
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250603115055.htm'>Epilepsy is more common in patients with frontotemporal dementia than expected</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 18:14:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>According to a recent study, in patients with frontotemporal dementia (FTD), epileptic seizures are significantly more common than previously known. Epilepsy is known to be associated with Alzheimer's disease, for example, but data on the on the connection between FTD and epilepsy remains scarce. The newly published study fills this information gap and shows that epilepsy is considerably more common in patients with FTD than previously estimated. "Our results show that epilepsy is considerably more common among those with FTD than those with Alzheimer's disease or in healthy controls," says Doctoral Researcher Annemari Kilpeläinen, the first author of the research article and a medical specialist in neurology. "It is noteworthy that epilepsy occurred in some patients with FTD already ten years before their dementia diagnosis, and it was more common in all the examined stages of the disease than previous international studies have reported." The prevalence of epilepsy was assessed at several time points from ten years before to five years after the dementia diagnosis. In addition to epilepsy diagnoses, medications used for epilepsy were more common in patients with FTD, which strengthens the reliability of the results. This may lead to underdiagnosis and delays in treatment. "Identifying epilepsy is important because its treatment can improve patients' functional capacity and quality of life. Knowledge about the association between epilepsy and FTD raises new research questions: do these diseases share some pathophysiological mechanisms and could some FTD symptoms be caused by alterations in the specific electrical systems of the brain," says the project's principal investigator, Associate Professor and Director of UEF Brain Research Unit Eino Solje. An extensive research project brings together different fields of science The recently published study is part of an extensive project that combines exceptionally extensive real-life patient data with different kinds of unique registers. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250603114813.htm'>Baboons walk in line for friendship, not survival, new study finds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 17:31:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at Swansea University have discovered that baboons walk in lines, not for safety or strategy, but simply to stay close to their friends. Some proposed that the order was random, while others argued that baboons strategically positioned themselves, with vulnerable individuals walking in the center to reduce their risk of attack. Now, using high-resolution GPS tracking, researchers from Swansea University have re-examined this behavior in a group of wild chacma baboons (Papio ursinus) on South Africa's Cape Peninsula. Their findings, published in the journal Behavioral Ecology, reveal that baboon movement patterns are driven by social bonds rather than survival strategies. Dr Andrew King, Associate Professor at Swansea University said: "Surprisingly, the consistent order we see for the baboons we studied isn't about avoiding danger like we see in prey animals when they position themselves in the middle of their social group, or for better access to food or water like we see in like we see in the movements of plains zebra. Instead, it's driven by who they're socially bonded with. They simply move with their friends, and this produces a consistent order. During these group movements -- like heading to a familiar sleeping spot -- it's likely that the group already knows where they're going. In this case, the consistent travel patterns among baboons emerge naturally from their social affiliations with each other, and not as an evolved strategy for safety or success. Marco Fele, the study's lead author and PhD student at Swansea University, said: "We know that strong social bonds are important for baboons -- they're linked to longer lives and greater reproductive success. But in this context, those bonds aren't serving a specific purpose. Our study highlights the potential for these kinds of spandrels in collective animal behaviour." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01297-8'>Cysteine depletion triggers adipose tissue thermogenesis and weight loss</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 13:41:57
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Caloric restriction and methionine restriction-driven enhanced lifespan and healthspan induces ‘browning' of white adipose tissue, a metabolic response that increases heat production to defend core body temperature. However, how specific dietary amino acids control adipose thermogenesis is unknown. Here, we identified that weight loss induced by caloric restriction in humans reduces thiol-containing sulfur amino acid cysteine in white adipose tissue. Systemic cysteine depletion in mice causes lethal weight loss with increased fat utilization and browning of adipocytes that is rescued upon restoration of cysteine in diet. Mechanistically, cysteine-restriction-induced adipose browning and weight loss requires sympathetic nervous system-derived noradrenaline signalling via β3-adrenergic-receptors that is independent of FGF21 and UCP1. In obese mice, cysteine deprivation induced rapid adipose browning, increased energy expenditure leading to 30% weight loss and reversed metabolic inflammation. These findings establish that cysteine is essential for organismal metabolism as removal of cysteine in the host triggers adipose browning and rapid weight loss. The Comprehensive Assessment of Long-term Effects of Reducing Intake of Energy (CALERIE-II) clinical trial in healthy adults demonstrated that a simple 14% reduction of calories for 2 years without specific dietary prescription to alter macronutrient intake or meal timings can reprogramme the immunometabolic axis to promote healthspan1,2,3. Thus, harnessing the pathways engaged by caloric restriction (CR) in humans may expand the current armament of therapeutics against metabolic and immune dysfunction. Induction of negative energy balance and resultant activation of mitochondrial fatty acid oxidation by CR is thought to underlie some of its beneficial effects on healthspan1. However, it has also been suggested that CR-induced metabolic effects may be due to decreased protein intake in food-restricted animal models4,5. Adding back individual amino acids to calorie-restricted Drosophila abolished the longevity effects, and traced to the limitation of methionine, an important node for lifespan extension5. Indeed, methionine restriction (MR) in rodents increases lifespan6 with enhanced insulin sensitivity, adipose tissue thermogenesis and mitochondrial fatty acid oxidation7. Surprisingly, in long-lived Drosophila fed an MR diet, adding back methionine did not rescue the pro-longevity effect of diet, and it was hypothesized that activation of the methionine cycle may impact longevity5. Commercial MR diets contain 0.17% methionine compared with normal levels of 0.86%, but notably, the MR diets also lack cystine8,9, another sulfur-containing amino acid (SAA), which is required for protein synthesis, including synthesis of glutathione, taurine, CoA and iron–sulfur clusters10,11. Of note, in rats, MR-induced anti-adiposity and pro-metabolic effects, including reduction of leptin, insulin, IGF1 and elevation of adiponectin, were reversed when animals were supplemented with cysteine in the diet12. Furthermore, cysteine supplementation in MR rats did not restore low methionine, suggesting no increase in the methionine cycle12, where homocysteine is converted into methionine via the enzyme betaine-homocysteine S-methyltransferase (BHMT)10. Cysteine is an ancient molecule that evolved to allow early life to transition from anoxic hydrothermal vents into oxidizing cooler environment14,15. Thus, cysteine, the only thiol-containing proteinogenic amino acid, is essential for disulfide bond formation and redox signalling, including nucleophilic catalysis10,13. It remains unclear whether cysteine specifically controls organismal metabolism and whether sustained CR in healthy humans can help understand the fundamental relationship between energy balance and SAA homeostasis pathways that converge to improve healthspan and lifespan. Adipose tissue regulates organismal metabolism by orchestrating inter-organ communication required for healthy longevity. To study the mechanisms that drive CR's beneficial effects on human metabolism, we conducted an unbiased metabolomics analysis of the subcutaneous adipose tissue (SFAT) of participants in the CALERIE-II trial at baseline and 1 year after 15% achieved CR and weight loss1,2,3. The partial least squares discriminant analyses (PLSDA) of abdominal SFAT biopsies revealed that 1 year of mild sustained CR substantially altered the adipose tissue metabolome (Fig. The unbiased metabolite sets enrichment analyses demonstrated significant increases in cysteine, methionine and taurine metabolism, which indicates rewiring of cysteine metabolism (Fig. Further analyses of our previously reported RNA sequencing data of humans that underwent CR1,2. revealed that compared to baseline, 1 and 2 years of CR in humans increased the adipose expression of CTH (Fig. 1d) with a concomitant reduction in the expression of BHMT (Fig. These data suggest that rewiring of cysteine metabolism may involve reduction in methionine cycle and changes in trans-sulfuration pathway metabolites (Fig. Notably, previous studies have found that long-lived rodents upregulate metabolites in TSP that generates cysteine from methionine16,17. Consistent with our findings in humans that underwent CR, data from multiple lifespan-extending interventions in rodents identified upregulation of CTH as a common signature or potential biomarker of longevity18. b, Metabolite set enrichment analysis shows that compared to baseline, 1 year of CR in humans activates TSP, with increased cysteine and taurine metabolism. c, Schematic summary of TSP and metabolites from baseline to 1 year CR, measured in human SFAT. Blue lines indicate unchanged metabolites, green and red arrows indicate significantly increased or decreased metabolites or genes respectively, via paired t-test (P < 0.05). d,e, Normalized expression of changes in CTH and BHMT in human SFAT at baseline, and after 12 and 24 months of CR. Adjusted P values were calculated in the differential gene expression analysis in a separate cohort from metabolome analyses in the CALERIE-II trial1 (n = 8). f,g, Change in metabolites in human SFAT at baseline and 12 months of CR. Significance was calculated using paired t-tests (n = 14). h, Mouse model used to achieve cysteine deficiency utilizing Cth−/− mice fed a CysF diet. Per cent body weight represented over 6 days of diet. j, Cth−/− mice were fed purified control diet (black line) or a diet containing 75% cysteine (green line) alternately switched to CysF diet (green line with red dots n = 6 per group). k, Box plots of metabolites involved in TSP in the serum of Cth−/− mice fed CTRL or CysF diet for 6 days (n = 4 Cth−/− CTRL, n = 5 Cth−/− CysF). l, Schematic summary of changes in the metabolites in the serum of Cth−/− mice fed CTRL or CysF diet for 6 days. Blue lines represent measured, but unchanged metabolites, red and green arrows indicate significantly decreased or increased metabolites, respectively (P < 0.05). See Supplementary Table 1 for the full list of metabolites. m, Total GSH content in subcutaneous (SFAT), brown (BAT) adipose depots and liver of Cth−/− mice fed with CTRL or CysF diet for 5 days (n = 7 per group), determined by colorimetric assay. n, Box plots of GSSG and threonine quantification in the SFAT of Cth−/− mice fed CTRL or CysF diet for 6 days (n = 6 per group). o,p, RNA-seq based expression of Gclc, Gss (o) and Bola3 (p) in the SFAT of Cth−/− mice fed with CTRL or CysF for 6 days. q, Coenzyme A (CoA) content in SFAT, BAT and liver samples of Cth−/− mice fed with CTRL or CysF diet for 5 days, determined by fluorometric assay (n = 7 per group, ND, not detectable). r, Analysis of EPR spectra of POBN-lipid radical adducts measured in Folch extracts of visceral adipose depot (VFAT), SFAT and BAT tissues from Cth−/− mice fed with CTRL or CysF diet for 5 days, normalized to 100 mg (ND, not detectable; n = 5–6 per group). s, Aconitase activity determined in SFAT, VFAT and BAT tissues from Cth−/− mice fed with CTRL or CysF diet for 5 days (n = 6 CTRL and 7 CysF). Data are represented as mean ± s.e.m. Whiskers are plotted down to the minimum and up to the maximum value. Unless mentioned, differences were determined with unpaired two-tailed t-tests. Panels c, h and l created with BioRender.com. Metabolomic analyses revealed that despite an increase in CTH expression post-CR, adipose cysteine levels were significantly reduced upon CR (Fig. 1f) with no change in homocysteine and cystathionine (Extended Data Fig. Consistent with the reduced expression of BHMT in methionine cycle, there was a decline in concentration of dimethylglycine (DMG) (Fig. CR caused a reduction in cysteine derived metabolites, γ-glutamyl–cysteine (γ-Glu–Cys), glutathione (GSH) and cysteinyl glycine (Cys–Gly) (Fig. Collectively, these results suggests that food restriction that resulted in 15% CR in humans reduces enzymes and metabolites that feed into methionine cycle and lowers cysteine (Fig. Cysteine is thought to be biochemically irreplaceable because methionine, the other sole proteinogenic SAA, lacks a thiol group and hence cannot form complexes with metals to control redox chemistry19. To determine whether cysteine is required for survival and organismal metabolism, we created a loss of function model where cysteine becomes an essential amino acid requiring acquisition from the diet by deletion of CTH (Cth−/− mice) (Fig. Cysteine deficiency was thus induced by feeding adult Cth−/− mice a custom amino acid diet that only lacks cystine (cystine-free (CysF) diet), whereas control mice were fed an isocaloric diet that contained cystine (CTRL diet) (Fig. Utilizing this model, we found that mice with cysteine deficiency rapidly lost ~25–30% body weight within 1 week compared to littermate Cth+/+ mice fed a CysF diet or Cth−/− fed a control diet (Fig. Upon clinical examination of the cysteine-deficient mice, 30% weight loss is considered a moribund state that required euthanasia. The weight loss in mice lacking CTH and cystine in the diet was associated with significant fat mass loss relative to lean mass (Extended Data Fig. Pair feeding of cystine depleted diet fed animals produced similar weight loss (Extended Data Fig. This rapid weight loss was not due to malaise or behavioural alteration, as Cth−/−CysF mice displayed normal activity and a slight reduction in food intake in the first 2 days after CysF diet switch that was not significantly different (Extended Data Fig. The Cth-deficient mice on the control diet were indistinguishable from control littermates in parameters indicative of health, they displayed higher nest building and no change in grip strength, gait, ledge test, hindlimb clasping and displayed no clinical kyphosis (Extended Data Fig. Furthermore, compared with Cth−/− mice on control diet, the analyses of liver, heart, lungs and kidneys of Cth−/−CysF mice did not reveal pathological lesions indicative of tissue dysfunction (Extended Data Fig. Notably, restoration of up to 75% cystine levels in the diet of Cth−/− CysF mice that were undergoing weight loss was sufficient to completely rescue the body weight over three weight-loss cysteine-depletion cycles, demonstrating the specificity and essentiality of cysteine for the organism (Fig. To identify changes in metabolites upon cysteine-deprivation-induced fat mass loss, we conducted serum and subcutaneous adipose tissue (SFAT) metabolomics analyses (Extended Data Fig. Compared with Cth-deficient mice fed a normal diet, the Cth−/−CysF mice had reduced cystine levels, suggesting that cysteine deficiency is maintained by a reduction in systemic cystine levels (Fig. Cysteine depletion also elevated the cystathionine and l-serine levels, compared to control diet fed animals (Fig. Other SAA metabolites such as methionine, homocysteine (HCys) and glutamic acid were not significantly changed (Extended Data Fig. Taurine levels in the Cth-deficient mice on a cystine-free diet also did not change compared with control animals. Notably, the γ-glutamyl peptide analogues of cysteine and GSH such as 2-aminobutyric acid (2AB) and ophthalmic acid (OA or γ-glutamyl-2-aminobutyryl-glycine) were increased in the serum of cysteine-deficient mice (Fig. Notably, in subcutaneous and brown adipose (BAT) depots and in liver, cysteine deficiency induces total glutathione (GSH) and oxidized GSH (GSSG) depletion (Fig. 2b,d).The increase in γ-glutamyl peptides (2AB and OA) in cysteine-limiting conditions in vivo is consistent with studies that show that GCLC can synthesize γ-glutamyl-2-aminobutyryl-glycine in a GSH-independent manner and prevents ferroptosis by lowering glutamate generated oxidative stress20. OA is a GSH analogue in which the cysteine group is replaced by l-2-aminobutyrate (2AB). On a normal cysteine-replete diet, 2AB is produced from 2-oxobutyrate (2OB) and glutamate in the presence of aminotransferases21. Thus, the increase in 2AB despite the removal of cystine in diet could be due to an alternative pathway of deamination of threonine into 2AB22. Indeed, l-threonine levels are increased upon cysteine depletion in mice (Fig. Previous studies found that GSH can inhibit glutamate cysteine ligase (GCLC)23,24 regulating its production by a feedback mechanism. Thus, the removal of cysteine and reduction of GSH may release this disinhibition (Fig. Consistent with this hypothesis and elevated OA levels, Gclc and Gss expression were increased in cysteine-starved mice (Fig. The increased OA production versus GSH production reveals adaptive changes induced by systemic cysteine deficiency. Cysteine is required for protein synthesis. However, in mice where cysteine is depleted for 5 days, protein synthesis was not impaired in subcutaneous, visceral, brown adipose depots and in the liver (Extended Data Fig. Cysteine is also required for Fe–S clusters21,25. The RNA sequencing analysis revealed that the expression of genes involved in Fe–S cluster assembly, including Bola3, was unimpaired or increased during cysteine depletion25 (Fig. Consistent with the association between increased Bola3 and adipose browning in a cysteine-deficient state, adipose-specific deletion of Bola3 decreases energy expenditure (EE) and increases adiposity in mice upon aging26. Upon short-term cysteine starvation, Fe–S clusters are preserved in adipose depots. Additionally, cysteine is crucial for coenzyme A (CoA) synthesis. We observed a significant reduction in CoA levels in BAT and liver upon cysteine starvation (Fig. The in vivo spin trapping and electron paramagnetic resonance (EPR) spectroscopy revealed that cysteine deficiency significantly increased lipid-derived radicals in BAT with undetectable signals in white adipose tissue (WAT) (Fig. Also, given aconitase is regulated by reversible oxidation of (4Fe–4S)2+ and cysteine residues, depletion of cysteine also reduced aconitase activity in SFAT with no change in BAT (Fig. Together, these data demonstrate that removing cysteine causes lethal weight loss and induces adaptive changes in organismal metabolism, including upregulation of Gclc, elevated γ-glutamyl peptides with depletion of CoA and GSH (Fig. 1l,m,q and Extended Data Fig. The decrease in fat mass during cysteine deficiency is driven by loss of all major fat depots including subcutaneous fat (SFAT), visceral epididymal/ovarian adipose fat (VFAT), and brown adipose tissue (BAT) (Extended Data Fig. Histological analyses revealed that this reduction in adipose tissue size is associated with transformation of white adipose depots into a BAT-like appearance, with the formation of multilocular adipocytes, enlarged nuclei, and high UCP1 expression, a phenomenon known as ‘browning' that increases thermogenesis27,28 (Fig. Of note, the SFAT browning in cysteine-deficient mice was reduced upon cystine-restoration in diet (Fig. Similar response was observed in visceral fat (VFAT) (Extended Data Fig. Consistent with the browning of SFAT, the cysteine-deficient animals showed significantly increased expression of uncoupling protein 1 (UCP1) (Fig. The UCP1 and adipose triglyceride lipase (ATGL) induction upon cysteine deficiency in adipose tissue was reversed by cysteine repletion (Fig. Consistent with 30% weight loss at day 5, the glycerol concentrations were depleted in the sera of cysteine-deficient mice and were restored by cysteine-repletion-induced weight regain (Extended Data Fig. The differentiation of Cth-deficient preadipocytes to mature adipocytes and subsequent exposure to cysteine-free culture conditions did not affect their thermogenic genes or UCP1, suggesting that a non-cell autonomous mechanism may control adipocyte browning (Extended Data Fig. a, Representative images of subcutaneous (SFAT) and visceral (VFAT) fat sections stained for UCP1 from Cth−/− mice fed CTRL or CysF diet for 6 days (scale bar, 100 μm). b, Representative H&E-stained sections of SFAT of Cth−/− mice fed CTRL or CysF diet for 6 days or CysF diet followed by Cys-supplemented diet for 4 days (CysF + Cys) (scale bar, 100 μm). c, Western blot detection of ATGL and UCP1 in SFAT from Cth−/− mice after 6 days of CTRL or CysF diet or Cys supplementation after CysF-induced weight loss. Actin is used as a loading control. d, qPCR analysis of thermogenic genes in SFAT of Cth+/+ and Cth−/− mice fed CysF diet for 6 days (n = 8 Cth+/+ and n = 10 Cth−/−). e,f, Faecal calorie content (n = 6 per group) (e) and cumulative food intake of Cth−/− mice fed CTRL or CysF diet for 4 days (f) (n = 6 per group). g, Linear regression analysis of EE against body mass during dark cycle at 4 and 5 days of weight loss (n = 10 Cth+/+ CysF and n = 12 Cth−/− CysF). h, Per cent body weight change of Cth−/− mice fed with CTRL diet or CysF diet (red line) for 5 days and then switched to Cys-containing diet (orange line) for 3 days (n = 6 per group). i, RER measured in metabolic cages, of Cth−/− mice fed with CTRL diet (n = 6) or Cys-containing diet after CysF-induced weight loss (n = 4). j, Average food intake of Cth−/− mice fed with CysF diet and then switched to Cys-containing diet for 2 days (n = 7 per group). Significance was measured with paired t-test. k, Linear regression analysis of EE against body mass during dark cycle of Cth−/− mice fed with CTRL (n = 6) or Cys-supplemented diet after CysF-induced weight loss (n = 4), average values of the first two nights after diet switch. l, t-SNE plot of scRNA-seq showing cluster identities from SFAT SVF from Cth−/− mice fed CTRL or CysF diet at day 4 of weight loss and bar chart showing population fold change (FC) in relative abundance of each cluster comparing Cth−/− CysF versus Cth−/− CTRL. m, t-SNE plot displaying Pdgfra expression in red across all populations and Monocle analysis of clusters 0, 1 and 2, with colouring by pseudotime to show right most cluster giving rise to two separate clusters. Each cluster represented by colour in Cth−/− CTRL and Cth−/− CysF. Statistical differences were calculated by two-way ANOVA with Sidak's correction for multiple comparisons or unpaired two-tailed t-tests. We next investigated whether changes in energy absorption, energy intake or EE contribute to cysteine-depletion-induced weight loss. Analysis of energy absorption by faecal bomb calorimetry revealed no significant difference in control and cysteine-deficient mice (Fig. Moreover, although the cumulative food intake over 5 days of weight loss was not statistically different (Fig. 3e) after switching to CysF diet was lower (P < 0.05), which may contribute to early weight loss. Calculation of the analysis of covariance (ANCOVA) or representation of the data as regression between EE and body mass, demonstrated that EE was increased in cysteine-deprived animals during the dark cycle (Fig. 2g) and not in the light cycle (Extended Data Fig. In addition, there was no difference in locomotor activity between control or cysteine-deficient mice (Extended Data Fig. 3h), suggesting that cysteine depletion increases EE. Of note, the increase in EE was supported by increased fat utilization, as the respiratory exchange ratio (RER) in cysteine-deprived animals was significantly reduced (Extended Data Fig. We next determined the specificity of cysteine on mechanisms that may contribute to rapid weight loss. Weight regain after cysteine repletion substantially reversed adipose browning (Fig. Furthermore, replacement of single amino acid cysteine, also reversed the cysteine-starvation-driven reduction in RER, suggesting restoration of organismal metabolism to normal carbohydrate utilization instead of fatty acid oxidation (Fig. Of note, cysteine repletion significantly increased food intake for the first two days, suggesting that animals can sense cystine in diet and compensate via hyperphagia to restore bodyweight setpoint (Fig. The EE upon cysteine replacement was not significantly different during weight rebound (Fig. These data suggest that cysteine replacement can reverse weight loss by mechanisms that involve reduced adipose browning and decreased fat utilization while change in EE alone does not account for rapid weight restoration. We conducted the RNA sequencing of the major adipose depots to investigate the mechanisms that control adipose tissue browning and associated remodelling. As displayed by the heatmap, cysteine deficiency profoundly altered the transcriptome of adipose tissue (Extended Data Fig. Gene set enrichment analysis comparing Cth−/− CTRL and Cth−/− CysF identified that the top downregulated pathways were involved in the extracellular matrix and collagen deposition, highlighting the broad remodelling of the adipose tissue (Extended Data Fig. In addition, multiple metabolic pathways seemed to be regulated by cysteine deficiency within the SFAT with ‘respiratory electron transport chain and heat production' as the top pathway induced during cysteine deficiency (Extended Data Fig. Indeed, numerous genes identified by the ‘thermogenesis' Gene Ontology term pathway such as Ucp1, Cidea, Cox7a1, Cox8b, Dio2, Eva1, Pgc1, Elovl3 and Slc27a2, were differentially expressed comparing Cth+/+ CysF and Cth−/− CysF in the SFAT (Extended Data Fig. These results demonstrate that cysteine depletion activates the thermogenic transcriptional programme. To investigate the cellular basis of adipose tissue remodelling during cysteine deficiency, we isolated the stromal vascular fraction (SVF) by enzymatic digestion and conducted single-cell RNA sequencing (scRNA-seq) of SFAT. We isolated SVF cells from Cth+/+ and Cth−/− fed CTRL or CysF diet with each sample pooled from four animals (Extended Data Fig. Comparison of Cth−/− CysF with other groups revealed dramatic changes in cellular composition (Fig. Particularly, loss of clusters 0, 1 and 2 were apparent upon cysteine deficiency (Fig. Of note, these clusters contained the highest numbers of differentially expressed genes induced by β3-adrenergic receptor agonist CL-316243 (ref. 4e), highlighting the potential role of sympathetic nervous system (SNS)-derived noradrenaline (NA) in regulating the effects of cysteine deficiency. By expression of Pdgfra, we identified these clusters as enriched for adipocyte progenitors (Fig. We conducted a pseudotime analysis to place these clusters on a trajectory and illuminate their cell lineage. Trajectory analysis based on pseudotime suggested that cluster 2 may differentiate into two separate preadipocyte clusters, clusters 0 and 1 (Fig. Cth−/− CysF animals proportionally lost clusters 0 and 1, while relatively maintaining cluster 2 compared with the other groups (Fig. 2m), suggesting that more differentiated preadipocytes are mobilized during cysteine deficiency. Indeed, cluster 2 expressed Dpp4, an early progenitor marker that has been shown to give rise to different committed preadipoctyes32 (Extended Data Fig. Cluster 0 was enriched for both Icam1 and F3, which are expressed by committed adipogenic and antiadipogenic preadipocytes, respectively29,32 (Extended Data Fig. Cd9, a fibrogenic marker in preadipocytes31,33, along with the collagen gene, Col5a3, were broadly expressed across clusters 0 and 1, and was specifically lost by day 4 of inducing cysteine deficiency (Extended Data Fig. The loss of these preadipocyte clusters were orthogonally validated by FACS (Extended Data Fig. We next sought to identify beige/brown adipocyte precursors in our scRNA-seq dataset to understand whether there was an increased commitment towards brown adipocytes. Clearly, Tagln or Sm22, which has been previously described in beige adipocytes34,35, is specifically expressed by a subset of cells in cluster 1 (Extended Data Fig. Of note, these Tagln-expressing cells are lost with cysteine deficiency (Fig. Given the strong browning phenotype observed on day 6, it is possible that these cells become mobilized and differentiate early on during cysteine deficiency, leading to the absence of these cells as mature adipocytes are not captured within the SVF. Indeed, when we performed pathways analysis on cluster1, comparing gene expression of Cth−/− CysF with Cth−/− CTRL, we found that one of the top upregulated pathways was ‘adipogenesis' (Extended Data Fig. Furthermore, examination of the expression of stem associated markers and mature adipocyte markers in the adipocyte progenitor clusters revealed a clear downregulation of stem markers and an increase in mature adipocyte markers, suggesting that cysteine deficiency was driving the maturation of progenitor cells (Fig. However, given the robust transformation of the adipose tissue during cysteine deficiency towards browning, it is unlikely that mobilization of brown precursors alone is mediating this response. Previous studies have found that in certain models, beige adipocytes can originate from pre-existing white adipocytes, in addition to de novo adipogenesis36. The potential role of cysteine in the trans-differentiation of mature white adipocytes into brown-like adipocytes needs to be further examined using future lineage-tracking studies. To determine the mechanism of adipose thermogenesis caused by cysteine starvation, we next investigated the processes upstream of increased fatty acid oxidation. We measured the lipolysis regulators, phosphorylation of hormone-sensitive lipase (pHSL) and ATGL and found that cysteine deficiency increased ATGL expression without consistently affecting pHSL levels (Fig. ATGL preferentially catalysers the first step of triglyceride hydrolysis, whereas the hormone-sensitive lipase (HSL) has a much broader range of substrates with a preference for diacylglycerols and cholesteryl esters37. Given a dramatic browning response in WAT post-cysteine deficiency, the increased ATGL is consistent with previous work that shows BAT relies heavily on the action of ATGL to mobilize lipid substrates for thermogenesis38. This is further supported by a decrease in most lipid species, particularly triglycerides and diacylglycerol in the BAT of cysteine-deficient mice (Fig. Considering dramatic adipose tissue browning and elevated UCP1 expression upon cysteine starvation, we next sought to investigate whether this is a homeostatic response to defend core body temperature (CBT) or whether temperature setpoint is perturbed to causes hyperthermia. We measured CBT utilizing loggers surgically implanted into the peritoneal cavity in Cth−/− mice on CTRL or CysF diet over 6 days when animals lose weight. Of note, despite conversion of WAT into brown-like thermogenic fat, the CBT was not different between control and cysteine-deficient mice (Extended Data Fig. These data suggest that either cysteine- may signal the host to defend CBT within tight normal physiological range or any metabolic heat that is generated is dissipated due to the animal housing in the sub-thermoneutral temperature. To confirm adipose thermogenesis in vivo, we utilized a highly sensitive and specific magnetic resonance spectroscopic imaging method called biosensor imaging of redundant deviation in shifts (BIRDS)39 to determine the temperature of BAT in Cth+/+ and Cth−/− animals after 6 days of CysF diet. This method relies on measuring the chemical shift of the four nonexchangeable methyl groups from an exogenous contrast agent, TmDOTMA, which has a high-temperature sensitivity (0.7 ppm per °C). The TmDOTMA− methyl resonance has ultrafast relaxation times (<5 ms), allowing high signal-to-noise ratio by rapid repetition for superior signal averaging39. Compared to cysteine-replete animals, the in vivo local temperature in BAT of cysteine-deficient mice was significantly greater than surrounding tissue (Fig. a, Western blot detection of lipolysis regulators pHSL, HSL and ATGL in SFAT from Cth−/− mice after 6 days of CTRL or CysF diet, actin is used as loading control. c,d, in vivo measurement of BAT temperature by BIRDS imaging (c) and quantification of local temperature differences in BAT (d) compared to surrounding tissue in Cth+/+ and Cth−/− mice on CysF diet for 6 days (n = 5 per group). f, Serum GDF15 concentrations in Cth−/− CTRL, Cth−/− CysF for 4 days and Cth−/− CysF followed with 3 days of Cys supplementation (n = 6 per group). g, Immunoblot analysis of CHOP, calnexin, IRE1a, BiP in the liver of Cth−/− mice fed with CTRL or CysF diet at day 6. Actin was used as loading control. h, Percentage body weight change of Cth−/− and Cth−/−CHOP−/− mice fed with CysF diet for 5 days (n = 17 Cth−/− and n = 15 Cth−/−CHOP−/−). i, Percentage body weight change of Cth−/− and Fgf21−/−Cth−/− mice fed with CysF diet for 5 days (n = 13 Cth−/− and n = 18 Fgf21−/−Cth−/−). j, Energy expenditure measured in metabolic cages of Cth−/− and Cth−/− Fgf21−/− mice on days 3–4 of CysF diet (n = 5 per group). k, Immunoblot analysis of pHSL, HSL, ATGL and UCP1 in SFAT of Cth+/+, Cth−/− and Cth−/−Fgf21−/− mice fed CysF diet for 6 days. l, Representative H&E-stained SFAT sections of Cth−/− and Fgf21−/−Cth−/− mice after 6 days of CysF diet (scale bar, 500 μm). m–p, Cth+/+ and Cth−/− mice were fed with CysF diet and housed at 20 °C or 30 °C for 6 days. Statistical differences were calculated by one-way ANOVA with Tukey's correction for multiple comparisons or two-way ANOVA with Sidak's correction for multiple comparisons or unpaired two-tailed t-tests. Changes in nutritional stress induced by CR, MR or low-protein diets upregulate the expression of FGF21, which, when overexpressed, increases lifespan and also upregulates EE40,41. 3e) and Fgf21 expression in the liver (Extended Data Fig. 5f), which was reversed by cysteine-repletion-induced weight restoration (Fig. Similar to FGF21, the hormone GDF15, can also be induced by cellular or nutritional stress-mediated signalling42. Cysteine depletion at day 4 after weight loss significantly increased GDF15, which was not restored after cysteine-repletion-induced weight regain (Fig. Given the cysteine-repleted diet switch increases food intake, the higher GDF15 levels during weight rebound are likely insufficient to cause food aversion. Notably, recent studies suggest elevated endoplasmic-reticulum (ER) stress in Bhmt−/− mice with reduced methionine cycle, is associated with increased FGF21 and adipose browning43. Cysteine deficiency led to induction of ER-stress proteins CHOP, calnexin, IRE1α and BIP (Fig. However, deletion of CHOP in cysteine-starved Cth−/− mice did not rescue weight loss (Fig. 3h) nor did it alter the FGF21 and GDF15 serum levels (Extended Data Fig. 5g,h) demonstrating that induction of the CHOP-dependent ER-stress response does not drive cysteine's metabolic and neuroendocrine effect. Given that cysteine specifically regulated FGF21 during weight loss and regain (Fig. 3e), we generated Fgf21−/−Cth−/− double knockout (DKO) mice to test whether FGF21 controls adipose browning and weight loss in cysteine-starved mice. In the absence of FGF21, cysteine deficiency-induced weight loss and reduction in adiposity in Cth−/− mice were blunted, but the weight-loss trajectory continued and was not rescued (Fig. The Fgf21−/−Cth−/− DKO mice had lower EE compared to Cth−/− mice on the CysF diet (Fig. However, the RER was not different, indicating that Fgf21−/−Cth−/− mice still substantially utilized fat as an energy source (Extended Data Fig. This was supported by maintenance of lipolysis signalling observed by levels of pHSL and ATGL in Cth−/− mice, but reduced UCP1 protein and mRNA expression in WAT of Fgf21−/−Cth−/− (Fig. Of note, the WAT of Fgf21−/−Cth−/− DKO mice maintained classical multilocular browning characteristics (Fig. 3l) suggesting that FGF21 is not required for adipose browning. These results suggest that FGF21 is partially required for weight loss but does not mediate lipid mobilization or adipose browning caused by cysteine deficiency. Cysteine deprivation revealed a metabolic crisis that may signal the host to activate thermogenic mechanisms. However, across animal vivarium, including ours, mice are housed at sub-thermoneutral 20 °C temperatures and are constantly under thermogenic stress due to slight cold challenge28. To further confirm that mice were indeed inducing thermogenesis to defend CBT, we housed cysteine-deficient animals at 30 °C thermoneutrality. The cysteine deficiency in Cth−/− mice housed at 30 °C also led to similar weight loss as 20 °C with significant browning of adipose tissue (Fig. The degree of browning and gene expression of Ucp1 and Elovl3 in CysF Cth-deficient mice at thermoneutrality were relatively lower than inductions observed at 20 °C (Fig. Furthermore, expression of genes involved with lipid regulation and browning such as Prdm16, Ppargc1a, Ppara, Pparg and Cpt1 (Fig. 3p) were significantly increased in SFAT, suggesting that even at thermoneutral temperatures, Cth−/− CysF-fed mice activate fat metabolism and have increased thermogenesis caused by cysteine deficiency. In addition, compared to controls, the cysteine-deficient mice at thermoneutrality retained higher Ucp1 expression in BAT (Extended Data Fig. Together, cysteine-depletion-induced weight loss and adipose browning are maintained at thermoneutrality. The liver is believed to be the primary organ responsible for maintaining systemic cysteine homeostasis10,13. Immunoblot analyses confirmed the highest CTH expression in the liver, followed by the kidney, thymus and adipose tissue (Extended Data Fig. Given that CR in humans lowers cysteine levels in adipose tissue; we generated adipocyte- and hepatocyte-specific Cth-deficient mice to explore the cell-type-specific mechanism of cysteine in weight loss (Fig. As expected, liver-specific deletion of Cth did not alter CTH expression in the kidney and adipose-specific ablation of Cth maintained the expression in the liver (Extended Data Fig. Neither liver- nor adipose-specific deletion of Cth led to a reduction in serum cysteine levels (Fig. 4c,d,g,h and Extended Data Fig. 6c,d) or caused fat mass loss when cysteine was restricted in the diet (Fig. a, Immunoblot analyses of CTH in the liver of male and female Cthf/f Alb:Cre− or Alb:Cre+ mice. b, Western blot detection of CTH in the SFAT of male and female Cthf/f Adipoq:Cre− or Adipoq:Cre+ mice. c,d, Serum cysteine and cystine determined by LC–MS/MS in Alb:CreCthf/f mice (n = 5 per group) (c) and Adipoq:Cre;Cthf/f mice (n = 4 CTRL and n = 5 CysF) (d) after 6 days of CTRL or CysF diet. e,f, Percentage body weight changes of Alb-Cre;Cthf/f mice (n = 5 Cthf/f CTRL, n = 6 Cthf/f CysF and n = 3 Alb-Cre;Cthf/f CTRL and CysF) (e) and Adipoq-Cre;Cthf/f mice (n = 5 per group) after 6 days of CTRL or CysF diet (f). g,h, Volcano plot of serum metabolites identified by LC–MS/MS in Alb-Cre;Cthf/f mice (n = 5 per group) (g) and Adipoq-Cre;Cthf/f mice (n = 4 CTRL and n = 5 CysF) (h) after 6 days of CTRL or CysF diet. Trans-sulfuration pathway related metabolites are highlighted in red. Significantly increased or decreased metabolites (−log10(P) >1.3 and ∣log2(FC)∣>1) are highlighted in blue and listed on the right. Supplementary Tables 3 and 6 provide the full list of metabolites. i–l, Cth−/− and Cth−/− Ucp1−/− mice were fed a CysF diet for 6 days (n = 8 per group). Per cent body weight change over 6 days of diet (i). Energy expenditure measured in metabolic cages on days 4 and 5 of CysF diet (k). Linear regression analysis of EE against body mass during dark and light cycles at 4 and 5 days of weight loss (when adjusted to body mass covariate, EE of Cth−/− Ucp1−/− is significantly decreased, during both night and day) (l). m, CBTs measured in the peritoneal cavity by implantation of Star-Oddi loggers over 6 days of diet in male Cth−/− and Cth−/−Ucp1−/− mice fed CysF diet. n,o, Immunoblot staining of ATGL, TH and UCP1 in BAT of Cth−/− and Cth−/−Ucp1−/− fed a CysF diet for 6 days (n) and quantification using tubulin as loading control (o). p, Thermogenic markers gene expression analysis in BAT of Cth−/− and Cth−/−Ucp1−/− mice fed a CysF diet for 6 days, measured by qPCR (n = 8 Cth−/−, n = 10 Cth−/−Ucp1−/−). q, Gene expression of genes involved in futile creatine cycle in BAT of Cth−/− and Cth−/−Ucp1−/− mice fed a CysF diet for 6 days (n = 16 Cth−/−, n = 15 Cth−/−Ucp1−/−), quantified by qPCR. Statistical differences were calculated by two-way ANOVA with Sidak's correction for multiple comparisons or by unpaired two-tailed t-tests. Further liquid chromatography–mass spectrometry (LC–MS) analyses of sera from hepatocyte-specific Cth-deficient mice maintained on CysF diet showed no changes in cystathionine, γ-glutamyl-dipeptides, cysteine or cystine (Fig. Consistent with low CTH activity, livers of the CysF-fed mice (Alb-Cre:Cthf/f, CysF) had lower levels of cysteine, cystathionine, S-adenosyl homocysteine, 2AB and ophthalmate (Extended Data Fig. Of note, the levels of cystathionine and cysteine/cystine in subcutaneous adipose tissue of liver-specific Cth-deficient mice were unchanged (Extended Data Fig. Consistent with these data, no change in serum cysteine/cystine were detected in adipose tissue specific Cth−/− mice that had no weight loss on a cysteine-free diet (Fig. The Cth−/− animals co-housed together with Cth+/+ mice still maintained weight loss when fed a CysF diet, suggesting that microbiota derived metabolites do not account for the weight loss (Extended Data Fig. These results demonstrates that Cth across multiple tissues may defend systemic cysteine pool to prevent uncontrolled thermogenesis and death when cysteine content is low in diet. Given that UCP1 is a canonical regulator of non-shivering adipose thermogenesis44,45 and as cysteine elimination induced UCP1 expression in WAT, we next deleted UCP1 in cysteine-deficient mice to determine its role in adipose browning. Notably, we found that Cth−/−Ucp1−/− double knockout (DKO) mice had equivalent food intake (Extended Data Fig. 6l), lost weight at a similar rate to its Cth−/− littermates on a CysF diet and displayed similar browning-like features with multilocular adipocytes (Fig. The ablation of UCP1 in cysteine-deficient mice lowered EE during dark and light cycles (Fig. 4k,l) but did not affect the CBT (Fig. Despite similar body weight loss and browning phenotype, Cth−/− and Cth−/−Ucp1−/− DKO mice show significant differences in the interaction between EE and body weight (Fig. Therefore, UCP1 depletion and other parameters such as body composition may account for this significant difference in the correlative trend between EE and body weight. The lack of UCP1 in Cth-deficient mice undergoing cysteine starvation displayed elevated ATGL and tyrosine hydroxylase (TH) expression, suggesting increased lipolytic signalling (Fig. Despite lack of UCP1, gene expression indicative of the thermogenic programme, such as Ppargc1, Cidea and Cpt1 are significantly increased in Cth−/−Ucp1−/− DKO mice compared to Cth−/− in the BAT after 6 days of CysF diet (Fig. Furthermore, gene expression of other mediators of the thermogenic genes such as Acadm, Cox7a1, Elovl3 and Slc27a2 are also significantly increased in Cth−/−Ucp1−/− DKO mice compared to Cth−/− animals fed cysteine-restricted diet (Fig. The futile creatine cycle is proposed to regulate UCP1-independent thermogenesis47. Compared to control animals, the creatine cycle genes Ckb and Alpl were not significantly different in SFAT of cysteine-deficient animals (Extended Data Fig. The creatine synthesis genes, Gatm and Gamt were significantly reduced with cysteine deficiency in SFAT (Extended Data Fig. The expression of one of the creatine kinases that utilizes ATP, Ckmt2 and the transporter for creatine, Slc6a8 were also not differentially regulated in SFAT (Extended Data Fig. Notably, Slc6a8, Ckmt1 and Ckmt2 expression was increased in BAT of Cth−/−Ucp1−/− animals compared to cysteine-deficient animals (Fig. 4q), suggesting a potential role of these effectors in BAT thermogenesis in the cysteine-starvation model. The alternative UCP1-independent thermogenic regulatory genes Atp2a2 and Ryr2 that control calcium cycling48 were not impacted by cysteine deficiency (Extended Data Fig. Similarly, Sarcolipin and Atp2a2, which can increase muscle driven thermogenesis49 were also not affected in skeletal muscle of Cth-deficient mice lacking cysteine (Extended Data Fig. The futile lipid cycle is also implicated in UCP1 independent thermogenesis50. Of note, Cth−/− mice on a CysF diet have significantly elevated expression of Dgat1, Pnpla2 and Gk with no change in Lipe in SFAT (Extended Data Fig. The expression of these genes was also induced in absence of UCP1 in BAT and SFAT (Extended Data Fig. However, absence of association between changes in gene expression of major UCP1 independent regulators does not rule out causal role of some of these mechanisms in cysteine-deprivation-driven adipose browning. These results suggest that systemic cysteine deficiency-induced thermogenesis depends mainly on an as-of-yet unknown non-canonical UCP1-independent thermogenic mechanism. As cysteine-elimination-induced adipocyte browning is non-cell autonomous (Extended Data Fig. 3d), we evaluated upstream mechanisms that control cysteine-starvation-induced thermogenesis. We conducted an unbiased whole-brain activity mapping screen to determine the circuitry responsible for regulating the thermogenic induction. Using whole-brain immunolabelling and clearing tandem iDISCO+51, we mapped the differential expression of the immediate–early gene c-Fos (Fig. Subsequently, we quantified c-Fos positive cells across brain regions and registered them to the Allen Brain Atlas with CLEARMAP52. Notably, key components of the canonical thermogenesis circuitry (Fig. 5a–d) were significantly activated upon 5 days of exposure to a cysteine-free diet. a, Tissue clearing and whole-brain c-Fos immunolabelling approach using iDISCO+ and CLEARMAP in Cth−/−mice fed CTRL or CysF for 5 days. LPBN, lateral parabrachial nucleus; MPOA, medial preoptic area; DMH, dorsomedial hypothalamus; PVH, paraventricular hypothalamus; RPA, raphe palladus; DRN/vlPAG, dorsal raphe nucleus/ventrolateral portion of the periaqueductal grey. c, Automated analysis of c-Fos+ cell distribution in Cth−/− brains collected after 5 days CTRL (n = 6) or CysF feeding (n = 5). Panels show the reference annotation (Allen Brain Atlas; ABA) with details from the averaged density maps (5–6 brains averaged) between the two conditions, P value maps (25-μm orthogonal projection) for the canonical thermogenic regions in the brain as coronal projections. Third, fourth and fifth lanes show DMH, BNST and the vlPAG, respectively; three critical sites for transmitting information received by the MPOA to the SNS-mediated thermogenic outflow. d, Quantification of c-Fos staining in the parabrachial nucleus, MPOA, DMH and BNST of Cth−/− mice after 5 days of CTRL (n = 6) or CysF feeding (n = 5). e) Measurement of noradrenaline by orbitrap MS/MS in the SFAT of Cth+/+ and Cth−/− fed 6 days of CTRL or CysF diet (n = 5 Cth+/+ CTRL, n = 5 Cth+/+ CysF, n = 6 Cth−/− CTRL, n = 6 Cth−/− CysF). f, qPCR gene expression Maoa in SFAT of Cth+/+ (n = 8) and Cth−/− (n = 10) mice fed with CysF diet for 6 days. g,k, Cth−/− mice were fed with CysF diet for 5 days and treated daily with a β-3 adrenergic receptor antagonist (L748337) or vehicle (PBS) (n = 7 per group). h, Representative images of hematoxylin and eosin (H&E) staining of SFAT sections (scale bar, 50 μm). i, qPCR gene expression of Ucp1 in BAT depots. j, Immunoblot analysis of lipolysis regulators (pHSL, HSL and ATGL) in BAT samples. Actin is used as a loading control. k, Quantification of pHSL and ATGL signals (n = 6 per group). Whiskers are plotted down to the minimum and up to the maximum value. Statistical differences were calculated by two-way ANOVA with Sidak's correction for multiple comparisons or by unpaired two-tailed t-tests. Panels a and b created with BioRender.com. In brief, thermogenic signals seem to converge in the dorsal lateral parabrachial nucleus (LPBN), a critical hub that integrates inputs related to both environmental temperature changes and internal metabolic shifts53. From the LPBN, thermogenic signals are relayed to the medial preoptic area (MPOA), a key sensory integrator that regulates thermogenesis54,55. The MPOA then initiates a thermogenic response primarily through activation of the SNS. This response can be mediated by direct monosynaptic projections from the MPOA to premotor regions involved in sympathetic activation or through polysynaptic pathways involving the dorsomedial hypothalamus (DMH), bed nucleus of the stria terminalis (BNST) and the ventrolateral periaqueductal grey (vlPAG) adjacent to the dorsal raphe nucleus (DRN)56,57,58. All these regions were significantly activated (Fig. By activating this well-established thermogenic circuitry, cysteine deficiency induces a potent metabolic response, highlighting a critical mechanism by which systemic amino acid depletion can modulate EE and adipose browning. This underscores the broader physiological relevance of cysteine metabolism in energy homeostasis and thermoregulation. Upstream of lipolysis, nonshivering thermogenesis is mainly activated by the SNS-derived adipose noradrenaline59. Mass-spectrometric analyses of subcutaneous adipose tissue (Fig. 5e), including imaging mass spectrometry of BAT (Extended Data Fig. 7a) revealed that cysteine-starvation-induced browning is associated with increased NA concentrations. This was coupled with a significant reduction in NA-degrading enzyme monoamine oxidase-a (Maoa) (Fig. 5f), without affecting catechol-o-methyl transferase (Comt), suggesting increased adipose NA bioavailability (Extended Data Fig. Finally, to test whether SNS-derived NA is required for adipose browning, the inhibition of β3-adrenergic receptors (ADRB3) by L748337 in Cth-deficient mice lacking cysteine-protected animals against weight loss (Fig. 5g), despite having a similar food intake (Extended Data Fig. Inhibition of β3-adrenergic signalling blunted adipose browning (Fig. 5i), as well as lipolysis inducers pHSL and ATGL that are downstream of ADRB3 signalling (Fig. This was consistent with our unbiased RNA sequencing analyses that showed that cysteine-regulated adipose clusters contained the highest numbers of differentially expressed genes induced by β3-adrenergic receptor agonist (Extended Data Fig. Together our findings suggest that cysteine-depletion-induced browning is non-cell autonomous and lack of cysteine drives increased SNS activity leading to augmented ADRB3-mediated NA signalling that controls adipose browning to weight loss. We next tested whether cysteine deficiency could be utilized to induce an adaptive thermogenic mechanism for fat mass reduction in the high-fat diet (HFD)-induced obesity model. The Cth−/− mice that had been fed HFD for 12 weeks were switched to an isocaloric HFD containing (HFD-CTRL) or lacking cystine (HFD-CysF). The Cth−/− mice fed a HFD-CysF diet were able to lose approximately 30% body weight within 1 week despite maintaining a high calorie intake (Fig. This weight loss was associated with major reductions in fat mass (Extended Data Fig. With weight loss, cysteine-deficient mice had reduced fasting glucose levels, improved glucose tolerance (Fig. Furthermore, cysteine deficiency in obese mice reduced RER suggesting higher fat utilization (Fig. Notably, immunohistological analysis of the white adipose depots demonstrated that cysteine deficiency induced browning even while on HFD with increased expression of UCP1 in SFAT and VFAT (Fig. Additionally, consistent with improvement of metabolic function in obesity, the gene expression of inflammasome components Il1b, Il18, Nlrp3, Casp1 and pro-inflammatory cytokines Il6 and Tnf were reduced in F4/80+CD11b+ adipose tissue macrophages in visceral adipose tissue (Fig. 6g) These results demonstrate that induction of cysteine deficiency can cause rapid weight loss in mouse model of diet-induced obesity, opening new avenues for future drug development for excess weight loss. Cth−/− mice that had been fed HFD for 12 weeks were switched to HFD-CTRL or HFD-CysF. b, Fasting blood glucose measured 1 week post diet switch (Cth−/− HFD-CTRL n = 19, Cth−/− HFD-CysF, n = 20). c, The glucose tolerance test (GTT) in Cth−/− after diet switch from HFD-CTRL to HFD-CysF (Cth−/− HFD-CTRL n = 19, Cth−/− HFD-CysF, n = 20). The glucose administration is based on total body weight. d, EE of Cth−/− mice fed with HFD-CTRL or HFD-CysF, average values of nights 4 and 5 of diet switch (n = 6 Cth−/− HFD-CTRL, n = 5 Cth−/− HFD-CysF). e, RER measured in metabolic chambers on days 4 and 5 of diet switch (n = 6 Cth−/− HFD-CTRL, n = 5 Cth−/− HFD-CysF). f, Representative histological sections of SFAT and VFAT stained for UCP1, 6 days after diet switch. g, qPCR analysis of inflammatory genes in CD11b+ F4/80+ VFAT macrophages of Cth−/− mice after diet switch to HFD-CTRL or HFD-CysF (n = 4 per group). Statistical differences were calculated by two-way ANOVA with Sidak's correction for multiple comparisons or by unpaired two-tailed t-tests. Adipose tissue regulates metabolism by orchestrating inter-organ communication required for healthy longevity60. Analyses of adipose tissue of humans that underwent moderate CR in free-living conditions have highlighted genes and pathways that link energy metabolism and inflammation to influence healthspan1,2. In rodents, restriction of calories up to 40% reduces CBT and induces browning of the adipose tissue of mice reared in sub-thermoneutral temperature61. The CR in humans upregulated the fatty acid oxidation and futile lipid cycling induced-thermogenic pathways but UCP1 was undetectable in adipose tissue of CALERIE-II participants1. Similarly, weight loss in obese humans is not associated with classical UCP1 adipose tissue browning62. This suggests that alternative UCP1-independent mechanisms maybe at play in human in response to CR. Our studies demonstrated that reduction of cysteine and subsequent rewiring of downstream cysteine metabolism is linked to adipose browning and weight loss. Expression and activity of the TSP genes CBS and CTH increase when cysteine is low10. Indeed, during CR, the TSP is induced to defend against the depletion of cysteine levels. MR regimens that improve lifespan are also restricted or deficient in cysteine12, and it is unclear whether MR or cysteine restriction drives pro-longevity effects. Thus, to understand the metabolic requirement of dietary non-essential amino acid such as cysteine, a genetic mouse model is required that lacks Cth in conjunction with restriction of cysteine. Notably, previously reported Cth mutant mice originally generated on a 129SvEv mouse strain maintained on cysteine-replete normal chow diet were reported to display hypertension and motor dysfunction characteristic of neurodegenerative changes in corpus striatum64,65. Using this same model cysteine depletion in this global Cth-deficient mice also causes weight loss and adipose tissue browning together with decrease of CoA levels similar to our findings66. We also demonstrate that conditional deletion of Cth (on a pure C57/B6 background) in adipose tissue and liver is not sufficient to induces weight loss and cause adipose browning as other tissues compensate to maintain to systemic cysteine concentrations after cysteine elimination in the diet. Our data establish that systemic cysteine depletion drives adipose tissue thermogenesis without causing behavioural defects or pathological lesions. While it is still unclear why cysteine deficiency triggers the activation of adipose browning, the mechanism of thermogenesis depends on upstream SNS-mediated sympathetic β3-adrenergic signalling and partially requires FGF21 and can be successfully maintained even in the absence of UCP1 and at thermoneutrality. Future studies of specific ablation of UCP1-independent thermogenic genes in Cth−/− mice on cysteine restriction are required to determine the causal downstream pathway that causes thermogenesis. The model of cysteine-deprivation-induced strong browning response may thus allow the discovery of an alternate UCP1-independent mechanism of adipose tissue thermogenesis. In healthy humans undergoing CR, consistent with reduced cysteine, glutathione, a major redox regulator, was reduced in adipose tissue. The Cth-deficient mice on a cysteine-free diet show a decrease in CoA and GSH with a compensatory increase in Gclc, Gss and accumulation of γ-glutamyl-peptides. Despite increased oxidative stress, the adipose tissue histology, RNA sequencing and lipidomic analysis of BAT did not reveal overt ferroptosis in cysteine-depletion-induced weight loss. Future studies may reveal cysteine-dependent alternative protective mechanisms that control redox balance and ferroptosis while sustaining UCP1-independent thermogenesis. Taken together, this study expands our understanding of pathways activated by pro-longevity dietary interventions that confer metabolic adaptation required to maintain tissue homeostasis. Thus, the manipulation of TSP activity to drive adipose tissue browning also has implications for developing interventions that control adiposity and promote longevity. In humans, restriction of methionine and cysteine increased FGF21 and caused a reduction in body weight with improvement of metabolic parameters67. Similar to our findings, the metabolic benefits of methionine and cysteine dietary restriction in humans were greater than MR alone67. Here, based on human dietary restriction studies and mouse models of cysteine deficiency, we demonstrate that cysteine is essential for organismal metabolism as its absence triggers adipose browning with progressive weight loss. The participants in this study were part of the CALERIE Phase 2 study68, which was a multicentre, parallel-group, randomized controlled trial by recruitment of non-obese healthy individuals. Overall, 238 adults participated at three different locations: Pennington Biomedical Research Center, Washington University and Tufts University (NCT00427193, registered on ClinicalTrials.gov). Duke University served as a coordinating centre. Participants were randomly assigned to of 25% CR or ad libitum caloric intake for 2 years. CR group participants actually reached 14% of CR1,3. Their body mass index was between 22.0 and 27.9 kg m–2 at the initial visit. Samples were collected at baseline, 1 year and 2 years of intervention. Abdominal subcutaneous adipose tissue biopsy was performed on a portion of CR group participants and used for RNA sequencing and metabolomics in this study. All studies were performed under a protocol approved by the Pennington institutional review board with written informed consent from all participants. All mice were on a C57BL/6J (B6) genetic background. Cth−/− mice (C57BL/6NTac-Cthtm1a(EUCOMM)Hmgu/Ieg) were purchased from the European Mouse Mutant Cell Repository. Breeding these mice to Flipase transgenic mice from The Jackson Laboratories generated Cthfl/fl mice, which were crossed to Adipoq-cre and Albumin-cre, purchased from Jackson Laboratories. Ucp1−/− and CHOP−/− mice were purchased from The Jackson Laboratories and crossed to Cth−/− mice. Fgf21−/− mice were kindly provided by S. Kliewer (UT Southwestern) as described previously40 and crossed to Cth−/− mice. All mice used in this study were housed in specific-pathogen-free facilities in ventilated cage racks that deliver HEPA-filtered air to each cage with free access to sterile water through a Hydropac system at Yale School of Medicine. Mice were fed ad libitum with a standard vivarium chow (Harlan 2018s), unless a special diet was provided, and housed under a 12-h light–dark cycle with controlled temperature and humidity conditions (approximately 22 °C and 60% humidity). All experiments and animal use were approved by the Institutional Animal Care and Use Committee at Yale University. Animals were either allocated to experimental groups depending on their genotype or randomly when working with the same genotype. For pair-feeding studies, mice were provided with either ad libitum or 2.22–2.27 g of diet daily. Cell lysates were prepared using RIPA buffer and optionally frozen and stored at −80 °C. For tissue samples, snap-frozen tissues were ground by mortar and pestle in liquid nitrogen and resuspended in RIPA buffer with protease and phosphatase inhibitors. Samples were centrifuged at 14,000g for 15 min and the supernatant was collected protein concentration was determined using the DC Protein Assay (Bio-Rad) and transferred to a nitrocellulose membrane. Cells or ground tissue (described above) were collected in STAT-60 (Tel-test). RNA from cells were extracted using QIAGEN RNeasy micro kits following the manufacturer's instructions. For tissue samples, RNA was extracted using Zymo mini kits following the manufacturer's instructions. During RNA extraction, DNA was digested using RNase free DNase set (QIAGEN). Synthesis of complementary DNA was performed using an iScript cDNA synthesis kit (Bio-Rad) and real-time quantitative PCR (qPCR) was conducted using Power SYBR Green detection reagent (Thermo Fisher Scientific) on a Light Cycler 480 II (Roche). Primer sequences are listed in Supplementary Table 7. Cth−/− HFD-CTRL and HFD-CysF mice were fasted 14 h before glucose tolerance test. injection based on body weight (0.4 g kg−1). Cth−/− CTRL and CysF mice were fasted for 4 h. Glucose was given by i.p. Blood glucose levels were measured by handheld glucometer (Breeze, Bayer Health Care). Adipose tissue was digested at 37 °C in HBSS (Life Technologies) + 0.1% collagenase I or II (Worthington Biochemicals). Samples were acquired on a custom LSR II or sorted with a FACSARIA cell sorter, using Diva software (v.8.0.1). For SVF, female Cth+/+ and Cth−/− mice were fed CTRL of CysF diet for 4 days. SFAT was collected, with lymph nodes removed, pooled and digested. Isolated cells were subjected to droplet-based 3′ end massively parallel scRNA-seq using Chromium Single Cell 3′ Reagent kits as per manufacturer's instructions (10x Genomics). The libraries were sequenced using a HiSeq3000 instrument (Illumina). Sample demultiplexing, barcode processing and single-cell 3′ counting was performed using the Cell Ranger Single-Cell Software Suite (10x Genomics). The Cell Ranger count was used to align samples to the reference genome (mm10), quantify reads, and filter reads with a quality score below 30. The Seurat package in R was used for subsequent analysis31. Cells with mitochondrial content greater than 0.05% were removed and data were normalized using a scaling factor of 10,000, and number of unique molecular identifiers (nUMI) was regressed with a negative binomial model. Principal-component analysis (PCA) was performed using the top 3,000 most variable genes and t-distributed stochastic neighbour embedding (t-SNE) analysis was performed with the top 20 principal components. Clustering was performed using a resolution of 0.4. The highly variable genes were selected using the FindVariableFeatures function with mean >0.0125 or <3 and dispersion >0.5. These genes are used in performing the linear dimensionality reduction. PCA was performed before clustering and the first 20 principal components were used based on the ElbowPlot. Clustering was performed using the FindClusters function, which works on k-nearest neighbour graph model with granularity ranging 0.1–0.9 and selected 0.4 for the downstream clustering. For identifying the biomarkers for each cluster, we performed differential expression between each cluster to all other clusters, identifying positive markers for that cluster. To understand the trajectory of the adipocyte progenitors, we used Monocle2 to analyse scRNA-seq data of clusters 0, 1 and 2 (ref. Snap-frozen tissues were ground by mortar and pestle in liquid nitrogen and resuspended in STAT-60. RNA was extracted using Zymo mini kits. The quality of raw reads was assessed with FastQC. Raw reads were mapped to the GENCODE vM9 mouse reference genome (GENCODE) using STAR Aligner with the following options: –outFilterMultimapNmax 15 –outFilterMismatchNmax 6 –outSAMstrandField All –outSAMtype BAM SortedByCoordinate –quantMode TranscriptomeSAM. The quality control of mapped reads was performed using in-house scripts that employ Picard tools. The list of rRNA genomic intervals that we used for this quality control was prepared on the basis of UCSC mm10 rRNA annotation file (UCSC) and GENCODE primary assembly annotation for vM9 (GENCODE). rRNA intervals from these two annotations were combined and merged to obtain the final list of rRNA intervals. These intervals were used for the calculation of the percentage of reads mapped to rRNA genomic loci. Strand specificity of the RNA-seq experiment was determined using an in-house script, on the basis of Picard mapping statistics. Expression quantification was performed using RSEM. For the assessment of expression of mitochondrial genes, we used all genes annotated on the mitochondrial chromosome in the GENCODE vM9 mouse reference genome. Gene differential expression was calculated using DESeq2. Pathway analysis was conducted using fgsea (fast GSEA) R-package (fgsea) with the minimum of 15 and maximum of 500 genes in a pathway and with 1 million of permutations. The elimination of redundant significantly regulated pathways (adjusted P < 0.05) was carried out using an in-house Python script in the following way. We considered all ordered pairs of pathways, where the first pathway had normalized enrichment score equal to or greater than the second pathway. For each ordered pair of pathways, we analysed the leading gene sets of these pathways. The leading gene sets were obtained using fgsea. If at least one of the leading gene sets in a pair of pathways had more than 60% of genes in common with the other leading gene set, then we eliminated the second pathway in the pair. Frozen tissues or serum samples, together with internal standard compounds (mentioned below), was subjected to sonication in 500 μl of ice-cold methanol. To this, an equal volume of ultrapure water (LC–MS grade, Wako) and 0.4 volume of chloroform were added. The aqueous phase was then filtered using an ultrafiltration tube (Ultrafree MC-PLHCC, Human Metabolome Technologies), and the filtrate was concentrated by nitrogen spraying (aluminium block bath with nitrogen gas spraying system, DTU-1BN/EN1-36, TAITEC). The concentrated filtrate was dissolved in 50 μl of ultrapure water and utilized for ion chromatography (IC)–MS and LC–MS/MS analysis. Methionine sulfone and 2-morpholinoethanesulfonic acid were employed as internal standards for cationic and anionic metabolites, respectively. The recovery rate (%) of the standards in each sample measurement was calculated to correct for the loss of endogenous metabolites during sample preparation. Anionic metabolites were detected using an orbitrap-type MS (Q-Exactive focus; Thermo Fisher Scientific) connected to a high-performance IC system (ICS-5000+, Thermo Fisher Scientific) that allows for highly selective and sensitive metabolite quantification through IC separation and a Fourier transfer MS principle. The IC system included a modified Thermo Scientific Dionex AERS 500 anion electrolytic suppressor, which converted the potassium hydroxide gradient into pure water before the sample entered the mass spectrometer. Separation was carried out using a Thermo Scientific Dionex IonPac AS11-HC column with a particle size of 4 μm. The IC flow rate was 0.25 ml min−1, supplemented post-column with a makeup flow of 0.18 ml min−1 methanol. The potassium hydroxide gradient conditions for IC separation were as follows: from 1 mM to 100 mM (0–40 min), to 100 mM (40–50 min) and to 1 mM (50.1–60 min), with a column temperature of 30 °C. The Q-Exactive focus mass spectrometer was operated in the ESI-negative mode for all detections. The automatic gain control target was set at 3 × 106 ions, and the maximum ion injection time was 100 ms. The LC–MS-8060 triple-quadrupole mass spectrometer (Shimadzu corporation) with an electrospray ionization (ESI) ion source was employed to perform multiple reaction monitoring in positive and negative ESI modes. The samples were separated on a Discovery HS F5-3 column (2.1 mm internal diameter × 150 mm length, 3-μm particle, Sigma-Aldrich) using a step gradient of mobile phase A (0.1% formate) and mobile phase B (0.1% acetonitrile) with varying ratios: 100:0 (0–5 min), 75:25 (5–11 min), 65:35 (11–15 min), 5:95 (15–20 min) and 100:0 (20–25 min). The flow rate was set at 0.25 ml min−1 and the column temperature was maintained at 40 °C. For low concentration monoamine measurements, extracted tissue metabolites by abovementioned protocol were injected with an autosampler (M-510, Eicom) into a HPLC unit (Eicom) coupled to an ECD (ECD-300, Eicom). The samples were resolved on the Eicompak SC-5ODS column (φ3.0 × 150 mm, Eicom), using an isocratic mobile phase (5 mg l−1 EDTA-2Na, 220 mg l−1 sodium 1-octanesulfonate in acetate/citrate buffer (0.1 M, pH 3.5)/methanol (83:17, v/v)), at a flow rate of 0.5 ml min−1 and a column temperature of 25 °C. At the ECD, analytes were subjected to oxidation reactions within the ECD unit with WE-3G graphite electrode (applied potential is +750 mV against an Ag/AgCl reference electrode). Resulting chromatograms were analysed using the software EPC-300 (Eicom). To extract total lipids, frozen tissues were mixed with 500 μl of 1-butanol/methanol (1:1, v/v) containing 5 mM ammonium formate. The mixture was vortexed for 10 s, sonicated for 15 min in a sonic water bath and then centrifuged at 16,000g for 10 min at 20 °C. The supernatant was transferred to a 0.2-ml glass insert with a Teflon insert cap for LC ESI–MS analysis. For lipidomic analysis, a Q-Exactive focus orbitrap mass spectrometer (Thermo Fisher Scientific) was connected to an HPLC system (Ultimate3000, Thermo Fisher Scientific). The samples were separated on a Thermo Scientific Accucore C18 column (2.1 × 150 mm, 2.6 μm) using a step gradient of mobile phase A (10 mM ammonium formate in 50% acetonitrile and 0.1% formic acid) and mobile phase B (2 mM ammonium formate in acetonitrile/isopropyl alcohol/water, ratios of 10:88:2, v/v/v, with 0.02% formic acid). The Q-Exactive focus mass spectrometer operated in both positive and negative ESI modes. It performed a full mass scan (m/z 250–1,100), followed by three rapid data-dependent MS/MS scans, at resolutions of 70,000 and 17,500, respectively. The automatic gain control target was set at 1 × 106 ions, and the maximum ion injection time was 100 ms. The source ionization parameters included a spray voltage of 3 kV, transfer tube temperature of 285 °C, S-Lens level of 45, heater temperature of 370 °C, sheath gas at 60 and auxiliary gas at 20. The acquired data were analysed using LipidSearch software (Mitsui Knowledge Industry) for major phospholipids (PLs). The search parameters for LipidSearch software were as follows: precursor mass tolerance of 3 ppm, product mass tolerance of 7 ppm and m-score threshold of 3. The tissue block was frozen and secured onto a disc using a cryoembedding medium (Super Cryoembedding Medium, SECTION-LAB), then equilibrated at −16 °C in cryostats (Leica Biosystems). Tissue sections, 8-µm thick, were cut and mounted onto conductive indium-tin-oxide (ITO)-coated glass slides (Matsunami Glass Industries). A solution of tetrafluoroborate salts of 2,4-diphenyl-pyrylium (DPP) (1.3 mg ml−1 in methanol) for on-tissue derivatization of monoamines and DHB-matrix (50 mg ml−1 in 80% ethanol) were manually sprayed onto the tissue using an airbrush (Procon Boy FWA platinum; Mr Hobby). The manual spray was performed at room temperature, applying 40 μl mm−2 with a distance of approximately 50 mm. The samples were analysed using a linear ion trap mass spectrometer (LTQ XL, Thermo Fisher Scientific). The raster scan pitch was set at 50 µm. Ion images were reconstructed using ImageQuest v.1.1.0 software (Thermo Fisher Scientific). Mice were transcardially perfused with PBS, followed by 4% PFA. Brains were then put through a 24 h post-fixing period, after which, immunolabelling and whole-brain clearing were performed according to previously established protocols. Antibodies used for c-Fos labelling were Synaptic Systems rabbit c-Fos 226008 (primary) and Alexa-Fluor 647 donkey anti-rabbit (secondary), respectively. Fos labelling studies were next analysed using ClearMap52. For acquisition, cleared samples were imaged in a sagittal orientation (left lateral side up) on a light-sheet microscope (Miltenyi Blaze) equipped with a sCMOS camera and LVMI-Fluor ×4 objective lens equipped with a 6-mm working distance dipping cap. Inspector Microscope controller software was used. Samples were scanned in the 640 nm channel. Images were taken every 6 μm and reconstructed with ClearMap software52 for quantification or Imaris v.10.1 software for visualization. The 480-nm channel was used with a ×1.3 objective lens for autofluorescence. Animals were anaesthetized with isoflurane, first at a rate of 2–3% and maintained at 0.5–2% in oxygen during surgery. Mice were kept on a heating pad throughout surgery. Mice were injected with buprenorphine and bupivacaine as pre-emptive analgesia. A small ventral incision of 1 cm was made after clipping hair and disinfection with betadine and 70% ethanol. DST nano-T temperature loggers (Star-Oddi) were placed in the peritoneal cavity, and abdominal muscle and skin were sutured closed. After surgery, mice were singly housed and provided with Meloxicam for 48 h. After 7 days, sutures were removed. Ten days after surgery, mice were started on CTRL or CysF diet, and loggers were removed for data collection after killing. Loggers were programmed to take temperature readings every 30 min. The EE, RER, activity and food intake of mice were monitored using the TSE PhenoMaster System (v.3.0.3) Indirect Calorimetry System. Each parameter was measured every 30 min. EE and RER were calculated based on the oxygen consumption (O2) and carbon dioxide production (CO2). Mouse activity was detected by infra-red sensors, and food intake and water consumption were measured via weight sensors on food and water dispensers located in the cage. The parameters of body composition were measured in vivo by magnetic resonance imaging (EchoMRI; Echo Medical Systems). The amount of fat mass, lean mass and free water were measured by the analysis. For the analysis, each mouse was placed in an acrylic tube with breathing holes and the tube was inserted in the MRI machine. The analysis per mouse takes approximately 90 s and automatically calculated numerical results were analysed. Mice were acclimated in climate chambers (model 7000-10, Caron) at either 30 °C or 20 °C, with humidity maintained at 50% under a 12 h light–dark cycle. After 1 week acclimation, mice were switched to either CTRL or CysF diet for 6 days, while maintained in the climate chambers. Mice were handled daily to measure body weight. Faeces were collected daily over the course of CTRL or CysF feeding. Samples were dried for 72 h. Faecal bomb calorimetry was performed at UT Southwestern Medical Center Metabolic Phenotyping Core using a Parr 6200 Isoperibol Calorimeter equipped with a 6510 Parr Water Handling. After blood collection by cardiac puncture, samples were allowed to clot for 2 h. Serum was collected after centrifugation. FGF21 and GDF15 levels in the serum were measured by ELISA (R&D). Cysteine levels were determined by competitive EIA (LS-Bio). Glycerol levels were determined by colorimetric assay (Sigma-Aldrich). Snap-frozen tissues were homogenized in PBS supplemented with EDTA and protease and phosphatase inhibitors. Samples were filtered through 10 kDa Amicon Ultra Centrifugal filters (Millipore). Total GSH and GSSG levels were determined by colorimetric assay (Cayman Chemical). Coenzyme A levels were measured by fluorometric assay (Abcam). Data were normalized to protein concentration, determined beforehand with colorimetric assay (Bio-Rad). SUnSET assay was performed to assess protein synthesis70. Mice were injected with puromycin (Invivogen) (40 nmol g−1 body weight). Mice were killed 30 min after administration and tissues were collected and snap-frozen in liquid nitrogen. Samples were processed for western blotting to assess incorporation of puromycin in newly synthesized proteins. Mice were administered twice daily L748337 (Santa Cruz Biotechnology) (5 mg kg−1) by i.p. Mice were weighed daily and assessed for their health. Tissues were collected in 10% formalin, embedded in paraffin and sectioned into 5-μm thick sections. Tissues were stained with haematoxylin and eosin (H&E) or stained for UCP1 (Abcam) and goat anti-rabbit HRP (DAKO) and developed for colour using Abcam DAB substrate kit. The animals were anaesthetized with 3% isoflurane in an induction chamber and then kept at 2–3% during surgery. The animal was laid back on a microwaveable heating pad. Before incision, a single dose of bupivacaine was given for analgesia. A 1–2-cm midline incision was made on the neck to expose the jugular vein. A sterile polyurethane or silicone catheter with a metal guide was inserted from the back of the neck, where the vascular port was fixed to the jugular vein. Before implantation the port and the catheter were flushed with heparinized saline (25 IU ml−1). The skin was closed with surgical sutures after application of triple antibiotic ointment and the vascular port was fixed. Temperature mapping with BIRDS was performed on a 9.4T Bruker scanner. A 200 mM TmDOTMA− solution was infused at a rate of 60 to 80 µl h−1 for 1–2 h. The infusion rate was adjusted according to animal physiology. The T2-weighted MR images were acquired with a field of view of 23 × 23 mm2, 128 × 128 matrix, 23 slices of 0.5-mm thickness, TR = 3 s and TE = 9 ms. The extremely short T1 and T2 relaxation times (<5 ms) of the TmDOTMA− methyl group allowed ultrafast temperature mapping with BIRDS using 3D chemical shift imaging (CSI) acquisition with a short TR (10 ms) and wide bandwidths (±150 ppm). Temperature mapping with BIRDS was started immediately after detection of global MR signal of TmDOTMA− methyl group, at about 1 h after the start of the infusion. The CSI was acquired using a field of view of 23 × 15 × 23 mm3, 809 spherical encoding steps, 21 min acquisition and reconstructed to 23 × 15 × 23, with a voxel resolution of 1 × 1 × 1 mm3. Selective excitation of the TmDOTMA− methyl group was achieved using a single-band, 200-µs Shinnar-Le Roux (SLR) RF pulse. The MR spectrum in each voxel was line broadened (200 Hz) and phased (zero-order) in MATLAB (MathWorks), and the corresponding temperature Tc was calculated from the chemical shift \({\delta }_{C{H}_{3}}\) of the TmDOTMA− methyl group according to where δ0 = −103.0 ppm and the coefficients a0 = 34.45 ± 0.01, a1 = 1.460 ± 0.003 and a2 = 0.0152 ± 0.0009 were calculated from the linear least-squares fit of temperature as a function of chemical shift \({\delta }_{C{H}_{3}}\). Statistical analysis was carried out using Student's t-test with two tails, with P < 0.05 used as a cutoff for significance. POBN (α-(4-pyridyl-1-oxide)-N-t-butylnitrone, Enzo) was used for spin trapping; POBN was dissolved in saline and administered i.p. Tissue samples (VFAT, SFAT and BAT) were collected 45 min after injection, immediately frozen in liquid nitrogen and stored at −80 °C until EPR measurements. All EPR spectra were recorded in a quartz flat cell using an X-band EMX plus EPR Spectroscope (parameters: 3,480 ± 80 G scan width, 105 receiver gain and 20 mW microwave power; time constant of 1,310 ms and conversion time of 655 ms). Freshly collected SFAT and VFAT samples were measured at 500 μg total protein pe ml, and BAT samples were measured at 100 μg total protein per ml. All results were normalized to 500 μg ml−1 total protein concentration. Stromal vascular fraction from visceral depots of Cth−/− was isolated as previously described. Cells were plated in growth medium (DMEM supplemented with 10% FBS and 1% penicillin–streptomycin) and expanded for 3–5 days. Adipocyte differentiation was induced with growth medium supplemented with insulin (5 μg ml−1), rosiglitazone (1 μM), iso-butyl-methylxanthine (0.5 mM) and dexamethasone (1 μM) for 48 h. Cells were maintained on differentiation medium containing insulin (5 μg ml−1) and rosiglitazone (1 μM) for 96 h. Fully differentiated cells were then treated with various concentrations of cystine (0–200 μM) for 48 h, in cystine and methionine-free DMEM (Gibco) supplemented with 10% dialysed FBS, 1% penicillin–streptomycin and 200 μM methionine. Data collection and analysis were not performed blind to the conditions of the experiments. Data were analysed with GraphPad (v.9.4.0) or R (v.3.4.2). Statistical differences between groups were calculated by unpaired two-tailed t-tests. For comparing groups over time, mice were individually tracked, and groups were compared using two-way analysis of variance (ANOVA) with Sidak's correction for multiple comparisons. For all experiments P ≤ 0.05 was considered significant. No statistical methods were used to pre-determine sample sizes, but our sample sizes are similar to those reported in the field. Data distribution was assumed to be normal, but this was not formally tested. Some mice were excluded from the study due to abnormal health conditions or reaching the humane end point. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Sequencing data are accessible in the Gene Expression Omnibus repository. Murine bulk RNA-seq data of the adipose depots after cysteine restriction is under accession code GSE292788; scRNA-seq of the adipose SVF is under GSE293660; and human RNA-seq data from the CALERIE-II study were published previously1. Source data are provided with this paper. Spadaro, O. et al. Caloric restriction in humans reveals immunometabolic regulators of health span. The matricellular protein SPARC induces inflammatory interferon-response in macrophages during aging. Redman, L. M. et al. Metabolic slowing and reduced oxidative damage with sustained caloric restriction support the rate of living and oxidative damage theories of aging. Green, C. L., Lamming, D. W. & Fontana, L. Molecular mechanisms of dietary restriction promoting health and longevity. Grandison, R. C., Piper, M. D. & Partridge, L. Amino-acid imbalance explains extension of lifespan by dietary restriction in Drosophila. Low methionine ingestion by rats extends life span. Wanders, D. et al. FGF21 mediates the thermogenic and insulin-sensitizing effects of dietary methionine restriction but not its effects on hepatic lipid metabolism. Xu, Q. et al. HNF4α regulates sulfur amino acid metabolism and confers sensitivity to methionine restriction in liver cancer. Plaisance, E. P. et al. Role of beta-adrenergic receptors in the hyperphagic and hypermetabolic responses to dietary methionine restriction. Stipanuk, M. H. Metabolism of sulfur-containing amino acids. Elshorbagy, A. K. et al. Cysteine supplementation reverses methionine restriction effects on rat adiposity: significance of stearoyl-coenzyme A desaturase. & Gaskins, H. R. Redox control of the transsulfuration and glutathione biosynthesis pathways. Moosmann, B., Schindeldecker, M. & Hajieva, P. Cysteine, glutathione and a new genetic code: biochemical adaptations of the primordial cells that spread into open water and survived biospheric oxygenation. Jordan, S. F. et al. Spontaneous assembly of redox-active iron-sulfur clusters at low concentrations of cysteine. & Brown-Borg, H. M. Methionine flux to transsulfuration is enhanced in the long living Ames dwarf mouse. Hine, C. et al. Endogenous hydrogen sulfide production is essential for dietary restriction benefits. Identification and application of gene expression signatures associated with lifespan extension. Shalayel, I. et al. Cysteine chemistry in connection with abiogenesis. Kang, Y. P. et al. Non-canonical glutamate-cysteine ligase activity protects against ferroptosis. OG, Lien Jr & Greenberg, D. M. Identification of α-aminobutyric acid enzymatically formed from threonine. Seelig, G. F., Simondsen, R. P. & Meister, A. Reversible dissociation of gamma-glutamylcysteine synthetase into two subunits. Catalytic and regulatory properties of the heavy subunit of rat kidney γ-glutamylcysteine synthetase. Rouault, T. A. Mammalian iron-sulphur proteins: novel insights into biogenesis and function. Tajima, K. et al. Mitochondrial lipoylation integrates age-associated decline in brown fat thermogenesis. Chouchani, E. T., Kazak, L. & Spiegelman, B. M. New advances in adaptive thermogenesis: UCP1 and beyond. Brown adipose tissue: function and physiological significance. Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Goldberg, E. L. et al. IL-33 causes thermogenic failure in aging by expanding dysfunctional adipose ILC2. Burl, R. B. et al. Deconstructing adipogenesis induced by β3-adrenergic receptor activation with single-cell expression profiling. Identification of a mesenchymal progenitor cell hierarchy in adipose tissue. Marcelin, G., Silveira, A. L. M., Martins, L. B., Ferreira, A. V. & Clément, K. Deciphering the cellular interplays underlying obesity-induced adipose tissue fibrosis. & Graff, J. M. Mouse strains to study cold-inducible beige progenitors and beige adipocyte formation and function. Oguri, Y. et al. CD81 controls beige fat progenitor cell growth and energy balance via FAK signaling. Lee, Y. H., Petkova, A. P., Mottillo, E. P. & Granneman, J. G. In vivo identification of bipotential adipocyte progenitors recruited by β3-adrenoceptor activation and high-fat feeding. Morak, M. et al. Adipose triglyceride lipase (ATGL) and hormone-sensitive lipase (HSL) deficiencies affect expression of lipolytic activities in mouse adipose tissues. Coman, D., Trubel, H. K. & Hyder, F. Brain temperature by biosensor imaging of redundant deviation in shifts (BIRDS): comparison between TmDOTP5- and TmDOTMA-. Hill, C. M. et al. FGF21 is required for protein restriction to extend lifespan and improve metabolic health in male mice. Patel, S. et al. GDF15 provides an endocrine signal of nutritional stress in mice and humans. Warrier, M. et al. Homocysteine-induced endoplasmic reticulum stress activates FGF21 and is associated with browning and atrophy of white adipose tissue in Bhmt knockout mice. Nicholls, D. G. Mitochondrial proton leaks and uncoupling proteins. Kozak, L. P. & Harper, M. E. Mitochondrial uncoupling proteins in energy expenditure. Ukropec, J., Anunciado, R. P., Ravussin, Y., Hulver, M. W. & Kozak, L. P. UCP1 independent thermogenesis in white adipose tissue of cold-acclimated Ucp1-/- mice. Kazak, L. et al. A creatine-driven substrate cycle enhances energy expenditure and thermogenesis in beige fat. Ikeda, K. et al. UCP1-independent signaling involving SERCA2b-mediated calcium cycling regulates beige fat thermogenesis and systemic glucose homeostasis. Bal, N. C. et al. Sarcolipin is a newly identified regulator of muscle-based thermogenesis in mammals. Oeckl, J. et al. Loss of UCP1 function augments recruitment of futile lipid cycling for thermogenesis in murine brown fat. Renier, N. et al. iDISCO: a simple, rapid method to immunolabel large tissue samples for volume imaging. Renier, N. et al. Mapping of brain activity by automated volume analysis of immediate early genes. Nakamura, K. & Morrison, S. F. A thermosensory pathway that controls body temperature. Tan, C. L. et al. Warm-sensitive neurons that control body temperature. Regulation of body temperature by the nervous system. Crestani, C. C. et al. Mechanisms in the bed nucleus of the stria terminalis involved in control of autonomic and neuroendocrine functions: a review. Zhao, Z. D. et al. A hypothalamic circuit that controls body temperature. Schneeberger, M. et al. Regulation of energy expenditure by brainstem GABA neurons. & Bamshad, M. Innervation of mammalian white adipose tissue: implications for the regulation of total body fat. Lee, A. H. & Dixit, V. D. Dietary regulation of immunity. Fabbiano, S. et al. Caloric restriction leads to browning of white adipose tissue through type 2 immune signaling. Barquissau, V. et al. Caloric restriction and diet-induced weight loss do not induce browning of human subcutaneous white adipose tissue in women and men with obesity. Conti, B. et al. Transgenic mice with a reduced core body temperature have an increased life span. Yang, G. et al. H2S as a physiologic vasorelaxant: hypertension in mice with deletion of cystathionine γ-lyase. Paul, B. D. et al. Cystathionine γ-lyase deficiency mediates neurodegeneration in Huntington's disease. Varghese, A. et al. Unravelling cysteine-deficiency-associated rapid weight loss. Richie, J. P. Jr et al. Dietary methionine and total sulfur amino acid restriction in healthy adults. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Schmidt, E. K., Clavarino, G., Ceppi, M. & Pierre, P. SUnSET, a nonradioactive method to monitor protein synthesis. Nakai, K., Kadiiska, M. B., Jiang, J. J., Stadler, K. & Mason, R. P. Free radical production requires both inducible nitric oxide synthase and xanthine oxidase in LPS-treated skin. We thank all investigators and staff involved in coordinating and executing CALERIE-II clinical trial and Yale comparative medicine pathology core led by C. Booth for support with autopsies and histology. We also thank University of Texas Southwestern Medical Center (UTSW), Dallas Metabolic Core facility (supported by National Institute of Diabetes and Digestive and Kidney diseases (NIDDK) P30DK127984–National Institutes of Health (NIH) NORC programme) for bomb calorimetry analysis.K.S. The research in the Dixit Laboratory was supported in part by NIH grants AG031797, AG073969, AG068863 and P01AG051459. These authors contributed equally: Aileen H. Lee, Lucie Orliaguet. Aileen H. Lee, Lucie Orliaguet, Yun-Hee Youm, Tamara Dlugos, Yuanjiu Lei & Vishwa Deep Dixit Department of Comparative Medicine, Yale School of Medicine, New Haven, CT, USA Aileen H. Lee, Lucie Orliaguet, Yun-Hee Youm, Tamara Dlugos, Yuanjiu Lei, Tamas L. Horvath & Vishwa Deep Dixit Aileen H. Lee, Lucie Orliaguet, Yun-Hee Youm, Tamara Dlugos, Yuanjiu Lei & Vishwa Deep Dixit Multiomics Platform, Center for Cancer Immunotherapy and Immunobiology, Kyoto University Graduate School of Medicine, Kyoto, Japan Department of Radiology and Biomedical Imaging, Yale School of Medicine, New Haven, CT, USA Department of Biomedical Engineering, School of Engineering and Applied Science, Yale University, New Haven, CT, USA Department of Pathology and Immunology, Washington University School of Medicine, St. Louis, MO, USA Irina Shchukina, Prabhakar Sairam Andhey & Maxim N. Artyomov Translational Research Institute for Metabolism and Diabetes, AdventHealth, Orlando, FL, USA Pennington Biomedical Research Center, Baton Rouge, LA, USA Department of Cellular and Molecular Physiology, Yale School of Medicine, New Haven, CT, USA Yale Center for Research on Aging, Yale School of Medicine, New Haven, CT, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar performed experiments, data analysis and prepared the manuscript. performed c-fos mapping in the brain. provided scientific input on analyses of SNS. All authors provided intellectual input and assisted with the preparation of the manuscript. The authors declare no competing interests. Nature Metabolism thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Revati Dewal, in collaboration with the Nature Metabolism team. This manuscript has been originally submitted and reviewed at another Springer Nature journal. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Cystathionine and homocysteine measurements by MS/MS in human SFAT at baseline (B) and after 12 months of caloric restriction (n = 14). b) Schematic of Cth−/− and Cthfl/fl mice generation (KOMP construct) used to cross to either Alb:cre or Adipoq:cre. d) Fat mass and lean mass measured by EchoMRI of male Cth+/+ and Cth−/− after 6 days of CTRL or CysF diet (n = 5 Cth+/+ CTRL, n = 12 Cth+/+ CysF, n = 8 Cth−/− CTRL, n = 17 Cth−/− CysF). Percentage body weight change over 6 days of diet. f) Accumulated food intake of Cth+/+ and Cth−/− mice over 6 days of CysF feeding measured in metabolic cages (n = 10 Cth+/+ and n = 12 Cth−/−). Cage image and video show that Cth−/− mice on CysF diet at day 5 have normal activity. g) Qualitative assessment of nest building (score from 0 to 4) and presence (score=1) or absence (score=0) of kyphosis in WT and Cth−/− mice (n = 12/group). h) Gait assessment, ledge test and hindlimb clasping test were performed to measure motor coordination in WT and Cth−/− mice. i) Representative H&E-stained sections of kidney, lung, heart, and liver from female Cth−/− mice fed control diet or cystine-deficient diet for 6 days, lack notable pathologic changes and do not differ in microscopic changes by diet in the tissues examined. C = renal cortex, M = renal medulla A = airway, P = pulmonary artery, > = central vein, and * = portal triad. Statistical differences were calculated by 2-way ANOVA with Sidak's correction for multiple comparisons, or by unpaired two-tailed t-tests. a, b) Volcano plot of a) serum and b) subcutaneous adipose tissue (SFAT) metabolites identified by LC-MS/MS in Cth−/− mice after 6 days of CTRL or CysF diet (n = 4 CTRL and n = 5 CysF for serum, n = 6/group for SFAT). Transsulfuration pathway related metabolites are highlighted in red. Significantly increased or decreased metabolites (-log10(pvalue)>1.3 and ∣log2(FC)∣>1) are highlighted in blue and listed on the right. See Supplementary Table 1 and 2 for the full list of metabolites. c) Serum L-methionine, L-homocysteine, glutamic acid quantified by mass spectrometry in Cth−/− mice fed with CTRL (n = 4) or CysF (n = 5) diet for 6 days. d) Total GSSG content in SFAT, brown adipose tissue (BAT) and liver samples of Cth−/− mice fed CTRL or CysF diet for 5 days, determined by colorimetric assay (n = 7/group). e) Surface sensing of translation (SUnSET) assay was performed to assess protein synthesis. Immunoblot detection of puromycin incorporation in neosynthesized proteins in SFAT, visceral adipose tissue (VFAT), BAT and liver samples of Cth−/− mice fed with CTRL or CysF diet for 5 days (n = 6/group). f) Heatmaps of RNA-seq based gene expression of genes involved in Fe-S cluster assembly in SFAT, VFAT and BAT of Cth+/+ and Cth−/− mice, fed with CTRL or CysF diet (n = 4/group). g) Representative EPR spectra of POBN-lipid radical adducts measured in Folch extracts of SFAT, VFAT and BAT tissues of Cth−/− mice fed with either CTRL or CysF diet. The six-line spectrum (red arrows) is consistent with carbon-centred lipid-derived radicals, indicative of lipid peroxidation (identified through hyperfine coupling constants aN = 15.75 ± 0.06 G and aβH = 2.77 ± 0.07 G). Whiskers are plotted down to the minimum and up to the maximum value. Statistical differences were calculated by 2-way ANOVA with Sidak's correction for multiple comparisons, or by unpaired two-tailed t-tests. a) Representative subcutaneous (SFAT), visceral (VFAT), and brown adipose depots (BAT), indicated with white arrows, of Cth+/+ and Cth−/− after 6 days of CysF diet. b) Representative H&E-stained sections of VFAT of Cth−/− mice fed CTRL or CysF diet for 6 days or after Cys supplementation following CysF weight loss (scale bar=100 μm). c) Serum glycerol levels of Cth−/− mice fed with CTRL (n = 20) or CysF (n = 8) or switched to Cys-containing diet after CysF feeding (n = 10). d) Ucp1, Cidea and Pparg gene expression in Cth−/− pre-adipocytes differentiated in vitro and treated with increasing concentration of Cysteine for 48 h (n = 6/condition). e) Cumulative food intake during the initial two days of CysF feeding in Cth+/+ and Cth−/− mice (n = 10 Cth+/+ and n = 12 Cth−/−). f) Energy expenditure during CysF feeding. g) Linear regression analysis of unnormalized average energy expenditure measured by indirect calorimetry against body mass on days 4 and 5 of CysF diet. i) Respiratory exchange ratio (RER) and j) area under the curve (AUC) quantified for RER. k, l) Whole tissue RNA-seq of SFAT, VFAT, and BAT of Cth+/+ and Cth−/− fed 6 days of CTRL or CysF diet (n = 4/group). k) Heat map highlighting changes specifically occurring in cysteine deficiency. l) Selected top pathways being up- and down-regulated in Cth−/− CysF vs CTRL in SFAT after gene set enrichment analysis. m) Gene expression of selected thermogenesis markers confirmed by qPCR in SFAT, in Cth+/+ and Cth−/− mice fed with CysF diet (n = 8 Cth+/+ and n = 10 Cth−/−). Statistical differences were calculated by one-way ANOVA, or by 2-way ANOVA with Sidak's correction for multiple comparisons, or by unpaired two-tailed t-tests. a) Experimental design schematic of cell processing of subcutaneous adipose depot (SFAT) stromal vascular fraction (SVF) for scRNA-seq. b) t-SNE plot of scRNAseq from SFAT stromal vascular fraction with c) cluster identities. d) Heat map of normalized gene expression of selected markers to identify major cell lineages. e) Enrichment of CL-316,243 activated gene signature overlaid on all populations in all samples. f) t-SNE plots displaying Dpp4, Cd9, Icam1, Col5a3, F3, and Tagln expression in red across all populations in Cth−/− CTRL and Cth−/− CysF samples. g) Volcano plot of differentially expressed genes comparing Cth−/− CysF and Cth+/+ CysF in cluster 1. h) Orthogonal validation of adipocyte progenitor changes using FACS analysis of SFAT SVF in Cth+/+ and Cth−/− mice on CTRL and CysF diet for 4 days (n = 6 Cth+/+ CTRL and CysF, n = 5 Cth−/− CTRL and CysF). i) Selected top pathways from gene set enrichment comparing Cth−/− CysF vs. Cth+/+ CysF in cluster 1. j) Heatmap of gene expression of select stem and mature adipocyte genes in clusters 0, 1 and 2 showing the impact of cysteine depletion in mice. Statistical differences were calculated by 2-way ANOVA with Sidak's correction for multiple comparisons, and by unpaired two-tailed t-tests. a) Quantification of ATGL immunoblot shown in Fig. 3a, Actin was used as a loading control. b, c) Tissue lipidomics of brown adipose depot (BAT) from Cth−/− mice fed CTRL (n = 4) or CysF diet (n = 5) for 6 days with b) triglycerides (TG) and c) diacylglycerol species highlighted. d) Core body temperature (CBT) measured in the peritoneal cavity by implantation of Star-Oddi logger of Cth−/− mice fed with CTRL or CysF diet over 6 days and e) average day and night CBT of Cth−/− mice fed with CTRL or CysF diet. g, h) Serum levels of g) FGF21 and h) GDF15 in Cth−/− and Cth−/−CHOP−/− mice after 5 days of CysF feeding, measured by ELISA (n = 9 Cth−/− and n = 7 Cth−/−CHOP−/−). i) SFAT, VFAT and BAT weight normalized to body weight of Cth−/− and Fgf21−/−Cth−/− mice after CysF feeding (n = 5/group). j) Respiratory exchange ratio (RER) of Cth−/− and Fgf21−/−Cth−/− mice upon CysF feeding, measured at day 3 and 4 in metabolic cages (n = 5/group). k) Ucp1 gene expression in SFAT of Cth−/− and Fgf21−/−Cth−/− mice after 6 days of CysF feeding (n = 11 Cth−/− and n = 12 Fgf21−/−Cth−/−). l) Representative H&E histology images of SFAT showing increased browning at day 6 in Cth+/+ and Cth−/− mice fed CysF diet and housed at 20 °C or at 30 °C. m) Ucp1 gene expression measured by qPCR in BAT of Cth+/+ and Cth−/− mice fed CysF diet and housed at 20 °C or at 30 °C for 6 days (n = 3 Cth+/+20 C, n = 4 Cth+/+30 C and Cth−/−20C, n = 5 Cth−/−30C). Whiskers are plotted down to the minimum and up to the maximum value. Statistical differences were calculated by 2-way ANOVA with Sidak's correction for multiple comparisons, and by unpaired two-tailed t-tests. a) Immunoblot analysis of CTH in liver, kidney, subcutaneous (SFAT), visceral (VFAT), brown (BAT) adipose depots, lung, heart, spleen, and thymus. Actin is used as a loading control. c, d) Cysteine serum levels of c) Cthf/f and Alb-Cre;Cthf/f mice (n = 4 Cthf/f CysF, n = 5 Cthf/f CTRL, Alb-Cre;Cthf/f CTRL and CysF) and d) Cthf/f and Adipoq-Cre;Cthf/f (n = 4 Cthf/f CTRL and CysF, Adipoq-Cre;Cthf/f CTRL and n = 5 Adipoq-Cre;Cthf/f CysF) mice after 6 days of CTRL or CysF diet. e–i) Alb-Cre;Cthf/f mice were fed CTRL or CysF diet for 6 days. g) Volcano plot of metabolites identified by MS/MS in the liver. h) Schematic summary of changes in the metabolites and i) volcano plot of metabolites identified by MS/MS in the SFAT. Transsulfuration pathway related metabolites are highlighted in red. Significantly increased or decreased metabolites (-log10(pvalue)>1.3 and ∣log2(FC)∣>1) are highlighted in blue and listed on the right. j) Schematic summary of changes in serum metabolites of Adipoq-Cre;Cthf/f fed with CTRL or CysF diet for 6 days. Blue lines represent measured, but unchanged metabolites, red and green arrows indicate significantly decreased or increased metabolites, respectively (p < 0.05). See Supplementary Table 6 for the full list of metabolites. k) Percentage body weight change of Cth+/+ and Cth−/− mice that were co-housed and fed CysF diet for 6 days (n = 4/group). l) Accumulated food intake of Cth−/− and Cth−/− Ucp1−/− mice during 6 days of CysF diet (n = 7 Cth−/− and n = 8 Cth−/− Ucp1−/−). m, n) RNA-seq based expression of genes associated with m) futile creatine cycle (Slc6a8, Gatm, Gamt, Ckmt2, Alpl and Ckb) and n) futile calcium cycle (Atp2a2 and Ryr2) in the SFAT of Cth+/+ and Cth−/− mice fed CTRL or CysF diet for 6 days (n = 4/group). p) RNA-seq based expression of genes associated with triglyceride and fatty acid metabolism (Dgat1, Pnpla2, Lipe, Gk) in the SFAT of Cth+/+ and Cth−/− mice fed CTRL or CysF diet for 6 days (n = 4/group). q–r) Heatmaps of gene expression of genes involved in creatine, calcium and lipid futile cycles in q) BAT and r) SFAT of Cth−/− and Cth−/−Ucp1−/− mice fed a CysF diet for 6 days (n = 16 Cth−/−, n = 15 Cth−/− Ucp1−/−), quantified by qPCR. Whiskers are plotted down to the minimum and up to the maximum value. Statistical differences were calculated by 2-way ANOVA with Sidak's correction for multiple comparisons, and by unpaired two-tailed t-tests. Panels e, f, h and j created with BioRender.com. b) qPCR gene expression of Comt in SFAT of Cth+/+ (n = 8) and Cth−/− (n = 10) mice fed with CysF diet for 6 days. c) Cumulative food intake during CysF feeding of Cth−/− treated with L748337, a β-3 adrenergic receptor antagonist or vehicle (PBS) for 5 days (n = 6 PBS and n = 4 L748337). d) Body composition measured by Echo-MRI on day 6 post diet switch (n = 6 Cth−/− HFD-CTRL and n = 4 Cth−/− HFD-CysF). e) The glucose tolerance test (GTT) in Cth−/− mice fed CTRL or cysF diet for 4 days (n = 5 CTRL and n = 6 CysF). Glucose dose was based on lean mass. f) Linear regression analysis of energy expenditure (EE) against body mass during dark cycle of Cth−/− mice fed with HFD-CTRL or HFD-CysF, average values of nights 4 and 5 of diet switch. Statistical differences were calculated by 2-way ANOVA with Sidak's correction for multiple comparisons, and by unpaired two-tailed t-tests. Table 3: metabolite list of Alb-Cre Cthf/f serum. Table 4: metabolite list of Alb-Cre Cthf/f serum. Table 5: metabolite list of Alb-Cre Cthf/f SFAT. Table 6: metabolite list of Adipo-Cre Cthf/f serum. Statistical source data for Fig. Statistical source data for Fig. Raw images from western blot (WB) in Fig. Raw images from WB in Fig. Raw images from WB in Fig. Raw images from WB in Fig. Raw images from WB in Extended Data Fig. Raw images from WB in Extended Data Fig. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Cysteine depletion triggers adipose tissue thermogenesis and weight loss. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/green-tech/a64947129/lichen-concrete/'>A New Kind of Concrete Can Grow and Repair Itself. How? It's Alive.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Self-healing concrete could be greener, more resilient, and longer lasting than the standard variety. So, it's no surprise that scientists and engineers are trying to figure out ways to lower this building material's impact. One solution is to not use it altogether—the creation of mass timber buildings in the U.S. has been rapidly increasing in recent years. The most common method for increasing concrete's longevity is by ripping a page out of the Roman playbook and developing means for concrete to essentially “self-heal.” The reason why the 2,000-year-old aqueducts throughout Italy don't look a day over 500 is because Romans used “lime clasts” mixed in with concrete that essentially filled cracks before they formed. Of course, self-healing concrete isn't new—headlines have popped up for years touting new techniques for making concrete long lasting, and some have even used microbes. But the authors of this new study say they've stumbled upon a technique that improves on previous attempts in one major way. “Microbe-mediated self-healing concrete has been extensively investigated for more than three decades” Congrui Grace Jin, senior author of the study from Texas A&M, said in a press statement, “but it still suffers from one important limitation—none of the current self-healing approaches are fully autonomous since they require an external supply of nutrients for the healing agents to continuously produce repair materials.” Lichen, often found clinging to trees and rocks, is actually a complex symbiotic system filled with cyanobacteria, fungi, and algae. Specifically, the fungi can pull in ionized calcium, which spurs the production of calcium carbonate—the ultra-hard material found in seashells and coral. (Calcium carbonate is actually the same material that made Rome's concrete extremely strong in antiquity.) Using this autonomous repair system can not only extend the life of concrete, but also cut down on repair costs. The authors are now testing whether this synthetic fungi could repair existing cracks as well. Challenges still remain for the widespread adoption of self-healing concrete. Right now, not many companies make the stuff, and it's still more expensive to produce that typical, run-of-the-mill concrete. Some experts say it could still be a decade before self-healing additives really take off. When that happens, though, the lichen will be ready.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41550-025-02565-z'>A temperate 10-Earth-mass exoplanet around the Sun-like star Kepler-725</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 12:47:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The detection of low-mass exoplanets (≤10 Earth masses (M⊕)) yields fundamental inputs for current theories of planet formation and evolution, and supplies critical information for the planned direct-imaging missions that aim to detect and characterize Earth-like planets in the habitable zones around solar-like stars. However, the most efficient detection techniques available for low-mass exoplanets (that is, photometric transit and radial velocity methods) are heavily biased towards the detection of short-period planets (for example, ≤100 days) and strongly favour late-type stars. Here we report the discovery of Kepler-725 c, a 10 ± 3 M⊕ exoplanet within the habitable zone of the late G-type dwarf Kepler-725. Through analysis of the transit timing variations of the relatively short-period (39.64 days) warm Jupiter Kepler-725 b, we find that Kepler-725 c has a period of 207.5 days and travels in an eccentric orbit (with an eccentricity of 0.44 ± 0.02 and an orbital semi-major axis of 0.674 ± 0.002 au), receiving a time-averaged insolation of 1.4 times the Earth's value. This discovery demonstrates that the transit timing variation method enables the detection and accurate mass measurement of a super-Earth/mini-Neptune within a solar-like star's habitable zone. Similar searches for such exoplanets could be conducted in other exoplanetary systems in the era of the Transiting Exoplanet Survey Satellite mission and upcoming PLAnetary Transits and Oscillations of stars and Earth 2.0 missions. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The Kepler data were downloaded from https://archive.stsci.edu. The TTV measurements obtained by Holczer et al.13 plotted in Fig. 1 were downloaded from https://content.cld.iop.org/journals/0067-0049/225/1/9/revision1/apjsaa25b0t3_mrt.txt. Further data used to create the figures in this article are available on figshare at https://doi.org/10.6084/m9.figshare.28720076 (ref. 64), which includes the data for Extended Data Figs. Source data are provided with this paper. The dynamical simulations are based on the publicly available packages TTVFast (https://github.com/kdeck/TTVFast) and REBOUND (https://rebound.readthedocs.io/en/latest/). Both the transit modelling code STMT and the TTV inversion code GEMC are mainly based on the MCMC sampling strategy in the PRISM_GEMC code (https://github.com/JTregloanReed/PRISM_GEMC). The TTV inversion code GA is mainly based on the geneticalgorithm code (https://github.com/rmsolgi/geneticalgorithm). The TTV inversion code using MultiNest is available on GitHub at https://github.com/JohannesBuchner/MultiNest. Other scripts used in the article are available from the corresponding authors upon request. Bryson, S. et al. The occurrence of rocky habitable-zone planets around solar-like stars from Kepler data. Google Scholar Akeson, R. L. et al. The NASA Exoplanet Archive: data and tools for exoplanet research. Google Scholar Queloz, D. et al. No planet for HD 166435. Google Scholar Measuring precise radial velocities on individual spectral lines. I. Validation of the method and application to mitigate stellar activity. Google Scholar Pepe, F. et al. ESPRESSO at VLT. On-sky performance and first results. Google Scholar Jontof-Hutter, D., Rowe, J. F., Lissauer, J. J., Fabrycky, D. C. & Ford, E. B. The mass of the Mars-sized exoplanet Kepler-138 b from transit timing. Google Scholar Agol, E., Steffen, J., Sari, R. & Clarkson, W. On detecting terrestrial planets with timing of giant planet transits. Google Scholar & Murray, N. W. The use of transit timing to detect terrestrial-mass extrasolar planets. Google Scholar Sun, L. et al. A highly mutually inclined compact warm-Jupiter system KOI-984? Google Scholar Borucki, W. J. et al. Kepler planet-detection mission: introduction and first results. Google Scholar Ford, E. B. et al. Transit timing observations from Kepler. V. Transit timing variation candidates in the first sixteen months from polynomial models. Google Scholar Batalha, N. M. et al. Planetary candidates observed by Kepler. Analysis of the first 16 months of data. Google Scholar Holczer, T. et al. Transit timing observations from Kepler. Catalog of the full long-cadence data set. Google Scholar Berger, T. A., Huber, D., Gaidos, E. & van Saders, J. L. Revised radii of Kepler stars and planets using Gaia Data Release 2. Google Scholar The California-Kepler Survey. Precise physical properties of 2025 Kepler planets and their host stars. Google Scholar McQuillan, A., Mazeh, T. & Aigrain, S. Stellar rotation periods of the Kepler objects of interest: a dearth of close-in planets around fast rotators. Google Scholar Sun, L. et al. Refined system parameters and TTV study of transiting exoplanetary system HAT-P-20. Google Scholar Nesvorný, D. et al. The detection and characterization of a nontransiting planet by transit timing variations. Google Scholar Sun, L. et al. Kepler-411: a four-planet system with an active host star. Google Scholar Kopparapu, R. K. et al. Habitable zones around main-sequence stars: new estimates. Google Scholar & Rivera-Valentín, E. G. The equilibrium temperature of planets in elliptical orbits. Google Scholar Brewer, J. M. et al. EXPRES. Two additional planets orbiting ρ Coronae Borealis reveal uncommon system architecture. Google Scholar & Kipping, D. Probabilistic forecasting of the masses and radii of other worlds. Google Scholar Kopparapu, R. K. et al. Habitable zones around main-sequence stars: dependence on planetary mass. Google Scholar Luque, R. & Pallé, E. Density, not radius, separates rocky and water-rich small planets orbiting M dwarf stars. Google Scholar Piaulet, C. et al. Evidence for the volatile-rich composition of a 1.5-Earth-radius planet. Google Scholar Rogers, J. G., Schlichting, H. E. & Owen, J. E. Conclusive evidence for a population of water worlds around M dwarfs remains elusive. Google Scholar Mol Lous, M., Helled, R. & Mordasini, C. Potential long-term habitable conditions on planets with primordial H–He atmospheres. Google Scholar Madhusudhan, N., Piette, A. & Constantinou, S. Habitability and biosignatures of Hycean worlds. Google Scholar Johansen, A., Youdin, A. & Mac Low, M. Particle clumping and planetesimal formation depend strongly on metallicity. Google Scholar Podlewska-Gaca, E., Papaloizou, J. C. B. & Szuszkiewicz, E. Outward migration of a super-Earth in a disc with outward propagating density waves excited by a giant planet. Google Scholar Izidoro, A., Raymond, S. N., Morbidelli, A., Hersant, F. & Pierens, A. Gas giant planets as dynamical barriers to inward-migrating super-Earths. Google Scholar Lithwick, Y., Xie, J. Extracting planet mass and eccentricity from TTV data. Google Scholar Kovács, G., Zucker, S. & Mazeh, T. A box-fitting algorithm in the search for periodic transits. Google Scholar Zhan, H. The wide-field multiband imaging and slitless spectroscopy survey to be carried out by the Survey Space Telescope of China Manned Space Program. Google Scholar The Large Ultraviolet/Optical/Infrared Surveyor. Google Scholar Gaudi, B. S. et al. The Habitable Exoplanet Observatory. Google Scholar Ricker, G. R. et al. Transiting Exoplanet Survey Satellite (TESS). Google Scholar Rauer, H. et al. The PLATO 2.0 mission. Google Scholar China is hatching a plan to find Earth 2.0. Google Scholar Hippke, M., David, T. J., Mulders, G. D. & Heller, R. Wōtan: comprehensive time-series detrending in Python. Google Scholar Lightkurve Collaboration et al. Lightkurve: Kepler and TESS time series analysis in Python. Astrophysics Source Code Library ascl:1812.013 (2018). Lindegren, L. et al. Gaia Data Release 2. The astrometric solution. Google Scholar Zechmeister, M. & Kürster, M. The generalised Lomb-Scargle periodogram. A new formalism for the floating-mean and Keplerian periodograms. Google Scholar Barnes, S. A. Ages for illustrative field stars using gyrochronology: viability, limitations, and errors. Google Scholar Mamajek, E. E. & Hillenbrand, L. A. Improved age estimation for solar-type dwarfs using activity-rotation diagnostics. Google Scholar Spada, F. & Lanzafame, A. C. Competing effect of wind braking and interior coupling in the rotational evolution of solar-like stars. Google Scholar Angus, R. et al. Toward precise stellar ages: combining isochrone fitting with empirical gyrochronology. Google Scholar Claret, A., Hauschildt, P. H. & Witte, S. New limb-darkening coefficients for PHOENIX/1D model atmospheres. Calculations for 5000 K ≤ Teff ≤ 10 000 K Kepler CoRot Spitzer uvby UBVRIJHK Sloan and 2MASS photometric systems. Google Scholar Enoch, B., Collier Cameron, A., Parley, N. R. & Hebb, L. An improved method for estimating the masses of stars with transiting planets. Google Scholar Torres, G., Andersen, J. Accurate masses and radii of normal stars: modern results and applications. Google Scholar Deck, K. M. et al. TTVFast: an efficient and accurate code for transit timing inversion problems. Google Scholar Tregloan-Reed, J., Southworth, J. & Tappert, C. Transits and starspots in the WASP-19 planetary system. Google Scholar Feroz, F., Hobson, M. P. & Bridges, M. MULTINEST: an efficient and robust Bayesian inference tool for cosmology and particle physics. Google Scholar Southworth, J. Homogeneous studies of transiting extrasolar planets - IV. Thirty systems with space-based light curves. Google Scholar Zeng, L., Sasselov, D. D. & Jacobsen, S. B. Mass-radius relation for rocky planets based on PREM. Google Scholar REBOUND: an open-source multi-purpose N-body code for collisional dynamics. Google Scholar Rein, H. & Tamayo, D. WHFAST: a fast and unbiased implementation of a symplectic Wisdom-Holman integrator for long-term gravitational simulations. Google Scholar Gladman, B. Dynamics of systems of two close planets. Google Scholar Cincotta, P. M. & Simó, C. Simple tools to study global dynamics in non-axisymmetric galactic potentials - I. Astron. Google Scholar Williams, D. M. & Pollard, D. Earth-like worlds on eccentric orbits: excursions beyond the habitable zone. Google Scholar Bolmont, E., Libert, A., Leconte, J. & Selsis, F. Habitability of planets on eccentric orbits: limits of the mean flux approximation. Google Scholar The equilibrium temperature of planets on eccentric orbits: time scales and averages. Sun, L. et al. Kepler-725_NA_data. This article includes data collected by the Kepler mission. Funding for the Kepler mission is provided by the NASA (National Aeronautics and Space Administration) Science Mission directorate. This research has made use of the NASA Exoplanet Archive, which is operated by the California Institute of Technology under contract with NASA under the Exoplanet Exploration Program. This research has made use of the VizieR catalogue access tool, CDS, Strasbourg, France. This work has made use of data from the European Space Agency mission Gaia (https://www.cosmos.esa.int/gaia), processed by the Gaia Data Processing and Analysis Consortium (DPAC, https://www.cosmos.esa.int/web/gaia/dpac/consortium). Funding for DPAC has been provided by national institutions, in particular the institutions participating in the Gaia Multilateral Agreement. S.G. acknowledges financial support from the National Natural Science Foundation of China (grant numbers 12288102 and U1531121) and the Yunnan Fundamental Research Project (grant number 202305AS350009), the science research grant from the China Manned Space Project. acknowledge financial support from the National Natural Science Foundation of China (grant numbers 10873031, 11473066 and 12003063). also acknowledges support from International Centre of Supernovae, Yunnan Key Laboratory (grant number 202302AN360001). acknowledges support from the National Natural Science Foundation of China (grant number 11573004) and the Research Development Fund (grant number RDF-16-01-16) of Xi'an Jiaotong-Liverpool University. acknowledges support from the National Natural Science Foundation of China (grant numbers 11827804 and U2031210), the China Manned Space Project (grant number CMS-CSST-201906) and the China Manned Space Program (grant number CMS-CSST-2025-A18). This work has been carried out under the cooperation frame between the China Scholarship Council and Deutscher Akademischer Austausch Dienst. The joint research project between the Yunnan Observatories and Hamburg Observatory is funded by the Sino-German Center for Research Promotion (grant number GZ1419). Yunnan Observatories, Chinese Academy of Sciences, Kunming, China L. Sun, S. Gu & X. Wang Key Laboratory for the Structure and Evolution of Celestial Objects, Chinese Academy of Sciences, Kunming, China L. Sun, S. Gu & X. Wang International Centre of Supernovae, Yunnan Key Laboratory, Kunming, China School of Astronomy and Space Science, University of Chinese Academy of Sciences, Beijing, China Hamburger Sternwarte, Universität Hamburg, Hamburg, Germany Department of Physics, School of Mathematics and Physics, Xi'an Jiaotong-Liverpool University, Suzhou, China Nanjing Institute of Astronomical Optics & Technology, Chinese Academy of Sciences, Nanjing, China CAS Key Laboratory of Astronomical Optics & Technology, Nanjing Institute of Astronomical Optics & Technology, Nanjing, China You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar proposed the study and wrote the paper. carried out all calculations and simulations. contributed to discussions of the results. Correspondence to L. Sun or X. Wang. The authors declare no competing interests. Nature Astronomy thanks Eric Agol and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The contour in each panel encloses 68.3% credible region. The green dashed lines enclose the 68.3% credible intervals. Seven optimum solutions for the three-planet framework that can well reproduce the observed TTV signals of Kepler-725 b are identified. The contours in each panel enclose 68.3%, 95.4% and 99.7% credible regions, respectively. While the green dashed lines enclose the 68.3% credible intervals. Only known low-mass planets (gray filled circle + cross) that have mass measured with precision better than 33% and radius measured with precision better than 33% are shown (source: TEPCat). The different lines represent the theoretical models of various compositions for exoplanets. All error bars represent 1-standard deviation. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Sun, L., Gu, S., Wang, X. et al. A temperate 10-Earth-mass exoplanet around the Sun-like star Kepler-725. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03743-2'>A generative AI-discovered TNIK inhibitor for idiopathic pulmonary fibrosis: a randomized phase 2a trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 10:20:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite substantial progress in artificial intelligence (AI) for generative chemistry, few novel AI-discovered or AI-designed drugs have reached human clinical trials. Here we present the results of the first phase 2a multicenter, double-blind, randomized, placebo-controlled trial testing the safety and efficacy of rentosertib (formerly ISM001-055), a first-in-class AI-generated small-molecule inhibitor of TNIK, a first-in-class target in idiopathic pulmonary fibrosis (IPF) discovered using generative AI. IPF is an age-related progressive lung condition with no current therapies available that reverse the degenerative course of disease. Patients were randomized to 12 weeks of treatment with 30 mg rentosertib once daily (QD, n = 18), 30 mg rentosertib twice daily (BID, n = 18), 60 mg rentosertib QD (n = 18) or placebo (n = 17). The primary endpoint was the percentage of patients who have at least one treatment-emergent adverse event, which was similar across all treatment arms (72.2% in patients receiving 30 mg rentosertib QD (n = 13/18), 83.3% for 30 mg rentosertib BID (n = 15/18), 83.3% for 60 mg rentosertib QD (n = 15/18) and 70.6% for placebo (n = 12/17)). Treatment-related serious adverse event rates were low and comparable across treatment groups, with the most common events leading to treatment discontinuation related to liver toxicity or diarrhea. Secondary endpoints included pharmacokinetic dynamics (Cmax, Ctrough, tmax, AUC0–t/τ/∞ and t1/2), changes in lung function as measured by forced vital capacity, diffusion capacity of the lung for carbon monoxide, forced expiry in 1 s and change in the Leicester Cough Questionnaire score, change in 6-min walk distance and the number and hospitalization duration of acute exacerbations of IPF. We observed increased forced vital capacity at the highest dosage with a mean change of +98.4 ml (95% confidence interval 10.9 to 185.9) for patients in the 60 mg rentosertib QD group, compared with −20.3 ml (95% confidence interval −116.1 to 75.6) for the placebo group. These results suggest that targeting TNIK with rentosertib is safe and well tolerated and warrants further investigation in larger-scale clinical trials of longer duration. The process of developing novel therapeutics is increasingly slow and costly1, with an average of US$2–3 billion spent over the course of 10–15 years to bring a new drug to market2. Advances in artificial intelligence (AI) have demonstrated proof-of-concept innovation and potential to accelerate and decrease costs across all stages of pharmaceutical research and development by facilitating disease-associated target prioritization3,4,5, compound design and optimization6,7,8,9,10, and clinical trial performance11,12,13,14,15. Shifting more of the time-, material- and labor-intensive steps of drug discovery to AI platforms enables more rapid nomination of therapeutic candidate compounds that explore the possible chemical space more thoroughly16,17 and with a concomitant reduction in hands-on screening18,19, thus allowing more preclinical candidate compounds to enter clinical testing and, eventually, clinical practice. Despite these advancements, few AI-discovered or AI-designed drugs have reached clinical trials. AI-discovered drugs have experienced similar levels of phase 2 trial failure as non-AI-discovered drugs20,21, and none has so far progressed through phase 3 trials21. The question of whether AI can impart meaningful, sustained disruption to drug development has remained unanswered21. As an answer to this question, our group used generative AI-driven discovery tools22,23 to identify Traf2- and Nck-interacting kinase (TNIK) de novo as a critical regulator of idiopathic pulmonary fibrosis (IPF) pathology as it orchestrates multiple profibrotic and proinflammatory cellular programs9. Our approach resulted in the first antifibrotic, small-molecule TNIK inhibitor, rentosertib (formerly ISM001-055), for the treatment of IPF designed by generative AI. We reported the development and positive phase 0 and 1 trial results of rentosertib9 (NCT05154240 and CTR20221542 (accessed via http://www.chinadrugtrials.org.cn/)), the first report of a targeted TNIK inhibitor entering clinical testing, which found rentosertib to be safe and well tolerated by healthy individuals, exhibiting a favorable pharmacokinetic (PK) profile in both trials9. This was also, importantly, the first reported instance of AI platform-enabled discovery of both a disease-associated target and a compound for that target. Our generative AI-powered approach streamlined preclinical candidate nomination to a mere 18 months and completion of phase 0/1 clinical testing to under 30 months from the initiation of target discovery, representing a revolutionary shift in the streamlining of drug discovery9. IPF is an age-related, progressive lung disease diagnosed by excluding other potential causes of interstitial lung disease (for example, environmental exposures, connective tissue disease, hypersensitivity pneumonitis or drug toxicities)24 and marked by interstitial pneumonia, dyspnea and cough25, histologically characterized by fibroblast proliferation and extracellular matrix remodeling26,27,28,29. Affecting between 10 and 60 per 100,000 individuals in the USA, with an approximately tenfold greater incidence in those over 65, IPF occurs at a rate similar to stomach, brain and testicular cancers25. The median survival time after diagnosis is 2–4 years despite widespread availability of targeted antifibrotic therapies26,30, highlighting the ongoing need for novel, effective therapies to treat IPF. The current standard-of-care (SOC) therapy regimen to treat IPF includes either nintedanib, a broadly acting inhibitor of receptor tyrosine kinases31, or pirfenidone, an inhibitor of TGFβ-mediated fibroblast-to-myofibroblast transition32. Clinical trials investigating these two drugs in patients with IPF have demonstrated only a slowing of disease progression33,34, as measured by forced vital capacity (FVC) and self-reported quality of life (QOL) metrics, and unclear benefit to survival26, making the development of therapies that can restore lung function and reverse the course of disease a priority. Here, we present the results of a phase 2a multicenter, double-blind, randomized, placebo-controlled trial to assess the safety, tolerability, PK and impact on FVC of rentosertib at a range of doses for up to 12 weeks in patients with IPF. Additional secondary and exploratory endpoints and analyses were designed to further measure lung function, QOL metrics and cellular phenotypes associated with rentosertib treatment and response. Among the 128 patients screened for inclusion, 71 (55.5%) patients were enrolled and randomly assigned to receive placebo (n = 17), 30 mg rentosertib once daily (QD, n = 18), 30 mg rentosertib twice daily (BID, n = 18) or 60 mg rentosertib QD (n = 18) (Extended Data Fig. The most common reason for screening-based exclusion was obstructive pulmonary disease comorbidity indicated by forced expiry in 1 s (FEV1)/FVC less than 0.7 at visit 1 (baseline) or diffusion capacity of the lung for carbon monoxide (DLCO) percent of predicted normal falling outside the range of ≥25% and <80% at baseline. Patients were also excluded for a recent history of upper or lower respiratory tract infection (within 4 weeks) at visit 1 (baseline) or before visit 2 (week 1). The intention-to-treat population included 71 patients, and 55 (77%) completed the 12-week placebo-controlled period (15 (88%) in the placebo group, 16 (89%) in the 30 mg rentosertib QD group, 12 (67%) in the 30 mg rentosertib BID group and 12 (67%) in the 60 mg rentosertib QD group). Overall, 16 (22.5%) patients discontinued treatment before the conclusion of the 12-week study due to either adverse events (AEs) or other withdrawal; 2 from placebo, 2 from 30 mg rentosertib QD, 6 from 30 mg rentosertib BID and 6 from 60 mg rentosertib QD groups. All patients were included in the safety and efficacy analyses (Fig. Of 128 patients screened for inclusion, 71 were randomized to receive 30 mg rentosertib QD (n = 18), 30 mg rentosertib BID (n = 18), 60 mg rentosertib QD (n = 18), or placebo (n = 17) over the course of 12 weeks. Sixteen patients discontinued treatment prior to the end of treatment. Clinical and demographic characteristics, including age, body mass index (BMI) and baseline lung function, were similar across treatment groups (Table 1). The rates of treatment-emergent AEs (TEAEs) were similar across all treatment groups (13/18 (72.2%) in the 30 mg rentosertib QD group, 15/18 (83.3%) in the 30 mg rentosertib BID group, 15/18 (83.3%) in the 60 mg rentosertib QD group and 12/17 (70.6%) in the placebo group) (Table 2), with similar incidence for patients concurrently taking SOC antifibrotic therapy or not (Supplementary Table 1). Treatment-related AEs were more common among patients who received rentosertib compared with placebo, with 5/17 (29.4%) in the placebo group versus 9/18 (50.0%) in the 30 mg QD group, 11/18 (61.1%) in the 30 mg BID group, and 14/18 (77.8%) in the 60 mg QD group. Of these, few were serious AEs (SAEs), with no treatment-related SAEs reported in the placebo group, 1/18 (5.6%) of the rentosertib 30 mg QD group, 2/18 (11.1%) in the 30 mg BID group and 2/18 (11.1%) in the 60 mg QD group (Table 2). A complete listing of TEAEs is presented in Supplementary Table 2. The most frequent reason for treatment discontinuation was AEs (12/16 (75%)), and the treatment of 7 of the 12 patients was discontinued for AEs due to liver injury or dysfunction (0/18 (0%) in the 30 mg QD group, 4/18 (22.2%) in the 30 mg BID group and 3/18 (17%) in the 60 mg QD group) (Supplementary Table 3). Four of the seven (57.1%) total participants who withdrew because of liver toxicity were concurrently administered nintedanib antifibrotic therapy. Three of the four (75%) remaining discontinuations were due to patient withdrawal from the trial (3/71 patients (4.2%)). One patient died of heart failure before the week 12 end-of-treatment (EOT) visit determined to be unrelated to the trial owing to a history of degenerative aortic valve disease, aortic valve incompetence, left ventricular dysfunction, aortic dilatation, pulmonary hypertension, increased blood glucose and hypercholesterolemia, as well as a lack of other AEs reported and no notable findings for vital signs, physical examination, 12-lead electrocardiogram (ECG) or laboratory tests since randomization. A complete listing of patients whose treatment was discontinued due to TEAE is presented in Supplementary Table 4. FVC is the gold-standard metric for assessing the lung function of patients with IPF and response to therapeutic intervention33,34,35. After 12 weeks of treatment, patients who received placebo experienced deteriorating FVC with a mean change of −20.3 ml (95% confidence interval (CI) −116.1 to 75.6). Patients who received 30 mg rentosertib QD showed a similar mean reduction in FVC of −27.0 ml (95% CI −88.8 to 34.8), patients who received 30 mg rentosertib BID experienced a mean change of +19.7 ml (95% CI −60.5 to 99.9) and patients receiving 60 mg rentosertib QD showed improved FVC, with and +98.4 ml (95% CI 10.9 to 185.9) (Fig. 2 and Extended Data Fig. Patients receiving 60 mg rentosertib QD not concurrently taking SOC antifibrotic therapy exhibited significant improvement in FVC (+187.8 ml, 95% CI 68.6 to 306.9 ml), whereas patients concurrently taking 60 mg rentosertib QD with either nintedanib or pirfenidone did not exhibit significant changes in FVC (Extended Data Fig. a, The absolute change in FVC ± 95% CI. b, The absolute change in FVC ± 95% CI ANCOVA model with multiple imputation assuming missing at random (MAR). The changes in DLCO and FEV1 were relatively small and similar across treatment regimens (Extended Data Fig. Changes in self-reported QOL and physical ability metrics, such as responses to the Leicester Cough Questionnaire (LCQ) and 6-min walk distance (6MWD), respectively, were likewise similar across treatment groups (Extended Data Fig. 4f–i), although our modeling indicated a significant increase of least-squares mean change of the LCQ scores of patients receiving 60 mg rentosertib compared with those receiving placebo (two-sided P = 0.0495). Three patients (16.7%) receiving 60 mg rentosertib QD and one patient (5.9%) receiving placebo experienced acute exacerbation of IPF (AE-IPF) (Supplementary Table 5). The patients in the 60 mg QD group were all hospitalized in response to their acute exacerbations with a mean duration of 23.3 days, whereas the patient in the placebo group was not hospitalized. A PK analysis of rentosertib concentration in the blood plasma of patients with IPF after single and multiple doses shows a favorable PK profile, with greater exposure in patients receiving 60 mg rentosertib QD compared with patients receiving 30 mg rentosertib QD or 30 mg rentosertib BID, peaking 1 h after administration (Fig. The area under the plasma–concentration curve (AUC0–t), measuring total exposure to the drug to the last available time point, was greater for the 60 mg rentosertib QD at both week 0 (first dose) and week 12 (EOT; arithmetic mean 1,630 and 3,450 h ng−1 ml−1, respectively) than for 30 mg rentosertib BID (315 and 1,390 h ng−1 ml−1) and 30 mg rentosertib QD (553 and 788 h ng−1 ml−1) (Fig. The steady state was achieved at week 2, and no obvious drug accumulation was observed by week 12. The geometric mean half-life (t1/2) ranged from 10.9 to 12.0 h at week 12. a–c, Pharmacokinetic dynamics of rentosertib in patient sera collected predose and periodically throughout 24 h posttreatment at week 0 (a), at 0 h posttreatment at weeks 2, 4 and 8 (b) and 48 h after administration at week 12 (c). d, AUC0–t of rentosertib exposure; n = 22 in 30 mg QD, n = 21 in 30 mg BID, n = 20 in 60 mg QD. e, Cmax of rentosertib exposure; n = 22 in 30 mg QD, n = 21 in 30 mg BID, n = 20 in 60 mg QD. f, Ctrough of rentosertib exposure in patient sera collected pretreatment at weeks 0, 2, 4, 8 and 12. All data represent arithmetic mean ± s.d. A higher AUC0–t and t1/2 at week 12 for all dosages indicates an increased exposure to rentosertib with increases time on treatment (Fig. 3a,c and Extended Data Fig. Change in patients' FVC from baseline to week 12 was positively correlated with AUC0–t and trough concentration (Ctrough) (Extended Data Fig. Not surprisingly, the Cmax exposure was greatest in patients receiving the 60 mg QD dose, with both 30 mg QD and 30 mg BID showing similarly lower Cmax (Fig. 3e), whereas Ctrough was similar between 30 mg BID and 60 mg QD dosages (Fig. Together, the PK dynamics suggest that higher dosage leads to increased exposure from higher initial net absorption, with a more pronounced effect by the end of the treatment period associated with slower clearance. We performed proteomic profiling of patient serum samples to understand the mechanism of action of rentosertib, guide development of biomarkers of response and evaluate the hypothesis that inhibition of TNIK may target biological aging36. We identified a total of 2,841 proteins using the Olink Explore 3072 panel across serum samples collected from patients in all treatment groups at baseline, 2 weeks, 4 weeks and 12 weeks. Using a linear regression model to evaluate time-dependent effects of rentosertib treatment on protein expression changes, we found that treatment duration substantially impacted protein expression patterns, with 20, 82 and 192 proteins significantly changing with treatment at weeks 2, 4 and 12, respectively (Benjamini–Hochberg-adjusted P value <0.1) (Extended Data Fig. Paired statistical testing to assess dose-associated changes in protein expression compared with placebo identified 1, 8 and 22 high-confidence (Padj < 0.05) differentially abundant proteins in sera from patients receiving 30 mg QD, 30 mg BID and 60 mg QD rentosertib, respectively, using a generalized linear model (Fig. 4a,b and Supplementary Tables 7–9). Restricting analysis of differential proteins from baseline to week 12, the number of significantly altered proteins increased substantially with higher dosing: 0, 39 and 115 in the 30 mg QD, 30 mg BID and 60 mg QD groups, respectively (Extended Data Fig. Together, this suggests that the duration of rentosertib treatment and the dose are associated with changes in serum protein profiles. a,b, Differentially abundant proteins in serum from patients receiving rentosertib at 30 mg BID (a) and 60 mg QD (b) identified with a generalized linear model. c, Serum levels of COL1A1, FAP, FN1 and MMP10 decrease with rentosertib dose and time on treatment. n = 11 patients in placebo, n = 11 patients in 30 QD, n = 11 patients in 30 BID, n = 10 patients in 60 QD. d, The changes in serum levels of COL1A1, FAP, FN1 and MMP10 from baseline to week 12 are inversely correlated with change in FVC from baseline to week 12. n = 43 patients at each visit; P value calculated by Pearson correlation analysis. e,f, Reactome pathway enrichment for differentially abundant proteins in serum from patients receiving rentosertib at 30 mg BID (e) and 60 mg QD (f). Downregulated proteins associated with 30 mg BID and 60 mg QD treatment include known fibrosis-associated proteins such as MMP10, PTPRZ1, COL1A1, FAP, FN1, ROBO2, ASPN and LTBP2 (Fig. Analyzing the differentially abundant proteins to find association with change in FVC found that ASPN, PTPRZ1, MMP10 and CHAD were both significantly inversely correlated to change in FVC (P < 0.05) and significantly downregulated after administration of rentosertib at the week 12 time point, suggesting an array of potential biomarkers for response to treatment (Fig. Pathway enrichment analysis identified extracellular matrix organization as the most downregulated Reactome pathway gene set in both treatment groups, suggesting a reduction in excessive extracellular matrix production (Fig. We queried diverse published gene expression datasets profiling the tissue of patients with IPF and that of healthy individuals using our AI-powered target discovery platform, PandaOmics37, for expression patterns of these top genes to validate their relevance to IPF. COL1A1, MMP10 and FAP were found to be universally upregulated in IPF patient samples compared with healthy individuals, and FN1 was upregulated in most datasets (Extended Data Fig. We further found that the abundance at week 12 of proteins previously reported to be associated with lung function38 and transplant-free survival39 in patients with IPF, such as ADAMTSL2, COL6A3, CCN3, COL24A1, KRT19 and LTBP2, were associated change in FVC (Extended Data Fig. The expression of inflammation-associated pathways, other immune-related proteins, such as IL-10 and CD5, and other extracellular matrix-related proteins, such as COL6A3, were found to have significant associations with time on treatment and change in FVC, further suggesting that TNIK plays a diverse role in various aging-related dysregulated pathways36 (Fig. 4e,f and Extended Data Fig. In this phase 2a study, treatment with rentosertib at 30 mg QD, 30 mg BID and 60 mg QD over 12 weeks was safe and well tolerated. Treatment-related SAEs were rare, with a similarly low rate across treatment and placebo groups. The most common TEAEs leading to discontinuation of treatment were liver toxicity related and occurred mostly in patients who were also treated with nintedanib antifibrotic therapy, although notably not pirfenidone. In addition to liver toxicities, diarrhea and hypokalemia were among the most common TEAEs among participants treated with rentosertib. To determine the respective and synergistic contributions of rentosertib and SOC antifibrotic therapies to liver and gastrointestinal toxicity, as well as any potential effects on absorption or trough levels, we require further drug–drug interaction studies and assessment in a larger cohort of mixed SOC antifibrotic recipients and nonrecipients. Treatment with 60 mg rentosertib QD over 12 weeks was associated with a trend toward an increase in FVC in patients with IPF. The 60 mg rentosertib QD dose exhibited the greatest mean improvement in lung function, as measured by FVC, whereas patients receiving placebo experienced an average decline in FVC. Patients receiving 60 mg rentosertib QD experienced a mean increase in FVC percentage change of 2.82% by our modeling, meeting the minimal clinically important difference (MCID) reported for FVC in IPF of 2–6% (ref. Subgrouping patients by concurrent SOC antifibrotic treatment showed patients receiving 60 mg rentosertib QD without SOC antifibrotic exhibited the strongest increase in FVC, suggesting potential interaction between the drugs that will be investigated in future trials. The incongruity of increased FVC with stable DLCO may be attributed to low numbers of patients in each treatment arm and the inherent variability of DLCO measurement41. The impact of rentosertib treatment on QOL metrics was largely inconclusive, with large variances within treatment arms, although our modeling found that patient-reported coughing-related QOL (via LCQ) was significantly improved in patients receiving 60 mg rentosertib QD. As various QOL metrics integrate the functionality of a number of organ systems, of which lung health is one contributing factor, a lack of difference among the treatment groups may be due to the relatively short data collection period and small cohort size. AE-IPF is a strong indication of dismal patient prognosis42 with a median survival of 2.2 months after onset43, and incidence informs long-term therapeutic efficacy. Three acute exacerbations occurred in the 60 mg rentosertib arm and one in the placebo arm; however, a 12-week study is a relatively short time frame to capture such long-term disease events or trajectory44. An unintended consequence of rentosertib's potential immunomodulatory mechanism of action9 may be a dampened site-specific immune response, potentially leading to increased susceptibility to infection, a known trigger for a subset of AE-IPF45,46. Because AE-IPF is a clinically important occurrence, we intend to assess its incidence in longer trials with larger cohorts in the future, which would assess patients for at least 12 months to more fully capture the incidence of AE-IPF and other potential AEs. Assaying the serum proteome in patients provided insight into changes in circulating proteins associated with higher doses of rentosertib. The top up- and downregulated proteins showed high overlap between the 30 mg BID and 60 mg QD doses, with downregulated proteins enriched for association with extracellular matrix organization, a key feature of fibrotic progression in IPF. Moreover, a number of the top downregulated proteins are known to be associated with pulmonary fibrosis: MMP1047, LTBP239,48,49, KRT1939 and COL24A139 have been nominated as biomarkers for IPF and transplant-free survival; PTPRZ150 activates β-catenin signaling, which contributes to IPF development51, and BHLHE40 regulates epithelial-to-mesenchymal transition in IPF via β-catenin52; COL1A1 is a key component of dysregulated extracellular matrix deposition characteristic of fibrosis53,54,55; ASPN plays a role in myofibroblast transformation56 and may play a central role in IPF tissue remodeling55,57; circulating ROBO2 is associated with poor IPF prognosis58; FAP is expressed exclusively in fibrotic loci in the lungs of patients with IPF59; and CHAD binds to type II collagen in cartilaginous fibrotic loci60,61. Downregulation of this IPF-associated protein profile supports the hypothesis that inhibition of TNIK modulates IPF pathophysiological pathways and points to a potential serum protein signature as a biomarker for response to rentosertib treatment. The limitations of this study include the small cohort size of each arm, the geographical and demographic homogeneity of the participants (all were residents of China of similar race) and a short period of follow-up, which limit the assessment of long-term safety and efficacy. Despite the short duration and the number of withdrawals from the trial across all arms (n = 16/71 (22.5%)), the results are encouraging for additional study of this drug candidate and target, suggesting that rentosertib is generally safe and potentially effective and, more broadly, that using AI in both target identification and in drug design may enhance the efficiency of the drug development process. Longer phase 2 or 3 trials studying rentosertib in a larger, heterogeneous sample of patients with IPF from around the globe are warranted to further evaluate the efficacy and safety of rentosertib treatment for IPF. This phase 2a, multicenter, double-blind, placebo-controlled, randomized, multidose trial of rentosertib in adults with IPF was performed at 21 sites across China beginning 19 July 2023 and running through 11 June 2024. Adults with IPF were randomly assigned in a 1:1:1:1 ratio via interactive response technology to receive oral rentosertib (at a dose of 30 mg (QD), 30 mg (BID) or 60 mg (QD)) or placebo (QD) for 12 weeks, along with the continued use of SOC medications (Fig. Patient screening occurred, on average, 30 days before the first dosing at the second visit. The study treatment rentosertib was provided by Insilico Medicine or a designated contract research organization, packaged and labeled in accordance with the principles of Good Manufacturing Practice. Resupply to the sites was managed via an interactive response technology system, which also monitored expiry dates of supplies available at the sites. Access to the randomization codes was controlled and documented. The trial was conducted following the principles outlined in the Declaration of Helsinki and the International Council for Harmonization guidelines for Good Clinical Practice. The institutional review board or ethics committee at participating centers approved protocols and adhered to local laws before initiation of the clinical trial. All patients in this study provided written informed consent. Trial management and data processing, summarization and analyses were conducted by Fortrea Clinical Pharmacology Services, Leeds, UK. Authors used by the study sponsor were involved in the trial design, collection, analysis and data interpretation. Subjects, investigators, site study staff, reviewers and everyone involved in study conduct or analysis were blinded with regard to the randomized treatment assignments until after data freeze. Bioanalytics staff were allowed to identify samples from subjects assigned to placebo treatment but did not disclose randomization until trial unblinding. These authors contributed to manuscript preparation across all iterations of the manuscript. The first and last authors of this study vouch for the completeness of all data reported in this publication. Eligible patients were 40 years of age or older with an IPF diagnosis as defined by the American Thoracic Society, European Respiratory Society, Japanese Respiratory Society and Latin American Thoracic Association guidelines62. Enrolled patients presented with stable IPF and were deemed suitable for study participation on the basis of medical history, physical examination, vital signs, 12-lead ECG and laboratory evaluation. Patients previously receiving nintedanib or pirfenidone were enrolled if their antifibrotic therapy had been stable for >8 weeks before the first (screening) visit. Eligible patients had to meet all three of the following criteria during the screening visit: (1) FVC >40% predicted of normal, (2) DLCO corrected for hemoglobin ≥25% and <80% predicted of normal, and (3) FEV1/FVC ratio >0.7 based on prebronchodilator value. Patients were excluded for concomitant respiratory disorders or health concerns, including, but not limited to, cystic fibrosis, active aspergillosis, active tuberculosis, confirmed coronavirus disease 2019 (COVID-19) at visits 1 or 2 or severe COVID-19 requiring hospitalization within 6 months of visit 1 or long COVID-19, clinically relevant or severe pulmonary hypertension, acute IPF exacerbation within 4 months before visit 1 and/or during the screening period, upper or lower respiratory tract infection that has not fully resolved within 4 weeks before visit 1 and/or before visit 2 (day 1), active on the lung transplantation register or expected to become active on the lung transplant register within 6 months before visit 1, a history of lung volume reduction surgery or lung transplant, current smoker, BMI >40 kg m−2, aspartate aminotransferase (AST) or ALT ≥1.5 times upper level of normal or total bilirubin ≥1.5 times upper level of normal at visit 1, eGFR ≤60 ml min−1 1.73 m−2 (Chronic Kidney Disease Epidemiology Collaboration formula) at visit 1, uncontrolled hypertension (systolic pressure >160 mmHg or diastolic pressure >95 mmHg despite treatment with antihypertensive treatments) at visit 1, unstable cardiac angina or myocardial infarction within 6 months before visit 1, taking oral corticosteroids, any documented active or suspected malignancy or history of malignancy within 5 years before visit 1, known hypersensitivity or contraindications to serine/threonine kinase inhibitors, any condition or treatment possibly affecting drug absorption (for example, gastrectomy or metoclopramide), taking restricted medications (for example, moderate-to-strong CYP3A4–CYP1A2 inhibitors or inducers or medications primarily metabolized by CYP3A4–CYP1A2) or consumption of grapefruit or grapefruit juice, pomelo, Seville orange or Seville orange-containing products within 48 h before day 1, substantial trauma or major surgery within 3 months before visit 1 or planned major surgery during the study and 12-lead ECG demonstrating corrected QT interval by Fridericia (QTcF) >450 ms for males and >470 ms for females, or a QRS interval >120 ms at visit 1. Full inclusion and exclusion criteria are detailed in the associated study protocol. Given an approximate sample size of 15 subjects per treatment arm, there exists a 90% probability of observing at least one AE if the true population rate is approximately 15%, which is sufficient to assess the feasibility of safety parameters. The primary objective of this study was to evaluate the safety and tolerability of orally administered rentosertib for up to 12 weeks in adult patients with IPF compared with placebo. The primary endpoint for this study was the percentage of patients who have at least one TEAE. Patients were continually monitored for AEs during the 12 weeks of rentosertib treatment and for 1 week after administration of the final dose, with study drug-related SAE and AEs of special interest collected if they occurred beyond the study period. The secondary efficacy endpoints measured the relative and percent change in FVC from week 0/visit 2 up to week 12. The absolute and relative change in FVC (%) predicted from week 0/visit 2 up to week 12 as well as the change in DLCO (%) predicted from week 0 up to week 12 was also measured. Twenty-one subjects (29.6%) had FVC and FEV1 measured at baseline (randomization with spirometry equipment sourced from the local site meeting the 2019 American Thoracic Society and European Respiratory Society guideline criteria)63, provided that the subject's baseline and end-of-treatment spirometry were performed with the same equipment. Fifty subjects (70.4%) had FVC and FEV1 measured at baseline/randomization with centrally sourced SpiroSphere devices (Clario) provided by the sponsor with study-specific training and proficiency tests to the operating staff to meet the 2019 ATS/ERS guideline criteria63 with central overread. All DLCO measurements were assessed with devices from each site according to the ATS/ERS guidelines64, and all measurements were conducted with the same equipment for each site. FVC MCID has been reported to be a change of 2–6% in patients with IPF40. The change in LCQ, a 19-item questionnaire that assesses cough-related QOL65, from week 0 to weeks 4, 8 and 12 was also analyzed. The LCQ examines three domains (physical, psychological and social) related to the patient's experience within a 24-h time frame. The range for the total score on the LCQ is 3–21, with the sum of each domain score ranging from 1 to 7. A higher score indicates a better QOL, with an MCID reported to be a change of 1.3 points for patients with chronic obstructive pulmonary disease66. The LCQ questionnaire has mainly been validated in disease settings such as chronic obstructive pulmonary disease67, noncystic fibrosis bronchiectasis68 and chronic cough69. The change in 6MWD in meters from week 0 to week 12 was also evaluated. Other secondary endpoints included pharmacokinetic parameters of rentosertib and related metabolites after the first dose at visit 2 and the last dose at visit 6 (EOT). This study also collected data on the number of acute IPF exacerbations and the number of days hospitalized after acute IPF exacerbations from week 0 through week 12. The exploratory endpoints analyzed include the change in IPF blood biomarkers after rentosertib treatment at weeks 0, 2, 4, 8 and 12. Changes in the blood proteome in treated patients were also measured at weeks 0, 2, 4, 8 and 12. An analysis of covariance (ANCOVA) model was performed for endpoints including FVC. The model includes treatment as a fixed effect and baseline value as a covariate. LCQ scores were analyzed using a mixed model for repeated measures. FVC and other spirometry measurements were collected at baseline and at week 12, with all patients being measured at baseline and a total of 22 patients missing week 12 measurements (5 receiving 30 mg rentosertib QD, 6 receiving 30 mg rentosertib BID, 8 receiving 60 mg rentosertib QD and 3 receiving placebo). Missing values at week 12 were imputed using a multiple imputation method assuming missing at random. All statistical comparisons were made using two-sided tests. No statistical comparisons of AEs between treatment groups were performed. Given an approximate sample size of 15 subjects per treatment arm, there exists a 90% probability of observing at least one AE if the true population rate is approximately 15%, which was sufficient to assess the feasibility of safety parameters, although a sample size calculation based on statistical power considerations was not performed. Spearman correlation analysis was used to assess the association between FVC change and AUC0–t or Ctrough. The safety population included all subjects who received at least one dose of study treatment. The intent-to-treat population included any randomized subjects. The intent-to-treat population was used for summary of demographic and baseline characteristics and for analysis of secondary endpoints, except PK analysis. Assessment of PK parameters included subjects who received at least one dose of study treatment and had at least one measurable concentration collected after dosing. Assessment of serum biomarkers was performed on subjects who had a predose baseline value and at least one value after treatment initiated. For Olink proteomics analyses, we utilized a generalized linear mixed model to assess the association between each protein and dose or FVC change. The lme4 package was used for the generalized linear mixed model analysis. For modeling the FVC changes, we conducted the statistical analyses in two steps. In the first step, we regressed the protein level (NPX) against the visit to obtain the slope of protein change rate for each individual. In the second step, we regressed the FVC changes against the slope of protein change rates obtained from the first step. This was conducted using the lm package. Pathway analyses were performed using the clusterProfiler R package, with Reactome processes70 as the curated pathway set. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Complete deidentified Olink proteomics data, and FVC data, have been deposited at the OMIX database under accession codes at OMIX008341 (https://ngdc.cncb.ac.cn/omix/release/OMIX008341). Qualified researchers may request access to Olink proteomics data through the OMIX database at https://ngdc.cncb.ac.cn/omix/. Study protocol and statistical analysis plan will be provided in a secure data sharing environment upon academic or research request. Source data are provided with this paper. Custom code used to analyze Olink proteomics data is available via GitHub at https://github.com/HUICUI1992/Code-for-NM. Scannell, J. W., Blanckley, A., Boldon, H. & Warrington, B. Diagnosing the decline in pharmaceutical R&D efficiency. Berdigaliyev, N. & Aljofan, M. An overview of drug discovery and development. Pun, F. W., Ozerov, I. V. & Zhavoronkov, A. AI-powered therapeutic target discovery. Qureshi, R. et al. AI in drug discovery and its clinical relevance. Artificial intelligence in cancer target identification and drug discovery. Du, Y. et al. Machine learning-aided generative molecular design. Cheng, Y., Gong, Y., Liu, Y., Song, B. & Zou, Q. Molecular design in drug discovery: a comprehensive review of deep generative models. Sousa, T., Correia, J., Pereira, V. & Rocha, M. Generative deep learning for targeted compound design. Ren, F. et al. A small-molecule TNIK inhibitor targets fibrosis in preclinical and clinical models. Discovery of a novel and potent cyclin-dependent kinase 8/19 (CDK8/19) inhibitor for the treatment of cancer. Prediction of clinical trials outcomes based on target choice and clinical trial design with multi-modal artificial intelligence. Feuerriegel, S. et al. Causal machine learning for predicting treatment outcomes. Feijoo, F., Palopoli, M., Bernstein, J., Siddiqui, S. & Albright, T. E. Key indicators of phase transition for clinical trials through machine learning. Integrated deep learned transcriptomic and structure-based predictor of clinical trials outcomes. Preprint at bioRxiv https://doi.org/10.1101/095653 (2016). Gayvert, K. M., Madhukar, N. S. & Elemento, O. A data-driven approach to predicting successes and failures of clinical trials. Lavecchia, A. Navigating the frontier of drug-like chemical space with cutting-edge generative AI models. Jiménez-Luna, J., Grisoni, F., Weskamp, N. & Schneider, G. Artificial intelligence in drug discovery: recent advances and future perspectives. Mak, K.-K. & Pichika, M. R. Artificial intelligence in drug development: present status and future prospects. Jayatunga, M. K. P., Xie, W., Ruder, L., Schulze, U. & Meier, C. AI in small-molecule drug discovery: a coming wave? Wong, C. H., Siah, K. W. & Lo, A. W. Estimation of clinical trial success rates and related parameters. KP Jayatunga, M., Ayers, M., Bruens, L., Jayanth, D. & Meier, C. How successful are AI-discovered drugs in clinical trials? A first analysis and emerging lessons. Identification of therapeutic targets for amyotrophic lateral sclerosis using PandaOmics—an AI-enabled biological target discovery platform. Pun, F. W. et al. Hallmarks of aging-based dual-purpose disease and age-associated targets predicted using PandaOmics AI-powered discovery engine. Diagnosis of idiopathic pulmonary fibrosis. An official ATS/ERS/JRS/ALAT clinical practice guideline. & Martinez, F. J. Idiopathic pulmonary fibrosis. Richeldi, L., Collard, H. R. & Jones, M. G. Idiopathic pulmonary fibrosis. Raghu, G. & Richeldi, L. Current approaches to the management of idiopathic pulmonary fibrosis. Martinez, F. J. et al. Idiopathic pulmonary fibrosis. Raghu, G. Idiopathic pulmonary fibrosis: lessons from clinical trials over the past 25 years. A., Schwarz, M. I., Brown, K. R. & Cherniack, R. M. Predicting survival in idiopathic pulmonary fibrosis. Wollin, L. et al. Mode of action of nintedanib in the treatment of idiopathic pulmonary fibrosis. Aimo, A. et al. Pirfenidone for idiopathic pulmonary fibrosis and beyond. Noble, P. W. et al. Pirfenidone in patients with idiopathic pulmonary fibrosis (CAPACITY): two randomised trials. Richeldi, L. et al. Efficacy and safety of nintedanib in idiopathic pulmonary fibrosis. King, T. E. et al. A phase 3 trial of pirfenidone in patients with idiopathic pulmonary fibrosis. Ewald, C. Y. et al. TNIK's emerging role in cancer, metabolism, and age-related diseases. Kamya, P. et al. PandaOmics: an AI-driven platform for therapeutic target and biomarker discovery. et al. Multi-ancestry proteome-phenome-wide Mendelian randomization offers a comprehensive protein-disease atlas and potential therapeutic targets. Preprint at medRxiv https://doi.org/10.1101/2024.10.17.24315553 (2024). Oldham, J. M. et al. Proteomic biomarkers of survival in idiopathic pulmonary fibrosis. du Bois, R. M. et al. Forced vital capacity in patients with idiopathic pulmonary fibrosis: test properties and minimal clinically important difference. McCormack, M. C. Facing the noise: addressing the endemic variability in DLCO testing. Acute exacerbations of idiopathic pulmonary fibrosis. Song, J. W., Hong, S.-B., Lim, C.-M., Koh, Y. & Kim, D. S. Acute exacerbation of idiopathic pulmonary fibrosis: incidence, risk factors and outcome. Richeldi, L. et al. Trial of a preferential phosphodiesterase 4B inhibitor for idiopathic pulmonary fibrosis. Viral infection in acute exacerbation of idiopathic pulmonary fibrosis. Acute exacerbation of idiopathic pulmonary fibrosis. An International Working Group Report. Sokai, A. et al. Matrix metalloproteinase-10: a novel biomarker for idiopathic pulmonary fibrosis. Enomoto, Y. et al. LTBP2 is secreted from lung myofibroblasts and is a potential biomarker for idiopathic pulmonary fibrosis. Plasma LTBP2 as a potential biomarker in differential diagnosis of connective tissue disease-associated interstitial lung disease and idiopathic pulmonary fibrosis: a pilot study. Shang, D., Xu, X., Wang, D., Li, Y. Protein tyrosine phosphatase ζ enhances proliferation by increasing β-catenin nuclear expression in VHL-inactive human renal cell carcinoma cells. Chilosi, M. et al. Aberrant Wnt/β-catenin pathway activation in idiopathic pulmonary fibrosis. Hu, X. et al. Dec1 deficiency ameliorates pulmonary fibrosis through the PI3K/AKT/GSK-3β/β-catenin integrated signaling pathway. Devos, H., Zoidakis, J., Roubelakis, M. G., Latosinska, A. Reviewing the regulators of COL1A1. Bibaki, E. et al. miR-185 and miR-29a are similarly expressed in the bronchoalveolar lavage cells in IPF and lung cancer but common targets DNMT1 and COL1A1 show disease specific patterns. Identification of Hub genes and pathways associated with idiopathic pulmonary fibrosis via bioinformatics analysis. Huang, S. et al. Asporin promotes TGF-β–induced lung myofibroblast differentiation by facilitating Rab11-dependent recycling of TβRI. Åhrman, E. et al. Quantitative proteomic characterization of the lung extracellular matrix in chronic obstructive pulmonary disease and idiopathic pulmonary fibrosis. Todd, J. L. et al. Association of circulating proteins with death or lung transplant in patients with idiopathic pulmonary fibrosis in the IPF-PRO Registry Cohort. Acharya, P. S., Zukas, A., Chandan, V., Katzenstein, A.-L. A. & Puré, E. Fibroblast activation protein: a serine protease expressed at the remodeling interface in idiopathic pulmonary fibrosis. Regional and disease specific human lung extracellular matrix composition. Shum, L. Chondroadherin binds to type II collagen. Raghu, G. et al. Idiopathic pulmonary fibrosis (an update) and progressive pulmonary fibrosis in adults: an official ATS/ERS/JRS/ALAT clinical practice guideline. Standardization of Spirometry 2019 Update. An Official American Thoracic Society and European Respiratory Society Technical Statement. Graham, B. L. et al. 2017 ERS/ATS standards for single-breath carbon monoxide uptake in the lung. Birring, S. S. et al. Development of a symptom specific health status measure for patients with chronic cough: Leicester Cough Questionnaire (LCQ). Minimal clinically important differences for patient-reported outcome measures of cough and sputum in patients with COPD. The validity and precision of the Leicester Cough Questionnaire in COPD patients with chronic cough. Murray, M. P., Turnbull, K., MacQuarrie, S., Pentland, J. L. & Hill, A. T. Validation of the Leicester Cough Questionnaire in non-cystic fibrosis bronchiectasis. Nguyen, A. M. et al. Leicester Cough Questionnaire validation and clinically important thresholds for change in refractory or unexplained chronic cough. The Reactome Pathway Knowledgebase 2024. We thank Fortrea Clinical Pharmacology Services (Leeds, UK) for logistical and organizational support, C. Wang and Y. Jiang of Insilico Medicine (Shanghai, China) for data management expertise, Y. Liu of Insilico Medicine (Shanghai, China) for clinical operations support and T. Astor of Insilico Medicine (Cambridge, MA, USA) for clinical data interpretation support. Department of Pulmonary and Critical Care Medicine, Peking Union Medical College Hospital, Beijing, China Zuojun Xu & Ping Wang Feng Ren, Sang Liu, Yuan Lv, Heng Zhao, Shan Chen, Hui Cui & Alex Zhavoronkov Department of Respiratory and Critical Care Medicine, Tianjin Medical University General Hospital, Tianjin, China Department of Pulmonary and Critical Care Medicine, Beijing Friendship Hospital, Capital Medical University, Beijing, China Department of Respiratory Disease, Qilu Hospital of Shandong University, Jinan, China Department of Respiratory Medicine, Shengjing Hospital of China Medical University, Shenyang, China Department of Pulmonary and Critical Care Medicine, Nanjing Drum Tower Hospital, Nanjing University, Nanjing, China Department of Respiratory and Critical Care Medicine, Hainan General Hospital, Haikou, China Department of Respiratory and Critical Care, Anhui Chest Hospital, Hefei, China Department of Respiratory Medicine, Shanghai Pulmonary Hospital, Tongji University, Shanghai, China Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Zhengzhou University, Zhengzhou, China Department of Pulmonary and Critical Care Medicine, West China Hospital, Sichuan University, Chengdu, China Department of Respiratory and Critical Care Medicine, Nanfang Hospital, Southern Medical University, Guangzhou, China Department of Respiratory Medicine, Xiangya Hospital, Central South University, Changsha, China Department of Respiratory and Critical Care Medicine, Peking University Shougang Hospital, Beijing, China Department of Respiratory and Critical Care Medicine, Jiangxi Provincial People's Hospital, Nanchang, China Insilico Medicine US, Cambridge, MA, USA Sujata Rao, Carol Satler, David Gennert & Alex Zhavoronkov Insilico Medicine AI, Abu Dhabi, United Arab Emirates Mikhail Korzinkin & Alex Zhavoronkov Insilico Medicine Hong Kong, Hong Kong SAR, China You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the clinical study. participated in patient enrollment, carried out the trial and acquired data. analyzed and interpreted the data. wrote and revised the paper. All authors have seen and approved of the paper before submission. Correspondence to Zuojun Xu or Alex Zhavoronkov. are employees of Insilico Medicine. Insilico Medicine was the study sponsor. The other authors declare no competing interests. Nature Medicine thanks Yuben Moodley, Luca Richeldi, Jeff Swigris and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Lorenzo Righetto, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Eligible patients were randomized to receive 30 mg rentosertib QD, 30 mg rentosertib BID, 60 mg rentosertib QD, or placebo administered over 12 weeks with periodic assessment and plasma sampling. IPF, idiopathic pulmonary fibrosis; FVC, forced vital capacity; FEV1, forced expiratory volume in one second; DLCO, diffusion capacity of the lung for carbon monoxide; SOC, standard-of-care; QD, once-daily; BID, twice-daily; HRCT, high-resolution computed tomography; 6MWD, six-minute walk distance; LCQ, Leicester cough questionnaire; PK, pharmacokinetics; EOT, end-of-trial; EOS, end-of-study. a-b) Changes in forced vital capacity (FVC) ± 95% CI after 12 weeks of treatment compared to baseline excluding n = 1 patient from the placebo group and n = 1 patient from the rentosertib 30 mg QD group who exhibited >600 mL difference between screening and baseline FVC measurements, making uncertain the baseline FVC values in those patients. Absolute change in FVC ± 95% CI (a) and absolute change in FVC ± 95% CI ANCOVA Model with Multiple Imputation assuming missing at random (MAR) (b). (c) Absolute change in FVC (% Predicted) ± SE, ANCOVA Model with Multiple Imputation assuming MAR. (d) Percentage change in FVC ± SE, ANCOVA Model with Multiple Imputation assuming MAR. (e) Change in percent predicted FVC ± SD. (a) Absolute change in FVC ± 95% CI in patients not concurrently taking SOC antifibrotic therapy (left) or in patients concurrently taking antifibrotic therapy (right). (b) Absolute change in FVC ± 95% CI by ANCOVA Model with Multiple Imputation assuming missing at random (MAR) in patients not concurrently taking SOC antifibrotic therapy (left) or in patients concurrently taking antifibrotic therapy (right). (a) Absolute change in DLCO ± 95% CI. (b) Absolute Change in DLCO (% Predicted) ± SE, ANCOVA Model with Multiple Imputation assuming missing at random (MAR). (c) Absolute change in percent predicted HGB-corrected DLCO (%) ± SD. (d) Absolute change in FEV1 ± SD. (e) Mean percent-change in FEV1 ± SD. (f) Absolute change in LCQ score ± SD. (g) Absolute change in LCQ score ± SE by Mixed model for repeated measures (MMRM). (h) Absolute change in 6MWD ± SD. (i) Absolute change in 6MWD ± SE, ANCOVA Model with Multiple Imputation assuming MAR. (a) AUC0-t of rentosertib at week 0 and week 12, data are mean ± SD. n = 11 at week 0 and n = 11 at week 12 in 30QD, n = 11 at week 0 and n = 10 at week 12 in 30 BID, n = 10 at week 0 and n = 10 at week 10 in 60 QD. (b) t1/2 of rentosertib at week 0 and week 12, data are mean ± SD. n = 11 at week 0 and n = 11 at week 12 in 30QD, n = 11 at week 0 and n = 10 at week 12 in 30 BID, n = 10 at week 0 and n = 10 at week 10 in 60 QD. (c) Correlation analysis of AUC0-t and change in FVC between week 0 and week 12. (d) Correlation analysis of Ctrough measured throughout the trial and change in FVC between week 0 and week 12. P values calculated by two-sided Spearman correlation analysis. (a) Change in Normalized Protein eXpression (NPX) from baseline to subsequent visits (weeks 4, 8, and 12) with a simple linear regression model (delta_NPX ~ treatment_cat) fitted to assess the relationship between treatment dose and protein expression changes using Benjamini-Hochberg (BH)-adjusted P values. (b) Change in NPX between baseline and week 12 by two-sided paired BH-adjusted t-test. Red dots denote proteins with BH-adjusted P value < 0.1. Yellow boxes highlight fibrosis-related proteins. (a) PandaOmics analysis of published RNA-seq gene expression datasets profiling patients with IPF and healthy individuals of four top proteomics hit genes in our trial cohort. (b) Correlation analysis of protein abundance of genes previously associated with lung function and IPF TFS with change in FVC from week 0 to week 12, treatment regimen, and time on treatment. n = 43 patient at each visit, and the P value was calculated by two-sided Pearson correlation analysis; n = 11 patients in Placebo, n = 11 patients in 30 QD; n = 11 patients in 30 BID, n = 10 patients in 60 QD. P values calculated by paired t-test. Correlation analysis of protein abundance of genes (IL10, CD5, COL6A3) previously associated with aging-related dysfunction, such as immunity and ECM remodeling, with change in FVC from week 0 to week 12 and time on treatment. Left panel: n = 43 patient at each visit; P values calculated by two-sided Pearson correlation analysis; Right panel: n = 11 patients in placebo, n = 11 patients in 30 QD; n = 11 patients in 30 BID, n = 10 patients in 60 QD, boxes and whiskers are quartiles; P values calculated by paired t-test. Statistical source data (all figures). Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Xu, Z., Ren, F., Wang, P. et al. A generative AI-discovered TNIK inhibitor for idiopathic pulmonary fibrosis: a randomized phase 2a trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03693-9'>Pooled analysis of 3,741 stool metagenomes from 18 cohorts for cross-stage and strain-level reproducible microbial biomarkers of colorectal cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 10:20:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Associations between the gut microbiome and colorectal cancer (CRC) have been uncovered, but larger and more diverse studies are needed to assess their potential clinical use. We improved CRC prediction accuracy based solely on gut metagenomics (average area under the curve = 0.85) and highlighted the contribution of 19 newly profiled species and distinct Fusobacterium nucleatum clades. Specific gut species distinguish left-sided versus right-sided CRC (area under the curve = 0.66) with an enrichment of oral-typical microbes. We identified strain-specific CRC signatures with the commensal Ruminococcus bicirculans and Faecalibacterium prausnitzii showing subclades associated with late-stage CRC. Our analysis confirms that the microbiome can be a clinical target for CRC screening and characterizes it as a biomarker for CRC progression. It has a 30% higher incidence in men2 and 60–65% of all CRC cases occur in individuals with no previous family history (sporadic cancers)3. Only 40% of cases are diagnosed before metastasis2, with highest survival rates when the tumor is diagnosed at an early stage and a 5-year survival rate for stage IV for colon and rectal cancer of 11% and 15%, respectively4. CRC originates in the epithelial layer of either the proximal or distal colon plus rectum5, usually referred to as right- and left-sided CRC, respectively. Progression from benign precursor lesion (adenoma) to a malignant tumor (carcinoma), termed the adenoma–carcinoma sequence, may take several years6 and is characterized by an accumulation of mutations in tumor cells5, impairment in the gut mucosal barrier and intestinal inflammation7,8. Interest in the tumor microenvironment has increased alongside advances in distinguishing tumor histological features and expression patterns of CRC9, with the gut microbiome suggested as another important hallmark of cancer9. Specific microbes have been proposed as major contributors to carcinogenesis, particularly pks+ Escherichia coli and Fusobacterium nucleatum10,11. Several individual cohort studies and earlier meta-analyses have observed distinct microbiome signatures in patients with CRC when compared with patients with adenomas or healthy controls12,13,14,15,16,17, consistently across different countries and cohorts18,19,20. A few noteworthy metagenomic studies also interrogated microbiome changes along the adenoma–carcinoma sequence and according to primary neoplasia location15,21, and links between CRC and oral species have been suggested15. Further evidence points toward the enrichment of oral-typical microbes (at the genus level21) and of oral biofilm-forming species22 in the gut metagenomes of patients with proximal CRC. However, no metagenomic studies have gone beyond characterizing already well-known strain-specific factors influencing CRC risk (for example, pks island, fragilysin), and no untargeted searches for subspecies and strain-level genomic associations with CRC phenotypes are available. These gaps in the state-of-the-art currently limit the microbiome's potential to be used as a screening tool in clinical settings. Here, we investigated gut microbiome composition along the adenoma–carcinoma sequence and across different primary tumor locations using a meta-analytical approach comprised of an unprecedented number of cohorts (12 public studies and 6 new cohorts generated in this study) and samples (2,116 from public studies and 1,625 from our new CRC cohorts). We also used new computational, statistical and machine learning (ML) strategies to achieve higher profiling resolution extended to previously unknown species and differentiated clades of F. nucleatum23. We established a large and diverse set of gut metagenomic cohorts associated with sporadic CRC and with information on CRC stage (stages 0–IV) and primary tumor location (right-sided or left-sided). To this end, we sequenced 1,625 new stool metagenomes from 6 previously unpublished CRC cohorts (Methods) and integrated them with 2,116 stool metagenomes from 12 public studies. In total, we leveraged 1,471 samples from patients with CRC (1,191 with staging information and 989 with primary tumor location information), 702 from patients with colorectal adenoma and 1,568 from control participants, from 16 case-control and two CRC-only studies (Supplementary Tables 1 and 2). Four of the six newly sequenced cohorts (cohorts 1–4, n = 671) are part of the European ONCOBIOME initiative (Methods and ‘Data Availability'), whereas the fifth (cohort 5) is part of the Micro-N Nurses' Health Study II (NHSII) (n = 897)24. Considering the 3,741 metagenomes in the 18 integrated datasets, we gathered 94 stool metagenomes from patients with stage 0 CRC or carcinoma in situ, and more than 250 for each single stage from stage I to stage IV. In total, 344 samples were from individuals whose primary tumors originated in the right colon (cecum, ascending and transverse colon (10 cohorts)) and 645 samples were from patients whose primary tumors originated in the left colon and rectum (11 cohorts) (Fig. In addition, cohort 1 includes patients with stage IV CRC with either resected primary tumor (n = 68) or in situ primary tumor (n = 95). a, Overview of the cohorts (n = 18) and sample sizes (n = 3,741) according to case-control, cancer stage and primary tumor location, along with the cohorts used to define oral-typical species. b, Number of samples available from each CRC stage and the two primary tumor locations. c, Sample microbiome richness at each stage and primary tumor location. d, Meta-analyzed SMDs of the associations between alpha-diversity (Shannon diversity (upper) and SGB richness (lower)) and all paired comparisons. The 95% CIs for each meta-analysis model are indicated by a horizontal line. P values were computed via two-tailed t-test. Significant associations (P < 0.05) are indicated by a light blue diamond. SMD values corrected for age, sex and BMI (Methods) are indicated by a star (blue when P < 0.05). No correction for multiple hypothesis testing was performed. e, Meta-analysis of the association between the cumulative relative abundance of oral species (oral-to-gut score) and all paired comparisons (left) and between the number of oral species (oral-to-gut richness) and all paired comparisons (right). Symbols and axes are similar to d. P values were computed via two-tailed t-test. No correction for multiple hypothesis testing was performed. SMD values corrected for age, sex and BMI are indicated by a star. f, PERMANOVA (stratified by dataset) derived R2 according to CRC stage and primary tumor location, computed via adonis2 (Methods) on Bray–Curtis distances. Comparisons with P < 0.01 are highlighted in dark blue. Circles indicate the R2 explained by strain-level microbial features, and comparisons with P < 0.01 are marked with an asterisk. 25), which leverages species-level genome bins (SGB)26 to enumerate and quantify characterized (known SGBs (kSGBs) having at least one cultivated reference) and uncharacterized species (unknown SGBs (uSGbs) lacking cultured representatives). In total, we detected 3,866 bacterial, 15 eukaryotic and 23 archaeal SGBs. Some bacterial species spanned multiple SGBs, as was the case for CRC-associated F. nucleatum species for which five SGBs described known and unknown subspecies found by MetaPhlAn 4 (that is, SGB6001, SGB6007, SGB6011, SGB6013, SGB6014), with SGB6007 and SGB6013 recently independently investigated23 and corresponding to F. nucleatum subspecies animalis (Fna) clade 2 (C2) and Fna clade 1 (C1)23. To test the relevance of the presence and overall abundance of oral microbial species in the CRC gut ecosystem, we defined a panel of typically oral SGBs. These were defined on an independent set of 990 matched oral and stool samples from 495 healthy individuals in 5 public microbiome studies27 (Methods). In particular, we considered oral SGBs to be those prevalent (>20%) in the oral microbiome but not (<5%) in the gut microbiome (Methods and Supplementary Table 3). Functional profiles were also generated with HUMAnN 3.6 (ref. 28), and used for a comprehensive analysis on UniRef90 (UR90) gene profiles and corresponding functional grouping according to MetaCyc Pathways, Enzyme Commission (EC) and Gene Ontology (GO) terms. In addition, we investigated within-species phylogenetic structure for uSGBs using StrainPhlAn 4 (ref. 25) and evaluated the resulting 112 within-SGB phylogenies to assess differential strain carriage by CRC phenotypes and for subclade association with CRC-related microbial genes. Consistent with previous reports18, gut microbial alpha-diversity was higher in CRC than controls in 9 of 16 cohorts (SMD > 0, only two with P < 0.05), but this was not a particularly strong effect according to the meta-analytic approach via standardized mean differences (SMD), which was not statistically significant (P ≥ 0.05) (Fig. We observed no clear relationship between richness and clinical stage compared with controls. Estimated oral-to-gut microbiome score (Methods and Extended Data Fig. 2a–e) was instead higher both in CRC cases (Hedges' SMD = 0.47, P < 0.001) (Fig. 1e) and in later CRC stages (Hedges' SMD = 0.14, P = 0.003). In addition, CRC originating from the right colon presented lower richness (Hedges' SMD = 0.25, P = 0.07) (Fig. Stage 0–III microbiomes were not different from stage IV (R2 = 0.01, P ≥ 0.05), and stages 0–II (early) were not different from stages III–IV (late) (R2 = 0.01, Bray–Curtis; PERMANOVA P ≥ 0.05) (Fig. In addition, the microbiome of patients with adenoma did not differ significantly from controls (Fig. 1f and Supplementary Table 5), suggesting a more crucial role for the gut microbiome in the adenoma–carcinoma transition compared with earlier phases. Primary locations (right versus left) showed microbiome differences (R2 = 0.017, P = 0.002) (Fig. 1f and Supplementary Table 5) with no strain-level contribution to the separation (Fig. Altogether, the combined data support the potential of enriched oral microbial infiltration into the gut microbiome as a differentiator of CRC stages and locations (Fig. ML applied to stool metagenomics can be a potential option for noninvasive CRC screening18,19,28. Here, we tested whether leveraging increased sample sizes and methods could further improve predictions of CRC cases. To do so, we exploited ML algorithms models18,28,29 in three different ways: (1) 10-fold cross-validation (CV) applied 20 times on each dataset separately (per-dataset CV); (2) a training–testing approach applied to pairs of distinct datasets (between-dataset CV); and (3) a leave-one-dataset-out (LODO) setting, in which the classifier was trained on all but one dataset and tested on the left-out dataset (iterated over each left-out dataset) (Methods and Fig. a, Cross-prediction matrix of the ML classifier trained and tested to predict CRC versus control samples. Dataset-wise CV AUCs are reported along the diagonal. Bottom six rows report different LODO validations obtained by training a classifier on all the cohorts but one and testing on the left-out cohort (column), on the complete taxonomic profiles (All SGBs), the subset of oral SGBs (Oral SGBs), the subset of all SGBs except the oral SGBs (Nonoral SGBs), gene families clustered according to the EC numbers (EC genes) and MetaCyc pathways (Pathways), respectively, and a filtered set of gene families (Genes) (Methods). Positive values of Hedges' g SMD indicate a positive association with CRC (higher abundance in CRC than controls), while negative values indicate a positive association with controls (higher abundance in controls than CRC). Associations with q < 0.1 are reported in blue. I2 values for heterogeneity in meta-analysis are reported if ≥50% c, Twenty strongest SGBs associated with CRC and ten with controls. Each line on the y axis reports the set of single-dataset and Hedges' model SMD (values on the x axis, with the same directions as in b, represented by symbols and diamonds, respectively. Significant single-dataset comparisons (q < 0.1) are colored dark gray. SMD values corrected for age, sex and BMI (Methods) are indicated by a star (colored blue when q < 0.1). Predictions of CRC status using a LODO approach achieved the highest and most stable area under the curve (AUC) values (average AUC = 0.85, ranging from 0.71 to 0.97) (Fig. 2a) and were an improvement compared with previous studies (average LODO AUC = 0.81)18. Predictions based on CV were, as expected, generally high but variable across datasets (average AUC ± s.d. = 0.87 ± 0.09, ranging from 0.68 to 0.96), with similar results for between-dataset CV (average AUC > 0.72 ± 0.11) (Fig. We then tested the use of only oral and nonoral SGBs for CRC case versus control classification and obtained similar AUC values to the model considering all SGBs (average LODO AUC = 0.83 compared with 0.85 when considering only oral SGBs and 0.79 when considering nonoral SGBs) (Fig. By contrast, ML models using different sets of microbiome functional features were less predictive (average LODO AUC of 0.68 to 0.72). These results reinforce the potential of predictive tools applied to stool metagenomics to be useful for CRC screening when trained on large and diverse datasets and highlight the predictive importance of oral species present in the gut during CRC. We next aimed to pinpoint specific microbiome biomarkers associated with CRC using the increased power of our multicohort framework (Methods). We identified 125 SGBs with increases relative abundance in CRC (q < 0.1, 106 kSGBs and 19 uSGBs) and 83 SGBs more abundant in controls (53 kSGBs and 30 uSGBs) (Fig. Bacterial biomarkers for CRC encompassed not only known associations, such as Parvimonas micra, Gemella morbillorum and Peptostreptococcus stomatis (SMD = 0.63, 0.59 and 0.58, respectively)18, but also newly associated SGBs such as multiple genomically distinct F. nucleatum SGBs (SGB6007 (Fna C2 in ref. 23) F. nucleatum vincentii; SMD = 0.54, 0.5, 0.47, 0.43, and 0.34, respectively) (Extended Data Fig. Only four SGBs were uncharacterized at the species level (that is, belonging to known genera): Solobacterium SGB6833 (distinct form the previously associated Solobacterium moorei18), Peptostreptococcus SGB749 and two Porphyromonas SGBs. A considerable fraction of the identified CRC gut biomarkers were oral-typical species: 21 of the 125 SGBs (16.8%) positively associated with CRC were oral-typical, in contrast to 34 of 488 (7.0%) nonsignificantly associated with CRC (Fisher's test, P < 0.01). No oral SGB was associated with controls and a greater proportion of oral SGBs were associated with CRC at lower q thresholds (18 of 90 at q < 0.05). By reconstructing strain- and subspecies-level phylogenies for oral-typical species associated with CRC via PhyloPhlAn 3 (ref. 30) using metagenome-assembled genomes and isolates, we further identified several subclades of taxa that appear to be more prevalent in the oral cavity or the gut, including clades of F. nucleatum SGB6007 and three Veillonella species (Extended Data Fig. Within oral species, there is thus evidence of genomic adaptation to the intestinal environment. In addition, to better characterize the tropism of the 21 oral SGBs that are more abundant in CRC (Extended Data Fig. 5a), we exploited datasets with dental plaque and tongue dorsum metagenomes from the same individuals31. We determined that 11 SGBs were more abundant in the dental plaque (7 of the top 8 species), whereas 5 were more abundant in the tongue dorsum (Extended Data Fig. 5a), thus hinting at a potential major contribution of biofilm-forming microbes in the intestinal CRC microbiome. To test whether the microbiome biomarkers for CRC were associated with the presence of the primary tumor in the gut, we evaluated the microbiome potential to discriminate between patients with stage IV CRC who had an in situ primary tumor and patients with resected primary tumor from the AtezoTRIBE study. We obtained a LODO AUC = 0.78 with a classifier trained on all the other studies for distinguishing between cases and controls. In addition, 13 (11 oral) of the 20 SGBs most associated with CRC (Fig. 2b) were also significantly (P < 0.05) more abundant when the primary tumor is present rather than resected (Extended Data Fig. Overall, this reinforces the relevance of oral-to-gut introgression by oral commensals18 and that the primary tumor microenvironment determines the overall stool microbiome signature in CRC. We investigated the microbiome's functional repertoire alteration in CRC-affected individuals (Supplementary Table 6). In agreement with previous work18, the cutC gene showed higher abundance in CRC-associated metagenomes (CRC versus control SMD = 0.28; q = 0.001) (Supplementary Table 6). In total, 241 MetaCyc pathways were also positively associated with CRC (q < 0.1) and 68 with controls (Fig. At the enzyme level, sulfur-producing enzymes were associated with CRC (including EC 4.4.1.2 homocysteine desulfhydrase (SMD = 0.52, q < 0.001) and EC 1.8.1.8 protein disulfide reductase (SMD = 0.41, q < 0.001)), consistent with previous work32 (Extended Data Fig. In addition, the association between CRC and two pathways involved in the production of oleate in aerobes (PWY-6282 (SMD = 0.4, q < 0.1) and PWY-7664 (SMD = 0.9, q < 0.1)) (Extended Data Fig. 6, Supplementary Table 6) corroborated a previous hypothesis of an association between oleate and the proliferation of cancerous cells33. We then tested whether increased ammonia levels are characteristic of CRC tumor microenvironments34 and found several pathways and enzymes involved in ammonia production or sequestration that were significantly altered in the presence of CRC. In particular, l-histidine degradation pathways were more frequently encoded in CRC metagenomes (meta-analysis SMD = 0.41, 0.32, respectively, q < 0.1) (Supplementary Table 6). The first step of this pathway involves the enzyme histidase (EC 4.3.1.3) cleaving an amino group off l-histidine to create urocanate and ammonia as by-products, and this histidase was similarly enriched in CRC (SMD = 0.46, q < 0.1), late CRC (SMD = 0.18, P = 0.02) and metastatic CRC metagenomes (SMD = 0.35, q < 0.1). In addition, a second ammonia lyase enzyme, methylaspartate ammonia lyase (EC 4.3.1.2), was also highly associated with CRC metagenomes (SMD = 0.45, q < 0.1) and the opposite for l-histidine biosynthesis (SMD = −0.33, q < 0.1) (Extended Data Fig. The tumor microenvironment of CRC was previously characterized as having increased levels of host-produced ammonia, leading to T cell exhaustion34, but our results suggest a potential role for gut microbiota in ammonia regulation in the tumor microenvironment. To assess whether CRC-associated gut microbiome alterations are stage specific, we considered samples in each stage separately as well as combined into early (0–II) versus late (III–IV) CRC. We observed no strong differences when discriminating between healthy individuals and patients with colorectal adenoma (Fig. By contrast, controls and adenomas were distinct when contrasted to early or late CRC stage metagenomes (Fig. a, LODO AUC predictions based on taxonomic microbiome composition (upper) and microbial functional potential (lower). b,c, Significant species (Hedges' model effect size P < 0.01, none presented q < 0.1) found in association either with earlier or later stages in meta-analysis considering the following comparisons (all presented I2 < 50%). The shape of each point indicates the dataset-specific effect size for each species, and the blue diamonds indicate Hedges' model effect size on SMD. P values were computed via two-tailed t-test. SMD values were corrected for age, sex and BMI and indicated by a star (blue if P < 0.01). We then investigated which microbial SGBs were differentially abundant in early and late CRC stages, as well as in metastatic CRC. We found 17 SGBs associated with late CRC, but only four associated with early CRC (Fig. Among the former were five SGBs of oral origin: Slackia exigua SGB14784, P. micra SGB6653, Solobacterium SGB6833, Dialister pneumosintes SGB5842 and Streptococcus mutans SGB8000. Interestingly, P. micra SGB6653 was already increased in stage I, along with G. morbillorum SGB7295, while F. nucleatum SGB6007, despite being significantly increased in stage I, appeared to be consistently more abundant starting in stage II of CRC (Supplementary Table 8). We note that stage-specific taxa are usually part of the whole CRC versus healthy state signature (8 of the 17 significant SGBs between early versus late CRC) indicating a continuum in microbiome trends along stages rather than distinct configurations. Among the nine SGBs significantly more abundant in stage IV, H. hathewayi SGB4741 showed the greatest increase in abundance (Fig. 3c) with Methanobrevibacter smithii SGB714 also among the top associated species (Fig. Considering pathways, we found four pathways differential between early versus late CRC (P < 0.01) (Extended Data Fig. 8a and Supplementary Table 6), while 14 were increased in stage IV versus all the other stages combined (q < 0.1) (Extended Data Fig. Although few microbial pathways were found to be associated with late or early CRC stages, we confirmed the previously reported association between methane metabolism and stage IV CRC15 (METHANOGENESIS-PWY, SMD = 0.4, P < 0.01) (Supplementary Table 6). We then investigated SGBs showing particularly consistent trends of increased or decreased abundance across all CRC stages (Methods, Fig. Two such examples were P. micra SGB6653 and F. nucleatum SGB6007, in which the abundance started to increase at stage I (Fig. Conversely, Akkermansia muciniphila (SGB9226 and SGB9228) and Parabacteroides distasonis SGB1934 were generally more abundant in later stages of CRC (Fig. These results suggest that the microbiome changes in late-stage CRC17,18,19 occur predominantly on a continuum and become more pronounced as the cancer progresses (Fig. a, Linear mixed model coefficients (in the heatmap cells) showing the associations between each microbial species and each stage when compared with controls. Positive values (from orange to brown) indicate increased stage SGB abundances compared with in controls, while negative coefficients (blue) indicate decreased abundances. Significant associations (q < 0.1) are indicated by a star. Associations also found significant in either right- or left-sided CRC for each stage (q < 0.1) are indicated by a right- or left-pointing triangle, respectively. Box plots represent the distribution of three SGBs with significant changes in the abundances in CRC stages. b, Jaccard similarities between the signatures of sequential CRC stages. c, Overlap for CRC stages signatures with oral SGBs and the species associated with cardiometabolic risk in the PREDICT 1 (P1) study, with T2D, IBD, CD and inflammatory diseases (Intest./Syst. The number of SGBs in each signature is reported at the end of each row. d, LODO AUC for right-sided versus left-sided CRC classification, considering all SGBs, oral SGBs only, nonoral SGBs, EC numbers, MetaCyc pathways or a subset of all the gene families (Methods). e, SGBs significantly associated (q < 0.1) either with right- or left-sided CRC. Meta-analysis Hedges' g is indicated by a diamond, while the SMD values corrected for age, sex and BMI are indicated by a star (blue if q < 0.1). f, SMD values of a meta-analysis of right- versus left-sided tumor-related microbiome composition (y axis) versus the coefficients of a meta-analysis of stages 0–II versus stages III–IV (x axis, left), and 0–III versus stage IV (x axis, right) for taxonomic profiles. ab., arcsine square root transformed relative abundance; C, Candidatus; g, group; Intest., ; N.T., not tested in the corresponding analysis. Because cardiometabolic disorders and CRC share many risk factors35, we quantified the overlap in microbial biomarkers between adenoma and CRC stages with respect to oral species, other human diseases and cardiometabolic markers36 (Fig. SGBs characteristic of CRC stages I–IV included ~25% of species associated with cardiometabolic risk; stage 0 CRC shared the highest proportion of oral bacteria compared with the other stages (58%); stage I CRC showed the highest percentage of species in common with poor cardiometabolic health (27%) (Fig. All CRC stages shared SGBs with Crohn's disease (CD) and immune-mediated diseases (Fig. 4c), while stage IV shared 21% of SGBs with poor cardiometabolic health markers and less with immune-mediated diseases (Fig. Importantly, the microbial signatures of these three conditions have only two SGBs in common (H. hathewayi SGB4741 and Enterocloster aldensis SGB476). Overall, these results indicate a high proportion of SGBs associated with poor cardiometabolic health in all stages of CRC and in adenomas, and a high degree of oral species during CRC development, which is also shared across inflammatory diseases (intestinal or systemic)37,38, and generalized inflammatory conditions characterizing metastatic tumors. We found that differences in the mucosal microbiome according to primary tumor location21,22 extend to stool metagenomics (average AUC = 0.66 across cohorts) when using all SGBs (min = 0.58, max = 0.77) and similarly when limited to oral SGBs (average AUC = 0.6) (Fig. This underscores a difference in microbiome composition that can be relevant for side-specific mechanistic models. Among the 61 tumor location differential SGBs (q < 0.1), we found that three oral-typical SGBs (Veillonella parvula SGB6939, Veillonella atypica SGB6936 and Trueperella pyogenes SGB17137) were significantly increased in right-sided CRC (Fig. In addition, seven of the ten SGBs associated with right-sided CRC (q < 0.1) were nonoral SGBs. Among these, we identified Streptococcus parasanguinis SGB8071, which was shown to form biofilms with Veillonella spp.39, suggesting that such interactions may be more characteristic of right-sided CRC22. Importantly, only a few SGBs were found in common when comparing the signature for primary tumor location with CRC stages: Christensenellaceae NSJ_44 SGB7265 and P. micra SGB6653 when considering early versus late stages (Fig. Marseille P3244 SGB29302 and GB45495 SGB63167 when considering nonmetastatic versus metastatic CRC (Fig. We observed a similar behavior also when considering microbial pathways (Extended Data Figs. 8 and 10), reinforcing the notion that primary tumor location is a subtle but detectable factor to account for when studying gut microbiome alterations in patients with CRC. We then investigated within-species differential gene carriage and diverging phylogenetic features among strains associated with CRC status and stage. We assessed differential gene family carriage (UniRef90s) for 179 SGBs detectable at sufficient prevalence using generalized linear models (GLMs). This identified 62 species that are typically not differentially abundant between cancer conditions, but rather whose strains differentially carried at least one gene family (UR90) in CRC (GLM false discovery rate (FDR) global q < 0.05 and absolute coefficient estimate >1) (Methods and Supplementary Table 9). Many of the species containing the most genetically differential strains were also identified as nonoral CRC-associated, including species of the genus Klebsiella, E. coli, B. fragilis and H. hathewayi (Fig. Nine of the 20 species with the highest number of differentially carried genes had genes enriched in both CRC and controls, highlighting the degree to which strains in the same SGB can differ in functional potential during CRC. Intriguingly, several species carrying a subset of genes solely in CRC were more typically quantified as human commensals, such as Odoribacter splanchnicus and Dorea longicatena, suggesting that some strains of these species may be unusually detrimental or CRC-responsive. a, Top 20 species by the number of significantly differentially carried genes among their strains in CRC (FDR global q < 0.05 and absolute GLM coefficient estimate >1), likely representing species diversity. For the top 20 species, we quantified the number of genes they were differentially carrying along with the dispersion of such genes in CRC and healthy subjects as quantified by Anpan. b, GO terms identified as having the highest ratio of significantly called UniRef90s (genes) in CRC to total genes defined in the term. GO terms are drawn from biological processes (BP) and molecular functions (MF) and are split across known gene families (annotated directly to GO by HUMAnN and predicted from FUGAsseM, https://huttenhower.sph.harvard.edu/fugassem/) using samples with paired metagenomic and metatranscriptomic data to assess the likely function of undescribed gene families. (1) Respiratory electron transport chain; (2) bacterial-type flagellum-dependent cell motility; (3) bacterial-type flagellum organization; (4) amino acid transmembrane transport; (5) site-specific DNA-methyltransferase (adenine-specific) activity; (6) phosphorelay sensor kinase activity; (7) oxidoreductase activity, acting on other nitrogenous compounds as donors; and (8) NAD(P)H dehydrogenase (quinone) activity. c, Carriage of genes previously known to associate broadly with CRC; colibactin, fragilysin and cutC genes, by specific clades' strains in the CRC ecosystem as quantified by Anpan's gene model. Although these genes did not show significance when assessed in species, they did at the global level (for example, when considering total carriage of the function in CRC metagenomes). For each species-gene pair, we quantified the prevalence of the gene's carriage in healthy controls and CRC cases. Many of the genes predicted to be in this category were annotated by FUGAsseM and as such are predictions of the potential function but are otherwise genes of unknown function. AA, amino acid; ETC, electron transport chain; rRNA, ribosomal RNA; TMA, trimethylamine. We then identified pathways specific to CRC-enriched or CRC-depleted strains using a subset of these differential genes that possessed at least some functional characterization. Many molecular functions were altered in CRC-associated microbes. Among the known functions identified using HUMAnN28 (Methods), the SOS response (GO:0009432)—a broad term for cellular response to DNA damage—was more frequently carried by Streptococcus equinus strains in CRC. Gene families from Tyzzerella nexilis and Butyricimonas virosa were also predicted to fall in this GO term (Methods) and be differentially carried in CRC. Consistent with community-wide results, we also found that succinate dehydrogenase activity was more encoded by Parasutterella excrementihominis strains in CRC, which also fits the hypothesis that this fatty acid is more readily available in the CRC-associated ecosystem40 (Fig. Although not significant, the carriage of colibactin-producing genes by E. coli and Klebsiella spp. was increased in CRC (Anpan GLM; q > 0.01 and abs(coefficient estimate) < 1 (a measure of effect size)). This could indicate several potential hypotheses including: (1) that we did not capture the correct time point for an impact of pks+ E coli on CRC progression; (2) low levels of this gene are always encoded and activation is required for the toxicity41; or (3) colibactin is responsible for a minority of CRCs (Fig. We propose similar hypotheses for the B. fragilis toxin fragilysin, for which the gene was not enriched in CRC (Fig. Finally, no significant enrichment of cutC-related enzymes was observed, suggesting that the increased prevalence of this gene was due to the increased abundance of the species carrying this gene, and not the selection for this gene in a species (Fig. Carbon–nitrogen lyase carriage was also of interest (Fig. 5b,d), because genes in this molecular function produce ammonia. Increased ammonia levels have been shown to contribute to T cell exhaustion and suppressed immune activity in CRC, and recently the microbiome was potentially implicated in this process in mice34. Here, we identified five species encoding genes involved in ammonia production in the CRC ecosystem, including Klebsiella oxytoca, O. splanchnicus, Bacteroides intestinalis, P. excrementihominis and Clostridioides difficile (Fig. These included the argininosuccinate lyase gene, which was carried by P. excrementihominis (Fig. 5d) and has ammonia as a known product of its molecular activity34. We then expanded the gene carriage model that indicated the likelihood of distinct strain carriage in species in the CRC ecosystem, with a complementary within-species phylogenetic model via dominant strain profiling at single-nucleotide resolution using StrainPhlAn 4 (ref. This identified several species with an expected log point-wise predictive density (ELPD) of 4, thus carrying dominant strains in distinct phylogenetic lineages in CRC (Fig. We considered a clade significant if the phylogenetic model improved ELPD over a base GLM (the same model without phylogenetic information) by more than a factor of 2. Eight species exhibited differential strain carriage in the broad definition of CRC (stages 0–IV) (Fig. 6a, Supplementary Table 10 and Methods), while only two species were associated with primary tumor location (Fig. Of these associations, only three were identified in species that were also overall significantly differentially abundant (SMD q < 0.1) in the same contrast (Fig. 6a and Supplementary Tables 6 and 9), indicating that subspecies phylogenetic differentiation can be driven orthogonally to the enrichment or depletion of species inhabiting these ecosystems. a, Several species exhibited within-species subclade associations with CRC, late CRC, metastatic CRC and the primary tumor location (right or transverse colon versus left colon or rectum). Only two of the associations were identified in species that were themselves differentially abundant with CRC in the meta-analysis, emphasizing the different pressures on colonization and growth versus phylogeny and evolution. Species are shown if the GLM improvement with phylogeny was greater than an ELPD of 4. b, Within-species subclade clustering obtained via Anpan analysis with CRC as the outcome of interest. Here, we highlighted two examples of Lachnospira eligens and E. rectale (full model results from Anpan in Supplementary Table 10) exhibiting phylogenetically distinct subclades associated with CRC or healthy individuals. For each cladogram, the inner ring is colored by CRC or control, tips are colored by stage and the outer ring is the mean posterior phylogenetic effect as calculated by Anpan's phylogenetic generalized linear mixed model (PGLMM) model with covariates of age, sex and study. c,d, Within-CRC comparisons had more significant hits than global analysis (CRC or control). Here, we present some of the top hits from the model with Collinsella aerofaciens SGB14546, Clostridium fessum SGB4705 and F. prausnitzii 15318 (c) for the metastatic comparisons, and R. bicirculans SGB4262 (d) in late stages. Similar to b, the inner ring is the CRC stage, while tips are metastatic status and early or late, respectively, and the outer ring is the mean posterior phylogenetic effect (full model results from Anpan are given in Supplementary Table 10). e, Significant gene carriage differences by taxon associations in R. bicirculans (Anpan, q < 0.05 and abs(estimate) > 1). Genes differentially carried by R. bicirculans included those in carbohydrate metabolism, DNA mobility, and response to oxidative environments; all genes were found to be more present in late-stage CRC (III–IV) than in early-stage CRC (0–III). Specifically, after controlling for sex, age and study, we identified both Lachnospira eligens (formerly Eubacterium eligens) and Eubacterium rectale strain phylogenies as differential in CRC (Fig. Although E. rectale has previously been shown to exhibit distinct genetics by geographic origin42, we did control for study as proxy of geographic location, thus indicating that this species presents further genetic differentiation in CRC in addition to geography. The Eubacterium genus is generally considered health-associated43, although some publications have hypothesized a potential role for Eubacterium spp. Genetic differentiation among Eubacterium strains may thus help to explain these disagreements. Among other clades, many strains associated with CRC were in uSGBs, including several from Ruminococcaceae (SGB15265, SGB15260, SGB4181) and one from Lachnospiraceae (SGB5089) (Fig. 6a) families, indicating that not yet fully identified species may contribute to the tumor microenvironment. The strongest signals from this phylogenetic lineage analysis differentiated early and late CRC, as well as metastatic and nonmetastatic CRC (Fig. All species identified as having subclades associated with early or late CRC were also identified in the metastatic–nonmetastatic comparisons, likely indicating that stage IV is a driver of these differences, which can also be observed across the two highlighted species, R. bicirculans (Fig. In addition, R. bicirculans was a top hit in both models and was found through the gene carriage model to also carry genes in CRC (Anpan taxon-wise q < 0.1 and abs(estimate) > 1) (Fig. Of potential interest, we identified several genes involved in carbohydrate metabolism as having higher carriage by R. bicirculans strains in later stages of CRC (Fig. 6e), for which it is known that carbohydrate metabolism is altered, and hypothesized that Firmicutes-specific metabolism could promote tumor differentiation46. Noninvasive, early CRC screening and the identification of consistent alterations in microbial components of the tumor microenvironment and bowel still have substantial room for improvement. Metagenomic profiling of the gut microbiome was shown to be highly useful for both tasks. Here, we expanded on previous metagenomic work12,13,14,15,16,17,18,19,20,28 to: (1) improve the accuracy and generalizability of metagenomic-based classifiers across populations; (2) identify additional relevant microbial biomarkers of tumor presence; (3) assess how tumor stage and other clinical variables are linked with specific microbiome configurations; and (4) investigate whether and how strain-level microbial features are linked with tumor presence and stage. By leveraging a total of 3,741 samples from 18 cohorts and applying new strain-level computational methodologies, our study has power and resolution to assess these clinically relevant outcomes. Reproducibility of microbiome signatures in new cohorts and populations is particularly relevant for clinical screening. Based on our results, new cohorts should expect to observe an AUC of ~0.85 in CRC classification based on metagenomics, and this baseline value will further improve as more diverse and larger datasets are incorporated and as uncharacterized species are profiled by sensitive taxonomic profiling approaches25. We found all five SGBs assigned to the F. nucleatum species to be more abundant in CRC than controls, namely F. nucleatum subsp. animalis, vincentii, nucleatum, polymorphum, and a second SGB of vincentii, in decreasing order of association strength. This was in addition to other well-characterized CRC-associated microbes such as P. micra and B. fragilis. We also identified 19 additional uncharacterized SGBs with neither cultivated strains nor taxonomically defined species, highlighting a more complex CRC-associated microbial signature than previously appreciated. Our study also demonstrates that the in situ primary tumor is linked to the usual stool CRC microbiome signature, independent of the sidedness, confirming previous evidence that the primary tumor harbors usual CRC biomarkers, such as F. nucleatum47. Investigations of the gut microbiome changes during early, late and metastatic CRC are key to better characterizing progression along the adenoma–carcinoma sequence. Although interstage microbiome shifts along with CRC progression are not as strong as those observed between CRC and controls, we found several biomarkers for late and metastatic CRC, as well as several microbial species consistently and monotonically increasing (or decreasing) from control to precursor lesion to bona fide cancer or advanced disease. Compared with the other stages, metastatic CRC presented a higher abundance of Methanobrevibacter smithii, supporting previous findings that link methane producers with stage IV CRC15. On primary tumor location, we found that the stool samples derived from patients with CRC originating from the right-sided and transverse colon were also consistently enriched in oral species. Because several oral microorganisms form biofilms when they accumulate in the oral cavity50, they may show the same capacity when they grow in the gut, which is consistent with previous observations of the tumor-free mucosa in patients with right-sided CRC22. Coupled with the observation that left-side originating tumors are more enriched in unclassified Clostridia species, this outcome indicates the potential for small differences in gut microbes based on the location of the primary tumor, which could be related to variation in the tumor microenvironment, and carcinogenic triggers. Prokaryotic species are remarkably genetically and functionally diverse26, and part of the microbiome–CRC link may be because of differences among strains or lineages in SGBs. This study performed a comprehensive strain-level analysis for CRC and found several relevant associations, including the case of otherwise-typical gut clades exhibiting differential dominant strain genetics in CRC, as well as CRC-associated species showing increased encoding of some accessory genes. Such associations were stronger than those for genes known to be directly involved in carcinogenesis (for example, pks island and fragilysin), suggesting more prevalent shifts in microbiome composition for several genes potentially relevant to adaptation to the tumor microenvironment. Dominant strain carriage was particularly associated with CRC stages, with many species (27 of the 213 tested) having significant phylogenetic associations with metastatic disease, all of which were independent of significant species-level abundance changes during CRC. Although even larger investigations are needed to assess the extent to which these strain associations are clinically relevant, they provide targeted potential mechanisms to be validated. We found several common patterns across multiple lines of investigation. These included the role of orally derived bacteria in shaping the gut microbiome in CRC, as previously observed at lower resolution18. We not only strengthened the notion that the number and cumulative abundance of orally derived species are significantly higher in CRC samples than controls and adenomas, but also found that later stages of CRC were particularly enriched for oral species. Similarly, to a lesser extent, patients with right-sided CRC presented a higher number of oral-typical commensals in the gut than left-sided CRC individuals. However, many additional nonoral bacteria were also associated with CRC, including those that have been previously associated with high cardiometabolic risk36. Interestingly, both adenoma and later cancer stages were enriched in species linked with poor cardiometabolic health and immune-mediated diseases, possibly indicating a role for such species as proinflammatory risk factors in CRC. Despite reported evidence that microbiome changes along CRC stages act more like a continuum than as discrete and highly differentiating configurations, we still lack better characterization of the gut microbiome significantly associated with single-stage transition and of its potential impact on the development of distant metastasis. The increased availability of stool-based metagenomic screening tests can be further exploited in new large cohorts to also improve the detection of early-phase microbiome changes occurring in this transition. Our data suggest that translation into clinical application is an option ready to be explored. Our study has some limitations in respect to being an association-based study, thus limiting our conclusions in determining any causal relationship between microbiome configurations and tumor progression and onset, for which, however, independent evidence has been reported10,11. Overall, our study reinforces the robust identification of microbiome biomarkers that can be used in stool-based screening strategies and identifies compositional and structural characteristics of the microbiome associated with disease progression to be prioritized for mechanistic studies. In this study, we performed a pooled analysis expanding the set of publicly available gut microbiome sporadic CRC cohorts with six newly collected and sequenced in house cohorts (cohort 1, 2, 3, 4, 5 and 6), for a total of 1,625 new shotgun gut metagenomes. In total, 555 such new microbiome samples were collected, consistently sequenced and profiled under the ONCOBIOME Consortium, a European effort to unravel associations between intestinal microbiome alterations and different cancer types (https://www.oncobiome.eu). Samples from cohort 5 derived from a subpopulation of the NHSII24,51, and cohort 6 includes CRC samples and controls collected at the Umraniye Training and Research Hospital and the Department of Medical Biology, Yeditepe University (Istanbul, Turkey). Cohort 1 includes stool samples from 163 Italian patients with stage IV CRC enrolled in a multicenter phase II clinical trial (AtezoTRIBE, collected in the University Hospital of Pisa, Italy, NCT number: NCT03721653). Cohort 2 and cohort 3 (COLOBIOME and IIGM-CZ) comprise fecal samples from the Czech Republic collected by two different research institutes (Masaryk University in collaboration with Masaryk Memorial Cancer Institute in Brno and Institute of Experimental Medicine in Prague; n = 204 and 124, respectively). Cohort 5 includes fecal samples from 448 healthy individuals, 435 patients with adenoma and 14 patients with CRC from NHSII. Cohort 6 includes 18 patients with CRC and 39 control individuals from the Umraniye Training and Research Hospital and the Department of Medical Biology, Yeditepe University (Istanbul, Turkey). Public data considered here include four cohorts from China16,52,53,54, and eight cohorts from Austria, France, Germany, India, Italy, Japan, Spain and the United States12,13,14,15,17,18,19,55, respectively (‘Data Availability'). In total, we considered 1,471 CRC samples, 1,191 of which have detailed information about the stage of the disease. In addition, 344 stool samples derived from patients with right-sided CRC, and 645 from patients with left-sided CRC. Tumor staging was defined based on the TNM and AJCC systems56. In particular, it accounts for growth of the tumor to the intestinal wall or nearby organs, with no invasion of lymph nodes (T), the amount of invasion of regional lymph nodes (N) and metastases (M) in distant sites. CRC was categorized based on primary tumor location in two main classes: right-sided CRC, namely originating from the cecum, ascending colon, hepatic flexure and transverse colon; and left-sided CRC, namely originating from the splenic flexure, descending or sigmoid colon, rectosigmoid junction and rectum57. AtezoTRIBE (NCT03721653) is a prospective phase II clinical trial to study upfront systemic regimens in patients with unresectable stage IV CRC. Patients were not subjected to any other treatment at the time of the first stool sample collection. A further 16 samples with uncertain tumor location were considered only to study microbial trends in stages of CRC analysis. Sample collection followed the same procedure as described for cohort 3. AtezoTRIBE is a multicenter study and the protocol was approved by the ethics committees at each participating center. Participants gave written informed consent before enrollment. Patients were enrolled at Masaryk Memorial Cancer Institute (Brno, Czech Republic) from 2015 to 2019, as reported previously21. Patient inclusion criteria were: (1) scheduled for resection based on preliminary screening (such as a colonoscopy), (2) no neoadjuvant treatment, (3) no previous CRC diagnosis and (4) with confirmed stage 0–IV CRC without multiplicities (single tumor). Stool samples were collected from untreated patients before the scheduled surgery. Sixty-four samples derived from individuals affected with CRC primary location in the cecum or ascending colon, 21 from individuals with CRC primary location in the transverse colon and 107 from individuals with CRC primary location in the splenic flexure descending, sigmoid, rectosigmoid or rectum. Patients provided written informed consent according to the Declaration of Helsinki. Stool specimens and clinical and demographic data were collected from 124 Czech individuals recruited in two hospitals in Prague and one in Plzen, Czech Republic58. The individuals included in this study, like those of cohort 4, were not included in a CRC screening program, but because they were considered at risk for CRC and thus recommended to have a colonoscopy test. Based on colonoscopy results, participants were divided into: (1) 59 patients with CRC; (2) 19 patients with colorectal adenoma (13 nonadvanced and 6 advanced adenomas; no serrated lesions were collected); and (3) 38 colonoscopy-negative individuals and with 8 hyperplastic polyps58. All the samples from CRC cases were collected at diagnosis, before any treatment. Naturally evacuated fecal samples were obtained from all participants previously instructed to self-collect the specimen at home. Stool samples were collected in nucleic acid collection and transport tubes with RNA stabilizing solution (Norgen Biotek) and returned to the endoscopy unit. All the samples were from sporadic CRC cases, collected at diagnosis before any treatment. Samples were collected in the same way as cohort 3. The local ethics committees of Azienda Ospedaliera SS. Colorectal miRNA CEC2014), AOU Città della Salute e della Scienza di Torino (Italy), the Institute of Experimental Medicine of Prague (Czech Republic), Masaryk Memorial Cancer Institute (protocol no. 2018/865/MOU) and Masaryk University of Brno (Czech Republic, protocol no. All patients gave written informed consent following the Declaration of Helsinki before participating in the study. NHSII is a cross-sectional, prospective study of CRC-related gut microbial composition. Participants provided written informed consent before study enrollment and stool collection. Specifically, this study recruited a subpopulation of NHSII24,51, a long-running prospective cohort from the United States. All participants contributed a stool sample, with the research aiming to investigate the role of the gut microbiome specifically in participants with recent adenomas. The adenoma (n = 435) and CRC cases (n = 14) in the study were one-to-one matched with healthy control samples (n = 448) based on age at stool collection, ethnicity, month of collection, state of residence and total number of, reason for and date of recent endoscopy. For a subset of cases (n = 39) the matching criteria for ethnicity (expanded definition of Caucasian) and age at collection were relaxed. Adenomas were further defined as high or low risk based on the location, number and histology of the cells. The CRC samples ranged from all stages and were limited as only a limited number of participants in NHSII have been diagnosed with CRC and recently contributed a stool sample. Previously stored, Genotek's OMNIgene fixed, and −80 °C frozen stool samples were processed and sequenced for shotgun metagenomics at Diversigen. Patients with CRC were recruited at the Umraniye Training and Research Hospital while healthy volunteers contributing to science used as controls were recruited at the Department of Medical Biology, Yeditepe University (both in Istanbul, Turkey). Naturally evacuated fecal samples were obtained from subjects previously instructed to self-collect the specimen at home. For patients with CRC, collection was performed before surgical resection. Samples from participants who had used antibiotics within 1 month before the sample collection were excluded. Samples were collected in nucleic acid collection and transport tubes with RNA stabilizing solution (Norgen Biotek). Stool aliquots (200 μl) were stored at −80 °C until RNA extraction. DNA was extracted from stool samples with the DNeasy PowerSoil Pro Kit (Qiagen), and sequencing libraries were prepared using the Illumina DNA Prep, (M) Tagmentation kit (Illumina), following the manufacturer's guidelines. The library pool was subjected to a cleaning step with 0.7× Agencourt AMPure XP beads. Samples were sequenced on a NovaSeq 6000 S4 flow cell (Illumina) at the University of Trento sequencing facility. Sequenced metagenomes were preprocessed using the pipeline available at https://github.com/SegataLab/preprocessing for: (1) removal of low-quality reads (quality <20), too short fragments (length <75 bp), and reads with two or more ambiguous nucleotides; (2) host contaminant DNA removal using Bowtie 2 (ref. Once preprocessed, ONCOBIOME samples presented an average of 37 million reads. A final clean-up of the library pool was performed with 0.6× AMPure XP beads (Beckman-Coulter), and then resuspended with one-third of the initial pool volume. For DNA extraction and sequencing, samples were sent to Diversigen and all steps were completed according to their standardized DEEPSEQ protocol. This used Powerbead Pro Plates (Qiagen) with 0.5 and 0.1 mm ceramic beads, but otherwise followed the manufacturer's protocol. DNA amount and quality were assessed with a Quant-iT PicoGreen dsDNA Assay (Invitrogen) post extraction. Sequenced samples were then filtered for host contamination via the KeandData pipeline (https://github.com/biobakery/kneaddata). In particular, this pipeline consists of three main steps: a first trimming of poor-quality reads with trimmomatic60, specifically we applied a sliding window trim removing reads after four subsequent bases had a Phred score of 20 or less, and then reads with fewer than 60 base pairs were removed. Finally, host and common sequencing components decontamination was completed with Bowtie 2 against PhiX and the human genome (hg37). We applied MetaPhlAn 4 (v.4.0.0, database vJan21, with the ‘--statq 0.1')25 and HUMAnN 3.6 (ref. 28) profiling tools to produce microbial taxonomic and functional profiles, respectively. In addition, StrainPhlAn 4 (v.4.0.3)25 was run to generate dominant single nucleotide variant profiles for any species that passed the filtering steps in StrainPhlAn (213; species are filtered for sufficient markers and samples to run the tool). Newly sequenced samples and public data considered in this study were profiled consistently. We considered metagenomic samples from 11 public CRC–control studies, 8 of which had already been included in previous meta-analyses18,19. Metagenomic samples for three additional public studies (Liu, N.N. (2021)53) were available in the European Nucleotide Archive (ENA) (accession numbers: PRJNA731589, PRJNA429097 and PRJNA763023, respectively). This cohort12 comprises 154 Austrian individuals (61 controls, 47 adenomas and 46 CRC). This cohort13 includes 60 stool samples from the same number of individuals from India (equally distributed between Bhopal and Kerala) divided into 30 controls and 30 CRC cases. All the fecal samples in this cohort were collected from people who were not subject to antibiotics close to the sampling date and had not been diagnosed with other diseases. This cohort54 comprises 164 stool samples from an equal number of individuals from China (85 controls and 79 CRC cases). The cohort derives from the ‘Chinese cohort in Shanghai (CHN_SH)', whose patients were sampled after CRC diagnosis and before any treatment. All the cases include exclusively sporadic CRC. Control individuals were recruited in the Taizhou Imaging Study. Age, sex, body mass index (BMI) and case or control condition were retrieved from the original publication and the corresponding ENA portal project (PRJNA731589). Participants were asked to collect the stool sample 1 week before colonoscopy preparation and participants who reported use of antibiotics or probiotics within 1 month before sample collection were excluded. This is ‘cohort 2' of the study18 and comprises 60 stool samples, collected from the same number of individuals recruited at the European Oncology Institute in Milan, Italy. In particular, the cohort consisted of 28 controls and 32 CRC cases for which no staging or primary tumor location information was available. For CRC cases, sampling was performed before surgery or any cancer treatment. In total, 110 stool samples were collected from an equal number of US individuals divided into 58 controls and 52 CRC cases (12 stage II, 21 stage III and 18 stage IV; 15 right-sided CRC, 32 left-sided CRC). CRC samples were collected before surgery or any other cancer treatment14. This cohort19 comprises 125 stool samples, collected from the same number of German individuals. CRC samples were recruited in the ColoCare study and fecal samples were collected after colonoscopy. Control samples were recruited in the PRÄVENT study. Stool samples were excluded if individuals used antibiotics, were subjected to radiotherapy or corticosteroids in the month before the sampling. Age, sex, case or control, TNM, stage and primary location were obtained from the original publication and the corresponding ENA portal project (PRJNA429097). This cohort53 comprises 200 stool samples from the same number of individuals from the Fudan cohort (China) and includes 100 controls and 100 CRC cases. Only samples from individuals who did not use antibiotics or probiotics for 1 month before recruitment were included in the study. CRC stool samples were collected before colonoscopy or other cancer therapies and surgery. Only sporadic CRC cases were included, with no history of inflammation-associated CRC, intestinal bowel syndrome or other cancers. Disease categories (CRC or control) were retrieved from the original publication; raw metagenomes were obtained from the ENA portal accession number PRJNA763023. This study15 comprises 616 stool samples from the same number of Japanese individuals subjected to colonoscopy. Only sporadic CRC cases were considered, with no inflammatory bowel disease (IBD) or abdominal surgical history. This cohort16 comprises 128 samples from the same number of individuals, collected in Hong Kong, China, and included 53 controls and 75 CRC cases (12 Stage I, 24 Stage II, 24 Stage III, and 8 Stage IV; 11 right-sided and 54 left-sided CRC). For the definition of the oral signature, we collected the data available from 5 datasets for a total of 495 healthy individuals for whom both stool and oral (either from saliva or tongue dorsum) samples were available for the same subject and time point. The oral signature was defined based on the distribution of specific microbial species that met the following criteria: (1) present exclusively in the oral cavity of at least 20% of participants; (2) found in both the oral cavity and stool of fewer participants than those that were exclusively oral; and (3) present exclusively in stool in fewer than 5% of participants. These constraints resulted in a signature of 235 oral-typical species (Supplementary Table 3). PERMANOVA was performed using the adonis2 function from the vegan R package with 999 permutations and blocked for study of origin by the setBlocks function, with and without including age, sex and BMI in the model. To quantify gut colonization by typically oral commensal species (defined in the previous section), we developed two quantitative scores. The ‘oral-to-gut richness,' in contrast, counts the number of distinct oral SGBs present in each stool sample. Because this work is a multicohort study and a batch effect exists in data from different origins, we used the meta-analysis of SMDs computed in each dataset instead of effect sizes computed from batch-effect corrected data as the primary approach for biomarker discovery. Our choice is motivated by the fact that correcting for batch effect is a difficult task, because of both incomplete information on batch effects (not only between cohorts, but also within cohorts), and the lack of a consensus approach for batch correction in microbiome studies. In particular, SMDs were computed with Hedges' method66 which adds a correction for low sample bias to Cohen's d estimator. Between-study variance (𝜏2) was estimated via restricted maximum likelihood and CIs of the summary effect were adjusted with the Hartung and Knapp method67. This procedure was applied to all microbiome features with at least 10% prevalence and present in at least five samples in one of the testing sets when at least three studies presented a minimum of ten samples for each class. Significance was determined as Benjamini–Hochberg q < 0.1 or P < 0.01. Meta-analysis of standardized linear model estimates was applied to determine the effect sizes corrected for age, sex and BMI. Per-cohort linear models were fitted for each feature relative abundance (arcsine square root transformed) with the additive effect of age, sex and BMI. An ensemble of 1,000 trees with a minimum of 5 samples per leaf (grid-search optimal max features per split to consider in CV, and no other normalization performed) was trained and tested in the following settings on the data: (1) per-dataset CV (10-fold CV repeated 20 times); (2) across-study prediction (for each pair of studies the classifier is trained on one and tested on the other); and (3) LODO approach (each cohort becomes the testing set while all the others are used for training). CV comparisons were considered when presenting at least 15 samples for each class in one cohort, whereas between-dataset CV and LODO comparisons were considered when 15 samples were available in each class both in the training and validation sets. NHSII was included in the training set for comparison of controls versus CRC in LODO, but was not considered a validation cohort for the unbalanced sample sizes between the two classes. This setup was extensively applied in previous works18,28, allowing for robust comparison of our results with those in the literature. Relative abundance profiles (values in the [0, 1] range) were previously arcsine square root transformed. When testing for oral-typical or non-oral species, after selection, we rescaled the relative abundance to [0,1], and then transformed via arcsine square root. No other feature selection was performed otherwise. For the reasons mentioned above, we decided not to integrate the studies in a single large cohort and perform batch-effect correction, previous to ML. Our approach treats each study independently and tests the strength of the trained model in the same study (per-dataset CV), in a different study (across-studies prediction) or in the left-out study when validated in LODO. This ensures that batch-effect correction does not introduce favorable bias in the classification tasks. Linear mixed models, via the MaAsLin 2 R package68, were iteratively applied fitting each microbial abundance profile (after arcsine square root transformation) with sample condition (control, adenoma, CRC stage 0, I, II, III, IV) as the fixed effect, and originating cohort as the random effect. For the fitting of each model, species were considered if they were at least 10% prevalent across all controls, adenoma and stages 0–IV. We applied MaAsLin 2 with the same setting to test SGB differential abundant between primary tumor locations in stage IV CRC. In both analyses, only associations with q < 0.1 were considered significant. To complete subspecies clade-level association analysis with CRC, we used Anpan (v.0.3.0, https://huttenhower.sph.harvard.edu/anpan)69, an R package that quantifies the associations between differential gene carriage, subspecies phylogenetic structure and host phenotypic outcomes. The gene model in Anpan addresses two key issues: robust and accurate detection of species whose genes are well-covered in shotgun metagenomes; and sensitive detection of consistently associated genes with phenotypic outcomes. Anpan first filters samples to remove any without enough species-specific gene coverage to accurately assess gene-level effects. A GLM was then used to model each gene's association with the outcome (accounting for metadata covariates), followed by FDR correction. Here, our outcomes of interest were CRC or control, early- or late-stage CRC and primary tumor location (right-sided versus left-sided), adjusted for age, sex and study (which accounts for the geographic location of collection). From the predicted genes with Anpan, we first quantified the number of significant hits per species, with a significance threshold of an absolute coefficient of 2 and q < 0.05. Next, we regrouped the UniRef90 genes to GO terms by direct matching. We also used annotations from FUGAsseM to add predicted GO term annotations based on metagenomic and metatranscriptomic covariation patterns. It extends ‘guilt by association' approaches by building an individual random forest classifier predicting gene function based on each data type, followed by an ensemble tier that builds an integrated classifier combining the learning results from the first tier. As a result, putative functional annotations are assigned to uncharacterized proteins that achieve high prediction probability. Next, we assessed the phylogenetic associations with our outcomes of interest. StrainPhlAn trees (http://segatalab.cibio.unitn.it/tools/strainphlan/) were used as input to phylogenetic generalized linear mixed models to assess phylogenetic associations with the outcomes. Phylogenetic generalized linear mixed models are probabilistic models that account for phylogenetic structure by encoding the tree structure as a correlation matrix. For these models, we also used age and sex as covariates and study as an offset variable. We referred to associations as hits if the phylogenetic model improved the ELPD over a base GLM (the same model except without phylogenetic information) by more than 2. ELPD is a model comparison metric akin to the Akaike information criterion. We tested whether the inclusion of strain-related microbial features in a PERMANOVA leads to improvement in the model association or prediction. Because phylogenetic information was already tested via anpan, we developed a complementary approach for defining strain-level features based on strain preferences for a given nucleotide in marker genes. In particular, starting from StrainPhlAn 4 reconstructed multiple sequence alignments of the marker genes for the 213 SGBs, we selected genetic positions in marker genes with binary entropy at least 0.5 (to remove positions with little strain-level variability), by selecting those that presented the minimum number of gaps in clusters of 1-ANI (average nucleotide identity) ≤0.05 (to remove features very correlated to each other). We then expanded each position into five features using one-hot encoding, one for each nucleotide and one representing a gap. The value in each of these features can be either 1 when the corresponding nucleotide or gap is present in that position, or 0 otherwise. In this way, we obtained a total of 1,382,825 features across all the SGBs. Given the large number of features produced, we then applied an additional set of thresholds based on prevalence and removing collinear ones. In particular, features with <20% prevalence or prevalence >80% in the controls and CRC samples were removed, ensuring that very rare or too common base preferences were not considered, thus obtaining 42,094 features. We then removed features highly correlated (Pearson correlation >0.5) with any other feature, selecting the first occurrence as the representative and discarding all other features that correlated with it with a higher absolute Pearson coefficient than the threshold selected (0.5). This step produced a set of 2,722 features for the 0.5 Pearson correlation threshold. PERMANOVA tests with this feature set were performed as described earlier in Methods, specifying Jaccard as the distance measure. The cardiometabolic microbial signature was estimated, as reported in our previous work25,36, as the species most associated with the set of cardiometabolic indices defined in ref. In brief, partial Spearman correlations were computed between each SGB and the set of indices associated with cardiometabolic risk, correcting for sex, age and BMI. Partial correlations were ranked and averaged first in each category and then across categories to derive a global rank. Ranks ranged between 0 and 1 for the most favorable and unfavorable species, respectively, and we considered those SGBs with a rank above the third quartile of the distribution. We identified 115 SGBs representing a higher cardiometabolic risk and that account for 2.97% of the detected SGBs across all analyzed cohorts. We compared the signatures found for CRC in meta-analysis with signatures for other disease types or groups or diseases. Specifically, we searched in the curatedMetagenomicData 3 repository for case-control studies for T2D (control n = 882, cases n = 750, four cohorts)70,71,72,73, ulcerative colitis (n = 247 and 84), CD (n = 291 and 83, three cohorts in total)38,74,75, inflammatory diseases (including asthma, Behcet syndrome, multiple sclerosis, rheumatoid arthritis and myalgic encephalomyelitis, n = 918 and 827, five cohorts)76,77,78,79,80. IBD was obtained with a quarry of ulcerative colitis and CD. Then, to compare the microbial signature associated with CRC, we arcsine square root transformed all the MetaPhlAn 4 SGB-level relative abundances, we performed a meta-analysis of SMDs computed starting from a linear regression linking the disease state to the SGB transformed abundance and adjusted by country to take into account potential population effect. SMDs and uncertainty estimations were meta-analyzed via inverse variance weighting using Paule–Mandel heterogeneity. From the resulting tables, signatures for the six disease types were retrieved by selecting those SGBs having an FDR for the meta-analysis P value <0.1 and being found in a minimum of three datasets. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The NHSII cohort is available in NCBI Sequence Read Archive (SRA) with the project id PRJNA1237248. Metagenomic sequences for Cohort 6 are available in NCBI via the project number PRJNA1167935. MetaPhlAn 4 and HUMAnN 3.6 profiles and metadata for the cohorts included in this study are available via Zenodo at https://doi.org/10.5281/zenodo.15069069 (ref. No original tool was developed for this manuscript. All the applied approaches have been previously published or are publicly available and referenced in the manuscript. Global cancer statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. Keum, N. & Giovannucci, E. Global burden of colorectal cancer: emerging trends, risk factors and prevention strategies. Miller, K. D. et al. Cancer treatment and survivorship statistics, 2022. Stem vs non-stem cell origin of colorectal cancer. Comparative lesion sequencing provides insights into tumor evolution. Elinav, E. et al. Inflammation-induced cancer: crosstalk between tumours, immune cells and microorganisms. Schmitt, M. & Greten, F. R. The inflammatory pathogenesis of colorectal cancer. The human gut bacterial genotoxin colibactin alkylates DNA. Gut microbiome development along the colorectal adenoma–carcinoma sequence. Gupta, A. et al. Association of Flavonifractor plautii, a flavonoid-degrading bacterium, with the gut microbiome of colorectal cancer patients in India. Vogtmann, E. et al. Colorectal cancer and the human gut microbiome: reproducibility with whole-genome shotgun sequencing. Yachida, S. et al. Metagenomic and metabolomic analyses reveal distinct stage-specific phenotypes of the gut microbiota in colorectal cancer. Yu, J. et al. Metagenomic analysis of faecal microbiome as a tool towards targeted non-invasive biomarkers for colorectal cancer. Thomas, A. M. et al. Metagenomic analysis of colorectal cancer datasets identifies cross-cohort microbial diagnostic signatures and a link with choline degradation. Wirbel, J. et al. Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer. Zwinsová, B. et al. Colorectal tumour mucosa microbiome is enriched in oral pathogens and defines three subtypes that correlate with markers of tumour progression. Dejea, C. M. et al. Microbiota organization is a distinct feature of proximal colorectal cancers. Zepeda-Rivera, M. et al. A distinct Fusobacterium nucleatum clade dominates the colorectal cancer niche. Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4. Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. Accessible, curated metagenomic data through ExperimentHub. Integrating taxonomic, functional, and strain-level profiling of diverse microbial communities with bioBakery 3. eLife 10, e65088 (2021). Pasolli, E., Truong, D. T., Malik, F., Waldron, L. & Segata, N. Machine learning meta-analysis of large metagenomic datasets: tools and biological insights. Precise phylogenetic analysis of microbial isolates and genomes from metagenomes using PhyloPhlAn 3.0. Structure, function and diversity of the healthy human microbiome. Nguyen, L. H. et al. Association between sulfur-metabolizing bacterial communities in stool and risk of distal colorectal cancer in men. Zhang, Y. et al. Oleic acid and insulin as key characteristics of T2D promote colorectal cancer deterioration in xenograft mice revealed by functional metabolomics. Bell, H. N. et al. Microenvironmental ammonia enhances T cell exhaustion in colorectal cancer. Global, regional, and national burden of colorectal cancer and its risk factors, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Asnicar, F. et al. Microbiome connections with host metabolism and habitual diet from 1,098 deeply phenotyped individuals. Thompson, K. N. et al. Alterations in the gut microbiome implicate key taxa and metabolic pathways across inflammatory arthritis phenotypes. Lloyd-Price, J. et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. & Mazlan, M. Metabolomics profiling on different stages of colorectal cancer: a systematic review. Dougherty, M. W. & Jobin, C. Shining a light on colibactin biology. Analysis of 1321 Eubacterium rectale genomes from metagenomes uncovers complex phylogeographic population structure and subspecies functional adaptations. Mukherjee, A., Lordan, C., Ross, R. P. & Cotter, P. D. Gut microbes from the phylogenetically diverse genus Eubacterium and their various contributions to gut health. Wang, Y. et al. Eubacterium rectale contributes to colorectal cancer initiation via promoting colitis. Gut microbiota Eubacterium callanderi exerts anti-colorectal cancer activity. Gut microbiota and colon cancer: the carbohydrate link. Roelands, J. et al. An integrated tumor, immune and microbiome atlas of colon cancer. Zhao, L. et al. Parvimonas micra promotes colorectal tumorigenesis and is associated with prognosis of colorectal cancer patients. Xia, X. et al. Bacteria pathogens drive host colonic epithelial cell promoter hypermethylation of tumor suppressor genes in colorectal cancer. Oral microorganisms and biofilms: new insights to defeat the main etiologic factor of oral diseases. The sulfur microbial diet is associated with increased risk of early-onset colorectal cancer precursors. Yang, J. et al. Establishing high-accuracy biomarkers for colorectal cancer by comparing fecal microbiomes in patients with healthy families. Yang, Y. et al. Dysbiosis of human gut microbiome in young-onset colorectal cancer. Liu, N.-N. et al. Multi-kingdom microbiota analyses identify bacterial–fungal interactions and biomarkers of colorectal cancer across cohorts. Obón-Santacana, M. et al. Meta-analysis and validation of a colorectal cancer risk prediction model using deep sequenced fecal metagenomes. Amin, M. B. et al. (eds) in AJCC Cancer Staging Manual 8th edn, Ch. Difference between left-sided and right-sided colorectal cancer: a focused review of literature. Pardini, B. et al. A fecal microRNA signature by small RNA sequencing accurately distinguishes colorectal cancers: results from a multicenter study. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Benson, G. Tandem repeats finder: a program to analyze DNA sequences. Mobile genes in the human microbiome are structured from global to individual scales. Ferretti, P. et al. Mother-to-infant microbial transmission from different body sites shapes the developing infant gut microbiome. Kartal, E. et al. A faecal microbiota signature with high specificity for pancreatic cancer. Nagata, N. et al. Metagenomic identification of microbial signatures predicting pancreatic cancer from a multinational study. Hedges, L. V. Distribution theory for Glass's estimator of effect size and related estimators. & Knapp, G. On tests of the overall treatment effect in meta-analysis with normally distributed responses. Mallick, H. et al. Multivariable association discovery in population-scale meta-omics studies. Ghazi, A. R., et al. Quantifying metagenomic strain associations from microbiomes with Anpan. Gut metagenome in European women with normal, impaired and diabetic glucose control. Qin, J. et al. A metagenome-wide association study of gut microbiota in type 2 diabetes. Forslund, S. K. et al. Combinatorial, additive and dose-dependent drug–microbiome associations. Identification and assembly of genomes and genetic elements in complex metagenomic samples without using reference genomes. He, Q. et al. Two distinct metacommunities characterize the gut microbiota in Crohn's disease patients. Gut microbiome of multiple sclerosis patients and paired household healthy controls reveal associations with disease risk and course. The oral and gut microbiomes are perturbed in rheumatoid arthritis and partly normalized after treatment. Nagy-Szakal, D. et al. Fecal metagenomic profiles in subgroups of patients with myalgic encephalomyelitis/chronic fatigue syndrome. Zhou, C. et al. Metagenomic profiling of the pro-inflammatory gut microbiota in ankylosing spondylitis. Shotgun metagenomics of 250 adult twins reveals genetic and environmental impacts on the gut microbiome. Piccinno, G. et al. Pooled analysis of 3,741 stool metagenomes from 18 cohorts for cross-stage and strain-level reproducible microbial biomarkers of colorectal cancer. Further, NHSII work cannot be completed without the constant support of the Channing Division of Network Medicine, Department of Medicine, Brigham and Women's Hospital as home of the Nurses' Health Studies, which is vital in the collection and storage of all data relating to NHSII. The research leading to these results has received funding from the European Union's Horizon 2020 research and innovation programme under grant agreement no. This work was also delivered as part of the PROSPECT team supported by Cancer Grand Challenges partnership funded by Cancer Research UK (grant nos CGCATF-2023/100036, CGCATF-2023/100041), the National Cancer Institute (grant nos OT2CA297680, 1OT2CA297205-01), the Bowelbabe Fund for Cancer Research UK and Institut National Du Cancer (to A.T.C., C.H., N.S.). This study was also supported by the National Cancer Institute of the National Institutes of Health grant nos 1U01 CA230551 (to N.S. ); American Cancer Society Research Professor (to A.T.C. ); the European Research Council (ERC-StG project MetaPG-716575 and ERC-CoG microTOUCH-101045015 to N.S. ); the Premio Internazionale Lombardia e Ricerca 2019 (to N.S. ); the Associazione italiana per la ricerca sul cancro AIRC under IG 2020—ID 24882—P.I. ); the European Union's Horizon 2020 research and innovation program under grant agreement no. also thank RECETOX RI (grant no. LM2023069) financed by the Czech MEYS for supportive background. was partially funded at UNIPI by European Union—NextGenerationEU through the Italian Ministry of University and Research under PNRR - M4C2-I1.3 Project PE_00000019 ‘HEAL ITALIA' to Chiara Cremolini CUP: I53C22001440006, and by PRIN2022 CUP: I53D23005120006; L.Z. was supported by ANR RHU5 ‘ANR-21-5 RHUS-0017' IMMUNOLIFE', MAdCAM INCA_ 16698, by the European Research Council (ERC) under grant agreement no. 964590 (project acronym: IHMCSA, project entitled International Human Microbiome Coordination and Support Action), by the European Union's Horizon Europe research and innovation program under grant agreement no. thank G. Roussy for the CLINICOBIOME PMS support. This publication reflects only the authors' view, and the European Commission is not responsible for any use that may be made of the information it contains. These authors jointly supervised this work: Curtis Huttenhower, Alessio Naccarati, Eva Budinska, Nicola Segata. Gianmarco Piccinno, Paolo Manghi, Andrew Maltez Thomas, Aitor Blanco-Míguez, Francesco Asnicar, Katarina Mladenovic, Federica Pinto, Federica Armanini, Michal Punčochář, Elisa Piperni, Vitor Heidrich, Gloria Fackelmann & Nicola Segata Chan School of Public Health, Boston, MA, USA Chan School of Public Health, Boston, MA, USA Kelsey N. Thompson, Andrew R. Ghazi & Curtis Huttenhower IEO, European Institute of Oncology, IRCCS, Milan, Italy Sonia Tarallo, Barbara Pardini & Alessio Naccarati Sonia Tarallo, Barbara Pardini & Alessio Naccarati Clinical and Translational Epidemiology Unit, Massachusetts General Hospital, Boston, MA, USA Long H. Nguyen, Mingyang Song & Andrew T. Chan Graduate School of Natural and Applied Sciences, Yeditepe University, Istanbul, Turkey Biomedical Center, Faculty of Medicine in Pilsen, Charles University, Prague, Czech Republic Veronika Vymetalkova, Vaclav Liska & Pavel Vodicka Department of Colorectal Surgery, Clinica S. Rita, Vercelli, Italy Department of Surgery, University Hospital and Faculty of Medicine in Pilsen, Charles University, Prague, Czech Republic Department of Hepatogastroenterology, Institute for Clinical and Experimental Medicine, Prague, Czech Republic Masaryk Memorial Cancer Institute, Brno, Czech Republic RECETOX, Faculty of Science, Masaryk University, Brno, Czech Republic Martina Čarnogurská, Vlad Popovici & Eva Budinska Department of Translational Research and New Technologies in Medicine and Surgery, University of Pisa, Pisa, Italy Chan School of Public Health, Boston, MA, USA Faculté de Médecine, Université Paris-Saclay, Kremlin-Bicêtre, France Institut National de la Santé Et de la Recherche Médicale (INSERM) U1015, Equipe Labellisée—Ligue Nationale contre le Cancer, Villejuif, France You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar coordinated the project in ONCOBIOME and PROSPECT. is a founder and shareholder of PreBiomics Srl and is on the scientific advisory board of ZOE Ltd and received consultancy fees from them. received research contract fundings from Daiichi Sankyo and Biomérieux. The other authors declare no competing interests. Nature Medicine thanks Amiran Dzutsev and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Joao Monteiro, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Richness of microbial enzymes and pathways in controls, adenomas, the different CRC stages and primary tumor locations. b) t-SNE based on Bray-Curtis dissimilarity for taxonomic and functional profiles. The cohort derivation (either from ONCOBIOME, NHSII or public studies) is indicated as the shape of each point (triangle for ONCOBIOME, square for NHSII and a circle for public cohorts). Relative abundance values higher than this threshold are represented with the same color. E) Meta-analysis effect sizes (Hedges' g) of the oral-to-gut score computed based on different thresholds of the ‘Exclusive oral prevalence'. A) The left-most phylogenetic tree shows the relationships between F. nucleatum SGBs and previously characterized F. nucleatum subspecies. Each leaf represents either an isolated genome (circle) or a metagenome-assembled genome (MAG, reported as a triangle), as available in the Jun23 release of the MetaPhlAn 4 database. B) Phylogenetic reconstruction via PhyloPhlAn 3 of isolate and MAGs either derived from oral, stool, or unknown body site for Fusobacterium nucleatum SGB6007 (F. nucleatum subspecies animalis), Veillonella atypica SGB6936, Veillonella rogosae SGB6956, Veillonella dispar SGB6952, Streptococcus mutans SGB8000, and Streptococcus parasanguinis SGB8071. Significant SGBs detected via the meta-analysis for the comparisons tested in the manuscript (controls vs CRC, early stages CRC vs late stages CRC, non-metastatic vs stage IV CRC and right-sided vs left-sided CRC) are in black (q < 0.1) and red (P < 0.01). The comparison controls vs CRC showed the highest Spearman's correlation coefficient (0.41), with right-CRC vs left-CRC Spearman's correlation coefficient of 0.21. This confirms the stronger biomarkers obtained by these comparisons, while the comparisons on stages presented non-significant correlation coefficients (Spearman's correlation coefficient of 0.06 and 0.05 for early-stages vs late-stages and non-metastatic vs metastatic CRC, respectively). A) Differential abundant oral SGBs in the signature of CRC between the oral plaque and tongue dorsum in the Human Microbiome Project (HMP) study. We selected 86 participants of the HMP study with matched microbiome samples for the two oral cavity sites. Differential abundance analysis was performed via Wilcoxon signed-rank test and adjusted p-values (q) were computed via Benjamini-Hochberg (BH) correction. log2 fold-change (FC) between average relative abundances (RA) in the two sites is reported in the heatmap. B-D) Validation of the machine learning and CRC gut microbiome signature between patients with in situ or resected primary tumor in the AtezoTRIBE study. Differentially abundant microbial pathways (meta-analysis model q < 0.1) between controls and CRC. Associations with CRC present Hedges' g SMD > 0, and associations with controls present Hedges' g SMD < 0. Significant single-dataset comparison (q < 0.1) is reported in dark gray, while non-significant single-dataset associations (q ≥ 0.1) is in lighter gray. A) Meta-analysis of sulfur-related pathways for controls vs CRC. Hedges' g meta-analysis SMD and 95% confidence intervals are reported in the first plot. Significance level is indicated via a color gradient. P-values have been adjusted via Benjamini-Hochberg procedure for controlling FDR. Single cohort effect size levels are reported in the heatmap, ranging from dark blue for SMD = −1.5 to dark red for SMD = 1.5. Mean difference in abundance with species contribution for each pathway is reported in the stacked barplot on the right. B-C) Overview of histidine-related microbiome pathways and their associations with CRC, staging and primary tumor location. Hedges' g meta-analysis SMD and 95% confidence intervals are reported in the first plot. Significance level is indicated via a color gradient. P-values have been adjusted via Benjamini-Hochberg procedure for controlling FDR. Single cohort effect size levels are reported in the heatmap, ranging from dark blue for SMD = −1.5 to dark red for SMD = 1.5. Mean difference in abundance with species contribution for each pathway is reported in the stacked barplot on the right. C) Visual representation of the biochemical mechanisms that lead to L-glutamate production from L-histidine, with the production as a side product of ammonium. A) Differentially abundant microbial pathways (meta-analysis model P < 0.01) between early (stages 0-II) and late (stages III-IV) CRC. Associations with CRC Stage III-IV present Hedges' g SMD > 0, and associations with CRC Stages 0-II present Hedges' SMD < 0. Significant single-dataset comparison (P < 0.01) is reported in dark gray, while non-significant single-dataset associations (P ≥ 0.01) is in lighter gray. B) Differentially abundant microbial pathways (meta-analysis model q < 0.1) between non-metastatic (stages 0-III) and metastatic (stage IV) CRC. Significant single-dataset comparison (q < 0.1) is marked in dark gray, while non-significant single-dataset associations (q ≥ 0.1) are in lighter gray. Each line-plot represents the trends of an SGB in adenoma and the different stages of CRC in relation to controls' abundances. Non-significant coefficients (q ≥ 0.1) are reported with a violet circle, while significant coefficients (either positive or negative, q < 0.1) are reported as a brown diamond. Differentially abundant microbial pathways (meta-analysis model P < 0.01) between right-sided and left-sided CRC. Pathways associated with right-sided CRC present SMD < 0, and pathways associated with left-sided CRC present SMD > 0. Significant single-dataset comparisons (P < 0.01) are marked in dark gray, while non-significant single-dataset associations (P ≥ 0.01) are in lighter gray. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Piccinno, G., Thompson, K.N., Manghi, P. et al. Pooled analysis of 3,741 stool metagenomes from 18 cohorts for cross-stage and strain-level reproducible microbial biomarkers of colorectal cancer. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01690-z'>Exclusive: Inside the thriving wild-animal markets that could start the next pandemic</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 10:17:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Jane Qiu is an independent science writer in Beijing. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. In the bustling Jatinegara market in Jakarta, cages are stacked three metres high, holding creatures from all corners of Indonesia and beyond. Bats, raccoon dogs, macaques and songbirds — sold as pets or food — are crammed together, their musky odours mingling with the stench of urine and faeces in the damp tropical air. These markets remain “the best way of transmitting diseases”, says James Wood, a veterinary epidemiologist at the University of Cambridge, UK. Jatinegara's location, in an international travel hub with a population of 11 million people, increases that risk considerably. The world is still recovering from the COVID-19 pandemic, which many researchers say probably started, or was at least amplified, at a market selling live animals in Wuhan, China1,2. China banned the farming and trading of most wildlife species for food in 2020, but these practices have simply gone underground. “We are back to business as usual,” says Vincent Nijman, a conservation biologist at Oxford Brookes University, UK, with “millions and millions of animals being traded on a daily basis”. The wildlife trade acts as a vast global network of unregulated natural laboratories, through which potential pathogens freely circulate, evolve and ultimately congregate in urban centres, says Andrew Cunningham, a wildlife epidemiologist at the Institute of Zoology in London. “It's the scariest thing we are doing,” he says. These efforts were driven by the idea that it might be possible to predict which viruses could spark major disease outbreaks. But many researchers now say that this assumption was flawed. Increasingly, scientists are looking to human–wildlife interfaces — including markets and the trade networks that supply them — as crucial sites for the study of zoonoses, human diseases caused by pathogens that normally infect other species. A handful of research groups are now working to understand how pathogens jump between species, why some jumps cause outbreaks and others don't, and what kinds of intervention can reduce the risks. But such work is costly, can be dangerous and demands sustained support, which has become increasingly hard to secure. Without investing in such research, “you're really flying blind”, says Maria Van Kerkhove, head of the emerging diseases and zoonoses unit at the World Health Organization in Geneva, Switzerland. “You're making recommendations that may not be the most appropriate.” In the veterinary room of Cuc Phuong National Park in Vietnam, Tran Nam Trieu carefully lifts a Sunda pangolin (Manis javanica) onto an examination table. The creature is curled into a ball, its scales subtly rising and falling with each breath. Delicately unrolling it, Trieu, a veterinary surgeon, reveals its soft pink belly and the wound from an amputation; its left hind leg was damaged in a snare. Other coronaviruses found in these pangolins are relatives of those that cause Middle East respiratory syndrome (MERS). A live pangolin seized by the Malaysian wildlife department was probably on its way to a market in China.Credit: Bazuki Muhammad BM/JS via Reuters But that “should not prevent us from focusing on the bigger picture”, he says: the wildlife trade poses a much greater zoonotic risk than do lab accidents. 25 million deaths: what could happen if the US ends global health funding 25 million deaths: what could happen if the US ends global health funding To gauge those risks, Nguyen Thi Thanh Nga, a researcher at the Wildlife Conservation Society (WCS) in Hanoi, and her colleagues are working to identify potential pathogens circulating in trafficked pangolins in Vietnam — a major transit hub for moving wildlife into China — and exploring how these microorganisms relate to those found in source and destination countries. Of 246 pangolins confiscated across Vietnam between 2015 and 2018 — many from the rescue centre in Cuc Phuong — 7 were infected with coronaviruses, although none had signs of respiratory or other systemic illnesses5. The increasing detection rate of coronaviruses along the supply chain is consistent with another study by WCS researchers, on rats captured and sold for food in Vietnam7. Capturing and handling bats can be dangerous work, he says, and some hunters develop fevers when wounds from a scratch or bite become infected. Rather than seeking hospital care, they typically take herbs and over-the-counter medications, he says. Researching trade networks, he says, is sensitive work that requires significant trust-building. Fur farming a ‘viral highway' that could spark next pandemic, say scientists Fur farming a ‘viral highway' that could spark next pandemic, say scientists Ten years on, says Brown, people there are still negatively affected by the policy and remain suspicious of the authorities and international researchers. Some scientists have managed to cultivate the trust of wildlife traders in Indonesia. At the Langowan market in North Sulawesi, which sells both wild-animal meat and live animals, Tiltje Ransaleleh asks vendors about their supplies, the species they've sold and their origins. Beneath a canopy of red fabric that wards off the tropical Sun, Ransaleleh, a zoologist at Sam Ratulangi University in Manado, Indonesia, and her colleagues collect swab samples from bats lining the wooden stalls. Her team has mapped an intricate network of supply chains and identified the intermediaries — who purchase wild animals (including up to one million bats a year) from hunters and transport them to markets — as a potential vehicle for disease transmission8. One insight gleaned, she says, is that festive periods are the riskiest, when sales can surge to 5 times their usual volume, with more than 10,000 bats sold in a single day at Langowan. These in-depth studies of trade networks and human behaviour are essential for tracking the movement of wild animals and the potential pathogens they carry, says Stephen Luby, a disease ecologist at Stanford University in California. 25 million deaths: what could happen if the US ends global health funding Fur farming a ‘viral highway' that could spark next pandemic, say scientists What toilets can reveal about COVID, cancer and other health threats Trade wars could affect food security in low-income nations The path for AI in poor nations does not need to be paved with billions Black Death bacterium has become less lethal after genetic tweak The Last of Us science adviser: COVID changed our appetite for zombies Trump freezes ‘gain of function' pathogen research ― threatening all US virology, critics say 25 million deaths: what could happen if the US ends global health funding Fur farming a ‘viral highway' that could spark next pandemic, say scientists What toilets can reveal about COVID, cancer and other health threats An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01568-8'>Substrate recognition and allosteric regulation of synaptic vesicle glutamate transporter VGLUT2</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 09:52:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Structural & Molecular Biology (2025)Cite this article The concentration of neurotransmitters inside synaptic vesicles (SVs) underlies the quantal nature of synaptic transmission. Uptake of glutamate, the principal excitatory neurotransmitter, is driven by membrane potential. To prevent nonquantal efflux of glutamate after SV exocytosis, the vesicular glutamate transporters (VGLUTs) are allosterically inhibited by the neutral pH of the synaptic cleft. To elucidate the mechanism, we determined high-resolution structures of rat VGLUT2 with a cyclic analog of glutamate. We propose a mechanism of substrate recognition in which a positively charged cytoplasmic vestibule electrostatically attracts the negatively charged substrate. We also identify modification of VGLUT2 by palmitoylation and find that this promotes retrieval of the transporter after exocytosis. The structure also reveals an extensive network of electrostatic interactions that forms the cytoplasmic gate. Functional analysis of a mutant that disrupts the network shows how this cytoplasmic gate confers the allosteric requirement for lumenal H+ required to restrict VGLUT activity to SVs. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout The atomic coordinates of rVGLUT2 have been deposited in the Protein Data Bank (PDB) under accession codes PDB 7T3N for the R184Q/E191Q structure and PDB 7T3O for the WT structure. The corresponding maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes EMD-25665 and EMD-25666, respectively. The mass spectrometry data, including search results and peptide identification, are available at the ProteomeXchange Consortium through the PRIDE partner repository, with dataset identifier PXD061514. All data supporting the findings of this study are available from the corresponding authors on reasonable request. Source data are provided with this paper. Katz, B. Quantal mechanism of neural transmitter release. Naito, S. & Ueda, T. Characterization of glutamate uptake into synaptic vesicles. Maycox, P. R., Deckwerth, T., Hell, J. W. & Jahn, R. Glutamate uptake by brain synaptic vesicles. Energy dependence of transport and functional reconstitution in proteoliposomes. Bellocchio, E. E., Reimer, R. J., Fremeau, R. T. & Edwards, R. H. Uptake of glutamate into synaptic vesicles by an inorganic phosphate transporter. Takamori, S., Rhee, J. S., Rosenmund, C. & Jahn, R. Identification of a vesicular glutamate transporter that defines a glutamatergic phenotype in neurons. The expression of vesicular glutamate transporters defines two classes of excitatory synapse. Gras, C. et al. A third vesicular glutamate transporter expressed by cholinergic and serotoninergic neurons. Fremeau, R. T. et al. Vesicular glutamate transporters 1 and 2 target to functionally distinct synaptic release sites. Wojcik, S. M. et al. An essential role for vesicular glutamate transporter 1 (VGLUT1) in postnatal development and control of quantal size. Li, F., Eriksen, J., Finer-Moore, J., Stroud, R. M. & Edwards, R. H. Diversity of function and mechanism in a family of organic anion transporters. Drew, D. & Boudker, O. Ion and lipid orchestration of secondary active transport. Morin, P., Sagné, C. & Gasnier, B. Functional characterization of wild-type and mutant human sialin. Wreden, C. C., Wlizla, M. & Reimer, R. J. Varied mechanisms underlie the free sialic acid storage disorders. Structures suggest a mechanism for energy coupling by a family of organic anion transporters. & Ueda, T. Artificially imposed electrical potentials drive l-glutamate uptake into synaptic vesicles of bovine cerebral cortex. & Takahashi, T. A single packet of transmitter does not saturate postsynaptic glutamate receptors. Omote, H., Miyaji, T., Juge, N. & Moriyama, Y. Vesicular neurotransmitter transporter: bioenergetics and regulation of glutamate transport. Cavelier, P. & Attwell, D. Neurotransmitter depletion by bafilomycin is promoted by vesicle turnover. Takami, C., Eguchi, K., Hori, T. & Takahashi, T. Impact of vesicular glutamate leakage on synaptic transmission at the calyx of Held. Eriksen, J. et al. Protons regulate vesicular glutamate transporters through an allosteric mechanism. Tabb, J. S., Kish, P. E., Van Dyke, R. & Ueda, T. Glutamate transport into synaptic vesicles. Roles of membrane potential, pH gradient, and intravesicular pH. & Jahn, R. An anion binding site that regulates the glutamate transporter of synaptic vesicles. Wolosker, H., de Souza, D. O. & de Meis, L. Regulation of glutamate transport into synaptic vesicles by chloride and proton gradient. Schenck, S., Wojcik, S. M., Brose, N. & Takamori, S. A chloride conductance in VGLUT1 underlies maximal glutamate loading into synaptic vesicles. & Edwards, R. H. The dual role of chloride in synaptic vesicle glutamate transport. Hori, T. & Takahashi, T. Kinetics of synaptic vesicle refilling with neurotransmitter glutamate. Preobraschenski, J., Zander, J. F., Suzuki, T., Ahnert-Hilger, G. & Jahn, R. Vesicular glutamate transporters use flexible anion and cation binding sites for efficient accumulation of neurotransmitter. Martineau, M., Guzman, R. E., Fahlke, C. & Klingauf, J. VGLUT1 functions as a glutamate/proton exchanger with chloride channel activity in hippocampal glutamatergic synapses. Herring, B. E., Silm, K., Edwards, R. H. & Nicoll, R. A. Is aspartate an excitatory neurotransmitter? Naito, S. & Ueda, T. Adenosine triphosphate-dependent uptake of glutamate into protein I-associated synaptic vesicles. Kolen, B. et al. Vesicular glutamate transporters are H+-anion exchangers that operate at variable stoichiometry. Eriksen, J., Li, F., Stroud, R. M. & Edwards, R. H. Allosteric inhibition of a vesicular glutamate transporter by an isoform-specific antibody. Winter, H. C. & Ueda, T. The glutamate uptake system in presynaptic vesicles: further characterization of structural requirements for inhibitors and substrates. Li, F. et al. Ion transport and regulation in a synaptic vesicle glutamate transporter. Fukata, Y., Murakami, T., Yokoi, N. & Fukata, M. Local palmitoylation cycles and specialized membrane domain organization. & Chamberlain, L. H. Palmitoylation of the synaptic vesicle fusion machinery. Collins, M. O., Woodley, K. T. & Choudhary, J. S. Global, site-specific analysis of neuronal protein S-acylation. Miesenböck, G., De Angelis, D. A. & Rothman, J. E. Visualizing secretion and synaptic transmission with pH-sensitive green fluorescent proteins. Distinct endocytic pathways control the rate and extent of synaptic vesicle protein recycling. The conserved motif in hydrophilic loop 2/3 and loop 8/9 of the lactose permease of Escherichia coli. Analysis of suppressor mutations. Structure of the YajR transporter suggests a transport mechanism based on the conserved motif A. Proc. Bavnhoj, L., Paulsen, P. A., Flores-Canales, J. C., Schiott, B. & Pedersen, B. P. Molecular mechanism of sugar transport in plants unveiled by structures of glucose/H+ symporter STP10. Custodio, T. F., Paulsen, P. A., Frain, K. M. & Pedersen, B. P. Structural comparison of GLUT1 to GLUT3 reveal transport regulation mechanism in sugar porter family. Identification of a vesicular ATP release inhibitor for the treatment of neuropathic and inflammatory pain. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Scheres, S. H. RELION: implementation of a Bayesian approach to cryo-EM structure determination. Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Zivanov, J., Nakane, T. & Scheres, S. H. W. A Bayesian approach to beam-induced motion correction in cryo-EM single-particle analysis. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Guan, S., Price, J. C., Prusiner, S. B., Ghaemmaghami, S. & Burlingame, A. L. A data processing pipeline for mammalian proteome dynamics studies using stable isotope metabolic labeling. Clauser, K. R., Baker, P. & Burlingame, A. L. Role of accurate mass measurement (±10 ppm) in protein identification strategies employing MS or MS/MS and database searching. Silm, K. et al. Synaptic vesicle recycling pathway determines neurotransmitter content and release properties. Schneider, C. A., Rasband, W. S. & Eliceiri, K. W. NIH Image to ImageJ: 25 years of image analysis. Morin, A. et al. Collaboration gets the most out of software. Bond, C. S. & Schüttelkopf, A. W. ALINE: a WYSIWYG protein-sequence alignment editor for publication-quality alignments. We thank D. Bulkley, Z. Yu, G. Gilbert and M. Harrington at the UCSF cryo-EM facility and C. Hecksel, P. Mitchell, L.-M. Joubert and L. Dunn at Stanford-SLAC Cryo-EM Center (S2C2) for their support in data acquisition and computation. We thank D. Cawley at the Vaccine and Gene Therapy Institute of Oregon Health & Science University for generating the monoclonal antibody and for advice on working with the antibodies. This work was supported by R01NS089713 to R.M.S. was supported by postdoctoral fellowships from the American Heart Association (17POST33660928) and National Institute of Mental Health (K99MH119591) and Y.K.G. by the National Science Foundation Graduate Research Fellowship Program (2034836). The UCSF EM facility was supported by NIH grants S10OD020054 and S10OD021741. Some of this work was performed at the Stanford-SLAC Cryo-EM Center (S2C2), which is supported by the NIH Common Fund Transformative High-Resolution Cryo-Electron Microscopy program (U24 GM129541). The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. Structural biology applications used in this project were compiled and configured by SBGrid57. Mass spectrometry was provided by the Mass Spectrometry Resource at UCSF (A.L. Burlingame, Director), supported by the Dr. Miriam and Sheldon G. Adelson Medical Research Foundation (AMRF) and the UCSF Program for Breakthrough Biomedical Research. Present address: Department of Structural Biology, Amgen Research, South San Francisco, CA, USA Present address: Nine Square Therapeutics, South San Francisco, CA, USA These authors contributed equally: Fei Li, Jacob Eriksen. Department of Biochemistry and Biophysics, UCSF School of Medicine, San Francisco, CA, USA Fei Li, Janet Finer-Moore, Phuong Nguyen, Alisa Bowen, Andrew Nelson & Robert M. Stroud Departments of Neurology and Physiology, UCSF School of Medicine, San Francisco, CA, USA Fei Li, Jacob Eriksen, Hongfei Xu, Surabhi Hareendranath, Poulomi Das & Robert H. Edwards Department of Pharmaceutical Chemistry, UCSF School of Medicine, San Francisco, CA, USA Juan A. Oses-Prieto, Yessica K. Gomez, Alma Burlingame, Michael Grabe & Robert M. Stroud You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar purified the protein, prepared all samples for EM, acquired cryo-EM data at UCSF and the SLAC cryo-EM facilities and determined the structures. performed the functional experiments. J.A.O.-P. and A. Burlingame performed the mass spectrometry experiments. performed the docking and molecular dynamics simulations. wrote the paper with input from all authors. Correspondence to Fei Li, Robert M. Stroud or Robert H. Edwards. The authors declare no competing interests. Nature Structural & Molecular Biology thanks Albert Guskov and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Katarzyna Ciazynska and Melina Casadio, in collaboration with the Nature Structural & Molecular Biology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Sequence alignment of selected VGLUT homologs (line 1-5), other human SLC17 family members (line 6-11) and DgoT from E. coli demonstrates conservation across evolution and the SLC17 family of organic anion transporters. The sequences are listed in order of their sequence identity to rat VGLUT2, as in Fig. Residues are colored based on their sequence conservation, with red the most conserved and no color the least. This figure was generated with ALINE58. Untransfected (UN) HEK293T cells and HEK293T cells transfected with internalization-defective wild-type (WT) VGLUT2, R88K VGLUT2 and empty vector (EV) were loaded with 3H-aspartate (a) or 3H-glutamate (b) and the efflux monitored at pH 7.4 and 5.5. Neither WT nor R88K VGLUT2 mediate the flux of aspartate. c, Effect of mutations on the surface expression of VGLUT2. HEK cells were transfected with WT, mutant VGLUT2 or empty vector (EV), biotinylated and the total protein (input) or the biotinylated fraction (isolated with streptavidin) immunoblotted for VGLUT2. The quantitation of fluorescent western analysis from 3 biological replicates is shown below as % WT. The data indicate mean ± SEM. **, p<0.01; ****, p<0.0001; ns, not significant by one-way ANOVA. a, b, Size exclusion chromatography (SEC) and SDS-PAGE of WT (a) and R184Q/E191Q VGLUT2 (b) in complex with an Fab show high purity and homogeneity. c, Glutamate uptake by purified VGLUT2 reconstituted into liposomes with TF0F1 ATP synthase. The R184Q/E191Q mutant shows significantly less glutamate uptake than WT. The activity was measured as ATP-driven 3H-glutamate uptake for 15 min at 29 °C. The ionophores nigericin and valinomycin (100 nM each) were used to dissipate ΔpH and ΔΨ, respectively. d, The assay for efflux from transfected HEK cells shows that the R184Q/E191Q double mutant confers robust glutamate flux. Error bars indicate SEM. a, Details of the data processing strategy are described in the Methods section. Briefly, 3 rounds of 3D classifications were carried out in RELION to select high quality particles in the same conformation. The resulting particle stack was then refined and polished in RELION. To obtain the final map, polished particles were refined using RELION 3D auto-refine with a manually generated mask including only the VGLUT2 and Fab or using the nonuniformal refinement strategy in cryoSPARC with an automatically generated mask. Both maps resulted in similar resolution and quality. Final maps were then sharpened using Phenix Auto-sharpening and used for refinement and model building. Statistics of the final map were calculated using Phenix_comprehensive_validation tool51 and reported in Table 1. b, Sharpened coulombic density map of R184Q/E191Q was of sufficient quality to resolve the side chains of most residues. The coulombic potential density map is shown in gray mesh and model colored as in Fig. a and b, Docking of ACPD and glutamate. ACPD pose from the solved R184Q/E191Q-ACPD complex is shown in transparent green, interacting sidechains in blue (N domain) or pink (C domain) for R184Q/E191Q VGLUT2 (a), and green (N domain) or orange (C domain) for wild-type VGLUT2 (b). Best ACPD poses from flexible and rigid mode docking into R184Q/E191Q VGLUT2 are overlaid in opaque green and dark green, respectively. These poses are virtually identical, with all three functional groups in the same location as the solved pose (transparent). b, Best glutamate pose from flexible mode docking into WT VGLUT2, shown in opaque gray. The location within the binding pocket is similar to the solved pose of ACPD (transparent green); there is an average distance of 1.5 Å between the corresponding carboxyl groups of glutamate and ACPD near R88 and 2 Å between the corresponding carboxyl groups of glutamate and ACPD near R322. c, Position of aspartate at the beginning of an 800 ns simulation, determined by first placing functional groups in positions similar to the solved ACPD then choosing favorable rotamers. d-g, Tracking substrate recognition by MD simulation. d, Distance between the amine group of the ligand and the ring of F326 (dotted trace) or the sidechain oxygen atom of N450 (solid trace) is shown for the 800 ns R184Q/E191Q VGLUT2-ACPD simulation (green), the 1000 ns WT VGLUT2-glutamate simulation (gray), and the 800 ns WT VGLUT2-aspartate simulation (cyan). Gray bars indicate the time at which the snapshots on the left (e-g) were taken. ACPD adopts two stable poses during the simulation, with the amine pointed either towards the F326 ring (e, top) or away from it and towards N450, which also removed one interaction between the carboxyl and R88 (e, bottom). f, Glutamate adopts a single stable pose during the simulation. g, Aspartate is unable to interact with both arginines simultaneously and does not adopt a single stable pose. h, Root mean squared deviation (RMSD) of each ligand during its simulation. a, Details of the data processing strategy are described in the Methods section and similar to that used for VGLUT2-R184Q/E191Q (Extended Data Fig. b, Sharpened coulombic density map of WT VGLUT2 was of sufficient quality to resolve the side chains of most residues. The coulombic potential density map is shown in gray mesh and model colored as in Fig. Models of cytoplasmic-open WT (a) and R184Q/E191Q (b) VGLUT2 were produced by overlapping the N and C domains separately with the crystal structure of DgoT in the cytoplasm-open conformation. The sidechains of R88 and R322 in VGLUT2 overlap well with the equivalent residues in DgoT. c, Structure of the binding cavity in DgoT is colored by sequence conservation (Extended Data Fig. 3. d, Electrostatic surface of the inward/cytoplasm-open WT DgoT structure. e, Electrostatic surface of model of WT DgoT in outward-open conformation. The outward-open conformation of DgoT was generated by overlaying N and C domains separately with the structure of WT VGLUT2 in outward/lumen-open conformation. MS/MS spectra of tryptic peptides spanning amino acids K56 to R69 (a, b), and A57 to R69 (c) of R184Q/E1891Q VGLUT2, obtained by HCD fragmentation of precursor ions 626.0019+3, 626.0023+3 and 874.4525+2, respectively. Excess mass of 238.229 relative to the unmodified sequence, corresponding to a palmitoyl group, was observed in the precursor and in sequence ions including particular cysteine residues. a, Masses of b5 [Roepstorff-Fohlmann-Biemann nomenclature] and higher and y10 and higher indicate palmitoylation at C60 in peptide K56 to R69. b, Masses of b7 and higher and y8 and higher pinpoint palmitoylation at C62 in peptide K56 to R69. c, masses observed for y6 and higher indicate palmitoylation of C64 in peptide K57 to R69. Experimental masses of the most representative sequence ion peaks are labelled in the spectra, further indicating the fragment type. The position of fragmentation events generating these ions are indicated in the sequences over the spectra. A table indicating the theoretical masses of sequence ions according to the proposed modified sequences is shown on the right, with red indicating the observed fragments. Representative current traces from oocytes injected with WT, R213S, D371N pmVGLUT2-HA cRNA and uninjected controls. Recordings were performed in steps of 10mV from −120 to 60 mV. Holding current was −30 mV. All-atom MD simulation of WT VGLUT2 with l-Glu starting in a top-scoring docked pose (pose 1). All-atom MD simulation of WT VGLUT2 with l-Glu starting in the second-best docked pose (pose 2). All-atom MD simulation of WT VGLUT2 with l-Glu starting in a lower-scoring docked pose (pose 3). All-atom MD simulation of WT VGLUT2 with l-Glu starting in a lower-scoring docked pose (pose 3) (run #2). VGLUT2-pH (c) and GFP–VGLUT (d,e) imaging. Transport assays and I–V curves. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Li, F., Eriksen, J., Oses-Prieto, J.A. et al. Substrate recognition and allosteric regulation of synaptic vesicle glutamate transporter VGLUT2. Nat Struct Mol Biol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Structural & Molecular Biology (Nat Struct Mol Biol) Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250602155513.htm'>Geological time capsule highlights Great Barrier Reef's resilience</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 01:17:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research led by the University of Sydney adds to our understanding of how rapidly rising sea levels due to climate change foreshadow the end of the Great Barrier Reef as we know it. The findings suggest the reef can withstand rising sea levels in isolation but is vulnerable to associated environmental stressors arising from global climate change. It draws from a geological time capsule of fossil reef cores, extracted from the seabed under the Great Barrier Reef. Rather, associated environmental stressors like poor water quality and warming climates led, in combination, to its demise about 10,000 years ago (towards the end of the last ice age). The ensuing one to two thousand years saw Reef 4 transition. "This research shows us a healthy, active barrier reef can grow well in response to quite fast sea level rises," said Professor Webster. "The modern reef faces rising sea levels, more heat waves and extensive bleaching, along with increasing sediment and nutrient input. "It won't die but its characteristics may change. We will see a different collection of coral species, perhaps simpler and not as structurally complex." The 15 to 20-metre cores underpinning this research comprise a mix of fossil coral, algae and sediments. They reveal how the reef's previous incarnations responded to rapid sea level rise. Of particular interest to Professor Webster's team was the period known as Meltwater pulse 1B, between 11,450 and 11,100 years ago, when sea levels rose very rapidly. "This 350-year period is crucial; it covers a time when global sea levels rose very rapidly," Professor Webster said. Based on records from Barbados, we previously thought sea levels were rising by about 40 millimetres a year at this time. "Our research shows the rise wasn't so large and fast. Extracted by a drilling ship from beneath the Great Barrier Reef's shelf edge at a depth of 40 to 50 metres, the cores offered new insight into how Reef 4, also known as the proto-Great Barrier Reef, was impacted by rising sea levels. "Understanding the environmental changes that influenced it, and led to its ultimate demise, therefore offers clues on what might happen to the modern reef." Professor Webster and colleagues used radiometric dating and reef habitat information to accurately pinpoint core samples pertaining to Meltwater pulse 1B. They provide paleoclimate and paleoenvironmental data, going far further back in time than instrumental records which go back only 50 to 100 years. "These data allow us to more precisely understand how reef and coastal ecosystems have responded to rapid environmental changes, like the rises in sea level and temperature we face today." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250602155505.htm'>The sweet spot: sugar-based sensors to revolutionize snake venom detection</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-03 01:08:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Most approaches to diagnose and treat snake venom, a WHO neglected tropical disease, rely on antibodies. In this new research paper, published in ACS Biomacromolecules, Warwick researchers have shown the first proof of concept for a cheap and rapid alternative -- a glycopolymer-based ultraviolet-visible (UV-vis) test to detect snake venom. They showcase a version of this diagnostic assay in the paper that specifically detects Western Diamondback Rattlesnake (Crotalus atrox) venom. Dr. Alex Baker, Assistant Professor at Warwick, lead researcher of the Baker Humanitarian Chemistry Group and senior author of the paper said: "Snake venoms are complex and detecting the toxins at work is challenging but essential to save lives. Western Diamondback Rattlesnake venom has evolved to bind to specific sugar molecules on the surface of cells in the body, such as red blood cells and platelets. Specifically, the toxin binds to galactose-terminal glycans (sugar chains ending in galactose). Mahdi Hezwani, first author and alumni of Dr Baker's research group said: "This assay could be a real game-changer for snake envenomation. Venoms from other snake species do not interact with glycans in the body. For example, when we tested venom from the Indian Cobra (Naja naja) we did not see binding to the synthetic glycans that bind to C.atrox venom. This is the first example of a diagnosis test using sugars for detecting snake venom in a rapid detection system, and builds on the work of this Warwick research group using a glyconanoparticle platform in COVID-19 detection. This new assay is faster, cheaper, easier to store, is modifiable since the sugars can be custom-made to recognise a specific toxin and is an example of the bold, innovative solutions that will continue to be made possible through The University of Warwick's new STEM Connect programme. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            