
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-12</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/measles-death-shows-how-virus-can-hide-in-the-brain-for-years/'>Child's Death Shows How Measles in the Brain Can Kill Years after an Infection</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 16:50:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Child's Death Shows How Measles in the Brain Can Kill Years after an Infection An MRI scan showing subacute sclerosing panencephalitis, a complication of measles infection. A school-aged child in Los Angeles County has died from a rare but always fatal complication from a measles infection they acquired when they were an infant who was too young to be vaccinated. “This case is a painful reminder of how dangerous measles can be, especially for our most vulnerable community members,” said Los Angeles County Health Officer Muntu Davis in a recent statement. The child who died suffered from subacute sclerosing panencephalitis (SSPE), a progressive brain disorder that usually develops two to 10 years after a measles infection. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “It's just a virus that goes unchecked and destroys brain tissue, and we have no therapy for it,” said Walter Orenstein, an epidemiologist and professor emeritus at Emory University, to Scientific American earlier this year. People with SSPE experience a gradual, worsening loss of neurological function and usually die within one to three years after diagnosis, according to the Los Angeles County Health Department. The disorder affects only about one in every 10,000 people who contract measles. Children who suffer from this will always die,” said Paul Offit, director of the Vaccine Education Center and an attending physician in the Division of Infectious Diseases at Children's Hospital of Philadelphia, in a previous interview with Scientific American. Measles can also cause encephalitis, a faster-occurring brain inflammation, in one in every 1,000 people who are infected because the virus causes the immune system to attack a protein produced by certain brain cells. This inflammation kills about one in five people who develop it. There is some evidence this effect can last for a couple of years, making those who get measles more susceptible to other infectious diseases. These side effects are of particular concern because the measles virus is highly contagious—an order of magnitude more than seasonal influenza. With measles, viral particles emitted by coughing or sneezing can linger in a room for hours after the infected person has left. Most of those infected were unvaccinated or had an unknown vaccination status. Measles used to infect three million to four million people in the nation every year until vaccines became available in 1963. Contrary to claims by Secretary of Health and Human Services Robert F. Kennedy, Jr., measles cannot be treated by vitamin A or cod liver oil. The only effective means of combatting measles is widespread vaccination. At least 95 percent of a population must be vaccinated to prevent the spread of the disease and to protect either those who are too young to receive it or those who cannot be vaccinated because of other health conditions. “Infants too young to be vaccinated rely on all of us to help protect them through community immunity,” Davis said in his recent statement. “Vaccination is not just about protecting yourself—it's about protecting your family, your neighbors, and especially children who are too young to be vaccinated.” Andrea Thompson is a senior editor covering the environment, energy and earth sciences. She has been covering these issues for 16 years. Prior to joining Scientific American, she was a senior writer covering climate science at Climate Central and a reporter and editor at Live Science, where she primarily covered earth science and the environment. She has moderated panels, including as part of the United Nations Sustainable Development Media Zone, and appeared in radio and television interviews on major networks. in atmospheric chemistry from the Georgia Institute of Technology. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02981-1'>Meet Europe's first exascale supercomputer — can it compete in the global AI race?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 14:32:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. German Chancellor Friedrich Merz inaugurated the new European supercomputer called JUPITER on 5 September.Credit: Ina Fassbender/AFP via Getty Having surpassed one quintillion (1018) operations a second, it joins an exclusive league of exascale supercomputers. It gives researchers in Europe access to their own top-level supercomputer, rather than having to rely on machines in the United States and elsewhere. The milestone is “absolutely” a big deal for Europe, says Kirk Cameron, a computer scientist at Virginia Polytechnic Institute and State University (Virginia Tech) in Blacksburg, particularly with regards to AI and large language models (LLMs). “It's taken a little bit to get [Europe] into the race. So it's really nice to see them making that progress.” Running on some 24,000 NVIDIA chips, JUPITER is capable of exceeding 1,000 petaflops — one exaflop, or one million trillion operations per second — at its peak performance. Lippert says that having a supercomputer such as JUPITER will enable Europe to develop the talent necessary to build and operate such machines in the future. “Our economy and welfare depend on these technologies,” he says. Lippert says that this is achieved by paying to use only renewable energy from Germany's national grid. Cameron says such supercomputers can be extremely power-hungry and cause problems. Nuclear power for AI: what it will take to reopen Three Mile Island safely DeepMind AI crushes tough maths problems on par with top human solvers Synthetic data can benefit medical research — but risks must be recognized AI chatbots are already biasing research — we must establish guidelines for their use now We are currently seeking highly qualified candidates for multiple faculty positions across all academic ranks within the field of biomedical sciences. Henry Ford + MSU is seeking up to 10 exceptional CANCER researchers with a record of innovation and high-impact research. Nuclear power for AI: what it will take to reopen Three Mile Island safely DeepMind AI crushes tough maths problems on par with top human solvers An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/deep-space/a66042152/high-res-black-hole-merger/'>Scientists Just Proved Einstein and Hawking Right About Black Holes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ten years after the first gravitational wave detection, a new suite of instruments has captured distinctive ringing signals before and after a black hole merger. After all, how do you analyze an object with such massive gravity that not even light can escape? But to astrophysicists studying black holes, the ringing frequencies of these mergers—both leading up to and following the collision—can provide an immense amount of information. Due to technological limitations at the time, records from the original 2015 merger didn't contain enough data to confirm two prominent black hole theories: The idea that black holes could be described by just mass and spin, and Stephen Hawking's area theorem—that a black hole's event horizon can only grow over time. After a decade of improving detection technologies and methods, scientists finally have “an exquisitely detailed view of the signal both before and after the black hole merger,” Maximiliano Isi, an astrophysicist at the Flatiron Institute's Center for Computational Astrophysics, said in a press statement. Isi, along with dozens of other members of the LIGO-Virgo-KAGRA Collaborations, published a study in the journal Physical Review Letters detailing this new high-fidelity “ringing” data on black hole mergers. This study builds upon Isi's previous work, which tried to isolate these deep-space tones using the 2015 data. “We've found some of the strongest evidence yet that astrophysical black holes are the black holes predicted from Albert Einstein's theory of general relativity.” Confirmation of Hawking's area theorem draws parallels to the Second Law of Thermodynamics, which states that entropy must increase or remain constant over time. The authors suggest that by probing the thermodynamics of black holes, we could learn more about about even bigger cosmic mysteries—most notably, quantum gravity. “It's really profound that the size of a black hole's event horizon behaves like entropy,” Isi said in a press statement. “It has very deep theoretical implications and means that some aspects of black holes can be used to mathematically probe the true nature of space and time.” In the next decade, gravitational wave detectors will be 10 times more sensitive than they are today, and plans are already in motion to build LIGO's successor, known as the Laser Interferometer Space Antenna (LISA), which will investigate gravitational waves at even lower frequencies (including those originating from supermassive black holes). A Scientist Says This Object May Be an Alien Craft</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/forensic-expert-explains-how-3d-laser-scanning-could-reconstruct-the-charlie/'>Forensic Expert Explains How 3D Laser Scanning Could Reconstruct the Charlie Kirk Shooting</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 12:59:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>How 3D Laser Scanning Could Reconstruct the Charlie Kirk Shooting Forensic scientist Michael Haag explains how laser scanners could be used to lock down the crime scenes where Charlie Kirk was fatally shot, letting investigators revisit angles, trajectories and vantage points long after the fact. A crowd watches Charlie Kirk, founder of Turning Point USA, speaking during his "American Comeback Tour" at Utah Valley University when he was shot in the neck and killed on September 10, 2025. Charlie Kirk, 31-year-old founder of the conservative nonprofit organization Turning Point USA, was shot on Wednesday while speaking outdoors at Utah Valley University. Officials say the single shot may have been fired from a nearby rooftop. On Friday morning, with a massive manhunt still underway, President Donald Trump has announced that a suspect is in custody. Chairs, tables, cars—whatever was in the space—may be moved, or people may lose track of them. One of the best ways to immediately preserve a crime scene is with three-dimensional laser scans, which use light to map every object present. Today this technique is routinely used at major crime scenes. And experts say it's likely to be important in the investigation of Kirk's death. After majoring in chemistry and minoring in mathematics and physics at the University of Arizona, Haag spent nearly 25 years at New Mexico's Albuquerque Police Department, where he worked as a forensic scientist on a team that handled major crime scenes ranging from homicides to officer-involved shootings. He also now owns a private company that conducts many of the same services and provides training in shooting incident reconstruction worldwide. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. For more than two decades, Haag has worked on complex cases, taught investigators across the U.S. and testified in court on how 3D laser scans of crime scenes can be used. In a conversation with Scientific American, he explained how this technique would likely be used on the scene linked to Kirk's death. In plain language, he walked us through how a laser scanner captures millions of measurements to map rooftops and sightlines and how trajectory analysis checks whether impacts line up with a single shooter location. Can you walk me through every step of what 3D laser scanning of a crime scene looks like and how you use that information afterward? The first thing to keep in mind is that, with 3D laser scanning, the massive benefit is that you're documenting the spatial relationships of everything within that scene very accurately in three dimensions. You can look at measurements, angles and distances—anything that has to do with physical measurement and relationships, you can redo. It's an engineering tool that we've been using in crime scenes since about 2004. These scanners were initially designed to make very accurate measurements of construction sites, factories and bridges. It's like a laser range finder but much more capable. A 3D laser scanner puts that same range-finding device on a stable tripod platform that rotates 360 degrees horizontally and also moves vertically. With today's scanners, like the Leica RTC that I use, it's literally shooting out millions of these bullets of light in a very short time frame. From any one position, you build this three-dimensional world around you that is very accurate—we're talking data points with about three millimeters of accuracy within 20 meters of the scanner. You then move the scanner from position to position, each time building a three-dimensional sphere of data points, and then you blend all these spheres together. For example, I've done a lot of work on the Kennedy assassination and have scanned Dealey Plaza [the Dallas, Tex., site of President John F. Kennedy's assassination on November 22, 1963]. You can very quickly put yourself in any position you want in that scene and look at what trajectories are or are not possible. We'll see whether the bullet ... went on to hit something else downrange. Do they put on a virtual reality headset? It's literally like a computer game where you can get up in the sky and fly down through the scene. You can put yourself above buildings looking directly down. You can put yourself behind windows in buildings looking out into the courtyard. You could put yourself in Mr. Kirk's position and look out to see what he would have seen. In this situation, where people have video footage of a shooter and they know the basics, what would this 3D scan add? If it turns out that some of the video we've seen of a human-shaped form on top of the building is a viable shooting position, it may most commonly be used to put you in the position of that shooter so you can see what that vantage point would look like—and vice versa, from Mr. Kirk's position looking uprange. Is there a way to determine exactly what the trajectory was? If we look at the gunshot wound to Mr. Kirk, and we have video that shows his orientation and general body positioning, it looks like a neck shot to me. If we look at that path, we can start to look at where in that scene the bullet would have come from. In the example I just went through—front to back, left to right and downward—that would include somebody shooting him from his left as he's sitting in the scene, off in the distance, on top of a building over there. Again, it's not down to the degree that you might expect from TV, where crime scene investigators put a laser in and say the gun was right along this laser beam. There's usually about plus or minus five degrees of accuracy, even on a good trajectory with gunshot wounds to bodies. It's a little bit more conservative than that, but you can get a general idea of where it would have come from. Deni Ellis Béchard is Scientific American's senior tech reporter. His most recent novel, We Are Dreams in the Eternal Machine, explores the ways that artificial intelligence could transform humanity. You can follow him on X, Instagram and Bluesky @denibechard. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250911073204.htm'>Toxic “forever chemicals” found in 95% of beers tested in the U.S.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 11:02:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Infamous for their environmental persistence and potential links to health conditions, per- and polyfluoroalkyl substances (PFAS), often called forever chemicals, are being discovered in unexpected places, including beer. "As an occasional beer drinker myself, I wondered whether PFAS in water supplies was making its way into our pints," says research lead Jennifer Hoponick Redmon. "I hope these findings inspire water treatment strategies and policies that help reduce the likelihood of PFAS in future pours." PFAS are human-made chemicals produced for their water-, oil- and stain-repellent properties. Although breweries typically have water filtration and treatment systems, they are not designed to remove PFAS. By modifying a U.S. Environmental Protection Agency (EPA) testing method for analyzing levels of PFAS in drinking water, Hoponick Redmon and colleagues tested 23 beers. The test subjects were produced by U.S. brewers in areas with documented water system contamination, plus popular domestic and international beers from larger companies with unknown water sources. These include perfluorooctanesulfonate (PFOS) and perfluorooctanoic acid (PFOA), two forever chemicals with recently established EPA limits in drinking water. This work shows that PFAS contamination at one source can spread into other products, and the researchers call for greater awareness among brewers, consumers and regulators to limit overall PFAS exposure. 90% Chance: Physicists Predict a Black Hole Could Explode This Decade This Simple Blood Test Might Catch Alzheimer's Before It Steals Your Memory Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02810-w'>Bioactive molecules unearthed by terabase-scale long-read sequencing of a soil metagenome</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 09:27:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Metagenomics provides access to the genetic diversity of uncultured bacteria through analysis of DNA extracted from whole microbial communities. Long-read sequencing is advancing metagenomic discovery by generating larger DNA assemblies than previously possible. However, harnessing the potential of long-read sequencing to access the vast diversity within soil microbiomes is hampered by the challenge of isolating high-quality DNA. Here we introduce a method that can liberate large, high-quality metagenomic DNA fragments from soil bacteria and pair them with optimized nanopore long-read sequencing to generate megabase-sized assemblies. Using this method, we uncover hundreds of complete circular metagenomic genomes from a single soil sample. Through a combination of bioinformatic prediction and chemical synthesis, we convert nonribosomal peptide biosynthetic gene clusters directly into bioactive molecules, identifying antibiotics with rare modes of action and activity against multidrug-resistant pathogens. Our approach advances metagenomic access to the vast genetic diversity of the uncultured bacterial majority and provides a means to convert it to bioactive molecules. As adoption of this technology rose and costs were driven down, there was an explosion of bacterial isolate whole-genome sequences1. The majority of bacteria, however, remain recalcitrant to laboratory culture, preventing direct analysis of their genetic diversity. While short-read-based 16S amplicon profiling of metagenomes (that is, whole-DNA extracts of microbial communities) has been used to provide glimpses at microbial diversity hidden in the uncultured majority2, a key limitation of direct short-read exploration is the inability to deconvolute complex metagenomes and reconstruct sequences of individual microorganisms within the community. Binning of short-read assemblies by computationally predicted similarity into metagenome-assembled genomes (MAGs) as a means to explore the genetic potential of environmental bacteria has now become routine; however, MAGs tend to be smaller than bacterial genomes, fragmented and often contain sequence contamination through erroneous binning1,3,4,5. Exploration of soil microbiomes presents a particularly difficult challenge because of not only the immense microbial diversity they contain but also the challenges associated with isolating high-quality metagenomic DNA6,7. Efficient exploration of environmental microbial genetic potential requires the development of methods that can generate large contiguous assemblies. This is especially important when exploring the biosynthetic potential of the global metagenome, as biosynthetic gene clusters (BGCs) can be tens if not hundreds of kilobases long and are challenging to identify within fragmented assemblies8. Long-read sequencing technologies have advanced metagenomic sequencing by producing larger assemblies that can be efficiently interrogated using bioinformatic algorithms (Fig. 1a)9,10,11,12,13 and updated nanopore flow cells (R10.4) and sequencing chemistries (V14) can now generate high-quality assemblies without the need for short-read polishing14,15. a, The limitations of short-read technology restrict complete genomic sequencing to cultured isolates; emerging long-read sequencing may overcome this challenge. b, Nanopore sequencing read-length distribution comparison of soil metagenomic DNA prepared by direct hot CTAB extraction (old) and the newly developed and optimized nycodenz method (new). c, Assembly summary of high-quality nanopore soil metagenomic sequencing at metaFlye's input limit. d, Assembly length plot of >1 Mbp metaFlye generated contigs with an estimation of genome completeness based on the presence of all rRNA genes (5S, 16S and 23S) and at least 18 unique tRNAs. Plots are separated into circular and linear assemblies. Numerous methods have been developed to extract metagenomic DNA16 but the focus has been on DNA quality without the need to optimize these protocols for isolating large DNA fragments. Here, we developed a nanopore long-read sequencing pipeline with size-optimized DNA isolation and preparation that, when applied to a complex soil metagenome, yielded read lengths with N50 > 30 kbp. This is 200 times longer than the commonly used 150-bp short-read technology. The increased read size led to assemblies of contiguous DNA sequences on the order of megabase pairs, including hundreds of complete circular metagenomic genomes. Using a synthetic bioinformatic natural product (synBNP) approach, which couples bioinformatics and total chemical synthesis to abiotically decode BGCs17, we directly converted unearthed nonribosomal peptide BGCs within the large contiguous assemblies into bioactive molecules. This led to the discovery of a potent cardiolipin-binding broad-spectrum antibiotic and a ClpX-targeting antibiotic. A critical barrier to sequencing soils is the copurification of contaminants when isolating metagenomic DNA6,7. To improve soil DNA extraction, we developed a method that first separated bacteria from the soil matrix by nycodenz gradient centrifugation and then used a skim-milk wash to remove impurities from the isolated cells. This process generated a bacterial suspension that appeared similar to a lab-grown culture (Supplementary Fig. Using standard hot CTAB metagenomic DNA extraction18 on this cell suspension yielded a cleaner metagenomic DNA extraction with far larger fragments compared to direct soil extraction (Supplementary Fig. The ‘clean' microbial suspension allowed us to test a variety of metagenomic DNA extraction and size-selection approaches to maximize nanopore read length (Supplementary Note 1, Supplementary Fig. Our final method used commercially available DNA extraction (Monarch's high-molecular-weight (HMW) DNA extraction kit) and size selection (Oxford Nanopore's small fragment eliminator kit), allowing for simple and scalable DNA preparation. We expanded our approach using the original test forest soil sample to generate almost 1 Tbp of sequence data with an N50 of 32.82 kbp. For comparison, a representative nanopore sequencing result using our method and the direct soil hot CTAB extraction method is shown in Fig. In total, 2.5 Tbp of long-read sequencing data were generated from a single forest soil sample during testing and scaling. All data were pooled for bioinformatic analysis. A summary of all the sequencing runs is given in Supplementary Table 2. MetaFlye19, the state-of-the-art algorithm for metagenomic assembly, has an input limit of 239 bp (550 Gbp) and could not process all the data simultaneously because of the high sequencing depth. We, therefore, used two independent approaches to generate assemblies. This included global assembly at metaFlye's input limit using the highest-quality and longest reads and a targeted approach that used extraction and subassembly of reads containing sequences of interest. To gain broad insight into the metagenomic content of the forest soil, we assembled a 528.5-Gbp subset of our data consisting of the highest-quality and longest reads (Q20+ reads longer than 20,000 bp). In addition, a recent PacBio long-read exploration of a soil metagenome generated read lengths with a N50 of 4 kbp, leading to assemblies with an N50 of 36 kbp (ref. Our optimized nanopore exploration substantially increased assembly length and provided a unique opportunity to explore genome-sized contiguous metagenomic assemblies without the reliance on computationally inferred MAG binning. We focused our exploration on a subset of single contiguous assemblies that likely captured complete or near-complete bacterial genomes. To do this, we determined the subset of >1 Mbp of circular and linear assemblies that contained 5S, 16S and 23S ribosomal RNA (rRNA) genes with at least 18 transfer RNAs (tRNAs) (Fig. These combined criteria are part of the field benchmark for near-complete MAGs1 and yielded 563 single contiguous assemblies (Fig. These contained 95% (206) of the circular assemblies in the >1-Mbp dataset, which was in line with the expectation that large circular assemblies represent complete contiguous bacterial genomes (Fig. To put this advance into context, we collated MAGs from soil-linked samples within the Genome Taxonomy Database (GTDB) (Supplementary Table 4)23, the most up-to-date repository of bacterial sequences. From 5,640 entries across 91 bioprojects, we only identified two single-contig MAGs exceeding 1 Mbp that contained the required repertoire of rRNA and tRNA genes. In fact, only 26 total MAG bins contained an assembly over 1 Mbp, two orders of magnitude lower than the >3,200 in our single assembly. In addition to the rRNA and tRNA criteria, for a MAG to be considered near complete, it must be rated >90% complete with <5% contamination by CheckM, a software tool designed to assess the quality of genome bins. Irrespective of the rRNA and tRNA content, a MAG is considered of high quality at >70% completeness and medium quality at >50% completeness, both with <10% contamination1,24. Of the 206 circular assemblies, only three were below 80% completeness, with 79% (163) matching the near-complete criteria (Supplementary Table 3). Only 15 (2.7%) assemblies within the 563 complete or near-complete dataset exceeded the contamination criteria, with many only by a small margin (Supplementary Table 3). These high-quality MAGs were on average 3.05 ± 1.25 Mbp in size and composed of 567 ± 417 assemblies with an N50 of 11.8 ± 9.8 kbp. Overall, this analysis continued to demonstrate the advance in metagenomic exploration provided by long-read sequencing, with our optimized protocol greatly increasing the number of complete or near-complete contiguous genomes that can now be assembled from a complex soil metagenome. To gain an estimate of the species diversity found in each subset of our data, we used 16S profiling. In total, we detected >4,500 unique species within the 2.5-Tbp dataset. Over 90% of these were detected in the global assembly data, 25% were detected in the >1-Mbp dataset and over 12% were detected in the complete or near-complete genome dataset (Supplementary Table 5). Our complete and near-complete metagenome dataset resolved into 16 phyla (Fig. 2a), with only four sequences (0.7%) matched to known species (Supplementary Table 3). To highlight underexplored bacterial sequences within the dataset, we determined which underrepresented and largely uncultured taxa within the GTDB were most expanded by our assembled data (Supplementary Table 6). In addition, we also saw a notable expansion of diversity of the Palsa-744 genus within the well-represented Actinomycetota phylum (Supplementary Fig. Apart from a single completely sequenced cultured representative in the Capsulimonadaceae family, our data added the first complete contiguous genomes for each of the abovementioned taxa. Of these, the most striking was a 12.56-Mbp circular assembly belonging to the UBP17 phylum, which stood out because this mysterious phylum consisted of only ten total MAGs that were on average much smaller and highly fragmented. The theme of highly fragmented MAGs being the only genetic resource for a taxon was common throughout our comparisons (Fig. Assembly of large contiguous sequences provided not only an unprecedented resolution of a complex microbial community but also an opportunity for the bioinformatic exploration of biosynthetic potential within microbial dark matter. a, Phylogeny of high-quality contiguous assemblies from Fig. Generating complete BGC sequences within fragmented assemblies generated by short-read technology is challenging. Furthermore, the mobile nature and rapid evolution of BGCs25 may hamper MAG binning where computational similarity would need to link core genomic regions and varied BGC content. We rationalized that our long-read sequencing data would allow us to explore biosynthetic systems found in complex microbiomes without these limitations. To gain a broad overview of the biosynthetic content captured within our complete and near-complete metagenomic genome dataset, we bioinformatically identified and summarized BGC content at the taxonomic family level (Fig. On average, there were 1.3 ± 0.7 BGCs per Mbp across the families, with four families (UBA5704, Pseudonocardiaceae, Streptomycetaceae and UBA11063) exceeding 3 BGCs per Mbp. UBA stands for ‘uncultured bacteria or archaea' and was coined to phylogenetically characterize MAGs that provided the first example of a bacterial or archaeal lineage26. UBA5704 contains five genera and UBA11063 only contains a single genus (Aquella), for which a cultured representative was isolated and sequenced in 2019 (ref. Even with their relatively small genome sizes, our data indicated that Aquella species within our dataset contained a disproportionally large biosynthetic repertoire. Notably, the clusters were overwhelmingly nonribosomal peptide synthase (NRPS)-type BGCs (Fig. Even though the GTDB contained 37 Aquella entries (one isolate and 36 MAGs), analysis of the total 76.4-Mbp sequence yielded only 85 total BGCs (1.1 BGC per Mbp). This may indicate that, because of difficulties in assembling NRPS BGCs using short reads, this genus may have been previously overlooked as a rich source of natural products. Overall, our large contiguous assemblies offered an avenue for natural product discovery by pointing toward BGC-rich taxa that have remained hidden or overlooked because of a lack of quality assemblies. To access this previously hidden biosynthetic diversity, we used a total synthesis approach of bioinformatically predicted nonribosomal peptides. NRPS BGCs encode large modular proteins where each module functions to incorporate a monomer building block into a growing peptide-like structure28. The synBNP approach exploits this modular organization to predict the NRPS product by decoding which monomer is selected by each NRPS module to predict the assembled peptide; then, the predicted structure is accessed by total chemical synthesis. A small number of bacterial species dominate soil biomass29 and obscure the rare microbiome that contains the majority of genetic diversity within a metagenome. To explore the complete set of genetic diversity captured in our whole 2.5-Tbp dataset, we devised a partitioned assembly approach where a hidden Markov model (HMM) search was used to identify reads containing specific sequences of interest for subassembly. While we targeted NRPS adenylation (A) domains, this approach could be used to target any sequence of interest. The HMM search identified 6.17 million reads containing an A-domain, which assembled into 118,000 contigs with an N50 of 111 kbp (Fig. We then narrowed our search to contigs that contained four or more A domains to focus on large NRPS BGCs, which are generally problematic for short-read assemblies to fully capture. BiG-SCAPE analysis of the captured NRPS BGCs on these contigs revealed 366 clusters, of which only six could be found in the MiBig database of characterized molecules (Fig. Notably, only 20.5% of the clusters from our targeted assembly could be mapped to the global assembly dataset, indicating that our partitioned assembly approach did indeed access a larger proportion of the rare microbiome. a, Left, general outline of targeted metagenomic NRPS assembly through HMM identification of A-domain-containing reads. Right, BiG-SCAPE analysis of 328 assemblies containing ≥4 A domains. Colored circles are metagenomic BGCs, with white circles denoting clusters matched from the MiBig database. d, Antibiotic activity screening of 55 synBNPs synthesized based on 20 metagenomic NRPS BGCs against B. subtilis (Bs), S. aureus (Sa) and E. coli (Ec). Lines separate predictions generated by single BGCs. e, Expanded activity screening of synBNPs with MICs ≤8 µg ml−1 from b against the ESKAPE pathogens and activity against HEK and HeLa human cell lines. Results in d,e are representative of two and three biologically independent replicates, respectively. A possible limitation of the targeted assembly approach was that removing genomic context by restricting assembly to reads only containing NRPS genes may have led to erroneous or chimeric assemblies. While it is not possible to know the extent to which this problem may exist, we believe that it is unlikely to be pervasive because of two key points. First, previously characterized NRPS BGCs identified in the BiG-SCAPE analysis were correctly assembled. Second, as a control, we cultured and sequenced a collection of microorganisms from the same forest soil sample used in this study, which yielded multiple NRPS BGCs that matched those found in our targeted assembly dataset (Supplementary Table 7, Supplementary Fig. We selected 20 NRPS BGCs with diverse predicted phylogenic origins for synBNP structure prediction and synthesis (Fig. While the peptide sequence generated by an NRPS BGC can be predicted through the A-domain signatures present within each module, the final product can be released as either a linear or cyclic peptide, which remains difficult to predict bioinformatically. 3c; linear (L), cyclized through the terminal amine (H), cyclized through the 3-OH fatty acid (F) and cyclized through a nucleophilic amino acid side chain (C#)) to generate a library of 55 synBNPs (Supplementary Table 11). Myristic or hydroxymyristic acid was used when lipidation was predicted to be present as it is the most commonly seen lipid in characterized lipopeptides. All synBNPs were assayed for antibiotic activity against Gram-positive model organisms Bacillus subtilis and Staphylococcus aureus, as well as the Gram-negative model organism Escherichia coli (Fig. Five synBNPs, based on predictions from four NRPS clusters, had notable activity (minimum inhibitory concentration (MIC) ≤ 8 µg ml−1) against at least one of these bacteria and were, therefore, screened for wider activity against the ESKAPE pathogens (Enterococcus faecium, S. aureus, Klebsiella pneumoniae, Acetobacter baumannii, Pseudomonas aeruginosa and Enterobacter cloacae) and for human cell toxicity (Fig. We then selected the two most potent synBNPs for further characterization. SynBNP 25, inspired by the BGC found on contig 112034, was a potent broad-spectrum antibiotic and lacked cytotoxicity. This compound was named erutacidin after the Latin erutus for ‘dug out'. SynBNP 27, inspired by the BGC found on contig 97355, was selected for its potent activity against S. aureus (MIC = 2 µg ml−1). This compound was named trigintamicin after the Latin word for ‘30', the temperature at which activity was detected. In our initial screen, racemic 3-OH myristic acid was used for the synthesis of fatty acid cyclized synBNPs that included trigintamicin. Upon resynthesis and more thorough purification, the two trigintamicin isomers were resolved, with only the S-isomer showing antibiotic activity against S. aureus (Supplementary Table 12). To determine the MOAs of erutacidin and trigintamicin, we first attempted to raise resistant mutants. Direct plating of S. aureus on trigintamicin (Fig. Genomic sequencing of resistant clones revealed mutations at two genetic loci, ytrA and clpX (Supplementary Table 13). Mutations in ytrA have been previously linked to nonspecific antibiotic resistance30,31; hence, we focused on clpX. ClpX is an ATPase that unfolds and translocates proteins into the ClpP serine protease for degradation32. Our antibiotic activity assays were conducted at 30 °C and we found that shifting the assay conditions to 37 °C abolished trigintamicin activity, further suggesting ClpX as the target (Supplementary Table 12). We then performed isothermal titration calorimetry (ITC) to confirm the interaction between trigintamicin and ClpX. These studies found that trigintamicin bound ClpX, whereas its biologically inactive R-isomer did not (Fig. No thermal changes were observed in the absence of ClpX (Supplementary Fig. Collectively, our MOA studies indicated that trigintamicin was a ClpX-targeting antibiotic. The targeting of ClpX by trigintamicin explains the narrow antibacterial spectrum, as clpX is not essential in most bacteria. It also explains human cell cytotoxicity, as mitochondrial ClpXP is essential and an emerging target for cancer therapy34. b, ITC (top) and isotherm curves (bottom) of S. aureus ClpX against trigintamicin and its biologically inactive R-isomer. Data are representative of at least two independent determinations. d, Resistance changes of S. aureus and A. baumannii during serial passaging in the presence of erutacidin or apramycin. e, S. aureus kill curve elicited by erutacidin at varied concentrations. g, Fold change in erutacidin MIC against S. aureus caused by exogenous supplementation of various lipids. LOD, limit of detection; CL, cardiolipin; LPE, lysophosphatidylethanolamine; SH, sphingomyelin; AS, N-acetylsphingosine; MK, menaquinone-4. Data in d–g are presented as the mean values ± s.d. Neither direct plating nor prolonged passage of S. aureus or A. baumannii in the presence of erutacidin (Fig. Time-dependent killing curve analysis of S. aureus treated with erutacidin showed a rapid bactericidal effect (<10 min at ≥2× MIC) (Fig. This suggested disruption of membrane integrity as a potential MOA. To test erutacidin for membrane depolarization activity, we used a DiSC3(5)-based fluorescence assay. Treatment of S. aureus with erutacidin increased fluorescence similarly to the known depolarizer gramicidin, confirming membrane depolarization (Fig. For example, the antibiotics telomycin and daptomcyin interact with cardiolipin (also known as diphosphatidylglycerol) and phosphatidylglycerol (PG), respectively35,36. To further investigate erutacidin's MOA, we tested whether its activity could be suppressed by supplementation with various membrane lipids. Erutacidin's antibacterial activity was strongly inhibited by the addition of cardiolipin, reducing its activity eightfold at just a 0.5:1 molar ratio of cardiolipin to erutacidin (Fig. Supplementation of PG also caused a notable inhibition of activity, with an eightfold decrease in MIC at a 2:1 molar ratio of PG to erutacidin. Minor protection was provided by the supplementation of membrane lipids related to cardiolipin and PG, including phosphatidylethanolamine (PE) and phosphatidylcholine (PC) but no effect was observed for any other lipids tested or menaquinone (Fig. These data suggest that erutacidin interacts strongly and preferentially with cardiolipin but may also interact with related lipid structures. This may explain the lack of observed in vitro resistance for erutacidin, as acquiring mutations to overcome multiple inhibited molecular targets is very difficult. As such, we expected that multidrug-resistant (MDR) isolates would not show cross-resistance to erutacidin. Indeed, erutacidin maintained potent activity against all MDR S. aureus and MDR A. baumannii clinical isolates we tested (Supplementary Table 14). By achieving a substantial increase in read length, we show that it is possible to resolve complete contiguous metagenomic genomes from a complex soil sample. The distribution of bacterial taxa in soil is uneven, with a small number dominating the biomass29, and it is unclear what sequencing depth would be required to fully uncover very rare microbial dark matter. However, similar to trends observed with short-read sequencing38, continued improvements in efficiency and accuracy should allow deep long-read sequencing to increasingly become a routine tool in metagenomics. As with the rapid growth of short-read data38, this will necessitate the development of improved bioinformatic tools and computation infrastructure to assemble and explore terabase-scale long-read datasets. Nanopore specific assemblers are already becoming increasing powerful, with a very recent preprint introducing nanoMDBG as a fast and less computationally intensive assembly algorithm39. Bioinformatic algorithms for predicting the chemical output of BGCs are also improving, particularly with the incorporation of artificial intelligence and machine learning methods40,41, and we expect that the number and diversity of BGCs that can be decoded using a synBNP approach will continue to increase. Even in the small number of BGCs examined in this study through the synBNP approach, we could identify a broad-spectrum antibiotic with no detected resistance and with activity against very-difficult-to-treat MDR pathogens including the Gram-negative A. baumannii. By combining improved long-read access to metagenome sequence space with synBNP discovery, we established a pipeline to leverage the vast genetic diversity of microbial dark matter for the systematic detection, synthesis and screening of previously inaccessible bioactive molecules. This represents a scalable strategy to identify molecules inspired by BGCs from within the near-limitless uncultured bacterial majority. All strains, unless otherwise specified, were grown in Luria–Bertani (LB) at 30 °C with 200-rpm shaking. In order of collection, these comprised soil from a healthy forest (forest 1), sediment from a muddy bog, soil from the edge of a lake (forest 2), soil from a grove of dead trees (forest 3), decaying material from a dead tree and soil from a dry forest area (forest 4). Soil from a maximum depth of 10 cm was collected from an area of 50 cm2 into a gallon-sized Ziploc bag. A single collected bag of forest soil 1 was used for testing and scaling. Gradient centrifugation for separation of cells from the soil matrix and testing washes for removal of copurified contaminants were motivated by previous approaches in the field42,43,44. Soil (62.5 g) was added to 200 ml of prechilled 10 mM sodium pyrophosphate (Sigma) and blended (Hamilton Beach 10 speed) for 1 min using the LO/batter setting. The container was then placed on ice for 5 min. The 1-min blending and 5-min ice break were repeated two more times. Blended soil was then filtered through a cheesecloth (grade 90, bleached) to remove large insoluble particles and the filtrate was gently layered onto 8 ml of 36% w/v nycodenz (Accurate Chemical and Scientific Corporation) to the top of 50-ml conicals (Sarstedt). The tubes were centrifuged at 4 °C, 5,000g for 1.5 h in a swing bucket rotor yielding a fuzzy beige cell layer at the interface of the top (aqueous) and bottom (nycodenz) layer. The cell layer was gently aspirated and combined in a fresh 50-ml conical. The cell suspension was mixed 1:1 with sterile water and the mixture centrifuged at 4 °C, 5,000g for 1 h. The supernatant was discarded and the cell pellet was washed twice with 40 ml of JSM buffer (0.9% w/v NaCl and 1% w/v instant nonfat dry milk (Nestle)), followed by two washes with 40 ml of TE lysis buffer (25 mM Tris, 25 mM EDTA and 50 mM glucose). Lastly, the cell pellet was resuspended in a small volume (1–5 ml) of lysis buffer, the optical density at 600 nm (OD600) was checked and the bacterial suspension diluted as necessary for the desired DNA extraction protocol. DNA extraction directly from soil was performed as previously described18. Briefly, 2.4 ml of CTAB lysis buffer (100 mM Tris-HCl, 100 mM Na EDTA, 1.5 M NaCl, 1% w/v CTAB and 2% w/v SDS, pH 8.0) was added to 2 g of soil in a 5-ml tube and the slurry was incubated at 70 °C for 2 h. For the nycodenz-isolated bacteria, 3 ml of a 30-OD600 bacterial suspension was pelleted, then resuspended in 4 ml of CTAB lysis buffer and incubated at 70 °C for 2 h. After incubation, the suspensions were centrifuged at 20,000g for 10 min and the supernatant was moved to new tubes. Supernatants were combined with 0.7 volumes of isopropanol and incubated at room temperature for 10 min. Samples were centrifuged at 20,000g for 30 min and the resulting DNA pellets was washed twice with 1 ml of 70% ethanol. The pellets were then air-dried for 1–2 h at room temperature and resuspended in 100–200 µl of 5 mM Tris pH 8 overnight at 4 °C. This protocol is an adaptation of gel plug lysis described by Walter et al.45 Nycodenz-isolated bacteria were resuspended in lysis buffer to 10 OD600. The bacterial suspension was mixed 1:1 with 1.5% UltraPure low-melting-point agarose (Thermo Fisher Scientific) prepared in lysis buffer and cooled to 50 °C. The mixture was used to cast gel plugs in CHEF Mapper XA system 50-well plug molds (BioRad, 1703713). Gel plugs were then suspended in a solution of 5 mg ml−1 lysozyme and 1 mg ml−1 achromopeptidase prepared in lysis buffer and gently rotated at 37 °C overnight using a Roto-therm plus (Benchmark Scientific). The buffer was decanted and the plugs were topped up with TESP buffer (50 mM Tris, 100 mM EDTA, 1% w/v SDS and 1 mg ml−1 proteinase K) and incubated overnight with gentle rotation at 50 °C. The gel plug suspension was decanted and the plugs were topped up with fresh TESP and once again incubated overnight with gentle rotation at 50 °C. TESP was then decanted and the gel plugs were rinsed several times with milli-Q water before being suspended in T10E50 (10 mM Tris and 50 mM EDTA, pH 8) with 1 mM PMSF and incubated at room temperature for 2 h. The liquid was decanted and the plugs washed four times with T10E50 with each wash lasting 1 h on ice. Cell lysis was performed similarly to previously established approaches46,47. Nycodenz-isolated bacteria were resuspended in lysis buffer to 10 OD600 and 500 µl of the suspension was pelleted by centrifugation for 1 min at 20,000g. The cell pellet was resuspended in 800 µl of lysis buffer containing 5 mg ml−1 lysozyme and 1 mg ml−1 achromopeptidase and the suspension was incubated at 37 °C for 2 h. After incubation, 1 mg of proteinase K powder was added, followed by 200 µl of 5% w/v SDS. The sample was very gently inverted to mix and incubated for 3 h at 55 °C. The viscous solution was then layered onto a sucrose gradient for size selection. About 1 ml of 10-OD600 nycodenz-isolated bacteria was centrifuged at 20,000g for 1 min and the cell pellet was resuspended in 300 µl of TE lysis buffer containing 5 mg ml−1 lysozyme and 1 mg ml−1 achromopeptidase. The sample was incubated for 2 h at 37 °C and then carried forward using the New England Biolabs protocol for HMW DNA extraction from bacteria using the Monarch HMW DNA extraction kit for tissue (New England Biolabs, T3060). Briefly, 20 µl of proteinase K was added and the sample was incubated at 56 °C with varied shaking speeds (1,900, 1,700, 1,400 or 650 rpm) using a Fisherbrand Heat/Cool Thermal Mixer II (Fisher Scientific). After 30 min, 10 µl of RNase A was added and the sample was incubated a further 10 min with shaking at the same speed. Then, 300 µl of protein separation solution was added and the sample was mixed by inversion for 1 min. The sample was centrifuged for 10 min at 16,000g and the upper phase was then transferred to a Monarch 2-ml tube using a wide-bore pipette tip. Two DNA capture beads were added, followed by 550 µl of isopropanol. The liquid was then discarded and the beads were washed twice with 500 µl of genomic DNA wash buffer. Finally, DNA was eluted by addition of 100 µl of elution buffer II and incubation at 56 °C for 10 min with 300-rpm agitation. The DNA solution was left to resuspend at 4 °C overnight. The electroelution protocol was performed as previously described18 with minor modifications. Briefly, a 0.9% agarose gel was run in 0.5× TBE (VWR) at 6 V cm−1 for 2 h to size-select crude DNA extracts. The compression band containing HMW DNA was then excised, the gel slice was placed in dialysis tubing (Spectra/Por; 12,000–14,000-kDa molecular weight cutoff (MWCO)) and topped up with 0.5× TBE. DNA was then electroeluted in 0.5× TBE at 6 V cm−1 for 2 h. The gel slice was removed from the dialysis bag and the remaining liquid was concentrated to dryness against 25% PEG 8000 with 5 mM Tris pH 8. DNA was then rehydrated in the dialysis bag against 5 mM Tris pH 8 for isolation. Linear 10–40% sucrose gradients, with 10 mM Tris pH 8, were prepared using the freeze–thaw method48. Gradients were prepared by liquid nitrogen flash-freezing at the end of the day and immediately placed to thaw at 4 °C until use 22–24 h later. Two sizes of polypropylene tubes were used, 25 × 89 mm and 14 × 95 mm (Beckman), with 8-ml and 2.9-ml sucrose layers, respectively. Larger tubes were centrifuged using an SW28 rotor, while the smaller tubes were centrifuged using an SW40 Ti rotor. In both cases, centrifugation was at 4 °C, 40,000g for 16 h. After centrifugation, fractions were gently pipetted from the top of the tubes using wide-bore tips. Pulse-field gel electrophoresis was used to analyze the fractions. For shorter-fragment purification (that is, DNA from hot CTAB and Monarch 1,900 rpm), analysis was conducted using 0.9% agarose with 1× TAE at 6 V cm−1 for 4 h with a 5-s switch time. For ultralarge DNA, analysis was conducted using 0.9% agarose with 1× TAE at 6 V cm−1 for 16 h with a linear 3–7-s switch time. Ultralarge DNA from direct lysis was isolated using large tubes from fractions at 14 ml to 32 ml. DNA from Monarch 1,900 rpm was purified using large tubes from fractions at 6 ml to 8.5 ml, aiming to isolate DNA ~40 kb in size. DNA size selection with the SFE kit (Oxford Nanopore EXP-SFE001) was performed according to the manufacturer's instructions. DNA from gel-plug-extracted metagenomic DNA was isolated using the NucleoSpin gel and PCR cleanup kit (Macherey-Nagel) according to the manufacturer's instructions. A-domain amplicon sequencing was performed as previously described49. A-domain amplicons were trimmed to remove primer sequences using vsearch (version 2.28.1)50 fastx_filter and then assembled into operational taxonomic units (OTUs) clustered at 95% nucleotide similarity using vsearch cluster_size. A-domain OTUs for each soil were then compared across all samples at 95% similarity. Ultralong DNA sequencing kit V14 (SQK-ULK114) preparation was performed according to the manufacturer's instructions. Three DNA inputs were attempted, one following Oxford Nanopore directions on the Monarch HMW DNA extraction kit for tissue extraction, the manufacturer-recommended protocol for Monarch HMW DNA extraction kit for tissue and sucrose gradient purification. An additional attempt using the Monarch extracted DNA input was tried with a slightly altered protocol; instead of heat-killing the fragmentation mix, proteinase K was added to 1 mg ml−1 for 1 h at room temperature, followed by the addition of 3 mM PMSF for 1 h at room temperature to inhibit the proteinase K before continuing the protocol. Oxford Nanopore's ligation sequencing kit V14 (SQK-LSK114) was used to prepare sequencing samples, with DNA extraction and size selection as indicated, according to the manufacturer's instructions and minor modifications. Input DNA was increased to 3 µg, blunting and end-repair mix reaction incubation times were extended to 15 min at 20 °C and 10 min at 65 °C and adaptor ligation incubation time was increased to 30 min. Library solution was used for all samples and the DNA concentrations loaded are indicated in Supplementary Table 2. On the basis of our optimization experiments, the 1,700-rpm Monarch isolation and SFE size-selection protocol applied to 62.5 g of the forest soil would be sufficient to generate >2 Tbp of sequence. PromethION R10.4.1 flow cells were used to generate all of the sequencing data. Nanopore sequencing was conducted using a P2 solo with MinKnow 23.04.6 (ultra kits) or 23.11.4 (Ligation kits) and a P24 with MinKnow 23.07.12 (scaled ligation). All raw data were duplex-basecalled with standalone Dorado 0.4.1 + 6c4c636 for subsequent analysis. The resulting 528.5-Gbp dataset was assembled using Flye (version 2.9.3-b1797) in metagenomic mode (metaFlye) with parameters ‘--nano-hq --i 0 --threads 64 --meta' (ref. The assembly was run on Rockefeller University's large-memory high-performance computing node (64 cores, 3 TB of RAM) and took just under 8 days to complete with RAM usage peaking at 845 GB. The assembly was polished once (the emerging best practice53) with Medaka (version 1.11.3; https://github.com/nanoporetech/medaka) and annotated by Bakta (version 1.10.4)54. Assembled contigs > 1 Mbp were extracted and their taxonomy assigned using the GTDB toolkit (GTDB-tk version 2.4.0)55 classify workflow with GTDB release 09-RS220. The presence of rRNA was detected within the Bakta annotation using a GNU AWK script (version 5.3.0), while tRNAscan-SE (version 2.0.9) was used to quantify total unique tRNAs. A set of 563 contigs > 1 Mbp were identified containing all rRNAs and at least 18 unique tRNAs to establish the complete or near-complete metagenomic genome dataset. The taxonomic classification of these contigs was plotted as a Sankey diagram using the networkD3 R package 0.4 (Fig. AntiSMASH (version 6.1.1)58 results from the output JSON files were used to generate the appended BGC summaries. Subtrees expanding previously underexplored clades were generated by extracting the alignment information from the GTDB-tk classify workflow for desired sequences and plotting with FastTree (version 2.1.11)59 using default settings. HMMER (version 3.1b2)60 was used to scan the total sequencing dataset for AMP-binding domains using an available HMM (AMP-binding, Pfam PF00501.23). Reads containing AMP-binding domains were assembled (metaFlye), polished (Medaka) and annotated (Bakta), as described above. Similar to the complete and near-complete genome assembly dataset, frameshifts were minimized using proovframe and then the subset was analyzed by antiSMASH (version 6.1.1). For analysis of NRPS content, the antiSMASH results were analyzed by BiG-SCAPE (version 1.1.5) to generate Fig. 61) toolkit's easy-taxonomy workflow and the National Center for Biotechnology Information nonredundant protein database. Reads containing 16S rRNA sequences were extracted from the 2.5 Tbp of sequencing data using barrnap 0.9 (https://github.com/tseemann/barrnap). The 16S rRNA read subset was then assembled (metaFlye), polished (Medaka) and annotated (Bakta) as described above. Annotated 16S rRNA gene sequences were then extracted and, using vsearch, dereplicated and clustered at 99%, the recommended threshold for clustering full-length 16S sequences62. Lastly, to ensure full-length sequences, only OTUs between 1,400 and 1,700 bp were retained. The same extraction, dereplication and clustering process was used to identify 16S rRNA genes within the global assembly dataset. Phylogeny was assigned using mmseqs2 and the SILVA database (release 138.2)63. Forest soil 1 microorganisms were isolated using the nycodenz method with the last two washes replaced with Dulbecco's PBS (DPBS; Gibco). The purified suspension was diluted to 0.05 OD600 in DPBS and 200 µl was spread on individual agar plates (150 mm × 15 mm). Ten plates each were used with three different media: R2A agar (Fisher Scientific), which was previously used to isolate the Aquella species27, and two versions of dilute nutrient broth (Criterion). In one case, nutrient broth was diluted 200-fold compared to the manufacturer's recommendation; in the second case, dilute nutrient broth was with supplemented 1:1 (v/v) with soil extract64. In all cases, nystatin (50 µg ml−1) was added to limit fungal growth. Plates were incubated at 30 °C with R2A colonies picked after 3 days and nutrient broth colonies picked after 10 days. Liquid R2A or unmodified nutrient broth were used to culture isolates for sequencing (3 ml, 30 °C, 200-rpm shaking). Bacterial cells were pelleted from turbid cultures (1–1.5 ml) by centrifugation for 1 min at 20,000g. The supernatant was removed and the cell pellets were caried forward for genomic DNA isolation using the PureLink microbiome DNA purification kit (Thermo Fisher Scientific) as per manufacturer's instructions for soil samples with elution volume lowered to 50 µl. Oxford Nanopore's native barcoding kit 24 V14 (SQK-NBD114.24) was used as per the manufacturer's instructions to prepare sequencing samples. From R2A medium culturing, 22 genomes were individually barcoded and sequenced. From nutrient medium culturing, 20 genomes were individually barcoded and 12 were barcoded as pools of three. Each medium set was sequenced using a PromethION flow cell. Isolate genomes were assembled using Flye with Medaka polishing and Bakta annotation. Genome assemblies were dereplicated by extracting 16S rRNA genes, clustering sequences at 99% identity and then selecting a single assembly per cluster. BGCs were identified in each dereplicated cultured genome using antiSMASH. Large NRPS BGCs (that is, ≥4 A domains) were compared to those from the targeted metagenomic assembly using mmseqs2. SynBNP prediction was carried out according to our standard workflow65. BGCs were scanned for NRPS domains and modules. The order in which these domains are likely to function was predicted on the basis of canonical NRPS module organization combined with the principle of collinearity. The following BGCs were then removed: (1) BGCs lacking predicted initiation or termination domains; (2) BGCs with modules that do not follow canonical NRPS domain organization (that is, condensation domain, A-domain and thiolation domain); (3) BGCs rich in tailoring genes; and (4) BGCs having fewer than four A domains, as these make poor synBNP targets. Structure predictions were then generated on the basis of A-domain specificity and derived structural modifications predicted on the basis of an analysis of additional domains present in the BGC (for example, epimerization and N-methylation). Specificity codes were compared to both the antiSMASH output and an in-house specificity code table collated from previously characterized natural products. The highest A-domain matches were solely considered for predictions. In rare cases where multiple amino acids share the same code, one was selected from the consensus of predictions generated by NRPSPredictor2 within antiSMASH. If multiple predictions were tied after the consensus stage, one was selected at random. The set of bioinformatically tractable clusters considered and their corresponding predictions are given in Supplementary Table 8. This represented roughly 19% of detected NRPS BGCs. The average A-domain match per peptide was 89.9% ± 8.0%. From the subset of predictions that contained at most one <70% A-domain specificity code (~83% of the predictions passed this cutoff), we randomly selected 20for synthesis. These covered a range of phylogenetic classifications (Fig. Contig_44957 had two mismatched codes but was retained as the codes were closely related and predicted to specify the same amino acid. While we used a cutoff of NRPs containing at most a single A-domain that matched <70%, this can be loosened or more restrictive if desired. A breakdown of predictions that passed different cutoff levels is provided in Supplementary Table 10. Reagents and solvents used for synthesis were obtained from commercial sources and used without further purification. Peptides were purified using a CombiFlash EZ Prep purification system with ultraviolet detection and equipped with a RediSep gold HP C18 column (30 g), RediSep gold HP C8 column or Phenomenex Luna 5 µm C18 prepHPLC column using a 5–95% acetonitrile–water gradient supplemented with 0.1% formic acid. High-resolution mass spectrometry (HRMS) data were acquired on a SCIEX ExionLC ultra-HPLC (UPLC) instrument coupled to an X500R quadrupole time-of-flight mass spectrometer with a Phenomenex Kinetex PS C18 100-Å column (2.1 mm × 50 mm, 2.6 µm) and analyzed using SCIEXOS software. 1H and 13C nuclear magnetic resonance (NMR) spectra were obtained on a Bruker Avance DMX 600-MHz spectrometer equipped with cryogenic probe (Rockefeller University) and all spectra were analyzed using MestRe Nova software. Linear peptides were synthesized using standard Fmoc-based solid-phase peptide synthesis (SPPS) methods. 2-Chlorotritylchloride resin (0.1 mmol) was swollen in dichloromethane (DCM) at room temperature for 30 min. After draining, the resin was washed with DCM (three washes, 5 ml each). The C-terminal Fmoc-protected amino acid (four equivalents, 0.4 mmol) was dissolved in 5 ml of DCM and treated with N,N-diisopropylethylamine (DIPEA; five equivalents, 0.5 mmol). This mixture was added to the swollen resin and agitated for 1 h. The solution was drained and the resin was washed with DCM (three washes, 5 ml each). The resin was then capped by treatment with a 17:2:1 mixture of DCM, methanol and DIPEA. The resin was agitated for 30 min, drained and washed with DCM (three washes, 5 ml each) and dimethylformamide (DMF; three washes, 5 ml each). For peptide couplings, the Fmoc group was removed by treating the resin with 20% piperidine in 5 ml of DMF for 7 min. The next Fmoc amino acid to be coupled (three equivalents, 0.3 mmol) and hexafluorophosphate azabenzotriazole tetramethyl uronium (HATU; three equivalents, 0.3 mmol) was dissolved in 4 ml of DMF and treated with DIPEA (three equivalents, 0.3 mmol). This solution was added to the resin and agitated for 45 min. The solution was drained and the resin was washed with DMF (three washes, 5 ml each). This deprotection and coupling process was repeated for each subsequent amino acid. Finally, the N-terminal Fmoc amino acid was deprotected with 20% piperidine in DMF (two cycles of 5 ml for 7 min) and reacted with myristic acid (three equivalents, 0.3 mmol), HATU (three equivalents, 0.3 mmol) and DIPEA (three equivalents, 0.3 mmol) in DMF (4 ml). The reaction was agitated for 45 min, at which point the resin was washed with DMF (three washes, 5 ml each) and DCM (three washes, 5 ml each). The peptide was cleaved from resin and the side-chain-protecting groups were removed by treating the resin with a trifluoroacetic acid (TFA) cleavage cocktail containing 95% TFA, 2.5% water and 2.5% triisopropylsilane for 2 h. The solution was evaporated under air flow and the resulting residue was purified on a CombiFlash EZ Prep system (30-g HP C18 column, water–acetonitrile with 0.1% v/v formic acid 5%–95%). Peptides cyclized through the side chain of an amino acid were synthesized starting from the penultimate amino acid following the same Fmoc SPPS loading, Fmoc deprotection and coupling procedures described above. Side-chain serine and threonine residues at which cyclization was predicted to occur were added to the peptide with the side chain unprotected. After the N-terminal amino acid was deprotected, myristic acid was coupled to the terminal amine as described above (for peptides not predicted to have a lipid tail, the Boc-protected N-terminal amino acid was coupled). Next, the C-terminal amino acid was esterified onto the free serine or threonine side chain by adding a solution of Fmoc amino acid (15 equivalents, 1.5 mmol), N,N′-diisopropylcarbodiimide (DIC; 15 equivalents, 1.5 mmol) and 4-dimethylaminopyridine (DMAP; 0.5 equivalents, 0.05 mmol) in 5 ml of DMF to the resin and agitating for 16 h. The solution was drained and the resin was washed with DMF (three washes, 5 ml each) and DCM (three washes, 5 ml each), followed by deprotection of the Fmoc group using 20% piperidine in DMF (two cycles of 5 ml for 7 min). The linear precursor peptide was gently cleaved from resin by treating with a solution of 20% hexafluoroisopropanol in 6 ml of DCM for 45 min. After removal of solvent under air flow, the resulting peptide was dissolved in DMF (50 ml, 0.002 M) and treated with (7-azabenzotriazol-1-yloxy)tripyrrolidinophosphonium hexafluorophosphate (seven equivalents, 0.7 mmol) and DIPEA (30 equivalents, 3 mmol). After stirring for 2 h, the solution was diluted with ethyl acetate and washed with brine (three washes, 100 ml each). The organic layer was dried (Na2SO4), filtered and concentrated in vacuo. The resulting cyclic peptide was globally deprotected by treatment with TFA cleavage cocktail for 2 h. After evaporation under air flow, the crude cyclic peptide was purified on a CombiFlash EZ Prep system as described above. After synthesis of the linear precursor was completed, the N-terminal Fmoc group was removed using 20% piperidine in DMF (two cycles of 5 ml for 7 min) and reacted with (±)-3-hydroxymyristic acid (three equivalents, 0.3 mmol), HATU (three equivalents, 0.3 mmol) and DIPEA (three equivalents, 0.3 mmol) in DMF (4 ml). The reaction was agitated for 45 min, at which point the resin was washed with DMF (three washes, 5 ml each) and DCM (three washes, 5 ml each). Next, the C-terminal amino acid was esterified onto the hydroxyl group of the fatty acid by adding a solution of Fmoc amino acid (15 equivalents, 1.5 mmol), DIC (15 equivalents, 1.5 mmol) and DMAP (0.5 equivalents, 0.05 mmol) in 5 ml of DMF to the resin and agitating for 16 h. The solution was drained and the resin was washed with DMF (three washes, 5 ml each) and DCM (three washes, 5 ml each), followed by deprotection of the Fmoc group using 20% piperidine in DMF (two cycles of 5 ml for 7 min). The linear precursor was cleaved from resin, cyclized, deprotected and purified as described above. All compounds that displayed antibacterial activity were confirmed to have >95% purity. To avoid solubility issues, each synBNP concentration was prepared individually in DMSO before addition to LB. A series of eight concentrations were tested, with the last one being a DMSO only control. First, the synBNP was diluted to 3.2 mg ml−1 in DMSO and then serially diluted twofold six times in DMSO. Then, 4 µl of each concentration was added to 100 µl of LB dispensed across a 96-well plate (Costar, 3370). Test bacteria were grown overnight shaking at 200 rpm at 30 °C and the culture was diluted to an OD600 of 0.005. Next, 100 µl of the dilution was added to the antibiotic dilution series, yielding a test range of 64 to 1 µg ml−1 along with a control (0 µg ml−1). The plate was then incubated statically at 30 °C for 18 h, followed by visual inspection. MICs were determined by two biologically independent replicates for the initial synBNP screen and three independent replicates for all other determinations. The inhibitory activity of synBNPs was assessed using a 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay66. HEK239 and HeLa cells were cultured in DMEM supplemented with 10% FBS, 10 µg ml−1 penicillin–streptomycin, 2 µM glutamine and nonessential amino acids. Cells were passaged and seeded in 96-well plates at the exponential growth phase, inoculated at ~1 × 105 cells per ml and maintaining a cell density below 1 × 106 cells per ml. All assays were performed using 5,000 cells per well, with 50 µl of a homogeneous cell suspension at 1 × 105 cells per ml seeded in each well. For the high-concentration assays (32 µg ml−1), 1.5 µl of synBNP stocks were used to prepare a master solution by dilution in 150 µl of DMEM without phenol red. Two 50-µl aliquots of the master solution were added to cell-seeded plates, obtaining duplicate 100-µl wells with synBNPs at 32 µg ml−1 (0.5% DMSO). For the low concentrations (3.2 µg ml−1), 15 µl of the master solution was diluted to 150 µl with DMEM and 50 µl was added to cell-seeded plates to obtain duplicate wells of 100 µl at 3.2 µg ml−1 (0.05% DMSO). DMEM solutions containing 0.5% and 0.05% DMSO were used as vehicle controls to define 100% cell viability. Camptothecin at 40 µg ml−1 was used as a positive control to define 0% cell viability. After treatment, plates were incubated for 48 h followed by addition of 10 µl of 5 mg ml−1 MTT solution in DPBS and the plate was incubated for an additional 4 h (SK-N-SH was incubated for 5–6 h). After incubation, 90 µl of solubilization solution (40% DMF, 16% SDS and 2% acetic acid) was added to each well and formazan crystals were allowed to dissolve overnight. The absorbance of each well was measured at 570 nm using a TECAN Infinite M NANO+. Absorbance values were normalized to the positive and negative controls and percentage survival values were calculated. S. aureus SH1000 was streaked onto a LB agar plate and incubated overnight at 30 °C. Three independent colonies were then inoculated into 3 ml of LB and grown overnight at 30 °C with shaking at 200 rpm. The growing cultures were then diluted into a preprepared series of 3 ml of LB containing the indicated antibiotic concentrations to ~6 × 106 colony-forming units (cfu) per ml using the expectation that 1 OD600 of S. aureus contained 5 × 108 cfu per ml. The cultures were then placed shaking at 200 rpm and 30 °C and samples were taken at 10 min, 30 min, 1 h and then every subsequent hour for a total of 6 h. For cfu quantification, 100-µl samples were placed immediately into the top well of a 96-well plate and serially diluted tenfold (20 µl into 180 µl of LB) down to 10−7. Then, 6 µl of each dilution from the series was spotted onto LB agar, the spots were air-dried and the plate was incubated overnight at 30 °C. Resulting colonies were enumerated to calculate the cfu per ml. S. aureus SH1000 was streaked onto an LB agar plate and incubated overnight at 30 °C. A single colony was inoculated into 3 ml of LB and grown to confluence overnight at 30 °C. The culture was then diluted to roughly 1 × 107 cells per ml in LB containing antibiotic at 8× MIC. Five 96-well plates were then seeded with 100 µl of the dilution and grown stationary overnight at 30 °C. Mutants appeared as small colonies at the bottom of the well and were picked into fresh 8× MIC medium to ensure resistance. The confirmation culture was then streaked onto LB agar and incubated overnight at 30 °C; two colonies per mutant were inoculated for an overnight culture in LB at 30 °C. The MIC of each mutant was checked to confirm resistance. For erutamicin, no resistant colonies appeared and the direct plating was repeated at 4× and 2× MIC concentrations, which also did not produce resistant mutants. Three independent colonies for each of the bacteria were then inoculated into 3 ml of LB and grown overnight at 30 °C with shaking at 200 rpm. The cultures were used to prepare a MIC assay as described above with the indicated antibiotics. The following day, 1 µl was taken from the well containing the highest concentration of antibiotic exhibiting turbid growth and mixed into 1 ml of fresh LB. This dilution was used to seed a new MIC assay. Genomic DNA from resistant mutants was extracted for sequencing using the PureLink microbiome DNA purification kit (Thermo Fisher Scientific) per the manufacturer's instructions. Isolated DNA was then Illumina-sequenced using standard Nextera preparation and the MiSeq reagent kit v3 with 2× 300-bp reads. Reads for both parent and mutants were mapped to a reference genome using minimap2 (version 2.28-r1209)67. Changes detected in the parent strain were eliminated from consideration and mutations were uncovered using snippy (https://github.com/tseemann/snippy). The clpX gene was amplified from S. aureus SH1000 genomic DNA by PCR using the primers ClpXF (TATTACTCGAGATGTTTAAATTCAATGAAGATGAAG) and ClpXR (TAATAGGATCCACATCAATGATTAAGCTGATG). The pET19b (Sigma) vector backbone was amplified by 19BF (TAATAGGATCCGAAAGGAAGCTGAGTTGG) and 19BR (TATTACTCGAGCATATGCTTGTCGTCGTCGTC). The two PCR products were then digested with XhoI and BamHI and ligated together; the ligation reaction was transformed into E. coli EPI300 for colony screening on LB agar plates containing 100 µg ml−1 carbapenem. The pHisClpX vector was transformed into E. coli Rosetta 2(DE3) (Sigma) and a single colony was inoculated into 5 ml of LB containing carbapenem to grow overnight at 37 °C and 200-rpm shaking. The overnight culture was then added to 1 L of LB + carbapenem prewarmed to 37 °C and grown at 37 °C and 200-rpm shaking until an OD600 of 0.2–0.4. Once the desired OD range was achieved, the culture was transferred to 16 °C and 200-rpm shaking for 30 min, after which IPTG was added to 1 mM to induce protein expression, which continued overnight at 16 °C and 200 rpm. The suspension was sonicated using a Fisherbrand sonicator for a total of 7.5 min of sonication time, with 45 s on and 45 s off at 45% amplitude. The sonicated suspension was then centrifuged at 4,000g for 30 min. The supernatant was then centrifuged at 23,000g for an additional 30 min. Bound ClpX was then washed with 20 ml of increasing concentrations of imidazole in protein lysis buffer (10 mM, 20 mM and 30 mM), followed by elution with protein lysis buffer containing 250 mM imidazole, while collecting 0.5-ml fractions. The protein content in the eluted fractions was checked by absorbance at 280 nm, with any fractions at an OD > 0.6 pooled together. About 1 ml of the pooled fractions were then dialyzed (MWCO: 12,000–14,000 kDa; Spectra/Por) against 1 L of protein storage buffer (50 mM Tris pH 7.5, 200 mM KCl, 25 mM MgCl2, 1 mM β-mercaptoethanol and 10% glycerol) for 3 h at 4 °C, followed by an additional dialysis against 3 L of protein storage buffer overnight. The protein concentration was then determined to be 5.83 mg ml−1 (117 µM) by spectroscopy at 280 nm with the calculation of 1 OD corresponding to 3.69 mg ml−1 of His-ClpX. A sample (50 ml) of the 3-L storage buffer after dialysis was saved for resuspending trigintamicin and its isomer for ITC. The ligands (trigintamicin and its biologically inactive isoform) were dissolved in storage buffer taken post dialysis of ClpX. Then, 300 μl of ClpX solution was placed in the sample cell and titrated by ligand solution through 19 successive injections of 2 μl with a 2.5-min interval between injections and a reference power of 5 μcal s−1. Titration data were analyzed using the PEAQ-ITC analysis software (version 1.41). The isotherm curves were best fitted to a single-site binding model. An overnight culture of S. aureus SH1000 was pelleted, washed twice in equal volume of DPBS (Thermo Fisher Scientific) and diluted to 0.35 OD600 in DPBS. The mixture was incubated at room temperature in the dark for 15 min, followed by the addition of 50 µl of 2 M KCl and another 15-min incubation. The samples were then transferred into a 384-well flat, clear-bottom black microtiter plate with 30 µl per well. The initial fluorescence intensity of each well was recorded using a TECAN Infinite M NANO+ (excitation, 620 nm; emission, 675 nm) at 10-s intervals until the baseline stabilized. After signal stabilization, 30 µl of test compounds diluted in DPBS were added to the indicated concentrations and readings resumed. Gramicidin at 16 µg ml−1 (2× MIC) was used as a positive control for depolarization. Test lipids were dissolved in the manufacturer's recommended solvent to 0.5–1 mg ml−1. An aliquot containing the appropriate amount of lipid for the desired molar ratio to a target antibiotic was transferred to a 1.5-ml Eppendorf and dried under vacuum for 1 h. The dried pellet was then resuspended by pipetting with 6 µl of the test antibiotic dissolved in ethanol at 1.28 µg ml−1. The suspension was then topped up with 114 µl of LB and the sample was mixed by pipetting. An overnight culture of S. aureus SH1000 was diluted 1:1,000 into LB and 50 µl was added to the dilution series to generate a final antibiotic concentration range of 0.25 to 32 µg ml−1. The plate was then incubated overnight at 30 °C for 18 h, after which growth was visually inspected. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Publicly available GTDB release 09-RS220 was used for taxonomic assignment and comparison. Publicly available AMP-binding protein (Pfam PF00501.23) was used for HMM searches. Publicly available SILVA database release 138.2 was used for taxonomic assignment of 16S rRNA sequences. Source data are provided with this paper. Nanopore sequencing was performed using a P2 solo with MinKnow 23.04.6 (ultra kits) or 23.11.4 (ligation kits) and a P24 with MinKnow 23.07.12 (scaled ligation). All raw data were duplex-basecalled with standalone Dorado 0.4.1+6c4c636. Publicly available vsearch 2.28.1, GNU parallel 20230922, Chopper 0.6.0, Flye 2.9.3-b1797, Medaka 1.11.3, Bakta 1.10.4, GTDB-tk 2.4.0, GNU AWK script 5.3.0, tRNAscan-SE 2.0.9, networkD3 R package 0.4, antiSMASH 6.1.1, FastTree 2.1.11, HMMER 3.1b2, proovframe 0.9.7, CheckM 1.2.3, Barrnap 0.9, BiG-SCAPE 1.1.5, minimap2 2.28-r1209 and Snippy 4.6.0 were used for sequence assembly and analysis. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Thompson, L. R. et al. A communal catalogue reveals Earth's multiscale microbial diversity. The reliability of metagenome-assembled genomes (MAGs) in representing natural populations: insights from comparing MAGs against isolate genomes derived from the same fecal sample. Chen, L. X., Anantharaman, K., Shaiber, A., Eren, A. M. & Banfield, J. F. Accurate and complete genomes from metagenomes. Almeida, A. et al. A new genomic blueprint of the human gut microbiota. Meleshko, D. et al. BiosyntheticSPAdes: reconstructing biosynthetic gene clusters from assembly graphs. Meslier, V. et al. Benchmarking second and third-generation sequencing platforms for microbial metagenomics. Orellana, L. H., Kruger, K., Sidhu, C. & Amann, R. Comparing genomes recovered from time-series metagenomes using long- and short-read sequencing technologies. Connecting structure to function with the recovery of over 1,000 high-quality metagenome-assembled genomes from activated sludge using long-read sequencing. Eisenhofer, R. et al. A comparison of short-read, HiFi long-read, and hybrid strategies for genome-resolved metagenomics. Generating lineage-resolved, complete metagenome-assembled genomes from complex microbial communities. Zhao, W. et al. Oxford Nanopore long-read sequencing enables the generation of complete bacterial and plasmid genomes without short-read sequencing. Sereika, M. et al. Oxford Nanopore R10.4 long-read sequencing enables the generation of near-finished bacterial genomes from pure cultures and metagenomes without short-read or reference polishing. Verma, S. K., Singh, H. & Sharma, P. C. An improved method suitable for isolation of high-quality metagenomic DNA from diverse soils. & Brady, S. F. Bioactive synthetic–bioinformatic natural product cyclic peptides inspired by nonribosomal peptide synthetase gene clusters from the human microbiome. Brady, S. F. Construction of soil environmental DNA cosmid libraries and screening for clones that produce biologically active small molecules. Kolmogorov, M. et al. metaFlye: scalable long-read metagenome assembly using repeat graphs. Riley, R. et al. Terabase-scale coassembly of a tropical soil microbiome. Crits-Christoph, A., Diamond, S., Butterfield, C. N., Thomas, B. C. & Banfield, J. F. Novel soil bacteria possess diverse genes for secondary metabolite biosynthesis. Van Goethem, M. W. et al. Long-read metagenomics of soil communities reveals phylum-specific secondary metabolite dynamics. Parks, D. H. et al. A complete domain-to-species taxonomy for Bacteria and Archaea. Kazarina, A., Wiechman, H., Sarkar, S., Richie, T. & Lee, S. T. M. Recovery of 679 metagenome-assembled genomes from different soil depths along a precipitation gradient. Jensen, P. R. Natural products and the gene cluster revolution. Chan, K. G. et al. Aquella oligotrophica gen. nov. sp. nov.: a new member of the family Neisseriaceae isolated from laboratory tap water. Delgado-Baquerizo, M. et al. A global atlas of the dominant bacteria found in soil. & Rolff, J. Genomic signatures of experimental adaptation to antimicrobial peptides in Staphylococcus aureus. Kawada-Matsuo, M., Le, M. N. & Komatsuzawa, H.Antibacterial peptides resistance in Staphylococcus aureus: various mechanisms and the association with pathogenicity. & Sauer, R. T. ClpXP, an ATP-powered unfolding and protein-degradation machine. The ClpXP protease is dispensable for degradation of unfolded proteins in Staphylococcus aureus. & Schimmer, A. D. Mitochondrial ClpP serine protease—biological function and emerging target for cancer therapy. Taylor, S. D. & Palmer, M. The action mechanism of daptomycin. Assembly and clustering of natural antibiotics guides target identification. Kleijn, L. H. J. et al. A high-resolution crystal structure that reveals molecular details of target recognition by the calcium-dependent lipopeptide antibiotic laspartomycin C. Angew. The real cost of sequencing: higher than you think! High-quality metagenome assembly from nanopore reads with nanoMDBG. Huang, J. et al. DeepAden: an explainable machine learning for substrate specificity prediction in nonribosomal peptide synthetases. Mongia, M. et al. AdenPredictor: accurate prediction of the adenylation domain specificity of nonribosomal peptide biosynthetic gene clusters in microbial genomes. Soil pretreatment and fast cell lysis for direct polymerase chain reaction from forest soils for terminal restriction fragment length polymorphism analysis of fungal communities. Amalfitano, S. & Fazi, S. Recovery and quantification of bacterial cells associated with streambed sediments. Walter, J., Mangold, M. & Tannock, G. W. Construction, analysis, and β-glucanase screening of a bacterial artificial chromosome library from the large-bowel microbiota of mice. Morita, H. et al. An improved DNA isolation method for metagenomic analysis of the microbial flora of the human intestine. Luthe, D. S. A simple technique for the preparation and storage of sucrose gradients. Libis, V. et al. Uncovering the biosynthetic potential of rare metagenomic DNA using co-occurrence network analysis of targeted sequences. Rognes, T., Flouri, T., Nichols, B., Quince, C. & Mahe, F. VSEARCH: a versatile open source tool for metagenomics. De Coster, W. & Rademakers, R.NanoPack2: population-scale evaluation of long-read sequencing data. Luan, T. et al. Benchmarking short and long read polishing tools for nanopore assemblies: achieving near-perfect genomes for outbreak isolates. Schwengers, O. et al. Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification. Chaumeil, P. A., Mussig, A. J., Hugenholtz, P. & Parks, D. H. GTDB-Tk v2: memory friendly classification with the genome taxonomy database. Hackl, T. et al. proovframe: frameshift-correction for long-read (meta)genomics. Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. & Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Blin, K. et al. antiSMASH 6.0: improving cluster detection and comparison capabilities. Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. Steinegger, M. & Soding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Edgar, R. C. Updating the 97% identity threshold for 16S ribosomal RNA OTUs. The SILVA and ‘All-Species Living Tree Project (LTP)' taxonomic frameworks. George, I. F., Hartmann, M., Liles, M. R. & Agathos, S. N. Recovery of as-yet-uncultured soil acidobacteria on dilute solid media. Wang, Z., Koirala, B., Hernandez, Y., Zimmerman, M. & Brady, S. F. Bioinformatic prospecting and synthesis of a bifunctional lipopeptide antibiotic that evades resistance. Burian, J. et al. Datasets for ‘Bioactive molecules unearthed by terabase-scale long-read sequencing of a soil metagenome'. We thank J.-N. Audet at the Rockefeller University Field Research Center for coordinating sample collection. We are grateful to C. Mason at Weill Cornell Medicine for access to a P24 sequencer and his technician K. Ryon for coordinating use. We also thank the Rockefeller University High-Throughput Screening and Spectroscopy Resource Center for assistance with ITC experiments. This work was supported by National Institutes of Health grant R35GM122559 (S.F.B. Laboratory of Genetically Encoded Small Molecules, Rockefeller University, New York, NY, USA Ján Burian, Robert E. Boer, Yozen Hernandez, Adrian Morales-Amador, Linhai Jiang, Abir Bhattacharjee, Cecilia Panfil, Melinda A. Ternei & Sean F. Brady Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar J.B. conceptualized, designed, performed and analyzed the experiments. assisted J.B. in the sample collection and DNA preparation. All authors were involved in reviewing the manuscript. Nature Biotechnology thanks Rayan Chikhi, Kim Lewis and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Source data, in tabs, for panels b,d,e,f,g. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Burian, J., Boer, R.E., Hernandez, Y. et al. Bioactive molecules unearthed by terabase-scale long-read sequencing of a soil metagenome. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-62996-0'>Matrix directs trophoblast differentiation in a bioprinted organoid model of early placental development</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 09:27:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Trophoblast organoids can provide crucial insights into mechanisms of placentation, however their potential is limited by highly variable extracellular matrices unable to reflect in vivo tissues. Here, we present a bioprinted placental organoid model, generated using the first trimester trophoblast cell line, ACH-3P, and a synthetic polyethylene glycol (PEG) matrix. Bioprinted or Matrigel-embedded organoids differentiate spontaneously from cytotrophoblasts into two major subtypes: extravillous trophoblasts (EVTs) and syncytiotrophoblasts (STBs). Bioprinted organoids are driven towards EVT differentiation and show close similarity with early human placenta or primary trophoblast organoids. Inflammation inhibits proliferation and STBs within bioprinted organoids, which aspirin or metformin (0.5 mM) cannot rescue. We reverse the inside-out architecture of ACH-3P organoids by suspension culture with STBs forming on the outer layer of organoids, reflecting placental tissue. Our bioprinted methodology is applicable to trophoblast stem cells. We present a high-throughput, automated, and tuneable trophoblast organoid model that reproducibly mimics the placental microenvironment in health and disease. Organoids derived from single cells offer a powerful model for studying organ-like structures that recapitulate key features of native tissues1. However, their widespread reliance on animal-derived matrices such as Matrigel introduces variability and limits reproducibility, hindering biological insights and translational potential2,3. Given the critical role of the extracellular matrix (ECM) in guiding cell behaviour, including differentiation, organisation and function, there is a pressing need for tuneable, biologically relevant alternatives4,5,6,7,8. Emerging bioprinting technologies, including extrusion and droplet-based methods, enable precise deposition of cells within natural or synthetic hydrogels, offering scalable and reproducible platforms for 3D cell culture that are capable of supporting high-throughput workflows4. The placenta plays a vital role in supporting foetal development, and its dysfunction is implicated in numerous pregnancy complications, including preeclampsia, intrauterine growth restriction (IUGR), miscarriage, preterm birth, stillbirth, placental accreta, increta and percreta5,6,7,8,9,10,11,12,13,14. These conditions not only affect pregnancy outcomes but also have long-term health implications for both mother and child, increasing the risk of future cardiovascular, endocrine and neurological diseases15,16. Despite this, treatment options remain limited and deeper insights into placental biology are urgently needed. Placental organoids, first described in 2018, offer a promising model for studying early placental development and the pathogenesis of pregnancy complications17,18. While current protocols have advanced our understanding, they often rely on Matrigel, primary tissue from pregnancy terminations or invasive chorionic villous sampling, and costly media components, limiting reproducibility and accessibility19,20,21,22,23,24,25,26,27,28,29. Additionally, many trophoblast organoids display an inverted cellular architecture, hindering the study of key physiological processes, including diffusion across placental surfaces and cell secretions. In this study, we evaluate the organoid-forming potential of the immortalised first-trimester trophoblast cell line, ACH-3P, using both traditional Matrigel embedding and droplet-on-demand bioprinting with a synthetic, polyethylene glycol (PEG) based matrix. Organoids form in both conditions and differentiate into major trophoblast subtypes, with matrix composition influencing lineage specification: Matrigel driving organoids towards syncytialisation and PEG promoting an EVT phenotype. We further apply our bioprinted organoids to model inflammatory conditions and assess the effects of current and emerging therapies for preeclampsia, aspirin and metformin30,31,32. We report that these treatments at 0.5 mM concentration cannot rescue the impact on organoid metabolic activity or the reduction in STBs. To better mimic placental architecture, we implement a suspension method that reverses organoid polarity, promoting syncytialisation at the outer surface. Finally, we demonstrate that trophoblast stem cells (TSCs) form organoids in both matrices, although growth is restricted in the stiffer PEG matrix using bioprinting. Our findings underscore the importance of matrix properties in guiding organoid architecture and trophoblast differentiation, offering an automated, precise, scalable and reproducible platform for studying placental development and high-throughput drug testing for placental dysfunction disorders, including preeclampsia and IUGR. Firstly, we performed a matrix selection to determine which PEG matrix is optimal for ACH-3P organoid formation by comparing three 1.1 kPa PEG formulations: (i) a blank matrix with no adhesion peptides, (ii) a matrix containing α- and β-laminin peptide chains (IKVAV and YIGSR), and (iii) a matrix containing collagen type I peptide mimetic (GFOGER) based on proteins expressed in human placental tissue19,20,33. These were manufactured and provided by a company called Inventia Life Sciences in Australia without any further modification; thus, reproducibility is ensured through the manufacturer's QC methods for any lab utilising the RASTRUM bioprinting platform. The matrix components are tailored to the placental microenvironment by matching the stiffness and ECM components of the placental tissue and underlying decidua basalis21. A matrix stiffness of 1.1 kPa provides an initial scaffold that supports cell organisation into tissue structures, which are expected to progressively stiffen beyond the parameters defined by the starting matrix. ACH-3P cells formed significantly more organoids (15.7 ± 0.9) in the blank PEG matrix compared with those containing IKVAV and YIGSR (8.6 ± 0.7) or GFOGER (9.5 ± 0.7) peptides (Supplementary Fig. No significant difference in cell viability was observed between synthetic matrix conditions (blank 72.1% ± 3.7 vs. IKVAV and YIGSR 70.7% ± 2.5 vs. GFOGER 59.8% ± 5.7, p = 0.151; Supplementary Fig. For this reason, the blank PEG formulation was chosen for the subsequent bioprinting experiments. ACH-3P cells embedded in Matrigel or bioprinted matrix spontaneously formed organoids within 3–4 days in growth medium used for basic 2D culture (Fig. Bright-field imaging revealed that morphologically, organoids grown in Matrigel had prominent pseudopodia projecting from their outer surface and individual cells were also seen budding from organoids, both consistent with described features of EVT organoids (Fig. In comparison, bioprinted organoids did not appear to project pseudopodia into the matrix, instead they were observed invading through the matrix microenvironment, often fusing upon contact (Supplementary Movie 2). There was no significant difference in 2D area between the Matrigel and bioprinted organoids (31,355 µm2 ± 4806 vs. 27012 µm2 ± 4152, Fig. Cell viability was assessed using calcein AM (live) and ethidium homodimer-III (dead) staining, revealing a significant increase in the proportion of live cells in Matrigel compared to bioprinted organoids (Fig. However, incubation with a resazurin-based solution, Alamar Blue, revealed that both Matrigel and bioprinted organoids were metabolically active (Fig. When assessed over time, the metabolic activity and proliferation of cells in both conditions were seen to increase over time, with Matrigel organoids demonstrating a steep incline in cell activity (Supplementary Fig. Organoid-conditioned medium collected from Matrigel and bioprinted organoids tested positive for β-human chorionic gonadotropin (β-hCG) using over-the-counter pregnancy tests (Fig. a Single cells embedded in Matrigel or synthetic matrix (bioprinted) formed organoids and grew over 12 days in culture across three independent replicates. Outer cell projections and surfaces are denoted by white arrows in the inset image; scale bar = 100 µm. b Montage of whole-well; scale bar = 1 mm. c Organoid size measured at day 11 and plotted as organoid area. Unpaired, two-tailed t-test, data presented as mean ± SEM, individual data points represent means from three experimental repeats. d Whole-well images acquired and analysed using an IncuCyte imaging system; scale bar = 800 µm. Area occupied by green fluorescence (live cells) was compared to the total area occupied by cells to determine viability (%). Data presented as mean ± SEM, individual data points represent means from three experimental repeats, unpaired two-tailed t-test, **p < 0.01 (p = 0.0038). e Organoids labelled with markers for live (green) and dead cells (magenta); scale bar = 100 µm. f Day 11, Alamar blue fluorescence intensity. Unpaired two-tailed t-test, n = 3 experimental repeats, data presented as mean ± SEM. g Over-the-counter pregnancy tests incubated with organoid conditioned medium (CM). h Organoids immunolabelled for DAPI (grey), cytokeratin 7 (Cyk7, cyan hot), E-cadherin (E-cad, orange), human leucocyte antigen G (HLA-G, purple) and beta human chorionic gonadotropin (β-hCG, pink) were assessed from three independent repeats. Images further processed using the NIS Elements denoise.ai algorithm. Maximum intensity projections of 60 µm z stacks alongside a single optical section. A YZ orthogonal view and XY single optical section of syncytiotrophoblasts (STB) in β-hCG labelled organoids. i Three-dimensional visualisation of z-stacks using Bitplane Imaris rendering. Number of nuclei compared to number of cells positive for either HLA-G or β-hCG in each setting. Unpaired, two-tailed t-test, n = 3 experimental repeats containing at least 10 whole-mounted organoids each, data presented as mean ± SEM. j Histologically sectioned and processed organoids stained by haematoxylin and eosin (H&E). Internal cavities denoted by asterisks; scale bar = 50 µm. Organoids were investigated for the presence of trophoblast subtypes by whole-organoid immunofluorescence and confocal microscopy. E-cadherin, a marker of cytotrophoblasts (CTBs), was present extensively throughout the organoids grown in Matrigel and bioprinted conditions (Fig. EVTs were detected on the periphery of both Matrigel and bioprinted organoids with no significant difference in the number of human leukocyte antigen G (HLA-G)+ trophoblasts between each condition (Fig. Single β-hCG+ cells and syncytialised clusters were seen sporadically throughout the organoids, representing the STB subtype (Supplementary Movie 3). Image analysis of DAPI-labelled organoids revealed no significant difference in the number of β-hCG+ nuclei between Matrigel and bioprinted organoids (Fig. Organoid staining with haematoxylin and eosin (H&E) allowed further visualisation of the cellular organisation and architecture of organoids, and processing of whole matrix plugs provided the opportunity to view organoids in situ. Organoid sectioning revealed cavities in the inner cores of organoids (Fig. While cavities were seen in both groups, the bioprinted organoids generally appeared more compact and contained fewer cavities. Areas of syncytialisation were observed in organoids from both groups (Fig. Organoid transcriptomes were profiled to screen for trophoblast subtypes within Matrigel and bioprinted organoids at the single-cell level using the 10x Genomics Chromium platform. After quality control and normalisation, a total of 6081 cells and 17,159 genes were retained for subsequent analysis of Matrigel and bioprinted organoid samples. As previously established by the creators of the ACH-3P cell line22, these cells are devoid of vimentin, stably express Cyk7 and contain an HLA-G-expressing population of EVTs, which is consistent with our results (Fig. Uniform manifold approximation and projection (UMAP) analysis of Matrigel and bioprinted organoids was performed to further elucidate the populations of trophoblast subtypes. Through manual annotation based on published markers, the three major trophoblast cell types, including CTB, STB, and EVT, were identified (Fig. 2a): the proportion of CTB cells in each sample remained consistent, at 44% in Matrigel and 43% in bioprinted organoids. Although the EVT population made up 33% of cells in Matrigel organoids, they contributed a higher proportion at 54% in bioprinted organoids. In addition, the STB population formed 22% of cells in Matrigel organoids but only 3% of bioprinted. In further investigations, we observed two additional subclusters—a metal regulating CTB and maturing EVT population—totalling five distinct cell clusters (Fig. Single cell ordering by pseudotime trajectory analysis was performed to track the relatedness and differentiation of trophoblast subtypes, using SPINT1 expression as the origin point and root of cell differentiation (Fig. A differentiation pathway emerged from the major CTB cluster down two main arms: one extending to the EVT subtypes and another extending to the STB cluster. a Proportions of main trophoblast subtypes (CTB - cytotrophoblast, STB - syncytiotrophoblast, and EVT - extravillous trophoblasts) detected within Matrigel and bioprinted organoids. b Uniform manifold approximation and projection (UMAP) depicting an additional subcluster of CTBs and maturing EVTs. c UMAP plots of differentiation trajectory in organoids, depicted in pseudotime scale. d Dot plot representation of trophoblast lineage and subtype gene expression across the five distinguishable clusters. e UMAP plots of CTB markers, E-cadherin (CDH1) and epithelial cell adhesion molecule (EPCAM). f UMAP plot of metallothionein 2A (MT2A). h UMAP plot of ornithine decarboxylase 1 (ODC1). i UMAP plot of Epstein-Barr virus-induced gene 3 (EBI3). j and k UMAP plots of STB markers, syndecan 1 (SDC-1) and growth differentiation factor 15 (GDF15). Single-cell RNA sequencing data are deposited in the GEO under accession number GSE279994. l Area under the receiver-operating-characteristic curve (AUROC) similarity networks between cell types from Matrigel or bioprinted organoid transcriptomic datasets compared to published data from human placenta or primary trophoblast organoids. The CTB clusters were characterised by their increased expression of paternally expressed gene 10 (PEG10), integrin alpha-6/beta-1 (ITGA6) and cadherin 1 (CDH1) genes (Fig. A metal-regulating CTB subcluster (CTB_2) was distinguished by increased expression of metallothionein 2A (MT2A), a metal transporter protein (Fig. The EVT clusters progressively displayed less PEG10, ITGA6 and CDH1, with increasing expression of EVT markers, integrin subunit alpha 5 (ITGA5), HLA-G and matrix metallopeptidase 2 (MMP2) (Fig. One EVT cluster (EVT_1) displayed increased expression of proliferation marker, ornithine decarboxylase 1 (ODC1), and retained a high G2M phase profile (Fig. This suggests it may represent a progenitor population of cells transitioning from CTB to maturing EVTs. The second EVT subcluster (EVT_2) displayed increased expression of Epstein-Barr virus-induced gene 3 (EBI3), an interleukin 27 subunit shown to have roles in maternal immune tolerance during pregnancy (Fig. The EVT_2 subcluster also expressed increasing levels of integrin subunit alpha-1 (ITGA1), a marker of mature EVTs (Fig. On the other hand, the STB cluster showed increased gene expression of STB markers including chorionic gonadotropin subunit beta 3 (CGβ3), endogenous retrovirus group FRD member 1 (ERVFRD-1), SDC1, growth differentiation factor 15 (GDF15) and matrix metallopeptidase 15 (MMP15) (Fig. Differentially expressed gene (DEG) analysis revealed expression differences in the main cell clusters depending on their organoid type of origin (Supplementary Fig. To determine how confidently our Matrigel- and bioprint-derived trophoblast organoids recapitulate in vivo trophoblast biology, we compared the single-cell transcriptomes of CTB, EVT and STB in our organoids with (i) two primary placental tissue transcriptomes—the first-trimester chorionic villus (8–12-week) dataset28 and an independent 6-week placenta dataset29, and (ii) a published patient-derived trophoblast organoid scRNA-seq dataset34. MetaNeighbor analysis35, which ranks cross-dataset cell-type relatedness by the area under the receiver-operating-characteristic curve (AUROC), revealed strong concordance across all comparisons. Using an a priori threshold of AUROC > 0.88 to denote high similarity, each of CTB, EVT, and STB from our organoids formed exclusive, high-confidence links with their cognate in vivo counterparts in every reference dataset, while non-matched pairings remained below background (<0.5) (Fig. Collectively, the genetic comparisons to in vivo trophoblasts demonstrate that our ACH-3P organoid system preserves the key transcriptional identities of primary placental trophoblast lineages. This finding supports its utility as an experimental placental tissue platform that closely recapitulates the human placenta and can be used to elucidate mechanisms of placentation and as a drug screening platform for placental dysfunction disorders such as preeclampsia. Following a screening of trophoblast transcriptomes, we further analysed the expression profiles of Matrigel and bioprinted organoids at the protein level. Matrigel or bioprinted organoids were isolated from their matrices and characterised alongside 2D cultured ACH-3P cells and Matrigel matrix using liquid chromatography with tandem mass spectrometry. A total of 4984 proteins were detected across all samples and z-scores mapped (Fig. There were only 33 proteins significantly upregulated in Matrigel organoids compared to bioprinted, while 1.238 proteins were significantly upregulated in bioprinted compared to Matrigel organoids (Fig. When searched in Reactome (https://reactome.org/), this corresponded to a significant upregulation of 2 pathways in Matrigel organoids, namely the biosynthesis of ubiquinol and haem. On the other hand, 114 biological pathways (p < 0.05) were upregulated in bioprinted organoids, with many relating to mRNA processing. Organoids were digested and analysed by liquid chromatography with tandem mass spectrometry. a Heat map of mean protein z-score for 2D cells (2D), Matrigel organoids and bioprinted organoids. b Heat map of mean z-score for significantly differentially expressed proteins between Matrigel and bioprinted organoids. c Top proteins and pathways when ranked by significance for each group as determined by an unpaired, two-tailed Student's t-test. d Differences in key trophoblast subtype marker expression between Matrigel and bioprinted organoid groups. All protein mass spectrometry raw data is available in a ProteomeXchange partner repository with identifier PXD056796. Though a distinct TSC population was not noted at the transcriptome level, proteomic analysis revealed a significant increase in TSC markers such as ubiquitin-like interferon-stimulated gene 15 (ISG15), stearoyl-CoA desaturase (SCD) and sorting nexin-2 (SNX2) in bioprinted compared to Matrigel organoids (Fig. There was also a significant increase in abundance of CTB marker, CDH1, and proliferation marker, Ki67, in bioprinted organoids. Consistent with the scRNAseq data, bioprinted organoids had a greater abundance of EVT proteins, including HLA-G, ITGA5 and quiescin sulfhydryl oxidase 1 (QSOX1). Interestingly, there were no significant differences observed in the protein abundance of STB markers. Syncytin 2 (SYCY2), 3 beta- and steroid delta-isomerase 1 (3BHS1) or cytochrome P450 family 19 subfamily A member 1 (CP19A), between Matrigel and bioprinted organoids. In fact, the Matrigel expressions of these markers were highly variable between repeats. To establish a preeclampsia-like organoid model, bioprinted organoids were initially screened against varying concentrations of TNFα (1, 5, 20 ng/ml), an inflammatory cytokine that is highly expressed in women with preeclampsia (Supplementary Fig. While there was no significant difference in the number (p = 0.520), size (p = 0.244) or organoid viability (p = 0.890) between control or TNFα (1, 5, 20 ng/ml) groups (Supplementary Fig. 3a–c), and all concentrations of TNF-α reduced organoid proliferation (p < 0.05; Supplementary Fig. 3d); the concentration of C reactive protein (CRP) secreted by organoids exposed to 20 ng/ml TNFα was around 25× higher than all other groups (Supplementary Fig. 3e), demonstrating an inflammatory status. For this reason, we proceeded with 20 ng/ml TNFα. Aspirin is prescribed prophylactically to women at high risk of developing preeclampsia due to its established effects in preventing preterm preeclampsia (delivery <37 weeks)31. On the other hand, therapeutics already deemed safe for pregnancy, such as metformin, are being assessed for their repurposing capacity to treat severe preeclampsia with promising results and can ameliorate stress-induced placental cell dysfunction32,36,37. Bioprinted organoids were used for a drug screening with aspirin (0.5 mM) or metformin (0.5 mM) to investigate their ability to ameliorate inflammation-induced (TNFα 20 ng/ml) trophoblast dysfunction in the context of preeclampsia (Fig. Although there was a statistically significant difference across all the groups in the organoid number (Fig. 4b; p = 0.043), post-hoc tests showed no differences between any groups. Trophoblast organoid metabolic activity or proliferation was reduced from Day 9 onwards following TNF-α or aspirin/metformin treatment (Fig. 4d, e; p < 0.01), which was not rescued by aspirin (p < 0.001) or metformin (p < 0.01) and aligned with our previous findings in 2D culture36. Furthermore, we also determine the impact TNF-α ± aspirin/metformin treatment has on inflammatory markers, CRP and interleukin 6 (Il-6). CRP was significantly higher in organoid media containing TNF-α compared to all other conditions, which was substantially reduced in the presence of aspirin/metformin possibly suggesting that aspirin/metformin can reduce inflammation associated with trophoblast organoids (Fig. Although there were no statistically significant differences across the groups in Il-6 gene expression within trophoblast organoids, Il-6 gene expression is overall lower in the presence of TNF-α + aspirin, compared to TNF-α alone, which was not applicable to metformin (Fig. Next, we investigated the impact TNF-α ± aspirin/metformin had on ACH-3P cytotrophoblast differentiation within organoid culture (Fig. Interestingly, the number of β-hCG+ STBs was reduced in all groups (p < 0.05) except TNF-α + metformin (p = 0.07), compared to control (Fig. 4h, j), suggesting the presence of STB stress induced by TNF-α, which was not restored by aspirin/metformin at 0.5 mM concentration. On the other hand, no changes in the number of HLA-G+ EVTs were observed following treatment with aspirin/metformin and/or TNF-α (Fig. a Brightfield images of bioprinted organoids incubated with inflammatory cytokine tumour necrosis factor alpha (TNFα, 20 ng/ml) ± aspirin (0.5 mM) or metformin (0.5 mM). Images acquired using an IncuCyte. Organoids annotated with an orange outline. b Mean number of organoids counted across six bioprint units. Ordinary one-way ANOVA with Šídák's multiple comparisons post-hoc test, data presented as mean ± SEM, individual data points represent mean per bioprint unit. Ordinary one-way ANOVA with Šídák's multiple comparisons post-hoc test, data presented as mean ± SEM, individual data points represent mean per bioprint unit. d Alamar Blue fluorescence measured at 590 nm, normalised to the day 6 fluorescence intensity of each condition. Data presented as mean ± SEM, individual data points represent mean across three bioprint units. e Alamar Blue fluorescence on Day 12 normalised to the day 6 fluorescence intensity of each condition. Ordinary one-way ANOVA with Šídák's multiple comparisons post-hoc test, data presented as mean ± SEM, individual data points represent mean per bioprint unit, **p < 0.01, ***p < 0.001 (Control vs. TNFα p = 0.0026; Control vs. TNFα + Aspirin p = 0.0002; Control vs. TNFα + Metformin p = 0.0009; Control vs. Aspirin p = 0.0002; Control vs. Metformin p = 0.0002). f C-reactive protein (CRP) concentration of condition medium from 3 pooled bioprint units per condition. g Relative interleukin 6 (Il-6) mRNA expression of organoids harvested from each bioprint unit. Data presented as mean ± SEM, individual data points represent the mean per bioprint unit. Ordinary one-way ANOVA with Šídák's multiple comparisons post-hoc test. h Number of β-hCG positive nuclei divided by total number of nuclei per organoid. Ordinary one-way ANOVA with Šídák's multiple comparisons post-hoc test, data presented as mean ± SEM, data points represent individual organoids from pooled bioprint units, p*<0.05 (Control vs. TNFα p = 0.0496; Control vs. TNFα + Aspirin p = 0.012; Control vs. Aspirin p = 0.0281; Control vs. Metformin p = 0.0417). i Number of HLA-G positive nuclei divided by total number of nuclei per organoid. Ordinary one-way ANOVA with Šídák's multiple comparisons post-hoc test, data presented as mean ± SEM, data points represent individual organoids from pooled bioprint units. j Immunofluorescence images labelled with DAPI (grey), β-hCG (pink) and HLA-G (purple) acquired using a Leica Stellaris confocal microscope at ×25, scale bar = 100 µm. To reverse the polarity and organisation of trophoblast subtypes within organoids, we harvested 3-day-old organoids from their respective conditions and cultured them in suspension for up to 12 or 28 days. In the absence of a matrix to confine them, suspended organoids grew comparably larger over time and were occasionally seen fusing together (Fig. Further to this, since there was no adhesive matrix present to interact with, the organoids retrieved from both gel conditions displayed spherical morphology. Histological processing also revealed that internal cavities continued to increase in size as organoids grew (Fig. To examine the inversion of cellular organisation within the suspended organoids, another marker of STBs, syndecan-1 (SDC-1), was immunolabelled and imaged in whole organoids. Individual z slices of representative organoids are presented in Fig. SDC-1 expressing cells were observed on the periphery of organoids grown in suspension, with areas of syncytialisation depicted by arrows. Higher resolution images reveal the potential presence of a microvillus brush border on the outer edge of these syncytialised regions (Supplementary Fig. Suspended organoids were also immunolabelled for E-cadherin, HLA-G and β-hCG to confirm the presence of each trophoblast subtype (Supplementary Fig. Organoids grown in Matrigel or bioprinted conditions were harvested from the matrices after 3 days before transferring to a low-attachment plate for suspension culture. a Live cell images of gel-embedded and suspended organoids at days 12 (D12) and 28 (D28); scale bar = 100 µm. Suspension of organoids was independently repeated twice. b Images of harvested organoids cut at 5 µm thickness and stained by haematoxylin and eosin; scale bar = 100 µm. c Harvested organoids fixed, immunolabelled for syndecan-1 (SDC-1, pink) and co-stained with DAPI (grey). Confocal z-stacks of 60 µm depth were processed for visualisation using NIS Elements denoise.ai algorithm. Z-stack images of an organoid from each condition presented as maximum intensity projections (MIPs) with XY and XZ orthogonal views and optical z slices 30 µm apart. Arrows depict syncytialised areas on the outer surface of organoids. Suspension of organoids was independently repeated twice. To assess the translation of our bioprinting conditions to TSC organoid formation, we manually embedded CT29 ‘Okae' TSC line in Matrigel or bioprinted cells in the same blank 1.1 kPa matrix. Organoids formed in both conditions, but they were significantly smaller in the bioprinted matrix, even after stimulation with STB or EVT differentiation medium (STB-DM and EVT-DM, respectively; Fig. Organoids from both conditions secreted β-hCG (Fig. There was also a significant reduction in organoid formation efficiency in bioprinted conditions (Fig. Both Matrigel-embedded and bioprinted organoids could differentiate into SDC-1+ STB organoids, and HLA-G+ EVT organoids under predefined medium (Fig. There was no difference in the number and size between undifferentiated trophoblast organoid medium (TOM), STB-DM or EVT-DM treated organoids within each condition (Fig. However, within bioprinting conditions, ‘rich' matrix containing fibronectin, laminin, collagen IV peptides, laminin-511 and hyaluronic acid increased the number (p < 0.0001) and size (p < 0.05–0.001) of TSC-derived organoids compared to either subtype alone bioprinted with ‘blank' PEG matrix (Fig. Matrigel-embedded or bioprinted trophoblast stem cell (CT29) organoids supported by trophoblast organoid medium (TOM), syncytiotrophoblast differentiation medium (STB-DM) or extravillous trophoblast differentiation medium (EVT-DM). Bioprinted organoids were additionally generated in a rich PEG matrix. a Day 12 brightfield images, Nikon Ti, ×4, scale bar = 500 µm. b Day 12 brightfield images, Nikon Ti, ×20, scale bar = 100 µm. c Over-the-counter pregnancy test result of conditioned medium from STB-DM organoids in Matrigel (M) or bioprinted (B). d Organoid formation efficiency calculated as the number of cells seeded per unit/number of organoids. Unpaired, two-tailed t-test, data plotted as mean ± SEM, data points represent the mean of each bioprint unit, ****p < 0.0001. e Number of organoids and organoid area of Matrigel-embedded organoids. Ordinary one-way ANOVA with Tukey's post-hoc, data plotted as mean ± SEM, data points represent the mean of each bioprint unit. Violin plots display median ± interquartile range (IQR) in organoid size across bioprint units. f Number of organoids and organoid area of bioprinted organoids. Ordinary one-way ANOVA with Tukey's post-hoc, data points represent the mean of each bioprint unit. Violin plots display media ± IQR in organoid size across bioprint units. g Immunofluorescence images of organoids labelled for key trophoblast subtype markers. Leica Stellaris confocal microscope, 25x, scale bar = 100 µm. Early placental development is a complex and tightly regulated process that can be disrupted in various pregnancy complications. 3D cultures of cells from the maternal-foetal interface provide tools to understand the elusive mechanisms dictating placental physiology more thoroughly and screen potential treatments38. In this study, we describe the generation of trophoblast organoids from the ACH-3P first-trimester trophoblast cell line and TSCs within Matrigel or bioprinted into a synthetic, PEG-based matrix. Bioprinting offers an innovative opportunity to produce low-cost, high-throughput and reproducible organoid models with high precision and resolution. Here, we aimed to demonstrate the feasibility of bioprinting trophoblast organoids and assess the impact of ECM on trophoblast differentiation. Establishing the most appropriate matrix environment for 3D placental models is crucial, given the growing recognition of mechanical cues and cell–matrix interactions influencing stem cell differentiation, trophoblast fusion and trophoblast spheroid formation39. Alternative natural and synthetic hydrogels, such as PEG-based formulations, have been gaining attention for their tuneable composition and stiffness properties40. Based on our initial matrix selection, we demonstrated increased ACH-3P organoid formation efficiency within a naked PEG matrix containing no adhesion peptides or proteins. This condition may be due to the production of ECM components by trophoblasts, both degrading and contributing to the ECM environment they embed and invade into41. In fact, the decidua basalis, where trophoblasts embed within the uterus, has a reported stiffness of 1.25 kPa, likely due to the trophoblast deposition of ECM components including laminin, fibronectin, collagen IV, heparan sulfate, and fibrillin 121. Within both Matrigel and bioprinted PEG and basic growth medium, ACH-3P single cells gave rise to organoids that grew steadily over time with no observable differences in size. Both Matrigel and bioprinted organoids contained the three major trophoblast subtypes of CTBs, EVTs and STBs as shown by immunofluorescence labelling. This is consistent with the results of recent work comparing the organoid formation of ACH-3Ps alongside HTR-8/SVneo and Swan-71 trophoblast cell lines in Matrigel42. Despite rapid organoid formation, bioprinted cultures encountered reduced viability, likely due to shear stress applied to the cells during bioprinting, an important consideration for primary cell work. Alternatively, this result may highlight a need to further elucidate the ECM composition and stiffness requirements at different stages of organoid expansion and differentiation, as investigated in intestinal organoids43. Transcriptomic analysis confirmed the spontaneous differentiation of major trophoblast subtypes, originating from a CTB signature and directed towards either EVT or STB transcriptomic profiles. Interestingly, detailed clustering analysis revealed a distinct subpopulation of CTB in both organoids, distinguishable by the expression of MT2A, a metallothionein. The MT2A form of metallothionein is strongly expressed in the placenta and increases throughout gestation44,45. MT2A in the placenta is believed to have roles in detoxifying trace elements, protecting the developing foetus from heavy metals, clearing reactive oxygen species, trophoblast differentiation and as a regulator of labour-associated genes46,47,48,49,50. Metallothionein, and specifically MT2A, expression has previously been reported in HTR-8/SVneo, BeWo and JAR trophoblast cell lines and induced human pluripotent stem cells resembling trophoblasts45,47,51,52. Further, when Okae and colleagues were establishing their TSC lines, they performed transcriptomic comparisons of isolated primary trophoblast subtypes versus cultured TSCs stimulated to differentiate. RNAseq-based profiling revealed that MT2A had the highest expression in primary CTBs compared to other trophoblast subtypes, and in 3D TSC cultures being directed to differentiate into STBs53. Our data showed that single cells emerged from organoid structures, thereby reflecting the physiology of EVTs in vivo. Immunofluorescence labelling for HLA-G confirmed the presence of an EVT population of cells in both types of organoids, but no statistically significant difference in the proportion of these cells was observed by imaging (12.3% Matrigel vs. 11.2% bioprinted). However, single-cell transcriptomics indicated that bioprinted organoids contained a higher proportion of EVT cells (33% Matrigel vs. 54% bioprinted). This analysis also provided a greater insight into trophoblast subtypes, detecting at least two distinct populations of EVTs with varying expression of HLA-G that may have been indistinguishable by confocal fluorescence microscopy alone. Given this subcluster demonstrated greater expression of ITGA1, a marker of EVT maturity, we consider EVT_2 a more mature EVT subcluster22,54,55,56. The impact of a stiffer PEG matrix in directing EVT differentiation is corroborated by a recent study showing that trophoblast migration and invasion are increased in correlation with an increasing ECM stiffness57. Greater EVT presence was reinforced at the protein level using proteomics analysis, with bioprinted organoids showing increased abundance of several EVT markers. Concomitant to this, the activity of STBs in the organoids was first signalled by the positive result of a pregnancy test. At the morphological level, Matrigel and bioprinted organoids contained internal cavities, although bioprinted organoid cavity formation appeared less frequent. These cavities are consistent with those observed in published trophoblast organoid models and resemble the lacunae formed by early STBs where maternal blood pools for diffusion across the syncytial membrane17,18,58. Immunofluorescent quantification of β-hCG+ nuclei revealed no significant difference between Matrigel or bioprinted organoids (Matrigel 10.4% vs. bioprinted 9.2%), which is aligned with in vivo STB proportions of 9.7%29. This was supported by our proteomic analysis demonstrating no significant differences in STB protein markers between Matrigel and bioprinted organoids, although the key STB marker, hCG, was not detected. Contrarily, scRNAseq detected significantly fewer STB cells in bioprinted organoids compared to those in Matrigel. However, a caveat of the application of scRNAseq to placental tissue is that it is unable to separate individual nuclei within a shared cytoplasm, as occurs in multinucleated STBs. This limitation makes transcriptomic conclusions around trophoblast subtypes present in our organoids difficult to draw. Thus, methods such as single-nucleus RNA sequencing (snRNAseq), recently applied to STB analyses and to other multinucleated skeletal muscle myofibers, may be more suitable28,59. Comparison of our scRNAseq datasets to publicly available in vivo data from 6- and 8-week human first-trimester placental tissue and first-trimester primary trophoblast organoids demonstrated strong similarity to these tissues. In fact, the bioprinted ACH-3P organoids showed stronger associations with the 6-week placental tissue than our Matrigel-derived organoids, with the CTB subtype lacking significant similarity in Matrigel organoids compared to those from this tissue. The utility of our bioprinted trophoblast organoids to be used for drug screening of conditions caused by placental dysfunction, such as preeclampsia, was also demonstrated. We applied inflammatory stimuli, TNF-α, to mimic a preeclamptic placenta and treated organoids with clinically used treatments, aspirin and metformin. We showed that TNF-α induced a pro-inflammatory environment that impeded metabolic activity or proliferation of bioprinted trophoblast organoids, and reduced the percentage of STBs, which aspirin or metformin were unable to rescue at the concentration of 0.5 mM. STB stress plays a central role in the pathogenesis of preeclampsia, suggesting that our model of TNF-α-induced stress in bioprinted trophoblast organoids resembles a preeclamptic placenta60,61. Our previous work in 2D in vitro ACH-3P models shows the ability of aspirin and metformin to rescue stress-induced trophoblast dysfunction; however, not in the presence of TNF-α but rather hypoxia mimic or oxidative stress36. Aspirin has been shown to increase EMT transition and trophoblast invasion, likely causing a shift in proliferation towards EVT formation in these organoids62. Here, aspirin in the presence of TNF-α showed a reduction in the number of STBs, but it did not affect the EVTs abundance. Metformin has been shown to regulate trophoblast metabolism and increase expression of STB markers63. In our drug screen, metformin did not have a significant impact on the number of STBs or EVTs in the presence of TNF-α, although it reduced STBs in the absence of TNF-α. These differences with aspirin or metformin treatment in relation to the STB or EVT content of organoids could be dose- or context-dependant, as we have shown previously36. Additionally, pathway analysis of aspirin and metformin treatment in inflamed trophoblast organoids could provide further insights into the effect of these agents on placental growth and development. Overall, we demonstrated that our automated, high-throughput and precise bioprinted trophoblast organoid model can provide a myriad of useful parameters to screen potential treatments for preeclampsia and accelerate drug discovery by investigating organoid number, growth, metabolism, proliferation, viability and differentiation. This is particularly important as preeclampsia is still a condition without a cure. The same platform can be used to extract mRNA and protein from bioprinted organoids and investigate mechanisms of placentation or mechanisms of action of potential treatments. To mitigate the inside-out architecture of trophoblast organoids, we performed a suspension culture method like that applied in other organoid models64,65,66,67. Suspension of immature Matrigel or bioprinted organoids resulted in STB localisation to the outer surface, detected by expression of SDC-1. Higher magnification imaging of suspended organoids also revealed microvilli-like structures on the periphery of SDC-1+ cells, potentially reflecting the brush border of the in vivo STBs. The reversal of trophoblast organoid polarity by suspension culture is in line with a recently published study, where primary trophoblast-derived organoids were grown in suspension with gentle agitation, resulting in consistent layers of STBs on their outer surface68. Orbital rotation was also proposed to improve polarity reversal by inducing gentle shear stress that mimics early placental physiology and promotes trophoblast fusion69. Fluid flow in trophoblast organoid culture could also improve the viability of suspended organoids for long-term culture by facilitating nutrient permeation through to the core and preventing necrosis70. Finally, we tested TSC organoid formation in Matrigel compared to a bioprinted PEG matrix. It was interesting, though somewhat unsurprising, to observe reduced organoid formation and growth of TSCs in the stiffer matrix. Given that the endometrium is reduced in stiffness during the secretory phase of the menstrual cycle and decidualisation during the window of implantation, we expect that TSCs arising from this early blastocyst stage would require a softer environment for expansion71,72. Additionally, the secretory phase is marked by altered expression of ECM proteins, with the upregulation of collagen IV and V, laminin and fibrillin73,74,75, which is aligned with TSC organoids showing improved growth profile in the ‘rich' matrix in our bioprinted model. This highlights an important distinction in the ECM environment requirements of TSCs in the stem cell niche and their mature progeny, such as ACH-3Ps. Since increased placental stiffness has been observed in tissue from pregnancy complications, a system that supports screening of a multitude of matrix stiffnesses and proteins would enable further investigation into these crucial mechanisms. To extend this model, the matrix stiffness could be tuned to induce changes consistent with gestational age or pregnancy disorders, including preeclampsia, foetal growth restriction and gestational diabetes mellitus76,77,78,79,80,81,82. Organoid proliferation and cell fate may be further dictated by introducing adhesion factors required for growth and differentiation. To advance this bioprinted model of placental organoids to investigations of maternal-foetal interactions, uterine endothelial cells, stromal cells, fibroblasts or immune cells (e.g. decidual natural killer cells or macrophages) can be incorporated. Additionally, while most cell cultures are maintained at 21% oxygen, the early placenta develops within a physiologically low oxygen environment of around 2.5%83. Given that stem cell niches also often reside in 2–8% oxygen, culture in low oxygen conditions may improve the expansion of organoids and their biological relevance84. While we acknowledge that a limitation of this study is the influence of choriocarcinoma within the ACH-3P cell line, we have demonstrated at the transcriptomic level that ACH-3P bioprinted organoids closely resemble human first-trimester placental tissue. Also, we have shown the application of our approach to biofabricating TSC organoids that need further matrix tuning, perhaps using a softer matrix, in the future. This study has demonstrated that ACH-3P organoids can be used as a low-cost, low-risk, reproducible and biologically relevant model to supplement the use of primary cells that require intensive sampling and culture conditions with expensive components. Likewise, PEG matrix formulations open opportunities to control organoid formation and differentiation, with a 1.1 kPa blank matrix suitable for studies of placental EVT formation and invasion. Further, the application of bioprinting facilitates accurate orientation of low volumes of PEG components, ensuring precise cross-linking and reproducible organoid formation. This bioprinted organoid model can therefore be used to study important physiological processes, pathogenic mechanisms driving impaired EVT differentiation and invasion, and assess potential therapeutics at low cost and risk. This opens new avenues to assess the molecular pathways involved in EVT differentiation by removing the need for exogenous chemical stimuli to drive differentiation. At the same time, we have shown that suspended organoid cultures offer the opportunity to more accurately study STB physiology, including barrier effects and syncytial shedding of nuclear aggregates or extracellular vesicles for the diagnosis of pregnancy complications60,61,85,86,87,88. The use of ACH-3Ps and TSCs in this study was approved by the University of Technology Sydney Human Research Ethics Committee in accordance with the requirements of the NHMRC National Statement on Ethical Conduct in Human Research (2007). The ACH-3P first-trimester trophoblast cell line was generously donated by Professor Gernot Desoye (Graz Medical University, Austria), which was established in 200722. ACH-3Ps were immortalised by the fusion of primary first-trimester trophoblasts isolated from a 12-week gestation placenta and choriocarcinoma cell line AC1-1, resulting in a hybrid male cell line. This cell line was chosen due to its cytotrophoblast-like phenotype and demonstrated capacity to form HLA-G− and HLA-G+ subtypes, suggesting differentiation potential22,89. ACH-3P cells were cultured in Ham's F12 nutrient mix (Gibco; Thermo Fisher Scientific, cat. 11765062) supplemented with 10% foetal bovine serum (Gibco; Thermo Fisher Scientific, cat. 10099141) and 1% penicillin–streptomycin (Gibco; Thermo Fisher Scientific, cat. Every 2–5 passages, cells were treated with a selection medium containing azaserine (5.7 µM, Sigma-Aldrich) and hypoxanthine (100 µM, Sigma-Aldrich) for 24 h to prevent the overgrowth of choriocarcinoma. Cells were dissociated using Accutase (Sigma-Aldrich, cat. A6964) and experiments were performed with passages 13–15. A male TSC (“Okae”) line, CT29, was obtained from Riken (Japan). Culture surfaces were pre-coated with 0.5 µg/ml iMatrix-511 (FUJIFILM Wako, cat. CT29 cells were maintained in TSC 2D medium consisting of DMEM/F12 with 2.5 mM L-glutamine (Thermo Fisher Scientific, cat. 11320033), 0.15% bovine serum albumin (BSA, Sigma-Aldrich, cat. I3146), 1% KnockOut™ serum replacement (KSR, Thermo Fisher Scientific, cat. 10828010), 0.2 mM L-ascorbic acid (FUJIFILM Wako, cat. 1713265-25-8), 0.5% penicillin–streptomycin (P/S, Thermo Fisher Scientific), 2.5 µM Y27632 (Selleck Chemicals, cat. S6390), 25 ng/ml epidermal growth factor (EGF, FUJIFILM Irvine Scientific, cat. 100-26), 0.8 mM valproic acid (VPA, Selleck Chemicals, cat. S7692), 2 µM CHIR99021 (Selleck Chemicals, cat. Organoids were generated using passage 24. Cell cultures were incubated at 37 °C, 5% CO2 in a humidified atmosphere and were routinely tested for mycoplasma contamination. To prevent two-dimensional cell attachment, the wells of a black-walled, 96-well plate (Perkin Elmer, cat. 6055300) were precoated with 15 µL of 0.5% agarose (Sigma-Aldrich, cat. Dissociated cells were filtered through a 70 µm filter to separate aggregates before 500 ACH-3P cells were seeded in 20 µL droplets or 1200 TSC cells were deposited in 5 µL droplets using cold, growth-factor reduced Matrigel (Corning, cat. The cell-laden Matrigel droplets were incubated at 37 °C for 40 min to polymerise prior to the addition of 150 µL ACH-3P growth medium or TSC organoid medium (TOM), respectively. TOM contained DMEM/F12 with 2.5 mM L-glutamine (Thermo Fisher Scientific, #11320033), 0.15% bovine serum albumin (BSA, Sigma-Aldrich, #A8806), 1% insulin–transferrin–selenium–ethanolamine (ITS-X, Sigma-Aldrich, #I3146), 1% KnockOut™ serum replacement (KSR, Thermo Fisher Scientific, #10828010), 0.2 mM L-ascorbic acid (FUJIFILM Wako, #1713265-25-8), 0.5% penicillin–streptomycin (P/S, Thermo Fisher Scientific), 2.5 µM Y27632 (Selleck Chemicals, #S6390), 25 ng/mL epidermal growth factor (EGF, FUJIFILM Irvine Scientific, #100-26), 1 µM A83-01 (Selleck Chemicals, #S7692) and 2 µM CHIR99021 (Selleck Chemicals, #S1263). Matrigel cultures were grown for 12 days, with culture medium changed every 3–4 days in ACH-3P cultures or 2–3 days in TSC cultures. Large Plug 3D cell models (7.2 µL/well) were generated using the RASTRUM drop-on-demand bioprinter (Inventia Life Science). The wells of a black-walled, 96-well plate were initially printed with an Inert Base to prevent 2D attachment. Dissociated and filtered ACH-3P cells were primed and printed at 10,000 cells/well in a synthetic PEG-based matrix (Inventia Life Science, cat. The system uses a two-droplet process, with the PEG-4MAL bioink meeting the cells suspended in an MMP-cleavable activator that immediately polymerises as the two active bioink components make contact and crosslink90. The RASTRUM matrix formulations used were all 1.1 kPa stiffness, resembling that of the decidua basalis, the site of placental invasion. A matrix selection was performed to compare (i) a naked ‘blank' PEG containing no additional adhesion peptides, (ii) a matrix containing α- and β-laminin peptide chains (IKVAV and YIGSR), and (iii) a matrix containing collagen type I peptide mimetic (GFOGER) given the high expression of laminin and collagens in the native placenta75,91,92,93. Bioprinted TSC organoids were produced in Large Plug models in either naked PEG, as used for ACH-3P, or a ‘rich' matrix of ~1.1 kPa and containing RGD (fibronectin, laminin), YIGSR (laminin) and CNYYSNS (collagen IV) adhesion peptides with laminin-511 and hyaluronic acid (Inventia Life Science, cat. These cells were printed at a seeding concentration of 2,780,000 cells/mL which equated to a density of 10,000 cells/well. Bioprinted cultures were immediately overlaid with ACH-3P or TSC organoid growth medium and grown for 12 days, with culture medium changed every 3–4 days for ACH-3P or 2–3 days for TSCs. TSC organoids were differentiated from day 7 using either STB or EVT differentiation medium (STB-DM and EVT-DM, respectively). STB-DM contained DMEM/F12 with 2.5 mM L-glutamine (Thermo Fisher Scientific, cat. 11320033), 0.3% bovine serum albumin (BSA, Sigma-Aldrich, cat. I3146), 4% KnockOut™ serum replacement (KSR, Thermo Fisher Scientific, cat 10828010), 0.2 mM L-ascorbic acid (FUJIFILM Wako, cat. 1713265-25-8), 0.5% penicillin–streptomycin (P/S, Thermo Fisher Scientific), 2.5 µM Y27632 (Selleck Chemicals, cat. S6390), 50 ng/mL epidermal growth factor (EGF, FUJIFILM Irvine Scientific, cat. 100-26) and 2 µM forskolin (FUJIFILM Wako, cat. EVT-DM contained DMEM/F12 with 2.5 mM L-glutamine (Thermo Fisher Scientific, cat. 11320033), 0.3% bovine serum albumin (BSA, Sigma-Aldrich, cat. I3146), 4% KnockOut™ serum replacement (KSR, Thermo Fisher Scientific, cat. 10828010), 0.5% penicillin–streptomycin (P/S, Thermo Fisher Scientific), 2.5 µM Y27632 (Selleck Chemicals, cat. S6390), 7.5 µM A83-01 (Selleck Chemicals, cat. ACH-3P cells were bioprinted into a blank PEG matrix at a cell density of 8,000-14,000 cells/well. Following bioprinting, 150 μL of ACH-3P growth medium was added to each well, and the plate was incubated at 37 °C with 5% CO2. To assess the impact of TNFα concentration on bioprinted organoids, we first exposed them to varying concentrations (0, 1, 5 and 20 ng/mL) from day 5 of culture. For subsequent experiments, TNFα (20 ng/mL) was added to respective wells from day 5 of culture. Aspirin (0.5 mM) and metformin (0.5 mM) were added to respective wells from day 8 of culture. Imaging and downstream analysis were conducted as described below. After three days in matrix-embedded culture, wells were washed with phosphate-buffered saline (PBS) and organoids from both Matrigel and bioprinted conditions were retrieved from their matrices. To retrieve organoids from Matrigel, 100 µL of 1 U/mL dispase (StemCell Technologies, cat. 07913) was added to each well and incubated for 30 min at 37 °C until gel dissolved. Bioprinted organoids were retrieved from their matrix by incubating each well with 75 µL of Cell Retrieval Solution (Inventia Life Science) for 30 min at 37 °C. Matrix digestion was halted by adding complete growth medium, well contents were collected and washed twice with PBS. Organoids were resuspended in complete growth medium and transferred to an ultra-low attachment 24-well plate (Corning, United States). Organoids were maintained in suspended culture for up to either 12 or 28 days with medium changed every 3–4 days. Live cell imaging was performed using an IncuCyte organoid assay module (Sartorius, Germany) with ×10 brightfield and phase images acquired every 6 h per well. Higher resolution brightfield images were acquired with a Nikon Ti inverted microscope (Nikon, Japan) using a Nikon DS-Qi2 camera, NIS Elements Advanced Research software (v. 5.30.06), ×4 (Plan Apo λ, NA 0.2), ×10 (Plan Fluor Ph1 DLL, NA 0.3), and ×20 (Plan Apo λ ×20 Ph2 DM, NA 0.75) objectives. Organoid count was taken at day 7 in Matrigel versus bioprint experiments to account for progressive fusion of organoids. Organoid area was measured using ×10 magnification images acquired from 5 wells of each experimental replicate, totalling at least 800 organoids per matrix condition. Cell viability was assessed in Matrigel and bioprinted models using a live/dead cell viability kit for mammalian cells (Biotium, cat. Encapsulated cells were washed with PBS and incubated with calcein-AM (1 µM), ethidium homodimer 3 (EthD-III, 2 µM), and Hoechst 33342 (10 µg/mL; Invitrogen, cat. H3570) in PBS to label live cells, dead cells and nuclei, respectively. Wells were incubated with the fluorescent dyes for 30 min at 37 °C and washed twice with PBS prior to imaging. Images were acquired at ×10 magnification using the IncuCyte green and red channels for live and dead cells, respectively. Higher resolution representative fluorescent images were acquired as a z-stack of 20 slices at 1 µm step size using a Leica Stellaris confocal microscope. Images were acquired using a ×20 objective (HC PL APO CS2 ×20, NA 0.75), DAPI (ex 450 nm), FITC (ex 495 nm) and TRITC (ex 590 nm) channels and a pinhole size of 66.7 µm at 590 nm. Wells containing Matrigel or inert base only, calcein-AM only or EthD-III only were used as staining controls. DAL1100) was prepared in complete cell culture medium (10%) and added to culture wells on days 1, 4, 7 and 11 in Matrigel versus bioprinted conditions or days 6, 9 and 12 in the drug screening assay. After 2 h incubation, the metabolised Alamar Blue medium was transferred to a new plate and the fluorescent intensity at 530 nm excitation and 590 nm emission was recorded using a Tecan M Plex plate reader. At least 3 wells were measured per time point per condition. Organoid growth medium from the TNF-α concentration curve and drug screen assays was collected at day 12. Medium from the 3 wells of each condition was pooled and assessed using a CRP ELISA (Abcam, cat. Experiments were conducted according to the manufacturer's instructions. Matrigel-embedded organoids were harvested using dispase enzyme, and bioprinted organoids were collected using Cell Retrieval Solution as described above. Retrieved organoids from both conditions were washed twice with PBS to remove residual matrix prior to downstream analysis. A whole organoid immunolabelling protocol was adapted from Rios et al.95. Harvested or in situ ACH-3P organoids and in situ TSC organoids were fixed with 4% paraformaldehyde (PFA; Sigma-Aldrich, P6148) for 1 h at 4 °C and washed three times with PBS. When immunolabelling for intracellular proteins (β-hCG and Cyk-7), organoids were first permeabilised for 30 min at room temperature in a penetration buffer (PBS, 20% DMSO, 0.3 M glycine, 0.2% Triton X-100). Permeabilised organoids were washed twice with organoid washing buffer (OWB; containing PBS, 1% bovine serum albumin (BSA), 0.1% Triton X-100) for 5 min at room temperature. All organoids were incubated in blocking buffer (PBS, 1% bovine serum albumin, 3% normal goat serum, 0.1% Triton X-100) for 1 h at 4 °C by gentle rocking. Organoids were incubated with primary antibodies prepared in blocking buffer according to dilutions (Supplementary Table 1) overnight at 4 °C with gentle rocking. Primary antibodies were removed, and organoids were washed three times with OWB for 2 h at 4 °C with gentle rocking. Secondary antibodies were prepared in blocking buffer and incubated with organoids overnight at 4 °C with gentle rocking. Secondary antibodies were removed, and organoids were washed three times with OWB for 2 h at 4 °C with gentle rocking. OWB was removed and organoids were optically cleared using a fructose (2.5 M)–glycerol (60% v/v) clearing solution, incubated at room temperature for 20 min and transferred to a microscope slide. Organoids incubated with secondary antibodies only were used as labelling controls. Fluorescent images of mounted ACH-3P organoids were acquired using a Nikon A1R inverted confocal microscope with PMT and GaAsP detectors and NIS Elements Advanced Research software (v. 5.30.06). Z-stacks taken at 1 µm slices to a depth of 60 µm were acquired by Nyquist sampling for each organoid using a ×20 long working distance objective (S Plan Fluor LWD, NA 0.7). Laser excitation lines of 405, 488, 561 and 637 nm were used with a line average count of 2 and 33.2 µm pinhole diameter. Higher resolution images were also acquired with a ×100 oil immersion objective (Plan Apo λ, NA 1.45). Representative images were processed using NIS Elements denoise.ai. Fluorescent images of in situ ACH-3P or TSC organoids were acquired using a Leica Stellaris (LASX 4.8.1) inverted confocal microscope with HyD X and HyD S detectors, white light laser and a ×25 water immersion objective (NA 0.95). Z stacks were taken at 1.5 µm slices with a line accumulation count of 2 and a pinhole of 55.82 µm (1 Airy unit). Organoid viability was calculated using IncuCyte software (v2022B.rev2) by applying a mask to the areas occupied by green or red fluorescence. Viability data were presented as an area positive for calcein-AM (live)/total cell area (live + dead). Maximum intensity projections were created using ImageJ (v1.53j; https://imagej.net/software/fiji/). The proportion of cells positive for HLA-G and β-hCG was assessed using Bitplane Imaris (v9.6.0) software. At least 10 confocal z stacks were processed using Bitplane Imaris Surfaces module to quantify nuclei and respective immunofluorescent markers for each condition and experimental repeat (n = 10–92 per experimental replicate). Organoids and bioprinted matrix plugs harvested for histology were fixed in 4% PFA at room temperature for 1 h. Organoids were washed with PBS and stored in 70% ethanol overnight at 4 °C. The ethanol was removed, organoids were suspended in approximately 100 µL molten 1% low-gelling temperature agarose (Sigma-Aldrich, A4018) and transferred to a plastic surface to set as a droplet. After 20 min, the agarose droplet was transferred to a histology cassette and processed through the following for 45 min each: 80% ethanol, 90% ethanol, 100% ethanol, 100% ethanol, xylene, xylene, molten paraffin wax. The wax-perfused agarose droplets were then embedded in paraffin wax for microtome sectioning at 5 µm thickness. Dried microscope slides containing sections of organoid droplets were passed through the following reagents for 2 min each, unless otherwise stated: xylene (3 times), 100% ethanol (3 times), 95% ethanol, 70% ethanol, water, Harris Haematoxylin (5 min), water, acid alcohol (15 s), water, Scott's Bluing solution, water, 1% alcoholic Eosin (3 min), 100% ethanol (3 times), xylene (3 times). Coverslips were applied using DPX mounting medium (LabChem, AJA3197) and images acquired at ×20 (NA 0.50) using an Olympus BX51 upright microscope. To isolate RNA, harvested organoids were washed twice with PBS before extraction with 1 mL QIAzol (QIAGEN, 79306) according to the manufacturer's protocols. RNA lysates were assessed using a NanoDrop One spectrophotometer (Thermo Fisher Scientific) and gene expression quantified by RT-qPCR using a Luna Universal One-Step kit (New England Biolabs, E3005) and C1000 Touch Thermal Cycler (Thermo Fisher Scientific). Relative gene expression was quantified using the 2−ΔΔCt method. The primers were purchased from Sigma-Aldrich: β-actin (#NM_001101; forward: GACGACATGGAGAAAATCTG, reverse: ATGATCTGGGTCATCTTCTC); Il-6 (#NM_000600; forward: GCAGAAAAAGGCAAAGAATC, reverse: CTACATTTGCCGAAGAGC). For single-cell RNA sequencing (scRNA-seq), 7 wells of each condition were harvested and pooled, producing roughly 700 organoids. Organoids were washed twice with PBS and centrifuged for 5 min at 300 × g. Organoids were dissociated into single cells by adding 1 mL Accutase to each sample, followed by incubation on a shaker for 30 min at 800 rpm. The cell suspension was centrifuged for 5 min at 300 × g and washed twice with sterile 0.04% BSA in PBS, centrifuging between each step. Cells were then resuspended in 100 µL 0.04% BSA and passed through a 40 µm filter. Cell viability and count were assessed using trypan blue, ensuring viability was >80%. Single cells were prepared for sequencing according to the 10× Genomics Chromium Single Cell 3' Kit user guide at the Single Cell Technology Facility at the Faculty of Engineering and IT at UTS. Briefly, to create Gel Beads-in-emulsion (GEMs), approximately 5000 single cells from each sample were suspended in a Chromium master mix and combined with barcoded Single Cell 3' v3.1 Gel Beads and partitioning oil was loaded onto a Chromium Next GEM Chip G for partitioning. In each resulting GEM, the Gel Bead was dissolved and co-partitioned with cell lysed, incubated with reverse transcription reagents and a unique molecular identifier (UMI) to produce barcoded, full-length cDNA. Barcoded, full-length cDNA was amplified via 12 PCR cycles to generate sufficient mass for library construction. A sample index PCR was performed to generate a 3' gene expression dual index library, which was double-sided size selected. Quality control was performed using an Agilent TapeStation and analysed using TapeStation Analysis Software (v4.1.1). The libraries were sequenced by the Australian Genome Research Facility (AGRF) on an Illumina NovaSeq 6000 instrument using a NovaSeq S4 kit (200 cycles) v1 chemistry at an aimed depth of 30,000 paired-end reads per cell. The sequencer generated raw data files in binary base call (BCL) format. The BCL files were demultiplexed and converted to the FASTQ file format using Illumina Conversion Software (bcl2fastq v2.19.0.316). The 10x Genomics cellranger-v (3.1.0) count pipeline was used for alignment, filtering, barcode counting, and UMI counting from FASTQ files and was executed on a high-performance cluster with 2.6.32-754.17.1.el6.x86_64 operating system. The single-cell datasets were analysed with Seurat. Cells with at least 100 genes and <15% mitochondrial genes were kept. Genes expressed in fewer than 5 cells were discarded. The 2000 most variable genes were used in PCA calculations. Cells were analysed with 30 principal components (PCs) and clustered with 0.2 and 0.3 resolution. DEG between clusters for each compartment were calculated with FindMarkers with min.pct = 0.25 and logfc.threshold = 0.4 using the Wilcoxon algorithm. Gene set enrichment analysis of cluster-specific markers was performed with the fgsea package. Reactome or Gene Ontology gene sets were downloaded from the MSigDB and obtained from “h.all.v2023.2.Hs.entrez”. The Monocle3 R package was used to explore the differentiation of ACH-3P trophoblasts. The raw counts were loaded into Monocle3 for semi-supervised single-cell ordering in pseudo-time, using SPINT1 expression to denote the origin. Branched expression analysis modelling (BEAM) was applied to observe the expression of the CTB progenitor gene markers along pseudo-time trajectory towards either STB or EVT. To quantify how closely our organoid trophoblast lineages resemble their in vivo counterparts, we benchmarked scRNA-seq profiles from Matrigel- and bioprint-derived organoids against three reference datasets: (i) a first-trimester (8–12 weeks) chorionic-villus atlas28, (ii) a 6-week placental trophoblast dataset29, and (iii) a published patient-derived trophoblast organoid dataset34. All raw count matrices were processed in Seurat v4.4 with log-normalisation. Cells annotated as CTB, EVT or STB were subset. Organoid cells were labelled by culture platform (Matrigel, Bioprint), and public cells retained their native study identifiers at the resolution of only CTB, EVT and STB. SCE objects were merged, and MetaNeighbor35 was applied to identify the set of study-invariant, highly variable genes (variableGenes function) and to compute pairwise area-under-the-ROC (AUROC) similarity scores between every cell-type–by-study pair using the fast_version = TRUE implementation. AUROC values range from 0.5 (random overlap) to 1 (perfect concordance); we considered scores > 0.88 as evidence of strong transcriptional similarity. Results were visualised as a similarity network (igraph v1.6.0, ggraph v2.2.1) in which edges connecting two nodes (cell-type-by-study labels) were drawn only when AUROC > 0.88, and node size was scaled to the within-dataset self-AUROC. The analysis was repeated with organoid cells pooled (Matrigel + Bioprint) and with each platform analysed separately to assess platform-specific fidelity (Supplementary Fig. For proteomic analysis, 8 wells of each condition were harvested as described above and pooled, amounting to approximately 800 organoids per sample for each of three experimental repeats. Harvested organoids were washed twice with PBS, PBS was removed, and the dry organoid pellet was stored at −80 °C prior to downstream analysis. Undiluted Matrigel (20 µL) and ACH-3P cells grown in 2D were prepared as controls. Organoid pellets were thawed and solubilised with 30 µL SDC Master Mix (Supplementary Table 2) at 95 °C for 10 min. Samples were cooled at room temperature for 50 min prior to the addition of 2 µL trypsin solution and overnight incubation at 37 °C. Load Buffer was added to each sample (300 µL) and samples were centrifuged for 5 min at 15,000 × g. Stage tips were prepared using styrene divinylbenzene- reverse phase sulfonate (SDB-RPS), and 165 µL of supernatant was transferred to each tip96. Tubes with stage tips were centrifuged for 2 min at 2100 x g. Stage tips were washed with 100 µL Wash Buffer and centrifuged for 2 min at 2100 × g. Stage tips with bound peptides were transferred to new tubes containing inserts (Agilent), 50 µL fresh Elution Buffer was added to each sample and then centrifuged for 2 min at 2100 × g. Insets containing eluted peptides were evaporated in a centrifugal vacuum concentrator for 1 h. Peptides were resuspended in 25 µL MS Loading Solvent. Using an Acquity M-class nanoLC system (Waters, USA), 5 µL of the sample was loaded at 15 µL/min for 3 min onto a nanoEase Symmetry C18 trapping column (180 µm × 20 mm) before being washed onto a PicoFrit column (75 µm ID × 350 mm; New Objective, Woburn, MA) packed with SP-120-1.7-ODS-BIO resin (1.7 µm, Osaka Soda Co, Japan) heated to 45 °C. Peptides were eluted from the column and into the source of a Q Exactive Plus mass spectrometer (Thermo Scientific) using the following programme: 5–30% MS buffer B (98% Acetonitrile + 0.2% formic acid) over 90 min, 30–80% MS buffer B over 3 min, 80% MS buffer B for 2 min, 80–5% for 3 min. The eluting peptides were ionised at 2400 V. A data-dependant MS/MS (dd-MS2) experiment was performed, with a survey scan of 350–1500 Da performed at 70,000 resolution for peptides of charge state 2+ or higher with an AGC target of 3e6 and maximum Injection Time of 50 ms. The Top 12 peptides were selected and fragmented in the HCD cell using an isolation window of 1.4m/z, an AGC target of 1e5 and a maximum injection time of 100 ms. Fragments were scanned in the Orbitrap analyser at 17,500 resolution and the product ion fragment masses measured over a mass range of 50–2000 Da. The mass of the precursor peptide was then excluded for 30 s. LC/MS/MS was performed on each sample in triplicate injections. The MS/MS data files were searched using Peaks Studio 11 against the UniProt Human reference proteome database (20230323) and a database of common contaminants with the following parameter settings. Variable modifications: propionamide, oxidised methionine, deamidated asparagine. Number of allowed missed cleavages: 3. Peptide mass tolerance: 10 ppm. The results of the search were then filtered to include peptides with a –log10P score that was determined by the false discovery rate (FDR) of <1%, the score being that where decoy database search matches were <1% of the total matches. The protein.db file was exported from peaks, followed by a low-value imputation of 5000, representing a low abundance area value for any value that had a numerical value of 0. This approach was used in favour of other methods that use random values within the distribution, or values reduced by factors of the standard deviation. This dataset had a wide distribution of area values across several orders of magnitude due to large changes in protein abundance between the different experimental groups. This meant that reduction by SD or assigning random values within the distribution was inappropriate and sometimes resulted in negative values. For this reason, a representative value of 5000 was chosen as this was similar to other very low abundance proteins. Student's t-tests were used between comparable groups. A log2-fold change of <−0.4 or >0.4 with a p-value of <0.05 was considered significant and included in downstream analyses. Significantly differentially abundant proteins were searched in Reactome.org. Due to the limitations of the Reactome database and our sample type not being very well represented in the pathway analysis, a p-value of <0.05 was used as a cutoff for significantly enriched pathways instead of a more stringent FDR of 1%. Significantly enriched pathways were then further interrogated in the literature to validate that the record in the database was correct. Organoid data were statistically analysed using GraphPad Prism (v9.4.1). For normally distributed data, an unpaired, two-tailed t-test was used to compare the means of experimental replicates, where 2 groups were considered, and a p-value < 0.05 was considered statistically significant. Where more than one condition was assessed, an ordinary one-way ANOVA was applied with either Tukey's or Šídák's multiple comparisons post-hoc test. Data are presented as mean ± standard error of the mean (SEM). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The raw single-cell RNA-sequencing data for the trophoblast organoids have been deposited in the GEO under accession number GSE279994. All protein mass spectrometry raw data is available in a ProteomeXchange partner repository with identifier PXD056796. Large raw image/micrograph files that support the findings in this study are not provided with the source data and are available from the corresponding author upon request. To request the data, please contact A/Prof Lana McClements at lana.mcclements@uts.edu.au. Requests will be evaluated and responded to within 10 business days. The data will be shared from the institutional repository and available for five years from the date of publication. Source data are provided with this paper. A. Organogenesis in a dish: modeling development and disease using organoid technologies. Kleinman, H. K. & Martin, G. R. Matrigel: basement membrane matrix with biological activity. Matai, I., Kaur, G., Seyedsalehi, A., McClinton, A. & Laurencin, C. T. Progress in 3D bioprinting technology for tissue/organ regenerative engineering. Napso, T., Yong, H. E. J., Lopez-Tello, J. & Sferruzzi-Perri, A. N. The role of placental hormones in mediating maternal adaptations to support pregnancy and lactation. Villar, J. et al. Preeclampsia, gestational hypertension and intrauterine growth restriction, related or independent conditions?. Consensus definition of fetal growth restriction: a Delphi procedure. The placenta in fetal growth restriction: what is going wrong?. Mayhew, T. M., Wijesekara, J., Baker, P. N. & Ong, S. S. Morphometric evidence that villous development and fetoplacental angiogenesis are compromised by intrauterine growth restriction but not by pre-eclampsia. Metzger, B. E. et al. Hyperglycemia and adverse pregnancy outcomes. Brosens, I., Puttemans, P. & Benagiano, G. Placental bed research: I. The placental bed: from spiral arteries remodeling to the great obstetrical syndromes. Cahill, A. G., Beigi, R., Heine, R. P., Silver, R. M. & Wax, J. R. Placenta accreta spectrum. & Thornburg, K. Placental origins of chronic disease. McNestry, C., Killeen, S. L., Crowley, R. K. & McAuliffe, F. M. Pregnancy complications and later life women's health. Turco, M. Y. et al. Trophoblast organoids as a model for maternal–fetal interactions during human placentation. Haider, S. et al. Self-renewing trophoblast organoids recapitulate the developmental program of the early human placenta. Sayres, L. et al. Angiogenic function of human placental endothelial cells in severe fetal growth restriction is not rescued by individual extracellular matrix proteins. Barreto, R. et al. Mice placental ECM components may provide a three-dimensional placental microenvironment. Abbas, Y. et al. Tissue stiffness at the human maternal–fetal interface. The first trimester human trophoblast cell line ACH-3P: a novel tool to study autocrine/paracrine regulatory loops of human trophoblast subpopulations—TNF-α stimulates MMP15 expression. Zhuang, B.-M. et al. Single-cell characterization of self-renewing primary trophoblast organoids as modeling of EVT differentiation and interactions with decidual natural killer cells. Expression of interleukin-27 by human trophoblast cells. Liu, J. et al. Human placental trophoblast cells contribute to maternal–fetal tolerance through expressing IL-35 and mediating iTR35 conversion. Lombardelli, L. et al. At embryo implantation site IL-35 secreted by trophoblast, polarizing T cells towards IL-35 + IL-10 + IL-4 + Th2-type cells, could favour fetal allograft tolerance and pregnancy success. Mao, H. et al. Human placental trophoblasts express the immunosuppressive cytokine IL-35. Wang, M. et al. Single-nucleus multi-omic profiling of human placental syncytiotrophoblasts identifies cellular trajectories during pregnancy. Cell trajectory modeling identifies a primitive trophoblast state defined by BCAM enrichment. The role of TNF-α and anti-TNF-α agents during preconception, pregnancy, and breastfeeding. The hypertensive disorders of pregnancy: ISSHP classification, diagnosis & management recommendations for international practice. Neola, D. et al. Unveiling therapeutic potentials and exploring maternal-fetal health benefits of metformin in pregnancy: a scoping review. Franczyk, M., Lopucki, M., Stachowicz, N., Morawska, D. & Kankofer, M. Extracellular matrix proteins in healthy and retained placentas, comparing hemochorial and synepitheliochorial placentas. Shannon, M. J. et al. Single-cell assessment of primary and stem cell-derived human trophoblast organoids as placenta-modeling platforms. Crow, M., Paul, A., Ballouz, S., Huang, Z. J. & Gillis, J. Characterizing the replicability of cell types defined by single cell RNA-sequencing data using MetaNeighbor. Evaluating oxidative stress targeting treatments in in vitro models of placental stress relevant to preeclampsia. Cluver, C. A. et al. Use of metformin to prolong gestation in preterm pre-eclampsia: randomised, double blind, placebo controlled trial. Ghorbanpour, S. M. et al. A placenta-on-a-chip model to determine the regulation of FKBPL and galectin-3 in preeclampsia. Interplay of matrix stiffness and protein tethering in stem cell differentiation. Fully synthetic matrices for in vitro culture of primary human intestinal enteroids and endometrial organoids. & Anderson, P. Support cells and the extracellular matrix. In Stevens and Lowe's Human Histology (eds Lowe, J. S. & Anderson, P. G.) 55–70 (Mosby, 2015). Dietrich, B., Kunihs, V., Pollheimer, J., Knöfler, M. & Haider, S. 3D organoid formation and EVT differentiation of various trophoblastic cell lines. Designer matrices for intestinal stem cell and organoid culture. Furukawa, S., Usuda, K., Abe, M., Hayashi, S. & Ogawa, I. Histological expression of metallothionein in the developing rat placenta. Nikulin, S. V. et al. Impedance spectroscopy and transcriptome analysis of choriocarcinoma BeWo b30 as a model of human placenta. Differences and interactions in placental manganese and iron transfer across an in vitro model of human villous trophoblasts. Widhalm, R. et al. Human placental cell line HTR-8/SVneo accumulates cadmium by divalent metal transporters DMT1 and ZIP14. et al. Mammalian metallothionein-2A and oxidative stress. Lui, S. et al. Delineating differential regulatory signatures of the human transcriptome in the choriodecidua and myometrium at term labor. Ogushi, S., Nakanishi, T. & Kimura, T. Cadmium inhibits differentiation of human trophoblast stem cells into extravillous trophoblasts and disrupts epigenetic changes within the promoter region of the HLA-G gene. Analysis of differential gene-regulatory responses to zinc in human intestinal and placental cell lines. Tsuchida, N. et al. Transcriptomic features of trophoblast lineage cells derived from human induced pluripotent stem cells treated with BMP 4. Okae, H. et al. Derivation of human trophoblast stem cells. Karvas, R. M. et al. ITGA1 is upregulated in response to oxygen over time in a BMP4 model of trophoblast. Knöfler, M. et al. Human placenta and trophoblast development: key molecular mechanisms and model systems. Damsky, C. H., Fitzgerald, M. L. & Fisher, S. J. Distribution patterns of extracellular matrix components and adhesion receptors are intricately modulated during first trimester cytotrophoblast differentiation along the invasive pathway, in vivo. Stiff extracellular matrix promotes invasive behaviors of trophoblast cells. Establishment of trophoblast-like tissue model from human pluripotent stem cells in three-dimensional culture system. Petrany, M. J. et al. Single-nucleus RNA-seq identifies transcriptional heterogeneity in multinucleated skeletal myofibers. Awoyemi, T. et al. Preeclampsia and syncytiotrophoblast membrane extracellular vesicles (STB-EVs). Levine, L. et al. Syncytiotrophoblast extracellular microvesicle profiles in maternal circulation for noninvasive diagnosis of preeclampsia. Su, M.-T., Tsai, P.-Y., Wang, C.-Y., Tsai, H.-L. & Kuo, P.-L. Aspirin facilitates trophoblast invasion and epithelial–mesenchymal transition by regulating the miR-200–ZEB1 axis in preeclampsia. Nashif, S. K. et al. Metformin impairs trophoblast metabolism and differentiation in a dose-dependent manner. Controlling epithelial polarity: a human enteroid model for host–pathogen interactions. Fiorotto, R. et al. Cell–matrix interactions control biliary organoid polarity, architecture, and differentiation. Joo, S.-S. et al. Porcine intestinal apical-out organoid model for gut function study. Chiu, M. C. et al. Apical-out human airway organoids modeling SARS-CoV-2 infection. Yang, L., Liang, P., Yang, H. & Coyne, C. B. Trophoblast organoids with physiological polarity model placental structure and function. & Gauster, M. Go with the flow-trophoblasts in flow culture. In vivo high-resolution magnetic resonance elastography of the uterine corpus and cervix. Zambuto, S. G., Jain, I., Clancy, K. B. H., Underhill, G. H. & Harley, B. A. C. Role of extracellular matrix biomolecules on endometrial epithelial cell attachment and cytokeratin 18 expression on gelatin hydrogels. De Vriendt, S., Casares, C. M., Rocha, S. & Vankelecom, H. Matrix scaffolds for endometrium-derived organoid models. Rossi, F. et al. Extracellular matrix and pregnancy: functions and opportunities caught in the net. Kiyozumi, D., Nakano, I., Sato-Nishiuchi, R., Tanaka, S. & Sekiguchi, K. Laminin is the ECM niche for trophoblast stem cells. Spiliopoulos, M. et al. Characterizing placental stiffness using ultrasound shear-wave elastography in healthy and preeclamptic pregnancies. Evaluation of the relationship between placental stiffness measured by shear wave elastography and perinatal outcomes in women with gestational diabetes mellitus. Ma, Z., Sagrillo-Fagundes, L., Mok, S., Vaillancourt, C. & Moraes, C. Mechanobiological regulation of placental trophoblast fusion and function through extracellular matrix rigidity. Kılıç, F. et al. Shear wave elastography of placenta: in vivo quantitation of placental elasticity in preeclampsia. Cimsit, C., Yoldemir, T. & Akpinar, I. N. Shear wave elastography in placental dysfunction. Comparison of placental elasticity in normal and pre-eclamptic pregnant women by acoustic radiation force impulse elastosonography. Yuksel, M. A. et al. Shear wave elastography of the placenta in patients with gestational diabetes mellitus. Evaluation of respiratory gases and acid-base gradients in human fetal fluids and uteroplacental tissue between 7 and 16 weeks' gestation. Oxygen in stem cell biology: a critical component of the stem cell niche. Rajakumar, A. et al. Transcriptionally active syncytial aggregates in the maternal circulation may contribute to circulating soluble fms-like tyrosine kinase 1 in preeclampsia. Gebara, N., Correia, Y., Wang, K. & Bussolati, B. Angiogenic properties of placenta-derived extracellular vesicles in normal pregnancy and in preeclampsia. Associations of maternal and placental extracellular vesicle miRNA with preeclampsia. Tong, M. et al. Proteomic characterization of macro-, micro- and nano-extracellular vesicles derived from the same first trimester placenta: relevance for feto-maternal communication. Pastuschek, J. et al. Molecular characteristics of established trophoblast-derived cell lines. Du, E. Y. et al. 3D bioprintable hydrogel with tunable stiffness for exploring cells encapsulated in matrices of differing stiffnesses. Liu, M., Yin, Y., Yu, H. & Zhou, R. Laminins regulate placentation and pre-eclampsia: focus on trophoblasts and endothelial cells. Fu, Q., Tao, Y., Piao, H., Du, M.-R. & Li, D.-J. Trophoblasts and decidual stromal cells regulate decidual NK cell functions via interaction between collagen and LAIR-1. Shi, J. W. et al. Collagen at the maternal–fetal interface in human pregnancy. Role of C-reactive protein at sites of inflammation and infection. Dekkers, J. F. et al. High-resolution 3D imaging of fixed and cleared organoids. Rappsilber, J., Mann, M. & Ishihama, Y. Protocol for micro-purification, enrichment, pre-fractionation and storage of peptides for proteomics using StageTips. The authors would like to acknowledge the Single Cell Technology Facility, Faculty of Engineering and Information Technology, University of Technology Sydney. We would like to thank Holly Pearson for their scientific input and technical assistance. The authors acknowledge the use of the Nikon Ti widefield, Nikon A1R confocal, Leica Stellaris confocal and Olympus BX51 microscopes and Bitplane Imaris software at the Microbial Imaging Facility at AIMI in the Faculty of Science, the University of Technology Sydney. The authors acknowledge the Proteomics, Lipidomics and Metabolomics Core Facility for access to proteomics methodologies and instrumentation, as well as technical assistance. ), Australian Government Research Training Programme Scholarship (C.R. and Seed Funding provided by the Faculty of Science, UTS (C.R.). H.C. was supported by the UTS President's Scholarship and International Research Training Programme (IRTP) Scholarship. was supported by a Future Leader Fellowship Level 1 (106628) from the National Heart Foundation of Australia. Claire Richards, Ashley Bannister, Grace Owen, Catherine A. Gorrie, Matthew P. Padula, Kristine C. McGrath & Lana McClements Institute for Biomedical Materials and Devices, Faculty of Science, University of Technology Sydney, Sydney, NSW, Australia Claire Richards & Lana McClements Centre for Inflammation, Centenary Institute and University of Technology Sydney, Faculty of Science, School of Life Sciences, Sydney, NSW, Australia Hao Chen, Matthew O'Rourke & Philip M. Hansbro Inventia Life Science Pty Ltd, Sydney, NSW, Australia School of Biomedical Sciences and Pharmacy, University of Newcastle, Callaghan, University of Newcastle, Callaghan, Newcastle, NSW, Australia Centre for Drug Repurposing and Medicines Research Program, Hunter Medical Research Institute, New Lambton Heights, Newcastle, NSW, Australia Single Cell Genomics Facility, School of Biomedical Engineering, Faculty of Engineering and Information Technology, University of Technology Sydney, Sydney, NSW, Australia Garvan Institute of Medical Research, The Kinghorn Cancer Centre, Darlinghurst, NSW, Australia School of Clinical Medicine, Faculty of Medicine, University of New South Wales Sydney, Kensington, NSW, Australia Microbial Imaging Facility at the Australian Institute for Microbial Imaging, Faculty of Science, University of Technology Sydney, Sydney, NSW, Australia Amy L. Bottomley & Louise Cole Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualised, optimised, and carried out the experiments, analysed and interpreted the data and wrote the manuscript. contributed to proteomics experiments, data analysis and interpretation. performed experiments and data analysis. contributed to experimental design and data interpretation. provided histology training, assisted in experimental design and optimisation. contributed to experimental design, technical support and data acquisition through the Single Cell Technology Facility. provided significant input to experimental design and optimisation, image acquisition, image analysis and interpretation. conceptualised the study, acquired funding, contributed to data analysis and interpretation and manuscript writing. was an employee of Inventia Life Science at the time of the study. The remaining authors declare no competing interests. Nature Communications thanks Alexander Beristain, and the other, anonymous, reviewer for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Richards, C., Chen, H., O'Rourke, M. et al. Matrix directs trophoblast differentiation in a bioprinted organoid model of early placental development. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250911073158.htm'>Black holes just proved Stephen Hawking right with the clearest signal yet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 06:35:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>On September 14, 2015, a signal arrived on Earth, carrying information about a pair of remote black holes that had spiraled together and merged. The signal had traveled about 1.3 billion years to reach us at the speed of light -- but it was not made of light. It was a different kind of signal: a quivering of space-time called gravitational waves, first predicted by Albert Einstein 100 years prior. Light waves, such as X-rays, optical, radio, and other wavelengths of light, as well as high-energy particles called cosmic rays and neutrinos had been captured before, but this was the first time researchers had witnessed a cosmic event through its gravitational warping of space-time. For this achievement, first dreamed up more than 40 years prior, three of the LIGO founders won the 2017 Nobel Prize in Physics: MIT's Rainer Weiss, professor of physics, emeritus (who recently passed away at age 92); Caltech's Barry Barish; and Caltech's Kip Thorne. Together, the gravitational-wave-hunting network, known as LVK (LIGO, Virgo, KAGRA), has captured a total of more than 300 black hole mergers, most of which are already confirmed while others await further analysis. The space-time distortions induced by gravitational waves are incredibly minuscule. To sense them, LIGO and Virgo must detect changes in space-time smaller than 1/10,000 the width of a proton. That's 700 trillion times smaller than the width of a human hair. The event was not that different from the first-ever detection (called GW150914) -- both involve colliding black holes about 1.3 billion light-years away with masses between 30 to 40 times that of our Sun. But thanks to 10 years of technological advances reducing instrumental noise, the GW250114 signal is dramatically clearer. "We can hear it loud and clear, and that lets us test the fundamental laws of physics," says LIGO team member Katerina Chatziioannou, Caltech assistant professor of physics and William H. Hurt Scholar, and one of the leading authors of a new study on GW250114 published in the Physical Review Letters. When black holes merge, their masses combine, increasing the surface area. Additionally, the merger can cause the combined black hole to increase its spin, which leads to it having a smaller area. Later, Hawking and physicist Jacob Bekenstein concluded that a black hole's area is proportional to its entropy, or degree of disorder. In essence, the detection (made just by LIGO, since Virgo was undergoing routine maintenance and KAGRA was offline during this particular observation) allowed the team to "hear" two black holes growing as they merged into one, verifying Hawking's theorem. Kip Thorne recalls Hawking phoning him to ask whether LIGO might be able to test his theorem immediately after he learned of the 2015 gravitational-wave detection. Hawking died in 2018 and sadly did not live to see his theory observationally verified. The surface areas of pre-merger black holes can be more readily gleaned as the pair spiral together, roiling space-time and producing gravitational waves. During this so-called ringdown phase, the final black hole vibrates like a struck bell. The modes are like characteristic sounds a bell would make when struck; they have somewhat similar frequencies but die out at different rates, which makes them hard to identify. "Analyzing strain data from the detectors to detect transient astrophysical signals, send out alerts to trigger follow-up observations from telescopes or publish physics results gathering information from up to hundreds of events is quite a long journey -- adds Nicolas Arnaud, CNRS researcher in France and Virgo coordinator of the fourth science run -- Out of the many skilled steps that such a complex framework requires, I see the humans behind all these data, in particular those who are on duty at any time, watching over our instruments. There are LVK scientists in all regions, pursuing a common goal: literally, the Sun never goes down above our collaborations!" LIGO and Virgo have also unveiled neutron stars over the past decade. The "multi-messenger" astronomy event marked the first time that both light and gravitational waves had been captured in a single cosmic event. "The global LVK network is essential to gravitational-wave astronomy," says Gianluca Gemme, Virgo spokesperson and director of research at INFN (Istituto Nazionale di Fisica Nucleare). "With three or more detectors operating in unison, we can pinpoint cosmic events with greater accuracy, extract richer astrophysical information, and enable rapid alerts for multi-messenger follow-up. Other LVK scientific discoveries include the first detection of collisions between one neutron star and one black hole; asymmetrical mergers, in which one black hole is significantly more massive than its partner neutron star; the discovery of the lightest black holes known, challenging the idea that there is a "mass gap" between neutron stars and black holes; and the most massive black hole merger seen yet with a merged mass of 225 solar masses. In the coming years, the scientists of LVK hope to further fine tune their machines, expanding their reach deeper and deeper into space. Looking farther into the future, scientists are working on a concept for even larger detectors.The European project, called Einstein Telescope, plans to build one or two huge underground interferometers with arms of more than 10 kilometers, The US one, called Cosmic Explorer, would be similar to the current LIGO but with arms 40 kilometers long. "This is an amazing time for gravitational wave research: thanks to instruments such as Virgo, LIGO and KAGRA, we can explore a dark universe that was previously completely inaccessible. We are already preparing a new generation of detectors such as the Einstein Telescope in Europe and Cosmic Explorer in the US, as well as the LISA space interferometer, which will take us even further into space and back in time. Note: Content may be edited for style and length. 90% Chance: Physicists Predict a Black Hole Could Explode This Decade This Simple Blood Test Might Catch Alzheimer's Before It Steals Your Memory Overworked Brain Cells May Hold the Key to Parkinson's Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250911073145.htm'>Black hole explosion could change everything we know about the Universe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-12 02:40:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Physicists have long believed that black holes explode at the end of their lives, and that such explosions happen -- at most -- only once every 100,000 years. But new research published in Physical Review Letters by physicists at the University of Massachusetts Amherst has found a more than 90% probability that one of these black-hole explosions might be seen within the decade, and that, if we are prepared, our current fleet of space and earthbound telescopes could witness the event. This catalog would finally answer one of humankind's oldest questions: from where did everything in existence come? We know that black holes exist, and we have a good understanding of their life cycle: an old, large star runs out of fuel, implodes in a massively powerful supernova and leaves behind an area of spacetime with such intense gravity that nothing, not even light, can escape. But, as physicist Stephen Hawking pointed out in 1970, another kind of black hole -- a primordial black hole (PBH), could be created not by the collapse of a star but from the universe's primordial conditions shortly after the Big Bang. However, despite their density, PBHs could be much lighter than the black holes we have so far observed. Furthermore, Hawking also showed that black holes have a temperature and could, in theory, slowly emit particles via what is now known as "Hawking radiation" if they got hot enough. As PBHs evaporate, they become ever lighter, and so hotter, emitting even more radiation in a runaway process until explosion. Yet, while we should be able to, no one has ever directly observed a PBH. "We know how to observe this Hawking radiation," says Joaquim Iguaz Juan, a postdoctoral researcher in physics at UMass Amherst. Though physicists since Hawking's time have thought that the chances of seeing an exploding PBH are infinitesimally slight, Iguaz Juan notes that "our job as physicists is to question the received assumptions, to ask better questions and come up with more precise hypotheses." "We believe that there is up to a 90% chance of witnessing an exploding PBH in the next 10 years," says Aidan Symons, one of the paper's co-authors and a graduate student in physics at UMass Amherst. Standard black holes have no charge, and it was assumed that PBHs are likewise electrically neutral. "We make a different assumption," says Michael Baker, co-author and an assistant professor of physics at UMass Amherst. "We show that if a primordial black hole is formed with a small dark electric charge, then the toy model predicts that it should be temporarily stabilized before finally exploding." Taking all known experimental data into account, they find that we could then potentially observe a PBH explosion not once every 100,000 years as previously thought, but once every 10 years. Iguaz Juan adds, "this would be the first-ever direct observation of both Hawking radiation and a PBH. We would also get a definitive record of every particle that makes up everything in the universe. 90% Chance: Physicists Predict a Black Hole Could Explode This Decade This Simple Blood Test Might Catch Alzheimer's Before It Steals Your Memory Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            