
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01029-8'>Exclusive: Trump White House directs NIH to study ‘regret' after transgender people transition</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 21:26:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You can also search for this author in PubMed Google Scholar As the US National Institutes of Health (NIH) continues to defund nearly every research project on transgender health, the White House has directed the agency to focus on studying “regret” after a person transitions to align their body with their gender identity. ‘Mind-boggling': US CDC orders gender-related terms cut from scientific papers ‘Mind-boggling': US CDC orders gender-related terms cut from scientific papers Many scientists, reeling from the abrupt cancellation of more than US$180 million in NIH funding for research on transgender health, slammed the proposed studies as ideologically driven. “It's really pigeonholing trans people into this medical lens where the only thing important to know about them is that they seek medical transition” and regret it, says Harry Barbee, who studies the health of people from gender and sexual minorities (LGBT+) at the Johns Hopkins Bloomberg School of Public Health in Baltimore, Maryland, and identifies as non-binary and queer. Estimates suggest that 1.6 million people in the United States identify as transgender, and about one-quarter of them obtain gender-affirming surgeries. Research suggests1 that access to these procedures can reduce anxiety, depression and suicidal ideation in transgender people. Exclusive: NIH to cut grants for COVID research, documents reveal Exclusive: NIH to cut grants for COVID research, documents reveal US President Donald Trump signed an executive order on his first day in office on 20 January that directed the US government to stop acknowledging that gender can differ from sex at birth. The e-mail from Memoli, obtained by Nature, specifies two areas of research that the Trump administration wishes to fund about “social transition”, which is when a person changes how they express their gender to others — for example, by changing their name or appearance. More research on the experience of trans people — including understanding reasons for dissatisfaction after transition — is direly needed, Barbee says. But they worry that the sole focus on negative consequences is misplaced: fewer than 1% of transgender people who undergo gender-affirming surgery regret it, according to an analysis published by Barbee and their colleagues2. So far, about 187 NIH grants funding trans-health research have been terminated, according to an online effort to track the Trump team's research cuts by Brittany Charlton, an epidemiologist at the Harvard T. H. Chan School of Public Health in Boston, Massachusetts, who studies LGBT+ health, and her colleagues. If these proposed studies move forwards, it will create a “a distorted research ecosystem where only politically favourable findings are permitted to exist” and an “evidence vacuum for clinicians who are trying to do right by their patients”, Barbee says. ‘One of the darkest days': NIH purges agency leadership amid mass layoffs ‘Mind-boggling': US CDC orders gender-related terms cut from scientific papers How Europe aims to woo US scientists and protect academic freedom A brain drain would impoverish the United States and diminish world science Superpowers want to control critical mineral supplies — local communities need a stronger say Job Title: Chief Editor, Nature Biomedical Engineering Locations: New York, Beijing or Shanghai (Hybrid Working Model) Application Deadline: April ... ‘One of the darkest days': NIH purges agency leadership amid mass layoffs ‘Mind-boggling': US CDC orders gender-related terms cut from scientific papers An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08951-x'>Strategic atom replacement enables regiocontrol in pyrazole alkylation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 15:41:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Please note there may be errors present which affect the content, and all legal disclaimers apply. Pyrazoles are heterocycles commonly found as key substructures in agrochemicals and medicinally active compounds alike.1,2 Despite their pervasiveness, established methods fall notably short in delivering complex pyrazoles selectively due to issues of differentiation during either assembly or N-functionalization.3 This is a direct consequence of a dominant synthetic strategy that attempts to control selectivity-determining bonds between poorly differentiated starting materials. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Alexander Fanourakis, Yahia Ali, Liao Chen, Patrick Q. Kelly, Abigail J. Bracken & Mark D. Levin Fanourakis, A., Ali, Y., Chen, L. et al. Strategic atom replacement enables regiocontrol in pyrazole alkylation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/why-some-people-follow-authoritarian-leaders-and-the-key-to-stopping-it/'>Why Some People Follow Authoritarian Leaders—And The Key to Stopping It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 12:01:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Why Some People Follow Authoritarian Leaders—And The Key to Stopping It From media pundits to conversations held in coffee shops, people are talking about authoritarian leaders. President Trump and his warning of being a dictator “on day one,” coupled with his attempts to consolidate power, eliminate government oversight and silence his opposition, poses a grave threat to our democratic institutions. Without downplaying the dangers of authoritarian leaders, studies from my research group and other labs from across the globe identify an equally serious threat to democracy: “authoritarian followers” who instinctively comply with a dictator. For over 80 years, political psychologists like me have studied the authoritarian personality—a collection of attitudes and behaviors that increase a person's susceptibility to authoritarian leaders. We have found that authoritarian followers share three tendencies: they obey authority figures from their in-group (called authoritarian submission); they punish rule breakers (authoritarian aggression); and they rigidly endorse long-held traditions (conventionalism). If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Work from my lab and others reveals that authoritarian followers express a range of anti-democratic attitudes including anti-gay prejudice, anti-immigrant attitudes, generalized prejudice, nationalism and even the belief in conspiracies. Although peer-reviewed work on 2024 Trump voters awaits, authoritarian followers were more likely to vote for Trump than for either Hillary Clinton or Joe Biden in the previous two elections. Thus authoritarian followers are a powerful force within the MAGA movement. Crucially, the social, economic and physical environment also matter. Unfortunately for our democratic institutions, placing trust in a dictator and becoming an authoritarian follower is one way to reestablish a sense of control. Movements including Brexit, the rise of the Alternative for Germany (AfD), Marine Le Pen's successes in France, and Donald Trump's return to power in 2024 all gained momentum by stoking voters' fears that our way of life is under threat. Attacks against transgender rights and DEI initiatives are similarly rooted in perceived threats to traditional values. What can people do to address the recent global surge of authoritarian leaders and followers? Research on this topic shows that increases in education and political knowledge may make authoritarian followers more, not less, likely to express anti-democratic attitudes. For example, a study in the U.S. found that the relationship between authoritarianism and two core features of conservatism are stronger among those who are knowledgeable about politics. Alternatively, pro-democracy advocates could employ “jiujitsu” persuasion to target the motivations underlying anti-democratic beliefs. For example, because authoritarianism arises from the need to mitigate perceived threats in the environment, we can expose people to safer ones. Indeed a newly published study by my colleagues and me shows that the diversity of one's neighborhood correlates negatively with authoritarianism. In turn these experiences dispel worries that immigrants threaten deeply held cultural values or will take their jobs. For example, research from Singapore—where authorities endorse multiculturalism—shows that authoritarian followers support cultural diversity. Other work from Poland indicates that authoritarian followers support prohibiting hate speech against minority groups including members of the LGBTQ community, Muslims and people of African descent, because this hostile rhetoric violates social norms. These studies show that, under some limited conditions, leaders can harness authoritarian followers' destructive impulses for the social good. Although Trump's return to power has reignited popular interest in authoritarianism, social scientists have long wrestled with its origins. Their insights can help us predict what the next four years will look like, as well as identify ways to address the challenges that are likely to come. Democracy rarely falls at the hands of a single individual. Rather it dies through the complacency and obedience of otherwise well-intentioned authoritarian followers. This is an opinion and analysis article, and the views expressed by the author or authors are not necessarily those of Scientific American. He is also the lead editor of the Cambridge Handbook of Political Psychology (2022) and has published over 175 peer-reviewed manuscripts on topics relevant to political psychology.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02199-6'>Precise synthesis of advanced polyarylamines for efficient perovskite solar cells</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 10:16:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Although being highly demanded in organic electronics, functional conjugated polymers face challenges on scalable synthesis with batch uniformities. Here a reactivity-regulated sequent cross-coupling carbon–nitrogen polycondensation method is developed to enable the precise synthesis of functional polyarylamines with excellent batch-to-batch uniformity. It is revealed that the stepwise regulation of intermediate reactivities is key to accomplish controllable polycondensation via two sequent palladium-promoted carbon–nitrogen coupling cycles, which is distinct to the unicyclic carbon–carbon coupling. A variety of polyarylamines are prepared to improve the material functionalities, where a ternary polymer consisting of polar substituents is shown to optimize the interfacial and bulk properties of perovskite layers fabricated on top. The corresponding inverted perovskite solar cells achieved remarkable power conversion efficiencies of 25.2% (active area, 5.97 mm2) and 23.2% (active area, 128 mm2), along with decent operational stabilities. Overall, this work provides an effective polymerization method for advanced conjugated polymers to enable high-performance optoelectronics. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The main data supporting the findings of this study are available within the Article and its Supplementary Information. Swager, T. M. 50th anniversary perspective: conducting/semiconducting conjugated polymers. A personal perspective on the past and the future. Ding, L. et al. Polymer semiconductors: synthesis, processing, and applications. Google Scholar Yu, Z.-P., Yan, K., Ullah, W., Chen, H. & Li, C.-Z. Conjugated polymers for photon-to-electron and photon-to-fuel conversions. Google Scholar Sun, F. et al. Soft fiber electronics based on semiconducting polymer. Google Scholar An, C. & Hou, J. Benzo[1,2-b:4,5-b′]dithiophene-based conjugated polymers for highly efficient organic photovoltaics. Google Scholar Freudenberg, J., Jansch, D., Hinkel, F. & Bunz, U. H. F. Immobilization strategies for organic semiconducting conjugated polymers. Google Scholar Fei, C. et al. Strong-bonding hole-transport layers reduce ultraviolet degradation of perovskite solar cells. Google Scholar Li, B. et al. Highly efficient and scalable p-i-n perovskite solar cells enabled by poly-metallocene interfaces. Google Scholar Li, H. et al. 2D/3D heterojunction engineering at the buried interface towards high-performance inverted methylammonium-free perovskite solar cells. Google Scholar Liang, Z. et al. Homogenizing out-of-plane cation composition in perovskite solar cells. Google Scholar Fei, C. et al. Lead-chelating hole-transport layers for efficient and stable perovskite minimodules. Google Scholar Luo, D. et al. Enhanced photovoltage for inverted planar heterojunction perovskite solar cells. Google Scholar Huang, Y. et al. Finite perovskite hierarchical structures via ligand confinement leading to efficient inverted perovskite solar cells. Google Scholar Sun, Y. et al. Bright and stable perovskite light-emitting diodes in the near-infrared range. Google Scholar & Tan, Z.-K. Large-area near-infrared perovskite light-emitting diodes. Google Scholar & Zhu, K. Rapid advances enabling high-performance inverted perovskite solar cells. Google Scholar Veres, J., Ogier, S. D., Leeming, S. W., Cupertino, D. C. & Khaffaf, S. M. Low‐k insulators as the choice of dielectrics in organic field‐effect transistors. Google Scholar Mathijssen, S. G. J. et al. Dynamics of threshold voltage shifts in organic and amorphous silicon field-effect transistors. Google Scholar Zhang, W. et al. Systematic improvement in charge carrier mobility of air stable triarylamine copolymers. Google Scholar Wang, Y. et al. PTAA as efficient hole transport materials in perovskite solar cells: a review. Google Scholar Xu, X. et al. Improving contact and passivation of buried interface for high‐efficiency and large‐area inverted perovskite solar cells. Google Scholar Chen, R. et al. Robust hole transport material with interface anchors enhances the efficiency and stability of inverted formamidinium–cesium perovskite solar cells with a certified efficiency of 22.3%. Google Scholar Wu, X. et al. Backbone engineering enables highly efficient polymer hole‐transporting materials for inverted perovskite solar cells. Google Scholar Miyaura, N. & Suzuki, A. Palladium-catalyzed cross-coupling reactions of organoboron compounds. Google Scholar Green chemistry for organic solar cells. Google Scholar Xiong, H. et al. General room-temperature Suzuki–Miyaura polymerization for organic electronics. Google Scholar Yamamoto, T., Hayashi, Y. A novel type of polycondensation utilizing transition metal-catalyzed C–C coupling. Preparation of thermostable polyphenylene type polymers. Google Scholar Kim, Y. et al. Sequentially fluorinated PTAA polymers for enhancing Voc of high-performance perovskite solar cells. Google Scholar Kim, Y. et al. Methoxy-functionalized triarylamine-based hole-transporting polymers for highly efficient and stable perovskite solar cells. ACS Energy Lett. Google Scholar Lim, C. et al. Oligo(ethylene glycol)-incorporated hole transporting polymers for efficient and stable inverted perovskite solar cells. Google Scholar Wang, K.-L. et al. High-performance organic photorefractive materials containing 2-ethylhexyl plasticized poly(triarylamine). Google Scholar Uehling, M. R., King, R. P., Krska, S. W., Cernak, T. & Buchwald, S. L. Pharmaceutical diversification via palladium oxidative addition complexes. Google Scholar Arora, R. & Lautens, M. Photoexcited nickel-catalyzed carbohalogenation. Google Scholar Yokoyama, A. et al. Chain-growth polymerization for the synthesis of polyfluorene via Suzuki-Miyaura coupling reaction from an externally added initiator unit. Xu, C., Dong, J., He, C., Yun, J. Precise control of conjugated polymer synthesis from step-growth polymerization to iterative synthesis. Lee, S. M., Park, K. H., Jung, S., Park, H. & Yang, C. Stepwise heating in Stille polycondensation toward no batch-to-batch variations in polymer solar cell performance. McCann, S. D., Reichert, E. C., Arrechea, P. L. & Buchwald, S. L. Development of an aryl amination catalyst with broad scope guided by consideration of catalyst stability. This research was funded by the National Natural Science Foundation of China (no. 22125901), the National Key Research and Development Program of China (no. 2019YFA0705900), the Fundamental Research Funds for the Central Universities (no. 226-2024-00005) and the Scientific Research Project of China Three Gorges Corporation (no. We thank P. Qian and B. Shi from the Department of Chemistry, Zhejiang University, for technical support on scale synthesis. These authors contributed equally: Ziqiu Shen, Yanchun Huang. State Key Laboratory of Silicon and Advanced Semiconductor Materials, Department of Polymer Science and Engineering, Zhejiang University, Hangzhou, People's Republic of China Ziqiu Shen, Yanchun Huang, Yuan Dong, Kangrong Yan, Hongzhen Chen & Chang-Zhi Li You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar developed the concept. designed the experiments. and H.C. supervised the project. conducted the synthesis experiments. performed the PSC fabrication and characterization. carried out the film measurements and analysis. prepared the manuscript. All authors provided feedback and commented on the manuscript. Correspondence to Chang-Zhi Li. The authors declare no competing interests. Nature Materials thanks Mohammad Khaja Nazeeruddin, Yanlin Song and Atsushi Wakamiya for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Shen, Z., Huang, Y., Dong, Y. et al. Precise synthesis of advanced polyarylamines for efficient perovskite solar cells. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Materials (Nat. © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02612-0'>Scalable spatial transcriptomics through computational array reconstruction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 09:40:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Spatial transcriptomics enables gene expression mapping within tissues but is often limited by imaging constraints. We present an imaging-free approach that reconstructs spatial barcode locations using molecular diffusion and dimensionality reduction. Validated against ground truth imaging, our method achieves high fidelity and scales to centimeter-sized tissues. This approach enhances spatial transcriptomics' accessibility and throughput, enabling large-scale studies without specialized imaging equipment. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout All data is available at the Broad Single Cell Portal: https://singlecell.broadinstitute.org/single_cell/study/SCP2577. Raw sequencing data is available on SRA: https://www.ncbi.nlm.nih.gov/sra/PRJNA1221542 (ref. All code is available at http://github.com/Chenlei-Hu/Slide_recon.git (ref. Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S. & Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. Shah, S., Lubeck, E., Zhou, W. & Cai, L. In situ transcription profiling of single cells reveals spatial organization of cells in the mouse hippocampus. Ståhl, P. L. et al. Visualization and analysis of gene expression in tissue sections by spatial transcriptomics. Wang, X. et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states. Rodriques, S. G. et al. Slide-seq: a scalable technology for measuring genome-wide expression at high spatial resolution. Moses, L. & Pachter, L. Museum of spatial transcriptomics. Liu, Y. et al. High-spatial-resolution multi-omics sequencing via deterministic barcoding in tissue. Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. A remark on global positioning from local distances. Novembre, J. et al. Genes mirror geography within Europe. Glaser, J. I., Zamft, B. M., Church, G. M. & Kording, K. P. Puzzle imaging: using large-scale dimensionality reduction algorithms for localization. A., Ellington, A. D. & Marcotte, E. M. Bringing microscopy-by-sequencing into view. & Zhang, F. DNA microscopy: optics-free spatio-genetic imaging by a stand-alone chemical reaction. & Qian, N. Volumetric imaging of an intact organism by a distributed molecular network. Preprint at bioRxiv https://doi.org/10.1101/2023.08.11.553025 (2023). Hoffecker, I. T., Yang, Y., Bernardinelli, G., Orponen, P. & Högberg, B. A computational framework for DNA sequencing microscopy. Greenstreet, L. et al. DNA-GPS: A theoretical framework for optics-free spatial genomics and synthesis of current methods. McInnes, L., Healy, J. & Melville, J. UMAP: Uniform Manifold Approximation and Projection for dimension reduction. Preprint at arXiv https://doi.org/10.48550/arxiv.1802.03426 (2018). Stickels, R. R. et al. Highly sensitive spatial transcriptomics at near-cellular resolution with Slide-seqV2. Cable, D. M. et al. Robust decomposition of cell type mixtures in spatial transcriptomics. Russell, A. J. C. et al. Slide-tags enables single-nucleus barcoding for multimodal spatial genomics. Yu, T. et al. Differentially expressed transcripts from phenotypically identified olfactory sensory neurons. Cell type-specific inference of differential expression in spatial transcriptomics. Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. Kajihara, T. et al. Non-rigid registration of serial section images by blending transforms for 3D reconstruction. Lee, B. C., Tward, D. J., Mitra, P. P. & Miller, M. I. On variational solutions for whole brain serial-section histology using a Sobolev prior in the computational anatomy random orbit model. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Nolet, C. J. et al. Bringing UMAP closer to the speed of light with GPU acceleration. Palla, G. et al. Squidpy: a scalable framework for spatial omics analysis. Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Fleming, S. J. et al. Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender. Qiu, C. et al. A single-cell time-lapse of mouse prenatal development from gastrula to birth. Hu, C. et al. Scalable spatial transcriptomics through computational array reconstruction. Hu, C. et al. Scalable spatial transcriptomics through computational array reconstruction. We thank D. Sun, R. Raichur and S. Alakwe for assistance with tissue handling and library preparation; D. Cable for the initial computational implementation of the idea; and K. Cao, J. Zhang, M. Dai, X. Ye and P. Yadollahpour for discussion on the computational algorithm. We thank L. Gaffney for helping with schematics and illustrations. This work was supported by the National Institutes of Health (grant R01HG010647 to F.C. and a subaward from NHGRI TDCC (U24HG011735 to F.C. also acknowledges support from the Searle Scholars Award, the Burroughs Wellcome Fund CASI award and the Merkin Institute. This research was supported by the New York Stem Cell Foundation. is a New York Stem Cell Foundation – Robertson Investigator. Broad Institute of Harvard and MIT, Cambridge, MA, USA Chenlei Hu, Mehdi Borji, Giovanni J. Marrero, Vipin Kumar, Jackson A. Weir, Sachin V. Kammula, Evan Z. Macosko & Fei Chen Department of Molecular and Cellular Biology, Harvard University, Cambridge, MA, USA Biological and Biomedical Sciences Program, Harvard University, Cambridge, MA, USA Department of Psychiatry, Massachusetts General Hospital, Boston, MA, USA Department of Stem Cell and Regenerative Biology, Harvard University, Cambridge, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar developed the protocol and performed experiments with help from G.J.M., V.S. wrote the reconstruction script. synthesized the barcoded beads. wrote the paper, with contributions from all authors. Correspondence to Fei Chen. is an academic founder of Curio Biosciences and Doppler Biosciences and scientific advisor for Amber Bio. 's interests were reviewed and managed by the Broad Institute in accordance with their conflict-of-interest policies. are listed as inventors on a patent application related to the work. is an academic founder of Curio Biosciences. The remaining authors declare no competing interests. Nature Biotechnology thanks Song Chen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Simulated locations of capture beads and diffusible beads in a 3 mm circle. b, Simulated diffusion pattern of a capture bead on its associated diffusible beads, colored by simulated UMI counts. The distribution plots on the top and right represents the diffusion distribution on the x and y axis respectively. c, Simulated locations of capture beads, colored by a two dimensional color gradient depending on the locations. d, UMAP reconstructed locations of capture beads, colored the same as in c. e, Absolute error of capture beads plotted in ground truth locations. f, Displacement vectors of capture beads. Each arrow starts from the capture bead's ground truth location and ends at the reconstruction location. g, Histogram plot of capture beads' absolute error. a, Absolute error of capture beads plotted in ground truth locations. b, Displacement vectors of capture beads. Each arrow starts from the capture bead's ground truth location and ends at the reconstruction location. c, Histogram plot of capture beads' absolute errors. d, Spatial location of capture beads in ground truth, colored by decomposed cell types. e, Relative RMS error of measurement lengths as a function of measurement length. Data shown in Fig. 1f (blue) and two biological replicates (orange and green) are presented. Solid lines represent average values across beads and shaded areas represent one standard deviation. f, CA1 width measured in ground truth and reconstruction (N = 3 biological replicates). Data shown in Fig. 1f (blue) and two biological replicates (orange and green) are shown. Gray lines showed the mean width of each group. g, Neighborhood enrichment analysis between cell-type pairs in reconstruction (left) and ground truth (right). The enrichment scores are plotted in the same color scale, higher scores represent more enriched in the neighborhoods. h, Barcode matching between Slide-seq library and in situ sequencing barcode list or reconstruction barcode list. Bead barcodes with >20 UMI counts were matched with hamming distance ≤1. Blue rectangle represents total barcodes from in situ sequencing and dark blue represents barcodes matched with Slide-seq library barcodes (shown as green rectangle). Yellow rectangle represents total barcodes from reconstruction and dark yellow represents barcodes matched with Slide-seq library barcodes. i, Violin plot of UMI count per bead with the same Slide-seq library matched to reconstruction results and in situ sequencing results. Scale bars: 500 µm. a, Absolute error of capture beads plotted in ground truth locations. b, Displacement vectors of capture beads. Each arrow starts from the capture bead's ground truth location and ends at the reconstruction location. c, Histogram plot of capture bead locations' absolute errors. d, Spatial representation of reconstruction error on each nucleus. e, Displacement vectors of located nuclei. Each arrow starts from the nuclei's ground truth location and ends at the reconstruction location. f, Histogram plot of nuclei locations' absolute errors. f, RMS error of measurement lengths between bead pairs as a function of measurement length. Solid lines represent average values, and shaded areas represent one standard deviation. C-SIDE differential expression genes. Reconstruction cost and production time. UMAP epochs during reconstruction of P1 Mouse Embryo. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions et al. Scalable spatial transcriptomics through computational array reconstruction. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Biotechnology (Nat Biotechnol) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03602-0'>Immune and metabolic effects of African heritage diets versus Western diets in men: a randomized controlled trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 09:09:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. African heritage diets are increasingly being replaced by Western-style dietary patterns because of urbanization, economic development, increased access to processed foods, globalization and changing social norms. The health consequences of this nutrition transition are not well understood. We conducted a randomized controlled trial in the Kilimanjaro region in Northern Tanzania to investigate the immune and metabolic effects of switching between Kilimanjaro heritage-style and Western-style diets for 2 weeks and consuming a traditional fermented banana beverage (‘Mbege') for 1 week. Seventy-seven young and healthy volunteers assigned male at birth, some living in urban areas and some living in rural areas, were recruited in the trial. The switch from heritage-style to Western-style diet affected different metabolic pathways associated with noncommunicable diseases and promoted a pro-inflammatory state with impaired whole-blood cytokine responses to microbial stimulation. In contrast, the switch from Western-style to heritage-style diet or consuming the fermented beverage had a largely anti-inflammatory effect. These findings show the metabolic and immune effects of dietary transitions and the consumption of fermented beverages, underscoring the importance of preserving indigenous dietary practices to mitigate noncommunicable disease risk factors in sub-Saharan Africa. Sub-Saharan Africa faces a surge in noncommunicable diseases (NCDs), including cardiovascular disease (CVD), diabetes and chronic inflammatory conditions1, straining health systems. Urban adult mortality rates now exceed rural rates2, largely because of lifestyle changes, including the adoption of calorie-dense Western diets (WDs). Africa has a rich diversity of heritage diets, often rich in vegetables, fruits, legumes, grains and fermented products. Plant-based diets and fermented foods are increasingly recognized for their beneficial immune and metabolic effects3,4,5. In contrast, Western-style diets, high in calorie-dense and processed foods, drive metabolic dysfunction, systemic inflammation and long-term immune cell reprogramming, a phenomenon called ‘maladaptive trained immunity'6,7, which are key drivers of NCDs8. African populations are severely underrepresented in nutritional and immunological research, with limited data available on the immune and metabolic impact of African heritage diets (AHDs) compared to well-studied diets such as the Mediterranean or Japanese diet. In a previous cross-sectional study of healthy Tanzanian individuals from the Kilimanjaro region, we found that dietary patterns were associated with cytokine responses, plasma proteome and coagulation markers9,10,11. Urban residents, with greater exposure to WD environments, exhibited a stronger pro-inflammatory profile than those living in rural areas. Most rural dwellers in the study were from the Chagga tribe, whose heritage diet is predominantly plant-based, rich in fiber and polyphenols, and includes ‘Mbege', a fermented banana beverage made with finger millet12. Fermented foods, rich in bioactive microbes, have garnered renewed interest for their health benefits, yet they are becoming increasingly scarce in modern diets because of industrial food processing13,14. Cross-sectional observational studies are limited in establishing causal relationships and controlling confounders. Therefore, we performed an open-label, randomized controlled trial to evaluate the short-term and long-term immunometabolic effects of a short dietary intervention in young Tanzanian men from rural and urban backgrounds. Seventy-seven healthy adult men (median age = 25.6 years; interquartile range (IQR) = 21–27.2) participated in an open-label, short dietary intervention with 4-week follow-up to assess its longer-term immunometabolic effects. Participants were recruited from 22 April 2021 to 3 July 2021 with the final sampling on 9 August 2021. Baseline characteristics are summarized in Extended Data Table 1. Participants with a body mass index (BMI) of 18–25.9 were eligible for inclusion; however, four participants with a BMI > 26 were inadvertently included. Dietary habits were assessed using three nonconsecutive 24-h recalls, including a festival or weekend day, and are summarized in Supplementary Table 1. 1a, consisted of three arms: (1) men living in rural areas (n = 23) who habitually consumed a Kilimanjaro heritage-style diet and switched to a Western-style diet for 2 weeks; (2) men living in urban areas (n = 22) who habitually consumed a Western-style diet and switched to a heritage-style diet for 2 weeks; and (3) men who habitually consumed a Western-style diet (n = 22) and who consumed Mbege, a traditional fermented banana beverage, for 1 week. Five participants who maintained their habitual diets were included in the first two arms as controls to assess diet-independent effects (Supplementary Tables 2 and 3). Blood samples were collected at three time points: baseline; after the intervention (immediately after the 2-week dietary intervention or 1-week fermented beverage intervention); and at the follow-up (4 weeks after the intervention). After one participant withdrew from the study before the end of the intervention, the first group included young rural-living men (n = 22) who entered the intervention, whose habitual diet mainly consisted of a Kilimanjaro AHD and who changed for 2 weeks to a WD (top). The second group consisted of urban-living young men (n = 22) whose habitual diet mainly consisted of a WD, and who were switched for 2 weeks to a plant-based, fiber-rich heritage-style diet (middle). In both groups, five controls who remained on their habitual diet were also enrolled. The third group consisted of young men (n = 22) whose diet remained unchanged from their habitual WD, but who were supplemented with consumption of a locally brewed fermented banana beverage (Mbege), approximately 1 l per day for 1 week. Below each group, the number of participants sampled at each of the time points (baseline, t0; after the intervention, t1; and at the follow-up, t2), are presented in green, orange and purple filled squares, respectively. b, In all study arms, samples for hemocytometry, whole-blood cytokine responses and RNA sequencing (RNA-seq), as well as plasma proteome and metabolome, were collected at three time points: baseline (t0); on completion of the intervention (after the intervention) (t1) (at week 2 for the dietary intervention and week 1 for the banana beverage intervention); and 4 weeks after completion of the intervention (t2) (‘follow-up'). c, Images of example foods and the fermented beverage provided in the study arms. In the group of participants switching from AHD to WD, one participant withdrew before the start of the intervention and was excluded from the analyses, while two participants in this arm and one control lacked follow-up samples (CONSORT diagram in Extended Data Fig. Outcomes were assessed based on sample availability and quality control (Fig. Meals were provided three times daily in the Uru Shimbwe Juu village (AHD to WD switch) or Moshi town (WD to AHD switch). Examples and a detailed description of switched diets are shown in Fig. Weight monitoring showed a notable increase in the group who switched to a WD (median = 2.6 kg, IQR = 2.0–3.4 kg, P < 0.001; Mann–Whitney U-test). To assess the impact of the intervention on the variance in plasma proteome, cytokine production, plasma metabolome and whole-blood transcriptome, which are primary outcomes of the trial, we conducted variance partition analysis, using the intervention time points as the primary variable, with age, BMI and physical activity as covariates, and participant ID as a random effect (Extended Data Fig. Each intervention induced distinct immune and metabolic adaptations both in the short (after the intervention versus baseline) and long (follow-up versus baseline) term. Switching from an AHD to a WD accounted for over 5% of variance in plasma proteome (25.9%), cytokine production (40%), plasma metabolome (41.3%) and whole-blood transcriptome (19.1%) features, respectively. These effects largely persisted at the follow-up except for plasma proteome variance, which reduced to 9.9%. Conversely, switching from a WD to an AHD mainly influenced plasma proteome and metabolome, explaining over 5% of variance in 27% and 42% of features, respectively. Fermented beverage consumption explained over 5% of variance in plasma proteome (21%), cytokine production (23.3%), plasma metabolome (35.6%) and whole-blood transcriptome (14.5%) features, with sustained variation primarily in cytokine production and whole-blood transcriptome. Next, we analyzed changes in circulating proteins related to inflammation and metabolism, a prespecified primary outcome, using the 92-plex Olink inflammatory and cardiometabolic panels (proximity extension assays, Olink Proteomics AB)15. Relative protein concentrations were analyzed for 162 proteins (89 cardiometabolic, 73 inflammatory; Supplementary Table 6) after excluding proteins with values below the detection limits in more than 25% of samples. Differentially abundant proteins (DAPs) were identified using a linear mixed model (LMM) analysis and annotated to biological processes according to the Human Protein Atlas (v.23.0). Analysis focused on proteins that the time point variable explained over 5% of the variance (Methods and Supplementary Table 6). Principal component analysis (PCA) of participant-corrected values revealed distinct shifts, which were more pronounced in participants switching from a WD to an AHD (Extended Data Fig. Participants switching from an AHD to a WD exhibited a significant increase in 26 proteins (19 cardiometabolic and seven inflammation panel) and a decrease in one protein after the intervention compared to baseline (Fig. Among the most upregulated proteins were CNDP1, THBS4 (implicated in atherogenesis16), ANGPTL3 and TWEAK (Fig. Key biological processes linked to DAPs included ‘cell adhesion' (for example, THBS2, NID1, CDH1), ‘hydrolase activity' (for example, CNDP1, PAM), ‘apoptosis' (for example, TWEAK, TRAIL, TGFB1) and ‘protease activity' (for example, ANGPTL3, uPA, F7) (Extended Data Fig. 3b), suggesting early pathological adaptations to a WD, characterized by systemic inflammation and metabolic dysregulation. Targeted plasma proteomics was conducted using the inflammatory and cardiometabolic Olink panel. Plasma CRP concentrations were measured using ELISA. Samples were obtained at baseline (t0), after the intervention (t1, week 2 for the dietary intervention and 1 week for the fermented beverage intervention) and 4 weeks later (t2, follow-up). Statistical analysis was carried out for participants who completed the full study, that is n = 20, 22 and 22 in the WD, AHD and fermented beverage intervention groups, respectively. Statistical analysis was carried out using an LMM for each comparison with a two-sided hypothesis test. Fixed effects included time point, baseline BMI, age and physical activity level; participant ID was included as a random effect. a, Triangle plots present DAPs in the (I) WD, (II) AHD and (III) fermented beverage intervention groups. Protein concentrations are presented as Normalized Protein eXpression (NPX) values for Olink measures (switch to WD (b), switch to AHD (c) and fermented beverage (d)) or log2-transformed CRP levels (mg ml−1, e). In contrast, participants switching from a WD to an AHD exhibited a significant reduction in 28 proteins (22 inflammation, 6 cardiometabolic) and an increase in 6 proteins after the intervention compared to baseline (Fig. These DAPs were linked to ‘cytokines,' ‘inflammatory response,' ‘chemotaxis' and ‘host–virus interactions' (Extended Data Fig. At the follow-up, the changes had partially reversed; however, several proteins, including chemokines (CXCL1, CXCL5, CXCL6, CXCL8, CXCL10, CXCL11, CCL4, CCL5, MCP2, MCP4), cytokines (IL-6, OSM, IL-17C, TNFSF14) and others (CASP8, MMP1, CA1, CA3, STAMBP, ST1A1, 4E-BP), remained significantly decreased compared to baseline. In the control group, most DAPs remained unchanged, except for reduced values of CA1 (follow-up versus after the intervention) and CCL5 (follow-up versus baseline) (Extended Data Fig. Consumption of the fermented beverage resulted in a more diverse pattern of protein changes, with plasma concentrations of 29 proteins (27 cardiometabolic, two inflammatory) increasing and three inflammatory proteins decreasing after the intervention compared to baseline (Fig. At the follow-up time point, cardiometabolic proteins returned to baseline but inflammatory markers, including CC chemokines (CCL5, CCL4, CCL28), MMP1, IL-17A and OSM, remained lower than baseline. This is consistent with Wastyk et al.5, who observed reductions in inflammatory markers with a highly fermented food diet. In the control group, most DAPs remained unchanged, except for a decrease in CCL5 (follow-up versus baseline; Extended Data Fig. To confirm the inflammatory effects, plasma C-reactive protein (CRP) concentrations were measured using a high-sensitivity enzyme-linked immunosorbent assay (ELISA). Using an LMM analysis as for the Olink data, a significant increase in plasma CRP concentrations was observed in participants switching to a WD (after the intervention versus baseline, P = 0.020), while a trend toward reduced CRP concentrations was observed in participants switching to an AHD (follow-up versus after the intervention, P = 0.059), with no significant changes in the fermented beverage group. Overall, switching to a Kilimanjaro heritage-style diet resulted in sustained reductions in inflammatory and metabolic plasma proteins. Conversely, switching to a WD increased metabolic proteins and, to a lesser extent, inflammatory proteins, with minimal lasting effects. Fermented beverage consumption induced a rapid shift in cardiometabolic proteins and a sustained reduction in inflammatory proteins. Subsequently, we analyzed the effect of the interventions on cytokine production capacity, a prespecified primary outcome. Whole blood was incubated with Candida albicans, Escherichia coli lipopolysaccharide (LPS), Mycobacterium tuberculosis (MTB), Salmonella enteritidis, Staphylococcus aureus and the TLR3 agonist polyinosinic:polycytidylic acid (poly(I:C)) for 48 h. Cytokines (tumor necrosis factor (TNF), interleukin-6 (IL-6), IL-1β, IL-10, interferon-y (IFNγ)) were measured in supernatants using ELISAs, producing 30 cytokine response measures. LMM analysis revealed that cytokine responses in participants switching from an AHD to a WD declined in 12 and 14 measures after the intervention and at the follow-up, compared to baseline, respectively, while four measures increased after the intervention compared to baseline. Despite elevated plasma inflammatory proteins (Fig. 2a), TNF and IFNγ responses to all stimuli, except poly(I:C), decreased after the intervention (Fig. 4a and Supplementary Table 8), which is consistent with reduced cytokine production commonly observed in inflammatory conditions such as aging17, obesity18, endotoxemia19 or sepsis20. The largest decline was observed for C. albicans responses, especially in IFNγ, but also in other cytokines (Fig. IL-1β and IFNγ responses to poly(I:C) initially increased after the intervention, but this effect was not sustained at the follow-up (Fig. IL-6 responses to LPS, S. aureus, C. albicans, MTB and S. enteritidis decreased at the follow-up compared to baseline (Fig. Whole blood collected at baseline (t0), after the intervention (t1: week 2 for the dietary intervention group and week 1 for the banana beverage intervention group) and 4 weeks later (t2, follow-up), was subjected to 48-h ex vivo stimulation using the following stimuli: C. albicans hyphae (1 × 107 ml−1), E. coli LPS (10 ng ml−1), MTB (5 µg ml−1), S. enteritidis (1 × 106 ml−1), S. aureus (1 × 106 ml−1) and the TLR3 agonist poly(I:C) (50 µg ml−1). Cytokine concentrations were measured in the supernatants at three time points for participants in the WD (n = 21), AHD (n = 22) and fermented beverage (n = 22) study arms using ELISA. Statistical analysis was carried out using an LMM for each comparison with a two-sided hypothesis test. Fixed effects included time point, baseline BMI, age and physical activity level, while participant ID was included as a random effect. a,c,e, Arrow plots depicting statistical comparisons between time points for switch to WD (a), switch to AHD (c) and fermented beverage (e): after the intervention (t1) versus baseline (t0), follow-up (t2) versus after the intervention (t1), and follow-up (t2) versus baseline (t0). b,d,f, Combined box and line plots with the group trend line indicating concentrations of measured parameters at the three time points (switch to WD (b), switch to AHD (d) and fermented beverage (f)). Cytokine production is presented as the log2 concentrations of IFNγ, IL-10, IL-1β, IL-6 and TNF in the supernatant. Conversely, switching from a WD to an AHD resulted in only few changes (Fig. At the follow-up, TNF, IL-1β, and IFNγ responses to S. aureus and TNF responses to S. enteritidis were decreased compared to both baseline and after the intervention. In contrast, IL-1β responses to MTB showed an increase at the follow-up compared to baseline and after the intervention (Fig. In contrast, the transitioning to a WD led to a marked decline in C. albicans responses. In summary, the interventions had notable effects on cytokine production. Switching to a WD reduced TNF and IFNγ production, while the fermented beverage increased anti-inflammatory IL-10 responses and enhanced C. albicans cytokine responses. Diet influences plasma metabolome composition21, with immunomodulatory effects from food-derived and microbiome-derived metabolites. We analyzed the plasma metabolome, a prespecified primary outcome, using untargeted metabolomics, identifying 1,266 ions and 1,339 molecular formulas, with 935 metabolites annotated in the Human Metabolome Database (HMDB) (Methods and Supplementary Table 9). PCA revealed altered metabolic profiles after the intervention across all arms compared to baseline (Extended Data Fig. An LMM analysis was applied to detect differentially abundant metabolites (DAMs) (absolute fold change greater than 1.2, Padj < 0.05), focusing on metabolites where time points accounted for more than 5% of the variance (Supplementary Table 10). Metabolites were classified into 14 chemical groups; pathway enrichment analysis was conducted using the Relational Database of Metabolomics Pathways (RaMP) (https://rampdb.nih.gov/), focusing on WikiPathways terms (1,377 pathways, 3,966 metabolites and 14,191 metabolite–pathway associations) to identify metabolites within or influencing specific pathways (Supplementary Table 11)22. Transitioning from the AHD to WD resulted in 309 DAMs. Top DAMs and their chemical classes are shown in Extended Data Fig. Pathway enrichment analysis revealed increases in metabolites linked to ‘glucose homeostasis', the urea cycle (for example, ‘biomarkers for urea cycle disorders'), ‘tRNA aminoacylation' and ‘tryptophan catabolism' after the intervention (Fig. 4a and Supplementary Table 11), presumably reflecting the higher animal protein content in the WD, also indicated by increases in metabolites as 1-methylhistidine. Conversely, metabolites linked to the ‘flavan-3-ol' pathway, a flavonoid subclass, decreased. The omega-3 polyunsaturated fatty acid eicosapentaenoic acid (EPA) significantly decreased after the intervention compared to the baseline, while docosahexaenoic acid (DHA) showed a trend toward decreasing (P = 0.09), with both increasing again at the follow-up (Fig. Arachidonic acid (ARA) and its pro-inflammatory metabolite leukotriene B4 transiently decreased after the intervention and significantly increased at the follow-up, alongside a rise in 20-carboxy leukotriene B4 after the intervention (Fig. Lastly, dopamine and dopamine sulfate transiently decreased after the intervention, which is consistent with studies linking high-fat diets to hypodopaminergic effects23. a, For each dietary arm and time comparison, the top bar plots present the number of DAMs at each time point. An LMM was used for each comparison with a two-sided hypothesis test. Fixed effects included time point, baseline BMI, age and baseline physical activity level; participant ID was included as a random effect. Significance level was set to Benjamini–Hochberg-adjusted P ≤ 0.05 and absolute paired fold change greater than 1.2. The lower dot plots present the top eight significantly enriched WikiPathway metabolic pathways for DAMs using a Fisher's exact test via the RaMP database. b,d,f, Heatmaps showing examples of the enriched pathways discussed in the main text for switch to WD (b), switch to AHD (d) and fermented beverage (f), with the fill color indicating the relevant log2 paired fold change. c,e,g, Respective associated DAMs for switch to WD (c), switch to AHD (e) and fermented beverage (g) are shown in combined box and line plots with the group trend line indicating concentrations of measured parameters at the three time points. Switching from a WD to an AHD resulted in 334 DAMs. Among these, 93 metabolites increased and 41 decreased after the intervention, while 74 increased and 24 decreased after the intervention compared to the baseline (Fig. Additional DAMs that increased included benzoic acid, trans-piceid (a resveratrol glycoside with cardiovascular benefits)25 and fumaric acid (a tricarboxylic acid cycle intermediate and immunomodulator)26. In the controls, with the exception of benzoic acid, these metabolites were not differentially abundant or were only observed as differentially abundant at the follow-up compared to the baseline (Supplementary Fig. Among the top DAMs that decreased after the intervention were cresol, p-cresol sulfate and indoxyl sulfate, which returned to baseline levels at the follow-up. The latter two metabolites, produced via tyrosine and tryptophan fermentation by gut bacteria, are known to decrease with vegetarian diets27. Pathway enrichment analysis linked increased DAMs, such as the omega-3 polyunsaturated fatty acids docosahexaenoic acid (DHA), docosapentaenoic acid (DPA) and EPA, as well as arachidonic acid, dodecanoid acid and fumaric acid, to ‘omega-9 fatty acid synthesis', ‘omega-3/omega-6 fatty acid synthesis', ‘neurotransmitter release cycle' and ‘incretin synthesis, secretion and inactivation' pathways, with varying roles depending on the metabolite (Fig. These metabolites, formed by gut microbial breakdown of dietary polyphenols, have been associated with vascular health and anti-inflammatory effects28. The sustained decrease in cysteinylglycine (a glutathione breakdown product) and transient increase in tyrosol (a microbial metabolite of polyphenols), and organic sulfates like 4-ethylphenyl sulfate, may highlight microbial fermentation effects (Fig. In summary, the dietary switch and fermented beverage both had distinct effects on metabolic pathways related to glucose, lipid, fatty acid and amino acid metabolism, and pathways associated with inflammation. We also analyzed changes in the prespecified primary outcome of whole-blood transcriptome. We performed differential expression analysis (Padj < 0.2, no fold change cutoff), considering both the individual and control group of each intervention arm (Methods). Fermented beverage consumption resulted in 26 upregulated and 104 downregulated DEGs after the intervention compared to the baseline and 15 upregulated and 58 downregulated genes at the follow-up compared to the baseline (Fig. Whole-blood transcriptome data were analyzed for participants in the AHD to WD switch group (n = 22), the WD to AHD switch group (n = 22) and the fermented beverage group (n = 21). Five control individuals for each dietary arm were included in the differential expression analysis, using DESeq2. Differential expression analysis was performed between time points using the Benjamini–Hochberg method for P value adjustment and independent hypothesis weighting (IHW). A two-sided significance threshold was set at adjusted P value ≤ 0.2, with no fold change cutoff. a, A scheme presenting the number of DEGs for each comparison in each of the intervention arms are presented. b,e, Dot plots presenting the results of gene set overrepresentation analyses for GO terms performed with the DEGs from each comparison for the switch to WD (b) and fermented beverage (e). c,d,f,g, Tile plots presenting the log2 fold change of the genes related to selected enriched GO terms for switch to WD (after intervention (t1) versus baseline (t0) (c) and follow-up (t2) versus baseline (t0) (d)), and the fermented beverage (after intervention (t1) versus baseline (t0) (f) and follow-up (t2) versus baseline (t0) (g)). In participants switching to a WD, upregulated genes after the intervention were enriched for Gene Ontology (GO) terms related to reactive oxygen species (ROS), including ‘response to ROS', ‘cellular response to hydrogen peroxide' and ‘positive regulation of ROS production', and pathogen defense terms such as ‘response to fungus' and ‘defense response to bacterium' (Fig. In contrast, downregulated DEGs at the follow-up compared to the baseline were related to adaptive immunity, showing enrichment in processes such as ‘immunoglobulin production', ‘positive regulation of T cell activation' and ‘MHC class II protein complex assembly', including several human leukocyte antigen (HLA) genes (Fig. Upregulated genes at the follow-up compared to the baseline were associated with posttranscriptional modification terms, such as ‘production of miRNA involved in gene silencing' and ‘histone modification' (Fig. In the fermented beverage group, downregulated genes after the intervention compared to the baseline were linked to cellular migration processes, including ‘positive regulation of cell migration' and ‘regulation of cell development', including genes such as ACTN4, STAT3, DIAPH1, NOTCH1 and NOTCH2, LRP1, JAK1 and SEMA4D (Fig. Transcription factor enrichment analysis identified STAT3, STAT5B and SON DNA binding protein (an intracellular infection mediator in macrophages)31, and ELF4 (a known regulator of IFN induction)32, as key regulators of downregulated DEGs (Extended Data Fig. Moreover, aryl hydrocarbon receptor expression encodes a transcription factor responsive to metabolites such as indole-lactic acid and from fermented foods; its targets were increased (Extended Data Fig. Comparing the follow-up with the baseline, a decrease in genes related to inflammation progression and resolution pathways was observed. This included pathways such as ‘cell activation involved in immune response', ‘response to wounding', ‘acute phase response' and ‘positive regulation of cellular component movement', with key genes such as IL6R, DOCK2, DOCK5, DOCK8, DIAPH1 and LRP1 (Fig. In summary, these results support that a WD is associated with upregulation in genes of the innate immune system, whereas the fermented banana beverage induced a more anti-inflammatory transcriptional profile. Possible adverse events were monitored through spontaneous reporting because the risks of the short-term dietary interventions were considered minimal. Participants were instructed to report any adverse events to the study team, but none occurred. We conducted a post-hoc analysis to assess the intervention's impact on circulating leukocyte subsets and activation status. Using a Sysmex XN-450 hematology analyzer with its extended inflammation parameters, we measured complete blood counts and activation parameters for lymphocytes (reactive lymphocytes (RE-LYMPHs)), monocytes (reactive monocytes (RE-MONOs)) and neutrophils (neutrophil reactivity intensity (NEUT-RI) and neutrophil granularity intensity (NEUT-GI))34. LMM analysis revealed significant leukocyte changes, predominantly in participants switching to a WD, with smaller effects in participants consuming the fermented beverage (Fig. Switching to a WD increased white blood cells, neutrophils, immature granulocytes, NEUT-RI, monocytes, RE-MONOs and RE-LYMPHs from baseline to after the intervention (Fig. While white blood cell counts returned to baseline levels at the follow-up, activation markers (NEUT-RI, NEUT-GI, RE-MONOs, RE-LYMPHs) remained elevated, suggesting persistent immune cell activation. Conversely, switching to an AHD showed no significant leukocyte changes. The fermented beverage reduced NEUT-RI after the intervention, with effects persisting at the follow-up, alongside lower immature granulocytes and higher basophil percentages (Fig. No significant changes were observed in the control participants maintained on their habitual diet (Supplementary Table 18). a, Triangle plot presenting the significantly different leukocyte parameters across the baseline (t0), after the intervention (t1, 2 weeks for diets and 1 week for fermented beverage) and at the follow-up (t2, 4 weeks after t1) time points (n = 22 from each diet). Parameters were measured using the XN-450 hematology analyzer (Sysmex). An LMM was used for each comparison, with a two-sided hypothesis test. Fixed effects included time point, baseline BMI, age and baseline physical activity level, while participant ID was included as a random effect. b,c, Combined violin, box and line plots depicting the values of the measured parameters (switch to WD (b) and fermented beverage (c)) across the three time points with participant-specific lines linking observations. NEUT-RI and NEUT-GI values are expressed in fluorescence intensity (FI) units and scatter intensity (SI) units, respectively. Overall, the intervention substantially influenced leukocyte numbers and phenotype, with the WD promoting myelopoiesis and immune cell activation, while the fermented beverage reduced neutrophil activation. To validate the findings, we analyzed data from the 300 Tanzania functional genomics (TZFG) cohort, a cross-sectional study of urban- or rural-living Tanzanians (n = 295) from the same region. Participants were previously categorized into ‘Kilimanjaro heritage-style diet' (n = 138) and ‘Western-style diet' clusters (n = 157) based on food-derived plasma metabolome profiles (Extended Data Fig. Differential analysis using the Olink inflammation panel identified 18 proteins that were lower in the heritage-style diet cluster compared to the Western-style diet cluster (Supplementary Table 19). Among these, 83% (15 proteins) also decreased in the current WD to AHD arm (Extended Data Fig. These proteins included the chemokines CXCL1, CXCL5, CXCL6, CXCL11, MCP2 and MCP4. AHDs are rapidly being replaced by WDs across the continent. In a randomized controlled trial, we found that switching to a WD impacted key metabolic pathways linked to NCDs and induced a pro-inflammatory state involving leukocytes, inflammatory mediators and gene expression. In contrast, the switch to a largely plant-based heritage diet typical for the Kilimanjaro region and the fermented beverage primarily induced anti-inflammatory effects. Some of these changes persisted for 4 weeks after the intervention, demonstrating a sustained impact. This study highlights the immunometabolic consequences of dietary transitions and suggests their potential effects beyond a short dietary intervention. The heritage diet in this study is rich in green vegetables, legumes, plantains, root and tuber crops, including cassava and taro, and whole grains (millet, sorghum), providing dietary fiber and bioactive compounds like polyphenols, known for their anti-inflammatory properties and beneficial metabolic effects3,35,36,37,38,39. Additionally, this diet promotes a favorable gut microbiome composition and production of beneficial metabolites40. Our plasma proteomics and metabolomics analyses revealed changes in several classes of immunomodulating compounds and metabolic pathways, including microbiota-derived metabolites. Notably, in the AHD group, inflammatory proteins decreased without changes in circulating leukocytes or gene transcription, suggesting that its anti-inflammatory effects stem from modulation of metabolic processes in tissues and possibly the microbiome, rather than leukocyte transcriptional regulation. The immunometabolic benefits of this diet probably stem from synergistic interactions among diverse foods and bioactive compounds, rather than any single component, a pattern also seen with the Mediterranean diet41. The WD, rich in calorie-dense processed foods, saturated fats, salt and refined carbohydrates, promotes inflammation. This probably results from both direct immunometabolic effects of cholesterol, saturated lipids and sugars, and indirect effects via gut dysbiosis and weight gain6,42. Our study also highlights the potential health benefits of traditional African fermented beverages43. The Chagga tribe's heritage beer, Mbege, is prepared by fermenting boiled bananas with germinated finger millet porridge (Eleusine coracana). Besides the health-promoting compounds naturally present in bananas and millet, fermentation enriches Mbege with bioactive metabolites and microbes, such as Saccharomyces cerevisiae and lactobacilli44. Consumption of Mbege reduced, among others, neutrophil activation and boosted cytokine responses to Candida. We previously showed that β-glucan from S. cerevisiae is a potent inducer of trained immunity45. Metabolites like tyrosol, produced by S. cerevisiae46, were elevated in the participants in the Mbege arm, and therefore support its immunomodulating effects. Moreover, the flavan-3-ol pathway, which involves a subclass of polyphenols with recognized vascular benefits28, emerged as one of the most upregulated metabolic pathways in the Mbege arm. However, consumption of these fermented beverages should be balanced against risks such as the alcohol content and potential mycotoxin contamination. Chronic inflammation is a key driver of many NCDs8; studies have linked low-inflammatory diets to reduced risks of CVD, type 2 diabetes and cancer47,48. Adherence to the Mediterranean diet has shown benefit in both primary and secondary prevention of CVD49,50. Our study found that even brief dietary interventions can have prolonged effects on the immune and metabolic systems. While we cannot entirely rule out that some sustained effects were due to continued adherence to the intervention diets, participants were instructed to revert to their habitual diets after the intervention. Additionally, the limited availability of Western-style food products in rural areas makes continuous consumption of a WD unlikely. The persistent changes in neutrophil count and activation, inflammatory and cardiometabolic proteins, and transcription of immune-related genes at 4 weeks after the intervention align with prior experimental murine studies showing that the WD induces a maladaptive ‘trained immunity' program and that alternating high-fat diets stimulate myelopoiesis, increasing neutrophil counts and progenitor reprogramming, thereby promoting atherosclerosis7,51, The role of neutrophils in CVD is increasingly recognized52. Notably, the Mediterranean diet has been reported to reduce neutrophil counts and carotid intima-media thickness53. The rapid rise in obesity and NCDs across Africa underscores the need for sustainable, region-specific interventions. Preserving the consumption of heritage diets or incorporating indigenous and traditional foods into dietary practices could have a critical role in preventing and managing NCDs. However, Africa's diverse heritage diets are largely understudied, and the field of nutritional immunology in this context is still emerging. Policymakers often rely on findings from the Global North, which do not account for regional dietary, cultural and genetic differences. Our study provides direct evidence of the health risks of dietary westernization and the benefits of heritage diets and indigenous beverages, emphasizing the need for further research to explore their health impacts before they are lost to globalization, ensuring culturally appropriate and sustainable public health solutions. Our study also has a few limitations. First, food was offered ad libitum, and the intervention was not isocaloric, resulting in a weight gain in the AHD to WD switch group, which may have influenced immunometabolic profiles54. Second, the dietary intervention was short, although similar or even shorter interventions have demonstrated important immune effects. For example, a 2-week intervention with a vegan or ketogenic diet induced marked immune and gut microbiome changes4, while a 5-day fiber-poor diet was sufficient to alter host responses in both mice and humans55. Furthermore, our controlled setting established a baseline for short-term dietary responses in African environments, laying the foundation for future research on long-term effects. Third, our study included only men because of cultural logistics and confounding variables. Future studies should include women and diverse age groups. Fourth, the sample size is relatively small, although it is comparable to other multiomics dietary studies4,5. Sample size calculations were based on functional immune responses rather than multiomics data, given the challenges in defining meaningful effect sizes for large, complex multiomics datasets. Fifth, the study only provided molecular insights into short-term changes in immunometabolic features and pathways because of dietary practices but did not assess long-term clinical outcomes. Sixth, plasma metabolomics analysis used a nontargeted, high-throughput method relying on m/z ratios, which may lack specificity for definitive metabolite identification. Lastly, while the Chagga tribe's plant-rich and fiber-rich diet aligns with healthy dietary practices, Tanzania has over 100 tribes with varied diets. Thus, findings may not be generalizable to all heritage diets in the region. In summary, our findings highlight the substantial impact of dietary changes on both immune and metabolic systems, with implications for infection susceptibility and NCDs. They underscore the importance of preserving essential elements of AHDs, including fermented products, to mitigate the rising burden of NCDs and immune dysregulation in sub-Saharan Africa. As WDs increasingly replace indigenous diets, documenting the health benefits of heritage diets remains important. This study was conducted in the Moshi district in the Kilimanjaro region of northeastern Tanzania between April and August 2021, which coincides largely with the wet season. Moshi town serves as the administrative, commercial and educational center for the region, with a population of approximately 220,000 inhabitants. These foothills are rural areas where people live in smaller villages and are engaged in subsistence agriculture and animal husbandry. For this study, rural dwellers were mainly recruited from the village of Uru Shimbwe Juu, which is located approximately 15 km from Moshi town at an elevation of 1,978 m. Two cohorts of healthy volunteers assigned male at birth were recruited: (1) males residing in the rural village of Uru Shimbwe Juu or its environs, adhering to a predominantly Kilimanjaro heritage diet; and (2) males residing in Moshi town, adhering to a predominantly Western-style diet. Sex was determined based on self-report. Information about the study was disseminated through leaflets, community talks and announcements during church gatherings. Dietary habits in eligible volunteers who agreed to participate were assessed using a dietary recall questionnaire, conducted over three nonconsecutive 24-h periods in 1 week, including 1 weekend day. The questionnaire captured details on different foods, as well as estimated portion sizes (small, medium or large) and preparation methods for each food item. Questionnaires were evaluated by a nutritionist to ensure participants were adhering to either a diet typical of the Kilimanjaro region or a Western-style diet. This assessment relied primarily on the nutritionist's expertise and knowledge of local dietary habits, rather than a standardized set of criteria, incorporating traditional understanding of food patterns. Only participants adhering to either a Kilimanjaro heritage-style diet or Western-style diet were further considered for participation. Other inclusion criteria were age 20–40 years, BMI between 18 and 25 kg m−2, residing in either the rural or urban areas of Moshi for a minimum of 1 month preceding the study, and commitment to stay in the study area throughout the intervention period. Exclusion criteria were a positive result of human immunodeficiency virus or malaria rapid test, notable blood pressure abnormalities, elevated fasting blood sugar level, food allergies, recent acute illness, use of medication, antibiotics or vaccination in the last 3 months, previous hospitalization within the past year, presence of chronic conditions such as active malignancy, liver or kidney disease, tuberculosis, chronic hepatitis B or C infection, or unwillingness to consume alcohol. A standardized questionnaire was used to collect meta-data, lifestyle information, health status and daily activities, including tobacco use, physical activity level, medical history, medication use and vaccination records. Participants' physical activity was assessed by inquiring about their engagement in daily walking and sports activities. For descriptive purposes (Extended Data Table 1), sports activity levels were classified as follows: very active (more than 2 h of daily activity); active (up to 1 h daily or up to 2 h on some days); less active (up to 1 h on some days or only on weekends); and not active (no reported activity). For downstream analysis, we summarized these two estimations of physical activity with the scoring system provided in Supplementary Table 1. Briefly, engaging in any sports activity was scored as 2, and no sports activity as 1. The frequency of sports activity was scored as 1, 2 or 3 for ‘only weekend', ‘some days in a week' and ‘every day', respectively. Duration of daily walking was scored between 1, 2 and 3 for less than 30 min, between 30 and 60 min and more than 60 min per day, respectively. Scores values were scaled, and a sum of physical activity level was generated for each individual (Supplementary Table 5). 2483) and the National Institute for Medical Research in Tanzania (no. Participant recruitment and experimental procedures were conducted according to the principles mentioned in the Declaration of Helsinki (2013). All participants gave written informed consent before recruitment. The study was designed as an open-label, prospective intervention study with three arms: (1) rural-living men with a Kilimanjaro heritage diet who shifted to a Western-style diet for two weeks; (2) urban-living men with a Western-style diet shifting to a heritage diet for 2 weeks; and (3) men with a Western-style diet starting daily consumption of the fermented drink ‘Mbege' for 1 week (fermented beverage arm). To assess for dietary-independent effects, five controls who remained on their habitual diet were also included in the first two arms. Participants in the dietary intervention arms were randomized to either the dietary intervention (n = 23 in the AHD to WD arm; n = 22 in the WD to AHD arm) or remaining on their habitual diet (n = 5 in each arm, ‘controls'). No controls were included in the fermented beverage arm. Randomization was conducted by an independent statistician using a lottery method. Each preselected participant was assigned a number based on the dietary assessment, from which five numbers (controls) were randomly selected. The switch group received freshly prepared meals from a cook three times daily (breakfast, lunch and dinner) at a central cooking location in either Uru Shimbwe Juu village or in Moshi town. Food was provided ad libitum and consumed under supervision. In the fermented beverage arm, participants were directed to consume at least 1 l of the beverage daily between 16:00 to 17:00 while maintaining their usual diet. The beverage, produced fresh daily by one local manufacturer, was dispensed at a central location in Moshi town and intake was supervised. Intake of this fermented beverage was restricted in the other groups. For the rural group, samples were collected at the field site in Uru Shimbwe Juu, while for the urban group, samples were collected at the Kilimanjaro Clinical Research Institute (KCRI) research unit in Moshi town. The dietary intervention diets are detailed in Extended Data Table 2. Because of the absence of recorded portion sizes, macronutrient contributions to energy intake were calculated per 100 g of food and adjusted based on typical daily intake, assuming standard portion sizes. In the WD, carbohydrates contributed approximately 55% of total energy, primarily from refined sources such as refined maize flour (used for making stiff porridge, ‘ugali'), pancakes, potato chips, white rice, macaroni and fried plantains. Fats accounted for 23% of total energy, mainly from fried foods like fried chicken, beef, potato chips, ‘chips mayai' (a local dish combining potato chips and eggs) and pancakes. Protein contributed 16% of total energy, with chicken, beef and eggs as the primary sources. In contrast, the Kilimanjaro AHD provided 60% of total energy from carbohydrates, mainly derived from whole grains, roots and tubers, such as taro and boiled sweet potatoes. Protein contributed 11% of total energy, primarily from meat, local free-range chicken and fish. Fats accounted for 7% of total energy, while simple sugars and dietary fiber contributed 7% and 14% of total energy, respectively. A secondary outcome was the change in gut microbiome composition, assessed through untargeted metagenomics. These results will be reported separately. The sample size for this study was determined based on functional immune response outcomes rather than high-dimensional omics data, which typically generate large datasets unsuitable for conventional power calculations. Specifically, the calculation was based on previously reported differences in whole-blood TNF production after LPS stimulation between urban and rural populations in a cross-sectional study conducted in the same region11. The sample size was calculated to achieve 80% statistical power to detect significant differences at a two-sided alpha level of 0.05. Regarding harm, safety and possible adverse events, the risk associated with participation in this short dietary intervention was considered minimal. Therefore, spontaneous adverse event reporting was applied. Participants were instructed to report any adverse events they experienced during the study period directly to the study team. In addition, body weight was monitored throughout the study. Fasting glucose and blood pressure data from the after the intervention and follow-up periods, although initially planned according to the study protocol, were not recorded because of logistical challenges. Difficult access to the remote study site on Mount Kilimanjaro during the rainy season led to delays that compromised the ability to obtain early-morning, fasting measurements. EDTA-anticoagulated blood samples were analyzed using the Sysmex XN analyzer (Sysmex Corporation) to obtain a complete blood count and the Sysmex Extended Inflammation Parameters, including RE-MONO, RE-LYMPH and NEUT-RI. Blood samples were analyzed at the KCRI in Moshi, Tanzania within 2 h after venipuncture. Statistical comparisons across the study time points were performed using an LMM. The dream function (variancePartition package v.1.24) was applied to log2-transformed concentration values, with the time point as a fixed variable, participant as a random effect and adjustments for age, BMI and baseline physical activity level. Statistical summary results were extracted using the eBayes and topTable functions, applying false discovery rate (FDR) adjustment for multiple testing using Benjamini–Hochberg correction, with the significance cutoff for Padj set to 0.05. Paired fold change values were calculated using the fcrosFCmat function from the package fcros (v.1.6.1). Statistical results for each dietary arm were presented in a triangle plot, with the direction of change and paired log2 fold change presented for each time point comparison using ggplot2 (v.3.5.1). Variance partition analysis was conducted using the fitExtractVarPartModel function from the variance partition package (v.1.24.1) using time point, age, BMI and physical activity level at baseline as fixed variables and participants' ID as a random effect. The ex vivo cytokine stimulation experiments were conducted at the KCRI in Moshi, Tanzania. Whole-blood samples, obtained with heparin, were stimulated with bacterial, fungal and TLR3 and TLR4 agonists in a 48-well culture plate as described previously11. The following concentration stimuli were used: E. coli LPS (100 ng ml−1, Sigma-Aldrich); MTB (5 μg ml−1, H37Rv strain, in-house); poly(I:C) (50 µg ml−1, InvivoGen); S. aureus (106 ml−1 colony-forming units (CFU), ATCC strain 29213, in-house); C. albicans (106 CFU ml−1, UC820 strain, in-house); and S. enteritidis (106 CFU ml−1, in-house). Stimuli were prepared in Roswell Park Memorial Institute culture medium (Dutch-modified, Invitrogen) supplemented with 50 µg ml−1 gentamicin, 2 mM GlutaMAX and 1 mM pyruvate. The resulting supernatants were stored at −80 °C. Cytokine concentrations in the supernatant were quantified using ELISA according to the manufacturer's instructions provided by the respective kits (IL-6, IL-1β, IL-10 and TNF, R&D Systems; IFNγ, Sanquin). To ensure consistency, samples were measured using kits from the same lot. Statistical comparisons across the study time points were performed using an LMM. The dream function (variancePartition package v.1.24) was applied to log2-transformed concentration values, with time point as a fixed variable, participant as a random effect and adjustments for age, BMI and baseline physical activity level. Statistical summary results were extracted using the eBayes and topTable functions, applying FDR adjustment for multiple testing using Benjamini–Hochberg correction, with significance cutoff for Padj set to 0.05. Paired fold change values were calculated using the fcrosFCmat function from the package fcros (v.1.6.1). Statistical results for each dietary arm were presented in a triangle plot, presenting the direction of change and paired log2 fold change for each time point comparison using ggplot2 (v.3.5.1). Plasma proteins were assessed using proximity extension technology (Olink Proteomics AB)15. The assay involves the binding of target proteins to paired oligonucleotide antibody probes, followed by subsequent hybridization and amplification. NPX values, represented as arbitrary units on a log2 scale, were calculated from normalized cycle threshold data. Comprehensive validation details for the assay can be accessed on the Olink website (https://www.olink.com). Proteins with values below the limit of detection in more than 25% of subsequent samples were excluded. This resulted in the removal of 22 proteins, leaving 162 out of 184 proteins for further analysis. Most of these proteins had values below the limit of detection in almost all samples. Quality control measures and outlier detection were performed, revealing no significant batch effects or outliers attributable to technical issues. Additionally, control participants from both urban and rural groups exhibited consistent overall proteomic profiles, suggesting minimal batch-related bias. PCA was conducted using the prcomp function (stats v.3.6.2) on scaled NPX values or scaled NPX values corrected for participants (removeBatchEffect function, limma package v.3.50.3). To decrease the load of multiple testing, the proteins further tested in the differential analysis showed more than 5% of variance explained by the study time point (considering time point, age, BMI, physical activity level and participant as variables in the equation). The differential analysis for upregulated and downregulated proteins among the study time points was conducted using an LMM (variancePartition package v.1.24.) including time point as the main fixed variable, age, BMI and physical activity level at baseline as covariates, and the individual as a random effect (dream function). Statistical summary results were extracted for each time point comparison using the eBayes and topTable functions, FDR-adjusted for multiple testing using the Benjamini–Hochberg correction, setting the significance cutoff for Padj ≤ 0.05. Paired fold change values were calculated using the fcrosFCmat function from the package fcros (v.1.6.1), calculating the fold changes as a trimmed mean of the fold changes obtained using pairs of samples. The statistical results for each dietary arm were summarized in a triangle plot, presenting the direction of change and paired log2 fold change for each time point comparison using ggplot2 (v.3.5.1). In addition, beyond protein assignment to the designated Olink panels, DAPs were annotated to biological processes, based on biological process information obtained from the Human Protein Atlas (v.23.0). Plasma concentrations of highly sensitive CRP were measured using an ELISA, according to the manufacturer's instructions (Quantikine ELISA Kit, R&D Systems). For samples with CRP levels below the detection limit, a value equivalent to the lower limit of detection (0.2 mg l−1) was assigned. A comparative analysis of CRP among the study time points in each study arm was conducted using an LMM (variancePartition package v.1.24. ), with statistical summary results extracted for each time point comparison using the eBayes and topTable functions, FDR-adjusted for multiple testing using Benjamini–Hochberg correction, setting the significance cutoff for Padj at 0.05. Individuals from the 300 TZFG cohort (n = 295) were previously classified into two food clusters based on their food-associated plasma metabolome profiles: a Kilimanjaro heritage diet food cluster (n = 138) and a Western-type diet food cluster (n = 157). Differential proteins were detected using a linear model correcting for age, sex and BMI with the variancePartition package dream function followed by eBayes and topTable functions to extract summary statics (v.1.24). Differential proteins were visualized in combined violin-box-scatter plots, presenting the NPX values measured for each food cluster using ggplot2 (v.3.5.1). A qualitative comparison was conducted between the differential proteins in the 300 TZFG cohort and those in the heritage diet dietary arm, with results presented in a colored dot plot indicating proteins associated with the heritage diet in both datasets. Untargeted plasma metabolomics was performed by General Metabolics as described previously57. Briefly, an untargeted metabolomics approach was used to analyze plasma metabolites via high-throughput mass spectrometry (Agilent Technologies). The high-throughput method used is based on flow injection, without separation of compounds using a liquid chromatography gradient, allowing for efficient analysis of large cohorts. Polar metabolites were analyzed using the General Metabolics' high-throughput, nontargeted metabolomics platform in negative ion acquisition mode. Technical replicates were acquired for all samples. The data obtained, covering a mass-to-charge ratio (m/z) range of 50–1,000 at a flow rate of 150 μl min−1, were subjected to further analysis through centroiding on a high-performance computing cluster using MATLAB R2018a. The resultant matrix was constructed from centroid lists. Annotation coverage was typical for human plasma, with 1,266 annotated ions and 1,339 unique molecular formulas matched against ion libraries derived from the HMDB (https://www.hmdb.ca/), Kyoto Encyclopedia of Genes and Genomes (KEGG) (https://www.genome.jp/kegg/) and Chemical Entities of Biological Interest (ChEBI) (https://www.ebi.ac.uk/chebi/) databases within 0.001 atomic mass unit of theoretical mass. Given that the data were derived from human plasma, we prioritized annotation using the HMDB and converted KEGG or ChEBI IDs to HMDB IDs when available, using the Metabolite ID Conversion (https://www.metaboanalyst.ca/MetaboAnalyst/upload/ConvertView.xhtml, accessed March 2024). Drugs that were detected using manual curating in the downstream analysis were flagged in the summary table and were disregarded in the analysis. Differential analysis for upregulated and downregulated metabolites between study time points was conducted using an LMM, including time point as a main fixed variable, age, BMI and physical activity level at baseline as covariates, and the individual as a random effect using the variancePartition package dream function followed by eBayes and topTable functions to extract the summary statics (v.1.24). Paired fold change values were calculated using the fcrosFCmat function from the fcros package (v.1.6.1), calculating the fold changes as a trimmed mean of the fold changes obtained using pairs of samples. DAMs were enriched for and assigned to chemical classes and enriched for metabolic pathways by querying the RaMP database (v.2.3.0, last updated 27 July 2023, accessed August 2024) using the website's Application Programming Interface /api/combined-fisher-test and /api/combined-fisher-test using the R json function (jsonlite, v.1.8.4). Whole blood was collected in Qiagen PAXgene tubes. Total RNA was isolated using the QIAGEN PAXgene RNA isolation kit according to the manufacturer's guidelines; eluted RNA was dissolved in RNase-free water. RNA quality was evaluated in the TapeStation 4200 (Agilent Technologies) by estimating 28S and 18S band integrity. TruSeq Stranded Total RNA with Ribo-Zero Globin (Illumina) was used to convert total RNA into double-stranded complementary DNA molecule libraries. Briefly, depletion of ribosomal and globin mRNA was performed on 750 ng purified total RNA using biotinylated, target-specific oligonucleotides combined with Ribo-Zero rRNA removal beads. Under elevated temperature, the remaining RNA was fragmented using divalent cations. The first strand was generated using SuperScript2 RT (Invitrogen) supplemented with actinomycin D, followed by second-strand synthesis with deoxyuridine triphosphate replacing deoxythymidine triphosphate. Before PCR amplification, adenylation of the 3′ ends and index adapter ligation were done before subsequent yielding of the final library. The remaining overhangs were converted into blunt ends via exonuclease and polymerase activities, and enzymes were removed. Selective DNA fragment enrichment with ligated adapter molecules was performed using Illumina PCR primers in 15 cycles of PCR reaction, followed by purification of complementary DNA using SPRI Beads (Beckman Coulter). High-throughput sequencing was carried out with the Illumina NovaSeq 6000 Sequencing System on an S4 flow cell with v.1.5 chemistry (76-bp paired-end reads); data were converted into FASTQ files using bcl2fastq2 v.2.20. Raw RNA-seq data were aligned using STAR (v.2.7.3a) and human reference transcriptome version HG38 to generate the gene counts table. Raw gene counts were normalized using DESeq2 (v.1.26.0) in R (v.4.1.2)58. Each one of the dietary arms was analyzed separately, together with its control group; the following analysis was applied: low-expressed gene counts (genes with fewer than five counts in more than 10% of the samples) as well as hemoglobin or ribosomal genes were excluded, resulting in 19,887–20,585 genes present in each of the dietary groups and comparisons. Variance stabilizing transformation (VST) was applied on normalized data (exact numbers used for the analysis are provided in Supplementary Table 4). Before differential analysis, quality control was done with all study samples using robust PCA on VST-transformed corrected counts of all present genes using the PcaHubert function from the rrcov package (v.1.7-5)59, excluding six samples that presented high score and orthogonal distance. PCA was conducted using the prcomp function (stats, v.3.6.2) on scaled VST values or scaled VST values corrected for participants (removeBatchEffect function, limma package v.3.50.3). For the differential analysis, we followed the strategy for group-specific condition effects, with individuals nested within groups, which assumes a common time effect in the diet or control groups. Differential expression analysis was performed between each two time points using the FDR correction method for PadjIHW60, setting the threshold for Padj at 0.2, with no fold change threshold. Gene set enrichment analysis for the GO terms of DEGs was performed using the clusterProfiler package (v.4.2.2) using the compareCluster function with Padj < 0.05 (with Benjamini–Hochberg correction) defining all expressed genes as the universe. The function simplify was used to reduce the number of similar terms (clusterProfiler), setting the threshold to 0.7; further selection of pathways of interest was done manually for visualization using dotplot or heatplot with the respective clusterProfiler visualization functions. The full list of DEGs and enriched GO terms is found in Supplementary Tables 14–19. Transcription factor enrichment analysis was performed using the ChIP-X Enrichment Analysis 3 (ChEA3) tool via its Application Programming Interface (https://maayanlab.cloud/chea3/api/enrich/; accessed April 2024), using the jsonlite package in R (v.1.8.4)61. Transcription factor and gene targets found in the same DEG group were visualized in a circular plot using igraph (v.1.2.11) (Supplementary Table 19). All statistical analyses and visualizations were conducted in R (v.4.1.1. For all statistical analyses, intra-individual measurements were assessed using an LMM, with time point as the main fixed variable, and age, BMI and activity ranking at baseline as covariates. Measurements from the control groups were used for qualitative comparison with the intervention groups or as a contrast group for modeling transcriptional differences. Paired fold change values were calculated using the fcrosFCmat function from the fcros package (v.1.6.1), calculating the fold changes as a trimmed mean of the fold changes obtained using pairs of samples. For gene expression, fold change values were calcualted using results function (see the DESeq analysis above). To reduce the burden of multiple comparisons in the omics data, differential plasma protein (Olink assay) and metabolite analysis was performed on analytes where time points contributed more than 5% of the variance. P values were calculated using two-tailed hypothesis testing, and Benjamini–Hochberg correction adjusting for multiple testing on the overall tests performed at each time point contrast to control the FDR, with a significance threshold set at Padj < 0.05 when using mixed model analysis (setting the correction to ‘BH', topTable function, variancePartition package). Researchers from KCMC University in Moshi, Tanzania, had an integral role throughout the research process, including study design and implementation, data ownership, intellectual property rights, and authorship of publications. The local relevance of the research was confirmed through collaboration with KCMC University partners. Roles and responsibilities were clearly defined among collaborators at the outset, and plans for capacity building of local researchers were discussed. Furthermore, local and regional research relevant to the study was carefully considered and appropriately cited. Lastly, benefit-sharing measures were discussed in collaboration with KCMC University researchers. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All sequencing data have been deposited in the European Genome–phenome Archive under accession no. All additional datasets, including anonymized participant meta-data, nutritional information, circulating leukocyte data, whole-blood cytokine responses, and plasma proteome and metabolome data, are deposited in the Radboud Data Repository (https://data.ru.nl). Access to these datasets will be evaluated by a Data Access Committee to ensure compliance with the Tanzanian regulatory framework for data sharing (https://www.costech.or.tz/Files/Documents/1728287181.pdf). Requests will be reviewed by the Data Access Committee and a decision will be provided within 4 weeks. All code scripts used for the analysis of the data are available at https://github.com/TalPecht/dietstudy_Temba_Pecht_2024/. Burden of non-communicable diseases in sub-Saharan Africa, 1990-2017: results from the Global Burden of Disease Study 2017. Hemler, E. C. & Hu, F. B. Plant-based diets for cardiovascular disease prevention: all plant foods are not created equal. Differential peripheral immune signatures elicited by vegan versus ketogenic diets in humans. Gut microbiota-targeted diets modulate human immune status. & Tornero-Aguilera, J. F. Global impacts of Western diet and its effects on metabolism and health: a narrative review. Christ, A. et al. Western diet triggers NLRP3-dependent innate immune reprogramming. Temba, G. S. et al. Urban living in healthy Tanzanians is associated with an inflammatory status driven by dietary and metabolic changes. Kubo, R. & Kilasara, M. Brewing technique of mbege, a banana beer produced in northeastern Tanzania. Leeuwendaal, N. K., Stanton, C., O'Toole, P. W. & Beresford, T. P. Fermented foods, health and the gut microbiome. Shah, A. M., Tarfeen, N., Mohamed, H. & Song, Y. Fermented foods: their health-promoting components and potential effects on gut microbiota. Assarsson, E. et al. Homogenous 96-Plex PEA immunoassay exhibiting high sensitivity, specificity, and excellent scalability. Frolova, E. G. et al. Thrombospondin-4 regulates vascular inflammation and atherogenesis. Defective signaling in the JAK-STAT pathway tracks with chronic inflammation and cardiovascular risk in aging humans. Decreased interferon-α and interferon-β production in obesity and expression of suppressor of cytokine signaling. Leentjens, J. et al. Reversal of immunoparalysis in humans in vivo: a double-blind, placebo-controlled, randomized pilot study. Hotchkiss, R. S., Monneret, G. & Payen, D. Sepsis-induced immunosuppression: from cellular dysfunctions to immunotherapy. Influence of the microbiome, diet and genetics on inter-individual variation in the human plasma metabolome. Pico, A. R. et al. WikiPathways: pathway editing for the people. Estes, M. K. et al. A high fat western diet attenuates phasic dopamine release. Tham, Y. Y. et al. Lauric acid alleviates insulin resistance by improving mitochondrial biogenesis in THP-1 macrophages. Karami, A., Fakhri, S., Kooshki, L. & Khan, H. Polydatin: pharmacological mechanisms, therapeutic targets, biological activities, and health benefits. Peace, C. G., O'Carroll, S. M. & O'Neill, L. A. J. Fumarate hydratase as a metabolic regulator of immunity. Patel, K. P., Luo, F. J.-G., Plummer, N. S., Hostetter, T. H. & Meyer, T. W. The production of p-cresol sulfate and indoxyl sulfate in vegetarians versus omnivores. The role of bacterial-derived aromatic amino acids metabolites relevant in autism spectrum disorders: a comprehensive review. Nauseef, W. M. Myeloperoxidase in human neutrophil host defense. SON DNA-binding protein mediates macrophage autophagy and responses to intracellular infection. You, F. et al. ELF4 is critical for induction of type I interferon and the host antiviral response. Ehrlich, A. M. et al. Indole-3-lactic acid associated with Bifidobacterium-dominated microbiota significantly decreases inflammation in intestinal epithelial cells. Urrechaga, E., Mugertza, G., Fernández, M., España, P. P. & Aguirre, U. Leukocyte differential and reactive lymphocyte counts from Sysmex XN analyzer in the evaluation of SARS-CoV-2 infection. Malongane, F., Phoswa, W. N. & Berejena, T. The effect of indigenous African diet on inflammatory markers linked to type 2 diabetic mellitus. Popoola, J. et al. Nutritional, functional, and bioactive properties of African underutilized legumes. & Mackay, C. R. A major mechanism for immunomodulation: dietary fibres and acid metabolites. Yahfoufi, N., Alsadi, N., Jambi, M. & Matar, C. The immunomodulatory and anti-inflammatory role of polyphenols. Gut microbiome-mediated metabolism effects on immunity in rural and urban African populations. Witkamp, R. F. Bioactive components in traditional foods aimed at health promotion: a route to novel mechanistic insights and lead molecules? Christ, A., Lauterbach, M. & Latz, E. Western diet and the immune system: an inflammatory connection. Shumye Gebre, T. et al. Unveiling the potential of African fermented cereal-based beverages: probiotics, functional drinks, health benefits and bioactive components. Potent induction of trained immunity by Saccharomyces cerevisiae β-glucans. Soldevila-Domenech, N. et al. Generation of the antioxidant hydroxytyrosol from tyrosol present in beer and red wine in a randomized clinical trial. Wang, P. et al. Optimal dietary patterns for prevention of chronic disease. Estruch, R. et al. Primary prevention of cardiovascular disease with a Mediterranean diet supplemented with extra-virgin olive oil or nuts. Alternating high-fat diet enhances atherosclerosis by neutrophil reprogramming. Boughanem, H. Mediterranean diet, neutrophil count, and carotid intima-media thickness in secondary prevention: the CORDIOPREV study. Jordan, S. et al. Dietary Intake regulates the circulating inflammatory monocyte pool. Siracusa, F. et al. Short-term dietary changes can result in mucosal and systemic immune depression. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Todorov, V. & Filzmoser, P. An object-oriented framework for robust multivariate analysis. & Huber, W. Data-driven hypothesis weighting increases detection power in genome-scale multiple testing. B. et al. ChEA3: transcription factor enrichment analysis by orthogonal omics integration. We thank all volunteers for their participation in the dietary intervention study. We thank J. Kwayu and J. Lemunge at KCRI Moshi Tanzania for help with sample collection; H. Lemmers and H. Toenhake-Dijkstra at Radboudumc, the Netherlands, for help with the laboratory analysis; and C. Amour at KCMC University for helping with data entry and cleaning. We also thank M. Kraut and H. Theis at Platform for Single Cell Genomics and Epigenomics, German Center for Neurodegenerative Diseases, for their significant contribution to RNA-seq and K. Dahm at Systems Medicine, German Center for Neurodegenerative Diseases, for consultations on RNA-seq analysis and visualization. This study was funded by the Joint Programming Initiative: A Healthy Diet for a Healthy Life (JPI-HDHL) and ZonMW (no. and D.C; the Radboud Revolving Research Funds (3R-Fund) awarded to G.S.T. NWO SPI94-212) and a European Research Council Advanced Grant (no. ; and the German Research Foundation under Germany's Excellence Strategy (no. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Radboudumc and KCMC University are partners in the Africa-Europe Cluster of Research Excellence (CoRE) ‘Translational Research in Infection, Immunity and Inflammation', led by the African Research Universities Alliance (ARUA) and The Guild. These authors contributed equally: Godfrey S. Temba, Tal Pecht. Department of Medical Biochemistry and Molecular Biology, KCMC University, Moshi, United Republic of Tanzania Godfrey S. Temba, Vesla I. Kullaya & Reginald A. Kavishe Department of Internal Medicine, Radboudumc Research Institute for Medical Innovation (RIMI), Radboud University Medical Center, Nijmegen, the Netherlands B. Joosten, Mihai G. Netea & Quirijn de Mast Department for Genomics and Immunoregulation, Life & Medical Sciences (LIMES) Institute, University of Bonn, Bonn, Germany Tal Pecht, Thomas Ulas, Sneha Kanungo, Lorenzo Bonaguro, Jonas Schulte-Schrepping & Joachim L. Schultze Systems Medicine, German Center for Neurodegenerative Diseases (DZNE), Bonn, Germany Tal Pecht, Thomas Ulas, Sneha Kanungo, Lorenzo Bonaguro, Jonas Schulte-Schrepping & Joachim L. Schultze Vesla I. Kullaya, Elias Mafuru & Blandina T. Mmbaga PRECISE Platform for Single Cell Genomics and Epigenomics, German Center for Neurodegenerative Diseases (DZNE) and University of Bonn, Bonn, Germany Lorenzo Bonaguro, Jonas Schulte-Schrepping & Joachim L. Schultze Gastroenterology and Nutrition Unit, Meyer Children's Hospital IRCCS, Florence, Italy Department of Human Genetics, Radboud University Medical Center, Nijmegen, the Netherlands Department of Immunology and Metabolism, Life & Medical Sciences (LIMES) Institute, University of Bonn, Bonn, Germany You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar were responsible for recruitment, data collection and laboratory analysis. carried out RNA-seq analysis and interpretation. carried out additional data analysis and quality control. were responsible for project administration and data interpretation. AB Biotek provided funding for future studies to Q.D.M., M.G.M. AB Biotek had no role in the conceptualization, design, data collection and analysis, decision to publish or preparation of the manuscript. The other authors declare no competing interests. Nature Medicine thanks Jiaqi Huang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ming Yang, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Flow diagram illustrating the enrollment, allocation, follow-up, and analysis of participants. Histograms of the percentage of explained variance by dietary intervention in the switch from African heritage diet (AHD) to Western diet (Switch to WD), switch from Western to African heritage diet (Switch to AHD), consumption of the fermented beverage (top, middle and bottom, respectively). Variance partition analysis was performed considering the change in features in each omics layer between post-intervention (t1, orange) or follow-up (t2, purple) vs baseline (t0) measurements. Age, BMI and physical activity levels at baseline were included as additional fixed variables and individual ID as random effect. The percentage of features in each data layer that intervention contributed to more than 5% in its variance is marked in each plot. Targeted plasma proteomics was conducted using the inflammatory and cardiometabolic Olink panel (Olink® Proteomics AB). Samples from N = 20, 22 and 22 individuals in the switch from African heritage to Western diet (Switch to WD), switch from Western to African heritage diet (Switch to AHD) and the fermented beverage intervention arms were obtained at baseline (t0), post-intervention (t1), and four weeks follow-up (t2). a- PCA was calculated on (i) scaled original NPX values, or (ii) scaled values corrected for participant ID. Dot plots depict the principal component 1 (PC1) and PC2, with color indicating the measurement time point (t0, t1 and t2 in green, orange and purple) and geometric ellipses drawn at 95% confidence for each time point. b- Dot plot showing the biological terms (derived from the Human Protein Atlas) for the differential abundant proteins in each intervention arm and time point comparison, with size indicating the number of proteins related to each term. Plasma metabolome was performed using untargeted metabolomics (General Metabolics, Cambridge, MA) to assess changes during the transition to a Western diet (WD), African heritage-style diet (AHD) and fermented beverage consumption. Samples were obtained at baseline (t0) post-intervention (t1, week 2 for the WD or AHD diet interventions and one week for the fermented beverage intervention, N = 22/group), and after four weeks follow-up (t2, N = 22/group). a- Dotplots showing principal component 1 (PC1) and PC2 from principal components analysis (PCA) of log2-scaled ion intensity scaled values using all metabolites as input. Color indicates the time points (green, orange, purple for t0, t1 and t2, respectively). A geometric ellipse is drawn at 95% confidence for each time point. b- Triangle plots present top differentially abundant metabolites (DAMs, adjusted P value < 0.05 and absolute paired fold-change > 1.2). Arrows' direction (up or down) indicates the direction of change, comparing late vs. early time points in each comparison as listed at the bottom of the plot, with color fill depicting the log2 paired fold-change value for each comparison. Statistical analysis was carried out using a linear mixed model for each comparison, including time point, BMI, physical activity level and age as fixed variables, and participant ID as random effect. Asterisks indicate significance level based on BH-adjusted P value (*, **, ***, **** represents adj. Whole blood transcriptome data were analysed for participants in the heritage to Western diet switch group (Switch to WD, N = 22), the Western to heritage diet switch group (Switch to AHD, N = 22) and the fermented beverage group (N = 21). Blood samples were collected at 3 time points: baseline (t0), post-intervention (t1), and follow-up (t2), four weeks after the end of intervention. Five control individuals for each dietary arm were included in the DE analysis using DESeq2, with model design including the individual (ind.n), treatment (diet or control) and the time point (baseline, post-intervention or follow-up): ~ treatment + treatment:ind.n + treatment:time point. DE analysis was performed between each two time points using the Benjamini-Hochberg (BH) controlling method for P value adjustment and independent hypothesis weighting (IHW). The two-sided significance level was set to adjusted Padj ≤ 0.2, with no fold change threshold, a, PCA was calculated on scaled variance stabilized transformed (vst) count values. Dot plots depict the data on based on principal component (PC)1 and PC2 (x and y-axis respectively). Dot colors represent time points: t0 (green), t1 (orange), and t2 (purple). b, c - Tile plots display the log2 fold-change (log2FC) of genes related to selected GO terms enriched in the switch to the Western Diet (WD) group (indicated by color). d- Circular plot shows the predicted transcription factors (TF) and their targets among upregulated genes in the fermented beverage intervention group (t1 vs t0, based on ChEA3 analysis). a- Participants in the 300 TZFG cohort (n = 295) were assigned, based on their food-derived metabolome profile, to a food cluster associated with a more African heritage-style diet (AHD food cluster; n = 138) or a more Western diet (WD food cluster; n = 157), as previously described9. b- Results of the Olink 92 inflammation panel were compared between the African heritage diet and Western diet food clusters using a linear model, correcting for age, sex and BMI. The dot plot shows differentially expressed proteins between the two clusters, with blue indicating proteins that also exhibited significant changes in the Western to heritage diet intervention arm of the current study. Proteins levels are presented as NPX values. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Immune and metabolic effects of African heritage diets versus Western diets in men: a randomized controlled trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-01789-w'>Activity of GPCR-targeted drugs influenced by human gut microbiota metabolism</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 09:03:44
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Microbiota-mediated drug metabolism can affect pharmacological efficacy. Here we conducted a systematic comparative metabolomics investigation of drug metabolism modes by evaluating the impacts of human gut commensal bacteria on 127 G-protein-coupled receptor (GPCR)-targeted drugs. For the most extensively metabolized drugs in our screen, we elucidated both conventional and unconventional drug transformations and the corresponding activities of generated metabolites. Comparisons of drug metabolism by a gut microbial community versus individual species revealed both taxon intrinsic and collaborative processes that influenced the activity of the metabolized drugs against target GPCRs. We also observed iloperidone inactivation by generating unconventional metabolites. The human gut commensal bacteria mixture incorporated sulfur in the form of a thiophene motif, whereas Morganella morganii used a cascade reaction to incorporate amino-acid-derived tricyclic systems into the drug metabolites. Our results reveal a broad impact of human gut commensal bacteria on GPCR-targeted drug structures and activities through diverse microbiota-mediated biotransformations. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data generated during this study are included in the Article and its Supplementary Tables 1–25 and Supplementary Figs. The information on all GPCR drugs discussed in this Article was sourced from DrugBank Online at https://go.drugbank.com/. Source data are provided with this paper. Rosenbaum, D. M., Rasmussen, S. G. F. & Kobilka, B. K. The structure and function of G-protein-coupled receptors. Google Scholar Hauser, A. S., Attwood, M. M., Rask-Andersen, M., Schiöth, H. B. & Gloriam, D. E. Trends in GPCR drug discovery: new agents, targets and indications. Google Scholar Sriram, K. & Insel, P. A. G protein-coupled receptors as targets for approved drugs: how many targets and how many drugs? Google Scholar Yang, D. et al. G protein-coupled receptors: structure- and function-based drug discovery. Google Scholar Hauser, A. S. et al. Pharmacogenomics of GPCR drug targets. Google Scholar A., Lodowski, D. T. & Palczewski, K. The significance of G protein-coupled receptor crystallography for drug discovery. Google Scholar Mantas, I., Saarinen, M., Xu, Z.-Q. D. & Svenningsson, P. Update on GPCR-based targets for the development of novel antidepressants. Google Scholar & Jackson, S. H. D. Age-related changes in pharmacokinetics and pharmacodynamics: basic principles and practical applications. Google Scholar Pharmacogenomics of drug metabolizing enzymes and transporters: relevance to precision medicine. Genomics Proteomics Bioinform. Google Scholar Roden, D. M., Wilke, R. A., Kroemer, H. K. & Stein, C. M. Pharmacogenomics. Google Scholar Koziolek, M. et al. The mechanisms of pharmacokinetic food–drug interactions—a perspective from the UNGAP group. Google Scholar Tariq, R. A., Vashisht, R., Sinha, A. Medication Dispensing Errors and Prevention (StatPearls, 2024). Li, Y. et al. Current trends in drug metabolism and pharmacokinetics. Google Scholar Zanger, U. M. & Schwab, M. Cytochrome P450 enzymes in drug metabolism: regulation of gene expression, enzyme activities, and impact of genetic variation. Google Scholar & Tang, W. Drug metabolism in drug discovery and development. Google Scholar Lai, Y. et al. Recent advances in the translation of drug metabolism and pharmacokinetics science for drug discovery and development. Google Scholar Zimmermann, M., Zimmermann-Kogadeeva, M., Wegmann, R. & Goodman, A. L. Mapping human microbiome drug metabolism by gut bacteria and their genes. Google Scholar Javdan, B. et al. Personalized mapping of drug metabolism by the human gut microbiome. Google Scholar Pant, A., Maiti, T. K., Mahajan, D. & Das, B. Human gut microbiota and drug metabolism. Google Scholar Drug metabolism by the host and gut microbiota: a partnership or rivalry? Google Scholar Zimmermann-Kogadeeva, M. Quantifying host–microbiota interactions. Google Scholar Li, H., He, J. & Jia, W. The influence of gut microbiota on drug metabolism and toxicity. Google Scholar & Goodman, A. L. Integrating the gut microbiome and pharmacology. Google Scholar Thursby, E. & Juge, N. Introduction to the human gut microbiota. Google Scholar Lozupone, C. A., Stombaugh, J. I., Gordon, J. I., Jansson, J. K. & Knight, R. Diversity, stability and resilience of the human gut microbiota. Google Scholar Hou, K. et al. Microbiota in health and diseases. Google Scholar Qin, J. et al. A human gut microbial gene catalogue established by metagenomic sequencing. Google Scholar Weersma, R. K., Zhernakova, A. & Fu, J. Interaction between drugs and the gut microbiome. Google Scholar & Zuo, T. Interplays between drugs and the gut microbiome. Google Scholar Spanogiannopoulos, P. et al. Host and gut bacteria share metabolic pathways for anti-cancer drug metabolism. Google Scholar Haiser, H. J. et al. Predicting and manipulating cardiac drug inactivation by the human gut bacterium Eggerthella lenta. Google Scholar Koppel, N., Maini Rekdal, V. & Balskus, E. P. Chemical transformation of xenobiotics by the human gut microbiota. Google Scholar Chen, H. et al. A forward chemical genetic screen reveals gut microbiota metabolites that modulate host physiology. Google Scholar Kroeze, W. K. et al. PRESTO-Tango as an open-source resource for interrogation of the druggable human GPCRome. Google Scholar Burapan, S., Kim, M. & Han, J. Demethylation of polymethoxyflavones by human gut bacterium, Blautia sp. Google Scholar Schendel, R. R. et al. Structural transformation of 8–5-coupled dehydrodiferulates by human intestinal microbiota. Google Scholar Rich, B. E. et al. Alternative pathway for dopamine production by acetogenic gut bacteria that O‐demethylate 3‐methoxytyramine, a metabolite of catechol O‐methyltransferase. Google Scholar Kumano, T. Specialized metabolites degradation by microorganisms. Google Scholar Bess, E. N. et al. Genetic basis for the cooperative bioactivation of plant lignans by Eggerthella lenta and other human gut bacteria. Google Scholar Casey Laizure, S., Herring, V., Hu, Z., Witbrodt, K. & Parker, R. B. The role of human carboxylesterases in drug metabolism: have we overlooked their importance? Google Scholar et al. Glucuronidation of a sarpogrelate active metabolite is mediated by UDP-glucuronosyltransferases 1A4, 1A9, and 2B4. Google Scholar Malhotra, B., Guan, Z., Wood, N. & Gandelman, K. Pharmacokinetic profile of fesoterodine. Google Scholar Ishizuka, T. et al. Human carboxymethylenebutenolidase as a bioactivating hydrolase of olmesartan medoxomil in liver and intestine. Google Scholar Kuze, Y., Kogame, A., Jinno, F., Kondo, T. & Asahi, S. Development, validation and application of the liquid chromatography tandem mass spectrometry method for simultaneous quantification of azilsartan medoxomil (TAK-491), azilsartan (TAK-536), and its 2 metabolites in human plasma. Google Scholar Asaki, T. et al. Selexipag: an oral and selective IP prostacyclin receptor agonist for the treatment of pulmonary arterial hypertension. Google Scholar Helf, M. J., Fox, B. W., Artyukhin, A. B., Zhang, Y. K. & Schroeder, F. C. Comparative metabolomics with Metaboseek reveals functions of a conserved fat metabolism pathway in C. elegans. Google Scholar Vizcaino, M. I., Engel, P., Trautman, E. & Crawford, J. M. Comparative metabolomics and structural characterizations illuminate colibactin pathway-dependent small molecules. Google Scholar Böcker, S., Letzel, M. C., Lipták, Z. & Pervukhin, A. SIRIUS: decomposing isotope patterns for metabolite identification. Google Scholar Dührkop, K. et al. SIRIUS 4: a rapid tool for turning tandem mass spectra into metabolite structure information. Google Scholar Zheng, R., Wu, Y.-H., Jiang, D.-X. & Zhang, D. Determination of metabolite of nicergoline in human plasma by high-performance liquid chromatography and its application in pharmacokinetic studies. Google Scholar et al. Carboxylesterase 1 as a determinant of clopidogrel metabolism and activation. Google Scholar Kaye, B., Cussans, N., Faulkner, J., Stopher, D. & Reid, J. The metabolism and kinetics of doxazosin in man, mouse, rat and dog. Google Scholar Jia, M. et al. Simultaneous determination of iloperidone and its two active metabolites in human plasma by liquid chromatography–tandem mass spectrometry: application to a pharmacokinetic study. Google Scholar Subramanian, N. & Kalkman, H. O. Receptor profile of P88-8991 and P95-12113, metabolites of the novel antipsychotic iloperidone. Google Scholar Reher, R. et al. A convolutional neural network-based approach for the rapid annotation of molecularly diverse natural products. Google Scholar Dunbar, K. L., Scharf, D. H., Litomska, A. & Hertweck, C. Enzymatic carbon–sulfur bond formation in natural product biosynthesis. Google Scholar Zhang, X. et al. Biosynthesis of chuangxinmycin featuring a deubiquitinase-like sulfurtransferase. Google Scholar Wang, Y. et al. Identifying the minimal enzymes for unusual carbon–sulfur bond formation in thienodolin biosynthesis. Google Scholar Ouyang, H. et al. Mining the metabolic capacity of Clostridium sporogenes aided by machine learning. Google Scholar Bittremieux, W. et al. Comparison of cosine, modified cosine, and neutral loss based spectrum alignment for discovery of structurally related molecules. Google Scholar Wen, W.-H. et al. Reductive inactivation of the hemiaminal pharmacophore for resistance against tetrahydroisoquinoline antibiotics. Google Scholar Lichman, B. R. The scaffold-forming steps of plant alkaloid biosynthesis. Google Scholar Mancinotti, D., Michiko Frick, K. & Geu-Flores, F. Biosynthesis of quinolizidine alkaloids in lupins: mechanistic considerations and prospects for pathway elucidation. Google Scholar Cao, Y. et al. Commensal microbiota from patients with inflammatory bowel disease produce genotoxic metabolites. Google Scholar Wu, Q. et al. Metabolomics and genomics enable the discovery of a new class of nonribosomal peptidic metallophores from a marine micromonospora. Google Scholar Rampino, A. et al. Antipsychotic drug responsiveness and dopamine receptor signaling; old players and new prospects. Google Scholar Wang, S. et al. Structure of the D2 dopamine receptor bound to the atypical antipsychotic drug risperidone. Google Scholar Rabin, C. et al. In vitro and in vivo demonstration of risperidone implants in mice. Google Scholar Spina, E. & De Leon, J. Metabolic drug interactions with newer antipsychotics: a comparative review. Google Scholar Wilson, I. D. & Nicholson, J. K. Gut microbiome interactions with drug metabolism, efficacy, and toxicity. Google Scholar Vich Vila, A. et al. Impact of commonly used drugs on the composition and metabolic function of the gut microbiota. Google Scholar Zhao, Q., Chen, Y., Huang, W., Zhou, H. & Zhang, W. Drug–microbiota interactions: an emerging priority for precision medicine. Park, H. B. et al. Bacterial autoimmune drug metabolism transforms an immunomodulator into structurally and functionally divergent antibiotics. Bustion, A. E., Nayak, R. R., Agrawal, A., Turnbaugh, P. J. & Pollard, K. S. SIMMER employs similarity algorithms to accurately identify human gut microbiome species and enzymes capable of known chemical transformations. Heinken, A. et al. Genome-scale metabolic reconstruction of 7,302 human microorganisms for personalized medicine. Zimmermann, M., Zimmermann-Kogadeeva, M., Wegmann, R. & Goodman, A. L. Separating host and microbiome contributions to drug pharmacokinetics and toxicity. Loureiro, A. I. et al. Absorption, metabolism and excretion of opicapone in human healthy volunteers. Surapaneni, S. et al. Absorption, metabolism, and excretion, in vitro pharmacology, and clinical pharmacokinetics of ozanimod, a novel sphingosine 1-phosphate receptor modulator. Overton, H. A., Fyfe, M. C. T. & Reynet, C. GPR119, a novel G protein-coupled receptor target for the treatment of type 2 diabetes and obesity. Flock, G., Holland, D., Seino, Y. & Drucker, D. J. GPR119 regulates murine glucose homeostasis through incretin receptor-dependent and independent mechanisms. Qian, Y. et al. Activation and signaling mechanism revealed by GPR119-Gs complex structures. Targeting the GPR119/incretin axis: a promising new therapy for metabolic-associated fatty liver disease. Goodman, A. L. et al. Identifying genetic determinants needed to establish a human gut symbiont in its habitat. Cell Host Microbe 6, 279–289 (2009). Kozich, J. J., Westcott, S. L., Baxter, N. T., Highlander, S. K. & Schloss, P. D. Development of a dual-index sequencing strategy and curation pipeline for analyzing amplicon sequence data on the MiSeq Illumina sequencing platform. Smith, C. A., Want, E. J., O'Maille, G., Abagyan, R. & Siuzdak, G. XCMS: processing mass spectrometry data for metabolite profiling using nonlinear peak alignment, matching, and identification. Marzullo, P. et al. Ammonium formate-Pd/C as a new reducing system for 1,2,4-oxadiazoles. synthesis of guanidine derivatives and reductive rearrangement to quinazolin-4-ones with potential anti-diabetic activity. Hanwell, M. D. et al. Avogadro: an advanced semantic chemical editor, visualization, and analysis platform. & Olson, A. J. AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. We thank members of the Crawford lab for helpful discussions. We also thank T. Wu and J. Kim at the Yale West Campus Analytical Core for their assistance with LC–MS. This work was primarily supported by the National Institute of General Medical Sciences (1RM1GM141649 to J.M.C. The mouse model experiment (DP2DK125119 to N.W.P.) and preparation of the microbial community (R01AT010014 to A.L.G.) were supported by the NIH. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. was in part supported by the NIH Chemistry-Biology Interface Training Program (T32GM067543) and the NSF Predoctoral Fellowship Program (2020293597). was in part supported by the NIH Individual Predoctoral Fellowship program (F31DK132941). B.D.-L. is a Cancer Research Institute Irvington Fellow supported by the Cancer Research Institute (CRI4515). was in part supported by a Peter Moore fellowship in the Department of Chemistry. Present address: Department of Pharmaceutical Sciences, University of Pittsburgh, Pittsburgh, PA, USA These authors contributed equally: Qihao Wu, Deguang Song. Department of Chemistry, Yale University, New Haven, CT, USA Qihao Wu, Yanyu Zhao, Tayah Turocy & Jason M. Crawford Institute of Biomolecular Design and Discovery, Yale University, West Haven, CT, USA Qihao Wu, Yanyu Zhao, Tayah Turocy & Jason M. Crawford Department of Immunobiology, Yale University School of Medicine, New Haven, CT, USA Deguang Song, Brianna Duncan-Lowey & Noah W. Palm Department of Microbial Pathogenesis, Yale University School of Medicine, New Haven, CT, USA Andrew A. Verdegaal, Andrew L. Goodman & Jason M. Crawford Microbial Sciences Institute, Yale University, West Haven, CT, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived the project. conducted the experiments. B.D.-L. performed molecular docking. carried out data analyses. prepared graphical illustrations and wrote the first draft of the manuscript. All authors reviewed, edited and contributed to the final version of the manuscript. Correspondence to Andrew L. Goodman, Noah W. Palm or Jason M. Crawford. is a co-founder of Artizan Biosciences and has received research funding for unrelated studies from Artizan Biosciences and F. Hoffmann-La Roche. serves on the scientific advisory boards for Seres Therapeutics, Piton Therapeutics, Nuanced Health and Taconic Biosciences. The others authors declare no competing interests. Nature Chemistry thanks Aaron Wright and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Number of GPCRs and their class that interacted with the 127 selected drugs. The drug target information was obtained from the DrugBank database. (b) Mapping specific interactions between drugs and GPCRs. Agonist interactions were labeled in red and antagonist interactions were labeled in grey. Aligned with the prevalence of antagonistic GPCR drugs in the DrugBank database, our drug panel was composed primarily of antagonists (74.8 %, 95/127), but they were complemented with some agonists (20.5 %, 26/127) and dual-function drugs (4.7 %, 6/127). (a) The bar graph illustrates that out of the screened 127 drugs, 12 were heavily consumed (drug consumed ≥ 80%). n = 3 biological replicates. Data represent mean ± s.d. (b) The EICs of parent drug ions for both Bac+ and Bac- (after 24 h of drug incubation) offer a relative quantification of their concentration in the supernatants. The significant differences in drug concentration observed between the presence and absence of bacteria strongly suggest that drug consumption is dependent on the presence of bacteria. The EICs (with a ± 20 ppm window) of drug ions were determined using the following ions: [M + H]+, m/z 559.2300 for olmesartan medoxomil; [M + H]+, m/z 569.1667 for azilsartan medoxomil; [M + H]+, m/z 389.2071 for trimethobenzamide; [M + H]+, m/z 412.2846 for fesoterodine; [M + H]+, m/z 497.2217 for selexipag; [M + H]+, m/z 427.2028 for iloperidone; [M + H]+, m/z 405.1921 for ozaminod; [M + H]+, m/z 461.1296 for ponesimod; [M + H]+, m/z 430.2224 for sarpogrelate; [M + H]+, m/z 501.1360 for SB756050; [M + H]+, m/z 388.2118 for Trimebutine; [M + H]+, m/z 457.1904 for GSK1292263. The experiment was conducted in triplicates for both Bac+ and Bac- conditions. Data are means ± SEM. Statistical significance was determined using two-tailed t-test. (a) Sample preparation procedure for GPCR activity tests. (b) GPCR activity of 12 drugs were tested and compared under two conditions: drugs were administered without undergoing metabolism (at 0 h, blue), and drugs undergoing complete metabolization (at 24 h, red). Drug activities were quantified by EA50 values, Log10[Conc, M] refers to the starter drug concentration of 10 μM in the complex medium extracts. n = 3 biological replicates. Data are means ± SEM. Volcano plot illustrates the FC (molecular features with Bac+ versus molecular features with Bac-) and FDR values in LC-QTOF-MS positive ion mode. (a) The raw LC-MS files of Bac+ and Bac- were processed using XCMS modules to transform them into molecular feature information through feature detection, grouping, and filtering ( < 3,000 molecular features were prioritized based on a strict intensity cutoff to focus on robust drug transformations in this study). (b) Molecular feature selection involved volcano plot analysis with a p-value and fold change cutoff, followed by a manual inspection to create a conclusive validated hit list. (c) Structural elucidation of molecular features of interest was carried out through MS/MS and NMR techniques, as well as chemical standard comparisons. (a) Information on the metabolites of six drugs. The bar graphs on the left and the EIC traces on the right depict a comparison between Bac- and Bac+ for ponesimod (b), SB756050 (c), trimethobenzamide (d), GSK1292263 (e), ozanimod (f), and trimebutine (g). The metabolism of five other drugs, including sarpogrelate, olmesartan medoxomil, fesoterodine, and azilsartan medoxomil, was characterized through standard comparison (Supplementary Fig. The metabolome of iloperidone is detailed in Fig. 4. n = 3 biological replicates. Data are means ± SEM. Statistical significance was determined using two-tailed t-test. (a) Selected 12 isolates from bacterial collection represent their corresponding clades for the examination of drug metabolism. Proteobacteria, red; Actinobacteria, purple; Bacteroidetes, green; Firmicutes, blue. Mono bacterial drug metabolism for azilsartan medoxomil (b), olmesartan medoxomil (c), sarpogrelate (d), fesoterodine (e), selexipag (f), ozanimod (g), GSK1292263 (h), ponesimod (i), and iloperidone (j). No significant O-demethylation metabolism of the three drugs (trimebutine, trimethobenzamide, and SB756050) was detected in the 12 selected bacterial isolates. Further details are provided in Supplementary Fig. n = 3 biological replicates. Data are means ± SEM. (a) Iloperidone metabolism pathways by human gut commensal bacteria described in this study. (b) Reported biosynthetic pathway for indolethiophene skeleton in Streptomyces sp. (c) Reported quinolizidine alkaloids in the biosynthesis of lupins. (a) Chemical transformation route for generating a GSK1292263 metabolite, GSK-459. (b) Structural elucidation of synthetic GSK-459 by NMR analyses. (c) LC-MS-based retention time comparison between synthetic standard (trace A in blue), drug metabolite from bacterial transformation (trace B in black) and co-injection (trace C in red). The EIC (with a ± 20 ppm window) for GSK-459 ion was determined using [M + H]+, m/z 459.2061. (d) MS2 similarity comparison with standard obtained from chemical reaction (top spectrum in blue) and drug metabolite from bacterial transformation (bottom spectrum in red). The predicted fragments from MS2 are indicated in panel b. (e) GPCR activity test (GPR119 agonist) on both GSK1292263 and its inactive metabolite GSK-459. n = 3 biological replicates. Data represents mean ± s.d. (f) Chemical structures of iloperidone and its metabolites. All metabolites were isolated from bacterial transformation culture broth, with their structures determined using NMR, excluding metabolite ilo-477, which was inferred by comparative tandem MS and isotope labelling studies. (g) Key 2D NMR (COSY and HMBC) correlations for the structural assignments of ilo-430. (h) GPCR activity test (DRD2 antagonist) on iloperidone and its metabolites. n = 3 biological replicates. Data represents mean ± s.d. The metabolism of Iloperidone is summarized in Extended Data Fig. The chemical structures of azilsartan medoxomil (a), olmesartan medoxomil (b), ponesimod (c), clopidogrel (d), selexipag (e), risperidone (f), candesartan cilexetil (g), sarpogrelate (h), trimebutine (i), nicergoline (j), trimethobenzamide (k), MK-0557 (l), ozanimod (m), doxazosin (n), fesoterodine (o), SB756050 (p), GSK1292263 (q) and their metabolites are listed. The chemical structures of SB756050 metabolites, including SB-487, SB-473, and SB-459, were determined through 2D NMR analysis of materials isolated from the culture broth. A chemical synthesis approach and/or standard comparison was used to determine the structures of the metabolites of GSK1292263 (GSK-459), olmesartan medoxomil (olmesartan), azilsartan medoxomil (azilsartan), selexipag (ACT-333679), sarpogrelate (sarpogrelate-330), and fesoterodine (fesoterodine-342). MS/MS analysis was used to predict the chemical structures of ponesimod-463a/463b, trimebutine-374, trimethobenzamide-361, trimethobenzamide-347, ozanimod-407, ozanimod-408, doxazosin-438, doxazosin-424, SB-445, and GSK-460. The chemical structures of the remaining metabolites were proposed based on MS analysis. Ten microbial transformations, including those of azilsartan medoxomil, olmesartan medoxomil, clopidogrel, selexipag, candesartan cilexetil, sarpogrelate, nicergoline, ozanimod, doxazosin, and fesoterodine, were conserved with the transformations observed in human metabolism. NMR data of nine identified compounds. The properties of all GPCR drugs studied in this Article. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Wu, Q., Song, D., Zhao, Y. et al. Activity of GPCR-targeted drugs influenced by human gut microbiota metabolism. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemistry (Nat. © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250402160651.htm'>Study finds nearly five-fold increase in hospitalizations for common cause of stroke</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-03 00:28:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Cervical artery dissection is a tear in an artery in the neck that provides blood flow to the brain. Such a tear can result in blood clots that cause stroke. A dissection in the artery wall is most often caused by trauma due to motor vehicle accidents but can also occur with smaller injuries. Heavy lifting has also been shown to cause dissection in some people. "Cervical artery dissection is an important cause of stroke, especially in people under 50, so it is crucial to detect it right away," said Shadi Yaghi, MD, of Brown University in Providence, Rhode Island. "Strokes that are not fatal can lead to long-term disability, poor mental health and reduced quality of life. Our research found a dramatic increase in the number of hospitalizations for cervical artery dissection with rates rising steadily year over year." They then calculated the average annual percentage change in those rates. Researchers found the number of dissections increased from 11 cases per one million people in 2005 to 46 cases per one million people in 2019, with an average annual increase of 10%. Results were similar for both female and male participants. "Possible reasons for this nearly five-fold increase over 15 years include greater awareness of cervical artery dissection by health care professionals, better access to imaging to help identify it and an overall increase in this condition for which a cause has yet to be determined," said Yaghi. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            