
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-10</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09374-4'>Fluctuating DNA methylation tracks cancer evolution at clinical scale</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 18:14:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour. Cancer development is an evolutionary process1,2; consequently, the evolutionary history of a cancer may set its future trajectory and allow inference of the clinical path of a patient5. However, testing this hypothesis directly is challenging because longitudinal patient samples are required to document evolutionary history. Consequently, evolutionary histories are typically inferred from single timepoint data; for example, somatic (epi)mutations are patterned in distinctive ways by differing evolutionary dynamics3. In the haematological system, genome sequencing of single cells or single-cell colonies have been used to infer the phylogenetic relationships among cells6,7,8. The expense of this approach has restricted analyses to small numbers of cases, limiting suitability for clinical translation. DNA methylation can serve as a lineage marker, recording the clonal architecture of cell populations9,10,11,12,13 or the proliferative history14,15. We have recently identified DNA methylation at some CpG sites, which stochastically fluctuates over time at a timescale measured in years4. These fluctuating CpGs (fCpGs) function as a ‘methylation barcode', providing a low-cost strategy to provide high temporal resolution lineage tracing in patient samples4. In this study, we constructed a quantitative modelling framework called evolutionary inference using fluctuating methylation (EVOFLUx). This framework enables precise quantitative inference of the evolutionary history of cancer cells from input fCpG data derived from clinical specimens, at scale (Fig. a, Schematic representing the study design. Bulk cancer tissue samples were collected, DNA extracted and methylation arrays were performed. Evolutionary dynamics were inferred and these were correlated with clinical variables and outcomes. The illustrations of the body, tumour, eppendorf tube and machine were reproduced courtesy of NIAID. The illustration of the physician was adapted from Science Figures, under an Open Design Licence 1.0. The illustration of the screen was created by Simon Dürr under a CC0 1.0 licence. b, fCpG methylation patterns reflect population evolutionary dynamics. Three scenarios are depicted: in a polyclonal population with a distant MRCA, diverse fCpG methylation (unmethylated in white, heterozygous in grey and methylated in black) results in a unimodal distribution of average methylation near the steady state (top). Following a recent clonal expansion (cell number 3 of the top panel), identical fCpG methylation across cells yields a characteristic ‘W-shape' in the bulk methylation distribution (middle). Post-bottleneck, ongoing fluctuations generate diverse fCpG methylation patterns, changing the distribution of bulk methylation values (bottom). c, A hierarchically clustered heatmap of the 978 fCpGs identified in lymphoid cells (n = 2,204 samples). Magnified regions represent the homogeneous intermediate methylation pattern in normal lymphoid cells and the speckled pattern in cancer samples. PBMC, peripheral blood mononuclear cell; MGUS, monoclonal gammopathy of undetermined significance; MM, multiple myeloma; NOS, not otherwise specified; RT, Richter transformation. d, Example histograms of fCpG methylation distributions from healthy and neoplastic lymphoid cells (CLLs). e,f, Heatmaps showing the log2 fold change (FC) in genomic locations (e) and chromatin status (f) of fCpGs compared with non-fCpGs. g, Comparison between fCpG-associated and non-fCpG-associated genes in a single CLL sample (left; P = 5.14 × 10−12, Wilcoxon test, sample SCLL-328), and expression of fCpG-associated genes separated by discretized allele methylation status (right; P values were determined by two-sided Wilcoxon test, no multiple correction; n = 505 for fCpG genes and n = 15,736 for non-fCpG genes). EVOFLUx works by considering the heterogeneity of fCpG methylation values within a sample. At a diploid locus, each fCpG can take one of three states: neither allele methylated, one allele methylated or both alleles methylated (0%, 50% or 100% methylated, respectively), so n fCpG sites can take 3n possible methylation patterns. fCpGs fluctuate methylation status independently, meaning that they function as an ‘evolving barcode' to track clonal evolution: two somatic cells with close ancestry will share a near-identical pattern of fCpG methylation, whereas distantly related cells will have divergent fCpG methylation patterns (Fig. Therefore, the precise distribution of fCpG methylation is determined by the evolutionary history of the population, meaning that mathematical modelling can be used to recover the evolutionary history of a sample from input fCpG data. Here we focused on lymphoid neoplasms, which cover a broad spectrum of diseases and subtypes with highly variable clinicobiological features, from highly proliferative acute disease to indolent chronic leukaemia, arising in infants to older adults, with tumour samples across disease stages16,17. These tumours have been extensively profiled by DNA methylation arrays, which have provided insights into their cellular origin, pathogenesis and clinical behaviour18. Although their temporal clonal dynamics has been partially analysed in few patients19,20, their precise evolutionary histories remain poorly characterized. Applying EVOFLUx to 1,976 well-characterized lymphoid malignancies, we precisely measured individual tumour evolutionary history and show that these histories are associated with disease outcome. We assembled bulk Illumina methylation array data of normal and neoplastic lymphoid cells from 2,430 samples14,21,22,23,24,25,26,27,28,29,30 (Methods; Supplementary Tables 1–3). Following quality control, we retained 2,204 samples from 2,054 patients and 389,180 CpGs. As fCpG loci are tissue specific4, we constructed a pipeline to identify lymphoid-specific fCpGs (Methods; Extended Data Fig. 1a) using 1,471 samples from multiple lymphoid tumour entities (Supplementary Tables 1 and 2). We identified 978 pan-lymphoid cancer fCpGs (Supplementary Table 4). Methylation at fCpGs shows a characteristic ‘speckled' pattern across cancers (Fig. 1c) because (de)methylation occurs independently in each tumour, in stark contrast to the orderly patterns observed for traditional methylation clocks or a random subset of CpGs (Extended Data Fig. fCpGs did not cluster the samples by disease (Fig. 1c), and there was also no clustering based on disease subtype or array platform (Extended Data Fig. 1e–i), except for some B and T cell acute lymphoblastic leukaemias (B/T-ALLs) and multiple myeloma cases that exhibited global hypermethylation or hypomethylation, respectively, as previously reported14. This is consistent with fCpGs behaving as a stochastic ‘barcode' encoding lineage information. By comparison, methylation at CpGs excluded by our selection filters either did cluster by disease, had very low heterogeneity across samples or had unequal methylation and demethylation rates (Supplementary Figs. We then examined the methylation value distribution of these 978 fCpG in individual samples (Supplementary Table 5). In each cancer sample, the fCpGs followed a characteristic ‘W-shaped' distribution that depicts the fCpG methylation pattern of the founder cell of the cancer sample (Fig. By contrast, the healthy B cell subpopulations, which were not included in the discovery set, had unimodal distributions with intermediate methylation levels consistent with these being polyclonal populations (that is, average of the three methylation states; Fig. Methylation values across fCpGs were uncorrelated, except for a small number of fCpGs located within 1 kb of another fCpG (74 of 978; Extended Data Fig. In whole-genome bisulfite sequencing (WGBS) data of sorted bulk B and T cell populations31, methylation at fCpG loci in these polyclonal normal samples was largely intermediate. Over a small window of 100 bp, as the distance from the fCpGs increased, an increasing fraction of the neighbouring CpGs were either hypermethylated or hypomethylated (Extended Data Fig. Together, these analyses suggest that the local 3D genome structure influences (de)methylation processes. We sought to verify fCpGs as ‘evolving barcodes'. Analysis of fCpG methylation fluctuation over time confirmed that inter-patient fCpG heterogeneity was not caused by common single-nucleotide polymorphisms (SNPs; Methods; Extended Data Fig. We generated long-read nanopore sequencing32 on normal B cells (n = 6) and matched chronic lymphocytic leukaemia (CLL)-Richter transformation samples (n = 2 pairs) to simultaneously detect genetic mutations and DNA methylation, and confirmed that fCpG methylation variation is not a consequence of underlying somatic mutation (Extended Data Fig. In matched data, fCpG methylation levels measured by bead array or long-read sequencing were highly concordant (Extended Data Fig. 2e), and similar excellent concordance was observed in additional WGBS data (Extended Data Fig. We constructed fCpG methylation haplotypes using long-read sequencing (Extended Data Fig. 2g) and additional single-cell reduced representation bisulfite sequencing6, and detected lower intra-haplotype heterogeneity within CLL samples than normal B cell samples (Extended Data Fig. 5), consistent with the leukaemia being a clonal expansion, whereas normal B cells are polyclonal. We utilized somatic copy number alterations in 492 CLL and 85 mantle cell lymphoma (MCL) samples26,33 (Supplementary Tables 6 and 7) to distinguish between alleles and show the (de)methylation at fCpGs occurred independently on each allele (Extended Data Fig. 3a), despite copy number alterations being rare in our cohorts (Supplementary Fig. Thus, fCpGs show independent ongoing allele-specific changes to methylation, uniquely labelling cell lineages. As the DNA methylome is influenced by age34, we tested whether fCpGs showed evidence of age-dependent epigenetic modulation. In normal blood samples, mean fCpG methylation was not correlated with age, suggesting that fluctuations continue throughout life, whereas fCpG methylation variance increased with age (Extended Data Fig. Variance is higher in samples where there has been a recent clonal expansion (that is, homozygous methylated or unmethylated alleles become more prominent; Extended Data Fig. 3c), suggesting that fCpGs were detecting age-related clonal expansions of cells of the haematopoietic system4,35,36. We analysed the genomic features of fCpG sites. fCpGs were enriched on the shores of CpG islands (Fig. 1e), underrepresented in gene-associated regions (Extended Data Fig. 3d) and, notably, were distinct from CpGs used in other epigenetic clocks (Extended Data Fig. At the chromatin level, fCpGs were enriched in normal and neoplastic B cell weak promoters and enhancers as well as H3K27me3-marked regions, and significantly underrepresented in active promoters and H3K36me3-marked regions (Fig. RNA-sequencing analysis of CLL samples demonstrated that genes associated with fCpGs have significantly lower expression levels (Fig. 3f), with no association between fCpG methylation status and associated gene expression in matched cases (Fig. No correlation was observed between fCpG methylation and the expression of key DNA methylation modifier genes (Extended Data Fig. Pathway enrichment analysis revealed that fCpG-associated genes were underrepresented in pathways ubiquitously expressed across multiple tissue types but enriched in developmental pathways (Supplementary Tables 10 and 11). Although these results do not provide a detailed molecular understanding of the mechanisms underpinning fCpG fluctuation; together, they indicate that fCpGs tend to be located in silent regions of the genome and do not regulate transcription, so are likely to be neutral lineage markers. We developed EVOFLUx, a stochastic mathematical modelling and Bayesian inference framework, to simulate how clonal evolution quantitatively determines fCpG methylation values and enable inference of evolutionary history of individual tumour samples (Fig. a, Schematic of the EVOFLUx method that simulates the evolutionary dynamics of a growing cancer. EVOFLUx relies on a mathematical model describing how the fCpG distributions vary with the evolutionary parameters of a cancer. The model is split into two phases: before the MRCA (τ) in which methylation changes occur in the single-cell lineage that is subsequently transformed, and following the MRCA in which the population grows exponentially (θ). At each time step, epigenetic switching is allowed to occur between the three possible states (rates μ, ν, γ and ζ). b,c, Simulated fCpG methylation distributions for cancer with a distant versus recent MRCA (b), and rapid versus slow growth (c). d, Fisher plot showing the change in the subclonal composition within a simulated set of longitudinal samples (top). Scatter plots showing the marginal fCpG methylation distribution between simulated pairwise longitudinal samples with the subclonal proportions depicted in each corresponding sample timepoint (n = 1,000 simulated fCpGs; bottom). e, Histogram overlaying the fCpG methylation distributions of data simulated at T1 (neutral) and T3 (subclonal; left), and comparison of the fraction of fCpGs at intermediate values between T1 and T3 (right; the P value was determined by chi-squared test). EVOFLUx simulates the ongoing gain and loss of methylation at fCpGs within a lineage from the birth of a patient until the beginning of a cancer-associated clonal expansion at some specified time, and then continues to simulate methylation fluctuations within the growing population of cancer cells until the cancer sample was collected at time T (Methods; Fig. Cancer growth rate per year (θ), assuming an exponentially growing population. Four parameters corresponding to the four possible transitions between homozygous unmethylated, heterozygous methylated and homozygous methylated (μ, ν, γ and ζ). By combining the cancer growth rate and age, the cancer effective population size (Ne) — the number of long-lived lineages in the cancer — is calculated as Ne = eθ(T −τ). Computational simulations of the model recapitulated the observed W-shaped fCpG methylation distribution. Altering model parameters caused notable shifts in the distribution: increasing cancer age caused the flanking peaks of the W-shaped distribution to move towards the central peak, whereas slower growth broadened peak width (Fig. Hence, the distribution of fCpGs encoded the evolutionary history of a tumour. We added simulations of a single advantageous subclone within the cancer (Methods). Sampling longitudinally from model simulations and comparing fCpG methylation between timepoints showed that subclone outgrowth was marked by the small number of fCpGs with distinct methylation status in the subclone, becoming detectable only when the subclone was sufficiently large (Fig. EVOFLUx contained an extensive Bayesian inference method to learn model parameters from input fCpG methylation distribution data, accounting for tumour purity and the technical noise introduced by the methylation array (Methods; Fig. We generated simulated fCpG distributions with prespecified (that is, known) model parameters and validated the ability of EVOFLUx to accurately recover the ‘ground truth' in these simulated data (Extended Data Fig. 4a,b and Supplementary Information), even when the assumptions underlying the method were weakened (Extended Data Fig. a, Schematic showing the EVOFLUx Bayesian inference pipeline used to infer patient-specific tumour evolutionary history from input bulk methylation data, patient age and tumour purity. The illustration of human ageing was adapted from Pixabay under a CC0 1.0 licence. b, Scatter plot of the inferred growth rate (θ) versus effective population size (Ne = eθ(T − τ)) per sample from patients with cancer, using the posterior median as a descriptive summary statistic of the probability distribution inferred for each evolutionary variable (n = 1,885 samples; coloured according to cancer type). c, Scatter plot of the inferred time since the MRCA (τ) versus the mean epigenetic switching rate (averaged across different rate parameters) per sample (n = 1,885 samples). d–f, Inferred growth rate (top) and effective population size (Ne; bottom) of individual cancer samples separated by molecular subtype in B-ALL (d; two-sided MWU test, n = 767 samples), MCL (e; two-sided MWU test, n = 126 samples) and CLL (f; two-sided MWU test, n = 702 samples). Differences between subtypes were tested using MWU tests, with Holm–Bonferroni corrections applied. In panel d, B-ALL cases with MLL rearranged had significantly higher growth rate than each of the other B-ALL subtypes (all P < 0.005); however, for ease of presentation, only the comparison with t(1;19) is shown here. g, Inferred growth rate (top) and effective population size (bottom) for CLL cases with a TP53 driver mutation in each IGHV CLL subtype (two-sided MWU, n = 646 samples, multiple corrected using false discovery rate together with analysis of other driver mutations; Extended Data Fig. We applied EVOFLUx to 1,976 samples of lymphoid cancers (including T-ALL, B-ALL, CLL, MCL, diffuse large B cell lymphoma (DLBCL) and multiple myeloma) and premalignant conditions (that is, monoclonal B cell lymphocytosis (MBL) and monoclonal gammopathy of undetermined significance) for which we had age and tumour cell purity information, to infer the individual growth rate of each cancer, time since the most recent common ancestor (MRCA) and epigenetic switching rates (Supplementary Table 12). Posterior distributions were well formed (Extended Data Fig. 5a) and posterior predictive distributions recapitulated the input data well (Extended Data Fig. Inferred parameters were not significantly affected by tumour cell content of samples (Supplementary Fig. 8) or exclusion of copy number alteration-altered regions (Extended Data Fig. Most parameters were also insensitive to the number of fCpGs excluded (Extended Data Fig. 5f,g), except the effective population size (Extended Data Fig. Paediatric ALL and adult lymphoid neoplasms exhibited markedly different evolutionary histories (Fig. ALLs demonstrated much higher growth rates (θ; Extended Data Fig. 6a; P = 9.3 × 10−306, Mann–Whitney U (MWU) test, Holm–Sidak correction), smaller effective population sizes (Ne; Extended Data Fig. 6b; P = 8.1 × 10−25) and shorter times since the MRCA (Extended Data Fig. 6c; P = 6.0 × 10−306) than other lymphoid malignancies. T-ALL grew faster than B-ALL (P = 0.0017, Holm–Sidak correction) and showed more homogenous growth rates (P = 0.00044, Levene test). In adult cancers, MBL (a precursor to CLL) displayed lower growth rates and longer time since the MRCA than CLL (Extended Data Fig. DLBCL notably had the largest Ne despite comparable growth rates. fCpG switching rates varied significantly across diseases, with paediatric ALLs showing much faster switching than adult malignancies (Fig. Paediatric ALLs also demonstrated a strong positive correlation between fCpG switching rate and growth rate (Extended Data Fig. These findings suggest that fCpG (de)methylation rates are decreased in adult lymphoid cancers. In CLL, we estimated the ‘contemporary' growth rate derived from multiple longitudinal clinical measurements of the lymphocyte count preceding treatment. We examined how cancer evolutionary history related to molecular subtypes. In B-ALL, MLL-rearranged cases had a significantly higher growth rate (Fig. 3d; P = 1.3 × 10−13, MWU, 44.3 ± 6.1 versus 11.7 ± 0.2 per year (mean ± standard error)), but lower effective population size Ne (P = 3.7 × 10−7, 1.8 × 105 ± 0.2 × 105 versus 3.0 × 105 ± 0.06 × 105 cells) than the other subtypes, consistent with their distinct clinical behaviour37. In MCL, the generally more indolent leukaemic non-nodal MCL26 had a lower growth rate (Fig. 3e; P = 1.1 × 10−3, 1.7 ± 0.1 versus 2.1 ± 0.1 per year) and Ne (P = 7.4 × 10−5, 4.7 × 105 ± 1.4 × 105 versus 1.5 × 106 ± 0.2 × 106 cells) than the more aggressive conventional MCL. In DLBCL transcriptomic subtypes38, there was no significant differences, probably due to the smaller number of cases and the lower sample purity (Supplementary Fig. In CLL, two major molecular subtypes are defined based on the extent of somatic hypermutation in the heavy-chain variable region of the IG gene (IGHV): unmutated CLL (U-CLL) and mutated CLL (M-CLL). The more aggressive U-CLL subtype39,40 showed significantly higher growth rates (Fig. 3e; P = 1.3 × 10−32, 2.3 ± 0.04 versus 1.8 ± 0.02 per year) and larger Ne (P = 2.1 × 10−22, 7.2 × 105 ± 0.3 × 105 versus 4.1 × 105 ± 0.3 × 105 cells) than M-CLL, independent of tumour purity (Supplementary Fig. Similar results were obtained when analysing its precursor condition MBL (Supplementary Fig. Patients with mutations in specific driver genes, such as TP53, are well known to have a worse prognosis41. We compared the inferred growth rates and effective population sizes accounting for IGHV status for the most prevalent driver genetic alterations in CLL: TP53, SF3B1, NOTCH1, ATM, POT1 and IGLV3-21R110, del(11)(q22.3), del(13)(q14.3), del(17)(p13.1) and trisomy 12 (Fig. Patients with M-CLL with TP53 mutations had a higher growth rate and effective population size (P = 0.030 and P = 0.036, respectively, MWU test, false discovery rate corrected; Fig. Trisomy 12 was associated with increased effective population size in both U-CLL and M-CLL (P = 0.036 and P = 0.036, respectively), but no difference in growth rate. As new advantageous subclones can arise during cancer evolution, we also examined subclonal architecture in our cohort. In CLL, a small fraction of cases presents two or more clones with independent origins42. 8a) architectures are distinguishable by characteristic patterning of mutation allele frequencies43,44. Similarly, in fCpG data, clonal architectures are depicted by additional intermediate peaks in the methylation distribution (Fig. Simulations showed that subclone inference by EVOFLUx was limited to detect only strongly selected subclones arising at an intermediate timepoint in the history of the tumour (Supplementary Fig. 13 and Supplementary Information), for reasons analogous to limitations of subclone detection in genetic sequencing data45. We describe the evolution in tumours without detectable subclones as effectively-neutral. Applying EVOLFUx in our cohort revealed that most cancers (1,610 of 1,976) showed no evidence of either subclonal or independent clones (Extended Data Fig. The frequency of subclone detection varied considerably between cancer types, ranging from over 30% in CLL (232 of 718) to less than 5% in DLBCL (1 of 57). We verified EVOFLUx inferences with matched whole-exome sequencing (WES) data from 425 CLL cases (Supplementary Table 13). Using the MOBSTER subclonal deconvolution tool43, subclones were detected in 78 of 425 cancers (Supplementary Table 14), and these cancers had significantly higher EVOLUFx subclonality weights (P = 2.0 × 10−4, MWU test; Extended Data Fig. MOBSTER was more likely to detect subclones in cancers with more mutations (Extended Data Fig. 8d), suggesting limited power to detect subclones in WES. We therefore obtained matched whole-genome sequencing (WGS) data for 127 CLL samples (Supplementary Table 15) and observed better agreement between EVOFLUx and MOBSTER subclone calls (P = 3.9 × 10−4; Extended Data Fig. 8e and Supplementary Table 16), in which MOBSTER subclone calls were then independent of single-nucleotide variant count (P > 0.05). A classifier to predict WGS subclone architecture using EVOFLUx outperformed a WES-based classifier (area under the curve (AUC) = 0.73 versus AUC = 0.62) and performed equivalently to a classifier using both EVOFLUx and WES (AUC = 0.74; Extended Data Fig. Hence, EVOFLUx was more effective at detecting ongoing subclonal selection than MOBSTER applied to WES data. CLLs with two independent clonal origins were detected in 22 of 718 cases. Validation through comparing IG gene rearrangements from WES or WGS and RNA sequencing46 (Supplementary Table 17) showed that patients with multiple IG gene rearrangements had elevated independent origin model weightings (Extended Data Fig. Some patients with CLL undergo Richter transformation, the emergence of an aggressive phenotype with dismal prognosis. We assembled longitudinal matched WGS and methylation data27 for two patients with CLL developing Richter transformation followed for 19.5 and 14.5 years, respectively. WGS data provided ground-truth measurement of clonal evolution during the decades of longitudinal follow-up, which we contrasted with clonal inference from methylation data (Fig. a,b, Timelines and Fisher plots derived from WGS of two patients with CLL with longitudinal samples, annotated with treatment received (top). The Richter-transformed clone is shown in puce. Scatter plots showing the marginal fCpG methylation distribution between pairwise samples from above (middle). Reconstructed phylogenies of the relationship between samples, annotated with the sample timepoints (bottom). The methylation fraction of the 978 fCpG loci are presented as heatmaps (0% blue and 100% red). Benda.-Obi, bendamustine, obinutuzumab; CLB, chlorambucil; CLB-R, chlorambucil and rituximab; CP, cyclophosphamide and prednisone; R-CVP, rituximab, cyclophosphamide, vincristine and prednisone; RFCM, rituximab, fludarabine, cyclophosphamide and mitoxantrone. c, An example of longitudinal samples (SW-BCP-ALL-375) showing the development of the fCpG distribution from diagnostic B-ALL, through remission and relapse. d, Scatter plot showing the marginal fCpG methylation distribution between diagnosis and relapse from panel c. In CLL case 12, between T1 and T2 (13.1 years), WGS data showed that a subclonal expansion occurred, and this was mirrored in fCpGs (Fig. In the short (1 month) period between T2 and T3, the patient received ibrutinib treatment, but there was no clonal expansion detected in WGS nor methylation data. The patient then received rituximab, cyclophosphamide, vincristine and prednisone (R-CVP) combination treatment, but by T4 (5.6 months after T3) presented a clinical manifestation of Richter transformation, and WGS showed that the Richter transformation clone had expanded to form 77% of the tumour. This very large clonal expansion was clearly evident in fCpG methylation data. In CLL case 19, the initial samples at T1–T4 spanned a period of 7.1 years. WGS showed gradual expansion of a subclone that was mirrored in fCpG methylation data (Fig. At T5 (5.5 years later), there had been a large nested subclonal expansion detected by WGS that was also evident in fCpG data. At T6 (2.8 years later), the patient was diagnosed with Richter transformation. WGS showed near-fixation of the Richter transformation clone and there was a correspondingly stark signal in fCpG data. We used EVOFLUX on these longitudinal fCpG methylation data to construct phylogenetic relationships between samples (Methods). In both cases, the phylogenies (Fig. 4a,b) showed that the Richter transformation clone diverged exceptionally early, roughly a decade before the MRCA of the samples containing non-transformed CLL cells (9 and 12 years for cases 12 and 19, respectively). This was consistent with our previous analysis of single-cell RNA sequencing and DNA sequencing that detected Richter transformation cells at low frequencies within the diagnostic CLL sample27, but suggests, remarkably, that the initial Richter transformation divergence occurred well before diagnosis, over 30 years before the clinical presentation of Richter transformation. We validated the fCpG phylogenetic inferences by comparing them with phylogenetic trees from matched WGS data27 and comparing it with methylation data (Methods; Extended Data Fig. fCpGs exactly recapitulated the WGS tree topology and had highly similar branch lengths unlike other CpG sets (Supplementary Figs. Hence, fCpGs are a high-resolution phylogenetic character. We also performed phylogenetic analysis on B-ALL cases from diagnosis to relapse (Extended Data Fig. All relapse samples formed a separate clade from the initial diagnostic sample, suggesting a major treatment-induced evolutionary bottleneck. In patients with B-ALL with matched cancer–remission samples, we consistently observed that the W-shape (indicating a clonal expansion) was replaced by a unimodal distribution (indicating no clonal expansion) following successful treatment, with similar variance as normal blood (Extended Data Fig. In two patients with matched diagnosis, remission and relapse samples, we found that the unimodal fCpG distribution during remission was replaced with a W-shaped distribution at relapse similar in shape to the diagnostic sample, due to the clonal expansion driving recurrence (Fig. Comparing the fCpG distributions between diagnostic versus relapse samples revealed subclonal evolution through treatment (Fig. To investigate the relationship between the evolutionary history and future clinical trajectory of a cancer, we leveraged a series of 478 CLL cases with well-annotated follow-up data. Using univariate Cox models, we tested the effect of evolutionary parameters on time to first treatment (TTFT), which reflects the natural cancer biology, and then on overall survival, which is a more complex end point as it convolves disease biology with treatment responses. In univariate analysis, faster growing CLLs had markedly shorter TTFT (Fig. The Ne of a cancer did not have a strong effect on TTFT (P = 0.058, HR = 1.17), but was associated with shorter overall survival (P = 1.3 × 10−4, HR = 1.41). 17), so unsurprisingly, older patients had worse overall survival (P = 2.3 × 10−11, HR = 1.79). The decrease in risk of progression with cancer age (P = 3.8 × 10−17, HR = 0.65), measured by the time since the MRCA, was probably due to confounding with the growth rate, as these parameters were negatively correlated (Supplementary Fig. The epigenetic switching rate parameters were largely uninformative of prognosis. a, Univariate survival analysis of the TTFT (blue) and overall survival (OS; red) in the discovery CLL cohort for evolutionary variables inferred via EVOFLUx. b, Kaplan–Meier curves comparing the TTFT between patients with high versus low inferred cancer growth rates, separated by IGHV mutational status. c, Multivariate Cox regression model of the TTFT shows that the cancer growth rate is significant when controlling for IGHV status, TP53 alterations and age at sampling. The error bars represent 95% confidence intervals. A log-rank test was used in the Kaplan–Meier curves and Wald tests for Cox models. No multiple comparison adjustments were done. As the growth rate was different between U-CLL and M-CLL (Fig. 5b), we analysed its prognostic impact within each group separately. Higher growth rates consistently correlated with shorter TTFT in both subtypes (Fig. In a multivariate Cox regression model, growth rate maintained a strong independent prognostic impact as quantitative variable (P = 2.2 × 10−10, HR = 2.28) even when controlling for the IGHV mutational status and TP53 aberrations, as well as age (Fig. Of note, the effect of TP53 mutations on TTFT appeared mediated by increased growth rate. The cancer Ne was more significantly correlated with overall survival than the growth rate, and this effect was preserved in the U-CLL subtype (Extended Data Fig. 10a; P = 0.55 for M-CLL, P = 9.90 × 10−7 for U-CLL and overall P = 4.72 × 10−9) and in the multivariate setting with IGHV status, TP53 aberrations and age at sampling (Extended Data Fig. Although the inference of the evolutionary parameters on our initial cohort was wholly blinded to the clinical outcomes, we also validated our findings using a second independent cohort of 209 patients with CLL (135 untreated at sampling)28,29 (Supplementary Table 2). These results verified tumour initial growth rate as a predictor of TTFT (Extended Data Fig. Furthermore, the EVOFLUx-derived growth rate was prognostic even when controlling for the contemporary rate of change of lymphocyte counts (P = 0.018; Extended Data Fig. These results demonstrate that the evolutionary parameters inferred from a cost-effective methylation array could have a direct clinical application by contributing to predict the clinical behaviour of patients with CLL independently from well-established prognostic variables. Our study establishes a computational framework called EVOFLUx, which enables quantitative measurement of the evolutionary history of human malignancies at massive scale using only widely available and low-cost bulk methylation data as input. Evolutionary histories are fundamentally distinct from characterizations of the contemporary phenotype of cancer cells, such as the fraction of proliferating cells. EVOFLUx methodology should also work identically for sequencing-based methods to measure methylation such as bifulfite-based47,48 and bisulfite-free approaches32,49 (for example, long-read nanopore), which show an excellent correlation with our array data. In theory, these methods allow for assessment of many more fCpGs, increasing inference accuracy. EVOFLUx should also be applicable to tumour-derived cell-free DNA extracted from blood. Evolutionary histories are strongly associated with disease phenotype and clinical outcomes across lymphoid disease types. We consider this strong evidence that clonal evolution, the fundamental cellular process of disease development, underlies the clinical course of the disease. Consequently, we expect these results to generalize across all cancer types. We note that genome-wide DNA methylation analyses also measure other important biological features of a cancer (for example, molecular subtype14) that could be combined with EVOFLUx-based inference of evolutionary history to further improve the prognostic value of DNA methylation data. In summary, we present a cost-effective high-throughput platform for measuring cancer evolutionary dynamics at the population scale in patient samples. These fundamental measurements of the disease biology hold substantial prognostic value and represent an innovative asset in the field of precision oncology. We assembled and processed with a harmonized pipeline14 (v4.1; see Code availability section) 2,430 bulk sample Illumina methylation array data of normal and neoplastic lymphoid cells from previous publications14,21,22,23,24,25,26,27,28,29,30. As tumour samples, we included precursor 797 B-ALLs and 90 T-ALLs at diagnosis, 28 B-ALLs and 2 T-ALLs at relapse, as well as 74 B-ALLs and 12 T-ALLs at complete remission (that is, normal blood); 149 MCLs; 722 CLLs, 55 of its precursor condition MBL and 6 samples from patients with CLL undergoing a DLBCL transformation called Richter transformation; 62 primary DLBCL, not otherwise specified; and 104 multiple myeloma and 16 of its precursor condition monoclonal gammopathy of undetermined significance. In brief, raw idat files were loaded and processed with R (v4.3.1) using the minfi package50,51 (v1.46.0) in batches as specified in the column ‘SSNOB_NORMALIZATION_BATCH' of Supplementary Table 2. In brief, the data were processed for each batch as follows. First, idats files were loaded into a RGChannelSet object, and minfi quality metrics using the qcReport function were performed, removing samples with unexpected distributions of methylation values (that is, distributions markedly distinct from a bimodal centred around 0 and 1 β-values and/or from the remaining samples) and low signal intensities of internal control probes for each sample, including bisulfite conversions I and II, extension hybridization, hybridization, non-polymorphic, specificities I and II, and target removal probes. Next, further quality metrics were derived using the function minfiQC on the unnormalized RGChannelSet obejct. Those samples with median signal intensities of unmethylated and methylated channels of at least 10.5 in log2 scale were considered as having good signal intensities. Subsequently, detection P values were calculated across all CpGs and samples using the detectionP function for the unnormalized RGChannelSet object. Samples were considered as good if having a mean detection P value across all CpGs of P ≤ 0.01. On a CpG level, we retained CpGs with a detection P ≤ 1 × 10−16 in 90% or more of the samples, which has been shown to improve the quality of downstream analyses52,53. The RGChannelSet object was normalized with the single-sample batch-independent preprocessNoob function with dye bias correction. We next retained only CpGs (excluding CH probes) that did not contain any SNP neither in the interrogated CpGs nor in the probe extension using the dropMethylationLoci and dropLociWithSnps functions with default options (minor allele frequency (MAF) = 0). Further analyses using long-read nanopore data, Illumina array control probes, annotation packages and a data-driven approach were used to ensure the lack of any genetic confounding in the methylation values of the resulting fCpGs (see the next sections). Furthermore, CpGs with any previous evidence of potential cross-hybridization were excluded54 and only CpGs mapping to autosomal chromosomes were subsequently retained for downstream analyses. Finally, to further confirm the accuracy of the filtering criteria, we checked the distribution of normalized methylation values and performed principal component analyses separately for samples passing all quality checks as well as those considered as bad samples. The final DNA methylation matrix contained 2,204 samples and 389,180 CpGs passing all the aforementioned quality controls, and included 2,054 patients (22 technical replicates, 3 synchronic and 125 longitudinal samples from the same patients)55 (Supplementary Table 2). To determine the purity of samples, we used our previously deconvolution strategy to infer tumour cell content by DNA methylation14, which was used as a consensus purity in all the tumour samples except for DLBCL and multiple myeloma. In these two tumour entities, we have previously identified a DNA methylation signature loss causing inaccurate tumour purity predictions using DNA methylation data, and therefore we used available genetic or flow cytometry data for DLBCL and multiple myeloma, respectively. We constructed a pipeline to identify fCpGs in lymphoid tumours, based on the following criteria: Heterogeneous across different participants with the same disease (by accepting CpG loci with the top 5% of standard deviation of methylation value within a cancer type). Equally likely to be methylated or unmethylated (by selecting CpGs with average methylation of approximately 0.5 within a cancer type). Unlikely to be associated with specific cell or cancer types. We used an unsupervised Laplacian score feature selection metric56 to rank CpG loci by their tendency to preserve the nearest-neighbour graph, and accepted the 5% least-informative CpGs. We performed a series of analyses to exclude the potential genetic confounding (germline SNPs and somatic SNVs) on our fCpGs. We first excluded the possibility that common germline SNPs caused methylation heterogeneity at fCpG sites between individuals. We observed very distinct methylation dynamics of array control probes containing SNPs (which had been removed during the initial array processing) versus fCpGs. SNP probes showed the same distribution in all samples (Extended Data Fig. 2c), including longitudinally followed cases (Supplementary Fig. 3), whereas fCpGs only showed a W distribution in cancer samples with ongoing fluctuations over time. Thus, although SNPs reflect the stable genetic identity of the individual, fCpGs reflect the identity of a single cell and its evolving lineage. In addition, we used the packages SNPlocs.Hsapiens.dbSNP155.GRCh38 (v0.99.24) and MafH5.gnomAD.v4.0.GRCh38 (v3.19) to check for any known significant germline or somatic genetic confounding on the resulting 978 fCpGs. We found approximately 60% of fCpGs reported in the gnomAD v4 database (with the array background having approximately 65%), but with a very low MAF (median of 1 × 10−5 and mean of 1 × 10−3). To exclude the possibility of unknown or very rare genetic confounding, we used the data-driven gaphunting algorithm57 available in the minfi R package, which further discarded a possible cancer-specific single-nucleotide variation (SNV) that could confound the methylation values at the 978 identified fCpGs. Finally, Oxford Nanopore long read of a subset of normal and neoplastic samples further validated that fCpGs represent de/methylated cytosines (Extended Data Fig. For long-read methylation sequencing in CLL and Richter transformation samples, concentration was assessed using the Qubit assay and DNA integrity was analysed either with the Femto Pulse System (Agilent) or the Fragment Analyzer (Agilent). When more than 6 µg of material with good integrity was available, DNA was additionally treated with the Short Fragment Eliminator Kit XS (PacBio) and eluted in EB buffer. Approximately 4 µg of DNA was used for library preparation according to the standard LSK114 kit and protocol from Oxford Nanopore. The time for DNA repair and end-prep was increased up to 30 min at 20 °C and 30 min at 65 °C. Adapter ligation was performed for 1 h at room temperature. All elutions were performed at 37 °C for 1.5 h, and 550–600 ng of DNA was loaded onto a FLO-PRO114M (CLL cells) flow cells. Flow cells were washed (EXP-WSH004) after 1–2 days, if pore count decreased to less than 30%. A total of 1–4 washes were performed for each flow cell. Flow cells were run for 100 (CLL cells) hours in total with the Fast model (MinKNOW 23.11.7, Dorado 7.2.13). The raw data were rebasecalled using dorado duplex (v0.5.3) and applying the SUP and modified call to detect 5mC and 5hmC, (model dna_r10.4.1_e8.2_400bps_sup@v4.3.0_5mCG_5hmCG@v1). In normal B cell samples, 1–3 µg of DNA was used for WGS. Libraries were loaded onto a flow cell version FLO-PRO002 (R9.4) and were run for 90–110 h. The basecalling was performed on live mode with the Guppy basecaller (v6.2.7), included in the MinKNOW (v22.08.6), using the SUP model for base modification detection of 5mC and 5hmC (dna_r9.4.1_450bps_modbases_5hmc_5mc_cg_sup.cfg). In all samples, the generated unmapped BAM files after the basecalling were converted to FASTQ files using the SAMtools fastq -T Mm, Ml command. The FASTQ files were then mapped to BAM files using the command minimap2 -ax map-ont -y../GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.mmi. Reads from each strand were combined to generate DNA matrices for each CpG and were used for obtaining the methylation values of all fCpGs. In addition, mini BAM files containing all reads from the 976 fCpGs were generated (in hg38 genome assembly). The reads showed excellent mappability, with a mean of perfect nucleotide matches (NM tag; Levenshtein distance) for all fCpGs across samples of 96.41% (range of 73.31–97.90), and mean mapping quality (MAPQ) of all the reads covering all fCpGs across samples of 59.510 (range of 2–60). Subsequently, long reads were phased using variants called using Clair 3 (v1.0.9, model r941_prom_hac_g360 + g422)58 with the Longphase package (v1.7)59. At fCpGs, only 2.7% of the reads were non-canonical bases (Extended Data Fig. The variant allele frequency (VAF) of these mutations tended to be low and was negatively correlated with the coverage at that site (Supplementary Fig. Hence, the majority of these non-canonical base pairs are probably due to errors in nucleotide assignment. There is also no association between the methylation status of different reads and the variants present within a 50-bp window of each fCpG locus (Supplementary Fig. Hence, assessment of fCpG methylation via bead array was not majorly confounded by miscalled variants. The fCpG methylation patterns seen in the bead array data were replicated in the long-read data (Extended Data Fig. 2e) and the correlation between the fraction methylated measured via bead array and long-read sequencing at fCpGs was excellent (Extended Data Fig. To assess the intra-sample long-read diversity for each sample, the pairwise Hamming distances were calculated between every read on both haplotypes. The two lists of Hamming distances were concatenated, and the mean calculated as a summary statistic of the read diversity for each sample. One normal B cell sample contained only two reads from one haplotype, and zero from the other, and so was excluded from further analysis. Previously published single-cell reduced representation bisulfite sequencing (scRRBS) data were obtained6 and the fCpG methylation values extracted methylation values for normal B cells from 6 donors and CLL cells from 12 patients. There was a high dropout rate, so to extract meaningful patterns we plotted a subset of 40 cells and 20 fCpGs with a high density and overlap of fCpGs across single cells as examples (Supplementary Fig. To compare the full set of data accounting for the high degree of missing data, we used a metric of heterogeneity at a given fCpG that weights by the number of non-missing fCpGs according to: Where ni is the number of non-NaN values for the ith fCpG, \(\frac{n(n-1)}{2}\) is the total possible pairwise comparisons between a set of n objects and σ(βi) is the standard deviation across the methylation values of the ith fCpG (Supplementary Fig. To characterize the genomic and regulatory context of fCpGs, we used a series of statistical analyses and database annotations. We annotated fCpGs using Illumina manifest and other genomic annotation packages available at Bioconductor including IlluminaHumanMethylation450kanno.ilmn12.hg19 (v0.6.1) and IlluminaHumanMethylationEPICanno.ilm10b2.hg19 (v0.6.0). We additionally used the packages SNPlocs.Hsapiens.dbSNP155.GRCh38 (v0.99.24) and MafH5.gnomAD.v4.0.GRCh38 (v3.19) to check any possible germline or somatic genetic confounding on the resulting 978 fCpGs. We found approximately 60% of fCpGs reported in the gnomAD v4 database (with the array background having approximately 65%), but with a very low MAF (median of 1 × 10−5 and mean of 1 × 10−3). In addition, we used the Illumina 450k and EPIC array internal SNP probes and showed a dramatically distinct methylation dynamics compared with fCpGs in single-timepoint (Extended Data Fig. Finally, the data-driven gaphunting algorithm available in the minfi R package was applied with all the previously published thresholds and cut-offs57, which further discarded possible cancer-specific SNV that could confound the methylation values at the 978 identified fCpGs. We used Chi-squared tests to assess the enrichment of fCpGs in distinct genomic regions or elements. We performed gene-set enrichment analysis on the fCpG-associated genes using gProfiler60, specifically focusing on the Gene Ontology biological processes61 and the Human Protein Atlas62. The statistical domain space was limited to genes targeted by at least one CpG in the 389,180 candidate CpG set and significance was determined using the g:SCS algorithm63. Previous chromatin segmentation of normal and neoplastic B cells was used to assess the chromatin-state enrichment of fCpG14,64. fCpGs were checked for their overlap with previous ‘epigenetic clocks', including mitotic14,65,66,67,68, chronological age69,70,71,72,73,74,75,76,77,78, gestational age79,80,81,82,83, biological age and mortality84,85,86 and trait predictors87,88. The package methylclock (v1.10.0) was used to calculate all epigenetic clocks but epiCMIT, which was derived as previously described14. Previously available RNA sequencing data for 294 patients with CLL were obtained33 and processed as previously described26. Matched RNA sequencing data and DNA methylation data for the same patients at the same timepoint were available for 224 patients with CLL. Transcript per million counts were used to represent differential gene expression values across genes and samples. We used the gene annotation provided in the R Bioconductor package IlluminaHumanMethylationEPICanno.ilm10b2.hg19 to classify genes associated with fCpGs. In each methylation sample, the 978 fCpGs were discretized as homozygous demethylated, heterozygous methylated or homozygous methylated (coded as [0,1,2], respectively). This was done by separately fitting a β-mixture model with three components to each sample using Stan89 and extracting the component mixture probability. The gene expression value for genes classified as having and fCpG with 0, 1 or 2 alleles methylated were plotted as previously described. External DNA methylation data were download from the Gene Expression Omnibus database using the GEOquery R package (v2.72.0). For sorted immune cells, these include GSE137594 and GSE184269. For whole-blood samples, these include GSE72773, GSE55763, GSE40279 and GSE36054. Data were analysed with the normalization procedure used in each study together with the metadata provided. Mean and standard deviation for fCpGs were calculated with fCpGs present in the provided normalized matrices. We built a generative computational model of how the patterns of fCpGs vary over time (t) according to the evolutionary history of a cancer. Initially, our model focused on neutral evolution, before expanding to non-neutral modes of tumour evolution below. At time t = 0, the fCpGs were assumed to be equally likely to be homozygously methylated or demethylated. The fCpG status of the MRCA at time t = τ was calculated by applying matrix exponentiation. The second phase of the model consisted of a discrete time Markov process. The effective population size of the growing cancer was modelled as growing according to a deterministic exponential growth equation, Ne = eθ(T − τ). Each fCpG was considered independently; at each time step, t → t + δt, the number of homozygous-methylated (m), heterozygous-methylated (k) and homozygous-demethylated cells (w) at a specific fCpG was updated according to the epigenetic switching rates. At the time of sample, T, the fraction methylation of each simulated fCpG was calculated by summing the number of methylated alleles and normalizing by the total number of alleles in the population: We further accounted for contaminating normal cells and the technical noise introduced by the methylation bead array. Following our previous work, the bead array was assumed to saturate at extreme methylation values, shifting the minimum and maximum methylation by δ and ε, respectively4. The noise of the bead array was assumed to be β-distributed, with precision parameter κ. Alongside our model of neutral exponentially growing cancer populations, we devised two alternative models of cancer growth: A subclonal selection model in which a single cell within the cancer develops a selective advantage and begins to grow at an increased growth rate. An independent clonal origins model, in which a patient has developed two distinct cancers concurrently. We assumed that the initial cancer population began exponentially growing at τ1 as above, but at time t = τ2, we selected a single cell with a set of fCpG states drawn according to the cancer population and allowed this second population to grow concurrently with a growth rate θ2. The independent-cancer model followed the same scheme as the nested subclonal selection model, except the methylation status of the emerging cancer was that of an independent cell that experienced random fluctuations between t = 0 and t = τ2. If we let the number of cells in the less fit subclone in each methylation state be {m1, k1, w1} and in the fitter subclone be {m2, k2, w2}, following the convention above, then in both cases the measured methylation patterns at the time of sample are: We modified the simulations of how the fCpG methylation distribution changes over time to allow for multiple sequential sample collections. These simulations allow for neutral, independent clones, a single subclonal expansion or two subclonal expansions, which can either be nested or emerge from the clonal trunk in parallel. This required pre-specification of sampling times, along with the emergence times of any subclones or independent clones, which we collected to form a set of ‘landmark times'. The discrete time steps of the simulation were split into phases between the landmark times, which evolved according to the discrete time Markov process outlined above. At each sampling time, the fCpG methylation fraction was calculated as above and stored as a column in the output matrix. For each methylation array blood sample, we had matched age (T) and purity (ρ) information. These parameters are constrained either to be positive (θ, μ, ν, γ, ζ, κ > 0) or to lie within a specified range (0 < τ/T, δ, ε < 1), which we achieved using appropriate prior distributions. To better allow for priors to be set on a biologically meaningful scale, the priors for the log-normal distribution were set in terms of the real scale mean and standard deviation, rather than the standard log-scale. To reduce correlations in the posterior and make sampling more efficient, the variables ν and ζ were normalized by μ and γ, respectively. The priors are as follows: When fitting non-neutral models of tumour growth, the inference was parameterized in terms of the relative growth of the fitter subclone, \({\tilde{\theta }}_{2}=\frac{{\theta }_{2}}{{\theta }_{1}}\), and the fraction of the population consisting of the fitter subclone, \(f=\frac{{e}^{{\theta }_{2}(t-{\tau }_{2})}}{{e}^{{\theta }_{1}(t-{\tau }_{1})}+{e}^{{\theta }_{2}(t-{\tau }_{2})}}\). This parameterization induces less correlation in the resulting posterior, which greatly improves the sampling efficiency. The priors on these additional parameters are: We developed a stochastic estimator of the log-likelihood function at a given set of parameters by simulating the fCpG methylation distribution a large number of times, correcting for the bias inherent with using a finite number of simulations and penalizing the log-likelihood for extreme values of the Ne (see Supplementary Information for details). The standard Bayesian algorithms developed to infer the posterior for a given set of data (for example, Markov chain Monte Carlo (MCMC), nested sampling) are typically used when the log-likelihood is analytically tractable and can be calculated exactly. It has been shown that, as long as the stochastic approximation of the log-likelihood is unbiased, MCMC methods can obtain an exact Bayesian inference of the true posterior, as in pseudo-marginal Metropolis–Hastings90. Here we used a nested sampling approach using the dynesty package91,92,93. Unlike pseudo-marginal Metropolis–Hastings, nested sampling is able to efficiently explore multimodal posterior landscapes (which can occur under the subclonal and independent cancer models). We used an expected log pointwise predictive density94 approach to compare our competing models of evolution for each sample using the arviz Python package95, which uses PSIS-LOO-CV to compare the out-of-sample prediction accuracy between models while naturally penalizing more complex models. The weights of the respective models were calculated using pseudo-Bayesian model averaging using Akaike-type weighting, stabilized using the Bayesian bootstrap96. Previous mutated annotation files from WES46 and WGS27 data were used to further validate our distinct EVOFLUx evolutionary modes (that is, neutral, subclonal and independent) and Richter transformation phylogenies. To detect subclones in bulk WES and WGS data, we used MOBSTER43, which fits the VAF spectrum with a mixture model containing a Pareto distribution to account for the neutral tail97 and a variable number of β-distributions to account for the clonal and subclonal peaks. We ran MOBSTER using the default parameters, except using a minimum 5% VAF threshold and lowering the minimum number of mutations to compose a cluster to five in WES samples due to the low number of mutations. We then manually quality controlled all 377 WES samples and 10 WGS, tuning the fitting parameters to better represent the data (for instance, when the clonal peak had been called at a low frequency despite the median tumour purity being 95%). A novel Bayesian phylogenetic method was used to reconstruct the evolutionary relationships and the time to MRCA of longitudinal samples from the same patients. This was carried out in the BEAST (v1.8.4) framework98,99 using custom models implemented in PISCA100 (v1.1; available from https://github.com/adamallo/PISCA). EVOFLUx provided an estimate of the age of the patient when the MRCA of each bulk sample emerged. To estimate the methylation status of each fCpG at the MRCA of the sample in each of our longitudinal samples, we discretized the fCpGs as described above (see the section ‘CLL RNA sequencing data'). We implemented a four-parameter biallelic binary substitution model analogous to the pre-growth EVOFLUx model in PISCA. This plugin contains all the required statistical machinery to use this model for somatic phylogenetic estimation. The biallelic binary substitution model has three relative rate parameters: (1) heterozygous methylation \(\tilde{\upsilon }\), (2) homozygous demethylation \(\tilde{\gamma }\), and (3) heterozygous demethylation \(\tilde{\zeta }\), where homozygous methylation \(\tilde{\mu }\) was normalized to 1. For all relative transition rate parameters, a log-normal prior with mean of 1 and standard deviation of 0.6 was used, with a half-normal prior with mean of 0 and standard deviation of 0.13 for the molecular clock rate, using a strict clock model for the rate of evolution across the tree. Two demographic tree models, constant population size101 and exponential growth102, were compared by marginal likelihood estimation using path-sampling103 and a constant population model was deemed more appropriate. MCMC chains were run for 100 million generations sampled every 100,000 generations and convergence was assessed using Tracer (v.1.7)104, ensuring effective sample sizes (ESS) greater than 500 for all parameters. Maximum clade credibility trees were then made using 10% burn-in and medium node heights. The resulting trees were plotted using ggtree105. Each bulk sample is represented by a set of clonal mutations found during the deconvolution of WGS data (see above). Where a mutation was deemed absent in the clonal peak, the reference nucleotide was used. Mutational signature assignment106 was used to select mutations in the clock-like SBS1 channel107. BEAST (v1.10)108 was then used with the simple binary substitution model (as SBS1 effectively represents just C-to-T substitutions), a strict clock model, a constant population size prior101 and a flat prior on the age of MRCA (from zero to earliest patient sample), with ancestral state estimation at the root. Chains were run and ESS values assessed as described above. Clinical analyses were performed in CLL for TTFT and overall survival from the time of sampling. Tumour growth rate (θ), effective population size (Ne) and epigenetic switching rates were analysed as continuous variables in univariate Cox regression models for both TTFT and overall survival. The effect size of HRs for each evolutionary variable were analysed considering different scaling factors. In particular, the growth rate was analysed assuming exponential growth (that is, for θ = 1, the population is e = 2.71 times bigger per year), the Ne was considered per million cells, and the cancer age or time from the MRCA was analysed for each 10 years. Individual switching rate parameters (μ, ν, γ and ζ) were largely uninformative of prognosis and were summarized into a mean epigenetic switching rate, which was scaled by a factor of 100. In addition, growth rate and effective population were analysed as continuous variables in multivariate Cox regression models together with TP53 aberrations (considering mutations and deletions together), IGHV gene mutational status and the age of patients at sampling. Kaplan–Meier curves were generated for low and high growth rates and effective population size within IGHV subtypes using maximally selected log-rank statistic using the maxstats package (v0.7-25). P values from Kaplan–Meier curves were derived using the log-rank statistic. Survival (v3.5-7), survminer (v0.4.9) and ggsurvfit (v0.3.1) packages were used under R (v4.3.1). Historical records of the absolute number of lymphocytes in blood obtained via haemocytometer were collected for patients with CLL over the whole disease course (that is, an approximate of the number of malignant CLL cells in blood). In 231 patients with CLL, we could obtain at least 10 sample timepoints (that is, at least 10 medical appointments, median n = 27 and mean n = 34) before the first treatment, allowing us to track the natural history of the disease before treatment intervention for the tumour (Supplementary Fig. We fitted a linear model to all 231 cases and obtained the slope of the observed log number of lymphocytes (that is, the coefficient of the univariate linear model) and compared it with growth rate estimates derived from EVOFLUx. Appropriate multiple test correction, such as the Holm–Sidak correction, is noted when applied. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. No new methylation bead array data were generated in the course of this study. The harmonized and filtered methylation matrix has been deposited to Zenodo55 (https://doi.org/10.5281/zenodo.15479736). External DNA methylation data for sorted immune cells can be found under the accession codes GSE137594 and GSE184269. For whole-blood samples, the accession codes are GSE72773, GSE55763, GSE40279 and GSE36054. CLL gene expression data are available under the accession codes EGAS00001000374 and EGAS00001001306. Chromatin immunoprecipitation followed by sequencing datasets are available from Blueprint (https://www.blueprint-epigenome.eu/) under the accession code EGAS00001000326. Matched WES and WGS data are available under the accession codes EGAS00000000092 and EGAD00001008954, respectively, under controlled access. Pathway analysis was run using the Gene Ontology: Biological Processes release 2023-03-06 and the Human Protein Atlas (v10.0) databases. Matched Oxford Nanopore long-read data were generated for six normal B cells and samples and two CLL-Richter transformation sample pairs. Long-read data are available at the European Genome-Phenome Archive repository under the accession code EGAS50000001192. Source data are provided with this paper. The codes for EVOFLUx used to infer the evolutionary history of cancer samples from methylation array data (https://github.com/CalumGabbutt/evoflux), to curate DNA methylation data and to perform clinical and additional bioinformatic analyses (https://github.com/Duran-FerrerM/evoflux) and for the phylogenetic method (https://github.com/adamallo/PISCA), are available on GitHub. Nowell, P. C. The clonal evolution of tumor cell populations. & Maley, C. C. Cancer as an evolutionary and ecological process. Turajlic, S., Sottoriva, A., Graham, T. & Swanton, C. Resolving genetic heterogeneity in cancer. Gabbutt, C. et al. Fluctuating methylation clocks for cell lineage tracing at high temporal resolution in human tissues. Greaves, M. & Maley, C. C. Clonal evolution in cancer. Gaiti, F. et al. Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia. Lee-Six, H. et al. Population dynamics of normal human blood inferred from somatic mutations. Williams, N. et al. Life histories of myeloproliferative neoplasms inferred from phylogenies. Yatabe, Y., Tavaré, S. & Shibata, D. Investigating stem cells in human colon by using methylation patterns. Hong, Y. J., Marjoram, P., Shibata, D. & Siegmund, K. D. Using DNA methylation patterns to infer tumor ancestry. Brocks, D. et al. Intratumor DNA methylation heterogeneity reflects clonal evolution in aggressive prostate cancer. Hao, J. J. et al. Spatial intratumoral heterogeneity and temporal clonal evolution in esophageal squamous cell carcinoma. Siegmund, K. D., Marjoram, P., Woo, Y. J., Tavaré, S. & Shibata, D. Inferring clonal expansion and cancer stem cell dynamics from DNA methylation patterns in colorectal cancers. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Endicott, J. L., Nolte, P. A., Shen, H. & Laird, P. W. Cell division drives DNA methylation loss in late-replicating domains in primary human cells. de Leval, L. et al. Genomic profiling for clinical decision making in lymphoid neoplasms. Arber, D. A. et al. International consensus classification of myeloid neoplasms and acute leukemias: integrating morphologic, clinical, and genomic data. Duran-Ferrer, M. & Martín-Subero, J. I. Epigenomic characterization of lymphoid neoplasms. Gruber, M. et al. Growth dynamics in naturally progressing chronic lymphocytic leukaemia. Gutierrez, C. et al. Multifunctional barcoding with ClonMapper enables high-resolution study of clonal dynamics during tumor evolution and treatment. Kulis, M. et al. Whole-genome fingerprint of the DNA methylome during human B cell differentiation. Nordlund, J. et al. Genome-wide signatures of differential DNA methylation in pediatric acute lymphoblastic leukemia. Differential DNA methylation in purified human blood cells: implications for cell lineage and studies on disease susceptibility. Lee, S. T. et al. Epigenetic remodeling in B-cell acute lymphoblastic leukemia occurs in two tracks and employs embryonic stem cell-like signatures. Queirós, A. C. et al. Decoding the DNA methylome of mantle cell lymphoma in the light of the entire B cell lineage. Nadeu, F. et al. Genomic and epigenomic insights into the origin, pathogenesis, and clinical behavior of mantle cell lymphoma subtypes. Oakes, C. C. et al. DNA methylation dynamics during B cell maturation underlie a continuum of disease phenotypes in chronic lymphocytic leukemia. Dietrich, S. et al. Drug-perturbation-based stratification of blood cancer. Agirre, X. et al. Whole-epigenome analysis in multiple myeloma reveals DNA hypermethylation of B cell-specific enhancers. Loyfer, N. et al. A DNA methylation atlas of normal human cell types. Continuous base identification for single-molecule nanopore DNA sequencing. Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Seale, K., Horvath, S., Teschendorff, A., Eynon, N. & Voisin, S. Making sense of the ageing methylome. Relationship among three common hematological premalignant conditions. Jaiswal, S. & Ebert, B. L. Clonal hematopoiesis in human aging and disease. Rice, S. & Roy, A. MLL-rearranged infant leukaemia: a ‘thorn in the side' of a remarkable success story. Distinct types of diffuse large B-cell lymphoma identified by gene expression profiling. Hamblin, T. J., Davis, Z., Gardiner, A., Oscier, D. G. & Stevenson, F. K. Unmutated Ig VH genes are associated with a more aggressive form of chronic lymphocytic leukemia. Damle, R. N. et al. Ig V gene mutation status and CD38 expression as novel prognostic indicators in chronic lymphocytic leukemia. Zenz, T. et al. TP53 mutation and survival in chronic lymphocytic leukemia. Multiple productive immunoglobulin heavy chain gene rearrangements in chronic lymphocytic leukemia are mostly derived from independent clones. Caravagna, G. et al. Subclonal reconstruction of tumors by using machine learning and population genetics. Williams, M. J. et al. Quantification of subclonal selection in cancer from bulk sequencing data. Reply to ‘Neutral tumor evolution?'. A. et al. Molecular map of chronic lymphocytic leukemia and its impact on outcome. Leitão, E. et al. Locus-specific DNA methylation analysis by targeted deep bisulfite sequencing. Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis. Simultaneous sequencing of genetic and epigenetic bases in DNA. Aryee, M. J. et al. Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays. Fortin, J.-P., Triche, T. J. Jr & Hansen, K. D. Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi. Lehne, B. et al. A coherent approach for analysis of the Illumina HumanMethylation450 BeadChip improves data quality and performance in epigenome-wide association studies. Zhou, W., Triche, T. J., Laird, P. W. & Shen, H. SeSAMe: reducing artifactual detection of DNA methylation by Infinium BeadChips in genomic deletions. Discovery of cross-reactive probes and polymorphic CpGs in the Illumina Infinium HumanMethylation450 microarray. & Graham, T. Harmonised methylation array matrix related to the article Fluctuating DNA methylation tracks cancer evolution at clinical scale. He, X., Cai, D. & Niyogi, P. Laplacian score for feature selection. Andrews, S. V., Ladd-Acosta, C., Feinberg, A. P., Hansen, K. D. & Fallin, M. D. “Gap hunting” to characterize clustered probe signals in Illumina methylation array data. Zheng, Z. et al. Symphonizing pileup and full-alignment for deep learning-based long-read variant calling. Lin, J. H., Chen, L. C., Yu, S. C. & Huang, Y. T. LongPhase: an ultra-fast chromosome-scale phasing algorithm for small and large variants. Raudvere, U. et al. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). Ashburner, M. et al. Gene Ontology: tool for the unification of biology. Uhlén, M. et al. Tissue-based map of the human proteome. Reimand, J., Kull, M., Peterson, H., Hansen, J. & Vilo, J. g:Profiler — a web-based toolset for functional profiling of gene lists from large-scale experiments. The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia. Correlation of an epigenetic mitotic clock with cancer risk. Teschendorff, A. E. A comparison of epigenetic mitotic-like clocks for cancer risk prediction. & Wang, S. The MiAge Calculator: a DNA methylation-based mitotic age calculator of human tissue types. Zhou, W. et al. DNA methylation loss in late-replicating domains is linked to mitotic cell division. Bocklandt, S. et al. Epigenetic predictor of age. Garagnani, P. et al. Methylation of ELOVL2 gene as a new epigenetic marker of age. Hannum, G. et al. Genome-wide methylation profiles reveal quantitative views of human aging rates. Horvath, S. DNA methylation age of human tissues and cell types. Lin, Q. et al. DNA methylation levels at individual age-associated CpG sites can be indicative for life expectancy. & Gonzalez, A. Simplified assay for epigenetic age estimation in whole blood of adults. Aging of blood can be tracked by DNA methylation changes at just three CpG sites. Improved precision of epigenetic clock estimates across tissues and its implication for biological ageing. Horvath, S. et al. Epigenetic clock for skin and blood cells applied to Hutchinson Gilford progeria syndrome and ex vivo studies. Shireby, G. L. et al. Recalibrating the epigenetic clock: implications for assessing biological age in the human cortex. Prediction of gestational age based on genome-wide differentially methylated regions. Knight, A. K. et al. An epigenetic clock for gestational age at birth based on blood methylation data. Lee, Y. et al. Placental epigenetic clocks: estimating gestational age using placental DNA methylation levels. Accelerated placental aging in early onset preeclampsia pregnancies identified by DNA methylation. The PedBE clock accurately estimates DNA methylation age in pediatric buccal cells. Belsky, D. W. et al. DunedinPACE, a DNA methylation biomarker of the pace of aging. Levine, M. E. et al. An epigenetic biomarker of aging for lifespan and healthspan. Lu, A. T. et al. DNA methylation GrimAge strongly predicts lifespan and healthspan. McCartney, D. L. et al. Epigenetic prediction of complex traits and death. Liang, X. et al. DNA methylation signature on phosphatidylethanol, not on self-reported alcohol consumption, predicts hazardous alcohol consumption in two distinct populations. Carpenter, B. et al. Stan: a probabilistic programming language. The pseudo-marginal approach for efficient Monte Carlo computations. Skilling, J. Nested sampling for general Bayesian computation. Speagle, J. S. dynesty: A dynamic nested sampling package for estimating Bayesian posteriors and evidences. Practical Bayesian model evaluation using leave-one-out cross-validation and WAIC. & Martin, O. ArviZ a unified library for exploratory analysis of Bayesian models in Python. Yao, Y., Vehtari, A., Simpson, D. & Gelman, A. Using stacking to average Bayesian predictive distributions. Williams, M. J., Werner, B., Barnes, C. P., Graham, T. A. Identification of neutral tumor evolution across cancer types. BEAST: Bayesian evolutionary analysis by sampling trees. Drummond, A. J., Suchard, M. A., Xie, D. & Rambaut, A. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Martinez, P. et al. Evolution of Barrett's esophagus through space and time at single-crypt and whole-biopsy levels. Griffiths, R. C. & Tavaré, S. Sampling theory for neutral alleles in a varying environment. Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty. Rambaut, A., Drummond, A. J., Xie, D., Baele, G. & Suchard, M. A. Posterior summarization in Bayesian phylogenetics using Tracer 1.7. Yu, G., Smith, D. K., Zhu, H., Guan, Y. & Lam, T. T. Y. ggtree: An r package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Díaz-Gay, M. et al. Assigning mutational signatures to individual samples and individual somatic mutations with SigProfilerAssignment. Tate, J. G. et al. COSMIC: the catalogue of somatic mutations in cancer. Suchard, M. A. et al. Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10. This research was primarily funded by an Accelerator award Cancer Research UK/AIRC/AECC joint funder-partnership (to J.F., J.I.M.-S. and T.A.G.) Additional funding was provided by Cancer Research UK via direct grants (A19771 and DRCNPG-May21_100001 to T.A.G. ), through the CRUK Convergence Science Centre (CTRQQR-2021\100009 to T.A.G.) and via an early detection primer award (EDDPMA-May23/100059 to T.A.G. Further funding was provided by the European Research Council under the European Union's Horizon 2020 Research and Innovation Program (810287, BCLLatlas, to I.G.G., E.C. ), Generalitat de Catalunya Suport Grups de Recerca AGAUR (2021-SGR-01343 to J.I.M.-S. and 2021-SGR-01172 to E.C.). was further supported by the BBSRC London Interdisciplinary Doctoral Programme (1902605), Thorton Foundation funding to the Institute of Cancer Research and the Eric and Wendy Schmidt Foundation. M.D.-F. was supported by a postdoctoral fellowship of the AECC Scientific Foundation. were supported by the Swedish Research Council (2019-01976), the Swedish Childhood Cancer Fund (PR2019-0046/PR2022-0082) and the Swedish Cancer Society (CAN2022-2395). This research utilized the Cancer Research UK City of London High Performance Computing facility, and was partially developed at the Centro Esther Koplowitz. We acknowledge the SciLifeLab National Genomics Infrastructure, SNP&SEQ Technology Platform, funded by the Swedish Research Council and the Knut and Alice Wallenberg Foundation, for assistance with DNA methylation analyses. We thank D. Landau and F. Gaiti for supplying the single-cell reduced representation bisulfite sequencing data for analysis. These authors contributed equally: Calum Gabbutt, Martí Duran-Ferrer These authors jointly supervised this work: José I. Martin-Subero, Trevor A. Graham Calum Gabbutt, Heather E. Grant, Jacob Househam & Trevor A. Graham Centre for Genomics and Computational Biology, Barts Cancer Institute, Queen Mary University of London, London, UK Calum Gabbutt, Jacob Househam, Jude Fitzgibbon & Trevor A. Graham Fundació de Recerca Clínic Barcelona-Institut d'Investigacions Biomèdiques August Pi i Sunyer (FRCB-IDIBAPS), Barcelona, Spain Martí Duran-Ferrer, Ferran Nadeu, Neus Villamor, Elias Campo, Armando Lopez-Guillermo & José I. Martin-Subero Martí Duran-Ferrer, Ferran Nadeu, Neus Villamor, Elias Campo, Armando Lopez-Guillermo & José I. Martin-Subero Arizona Cancer Evolution Center, Biodesign Institute and School of Life Sciences, Arizona State University, Tempe, AZ, USA Neus Villamor, Elias Campo & Armando Lopez-Guillermo Centro Nacional de Analisis Genomico (CNAG), Barcelona, Spain Madlen Müller, Simon Heath, Emanuele Raineri & Ivo G. Gut Department of Medical Sciences, Molecular Precision Medicine and Science for Life Laboratory, Uppsala University, Uppsala, Sweden Department of Medical Oncology and Hematology, University Hospital and University of Zürich, Zurich, Switzerland Ivo G. Gut, Elias Campo & José I. Martin-Subero Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the modelling scheme and the Bayesian inference framework, with support from C.P.B. performed the experiments and primary analyses of the long-read nanopore sequencing data. This study builds on the earlier conception of the fluctuating methylation phenomenon by D.S. Correspondence to Calum Gabbutt, José I. Martin-Subero or Trevor A. Graham. T.A.G., J.I.M.-S., M.D.-F. and C.G., the Institute of Cancer Research, Fundació de Recerca Clínic Barcelona-Institut d'Investigacions August Pi i Sunyer and Institució Catalana de Recerca i Estudis Avançats have filed for a patent on a method to measure the evolutionary dynamics in cancers using DNA methylation (GB2317139.0). T.A.G is named as a co-inventor on adjacent patent applications that describe a method for TCR sequencing (GB2305655.9), and a method to infer drug resistance mechanisms from barcoding data (GB2501439.0). T.A.G has received an honoraria from Genentech and consultancy fees from DAiNA therapeutics. received honoraria from AbbVie, AstraZeneca, Janssen and SOPHiA GENETICS for speaking at educational activities, research funding from Gilead, and licensed the use of the protected IgCaller algorithm for SOPHiA GENETICS and Diagnóstica Longwood. All other authors declare no competing interests. Nature thanks Pavlo Lutsik, George Vassiliou and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a: fCpGs were selected by combining three filters: (1) CpGs with low intra-disease heterogeneity were removed as sites that were likely not fluctuating, (2) CpGs with a mean methylation far from 0.5 were removed as likely belonging to CpG sites with skewed methylation/demethylation rates, and (3i) CpGs which preserved the nearest-neighbour graph (quantified using the Laplacian Score) were removed as sites that were likely under selection/strict regulation. b-d: Heatmaps of b: Horvath's CpGs72, c: epiCMIT CpGs14, and d: random subset of 1,000 CpGs from the candidate CpGs on the bead array. e-i: Heatmaps of our set of 978 pan-lymphoid fCpGs with disease-specific subtypes annotated in: e chronic lymphocytic leukaemia (CLL), f B-cell acute lymphoblastic leukaemia (B-ALL), g mantle cell lymphoma (MCL), h diffuse large B-cell lymphoma - not otherwise specified (DLBCL-NOS), and i multiple myeloma (MM). Hierarchical clustering with average linkage and a Euclidean metric was used in heatmaps. a: Scatterplot showing the pairwise correlation coefficient in the fCpG methylation values of lymphoid cancer samples with other fCpG methylation values on the same chromosome, and the genomic distance between them. b: The average absolute difference from 0.5 methylation value of fCpGs with their local neighbourhood in whole genome bisulphite sequencing (WGBS) data in different B- and T- cell populations31. The fraction of hypo- and hyper- methylated CpG loci increases as a function of distance from the reference fCpG locus. c: (left) Heatmap of control single nucleotide polymorphism (SNP) probes from Illumina arrays, showing distinct methylation dynamics compared to fCpGs, with normal and remission samples intermingled with tumours. (middle) A distinct methylation distribution of control SNP probes and the same number of random fCpGs is shown for one healthy naïve B cell sample. (right) The percentage of CpGs in intermediate peaks (i.e., ≥0.2 and ≤0.4 and ≥0.6 and ≤0.8) is notably higher in fCpGs compared to control SNP probes. d: A bar plot showing the fraction of fCpGs that were called as unmodified cytosine (C), 5-Methylcytosine (5mC), 5-hydroxymethylcytosine (5hmC) or a non-canonical base (A, G or T). e-f: Heatmap, histograms and scatterplots showing that the methylation status of fCpGs is validated in Oxford nanopore long-read (e) and WGBS data (f). g: fCpG methylation of phased Oxford nanopore long-reads of 2 CLL and 2 matched RT samples, 3 sorted memory and 3 naïve B cells samples. h: (left) A box plot comparing the mean intra-haplotype Hamming distance of phased reads from cancer and normal samples (p = 0.016, MW-U test). One normal B-cell sample contained only 2 reads and was therefore removed. (right) Paired comparison of the mean intra- and inter-haplotype Hamming distances in 1 CLL and 2 RT samples (p = 0.030, paired T-test). One of the CLL samples only had reads from one haplotype, and thus was not informative for this plot. a: (top) Illustration of the possible combinations of fCpG states in monosomic and trisomic regions, along with the corresponding fraction methylation value. (bottom) Example histograms showing the fCpG methylation distribution of fCpGs located on just monosomic) or trisomic regions within 2 patient samples. b: Mean and standard deviation of fCpGs as a function of age in 2 independent cohorts of normal lymphoid cells, showing consistent methylation fluctuations but increased variance during aging. c: Boxplots of standard deviation of fCpG methylation values of whole-blood samples divided by age groups together with lymphoid tumour samples. P-values were derived using two-tailed t tests. d: fCpG loci are significantly less likely to be associated with a gene (annotations provided by UCSC, p = 1.6e−15, chi-sq test). e: A comparison of the mutual overlap between different epigenetic clocks: mitotic14,65,66,67,68, chronological age69,70,71,72,73,74,75,76,77,78, gestational age79,80,81,82,83, biological age and mortality84,85,86, and trait predictors87,88 and fCpGs. f: P-P plot demonstrating fCpG associated genes have a lower expression (in transcripts per million) than non-fCpG associated genes. g: Linear regressions of the RNA expression of key methylation maintenance genes (DNMT1, DMT3a, DNMT3B and TET2) vs and the mean (left) and standard deviation (right) of the fCpG distribution. a: Posterior distributions after fitting the simulated data in Fig. 2b with the Bayesian inference method. b: Histogram of the simulated fCpG methylation distribution (blue) with the posterior predictive of the model fit overlaid (orange). c: Histograms showing the methylation distributions of 10,000 synthetic fCpGs simulated under the logistic growth model with varying carrying capacity (Supplementary Information). d-e: EVOFLUx posterior median and 95% credible interval were run on a subset of 2,000 of the simulated fCpGs in c as a function of the carrying capacity for the growth rate (d) and most recent common ancestor age (e). f: Histograms showing the fCpG methylation distributions of 10,000 synthetic fCpGs simulated under the heterogenous epigenetic switching model with varying switching rate standard deviation (Supplementary Information). g-h: EVOFLUx posterior median and 95% credible interval when run on a subset of 2,000 of the simulated fCpGs in f as a function of the switching rate standard deviation for the growth rate (g) and most recent common ancestor age (h). a: Example posterior resulting from running EVOFLUx on a CLL sample – a pairs plot showing the marginal (diagonal) and pairwise (off-diagonal) inferred posterior (orange) distributions, with the prior distributions overlaid (gray). b: A histogram of the fCpG methylation distribution (blue) with the posterior predictive of the model fit overlaid (orange). c-e: Regression plots between the parameters inferred by running EVOFLUx on all 978 fCpGs (x-axis) vs just those fCpGs present on diploid regions (y-axis) in the CLL cohorts for the growth rate (c), the most recent common ancestor age (d) and the effective population size (e). f: Example histograms showing the distribution of all 978 fCpGs (blue) in a CLL sample (SCLL-001), with a randomly downsampled subset of 20%, 40%, 60% and 80%. For each set of 10% increment, 10 replicate fCpG subsets were generated (grey dots) and the EVOFLUx inference repeated. Mean and standard error of the replicates represented with a blue dot and error bars respectively. a-d: Boxplots (whiskers extending to ±1.5×IQR) showing the distribution of inferred growth rate (a), effective population size (b), time since the most recent common ancestor (c), and mean epigenetic switching rate (i.e. mean of μ, ν, γ, ζ; d) by disease. For interpretability, only a subset of the pairwise p values are annotated (Mann-Whitney U tests, hs correction). e: Linear regression between the growth rate and mean epigenetic switching rates separated by cancer types. There is a positive association in B-ALL (P = 2.4e-98, R2 = 0.44) and T-ALL (P = 5.9e-06, R2 = 0.22), a weak negative association in MM (P = 1.6e-05, R2 = 0.18) and no association in CLL (P = 0.060, R2 = 0.005) or the other entities. f-k: Linear regressions between the patient age at sampling and the mean inferred epigenetic switching rate in B cell acute lymphoblastic leukaemia (B-ALL, f), T cell acute lymphoblastic leukaemia (T-ALL, g), chronic lymphocytic leukaemia (CLL, h), mantle cell lymphoma (MCL, i) multiple myeloma (MM, j) and diffuse large B cell lymphoma (DLBCL, k). l: Linear regression between the EVOFLUx inferred initial evolutionary growth rate and the estimate of a linear model of the number of historical lymphocyte counts with the sampling dates of patients with at least 10 sample timepoints before treatment (P = 2e-5, Supplementary Fig. Inferred growth rate (a) and effective population size (b) of individual CLL samples separated by driver mutational status for common driver alterations (mutations in TP53, SF3B1, NOTCH1, ATM, POT1 and IGLV3-21R110, and copy number alterations (CNAs) in del(11q22.3), del(13q14.3), del(17p13.1) and trisomy12). Differences between genotypes were tested using Mann-Whitney U tests and Benjamini-Hochberg FDR corrected. Tests were performed on U-CLL and M-CLL patients separately to remove differences solely due to with IGHV status. a: Illustrations of three alternative evolutionary models: neutral evolution, in which the cancer population with a MRCA emerging at time τ grows exponentially at rate θ; subclonal selection, in which an initial population emerging at time τ1 growing at rate θ1 is outcompeted by a fitter subclonal emerging at time τ2 with growth rate θ2; and independent clonal origins, in which the fitter clone emerging at time τ2 with growth rate θ2 bears no clonal relationship to the initial clone. b: Bar chart comparing the fraction of cancer samples identified as subclonal by EVOFLUx across disease (pairwise chi-sq tests, holm-sidak (hs) correction). c: Boxplots comparing the distribution of subclonal weightings inferred by EVOFLUx in samples called as neutral vs under subclonal selection via whole exome sequencing (WES) data. d: Logistic regression between the probability of a cancer being identified as subclonal via running MOBSTER43 on WES data and the number of mutations detected in the sample. Regression were run separately for samples also identified as subclonal/neutral via EVOFLUx (subclonal weighting > 95%). e: Boxplots comparing the distribution of subclonal weightings inferred by EVOFLUx in samples called as neutral vs under subclonal selection within whole genome sequencing (WGS) data. f: Receiver operating characteristic (ROC) curves showing the accuracy at predicting the subclonality of WGS data of competing logistic classifiers trained on just the EVOFLUx subclonality weighting, just the WES subclonality call and a model trained on both sources of data. g: Boxplots comparing the distribution of independent model weightings inferred by EVOFLUx in samples containing multiple IGHV rearrangements (independent cancers) vs those with only a single IGHV rearrangement detected via WGS, WES and/or RNA-seq46 (likely a single clonal origin). a-b: Corresponding phylogenies reconstructed on matched WGS SNVs for CLL case 12 and 19 to Fig. c-e: The reconstructed phylogenies of the relationship between samples collected longitudinally in 3 individual ALL patients, annotated with the clinical classification of each sample. f: Paired boxplot showing the standard deviation of the fCpG methylation distributions of B-ALL samples is greater than their matched remission sample (p = 9.6e-39, paired t-test). g: Comparison of the standard deviation of the fCpG methylation distributions of B-ALL patients in remission (i.e. no cancer cells present in blood) vs normal whole blood (p = 0.067, MW-U test). h: Example longitudinal samples from one patient showing the development of the fCpG distribution from diagnostic B-ALL, through remission and relapse. i: Scatterplots showing the marginal fCpG methylation distribution between diagnosis and relapse for the ALL case in h. a: Kaplan-Meier curves comparing the OS between patients with high vs low (cut-off values identified using maxstat statistics) inferred effective population sizes (Ne) in the discovery cohort33, separated by IGHV mutational status. b Multivariate Cox regression of the OS shows the Ne is significant when controlling for IGHV status, TP53 alterations and age at sampling in the discovery cohort. c: Univariate survival analysis of the time to first treatment (TTFT, blue) and overall survival (OS, red) in the validation CLL cohort28,29 for evolutionary variables inferred via EVOFLUx. Note this cohort contains a mixture of treated and untreated samples, of which only the untreated samples were included in the TTFT analysis. d: Kaplan-Meier curves comparing the TTFT between patients with high vs low (cut-off values identified using maxstat statistics) inferred cancer growth rates in the validation cohort, separated by IGHV mutational status. e: Multivariate Cox regression of the effect of the cancer growth rate on the TTFT in the validation cohort, controlling for IGHV status, TP53 alteration and age at sampling. f: Multivariate Cox proportional hazard regression on the TTFT for 229 CLL patients where longitudinal measurements of the lymphocyte numbers were available and therefore the contemporary growth rate could be estimated. The contemporary lymphocyte counts refer to the estimate of a linear model of the number of historical lymphocyte counts with the sampling dates of patients with at least 10 sample timepoints before treatment. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Fluctuating DNA methylation tracks cancer evolution at clinical scale. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09450-9'>Systematic attribution of heatwaves to the emissions of carbon majors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 16:09:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 645, pages 392–398 (2025)Cite this article Extreme event attribution assesses how climate change affected climate extremes, but typically focuses on single events1,2,3,4. Furthermore, these attributions rarely quantify the extent to which anthropogenic actors have contributed to these events5,6. Here we show that climate change made 213 historical heatwaves reported over 2000–2023 more likely and more intense, to which each of the 180 carbon majors (fossil fuel and cement producers) substantially contributed. This work relies on the expansion of a well-established event-based framework1. Owing to global warming since 1850–1900, the median of the heatwaves during 2000–2009 became about 20 times more likely, and about 200 times more likely during 2010–2019. Overall, one-quarter of these events were virtually impossible without climate change. The emissions of the carbon majors contribute to half the increase in heatwave intensity since 1850–1900. Depending on the carbon major, their individual contribution is high enough to enable the occurrence of 16–53 heatwaves that would have been virtually impossible in a preindustrial climate. We, therefore, establish that the influence of climate change on heatwaves has increased, and that all carbon majors, even the smaller ones, contributed substantially to the occurrence of heatwaves. Our results contribute to filling the evidentiary gap to establish accountability of historical climate extremes7,8. Human-induced global warming not only causes long-term changes of state variables, energy and water fluxes in the Earth system but also manifests through climate extremes9. Every region of the world exhibits changes in intensity and frequency of extreme weather and climate events10,11, and events that were near impossible in the past are now occurring10,12. To assess the extent of contribution of climate change to these events, the field of extreme event attribution (EEA) has developed over the past years, through approaches promoted by the World Weather Attribution (WWA) initiative1 and other methods2,3,4. These approaches have been used to study many individual extreme events13, often showing an important contribution of climate change. However, to our knowledge, there is no framework to systematically and collectively conduct attribution exercises on a set of events identified in past records14, implying that impactful extreme events may still not be assessed. Moreover, EEA studies typically attribute events to climate change, but rarely to its causes5,6. Extending EEA to source attribution provides the quantification of the causal chain from emitters to climate extremes. It has been proven unambiguously that anthropogenic activities are largely responsible for climate change, and that combustion of fossil fuels is the main contributor15. Three categories of emitters may be used: countries5, individuals6 or businesses. In the first case, the source allocation of emissions can be based on the territorial origin of emissions produced within country borders16,17. Consumption-based allocations can be pursued, as well as approaches based on individual emission profiles6,18. Finally, emissions can also be allocated to businesses that directly profit from fossil fuel production or other high-emitting activities19,20,21,22,23,24. Businesses with particularly high emission profiles are referred as carbon majors, encompassing not only investor-owned companies (for example, ExxonMobil) but also state-owned companies (for example, Saudi Aramco) or nation-state producers (for example, the former Soviet Union)19. Here, we address both issues: the lack of systematic attribution of extreme events and the absence of quantitative analysis establishing a causal chain from individual emitters to these events. We build on an existing and widely used EEA framework1, systematizing the approach. We assess how much climate change has contributed to 213 heatwaves reported in the international disaster database EM-DAT over 2000–2023, owing to their particularly significant impact, most of which were previously unattributed. Then, we build on existing approaches to assess contributions to climate change5,6, extending the attribution upstream to the emitters. We assess how much the emissions of the 180 biggest carbon majors19 contributed to global mean surface temperature and to the likelihood and severity of historical heatwaves. In the EM-DAT database (www.emdat.be), 226 heatwaves are reported over 2000–2023, across 63 countries (Fig. These events were reported because of significant economic losses or casualties, a declaration of state of emergency or a call for international assistance. These societal impacts warrant their relevance for event attribution. Despite EM-DAT being the most widely used disaster database, the reporting of heatwaves across countries is highly uneven, with only nine heatwaves out of 226 reported over Africa, Latin America and the Caribbean, although these regions are also prone to heatwaves10. This known reporting bias in the EM-DAT database25 calls for more complete reporting to enable a more exhaustive analysis. a, The number of heatwaves reported per country in EM-DAT (www.emdat.be) over 2000–2023. An EEA is performed for each of them, as shown for four examples with ERA5 (ref. b, The 2021 Pacific Northwest heat dome. c, The 2003 heatwave in France. d, The 2022 Indian heatwave. e, The 2013 eastern China heatwave. For each example, the average temperatures during the event are mapped, with the outlines of the reported region (lime green contours). Moreover, the intensity (average temperature T (°C) during the period and the region of the event) and change in GMST (°C) are represented over 1950–2023 (black dots), with their conditional distribution represented through the median (red line) and ranges of the distribution (red shading). Finally, the change in intensity and change in return period (year) compared with the preindustrial reference period are shown for each example. Uncertainties inferred using bootstrapping are not shown here for the sake of clarity. Further details are provided in the Methods. Each of these heatwaves is systematically characterized and analysed in a consistent framework, following the method promoted by the WWA initiative1. This method is shown with the Pacific Northwest heatwave of 2021 as reported for the United States (Fig. 1b), thus without British Columbia in Canada. This event was reported in Oregon, Washington, Northern California, Idaho and Western Nevada over 26–30 June 2021 (www.emdat.be). Usually, EEA defines the heatwave with a box surrounding the region1. Here the heatwave is defined using the exact spatial characterization in the EM-DAT database, as it represents how the disaster was experienced by the local populations1 (Fig. Daily temperatures are averaged over this period and region for every year of available observations. Although many indicators could be used to characterize the heatwave26, the choice of the average over the period is motivated by its relevance for the reported impact rather than its meteorological rarity (Methods). Following the method by WWA1 and justified by the extreme value theory27, a statistical relationship can be inferred that links the probability distribution of the event to the change in global mean surface temperature1 (GMST) (Fig. This relationship allows us to calculate the probability and the intensity of the heatwave, both under observed conditions with climate change and under the preindustrial climate of 1850–1900 without such perturbations (Fig. Using both observation-derived estimates (ERA5 (ref. 29)) and Earth system models30, these results synthesize how climate change has affected the heatwave through a change in intensity and how many times more likely the event has become, which is called the probability ratio1. More details on the systematization of the WWA approach are provided in the Methods. Using only ERA528, the Pacific Northwest heatwave of 2021 over the United States had climate change increasing its intensity by 4.4 °C compared with that in 1850–1900, with a 95% confidence interval of 2.2–6.8 °C. Adding all other datasets29,30 relevant for the region decreases the influence of climate change to a change in intensity of the Pacific Northwest heatwave of 2021 over the United States of 3.1 °C (1.4–5.1 °C). The median estimate indicates that climate change has also increased the probability of heatwaves by more than 10,000, and at least seven times according to the lower bound of the confidence interval. This attribution is consistent with existing works on this heatwave: a previous work31 found a probability ratio of at least 150 and a change in intensity of 2.0 °C (1.2– 2.8 °C), whereas another analysis32 suggested a change in intensity of more than 2.9 °C. Although consistent, the results differ because of the characterization of the event and its very unlikely nature. The region and period are here determined through the reporting of the disaster for relevance to the impact rather than choices motivated by its meteorological rarity. Very unlikely events such as the Pacific Northwest heatwave of 2021 are more difficult to investigate, increasing the dispersion across several analysis34. However, this increased dispersion does not lower the confidence in the conclusion, which is the strong influence of climate change on these unlikely events. The method described for this event is applied to the 226 heatwaves, with three other cases shown in Fig. 1c–e, with the results also consistent with available attribution studies. Additional tests are conducted to assess the adequacy of the method for each event. The goodness of fit is assessed, validating 217 out of the 226 heatwaves, whereas the remaining nine are removed from the ensuing analysis. Furthermore, although there are strong physical justifications that GMST has a causal link to the heatwave1,10, this statistical model does not necessarily imply statistical causation. Thus, we also infer the non-linear Granger causality4. For 214 out of the 217 heatwaves, we prove with more than 95% certainty that GMST is a Granger-causing indicator of the heatwave. The three other events are removed from the ensuing analysis. Finally, another heatwave is removed because of the ensuing analysis related to the carbon majors. All details on these tests are provided in the Methods. Our analysis shows that human-induced climate change has contributed to increasing the intensity of all 213 heatwaves analysed here (Fig. With reference to 1850–1900, the median estimates for the changes in intensity range across events from +0.3 °C to +2.9 °C. The latter is the heatwave introduced in Fig. 1b, the Pacific Northwest heatwave of 2021 over the United States, whereas the heatwave with the mildest change in intensity occurred in Pakistan in June 2000. Over the study period, attributed heatwaves have become more and more intense (Fig. The median of the events shows that climate change has increased the intensity by 1.4 °C over 2000–2009, 1.7 °C over 2010–2019 and 2.2 °C over 2020–2023. This is consistent with GMST increasing by more than 0.2 °C per decade over the study period, and land warming faster35. Each heatwave is allocated a category depending on its change in intensity (colour) and its probability ratio (vertical bars in per cent) with reference to 1850–1900. a–c, Events are categorized based on the year of the event: 78 heatwaves attributed over 2000–2009 (a), 54 heatwaves attributed over 2010–2019 (b) and 81 heatwaves attributed over 2020–2023 (c). Median results are shown here. Further details on the attribution of each heatwave event are provided in the Methods, and all results are provided in the Supplementary Information. Apart from increasing the intensity, climate change has also increased the probability of all 213 heatwaves. The lowest probability ratio is observed for the heatwave of May 2006 in India, in which the event became only 22% more likely. However, the median estimates show that climate change has made 55 heatwaves out of 213 (26%) at least 10,000 times more likely, with a 95% confidence interval ranging from 7 to 158. This probability ratio is equivalent to saying that these heatwaves would have been virtually impossible without anthropogenic influence. Over the study period, the contribution of climate change to the likelihood of these events is also increasing (Fig. Figure 2 shows that the probability ratios of the heatwaves are shifting from low values to higher values, although this trend is highly affected by natural variability. The median probability ratios show that, with reference to 1850–1900, climate change made heatwaves about 20 times more likely over 2000–2009, and about 200 times more likely over 2010–2019. Overall, this systematic attribution of 213 heatwaves enhances the capacity of EEA for analyses across events, proving that climate change made all events more intense and more likely, and that this influence is increasing over time with increased global warming. EEA has been extensively used to quantify how human-induced climate change influences extreme events13. A growing literature has also investigated the contributions of anthropogenic actors to climate change16,18,21. However, the quantification of the causal chain from individual emitters to the extreme events has only been pursued in selected cases5,6. Here we build on established approaches while also introducing key methodological advancements in the decomposition (Methods) and developing the framework for simultaneous investigation over a large set of events. Unlike previous works that focus on emissions by countries5 or individuals6, we here investigate the attribution of emissions from businesses and specifically the carbon majors. Following established approaches20,21,22,23, we assign to each carbon major the emissions associated with the full value chain of their products, including all emissions in line with established accounting and reporting standards for corporates. This modelling choice aims at filling a gap in the scientific literature and does not preclude broader reflections on emission allocations and business responsibilities20. The emissions from carbon majors are estimated from company production records and associated emission factors19, leading to a dataset that provides CO2 and CH4 emissions for 180 carbon majors over 1854–2023 (Fig. Altogether, the emissions from these carbon majors represent 57% of the total cumulative anthropogenic CO2 emissions, including land use over the 1850–2023 period36. When considering only the emissions from fossil fuels and cement, the emissions from these carbon majors represent 75% of the cumulative CO2 emissions over 1850–2023 (ref. 36). The carbon majors have heterogeneous contributions to the CO2 emissions. The 14 top carbon majors (the former Soviet Union, People's Republic of China for coal, Saudi Aramco, Gazprom, ExxonMobil, Chevron, National Iranian Oil Company, BP, Shell, India for coal, Pemex, CHN Energy, People's Republic of China for cement) represent 30% of the total cumulative anthropogenic CO2 emissions, including land use, about as much as the 166 other carbon majors combined (27%). From a national perspective, 33 carbon majors are headquartered in the United States, accounting for 10% of the total CO2 emissions, and 33 carbon majors are headquartered in China (12% of the total CO2 emissions). a,b, Contributions of the carbon majors to the cumulative CO2 emission since 1850 (all sectors) (a) as reported in the Carbon Majors database (https://carbonmajors.org/) and compared with the Global Carbon Budget36 and the ensuing GMST as simulated by the OSCAR model (b). The GMST of ERA528 and BEST29 have been rebased to 1850–1900 (ref. 9). c–q, Attribution of historical heatwaves to the emissions of carbon majors for 15 selected carbon majors. In each of these panels, the 213 heatwaves are allocated into categories of contributions of the carbon majors to the change in intensity (colours) and how many times the carbon major increased the likelihood of the heatwave compared to 1850–1900 (x-axis). The results are shown through the median, but all results are provided in the Supplementary Information. Based on the CO2 and CH4 emissions of the carbon majors, we compute the contributions of these carbon majors to GMST. Climate models may be used to calculate climate change over the historical period, but also counterfactual worlds, such as a world in which a given carbon major would not have emitted. The difference informs how much this single actor has warmed the Earth over time. This method has already been applied to a former version of this database using a simple impulse-response model for CO2 (ref. 21). Here we use the reduced-complexity Earth system model OSCAR, for its non-linear representation of both the carbon cycle and the atmospheric chemistry of methane, as well as its capacity to integrate observational constraints to improve the robustness of the assessment37 (Methods). Closely aligned with the estimates based on IPCC methodologies35, we estimate an increase in GMST of about 1.30 °C in 2023 with respect to 1850–1900 (ref. 28), of which 0.67 °C is due to the emissions of all carbon majors and 0.33 °C is due to the emissions of the 14 biggest carbon majors (Fig. The unattributed 0.63 °C is due to other actors responsible for unaccounted fossil fuel burning, agricultural and land-use activities, other industrial processes, as well as to non-attributed greenhouse gases (N2O and halogenated species) and short-lived climate forcers. For comparison, a former assessment associates 0.40 °C in 2010 with 90 carbon majors21, whereas we find 0.48 °C with 180 carbon majors in 2010. Knowing the contributions of the carbon majors to GMST and knowing the relationship between GMST and the heatwaves from the event attribution, we subsequently compute how each carbon major has affected each heatwave (for details, see the Methods). For each heatwave, the total effect of climate change on the intensity and probability of the event is decomposed into the contributions from individual carbon majors and the combined effect of other unidentified contributors, anthropogenic and natural. Contributions from carbon majors to the intensities of all heatwaves range from 0 °C to 0.18 °C (Fig. As expected, the higher the emissions from a carbon major, the higher its contributions to the intensities of the heatwaves. The median contributions to heatwaves from the 14 top carbon majors range from 0.01 °C to 0.09 °C. The other carbon majors have lower contributions, although the 166 of them combined have about the same importance as the biggest carbon majors. We calculate the influence of carbon majors on heatwaves reported over each decade of our dataset. With reference to 1850–1900, climate change has increased the median intensity of heatwaves by 1.36 °C over 2000–2009, of which 0.44 °C is traced back to the 14 top carbon majors and 0.22 °C to the 166 others. These contributions correspond, respectively, to 32% and 16% of the overall effect of climate change. Over 2010–2019, the influence of climate change increased to 1.68 °C, with 0.47 °C (28%) from the 14 top carbon majors and 0.38 °C (22%) from the 166 others. These results show that the emissions of carbon major contributed to about half of the increase in intensity of heatwaves since preindustrial times, and that this contribution is rising. Apart from intensities, all the carbon majors have also increased the probability of all the heatwaves. For heatwaves that climate change made only slightly more likely, or for carbon majors with much lower emissions, the contributions are limited to an increase by 10% of the preindustrial probability. However, there are heatwaves that the carbon majors have made at least 10,000 times more likely compared with preindustrial levels, and which would have otherwise been virtually impossible without anthropogenic influence. Even relatively minor shares in total emissions lead to very substantial increases in the frequency of these events. Specifically, emissions associated with the biggest emitter, the former Soviet Union, have made 53 heatwaves (25%) at least 10,000 times more likely. For the smallest carbon major by emissions, Elgaugol, this is still the case for 16 heatwaves (8%). It means that the sole emissions of these carbon majors would have rendered possible these heatwaves otherwise virtually impossible. We have systematized the process of EEA, based on the widely used method promoted by the WWA initiative1. We achieve the analysis of 213 heatwaves, thus extending the coverage of existing event attribution studies. We validate the goodness of fit and the causality for each of these events. We show that climate change has increased the probability and intensity of all these heatwaves. Owing to the consistent protocol across all events, their meta-analysis over time shows that the extremeness of the heatwaves is rising more and more rapidly because of climate change, both in intensity and probability. We also extend the attribution analysis upstream along the causal chain, providing a coherent attribution to individual emissions on the company level for 180 carbon majors. The contributions of the carbon majors are very heterogeneous, with 14 carbon majors (the former Soviet Union, People's Republic of China for coal, Saudi Aramco, Gazprom, ExxonMobil, Chevron, National Iranian Oil Company, BP, Shell, India for coal, Pemex, CHN Energy, People's Republic of China for cement) contributing as much as the 166 others. Considering all reporting heatwaves, we show that carbon majors represent about half of the change in intensity since 1850–1900 and that their contributions are rising, in particular the ones of the smaller carbon majors. The analysis of their contributions to the probabilities of the heatwaves shows that although the contributions scale well with their cumulative emissions, smaller carbon majors cannot be neglected. Depending on the carbon major, between 16 and 53 of these heatwaves are made possible with the sole contribution of the smaller carbon majors. Although this assessment builds on well-established methods, there are still two limitations in this work. Although the EM-DAT is the most complete existing database for disasters, many heatwaves are still not reported, calling for a more exhaustive coverage of the events. Moreover, the contributions of the carbon majors remain incomplete. On the one hand, not all CO2 and CH4 emissions are covered in this database because of underreporting19. For instance, this database represents only 75% of the fossil fuel and cement CO2 emissions reported over 1850–2023 (ref. 36). The actual contributions of the carbon majors are thus expected to be higher if all the emissions from fossil fuel and cement producers are included. On the other hand, the burning of fossil fuels can release aerosols that would have a local effect on the climate. As a whole, the aerosols emitted by the fossil fuel sector reduce their contribution by approximately 10% (ref. 38). However, attributing aerosol climate effects to individual companies would be highly challenging. The effects of aerosols on climate are local to regional, yet fossil fuels are globally traded. Furthermore, aerosol emissions from fossil fuel combustion strongly depend on the use of filter technology, which differs between regions, sectors and combustion techniques. If these challenges are overcome, it would pave the way for attributing the aerosol health effects to individual emitters. Aerosols are also harmful air pollutants, with the emissions by the use of fossil fuel causing about 5 million excess deaths per year (ref. 39). Accounting for these effects remains beyond the scope of our analysis. Our framework could be adapted to other physical hazards, such as ocean acidity22, sea-level rise40, fires41 or droughts42. Extending the attribution from physical hazards to societal impacts remains a challenge. We may use directly the fraction of attributable risk to deduce the fraction of the impact imputed to the actor6, but it neglects complex and non-linear aspects related to the vulnerability and the exposure to the hazard43. Nevertheless, this attribution framework may be extended to heat-related mortality44 or economic damages24. Finally, other top-down approaches can complement our findings45. These results are relevant not only in the scientific community but also for climate policy, litigation and wider efforts concerning corporate accountability8. Climate-related legal proceedings are proliferating, with defendants seeking compensation for losses and damages or requiring more ambitious climate actions from corporations and nations8. However, the scientific evidence backing the claims is often lagging behind the state of the art in climate science, thus failing to adequately draw causality links7. Although this work aims at filling in scientific gaps, the results also fill in evidentiary gaps. This systematic attribution improves the coverage in extreme events, thus reinforcing the potential of attribution science for climate litigation7,43. Furthermore, if the fact that fossil fuels are the main driver of climate change has been unambiguously established15,36, as acknowledged by the carbon majors themselves46,47,48, proving and quantifying the causality from the emitters to the events provides important new resources to assess legal responsibilities. Further strengthening of the links between climate scientists, legal scholars and practitioners is beneficial to ensure that the overwhelming scientific literature is correctly accounted for49. From the EM-DAT database, we select the events only after 2000, because the reporting is more complete after this date25,50, and since climate change has been shown to exert increasing influence on extremes over this period51. The locations reported for the events in the EM-DAT database are names of cities, provinces, states or whole countries (for example, France). Geographical boundaries are necessary for the analysis, so the locations of EM-DAT are matched to spatial elements from GADM52 using the following algorithm. The reported ISO code is used to pre-select the spatial elements of GADM for the country and any attached disputed territories. The reported location is prepared: replacing spatial characters (accents, numbers and punctuation); removing extra spaces; lowercase letters for all characters; synthesizing specific sentences (for example, ‘Kadamjay district in Batken oblast' becoming ‘Kadamjay'); correcting for any change in regional aggregation (for example, ‘Haute & Basse Normandie' becoming ‘Normandie'); translating any region without its variant in GADM (for example, ‘Voreio Aigaio' becoming ‘North Aegean'). Each preselected spatial element is compared with each element of the prepared reported location by applying a character matcher53 on its names and variants. Each retained spatial element is filtered using a prepared list of false positives. For instance, the location for the state of ‘Ohio' triggers the identification of the county ‘Ohio' in the states of Kentucky and West Virginia, which have not been reported. The list of spatial elements is compared with the initial reported location, checking whether it matches correctly. If not, the issue is implemented through the prepared lists of known issues in steps 2 and 4. Another work, the dataset GDIS, is also used to match EM-DAT locations to geographical boundaries54. Both approaches have been developed independently. GDIS differs in that the locations of all categories of hazards are analysed, but only up to 2018. Here, only the hazards for heatwaves were analysed, but up to 2023. Moreover, GDIS uses GADM v.3.6, whereas our work uses GADM v.4.1. In the EM-DAT database, the dates of the heatwaves are often reported with the starting and ending days. When both days are provided, we use the average of the daily average temperature over this exact period. Other indicators may be possible26, but the average aggregates the essential features of these heatwaves55. In particular, this choice is motivated by its relevance for the reported impact rather than its meteorological rarity. Heatwaves affect local populations not only through daily maximum temperatures but also through lack of cooling at night, which can be estimated using daily average temperatures. Sustaining high temperatures over time modifies the impact of a heatwave as well, justifying the use of averages over the period of the heatwave rather than its peak. The annual indicator is calculated first on each grid point, then averaged over the defined region to maximize the relevance of the indicator1,56. Furthermore, some events were reported without the starting and/or ending day(s). We observe that heatwaves reporting both days and lasting less than a month last on average for 8 days. Therefore, we use as an indicator for events with missing days the maximum of the 8 days running average over the reported month. In the case that several months were reported with missing starting and/or ending days, we lengthen the duration of the running average by 1 month for each supplementary month reported. The statistical model of attribution studies is carefully chosen to model the frequencies and intensities of extreme events1,56. To capture possible trends and non-stationarities, the distribution depends on the parameters driven by explanatory variables. In this study, we opt for the generalized extreme value (GEV) distribution with a linear evolution of its location as outlined in equation (1). For every year y in the sample, the average temperature over the period and region of the event Ty is assumed to follow a GEV distribution of location μ, scale σ and shape ξ, whereas the location varies with the change in global mean surface temperature smoothed over the 3 previous years (GMST). Although the statistical model in equation (1) is common for the EEA of heatwaves1,56, we have compared its performance with other potential models. Apart from this GEV model, we have tested three other distributions with linear and non-linear evolutions of the parameters: Gaussian, skew normal and generalized Pareto. Overall, our comparisons assessed through quantile–quantile plots indicate that the GEV performs the best among the four distributions, especially in terms of upper tail behaviours. We calculate the classical Bayesian information criteria (BIC) to compare their performances while reducing the risk of over-fitting57. We observe that for all heatwaves, a stationary GEV has the lowest BIC. We note that the linear model of equation (1) is not always the best distribution according to the BIC, although the gain in BIC from the linear model to the best solution is always marginal. More quantitatively, the improvement in BIC from the stationary GEV to the linear model represents between 88% and 100% of the improvement from the stationary model to the best solution over all heatwaves analysed, with an average of 98%. In other words, sophisticating further the statistical model would, on average, improve the performance by only 2%. This result confirms that this expression is the most appropriate for most heatwaves. These fits are obtained by minimization of the negative log likelihood (NLL) of the training sample58. The first guess has its robustness improved using initial regression to approximate the coefficients42,59. The shape parameter is bounded between −0.4 and 0.4 (ref. 1). Moreover, the sample is weighted during minimization of the NLL, with weights equal to the inverse of the density of the GMST. This approach helps in providing equal performance over the full interval of GMST. The choice of whether to include the event or not when estimating the statistical model has been extensively discussed, although no final consensus has been reached1,31,60. The results presented in this paper have been obtained by estimating the event, to prevent removing points from the observational record. To ensure numerical convergence, a minimum probability of 10−9 was set for each point of the full sample. It implies that the attributed events under factual conditions will not have return periods higher than a thousand million years, which we consider long enough. Estimating return periods for unlikely events with relatively short observational records remains difficult34; thus, we append additional lines of evidence1,56. Conditional distributions are trained for ERA5, used as reference, but also with BEST29 (Extended Data Fig. 1) and simulations from Climate Model Intercomparison Phase 6 (CMIP6)30,61. The following Earth system models (ESMs) from CMIP6 are used: ACCESS-CM2, ACCESS-ESM1-5, AWI-CM-1-1-MR, BCC-CSM2-MR, CESM2, CESM2-WACCM, CMCC-CM2-SR5, CMCC-ESM2, CanESM5, EC-Earth3, EC-Earth3-CC, EC-Earth3-Veg, EC-Earth3-Veg-LR, FGOALS-g3, GFDL-CM4, GFDL-ESM4, INM-CM4-8, INM-CM5-0, IPSL-CM6A-LR, KACE-1-0-G, KIOST-ESM, MIROC6, MPI-ESM1-2-HR, MPI-ESM1-2-LR, MRI-ESM2-0, NESM3, NorESM2-LM, NorESM2-MM and TaiESM1. For every heatwave, only the ESMs with sufficient performance are used, as described in the next section. For ERA5 and BEST, we start the time series in 1950 for adequate spatial coverage over all regions62, and these time series finish in 2022 for BEST and 2023 for ERA5. For CMIP6, the time series is calculated over the historical (1850–2014) (ref. This scenario is chosen because its emissions are the closest to those observed over 2015-202 (ref. 36). Only runs with the initial conditions termed r1i1p1f1 are used, as it was run by most ESMs. Only one ensemble member is used to facilitate the comparison of the parameters and probabilities from ESMs to those based on observations. During the extreme event analysis, two sources of uncertainties are handled—namely, on the conditional distributions and on the handling of observations and simulations. During the training of conditional distributions, the uncertainties on the parameters are obtained using an ensemble of 1,000 bootstrapped members, with replacements allowed during the resampling. The conditional distributions are then used to assess the probabilities and intensities of the event, under a factual climate and a counterfactual climate. The counterfactual climate is defined as the average of the GMST over 1850–1900. ESMs exhibit different performance in reproducing local climates; thus, not all models may be useful for event attribution1. We calculate the seasonalities of ERA5 and each ESM over 1950–2020 in each grid point over the region. We then average their correlation. The most appropriate ESMs are the 10 most representative ESMs that maximize this average correlation. Following the WWA approach, not all models are retained for further analysis1,65. The factual distributions at the time of the event are compared with those of ERA5. Both the scale and the shape parameters of ERA5 and the model must have their 95% confidence intervals overlapping. Otherwise, the model will be discarded. Thus, the overall selection process is to sort the ESMs by correlation with ERA5 seasonality, remove those with parameters inconsistent with ERA5 and select the 10 best ESMs in this list. At this point, probability ratios and change in intensities are obtained for an ensemble of datasets, each with uncertainties. To synthesize over this large ensemble, equal weights are given to each bootstrap member of ERA5 and BEST, summarized into one distribution for observations. All kept ESMs are also given equal weights and summarized into one distribution for models. Finally, these distributions are averaged to deduce the median and 95% confidence intervals. We point out that synthesizing these lines of evidence could be conducted with other approaches1, although without affecting the main messages of this work. Although using a non-stationary GEV with its location varying linearly with GMST is a well-established approach for EEA to statistically model extremes under global warming, this setup may not be well-suited in isolated cases66. To ensure that the GEV model represents the data adequately, the goodness of fit is verified for each conditional distribution used in this analysis with the method used in ref. For every heatwave, several datasets are used for analysis, from which conditional distributions are fitted. The location and scale of each of these fitted conditional distributions are used to transform the respective training sample onto a stationary GEV(0, 1, ξ) with the same shape as the fitted conditional distribution. This transformed sample has observed quantiles, which are compared with the theoretical quantiles of a GEV(0, 1, ξ) in a quantile–quantile plot. This quantile–quantile plot describes how well the GEV model describes the sample. The uncertainty in the GEV parameters determines a confidence band in the quantiles around the identity line, as shown in Extended Data Fig. The fraction of the sample out of the confidence band is deduced, estimated as a 95% confidence interval on the ensemble obtained from bootstrapping of the training of the conditional distribution. For each heatwave, the ensemble of conditional distributions is compared with an out-of-sample threshold at 5%. The results are shown in Extended Data Fig. If the median out-of-sample fraction across the conditional distributions is below the threshold, the goodness of fit is confirmed, and the heatwave is kept for the ensuing analysis. As shown in Extended Data Fig. 3, 217 heatwaves are retained, and nine are removed from the ensuing analysis. These nine events removed from analysis are summarized in Extended Data Table 1. Out of the nine events, eight occurred in India, and the last one occurred in Japan. In the 217 heatwaves kept for analysis, eight events occurred in India. Besides this apparent regional clustering, no discernible traits emerge with regard to the season or length of the heatwave. More research is required to investigate why these fits do not perform as well as elsewhere for these specific events, which lies beyond the scope of this study. The well-established approach1 for EEA combines observations and simulations by using non-stationary distributions. These distributions correlate the evolution of the climate indicator T for the heatwave to GMST. Deducing causality, that climate change caused the event, using this correlation relies on the strong physics-based understanding5,6,31,41,65,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97 that increasing GMST tends to also increase regional temperatures, not only through its mean but also through the whole distribution inferred from natural variability, thus shifting the regional extremes as well. Yet, although there is a strong physical basis for this causality, we can also investigate the validity of this causality from a statistical perspective. Using Granger causal inference98, we may assess the predictive relationship between GMST and T, the climate indicator of the event4. The common approach for Granger causality4,98 requires that the input variables are stationary to train vector auto-regressive models99. This is usually verified by differentiating the variables, in other words, taking the interannual variability. This method would then assess whether the interannual variability of GMST can predict the interannual variability of T, thus focusing on the predictability of short-term shocks. However, the trend contains a stronger signal compared with the interannual variability. To account for long-term trends in GMST and T, Granger causality can be generalized using the vector error correction model (VECM)100. It requires the search for an adequate VECM model based on the Akaike information criterion101 and a co-integration test, for instance, using a Johansen test99. Nevertheless, this method still fails to account for non-linear effects. An alternative is to use machine learning, such as Random Forest models trained to predict T with GMST through their lagged effects102. Permutation tests are conducted to assess the performance of Random Forest models trained on permuted lagged GMST, compared with the non-permuted version103. Applying this method accounts for the evolution of GMST and T, while also accounting for non-linear effects. By using the latter method, only three events have a median value for the test above 0.05. As shown in Extended Data Fig. 4, for 214 events out of 217, we reject the null hypothesis, concluding that the evolution of GMST is Granger-causing the evolution of T. The three other events are listed in Extended Data Table 2. We notice that the median value for the Granger causality remains relatively low. Using IPCC terms, it is likely (>66%) that GMST Granger-caused the evolution of T for the event in the United States in 2011, whereas it is very likely (>90%) for all the others. These three events are removed from this analysis. The contributions of emissions of the carbon majors to global mean surface temperature are assessed with the reduced-complexity Earth system model OSCARv3.3 (refs. 104,105). The model embeds an ensemble of modules that replicate the behaviour of models of higher complexity105. OSCAR features the ocean and land carbon cycles with a bookkeeping module for CO2 emissions from land use and land cover change, wetlands, permafrost, tropospheric and stratospheric chemistry, and global and regional climate responses to these forcers. It accounts for the effects of greenhouse gases (CO2, CH4, N2O and 37 halogenated compounds), short-lived climate forcers (stratospheric water vapour, tropospheric and stratospheric ozone, primary and secondary organic aerosols, nitrates, sulfates and black carbon), surface albedo change, volcanic activity, solar radiation and contrails37,105. OSCAR is run over the historical period (1750–2023), following three sets of simulations: (1) The first set of simulations is driven by concentrations of greenhouse gases to ensure a match with the latest observations. (2) The second set is driven by emissions, using the compatible emissions from the first set obtained through mass balance106,107. This is a control run that confirms that the estimated compatible emissions lead to the observed atmospheric concentrations and is used as a reference for the following attribution runs. (3) In the third set of simulations, for each carbon major, the control run is repeated, but the CO2 and CH4 emissions of the major are subtracted from the compatible emissions. The difference in outcome (for example, global temperature) between the control and this simulation gives the contribution of the major. This approach is called a residual attribution method108. In all simulations, the radiative forcings from species or forcers that are neither CO2 nor CH4 (that is, forcers that are not attributed in this study) are prescribed as global time series based on the latest version of the Indicators of Global Climate Change35. Global time series of atmospheric concentrations for the first set of simulations come from the same source. Emissions of short-lived species (that affect the atmospheric sink of CH4) are taken from the latest version of the CEDS dataset109,110 and the updated GFED4s dataset111 that extends the original CMIP6 emissions from biomass burning112. Land use and land cover change data are the same as in the latest Global Carbon Budget36, in which we use both an updated LUH2 dataset113 and the FAO-based dataset114. OSCAR runs in a probabilistic framework to represent the uncertainty in the modelling of the Earth system. This uncertainty is sampled through a Monte Carlo approach with n = 2,400 elements. The uncertainty in the natural processes governing the atmospheric concentration of CO2 and CH4 comes from the available parametrizations of OSCAR105,115,116. The uncertainty in the input radiative forcing follows that of the IPCC AR6 (ref. 117) and is applied uniformly to the whole time series. The uncertainty in the input land use and land cover change is sampled by running one-half of the simulations with one dataset and the other half with the other dataset. There is no uncertainty in the input emissions. Finally, the raw uncertainty range from the Monte Carlo is constrained with observational data by weighting the elements of the ensemble based on their distance to the observations in the control simulations37,116. As constraining values, we use decadal CO2 emissions from fossil fuels and industry over 2012–2021 from the GCB36, decadal anthropogenic CH4 emissions over 2008–2017 from the AR6 (ref. 118) offset with their preindustrial value from PRIMAP third-party-based estimates119,120, and decadal global mean surface temperature change over 2011–2020 from the AR6 (ref. We assess whether the probability can be written as a sum of terms, with each term associated with contributions from anthropogenic actors or natural drivers. We define a region in space S. Every year y, the temperature field over the region is averaged over a period p of the year, then over the region S, resulting in the temperature Ty. The heatwave is characterized by the exceedance of the heatwave level u by Ty, with u a real-valued scalar. Ty represents a real-valued continuous random variable (Borel σ-algebra on the reals). Given the heatwave level u, the target probability is a survival function P(Ty > u). We assume that the probability of the heatwave is conditional on GMTy and that it follows the statistical model introduced in equation (1) and represented in equation (2). Every year, the temperature over the region and the period Ty is sampled from a non-stationary GEV distribution27. The parameters of this GEV distribution are the location μ, the scale σ and the shape ξ. The location varies linearly with a covariate, the change in GMSTy at the corresponding year. With the analytical expression for the cumulative distribution function of the GEV that follows equation (3): This well-established statistical model is widely used for EEA1,56 and has already been used extensively for heatwaves. We acknowledge that a more sophisticated model with additional covariates may further improve the performance121,122. However, this statistical model has been shown to have good performance for heatwaves in general13, and additional covariates can prevent the use of climate models as additional lines of evidence. The former section provides additional grounds for the choice of this model. The causal theory applied to climate change justifies the decomposition of probabilities in a Gaussian case123,124. Given a statistical model built on a non-stationary Gaussian distribution linearly driven by GMST, if GMST can be split into a sum of contributions, then the probabilities can be approximated as a sum of their associated contributions123,124. However, the statistical model presented in equation (2) uses a GEV instead of a Gaussian. Even by attempting to write the decomposition using Bayes's theorem and the inclusion–exclusion principle, the exact analytical form of each term remains challenging. This is mostly because the differences in probability when removing a contribution to GMST depend on the initial value of GMST. In other words, the non-linearity and the high number of terms lead to a solution that cannot be computed exactly. Instead, we propose to approximate the solution and to investigate the quality of this approximation. The usual approach to calculate contributions to climate change is to run the statistical model with all contributors, then to run it again without one contributor, the difference corresponding to the contributor. This approach is thereafter called All-But-One (ABO). Thus, an emitter e with a contribution to global warming GMSTy,e would contribute to the probability of the event using this approach. To account for non-linear effects in the decomposition of probabilities, this approach is complemented with a second approach that calculates the difference in GMSTy introduced by adding only the emitter (Add-One-to-None, AON). According to this approach, the emitter e would contribute to the probability of the heatwave as follows: The approach based on the removal of a single entity (ABO) estimates the contribution of a state perturbed by all the other contributors. The approach based on the addition of a single entity (AON) evaluates the contribution in an unperturbed state, without the other contributors interfering. Given the non-linearity of the system, we expect the physical contribution to be between the two values. We choose to calculate both approaches and average them. This approach, calculated using equation (6), is called the combined ABU & AON (Extended Data Fig. For each event, the probability is calculated for all datasets for the region and averaged over the datasets. Its 95% confidence interval is calculated using bootstrapping. The total probability of the event is decomposed into contributions of each carbon major, other climate forcers and preindustrial probability. After decomposition, these terms are summed up for comparison with the total probability. The 95% confidence interval is shown for all events, and only one event (Cyprus, May–September 2022) does not reproduce the total probability. This event has been removed from the analysis of extreme events. As shown in Extended Data Fig. 5, the average of ABO and AON provides the best estimate, because it accounts for non-linear effects. In EEA, probabilities are often communicated using probability ratios, quantifying how many times climate change has made the event more likely. It is calculated using the probability of the event in a preindustrial climate, thus with a GMST averaged over 1850–1900: Because the contribution of the emitter e to the probability of the heatwave Py,e is a perturbation, the emitter multiplies the probability of the heatwave as in equation (8): Alternatively to the approach based on GMST, a basic approach would be to assess the contributions directly with the emissions. This fraction can be used for the change in intensity and the change in probability of the event. This approach can be compared with the principle applied for the attributional life cycle assessments, taking the Earth system as a whole and using the shares in its inputs to trace the perturbation125,126, whereas the approach based on GMST traces the effects of the carbon majors through the Earth system. Therefore, GMST is more similar to the principle of the consequential life cycle assessment. However, the approach based on cumulative emissions has several drawbacks. First, the carbon majors fuel climate change with CO2 and other compounds, such as CH4. As an approximation, it would still be possible to aggregate these compounds using a global warming potential for fossil CH4. Then, the carbon cycle partially absorbs the emitted carbon over time. Thus, two companies with the same cumulated emissions may not share the same responsibility, if one has older emissions, thus with a lower contribution to the atmospheric concentration of CO2. Still, these old emissions contributed to warming up the Earth system and saturating the carbon sinks. Finally, the attribution analysis may not respond linearly to changes in GMST. For our study, heatwaves are represented with a GEV with the location varying linearly with GMST. According to the Transient Climate Response to Emissions (TCRE), the GMST varies almost linearly with cumulative emissions. Thus, the approach based on cumulative emissions would lead to similar results to ours. However, for events for which the distributions do not vary linearly with GMST, as it may for extreme precipitations1,56, non-linearities would be introduced. To conclude, the approach based on cumulative emissions is an approximation that relies on the linearity of the Earth system. However, this system is not entirely linear, and the TCRE is known as an approximation with its limits127. Under the assumptions that the linearity of the system would be respected, this simple approach would then lead to similar results as those based on the approach used in this work. The data that support the findings in this study are available through the following references: disaster database EM-DAT (https://public.emdat.be/), geographical boundaries database GADM (https://gadm.org/download_world.html), Carbon Majors database (https://carbonmajors.org/Downloads), ERA5 (https://cds.climate.copernicus.eu/datasets/reanalysis-era5-single-levels), BEST (https://berkeleyearth.org/data/), and CMIP6 on the Earth System Grid Federation data nodes (http://esgf-node.llnl.gov/search/cmip6/). Detailed data for the search query are as follows: Experiment ID (historical, ssp245), Variant Label (r1i1p1f1), Frequency (day) and Variable ID (tas). The outputs of this study are provided in the Supplementary Information. All the codes that support the findings in this study are available at Zenodo128 (https://doi.org/10.5281/zenodo.15569401). Moreover, the code for OSCARv3.3 can be accessed at Zenodo129 (https://doi.org/10.5281/zenodo.10548477). Philip, S. et al. A protocol for probabilistic extreme event attribution analyses. Yiou, P. et al. A statistical framework for conditional extreme event attribution. Faranda, D. et al. ClimaMeter: contextualizing extreme weather in a changing climate. Risser, M. D., Ombadi, M. & Wehner, M. F. Granger causal inference for climate change attribution. Otto, F. E. L., Skeie, R. B., Fuglestvedt, J. S., Berntsen, T. & Allen, M. R. Assigning historic responsibility for extreme weather events. Lott, F. C. et al. Quantifying the contribution of an individual to making extreme weather events more likely. Stuart-Smith, R. F. et al. Filling the evidentiary gap in climate litigation. & Higham, C. Global Trends in Climate Change Litigation: 2025 Snapshot (Grantham Research Institute on Climate Change and the Environment, 2025). Gulev, S. K. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) 287–422 (Cambridge Univ. Seneviratne, S. I. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) 1513–1766 (Cambridge Univ. Weather, Climate and Catastrophe Insight 2023 (AON, 2023). Fischer, E. M. & Knutti, R. Anthropogenic contribution to global occurrence of heavy-precipitation and high-temperature extremes. Science Sources: Detection and Attribution. Perkins-Kirkpatrick, S. E. et al. Frontiers in attributing climate extremes and associated impacts. Canadell, J. G. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) 673–816 (Cambridge Univ. Jones, M. W. et al. National contributions to climate change due to historical emissions of carbon dioxide, methane, and nitrous oxide since 1850. Beusch, L. et al. Responsibility of major emitters for country-level warming and extreme hot years. Schöngart, S., Nicholls, Z., Hoffmann, R., Pelz, S. & Schleussner, C.-F. High-income groups disproportionately contribute to climate extremes worldwide. Heede, R. Tracing anthropogenic carbon dioxide and methane emissions to fossil fuel and cement producers, 1854–2010. Frumhoff, P. C., Heede, R. & Oreskes, N. The climate responsibilities of industrial carbon producers. Ekwurzel, B. et al. The rise in global atmospheric CO2, surface temperature, and sea level from emissions traced to major carbon producers. Licker, R. et al. Attributing ocean acidification to major carbon producers. Dahl, K. A. et al. Quantifying the contribution of major carbon producers to increases in vapor pressure deficit and burned area in western US and southwestern Canadian forests. Callahan, C. W. & Mankin, J. S. Carbon majors and the scientific case for climate liability. & Tubeuf, S. The untold story of missing data in disaster research: a systematic review of the empirical literature utilising the Emergency Events Database (EM-DAT). Russo, E. & Domeisen, D. I. V. Increasing intensity of extreme heatwaves: the crucial role of metrics. Coles, S. An Introduction to Statistical Modeling of Extreme Values (Springer, 2001). Hersbach, H. et al. The Berkeley Earth land/ocean temperature record. Tebaldi, C. et al. Climate model projections from the Scenario Model Intercomparison Project (ScenarioMIP) of CMIP6. Philip, S. Y. et al. Rapid attribution analysis of the extraordinary heat wave on the Pacific coast of the US and Canada in June 2021. & Fischer, E. M. Quantifying the statistical dependence of mid-latitude heatwave intensity and likelihood on prevalent physical drivers and climate change. Pons, F. M. E., Yiou, P., Jézéquel, A. & Messori, G. Simulating the Western North America heatwave of 2021 with analogue importance sampling. Zeder, J., Sippel, S., Pasche, O. C., Engelke, S. & Fischer, E. M. The effect of a short observational record on the statistics of temperature extremes. Forster, P. M. et al. Indicators of Global Climate Change 2024: annual update of key indicators of the state of the climate system and human influence. Friedlingstein, P. et al. Quilcaille, Y., Gasser, T., Ciais, P. & Boucher, O. CMIP6 simulations with the compact Earth system model OSCAR v3.1. Jiang, K. et al. Attributed radiative forcing of air pollutants from biomass and fossil burning emissions. Lelieveld, J. et al. Air pollution deaths attributable to fossil fuels: observational and modelling study. Nauels, A. et al. Attributing long-term sea-level rise to Paris Agreement emission pledges. Liu, Z., Eden, J. M., Dieppois, B. & Blackett, M. A global view of observed changes in fire weather extremes: uncertainties and attribution to climate change. Quilcaille, Y., Gudmundsson, L. & Seneviratne, S. I. Extending MESMER-X: a spatially resolved Earth system model emulator for fire weather and soil moisture. Clarke, B., Otto, F., Stuart-Smith, R. & Harrington, L. Extreme weather impacts of climate change: an attribution perspective. The burden of heat-related mortality attributable to recent human-induced climate change. Schleussner, C.-F. et al. Carbon Majors' Trillion Dollar Damages (Climate Analytics, 2023). Early oil industry knowledge of CO2 and global warming. Bonneuil, C., Choquet, P.-L. & Franta, B. Early warnings and emerging accountability: total's responses to global warming, 1971–2021. Supran, G., Rahmstorf, S. & Oreskes, N. Assessing ExxonMobil's global warming projections. Blattner, C. E., Vicedo-Cabrera, A. M., Frölicher, T. L., Ingold, K., Raible, C. C. & Wyttenbach, J. How science bolstered a key European climate-change case. Jones, R. L., Guha-Sapir, D. & Tubeuf, S. Human and economic impacts of natural disasters: can we trust the global data? Hawkins, E. et al. Observed emergence of the climate change signal: from the familiar to the unknown. Global administrative area data v.4.1. SequenceMatcher from difflib. Rosvold, E. L. & Buhaug, H. GDIS, a global dataset of geocoded disaster locations. Xu, Z., Cheng, J., Hu, W. & Tong, S. Heatwave and health events: a systematic evaluation of different temperature indicators, heatwave intensities and durations. van Oldenborgh, G. J. et al. Pathways and pitfalls in extreme event attribution. Schwarz, G. Estimating the dimension of a model. Statistical methods for extreme event attribution in climate science. Quilcaille, Y., Gudmundsson, L., Beusch, L., Hauser, M. & Seneviratne, S. I. Showcasing MESMER-X: spatially resolved emulation of annual maximum temperatures of Earth system models. & Davison, A. C. Timing and spatial selection bias in rapid extreme event attribution. Fan, X., Duan, Q., Shen, C., Wu, Y. & Xing, C. Global surface air temperatures in CMIP6: historical performance and future changes. Bell, B. et al. The ERA5 global reanalysis: preliminary extension to 1950. Eyring, V. et al. Overview of the Coupled Model Intercomparison Project Phase 6 (CMIP6) experimental design and organization. O'Neill, B. C. et al. The Scenario Model Intercomparison Project (ScenarioMIP) for CMIP6. Ciavarella, A. et al. Prolonged Siberian heat of 2020 almost impossible without human influence. Bercos-Hickey, E. et al. Anthropogenic contributions to the 2021 Pacific Northwest heatwave. Risser, M. D., Zhang, L. & Wehner, M. F. Data-driven upper bounds and event attribution for unprecedented heatwaves. The most at-risk regions in the world for high-impact heatwaves. Bartusek, S., Kornhuber, K. & Ting, M. 2021 North American heatwave amplified by climate change-driven nonlinear interactions. Leach, N. J. et al. Heatwave attribution based on reliable operational weather forecasts. Ye, Y. et al. Attribution of a record-breaking cold event in the historically warmest year of 2023 and assessing future risks. Tradowsky, J. S. et al. Attribution of the heavy rainfall events leading to severe flooding in Western Europe during July 2021. Interplay between climate change and climate variability: the 2022 drought in Central South America. A. et al. 2022 early-summer heatwave in Southern South America: 60 times more likely due to climate change. Li, S. & Otto, F. E. L. The role of human-induced climate change in heavy rainfall events such as the one associated with Typhoon Hagibis. Luu, L. N. et al. Attribution of typhoon-induced torrential precipitation in Central Vietnam, October 2020. B., Burger, F. A., Henson, S. A., Britten, G. L. & Frölicher, T. L. Historical and future maximum sea surface temperatures. Morim, J. et al. Understanding uncertainties in contemporary and future extreme wave events for broad-scale impact and adaptation planning. Climate change increased extreme monsoon rainfall, flooding highly vulnerable communities in Pakistan. Philip, S. Y., Kew, S. F., van der Wiel, K., Wanders, N. & Jan van Oldenborgh, G. Regional differentiation in climate change induced drought trends in the Netherlands. Rapid attribution of the record-breaking heatwave event in North China in June 2023 and future risks. Zachariah, M. et al. Attribution of 2022 early-spring heatwave in India and Pakistan to climate change: lessons in assessing vulnerability and preparedness in reducing impacts. Harrington, L. J. et al. Limited role of climate change in extreme low rainfall associated with southern Madagascar food insecurity, 2019–21. & Zachariah, M. On the role of climate change in the 2018 flooding event in Kerala. Dunne, K. B. J., Dee, S. G., Reinders, J., Muñoz, S. E. & Nittrouer, J. Examining the impact of emissions scenario on lower Mississippi River flood hazard projections. van Oldenborgh, G. J. et al. Attribution of the Australian bushfire risk to anthropogenic climate change. Hazards Earth Syst. Rousi, E. et al. The extremely hot and dry 2018 summer in central and northern Europe from a multi-faceted weather and climate perspective. Hazards Earth Syst. Sippel, S. et al. Could an extremely cold central European winter such as 1963 happen again despite climate change? Kew, S. F. et al. Impact of precipitation and increasing temperatures on drought trends in eastern Africa. Pietroiusti, R. et al. Possible role of anthropogenic climate change in the record-breaking 2020 Lake Victoria levels and floods. Vautard, R. et al. Human influence on growing-period frosts like in early April 2021 in central France. Hazards Earth Syst. Schumacher, D. L. et al. Detecting the human fingerprint in the summer 2022 western–central European soil drought. Vautard, R. et al. Human influence on European winter wind storms such as those of January 2018. Carrasco-Escaff, T., Garreaud, R., Bozkurt, D., Jacques-Coper, M. & Pauchard, A. The key role of extreme weather and climate change in the occurrence of exceptional fire seasons in south-central Chile. Qian, C., Ye, Y., Bevacqua, E. & Zscheischler, J. Human influences on spatially compounding flooding and heatwave events in China and future increasing risks. Kimutai, J., New, M., Wolski, P. & Otto, F. Attribution of the human influence on heavy rainfall associated with flooding events during the 2012, 2016, and 2018 March-April-May seasons in Kenya. Zachariah, M., Kumari, S., Mondal, A., Haustein, K. & Otto, F. E. L. Attribution of the 2015 drought in Marathwada, India from a multivariate perspective. Investigating causal relations by econometric models and cross-spectral methods. Lütkepohl, H. New Introduction to Multiple Time Series Analysis (Springer, 2005). & Newbold, P. Spurious regressions in econometrics. Springer Series in Statistics (eds Parzen, E. et al.) 199–213 (Springer, 1998). Papagiannopoulou, C. et al. A non-linear Granger-causality framework to investigate climate–vegetation dynamics. Leng, S., Xu, Z. & Ma, H. Reconstructing directional causal networks with random forest: causality meeting machine learning. Gasser, T. et al. The compact Earth system model OSCAR v2.2: description and first results. Jones, C. et al. Twenty-first-century compatible CO2 emissions and airborne fraction simulated by CMIP5 Earth system models under four representative concentration pathways. Gasser, T., Guivarch, C., Tachiiri, K., Jones, C. D. & Ciais, P. Negative emissions physically needed to keep global warming below 2 °C. Trudinger, C. & Enting, I. Comparison of formalisms for attributing responsibility for climate change: non-linearities in the Brazilian Proposal approach. Hoesly, R. M. et al. Historical (1750–2014) anthropogenic emissions of reactive gases and aerosols from the Community Emissions Data System (CEDS). Hoesly, R. & Smith, S. CEDS v_2024_04_01 Release Emission Data (v_2024_04_01). Guido, R. et al. Global fire emissions estimates during 1997–2016. Earth System Science Data 9, 697–720 (2017). Van Marle, M. J. E. et al. Historic global biomass burning emissions for CMIP6 (BB4CMIP) based on merging satellite observations with proxies and fire models (1750-2015). Hurtt, G. C. et al. Harmonization of global land use change and management for the period 850–2100 (LUH2) for CMIP6. Annual emissions of carbon from land use, land-use change, and forestry from 1850 to 2020. Gasser, T. et al. Path-dependent reductions in CO2 emission budgets caused by permafrost carbon release. Gasser, T. et al. Historical CO2 emissions from land use and land cover change and their uncertainty. Forster, P. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) 923–1054 (Cambridge Univ. Szopa, S. et al. in Climate Change 2021: The Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (eds Masson-Delmotte, V. et al.) 817–922 (Cambridge Univ. Gütschow, J. et al. The PRIMAP-hist national historical emissions time series. Gütschow, J., Pflüger, M. & Busch, D. The PRIMAP-hist national historical emissions time series (1750-2022) v2.5.1 (2.5.1). Van Oldenborgh, G. J. et al. Attributing and projecting heatwaves is hard: we can do better. Zhang, L., Risser, M. D., Wehner, M. F. & O'Brien, T. A. Leveraging extremal dependence to better characterize the 2021 Pacific Northwest heatwave. Hannart, A., Pearl, J., Otto, F. E. L., Naveau, P. & Ghil, M. Causal counterfactual theory for the attribution of weather and climate-related events. Weidema, B. P., Pizzol, M., Schmidt, J. & Thoma, G. Attributional or consequential life cycle assessment: a matter of social responsibility. Brander, M., Burritt, R. L. & Christ, K. L. Coupling attributional and consequential life cycle assessment: a matter of social responsibility. MacDougall, A. H. & Friedlingstein, P. The origin and limits of the near proportionality between climate warming and cumulative CO2 emissions. Extension for extreme event analysis. This work was supported by funding from the Horizon 2020 and Horizon Europe research and innovation programmes of the European Union (grant agreement nos. 101003687 (PROVIDE), 101081369 (SPARCCLE) and 101003469 (XAIDA)). also acknowledges support from the Horizon Europe research and innovation programme of the European Union (grant agreement no. R.H. acknowledges funding from the Rockefeller Brothers Fund. We thank L. Pierini for verifying the equations for the decomposition of contributions. acknowledges funding from the European Research Council under the Horizon Framework research and innovation programme of the European Union (grant agreement no. 101124572 and European Research Council consolidator grant ‘LACRIMA'). Views and opinions expressed are, however, those of the authors only and do not necessarily reflect those of the European Union or the European Research Council Executive Agency; neither the European Union nor the granting authority can be held responsible for them. We thank the referees for their careful review of the paper. Open access funding provided by Swiss Federal Institute of Technology Zurich. Institute for Atmospheric and Climate Science, Department of Environmental Systems Science, ETH Zurich, Zurich, Switzerland Yann Quilcaille, Lukas Gudmundsson, Dominik L. Schumacher & Sonia I. Seneviratne International Institute for Applied Systems Analysis (IIASA), Laxenburg, Austria Thomas Gasser & Carl-Friedrich Schleussner Climate Accountability Institute, Snowmass, CO, USA Department of Public Law and Governance, Tilburg University, Tilburg, The Netherlands Climate Analytics, Berlin, Germany Atmospheric, Oceanic and Planetary Physics, Department of Physics, University of Oxford, Oxford, UK Laboratoire des Sciences du Climat et de l'Environnement, ESTIMR, CNRS-CEA-UVSQ, Gif-sur-Yvette, France Vrije Universiteit Brussel, Department of Water and Climate, Brussels, Belgium Integrative Research Institute on Transformations of Human-Environment Systems (IRI THESys) and the Geography Department, Humboldt-Universität zu Berlin, Berlin, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the software and carried out the analysis. R.H. provided guidance on the Carbon Majors database. conducted the analysis with OSCAR. revised the statistical decomposition. drafted the text, and Y.Q., L.G., D.L.S., T.G., R.H., C.H., Q.L., S.N., P.N., W.T., C.-F.S. contributed to interpreting the results and refining the text. Correspondence to Yann Quilcaille. The authors declare no competing interests. Nature thanks Karsten Haustein, Mark Risser, Likun Zhang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Number of heatwaves reported per country in EM-DAT over 2000–2023. An extreme event attribution is performed for each of them, as illustrated for four examples with BEST29 data: b, the 2021 Pacific Northwest heat dome, c, the 2003 heatwave in France, d, the 2022 Indian heatwave, and e, the 2013 eastern China heatwave. For each example, the average temperatures during the event are mapped, with the outlines of the reported region (lime green contours). In addition, the intensity (average temperature T (°C) during the period and region of the event) and change in Global Mean Surface Temperature (GMST, °C) are represented over 1950–2023 (black dots), with their conditional distribution represented through the median (red line) and ranges of the distribution (red shading). Finally, the change in intensity and change in return period (year) compared to the pre-industrial reference period are shown for each example. Uncertainties inferred using bootstrapping are not shown here for the sake of clarity. For a perfect fit, the quantiles of the sample (red dots) would be distributed over the identity line (black line). The uncertainties on the shape parameter provide a confidence band for the theoretical quantiles (grey shaded area), while the uncertainties on the location and scale provide a confidence band for the quantiles of the sample (red lines). For each of the heatwaves and datasets used for analysis, the fraction of sample outside confidence band is calculated (black circles). An event is rejected if the sample has a fraction higher than 5% out of the confidence band (red dashed line). For each heatwave, results are summarized over datasets in boxplots (black box) and the median (purple dot). For each of the heatwaves and datasets used for analysis, the non-linear Granger causal inference is tested through comparisons of performance of Random Forest models (black circles). The hypothesis of non-causality is rejected if the p-value is lower than 0.05 (red dashed line). For each heatwave, results are summarized over datasets in boxplots (black box) and the median (purple dot). Each heatwave attributed in this analysis is shown here (coloured dots). The total probability of each event is the result from the attribution analysis. The reconstructed probability results from the decomposition of the probabilities using the three approaches (ABU in yellow, AON in blue, combined in green), then all contributions are summed up to facilitate the comparison. The event in the red box in the upper right corner is the only event for which the decomposition fails. The 95% confidence interval is shown for the combined ABU & AON (green lines) only for the sake of clarity. Contributions of climate change and each carbon majors for each analysed heatwave. The results are ordered by geographical occurrence of the heatwaves and geographical headquarters of the carbon majors, and provide the contribution to the intensity and probability ratio, with median and 95% confidence intervals Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Systematic attribution of heatwaves to the emissions of carbon majors. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09493-y'>Myeloid progenitor dysregulation fuels immunosuppressive macrophages in tumours</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 15:59:04
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Monocyte-derived macrophages (mo-macs) often drive immunosuppression in the tumour microenvironment (TME)1 and tumour-enhanced myelopoiesis in the bone marrow fuels these populations2. Here we performed paired transcriptome and chromatin accessibility analysis over the continuum of myeloid progenitors, circulating monocytes and tumour-infiltrating mo-macs in mice and in patients with lung cancer to identify myeloid progenitor programs that fuel pro-tumorigenic mo-macs. We show that lung tumours prime accessibility for Nfe2l2 (NRF2) in bone marrow myeloid progenitors as a cytoprotective response to oxidative stress, enhancing myelopoiesis while dampening interferon response and promoting immunosuppression. NRF2 activity is amplified during monocyte differentiation into mo-macs in the TME to regulate stress and drive immunosuppressive phenotype. NRF2 genetic deletion and pharmacological inhibition significantly reduced the survival and immunosuppression of mo-macs in the TME, restoring natural killer and T cell anti-tumour immunity and enhancing checkpoint blockade efficacy. Our findings identify a targetable epigenetic node of myeloid progenitor dysregulation that sustains immunoregulatory mo-macs in the lung TME and highlight the potential of early interventions to reprogram macrophage fate for improved immunotherapy outcomes. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Accession numbers for re-analysed published datasets are listed in Supplementary Table 7. Processed matrix files and metadata for the mouse tissue scRNA-seq and scATAC-seq generated in this study are made publicly available on GEO (GSE270148). Processed matrix files and metadata for human tissue scATAC-seq and 10x Multiome data generated in this study are also made publicly available at the time of publication on GEO (GSE270148). Source data are provided with this paper. Notable software package versions are listed in the Methods and in Supplementary Table 7. No new pipelines were used in the study beyond those described in the relevant Methods sections. Exemplar data objects and code for notable analyses in this study are provided at github.com/Merad-Lab/Hegde_Myelopoiesis_Epigenetics. Any additional information required to interpret data reported in this paper is available from the corresponding author upon request. DeNardo, D. G. & Ruffell, B. Macrophages as regulators of tumour immunity and immunotherapy. Swann, J. W., Olson, O. C. & Passegue, E. Made to order: emergency myelopoiesis and demand-adapted innate immune cell production. Goswami, S., Anandhan, S., Raychaudhuri, D. & Sharma, P. Myeloid cell-targeted therapies for solid tumours. Sica, A., Guarneri, V. & Gennari, A. Myelopoiesis, metabolism and therapy: a crucial crossroads in cancer progression. Giles, A. J. et al. Activation of hematopoietic stem/progenitor cells promotes immunosuppression within the pre-metastatic niche. Casbon, A. J. et al. Invasive breast cancer reprograms early myeloid differentiation in the bone marrow to generate immunosuppressive neutrophils. Circulating hematopoietic stem and progenitor cells are myeloid-biased in cancer patients. Porembka, M. R. et al. Pancreatic adenocarcinoma induces bone marrow mobilization of myeloid-derived suppressor cells which promote primary tumor growth. Classical monocyte ontogeny dictates their functions and fates as tissue macrophages. The early neutrophil-committed progenitors aberrantly differentiate into immunoregulatory monocytes during emergency myelopoiesis. LaMarche, N. M. et al. An IL-4 signalling axis in bone marrow drives pro-tumorigenic myelopoiesis. Hao, X. et al. Osteoprogenitor–GMP crosstalk underpins solid tumor-induced systemic immunosuppression and persists after tumor removal. Breast cancer remotely imposes a myeloid bias on haematopoietic stem cells by reprogramming the bone marrow niche. Dey, S., Curtis, D. J., Jane, S. M. & Brandt, S. J. The TAL1/SCL transcription factor regulates cell cycle progression and proliferation in differentiating murine bone marrow monocyte precursors. Dynamic epigenetic enhancer signatures reveal key transcription factors associated with monocytic differentiation states. Mandula, J. K. & Rodriguez, P. C. Tumor-related stress regulates functional plasticity of MDSCs. Paul, F. et al. Transcriptional heterogeneity and lineage commitment in myeloid progenitors. Kwart, D. et al. Cancer cell-derived type I interferons instruct tumor monocyte polarization. Immune suppressive activity of myeloid-derived suppressor cells in cancer requires inactivation of the type I interferon pathway. Kwak, H. J. et al. Myeloid cell-derived reactive oxygen species externally regulate the proliferation of myeloid progenitors in emergency granulopoiesis. Pizzato, H. A. et al. Mitochondrial pyruvate metabolism and glutaminolysis toggle steady-state and emergency myelopoiesis. Pietras, E. M. et al. Re-entry into quiescence protects hematopoietic stem cells from the killing effect of chronic exposure to type I interferons. Molgora, M. et al. TREM2 modulation remodels the tumor myeloid landscape enhancing anti-PD-1 immunotherapy. Coupled scRNA-seq and intracellular protein activity reveal an immunosuppressive role of TREM2 in cancer. Matusiak, M. et al. Spatially segregated macrophage populations predict distinct outcomes in colon cancer. Mulder, K. et al. Cross-tissue single-cell landscape of human monocytes and macrophages in health and disease. The temporal progression of lung immune remodeling during breast cancer metastasis. Beury, D. W. et al. Myeloid-derived suppressor cell survival and function are regulated by the transcription factor Nrf2. Namgaladze, D., Fuhrmann, D. C. & Brune, B. Interplay of Nrf2 and BACH1 in inducing ferroportin expression and enhancing resistance of human macrophages towards ferroptosis. Kobayashi, E. H. et al. Nrf2 suppresses macrophage inflammatory response by blocking proinflammatory cytokine transcription. Ryan, D. G. et al. Nrf2 activation reprograms macrophage intermediary metabolism and suppresses the type I interferon response. Olagnier, D. et al. Nrf2 negatively regulates STING indicating a link between antiviral sensing and metabolic reprogramming. Ting, K. K. Y. et al. Oxidized low-density lipoprotein accumulation suppresses glycolysis and attenuates the macrophage inflammatory response by diverting transcription from the HIF-1alpha to the Nrf2 pathway. Park, M. D. et al. TREM2 macrophages drive NK cell paucity and dysfunction in lung cancer. Alaluf, E. et al. Heme oxygenase-1 orchestrates the immunosuppressive program of tumor-associated macrophages. Leader, A. M. et al. Single-cell analysis of human non-small cell lung cancer lesions refines tumor classification and patient stratification. Hu, J. et al. Tumor microenvironment remodeling after neoadjuvant immunotherapy in non-small cell lung cancer revealed by single-cell RNA sequencing. Zelenay, S. et al. Cyclooxygenase-dependent tumor growth through evasion of immunity. In vivo induction of activin A-producing alveolar macrophages supports the progression of lung cell carcinoma. Gomez-Chou, S. B. et al. Lipocalin-2 promotes pancreatic ductal adenocarcinoma by regulating inflammation in the tumor microenvironment. Li, Z. et al. Proinflammatory S100A8 induces PD-L1 expression in macrophages, mediating tumor immune escape. & Garcia-Sastre, A. ISRE-reporter mouse reveals high basal and induced type I IFN responses in inflammatory monocytes. Singh, A. et al. Small molecule inhibitor of NRF2 selectively intervenes therapeutic resistance in KEAP1-deficient NSCLC tumors. Schaer, D. J. et al. Hemorrhage-activated NRF2 in tumor-associated macrophages drives cancer growth, invasion, and immunotherapy resistance. Fate mapping via Ms4a3-expression history traces monocyte-derived cells. Ren, D. et al. Brusatol enhances the efficacy of chemotherapy by inhibiting the Nrf2-mediated defense mechanism. Perrone, M. et al. ATF3 reprograms the bone marrow niche in response to early breast cancer transformation. Zhang, M. et al. Selective activation of STAT3 and STAT5 dictates the fate of myeloid progenitor cells. Laurenti, E. et al. Hematopoietic stem cell function and survival depend on c-Myc and N-Myc activity. Villar, J. et al. ETV3 and ETV6 enable monocyte differentiation into dendritic cells by repressing macrophage fate commitment. & Kucia, M. Hematopoiesis and innate immunity: an inseparable couple for good and bad times, bound together by an hormetic relationship. Ng, M. S. F. et al. Deterministic reprogramming of neutrophils within tumors. Zhao, Y. et al. Neutrophils resist ferroptosis and promote breast cancer metastasis through aconitate decarboxylase 1. Garner, H. et al. Understanding and reversing mammary tumor-driven reprogramming of myelopoiesis to reduce metastatic spread. Daman, A. W. et al. Microbial cancer immunotherapy reprograms hematopoiesis to enhance myeloid-driven anti-tumor immunity. Cancer Cell https://doi.org/10.1016/j.ccell.2025.05.002 (2025). Singh, A. et al. NRF2 activation promotes aggressive lung cancer and associates with poor clinical outcomes. Chirnomas, D., Hornberger, K. R. & Crews, C. M. Protein degraders enter the clinic—a new approach to cancer therapy. The unfolded protein response mediator PERK governs myeloid cell-driven immunosuppression in tumors through inhibition of STING signaling. PERK is a critical metabolic hub for immunosuppressive function in macrophages. The integrated stress response effector ATF4 is an obligatory metabolic activator of NRF2. Boumelha, J. et al. An immunogenic model of KRAS-mutant lung cancer enables evaluation of targeted therapy and immunotherapy combinations. The differential effects of mutant p53 alleles on advanced murine lung cancer. Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Hao, Y. et al. Integrated analysis of multimodal single-cell data. Martin, J. C. et al. Single-cell analysis of Crohn's disease lesions identifies a pathogenic cellular module associated with resistance to anti-TNF therapy. Andreatta, M. & Carmona, S. J. UCell: robust and scalable single-cell gene signature scoring. The myeloid type I interferon response to myocardial infarction begins in bone marrow and is regulated by Nrf2-activated macrophages. Agrawal, A. et al. WikiPathways 2024: next generation pathway database. Crowell, H. L. et al. muscat detects subpopulation-specific state transitions from multi-sample multi-condition single-cell transcriptomics data. Granja, J. M. et al. ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis. Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Schep, A. N., Wu, B., Buenrostro, J. D. & Greenleaf, W. J. chromVAR: inferring transcription-factor-associated accessibility from single-cell epigenomic data. The cis-regulatory atlas of the mouse immune system. McLean, C. Y. et al. GREAT improves functional interpretation of cis-regulatory regions. Ma, S. et al. Chromatin potential identified by shared single-cell profiling of RNA and chromatin. Kartha, V. K. et al. Functional inference of gene regulation using single-cell multi-omics. Early transcriptional effects of inflammatory cytokines reveal highly redundant cytokine networks. Chen, Y. et al. Spatiotemporal single-cell analysis decodes cellular dynamics underlying different responses to immunotherapy in colorectal cancer. Bassez, A. et al. A single-cell map of intratumoral changes during anti-PD1 treatment of patients with breast cancer. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Gayoso, A. et al. A Python library for probabilistic analysis of single-cell omics data. Bravo Gonzalez-Blas, C. et al. SCENIC+: single-cell multiomic inference of enhancers and gene regulatory networks. McDavid, A. et al. Data exploration, quality control and testing in single-cell qPCR-based gene expression experiments. was supported by National Institutes of Health (NIH) grants CA257195, CA254104 and CA154947. was supported by the National Cancer Institute (NCI) fellowship K00CA223043. was supported by NIH Medical Scientist Training grant T32GM146636. was supported by the AACR-AstraZeneca Immuno-oncology fellowship 21-40-12-MATT. This work was supported by grant no. R24-072073 to the Immunological Genome Project (ImmGen) consortium. We thank the patients and their families for participating in the clinical study; members of the Merad laboratory and Brown laboratory at the Marc and Jennifer Lipschultz Precision Immunology Institute at Mount Sinai for insightful discussions and feedback; and E. Bernstein, D. Hasson and D. Filipescu for their technical advice and feedback. We acknowledge the Human Immune Monitoring Center, the Mount Sinai Biorepository and Pathology Core and the Mount Sinai Cytometry Core for extensive support and resources. This work was supported in part through the computational resources and staff expertise provided by the Bioinformatics for Next Generation Sequencing (BiNGS) Shared Resource Facility funded by NCI P30 Cancer Center and NIH/NIAMS P30 Skin Biology Disease Resource-based Center support grants. This work was also supported in part through the computational and data resources and staff expertise provided by Scientific Computing and Data at the Icahn School of Medicine at Mount Sinai and supported by the Clinical and Translational Science Awards (CTSA) grant UL1TR004419, and the Office of Research Infrastructure award S10OD026880 and S10OD030463. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. These authors contributed equally: Bruno Giotti, Brian Y. Soong Marc and Jennifer Lipschultz Precision Immunology Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Samarth Hegde, Bruno Giotti, Brian Y. Soong, Laszlo Halasz, Jessica Le Berichel, Maximilian M. Schaefer, Raphaël Mattiuz, Matthew D. Park, Assaf Magen, Adam Marks, Meriem Belabed, Pauline Hamon, Theodore Chin, Leanna Troncoso, Kai Nie, Grace Chung, Darwin D'souza, Krista Angeliadis, Seunghee Kim-Schulze, Alexander M. Tsankov, Thomas U. Marron, Brian D. Brown & Miriam Merad Department of Immunology and Immunotherapy, Icahn School of Medicine at Mount Sinai, New York, NY, USA Samarth Hegde, Brian Y. Soong, Laszlo Halasz, Jessica Le Berichel, Maximilian M. Schaefer, Raphaël Mattiuz, Matthew D. Park, Adam Marks, Meriem Belabed, Pauline Hamon, Theodore Chin, Leanna Troncoso, Kai Nie, Grace Chung, Darwin D'souza, Krista Angeliadis, Seunghee Kim-Schulze, Thomas U. Marron, Brian D. Brown & Miriam Merad Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Bruno Giotti, Adam Marks, Alexander M. Tsankov, Thomas U. Marron, Sai Ma & Brian D. Brown Institut Gustave Roussy, INSERM U1015, Villejuif, France Benoit Kloeckner & Florent Ginhoux The Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Adam Marks, Raja M. Flores, Andrew J. Kaufman, Alexander M. Tsankov, Thomas U. Marron, Sai Ma, Brian D. Brown & Miriam Merad Department of Immunology, Harvard Medical School, Boston, MA, USA Department of Stem Cell and Regenerative Biology, Harvard Medical School, Boston, MA, USA Kaili Fan & Jason D. Buenrostro Immunology and Microbial Pathogenesis Program, Weill Cornell Graduate School of Medical Sciences, New York, NY, USA Dughan Ahimovic, Michael J. Bale & Steven Z. Josefowicz Laboratory of Epigenetics and Immunity, Department of Pathology and Laboratory Medicine, Weill Cornell Medicine, New York, NY, USA Dughan Ahimovic, Michael J. Bale & Steven Z. Josefowicz The Human Immune Monitoring Center, Icahn School of Medicine at Mount Sinai, New York, NY, USA Kai Nie, Grace Chung, Darwin D'souza, Krista Angeliadis, Seunghee Kim-Schulze & Miriam Merad Division of Hematology/Oncology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Raja M. Flores, Andrew J. Kaufman & Thomas U. Marron Center for Thoracic Oncology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Raja M. Flores, Andrew J. Kaufman & Thomas U. Marron Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Acquisition and analysis of data: S.H., J.L.B., R.M., M.D.P., A. Computational investigation of data: S.H., B.G., B.Y.S., L.H., M.M.S., B.K., M.D.P., A. Magen, D.D., K.F., J.J.L., D.A. Correspondence to Miriam Merad. serves on the scientific advisory board and holds stock from Compugen Inc., Dynavax Inc., Innate Pharma Inc., Morphic Therapeutics, Asher Bio Inc., Dren Bio Inc., Nirogy Inc., Genenta Inc., Oncoresponse, Inc. and Owkin Inc. M.M. also serves on the ad hoc scientific advisory board of DBV Technologies Inc. and Genentech Inc., and on the foundation advisory board of Breakthrough Cancer. receives funding for contracted research from Genentech, Regeneron and Boehringer Ingelheim. has served on advisory and/or data safety monitoring boards for Rockefeller University, Regeneron Pharmaceuticals, Abbvie, Bristol-Meyers Squibb, Boehringer Ingelheim, Atara, AstraZeneca, Genentech, Celldex, Chimeric, Glenmark, Simcere, Surface, G1 Therapeutics, NGMbio, DBV Technologies, Arcus and Astellas, and receives contracted grants from Regeneron, Bristol-Myers Squibb, Merck and Boehringer Ingelheim. The above interests are not directly relevant to this manuscript. The remaining authors declare no competing interests. Nature thanks Tim Greten, Renato Ostuni and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Averaged heatmap for abundance of LT-HSCs, progenitors, and mature populations in bone marrow (BM) of KP tumor-bearing mice at different timepoints, normalized to tumor-naïve mice (left to right, n = 10,7,12,12). Pooled from two independent experiments. b, Granulocytic-monocytic colony forming units (CFU-GM) relative to erythroid blasts (BFU-E) after 8 days of incubation. Pooled from two independent experiments. c, Longitudinal expansion of sorted GMPs from naïve and KP tumor-bearing mice. n = 3 replicates from one experiment. d, Number of Ly6Chi monocytes and Ly6Ghi neutrophils in blood of naïve and KP tumor-bearing mice at different time points, correlated with tumor burden. Data are individual data points with confidence intervals for linear regression (left to right, n = 5,7,12,12). Pooled from two independent experiments. e, Number of GMPs, cMoPs, and CD157+ Ly6Chi monocytes in BM, and number of Ly6Chi monocytes in blood of KP GEMM at 12 weeks post tumor initiation, compared to tumor-free mice. n = 3 mice per group, representative of two independent experiments. f, scRNA-seq heatmap of per-cell UMI counts across indicated myeloid cell subclusters in BM of tumor-bearing and naïve mice. Pooled over n = 3 mice per group. g, Gene ontology (GO) terms enriched in KP tumor-bearing mouse GMPs (left) and cMoPs (right) compared to naïve counterparts. Curated terms arranged by adjusted p-value (log q-value). h, Gene ontology (GO) terms enriched in PyMT tumor-bearing mouse GMPs compared to naïve counterparts from Gerber-Ferder et al.13 (left) and Hao et al. 2023 (right). Curated terms arranged by adjusted p-value (log q-value). i, exemplar scATAC-seq UMAP and heatmap of column-normalized gene scores across indicated myeloid cell subclusters in BM of tumor-bearing and naïve mice. Pooled over n = 3 mice per group. j, scATAC-seq heatmap of normalized transcription factor (TF) motif accessibility enrichment in marker peak regions of indicated myeloid cell states. Pooled from n = 3 mice per group. k, GREAT pathway terms enriched in indicated H3K4me3 regions from BM GMPs and Ly6Chi monocytes of KP tumor-bearing mice (Fig. Curated terms arranged by adjusted p-value (q-value). l–m, H3K4me1, H3K27ac, and H3K27me3 CUT&RUN signal clustering (l) and relative DORC scores for exemplar genes (m) in KP tumor-associated GMP relative to naïve GMP. n, Mean difference of ChromVAR-scores for TFs enriched in tumor-associated H3K4me3 signal versus H3K4me1 (left) and H3K27ac (right). o, scRNA-seq per-cell gene expression heatmap scaled across indicated myeloid subclusters in peripheral blood of human patients with NSCLC. Pooled from n = 4 patients. p, GREAT pathway terms enriched in indicated differentially accessible cisTopic region #5 of CD14+ monocytes from blood of patients with NSCLC (Fig. q, ChIP-X Enrichment Analysis (ChEA) calculated TF regulators for differentially expressed genes in CD14+ monocytes from blood of patients with NSCLC (n = 3) compared to healthy donors (n = 2). Dot color indicate major known biological pathways. Statistics computed by two-tailed Welch's t-test (b), unpaired t-test with Holm-Sidak multiple comparison (c), unpaired two-tailed Student's t-test (e), hypergeometric test with multiple test correction (g,h,k,j,p,q). error bars represent mean +/– SEM (c), colorscale indicates significance of motif enrichment based on hypergeometric test (j). a, Representative gating strategy for mouse bone marrow stem cell, progenitors, and differentiated monocyte and neutrophil populations. b, Gating strategy for mouse blood myeloid populations, and CD157+ phenotype of Ly6Chi monocytes. c, Representative plots for GFP in tumor-naïve and GFP-expressing KP-burdened bone marrow (BM) and lung. d, Representative broad sorting strategy for myeloid populations in mouse BM and blood. e, Representative gating strategy for human blood stem cell, progenitors, and myeloid populations. f, Representative sorting strategy for mouse BM GMPs in adoptive transfer experiments. g, Representative broad sorting strategy for myeloid populations in mouse lung. LT-HSC, long-term hematopoietic stem cell. LMPP, lymphoid-myeloid primed multipotent progenitors. CMP, common myeloid progenitor. MDP, myeloid dendritic cell progenitor. GP, committed granulocytic progenitor. cMoP, common monocytic progenitor. a, Representative gating strategy for mouse lung myeloid populations. b, Representative plots for intratumoral TREM2hi mo-macs and CD86+MHCII+ mo-macs, overlaid on phenotypic markers Arg1, CD206, PDL1, CD73, CD11c. c, CD64+MERTK+ mo-macs differentiated from donor CD45.2+ GMPs of naïve or tumor-primed origin (related to Fig. d, Frequency of BMDMs expressing Arg1, PDL1, MHCII, CD86 upon in vitro differentiation with KP tumor conditioning early (48 hr) or no conditioning (no prime). e, Frequency of BMDMs expressing Arg1, MHCII, CD86, and Arg1hi mo-macs expressing PDL1 upon in vitro differentiation with KP tumor conditioning early (48 hr prime) followed by washout. f, Frequency of CellROX-, LiperFluo- (n = 5 per group), and MitoSOX-expressing Ly6Chi monocytes (n = 3 per group) in KP tumor-bearing lung at day 21. g, Median Fluorescence intensity (MFI) for CellROX in KP tumor-infiltrating mo-macs. h, Representative cytometry plots and frequency of NRF2+ and HO-1+ cells in KP tumor-infiltrating mo-mac subsets. i, NRF2 downstream gene activation per-patient score in lung-infiltrating myeloid cells from independent cohorts of NSCLC (Leader et al., n = 35 and Hu et al. n = 15 patients). j, Frequency of BMDMs expressing NRF2 and HO-1 after 48 h prime and washout of KP CM, related to Extended Data Fig. k, Frequency of intratumoral donor mo-macs expressing NRF2 and HO-1 four days after GMP transfer from naïve (n = 3) or tumor-primed (n = 5) origin into congenic tumor-bearing hosts (related to Fig. l, Browser plots for H3K4me3 signal at known NRF2-associated gene loci in KP tumor-associated and naïve GMPs. Highlighted regions indicate differential signal near known cis-regulatory elements and open chromatin (OCR) containing Nfe2l2 motif. Individual data points with bar denoting mean, representative of two independent experiments. Statistics computed by one-way ANOVA with Tukey's multiple comparison (d), one-way ANOVA with Dunnett's multiple comparison (i), and unpaired two-tailed Student's t-test (e)–(h),(j)–(k). a, scRNA-seq heatmap of per-cell UMI counts (left) and relative abundance (right) of myeloid sub-clusters in lung tissue of naïve and KP tumor-bearing mice. b, Normalized UCell score for gene module# M24 across KP lung tumor-infiltrating myeloid cell clusters (left), with ENCODE/ChEA calculated TF regulators (right). c, exemplar scATAC-seq UMAP and heatmap of column-normalized gene scores across indicated myeloid cell clusters in lung of naïve and tumor-bearing mice. d, scATAC-seq heatmap of normalized transcription factor (TF) motif accessibility enrichment in marker peak regions of indicated myeloid cell states. e, snRNA-seq heatmap of per-cell UMI counts for indicated myeloid clusters in human NSCLC primary lung tumors. n = 5 patients pooled. f, UMAP of myeloid cells in snRNA-seq data from human NSCLC primary lung tumors annotated by marker genes (left, n = 5 pooled). Relative frequency of WCGNA gene programs (colors) in indicated clusters of macrophage-subsetted dataset (right). g–h, scATAC-seq heatmap depicting column-normalized gene scores (g) and normalized TF motif accessibility enriched in marker peaks (h) of indicated myeloid clusters in tumors of patients with NSCLC. n = 14 patients pooled. i, Candidate TF regulators in KP tumor-infiltrating monocyte and mo-mac clusters, prioritized by maximum TF motif deviation (Δ) across clusters. Dot color indicate known curated biological pathways. j, Reactome pathway terms enriched for conserved TF regulators from mouse and human analysis, ranked by adjusted p-value (log q-value). Red color indicates association to stress-associated cytoprotective signaling. Colorscale indicates significance of motif enrichment based on hypergeometric test (d,h). a, Frequency of NRF2+ Ly6Chi monocytes in BM (left), Ly6Chi monocytes in BM (mid), and frequency of NRF2+ mo-macs in lung (right) of KP GEMM at 12 weeks post tumor initiation, compared to tumor-free mice, n = 3 per group. b, ChromVAR-computed TF motif deviation for Nfe2l2 in tumor condition relative to naïve condition (left) and UCell-computed NRF2 downstream gene activation score (right) across indicated myeloid populations. c, Frequency of NRF2+ cells (top, n = 5 per group) and GFP+ cells (bottom, n = 4 per group) within indicated myeloid cell types from tumor-bearing and naïve (n = 4) Mx1GFP mice. d, Frequency of GFP+ cells within CD64+MERTK+ mo-macs of indicated phenotype in tumor-bearing Mx1GFP mice, n = 4 per group. e, Browser plots at indicated gene loci for H3K27ac signal in KP tumor-associated and naïve GMPs. Highlighted regions indicate differential signal near known cis-regulatory elements and open chromatin (OCR) containing Nfe2l2 motif. Unless noted, data shown as individual data points with bar denoting mean, representative of two independent experiments (a,c,d). Statistics computed by unpaired two-tailed Student's t-test (a,d), and unpaired t-test with Holm-Sidak multiple comparison (c). a, Frequency of HO-1+, Arg1+, MHCII+, and CD86+ BMDMs after culture with >3 kDa KP CM protein retentate fraction, and <3 kDa KP CM deproteinated fraction. b, Frequency of HO-1+, Arg1+, MHCII+, and CD86+ BMDMs after culture with heat-inactivated KP CM. c, Protein abundance of indicated analytes in tumor-associated BM sera, represented as relative fold-change over naïve sera. n = 2 per condition, one experiment. d–e, Analytes upregulated in BM sera (d) and blood sera (e) of tumor-bearing mice compared to naïve mice as assessed by O-link, with relative concentrations of GM-CSF and IL-6 (box). n = 3 per group, one experiment. f, Comparison of analyte concentration from BM sera and blood sera, data from (d) and (e). g, Ex vivo progenitor expansion assay, with frequency of ki67+ and HO-1+ myeloid cells upon culturing BM Kit+ cells with indicated growth factors for 2 days. h, Relative expansion (left, n = 4) and frequency of ki67+, HO-1+, and Arg1+ myeloid cells (right, n = 6) upon culturing progenitors with KP CM or indicated growth factors for 2 days. i–j, Relative expansion of myeloid cells with frequency of ki67+, HO-1+, and Arg1+ myeloid cells (i) and median fluorescence intensity (MFI) for CellROX and MitoSOX (j) after culturing progenitors with KP CM and indicated depletion antibodies for 2 days. k-l, Relative expansion of myeloid cells with frequency of ki67+, HO-1+, and NRF2+ myeloid cells (k) and MFI for CellROX and MitoSOX (l) after culturing progenitors with KP CM and ML385 for 2 days. m–n, Representative histogram and MFI quantification for phospho-p38 MAPK in myeloid progenitors treated with KP CM and indicated neutralizing antibodies (m, n = 3 per group) or KP CM and ML385 (n, n = 4 per group). o–p, Relative expansion of myeloid cells with frequency of ki67+, HO-1+, and Arg1+ myeloid cells (o) and MFI for CellROX and MitoSOX (p) after culturing BM progenitors with indicated growth factors in the presence of SB203580 for 2 days. q, Schematic for reactive oxygen species (ROS)-burden dependent poising of NRF2 pathways in growth factor-induced myeloid progenitor expansion in BM which enables NRF2 activation within TME. Individual data points with bar denoting mean, representative of two independent experiments (a,g,h,i–p). Statistics computed by one-way ANOVA with Tukey's multiple comparison (a,b), Dunnett's multiple comparison (g)–(p), hypergeometric test for O-link (d,e). a, Gene expression in KP conditioned media (CM)-exposed BMDMs as measured by reverse transcription quantitative PCR (RT–qPCR). Relative values depicted as fold-change compared to control media, after normalization to Hprt expression. n = 3 replicates, representative of two independent experiments. b, Gene expression in KP CM-exposed BMDMs treated with ML385, as measured by RT–qPCR. Relative values depicted as fold-change compared to BMDMs treated with vehicle DMSO, after normalization to Hprt expression. n = 3 replicates, representative of two independent experiments. c, Relative MFI quantification of CellROX, LiperFluo, and MitoSOX in Nfe2l2TKO (KO) or control (WT) BMDMs exposed to KP CM. d, Representative cytometry plots for Annexin-V and Propidium iodide (PI) in Nfe2l2TKO (KO) or control (WT) BMDMs exposed to KP CM, with relative frequency of apoptotic cells. e, Relative frequency of apoptotic cells in Nfe2l2TKO (KO) or WT BMDMs without any tumor conditioning. f–g, Number of LT-HSCs and myeloid progenitors in BM (f) and myeloid cells in blood (g) of tumor-free Nfe2l2ΔMs4a3 mice (n = 7) or Nfe2l2fl/fl controls (n = 5). h, MFI for indicated phenotypic markers in blood-circulating Ly6Chi monocytes from tumor-free Nfe2l2ΔMs4a3 mice (n = 5) or Nfe2l2fl/fl controls (n = 4). i, Fluorescent bead phagocytosis assay using tumor-free Nfe2l2ΔMs4a3 or Nfe2l2fl/fl BM Ly6Chi monocytes at different ratio of cells to beads. j, frequency of NRF2+ CD45.2+ donor-derived mo-macs derived from GMPs in Brusatol-treated tumor-bearing mice or N-acetyl cysteine-treated tumor-naïve mice transferred into KP tumor-bearing CD45.1 hosts, four days post transfer. (left to right, n = 3,4,4,3). Individual data points with bar denoting mean, representative of two independent experiments (a,b,f–j) or three independent experiments (c,d,e). Statistics computed by unpaired two-tailed Student's t-test (c–h), two-way ANOVA with Fisher's LSD test (a,b) or Sidak's multiple comparison (i). a, Kaplan-Meier plot for overall survival of KP tumor-bearing Nfe2l2ΔMs4a3 mice (n = 10) or Nfe2l2fl/fl littermates (n = 8). b, Lung tumor burden in LLC1 tumor-bearing (left) and B16-F10 tumor-bearing (right) Nfe2l2ΔMs4a3 mice (n = 9,10) or Nfe2l2fl/fl littermates (n = 8,9). c, Tumor burden in KP tumor-bearing Keap1ΔMs4a3 mice (n = 7) or Keap1fl/fl littermates (n = 8). d, Frequency of Annexin-V+ cells within indicated intratumoral myeloid cells of KP tumor-bearing Nfe2l2ΔMs4a3 mice (n = 7) or Nfe2l2fl/fl littermates (n = 5). e, Number of tumor-infiltrating TREM2hi mo-macs, CD86+MHCII+ mo-macs, and TREM2hi mo-macs expressing PDL1 in Keap1ΔMs4a3 mice or Keap1fl/fl littermates (n = 7 each). f, Representative flow plots (left) and quantification (right) of donor chimerism in blood and lung of Nfe2l2WT 1:1 Nfe2l2ΔMs4a3 chimera mice or Nfe2l2ΔMs4a3-only chimera mice (left). Individual data points with lines connecting same mouse. g–h, Frequency of BM GMPs and blood CD157+ Ly6Chi monocytes (g), frequency of lung-infiltrating TREM2hi mo-macs, CD86+MHCII+ mo-macs, and lung-infiltrating TREM2hi mo-macs expressing PDL1 (h) derived from indicated BM donors. Individual data points with lines connecting same mouse (n = 8). i, scRNA-seq heatmap of per-cell UMI counts across indicated myeloid cell subclusters in BM of tumor-bearing Nfe2l2ΔMs4a3 and Nfe2l2fl/fl mice. n = 3 mice pooled per group. j–k, Number of differentially expressed genes i.e., nDEGs (j) (|logFC | >1, adjP<0.05) and number of differentially accessible regions i.e., nDARs (k) (|logFC | >0.25 and adjP<0.05) within indicated BM myeloid clusters of Nfe2l2ΔMs4a3 mice. l, Differentially expressed genes in BM Ly6Chi monocytes of tumor-bearing Nfe2l2ΔMs4a3 mice compared to Nfe2l2fl/fl littermates. m, Transcription factor (TF) motifs differentially enriched in BM Ly6Chi monocytes of tumor-bearing Nfe2l2ΔMs4a3 compared to Nfe2l2fl/fl littermates; ranked by FDR. n, scRNA-seq UMAP and heatmap of per-cell UMI counts (left) alongside relative abundance (right) of indicated myeloid sub-clusters in lung tissue of tumor-bearing Nfe2l2ΔMs4a3 and Nfe2l2fl/fl mice. n = 3 mice pooled per group. o, Number of DEGs (|logFC| >1, adjP<0.05) within indicated lung tumoral myeloid clusters of Nfe2l2ΔMs4a3 mice. p, Normalized UCell-computed scores for NRF2 activation and Type I/III Interferon (IFN)-stimulation in lung-infiltrating Ly6Chi monocytes from scRNA-seq of tumor-bearing Nfe2l2ΔMs4a3 and Nfe2l2fl/fl mice. Data are per-cell distribution plot. q, Differentially expressed genes in tumor-infiltrating mo-macs (Arg1hi,Trem2hi) of tumor-bearing Nfe2l2ΔMs4a3 mice compared to Nfe2l2fl/fl littermates. Individual data points with bar denoting mean, representative of two independent experiments (a,b,c,d,e). Scale bar, 2 mm (c). Statistics computed by Log-rank (Mantel-Cox) test (a), unpaired two-tailed Student's t-test (b–e), paired two-tailed t-test (f–h), hypergeometric test with multiple test correction (l,p,q). a–b, Number of blood Ly6Ghi neutrophils in blood (a) and lung (b) of tumor-bearing Nfe2l2ΔMs4a3CreERT2 mice with early (n = 5) vs late deletion (n = 6) of NRF2. c, Representative gating strategy for major mouse lung lymphoid (T, B, NK cell) populations, with exemplar phenotypic markers depicted for CD8+ T cells and NK cells. d–e, Number of lung-infiltrating NK cells (d) and frequency of NK cells expressing CD69, NKG2D, and producing IFNγ (e) in KP tumor-bearing Nfe2l2ΔMs4a3 mice (n = 8) or negative littermates (n = 7) at day 21. f–g, Number of lung-infiltrating CD8+ T cells (f) and frequency of lung-infiltrating CD8+ T cells expressing PD-1, LAG3, CD69, or producing IFNγ, TNFα (g) in KP tumor-bearing Nfe2l2ΔMs4a3 mice (n = 8) or negative littermates (n = 7) at day 21. h, Representative histology and tumor burden for KP tumor-bearing Nfe2l2ΔMs4a3 mice and Nfe2l2fl/fl control that received anti-CSF1R depletion antibodies or isotype. n = 5 mice per group. i, Representative histology and tumor burden for KP tumor-bearing Nfe2l2ΔMs4a3 mice that received anti-NK1.1 antibodies or isotype. n = 7 mice per group. Unless noted, data shown as individual data points with bar denoting mean, representative of two independent experiments (a,b,d–g). Scale bar, 2 mm (h,i). Statistics computed by unpaired two-tailed Student's t-test (a,b),(d–g),(i), one-way ANOVA with Sidak's multiple comparison (h). a, ChIP-X Enrichment Analysis (ChEA) calculated TF regulators for differentially expressed genes in tumoral mo-macs enriched in non-responders (NR cohort) to neoadjuvant immunotherapy (NAIT) for lung cancer (NSCLC, Hu et al., n = 12 patients), compared to responders (R cohort). arranged by adjusted p-value (log q-value). b, ChEA calculated TF regulators for differentially expressed genes in tumoral mo-macs from patients with no clonal T cell expansion (NE cohort) after immune checkpoint blockade (ICB) in breast cancer (BC, Bassez et al., n = 29 patients) compared to macs from patients with clonal expansion (E cohort). c–d, ChEA calculated TF regulators for differentially expressed genes in tumoral mo-macs (c) and peripheral blood CD14+ monocytes (d) enriched in non-responders exhibiting stable disease (SD) to immune checkpoint blockade (ICB) for colorectal cancer (CRC, Chen et al., n = 22 patients) compared to complete response (CR). e, Kaplan-Meier plot depicting overall survival of KP tumor-bearing mice treated with Brusatol in conjunction with anti-PD-1 immunotherapy (n = 8), Brusatol alone (n = 6), anti-PD-1 alone (n = 7), or vehicle (n = 6). f, Kaplan-Meier plot depicting overall survival of KP tumor-bearing mice treated with ML385 in conjunction with anti-PD-1 immunotherapy (n = 6), ML385 alone (n = 5), or vehicle (n = 5). g, Representative histology and tumor burden for KP tumor-bearing Nfe2l2ΔMs4a3 and Nfe2l2fl/fl mice that received Brusatol in conjunction with anti-PD1 (left to right, n = 6,4,5,4,5,5) h, Frequency of NK cells within CD45+ cells (left), and percentage of NK cells expressing CD69 and producing IFNγ (right) in tumor-bearing WT mice treated as indicated. n = 5 mice per group. i, Frequency of CD8+ T cells within CD45+ cells (left), and percentage of CD8+ T cells expressing PD-1, LAG3 or producing IFNγ, TNFα (right) in tumor-bearing WT mice treated as indicated. Individual data points with bar denoting mean (g,h,i), representative of two independent experiments (e,f,g,h,i). Scale bar, 2 mm (g). Statistics computed by hypergeometric test with multiple test correction (a)–(d), Log-rank (Mantel-Cox) test (e,f), and one-way ANOVA with Sidak's multiple comparison (g) or Dunnett's multiple comparison (h,i). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. et al. Myeloid progenitor dysregulation fuels immunosuppressive macrophages in tumours. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02597-5'>Mystery Martian minerals hint at the planet's complex geochemical past</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 15:27:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Janice L. Bishop is at the SETI Institute, Mountain View, California 94043, USA. Search author on: PubMed Google Scholar Mario Parente is at the College of Engineering, University of Massachusetts Amherst, Amherst, Massachusetts 01003, USA. Search author on: PubMed Google Scholar In 2020, NASA dispatched the Perseverance rover to Jezero Crater on Mars to investigate and cache surface material. Writing in Nature, Hurowitz et al.1 report Perseverance's observations of small nodules of unusual minerals (Fig. 1) in the clay-rich ‘mudstone' rocks of Neretva Vallis — a valley on the western edge of Jezero Crater, characterized by sedimentary rocks that were deposited by ancient water systems. The findings shed light on the remarkable complexity of the oxidation–reduction (redox) chemistry that took place on ancient Mars, and might provide a fresh perspective on prebiotic chemical processes — reactions that could form a basis for the emergence of life. Get Nature+, our best-value online-access subscription Receive 51 print issues and online access Rent or buy this article Prices vary by article type Prices may be subject to local taxes which are calculated during checkout Parente, M. et al. Preprint at Zenodo https://doi.org/10.5281/zenodo.5575824 (2021). Szynkiewicz, A., Moore, C. H., Glamoclija, M. & Pratt, L. M. Geochim. The authors declare no competing interests. Read the paper: Redox-driven mineral and organic associations in Jezero Crater, Mars Changes in Mars's habitability could have been driven by carbonate formation and transient oases Deep Mars is surprisingly soft Redox-driven mineral and organic associations in Jezero Crater, Mars Changing tides ushered in the world's first civilization Redox-driven mineral and organic associations in Jezero Crater, Mars Two-billion-year transitional oxygenation of the Earth's surface A two-year post-doc position in the field of Organic Electronics, with the focus on the development of light-emitting electrochemical cells (LECs). Title: Associate or Senior Editor (Photocatalysis), Nature Communications Location: Berlin, New York, Jersey City, Shanghai, Beijing, Nanjing, (Hyb... Seeking exceptional Senior/Junior PIs, Postdocs, and Core Specialists globally year-round Job Title: Associate or Senior Editor, Nature Machine Intelligence Locations: New York, New Jersey, Beijing or Shanghai (Hybrid working model) Appl... Developing machine learning methods to infer multi-modal, condition-dependent causal gene networks from large-scale single-cell multi-omic datasets. UMass Chan Medical School - GRN Lab Read the paper: Redox-driven mineral and organic associations in Jezero Crater, Mars Changes in Mars's habitability could have been driven by carbonate formation and transient oases Deep Mars is surprisingly soft An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09413-0'>Redox-driven mineral and organic associations in Jezero Crater, Mars</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 15:09:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 645, pages 332–340 (2025)Cite this article Upon entering Neretva Vallis, on Jezero Crater's western edge8, Perseverance investigated distinctive mudstone and conglomerate outcrops of the Bright Angel formation. Here we report a detailed geological, petrographic and geochemical survey of these rocks and show that organic-carbon-bearing mudstones in the Bright Angel formation contain submillimetre-scale nodules and millimetre-scale reaction fronts enriched in ferrous iron phosphate and sulfide minerals, likely vivianite and greigite, respectively. This organic carbon appears to have participated in post-depositional redox reactions that produced the observed iron-phosphate and iron-sulfide minerals. Geological context and petrography indicate that these reactions occurred at low temperatures. Within this context, we review the various pathways by which redox reactions that involve organic matter can produce the observed suite of iron-, sulfur- and phosphorus-bearing minerals in laboratory and natural environments on Earth. Ultimately, we conclude that analysis of the core sample collected from this unit using high-sensitivity instrumentation on Earth will enable the measurements required to determine the origin of the minerals, organics and textures it contains. NASA's Mars 2020 Perseverance rover mission is the first in a sequence of missions designed to return a scientifically selected suite of Martian rock, regolith and atmosphere samples to Earth for laboratory investigation. The Perseverance rover carries an instrument payload designed to fulfil these goals, with the capability to characterize rock targets, their submillimetre-scale textural attributes, and potential organic and inorganic biosignatures, placing these features into an outcrop-scale context1. Perseverance has explored three geologic terrains in Jezero Crater (Supplementary Fig. 1): (1) the crater floor, which includes lava flows and igneous cumulates that have experienced aqueous alteration under a variety of conditions2,3; (2) the Western Fan, a sequence of sedimentary rocks derived from mafic to ultramafic sources and deposited in a fluvial-deltaic-lacustrine setting4,5,6; (3) the Margin Unit, a layered-to-massive sequence of rocks with strong orbital spectroscopic signatures of olivine and carbonate that is exposed between the crater rim and the Western Fan7. This study focuses on a suite of rocks exposed in Neretva Vallis, a valley incised through the Jezero Crater rim and Margin Unit, which was the feeder channel for the Western Fan8 (Supplementary Fig. Perseverance initially explored a distinct, bright-toned outcrop exposed on the northern margin of Neretva Vallis. This outcrop area is informally named ‘Bright Angel' (Fig. In High Resolution Imaging Science Experiment (HiRISE) images of this deposit, albedo variations appeared to indicate layering at the metre scale. Subsequently, Perseverance explored strata exposed along the southern margin of Neretva Vallis, in an area informally named ‘Masonic Temple' (Fig. 1a), where rocks with similar characteristics crop out. As described below, the outcrops in these areas share many characteristics and are referred to collectively as the Bright Angel formation. Subsurface structures detected by the Radar Imager for Mars' subsurface experiment (RIMFAX) ground-penetrating radar (GPR; Methods and Supplementary Fig. a, Orbital context image with the rover traverse overlain in white. This mosaic was collected on sol 1178 from the location of the Walhalla Glades target before abrasion. Downslope, about 50 m distant, the approximate location of the target Grapevine Canyon is also shown. Mastcam-Z enhanced colour RGB cylindrical projection mosaic from sol 1178, sequence IDs zcam09219 and zcam09220, acquired at 63-mm focal length. A flyover of this area is available at https://www.youtube.com/watch?v=5FAYABW-c_Q. The Bright Angel formation consists of approximately metre-scale blocks formed by fracturing and physical weathering of the exposed outcrop (Fig. In RIMFAX GPR profiles, radar-reflective layers express a range of apparent dip angles, from horizontal up to about 30° at the northern contact with the Margin Unit (Supplementary Fig. The Bright Angel formation appears to be subdivided into concave-upwards to flat-lying bodies of layered rock lying within the Neretva Vallis channel (Supplementary Fig. A representative example of Bright Angel formation outcrop is visible in the ‘Beaver Falls' workspace (Fig. In this rock, centimetre-scale, reddish-to-tan-coloured, recessive layers are separated by thinner, relatively resistant, light-toned layers (Figs. The layers in this rock dip more steeply and strike at an angle to layers in the nearby outcrop, implying that it may have been displaced from its original orientation. Topographically above but stratigraphically below this rock is a darker-toned rock with a coarsely granular texture, the site of the target ‘Steamboat Mountain' (Fig. This darker-toned rock was investigated as a possible transitional lithology between the Bright Angel formation and the Margin Unit. Mastcam-Z image mosaic of the Beaver Falls workspace on sol 1217. Downhill is to the left on this image. Mastcam-Z enhanced colour RGB vertical projection mosaic from sol 1217, sequence zcam09264, acquired at 110-mm focal length. a, WATSON nighttime image of Cheyava Falls with both white-light LED groups on acquired on sol 1188 at a stand-off distance of 3.91 cm. b, Colourized SHERLOC ACI image acquired on sols 1201–1202 outlined with a white box and overlain on the WATSON image from a. The ACI image is a focus merge of 13 ACI images acquired between stand-off distances of 4.035 cm and 4.335 cm and has an image resolution of about 10 µm per pixel. One 1 × 1 mm spectral scan was acquired at the blue square location at a stand-off distance of 4.035 cm. d, SHERLOC Raman spectra from representative targets in the Bright Angel unit with fits to an instrument –O-stretching overtone feature from the SHERLOC fused-silica optics (light grey, labelled ‘Bknd') and the G-band signal at about 1,600 cm−1 associated with organic carbon in the targets Walhalla Glades (blue), Cheyava Falls (red) and Apollo Temple (green fit on grey spectrum). Malgosa Crest (yellow) shows no G-band signal above the instrument background. In general, macroscopic rock textures in the Bright Angel outcrop area are diverse and complex. Intervals of rock are wind-fluted and massive in appearance (Supplementary Fig. 3a), show large (centimetre scale) nodular features (Supplementary Fig. Primary textures include layered and structureless intervals with limited evidence for transport and deposition by currents, such as cross bedding or plane bed laminations. Across Neretva Vallis, in the Masonic Temple area (Fig. 1a), outcrops express wind-fluted, massive, layered and granular surface textures such as those seen in the Bright Angel area (Supplementary Figs. However, the Masonic Temple area also includes poorly sorted conglomerates composed of rounded to subangular millimetre- to centimetre-scale clasts embedded in a fine-grained matrix, as seen in the targets ‘Bass Camp‘ (Supplementary Fig. Despite the textural diversity of rocks in the Bright Angel formation, they all contain a fine-grained component, or facies, that comprises much of the rock volume. Individual grains in this facies are not visible in WATSON images (17.9–36.3 μm per pixel) or in SHERLOC Autofocus and Context Imager (ACI) images (10.1 μm per pixel) of the Cheyava Falls target (Fig. 3b,c), indicating grains approximately ≤30–110 μm in diameter, that is, finer than coarse silt or very fine sand. The image resolution is insufficient to determine the relative proportions of clay and silt; accordingly, we classify this facies as mudstone. The Masonic Temple conglomerates contain a matrix made of the same mudstone, as well as abundant millimetre- to centimetre-scale clasts, also composed of mudstone (Supplementary Figs. These clasts formed by mud deposition, partial consolidation, and later erosion and transport as intraclasts. The mud facies is red, tan or whitish-grey coloured in different rocks. These colour differences are apparent in visible-light reflectance spectra collected by Mastcam-Z (Supplementary Fig. Masonic Temple targets are characterized by long-wavelength spectral features that indicate high abundances of ferric iron. Targets in Bright Angel show spectral features that indicate less Fe3+, resulting in tan to whitish-grey colouration. Near-infrared (NIR) absorption spectra collected by SuperCam (Supplementary Fig. 11a–c) are characterized by shallow, approximately 1.92-μm band depths, indicating that Bright Angel formation rocks are weakly hydrated, especially when compared with sedimentary rocks from the Western Fan (Supplementary Fig. Other NIR spectral features are also relatively weak and cannot be uniquely attributed to specific minerals; however, candidate phases include phyllosilicate clays and opaline silica, consistent with expectations for a mudstone. Calcium sulfate with spectral features consistent with bassanite, CaSO4·0.5H2O, is also identified in NIR spectra (Supplementary Fig. Organic matter was detected in the Bright Angel area mudstone targets Cheyava Falls, Walhalla Glades and Apollo Temple by the SHERLOC instrument based on the presence of an approximately 1,600-cm−1 G band in the Raman spectra9,10,11,12 (Fig. The G band is most intense in Apollo Temple and less intense in Walhalla Glades and Cheyava Falls (Methods). In contrast, no G band was detected at Masonic Temple in the abrasion target ‘Malgosa Crest'. SuperCam Raman spectra collected from Apollo Temple show a strong continuum fluorescence signature (Supplementary Fig. 12) consistent with, but not uniquely attributable to, organic matter; this signature is weak to absent in Malgosa Crest. PIXL micro-X-ray fluorescence (XRF) elemental analyses indicate that Bright Angel formation mudstone is rich in SiO2, Al2O3 and FeO, and poor in MgO and MnO (Supplementary Information). On ternary diagrams used to infer mineralogy (Fig. 4a,b), the mudstone plots in locations consistent with abundant silica and aluminosilicate clays. PIXL analysis of the X-ray diffraction properties of the mudstone in abraded patches (Supplementary Fig. 13a–d) shows that it contains only randomly distributed, low-intensity diffraction peaks with no large domains that exhibit coherent monocrystalline diffraction13,14,15. These properties indicate crystalline domain sizes at or near the PIXL diffraction detection limit of 40–60 μm (ref. 15), in accord with image-based grain size estimates. There is no variation in the crystallinity or textural properties of the mudstone with stratigraphic position or elevation (Supplementary Fig. 13a–d), and thus no evidence for contact metamorphic recrystallization in proximity to adjacent geologic units. Analysis of diffraction peaks detected in calcium-sulfate veins and nodules in three abrasion targets indicates that Ca-sulfate is predominantly non-diffracting as well (Supplementary Figs. A possible explanation for the high proportion of non-diffracting Ca-sulfate is that it represents fine-grained bassanite, consistent with SuperCam spectra (Supplementary Fig. On Mars, bassanite is known to form by dehydration of gypsum16, with gypsum precipitation implying low temperature and salinity for Ca-sulfate-precipitating fluids17. Ternary diagrams plotting molar proportions of the indicated oxides and elements. FeOT is total iron reported as FeO. Green points are all XRF analyses from the Bright Angel area and black points are from the Masonic Temple area. Expected trajectories for oxidative chemical weathering of igneous rocks shown as black arrows; most data points cluster in locations on the diagrams expected for sediments rich in SiO2, Al2O3 and FeOT, formed by weathering of their sources. In a, the cluster of green points near olivine (Ol) represents olivine grains in Steamboat Mountain and Cheyava Falls. Additional linear trends shown with dashed arrowheaded lines indicate mixing with Ca-sulfate (CaSO4) and Fe-rich phases that include Fe-phosphates, Fe-oxides, Fe-sulfides, (Mg, Fe)SO4 and (Mg, Fe)CO3. Kln, kaolinite; Opx, orthopyroxene; Cpx, clinopyroxene; An, anorthite; Ab, albite; Fld, all feldspars. c, Compositional arrays in Bright Angel indicate that an iron-phosphate with a molar 3FeO:2P ratio (stoichiometrically like vivianite, Fe3(PO4)2∙8H2O and its oxidation products) is the primary P-bearing phase. The location of 1FeO:1P ferric phosphate (typically strengite, FePO4·2H2O) is also shown. d, Compositional arrays in Bright Angel and Masonic Temple indicate the presence of CaSO4. In Bright Angel, an endmember with a molar 3Fe:4S ratio (stoichiometrically like greigite, Fe3S4) is also present. Pyrite (FeS2), troilite (FeS) and jarosite (Jar; KFe3+3(SO4)2(OH)6) are also plotted. The location of the Fe-sulfide minerals on d is based on calculations described in Methods. In Cheyava Falls, PIXL analysed an approximately 0.5–1-mm-diameter olivine grain in an approximately centimetre-thick, light-toned Ca-sulfate layer (Fig. In Steamboat Mountain, coarse to very coarse sand-sized grains (0.5–2 mm) of olivine and Fe–Mg carbonate are locally surrounded by Ca-sulfate and set within mudstone (Supplementary Fig. Mixing relationships between mudstone, olivine, carbonate and Ca-sulfate are illustrated on Fig. Where coarse-crystalline olivine grains and mudstone are in contact with each other (Supplementary Fig. 13b,f), there is no indication that the mudstone has been recrystallized by interaction with this igneous mineral, indicating that olivine is a detrital phase. In Steamboat Mountain, SHERLOC targeted an area containing olivine, carbonate and Ca-sulfate, rather than mudstone. No organic matter was detected, indicating that these phases are not important carriers of organic matter. SuperCam Raman spectra from the Steamboat Mountain mudstone, however, do show the same continuum fluorescence seen in the organic-matter-bearing Apollo Temple target (Supplementary Fig. Observations of the Bright Angel outcrop area are consistent with their interpretation as mudstones deposited from suspension as layered and massive beds. These characteristics are unlike those observed in Western Fan sedimentary rocks, which show little to no fractionation of Fe from Mg or Mn (Supplementary Fig. 16), indicating that they were formed under anoxic conditions18. 2), we observe interstratified bedsets containing both fine-grained mudstone and coarse-grained olivine-bearing layers (for example, Cheyava Falls). We also observe poorly sorted, mud-rich olivine- and carbonate-bearing lithologies (for example, Steamboat Mountain). These coarser-grained lithologies appear to have provided a higher permeability medium through which later Ca-sulfate-precipitating fluids could migrate. Across Neretva Vallis, at Masonic Temple, the poorly sorted, coarse conglomeratic nature of some outcrops (Supplementary Figs. 6), indicate significant local or temporal variation in current velocities. Accordingly, we interpret the Bright Angel formation to have formed from sedimentary processes that included weathering, erosion, transport and deposition from water by fallout from suspension and energetic currents or debris flows, forming mudstone and coarser-grained and conglomeratic lithologies, respectively. Dispersed throughout the fine-grained mudstone in the Bright Angel outcrop area, we observe approximately 100–200 μm circular to irregularly shaped masses (informally referred to as ‘poppy seeds' by the Mars 2020 Science Team) that are black to dark blue to dark green coloured in daytime and nighttime white light-emitting diode (LED) illumination (Fig. 17c–19c) and PIXL Micro Context Camera (MCC; Supplementary Fig. 20) decorrelation stretch images enhance the blue to green colour of these features relative to the mudstone they occur in. PIXL XRF data indicate that the masses are enriched in Fe, P and Zn relative to their host mudstone (Fig. 4c), they extrapolate towards a molar FeO:P ratio of about 3:2 (see also Supplementary Fig. The X-ray diffraction properties of these masses are indistinguishable from the surrounding mudstone, indicating crystallites smaller than 40–60 μm (Supplementary Fig. In the Apollo Temple abrasion, PIXL analyses also reveal abundant Fe-phosphate masses. The colour properties of some of the Fe-phosphate masses in this target are distinctive: colourless in daytime WATSON images, tan/orange-toned under nighttime white-light LED illumination, and red in decorrelation stretch images relative to the surrounding white–grey mud (Supplementary Fig. Combined, these properties are consistent with accumulations of microcrystalline vivianite (Fe2+3(PO4)2·8H2O) or lower-hydration-state ferrous phosphates (for example, phosphoferrite: Fe2+3(PO4)2·3H2O) and their oxidation products, for example, metavivianite (Fe2+2.5Fe3+0.5(PO4)2·7.5H2O), ferrostrunzite (Fe2+Fe3+2(PO4)2(OH)2·6H2O) and santabarbaraite (Fe3+3(PO4)2(OH)3·5H2O), all of which have molar FeO:P ratios of 3:2 and colour properties that match our observations in Bright Angel. The Fe-phosphate masses do not appear to have been sorted into laminations or lenses of relative enrichment, as might be expected for 100–200 μm accumulations of vivianite co-deposited with finer-grained aluminosilicates and silica. They are thus unlikely to have been transported and deposited as grains along with the surrounding muddy sediment. Instead, the distribution of the Fe-phosphate masses indicates they formed as the result of chemical processes after mudstone deposition and accordingly can be considered authigenic nodules. In the proximity targets ‘Grapevine Canyon' and Walhalla Glades, Fe-phosphate nodules are encased in larger, centimetre-scale Ca-sulfate nodules (Supplementary Figs. 13h and 20), which are themselves cross-cut by a later generation of light-toned, mineralized fractures (Supplementary Fig. Fe-phosphate nodule abundances and sizes seem unrelated to their proximity to either generation of Ca-sulfate, indicating that Ca-sulfate nodule and vein emplacement occurred after Fe-phosphate nodule formation. Fe-phosphate is rare in Masonic Temple area targets. A few isolated spots that might contain Fe-phosphate masses were detected by PIXL in the Malgosa Crest abrasion target and WATSON images showed a few large (about 1–5 mm) blue–green masses (Supplementary Fig. These could be authigenic nodules of a different scale or transported Fe-phosphate-enriched clasts. A striking feature observed in the Cheyava Falls target (and the corresponding Sapphire Canyon core sample), is distinct spots (informally referred to as ‘leopard spots' by the Mars 2020 Science Team) that have circular to crenulated dark-toned rims and lighter-toned cores (Fig. The spots range in size from about 200 μm to 1 mm in diameter and their cores are less red than their surrounding mudstone. Like the previously described authigenic nodules they co-occur with, the spots are not concentrated in layers or laminae; together with their irregular shapes, this indicates that they were not deposited as grains. Instead, these multi-coloured features appear to represent in situ reaction fronts. PIXL XRF analyses of reaction front rims reveal they are enriched in Fe, P and Zn relative to the mudstone they occur in (Fig. 5a,b); we interpret the rims to be made of the same Fe-phosphate minerals found in authigenic nodules. In the reaction front cores, a phase enriched in S-, Fe-, Ni- and Zn was detected (Fig. Insight into the identity of this phase comes from comparison with PIXL analyses of the Apollo Temple target in the same outcrop block as Cheyava Falls (Fig. 4d) shows that several spot analyses trend towards a composition on the Fe–S join with an Fe:S ratio of about 3:4. Most of these points are associated with a non-diffracting, millimetre-scale region in the sol 1213 Apollo Temple XRF map shown in Supplementary Fig. This region, like the reaction front cores, is also enriched in Zn and Ni, approaching values as high as 2,300 ± 670 ppm and 2,000 ± 570 ppm, respectively (Supplementary Information), We also detect copper in this region at a concentration of 419 ± 355 ppm (Supplementary Information). X-ray scattering properties indicate that both this region and the reaction front cores are light-element-deficient relative to FeO and SO3 (Supplementary Fig. In LED-illuminated colour images, the high Fe–S region of Apollo Temple contains small dull brown-to-black-coloured masses (Supplementary Fig. Taken together, these chemical and colour properties are consistent with the Fe-sulfide mineral greigite (Fe3+2Fe2+S4; also see Supplementary Fig. The sulfide-bearing region is adjacent to other submillimetre-scale mineral accumulations having properties consistent with Fe and S in a range of oxidation states, including jarosite [KFe3+2(SO4)3(OH)6], which is an oxidation product of Fe-sulfide19, and a red–brown Fe-rich phase with low SO3, SiO2, Al2O3 and analytical totals, consistent with siderite (Supplementary Figs. a, PIXL MCC colour image of the target Cheyava Falls showing the outline of the PIXL scan area, and the individual XRF analysis points summed together to determine the bulk chemical composition (Supplementary Tables 1–3) of mudstone (tan), reaction front cores (purple), and nodule and reaction front rims (green). b, Element mobility index (τi,TiO2; Methods) for the indicated elements in Cheyava Falls showing enrichment and depletion patterns for nodules and reaction front rims (green) and reaction front cores (purple) relative to the mudstone they are contained in. c, Normalized probability density function for the NIR/blue reflectance value of individual pixels in PIXL MCC images of the mudstone facies in each of the indicated targets. A higher NIR/blue ratio is associated with higher Fe3+ concentration, consistent with a more oxidized substrate (and vice versa). d, Maximum percentage of the total iron in each of the indicated targets that could be present as vivianite (bars with stippled fill), greigite (bars with diagonal line fill) and their sum (grey bars); calculations described in Methods. Malgosa Crest has the lowest calculated vivianite + greigite abundance, organic matter is not detected in this target and it is the reddest target of the group (Fig. Walhalla Glades and Cheyava Falls fall between these two extremes. Error bars in b and d represent propagated standard errors. Finally, in the Bright Angel formation mudstone facies, there is an inverse relationship between the inferred abundances of vivianite and greigite (Methods and Supplementary Information) versus mudstone oxidation state, inferred from the NIR/blue reflectance ratio in PIXL MCC images20. The NIR/blue ratio is sensitive to the relative abundance of Fe3+, like observations from Mastcam-Z (Supplementary Fig. This inverse relationship is shown in Fig. Mudstone oxidation state also appears to be inversely related to the strength of the SHERLOC Raman G band. In contrast, Malgosa Crest has no detectable organic matter, the lowest inferred vivianite + greigite abundance, and is the most-oxidized target analysed in the Bright Angel formation. Walhalla Glades and Cheyava Falls are intermediate between these two extremes. Except when found in authigenic nodules and reaction front rims, phosphate is not associated with a mineral phase (for example, there is no indication that apatite or merrillite are present; Fig. Accordingly, we suggest that during deposition, phosphate was adsorbed on Fe3+-, Al- and Si-rich sediment grains21. In the Bright Angel area, iron and phosphate have been redistributed into authigenic nodules and reaction front rims; mass balance calculations suggest closed-system reorganization of these chemical components into vivianite (Supplementary Text). Fe-phosphate-enriched masses are not associated with Al2O3 (Supplementary Fig. 21h), which might otherwise suggest the co-existence of Al-phosphate minerals typical of transport of Al3+ and Fe3+ under oxidizing, low-pH conditions, such as variscite (AlPO4·2H2O) and strengite (FePO4·2H2O)22,23 (Fig. Instead, transport of Fe2+, Zn2+ and PO43− probably occurred under non-oxidizing conditions, which, combined with moderate pH, prevented mobilization of Al3+. Such conditions favour the precipitation of vivianite23. The apparent absence of Fe-phosphate nodules and reaction fronts in most of the conglomerate-bearing Masonic Temple area suggests a depositional facies control on the development of these specific features. In the Bright Angel area, Fe-phosphate minerals are associated with organic matter (Fig. A pathway to the formation of vivianite is via the oxidation of this organic matter, which would have been coupled to the reductive dissolution of Fe3+ in sediment grains. Similar precipitation and redox reactions have been considered for an occurrence of Mn–P-rich nodules in Gale Crater24, and for submillimetre-scale mixed valence Fe-phosphate grains in a conglomerate outcrop in the Jezero Western Fan25,26. Sulfate reduction coupled to organic matter oxidation could also be responsible for precipitation of Fe-sulfide in the Apollo Temple target and in the cores of reaction fronts in Cheyava Falls. As reduced Fe- and S-bearing phases formed, the mudstone colour properties were modified by iron reduction, bleaching it of its red colour in proportion to the abundance of available organic matter (Fig. Here we consider the null hypothesis: that within the low-temperature sedimentary-diagenetic setting we have proposed for the Bright Angel formation, abiotic reactions produced ferrous Fe and reduced S and concentrated them in authigenic nodules and reaction fronts. The null hypothesis predicts that abiotic reactions can reduce sedimentary Fe3+ to aqueous Fe2+, which is then incorporated in the Fe-phosphate and Fe-sulfide minerals we have identified. A wide variety of organic carbon compounds are known to promote the abiotic reductive dissolution of ferric iron oxide minerals at temperatures between 10 °C and 80 °C (refs. The presence of organic matter in Bright Angel formation mudstone (Fig. 3d), which could have been produced on Mars through abiotic synthesis30,31 or delivered from non-biological exogenic sources30,32, suggests that such reactions could have occurred. Further analysis is required to determine whether the specific organic compounds present in the Bright Angel formation can drive the reduction of mineral-hosted sedimentary Fe3+ at low temperature. Another possible pathway to the production of Fe2+ is through the abiotic oxidation of pyrite by Fe3+ (aq)33. This process would require both the presence of detrital pyrite and low solution pH, which would permit Fe3+ (aq) to be present. As previously discussed, neither condition appears to be met in the Bright Angel formation. The null hypothesis also predicts that an abiotic source of dissolved sulfide was available to be incorporated in authigenic Fe-sulfide. Dissolved sulfide facilitates the reductive dissolution of ferric iron oxides, with half-lives ranging from years to hours depending on Fe-oxide mineralogy, crystallinity and pH34,35, providing another potential pathway to the production of Fe2+ (aq). Magmatic degassing of reduced sulfur-bearing gases (for example, ref. 36) to local groundwater could provide a potential source of dissolved sulfide during diagenesis. However, geological constraints demand that this sulfide migrate in from a distal, high-temperature sulfide-gas-producing system, to the low-temperature depositional-diagenetic environment of the Bright Angel formation. No evidence for sulfide-producing hydrothermal or magmatic systems was observed in the Crater Floor, Western Fan or Margin Unit before investigation of the Bright Angel formation. Abiotic reduction of sulfate to sulfide by organic matter is another possible source of dissolved sulfide that could both reduce Fe3+-bearing sediment and provide the reduced sulfur required to form Fe-sulfide minerals37. However, sulfate reduction by reduced carbon compounds is energetically demanding and kinetically inhibited by the symmetry of the SO42− ion38, so abiotic reaction rates are exceedingly slow at temperatures <150–200 °C (refs. As discussed previously, the Bright Angel formation shows no unambiguous evidence that it was heated in contact with adjacent geologic units, and burial to depths in excess of about 5 km would be required to achieve temperatures >150 °C during the Noachian39. On Earth, vivianite nodules are known to form in fresh water23,40,41 and marine42,43 settings as a by-product of low-temperature microbially mediated Fe-reduction reactions. Fe-sulfide minerals, such as greigite, pyrite and mackinawite, can also be formed as products of microbial sulfate reduction44,45, and have been observed in close spatial association with vivianite42. Greigite precipitation, from precursor mackinawite, is favoured by a high dissolved Fe2+/SO42− ratio, typical of diagenetic fluids in freshwater and ferruginous marine environments44. Repeated sulfidation of Fe2+ in vivianite followed by sulfide oxidation promotes stable incorporation of Zn and other heavy metals into vivianite46. Thus, Zn enrichment in nodules is consistent with a hybrid iron- and sulfate-reducing mechanism. Minerals like these, produced by Fe- and S-based metabolisms, provide some of the earliest chemical evidence for life on Earth47,48, and are thought to represent potential biosignatures in the search for life on Mars49. The fact that the reaction fronts observed in the Cheyava Falls target are defined by small, spot-shaped, bleached zones in an overall Fe-oxide-bearing, red-coloured rock invites comparison to terrestrial ‘reduction halos' in modern marine sediments50 and ‘reduction spots', which are concentrically zoned features found in rocks of Precambrian and younger age on Earth51. A biologically influenced origin has been proposed by some for reduction spots51, although this is not a universally held perspective52. Under a biological scenario, the mixture of reactants available in the Bright Angel formation at the time of deposition could have provided raw ingredients for a set of biological redox reactions that drove Fe and S reduction, organic matter oxidation, and precipitation of Fe2+-phosphate and Fe-sulfide minerals. In this scenario, oxidized iron and sulfate would be used as terminal electron acceptors for organic matter consumption, promoting the formation of minerals through the release of chemical by-products: Fe-phosphate minerals in the case of iron reduction and Fe-sulfide minerals in the case of sulfate reduction. Where authigenic nodules were formed, the reaction would have shut off before additional reductive processes occurred. In the places where larger reaction fronts formed, the presence of sulfide-bearing cores suggests that sulfate-reducing metabolisms with lower energy yields could have taken hold once those regions of the rock had been depleted of available Fe3+, but had not yet exhausted organic carbon in the reaction front core. In summary, our analysis leads us to conclude that the Bright Angel formation contains textures, chemical and mineral characteristics, and organic signatures that warrant consideration as ‘potential biosignatures'53,54,55, that is, “a feature that is consistent with biological processes and that, when encountered, challenges the researcher to attribute it either to inanimate or to biological processes, compelling them to gather more data before reaching a conclusion as to the presence or absence of life53”. This assessment is further supported by the geological context of the Bright Angel formation, which indicates that it is sedimentary in origin and deposited from water under habitable conditions. Many significant questions remain about the origin of the nodules and reaction fronts encountered by Perseverance. We suggest that further in situ, laboratory, modelling and field analogue research into both abiotic and biological processes that give rise to the suite of mineral and organic phases observed in the Bright Angel formation will improve our understanding of the conditions under which they formed. Naturally exposed rock surfaces were prepared by removing surface dust with the gas dust removal tool61. To observe less weathered or flatter surfaces, several rocks were abraded using the rover's abrading bit61 to produce circular approximately 5-cm-diameter-wide by approximately 5–10-mm-deep ‘abrasion patches'. Target names, surface preparation techniques, unit and outcrop associations, and the images that each target appears in are tabulated in Supplementary Table 3. An overview of SHERLOC image and spectral map acquisition and standard processing methods are described in refs. Colourized ACI images were produced using methods described in ref. A focus mechanism motor failure on sol 1024 required SHERLOC measurements reported here to rely entirely on positioning by Perseverence's robotic arm, without the capability to refine focus internally. Bright Angel measurements were therefore made at a systematic offset from best focus: sol 1180 Walhalla Glades was collected at approximately 1.2-mm offset, sol 1201 Cheyava Falls was collected at approximately 1.8-mm offset, and sol 1217 Apollo Temple was collected at approximately 1.1-mm offset. Sol 1242 Malgosa Crest was collected at <0.3-mm offset from best focus; the absence of a G-band signal at Malgosa Crest, despite its better placement, provides further confidence in the G-band peak classification of the other Bright Angel spectra. The spectral intensity is attenuated for measurements collected out of focus, with approximately 35% signal loss expected for 1.5-mm focus offsets. No other spectral artefacts are expected for out of focus measurements. The Raman D band, normally associated in visible Raman spectra with disordered carbonaceous matter such as kerogen, is not present in SHERLOC spectra from the Bright Angel targets. The D band is generally less prominent in deep-ultraviolet (for SHERLOC, 248.6 nm) Raman spectra than Raman spectra that rely on visible-light-wavelength excitation sources63. Furthermore, the focus offset associated with measurements collected at Bright Angel is expected to reduce the observed Raman signal of a potential D-band peak to the instrument noise level, as confirmed with focus offset tests on the organic-rich SaU 008 SHERLOC calibration target and with SHERLOC analogue laboratory instruments. The SHERLOC fused-silica window generates a minor optical background contribution that must be accounted for in SHERLOC spectra, as described by ref. 3d (grey traces) for comparison with target spectra, demonstrating the significantly larger contribution of the G-band signal to this spectral region for Bright Angel measurements. The sol number that the XRF and MCC data were collected on is shown, along with the number of XRF spectra (‘# XRF points') that were collected from each target. The bulk element oxide and element abundances provided were calculated using the PIQUANT software package57,65 and have been corrected to remove the effects of diffraction and topographic roughness using the methods described in refs. Where needed to support discussion in the paper or Supplementary Text, the bulk chemical composition of regions of interest (ROIs) within PIXL XRF scans are also shown. These are accompanied by PIXL MCC images that show the locations of the individual PIXL XRF points that are included in the ROI. For all ROIs included in the Supplementary Data Tables, we have also included a list of the PIXL Motion Counter (PMC #s) positions57 for each XRF point in the ROI. These PMC locations describe which individual points are included in the ROI. The element mobility index (τi,TiO2), shown in Fig. 5b, was calculated using the methods described in ref. 5d, was performed using chemical abundance data for individual PMCs in the ‘mudstone' ROIs shown in the Supplementary Data Tables for the targets Apollo Temple, Cheyava Falls, Walhalla Glades, Steamboat Mountain and Malgosa Crest. For each PMC in each mudstone ROI, the Fe and P abundances (in mmol g−1) were combined in a molar Fe:P ratio of 3:2 until all P had been combined with Fe to form vivianite. In the next step, the remaining Fe in each PMC (after calculation of vivianite abundance) was combined with S in the appropriate molar Fe:S ratio until either all the Fe or all the S had been combined to form greigite. Because PIQUANT assumes that elements are paired with oxygen atoms for the purposes of determining element abundances from the fitting of X-ray spectra, the Fe:S stoichiometry used for estimation of greigite abundance was modified to a value 3:4.42, which differs slightly from idealized griegite stoichiometry (Fe:S = 3:4). The oxide assumption in PIQUANT causes it to overestimate the abundance of S in greigite because it expects the S fluorescence signal to be attenuated by matrix oxygen. To account for this effect, we modelled idealized spectra in PIQUANT for Fe3S4 and FeOSO3 and then determined the ratio of Fe/S Kα1 fluorescence from each calculated spectrum. PIQUANT correctly reports a 1:1 molar ratio for FeOSO3. The same calculation for the Fe-sulfide minerals troilite (FeS), pyrrhotite (Fe0.875S) and pyrite (FeS2) yields Fe:S ratios of 1:1.15, 1:1.29 and 1:2.11, respectively. PIXL has two detectors and so can detect X-ray diffraction from crystalline materials when energies of PIXL's incident radiation happen to be in a diffracting condition with d-spacings and orientations of crystal lattices in the target materials14,15. When detected, X-ray diffraction can also be used to assess crystallinity, grain size and grain texture of a target14,15,66,68 and to distinguish among minerals with similar elemental compositions13,26. Here diffraction from the Ca-sulfate-rich areas in PIXL scans in Apollo Temple, Walhalla Glades and Steamboat Mountain abrasion patches were compared with modelled diffraction patterns for gypsum, anhydrite and bassanite to identify the most likely mineral phase that matches the observed diffraction as described in detail in ref. A mineral was selected as indexed when the P value from the Fisher-transformed cross-correlation results13,69 was ≤0.01. Where P > 0.01, no mineral was selected as indexed. Owing to the difficulty phasing bassanite16, beam locations that indexed as bassanite were also treated as not indexed. A summary of the PIXL diffraction results is presented in Supplementary Table 2. The data presented in this paper are available on the NASA Planetary Data System Geoscience Node and Imaging and Cartography Node, which host dedicated repositories for data derived from the Mars 2020 Rover mission. Quantification of PIXL XRF data was conducted using PIQUANT65, a fundamental parameters XRF analysis software package developed for PIXL57. PIQUANT is embedded in the data visualization software package called PIXLISE57,70,71, which was used for analysis of quantified PIXL XRF data. The PIXLISE and PIQUANT software packages can be accessed at PIXLISE.org. PIXLISE source code versions are archived for reproducibility at OFS.io72. Farley, K. A. et al. Mars 2020 mission overview. Farley, K. A. et al. Aqueously altered igneous rocks sampled on the floor of Jezero Crater, Mars. Perseverance rover reveals an ancient delta-lake system and flood deposits at Jezero Crater, Mars. Stack, K. M. et al. Sedimentology and stratigraphy of the Shenandoah Formation, Western Fan, Jezero Crater, Mars. Bosak, T. et al. Astrobiological potential of rocks acquired by the Perseverance rover at a sedimentary fan front in Jezero Crater, Mars. Horgan, B. H. N., Anderson, R. B., Dromart, G., Amador, E. S. & Rice, M. S. The mineral diversity of Jezero Crater: evidence for possible lacustrine carbonates on Mars. Steele, A. et al. Comprehensive imaging and Raman spectroscopy of carbonate globules from Martian meteorite ALH 84001 and a terrestrial analogue from Svalbard. Steele, A. et al. Macromolecular carbon in Martian basalts. Scheller, E. L. et al. Inorganic interpretation of luminescent materials encountered by the Perseverance rover on Mars. Orenstein, B. J. et al. In-situ mapping of monocrystalline regions on Mars. Tice, M. M. et al. Alteration history of Seitah Formation rocks inferred by PIXL X-ray fluorescence, X-ray diffraction, and multispectral imaging on Mars. Vaniman, D. T. et al. Gypsum, bassanite, and anhydrite at Gale Crater, Mars. Hardie, L. A. Gypsum-anhydrite equilibrium at one atmosphere pressure. Babechuk, M. G., Widdowson, M. & Kamber, B. S. Quantifying chemical weathering intensity and trace element release from two contrasting basalt profiles, Deccan Traps, India. Burns, R. G. & Fisher, D. S. Iron–sulfur mineralogy of Mars—magmatic evolution and chemical-weathering products. Henneke, J. et al. A radiometric correction method and performance characteristics for PIXL's multispectral analysis using LEDs. Rampe, E. B., Morris, R. V., Archer, P. D., Agresti, D. G. & Ming, D. W. Recognizing sulfate and phosphate complexes chemisorbed onto nanophase weathering products on Mars using in-situ and remote observations. Roncal-Herrero, T., Rodríguez-Blanco, J. D., Benning, L. G. & Oelkers, E. H. Precipitation of iron and aluminum phosphates directly from aqueous solution as a function of temperature from 50 to 200 °C. & Dell, C. Diagenetic formation of iron phosphates in recent lake sediments. Treiman, A. H. et al. Manganese–iron phosphate nodules at the Groken Site, Gale Crater, Mars. Hausrath, E. M. et al. Phosphates on Mars and their importance as igneous, aqueous, and astrobiological indicators. Kizovski, T. et al. Fe-phosphates in Jezero Crater as evidence for an ancient habitable environment on Mars. Miller, W. P., Zelazny, L. W. & Martens, D. C. Dissolution of synthetic crystalline and noncrystalline iron oxides by organic acids. & Matijević, E. Interactions of metal hydrous oxides with chelating agents: IX. Reductive dissolution of hermatite and magnetite by aminocarboxylic acids. Ionescu, D., Heim, C., Polerecky, L., Thiel, V. & De Beer, D. Biotic and abiotic oxidation and reduction of iron at circumneutral pH are inseparable processes under natural conditions. Eigenbrode, J. L. et al. Organic matter preserved in 3-billion-year-old mudstones at Gale Crater, Mars. Steele, A. et al. Organic synthesis associated with serpentinization and carbonation on early Mars. Flynn, G. J., Nittler, L. R. & Engrand, C. Composition of cosmic dust: sources and implications for the early Solar System. Rimstidt, J. D. & Vaughan, D. J. Pyrite oxidation: a state-of-the-art assessment of the reaction mechanism. Afonso, M. D. & Stumm, W. Reductive dissolution of iron(III) (hydr)oxides by hydrogen-sulfide. Canfield, D. E., Raiswell, R. & Bottrell, S. The reactivity of sedimentary iron minerals toward sulfide. The sulfur content of volcanic gases on Mars. McSween, H. Y. Jr., Labotka, T. C. & Viviano-Beck, C. E. Metamorphism in the Martian crust. Cosmidis, J. et al. Biomineralization of iron-phosphates in the water column of Lake Pavin (Massif Central, France). Vuillemin, A. et al. Vivianite formation in ferruginous sediments from Lake Towuti, Indonesia. Hsu, T. W., Jiang, W. T. & Wang, Y. S. Authigenesis of vivianite as influenced by methane-induced sulfidization in cold-seep sediments off southwestern Taiwan. Liu, J. R. et al. Vivianite formation in methane-rich deep-sea sediments from the South China Sea. Picard, A., Gartman, A., Clarke, D. R. & Girguis, P. R. Sulfate-reducing bacteria influence the nucleation and growth of mackinawite and greigite. Xu, Z. Y. et al. Sulfidation-reoxidation enhances heavy metal immobilization by vivianite. In situ Fe and S isotope analyses in pyrite from the 3.2 Ga Mendon Formation (Barberton Greenstone Belt, South Africa): evidence for early microbial iron reduction. & Little, B. Mineralogical biosignatures and the search for life on Mars. Thomson, J., Higgs, N. C. & Colley, S. A geochemical investigation of reduction haloes developed under turbidites in brown clay. Kawahara, H. et al. Bleached-spot formation in Fe-oxide-rich rock by inorganic process. Gillen, C., Jeancolas, C., McMahon, S. & Vickers, P. The call for a new definition of biosignature. Mustard, J. F. et al. Report of the Mars 2020 Science Definition Team (Mars Exploration Program Analysis Group, 2013). Hamran, S.-E. et al. Radar Imager for Mars' Subsurface Experiment—RIMFAX. Allwood, A. C. et al. PIXL: Planetary Instrument for X-ray Lithochemistry. The SuperCam instrument suite on the NASA Mars 2020 rover: body unit and combined system tests. The Mars 2020 Perseverance rover Mast Camera Zoom (Mastcam-Z) multispectral, stereoscopic imaging investigation. Diverse organic-mineral associations in Jezero Crater, Mars. B., Czaja, A. D. & Williford, K. H. Deep-UV Raman spectroscopy of carbonaceous Precambrian microfossils: insights into the search for past life on Mars. Jakubek, R. S. et al. Spectral Background Calibration of Scanning Habitable Environments with Raman and Luminescence for Organics and Chemicals (SHERLOC) spectrometer onboard the rover enables identification of a ubiquitous Martian spectral component. Heirwegh, C. M., Elam, W. T., O'Neil, L. P., Sinclair, K. P. & Das, A. The focused beam X-ray fluorescence elemental quantification software package PIQUANT. Diverse and highly differentiated lava suite in Jezero Crater, Mars: constraints on intracrustal magmatism revealed by Mars 2020 PIXL. Liu, Y. et al. An olivine cumulate outcrop on the floor of Jezero Crater, Mars. Frequency distribution of the values of the correlation coefficient in samples from an indefinitely large population. 28th International Conference on Intelligent User Interfaces 91–105 (Association for Computing Machinery, 2023). Schurman, D. et al. PIXELATE: novel visualization and computational methods for the analysis of astrobiological spectroscopy data. Davidoff, S. et al. PIXLISE spectroscopy analysis software: released versions for published analyses. This work was carried out by A.C.A., M.L.C., K.P.H., K.U., S.D., K.A.F., S.W.L., Y.L., K.M.S., L.A.W., C.M.H. at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with the National Aeronautics and Space Administration (80NM0018D0004). Department of Geology and Geophysics, Texas A&M University, College Station, TX, USA Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, USA A. C. Allwood, M. L. Cable, K. P. Hand, K. Uckert, S. Davidoff, Y. Liu, K. M. Stack, L. A. Planetary Science Institute, Tucson, AZ, USA School of Earth and Space Exploration, Arizona State University, Tempe, AZ, USA Department of Earth, Atmospheric, and Planetary Sciences, Purdue University, West Lafayette, IN, USA Deutsches Zentrum für Luft- und Raumfahrt (DLR), Institute of Optical Sensor Systems, Berlin, Germany Institut de Recherche en Astrophysique et Planétologie, CNRS, Univ. Johns Hopkins University Applied Physics Laboratory, Laurel, MD, USA Earth, Environmental and Planetary Sciences, University of Tennessee, Knoxville, TN, USA Department of Earth Sciences, Brock University, St. Catharines, Ontario, Canada Astromaterials Research and Exploration Science Division, NASA Johnson Space Center, Houston, TX, USA Department of Earth, Environmental, and Planetary Sciences, Rice University, Houston, TX, USA Joanneum Research Institute for Digital Technologies, Graz, Austria Department of Earth, Planetary and Space Sciences, University of California Los Angeles, Los Angeles, CA, USA Malin Space Science Systems, San Diego, CA, USA Geology Department, Western Washington University, Bellingham, WA, USA Carnegie Science Earth and Planets Laboratory, Washington DC, USA Lunar and Planetary Institute, Houston, TX, USA Blue Marble Space Institute of Science, Seattle, USA IMPMC, UMR 7590 SU, CNRS, MNHN, IRD Biomineralogy Team Jussieu Campus, Paris, France Museum National d'Histoire Naturel, CNRS UMR 7590, Sorbonne Université, IMPMC, Paris, France Centro de Astrobiología (CAB), CSIC-INTA, Madrid, Spain INAF-Astrophysical Observatory of Arcetri, Florence, Italy Department of Geoscience, UNLV, Las Vegas, NV, USA Laboratoire Planétologie et Géosciences, CNRS UMR6112, Nantes Université, Univ. Institute of Geosciences, CSIC-UCM, Madrid, Spain Mars Exploration Program, NASA Headquarters, Washington DC, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Investigation: R. Barnes, A.B., P.B., K.B., S.B., O.B., R. Bhartia, T.B., A.J.B., A.P.B., G.C., E.L.C., E.C., E.A.C., A.C., E.D., K.A.F., D.T.F., T. Fornaro, T. Fouchet, B.G., S.G., S.-E.H., K.P.H., K.H.-L., J.A.H., J.R.J., A.J.J., M.W.M.J., P.S.J., L.C.K., H.K., T.V.K., D.A.K., S.W.L., A.Y.L., Y.L., J.N.M., L.M., N.M., J.A.M., J.M.-F., E.L.M., A.E.M., J.I.N., L.P.O'N., B.J.O., D.A.P., C.Q.-N., M.S.R., P.R., E.S., M.E.S., M.D.S., S. Sharma, D.L.S., K.L.S., S. Siljeström, J.I.S., K.M.S., A. Steele, M.M.T., A.H.T., K.U., S.J.V., L.A.W., B.P.W., R.C.W., K.H.W. Writing—review and editing: R. Barnes, P.B., K.B., S.B., O.B., R. Bhartia, T.B., A.B., A.P.B., M.L.C., G.C., E.L.C., E.C., E.A.C., E.D., A.G.F., D.T.F., T. Fornaro, T. Fouchet, F.G., S.G., K.P.H., E.M.H., C.D.K.H., K.H.-L., J.A.H., J.R.J., A.J.J., M.W.M.J., P.S.J., L.C.K., T.V.K., D.A.K., A.Y.L., Y.L., L.M., N.M., J.A.M., J.M.-F., F.M.M., E.L.M., A.E.M., J.I.N., L.P.O'N., B.J.O., C.Q.-N., M.S.R., P.R., M.E.S., M.D.S., S. Sharma, D.L.S., K.L.S., S. Siljeström, J.I.S., A. Srivastava, K.M.S., A. Steele, M.M.T., N.J.T., A.H.T., K.U., S.J.V., L.A.W., B.P.W., R.C.W. Nature thanks Janice Bishop, Aude Picard and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. This file contains Supplementary Text, Tables 1–3, Figs. File containing tabulated PIXL XRF data for rock bulk and region of interest compositions, as well as data used in the construction of Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Hurowitz, J.A., Tice, M.M., Allwood, A.C. et al. Redox-driven mineral and organic associations in Jezero Crater, Mars. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/is-there-life-on-mars-this-rock-may-hold-the-answer/'>This Martian Rock Might Be the Closest We've Come to Finding Alien Life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 15:05:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>NASA's Perseverance Mars rover took this selfie, made up of 62 individual images, on July 23, 2024. Were it found on Earth instead of Mars, the rock's speckles would likely be interpreted as evidence for a microbial feeding frenzy that occurred long ago. But getting certainty about what this rock truly contains likely requires hauling it off the Red Planet and delivering it back to Earth—an ambitious multiphase mission that NASA calls Mars Sample Return. The rock in question is packed with organic carbon—another promising prerequisite for life—and lies within a lithic formation called Bright Angel, which is exposed along a channel called Neretva Vallis. Eons ago, that now dry channel was a river valley, formed by water rushing into and feeding a lake-and-delta system in what's now known as Jezero Crater. Those apparently warm, wet origins led to NASA targeting Jezero as the landing spot for the space agency's Perseverance rover, which has been exploring the site for any past or present signs of life since it touched down in 2021. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. “This result gives us reason to consider the possibility that Mars was host to microbial life,” says the new study's co-author Joel Hurowitz, a geoscientist at Stony Brook University. Although this is nowhere close to direct, clinching evidence of Martian life, Hurowitz and his team hope that the findings help scientists “make progress in our quest to understand whether there's life on other planets in the solar system and beyond.” After NASA's initial teaser, scientists who were eager for more information paid close attention to a presentation by Hurowitz in mid-March of this year at the Lunar and Planetary Science Conference in the Woodlands, Tex. During his talk, Hurowitz recounted how he and his colleagues on the Perseverance team had found a carbon-rich rock with small patches ranging from black to dark blue to dark green, which they called “poppy seeds,” as well as larger patches with dark-toned rims and lighter-toned centers, which they referred to as “leopard spots.” They called the rock itself “Cheyava Falls.” But it can also occur abiotically via chemical reactions that happen at high temperatures. In their Nature study, Hurowitz and his co-authors elaborate on the likely mineralogical composition of Cheyava Falls and its speckles, as well as on the conditions in which it probably formed. Perseverance's measurements suggested the speckles contained the minerals vivianite, an iron phosphate, and greigite, an iron sulfide—and that both formed in close association with organic carbon. When found together on Earth, these minerals and organic molecules are usually considered a sort of biosignature, a physical sign of past or present life—at least if they can be shown to have formed at low temperatures rather than via hotter, less life-friendly conditions. A rock nicknamed “Steamboat Mountain” (right) also shows an abrasion patch. “We believe that these features happened early in the life of the sediment, shortly after it was deposited and likely before it had been ‘lithified' to form hard rock,” Hurowitz says. The team stops short of calling the speckles a sign of life, however, preferring the more prudent label of “potential biosignature.” Further evidence for or against a possibly biological origin, they say, is unlikely to arrive unless and until NASA brings a sample of the rock back to Earth for more thorough studies. In July 2024 Perseverance gathered just such a specimen from the rock—a core sample dubbed “Sapphire Canyon”—but NASA's beleaguered, multibillion-dollar plan to bring this and other samples back to Earth is far from a sure thing, given that it was targeted for cancellation in the White House's proposed budget earlier this year. “These results are super exciting,” says Janice Bishop, a planetary scientist at the SETI Institute, who has tested chemical reactions using analogue Martian rocks and co-authored a commentary that accompanies the new Nature paper. These reactions between substances also demonstrate energy sources that could have been used by early microorganisms. Yet because these Mars rocks are billions of years old, Bishop says, that offers abundant time in which the speckles could conceivably have emerged from abiotic processes: the small pockets of reduced vivianite and sulfides in the ancient mudstones at Jezero Crater could have been formed long ago by run-of-the-mill chemical reactions from organic compounds or other sources rather than by hungry microbes. More than once, scientists have declared they've found life on Mars based on piecemeal evidence—and each time so far, those claims have been ultimately dismissed as misinterpretations of entirely abiotic phenomena. Time will tell whether this latest case proves to be part of that same trend—or a genuine new discovery. Humberto Basilio is a Mexican science journalist covering policy, health, misconduct, archaeology and the environment. His work has been published in the New York Times, National Geographic, Science, Nature, and more. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a66016732/time-crystal-visible/'>Scientists Created the First Time Crystal You Can See With Your Own Eyes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. However, time crystals (as the name suggests) is similarly well-organized but across time, moving and transforming in a repeating cycle. Nobel laureate Frank Wilczek first proposed the idea of time crystals back in 2012, and since then, scientists and quantum computers alike have constructed various types of time crystals in order to understand their mind-bending properties. Now, researchers at the University of Colorado Boulder (UC Boulder) have achieved a new milestone: They've created a time crystal you can actually see with the naked eye… at least, under certain circumstances. These liquid crystals are sandwiched between two plates of glass coated with a photoresponsive dye, and light polarizes the dye, exerting pressure on the crystal and creating a complex series of interactions that repeat over time. All you do is shine a light, and this whole world of time crystals emerges,” Smalyukh said in a press statement. You shine a light, and the whole thing happens.” The resulting effect was a repeating wave—stripes repeating one after the other—that lasted for hours even when subjected to different temperature and light conditions. All of this comes with a big “so what?” question attached. Well, UC Boulder notes that future technologies could create more elaborate patterns by stacking time crystals, creating a novel method of storing data. More practically, time crystals could be used as an anti-counterfeiting technology. “I think there are opportunities to push this technology in all sorts of directions.” The Next Computer Revolution Will Come From a Lab ‘Fairy Circles' May Be the Future of Green Energy Information Could Be the Fifth State of Matter The Maya Kingdom Collapsed Due to Burning Events Humans May Be Able to Grow New Teeth in 5 Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/car-technology/a66016667/carbon-fiber-flax-fiber/'>Move Over Carbon Fiber—There's A New High-Performance Material In Town</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Carbon fiber has become ubiquitous in performance automobiles (and many other industries), primarily for its superior strength-to-weight properties. That's why many engineers hope that more natural materials could be a way forward, and have started to use flax fibers as an alternative to carbon fibers. By themselves, these fibers have considerably high tensile (pulling) strength. Once that's been done, the object is heat-treated for additional strength, and voila! One Swiss company, Bcomp, is currently leading the charge in developing these natural composite materials. Its product, called ampliTex, can be created using existing methods used to make carbon fiber and fiberglass components—Bcomp claims that manufacturers will even be able to using the same molds. Bcomp, a leader in environmentally friendly composite materials, has outlined some others. They add that using a bio-based resin can reduce CO2 emissions by up to 96 percent. Carbon fiber production creates a considerable amount of waste, and while it's true that these fibers can be recycled, that process can be energy and labor-intensive. However, Bcomp can easily recycle its production scrap into pellets for use in other applications, or to be fed back into a different step of the production process. While they still release carbon dioxide back into the atmosphere after being incinerated, Bcomp claims that the same process happens naturally when plant matter decomposes. Effectively, they're saying their product is following the natural carbon cycle. BMW recently announced it would be using Bcomp's flax-based materials for many of its exterior parts, interior parts, and pieces that were previously made from carbon fiber. This includes components such as the roof, mirrors, diffusers, and even interior trim pieces. We weren't able to find any performance claims about these new components directly from BMW, but we can confirm that they've already been tested at the racetrack. Before its recent announcement, BMW and Bcomp worked together to prove these flax composites in the crucible of motorsport. Racing is much more mechanically demanding than your daily commute, so if your material, part, or piece can survive the race track, there's a good chance it'll fare just fine on the road. Back in 2020, McLaren's Formula 1 team trialled the idea of a flax fiber seat for its drivers. For instance, the team's engineers discovered that Bcomp's natural ampliTex material provided better vibration absorption and impact resistance than conventional carbon fiber. It's not rocket science either: getting shaken around zaps your energy, and that can add up over the course of a two hour grand prix. You're probably not surprised to hear that many of the components inside your own car are made from plastic—it's been the go-to option for many years, as it's cheap to make. Not only is producing these components much kinder to the environment, but Polestar claims that they're lighter and reduce vibrations by up to 250 percent. Conveniently, flax fiber composites can be utilized for many applications beyond the automotive industry. The new environmentally conscious material is also being optimized for use in mass transportation, sports, marine, lifestyle, and infrastructure. While the performance advantages of flax fiber materials are not to be sniffed at, it might have even bigger implications. There's plenty to be learned, even if we focus on just Municipal Solid Waste—single-use plastics and cardboard are two of the most common and plentiful materials that many of us see every day. Looking at the Environmental Protection Agency's graphic above, you'll notice that even though we've implemented recycling, landfill usage has actually increased in recent years. Material science a critical tool that can steer us into the future with composites like flax fiber, which minimize environmental impact while optimizing performance. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. The Pros and Cons of Leasing vs Buying a Car</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250910000258.htm'>Don't toss cannabis leaves. Scientists just found rare compounds inside</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 11:20:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Sixteen of these compounds were tentatively identified as flavoalkaloids. "Most plants contain highly complex mixtures of phenolic compounds, and while flavonoids occur widely in the plant kingdom, the flavoalkaloids are very rare in nature," she explains. "We know that Cannabis is extremely complex - it contains more than 750 metabolites - but we did not expect such high variation in phenolic profiles between only three strains, nor to detect so many compounds for the first time in the species. Especially the first evidence of flavoalkaloids in Cannabis was very exciting." For her postgraduate studies in SU's Department of Chemistry and Polymer Science, she developed powerful analytical methods combining comprehensive two-dimensional liquid chromatography and high-resolution mass spectrometry for the detailed characterisation of phenolic compounds. "We were looking for a new application for the methods that I developed, after successfully testing them on rooibos tea, grapes and wine. I then decided to apply the methods to Cannabis because I knew it was a complex sample, and that Cannabis phenolics have not been well characterised," she explains. Prof. De Villiers says it is obvious there is still much to gain from studying Cannabis, as the bulk of research in this field to date has been focused on the pharmacological properties of the mood-effecting cannabinoids. "Our analysis again highlights the medicinal potential of Cannabis plant material, currently regarded as waste. New Smart Pimple Patch Clears Acne in Just 7 Days Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/atlantic-hurricane-season-has-gone-quiet-heres-why/'>At the Peak of Hurricane Season, the Atlantic Is Quiet. Here's Why</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Hurricane activity in the Atlantic basin is historically at its peak on September 10—but not this year In May, as Atlantic hurricane season loomed, meteorologists worried that above-average tropical activity, combined with cuts to the federal government's weather agency, could result in disaster. But so far, the season's effects have been mild. Here's what you should know about hurricane activity right now and this year in general. The most recent tropical storm, Fernand, dissipated on August 28, nearly two weeks ago. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. And today the National Oceanic and Atmospheric Administration's National Hurricane Center displays that most optimistic of Atlantic basin forecasts: an empty map labeled “no disturbances”—an indication that meteorologists don't expect any tropical activity of note within the next seven days. READ MORE: Hurricane Science Has a Lot of Jargon—Here's What It All Means Such a stretch of tropical quiet isn't particularly strange, says Matthew Rosencrans, a meteorologist and lead hurricane season forecaster at the National Oceanic and Atmospheric Administration's National Centers for Environmental Prediction, but it's rarer to see such a lull right over the traditional peak of hurricane season. Rosencrans says that the last September 10 with no tropical storm activity at all was in 2016. And sure enough, Hurricane Erin formed and then exploded into a Category 5 storm, albeit one that predominantly stayed over the ocean. And besides atmospheric pressure, many of the key factors for hurricane formation are present. Further, hurricanes feed off hot ocean water, which has been plentiful this year. “The ocean conditions are absolutely ready for anything,” McNoldy says. Wind shear—in which winds at different heights point in different directions, a phenomenon that can tear apart a brewing storm—has been a little high but not enough to prevent serious tropical activity if other factors align. READ MORE: 20 Years after Katrina, Major Hurricane Forecasting Advances Could Erode That means it's far too early to write off this hurricane season, Rosencrans and McNoldy agree. So far a total of six tropical storms have formed. That means the season overall—which officially runs from June 1 to November 30—is unfolding to be about as strong as predicted—and has plenty potential risk remaining. Even just last year offers a cautionary tale, Rosencrans says: after a similar lull in late August and early September, tropical activity spiked to the highest on record for late September and onward, including the deadly hurricanes Helene and Milton. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/what-is-alpha-gal-syndrome-symptoms-causes-and-way-to-get-tested/'>Imagine Becoming Allergic to Burgers, Bacon—Even Lipstick. A Tick Bite Can Do That</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Alpha-Gal Syndrome Explained: The Tick-Borne Allergy Affecting Diets Worldwide By Rachel Feltman, Fonda Mwangi & Alex Sugiura And the culprit is a tiny tick bite. Alpha-gal syndrome is one of the strangest allergies we know of: it's a tick-bite-triggered sensitivity to a sugar molecule found in the meat and fat of almost all nonhuman mammals. And as our winters get warmer and tick populations expand more people are finding themselves unable to enjoy their favorite foods—sometimes permanently. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Here to explain how a tick bite can completely change your dietary requirements and what you can do to protect yourself is Lee Haines, an associate research professor at the University of Notre Dame. Thanks so much for coming on to talk with us today. Lee Haines: I am chuffed to be here. Haines: So alpha-gal syndrome is a bizarre thing. And this sugar is found primarily in red meats, so people sometimes call it a red meat allergy, which isn't quite true because it's in [almost all] meat products that are not human. And people have to really change their lifestyle when they are diagnosed with this allergy. Feltman: And why did we start hearing about it in the U.S. a few years ago? Haines: It took a while to be discovered in the U.S. There were people talking about becoming allergic to their hamburgers, and it took a while for collective data to get on the radar of the medical system to say, “Hey, something is going on here. And at the same time this is not just a U.S. problem; it's global. So a doctor in Australia was noticing the same thing in her patients, and it just took time for the science to catch up to figure out why people were getting sick and to link it to the bite of a tick, which is an unusual thing to think about triggering an allergy. Haines: Yes, it is becoming more common. So first of all, it's a funky story, no [laughs]? Haines: So the media liked to pick up on it, which means that more people hear about it. And then the second is—maybe not in this order—but our winters are getting warmer, so there are a lot more ticks surviving, which means [there's] more of an opportunity to be bitten. But now we also have diagnostics to make sure that we know if you do have alpha-gal or not, so we can give some hard evidence that this is here, whereas, say, 25 years ago we didn't even have a diagnostic. Feltman: And so what are the signs that someone might have alpha-gal syndrome? Haines: Oh, you know, the range is incredibly diverse. It's quite often mistaken as [irritable bowel syndrome] ... So you kind of get stomach cramps, vomiting, diarrhea, and then it progressively gets worse. So it depends on how big of a dose of sugar you got, or you ate, 'cause some meat has a lot more of this sugar than others, like pork kidney—whoa. Steak and kidney pie [laughs], that will trigger an allergy even if you're just slightly sensitized. So some people have to carry an EpiPen. And then arrange to either talk with your doctor or get a referral to an allergist, and take your journal with you and say, “Look, this is what I've noticed.” And they should, right away, ask you if you want to get tested for alpha-gal syndrome, which is just a simple blood test. Haines: You might have to carry an EpiPen if you really are allergic to everything 'cause you can become allergic to any type of product coming from an animal, so dairy, cheese [laughs], cosmetics, medications, gelatin—like, many things could trigger your allergy. Feltman: Mm, and is there any treatment or cure? It hasn't been around long enough for us to, as scientists, to work on that and have definitive answers. But certainly, people are thinking about it as a peanut allergy—you know, you can go through desensitization—and I think there's a movement to try and develop something for sensitization against a meat allergy. But I would say the cure, in some cases, is to cut out anything that will activate your immune system ... And if you can do that, say, for five years [laughs]—not eat any meat products or become exposed to tick bites—oftentimes the person, their antibody levels will drop ... So if you can get those levels in your blood to drop lower, we are finding that people can go back to eating meat products. So it's about a long period of abstinence, and then maybe you can enjoy your bacon again. Feltman: Well, at least there's some hope [laughs] for bacon lovers who are suffering. Other people, their antibody levels are so high [laughs] that they probably won't be able to eat meat again. Feltman: So how can folks avoid this and, of course, other tick-borne illnesses? Haines: Oh, well, I have some bad news: ticks like surviving, and they are incredibly difficult to control and kill. We can't just spray and hope to get rid of all the ticks. So it really depends on our own ability to protect ourselves from tick bites, which means using 30 percent DEET or 20 percent picaridin. So if you're outside wandering around, when you come in at night check your body to make sure you have no bloodthirsty hitchhikers, and throw your clothing into the dryer for 10 minutes. They have an Achilles tendon that you can exploit. So other than wearing repellents and doing the tick check, you can also buy permethrin-treated clothing, and ticks, they'll crawl on you, but they'll die while they're crawling on you [laughs]. So it's quite satisfying to see ticks all over your lower legs, and they're just kind of clinging on there [laughs], they're dying [laughs], so they won't feast on you. But those are the main ways that you can prevent tick bites from happening in the first place. So when a tick first lands on your skin, it has to drill through your skin to access your blood, and it takes time to do that; it's not instant like a mosquito. So in that time period, usually 12 to 24 hours, it starts to embed its head into your skin, and you grab it with tweezers, really tight, pinch around the head and pull straight up. And the reason you do this and you don't try to squeeze his body or light a match to it or put petroleum jelly or anything like this: you don't wanna stress the tick. It also can deliver things like bacteria, borrelia that causes Lyme disease, or other nasty diseases like a parasite disease called [babesiosis], which is like the malaria of the tick world—very dangerous, quite rare but very dangerous. But there's many different pathogens—like, intruders, invaders—that a tick can transmit. And unfortunately, in North America there are [laughs] many things that they carry. And the thought was that they were the only tick that could spread this allergy, but now, if you look globally, many different species of ticks spread this allergy, including the tick that spreads Lyme disease. So there was a case just recently in Maine where the woman developed alpha-gal syndrome, and she was never bitten by a lone star tick, just [a] black-legged [tick], which is really scary ... And the lone star ticks are moving north, which is also scary because [laughs] they're very aggressive ticks that lay thousands of eggs, so they quite quickly can take over a system. So you're like, “Is that freckle moving?” [Laughs.] Feltman: Well, thank you so much for coming on to chat with us about this. We'll be back on Friday with a mind-bending conversation about the very nature of reality. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper and Jeff DelViscio. This episode was edited by Alex Sugiura. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. You can even gift someone a subscription. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. David M. Ewalt, Editor in Chief, Scientific American Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250910000244.htm'>Scientists finally solve the mystery of ghostly halos on the ocean floor</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-10 09:07:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Now, new research from UC San Diego's Scripps Institution of Oceanography reveals that the barrels with halos contained caustic alkaline waste, which created the halos as it leaked out. Though the study's findings can't identify which specific chemicals were present in the barrels, DDT manufacturing did produce alkaline as well as acidic waste. Other major industries in the region such as oil refining also generated significant alkaline waste. "One of the main waste streams from DDT production was acid and they didn't put that into barrels," said Johanna Gutleben, a Scripps postdoctoral scholar and the study's first author. "It makes you wonder: What was worse than DDT acid waste to deserve being put into barrels?" The study also found that the caustic waste from these barrels transformed portions of the seafloor into extreme environments mirroring natural hydrothermal vents -- complete with specialized bacteria that thrive where most life cannot survive. Despite these unknowns, Paul Jensen, emeritus marine microbiologist at Scripps and senior author of the study, said that he would have expected the alkaline waste to quickly dissipate in seawater. Instead, it has persisted for more than half a century, suggesting this alkaline waste "can now join the ranks of DDT as a persistent pollutant with long-term environmental impacts." The findings also provide a way of visually identifying barrels that formerly contained this caustic alkaline waste. Nobody was thinking about alkaline waste before this and we may have to start looking for other things as well." Sediments in the area are heavily contaminated with the pesticide DDT, a chemical banned in 1972 now known to harm humans and wildlife. Scant records from this time period suggest DDT waste was largely pumped directly into the ocean. Gutleben said she and her co-authors didn't initially set out to solve the halo mystery. In 2021, aboard the Schmidt Ocean Institute's Research Vessel Falkor, she and other researchers collected sediment samples to better understand the contamination near Catalina. Using the remotely operated vehicle (ROV) SuBastian, the team collected sediment samples at precise distances from five barrels, three of which had white halos. After some unsuccessful troubleshooting in the lab, Gutleben tested one of these samples' pH. This explained the limited amount of microbial DNA she and her colleagues had been able to extract from the halo samples. Where this high pH meets the surrounding seawater, it forms calcium carbonate that deposits as a white dust, creating the halos. "It's shocking that 50-plus years later you're still seeing these effects. We can't quantify the environmental impact without knowing how many of these barrels with white halos are out there, but it's clearly having a localized impact on microbes." Prior research led by Lisa Levin, study co-author and emeritus biological oceanographer at Scripps, showed that small animal biodiversity around the barrels with halos was also reduced. Jensen said that roughly a third of the barrels that have been visually observed had halos, but it's unclear if this ratio holds true for the entire area and it remains unknown just how many barrels are sitting on the seafloor. The researchers suggest using white halos as indicators of alkaline waste could help rapidly assess the extent of alkaline waste contamination near Catalina. Next, Gutleben and Jensen said they are experimenting with DDT contaminated sediments collected from the dump site to search for microbes capable of breaking down DDT. Jensen said that trying to physically remove the contaminated sediments would, in addition to being a huge logistical challenge, likely do more harm than good. "The highest concentrations of DDT are buried around 4 or 5 centimeters below the surface -- so it's kind of contained," said Jensen. "If you tried to suction that up you would create a huge sediment plume and stir that contamination into the water column." In addition to Gutleben, Jensen and Levin, Sheila Podell, Douglas Sweeney and Carlos Neira of Scripps Oceanography co-authored the study, alongside Kira Mizell of the U.S. Geological Survey. Note: Content may be edited for style and length. New Smart Pimple Patch Clears Acne in Just 7 Days New Research Reveals Not All Ultra-Processed Foods Are Bad Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            