
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-14</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02181-x'>Why do ageing rates vary by country? Massive study says politics play a part</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 16:25:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Julian Nowogrodzki is a science writer and editor in Boston, Massachusetts. Air pollution is a risk factor for faster ageing.Credit: Per-Anders Pettersson/Getty Social inequality and weak democratic institutions are linked to faster ageing, as are other environmental features such as high levels of air pollution, finds a study spanning four continents1. Education was one of the top factors that protected against faster ageing. The study also showed that ageing is accelerated by less-surprising factors such as high blood pressure and heart disease. What molecular ‘clocks' can tell you about your health What molecular ‘clocks' can tell you about your health “It's a very important study”, says Claudia Kimie Suemoto, a geriatrician at the University of São Paulo in Brazil who was not involved in the work. Political polarization and uncertainty mean that “we are living in a world of despair”, and that ages people, says lead author Agustín Ibañez, who directs the Latin American Brain Health Institute in Santiago. Just the process of harmonizing these data sets — such as checking that variables were measured in similar ways in different countries — took about 3 years, says Ibañez. The top medical risk factors for faster ageing were high blood pressure, hearing impairment and heart disease. Other risk factors included unhealthy weight, alcohol consumption, sleep problems, diabetes and impaired vision. The factors that provide the best protection against speedy ageing were education, ability to perform activities of daily living and sound cognitive abilities. Other protective factors included physical activity, good memory and the ability to walk well. Ageing is linked to inflammation — but only in the industrialized world Ageing is linked to inflammation — but only in the industrialized world Research-integrity sleuths say their work is being ‘twisted' to undermine science Deaf scientists hit by drastic NIH cuts — the research community must support them The Faculty of Arts and Science at the University of Toronto invites applications for a full-time tenure stream position in Sustainable Food Systems. Ageing is linked to inflammation — but only in the industrialized world An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250714014133.htm'>Deadly disguise: How candy-like nicotine pouches caused a 763% spike in child poisonings</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 15:56:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new study reveals that ingestions of nicotine pouches by young children have surged in recent years. Researchers at the Center for Injury Research and Policy of the Abigail Wexner Research Institute at Nationwide Children's Hospital and the Central Ohio Poison Center analyzed calls to U.S. poison centers and found an alarming 763% increase in the rate of reported nicotine pouch ingestions among children younger than 6 years old from 2020 to 2023. Nicotine pouches were also more likely to be associated with serious medical outcomes or hospital admissions than other nicotine products like gum/lozenges, e-liquids, powder/granules, and tablets/capsules/caplets. "Nicotine pouches are a serious and growing toxic ingestion hazard among young children," said Hannah Hays, MD, co-author of the study and medical director of the Central Ohio Poison Center. The study, published in Pediatrics, also investigated other nicotine products and formulations. Researchers examined nearly 135,000 cases of nicotine ingestions among children younger than 6 years old that were reported to U.S. poison centers from 2010 through 2023. Most ingestions occurred at home and involved children under the age of 2 years. While most exposures resulted in minor or no effects, there were 39 cases with major medical outcomes and two deaths. This rate was primarily driven by the ingestion rate for liquid nicotine and nicotine solid formulations such as tablets, capsules, and caplets. "Many nicotine products are flavored and sold in colorful packaging that may be attractive to a young child," said Dr. Smith. Researchers also shared a few safety tips for parents and caregivers of young children. Note: Content may be edited for style and length. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09249-8'>Microbiota-driven antitumour immunity mediated by dendritic cell migration</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 15:29:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Gut microbiota influence the antitumour efficacy of immune checkpoint blockade1,2,3,4,5,6, but the mechanisms of action have not been fully elucidated. Here, we show that a new strain of the bacterial genus Hominenteromicrobium (designated YB328) isolated from the faeces of patients who responded to programmed cell death 1 (PD-1) blockade augmented antitumour responses in mice. YB328 activated tumour-specific CD8+ T cells through the stimulation of CD103+CD11b− conventional dendritic cells (cDCs), which, following exposure in the gut, migrated to the tumour microenvironment. Mice showed improved antitumour efficacy of PD-1 blockade when treated with faecal transplants from non-responder patients supplemented with YB238. This result suggests that YB328 could function in a dominant manner. YB328-activated CD103+CD11b− cDCs showed prolonged engagement with tumour-specific CD8+ T cells and promoted PD-1 expression in these cells. Moreover, YB238-augmented antitumour efficacy of PD-1 blockade treatment was observed in multiple mouse models of cancer. Patients with elevated YB328 abundance had increased infiltration of CD103+CD11b− cDCs in tumours and had a favourable response to PD-1 blockade therapy in various cancer types. We propose that gut microbiota enhance antitumour immunity by accelerating the maturation and migration of CD103+CD11b− cDCs to increase the number of CD8+ T cells that respond to diverse tumour antigens. Immune checkpoint blockade (ICB) therapies, including monoclonal antibodies against PD-1 and PD-1 ligand (PD-L1), are used in many cancer types7. PD-1 blockade therapies have led to significant improvements in progression-free survival (PFS) and overall survival. However, the complete response rate remains low8,9,10,11. Therefore, predictive biomarkers for clinical efficacy and new approaches to overcome resistance mechanisms are needed. Notably, high ratios of PD-1+CD8+ T cells to PD-1+ regulatory T (Treg) cells in the tumour microenvironment (TME) correspond to the responsiveness to PD-1 and PD-L1 inhibitors12. Moreover, PD-1 expression by CD8+ T cells is induced by T cell activation signalling, especiallyNFATC1, through the stimulation of fully matured dendritic cells (DCs)13,14. In gut microbiota, the abundance of certain bacteria (for example, Ruminococcus spp. is associated with the clinical efficacy of ICB therapies1,2,3,4,5,6. Several mechanisms of microbiome-mediated antitumour immunity have been proposed, including stimulation of macrophages and monocytes15, antigenic mimics between microbiota and tumour antigens16,17,18 or direct activation of tumour-infiltrating CD8+ T cells by microbiota-derived metabolites19,20,21,22. Moreover, a consortium of bacteria isolated from healthy individuals reportedly stimulated CD103+ DCs, which led to the activation of effector CD8+ T cells23. However, the detailed mechanisms that underlie the connection between DC maturation by gut microbiota and the activation of tumour antigen-specific PD-1+CD8+ T cells in the distant TME have not been fully elucidated. Here, we identified a new bacterial strain, designated YB328, that was significantly enriched in faeces of patients who responded to PD-1 blockade therapies. YB328 accelerated the differentiation of CD103+CD11b− cDCs, which is an essential DC subset for antigen cross-presentation to CD8+ T cells24,25. YB328-stimulated CD103+CD11b− cDCs were further activated by other bacteria and migrated to tumour-draining lymph nodes (dLNs) and the TME. Once in these locations, the cDCs primed and activated CD8+ T cells and induced PD-1+CD8+ T cells specific for a broad range of tumour antigens. Overall, we describe in detail the mechanism of the activation of antigen-specific CD8+ T cells in the distant TME by gut microbiota. We prospectively collected stool samples from 50 Japanese patients with cancer (15 patients with non-small cell lung cancer (NSCLC) and 35 patients with gastric cancer (GC)) who received PD-1 blockade therapy. These patients were classified as responders or non-responders (Supplementary Tables 1 and 2). Clinicopathological features that are reportedly associated with a favourable clinical response to PD-1 blockade (listed in Supplementary Tables 3–5) were not significantly different in our cohort, except for PD-L1 expression by tumour cells in patients with GC. 16S rRNA gene amplicon sequencing of faecal samples showed that the Shannon diversity of gut microbiota was significantly higher in responders than in non-responders (Fig. Receiver operating characteristic (ROC) analysis further showed that the Shannon diversity was a significant predictive biomarker for identifying responders (Fig. Moreover, patients with high diversity of gut microbiota exhibited significantly prolonged PFS, regardless of the cancer type (Fig. 1c, Supplementary Table 6 and Extended Data Fig. Principal coordinate analysis (PCoA) and analysis of similarities (ANOSIM) based on unweighted unique fraction metric (UniFrac) distances revealed a significant difference in gut microbiota between responders and non-responders (Fig. Linear discriminant analysis effect size (LEfSe) showed that members of several families, namely, Ruminococcaceae, Propionibacteriaceae, Dehalobacteriaceae and Turicibacteraceae, were enriched in responders (Fig. Among them, Ruminococcaceae abundance was significantly correlated with prolonged PFS after PD-1 blockade therapy and showed the highest AUC score (Fig. In contrast to previous reports1,2,3,4,18, the abundance of the genera Faecalibacterium, Enterococcus, Bifidobacterium and Akkermansia did not differ significantly between responders and non-responders (Extended Data Fig. Notably, members of the families Bacteroidaceae and Veillonellaceae, which were enriched in non-responders, were associated with a significantly shorter PFS (Fig. 1e,f,h and Extended Data Fig. We also confirmed that patients with high Shannon diversity values, a high abundance of Ruminococcaceae and a low abundance of Bacteroidaceae, when stratified by the median cut-off, had prolonged PFS (Extended Data Fig. Faecal samples were collected from patients (n = 15 NSCLC, n = 35 GC) who received PD-1 blockade therapy. c, Kaplan–Meier curves for PFS of patients with high or low Shannon diversity of gut microbiota. High and low Shannon diversity of gut microbiota in patients were stratified by ROC analysis (b). d, Differences between gut microbiota of responders and non-responders as visualized by PCoA based on unweighted UniFrac distances. e, Cladogram of differentially abundant taxa identified by LEfSe analysis. Symbol sizes are scaled by the relative abundance. f, Linear discriminant analysis (LDA) scores of the significantly different taxa between responders and non-responders. g,h, ROC analysis (left) and Kaplan–Meier curves (right) for PFS in patients with high or low Ruminococcaceae (g) or Bacteroidaceae (h) abundance. i–o, Immune profiles of tumour-infiltrating lymphocytes (TILs) in patients with both biopsy and faecal samples available (n = 4 NSCLC, n = 22 GC). Correlations between subsets of TILs and bacterial families enriched in responders and non-responders (i). The frequency of PD-1+CD8+ T cells (representative staining (j,m) and summary (k,n)) in patients with high or low abundances of Ruminococcaceae (k) or Bacteroidaceae (n) stratified by ROC analysis in g and h. Cells were gated on CD8 TILs (j,m). The correlation between PD-1 expression by tumour-infiltrating CD8+ T cells and the abundance of Ruminococcaceae (l) or Bacteroidaceae (o). Significance was assessed using two-sided Mann–Whitney U-test, with data presented as the median (a,k,n); two-sided log-rank test (c, g, right, h, right); ANOSIM with one-tailed significance computed using 999 permutations) (d); or two-sided Pearson's correlation (i,l,o). By contrast, the abundance of Bacteroidaceae was associated with a decrease in PD-1+CD8+ T cells and an increase in Treg cell infiltration in the TME (Fig. These findings indicate a clear correlation between specific gut microbiota compositions and immune responses. Germ-free (GF) mice colonized with faeces from responders, but not from non-responders, exhibited strong antitumour effects against MC38 tumours after anti-PD-1 monoclonal antibody (anti-PD-1) treatment (Extended Data Fig. 2a,b) and showed increased Shannon diversity in the faeces (Extended Data Fig. Moreover, Ruminococcaceae was enriched in mice who received faecal microbiota transplantation (FMT) derived from responders (R-FMT), whereas Bacteroidaceae was enriched in mice who received FMT derived from non-responders (NR-FMT) (Extended Data Fig. To exclude the influence of congenital immunodeficiency in GF mice, we performed the same experiments in specific-pathogen-free (SPF) mice preconditioned with antibiotics (ATB-SPF mice). Consistent with GF mice, anti-PD-1 treatment significantly inhibited the growth of MC38 and EMT6 tumours in ATB-SPF mice that received R-FMT (Extended Data Fig. Analyses of the TME demonstrated that activated CD8+ T cells (PD-1+CD8+ T cells and CD62L−CD44+ effector memory CD8+ T cells) and cytokine-producing CD8+ T cells (IFNγ+TNF+) were significantly higher in mice treated with R-FMT and anti-PD-1 than in mice treated with NR-FMT and anti-PD-1 or in control mice (Extended Data Fig. Moreover, tumour-infiltrating PD-1+CD8+ T cells from mice treated with R-FMT and anti-PD-1 exhibited a broader T cell receptor (TCR) Vβ repertoire diversity than mice treated with NR-FMT and anti-PD-1 (Extended Data Fig. As PD-1 expression by T cells correlates with the strength of T cell activation signalling13,29, we examined the maturation of DCs in the TME, a process that has a crucial role in T cell activation. The expression levels of DC maturation markers (CD86, CD40 and CD80) were significantly higher in R-FMT-treated mice than in NR-FMT-treated mice (Extended Data Fig. Therefore, gut microbiota augment PD-1 blockade efficacy by promoting mature DC infiltration and PD-1+CD8+ T cell accumulation in the TME. Given the strong antitumour efficacy of R-FMT, we aimed to isolate bacterial strains from the faeces of responders. A previously undescribed strain, designated YB328, was isolated, and its characteristics were investigated (Fig. Taxonomic assignment based on its full-length 16S rRNA gene sequence and the Greengenes database (v.13 8) indicated that strain YB328 belongs to the family Ruminococcaceae. Based on its whole-genome sequence and the Genome Taxonomy Database (GTDB; v.220), strain YB328 was assigned to the recently described species Hominenteromicrobium mulieris30 (Extended Data Fig. 4a), sharing an average 97.3% nucleotide identity to the GTDB species-representative genome. Notably, the family-level name depended on the taxonomic framework, namely, Ruminococcaceae (Greengenes), Oscillospiraceae (NCBI taxonomy) or Acutalibacteraceae (GTDB taxonomy). Moreover, YB328 was detected not only in the Japanese population but also in individuals worldwide (Extended Data Fig. a, Left, scanning electron microscopy image. Right, transmission electron microscopy image of negatively stained cells. The arrows indicate extracellular membrane vesicles secreted by YB328. b,d, Relative abundances of YB328 (b) and P. vulgatus (d) in faecal samples of patients shown in Fig. c,e, ROC curves (left) and Kaplan—Meier curves (right) of the PFS of patients with high or low abundances of YB328 (c) or P. vulgatus (e) in faeces. f,g, Tumour growth curves of SPF mice with B16F10 (f) or MC38 (g) tumours (left, summary; right, each mouse). Here and in other figures, control indicates mice treated with an isotype control antibody. Cells were gated on CD8+ TILs (h–j). n–p, Tumour growth curves (n, n = 5–7 mice per group), TCR Vβ repertoire (o) and numbers of skewed TCR Vβ clones (>10% of each clone) (p) among PD-1+CD8+ T cells (n = 5 mice per group). For b and d, each dot indicates one patient, and the data are presented as the median. For f (left), g (left) and n, the average tumour sizes of the groups on a certain day are shown as dots and are presented as the mean ± s.d. For h–m and p, each dot in the summary graphs indicates one mouse. Data are presented as the mean ± s.d. Significance was assessed using two-sided Mann—Whitney U-test (b,d,p), two-sided log-rank test (c, right, e, right), two-way analysis of variance (ANOVA) with the Tukey—Kramer method (f,g,n) or one-way ANOVA with Bonferroni correction (h–m). We further evaluated the abundance of YB328 in the gut microbiome through metagenome sequencing and species-level quantification using Kraken2 and Bracken against a custom version of the GTDB (v.220). This analysis showed that the abundance of YB328 was significantly higher in the faeces of responders than in non-responders (Fig. Patients with abundant YB328, defined by a ROC-based cut-off value, had a significantly longer PFS (Fig. 2c), even after excluding patients with high microsatellite instability (Extended Data Fig. By contrast, Phocaeicola vulgatus (represented by strain AE61, a newly isolated strain from this study belonging to the family Bacteriodeceae and enriched in non-responders), showed an increased abundance in non-responders. Moreover, patients with abundant P. vulgatus had a shorter PFS (Fig. As P. vulgatus is a well-known commensal bacterium31, we used the patient-isolated P. vulgatus strain AE61 to represent gut microbiota of non-responders and evaluated its effects on immunotherapy responses in animal models together with YB328. Although YB328 administration alone did not inhibit tumour growth (Extended Data Fig. 4e), YB328 augmented the antitumour effect of anti-PD-1 treatment more than P. vulgatus in ATB-SPF mice. This effect was observed in both poorly immunogenic B16F10 and immunogenic MC38 tumour models, with colonization beyond the gavage period (Fig. 2f,g and Extended Data Fig. Other bacterial strains enriched and isolated from the faeces of responders exhibited few or no antitumour effects compared with YB328 and PD-1 blockade therapy (Extended Data Fig. Mice treated with YB328 and anti-PD-1 had higher abundances of activated CD8+ T cells and cytokine-producing CD8+ T cells in the TME than mice treated with P. vulgatus and anti-PD-1 or control mice (Fig. However, the frequencies of Treg cells and PD-1+ Treg cells were comparable across groups (Extended Data Fig. Nevertheless, DCs in the TME of YB328-colonized mice exhibited significantly increased expression of co-stimulatory molecules (Fig. 2k,l and Extended Data Fig. 5f) and recapitulated the phenotypes of DCs in mice that received R-FMT. Moreover, major histocompatibility class I (MHCI) expression was significantly upregulated in these DCs (Fig. Taken together, these results suggest that YB328 colonization induces DC maturation with high efficiency antigen presentation to activate tumour-infiltrating CD8+ T cells. Furthermore, the administration of YB328, but not P. vulgatus, favourably changed the Shannon diversity of gut microbiota (Extended Data Fig. ATB-SPF mice treated with NR-FMT and colonized with YB328, but not P. vulgatus, showed increased sensitivity to anti-PD-1 therapy (Fig. Moreover, YB328 administration increased the diversity of the TCR Vβ repertoire of tumour-infiltrating PD-1+CD8+ T cells among skewed TCRs in these mice (Fig. The administration of YB328, but not P. vulgatus, favourably altered the Shannon diversity of gut microbiota (Extended Data Fig. Consistently, in our GC and NSCLC cohort of patients, increased levels of YB328 were positively correlated with the Shannon diversity of gut microbiota (Extended Data Fig. However, the co-administration of YB328 and P. vulgatus abolished the antitumour immunity induced by YB328 (Extended Data Fig. This effect was in contrast to YB328 single administration in mice treated with NR-FMT, which showed effective induction of antitumour immunity in combination with anti-PD-1 treatment. However, YB328 sufficiently colonized the gut of mice treated with NR-FMT at a similar level to that observed with YB328 administration alone. This finding indicated that YB328 colonization is not significantly influenced by the microbial environment shaped by NR-FMT (Extended Data Fig. Indeed, P. vulgatus administration led to significantly greater engraftment of P. vulgatus than NR-FMT (Extended Data Fig. These observations underscore the impact of competitive dominance on the efficacy of microbiome-based therapies. To understand the mechanisms of DC maturation, we analysed bone-marrow-derived DCs (BMDCs) treated with P. vulgatus or YB328. When ovalbumin (OVA)-specific (OT-I) TCR transgenic CD8+ T cells were cultured with bacteria-treated DCs pulsed with OVA peptides, YB328-treated BMDCs were more strongly attracted and were in contact with CD8+ T cells for a longer time than P. vulgatus-treated BMDCs (Fig. No difference was observed in CCL22 production, which recruits Treg cells, among YB328-treated, P. vulgatus-treated and unstimulated BMDCs (Extended Data Fig. This result is consistent with the comparable Treg cell infiltration levels observed in the TME across treatments. a–d, BMDCs were stimulated with YB328 or P. vulgatus. MFI summaries of the maturation markers CD86 (a, n = 6 wells per group) and H-2Kb (b, n = 5 wells per group), IL-12p70 production (c, n = 5 wells per group) and chemokine (CXCL9 and CXCL10) expression (d, n = 8 wells per group) in BMDCs are shown. e,f, Dynamic imaging of CD8+ T cells and BMDCs stimulated with YB328 or P. vulgatus. e, Contact events were evaluated at 10-min intervals (right), and a representative trajectory of the indicated cell (left) was extracted from Supplementary Video 1. f, Images showing the interaction between DCs and CD8+ T cells. The elapsed time is shown in minutes and seconds. g–m, OT-I CD8+ T cells were cultured with bacteria-treated BMDCs pulsed with the indicated concentrations of antigen peptides. g,i, Expression levels of TCR signalling molecules by CD8+ T cells induced by N4 peptide (g) or Q4H7 peptide (i). j–m, The frequency of expression of the effector molecules PD-1 (j,l) and CD62L and CD44 (k,m) in CD8+ T cells induced by treatment with N4 peptide (j,k) or Q4H7 peptide (l,m). (a–f,h,j–m) or the mean (g,i). Significance was assessed using one-way ANOVA with Bonferroni correction (a–d) or two-sided unpaired t-test (e–m). OT-I CD8+ T cells were cultured with DCs and then pulsed with different OVA peptides capable of delivering titrated levels of TCR signalling32. YB328-treated and P. vulgatus-treated BMDCs pulsed with higher doses (100 nM to 1 μM) of high-affinity OVA peptide (N4) presented comparable levels of phosphorylated ZAP70. YB328-treated BMDCs also stimulated strong phosphorylation of co-stimulatory signals (pJNK, pERK1 and pERK2 (pERK1/2), pAKT and pS6K) in CD8+ T cells (Fig. YB328-treated BMDCs pulsed with a low dose (10 nM) of the N4 peptide induced extensive NFATC1 nuclear translocation in CD8+ T cells, whereas P. vulgatus-treated BMDCs induced only limited translocation (Fig. YB328-treated BMDCs pulsed with N4 peptides, regardless of the dose, were correlated with the successful activation of and increased cytokine production by CD8+ T cells, whereas P. vulgatus-treated BMDCs induced PD-1 expression only when pulsed with high N4 doses (Fig. Moreover, when supplied with low-affinity OVA peptide (Q4H7), YB328-treated BMDCs induced ZAP70 phosphorylation and co-stimulatory signalling (Fig. 7e) at relatively high doses and promoted the nuclear translocation of NFATC1 in CD8+ T cells (Fig. YB328-treated BMDCs consistently induced increased PD-1 expression, which correlated with the activation of and cytokine production by CD8+ T cells, regardless of the dose (Fig. Analysis of the transcriptional profile revealed higher expression of Irf8 and Batf3 in YB328-treated DCs than in P. vulgatus-treated DCs and untreated DCs (Fig. BATF3 maintains IRF8 autoactivation to commit to the terminal differentiation of CD103+CD11b− cDCs, which have a crucial role in antitumour immunity by effectively activating CD8+ T cells33,34. Accordingly, the number of CD103+CD11b− cDC progenies increased in response to YB328 treatment (Fig. By contrast, IRF4, which is a key transcription factor of CD103−CD11b+ cDCs35, was highly expressed by P. vulgatus-treated DCs (Fig. To further understand the mechanisms of CD103+CD11b− cDC differentiation induced by YB328 treatment, common DC progenitors (CDPs) were cultured with YB328 or P. vulgatus and with FLT3 ligand (FLT3L), a factor essential for the differentiation of CDPs into CD103+CD11b− cDCs36,37. IRF8 expression was significantly upregulated in CDPs stimulated with YB328 at low concentrations of FLT3L, although IRF8 expression was comparable between the groups treated with high concentrations of FLT3L (Fig. This result suggests that YB328 activates an alternative signalling pathway in CDPs that compensates for FLT3L signalling. FLT3L treatment promotes both PI3K–mTOR and STAT3 signalling for the development of CD103+CD11b− cDCs38,39, and these two pathways regulate Batf3 expression40,41. YB328-induced Irf8 expression was reduced by an S6K inhibitor (rapamycin) or a STAT3 inhibitor (HJC0152) and was more significantly inhibited by the administration of both inhibitors (Fig. Consequently, the number of CD103+CD11b− cDCs induced by YB328 stimulation was decreased by rapamycin and/or HJC0152 treatment (Fig. The antitumour effects induced by combination treatment with YB328 and anti-PD-1 were completely abrogated in Batf3−/− mice, which lack a subset of CD103+CD11b− cDCs42 (Fig. By contrast, the depletion of macrophages with an anti-CSF1R monoclonal antibody did not affect the antitumour efficacy induced by YB328 and anti-PD-1 treatment (Extended Data Fig. Moreover, YB328-stimulated macrophages were less effective in activating CD8+ T cells than YB328-stimulated DCs (Extended Data Fig. These data provide further support for the importance of CD103+CD11b− cDCs in YB328-driven antitumour activity. a–e, DCs were treated with P. vulgatus, YB328 or vehicle. Comprehensive gene expression was examined (a), as was the expression of Batf3 (b, n = 6), IRF8 (c, n = 5), IRF4 (e, n = 3) and changes in the progeny of the CD103+CD11b− cDCs (d, n = 3). f, IRF8 expression by CDPs treated with bacteria and Flt3L as indicated, n = 3–4. g–j, BMDCs were treated with the indicated stimuli, and expression of pS6K (g), pSTAT3 (h) and IRF8 (i) and changes in the progeny of CD103+CD11b− cDCs (j) were examined (n = 4–6). k, Tumour growth curves of wild-type (WT) and Batf3−/− mice (n = 6–9) treated as indicated. l, Gene expression signatures of BMDCs stimulated with YB328 were extracted. Red text indicates TLR proteins that were highly expressed in the YB328-specific signature. m, Changes in the progeny of CD103+CD11b− cDCs (n = 3–4). p,q, The expression of MHCI (p; gated on CD11c+MHCII+ cells) and the changes in the progeny of CD103+CD11b− cDCs (q) (n = 6). r, Frequency of PD-1 expression in CD8+ T cells (n = 4–6). s, Tumour growth curves of WT and Tlr7−/–Tlr9−/− mice (n = 5–6) treated as indicated. t, Experimental scheme (top) and tumour growth curves (bottom). The number of mice showing a complete response (CR) in n = 4–6 mice is shown in parentheses. u, Changes in the progeny of CD103+CD11b− cDCs (n = 7–9). v, Experimental scheme (top) and tumour growth curves (bottom). Each dot in the summary graphs indicates a well (b–j,m,p–r,u). The average tumour size of the groups on a certain day is shown as a dot (k,n,s,t,v). Significance was assessed using one-way ANOVA with Bonferroni's correction (b–j,p–r,u), two-way ANOVA with the Tukey—Kramer method (k,n,s,v) or two-sided unpaired Student's t-test (m). We next investigated how YB328 stimulates multiple signalling pathways in DCs. The expression of TLR family members, particularly TLR5 and TLR7–TLR9, was markedly upregulated in YB328-treated BMDCs compared with P. vulgatus-treated or untreated BMDCs (Fig. Similarly, YB328 administration increased the diversity of TLR expression by DCs in the gut-associated lymphoid tissues (GALTs) of mice treated with NR-FMT (Extended Data Fig. Myd88−/− BMDCs treated with YB328 did not lead to increases in the number of CD103+CD11b− cDC progeny (Fig. Moreover, the increase in antitumour efficacy induced by YB328 and anti-PD-1 treatment was abrogated in Myd88−/− mice (Fig. The frequency of CD103+CD11b− cDCs was significantly lower in the lamina propria and Peyer's patch, but not in the spleen, of Myd88−/− mice than in wild-type mice (Extended Data Fig. This result suggests that YB328-triggered CD103+CD11b− cDC generation requires the activation of MYD88 signalling. 2a), the ability of YB328 to stimulate endosomal TLRs (TLR7–TLR9) to promote the differentiation of CD103+CD11b− cDCs was examined. YB328 was phagocytosed by BMDCs (Fig. The stimulation of THP-1-based TLR7, TLR8 and TLR9 reporter cells revealed that YB328 activated both the IRF and NF-ĸB signalling pathways (Extended Data Fig. As mouse TLR8 is nonfunctional44, the roles of TLR7 and TLR9 in YB328-mediated antitumour immunity were investigated. BMDCs derived from Tlr7−/− or Tlr9−/− mice partially reduced the upregulation of MHCI expression (Fig. 4p) and abrogated the differentiation of CD103+CD11b− cDCs induced by YB328 (Fig. Furthermore, BMDCs derived from Tlr7–/–Tlr9−/− mice did not respond to YB328. Therefore increased MHCI expression, differentiation into CD103+CD11b− cDCs and induction of the production of PD-1-expressing CD8+ T cells were not observed in these cells (Fig. YB328 administration to Tlr7–/–Tlr9−/− mice did not augment the antitumour efficacy of anti-PD-1 treatment (Fig. These data suggest that endosomal TLR7–TLR9 are key mediators of YB328-induced antitumour immunity. However, YB328 probably does not have genes encoding components of flagellum, based on whole-genome sequence annotation, and flagella were not detected by electron microscopy (Fig. Therefore, we examined whether TLR ligands such as flagellin derived from the gut microbiome could further augment the antitumour immunity induced by YB328-stimulated DCs. PD-1 expression by CD8+ T cells stimulated with YB328-treated DCs, but not those stimulated with P. vulgatus, was significantly increased after flagellin treatment (Extended Data Fig. Although treatment with YB328 and anti-PD-1 inhibited tumour growth, combined treatment with NR-FMT, which contained a pool of TLR ligands, led to enhanced antitumour efficacy (Fig. Therefore, YB328 sensitizes DCs to multiple TLR ligands derived from not only YB328 but also from other gut microbes to augment antitumour immunity. As YB328 upregulated multiple TLRs in DCs, we explored whether the combined activation of TLR signalling recapitulated the phenotype of YB328-treated DCs. BMDCs treated with the TLR agonist R848 or ODN-1826 exhibited increased S6K and STAT3 phosphorylation, and combination treatment further increased the phosphorylation of both these proteins. The triple combination of flagellin, R848 and ODN-1826 led to the most effective induction of S6K and STAT3 phosphorylation, which in turn resulted in the highest levels of IRF8 expression and the strongest induction of CD103+CD11b− cDCs out of all the treatment combinations (Fig. Moreover, combination treatment with an anti-PD-1 and a TLR agonist cocktail—at doses in which each agonist alone was insufficient to completely control tumour growth (Extended Data Fig. We next examined whether the immunomodulatory functions of YB328 are specific to this strain. Five representative bacterial strains were chosen for analysis: one strain isolated from the faeces of responders (Ruminococcus torques AE30) and four strains from species phylogenetically close to YB328 (the type strains of the species Ruminococcus albus, Clostridium leptum, Acutalibacter muris and Neglectibacter timonensis) (Extended Data Fig. Among these, YB328 treatment led to the most significant induction of PD-1 expression in CD8+ T cells (Extended Data Fig. 10a) and specifically augmented antitumour effects of anti-PD-1 compared with the other selected bacteria (Extended Data Fig. We then compared the whole-genome sequences of YB328, the five bacterial strains that lacked antitumour efficacy and P. vulgatus (Extended Data Fig. Different cellular fractions of YB328 were also prepared to determine which components of YB328 were responsible for its antitumour activity (Extended Data Fig. We also analysed whether the bacteria Akkermansia muciniphila1,15 and Bifidobacterium longum3,4, which reportedly improve responses to cancer immunotherapy, possessed properties similar to those of YB328. A. muciniphila, but not B. longum, exhibited a significant antitumour effect when combined with anti-PD-1 treatment (Extended Data Fig. A. muciniphila-treated BMDCs promoted CD103+CD11b− cDC differentiation, which in turn induced PD-1 expression by CD8+ T cells (Extended Data Fig. However, A. muciniphila induced the expression of IRF8 but not S6K or STAT3 phosphorylation in DCs (Extended Data Fig. Together, these results demonstrate the uniqueness of YB328. We next investigated the location of CD103+CD11b− cDC differentiation and CD8+ T cell activation. The abundance of CD103+CD11b− cDCs, but not CD103−CD11b+ DCs, in the lamina propria and Peyer's patch was significantly higher after YB328 treatment than after P. vulgatus treatment (Fig. The frequencies of migratory CD103+CD11b− cDCs were also significantly increased in the dLNs (inguinal LNs) and the TME of mice treated with YB328 and anti-PD-1 (Fig. Moreover, the frequency of CD103+CD11b− cDCs was comparable to that observed after P. vulgatus treatment in tumour nondraining LNs (NdLNs: inguinal LNs on the opposite flank of the tumour) and the spleen (Extended Data Fig. CD103+CD11b− cDCs in the mesenteric lymph nodes (MLNs) of mice treated with YB328 exhibited significantly increased expression of CCR7, a crucial chemokine receptor for migration45,46 (Fig. PD-1+CD8+ T cells co-expressing TCF-1 significantly proliferated in the tumour dLNs, but not in the MLNs, of YB328-treated mice, and this phenotype was not observed in P. vulgatus-treated mice (Fig. a–f, ATB-SPF mice were subcutaneously injected with MC38 cells and treated with the indicated bacteria. The phenotype and frequency (representative staining (left) and frequency (right)) of DC populations in lamina propria (gated on CD11c+MHCII+ cells) (a), Peyer's patch (gated on CD11c+MHCII+ cells) (b), dLNs (gated on live CD45+ cells) (c) and tumour (gated on CD11c+MHCII+ cells) (d). Expression (representative staining (left) and MFI summary (right)) of CCR7 by CD103+CD11b− cDCs in the MLNs (e, n = 3). Expression (representative staining (left) and MFI summary (right)) of Ki67 by PD-1+CD8+ T cells in dLNs and MLNs (f, n = 5; gated on PD-1+CD8+CD3+ cells). g, Representative image (left) and staining (right) of KikG-expressing or KikR-expressing intestinal cells (gated on CD45+ cells). h, A representative image of the localization of migrated intestinal cells after the indicated bacterial treatment. j, KikR+CD103+ DCs (representative staining (left) and number summary (right)) in tumours (per mg) (gated on CD45+ cells). k, The KikR−/KikR+ ratio of PD-1+CD8+ T cells (representative staining (left) and ratio summary (right)) in tumours. l, The TCR Vβ repertoire in CD8+ T cells in the indicated organs. For i–l, n = 3–6 mice per group. For a–l, each dot in the summary graphs indicates one mouse. m, Representative multiplex immunohistochemical staining of tumours from patients with GC. n,o, Correlation between the YB328 abundance in the faeces of patients and the density of PD-1+CD8+ T cells (n) or CLEC9A+IRF8+ cells (o). For n–p, each dot indicates one patient, and the relative correlation was assessed by two-sided Pearson's correlation. Significance was assessed using two-sided unpaired Student's t-test (a–c,e,f,j–k), one-way ANOVA with Bonferroni correction (d,i) or ANOSIM with one-tailed significance using 999 permutations (l). We next used mice expressing photoconvertible Kikume green-red (KikGR) to trace intestinal immune cells (Fig. In contrast to those in P. vulgatus-treated mice, intestinal immune cells (KikR+ cells) showed substantial infiltration in the tumour dLNs of YB328-treated mice (Fig. The intestinal infiltration of KikR+CD103+CD11b− cDCs into the dLNs and MLNs was higher in YB328-treated mice than in P. vulgatus-treated mice but was not higher in NdLNs, spleens or cervical lymph nodes (Fig. Moreover, treatment with YB328, but not P. vulgatus, increased the abundance of KikR+CD103+ DCs and the number of KikR−PD-1+CD8+ T cells in the TME (Fig. Although the frequency of KikR+PD-1+CD8+ T cells was slightly increased in the MLNs of YB328-treated mice, the TCR Vβ repertoire of tumour-infiltrating CD8+ T cells was significantly different from that of CD8+ T cells in MLNs (Fig. 11e), which indicated that there was local activation of tumour-antigen-specific CD8+ T cells. By contrast, the frequencies of KikR+PD-1+CD8+ T cells and KikR+CD11b+ cells were comparable in the dLNs, NdLNs, spleens and cervical lymph nodes after YB328 or P. vulgatus administration. However, KikR+FOXP3+ T cells were slightly increased in the dLNs of YB328-treated mice (Extended Data Fig. Moreover, YB328 itself was not detected in the tumour (Extended Data Fig. These data support the notion that YB328 promotes the migration of intestinal CD103+CD11b− cDCs into tumour dLNs and the TME and primes and activates CD8+ T cells in situ, which augments the antitumour efficacy of PD-1 blockade treatment. YB328 abundance was significantly associated with the infiltration of PD-1+CD8+ T cells and IRF8+CLEC9A+ DCs, which correspond to mouse CD103+CD11b− cDCs25, in the TME of patients treated with PD-1 blockade therapies (Fig. The abundance of PD-1+CD8+ T cells was significantly correlated with that of IRF8+CLEC9A+ DCs but not with that of CD206+ or CD14+ cells (Fig. We confirmed these findings in our validation cohort (GC, n = 14; NSCLC, n = 7; collected from January 2018 to September 2018; see Methods) (Supplementary Tables 1–5). In accordance with our discovery cohort (GC, n = 35; NSCLC, n = 15; collected from March 2017 to December 2017; see Methods), the presence of YB328 in gut microbiota showed a clear correlation with a favourable clinical outcome, and patients with abundant YB328 exhibited a significantly longer PFS (Extended Data Fig. In a prospective cohort of patients with head and neck squamous cell carcinoma (HNSCC) (Supplementary Table 11), the YB328 phylotype was more abundant in the faeces of the responder group than in the non-responder group (Extended Data Fig. We also analysed datasets from the Japanese nationwide MONSTAR project (MONSTAR cohort)47 (Supplementary Table 12). Patients with malignant melanoma (MM) in the MONSTAR cohort had a higher abundance of Akkermansia in the responder group than in the non-responder group, as previously observed1. Moreover, YB328 was more abundant in the responder group of not only patients with MM and in patients with renal cell carcinoma, both of which are known to be immunotherapy-favourable cancer types, but also in patients with GC (thereby validating our results) and in patients with oesophageal cancer (Extended Data Fig. Furthermore, in a public dataset48 from patients with MM treated with anti-PD-1, responders exhibited significantly higher YB328 colonization than non-responders 3 months after FMT and anti-PD-1 treatment (Extended Data Fig. Thus, a high abundance of YB328 in faeces correlates with responsiveness to anti-PD-1 treatment in multiple types of cancer. Similarly, YB328 treatment also significantly increased the antitumour efficacy of the adoptive cell transfer of tumour antigen-specific CD8+ T cells (Extended Data Fig. Overall, patients with a high abundance of YB328 in gut microbiota have a substantial number of CD103+CD11b− cDCs, which enable CD8+ T cells to respond to diverse tumour antigens and consequently improving the clinical efficacy of immunotherapies (Extended Data Fig. Restoring and cultivating anaerobic gut microbiota from faecal preservation is technically challenging. Currently, most studies use commercially available bacterial strains, which may limit the clinical relevance of these studies. Here, we successfully isolated a previously undescribed bacterial strain, YB328, from the faeces of patients who responded to PD-1 blockade therapy and revealed the mechanism by which YB328 improves responsiveness to immunotherapy. On the basis of publicly available 16S rRNA gene sequence data, YB328 was estimated to be in the gastrointestinal tract of approximately one-fourth of Japanese individuals. As YB328 abundance ranges from 0.01% to 1%, which indicates that YB328 constitutes a relatively small fraction of microbiota, intervention with YB328 provides a large window to increase its abundance and to improve antitumour immunity (Extended Data Fig. The occupation of immunodominant antigen-specific CD8+ T cells in the TME rapidly exhausts CD8+ T cells, which hampers the antitumour efficacy of ICB therapies in patients49. YB328 optimizes the range of tumour antigens that contribute to the antitumour efficacy of PD-1 blockade therapy. YB328-stimulated cDCs provided enhanced T cell stimulation signalling to reduce the threshold required for CD8+ T cell antigen recognition. Consequently, PD-1+CD8+ T cell clones with diverse TCRs against both immunodominant and subdominant tumour antigens were primed and activated, which resulted in a favourable PD-1 blockade treatment outcome (Extended Data Fig. Our findings, together with previous reports2,50,51,52, suggest that gut microbiota diversity can serve as a pan-ethnic predictive biomarker for response to PD-1 blockade therapy across different cancer types. One can then envision a plausible explanation for the association between high diversity of the gut microbiota and clinical responses to ICB53. That is, bacteria that are beneficial for antitumour immunity, such as YB328, first induce CD103+CD11b− cDC differentiation and activation and further sensitize these cDCs to TLR ligands. We propose a two-step activation system that involves YB328 for CD103+CD11b− cDC progenies in the gut to activate CD8+ T cells in the TME, namely, primary direct activation and secondary activation induced by other bacteria (Extended Data Fig. In summary, we demonstrated that a new bacterial strain, YB328, stimulates DCs in GALTs to augment antitumour immunity in the TME. We also presented the mechanism of spatiotemporal regulation of DCs by the microbiome from GALTs to the distant TME for the priming and activation of tumour antigen-specific CD8+ T cells. Patients with advanced NSCLC (n = 22) and patients with GC (n = 49) who received PD-1 blockade monotherapy (nivolumab or pembrolizumab) from March 2017 to September 2018 were enrolled in this study. Patients were separated into two independent cohorts in this study: the discovery cohort, in which patients with NSCLC or GC received PD-1 blockade monotherapy from March 2017 to December 2017 (n = 50; NSCLC, n = 15; GC, n = 35), and the validation cohort, in which patients with NSCLC or GC received PD-1 blockade monotherapy from January 2018 to September 2018 (n = 21; NSCLC, n = 7; GC, n = 14). Patients with HNSCC (n = 16) who received PD-1 blockade monotherapy (nivolumab or pembrolizumab) from June 2022 to October 2023 were also enrolled in this study. Patients with HNSCC who tested positive for human papillomavirus were excluded. In the MONSTAR cohort47, patients who received PD-1 blockade monotherapy (nivolumab or pembrolizumab) were enrolled from July 2019 to February 2022. Patients who were treated with antibiotics or microbiome intervention therapy within 1 month (the NSCLC and GC cohort and the HNSCC cohort) or 3 months (MONSTAR cohort) before the initiation of treatment were excluded from this study. Patients who achieved a complete or partial response or stable disease lasting >6 months were classified as responders and those who progressed on therapy or had stable disease <6 months were classified as non-responders per the Response Evaluation Criteria in Solid Tumors (RECIST; v.1.1). Forty-seven patients (discovery cohort: NSCLC, n = 4 and GC, n = 22; validation cohort: NSCLC, n = 7 and GC, n = 14) had an available quantity of TIL isolated; these samples were subjected to multiple immunological analyses. Stool samples were collected within 1 week before initial treatment with a DNA stabilizer (FS-0009, TechnoSuruga Laboratory) according to the manufacturer's instructions and then stored at −80 °C. Fresh tumour samples were obtained from primary or metastatic tumours by needle or endoscopic biopsy within 2 weeks before initial administration and subjected to immunological analyses. Clinical information of patients was obtained from their medical records. Anti-PD-L1 monoclonal antibody (22C3, Dako; SP142 or SP263, Roche) was used for immunohistochemistry (IHC) with an automatic staining instrument (BenchMark ULTRA, Roche) as previously described54. PD-L1 positivity was defined as staining in 1% or more of the tumour cells. Anti-mutL homologue 1 (MLH1; ES05) monoclonal antibody, anti-mutS homologue 2 monoclonal antibody (MSH2; FE11), anti-postmeiotic segregation increased 2 monoclonal antibody (PMS2; EP51) and anti-mutS homologue 6 monoclonal antibody (MSH6; EP49) (all from Dako) were used for IHC. Tumours were considered negative for MLH1, MSH2, PMS2 or MSH6 expression if there was a complete absence of nuclear staining in tumour cells. Normal epithelial cells and lymphocytes were used as internal controls. Tumours lacking MLH1, MSH2, PMS2 or MSH6 expression were considered deficient in mismatch repair, whereas tumours that expressed all these markers were considered proficient in mismatch repair. Chromogenic in situ hybridization for RNA encoding Epstein–Barr virus was performed with fluorescein-labelled oligonucleotide probes (INFORM EBER probe) with enzymatic digestion (ISH protease 3, Roche) and an iViewBlue detection kit (Roche) with the BenchMark ULTRA staining system. Tumour tissues were minced and treated within 72 h after surgery according to a TIL preparation protocol using an optimized tissue preservation reagent (Tumor & Tissue Preservation Reagent (TTPR)) and a TIL isolation reagent (Tumor & Tissue Dissociation Reagent (TTDR)), which were codeveloped by BD Biosciences55. Alternatively, tumour tissues were treated immediately with a gentleMACS Dissociator (Miltenyi Biotec) as previously described56. Cells were washed with PBS containing 2% FCS and stained with the indicated antibodies and fixable viability dye. After staining for cell surface markers, the cells were intracellularly stained with antibodies with a FOXP3 staining buffer set (Thermo Fisher Scientific) according to the manufacturer's instructions. The data were analysed with FlowJo software (v.10.8.1; BD Biosciences). To analyse the similarity of the CD8+ TCR Vβ repertoires, the expression of TCR Vβ was evaluated and presented as an ordination plot by uniform manifold approximation and projection. To analyse the diversity of TLR expression on DCs, the Shannon index of TLR expression was calculated based on the expression of TLR1–TLR9. Total DNA extraction was conducted using a QIAamp DNA Stool Mini kit (Qiagen) according to the manufacturer's instructions. To increase the recovery of bacterial DNA, particularly from Gram-positive bacteria, pretreatment with lytic enzymes was performed before extraction using a stool kit. In brief, 100 mg faecal sample was suspended in 10 ml Tris-EDTA buffer (pH 7.5), and then 50 μl of 100 mg ml–1 lysozyme type VI purified from chicken egg white (MP Biomedicals) and 50 μl of 1 mg ml–1 purified achromopeptidase (Fujifilm Wako Pure Chemical) were added. The solution was incubated at 37 °C for 1 h with mixing. Next, 0.12 g SDS (final concentration of 1%) was added, and the suspension was mixed until it became clear. Next, 100 μl of 20 mg ml–1 proteinase K (Fujifilm Wako Pure Chemical) was added, followed by incubation at 55 °C for 1 h with mixing. The cell lysate was then subjected to ethanol precipitation. The precipitate was dissolved in 1.6 ml ASL buffer from the stool kit and subsequently purified using a QIAamp DNA Stool Mini kit (Qiagen). For the HNSCC cohort, DNA was extracted using an ISOSPIN Fecal DNA kit (Nippon Gene) following protocol N as previously described57. Each library was prepared according to the ‘Illumina 16S Metagenomic Sequencing Library Preparation Guide' with a primer set (27Fmod: 5′-AGR GTT TGA TCM TGG CTC AG-3′ and 338R: 5′-TGC TGC CTC CCG TAG GAG T-3′) targeting the V1–V2 region of the 16S rRNA gene. Next, 251-bp paired-end sequencing of the amplicon was performed using MiSeq (Illumina) with a MiSeq v.2 500 cycle kit. The raw sequences were demultiplexed and quality-trimmed with a FASTX-Toolkit (http://hannonlab.cshl.edu/fastx_toolkit/index.html, v.0.0.14) and BBtrim (http://bbmap.sourceforge.net, v.33.21). A total of 20,000 reads per sample were randomly selected using seqtk (v.1.3-r106) for further analyses. The processed sequences were clustered into operational taxonomic units (OTUs) defined at a 97% similarity cut-off using UCLUST (v.1.2.22q). Representative sequences for each OTU were then classified taxonomically by using RDP Classifier (v.2.2) with the Greengenes (v.13 8) database. The bioinformatics pipeline QIIME (v.1.9.1) was used as the informatics environment for all relevant processing of raw sequencing data and the determination of relative bacterial abundances. For the HNSCC cohort, sequencing libraries of the V4 hypervariable region of the 16S rRNA gene were prepared as previously described58 and sequenced on an Illumina MiSeq instrument with V2 chemistry (2 × 251 bp reads). Alpha diversity was estimated using the Shannon index, which captures both microbiota richness and evenness, after subsampling to an even depth (10,000 reads per sample). Beta diversity was assessed by unweighted UniFrac distance matrices and visualized by PCoA. The PCoA of unweighted UniFrac distances was performed with QIIME, and the results were plotted using R (v.4.02). ANOSIM was used to evaluate the significant differences between groups. Kruskal–Wallis and pairwise Wilcoxon tests were performed, followed by LDA to assess the effect size of each differentially abundant taxon59. Bacteria with markedly increased abundance were defined as those with a log10[LDA score] > 3. Phylotype analysis, as delineated on the basis of the YB328 amplicon sequence variant (ASV) using 16S rRNA gene amplicon sequencing data, was performed for the following datasets: (1) prospective cohort of patients with HNSCC (V4 region, sequencing performed in this study); (2) MONSTAR cohort (V3V4 region, sequencing data obtained with permission from the MONSTAR alliance); and (3) two publicly available sequencing datasets48,60 (both V3V4 region). 60, data captured two healthy Japanese cohorts, namely the MORINAGA cohort (n = 704 for study accession DRP005906) and the NIBIOHN cohort (n = 1,280 for study accessions DRP007219, DRP007221, DRP007218, DRP007222 and DRP007220); the data were downloaded from the DDBJ Sequence Read Archive (DRA) in fastq format. 48, sequence data were obtained from the NCBI Sequence Read Archive (SRA) under BioProject PRJNA928744. In all cases, sequencing reads were denoised to obtain ASVs using DADA2 (v.1.26.0)61. For this purpose, primers in the reads were trimmed using Cutadapt (v.4.2)62, and read pairs without identifiable primers and undetermined bases (Ns) were discarded. Reads were then further trimmed by length (truncLen option) and filtered on the basis of their expected error (maxEE option) using the filterAndTrim function of DADA2. For the publicly available sequencing data, forward and reverse reads were denoised on a per-sample basis using the dada function in DADA2 with the options err = NULL and selfConsist = TRUE. For the data generated in this study (that is, the prospective cohort of patients with HNSCC) and the MONSTAR cohort data, error models for the forward and reverse reads were generated using the learnErrors function in DADA2 on a per-sequencing run basis. After denoising using the dada function with the run-specific error models, ASV tables were merged using mergeSequenceTables function in DADA2 and chimeras were removed using the removeBimeraDenovo function with method = “consensus”. For all datasets, a single ASV that perfectly matched the 16S rRNA gene sequence of YB328 was retained and used to calculate the relative abundance of YB328 (phylotype). To verify the specificity of detection of YB328, the V4 and V3V4 ASVs of YB328, as extracted from its whole-genome sequence, were compared against the Greengenes2 database (release 2024.09; Supplementary Data 1). The shotgun sequencing library for each microbial sample was constructed using Illumina DNA Prep (Illumina). Sequencing was performed on a DNBSEQ-G400RS (MGI) in 150-bp paired-end mode. All procedures were performed according to the manufacturer's instructions. The genomic DNA of YB328 and P. vulgatus AE61 was extracted using an enzymatic lysis method with Qiagen Genomic-tip 100/G columns according to the manufacturer's protocol. The complete genome sequence of strain YB328 and a high-quality draft genome of P. vulgatus AE61 were determined using PacBio HiFi (high-fidelity) reads (Supplementary Table 7). PacBio sequencing, including library preparation and library quality control, were performed by Bioengineering Lab. In brief, short DNA fragments were removed using a Short Read Eliminator kit (PacBio) and retained DNA was sheared using a Covaris instrument to a size of 10–20 kbp. Sequencing libraries were then prepared using a PacBio SMRTbell Prep kit 3.0 and a SMRTbell gDNA Sample Amplification kit and quantified using a QuantiFluor dsDNA system (Promega) and an Agilent HS Genomic DNA 50 kb kit (Agilent Technologies) for quantification and size distribution, respectively. The library was then treated with a Revio Polymerase kit (PacBio) and sequenced on a Revio instrument. HiFi reads (Q20) were generated using SMRT Link (v.13.0.0.207600) to remove overhang adapter sequences and to create consensus sequences based on the resultant subreads. Lima (v.2.9.0) and pbmarkdup (v.1.0.3) were used for removing ultra-low PCR adapter sequences and PCR duplex reads, respectively. The resulting reads were assembled using Flye (v.2.9.5) and specifying the parameters --genome-size 3 m --asm-coverage 40. A phylogenetic tree showing the phylogenetic placement of YB328 and related taxa was constructed based on 120 single-copy marker genes using GTDB-Tk (v.2.4.0) with GTDB release 220 reference data. Protein sequences of identified marker genes (using the identify function of GTDBTk) were aligned using the align function of GTDBTk with the option skip_trimming. Aligned protein sequences were then loaded into an ARB (v.7.0) database, representative genomes were selected and the data were exported with custom masks for uninformative sites. Approximate maximum-likelihood phylogenetic trees were inferred using FastTree (v.2.1.1053), and the trees were visualized in ARB. The average nucleotide identity (ANI) values were then calculated using fastANI63 through comparisons of YB328 and closely related species. The predicted protein sequences were further analysed by matching against the Clusters of Orthologous Groups (COGs) database (release 3.10 in the Conserved Domain Database)65 using RPS-Blast (BLAST v.2.9.0+; E value cut-off of 0.01), the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (release 2019-11-18)66 and the Virulence Factor Database (VFDB; release 2022.04)67 and by protein alignment using DIAMOND (v.2.0.13)68. Coding sequences unique to strain YB328 (see Extended Data Fig. 4a for genome accessions) were identified using CompareM (v.0.1.2; https://github.com/dparks1134/CompareM) with default settings. Taxonomic profiling using metagenome sequencing data was performed through read-level taxonomic assignment with Kraken2 (v.2.1.3)69, followed by estimation of relative abundances using Bracken (v.2.9)70. For this purpose, paired-end reads were processed using fastp (v.0.23.2 or v.0.24.0)71 with the following settings: --trim_tail1 1 --trim_tail2 1 --cut_right --cut_right_window_size 4 --cut_right_mean_quality 15 --qualified_quality_phred 15 --unqualified_percent_limit 40 --trim_poly_x --poly_x_min_len 10 --n_base_limit 0 --length_required 75. Reads were then filtered using NCBI's BMTagger (v.3.101) to remove reads derived from the human genomic DNA. Taxonomic assignment of the reads was performed using Kraken2 against a customized version of the GTDB (v.220), with options --confidence 0.30 --use-names --paired. Species abundances were then estimated based on the Kraken2 output using Bracken, with options -r 150 -t 10. To construct the custom GTDB-based database, we built Kraken2 and Bracken databases using all species-representative genomes in the GTDB (v.220; n = 113,104 total bacterial and archaeal genomes), after replacing the corresponding genomes of the species P. vulgatus (representative: GCF_000012825.1) and H. mulieris (representative: GCF_020687165.1) by the two genome sequences generated in this study. Furthermore, to reduce the potential of false-positive assignments, a representative human genome sequence (GRCh38) was also included in the database. Construction of the Kraken2 and Bracken databases largely followed the scheme implemented in Struo2 (ref. 72), with a default k-mer length of 35 and specifying a read length of 150 for the bracken-build command. To investigate the abundance of YB328 on a global scale, we analysed 1,188 human faecal metagenomic datasets from NCBI's SRA. In this study, we focused on samples from healthy individuals; these were identified on the basis of the metadata available in GMrepo2 (ref. Data were retrieved in FASTQ format using the SRA Toolkit's prefetch and fasterq-dump commands, and then uniformly processed and analysed using fastp and Kraken2 as described above for the in-house generated sequencing data. All bacterial strains were isolated from fresh human faecal samples. The collected faecal samples were transported under 100% CO2 using a deoxygenating agent (Mitsubishi Gas Chemical) to a vinyl anaerobic chamber (COY) with an atmosphere composed of N2, H2 and CO2 (8:1:1). The faecal samples were diluted with pre-reduced dilution buffer to 1.0 × 10−8 g ml–1 in a tenfold dilution series. Fifty microlitres of 1.0 × 10−6 g ml–1, 1.0 × 10−7 g ml–1 and 1.0 × 10−8 g ml–1 diluted faecal fluid was inoculated on Eggerth–Gagnon (EG) agar plates75 supplemented with 5% (v/v) horse blood and M98-5 agar plates76. After incubation for 48 h at 37 °C, every colony grown on the plates was picked and subcultured on new plates for purification and growth. The obtained strains were preserved in pre-reduced nutrient broth (BD Biosciences) supplemented with 10% glycerol at −80 °C, and a portion of the colony was used for bacterial taxon identification. For bacterial taxon identification, the 16S rRNA gene sequence of the strains was amplified using the universal primers 27F (5′-AGAGTTTGATCCTGGCTCAG-3′) and 1492R (5′-GGTTACCTTGTTACGACTT-3′), and almost-full-length 16S rRNA genes were sequenced with the universal primers using a previously described procedure77. Bacteroides vulgatus AE61 (renamed P. vulgatus), Sellimonas intestinalis AE3, Clostridium colicanis AE66, Eggerthella lenta AE7, Ruminococcus torques AE30 and YB328 were isolated from EG agar plates. Erysipelatoclostridium ramosum AM28 was isolated using M98-5 agar plates. Ruminococcus albus JCM 14654T was obtained from the Japan Collection of Microorganisms (JCM) (RIKEN). Acutalibacter muris DSM 26090T, Neglectibacter timonensis DSM 102082T and Clostridium leptum DSM 753T were obtained from the German Collection of Microorganisms and Cell Cultures. A. muciniphila BAA-835T and B. longum BAA-999 were obtained from the American Type Culture Collection (ATCC). For inoculation of these bacterial strains into mice, the cells were collected using degassed GAM broth (Nissui) in an anaerobic chamber and were then suspended at a concentration of approximately 1.0 × 109 cells per ml in pre-reduced PBS containing 0.05% (w/v) l-cysteine·HCl·H2O or 15% (w/v) glycerol solution for storage. Until inoculation, the suspended bacterial strains were kept at −80 °C. The mouse colon cancer cell line MC38 (ENH204, Kerafast) and the mammary carcinoma cell line EMT6 (CRL-2755, ATCC) were cultured in high-glucose DMEM (Fujifilm Wako Pure Chemical) supplemented with 10% FBS (Biosera) and 2 mM l-glutamine. Mouse B16F10 (CRL-6475, ATCC) and B16F10-OVA (stable transfectant derived from B16F10 cells, which were generated in house) melanoma cell lines were cultured in RPMI-1640 (Fujifilm Wako Pure Chemical) supplemented with 10% FBS (Biosera) and 2 mM l-glutamine. All cell lines are confirmed to be free of mycoplasma contamination. For the induction of BMDCs from bone marrow (BM), 4 × 106 BM cells were cultured in tissue-culture-treated 6-well plates in 4 ml complete medium (RPMI-1640 (Fujifilm Wako Pure Chemical) supplemented with 2 mM l-glutamine and 10% FBS) containing 20 ng ml–1 GM-CSF (Peprotech) as previously described78. Half of the medium was removed on day 2, and fresh complete medium containing 40 ng ml–1 GM-CSF was added. The culture medium was entirely discarded on day 3 and replaced with complete medium containing 20 ng ml–1 GM-CSF. of bacteria used to infect BMDCs was 100 or indicated in each figure legend. CDPs were identified as LIN−CD117intCD135+CD115+CD11c−MHCII− BM cells, which lacked the markers of other cell lineages, including CD19, B220, CD3, NK1.1 and TER-119 (refs. Bone marrow-derived macrophages were induced from BM cells as previously described82 and cultured in DMEM containing 10 ng ml–1 M-CSF (Peprotech). THP-1-derived reporter cell lines expressing human TLRs (hTLR7, hTLR8 and hTLR9) were cultured according to the manufacturer's instructions (InvivoGen). Cells were stimulated with YB328 for 24 h, and supernatants were collected for downstream analyses. For YB328 fractionation, YB328 was cultured with GAM broth (Nissui) at 37 °C for 16 h. Using the collected cells, the cell membrane fraction and cytoplasm fraction were prepared using the bead-beating method as previously described83,84. Using the culture supernatant, the membrane vesicle fraction and supernatant fraction (without membrane vesicles) were prepared by ultracentrifugation previously as described85,86. The MC38, EMT6 and B16F10 cell lines were pre-authenticated by ATCC using short tandem repeat sequencing. OVA expression in B16F10-OVA cells was confirmed by flow cytometry. THP1-Dual™ hTLR7, THP1-Dual™ hTLR8 and THP1-Dual™ hTLR9 cells were not authenticated but were functionally validated for responsiveness to their respective TLR agonists, serving as positive controls. OT-I TCR transgenic mice87 (provided by W. R. Heath, University of Melbourne), Myd88−/− mice88, Tlr7−/−, Tlr9−/− and Tlr7–/–Tlr9−/− mice (Oriental Bio Service, provided by S. Akira, WPI Immunology Frontier Research Center, Osaka University), Batf3−/− mice89 (The Jackson Laboratory, provided by K. Murphy, Washington University School of Medicine) and Kikume Green—Red (KikGR) mice90 (RIKEN BioResource Research Center, provided by M. Tomura, Kyoto University Graduate School of Medicine) were used. GF mice were bred and maintained in the gnotobiotic facilities of the Central Institute for Experimental Animals or purchased from CLEA Japa or Sankyo Labo and maintained within the gnotobiotic facilities of the National Institute of Advanced Industrial Science and Technology. Other mice were housed in cages under SPF conditions, provided with standard food, given free access to hypochlorous weak-acid water and housed with a 12–12-h light–dark cycle with lights on at 8:00. In tumour challenge experiments, syngeneic mice were subcutaneously inoculated with 1 × 106 MC38 colon cancer cells, 1 × 106 EMT6 mammary carcinoma cells, 5 × 105 B16F10 melanoma cells or 5 × 105 B16F10-OVA melanoma cells in a total of 100 μl of PBS. The mice were randomized into different groups, ensuring that the tumour size was similar across all groups before the initiation of treatment. Anti-PD-1 monoclonal antibody (200 μg per mouse; clone: RMP1-14; BioLegend), anti-CSF1R monoclonal antibody (100 μg per mouse; clone: AFS98; Bio X Cell) or isotype control antibody (clone RTK2758; BioLegend) was intraperitoneally administered 3 times at 3-day intervals. The immunological profile of the TME was analysed on days 13 or 15 after tumour inoculation. Mice were monitored twice a week and euthanized when the subcutaneous tumour diameter exceeded 20 mm. In some FMT and bacterial administration experiments, mice were preconditioned with antibiotics (ampicillin 500 mg l–1 (Merck), neomycin 500 mg l–1 (Merck), metronidazole 1 g l–1 (Merck) and vancomycin 500 mg l–1 (Merck)) dissolved in sterile drinking water. Mice received bacterial or faecal transplantation by oral gavage using feeding needles. For FMT, we were able to collect faecal samples sufficient for in vivo animal models after examining gut microbiota from a total of 6 patients—3 responders and 3 non-responders—and used the faecal samples from these patients for each independent experiment. Faecal samples (0.1 g) were dissolved in 10 ml PBS and administered to mice by oral gavage (100 μl per mouse). For the bacterial administration experiments, we prepared the bacterial stock at 108 cells per ml and administered it to mice by oral gavage (100 μl per mouse). Intestinal-cell-labelling experiments were performed as previously described90. In brief, the small intestine was drawn from the abdominal cavity and photoconverted by exposure to violet light (435 nm, 40–50 mW cm–2) for 90 s by laparotomy. Nontargeted regions were protected from light using aluminium foil. For adoptive transfer experiments, splenocytes were collected from OT-I TCR transgenic mice, and CD8+ T cells were purified with CD8a microbeads (Miltenyi Biotec). SPF mice pretreated with antibiotics were subcutaneously injected with B16F10-OVA cells, and the sorted CD8+ T cells (1 × 106 cells per mouse) were transferred by intravenous injection. Sample sizes for animal experiments were chosen according to preliminary pilot studies12,91,92, and are in line with standards in the field. To isolate TILs and lymphocytes from the lymph nodes, tissue samples were collected and minced into small pieces, followed by digestion with BD Horizon Dri TTDR reagent at room temperature for 20 min. After dissociation, the cell suspension was filtered through a 70-μm strainer and washed with PBS supplemented with 1% BSA. For intestinal tissues, small intestines (duodenum, jejunum and ilium) were excised. The intestinal contents were removed by washing with HBSS. Fat tissue was also removed. Peyer's patches and mesenteric tissues were carefully removed and digested in BD Horizon Dri TTDR according to the manufacturer's instructions. The intestines were further opened longitudinally and cut into 5-mm long segments. Tissues were digested using a Lamina Propria Dissociation kit (Miltenyi Biotec) according to the manufacturer's instructions. BMDCs were stimulated with the indicated bacteria for 12 h, and chemokines and cytokines were measured using ELISA kits according to the manufacturer's instructions (CXCL9, Abcam; CXCL10, Thermo Fisher Scientific; Proteome Profiler Mouse Chemokine Array kit, R&D Systems; and IL12p70, Thermo Fisher Scientific). For profiling chemokine expression with a Proteome Profiler Mouse Chemokine Array kit, the membrane was visualized using an LAS-4000 instrument (GE Healthcare), and the raw imaging data were processed and analysed with ImageJ software (Fiji, v.2.0.0). The cell structure of YB328 was observed by scanning electron microscopy (S-4500; Hitachi) and transmission electron microscopy (H-7600; Hitachi) as previously described93,94. In brief, for scanning electron microscopy observation, YB328 cells were fixed with 2% glutaraldehyde, post-fixed with 1% osmium tetroxide and dehydrated through a graded ethanol series followed by 3-methylbutyl acetate. For transmission electron microscopy observation, the cells were fixed with 2.5% glutaraldehyde and then post-fixed with 1% osmium tetroxide. The fixed cells were suspended in 1% aqueous uranyl acetate. The biochemical properties of YB328 were evaluated using an API ZYM system (bioMérieux) according to the manufacturer's instructions. CD8+ T cells from OT-I TCR transgenic mice were prestained with 1 mg ml–1 Hoechst 33342 (Dōjindo Laboratories) at 37 °C for 10 min, washed and then cultured with the indicated bacteria-stimulated DCs pulsed with OVA peptides. After 4 h of culture, the cells were collected. Bacteria were prestained with SYTO Green Fluorescent Nucleic Acid stains (Thermo Fisher Scientific) and cultured with BMDCs prestained with CellTracker Deep Red dye (Thermo Fisher Scientific). All staining procedures were performed according to the manufacturer's instructions. After 15 min of culture, the cells were collected. Cells were fixed with 4% formaldehyde solution for 30 min, permeabilized with 0.5% Triton X-100 for 10 min and blocked with PBS supplemented with 10% BSA for 1 h. The fixed cells were stained with an anti-NFATC1 (Thermo Fisher Scientific) monoclonal antibody overnight and then with a DyLight 488 secondary antibody (Thermo Fisher Scientific). Coverslips were mounted with ProLong Diamond Antifade mountant (Thermo Fisher Scientific). The DCs and T cells were prestained with 0.5 μM CellTracker Green CMFDA dye, 0.5 μM CellTracker Red CMTPX dye, 0.5 μM CellTracker Deep Red dye (Thermo Fisher Scientific) or 1 mg ml–1 Hoechst 33342 at 37 °C for 10 min. Bacteria were prestained with SYTO Green Fluorescent Nucleic Acid stains (Thermo Fisher Scientific) according to the manufacturer's instructions. Prestained DCs (2 × 105) were mixed with prestained CD8+ T cells from OT-I TCR transgenic mice (1 × 105) or bacteria (m.o.i. = 100) and seeded into a culture dish. The videos were recorded using an A1R MP+ multiphoton microscope system (Nikon) or a fluorescence microscope (Carl Zeiss LSM-880, Zeiss). The raw imaging data were processed and analysed with ImageJ software (Fiji, v.2.0.0). Mice were depilated by hair removal cream 1 week before observation using an IVIS spectrum imaging system (IVIS kinetics, PerkinElmer), and the iVid-neo diet (Oriental Bio) was applied 2 weeks before observation to reduce autofluorescence from the diet. Analysis was conducted using Living Image software (v.4.7.3, PerkinElmer). Total RNA was extracted from BMDCs using a RNeasy Micro kit (Qiagen) and further processed with a SMART-Seq v4 Ultra Low Input RNA kit (Takara Bio). cDNA library preparation was conducted with a Nextera XT DNA Library Preparation kit (Illumina). The prepared RNA sequencing libraries were subjected to next-generation sequencing with a NovaSeq 6000 (Illumina) using 150-bp read lengths in paired-end mode. The reads generated for each RNA sample were analysed and compared using the Illumina DRAGEN Bio-IT Platform. Sequencing reads were aligned and annotated to the UCSC mouse reference genome (GRCm38.p6) from Gencode. The number of transcripts per kilobase million was calculated and used for downstream analyses. Expression levels across samples were normalized by z score transformation. To determine the gene expression of BMDCs, total RNA was extracted from BMDCs using a RNeasy Micro kit (Qiagen). cDNA synthesis was performed with SuperScript IV VILO master mix (Thermo Fisher Scientific). The TaqMan gene assay probe was purchased from Thermo Fisher Scientific (assay ID: Batf3/Mm01318273_m1; 18S rRNA/Hs03003631_g1). TaqMan Fast Advanced master mix (Thermo Fisher Scientific) was used to perform PCR, and the results were examined with a QuantStudio 7 Flex Real-Time PCR system (Thermo Fisher Scientific). The PCR conditions were as follows: 50 °C for 2 min, 95 °C for 20 s, and 40 cycles of 95 °C for 3 s and 60 °C for 30 s. Gene expression was normalized to that of the endogenous control gene 18S rRNA. Relative gene expression between groups was determined using the comparative threshold cycle method. YB328 and P. vulgatus were quantified in faeces and tumour tissues by real-time PCR with a primer set targeting the recA gene of YB328 (forward primer: 5′-CCTCTTGGACCTTGCCGAAA-3′; reverse primer: 5′-ATACGCGTGCCGTTATACGA-3′) and the 16S rRNA gene of P. vulgatus (5′-forward primer: CGGGCTTAAATTGCAGATGA; reverse primer: 5′-CATGCAGCACCTTCACAGAT-3′) as previously described95 using a QuantStudio 5 Real-Time PCR instrument. The reactions (20 µl) contained 1 × Power SYBR Green PCR master mix and 500 nM each of the forward and reverse primers, and the thermal cycling conditions were as follows: 95 °C for 1 min, 40 cycles of 95 °C for 30 s, 55 °C for 30 s, and 72 °C for 15 s. A standard curve was prepared using purified genomic DNA of YB328 and P. vulgatus. Formalin-fixed and paraffin-embedded blocks of tumour specimens were sliced into 4-μm-thick sections on adhesion microscope slides (Matsunami). The tissue slides were deparaffinized and rehydrated for multiplex IHC staining. Antigen retrieval and subsequent staining were performed using an Opal 7-Colour IHC kit (Akoya Biosciences) according to the manufacturer's instructions. Statistical analyses were performed with Prism software (v.8; GraphPad), SPSS (v.21.0; IBM) and R (v.4.02; R Foundation for Statistical Computing). Patient characteristics were compared between the two groups using Pearson's chi-square test and Fisher's exact test. ROC curves for continuous variables were created by plotting the true-positive rate against the false-positive rate at each threshold. The cut-off values of continuous variables were determined as the maximum sum of the sensitivity and specificity in ROC analyses. Survival curves were estimated using the Kaplan–Meier method and compared using the log-rank test. No statistical methods were used to predetermine sample size. Data were analysed for a normal distribution using the Shapiro-Wilk test before comparisons. The relationships between two groups were compared using a two-sided Student's t-test for normally distributed data or the nonparametric Mann–Whitney U-test. For multiple testing, significance was determined by one-way ANOVA followed by Bonferroni's correction. Tumour volume curves were compared using two-way ANOVA with the Tukey–Kramer method. Univariate or multivariate analyses were performed with the Cox regression model. All graphs of animal and in vitro experiments show representative data from at least two independent experiments. P < 0.05 was considered to indicate significance. This study was conducted in Japan with the involvement of local researchers throughout the research process, from conceptualization to data analyses and manuscript preparation. All patients provided written informed consent before sampling, according to the Declaration of Helsinki. Analyses of cohort samples were conducted in a blinded manner and approved by the National Cancer Center Ethics Committee (IRB protocol numbers: GC and NSCLC cohort: 2015-048 and 2017-007; HNSCC cohort: 2022-346; MONSTAR cohort: 2018-367). The GC and NSCLC cohort and the MONSTAR cohort are registered in the UMIN Clinical Trials Registry (https://www.umin.ac.jp/ctr/, IDs: UMIN000019129 and UMIN000036749, respectively). The study was not designed to evaluate outcomes based on participant characteristics such as sex or ethnicity. Animal care and experiments were conducted according to the guidelines of the animal committee of the National Cancer Center/Institutional Animal Experiment Committee of National Institute of Advanced Industrial Science and Technology after approval by the Ethics Review Committee for Animal Experimentation of the National Cancer Center (protocol numbers: K24-007 and K24-010) and the Institutional Animal Experiment Committee of National Institute of Advanced Industrial Science and Technology (protocol number: 2022-0413). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The whole genome sequences of YB328 and P. vulgatus AE61 have been deposited in the NCBI BioProject under accession PRJDB17635. The metagenome and 16S rRNA gene amplicon sequencing data generated in this study are available under BioProject accession PRJDB17628. The datasets used in this study, including the HNSCC cohort and the MONSTAR cohort, are not publicly available because of their association with unpublished studies. Access to these datasets is therefore restricted in accordance with institutional policies and data protection requirements. Researchers interested in accessing these data may submit a formal request to the corresponding author H.N. The datasets for the Healthy Japanese cohort (the data have been deposited in DNA Databank of Japan (DDBJ) under accession numbers DRP005906, DRP007219, DRP007221, DRP007218, DRP007222 and DRP007220), the MM cohort (BioProject PRJNA928744)48 and the faecal metagenomics analysis of global abundance of YB328 in healthy donors (BioProjects PRJEB6456, PRJEB21528, PRJNA422434, PRJNA397112, PRJDB4176, PRJNA517801, PRJNA529400, PRJEB27005, PRJNA529124, PRJEB8094, PRJEB17896, PRJNA328899, PRJEB7369, PRJEB6092, PRJNA268964, PRJNA447983, PRJEB1220, PRJEB1690, PRJNA289586, PRJNA421881, PRJEB17784, PRJEB5224, PRJNA504891, PRJEB9576, PRJEB6070, PRJNA438847, PRJEB27928, PRJEB12357 and PRJEB1786) were downloaded through the SRA. The RNA sequences of bacteria-stimulated BMDCs are available from the Gene Expression Omnibus under accession number GSE285376. Source data are provided with this paper. No custom code was developed in this study. Gut microbiome influences efficacy of PD-1-based immunotherapy against epithelial tumors. Gut microbiome modulates response to anti-PD-1 immunotherapy in melanoma patients. The commensal microbiome is associated with anti-PD-1 efficacy in metastatic melanoma patients. Sivan, A. et al. Commensal Bifidobacterium promotes antitumor immunity and facilitates anti-PD-L1 efficacy. Vetizou, M. et al. Anticancer immunotherapy by CTLA-4 blockade relies on the gut microbiota. Lee, K. A. et al. Cross-cohort gut microbiome associations with immune checkpoint inhibitor response in advanced melanoma. & Wolchok, J. D. Cancer immunotherapy using checkpoint blockade. Seven-year follow-up of the phase III KEYNOTE-006 study: pembrolizumab versus ipilimumab in advanced melanoma. Robert, C. et al. Pembrolizumab versus ipilimumab in advanced melanoma (KEYNOTE-006): post-hoc 5-year results from an open-label, multicentre, randomised, controlled, phase 3 study. Five-year survival with combined nivolumab and ipilimumab in advanced melanoma. Five-year outcomes with pembrolizumab versus chemotherapy for metastatic non-small-cell lung cancer with PD-L1 tumor proportion score ≧ 50. The PD-1 expression balance between effector and regulatory T cells predicts the clinical efficacy of PD-1 blockade therapies. Oestreich, K. J., Yoon, H., Ahmed, R. & Boss, J. M. NFATc1 regulates PD-1 expression upon T cell activation. Expression of the PD-1 antigen on the surface of stimulated mouse T and B lymphocytes. Lam, K. C. et al. Microbiota triggers STING-type I IFN-dependent monocyte reprogramming of the tumor microenvironment. Fluckiger, A. et al. Cross-reactivity between tumor MHC class I-restricted antigens and an enterococcal bacteriophage. Bessell, C. A. et al. Commensal bacteria stimulate antitumor responses via T cell cross-reactivity. Griffin, M. E. et al. Enterococcus peptidoglycan remodeling promotes checkpoint inhibitor cancer immunotherapy. Hezaveh, K. et al. Tryptophan-derived microbial metabolites activate the aryl hydrocarbon receptor in tumor-associated macrophages to suppress anti-tumor immunity. Singh, N. et al. Activation of Gpr109a, receptor for niacin and the commensal metabolite butyrate, suppresses colonic inflammation and carcinogenesis. Kim, O. Y. et al. Bacterial outer membrane vesicles suppress tumor by interferon-γ-mediated antitumor response. Mager, L. F. et al. Microbiome-derived inosine modulates response to checkpoint inhibitor immunotherapy. Tanoue, T. et al. A defined commensal consortium elicits CD8 T cells and anti-cancer immunity. Gutierrez-Martinez, E. et al. Cross-presentation of cell-associated antigens by MHC class I in dendritic cell subsets. Cabeza-Cabrerizo, M., Cardoso, A., Minutti, C. M., da Costa, M. P. & Sousa, C. R. E. Dendritic cells revisited. Gros, A. et al. PD-1 identifies the patient-specific CD8+ tumor-reactive repertoire infiltrating human tumors. A. et al. PD-1 status in CD8+ T cells associates with survival and anti-PD-1 therapeutic outcomes in head and neck cancer. Thommen, D. S. et al. A transcriptionally and functionally distinct PD-1+ CD8+ T cell pool with predictive potential in non-small-cell lung cancer treated with PD-1 blockade. & Kurachi, M. Molecular and cellular insights into T cell exhaustion. Afrizal, A. et al. Anaerobic single-cell dispensing facilitates the cultivation of human gut bacteria. Wexler, A. G. & Goodman, A. L. An insider's perspective: Bacteroides as a window into the microbiome. Complete but curtailed T-cell response to very low-affinity antigen. & Gajewski, T. F. Tumor-residing Batf3 dendritic cells are required for effector T cell trafficking and adoptive T cell therapy. Grajales-Reyes, G. E. et al. Batf3 maintains autoactivation of Irf8 for commitment of a CD8α+ conventional DC clonogenic progenitor. Suzuki, S. et al. Critical roles of interferon regulatory factor 4 in CD11bhighCD8α− dendritic cell development. Origin of the lamina propria dendritic cell network. Murphy, T. L. et al. Transcriptional control of dendritic cell development. Sathaliyawala, T. et al. Mammalian target of rapamycin controls dendritic cell development downstream of Flt3 ligand signaling. & Flavell, R. A. STAT3 is required for Flt3L-dependent dendritic cell differentiation. Vrzalikova, K. et al. S1PR1 drives a feedforward signalling loop to regulate BATF3 and the transcriptional programme of Hodgkin lymphoma cells. Lollies, A. et al. An oncogenic axis of STAT-mediated BATF3 upregulation causing MYC activity in classical Hodgkin lymphoma and anaplastic large cell lymphoma. Peripheral CD103+ dendritic cells form a unified subset developmentally related to CD8α+ conventional dendritic cells. Kawai, T. & Akira, S. Signaling to NF-κB by toll-like receptors. Kugelberg, E. Innate immunity: making mice more human the TLR8 way. Jang, M. H. et al. CCR7 is critically important for migration of dendritic cells in intestinal lamina propria to mesenteric lymph nodes. & Cao, X. Dendritic cell migration in inflammation and immunity. The SCRUM-MONSTAR cancer-omics ecosystem: striving for a quantum leap in precision medicine. Routy, B. et al. Fecal microbiota transplantation plus anti-PD-1 immunotherapy in advanced melanoma: a phase I trial. Burger, M. L. et al. Antigen dominance hierarchies shape TCF1+ progenitor CD8 T cell phenotypes in tumors. Zitvogel, L., Ma, Y., Raoult, D., Kroemer, G. & Gajewski, T. F. The microbiome in cancer immunotherapy: diagnostic tools and therapeutic strategies. Relating the gut metagenome and metatranscriptome to immunotherapy responses in melanoma patients. The diversity of gut microbiome is associated with favorable responses to anti-programmed death 1 immunotherapy in Chinese patients with NSCLC. Matson, V., Chervin, C. S. & Gajewski, T. F. Cancer and the microbiome-influence of the commensal microbiota on cancer, immune responses, and immunotherapy. Targeting VEGFR2 with ramucirumab strongly impacts effector/activated regulatory T cells and CD8+ T cells in the tumor microenvironment. Isolation of tumor-infiltrating lymphocytes from preserved human tumor tissue specimens for downstream characterization. Saito, T. et al. Two FOXP3+CD4+ T cell subpopulations distinctly control the prognosis of colorectal cancers. Tourlousse, D. M. et al. Validation and standardization of DNA extraction and library construction methods for metagenomics-based human fecal microbiome measurements. Characterization and demonstration of mock communities as control reagents for accurate human microbiome community measurements. Segata, N. et al. Metagenomic biomarker discovery and explanation. Park, J. et al. Comprehensive analysis of gut microbiota of a healthy population and covariates affecting microbial variation in two large Japanese cohorts. Callahan, B. J. et al. DADA2: high-resolution sample inference from Illumina amplicon data. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Jain, C., Rodriguez, R. L., Phillippy, A. M., Konstantinidis, K. T. & Aluru, S. High throughput ANI analysis of 90 K prokaryotic genomes reveals clear species boundaries. Seemann, T. Prokka: rapid prokaryotic genome annotation. Galperin, M. Y. et al. COG database update: focus on microbial diversity, model organisms, and widespread pathogens. Kanehisa, M., Furumichi, M., Sato, Y., Ishiguro-Watanabe, M. & Tanabe, M. KEGG: integrating viruses and cellular organisms. Liu, B., Zheng, D., Zhou, S., Chen, L. & Yang, J. VFDB 2022: a general classification scheme for bacterial virulence factors. Buchfink, B., Reuter, K. & Drost, H. G. Sensitive protein alignments at tree-of-life scale using DIAMOND. Lu, J., Breitwieser, F. P., Thielen, P. & Salzberg, S. L. Bracken: estimating species abundance in metagenomics data. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Youngblut, N. D. & Ley, R. E. Struo2: efficient metagenome profiling database construction for ever-expanding microbial genome datasets. Dai, D. et al. GMrepo v2: a curated human gut microbiome database with special focus on disease markers and cross-dataset comparison. Hiseni, P., Rudi, K., Wilson, R. C., Hegge, F. T. & Snipen, L. HumGut: a comprehensive human gut prokaryotic genomes collection filtered by metagenome data. Application of a single-colony coculture technique to the isolation of hitherto unculturable gut bacteria. Salanitro, J. P., Fairchilds, I. G. & Zgornicki, Y. D. Isolation, culture characteristics, and identification of anaerobic bacteria from the chicken cecum. Shigeno, Y., Kitahara, M., Shime, M. & Benno, Y. Phascolarctobacterium wakonense sp. nov., isolated from common marmoset (Callithrix jacchus) faeces. Helft, J. et al. GM–CSF mouse bone marrow cultures comprise a heterogeneous population of CD11c+MHCII+ macrophages and dendritic cells. Identification of clonogenic common Flt3+M-CSFR+ plasmacytoid and conventional dendritic cell progenitors in mouse bone marrow. In vivo analysis of dendritic cell development and homeostasis. Kim, S. et al. High amount of transcription factor IRF8 engages AP1-IRF composite elements in enhancers to direct type 1 conventional dendritic cell identity. Toda, G., Yamauchi, T., Kadowaki, T. & Ueki, K. Preparation and culture of bone marrow-derived macrophages from mice for functional analysis. Barel, M. et al. A novel receptor–ligand pathway for entry of Francisella tularensis in monocyte-like THP-1 cells: interaction between surface nucleolin and bacterial elongation factor Tu. Jalal, N. & Lee, S. F. The MsrAB reducing pathway of Streptococcus gordonii is needed for oxidative stress tolerance, biofilm formation, and oral colonization in mice. Morinaga, K., Yamamoto, T., Nomura, N. & Toyofuku, M. Paracoccus denitrificans can utilize various long-chain N-acyl homoserine lactones and sequester them in membrane vesicles. Yasuda, M. et al. Phage genes induce quorum sensing signal release through membrane vesicle formation. Hogquist, K. A. et al. T cell receptor antagonist peptides induce positive selection. Hildner, K. et al. Batf3 deficiency reveals a critical role for CD8α+ dendritic cells in cytotoxic T cell immunity. Kumagai, S. et al. Lactic acid promotes PD-1 expression in regulatory T cells in highly glycolytic tumor microenvironments. Itahashi, K. et al. BATF epigenetically and transcriptionally controls the activation program of regulatory T cells in human tumors. Tamaki, H. et al. Armatimonas rosea gen. nov., sp. nov., of a novel bacterial phylum, Armatimonadetes phyl. Isolation of a member of the candidate phylum ‘Atribacteria' reveals a unique cell membrane structure. Tong, J., Liu, C., Summanen, P., Xu, H. & Finegold, S. M. Application of quantitative real-time PCR for rapid identification of Bacteroides fragilis group and related organisms in human wound samples. We thank members of the SCRUM-Japan MONSTAR-SCREEN alliance for providing the cohort study data; T. Kobayashi, T. Owari, H. Nishinakamura, D. Sugiyama and M. Tomura for experimental guidance and discussion; and Y. Ka, T. Ogura, R. Takahashi, Y. Tada, T. Takaku, M. Nakai, K. Onagawa, M. Takemura, M. Hoshino, C. Ozawa, A. Takakura, K. Yoshida, Y. Osada, M. Ozawa, Y. Ohira, S. Yoshimatsu, the animal facility team at the National Cancer Center Japan, K. Morinaga, M. Takashino and R. Norimine for their technical assistance. This study was supported by the Grants-in-Aid for Scientific Research (A grant no. 24K02306 to S. Koyama) from the Ministry of Education, Culture, Sports, Science and Technology of Japan; the Japan Agency for Medical Research and Development (AMED)-CREST grant (JP18gm1010005 to H.N., H.M. and S.N. ); the Moonshot Research and Development Program (grant no. 22zf0127009h0001 to H.N., H.M. and S. Koyama); the Next Generation Drug Discovery and Development Technology on Regulating Intestinal Microbiome (no. ); the program ‘The next-generation drug discovery and development technology on regulating intestinal microbiome' (NeDDTrim) from Japan Agency for Medical Research and Development (AMED) (under the Grant Number JP 21ae0121037h0001 to H.N. and S. Koyama); the Development of Technology for Patient Stratification Biomarker Discovery (grant no. ); the Project for Promotion of Cancer Research and Therapeutic Evolution (P-PROMOTE, no. ); the Practical Research for Innovative Cancer Control (no. from AMED; the National Cancer Center Research and Development Fund (no. These authors contributed equally: Nina Yi-Tzu Lin, Shota Fukuoka, Shohei Koyama Nina Yi-Tzu Lin, Shota Fukuoka, Shohei Koyama, Takuma Irie, Eri Sugiyama, Shogo Kumagai, Kota Itahashi, Tokiyoshi Tanegashima, Keisuke Watanabe, Yuka Maeda & Hiroyoshi Nishikawa Nina Yi-Tzu Lin, Kaori Fujimaki, Sachiko Ito & Hiroyoshi Nishikawa Department of Respiratory Medicine and Clinical Immunology, Osaka University Graduate School of Medicine, Osaka, Japan Department of Infection Metagenomics, Genome Information Research Center, Research Institute for Microbial Diseases, Osaka University, Osaka, Japan Daisuke Motooka, Yuki Matsumoto, Hiroyuki Yamano & Shota Nakamura Molecular Biosystems Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Ibaraki, Japan Dieter M. Tourlousse, Kazutoshi Murotomi & Yuji Sekiguchi Benno Laboratory, RIKEN Baton Zone Program, RIKEN Cluster for Science Technology and Innovation Laboratory, Saitama, Japan Biomanufacturing Process Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Ibaraki, Japan Department of Thoracic Oncology, National Cancer Center Hospital East, Chiba, Japan Division of Cellular Signaling, National Cancer Center Research Institute, Tokyo, Japan Department of Anatomy and Molecular Cell Biology, Nagoya University Graduate School of Medicine, Nagoya, Japan Mariko Shindo, Takahiro Tsuji & Hiroaki Wake Department of Head and Neck Medical Oncology, National Cancer Center Hospital East, Chiba, Japan Tomohiro Enokida, Makoto Tahara & Takao Fujisawa Division of Translational Informatics, Exploratory Oncology Research and Clinical Trial Center (EPOC), National Cancer Center, Chiba, Japan Translational Research Support Office, National Cancer Center Hospital East, Chiba, Japan Department of Head and Neck Oncology and Innovative Treatment, Graduate School of Medicine, Kyoto University, Kyoto, Japan Department of Gastroenterology and Gastrointestinal Oncology, National Cancer Center Hospital East, Chiba, Japan Akihito Kawazoe, Toshihiko Doi & Kohei Shitara Division of Cancer Immune Multicellular System Regulation, Center for Cancer Immunotherapy and Immunobiology, Kyoto University Graduate School of Medicine, Kyoto, Japan Kindai University Faculty of Medicine, Osaka, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar N.Y.-T.L., S.F., S. Koyama, D.M., D.M.T., Y. Shigeno, T.I., Y. Matsumoto, H.Y., K.M., H.T., Y. Sekiguchi, E.S., S. Kumagai, K.I., T. Tanegashima, K.F., S.I., M.S., T. Tsuji, H.W., K.W., Y. Maeda, H.M., S.N. isolated YB328 and developed the culture system. Correspondence to Yoshimi Benno or Hiroyoshi Nishikawa. S. Koyama receives research funding from Otsuka Pharmaceutical and Chugai Pharmaceutical outside this study and receives honoraria from MSD and Chugai Pharmaceutical. receives honoraria from MSD and Chugai Pharmaceutical. receives honoraria (lecture fees) from MSD, Ono Pharmaceutical and Bristol-Myers Squibb (BMS). receives personal fees from Daiichi-Sankyo, Lilly, Ono Pharmaceutical, Taiho Pharmaceutical, BMS, Merck Pharmaceutical, Sumitomo Dainippon Pharma and AstraZeneca outside this study. serves as a board member and a founder of ARC Therapies outside of this study. receives personal fees for advisory roles from Sumitomo Dainippon Pharma, Taiho Pharmaceutical, Takeda Pharmaceutical, Chugai Pharmaceutical, AbbVie, Bayer, Rakuten Medical, Otsuka Pharmaceutical, KAKEN Pharmaceutical, Kyowa Kirin, SHIONOGI, PRA Health Science, A2 Health Care, Noile-Immune Biotech, MSD, Daiichi-Sankyo, Amgen, Novartis, Boehringer Ingelheim, Janssen Pharmaceutical and Astellas Pharmaceutical; receives honoraria (lecture fees) from BMS, Rakuten Medical, Ono Pharmaceutical, Daiichi-Sankyo and AstraZeneca; and receives research funding from Lilly, MSD, Daiichi-Sankyo, Sumitomo Dainippon Pharma, Taiho Pharmaceutical, Novartis, Merck Pharmaceutical, Janssen Pharmaceutical, Boehringer Ingelheim, Pfizer, BMS, AbbVie, Eisai, IQVIA, Chugai Pharmaceutical and SHIONOGI outside this study. reports receiving personal fees for consulting and advisory roles from BMS, Takeda, Ono Pharmaceutical, Novartis, Daiichi Sankyo, Amgen, Boehringer Ingelheim, Merck Pharmaceutical, Astellas, Guardant Health Japan, Janssen, AstraZeneca, Zymeworks Biopharmaceuticals, ALX Oncology and Bayer; receiving honoraria from BMS, Ono Pharmaceutical, Janssen, Eli Lilly, Astellas and AstraZeneca; and receiving research funding (all to the institution) from Astellas, Ono Pharmaceutical, Daiichi Sankyo, Taiho Pharmaceutical, Chugai, Merck Pharmaceutical, Amgen, Eisai, PRA Health Sciences and Syneos Health, outside the submitted work. H.M. receives research grants from Ono Pharmaceutical, Daiichi Sankyo, PFDeNA, Konica-Minolta and Ambry Genetics and serves as a board member of CureGene outside this study. receives research funding and honoraria (lecture fees) from Ono Pharmaceutical, BMS, Chugai Pharmaceutical, BD Japan and MSD; receives honoraria (lecture fees) from Amgen; receives research funding from Taiho Pharmaceutical, Daiichi-Sankyo, Kyowa Kirin, Zenyaku Kogyo, Oncolys BioPharma, Debiopharma, Asahi-Kasei, Sysmex, Fujifilm, SRL, Astellas Pharmaceutical, Sumitomo Dainippon Pharma, ARC Therapies and RIKAKEN Holdings; and serves as a scientific advisor and a founder of ARC Therapies and a scientific advisor of LTZ Therapeutics outside this study. are the primary inventors on pending patents 2020-165470 belonging to RIKEN and the National Cancer Center Japan. The other authors declare no competing interests. Nature thanks Ben Boursi and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Only families with an abundance of >0.1% are shown. High/low Shannon diversity of the gut microbiota in patients was stratified by ROC analysis (left) Kaplan–Meier curves (right) for PFS of patients with high/low Shannon diversity of the gut microbiota (c, g). In NSCLC patients, the median PFS was 266 days (95% CI, 0 to 696) for those with high diversity, and 23 days (95% CI, 20 to 25) for those with low diversity. In patients with GC, the median PFS was 92 days (95% CI, 17 to 166) for those with high diversity, and 31 days (95% CI, 15 to 47) for those with low diversity (c, g). PCoA of the composition of the faecal microbiota based on unweighted Unifrac distances (d, h). Bar plots of the LDA scores of differentially abundant taxa in the Resp and N-Resp groups as determined by LEfSe analysis (e, i). j-l, Abundances of selected bacterial taxa. 1f; l: bacterial genera previously associated with response to immunotherapy). The ROC analysis (left) and Kaplan–Meier curves for PFS (right) for the enriched bacterial families in Fig. High/low abundances of the indicated bacterial families in patients were stratified by ROC analysis (k). m, Kaplan–Meier curves for the PFS of patients stratified by the median cut-off for each indicated factor. n, The correlations between bacteria that were significantly correlated with the clinical course and PD-1 expression by CD8+ T cells are shown. (c, g, k, m) Kaplan–Meier curves were analysed by the two-sided log-rank test. (b, f) Two-sided unpaired t test, and data are presented as the median. (j, l) Two-sided Mann–Whitney U test, and data are presented as the median. (d, h) ANOSIM, with one-tailed significance computed by permutation (n = 999 permutations). a-g, GF mice transplanted with faeces from Resps or N-Resps were subcutaneously injected with MC38 tumour cells. Tumour growth curves (b, left: summary, right: each mouse). c-g, The faecal samples were subjected to 16S rRNA gene sequencing to analyse the bacterial composition. Only families with an abundance of >0.1% are shown (d). Differentially abundant taxa were analysed by LEfSe, the symbol sizes are scaled by relative abundance (e). The relative abundance of Ruminococcaceae (left) and Bacteroidaceae (right)(g). Each dot indicates one mouse, and the data are presented as the median (c, g). (c, g) Two-sided unpaired t test. a-l, ATB-SPF mice were transplanted with faeces from Resp or N-Resp mice following the subcutaneous injection of tumour cells. Tumour growth curves of MC38 (b, left: summary, right: each mouse) and EMT6 (c, left: summary, right: each mouse) tumour models. d-f, The frequency [representative staining (left) and % summary (right)] of effector molecule expression [PD-1 (d, n = 10) and CD62L/CD44 (e, n = 12)] and cytokine (IFN-γ/TNFα) production (f, n = 12) by CD8+ T cells. g-i, TCR Vβ repertoire of PD-1+CD8+ T cells (g, h). j-l, The expression [representative staining (left) and MFI summary (right)] of maturation markers [CD86 (j), CD40 (k) and CD80 (l)] by DCs. (b, left; c, left), The average tumour sizes of the groups on a certain day of the experiment are shown as dots and are presented as the mean ± SD. (d-l) Each dot indicates one mouse, and the data are presented as the mean (i) or mean ± SD (d-f, h, j-l). (b, c) Two-way ANOVA with the Tukey—Kramer method. (d-f, j-l), One-way ANOVA with Bonferroni correction. Species names, strain identifiers and genome sequence accession numbers are presented in the tip labels, if applicable. Scale bar represents 10% amino-acid substitutions per site. b, Abundance of the YB328-related bacteria in: (upper panel) healthy Japanese individuals based on publicly available 16S rRNA gene sequencing data of two cohorts (MORINAGA cohort: n = 704 patients; NIBIOHN cohort: n = 1280 patients), and (lower panel) individuals from 21 countries across the world based on shotgun metagenomics data. For the upper panel, each symbol represents a different dataset. For the lower panel, abundances were calculated as the percentage of metagenomic reads assigned to the YB328 reference genome in our custom GTDB-based database. c, Faecal samples from patients, excluding MSI-H patients (NSCLC: n = 15; GC: n = 32), were subjected to shotgun metagenomic sequencing. High/low bacterial abundance was stratified by a ROC curve analysis (middle panel). Kaplan–Meier curves (right panel) of the PFS of patients with high/low faecal bacterial abundance are shown. d, High/low bacterial abundance of all GC/NSCLC patients (NSCLC: n = 22; GC: n = 49), stratified by the median cut-off value, was used to plot Kaplan–Meier curves of the PFS of patients. e, Experimental scheme (left) and tumour growth curves (middle: summary, right: each mouse). Average tumour sizes of the groups on a certain day of the experiment are shown as dots and are presented as the mean ± SD. f, YB328 colonisation detected in faecal sample by qPCR, n = 3 mice per group. g, The tumour growth curves are shown. Each line represents one mouse. (c, left panel) Two-sided Mann—Whitney U test and data are presented as the median. (c, right panel; d) Two-sided log-rank test. The frequency [representative staining (left) and % summary (right)] of effector molecule [perforin (b) and granzyme B (c)] production by CD8+ T cells. The frequency [representative staining (left) and % summary (right)] of Foxp3+ cells in CD4+ T cells (d) and PD-1 expression by Foxp3+CD4+ T cells (e). Expression [representative staining (left) and MFI summary (right)] of the maturation marker CD80 (f) by CD11c+MHCII+ DCs. g, Shannon diversity of microbiota after the administration of the indicated treatment. Tumour growth curves (i, n = 5–7 mice per group). Profile of the TCR Vβ repertoire in PD-1+CD8+ T cells (j, n = 5 mice per group). Changes in the Shannon index of microbiota diversity (k, n = 11–12 mice per group). l, The correlation between the abundance of YB328 and the Shannon index of microbiota diversity in all GC/NSCLC patients (NSCLC: n = 22; GC: n = 49) in this study. m, The experimental scheme is shown (left panel). n-o, ATB-SPF or N-Resp-FMT recipient mice received the indicated bacterial treatment. (m, middle) The average tumour size of the groups on a certain day is shown as a dot. (i, m left panel) Each line indicates one mouse for the in vivo experiment. (l) Each dot indicates one patient. (b-g) One-way ANOVA with Bonferroni's correction. (k) Two-sided paired t test. (m) Two-way ANOVA with the Tukey—Kramer method. (o) Two-sided unpaired t test. a, BMDCs were stimulated with P. vulgatus or YB328. The expression [representative staining (left) and MFI summary (right)] of the maturation marker CD80. b, The complexity and size of DCs (n = 30 in each group from Supplementary Video 1) stimulated with YB328 or P. vulgatus were evaluated. A representative correlation model for DCs is shown in the binary image. c, Chemokine production by YB328-, P. vulgatus- or vehicle-treated BMDCs was examined with a proteome profiler chemokine antibody array. Representative data (left) and quantified as fold changes (right) are shown. (a) One-way ANOVA with Bonferroni correction. (b) Each dot indicates one cell. (c) Data are presented as the mean. a-h, OT-I CD8+ T cells were cocultured with bacteria-treated BMDCs pulsed with the indicated antigen peptides (a-d, N4 peptide; e-h, Q4H7 peptide). (a, e), Representative data showing the expression of TCR signalling molecules (p-ZAP70, p-JNK, p-Erk1/2, p-Akt and p-S6K) by CD8+ T cells after stimulation with DCs pulsed with the indicated concentrations of peptides. (b, f), Representative data of PD-1 expression by CD8+ T cells after stimulation with DCs pulsed with the indicated concentrations of peptides. (c, g), Representative data of CD62L/CD44 expression by CD8+ T cells after stimulation with DCs pulsed with the indicated concentrations of peptides. (d, h), Cytokine (IFN-γ and TNFα) production [representative staining (left) and % summary (right)] by CD8+ T cells after stimulation with DCs pulsed with the indicated concentrations of peptides. The data were analysed by a two-sided unpaired t test and presented as the mean ± SD. a, A representative gating strategy for evaluating the progeny of CD103+CD11b− cDCs in BMDCs stimulated with YB328, P. vulgatus, or vehicle, corresponding to Fig. b-c, Representative staining (b) of the phenotype of CD103+CD11b− cDCs (% summary: c, left panel) or CD103+ DCs (% summary: c, right panel) in the indicated organs of WT or Batf3−/− MC38 tumour-bearing mice (n = 3). d, Tumour growth curves (n = 6–9). e-f, ATB-SPF mice were subcutaneously injected with MC38 cells and administered an anti-PD-1 mAb, anti-CSF1R mAb or control Ab with/without YB328 treatment. Changes in the immunological landscape are shown (e, n = 3). g, OT-I CD8+ T cells were cocultured with bacteria-treated BMDCs (DC) or BMDMs (MF) pulsed with antigen peptides. The frequency of PD-1 expression [representative staining (left) and % summary (right)] by OT-I CD8+ T cells is shown (n = 5 biologically replicates). (d; f, lower-right) Each line indicates one mouse for the in vivo experiment. The average tumour size of the groups on a certain day is shown as a dot and are presented as the mean ± SD (f, lower-left). (c, e), Two-sided unpaired t test. (f) Two-way ANOVA with the Tukey—Kramer method. (g) One-way ANOVA with Bonferroni's correction. a, The expression of TLRs (TLR1—TLR9) by GALT-isolated DCs was examined (upper panel, experimental scheme). b, Tumour growth curves of each mouse in Fig. c, The frequency of CD103+CD11b− cDCs [representative staining (left panel) and % summary (right panel)] in the indicated organs of WT or MyD88−/− mice (n = 3). d, Assessment of IRF signalling (upper panel) and NF-ĸB activity (lower panel) in TLR reporter THP-1 cells treated with/without YB328 (n = 3–4). e, Tumour growth curves of each mouse in Fig. f, BMDCs were stimulated with YB328 or P. vulgatus at an MOI of 25 and then treated with flagellin. OT-I CD8+ T cells were cocultured with the stimulated BMDCs pulsed with 1 nM N4 peptides. PD-1 expression by CD8+ T cells [representative staining (left panel) and % summary (right panel)] is shown, n = 5. g. Tumour growth curves of each mouse in Fig. h, BMDCs were treated with the indicated TLR ligands. The levels [representative staining (left panel) and MFI summary (right panel)] of p-S6K, p-STAT3 and IRF8 are shown (n = 4). i, Tumour growth curves (n = 3–4). j, Tumour growth curves of each mouse in Fig. (b, e, g, i-j) Each line indicates one mouse for the in vivo experiment. (d, f, h) Each dot in the summary graphs indicates a well, and the data are presented as the mean ± SD. (a, c-d), Two-sided unpaired t test. (f, h) One-way ANOVA with Bonferroni's correction. The frequency [representative staining (left) and % summary (right)] of PD-1 expression by CD8+ T cells. b, Tumour growth curves (left: summary, right: each mouse). c-f, Comparative genomics of YB238 and other six strains (referred to as “Reference strains”) Venn diagram indicating the number of genes unique to YB328 (c). Bar plots of the distribution of genes annotated to different categories according to COG (d), KEGG (e), and VFDB (f). Values were calculated for all genes of YB328 (red bars), genes unique to YB328 (green bars), and the other six strains (blue bars show the mean ± SD). g, BMDCs were treated with each of four cell culture fractions of YB328 [G1: cytoplasm, G2: cell membrane, G3: culture supernatant (without membrane vesicle) or G4: membrane vesicle] and cocultured with OT-I CD8+ T cells stimulated with N4 peptide-pulsed DCs. The expression of the indicated proteins was measured by flow cytometry (n = 3) and normalized between samples. h, Tumour growth curves (upper: summary, lower: each mouse). i-m, BMDCs were stimulated with the indicated bacterial strains. The frequency [% summary (left) and representative staining (right)] of PD-1 expression by CD8+ T cells (j). The expression of IRF8 (k), p-S6K (l) and p-STAT3 (m) in BMDCs. The MFI summary (left) and representative staining (right) are shown. (a, i-m) Each dot indicates one well for the in vitro experiment in the summary data. (b, right; h, bottom) Each line represents one mouse. (a, i-m) Data are presented as the mean ± SD. (i-m) One-way ANOVA with Bonferroni correction. (h) Two-way ANOVA with the Tukey–Kramer method. a-c, ATB-SPF mice were subcutaneously injected with MC38 cells and treated with the indicated bacteria. The phenotype and frequency [representative staining (left) and % summary (right)] of DC populations in the indicated organs were examined: NdLNs (a) and SPs (b). TCF-1 expression by PD-1+CD8+ T cells in the indicated organs after the indicated treatment. Representative staining (upper) and % summary (lower) (c). Before bacterial treatment, KikGR mice were surgically photoconverted. Migratory intestinal cells that expressed KikR+ were profiled in the indicated organs. h, WT GF mice were subcutaneously injected with MC38 cells and were administered an anti-PD-1 mAb with YB328 treatment. Tumour were collected at days 20. Each dot in the summary graphs indicates one mouse. (a, b, d-g) Two-sided unpaired Student's t test. a-c, Representative images of multiplex immunohistochemical staining of tumours from GC patients (a) and the calculated density of immune cells. Correlations between the relative abundance of YB328 in the gut microbiota and the density of CD206+ cells (left) or CD14+ cells (right) (b, n = 20). e, High/low YB328 abundance was stratified by a ROC curve analysis (left panel) Kaplan—Meier curves for the PFS of patients with high/low YB328 abundance in the validation cohort (right panel). g, h, Relative abundance of the YB328 phylotype (g, h), and Akkermansia (g) in faecal samples across datasets from multiple cohorts. i, ATB-SPF mice were subcutaneously injected with B16F10-OVA cells, and OT-I CD8+ T cells were transferred with/without YB328 treatment. The graph shows representative data from two independent experiments. (k) was created using BioRender (https://www.biorender.com). Data are presented as the median (d) or mean ± SD (f). (b-d, f-h) Each dot indicates one patient. (d, f, g) Two-sided Mann–Whitney test. (e, right) Two-sided log-rank test. (h) Two-sided paired t test. (i) Two-way ANOVA with the Tukey–Kramer method. Genomic proximity (% ANI) between strain YB328 and other strains of the H. mulieris species. Closeness of strain YB328 to other strains within the H. mulieris species. Phenotypic traits and biochemical properties of YB328 compared with H. mulieris, N. timonensis, A. muris and Scatolibacter rhodanostii. Unique genes in YB238 in comparison with the genomes of Acutalibacter muris DSM 26090, N. timonensis DSM 102082, C. leptum DSM 753, R. albus JCM 14654, R. torques strain AE30, and P. vulgatus AE61. Correlations of clinicopathological features with the clinical responses in MONSTAR cohort. Clinicopathological features, including age, sex, the number of previous chemotherapy treatments and the number of metastatic organ sites, are reported. Dynamic imaging of CD8+ T cells (cyan) and BMDCs stimulated with YB328 (green) or P. vulgatus (magenta). Elapsed time is shown as mm:ss. The interaction of T cells with YB328-stimulated BMDCs. Dynamic imaging of CD8+ T cells (cyan) and BMDCs stimulated with YB328 (green). Elapsed time is shown as mm:ss. The interaction of T cells with P. vulgatus-stimulated BMDCs. Dynamic imaging of CD8+ T cells (cyan) and BMDCs stimulated with P. vulgatus (magenta). Elapsed time is shown as mm:ss. Dynamic imaging of BMDCs (red) co-cultured with YB328 (green). Elapsed time is shown as mm:ss. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Lin, N.YT., Fukuoka, S., Koyama, S. et al. Microbiota-driven antitumour immunity mediated by dendritic cell migration. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03808-2'>The exposome of healthy and accelerated aging across 40 countries</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 15:19:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Protective and risk factors can drive healthy or accelerated aging, with distinct environments modulating their effects. The impact of the exposome—the combined physical and social exposures experienced throughout life—on accelerated aging remains unknown. We assessed delayed and accelerated aging in 161,981 participants from 40 countries (45.09% female; mean age, 67.06; s.d., 9.85) by measuring biobehavioral age gaps (BBAGs), defined as the difference between estimated age from protective and risk factors and chronological age, in cross-sectional and longitudinal designs. BBAGs predicted chronological age, followed by regional and exposomal factor analyses, linked to accelerated aging. Europe led in healthy aging, while Egypt and South Africa showed the greatest acceleration; Asia and Latin America fell in between (Cliff's delta (δd) = 0.15–0.52; all P < 0.0001). Accelerated aging was more evident in eastern and southern Europe; globally, it was also associated with lower income (δd = 0.48–0.56, P < 1 × 10−15). Exposomal factors of accelerated aging include physical (air quality), social (socioeconomic and gender inequality, migration) and sociopolitical (representation, party freedom, suffrage, elections and democracy) determinants (all Cohen's d (d) > 0.37, P < 0.0001). BBAGs predicted future functional (r (Pearson correlation) = −0.33, P < 1 × 10−15, d = 0.70) and cognitive declines (r = −0.22, P < 1 × 10−15, d = 0.44), and larger BBAGs (P < 0.0001, d = 1.55). Healthy and accelerated aging are influenced by physical, social and sociopolitical exposomes, with considerable disparities across nations. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Preprocessed data are freely available via GitHub (https://github.com/euroladbrainlat/Biobehavioral-age-gaps). Additional details can be found in Supplementary Table 11. The datasets come from sources that are either publicly available for direct download after registration or accessible upon request. Data may be shared upon approval of a brief research proposal and signing of a data use agreement. The estimated response time is 3–4 weeks. For India (LASI), requests can be directed to S. Bajpai (swati.bajpai@gbhi.org). Access is granted based on a research purpose review and data sharing agreement. Timelines for access may vary depending on the review process, typically within 3–4 weeks. For South Africa (GHS), data requests should be addressed to C. Mostert (cyprian.mostert@aku.edu). Data may be shared upon approval of a brief research proposal and the signing of a data use agreement. The estimated response time is typically 3–4 weeks. Country-level indicators of GNI, GDP, air quality, socioeconomic inequality (Gini index) and migration were obtained from the World Bank (https://databank.worldbank.org/). Sociopolitical indicators were sourced from the Global State of Democracy Indices (https://www.idea.int/democracytracker/dataset-resources). Analysis codes are freely available via GitHub (https://github.com/euroladbrainlat/Biobehavioral-age-gaps). Fried, L. P. & Rowe, J. W. Health in aging—past, present, and future. Brain clocks capture diversity and disparities in aging and dementia across geographically diverse populations. Tian, Y. E. et al. Heterogeneous aging across multiple organ systems and prediction of chronic disease and mortality. This AI ‘brain clock' points to answers. Allada, R. & Bass, J. Circadian mechanisms in medicine. Rutledge, J., Oh, H. & Wyss-Coray, T. Measuring biological age using omics data. Nativio, R. et al. An integrated multi-omics approach identifies epigenetic alterations associated with Alzheimer's disease. Thuault, S. Reflections on aging research from within the National Institute on Aging. Tran Van Hoi, E. et al. Biomarkers of the ageing immune system and their association with frailty—a systematic review. Even the field's researchers can't agree. Keshavarz, M., Xie, K., Schaaf, K., Bano, D. & Ehninger, D. Targeting the “hallmarks of aging” to slow aging and treat age-related disease: fact or fiction? Zampino, M. et al. Biomarkers of aging in real life: three questions on aging and the comprehensive geriatric assessment. Livingston, G. et al. Dementia prevention, intervention, and care: 2024 report of the Lancet standing Commission. Paradela, R. S. et al. Population attributable fractions for risk factors for dementia in seven Latin American countries: an analysis using cross-sectional survey data. & Livingston, G. Population attributable fractions for risk factors for dementia in low-income and middle-income countries: an analysis using cross-sectional survey data. Gonzalez-Gomez, R. et al. Qualitative and quantitative educational disparities and brain signatures in healthy aging and dementia across global settings. Santamaria-Garcia, H. et al. Factors associated with healthy aging in Latin American populations. Healthy aging meta-analyses and scoping review of risk factors across Latin America reveal large heterogeneity and weak predictive models. Gonzalez-Gomez, R. et al. Educational disparities in brain health and dementia across Latin America and the United States. Structural inequality linked to brain volume and network dynamics in aging and dementia across the Americas. Miranda, J. J. et al. Understanding the rise of cardiometabolic diseases in low- and middle-income countries. Ibanez, A. et al. Neuroecological links of the exposome and One Health. & Eyre, H. Brain capital, ecological development and sustainable environments. Abolhasani, E. et al. Air pollution and incidence of dementia: a systematic review and meta-analysis. Fuller, R. et al. Pollution and health: a progress update. Zugman, A. et al. Country-level gender inequality is associated with structural differences in the brains of women and men. Lorenz-Spreen, P., Oswald, L., Lewandowsky, S. & Hertwig, R. A systematic review of worldwide causal and correlational evidence on digital media and democracy. Van Bavel, J. J., Gadarian, S. K., Knowles, E. & Ruggeri, K. Political polarization and health. The Aging Dimension: Perspectives in Behavioral Medicine (Routledge, 2024). The relationship between epigenetic age and the hallmarks of aging in human cells. Farina, F. R. et al. Next generation brain health: transforming global research and public health to promote prevention of dementia and reduce its risk in young adult populations. Structural inequality and temporal brain dynamics across diverse samples. Wu, Y. T. et al. Education and wealth inequalities in healthy ageing in eight harmonised cohorts in the ATHLOS consortium: a population-based study. Livingston, G. et al. Dementia prevention, intervention, and care: 2020 report of the Lancet Commission. Livingston, G. et al. Dementia prevention, intervention, and care. Ibanez, A., Kringelbach, M. & Deco, G. A synergetic turn in cognitive neuroscience of brain diseases. & Vahia, I. V. Successful cognitive and emotional aging. Moffitt, T. E. Behavioral and social research to accelerate the geroscience translation agenda. Local Burden of Disease Educational Attainment CollaboratorsMapping disparities in education across low- and middle-income countries. & Ruiz-Adame, M. Addressing the gaps between socioeconomic disparities and biological models of dementia. Global South research is critical for understanding brain health, ageing and dementia. Global and regional hearing impairment prevalence: an analysis of 42 studies in 29 countries. Unequal burdens: how structural socioeconomic inequality shapes brain health in aging and dementia. Artero, S. et al. Risk profiles for mild cognitive impairment and progression to dementia are gender specific. Investigating sex differences in risk and protective factors in the progression of mild cognitive impairment to dementia: a systematic review. Sex differences in biological aging with a focus on human studies. Gender differences in the association between hearing loss and cognitive function. Baez, S., Castro-Aldrete, L., Britton, G. B., Ibañez, A. Enhancing brain health in the Global South through a sex and gender lens. Z. K. & Isenberg, N. The aging brain: risk factors and interventions for long term brain health in women. Is the political system really related to health? The relationships between democratic experience, adult health, and cause-specific mortality in 170 countries between 1980 and 2016: an observational analysis. Fraser, T., Aldrich, D. P., Panagopoulos, C., Hummel, D. & Kim, D. The harmful effects of partisan polarization on health. Santamaría-García, H. et al. Allostatic interoceÿptive overload across psychiatric and neurological conditions. A predictive coding framework of allostatic–interoceptive overload in frontotemporal dementia. & Slachevsky, A. Environmental–genetic interactions in ageing and dementia across Latin America. Allison, P. Multiple imputation for missing data: a cautionary tale. Friedman, J. Greedy function approximation: a gradient boosting machine. Muller, A. C. & Guido, S. Introduction to Machine Learning with Python: A Guide for Data Scientists (O'Reilly Media, 2018). Selya, A. S., Rose, J. S., Dierker, L. C., Hedeker, D. & Mermelstein, R. J. A practical guide to calculating Cohen's f2, a measure of local effect size, from PROC MIXED. Breiman, L. Classification and Regression Trees (Routledge, 2017). Feurer, M. & Hutter, F. in Automated Machine Learning: Methods, Systems, Challenges (eds Hutter, F. et al.) 3–33 (Springer, 2019). Bzdok, D., Altman, N. & Krzywinski, M. Statistics versus machine learning. Chan, J. Y.-L. et al. Mitigating the multicollinearity problem and its machine learning approach: a review. Smith, S. M., Vidaurre, D., Alfaro-Almagro, F., Nichols, T. E. & Miller, K. L. Estimation of brain age delta from brain imaging. Economy and Growth Indicators (World Bank, accessed September 2024); https://data.worldbank.org/indicator Meissel, K. & Yao, E. S. Using Cliff's delta as a non-parametric effect size measure: an accessible web app and R tutorial. Evaluating effect size in psychological research: sense and nonsense. A. Subjective wellbeing, health, and ageing. Andrade, C. Understanding relative risk, odds ratio, and related terms: as simple as it can get. Paul, P., Pennell, M. L. & Lemeshow, S. Standardizing the power of the Hosmer–Lemeshow goodness of fit test in large data sets. is supported by Davos Alzheimer's collaborative. is supported by grants from the Multi-Partner Consortium to Expand Dementia Research in Latin America (ReDLat, supported by Fogarty International Center (FIC), National Institutes of Health, National Institutes of Aging (R01 AG057234, R01 AG075775, R01 AG21051, R01 AG083799, CARDS-NIH), Alzheimer's Association (SG-20-725707), Rainwater Charitable Foundation—The Bluefield Project to Cure FTD, and Global Brain Health Institute), ANID/FONDECYT Regular (1210195 and 1210176 and 1220995), ANID/PIA/ANILLOS ACT210096, FONDEF ID20I10152 and ANID/FONDAP 15150012. is supported by Alzheimer's Association Fellowship (AARF-21-848281). is supported by ANID/FONDECYT Regular 1210622, ANID/PIA/ANILLOS ACT210096, Alzheimer's Association (AARGD-24- 1310017), ANID/FOVI240065 and ANID/Proyecto Exploración 13240170. J. Cruzat is supported by ANID (FONDECYT Postdoctorado number 3240042). H.S.-G. is supported by NIH R01 (Social Epigenetics of Alzheimer's Disease and Related Dementias in Latin American Countries, number: 1R01AG082056-01A1), Global Brain Health Institute and Alzheimer Association (‘Brain Health in Individuals with Exposition to High Violence in Colombia', number: GBHI ALZ UK-23-971135). In addition, research reported in this publication was supported by the Fogarty International Center of the National Institutes of Health under Award Number D43TW012455. acknowledges having received support from the Alliance for Health Policy and Systems Research (2009/32034, 2012/253750), Bloomberg Philanthropies (grant 46129, via University of North Carolina at Chapel Hill School of Public Health), FONDECYT via CIENCIACTIVA/CONCYTEC, British Council, British Embassy and the Newton–Paulet Fund (223-2018, 224-2018), DFID/MRC/Wellcome Global Health Trials (MR/M007405/1), Fogarty International Center (R21TW009982, D71TW010877, R21TW011740, K01TW011478), Grand Challenges Canada (GMH-POC-0335-04), International Development Research Center Canada (IDRC 106887, 108167), Inter-American Institute for Global Change Research (IAI CRN3036), National Cancer Institute (NCI 1P20CA217231), National Council for Scientific and Technological Development (CNPq Brasil 408523/2023-9), National Health and Medical Research Council (NHMRC 2022566), National Heart, Lung and Blood Institute (NHLBI HHSN268200900033C, 5U01HL114180, 1UM1HL134590), National Institute for Health and Care Research (NIHR 150261, NIHR 150287), National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK K23DK135798), National Institute of Mental Health (NIMH 1U19MH098780), NSW Health, Cardiovascular Elite Postdoctoral Researcher Grants (H23/37663), Swiss National Science Foundation (40P740-160366), UKRI BBSRC (BB/T009004/1), UKRI EPSRC (EP/V043102/1), UKRI MRC (MR/P008984/1, MR/P024408/1, MR/P02386X/1, MR/X004163/1, MR/X020851/1), Wellcome (074833/Z/04/Z, 093541/Z/10/Z, 103994/Z/14/Z, 107435/Z/15/Z, 205177/Z/16/Z, 214185/Z/18/Z, 218743/Z/19/Z), World Diabetes Foundation (WDF15-1224) and the World Health Organization (2021/1189041, 2022/1249357). is supported by postdoctoral fellowships granted by the Multi-Partner Consortium to Expand Dementia Research in Latin America (ReDLat) and the Atlantic Fellows for Equity in Brain Health program. Although not directly funding this work, CU receives research funding support from The UKRI Medical Research Council (MR/Y019822/1), Alzheimer's Association (SAGA23-1141999), The Wellcome Leap Dynamic resilience program (co-funded by Temasek Trust), National Institute of Health (NIH) (RO1-AG074562), Office for Veterans' Affairs UK Defense and Security Accelerator (DASA) Fund (G2-SCH-2022-11-12245), Global Brain Health Institute (UFRA-424|CA-0241758), Davos Alzheimer's Collaborative Global Cohorts Fund and RoseTrees Foundation (Seedcorn2021\100220). is supported by Global Brain Health Institute, Alzheimer's Association, Alzheimer's Society UK, Pilot Awards for Global Brain Health Leaders (Grant Number: GBHI ALZ UK- 25-1289623). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. These authors contributed equally: Hernan Hernandez, Hernando Santamaria-Garcia. Latin American Brain Health Institute (BrainLat), Universidad Adolfo Ibañez, Santiago de Chile, Chile Hernan Hernandez, Sebastian Moguilner, Agustina Legaz, Jhosmary Cuadros, Raul Gonzalez-Gomez, Joaquín Migeot, Carlos Coronel-Oliveros, Enzo Tagliazucchi, Marcelo Adrián Maito, María E. Godoy, Josephine Cruzat, Sol Fittipaldi, Vicente Medel, Claudia Duran-Aniotz & Agustin Ibanez PhD Program in Neuroscience, Pontificia Universidad Javeriana, Bogotá, Colombia Center of Memory and Cognition Intellectus, Hospital Universitario San Ignacio, Bogotá, Colombia Cognitive Neuroscience Center, Universidad de San Andrés, Buenos Aires, Argentina Sebastian Moguilner, Agustina Legaz, Marcelo Adrián Maito, María E. Godoy, Florencia Altschuler & Agustin Ibanez Department of Neurology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Francesca R. Farina, Joaquín Migeot, Carlos Coronel-Oliveros, Swati Bajpai, Mohamed Salama, Sara A. Moustafa, Sol Fittipaldi, Brian Lawlor, Sandra Baez & Agustin Ibanez Grupo de Bioingeniería, Decanato de Investigación, Universidad Nacional Experimental del Táchira, San Cristóbal, Venezuela Advanced Center for Electrical and Electronic Engineering, Universidad Técnica Federico Santa María, Valparaíso, Chile Department of Medicine, University College Hospital, Ibadan, Nigeria Graduate Program in Biological Sciences: Pharmacology and Therapeutics, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil Lucas Uglione Da Ros & Wyllians V. Borelli Department of Morphological Sciences, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil Graduate Program in Biological Sciences: Biochemistry (PPGBioq), UFRGS, Porto Alegre, Brazil Department of Pharmacology, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil Neuroscience Lab, Faculty of Medicine and Biomedical Sciences, The University of Yaounde I, Yoaunde, Cameroon Brain and Mind Institute, Aga Khan University, Nairobi, Kenya Cyprian Mostert, Zul Merali & Chinedu Momoh Department of Population Health, Aga Khan University, Nairobi, Kenya Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada Institute of Global Health and Human Ecology, The American University in Cairo, New Cairo, Egypt Center for Social and Cognitive Neuroscience, School of Psychology, Universidad Adolfo Ibáñez, Santiago, Chile Department of Psychiatry and Behavioral Sciences, Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA, USA Department of Neurology, University of California, San Francisco, San Francisco, CA, USA Neuro-Policy Program, Center for Health and Biosciences, Baker Institute for Public Policy, Rice University, Houston, TX, USA Euro-Mediterranean Economists Association, Barcelona, Spain Department of Psychiatry and Behavioral Sciences, University of California, San Francisco, San Francisco, CA, USA International Center for Future Generations (ICFG), Brussels, Belgium Brain Capital Alliance, San Francisco, CA, USA Sydney School of Public Health, Faculty of Medicine and Health, University of Sydney, Sydney, New South Wales, Australia CRONICAS Center of Excellence in Chronic Diseases, Universidad Peruana Cayetano Heredia, Lima, Peru Universidad de los Andes, Bogota, Colombia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar were responsible for conceptualization and formal analysis, and wrote the original draft. All authors contributed to the review and editing. had access to the raw data. All authors had access to all data in the study, and the corresponding authors had final responsibility for the decision to submit for publication. Correspondence to Sandra Baez or Agustin Ibanez. has served on scientific advisory boards for Nintx, Novo Nordisk and Masima. The other authors declare no competing interests. Nature Medicine thanks Yao Yao and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ming Yang, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The figure illustrates the age distribution across countries for three different datasets. a) Age distribution for the cross-sectional dataset, which includes a total of 161,981 participants. b) Age distribution for the cross-sectional subsample consisting of 102,725 participants, with additional predictors available for analysis. c) Age distribution for the longitudinal dataset, where the same group of 21,631 participants is observed across two waves (Wave 1 and Wave 2), highlighting the distribution of age by country in each wave. (a) Feature importance, assessed via mean decrease in impurity (MDI), enabled the prediction of chronological age using biobehavioral factors. The sample size for the analyses reported in this figure included 102,725 individuals. Goodness-of-fit and feature importance metrics are provided. (b) MDI facilitated the characterization of groups with more delayed (left panel) and more accelerated (right panel) aging. Goodness-of-fit and feature importance metrics are provided. (c) Average BBAGs distribution by continent. The color bar indicates younger (blue) and older (red) BBAGs. (d) BBAGs comparisons by continent (left panel) and by European regions (right panel). (e) BBAGs comparisons between low- and high-income countries, based on gross national income (GNI) and gross domestic product (GDP) indicators. (f) Linear regression models were used to assess the relationship between BBAGs and all exposomes, as well as combined social, physical, and sociopolitical exposomes. (g) Linear regression models were also used to assess the associations between BBAGs and individual social (gender equality, migration, and structural equality) and physical exposomes (air quality). (h) Linear regression models were used to examine the association between BBAGs and individual sociopolitical exposomes (democracy indicators). All P-values reported in Panels f, g, and h were < 1e-15. All other illustrations and icons were designed using GIMP (https://www.gimp.org). The sample size for the analyses reported in this figure included 161,981 individuals. (a) Linear regression models tracked the relationship between BBAGs and combined social exposomes, controlling for physical and sociopolitical exposomes. (b) Linear regression models assessed the role of individual social exposomes (gender equality, migration, and structural equality) and physical exposomes (air quality) on BBAGs, controlling for sociopolitical exposomes. (c) Linear regression models examined the association between BBAGs and individual sociopolitical exposomes (democracy indicators), controlling for social and physical exposomes. We used a one-sided F-test to evaluate the statistical significance of the linear regression model, testing whether the model explains a substantially greater proportion of variance than expected by chance. (d) Linear fit between BBAGs and all exposomes, excluding zones with sparse exposome density, and combined social, physical, and sociopolitical exposomes. (e) Linear regression models assessing the relationship between BAGs and individual social (gender equality, migration, and structural equality) and physical exposomes (air quality), excluding zones with sparse exposome density. (f) Linear regression models examining the association between BBAGs and individual sociopolitical exposomes (democracy indicators), excluding zones with sparse exposome density. All P-values reported in Panels (a) to (f) were < 1e-15. The sample size for the analyses reported in this figure included 21,631 individuals. (a) Linear regression models tracked the relationships between recalculated BBAGs in wave 1 (excluding the specific healthy aging factor used as the outcome) and healthy aging factors (cognition, functional ability, and well-being) in wave 2. Recalculated BBAGs in wave 1 significantly predicted poorer cognition, functional ability, and well-being in wave 2. The sample size for the analyses reported in this figure included 5,431 individuals. (a) Linear regression models were used to assess the relationships between BBAGs in wave 1 and healthy aging factors (walking and memory) in wave 2. BBAGs in wave 1 significantly predicted poorer cognition, functional ability, and well-being in wave 2. (b) Linear regression models tracked the relationships between BBAGs in wave 1 as significant predictors of BBAGs in wave 2. (a) Odds ratios and attributable risk for the cross-sectional subsample. Results showed that BBAGs are significantly associated with poorer functional ability and cognition. BBAGs in wave 1 were significantly associated with declines in walking and memory, despite the effects on walking ability showing an uncertain confidence interval. Analyses reported in this panel included 3,868 individuals. (c) Meta-analysis on cross-sectional subsample for cognition, functional ability, and wellbeing using common-effects and random-effects models. Larger BBAGs are linked to poorer outcomes in all domains, with high heterogeneity across countries. All P-values reported in this panel were < 0.01. We used Cochran's Q test to assess heterogeneity across studies, reporting the associated p-value, degrees of freedom, and I2 statistic. (d) Summary of cross-sectional results, showing the average of common- and random-effects models by income level (low or high) based on gross national income (GNI) and gross domestic product (GDP) classifications. Color bar indicates effect sizes (small: blue and large: red). Accelerated aging is more strongly linked to poorer healthy aging outcomes—cognition, functional ability, and well-being—in low-income countries compared to high-income countries for both classifications (GNI: p = 4.05e-9, r = 0.82 and GDP: p = 8.35e-4, r = 0.47). All other illustrations and icons were designed using GIMP (https://www.gimp.org). (a) Feature importance analyses enabled the prediction of chronological age using biobehavioral factors. The sample size for the analyses reported in this figure included 148,188 individuals. Goodness-of-fit and feature importance metrics are provided. (b) Mean decrease in impurity (MDI) facilitated the characterization of groups with more delayed (left panel) and more accelerated (right panel) aging. Goodness-of-fit and feature importance metrics are provided. The sample size for the analyses reported in this figure was 148,188 individuals. Panels (a) and (c): Feature importance analyses enabled the prediction of chronological age using biobehavioral factors in females and males, respectively. Goodness-of-fit and feature importance metrics are provided. Panels (b) and (d): Feature importance analyses characterized groups with more delayed (left panel) and more accelerated (right panel) aging in females and males, respectively. Goodness-of-fit and feature importance metrics are provided. Panels (e), (f), and (g): Linear regression models tracked the relationship between BBAGs and various social exposomes across sex: Panel (e): Combined social exposomes. Panel (f): Social and physical exposomes. Supplementary Notes 1–6, Tables 1–11 and References. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Hernandez, H., Santamaria-Garcia, H., Moguilner, S. et al. The exposome of healthy and accelerated aging across 40 countries. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250714052108.htm'>It looked like nothing—then scientists found a world 10x the size of Jupiter</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 13:56:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Astronomers have detected a giant exoplanet - between three and ten times the size of Jupiter - hiding in the swirling disc of gas and dust surrounding a young star. Earlier observations of this star, called MP Mus, suggested that it was all alone without any planets in orbit around it, surrounded by a featureless cloud of gas and dust. This is the first time that Gaia has detected an exoplanet within a protoplanetary disc. By studying how planets form in the protoplanetary discs around young stars, researchers can learn more about how our own Solar System evolved. Through a process known as core accretion, gravity causes particles in the disc to stick to each other, eventually forming larger solid bodies like asteroids or planets. As young planets form, they start to carve gaps in the disc, like grooves on a vinyl record. To date, only three robust detections of young planets in a protoplanetary disc have been made. Dr Álvaro Ribas from Cambridge's Institute of Astronomy, who led the research, specialises in studying protoplanetary discs. "We first observed this star at the time when we learned that most discs have rings and gaps, and I was hoping to find features around MP Mus that could hint at the presence of a planet or planets," he said. Using ALMA, Ribas observed the protoplanetary disc around MP Mus (PDS 66) in 2023. "Our earlier observations showed a boring, flat disc," said Ribas. "But this seemed odd to us, since the disc is between seven and ten million years old. "My first reaction was that I must have made a mistake in my calculations, because MP Mus was known to have a featureless disc," said Vioque. "Our modelling work showed that if you put a giant planet inside the new-found cavity, you can also explain the Gaia signal," said Ribas. It also means that many more hidden planets might exist in other discs, just waiting to be found. "We think this might be one of the reasons why it's hard to detect young planets in protoplanetary discs, because in this case, we needed the ALMA and Gaia data together," said Ribas. Ribas says that upcoming upgrades to ALMA, as well as future telescopes such as the next generation Very Large Array (ngVLA), may be used to look deeper into more discs and better understand the hidden population of young planets, which could in turn help us learn how our own planet may have formed. Note: Content may be edited for style and length. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a65370112/tesla-jack-mode/'>Tesla's Fancy Suspensions Can Blow If You Skip One Simple Step. Most Owners Have No Idea.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For better or worse, car ownership will always require some amount of maintenance that may sometimes require you to jack your car up into the air. This is especially important for Tesla vehicles, as forgetting to engage the maintenance setting could cause significant damage to your vehicle—and your eardrums. Tesla's vehicles with air suspension—the Model X, Model S, and Cybertruck—will have a “jack mode” that needs to be engaged before lifting the car. You'll likely know if your Tesla is equipped with air suspension, but if you're unsure, it's easy to check. (Both vehicles were designed to accommodate air suspension, but the idea was scrapped before they went into production.) As it turns out, there's more reason to jack up your Tesla than merely fixing a flat tire. At the more basic end, Unplugged Performance makes aftermarket wheels and tires. However, they also offer much more advanced components like bigger brakes, more complicated springs and dampers, racing equipment installations, and even a full race prep program (if you're looking to go racing). For the Cybertruck, Unplugged offers more off-road focused components like lift kits, racks, bumpers, and even rock sliders, among many other accessories. It would be easy to dismiss the potential damages that would result from not engaging jack mode, as the majority of motorists don't actually wrench on their cars. Most cars in Tesla's range come with a tire repair kit and an air pump instead of a spare tire. Though that may come as a surprise to some, many automakers have done their research, and believe that the improvement in efficiency cause by not hauling around an extra tire outweighs the risk of leaving their customers stranded. Matt Crisara is a native Austinite who has an unbridled passion for cars and motorsports, both foreign and domestic. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. AI Is Giving This Car's Engine a Second Life In Hyundai's Factory, Your Coworkers May Be Robots You Can Soon Simply Swap Out Your Car's Battery You Can Charge Your Car Using This Curb Hyundai Built a Robot That Will Charge Your EV</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a65374656/dams-pole-movement/'>Whoopsie, Humans Built So Many Dams That We Shifted the North Pole by 3 Feet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Dams have long been one of humanity's greatest tools for controlling the world's water. The very first dams, constructed in ancient Mesopotamia, were used for many of the same purposes as they are today: water irrigation and flood control (though, a few centuries later, we now also use them to generate hydropower). They drastically disturb natural ecosystems, negatively impact water quality, and exacerbate soil erosion—oh, and they've also altered the location of the Earth's poles. In a new study published in the journal Geophysical Research Letters, a team of scientists at Harvard University details their discovery that the rampant construction of dams from 1835 to 2011 has locked up so much water that the Earth's poles have moved slightly from their axial rotation. Because Earth's outermost layer rests upon another layer of liquid rock, a redistribution of mass on the surface—whether accomplished by glaciers during an ice age or by locking up water behind dams—it can influence the orientation of the Earth's poles. Thankfully, Earth is not destined to end up like the planet Uranus, forever spinning on its side. Two centuries of frantic dam-building (about 7,000 of them in total) has only nudged the poles about three feet and caused a 0.83-inch drop in global sea levels. “As we trap water behind dams, not only does it remove water from the oceans, thus leading to a global sea level fall, it also distributes mass in a different way around the world,” Harvard University graduate student Natasha Valencic, the lead author of the study, said in a press statement. “We're not going to drop into a new ice age, because the pole moved by about a meter [or about three feet] in total, but it does have implications for sea level.” Between 1835 and 1954, a majority of dams were constructed in North America and Europe, moving the North Pole 8 inches to the 103rd meridian east. In 2005, NASA scientist Benjamin Fong Chao showed that when full, this mega-dam alone slowed the rotation of the Earth by 0.06 microseconds, or 60 billionths of a second. Humanity continues to impact our planet in a myriad of complex ways. Hopefully, the more we learn about these impacts, the more we can mitigate any negative impacts they may bring along with them. Past CO2 Rise Can't Even Compare to Climate Change Are There More Humans on Earth Than We Thought?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250714052105.htm'>This AI-powered lab runs itself—and discovers new materials 10x faster</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 12:25:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have demonstrated a new technique that allows "self-driving laboratories" to collect at least 10 times more data than previous techniques at record speed. The advance - which is published in Nature Chemical Engineering - dramatically expedites materials discovery research, while slashing costs and environmental impact. Self-driving laboratories are robotic platforms that combine machine learning and automation with chemical and materials sciences to discover materials more quickly. "Imagine if scientists could discover breakthrough materials for clean energy, new electronics, or sustainable chemicals in days instead of years, using just a fraction of the materials and generating far less waste than the status quo," says Milad Abolhasani, corresponding author of a paper on the work and ALCOA Professor of Chemical and Biomolecular Engineering at North Carolina State University. "This established approach to self-driving labs has had a dramatic impact on materials discovery," Abolhasani says. "It allows us to identify promising material candidates for specific applications in a few months or weeks, rather than years, while reducing both costs and the environmental impact of the work. That means the system sits idle while the reactions take place, which can take up to an hour per experiment. "We've now created a self-driving lab that makes use of dynamic flow experiments, where chemical mixtures are continuously varied through the system and are monitored in real time," Abolhasani says. "In other words, rather than running separate samples through the system and testing them one at a time after reaching steady-state, we've created a system that essentially never stops running. "This streaming-data approach allows the self-driving lab's machine-learning brain to make smarter, faster decisions, honing in on optimal materials and processes in a fraction of the time. "This breakthrough isn't just about speed," Abolhasani says. "By reducing the number of experiments needed, the system dramatically cuts down on chemical use and waste, advancing more sustainable research practices. "The future of materials discovery is not just about how fast we can go, it's also about how responsibly we get there," Abolhasani says. "Our approach means fewer chemicals, less waste, and faster solutions for society's toughest challenges." Co-lead authors of the paper are Fernando Delgado-Licona, a Ph.D. student at NC State; Abdulrahman Alsaiari, a master's student at NC State; and Hannah Dickerson, a former undergraduate at NC State. The paper was co-authored by Philip Klem, an undergraduate at NC State; Arup Ghorai, a former postdoctoral researcher at NC State; Richard Canty and Jeffrey Bennett, current postdoctoral researchers at NC State; Pragyan Jha, Nikolai Mukhin, Junbin Li and Sina Sadeghi, Ph.D. students at NC State; Fazel Bateni, a former Ph.D. student at NC State; and Enrique A. López-Guajardo of Tecnologico de Monterrey. This work was done with support from the National Science Foundation under grants 1940959, 2315996 and 2420490; and from the University of North Carolina Research Opportunities Initiative program. Note: Content may be edited for style and length. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02199-1'>China's shift towards ‘organized research': how can coordination and innovation co-exist?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 12:07:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). For example, the country still struggles to deliver groundbreaking discoveries and convert its research into commercial successes. The commercialization rate of Chinese university patents has remained below 5%. Several Chinese companies also continue to rely on imported technologies, particularly from the United States. This innovation bottleneck is compounded by global trends: the COVID-19 pandemic, rising protectionism in trade and escalating US–China technological tensions, which have disrupted international cooperation and deepened the need for domestic technological independence. China can build on advances it has made since 2022, when it shifted its science policy towards an ‘organized research' model — emphasizing mission-driven science, nationwide resource mobilization and cross-institutional collaboration. By forming task-focused teams and tackling urgent challenges at the frontiers of science and economic needs, the policy seeks to drive scientific breakthroughs in areas that are crucial to the country's strategic priorities, such as artificial intelligence (AI), renewable energy, biomedicine and fundamental science. Here, we outline the shifts in policy, explore the challenges and suggest ways to find a balance between pursuing collective national goals and fostering individual scientific creativity. Modern science is deeply influenced by social, political and institutional factors. At the national level, more than 300 state key laboratories (SKLs) — the country's top research centres — have been restructured since 2022 to promote cross-institutional collaboration. For example, the SKL for Photovoltaic Science and Technology was restructured around a partnership between photovoltaics company Trina Solar in Changzhou and Fudan University in Shanghai. At the institutional level, universities are building new research platforms. For example, last year, China Pharmaceutical University in Nanjing launched its Intelligent Pharmacy Institute, bringing together AI and drug discovery under a unified, centralized framework. By consolidating funding, personnel and interdisciplinary expertise, the institute seeks to streamline AI-driven pharmaceutical research. It also collaborates closely with leading pharmaceutical companies through initiatives such as the Intelligent Pharmacy Education–Industry Alliance to address real-world industrial challenges. Big science projects, which need massive resources, go beyond the capacity of a single institution or small research team. In 2024, to better channel resources and align research with pressing national priorities, China shifted the responsibility for key research and development programmes from the Ministry of Science and Technology to sector-specific government agencies, such as the Ministry of Water Resources. To boost China's participation in international big-science initiatives, the latest key project for Intergovernmental International Cooperation on Science and Technology Innovation was launched in May. By contrast, individual researchers are facing tighter competition and fewer resources. The General Programs of the National Natural Science Foundation of China (NSFC) — a major funding source for individual scientists — saw average grant sizes decline by 8.1% between 2022 and 2024, while applications surged by 53%, signalling increasingly limited funding opportunities for independent research3,4. For instance, the updated Interim Measures for the Administration of the National Key R&D Program, issued last year, places greater emphasis on research outcomes than it does on academic publications, with a particular focus on how breakthroughs can translate into practical applications. Researchers in China are now encouraged to embody the ideal of Ding Tian Li Di scientists. Although success will surely follow, we identify three challenges faced by the organized-research policy that might prevent China from achieving all its desired goals. First, Chinese universities remain organized mainly around conventional academic disciplines, with government resources allocated to specific departments. However, this structure is often ill-suited to addressing complex national priorities — such as achieving carbon neutrality — that demand coordinated efforts across disciplines, such as energy systems, chemistry, environmental science and management. In response, research units and interdisciplinary platforms have begun to emerge. Yet, these often remain entangled with existing discipline-based structures. For example, the Institute of Carbon Neutrality at Peking University in Beijing, established in 2023, significantly overlaps with conventional departments: 113 of its researchers hold dual appointments with their original academic institutions. This dual affiliation creates competition for faculty members' time and research funding, limiting the institute's ability to function as an independent driver of interdisciplinary research. Faculty members must still declare a disciplinary affiliation when applying for grants and promotions, and their work is typically evaluated by their peers from the same field. As a result, there is little institutional incentive to engage in cross-disciplinary or cross-unit collaboration. Second, the way in which the Chinese government is organizing research and allocating resources might draw scientists into a narrow set of fields, which could discourage grassroots innovation and limit academic freedom. One indicator of this is the shrinking share of General Programs in the NSFC budget — these programmes, known for their broad scope and investigator-driven nature, accounted for only 31.5% of total funding in 2023, down from 51.0% in 2013. Chinese universities rely on strategic collaborations with industry to expand battery research.Credit: Qilai Shen/Bloomberg/Getty Although the mission-oriented model can mobilize national capacity for targeted goals, it risks producing scientific uniformity and undermining small-scale research efforts, which some studies suggest are more productive5 and more likely to yield original or disruptive discoveries6. Another risk lies in the volatility of government priorities. When state support is withdrawn, entire research areas can stagnate. A historical example is the termination of the US Superconducting Super Collider (SSC) project in 1993 owing to budget overruns, management challenges and waning political support. The SSC's cancellation serves as a cautionary tale of how shifts in political will can derail long-term scientific efforts, regardless of their potential. Here, we propose a few suggestions to mitigate these challenges. Enhancing the Effectiveness of Team Science (National Academies Press, 2015). China made waves with Deepseek, but its real ambition is AI-driven industrial innovation Plastics pollution is surging — the planned UN treaty to curb it must be ambitious Survey of US postdocs finds threefold increase in job losses Deaf scientists hit by drastic NIH cuts — the research community must support them What we stand to lose when foreign students are seen as a threat Research-integrity sleuths say their work is being ‘twisted' to undermine science Rubbish years: how to boost lab group morale when world events crash in AI ‘scientists' joined these research teams: here's what happened China made waves with Deepseek, but its real ambition is AI-driven industrial innovation An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/what-new-biological-age-clocks-say-about-longevity-according-to-eric-topol/'>A Longevity Expert Breaks Down the Science and Hype of Biological Aging Tests</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A Longevity Expert Breaks Down the Science and Hype of Biological Aging Tests Super Agers author Eric Topol unpacks the rise of biological age tests—from organ clocks to immune system clocks—and how they might revolutionize early diagnosis of disease Counting birthdays may be a common tally, but your “age” isn't determined by time alone. New research increasingly shows the importance of considering chronological age as something very different from biological age—in which the body and its cells, tissues and organs all have separate “clocks” that can tick at different speeds. “Calculating biological age, I think, is core to the advances we've made in the science of aging,” says Eric Topol, a cardiologist and genomics professor at Scripps Research in California. “It's a way you can tell if a person, organ, or any biological unit is at pace of aging—if it's normal, abnormal or supernormal.” In his new book Super Agers: An Evidence-Based Approach to Longevity, Topol delves into the recent surge in public interest in biological aging and the accelerating quest to refine ways to measure it—giving a more precise picture of a person's longevity prospects and of potential ailments that can be prevented or treated early. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The real beginning of this research started more than a decade ago by geneticist Steven Horvath with his “clock” [test], with which, basically using saliva, you could look at specific genetic markers in a genome and predict a person's biological age. I kind of liken it to the body rusting out. Basically, you're getting marks at specific parts of the genome that track with aging in humans and every other species of mammal. In Horvath's initial test, there clearly was a detection of both alignment with the person's real age, or chronological age—and when it wasn't matching up. In other words, if a person's biological age was off by a few years from their real age, you'd wonder why that is. Using plasma proteins from a blood sample, we can also clock organs—whether it's the heart, brain, liver or kidney. So we have seen just enormous advances in these clocks, and they keep getting refined with added features. What can biological age tests tell us clinically? We can detect in an individual if something's not right at different levels. For example, if your biological age is five years older than your real age, is there an organ that might be linked with that? Then you can use these clocks to see if lifestyle, prevention or treatment can slow down the pace of aging and get it into alignment with your actual age. The question is: When will doctors actually start using them? The medical community is very hard to change. Tests are also made available by commercial companies, but they can be very expensive. You can run an epigenetic test in a very simple way for $10 or $20, while some of these companies are charging $200. I haven't seen their publications to be able to say with confidence that they are doing things right, and the lack of standards from one company to the next is disconcerting. Eventually, I believe, we're going to have high-fidelity epigenetic clocks with no motivation for a provider to hold things back if a person's data are really bad. If you had to pick one mechanism behind why biological age and chronological age are misaligned, it would most likely be because there are some genes that are either protective or linked with accelerated aging—but that's such a small part of the story. Another root cause appears to be that our immune system gets weaker and less functional as we get older. In the average person, this starts around age 55 to 60. We can't get everybody to do all these things that we know help them, but if they get their own data and they see something's off track, the hope is that they'd [change their lifestyle]. What environmental factors are also important to consider? And we have forever chemicals as well that are pervasive. These all play a factor in lifestyle, health and aging. Let's talk more about “inflammaging.” We know some inflammation can be good for the body to fight infections, for instance, but a lot can be bad. But what's so exciting is we can dial up or down the immune system now. For example, [there have been] natural, amazing experiments with the shingles vaccines, which reduce dementia and Alzheimer's disease 20 to 25 percent. That's going to be the critical thing in using these metrics: zooming in on the immune system and inflammation to keep people's immune system intact and stop it when it starts to go haywire. Maybe we don't have to do a systematic, comprehensive assessment of our immunome [that would include checking antibody titers and] sequencing B cells, T cells and interferons. If we can use just a group of plasma proteins, that would be terrific. There's a human immunome project just getting started to try to compare things such as the proteins with the much more sophisticated and expensive ways to get at the health of an immune system. What are the downsides of slowing down biological aging, or extending lifespan? “Super agers” who don't get one of the big four age-related diseases [type 2 diabetes, cancer, or heart or neurodegenerative disease] say “Well, I did it.” Of course, if you get up to age 98, you're really doing well. I think we're going to have a whole lot more super agers. But that's not going to get around the fact that eventually they're going to develop some problems—one of the big four or other conditions. It could be you get an infection because your immune system is just too weak. Or it could be you break your hip because your bone density is so low and you wind up with a pulmonary embolus [a clot that blocks blood flow to the lungs]. I don't want to put a sense out there that super agers won't see problems in the latter stages of their lives. Young is an associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02198-2'>AI hype, crackling northern lights and more: take it all in with these holiday reads</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 10:56:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Leor Zmigrod is a political neuroscientist based in London, and author of The Ideological Brain. Nyalleng Moorosi is a researcher at the Distributed AI Research Institute based in Hlotse, Lesotho. Farhana Sultana is a geographer at Syracuse University, New York. Anna Abalkina is a researcher at the Free University of Berlin. Candice L. Odgers is a psychologist at the University of California, Irvine. Renata Libonati is a fire scientist at the Federal University of Rio de Janeiro, Brazil. Àlex Gómez-Marín is a theoretical physicist and neuroscientist at the Institute of Neuroscience (CSIC-UMH) in Alicante, Spain. Monica McLemore is a researcher in reproductive health, rights and justice at the New York University Rory Meyers College of Nursing, New York City. Genevieve L. Wojcik is an epidemiologist at the Johns Hopkins Bloomberg School of Public Health, Baltimore, Maryland. Even plants can respond to the sound vibrations that nearby animals produce. The buzz of a bee's wings, for example, can induce nearby suncup flowers to make nectar that has a higher concentration of sugar, to attract pollinators. In its scope and passion, this book appeals to readers' most romantic instincts — reminding us of our entanglement with the cosmos and the organisms that live in it. “Do you believe in hope after hype?” ask linguist Emily Bender and sociologist Alex Hanna in their sharp, witty book, The AI Con, after taking us through the hellscape that is artificial intelligence (AI) hype. With a humorous but critical eye, they run through some of the largest AI claims made in the past few years — and how some collapsed on closer inspection. AI models to predict sepsis, for example, have shown low performance and high false-positive rates. Imparting too much trust in machine-learning models has harmed the health, education and law sectors. Simple questions can reveal flaws in the tools' promises: who benefits from AI adoption? What nuance is lost when perusing AI-compiled summaries instead of whole texts? Yet, AI technologies can be powerful for their intended purposes. With careful measurement, evaluation and a focus on capabilities over profit, they can automate mundane, repetitive tasks. Long before today's global discussions on climate change or gender equity, educator and political activist Rokeya Sakhawat Hossain envisioned a future in which science served justice and sustainability. In Sultana's Dream (1905), writing from the colonial Bengal Presidency, she evokes ‘Ladyland' — a feminist utopia in which men take on domestic work while women run society, centring innovation around care, community and ecological balance. Her provocative vision is remarkable considering the structural exclusions she faced. The solar kitchens, electric transport, geoengineering balloons (to harvest water, as a resource and to control rain) and zero-emission agriculture she describes seem fantastical — but they are part of a long-overlooked lineage of feminist techno-visions. As a Bangladeshi scholar and feminist scientist, I find this manifesto against the extractivist science and capitalist consumption that are ruining the planet discerning. Sultana's Dream deserves renewed attention as a creative, radically prescient blueprint from the margins. Pharmacologist Csaba Szabo is alarmed by the reproducibility crisis in biomedical science. In Unreliable, Szabo guides readers through the issues that have led to this, including methodological flaws, structural problems in science and the hyper-competitive environment for grants. The author pays special attention to scientific fraud, both by individuals and in the form of mass-produced fraudulent research churned out by paper mills. Dodgy papers are too often included in systematic reviews and meta-analyses, which could lead to people receiving ineffective treatments. He argues for replacing the for-profit publishing industry with data repositories and online information-storage systems, and for transforming systematic reviews, which need post-publication review to ensure lasting quality. Renowned psychologist and former teacher David Yeager has spent more than a decade identifying adults who excel at helping young people to exceed expectations. These mentors share a common approach, he observes in 10 to 25. Whether they are training surgeons, perfecting elite athletes' basketball shots or teaching calculus to secondary-school students, they do not just tell young people what to do. Yeager points out that these exceptional adults stand apart from many of us, who, despite our best intentions, get it wrong. ‘Just say no' campaigns are a classic example, often backfiring because of failed assumptions about youth and an over-reliance on authority. Through a skilful synthesis of the science, Yeager frames adolescence not as a problem to be solved, but as something to invest in. With a frightening account of a devastating wildfire that ravaged Fort McMurray, an oil town in Canada, in 2016, writer John Vaillant gives us a glimpse of what fire events will be like in our warming planet. This gripping tale is reminiscent of a disaster movie in which scientists' warnings are ignored until it's too late. Vaillant's depiction of the fire, which forced more than 90,000 people to evacuate and destroyed thousands of homes, seems apocalyptic — but every detail is true. His focus on geology, meteorology and ecology highlights a warning: fire, once a natural part of the Earth system, has become something new. It's about what the future could look like if we fail to act. Four Greek deities meet in a blooming garden to debate the nature of reality. By presenting these ideas in the form of a 500-page Platonic dialogue, Hart can examine objections to his thesis through the voice of Hephaistos. All Things are Full of Gods is an intellectual blessing in an age in which it can feel like algorithms threaten our souls. Political scientist Grace Howard's gripping book, The Pregnancy Police, is a must-read for anyone attempting to make sense of our current environment — one in which Adriana Smith, a Black mother who was legally dead, was kept alive for four months to gestate a nine-week-old fetus to comply with Georgia's anti-abortion law, without her family being consulted. Could crabs be conscious, can you beat hypochondria and more: top reads for summer Sci-fi icon Kim Stanley Robinson: ‘there's so much bad fiction about anthropomorphizing AI' Science on the catwalk: Iris van Herpen gives gowns a glow up How DEI is misunderstood — and the real route to gender equality Research-integrity sleuths say their work is being ‘twisted' to undermine science Deaf scientists hit by drastic NIH cuts — the research community must support them Could crabs be conscious, can you beat hypochondria and more: top reads for summer Sci-fi icon Kim Stanley Robinson: ‘there's so much bad fiction about anthropomorphizing AI' Science on the catwalk: Iris van Herpen gives gowns a glow up How DEI is misunderstood — and the real route to gender equality An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/07/250713031451.htm'>This Algorithm Just Solved One of Physics' Most Infamous Problems</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-14 06:47:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Caltech scientists have found a fast and efficient way to add up large numbers of Feynman diagrams, the simple drawings physicists use to represent particle interactions. In the 1940s, physicist Richard Feynman first proposed a way to represent the various interactions that take place between electrons, photons, and other fundamental particles using 2D drawings that involve straight and wavy lines intersecting at vertices. Though they look simple, these Feynman diagrams allow scientists to calculate the probability that a particular collision, or scattering, will take place between particles. Therefore, by summing all the possible diagrams, scientists can arrive at quantitative values related to particular interactions and scattering probabilities. In a paper published in Nature Physics, the Caltech team uses its new method to precisely compute the strength of electron-phonon interactions and to predict associated effects quantitatively. For such materials, scientists can use a method called perturbation theory to describe the interactions that occur between electrons and phonons, which can be thought of as "units" of atomic vibration. Perturbation theory is a good approximation in these systems because each successive order or interaction becomes decreasingly important. But for many other materials, electrons interact much more strongly with the atomic lattice, forming entangled electron-phonon states known as polarons. Polarons are electrons accompanied by the lattice distortion they induce. For example, an electron placed in a material with ionic bonds will distort the surrounding lattice and form a localized polaron state, resulting in decreased mobility due to the strong electron-phonon interaction. Scientists can study these polaron states by measuring how conductive the electrons are or how they distort the atomic lattice around them. Perturbation theory does not work for these materials because each successive order is more important than the last. "It's basically a nightmare in terms of scaling," says Bernardi. Thus far such calculations have been dominated by methods where scientists can tune certain parameters to match an experiment. Instead, his group focuses on solving problems from "first principles," meaning beginning with nothing more than the positions of atoms within a material and using the equations of quantum mechanics. When thinking about the scope of this problem, Luo says to imagine trying to predict how the stock market might behave tomorrow. "The only thing we can do is use a smart way of sampling all these scattering processes." Caltech researchers are addressing this problem by applying a technique called diagrammatic Monte Carlo (DMC), in which an algorithm randomly samples spots within the space of all Feynman diagrams for a system, but with some guidance in terms of the most important places to sample. "We set up some rules to move effectively, with high agility, within the space of Feynman diagrams," explains Bernardi. The Caltech team overcame the enormous amount of computing that would have normally been required to use DMC to study real materials with first principle methods by relying on a technique they reported last year that compresses the matrices that represent electron-phonon interactions. Another major advance is nearly removing the so-called "sign problem" in electron-phonon DMC using a clever technique that views diagrams as products of tensors, mathematical objects expressed as multi-dimensional matrices. In the new paper, the researchers have applied DMC calculations in diverse systems that contain polarons, including lithium fluoride, titanium dioxide, and strontium titanate. The scientists say their work opens up a wide range of predictions that are relevant to experiments that people are conducting on both conventional and quantum materials -- including electrical transport, spectroscopy, superconductivity, and other properties in materials that have strong electron-phonon coupling. "We have successfully described polarons in materials using DMC, but the method we developed could also help study strong interactions between light and matter, or even provide the blueprint to efficiently add up Feynman diagrams in entirely different physical theories," says Bernardi. The paper is titled, "First principles diagrammatic Monte Carlo for electron-phonon interactions and polaron." Luo was partially funded by an Eddleman Graduate Fellowship. NASA Just Flew Through the Sun's Atmosphere – And What It Saw Is Jaw-Dropping AI Designs New Material To Cool Your Home and Slash Energy Bills Killer Whales Are Giving Fish to Humans Worldwide – What's Going on? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            