
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-27</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a69097435/eye-implant-ar-glasses/'>Scientists May Have Found the Incredible Way to Reverse Blindness</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The human eye is an indisputable marvel of evolution. But when things go wrong in the eye, all of that sensitive biological engineering, all backed with a 24mm-diameter sphere, can be hard to repair. One of the most pernicious conditions that afflicts around five million people worldwide is “geographic atrophy,” or GA, an advanced form of dry age-related macular degeneration that is one of the leading causes of blindness in people over 50. This eventually impacts a person's ability to read, drive, or even recognize faces. In some cases it can even lead to total blindness. Today there is no known cure for this vision affliction, but a new study led by an international team of scientists has made some serious progress on that front. “In the history of artificial vision, this represents a new era. Blind patients are actually able to have meaningful central vision restoration, which has never been done before,” UCL's Mahi Muqit, a co-author of the study, said in a press statement. This system also comes with a zoom feature so that patients can learn to read effectively after training for several months. Now they can read up to five lines on it. “The rehabilitation process is key to these devices. “My feeling is that the door is open for medical devices in this area, because there is no treatment currently licensed for dry AMD…I think it's something that, in future, could be used to treat multiple eye conditions.” Eyes may be an evolutionary wonder, but so too is our scientific ability to understand them and hopefully one day fix them when things go wrong. Scientists Found an Ancient Figurine With No Face We May Have Been Wrong About the Origin of Life Will We Really Reach Singularity in 3 Months? This New Shot Could Permanently Cure Balding Scientists Found a Weird Way to Treat Hair Loss</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69164694/roman-army-camp-alps-hidden/'>Archaeologists Found an Ancient Roman Military Camp Hiding 7,000 Feet High in the Sky</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 13:55:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The ancient Roman army didn't seem to be afraid of anything, but they definitely weren't afraid of heights, as evidenced by the discovery of a 2,000-year-old Roman military camp located 7,000 feet above sea level in the Swiss Alps. As archaeologists continue to explore the history of the Roman army in what is now Switzerland, a volunteer unearthed a previously unknown military camp in the mountains, strategically positioned to have tactical views of the surrounding valleys and mountain passes, according to a translated statement from the Canton of Graubünden. Resting undisturbed for two millenia 7,000 feet high in the Swiss Alps, the previously unknown Roman military camp was fortified by three ditches and a wall with ramparts. The site of the camp offers a view of four key valleys—Landwassertal, Albulatal, Domleschg, and Surses—and Lenzerheide, a heavily traveled mountain passageway. The perch gave Roman soldiers stationed at the camp a perfect vantage point to spot any incoming enemies. So far, the find has yielded weapons and equipment belonging to Roman soldiers, including lead sling bullets and boot nails. The statement said that the “sensational discovery of a Roman military camp in Graubünden shows once again that archaeological research into ‘Roman Switzerland' continues to hold great surprises.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Scientists Found an Ancient Figurine With No Face We May Have Been Wrong About the Origin of Life Will We Really Reach Singularity in 3 Months? Geologists Found the World's Largest Gold Deposit This New Shot Could Permanently Cure Balding Scientists Found a Weird Way to Treat Hair Loss Why Did the Legendary Edmund Fitzgerald Sink?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64486-9'>Super mitochondria-enriched extracellular vesicles enable enhanced mitochondria transfer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 12:53:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Mitochondria transfer is a spontaneous process that releases functional mitochondria to damaged cells via different mechanisms including extracellular vesicle containing mitochondria (EV-Mito) to restore mitochondrial functions. However, the limited EV-Mito yield makes it challenging to supply a sufficient quantity of functional mitochondria to damaged cells, hindering their application in mitochondrial diseases. Here, we show that the release of EV-Mito from mesenchymal stem cells (MSCs) is regulated by a calcium-dependent mechanism involving CD38 and IP3R signaling (CD38/IP3R/Ca2+ pathway). Activating this pathway through our non-viral gene engineering approach generates super donor MSCs which produce Super-EV-Mito with a threefold increase in yield compared to Ctrl-EV-Mito from normal MSCs. Leber's hereditary optic neuropathy (LHON), a classic mitochondrial disease caused by mtDNA mutations, is used as a proof-of-concept model. Super-EV-Mito rescues mtDNA defects and alleviates LHON-associated symptoms in LHON male mice. This strategy offers a promising avenue for enhancing mitochondria transfer efficiency and advancing its clinical application in mitochondrial disorders. Mitochondrial dysfunction is the source and key hallmark of mitochondrial disorders, with a minimum prevalence of 1 in 5000 adults1. However, effective therapeutic strategies are currently lacking for many mitochondrial diseases in clinical practice2,3. Notably, Leber's hereditary optic neuropathy (LHON) refers to a classical mitochondrial disorder caused by mutations in mitochondrial DNA (mtDNA) which encodes key proteins related to mitochondrial function4,5. Although some antioxidant drugs and nutritional support can assist in the treatment of LHON, there is currently no effective therapeutic approach targeting the mitochondrial dysfunction underlying LHON6. Mitochondria transplantation is an effective approach for restoring mitochondrial functions in patients with mitochondrial diseases because the dysfunctional mitochondria are replenished by exogenous functional mitochondria7,8,9. This approach involves directly transplanting isolated free mitochondria into the lesion site. However, studies on mitochondria transplantation are still in their relatively early stages. Since the first reported application of mitochondria transplantation in treating mitochondrial disorders in 2009, only two clinical studies have been published (ClinicalTrials.gov: NCT02586298 and NCT04976140). Therefore, developing an effective method to deliver functional mitochondria specifically into damaged cells is a critical yet challenging objective. Inspired by the natural intercellular transfer of mitochondria, healthy cells can deliver mitochondria to injured cells via extracellular vesicles (EVs)12, intercellular nanotube connections13, and other mechanisms in an attempt to repair mitochondrial dysfunction14. Accumulating evidence indicated that the paracrine effect of stem cells is a key mechanism underlying their ability to repair damaged cells15,16. This is primarily because stem cells transfer healthy mitochondria to injured recipient cells via extracellular vesicles containing mitochondria (EV-Mito), thereby compensating for the functional impairment of damaged mitochondria17. In this phenomenon, EV-Mito not only protects mitochondrial activity through the membrane structure of EVs but also selectively recognizes damaged recipient cells for functional mitochondrial delivery18,19,20,21. This approach appears to address the main challenges of mitochondria transplantation technology: the lack of selectivity and the rapid deactivation of isolated mitochondria. Consequently, some studies are now focusing on collecting EV-Mito for the treatment of mitochondrial diseases. However, these studies have uncovered a challenge: the limited mitochondrial mass of stem cells as donor cells restricts the mitochondrial content of EV-Mito22,23. Addressing how to create super donor cells capable of generating more high-quality EV-Mito would expand this platform's scalability and enhance its feasibility for treating mitochondrial diseases. In this study, we demonstrate that the release of EV-Mito from mesenchymal stem cells (MSCs) is regulated by a calcium-dependent mechanism involving CD38 and inositol trisphosphate receptor (IP3R) signaling (CD38/IP3R/Ca2+ pathway). Furthermore, we develop our non-viral gene delivery vectors called CAP (C: N,N-cystamine-bis-acrylamide, A: agmatine dihydrochloride, P: p-aminoethylbenzenesulfonic acid) to load a CD38 plasmid, forming CAP/pCD38 complexes24,25. These CAP/pCD38 complexes are used to genetically engineer adipose-derived MSCs, thereby activating the CD38/IP3R/Ca2+ pathway in MSCs. Genetically engineered MSCs using CAP/pCD38 complexes release extracellular mitochondrial particles at 3 times the level of untreated MSCs, indicating that this approach produces super donor MSCs for more EV-Mito. To validate the therapeutic efficacy of super donor MSC-derived EV-Mito (Super-EV-Mito) in mitochondrial diseases, we introduce LHON cell and LHON-like male mouse models. We find that Super-EV-Mito significantly enhances the cellular uptake of exogenous mitochondria, alleviating LHON disease symptoms. Meanwhile, Super-EV-Mito demonstrates mitochondrial function restoration in both rotenone (Rot)-induced widespread mitochondrial damage models at the cellular and mouse levels, further expanding their potential applications in other mitochondrial diseases. As mitochondria serve as the primary therapeutic agent in mitochondria transplantation, their activity and quantity inevitably influence the efficiency of mitochondria transplantation. Our study suggests that genetically engineered stem cells can generate super-donor cells, thereby providing a greater quantity and higher quality mitochondrial material for mitochondria transplantation. Mitochondria transplantation is an effective treatment for mitochondrial disorders; however, it is restricted by the limited selectivity in recipient cells and loss of mitochondrial activity in isolated mitochondria. Fortunately, mitochondria transfer as a spontaneous process selectively delivers functional mitochondria to damaged cells, thereby overcoming the aforementioned limitations26. Meanwhile, accumulating evidences suggest that the paracrine effects of stem cells, including the extracellular release of mitochondrial particles, constitute a key mechanism underlying their ability to repair damaged cells15. We attempted to investigate the pathways regulating the secretion of EV-Mito by MSCs and explored whether enhancing these pathways could increase the yield of EV-Mito. Previous studies have identified that the release of extracellular mitochondrial particles by donor cells, such as astrocytes and osteoblasts, is mediated by CD38 signaling29,30. Therefore, we aim to elucidate how CD38-mediated pathways influence the release of EV-Mito. We employed the exogenous supplementation of CD38 plasmid DNA (pCD38) to upregulate CD38 expression in MSCs and observe changes in the genetically engineered MSCs. To construct genetically engineered MSCs, we introduced our previously developed polymeric material, named CAP [C: N,N'-cystamine-bis-acrylamide (CBA), A: agmatine dihydrochloride (Agm), P: 4-(2-aminoethyl) benzenesulfonamide (PAEBS)]24. CAP was synthesized following the procedure described in a previous study (Supplementary Fig. 1) and verified using 1H NMR spectroscopy (Supplementary Fig. Subsequently, the pDNA binding capacity of CAP was detected by agarose gel electrophoresis. The results demonstrated that CAP could completely bind pDNA at the weight ratio of 5:1 (Supplementary Fig. Additionally, nanoparticle sizes and zeta potentials of the CAP/pDNA complex were approximately 102.3 ± 1.8 nm and 20.9 ± 0.9 mV, respectively (Supplementary Fig. CAP contains disulfide bonds within its linkers, allowing degradation in the intracellular reductive environment of MSCs and the rapid gene release. To simulate the intracellular reductive conditions and investigate the degradation of CAP, we used dithiothreitol (DTT) to mimic the intracellular reductive environment of MSCs. The pDNA was released after 10 mM DTT treatment (2 h), indicating that the vector material had degraded for gene release (Supplementary Fig. The transfection efficiency of CAP/pGFP complexes was significantly superior to that of PEI 25 K/pGFP complexes and Lipofectamine 3000 Transfection Reagent (Lipo3000)/pGFP complexes, where PEI 25 K and Lipo3000 are widely considered the gold standards for plasmid gene transfection (Supplementary Fig. One of the purposes of gene delivery of stem cells is to engineer stem cells to have certain functions or secrete certain proteins to act on target sites33. It can be found that CAP/pDNA complexes had no obvious toxicity (Supplementary Fig. These results indicated that CAP offered both higher transfection efficiency and better safety compared to Lipo3000, making it more suitable for engineering MSCs in the subsequent research applications. We obtained genetically engineered MSCs and analyzed them using gene ontology (GO) analysis. 1A, these results between normal MSCs and CD38-enhanced MSCs indicated that CD38 upregulation affected calcium-dependent pathways, accompanied by alterations in the extracellular region, extracellular space, positive regulation of cytosolic Ca2+ concentration and positive regulation of mitochondrial fission. Therefore, we hypothesized that the mitochondrial functions and the release of EV-Mito from MSCs may be regulated by a mechanism involving CD38 and Ca2+ up-regulation. A GO analysis based on the significant differentially expressed genes between untreated and CAP/pCD38-treated MSCs. B Gene expression of pCD38 in MSCs after different treatments. C Expression of IP3R in MSCs after different treatments, red: IP3R protein, blue: DAPI. E Normalized fluorescence intensity of Mitotracker green (MTG) of extracellular mitochondria in a condition medium. J Oxygen consumption rate (OCR) from Seahorse cell mito stress test of MSCs after pCD38 transfection. K Maximum respiration of MSCs after pCD38 transfection. M A schematic diagram illustrating the release of EV-Mito from MSCs regulated by the CD38/IP3R/Ca2+ pathway. Data are presented as mean ± SD. n = 3 biologically independent samples for each in (B, D–K), n = 5 biologically independent samples for each in (L). GO enrichment analysis was performed via two-sided hypergeometric test (A). Statistical significances were performed via one-way ANOVA with two-sided Tukey's HSD post hoc test (B, E–I, K, L) or two-sided Games-Howell test (D). We confirmed the expression of CD38 based on CAP. Lipo3000 was set as the positive control group. CAP-mediated plasmid gene delivery obviously promoted the expression of CD38 in MSCs compared with the untreated and Lipo3000 groups, suggesting that CAP enabled the upregulation of CD38 expression in MSCs (Fig. In addition, CD38 catalyzes the synthesis of the calcium-mobilizing second messenger cyclic ADP-ribose (cADPR)29, resulting in a significant elevation of cytoplasmic calcium levels (Supplementary Fig. However, it is currently unclear whether CD38 regulates mitochondrial Ca2+ to control the release of EV-Mito in MSCs. Based on the fact that IP3R is the receptor responsible for calcium transfer from the endoplasmic reticulum to mitochondria and that most actively secreted cellular events involve calcium regulation, we examined IP3R/mitochondrial Ca²⁺ signaling as a possible mechanism driving extracellular mitochondria release from MSCs36. Immunofluorescence and Western blot assays demonstrated that CAP-mediated upregulation of CD38 enhanced IP3R expression in MSCs (Fig. Subsequently, CAP/pDNA enhanced mitochondrial Ca²⁺ level compared to the other groups (Fig. Furthermore, Mitotracker Green (MTG) was carried out to stain mitochondria of MSCs. After untreated, Lipo3000/pCD38, or CAP/pCD38 complex treatment, the normalized fluorescence intensity (FI) of MTG-labeled EV-Mito in condition medium was detected using the multifunctional microplate reader. 1E, CAP exhibited a higher normalized FI of MTG compared with the untreated and Lipo3000, revealing that the upregulation of CD38 led to an increased release of mitochondria in extracellular spaces by MSCs. These findings suggested that the upregulation of CD38 may promote IP3R expression and mitochondrial Ca²⁺ level in MSCs, enhancing an increased release of EV-Mito in the extracellular space. Furthermore, we identified the IP3R siRNA to suppress IP3R expression. However, IP3R knockdown (KD) did not affect CD38 expression (Fig. Notably, IP3R KD significantly inhibited CD38-mediated IP3R activation (Fig. These findings suggested that CD38 upregulation enhanced IP3R expression and the inhibition of IP3R did not exert a negative feedback effect on CD38 expression. In addition, the upregulation of CD38 increased mitochondrial Ca²⁺ concentration, while blocking IP3R expression significantly inhibited the CD38-induced rise in mitochondrial Ca²⁺ levels (Fig. Additionally, IP3R knockdown also reduced the release of mitochondria into the extracellular space by MSCs (Fig. Previous studies have shown that elevated intracellular Ca²⁺ promotes vesicle-plasma membrane fusion, facilitating the release of EVs and their cargo37,38. Pharmacological inhibition studies further demonstrated that the release of EV-Mito was markedly suppressed by actin polymerization inhibitors, whereas tubulin inhibitors had no obvious effect (Supplementary Fig. 8C, D), suggesting that EV-Mito release is actin-dependent but independent of tubulin dynamics39. This mechanistic insight highlighted a calcium-sensitive, actin-dependent vesicle trafficking pathway as a key contributor to EV-Mito secretion. To investigate whether the upregulation of CD38 gene expression mediated by the CAP/pCD38 complex negatively impacts mitochondrial function in MSCs, we assessed mitochondrial respiratory function in MSCs using the oxygen consumption rate (OCR) analysis. 1J, K), demonstrating that the upregulation of CD38 expression not only did not impair the mitochondrial respiratory chain function in MSCs but may actually enhance it. This could be due to the upregulation of CD38 activating processes such as cellular metabolic process (Fig. 1A), which in turn enhanced mitochondrial functions. MMP measurement was used to assess the effects of CAP/pCD38 treatment on mitochondrial integrity. 1L, the ratio of mitochondria with normal MMP to those with depolarized MMP in CAP/pCD38-treated MSCs was significantly higher than that in untreated MSCs, indicating that CD38-mediated Ca²⁺ elevation did not compromise MMP. At the same time, the upregulation of CD38 did not negatively affect mitochondrial respiratory function and MMP level in donor cells (MSCs). A series of experimental validations demonstrated that the release of EV-Mito from MSCs was regulated by the CD38/IP3R/Ca²⁺ pathway. Based on these findings, we genetically engineered MSCs using CAP/pCD38 complexes to create super donor MSCs. In brief, we utilized the gold-standard plasmid gene transfection reagent (Lipo3000) and our previously developed non-viral gene delivery vector (CAP) to load the pCD38, forming Lipo3000/pCD38 and CAP/pCD38 complexes. These complexes were used to genetically engineer adipose-derived MSCs, thereby activating the CD38/IP3R/Ca²⁺ pathway in MSCs. After treating MSCs with Lipo3000/pCD38 or CAP/pCD38 for 4 h, the medium was replaced with a fresh medium. MSCs were incubated for an additional 20 h or 44 h. The condition medium was centrifuged at 300 × g for 10 min to remove cell debris, followed by centrifugation at 2500 × g for 10 min to eliminate residual impurities. The supernatant was centrifuged at 18,000 × g for 30 min to obtain EV-Mito as a pellet. In subsequent experiments, EV-Mito collected from untreated MSCs was designated as Ctrl-EV-Mito, this from Lipo3000/pCD38-treated MSCs as Lipo-EV-Mito, and this from CAP/pCD38-treated MSCs as Super-EV-Mito (Fig. A Extraction and purification of Super-EV-Mito. D The proportion of released mitochondria encapsulated in Super-EV-Mito using nano-flow cytometry analysis. E The proportion of released mitochondria within EVs calculated by TEM images. F Western blots of EV-Mito and mitochondrial markers in different EV-Mito groups, representative images from n = 3 independent experiments. I Mitochondrial basal respiration detection in different EV-Mito. i: Standard curve between mitochondrial protein and fluorescence intensity of Mitotracker Green probe; ii: OCR normalized by mitochondrial protein in different EV-Mito groups. Data are presented as mean ± SD. n = 3 biologically independent samples for each in (E, G–J). Statistical significances were performed via one-way ANOVA with two-sided Tukey's HSD post hoc test (G (ii), I (ii)), two-sided Games-Howell test (G (i), H), or two-sided t-test (J). The zeta potential of Super-EV-Mito was −1.76 ± 0.27 (Supplementary Table 1). Multimodality Structured Illumination Microscopy (multi-SIM) imaging system was used to observe Super-EV-Mito. Before multi-SIM imaging, MSC membranes and mitochondria were labeled with DiO and Mitotracker Red dyes, respectively. 2C, DiO-labeled membrane structures enveloped Mitotracker Red-labeled mitochondria. Mitotracker Red fluorescence signals were observed within individual Super-EV-Mito, confirming that Super-EV-Mito contained mitochondria. It has been reported that not all released mitochondria are encapsulated within EVs; a portion may indeed exist in the form of free mitochondria. We conducted nano-flow cytometry analysis to quantify the proportion of released mitochondria encapsulated in EVs. The result showed that approximately 24.87 ± 2.76% of mitochondrial particles were double-positive, indicating their encapsulation within EVs, while the remaining mitochondria were FITC-positive, suggesting that they existed as free mitochondria (Fig. However, the double-positive population in nano-flow cytometry only reflects the proportion of mitochondria associated with EVs and does not directly visualize how many mitochondria are present within individual EVs. To address this, we further quantified the mitochondrial content within EVs using transmission electron microscopy (TEM) analysis. Statistical evaluation of Super-EV-Mito TEM images revealed that 6 out of every 10 released mitochondria were encapsulated in EVs, while the remaining 4 existed in a free form (Fig. There is heterogeneity in EV-Mito structures. To further investigate whether the quantity and quality of mitochondria contained within Super-EV-Mito are improved after activation of the CD38/IP3R/Ca2+ pathway, we used Western blot assay and flow cytometry for assessing the quantity of EVs collected from each group. Subsequently, we evaluated the mitochondrial quality within the EVs using mitochondrial ROS level and basal respiration detection. Western blot assay was carried out to explore the expressions of key EV membrane markers (CD81) and mitochondrial proteins (MT-ND4 and COX IV) under equal protein loading conditions. Flow cytometry was further employed to quantify mitochondrial content, based on MTG mean fluorescence intensity (MFI) and positivity ratios. Furthermore, we assessed the mitochondrial activity in the Super-EV-Mito by measuring mitochondrial ROS and basal respiration. MitoSOX probe was employed to assess mitochondrial ROS levels within EV-Mito, reflecting the functional activity of mitochondria in these EVs. The MitoSOX/MTG intensity in Super-EV-Mito was lower than the EV-Mito group, suggesting that CD38 activation enhanced the quality of mitochondria encapsulated within these EVs (Fig. In addition, we investigated whether Super-EV-Mito contains more functional mitochondria by assessing the basal respiration. We established a standard curve correlating Mitotracker Green fluorescence intensity with mitochondrial protein concentration (Fig. Based on this calibration, we quantified the basal respiration per unit of mitochondrial protein using OCR assays. 2I (ii), the basal respiration per unit mitochondrial protein in Super-EV-Mito was significantly higher than that in Ctrl-EV-Mito and Lipo-EV-Mito. We evaluated the mitochondrial function in EV-Mito at different time points to reflect the stability of mitochondria within EV-Mito. We isolated free mitochondria from CAP/pCD38 complex-treated MSCs as a control group (called isolated mitochondria from Super MSCs). 2J, the basal respiration of Super-EV-Mito, as measured by OCR, remained relatively stable for up to two days. In contrast, the mitochondrial function of isolated mitochondria from Super MSCs dropped significantly starting on the first day, showing lower activity compared to the Super-EV-Mito group. The finding indicated that mitochondria encapsulated within EVs exhibited superior functional stability, maintaining their activity for at least two days. We speculated that this stability was mainly attributed to the membrane structure of Super-EV-Mito, which protected mitochondrial activity. To investigate this hypothesis, we compared the mitochondrial activity in Super-EV-Mito and isolated mitochondria under conditions of high extracellular Ca²⁺. The Ca²⁺ concentration gradient (0, 0.0001, 1.25, and 2.5 mM) was used to simulate the mitochondrial functionality of Super-EV-Mito under cytosolic, extracellular, and blood calcium conditions, respectively. Accordingly, we assessed changes in MMP using the MMP-dependent Mitotracker Red probe (MTR) to characterize mitochondrial functional alterations under high extracellular Ca2+ concentrations. We measured the FI of MTR in Super-EV-Mito and isolated mitochondria from Super MSCs without any treatment at 0 h using a multifunctional microplate reader, designated as FIS and FII, respectively. As the concentrations of Ca²⁺ increased, the normalized FI of MTR in the Super-EV-Mito group remained above 70%, whereas the normalized FI of MTR in isolated mitochondria showed an obvious decrease compared with Super-EV-Mito. Our results demonstrated that the membrane structure of Super-EV-Mito protected mitochondria from Ca²⁺-induced MMP loss, suggesting that Super-EV-Mito was likely to preserve mitochondrial function under high Ca²⁺ conditions and resist external stress (Supplementary Fig. Therefore, the Super-EV-Mito collected by activating the CD38/IP3R/Ca2+ pathway in MSCs exhibited a higher quantity and mitochondrial activity compared to the Ctrl-EV-Mito collected from normal MSCs. This also indicated that the activation of the CD38/IP3R/Ca2+ pathway led to the generation of super-donor cells, thereby holding the potential to provide a greater quantity and higher quality mitochondrial material for mitochondria transfer therapy. LHON is a classical ocular mitochondrial disease resulting from mutations in mtDNA that encode key proteins related to mitochondrial function. Abnormalities in mtDNA encoding three subunits of complex I in the mitochondrial respiratory chain are primary pathogenic factors in more than 95% of LHON cases. Notably, 90% of Asian LHON patients are closely associated with a homoplasmic nucleotide substitution of guanine to adenine at position 11778 in mtDNA41. The mutations at this site lead to a functional abnormality in the key protein ND4 of mitochondrial complex I, causing mitochondrial dysfunctions, including mitochondrial respiratory chain dysfunction, decreased ATP generation, and MMP depolarization42. To investigate the therapeutic effects of Super-EV-Mito in mitochondrial diseases, we utilized LHON model cells (GM10742 cells) and evaluated the mitochondrial functional restoration capacity of different formulations after administration43. The cellular uptake of Super-EV-Mito in GM10742 cells was explored by flow cytometry analysis. We transduced MSCs with a viral vector encoding mEmerald-TOMM20 to specifically label mitochondria within the MSCs. We then utilized mEmerald-TOMM20-labeled Ctrl-EV-Mito, Lipo-EV-Mito, and Super-EV-Mito to track uptake in recipient cells. Ctrl-EV-Mito, Lipo-EV-Mito, and Super-EV-Mito exhibited varying levels of mitochondrial uptake in recipient GM10742 cells. Among these, Super-EV-Mito increased the MFI of mEmerald-TOMM20 compared to the other EV-Mito, likely due to the higher content of mitochondria in Super-EV-Mito (Fig. A Cell uptake of different groups evaluated by flow cytometry analysis. B The expression of MT-ND4 and COX IV protein detected by Western blot assay, representative images from n = 3 independent experiments. The quantitative data (C) and CLSM images (D) of MMP detection. E OCR from Seahorse cell mito stress test of GM10742 cells in various treatment groups. Spare respiratory capacity (F) and maximum respiration (G) of GM10742 cells after varying treatment groups. I Detection of mitochondrial ROS level. K Cell proliferation evaluated using EdU assay. L KEGG pathway enrichment analysis based on the significant differentially expressed genes between untreated GM10742 cells and Super-EV-Mito-treated GM10742 cells. Data are presented as mean ± SD. n = 3 biologically independent samples for each in (A, E–K). n = 8 biologically independent samples for each in (C). Statistical significances were performed via one-way ANOVA with two-sided Tukey's HSD post hoc test (A, F, H, J, K) or two-sided Games-Howell test (C, G, I). KEGG pathway enrichment analysis was performed via two-sided hypergeometric test (L). Subsequently, we systematically investigated whether the introduction of EV-Mito could effectively improve mitochondrial function in GM10742 cells with mtDNA mutations. The expression of mitochondrial ND4 protein (MT-ND4) in GM10742 cells was limited due to the mutations at position 11778 in mtDNA. Super-EV-Mito promoted the MT-ND4 expression in GM10742 cells (Fig. This demonstrated that EV-Mito could increase the levels of mitochondrial proteins. The JC-1 is a fluorescent dye commonly used to assess MMP44,45. In functional mitochondria with high MMP, JC-1 aggregates form and emit red fluorescence. JC-1 remains in its monomeric form when MMP is low. The ratio of red to green fluorescence provides a quantitative measure of MMP, making it a reliable indicator of mitochondrial health and function. The value of red/green fluorescence intensity after Super-EV-Mito treatment was obviously higher than the untreated group (Fig. It suggested that untreated GM10742 cells exhibited abnormalities in MMP, which could be improved through the supplementation of exogenous mitochondria provided by Super-EV-Mito. Furthermore, the mitochondrial respiratory chain function of GM10742 cells after treatment with different formulations was evaluated using OCR assays. Super-EV-Mito significantly enhanced the spare respiratory capacity and maximum respiration of GM10742 cells, indicating that Super-EV-Mito treatment improved the mitochondrial respiratory chain function in mtDNA-mutant GM10742 cells (Fig. Additionally, intracellular ATP levels and mitochondrial ROS levels were measured, revealing that Super-EV-Mito effectively increased ATP production while reducing mitochondrial ROS levels in GM10742 cells (Fig. Excessive opening of mPTP is often associated with mitochondrial dysfunction. The degree of mPTP opening was assessed using Calcein fluorescence. 3J, untreated GM10742 cells exhibited relatively low Calcein fluorescence. After treatment with Ctrl-EV-Mito, Lipo-EV-Mito, and Super-EV-Mito, Calcein fluorescence values progressively increased, with the highest levels observed in the Super-EV-Mito group, suggesting that Super-EV-Mito produced the best therapeutic effect on mitochondrial function. Furthermore, the EdU assay was applied to investigate cellular proliferation. Due to mitochondrial dysfunction caused by mtDNA mutations, untreated GM10742 cells displayed limited proliferative capacity. Super-EV-Mito provided “energy” to GM10742 cells by supplementing exogenous mitochondria, thereby enhancing their proliferative capacity (Fig. After being treated with Super-EV-Mito, GM10742 cells were collected for KEGG pathway enrichment analysis. The significantly differentially expressed genes between untreated GM10742 cells and Super-EV-Mito-treated GM10742 cells were enriched in mitochondrial metabolism, AMPK signaling pathway, and mitophagy (Fig. We further investigated the long-term therapeutic potential of Super-EV-Mito by examining both their persistence in GM10742 cells and their sustained ability to restore mitochondrial function. Our results showed that mtDNA originating from donor rat mitochondria could persist in GM10742 cells for up to 5 days post-administration (Supplementary Fig. Furthermore, a single administration of Super-EV-Mito led to significantly improved mitochondrial function in GM10742 cells, which was maintained for up to 5 days (Supplementary Fig. In summary, Super-EV-Mito effectively delivered exogenous mitochondria to LHON model cells, restoring mitochondrial functions, including MMP, ATP production, mitochondrial ROS levels, and mPTP opening. This restoration enhanced the proliferative capacity of LHON model cells. Furthermore, as Super-EV-Mito contain a greater quantity of mitochondria with superior functionality, the therapeutic efficacy of the Super-EV-Mito treatment group in vitro was superior to the Ctrl-EV-Mito treatment group. Given the exceptional mitochondrial function restoration achieved by Super-EV-Mito in GM10742 cells, we investigated the therapeutic potential of Super-EV-Mito for mitochondrial disease in the LHON male mouse model. The safety of Super-EV-Mito was assessed in vivo. Various doses of Super-EV-Mito were set and administrated in the normal C57BL/6J male mice, including 1-fold (Super-EV-Mito-1), 2-fold (Super-EV-Mito-2) and 4-fold (Super-EV-Mito-4) Super-EV-Mito. Different doses of Super-EV-Mito were administered again 7 days after the first administration. 13A, B, even at high doses, Super-EV-Mito-4 did not induce systemic toxicity in mice, nor did it cause significant damage to the retina or major organs. However, direct transplantation of MSCs into the eye appeared to negatively impact the structural integrity of the retinal layers and led to elevated expression of inflammation-related proteins (Supplementary Fig. Therefore, Super-EV-Mito exhibited a high level of in vivo safety. Prior to conducting formal efficacy studies in vivo, we assessed the in vivo persistence of donor-derived mtDNA in ocular tissues to evaluate the long-term therapeutic potential of Super-EV-Mito. Following a single intravitreal administration, donor rat mtDNA remained detectable in the eye for up to 14 days (Supplementary Fig. LHON model male mice were purchased from Aurora Bioscience Co., Ltd. (Suzhou, China). A mitochondria-targeted adeno-associated virus (MTS-AAV) containing the mutant human ND4 gene (mutND4), followed by mitochondrial-encoded mCherry, was microinjected into zygotes. Female founders (F0) exhibiting mCherry fluorescence on ophthalmoscopy were backcrossed with normal males for producing the first generation of offspring (F1): mutant ND4 C57BL/6 J male mice48. The therapeutic effect of Super-EV-Mito was evaluated in LHON model male mice (Fig. Specifically, three-month-old LHON model male mice received an initial treatment, followed by a second treatment seven days later. After two weeks, the optomotor test and electroretinography (ERG) assessments were conducted on all groups of mice. Furthermore, the eyeballs were collected for intraocular ATP level measurement and retinal H&E sections. Additionally, we introduced Idebenone (Ide), the only drug currently approved for adjunctive treatment of LHON, as a positive control group49. Visual recovery in each group was evaluated using the optomotor test and ERG assessments. For the optomotor test, a rotating drum with alternating black-and-white striped circular panels was constructed. Video recordings were used to count the number of head turns made by each mouse within the 2-minute rotation periods. The results showed that wild-type male mice (WT group) exhibited an average of 15.00 ± 1.41 head movements/2 min, while PBS-treated LHON model male mice showed only 6.71 ± 0.76 head movements/2 min, indicating severe visual impairment in the LHON male mice. Ide treatment via daily oral administration resulted in partial visual recovery, whereas mice treated with Super-EV-Mito demonstrated significantly better visual recovery compared to the Ide group (Fig. ERG, as an important diagnostic tool for detecting retinal and choroidal hereditary diseases, was utilized to evaluate retinal function in mice after different treatments. ERG assessments were conducted on mice following 24 h of dark adaptation. The ERG waveforms of WT male mice, PBS-treated, Ide-treated, and Super-EV-Mito-treated LHON model male mice were shown in Fig. Statistical analysis of the amplitude variations in a-wave and b-wave revealed that PBS-treated LHON model male mice showed a significant decrease in the absolute amplitudes of both the a-wave and b-wave relative to wild-type males. However, treatment with Super-EV-Mito markedly restored the absolute values of both waves, with the b-wave amplitude recovering to levels comparable to those of wild-type male mice (Fig. H&E staining sections were carried out to analyze the retinal structure (Fig. Retinal layer thickness and RGC counts were quantified from H&E-stained sections. Super-EV-Mito significantly increased retinal layer thickness and restored RGC numbers (Fig. Both parameters in the Super-EV-Mito-treated group showed no significant differences compared to WT mice, indicating that Super-EV-Mito effectively repaired retinal structures. Further investigation into the total expression of MT-ND4 protein in LHON model male mice after different treatments revealed that Super-EV-Mito treatment restored ND4 protein expression in the eyes by supplying exogenous functional mitochondria. Western blot analysis of mitochondrial COX IV protein showed that mitochondria transplantation mediated by Super-EV-Mito increased the number of exogenous mitochondria, thereby elevating COX IV protein expression in the eyes of the mice (Fig. The mitochondrial dysfunction caused by mtDNA mutations resulted in significantly lower ATP content in the eyes of PBS-treated LHON model male mice compared to wild-type mice. Super-EV-Mito treatment significantly increased ATP content in the eyes (Fig. A Flowchart for Super-EV-Mito treatment in the LHON male mouse model. B The optomotor test (CW clockwise, CCW counterclockwise). C ERG waveforms in varying treatment group. Amplitude variations of a- (D) and b-wave (E) in ERG. F H&E sections of the different treatment groups. Thickness of general retina (G) and retinal ganglion cells (RGC) numbers (H) were summarized to evaluate the restoration of retinal structure. I The expression of MT-ND4 and COX IV protein detected by Western blot assay, representative images from n = 3 independent experiments. J The quantitative results of MT-ND4 and COX IV protein expression were obtained using ImageJ software. K The ATP contents after varying treatments. A and B were created in BioRender. Data are presented as mean ± SD. n = 3 eyes for each in (D, E, G, H, J, K). n = 7 independent process of optomotor tests for each in (B). Statistical significances were performed via one-way ANOVA with two-sided Tukey's HSD post hoc test (B, D, E, G, H, J, K). Inspired by the therapeutic efficacy of Super-EV-Mito in the mtDNA mutant LHON male mice, we investigated their potential to supplement exogenous functional mitochondria for the treatment of a broader range of mitochondrial disorders, beyond primary mitochondrial disorder stemmed from mtDNA mutations. To this end, we utilized Rot, a chemical inhibitor of mitochondrial complex I, to establish a Rot-induced mitochondrial disease model. As noted in previous studies, Rot-induced cell and male mouse models have been widely used as models for various mitochondrial diseases, including LHON and Parkinson's disease. The binding sites of Rot are located within complex I, specifically targeting regions such as decylubiquinone and the ND4 subunit, resulting in complex I dysfunction50,51. The Rot-damaged cell model was established by pre-treating HeLa cells with 0.1 µM Rot, and the cellular uptake of exogenous mitochondria and mitochondrial functional restoration effects of Super-EV-Mito were assessed in this model. Mitochondria from donor cells (MSCs) were labeled by mEmerald-TOMM20, and mitochondria-enriched vesicles were obtained following untreated, Lipo3000/pCD38, or CAP/pCD38 transfection. Rot-induced cells were treated with EV-Mito from different groups, and the MFI of mEmerald-TOMM20 in Rot-induced cells was analyzed using flow cytometry and confocal laser scanning microscopy (CLSM). The data exhibited that Rot-damaged cells treated with Super-EV-Mito exhibited the highest fluorescence intensity, indicating that Super-EV-Mito-mediated mitochondria transfer was feasible even in Rot-damaged cells (Fig. In addition, we conducted uptake inhibition experiments to investigate the mechanism by which mEmerald-TOMM20-labeled Super EV-Mito was internalized into the cytoplasm of recipient cells. As shown in the Supplementary Fig. 14, we found that Super-EV-Mito entered recipient cells via multiple endocytic pathways, with macropinocytosis and caveolae-mediated endocytosis being the predominant mechanisms. Cell uptake of mEmerald-TOMM20-labeled EV-Mito evaluated by flow cytometry analysis (A) and CLSM (B), green: mEmerald-TOMM20-labeled EV-Mito. CLSM images (C) and quantitative data (D) of MMP detection. E Western blots of MT-ND4 and COX IV protein expressions. F The ATP contents after varying treatments. G Mitochondrial ROS level detected by MitoSOX probe. H The detection of mPTP opening. I Cell proliferation evaluated using EdU assay. J Flowchart for Super-EV-Mito treatment in the Rot-induced LHON male mice. L Immunofluorescence staining of mitochondrial complex I after varying treatments (red: mitochondrial complex I; blue: nucleus), GCL ganglion cell layer, INL inner nuclear layer, ONL outer nuclear layer. M Representative H&E images in the varying treatment groups. N Thickness of the general retina after different treatments. O RGC numbers in different treatment groups. Data are presented as mean ± SD. n = 7 biologically independent samples for each in (D), n = 7 independent process of optomotor tests for each in (K). Statistical significances were performed via one-way ANOVA with two-sided Tukey's HSD post hoc test (A, D, G–I, N, O) or two-sided Games-Howell test (F, K). In Rot-induced cells, mitochondrial damage led to reduced MMP, whereas Super-EV-Mito treatment restored MMP to levels comparable to the PBS group without Rot exposure (Fig. Given that Rot also damages the ND4 protein site within mitochondrial complex I, the Rot + untreated group exhibited lower MT-ND4 expression than the untreated group. Super-EV-Mito restored MT-ND4 expression by introducing exogenous mitochondria (Fig. Moreover, Ctrl-EV-Mito, Lipo-EV-Mito, and Super-EV-Mito all increased ATP content in Rot-damaged cells (Fig. 5F) and reduced mitochondrial ROS accumulation induced by Rot (Fig. Super-EV-Mito markedly reduced mPTP opening, as indicated by higher Calcein Green fluorescence (Fig. Most importantly, all vesicle formulations enhanced the proliferation capacity of Rot-induced cells to varying degrees, with Super-EV-Mito showing the most pronounced effect (Fig. These findings suggested that Super-EV-Mito not only restored mitochondrial functions in mtDNA-mutated LHON cells but also regulated mitochondrial function in Rot-induced cells with widespread mitochondrial damage, ultimately rescuing cell proliferation activity. Based on our research and studies from other investigators, a Rot-induced male mouse model was established to simulate ocular mitochondrial diseases associated with widespread mitochondrial dysfunction by intravitreal injection of 2.5 mM Rot42,52. The first treatment was administered on day 7 post-modeling, followed by a second treatment on day 14. On day 28, visual function in mice was evaluated (Fig. Optomotor response testing showed that Super-EV-Mito significantly restored visual impairment caused by Rot (Fig. Treatment with Super-EV-Mito promoted the upregulation of complex I expression (Fig. H&E staining revealed that Rot induced severe retinal structural damage, including a marked reduction in retinal layer thickness and RGC numbers. After Super-EV-Mito treatment, both retinal layer thickness and RGC counts were significantly improved (Fig. A series of studies conducted on gene-mutated LHON cells and the LHON male mouse model demonstrated that Super-EV-Mito could effectively supplement exogenous mitochondria via efficient mitochondria transfer, thereby restoring mitochondrial function and achieving effective treatment of LHON (Fig. Furthermore, Super-EV-Mito effectively mitigated disease progression by enhancing mitochondria transfer efficiency and promoting the mitochondrial functions in the Rot-induced pan-mitochondrial disease model. B The generation of super donor cells for EV-Mito and a schematic representation of Super-EV-Mito for mitochondrial repair and delaying the progression of LHON. A and B were created in BioRender. CAP a non-viral gene delivery vector, EV-Mito extracellular vesicle containing mitochondria, MSC mesenchymal stem cells. However, the majority of mitochondrial diseases currently lack effective treatments. Mitochondria transfer is a natural process where functional mitochondria are released into damaged cells via EV-Mito to restore mitochondrial function under pathological conditions. To overcome the challenges of limited yield and quality of mitochondria-enriched EVs, which hinder their therapeutic potential for mitochondrial diseases, we employed our previously engineered non-viral transfection vector (CAP) to deliver the pCD38 into MSCs. This approach activates the CD38/IP3R/Ca²⁺ pathway, thereby generating super donor cells. These super donor cells produced EV-Mito (termed Super-EV-Mito) containing 3 times the mitochondrial content of normal MSC-derived EVs, with significantly enhanced mitochondrial quality. This strategy not only enabled the collection of a greater number of high-quality EV-Mito, reducing the production cost of Super-EV-Mito-mediated mitochondria transfer therapy but also utilized the membrane structure of EVs to preserve mitochondrial activity. We explored the mechanism by which CD38 signaling promotes the release of EV-Mito: (1) CD38 catalyzes the synthesis of the calcium-mobilizing second messenger cADPR, resulting in a significant elevation of cytoplasmic calcium levels; (2) Mitochondria are important intracellular calcium stores. We observed that upregulation of CD38 expression led to increased expression of IP3Rs on the ER. These IP3Rs rapidly mediate Ca²⁺ transfer to mitochondria. (3) Previous studies have shown that elevated intracellular Ca²⁺ levels can trigger the fusion of vesicles with the plasma membrane, driving vesicle-plasma membrane fusion and facilitating EV release. To further determine whether activation of the CD38/IP3R/Ca²⁺ signaling pathway enhances both the quantity and quality of mitochondria encapsulated within Super-EV-Mito, we conducted a series of assays including Western blot, flow cytometry, and TEM to quantify EVs derived from different groups. These results collectively demonstrated that activation of the CD38/IP3R/Ca²⁺ axis induces the generation of super donor MSCs, which produce EVs with both increased mitochondrial payload and improved mitochondrial functionality. This strategy may thus serve as a promising approach for enhancing the therapeutic efficacy of mitochondria transfer. In both a typical ocular mitochondrial disease model of LHON and a Rot-induced mitochondrial disease model, Super-EV-Mito alleviated mitochondrial dysfunction-associated vision loss in mice. The severity of such diseases is closely linked to the proportion of mutant mtDNA, with a well-recognized threshold effect-clinical symptoms typically emerge when the proportion of mutant mtDNA exceeds ~60%. Consequently, strategies aimed at reducing the mutant load below this pathological threshold are considered promising therapeutic avenues. Mitochondria transfer represents one such strategy by introducing exogenous, healthy mitochondria into affected cells. This approach can restore mitochondrial function through two complementary mechanisms: (1) providing wild-type mtDNA to dilute the mutant load and shift the heteroplasmic ratio, and (2) stimulating mitophagy to selectively eliminate dysfunctional mitochondria enriched in mutant mtDNA. Importantly, the therapeutic benefit of mitochondria transfer does not rely on directly editing or repairing mutant mtDNA, but rather on reestablishing mitochondrial homeostasis through the combined effects of mitochondrial supplementation and mitophagy activation. Although our results demonstrated that CD38 overexpression significantly enhanced mitochondrial activity and functionality in both MSCs and their derived Super-EV-Mito, it is important to note that such alterations in mitochondrial function may also influence MSC metabolism and differentiation potential. To minimize these potential risks, we limited the transfection procedure to a single round per MSC batch and collected EVs only once from each batch. This precautionary measure was taken to avoid repeated use of the same MSC population for EV production. In future studies, we aim to further optimize MSC engineering and EV harvesting strategies to maximize the proportion of mitochondria encapsulated within EVs while minimizing production costs, thereby enhancing the translational feasibility of this approach. Therefore, this study suggests that engineering super donor cells is an advanced source for the scalable production of EV-Mito to treat mitochondrial disorders. This approach offers a promising strategy for treating primary mitochondrial diseases such as LHON caused by mtDNA mutations, while also opening a avenue for addressing secondary mitochondrial dysfunction associated with broader disease contexts. Cystamine dihydrochloride, acryloyl chloride, agmatine dihydrochloride (Agm), and anhydrous DMSO from J&K Chemical (Beijing, China). Mitotracker Red or Green probe, MitoSOX probe, Mitochondria Isolation Kit, Lipofectamine 3000 (Lipo3000), LysoTracker Deep Red probe, SDS-PAGE and agarose from Thermo Fisher Scientific (Waltham, USA). Bicinchoninic acid (BCA) and Dulbecco's Modified Eagle Medium (DMEM) from KeyGEN Biotech (Nanjing, China). MMP Kit of JC-1, 4,6-diamino2-phenylindole (DAPI), Hoechst 33342, Calcein-AM, Enhanced ATP Assay Kit (s0027), HRP-Goat Anti-Mouse IgG (A0216), HRP-Goat Anti-Rabbit IgG (A0208), Alexa Fluor 647-Goat Anti-Rabbit IgG (A0468) and BeyoClick™ EdU Cell Proliferation Kit with TMB (C0088S) from Beyotime Biotechnology (Shanghai, China). Anti-NDUFB8 Recombinant Rabbit mAb (ET7108-25) from HuaBio (Hangzhou, China). The siRNA targeting IP3R (siIP3R) from Riobio Company Limited (Supplementary Sequence 1, Guangzhou, China). Rotenone (Rot) from Aladdin Bio-Chem Technology (Shanghai, China). NaCl, NaOH, and HCl were purchased from Yuanye Bio-Technology (Shanghai, China). Firstly, cystamine dihydrochloride was dissolved in water at a concentration of 50 mM. Acryloyl chloride (0.1 M) was dissolved in dichloromethane at a volume of 10 mL. After that, NaOH solution (100 mM, 10 mL) was prepared. 10 mL of acryloyl chloride (0.05 M), as well as NaOH solution, were transferred in two separate dripping funnels. Then the two components were added drop by drop in a three-necked flask at the same time. The mixing process lasted for 60 min at 0 °C while stirring. Another 6 h was spent to perform the reaction at room temperature (r.t.). Finally, dichloromethane was used to dissolve the mixture. The purification of CBA was performed by crystallization using ethyl acetate. CAP was synthesized by Michael's addition reaction in which the N, N-cystamine-bis-acrylamide (C), agmatine dihydrochloride (A), and p-aminoethylbenzenesulfonic acid (P) were used as monomers. The reaction should avoid the light and be protected by argon for 24 h at 90 °C after adding triethylamine. Impurities were removed from CAP to ultrapure water by a 24-hour-long dialysis procedure with a Dialysis membrane (MWCO = 500), and then CAP was obtained by lyophilization. 1H Nuclear magnetic resonance (1H NMR) was performed to confirm the structures. The pDNA used in this study was amplified in E. coli strain DH5α, and then an Endo-free Plasmid Maxi Kits were performed to extract pDNA. The pDNA (pCD38 or pGFP) was dissolved in H2O. Anhydrous dimethyl sulfoxide (DMSO) was used to dissolve and store CAP (200 mg/mL). The CAP mother solution was diluted with H2O into various concentrations. Then pDNA solution was added drop by drop to CAP solution in vortex condition. Dynamic light scattering analyzer (DLS) performed by Nano ZS90 zetasizer was used to detect the particle sizes and zeta potentials of NPs. Adipose tissues from Sprague-Dawley rats were collected and digested by type I collagenase. Then, DMEM with 10% FBS was used to culture MSCs. After the 3-day culture, unadhered impure cells were removed by fresh medium. Cells were cultured when 90% confluence was achieved. HeLa cells (human breast cancer cell line) were obtained from the Shanghai Institute of Cell Bank, Chinese Academy of Sciences (Shanghai, China). GM10742 cells were purchased from the Coriell Institute for Medical Research. MSCs and HeLa cells were cultured in DMEM supplemented with 10% FBS, 100 U/mL penicillin, and 100 μg/mL streptomycin, while GM10742 cells were maintained in RPMI-1640 containing 10% FBS, 100 U/mL penicillin, and 100 μg/mL streptomycin. All cells were incubated at 37 °C in a humidified atmosphere with 5% CO₂. For all cell experiments in this study, the untreated group served as the blank control. HeLa cells were constantly exposed to either an FBS-free medium or a Rot-contained FBS-free medium at the concentration of 0.1 μM for 12 h. Finally, the medium was replaced for the subsequent studies. MSCs were seeded on 24-well culture plates. Medium was replaced by different formulations at a concentration of 1 μg pDNA/well diluted by DMEM without FBS. The different formulations were incubated with the cells for 4 h. Medium was replaced with 1 mL fresh DMEM. Another 20 h cultivation was performed before detection. Finally, the positive rate (%) and mean fluorescence intensity were assessed by flow cytometry. MSCs were cultured in tissue culture flasks with a 75 cm2 growth area (T75). After 48 h, MSCs were transfected by the different formulations (10 μg pCD38/flask). The culture medium was replaced with EV-depleted medium and collected after 48 h. The medium was sequentially centrifuged at 300 × g for 10 min, 2500 × g for 10 min, and finally at 18,000 × g for 30 min at 4 °C to isolate EV-Mito from the supernatant. EVs marker expression of CD81 and the mitochondrial marker expression of COX IV and MT-ND4 were measured by Western blot. EV-Mito was resuspended at 0.1 mg/mL (protein concentration) in PBS for DLS and zeta potential detections. For nanoparticle tracking analysis (NTA) detection, EV-Mito was resuspended in 40 μL PBS and measured using a ZetaView Particle Metrix (Particle Metrix, PMX-120, Germany). Polystyrene beads at 100 nm (Thermo Fisher Scientific, Fremont, CA) were used to calibrate the instrument. NTA software was carried out to identify the concentrations of nanoparticles. For multi-SIM imaging system (NanoInsights-Tech Co., Ltd.), EV-Mito was resuspended at 0.1 mg/mL (protein concentration). The multi-SIM images were taken using single slice mode with 50 mW laser power and 30 ms exposure time. Images were then reconstructed using the SIM Imaging Analyser software53. For nano-flow detection, MSCs were transfected with mEmerald-TOMM20 using recombinant adenovirus (Ad-mEmerald-TOMM20) at a virus titer of 2.0 × 108 PFU/mL for 24 h. Then, MSCs were transfected with CAP/pCD38 complexes. The EV-Mito was extracted and purified. CD81 rabbit mAb was used as the label of EV-membrane. ABflo® 647-conjugated Goat anti-Rabbit IgG (H + L) was set as the secondary antibody. The proportion of released mitochondria encapsulated in EVs were analyzed using the Flow NanoAnalyzer U30E (NanoFCM, Xiamen, China). For transmission electron microscopy (TEM) imaging, EV-Mito was pipetted onto a 150-mesh copper and incubated for 3 min. Mesh copper was stained with 2% uranyl acetate for 8 min, and washed with distilled water. Then it was stained with 2.6% lead citrate. The images were recorded by TEM (HT7800, HITACHI, Japan). For flow cytometry analysis, MSCs were cultured in T75 flasks. After 48 h, MSCs were treated with the different formulations (10 μg pCD38/flask). Then the medium was replaced with 100 nM of Mitotracker Green (MTG) for 45 min in serum-free medium. The EV-Mito was extracted and washed with PBS for flow cytometry analysis. For Ca2+ protection, Super-EV-Mito was obtained and mixed with a series of varying concentrations of Ca2+ for 24 h at 4 °C. The fluorescence intensity of Mitotracker Red was detected by a multifunctional microplate reader. For pharmacological inhibition studies, MSCs were seeded on 24-well culture plates, transfected with CAP/pCD38 complex, and stained with MTG. Then the medium was replaced by an EV-depleted medium with different inhibitors (cytochalasin B (20 μM), Paclitaxel (20 nM)) and collected after 24 h. Then, EV-Mito was extracted and purified for the flow cytometry analysis. MSCs were seeded on 24-well culture plates and transfected with pCD38 at a concentration of 1 μg pCD38/well. The PE-labeled Anti-CD38 antibody was used for immunofluorescence detection. In addition, the Anti-IP3R antibody and Alexa Fluor 647-Goat Anti-Rabbit IgG antibody were used for immunofluorescence detection. The fluorescence intensity was assessed by flow cytometry. MSCs were seeded on 24-well culture plates and transfected with CAP/pCD38. The rhod-2-AM (2 μM) was stained for 30 min at 37 °C. The MFI of rhod-2 was assessed by flow cytometry. MSCs were seeded on 24-well culture plates and transfected with CAP/pCD38 complexes. The MFI of fluo-4 was assessed by flow cytometry. MSCs were seeded on 10 cm culture dishes and transfected with pCD38. MTG was carried out to stain mitochondria in MSCs. After 48 h of Lipo3000/pCD38 or CAP/pCD38 complex treatment, condition medium was collected and the normalized FI of MTG-labeled EV-Mito in condition medium was detected using a multifunctional microplate reader. MSCs were transfected with mEmerald-TOMM20 using recombinant adenovirus (Ad-mEmerald-TOMM20) at a virus titer of 2.0 × 108 PFU/mL for 24 h. Then, MSCs were transfected with different treatment groups. The EVs were extracted and purified. Then cells were treated at 6 μg EV protein/well in complete growth medium for 24 h. After washing by PBS, cell uptake was detected by CLSM (Zeiss LSM800, Germany) and flow cytometry. The GM10742 cells were plated in the 24-well plate at 1 × 105 cells/well and cultured with different EV-Mito at 6 μg EV-Mito protein/well. After different time points (1, 2, 3, 5 days), cells were collected for qPCR detection. The primers of rat MT-ND4 DNA were as follows: Forward 5'-CGATCCATTATCCACCCCAC-3', Reverse 5'- TGTTGGGATTAGAGTGGCTTCG-3', and the primers of human GAPDH DNA were: Forward 5'- GGTCTGAGGTTAAATATAGCTGCTG-3', Reverse 5'-TTGATTTGCCAAGTTGCCTG-3'. The relative copy numbers of rat MT-ND4 were quantified through 2−(∆∆Ct) by real-time PCR analysis (CFX connect, Bio-Rad, USA). The HeLa cells or Rot-induced HeLa cells were plated in the 35 mm glass bottom dishes (8 × 104 cells/well). Cells were cultured without Mito-EVs (untreated group) or with Ctrl-EV-Mito, Lipo-EV-Mito and Super-EV-Mito (6 μg EV-Mito protein/well). After 48 h of incubation, the cells were covered by 1 mL 1 × JC-1 dye for 15 min. GM10742 cells, HeLa cells, or Rot-induced HeLa cells were plated at 24-well plates overnight and then treated with different groups at 6 μg EV-Mito protein/well for 48 h. Then the ATP Bioluminescent Assay Kits were used to assess the ATP content in different cells. The luminescence was recorded by chemiluminometer (Luminoskan Ascent, Thermo Fisher Scientific, USA). The oxygen consumption rate (OCR) of MSCs and GM10742 cells was determined with a Seahorse XFe96 Analyzer (Agilent, USA). MSCs were plated onto a cell culture plate at 3 × 104 cells per well for 24 h, then transfected with Lipo3000/pCD38 or CAP/pCD38 complexes for 12 h before Seahorse analysis. In this test, the medium was replaced with the assay medium, the medium (pH 7.4) containing glucose (10 mM) (Agilent, USA), L-glutamine (2 mM) and sodium pyruvate (1 mM) (Gibco, USA). GM10742 cells were plated onto a 24-well plate and treated with Ctrl-EV-Mito, Lipo-EV-Mito and Super-EV-Mito (6 μg EV-Mito protein/well) for 48 h. On the test day, the GM10742 cells were collected and resuspended with the assay medium, then seeded onto the cell culture plate at 1 × 104 cells per well. After equilibration, the OCR was measured. The OCR of different EV-Mito and isolated mitochondria was performed by the similar method as previous study54. After centrifugation, 180 μL of 37 °C-preheated MAS containing substrate (10 mM pyruvate, 2 mM malate, 4 μM FCCP) was added to each well. The OCR was measured by XFe96 analyzer. The mPTP opening was measured with Calcein-AM and CoCl2. In brief, the cells in each group were treated, collected and cultured with Calcein-AM (2 μM) and CoCl2 (1 mM) at 37 °C for 30 min. Finally, cells were washed with 1 mM CoCl2 on a shaker incubator for 25 min. The MFI of Calcein was assessed by flow cytometry. GM10742 cells, HeLa cells, or Rot-induced HeLa cells were plated at 24-well plates overnight and then treated with Ctrl-EV-Mito, Lipo-EV-Mito and Super-EV-Mito (6 μg EV-Mito protein/well) for 48 h. Then the cells were incubated with 500 nM of MitoSOX (M36007, Thermo Fisher Scientific, USA) at 37 °C for 30 min. The MFI of MitoSOX was assessed by flow cytometry. GM10742 cells, HeLa cells or Rot-induced HeLa cells were treated with Ctrl-EV-Mito, Lipo-EV-Mito and Super-EV-Mito (6 μg EV-Mito protein/well) for 48 h. Cell proliferation was detected with a BeyoClick™ EdU Cell Proliferation Kit with TMB (C0088S, Beyotime Biotechnology, China). The absorbance was investigated by the multifunctional microplate reader at 370 nm (Thermo Fisher Scientific Multiskan GO, USA). Conduct quality control using NanoDrop One spectrophotometer (NanoDrop Technologies, Wilmington, DE), Qubit 3.0 Fluorometer (Life Technologies, Carlsbad, USA), and electrophoresis. RNA integrity and concentration was assessed using the RNA Nano 6000 Assay Kit of the Bioanalyzer 2100 system (Agilent Technologies, CA, USA). The library preparation using VAHTS Universal V6 RNA-seq Library Prep Kit for MGI (Vazyme, Nanjing, China). RNA sequencing was performed using DNBSEQ T7 Sequencer (Complete Genomics, USA). Raw sequencing data were processed with FastQC for quality control, and clean reads were aligned to the rat or human reference genome using HISAT. Transcript abundance was quantified with RSEM and expressed as FPKM. Differentially expressed genes (DEGs) were identified by comparing treatment and control groups (fold change ≥ 2.0, posterior probability ≥ 0.8, FDR-adjusted P < 0.05). Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses were performed with clusterProfiler (v3.14.3), with terms considered significant at adjusted P < 0.05. The cells were collected for extracting proteins which were quantized by the BCA Protein Kit. After incubation for 2 h with 5% skim milk, the membranes were put with primary antibody (MT-ND4, GAPDH and COX IV antibody) at 4 °C all night. Eyes were collected for the extraction of protein. MT-ND4 and COX IV antibodies were used for analysis, and GAPDH was used as a loading control. C57BL/6J male mice (4-6 weeks) were purchased from East China Normal University Laboratory Animal Technology Co. Ltd. (Shanghai, China) and mutant mtND4R340H mitochondria transgenesis (mtTg) male mice (3 months of age) were purchased from Aurora Bioscience Co. Ltd. (Suzhou, China). All mice were housed under a 12 h light-dark cycle at 25 °C with 50% relative humidity. Mice were randomly assigned to pharmacodynamic studies. Three-month-old mutant mtND4R340H mtTg LHON male mice were randomly divided into 3 groups. Mice receiving an intravitreal injection of 2 μL PBS or Super-EV-Mito (1 μg EV-Mito protein/eye) were designated as the PBS group and the Super-EV-Mito group, respectively. Mice administered Ide orally at a daily dose of 60 mg/kg were designated as the Ide group. Age-matched wild-type male mice were included as the WT group. After 2 weeks of the second time of administration, the optomotor test and ERG were performed. The eyes were harvested for further analysis. To establish the Rot-induced LHON male mouse model, the mice were injected with 3 μL of 2.5 mM Rot (dissolved in trilaurin) into the vitreous chamber5. After model establishment, male Rot-induced LHON mice were randomly divided into five groups. Mice receiving an intravitreal injection of 2 μL PBS were designated as the Rot + PBS group. Mice treated intravitreally with Ctrl-EV-Mito, Lipo-EV-Mito, or Super-EV-Mito (1 μg EV-Mito protein/eye) were defined as the Rot + Ctrl-EV-Mito group, Rot + Lipo-EV-Mito group, and Rot + Super-EV-Mito group, respectively. Mice administered Ide orally at a daily dose of 60 mg/kg were designated as the Rot + Ide group. Age-matched male mice without Rot were included as the PBS group. For the optomotor test, a circular platform (6 cm radius) in black alternating with a white electric roller was settled to put the mouse. Mice were put to habituate the experiment for 5 min. For ERG measurement, mice were dark-adapted overnight, anesthetized, and placed on the recording platform. Corneal ring electrodes and subcutaneous needle electrodes in the skin and tail were positioned, and dark-adapted 0.01 ERG responses (flash intensity = 10 cd·s/m²) were recorded using the Espion Visual Electrophysiology System (Diagnosys LLC, USA). The mitochondrial complex I expression was analyzed using immunofluorescence. The eyes of mice were collected for H&E staining. After 2 weeks of the second time of administration, the eyes, heart, kidney, liver, lung and spleen of mice were collected for H&E staining. After blood was obtained, the blood was placed in an EP tube at r.t. for 1 h and centrifuged at 900 × g for 10 min to collect the serum. The serum samples were taken for ALT, AST, and BUN using the relevant assay kits. The two eyes of all groups' mice were injected with Super-EV-Mito (1 μg EV-Mito protein/eye). The primers of rat MT-ND4 DNA were as follows: Forward 5'-CGATCCATTATCCACCCCAC-3', Reverse 5'-TGTTGGGATTAGAGTGGCTTCG-3', and the primers of mouse Actin DNA were: Forward 5'-GATCACTCAGAACGGACACCAT-3', Reverse 5'-GGCTCATCAAATGCCCACA-3'. The relative copy numbers of rat MT-ND4 were quantified through 2−(∆∆Ct) by real-time PCR analysis (CFX connect, Bio-Rad, USA). 5F, data was normalized by cell numbers. Quantitative data in these experiments were presented as mean ± standard deviation (SD) from sample numbers (n). And the pairwise comparisons adopted t-test. Comparisons of all groups were analyzed using the SPSS 23.0. Statistical tests utilized for each experiment and the reproducibility of experiments were specified in the legends of figures. No data were excluded from the analysis. The ImageJ software (National Institutes of Health, USA) was used for quantitative analysis in fluorescence intensity for confocal images and Western blot. All the animal protocols and procedures were performed under the guidelines for human and responsible use of animals in research approved by the regional ethics committee of China Pharmaceutical University (2024-07-036) and JOINN Laboratories Co., Ltd. (Suzhou, China) (S-ACU24-0975). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The RNA sequencing data of rat primary mesenchymal stem cells and human GM10742 cells generated in this study have been deposited in the NCBI Sequence Read Archive (SRA) under accession code PRJNA1307022 and PRJNA1306802. Source data are provided with this paper. Zong, Y. et al. Mitochondrial dysfunction: mechanisms and advances in therapy. Murphy, M. P. & Hartley, R. C. Mitochondria as a therapeutic target for common pathologies. Wen, H. et al. Mitochondrial diseases: from molecular mechanisms to therapeutic advances. Wallace, D. C. & Lott, M. T. Leber hereditary optic neuropathy: exemplar of an mtDNA disease. & Jiang, H. L. Pathologically responsive mitochondrial gene therapy in an allotopic expression-independent manner cures Leber's hereditary optic neuropathy. Lyseng-Williamson, K. A. Idebenone: a review in Leber's hereditary optic neuropathy. & Lee, Y. S. Mitochondria as secretory organelles and therapeutic cargos. Nakai, R. & Varnum, S. Mitochondria transfer-based therapies reduce the morbidity and mortality of Leigh syndrome. Oral mitochondrial transplantation using nanomotors to treat ischaemic heart disease. Bertero, E., Maack, C. & Rourke, B. Mitochondrial transplantation in humans: “magical” cure or cause for concern?. Dave, K. M. et al. Mitochondria-containing extracellular vesicles (EV) reduce mouse brain infarct sizes and EV/HSP27 protect ischemic brain endothelial cultures. Saha, T. et al. Intercellular nanotubes mediate mitochondrial trafficking between cancer and immune cells. Malekpour, K., Hazrati, A., Soudi, S. & Hashemi, S. M. Mechanisms behind therapeutic potentials of mesenchymal stem cell mitochondria transfer/delivery. Song, X. et al. Encapsulation of single-walled carbon nanotubes with asymmetric pyrenyl-gemini surfactants. Han, J., Sul, J. H. & Jo, D. G. Engineered exosomes with a photoinducible protein delivery system enable CRISPR-Cas-based epigenome editing in Alzheimer's disease. & Chopp, M. Exosomes-beyond stem cells for restorative therapy in stroke and neurological injury. Harrell, C. R. & Jovicic, N. Mesenchymal stem cell-derived exosomes and other extracellular vesicles as new remedies in the therapy of inflammatory diseases. Phinney, D. G. et al. Mesenchymal stem cells use extracellular vesicles to outsource mitophagy and shuttle microRNAs. Yu, M. et al. BMSCs-derived mitochondria improve osteoarthritis by ameliorating mitochondrial dysfunction and promoting mitochondrial biogenesis in chondrocytes. Qi, L. Y., Wang, Y., Hu, L. F., Zhao, P. S. & Jiang, H. L. Enhanced nuclear gene delivery via integrating and streamlining intracellular pathway. & Jiang, H. L. Nanoengineered mesenchymal stem cell therapy for pulmonary fibrosis in young and aged mice. The power and potential of mitochondria transfer. Iron oxide nanoparticles augment the intercellular mitochondrial transfer-mediated therapy. Morrison, T. J. et al. Mesenchymal stromal cells modulate macrophages in clinically relevant lung injury models by extracellular vesicle mitochondrial transfer. Transfer of mitochondria from astrocytes to neurons after stroke. A boronic acid-rich dendrimer with robust and unprecedented efficiency for cytosolic protein delivery and CRISPR-Cas9 gene editing. & Ding, Y. Serum-tolerant polymeric complex for stem-cell transfection and neural differentiation. Enforced expression of mitochondrial calcium uniporter in donor T cells abolishes gvhd progression. & Hajnoczky, G. The roles of mitochondria in global and local intracellular calcium signalling. Katona, M. et al. Exploratory analysis of eating- and physical activity-related outcomes from a randomized controlled trial for weight loss maintenance with exercise and liraglutide single or combination treatment. Kaeser, P. S. et al. RIM proteins tether Ca2+ channels to presynaptic active zones via a direct PDZ-domain interaction. Chapman, E. R. Synaptotagmin: a Ca2+ sensor that triggers exocytosis?. Boudreau, L. H. et al. Platelets release mitochondria serving as substrate for bactericidal group IIA-secreted phospholipase A2 to promote inflammation. Ikeda, G. et al. Mitochondria-rich extracellular vesicles from autologous stem cell-derived cardiomyocytes restore energetics of ischemic myocardium. & Nikoskelainen, E. K. Mitochondrial DNA mutation associated with Leber's hereditary optic neuropathy. & Wei, W. Strand-selective base editing of human mitochondrial DNA using mitoBEs. Yuan, J., Liu, H., Zhang, H., Wang, T. & Li, Z. & Cai, X. Prussian blue nanozyme as a pyroptosis inhibitor alleviates neurodegeneration. Deng, C. L. et al. Photoreceptor protection by mesenchymal stem cell transplantation identifies exosomal MiR-21 as a therapeutic for retinal degeneration. & Martin, K. R. Molecular mechanisms mediating retinal reactive gliosis following bone marrow mesenchymal stem cell transplantation. Mouse mtDNA mutant model of Leber hereditary optic neuropathy. Klopstock, T. et al. Persistence of the treatment effect of idebenone in Leber's hereditary optic neuropathy. The coupling mechanism of mammalian respiratory complex I. & Yagi, T. Successful amelioration of mitochondrial optic neuropathy using the yeast NDI1 gene in a rat animal model. Guo, Y. T., Li, D. & Li, D. Visualizing intracellular organelle and cytoskeletal interactions at nanoscale resolution on millisecond timescales. Zhou, J. et al. Astrocytic LRP1 enables mitochondria transfer to neurons and mitigates brain ischemic stroke by suppressing ARF1 lactylation. This work was also supported by Natural Science Foundation of Jiangsu Province (BK20231016—Y.W., China), the Jilin Provincial Foundation of Changbai Talent Outstanding Team (20241000—X.W.C. We would like to thank Xiaonan Ma of China Pharmaceutical University (Nanjing, China) for providing technical assistance of Carl Zeiss LSM 800 on the Public Experimental Platform. These authors contributed equally: Yi Wang, Hao-Yuan Yu. State Key Laboratory of Natural Medicines, Department of Pharmaceutics, China Pharmaceutical University, Nanjing, China Yi Wang, Hao-Yuan Yu, Zi-Juan Yi, Lian-Yu Qi, Jing-Song Yang, Hai-Xin Xie, Min Zhao, Na-Hui Liu, Jia-Qi Chen, Tian-Jiao Zhou, Lei Xing & Hu-Lin Jiang Jilin Provincial Key Laboratory of Stress and Cardiovascular Disease, Department of Cardiology and Hypertension, Yanbian University Hospital, Yanji, China Joint International Research Laboratory of Target Discovery and New Drug Innovation, MOE, China Pharmaceutical University, Nanjing, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar provided financial support for this work, and all authors edited the manuscript. The authors declare no competing interests. Nature Communications thanks Jonathan Brestoff and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. Super mitochondria-enriched extracellular vesicles enable enhanced mitochondria transfer. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251027023751.htm'>Scientists just found a surprising twist in Earth's extinction story</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 12:34:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For years, many scientists have warned that Earth is undergoing another mass extinction, with species disappearing faster than ever before. Their analysis shows that over the past five centuries, extinctions among plants, arthropods, and land vertebrates actually reached their highest point about a century ago and have been declining since. The study also found that earlier extinction waves were driven mostly by invasive species on islands, whereas today's greatest danger to wildlife is the loss of natural habitats. The researchers argue that predictions of an ongoing mass extinction may rely on oversimplified or outdated assumptions. Many of these projections treat past extinction patterns as if they apply equally to modern times, ignoring how the main causes of species loss have shifted over history. "We discovered that the causes of those recent extinctions were very different from the threats species are currently facing," said Wiens, professor of ecology and evolutionary biology. "This makes it problematic to extrapolate these past extinction patterns into the future, because the drivers are rapidly changing, particularly with respect to habitat loss and climate change." Saban and Wiens noted that most of what we know about species loss comes from recent centuries, yet many studies have assumed those patterns predict future risks. In reality, the factors leading to extinction are far from uniform across time, habitats, or taxonomic groups. "To our surprise, past extinctions are weak and unreliable predictors of the current risk that any given group of animals or plants is facing," said lead author Saban, who recently graduated from the U of A and is currently a doctoral student at Harvard University. Many of the recorded extinctions occurred on isolated islands, including the Hawaiian Islands, where invasive species introduced by humans -- such as rats, pigs, and goats -- wiped out native wildlife. On continental regions, however, most extinctions took place in freshwater environments. "It just means that past extinctions do not reflect current and future threats." "Biodiversity loss is a huge problem right now, and I think we have not yet seen the kinds of effects that it might have," she said. "But it's important that we talk about it with accuracy, that our science is rigorous in how we're able to detail these losses and prevent future ones." "We show that extinction rates are not getting faster towards the present, as many people claim, but instead peaked many decades ago," Wiens said. For some groups, such as arthropods and plants and land vertebrates, extinction rates have actually declined over the last 100 years, notably since the early 1900s, he added. One of the reasons for declining extinction rates "is many people are working hard to keep species from going extinct. And we have evidence from other studies that investing money in conservation actually works." "By looking at the data in this way, we hope that our study helps inform our overall understanding of biodiversity loss and how we can come up with better ways to address it." Scientists Discover Brain Chemical Linked to Depression and Suicidal Thoughts The Universe “Will End in a Big Crunch,” Physicists Warns Brain's “Pain Map” Discovered: Scientists Find Opioid-Free Path to Relief Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03519-1'>This ‘minor' bird flu strain has potential to spark human pandemic</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 12:31:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Attention has been focused on avian influenza virus H5N1 in the past few years, but scientists are concerned about the spread of other bird viruses.Credit: Ralf Hirschberger/AFP via Getty In the past few years, surveillance has been focused on the avian influenza virus H5N1, which has spread across most continents and can cause severe disease and death in people. In North America, the virus is also spreading among dairy cows. H9N2 might be more prevalent than we realize, says Michelle Wille, who studies bird flu at the Peter Doherty Institute for Infection and Immunity in Melbourne. Infections are probably being missed because they do not result in severe infection or hospitalization in people, or because people are more commonly tested for the H5N1 instead, she adds. The modern version also showed improved binding to various receptors on human cells. The virus would have to undergo several more changes before it could cause sustained transmission between people, says Wille. If bird flu sparks a human pandemic, your past immunity could help How Napoleon's army met its doom: DNA reveals surprise illnesses had a role Japan declares a flu epidemic — what this means for other nations The rise of ‘nightmare bacteria': antimicrobial resistance in five charts Birds' intruder alert hints at how sounds took on new meanings European bats capture migrating birds and eat them on the wing The search for mutations that sperm acquire as men age Japan declares a flu epidemic — what this means for other nations SLS invites applications for multiple tenure-track/tenured faculty positions at all academic ranks. SLS invites applications for multiple tenure-track/tenured faculty positions at all academic ranks. We are seeking outstanding scientists to lead vigorous independent research programs focusing on all aspects of chemical biology including... If bird flu sparks a human pandemic, your past immunity could help An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41929-025-01429-z'>Single-molecule reaction mapping uncovers diverse behaviours of electrocatalytic surface Pd–H intermediates</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 11:08:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Many vital electrocatalytic transformations hinge on reactive surface metal–hydrogen intermediates (M–H*), yet the low concentration and transient nature of such intermediates present formidable challenges to in-depth investigation. Here we use single-molecule super-resolution reaction imaging to directly probe surface palladium–hydrogen (Pd–H*) intermediates on individual palladium nanocubes during electrocatalytic hydrogen evolution. Our approach visualizes hydrogen spillover from palladium to the surrounding substrate surface over hundreds of nanometres away and dissects substantial inter- and intraparticle heterogeneity. Through Gaussian-broadening kinetic analysis, we reveal that ensemble-averaged measurements systematically overestimate the stability of Pd–H*. Moreover, we resolve three subpopulations of palladium nanocubes with distinct reactivity features, uncovering critical correlations between intermediate stability, hydrogenation reactivity and transition-state properties. Our findings highlight the necessity of single-particle resolution for capturing the intrinsic complexity of electrocatalysts; our approach is also broadly applicable to interrogate surface-reactive intermediates across a wide array of electrocatalytic pathways. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Raw data comprising a large amount of movie files supporting the findings of this study are available from the authors upon reasonable request. Source data are provided with this paper. Computer codes for image and data analysis were previously published as cited in Methods and Supplementary Information. MATLAB codes for fitting the potential dependence of reaction rate are provided as Supplementary Software 1. Liu, C., Chen, F., Zhao, B.-H., Wu, Y. & Zhang, B. Electrochemical hydrogenation and oxidation of organic species involving water. Article PubMed Google Scholar Oumellal, Y., Rougier, A., Nazri, G. A., Tarascon, J. M. & Aymard, L. Metal hydrides for lithium-ion batteries. Article PubMed Google Scholar Darmadi, I., Nugroho, F. A. & Langhammer, C. High-performance nanostructured palladium-based hydrogen sensors—current limitations and strategies for their mitigation. Google Scholar Chaplin, B. P. et al. Critical review of Pd-based catalytic treatment of priority contaminants in water. Google Scholar Adams, B. D. & Chen, A. The role of palladium in a hydrogen economy. Google Scholar Jung, O., Jackson, M. N., Bisbey, R. P., Kogan, N. E. & Surendranath, Y. Innocent buffers reveal the intrinsic pH- and coverage-dependent kinetics of the hydrogen evolution reaction on noble metals. Google Scholar Ramaker, D. E. & Roth, C. Nature of the intermediate binding sites in hydrogen oxidation/evolution over Pt in alkaline and acidic media. Google Scholar Boosting formate production in electrocatalytic CO2 reduction over wide potential window on Pd surfaces. Article PubMed Google Scholar Mu, Z. et al. Critical role of hydrogen sorption kinetics in electrocatalytic CO2 reduction revealed by on-chip in situ transport investigations. Google Scholar Vogt, C. et al. Unravelling structure sensitivity in CO2 hydrogenation over nickel. Google Scholar Guo, S. et al. Insights into nitrate reduction over indium-decorated palladium nanoparticle catalysts. Google Scholar Zalineeva, A., Baranton, S., Coutanceau, C. & Jerkiewicz, G. Octahedral palladium nanoparticles as excellent hosts for electrochemically adsorbed and absorbed hydrogen. Google Scholar Martin, M. H. & Lasia, A. Study of the hydrogen absorption in Pd in alkaline solution. Google Scholar Syrenova, S. et al. Hydride formation thermodynamics and hysteresis in individual Pd nanocrystals with different size and shape. Article PubMed Google Scholar Tiburski, C., Nugroho, F. A. & Langhammer, C. Optical hydrogen nanothermometry of plasmonic nanoparticles under illumination. Google Scholar Tang, M. L., Liu, N., Dionne, J. & Alivisatos, A. P. Observations of shape-dependent hydrogen uptake trajectories from single nanocrystals. Article PubMed Google Scholar Chen, J. et al. Measuring the activation energy barrier for the nucleation of single nanosized vapor bubbles. Google Scholar Polo-Garzon, F. et al. Neutron scattering investigations of hydride species in heterogeneous catalysis. Article PubMed Google Scholar Manchester, F. D., San-Martin, A. & Pitre, J. M. The H–Pd (hydrogen–palladium) system. Google Scholar et al. High-spatial-resolution mapping of catalytic reactions on single particles. Article PubMed Google Scholar & Knappenberger, K. L. Jr. Distinguishing single-metal nanoparticles with subdiffraction spatial resolution using variable-polarization Fourier transform nonlinear optical microscopy. Google Scholar Hao, R., Fan, Y., Anderson, T. J. & Zhang, B. Imaging single nanobubbles of H2 and O2 during the overall water electrolysis with single-molecule fluorescence microscopy. Article PubMed Google Scholar Liu, X., Chen, T., Song, P., Zhang, Y. & Xu, W. Single-molecule nanocatalysis of Pt nanoparticles. Google Scholar Chen, T. et al. Single-molecule nanocatalysis reveals facet-dependent catalytic kinetics and dynamics of pallidium nanoparticles. Google Scholar Chen, T., Zhang, Y. & Xu, W. Size-dependent catalytic kinetics and dynamics of Pd nanocubes: a single-particle study. Article PubMed Google Scholar Zhou, Y., Wei, W., Su, H. & Wang, W. Sensitively fluorescent detection of H2 with resazurin hydrogenation reactions catalyzed by Pd/C nanocomposites. Google Scholar Narayan, T. C. et al. Direct visualization of hydrogen absorption dynamics in individual palladium nanoparticles. Google Scholar Sytwu, K. et al. Visualizing facet-dependent hydrogenation dynamics in individual palladium nanoparticles. Article PubMed Google Scholar Antolini, E. Palladium in fuel cell catalysis. Google Scholar Lasia, A. Mechanism and kinetics of the hydrogen evolution reaction. Google Scholar Kinetic criteria for the mechanism of the hydrogen evolution reaction. Google Scholar Shinagawa, T., Garcia-Esparza, A. T. & Takanabe, K. Insight on Tafel slopes from a microkinetic analysis of aqueous electrocatalysis for energy conversion. Niu, W. et al. Seed-mediated growth of nearly monodisperse palladium nanocubes with controllable sizes. Goyal, A., Marcandalli, G., Mints, V. A. & Koper, M. T. M. Competition between CO2 reduction and hydrogen evolution on a gold electrode under well-defined mass transport conditions. Chierchie, T., Mayer, C. & Lorenz, W. J. Structural changes of surface oxide layers on palladium. Doneux, T., Bouffier, L., Goudeau, B. & Arbault, S. Coupling electrochemistry with fluorescence confocal microscopy to investigate electrochemical reactivity: a case study with the resazurin–resorufin fluorogenic couple. Xu, W. et al. Single-molecule electrocatalysis by single-walled carbon nanotubes. Wei, W. et al. Accessing the electrochemical activity of single nanoparticles by eliminating the heterogeneous electrical contacts. Zhao, J. et al. Exploring the strain effect in single particle electrochemistry using Pd nanocrystals. Shen, L. et al. Shape-controlled hydrogen evolution reaction of single Pd nanocrystals. Hao, R., Fan, Y., Howard, M. D., Vaughan, J. C. & Zhang, B. Imaging nanobubble nucleation and hydrogen spillover during electrocatalytic water splitting. Karim, W. et al. Catalyst support effects on hydrogen spillover. Li, J., Ma, Y., Ho, J. C. & Qu, Y. Hydrogen spillover phenomenon at the interface of metal-supported electrocatalysts for hydrogen evolution. Liu, Y. et al. Direct observation of accelerating hydrogen spillover via surface-lattice-confinement effect. Tang, Z. et al. Interfacial hydrogen spillover on Pd–TiO2 with oxygen vacancies promotes formate electrooxidation. ACS Energy Lett. Benck, J. D., Pinaud, B. & Jaramillo, T. F. Substrate selection for fundamental studies of electrocatalysts and photoelectrodes: inert potential windows in acidic, neutral, and basic electrolyte. Wilke, S., Hennig, D., Löber, R., Methfessel, M. & Scheffler, M. Ab initio study of hydrogen adsorption on Pd(100). Behm, R. J., Christmann, K. & Ertl, G. Adsorption of hydrogen on Pd(100). Lu, L. et al. Robust removal of ligands from noble metal nanoparticles by electrochemical strategies. A., Krishnan, Y., Buchauer, F. L., Hansen, H. A. & Hjelm, J. Unifying the ORR and OER with surface oxygen and extracting their intrinsic activities on platinum. Arblaster, J. W. Crystallographic properties of palladium. Han, K. S., Liu, G., Zhou, X., Medina, R. E. & Chen, P. How does a single Pt nanocatalyst behave in two different reactions? A single-molecule study. Montejano, H. A., Gervaldo, M. & Bertolotti, S. G. The excited-states quenching of resazurin and resorufin by p-benzoquinones in polar solvents. Zhao, M. et al. Long-range enhancements of micropollutant adsorption on metal-promoted photocatalysts. et al. Concentration- and chromosome-organization-dependent regulator unbinding from DNA for transcription regulation in living cells. MATLAB code package: iQPALM (image-based quantitative photo-activated localization microscopy). figshare https://figshare.com/articles/code/MATLAB_code_package_iQPALM_image-based_quantitative_photo-activated_localization_microscopy_/12642617 (2020). The research is supported by the NSF (grant CHE-2303933). Preliminary studies earlier were supported by the Army Research Office (grant number W911NF2310105) and the US Department of Energy, Office of Science, Basic Energy Sciences, Catalysis Science Program (grant number DE-SC0004911). This work made use of the Cornell Center for Materials Research Shared Facilities which are supported through the NSF MRSEC programme (DMR-1719875). We thank K. Termini for fabricating custom-designed parts. Consulting, Boston, MA, USA Ming Zhao Present address: Department of Materials Science and Engineering, National University of Singapore, Singapore, Singapore Present address: Department of Chemical Engineering and Catalysis Science and Technology Institute, University of Michigan, Ann Arbor, MI, USA Present address: Department of Biomedical Engineering, City University of Hong Kong, Hong Kong, China Department of Chemistry and Chemical Biology, Cornell University, Ithaca, NY, USA Wenjie Li, Muwen Yang, Zhiheng Zhao, Ming Zhao, Rong Ye, Bing Fu & Peng Chen Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and performed experiments, synthesized particles, derived kinetic models, wrote computer codes and analysed data. contributed to imaging experiments. contributed to reagent preparation. contributed to materials synthesis. analysed results and wrote the paper. All authors contributed to discussion. Correspondence to Peng Chen. The authors declare no competing interests. Nature Catalysis thanks Qianjin Chen, Ning Fang and Florian Meirer for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Li, W., Yang, M., Zhao, Z. et al. Single-molecule reaction mapping uncovers diverse behaviours of electrocatalytic surface Pd–H intermediates. DOI: https://doi.org/10.1038/s41929-025-01429-z Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02857-2'>Improved reconstruction of single-cell developmental potential with CytoTRACE 2</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 10:44:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Here we introduce CytoTRACE 2, an interpretable deep learning framework for predicting absolute developmental potential from single-cell RNA sequencing data. Across diverse platforms and tissues, CytoTRACE 2 outperformed previous methods in predicting developmental hierarchies, enabling detailed mapping of single-cell differentiation landscapes and expanding insights into cell potency. All cells, from the fertilized egg to its mature progeny, are hierarchically organized in multicellular life. Each cell has distinct potency, or ability to differentiate into specialized cell types, ranging from totipotent (capable of generating an entire organism) and pluripotent (capable of generating all adult cells) to multipotent, oligopotent, unipotent and differentiated cells, each with increasingly restricted developmental potential1 (Fig. While lineage tracing, functional transplantation assays and single-cell genomics have expanded our understanding of cell potency2, there remains a need for interpretable methods that can learn developmental programs, predict potency states and generate insights applicable to regenerative and cancer biology. a, Overview of cell potency across six developmental categories. b, Summary of the 33-dataset single-cell potency atlas. d, CytoTRACE 2 performance across six broad potency categories in training and held-out test sets, with mean potency scores shown for each standardized phenotype–dataset pair (circles). e, CytoTRACE 2 performance across 17 evaluable granular potency levels in held-out test data. Points denote mean potency score per phenotype; large circles indicate the median across these points for each granular potency level. Thick black lines (x axis) separate broad potency categories. A linear regression line with 95% confidence band is shown. f, Same as e, but using a leave-clade-out strategy, where each of 19 developmentally distinct clades (b) was held out during training. For d–f, concordance with ground truth was assessed using weighted Kendall correlation (τ) applied to single cells, with significance assessed by two-sided z-test. Box plots show medians, quartiles and 1.5 × interquartile range (IQR). g, Uniform Manifold Approximation and Projection (UMAP) of three held-out datasets showing ground truth (top), CytoTRACE 2 (middle) and CytoTRACE 1 (bottom). h, Violin plots comparing nine methods for reconstructing 57 developmental systems. P values were calculated by two-sided Wilcoxon tests against CytoTRACE 2; **P < 0.01; ****P < 0.0001. i, Performance comparison with eight previous methods and 18,706 gene sets in the test set (left) and Tabula Sapiens (right) using weighted τ to assess absolute (six broad potency levels) and relative order (median correlation across individual trajectories). 3), a computational method for predicting cellular maturity from single-cell RNA sequencing (scRNA-seq) data, based on the number of genes expressed per cell. However, like other trajectory inference methods4,5,6,7,8, CytoTRACE 1 provides predictions that are dataset-specific, making it difficult to unify results across datasets and contextualize them within the broader framework of cellular potency. To overcome these challenges, we developed CytoTRACE 2, an interpretable deep learning framework for determining single-cell potency categories and absolute developmental potential from scRNA-seq data. Unlike most deep learning methods9, CytoTRACE 2 learns multivariate gene expression programs that are readily interpretable and enable accurate predictions of developmental potential. Moreover, it suppresses batch and platform-specific variation through multiple mechanisms, including competing representations of gene expression and training set diversity (Methods). Our approach uncovers cross-tissue correlates of cell potency and highlights the value of interpretable deep learning for characterizing single-cell developmental states in health and disease (https://cytotrace2.stanford.edu). To develop CytoTRACE 2, we curated an extensive atlas of human and mouse scRNA-seq datasets with experimentally validated potency levels, spanning 33 datasets, nine platforms, 406,058 cells and 125 standardized cell phenotypes (Fig. Phenotypes were grouped into six broad potency categories—totipotent, pluripotent, multipotent, oligopotent, unipotent and differentiated—and further subdivided into 24 granular levels based on expected developmental order from lineage tracing and functional assays (Fig. A training set of 93 cell phenotypes from 16 tissues and 13 studies was used to develop the model, with the remaining data reserved for performance evaluation (Fig. CytoTRACE 2 decodes developmental potential using a novel, explainable deep learning architecture called a gene set binary network (GSBN). Inspired by binarized neural networks10, GSBNs assign binary weights (0 or 1) to genes, identifying highly discriminative gene sets that define each potency category (Fig. Multiple gene sets can be learned for each potency group, and the informative genes driving model predictions can be easily extracted—an advantage over conventional deep learning architectures. As such, CytoTRACE 2 provides two key outputs for each single-cell transcriptome: (1) the potency category with maximum likelihood and (2) a continuous ‘potency score' generated by integrating GSBN predictions across potency categories and calibrating the range from 1 (totipotent) to 0 (differentiated) (Fig. Based on the assumption that transcriptionally similar cells occupy related differentiation states, CytoTRACE 2 also leverages Markov diffusion combined with a nearest neighbor approach to smooth individual potency scores (Extended Data Fig. We used two definitions of development ordering: ‘absolute order', which compares predictions to known potency levels across datasets, and ‘relative order', which ranks cells within each dataset from least to most differentiated (Extended Data Fig. The agreement between known and predicted developmental orderings was quantified using weighted Kendall correlation to ensure balanced evaluation and minimize bias (Supplementary Table 5). We started by evaluating model hyperparameters through cross-validation and observed minimal performance variation across a wide range of values (Extended Data Fig. Based on this, we selected stable hyperparameters and retrained the model. On the training data, we demonstrated that CytoTRACE 2 achieves high accuracy in distinguishing absolute potency for broad potency labels (Fig. To validate our approach, we next extended our analysis to unseen data, comprising 14 held-out datasets spanning nine tissue systems, seven platforms and 93,535 evaluable cells. Performance on broad and granular potency labels was consistently high in testing (Fig. 1d,e) and robust to differences in species, tissues, platforms or phenotypes that were absent during training (Extended Data Fig. To rigorously assess generalizability, we retrained CytoTRACE 2 on different subsets of the potency atlas, including random train–test splits and scenarios where distinct developmental systems, termed ‘clades', were held out from training. In all cases, results were well correlated with ground truth (Fig. We also found that CytoTRACE 2 is resistant to moderate annotation errors and performs reliably under practical data limitations (Extended Data Fig. A key advantage of CytoTRACE 2 is its ability to predict absolute developmental potential on a continuous scale from 1 (totipotent) to 0 (differentiated), which enables cross-dataset comparisons and avoids imposing a developmental order where none exists. For example, unlike its predecessor, CytoTRACE 2 corroborated a pluripotency program in cranial neural crest cell precursors11 and correctly distinguished datasets with and without immature cells12,13 (Fig. It also outperformed other methods3,14,15,16,17,18,19,20 in ordering mouse single-cell transcriptomes from six datasets2,21,22,23,24,25 across 62 developmental time points (Extended Data Fig. 5a–c) and accurately captured the progressive decline in potency across 258 evaluable phenotypes during mouse development (Extended Data Fig. 5d,e)—without requiring data integration or batch correction. CytoTRACE 2 potency predictions also aligned with known leukemic stem cell signatures in acute myeloid leukemia (Extended Data Fig. 6a)26 and identified known multilineage potential in oligodendroglioma27, highlighting its applicability to cancer (Extended Data Fig. Next, we benchmarked CytoTRACE 2 against multiple strategies for cell potency classification and developmental hierarchy inference (Supplementary Table 11). CytoTRACE 2 outperformed eight state-of-the-art machine learning methods28,29,30,31,32 for cell potency classification in 33 datasets, achieving a higher median multiclass F1 score and lower mean absolute error (Extended Data Fig. Moreover, it surpassed eight developmental hierarchy inference methods for cross-dataset (absolute) and intra-dataset (relative) performance3,14,15,16,17,18,19,20, demonstrating over 60% higher correlation, on average, for reconstructing relative orderings in 57 developmental systems, including data from Tabula Sapiens33 (Fig. Similar results were observed when comparing CytoTRACE 2 against nearly 19k annotated gene sets34,35,36 (Fig. 1i and Supplementary Table 13) and scVelo5, a generalized RNA velocity model for predicting future cell states (Extended Data Fig. Previous genomic studies of stemness largely focused on pluripotency, with limited insight into other potency states. Given the inherent interpretability of our GSBN design, we next explored the molecular programs driving potency predictions (Fig. Across our potency atlas, GSBN modules produced a cohesive gradient of differentiation states (Fig. The top-ranking genes showed conserved signatures across species, platforms and developmental clades, identifying both positive and negative correlates of cell potency (Fig. a, Schematic for characterizing CytoTRACE 2 gene sets and feature importance. b, UMAP of gene set expression levels in training–test sets, aggregated in a 0.5 × 0.5 grid, colored by CytoTRACE 2 (top) or ground truth potency (bottom). markers per potency category, shown across 237 pseudo-bulks aggregated by phenotype, species and platform from training–test sets. d, Overview of a CRISPR knockout (KO) screen assessing in vivo differentiation effects in hematopoietic stem cells (HSCs)38. e, Enrichment of top CytoTRACE 2 multipotency markers among genes whose knockout promotes or inhibits HSC differentiation (from d), using GSEA. f, GSEA of 537 pathways in genes ranked by multipotency scores, highlighting ‘cholesterol metabolism'. g, Top: overview of UFA pathways, inspired by ref. h, Single-sample GSEA of UFA genes across 237 pseudo-bulk samples, colored by tissue type as in c. ****P < 0.0001 (one-sided permutation testing). Box plots show medians, quartiles and 1.5 × IQR. i, qPCR of UFA genes in FACS-purified mouse hematopoietic subsets (n = 3), normalized to HSC/MPP; Actb as internal control. Higher magnification views (bottom) highlight boxed regions. Cell boundaries were visualized with E-cadherin immunostaining; asterisks mark representative Lgr5+ crypt base columnar (CBC) cells. Images in a, d, g, i, j were created using BioRender.com. Given these results, we hypothesized that CytoTRACE 2 might enrich for key potency-specific factors. To further explore this hypothesis, we analyzed data from a large-scale CRISPR screen, in which ~7,000 genes in multipotent mouse hematopoietic stem cells were individually knocked out and assessed for developmental consequences in vivo38 (Fig. Among the 5,757 genes overlapping CytoTRACE 2 features, the top 100 positive multipotency markers were enriched for genes whose knockout promotes differentiation, whereas the top 100 negative markers were enriched for genes whose knockout inhibits differentiation (Q = 0.04; Fig. This trend was consistent across different numbers of top markers and highly specific for multipotency, underscoring the fidelity of learned potency representations (Extended Data Fig. To more deeply analyze multipotency in mouse and human tissues and explore the potential of CytoTRACE 2 for biomarker discovery, we next applied pathway enrichment analysis to genes ranked by feature importance. Remarkably, cholesterol metabolism emerged as a leading multipotency-associated pathway (Fig. Within this pathway, three genes related to unsaturated fatty acid (UFA) synthesis (Fads1, Fads2 and Scd2) were among the top-ranking markers (Fig. 2h; train–test area under the curve (AUC) values of 0.87 and 0.92, respectively). To experimentally confirm these findings, we performed quantitative PCR on mouse hematopoietic cells sorted into multipotent, oligopotent, and differentiated subsets (Fig. 10a,b) and multiplexed in situ mRNA imaging on mouse intestinal epithelium co-stained with multipotency markers, Lgr539 and Fgfbp140 (Fig. While fatty acid metabolism has been linked to stem cell biology41, no study has specifically attributed lipid metabolism genes to distinct potency levels. In summary, CytoTRACE 2 is an interpretable deep learning framework that predicts cell potency and continuous differentiation states from scRNA-seq data. Unlike previous methods, it links stemness and pseudotime to absolute developmental potential, offering cross-dataset compatibility and transparency into the molecular profiles driving its predictions. Like all supervised machine learning approaches, CytoTRACE 2 depends on the quality and breadth of its training data, although robust results were observed across diverse training–test splits, and moderate labeling variation was well tolerated. Performance may decline when analyzing cells with very low RNA content or number of expressed genes (Extended Data Fig. While some phenotypes were misclassified in held-out data, absolute errors remained low and outcompeted existing methods. Finally, although the current model is trained on human and mouse data, ortholog mapping may expand its applicability to other species. Given its demonstrated advantages, we anticipate that CytoTRACE 2 will have immediate utility for improving our understanding of cell potency, with implications for the identification of novel biomarkers and therapeutic targets in diseases where altered developmental hierarchies play a role. All animal procedures were performed in compliance with ethical regulations and conducted according to a protocol approved by the Stanford University Administrative Panel for Laboratory Animal Care committee (protocol no. Developmental potency reflects a cell's capacity to differentiate into various cell types, with six widely recognized categories in stem cell biology: totipotency, pluripotency, multipotency, oligopotency, unipotency, and differentiated (Fig. These broad classifications are based on decades of research, including lineage tracing, transplantation and colony-formation experiments across multiple tissues and species. Each category represents a progressively restricted ability to generate downstream cell types, from totipotent cells capable of forming all embryonic and extra-embryonic lineages to unipotent cells restricted to producing a single mature cell type; however, as developmental potential exists on a continuum, we also devised a more granular classification system, as described in Supplementary Note and Supplementary Tables 2 and 3. Of note, classically defined potency levels are not directly annotated in publicly available scRNA-seq datasets. As part of this selection process, we applied the following inclusion and exclusion criteria to enhance experimental rigor: Only functionally validated developmental states supported by lineage tracing or transplantation assays were considered for analysis. Datasets with transient cell changes, such as from metabolic activation or suppression, cell cycle transitions or environmental perturbations were excluded, as these do not represent durable developmental processes. Datasets with irreconcilable technical batches resulting in major imbalances in the number of cells per phenotype were excluded. Single-nucleus RNA sequencing datasets were excluded, as they do not capture cytoplasmic RNA and include immature transcripts. Among datasets satisfying these conditions, author-supplied cell type annotations were mapped to one of six broad potency categories (totipotent, pluripotent, multipotent, oligopotent, unipotent and differentiated) or not evaluable using established definitions (‘Potency annotation scheme', Supplementary Note). These potency categories were further subdivided into 24 granular categories, ranging from 1 (least differentiated) to 24 (most differentiated) (Supplementary Tables 2 and 3). Cellular phenotypes were hierarchically grouped into these categories based on potency, developmental timing and sequence, and self-renewal capacity. Where possible, we also examined single-cell developmental states in a dataset-specific manner and without regard to potency categories, as previously described3. Such ‘relative' orderings, most of which were obtained from Gulati et al.3, ranged from 1 (least differentiated) to N (most differentiated) in a given dataset, and exceeded the number of resolvable potency categories in some datasets (Supplementary Table 4), permitting a more granular assessment Our comprehensive potency atlas catalogs experimentally confirmed cell states and their corresponding potency levels, providing a structured reference for model training and validation. This format allows for consistent annotation and comparison across datasets. For full details of potency annotations and associated rationale, see ‘Potency annotation scheme' (Supplementary Note) and Supplementary Tables 2–4. Using the abovementioned criteria, we assembled a 33-dataset potency atlas (Fig. 1b), from which we selected a training cohort consisting of seven human and 12 mouse scRNA-seq datasets from 13 studies (Supplementary Table 1). We ensured that all six broad potency categories were represented in both species along with a diverse array of biological (for example, tissue types) and technical characteristics (for example, sequencing platforms). As part of this effort, and to align with precedent in the field, we incorporated all human and mouse scRNA-seq datasets (n = 13) with annotatable potency categories analyzed by Gulati et al.3. To broadly cover tissue types, we also included cell phenotypes from the Tabula Muris scRNA-seq atlas43 for which potency categories could be determined (15 tissue types and 43 phenotypes). The resulting training cohort encompasses 312,523 cells, 16 tissue types, 93 phenotypes and six scRNA-seq platforms (Fig. The remaining datasets served as a held-out test cohort, which mirrors the training cohort with respect to species representation in each broad potency category (Supplementary Table 1). Consisting of three human and 11 mouse scRNA-seq datasets from 14 studies, the test cohort spans 93,535 cells, 73 phenotypes, nine tissue types and seven scRNA-seq platforms, including two tissue types and 21 phenotypes that were absent from training (Fig. To augment these data, we annotated potency categories in 459,320 evaluable cells from Tabula Sapiens, a multi-tissue scRNA-seq atlas from postmortem human donor biopsies33 (Supplementary Table 1); however, given the confounding influence of postmortem intervals on human tissue messenger RNA levels44, we hypothesized that Tabula Sapiens might exhibit reduced data quality. To test this, we calculated the ratio of mitochondrial reads to total reads (MTR) within each single-cell transcriptome as a proxy for overall data quality. Accordingly, we omitted Tabula Sapiens from the primary test cohort and evaluated it as a secondary benchmark in Fig. Author-supplied phenotypes in Tabula Sapiens with fewer than five cells in a tissue–platform pair were excluded from further analysis. Collectively, these ground truth datasets with newly annotated potency levels represent a unique community resource for systematic characterization of absolute developmental states and their molecular programs in humans and mice. Depending on platform, all scRNA-seq expression matrices were normalized to transcripts per million (TPM) or counts per million (CPM) as appropriate. These data can also be interactively explored at https://cytotrace2.stanford.edu. For cells with identical phenotypes but different author-supplied labels, we unified the annotations (Supplementary Table 3). For example, ‘HSC-MPPs' from ‘HSC development (Smart-seq2)' and ‘Hematopoietic stem cell progenitor (HSCP)' from ‘HSPCs (C1)' were annotated as ‘Hematopoietic stem and early progenitor'. To balance the representation of cells from distinct lineages within a given broad potency category, we also re-annotated related cell subsets sharing a common parental phenotype. This was crucial when training CytoTRACE 2 as the probability of sampling individual cells was weighted based on phenotype. In this way, each major phenotype contributed equally during model training regardless of the number of evaluable cells, mitigating the chance of overweighting and overfitting (see ‘Training and hyperparameter tuning' below). Existing RNA-based surrogates of cellular differentiation status have notable limitations for imputing absolute differentiation states and potency categories from scRNA-seq data. For example, the original CytoTRACE, termed CytoTRACE 1 in this work, employs gene counts as an unbiased strategy for identifying immature cells3. Despite the utility of this approach, gene counts are subject to dataset-specific biases, making them suboptimal for potency assessment. Measures based on transcriptional entropy and RNA velocity also suffer from dataset-specific biases, a nonspecific relationship to absolute differentiation status, or the requirement for continuous developmental processes within a narrowly defined time window4,5,14,15,16. Supervised machine learning models offer a potentially robust alternative to the abovementioned strategies when adequate training data are available; however, machine learning methods also face key challenges when applied to scRNA-seq data, including sparsity, high dimensionality and data heterogeneity encompassing both biological and technical variation. While deep learning is a promising subtype of machine learning, often achieving remarkable performance gains over other machine learning methods (especially in the presence of high complexity, noise and uncertainty) most existing architectures lack inherent interpretability, limiting their broad applicability. To address these challenges, we designed a novel deep learning framework that can handle the complexities of single-cell potency assessment while achieving direct biological interpretability. Unlike recent methods46,47 that decompose single-cell expression data into a combination of previously known and simultaneously learned new gene programs, our approach, termed a GSBN, is anchored to known phenotypic states but not known gene sets. As such, GSBNs have the flexibility to discover new gene programs for known phenotypic states, such as potency categories, from scRNA-seq data. As part of their design, GSBNs are highly robust and fully interpretable, meaning they can be directly interrogated to extract meaningful markers for each phenotypic class of interest across datasets, platforms and tissues. CytoTRACE 2 consists of five high-level components, schematically depicted in Fig. GSBNs: identification of interpretable potency-associated gene sets for each potency category. Enrichment assessment: evaluation of gene set activation levels in single cells. Integration of scores: integration of gene set activation levels, both within and across gene set binary networks. Postprocessing: leveraging transcriptional covariance and uncertainty in model predictions to smooth single-cell potency scores and produce the final output. Among these five components, GSBNs, enrichment assessment and integration of scores constitute the CytoTRACE 2 core model, a neural network architecture consisting of a shared input layer; a set of G GSBN modules, where G denotes the number of potency categories; and a shared output layer (Extended Data Fig. Within the core model, each GSBN module is trained to discriminate a single potency category and contains (1) a binary neural network (BNN) component, which encodes potency-associated gene sets and (2) downstream functions to calculate and integrate gene set enrichment scores (Fig. Notably, because weights in BNNs are constrained to binary rather than continuous values, BNNs also allow for more efficient computation and provide an implicit form of model regularization48. Let input scRNA-seq dataset X be an \(I\times C\) gene expression matrix over I genes and C cells. First, gene symbols in X are mapped and filtered using dictionary \({\mathbb{D}}\), a collection of gene symbols that harmonizes all HGNC (human) and MGI (mouse) identifiers supported by CytoTRACE 2 (‘Dictionary of input genes' below). Following this step, the resulting expression matrix, denoted \({{\bf{X}}}^{{\boldsymbol{{\prime} }}}\), consists of \(n=\mathrm{14,271}\) genes and C cells. As part of this process, any genes in \({{\bf{X}}}^{{\boldsymbol{{\prime} }}}\) not present in \({\bf{X}}\) through mapping are set to zero. In the second step, \({{\bf{X}}}^{{\boldsymbol{{\prime} }}}\) is converted into dual representations: for the first, it is normalized to CPM/TPM and log2-adjusted, yielding an \(N\times C\) matrix \({\bf{L}}\); for the second, it is mapped to rank space, yielding an \(N\times C\) matrix \({\bf{R}}\), with the genes of each single-cell transcriptome \({{\bf{X}}}_{c}^{{\boldsymbol{{\prime} }}}\) assigned relative integer rank such that rank 1 corresponds to the gene with highest expression. In tandem, these two representations provide an inherent regularization to model inputs. \({\bf{R}}\) and \({\bf{L}}\) are subsequently passed to the CytoTRACE 2 core model where they jointly constitute the model input layer. Inputs \({\bf{R}}\) and \({\bf{L}}\) are passed to each of \(G\) GSBN modules within the CytoTRACE 2 core model. These modules begin by thresholding \({\bf{R}}\) (Extended Data Fig. 1a) to learnable maximum rank \(\tau {\mathbb{\in }}{\mathbb{N}}\), yielding \(N\times C\) matrix \({\bf{T}}\): This rank trimming (see also ‘Model initialization and updates') enables calculation of the rank-based enrichment score, described in ‘Enrichment assessment' below. Next, within each GSBN module, \(M\) gene sets are learned in binary \(N\times M\) matrix \({{\bf{W}}}^{{\rm{B}}}\), where \(M{\mathbb{\in }}{\mathbb{N}}\) is prespecified and all entries \({{\bf{W}}}_{i,j}^{{\rm{B}}}\in \left\{\mathrm{0,1}\right\}\). \({{\bf{W}}}^{{\rm{B}}}\) constitutes the gene set selection layer of the CytoTRACE 2 core model; it has a continuous equivalent \({\bf{W}}\) used for model initialization and backpropagation (see also ‘Training and hyperparameter tuning'). At each forward iteration for model training, \({\bf{W}}\) undergoes binarization: where binarize denotes the following utility function: \({{\rm{Score}}}_{U}\) aggregates overall expression activity of a given gene set j in rank space whereas \({{\rm{Score}}}_{A}\) compares the average expression of genes in j versus background levels. By integrating both scores, each providing a different axis of information, CytoTRACE 2 can learn more complex expression patterns while also achieving additional regularization through enrichment score competition. The two scores are defined as follows. \({{\rm{Score}}}_{U}\) calculates the commonly used nonparametric UCell score49 for each gene set, or column of \({{\bf{W}}}^{{\rm{B}}}\). For each cell \(1\le k\le C\) and module gene set \(1\le j\le M\), \({{\rm{Score}}}_{A}\) implements a scoring system based on Seurat's AddModuleScore (AMS), computing the average expression of genes within a gene set subtracted by the aggregated expression of control, or background, feature sets50. To select background features, AMS groups genes into \({n}_{\mathrm{bins}}\) bins according to their average expression within a dataset. Then, for each gene, a ‘background' set of \({n}_{\mathrm{sample}}\) genes from the same average expression bin is sampled, ensuring that each gene is compared to other genes with similar average expression. Here, for computational efficiency and to avoid introducing a dependency on dataset composition, we use our entire curated training cohort (see ‘Single-cell potency atlas') as the ‘dataset' in which to rank genes by average expression. We then compute a constant set of background genes to use for each gene. We encode the mapping of genes to their background genes in the binary \(N\times N\) matrix \({\bf{G}}\), where each row represents a gene as used in a gene set, and the jth entry of row i is 1 if gene j is used as background for gene i, and 0 otherwise. In detail, we construct \({\bf{G}}\) as follows. We then rank the results and uniformly partition genes into \({n}_{\mathrm{bins}}=24\) bins of size \({s}_{\mathrm{bin}}\) according to rank, following the Seurat default50. Next, for each gene (each row of \({\bf{G}}\)), we randomly select without replacement a set of background genes, where the number of background genes follows a Gaussian distribution with mean \({\mu} = {n}_{\mathrm{sample}}\) and variance This approach provides an additional regularizing effect compared to constant selection of a uniform number of background genes per gene. Note that left-multiplying a gene set matrix \({{\bf{W}}}^{{\rm{B}}}\) by \({\bf{G}}\) maps the genes in the gene sets (columns) of \({{\bf{W}}}^{{\rm{B}}}\) to their corresponding background genes. Then, given \({\bf{G}}\), for each cell \(1\le k\le C\) and module gene set \(1\le j\le M\), The two resulting enrichment score matrices are subsequently concatenated into a single \(C\times 2M\) matrix \({\bf{K}}\): To transfer these enrichment scores into comparable spaces, CytoTRACE 2 standardizes each score across cells, yielding \(C\times 2M\) matrix \({{\bf{K}}}^{{\rm{norm}}}\). This standardization, implemented via torch.nn.BatchNorm1d from PyTorch v.2.0.0 with affine = False, tracks the mean and variance of each score during training. Once trained, the model applies these learned values, rather than dataset-specific values, for standardization at inference. To convert the gene set enrichment scores to a single score per cell per GSBN module, the normalized scores \({{\bf{K}}}^{{\rm{norm}}}\) are passed through a feedforward layer, termed the ‘enrichment layer' in the CytoTRACE 2 core model, containing the associated length \(2M\) gene set enrichment score weight vector \(\mathop{{\rm{V}}}\limits^{\rightharpoonup }\) and yielding length \(C\) potency category score vector \(\mathop{q}\limits^{\rightharpoonup }\). As part of this process, dropout is applied to reduce overfitting during model training, with a predetermined fraction of the normalized scores set at random to zero. From the weights in each \(\mathop{{\rm{V}}}\limits^{\rightharpoonup }\), concatenated across potency categories into matrix \({\bf{V}}\), the directionality and importance of each gene set can be interpreted (see ‘Interpretability' below). The model then integrates across the potency category scores produced by each GSBN module, concatenating the potency category score vectors into \(C\times G\) potency score matrix \({\bf{Q}}\). The model then predicts cellular potency by assigning the potency category with highest likelihood for each cell, yielding length \(C\) vector \(\hat{{\boldsymbol{y}}}\): For this aspect, we introduce length \(G\) ordered vector \(\mathop{t}\limits^{\rightharpoonup }\) to be multiplied by the potency category likelihood matrix: where \(\mathop{\mathrm{RPS}}\limits^{\rightharpoonup }\) is the length \(C\) raw potency score vector. As the potency categories are ordered based on their absolute developmental potential, the resulting raw potency score will be closer to one for higher potency categories, such as totipotent, and closer to zero for lower potency categories, such as differentiated. As \(\mathop{\mathrm{RPS}}\limits^{\rightharpoonup }\) directly incorporates model uncertainty, it is passed to ‘Postprocessing' below to define a more granular developmental ordering. As the fully trained CytoTRACE 2 model predicts potency for each cell individually, CytoTRACE 2 further processes the output (raw potency score \(\mathop{\mathrm{RPS}}\limits^{\rightharpoonup }\) and predicted potency categories \(\hat{{\boldsymbol{y}}}\)) to incorporate the neighborhood structure of transcriptionally similar cells. We reasoned that doing so could further improve performance given our previous experience combining gene counts with transcriptional covariance in CytoTRACE 1 (ref. To this end, we devised and validated a three-step procedure using the training cohort, as described below. Notably, this procedure improves correlations with relative developmental orderings (see ‘Metrics' below) over \(\mathop{\mathrm{RPS}}\limits^{\rightharpoonup }\) or \(\hat{{\boldsymbol{y}}}\) alone without sacrificing the potency classification performance achieved by \(\hat{{\boldsymbol{y}}}\) (Extended Data Fig. In the first step, CytoTRACE 2 applies Markov diffusion to smooth \(\mathop{\mathrm{RPS}}\limits^{\rightharpoonup }\) using the same implementation as CytoTRACE 1 (ref. This similarity matrix is then used to smooth \(\mathop{\mathrm{RPS}}\limits^{\rightharpoonup }\) with diffusion parameter α = 0.9 as previously described3, yielding smoothed potency score \(\mathop{\mathrm{SPS}}\limits^{\rightharpoonup }\). Using the same sampling procedure described in our previous work3, the running time of this step can be significantly reduced without loss of performance (Extended Data Fig. In this study, sampling was restricted to datasets with >10,000 cells (Supplementary Table 1). To reconcile \(\mathop{\mathrm{SPS}}\limits^{\rightharpoonup }\) with predicted potency categories \(\hat{{\boldsymbol{y}}}\), in the second step CytoTRACE 2 performs a binning procedure to maintain \(\hat{{\boldsymbol{y}}}\) while preserving relative potency ordering within each category. To do so, CytoTRACE 2 first separates cells by their predicted potency category and assigns each cell \(1\le w\le C\) a rank \({\mathcal{R}}\left(k,{\hat{{\boldsymbol{y}}}}_{w}\right)\) relative to all cells sharing predicted potency category \({\hat{{\boldsymbol{y}}}}_{w}\). For this transformation, within each potency category \(1\le p\le G\), the cell with lowest potency score receives rank 1 while the cell with highest potency score receives maximum rank \({r}_{\max }\left(p\right)\). Cells are then arranged uniformly by rank per potency category within equal length partitions of the unit interval, yielding binned smooth potency score \({\mathop{\mathrm{SPS}}\limits^{\rightharpoonup}}{}^{B}\). Thus, the binned smooth potency score for differentiated cells extends from 0 to 1/6, unipotent from 1/6 to 2/6, and so on, with relative ordering within each bin matching that of the original smoothed potency score. In the third step, to further smooth \({\mathop{\mathrm{SPS}}\limits^{\rightharpoonup }}{}^{B}\) while minimizing the impact on \(\hat{{\boldsymbol{y}}}\) and allowing for the preservation of rare cell states (Extended Data Fig. 3f), CytoTRACE 2 applies a variation of k-nearest neighbor (k-NN) smoothing to datasets with >100 cells. Here, we introduce an efficient heuristic approach for adaptive neighborhood smoothing guided by two key assumptions: (1) cells with more similar gene expression profiles are more likely to share a potency phenotype; and (2) prediction errors for cells with the same ground truth potency exhibit a random distribution around a central mean. To balance these two considerations and identify an appropriate neighborhood size, we select k adaptively for each cell according to the following process. First, given log2-adjusted CPM/TPM gene expression profiles for the selected cell, we standardize expression per cell to zero mean and unit variance, then perform dimension reduction of standardized gene expression profiles over all cells to the top 30 principal components (PCs). Using the top 30 PCs, we then compute pairwise Euclidean distances for all cells, rescaling the resulting distances to unit maximum per cell of interest. Next, we define the neighborhood around each center cell w through an iterative procedure, allowing a maximum neighborhood size of 30 cells. We start with the nearest cell to \(w\), denoted \({c}_{1}\), and calculate the average potency score prediction for \(w\) and \({c}_{1}\), mapping the result to one of six broad potency categories, yielding \({P}_{1}\). If not identical, we repeat the procedure increasing the group size by one, in other words, comparing the nearest two cells to \(w\) (yielding three total cells) with the next nearest three cells (\({c}_{3}\), \({c}_{4}\) and \({c}_{5}\)). We repeat this process until the resulting potency categories are the same between two groups, in which case we select \(k\) to encompass all cells considered between the two groups, or until we exhaust our candidate nearest neighbor cells (reach a group size of 15). If concordance between nearest and next nearest groups is not found, we keep our initial selection of \(k\) = 3. Once \(k\) is determined, we update our prediction for \(w\) according to the distance-weighted mean of neighborhood potencies to obtain the final potency score prediction: where \(N(w)\) denotes the set of all cells within the selected neighborhood of center cell \(w\), including \(w\) itself, and \({d}_{c}\) denotes the Euclidean distance of cell \(c\) to cell \(w\). Categorical potency predictions are updated based on the defined intervals above, yielding \({\hat{{\boldsymbol{y}}}}^{* }\). We found empirically that combining these three approaches yielded superior performance on the training cohort (Extended Data Fig. For model training, we defined a loss function combining cross-entropy loss with an additional term penalizing gene set size based on the binary weighting matrix \({{\bf{W}}}_{p}^{{\rm{B}}}\) originating from each GSBN module, \(1\le p\le G\). More precisely, we define the loss function as the sum of gene set size penalty loss \({J}_{S}\) and a prediction loss per cell \({J}_{P}\): In detail, given potency category predictions \({\hat{{\boldsymbol{y}}}}_{w}\) and ground truth potency categories \({{\boldsymbol{y}}}_{w}\) for cell \({w}\) (see ‘Single-cell potency atlas' above), we defined prediction loss \({J}_{P}\) as: Loss weights for all cells are contained in the length \(C\) weighting vector \(\mathop{\boldsymbol{v}}\limits^{\rightharpoonup}\), which has unit sum and is constructed hierarchically to assign equal weight (1) to all broad potency categories, (2) to all phenotypes within each broad potency category, and (3) to all datasets contributing to each phenotype. We defined gene set size penalty loss \({J}_{S}\) as: To promote model generalizability, we introduced two explicit regularization aspects. We included a dropout layer to avoid model overfitting to specific enrichment scores (“Integration of scores”). This layer was applied to the normalized scores \({{\bf{K}}}^{{\rm{norm}}}\) during training only. Additionally, a penalty term was added to the loss function to constrain the number of genes in each gene set of \({{\bf{W}}}^{{\rm{B}}}\) (“Loss function”). Model weights were initialized according to PyTorch v.2.0.0 default except for the binary weighting matrices, which were initialized at random with values sampled from the Gaussian distribution with mean of –0.1 and s.d. of 0.055 to produce a sparse initial binarization with approximately 500 genes selected per gene set. Model training was performed with mini-batch learning using a batch size of 1,024. To balance batches and ensure equal representation for the model learning process, each batch was constructed via uniform sampling across datasets and phenotypes (Supplementary Tables 1 and 3) as implemented by torch.utils.data.WeightedRandomSampler in PyTorch. Following initialization, forward propagation proceeded for each iteration as described in ‘Core model architecture', with parameters updated according to their definition. For numeric stability, the cutoff rank τ (‘Gene set binary networks') for trimming input rank space expression matrix \({\bf{R}}\) was not learned directly but rather computed as a function of learnable parameter \({\tau }_{m}{\mathbb{\in }}{\mathbb{R}}\), which was initialized uniformly at random from \({0\le \tau }_{m}\le 1\) per module and suitably scaled. As gene set enrichment score calculation (‘Enrichment assessment') requires a gene set pool larger than the gene set itself for comparison, τ was computed from \({\tau }_{m}\) in such a way as to ensure that the ranks of at least ten more genes beyond the maximum gene set size of the module were preserved following trimming to \({\bf{T}}\). Thus, at each iteration, the updated \({\tau }_{m}\) was scaled and constrained as follows: Model predictions were assessed at each iteration against ground truth, with the loss function and its gradient computed and used to backpropagate updates to network weights using PyTorch's NAdam optimizer with custom learning rate \({\rm{lr}}=0.001\) (see ‘Hyperparameter optimization' below) and otherwise default parameters. Given the role of inertia in successfully training binary neural networks52,53, we employed cross-epoch gradient accumulation to dampen binary weight flipping and achieve a stabilizing effect. This approach additionally facilitates broader hyperparameter space exploration while validation-based early stopping (see ‘Model evaluation and stopping') ensures that the most performant model encountered during training is retained. Backpropagation for the binary neural network component of each GSBN module was implemented with Straight-Through Estimator and hardtanh activation function as previously described48. We evaluated model validation performance via weighted accuracy, defined as the mean F1 score across evaluable potency categories. To do this, we first calculated the F1 score for each phenotype (standardized as in Supplementary Table 3) and dataset pair using metrics.precision_recall_fscore_support from sklearn v.1.0.2. We then averaged the resulting scores across datasets per phenotype, across phenotypes within each broad potency category, and across broad potency categories, yielding the final weighted accuracy. For the standard CytoTRACE 2 model, each validation set consisted of a single dataset; however, for the leave-clade-out model (see ‘Generalizability to unseen cell-type clades'), validation sets included all cells covering a clade, regardless of dataset. To evaluate the hyperparameter space of CytoTRACE 2, we performed a hyperparameter sweep over the training cohort using wandb (v.0.16.4) (https://wandb.ai). We explored the learning rate lr over \(\left\{\mathrm{0.01,0.005,0.001,0.0005,0.0001}\right\}\), number \(M\) of gene sets per broad potency category over \(\left\{\mathrm{1,2,4,8,12,16,24,32,48}\right\}\), gene set size penalty weight λ over \(\left\{\mathrm{0.5,0.1,0.05,0.01,0.005,0.001}\right\}\), dropout rate \(\rho\) over \(\left\{\mathrm{0,0.25,0.5}\right\}\), and enrichment considering whether to use AMS enrichment, UCell enrichment, or the combination of both as described in ‘Enrichment assessment' above. For every iteration of leave-one-dataset-out nested cross-validation, we trained models across 500 different combinations of these hyperparameters sampled based on the random hyperparameter search. While one dataset was held out from training and evaluated as a validation set, another dataset was also held out from training but used to determine the early stopping point as described in ‘Model evaluation and stopping'. We scored each hyperparameter combination by weighted accuracy over model validation sets (Supplementary Table 3; see ‘Model evaluation and stopping'). We observed that variation in hyperparameter values had minimal impact on performance, underscoring overall model robustness (Extended Data Fig. Final hyperparameter selection was carried out by a manual curation process identifying values yielding consistently (albeit modestly) higher weighted accuracy. In selecting the number of gene sets \(M\) per potency category, we found that model performance increased with \(M\) before plateauing (Extended Data Fig. 1e, right); as such, we selected \(M\) slightly larger than the number corresponding to the elbow of this curve. The final hyperparameters used were \(M=24\) gene sets per potency; \(\rho =0.5\) dropout probability; \(\lambda =0.01\) gene set size penalty weight; and \({\rm{lr}}=0.001\) learning rate. Among all models, we limited to 84 models with hyperparameter values in ranges of plateau (\(M\ge 2\) gene sets per potency; \(\rho =0.5\) dropout probability, \(\lambda \le 0.01\), \({\rm{lr}}\le 0.001\)). Models were trained via leave-one-dataset-out cross-validation for each of the training datasets, with final CytoTRACE 2 predictions in non-training data obtained as the result of integrating predictions across the 19 resulting models followed by an additional postprocessing step. As described in ‘Integration of scores' above, each model m yields a \(C\times G\) potency category likelihood matrix \({{\bf{P}}}^{m}\). Models were integrated by entry-wise averaging of potency category likelihood matrices to yield a single potency category likelihood matrix \({{\bf{P}}}^{{ensemble}}\) from which potency category predictions and raw potency scores were computed as described above, before passing them to ‘Postprocessing'. To create dictionary \({\mathbb{D}}\) (‘Preprocessing' above), all human gene symbols were mapped to their closest mouse orthologs, as determined by gene sequence similarity, using the GRCh38.p13 and GRCm39 annotation files available from Ensembl v.109, respectively. In cases where a single mouse gene \(g\) was identified as the best hit for multiple human genes, the human gene with maximum sequence similarity to \(g\) was selected and the remaining human gene(s) excluded from further consideration. Unique human gene symbols without orthologs by the above process were also included for completeness. To define a common subset, only genes present in at least 80% of datasets from an initial development cohort, a subset of the final training cohort, were retained. Combining these steps, \({\mathbb{D}}\) was assembled with 14,271 unique gene symbols, including 13,750 orthologous pairs and 521 genes without orthologs in Ensembl via the mapping step above. When mapping human datasets to \({\mathbb{D}}\), gene symbol aliases are resolved using linked aliases available from https://biomart.genenames.org. When mapping to mouse datasets, alias gene symbols are resolved using data available from https://www.informatics.jax.org/mgihome/nomen/. By examining the orientation of the output layer weights for each gene set, we found that gene sets with positive weights (polarity) were highly enriched in a given potency category, whereas those with negative weights (polarity) were preferentially depleted (Fig. Additionally, we reasoned that genes repeatedly selected for a given potency category were more likely to be important for effective classification. As such, we designed a metric to quantify feature importance, assigning importance scores to genes according to the frequency at which they were selected in positively versus negatively weighted gene sets. Here, we incorporate gene selection frequency across all 19 training models computed by leave-one-out cross-validation (LOOCV) over the training cohort datasets. More formally, we define \(N\times G\) feature importance score matrix \({\bf{F}}\) (Supplementary Table 15) containing the feature importance score of each gene \(1\le i\le N\) for each potency category \(1\le p\le G\) based on the gene set compositions and enrichment weights across models. Given gene set enrichment weight matrix \({{\bf{V}}}^{l}\) of model \(l\), we calculate the polarity \({\rm{Polarity}}\left({{\bf{V}}}^{l},j,p\right)\) of gene set \(j\) defined within model \(l\) for potency category module \(p\) as the sign of the average of these two weights. Then, relying on model binary weighting matrices to encode gene set composition, we construct feature importance score matrix \({\bf{F}}\) entry-wise as where \({{\bf{W}}}_{p,l}^{{\rm{B}}}\left[i,j\right]\) denotes the \(\left[i,j\right]\)th entry of the binary weighting matrix from module \(p\) of model \(l\). Two key metrics, illustrated in Extended Data Fig. Absolute order quantifies cross-dataset performance, whereby predicted orderings from all cells with annotated potency levels are analyzed together, regardless of dataset, tissue type or platform (Supplementary Tables 2 and 3). Relative order quantifies performance within a given dataset and tissue type, akin to conventional pseudotime and ranges from 1 (least differentiated) to \(N\) (most differentiated) in each dataset (Supplementary Table 4). For both metrics, we applied weighted Kendall correlation (\(\tau\)) (wdm package v.0.2.4 in R) to assess concordance between known and predicted developmental orderings, with weighting schemes provided in Supplementary Table 5. Similar to our previous work3, ground truth phenotypes corresponding to less mature cells were coded with lower ranks (starting at 1); therefore, higher predictions of developmental order were ranked such that higher values received lower ranks and vice versa. For categorical predictions (CytoTRACE 2 and potency classification benchmarking outputs only), we evaluated potency classification performance as well. Binary correctness of predicted versus ground truth broad potency categories was assessed via mean multiclass F1 score, implemented with function f1_score from sklearn.metrics with average = none (Extended Data Figs. To account for the magnitude of deviations from ground truth potency, we also considered mean absolute error (MAE), assigning each broad potency class an integer label corresponding to the class ordering, with labels ranging from 1 (differentiated) to 6 (totipotent), and computing the absolute value of the difference between predicted and ground truth categories (Extended Data Figs. For both metrics, scores were computed per ground truth potency category then aggregated by mean across potencies. To test the generalizability of CytoTRACE 2 to unseen developmental systems, we trained a version with a leave-clade-out framework (Fig. 1f), grouping phenotypes into 18 mutually exclusive developmental clades as detailed in Fig. Of note, to ensure representation of some totipotent and pluripotent phenotypes for all training sets, we partitioned embryonic phenotypes into two clades by alternating granular potency level annotation, corresponding to distinct time points during development and resulting in 19 total clades for this analysis (Supplementary Table 2). Stem and progenitor cells that produce a given clade were included in the same partition as that clade (for example, pancreatic multipotent progenitors were included with pancreatic epithelial cells). Epithelial cells were separated by tissue to avoid conflating tissue-specific developmental hierarchies. For each clade, we trained an ensemble of two models over the remaining 18 clades, selecting at random 17 clades for training and one clade as a held-out validation set to be used for early stopping (see ‘Model evaluation and stopping') for each model. We then applied the resulting ensemble to the unseen test clade, assessing performance across all held-out clades in Fig. To ensure minimum adequate representation within each category, we confirmed that each fold contained at least one phenotype per broad potency category. Tabula Muris, which was divided into two sub-datasets according to platform for the original CytoTRACE 2 training cohort due to its size and diversity, was again divided, with one of its sub-datasets assigned to another fold at random. For each split, two folds were combined to form the training cohort and the remaining one left as a test set for evaluation (2:1 training–test split; Supplementary Table 8). Performance per test set of these three randomized splits, along with the original CytoTRACE 2 test set, was assessed by absolute order, relative order, mean multiclass F1 score and MAE (see ‘Metrics'), showing strong consistency across folds (Extended Data Fig. Performance for the three randomized splits was additionally assessed across all held-out datasets jointly in Extended Data Fig. To evaluate the robustness of CytoTRACE 2 to potential noise within potency annotations, we trained models across two scenarios of training cohort annotation error, then evaluated model performance over the test cohort (see ‘Training and test datasets'). To simulate annotation error, we formulated label noise as a transition matrix54, encoding the probability of perturbation from one potency to another (Extended Data Fig. Transition matrix perturbation probabilities were designed to follow a Gaussian distribution based on the rank distance between the original potency and perturbed potency. where potencies \(i,j\) are represented by their rank within the six broad potency categories. values (\(\sigma\)) were selected to yield a titration of 5%, 10%, 20%, 50% and 80% perturbation levels. Rows were normalized to unit sum for a net probability of one. For the first annotation error scenario, we considered cell-level annotation error and perturbed the potency annotations of individual cells independently (Extended Data Fig. To determine the influence of variable gene counts and unique molecular identifier (UMI) counts on CytoTRACE 2, we performed two experiments in which scRNA-seq expression data from all 14 datasets in the test cohort were perturbed by downsampling gene counts (Extended Data Fig. 3d) and all seven droplet-based datasets in the test cohort (Supplementary Table 1) were perturbed by downsampling UMIs (Extended Data Fig. Cells with UMIs lower than a given threshold were unaltered. We repeated each process for five replicates, then assessed performance for standard metrics as described above (see ‘Metrics') relative to the CytoTRACE 2 predictions without perturbation. Given the inclusion of neighborhood-based smoothing in model postprocessing, we performed a titration experiment applying CytoTRACE 2 to test datasets with selected phenotypes downsampled to increasingly rare abundance. For 11 phenotypes spanning a range of potencies, we downsampled cells of the selected phenotype to predefined abundances of 50, 20, 10, 8, 5, 2 and 1 cell(s), leaving the remaining cells in the dataset unchanged. We repeated this titration process five times for each phenotype, observing robust predictions down to five cells per phenotype (Extended Data Fig. As such, we recommend that the final postprocessing step (adaptive k-NN smoothing) be omitted when exceedingly rare cell states (consisting of <5 cells each) are of interest. For the analyses presented in Extended Data Fig. 5, we downloaded and curated six publicly available scRNA-seq datasets spanning each embryonic day during mouse prenatal development2,21,22,23,24,25 (Supplementary Table 1). One dataset, which covers pre-implantation through early implantation (E0.5–E4.5) (Deng et al.22), was obtained from the 19-dataset training cohort (Supplementary Table 1) and evaluated using a CytoTRACE 2 model trained on the remaining 18 datasets to avoid overfitting (see ‘Benchmarking developmental potential inference methods and annotated gene sets'). Four datasets21,23,24,25 covering embryogenesis periods from implantation to organogenesis were previously assembled by Qiu et al.25 and are accessible through http://tome.gs.washington.edu. Finally, a single-nucleus RNA-seq dataset2 covering organogenesis through birth (E8.75-P0) and generated by sci-RNA-seq3 was downloaded from http://mouse.gs.washington.edu. As we compared CytoTRACE 2 against multiple methods with highly variable time complexity (‘Benchmarking developmental potential inference methods and annotated gene sets'), all cells were randomly downsampled to 30 cells per author-supplied phenotype per time point, resulting in a combined dataset of 183,771 cells. This allowed us to balance considerations of performance versus computational efficiency. We ran each method on each dataset individually as described in ‘Benchmarking developmental potential inference methods and annotated gene sets'. No dataset integration or batch normalization procedures were applied. For Organogenesis (E8.5)25 and Organogenesis (E8.5–P0)2, which were sequenced using sci-RNA-seq3, we used count data after running SCTransform of Seurat (v.4.3.0) with default parameters. Due to the large size of the dataset, Organogenesis (E8.75–P0)2 was run with ten randomly divided batches for SCENT (SR) and SLICE. Primordial germ cells were excluded owing to the wide range of potency levels reported in previous literature55. For the analyses in Extended Data Fig. Although the tree was constructed using a heuristic approach based on transcriptional covariance across embryonic time, it reflects many known parent-daughter relationships2. It thus serves as a proxy for developmental potential. We defined ground truth as the distance from the root (zygote) to each daughter node (Extended Data Fig. For each direct path in the tree (from root to leaf), the resulting scores were then converted to rank space (Extended Data Fig. To reconcile cases where a given node i participates in multiple paths, we used the average rank for i. CytoTRACE 1 predictions were processed in the same manner (Extended Data Fig. The resulting ranks were correlated with ground truth distances (distance from the root) in Extended Data Fig. For the analysis presented in Extended Data Fig. 6a, we downloaded the Galen et al.56 acute myeloid leukemia (AML) dataset (Gene Expression Omnibus (GEO) accession number GSE116256; PMID 30827681) from the Curated Cancer Cell Atlas website on 28 June 2023 (https://www.weizmann.ac.il/sites/3CA/)57. We leveraged author-supplied cell type annotations, including classifications of malignant and non-malignant cells from 3CA57. We ran CytoTRACE 2 with default parameters (‘Benchmarking developmental potential inference methods and annotated gene sets') on all annotated malignant cells from each tumor sample. For quality control, we further excluded samples for which each predicted potency label contained <10 malignant cells. For each of the resulting tumor samples (n = 19), we created a single matrix of malignant cells and non-malignant cells, with the latter uniformly downsampled from all patients to 100 cells per author-supplied phenotype (‘B_cell', ‘erythrocyte', ‘myeloid', ‘NK_cell', ‘plasma' and ‘T_cell'; non-malignant cells labeled as ‘undifferentiated' were excluded from additional analysis). We then calculated the log2 fold changes (LFCs) of each potency category versus all other phenotypes by tumor sample and averaged by potency category across tumor samples. Finally, we normalized the logFC values of each gene to mean zero and unit variance across potency categories and plotted the enrichment of AML cell-type-specific gene signatures26 (‘LSPC-Primed-Top100', ‘LSPC-Quiescent', ‘GMP-like-Top100' and ‘Mono-like-Top100'; https://github.com/andygxzeng/AMLHierarchies), each expected to be enriched in multipotent, multipotent, oligopotent and unipotent/differentiated cells, respectively (Extended Data Fig. 6b, we applied CytoTRACE 2 to scRNA-seq profiles of six oligodendrogliomas27, with coordinates for the associated oligodendroglioma 2D lineage hierarchy embedding obtained from https://singlecell.broadinstitute.org/single_cell/study/SCP12/oligodendroglioma-intra-tumor-heterogeneity. For the latter, we separated cells according to the stemness score by partitioning them into successive intervals of 0.25 units. We then displayed CytoTRACE 2 potency scores as a function of each interval (Extended Data Fig. To evaluate CytoTRACE 2 against supervised machine learning approaches commonly employed in cell type prediction tasks (Extended Data Fig. 7a,b), we selected three dedicated single-cell annotation methods with superior performance in a benchmarking study28 (scPred30, SingleCellNet31 and scmap32) and five general-purpose classifiers (below), each trained to predict six broad potency labels based on single-cell expression profiles. All tools were trained and tested over a series of four folds, including the original CytoTRACE 2 training–test split (Fig. 1b) along with three randomized splits (see ‘Randomization of training and test sets'), collectively encompassing all 33 ground truth datasets in the single-cell potency atlas described above, with classification performance per test cohort assessed by mean multiclass F1 score and MAE (Extended Data Fig. For all methods, expression data were first mapped into the uniform feature space used by CytoTRACE 2 (see ‘Preprocessing' and ‘Dictionary of input genes'). Other normalization schemes generally yielded worse performance and were thus omitted from further consideration (log2-adjusted CPM/TPM data, either used alone or with gene-level standardization). No explicit dataset integration or batch correction was performed. All parameters were set to default values unless otherwise specified. We applied CytoTRACE 2 with model ensembling and postprocessing as described in ‘The CytoTRACE 2 framework' to predict cell potency categories. A dedicated cell type classification method, scPred first performs a dimension reduction, identifying PCs exhibiting significant variation across classes, then, as the default option, applies a support vector machine approach for classification30. Following the recommended pipeline for scPred (v.1.9.2) as described at https://powellgenomicslab.github.io/scPred/articles/introduction.html, we first normalized and scaled expression data using the NormalizeData() and ScaleData() functions in Seurat (v.5.1.0), respectively. We then used scPred's getFeatureSpace() function to identify class-informative PCs, trainModel() to train the default support vector machine (SVM) with radial kernel model for each potency category (one-versus-rest), and scPredict() for classification. A relaxed probability threshold of 0 was used to avoid ‘unassigned' labels. Here, we trained the method over unnormalized expression data via the scn_train function of pySingleCellNet (v.0.1.1) with nTopGenes = 200, nTopGenePairs = 200, nRand = 100, nTrees = 1,000, stratify = False, and propOther= 0.4, following the tutorial provided at https://pysinglecellnet.readthedocs.io/en/latest/notebooks/train_classifier.html. The scn_classify() function with nrand = 0 was used for classification. Following the recommended pipeline for scmap (v.1.26.0) provided at https://bioconductor.org/packages/devel/bioc/vignettes/scmap/inst/doc/scmap.html, we log2-transformed expression data, then used selectFeatures() to select informative genes and indexCell() to create a scmapCell index for the training dataset. For classification, we used scmapCell() to project the index onto the test dataset and scmapCell2Cluster() to obtain label assignments. A relaxed probability threshold of 0 was set to assign labels to as many cells as possible regardless of assignment confidence. We trained a logistic regression model to perform cell potency classification using the SGDClassifier from scikit-learn (v.1.4.2) with loss = ‘log_loss', default L2 regularization, and sample weights provided for class balancing. This function internally employs a one-versus-rest (OVR) strategy, training a separate binary classifier for each potency category and selecting the potency category with highest confidence at evaluation. We trained and applied the XGBClassifier function from the XGBoost library (v.2.1.1) with default parameters and without sample weights. Like logistic regression, this method uses the OVR approach. We implemented a linear SVM model using Scikit-learn's SGDClassifier with loss = ‘hinge' for linear support vector classification with OVR. We implemented an additional SVM version using SVC from scikit-learn (v.1.4.2) with the default radial basis function kernel and γ = ‘auto'. The default decision function, which employs an inference of OVR from one-versus-one fits internally, was used. Sample weights were not provided during training. Using LogisticRegression from scikit-learn (v.1.4.2) with multi_class = ‘multinomial', we fit a single logistic regression model for all potency categories simultaneously using cross-entropy loss and the ‘sag' solver. A maximum number of iterations (max_iter = 500) and tolerance (tol = 1 × 10−3) were set to ensure convergence. Sample weights were not provided during training. To rigorously assess performance on our compendium of 33 curated scRNA-seq datasets, we compared CytoTRACE 2 with eight published methods for predicting developmental potential from scRNA-seq data as well as nearly 19,000 previously annotated gene sets (Fig. Unless otherwise stated, all evaluated methods and gene sets were applied to scRNA-seq datasets individually, without batch correction or integration across datasets, with expression data normalized per author recommendations and with default parameters. All expression data were subset to the cells with known potency. Several methods rely on human gene symbols, as noted below. For all such instances, we mapped mouse dataset gene symbols to their closest human orthologs, as determined by gene sequence similarity, using the GRCm39 and GRCh38.p13 annotation files available from Ensembl, respectively. In cases where a single human gene \(g\) was identified as the best hit for multiple mouse genes, the mouse gene with maximum sequence similarity to \(g\) was selected. As several methods have slower running times, to promote an equitable comparison while achieving computational feasibility, larger datasets were first downsampled. The Tabula Muris43 dataset was downsampled to 30 cells per phenotype, separated by tissue and platform pair, and the ‘Immune cell atlas (10x)', ‘Human breast 1 (10x)', ‘Human breast 2 (10x)', and Tabula Sapiens33 datasets were downsampled to 100 cells per phenotype (Supplementary Table 1). Cell types in Tabula Sapiens33 with fewer than five cells were removed after the prediction of each method to overcome the reduced data quality of Tabula Sapiens33 (‘Training and test datasets'). We applied CytoTRACE 2 with model ensembling and postprocessing as described in ‘The CytoTRACE 2 framework' to predict cell potency categories and scores. All other datasets were evaluated with the primary version of CytoTRACE 2 trained over all training datasets. CytoTRACE 1, the predecessor of CytoTRACE 2, introduced transcriptional diversity quantified through gene counts as a correlate of developmental potential and exploited this concept to predict relative cellular potency from scRNA-seq3. SCENT estimates relative cellular potency from scRNA-seq and a reference protein–protein interaction (PPI) network using single-cell signaling entropy (SR), a measure of the diversity of molecular pathway activity in a cell15. SCENT (v.1.0.3) was executed with the ‘net13Jun12' human PPI network provided with the package and otherwise default parameters. For mouse datasets, genes were first mapped to human orthologs as described above. All gene symbols were converted to Entrez ID using org.Hs.eg.db (v.3.15.0) in R. Gene expression matrices were normalized per documentation recommendation (https://github.com/aet21/SCENT/blob/master/vignettes/SCENT.Rmd). CCAT was applied with the same package, PPI network, and preprocessing steps described above (‘SCENT (SR)') with expression datasets prepared as per documentation recommendations. Similar to SCENT (CCAT), FitDevo infers cellular potency from the correlation between gene expression and a measure of gene weights20. FitDevo (v.1.2.0) was applied following tutorial instructions with binary gene weight matrix downloaded from the same source (https://github.com/jumphone/FitDevo/#demo-1–infer-developmental-potential-dp-using-expression-matrix-of-scrna-seq-data). SLICE relies on transcriptomic entropy for cellular potency prediction and lineage reconstruction, estimating entropy over functional groups of genes computed from Gene Ontology annotations17. SLICE (v.0.99.0) was applied according to demo details from the method's GitHub page (https://github.com/xu-lab/SLICE/blob/master/demo/FB.R). StemID infers cellular differentiation trajectories from scRNA-seq data with a clustering-based algorithm analyzing links between clusters16. StemID, implemented in RaceID (v.0.1.4), was run according to documentation vignette instructions (https://cran.r-project.org/web/packages/RaceID/vignettes/RaceID.html). For each dataset, an SCseq object was initialized from each input gene expression matrix using filterData() with mintotal = 10. Ltree() and compentropy() were then applied consecutively to obtain the StemID score for cell potency. scTour implements a deep learning architecture combining a variational autoencoder with a neural ordinary differential equation to reconstruct the developmental trajectory of an input scRNA-seq dataset, oriented according to gene counts19. scTour (v.1.0.0) was trained and applied to each dataset individually per ‘Model training' documentation vignette instructions at https://sctour.readthedocs.io/en/latest/notebook/scTour_inference_PostInference_adjustment.html. Cell potency scores were obtained from the developmental pseudotime predictions extracted from the model training output with get_time(). mRNAsi utilizes a one-class logistic regression framework to construct a cellular stemness index applicable to cell potency estimation from bulk and scRNA-seq data18. All input gene expression matrices were CPM/TPM normalized and log2-transformed. For each gene set, the AddModuleScore() function with default parameters from Seurat (v.4.3.0) was applied to each expression matrix normalized via Seurat's NormalizeData() function. As scVelo5 relies on splicing kinetics, necessitating the processing of raw sequencing data, we limited our analyses to nine ground truth datasets from the test cohort that were generated by platforms with built-in support by velocyto and for which raw sequencing data are publicly available (Supplementary Tables 1 and 14). Raw FASTQ files for seven of these datasets, namely ‘BM-MNC (CITE-seq)', ‘Retinal neurons (10x)', ‘Pancreas (10x)', ‘Peripheral glia (Smart-seq2)', ‘Skeletal stem cell (C1)' and ‘HSCs and MPPs (inDrop)', were obtained from the Sequence Read Archive (SRA) from NCBI, with study IDs SRP188993, SRP168426, SRP200419, SRP109011, SRP239468 and SRP094420, respectively. For ‘Peripheral glia (Smart-seq2)', we analyzed sample IDs prefixed with ‘E12.5'. Notably, raw FASTQ files were only available for 227 of 473 cells in the ‘Skeletal stem cell (C1)' dataset. For the remaining two datasets, ‘Mouse neurogenesis (10x)' and ‘Mouse mature neural cell types (10x)', data were obtained as BAM files from SRA study ID SRP476153. FASTQ files were downloaded using sra-tools v.3.1.1 and processed with cutadapt v.4.9 for adaptor trimming of Smart-seq2/C1 reads. For preprocessing of inDrop samples, dropest v.0.8.6 was used (according to recommended workflow at https://velocyto.org/velocyto.py/tutorial/cli.html#run-dropest-run-on-dropseq-indrops-and-other-techniques). Loom files containing spliced, unspliced and spanning reads were then generated from the BAM files along with corresponding Gene Transfer Format files using the velocyto.py v.0.17.17 Python command line tool. Following quantification of spliced/unspliced counts, the scVelo v.0.3.1 Python velocity estimation workflow was run as described in the tutorial at https://scvelo.readthedocs.io/en/stable/. For all datasets, both a generalized dynamical model (as detailed at https://scvelo.readthedocs.io/en/stable/DynamicalModeling.html) and a differential kinetics adjusted model with grouping by the CytoTRACE 2 standardized phenotypes (as detailed at https://scvelo.readthedocs.io/en/stable/DifferentialKinetics.html) were employed. With the exception of random_state in scvelo.pp.neighbors(), which was set to 0 to ensure reproducible results, all other parameters were set to those in the respective vignettes, including min_shared_counts in scvelo.pp.filter_and_normalize(), which was set to 20 for dynamical models and 30 for differential kinetics models. Following velocity estimation, cell-internal latent time was inferred using scvelo.tl.latent_time(). The resulting outputs were then evaluated via absolute and relative order (see ‘Performance assessment' above) and CytoTRACE 2 outputs were assessed over the same cells for comparison (Extended Data Fig. 9a, we ran CytoTRACE 2 on each of the training and test datasets, then extracted positive potency score matrix \({{\bf{Q}}}^{{\rm{pos}}}\) from each of the 19 models per dataset. \({{\bf{Q}}}^{{\rm{pos}}}\) is derived from the final layer of each GSBN module; obtained similarly to the potency score matrix \({\bf{Q}}\) in ‘Integration of scores' above, but by multiplying \({{\bf{K}}}^{{\rm{norm}}}\) by only positive weights from the enrichment layer; and has a dimensionality of \(C\) input cells by 114 when combined across the 19 models and six potency category modules. We concatenated \({{\bf{Q}}}^{{\rm{pos}}}\) matrices from the 19 models across all 33 datasets (\(C\) = 124,231 cells; downsampled as described in ‘Benchmarking developmental potential inference methods and annotated gene sets' above) to produce \({{\bf{Q}}}_{{\rm{all}}}^{{\rm{pos}}}\), standardized across all cells in the training and test sets separately. To adjust for differences in cell density that confound visualization, we averaged CytoTRACE 2 potency scores within each window of 0.5 UMAP units squared across the two components of UMAP space. 2c, we examined the expression of the top 500 potency-associated markers learned by CytoTRACE 2 (matrix \({\bf{F}}\) in ‘Interpretability') in training and test sets from the single-cell potency atlas. We first filtered and mapped gene symbols in every dataset to CytoTRACE 2 input features (n = 14,271), then CPM/TPM normalized as appropriate and log2-adjusted the data. Keeping training and test data separate, the expression matrices from each dataset were mean-aggregated into pseudo-bulk expression profiles by phenotype. We then further averaged shared phenotypes across datasets profiled by the same general platform (droplet/UMI or plate-seq/non-UMI) and finally, by species identifier (human or mouse). Using this matrix, we calculated the mean expression of the top 500 positive/negative genes per potency category (matrix \({\bf{F}}\) in ‘Interpretability'; Supplementary Table 15), then unit variance normalized the resulting expression signatures across pseudo-bulk samples separately for training and test sets (Fig. To assess the biological relevance of CytoTRACE 2 model features, we analyzed large-scale in vivo CRISPR screening data of mouse hematopoiesis38 (Fig. These data encompass ~7,000 genes along with CasTLE –log10 P values, representing the effect of knockout (KO) on HSC differentiation. Although these effects were separately measured for lymphoid (n = 6,783 genes) and myeloid (n = 6,732 genes) lineages, directed –log10 P values were well correlated between them (r = 0.78). Therefore, to create a single ordered list of KO effects, we combined directed –log10 P values for genes with effect score data in both lineages, keeping the most significant directed –log10 P value for each gene (positive or negative). Contributions from each lineage were nearly perfectly balanced, with higher positive scores and higher negative scores implying that KO of a gene promotes or inhibits HSC differentiation, respectively. We then intersected the resulting vector with those within the CytoTRACE 2 model space, resulting in n = 5,757 genes. P values were computed using the adaptive multilevel Monte Carlo method and Q values represent false discovery rates calculated using the Benjamini–Hochberg procedure. To assess robustness across the number of top multipotency markers selected, we repeated the above process checking 50, 100, 200 and 500 markers (Extended Data Fig. We also compared the median –log10 Q value of gene set enrichment over the four gene set sizes against the same process repeated for CytoTRACE 2 markers for all other potency categories (Extended Data Fig. To interpret potency-associated genes learned by CytoTRACE 2, we applied fgsea (v.1.25.1) to each rank-ordered gene list in \({\bf{F}}\) with minSize = 15 and otherwise default parameters. \({\bf{F}}\) is an \(N\times G\) matrix consisting of model importance scores for all \(N\) evaluable genes (14,271) in each of \(G=6\) potency categories learned on the training cohort (‘Interpretability' above; Supplementary Table 15). We ran fgsea on mouse and human gene sets separately, and human gene sets were limited to those with no counterpart in mouse gene sets. When running human gene sets, genes in \({\bf{F}}\) were first mapped to human orthologs by dictionary \({\mathbb{D}}\) (‘Dictionary of input genes'). Gene sets with an adjusted P value < 0.05 in at least one potency category are provided in Supplementary Table 17. We selected a subset of representative molecular signatures for display in Extended Data Fig. 9e, highlighting both canonical and poorly understood potency-related biology. For this purpose, we analyzed pseudo-bulk-expression profiles of each phenotyp–dataset pair in our 33-dataset potency atlas using single-sample GSEA (ssGSEA) from the GSVA package in R (v.1.46.0)58 to mitigate technical variation. Once ssGSEA scores were obtained for the UFA factors, we then averaged them into the same 237 phenotype, species, and platform combinations described in ‘Top potency-associated genes learned by CytoTRACE 2'. Keeping training and test cohorts separate, we further averaged ssGSEA scores by developmental system (here, denoted ‘tissue'), using the phenotype-to-clade mapping scheme provided in Supplementary Table 9. Mean-aggregated ssGSEA scores across 237 phenotype, species and platform combinations in training and test sets are displayed in Fig. Next, we calculated the pairwise difference \({\Delta }_{i}\) between the median ssGSEA scores of multipotent and each other potency category i. To simulate a null distribution, we permuted the phenotype-level ssGSEA scores, recomputed the median ssGSEA score for each ground truth potency category and calculated both statistics. To determine an empirical P value, we tallied the proportion of times both statistics were as high (or higher) than the test statistics from the original data. AUCs of UFA genes (main text) were calculated for training and test sets separately using the ssGSEA scores described above, but after averaging the scores by tissue type to address imbalances. AUCs were first calculated in a pairwise manner between multipotency and each other potency category, then averaged. 10, 8–12-week-old mice were used, with equal numbers of males and females. The animals were housed under a 12-h light–dark cycle at room temperatures between 20–26 °C, with humidity levels ranging from 30–70%. In brief, hips, femurs, tibia and humeri were collected from C57BL/6 mice. Bones were cleaned, cut and flushed with a syringe filled with ice-cold FACS buffer (2% fetal bovine serum in Hanks' balanced salt solution buffer). Cells in FACS buffer were filtered through a 40-μm filter, pelleted and then incubated in ammonium–chloride–potassium (ACK) lysis buffer for 5 min on ice. Cells were then spun down and resuspended in 400 µl FACS buffer per mouse. Lineage depletion beads (Miltenyi Biotec 130-110-470) were added to the cells (50 µl per mouse) and incubated for 10 min at 4 °C. After incubation, the cells were loaded onto an LS magnetic separation column (Miltenyi Biotec 130-042-401), which was subsequently washed with 3 × 3 ml of FACS buffer. The cells were then incubated with the remaining antibodies for an additional 20 min on ice, followed by washing and FACS analysis. Blood samples were collected from the same mice for the isolation of CD8a+ T cells (CD3+ CD8a+) and B (CD19+) cells. Peripheral blood mononuclear cell (PBMC) isolation was performed using a SepMate-15 tube (STEMCELL Technologies 85415) according to the manufacturer's instructions. Enriched PBMCs were resuspended in FACS buffer and incubated with either T cell antibodies (anti-CD3-BV711, BioLegend 100241, 1:100; anti-CD8a-BV605, BioLegend 100743, 1:100 dilution) or B cell antibodies (anti-CD19-BV605, BioLegend 115539, 1:100 dilution) on ice for 20 min. The cells were then washed with FACS buffer and analyzed on a BD FACSAria II using FACSDiva software (v.9.7). Flow cytometry data were analyzed with FlowJo (v.10.9.0). 10b, 20,000 sorted cells from each bone marrow and blood population noted in ‘Flow cytometry' were lysed in RNA lysis buffer (RLT) and subjected to RNA extraction using the RNeasy Plus Micro kit (QIAGEN, 74034). RNA was then reverse transcribed into cDNA with SuperScript III First Strand Synthesis kit (Thermo Fisher Scientific, 11752-050) according to the manufacturer's instructions. Actb was used as an internal control. 10c–e were collected from C57BL/6 mice, cleaned with cold PBS and fixed in 10% neutral buffered formalin at 4 °C overnight. Protease Plus (Advanced Cell Diagnostics, 322331) was used for tissue pre-treatment. Following the last round of in situ hybridization imaging, fluorophores were cleaved using fresh 10% cleaving solution v.2. The intestinal tissues were then subjected to immunofluorescence staining. In brief, tissues were washed with PBS, permeabilized with 0.1% Triton X-100 in PBS and then blocked with 5% bovine serum albumin (BSA) in PBS for 30 min at room temperature. The tissues were then incubated with anti-E-Cadherin-Alexa Fluor 488 antibody (BD Biosciences 560061, 1:50 dilution) diluted in staining buffer (5% BSA in PBS with 0.1% Triton X-100) for 1 h at room temperature, followed by washing and imaging. All fluorescence images were acquired on a Zeiss LSM 980 confocal microscope. To quantify colocalization, cells along the crypts–villi axis were first categorized into different cell zones, as described in the caption of Extended Data Fig. Mean fluorescence intensities were then determined using ImageJ (v.1.53t). Relationships between two ordered variables were assessed by correlation tests or linear regression. Unless otherwise stated, statistical significance for Kendall correlations was determined by a two-sided z-test. Two-group comparisons were assessed using unpaired or paired tests, as appropriate. No statistical method was used to predetermine sample size. Data analyses were performed using Python (v.3.9.0) and R (v.4.2.0+). Software packages and versions specific to each analysis are detailed in the Methods. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All datasets comprising the single-cell potency atlas assembled in this work (Supplementary Table 1) are publicly available from GEO, ArrayExpress or the SRA with the following accession codes: GSE52583 (‘AT2/AT1 lineage (C1)'), GSE109774 (‘Bone marrow (10x)', ‘Bone marrow (Smart-seq2)' and ‘Tabula Muris (Smart-seq2/10x)'), GSE60783 (‘Dendritic cells (C1)'), GSE97391 (‘Direct in vitro neuron (inDrop)' and ‘Standard in vitro neuron (inDrop)'), GSE70245 (‘HSPCs (C1)'), GSE113197 (‘Human breast 1 (10x)' and ‘Human breast 1 (C1)'), GSE161529 (‘Human breast 2 (10x)'), GSE36552, (‘Human embryo (Tang et al.) (ref. 60)'), SRP073767 (‘Peripheral blood (10x)'), GSE128639 (‘BM-MNC (CITE-seq)'), GSE100866 (‘Cord blood (CITE-seq)'), E-MTAB-9067 (‘HSC development (Smart-seq2)'), GSE90742 (‘HSCs and MPPs (inDrop)'), E-MTAB-11536 (‘Immune cell atlas (10x)'), GSE76408 (‘Lgr5-CreER intestine (CEL-seq)'), E-MTAB-3321 (‘Mouse embryo 2 (Smart-seq2)'), GSE59892 (‘Mouse embryo 3 (Smart-seq)'), GSE162044 (‘Neural crest (Smart-seq2)'), GSE132188 (‘Pancreas (10x)'), GSE99933 (‘Peripheral glia (Smart-seq2)'), GSE122466 (‘Retinal neurons (10x)'), GSE64447 (‘Skeletal stem cell (C1)') and GSE201333 (‘Tabula Sapiens (Smart-seq2/10x)'). Raw FASTQ or BAM files analyzed in this work are available from the SRA with the following accessions: SRP188993 (‘BM-MNC (CITE-seq)'), SRP168426 (‘Retinal neurons (10x)'), SRP200419 (‘Pancreas (10x)'), SRP109011 (‘Peripheral glia (Smart-seq2)'), SRP239468 (‘Skeletal stem cell (C1)'), SRP094420 (‘HSCs and MPPs (inDrop)') and SRP476153 (‘Mouse neurogenesis (10x)' and ‘Mouse mature neural cell types (10x)'). Five expression datasets covering mouse embryogenesis periods from implantation to organogenesis are accessible from GEO or ArrayExpress with the following accessions: GSE100597 (‘Implantation (E3.5–E6.5)'), GSE109071 (‘Implantation (E5.5–E6.5)'), E-MTAB-6967 (‘Gastrulation (E6.5–E8.5)'), GSE186069 (‘Organogenesis (E8.5)'), and GSE228590 (‘Organogenesis (E8.75–P0)'). The publicly available oligodendroglioma and AML expression data analyzed in this work are available with GEO accession numbers GSE70630 and GSE116256, respectively. Reference genomes and annotation files for GRCm39 (mouse) and GRCh38.p13 (human) were obtained from Ensembl release 109 (February 2023) via the archive at https://feb2023.archive.ensembl.org. R and Python packages for running CytoTRACE 2 with the pre-trained model are freely available for non-profit academic use at https://github.com/digitalcytometry/cytotrace2. Both packages implement optional parallel processing for efficient execution and provide built-in plotting functions (UMAPs and box plots). Documentation, vignettes and input examples are provided. An interactive RShiny web application is available, allowing users to: • Run CytoTRACE 2 on user-provided datasets via an intuitive, interactive interface • Browse CytoTRACE 2 results for 33 datasets with ground truth potency annotations • Explore potency-associated genes learned by CytoTRACE 2 and investigate potency enrichment of user-defined genes and gene sets across the single-cell potency atlas This application can be accessed at https://cytotrace2.stanford.edu. Qiu, C. et al. A single-cell time-lapse of mouse prenatal development from gastrula to birth. Gulati, G. S. et al. Single-cell transcriptional diversity is a hallmark of developmental potential. La Manno, G. et al. RNA velocity of single cells. & Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. Reversed graph embedding resolves complex single-cell trajectories. Lange, M. et al. CellRank for directed single-cell fate mapping. Weiler, P., Lange, M., Klein, M., Pe'er, D. & Theis, F. CellRank 2: unified fate mapping in multiview single-cell data. Rudin, C. Stop explaining black box machine learning models for high stakes decisions and use interpretable models instead. Hubara, I., Courbariaux, M., Soudry, D., El-Yaniv, R. & Bengio, Y. Binarized neural networks. Stuart, T. et al. Comprehensive integration of single-cell data. Teschendorff, A. E., Maity, A. K., Hu, X., Weiyan, C. & Lechner, M. Ultra-fast scalable estimation of single-cell differentiation potency from scRNA-seq data. Teschendorff, A. E. & Enver, T. Single-cell entropy for accurate estimation of differentiation potency from a cell's transcriptome. Herman, J. S., Sagar & Grün, D. FateID infers cell fate bias in multipotent progenitors from single-cell RNA-seq data. Malta, T. M. et al. Machine learning identifies stemness features associated with oncogenic dedifferentiation. Li, Q. scTour: a deep learning architecture for robust inference and accurate prediction of cellular dynamics. Zhang, F. et al. FitDevo: accurate inference of single-cell developmental potential using sample-specific gene weight. Cheng, S. et al. Single-cell RNA-seq reveals cellular heterogeneity of pluripotency transition and x chromosome dynamics during early mouse development. & Sandberg, R. Single-cell RNA-seq reveals dynamic, random monoallelic gene expression in mammalian cells. Mohammed, H. et al. Single-cell landscape of transcriptional heterogeneity and cell fate decisions during mouse early gastrulation. Pijuan-Sala, B. et al. A single-cell molecular map of mouse gastrulation and early organogenesis. Systematic reconstruction of cellular trajectories across mouse embryogenesis. Zeng, A. G. X. et al. A cellular hierarchy framework for understanding heterogeneity and predicting drug response in acute myeloid leukemia. Tirosh, I. et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Abdelaal, T. et al. A comparison of automatic cell identification methods for single-cell RNA sequencing data. Cao, X. et al. A systematic evaluation of supervised machine learning algorithms for cell phenotype classification using single-cell RNA sequencing data. & Powell, J. E. scPred: accurate supervised method for cell-type classification from single-cell RNA-seq data. The Tabula Sapiens: a multiple-organ, single-cell transcriptomic atlas of humans. Architecture of the human regulatory network derived from ENCODE data. Lachmann, A. et al. ChEA: transcription factor regulation inferred from integrating genome-wide ChIP-X experiments. The Molecular Signatures Database hallmark gene set collection. The Oct4 and Nanog transcription network regulates pluripotency in mouse embryonic stem cells. Large-scale in vivo CRISPR screens identify SAGA complex members as a key regulators of HSC lineage commitment and aging. Identification of stem cells in small intestine and colon by marker gene Lgr5. Capdevila, C. et al. Time-resolved fate mapping identifies the intestinal upper crypt zone as an origin of Lgr5+ crypt base columnar cells. & He, C. Concise review: regulation of stem cell proliferation and differentiation by essential fatty acids and their metabolites. Jin, H.-R. et al. Lipid metabolic reprogramming in tumor microenvironment: from mechanisms to therapeutics. Schaum, N. et al. Single-cell transcriptomics of 20 mouse organs creates a Tabula Muris. The effects of death and post-mortem cold ischemia on human tissue transcriptomes. Systematic determination of the mitochondrial proportion in human and mice tissues for single-cell RNA-sequencing data quality control. Lotfollahi, M. et al. Biologically informed deep learning to query gene programs in single-cell atlases. Kunes, R. Z., Walle, T., Land, M., Nawy, T. & Pe'er, D. Supervised discovery of interpretable gene programs from single-cell data. & David, J.-P. BinaryConnect: training deep neural networks with binary weights during propagations. 28th International Conference on Neural Information Processing Systems. Andreatta, M. & Carmona, S. J. UCell: robust and scalable single-cell gene signature scoring. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. & Salakhutdinov, R. Dropout: a simple way to prevent neural networks from overfitting. Milad, A., Javier, F.-M., Nicholas, D. L. & Yarin, G. A systematic study of binary neural networks' optimisation. In International Conference on Learning Representations (ICLR, 2019). Latent weights do not exist: rethinking binarized neural network optimization. of the 33rd International Conference on Neural Information Processing Systems 7533–7544 (NeurIPS, 2019). & Verleysen, M. Classification in the presence of label noise: a survey. Determining the potency of primordial germ cells by injection into early mouse embryos. van Galen, P. et al. Single-cell RNA-seq reveals aML hierarchies relevant to disease progression and immunity. Gavish, A. et al. Hallmarks of transcriptional intratumour heterogeneity across a thousand tumours. Hänzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-Seq data. Zaro, B. W. et al. Proteomic analysis of young and old mouse hematopoietic stem cells and their progenitors reveals post-transcriptional regulation in stem cells. Tang, F. et al. mRNA-Seq whole-transcriptome analysis of a single cell. Mitiku, N. & Baker, J. C. Genomic analysis of gastrulation and organogenesis in the mouse. The authors acknowledge A. Gentles, S. Sikandar, A. Usmani, N. Earland, R. Fields and D. Chen for providing critical feedback on this manuscript. We are grateful to M. Haney for sharing CRISPR screening data, A. Alizadeh and C.L. This work was supported by a Stanford Bio-X Interdisciplinary Graduate Fellowship (M.K. is a Chan Zuckerberg Biohub – San Francisco Investigator. These authors contributed equally: Minji Kang, Gunsagar S. Gulati, Erin L. Brown, Zhen Qi. Minji Kang, Erin L. Brown, Zhen Qi, Jose Juan Almagro Armenteros, Rachel Gleyzer, Wubing Zhang, Jeremy Philip D'Silva, Janella Schwab, Michael F. Clarke & Aaron M. Newman Minji Kang, Erin L. Brown, Susanna Avagyan, Rachel Gleyzer, Wubing Zhang, Jeremy Philip D'Silva, Janella Schwab & Aaron M. Newman Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA, USA Department of Radiation Oncology, Mayo Clinic, Rochester, MN, USA Mayo Clinic Comprehensive Cancer Center, Rochester, MN, USA conceived of the study, developed strategies for related experiments and wrote the paper with substantial assistance from Z.Q. assisted with the initial development of CytoTRACE 2. performed experimental validation experiments under the supervision of A.M.N. have a patent application (US PCT/US25/18429) related to the identification of developmental states from scRNA-seq data. has ownership interests in CiberMed and LiquidCell Dx. has ownership interests in Droplet Biosciences and LiquidCell Dx. All other authors declare no competing interests. Nature Methods thanks Sridhar Hannenhalli and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Madhura Mukhopadhyay, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic overview of the CytoTRACE 2 core model, focusing on architectural details and key operations between the input layer and output layer for a single gene set binary network (GSBN) module. Note that six GSBN modules, one per broad potency category, are included in the full model, as illustrated in Fig. b, Serial impact of three postprocessing procedures (“Postprocessing” in Methods) on the accuracy of predicting (i) relative developmental orderings (n = 33 systems; bottom) and (ii) potency classes (F1 score, mean-aggregated across six broad potency categories; top) on training datasets (Methods). All results were obtained via leave-one-out cross-validation (LOOCV). All training datasets were analyzed using LOOCV (Supplementary Table 1). d, Illustration of the difference between absolute and relative developmental potential using two hypothetical scRNA-seq datasets, one spanning totipotent through pluripotent cells (‘Embryo') and the other encompassing multipotent through differentiated cells (‘Blood'). e-f, Impact of hyperparameter values on training set performance, showing weighted accuracy for single-cell potency classification using nested LOOCV (Supplementary Table 6; Methods). Right: Same as the left but showing weighted accuracy plotted by the number of gene sets per GSBN module. f, Weighted accuracy of models trained using distinct gene set enrichment procedures (AMS and/or UCell) after selecting robust hyperparameter values as described in Methods. Note that tissue-specific expression matrices from plate- and droplet-based platforms within Tabula Muris43 were analyzed individually in panels b and c for clarity, yielding 33 total systems with known developmental orderings. a, Performance of CytoTRACE 2 for reconstructing relative developmental trajectories (left) and six broad potency categories (right) in 33 ground truth scRNA-seq datasets (Fig. Performance was evaluated at the single-cell level using weighted Kendall correlation (τ), as described in Supplementary Table 5 and Methods. To promote a fair comparison, we evaluated absolute order correlations (right) with and without the inclusion of totipotent and pluripotent cells, as the corresponding potency categories were not available for human and droplet-only datasets in the test cohort. ns, not significant (two-sided unpaired Wilcoxon test). b, Same as panel a (left) but showing relative order performance stratified by developmental clade (Fig. 1b) with three additional splits, the latter of which were randomized to balance potency categories between cohorts (Supplementary Table 8; Methods). Performance was evaluated with four metrics, each calculated at the single-cell level in held-out data: absolute order (weighted τ), relative order (median weighted τ across evaluable systems with known developmental orderings), multiclass F1 for predicting broad potency classes (n = 6), and one minus the mean absolute error (MAE) for predicting broad potency classes (n = 6). a-c, Impact of perturbing potency labels in the training set. a, Heat maps depicting individual potency labels and their transition probabilities at different perturbation levels (Methods). b-c, Performance of CytoTRACE 2 models trained on potency labels with defined perturbation levels (as illustrated in a and described in Methods) applied to individual cells (b) or phenotypes (c) in the training set. Performance in held-out test data was evaluated with four metrics: absolute order (weighted τ of broad potency levels across datasets), relative order (median weighted τ of each dataset analyzed individually), multiclass F1 for predicting broad potency classes (n = 6), and the mean absolute error (MAE) for predicting broad potency classes (n = 6). Absolute order (τ), relative order (τ), and F1 score are expressed as a percentage of the results obtained with the unperturbed CytoTRACE 2 model. Each point represents the mean across five replicates of random perturbation. d, Analysis of robustness to the number of genes per cell using all test datasets (n = 14) (Supplementary Table 1) assessed with the same metrics as panels b and c. Results for each dataset represent the average across five rounds of gene count downsampling and are expressed relative to results with no downsampling. e, Same as panel d but shown for UMIs per cell in evaluable test datasets (n = 7). Eleven phenotypes spanning a range of potencies were titrated in defined amounts (x-axis) while other cells were left unchanged. CytoTRACE 2 was then applied to predict potency scores. Points represent averages from five random samplings (without replacement) per phenotype and error bars represent 95% confidence intervals. 1g but focusing on mouse cranial neural crest cells profiled by Zalc et al.11. Left inset: Log expression levels of core pluripotency factor Pou5f1 in mouse cranial neural crest cells (CNCCs). Right inset: Cells predicted as pluripotent by CytoTRACE 2. Cells predicted as pluripotent by CytoTRACE 2 showed significantly higher Pou5f1 expression than others, with P = 2.6 × 10−5 by two-sided Wilcoxon test. a, Overview of single-cell expression datasets analyzed in b-e and corresponding developmental time points profiled (n = 62). Icons denoting key stages of mouse embryogenesis were created using BioRender.com. b, Linearity between the average CytoTRACE 2 potency score per time point (weighted equally across author-annotated phenotypes) expressed in rank space (y-axis) and the corresponding time points (n = 62; x-axis). Concordance was calculated using linear regression (dashed line) and Kendall correlation (τ), with the latter weighted by the number of time points per embryonic day. c, Scatter plot comparing the performance of CytoTRACE 2 to previous approaches for reconstructing the temporal hierarchy of 45 time points spanning organogenesis (beginning at E8.061) to birth (y-axis) versus 17 time points preceding organogenesis (x-axis). Correlations are weighted by whole day intervals to account for imbalances in the number of evaluable time points per day. Point sizes represent the average weighted Kendall correlation per approach. d, Top: Data-driven lineage tree of mouse embryogenesis, where nodes represent cell types (n = 259), edges represent developmental transitions inferred by Qiu et al.2, and colors represent the corresponding rank distance from each cell type to the root (“Ground truth”). Center and bottom: Same as top, but with CytoTRACE 2 and CytoTRACE 1 predictions each averaged by phenotype, then rank-ordered along the path to the root. Distances were averaged for cell types with multiple direct paths to the root. e, Scatter plots showing all distances in d, with concordance between CytoTRACE methods (center and bottom panels of d) and lineages inferred by Qiu et al.2 (top panel of d). The significance of τ in b and e was determined using a two-sided Z-test. a, Box plots showing relative expression levels of cell state signatures from patients with acute myeloid leukemia (AML)26 in 13,445 AML cells stratified by potency categories identified by CytoTRACE 2. Each point denotes a single gene from the corresponding gene set ID indicated above the plot (Supplementary Table 10). Genes were internally normalized within each tumor sample as the mean log2 fold change (FC) within a given potency category versus the remaining cells in the tumor, as described in Methods, then z-score normalized (standardized) across potency categories. The four signatures, LSPC-Primed, LSPC-Quiescent, GMP-like, and Mono-like, are expected to be most highly expressed in multipotent, multipotent, oligopotent, and unipotent/differentiated cells, respectively (Supplementary Table 10). Statistical significance comparing the expected potency level(s) with each other potency level was determined by a two-sided Wilcoxon test. b, Left: Scatter plot of oligodendroglioma cells from six tumors organized by previously described stemness and lineage enrichment scores27. Right: Stacked bar plot showing how the fractional representation of cells with predicted potency categories (CytoTRACE 2) changes as a function of author-supplied stemness scores (y-axis). Cells predicted to have the highest oligo- and multilineage potential by CytoTRACE 2 correspond to those annotated as stem-like by Tirosh et al.27. a, Box plots comparing the performance of CytoTRACE 2 against eight baseline methods (supervised machine learning models, including leading tools for reference-guided annotation of scRNA-seq data) implemented for single-cell potency classification (Methods). Four-fold cross-validation was performed for each method, where each point represents performance in one fold of held-out data (the original training-test split [Fig. Performance was assessed at the single-cell level using multiclass F1 (left) and one minus the mean absolute error (MAE; right) for predicting broad potency classes (n = 6). b, Scatter plot comparing median performance scores for all methods from panel a. a-b, Representative test datasets comparing CytoTRACE 2 and scVelo. a, UMAP representation of mouse pancreas development (10x) (Supplementary Table 1). Left: Cells colored by ground truth granular potency level (Fig. Center: Cells colored by CytoTRACE 2 potency scores. Right: Cells colored by scVelo latent time (differential kinetics model). b, Same as a but showing human bone marrow (CITE-seq) (Supplementary Table 1). c, Left: Bar plot showing mean absolute order (weighted τ applied to single cells) performance across six broad potency levels (circles) and ten granular order potency levels (triangles) for nine test datasets evaluable by CytoTRACE 2 and scVelo (Supplementary Tables 3 and 14; Methods). Right: Violin and box plots showing relative order performance (weighted τ applied to single cells) on the same test datasets (n = 8 evaluable datasets with relative developmental orderings, Supplementary Tables 4 and 14). Statistical significance was determined by two-sided paired t test. Violin plot bounds denote minimum and maximum values. 1e but shown for the nine evaluable test datasets in c. Left: CytoTRACE 2 potency scores. Right: scVelo latent time (differential kinetics model). Statistical significance was determined using a one-sided Z-test. 2b but separated by cohort, species, cellular system (three general categories shown for clarity), and scRNA-seq platform. 2b is shown as a reference in the upper left. Colors denote potency scores (same as the color bar in Fig. b, Heat map depicting pairwise similarity of gene sets learned by CytoTRACE 2 across all 19 ensemble models from leave-one-out cross-validation on the 19-dataset training cohort. Overlap was quantified by Jaccard index and stratified into gene sets with positive (left, n = 1,490) and negative weights (right, n = 1,246); gene set polarity was determined as described in “Interpretability,” Methods. 2e but showing the consistency between CytoTRACE 2 multipotency markers and hematopoietic stem cell (HSC) knockout (KO) phenotypes across a range of top \(k\) markers, whether positive or negatively associated with multipotency (\(k\) = 50, 100, 200, and 500). GSEA statistics are expressed as directed –log10 Q values. Statistical significance between groups was determined using a two-sided unpaired Wilcoxon test. d, Same as c, but showing the median directed –log10 Q value across all top \(k\) markers shown in c, stratified by positive and negative markers, and extended to all potency categories in the CytoTRACE 2 feature importance matrix (Supplementary Table 15). e, Enrichments of selected gene sets from MSigDb in the CytoTRACE 2 feature importance matrix (Fig. All –log10 adjusted p-values, including those exceeding the color bar range, are provided in Supplementary Table 17. Bubble sizes are proportional to unsigned –log10 adjusted p-values within the color bar. a, Representative gating schemes for FACS-purification of mouse hematopoietic subsets analyzed in Fig. KLS, Kit+ Lin− Sca1+ multipotent cell subset consisting of HSCs and MPPs (multipotent progenitors); KL, Kit+ Lin− Sca1− subset devoid of multipotent cells; CMP, common myeloid progenitor; CLP, common lymphoid progenitor. b, Bar plots showing biological replicates and controls for quantitative PCR experiments, related to Fig. Each gene is shown normalized to the maximum mean expression across all groups. Top: markers of HSC/MPP (Hoxb5, Fgd5, Procr), progenitors (Cd34), and differentiated lineages (Cd8a, Cd19). Bottom: unsaturated fatty acid (UFA) synthesis genes identified as markers of multipotency by CytoTRACE 2 (Fads1, Fads2, and Scd2). Actb was used as an internal control. (standard error of the mean) across three biological replicates. 2j but shown for mouse duodenum (c) and ileum (d). e, Quantification of mRNA hybridization signal in multipotent and unipotent/differentiated zones of mouse jejunum (left, corresponding to images in Fig. 2j), duodenum (center, corresponding to confocal images in c), and ileum (right, corresponding to images in d). Multipotent zones are divided as previously described40 on the basis of cell location from the crypt base, with red and green regions expected to be most enriched in Lgr5 and Fgfbp1, respectively. (n = 20 paired crypts and villi from each intestinal region [jejunum, duodenum, ileum] pooled from two mice, with a total of 10 paired crypts and villi per region per mouse). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Improved reconstruction of single-cell developmental potential with CytoTRACE 2. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-64512-w'>PLM-interact: extending protein language models to predict protein-protein interactions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 10:36:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Computational prediction of protein structure from amino acid sequence alone has been achieved with unprecedented accuracy, yet the prediction of protein-protein interactions remains a challenge. Here, we assess the ability of protein language models (PLMs), routinely applied to protein folding, to be retrained for protein-protein interaction prediction. Existing models that exploit PLMs use a pre-trained PLM feature set, ignoring that the proteins are physically interacting. We propose PLM-interact, which goes beyond single proteins by jointly encoding protein pairs to learn their relationships, analogous to the next-sentence prediction task from natural language processing. This approach achieves state-of-the-art performance in a widely adopted cross-species protein-protein interaction prediction benchmark: trained on human data and tested on mouse, fly, worm, E. coli and yeast. In addition, we develop a fine-tuning method for PLM-interact to detect mutation effects on interactions. Finally, we report that the model outperforms existing approaches in predicting virus-host interaction at the protein level. Our work demonstrates that large language models can be extended to learn the intricate relationships among biomolecules from their sequences alone. Proteins are the main structural components of cells and mediate biological processes by interacting with other proteins1. Disruption of these protein-protein interactions (PPIs), e.g., mediated by mutations, can underlie human disease2. In virology, PPIs are particularly important as viruses depend entirely on the host cell for replication, achieved mainly through specific interactions with host proteins. In response to infection, our immune system counteracts pathogens via targeted PPIs. Understanding PPI mechanisms offers the potential for developing novel therapy strategies for both human disease and pathogen infections3. Unfortunately, experimentally identifying PPIs is both costly and time-consuming, such that interaction datasets remain sparse with only a few species having comprehensive coverage4,5. Computational algorithms offer an efficient alternative to the prediction of PPIs at scale. Existing prediction approaches mainly leverage protein properties such as protein structures, sequence composition and evolutionary information6,7,8,9. Applying these features to pairs of proteins, classifiers have been trained using classical machine learning10 and deep learning approaches11. Recently, protein language models (PLMs) trained on large public protein sequence databases have been used for encoding sequence composition, evolutionary and structural features12,13,14, becoming the method of choice for representing proteins in state-of-the-art PPI predictors. Despite the use of PLMs in PPI prediction, identifying PPIs remains challenging. a Typical PPI prediction models use pre-trained ‘frozen' protein language models to extract single-protein embeddings with a trainable interaction classifier. Both the mask language modelling task and a binary classification task predicting interaction status are used to train the model. The main issue is PLMs are primarily trained using single protein sequences, i.e., while they learn to identify contact points within a single protein16, they are not ‘aware' of interaction partners. In a conventional PLM-based PPI predictor architecture, a classification head is used to extrapolate the signals of inter-protein interactions by grouping common patterns of intra-protein contacts in interacting and non-interacting pairs, respectively (Fig. However, this strategy relies on the classification head being generalisable. Unfortunately, with the use of ‘frozen' embeddings and a feedforward neural network being the dominant option, these classifiers have limited parameters to deal with complex interaction patterns. To address the lack of inter-protein context in model pre-training, we propose PLM-interact, which directly models PPIs by extending and fine-tuning a pre-trained PLM, ESM-217. PLM-interact (trained on human PPI data) achieves a significant improvement compared to other predictors when applied to mouse, fly, worm, yeast and E. coli datasets, and can be applied to virus-host PPI prediction. We also demonstrate that a fine-tuned version of PLM-interact can predict mutation effects on interactions. To directly model PPIs, two extensions to the widely used PLM ESM-217 are introduced (Fig. 1b): (1) longer permissible sequence lengths in paired masked-language training to accommodate amino acid residues from both proteins; (2) implementation of “next sentence” prediction18 to fine-tune all layers of ESM-2 where the model is trained with a binary label indicating whether the protein pair is interacting or not (see Methods for details). Our training task is, thus, a mixture of the next sentence prediction and mask language modelling tasks. This architecture enables amino acids in one protein sequence to be associated with specific amino acids from another protein sequence through the transformer's attention mechanism. We fine-tune it for PPIs by showing it pairs of known interacting and non-interacting proteins. In contrast to similar training strategies in machine learning18, we find that the next sentence prediction and mask language modelling objectives need to be balanced. We therefore conducted comprehensive benchmarking for different weighting options, before selecting a 1:10 ratio between classification loss and mask loss, combined with initialisation using the ESM-2 (with 650 M parameters), as this achieved the best performance (see Methods for details, Supplementary Fig. To examine the performance of PLM-interact, we benchmark the model against six PPI prediction approaches: TUnA19, TT3D13, Topsy-Turvy20, D-SCRIPT15, PIPR6 and DeepPPI21. We use a multi-species dataset created by Sledzieski et al.15. Each model is trained on human protein interaction data and tested on five other species. The positive PPIs in these datasets are experimentally-derived physical interactions, while the negative pairs are randomly paired proteins not reported to interact. PLM-interact achieves the highest AUPR (area under the precision-recall curve)22 followed by TUnA19 and TT3D13 (Fig. The predictions for yeast and E. coli PPIs are more challenging because they are more evolutionarily divergent from the human proteins used for training than the other species (see Fig. Our model achieved an AUPR of 0.706 on yeast, a 10% improvement over TUnA's AUPR of 0.641, and a 7% improvement on E. coli with an AUPR of 0.722; it also shows a 28% improvement over TT3D's AUPR of 0.553 on yeast and a 19% improvement over TT3D's AUPR of 0.605 on E. coli. a The data size of training, validation and test protein pairs. b The taxonomic tree of the training and test species is aligned with the precision-recall curve of each model on each test species. A bar plot of AUPR values illustrates the PPI prediction benchmark. The distribution of predicted interaction probabilities of positive and negative protein pairs for each PPI model is shown in Supplementary Fig. All species icons in panel (b) are created in BioRender (Liu, D., 2025; https://BioRender.com/1ezsj7q). Importantly, the improvement in PLM-interact is due to its ability to correctly identify positive PPIs: Comparing the predicted interaction probabilities, PLM-interact consistently assigned higher probabilities of interaction to true positive PPIs. In contrast, other methods give lower interaction probability estimates in all held-out species. The distribution of predicted interaction probabilities of positive and negative protein pairs for each model is shown in Supplementary Fig. Next, we showcase five positive PPI instances, one for each test species, for which our model produces a correct prediction, and both TUnA and TT3D produce incorrect predictions (Fig. These PPIs are necessary for essential biology processes including inducing leukaemia cell differentiation23, dynein light chain roadblock24, RNA polymerisation25, import of protein precursors into mitochondrial26 and protein transportation27. We use Chai-128 and AlphaFold329 to predict and visualise these interacting protein structures – these visualisations are shown in Fig. The self-assessment scores pTM and ipTM from Chai-1 and AlphaFold3 are reported in Supplementary Table 2. Protein-protein structures are predicted by Chai-128 and visualised with ChimeraX45. The predicted interaction probabilities of PPI models range between 0 and 1. Interacting proteins are shown from left (yellow) to right (green), respectively. Mouse: P97287 (Induced myeloid leukaemia cell differentiation protein Mcl-1 homologue) and P63028 (Translationally-controlled tumour protein); Fly: Q9W0F0 (Dynein light chain roadblock) and Q7K035 (AT23443p); Worm: Q21955 (Mediator of RNA polymerase II transcription subunit 15) and Q9N4F2 (Mediator of RNA polymerase II transcription subunit 19); Yeast: P23644 (Mitochondrial import receptor subunit TOM40) and P53507 (Mitochondrial import receptor subunit TOM7); and E. coli: A0A454A7G5 (ABC transporter permease protein) and A0A454A7H5 (Possible ABC-transport protein, ATP-binding component). 4 for the corresponding AlphaFold329 predicted structures. The ipTMs of both Chai-1 and AlphaFold3 for each structure are shown in Supplementary Table 2. To investigate if the order of protein in each test pair has an impact on the prediction results, we also perform inference on the test protein pairs with the reversed order. We observe almost identical AUPR performance (Supplementary Fig. 5) and predicted interaction probability distributions (Supplementary Fig. To probe the role of sequence identity on PLM-interact's performance, we evaluate the model's performance with the different protein identities between training and test datasets (see “Methods” for details). As expected, we observe marked sequence similarity between human and mouse, similarities reduce significantly between fly, worm, yeast and E. coli with human. Notably, both PLM-interact and the second-best performer, TUnA, in the cross-species benchmark benefit from sequence similarity. Their performance improves as protein identity increases, with PLM-interact consistently outperforming TUnA (Supplementary Fig. To further evaluate our model's performance in relation to sequence similarity, we train PLM-interact on a leakage-free human ‘gold' standard training dataset created by Bernett et al.30 and compare with the state-of-the-art PPI approaches. In this benchmarking dataset, there are no overlaps and minimal sequence similarities among the training, validation and test datasets. Due to computing limitations on the maximum sequence length of a pair, we only use 80% of the training set for training. 4, where PLM-interact exhibits identical AUPR (0.69) and AUROC (0.7) to TUnA. Interestingly, when adopting a neutral 0.5 threshold on predicted interaction probabilities for final classification, PLM-interact outperforms TUnA and other baselines in F1-score and recall. The improvement in recall is 9% over TUnA, while the precision is comparable with TUnA, indicating PLM-interact performs better at predicting positive interactions. a The x-axis shows the evaluation metrics (AUPR, Precision, Recall, AUROC and F1-score) for six PPI models: PLM-interact, TUnA, Topsy-Turvy, D-SCRIPT, PIPR and DeepPPI, the y-axis represents the corresponding metric values. b The distribution of predicted interaction probabilities of PLM-interact and TUnA for the positive and negative protein pairs, respectively. Here, we examine PLM-interact's ability to predict the mutation effect on interactions. We use mutation data from IntAct31, specifically, mutations that increase (IntAct ID: MI:038232) or decrease (IntAct ID: MI:011933) interaction rate or strength of binding (Fig. These collectively give us 6,979 total annotated mutation effects. Each annotated mutation effect sample consists of the wild type and mutant sequences of one protein and its interacting protein, which is always in its wild type, i.e., only a single protein is mutated in a PPI. The effect is treated as a binary label for increasing (+) or decreasing interaction (-). To make predictions on the mutation effect, we compute the log-predicted interaction probability ratio between the mutant and canonical (wild type) pairs. A positive log ratio indicates the increasing interaction class and negative otherwise (Fig. This strategy is similar to the log-likelihood ratio approach for variant effect prediction in single proteins34,35. a Diagrammatic overview of the binary mutation effect classification task. b Inference and fine-tuning of PLM-interact to predict mutation effects that increase or decrease interaction rate/strength. c Precision-Recall curves of two fine-tuned PLM-interact models and four zero-shot models (PLM-interact, TUnA, Topsy-Turvy and D-SCRIPT) on the test dataset. We develop a fine-tuning strategy for predicting mutation effect on interactions (see Fig. Our method leverages the log-predicted interaction probability ratio as input to a cross-entropy loss, allowing the gradient to be backpropagated to update all layers of PLM-interact. This training allows the model to calibrate changes in predicted interaction probability in mutant case in relation to the canonical scenario. We construct training, validation and testing sets from the combined IntAct increasing and decreasing interaction set (see Datasets for details) and benchmarked two fine-tuned PLM-interact and four models without fine-tuning (i.e., zero-shot), including TUnA, Topsy-Turvy, D-SCRIPT, and a zero-shot PLM-interact (Fig. We find all zero-shot models to perform poorly, with close to random performance in AUPR and AUROC, despite the overlapping proteins existing between the human PPI training set and the mutation PPI test set. Remarkably, we observe marked improvement (150% in AUPR and 36% AUROC) when fine-tuning all layers of PLM-interact. To assess how traditional methods perform in the fine-tuning setting, we also train a version of PLM-interact where only the classification head is fine-tuned; the result is significantly worse than fine-tuning the full model. We show an example of a successfully predicted mutation that increases interaction strength or rate in Fig. 6a, the DNA replication licensing factor MCM7, which is important for DNA replication in human cells. MCM7 is reported as a biomarker in human cancers such as hepatocellular carcinoma and lung cancer36,37. MCM7 Y600 phosphorylation is associated with breast cancer, and the mutant Y600E of MCM7 enhances the interaction levels with MCM members38. PLM-interact predicts a positive log ratio of 0.165 between the mutant PPI and the canonical PPI, correctly suggesting an increase in interaction. These PPI structures are predicted using Chai-128 and visualised with ChimeraX45; here, the mutated amino acids are highlighted in purple. In each panel, the log-predicted interaction probability ratio between the mutant and canonical protein pairs is shown for fine-tuned PLM-interact and zero-shot TUnA and Topsy-Turvy, respectively. The positive log ratio indicates a mutation-increasing PPI, while the negative log ratio indicates a mutation-decreasing PPI. The ipTMs of both Chai-1 and AlphaFold3 for each structure are shown in Supplementary Table 3. Interacting protein structures are shown from left (yellow) to right (green). a Residue 600 Tyrosine (Y) of P33993 (DNA replication licensing factor MCM7) is mutated to Glutamic Acid (E), increasing its interaction with P33992 (DNA replication licensing factor MCM5). b Residue 151 Asparagine (N) of Q16595 (Frataxin, mitochondrial) is mutated to Alanine (A), decreasing interaction with Q9H1K1 (Iron-sulfur cluster assembly enzyme ISCU). 8 for the corresponding AlphaFold3 predicted structures. Next, we show an example of the mutation that decreases interaction (Fig. The protein Frataxin encoded by FXN is important for the synthesis of iron–sulfur cluster, and a mutation of FXN has been discovered to be associated with a neurodegenerative disease Friedreich's ataxia (FRDA)39. PLM-interact correctly predicts that the missense mutation N151A reduces the interaction probability (log ratio is − 0.649). To study virus-host PPI prediction, we train PLM-interact on a virus-human PPIs dataset from Tsukiyama et al.11. The dataset is derived from the Host-Pathogen Interaction Database (HPIDB) 3.041, and comprises a total of 22,383 PPIs, which include 5882 human and 996 virus proteins. We compare our model with three recent virus-human PPI models: PLM-based approach STEP14 and the protein embeddings-based approaches LSTM-PHV11 and InterSPPI42. STEP is similar to existing PPI models benchmarked previously in our study; it leverages protein sequence embedding extracted by the pre-trained PLM ProtBERT43. For the STEP comparison, this corresponds to improvements in AUPR, F1 and MCC scores of 5.7%, 10.9% and 11.9%, respectively (Fig. The length of virus, human proteins and the combined length of virus-human PPIs are shown in Fig. To further analyse our model's performance, we select three pairs of virus-human PPIs from our test data, all with corresponding experimental virus-human complex structures available in the HVIDB44. We then use ChimeraX45 to visualise these structures and present PLM-interact's predicted interaction probability for each example (see Fig. a Comparison of AUPR, F1 and MCC metrics of PLM-interact against recent virus-human PPI models. b The distribution of the length of virus proteins, human proteins and virus-human protein pairs. c The virus-human PPIs shown are correctly predicted by our model, and their 3D complex structures are experimentally verified and obtained from the human-virus PPI database (HVIDB44). From left (green) to right (yellow), these interacting protein structures are: Tumour necrosis factor receptor superfamily member 14 (Human protein: Q92956) with Envelope glycoprotein D (human herpes simplex virus 1: P57083), Ephrin-B2 (human protein: P52799) with Glycoprotein G (Nipah virus protein: Q9IH62) and Retinoblastoma-associated protein (human protein: P06400) with Large T antigen (Simian virus 40: P03070). Note: The metrics results of the other three models in panel (a) are taken from STEP14 paper. In this study, we have developed PLM-interact, a PPI predictor that extends single protein-focused PLMs to their interacting protein partner. We report significant improvements in held-out species comparisons and with further fine-tuning highlight successful examples of predicting mutational effects on protein interactions. We further demonstrate PLM-interact's performance in a virus-human PPI prediction task, showing a significant improvement over state-of-the-art prediction approaches. Notably, PLM-interact, based solely on a large language model approach, significantly outperform two baselines that incorporate multi-modal input, namely TT3D13 and Topsy-Turvy20. Inclusion of such additional features should further improve PLM-interact's performance. Furthermore, our fine-tuning experiments show the potential of predicting mutation effects on PPIs from sequence alone. This could lead to interaction-aware in-silico variant effect predictors where methods rely on PLMs of the single proteins34,47,48. However, current training data remains limited. The number of high-quality structures of mutant proteins and their interaction partners are low. Algorithmically, models with long and multimodal context49,50,51 that include multiple proteins, structures and nucleotides could be specialised for interaction tasks. Finally, effective sequence-based virus-host PPI predictors could provide the much-needed molecular detail to conventional virus-host species prediction tools, which tend to rely on genome composition signals, ignoring host molecules that are interacting physically with viral molecules52,53,54,55. In those approaches, the host species only acts as a label. Recent progress within SARS-CoV-2 PPI studies mapped out a complex interaction landscape between the virus and human proteome56,57. Other viruses are likely to have similarly complex interactions with human and animal hosts. Leveraging these interactions could lead to tools that are better at predicting zoonotic events and the potential for the emergence of novel viruses. While PLM-interact has demonstrated significant improvements, there is much to do in terms of generating reliable predictions, in particular, the need for high-quality virus-host experimental PPI data for training. What is clear is that attention-based large language models applied to longer-range sequence interactions are enhancing our understanding of both proteins and their interactions--the fundamental ‘language' of molecular biology. Overview of all datasets used in this paper: Cross-species dataset: The human training and five test datasets: mouse, fly, worm, yeast and E. coli from Sledzieski et al.15 are used for benchmarking between PLM-interact and the state-of-the-art PPI approaches (Fig. 2), and PPI model evaluation under the different protein sequence identity between training and test datasets (Supplementary Fig. Bernett dataset: The leakage-free human training, validation and test benchmarking dataset created by Bernett et al.30 is used to further benchmark PLM-interact and other PPI approaches (Fig. Mutation effect dataset: The mutation dataset collected from IntAct31 is used to fine-tune PLM-interact for predicting increasing or decreasing strengths of interactions associated with mutations (Figs. Virus-human PPI dataset: The virus-human benchmarking dataset from Tsukiyama et al.11 is used to compare PLM-interact and other virus-human PPI models (Fig. STRING V12 training dataset: The database STRING V1258 is used to construct a larger dataset for PLM-interact training. The benchmarking human PPI dataset, from Sledzieski et al.15, comprises human training and validation data, and test data from five other species: mouse, Mus musculus; fly, Drosophila melanogaster; worm, Caenorhabditis elegans; yeast, Saccharomyces cerevisiae; and E. coli, Escherichia coli, all retrieved from STRING V1159. We train and validate our model on human PPIs and then conduct inference on PPIs from five other species. All training, validation and test datasets maintain a 1:10 ratio of positive to negative pairs, reflecting the fact that positive PPIs are significantly fewer than negative pairs in PPI networks. Negative pairs are generated by randomly pairing proteins not reported to interact. The length of protein sequences ranges from 50 to 800, and PPIs are clustered at 40% identity using CD-HIT60 to remove the redundant PPIs. The human training dataset includes 38,344 positive PPIs, whereas the validation set includes 4794 positive PPIs. Each of the five species includes 5000 positive interactions, except for E. coli, which only has 2000 positive interactions due to the fewer positive PPIs in the STRING dataset used15. The human benchmarking dataset constructed by Bernett et al. explicitly to minimise data leakage, is also used to evaluate PPI models' performance30. The positive protein pairs are from the HIPPIE v2.361 human PPI dataset, and negative protein pairs are randomly selected, and the ratio between positive and negative pairs is 1:1. Their data-splitting strategy removes overlaps and minimises sequence similarity among the training, validation and test datasets. Due to the storage limitation, we train on protein pairs that have the maximum combined paired length of 2193, which covers 80% of this benchmarking training data. The mutation effect dataset is obtained from IntAct31. We collect mutations that increase (MI: 0382) or decrease (MI: 0119) interaction rate or strength. We remove PPIs that have the same canonical and participant proteins, i.e., self-interactions. The resulting dataset contains 1281 mutation-increasing PPI samples and 5698 mutation-decreasing PPI samples (Supplementary Fig. Due to GPU memory limitations on the length of protein pairs, we prioritise short protein pairs (< 2201 amino acids, ~ 80% of the data) to maximise the training size, while allowing the validation and test sets to have pairs with much longer combined lengths. In the end, we have 5103 training, 841 validation and 841 test samples. The benchmarking dataset of 22,383 virus-human PPIs includes 5882 human and 996 virus proteins. This dataset was obtained from Tsukiyama et al.11, sourced from the HPIDB 3.0 database41; the ratio of positive to negative pairs is 1:10 and negative pairs are chosen based on sequence dissimilarities. The length of protein sequences ranges from 30 to 1000, and the redundant PPIs are filtered based on a threshold of 95% identity using CD-HIT60. The processed dataset was split into training and test datasets with a ratio of 8:2. Our training and test split is identical to the one in Tsukiyama et al.11. In addition, we provide a model trained on human PPIs from STRING V1258. The positive PPIs are selected by collecting physical links with positive experimental scores, while excluding PPIs with positive homology scores and confidence scores below 400. Previous studies have typically limited the maximum length of protein sequences to 800 or 1000 due to GPU memory limitations. We process the training protein sequences with a combined length threshold for protein pairs of 2101. This human dataset includes 60,308 positive PPIs for training and 15,124 positive PPIs for testing. Furthermore, protein sequences are clustered at 40% identity using MMSeq262, and only PPIs from the distinct clusters are chosen to eliminate redundant PPIs. Again, the positive-to-negative protein pair ratio is 1:10, consistent with the aforementioned two benchmarking datasets. We use ESM-2 as the base model in PLM-interact. ESM-2 is an encoder transformer model with a parameter size range from 8 million to 15 billion. The results presented are PLM-interact based on ESM-2 with 650 M parameters. We also provide PLM-interact model checkpoints trained with ESM-2 35 M on our Hugging Face repository to help with testing. The input representation contains amino acid token representations from two proteins. This setup is similar to the original BERT model63, also known as the cross-encoder, which simultaneously encodes a pair of query and answer sentences. A standard input sequence of PLM-interact, \(x\), can be shown as the following: where \({CLS}\) is the classification token, \({P}_{1}\) contains amino acid tokens of protein 1, \({P}_{2}\) contains amino acid tokens of protein 2, and \({EOS}\) is the end-of-sentence token. This setting allows us to use the original ESM-2 tokenizer to generate embedding vectors \(e\), and pass them to the transformer encoder of the ESM-2: where \(f\) is ESM-2, \(e\) contains the token embeddings of \(x\), and \(h\) contains the output embeddings of all input tokens. where \({h}_{{a}_{1}}\) and \({h}_{{a}_{n}}\) represent amino acid tokens in proteins 1 and 2. Then, we use the \({CLS}\) token embedding to aggregate the representation of the entire sequence pair and as the features for a linear classification function \(\varphi\), and parameterised as a single feed-forward layer with a ReLU activation function. PLM-interact is trained with two tasks: (1) a mask language modelling (MLM) task predicting randomly masked amino acids and (2) a binary classification task predicting the interaction label of a pair of proteins. PLM-interact is trained for 10 epochs using a batch size of 128 on both benchmarking datasets of human PPIs and virus-human PPIs. For all training runs, the input protein pairs are trained using both orders as the interaction between protein 1 and protein 2 is the same as the protein 2 and protein 1, which leads to doubling of the training set size. The validation and testing sets are not subject to the same data argumentation. The learning rate is 2e-5, weight decay is 0.01, warm-up is 2000 steps, and the scheduler is WarmupLinear, which linearly increases the learning rate over the warm-up steps. During training, we evaluate the model's performance at every 2000 steps on the validation set. For every evaluation, a set of 128 protein pairs are randomly sampled from the validation set, and the results are averaged over 100 times to ensure metric reliability. Here, we use both masking and classification losses to optimise our model, the loss function for each data point \(l\) can be represented as: where \({l}_{{\mbox{mlm}}}\) and \({l}_{{ce}}\) are separately represent the MLM loss and classification (i.e., cross entropy) loss. \(\alpha\) and \(\beta\) are weights for the MLM and classification losses, and they are determined in the following “Technical benchmark for hyperparameter selection” section. All of the models are trained on the DiRAC Extreme Scaling GPU cluster Tursa. A typical 10-epoch training run of the model with ESM-2 (650 M) with human PPIs takes 31.1 h on 16 A100-80 GPUs. A typical 10-epoch training run of the model with ESM-2 (650 M) trained on virus-human PPIs takes 30.5 h on 8 A100-80 GPUs. The model with ESM-2 (650 M) trained on STRING V12 human PPIs used 16 A100-80 GPUs for 86.4 h. For model training time with different ratios and model sizes, see the following section, technical benchmark and Supplementary Table 1 for details. We provide model checkpoints that include human PPI models trained on the benchmarking dataset constructed by Sledzieski et al.15 and retrieved from STRING V1159, a human PPI model trained on a benchmarking dataset created by Bernett et al.30 and sourced from HIPPIE v2.361, a virus-human PPI model trained on the benchmarking virus-human PPIs created by Tsukiyama et al.11 and sourced from HPIDB 3.041, as well as a human PPI model trained on human PPIs that we collected from STRING V1258. To find the optimal value of \(\alpha\) and \(\beta\) in Eq. (6), we benchmark a range of different options between mask loss and classification loss on human benchmarking data. For each ESM-2-35M and ESM-2-650M model, we train five models with different settings of ratios \(\alpha\): \(\beta\) between mask loss and classification loss. The ratios are \(\alpha\): \(\beta\) = 1:1, 1:5, 1:10, 0:1 (with mask), and 0:1 (without mask, denoted as classification) (Supplementary Fig. Next, the final model is selected based on testing on five other host PPIs. We find that a ratio of 1:10 is the optimal choice for ESM-2-650M. The AUPR of E. coli shows a 4.3% improvement over the second-best model, while other species remain comparable or better with this 1:10 ratio (Supplementary Fig. According to these results, we select a loss ratio of 1:10 for ESM-2-650M. The ratio setting is implemented in benchmarking of human PPI training and virus-human PPI training, as well as human PPI training using the STRING V12 database. Typically, 15% tokens are masked out to train protein language models, such as ESM-1b47 and ESM-217. Given that PLM-interact deals with much longer sequence lengths than typical ESM-2, we test the model's performance under different masking percentages. We test a range of different masking percentages (7%, 15%, 22% and 30%) to train ESM-2-650M with a mask-to-classification loss ratio of 1:10. We compare this base model with masking models that are trained with different masking ratios to determine the benefit of masking percentages on performance. We use McNemar's test to evaluate if the masking model is significantly better than the binary model (p-value < 0.05). As this is an unbalanced binary classification task, we determine true and false predictions using the threshold that gives the best F1 score. Finally, we compare the AUPR performance of each model on five test species. The results are shown in Supplementary Fig. For each species, we show a line plot of AUPR performance with the different masking ratios: 0% (binary), 7%, 15%, 22% and 30%. A gold star is marked if the masking model is significantly better than the binary classification model (0% masking). Overall, we also observe insignificant differences among the masking percentages. Let \({x}_{{mutant}}\), \({x}_{{canonical}}\), and \({o}_{{effect}}\) be the mutant, canonical interaction pair representation and binary effect label, respectively. The basic data unit in this task contains a triplet \(\{{x}_{{mutant}},{x}_{{canonical}},{o}_{{effect}}\}\). Inference is performed based on computing the log-predicted interaction probability ratio between the mutant and canonical protein pairs: Where \(g\) is the predicted interaction probability in Eq. Both AUPR and AUROC are computed based on the log ratio itself. We use a binary cross-entropy loss to fine-tune PLM-interact as follows: We fine-tune two versions of PLM-interact models, one with all layers fine-tuned, the other with only the classification layer being updated. The latter is designed to mimic fine-tuning the traditional approach of a frozen pretrained protein encoder followed by a learnable classification head. We use the checkpoint trained with the human dataset from Sledzieski et al.15 for two versions of PLM-interact fine-tuning. Both fine-tuned models are trained for 40 epochs, and the best epoch is obtained when the validation loss reaches its minimum (Supplementary Fig. We compute the prediction interaction probabilities based on checkpoints of TUnA19, TT3D13, Topsy-Turvy20 and D-SCRIPT15 to generate precision-recall (PR) curves in Fig. Due to the absence of publicly available checkpoints for DeepPPI21 and PIPR6, these methods are excluded from the PR curve comparison. The AUPR value for TUnA is sourced from the TUnA paper, the AUPR values for DeepPPI and Topsy-Turvy are sourced from the Topsy-Turvy paper20, those for D-SCRIPT and PIPR are from the D-SCRIPT paper15, and the AUPR value of TT3D is obtained through email communication. For the benchmarking on the Bernett dataset30, we obtain the AUPR value of TUnA from the TUnA paper19 and other models' AUPR values are obtained from Bernett et al.30. A complete list of the main features, architectures, references and code links for each baseline method can be found in Supplementary Table 4. As for model comparison on the mutation effect prediction task, Topsy-Turvy and D-SCRIPT cannot handle protein pairs longer than 2000, resulting in 598 out of 841 test samples being used for prediction. Therefore, we show Precision-Recall and ROC curves under these 598 predictions in Fig. We use the sequence search and clustering tool MMseq2 Release 13-4511162 to obtain the protein sequence-based alignment results between each pair of proteins; the parameters setting is: --threads 128 --min-seq-id 0.4 --alignment-mode 3 --cov-mode 1. To evaluate each PPI model's performance with different levels of protein sequence similarities between training and test datasets, we use MMseq262 to obtain the protein sequence identity for each protein in the test pairs from five test species against all human training proteins, with identity values ranging from 0 to 100. The sequence identity between each test protein pair and training proteins is determined by the maximum identity of any protein in each test pair. 7, we report the AUPR values of PLM-interact and TUnA under different levels of identity “bins” [0, 20, 40, 60, 80, 100] between training and test proteins. Chai-128 is a state-of-the-art model for molecular structure prediction, available at https://lab.chaidiscovery.com/. We use Chai-1 with the “specify restraints” option to predict protein-protein structure complexes and visualise predicted PPI structures using the molecular visualisation programme ChimeraX-1.7.145. AlphaFold3 is a tool to predict the biomolecular interactions, including protein, DNA, small molecules, ions and modified residues29, available at https://alphafoldserver.com/. We use AlphaFold3 in its PPI mode to predict protein structure complexes. IPTM scores below 0.6 indicate failed predictions, while scores above 0.8 indicate high confidence predictions. pTM scores above 0.5 suggest that the predicted structures are similar to the ground truth. For more details about these self-assessment scores, refer to the AlphaFold329 paper and official description at https://alphafoldserver.com/faq#how-can-i-interpret-confidence-metrics-to-check-the-accuracy-of-structures. McNemar's test64 is a statistical test that determines if there are significant differences between paired nominal data. Here, b represents the count of correct predictions obtained by \({model}1\) and incorrect predictions by \({model}2\), while \(c\) represents the count of incorrect predictions obtained by \({model}1\) and correct predictions by \({model}2\). To investigate if our models with masking perform significantly differently from a binary model without masking, we conducted a McNemar's test for any models under different masking percentages with the binary classification model (0% masking). This test is based on the number of correct and incorrect between two models. Predicted interaction probabilities from each model are used to get predicted labels, which are used to obtain the counts of correct and incorrect predictions. A McNemar's test p-value < 0.05 indicates a significant difference between the predictive performance of two models. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The cross-species benchmarking dataset created by Sledzieski et al.15 is available at https://d-script.readthedocs.io/en/stable/data.html. The human dataset created by Bernett et al.30 is available at https://doi.org/10.6084/m9.figshare.21591618.v3. The virus-human benchmarking PPI dataset created by Tsukiyama et al.11 is available at http://kurata35.bio.kyutech.ac.jp/LSTM-PHV/download_page. Protein sequences are retrieved from UniProt66 (https://www.uniprot.org/). The 3D complex structures of human-virus PPIs are obtained from HVIDB44 (http://zzdlab.com/hvidb/download.php). The STRING V12 human training data are sourced from the STRING PPI database V1258 (https://stringdb-downloads.org/download/protein.physical.links.v12.0.txt.gz) and are available at https://huggingface.co/datasets/danliu1226/STRING_V12_TrainingSet. The training, validation and test datasets for the mutation effect classification task are sourced from the IntAct molecular interaction database31 (https://ftp.ebi.ac.uk/pub/databases/intact/current/various/mutations.tsv) and are available at https://huggingface.co/datasets/danliu1226/Mutation_effect_dataset. The data generated in this study (training, validation and test datasets used for PPI benchmarking and mutation effect prediction tasks, cross-species PPI model checkpoints; the Bernett PPI model checkpoint; mutation effect classification model checkpoint; predicted interaction probabilities for PPI models on benchmarking tasks; prediction results for mutation effect classification and PPI model evaluations under varying protein sequence identities between training and test datasets) are available at Hugging face (https://huggingface.co/danliu1226) and the Source Data file. All datasets in this study are publicly accessible without restrictions. Source data are provided in this paper. The code in this study is publicly available and has been deposited in GitHub at https://github.com/liudan111/PLM-interact, under the MIT license. The specific version of the code associated with this publication is archived in Zenodo and is accessible via https://doi.org/10.5281/zenodo.1664332465. Trained model checkpoints and datasets used in this study are available at https://huggingface.co/danliu1226. Berggård, T., Linse, S. & James, P. Methods for the detection and analysis of protein–protein interactions. & Sternberg, M. J. E. The contribution of missense mutations in core and rim residues of protein–protein interfaces to human disease. Kotlyar, M., Pastrello, C., Sheahan, N. & Jurisica, I. Integrated interactions database: tissue-specific view of the human and model organism interactomes. Shin, W.-H., Kumazawa, K., Imai, K., Hirokawa, T. & Kihara, D. Current challenges and opportunities in designing protein–protein interaction targeted drugs. Chen, M. et al. Multifaceted protein–protein interaction prediction based on Siamese residual RCNN. Predicting protein–protein interactions through sequence-based deep learning. & Zhang, Z. SGPPI: structure-aware prediction of protein–protein interactions in rigorous conditions with graph convolutional network. Yuan, Q., Chen, J., Zhao, H., Zhou, Y. & Yang, Y. Structure-aware protein–protein interaction site prediction using deep graph convolutional network. Predicting protein–protein interactions based only on sequences information. Tsukiyama, S., Hasan, M. M., Fujii, S. & Kurata, H. LSTM-PHV: prediction of human-virus protein–protein interactions by LSTM with word2vec. Hallee, L. & Gleghorn, J. P. Protein-protein interaction prediction is achievable with large language models. Sledzieski, S., Devkota, K., Singh, R., Cowen, L. & Berger, B. TT3D: Leveraging precomputed protein 3D sequence models to predict protein–protein interactions. Madan, S., Demina, V., Stapf, M., Ernst, O. & Fröhlich, H. Accurate prediction of virus-host protein-protein interactions via a Siamese neural network using deep protein sequence embeddings. Sledzieski, S., Singh, R., Cowen, L. & Berger, B. D-SCRIPT translates genome to phenome with sequence-based, structure-aware, genome-scale predictions of protein-protein interactions. Protein language models learn evolutionary statistics of interacting sequence motifs. Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. Reimers, N. & Gurevych, I. Sentence-BERT: Sentence embeddings using siamese BERT-networks. Proceedings of the 2019 Conference on Empirical Methods in Natural Language In Processing and the 9th International Joint Conference on Natural Language Processing. Ko, Y. S., Parkinson, J., Liu, C. & Wang, W. TUnA: An uncertainty aware transformer model for sequence-based protein-protein interaction prediction. Singh, R., Devkota, K., Sledzieski, S., Berger, B. & Cowen, L. Topsy-Turvy: integrating a global view into sequence-based PPI prediction. Richoux, F., Servantie, C., Borès, C. & Téletchéa, S. Comparing two deep learning sequence-based models for protein-protein interaction prediction. & Goadrich, M. The relationship between Precision-Recall and ROC curves. In Proceedings of the 23rd international conference on Machine learning, 233–240 (2006). Maurer, U., Charvet, C., Wagman, A. S., Dejardin, E. & Green, D. R. Glycogen synthase kinase-3 regulates mitochondrial outer membrane permeabilization and apoptosis by destabilization of MCL-1. Terenzio, M. et al. DYNLRB1 is essential for dynein mediated transport and neuronal survival. Architecture of the RNA polymerase II–Mediator core initiation complex. The ATP binding cassette (ABC) transport systems of Mycobacterium tuberculosis. Chai Discovery et al. Chai-1: Decoding the molecular interactions of life. Accurate structure prediction of biomolecular interactions with AlphaFold 3. & List, M. Cracking the black box of deep sequence-based protein–protein interaction prediction. The IntAct molecular interaction database in 2012. Frazer, J. et al. Disease variant prediction with deep generative models of evolutionary data. Brandes, N., Goldman, G., Wang, C. H., Ye, C. J. & Ntranos, V. Genome-wide prediction of disease variant effects with a deep protein language model. Qu, K. et al. MCM7 promotes cancer progression through cyclin D1-dependent signaling and serves as a prognostic marker for patients with hepatocellular carcinoma. Toyokawa, G. et al. Minichromosome Maintenance Protein 7 is a potential therapeutic target in human cancer and a novel prognostic marker of non-small cell lung cancer. Huang, T.-H. et al. EGFR potentiates MCM7-mediated DNA replication through tyrosine phosphorylation of Lyn kinase in human cancers. Santos, R. et al. Friedreich ataxia: Molecular mechanisms, redox considerations, and therapeutic opportunities. Structure of the human frataxin-bound iron-sulfur cluster assembly complex provides insight into its activation mechanism. Ammari, M. G., Gresham, C. R., McCarthy, F. M. & Nanduri, B. HPIDB 2.0: a curated database for host–pathogen interactions. Prediction of human-virus protein-protein interactions through a sequence embedding-based machine learning method. Elnaggar, A. et al. ProtTrans: Towards cracking the language of life's code through self-supervised learning. Yang, X. et al. HVIDB: a comprehensive database for human–virus protein–protein interactions. Pettersen, E. F. et al. UCSF ChimeraX: Structure visualization for researchers, educators, and developers. van Kempen, M. et al. Fast and accurate protein structure search with Foldseek. Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Hayes, T. et al. Simulating 500 million years of evolution with a language model. The OMG dataset: An Open MetaGenomic corpus for mixed-modality genomic language modeling. Wang, Y. et al. LC-PLM: Long-context Protein Language Model.https://doi.org/10.1101/2024.10.29.620988 (2024). & Streicker, D. G. Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes. Young, F., Rogers, S. & Robertson, D. L. Predicting host taxonomic information from viral genomes: A comparison of feature representations. Liu, D., Young, F., Robertson, D. L. & Yuan, K. Prediction of virus-host associations using protein language models and multiple instance learning. Roux, S. et al. iPHoP: An integrated machine learning framework to maximize host prediction for metagenome-derived viruses of archaea and bacteria. Stukalov, A. et al. Multilevel proteomics reveals host perturbations by SARS-CoV-2 and SARS-CoV. J, L. et al. Virus-host interactome and proteomic survey reveal potential virulence factors influencing SARS-CoV-2 pathogenesis. The STRING database in 2023: protein–protein association networks and functional enrichment analyses for any sequenced genome of interest. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Fu, L., Niu, B., Zhu, Z., Wu, S. & Li, W. CD-HIT: accelerated for clustering the next-generation sequencing data. & Schaefer, M. H. HIPPIE v2.0: enhancing meaningfulness and reliability of protein–protein interaction networks. Steinegger, M. & Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Devlin, J., Chang, M.-W., Lee, K. & Toutanova, K. BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding. Note on the sampling error of the difference between correlated proportions or percentages. Liu, D. PLM-interact: extending protein language models to predict protein-protein interactions. acknowledges funding from the European Union's Horizon 2020 research and innovation 562 programme, under the Marie Sklodowska-Curie Actions Innovative Training Networks 563 grant agreement no. ; a UK Medical Research Council (MRC) Doctoral Training Programme in Precision Medicine studentship (MR/N013166/1) for K.D.L. acknowledges support from Cancer Research UK (EDDPGM-Nov21\100001 and DRCMDP-Nov23/100010), Biotechnology and Biological Sciences Research Council (BBSRC) BB/V016067/1, Prostate Cancer UK MA-TIA22-001 and EU Horizon 2020 grant ID 101016851. acknowledge support from Cancer Research UK core funding to the CRUK Scotland Institute (A31287) and a core programme award to C.J.M. acknowledges support from Cancer Research UK core funding to the CRUK Scotland Institute (A31287). This work used the DiRAC Extreme Scaling service (Tursa) at the University of Edinburgh, managed by the Edinburgh Parallel Computing Centre on behalf of the STFC DiRAC HPC Facility (www.dirac.ac.uk). The DiRAC service at Edinburgh was funded by BEIS, UKRI and STFC capital funding and STFC operations grants. DiRAC is part of the UKRI Digital Research Infrastructure. These authors jointly supervised this work: Craig Macdonald, David L Robertson, Ke Yuan. Dan Liu, Francesca Young, Kieran D. Lamb & David L. Robertson Adalberto Claudio Quiros, Crispin J. Miller & Ke Yuan Adalberto Claudio Quiros, Craig Macdonald & Ke Yuan Cancer Research UK Scotland Institute, Glasgow, United Kingdom Alexandrina Pancheva, Crispin J. Miller & Ke Yuan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar provided suggestions for model training and implementing models on the HPC system. Correspondence to Craig Macdonald, David L. Robertson or Ke Yuan. The authors declare no competing interests. Nature Communications thanks Samuel Sledzieski and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. PLM-interact: extending protein language models to predict protein-protein interactions. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/mosquitoes-found-in-iceland-for-the-first-time-amid-climate-change/'>Mosquitoes Found in Iceland for the First Time amid Climate Change</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Mosquitoes Invade Iceland, Earth Darkens, and Bird Flu Returns By Rachel Feltman, Fonda Mwangi, Jeffery DelViscio & Alex Sugiura For Scientific American's Science Quickly, I'm Rachel Feltman. Let's kick things off with some climate news. Back in 2016 an article in the New York Times referred to Iceland as a “mosquito-free island paradise.” While nearby countries host mosquito populations during warmer times of the year, one of the going theories was that Iceland's propensity for harsh swings between thawing and freezing helped keep the bloodsuckers from getting a foothold. Now mosquitoes have made themselves at home in Iceland for the first time. According to local news sources, an insect enthusiast got in touch with the Natural Science Institute of Iceland after finding three suspected mozzies out in the wild. This development is a troubling reminder of rapid global warming's potential impacts on our planet, especially up in the Arctic: one 2022 study in Nature reported that this region had warmed almost four times faster than the rest of the planet between 1979 and 2021. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The measure of how much sunlight our planet reflects back into space is called albedo, and this reflectivity has been decreasing for decades. This could be creating somewhat of a climate-change feedback loop. The Northern Hemisphere is experiencing more warming, and warmer air holds more water vapor, which tends to absorb solar radiation. Meanwhile, melting ice and snow in the north means less white surface is available to reflect sunlight back into space. As the Northern Hemisphere gradually reflects less sunlight, it is expected to absorb more heat, which scientists think could then decrease its albedo even more. After a summer lull bird flu is back on the rise. The surge aligns with the annual southern migration of wild birds, which creates opportunities for domestic flocks to mingle with infected outsiders and their droppings. That includes more than a million turkeys, the New York Times reported, and according to Axios, wholesale turkey prices are roughly 40 percent above last year's rates. At the same time the Associated Press said last week that the ongoing government shutdown would force Centers for Disease Control and Prevention researchers to skip IDWeek, a “pivotal conference on infectious disease” in Atlanta, Georgia, where experts were expected to share findings on diseases like bird flu, HIV and more. If you haven't already given it a listen, check out our recent miniseries on the history and future of bird flu. Meanwhile, also at the intersection of agriculture and health, a study published last Thursday in the journal mBio suggests that our conversations about food safety might need to include urinary tract infections. The team then screened meat samples from retail sources located in the patients' neighborhoods in an attempt to determine which infections came from nonhuman animal sources. Ultimately, the researchers linked 18 percent of the UTI cases they examined to E. coli strains from animals, with chicken and turkey samples most likely to contain infection-causing bacteria. While the new study is localized and can't prove a causal link between contaminated meat and human infections, it's never a bad idea to up your food-safety game. Those precautions can help protect you from zoonotic viruses such as bird flu, too, in addition to, of course, lowering your risk of common foodborne-illness symptoms like vomiting. In other California health news earlier this month the state's Department of Public Health announced three unrelated cases of clade I mpox in residents who had not recently traveled abroad. Clade II has been circulating in the U.S. at low levels in recent years, but cases of clade I have always been tied to international travel. The news of suspected transmission in California is somewhat concerning because clade I has historically had a higher mortality rate than clade II. Mpox is mostly spread through close contact, including sex, and these new cases align with past clade II surges that primarily hit communities of men who have sex with men, as well as transgender and nonbinary individuals and people with HIV. If you or any of your sexual partners fall into one of those categories and you're not vaccinated for mpox yet, now is a great time to go get your first shot. You can learn more about the changing science of peanut allergies and other food intolerances by listening to our August 22 episode or by reading SciAm's recent feature article on the subject. We'll wrap things up on a high note with some cautiously optimistic news about the North Atlantic right whale. According to a report from the North Atlantic Right Whale Consortium published last Tuesday, these critically endangered behemoths continue to show slow but steady population growth. Scientists estimate that the whale's population now totals 384, which is eight higher than last year. The population decline is reportedly thanks to threats such as boat collisions, fishing nets, and decreased food on their migration path as a result of global warming. That's all for this week's science news roundup. Tune in on Wednesday for a conversation about how the highly anticipated sale of TikTok's U.S. operations could impact its infamously powerful algorithm—and what a skew in “For You” page content could mean for the country. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. Jeffery DelViscio is currently chief multimedia editor/executive producer at Scientific American. Before that he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251027023806.htm'>Hidden 5-mile wide asteroid crater beneath the Atlantic revealed in stunning 3D</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 08:52:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers from Heriot-Watt University have released striking images of an asteroid impact crater hidden deep beneath the floor of the Atlantic Ocean. Using the new data, scientists have pieced together what unfolded immediately after the Nadir impact: the crater initially formed as a deep bowl, molten rock surged upward from below, a vast zone of fractured rock spread outward for thousands of square kilometers, and an enormous tsunami more than 800 meters (~2600 feet) high swept across the Atlantic. The research was published in Nature Communications Earth & Environment. The data revealed a circular depression measuring more than 8.5 km across, leading Dr. Nicholson to suspect it was the site of an ancient asteroid strike. Early results suggested the crater was formed by an asteroid several hundred meters wide about 66 million years ago, but proof remained uncertain. High-resolution, three-dimensional seismic data collected by the global geophysical company TGS and shared with Dr. Nicholson provides clear evidence that an asteroid created the Nadir Crater. Dr. Nicholson said: "There are around 20 confirmed marine craters worldwide, and none of them has been captured in anything close to this level of detail. "Craters on the surface are usually heavily eroded and we can only see what is exposed, whereas craters on other planetary bodies usually only show the surface expression. "These data allow us to image this fully in three dimensions and peel back the layers of sedimentary rock to look at the crater at all levels. "We originally thought the asteroid would have been around 400m wide. "We can tell it came from about 20-40 degrees to the northeast, because of spiralling thrust-generated ridges surrounding the crater's central peak -- those are only formed following a low-angle oblique impact. "And we think it would have hit Earth at about 20 km per second, or 72,000 km per hour, although we still need to confirm this with a new set of impact models." "As well as this, we see evidence for a train of tsunami waves going away from, then back towards the crater, with large resurge scars preserving evidence of this catastrophic event." Dr. Nicholson points out that humans have never witnessed an asteroid of this size crashing into Earth. These will give us more information about the shock pressures experienced during impact, and the precise age and sequence of events that occurred after this event." Collaborator Dr. Veronica Bray of the University of Arizona, an expert in impact cratering across the solar system, commented: "We see pristine impact craters on airless bodies like the Moon, but don't have subsurface structural information. "The new 3D seismic imaging of Nadir gives us both. It's a startlingly good look at an impact crater!" The rubble pile asteroid Bennu is around 400m in diameter. It is considered the most hazardous object in near-Earth orbit. According to NASA scientists, its total impact probability through the year 2300 is about 1 in 1,750 (or 0.057%). The researchers were also able to identify September 24, 2182, as the most significant single date in terms of a potential impact, with an impact probability of 1 in 2,700 (or about 0.037%) Scientists Discover Brain Chemical Linked to Depression and Suicidal Thoughts The Universe “Will End in a Big Crunch,” Physicists Warns Brain's “Pain Map” Discovered: Scientists Find Opioid-Free Path to Relief Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251027023802.htm'>Melting ice is hiding a massive climate secret beneath Antarctica</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 08:43:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Climate models have long predicted that global warming would weaken the Southern Ocean's ability to absorb carbon dioxide (CO2). But climate change is steadily altering these delicate ocean layers and could soon disrupt this natural carbon storage system. The world's oceans capture roughly one quarter of all human-produced CO2 emissions. The Southern Ocean alone accounts for about 40 percent of that total, making it one of the planet's most powerful natural defenses against global warming. The balance depends on how much natural CO2 from ancient deep waters resurfaces. This interplay is governed by the layering, or stratification, of different water masses and by the strength of ocean currents. The deep water that resurfaces in the Southern Ocean has been isolated for centuries or even millennia, accumulating large amounts of CO2. However, despite these stronger winds, data collected over recent decades show that the Southern Ocean remains a strong carbon sink. The new AWI research helps explain why: ocean layering has changed in a way that keeps much of the deep carbon locked away. "Deep water in the Southern Ocean is normally found below 200 meters," says Dr. Léa Olivier, AWI oceanographer and lead author of the study. "It is salty, nutrient-rich and relatively warm compared to water nearer the surface." This deep water contains large stores of dissolved CO2 that entered the ocean long ago. Although such wind intensification has been observed and linked to human activity, measurements still show no major decline in the ocean's carbon uptake -- at least not yet. Long-term monitoring by AWI and other research institutions shows that climate change is already altering the characteristics of both surface and deep waters. "In our study, we used a dataset comprising biogeochemical data from a large number of marine expeditions in the Southern Ocean between 1972 and 2021. In doing so, we only considered processes related to the exchange between the two water masses, namely circulation and mixing, and not biological processes, for example," explains Léa Olivier. The Southern Ocean's surface water salinity has reduced as a result of increased input of freshwater caused by precipitation and melting glaciers and sea ice. However, this situation could reverse if the stratification were to weaken," summarizes Léa Olivier. As CO2-rich water replaces more of the surface layer, the boundary between them becomes more vulnerable to mixing, likely caused by those same winds. Once mixing increases, stored CO2 could begin to leak upward and escape into the atmosphere. Recent research suggests that this process may already be starting. Materials provided by Alfred Wegener Institute, Helmholtz Centre for Polar and Marine Research. Note: Content may be edited for style and length. Scientists Discover Brain Chemical Linked to Depression and Suicidal Thoughts The Universe “Will End in a Big Crunch,” Physicists Warns Brain's “Pain Map” Discovered: Scientists Find Opioid-Free Path to Relief Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251026021744.htm'>Scientists just uncovered what's really happening beneath this mysterious volcano</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'Science Daily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-27 06:07:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>What drives the rumbling vibrations, called tremor, that occur when molten rock or gases travel upward through underground channels? Professor Dr. Miriam Christina Reiss, a volcano seismologist at Johannes Gutenberg University Mainz (JGU), and her research group have traced these tremor signals beneath Tanzania's Oldoinyo Lengai volcano. "We were not only able to detect tremor, but also to determine its exact position in three dimensions -- its location and depth below the surface," said Reiss. Their research also carries practical importance, as it may one day make volcanic eruption forecasts more accurate. When magma ascends from deep within Earth, it can make the ground shake. Under high pressure, magma can crack surrounding rock, producing earthquakes. "For volcano seismology, it is extremely interesting to study these signals and wave types that arise when magma moves below the surface," explained Reiss. Her research centers on two key questions: Where exactly does tremor begin, and what causes it to form? The answers reveal important clues about a volcano's internal state and level of activity. Reiss and her colleagues monitored Oldoinyo Lengai for 18 months using an array of seismometers positioned around the volcano to record ground vibrations. It is clear that these signals are connected, thus we see a directly linked system here." Oldoinyo Lengai is unlike any other volcano on the planet. It is the world's only active carbonatite volcano, producing lava that is unusually fluid and relatively cool at about 550 degrees Celsius, compared to the 650 to 1,200 degrees typical of most magmas. The team's results mark an important step forward in understanding how magma moves within volcanoes. "Tremor occurs whenever magma is moving -- including before eruptions," Reiss explained. Scientists Discover Brain Chemical Linked to Depression and Suicidal Thoughts The Universe “Will End in a Big Crunch,” Physicists Warns Brain's “Pain Map” Discovered: Scientists Find Opioid-Free Path to Relief Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            