
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-19</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/u-s-cuts-antarcticas-only-research-icebreaker-ship-under-trump-budget/'>U.S. Researchers May Never Probe Deep into Antarctica's ‘Doomsday Glacier' Again Without This Ship</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>U.S. Cuts Antarctica's Only Research Icebreaker Ship under Trump Budget Squeeze The National Science Foundation will stop operating the Nathaniel B. Palmer icebreaker and slash polar science funding by 70 percent, devastating Antarctic research Bow shot of National Science Foundation research icebreaker Nathaniel B. Palmer during sea ice sampling. Many ships have tried to reach the floating tongue of Thwaites Glacier—a 130-kilometer-wide conveyor belt of ice that slides off West Antarctica's coast and splinters into the sea. Thwaites is rapidly destabilizing, and precise mapping of the seafloor and ocean currents surrounding it are urgently needed to know how much damage the crumbling glacier and resulting sea-level rise could do to coasts worldwide. Only one vessel—the U.S. research icebreaker Nathaniel B. Palmer (NBP)—has successfully penetrated the area's phalanx of sea ice and billion-metric-ton icebergs to reach a critical location on Thwaites, widely considered the world's most dangerous glacier. The NBP has made about 200 research cruises to Antarctica in the past 30 years, in many cases reaching places never before visited. But if the Trump administration has its way, this will all come to an end in October. If the Thwaites Glacier were to implode, it could raise the average global sea level by 65 centimeters, and it could potentially trigger wider Antarctic ice sheet collapse that could raise global sea level by more than three meters. Frequent measurements in Antarctica's remote locations are needed to project how quickly this might happen. Expeditions also monitor marine ecosystems that are rapidly shifting as a result of climate change and affecting the large commercial fisheries in surrounding waters. A replacement vessel—if one is even funded—“will probably take 10 years to build,” he says. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. This latest move caps what feels like a major U.S. retreat from Antarctica, says Theodore Scambos, a polar glaciologist at the University of Colorado Boulder. This loss will have widespread consequences, according to a wide range of scientists, logistics experts, former NSF officials and former diplomats interviewed for this story. It will hamstring U.S. influence in the Antarctic at a time when geopolitical competition and resource exploitation are intensifying in the region. It will also undermine scientists' ability to monitor rapid changes in Thwaites and other remote areas—at the very moment climate impacts on the continent are accelerating. Researchers first learned of the NBP's situation on May 30, when the Trump administration released its full fiscal year 2026 budget request for NSF to Congress. The budget request also stipulated a 70 percent funding cut for polar science research projects overall. “None of us saw that coming,” says Patricia Yager, a polar marine biologist at the University of Georgia, who called the move “shocking.” In response to a request for comment from Scientific American, an NSF spokesperson confirmed via e-mail that NSF had proposed terminating the lease of the NBP in Trump's 2026 fiscal year budget request but provided little additional information. Storis, which has a troubled history, will also be used in the Arctic. It can plow through meter-thick sea ice at three knots with little more than gentle side-to-side rocking. And it can remain at sea for 65 days, a vital capability given that a round trip to Thwaites takes nearly two weeks from the nearest port. When the NBP is in Antarctic waters, it continuously collects data. Multibeam sonar captures a three-dimensional map of the seafloor, revealing features such as undersea canyons that influence the stability of coastal glaciers. Another sonar system traces sediment layers below the seafloor, which can provide important records of past climate periods. Even in rough seas and high winds, the ship's dynamic positioning system allows it to hover within a few feet of its intended location while technicians do the delicate work of collecting sediment cores or piloting remote-operated submersibles on the seafloor. The NBP can carry and launch two helicopters, which proved decisive during a 2010 cruise, allowing researchers to leapfrog over impassable sea ice and install instruments for monitoring crustal movements and glacial retreat as far as 200 kilometers away. The NSF is scrambling for ways to salvage at least some of them. For example, the agency might send the University of Alaska Fairbanks's Arctic research vessel Sikuliaq all the way down to Antarctica in January to perform a major ecological survey that has been conducted annually since 1999. And compared with the NBP, it has fewer berths for scientists, can spend fewer days at sea and has more limited ice-breaking capabilities—effectively excluding it from Thwaites and other heavily iced sections of the Antarctic coastline. A few other countries have research icebreakers, including the U.K., Germany, Australia, South Korea and China. It is possible that a couple U.S. researchers could occasionally find ride-along spots. But that may not advance U.S. research priorities, Scambos says: “You're not going to get somebody else's icebreaker to bring you and 20 of your colleagues and undertake a major mission that has U.S. research interest behind it.” The NSF spokesperson stated that the agency “has started the process to identify vessels and partnerships to continue support of marine science.” A new icebreaker that would eventually replace the NBP is in early design stages, but it would take a decade to build. But spending an ensuing decade without a research icebreaker could have major geopolitical consequences. Antarctica is Earth's largest remaining territory not unilaterally controlled by any particular nation. If international commitment to the treaty ever faltered, however, nations might pursue territorial claims on the continent, and those claims would be bolstered by having maintained research stations, communications hubs, deep water ports and air transport and driving routes. “Operations and logistics are 100 percent necessary for U.S. national security interests,” says William Muntean III, a former senior adviser for Antarctic affairs and a deputy representative for the U.S. State Department to the annual Antarctic Treaty Consultative Meeting. The scientific research conducted there is a fundamental part of that equation. “In the Antarctic Treaty System,” Muntean says, “knowledge is what equals power.” That's because the rules of play for Antarctica and its surrounding seas are constantly negotiated and monitored by more than two dozen countries with research stations there—and any negotiations become especially heated when it comes to resource extraction. The U.S. and a handful of other countries have sought to establish several marine protected areas around the continent where fishing would be prohibited, but they have encountered resistance from Russia and China. Negotiations often hinge on data collected from research vessels, such as those that document changes in ecosystems. In this way, Muntean says, “knowing what is actually happening in Antarctica gives a country the ability to then influence” what happens. The ship's fate is on hold until early September, when Congress returns from recess. Funding bills that were sitting in Congress before recess add confusion. Senate bill 2354, which passed out of the Appropriations Subcommittee on Commerce, Justice, Science and Related Agencies in mid-July, appears to restore full funding to the NSF. Others will have to do so in the coming weeks, before Congress returns. The administration is already stripping climate data from government websites, preparing to halt EPA regulation of carbon dioxide emissions and quietly discussing plans to scuttle a state-of-the-art NASA satellite that monitors carbon dioxide—allowing the spacecraft to burn up in the atmosphere. Canceling the NBP, he says, looks like “a political decision of not wanting to support climate change research.” In this decision, Schofield sees an important lever of soft power being abandoned. “Since World War II,” he says, “there was always a strong belief that if the U.S. had the strongest scientists and the strongest engineers, it would serve national security.” Those generational investments, he says, are now being undone. Douglas Fox writes about biology, geology and climate science from California. He wrote the November 2022 article “The Coming Collapse,” which revealed that Antarctica's Thwaites Ice Shelf could splinter apart in less than a decade.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/green-tech/a65797286/soil-3d-printed-house/'>Engineers Printed a 1000-Square-Foot House Out of Soil—No Concrete Needed</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Using dirt as a construction material could help make one of the biggest carbon-emitting sectors significantly greener. While many countries need to build fast to address the ongoing housing crisis, all of that building would come with a major environmental toll. A 2022 estimate by MIT Climate states that constructing a single house creates 15 to 100 tons of CO2, and a lot of that carbon comes from concrete, which requires high temperatures powered by fossil fuels to create. While valiant efforts have been made to clean up concrete and cement production, the industry is still one of the biggest greenhouse gas emitters globally. This unfortunate truth has inspired some engineers to think outside the box in order to help close the gap between needed housing and environmental sustainability. Granted, most 3D-printed homes still rely on some form of concrete mix (including cement, sand, gravel, and aggregates). But now, companies are trying to 3D-build with a different kind of extruded material—local soil. Lib Work claims that the Model B (built in Yamaga, Kumamoto Prefecture) is five times stronger than the previous model, which was constructed earlier this year. According to Archinect, Lib Work is the world's first fully automated home construction system. “We have succeeded in developing a sustainable earthen wall made entirely from earth and natural materials, without using any cement,” according to the company's website, translated from Japanese. An in-wall condensation detection system constantly monitors for deterioration, and the home is fully off the grid, relying on solar panels and a Tesla Powerwall battery for energy. The home is also fully remote controlled through a smartphone app. In 2021, for example, Mario Cucinella Architects in Bologna, Italy, partnered with the 3D-printer manufacturer WASP (which also worked with Lib Work on the Model B) to create a clay-based construction material derived from local soil. Although the drying process took longer than other quick-dry concrete products, the material allowed for increased flexibility in terms of construction location. Building affordable housing and limiting greenhouse gas emissions will require innovative solutions—from broader city planning to home construction. Increased housing density can help keep utility bills low and limit reliance on cars, while sustainable materials can build homes without adding to humanity's growing CO2 budget. Hot Debate: How Cool Should You Keep Your House? A New Kind of Concrete Can Grow and Repair Itself</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02821-5'>Global covariation of forest age transitions with the net carbon balance</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 10:27:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Forest age transitions are critical in shaping the global carbon balance, yet their influence on carbon stocks and fluxes remains poorly quantified. Here we analyse global forest age dynamics from 2010 to 2020 using the Global Age Mapping Integration v2.0 dataset, alongside satellite-derived aboveground carbon (AGC) and atmospheric inversion-derived net CO2 flux data. We reveal widespread declines in forest age across the Amazon, Congo Basin, Southeast Asia and parts of Siberia, primarily driven by stand-replacing disturbances such as fire and harvest, leading to the replacement of older forests by younger stands. Meanwhile, forests in China, Europe and North America experienced net ageing. Globally, stand replacement resulted in substantial AGC losses, with old forests (>200 years, ~98.0 MgC ha−1) transitioning to younger, carbon-poor stands (<20 years, ~43.5 MgC ha−1), leading to a net AGC loss of ~0.14 PgC per year. Despite this, regions with high rates of young stands replacing old forests exhibited a temporary strengthening of the carbon sink, driven by the rapid regrowth of these young stands. Crucially, these young forests do not compensate for the long-term carbon storage of old forests. Our findings underscore the importance of protecting old forests while optimizing forest management strategies to maximize carbon gains and enhance climate mitigation. In this study, forests encompass natural forests, managed stands and planted tree cover (for example, agroforestry, tree crop and plantations), characterized by at least 5 metres height4. Effectively managing and conserving these forests is crucial for mitigating climate change, especially given the decline in forest resilience under climate change6. Several processes, including deforestation, degradation, afforestation, natural disturbances, management practices and regrowth, influence the distribution of forest ages worldwide7. These factors, in turn, affect the forests' capacity to sequester carbon8. European forests, for instance, have consistently acted as carbon sinks9 due to their recovery from extensive clear-cutting after World War II10. Their growth exceeds harvest removals and natural mortality9,11, although the European carbon sink has shown a recent decline12. In regions such as the Amazon Basin, old forests were once ‘hotspots' of younger forests, shaped substantially by ancient civilization activities13. Recent regional transformations have created a diverse mosaic of forest age classes (Fig. Complex interactions between stand-replacement events, regrowth following disturbances, afforestation and the natural ageing of established forests drive shifts in forest age distributions. These factors collectively contribute to ecological transformations in forest landscapes. a,b, Average forest age for 2010 and 2020 at a one-degree pixel resolution (a) and the total area of each forest age class for 2010 (blue) and 2020 (green) (b). c, The forest age difference map between 2020 and 2010, relative to the expected 10-year ageing. This map was calculated by subtracting ten years from the difference between 2020 and 2010. A value of zero indicates that forests have aged as expected over the decade, consistent with the gradual maturation of undisturbed forests. By contrast, negative values reflect forests becoming younger, probably due to stand replacement, disturbances or shifts in forest composition favouring younger stands. One-degree grid cells with less than 20% forest cover have been masked. Traditional methods, such as national forest inventories and research plot measurements, offer valuable insights into local forests but have limited spatial and temporal coverage. Several factors constrain their effectiveness, including the absence, complexity or limited availability of sample-based inventories in many regions, the infrequency of repeated measurements and the variability in what they measure. These limitations hinder their ability to comprehensively capture global forest dynamics. Atmospheric CO2 flux estimates from inversion techniques offer an integrated view of biosphere–atmosphere carbon exchange, considering various sources and sinks across different spatial and temporal scales. Yet, their coarse spatial resolution, typically around 110 km, limits their ability to distinguish between forests and non-forests or accurately capture the different dynamics of forests. Additionally, uncertainty in atmospheric transport models and the lack of measurement stations in certain regions limit their regional accuracy15. Efforts to map forest age at global and regional levels have been made16,17,18, but these typically result in static snapshots of forest age distribution. A few satellite-derived maps monitor temporal changes in forest age, but these are limited in space and can only monitor forests since the beginning of the Landsat era19,20. This limitation impedes our ability to effectively track and understand temporal changes in forest age across all age classes. Using the Global Age Mapping Integration v2.0 (GAMIv2.0) dataset16,21 (Methods), our study analyses changes in forest age distribution from 2010 to 2020 and their implications for the carbon cycle. We integrate forest age maps with independent atmospheric CO2 inversion flux data from the Global Carbon Project22 and European Space Agency Climate Change Initiative (ESA-CCI) satellite-based biomass maps23. Our analysis examines how shifts in forest age, caused by rejuvenation from natural disturbances and clear-cutting, and the natural ageing of established forests affect the net carbon balance. The study has three main objectives: (1) to map global changes in forest age distribution over the last decade, (2) to infer the impact of replacing older, carbon-rich forests with younger ones on the net carbon balance, and (3) to hypothesize the effects of these shifts on aboveground carbon (AGC) stocks under a (1) business-as-usual (BAU) forest management and (2) forest conservation scenario. Our findings highlight the interplay between forest age dynamics and the global carbon cycle, emphasizing the need for strategic, targeted forest management and informed policies to optimize the role of forests in climate change mitigation. We analysed forest age distribution shifts from 2010 to 2020 using the GAMIv2.0 dataset (Fig. We focused on this period due to the availability of ESA-CCI biomass data (2010, 2017–2020) and the rising pressure on forests over the past decade24,25. Age classes were defined based on ecological stages (Table 1). From 2010 to 2020, global forest age shifts (Fig. 1b) were driven by stand replacement and the gradual ageing of established forests (definition in Table 2). Mature (81–200 years) forests increased by \(+{0.08}_{+0.05}^{+0.11}\) billion hectares \((+{8.0}_{+4.4}^{+12.6} \% )\), whereas young (1–20 years), maturing (21–80 years) and old forests (>200 years) declined by \(-{0.04}_{-0.08}^{-0.01}\) billion hectares \((-{5.2}_{-12.0}^{-1.5} \% ),-{0.02}_{-0.05}^{+0.05}\) billion hectares (\(-{2.7}_{-7.4}^{+7.4} \%\)) and \(-{0.03}_{-0.04}^{-0.02}\) billion hectares (\(-{2.6}_{-3.3}^{-2.1} \%\)), respectively (Extended Data Table 1). Regional variations are noteworthy, though (Extended Data Fig. For instance, the total area of young forests increased in Eurasian boreal from \({0.039}_{0.038}^{0.042}\) billion hectares in 2010 to \({0.058}_{0.058}^{0.059}\) billion hectares in 2020 (Extended Data Fig. By contrast, young forests remained relatively stable in European forests, with approximately \({0.036}_{0.032}^{0.042}\) billion hectares in 2010 and \({0.037}_{0.035}^{0.041}\) billion hectares in 2020 (Extended Data Fig. This stability suggests a balance between young forests maturing into older age classes and stand-replacement processes despite ongoing disturbances26. GAMIv2.0 estimates this relationship using a nonlinear model that considers the effects of local biogeographic and climatic conditions, enabling the representation of non-monotonic biomass accumulation across age classes (Extended Data Fig. However, our analysis does not fully capture the impact of forest management practices on the forest age–biomass relationship. Consequently, shifts between age classes may be biased, especially in regions with substantial management influence on forest dynamics (Supplementary Fig. To address this, we binned the age classes by 20 years, which smooths out variability and reduces sensitivity to errors in age estimation. Whereas this approach captures broader trends and minimizes noise from management-induced variations, it may mask the finer-scale impacts of different management practices. Therefore, more comprehensive forest management data are crucial to enhancing the accuracy and reliability of these models. The reduction in young forests is probably due to their progression to older stages, outpacing the development of young forests through frequent forest clearings and afforestation. Additionally, the saturation of afforestation programmes in countries such as China27, initiated in the 1980s and 1990s, and the possible exhaustion of old forests suitable for cutting in Southeast Asia due to increased protection efforts28 may have contributed to this reduction. We observe an increase in the global area of forests aged between 21 and 40 years by approximately +0.03 billion hectares (+17%) over 2010–2020 (Extended Data Table 1). The increase could be a response to prior variations in natural or human-induced disturbances, such as clear-cutting, leading initially to an increase in this age class. These dynamics indicate a more complex forest regrowth pattern29, where young forests mature and recover from earlier disturbances. Multiple factors, including the natural ageing of younger forests, enhanced conservation efforts and a shift towards sustainable practices such as thinning, probably drive the increase in mature forests. Additionally, land abandonment and migration to urban areas may indirectly contribute to this trend. The decline of old forests is alarming because it suggests the loss of the oldest and most ecologically valuable forest stands, vital for their substantial carbon storage capabilities and rich biodiversity. These forests not only sequester large amounts of carbon but also enhance the recovery and growth of nearby degraded forests through processes such as seed dispersal and the creation of stable microclimates20. This underscores the critical importance of preserving old forests. Beyond global changes in forest age distribution (Fig. 1b), we identified contrasting local and regional forest age transitions (Fig. In regions such as Amazonia, the Congo Basin and Southeast Asia, marked decreases in forest age were evident, with variations at the one-degree pixel level showing up to 30% changes since 2010. This trend is primarily attributed to increasing stand-replacing disturbances and mortality30,31, indicating a shift towards younger forest stands and the replacement of old forests. Traditional activities such as slash-and-burn agriculture also contributed to this trend in the Amazon Basin32. Whereas young forests grow faster27, their regrowth does not fully compensate for the carbon loss from older forests28. Transitions to younger stands often involve associated emissions fluxes—from direct emissions due to fire to delayed emissions resulting from increased substrate availability following the harvest of litter. In North America, especially in the Pacific Northwest, a mosaic of older stands and areas of stand replacement followed by regrowth is evident, probably influenced by clear-cutting, a high-burning frequency regime and other natural disturbances, such as insect outbreaks33. Forests in the North American boreal region experienced a net age increase of \(+{8.9}_{+8.8}^{+9.0}\) years (mean age of \({101.6}_{85.7}^{111.9}\) years in 2010, shifting to \({110.5}_{94.6}^{120.7}\) years in 2020). The age increase is only marginally lower than the expected net ten-year increase if no disturbances have occurred. It suggests that disturbances were relatively limited in this region, allowing the forests to age naturally. We also observed that local patches of Canadian forests were getting younger, probably related to recent fire activity34. These localized disturbances indicate that while the overall impact on forest ageing was minimal, specific areas experienced substantial shifts in forest age distribution. Siberian forests, suspected to harbour an important fraction of the global carbon sink12, predominantly sustained their older age class (that is, forests in Eurasian boreal regions showed a mean age of \({108.0}_{89.0}^{116.5}\) years in 2010, shifting to a mean age of \({113.6}_{95.1}^{122.0}\) years in 2020). However, substantial transitions towards younger forests (approximately 0.09 billion hectares, 7.2% of the Eurasia boreal region) indicate localized impacts from logging, increased wildfire events35 and tree recruitment failure after fire36. In China, extensive reforestation and afforestation programs since the 1980s37 have led to substantial shifts in the forest age, with younger planted forests maturing. As part of these large-scale afforestation efforts, the forest area in southern China increased from 9% to 35% between 1986 and 2018, reflecting a marked expansion in forest cover following the implementation of these policies37. These efforts have contributed to the overall net ageing of forests in Eurasia temperate by \(+{7.7}_{+6.4}^{+8.2}\) years, from a mean age of \({44.5}_{36.5}^{53.2}\) years in 2010 to \({52.5}_{42.9}^{60.4}\) years in 2020 (Supplementary Table 1). In southwestern Australia, frequent wildfires have led to younger age classes, probably due to increased turnover or the severity of recent fire events34. Overall, forests in Australia aged by \(+{2.1}_{-0.3}^{+3.4}\) years (a mean age of \({67.7}_{55.8}^{88.0}\) years in 2010, shifting to a mean age of \({69.9}_{57.9}^{87.7}\) years in 2020). This suggests that while fire events have caused localized rejuvenation, other areas have aged relatively undisturbed. Conversely, the Miombo woodlands have maintained consistent fire regimes34, resulting in relatively unchanged turnover and gross primary productivity38. In South America, forests outside the Amazonia, such as the Atlantic forests, have experienced a decrease in native forest cover due to human activities and disturbances. However, recent conservation efforts have led to some recovery39. This combination of factors has resulted in a slight net age increase in the temperate forests of South America by \(+{3.9}_{+1.2}^{+6.3}\) years (mean age of \({87.4}_{51.6}^{110.1}\) years in 2010, shifting to a mean age of \({91.0}_{57.7}^{110.5}\) years in 2020). European forests, covering around 33% of the continent, are experiencing complex age dynamics due to management and natural disturbances, such as fire, drought and insect outbreaks since 201040,41. Despite these disturbances, forests are generally ageing (mean age of \({81.5}_{72.1}^{91.2}\) years in 2010, shifting to a mean age of \({89.5}_{80.3}^{98.9}\) years in 2020, that is, \(+{8.0}_{+7.7}^{+8.2}\) years), except in areas such as Portugal, where large fires have led to younger forests34. Consistent management and stable harvest rates43 suggest that drastic forest age shifts due to human management are unlikely. However, intensified disturbances or increased harvesting could alter the future age structure of European forests, potentially amplifying the reported saturation of the European forest carbon sink44. The net ageing trend in Eurasian boreal forests (that is, forests aged by \(+{5.7}_{+5.4}^{+6.2}\) years), especially in western Russia, is mainly due to large-scale forest abandonment since 1990, a scale surpassing the European forest area45. Our analysis of global forest age shifts reveals distinct regional patterns, with a mix of undisturbed ageing (Fig. 2a) (\({3.21}_{2.84}^{3.46}\) billion hectares) and stand-replaced forests (Fig. This variation across regions underscores the complex interplay (Supplementary Fig. 2) of natural processes and human influences (Fig. 2d) in shaping the age distributions of forests. Regionally, stand-replaced forests cover the largest absolute areas in tropical Asia (\({0.26}_{0.22}^{0.28}\) of total stand-replaced forest fraction, \({0.059}_{0.040}^{0.070}\) billion hectares), northern Africa (\({0.22}_{0.11}^{0.25}\) of total stand-replaced forest fraction, \({0.048}_{0.018}^{0.067}\) billion hectares) and South America tropical (\({0.11}_{0.10}^{0.13}\) of total stand-replaced forest fraction, \({0.026}_{0.020}^{0.030}\) billion hectares) (Supplementary Table 3). By contrast, undisturbed ageing forests are primarily located in South America tropical (\({0.20}_{0.20}^{0.21}\) of total undisturbed ageing forest fraction, \({0.64}_{0.63}^{0.64}\) billion hectares), Eurasia boreal (\({0.17}_{0.17}^{0.17}\) of total undisturbed ageing forest fraction, \({0.54}_{0.54}^{0.54}\) billion hectares), Europe (\({0.10}_{0.10}^{0.11}\) of total undisturbed ageing forest fraction, \({0.32}_{0.32}^{0.32}\) billion hectares) and North America temperate (\({0.092}_{0.090}^{0.092}\) of total undisturbed ageing forest fraction, \({0.29}_{0.28}^{0.29}\) billion hectares) (Supplementary Table 3), primarily characterized by a mix of forests often subject to management (for example, selective harvesting) and unmanaged forests (Fig. Due to its vast forest extent, South America tropical contains substantial absolute areas of both stand-replaced and undisturbed ageing forests, making both classes particularly prevalent in the region. a,b, Spatial distribution of undisturbed ageing (a) and stand-replaced forest fraction (b) at one-degree pixel size. c,d, The forest age pre-stand-replacement (c) and the total area of forest management types46 (d) are also shown across the 11 TRANSCOM-land regions (Supplementary Fig. Naturally regenerated forests are forests that have regenerated without signs of management (for example, logging, clear-cuts). One-degree grid cells with less than 20% forest cover have been masked. NA, North American; SA, South America. Boreal forests typically have an average pre-stand-replacement age of approximately 120 years (\({123.75}_{99.51}^{132.48}\) years and \({113.05}_{96.00}^{122.79}\) years for Eurasia boreal and North American boreal, respectively) (Fig. By contrast, forests in the Northern Hemisphere's temperate regions are replaced at younger ages. Management in these temperate regions, which usually involves naturally regenerated and planted forests, results in an average replacement age of \({94.50}_{78.94}^{102.87}\) years in Europe and \({95.74}_{80.64}^{101.63}\) years in North America (Fig. This variability is influenced by regional rotation lengths, with areas such as northern Spain, Portugal (that is, fast-growing species, such as Eucalyptus, feed into fast rotation forestry projects), Les Landes, Sweden and Finland having shorter rotation periods (Extended Data Fig. By contrast, other forests are harvested at an age over 100 years (Extended Data Fig. We observed short regeneration cycles in Eurasia's temperate (\({33.55}_{21.84}^{39.58}\) years) and South American temperate (\({58.07}_{38.46}^{75.47}\) years) forests. This forest age pre-stand replacement is probably influenced by frequent fires34 and plantation forestry in these regions (Fig. 2d), where the typical rotation period is up to 15 years (ref. In the South American tropics, stand replacement often occurs in older forests, averaging \({190.71}_{166.29}^{215.73}\) years, including old forests over 300 years old, usually characterized by continuous recruitment, though with considerable local variation. The age variability before stand-replacement in South America's tropical region suggests a mix of deforested old forests and young secondary forests harvested and burned before maturity20 or part of a slash-and-burn agricultural system32. The scenario is more complex in northern (\({91.97}_{78.28}^{106.22}\) years) and southern (\({78.92}_{65.99}^{97.57}\) years) Africa regions, which experienced rapid replacement of young forests. These regions experience regular disturbances, primarily from fires34, which can rapidly replace young forests. However, these fires do not always lead to complete stand replacement. Instead, they often leave some trees unburned, especially in lower-intensity or sub-canopy fires. Additionally, in some forest ecosystems, the lower tree density per hectare in certain areas47 may limit the fire severity, reducing the likelihood of complete stand replacement. This highlights post-fire regeneration processes that differ from those of stand-replacing fires, allowing some trees to survive and creating a mix of age structures within the forests. We observe a distinctive spatial pattern in the distribution of stand-replaced forests across various age classes as of 2010 (Fig. Forests that were already young (that is 0–20 years) before stand replacement—covering \({0.14}_{0.083}^{0.20}\) billion hectares and accounting for \({58.97}_{46.14}^{66.55}\)% of all stand-replaced forests— are broadly distributed but primarily concentrated in tropical regions (for example, South America's tropics, tropical Asia and northern Africa) and the Eurasian temperate zone (Fig. This widespread area of young forests being replaced by new stands suggests a high turnover rate, probably driven by plantations (Fig. 2d) and natural disturbances, such as fire events. Maturing forests (that is 20–80 years) replaced by younger stands constitute \({12.51}_{9.97}^{15.96}\)% of the total stand-replaced forest areas. Mature forests (that is 81–200 years) that have undergone stand replacement, where younger stands have replaced previously mature forests, account for \({14.45}_{10.58}^{18.86}\)% of the total stand-replaced forest area. 3c) is sparser than that of younger forests, with large but dispersed patches in regions such as North American, European and Siberian forests. This pattern may indicate less frequent replacement events in mature forests, suggesting a period of relative stability or reduced human influence (Fig. Old forests (that is >200 years) were the least likely to undergo stand replacement, representing only \({14.38}_{11.78}^{19.92}\)% of the total and are primarily located in tropical regions (Extended Data Fig. By contrast, forests undisturbed during the study period display a more balanced distribution across different age classes: young (\({19.01}_{16.56}^{20.59}\)%), maturing (\({18.71}_{16.22}^{20.76}\)%), mature (\(3{0.80}_{26.30}^{32.98}\)%) and old forests (\({32.23}_{30.06}^{34.41}\)%) (Extended Data Fig. a,b, Total area (in billion hectares) of stand-replaced (a) and undisturbed ageing forests (b) per age class across the 11 TRANSCOM-land regions (Supplementary Fig. Using the ESA-CCI biomass map for 202023, we observe apparent differences in AGC stocks where younger stands replaced carbon-rich older forests from 2010 to 2020 (Fig. After replacement, the AGC stock of stand-replaced forests varies by prior age class: ~2.3 MgC ha−1 in young forests, increasing to 15.2 MgC ha−1 in maturing forests, 31.1 MgC ha−1 in mature and peaking at 31.9 MgC ha−1 in old forests. These differences reflect two main factors. First, not all biomass is lost during stand-replacement—legacy trees and residual carbon often remain, especially in partially disturbed areas. Second, younger stands replacing old forests usually grow faster due to nutrient-rich soils, existing seed banks and structural legacies, with open canopies enhancing light availability and promoting rapid regrowth48 (Extended Data Fig. a, AGCs stocks across forest age classes, distinguishing between stand-replaced forests and undisturbed ageing forests, expressed per unit area at a one-degree pixel level. The stand-replaced categories represent the AGC stock of forests at a given age class (young, maturing, mature or old) before stand replacement. b, Net carbon changes for stand-replaced forests across different forest age categories. c, The relationship between the fraction of old forests replaced by young stands (that is, stand-replacement extent) and changes in NEE from inversions between circa 2020 (average of 2019–2021) and 2010 (average of 2009–2011) is shown. One-degree grid cells with less than 20% forest cover have been masked. To smooth the spatial distribution of net CO2 fluxes, we applied a Gaussian filter (length = 500 km, equivalent to approximately four one-degree pixels). This smoothing technique reduces noise in the data and helps minimize the influence of local transport errors. To minimize noise and regional variability due to atmospheric transport errors, all spatial data were aggregated using area-weighted averaging over 5 × 5 grid cells (that is, a coarsening scale of 5° × 5°). CAMS, Copernicus Atmosphere Monitoring Service; CarboScope, Jena CarboScope Atmospheric Inversion System; CMS-Flux, Carbon Monitoring System Flux; CTE, CarbonTracker Europe; IAPCAS, Institute of Atmospheric Physics Carbon Assimilation System; ICT-NOAA, CarbonTracker NOAA; MIROC, Model for Interdisciplinary Research on Climate; NISMON-CO2, NICAM-based Inverse Simulation for Monitoring of CO2; UoE, University of Edinburgh Inversion System. Undisturbed ageing forests store substantially more carbon than recently disturbed ones across all age classes (Fig. The comparable carbon storage capacity in stand-replaced and undisturbed mature forests suggests that stand replacement often involves partial rather than complete biomass removal through selective logging or natural disturbance. Whereas the GAMIv2.0 product does not capture small-scale degradation or mortality effects, the ESA-CCI biomass map partially reflects these dynamics. This may explain the limited AGC gap observed in mature forests across disturbance types. However, ESA-CCI biomass data also have limitations, particularly in the calibration of AGC estimates from remote sensing observations, as the sensitivity of satellite-derived AGC retrievals decreases with increasing biomass, contributing to uncertainty in high-biomass forests49. Evaluating total carbon stocks across forest age classes (Supplementary Table 5) reveals that old, undisturbed forests store \({127.39}_{99.20}^{155.14}\) PgC. Conversely, young forests, including those that replaced older stands (\({1.85}_{1.35}^{3.17}\) PgC) and undisturbed ageing forests (\({17.99}_{16.02}^{21.32}\) PgC), collectively stored approximately 20 PgC in 2020 (approximately 9% of the total AGC stocks in 2020). Maturing and mature, undisturbed ageing forests cover large areas (Fig. The total global AGC stock in 2020 was estimated at ~\({235.75}_{192.50}^{286.11}\) PgC in 2020, aligning with independent datasets but substantially lower than previous estimates5 (~371.5 PgC), probably due to differences in disturbance representation, belowground biomass inclusion and forest definition (for example, a 20% forest fraction filter was applied in our study). The transition from carbon-rich old forests to younger forests, resulting from stand replacement, substantially reduces the overall carbon storage of these ecosystems (Fig. Between 2010 and 2020, forests undergoing stand replacement lost \(+{0.38}_{+0.35}^{+0.46}\) PgC yr−1 (where + denotes loss of AGC) (Supplementary Table 5 and Supplementary Fig. Despite covering only a small fraction of the global forest area (Fig. 3 ), these forests disproportionately influenced AGC losses, particularly where old forests were replaced by young stands, contributing to 0.10–0.16 PgC yr−1 of global AGC loss (~1% of total forest area). Despite the initial loss of AGC following stand replacement, young forests often transition quickly to a net carbon sink51. This is reflected in a significant correlation between the fraction of old forests (>200 years) replaced by young stands (<20 years) and the 10-year trend in net ecosystem exchange (NEE), based on GAMIv2.0 data and atmospheric inversion estimates from nine Global Carbon Budget 2023 (GCB2023) inversion models22 (Fig. The negative slope indicates that for each 1% annual increase in the fraction of old forests replaced by young forests, the carbon sink strengthens by 34.32 gC m−2 yr−1. This suggests that regions with a higher proportion of old forests replaced by young forests tend to exhibit a stronger carbon sink. However, this pattern should not be misinterpreted as an argument in favour of stand replacement. The strong carbon sink observed in these areas primarily results from the long, undisturbed period before stand replacement, during which old forests accumulated large amounts of carbon and created conditions that facilitate rapid regrowth52. The time since the previous-to-last disturbance has a critical role in shaping carbon sink strength, as long-undisturbed forests develop favourable soil conditions, structural complexity and nutrient availability that enhance post-disturbance regrowth. It is also essential to recognize the limitations of this analysis. At the spatial resolution of current atmospheric inversion models (~1°), it is beyond our capacity to disentangle NEE trends across all four age classes robustly. This limitation is particularly pronounced in regions with mixed-age forests and intensive management, where inversion signals integrate fluxes across diverse stand conditions. Whereas exploratory correlations by age class are presented in Extended Data Fig. More interpretable signals may emerge in tropical regions, where large areas of old-growth forest have been replaced by younger stands, resulting in more homogeneous age distributions within inversion grid cells. Still, robust separation of age-specific NEE responses remains challenging at this scale. Accordingly, our main interpretation focuses on the consistent NEE signal linked to old-to-young forest transitions in these areas (Extended Data Fig. 3d), which shows a consistent negative correlation with NEE trends across multiple inversion models (Fig. Still, broader factors such as CO2 fertilization, temperature shifts and climate variability may also influence NEE trends. While we acknowledge these influences, our analysis isolates the relationship between stand replacement and NEE without attempting to disentangle climate-driven effects. Young, fast-growing forests can sequester carbon at rates higher than old forests, particularly in tropical regions, where young regrowing forests (<20 years) can absorb CO2 up to 20 times faster than old forests52. Yet they do not recover the AGC stocks of old forests within a meaningful timeframe for climate policy. Complete restoration of species composition can take centuries53,54,55. The ability of forests to accumulate carbon over centuries highlights the importance of preserving old forests, which store far more carbon than young regrowing stands. Although undisturbed ageing forests contribute to long-term carbon storage (Fig. 3b), the net CO2 uptake trends suggest that stand-replacement processes—not just naturally ageing forests—are key drivers of recent flux changes. Nevertheless, it is essential to recognize that forest harvesting, leading to biomass loss, impacts atmospheric carbon levels in multiple ways. Harvesting does not immediately result in equivalent carbon emissions being released into the atmosphere. Forests continue to assimilate carbon, and the fate of harvested biomass—whether rapidly released through burning, gradually decomposed as coarse woody debris or stored in wood products for a long time—determines the timing and extent of carbon released into the atmosphere. The balance between immediate and delayed carbon release may explain the residuals between the proportion of forests replaced by young stands and net CO2 fluxes (Fig. 4c), which are further influenced by variations in hydrometeorological conditions56. It is crucial to contextualize the impact of stand replacement on carbon flux dynamics within the broader picture of global forest dynamics. Global forests act as a net carbon sink, particularly in boreal and temperate regions5,57. However, deforestation and land-use conversion increasingly challenge this climate-mitigation role, where forests are permanently lost rather than replaced. In deforestation hotspots, atmospheric inversion data mainly show these areas as net CO2 sources, primarily due to emissions from burning activities. For example, in regions such as the eastern edge of the Amazon, fire-related emissions can exceed the carbon uptake of remaining forests58, reinforcing the dominant role of deforestation in increasing atmospheric CO2. Thus, whereas stand-replacement processes contribute to enhanced CO2 uptake in some regions, they should not be viewed in isolation from the broader issue of forest loss and the protection of old forests. Furthermore, the relationship between old-stand replacement and NEE trends varies across atmospheric inversion models. Whereas five of the nine models indicate a significant negative correlation, others show no clear relationship (Fig. This variability underscores the need for cautious interpretation when linking stand-replacement processes to regional NEE trends. Nevertheless, the consistent signal across multiple models and spatial windows (Supplementary Fig. 8) supports the conclusion that old forest replacement has been key in shaping recent CO2 flux dynamics. Our findings emphasize the complex role of forest age transitions in shaping the global carbon balance. Whereas old forests store the highest carbon stocks, young regrowing forests, particularly those replacing old forests, can contribute substantially to CO2 uptake due to rapid biomass accumulation. However, this high sequestration rate primarily results from the favourable conditions left behind by the replaced old forests, including legacy soil carbon, nutrient availability and established seed banks. This does not imply that stand replacement is a viable climate mitigation strategy, as old forests take centuries to develop and cannot be restored within a policy-relevant timeframe. Young forests can never fully replace the long-term carbon storage capacity and ecological functions of old forests. Protecting existing forests remains the most effective and immediate strategy for mitigating climate change60. To explore the implications of age dynamics for future carbon storage, we projected forest age distributions and AGC stocks to 2050 under two idealized pathways: BAU and a forest conservation scenario that halts stand-replacement after 2030 (Methods, Extended Data Fig. Whereas BAU results in stable AGC stocks due to ongoing turnover, the conservation scenario projects an annual increase in carbon storage of +0.55 to +0.63 PgC yr−1 relative to 2020 levels, primarily in maturing and mature forests. Despite reduced AGC in young stands, total projected AGC by 2050 is higher under conservation (209–306 PgC) than BAU (200–296 PgC), illustrating the mitigation potential of conserving ageing forests. Yet, the modest gain suggests that conservation alone will not be sufficient for achieving large-scale sequestration goals. It is also crucial to distinguish between young forests emerging from stand replacement and those established through afforestation. Whereas afforestation represents an expansion of forest cover, stand replacement involves the replacement of existing forests, thereby reducing their long-term carbon storage potential. Although afforestation can enhance carbon sequestration27,37, its effectiveness depends on the forest type; planted, homogeneous forests provide fewer benefits for carbon storage, biodiversity and local ecosystem resilience than naturally regenerating forests. Beyond carbon storage, natural forests provide irreplaceable ecosystem and social benefits compared to monoculture plantations. Yet, only 34% of global restoration commitments prioritize natural forests, whereas 45% focus on monocultures61. Our study highlights the need for targeted conservation and forest management policies that strike a balance between preserving old forests and implementing sustainable harvesting and regrowth strategies. We call for targeted management and conservation strategies that carefully consider forest age and type to ensure forests remain functional, land-based systems for mitigating anthropogenic climate change. We used the ESA-CCI biomass v4 data, which provide estimates of AGB density in woody vegetation for 2010 and 2020 (https://data.ceda.ac.uk/neodc/esacci/biomass/data/agb/maps/v4.0). Those data are derived from a combination of Earth observation data from the Copernicus Sentinel-1 mission, Envisat's ASAR instrument and JAXA's Advanced Land Observing Satellite (ALOS-1 and ALOS-2), along with additional information from Earth observation sources. Specifically, for each time step and each member of our ensemble, we added a scaled version of the standard deviation of the biomass, with the scaling factor drawn from a normal distribution clipped to the range [−1, 1]. This approach allows us to simulate the natural variability and uncertainty in the biomass data. We then combined these perturbed maps to create a comprehensive dataset representing the possible range of biomass values. This enables us to provide confidence intervals and robust statistical measures of uncertainty in our estimates. where AGBi,j,k (t) is the perturbed aboveground biomass at location (i, j) and time t for the kth member, μi,j(t) is the mean aboveground biomass, σi,j(t) is the standard deviation of the aboveground biomass and Sk is the scaling factor drawn from a normal distribution \(N(0,\frac{1}{3})\) and clipped to the range [−1,1]. The use \(N(0,\frac{1}{3})\) ensures that the perturbations are centred around zero, introducing symmetric and controlled variability. Clipping the values to [−1,1] further ensures that the introduced variations are within a reasonable range. The AGB estimates, expressed as dry organic mass, were converted to AGC using the Intergovernmental Panel on Climate Change default carbon fraction of 0.47. GAMIv2.016,21 is an updated version of the Max Planck Institute for Biogeochemistry (MPI-BGC) forest age product, providing global forest age distributions for 2010 and 2020 at a 100-metre resolution. This version leverages the machine learning algorithm XGBoost to generate data-driven estimates of forest age, integrating over 40,000 forest inventory plots, biomass and height measurements, remote sensing observations and climate data. This multi-variable approach accounts for structural and ecological variations across climatic gradients, ensuring that forest age is not inferred from biomass alone but reflects broader growth patterns (Extended Data Fig. Unlike previous global age products that assume monotonic or simplified growth trajectories, GAMIv2.0 is explicitly designed to represent the full diversity of age–biomass relationships, including saturation and declines in older age classes, across biomes. This improves alignment with patterns observed in ground-based studies and enhances the suitability of the dataset for carbon modelling applications. A key improvement in GAMIv2.0 is the integration of Landsat-based disturbance history with machine-learning-based forest age estimates. The methodology for mapping time since the last disturbance and afforestation is based on Potapov et al.4 and Hansen et al.62. Mapping time since last disturbance and afforestation Last disturbance layer: disturbed forest areas are identified using the ‘forests affected by stand-replacement disturbances or degradation' layer from Potapov et al.4. Negative values (indicating losses in 2021 and 2022) are masked. Forest gain layer (Afforestation): afforested (regrown) forest areas are mapped using the ‘forest extent gain' layer and the canopy height data from 2020. Integrating Landsat-based and machine-learning-based age estimates To improve forest age predictions, we merged the Landsat-based time since disturbance estimates with the machine-learning-derived forest age estimates through the following decision rules: If the Landsat-based time since disturbance is ≥20 years, we retain the machine-learning estimate, regardless of its value. This fusion approach refines the estimation of time since the last stand-replacement event over the past 20 years, addressing biases that tend to overestimate the age of young forests in regrowing or afforested areas (Supplementary Fig. Despite these improvements, some biases remain, including a potential overestimation of young forest age and an underestimation of older forest age. This has important implications for our results: the observed decline in young forests may be less pronounced than suggested, whereas the reduction in older forests could be more substantial. The age maps were generated using an ensemble approach to account for uncertainties: Aleatoric uncertainty: different biomass data realizations were incorporated into the model to capture variability in input data. Epistemic uncertainty: multiple XGBoost models were trained with varying hyperparameter settings to assess variability in model performance and predictions. By incorporating these uncertainty quantifications, we provide a more robust estimate of forest age distribution. We also derived age-class fraction products at one-degree resolution, categorizing forests into two-decade intervals up to 200 years, followed by 201–300 and >300-year age classes. The GAMIv2.0 product and its associated uncertainties can be visualized in the following Google Earth Engine app: https://besnardsim.users.earthengine.app/view/globalforestage. We used the 1 × 1° gridded ‘co2flux' output generated by nine (CAMS, CMS-Flux, CT-NOAA, CTE, CarboScope, IAPCAS, MIROC, NISMON-CO2 and UoE) inversion models within the Regional Carbon Cycle Assessment and Processes Project (RECCAP-2)22 (https://meta.icos-cp.eu/objects/FHbD8OTgCb7Tlvs99lUDApO0). These models incorporate CO2 mole fraction measurements from surface stations and total column mole fraction data from satellites. The selected atmospheric inversion models include CAMS, sEXTocNEET, CTE2022, NISMON-CO2, CMS-Flux, UoE, GONGGA, THU and CAMS-Satellite. We adjusted the net carbon fluxes by excluding lateral fluxes from the calculation. Such adjustments included removing riverine carbon export to estuaries and coastal oceans and accounting for carbon transfers in crops and emissions from cement production. Finally, atmospheric inversion-based net CO2 fluxes estimates were smoothed with a Gaussian filter while maintaining the total intensity by redistributing values only among valid pixels. This approach uses a Gaussian distribution for intensity redistribution, considering only valid pixels. The filter's smoothing scale is set to a length of 500 km, defining the filter's physical length in kilometres. Sigma for the filter is calculated based on a specified degree of longitude at the equator. This approach effectively smooths the data while respecting its original structure and missing values. We created two net CO2 fluxes, circa 2010 (average between 2009–2011) and 2020 (average between 2019–2021), from which we determined the NEE changes by subtracting the latter from the former. Estimating changes in forest age distribution between 2010 and 2020 involved several steps using the 100 m GAMIv2.0 product. First, we aggregated the data to a one-degree pixel resolution for 2010 and 2020 using an average resampling method. The resulting datasets provided estimates of the mean forest age at a one-degree scale for both years (Fig. In parallel, we calculated forest age differences at GAMIv2.0's native resolution (100 m). This difference map was then resampled to a one-degree resolution using the same averaging method to maintain spatial consistency (Fig. This analysis was performed independently for each of the 20 forest age maps. In this study, it is essential to distinguish between forest ageing, which progresses at a constant rate over time, and forest growth, representing changes in biomass accumulation over time. Growth rates vary depending on stand dynamics, disturbance history and environmental conditions, whereas ageing itself remains uniform. However, because GAMI estimates forest age as a function of biomass, canopy height and climate variables, variations in modelled age primarily reflect inferred structural and ecological differences rather than the simple passage of time. This distinction is crucial for correctly interpreting trends in forest age distribution and their implications for carbon dynamics. We first computed the difference in modelled forest age between 2010 and 2020 at 100-m resolution using the GAMIv2.0 product to classify forests into stand-replaced and undisturbed ageing categories. Undisturbed ageing forests were identified as pixels where the age difference was exactly 10 years, indicating that the forest remained undisturbed and aged naturally over the decade. Stand-replaced forests were identified as pixels where the age difference was less than 10 years, meaning that a stand-replacing disturbance occurred between 2010 and 2020. This includes negative values, where the 2020 forest age is lower than in 2010, reflecting a more recent disturbance that had little time for regrowth to occur. To further analyse these patterns, we generated two derivative products for stand-replaced forests: Similarly, we created equivalent products based on an age difference of exactly 10 years for undisturbed ageing forests. To understand how initial forest age influences disturbance and ageing patterns, we stratified both stand-replaced and undisturbed ageing forests into four age classes, using their 2010 GAMIv2.0-estimated age: young forests (0–20 years), maturing forests (21–80 years), mature forests (81–200 years) and old forests (>200 years). Finally, the age difference and binary classification products were resampled to a one-degree resolution using an average resampling method, enabling large-scale analysis. This process was conducted independently for each of the 20 forest age maps to account for model variability. These resampled products served two purposes: the binary classification was used to compute the fraction of forest pixels in each one-degree grid cell that experienced stand replacement or undisturbed ageing. These metrics were then used to quantify the spatial extent and intensity of age transitions (for example, Figs. First, we analysed carbon stock across forest age classes, focusing on stand-replaced and undisturbed ageing forests. This involved partitioning 100-m pixels based on age difference, as previously described, to identify distinct categories of forests that had undergone varying degrees of change over the study period. Each partitioned class was then assigned an estimate of carbon stock. Using an average resampling method, those carbon stock estimate products were resampled to a one-degree pixel resolution. This was done independently for the stand-replaced and undisturbed ageing forest categories. Similarly, we calculated net carbon stock changes for stand-replaced forests by computing the difference in AGC stocks between 2010 and 2020. To express these changes in a flux-consistent sign convention, we multiplied the resulting values by −1 so that positive values represent carbon loss (biosphere-to-atmosphere flux, typically due to disturbance or decomposition). By contrast, negative values indicate carbon accumulation (biomass regrowth). This approach ensures consistency with commonly used flux representations while preserving the original stock change information. Finally, we estimate the average annual growth rate across age classes for stand-replaced and undisturbed ageing forests as follows: Such a procedure was done independently for the corresponding 20 forest age and biomass maps. To assess the covariation between forest age shifts and net CO2 flux changes, we analysed the relationship between stand-replacement fractions and changes in NEE derived from atmospheric inversions. To ensure the robustness of our analysis, we extract data within spatial windows of different resolutions (that is, 2° × 2°, 5° × 5° and 10° × 10°) before performing the regression analysis. This spatial window approach ensures that stand-replacement fractions and NEE changes remain spatially coherent within each inversion model. We compute the median stand-replacement fraction (20 members) and corresponding median NEE changes across all available inversion models (nine members) for each spatial window. To assess the robustness of our results, we apply Jackknife resampling, systematically excluding individual NEE inversion members to test their influence on the overall trend. To explore how changes in forest management could influence future age-class dynamics and AGC storage, we developed projections of forest age distributions to the year 2050 under two hypothetical management scenarios: BAU and forest conservation. These scenarios are inspired by, but do not strictly adhere to, the objectives of global forest policy initiatives, such as the New York Declaration on Forests, which aims to halt natural forest loss by 2030 and restore 350 million hectares of degraded landscapes and forests. We aim to provide a first-order estimate of how AGC stocks might evolve in response to shifts in forest age structures under idealized management pathways. Our projections do not account for climate-driven effects such as CO2 fertilization or altered disturbance regimes. This scenario assumes that current forest management practices and policies remain unchanged for a period of 30 years. As a result, the forest age structure in 2050 is expected to be similar to the one observed in 2020. To calculate the total carbon storage capacity for this scenario, we use a straightforward method: multiplying the total area of each age class in 2050 by its respective total carbon stock estimate and then dividing this product by the total area of each age class in 2020. In the BAU scenario, the total area of each age class in 2050 is expected to be the same as in 2020. Such carbon stock estimation can be estimated as follows: Total area of age classi, 2020 is the area of the ith age class (in hectares) in 2020. Total stocki, 2020 is the estimated total carbon stock for the ith age class (in MgC per hectare) in 2020. Aligning with the objectives of the New York Declaration on Forests, this scenario maintains the continuation of existing management practices until 2030, followed by a period of non-intervention in forest areas up to 2050. The aim is to halt BAU stand replacement by 2030 and allow forests to age naturally thereafter. In this scenario, the forest age-class distribution in 2030 is anticipated to be identical to that of 2020, whereas by 2050, all forests are expected to have matured by an additional 20 years. The total carbon stocks under this scenario are estimated by multiplying each age class's total area in 2050 by its corresponding total carbon stock estimates in 2020. This product will then be divided by the total area of each age class in 2020. The carbon stock changes estimation for the forest conservation scenario can be expressed as follows: Total area of age-class conservation2050 is the area of the ith age class (in hectares) in 2050. Total area of age classi, 2020 is the area of the ith age class (in hectares) in 2020. Total stocki, 2020 is the estimated total carbon stock for the ith age class (in MgC per hectare) in 2020. First, to investigate the influence of forest management on the relationship between forest age and biomass (Supplementary Fig. 1), we used mixed-effects models on a global scale. These models included the management category as a fixed effect and geographic coordinates as random effects to capture spatial heterogeneity. This approach allowed us to isolate the role of management while accounting for spatial variability across different regions. To manage regional variability effectively, the analysis was conducted within 2-degree latitude by 2-degree longitude windows, ensuring sufficient data points within each window for robust statistical analysis. The mixed-effects model can be expressed as: Biomass is the dependent variable, while management type (managed vs unmanaged) and forest age are fixed effects. Latitude and longitude are random effects that account for spatial variability. Within each 2-degree latitude by 2-degree longitude window, we extracted the P value of the interaction term between the management category and forest age to assess the influence of management practices on the relationship between forest age and biomass. Second, we investigated the relationship between forest age and logging fraction (Supplementary Fig. class 11 – naturally regenerating forest without any signs of management, including primary forests; class 20 – naturally regenerating forest with signs of management, for example, logging, clear-cuts and so on; class 32 – plantation forests (rotation time up to 15 years); Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All codes for the analysis of the data are available via GitHub at https://github.com/simonbesnard1/forest-age-upscale.git. Messetchkova, I. Glasgow leaders' declaration on forests and land use. UN Climate Change Conference (COP26) at the SEC–Glasgow 2021 https://webarchive.nationalarchives.gov.uk/ukgwa/20230418175226mp_/https://ukcop26.org/glasgow-leaders-declaration-on-forests-and-land-use/ (2021). Nature Restoration Law: Council Gives Final Green Light; https://www.consilium.europa.eu/en/press/press-releases/2024/06/17/nature-restoration-law-council-gives-final-green-light/ (2024). Texts Adopted, Deforestation Regulation, Wednesday, 19 April; https://www.europarl.europa.eu/doceo/document/TA-9-2023-0109_EN.html (2023). Potapov, P. et al. Mapping global forest canopy height through integration of GEDI and Landsat data. The enduring world forest carbon sink. Emerging signals of declining forest resilience under climate change. Pan, Y. et al. Age structure and disturbance legacy of North American forests. Besnard, S. et al. Quantifying the effect of forest age in annual net forest carbon balance. Fuchs, R., Herold, M., Verburg, P. H. & Clevers, J. G. P. W. A high-resolution and harmonized model approach for reconstructing and analysing historic land changes in Europe. Unprecedented pluri-decennial increase in the growing stock of French forests is persistent and dominated by private broadleaved forests. Winkler, K. et al. Changes in land use and management led to a decline in Eastern Europe's terrestrial carbon sink. Peripato, V. et al. More than 10,000 pre-Columbian earthworks are still hidden throughout Amazonia. Younger trees in the upper canopy are more sensitive but also more resilient to drought. Comparing national greenhouse gas budgets reported in UNFCCC inventories against atmospheric inversions. Besnard, S. et al. Mapping global forest age from forest inventories, biomass and climate data. Carbon sequestration potential of second-growth forest regeneration in the Latin American tropics. The carbon sink of secondary and degraded humid tropical forests. Santoro, M. & Cartus, O. ESA Biomass climate change initiative (Biomass_cci): global datasets of forest above-ground biomass for the years 2010, 2017, 2018, 2019 and 2020 v4 [dataset]. Abrupt increase in harvested forest area over Europe after 2015. Significant increase in natural disturbance impacts on European forests since 1950. Seidl, R. & Senf, C. Changes in planned and unplanned canopy openings are linked in Europe's forests. Leng, Y. et al. Forest aging limits future carbon sink in China. The effectiveness of global protected areas for climate change mitigation. Cook-Patton, S. C. et al. Mapping carbon accumulation potential from global natural forest regrowth. Gatti, L. V. et al. Amazonia as a carbon source linked to deforestation and climate change. Zhao, Z. et al. Central African biomass carbon losses and gains during 2010–2019. Jakovac, C. C., Peña-Claros, M., Kuyper, T. W. & Bongers, F. Loss of secondary-forest resilience by land-use intensification in the Amazon. Harris, N. L. et al. Attribution of net carbon change by disturbance type across forest lands of the conterminous United States. Fan, L. et al. Siberian carbon sink reduced by forest disturbances. Barrett, K. et al. Postfire recruitment failure in Scots pine forests of southern Siberia. Yu, Z. et al. Forest expansion dominates China's land carbon sink since 1980. Wild, B. et al. VODCA2GPP—a new, global, long-term (1988–2020) gross primary production dataset from microwave remote sensing. The Atlantic forest of South America: spatiotemporal dynamics of the vegetation and implications for conservation. Temperature extremes of 2022 reduced carbon uptake by forests in Europe. Bastos, A. et al. Impacts of extreme summers on European ecosystems: a comparative analysis of 2003, 2010 and 2018. Harvested area did not increase abruptly—how advancements in satellite-based mapping led to erroneous conclusions. et al. First signs of carbon sink saturation in European forest biomass. Potapov, P. V. et al. Eastern Europe's forest cover dynamics from 1985 to 2012 quantified from the full Landsat archive. Global forest management data for 2015 at a 100-m resolution. Brandt, M. et al. An unexpectedly large count of trees in the West African Sahara and Sahel. The key role of forest disturbance in reconciling estimates of the northern carbon sink. Santoro, M. et al. Design and performance of the Climate Change Initiative Biomass global retrieval algorithm. Luyssaert, S. et al. Old-growth forests as global carbon sinks. Global increase in biomass carbon stock dominated by growth of northern young forests over past decade. Heinrich, V. H. A. et al. Large carbon sink potential of secondary forests in the Brazilian Amazon to mitigate climate change. Poorter, L. Biomass resilience of Neotropical secondary forests. Rozendaal, D. M. A. et al. Biodiversity recovery of Neotropical secondary forests. Assessing biomass and biodiversity recovery in regenerating Amazonian forests. Global apparent temperature sensitivity of terrestrial carbon turnover modulated by hydrometeorological factors. Global maps of twenty-first century forest carbon fluxes. Basso, L. S. et al. Atmospheric CO2 inversion reveals the Amazon as a minor carbon source caused by fire emissions, with forest uptake offsetting about half of these emissions. Regional inversion shows promise in capturing extreme‐event‐driven CO2 flux anomalies but is limited by atmospheric CO2 observational coverage. Roe, S. et al. Land-based measures to mitigate climate change: potential and feasibility by country. Restoring natural forests is the best way to remove atmospheric carbon. Hansen, M. C. et al. High-resolution global maps of 21st-century forest cover change. Relationships between net primary productivity and stand age for several forest types and their influence on China's carbon balance. He, L., Chen, J. M., Pan, Y., Birdsey, R. & Kattge, J. Relationships between net primary productivity and forest stand age in U.S. forests. We thank the Global Land Monitoring group members at the GFZ Helmholtz Centre for Geosciences for providing feedback on the presented results. We acknowledge funding support by the European Union through the FORWARDS (https://forwards-project.eu/) and OpenEarthMonitor (https://earthmonitor.org/) projects. acknowledges support from the Consultative Group on International Agricultural Research (CGIAR) MITIGATE+ project, the World Resources Institute (WRI) Land and Carbon Lab and the Open Earth Monitor Project funded by the European Union (grant agreement number 101059548). acknowledge support by the Project Office BIOMASS (grant number 50EE1904) funded by the German Federal Ministry for Economic Affairs and Climate Action. acknowledges the support of the European Space Agency (ESA) under contract number 4000123662/18/I-NB (Climate Change Initiative BIOMASS). We recognize the use of OpenAI's ChatGPT and Grammarly AI tools to enhance the manuscript's sentence structure, conciseness and grammatical accuracy. Importantly, we emphasize that the conceptualization, research and results presented are entirely our own. Open access funding provided by GFZ Helmholtz-Zentrum für Geoforschung. GFZ Helmholtz Centre for Geosciences, Potsdam, Germany Simon Besnard, Viola H. A. Heinrich, Martin Herold & Daniela Requena Suarez Max Planck Institute for Biogeochemistry, Jena, Germany Laboratoire des Sciences du Climat et de l'Environnement, CEA-CNRS-UVSQ-UPSACLAY, Gif sur Yvette, France College of Urban and Environmental Sciences, Peking University, Beijing, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar prepared the outputs of GCP atmospheric inversions and P.C. The authors declare no competing interests. Nature Ecology & Evolution thanks Jing Chen, Jamis Bruening and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a–k, Total forest area per age class in 2010 (blue) and 2020 (green) across the eleven TRANSCOM land regions (see Supplementary Fig. Values represent the median estimates across 20 ensemble members, with error bars indicating the 5th and 95th percentiles. a–k, Relationship between forest biomass and age across hydro-climate zones, based on the GAMI dataset and ESA-CCI biomass v4 data for 2020. The figure illustrates consistent patterns of biomass accumulation with forest aging, reflecting broad ecological trends. Each pixel represents a median estimate of the 100 m pixels belonging to a specific category within each one-degree pixel. a–k, Annual growth rates across age classes for forests replaced by young stands (in orange) and undisturbed ageing forests (in green) in each TRANSCOM-Land region. Stand-replaced forests (orange) correspond to areas where older stands have been replaced by younger forests, which generally exhibit higher growth rates due to their early successional stage. In contrast, undisturbed ageing forests (green) represent forests that continue ageing without stand replacement, typically showing lower growth rates as they approach maturity. Spatial distribution of carbon stocks (in MgC ha-1) in undisturbed ageing established forests (a) young (forest age <= 20 years old), (b) intermediate (21–80 years old), (c) mature (81–200 years old), and (d) old forests (forest age > 200 years old) in 2010. Each pixel represents a median estimate of the 100 m pixels belonging to a specific category within each one-degree pixel. Relationship between the fraction of forests replaced by young stands (that is, stand-replacement extent) and changes in net CO2 fluxes between circa 2020 (average of 2019–2021) and 2010 (average of 2009–2011) for young (a), maturing (b), mature (c) and old (d) stand-replaced forests. To smooth the spatial distribution of net CO2 fluxes, we applied a Gaussian filter (length = 500 km, equivalent to approximately four one-degree pixels). This smoothing technique reduces noise in the data and helps minimise the influence of local transport errors. Potential total carbon stock (a) in 2050 across age classes for the two scenarios. We also show the forest age class distribution in 2050 (b) under BAU and forest conservation scenarios. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Global covariation of forest age transitions with the net carbon balance. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41477-025-02072-z'>A single-cell, spatial transcriptomic atlas of the Arabidopsis life cycle</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 10:13:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Arabidopsis has been pivotal in uncovering fundamental principles of plant biology, yet a comprehensive, high-resolution understanding of its cellular identities throughout the entire life cycle remains incomplete. Here we present a single-nucleus and spatial transcriptomic atlas spanning ten developmental stages, encompassing over 400,000 nuclei from all organ systems and tissues—from seeds to developing siliques. Leveraging paired single-nucleus and spatial transcriptomic datasets, we annotate 75% of identified cell clusters, revealing striking molecular diversity in cell types and states across development. Our integrated approach identified conserved transcriptional signatures among recurrent cell types, organ-specific heterogeneity and previously uncharacterized cell-type-specific markers validated spatially. Moreover, we uncover dynamic transcriptional programs governing secondary metabolite production and differential growth patterns, exemplified by detailed spatial profiling of the compact yet complex apical hook structure; this profiling revealed transient cellular states linked to developmental progression and hormonal regulation, highlighting the hidden complexity underlying plant morphogenesis. Functional validation of genes uniquely expressed within specific cell contexts confirmed their essential developmental roles, underscoring the predictive power of our atlas. Collectively, this comprehensive resource provides an invaluable foundation for exploring cellular differentiation, environmental responses and genetic perturbations at high resolution, advancing our understanding of plant biology. Multicellular organisms have evolved distinct organs to perform specific complex tasks and address challenges that may be present throughout their life. Unlike most animals, plants continuously undergo post-embryonic organogenesis to form new organs throughout development. In the model plant Arabidopsis, many tissues and organs present throughout its life cycle are composed of identical or similarly classified cell types that share functionality across organs and developmental stages1. For example, epidermal cells are the interface of the plant with the external environment for all organs, while internally localized vascular cells universally function in water and nutrient transport2,3. Despite the current understanding of plant anatomy, there is limited understanding of the molecular constituents that define cell types and states within the native context of diverse organs present throughout development. While cell-type marker genes exist for many recurrent cell types across distinct organs, the form and function of cell types can vary depending on the tissue context or cellular niche from which they originate. Intra-cell-type heterogeneity is exemplified by Arabidopsis sepal and silique epidermal pavement cells, which undergo rapid cellular elongation depending on transcript dosages of the epidermal master regulator ATML1 (ref. 4) or regulators of the cell cycle5 or in response to fertilization signals6, demonstrating that identically classified cell types can undergo varied developmental programs dependent on internal cues. In addition to cell-type classification, cellular states (which can be defined as variations in molecular phenotypes that do not impact the developmental potential7) may further differentiate individual cells within cell types and may reflect developmental factors, such as the cell cycle8, tissue/cellular age9, the spatial location and cellular neighbourhoods within tissues10, and external stimuli11. The molecular identity of a cell can thus be defined as the cumulative expression of transcripts that determine both cell type and state. Single-cell RNA sequencing has been used to provide detailed maps of cell types in plants12. Still, its application is generally restricted to selected organs, tissues and cell types13,14,15,16,17 within individual studies13,14,15,16,17,18,19, posing a bottleneck towards a standardized comprehensive understanding of cell types and states in this model organism. While a recent study addresses this gap through the generation of single-nucleus transcriptomic datasets of tissues present throughout Arabidopsis development20, a challenge remains in the annotation and validation of large numbers of new marker genes, many of which may have no known function or are uncharacterized. To overcome this challenge, spatial transcriptomic methods present a powerful tool for the native detection of transcripts in situ within tissues, which simultaneously facilitates the validation of newly identified marker genes and the elucidation of spatially regulated expression21,22,23,24,25. Furthermore, the complementary application of single-cell and spatial transcriptomic technologies enables the investigation and elucidation of spatially and temporally regulated cellular states that potentially underlie biological function26 and environmental interaction27 within individual cell types. To date, however, there has not been wide application of single-cell and spatial omics to diverse Arabidopsis organs within individual studies. Here we present a single-nucleus transcriptome atlas with companion spatial transcriptome atlases as a foundational dataset of the Arabidopsis life cycle. The paired application of single-nucleus and spatial transcriptomic methods facilitated the identification of new cell-type-specific genes across diverse organs and enabled confident annotation of 138/183 (75%) of the clusters identified in our study. Furthermore, our paired single-nucleus and spatial transcriptomic datasets enabled the transcriptome-wide analysis of cellular states associated with rapid differential growth within the etiolated Arabidopsis seedling hypocotyl. Lastly, the examination of mutants in genes that we found to be under the combined regulation of cell-type and developmental specificity revealed functional roles of these transcripts in plants at the physiological level only within their native expression contexts. This study will be a powerful foundation for hypothesis generation that may facilitate discoveries within highly specific cell populations throughout the Arabidopsis life cycle. To generate a comprehensive atlas of Arabidopsis development, we collected six distinct organ systems that encompass diverse tissues as well as whole organisms at ten discrete developmental time points throughout the Arabidopsis life cycle corresponding to established developmental road maps28,29,30. This includes imbibed and germinating seeds, three stages of seedling development, developing and fully emerged rosettes, the stem, flowers and siliques (Fig. For each organ system, we also generated a paired spatial transcriptomic dataset (Fig. For the single-nucleus datasets, 432,919 nuclei from the ten developmental time points passed accepted droplet-based single-nucleus filtering metrics31 (median unique molecular identifier (UMI), 916; Extended Data Fig. For the seedling and rosette samples that constitute developmental time series, we also performed integrative analyses that revealed clusters corresponding to known cell types (Extended Data Fig. 1c,d), which confirms the reproducibility of these datasets and enables the investigation of developmentally regulated processes, such as root hair development (Supplementary Fig. 3), as reported recently20, across developmentally distinct samples. We also merged all single-nucleus RNA sequencing (snRNA-seq) datasets into a global dataset (Fig. 1f) encompassing all cell types across the Arabidopsis life cycle. Developmental time points surrounded by solid lines denote matching single-nucleus and spatial transcriptomic datasets. b,c, Paired single-nucleus and spatial transcriptomic datasets generated in our study. d, Uniform manifold approximation and projection (UMAP) plots of each dataset with select cluster annotations displayed for each dataset. e, Spatial transcriptomic assays analysed in this study, including Arabidopsis seed, seedling, mature leaf, stem, flower cross and longitudinal sections, and silique. The scale bar lengths are shown. f, t-distributed stochastic neighbour embeddings of the fully integrated dataset. Nuclei are coloured according to the tissue of origin. g, Web browser access to the Arabidopsis Developmental Atlas Browser and CELLxGENE instances of flower single-nucleus and spatial datasets. h, A spatial transcriptomic dataset within the MERSCOPE Visualizer software. For each micrograph in e, results were observed across ten independent seeds, eight independent seedlings, two independent leaves, one stem, three independent flowers (one in longitudinal orientation and two in cross orientation) and three independent siliques. To compare cell types across organs and development, we analysed each dataset independently, which resulted in 183 clusters across all datasets (Fig. Cluster annotation was performed according to the following guidelines. First, an extensive list of marker genes was compiled with known cell-type- and tissue-specific expression in all Arabidopsis organs (Supplementary Table 1), including cell-type-specific markers recently identified from single-cell RNA-seq studies18,19,32,33,34,35. Many of these known cell-type-/tissue-specific expressed genes were enriched in specific clusters of our dataset (>1,000 genes), which aided cluster annotation (Supplementary Fig. Second, a cell-type enrichment score for each cluster was calculated on the basis of known cell-type markers to systematically infer cell types (Methods and Supplementary Fig. Third, we investigated newly identified cluster markers within each dataset using previously generated dissection-based and cell-type-specific transcriptomic studies (TAIR36 and ePlant37,38) to confirm the accuracy of our cluster annotations (Supplementary Table 2). Finally, we spatially validated a selection of cluster markers for each tissue/organ using sequencing- and imaging-based spatial transcriptomic technologies (Fig. a, Spatial transcriptomic visualization of selected gene expression (LHB1B1 (AT2G34430), XYLOGLUCAN ENDOTRANSGLUCOSYLASE/HYDROLASE 9 (XTH9; AT4G03210), NAD(P)H dehydrogenase subunit S (NdhS; AT4G23890) and PLASTID-SPECIFIC RIBOSOMAL PROTEIN 2 (PSRP2; AT3G52150)) in the Arabidopsis seedling (zoomed in on the cotyledons). The scale bar lengths are shown. b, UMAP plot of single-nucleus transcriptomes showing expression of PSRP2 (purple dots), enriched in cluster 3 (outlined in green). c, Spatial mapping of cluster 10 annotated as ‘dividing cells' in the three-day-old seedling, with transcripts from known and new cell-cycle- and division-related genes (AT2G47500, CSLD5 (AT1G02730), AT4G14330 and HIK (AT1G18370), marked in magenta, yellow, cyan and pink, respectively). The insets show individual dividing cells with colocalized transcripts. The scale bar lengths are shown. Our spatial transcriptome approach allowed us to validate examples of known cell-type marker genes across organs and development, as well as validate newly identified cell-type-specific and tissue-specific marker genes across all organs (see Supplementary Table 3 for 109 examples of new cell-type/tissue marker genes). In contrast, we were also able to identify and validate markers that do not universally specify cell types but rather demonstrate cell-type-specific expression only within the context of specific organs, as shown for epidermal cells of seedling hypocotyls (Extended Data Fig. Spatial transcriptomics confirmed the localization of cell-type-specific genes identified via snRNA-seq in most developmental stages and plant organs tested (Extended Data Figs. In the seedling datasets, we were able to confirm cell identities such as mesophyll cells expressing LIGHT-HARVESTING CHLOROPHYLL-PROTEIN COMPLEX II SUBUNIT B1 (LHB1B1; AT2G34430)38, epidermal cells (for example, GLYCOSYL HYDROLASE 9B8 (GH9B8; AT2G32990) and AT2G12462, new epidermal markers restricted to the apical hook and cotyledons in seedlings, respectively) and vascular cells (AINTEGUMENTA (ANT; AT4G37750), which was shown to regulate ERECTA-LIKE 1 (ERL1; AT5G62230) and PHLOEM INTERCALATED WITH XYLEM (PXY; AT5G61480), which maintain procambial cell identity39) that are co-expressed in the same cluster as SUCROSE-PROTON SYMPORTER 2 (SUC2; AT1G22710)40,41,42 (Fig. We were also able to annotate clusters to cell states through the cluster-specific expression of cell-division-associated genes such as CELLULOSE SYNTHASE LIKE-D (CSLD5; AT1G02730), involved in cell plate formation43, and HINKEL (HIK; AT1G18370), which is involved in cytokinesis44 (Fig. Visual inspection of spatial marker expression within the single-nucleus datasets did not always reveal cluster-specific expression patterns, as demonstrated by LHB1B1 (Extended Data Fig. 3g), but quantitative evaluation of these markers revealed cluster- and subcluster-specific and enriched expression of these spatial markers both within individual samples (Extended Data Fig. 3k, right), demonstrating conserved expression of these cell-type and cell-state markers across diverse tissues throughout the Arabidopsis life cycle. In stems, we identified cell-type-specific gene expression localized to the procambium (for example, USUALLY MULTIPLE AMINO ACIDS MOVE IN AND OUT TRANSPORTERS 11 (UMAMIT11; AT2G40900)), cortex (for example, B-BOX DOMAIN PROTEIN 15 (BBX15; AT1G25440)), phloem (H(+)-ATPase 3 (refs. 47) (IRX3; AT5G17420)), enabling high-resolution mapping of vascular differentiation in the stem (Extended Data Fig. Spatial expression of markers in Arabidopsis floral tissue revealed epidermal markers (for example, AT2G37540), vascular-expressed genes (for example, FLOWERING LOCUS T (FT; AT1G65480)) and carpel-specific expression of AGAMOUS-like 8 / FRUITFULL (AT5G60910; ref. Spatial transcriptomics also revealed distinct expression domains within the Arabidopsis silique, with VEGETATIVE STORAGE PROTEIN 2 (VSP2; AT5G24770) and the TPX2 protein family (TPXL6; AT5G37478) marking the exocarp cell layer, and AT2G13810 showing specific localization in developing seeds (Extended Data Fig. Together, these examples demonstrate the strength of our paired single-nucleus and spatial transcriptomic datasets on matched biological samples. While we were able to annotate many clusters to individual cell types, we found that some clusters instead corresponded to anatomical regions or broad cellular states, such as the developing embryo or dividing cells (Supplementary Table 2). This led us to hypothesize that further transcriptional complexity could be resolved within each cluster when examined independently. Through systematic re-clustering of individual clusters (subclustering), we were able to determine further complexity within clusters, resulting in a total of 655 subclusters (Supplementary Fig. Correlating the aggregated transcriptomes (pseudobulk) of each subcluster, we found that subclusters generally grouped by sample type, but we also found examples of subclusters with unique gene expression patterns that grouped independent of dataset (Supplementary Fig. Together, these results demonstrate that subclustering captures subtle transcriptional states and uncovers rare or transitional cell populations that may play distinct roles across developmental or environmental contexts. The silique represents a complex system that includes distinct maternal and progeny tissues. We therefore used the silique dataset as a use case example of dataset annotation that is enhanced by the paired single-nucleus and spatial transcriptomic datasets (Fig. Within the proliferating cell population (cluster 0) that may encompass multiple cell types in siliques, subclustering revealed three transcriptionally distinct groups, with each defined by unique marker genes (Fig. Expression of these markers was restricted to the seed coat layer49, prompting further spatial investigation. a, Visualization of all 43 transcripts targeted with MERFISH in siliques. Groups of transcripts are false-coloured according to cell type and region-specific expression. Magnified images of individual seed sacs are also depicted. The scale bar lengths are shown. b,c, Subclustering of cluster 0 from the silique dataset. Groups of clusters corresponding to the outer integument (blue), inner integument (pink) and flavonoid biosynthesis (orange) clusters are depicted (b), with violin plots of select cluster markers (c). d, Relevant cell types and structures of developing seed sacs. e, Magnified images of an individual seed sac in the spatial dataset. Subcluster markers are coloured corresponding to the annotated cluster grouping. The scale bar length is shown. For the micrographs in a and e, results were observed in three siliques and at least ten seed sacs within the siliques. By mapping these markers in the spatial dataset, we were able to finely annotate subcluster groups corresponding to the seed coat's outer and inner integument layers (Fig. This annotation of individual layers within developing seeds was further supported by a similar annotation of these genes in a single-nucleus transcriptome dataset of Arabidopsis seeds35. Consistent with a previous single-nucleus dataset of developing seeds35, we also identified subcluster markers corresponding to flavonoid biosynthetic processes within our subcluster groups (Fig. 6), but spatial mapping revealed that these transcripts are detected in both the seed coat and endosperm layers of seed sacs (Fig. While flavonoid secondary metabolites are known to function in seed coat colour and cause the transparent testa phenotype in mutants of flavonoid biosynthetic enzymes50,51, the spatial expression of TRANSPARENT TESTA 4 (TT4; AT5G13930) and other flavonoid biosynthesis genes within the endosperm layer may instead be associated with the alteration of fatty acid levels in embryos. This has been observed in tt2 (AT5G35550)52, tt4 (AT5G13930)53 and tt8 (AT4G09820)54 mutants. The simultaneous but spatially distinct expression of genes encoding flavonoid biosynthesis enzymes may suggest functionally diverse roles of flavonoid metabolites within discrete cell types of individual seeds. Together, these findings demonstrate that transcriptional states may be strongly associated with secondary metabolite production and inclusive of multiple cell types, further demonstrating that the paired application of single-nucleus and spatial transcriptome datasets enables the discovery and annotation of marker genes at the resolution of individual cell layers. Spatially confined expression of polarity regulators is a hallmark of de novo organogenesis in various Arabidopsis organs10,55,56. We hypothesized that our Arabidopsis life-cycle atlas would capture polarity-defined cells. In our single-nucleus datasets, we detected transcripts of canonical polarity markers across several clusters (Supplementary Fig. 7), suggesting the presence of such cells within our datasets. However, these transcripts were sparsely expressed, limiting our ability to fully resolve polarity domains on the basis of single-nucleus data alone (Supplementary Fig. To overcome this limitation, we leveraged our spatial transcriptomics dataset and examined the expression of known polarity regulators in our three-day-old seedling dataset. We detected transcripts of known polarity regulators in expected regions of cotyledons and were able to spatially reconstruct the three-domain model of leaf development57,58 with the detection of REVOLUTA (REV; AT5G60690), WUSCHEL-related homeobox 1 (WOX1; AT3G18010) and YABBY1 (YAB1) / ABNORMAL FLORAL ORGANS (AFO) (AT2G45190) in the adaxial, middle and abaxial domains of cotyledons, respectively (Fig. The scale bar length is shown. b, Simplified three-domain model of leaf development5. c, Expression of organ polarity regulators REV and KAN1 in the single-nucleus three-day-old seedling dataset. d,e, Expression of TT4 in the spatial (d) and single-nucleus (e) datasets of three-day-old seedlings. The scale bar length is shown. f, Identification of nuclei with co-expression of canonical polarity regulators and transcripts with polar expression patterns in the three-day-old seedling spatial transcriptome dataset. g,h, Normalized pseudobulk expression of TT4 in all 183 major clusters (g) and all 655 subclusters (h). The points are coloured according to dataset. Right, zoomed-in image of the carpel. The scale bar lengths are shown. se, sepal; an, anther; pe, petal; ca, carpel. The scale bar length is shown. k, Expression of the flavonoid biosynthesis enzymes TT4–7, TT3 and TT18 in clusters of the single-nucleus flower dataset. l, Average expression of all enzymes of the flavonoid biosynthesis pathway in the flower single-nucleus dataset. Cluster 10, which shows restricted expression of these transcripts, is circled. m,n, Subcluster results and expression of cell-type markers in cluster 10 of the flower dataset. For the micrographs in a and d, results were observed in eight seedlings. For the micrographs in i and j, results were observed in two cross-sectioned flowers and one longitudinally sectioned flower. We identified new marker genes that were spatially co-expressed in regions of known polarity regulators, as exemplified by TT4, which was detected prominently and solely within the adaxial region of cotyledons (Fig. Furthermore, TT4 transcripts were more abundant than those of the canonical adaxial polarity regulators REV and AS2 (Fig. 4a,d), which could be because canonical polarity regulators encode transcription factors that are known to be expressed at lower levels in cells. Despite low detection of canonical polarity regulators in our single-nucleus datasets (Fig. 7), we were able to identify nuclei that co-expressed polarity regulators with TT4 (Fig. 4e) and other spatially validated transcripts with polar expression patterns for both abaxial and adaxial defined cells (Fig. 9), in addition to known cell-type markers (Supplementary Fig. These results demonstrate that these new markers with polar expression patterns in cotyledons denote polarity-defined cells (polarity positive), which is supported by the observation that the mutation of individual TRANSPARENT TESTA genes affects the development of specific cell types in seedlings59. While the polarity-positive nuclei were not restricted to individual clusters (Fig. 4f), we instead found that the expression of canonical and newly identified polarity markers was more defined at the subcluster level (Fig. Together, these results reveal that polar specification demarcates cell subpopulations within individual cell types and may further explain the observed subcluster diversity (Supplementary Fig. While TT4 exhibited prominent adaxial localized expression in cotyledons of three-day-old seedlings, examining TT4 expression across all organs and developmental stages revealed that TT4 expression was the greatest in several flower clusters (Fig. 4g,h), which prompted us to examine the expression and spatial distribution of TT4 within flowers. Consistent with prior findings, we found that transcripts of canonical polarity markers were spatially restricted to the expected adaxial (REV, yellow) and abaxial (YAB1 (ref. 60), cyan) regions of the carpel (Fig. 4a), we found that TT4 transcripts were instead localized to abaxial layers of carpels (Fig. As TT4 encodes the sole enzyme involved in chalcone synthesis50 upstream of flavonoid biosynthesis, we hypothesized that genes encoding other enzymes in the flavonoid biosynthetic pathway may also be spatially regulated in flowers. We found that the expression of other flavonoid biosynthesis genes (TT3-7, LEUCOANTHOCYANIDIN DIOXYGENASE (LDOX) / TT18 (AT4G22880) and FLAVONOL SYNTHASE 1 (FLS1; AT5G08640)) was enriched in the same flower cluster (Fig. 4k,l; cluster 10) and spatial regions as that of TT4 (Fig. 4i,j), which suggests that the expression of entire secondary metabolite biosynthesis pathways may influence the transcriptional identity of individual cells (Fig. While we observed the highest expression of the flavonoid biosynthetic pathway within flower subclusters, some (but not all) organs were associated with clusters that specifically express flavonoid biosynthesis genes, revealing specific roles of flavonoids in specific organs throughout the Arabidopsis life cycle (Supplementary Fig. Because flavonoids have diverse roles in several biological processes in various flower tissues61, we questioned whether the flower cluster associated with high expression of flavonoid biosynthesis enzymes, which we consistently observed with altered clustering parameters (Supplementary Figs. 12 and 13), may instead represent diverse cell types that all produce flavonoids in contrast to a single cell type. 4m) revealed three subcluster populations that were enriched for markers of tapetum62 (subcluster 0, GLYCINE RICH PROTEINS), epidermal63 (subcluster 1, 3-KETOACYL-COA SYNTHASE 10 (KCS10; AT2G26250) and TRANSPARENT TESTA genes) and female gametophyte64 (subcluster 2, ER-TYPE Ca2+-ATPase 1 (ECA1) and ECA1-like gametogenesis family genes) cells (Fig. Together, these results highlight an example of transcriptional complexity driven by the expression of secondary metabolism pathways among several cell types, where cell-type identity is resolved only at the resolution of subclusters. In the integrated global dataset (Fig. 1f), we identified at least ten clusters that correspond to recurrent cell types, such as vascular (phloem), epidermal (epidermis, guard cells and trichomes) and meristematic (procambium) cell types, that are represented by nuclei from all tissues and developmental time points assayed in which those cell types are present (Supplementary Fig. While these recurrent cell types could be identified by the cluster-specific expression of individual or suites of cell-type markers, we questioned whether heterogeneity within individual cell types may exist at the whole-transcriptome level. We initially focused on phloem companion and guard cells that were identified by the specific expression of the phloem and guard cell marker genes SUC2 (AT1G22710; refs. Independent subclustering of the phloem companion (Fig. 5b) and guard cell populations from the global dataset (Fig. 5c) revealed subcluster complexity within both of these cell types. We observed similar trends within both the phloem companion and guard cell populations, where nuclei from both the seedling and rosette time-series datasets formed individual aggregate clusters (Fig. In contrast, nuclei from other organs, exemplified by silique nuclei, generally clustered separately (Fig. 5b,c), which suggests that organ-specific transcripts contribute to the observed complexity within individual cell types. Examining populations of other recurrent cell types identified from the global dataset (Supplementary Fig. 14a–c) consistently revealed similar trends where silique nuclei clustered separately from those of different organs. From these findings, we hypothesized that the observed heterogeneity within cell types may be due to (1) genes with expression that is restricted to individual organs or developmental time points yet expressed within multiple cell types, or (2) genes with expression under the dual regulation of cell-type and organ or developmental specificity. a, t-distributed stochastic neighbour embedding of the global dataset with expression levels of the stomatal lineage marker FAMA (blue, AT3G24140) and the phloem marker SUC2 (pink, AT1G22710). b,c, Re-clustering of the phloem (b) and guard cell (c) lineage clusters. Cells are coloured according to tissue of origin. d, Quantification of subcluster markers shared between phloem and guard cell lineage subclusters. Groups of subclusters with more than ten overlapping subcluster markers are depicted. Colour indicates the dataset of origin with the highest proportion of cells for each subcluster. e, Quantification of markers uniquely identified as subcluster markers in only phloem and guard lineage cell subclusters. Subclusters with at least ten uniquely identified genes are depicted. f,g, Expression of markers uniquely identified in phloem companion cells of the stem dataset. Panel f shows the average expression of all markers uniquely identified in stem phloem companion cells. Cluster 9 of the phloem companion cell analysis (Fig. Panel g shows the expression of three stem phloem companion cell markers in the single-nucleus dataset (left) and the magnified region depicted in Fig. 1e of a vascular bundle in the stem spatial transcriptomic dataset (right). The scale bar length is shown. h, Normalized pseudobulk expression of MIOX1 (AT1G14520) across 655 subclusters. The points are coloured according to dataset. i, Quantification of the length of fully elongated siliques in Col-0 and three miox1 T-DNA mutants. n > 9 individual fully elongated siliques were measured from individual plants for each genotype. Siliques from eight plants were measured for Col-0 and the miox1-1 mutant, and siliques from two plants were measured for the miox1-2 and miox1-3 mutants. The unpaired Wilcoxon test P value is reported for each miox1 mutant compared to Col-0. j, Representative image of fully elongated siliques in Col-0 and the miox1-1 T-DNA mutant. The scale bar length is shown. To investigate this question, we examined the intersection of newly identified markers from the subsets of phloem companion and guard cells. This revealed two classes of marker genes: (1) markers that were shared across cell types within the same tissue, organ or organ system (shared markers; Fig. 5d and Supplementary Table 4) and (2) unique cell-type markers that were identified within only phloem companion or guard cell populations of individual sample types (unique; Fig. For example, CYTOCHROME P450, FAMILY 709, SUBFAMILY B, POLYPEPTIDE 1 (CYP709B1; AT2G46960) and ARABIDOPSIS NAC DOMAIN CONTAINING PROTEIN 29 (ANAC029; AT1G69490) were both identified as markers within silique guard cells (Supplementary Fig. Examining the expression of these silique guard cell markers in the context of whole-plant development, we found that CYP709B1 was restricted to the silique organ but was expressed broadly across cell types (shared marker gene), while ANAC029 expression was unique to silique guard cells within the context of whole-plant development (unique marker gene; Supplementary Fig. Consistent with the broad expression of CYP709B1 across silique cell types, we also identified shared markers of other tissues in the phloem companion and guard cell populations (Fig. Together, these results demonstrate that transcriptional identities within individual cell types are defined both by the expression of genes shared among cell types within a tissue or organ (for example, flower specific) and by genes under the combined regulation of cell-type and developmental specificity (for example, expression restricted to only one cell type in only one organ, such as silique mesocarp). Extending this cell-type-level analysis to other recurrent cell populations revealed similar results (Supplementary Fig. 14f,g and Supplementary Table 4), demonstrating that this observation was not unique to the cell types of phloem companion and guard cells but is instead widespread across diverse cell types. To systematically identify organ- and development-specific markers across Arabidopsis development, we compared the pseudobulk expression of marker genes of all 183 major clusters (Supplementary Fig. 15a, calculated from Supplementary Tables 5 and 6; Methods). Of 4,528 genes identified as markers of at least one of the major clusters, 331 (7.3%) genes were uniquely identified as markers within only one dataset (Supplementary Table 4). Consistent with the unique expression patterns in the pseudobulk data, plotting the expression of these markers both in their natively expressed dataset (for example, stem phloem; Fig. 5f,g) and in the globally integrated dataset revealed that the expression of many of these markers (>70%) was restricted to individual clusters and/or subsets of clusters (Supplementary Fig. 15b), demonstrating that our atlas identifies transcripts uniquely expressed in certain cell types or cell states, present only within specific datasets. We next questioned whether genes with unique expression patterns may have functional relevance within the natively expressed sample type. We evaluated 94 transfer DNA (T-DNA) mutant alleles66 of 46 marker genes uniquely identified or enriched as cluster or subcluster markers within individual datasets (Fig. From these mutants, we identified diverse phenotypes for seven genes (15%), including alterations in leaf morphology and physiology, petiole length, bolting time, premature leaf senescence and abnormalities in silique length and morphology (Fig. Of note, six of the seven genes were not previously associated with phenotypes, demonstrating the capacity to identify new phenotypes in mutants of genes with unique expression patterns. As an example, we focused on mutants of MYO-INOSITOL OXYGENASE 1 (MIOX1; AT1G14520), where we found that high expression of this gene was restricted to the silique dataset and was enriched within individual subclusters of this organ (Fig. In miox1 mutants, we observed a reduction (~1 mm) in the length of fully elongated siliques (Fig. 5i,j) without observing extraneous phenotypes in other tissues and developmental stages. These results were consistent in several miox1 mutants, suggesting a unique cell-type-specific role of MIOX1 in silique development throughout the Arabidopsis life cycle. Examining the expression of the other MIOX family members revealed similar trends of subcluster-specific expression in siliques for MIOX2 (AT2G19800), MIOX4 (AT4G26260) and MIOX5 (AT5G56640) (Extended Data Fig. 7), although in contrast to MIOX1, the expression of the other MIOX family members was instead restricted to other clusters and cell types in the silique dataset, which may suggest specific yet distinct roles of myo-inositol metabolism enzymes during silique development. In dicotyledonous plants, the hypocotyl apical hook is a hallmark phenotype of dark-grown (etiolated) seedlings, which is necessary to prevent mechanical damage to the stem cells within the shoot apical meristem as seedlings emerge from soil67. To maintain the apical hook structure (Fig. 6a), it is understood that the opposing activity of the hormones auxin and ethylene causes epidermal and cortex cells to transiently undergo differential rates of cell elongation along the convex–concave axis, but as a ‘standing wave' of growth68, where individual cells transit from the shoot apical meristem, progress through and ultimately exit the apical hook in fewer than 12 hours69 (Fig. The transient positions of all individual cells as they pass through the apical hook thus represent unique cellular states at the intersection of developmental and hormonal regulation. We therefore sought to use the apical hook as a model to study cellular states. The scale bar lengths are shown. b, Model of cellular states associated with the development and maintenance of the apical hook. Maximal activity levels of ethylene and auxin are localized to the convex and concave apices of the apical hook, respectively. c, Illustration of the experiments performed. d,e, Clustering of Col-0 shoot apex cells and whole hls1 seedlings. In d, nuclei from both datasets are plotted separately within the co-embedded UMAP space. The number of nuclei for each genotype among all clusters is depicted as a bar plot (right). Clusters that are overrepresented for cells of the apical hook and underrepresented in hls1 are circled in red. In e, all nuclei from the Col-0 shoot apex and hls1 seedlings are depicted. f, Re-clustering of clusters that majorly represent cells of the apical hook structure. Annotations of select clusters are depicted. A cluster corresponding to cortex cells is circled. g, Left, re-clustering of the cortex cell cluster. Right, expression of a newly identified cortex subcluster marker, IAA32 (AT2G01200). h,i, Spatial locations of transcripts associated with cellular states in the apical hook. The shoot apices of three-day-old dark-grown seedlings of Col-0 (left) and hls1 (right) are depicted. Transcripts with spatial expression patterns enriched within domains of the apical–basal axis of development (h) and the convex–concave domains of the apical hook (i) are depicted. The scale bar lengths are shown. For the micrographs in a, h and i, results were observed in at least eight seedlings for each genotype. To provide a foundation for identifying cellular states within the apical hook, we generated paired single-nucleus and MERFISH datasets of dissected shoot apices containing the apical hook from wild-type seedlings, as well as whole seedlings of hookless1 (hls1; AT4G37580), which undergoes normal seedling development but specifically lacks the apical hook structure70 (Fig. Co-clustering of these datasets revealed that specific clusters (clusters 2, 3, 5 and 6) were overrepresented by nuclei of the Col-0 shoot apex and underrepresented in hls1 seedlings, revealing that these clusters represent cells in the apical hook (Fig. Re-clustering of the apical hook cells revealed additional clusters that correspond to known cell types in hypocotyls (Fig. Further investigation of heterogeneity within cell types through subclustering analyses revealed a total of 24 subclusters that may represent distinct cellular states in all cell types present in the apical hook. To focus on cell states that may regulate differential growth programs within the apical hook, we focused on cortex clusters, which represent the most abundant cell type in the apical hook by volume71. Subclustering of cortex cluster 2 revealed five discrete cell populations in this cluster (Fig. 6g), where we identified INDOLE-3-ACETIC ACID INDUCIBLE 32 (IAA32; AT2G01200) as a subcluster marker with known expression in the basal concave region of the apical hook72, which we also confirmed in our spatial datasets (Fig. We also identified new markers with spatial expression restricted to cortex cells in convex and concave regions of the apical hook (Fig. Additionally, expression of these markers was absent or reduced in straightened hypocotyls of hls1 (Extended Data Figs. 8 and 9), further suggesting a role of cell-state markers in apical hook regulation. While we were able to identify new markers of apical–basal and convex–concave positioning, we observed varying levels of co-expression of these markers both spatially and at the single-cell level, which suggests further spatial complexity in this tissue structure. This observation is exemplified by the basal apical hook cortex markers IAA32 and DEHYDRATION RESPONSE ELEMENT-BINDING PROTEIN 26 (DREB26; AT1G21910), where we observed that IAA32 expression is uniquely restricted to the basal apical hook, while DREB26 expression is similarly expressed in the basal apical hook but also extends into the basal hypocotyl (Extended Data Fig. This observation is also reflected in the apical hook single-nucleus dataset, where we observed IAA32 and DREB26 co-expression in the same apical hook cortex cluster (cluster 2; Extended Data Fig. 8b, middle), but IAA32 and DREB26 expression becomes more distinct in apical hook cortex subclusters (Extended Data Fig. Together, these results highlight the fine-tuned spatiotemporal regulation of cortex cell neighbourhoods in the apical hook and suggests that IAA32 and other markers of cellular states in the apical hook may be directly involved in the regulation of differential growth to maintain this structure. Finally, to understand the combinatorial interaction of cellular states that regulate orthogonal spatial axes aligned with developmental progression (apical–basal) and lateral asymmetry that underlies tissue bending (convex–concave), we examined cells that co-express markers of apical–basal and convex–concave patterning, which represent rare cell neighbourhoods of ten or fewer cells within a whole organism (for example, convex cells of the basal hypocotyl) (Extended Data Fig. Independent analysis of all four cell groups revealed that markers of each of these populations were enriched for distinct and diverse Gene Ontology (GO) terms, many of which are associated with the regulation of growth or development, such as anatomical structure arrangement and pattern specification process, which are associated with apical–convex and apical–concave cell states, respectively (Supplementary Fig. Furthermore, several newly identified markers in enriched GO categories for all cellular states analysed have demonstrated function in apical hook regulation. Our comprehensive analysis of cellular states within the apical hook thus revealed diverse cellular states, represented by over 20 subclusters, as well as a role of spatiotemporally regulated cell-state genes, with pinpoint precision of expression, in wild-type growth and development of the apical hook. In this study, we used single-nucleus and spatial transcriptomic technologies to characterize cellular identities along the entire life cycle of the model plant Arabidopsis. As a resource of broad interest, complementary application of single-nucleus and spatial transcriptomic technologies enabled the annotation of known and new cell types and regions (Figs. 2–4) to diverse organs and tissues, many of which have not been investigated at high resolution or at the single-cell level. Our sampling of diverse organs allowed us to evaluate the transcriptomes of recurrent and unique cell populations across organs and developmental stages at the single-nucleus level, which revealed heterogeneity within individual cell types (Fig. With an easily accessible web application, we believe that our life-cycle atlas will be of wide use to the plant science community and scientists interested in single-cell biology for hypothesis generation and reference for future single-cell genomics studies. The integration of our data with published single-cell protoplast-based datasets (Supplementary Fig. 17) demonstrates that our atlas can function as reference datasets onto which future single-cell omics data can be integrated to further annotate cell types and/or investigate cell-type-specific responses to environmental or chemical stimuli. Furthermore, proof-of-concept analyses demonstrate that spatial single-cell analyses are feasible using our datasets (Supplementary Figs. Across the ten developmental time points assayed, we identified immense transcriptional diversity represented by 183 major clusters and 655 subclusters. In our datasets, we identified known cell types and, with the aid of our paired spatial transcriptomic datasets, were able to more finely annotate cellular populations that were obscured in our single-nucleus datasets, including individual cell layers within seed sacs (Fig. 3) and polarity-defined cells within cell types (Fig. Our life-cycle atlas also highlights a cellular state defined by the expression of enzymes of the flavonoid biosynthetic pathway that promotes co-clustering of distinct cell populations (for example, tapetum, epidermis and gametophyte; Fig. 4) at the transcriptome-wide level of single cells. What remains undetermined is the level of influence of other secondary metabolite pathways and their effect on cellular transcriptomes throughout the Arabidopsis life cycle. For each of the 655 individual cell populations, newly identified marker genes may be of use for future characterization of other distinct and potentially new cell populations (Supplementary Fig. Comparative analyses of recurrent cell types across organs are a unique aspect of our developmental atlas compared with atlases of individual organs. By examining recurrent cell types, we revealed that the expression of organ-wide transcripts further influences the transcriptional identity of cells (Fig. The expression of transcripts broadly within organ systems may indicate high levels of coordinated activity across cell types of a tissue, as is the case for flavonoid biosynthesis (Fig. 4), and may contribute to further transcriptional complexity within cell types. Transcripts uniquely expressed in individual cell populations throughout development are of particular interest, as they may reveal unique functions and gene regulatory mechanisms. Interestingly, cell types of the reproductive tissues were associated with greater quantities of uniquely expressed transcripts than other tissues, which may be attributed to cell types present only within these tissues. A phenotypic investigation of genes with cell-type and developmental specificity revealed that loss-of-function mutants are associated with diverse phenotypes only within the tissue of native expression context (Fig. Many of the genes for which we identified new phenotypes are members of multigene families, demonstrating non-redundant functions of individual gene family members within their native expression contexts, as was the case for miox1, where we identified a new shortened silique phenotype (Fig. We found that the expression of the remaining members of the MIOX family (MIOX2, MIOX4 and MIOX5) are all enriched for distinct and separate cell types within siliques (Supplementary Fig. 7), which may suggest broad yet distinct spatially partitioned functional roles of myo-inositol metabolism in silique development. Higher-order mutation of MIOX family members may provide interesting future opportunities to investigate the role of myo-inositol metabolism in silique development in both cell-type-specific and organ-system contexts. Reproductive organs are associated with the production of diverse and unique secondary metabolites73,74,75, such as flavonoids (Fig. 4), which may also be a basis for the observed unique expression patterns. We found that the expression of enzymes of the full flavonoid biosynthesis pathway was consistently associated with a single cluster in our flower dataset that encompasses epidermal, tapetal and female gametophyte cell types (Fig. An undefined alteration of cellular status or state within these cell types may occur upstream of flavonoid biosynthesis, and it will be interesting to investigate potential upstream regulatory programs that define this cellular switch in diverse cell types. Finally, we demonstrated that the hypocotyl apical hook of Arabidopsis serves as an exemplary model for the investigation of complex, spatiotemporally regulated cellular states due to its intricate cellular composition and dynamic developmental processes. Our analysis uncovered 24 distinct cell-state subclusters in this compact anatomical structure that are associated with the apical hook's characteristic differential growth patterns. In cortex cells, which represent the most abundant cell type in the apical hook, we identified new subcluster-specific marker genes that exhibited precise spatial localization to regions of the apical hook, demonstrating clear transcriptional demarcation along both the apical–basal and convex–concave axes. Importantly, the spatial expression of these marker genes was markedly diminished or completely lost in the hls1 mutant, underscoring their critical roles in hook formation and curvature maintenance. Further detailed analysis revealed cellular subpopulations co-expressing markers of both axes, such as cells uniquely positioned at intersections of apical–basal and convex–concave developmental gradients. Interestingly, we found that several of the newly identified markers of these cellular neighbourhoods have known functions in apical hook development, demonstrating that the spatiotemporal regulation of cellular states can be functionally relevant to wild-type form and function (Supplementary Fig. Collectively, our comprehensive analysis demonstrates the intricate cellular architecture and transcriptional specificity throughout the Arabidopsis life cycle. The complementary application of single-nucleus and spatial transcriptomic methodologies to diverse samples facilitated the annotation of the majority of clusters (138/183; 75%) in our life-cycle atlas. Several of the samples profiled currently lack comprehensively annotated single-cell reference datasets. As in situ spatial transcriptomic methodologies require knowledge of up to thousands of transcripts with distinct expression patterns, our datasets also function as a foundation for future spatial transcriptomics studies. Overall, our atlas highlights the complexity of cell types present throughout an organism's entire life cycle. Moreover, this atlas serves as a foundation for future studies that seek to further characterize specific organs and developmental stages or investigate stress or stimulus-driven responses. Researchers can access the processed datasets for rapid exploration in a web-based interface available at our web portal (http://arabidopsisdevatlas.salk.edu/). For the imbibed seeds (0 days old), germinating seeds (1.25 days old), light-grown 3-, 6- and 12-day-old seedling developmental time points, and 3-day-old etiolated seedlings, Arabidopsis Col-0 seeds (CS6000) were surface-sterilized via incubation in 70% (v/v) ethanol for 5 min, followed by incubation in 50% (v/v) bleach and 0.01% (v/v) TWEEN-20 for 5 min, followed by triplicate washes in double distilled water for 5 min. Sterilized seeds were placed onto 1 X Linsmaier and Skoog media (1.0 LS salts (Caisson Labs), 0.8% agar (w/v), 1% w/v Suc., pH 5.7) in 120-mm2 petri dishes at a density of 100 seeds per row for the 0-day and 1.25-day samples and 30 seeds per row for the 3-day, 6-day and 12-day samples. The imbibed seeds were stratified via incubation at 4 °C for three days in complete darkness. Following stratification, the plates were placed in a long-day (16 h light/8 h dark) growth chamber at 23 °C. For the Col-0 and hls1 three-day-old etiolated seedlings, the plates were placed in sealed dark chambers and grown in a growth chamber without lights at 23 °C. For the rosette, flower and silique tissues, seeds were sterilized identically to the seed and seedling samples, directly sown onto soil and grown at 23 °C in long-day conditions. For all samples, at least two replicates, defined as multiple plates for seed and seedling time points or at least half a flat of plants (>18 plants) for later developmental stages, were prepared independently. All the following procedures were performed on ice or at 4 °C. Tissues were chopped in 500–1,000 µl of cold NPB with a razor blade on ice for 5 min to release nuclei and incubated in 10 ml of NPB. For the 0-day and 1.25-day seed samples, the nuclei were extracted in 5 ml of NPB with a Dounce homogenizer with 20 strokes of the loose pestle followed by 20 strokes of the tight pestle. The crude nucleus extract was sequentially filtered through 70-µm and 30-µm cell strainers. Nuclei were pelleted via centrifugation at 700 g for 5 min with a swing-rotor centrifuge. The pellet was resuspended in 10 ml of NPBd (NPB, 0.05% Triton X-100, 0.05% NP40, 2% BSA) via pipetting and incubated for 10 min, followed by centrifugation at 700 g for 5 min. For seedlings and rosettes, additional washes in NPBd were performed until the pellet became translucent. The pellet was then resuspended in 1 ml of NPB and centrifuged at 50 g for 3 min in a fixed-rotor centrifuge to pellet non-nucleus debris (including pollen grains from flower tissues), and the supernatant was taken. This step was repeated until the supernatant cleared or visible pellet was not observed. For root-containing tissues (seedlings), the nucleus suspension was filtered through Flowmi strainers (40 µm) to remove additional sources of debris. Finally, nuclei were pelleted via centrifugation at 700 g for 10 min with a swing-rotor centrifuge and resuspended in an appropriate volume of NPB. Nuclei were counted manually with a haemocytometer. Nuclei from each replicate were evenly distributed into at least four independent samples, with a targeted recovery of up to 20,000 nuclei per reaction. Each nucleus suspension was loaded into the 10x Genomics Chip M, which was then processed with a 10x Genomics Chromium X controller, and sequencing libraries were constructed following the manufacturer's instructions. Each sequencing library was sequenced with the Illumina NovaSeq 6000. Briefly, equal quantities of purified nuclei from each sample were individually labelled with separate CellPlex barcodes. Following labelling, the nuclei were pelleted and washed twice with PBS containing 2% BSA with RNase inhibitor. The nuclei were then quantified again using a haemocytometer and were pooled in equal quantities and brought to the appropriate volume for loading into the 10x Genomics Chip M as described previously. Demultiplexed FASTQ files were used to generate gene-by-cell matrices with Cell Ranger (v.7.0.0)76 and were aligned to the Arabidopsis nuclear transcriptome using the include-introns flag, which was prepared with the TAIR10 genome and Araport 11 transcriptome. Downstream analyses were performed with the R (v.4.1.3)77 package Seurat (v.4.1.0)78. Low-quality and potential doublet nuclei were removed from the datasets. Nuclei that did not meet a cut-off of >300 and <7,000 genes detected, >400 UMI per nucleus, <5% mitochondrial or <15% chloroplast reads were removed. For the analysis of single-tissue datasets, after filtering, 2,000 integration features for the aggregated datasets were determined for data scaling and dimensionality reduction using the SelectIntegrationFeatures function. Each of the datasets was then integrated using the IntegrateData function with 20 principal components (PCs). Following integration of each dataset, the entire dataset was scaled and dimensionality reduction was performed with PC analysis (PCA) followed by UMAP using 30 PCs. Identification of nearest neighbours and clustering were then performed using the FindNeighbors and FindClusters functions, respectively. Cluster markers were then identified for each cluster using the FindAllMarkers function with logfc.theshold = 0.25 and min.pct = 0.1. For iterative subclustering, each individual cluster was subset and re-normalized, 2,000 variable features were identified, the data were scaled, dimensions were reduced via PCA and UMAP using 30 PCs, and neighbours and clusters were identified. 79,80, 4,000 variable features were identified and used for PCA, and pre-clustering was performed with Harmony (v.0.1.0)81 using 50 PCs. t-distributed stochastic neighbour embedding was then performed according to ref. For cell annotation, we used marker genes identified in previous studies using promoters and GUS/GFP fusions, in situ hybridization assays, bulk RNA-seq studies of dissected tissues and single-cell atlases of roots, shoot apical meristem, young leaves and inflorescence stems (Supplementary Table 1). For newly identified marker genes of each cluster, the numbers of curated cell-type markers were divided by the number of cluster marker genes to calculate the cell-type enrichment score. We also explored the expression of top cluster markers in databases such as TAIR (www.arabidopsis.org) and ePlant (bar.utoronto.ca/eplant/). Finally, we used spatial transcriptomics generated during this study for seeds, seedlings, stems, flowers and siliques to validate cell identities and uncover identities of unannotated clusters. For annotating silique major clusters, a curated list of marker genes was used35. To integrate our 21-day-old and 30-day-old rosette single-nucleus datasets with a published leaf protoplast dataset19, each dataset was individually loaded, normalized and scaled using SCTransform83, and 30 PCs were used for dimensionality reduction. Cell-type label transfer, low-dimensional embedding correction and dataset projection of the nucleus datasets were performed using the MapQuery function. To integrate the seedling and rosette time-series datasets, each dataset was pre-processed as described previously, and each dataset within the developmental time series was merged. For each merged dataset, PCs were calculated, and Harmony (v.0.1.0)81 was used to integrate the datasets regressing variables of dataset input and percentage of mitochondrial and chloroplastic reads, rounded to the nearest integer. Dimensionality reduction, nearest neighbour identification and clustering were then performed using 20 harmony dimensions. Identification of cluster markers was performed as described previously for individual datasets. First, we identified marker genes for individual clusters of each tissue type (that is, for 183 clusters) using the FindAllMarkers function of Seurat (adjusted P < 0.0005, average log2(fold change) > 2). We then checked whether these cluster marker genes are highly expressed in other organs using pseudobulk expression data of each cluster. A marker gene candidate was removed when (1) its normalized expression level in the target cluster was not more than fourfold higher than that in any other clusters in the other organs or (2) the log2 TPM value of the gene was higher than five in any clusters in the other organs. Clusters identified from the global dataset that were well represented by several datasets were selected and individually subset. Re-clustering was performed as described previously. For cross-tissue comparisons of datasets, the tissue dataset that comprised the majority population of each subcluster was specified as the organ identity for each subcluster. Systematic identification of subcluster markers was performed, and unique and overlapping sets of subcluster marker genes were visualized using ComplexUpset (v.1.3.3)84. Intersections of subcluster groups with fewer than ten shared subcluster markers were filtered for visualization. GO term enrichment of cluster and subcluster markers was performed with ClusterProfiler (v.4.2.2)85 using the enrichGO function, and P values were adjusted via the Benjamini–Hochberg procedure. A list of marker genes of silique major clusters and subclusters were filtered using the following criteria: (1) more than 25 specific probes could be designed on the basis of Vizgen's software, and (2) the TPM value was below 1,000 in the silique pseudobulk RNA-seq data (obtained by aggregating all the silique cells). After the filtering, 140 genes representing most major clusters and selected subclusters were chosen from the remaining marker genes. The panel design was balanced so that the total TPM value of 140 transcripts was below 11,000 to avoid overcrowding of signal. A list of marker genes of three-day-old seedling shoot apex major clusters and subclusters was filtered using the following criteria: (1) more than 25 specific probes could be designed on the basis of Vizgen's software, and (2) the TPM value was below 1,000 in the three-day-old seedling pseudobulk RNA-seq data86. After filtering, 600 genes representing most major clusters and subclusters were selected in addition to 400 transcription factors with expression that was specific to clusters and/or subclusters in the three-day-old seedling dataset. The panel design was balanced so that the total TPM value of 1,000 transcripts was below 15,000 to avoid overcrowding of signal. Plants were grown according to the previously described methods. Tissues of three-day-old etiolated seedlings, the stem apex, mature unpollinated flowers and fully elongated siliques matching the aforementioned developmental stages were excised and immediately incubated in OCT (Fisher) for 5 min to acclimate the tissue. Following incubation, the tissue was placed into cryomolds (Sakura), which were filled with OCT and immediately frozen in an isopentane bath cooled by liquid N2. Tissue blocks were acclimated to −18 °C in a pre-cooled cryostat chamber (Leica) for 1 h. The tissue blocks were trimmed until the tissue was entered, after which 10-µm sections were visually inspected until the region of interest was exposed. Sample mounting and preparation were performed according to the MERSCOPE user guide, with slight modifications. Briefly, a 10-µm section was melted and mounted onto a room-temperature MERSCOPE slide (Vizgen, 20400001), placed into a 60-mm petri dish and re-frozen via incubation in the cryostat chamber for 5 min. The samples were then baked at 37 °C for 5 min and incubated in fixation buffer (1× PBS, 4% formaldehyde) for 15 min at room temperature. The samples were then washed with 1× PBS containing 1:500 RNAse inhibitor (Protector RNAse Inhibitor, Millipore Sigma) for 5 min at room temperature in triplicate. Following the final PBS wash, the samples were dehydrated via incubation in 70% EtOH at 4 °C overnight. Tissue sections were processed following Vizgen's protocol. After the 70% ethanol was removed, the sample was incubated in the Sample Prep Wash Buffer (PN20300001) for 1 min and then incubated in the Formamide Wash Buffer (PN20300002) at 37 °C for 30 min. After the Formamide Wash Buffer was removed, the sample was incubated in MERSCOPE Gene Panel Mix at 37 °C for 42 h. The 1,000-target panel was used for the three-day-old etiolated seedling (Col and hls1), stem apex and flower (cross section) samples, while the silique panel was used for the flower (longitudinal) and silique samples. After the probe hybridization, the sample was washed twice with the Formamide Wash Buffer at 47 °C for 30 min and once with the Sample Prep Wash Buffer at room temperature for 2 min. After the washing, the sample was embedded in hydrogel by incubating it in the Gel Embedding Solution (Gel Embedding Premix (PN20300004), 10% w/v ammonium persulfate solution and N,N,N′,N′-tetramethylethylenediamine) at room temperature for 1.5 h. The sample was then cleared by first incubating it in the Digestion Mix (Digestion Premix (PN 20300005) and 1:40 RNase inhibitor) at room temperature for 2 h, followed by the incubation in the Clearing Solution (Clearing Premix (PN 20300003) and Proteinase K) at 47 °C for 24 h and then at 37°C for 24 h. The cleared sample was washed twice with the Sample Prep Wash Buffer and stained with DAPI and PolyT Staining Reagent at room temperature for 15 min and then washed with the Formamide Wash Buffer at room temperature for 10 min and rinsed with the Sample Prep Wash Buffer. The sample was imaged with the MERSCOPE Instrument, and detected transcripts were decoded on the MERSCOPE Instrument using a Codebook generated by Vizgen. Transcripts were visualized on Vizgen's Vizualizer. Germinating seeds (1.25 days) were grown as previously described and were subsequently submerged in OCT in a cryomold (Sakura) and flash-frozen in a liquid N2 cooled isopentane bath. Cryosectioning of the 1.25-day-old germinating seeds was performed as described above. Following sectioning, the frozen section was arranged onto the centre of the Curio Seeker Spatial Mapping Kit (Curio Bioscience) and gently melted and stored at −80 °C until further processing according to the manufacturer's specifications. Briefly, the frozen tile was gently thawed to room temperature and submerged in Hybridization Reaction Mix for 15 min, followed by sequential transfers to wash buffer and RT Reaction Mix at room temperature. Reverse transcription was performed via incubation at 52 °C for 30 min followed by tissue digestion with the addition of Tissue Clearing Reaction Mix and incubation at 37 °C for 30 min. The released beads were transferred to a new tube, pelleted via centrifugation and washed twice in Bead Wash Buffer. Following the second wash, the beads were incubated at 95 °C for 5 min and then pelleted and immediately resuspended Second Strand Mix and incubated at 37 °C for 60 min. Following incubation, Bead Wash Buffer was added, and the beads were washed three times with Bead Wash Buffer. After the third wash, the beads were resuspended in and amplified in cDNA Amplification Mix via PCR. The amplified cDNA was then purified and quantified, followed by tagmentation and clean-up. Spot demultiplexing was performed according to the manufacturer's specifications using the Seeker Bioinformatic Pipeline (Curio Bioscience). The resulting spot-by-gene matrix was used for downstream analysis with Seurat (v.4.1.0). Empty and low-quality spots with fewer than 125 genes detected or 125 UMIs were removed. Spatially aware clustering was performed on a SpatialExperiment (v.1.12.0) object using Banksy (v.0.99.7)87 with the parameters lambda = 0.2 and k_geom = 10, and the resulting matrix was used for PCA followed by UMAP dimensionality reduction with 20 PCs, followed by nearest neighbour identification and clustering. Identification of cluster markers was performed as described previously. Cell segmentation was performed using Baysor88 (v.0.7.1) on the processed transcript coordinate matrix with the following parameters: s, 6; x, global_x; y, global_y; g, gene; m, 20; p; n-clusters, 10; scale-std, 90%; count-matrix-format, ‘tsv'. Cell segmentation boundaries, centroids, molecule positions and cell-by-gene count matrices were analysed with Seurat v.5.2.1. All datasets were analysed individually and were normalized with SCTransform for all targets detected in each spatial dataset with the parameter clip.range = c(−10, 10). PCs were calculated and used for UMAP dimensionality reduction with 30 PCs, followed by nearest neighbour identification and clustering. Identification of cluster markers was performed as described previously. Plants were grown as previously described until all flowers had dehisced. All fully elongated siliques from the primary stem (n > 9) were harvested from all individual plants. For Col-0 and the miox1-1 mutant, siliques from a total of eight individuals across three biological replicates were measured. For the miox1-2 and miox1-3 mutants, silique measurements were taken from two individuals from two biological replicates. For the statistical analysis, an unpaired Wilcoxon rank sum test was performed for each miox1 mutant compared with Col-0, and the figures were generated using ggplot2 (v.3.3.6)89 and ggpubr (v.0.6.0)90. For the spatial transcriptomic experiments, similar results were observed across individual samples and across independent experiments. For the germinating seed dataset, n > 10 independent seeds were profiled simultaneously in one experiment. For the Col-0 seedling dataset, n = 8 independent seedlings were simultaneously profiled in one experiment. Technical replication of adjacent tissue sections was performed for the same seedlings. Identical sampling of n = 8 independent seedlings was performed for the hls1 seedling dataset. For the silique dataset, n = 3 independent siliques were profiled in one experiment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All datasets can be accessed through our web tool at http://arabidopsisdevatlas.salk.edu/. The raw and processed sequencing datasets are available via GEO (accession number GSE226097). The processed spatial datasets are available for download at our web resource. The Arabidopsis Book (American Society of Plant Biologists, 2002–2019). How do cells know what they want to be when they grow up? Lessons from epidermal patterning in Arabidopsis. & Weijers, D. Plant vascular development: from early specification to differentiation. Variability in the control of cell division underlies sepal epidermal patterning in Arabidopsis thaliana. et al. Growth dynamics of the Arabidopsis fruit is mediated by cell expansion. A reference cell tree will serve science better than a reference cell atlas. A. H. Cell cycle-regulated gene expression in Arabidopsis. Cell polarity in plants: when two do the same, it is not the same. Qi, F. & Zhang, F. Cell cycle regulation in the plant response to stress. Cole, B. et al. Plant single-cell solutions for energy and the environment. Taylor, I. W. et al. Arabidopsis uses a molecular grounding mechanism and a biophysical circuit breaker to limit floral abscission signaling. & Wang, J.-W. A single-cell analysis of the Arabidopsis vegetative shoot apex. & Chen, Z. J. Single-cell RNA-seq analysis reveals ploidy-dependent and cell-specific transcriptome changes in Arabidopsis female gametophytes. Misra, C. S. et al. Transcriptomics of Arabidopsis sperm cells at single-cell resolution. Shahan, R. et al. A single-cell Arabidopsis root atlas reveals developmental trajectories in wild-type and cell identity mutants. Leaf cell-specific and single-cell transcriptional profiling reveals a role for the palisade layer in UV light protection. Guo, X. et al. An Arabidopsis single-nucleus atlas decodes leaf senescence and nutrient allocation. Peirats-Llobet, M. et al. Spatially resolved transcriptomic analysis of the germinating barley grain. Zhang, X. et al. A spatially resolved multi-omic single-cell atlas of soybean development. Large-scale single-cell profiling of stem cells uncovers redundant regulators of shoot development and yield trait variation. Long, K. A. et al. Spatial transcriptomics reveals expression gradients in developing wheat inflorescences at cellular resolution. The single-cell stereo-seq reveals region-specific cell subtypes and transcriptome profiling in Arabidopsis leaves. Nolan, T. M. et al. Brassinosteroid gene regulatory networks at cellular resolution in the Arabidopsis root. Nobori, T. et al. A rare PRIMER cell state in plant immunity. Boyes, D. C. et al. Growth stage–based phenotypic analysis of Arabidopsis: a model for high throughput functional genomics in plants. Mizzotti, C. et al. Time-course transcriptome analysis of Arabidopsis siliques discloses genes essential for fruit development and maturation. Lopez-Anido, C. B. et al. Single-cell resolution of lineage trajectories in the Arabidopsis stomatal lineage and developing leaf. Distinct identities of leaf phloem cells revealed by single cell transcriptomics. Denyer, T. et al. Spatiotemporal developmental trajectories in the Arabidopsis root revealed using high-throughput single-cell RNA sequencing. Picard, C. L., Povilus, R. A., Williams, B. P. & Gehring, M. Transcriptional and imprinting complexity in Arabidopsis seeds at single-nucleus resolution. The Arabidopsis Information Resource: making and mining the ‘gold standard' annotated reference plant genome. Fucile, G. et al. ePlant and the 3D Data Display Initiative: integrative systems biology on the world wide web. Waese, J. et al. ePlant: visualizing and exploring multiple levels of data for hypothesis generation in plant biology. A. et al. AINTEGUMENTA and AINTEGUMENTA-LIKE6 directly regulate floral homeotic, growth, and vascular development genes in young Arabidopsis flowers. The Arabidopsis repressor of light signaling SPA1 acts in the phloem to regulate seedling de-etiolation, leaf expansion and flowering time. Truernit, E. & Sauer, N. The promoter of the Arabidopsis thaliana SUC2 sucrose-H+ symporter gene directs expression of beta-glucuronidase to the phloem: evidence for phloem loading and unloading by SUC2. Stadler, R. & Sauer, N. The Arabidopsis thaliana AtSUC2 gene is specifically expressed in companion cells. Gu, F. et al. Arabidopsis CSLD5 functions in cell plate formation in a cell cycle-dependent manner. The Arabidopsis HINKEL gene encodes a kinesin-related protein involved in cytokinesis and is expressed in a cell cycle-dependent manner. DeWitt, N. D. & Sussman, M. R. Immunocytological localization of an epitope-tagged plasma membrane proton pump (H(+)-ATPase) in phloem companion cells. DeWitt, N. D., Harper, J. F. & Sussman, M. R. Evidence for a plasma membrane proton pump in phloem cells of higher plants. Turner, S. R. & Somerville, C. R. Collapsed xylem phenotype of Arabidopsis identifies mutants deficient in cellulose deposition in the secondary cell wall. Preston, J. C., Hileman, L. C. & Cubas, P. Reduce, reuse, and recycle: developmental evolution of trait diversification. Belmonte, M. F. et al. Comprehensive developmental profiles of gene activity in regions and subregions of the Arabidopsis seed. Analysis of Arabidopsis mutants deficient in flavonoid biosynthesis. Bowerman, P. A., Ramirez, M. V., Price, M. B., Helm, R. F. & Winkel, B. S. J. Wang, Z. et al. TRANSPARENT TESTA2 regulates embryonic fatty acid biosynthesis by targeting FUSCA3 during the early developmental stage of Arabidopsis seeds. Xuan, L. et al. TRANSPARENT TESTA 4-mediated flavonoids negatively affect embryonic fatty acid biosynthesis in Arabidopsis. Chen, M. et al. TRANSPARENT TESTA8 inhibits seed fatty acid accumulation by targeting several seed development regulators in Arabidopsis. Stable establishment of organ polarity occurs several plastochrons before primordium outgrowth in Arabidopsis. Cell type boundaries organize plant development. Nakata, M. & Okada, K. The three-domain model: a new model for the early development of leaves in Arabidopsis thaliana. Roles of the middle domain-specific WUSCHEL-RELATED HOMEOBOX genes in early development of leaves in Arabidopsis. Architectural phenotypes in the transparent testa mutants of Arabidopsis thaliana. Watanabe, K. & Okada, K. Two discrete cis elements control the abaxial side-specific expression of the FILAMENTOUS FLOWER gene in Arabidopsis. Taylor, L. P. & Grotewold, E. Flavonoids as developmental regulators. Hsieh, K. & Huang, A. H. C. Tapetosomes in Brassica tapetum accumulate endoplasmic reticulum-derived flavonoids and alkanes for delivery to the pollen surface. Characterization of the FIDDLEHEAD gene of Arabidopsis reveals a link between adhesion response and cell differentiation in the epidermis. Ohashi-Ito, K. & Bergmann, D. C. Arabidopsis FAMA controls the final proliferation/differentiation switch during stomatal development. O'Malley, R. C., Barragan, C. C. & Ecker, J. R. A user's guide to the Arabidopsis T-DNA insertion mutant collections. & Zhong, S. Activation of HLS1 by mechanical stress via ethylene-stabilized EIN3 is crucial for seedling soil emergence. & Guo, H. To curve for survival: apical hook development. Lehman, A., Black, R. & Ecker, J. R. HOOKLESS1, an ethylene response gene, is required for differential cell elongation in the Arabidopsis hypocotyl. Montenegro-Johnson, T. D. et al. Digital single-cell analysis of plant organ development using 3DCellAtlas. Cao, M. et al. TMK1-mediated auxin signalling regulates differential growth of the apical hook. Olas, J. J., Apelt, F., Watanabe, M., Hoefgen, R. & Wahl, V. Developmental stage-specific metabolite signatures in Arabidopsis thaliana under optimal and mild nitrogen limitation. New insights into the regulation of leaf senescence in Arabidopsis. Zheng, G. et al. Massively parallel digital transcriptional profiling of single cells. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Liu, H. et al. DNA methylation atlas of the mouse brain at single-cell resolution. Kobak, D. & Berens, P. The art of using t-SNE for single-cell transcriptomics. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Linderman, G. C., Rachh, M., Hoskins, J. G., Steinerberger, S. & Kluger, Y. Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. Choudhary, S. & Satija, R. Comparison and evaluation of statistical error models for scRNA-seq. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Zhang, Y., Li, N. & Wang, L. Phytochrome interacting factor proteins regulate cytokinesis in Arabidopsis. Singhal, V. et al. BANKSY unifies cell typing and tissue domain segmentation for scalable spatial omics data analysis. Cell segmentation in imaging-based spatial transcriptomics. Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer, 2016); https://ggplot2.tidyverse.org Kassambara, A. ggpubr: ‘ggplot2' based publication ready plots. We thank G. Zzyzyx for the illustrations. was supported by the Human Frontiers Science Program Long-Term Fellowship (no. N.I.-E. is a research fellow at the George E. Hewitt Foundation for Medical Research and an awardee of the Weizmann Institute of Science—Women's Postdoctoral Career Development Award in Science. Research reported in this publication was supported by the National Institute of General Medical Sciences of the National Institutes of Health under award no. is an investigator at the Howard Hughes Medical Institute. Present address: Sainsbury Laboratory, University of East Anglia, Norwich, UK These authors contributed equally: Travis A. Lee, Natanella Illouz-Eliaz, Tatsuya Nobori. Plant Biology Laboratory, Salk Institute for Biological Studies, La Jolla, CA, USA Travis A. Lee, Natanella Illouz-Eliaz, Tatsuya Nobori, Jiaying Xu, Bruce Jow, Joseph R. Nery & Joseph R. Ecker Genomic Analysis Laboratory, Salk Institute for Biological Studies, La Jolla, CA, USA Travis A. Lee, Natanella Illouz-Eliaz, Tatsuya Nobori, Jiaying Xu, Bruce Jow, Joseph R. Nery & Joseph R. Ecker Howard Hughes Medical Institute, Salk Institute for Biological Studies, La Jolla, CA, USA Travis A. Lee, Natanella Illouz-Eliaz, Tatsuya Nobori, Bruce Jow & Joseph R. Ecker Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar serves on the scientific advisory board of Zymo Research, Inc. and Cibus, Inc. The other authors declare no competing interests. Nature Plants thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Number of nuclei within each dataset. b, Number of genes detected per nucleus within each dataset. c, Integration of the 3d-, 6d-, and 12d-old seedling datasets. Barplot depicts the distribution of cells per dataset for each cluster. d, Integration of the 21d- and 30d-old rosette nuclei datasets. Barplot depicts the distribution of cells per dataset for each cluster. a and b, Quality metrics of the Curio spatial transcriptomics dataset of germinating seeds. c, Representative micrograph of a cryosectioned seed. Regions of the seedling that correspond to clusters in the spatial transcriptomic dataset are false colored accordingly. d, Dimensionality reduction and de novo clustering of spatial transcriptomics data of 1.25-day-old germinating seeds. e and f, Mapping of clusters to spatial coordinates. Colors correspond to cell types and anatomical regions of germinating seeds (c). g to m, Cross-comparison of spatial transcriptomics derived cluster markers in the matched droplet-based single nucleus dataset. (g) Dimensionality reduction and clustering of the droplet-based 1.25 d seed dataset in Fig. (h to j) Expression of cluster markers identified in the spatial transcriptomics dataset corresponding to the (h) stele (cluster 4), (i) cotyledon (cluster 5), and (j) unannotated (cluster 6) in the droplet-based dataset. (k to m) Average expression of cluster markers identified in the spatial transcriptomics dataset corresponding to the (k) epidermis (cluster 10), (l) root tip (cluster 10), and (m) root stele (cluster 14) in the droplet-based dataset. a, Merged image showing the spatial distribution of transcripts representing distinct cell types and regions in 3-day-old Arabidopsis seedlings. b, Spatial localization of transcripts for GLYCOSYL HYDROLASE 9B8 (GH9B8) and AT2G12462, marking epidermal cells in the hypocotyl and cotyledons, respectively. c, Spatial distribution of transcripts for AINTEGUMENTA (ANT), SUC2, and AST91, representing vascular cell identities. d, Spatial expression of transcripts associated with dividing cells in cluster 10 of the 3-day-old seedling dataset. Insets highlight cells with high transcript abundance of these division-associated genes. e-j, Expression of select spatial markers displayed in (a)-(d) in the 3-day-old seedling single nucleus dataset. (e), AT4G23890 is specifically enriched in cluster 3 (outlined in blue), associated with mesophyll identity. (f), XTH9, a cell wall remodeling gene, shows broad expression with enrichment in the same region as AT4G23890. (g), LHB1B1, a light-harvesting complex gene, is enriched in cluster 3 cells but is also highly expressed across a wide range of cells, indicating widespread photosynthetic activity. (h), HIK, required for cytokinesis and is involved in physical cell division, is enriched in cluster 10 (outlined in purple), corresponding to dividing cells. (i), CSLD5, a cellulose synthase-like gene involved in cell plate formation during cytokinesis, displays punctate expression across dividing populations. (j), AT2G47500 shows broad expression with enrichment in dividing clusters. Color intensity represents normalized expression levels (purple gradient, min to max). Scale bars: 1 mm ([a]–[c], left panel in d); 100 μm (left inset in d); 50 μm (right inset in d). k, Normalized pseudobulk expression of spatial transcriptomic markers depicted in Fig. For the micrographs depicted in (a)-(d), results were observed in eight seedlings. a, Merged image showing the spatial distribution of transcripts representing distinct cell types and regions in the Arabidopsis stem. b, Merged image of transcripts enriched in procambium cells. c, Spatial expression of AT4G29100, AT5G22090, UMAMIT11, and ENDOL9 in procambium regions. d, Merged image of transcripts localized to cortex cells in the stem. e-f, Spatial distribution of (e) BBX15 and (f) AT1G64500 transcripts in cortex cells. g, Merged image of transcripts localized to phloem cells in the stem. h-i, Spatial distribution of (h) AT1G67870 and (i) HA3 transcripts in phloem cells. j, Merged image of transcripts localized to xylem cells in the stem. k-l, Spatial distribution of (k) IRX3 and (l) XCP2 transcripts in xylem cells. m, Normalized pseudobulk expression of spatial transcriptomic markers depicted in (h) and (l), across 183 clusters. For the micrographs depicted in a-l, results were observed in one to one stem. a, Merged image showing transcripts expressed in distinct cell types and regions of the flower. b, Spatial distribution of AT2G37540 transcripts marking epidermal cells in the outer layers of sepals and petals. c, Spatial distribution of AT1G61660, AT1G65480 - FLOWERING LOCUS T (FT), and AT4G08290 transcripts identifying vascular tissues within the flower. d, Localized expression of AT3G20470 in vascular regions. g, Expression of AT1G64370 in vascular cells within a floral cross-section. h, Expression of AGAMOUS-LIKE 8 (AGL8) / FRUITFULL (FUL) in the carpel region of the flower. i, Normalized pseudobulk expression of spatial transcriptomic markers depicted in (b), (d), (e), and (g), across 655 subclusters. For the micrographs depicted in a-e, results were observed in one longitudinally sectioned flower. a, Merged image displaying transcripts expressed in distinct cell types and regions of the Arabidopsis silique. b, Merged image highlighting transcripts localized to the exocarp cells. d, Spatial distribution of AT5G24770 - VEGETATIVE STORAGE PROTEIN 2 (VSP2) transcripts in the exocarp cells. e, Spatial distribution of AT2G13810 transcripts in the developing seeds. f, Spatial distribution of AT5G37478 (TARGETING PROTEIN FOR XKLP2 protein family [TPX2]) transcripts in the exocarp cells. g, Normalized pseudobulk expression of spatial transcriptomic markers depicted in (e), and (f) across 655 subclusters. a to d, Expression of the four MIOX family members (MIOX1, MIOX2, MIOX4, and MIOX5) within (left) the 655 identified subclusters and visualized within the (middle) global and (right) silique datasets. Clusters with high expression are circled for emphasis. a and b, Expression of IAA32 (AT2G01200) and DREB26 (AT1G21910) that are spatially restricted to the basal hypocotyl cortex cells within the apical hook. (a) Spatial location of DREB26 and IAA32 transcripts in three-day-old etiolated seedlings of (left) Col-0 and (right) hls1. Quantification of DREB26 and IAA32 mRNA molecules in hypocotyls of Col and hls1 is depicted in parentheses. (b) Expression of (top) IAA32 and (bottom) DREB26 within the (left) whole seedling clustering, (middle) apical hook re-clustering, and (right) subclustering of individual apical hook clusters. c and d, Expression of DAG2 (AT2G46590) and RIP2 (AT2G37080) that are spatially restricted to the cortex cells within the apical region of the shoot apex. (c) Spatial location of DAG2 and RIP2 transcripts in three-day-old etiolated seedlings of (left) Col-0 and (right) hls1. Quantification of DAG2 and RIP2 mRNA molecules in hypocotyls of Col and hls1 is depicted in parentheses. (d) Expression of (top) DAG2 and (bottom) RIP2 within the (left) whole seedling clustering, (middle) apical hook re-clustering, and (right) subclustering of individual apical hook clusters. For the micrographs depicted in (a) and (c), results were observed in at least three seedlings per genotype. a and b, Expression of AT1G72290, AT2G34930, and BBM (AT5G17430) that are spatially restricted to the hypocotyl cortex cells within the convex region of apical hook. (a) Spatial location of AT1G72290, AT2G34930, and BBM transcripts in three-day-old etiolated seedlings of (left) Col-0 and (right) hls1. Quantification of AT1G72290, AT2G34930, and BBM mRNA molecules in hypocotyls of Col and hls1 are depicted in parentheses. (b) Expression of (top) AT1G72290 and (bottom) AT2G34930 within the (left) whole seedling clustering, (middle) apical hook re-clustering, and (right) subclustering of individual apical hook clusters. c and d, Expression of GH3.3 (AT2G23170) and AT3G51410 that are spatially restricted to the cortex cells within the concave region of the apical hook. (c) Spatial location of GH3.3 and AT3G51410 transcripts in three-day-old etiolated seedlings of (left) Col-0 and (right) hls1. Quantification of GH3.3 and AT3G51410 mRNA molecules in hypocotyls of Col and hls1 is depicted in parentheses. (d) Expression of (top) GH3.3 and (bottom) AT3G51410 within the (left) whole seedling clustering, (middle) apical hook re-clustering, and (right) subclustering of individual apical hook clusters. For the micrographs depicted in (a) and (c), results were observed in at least three seedlings per genotype. a to d, Cortex cells that co-express combinations of markers associated with apical-basal or convex-concave regions within the apical hook. Cluster markers and cell-type annotation of individual tissue datasets. Newly identified cell-type-/tissue-specific marker genes across all organs. Uniquely expressed and shared subcluster markers identified from the cross-cell-type analysis. Pseudobulk expression values of 183 major clusters. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Lee, T.A., Illouz-Eliaz, N., Nobori, T. et al. A single-cell, spatial transcriptomic atlas of the Arabidopsis life cycle. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/new-treatments-can-free-kids-from-the-deadly-threat-of-peanut-allergy/'>New Treatments Can Free Kids from the Deadly Threat of Peanut Allergy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Anabelle Terry, a slender, self-possessed 13-year-old, has heard the peanut butter story her entire life. At two and a half she ate nuts for the first time. Her mother, Victoria, had made a little treat: popcorn drizzled with melted caramel, chocolate and peanut butter. “And afterward, I felt really sick,” she says. A few minutes later she vomited on the kitchen floor. It might be a snack that someone hadn't scrutinized or a food package that didn't list all potential allergens. “Eating lunch, all my friends would have PB&Js. “And going over to friends' houses after school, we always had to make sure: ‘Hey, would you mind making a nut-free meal? If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Most of that caution is in Anabelle's past now. For the vast majority of patients, peanut allergy is an unpredictable, lifelong affliction. But thanks to a clinical trial that Anabelle entered when she was nine, she can now tolerate peanuts and tree nuts well enough to feel safe every day. “It's an amazing time right now,” says R. Sharon Chinthrajah, an associate professor at Stanford Medicine, who led the national trial Anabelle joined. In fact, medicine's entire understanding of how to keep children safe from ever developing allergies is being rethought. With peanut reactions, for instance, there are real hopes that children can be protected—definitely from the worst effects and maybe from any at all. “The future looks very bright for our patients to have more choices in different periods in their lifetime,” Chinthrajah says. “We're not yet at the cure, but we're definitely moving along on the therapeutic front to be able to deal with this chronic disease.” Peanut allergies are perplexing, in part because they appeared so recently. Food reactions have occurred throughout recorded history, but widespread peanut problems didn't begin to surface until the 1990s. The effects on everyday life were dramatic: airlines began to deprive passengers of peanuts and announce that certain snacks might threaten someone else onboard. Elementary schools set aside peanut-free tables at lunch, and food manufacturers began to label their baked goods “school-safe” to signal they were free of common allergens. Epinephrine auto-injectors, which can shut down severe allergic shock (and are usually called EpiPens, for the dominant trademarked version), were rare and carried mostly for the occasional beesting. Scott Sicherer, a clinician and director of the Jaffe Food Allergy Institute at the Icahn School of Medicine at Mount Sinai in New York City, watched reports of peanut threats rise in real time. In 1997 he and his colleagues conducted the first survey of peanut and tree-nut allergy in the U.S., finding that 1.6 percent of adults and 0.6 percent of children described themselves as allergic based on reactions they had experienced. In a third sampling, conducted 11 years after the first one, the overall rate tripled from that initial measurement to 2.1 percent of children, and peanut allergies were reported in 1.4 percent of kids. And what's more common is now also more dire: researchers at the Mayo Clinic have estimated that emergency-department visits for anaphylactic shock caused by foodborne allergies—the kind of reaction that can squeeze shut airways and trigger heart attacks—increased more than threefold between 2005 and 2014. “One out of 10 individuals in the U.S., more than 33 million, has a food allergy,” says Sung Poblete, CEO of Food Allergy Research and Education, an advocacy organization. Medicine's entire understanding of how to keep children safe from allergies is being rethought. The immune system recognizes certain proteins in a food as unwelcome and launches a cascading reaction that often involves an antibody called IgE. The antibody triggers a whole-body inflammatory response: hives, swelling, vomiting, and, in the worst cases, crashing blood pressure and an inability to breathe. “Eczema, type 2 diabetes, atherosclerotic cardiovascular disease, cancer, depression—all these things have inflammatory origins and are more common now.” Researchers have proposed that cleaner modern life, early antibiotic exposure, and microbiome damage from detergents and surfactants—all components of what's called the hygiene hypothesis—might influence how often allergies develop. There may be a clue as well in which foods provoke reactions. Up to 90 percent of food allergies are caused by just eight things: peanuts, milk, eggs, fish, crustaceans, tree nuts, wheat and soybeans. Why these foods are especially allergenic also puzzles researchers. They contain complex proteins, which remain intact during digestion and may trigger the immune system in ways other foods do not; these proteins also may have similarities to common environmental allergens. Regardless of the underlying causes, research is zeroing in on ways to mitigate food allergies. But the hope is that some of the new approaches can be applied to other allergies—and to help children such as Anabelle who experience more than one. The first priority in tackling peanut allergy has been children who are at extraordinary risk, the ones whose lives are at stake if they consume something with the smallest cross-contamination from a manufacturing error. People who suffer from seasonal allergies often receive allergy shots, a program of injections that gradually decreases their sensitivity and keeps their reactions at a level they can tolerate. Allergy shots were briefly tried for peanuts as well, but they were abandoned because of safety concerns, including the 1991 death of a trial participant who received a miscalculated dose. After that, patients' only remaining option was to change their diet, but mistakes and cross-contamination kept putting them at risk. A large international study in 2018 definitively proved that the approach worked, and it became the standard for treating kids whose families weren't willing to trust avoidance. In 2020 it led to the first-ever FDA approval of a therapy for peanut allergy, a powdered form of peanut protein with the trade name Palforzia that is dispensed over months in precisely metered doses. Initially Palforzia was not approved for children younger than four years of age. That has left some allergy families searching for alternatives to oral immunotherapy. “And up to 10 percent of patients experience anaphylaxis at some point while they're on the treatment itself.” The skin method involves a patch containing peanut protein that is applied daily at home for as many as three years; the patch always contains the same dose but is gradually applied for escalating amounts of time. In a phase 3 clinical trial, the results of which were published in 2023, 67 percent of toddlers who were too young to receive Palforzia and who wore the patch were able to raise the amount of peanut protein they could safely consume to the equivalent of three or four peanuts. The hope for the patch, which has not yet been approved by the FDA, is that it will be easier for kids to tolerate because of its lower dose and easier for parents to manage logistically. He was expected to finish the trial in August of this year, and his mother already thinks his sensitivity is diminished. “Some days he doesn't really complain about it; some days he's scratching his back against the wall trying to get the itchies out,” she says. In 2013 they began testing the efficacy of an existing drug, a monoclonal antibody named omalizumab (marketed as Xolair) that is already approved for severe asthma caused by allergies. “And the beauty of something like that, where you're targeting allergic inflammation, is that it has the potential to help all allergies.” Investigators launched a trial that admitted children and adults who showed allergies to peanuts and at least two other foods; Anabelle Terry was one of the participants. Using a complex study design with several stages, the scientists tested whether regular doses of the injectable drug worked better to reduce allergic sensitivity than did placebos; whether shorter or longer courses of the drug made a difference; whether it worked best alone or combined with oral immunotherapy; and how often and in what amounts people could consume allergenic food once they stopped the treatment. Based on those results, and anticipating more data, the FDA immediately approved Xolair as a protection against peanut allergy. Participating in the trial was a significant commitment for families. Jennifer Jennison's son, Jack, was two years old and allergic to eggs, peanuts and cashews—among other foods—when the trial accepted him at its Atlanta site. Every two weeks she or her husband, David, would take time off work to bring their son for an injection. After around seven months, the protocol added tests of small doses of food allergens in applesauce to the office visits; after several hours of observation to make sure the dose was safe, the family carried home boxes of premeasured allergen powder for Jack to eat every day. She was in the same arm of the trial and now eats a daily dose of peanuts, walnuts and cashews to keep her protection up. But what happened to the Jennisons afterward shows that no peanut-allergy protection is perfect yet. Convincing a child to eat the same foods every day is no small task. After a while he started to resist the Reese's Pieces, too. “For us, eggs are the most important,” Jennifer says. “I still feel more comfortable with the cross-contamination risk of peanut knowing that he had built up a tolerance. Because new approaches to desensitization have worked so well for severely affected kids, researchers have begun to address the needs of those who are somewhat less allergic. Such people, whom some immunologists call “high threshold,” include possibly 800,000 kids with peanut allergies just in the U.S. But their triggers are so different from those of highly allergic people that they had been excluded from some trials of desensitization strategies. Indeed, immunology didn't have a clear understanding of whether desensitization that started from their baseline would even achieve the same results as in highly allergic kids. All of that is now changing because after years of diagnosing patients in this class, medical practitioners could perceive that the group was being left behind. “And what we would tell those individuals is: ‘Your symptoms weren't so bad, so you're not really that much in danger. His mother, Holly, who was so thoughtful about her children's diets that she made her own baby food, was starting to introduce him to solids. She made a spinach pancake for her older daughter, slid a few morsels onto Jackson's high-chair tray, and then watched in horror as raised red hives rippled down his body. The diagnosis sent the Esteveses, who live on Long Island in New York State, hurtling into a landscape familiar to other allergy families. “I was suddenly attuned to every food label,” Holly says now, 10 years later. I had to bring everything for him.” What made it even more complex was that no one else in the family—Jackson's parents, his older sister, or a younger sister who was born soon after the pancacke incident—shared Jackson's allergies. Although he was extremely reactive to some foods, medicine considered him just minimally allergic to peanuts—and that made him eligible for a new trial launched by Sicherer and a team of researchers from several institutions, called CAFETERIA. (Allergy researchers seem to be exceptionally fond of complex acronyms. “CAFETERIA” comes from “Challenging to Foods with Escalating Thresholds for Reducing Food Allergy.” The Xolair study was known as OUtMATCH, which stood for “Omalizumab as Monotherapy and as Adjunct Therapy to Multi-Allergen Oral Immunotherapy in Food Allergic Children and Adults.”) Starting in 2019, children between four and 14 years old consumed escalating doses of peanut butter, first under medical supervision and then at home, first with a carefully measured eighth of a teaspoon and increasing every eight weeks until they were consuming one tablespoon daily. Then they were asked to eat two tablespoons of peanut butter—the amount that would go in a sandwich, which an allergic child would never try to consume—every week for 16 weeks but not necessarily in daily doses. Finally, they had to refrain for eight weeks before being tested a final time. “A 100 percent response rate is hard not to be happy with.” The study's authors say it needs to be repeated in more kids and at different medical centers. Jackson, who turned 11 this year, has been able to eat peanuts safely since he completed his participation in CAFETERIA; he'll even eat a PB&J once in a while, although it is not a favorite food. Most of his allergies to other foods remain unaffected, but “he was a success story” all the same, his mother says. The kids have to push themselves to swallow something that has made them ill in the past and that still, even in tiny doses, might produce an unpleasant reaction. Parents have to endure the stress of watching their children undergo food challenges to test their progress, knowing that life-threatening anaphylaxis might result. Plus, for highly allergic people, the most that desensitization can offer is to keep them safer, not completely safe. A newly proposed treatment might manage that by rewriting the immune system's memory of antigens—although research into it is in such early stages that results have been reported for only one patient, and the first small clinical trial is just beginning. The treatment involves successive administration of two drugs, both made by biotechnology company Regeneron. The first drug, dupilumab (marketed as Dupixent), is a monoclonal antibody that is already FDA-approved for treating moderate to severe eczema and asthma and a few other conditions; it works by blocking the action of specific cytokines, signaling proteins that encourage the production of IgE. The second, linvoseltamab, is also an antibody and recently received FDA approval for treating relapsed or refractory multiple myeloma. This cancer affects plasma cells, a category of white blood cell that produces antibodies, including IgE. Investigators initially thought that dupilumab could be a solo treatment for peanut allergy. But several trials showed that although peanut-specific IgE levels went down under its influence, there was no long-term practical benefit. Even immediately after a course of the drug in one trial, participants were unable to tolerate peanuts. In another test, drug recipients showed improved peanut tolerance right away, but it dropped three months later. That led researchers to look at the second drug, which deals a mortal blow to the cells where IgE is manufactured. In mice and monkeys, administering a dose of linvoseltamab during an ongoing course of dupilumab destroyed the cells producing IgE. “This is very different than other approaches of trying to build tolerance in patients or trying to just decrease IgE,” says Jennifer Maloney, who leads Regeneron's therapeutic work on immune, inflammatory and infectious diseases. The company has shared results from just one patient so far, a 20-year-old man with multiple severe allergies. His case has not yet been published in a peer-reviewed journal, but the company is now recruiting a small group of patients for an early-phase trial that will primarily test safety. “We're going to learn something really important,” he says. “If we wanted to cure the disease and make it go away, would this be a viable approach to doing so? If it doesn't work, we're going to learn things about why it didn't work and what we might need to do in the next trial.” It's the ultimate goal: prevention, not desensitization or cure. In the early 2000s Gideon Lack, an immunologist then at Imperial College London, went to Tel Aviv to give a talk on how food allergies were rising around the world. He asked the audience, all Israeli pediatricians and allergists, how many of them had treated children with peanut allergy. From his own experience in the U.K., he expected every hand to shoot up. This low show of hands was extraordinary, and it immediately presented an opportunity to ask why the U.K. and Israel were so different. After returning home, Lack set up a survey to compare national rates of peanut allergy. Before their first birthday, Israeli children frequently ate peanuts, often in a ubiquitous snack called Bamba—something like Cheetos but coated in peanut butter instead of cheese. By the time they were 14 months old, almost 80 percent of the Israeli children were eating at least a few grams of peanut protein every month. In contrast, 80 percent of the British children had never tasted peanuts. It's the right thing to do.” —Gideon Lack, King's College London Lack and his team wondered whether the Israeli experience showed that this well-meaning advice might be wrong. They set up a fresh study, recruiting families with infants who were between four and 10 months old and had severe eczema or showed evidence of egg allergy, signs that their IgE production was already disrupted. The babies were tested for preexisting peanut allergy, and if they were negative, they went into one of two groups. The families of one group were told to keep their children from eating peanuts until they were five years old. The rest of the families were encouraged to introduce their kids to peanut products, preferably Bamba or peanut butter. Among the children told to avoid peanuts, 13.7 percent developed peanut allergies. Lack published the results in 2015, working with a team primarily from King's College London, where he had moved to research pediatric allergies. This study, called LEAP (for “Learning Early About Peanut Allergy,” in a departure from long acronyms), caused an earthquake in allergy science. In one, published the following year, children from both arms of the LEAP study were asked to not eat peanut products in their sixth year. Allergy rates rose further among the children who had refrained from peanuts all along, but children who started eating peanut products early maintained their low rates of allergy. In a third, published in 2024, the team went back to children who had been in the LEAP study and were at least 12 years old to check whether the preventive effect lasted. In the group that ate peanut products early, only 4.4 percent had bad reactions. Early introduction “overwhelmingly will prevent peanut allergy,” Lack says. But there have been persistent challenges to implementing that idea. Health authorities no longer recommend that parents avoid feeding allergy-related foods for three years—but most national and international guidelines still recommend exclusive breastfeeding for six months, and the cultural pressure to maintain that time frame is immense. In 2019 the American Academy of Pediatrics did revise its guidance to allow the introduction of potential allergens at four to six months for children who seem likely to be at high risk, indicated by symptoms of eczema. Lack worries this approach doesn't provide exposure as early in life as the immune system needs; the children in the LEAP study and in an unrelated 2016 study of early introduction began peanut exposure at four and three months, respectively. “If you start the weaning process at four months, then the baby may not get peanut butter in significant quantities until five to six months of age. Immunologists and allergists are aware that early feeding prevents allergy. As many advances as there have been in the past decade, scientists worry that the fundamentals of peanut allergy still elude them. “This field is still relatively early in its development compared with oncology or respiratory medicine, which are targeting very specific biological pathways with very specific precision treatments,” Emory's Vickery says. But can I see that on the horizon? At the University of California, Los Angeles, a team led by Andre Nel has developed a lipid nanoparticle that uses mRNA—the same technology used in the COVID vaccines that were developed rapidly in 2020—to create fragments of peanut allergens. Peanut allergy may be due for particular attention from the Trump administration. The president's Secretary of Health and Human Services, Robert F. Kennedy, Jr., has several times endorsed an unsupported contention that peanut allergy is caused by childhood vaccinations. Earlier this year “peanut allergies” appeared on a list of topics that would cause grants to get extra scrutiny within the NIH. “If I go off to summer camp, I have to go away from the other kids for a while and take my nuts to make sure nobody else who has a nut allergy gets sick,” she says. “Going on vacations, I always have to bring a giant bag of nuts with me in my backpack. It would feel pretty nice just being able to go in for a little visit and just get a shot. Maryn McKenna is a journalist specializing in public health, global health and food policy and is a contributing editor at Scientific American.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/human-uniqueness-is-a-myth-mounting-evidence-shows/'>Whales Mourn, Birds Show Self-Awareness, Plants Remember—Are Humans Really So Special?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Explore our legacy of discovery and look ahead to the future. It was the telegram exchange that sparked an identity crisis for humankind. In 1960 a young Jane Goodall working in a remote forest in Tanzania observed a chimpanzee she named David Greybeard using blades of grass and twigs to fish nutritious termites out of their nest. For decades—centuries, even—scholars have attempted to draw a hard line between our kind and the other organisms with whom we share the planet. They have argued that only humans have culture—sets of learned behaviors, such as toolmaking, that are passed down from generation to generation. They have proposed that only humans think symbolically, using signs to represent objects or ideas. For his part, Charles Darwin, writing in the late 1800s, opined that nonhuman animals have the same cognitive abilities and emotions that humans have and that any differences were a matter of degree and not kind. In the absence of any way to reliably read animal minds, however, scientists who studied animal behavior and cognition took the position that ascribing human thoughts, feelings and motivations to animals—anthropomorphism—was a cardinal sin. The findings have spurred fresh thinking about what, exactly, distinguishes Homo sapiens, with our vaunted intellect, from every other species on Earth. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Let's look first at our evolutionary nearest and dearest. We H. sapiens possess much larger brains than our closest living relatives, the chimps and bonobos, do—around three times as large. Naturally, anthropologists have wondered why we evolved such energetically expensive brains. At the same time, we know that H. sapiens is the sole surviving member of what was once a diverse group of humanoids. Surely our big brains and all the clever things they allow us to do were a major reason for our success as a species, a vital factor in why we alone went on to spread across the globe and thrive in every ecosystem we set our sights on, outcompeting other branches of humanity until we were the last hominin standing. Yet virtually every trait that anthropologists have identified as one that might have set our kind apart has subsequently been found in another member of the family. Our closest evolutionary cousins, the Neandertals, left behind decorations that suggest they used symbols, which may indicate a capacity for language. The same goes for our smaller-brained relative Homo erectus. And some 3.3 million years ago, long before brain size began to expand in our lineage, an unknown hominin—possibly Australopithecus afarensis—shaped basalt cobbles into cutting tools, demonstrating an understanding of the material properties of stone and a vision for how to transform a lump of rock into a useful implement. Humans were long thought to be the only moral animals, uniquely equipped with a sense of right and wrong. But we now know that is not the case. The monkeys' rejection of unequal payment for equivalent work demonstrated that they have a sense of fairness and experience moral outrage when they get a raw deal. Mice, for instance, can share the emotional state of another individual, exhibiting increased sensitivity to pain if they see a companion showing signs of pain. Dogs recognize distress in their owners and will offer consolation. Rats will sacrifice their own gains to alleviate the suffering of a conspecific, forgoing a food reward if taking the food means inflicting pain on another rat. Empathy and other complex emotions were long considered beyond the experience of nonhuman creatures. But mounting evidence indicates that they are widespread among mammals. Some of the most striking examples involve emotional responses to death. In 2018 an orca known as Tahlequah made headlines around the world when she carried her dead calf with her for 17 days while she swam 1,000 miles across the Salish Sea. This time she held on to its corpse for at least 11 days before releasing it. Researchers characterized the mother orca's reaction to these losses as grief. Apes, monkeys and elephants have been observed to mourn the loss of bonded individuals, too. It's not just large-brained mammals that appear to express sorrow, however. Our fellow mammals are not the only animals to show signs of thinking and feeling as humans do. Fish feel pain, another conscious experience: when given an injection that causes discomfort, lab zebra fish will vacate their preferred habitat, which has been decorated with pleasing rocks and vegetation, to visit a barren habitat whose water is infused with a pain reliever. In 2024 dozens of scientists signed a declaration acknowledging that species quite different from humans may have experiences of consciousness and that this possibility needs to be considered in decisions that affect these animals. The document could help shape policies governing animal research ethics and welfare. In recent years researchers have increasingly begun to explore the idea that plants—traditionally viewed as noncognitive beings—can learn, remember, make decisions, communicate and experience the world uniquely. In this way, some investigators propose, they are conscious. Other plants sense when they are being munched by hungry insects and emit chemical signals that summon predators of their attackers. We're not separate from nature, we're connected to it, part of the weave of life, in all its dazzling diversity. Kate Wong is an award-winning science writer and senior editor at Scientific American focused on evolution, ecology, anthropology, archaeology, paleontology and animal behavior. Her reporting has taken her to caves in France and Croatia that Neandertals once called home, to the shores of Kenya's Lake Turkana in search of the oldest stone tools in the world, to Madagascar on an expedition to unearth ancient mammals and dinosaurs, to the icy waters of Antarctica, where humpback whales feast on krill, and on a "Big Day" race around the state of Connecticut to find as many bird species as possible in 24 hours. Kate is co-author, with Donald Johanson, of Lucy's Legacy: The Quest for Human Origins.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/why-some-smells-are-pleasant-and-others-are-off-putting-according-to-science/'>Why Some Smells Are Pleasant and Others Are Off-Putting, according to Science</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The sense of smell, it turns out, is highly individual—a scent that is enticingly floral to one person may be off-putting and chemical to another. “It depends on what people have associated with the odor,” says psychologist Antonie Louise Bierling of Friedrich Schiller University Jena, who led the study. “In my opinion, it would have been really astonishing if the same odor smelled pleasant to everyone. More familiar scents seem to smell more pleasant, for example, and odors from complex molecules seem to have greater appeal than those of simple compounds. Scientists hope to use the data to help develop “digital olfaction”—artificial noses that can sense scents as well as or even better than humans. Such devices could be helpful for disease detection or in “smart home” applications: a toaster, for instance, that can tell when toast is done by how burned it smells or refrigerators that can identify putrid decay. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Miriam Quick and Jen Christiansen; Source: “A Dataset of Laymen Olfactory Perception for 74 Mono-Molecular Odors,” by Antonie Louise Bierling et al., in Scientific Data, Vol. Researchers asked people to respond with “yes” or “no” to whether each smell matched 16 descriptive terms, including “sweet” and “musky.” By comparing these answers with the sliding-scale pleasantness scores the same molecules received, we can see which words were most predictive of pleasant—and unpleasant—smells. For example, the median pleasantness score for all molecules that the respondents considered “sweet” was 61, and the median pleasantness score for all molecules that were not marked as “sweet” was 34. Miriam Quick and Jen Christiansen; Source: “A Dataset of Laymen Olfactory Perception for 74 Mono-Molecular Odors,” by Antonie Louise Bierling et al., in Scientific Data, Vol. We grouped molecular complexity into three categories: high, medium and low. Vertical bars mark median pleasantness scores per category. Miriam Quick and Jen Christiansen; Source: “A Dataset of Laymen Olfactory Perception for 74 Mono-Molecular Odors,” by Antonie Louise Bierling et al., in Scientific Data, Vol. Clara Moskowitz is a senior editor at Scientific American, where she covers astronomy, space, physics and mathematics. Moskowitz has reported live from rocket launches, space shuttle liftoffs and landings, suborbital spaceflight training, mountaintop observatories, and more. She has a bachelor's degree in astronomy and physics from Wesleyan University and a graduate degree in science communication from the University of California, Santa Cruz. Miriam Quick is a data journalist and researcher specializing in information visualization. Jen Christiansen is author of the book Building Science Graphics: An Illustrated Guide to Communicating Science through Diagrams and Visualizations (CRC Press) and senior graphics editor at Scientific American, where she art directs and produces illustrated explanatory diagrams and data visualizations. Subsequently she moved to Washington, D.C., to join the staff of National Geographic (first as an assistant art director–researcher hybrid and then as a designer), spent four years as a freelance science communicator and returned to Scientific American in 2007. Christiansen presents and writes on topics ranging from reconciling her love for art and science to her quest to learn more about the pulsar chart on the cover of Joy Division's album Unknown Pleasures. in geology and studio art from Smith College. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-scientists-finally-learned-that-nerves-regrow-even-in-the-adult-brain/'>How Scientists Finally Learned That Nerves Regrow—Even in the Adult Brain</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Long dismissed as impossible, nerve regrowth is offering new hope for healing injuries and treating neurodegeneration Explore our legacy of discovery and look ahead to the future. Billions of nerve cells send signals coursing through our bodies, serving as conduits through which the brain performs its essential functions. For millennia physicians thought damage to nerves was irreversible. In ancient Greece, founders of modern medicine such as Hippocrates and Galen refused to operate on damaged nerves for fear of causing pain, convulsions or even death. In recent decades this knowledge has inspired promising treatments for nerve injuries and has led researchers to investigate interventions for neurodegenerative disease. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. It was only in the late 1800s, however, that scientists began to understand how, exactly, nerves regenerate. Through his experiments on frogs, British physiologist Augustus Waller described in detail what happens to a peripheral nerve after injury. Still, there remained fierce debate about whether stitching nerves together would harm more than help. It was against the backdrop of bloody world wars of the 20th century that physicians finally made significant advances in techniques to restore damaged neurons. Over time physicians learned that some peripheral nerve injuries are more conducive to repair than others. Crushed nerves are likelier than cut ones to be repaired, and injuries that occur closer to a nerve's target tissue have a greater chance of regaining function than those that occur farther away. In cases where, for example, a nerve is damaged far from its target muscle, existing techniques may not be sufficient to enable regrowth across the long distances involved within a time frame allowing for recovery. This detour provides an alternative pathway to regain function. Susan Mackinnon, a plastic and reconstructive surgeon at Washington University in St. Louis, has largely driven the advances in nerve transfer, enabling patients to use their limbs after peripheral nerve injuries that previously would have led to a permanent loss of movement in them. For instance, Oskar Hanson, a high school baseball player, lost sensation and movement in most of his left arm after a surgery to mend a ligament injury ended up damaging the ulnar nerve in that arm. “There was zero hope that he would be able to have use of his arm again,” says his mother, Patricia Hanson. But after Mackinnon performed a nerve transfer procedure, most of the function returned. “She saved his life with that surgery,” Hanson says. A pivotal moment came in the early 1980s, when Canadian neuroscientist Albert Aguayo and his colleagues demonstrated that in rats, neurons of the spinal cord and brain stem could regrow when segments of peripheral nerves were grafted into the site of injury. In succeeding years, neuroscientists worked to uncover what, exactly, that environment looked like. To do so, they searched for differences in the peripheral and central nervous systems that could explain why the former was better able to repair damaged neurons. For example, only injuries within the central nervous system led to the formation of glial scars—masses of nonneuronal cells known as glial cells. The purpose of these scars is still debated, however. At the time, these findings were met with skepticism, says Rusty Gage, a professor of genetics at the Salk Institute for Biological Studies in La Jolla, Calif. “It was just too hard to believe.” Inspired by these findings, researchers looked for signs of adult neurogenesis in other animals. In 1998 Gage and his colleagues revealed evidence of this process occurring in the brains of adult humans—specifically within the hippocampus, a region linked with learning and memory. Although support for adult neurogenesis in humans has amassed over the years, some experts still debate its existence. In 2018 a team co-led by Arturo Alvarez-Buylla, a neuroscientist at the University of California, San Francisco, who had worked with Nottebohm on songbirds, published a study stating that the formation of new neurons was extremely rare, and likely nonexistent, in adult human brains. Still, there's a growing consensus that neurogenesis does happen later in life—and that this growth appears to be largely limited to certain parts of the brain, such as the hippocampus. Researchers are now trying to understand the purpose of these budding nerve cells and asking whether they might offer clues for treating neurodegenerative disorders such as Alzheimer's disease. Some scientists are even exploring whether, by targeting neurogenesis, they can improve the symptoms of psychiatric conditions such as post-traumatic stress disorder. Understanding that a neuron can regrow and be repaired and identifying details of that process has been a great achievement, says Massimo Hilliard, a cellular and molecular neurobiologist at the University of Queensland in Australia. Diana Kwon is a freelance journalist who covers health and the life sciences.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03890-6'>A plasma proteomics-based candidate biomarker panel predictive of amyotrophic lateral sclerosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 09:55:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Here, in this cross-sectional study, we used the Olink Explore 3072 platform to investigate plasma proteomics as a biomarker tool for this neurodegenerative condition. Thirty-three proteins were differentially abundant in the plasma of patients with ALS (n = 183) versus controls (n = 309). We replicated our findings in an independent cohort (n = 48 patients with ALS and n = 75 controls). We then applied machine learning to create a model that diagnosed ALS with high accuracy (area under the curve, 98.3%). By analyzing plasma samples from individuals before ALS symptoms emerged, we estimated the age of clinical onset and showed that the disease process—impacting skeletal muscle, nerves and energy metabolism—occurs years before symptoms appear. Our research suggests that plasma proteins can be a biomarker for this fatal disease and offers molecular insights into its prodromal phase. Amyotrophic lateral sclerosis (ALS) is a progressive neurological disorder characterized by the degeneration of motor neurons. This degeneration leads to muscle weakness, atrophy and, ultimately, respiratory failure, typically resulting in death within 2–4 years after symptoms begin1. By 2040, the global population of individuals living with ALS worldwide is projected to approach 400,000 (ref. The substantial number of cases and the high cost associated with their care pose a substantial burden on communities and healthcare systems2. In the United States, the only approved treatments for sporadic ALS are riluzole and edaravone. There is an urgent need for better medications; however, the necessity to recruit large numbers of patients with ALS into clinical trials to assess drug effectiveness hinders therapeutic development3. Currently, diagnosing ALS relies on clinical symptoms and neurological evaluations, much like the methods used 155 years ago when Jean-Martin Charcot first identified the disease4. Although neurophysiological studies and genetic tests are supportive tools for clinical decision-making, no definitive diagnostic test exists5. Patients frequently experience delays of 6–18 months until their symptoms become severe enough to establish a diagnosis6. Even when there is clear evidence of motor neuron dysfunction, differentiating ALS from other neurological conditions can be difficult because the disease presents with heterogeneous clinical manifestations7. This uncertainty can be unsettling for patients and their families. Moreover, it causes delays in entering clinical trials and limits the number of eligible participants due to enrollment restrictions for patients with more severe disability8. In the future, when effective treatments become available for ALS, diagnostic delays may impede treatment initiation, harming patient outcomes. We addressed this critical knowledge gap by generating and analyzing data for 3,072 plasma proteins within a large ALS case–control cohort. These well-characterized patients had genetic data readily available, allowing us to incorporate these factors into our search for differentially abundant proteins and to use a multiomic approach to determine if the observed plasma protein changes occurred because of the disease process or were primary drivers in the pathogenesis of ALS. Using machine learning, we identified a distinct molecular signature in plasma that effectively differentiates ALS cases from healthy individuals and other neurological disorders. Additionally, we explored whether our model could predict when symptoms would start in asymptomatic individuals who later developed ALS. Notably, we established a public resource for the scientific community to encourage and advance biomarker research in this fatal neurodegenerative disease. After quality control, we obtained data for 2,886 proteins from 231 individuals diagnosed with ALS and 384 control individuals (n = 214 healthy individuals and n = 170 with other neurological conditions). Patients with other neurological disorders were included as control data, as our central hypothesis was to use proteomics to distinguish ALS cases both from healthy individuals and from other relevant conditions. Participants were recruited from an Italian population-based ALS registry and a US natural history study, and they were diagnosed with ALS based on established consensus criteria9,10. Extended Data Table 1 lists the demographics and clinical features of the cohorts used in this study. The proteomic data, measured using the Olink Explore 3072 assay in plasma samples, exhibited median intra-assay and inter-assay coefficients of variation of 9.9% and 22.3%, respectively, confirming the platform's reliability11. Figure 1 gives an overview of the study and analysis. We conducted a cross-sectional study to identify ALS biomarkers via proteomic analysis of plasma samples. We performed differential abundance analysis on proteomic data generated from plasma samples of patients with ALS, healthy controls and patients with other neurological diseases in both the Discovery Cohort and the Replication Cohort. After this, we applied supervised machine learning using plasma protein levels and clinical parameters to identify a molecular signature of ALS. For External Validation Set 2, we obtained proteomic data from the UK Biobank18. Additionally, a web tool was created for clinical researchers to analyze their own data. We conducted proteome-wide association testing of individual plasma protein levels using generalized linear regression. Our analyses were adjusted for the following confounding factors: age at sample collection, self-reported sex and the collection tube type (that is, EDTA or heparin). Furthermore, we incorporated the first two uniform manifold approximation and projection (UMAP) components derived from genetic data of the study participants to manage population stratification. In total, 33 plasma proteins exceeded the significance threshold in the Discovery Cohort (adjusted false discovery rate (FDR) P < 0.05; Table 1 and Fig. The protein with the highest differential protein abundance was neurofilament light chain (NEFL, log2 fold change = 2.34, adjusted FDR P = 2.22 × 10−88), which is known to be elevated in ALS and various neurological diseases12. Notably, LIF encodes leukemia inhibitory factor, a neurotrophic cytokine that may promote motor neuron survival in vivo and in vitro13. The remaining 31 proteins have not been definitively linked to ALS previously. a, Volcano plot showing the differential abundance of proteins in the Discovery Cohort (n = 183 ALS cases versus n = 172 healthy controls plus n = 137 other neurological diseases). Blue and red dots denote statistically significant downregulated or upregulated proteins determined by generalized linear regression (adjusted to 5% FDR for multiple comparisons). b, Volcano plot showing the differential abundance of proteins in the Replication Cohort (n = 48 ALS cases versus n = 42 healthy controls and n = 33 other neurological diseases). The error band is shown as a gray band representing the 95% confidence interval for the mean prediction (blue linear regression line) at each x value. Pearson correlation coefficients (R) were calculated to assess the linear association between variables. P values were computed using a two-sided test of the null hypothesis that the correlation coefficient equals 0. d, A comparison of the proteins significantly associated with ALS in plasma and their abundances in CSF. The CSF protein levels were derived from SomaScan data generated for n = 14 ALS cases and n = 89 healthy controls. We verified the differential abundance of the proteins (n = 14 of 33) identified in the Discovery Cohort within the independent Replication Cohort, which similarly included ALS cases, healthy controls and individuals with other neurological diseases (Table 1 and Fig. Additional validation was performed using sensitivity analyses (Extended Data Table 2), comparisons with quantitative ELISAs (Extended Data Fig. We performed a comparative analysis to assess plasma protein levels in patients with ALS with C9orf72 expansions (n = 29) against those lacking these expansions (n = 202). This exploratory analysis identified eight significantly upregulated proteins in ALS C9orf72 carriers compared with non-carriers (Extended Data Fig. We also compared ALS C9orf72 carriers (n = 29) with non-carrier patients with ALS (n = 202) plus a control group (n = 383, which included 169 neurological control non-carriers and 214 healthy control non-carriers). The same eight proteins were elevated among ALS C9orf72 carriers compared with controls, suggesting that these protein changes are associated with the presence of the C9orf72 repeat expansion (Extended Data Fig. We performed additional exploratory analysis to assess whether asymptomatic C9orf72 carriers (n = 12) displayed a different proteomic profile compared with symptomatic C9orf72 carriers (Extended Data Fig. Our results indicated that specific proteins (including EIF2S2, HPCAL1, JPT2, MTIF3, PDAP1 and SMAD3), which are elevated in symptomatic carriers, did not demonstrate significant changes in asymptomatic carriers and may, therefore, have potential as biomarkers for phenoconversion. We investigated whether the proteins with differential abundance in plasma were similarly impacted in the cerebrospinal fluid (CSF). For this analysis, we extracted individual protein measurements from SomaScan data generated using CSF from 14 ALS cases and 89 healthy controls for this analysis. Variations were modeled using a generalized linear regression, adjusting for age and sex. Notably, five proteins—HSPB6, MEGF10, NEFL, RNASE3 and TPM3—demonstrated significant increases in the CSF of patients with ALS (Fig. The other proteins did not show significant differences in the CSF of patients with ALS (Table 1). An enrichment analysis of the 33 plasma proteins linked to ALS uncovered significant associations with multiple biological processes (Fig. Most of the identified pathways showed a strong connection with skeletal muscle and neuronal function (Fig. These results highlight the importance of cellular pathways involved in ‘skeletal muscle development and degeneration', ‘energy metabolism' and ‘NMDA receptor-mediated excitotoxicity', corroborating earlier research in ALS14,15,16. a, Functional enrichment analysis of ALS based on the differentially abundant plasma proteins (n = 33) performed using the ‘clusterProfiler' software package. A one-sided t-test was used to test for enrichment of genes in a particular pathway, followed by FDR correction to account for multiple testing comparisons. Significant Gene Ontology (GO) enrichments for biological processes (BP, blue), molecular functions (MF, orange) and cellular functions (CC, green) as well as pathways from Reactome (magenta) and Kyoto Encyclopedia of Genes and Genomes (KEGG) (light blue) are shown. The proteins involved in each category are listed. To do this, we examined whether genetic factors affected the levels of the differentially abundant proteins. First, we searched for evidence of genetic association in the autosomal genes (n = 31) related to the proteins that we identified in ALS, using data from a large ALS genome-wide association study (GWAS) (n = 27,205 cases and n = 110,881 controls)17. Next, we implemented two-sample Mendelian randomization using merged summary-level ALS GWAS data17 alongside cis-protein quantitative trait loci (pQTL) data for the 33 plasma proteins with differential abundance in ALS18. This approach helps identify proteins whose levels correlate with ALS through shared causal variants. Extended Data Table 4 shows that none of the analyses reached statistical significance, reaffirming that the differential protein abundance levels seen in patients with ALS are not directly driven by inherited genetic variation encoding the levels of these proteins. We performed supervised machine learning, hypothesizing that this method would identify a molecular signature of ALS when applied to a proteomic dataset from a well-characterized cohort. Our main goal was to create a binary classification model to distinguish patients with ALS from control individuals. The initial features used to build the machine learning model included the differentially abundant proteins (n = 33) and clinical parameters, including age at sample collection, sex and blood collection tube type (total number of features = 36). Ten supervised machine learning algorithms were evaluated (listed in Extended Data Table 5). After secondary feature selection, 20 features (17 proteins, sex, age at sample collection and plasma collection tube) were identified as the most predictive for discriminating ALS cases from healthy individuals and patients with other neurological diseases (Fig. 4a); the area under the curve (AUC) and balanced accuracy of this model were 96.2% and 89.3%, respectively, when evaluated in the Testing Set (n = 48 ALS cases versus n = 42 healthy controls plus n = 33 other neurological diseases) (Fig. a, The mean importance score of the 20 features (17 proteins plus sex, age at collection and blood collection tube type) making up the random forest model. of the mean feature importance estimates across 100 repeated permutations. The white area on the right denotes ALS risk scores consistent with a diagnosis of ALS. In contrast, the gray area on the left, which was manually added to the plot, delineates scores consistent with healthy control status or other neurological diseases. When NEFL was removed as a feature, the area under the receiver operating characteristic curve (AUC-ROC) declined by 2.4% to 13.2% across the Testing (89.8%), External Validation Set 1 (96.2%) and External Validation Set 2 (85.9%) datasets, indicating that the model's performance is not solely dependent on NEFL. Instead, it relies on the contributions of the other 19 features, which include 16 proteins as well as factors such as sex, age at sample collection and the type of plasma collection tube, to accurately classify samples as either ALS or control. We validated our model using two external sample sets that were not part of the training or testing process (Fig. External Validation Set 1 consisted of 46 samples assayed at the National Institutes of Health (NIH), which were withheld from the primary analysis due to inadequate genetic data. External Validation Set 2 comprised Olink Explore 3072 data from the UK Biobank, including 13 ALS cases and 23,601 controls. Notably, the proteomic data for External Validation Set 1 were normalized with the same bridging samples used in the Testing Set and Training Set. In contrast, the data from External Validation Set 2 (UK Biobank) did not undergo this normalization process. Despite the absence of normalization or bridging samples, External Validation Set 2 (UK Biobank) still achieved high AUC and balanced accuracy values of 99.1% and 88.1%, respectively (Fig. Overall, the supervised machine learning algorithm diagnosed 62 patients (82.7%) out of 75 ALS and correctly identified 23,548 healthy individuals (99.3%) of the 23,707 in the Testing Set, External Validation Set 1 and External Validation Set 2 (Fig. We then used the Shapley additive explanations (SHAP) approach to illustrate each feature's contribution to the predictions made for each sample (Extended Data Fig. Finally, we calculated an ALS risk score for every sample from the aggregated SHAP values, scaled between 0 and 100 using the minimum and maximum aggregated SHAP values from the Training Set samples. We conducted additional analyses of the model's predictive performance for a group of individuals (n = 355) diagnosed with neuropathy (International Classification of Diseases, 10th revision (ICD-10) codes G60–G64) or myopathy (ICD-10 codes G70–G73) using data from the UK Biobank. The random forest model successfully identified 333 (93.8%) individuals as non-ALS cases, demonstrating its capability to differentiate ALS from similar conditions. We identified individuals (n = 110) who had plasma samples taken before the onset of their ALS symptoms (n = 109 samples from the UK Biobank and n = 1 sample from our cohort; mean duration from plasma sampling to symptom onset = 6.4 years, range: 2.0–14.0 years). For those individuals whose genetic data were available, 12 were confirmed to carry the C9orf72 repeat expansion. We used these data points to examine how our machine learning model behaved prior to symptom onset. A significant association was found between the ALS risk score and the time to symptom onset, with the ALS risk score increasing progressively as patients approached symptom onset (based on n = 110 asymptomatic samples and n = 251 ALS cases that were within 5 years of symptom onset; slope coefficient = 4.48, slope P = 1.40 × 10−49, R2 of the regression model = 0.74 and root mean square error (RMSE) = 14.39; Fig. Notably, this increase in ALS risk score was not due to aging, as individuals with other neurological diseases, healthy individuals or even patients with ALS after they had received their diagnosis did not display the same trend (Extended Data Fig. a, Scatter plot comparing the ALS risk score based on the random forest model and the years before symptom onset in patients with ALS. The purple dots represent plasma samples taken from presymptomatic individuals (n = 109 from the UK Biobank and n = 1 from this study) who subsequently developed ALS. The orange dots denote plasma samples taken from individuals with ALS cases in the Training Set, the Testing Set and External Validation Sets 1 and 2 (n = 251 ALS cases whose samples were collected within 5 years of symptom onset). The statistical significance of each coefficient was assessed using a two-sided hypothesis test with a null hypothesis that the coefficient equals 0. The vertical dashed line indicates the rough boundary between presymptomatic individuals and those diagnosed with ALS. The data sourced from the UK Biobank are represented in whole years, explaining why some cases seem to exhibit symptoms prior to 0 years. Data are presented as estimated coefficient values from linear regression ± s.e., and the P values derived from two-sided hypothesis testing are displayed. We investigated which proteins may be altered during the prodromal period. After adjusting for collection age, sex and plasma source, we found that 10 proteins making up the model were individually linked to the time until symptom onset when analyzing asymptomatic individuals and ALS cases together (n = 361) (Fig. In the analysis concentrating exclusively on patients with ALS (n = 251) or asymptomatic individuals (n = 110), only NEFL demonstrated a meaningful correlation with the time to symptom onset (Extended Data Fig. Interestingly, the slope coefficients revealed opposing trends: NEFL levels sharply increased in asymptomatic individuals as the time to onset approached zero, whereas, in patients with ALS, NEFL levels significantly decreased as the time since onset lengthened. Clinicians have long sought a reliable method to diagnose ALS. Such knowledge could substantially improve patient care, assist in clinical trials and offer insights into the underlying pathogenic mechanisms21,22. In this study, we analyzed proteomic data from a well-characterized cohort to identify a unique molecular signature in the plasma of patients with ALS that could serve as a biomarker. We identified a panel of 33 differentially abundant proteins in ALS and validated our findings across multiple independent datasets, showing that the same protein profile can differentiate ALS from other neurological diseases, including clinically relevant differential diagnoses such as myopathies and neuropathies. We used multiomic data (that is, genetic data) from this well-characterized cohort to determine that these changes are likely secondary to the disease process. Additionally, we demonstrated that the ALS risk score computed from our machine learning model could be used as a proxy for forecasting the age at which symptoms may appear in individuals. Our data also suggest that the underlying disease process happens considerably earlier than was previously recognized. Pathway analysis showed that the relevant proteins are linked to cellular processes known to be involved in ALS23, confirming the effectiveness of our data-driven approach and supporting the reliability of our findings. The key challenges in developing proteomic biomarkers for ALS have been the need for a sufficiently large dataset and the difficulty in analyzing multidimensional relationships. To address these issues, we used plasma samples from a population-based ALS registry that had enrolled patients over an extended period within a defined geographical area9,24. This registry collected data throughout the patientsʼ illness, providing detailed clinical information that informed our categorization efforts19. The availability of multiomic data, particularly genetic data that allow for a more careful delineation of the protein signals, adds to the strength of these cohorts for research. To date, the development of biomarkers for ALS has primarily relied on the study of single analytes, such as neurofilaments25. Although univariate biomarker systems hold clinical value, it is increasingly evident that they fail to capture the complexity of neurodegenerative diseases, raising concerns about their accuracy and sensitivity25,26. In contrast, the machine learning algorithms that we applied were adept at deciphering multifaceted relationships within data19,27. We generated data for almost 3,000 proteins across multiple samples to facilitate these analyses28. The accuracy of Olink has been validated through comparisons with proteomic measurements based on ELISA and mass spectrometry29,30,31,32,33,34, establishing it as an emerging standard in the field35. 4)—attributable to multiple pathways—confirms the multiplex nature of ALS and the appropriateness of our machine learning approach for multimodal biomarker development. We opted for plasma in our proteomic analysis due to its greater accessibility compared with CSF. Crucially, a biomarker derived from blood can facilitate early ALS evaluations, even by non-specialists36. Such testing may streamline referral to multidisciplinary ALS clinics, leading to earlier intervention and improved patient outcomes37. Additionally, biomarkers obtained through venipuncture simplify the enrollment process for clinical trials, thereby reducing costs and participant burden38. Examining the proteomic profile of CSF collected by lumbar puncture may incrementally improve diagnostic accuracy, as these biospecimens are in direct contact with the extracellular space of the brain and spinal cord, reflecting biochemical changes in the central nervous system39. Nevertheless, in neuromuscular diseases, plasma-based biomarkers may be better at measuring changes in the neuromuscular compartment than CSF, which is corroborated by our pathway analysis showing changes in the peripheral tissues. The biomarker field is evolving40; one possible outcome of this research is that plasma-based biomarkers could act as a screening tool for ALS, and CSF assays could provide subsequent confirmation. We identified a unique set of plasma proteins that distinguishes ALS from healthy individuals and other neurological diseases. The NEFL protein, a proven ALS marker, is part of this panel, supporting our data-driven hypothesis-free approach25,41. However, the real breakthrough is the discovery of 16 additional proteins through predictive modeling, which substantially enhances ALS classification accuracy. The selection of these proteins, which have not been previously linked to ALS, highlights how proteomics and machine learning can provide fresh insights into complex diseases. For instance, our enrichment analysis identified links between ALS and various biological pathways, particularly those involving skeletal muscle, neurons and energy metabolism. Several conclusions can be drawn from these findings. First, our research demonstrates that it is possible to detect the earliest stages of the disease at a molecular level. Our protein panel could act as a biomarker for this prodromal phase of ALS, similar to the biomarkers currently used in Alzheimer's disease42. This capability will greatly improve clinical trials for presymptomatic patients with ALS, which typically rely on natural history studies to determine the expected rate of symptom conversion. Although there may be some noise in these measurements, longitudinal biomarker monitoring may detect an increase in their ALS risk score, indicating a molecular disease onset and predicting phenoconversion. Second, the processes associated with the disease happen much earlier than previously thought; NEFL has been shown to increase 12 months before symptoms appear43. However, we discovered that the ALS risk score, determined by plasma protein levels, begins to shift in presymptomatic individuals up to 10 years before symptoms emerge, hitting the 50th percentile around 5 years prior to onset. This change in viewpoint again underscores the power of data-driven research to reveal unexpected insights into the pathophysiology of complex diseases. Our comprehensive genetic analyses revealed no evidence that genetic factors directly influence changes in plasma proteins. This suggests that the observed alterations in proteins are likely secondary effects or may originate from indirect mechanisms such as posttranscriptional or posttranslational regulation. Indeed, many of the observed changes, such as muscle regeneration and catabolic changes in energy metabolism, suggest a compensatory phase that occurs in patients with ALS long before symptom onset. However, we cannot rule out that primary muscle pathophysiology could still play an undiscovered role in the development of ALS15. Our primary analysis focused on general ALS; however, we also uncovered preliminary evidence of distinct plasma proteomic profiles among ALS subpopulations. Notably, we identified eight elevated proteins in patients with a pathogenic C9orf72 repeat expansion (Extended Data Table 3). None of these proteins was altered in the general ALS cohort, indicating that their changes may be specific to C9orf72 carriers. These preliminary findings need confirmation in a larger sample cohort, ideally accompanied by longitudinal data, to explore how these proteins might relate to phenoconversion in C9orf72 carriers. The proteomic platform that we used quantifies approximately 3,000 proteins, which represents a substantial number of the total number of proteins in human plasma44,45. The unmeasured portion likely includes tissue proteins acting as markers of leakage and immunoglobulins that mirror the host's immune response45, some of which may be connected to ALS. Additionally, the Olink assay cannot detect new proteins, limiting its effectiveness in investigating the role of novel proteins generated by cryptic splicing in ALS pathogenesis46. Future studies will investigate these broader proteomic approaches and incorporate longitudinal data collection to further validate and enhance the performance of our model. The CSF analysis was conducted on a limited number of samples (n = 14 ALS cases and n = 89 healthy controls), which might not entirely reflect the range of protein abundance variations. However, we showed that specific proteins identified in plasma also exhibited variations in the CSF of patients with ALS. We are in the process of increasing our collection of CSF samples to facilitate further research on these and other potential proteins. Despite the absence of shared bridging controls between our study and the UK Biobank's Olink data, our plasma protein biomarker panel achieved over 99% accuracy in predicting the ALS or control status of 23,614 European individuals in the UK Biobank cohort. It was also able to distinguish patients with diseases such as neuropathies and myopathies that can mimic ALS. Nonetheless, a larger case–control cohort with greater ancestral diversity is necessary for additional validation to assess the global applicability of our predictive model. In conclusion, we discovered a unique molecular profile through proteomic analysis of plasma samples from patients with ALS. This protein signature serves as a crucial factor for diagnosing ALS and distinguishes it from other neurological disorders with high specificity and sensitivity (AUC > 98%). Furthermore, the same proteomic approach enables us to predict symptom conversion in individuals who have not yet exhibited ALS symptoms. Our data-driven strategy, powered by machine learning, shows how proteomics can enhance ALS diagnosis and reshape understanding of this complex disease, suggesting that the prodromal phase happens much earlier than previously recognized. This study was a cross-sectional analysis of plasma proteomic data to identify a biomarker panel for ALS. Recruitment sites were located at the University of Turin in Turin, Italy; the NIH in Bethesda and Baltimore, Maryland; and Johns Hopkins University in Baltimore, Maryland. Written consent was obtained from all individuals enrolled in this study. The Italian samples consisted of neurologically healthy individuals (n = 196) and patients diagnosed with ALS (n = 236) living in Northern Italy and recruited in a population-based study known as the Piedmont and Valle d'Aosta Registry for ALS (PARALS; established 1 January 1995)9. The US plasma samples comprised patients diagnosed with ALS (n = 45) evaluated at the NIH Clinical Center in Bethesda, Maryland, as part of a natural history study (NCT03225144)47 and control samples collected as part of the NIH Baltimore Longitudinal Study of Aging (BLSA) (n = 53, NCT00233272)48 and at the Johns Hopkins Hospital (n = 9). The control samples for the CSF analyses (n = 89) were collected under an NIAID natural history protocol (NCT00794352). The patients with ALS were diagnosed according to the revised El Escorial criteria10 by a neurologist specializing in ALS. Patients with familial and sporadic ALS and self-declared diverse ancestry (n = 3 Hispanic samples) were included in the study, ensuring a comprehensive representation of the ALS population. The control individuals were selected based on their lack of a diagnosis of ALS, neurological disease and cognitive decline in their clinical history. The control cohorts were matched to the case cohorts for race and ethnicity but not for sex or age, although the age distribution was similar (Extended Data Table 1). We also assembled a cohort of individuals with other neurological conditions (n = 194, listed in Extended Data Table 1) as an additional comparison cohort. Plasma samples were collected from these participants at the NIH Clinical Center (n = 172) and Johns Hopkins University (n = 22). The other neurological disease cohort included patients diagnosed with the following conditions: corticobasal syndrome (n = 8 patients) was diagnosed according to the Armstrong criteria49; patients with Lewy body dementia (n = 8) were diagnosed with clinically probable disease according to consensus criteria50,51; the multiple system atrophy (n = 5) cases were diagnosed according to the Gilman criteria52; and Parkinson's disease (n = 153) and progressive supranuclear palsy (n = 19) were diagnosed based on the 2015 and 2017 Movement Disorders Society criteria, respectively53,54. One patient was labeled as having dementia, not otherwise specified, after clinical evaluation. The Italian samples (n = 427) were mainly collected using heparin tubes, and the remaining samples (n = 306) were collected using EDTA tubes. Blood cells were removed by centrifugation within 2 hours of collection, and the supernatant, consisting of the plasma, was carefully removed without disturbing the cell pellet. All plasma samples were aliquoted and stored at −80 °C. The number of freeze–thaw cycles between aliquoting and running the proteomic assays was minimized, typically one but no more than three. A 22-gauge Sprotte spinal needle was used and inserted at the L3/L4 level. The CSF was collected in polypropylene tubes, immediately placed on ice and sent to the laboratory for processing within 30 minutes. All samples were gathered at a single location, the NIH Clinical Center in Bethesda, Maryland (NCT03109288). The Olink Explore 3072 assay quantifies 2,926 proteins (https://olink.com/products/olink-explore-3072-384) with high accuracy and specificity11. In brief, the platform is based on oligonucleotide-labeled antibody pairs that bind to their target protein in solution11. When both antibodies are close, their oligonucleotide tails hybridize and undergo extension by a DNA polymerase. This molecular process generates a unique, double-stranded DNA barcode, which is subsequently amplified and quantified using next-generation sequencing (Illumina NovaSeq). Proteins that did not meet the default standard Olink quality control criteria were excluded from the analysis. These steps were conducted using Olink software (version 1.0). Normalized Protein eXpression (NPX) values, representing the log2-transformed ratio of sample assay counts to extension control counts, were used as the final assay readout; a higher value corresponded to a higher protein expression or abundance levels11. Replicate samples (n = 3 per plate), referred to as bridging samples, were used to calculate both intra-plate and inter-plate coefficients of variation. These bridging samples are independent of the inter-plate sample controls. Samples with median NPX values (calculated from all proteins in each sample) that deviated by more than 3 s.d. from the mean of the median NPX values of all samples, as well as samples exceeding 3 s.d. Olink data from the UK Biobank were obtained through the UK Biobank Research Analysis Platform (https://www.dnanexus.com/partnerships/ukbiobank). ALS cases were defined as individuals diagnosed with ICD-10 code G12.2 and confirmed at death55 and had their blood collected up to 1 year before symptom onset. Control samples were defined as individuals without a recorded diagnosis of myopathy (ICD-10 codes G70–G73) or neuropathy (ICD-10 codes G60–G64). DNA obtained from the same participants also underwent whole-genome sequencing on a HiSeq X10 sequencer (Illumina; n = 476, 150-bp paired-end reads, 35× coverage)56 or genotyping on Infinium GDA-8+ NeuroBooster BeadChips (version 1.0, Illumina; n = 245), according to the manufacturer's instructions. Standard sample-level and variant-level quality control procedures were applied to the genetic datasets, and variants were extracted for the principal component analysis, performed using ‘flashPCA' (version 2.0). Principal components 1–10 were reduced to two dimensions using UMAP version 0.2.10. These two UMAP values were used to correct for population stratification in the statistical analysis of the individual protein analytes57. Although methods such as ADMIXTURE can provide percentage estimates of ancestry, the UMAP approach offers a direct and efficient way to control for confounding in our association analyses. Notably, reanalysis using all ten principal components yielded nearly identical results, confirming that the two-component UMAP adjustment sufficiently controlled for stratification in our study cohorts. Samples that passed Olink's quality control were divided into two cohorts at an 80:20 ratio, labeled as the Discovery/Training and Replication/Testing datasets. The terms ‘Discovery' and ‘Replication' refer to differential protein abundance analysis, whereas ‘Training' and ‘Testing' pertain to machine learning analysis. To accurately represent cohort characteristics in the datasets, samples were matched by diagnostic groups (ALS, healthy controls and neurological controls) and the type of tube used for collection, as implemented in the ‘caret' software package (version 6.0-94). We performed a cross-sectional association analysis between plasma protein abundances and the diagnosis of ALS. The following generalized linear model was used to evaluate the relationship between an individual protein's abundance and ALS status: NPXi = β1 × ALSi + β2 × agei + β3 × sexi + β4 × tubetypei + β5 × UMAP1i + β6 × UMAP2i, where NPXi represents the NPX value for individual i; ALSi is a binary indicator for ALS diagnosis (1 for ALS, 0 for control); agei is the age at sample collection; sexi is the reported sex of the individual; tubetypei indicates the plasma collection tube type (heparin or EDTA); and UMAP1i and UMAP2i are the first two components of the UMAP projection of genetic data. The absence of a constant term means that there is no intercept (β0) in the model. Multiple comparisons were controlled using the FDR procedure defined by Benjamini and Hochberg, and two-sided FDR-adjusted P values are reported in Table 1, Fig. The significance threshold was set at an FDR P value of 0.05. Our Discovery Cohort, comprising 183 cases and 309 controls, had 80% power to detect a differentially abundant protein with an effect size (Cohen's f statistic) of 0.242, assuming a significance threshold P value of 1.73 × 10−5. All analyses were conducted using R (version 4.3.2). We used z-scores to compare the effect sizes of differentially abundant proteins in the Discovery and Replication cohorts. We used z-scores in this comparison because they provide an accurate statistical representation of the difference in effect size between cases and controls by taking into account the s.e. Without standardizing with z-scores, the slope of the relationship between protein abundance and disease status could be artificially inflated or deflated, resulting in misleading correlation statistics. Quantification was performed on plasma samples (n = 16 ALS cases and n = 16 healthy controls) using commercially available colorimetric ELISA or fluorescence-based ProQuantum high-sensitivity immunoassay kits according to the manufacturer's protocol. Immunoassay kits were available for 30 of the 33 differentially abundant proteins (Extended Data Fig. The optimal dilution factor for measuring each protein was determined empirically. Each plasma sample and protein standards were assayed in duplicates. A standard curve for each protein was generated by plotting the absorbance readings at 450 nm wavelength or FAM fluorescence emission at 517 nm against the concentrations of the protein standards. A four-parameter logistic curve was fitted using GraphPad Prism software (version 10.4.1). The Bland–Altman plot was created by calculating the differences in measurements between the ELISA or ProQuantum and Olink assays (log2-transformed concentration from the ELISA or ProQuantum assay minus Olink NPX) and comparing this to the mean measurement from both assays for each paired sample. This standard data visualization technique is recommended for comparing methods59. For the subset of samples (n = 16 ALS and n = 16 healthy controls), differential protein abundance analysis was performed using a generalized linear regression model with the following formula: ProteinAbundancei = β1 × ALSi + β2 × agei + β3 × sexi + β4 × tubetypei. All analyses and visualizations were conducted using GraphPad Prism software (version 10.4.1). Proteomic measurements were previously generated for a subset of the BLSA control samples (n = 9) using SomaScan 7K, a single-stranded DNA aptamer-based proteomic platform60. The data were categorized using the k-means clusters identified by Katz et al.11 (Extended Data Fig. CSF proteomic measurements were generated from ALS cases (n = 14) and healthy control individuals (n = 89) using the SomaScan 7K platform. Individual proteins were assessed using generalized linear modeling, adjusted for age at collection and sex. The proteins that were differentially abundant in ALS were analyzed for pathway enrichment using ‘clusterProfiler' (version 4.10.1) using the following databases: (1) GO61, (2) KEGG62 and (3) Reactome63. Pathways with a single gene were discarded, and significance was defined as an FDR-adjusted P value of less than 0.05. Regional association plots of the genes encoding the differentially abundant proteins identified in ALS cases were made with ‘locusZoomR' (version 0.3.5) using data downloaded from a large ALS GWAS17. Single-nucleotide polymorphisms (SNPs) showing a P value less than or equal to 5.0 × 10−8 were regarded as evidence of genetic variants in the locus that may influence the risk of developing ALS. We performed Mendelian randomization analyses using ‘TwoSampleMR' (version 0.6.1)64. These plasma pQTL profiles were obtained from GWAS data based on 54,219 UK Biobank participants, available from the Synapse web portal18. Only genetic variants with P < 1.0 × 10−5 from the pQTL summary statistics were included in the analysis. As the outcome, we used an ALS GWAS, comprising 29,612 cases and 122,656 control individuals17. To assess causality, we applied the inverse‐variance weighted method and two Mendelian randomization sensitivity tests (MR Egger and weighted median). Traits were considered consistent with a causal effect when P values were less than or equal to 0.05. We performed supervised machine learning on the proteomic data to identify a molecular signature of ALS. To do this, we randomly selected 80% of the samples as the Training Set (n = 183 ALS cases versus n = 172 healthy controls plus n = 137 other neurological diseases). We used ‘caret' (short for ‘classification and regression training', version 6.0-94), an open-source machine learning package that streamlines predictive model creation. The decision to limit the feature set to 33 proteins was made to enhance the model's stability and avoid overfitting, often seen in high-dimensional data analysis65. Additionally, using a minimal feature set makes the model more relevant in clinical environments, where cost and assay complexity play important roles. For instance, even though UMAP components are useful for addressing population stratification in statistical analyses, they were left out of the machine learning model. Their inclusion would require all patients to undergo genotyping, substantially increasing implementation costs and computational complexity. Consequently, we concentrated on proteins that can be directly measured in plasma, ensuring that the model remains both practical and affordable. Ten supervised machine learning algorithms were evaluated (listed in Extended Data Table 5). Adaptive resampling with a minimum of ten resamples, and a significance threshold of α = 0.05 using generalized least squares was employed to refine the tuning parameters for each model. The best model (random forest) was selected based on overall performance (AUC and balanced accuracy) when classifying unseen samples in the Testing Set, External Validation Set 1 and External Validation Set 2. The random forest model was further tuned on the Training Set, and key predictive features were prioritized using the genetic algorithm feature selection method; fivefold cross-validation was conducted and repeated ten times, and data were preprocessed with centering and scaling to standardize features. The parameters used for finetuning and feature selection using the genetic algorithm method were as follows: nbits = 36, popSize = 500, maxiter = 25, pmutation = 0.1, pcrossover = 0.8 and elitism = 3. For model tuning in the Discovery Cohort, the standard random forest majority-vote threshold of 0.5 was used and then directly applied to the Replication Cohort. We conducted a repeated classification analysis, excluding NEFL, to assess the impact of this protein on the model's performance. The remaining features (that is, 16 proteins, demographic factors (sex and age at sample collection) and plasma collection tube type) were incorporated into this model. To determine the ALS risk score, we summed the SHAP values for all features (n = 20, including 17 proteins, sex, age at sample collection and type of tube used for collection) for each sample. The summed SHAP values were then scaled to a range from 0 to 100 based on the minimum and maximum values from the Training Set. Subsequently, we conducted a generalized linear regression analysis, adjusting for age at sample collection, sex and plasma tube collection type, to assess the correlation between the time to symptom onset and (1) the ALS risk score and (2) individual protein abundances. All collaborators of this study who have fulfilled the criteria for authorship required by Nature Portfolio journals have been included as authors, as their participation was essential for the design and implementation of the study. Roles and responsibilities were agreed among collaborators ahead of the research. This work includes locally relevant findings, which have been determined in collaboration with local partners. This research was not severely restricted or prohibited in the researchers' setting and did not result in stigmatization, incrimination, discrimination or personal risk to participants. Local and regional research relevant to our study was considered in citations. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. We made our summary statistics and individual-level proteomic data publicly available (https://zenodo.org/records/14213102)66. Furthermore, we developed an online tool that enables researchers to explore the machine learning model and use it with their own Olink data (https://ndru-ndrs-lng-nih.shinyapps.io/web_server_shiny/). The pQTL data from the UK Biobank used for Mendelian randomization are available at https://www.synapse.org/Synapse:syn51365301, and the ALS GWAS data are available at https://gwas.mrcieu.ac.uk/datasets/ebi-a-GCST90027163/. The Olink data from the UK Biobank can be obtained via controlled access on their web portal (https://www.ukbiobank.ac.uk/). The interactive web portal was developed using ‘shiny' (version 1.9.1). We additionally made our programming code and data publicly available to facilitate replication and enable other researchers to use our analytical pipeline in their work (https://zenodo.org/records/14213102)66. Feldman, E. L. et al. Amyotrophic lateral sclerosis. Projected increase in amyotrophic lateral sclerosis from 2015 to 2040. Fournier, C. N. Considerations for amyotrophic lateral sclerosis (ALS) clinical trial design. Masrori, P. & Van Damme, P. Amyotrophic lateral sclerosis: a clinical review. Controversies and priorities in amyotrophic lateral sclerosis. Chiò, A. et al. Phenotypic heterogeneity of amyotrophic lateral sclerosis: a population based study. Piemonte and Valle d'Aosta Register for Amyotrophic Lateral Sclerosis (PARALS).Incidence of ALS in Italy: evidence for a uniform frequency in Western countries. Brooks, B. R., Miller, R. G., Swash, M. & Munsat, T. L. El Escorial revisited: revised criteria for the diagnosis of amyotrophic lateral sclerosis. Katz, D. H. et al. Proteomic profiling platforms head to head: leveraging genetics and clinical traits to compare aptamer- and antibody-based methods. Gaetani, L. et al. Neurofilament light chain as a biomarker in neurological disorders. Azari, M. F. et al. Behavioural and anatomical effects of systemically administered leukemia inhibitory factor in the SOD1G93A G1H mouse model of familial amyotrophic lateral sclerosis. Van Den Bosch, L., Van Damme, P., Bogaert, E. & Robberecht, W. The role of excitotoxicity in the pathogenesis of amyotrophic lateral sclerosis. Shefner, J. M. et al. Skeletal muscle in amyotrophic lateral sclerosis. Vandoorne, T., De Bock, K. & Van Den Bosch, L. Energy metabolism in ALS: an underappreciated opportunity? van Rheenen, W. et al. Common and rare variant association analyses in amyotrophic lateral sclerosis identify 15 risk loci with distinct genetic architectures and neuron-specific biology. Plasma proteomic associations with genetics and health in the UK Biobank. Faghri, F. et al. Identifying and predicting amyotrophic lateral sclerosis clinical subgroups: a population-based machine-learning study. Lundberg, S. M. & Lee, S. A unified approach to interpreting model predictions. 31st International Conference on Neural Information Processing Systems 4768–4777 (Curran Associates, 2017). Irwin, K. E., Sheth, U., Wong, P. C. & Gendron, T. F. Fluid biomarkers for amyotrophic lateral sclerosis: a review. Recent advances in the diagnosis and prognosis of amyotrophic lateral sclerosis. Saez-Atienzar, S. et al. Genetic analysis of amyotrophic lateral sclerosis identifies contributing pathways and cell types. Chiò, A. et al. Secular trends of amyotrophic lateral sclerosis: the Piemonte and Valle d'Aosta Register. & Wikkelso, C. Patients with amyotrophic lateral sclerosis and other neurodegenerative diseases have increased levels of neurofilament protein in CSF. Davies, J. C. et al. Limited value of serum neurofilament light chain in diagnosing amyotrophic lateral sclerosis. Chen-Plotkin, A. S. Unbiased approaches to biomarker discovery in neurodegenerative diseases. Petrera, A. et al. Multiplatform approach for plasma proteomics: complementarity of Olink proximity extension assay technology to mass spectrometry-based protein profiling. Xu, F. et al. Olink proteomics analysis uncovers inflammatory proteins in patients with different states of bipolar disorder. Wang, T. et al. Olink proteomics analysis uncovers the landscape of inflammation-related proteins in patients with acute compartment syndrome. Plasma Olink proteomics reveals novel biomarkers for prediction and diagnosis in dilated cardiomyopathy with heart failure. Large-scale plasma proteomics comparisons through genetics and disease associations. Fredriksson, S. et al. Multiplexed protein detection by proximity ligation for cancer biomarker validation. Cui, M., Cheng, C. & Zhang, L. High-throughput proteomics: a methodological mini-review. Jain, K. K. The Handbook of Biomarkers (Humana Press, 2010). Kiernan, M. C. et al. Amyotrophic lateral sclerosis. Clinical trial designs for predictive biomarker validation: theoretical considerations and practical challenges. Blennow, K., Hampel, H., Weiner, M. & Zetterberg, H. Cerebrospinal fluid and plasma biomarkers in Alzheimer disease. Lleo, A. Biomarkers in neurological disorders: a fast-growing market. Lu, C. H. et al. Neurofilament light chain: a prognostic biomarker in amyotrophic lateral sclerosis. Jack, C. R. Jr. et al. NIA-AA Research Framework: toward a biological definition of Alzheimer's disease. Bjornevik, K. et al. Prediagnostic neurofilament light chain levels in amyotrophic lateral sclerosis. Plasma proteome profiling to assess human health and disease. Irwin, K. E. et al. A fluid biomarker reveals loss of TDP-43 splicing repression in presymptomatic ALS–FTD. Portley, M. et al. Cognitive determinants of decisional capacity in neurodegenerative disorders. Ferrucci, L. The Baltimore Longitudinal Study of Aging (BLSA): a 50-year-long journey and plans for the future. Criteria for the diagnosis of corticobasal degeneration. Diagnosis and management of dementia with Lewy bodies: fourth consensus report of the DLB Consortium. Emre, M. et al. Clinical diagnostic criteria for dementia associated with Parkinson's disease. Gilman, S. et al. Second consensus statement on the diagnosis of multiple system atrophy. Postuma, R. B. et al. MDS clinical diagnostic criteria for Parkinson's disease. Hoglinger, G. U. et al. Clinical diagnosis of progressive supranuclear palsy: the movement disorder society criteria. Accuracy of routinely-collected healthcare data for identifying motor neurone disease cases: a systematic review. Dewan, R. et al. Pathogenic huntingtin repeat expansions in patients with frontotemporal dementia and amyotrophic lateral sclerosis. Diaz-Papkovich, A., Anderson-Trocme, L., Ben-Eghan, C. & Gravel, S. UMAP reveals cryptic population structure and phenotype heterogeneity in large genomic cohorts. Assessment of variability in the plasma 7k SomaScan proteomics assay. Ashburner, M. et al. Gene ontology: tool for the unification of biology. Kanehisa, M., Goto, S., Furumichi, M., Tanabe, M. & Hirakawa, M. KEGG for representation and analysis of molecular networks involving diseases and drugs. An introduction to variable and feature selection. Plasma proteomics defines a diagnostic biomarker panel for ALS. This work was supported in part by the Intramural Research Program of the National Institutes of Health (NIH), the National Institute on Aging (NIA) (1ZIAAG000933), the National Institute of Neurological Disorders and Stroke (NINDS) (ZIANS003154), the Division of Intramural Research of the National Institute of Allergy and Infectious Diseases (1ZIAAI001242) and Merck Sharp & Dohme Corporation, a subsidiary of Merck & Co., Inc. The contributions of the NIH authors were made as part of their official duties as NIH federal employees, are in compliance with agency policy requirements, and are considered Works of the United States Government. However, the findings and conclusions presented in this paper are those of the authors and do not necessarily reflect the views of NIH or the US Department of Health and Human Services. received additional support from the Centers for Disease Control and Prevention, the Muscular Dystrophy Association, Microsoft Research, the Packard Center for ALS Research at Johns Hopkins, the ALS Association and Cerevel Therapeutics (now part of AbbVie, Inc.). This work was partly supported by the Italian Ministry of Health (Ministero della Salute, Ricerca Sanitaria Finalizzata, grant RF-2021-12374238); the Progetti di Rilevante Interesse Nazionale program of the Ministry of Education, University and Research (grants 2017SNW5MB and 20228N7573); the Horizon 2020 program (project BRAINTEASER under grant agreement no. 101017598); and the Horizon Europe program (Project Hereditary under grant agreement no. receives salary support from the NIH (NINDS and NIA). This study also used genotype and clinical data from the HyperGenes Consortium. This research has been conducted using the UK Biobank Resource under application number 33601. A list of members and their affiliations appears in the Supplementary Information. These authors contributed equally: Ruth Chia, Ruin Moaddel, Clifton L. Dalgard, Sonja W. Scholz, Adriano Chiò, Bryan J. Traynor. Neuromuscular Diseases Research Section, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA Ruth Chia, Memoona Rasheed, Paola Ruffo, Rosario Vasta, Betul Comertpay & Bryan J. Traynor Laboratory of Clinical Investigation, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA Neurodegenerative Disorders Clinic, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA Neurodegenerative Diseases Research Section, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA Natalie Landeck, Paolo Reho, Anindita Ray, Kumar Arvind & Sonja W. Scholz Department of Environmental Health Sciences, Mailman School of Public Health, Columbia University, New York, NY, USA ‘Rita Levi Montalcini' Department of Neuroscience, Amyotrophic Lateral Sclerosis Center, University of Turin, Turin, Italy Rosario Vasta, Andrea Calvo, Cristina Moglia, Antonio Canosa, Umberto Manera, Maurizio Grassano, Maura Brunetti, Federico Casale & Adriano Chiò Andrea Calvo, Cristina Moglia, Antonio Canosa, Umberto Manera & Adriano Chiò Institute of Cognitive Sciences and Technologies, C.N.R, Rome, Italy Department of Neurology, The Ohio State University, Columbus, OH, USA Computational Biology Group, Laboratory of Neurogenetics, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA Department of Anatomy, Physiology and Genetics, Uniformed Services University of the Health Sciences, Bethesda, MD, USA Henry M. Jackson Foundation for the Advancement of Military Medicine, Bethesda, MD, USA Neuroregeneration and Stem Cell Programs, Institute for Cell Engineering, Johns Hopkins University School of Medicine, Baltimore, MD, USA Hall, Alexander Y. Pantelyat, Sonja W. Scholz & Bryan J. Traynor Inherited Movement Disorders Unit, Neurogenetics Branch, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA Parkinson's Disease Clinic, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA Laboratory of Behavioral Neuroscience, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA Laboratory of Clinical Immunology and Microbiology, Neuroimmunological Diseases Section, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, MD, USA RNA Therapeutics Laboratory, National Center for Advancing Translational Sciences, National Institutes of Health, Rockville, MD, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar A. Chiò also acquired the samples and phenotype information. holds patents on the clinical testing and therapeutic intervention for the hexanucleotide repeat expansion of C9orf72. The other authors declare no competing interests. Nature Medicine thanks Robert Bowser and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. ELISA assays were performed on plasma samples from the Discovery and Replication datasets (n = 16 ALS cases and n = 16 healthy controls). ELISA kits were available for 30 of the 33 differentially abundant proteins. We present exploratory data for 16 proteins, while the other ELISA assays are excluded due to either lack of protein detection (n = 13) or assay failure (n = 1). Each row displays data for an individual protein. The agreement between protein measurements from the ELISA or ProQuantumTM (abbreviated as ProQ) and Olink assays was evaluated using three methods: correlation (first panel), Bland-Altman analysis (second panel), and differential analysis comparing ALS (orange dots) with healthy controls (blue dots) through either ELISA or ProQuantumTM (third panel) or Olink (fourth panel) measurements. Pearson correlation coefficients (R) were calculated to assess the linear association between variables. P-values were computed using a two-sided test of the null hypothesis that the correlation coefficient equals zero. The shaded grey area reflects the 95% limits of agreement for the paired measurements from each sample, with upper and lower limit values shown in red font. Plasma profiling was performed on a random subset of the Baltimore Longitudinal Study of Aging (BLSA) samples (n = 9) using the SomaScan 7 K and Olink 3072 Explore platforms. Of these, 2,120 proteins overlapped with the SomaScan 7 K platform. K-means clustering, defined by Katz and colleagues, divided the distribution into three clusters. c, The heatmap illustrates Spearman correlations among overlapping protein measurements from the Olink and SomaScan platforms, specifically for 27 differentially expressed proteins in ALS cases. Data for the six other differentially expressed proteins (CORO6, DTNB, FGF21, NEB, RBFOX3, and SSC4D) were unavailable. These volcano plots illustrate the differential protein abundance in a, ALS patients with the pathogenic C9orf72 repeat expansion (n = 29) compared to ALS patients without the expansion (n = 202), b, ALS patients with C9orf72 (n = 29) compared to ALS patients without the expansion (n = 202) combined with control subjects (n = 383; comprising 169 neuro-control non-carriers, and 214 healthy control non-carriers), and c, asymptomatic individuals with C9orf72 (n = 12) compared to symptomatic ALS patients with the expansion (n = 29). Blue and red dots depict statistically significant down-regulated or up-regulated proteins identified through generalized linear regression. Significance was assessed using two-sided moderated t-tests with p-values adjusted by the false discovery rate (FDR) method. Each point represents a sample; orange dots represent ALS cases, dark blue dots represent healthy individuals, and light blue dots represent patients with other neurological diseases. a, The distributions of the ALS Risk Score according to the age at the time of plasma collection are shown for asymptomatic individuals (purple, n = 126), ALS cases (orange, n = 259), patients with other neurological diseases (light blue, n = 186), and a representative sample of the healthy controls (blue, n = 533). Asymptomatic individuals were the only ones showing a slight correlation with time with a 5 to 14-fold steeper slope coefficient compared to healthy individuals and those with ALS or other neurological diseases. This indicates that the increase in ALS Risk Score seen in preclinical ALS is not influenced solely by normal aging or pathological aging associated with neurodegenerative diseases. The statistical significance of each coefficient was assessed using a two-sided hypothesis test with a null hypothesis that the coefficient equals zero. b, Scatterplots comparing time to symptom onset in patients with ALS with expression levels for the 17 individual proteins included in the Random Forest model. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. A plasma proteomics-based candidate biomarker panel predictive of amyotrophic lateral sclerosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01342-6'>Fluorescent GLP1R/GIPR dual agonist probes reveal cell targets in the pancreas and brain</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 09:35:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Dual agonists targeting glucagon-like peptide-1 receptor (GLP1R) and glucose-dependent insulinotropic polypeptide receptor (GIPR) are breakthrough treatments for patients with type 2 diabetes and obesity. Compared to GLP1R agonists, dual agonists show superior efficacy for glucose lowering and weight reduction. However, delineation of dual agonist cell targets remains challenging. daLUXendins are potent GLP1R/GIPR dual agonists that advantageously show less functional selectivity for mouse GLP1R over mouse GIPR. Systemic administration of daLUXendin strongly labels GLP1R+ and GIPR+ neurons in circumventricular organs characterized by an incomplete blood–brain barrier but does not penetrate the brain beyond labelling seen with single (ant)agonists. At the single-molecule level, daLUXendin targets endogenous GLP1R–GIPR nanodomains, which differ in organization and composition from those targeted by a single agonist. daLUXendins reveal dual agonist targets in the pancreas and brain and exclude a role for brain penetration in determining the superior efficacy of dual agonists, shedding new light on different modes of action of dual agonists versus single agonists. Glucagon-like peptide-1 (GLP1) and glucose-dependent insulinotropic polypeptide (GIP) potently amplify insulin secretion and influence satiety to reduce food intake1,2,3. Stabilized GLP1R agonists were approved for the treatment of type 2 diabetes in 2008 (ref. Less attention has been paid to the GIP axis, as multiple studies showed that either genetic ablation or pharmacological blockade of GIPR protects against body weight gain6, and that GIPR agonism has minimal effects on glycemia in patients with type 2 diabetes7, potentially owing to GIPR downregulation8. However, dual agonists such as tirzepatide combine elements of GLP1R and GIPR agonism, such that the full benefits of both hormone axes are realized9. Although the pharmacological mechanisms underlying dual GLP1R/GIPR agonist efficacy are well established10,11,12, less is known about the cellular substrates that underlie actions on pancreatic hormone secretion as well as food intake. GLP1R and GIPR are low-abundance cell surface proteins, and their instability in solution makes it difficult to raise antibodies against specific epitopes, rendering their immunohistochemical visualization challenging13. Specific antibodies exist for GLP1R, but no reliable commercial antibody exists yet for the detection of GIPR in primary cells and tissues13. Compared to antibodies, GLP1R-Cre and GIPR-Cre reporter mice are sensitive, but only identify cells that express(ed) GLP1R/GIPR transcripts14,15. Finally, animals harbouring a self-labelling enzyme on the GLP1R amino terminus allow facile visualization using a range of fluorophores, although this model is restricted to rodents16. No approach to date provides information on GLP1Rs and GIPRs specifically accessed or bound by a dual agonist. To circumvent these issues, we previously developed LUXendins and sGIPs17,18,19, which are specific probes against GLP1R and GIPR, respectively. Using these probes, we confirmed that GLP1R is largely a β cell-specific protein marker18,20, whereas GIPR is expressed in both β cells and α cells within the islet17. However, we were unable to make any inferences about dual agonist binding sites within the islet and brain, as GLP1R and GIPR probes might target different receptor (sub)populations. Dual agonism might also influence GIPR/GLP1R localization, expression and access through effects in addition to those mediated by GLP1R and/or GIPR (ant)agonism alone; for example, release of paracrine mediators and functional selectivity21,22,23. Although fluorescent tirzepatide has been reported12, the probe was not pharmacologically validated for its specificity at GLP1R/GIPR. Structure–activity relationships and cryo-electron microscopy show that the binding affinity and efficacy of GLP1R agonists, GIPR agonists and dual agonists are determined by amino acids closer to the N-terminus24,25,26,27. We thus reasoned that carboxy-terminal functionalization with a fluorophore would be well tolerated, especially because this end is not commonly resolved in structural interpretations. Given that tirzepatide possesses a C-terminal serine, we decided to generate an S39C mutant, allowing the late-stage addition of a fluorophore (Fig. daLUXendin544 and daLUXendin660 were obtained using cysteine-maleimide ‘click' chemistry to install Cy3 and Cy5 fluorophores, respectively, in yields of ~50%. iso-butyric amino acid was incorporated into the structure instead of alanine at position two to confer degradation resistance against DPP-IV28 (Fig. a,b, Tirzepatide, modified at the C-terminus with S39C to allow maleimide conjugation with either Cy3 or Cy5 (model obtained from cryo-electron microscopic structure of non-acyl tirzepatide bound to activated GLP1R in complex with Gs) (PDB 7VBI)26. c, cAMP responses to GLP1, GIP, tirzepatide (TZP), daLUXendin544 (daLUX544) and daLUXendin660 (daLUX660) in AD293 cells expressing either hGLP1R, mGLP1R, hGIPR or mGIPR (n = 4 independent repeats). d,e, Binding affinity of daLUXendin660 against hGLP1R and hGIPR (d), with Kd values (e) in a separate bar graph (one-site fit specific binding) (n = 3 independent repeats). f,g, Competition binding for daLUXendin660 (f) and either Ex4-Cy5 or GIP-Cy5 (g) at hGLP1R and hGIPR versus GLP1 and GIP (one-site fit Ki) (n = 3 independent repeats). At dark onset, mice were injected subcutaneously with either vehicle, daLUXendin660 (daLUX; 10 nmol kg−1) or tirzepatide (10 nmol kg−1), and then food was returned. Graph showing cumulative food intake 12 h and 24 h post injection (n = 8 mice per investigated state) (repeated-measures two-way ANOVA with Šídák's multiple comparisons test). j, Tirzepatide and daLUXendin660 similarly lower glucose in GLP1RWT/WT and GLP1RWT/KO mice during glucose tolerance testing (n = at least five mice per investigated state). k,l, Efficacy of tirzepatide and daLUXendin660 is reduced in GLP1RKO/KO mice (k), shown also by area under the curve (AUC) (l) (n = at least five mice per investigated state) (one-way ANOVA with two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). m, mGLP1R:mGIPR selectivity is lower for daLUXendin660 versus tirzepatide (selectivity is calculated using the mean AUC ratio from at least five GLP1RWT/WT and five GLP1RKO/KO mice). Bar and line graphs show means; error bars, s.e.m. Exact P values are displayed on each graph. AD293 cells transfected with either human GLP1R (hGLP1R), mouse GLP1R (mGLP1R), human GIPR (hGIPR) or mouse GIPR (mGIPR) were used to determine cAMP signalling potency of daLUXendin544 and daLUXendin660 versus unmodified (acylated) tirzepatide, GIP and GLP1. At both hGLP1R and mGLP1R, tirzepatide and its fluorescent analogues showed reduced potency compared to GLP1, with the potency rank order favouring daLUXendin660 > tirzepatide > daLUXendin544 (Fig. As previously reported for tirzepatide11, there were species differences for GIPR signalling with each of the dual agonist ligands tested, with all showing a larger loss of potency at the mGIPR than at the hGIPR relative to native hGIP (Fig. Of note, even though daLUXendin544 showed globally reduced potency compared to daLUXendin660, half-maximum effective concentration (EC50) ratios suggested that both ligands show similar GIPR:GLP1R selectivity of ~2:1 (that is, favouring GIPR) for human receptors and ~1:2 (that is, favouring GLP1R) for mouse receptors. By comparison, tirzepatide showed GIPR:GLP1R selectivity of ~3:1 for human receptors and ~1:8 for mouse receptors in our assays. daLUXendin660 showed nM binding affinity at both hGLP1R and hGIPR, although assessment of the daLUXendin544 dissociation constant (Kd) was precluded by interference with the BRET signal (Fig. Competition assays showed that binding affinity (pKi) for hGLP1/hGLP1R and hGIP/hGIPR was similar when using daLUXendin660 as a probe compared to Ex4-Cy5 and GIP-Cy5, further confirming strong receptor binding (Fig. Therefore, based upon cAMP assays and binding affinity, daLUXendin544 and daLUXendin660 are potent GLP1R/GIPR dual agonists that advantageously show approximately fourfold less selectivity for mouse GLP1R over GIPR compared to tirzepatide. To establish whether in vitro pharmacology is mirrored by in vivo efficacy, mice were dosed with either tirzepatide or daLUXendin660 at 10 nmol kg−1 to maximally engage mGLP1R and mGIPR11,29. At this dose, daLUXendin660 and tirzepatide both reduced food intake during the dark phase and over a cumulative 24 h period. At 12 h and 24 h post injection, tirzepatide demonstrated ~40% greater efficacy in driving anorexia, presumably because of its acylation and plasma binding (Fig. Likewise, both tirzepatide and daLUXendin660 exerted substantial glucose-lowering effects in Glp1rWT/WT and Glp1rWT/KO mice, with similar area under the curves (Fig. Both ligands partially maintained their efficacy in Glp1rKO/KO mice, reflecting continued engagement of mGIPR at doses >1–3 nmol kg−1 (Fig. However, Glp1r knockout was less effective in daLUXendin660-treated versus tirzepatide-treated mice, demonstrating increased mGIPR dependence for daLUXendin660, as expected from in vitro functional selectivity data (Fig. Results should, however, be interpreted in light of increased mGIPR sensitivity or expression in Glp1rKO/KO mice11. Together, these data confirm that daLUXendin660 and tirzepatide both lower blood glucose and reduce food intake. In line with the cAMP results, daLUXendin544 led to concentration-dependent labelling of SNAP-hGIPR:AD293 cells, overlapping with cell-impermeable SBG-Oregon Green (OG) SNAP label (Extended Data Fig. The brightest labelling was detected at 50 nM–5 µM, corresponding to the cAMP max (Extended Data Fig. The strongest and most specific membrane labelling, without evidence of altered cell morphology (Extended Data Fig. 1a–c), was observed with 500 nM daLUXendin544; hence, this concentration was selected for experiments in cell lines and tissue. No labelling was observed in non-transfected AD293 cells (Extended Data Fig. Therefore, daLUXendin544 and daLUXendin660 are highly specific dual agonist probes suitable for cell labelling. MIN6-CB4 β cells were labelled with both daLUXendin660 and the specific GLP1R probe LUXendin551 (refs. Demonstrating specificity for GLP1R, LUXendin551 signal colocalized with daLUXendin660 (Extended Data Fig. 2a), probably owing to labelling of different GLP1R pools (see below). However, not all daLUXendin660 signal colocalized with LUXendin551, presumably owing to GIPR labelling, as well as increased GLP1R internalization (Extended Data Fig. To investigate this possibility further, MIN6-CB4 β cells were incubated with daLUXendin660 and the specific GIPR probe sGIP549 (ref. Demonstrating specificity for GIPR, sGIP549 signal colocalized with daLUXendin660, whereas not all daLUXendin660 signal colocalized with sGIP549, corresponding to GLP1R labelling (Extended Data Fig. Therefore, daLUXendin660 labels both endogenous GLP1R and GIPR in MIN6-CB4 β cells. We next repeated studies in isolated mouse islets. To circumvent competition between the various probes at GLP1R/GIPR, we used GLP1RSNAP/SNAP mice in which a SNAP-tag enzyme self-label is knocked into the extracellular domain of the endogenous GLP1R16, allowing GLP1R to be visualized without influencing orthosteric binding. Using this model, colocalization could be detected between the cell-impermeable SNAP label BG-Sulfo646 and daLUXendin544 (Extended Data Fig. We noticed that daLUXendin544 retained significant levels of GLP1R at the cell surface, as shown by the full width at half maximum (Extended Data Fig. By contrast, daLUXendin544 did not exhibit any detectable membrane labelling in α cells, presumably because they express little to no GLP1R transcript or protein (also used to identify α cells versus β cells) (Extended Data Fig. 2f,h)15,18,20, so α cell labelling is largely by GIPR. Using the GIPR probe sGIP648, daLUXendin544 was found to strongly label GIPR in mouse islets (Extended Data Fig. 2g,h), also apparent from studies with the GLP1R probes and GLP1R SNAP labels. Lastly, daLUXendin544 labelling was reduced ~50% in GLP1RKO/KO islets, with a further reduction in labelling seen in the presence of a saturating concentration of GIPAib2 (1 µM) (Extended Data Fig. To allow post hoc protein labelling and identification of cell types in complex tissue, we optimized a fixation protocol that retained strong and specific daLUXendin544 and daLUXendin660 labelling post-fixation with either 2% or 4% formalin for 15 min (Fig. Islets were labelled with daLUXendin544 before formalin-fixation, and immunostaining with antibodies against either insulin, glucagon or somatostatin to identify β cells, α cells or δ cells, respectively. As expected, given their abundant GLP1R and GIPR expression, β cells were strongly labelled with daLUXendin544 (Fig. Confirming results in live islets, α cells showed weaker but detectable daLUXendin544 labelling restricted to the cytoplasm (Fig. 2c), probably reflecting their abundant GIPR expression and largely absent GLP1R protein expression17,18,20. However, labelling was observed both in the cytoplasm and membrane, which might reflect low-moderate expression of GIPR and GLP1R protein (Fig. Identical results were obtained with daLUXendin660 (Fig. Quantification of daLUXendin544/660 labelling is shown in Fig. Demonstrating efficacy of daLUXendin660 in vivo, islets in GLP1R-tdRFP and GIPR-GFP reporter mice were strongly and specifically labelled 60 min following intravenous injection with 100 nmol kg−1 probe but not vehicle (Fig. a, Following labelling of live islets, daLUXendin660 can be fixed using either 2% or 4% formalin (FA) for 15 min (n = at least five islets from three mice). b, daLUXendin544 labels insulin-positive (INS+) cells throughout the mouse islet (n = 44 islets from five mice). c,d, daLUXendin544 labels glucagon-positive (GCG+) and somatostatin-positive (SST+) cells throughout the islet, although to a lesser extent than INS+ cells (INS, 44 islets from five mice; GCG, 25 islets from seven mice; SST, 29 islets from seven mice). f,g, Quantification of daLUXendin544 (f) and daLUXendin660 (g) labelling intensity in INS+, GCG+ and SST+ cell populations (Kruskal–Wallis test with Dunn's multiple comparisons test). injection of daLUXendin660, but not vehicle (inset), labels GLP1R+ cells in islets from GLP1R-tdRFP reporter mice (arrows show GLP1R− but daLUXendin660+ cells) (n = at least nine islets). i, As in h, but with GIPR-GFP reporter mice (n = 9 islets). k,l, daLUXendin660 labels INS+, GCG+ and SST+ cell populations in SC-islets (inset in k, a separate non-immunostained SC-islet showing membrane and cytoplasmic daLUXendin660 signal). Exact P values are displayed on each graph. Fluorescent labelling of low-abundance proteins in human islets is limited by high levels of autofluorescence, in particular from lipofuscin accumulation32. To circumvent these issues, we instead turned to human induced pluripotent stem (iPS) cell-derived islet-like structures (SC-islets), which are therapeutically important as a cell therapy for type 1 diabetes33,34. In keeping with the transcriptomic data, immunostaining revealed daLUXendin660 labelling in insulin (INS+), glucagon (GCG+) and somatostatin (SST+) cell populations (Fig. Some daLUXendin660 labelling was detected in INS−/GCG−/SST− cells, which were presumably GLP1R/GIPR-expressing enteroendocrine cells that are present in most differentiations (Fig. Similar to mouse islets, the strongest daLUXendin660 labelling was seen in β cells, with weaker but detectable labelling in α cells and δ cells (Fig. To expand the utility of daLUXendin660 for long-term dosing, we installed a C20 di-acid lipid chain on lysine at position 20 to confer albumin binding. The new molecule, termed daLUXendin660+, performed similarly to daLUXendin660 in pharmacology and receptor binding assays, demonstrating that acylation and lipid modification is well tolerated (Extended Data Fig. No differences in pancreatic islet cell labelling could be detected between daLUXendin660 and daLUXendin660+ applied to the same islet batch (that is, INS > GCG and SST) (Extended Data Fig. Following intravenous injection (100 nmol kg−1), daLUXendin660 signal could be readily detected in the circumventricular organs and choroid plexus but not in other areas of the brain (Fig. The circumventricular organs are characterized by an incomplete blood–brain barrier and include areas such as the median eminence, area postrema, subfornical organ and organum vasculosum of the lamina terminalis, brain regions known to bind GLP1R and GIPR (ant)agonists17,18. We did not observe any difference in the extent of brain penetration between daLUXendin660, daLUXendin660+ and the single GLP1R agonist LUXendin645 (Fig. To establish which incretin receptor-expressing neurons are labelled with daLUXendin660, we performed intravenous injection (100 nmol kg−1) of daLUXendin660 in transgenic mice Cre-dependently expressing fluorescent reporters in either GLP1R-expressing cells (GLP1R-tdRFP) or GIPR-expressing cells (GIPR-GCaMP6). 3e) neurons was observed in the area postrema. a–c, Intravenous administration of 100 nmol kg−1 daLUXendin660 (n = 3 mice) (a), 100 nmol kg−1 daLUXendin660+ (n = 2 mice) (b) and subcutaneous administration of 100 nmol kg−1 LUXendin645 (GLP1R antagonist) (n = 5 mice) (c) led to staining in circumventricular organs of the brain and the choroid plexus. Shown are the organum vasculosum of the lamina terminalis (i), subfornical organ (ii), choroid plexus (iii), median eminence (iv) and area postrema (v). d,e, Intravenous administration of 100 nmol kg−1 daLUXendin660 in a GLP1R-tdRFP mouse (n = 1) (d) or GIPR-GCaMP6 mouse (n = 2) (e) labels GLP1R-expressing and GIPR-expressing cells, respectively (indicated by white arrows). Note that images in e are from the same mouse as in a but with GIPR-GCaMP6 signal overlaid. The strength and extent of labelling does not necessarily equate to agonist efficacy, especially for peptidic ligands that diffuse freely through fenestrated vessels of the median eminence into the ventricular system22,36. Strong labelling was observed in tanycytes—identified through vimentin immunostaining—within the third ventricle (Extended Data Fig. daLUXendin660 labelling in tanycytes was localized to the apical surface as well as cellular processes extending into the brain parenchyma (Extended Data Fig. Neurons identified as GLP1R+ in GLP1R-tdRFP reporter mice were co-labelled with daLUXendin660 in the vicinity of the ventricles (Extended Data Fig. A small number of neurons positive for both GIPR and GLP1R were labelled with daLUXendin660 (Extended Data Fig. GPCRs possess higher organization at the cell membrane and within the cell, forming nanodomains that are important for signalling16,37,38. We therefore set out to determine how endogenous GLP1R/GIPR nanodomains might be targeted by single agonists versus a dual agonist in primary tissue; that is, pancreatic islets. We first established nanodomain organization in the non-stimulated but bound state using LUXendin645, a GLP1R antagonist coupled to Cy5. Following 60 min incubation with LUXendin645, fixation and dSTORM nanoscopy of whole islets, we confirmed our previous data showing that non-stimulated GLP1R form discrete nanodomains16,18 (Fig. The GIPR agonist sGIP648 also labelled discrete receptor nanodomains (GIPR), as defined by clustering (Fig. daLUXendin660, however, engaged more ordered receptor nanodomains compared to the GIPR agonist alone, with high levels of clustering probably reflecting increased receptor interactions (Fig. Localization number per cluster, as well as the density of each cluster, were similar across all ligands examined (Fig. Together, these data suggest that tirzepatide either engages GLP1R into clustering or increases interactions between GLP1R and GIPR, thus contributing to its unique nanodomain arrangement. Owing to the long acquisition times needed for localization counting, dSTORM is best suited to fixed tissue. We therefore set out to show the applicability of daLUXendin660 for TIRF-based GLP1R/GIPR single particle tracking at the cell membrane in live MIN6-CB4 cells (Fig. To open up super-resolution live imaging approaches, such as STED, we replaced the C-terminal Cy5 with fluorogenic deuterated silicon rhodamine (SiR-d12)39 to create daLUXendin651-d12 (Fig. Labelling with daLUXendin651-d12 was too dim to visualize endogenous receptor but was sufficient to map GLP1R in fixed CHO-K1:SNAP-GLP1R:Halo-GIPR cells. daLUXendin651-d12 was able to image GLP1R below the diffraction limit and could be recorded over multiple frames (Supplementary Fig. Following direct application of daLUXendin651-d12 to CHO-K1:SNAP-GLP1R:Halo-GIPR cells, far-red cell surface signal from SiR-d12 became rapidly visible (10 s) in live STED on the cell surface, colocalizing with BG-Sulfo549 and CA-AF488 (Fig. a, Schematic showing single-molecule labelling and localization strategy for GLP1R (LUXendin645; LUX645), GIPR (sGIP648) and GLP1R/GIPR (daLUXendin660) (PDB 7VBI)26. b, Representative dSTORM images show different single-molecule densities and organization (30 nm) of LUX645-bound GLP1R, sGIP648-bound GIPR and daLUXendin660-bound GLP1R/GIPR across an islet cell population (nucleus is bounded by dashed line) (n = 5 islets from three mice). c,d, daLUXendin660 labels more GLP1R/GIPR clusters compared to either LUX645 or sGIP648, shown by representative images (c) and bar graph (d) (n = 5 islets from three mice) (one-way ANOVA with two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). e, Distribution and mean (inset) localization per cluster are similar across LUX645, sGIP648 and daLUXendin660 probes (n = 5 islets from three mice) (one-way ANOVA with Šídák's multiple comparisons test). f, As in e, but for cluster density (one-way ANOVA with Šídák's multiple comparisons test). g, Single particle tracking of daLUXendin660-labelled GLP1R/GIPR, showing min–max displacement (n = 5 cells, two independent repeats). h,i, SiR-d12 can be installed on daLUXendin (h) to create daLUXendin651-d12 (i), optimized for STED nanoscopy. j, Confocal snapshots showing CHO-K1:SNAP-GLP1R:Halo-GIPR cells labelled for GLP1R (BG-Sulfo549) and GIPR (CA-AF488) before live STED imaging of daLUXendin651-d12 labelling (n = 5 wells). Scale bars are provided on each figure. Exact P values are displayed on each graph. In the present study, we synthesize and validate daLUXendins, red and far-red GLP1R/GIPR dual agonist probes. Nanomolar concentrations of daLUXendin specifically label GLP1R/GIPR in live cells, allowing endogenous binding sites and cellular targets for dual agonists such as tirzepatide to be visualized and interrogated. daLUXendin labelling can be achieved in most cells and tissues with a simple 1 h incubation or injection, followed by washing and analysis. Advantageously, daLUXendins show minimal loss in signal intensity when formaldehyde-fixed, allowing co-labelling for other cellular or neuronal proteins and markers. A key aspect of tirzepatide efficacy is increased potency and selectivity at human GIPR compared to GLP1R, with approximately 70% of activity at GIPR10,11,12. In mice, however, tirzepatide is selective for GLP1R over GIPR, and its insulinotropic and overall therapeutic efficacy is equivalent or non-superior to semaglutide (GLP1R agonist)10,11. Although this is unlikely to have any bearing on GLP1R and GIPR accessed and bound by tirzepatide, it remains a limitation to the use of preclinical mouse models to understand GLP1R/GIPR dual agonism. Although the exact mechanisms underlying the differences between daLUXendin544/660 and tirzepatide remain unknown, our previous studies showed that fluorophore modification can increase GLP1R agonist efficacy18. Therefore, daLUXendin660 could be a useful tool for probing dual agonist biology in the mouse. Within the pancreatic islet, tirzepatide labelled β cells > α cells = δ cells, in keeping with known protein or transcript expression levels for GLP1R and GIPR (as previously reviewed21). Although studies have shown that tirzepatide stimulates secretion of insulin, somatostatin and glucagon11,21, the cellular substrates have so far remained elusive. Given that α cells express GIPR >> GLP1R, direct effects of tirzepatide on glucagon secretion are probably mediated by GIPR signalling, a known stimulator of α cell function40,41. By contrast, δ cells express GIPR > GLP1R, inferring a predominant role for GIPR in tirzepatide-stimulated somatostatin secretion, with some GLP1R contribution. β cells express GLP1R = GIPR, but increased affinity of tirzepatide for human GIPR10 is likely to disproportionately contribute to insulin secretion. As well as exerting direct effects on islet cell function, tirzepatide is also likely to stimulate α cell-to-β cell and δ cell-to-β cell communication, further regulating insulin release (as previously reviewed21,42). At the same time, tirzepatide might engage β cell-to-α cell or δ cell-to-α cell communication to provide a brake on glucagon secretion. Peripherally injected daLUXendin660 was able to label the pancreas as well as readily access the median eminence, organum vasculosum of the lamina terminalis, subfornical organ and area postrema. The extent of daLUXendin660 labelling was similar to that seen for a GLP1R antagonist, LUXendin645, as well as a GIPR agonist, sGIP648 (refs. We therefore speculate that the superior efficacy of GLP1R/GIPR dual agonism versus single GLP1R agonism reflects the specific neuronal and supporting cell populations that are activated rather than the extent of brain access. Periventricular cell labelling is a common feature across GLP1R/GIPR (dual) (ant)agonists, pointing to a critical role for this cell type in the regulation of food intake and possibly the ligands themselves43. Outside of conventional pharmacology, it remains unknown how daLUXendin660 interacts with GLP1R/GIPR in cells that endogenously express both receptors. Single-molecule localization microscopy showed that daLUXendin660 densely labelled GLP1R/GIPR clusters or nanodomains in islets, which was in addition to that seen with a GIPR agonist alone. Therefore, daLUXendin660 probably influences either GLP1R clustering or GLP1R and GIPR localization to increase nanodomain formation. We speculate that GIPR and GLP1R signalling within or between cell nanodomains, rather than simple signal summation across the cytoplasm, contributes to tirzepatide efficacy, as recently shown for GLP1 and β2-adrenergic receptors44. Tirzepatide is just one of many emerging dual and triple agonists that might exhibit different receptor engagement profiles. Given that we are still piecing together single agonist function45, daLUXendin provides a powerful tool to understand dual agonist receptor engagement, brain or organ access and receptor synergism. Owing to the structural similarities between dual and triple agonists, in the future, daLUXendins can be easily modified to incorporate a glucagon receptor agonist component. The study has a number of limitations. Therefore, we cannot categorically state that brain areas or cell types accessed by tirzepatide in humans and rodents would be similar to those accessed by daLUXendins in rodents. Secondly, we did not provide any data on human islets. We have noticed heterogeneity in GLP1R/GIPR expression, probably reflecting culture time and donor variability, compounded by high levels of lipofuscin autofluorescence32. Thirdly, we restricted daLUXendin544/660/651-d12 to Cy3, Cy5 and SiR-d12 fluorophores, as they are well suited for complex tissue labelling16,18,19. In the future, the daLUXendin colour palette could be extended to include blue-near-infrared and even epitope tags to allow signal amplification. In summary, daLUXendin544/660 are highly specific probes that highlight dual agonist target cells within complex tissues such as the pancreatic islet and brain. We envisage that daLUXendin544/660 will provide interrogable mechanisms underlying dual agonist efficacy. Islet isolation is a nuisance variable, and as such, data are taken from independent islet preparations. To ensure that all states were represented in the different experiment arms, randomization was used to allocate samples and animals to treatment groups. All studies combined males and females, as no sex-specific phenotype has been reported for any of the genetically modified lines used, and male and female tissues were indistinguishable by their chemical probe or SNAP-tag labelling. Animal studies were regulated by the Animals (Scientific Procedures) Act 1986 of the UK (Personal Project Licences P2ABC3A83, PP1778740 and PP6526002). Studies with human iPS cells were reviewed by the University of Oxford Medical Sciences Interdivisional Research Ethics Committee (MS IDREC). Dry solvents were purchased from Acros Organics (Thermo Fisher Scientific). Amino acids and resins for solid-phase peptide synthesis were purchased from Novabiochem (Merck) or Iris Biotech. Ultra-high performance liquid chromatography with UV–visible light detection was performed for purity assessment, using an Agilent 1260 Infinity II LC System equipped with an Agilent SB-C18 column (1.8 µm, 2.1 × 50 mm). Buffer A consisted of 0.1% formic acid in H2O; buffer B consisted of 0.1% formic acid in acetonitrile. Retention times are given in minutes. Chromatograms were imported into Graphpad Prism 8, and purity was determined by calculating area under the curve ratios. Preparative or semi-preparative high-performance liquid chromatography was performed on an Agilent 1260 Infinity II LC System equipped with columns as follows: preparative column, Reprospher 100 C18 columns (10 μm: 50 × 30 mm at 20 ml min−1 flow rate; semi-preparative column, 5 μm: 250 × 10 mm at 4 ml min−1 flow rate. Eluents A (0.1% TFA in H2O) and B (0.1% TFA in MeCN) were applied as a linear gradient. Peak detection was performed at maximal absorbance wavelength. For high-resolution mass spectrometry, samples were analysed on an Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific). Mass spectrometry scans were acquired in a range of 350–1500 m/z. MS1 scans were acquired in the Orbitrap with a mass resolution of 120,000 with an AGC target value of 4 × 105 and 50 ms injection time. MS2 scans were acquired in the ion trap with an AGC target value of 1 × 104 and 35 ms injection time. Precursor ions with charge states two to four were isolated with an isolation window of 1.6 m/z and 40 s dynamic exclusion. Precursor ions were fragmented using higher-energy collisional dissociation with 30% normalized collision energy. Peptide synthesis and chemical characterization (high-resolution mass spectrometry, high-performance liquid chromatography) are detailed in the Supplementary Note 1. AD293 cells (RRID:CVCL_9804) were transiently transfected 24 h before the assay in six-well plates with plasmids encoding wild-type hGLP1R, mGLP1R, hGIPR or mGIPR using Lipofectamine 2000. AD293 cells were authenticated at source using STR profiling. Cells were detached and treated with a range of agonist concentrations in serum-free DMEM + 0.1% BSA for 30 min at 37 °C in 96-well plates, followed by the addition of lysis buffer 2 (Cisbio) at a 1:2 ratio. Lysates were further diluted 1:10 to avoid spectral interference from the higher concentrations of fluorescent ligands in the HTRF assay, and the concentration of cAMP was determined after the addition of cAMP Dynamic detection reagents (Cisbio). Three-parameter logistic fitting was used to determine signalling potencies for each ligand, with EC50 ratios relative to the cognate ligand for each receptor used to establish test agonist selectivity for GLP1R versus GIPR. HEK293T cells (RRID:CVCL_0063) were cultured in DMEM/F12 GlutaMAX (ThermoFisher), supplemented with 10% FBS and 1% antibiotic-antimycotic solution and cultured at 37 °C, 5% CO2. HEK293T cells were authenticated at source using STR profiling. Cells transiently expressing Nluc–GLP1R or Nluc–GIPR were seeded onto white 96-well plates and cultured for 24 h. Media was then removed and the cells were incubated in PBS supplemented with 0.1% BSA and 0.01% NanoGlo (Promega). For saturation binding assays, daLUXendins were added across the concentration range 0.1–31.6 nM, in the absence or presence of 1 μM GIP and 1 μM Exendin-9 to determine specific binding. For competition binding assays, cells were co-treated with 10 nM daLUXendin and GLP1 or GIP across the range 1 μM to 1 pM. BRET ratios (λacceptor/λdonor) were calculated, and saturation data fit using the ‘One Site – Specific Binding' equation of GraphPad Prism 10. 18) littermates (on a C57BL/6J background) were fasted for 4–6 h, with free access to water. Wild-type and heterozygous mice were used as controls, as loss of a single Glp1r allele was found to exert minimal influence on phenotype46. Tirzepatide and daLUXendin were administered at 10 nmol kg−1 before intraperitoneal injection of 2 g kg−1 glucose 60 min later. Glucose was measured in blood from the tail vein at 0, 15, 30, 60, 90 and 120 min post glucose challenge. Mice were socially housed in specific-pathogen-free conditions under a 12 h light–dark cycle with ad libitum access to food and water, relative humidity 55 ± 10% and temperature 21 ± 2 °C. Male and female 12–14 week old C57BL/N mice were singly housed in a 12 h dark–light cycle (14:00–02:00 h) and habituated to FED3 feeding devices for 7 days before commencing the study47. On test days, mice were fasted for 3 h before onset of the dark phase by removal of the FED3 devices. At dark onset, mice were injected subcutaneously with either vehicle, daLUXendin660 (10 nmol kg−1) or tirzepatide (10 nmol kg−1), the FED3 devices were returned and food intake was recorded over a 24 h period. The study was performed using a three-way crossover design. A 1-week washout period was observed between experimental days. FED3 devices were checked, serviced and food was topped up daily. AD293 cells (RRID:CVCL_9804) were grown in DMEM High Glucose supplemented with 10% FBS, 100 units per ml penicillin and 100 μg ml−1 streptomycin and 1% glutamine (complete DMEM). Cells were transfected with 0.3 μl Lipofectamine 2000 per well and 50 ng DNA (SNAP_hGIPR or SNAP_hGLP1R plasmid) per well in 110 μl OptiMEM per well and incubated overnight at 37 °C, 5% CO2. Cells co-transfected with plasmids for two receptors harbouring orthogonal enzyme self-labels (SNAP_GIPR and Halo_GLP1R plasmid) received 50 ng of each DNA. Control, non-transfected cells were cultured overnight in 50 μl OptiMEM without the addition of Lipofectamine 2000 or DNA. The next day, culture media was removed and 50 μl of labelling solution or complete DMEM (for unlabelled control) was added to each well. On the day after transfection with SNAP-GIPR and Halo-GLP1R, cells were incubated in triple probe solutions of either 500 nM daLUXendin544 with 500 nM SBG-Oregon Green (SBG-OG) and 500 nM CA-Sulfo646, or 500 nM daLUXendin660 with 500nM SBG-OG and 500 nM CA-Sulfo549, all in 50 μl complete DMEM for 55 min at 37 °C, 5% CO2. MIN6-CB4 cells were generated and phenotyped, as previously described48. Cells were grown to 70% confluency in MIN6 media (DMEM High Glucose supplemented with 15% FBS, 71 μM 2-mercaptoethanol, 2 mM glutamine, 100 units ml−1 penicillin, 100 μg ml−1 streptomycin). Cells were then seeded in 100 μl MIN6 media per well, on a glass-bottomed 96-well imaging plate previously coated with poly-l-lysine 0.01%. The day after seeding, cells were incubated with either 500 nM daLUXendin660, 100 nM sGIP549 (GIPR probe)17 or 100 nM LUXendin551 (GLP1R probe)18, a combination of two or just complete MIN6 media for 1 h at 37 °C, 5% CO2. 16) mice (both on a C57BL/6J background) were used as tissue donors. Mice were humanely killed using a schedule-1 method, and the common bile duct was injected with Serva NB8 (1 mg ml−1) collagenase. Islets were used up to 4 days after isolation. GLP1RSNAP/SNAP islets were incubated with 500 nM BG-Sulfo549 or BG-Sulfo646 and/or another orthogonal fluorescent ligand for 1 h at 37 °C, 5% CO2, before washing three times. For immunostaining, labelled islets were washed three times with PBS, then incubated with 2–4% formalin for 15 min at room temperature (16–25 °C). Islets were again washed three times with PBS, then incubated in permeabilizing blocking buffer for 1 h at room temperature before incubation with primary antibody overnight at 4 °C. The following day, islets were incubated with secondary antibody for 2 h at room temperature before mounting onto microscope slides with VECTASHIELD HardSet with DAPI or Everbrite Hardset. Primary and secondary antibodies are listed in Supplementary Table 5. To assess brain and pancreas access following peripheral administration, GLP1R-Cre:tdRFP (GLP1R-tdRFP) and GIPR-Cre:GFP (GIPR-GFP) mice15 (both on a C57BL/N background) were injected intravenously into the tail vein with 100 nmol kg−1 daLUXendin. Then, 40 min later, mice were terminally anaesthetized with pentobarbital (200 mg kg−1) and perfused with ice-cold 0.1 M PBS followed by 4% formalin solution. For brain tissue analysis, GLP1R-tdRFP mice and GIPR-Cre:GCaMP6 (GIPR-GCaMP6) mice14 were terminally anaesthetized using ketamine hydrochloride (Ketavet, Zoetis; 75 mg kg−1, intraperitoneally) and medetomidine hydrochloride (Domitor, OrionPharma; 1 mg kg−1, intraperitoneally). Meloxicam (Metacam, Boehringer Ingelheim; 5 mg kg−1, subcutaneously) was given for peri-surgery analgesia. Mice were placed in a stereotaxic frame, and daLUXendin was injected unilaterally into the lateral ventricle at doses of 5 nmol kg−1 body weight at a rate of 1 µl min−1. At 1 h post injection, mice were perfused with ice-cold 0.1 M PBS followed by 4% formalin solution. All brains were cryo-sectioned coronally at 30 µm and mounted onto microscope slides with VECTASHIELD antifade mounting medium (H-1000). Brains were cryo-sectioned coronally at 30 µm and sections processed for amplification of fluorescent reporter signals by immunofluorescence labelling of vimentin (a marker for ependymal cells and tanycytes). Antigen retrieval of free-floating sections was performed using sodium citrate buffer at 80 °C for 20 min. Sections were next blocked in 5% normal donkey serum, 0.2% Triton X-100 for 1 h and then incubated overnight in primary antibody (in blocking solution) at room temperature (primary antibody: chicken anti-vimentin, Ab24525, Abcam; 1:750). Sections were next incubated for 2 h in secondary antibody (in 1% normal donkey serum, 0.2% Triton X-100) at room temperature (secondary antibody: donkey anti-chicken IgY (H+L) highly cross-adsorbed secondary antibody, Alexa Fluor 488, A78948, Thermo Fisher Scientific; 1:500). Sections were stained with DAPI to mark nuclei, then mounted on Superfrost Plus slides and cover-slipped using Prolong Antifade medium. Sections were imaged on an inverted Leica SP8 confocal microscope using either a ×25 or ×63 oil immersion objective. Three-dimensional reconstructions of daLUXendin660-labelled tissue or antibody-stained cells were rendered using the Surfaces function in Imaris (v.10.1.1) (Oxford Instruments). Opaque surfaces are three-dimensional reconstructions of overlapping areas of daLUXendin660 labelling and antibody staining created using the Coloc function in Imaris. Primary and secondary antibodies are listed in Supplementary Table 5. Midbrain sections (30 µm) containing the ARH from a GIPR-Cre:GCaMP6 (GIPR-GCaMP6) mouse intracerebroventricularly administered daLUXendin660 (3.3 nmol kg−1 body weight) were collected on Superfrost Plus slides and allowed to air-dry at room temperature for 1 h. Slides were then dipped in molecular-grade ethanol and further air-dried overnight at room temperature. RNAscope in situ hybridization was performed on these sections using the RNAscope Multiplex Fluorescent Kit (v.2) (Advanced Cell Diagnostics) as per the manufacturer's instructions, with a modification to the pre-treatment procedure (Protease IV incubation conducted for 20 min at room temperature). Sections were processed for in situ hybridization of Glp1r mRNA (cat. Following hybridization, slides were cover-slipped using Prolong Antifade medium. Human iPS cells (ALSTEM, iPS11) were maintained on iPS cell-qualified Matrigel (Corning, 354277) in mTeSR+ media (StemCell Technologies, 05826). ALSTEM iPS11 cells were authenticated at source by testing for expression of OCT4 and TRA-1-60, as well as alkaline phosphatase activity. Differentiation to islet-like structures (SC-islets) was carried out in a suspension-based, magnetic CELLSPIN bioreactor system (PFEIFFER). Most of the supernatant was removed by aspiration; fresh media was added and bioreactors were placed back on the stirrer system. SC-islet differentiation was based on published protocols35,49 Live or fixed cell and tissue imaging was performed with either (1) Zeiss LSM780/LSM880 meta-confocal microscopes equipped with GaAsP spectral detectors and ×40 and ×63/1.2 NA water objectives; (2) an Olympus FV3000 confocal microscope equipped with GaAsP spectral detectors and UPLSAPO ×60/1.30 NA silicone and ×60/1.41 NA oil objectives; or (3) an Olympus FV4000 confocal microscope equipped with SilVIR spectral detectors and a UPLSAPO ×60/1.41 NA oil objective. Zen 2012 or Olympus cellSens (v.3.2) were used for image acquisition and analysis. Islets were labelled with LUXendin645 (500 nM), sGIP648 (1 µM) or daLUXendin (1 µM), before fixation in 2–4% formalin for 15–30 min. Islets were mounted on cavity slides, submerged in STORM buffer (Abbelight) and sealed using a 170 µm coverslip and dental resin. Samples were imaged in HILO mode on an Evident/Abbelight SAFe 180 system, using a ×100/1.5 NA Olympus UPLAPO100XOHR objective. LUXendin645, sGIP648 or daLUXendin660 (all Cy5) were pumped to the dark state using an Oxxius laser combiner and 600 mW 640 nm Coherent Obis diode laser before initiation of photoblinking. Single-molecule events were recorded using an LP650 filter with an integration time of 50 ms on a Hamamatsu ORCA-Fusion sCMOS for 20,000–40,000 frames. Localizations were extracted and images reconstructed using Abbelight NEO software. Density-based spatial clustering of applications with noise (DBSCAN) was used to determine localization clustering, implemented in Abbelight NEO software (v.39), with ε = 25 nm (the average precision of the data) and minPts = 8. Results were confirmed using a custom RNA segmentation and DBSCAN routine implemented in R Project (v.4.4.3) (https://cran.r-project.org/web/packages/dbscan/index.html)50. Live confocal and STED imaging of CHO-K1:SNAP-GLP1R:Halo-GIPR cells (CHO-K1, RRID:CVCL_0214) was performed on a STEDYCON system (Abberior Instruments), mounted on a Nikon Eclipse TI research microscope equipped with a Plan APO Lambda ×100/1.45 NA oil objective (Nikon) and controlled by NIS Elements (Nikon). An incubation chamber surrounding the microscope setup was set to 37 °C 24 h before live-cell imaging. Imaging was performed in FluoroBrite Medium (Gibco) at 37 °C. Excitation was delivered with a 405, 488, 568 or 640 nm diode laser, and emission was detected with avalanche photodiodes at 461, 520, 566 or 671 nm, respectively. Pixel size was set to 30 nm × 30 nm, with 10 µs pixel dwell time, and a line accumulation of 1 was used during acquisition. STEDYCON 9.0.799-g22f03ed2 software was used for image acquisition and analysis. MIN6-CB4 cells were labelled with 500 pM daLUXendin660 in complete media for 20 min at 37 °C and washed three times in HEPES-bicarbonate buffer, ensuring sparse labelling of GLP1R/GIPR. Cells were mounted on cavity slides and imaged in HEPES-bicarbonate buffer supplemented with 11 mM d-glucose before TIRF imaging at 40 Hz using an Evident/Abbelight SAFe 180 system and a ×100/1.5 NA Olympus UPLAPO100XOHR objective. Single particles were recorded using an LP650 filter with an integration time of 25 ms on a Hamamatsu ORCA-Fusion sCMOS for 1,750 frames. Single particle analysis was performed using the Trackmate plugin for ImageJ51, with trajectories shown as maximum displacement. Probe labelling was analysed using corrected total cell fluorescence, which is integrated density − (area of selected cell × mean fluorescence of background readings)52,53. Colocalization was determined using Manders' coefficient, which calculates the proportion of pixels in one channel that also show intensity from both channels. Full width at half maximum was used to analyse labelling patterns at the membrane versus cytoplasm. All analyses were performed in ImageJ (NIH), with brightness and contrast linearly adjusted across the entire image for presentation purposes and applied equally between all states under examination. Data distribution was assessed using the D'Agostino–Pearson normality test. Pairwise comparisons were made using unpaired t-tests. For non-parametric data, pairwise comparisons were made using a Mann–Whitney test. Multiple comparisons were made using either one-way ANOVA (parametric), two-way ANOVA (parametric) or Kruskal–Wallis test (non-parametric), with pairwise comparisons made using either Šídák's or Dunn's tests, or the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli. GraphPad Prism 9 and 10 were used for all statistical analyses. Values are presented as means; error bars, s.e.m. ; P values less than 0.05 were considered significant. For cell line studies, the independent replicate corresponds to experiments performed on different days or using cells split from separate flasks. Variation between individual islets or cells was greater than variation of the mean between animals; therefore, they are considered separate data points for analysis purposes. If the n number occupies a range of samples or animals, the lowest value is provided as per journal guidelines. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Datasets generated and/or analysed during the current study are available from the corresponding authors upon request. Owing to their large size, individual raw image files are available upon request. daLUXendins are subject to a Material Transfer Agreement and the ability to manufacture and supply. All requests for reagents and data will be handled by J.B. or D.J.H., who will respond within 30 working days. Source data are provided with this paper. Liskiewicz, A. et al. Glucose-dependent insulinotropic polypeptide regulates body weight and food intake via GABAergic neurons in mice. Scrocchi, L. A. et al. Glucose intolerance but normal satiety in mice with a null mutation in the glucagon-like peptide 1 receptor gene. Turton, M. D. et al. A role for glucagon-like peptide-1 in the central regulation of feeding. Wilding, J. P. H. et al. Once-weekly semaglutide in adults with overweight or obesity. Miyawaki, K. et al. Inhibition of gastric inhibitory polypeptide signaling prevents obesity. Preserved incretin activity of glucagon-like peptide 1 [7–36 amide] but not of synthetic human gastric inhibitory polypeptide in patients with type-2 diabetes mellitus. Campbell, J. E. et al. TCF1 links GIPR signaling to the control of beta cell function and survival. The incretin co-agonist tirzepatide requires GIPR for hormone secretion from human islets. Structural determinants of dual incretin receptor agonism by tirzepatide. Adriaenssens, A. E. et al. Glucose-dependent insulinotropic polypeptide receptor-expressing cells in the hypothalamus regulate food intake. Identification and characterization of GLP-1 receptor-expressing cells using a new transgenic mouse model. Revealing the tissue-level complexity of endogenous glucagon-like peptide-1 receptor expression and signaling. Adriaenssens, A. et al. Hypothalamic and brainstem glucose-dependent insulinotropic polypeptide receptor neurons employ distinct mechanisms to affect feeding. Ast, J. et al. Super-resolution microscopy compatible fluorescent probes reveal endogenous glucagon-like peptide-1 receptor distribution and dynamics. Gray, S. M. et al. Discordance between GLP-1R gene and protein expression in mouse pancreatic islet cells. Median eminence blood flow influences food intake by regulating ghrelin access to the metabolic brain. Michau, A. et al. Metabolism regulates exposure of pancreatic islets to circulating molecules in vivo. Zhang, Y. et al. Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein. Liang, Y.-L. et al. Phase-plate cryo-EM structure of a biased agonist-bound human GLP-1 receptor–Gs complex. Structural insights into multiplexed pharmacological actions of tirzepatide and peptide 20 at the GIP, GLP-1 or glucagon receptors. Thorens, B. et al. Cloning and functional expression of the human islet GLP-1 receptor: demonstration that exendin-4 is an agonist and exendin-(9–39) an antagonist of the receptor. Mroz, P. A. et al. Optimized GIP analogs promote body weight lowering in mice through GIPR agonism not antagonism. Coskun, T. et al. LY3298176, a novel dual GIP and GLP-1 receptor agonist for the treatment of type 2 diabetes mellitus: from discovery to clinical proof of concept. Poc, P. et al. Interrogating surface versus intracellular transmembrane receptor populations using cell-impermeable SNAP-tag substrates. Birke, R. et al. Sulfonated red and far-red rhodamines to visualize SNAP- and Halo-tagged cell surface proteins. The long lifespan and low turnover of human islet beta cells estimated by mathematical modelling of lipofuscin accumulation. Shapiro, A. M. J. et al. Insulin expression and C-peptide in type 1 diabetes subjects implanted with stem cell-derived pancreatic endoderm cells in an encapsulation device. Ramzy, A. et al. Implanted pluripotent stem-cell-derived pancreatic endoderm cells secrete glucose-responsive C-peptide in patients with type 1 diabetes. Balboa, D. et al. Functional, metabolic and transcriptional maturation of human pancreatic islets derived from stem cells. Rapid sensing of circulating ghrelin by hypothalamic appetite-modifying neurons. Buenaventura, T. et al. Agonist-induced membrane nanodomain clustering drives GLP-1 receptor responses in pancreatic beta cells. Sungkaworn, T. et al. Single-molecule imaging reveals receptor–G protein interactions at cell surface hot spots. Roßmann, K. et al. N-Methyl deuterated rhodamines for protein labelling in sensitive fluorescence microscopy. El, K. et al. GIP mediates the incretin effect and glucose tolerance by dual actions on α cells and β cells. Expression and functional activity of glucagon, glucagon-like peptide I, and glucose-dependent insulinotropic peptide receptors in rat pancreatic islet cells. El, K. & Campbell, J. E. The role of GIP in alpha-cells and glucagon secretion. Imbernon, M. et al. Tanycytes control hypothalamic liraglutide uptake and its anti-obesity actions. Anton, S. E. et al. Receptor-associated independent cAMP nanodomains mediate spatiotemporal specificity of GPCR signaling. The role of β cell glucagon-like peptide-1 signaling in glucose regulation and response to diabetes drugs. A., White, E. & Kravitz, A. V. Feeding experimentation device (FED): a flexible open-source device for measuring feeding behavior. Establishment of a long-term stable β-cell line and its application to analyze the effect of Gcg expression on insulin secretion. Rezania, A. et al. Reversal of diabetes with insulin-producing cells derived in vitro from human pluripotent stem cells. A density-based algorithm for discovering clusters in large spatial databases with noise. Knowledge Discovery and Data Mining (eds Simoudis, E. et al.) 226–231 (AAAI Press, 1996). Ershov, D. et al. TrackMate 7: integrating state-of-the-art segmentation algorithms into tracking pipelines. Progressive activation of CyclinB1-Cdk1 coordinates entry to mitosis. Viloria, K. et al. Vitamin-D-binding protein contributes to the maintenance of α cell function and glucagon secretion. Single cell portal: an interactive home for single-cell genomics data. This research was funded, in whole or in part, by UK Research and Innovation (EP/X026833/1) (to D.J.H.). This project has received funding from the European Research Council under the European Union's Horizon 2020 research and innovation programme (Starting Grant 715884 to D.J.H.) and from the European Union's Horizon Europe Framework Programme (deuterON, grant agreement no. This work was supported on behalf of the ‘Steve Morgan Foundation Type 1 Diabetes Grand Challenge' by Diabetes UK and the Steve Morgan Foundation (grant number 23/0006627 to D.J.H. was supported by a Novo Nordisk–Oxford Fellowship. was supported by a Diabetes UK R.D. is a Royal Society Industry Fellow (INF/R2/212001) and, together with A.P., was supported by the Biotechnology and Biological Sciences Research Council (BB/W014831/1). was supported by MRC (MR/X003604/1) and the ‘EFSD European Research Programme on New Targets for Type 2 Diabetes supported by an educational research grant from MSD'. is supported by the Novo Nordisk Fonden/European Federation for the Study of Diabetes Future Leaders Award. is supported by an MRC Clinician Scientist Fellowship (MR/Y00132X/1), as well as project grants from the MRC and Diabetes UK. The Section of Endocrinology at Imperial College London is funded by grants from the MRC, National Institute for Health Research (NIHR) and is supported by the NIHR Biomedical Research Centre Funding Scheme and the NIHR/Imperial Clinical Research Facility. The Oxford Centre for Diabetes, Endocrinology and Metabolism at the University of Oxford is supported by the NIHR Oxford Biomedical Research Centre. The project involves an element of animal work not funded by the NIHR but by another funder, as well as an element focussed on patients and people appropriately funded by the NIHR. We thank R. Birke and I. Kretzschmer (both FMP) for technical support. For the purpose of Open Access, the author has applied a CC BY public copyright licence to any Author Accepted Manuscript version arising from this submission. Anne de Bray, Katrina Viloria, Daniela Nasteska, Julia Ast & David J. Hodson Anne de Bray, Sarah Armour, Jason Tong, Ali H. Shilleh, Katrina Viloria, Daniela Nasteska, Jeremy W. Tomlinson & David J. Hodson Centre for Cardiovascular and Metabolic Neuroscience, Department of Neuroscience, Physiology & Pharmacology, UCL, London, UK Anna G. Roberts, Wanqing Jiang, Natalie S. Figueredo Burgos, James P. P. Trott, Alice E. Adriaenssens & Stefan Trapp Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Berlin, Germany Christiane Huhn, Blaise Gatin-Fraudet, Kilian Roßmann, Tolga Soykan & Johannes Broichhagen Division of Stem Cell Regulation Research, Center for Medical Research and Education, Osaka University Graduate School of Medicine, Suita, Osaka, Japan Julia Ast, Malgorzata Cyranka, Alexey Epanchintsev & Carina Ämmälä Institute of Metabolic Science-Metabolic Research Laboratories & MRC-Metabolic Diseases Unit, University of Cambridge, Cambridge, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed experiments and analysed the data. validated and provided a cell line. conceived the studies, designed and supervised experiments and wrote the manuscript with input from all authors. Correspondence to Johannes Broichhagen or David J. Hodson. is an employee of AstraZeneca plc. University of Birmingham and Leibniz-Forschungsinstitut für Molekulare Pharmakologie have filed a patent (WO2024133236, ‘Peptide conjugates for labelling endogenous GIPR and GLP1R', inventors David J. Hodson and Johannes Broichhagen), which covers daLUXendins described in the present study. Oxford University Innovation filed a PCT patent application on 14 March 2025 (PCT/GB2025/050523, entitled ‘Combination therapy' and with the inventors being David J. Hodson, Katrina Viloria, Ali Shilleh and Martin Hewison) that encompasses elements of the work described in the present study. This PCT patent application is due to be published on 14 September 2025. and J.B. receive licensing revenue from Celtarys Research for the provision of GLP1R/GIPR chemical probes used in the present study, including LUXendins, sGIPs and daLUXendins. The remaining authors declare no competing interests. Nature Metabolism thanks Timo Müller and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Christoph Schmitt, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, b) daLUXendin544 labels SNAP_hGIPR:AD293 cells in a concentration-dependent manner (SBG-OG; cell impermeable SNAP label), shown by representative images (a), and line graph (b) (n = at least 7 wells, 2 independent repeats). c) 500 nM daLUXendin544 brightly labels SNAP_hGIPR:AD293 cells without signs of impaired morphology (n = at least 7 wells, 2 independent repeats). d) daLUXendin544 and daLUXendin660 do not label non-transfected AD293 cells (n = at least 9 wells, at least 2 independent repeats). f) As for e), but daLUXendin660 (n = 19 wells, 3 independent repeats). a) daLUXendin660 and LUXendin551 (LUX551; GLP1R probe) labelling co-localize in MIN6-CB4 beta cells, which endogenously express GLP1R (n = 16 images from 4 wells, 2 independent repeats). b) daLUXendin660 and sGIP549 (GIPR probe) co-localize in MIN6-CB4 beta cells, which endogenously express GIPR (n = 16 images from 4 wells, 2 independent repeats). d, e) daLUXendin544 labels membrane and cytoplasmic (that is, internalized) GLP1R pools in GLP1RSNAP/SNAP mouse islets, labelled using the cell impermeable SNAP label, BG-Sulfo646, and shown using full width at half maximum (FWHM) (line placement used for FHWM calculation is shown) (n = at least 18 islets from at least 5 mice). f) daLUXendin544 engages distinct GLP1R/GIPR pools in α cells versus β cells, with more membrane labelling in β cells (arrows show membrane labelling) (n = 37 islets from 3 mice). h) Analysis showing strong co-localization between daLUXendin544 and either GLP1R or GIPR, labelled using LUXendin645 (LUX645) and sGIP648, respectively (n = 43 islets from 2 mice). i, j) daLUXendin544 labelling is reduced in GLP1RKO/KO islets, and further reduced in GLP1RKO/KO islets + 1 µM GIP Aib2 (n = 37 islets from 2 mice) (Kruskal-Walllis test with two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). Scale bar = 33.7 µm in a and b; scale bar = 53 µm in c, d, f, g and j. No changes are seen in vehicle-treated islets. Exact p values are displayed on each graph. a) daLUXendin660 can be further modified to generate daLUXendin660+ by installing a C20 di-acid lipid chain at lysine position 20. b) cAMP responses to daLUXendin660 (daLUX660) and daLUXendin660+ (daLUX660+) in AD293 cells expressing either human (h)GLP1R, mouse (m)GLP1R, human (h)GIPR or mouse (m)GIPR (n = 4 independent repeats). c, d) Binding affinity of daLUXendin660+ against hGLP1R and hGIPR (c), with Kd values (d) in a separate bar graph (one-site fit specific binding) (n = 3 independent repeats). e-h) daLUX660+ labels islets similarly to daLUX660, with strength of labelling β cells > α cells = δ cells, as shown by representative images (e, g) and summary bar graphs (f, h) (INS, insulin = at least 26 islets from 5 mice; GCG, glucagon = at least 16 islets from 5 mice; SST, somatostatin = at least 16 islets from 5 mice) (Kruskal-Wallis test with Dunn's multiple comparisons test). Scatter plots and line graphs show mean ± SEM. Violin plots shows min-max and median. Exact p values are displayed on each graph. a) Intracerebroventricular (icv) injection of daLUXendin660 (3.3–5 nmol/kg BW) labels cells surrounding the 3rd ventricle. daLUXendin660 colocalizes with tanycytes (indicated by vimentin immunolabeling) following icv administration (n = 4 mice). b) 3D-rendering showing extensive colocalization of daLUXendin660 with vimentin in tanycytes surrounding the 3rd ventricle (box labelled with i indicates region of magnified image). Labelling of GLP1R- and GIPR-expressing cells (examples indicated by arrows) was characterized in c) GLP1R-tdRFP (n = 2) and d) GIPR-GCaMP6 (n = 2) mice. e) A small number of cells positive for both GIPR and Glp1r are labelled with daLUXendin660 following icv administration (indicated by arrow heads). Box labelled with i indicates region of magnified image, with ', '' and ''' representing Glp1r, GIPR and daLUXendin660 signals, respectively. Statistical Source Data for Supplementary Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. de Bray, A., Roberts, A.G., Armour, S. et al. Fluorescent GLP1R/GIPR dual agonist probes reveal cell targets in the pancreas and brain. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44160-025-00866-9'>Photobiocatalytic benzylic C–H acylation enabled by the synergy of a thiamine-dependent enzyme, an organophotocatalyst and hydrogen-atom transfer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 09:35:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Synthesis (2025)Cite this article Direct functionalization of abundant C(sp3)–H bonds is highly attractive. Photobiocatalysis offers promise for expanding enzyme reactivity but has been limited to the use of preactivated radical precursors. Key challenges for C(sp3)–H bond activation include the lack of robust activation modes for the inert bonds under biocatalytic conditions and controlling the reactivity and stereochemistry of prochiral radicals. Here we report a triple activation strategy enabling photobiocatalytic C(sp3)–H bond acylation with aldehydes. By combining hydrogen-atom transfer for prochiral radical formation, organic-dye-modulated single-electron transfer and an engineered thiamine-dependent enzyme, we describe a radical acyl transferase for functionalizing C(sp3)–H bonds. This robust radical enzymatic system achieves benzylic C(sp3)–H and aldehyde C(sp2)–H oxidative coupling in an enantioselective manner (up to 97% e.e. This is a preview of subscription content, access via your institution Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data are available in the main text or Supplementary Information. Crystallographic data for the structures reported in this article have been deposited at the Cambridge Crystallographic Data Centre, under deposition numbers CCDC 2355842 (3g), 2358465 (3k) and 2416063 (3q). Copies of the data can be obtained free of charge via https://www.ccdc.cam.ac.uk/structures/. Chen, K. & Arnold, F. H. Engineering new catalytic activities in enzymes. Google Scholar Reetz, M. T. Biocatalysis in organic chemistry and biotechnology: past, present, and future. Google Scholar Buller, R. et al. From nature to industry: harnessing enzymes for biocatalysis. Google Scholar Schwizer, F. et al. Artificial metalloenzymes: reaction scope and optimization strategies. Google Scholar Jain, S., Ospina, F. & Hammer, S. C. A new age of biocatalysis enabled by generic activation modes. JACS Au 4, 2068–2080 (2024). Google Scholar Kissman, E. N. et al. Expanding chemistry through in vitro and in vivo biocatalysis. Google Scholar Stephenson, C. R. J., Yoon, T. P. & MacMillan, D. W. C. (eds) Visible Light Photocatalysis in Organic Chemistry (Wiley-VCH, 2018). Harrison, W., Huang, X. & Zhao, H. Photobiocatalysis for abiological transformations. Google Scholar Emmanuel, M. A. et al. Photobiocatalytic strategies for organic synthesis. Google Scholar Xu, Y., Liu, F., Zhao, B. & Huang, X. Repurposing naturally occurring enzymes using visible light. Google Scholar Emmanuel, M. A., Greenberg, N. R., Oblinsky, D. G. & Hyster, T. K. Accessing non-natural reactivity by irradiating nicotinamide-dependent enzymes with light. Google Scholar Huang, X. et al. Photoinduced chemomimetic biocatalysis for enantioselective intermolecular radical conjugate addition. Google Scholar Chen, B. et al. Modular access to chiral amines via imine reductase-based photoenzymatic catalysis. Google Scholar Liu, Y. et al. Photoredox/enzymatic catalysis enabling redox-neutral decarboxylative asymmetric C–C coupling for asymmetric synthesis of chiral 1,2-amino alcohols. JACS Au 3, 3005–3013 (2023). Google Scholar Biegasiewicz, K. F. et al. Photoexcitation of flavoenzymes enables a stereoselective radical cyclization. Google Scholar Huang, X. et al. Photoenzymatic enantioselective intermolecular radical hydroalkylation. Google Scholar Peng, Y. et al. Photoinduced promiscuity of cyclohexanone monooxygenase for the enantioselective synthesis of α-fluoroketones. Google Scholar Duan, X. et al. A photoenzymatic strategy for radical-mediated stereoselective hydroalkylation with diazo compounds. Google Scholar Zhao, B. et al. Direct visible-light-excited flavoproteins for redox-neutral asymmetric radical hydroarylation. Google Scholar Shi, Q. et al. Single-electron oxidation-initiated enantioselective hydrosulfonylation of olefins enabled by photoenzymatic catalysis. Google Scholar et al. Enantioselective decarboxylative alkylation using synergistic photoenzymatic catalysis. Google Scholar Li, M., Yuan, Y., Harrison, W., Zhang, Z. & Zhao, H. Asymmetric photoenzymatic incorporation of fluorinated motifs into olefins. Google Scholar Cheng, L. et al. Stereoselective amino acid synthesis by synergistic photoredox-pyridoxal radical biocatalysis. Google Scholar Wang, T.-C. et al. Stereoselective amino acid synthesis by photobiocatalytic oxidative coupling. Google Scholar Ouyang, Y., Page, C. G., Bilodeau, C. & Hyster, T. K. Synergistic photoenzymatic catalysis enables synthesis of α-tertiary amino acids using threonine aldolases. Google Scholar Xu, Y. et al. A light-driven enzymatic enantioselective radical acylation. Google Scholar Xing, Z. et al. Synergistic photobiocatalysis for enantioselective triple radical sorting. Google Scholar Liu, X., Xu, S., Chen, H. & Yang, Y. Unnatural thiamine radical enzymes for photobiocatalytic asymmetric alkylation of benzaldehydes and α-ketoacids. ACS Catal. Google Scholar Tseliou, V. et al. Stereospecific radical coupling with a non-natural photodecarboxylase. Saint-Denis, T. G. et al. Enantioselective C(sp3)-H bond activation by chiral transition metal catalysts. Mondal, S. et al. Enantioselective radical reactions using chiral catalysts. Zhang, C., Li, Z.-L., Gu, Q.-S. & Liu, X.-Y. Catalytic enantioselective C(sp3)–H functionalization involving radical intermediates. Chakrabarty, S., Wang, Y., Perkins, J. C. & Narayan, A. R. H. Scalable biocatalytic C–H oxyfunctionalization reactions. Li, F., Zhang, X. & Renata, H. Enzymatic C–H functionalizations for natural product synthesis. Zhang, R. K. et al. Enzymatic assembly of carbon-carbon bonds via iron-catalysed sp3 C–H functionalization. Zhang, J. et al. Chemodivergent C(sp3)–H and C(sp2)–H cyanomethylation using engineered carbene transferases. Athavale, S. V. et al. Enzymatic nitrogen insertion into unactivated C–H bonds. Roy, S. et al. Stereoselective construction of β-, γ- and δ-lactam rings via enzymatic C–H amidation. Mao, R. et al. Biocatalytic, enantioenriched primary amination of tertiary C–H bonds. Rui, J. et al. Directed evolution of nonheme iron enzymes to access abiological radical-relay C(sp3)–H azidation. Zhao, Q. et al. Engineering non-haem iron enzymes for enantioselective C(sp3)–F bond formation via radical fluorine transfer. Zhao, L.-P. et al. Biocatalytic enantioselective C(sp3)–H fluorination enabled by directed evolution of non-haem iron enzymes. Zetzsche, L. E. et al. Biocatalytic oxidative cross-coupling reactions for biaryl bond formation. Mukherjee, P. et al. Enantiodivergent synthesis of isoindolones catalysed by a Rh(III)-based artificial metalloenzyme. Hailes, H. C. et al. Engineering stereoselectivity of ThDP-dependent enzymes. FEBS J. Vasilopoulos, A., Krska, S. W. & Stahl, S. S. C(sp3)–H methylation enabled by peroxide photosensitization and Ni-mediated radical coupling. Romero, N. A. & Nicewicz, D. A. Organic photoredox catalysis. Liu, K., Schwenzer, M. & Studer, A. Radical NHC catalysis. ACS Catal. Meng, Q.-Y., Lezius, L. & Studer, A. Benzylic C–H acylation by cooperative NHC and photoredox catalysis. Cao, J., Zhu, J. L. & Scheidt, K. A. Photoinduced cerium-catalyzed C–H acylation of unactivated alkanes. Byun, S. et al. Light‐driven enantioselective carbene‐catalyzed radical–radical coupling. Jana, S. & Cramer, N. Tunable thiazolium carbenes for enantioselective radical three-component dicarbofunctionalizations. Huan, L., Shu, X., Zu, W., Zhong, D. & Huo, H. Asymmetric benzylic C(sp3)–H acylation via dual nickel and photoredox catalysis. Bao, Y., Xu, Y. & Huang, X. Focused rational iterative site-specific mutagenesis (FRISM): a powerful method for enzyme engineering. Lu, Y.-C. et al. Photobiocatalytic enantioselective C(sp3)–H acylation enabled by thiamine-dependent enzymes via intermolecular hydrogen atom transfer. Download references This work was supported by the Jiangsu Basic Research Center for Synthetic Biology (grant no. We appreciate financial support from the National Key Research and Development Program of China (2022YFA0913000 to X.H. ), the National Natural Science Foundation of China (22277053 to X.H. ), the Natural Science Foundation of Jiangsu Province (BK20220760 to X.H. ), the Fundamental Research Funds for the Central Universities (0205/14380351 and 0205/14380346 to X.H.) and the Excellent Research Program of Nanjing University (ZYJH004 to X.H. These authors contributed equally: Xichao Peng, Jianqiang Feng, Fulu Liu, Wen-Zheng Xie. State Key Laboratory of Coordination Chemistry, Chemistry and Biomedicine Innovation Center (ChemBIC), School of Chemistry and Chemical Engineering, Frontier Interdisciplinary Science Research Center, Nanjing University, Nanjing, China Xichao Peng, Fulu Liu, Wen-Zheng Xie, Hailong Sun, Yuanyuan Xu, Yang Ming, Zhongqiu Xing, Yi-Tao Long & Xiaoqiang Huang State Key Laboratory of Physical Chemistry of Solid Surfaces, Fujian Provincial Key Laboratory of Theoretical and Computational Chemistry, College of Chemistry and Chemical Engineering, Xiamen University, Xiamen, China Jianqiang Feng & Binju Wang Institute of Molecular Engineering Plus, College of Chemistry, Fuzhou University, Fuzhou, China Jianqiang Feng Jiangsu Co-Innovation Center of Efficient Processing and Utilization of Forest Resources, International Innovation Center for Forest Chemicals and Materials, Nanjing Forestry University, Nanjing, China Yu Zheng Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the catalysis. performed most of the experiments. assisted in synthetic experiments. performed theoretical calculations. contributed to UV–vis spectroelectrochemical and EPR experiments. wrote the paper with input from all authors. coordinated and conceived the project. Correspondence to Binju Wang, Yi-Tao Long or Xiaoqiang Huang. The authors declare no competing interests. Nature Synthesis thanks Robert Kourist and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Thomas West, in collaboration with the Nature Synthesis team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Conditions: 1 (0.004 mmol), 2 (0.016 mmol), HAT reagent (0.020 mmol), enzyme (PfBAL_T481L, PfBAL_T481L-A480G or PfBAL_T481L-A480G-Y397A-W163C, 2 mol%), Eosin Y (3 mol%), 20% v/v DMSO in 100 mM MOPS buffer (pH 8.0, containing 2.5 mM MgSO4 and 0.15 mM ThDP) were stirred for 14 h at room temperature under N2 atmosphere with the irradiation of 450–460 nm LEDs; total volume of the reaction is 0.8 ml. The reactions were analysed by GC-MS. A representative snapshot was selected in the final stage of 200 ns MD trajectory, which resembles the most populated structures from the clustering analysis of the MD trajectory (Supplementary Table 25). MD, Molecular Dynamics. Supplementary Figs. 1–12, Tables 1–25 and Methods. X-ray crystallographic data for 3g, CCDC 2355842. X-ray crystallographic data for 3k, CCDC 2358465. X-ray crystallographic data for 3q, CCDC 2416063. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Peng, X., Feng, J., Liu, F. et al. Photobiocatalytic benzylic C–H acylation enabled by the synergy of a thiamine-dependent enzyme, an organophotocatalyst and hydrogen-atom transfer. Download citation Received: 05 November 2024 Accepted: 23 July 2025 Published: 19 August 2025 DOI: https://doi.org/10.1038/s44160-025-00866-9 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Synthesis ISSN 2731-0582 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250818102949.htm'>This overlooked trait could explain anxiety and depression in millions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-19 04:46:57
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers found there was a significant, positive relationship between the two, concluding that highly sensitive people are more likely to experience depression and anxiety compared to those who are less sensitive. In the study, sensitivity was defined as a personality trait that reflects people's capacity to perceive and process environmental stimuli such as bright lights, subtle changes in the environment and other peoples' moods. Often overlooked in mental health studies and clinical practice, which tend to focus on neuroticism and its association with mental health conditions, this research shows that understanding a person's sensitivity level is important and can have therapeutic implications. For example, people with more sensitive personality traits may be more likely to benefit from treatment plans which involve techniques such as applied relaxation and mindfulness, which can also prevent relapse. Tom Falkenstein, a psychotherapist and a PhD student at Queen Mary University of London, said: "This is the most extensive systematic review on sensitivity and mental health in adolescents and adults to date, and is the first ever meta-analysis on the topic to estimate the impact of this relationship. "In addition, our findings could help improve treatment for these individuals. Around 31% of the general population are considered highly sensitive, and, as our findings show, are more likely to respond better to some psychological interventions than less sensitive individuals. Therefore, sensitivity should be considered when thinking about treatment plans for mental health conditions. Our work shows it is crucial that the awareness of sensitivity is improved among mental health care professionals, so clinicians and practitioners can recognize the trait in their patients, and tailor treatment to their sensitivity." "This is the first meta-analysis providing robust evidence that highly sensitive people are more prone to common mental health problems. Our results provide further evidence that sensitive people are more affected by both negative and positive experiences and that the quality of their environment is particularly important for their well-being." The systematic review and meta analysis of 33 studies was carried out by an academic team from several universities including Queen Mary University and the University of Surrey. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            