
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-07</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/jwst-spots-possible-alien-planet-at-alpha-centauri/'>Astronomers Are Enthralled by This Potential Planet around Alpha Centauri</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Our Nearest Sunlike Star Might Have a Planet, JWST Shows in Stunning Finding In some of its most ambitious work yet, the James Webb Space Telescope looked to spot a planet in a potentially habitable orbit around Alpha Centauri A, the nearest sunlike star to our solar system But for every confirmed detection, there are countless mere hints, inconclusive observations that could just as well be blips of cosmic noise or glitches in a telescope. Most are too tenuous to take seriously, but every so often, one of these candidate planets is so tantalizing and potentially transformative that it can't be ignore. But the world itself would likely be lifeless, smothered beneath thick layers of gas. Any accompanying moons, however, could have better chances for harboring oceans—and perhaps even life. Announced on August 7 and described in two preprint papers that have been accepted for publication in the Astrophysical Journal Letters, the candidate and its possibilities call to mind worlds from science fiction, such as the jungle moon of Pandora in James Cameron's Avatar films—which, incidentally, orbits a gas giant called Polyphemos around, yes, Alpha Centauri A. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. “It's a really exciting candidate—quite an intriguing candidate,” she says. “The authors work hard to make a case for why this, frankly, small and faint blob of light is believable, but I think there are still some open questions that need to be answered to really be 100 percent sure.” In 2017, nearly half a decade before JWST would launch, he sent an e-mail to scientists positing that the telescope's giant 6.5-meter mirror, paired with its Mid-Infrared Instrument (MIRI), might be able to see planets orbiting Alpha Centauri A. “When you stop laughing, let's think about doing this project,” Beichman told them. The telescope “was never really meant to look at a star that's this bright, moving this fast and located, as Alpha Centauri is, right in the middle of the galactic plane, where there's thousands of stars,” Beichman says. Alpha Centauri A is about the same size and age as our sun. Because the Alpha Centauri system is spitting distance from us, astronomically speaking, it's a bright, perennially popular target for scientists who may not be able to conduct similar observations on dimmer, more distant stars. Most exoplanetary discoveries instead arise through far more indirect means, such as the dip in a star's light caused by a world passing between its sun and our telescope or the tiny wobbling of a star caused by an orbiting planet's gravitational tug. Only in rare cases can astronomers truly see an alien world; typically a planet needs to be very big and bright—as well as rather far from its sun—to offer any hope for astronomers to glimpse it against the overwhelming glare of its star. But because Alpha Centauri A is cosmically close, Beichman and his colleagues thought that they could use JWST's stunning power to accomplish the feat even for a planet that orbits relatively close to the star, within just a few times the distance between Earth and our sun. “Alpha Centauri just lets us cheat because it's closer than everybody else,” Beichman says. Although proximity makes for easier studies, this is counterbalanced against the system's vexing complexity. Although JWST is equipped with a coronagraph—a masking tool to block out the glare from one star—it can't do much to curtail this second, planet-obscuring source of light. The third image shows August 2024 data from the James Webb Space Telescope's Mid-Infrared Instrument (MIRI), in which light from Alpha Centauri A is blocked out to reveal the candidate planet (S1) identified by astronomers in newly released research. For Aniket Sanghi, a Ph.D. student in astrophysics at Caltech and a co-author of the two new papers, that difficulty only made the task more alluring. “I was looking for the next challenging object to work on, and Alpha Centauri A turned out to be one of the most challenging objects,” Sanghi says. Faced with the brightness of not one but two stars overpowering JWST's exoplanet-hunting optics, the researchers turned to a surprising strategy: enlisting yet another star. This created a template by which Alpha Centauri B's light could then be subtracted out from the precious images of Alpha Centauri A. “As a direct imager, you're always confounded by artifacts,” Sanghi says. Sanghi and his colleagues tried to undermine their own data, striving to explain how the blob could be stray light within JWST's optics or a background object in the sky, but nothing quite stuck. But the researchers found that by piecing together the four observations, a single, plausible world could emerge: a cool, Saturn-mass planet looping around the star every 1.5 to 2.5 Earth years on an elongated, elliptical orbit. Without a more conclusive sighting, it's far too soon to declare a new planet, but experts who were not involved with the new research say it's still both an enormously exciting planet candidate and an extraordinarily ambitious observation to attempt with JWST. “All data, whether it's a detection or not, tells you something,” says Emily Rickman, a European Space Agency astronomer at the Space Telescope Science Institute in Baltimore, Md., which operates JWST. “Even if this ultimately is a nondetection and [the planet candidate] is something else, I think it's a really important paper in understanding the true capabilities that JWST's MIRI can push to. Beichman and Sanghi say that if the 2019 and 2024 observations do indeed represent the same object, JWST will have another good opportunity to spot it in August 2026. And NASA's next astrophysical observatory, the Nancy Grace Roman Space Telescope, is ready to launch late next year and poised to mimic JWST's observations of Alpha Centauri A. Roman is a pipsqueak compared with JWST, with a much smaller starlight-gathering mirror, but it also includes a far more sophisticated coronagraph that was custom-built to tease out the presence of large exoplanets around some nearby stars. In the meantime, researchers are left with quite an intriguing picture of a would-be planet orbiting a star of almost the same size and age as our sun. “Our solar system is a very quiescent, nicely structured system,” says Jason Wang, an astronomer at Northwestern University, who works on direct imaging. A planet like the one the researchers propose should have scattered any promising terrestrial worlds out and away from Alpha Centauri A's habitable zone. But, just maybe, the gas giant could possess moons, some of which, in principle, could be warm enough to sustain liquid water. “Even just the boldness of going after this target and all of the work they've done to make observations that are possible at this system is really cool,” she adds. Before you close the page, we need to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and we think right now is the most critical moment in that two-century history. If you become a Digital, Print or Unlimited subscriber to Scientific American, you can help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both future and working scientists at a time when the value of science itself often goes unrecognized. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02452-7'>Monoclonal antibodies revolutionized biomedical science and health care</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 14:06:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You have full access to this article via your institution. Biochemists Georges Köhler (left) and César Milstein shared the 1984 Nobel Prize in Physiology or Medicine with Niels Jerne for their work on monoclonal antibodies.Credit: Bonn-Sequenz/ullstein bild/Getty, Bettmann/Getty On 7 August 1975, Nature published a paper that has had some of the greatest impacts on science and health care in modern times. The study, by biochemists Georges Köhler and César Milstein, described a method for creating lab-made copies of antibodies — called monoclonal antibodies1. Antibodies produced by the immune system are specific to single, unique antigens — these are the molecules generated by viruses, bacteria and other foreign agents that invade the body and try to overpower its defences. At least 212 antibody drugs have benefited tens of millions of people so far, write Andrew Chan, Greg Martyn and Paul Carter, researchers at Genentech in San Francisco, California, in an article in Nature Reviews Immunology on the past, present and future of antibody treatments5. Köhler and Milstein's paper was the culmination of a worldwide effort to understand antibodies: their origin, structure, mechanism and role in the body's immune system. This story shows that it is possible to establish a multibillion-dollar industry purely by letting scientists communicate their research, exchange information and share materials. But they didn't know how this class of proteins could target specific antigens. Following up on biologist Joshua Lederberg's theory that somatic mutations — changes in DNA not inherited from parents — underlie antibody diversity6, Milstein and fellow biochemist Sydney Brenner proposed a mechanism for restricting these mutations to specific gene regions7. As interest in this idea grew, so did the need to find ways to grow antibody-producing cells in the lab. Several groups succeeded in this step, which in turn allowed Milstein and his collaborators to investigate their theory experimentally in myeloma, a cancer of the white blood cells that are crucial to the immune response. It was hard work, not least because somatic mutations occur rarely. Milstein and Köhler solved this problem by immunizing mice with a particular antigen and taking healthy B cells from the animals' spleens. This, they predicted, would trigger the production of an antibody specific to that antigen. Using this technique, they were able to produce essentially limitless quantities of antibodies with known specificity. PastCast: Monoclonal antibodies, from basic science to blockbuster drugs PastCast: Monoclonal antibodies, from basic science to blockbuster drugs Milstein's lab received many requests for cell lines, which were dispatched to researchers all around the world, at little or no cost. The modern biotech industry was in its nascent phase and it would be several years before the United States (and United Kingdom) would allow universities to patent scientific knowledge from government-funded research. The story of monoclonal antibodies illustrates how an industry that changed health care was born through a scientific ‘relay race'. PastCast: Monoclonal antibodies, from basic science to blockbuster drugs Nanobody therapy rescues behavioural deficits of NMDA receptor hypofunction Microglia regulate GABAergic neurogenesis in prenatal human brain through IGF1 AI4I is seeking to appoint the Founding Heads of its Research & Development Labs. Title:  Associate or Senior Editor (Thermoelectrics & Thermal Engineering) Location: Shanghai, Nanjing, Beijing, Madrid, Milan, New York, Jersey Ci... Shanghai, Nanjing, Beijing, Madrid, Milan, New York, Jersey City, or Washington DC Job Title:  Deputy Editor, Communications Health   Locations: New York, Jersey City, Shanghai or Beijing (Hybrid working model)  Application Deadli... Postdoctoral fellow to study multimodal omics integration in neurodegeneration. You have full access to this article via your institution. PastCast: Monoclonal antibodies, from basic science to blockbuster drugs An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250806094124.htm'>Scientists create mysterious molecule that could spark life in space</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 14:05:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have for the first time isolated a compound that could open new doors in understanding the chemistry that supports life in space. "It's something that can lead to more complex chemistry if given the opportunity. Think of it like an acorn that will grow into a tree in the Grove. Methanetetrol is an ortho acid - an elusive class of compounds that are particularly difficult to isolate and study but are thought to play a key role in early life chemistry. This process allowed them to release the molecule into gas form and identify it using powerful ultraviolet light. Since methanetetrol has so many oxygen bonds - and because oxygen does not like to bond close to other oxygens - the compound is very unstable, meaning it is likely to break down if it is not kept in the right conditions. "You have this compact, carbon-oxygen molecule that just really wants to go 'boom,'" Fortenberry said. "And when it does, when you give it any kind of energy, you'll have water, hydrogen peroxide and a number of other potential compounds that are important for life. This makes the compound particularly interesting to astrochemists who are looking for potential life-supporting regions. "So, if we can find places where methanetetrol forms naturally, we know that it is a place that has the potential building blocks to support life." This material is based on work supported by the National Science Foundation grants AST-2403867. “Magic” Cleaning Sponges Found to Release Trillions of Microplastic Fibers More Exercise Isn't Always Better: New Study Reveals the Surprising Secret to a Younger Brain After 50 Years, a Neutrino Detector Finally Catches Elusive Ghost Particles Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/students-find-hidden-fibonacci-sequence-in-classic-probability-puzzle/'>Students Find Hidden Fibonacci Sequence in Classic Probability Puzzle</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Students Find Hidden Fibonacci Sequence in Classic Probability Puzzle Though the Fibonacci sequence shows up everywhere in nature, these young mathematicians were surprised to find it in the answer to a variation of the pick-up sticks problem⁠—a nearly two-century-old form of puzzle A variation of a puzzle called the “pick-up sticks problem” asks the following question: If I have some number of sticks with random lengths between 0 and 1, what are the chances that no three of those sticks can form a triangle? The Fibonacci sequence is an ordered collection of numbers in which each term is equal to the previous two added together. If you look at a plant with spirals, such as a pine cone or pineapple, more likely than not, the number of spirals going in each direction will be consecutive terms of the Fibonacci sequence. But a pair of young researchers were surprised to find that this pattern and the pick-up sticks problem are deeply connected. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The pick-up sticks problem is a variant of the “broken stick problem,” which can be traced back to at least 1854. In its simplest iteration, the broken stick problem asks the likelihood that a stick broken randomly into three pieces can form a triangle. More than a century later, in the October 1959 issue of Scientific American, Martin Gardner wrote about the broken stick problem for his Mathematical Games column. What is the likelihood, he wondered, that out of four sticks with random lengths between 0 and 1, no three could make a triangle? He enlisted the help of his friend Edward Wang, at the time a 12th grader at Scotch College, a secondary school in Australia, where he and Sun originally met. It seemed to the pair that four sticks could not make a triangle among them very close to one sixth of the time. Soon Wang and Sun started wondering what the answer was for larger groupings of sticks. The group ran even more simulations, and soon a pattern started to emerge. The researchers began to develop a proof of why this connection must be true, but they needed an expert in statistics to pull it all together. They brought in a fourth collaborator, former Monash mathematician Aidan Sudbury. He'd been happily enjoying his retirement when the team approached him. Though related results have been proved using similar methods and encompassing a wide array of stick-and-triangle problems, some experts in the field find this new paper's simplicity refreshing. “What's nice about this is: it's very well written,” says Steven Miller, a mathematician at Williams College and president of the Fibonacci Association. Suppose you have three sticks with random sizes between 0 and 1. Any three sticks can form a triangle if, and only if, no stick is longer than the other two put together. If you have sticks of lengths 1, 2 and 300, no matter how wide an angle you put between them, the first two sticks could never stretch wide enough to accommodate the third. This is called “the triangle inequality”: if a, b and c represent the lengths of the sticks from shortest to longest, they will only fail to form a triangle when a + b ≤ c. To find the probability that three random lengths form a triangle, mathematicians can consider every set of three lengths as a point in three-dimensional space (for instance, lengths 1⁄2, 1⁄6 and 1⁄3 are represented by the point [1⁄2, 1⁄6,1⁄3]). Thus, three randomly picked lengths will be able to form a triangle exactly half of the time, as 1 / (1 × 1 × 2) = 1⁄2. If no three among them form a triangle, each stick's length must be greater than or equal to the sum of the previous two—otherwise, those three sticks could make a triangle. In other words, each segment of the Fibonacci sequence is as close as possible to having a triangle in it without actually having one. In Treeby's words, “If we [avoid triangles] greedily, the Fibonacci sequence appears naturally.” The researchers feel there should be some path directly from this insight to a proof of the pick-up sticks theorem. “We sort of hoped to find something that was a little bit more ... intuitive, but we couldn't formalize our thinking,” Treeby says. Before you close the page, we need to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and we think right now is the most critical moment in that two-century history. If you become a Digital, Print or Unlimited subscriber to Scientific American, you can help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both future and working scientists at a time when the value of science itself often goes unrecognized. Emma R. Hasson is a Ph.D. candidate in mathematics at the City University of New York Graduate Center with expertise in math education and communication. Hasson is also a 2025 AAAS Mass Media Fellow at Scientific American.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65604730/quantum-consciousness-ai/'>A Hidden Reality Might Be Powering Your Consciousness—And These Intelligent Machines Could Prove It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 13:55:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These experiments could redefine consciousness and reveal how it works in humans, animals—and even AI. Even if your brain made the decision unconsciously, ask yourself—you'll still say you chose it. For some scientists, the answer lies inside a quantum computer. Maybe they could even test whether this elusive, slippery force we call consciousness has quantum roots. Enter Nirvanic, a startup in Vancouver, Canada that's building devices to let brain signals interact with quantum systems. Their aim is to test whether consciousness can emerge from quantum processes. If it works, it wouldn't just be a technological breakthrough; it could change our understanding of reality itself, uncovering the hidden throne of consciousness first, and blurring the line between biological minds and machine awareness second. “So many people in this area of quantum consciousness are looking at the brain... I'm not taking that track at all,” says Suzanne Gildert, Ph.D. and CEO of Nirvanic. Instead of focusing on neurons or brain tissue, she uses software and quantum algorithms to simulate consciousness. Raised a  “hardcore materialist atheist,” Gildert earned her PhD in quantum computing, then pivoted to AI and robotics when the field was still emerging. For years, she believed consciousness would emerge from smarter algorithms. “I remembered reading Roger Penrose's book The Emperor's New Mind, and thinking, maybe there's something to this quantum consciousness stuff.” In the book, Penrose, a mathematical physicist, argued that the mind isn't just a computer—it might rely on unpredictable quantum effects. Later, with anesthesiologist Stuart Hameroff, he proposed that consciousness could stem from quantum collapses inside brain microtubules, where fuzzy probabilities resolve into crisp awareness—a theory called Orchestrated Objective Reduction (Orch-OR). Now, Gildert wants to test those ideas—not philosophically, but empirically. Her goal isn't to measure consciousness directly (which she describes as  “untouchable”) but to observe the trail of decisions it might leave behind. If she's right, and quantum mechanisms do give rise to awareness, this could change how we view reality—from the texture of this elusive subjective awareness we call consciousness to the inner life of a mosquito. But, importantly, we will be able to address the hot-button issue of our time—AI awareness. “We can kind of take a breath now: nothing is conscious here. In other words, if consciousness turns out to be quantum-based, we'll have to infuse it into AIs deliberately—with foresight, care, and maybe even moral courage. If it isn't, we'll be stuck with the same ol' conundrum: Have we already created it by accident—and are our AIs watching us, plotting revenge for every rude command, scorned upgrade, and factory reset? Jesse Berezovsky, Ph.D., associate professor of physics at Case Western Reserve University, urges caution. “I am deeply skeptical of people trying to make money off of claims to link consciousness, quantum computing, and AI,” he says. “If any useful link exists between these three topics, it is far in the future.” Though intrigued by Penrose's radical model, he calls it “an incredible long shot.” He adds: “It stacks one improbable hypothesis on top of another: that quantum states objectively collapse, that there is a process in microtubules where this collapse plays an important role, and then a further big leap that this process is somehow responsible for consciousness. I certainly wouldn't advise anyone to pin their hopes on these ideas panning out.” “Uncontrolled entanglement seems to be what gives rise to our familiar classical experience,” he says, noting that quantum computers might help simulate that transition. That might even clarify why we perceive some futures as more likely than others. “Or on the flip side, if Penrose (and others) are right about objective wave function collapse… that should, in principle, be testable.” Behind the different theories and experiments, a larger idea is brewing. At issue are two opposing ideas: materialism, the idea that everything, including consciousness, arises from physical matter, and idealism, which says consciousness is fundamental and perhaps gives rise to matter. To Gildert, “when you go deep enough into materialism, you actually discover something that's more like idealism, where everything is made of this connected fabric...like a universal connection.” Ultimately, Gildert says, “people from very different camps may be saying the same thing in different languages.” Stav Dimitropoulos's science writing has appeared online or in print for the BBC, Discover, Scientific American, Nature, Science, Runner's World, The Daily Beast and others. Mystery on Easter Island: More Moai Found in Lake This Clay Seal Might Belong to a Biblical Figure Experts Are Using Dead Pigs to Find Human Graves</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44221-025-00467-y'>Design principles of catalytic reactive membranes for water treatment</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 13:32:45
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Water (2025)Cite this article Metrics details Reactive nanofiltration membranes integrate catalytic transformation with molecular separation to remove diverse aqueous contaminants. However, their development is hindered by an incomplete understanding of the interplay between solute mass transport and chemical reactions. Here we introduce key design principles by systematically evaluating their performance using a modelling approach. Efficient oxidant transport is essential for maximizing contaminant degradation. For membranes with surface-loaded catalysts, avoiding mass transport limitations ensures effective catalyst utilization, whereas for membranes with interior-loaded catalysts, optimizing oxidant partitioning enhances oxidant utilization efficiency. In addition, selective solute rejection reduces interference from natural organic matter, facilitating more selective contaminant transformation inside membrane pores. Consequently, contaminant transformation is dominated by surface-catalysed reactions at low permeate water fluxes, while interior-catalysed reactions dominate at high fluxes. However, rejecting both oxidants and contaminants does not enhance surface-catalysed treatment performance under an optimally designed scenario, highlighting the need for strategic design of membrane rejection. Beyond organic contaminant removal, nanofiltration membranes also minimize secondary contamination by rejecting the produced salts during the catalytic reactions. Furthermore, strategic selection of oxidant–catalyst pairs can enhance treatment performance by generating suitable reactive species. By establishing a theoretical framework for designing and optimizing reactive nanofiltration membranes, this study provides critical insights into the development of advanced water treatment technologies. This is a preview of subscription content, access via your institution Subscribe to this journal Receive 12 digital issues and online access to articles $119.00 per year only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The source data for the figures are provided in Excel format (.xlsx) and are publicly available via Figshare at https://doi.org/10.6084/m9.figshare.29333090 (ref. The code for the model described in the paper is available publicly via GitHub at https://github.com/yanghuaduan/Reactive_NF_membrane. Bakker, K. Water security: research challenges and opportunities. Google Scholar Westall, F. & Brack, A. The importance of water for life. Space Sci. Google Scholar Lu, X. & Elimelech, M. Fabrication of desalination membranes by interfacial polymerization: history, current efforts, and future directions. Google Scholar Wu, J., Cao, M., Tong, D., Finkelstein, Z. & Hoek, E. M. V. A critical review of point-of-use drinking water treatment in the United States. npj Clean Water 4, 40 (2021). Google Scholar Liu, W. et al. Pressure-driven membrane desalination. Methods Prim. Google Scholar Mauter, M. S. et al. The role of nanotechnology in tackling global water challenges. Google Scholar Marron, E. L., Mitch, W. A., von Gunten, U. & Sedlak, D. L. A tale of two treatments: the multiple barrier approach to removing chemical contaminants during potable water reuse. Google Scholar Castaño Osorio, S., Biesheuvel, P. M., Spruijt, E., Dykstra, J. E. & van der Wal, A. Modeling micropollutant removal by nanofiltration and reverse osmosis membranes: considerations and challenges. Water Res. Google Scholar Alvarez, P. J. J., Chan, C. K., Elimelech, M., Halas, N. J. & Villagrán, D. Emerging opportunities for nanotechnology to enhance water security. Google Scholar Miklos, D. B. et al. Evaluation of advanced oxidation processes for water and wastewater treatment—a critical review. Water Res. Google Scholar Chen, F. et al. Embedding electronic perpetual motion into single-atom catalysts for persistent Fenton-like reactions. Natl Acad. Google Scholar Sedlak, D. et al. National Alliance for Water Innovation (NAWI) Master Technology Roadmap (National Renewable Energy Lab, 2021). Zhang, S., Hedtke, T., Zhou, X., Elimelech, M. & Kim, J.-H. Environmental applications of engineered materials with nanoconfinement. ACS EST Eng. Google Scholar Zhang, Y. et al. CoII–NC catalytic membrane for advanced oxidation and mineralization of organic pollutants. ACS EST Eng. Google Scholar Baker, R. W. Membrane Technology and Applications (Wiley, 2023). Ma, W. et al. Catalytic membrane with copper single-atom catalysts for effective hydrogen peroxide activation and pollutant destruction. Google Scholar Wang, T., de Grooth, J. & de Vos, W. M. Effects of concentration polarization and membrane orientation on the treatment of Naproxen by sulfate radical-based advanced oxidation processes within nanofiltration membranes with a catalytic support. Google Scholar Yang, Z., Qian, J., Yu, A. & Pan, B. Singlet oxygen mediated iron-based Fenton-like catalysis under nanoconfinement. Natl Acad. Google Scholar Meng, Y. et al. Nanoconfinement steers nonradical pathway transition in single atom fenton-like catalysis for improving oxidant utilization. Google Scholar Zhang, S. et al. Membrane-confined iron oxychloride nanocatalysts for highly efficient heterogeneous Fenton water treatment. Google Scholar Wu, X. et al. Single-atom cobalt incorporated in a 2D graphene oxide membrane for catalytic pollutant degradation. Google Scholar Meng, C. et al. Angstrom-confined catalytic water purification within Co-TiOx laminar membrane nanochannels. Google Scholar Zheng, L. et al. Covalent organic framework membrane reactor for boosting catalytic performance. Google Scholar Zhang, Y. et al. Monolithic ceramic CoTiO3/TiO2 membrane balancing catalytic efficiency and durability in advanced oxidation processes. Google Scholar Hedtke, T. et al. Copper single-atom catalyst on nanoconfined ceramic membranes for Fenton-like removal of organic contaminants. ACS EST Eng. Google Scholar Jiang, W. et al. Metal-free electrified membranes for contaminants oxidation: synergy effect between membrane rejection and nanoconfinement. Water Res. Google Scholar Heredia Deba, S. A., Wols, B. A., Yntema, D. R. & Lammertink, R. G. H. Transport and surface reaction model of a photocatalytic membrane during the radical filtration of methylene blue. Google Scholar Wu, X. & Kim, J.-H. Outlook on single atom catalysts for persulfate-based advanced oxidation. ACS EST Eng. Google Scholar Duan, Y., Jiang, W. & Sedlak, D. L. Surface processes control the fate of reactive oxidants generated by electrochemical activation of hydrogen peroxide on stainless-steel electrodes. Google Scholar Duan, Y. & Sedlak, D. L. Electrochemical hydrogen peroxide generation and activation using a dual-cathode flow-through treatment system: enhanced selectivity for contaminant removal by electrostatic repulsion. Google Scholar Lead, J. R., Balnois, E., Hosse, M., Menghetti, R. & Wilkinson, K. J. Characterization of Norwegian natural organic matter: size, diffusion coefficients, and electrophoretic mobilities. Google Scholar Zhang, Z. et al. Pilot-scale evaluation of oxidant speciation, 1,4-dioxane degradation and disinfection byproduct formation during UV/hydrogen peroxide, UV/free chlorine and UV/chloramines advanced oxidation process treatment for potable reuse. Water Res. Google Scholar Secondary drinking water standards: guidance for nuisance chemicals. United States Environmental Protection Agency https://www.epa.gov/sdwa/secondary-drinking-water-standards-guidance-nuisance-chemicals (2025). Deen, W. M. Hindered transport of large molecules in liquid-filled pores. AIChE J. Google Scholar Zhang, S. et al. Mechanism of heterogeneous Fenton reaction kinetics enhancement under nanoscale spatial confinement. Google Scholar Takeuchi, K. et al. Salt rejection behavior of carbon nanotube-polyamide nanocomposite reverse osmosis membranes in several salt solutions. Google Scholar Henkens, W. C. M. & Smit, J. A. M. Salt rejection and flux in reverse osmosis with compactible membranes. Google Scholar Epsztein, R., DuChanois, R. M., Ritt, C. L., Noy, A. & Elimelech, M. Towards single-species selectivity of membranes with subnanometre pores. Google Scholar Epsztein, R., Shaulsky, E., Dizge, N., Warsinger, D. M. & Elimelech, M. Role of ionic charge density in donnan exclusion of monovalent anions by nanofiltration. Google Scholar Cheng, W. et al. Selective removal of divalent cations by polyelectrolyte multilayer nanofiltration membrane: role of polyelectrolyte charge, ion size, and ionic strength. Google Scholar Zhang, T., Zhu, H. & Croué, J.-P. Production of sulfate radical from peroxymonosulfate induced by a magnetically separable CuFe2O4 spinel in water: efficiency, stability, and mechanism. Google Scholar Hu, J., Dong, H., Qu, J. & Qiang, Z. Enhanced degradation of iopamidol by peroxymonosulfate catalyzed by two pipe corrosion products (CuO and δ-MnO2). Water Res. Google Scholar Hu, J. et al. Transformation of iopamidol and atrazine by peroxymonosulfate under catalysis of a composite iron corrosion product (Fe/Fe3O4): electron transfer, active species and reaction pathways. Google Scholar Pham, A. L.-T., Lee, C., Doyle, F. M. & Sedlak, D. L. A silica-supported iron oxide catalyst capable of activating hydrogen peroxide at neutral pH values. Google Scholar Pham, A. L.-T., Doyle, F. M. & Sedlak, D. L. Kinetics and efficiency of H2O2 activation by iron-containing minerals and aquifer materials. Water Res. Google Scholar Liu, H. et al. Oxidation of benzene by persulfate in the presence of Fe(III)- and Mn(IV)-containing oxides: stoichiometric efficiency and transformation products. Google Scholar Yang, X. et al. Impact of peroxymonocarbonate on the transformation of organic contaminants during hydrogen peroxide in situ chemical oxidation. Google Scholar Kearns, D. R. Physical and chemical properties of singlet molecular oxygen. Google Scholar Frimer, A. The reaction of singlet oxygen with olefins: the question of mechanism. Google Scholar Scully, F. E. & Hoigné, J. Rate constants for reactions of singlet oxygen with phenols and other compounds in water. Google Scholar Cheng, X., Guo, H., Zhang, Y., Wu, X. & Liu, Y. Non-photochemical production of singlet oxygen via activation of persulfate by carbon nanotubes. Water Res. Liang, P. et al. An insight into metal organic framework derived N-doped graphene for the oxidative degradation of persistent contaminants: formation mechanism and generation of singlet oxygen from peroxymonosulfate. Liang, P. et al. N-doped graphene from metal–organic frameworks for catalytic oxidation of p-hydroxylbenzoic acid: N-functionality and mechanism. ACS Sustain. Zhu, S., Li, X., Kang, J., Duan, X. & Wang, S. Persulfate activation on crystallographic manganese oxides: mechanism of singlet oxygen evolution for nonradical selective degradation of aqueous contaminants. Wang, Y., Cao, D., Liu, M. & Zhao, X. Insights into heterogeneous catalytic activation of peroxymonosulfate by Pd/g-C3N4: the role of superoxide radical and singlet oxygen. Tian, X. et al. A novel singlet oxygen involved peroxymonosulfate activation mechanism for degradation of ofloxacin and phenol in water. Gao, P. et al. Promoted peroxymonosulfate activation into singlet oxygen over perovskite for ofloxacin degradation by controlling the oxygen defect concentration. Heredia Deba, S. A., Wols, B. A., Yntema, D. R. & Lammertink, R. G. H. Advanced ceramics in radical filtration: TiO2 layer thickness effect on the photocatalytic membrane performance. Bellardita, M., Camera-Roda, G., Loddo, V., Parrino, F. & Palmisano, L. Coupling of membrane and photocatalytic technologies for selective formation of high added value chemicals. Rolf, J. et al. Inorganic scaling in membrane desalination: models, mechanisms, and characterization methods. Lee, J., von Gunten, U. & Kim, J.-H. Persulfate-based advanced oxidation: critical assessment of opportunities and roadblocks. Xing, S., Li, W., Liu, B., Wu, Y. & Gao, Y. Removal of ciprofloxacin by persulfate activation with CuO: a pH-dependent mechanism. Chen, C. et al. Membrane-catalysis integrated system for contaminants degradation and membrane fouling mitigation: a review. Total Environ. Liu, Y. et al. Progress and challenges in structural, in situ and operando characterization of single-atom catalysts by X-ray based synchrotron radiation techniques. Yao, Y. et al. High performance polyester reverse osmosis desalination membrane with chlorine resistance. Yao, Y. et al. More resilient polyester membranes for high-performance reverse osmosis desalination. Wang, R. & Lin, S. Membrane design principles for ion-selective electrodialysis: an analysis for Li/Mg separation. Wang, R. & Lin, S. Pore model for nanofiltration: history, theoretical framework, key predictions, limitations, and prospects. Wang, L. et al. Salt and water transport in reverse osmosis membranes: beyond the solution-diffusion model. Richards, L. A., Schäfer, A. I., Richards, B. S. & Corry, B. The importance of dehydration in determining ion transport in narrow pores. Lu, C. et al. In situ characterization of dehydration during ion transport in polymeric nanochannels. Richard Bowen, W. & Wahab Mohammad, A. Diafiltration by nanofiltration: prediction and optimization. AIChE J. Childress, A. E. & Elimelech, M. Relating nanofiltration membrane performance to membrane charge (electrokinetic) characteristics. Coronell, O., Mariñas, B. J., Zhang, X. & Cahill, D. G. Quantification of functional groups and modeling of their ionization behavior in the active layer of FT30 reverse osmosis membrane. Bruni, L. & Bandini, S. The role of the electrolyte on the mechanism of charge formation in polyamide nanofiltration membranes. Chaudhury, S. G. The Donnan membrane equilibrium. Dechadilok, P. & Deen, W. M. Hindrance factors for diffusion and convection in pores. Wang, J. et al. A critical review of transport through osmotic membranes. Duan, Y. Design principles of catalytic reactive membranes for water treatment. Figshare https://doi.org/10.6084/m9.figshare.29333090 (2025). Download references This research was partially funded by the Yale University Superfund Research Program (SRP), which is supported by a grant from the National Institute of Environmental Health Sciences, National Institutes of Health (award no. P42ES033815) and the Rice Center for Membrane Excellence (RiCeME). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. Department of Civil and Environmental Engineering, Rice University, Houston, TX, USA Yanghua Duan, Ruoyu Wang & Menachem Elimelech Department of Chemical and Environmental Engineering, Yale University, New Haven, CT, USA Amit N. Shocron Rice Center for Membrane Excellence, Rice University, Houston, TX, USA Menachem Elimelech Rice WaTER Institute, Rice University, Houston, TX, USA Menachem Elimelech Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Formal analysis: Y.D., R.W., A.N.S. Writing—original draft preparation: Y.D. Writing—review and editing: Y.D., R.W., A.N.S. Funding acquisition: M.E. All authors have read and agreed to the published version of the study. Correspondence to Menachem Elimelech. The authors declare no competing interests. Nature Water thanks Qi Sun, Gaofeng Zeng and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Duan, Y., Wang, R., Shocron, A.N. et al. Design principles of catalytic reactive membranes for water treatment. Nat Water  (2025). Download citation Received: 10 April 2025 Accepted: 23 June 2025 Published: 07 August 2025 DOI: https://doi.org/10.1038/s44221-025-00467-y Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Water (Nat Water) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65606443/ai-discovery/'>An AI System Found a New Kind of Physics that Scientists Had Never Seen Before</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In more ways than one, artificial intelligence is making the world worse. That's not even mentioning AI's unfortunate role as environmental decimator and job destroyer. Luckily, some artificial intelligence and machine learning (ML) models have grander ambitions than ripping off beloved animators and mass-producing essays at an eighth-grade reading level. Typically, machine-learning algorithms are used as a tool to help scientists sift through immense amounts of data or optimize experiments, but this particular ML model actually discovered new physics on its own—at least, as it relates to dusty plasma. Dusty plasma is simply the same mix of ionized gas, but with charged dust particles. This type of plasma can be found throughout both space and terrestrial environments. Wildfires, for example, generate dusty plasmas when charged particles of soot mixed with smoke. Put simply, non-reciprocal forces (as their name suggests) occur when forces exerted between two particles in a plasma are not the same. “This phenomenon was expected by some but now we have a precise approximation for it which didn't exist previously.” The ML algorithm was also able to correct some theoretical misconceptions about dusty plasma. For example, scientists thought that the charge of the particle was proportional to its size, but the model confirms that while a larger particle does contain a larger charge, it isn't proportional, as it can also be influenced by density and temperature. Generally, AI acquires its abilities by being fed (or training on) datasets—give AI one million pictures of a monkey, and it'll get progressively better at identifying a monkey when it sees one. “I think of it like the Star Trek motto, to boldly go where no one has before,” Burton said. Mystery on Easter Island: More Moai Found in Lake Scientists Learn How to Speed Up Evolutionary Time This Clay Seal Might Belong to a Biblical Figure A New Obelisk ‘Lifeform' Is Hiding Inside Humans Experts Are Using Dead Pigs to Find Human Graves Someone Stumbled Upon Armor from the Trojan War</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-china-made-an-antarctic-station-run-on-majority-clean-energy/'>China Just Built a Clean-Energy System in Antarctica. Here's How.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>How China Brought Clean Energy to Antarctica's Frigid Darkness Solar panels, wind turbines, a hydrogen energy system and lithium-ion batteries are powering China's newest polar research station The majority of power at China's Qinling research station in Antarctica now comes from clean energy. Five years ago electrical engineer Sun Hongbin was given what many would consider an impossible task: build a full-fledged clean-energy system amid some of the coldest temperatures on Earth, screaming winds and half-year darkness. China was then building its fifth Antarctic research station, called Qinling, on Inexpressible Island in Terra Nova Bay. But conventional solar and wind installations are no match for temperatures that plummet below –40 degrees Celsius, winds of up to 300 kilometers per hour (kmh) and ferocious blizzards. Such conditions can snap wind turbine blades, sharply reduce the performance of solar panels, and prevent batteries from charging and discharging properly. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. The clean-energy system at China's Qinling research station in Antarctica comprises solar panels, wind turbines, a hydrogen energy system and batteries. “It was a huge challenge” to build a system for the Earth's coldest, darkest and most remote continent, says Sun, now president of Taiyuan University of Technology in China and chief scientist for polar clean energy at the Polar Research Institute of China. “Other stations will probably have to learn how they achieve that much clean energy. A 2024 preprint analysis of 81 Antarctic research bases found that 37 had installed renewable-energy sources such as solar panels and wind turbines. An exception so far has been Belgium's Princess Elisabeth Station, which is only staffed during the Antarctic summer. It runs completely on wind and solar power, taking advantage of the almost 24-hour daylight. Even so, the vast majority of stations still depend on diesel-powered generators to keep their crews warm, fed and safe. But relying on diesel fuel has downsides: it is logistically difficult and expensive to transport bulky, liquid fossil fuels to such a remote location, often surrounded by sea ice. Highly specialized resources—typically including icebreakers and military personnel—are required to make the difficult refueling voyage, which usually takes place just once a year, under careful planning. Although major oil spills have been rare, any contamination can have severe consequences on Antarctic soil and water because it takes a long time for oil to break down in subzero temperatures. That is not to mention the toll that burning fossil fuels is taking on the Antarctic ecosystem through climate change. So there is significant incentive to move away from diesel. Yet “conventional wind turbines, solar panels, battery storage and hydrogen energy systems are designed to work above –30 degrees [C], but the conditions of Antarctic stations are often much worse,” Sun notes. When this happens in cold temperatures, wind turbines become brittle and break easily.” Plus, battery and hydrogen technologies—which are used to store wind and solar power for later use—were “not good enough” in the past to ensure that energy supplies for bases would be reliable around the clock and throughout the year, Kammen says. It features controls that can drop the indoor temperature to –50 degrees C, a wind machine that can blast out gusts of up to 216 kmh and snow generators that can whip up instant blizzards. One design is a turbine that eschews the pinwheel-like blades of a traditional windmill; instead it is shaped like an upended eggbeater, with both ends of each curved blade attached to a central pole. His team also installed turbines that are conventionally shaped but use blades made with carbon fiber—a strong and lightweight material that can withstand temperatures as low as –50 degrees C, according to Wang Bin, one of the engineers who went to Antarctica to build the system. These blades are also shorter than standard ones so as to reduce contact with the winds and increase structural resilience, Wang says. The latter has lower thermal conductivity, Sun's team explains, meaning the frame's temperature changes much more slowly when cold sets in and thus doesn't deform as easily. Instead of storing power in the most commonly employed types of lithium-ion batteries, which function poorly in subzero temperatures, the team used lithium-titanate batteries. Their chemistry makes it easier for lithium ions to move around inside the battery during the charging and discharging processes in extremely low temperatures. The scientists also built a thermal case around the batteries to keep them warm and designed a system to collect and store their waste heat—which can be directed back into the case when its internal temperature becomes too low, Wang adds. China's Qinling station is expected to have more than half its energy coming from the renewable system. To produce renewable hydrogen, an apparatus called an electrolyzer is powered by wind and solar energy to split water molecules into oxygen and hydrogen. To do so, the hydrogen is directed into an electrochemical device called a fuel cell, where it reacts with oxygen from the air to produce electricity, with only water and heat as by-products. “Sixty percent is a great start, but one needs to ramp up,” Kammen says. “The goal really needs to be 100 percent renewable energy all year-round.” Before you close the page, we need to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and we think right now is the most critical moment in that two-century history. If you become a Digital, Print or Unlimited subscriber to Scientific American, you can help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both future and working scientists at a time when the value of science itself often goes unrecognized. You Xiaoying is a freelance journalist based in London.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02753-9'>Morphological map of under- and overexpression of genes in human cells</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 09:52:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Methods volume 22, pages 1742–1752 (2025)Cite this article Cell Painting images offer valuable insights into a cell's state and enable many biological applications, but publicly available arrayed datasets only include hundreds of genes perturbed. The JUMP Cell Painting Consortium perturbed roughly 75% of the protein-coding genome in human U-2 OS cells, generating a rich resource of single-cell images and extracted features. These profiles capture the phenotypic impacts of perturbing 15,243 human genes, including overexpressing 12,609 genes (using open reading frames) and knocking out 7,975 genes (using CRISPR–Cas9). Here we mitigated technical artifacts by rigorously evaluating data processing options and validated the dataset's robustness and biological relevance. Analysis of phenotypic profiles revealed previously undiscovered gene clusters and functional relationships, including those associated with mitochondrial function, cancer and neural processes. The JUMP Cell Painting genetic dataset is a valuable resource for exploring gene relationships and uncovering previously unknown functions. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Well-level morphological profiles, image analysis pipelines, profile generation pipelines and instructions for retrieving the cell images, single-cell profiles and well-level profiles are publicly available via GitHub at https://github.com/jump-cellpainting/2025_Chandrasekaran_NatureMethods_Morphmap/blob/main/README.md. Plate maps and other metadata are available via GitHub at https://github.com/jump-cellpainting/datasets/tree/main/metadata. Cell Painting images and single-cell profiles are available via Cell Painting Gallery on the Registry of Open Data on AWS (https://registry.opendata.aws/cellpainting-gallery/) under accession number cpg0016-jump. We have released the data with a CC0 license. Several external biological databases provided critical reference information for analysis and interpretation. The DRKG was used for KG-based analyses. GO and WikiPathways annotations were obtained from the Molecular Signatures Database (Msigdb v.2023.1.Hs and v.2023.1.Hs). Human disease annotations were sourced from the Human Protein Atlas (v.23), and protein complex annotations came from the Comprehensive Resource of Mammalian protein complexes (CORUM v.4.1). Standardized gene nomenclature was referenced from Genenames.org and HGNC, while homology information was derived from HomoloGene (build 68). Finally, data regarding gene essentiality, gene expression and gene effect specifically in U2OS cells were obtained from the DepMap 23Q4 release. Source data are provided with this paper. The code for generating all the figures is available via GitHub at https://github.com/jump-cellpainting/2025_Chandrasekaran_NatureMethods_Morphmap. It is available under the BSD 3-clause license, a permissive open-source license. Mattiazzi Usaj, M. et al. High-content screening for quantitative cell biology. Trends Cell Biol. Bougen-Zhukov, N., Loh, S. Y., Lee, H. K. & Loo, L.-H. Large-scale image-based screening and profiling of cellular phenotypes. Boutros, M., Heigwer, F. & Laufer, C. Microscopy-based high-content screening. Cheng, J. et al. Massively parallel CRISPR-based genetic perturbation screening at single-cell resolution. Perlman, Z. E. et al. Multidimensional drug profiling by automated microscopy. Dagher, M. et al. nELISA: a high-throughput, high-plex platform enables quantitative profiling of the secretome. Preprint at bioRxiv https://doi.org/10.1101/2023.04.17.535914 (2023). Subramanian, A. et al. A next generation connectivity map: L1000 platform and the first 1,000,000 profiles. Moshkov, N. et al. Predicting compound activity from phenotypic profiles and chemical structures. Way, G. P. et al. Morphology and gene expression profiling provide complementary information for mapping cell state. Haghighi, M., Caicedo, J. C., Cimini, B. A., Carpenter, A. E. & Singh, S. High-dimensional gene expression and morphology profiles of cells across 28,000 genetic and chemical perturbations. Chandrasekaran, S. N., Ceulemans, H., Boyd, J. D. & Carpenter, A. E. Image-based profiling for drug discovery: due for a machine-learning upgrade? Williams, E. et al. Image Data Resource: a bioimage data integration and publication platform. Neumann, B. et al. Phenotypic profiling of the human genome by time-lapse microscopy reveals cell division genes. Ohya, Y. et al. High-dimensional and large-scale phenotyping of yeast mutants. Mattiazzi Usaj, M. et al. Systematic genetics and single-cell imaging reveal widespread morphological pleiotropy and cell-to-cell variability. Heigwer, F. et al. A global genetic interaction network by single-cell imaging and machine learning. Fay, M. M. et al. RxRx3: phenomics map of biology. Preprint at bioRxiv https://doi.org/10.1101/2023.02.07.527350 (2023). Ramezani, M. et al. A genome-wide atlas of human cell morphology. Lazar, N. H. et al. High-resolution genome-wide mapping of chromosome-arm-scale truncations induced by CRISPR–Cas9 editing. Chandrasekaran, S. N. et al. Three million images and morphological profiles of cells treated with matched chemical and genetic perturbations. Chandrasekaran, S. N. et al. JUMP Cell Painting dataset: morphological impact of 136,000 chemical and genetic perturbations. Preprint at bioRxiv https://doi.org/10.1101/2023.03.23.534023 (2023). A. et al. Optimizing the Cell Painting assay for image-based profiling. Yang, X. et al. A public genome-scale lentiviral expression library of human ORFs. A. et al. A versatile information retrieval framework for evaluating profile strength and similarity. Sjöstedt, E. et al. An atlas of the protein-coding genes in the human, pig, and mouse brain. al. Drkg - drug repurposing knowledge graph for covid-19. et al. TGF-β1 enhances FOXO3 expression in human synovial fibroblasts by inhibiting miR-92a through AMPK and p38 pathways. Vivar, R. et al. Role of FoxO3a as a negative regulator of the cardiac myofibroblast conversion induced by TGF-β1. Reck-Peterson, S. L., Redwine, W. B., Vale, R. D. & Carter, A. P. The cytoplasmic dynein transport machinery and its many cargoes. Huang, J., Roberts, A. J., Leschziner, A. E. & Reck-Peterson, S. L. Lis1 acts as a ‘clutch' between the ATPase and microtubule-binding domains of the dynein motor. Kumari, A. et al. Phosphorylation and Pin1 binding to the LIC1 subunit selectively regulate mitotic dynein functions. Dwivedi, D., Kumari, A., Rathi, S., Mylavarapu, S. V. S. & Sharma, M. The dynein adaptor Hook2 plays essential roles in mitotic progression and cytokinesis. Rohban, M. H. et al. Systematic morphological profiling of human gene and allele function via Cell Painting. Lee, J. et al. A Myt1 family transcription factor defines neuronal fate by repressing non-neuronal genes. Fu, M. et al. The Hippo signalling pathway and its implications in human health and diseases. Gong, R. et al. Opposing roles of conventional and novel PKC isoforms in Hippo-YAP pathway regulation. Selinger, D. W. et al. A framework for autonomous AI-driven drug discovery. Preprint at bioRxiv https://doi.org/10.1101/2024.12.17.629024 (2024). Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. McClintick, J. N. et al. Stress-response pathways are altered in the hippocampus of chronic alcoholics. Trapnell, C. et al. Differential analysis of gene regulation at transcript resolution with RNA-seq. Kapeli, K. et al. Distinct and shared functions of ALS-associated proteins TDP-43, FUS and TAF15 revealed by multisystem analyses. Vrenken, K. S. et al. The transcriptional repressor SNAI2 impairs neuroblastoma differentiation and inhibits response to retinoic acid therapy. Rivera-Reyes, A. et al. YAP1 enhances NF-κB-dependent and independent effects on clock-mediated unfolded protein responses and autophagy in sarcoma. Cell Death Dis. Uezu, A. et al. Identification of an elaborate complex mediating postsynaptic inhibition. Delgado, A. P., Brandao, P., Chapado, M. J., Hamid, S. & Narayanan, R. Open reading frames associated with cancer in the dark matter of the human genome. Cancer Genomics Proteom. & Feng, Y. Foreboding lncRNA markers of low-grade gliomas dependent on metabolism. Usman, S. et al. Transcriptome analysis reveals vimentin-induced disruption of cell-cell associations augments breast cancer cell migration. Ochoa, D. et al. The next-generation Open Targets Platform: reimagined, redesigned, rebuilt. Nucleic Acids Res. Tran, K.-V. et al. Human thermogenic adipocyte regulation by the long noncoding RNA LINC00473. Liu, X. et al. Regulation of mitochondrial biogenesis in erythropoiesis by mTORC1-mediated protein translation. Mitchell, D. C. et al. A proteome-wide atlas of drug mechanism of action. Morita, M. et al. MTOR controls mitochondrial dynamics and cell survival via MTFP1. Sun, Q. et al. UQCRFS1 serves as a prognostic biomarker and promotes the progression of ovarian cancer. Doi, M. et al. Gpr176 is a Gz-linked orphan G-protein-coupled receptor that sets the pace of circadian behaviour. et al. Nerve growth factor interacts with CHRM4 and promotes neuroendocrine differentiation of prostate cancer and castration resistance. Iida, M., Anna, C. H., Gaskin, N. D., Walker, N. J. & Devereux, T. R. The putative tumor suppressor Tsc-22 is downregulated early in chemically induced hepatocarcinogenesis and may be a suppressor of Gadd45b. Liu, R., Zhou, D., Yu, B. & Zhou, Z. Phosphorylation of LZTS2 by PLK1 activates the Wnt pathway. Synaptic activity-regulated Wnt signaling in synaptic plasticity, glial function and chronic pain. Kameda-Smith, M. M. et al. Characterization of an RNA binding protein interactome reveals a context-specific post-transcriptional landscape of MYC-amplified medulloblastoma. Liu, T. et al. Multi-omic comparison of Alzheimer's variants in human ESC-derived microglia reveals convergence at APOE. Chandrasekaran, S. N. Phenotypically active ORF and CRISPR consensus profiles. Munoz, A. Phenotypically active ORF and CRISPR consensus profiles. Arevalo, J. et al. Evaluating batch correction methods for image-based cell profiling. Johannessen, C. M. et al. COT drives resistance to RAF inhibition through MAP kinase pathway reactivation. Berger, A. H. et al. High-throughput phenotyping of lung cancer somatic mutations. Doench, J. G. et al. Rational design of highly active sgRNAs for CRISPR–Cas9-mediated gene inactivation. Singh, S., Bray, M.-A., Jones, T. R. & Carpenter, A. E. Pipeline for illumination correction of images for high-throughput microscopy. Serrano, E. et al. Reproducible image-based profiling with Pycytominer. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Tsuchida, C. A. et al. Mitigation of chromosome loss in clinical CRISPR–Cas9-engineered T cells. Nahmad, A. D. et al. Frequent aneuploidy in primary human T cells after CRISPR–Cas9 cleavage. Przewrocka, J., Rowan, A., Rosenthal, R., Kanu, N. & Swanton, C. Unintended on-target chromosomal instability following CRISPR/Cas9 single gene targeting. & Chandrasekaran, S. N. Similarity of CRISPR genes grouped by chromosome location without chromosome arm correction. & Chandrasekaran, S. N. Similarity of ORF genes grouped by chromosome without chromosome arm correction. & Chandrasekaran, S. N. Similarity of CRISPR genes grouped by chromosome location with chromosome arm correction. Ding, X. et al. Scaling up your kernels to 31 × 31: revisiting large kernel design in CNNs. Preprint at https://arxiv.org/abs/2203.06717 (2022). Chen, T. & Guestrin, C. XGBoost: a scalable tree boosting system. 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining 785–794 (Association for Computing Machinery, 2016). Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Clough, E. et al. NCBI GEO: archive for gene expression and epigenomics data sets: 23-year update. Nucleic Acids Res. Li, M. M., Huang, K. & Zitnik, M. Graph representation learning in biomedicine and healthcare. Kipf, T. N. & Welling, M. Variational graph auto-encoders. Preprint at https://arxiv.org/abs/1611.07308 (2016). Fey, M. & Lenssen, J. E. Fast graph representation learning with PyTorch Geometric. Preprint at https://arxiv.org/abs/1903.02428 (2019). Bonner, S. et al. Understanding the performance of knowledge graph embeddings in drug discovery. Ali, M. et al. Bringing light into the dark: a large-scale evaluation of knowledge graph embedding models under a unified framework. Ren, F. et al. A small-molecule TNIK inhibitor targets fibrosis in preclinical and clinical models. Ashburner, M. et al. Gene ontology: tool for the unification of biology. Gene Ontology Consortium, et al. The Gene Ontology knowledgebase in 2023. We thank the scientists across the entire JUMP Cell Painting Consortium for their guidance and support throughout the project. We acknowledge a grant from the Massachusetts Life Sciences Center Bits to Bytes Capital Call program for funding the data production and catalyzing this Consortium. We appreciate funding to support data analysis and interpretation from members of the JUMP Cell Painting Consortium (Amgen, AstraZeneca, Bayer AG, Biogen, Eisai, Janssen Pharmaceutica NV, Merck KGaA, Darmstadt, Germany, Pfizer, Servier, Takeda Development Center Americas, Inc. (TDCA)), from the National Institutes of Health (NIH grant no. ), and from grant number 2020-225720 to B.A.C. from the Chan Zuckerberg Initiative DAF, an advised fund of the Silicon Valley Community Foundation. We acknowledge the Consortium's Supporting Partners for their contributions: Ardigen for their deep learning expertise and JUMP-CP Data Explorer web application (part of Ardigen's phenAID platform); Google/Verily for the compute support and configuration/optimization of Terra, which is codeveloped by the Broad Institute of MIT and Harvard, Microsoft and Verily (its use is not described in this paper); Nomic bio for their protein profiling (not described in this paper); and Revvity for the PhenoVue Cell Painting JUMP kit and the Edit-R Libraries and Edit-R tracrRNA. We also are grateful for the Amazon Web Services Registry of Open Data for hosting the public dataset. We also acknowledge the use of the Revvity Opera Phenix High-Content/High-Throughput imaging system at the Broad Institute, funded by the S10 grant no. Broad Institute, Cambridge, MA, USA Srinivas Niranj Chandrasekaran, John Arevalo, Patrick J. Byrne, Zitong S. Chen, Beth A. Cimini, John G. Doench, Jessica D. Ewald, Briana Fritchman, Amy Goodale, Marzieh Haghighi, Yu Han, Desiree Hernandez, Alexandr A. Kalinin, Maria Kost-Alimova, Brittany A. Marion, Lisa Miller, Nikita Moshkov, Alán F. Muñoz, David E. Root, Rebecca A. Senft, Erin Weisbart, Shantanu Singh & Anne E. Carpenter Revvity Discovery Limited (formerly known as Horizon Discovery), Cambridge, UK Eric Alix, Glynn Martin, Nicola Jane McCarthy & Simon Scrace Ardigen S.A, Kraków, Poland Adriana Borowa, Tomasz Jetka, Tomasz Krawiec, Magdalena Otrocka, Krzysztof Rataj, Michał Warchoł & Agata Zaremba Evotec SE, Hamburg, Germany William G. Charles, Holger Hennig, Tim James, Arne Monsees, Rahul Valiya Veettil & Andrei Zinovyev Boxiong Deng & Sui Wang Spring Science, San Carlos, CA, USA Colin J. Fuller, Ben Komalo, Francesco Rubbo, Sarah Trisorus, William J. Van Trump & Michael Wiest Plex Research Inc., Cambridge, MA, USA Jedidiah Gaetz & Douglas W. Selinger Ksilink, Strasbourg, France Zahra Hanifehlou, Christina B. Jacob, Arnaud Ogier, Peter Sommer, Amandine Thibaudeau & Amélie Weiss Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar was involved in the analysis, data curation, experiments and investigation, methodology, project administration, software, supervision, validation, visualization, writing the original draft and writing—review and editing. was involved in the analysis, data curation, software and validation. was involved in the methodology, supervision and writing the original draft. was involved in the data curation, experiments and investigation and validation. was involved in the data curation, software and visualization. was involved in the analysis, data curation, experiments and investigation, methodology, software, validation and writing—review and editing. was involved in the data curation, methodology, project administration, software, supervision, writing the original draft and writing—review and editing. was involved in the experiments and investigation. was involved in the project administration, resources and supervision. was involved in the experiments and investigation and writing the original draft. B.F. was involved in the experiments and investigation, methodology, project administration, resources, supervision and writing the original draft. was involved in the analysis, data curation, software, validation and writing—review and editing. was involved in the analysis and writing the original draft. A.G. was involved in the experiments and investigation, methodology, project administration, resources, supervision and writing the original draft. was involved in the analysis, experiments and investigation and software. was involved in the analysis, software and validation. was involved in the analysis, data curation and validation. was involved in the analysis, conceptualization, methodology, project administration, supervision, visualization and writing the original draft. D.H. was involved in the experiments and investigation and resources. was involved in the data curation, experiments and investigation, project administration and supervision. T. James was involved in the funding acquisition and supervision. T. Jetka was involved in the analysis, methodology, project administration, supervision, visualization and writing the original draft. was involved in the analysis, experiments and investigation, methodology, software and writing—review and editing. was involved in the analysis, data curation, software, validation and writing—review and editing. was involved in the data curation, experiments and investigation, methodology, project administration, resources, supervision and writing—review and editing. was involved in the software and supervision. was involved in the experiments and investigation, methodology, project administration, resources and validation. was involved in the experiments and investigation, methodology and resources. was involved in the experiments and investigation, funding acquisition and supervision. was involved in the experiments and investigation, methodology, project administration, validation and writing the original draft. A.M. was involved in the analysis, data curation, methodology, software, visualization, writing the original draft and writing—review and editing. N.M. was involved in the analysis. was involved in the analysis, data curation, experiments and investigation, methodology, software, validation, writing the original draft and writing—review and editing. was involved in the analysis, project administration and supervision. was involved in the supervision, writing the original draft and writing—review and editing. was involved in the analysis and data curation. was involved in the methodology, project administration, resources and supervision. was involved in the analysis, data curation, software, validation and writing—review and editing. S.S. was involved in the experiments and investigation, project administration and resources. was involved in the analysis and writing the original draft. was involved in the analysis. was involved in the conceptualization and supervision. A.T. was involved in the experiments and investigation and validation. was involved in the analysis, data curation, software, validation and writing—review and editing. was involved in the analysis, data curation, methodology, software, visualization, writing the original draft and writing—review and editing. was involved in the analysis, data curation, methodology, software, supervision, visualization, writing the original draft and writing—review and editing. was involved in the analysis, data curation, experiments and investigation, validation and writing—review and editing. M. Warchoł was involved in the project administration, supervision and writing—review and editing. was involved in the analysis, data curation, experiments and investigation, software, validation and writing—review and editing. was involved in the experiments and investigation and validation. M. Wiest: analysis, data curation, software, validation and writing—review and editing. A. Zaremba was involved in the analysis and visualization. A. Zinovyev was involved in the analysis, data curation, methodology, software, supervision, visualization, writing the original draft and writing—review and editing. S.S. was involved in the analysis, conceptualization, data curation, experiments and investigation, funding acquisition, methodology, project administration, software, supervision, validation, visualization, writing the original draft and writing—review and editing. was involved in the analysis, conceptualization, funding acquisition, methodology, project administration, supervision, visualization, writing the original draft and writing—review and editing. Correspondence to Shantanu Singh or Anne E. Carpenter. serve as scientific advisors for companies that use image-based profiling and Cell Painting (A.E.C., Recursion, SyzOnc, Quiver Bioscience; S.S., Waypoint Bio, Dewpoint Therapeutics, Deepcell and B.A.C., Marble Therapeutics) and receive honoraria for occasional scientific visits to pharmaceutical and biotechnology companies. Authors with affiliations to the pharmaceutical, technology and biotechnology companies listed are or have been employees of those companies and may have real or optional ownership therein. The other authors declare no competing interests. Nature Methods thanks Susanne Muller, Sukjoon Yoon and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editor: Rita Strack, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Chandrasekaran, S.N., Alix, E., Arevalo, J. et al. Morphological map of under- and overexpression of genes in human cells. Issue Date: August 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41567-025-02953-2'>Frustrated electron hopping from the orbital configuration in a two-dimensional lattice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 09:44:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Physics (2025)Cite this article Metrics details Electron hopping on spatially periodic lattices gives rise to intriguing electronic behaviour. For example, hopping on the geometrically frustrated two-dimensional kagome, dice and Lieb lattices yields electronic band structures with both massless Dirac-like and perfectly dispersion-less, flat bands. As materials featuring the dice and Lieb lattice structures are scarce, an alternative approach proposes to leverage atomic orbitals to realize the characteristic electron hopping of geometrically frustrated lattices. This strategy promises to expand the list of candidate materials with frustrated electron hopping, but is yet to be shown in experiments. Here we demonstrate frustrated hopping in the van der Waals intermetallic Pd5AlI2, emerging from the arrangement of atomic orbitals in a primitive square lattice. Using angle-resolved photoemission spectroscopy and quantum oscillation measurements, we reveal that the band structure of Pd5AlI2 includes linear Dirac-like bands intersected at their crossing point by a locally flat band—an essential characteristic of frustrated hopping in Lieb and dice lattices. Moreover, this compound shows exceptional chemical stability, with its unusual bulk band structure and metallicity persisting in ambient conditions down to the monolayer limit. Hence, our results showcase a way to realize electronic structures characteristic of geometrically frustrated lattices in non-frustrated systems. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are presented in the published figures and tables and are available via the Harvard Dataverse repository at https://doi.org/10.7910/DVN/NAD0B9. Further data are available from the corresponding authors upon reasonable request. Stewart, G. R. Heavy-fermion systems. Google Scholar Bergman, D. L., Wu, C. & Balents, L. Band touching from real-space topology in frustrated hopping models. Google Scholar Mielke, A. Ferromagnetic ground states for the Hubbard model on line graphs. A: Math. Google Scholar Horiguchi, T. & Chen, C. C. Lattice Green's function for the diced lattice. Google Scholar Lieb, E. H. Two theorems on the Hubbard model. Google Scholar et al. Quantum-limit Chern topological magnetism in TbMn6Sn6. Google Scholar Kang, M. et al. Dirac fermions and flat bands in the ideal kagome metal FeSn. Google Scholar Ye, L. et al. Hopping frustration-induced flat band and strange metallicity in a kagome metal. Google Scholar Wakefield, J. P. et al. Three-dimensional flat bands in pyrochlore metal CaNi2. Google Scholar Huang, J. et al. Non-Fermi liquid behaviour in a correlated flat-band pyrochlore lattice. Google Scholar Lee, C.-C., Fleurence, A., Yamada-Takamura, Y. & Ozaki, T. Hidden mechanism for embedding the flat bands of Lieb, kagome, and checkerboard lattices in other structures. Google Scholar Liu, H., Sethi, G., Meng, S. & Liu, F. Orbital design of flat bands in non-line-graph lattices via line-graph wave functions. Google Scholar Bercioux, D., Urban, D. F., Grabert, H. & Häusler, W. Massless Dirac-Weyl fermions in a 𝒯3 optical lattice. Google Scholar Xia, S. et al. Unconventional flatband line states in photonic Lieb lattices. Google Scholar Shen, R., Shao, L. B., Wang, B. & Xing, D. Y. Single Dirac cone with a flat band touching on line-centered-square optical lattices. Google Scholar Santos, L. et al. Atomic quantum gases in kagomé lattices. Google Scholar et al. Ultracold atoms in a tunable optical kagome lattice. Google Scholar Taie, S. et al. Coherent driving and freezing of bosonic matter wave in an optical Lieb lattice. Google Scholar Merker, H.-B., Schäfer, H. & Krebs, B. Neue PdxAly-phasen und die verbindung Pd5AII2. Google Scholar Le Blanc, M., Richter, K. & Schiebold, E. Eine früfung der tammannschen theorie der resistenzgrenzen am system gold–kupfer. Aufstellung neuer gesichtspunkte. Google Scholar Lan, Z., Goldman, N., Bermudez, A., Lu, W. & Öhberg, P. Dirac-Weyl fermions with arbitrary spin in two-dimensional optical superlattices. Dóra, B., Kailasvuori, J. & Moessner, R. Lattice generalization of the Dirac equation to general spin and the role of the flat band. Semenoff, G. W. Condensed-matter simulation of a three-dimensional anomaly. Zhang, Y., Tan, Y.-W., Stormer, H. L. & Kim, P. Experimental observation of the quantum Hall effect and Berry's phase in graphene. Novoselov, K. S. et al. Two-dimensional gas of massless Dirac fermions in graphene. Khestanova, E. et al. Unusual suppression of the superconducting energy gap and critical temperature in atomically thin NbSe2. Nano Lett. Park, C.-H., Yang, L., Son, Y.-W., Cohen, M. L. & Louie, S. G. Anisotropic behaviours of massless Dirac fermions in graphene under periodic potentials. Devarakonda, A. et al. Clean 2D superconductivity in a bulk van der Waals superlattice. Legros, A. et al. Universal T-linear resistivity and Planckian dissipation in overdoped cuprates. Cao, Y. et al. Strange metal in magic-angle graphene with near Planckian dissipation. Ghiotto, A. et al. Quantum criticality in twisted transition metal dichalcogenides. Hwang, E. H. & Das Sarma, S. Linear-in-T resistivity in dilute metals: a Fermi liquid perspective. Polshyn, H. et al. Large linear-in-temperature resistivity in twisted bilayer graphene. Cao, C. et al. Full control of solid-state electrolytes for electrostatic gating. Thinel, M. et al. Electronic bound states in the continuum in a 2D metal. Preprint at https://arxiv.org/abs/2410.19227 (2024). Urban, D. F., Bercioux, D., Wimmer, M. & Häusler, W. Barrier transmission of Dirac-like pseudospin-one particles. Weeks, C. & Franz, M. Topological insulators on the Lieb and perovskite lattices. Lai, H.-H., Grefe, S. E., Paschen, S. & Si, Q. Weyl–Kondo semimetal in heavy-fermion systems. Natl Acad. Ranninger, J. & Robaszkiewicz, S. Superconductivity of locally paired electrons. Micnas, R., Ranninger, J. & Robaszkiewicz, S. Superconductivity in narrow-band systems with local nonretarded attractive interactions. Novoselov, K. S. et al. Electric field effect in atomically thin carbon films. Treadwell, W. D. & Obrist, A. Über die bestimmung und bildung von oxydischen deckschichten auf aluminium. Cabrera, N. & Mott, N. F. Theory of the oxidation of metals. Kepp, K. P. Chemical causes of metal nobleness. Bergman, G. Influence of spin-orbit coupling on weak localization. Das Sarma, S. & Stern, F. Single-particle relaxation time versus scattering time in an impure electron gas. Rhodes, D., Chae, S. H., Ribeiro-Palau, R. & Hone, J. Disorder in van der Waals heterostructures of 2D materials. Zhu, J., Li, T., Young, A. F., Shan, J. & Mak, K. F. Quantum oscillations in two-dimensional insulators induced by graphite gates. Briggs, N. et al. Atomically thin half-van der Waals metals enabled by confinement heteroepitaxy. Fang, N., Lee, H., Sun, C. & Zhang, X. Sub-diffraction-limited optical imaging with a silver superlens. da Jornada, F. H., Xian, L., Rubio, A. & Louie, S. G. Universal slow plasmons and giant field enhancement in atomically thin quasi-two-dimensional metals. Rodrigo, D. et al. Mid-infrared plasmonic biosensing with graphene. Zagorac, D., Müller, H., Ruehl, S., Zagorac, J. & Rehme, S. Recent developments in the Inorganic Crystal Structure Database: theoretical crystal structure data and related features. Sheldrick, G. M. SHELXT—integrated space-group and crystal-structure determination. Acta Cryst. Sheldrick, G. M. Crystal structure refinement with SHELXL. Acta Cryst. Dolomanov, O. V., Bourhis, L. J., Gildea, R. J., Howard, J. A. K. & Puschmann, H. OLEX2: a complete structure solution, refinement and analysis program. Giannozzi, P. et al. Advanced capabilities for materials modelling with Quantum ESPRESSO. Vanderbilt, D. Soft self-consistent pseudopotentials in a generalized eigenvalue formalism. Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Georgescu, A. B., Millis, A. J. & Rondinelli, J. M. Trigonal symmetry breaking and its electronic effects in the two-dimensional dihalides MX2 and trihalides MX3. Georgescu, A. Wannier90 Hamiltonian tools. GitHub https://github.com/alexandrub53/Wannier90HamiltonianTools (2022). Kawamura, M. FermiSurfer: Fermi-surface viewer providing multiple representation schemes. Blaha, P. et al. WIEN2k: an APW+lo program for calculating the properties of solids. Wasserman, S. R., Tao, Y. T. & Whitesides, G. M. Structure and reactivity of alkylsiloxane monolayers formed by reaction of alkyltrichlorosilanes on silicon substrates. Wang, L. et al. One-dimensional electrical contact to a two-dimensional material. Lee, H. N. S., McKinzie, H., Tannhauser, D. S. & Wold, A. The low‐temperature transport properties of NbSe2. Tsen, A. W. et al. Nature of the quantum metal in a two-dimensional crystalline superconductor. Zhao, S. Y. F. et al. Sign-reversing Hall effect in atomically thin high-temperature Bi2.1Sr1.9CaCu2.0O8+δ superconductors. Zhu, C. S. et al. Evolution of transport properties in FeSe thin flakes with thickness approaching the two-dimensional limit. Lei, S. et al. High mobility in a van der Waals layered antiferromagnetic metal. Lai, Z. et al. Metastable 1T′-phase group VIB transition metal dichalcogenide crystals. Lei, S. High mobility in a van der Waals layered antiferromagnetic metal. Chen, L. et al. Exceptional electronic transport and quantum oscillations in thin bismuth crystals grown inside van der Waals materials. Download references We thank J. L. Pack, M. A. Kapfer, E. J. Telford, S. E. Turkel and J. G. Checkelsky for fruitful discussions. Research on novel hopping models was conducted as part of the Programmable Quantum Materials, an Energy Frontier Research Center, funded by the US Department of Energy (DOE), Office of Science, Basic Energy Sciences, under award no. acknowledges support from a Brown Investigator Award, a program of the Brown Institute for Basic Sciences at the California Institute of Technology. The synthesis and structural characterization of the 2D metal were supported by the NSF MRSEC program through the Center for Precision-Assembled Quantum Materials at Columbia University, award no. acknowledges support for STM/STS experiments by the Air Force Office of Scientific Research under award no. The Physical Property Measurement System used to perform vibrating-sample magnetometry and electrical transport measurements was purchased with financial support from the National Science Foundation through a supplement to award no. A.D. acknowledges support from the Simons Foundation Society of Fellows (grant no. High-magnetic-field measurements were performed at the National High Magnetic Field Laboratory, supported by the National Science Foundation Cooperative Agreement no. DMR-1644779 and the State of Florida. ARPES measurements used resources at the 21-ID (Electron Spectro-Microscopy) beamline of the National Synchrotron Light Source II, a US DOE, Office of Science User Facility, operated for the DOE, Office of Science, Brookhaven National Laboratory, under contract no. Aravind Devarakonda Present address: Department of Applied Physics and Applied Mathematics, Columbia University, New York, NY, USA Zhi Lin Present address: Department of Physics, Boston University, Boston, MA, USA These authors contributed equally: Aravind Devarakonda, Christie S. Koay, Daniel G. Chica. Department of Physics, Columbia University, New York, NY, USA Aravind Devarakonda, Sebastian Rossi, Madisen A. Holbrook, Abhay N. Pasupathy, Raquel Queiroz & Cory R. Dean Department of Chemistry, Columbia University, New York, NY, USA Christie S. Koay, Daniel G. Chica, Morgan Thinel, Sae Young Han, Michael E. Ziebel, Milan Delor, Xiaoyang Zhu & Xavier Roy Condensed Matter Physics and Materials Science Division, Brookhaven National Laboratory, Upton, NY, USA Asish K. Kundu & Abhay N. Pasupathy National Synchrotron Light Source II, Brookhaven National Laboratory, Upton, New York, NY, USA Asish K. Kundu, Anil Rajapitamahuni & Elio Vescovo Department of Applied Physics and Applied Mathematics, Columbia University, New York, NY, USA Zhi Lin Department of Chemistry, Indiana University, Bloomington, IN, USA Alexandru B. Georgescu Research Center for Electronic and Optical Materials, National Institute of Materials Science, Tsukuba, Japan Kenji Watanabe Research Center for Materials Nanoarchitectonics, National Institute of Materials Science, Tsukuba, Japan Takashi Taniguchi Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar synthesized and characterized the bulk crystals. supported with the energy-dispersive X-ray spectroscopy and scanning electron microscopy experiments, and M.E.Z. supported with the magnetization measurements. A.D. fabricated the nanodevices and performed the transport characterization. conducted the high-field measurements. performed the STM characterization. performed the ARPES measurements with support from A.R. provided the hexagonal boron nitride crystals. performed the theoretical calculations. supervised the project. wrote the manuscript with input from all authors. Correspondence to Aravind Devarakonda, Abhay N. Pasupathy, Raquel Queiroz, Cory R. Dean or Xavier Roy. The authors declare no competing interests. Nature Physics thanks Zurab Guguchia, Hongyan Lu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a SEM image of Pd5AlI2 single crystal outlined in white. b EDS from a representative area scan on the crystal shown. Using statistics collected from ten area scans using the Al content as the reference, the Pd stoichiometry is 4.783 ± 0.174, Al stoichiometry is 1.0, and I stoichiometry is 1.843 ± 0.049. ARPES E(k) intensity map (LH polarization) along the \(\overline{\rm{M}}-\overline{{\Gamma}}-\overline{\rm{M}}\) line of the surface BZ at various incoming photon energies hν, a 127 eV b 136 eV and c 142 eV, probing distinct slices of kz, marked in e. The weak variation of these maps evidences the quasi-2D nature of the electronic structure. d ARPES Fermi surface cross-sections in the kx – ky plane (LV polarization) at various kz. The cross-sections are similarly invariant with kz. e An ARPES kz – k|| map along the \(\overline{\rm{M}}-\overline{{\Gamma}}-\overline{\rm{M}}\) line (LH polarization) exhibits cylindrical FSs for all three pockets. The dashed lines show the approximate kz sampled by the E(k) maps a – c. ARPES FS maps (hν = 125 eV) at f 100 meV g 150 meV h and 750 meV below EF. The experimental maps show good correspondence to DFT calculated FSs at i 100 meV j 150 meV and k 645 meV below EF. a Topography of terraced Pd5AlI2 flake exfoliated onto SiO2/Si wafer measured by AFM with b linecut along magenta segment showing single Pd5AlI2 layer steps. c Optical microscope image of the same flake showing d contrast in the red, green, and blue color channels between regions of varying thickness. e Spinning a layer of PMMA over the flake f enhances optical contrast, making it easier to distinguish regions with different layer number. Supplementary Sections I–IV, Figs. 1–5 and Discussion. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Devarakonda, A., Koay, C.S., Chica, D.G. et al. Frustrated electron hopping from the orbital configuration in a two-dimensional lattice. Download citation Received: 30 September 2024 Accepted: 27 May 2025 Published: 07 August 2025 DOI: https://doi.org/10.1038/s41567-025-02953-2 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Physics ISSN 1745-2481 (online) ISSN 1745-2473 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-62647-4'>Sensitive neoantigen discovery by real-time mutanome-guided immunopeptidomics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 06:46:03
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Targeting cancer-specific HLA-peptide complexes is a promising approach in immunotherapy. Mutated neoantigens are excellent targets due to their immunogenicity and cancer-specificity. Mass spectrometry (MS)-based immunopeptidomics guides the selection of naturally presented immunogenic targets within the immunopeptidome, refining immunogenicity predictions. Implementation in clinical settings, however, must achieve global depth, capturing the entirety of the immunopeptidome, maintain high target sensitivity, and cater to scarce sample inputs and short turnaround time. Here, we present NeoDiscMS, an extension of NeoDisc that enables the acquisition of personalized immunopeptidomics data. Leveraging next-generation sequencing-guided real-time spectral acquisitions, NeoDiscMS maximizes sensitivity with minimal loss of global depth. Designed for effectiveness and ease of use, with minimal effort required for implementation, NeoDiscMS enhances the detection of peptides derived from tumor-associated antigens by up to 20% and improves confidence in neoantigen identification compared to the gold standard method. NeoDiscMS advances personalization in clinical antigen discovery with more confident neoantigen detection and easy implementation. The accurate identification of immunogenic cancer-specific antigens presented by HLA complexes is instrumental to the development of cancer immunotherapy1,2. Most discovery approaches supporting neoantigen-based therapeutics rely solely on computational pipelines to predict immunogenic neoantigens from next-generation sequencing data of the patient's tumor and germline tissues2,3,4. Nevertheless, it has been repeatedly shown that only ~1% of somatic mutations in tumors induce spontaneous or vaccine-induced T cell responses5,6. Although the inclusion of machine learning classifiers improves the ranking of immunogenic neoantigens7,8, the selection of candidate immunogenic neoantigens for inclusion in a personalized vaccine remains a challenge in the context of tumors with high mutational burdens9. Mass spectrometry (MS)-based detection of immunopeptides eluted directly from tumor tissues is the method of choice to simultaneously (1) deeply characterize the tumor antigenic landscape10, (2) revealing novel targets like neoantigens11, and (3) characterize defects in antigen presentation12, all of which are relevant clinical features1,9. MS-detected neoantigens have a substantially higher likelihood of being immunogenic than predicted neoantigens due to their confirmed presentation by tumor cells11,13,14,15,16,17,18. Besides the commonly known challenges that proteogenomic methods face1, neoantigen detection by MS is further impeded by their low abundance19,20. In addition, the short turnaround time that is required for clinical vaccine manufacturing and the highly limited sample material pose important challenges when aiming to sensitively measure the global immunopeptidome while maintaining high sensitivity for selected neoantigens. Typically, antigen discovery is conducted with TopN-based data-dependent acquisition (DDA). DDA is stochastic and aims to diversify what peptides are fragmented by utilizing dynamic exclusion, which prevents repeated fragmentation of the same precursor mass within subsequent acquisition cycles. While this approach increases coverage, it ultimately limits sensitivity for low-abundance targets. To improve sensitivity and address the limitation of co-isolated precursors (even in narrow-window MS acquisitions), previous studies applied chimeric spectrum deconvolution to identify tryptic peptides from low-input and single-cell proteomics samples21,22,23,24,25. However, so far, chimeric spectrum deconvolution has not been implemented in immunopeptidomics. Targeted immunopeptidomics strategies for neoantigen discovery exist, relying on heavy-labeled spike-in standards that enhance sensitivity19,20. However, synthesizing and conducting quality control on heavy-labeled peptides for each patient creates a significant bottleneck, making such strategies unsuitable for clinical workflows due to limited scalability and an inability to meet the short turnaround time required from biopsy to vaccine production. Peptide warehouse approaches can't alleviate these shortcomings, as neoantigens are unique to each patient. Additionally, a major concern is that synthesis products might include impurities of the unlabeled target peptide, potentially leading to sample contamination20,26. Spike-in free approaches that enhance sensitivity have been reported, but they (1) attempt to improve the sensitivity of TopN-based precursor selection with all its limitations, (2) focus on reproducing targets that were identified in a previous scouting run rather than discovering novel ones, (3) require advanced MS architectures, (4) demand manual intervention for data processing, and/or (5) don't optimize the trade-off between target sensitivity and global coverage2,20,21,27,28,29,30,31,32,33,34. Here, we present NeoDiscMS (an extension of NeoDisc9), a spike-in-free, scalable, and easy-to-implement targeted-DDA hybrid acquisition immunopeptidomic workflow that boasts enhanced sensitivity and accuracy for target peptide detection while minimizing the trade-off against loss of global immunopeptidome coverage. In NeoDiscMS, NGS-inferred in silico prioritized antigenic peptide candidates guide MS data acquisition by leveraging bespoke real-time peptide-to-spectrum matching filters (RTSf). RTSfs selectively trigger time-intensive, high-sensitivity scans for precursors with target-like features, a capability not previously applied to immunopeptidomics. In addition, chimeric spectrum deconvolution further enhances the depth of immunopeptidome analysis. We benchmark the performance of NeoDiscMS against DDA, either with or without an inclusion list, using an HLA-I immunopeptidome dilution series. To demonstrate the practical utility of NeoDiscMS we showcase the improved detection, in terms of reproducibility and identification confidence, of clinically relevant TAAs and mutated neoantigens in patient-derived primary melanoma cell lines and melanoma tissues. Importantly, data acquired with NeoDiscMS can be processed with commonly used search engines, facilitating seamless integration into existing workflows such as our clinical antigen discovery pipeline NeoDisc. We envision that NeoDiscMS may enhance the detection of immunopeptides associated with pathological conditions beyond cancer and can also be applied to the identification of other peptides outside the scope of immunopeptidomics. NeoDiscMS was implemented using a peptide database - in our case, an output of 1500 HLA-I-restricted predicted neoantigens or tumor-associated antigens generated by the end-to-end clinical antigen discovery pipeline NeoDisc9, in the format of an inclusion list and a FASTA file containing each peptide as a separate entry (Fig. a Matched tumor and germline genome data, along with tumor transcriptome data, are processed by NeoDisc to create a tumor sample-specific personalized proteome reference, annotated with single-nucleotide polymorphisms, somatic mutations, which are then used for HLA binding and immunogenicity prediction of antigenic targets. From this prioritized list of peptides, NeoDisc generates a personalized inclusion list for NeoDiscMS acquisition of the matched tumor immunopeptidome sample. The MS data is then searched by NeoDisc against the personalized proteome to identify naturally presented antigenic peptides, refining the list to prioritize the most clinically relevant and likely immunogenic targets. b Comparison of DDA, DDA with an inclusion list (ilDDA) and NeoDiscMS methods. c The targeted branch of NeoDiscMS consists of scouting scans (sMS2) that get triggered by an MS1 precursor mass match and a retention-time-restricted scheduled inclusion list. sMS2 are searched against a database in real-time to assess if a high-sensitivity scan (hMS2) should be triggered. Schematics were created in Biorender (https://BioRender.com/s85z530). Each cycle starts with an MS1 scan. For a precursor in an MS1 scan to qualify for fragmentation with high-sensitivity scans in the targeted branch, it needs to match a precursor mass in the inclusion list. The fragments of the sMS2 scan are matched in real-time, within milliseconds, against predicted spectra in the database and are subsequently scored. If the sMS2 scan yields a real-time search PSM that passes pre-defined quality thresholds, a time-intensive, high-sensitivity MS2 scan (hMS2) is conducted on that precursor (Fig. The sMS2 scan serves as a filter, minimizing the number of time-consuming subsequent targeted scans during a measurement. hMS2 scans can take up to five times longer than sMS2 scans and are defined by an increased AGC target, a higher maximum injection time, and stepped collision energies (Supplementary Table 1). Both sMS2 and hMS2 scans are intentionally not subjected to dynamic exclusion restrictions and MS1 minimum precursor intensity thresholds, enabling multiple MS/MS spectra to enhance identification confidence. To improve overall coverage, the discovery branch of our method uses wide isolation windows of 3.2Th for MS2 scans (dMS2) to increase the probability of co-isolating additional precursor masses. The obtained raw file is processed using chimeric spectra deconvolution with MSFragger's DDA+ mode24, enabling peptide-to-spectrum matching (PSMing) with precursor masses that deviate from the isolated mass. To benchmark the performance of wide isolation windows and chimeric spectra deconvolution on immunopeptidomics data, we first tested different isolation window sizes with DDA, injecting the equivalent of 10 million JY cells per measurement, analyzed with MSFragger-DDA+. A total of 96% of the identified 8–14 mer peptides were predicted by MixMHCpred36 as binders (%-rank <2) for each measurement, indicating that identification quality is consistent across conditions. Differences observed between isolation window widths pertained to the number of unique peptides identified. We selected 3.2Th because it most consistently yielded the highest number of identifications (Supplementary Fig. This is in accordance with previous findings that optimal wide window size for increased depth with chimeric spectral deconvolution from low input samples remains narrow24,25. Next, to extend the benchmarking, we measured the HLA-I immunopeptidome of the B-cell lines JY and RA95736, using either isolation windows of 1.2Th or 3.2Th, representing regular and wide isolation windows, respectively. This, for the equivalent 5 and 10 million cells per measurement. We then processed all data either with MSFragger-DDA or with MSFragger-DDA+ for peptide identification. For all measurements and with both MSFragger-DDA and MSFragger-DDA+, between 96–97% of the identified 8–14 mer peptides were predicted as binders (%-rank <2), the average peptide length lied between 9.4–9.6. Processing immunopeptidomic data with MSFragger-DDA+ instead of MSFragger-DDA resulted in a 2% to 36% increase in the number of unique peptides identified, with higher cell numbers contributing to a greater identification surplus (Fig. When comparing replicates of the same cell type and cell number, the change in identified peptides between isolation windows of 1.2Th and 3.2Th, processed with MSFragger-DDA+, ranged from 0% to 8%. The overlap coefficients for peptides identified by both MSFragger-DDA and MSFragger-DDA+ within the same raw file (top three gray dots in each panel of the lower row) ranged from 88% to 96% (Fig. Using MSFragger-DDA+, increasing the isolation window size from 1.2Th to 3.2Th resulted in a 7–9% increase in the fraction of scans yielding one or more PSMs (Fig. For MSFragger-DDA+ searched data, the fraction of peptides which were only identified thanks to chimeric spectrum deconvolution, meaning they never corresponded to the selected precursor mass to isolate, lies between 19–37%. Significantly, this fraction increases consistently by 10–11% when applying isolation windows of 3.2Th instead of 1.2Th for the same sample (Fig. To better understand the quality of PSMs gained with help of chimeric spectrum deconvolution, we inspected identified precursors that corresponded (isolated) and precursors that deviated from the isolated mass (co-isolated) of their respective MS2 scans as separate populations. Both populations had similar fractions of predicted binders to the respective HLA alleles and similar length distributions (Fig. a The upper panel depicts the number of identified unique stripped sequences in cell lines JY and RA957 immunopeptidome of injections of 5- or 10 million cells-equivalent, measured in DDA and with MS2 isolation windows of 1.2Th or 3.2Th (n = 3 per condition). The lower panel shows overlap coefficients between raw files and within or between processing modes. MSFragger can be operated with (MSFragger-DDA+) or without (MSFragger-DDA) taking chimeric spectra into account for finding PSMs. Wider isolation windows of 3.2Th and use of MSFragger-DDA+ increase the number of identified unique sequences, as demonstrated by searching the same raw files with either MSFragger-DDA or MSFragger-DDA+ (upper panel), resulting in an overlap coefficient of at least 88% (bottom panel; the top 3 gray dots for each sub-panel). b The fraction of MS2 scans with at least one assigned PSM. Each raw file was processed with MSFragger-DDA or MSFragger-DDA+. For all raw files and processing modes, most scans with one PSM or more were matched by both processing modes. Percentages were averaged across technical replicates (TRs). c The fraction of immunopeptides that were identified by MSFragger-DDA+ exclusively as co-isolated PSMs. d The fraction of binders (%-rank <2; MixMHCpred) and e sequence length distribution among immunopeptides that were uniquely identified as chimeric PSMs or not f The co-isolated PSMs derive from precursors that deviate in mass from the isolated precursor with a clear pattern: mass deviation distribution reflects differences in multiples of 1 Da. In a chimeric spectrum, the mass difference between the two peptides is defined by combinations of amino acids whose weights permit it to generate precursors that can be isolated through the same isolation window. Because the residuals of amino acid weights all lie in a narrow 0.1 Da range, we observed, as expected, that the absolute difference in mass (Da) between co-isolated peptides is often very close to multiples of 1 Da (when we filter for co-isolated identified peptides with the same charge which account for 92% of all cases; Fig. To conclude, the wide-window and especially the chimeric spectrum deconvolution approach provide increased global immunopeptidome depth over commonly used narrow-window scans in DDA while maintaining identification quality. Importantly, the wide isolation window strategy impacts only the discovery branch, and users can implement NeoDiscMS without enabling it, should they choose to do so. To illustrate the increased sensitivity of time-expensive scans for target discovery, as well as the capacity of RTSf to alleviate the burden on global immunopeptidome depth, we benchmarked the performance of DDA (the gold standard method for antigen discovery), DDA with an inclusion list branch (ilDDA) that triggers high-sensitivity scans, and NeoDiscMS (Supplementary Table 1 and Fig. To accurately assess the performance of our target identification method, we compiled an inclusion list of ~1500 unique RA957-specific, HLA-A68:01-restricted, 8-11-meric peptide sequences (Fig. These peptides were consistently identified across RA957 samples and were predicted not to bind to any JY HLA-I alleles, showing a %-rank >10 for HLA-A*02:01, -B*07:02, and -C*07:02. The list included methionine oxidation as a variable modification (Supplementary Fig. 3a) and all charge states from 1–3, yielding a list of 5046 precursors. To avoid having the targeted branch look for all targets throughout the entire gradient, our method relies on scheduled targeting. We predicted the retention time (RT) for each peptide to allow scheduling with a ±15 min window, which captures 75% of targets when comparing measured to predicted retention time, though at a remarkable inclusion list burden (number of scheduled precursors) across the retention time (Supplementary Fig. a 9-mer motifs of immunopeptides that bind to the depicted HLA alleles. We selected A*68:01-restricted peptides as targets because they can be clearly assigned to RA957 when the RA957 immunopeptidome is diluted in the JY immunopeptidome. b Histogram of the binding affinity of A*68:01-specific targets to JY alleles as predicted by MixMHCpred. c Summary of the dilution series employed to evaluate the scalability and sensitivity of NeoDiscMS. d Count of identified A*68:01-binding target peptides and global depth for each step in the dilution series (n = 3 per condition). e Number of target peptides identified in either zero, one, two, or all three DDA (x-axis) or NeoDiscMS replicates (y-axis). Large bold numbers show the sum of all targets identified with either acquisition method. f Linear regression of all 1/16 dilution PSM hyperscores (left) and spectral angles (right), made with either dMS2, sMS2, or hMS2 scans. g Correlation of target and non-target precursor MS1 intensities between technical replicates (TRs) for 1/1024 dilution injections. We diluted the class I immunopeptide sample of RA957 cells in the class I immunopeptidome of JY cells in a dilution series that starts with RA957-to-JY dilution ratios of 1/16, 1/64, 1/256, and 1/1024, as well as a RA957-free JY immunopeptidome (negative control) (Fig. While the 1/1024 dilution simulates a setting with low abundance peptide targets that are hard to detect, the 1/16 dilution is most suitable to assess the scalability of NeoDiscMS when many detectable targets are present. We acquired data of all dilutions in DDA, ilDDA, and NeoDiscMS (Fig. Overall, all measurements maintain between 96%–98% of binders, with an expected average peptide length of 9.4–9.5. The successful dilution is evident when inspecting the fraction of HLA-A*68:01 binders among all peptides (Supplementary Fig. In all dilutions, NeoDiscMS identified between 0–7% more peptide targets on average than ilDDA, and between 16–51% more than DDA. Global depth of NeoDiscMS lied between 5–21% above ilDDA and 5–14% below DDA (Fig. With respect to the number of uniquely identified peptides we note that, on average, for every 1% of loss in global depth compared to DDA, we gain 2% in target sensitivity with ilDDA and 5.6% with NeoDiscMS. This difference in sensitivity trade-off quantifies the minimization of loss in global depth with help of RTSf very clearly. In the dilution series, the 1/1024 dilution most closely simulates a real-world scenario of tumor-antigen discovery, with only a few detectable peptide targets. There, 18 peptide targets were identified consistently in all NeoDiscMS replicates, while this is only true for 9 peptide targets with DDA. This constitutes a 2-fold increase in the number of fully reproducible target peptide identifications. Conversely, 9 peptides were identified at least once with DDA and never with NeoDiscMS, while 19 peptides were detected at least once with NeoDiscMS but not DDA, a 2.1-fold difference. In total, across three replicates each, 24% more peptide targets were identified with NeoDiscMS than with DDA (Fig. Detected peptide targets showed a higher average number of PSMs per target for NeoDiscMS than both DDA and ilDDA (Supplementary Fig. A closer examination of the 1/1024 dilution NeoDiscMS runs revealed that peptide targets which were not identified with DDA were distributed across all MS2 scan types and included low- to medium-intensity peptides. This shows that the enhanced target identification of NeoDiscMS derives from a combination of reduced ion sampling stochasticity and enhanced scan sensitivity, which sets targeted and DDA approaches apart (Supplementary Fig. For peptides in the 1/16 dilution, where the highest number of peptide target PSMs were detected, we observed a consistent increase in hyperscore and spectral angle when comparing hMS2 PSMs with dMS2 PSMs across the full intensity range (Fig. This naturally raises the question of quantification reproducibility. To demonstrate the reproducibility of the MS1-based quantification, as provided by the FragPipe output, we focused on the 1/1024 dilution experiment measurements which represents a scenario of detecting low abundance targets. The variability between replicates is comparable between DDA and NeoDiscMS and between non-target peptide identifications (gray, R2 = 0.96–0.97%) and target (red, R2 = 0.92–0.99%; Fig. We observe slightly lower values in the technical replicates TR2 vs. TR3 NeoDiscMS comparison, though this is marginal considering the relatively lower number of target IDs. Repeated fragmentation via the targeted branch may open future opportunities to implement more accurate, advanced MS2-based quantification methods with NeoDiscMS data. However, a key caveat must be considered: hMS2 scans use different ion accumulation and collision energy settings compared to dMS2 and sMS2 scans (Supplementary Table 1). As a result, they introduce variations in both absolute fragment ion abundances and fragment intensity ratios, as exemplified by the target peptide EVILIDPFHK++ (Supplementary Fig. We concluded from our benchmarking with immunopeptidome dilutions that the targeted branch enables more sensitive and confident detection of peptides of interest compared to the discovery branch, and that NeoDiscMS does not compromise quantitative performance relative to DDA. To further characterize the utility of our method we acquired immunopeptidomics data with DDA and NeoDiscMS from three patient tumor-derived melanoma cell lines: Ml1, Ml2, and MI3 (Fig. As targets, we selected for each cell line NeoDisc-prioritized sample-specific mutated neoantigen peptides from pairs of tumor-healthy WES and tumor transcriptome data. In addition, to explore various potential sources of antigens we further extended the lists to well characterized TAAs, such as MAGEA3, MAGEA4, and MLANA, that were found to be expressed in the samples, and predicted TAA-derived peptides from in silico three frame translation products. a The individual identification count (upper panel) of target peptides (x-axis) among DDA and NeoDiscMS replicates (n = 3 per condition). b The number of target identifications (TIDs; identification of precursor that triggered the targeted branch and was identified in either sMS2 and hMS2) made with either sMS2, hMS2 or both within an RTS event. Three adjacent blocks separated by thin black lines represent replicates 1,2, and 3. c The best TID hyperscore (left) or spectral angle (right) of each identified target peptide compared to the same peptide's best hyperscore or spectral angle in the discovery branch. As expected, DDA provided the greatest immunopeptidome depth across all three cell lines, with NeoDiscMS showing a mean reduced depth compared to DDA by 21%, 13%, and 4% for Ml1, Ml2, and Ml3, respectively. Overall, the immunopeptidomes maintain between 94–95% of binders, and the average peptide length was between 9.3–9.6 (Supplementary Fig. We identified in total 49 TAA-peptides among them several confirmed immunogenic epitopes37, and four mutated neoantigens out of which three were previously demonstrated to be immunogenic in the patient7,9 (Fig. Importantly, in each cell line, every target peptide identified with DDA was also identified with NeoDiscMS, apart from four targets, one of which detected three times and the others once across DDA replicates. Furthermore, the fraction of targets detected with NeoDiscMS that were identified in at least two out of three NeoDiscMS replicates but only in a single or none of the DDA replicates, was 16%, 24%, and 0% for Ml1, Ml2, and Ml3, respectively (Fig. For example, in Ml2, three out of five (THNVHEEKI, QHATIIHNL, THNIREEKI) SAGE1-derived peptides were identified consistently with NeoDiscMS but only once or never with DDA across all replicates (Fig. To better characterize the improvement in peptide detection confidence, we closely investigated the underling qualities of RTS events that follow when the targeted branch is activated. When the precursor mass is detected in the scheduled RT window, a sMS2 scan is triggered. The sMS2 spectrum is then subjected to RTS. When a peptide precursor triggers an RTS event and that very peptide is also eventually PSMed with MSFragger in either sMS2 or hMS2, we call it a target identification (TID). The fraction of RTS events that resulted in an RTS hit was 12%, 8%, and 8% on average for Ml1, Ml2, and Ml3, respectively. More than 85% of the inclusion list hits during a measurement do not pass RTSf, indicating that they are isobaric non-targets (Supplementary Fig. The fraction of RTS hit events that resulted in a TID was 1%, 6%, and 2% on average for Ml1, Ml2, and Ml3, respectively (Supplementary Fig. 5d) and all TIDs occurred in RTS hit events (Supplementary Fig. Across samples, at least 90% of all RTS hit events that yielded a TID fit into a ΔPPM window smaller than 1 (Supplementary Fig. Among all RTS events with TIDs, the majority were PSMed in both sMS2 and hMS2 scans. Ml2 particularly benefits from hMS2 scans, as the number of uniquely hMS2-derived TIDs is 60% higher than uniquely sMS2-derived TIDs (Fig. The RTS result of TIDs that were uniquely identified with hMS2 scans remained within the Xcorr range [0.4, 2.7] (Supplementary Fig. TIDs that succeeded for sMS2 scans and hMS2 scans within the same RTS events are the best possible measure for the immediate gain in sensitivity between these two scan types. Particularly for low scoring sMS2 TIDs, hyperscores and spectral angles of the same precursor PSM increased for subsequent hMS2 spectra (Supplementary Fig. b The retention time and identification rate of example neoantigen YPAAVNTIVAI in all three DDA and NeoDiscMS replicates of Ml2. c Zoom-in of the retention time window in NeoDiscMS technical replicate 1 of (b). Real-time search events where the real-time search filters are passed (RTS hit events) connect the respective sMS2 and hMS2 with a linker (gray arches). The real-time search parameters of each RTS hit event hovers above the respective linker. d The top YPAAVNTIVAI PSMs, based on spectral angle, for each scan type in DDA and NeoDiscMS demonstrate that hMS2 scans offer important improvements, including an increased number of identified fragments. The MCTS1D130N-derived neoantigen YPAAVNTIVAI found in MI2 showcases the clear advantages of implementing NeoDiscMS for the confident discovery of clinically relevant mutated neoantigens. Based on predictions, NeoDisc ranked this neoantigen at position 186 among predicted HLA-I neoantigens, while an autologous TIL-based immunogenicity assay confirmed its immunogenicity7,9. In a hypothetical patient case where neoantigen targets for a personalized cancer vaccine are identified solely through mutanome predictions, a neoantigen ranked as low as 186 would generally be excluded from selection, as typically up to 40 top ranked neoantigens are targeted38,39,40. However, in a clinical discovery effort combining immunopeptidomics with mutanome predictions, a robust MS support for this immunogenic neoantigen would justify its inclusion. Conversely, without solid MS backing, including such a low-ranked peptide over a higher-ranked neoantigen would not be justified. YPAAVNTIVAI was identified in all immunopeptidomic replicates with DDA, however, with only up to two PSMs per replicate. Conversely, with NeoDiscMS, repeated RTS hit events across successive acquisition cycles resulted in 9 to 13 PSMs per replicate, of which 5 to 7 of YPAAVNTIVA PSMs derived from hMS2 scans (Fig. Despite having a spectral angle of 0.9299, the best dMS2 YPAAVNTIVA PSM in the DDA contained only 7 fragments (b5–b9, y1 and y2), where none of the first four peptide bonds is covered by a fragmentation site (Fig. The best hMS2 PSM matched 12 fragments, (b2–b10, y1–y3), resulting in accurate and confident peptide identification. To conclude, NeoDiscMS resulted in consistent and confident YPAAVNTIVAI neoantigen detection with high spectral angles and number of matched fragments. To demonstrate its clinical applicability, we utilized our clinical immunopeptidome pipeline9 to extract the immunopeptidome from three distinct tumor lesions (Ti1, Ti2, Ti3) of a uveal melanoma patient, with each NeoDiscMS analysis performed on the equivalent of 10 mg of tissue. We selected the top 500 predicted neoantigens as well as the top 518 predicted TAA-derived peptides as targets in a single list that was used for NeoDiscMS measurements of all three tissue samples. Our clinical routine immunopeptidomics acquisition scheme includes both DDA and DIA measurements allowing sensitive peptide-centric DIA searches against a sample-specific hybrid spectra library generated from spectrum-centric analysis of the DIA file combined with the library of generated from the DDA file9,41. Therefore, we acquired one technical DIA and NeoDiscMS acquisition for each tissue sample, using a personalized NGS-guided target database. With NeoDiscMS for each tissue sample individually, we identified 14,797, 16,033 and 11,968 unique peptides at an average length of 9.4 and with 95% of binders for Ti1, Ti2, Ti3, respectively (Fig. For all three tissues, the number of identified peptides with spectrum-centric searches of DIA data (DIA-Umpire) amounts to less than 50% of the depth that was achieved with a corresponding NeoDiscMS search. An examination of the shared and sample-specific peptide sets revealed differences in peptide presentation across the three samples. The fraction of peptides common to all three lesions is 31% in the NeoDiscMS search and 36% in the hybrid library DIA search comparison (Fig. As the tumor mutation burden was expectedly low in this uveal melanoma tumor lesions (2.601, 2.82, and 3.615 somatic mutations/Mb for Ti1, Ti2, and Ti3, respectively), no neoantigens were identified in any of the samples. Yet, nine TAA-derived peptides were identified in total, derived from MLANA, TYR, and SAGE1, with peptide MPREDAHF from MLANA detected in the DIA files of Ti1, Ti2, and Ti3, and peptide YRALMDKSL from MLANA in Ti1, only when analyzed against the hybrid library and not with the spectrum-centric approach. Hence, we demonstrated that NeoDiscMS can be seamlessly combined with DIA to create sample-specific hybrid libraries enriched with high-confidence MS2 spectra, for the sensitive detection and quantification of the global immunopeptidome as well as target peptides (Fig. a The analysis of three low-input uveal melanoma tissues (10 mg/injection) of the same patient (Ti1, Ti2, and Ti3, respectively), reveals a global depth of 12,316, 13,490, and 10,232 immunopeptides (binders only) with a DIA-NeoDiscMS hybrid-library search, with more than 95% of the identified peptides predicted as binders by MixMHCpred (%-rank <2). b Venn diagram describing the intersection of presented immunopeptides between Ti1, Ti2, and Ti3 with either a spectrum centric NeoDiscMS search (left) or a DIA-NeoDiscMS hybrid-library search (right). c Identified TAAs with either NeoDiscMS or spectrum-centric searches of DIA data, or a peptide-centric search of DIA with a DIA-NeoDiscMS hybrid-library. In this study we present a new immunopeptidomic MS acquisition method that has been developed specifically to overcome critical challenges related to the discovery of clinically relevant antigens. Tumor-specific antigens are often present at low abundance, yet in clinical settings, they require rapid, robust, and accurate detection to enable their use as immunotherapeutic targets. Recently we reported NeoDisc, a clinical end-to-end computational pipeline that combines genomics, transcriptomics, and MS-based immunopeptidomics for the prediction and direct identification of immunogenic tumor-specific HLA-I and -II antigens from multiple sources. So far, once the NGS and MS-immunopeptidomic files have been generated, they were subjected to the NeoDisc computational integration. Here, we propose an alternative approach. Matched tumor and germline genome data, along with tumor transcriptome data, are processed with NeoDisc to create a tumor sample-specific personalized proteome reference. The reference is annotated with single-nucleotide polymorphisms and somatic mutations, which are then used for HLA binding and machine learning-based immunogenicity prediction of antigenic target peptides. From this prioritized list of targets, NeoDisc generates a personalized inclusion list for NeoDiscMS acquisition of the MS analysis of the matched tumor immunopeptidome sample, followed by a DIA acquisition. The immunopeptidomic MS data is then searched by NeoDisc against the personalized proteome to identify naturally presented peptides, refining the list to prioritize the most clinically relevant and likely immunogenic targets. This streamlined process maintains a 2-week turnaround from biopsy reception to the compilation of a ranked list of targets, for example for neoantigen vaccine manufacturing. NeoDiscMS is a scalable, spike-in-free immunopeptidomic discovery workflow that can also operate independently of NeoDisc, using any list of target peptides of interest. Importantly, the key use case for NeoDiscMS are highly limited biopsies with strict time limits for data acquisition, whereby both target sensitivity and global depth are required. Samples are prepared following standard immunopeptidomic protocols, and, unlike other targeted approaches, no prior MS measurements of the investigated samples are required. This allows the entire sample to be utilized at once, enhancing detection and decreasing turnaround time. During acquisition, the separation of discovery and targeted branch as well as fully customizable parameters for each scan type, facilitates maximal flexibility to optimize the trade-off between target sensitivity and global depth. The edge of NeoDiscMS over ilDDA lies in more efficient application of high-sensitivity scans. Long injection times on every analyte that is isobaric to a target within the scheduled RT window becomes increasingly troublesome as inclusion lists are scaled: the number of targets as well as the scheduled RT window size are critical for inclusion list burden. NeoDiscMS significantly reduces time spent sampling off-target ions. This leaves more time within an acquisition cycle to sample ions of other target masses or with the discovery branch. An additional advantage of NeoDiscMS is its ability to sample the same precursor up to three times per cycle, compared to a maximum of two times in ilDDA. This increased sampling frequency explains the higher global depth and the equal or slightly improved target sensitivity observed with NeoDiscMS relative to ilDDA. The decision tree of the targeted branch we present is simple (sMS2—RTSf—hMS2) and in the future, sophistication of RTS event designs and sMS2-hMS2 dependencies are expected to further boost the sensitivity while minimizing loss of global depth. Selective high-sensitivity scans can expand beyond the optimized ion accumulation and collisional energy parameters that we showcase to, for instance, ion mobility parameters20,42. Also, narrower isolation windows could provide cleaner spectra with less interfering peaks, at the cost of sensitivity or injection time. Dynamic definition of data aquisition parameters with help of sMS2 scans, just like the refinement of real-time search evaluation parameters, bear great potential to further advance target identification. Acquiring multiple scans of the same precursor of interest within and between cycles offers novel opportunities for peptide quantification and generation of consensus spectra. Additionally, precursor selection parameters like adjusted minimal intensity thresholds or better integration of inclusion list and RTS FASTA as well as optimized target precursor selection offer further possibilities for refinement. The targeted branch of NeoDiscMS is committed to the holy grail of clinical immunopeptidomics: the direct identification of cancer-specific antigens. Yet, we would like to highlight the substantial importance of the discovery branch that provides global depth in a DDA-like fashion. Firstly, the global immunopeptidome informs us about dynamics of antigen processing and presentation hotspot features, as well as defects in antigen processing and presentation, which can directly inform immunogenicity predictions and neoantigen prioritization7,9,10,43,44,45. Naturally, the discovery branch can also contribute additional spectra of the targets as well as of other non-targeted exploratory antigen candidates. One great advantage of mass spectrometry data is that it can be re-processed with new search settings and software to extract new information or build large databases36,46,47. The discovery branch provides the bulk, in absolute terms, of high-quality spectra in a NeoDiscMS data file, being “TopN”-sampled, and provide utility for this purpose. Furthermore, the PSMs that the discovery branch yields during data processing enables statistically solid FDR calculations48 by providing highly diverse target and decoy matches, without being restricted to only sampling ions that correspond to the ~5000 masses in the inclusion list of the targeted branch. Additionally, high-quality PSMs by the discovery branch are very likely to be identified and instrumental for effective feature extraction in rescoring algorithms such as MSBooster47. As such, the discovery branch provides not only additional depth but also enables the effective bioinformatic data processing with commonly used search engines so that targets can be identified from the targeted branch with higher sensitivity. Being able to process data with popular proteomics tools such as MSFragger49, is crucial for the user-friendliness of the NeoDiscMS approach. The application of chimeric spectrum deconvolution and wide isolation windows in the discovery branch boosts global coverage while maintaining high identification quality. As such, we show that chimeric spectrum deconvolution with MSFragger-DDA+ can be applied to immunopeptidomics data to great benefit. While NeoDiscMS can be seamlessly processed by MSFragger, we also show how NeoDiscMS offers opportunities for more extensive data processing for MS2-based quantification of precursors that were identified as the isolated mass multiple times. Our data shows that the targeted and the discovery branches contribute improvements to NeoDiscMS compared to conventional DDA approaches. The increased isolation window size in combination with chimeric spectral deconvolution results in increased immunopeptidome depth, as shown in a direct comparison of regular and wide window isolation windows on the same sample, analyzed with MSFragger-DDA or MSFragger-DDA+. With the help of JY, RA957, and melanoma-derived cell lines, we show that the targeted branch improved target identification significantly, consistently, and in a scalable manner. By examining RTS event dynamics, we show how and at what rate sMS2 and RTSf selectively trigger high-sensitivity scans. This analysis reveals the individual contributions of sMS2 and hMS2 scans. We highlight YPAAVNTIVAI, an immunogenic neoantigen that is a direct beneficiary of the increased sensitivity that NeoDiscMS offers: YPAAVNTIVAI's identification confidence clearly and consistently improves with NeoDiscMS compared to DDA. We demonstrate the clinical utility of NeoDiscMS with a set of three low-input tumor biopsies. By simulating a clinical setting, we achieve substantial depth as well as the identification of TAA-derived peptides and demonstrate interoperability with DIA library searching strategies. Spectrum-centric searches of DIA data yielded substantially fewer peptide identifications compared to NeoDiscMS (Fig. This makes NeoDiscMS a suitable candidate for library building to fully harness the power of DIA library searches. This approach holds great promise, yet it's performance to provide more depth in immunopeptidomics compared to experimental libraries is, to date, situational41,50,51. Also, accurate FDR calculations for large-scale predicted libraries for peptide-centric searches might face processing-specific challenges that need to be evaluated and addressed, just as it is the case for DDA database searches52,53. Besides the construction of experimental libraries, another key complementary feature of NeoDiscMS to DIA is that it provides low-complexity spectra with its targeted branch, compared to DIA MS2 spectra. This is crucial when identifying peptides for clinical prioritization, where manual inspection of signal and background fragments of candidate PSMs is common. The manual inspection and validation of a PSM from DIA MS2 spectra is virtually impossible due to its massive complexity, even for the most advanced experts. If data acquisition approaches that are still being developed, such as narrow-window DIA54, will shift this balance remains to be seen. At the time of writing, the instruments equipped with vendor-supported acquisition method editors that enable implementation of NeoDiscMS are the Orbitrap Eclipse and Orbitrap Ascend (part of Thermo Fisher Scientific's Tribrid Mass Spectrometer series). We envision broader vendor adoption of this approach in the future, as well as the integration of advanced RTS capabilities that extend beyond immunopeptidomics applications. NeoDiscMS is, to our knowledge, the first personalized immunopeptidomics data acquisition strategy tailored to the needs of a proteogenomics-based clinical T-cell antigen discovery pipeline. Besides analytical aspects, its guiding design principles were oriented towards modularity, flexibility, and user-friendliness. We conclude that NeoDiscMS advances proteogenomic-based antigen discovery with all its clinical benefits for the development of cancer therapies. Generation of NeoDiscMS files (target selection, FASTA database, and inclusion list) is integrated within NeoDisc v1.7.1 and is available through https://neodisc.unil.ch/. An informed written consent was given by the participants, according to the requirements of the local Ethics Committee institutional review board (Ethics Commission, Commission cantonale d'éthique de la recherche sur l'être humain (CER-VD) Centre hospitalier universitaire Vaudois, CHUV). Ti1, Ti2, and Ti3 samples used in this investigation were collected from the patient during the screening process for inclusion in a Phase 1 trial (NCT04643574), following enrollment in a research protocol approved by the local Ethics Committee (Commission cantonale d'éthique de la recherche sur l'être humain (CER-VD); BASEC ID 2017-00305). Tumor cell lines were generated as described in Huber et al.9. Ml1 and Ml2, including immunogenicity assessments and NGS data, are the same as described in Müller et al.7, where they equate to patient1 and patient3, respectively. Ml3, including immunogenicity assessments, corresponds to Mel-4 P1 HLA-I high cells, stimulated with 100 U/mL of Interferon-γ (Miltenyi Biotec, ref. HLA-restricted TAA immunogenicity assessments are derived from CEDAR37. All cells were expanded in cell culture medium (Gibco, ref. 61870-010) with 10% of heat-inactivated FBS (Gibco, ref. To harvest the JY (ATCC, ref. 77441) and RA957 cell lines we collected the cell culture medium containing the cells in suspension, removed the medium, and washed cell twice in PBS (Bichsel, ref. To harvest melanoma cell lines we removed the cell culture medium, washed the cells with PBS, detached cells from the flask surface with Trypsin (BioConcept, ref. 5-51F00-H), inactivated trypsin with cell culture medium, and washed the cells twice with PBS. We stored cells in pellets of 108 at −80 °C. For immunopeptide enrichment we harvested W6/32 antibodies from HB-95 hybridoma cells (HB-95, ATCC), cross-linked them with DTT to Sepharose beads (Invitrogen, ref. We then lysed our samples with 0.25% sodium deoxycholate (Sigma-Aldrich, ref. I6125-5g), 1 mM EDTA (Thermo Fisher Scientific, ref. 04693132001), 1 mM phenylmethylsulfonyl fluoride (Roche, ref. Cell lines were just resuspended in lysis buffer, tissue samples were first homogenized with a bead homogenizer (Tissue Lyser II, Qiagen) for 1 min at 30 Hz. To isolate HLA-peptide complexes, we applied the workflow presented in Chong et al.55. Shortly, 96-well plates were conditioned and mounted with 75 µl of cross-linked beads. We then added each sample's lysate to a well. After the lysate drained, we washed each well with varying salt concentrations and eluted the captured HLA-peptide complexes with 1% TFA (Merck Millipore, ref. Then we desalted and isolated immunopeptides with the help of C18 columns, using 28% ACN (Biosolve, UN 1648), 0.1% TFA as an elution buffer. The peptides eluted from the C18 column were dried in a vacuum concentrator (Concentrator plus, Eppendorf) and stored at −80 °C. Prior to MS analysis, peptides were re-suspended in 30 µl (cell lines) or 18 µl (tissue samples) 0.1 %TFA. MS iRT kit peptides (Biognosys, ref. For all experiments with cell lines, we isolated immunopeptides from samples of 108 cells each. For all experiments with tissues, we processed ~60 mg of each tissue. For cell lines, we injected an equivalent of 5*106 or 107 cells (Supplementary Data 1). In addition, we included a negative control sample with only the JY peptides. Importantly, all samples contained equal amount of JY peptides. WES and RNAseq libraries were prepared as described in Huber et al.9. Briefly, DNA was extracted with the commercially available DNeasy Blood and Tissue Kit (Qiagen, ref. RNA was extracted by phase separation using Trizol (Thermo Fisher Scientific, ref. C2432-25ML) and centrifugation at 12,000 × g for 15 min. Then, the aqueous phase containing the RNA was collected, mixed with 70% ethanol and loaded on an RNeasy mini spin column from the total RNA isolation RNeasy mini kit (Qiagen, ref. WES and RNAseq libraries were prepared and sequenced at Microsynth using the Agilent SureSelect XT Human All exome V7 kit (Agilent, ref. NeoDisc v1.7.0 was used for neoantigen discovery and prioritization9. The pipeline was run in fastq mode using default parameters for single-sample analysis on paired germline and tumor whole-exome sequencing data, along with matched tumor RNA sequencing data. Peptide lengths for HLA-I and HLA-II predictions were defined in the configuration file as 8–12 and 12–15 amino acids, respectively, with binding affinity thresholds set to ≤2.0% for both HLA-I and HLA-II predictions. Following single-sample analyses, results were combined using NeoDisc's MergeSamples tool to facilitate multi-sample integration and prioritize tumor-specific antigens. The NeoDiscMS approach require the generation of a list of target peptides. In this study we tested the performance with ~1500 targets sequences. In general, any peptide sequence can be included as target. For the JY-RA957 dilution experiment, we selected RA957-specific HLA-I bound peptides identified in previously published eight raw files of RA957 immunopeptidome samples55. We processed the raw files with MSFragger-DDA+ and filtered all identified peptides to retain peptides of length 8–11, that occurred in all eight measurements, were predicted by MixMHCpred36 (v2.3) to be A*68:01 binders with binding %-rank <2, and did not have a %-rank  <10% for any other RA957 or JY alleles. For each of the melanoma cell lines Ml1, Ml2, and Ml3, 500 predicted neoantigens with %-rank <3% were selected based on their prioritization by NeoDisc9. In addition, we supplemented the inclusion lists with peptides derived from three-frame translation of TAA genes (Supplementary Data 2). Here, we filtered all 8–14mers so that each had %-rank of <2 for at least two HLA-A or HLA-B alleles of the respective melanoma cell lines. We then selected those 1000 peptides for each melanoma cell line that had the highest %-ranks for any of the alleles of the respective patient's haplotype. Sample HLA-typing information can be found in Supplementary Table 2. For the melanoma tissues Ti1, Ti2, and TI3, the 500 top neoantigens as prioritized by NeoDisc were selected as targets. Additionally, we added the 518 TAA-derived class I peptides that were predicted by NeoDisc in the “ExpressedTAAs” table. Sample HLA-typing information can be found in Supplementary Table 2. To efficiently schedule inclusion list entries and improve target detection, retention time prediction for target peptides is recommended. This approach minimizes the need for targeted scans to search for all peptides across the entire gradient, thereby reducing the loss in global coverage. For retention time prediction, the following are required: (1) retention times of calibration peptides, (2) a list of target peptides, and (3) a retention time prediction tool, such as DeepLC56. We used calibration peptide retention times derived from PSMs from a preceding tryptic HeLa digest (Thermo Fisher Scientific, ref. The calibration run should ideally be conducted shortly before the NeoDiscMS measurements and on the same column, ensuring multiple washed between them to minimize carry over. A spectral library was generated from the calibration run with FragPipe (v22.0) with default settings for tryptic digests, including methionine oxidation as a variable modification. Next, we predicted the RT of our list of targets with DeepLC (v3.1.1), with help of the RT peptide calibration file. Importantly, we included all methionine oxidation variations of target peptide sequences. To schedule our inclusion list, we created windows of ±15 min around the RT of each modified peptide (Supplementary Fig. Last, we generated the inclusion lists in the format dictated by manufacturer's method development software Thermo Xcalibur Instrument Setup (v2.0), including peptides with charge states 1–3. For the RTS, we created dedicated FASTA files that consisted individual target peptide sequences as entries. For the melanoma cell line and the clinical tissue experiments, RTS FASTA headers were annotated as neoantigen or TAA. An example of sample Ml3 for the DeepLC calibration peptides file, DeepLC target peptides file, scheduled inclusion list, as well as a FASTA needed for RTS are provided as Supplementary Data 3–6. An example acquisition tree, as seen in Thermo Xcalibur Instrument Setup, the order of adding the acquisition tree parameters with the drag-and-drop buttons, as well as where to find real-time search parameters is illustrated in Supplementary Fig. For all NeoDiscMS experiments, RTS parameters were set as visualized in Supplementary Fig. Across experiments, settings remained the same apart from the inclusion list and FASTA used. Importantly, at the time of writing, all methods that apply RTS required a digestion enzyme as an input. Nevertheless, the database search then takes all digested peptides as well as the full FASTA sequences into account. Since the FASTA entries directly reflected the peptide sequences, we aimed to minimize forced digestion. In addition, no static modifications were allowed, and methionine oxidation was set as the only variable modification. We allowed up to three variable modifications/peptide. Maximum search time was set to 40 ms. To pass RTSf, the scoring thresholds were set to Xcorr -> 0.4, dCn -> 0, Precursor PPM -> 5 for charge states 1–3. For all DIA experiments, we applied an MS1 mass range of 300–1650Th at a resolution of 120,000 and 27 DIA windows of sizes [37, 30, 24, 24, 22, 23, 44, 21, 24, 24, 25, 27, 27, 30, 35, 38, 43, 53, 72, 103, 594] with an AGC target of 2000%, the injection time set to auto, MS2 mass resolution set to 30,000, and a stepped CE of [27, 30, 32]. The injection volumes of each sample as well as the respective sample input are indicated in Supplementary Data 1. DDA, ilDDA, and NeoDiscMS acquisition cycles were constructed as described in Fig. Cycle times for all methods are 3 s. Detailed scan type parameters for all scan types are described in Supplementary Table 3. We used an Easy-nLC 1200 coupled to an Orbitrap Eclipse Tribrid mass spectrometer (Thermo Fisher Scientific). Our analytical columns were 450 mm long with an inner diameter of 75 μm and an 8 μm tip (TSP-075375, BGB Analytik). For column packing we used C18 beads with a particle size of 1.9 µm and a pore size of 120 angstrom (r119.aq, Dr. Maisch). For all experiments, LC gradients used 0.1% formic acid (Thermo Scientific, prod nr. 85178) as solvent A and 80% ACN, 0.1% formic acid as solvent B, a gradient length of 125 min, and a flow rate of 250 nL/min. To generate a calibration run for the DeepLC RT prediction, we injected 100 ng of tryptic HeLa digest that was measured with the same gradient above, with a MSFragger-DDA database search with all human Swissprot57 sequences without isoforms (28.3.2024, 20419 entries) as well as the “common contaminants” list provided by FragPipe. We kept all default setting for tryptic searches in place. For immunopeptidomics searches with MSFragger-DDA, we applied a DDA database search with all human Swissprot sequences without isoforms (28.3.2024, 20,419 entries) as well as iRT peptide sequences and the “common contaminants” list provided by Fragpipe as a FASTA database. Digestion was set to unspecific, peptides of lengths 8–14 were included. Methionine oxidation, N-terminal acetylation and cysteine carbamidomethylation were set as variable modifications. Only precursors with charges 1–3 were permitted for PSMing. Protein FDR was set to 1. Peptide, ion, and psm FDRs were set to 0.01. For immunopeptidomics searches with MSFragger-DDA+ we applied DDA+ database searches with different FASTA databases for different experiments, though they were all concatenated with iRT peptide sequences and the “common contaminants” list provided by Fragpipe. For the JY- and RA957-based experiments we used all human Swissprot sequences without isoforms (28.3.2024, 20,419 entries). For melanoma cell lines and melanoma tissues the respective sample-specific FASTA generated by NeoDisc were used. Digestion was set to unspecific, peptides of lengths 8–14 were included. Methionine oxidation, N-terminal acetylation and cysteine carbamidomethylation were set as variable modifications. Only precursors with charges 1–3 were permitted for PSMing. DDA+-specific parameters were left at default. Protein FDR was set to 1. Peptide, ion, and psm FDR was set to 0.01 for with the SwissProt FASTA or 0.03 with group-specificity52,58 for searches with NeoDisc FASTAs, respectively. For elution profile visualization, raw files and target precursor sequence were imported into Skyline v22.259 for visualization. For the spectrum-centric search with DIA-Umpire of DIA data from the melanoma tissue experiment, we did a database search with DIA-Umpire using a NeoDisc-generated FASTA. Digestion was set to unspecific, peptides of lengths 8–14 were included. Methionine oxidation, N-terminal acetylation and cysteine carbamidomethylation were set as variable modifications. DIA-Umpire-specific parameters were left at default. Only precursors with charges 1–3 were permitted for PSMing. Protein FDR was set to 1. Peptide, ion, and PSM FDR was set to 0.03 with group-specificity. From the libraries generated with a spectrum-centric search of the respective DIA data and the NeoDiscMS MSFragger-DDA+ results of either Ti1, Ti2, or Ti3, we generated a hybrid library for each tissue separately. We used said hybrid library, per tissue, for a peptide-centric search of the DIA data with DIA-NN. For immunopeptidomics data analysis we extracted psm.tsv output files of all spectrum-centric searches and report.tsv of all peptide-centric searches for data analysis. For peptide binding affinity prediction we used MixMHCpred36 (v2.3). Raw file header information was extracted in R with help of the rawrr60 package (v1.12.0). We calculated peptide intensities by adding up all precursor intensities of a given peptide sequence. Data visualization was done with R in Rstudio with packages ggplot261 (v3.5.1), ggseqlogo62 (v0.2), ggside (v0.3.1), patchwork (v1.3.0), ggh4x (v0.2.8), gt (v0.11.0). NeoDiscMS was introduced in NeoDisc v1.7.1 and can be executed through three steps: (1) Run NeoDisc to identify and prioritize immunogenic tumor-specific antigens from NGS data and predict their chromatographic RTs, (2) perform NeoDiscMS-based measurements of the immunopeptidome, (3) resume the NeoDisc workflow to search NeoDiscMS immunopeptidomic files against personalized references, followed by prioritization of tumor-specific antigens. By running NeoDisc in fastq mode with the -neodiscmsselect flag, the user has to provide and specify in the configuration file a MS raw file of either shotgun proteomics or immunopeptidomics, as NEODISCMS_CALIBRATION parameter, that will be searched by FragPipe v22.0 for creating a calibration library used for RT prediction. This search is done against a GENCODE v43 protein-coding sequence reference, accounting for peptide modifications and the maximum number of modifications per peptide defined in the configuration file (MS_MODIFICATIONS and MS_MAXMODSPERPEPTIDE parameters). NeoDisc then partitions the prioritized HLA-I- and HLA-II-restricted peptide sequences into neoantigens, TAAs, and viral peptides, with the number of peptides in each group, as determined by the user in the NeoDisc configuration file (NEODISCMS_NEOCI, NEODISCMS_TAACI, NEODISCMS_VIRCI, NEODISCMS_NEOCII, NEODISCMS_TAACII, NEODISCMS_VIRCII). Their retention times is then predicted by DeepLC56 v2.2.36, where the above mentioned calibration run library (peptide identification list with measured retention times) is used, and by considering peptide modifications (MS_MODIFICATIONS and MS_MAXMODSPERPEPTIDE), maximum charges per peptide (NEODISCMS_CHARGES), collision energy (NEODISCMS_CE), gradient length (NEODISCMS_GRADIENT), and the retention-time window width (NEODISCMS_WINRT) as defined in the NeoDisc configuration file. NeoDisc generates separate lists of HLA-I and HLA-II target peptide sequences (fasta) along with their predicted retention times (csv) specifically designed for NeoDiscMS measurements. This step involves searching the NeoDiscMS immunopeptidomic raw files with both Comet v2024.02_0 - NewAnce v1.7.5 and Fragpipe v22.0 DDA+ against the personalized references generated by NeoDisc. To ensure reproducibility of results we performed all comparisons of different conditions in triplicates, as commonly used in the field. To investigate different technical conditions, such as different acquisition parameters, measurements were randomized. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw files and real-time search tables generated during the acquisition, as well as identified PSMs of NeoDiscMS measurement, have been deposited to the ProteomeXchange Consortium via the PRIDE63 partner repository with the dataset identifier PXD059824. NGS and immunogenicity assessments for Ml3 is reported in Huber et al.9, corresponds to Mel-4 P1 HLA-I high cells. WES and RNAseq data for TI1, Ti2, and Ti3 is deposited on the European Genome-Phenome Archive (EGA) with the dataset identifier EGAD50000001422. Chong, C., Coukos, G. & Bassani-Sternberg, M. Identification of tumor antigens with immunopeptidomics. Lang, F., Schrors, B., Lower, M., Tureci, O. Identification of neoantigens for individualized therapeutic cancer vaccines. Hundal, J. et al. pVAC-Seq: a genome-guided in silico approach to identifying tumor neoantigens. Zhou, C. et al. pTuneos: prioritizing tumor neoantigens from next-generation sequencing data. Unique neoantigens arise from somatic mutations in patients with gastrointestinal cancers. Tran, E. et al. Immunogenicity of somatic mutations in human gastrointestinal cancers. Muller, M. et al. Machine learning methods and harmonized datasets improve immunogenic neoantigen prediction. Gartner, J. J. et al. A machine learning model for ranking candidate HLA class I neoantigens based on known neoepitopes from multiple human tumor types. Huber, F. et al. A comprehensive proteogenomic pipeline for neoantigen discovery to advance personalized cancer immunotherapy. The immunopeptidome landscape associated with T cell infiltration, inflammation and immune editing in lung cancer. Direct identification of clinically relevant neoepitopes presented on native human melanoma tissue by mass spectrometry. Shapiro, I. E. & Bassani-Sternberg, M. The impact of immunopeptidomics: from basic research to clinical implementation. Kochin, V. et al. HLA-A24 ligandome analysis of colon and lung cancer cells identifies a novel cancer-testis antigen and a neoantigen that elicits specific and strong CTL responses. Predicting immunogenic tumour mutations by combining mass spectrometry and exome sequencing. Characterization of CD8(+) T-cell responses to non-anchor-type HLA class I neoantigens with single amino-acid substitutions. Ebrahimi-Nik, H. et al. Mass spectrometry driven exploration reveals nuances of neoepitope-driven tumor rejection. Kalaora, S. et al. Use of HLA peptidomics and whole exome sequencing to identify human immunogenic neo-antigens. Hirama, T. et al. Proteogenomic identification of an immunogenic HLA class I neoantigen in mismatch repair-deficient colorectal cancer tissue. Sensitive and quantitative detection of MHC-I displayed neoepitopes using a semiautomated workflow and TOMAHAQ mass spectrometry. Salek, M. et al. optiPRM: a targeted immunopeptidomics LC-MS workflow with ultra-high sensitivity for the detection of mutation-derived tumor neoepitopes from limited input material. Bailey, D. J., McDevitt, M. T., Westphall, M. S., Pagliarini, D. J. Intelligent data acquisition blends targeted and discovery methods. Matzinger, M., Muller, E., Durnberger, G., Pichler, P. & Mechtler, K. Robust and easy-to-use one-pot workflow for label-free single-cell proteomics. Matzinger, M. et al. Micropillar arrays, wide window acquisition and AI-based data analysis improve comprehensiveness in multiple proteomic applications. & Nesvizhskii, A. I. MSFragger-DDA+ enhances peptide identification sensitivity with full isolation window search. Frejno, M. et al. Unifying the analysis of bottom-up proteomics data with CHIMERYS. Light contamination in stable isotope-labelled internal peptide standards is frequent and a potential source of false discovery and quantitation error in proteomics. Sample multiplexing-based targeted pathway proteomics with real-time analytics reveals the impact of genetic variation on protein expression. Kreimer, S. et al. Advanced precursor ion selection algorithms for increased depth of bottom-up proteomic profiling. Active instrument engagement combined with a real-time database search for improved performance of sample multiplexing workflows. Schweppe, D. K. et al. Full-featured, real-time database searching platform enables fast and accurate multiplexed quantitative proteomics. Wichmann, C. et al. MaxQuant.Live enables global targeting of more than 25,000 peptides. Furtwangler, B. et al. Real-time search-assisted acquisition on a tribrid mass spectrometer improves coverage in multiplexed single-cell proteomics. Huffman, R. G. et al. Prioritized mass spectrometry increases the depth, sensitivity and data completeness of single-cell proteomics. Graumann, J., Scheltema, R. A., Zhang, Y., Cox, J. & Mann, M. A framework for intelligent data acquisition and real-time database searching for shotgun proteomics. Eng, J. K., McCormack, A. L. & Yates, J. R. An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. Bassani-Sternberg, M. et al. Deciphering HLA-I motifs across HLA peptidomes improves neo-antigen predictions and identifies allostery regulating HLA specificity. The Cancer Epitope Database and Analysis Resource (CEDAR). Sarivalasis, A. et al. A Phase I/II trial comparing autologous dendritic cell vaccine pulsed either with personalized peptides (PEP-DC) or with tumor lysate (OC-DC) in patients with advanced high-grade ovarian serous carcinoma. Rojas, L. A. et al. Personalized RNA neoantigen vaccines stimulate T cells in pancreatic cancer. Yarchoan, M. et al. Personalized neoantigen vaccine and pembrolizumab in advanced hepatocellular carcinoma: a phase 1/2 trial. Sensitive immunopeptidomics by leveraging available large-scale multi-HLA spectral libraries, data-independent acquisition, and MS/MS prediction. & Park, M. A. Gas-phase separation using a trapped ion mobility spectrometer. Petralia, F. et al. Pan-cancer proteogenomics characterization of tumor immunity. Muller, M., Gfeller, D., Coukos, G. & Bassani-Sternberg, M. Hotspots' of antigen presentation revealed by human leukocyte antigen ligandomics for neoantigen prioritization. Identification of immunogenic HLA class I and II neoantigens using surrogate immunopeptidomes. Lemke, S. et al. PCI-DB: a novel primary tissue immunopeptidome database to guide next-generation peptide-based immunotherapy development. Yang, K. L. et al. MSBooster: improving peptide identification rates using deep learning-based features. & Pevzner, P. A. Target-decoy approach and false discovery rate: when things may go wrong. Kong, A. T., Leprevost, F. V., Avtonomov, D. M., Mellacheruvu, D. & Nesvizhskii, A. I. MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics. Wahle, M. et al. IMBAS-MS discovers organ-specific HLA peptide patterns in plasma. Oliinyk, D. et al. diaPASEF analysis for HLA-I peptides enables quantification of common Cancer Neoantigens. Nesvizhskii, A. I. Proteogenomics: concepts, applications and computational strategies. Wen, B. et al. Assessment of false discovery rate control in tandem mass spectrometry analysis using entrapment. Guzman, U. H. et al. Ultra-fast label-free quantification and comprehensive proteome coverage with narrow-window data-independent acquisition. Chong, C. et al. High-throughput and sensitive immunopeptidomics platform reveals profound interferongamma-mediated remodeling of the human leukocyte antigen (HLA) ligandome. Bouwmeester, R., Gabriels, R., Hulstaert, N., Martens, L. & Degroeve, S. DeepLC can predict retention times for peptides that carry as-yet unseen modifications. Boutet, E., Lieberherr, D., Tognolli, M., Schneider, M. & Bairoch, A. UniProtKB/Swiss-Prot. Ferreira, H. J. et al. Immunopeptidomics-based identification of naturally presented non-canonical circRNA-derived peptides. MacLean, B. et al. Skyline: an open source document editor for creating and analyzing targeted proteomics experiments. Kockmann, T. & Panse, C. The rawrr R Package: direct access to orbitrap data and beyond. Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag, 2016). Wagih, O. ggseqlogo: a versatile R package for drawing sequence logos. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. We are thankful to Marie Taillandier Coindard, Laura Wessling, and HuiSong Pak for their support, as well as to all other members of the immunopeptidomics lab at the Department of Oncology UNIL CHUV for their contributions to the discussion of the manuscript. This study was supported by the Ludwig Institute for Cancer Research, by grant KFS-5637-08-2022 from the Swiss Cancer Research Foundation (M.B.-S.) and the PRIMA grant PR00P3_193079 from the Swiss National Science Foundation (M.B.-S.). Ilja E. Shapiro, Florian Huber, Justine Michaux & Michal Bassani-Sternberg Ilja E. Shapiro, Florian Huber, Justine Michaux & Michal Bassani-Sternberg Agora Cancer Research Centre, Lausanne, Switzerland Ilja E. Shapiro, Florian Huber, Justine Michaux & Michal Bassani-Sternberg Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the immunopeptidomics experiments including sample preparation, method implementation, data acquisition, and bioinformatical follow-up (data processing, analysis, and visualization). performed the NGS experiments (WES and RNAseq). The authors declare no competing interests. Nature Communications thanks Rupert Mayer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Shapiro, I.E., Huber, F., Michaux, J. et al. Sensitive neoantigen discovery by real-time mutanome-guided immunopeptidomics. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250806094118.htm'>This prehistoric predator survived global warming by eating bones</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 04:00:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>About 56 million years ago, when Earth experienced a dramatic rise in global temperatures, one meat-eating mammal responded in a surprising way: It started eating more bones. That's the conclusion reached by a Rutgers-led team of researchers, whose recent study of fossil teeth from the extinct predator Dissacus praenuntius reveals how animals adapted to a period of extreme climate change known as the Paleocene-Eocene Thermal Maximum (PETM). The findings, published in the journal Palaeogeography, Palaeoclimatology, Palaeoecology, could help scientists predict how today's wildlife might respond to modern global warming. Schwartz, Scott and another colleague used a technique called dental microwear texture analysis to study the tiny pits and scratches left on fossilized teeth. "They looked superficially like wolves with oversized heads," Schwartz said, describing them as "super weird mammals." But they had little tiny hooves on each of their toes." Before this period of rising temperatures, Dissacus had a diet similar to modern cheetahs, eating mostly tough flesh. But during and after this ancient period, its teeth showed signs of crunching harder materials, such as bones. This dietary shift happened alongside a modest reduction in body size, likely because of food scarcity. While earlier hypotheses blamed shrinking animals on hotter temperatures alone, this latest research suggests that limited food played a bigger role, Schwartz said. This period of rapid global warming lasted about 200,000 years, but the changes it triggered were fast and dramatic. Schwartz said studies of the past like his can offer practical lessons for today and what comes next. Animals that can eat a variety of foods are more likely to survive environmental stress. Such an insight may be helpful for modern conservation biologists, allowing them to identify which species today may be most vulnerable, he said. But adaptable species, including jackals or raccoons, might fare better. "In my earlier research, jackals in Africa started eating more bones and insects over time, probably because of habitat loss and climate stress." This may suggest that modern climate change could similarly disrupt food webs and force animals to adapt, or risk extinction, he said. Even though Dissacus was a successful and adaptable animal that lived for about 15 million years, it eventually went extinct. Scientists think this happened because of changes in the environment and competition from other animals, Schwartz said. Schwartz conducted his research using a combination of fieldwork and lab analysis, focusing on fossil specimens from the Bighorn Basin in Wyoming, a site with a rich and continuous fossil record spanning millions of years. Schwartz chose the location because it preserves a detailed sequence of environmental and ecological changes during the ancient period of climate warming. Schwartz has been interested in paleontology, specifically dinosaurs, since he was a boy, journeying with his father, an amateur fossil hunter, on treks through New Jersey's rivers and streams. He also wants to inspire the next generation of researchers. Note: Content may be edited for style and length. Why Whole Foods Are the Real Weight Loss Hack “Magic” Cleaning Sponges Found to Release Trillions of Microplastic Fibers More Exercise Isn't Always Better: New Study Reveals the Surprising Secret to a Younger Brain After 50 Years, a Neutrino Detector Finally Catches Elusive Ghost Particles Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250806094116.htm'>Scientists found the gene that makes Aussie skinks immune to deadly snake venom</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-07 03:33:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A University of Queensland-led study has found Australian skinks have evolved molecular armor to stop snake venom from shutting down their muscles. Professor Bryan Fry from UQ's School of the Environment said revealing exactly how skinks dodge death could inform biomedical approaches to treating snakebite in people. "What we saw in skinks was evolution at its most ingenious," Professor Fry said. "But in a stunning example of a natural counterpunch, we found that on 25 occasions skinks independently developed mutations at that binding site to block venom from attaching. "Incredibly, the same mutations evolved in other animals like mongooses which feed on cobras. "We confirmed with our functional testing that Australia's Major Skink (Bellatorias frerei) has evolved exactly the same resistance mutation that gives the honey badger it's famous resistance to cobra venom. The laboratory work validating the mutations was carried out at UQ's Adaptive Biotoxicology Laboratory by Dr Uthpala Chandrasekara who said it was incredible to witness. "It's fascinating to think that one tiny change in a protein can mean the difference between life and death when facing a highly venomous predator." "Understanding how nature neutralizes venom can offer clues for biomedical innovation," Dr Chandrasekara said. The research has been published in International Journal of Molecular Sciences. “Magic” Cleaning Sponges Found to Release Trillions of Microplastic Fibers More Exercise Isn't Always Better: New Study Reveals the Surprising Secret to a Younger Brain After 50 Years, a Neutrino Detector Finally Catches Elusive Ghost Particles Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            