
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-25</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65888268/excavation-changes-roman-empire-history-1756134604/'>A Staggering Excavation Has Rewritten the Fall of the Roman Empire</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 15:20:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Interamna Lirenas has turned out to be far more than a “backwater town” of the Roman Empire. According to a published study in Roman Urbanism in Italy, this central Italian town thrived well beyond previous belief, using its impressive urban features and forward-thinking design to stave off the effects of the empire's collapse well into the 3rd century AD. “We started with a site so unpromising that no one had ever tried to excavate it,” Alessandro Launaro, the study's author and Interamna Lirenas Project lead at the University of Cambridge's Classics Faculty, said in a statement. It turns out that Interamna Lirenas survived for around 300 years longer than previously believed, and was a flourishing town to boot. “But what we discovered wasn't a backwater, far from it. We found a thriving town adapting to every challenge thrown at it for 900 years.” They then launched a series of targeted excavations to unearth the history. “We think many other average Roman towns in Italy were just as resilient. By focusing on common ware pottery used for cooking—and not the imported pottery that often shows evidence of high-status living—the team could better map the location and dates of citizen movement in the region. “Based on the relative lack of imported pottery,” Launaro said, “archeologists have assumed that Interamna Lirenas was a declining backwater. We now know that wasn't the case.” Instead of favoring imported pottery, the town—which was likely home to about 2,000 residents—was busy making their own way. Thanks to a found inscription, researchers also believe that the town was likely visited by Julius Caesar in 46 BC, likely because Intermana Lirenas was part of a regional urban network and ideally situated between a river and major road. “This town continually played its cards right,” Launaro said, “it was always forging relations with communities between Rome and southern Italy while thriving as a trading hub.” Of course, the River Liri may have helped in that department—the town may have served as a river port. “River ports didn't just need warehouses,” Launaro said. “People spent a lot of time working and resting in the vicinity, so they needed all kinds of amenities, just like the ones we found here.” The theater was in a state of growth, not decline. Three bath complexes—with evidence of continued use and upkeep beyond Roman Italy's decline—and housing that showed no signs of zoning or separation by social status further contributed to the town's apparently thriving status. Throughout the 60 acres of their survey,  the team identified 19 courtyard buildings that they believe could have been markets, guild houses, warehouses, or apartments. The archaeologists believe they found a sheep and cattle market, which would have been key to the region's thriving wool trade. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Is This Mosaic Evidence of the First Nursing Home? An Archaeology Student Hit Gold on Her Debut Dig Scientists Found a Lost Maya City in the Jungle This Clay Seal Might Belong to a Biblical Figure</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41566-025-01719-9'>Integrated electro-optic digital-to-analogue link for efficient computing and arbitrary waveform generation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 15:09:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The rapid growth in artificial intelligence and modern communication systems demands innovative solutions for increased computational power and advanced signalling capabilities. Integrated photonics, leveraging the analogue nature of electromagnetic waves at the chip scale, offers a promising complement to approaches based on digital electronics. To fully unlock their potential as analogue processors, establishing a common technological base between conventional digital electronics and analogue photonics is imperative for building next-generation computing and communications systems. However, the absence of an efficient interface has thus far critically challenged a comprehensive demonstration of the advantages of analogue photonic hardware, with the scalability, speed and energy consumption as primary bottlenecks. Here we address this challenge and demonstrate a general electro-optic digital-to-analogue link enabled using foundry-based lithium niobate nanophotonics. Using purely digital electronic inputs, we achieve the on-demand generation of both analogue optical and electronic waveforms at information rates of up to 186 Gb s−1. The optical waveforms address the digital-to-analogue electro-optic conversion challenge in photonic computing, showcasing high-fidelity Modified National Institute of Standards and Technology image encoding with an ultralow power consumption of 0.058 pJ b−1. The electronic waveforms enable a pulse-shaping-free microwave arbitrary waveform generation method with ultrabroadband tunable delay and gain. Our results pave the way for efficient and compact digital-to-analogue conversion paradigms enabled by integrated photonics, and underscore the transformative impact that analogue photonic hardware may have on various applications, such as computing, optical interconnects and high-speed ranging. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data needed to evaluate the conclusions of the paper are available within the Article and its Supplementary Information. LeCun, Y., Bengio, Y. & Hinton, G. Deep learning. Google Scholar Winzer, P. J., Neilson, D. T. & Chraplyvy, A. R. Fiber-optic transmission and networking: the previous 20 and the next 20 years. Google Scholar Graves, A. et al. Hybrid computing using a neural network with dynamic external memory. Google Scholar Akopyan, F. et al. TrueNorth: design and tool flow of a 65 mW 1 million neuron programmable neurosynaptic chip. Google Scholar Chen, X. et al. All-electronic 100-GHz bandwidth digital-to-analog converter generating PAM signals up to 190 GBaud. J. Lightwave Technol. Google Scholar & Lee, N. Terahertz communications: challenges in the next decade. Terahertz Sci. Google Scholar B. Attojoule optoelectronics for low-energy information processing and communications. J. Lightwave Technol. Google Scholar Wetzstein, G. et al. Inference in artificial intelligence with deep optics and photonics. Google Scholar Shastri, B. J. et al. Photonics for artificial intelligence and neuromorphic computing. Google Scholar Capmany, J. & Novak, D. Microwave photonics combines two worlds. Google Scholar Marpaung, D., Yao, J. & Capmany, J. Integrated microwave photonics. Google Scholar McMahon, P. L. The physics of optical computing. Google Scholar Marković, D., Mizrahi, A., Querlioz, D. & Grollier, J. Physics for neuromorphic computing. Google Scholar Rashidinejad, A., Li, Y. & Weiner, A. M. Recent advances in programmable photonic-assisted ultrabroadband radio-frequency arbitrary waveform generation. IEEE J. Quantum Electron. Google Scholar Bogaerts, W. et al. Programmable photonic circuits. Google Scholar Shu, H. et al. Microcomb-driven silicon photonic systems. Google Scholar Zhang, M., Wang, C., Kharel, P., Zhu, D. & Lončar, M. Integrated lithium niobate electro-optic modulators: when performance meets scalability. Google Scholar Xu, M. et al. Dual-polarization thin-film lithium niobate in-phase quadrature modulators for terabit-per-second transmission. Google Scholar Patel, D., Samani, A., Veerasubramanian, V., Ghosh, S. & Plant, D. V. Silicon photonic segmented modulator-based electro-optic DAC for 100 Gb/s PAM-4 generation. IEEE Photonics Technol. Google Scholar Dubé-Demers, R., LaRochelle, S. & Shi, W. Low-power DAC-less PAM-4 transmitter using a cascaded microring modulator. Google Scholar Simard, A. D., Filion, B., Patel, D., Plant, D. & LaRochelle, S. Segmented silicon MZM for PAM-8 transmissions at 114 Gb/s with binary signaling. Google Scholar Huynh, T. N. et al. Flexible transmitter employing silicon-segmented Mach–Zehnder modulator with 32-nm CMOS distributed driver. J. Lightwave Technol. Google Scholar Samani, A. et al. Experimental parametric study of 128 Gb/s PAM-4 transmission system using a multi-electrode silicon photonic Mach Zehnder modulator. Google Scholar Sobu, Y. et al. High-speed optical digital-to-analog converter operation of compact two-segment all-silicon Mach–Zehnder modulator. J. Lightwave Technol. Google Scholar Jafari, O., Zhalehpour, S., Shi, W. & LaRochelle, S. DAC-less PAM-4 slow-light silicon photonic modulator providing high efficiency and stability. J. Lightwave Technol. Google Scholar Wang, C. et al. Integrated lithium niobate electro-optic modulators operating at CMOS-compatible voltages. Google Scholar Hu, Y. et al. On-chip electro-optic frequency shifters and beam splitters. Google Scholar Hu, Y. et al. Integrated electro-optics on thin-film lithium niobate. Google Scholar Zhang, M., Wang, C., Cheng, R., Shams-Ansari, A. & Lončar, M. Monolithic ultra-high-Q lithium niobate microring resonator. Zhu, X. et al. Twenty-nine million intrinsic Q-factor monolithic microresonators on thin-film lithium niobate. Zhou, H. et al. Photonic matrix multiplication lights up photonic accelerator and beyond. Cundiff, S. T. & Weiner, A. M. Optical arbitrary waveform generation. Wang, J. et al. Reconfigurable radio-frequency arbitrary waveforms synthesized in a silicon photonic chip. Tan, M. et al. Photonic RF arbitrary waveform generator based on a soliton crystal micro-comb source. J. Lightwave Technol. Fischer, B. et al. Autonomous on-chip interferometry for reconfigurable optical waveform generation. Cox, C. H., Ackerman, E. I., Betts, G. E. & Prince, J. L. Limits on the performance of RF-over-fiber links and their impact on device design. Kharel, P., Reimer, C., Luke, K., He, L. & Zhang, M. Breaking voltage–bandwidth limits in integrated lithium niobate modulators using micro-structured electrodes. St-Arnault, C. et al. Net 3.2 Tbps 225 Gbaud PAM4 O-band IM/DD 2 km transmission using FR8 and DR8 with a CMOS 3 nm SerDes and TFLN modulators. Preprint at https://arxiv.org/abs/2503.24147 (2025). Della Torre, A. et al. Folded electro-optical modulators operating at CMOS voltage level in a thin-film lithium niobate foundry process. Xue, S. et al. Full-spectrum visible electro-optic modulator. Renaud, D. et al. Sub-1 Volt and high-bandwidth visible to near-infrared electro-optic modulators. He, L. et al. Low-loss fiber-to-chip interface for lithium niobate photonic integrated circuits. Liu, X. et al. Ultra-broadband and low-loss edge coupler for highly efficient second harmonic generation in thin-film lithium niobate. Cheng, R. et al. Single-drive electro-optic frequency comb source on a photonic-wire-bonded thin-film lithium niobate platform. A. et al. High-power and narrow-linewidth laser on thin-film lithium niobate enabled by photonic wire bonding. Zeng, B. et al. Cryogenic packaging of nanophotonic devices with a low coupling loss <1 dB. Assumpcao, D. et al. A thin film lithium niobate near-infrared platform for multiplexing quantum nodes. Botez, D., Garrod, T. & Mawst, L. J. High CW wallplug efficiency 1.5 micron-emitting diode lasers. In 2015 IEEE Photonics Conference (IPC) 551–552 (IEEE, 2015); https://doi.org/10.1109/IPCon.2015.7323726 Mashanovitch, M. et al. High-power, efficient DFB laser technology for RF photonics links. In 2018 IEEE Avionics and Vehicle Fiber-Optics and Photonics Conference (AVFOP) 1–2 (IEEE, 2018); https://doi.org/10.1109/AVFOP.2018.8550469 Jin, W. et al. Hertz-linewidth semiconductor lasers using CMOS-ready ultra-high-Q microresonators. Yu, M. et al. Integrated electro-optic isolator on thin-film lithium niobate. Xiang, C. et al. 3D integration enables ultralow-noise isolator-free lasers in silicon photonics. Guo, X. et al. High-performance modified uni-traveling carrier photodiode integrated on a thin-film lithium niobate platform. Shams-Ansari, A. et al. Electrically pumped laser transmitter integrated on thin-film lithium niobate. Lin, Z. et al. 120 GOPS Photonic tensor core in thin-film lithium niobate for inference and in situ training. Hu, Y. et al. Integrated lithium niobate photonic computing circuit based on efficient and high-speed electro-optic conversion. Preprint at https://arxiv.org/abs/2411.02734 (2024) Nahmias, M. A. et al. Photonic multiply-accumulate operations for neural networks. IEEE J. Sel. Shen, Y. et al. Deep learning with coherent nanophotonic circuits. Gupta, S., Agrawal, A., Gopalakrishnan, K. & Narayanan, P. Deep learning with limited numerical precision. 32nd International Conference on Machine Learning Vol. 37 (eds Bach, F. & Blei, D.) 1737–1746 (PMLR, 2015). DeepSeek-AI et al. DeepSeek-V3 technical report. Preprint at https://arxiv.org/abs/2412.19437 (2025). Dettmers, T. & Zettlemoyer, L. The case for 4-bit precision: k-bit inference scaling laws. 40th International Conference on Machine Learning Vol. 202 (eds Krause, A. et al.) 7750–7774 (PMLR, 2023). Hamerly, R., Bernstein, L., Sludds, A., Soljačić, M. & Englund, D. Large-scale optical neural networks based on photoelectric multiplication. Chen, Z. et al. Deep learning with coherent VCSEL neural networks. Ou, S. et al. Hypermultiplexed integrated photonics-based optical tensor processor. Hu, Y. et al. High-efficiency and broadband on-chip electro-optic frequency comb generators. Song, Y., Hu, Y., Zhu, X., Yang, K. & Lončar, M. Octave-spanning Kerr soliton frequency combs in dispersion- and dissipation-engineered lithium niobate microresonators. Song, Y., Zhu, X., Zuo, X., Huang, G. & Lončar, M. Stable gigahertz- and mmWave-repetition-rate soliton microcombs on X-cut lithium niobate. Xu, X. et al. 11 TOPS photonic convolutional accelerator for optical neural networks. Bai, B. et al. Microcomb-based integrated photonic processing unit. Chen, Z. et al. Efficient erbium-doped thin-film lithium niobate waveguide amplifiers. Zhou, J. et al. On-chip integrated waveguide amplifiers on erbium-doped thin-film lithium niobate on insulator. Jia, Y. et al. Integrated photonics based on rare-earth ion-doped thin-film lithium niobate. Cardenas, J. et al. Wide-bandwidth continuously tunable optical delay line using silicon microring resonators. Koenig, S. et al. Wireless sub-THz communication system with high data rate. Pfeifle, J. et al. Coherent terabit communications with microresonator Kerr frequency combs. Marin-Palomo, P. et al. Microresonator-based solitons for massively parallel coherent optical communications. Rizzo, A. et al. Massively scalable Kerr comb-driven silicon photonic link. Siddharth, A. et al. Ultrafast tunable photonic-integrated extended-DBR Pockels laser. Xue, S. et al. Pockels laser directly driving ultrafast optical metrology. Ghelfi, P. et al. A fully photonics-based coherent radar system. Zhu, S. et al. Integrated lithium niobate photonic millimetre-wave radar. Liu, W. et al. A fully reconfigurable photonic integrated signal processor. Feng, H. et al. Integrated lithium niobate microwave photonic processing engine. Maity, S. et al. Coherent acoustic control of a single silicon vacancy spin in diamond. Riedel, D. et al. Efficient photonic integration of diamond color centers and thin-film lithium niobate. Knaut, C. M. et al. Entanglement of nanophotonic quantum memory nodes in a telecom network. Menssen, A. J. et al. Scalable photonic integrated circuits for high-fidelity light control. Christen, I. et al. An integrated photonic engine for programmable atomic control. We thank K. Richard, N. Hoffman, M. Roberts and Keysight Technologies, Inc. for technological support, J. Jacobson for assistance and D. Plant and G. Hills for discussions. This work is supported by the Korea Advanced Institute of Science and Technology, grant number NRF-2022M3K4A1094782 (to Y.S., Y.H. ), the Defense Advanced Research Projects Agency, grant number HR001120C0137 (to Y.H. ), the National Science Foundation (NSF), grant numbers OMA-2137723 (to Y.H. and NSF EEC-1941583 (to K.P., H.K.W. ), the Department of the Navy, grant number N6833522C0413 (to X.Z. ), Amazon Web Services (AWS), grant number A50791 (to K.P., L.M. ), DRS Daylight Solutions, Inc., award A56097 (to L.M. ), NASA grant number 80NSSC22K0262 (to N.S. ), the National Institutes of Health, grant numbers NIH P41EB015903 (to N.L. and B.V.) and the Singapore National Research Foundation grant numbers NRF2022-QEP2-01-P07 and NRF-NRFF15-2023-0005 (to D.Z.). acknowledges support from the AWS Generation Q Fund at the Harvard Quantum Initiative. acknowledges support from the Capes-Fulbright and Behring Foundation fellowships. acknowledges support from the National Science Foundation Graduate Research Fellowship Program. acknowledges support from the A*STAR National Science Scholarship. The views, opinions and/or findings expressed are those of the authors and should not be interpreted as representing the official views or policies of the Department of Defense or the US Government. These authors contributed equally: Yunxiang Song, Yaowen Hu. John A. Paulson School of Engineering and Applied Sciences, Harvard University, Cambridge, MA, USA Yunxiang Song, Yaowen Hu, Xinrui Zhu, Keith Powell, Letícia Magalhães, Hana K. Warner, Shengyuan Lu, Xudong Li, Dylan Renaud, Norman Lippok, Neil Sinclair & Marko Lončar Quantum Science and Engineering, Harvard University, Cambridge, MA, USA State Key Laboratory for Mesoscopic Physics and Frontiers Science Center for Nano-optoelectronics, School of Physics, Peking University, Beijing, China HyperLight Corporation, Cambridge, MA, USA Fan Ye, Dylan Renaud & Mian Zhang Wellman Center of Photomedicine, Harvard Medical School and Massachusetts General Hospital, Boston, MA, USA Norman Lippok & Benjamin Vakoc Department of Materials Science and Engineering, National University of Singapore, Singapore, Singapore Di Zhu Centre for Quantum Technologies, National University of Singapore, Singapore, Singapore Di Zhu A*STAR Quantum Innovation Centre (Q.InC), Institute of Materials Research and Engineering (IMRE), Agency for Science, Technology and Research (A*STAR), Singapore, Singapore Di Zhu Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project. designed the experiments. performed the measurements and analysed the data with Y.H. led the foundry tapeout with F.Y. fabricated the early versions of the device. helped with the project. wrote the manuscript with contributions from all authors. supervised the project. Correspondence to Yunxiang Song, Yaowen Hu or Marko Lončar. are involved in developing lithium niobate technologies at HyperLight Corporation. The remaining authors declare no competing interests. Nature Photonics thanks Jose Capmany, Charles Roques-Carmes and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Discussion and Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Song, Y., Hu, Y., Zhu, X. et al. Integrated electro-optic digital-to-analogue link for efficient computing and arbitrary waveform generation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03861-x'>Pig-to-human lung xenotransplantation into a brain-dead recipient</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 15:01:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Genetically engineered pig lungs have not previously been transplanted into humans, leaving key questions unanswered regarding the human immune response in the context of a xenotransplanted lung and the possibility of hyperacute rejection. Severe edema resembling primary graft dysfunction was observed at 24 hours after transplantation, potentially due to ischemia–reperfusion injury. Antibody-mediated rejection appeared to contribute to xenograft damage on postoperative days 3 and 6, with partial recovery by day 9. Immunosuppression included rabbit anti-thymocyte globulin, basiliximab, rituximab, eculizumab, tofacitinib, tacrolimus, mycophenolate mofetil and tapering steroids, with adjustments made during the postoperative period based on assessments of immune status. Although this study demonstrates the feasibility of pig-to-human lung xenotransplantation, substantial challenges relating to organ rejection and infection remain, and further preclinical studies are necessary before clinical translation of this procedure. This is a preview of subscription content, access via your institution Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout These data are not publicly accessible due to ethical and institutional restrictions safeguarding research subject privacy. De-identified recipient data supporting the findings of this study are available upon request from the corresponding author, subject to ethics and institutional approval. Requests will be reviewed within 2 weeks. Chisholm-Burns, M., Kelly, B. S. & Spivey, C. A. Xenotransplantation could either be a friend or foe of healthcare equity. Mohiuddin, M. M. et al. Graft dysfunction in compassionate use of genetically engineered pig-to-human cardiac xenotransplantation: a case report. Mallapaty, S. & Kozlov, M. First pig kidney transplant in a person: what it means for the future. Wang, Y. et al. Pig-to-human kidney xenotransplants using genetically modified minipigs. Burdorf, L. et al. Pig-to-baboon lung xenotransplantation: extended survival with targeted genetic modifications and pharmacologic treatments. Li, Q. et al. Carbohydrate antigen expression and anti-pig antibodies in New World capuchin monkeys: relevance to studies of xenotransplantation. Bailey, J., Capaldo, T., Conlee, K., Thew, M. & Pippin, J. Experimental use of nonhuman primates is not a simple problem. & Kreisel, D. Immunopathogenesis of primary graft dysfunction after lung transplantation. Kubicki, N., Laird, C., Burdorf, L., Pierson, R. N. 3rd & Azimzadeh, A. M. Current status of pig lung xenotransplantation. Lung xenotransplantation: recent progress and current status. Pullen, L. C. Xenotransplantation moves toward clinical trials. & Alenghat, T. Respiratory epithelial cells orchestrate pulmonary innate immunity. Griffith, B. P. et al. Transplantation of a genetically modified porcine heart into a live human. Snell, G. I. et al. Report of the ISHLT Working Group on Primary Lung Graft Dysfunction, part I: definition and grading – a 2016 Consensus Group statement of the International Society for Heart and Lung Transplantation. Burdorf, L. et al. Thromboxane and histamine mediate PVR elevation during xenogeneic pig lung perfusion with human blood. Immune response after pig-to-human kidney xenotransplantation: a multimodal phenotyping study. Evaluation of human and non-human primate antibody binding to pig cells lacking GGTA1/CMAH/beta4GalNT2 genes. Anand, R. P. et al. Design and testing of a humanized porcine donor for xenotransplantation. Xu, H., Perez, S. D., Cheeseman, J., Mehta, A. K. & Kirk, A. D. The allo- and viral-specific immunosuppressive effect of belatacept, but not tacrolimus, attenuates with progressive T cell maturation. Mavragani, C. P. Mechanisms and new strategies for primary Sjogren's syndrome. Moazami, N. et al. Pig-to-human heart xenotransplantation in two recently deceased human recipients. Chen, J. Q. et al. Scan of the endogenous retrovirus sequences across the swine genome and survey of their copy number variation and sequence diversity among various Chinese and Western pig breeds. Chen, S. Ultrafast one-pass FASTQ data preprocessing, quality control, and deduplication using fastp. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. A., Bamford, K. B., Bell, P. M. & Callender, M. E. Community-acquired lower respiratory tract infection: implementation of an antibiotic protocol. Goodacre, N., Aljanahi, A., Nandakumar, S., Mikailov, M. & Khan, A. S. A reference viral database (RVDB) to enhance bioinformatics analysis of high-throughput sequencing for novel virus detection. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. We also thank the clinical and laboratory teams at the First Affiliated Hospital of Guangzhou Medical University and the National Clinical Centre for Respiratory Disease, including the lung xenotransplantation clinical team (Y. Zeng, J. Mao, L. Wang, W. Shen, J. Zhang, T. Xing, Y. Xue, J. Fan, Y. Ouyang and X. Zhang), the nursing team (Y. Chen, D. Huang, R. Li, X. Li, M. Kuang and X. Li), anesthesiologist Y. Zhou, laboratory staff (W. Wang, L. Liu, Z. Huang and M. Yu) and the organ procurement organization personnel (L. Wang, L. Wang, Y. Wang and K. Li) for their unwavering support. We acknowledge L. Burdorf and C. Laird's team for their pioneering work on NHP lung xenotransplantation, which inspired this research, particularly in the experimental design. These authors contributed equally: Jianxing He, Jiang Shi, Chao Yang, Guilin Peng, Chunrong Ju. Department of Thoracic Surgery and Oncology, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Research Centre for Respiratory Disease, Guangzhou, China Jianxing He, Jiang Shi, Chao Yang, Guilin Peng, Yi Zhao, Hengrui Liang & Xin Xu Department of Organ Transplantation, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China Jianxing He, Jiang Shi, Chao Yang, Guilin Peng, Chunrong Ju, Chuanbao Chen, Menyang Liu, Xiaoyou Liu & Xin Xu Department of Anesthesiology, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China Department of Pathology, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China Department of Critical Care Medicine, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China Department of Emergency Medicine, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China State Key Laboratory of Respiratory Disease & National Clinical Research Center for Respiratory Disease, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China Animal Experiment Center, State Key Laboratory of Respiratory Disease & National Clinical Research Center for Respiratory Disease, The First Affiliated Hospital of Guangzhou Medical University, National Clinical Centre for Respiratory Disease, Guangzhou, China Institute of Organ Transplantation, Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China Division of Cardiovascular and Thoracic Surgery, Department of Surgery, Duke University Medical Center, Durham, NC, USA Division of Pulmonary and Critical Care Medicine, Department of Medicine, Mayo Clinic, Rochester, MN, USA Department of Allergy and Clinical Immunology, The First Affiliated Hospital of Guangzhou Medical University, Guangzhou Institute of Respiratory Disease, State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, National Center for Respiratory Medicine, Guangzhou, China Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Guangzhou Medical University, Guangzhou Institute of Respiratory Disease, State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, National Center for Respiratory Medicine, Guangzhou, China National Clinical Eye Research Center of Eye Hospital and Institute for Clinical Data Science, Wenzhou Medical University, Wenzhou, China Institute for AI in Medicine and Faculty of Medicine, Macau University of Science and Technology, Macau, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the data collection and analysis. drafted the manuscript, with critical revisions from J.H., K.Z., J.S., X.L., Y.Z., N.Z. All authors reviewed and approved the final manuscript. Correspondence to Jianxing He, Xiaoyou Liu or Xin Xu. The other authors declare competing interests. Nature Medicine thanks Hidetaka Hara and Eckhard Wolf for their contribution to the peer review of this work. Primary Handling Editor: Michael Basson, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. He, J., Shi, J., Yang, C. et al. Pig-to-human lung xenotransplantation into a brain-dead recipient. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250825015657.htm'>The Higgs boson just revealed a new secret at the Large Hadron Collider</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 14:54:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Since the discovery of the Higgs boson in 2012, physicists have made major strides in exploring its properties. Does that mean the subject is done and dusted? In new results presented at the 2025 European Physical Society Conference on High Energy Physics (EPS-HEP), the ATLAS Collaboration narrowed in on two exceptionally rare Higgs-boson decays using data collected in Run 3 of the Large Hadron Collider (LHC). Despite its scarceness - occurring in just 1 out of every 5000 Higgs decays - this process provides the best opportunity to study the Higgs interaction with second-generation fermions and shed light on the origin of mass across different generations. This rare decay is especially intriguing, as it proceeds via an intermediate "loop" of virtual particles. If new particles contribute to this loop, the process could offer hints of physics beyond the Standard Model. Identifying these rare decays is quite the challenge. This signal can be easily hidden behind the thousands of muon pairs produced through other processes ("background"). Compounding the challenge are the operation conditions of LHC Run 3, which features more overlapping collisions, making it easier for particle jets to mimic real photons. They also developed a sophisticated method to better model background processes, categorised recorded events by the specific Higgs-production modes, and made further improvements to their event-selection techniques in order to maximize the likelihood of spotting genuine signals. The comparable CMS result reached an observed (expected) significance of 3 (2.5) standard deviations. This means that the chance that the result is a statistical fluctuation is less than one in 3000! As for the H→Zγ process, a previous ATLAS and CMS combined analysis used Run-2 data to find evidence of this decay mode. It reported an observed (expected) excess over the background-only hypothesis of 3.4 (1.6) standard deviations. The latest ATLAS result, combining Run-2 and Run-3 data, reported an observed (expected) excess over the background-only hypothesis of 2.5 (1.9) standard deviations. This outcome provides the most stringent expected sensitivity to date for measuring the decay probability ("branching fraction") of H→Zγ. Note: Content may be edited for style and length. This Diet Could Help Protect Your Brain From Alzheimer's Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250825015645.htm'>Google's quantum computer just simulated the hidden strings of the Universe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 14:30:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The research, published in the academic journal Nature, represents an essential step in quantum computing and demonstrates its potential by directly simulating fundamental interactions with Google's quantum processor. In the future, researchers could use this approach to gain deeper insights into particle physics, quantum materials, and even the nature of space and time itself. The aim is to understand how nature works at its most fundamental level, described by so-called gauge theories. "Our work shows how quantum computers can help us explore the fundamental rules that govern our universe," says co-author Michael Knap, Professor of Collective Quantum Dynamics at the TUM School of Natural Sciences. Tyler Cochran, first author and graduate student at Princeton, says: "By adjusting effective parameters in the model, we could tune properties of the strings. They can fluctuate strongly, become tightly confined, or even break." Materials provided by Technical University of Munich (TUM). Note: Content may be edited for style and length. This Diet Could Help Protect Your Brain From Alzheimer's Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/military/a65709166/navy-officer-says-underwater-ufos-are-legitimate-threats-the-evidence-is-hard-to-ignore/'>Navy Officer Says Underwater UFOs Are Legitimate Threats. The Evidence Is Hard to Ignore.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In mid-2014, during training flights off the coast of Virginia Beach, Virginia, F/A-18 pilot and U.S. Navy Lieutenant Ryan Graves began to notice something strange. The radar returns looked off—phantom blips moving with unfathomable speed and precision. At first he dismissed it as a glitch. They would hover in place—and then dart away at supersonic speeds. They were recorded from the ocean's surface to 40,000 feet. Graves reported seeing a dark gray or black cube inside a clear sphere, between five and 15 feet in diameter, coming within 50 feet of their jets. “We started treating it as a safety issue.” Over the next year, Graves's squadron recorded sightings of unidentified objects almost daily. Sometimes the object would rotate in place; others vanished when approached. They challenged every assumption held by aerospace engineers and radar operators. Graves doesn't claim to know what the objects were. “This wasn't business as usual,” he says. It wasn't just us on the East Coast. Since at least the 1950s, military sources have reported strange objects plunging into the ocean—what they call USOs (unidentified submerged objects). These phenomena exhibit characteristics that conflict with our understanding of physics and maritime navigation. Today Graves is one of the most vocal advocates for UAP transparency. He doesn't claim that these craft are alien, but he's certain that they aren't using known human tech. “Where that leaves us opens up options—extraterrestrial visitors, time travelers, breakaway civilizations . . . things that challenge the status quo and aren't easily accepted at face value,” he says. Two videos captured by Navy fighter jets that were made public in 2020 show strange craft flying at incredible speeds with no visible means of propulsion, sometimes rotating in midair. We don't test experimental aircraft in training ranges—it's too dangerous—and I had access to everything classified. Gallaudet, now retired from the Navy and currently the CEO of Ocean STL Consulting, is pushing for the U.S. government to treat these phenomena as a “national research priority.” Despite some public disclosures, many records remain classified, buried within defense contractor vaults or shielded by national security exemptions. If even part of what's been reported is true, then the encounters off Virginia, California, and elsewhere could be the opening chapter in a much deeper mystery—one that spans oceans, navies, and continents. This spring, Graves and Gallaudet briefed officials in Washington, D.C., on unidentified submerged objects. “We're at a unique moment in history,” Graves says. “People have access to tools that can reveal things. In 2023, Congress passed the UAP Disclosure Act, legislation requiring federal agencies to collect, catalog, and disclose records related to recovered nonhuman craft and biologics. For the first time, U.S. law openly acknowledged the potential existence of off-world or nonhuman intelligence, and even hinted at craft retrieval and reverse engineering programs. Those newly released records contain a trove of details on previously classified encounters with multiple transmedium UAPs. None definitively prove that otherworldly beings are hiding in or above our oceans, but they do raise questions that neither our military nor scientific experts can explain. “The possibility that they exist underwater is very real,” Gallaudet says. “Or why not—maybe they lived here for a long time, before we even evolved, and sought safety from the Earth's atmospheric and geologic cataclysms by creating a habitat or place to live beneath the seafloor. . .. The following four incidents have provided investigators with the most compelling—and perplexing—evidence so far of transmedium encounters with what experts like Gallaudet believe could be nonhuman in origin. In November 2004, several U.S. Navy pilots assigned to the USS Nimitz encountered a Tic Tac–shaped UAP darting and dashing over the Pacific. According to reports that followed, the object lacked visible control surfaces and heat signatures that would be typical of jet aircraft or rockets. The objects appeared at 80,000 feet, and then dropped to sea level in less than a second. As Fravor circled downward, the object suddenly shot away, vanishing in seconds. This time the object was captured on an infrared camera showing a white speck flying against the wind, rotating and maneuvering in ways no known aircraft can. One of the most hotly debated encounters of an unidentified submerged object came in April 2013, when an infrared surveillance camera operated by U.S. Customs and Border Protection captured something extraordinary over Aguadilla, Puerto Rico. A small spherical object flew inland from the ocean, crossed the airport, and then returned to sea—where it maneuvered in ways that appeared to defy physics. Seconds later it resurfaces, splits in two, and then submerges again. Any object passing between two mediums should generate drag or turbulence or, at the very least, a splash, according to the laws of hydrodynamics. But this one behaved as if those laws didn't apply. “It's not that they break the laws,” Gallaudet says. “It's that they demonstrate engineering capabilities we haven't yet developed. “I've spoken with the pilots who saw it,” he adds. “The near-instantaneous acceleration of that craft is something we just can't engineer.” In the 1990s, a Navy CH-53 Sea Stallion pilot was flying off the coast of Puerto Rico when he saw something he couldn't explain. The pilot, whose name has not been disclosed, would later report that the object was a large, dark mass unlike any submarine or marine animal he'd ever seen. Fravor then shared details on Joe Rogan's podcast in 2019. “He's looking at this thing going, ‘What the hell is that?' And then it just goes back down underwater. In July 2019, an infrared camera aboard the USS Omaha captured something that defied explanation: a spherical object moving swiftly over the Pacific before dropping into the ocean—again, no splash, no wake, no debris. The footage, later verified by Pentagon officials, stunned observers. The object didn't resemble any known drone, missile, or aircraft. And its disappearance beneath the waves raised new questions about transmedium tech. The video shows a recording of the AN/KAX-2 screen and appears to have been taken with a night vision camera. “It looked nothing like any known aircraft,” Gallaudet says. “After I testified before Congress last year, a sailor who had been on the bridge of the Omaha contacted me. Stav Dimitropoulos's science writing has appeared online or in print for the BBC, Discover, Scientific American, Nature, Science, Runner's World, The Daily Beast and others. Underwater UFO is a Threat, Says Ex-Navy Officer A Chinese Warship Injured Divers With Its Sonar Ukraine's Snipex Alligator Rifle Is a Real Beast Ex-Navy Official: UFOs Are Hiding in Our Oceans What We Know About the Air Force's X-37B Shuttle</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65864098/cow-tooth-stonehenge/'>A Single Cow Tooth Just Revealed a Major Clue About Stonehenge's Origins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists sliced open a Neolithic cow's molar, and now they believe they know quite a bit more about the history of Stonehenge. The fact that it was found alongside Stonehenge' south entrance (in a prominent ritual location) suggested the animal's importance, and its remains were eventually dated to between 2995 and 2990 B.C. But while that provided the “when,” there were still questions unanswered about why the fossil would have been placed in such an honored position. Recently, a team from the British Geological Survey, Cardiff University, and University College London used isotope analysis to track this history, chronicling the findings in a new study published in the Journal of Archaeological Science. To begin their inquiry, they sliced into the cow's third molar, since that tooth records chemical signals from the animal's second year of life. Through this method, they were able to determine where this cow spent the early years of its life grazing—and it wasn't always in the fields near Stonehenge. “This is yet more fascinating evidence for Stonehenge's link with southwest Wales,” Michael Parker Pearson, professor of British later prehistory at the University College London, said in a statement, “where its bluestones come from. “It tells us that very early in its life it incorporated lead into its skeleton and that lead was from old Paleozoic rocks, older than about 400 million years old,” Jane Evans, professor and British Geological Survey honorary research associate, told The Guardian. “This study has revealed unprecedented details of six months in a cow's life, providing the first evidence of cattle movement from Wales, as well as documenting dietary changes and life events that happened around 5,000 years ago,” Evans said in a statement. “A slice of one cow tooth has told us an extraordinary tale and, as new scientific tools emerge, we hope there is still more to learn from her long journey.” Richard Madgwick, professor of archaeological science at Cardiff University, highlighted what makes this bovine approach important in a statement: “So often grand narratives dominate research on major archaeological sites, but this detailed biographical approach on a single animal provides a brand-new facet of the story of Stonehenge.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. An Archaeology Student Hit Gold on Her Debut Dig Gut Feelings Are Memories From the Future, Scientists Say We May Have Just Found a Hidden Shipwreck</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65833714/singularity-moores-law/'>For the Singularity to Truly Arrive, We'd Need a Machine That Eats the Sun</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>AI optimists envision a future where artificial general intelligence (AGI) surpasses human intelligence, but the path remains riddled with scientific and logistical hurdles. Once upon a time, the world seemed like it was made of larger building blocks. Rain fell from what looked like opaque, puffy masses that also blocked the Sun. Even when alchemists were melting pieces of ore, they thought mercury must be related to silver. Higher resolutions, more powerful zoom, electron microscopes, particle accelerators, nuclear energy. For technology, powering these levels of detail has fallen into the broad purview of computer chips—and computer chips have also gotten higher-resolution, in a sense. Early punchcards (predecessors to the chips we have now) had openings so that portions of circuitry could literally form a connection, like playing certain notes on the piano using certain fingers, or connecting phone lines on an old switchboard. But our knowledge of electronics has grown so minuscule that it's almost hard to fathom. Moore was a cofounder of computer chip pioneer Intel, and it was there that he noticed that transistors—switches used to direct the current within electrical devices—were shrinking at a pretty consistent rate. This led to the 1958 invention of the integrated circuit, which could be installed in devices that were previously built one transistor at a time. But Moore's Law, as it was later called, has held for decades past this initial forecast. 1But for the last several years, those in the computing industry (and those who study it) have started to discuss the “end of Moore's Law.” There's a point at which transistors simply can't get any smaller because of the basics of physics itself—these tiny transistors must still be able to communicate with the rest of what's required to build an integrated circuit, be widely manufacturable, and stay cost effective.2 The slowdown of Moore's Law has been notable for a while. From MIT's Computer Science and Artificial Intelligence Lab (CSAIL): Companies like OpenAI make opaque promises about how generative AI will change all of our lives, save us hours a week, and make many sectors of human labor obsolete. Manufacturers and generative AI companies are already doing this. However, that's not a long-term solution to the growing demand for this amount of computing. Like leadership of the late Roman Empire or the icing on a dry cake, our computing components can't be spread too thin. (Today, people can't seem to—or don't want to—control whether or not their chatbots become racist, are “sensual” with children, or induce psychosis in the general population, but sure.) The goal of these AI boosters is known as artificial general intelligence, or AGI. They theorize, or even hope for, an AI so powerful that it thinks like... well... a human mind whose ability is enhanced by a billion computers. AI optimists want to accelerate the singularity and usher in this “godlike” AGI. One of the key facts of computer logic is that, if you can slow the processes down enough and look at it in enough detail, you can track and predict every single thing that a program will do. Algorithms (and not the opaque AI kind) guide everything within a computer. Over the decades, experts have written the exact ways information can be sent, one bit—one minuscule electrical zap—at a time through a central processing unit (CPU). From there, those bits are assembled into a slightly more concrete format as another type of code. Networks work the same way, with your video or document broken into pieces, then broken down further and further until tiny packets of data can be carted back and forth as electrical zaps over lengths of wire. The human brain is, in some ways, another piece of electrical machinery. The National Institute of Standards and Technology (NIST) quantifies it as an exaflop caliber computer: “a billion-billion (1 followed by 18 zeros) mathematical operations per second—with just 20 watts of power.” By this standard, you power dozens of human brains by plugging them into a single U.S. household outlet. It's possible that the human brain is also predictable when you understand all of its parts and influences enough. And despite an iconic, massively influential paper stating otherwise, the cell is not like a machine. There are countless ways the human brain could be boosted or hindered by factors we can't even measure yet. That seems especially true when aspiring “AI caretaker” engineers want their AIs to know everything from all of human history. But let's say that efficiency or quantity of information isn't an issue. Let's say we can build one-million-exaflop computers to run advanced AIs that will mimic human think tanks. How does the end of Moore's Law affect scientists who work toward that technological singularity? That's both the size of electrical energy required and the physical size associated with storage, processing, cooling, and everything else required to keep a computer running. There are a few directions we could go to solve the size problem, but none of them are easy to achieve.AI boosters push nuclear fusion (another technology that is still far away) as a cure-all for the energy problems associated with large AI computing. But no one knows for sure when (or if) nuclear fusion will produce more energy than what is required to run nuclear fusion facilities. The Kardashev scale is a thought exercise about Solar System- or galaxy-scale civilizations. But while Moore's Law was a forecast based on expertise in both technology and global supply chains, the Kardashev scale and Dyson spheres are thought exercises with no real-life analog at all. On a more grounded level, quantum computing has been touted as an advance toward the realm of AI, ultimately leading into the singularity. But quantum computing is in its infancy, to say the least. It currently requires extreme cooling unlike anything in today's traditional computer realm. There is no usable consumer version of a quantum computer, and we're not even close to one. All of that means we have a lot of options that are at least 10 years away—or even as much as 100 or 1,000 years away. Venture capitalists today are selling a vision of the future. In the huge field of artificial intelligence, there are countless ways to define and work toward goals like finding new prescription drugs or faraway galaxies. AGI is a separate, specific idea, but even within that there are variations. I personally believe that AGI is very far away—though some very smart people, like Google DeepMind and Imperial College London computer scientist Murray Shanahan, believe it's closer than I think. (Shanahan's book for MIT Press about the technological singularity is a great introduction.) But others, like OpenAI's Sam Altman, don't seem to know what they're talking about in any detail. Altman waves away questions about specifics of technologies he does not understand, while Shanahan writes detailed papers about the Wittgensteinian philosophical tests that AI models are growing ever more able to pass. Altman has suggested a Dyson sphere that encloses our Solar System, for example, as a back-of-the-napkin solution to the rising energy costs of AI. In 2019, over 750 million people on Earth still didn't have access to electricity, an additional over 400 million aren't able to use local available electricity, and both numbers are subject to stagnation or even worsening in the wake of the global COVID-19 pandemic. We would need to drain the entire Solar System (and more!) of certain elements to even build what Altman suggests. While Moore's Law is real, many factors of the singularity are not—at least, not this decade. The same way we can look around a room and categorize and remember many details at a glance, computers can plug away at enormous lists of ingredients without missing a beat or losing their place. In 2024, chemist Mas Subramanian (the creator of the novel pigment YInMn Blue) told Popular Mechanics that algorithms to discover new molecules are difficult to work with because of factors that the public doesn't really understand. It's just not that easy to find a new pigment, for example—YInMn blue has an unusual crystal structure. The chemical reaction that makes the color is found in a bipyramidal shape, Subramanian explains, rather than a tetrahedral or octohedral network. (Bipyramidal is like two tetrahedrons, or “D4” shapes, glued together. The octohedron has eight faces in a different form.) Graphite and diamond are different crystalline forms of the same element. That need for context is a major limitation of algorithms as we know them. So, Subramanian explains, the machine learning algorithm suggests a long list that must be vetted by a human, and many suggestions don't work in real life right off the bat. And because these models are trained on what already exists, they can't innovate, in the most literal sense. The landmarks of expert-level artificial intelligence studies don't sound like sales pitches or soundbytes—they sound more like Shanahan's clarifying note, written after he used some imprecise language in a paper that escaped containment and entered the mainstream press: He concludes: “The aim, rather, was to remind readers of how unlike humans LLM-based systems are, how very differently they operate at a fundamental, mechanistic level, and to urge caution when using anthropomorphic language to talk about them.” It's very different than Altman's public comment that he might need to Dyson-sphere the entire Solar System. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. An Archaeology Student Hit Gold on Her Debut Dig Gut Feelings Are Memories From the Future, Scientists Say</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a65836714/inner-thoughts-chip/'>Scientists Have Translated the Inner Monologue. The Implications Are Incredible.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This could be a game-changer for patients suffering from mobility issues, neurodegenerative diseases, or other speech related disabilities. We may earn commission if you buy from a link. The results of this final installment of this (unofficial) BCI trilogy was published last week in the journal Cell. “For people with paralysis, attempting to speak can be slow and fatiguing, and if the paralysis is partial, it can produce distracting sounds and breath control difficulties,” Willett, a co-author on all three studies (including the latest one), said in a press statement. To translate this world of imagined speech, Willett and his team relied on four participants in the Brain2Gate trial—a study initially designed for developing proof-of-principle for people with tetraplegia to control computer cursors and other assistive devices with their thoughts alone. At least two of the volunteers (a 68-year-old woman and a 33-year-old man) were diagnosed with ALS, and either couldn't intelligibly communicate or couldn't move their muscles at all. Participants were asked to perform speech attempts  or think about simple words. Willett and his team found that the resulting patterns were similar to (though less robust than) those of attempted speech signals from the 2023 study. They also developed unique in-system code phrases—phrases not common in usual speech (i.e. “Orange you glad I didn't say banana”) that could switch the BCI on and off so that participants' inner monologues remained private at their own discretion. “We found that we could decode these signals well enough to demonstrate a proof of principle, although still not as well as we could with attempted speech,” Willett said in a press statement. “This gives us hope that future systems could restore fluent, rapid and comfortable speech to people with paralysis via inner speech alone.” With just a 50-word vocabulary, the BCI produced an error rate up to 33 percent. “It's worth pointing out that implanted BCIs are not yet a widely available technology and are still in the earliest phases of research and testing,” Willett said in a press statement. Willett also said that future studies will explore how to make the decoding process even more accurate, specifically by exploring regions of the brain beyond just the motor cortex, including those associated with language and hearing. With better science, engineering, and a little luck, science may one day be able to return voices to those who—through accidents or neurodegenerative diseases—have forever lost their own. To Fall Asleep Faster, Do This Before Bed Did Scientists Just Find a Way to Reverse Aging? New Injectable Skin Could Help Heal Severe Wounds Some Brains Are Defying Time by Refusing to Age</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/the-way-people-search-the-internet-can-fuel-echo-chambers/'>The Way You Search the Internet Can Fuel Echo Chambers—Without You Realizing It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Users' Internet search questions can strengthen echo chambers, even on factual topics, but there are simple ways to lessen the effect This divergence doesn't just apply to political opinions; it also touches on factual topics, from climate change to vaccination. Online participants were asked to rate their beliefs on six topics, including nuclear energy and caffeine's health effects. They then chose search terms to learn more about each topic. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. These narrow terms tended to align with participants' existing beliefs, and generally less than 10 percent did this knowingly. “People often pick search terms that reflect what they believe, without realizing it,” says Eugina Leung of Tulane University's business school, who led the study. “Search algorithms are designed to give the most relevant answers for whatever we type, which ends up reinforcing what we already thought.” The same was true when participants used ChatGPT and Bing for searches aided by artificial intelligence. When the researchers randomly assigned participants to view different results, they saw those results affect people's opinions and even behavior. For instance, participants who saw search results using “nuclear energy is good” felt better about nuclear energy afterward than those using “nuclear energy is bad.” People who saw results using “caffeine health benefits,” rather than “risks,” were more likely to choose a caffeinated drink afterward. Pointing out biases in the search terms had only a small effect on people's final opinions. But changing the search algorithm either to always provide broad results or to alternate between results obtained with broad and user-provided terms mitigated the effects of narrow searches. “This would be really helpful,” Leung says, but whether it will ever happen “is hard to predict.” Simon Makin is a freelance science journalist based in the U.K. His work has appeared in New Scientist, the Economist, Scientific American and Nature, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/180-years-of-scientific-american-means-180-degree-turns-in-science-here-are/'>Science Thought Nerves Couldn't Heal. It Thought Mars Had Canals. It Thought Plastic Was Eco-Friendly. Then It Thought Again</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Drawings of Mars showing its 'canals' and polar ice caps in drawings created from observations made at the Lowell Observatory in 1907. Explore our legacy of discovery and look ahead to the future. For Scientific American's Science Quickly, I'm Rachel Feltman. Today we're doing something a little different from our usual weekly news roundup. Scientific American turns 180 this year, and we recently celebrated with a collection of print features about times in history when science has seemingly done a complete pivot—a 180-degree turn, if you will. First up, we have a story from freelance health and life sciences journalist Diana Kwon about nerve regeneration. For millennia doctors and scientists believed that any damage to the nerve cells that carry signals throughout the body must be irreversible. While many instances of nerve damage are, indeed, difficult to treat, scientists have realized over the past couple of centuries that nerves can and do regenerate. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Now we know that even these most precious neurons can regenerate under the right conditions. As research continues into exactly which mechanisms encourage (or block) neural regeneration throughout the body, scientists are also engaged in another debate: whether human brains are capable of producing new neurons throughout adulthood. The phenomenon of adult neurogenesis would have been unfathomable mere decades ago, but a growing body of evidence now supports it. Just imagine what secrets we'll have uncovered about the nervous system 180 years from now. In another example of a scientific turnabout, Scientific American senior features editor Jen Schwartz reminds readers that plastic was, ironically, invented as a sustainable alternative to another material: ivory. In fact, back in 1864 Scientific American published news of a contest from billiard-table manufacturing company Phelan & Collender seeking an alternative for vanishing elephant tusks, which, at the time, were used to make pool balls. The company offered $10,000 as a reward for this feat of materials science. A printer from Albany, New York, named John Wesley Hyatt came up with celluloid in response, though he chose to patent the invention for himself rather than accepting the prize money. In another scientific 180, detailed by Scientific American contributing editor Sarah Scoles, we learn how the search for alien life has periodically been turned on its head. In the late 19th century an Italian astronomer observed groovelike markings on Mars, which convinced an American astronomer that the Red Planet hosted a whole civilization. In 1906 that second astronomer, Percival Lowell, wrote a book positing that Martians had carved out a sophisticated network of watery canals. Even when a closer look at Mars in 1909 revealed that those canal-like markings had actually been an optical illusion, Lowell's theories persisted. In 1916 a Scientific American managing editor wrote in New York Times letters to the editor that he still believed Mars held sophisticated life and an irrigation system to prove it. Of course, when the Mariner 4 spacecraft gave us our first flyby view of Mars in 1965, we saw our planetary neighbor for the desolate world it is. While we're unlikely to do another 180 on the existence of intelligent life in our solar system anytime soon, scientists have recently become aware of the abundant potential opportunities for microbial life in our cosmic backyard and beyond. We may not be peering up at Mars hoping to see aliens traveling around by gondola anymore, but in some ways the hunt for alien life is more optimistic than ever. Check out our 180th-anniversary issue on newsstands, or go to ScientificAmerican.com for more fascinating stories of scientific swerves. We'll have additional anniversary treats popping up online in the next few weeks, too, so stay tuned. Just snap a pic of any Scientific American print issue in a setting that reflects or complements the cover theme. Then share it on social media using #SciAmInTheWild. Our usual science news roundup will be back next week. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Jeffery DelViscio is currently chief multimedia editor/executive producer at Scientific American. Before that he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02777-1'>inTRACKtive: a web-based tool for interactive cell tracking visualization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 09:45:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The field of developmental biology is undergoing a profound transformation, driven by groundbreaking advances across multiple disciplines. Light-sheet microscopy enables rapid, large-scale, multicolor imaging of developing embryos in unprecedented detail, allowing scientists to capture dynamic processes as they unfold in real time1,2,3. Simultaneously, the advent of advanced cell tracking technologies, such as Ultrack4, Linajea5, Elephant, TGMM6, Trackmate, Mastodon and TrackAstra, has made it possible to follow individual cells throughout embryonic development with remarkable precision and speed. These innovations, which have emerged in the past two decades, are only now realizing their full potential for embryology. While these imaging datasets hold the potential for biological insights, accessing and interacting with such complex data typically demands highly specialized technical expertise and substantial computational resources. Existing software tools allow visualization and analysis of cell tracking data (for example, napari8, TrackMate's TrackScheme, Mastodon and CellTracksColab), However, while powerful in their respective application domains, most of these approaches require native software, have limited 3D support, or are intended for the curation of tracks, not the interactive exploration of large cell tracking datasets. Consequently, only a limited subset of researchers can fully leverage these data, creating a barrier to broader adoption. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout The tracking data presented in this study are available for download from the Virtual Embryo Zoo website (https://embryozoo.org). All data have been converted and normalized to a common format (Supplementary Fig. The repository for inTRACKtive, including documentation, example code and videos, can be found at https://github.com/royerlab/inTRACKtive. Sofroniew, N. et al. napari: a multi-dimensional image viewer for Python. We extend our gratitude to all the original authors who kindly provided data and guidance: P. Keller, H. Shroff, K. McDole, A. Jain, R. Haase and G. Malandain. We thank the Royer group at CZ Biohub SF and E. Gomez-de-Mariscal for feedback on inTRACKtive and the Virtual Embryo Zoo website. We appreciate S. Schmid's constructive feedback on the manuscript. We thank CZI for the Science Design System (SDS) UI component library. Chan Zuckerberg Biohub San Francisco (CZB SF) funded this work. We thank CZB SF donors P. Chan and M. Zuckerberg for their generous support. Chan Zuckerberg Biohub, San Francisco, CA, USA Teun A. P. M. Huijben, Kyle Awayan, Jordão Bragantini, Merlin Lange & Loïc A. Royer Chan Zuckerberg Initiative, Redwood City, CA, USA Ashley G. Anderson III, Andrew Sweet, Erin Hoops, Connor Larsen & Chi-Li Chiu Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar developed the initial version of inTRACKtive. built the Virtual Embryo Zoo website. J.B. performed tracking of the Tribolium and zebrafish data using Ultrack. provided insight into the zebrafish tailbud migration. All authors contributed to editing the manuscript. Correspondence to Teun A. P. M. Huijben or Loïc A. Royer. The authors declare no competing interests. Nature Methods thanks Juhyun Lee, John Murray, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Overview of the Virtual Embryo Zoo Visualizing velocity, density and structure tensor How to visualize your own data: using inTRACKtive from the command line, from napari, from a Jupyter notebook, and within a Jupyter notebook Huijben, T.A.P.M., Anderson, A.G., Sweet, A. et al. inTRACKtive: a web-based tool for interactive cell tracking visualization. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02778-0'>Ultrack: pushing the limits of cell tracking across biological scales</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 09:05:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Tracking live cells across two-dimensional, three-dimensional (3D) and multichannel time-lapse recordings is crucial for understanding tissue-scale biological processes. Despite advancements in imaging technology, accurately tracking cells remains challenging, particularly in complex and crowded tissues where cell segmentation is often ambiguous. We present Ultrack, a versatile and scalable cell tracking method that tackles this challenge by considering candidate segmentations derived from multiple algorithms and parameter sets. Ultrack leverages temporal consistency to select optimal segments, ensuring robust performance even under segmentation uncertainty. We validate our method on diverse datasets, including terabyte-scale developmental time-lapse recordings of zebrafish, fruit fly and nematode embryos, as well as multicolor and label-free cellular imaging. We demonstrate that Ultrack achieves superior or comparable performance in the cell tracking challenge, particularly when tracking densely packed 3D embryonic cells over extended periods. Moreover, we propose an approach to tracking validation via dual-channel sparse labeling that enables high-fidelity ground-truth generation, pushing the boundaries of long-term cell tracking assessment. Our method is freely available as a Python package with Fiji and Napari plugins and can be deployed in a high-performance computing environment, facilitating widespread adoption by the research community. However, one key challenge remains: the accurate reconstruction of cell trajectories and lineages in complex biological systems11. Cell segmentation and tracking have been persistent challenges in bioimage analysis11. While cell segmentation has advanced rapidly with the advent of deep-learning-based methods18,19,20, cell tracking continues to be an open problem11. The primary challenge stems from the association of spurious segmentations over time, hindering long-term tracking, especially in dense and dynamic cellular environments. Most automatic tracking methods use a two-step approach: first segmenting cells, then linking them across time frames21,22,23,24. While computationally efficient, this approach struggles with the compounding of errors over time, particularly in dense tissues25 or when cells divide rapidly. Simultaneous segmentation and tracking offers a promising alternative26,27,28,29,30, wherein multiple candidate segments and/or inter-segment links are generated, and then optimal candidates are selected by satisfying biological constraints (that is, cells divide but do not merge) and optimizing specific criteria (that is, only keep links best supported by the data). However, existing methods in this category are constrained by their inability to utilize arbitrary segmentation inputs28,31, as well as limitations in data scale or dimensionality due to high computational costs29,30. These factors have limited their applicability to tissue-scale problems. To address these challenges, we developed Ultrack, a robust and scalable method for large-scale cell tracking that excels even under segmentation uncertainty. Ultrack can track cells (or nuclei) in two-dimensional (2D), three-dimensional (3D) and multichannel datasets, accommodating a wide range of biological contexts. By jointly evaluating candidate segmentations and tracks, Ultrack uses temporal consistency to select the most accurate segments (Fig. This approach leverages information from adjacent time points to resolve cell segmentation and tracking ambiguities, enhancing performance, particularly in complex, densely packed tissues where cell boundaries are often uncertain (Supplementary Video 1). a, The Ultrack pipeline can process a variety of input images, including 2D, 3D and multicolor datasets. These images are then processed by any segmentation algorithm or a combination of them. b, Ultrack builds segmentation hypotheses between frames for tracking and solves an ILP problem to identify cell segments and their trajectories. The resulting tracks can be exported in various formats compatible with multiple platforms. c, Illustration of how segmentation hypotheses are built using ultrametric contours: multiple segmentations are provided (first row), their binary contours are extracted (second row), and these are combined into a multilevel contour, equivalent to a hierarchy (third row). d, Joint segmentation and tracking is performed using an ILP formulation, modeling cell behaviors (for example, cell appearance xα., division xδ., death or exit x.β) while finding nonoverlapping cells with maximal association over time. e, Ultrack can process arbitrarily sized datasets and scales from a laptop to a HPC cluster. f, Projection of a 3D image of a developing zebrafish embryo, with cell tracks overlaid and colored to indicate track orientation in the xy plane. Logos in b reproduced with permission from: TrackMate, Jean-Yves Tinevez; Napari, Juan Nunez-Iglesias. We demonstrate the utility and robustness of Ultrack through a comprehensive set of benchmarks and real-world applications, including top-scoring performances on the cell tracking challenge (CTC)11. Our method shows improved tracking performance across various use cases and scales, from multicolor and label-free 2D datasets to large-scale 3D time-lapse sequences of developing embryos. Additionally, we introduce a new dual-channel sparse labeling approach that enables the generation of high-fidelity ground truth for tracking validation. This method combines ubiquitous fluorescence labeling with sparse, random labeling at a distinct wavelength, allowing for the creation of annotations that would be impossible to produce manually. This innovative validation strategy pushes the limits of cell tracking assessment, enabling the evaluation of tracking performance over extended time periods and in increasingly complex tissue environments. To ensure broad accessibility and ease of use, we provide intuitive interfaces in Fiji35, Napari36, Python and command line, catering to researchers with diverse computational backgrounds. We also made efforts to ensure interoperability with existing popular cell tracking frameworks such as TrackMate37 and Mastodon38 (Fig. Thus, by enabling more accurate, scalable and accessible cell tracking, Ultrack can accelerate research in developmental biology, cancer research and regenerative medicine, offering new insights into cellular behavior and tissue dynamics. To address this challenge, Ultrack considers multiple segment hypotheses instead of prematurely committing to a particular choice of segments (Fig. These hypotheses can be obtained from the original imaging data pixel intensities, multiple algorithms (for example, watershed plus Cellpose) or from one or several algorithms configured with different parameters. This approach allows the integration of various segmentation methods, maintaining compatibility with the growing array of novel off-the-shelf segmentation models39,40. However, the main challenge of this approach and previous joint segmentation and tracking methods, as well as the main obstacle for their practical application, is the potentially large number of possible segmentation hypotheses, especially for large multidimensional datasets. To address the challenge of managing multiple segmentation hypotheses, we use ultrametric contour maps (UCMs)41,42,43, which provide a compact representation of these hypotheses. Intuitively, UCMs are multilevel contour maps that represent a hierarchy of possible cell boundaries, where the strength of each boundary reflects its likelihood of being a true cell edge (Fig. This hierarchical structure allows for the efficient encoding of multiple possible segmentations within a single map. More formally, UCMs partition the image space into nested segments, where valid segments do not overlap, or are contained within another. This property enables Ultrack to efficiently consider multiple segmentation possibilities while tracking cells, even in terabyte-scale datasets (Fig. 1e), that are challenging for existing joint segmentation and tracking solutions26,28,29,30. Hence, the canonical Ultrack input consists of two maps for each time frame: (i) a foreground map distinguishing potential cells from the background and (ii) a grayscale image representing the multilevel contour map (that is, UCM; Fig. These two input maps can be generated from any method, making Ultrack compatible with a wide range of existing workflows (see Methods for technical details). Once all candidate segments and links between segments are determined, tracking involves solving a complex combinatorial problem of selecting and associating the correct cell segments across time from the set of multiple segmentation and linkage hypotheses (Fig. 1d, we formulate this combinatorial problem as an integer linear program (ILP)28,44 that simultaneously solves for the optimal selection of temporally consistent segments while adhering to biological constraints. These constraints include cell division, cells entering (for example, moving into the frame) or exiting (for example, cell death or moving out of the field of view), and ensuring that a pixel is not assigned to more than one cell45, denoted by auxiliary variables xδ., xα. This efficient and versatile mathematical formulation leverages highly optimized solvers (Gurobi46, Coin47) and thus can handle tens of millions of segments from terabyte-scale datasets (Fig. A 3D image of a developing zebrafish embryo recapitulating cell lineages and migration patterns is shown in Fig. Image segmentation techniques, from classical methods like watershed to advanced deep-learning models, face considerable challenges in parameter tuning due to several factors: (i) computational: extensive resources are required to explore a large parameter space; (ii) practical: users often lack the expertise to train or tune parameters effectively; and (iii) technical: a single optimal parameter set may not fit all aspects of a diverse dataset, such as when segmenting cells of varying sizes. Ultrack addresses these challenges by integrating multiple segmentation labels derived from different parameter settings into a single multilevel contour map. 1d), resulting in more effective segmentation than what could be achieved with any single parameter setting—even a single optimal one. To demonstrate this key feature of Ultrack, we tracked human hepatocarcinoma-derived cells expressing YFP-TIA-1 (ref. 48) in their nucleus and cytoplasm from the CTC11 using the Cellpose ‘cyto2r' weights32 without any fine-tuning (that is, deep-learning retraining). In this dataset, cells have varying intensities, impairing the model's ability to consistently segment every cell correctly. For example, using original image intensities (γ = 1.0), the segmentation model fails to detect dimmer cells (Fig. Conversely, altering the dynamic range by taking the power of the image intensities with values closer to zero (γ = 0.1, 0.25, 0.5) improved the segmentation of dimmer cells but led to incorrect segmentation or non-detection of now-saturated bright cells (Fig. a, Segmentation pipeline using gamma correction with varying settings (γ = 0.1, 0.25, 0.5 and 1.0) to generate diverse segmentation hypotheses. c, Impact of gamma correction on segmentation quality: original images (top row), segmentation results (second and third rows) with errors highlighted by colored arrows, foreground maps (fourth row) and contour maps (fifth row). d, Quantitative comparison of F1 score of detections, TRA and false-negative cell detections for individual gamma settings and Ultrack's combined approach (all four). Arrows within parenthesis indicate whether lower (↓) or higher (↑) values are better. To segment and track the combination of all parameterizations of gamma, we extracted the binary foreground and contour maps from the Cellpose labels (Fig. Next, we combined them into a single binary foreground and a multilevel contour map for tracking (Fig. We evaluated the tracking and segmentation performance for each individual gamma setting, as well as for the combined contour map approach. We used three metrics for evaluation: the F1 score of detections, the tracking accuracy (TRA)49 and false-negative detection rate. For false-negative detection, lower scores indicate better performance, while for F1 and TRA, higher scores are better (Methods). These results demonstrate that Ultrack achieves more accurate tracking by leveraging multiple segmentation hypotheses, even outperforming the optimal single parameter setting (γ = 0.5 or 0.25 depending on the metric). This indicates that combining diverse segmentation inputs is often more effective in practice, thus liberating the need for precise parameter tuning and offering a more robust approach to cell tracking. Multicolor labeling has been shown to enhance cell segmentation and tracking capabilities, particularly in complex cellular environments50,51. However, applying state-of-the-art deep-learning models to such data often requires training on dataset-specific annotations, as pretrained weights are typically optimized for single-channel data. Ultrack addresses this challenge by combining multiple segmentation inputs in two ways: by varying parameters, as demonstrated earlier, and by integrating outputs from different segmentation algorithms applied to separate color channels. We demonstrate this key feature of Ultrack on a three-channel ‘multicolor' dataset of metastatic breast adenocarcinoma cells (MDA-MB-231) with red, green and blue markings from a lentiviral gene ontology (LeGO) vector system52, and show that we can effectively use models like Cellpose's ‘cyto2'32 alongside classical algorithms by applying them independently to each channel and then combining the outputs into a single multilevel contour representation. This approach circumvents the need for retraining typically required by other approaches37,53. While classical image processing approaches like watershed are generally less precise at splitting cell instances, they provide greater control and allow for the detection of dimmer cells, complementing the more sophisticated Cellpose output. Furthermore, Ultrack can use the three color channels as a feature when associating segments between frames, which helps connect the correct segments across time (Fig. See Methods for a description of the color features association. a, Multichannel, multi-algorithm tracking pipeline: individual labels are generated for each color channel (R, red; G, green; B, blue), from which foregrounds and contours are derived and subsequently integrated for joint segmentation and tracking. b, Stepwise improvement of segmentation strategies (left to right): (i) individual algorithms applied to grayscale images, (ii) applications to multicolor data, (iii) combination of labels from both algorithms and colors and (iv) enhancement with color features in association scores. Colored arrows highlight specific error types. c, Representative subset of lineages recovered by Ultrack, showcasing cells' respective colors, and examples of captured division events. d, Quantitative evaluation of tracking accuracy for the approaches described in b. Final combined results are available at https://t.ly/K-hA7. We assessed the effectiveness of our multichannel multi-algorithm approach using progressively more sophisticated strategies: (i) individual pipelines using Cellpose and watershed algorithms on a grayscale version (maximum intensity across multiple channels); (ii) each algorithm applied independently on multiple channels; (iii) combination of outputs from both algorithms in the multicolor configuration (Fig. 3a); and (iv) integration of color features into the association scores to improve linking of segments. We manually curated 413 cell tracks, creating a gold-standard dataset for benchmarking our results. Incorporating color channels enhances cell distinction compared to grayscale versions, notably reducing under-segmentation (Fig. The combination of watershed and Cellpose algorithms proves particularly effective, improving the detection of dimmer cells while maintaining Cellpose's accurate segmentation. Furthermore, color-feature linking improves tracking in ambiguous scenarios (Fig. Overall, Ultrack steadily increased accuracy as additional features (that is, segmentation labels, color information) were incorporated. This demonstrates its effectiveness in creating valid lineages (Fig. 3c and Supplementary Video 3) from a large pool of segmentation hypotheses without requiring a fine-tuned segmentation model, highlighting its versatility and robustness in multicolor cell tracking scenarios. As demonstrated in the previous section, off-the-shelf segmentation models face challenges when datasets deviate greatly from the training distribution. Users then often resort to time-consuming tasks such as annotating multiple segmentation labels to fine-tune a new model or implementing custom algorithms based on their expertise. This is particularly common because of the high diversity of imaging modalities and datasets typically encountered in biology. Here, we show that our approach can avoid these requirements by directly tracking image intensities as if they were multilevel contours, thus circumventing the need for segmentation labels as input. Our method relies on a single assumption: the intensity map must highlight cell boundaries in some way. This could be achieved through membrane labeling, which delineates cells as a contour map, or through nuclear labeling, which highlights cell interiors. By inverting the latter, we obtain an image where boundaries have higher intensity values than cell interiors. To evaluate this approach, we tracked A549 cells using quantitative phase imaging (QPI) data acquired with recOrder57 on a widefield transmission microscope. Traditional segmentation models like Cellpose perform poorly on this data because QPI was not part of their training set58. We obtained the intensity map using VSCyto2D58, a convolutional neural network for virtual staining that predicts membrane and nuclear markers from label-free data. The foreground was computed by manually thresholding the membrane channel and applying morphological filtering to close holes and remove small objects. We then applied Ultrack to the foreground and the intensity map encoding cell contours (Supplementary Video 4). For benchmarking, we tracked the virtual stained nuclear channel and manually curated a set of 247 tracklets of lineages starting in the first frame to the last frame. We compared these data against Cellpose with ‘livecell_cp3' weights59 applied to the QPI channel, followed by tracking. With Ultrack applied to the combined membrane and nuclear virtual staining channels, we detected most of the cells (Extended Data Fig. 1b,c), while the off-the-shelf model failed to recognize them. We missed 2,802 fewer associations between cells (that is, false-negative edges) and 3,014 cell detections (that is, false-negative detections) than virtual staining with Ultrack, at the cost of detection of 1,000 false-positive cells yielding a much higher F1 score (0.812 versus 0.347). This shows that tracking from image intensity directly, in this case from virtual staining, offers a competitive alternative to supervised segmentation and tracking methods, which often fail when presented with unseen imaging modalities, such as label-free quantitative polarization, Zernike phase contrast, differential interference contrast or for different cell types, thus eliminating the need for extensive human annotation to train segmentation models. Imaging systems must balance multiple factors, including the field of view size, spatial resolution, imaging speed and depth60. A critical limitation often encountered is acquisition speed (that is, temporal resolution), which hampers cell tracking performance. This is especially problematic when cells move rapidly and have similar appearances, complicating the assignment of cells between consecutive frames. Registration of adjacent frames often compensates for limited time resolution. However, collective cell motion within growing or deforming tissues frequently demonstrates local coherence that does not conform to simple linear or affine transformations. This is due to the complex dynamics of some biological processes. This approach effectively minimizes apparent motion when cells present distinct migration patterns. Computing nonlinear registration can be extremely time-consuming or require complex, specialized algorithms. To address this, Ultrack provides its own graphics-hardware-accelerated (GPU) routines to compute vector fields for 2D and 3D time-lapse recordings. These routines are based on automatic differentiation from deep-learning models63, offering efficient computation without sacrificing accuracy (Methods). The movement vector fields can be integrated into the tracking step, applying local movement to each candidate segmentation. This approach avoids altering the original image data. Moreover, Ultrack accepts any vector fields represented as a grid, allowing users to utilize their preferred registration methods. This dataset features nuclear-labeled embryo cells imaged using selective plane illumination microscopy. The epithelial cells' ellipsoidal surface was transformed into planar cartography using the ImSAnE toolbox65 to facilitate downstream analysis. However, this cartographic transformation introduced distortion and excessive motion on the polar boundaries (Extended Data Fig. We compared the tracking accuracy with and without flow registration (Extended Data Fig. 2b) at various temporal resolutions, starting by skipping ten frames (Δt = 10) up to the original frame rate (Δt = 1; Extended Data Fig. The results show: (i) at Δt = 10 tracking accuracy improved from 0.443 to 0.623 with flow registration, in this scenario the most motile cells move more than 150 voxels between frames, a substantial distance given that the median cell diameter of approximately 19.64 voxels; (ii) at Δt = 5 accuracy increased from 0.571 to 0.756; (iii) at original time resolution (Δt = 1) similar results were obtained (0.927 versus 0.929). These results demonstrate that Ultrack's temporal registration capability enhances tracking accuracy, particularly when dealing with complex cell movements or lower temporal resolution datasets. Concurrent work has shown that Ultrack remains effective even in scenarios that prioritize speed over resolution, such as neural dynamics imaging, where z-resolution is often reduced to enable faster acquisition. The WormID benchmark66 demonstrates Ultrack's robustness across a range of imaging resolutions, from 1 × 0.1604 × 0.1604 μm to 1.5 × 0.32 × 0.32 μm per voxel67. All these features—including its ability to handle multiple segmentation hypotheses, integrate multichannel data, leverage intensity-based tracking and apply temporal registration—collectively enable robust cell tracking in complex biological contexts. These capabilities are particularly valuable when confronting the challenges posed by dense, 3D tissues with dynamic cellular behaviors. To demonstrate the power and versatility of this approach, we next evaluated Ultrack's performance on one of the highly competitive CTCs11. The CTC is a long-standing benchmark for cellular segmentation and tracking methods11. It provides participants with training datasets to train and tune their algorithm parameters. This structure prevents information leakage between the training and testing phases and offers an unbiased evaluation of each algorithm's performance. We assessed our method using five 3D datasets: breast cancer cells, Caenorhabditis elegans (worm)68, Drosophila melanogaster (fly)31 and T. castaneum (beetle), as well as cartographic projections of T. castaneum datasets64. The worm and breast cancer cells datasets include manually curated lineage data and cell coordinates, as well as silver-standard segmentation annotations derived from a consensus among prior challenge submissions69. Using these annotations, we trained a U-Net model45,70 that predicts two outputs per voxel: (i) whether the voxel is on a cell contour and (ii) whether that voxel is in the foreground. We optimized these predictions using three dice losses71: (i) binary foreground prediction, (ii) binary cells' contour prediction and (iii) U-Net's intermediate cell contour predictions72 (Methods). The cells' contours and foreground were then used as inputs for Ultrack. As shown in Extended Data Table 1, Ultrack achieves a top combined segmentation and tracking score of 0.844 for the worm dataset. For the cancer cells dataset, our approach underperformed, most likely because of over-fitting during training the segmentation model on limited data: our cross-validation reported an average TRA of 0.905 while achieving a much lower score of 0.818 on the test set. In general, sufficient training data for segmentation are not always available or easily obtained, which is a typical situation users face. This is the case for the fly, beetle and beetle projection datasets, which lack ground truth to train segmentation models. To circumvent this limitation, the leading deep-learning-based submissions are based on point coordinates and movement, relying only on track annotations. Instead, we asked whether we could forgo deep learning and use classical image processing techniques for segmentation. We opted for a straightforward method. First, we detected the foreground using a difference of Gaussians filter and applied Otsu's threshold54 to select the foreground voxels. Next, we used image intensity as a proxy for cell boundaries by inverting and normalizing these intensities between 0 and 1, so dimmer voxels, which are more likely to be the cells' contours, represent the highest values. Combining this simple image processing approach with Ultrack's handling of multiple hypotheses implicit in the contour representation yields noteworthy results compared to competing approaches that require training data. As shown in Extended Data Table 1, for the fly dataset, Ultrack achieved a combined segmentation and tracking score of 0.708, compared to 0.617 for the next best method. Similarly, for the beetle dataset, Ultrack scored 0.841, while the next best result was 0.804 (see Methods for additional details). Overall, these results demonstrate Ultrack's strong performance even without deep-learning models, achieving more first-place rankings on 3D datasets than any other method. The next best performer, MPI, achieved first place only twice on the TRA metric, while never ranking higher than third place in segmentation scores across their submissions (Extended Data Table 2). See Supplementary Video 6 for recordings of the results. While the CTC provides a valuable unbiased benchmark for evaluating segmentation and tracking, it relies primarily on human-generated annotations. These annotations, while crucial, are inherently limited in scale and potentially in quality due to the challenge of manually annotating the latest-generation 3D live microscopy datasets1,2,6,7,9,15,16. Such datasets often span hundreds of gigabytes to terabytes, recording thousands of cells over hundreds or more time points, making manual annotation a task that is not only time-consuming but often beyond human capability. Moreover, in densely labeled or multidimensional datasets, distinguishing individual cells can be highly uncertain and sometimes impossible15,16,24,31,64. Encouraged by Ultrack's performance on the CTC, we developed a higher-fidelity ground-truth dataset that could overcome these limitations and potentially provide a more challenging benchmark of a crowded and dynamic cellular environment from a developing zebrafish embryo. We applied a dual-labeling protocol that combines ubiquitous fluorescence nuclear labeling Tg(ef1α:H2B-mNeonGreen) with sparse random nuclear labeling at a distinct wavelength (pMTB-ef1-H2B-mCherry). Sparse labeling was accomplished by early-stage microinjection of a marker (Tol2 transposase-mediated integration; Methods) that propagates to daughter cells upon division, effectively recapitulating their lineage (Fig. a, Validation pipeline using sparse fluorescence labeling: (i) Embryos with ubiquitous green nuclear labeling Tg(ef1α:H2B-mNeonGreen) are injected with DNA plasmids expressing red fluorescent protein pMTB-ef1-H2B-mCherry; a single embryo is then selected for imaging. (ii) Simultaneous imaging of red and green channels. b, (iii) Independent tracking of cell nuclei in both channels. (iv) Selection and manual curation of a subset of cell nuclei tracks from the red channel. (v) Comparison of curated tracks to those obtained from ubiquitous labeling. c, Comparative visualization of red (first column) and green (second column) channel segmentation and tracking results at different time points, and combined results overlaid grayscale intensity colored in red and green (last column); cell density and tracking difficulty increase over time (top to bottom), d, Quantitative evaluation of tracking accuracy under different conditions and durations: using deep learning for cell contour estimation (first row) versus direct image intensity-based segmentation (second row); measurements taken up to the first 150 frames (first column) and 500 frames (second column). Metrics shown are proportions of false-negative edges (FN), identity switches (IS), false-positive divisions (FP-D), false-negative divisions (FN-D) and their sum. The red vertical line indicates the baseline sum error rate for 150 frames of 0.078 for comparison with Malin et al.24. Tracking results are available at https://t.ly/WZV-g. Schematic in a created with BioRender.com. The sparsity of this second channel greatly simplifies tracking and allows for rapid curation into a set of high-quality, or ‘platinum-level', annotations (Fig. Cells in the sparse channel were automatically tracked using Ultrack and then manually curated to ensure the highest quality standards (Fig. This curation process involved selecting long, green-overlapping, high-quality lineages, conducting visual inspections and making necessary corrections, resulting in 152 annotated tracklets spanning from 85 to 521 frames. This methodology not only simplifies the manual curation process but also reveals cell trajectories that are challenging to annotate manually using conventional methods on dense datasets (Supplementary Video 7). 4c, tracking is easier at earlier stages of development when cells are less numerous, less densely packed, and when the optical properties of the tissue are more favorable for imaging. Despite these challenges, cell nuclei in the sparse channel are consistently easier to track compared to the dense channel (Fig. We used these high-quality sparse-channel-derived tracking labels to evaluate Ultrack's accuracy on the ubiquitously labeled channel. We used two segmentation approaches as in ref. 45: (i) image processing to produce foreground maps and nuclei boundaries directly and (ii) a convolutional neural network to generate foreground and contour maps. To quantitatively assess the performance, we evaluated our automatic dense tracking results against the manually curated sparse tracking over two time periods (Fig. Assessing tracking performance with five metrics24, all expressed as proportions relative to the total number of annotated edges: (i) false-negative edges (that is, missing connections between cells), (ii) identity switches (that is, incorrect association between cells), (iii) false-positive divisions, (iv) false-negative divisions and (v) the sum error rate that combines the first four metrics. As expected, tracking accuracy drops for longer imaging periods (500 versus 150 frames; Fig. Notably, Ultrack with a deep-learning-based input, which benefits from available training data, consistently outperforms plain image processing contour and foreground maps, maintaining comparable accuracy over 500 frames to what the image intensity-based approach achieves in just 150 frames. To contextualize our results within recent advances in cell tracking for embryo-scale 3D datasets, we compared our findings to those reported by Malin et al.24. Their study, which used a similar multi-view light-sheet microscope setup73, established a baseline sum error rate of 0.078 for tracking neuronal cells in zebrafish embryos during 150 frames. 4d, Ultrack achieves a lower proportion of sum errors for both deep-learning and intensity-image-based contour maps on 150 frames (0.049 and 0.070, respectively). Moreover, when using deep learning, we maintain accuracy (0.073) comparable to the baseline (0.078) even over a much more challenging 500-frame period. This extended performance is particularly noteworthy, as it underscores the robustness of our method in maintaining high tracking accuracy over prolonged periods of embryonic development despite increasing tissue complexity, cell density and imaging artifacts. These results not only validate the effectiveness of our dual-channel approach but also highlight Ultrack's potential to push the boundaries of long-term cell tracking in complex biological systems. To demonstrate Ultrack's scalability and performance on massive datasets, we used it to recover cell lineages of nuclear-labeled zebrafish embryos9 acquired using the fast (one stack per minute), high-resolution (×20 magnification and 1.0-NA Olympus objective) and large-imaging-volume (≥1 mm3) DaXi light-sheet microscope6, producing uncompressed time-lapse data ranging from 1.7 to 3.7 terabytes, capturing 8.6 h to 13.2 h of zebrafish development (Fig. a, Pipeline for joint segmentation and tracking using a convolutional neural network (CNN) and Ultrack. c, Detailed views of fully automated lineage reconstructions, showcasing two rounds of cell divisions and their corresponding 2D slices. This indicates that, on average, 50% of the lineages are error-free for at least 62 frames. Tracking data results are available at https://t.ly/vXFyb. Our workflow began with training a U-Net to predict nuclei pixels (foreground) and their contours (Fig. We then applied Ultrack to these network predictions. Leveraging Ultrack's command-line interface (CLI) and SLURM74, we processed these massive datasets within hours using distributed computing in a high-performance computing (HPC) environment. For example, a 3.7-TB dataset (791 × 448 × 2,174 × 2,423 voxels, T × Z × Y × X) was processed in approximately 8.2 h (Fig. Ultrack's computational scaling extends across computing environments, from high-performance clusters to off-the-shelf laptops. Ultrack supports out-of-core processing: it can process arbitrarily large datasets, even those substantially larger than available memory (RAM)—as long as a single image (2D or 3D) can fit into memory (Methods). We could process the same 3.7-TB dataset on a 32-GB RAM laptop in 5 days, accessing data remotely via a 1-Gbps connection (Fig. To assess performance, we manually evaluated time-lapse recordings until the first tracking mistake, recording the duration of error-free tracks. We used a stratified sampling strategy, manually annotating distinct regions (head, body, tail bud and presomitic mesoderm) at various time points and image qualities. From these strata, we randomly sampled approximately 140 lineages per embryo (n = 3). Figure 5d shows each fish's cumulative proportion of error-free tracks and their mean. On average, it took 62 frames for a tracking error to appear in 50% of the lineages, meaning that for a period of approximately 1 h of development (60 frames), half of the tracks are completely error-free, demonstrating Ultrack's robust performance over extended periods. Figure 5c and Supplementary Video 8 provide a detailed view of an automatically reconstructed lineage, capturing two rounds of cell divisions and illustrating Ultrack's ability to maintain accurate tracking through multiple cell generations. These results underscore Ultrack's capacity to efficiently process and accurately track cells in multi-terabyte, high-resolution datasets utilizing HPC resources or standard hardware. This scalability and versatility make Ultrack a powerful tool for analyzing large-scale developmental processes at unprecedented resolution and scale. Cell tracking is often crucial in scenarios with fewer cells, where perfect lineage reconstruction is necessary to uncover precise details of cell behavior, such as division, migration and death75,76. To evaluate Ultrack's performance in such a context, we chose the zebrafish neuromast as our model system, given its well-defined structure and importance in studying sensory organ development. Previous efforts in segmenting and tracking cells in this organ included manual tracking of cells77 or semiautomated segmentation of single frames78. To our knowledge, no other study has reported the fully automatic segmentation and tracking of cells in this complex 3D organ for extended time periods. We imaged a zebrafish neuromast for 42 h using membrane and nuclei markers. Our segmentation model was trained to detect only cells that express both markers. Imaging was performed on an Olympus IX83 microscope equipped with a microlens-based, super-resolution spinning-disk confocal system (VT iSIM, VisiTech International), using a ×60 1.3-NA silicone-oil objective. To ensure optimal segmentation accuracy, we fine-tuned a 3D Cellpose model32 to segment cells using both color channels. We then converted the Cellpose predictions into foreground and contour maps for input into the Ultrack algorithm (Fig. This approach allowed us to reconstruct lineages spanning the entire time-lapse recording, capturing diverse cellular events such as migration within the organ, cell division and cell death (Fig. b, Dendrogram recovered by Ultrack for 71 cells over 500 frames (41.7 h), with orange lines representing cell lineages and blue highlights indicating examples detailed next. d, Quantitative comparison between Ultrack (blue) and TrackMate (pink). The bar chart shows error counts for various metrics, with Ultrack obtaining 442 fewer mistakes than TrackMate. e, Two examples demonstrating Ultrack's robustness to over-segmentation, showing the original image (left), fragmented TrackMate results (middle) and accurate Ultrack segmentation and tracking (right) despite imperfect Cellpose input. To assess Ultrack's accuracy, we visually inspected the generated lineages and manually corrected any errors, creating ground-truth reference lineages. Using the CTC evaluation routines, we quantified Ultrack's performance (Fig. Ultrack achieved a TRA score of 0.9989, demonstrating its high precision in this challenging scenario. We compared Ultrack's performance against TrackMate37, using identical Cellpose segmentations as input. To ensure a fair comparison, we first optimized both Ultrack's and TrackMate's parameters using a different 50-frame dataset. We then applied these optimized settings to the full 500-frame dataset reported here. In this case, Ultrack's tracking results exhibit higher F1 scores of 0.9951 versus 0.9895 and more than twofold reduction in errors (279 versus 721), with 442 fewer mistakes needing manual correction when compared to TrackMate. The primary source of errors in TrackMate's results stemmed from imperfect Cellpose segmentations (Fig. Despite TrackMate's segmentation filtering options, it struggled to correct issues such as over-segmentation and false merges. In contrast, Ultrack's approach naturally incorporates these imperfect segments into its hypotheses, allowing for more accurate merging and tracking (Fig. These results demonstrate Ultrack's ability to maintain high tracking accuracy in this complex, long-term imaging scenario and, thus, its utility for detailed studies of cellular behavior and tissue dynamics. Cell tracking software often represents one of the most complex components in image analysis pipelines, primarily due to the necessity of handling the time dimension—a factor frequently overlooked in segmentation and other image processing operations. Moreover, it operates on non-matrix data (that is, track lineages), where efficient data processing routines are less prevalent. This complexity has historically restricted accessibility to a broad user base, with only a few long-standing solutions like TrackMate37 and others21,22,31, alongside specialized solutions for handling larger datasets24. Ultrack addresses these challenges by offering a versatile, user-friendly solution that caters to a wide range of users, from biologists with minimal programming experience to machine learning experts requiring customizable inputs. By providing multiple interfaces and scalable processing capabilities, Ultrack democratizes access to advanced cell tracking technology, enabling efficient analysis of datasets of various sizes across different computational resources (Fig. At the heart of Ultrack's accessibility is its multi-interface approach. Ultrack offers two options for users who prefer a graphical interface: Napari and Fiji. The Napari plugin (Extended Data Fig. 3a) integrates transparently with the Python ecosystem, providing a comprehensive set of features for noncoding users. This interface offers graphics-hardware-accelerated preprocessing, vector-field flow computation and full configuration of Ultrack parameters. Complementing this, the Fiji plugin35 (Extended Data Fig. 3b) exposes Ultrack's algorithm to Fiji's mature ecosystem and broad user base, allowing seamless integration with existing Fiji workflows and ensuring compatibility with popular tools like TrackMate37 and Mastodon38. Supplementary Video 10 shows them in action. For users comfortable with programming, Ultrack provides a user-friendly Python API (Extended Data Fig. 3c,d) that exposes all of its functionalities, allowing for easy customization and reproducible workflows. This API accepts various array formats (for example, NumPy, Zarr and Dask), and can export data into multiple file formats, including networkx79, tables (that is, data frames)80, CTC11 and TrackMate37 formats, enhancing its interoperability with other tools and workflows. Additionally, Ultrack offers a CLI (Extended Data Fig. This CLI accommodates any file format supported by existing Napari reader plugins, eliminating the need for data conversion—a crucial feature for handling large datasets. Ultrack advances cell tracking by offering a robust, scalable solution for analyzing complex cellular dynamics across various scales and dimensions. Its integration of segmentation and tracking, leveraging multilevel contour maps (UCMs), enables effective analysis in challenging scenarios where previous methods often struggled. The method's versatility is demonstrated through its performance in 3D embryonic CTCs, even when using non-learned classical image processing algorithms. This ability to track without training data or deep fine-tuning addresses a critical challenge in the microscopy field, where 3D annotations are often scarce11. While Ultrack can leverage learned models when available for optimal performance, it remains effective even without them. Furthermore, Ultrack's capacity to combine results from parameter sweeps eliminates the common problem of identifying optimal segmentation parameters. This flexibility extends to working directly with image intensities thus circumventing the need for explicit segmentation in certain scenarios, for example, virtual staining of label-free images. This approach is not limited to label-free imaging, such as QPI, but extends to virtually any imaging method—whether conventional fluorescence microscopy, label-free techniques or virtual staining approaches—that can produce intensity maps highlighting cellular structures, such as nuclei or membranes. By leveraging these intensity-based representations and using direct tracking, Ultrack offers an adaptable and efficient solution to avoid extensive annotation or fine-tuning (that is, deep-learning retraining) of segmentation models. Ultrack also enables processing multichannel data by combining multiple single-channel labels, a feature that, when paired with technologies for engineering multicolored cells50,51, promises a new generation of high-accuracy lineage reconstruction. Finally, Ultrack addresses challenges associated with low temporal resolution through GPU-accelerated flow-field registration between frames, broadening its applicability to more experiments. The introduction of a dual-channel sparse labeling approach for validation offers a more challenging and heterogeneous benchmark compared to previous studies focused on specific cell subsets or biological contexts15,16,31,64. This approach paves the way for more robust validation of tracking algorithms over extended periods in large datasets. Ultrack's scalability, tested on both HPC clusters and laptops, addresses the growing demand for tools capable of handling increasingly large and complex datasets produced by advanced imaging technologies1,6,7. This flexibility in computational requirements democratizes access to high-quality cell tracking across diverse research settings. Ultrack's integration with popular platforms like Fiji35 and compatibility with established tracking frameworks such as TrackMate37 and Mastodon38 positions it for widespread adoption. Its multiple interfaces, ranging from graphical user interfaces (GUIs) to command-line tools for HPC environments, align with community efforts to make complex computational tools more accessible to biologists19,20,33,34,36,81. First, the segmentation hypotheses are only as good as the provided foreground and contour maps. While Ultrack is robust to occasional segmentation errors, it may fail when presented with an overwhelming number of incorrect segmentations or systematic errors that persist over time—in such cases, the ILP optimization might converge on erroneous solutions. Additionally, as shown in Extended Data Fig. 2, large cell movements between adjacent frames violate our assumption of segmentation consistency and can impair tracking performance unless registration is used. A few strategies can improve Ultrack's effectiveness. For instance, since Ultrack uses region-merging hierarchical segmentation, contour maps that initially over-segment cells are actually beneficial, as they generate more hypotheses for the algorithm to consider. To help users optimize these aspects, we maintain comprehensive documentation (https://royerlab.github.io/ultrack/optimizing.html), providing guidelines and practical tips for achieving optimal tracking results across different applications. One might question why we chose to use foreground and contour map prediction models rather than established instance segmentation approaches like Stardist18 or MicroSAM20 in Figs. The answer lies in the nature of network contour predictions: their probabilistic outputs capture subtle variations in cell boundary likelihood, producing smoothly varying contour values. This continuous representation generates a richer set of segmentation hypotheses compared to instance segmentation methods, which output discrete labels that must then be converted to foreground and contour maps. The probabilistic nature of contour predictions thus provides Ultrack with more information to work with during the optimization process. Looking ahead, several promising avenues for future development emerge: (i) Integration of advanced deep-learning techniques, particularly transformer-based models for sequence prediction82, could further improve long-term tracking accuracy. (ii) Expansion of the sparse labeling approach to generate large-scale, high-quality datasets for training and benchmarking future tracking algorithms. (iii) Development of interactive correction tools leveraging Ultrack's multiple segmentation hypotheses to facilitate efficient manual curation of tracking results. (iv) Exploration of Ultrack's potential applications beyond developmental biology, including cancer research, immunology and regenerative medicine. In conclusion, Ultrack's innovative approach for joint segmentation and tracking, combined with its scalability and accessibility, has the potential to accelerate research across a wide range of biological disciplines by enabling more accurate, efficient and accessible cell tracking. Ultrack opens new avenues for investigating complex cellular behaviors, tissue dynamics and developmental processes at unprecedented scales and resolutions. All zebrafish care and experimental procedures complied with protocols approved by the Institutional Animal Care and Use Committee at the University of California San Francisco (UCSF). Adult fish were maintained at 28.5 °C and fed twice daily using an automatic feeder83. Embryos were raised at 28.5 °C, and staged based on hours post fertilization (hpf). For light-sheet microscopy and injection experiments, we used two transgenic lines: Tg(h2afva:h2afva-mCherry; a gift from J. Huisken, University of Göttingen) and Tg(ef1α:H2B-mNeonGreen; a gift from D. Wagner, UCSF). Zebrafish neuromasts were imaged using the transgenic line Tg(cldnb:lyn-mScarlet) crossed with Tg(she:H2B-EGFP)84,85. MDA-MB-231 metastatic breast adenocarcinoma cells (American Type Culture Collection) were maintained in Dulbecco's Modified Eagle Medium (DMEM; VWR) with 10% (vol/vol) fetal bovine serum (FBS; Seradigm VWR), and penicillin and streptomycin (50 U ml−1; Thermo Fisher Scientific). Cells were stably modified with a variety of red, green and blue markings with a LeGO vector system, as described in Weber et al.52. Lentivirus was prepared as described in Shah et al.86. In brief, pseudoviral particles were generated using 293TN cells (System Biosciences, SBI) co-transfected with lentiviral packaging and envelope plasmids (psPAX and pMD2.G, gifts from D. Trono), and one plasmid of interest containing fluorophore eGFP, mCherry or EBFP2 (Addgene, 25917, 27339 and 85213). Supernatants containing lentivirus were collected, passed through a 0.45-μm filter, concentrated by ultracentrifugation at 200,000g for 4 h at 4 °C, resuspended in phosphate-buffered saline (PBS; VWR) and stored at −80 °C. MDA-MB-231 cells were transduced simultaneously with all three constructs at equal viral titer and a range of multiplicity of infections). To optimize color diversity, a fraction of cells expressing fluorophore was assessed at 5 days after transduction using fluorescence microscopy, and a condition with ≈50% transduction rate of all fluorophores (multiplicity of infection ≈ 0.7) was selected for subsequent experiments. Cells were seeded 12 h before imaging at a density of ≈10,000 per well on a glass-bottom plate coated with 50 μg ml−1 rat tail type I collagen (Corning) in 0.02 N glacial acetic acid (Sigma) overnight at 4 °C. Imaging was performed on a Keyence BZ- X800 imaging system equipped with temperature, humidity and CO2 control, and images were acquired at a magnification of ×10 at 10-min intervals for 2 days. DNA plasmids (pMTB-ef1-H2B-mCherry2 gifted by D. Wagner, UCSF) were extracted and purified with a QIAprep Spin Miniprep Kit (27104). Purified DNA was diluted to 100 ng l−1 in water (Invitrogen UltraPure DNase/RNase-free distilled water, 10977015) and stored at −20 °C. Zebrafish Tol2 mRNA was prepared using pT3TS-Tol2 plasmids shared by the Ekker lab87. pT3TS-Tol2 plasmid DNA was linearized by the restriction enzyme BamHI digestion. Linearized plasmids were gel-purified with a QIAquick Gel Extraction Kit (28704). The 5′-capped mRNAs were synthesized using Invitrogen mMESSAGE mMACHINE T3 Transcription Kit and purified by lithium included in the kit (Thermo Fisher, AM1348). The purified RNA was dried and dissolved with 20 l DNase/RNase-free water, diluted to 200 ng l−1 and saved as stock at −70 °C. Before the experiment, the stocked DNA and RNA were thawed, and 1 μl DNA plus 1 μl RNA were mixed with 3 μl of DNase/RNase-free water containing 0.05% phenyl red. All injections were done with an injector (WPI PV830 Pneumatic Pico Pump) and a micromanipulator (Narishige, Tokyo, Japan). Around 50 to 100 embryos were injected for each experiment. The injected embryos are ready for mounting and imaging at approximately 4.5 hpf. Embryos are first dechorionated and then gently embedded into a 1% solution of low-gelling-temperature agarose (Sigma, A0701). The samples are then transferred in a glass-bottom cell culture dish (35-mm cell culture dish with glass bottom 20 mm, Stellar Scientific) and positioned at the correct imaging angle using a custom-made capillary. When solid, the agarose surrounding the tail was cut off and removed using a dissection knife and forceps to permit full development and tail elongation. During the time lapses, the embryos are placed in an embryo medium solution with 0.016% tricaine to prevent embryo movement. The cells were seeded on a 12-well glass-bottom plate (Cellvis, P12-1.5H-N) and imaged using a widefield microscope (Leica Dmi8). Four hours after microinjection, injected embryos (~4 hpf) were screened using a Thunder Imager Model Organism (Leica Microsystems). An embryo with 20–30% H2B-mCherry2-expressing cells (compared to ubiquitous H2B-mNeonGreen cells) was selected and transferred to a glass-bottom petri dish (35-mm cell culture dish with glass bottom 20 mm, Stellar Scientific) containing 0.3× Danieau's embryo medium. The dechorionated embryo was then gently transferred to a petri dish containing 0.1% low-temperature melting agarose (Sigma, A0701) using a glass pipette. Subsequently, the embryo was transferred to FEP tubes (Valco-TFEP130; outer diameter, 1/16 inch; internal diameter, 0.03 inch; Vici Metronics) with agarose via a micropipette through a 200-μl tip (Eppendorf). Both ends of the FEP tube were sealed with 2% low-temperature melting agarose to secure the embryo. The tube was placed in the chamber of an adaptive multi-view multicolor light-sheet platform9 using a customized holder insertion tool filled with low-toxicity silicone (Kwik-Sil, World Precision Instrument). The mounting system was similar to that previously described in Zebrahub9, but with an upside-down mounting direction. For whole-embryo imaging, the multicolor-interleaved mode was used with an exposure time of 20 ms per channel with the OpenSimView9 microscope. The imaging volume comprised 493 z-planes with 1.625-μm z-steps. Images were captured at 90-s intervals, starting at the shield stage (6 hpf), and ending after 24 hpf. We use two orthogonal, oblique light sheets for each time point to improve the image field of view coverage. During imaging, the exposure was set to 20 ms, the z-step to 1.24 ms, the imaging volume was adapted to each sample and the time step was around 60 s. Time-lapse sessions are about 10-h long and usually cover a developmental window from 1 or 2 somites to 27 somites. The Ultrack package, including its REST API, and additional image analysis routine and plotting were implemented using Blosc, CuPy, FastAPI, imagecodecs, iohub, inTRACKtive, httpx, ggplot2, Napari, networkx, Numba, NumPy, Pandas, PyTorch, scikit-image, SciPy, SLURM, tifffile, traccuracy, uvicorn, websockets and Zarr36,63,74,79,80,88,89,90,91,92,93,94,95. Extended Data Table 3 presents the datasets evaluated in this paper. While not a requirement, some images of the CTC were upsampled for improved accuracy. Next, the raw multi-view images are fused. Phase cross-correlation techniques are applied to correct inter-frame movements to ensure stability throughout the time-lapse sequence. In cases where deep-learning methodologies are utilized, image normalization is performed, scaling pixel intensities to the range between 0 and 1 using user-specified lower and upper quantiles of 0.005 and 0.99999, respectively, and their background is subtracted using the subtract_background function from dexpv2 with σ = 20, which approximates a white tophat transform using morphological reconstruction by dilation. Flow-field estimation of large 3D volumes can be extremely challenging and computationally intensive, often following particle image velocimetry strategies that estimate local translation using phase cross-correlation for small patches of the data61. Instead of maximizing cross-correlation, which minimizes the mean-squared error between frames given a field in the coordinate space, we use gradient descent to optimize the coordinate space directly. This choice, justified by its enhanced computational efficiency and simplicity, allows for the integration of any differentiable function within the coordinate space. The loss function is defined according to equation (1): The grid_sample function, a differentiable data sampling method from PyTorch63, enables this process. This routine is tailored for specialized GPU hardware, greatly enhancing processing speed. The θ optimization is performed for a fixed number of iterations over multiple image resolutions, starting from the lowest resolution, which is then used to initialize the next θ optimization stage. For our experiment (Extended Data Fig. A detailed Jupyter notebook is available at https://github.com/royerlab/ultrack_supplementary/tree/main/configuration/flow_registration/. The conversion of multiple segmentation labels into foreground and contour maps involves two main processes. For the foreground map generation: (i) each label is transformed into a binary map, with nonzero pixels assigned a value of one (foreground) and zero pixels remaining as background; (ii) these individual maps are merged into a single foreground map by taking the maximum value across corresponding pixels from all individual foreground maps; (iii) this logical OR operation ensures that any pixel belonging to at least one segment is included in the final foreground map. For the contour map generation: (i) to preserve each label and its dual contour, we process each label map to create a binary contour map, where pixels are set to one if their neighboring pixels have differing labels, indicating a boundary; (ii) the final contour map is then created by averaging these individual binary contour maps—summing would yield the same results because the hierarchical segmentation reconstruction is based on ordering of values and not their absolute values, the benefit of averaging is to keep pixel values within the 0 to 1 range. We also implemented a deep-learning approach to generate foreground and contour maps, which we used for both the CTC submissions (where 3D segmentation annotations were available) and the large zebrafish dataset (Fig. Our implementation uses a U-Net architecture70 that processes single-channel grayscale images to produce two-channel outputs: foreground prediction and contour prediction, each with its sigmoid activation layer. During decoding, we upsample intermediate representations using linear interpolation and combine them with corresponding encoder features via skip connections. Network training minimizes a dice loss71, supplemented with auxiliary supervision at three intermediate resolutions during decoding, following the approach detailed in ref. All network weights and architectural details are publicly available at our supplementary repository (https://github.com/royerlab/ultrack_supplementary/), with the complete implementation available at our CTC submission repository (https://github.com/royerlab/ultrack_CTC_submission/). The goal of hierarchical segmentation is to compute a set of nested partitions (that is, segmentations) from a multilevel contour map, where each level corresponds to a different segmentation threshold. While a naive approach resorts to binarizing the contour map at each level and computing the connected components of the binary images, Ultrack uses a more sophisticated and computationally efficient method: the hierarchical watershed algorithm43,98. This algorithm represents the image as a graph G = (V, E), where V are the pixels or voxels, and E denotes the edges connecting adjacent voxels (four neighbors for 2D and six neighbors for 3D). It defines a non-decreasing order of edge weights as shown in equation (2): The algorithm then recovers nested segmentations using a region-merging approach based on Kruskal's minimum-spanning tree99 and Tarjan's union-find algorithms100. The process begins with an initial segmentation where each pixel forms its own segment. It then iteratively: (i) selects the minimum unseen edge (p, q); (ii) merges the segments of p and q if they are not already in the same segment; (iii) continues until all edges have been processed. This segmentation process operates within each connected component of the binary foreground map, allowing the algorithm to ignore background regions and thereby greatly speeding up the process in non-dense labeling (for example, nuclei). For further details on hierarchical segmentation processing and implementation, see ref. For each candidate segmentation, we compute the potential temporal associations (that is, tracks). We define ET as the set of pairs representing potential associations between these segmentations, with w(⋅,⋅) denoting their respective weights. The association process involves the following steps: (i) For each segmentation \(i\in {\boldsymbol{{\mathcal{H}}}}\), where t denotes the time point of segment i, we compute its candidates 2k-nearest neighbors within a predefined radius using their centroid coordinates at time t − 1; (ii) Optional step for additional filtering of neighbors, for example, color-feature association section. (iii) The association score between every segment i and their candidate neighbor j is calculated as w(i, j) = IoU(i, j)γ. and Si is the binary mask of region i (that is, segmentation hypothesis) and γ is consistently set to four throughout our study; (iv) Include into ET the k pairs per segment in t with the largest IoU, using their distance as a tie-breaker. Given \({\boldsymbol{{\mathcal{H}}}}\) and ET, as defined previously, we compute the joint segmentation and tracking by solving an ILP optimization problem. This ILP framework aims to find the optimal pairing between segments that maximizes their association while adhering to biological constraints. The ILP is formulated as follows in equations (3)–(9): In this formulation, yi indicates whether a segment is selected, and xij indicates the selected associations, xαi, xiβ and xδi are slack variables indicating an appearing, disappearing and dividing cell, respectively. Equations (4) and (5) ensure flow conservation, stipulating that an appearance, division or incoming association must correspond to at least an outgoing association or a cell disappearance. Equation (6) restricts cell division to preexisting cells, and equation (7) prevents selecting overlapping segmentations, thereby enforcing that each pixel is uniquely selected. Additional calculations were performed using the traccuracy Python package101. This is mathematically defined according to equations (10) and (11): The weights are set as follows: wNS = 5, wFN = 10, wFP = 1, wED = 1, wEA = 1.5 and wEC = 1, emphasizing the greater complexity in correcting missing units compared to deleting extra ones. For further details, refer to Matula et al.49. The SEG score calculates the average intersection over union (Jaccard index) across all segmentation instances. A ground-truth (gt) segmentation instance, Gj, is paired to a predicted instance Pi, as in equation (12), if: which provides a less-biased assessment of performance102 that combines the evaluation of both nFN and nFP. Because the data are sparse, only a subset of metrics are valid estimates for the remaining unlabeled data: FN: Proportion of false negative (that is, missing edges); FP-D: Proportion of false-positive divisions (that is, fake division) within a one-frame tolerance; FN-D: Proportion of false-negative divisions (that is, missing division) within a one-frame tolerance; The image intensity information can be used to filter out segments between adjacent frames that are unlikely to represent the same cell. To do that, we compute a normalized distance between the average intensities of segments, akin to a standard normal distribution z-score; if this distance exceeds a predefined threshold z, their association is excluded from the ILP formulation. where μj is the mean intensity vector within the mask of a segment Sj (represented as set) at time t and σj is the standard deviation. The operations are done element-wise per channel, and \(\max\) results in a scalar from it, here, compared to a neighboring segment Si at t − 1. where Ip is the intensity values at voxel p, such that for a three-channeled image Ip is a 3D vector. The motivation behind this formulation is the need for an intensity-invariant distance between colored cells. Hence, using a z-score formulation was a natural choice because the threshold z can be interpreted as how many standard deviations that a color distribution within a cell differs from other distributions of cells. The threshold z was set to 3.0 throughout the multicolor experiments, a symmetric quantile threshold equivalent to a 0.0027 probability under our z-score estimation. Fiji35 is a Java-based software that can be extended through plugins developed in the same language. To integrate Ultrack with Fiji, we developed a RESTful API103 using the FastAPI framework91, facilitating communication between Java and Python. This API uses websockets104 for persistent communication, enabling real-time event logging and output streaming from Ultrack to Fiji. We created a GUI in Java that serves as a Fiji plugin, leveraging the API provided with the Ultrack Python package. This interface grants access to features and functionalities of both systems. To promote broader interoperability, Ultrack adheres to API specifications detailed in our documentation (https://royerlab.github.io/ultrack/rest_api.html), allowing integration with various programming languages beyond Java and Python. Napari36 is a Python library for n-dimensional image visualization, annotation and analysis. We developed a custom plugin for Napari integration that still scales with large datasets. In addition to the tracking standard workflow, the plugin offers basic preprocessing operations to fit most users' needs, which can benefit from GPU speedups, if any are available. Ultrack's parameters can be tuned through the GUI. By the tracking execution finish, the results can be exported to several formats, such as TrackMate37, NetworkX graph79 and Zarr array. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The repository for the Ultrack Python package and its updated versions are available at https://github.com/royerlab/ultrack/. Additional supplementary scripts are available at https://github.com/royerlab/ultrack_supplementary/. The Fiji plugin can be found at https://github.com/royerlab/ultrack-imagej/. & Stelzer, E. H. Reconstruction of zebrafish early embryonic development by scanned light sheet microscopy. Shah, G. et al. Multi-scale imaging and analysis identify pan-embryo cell dynamics of germlayer formation in zebrafish. Chen, B.-C. et al. Lattice light-sheet microscopy: imaging molecules to embryos at high spatiotemporal resolution. Resolution doubling in light-sheet microscopy via oblique plane structured illumination. Shi, Y. et al. Smart lattice light-sheet microscopy for imaging rare and complex cellular events. Yang, B. et al. Daxi-high-resolution, large imaging volume and multi-view single-objective light-sheet microscopy. Swept confocally-aligned planar excitation (SCAPE) microscopy for high-speed volumetric imaging of behaving organisms. Lange, M. et al. A multimodal zebrafish developmental atlas reveals the state-transition dynamics of late-vertebrate pluripotent axial progenitors. Efficient processing and analysis of large-scale light-sheet microscopy data. The cell tracking challenge: 10 years of objective benchmarking. Feldman, D. et al. Optical pooled screens in human cells. Ivanov, I. E. et al. Mantis: High-throughput 4D imaging and analysis of the molecular and physical architecture of cells. & Campàs, O. Embryonic tissues as active foams. In toto imaging and reconstruction of post-implantation mouse development at the single-cell level. Wan, Y. et al. Single-cell reconstruction of emerging population activity in an entire developing circuit. Currie, J. D. et al. Live imaging of axolotl digit regeneration reveals spatiotemporal choreography of diverse connective tissue progenitor pools. Schmidt, U., Weigert, M., Broaddus, C. & Myers, G. Cell detection with star-convex polygons. In International Conference on Medical Image Computing and Computer-Assisted Intervention 265–273 (Springer, 2018). Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Ulicna, K., Vallardi, G., Charras, G. & Lowe, A. R. Automated deep lineage tree analysis using a Bayesian single cell tracking approach. & Averof, M. Tracking cell lineages in 3D by incremental deep learning. Automated reconstruction of whole-embryo cell lineages by learning from sparse annotations. Ulman, V. et al. An objective comparison of cell-tracking algorithms. Schiegg, M., Hanslovsky, P., Kausler, B. X., Hufnagel, L. & Hamprecht, F. A. Jug, F. et al. Optimal joint segmentation and tracking of Escherichia coli in the mother machine. In Bayesian and grAphical Models for Biomedical Imaging: First International Workshop, BAMBI 2014, Revised Selected Papers 25–36 (Springer, 2014). Türetken, E., Wang, X., Becker, C. J., Haubold, C. & Fua, P. Network flow integer programming to track elliptical cells in time-lapse sequences. Jug, F., Levinkov, E., Blasse, C., Myers, E. W. & Andres, B. Efficient algorithms for moral lineage tracing. Amat, F. et al. Fast, accurate reconstruction of cell lineages from large-scale fluorescence microscopy data. Pachitariu, M. & Stringer, C. Cellpose 2.0: how to train your own model. Accurate and versatile 3D segmentation of plant tissues at cellular resolution. Vijayan, A. et al. A deep learning-based toolkit for 3D nuclei segmentation and quantitative analysis in cellular and tissue context. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Sofroniew, N. et al. Napari: a multi-dimensional image viewer for Python. Ershov, D. et al. TrackMate 7: integrating state-of-the-art segmentation algorithms into tracking pipelines. The multimodality cell segmentation challenge: toward universal solutions. Edlund, C. et al. Livecell—a large-scale dataset for label-free live cell segmentation. Arbelaez, P. Boundary extraction in natural images using ultrametric contour maps. Najman, L. & Schmitt, M. Geodesic saliency of watershed contours and hierarchical segmentation. Najman, L. On the equivalence between hierarchical segmentations and ultrametric watersheds. & Nevatia, R. Global data association for multi-object tracking using network flows. Bragantini, J., Lange, M. and Royer, L. Large-scale multi-hypotheses cell tracking using ultrametric contours maps. In European Conference on Computer Vision 36–54 (Springer, 2024). Dynamic oscillation of translation and stress granule formation mark the cellular response to virus infection. Cell tracking accuracy measurement based on comparison of acyclic oriented graphs. Cai, D., Cohen, K. B., Luo, T., Lichtman, J. W. & Sanes, J. R. Improved tools for the brainbow toolbox. Skin cells undergo asynthetic fission to expand body surfaces in zebrafish. Weber, K., Thomaschewski, M., Benten, D. & Fehse, B. RGB marking with lentiviral vectors for multicolor clonal cell tracking. Hayashida, J., Nishimura, K. & Bise, R. MPM: joint representation of motion and position map for cell tracking. Otsu, N. A threshold selection method from gray-level histograms. The image foresting transform: Theory, algorithms and applications. Robust virtual staining of landmark organelles with Cytoland. Stringer, C. & Pachitariu, M. Cellpose3: one-click image restoration for improved cellular segmentation. B. in Handbook of Biological Confocal Microscopy 20–42 (Springer, 2006). Pereyra, M. et al. QuickPIV: Efficient 3D particle image velocimetry software applied to quantifying cellular migration during embryogenesis. & Dalca, A. V. Voxelmorph: a learning framework for deformable medical image registration. Paszke, A. et al. PyTorch: an imperative style, high-performance deep learning library. Jain, A. Molecular, cellular and mechanical basis of epithelial morphogenesis during tribolium embryogenesis. & Streichan, S. J. Tissue cartography: compressing bio-image data by dimensional reduction. Adhinarta, J. et al. WormID-Bench: a benchmark for whole-brain activity extraction in C. elegans. Sprague, D. Y. et al. Unifying community whole-brain imaging datasets enables robust neuron identification and reveals determinants of neuron position in C. elegans. Automated analysis of embryonic gene expression with cellular resolution in C. elegans. Akbaş, C. E., Ulman, V., Maška, M., Jug, F. & Kozubek, M. Automatic fusion of segmentation and tracking labels. European Conference on Computer Vision (ECCV) Workshops 446–454 (Springer, 2019). Ronneberger, O., Fischer, P. & Brox, T. U-net: convolutional networks for biomedical image segmentation. in International Conference on Medical Image Computing and Computer-Assisted Intervention 234–241 (Springer, 2015). In European Conference on Computer Vision 562–578 (Springer, 2018). Xie, S. and Tu, Z. Holistically-nested edge detection. Tomer, R., Khairy, K., Amat, F. & Keller, P. J. Quantitative high-speed imaging of entire developing embryos with simultaneous multiview light-sheet microscopy. & Grondona, M. SLURM: Simple Linux Utility for Resource Management. In Job Scheduling Strategies for Parallel Processing: 9th International Workshop 44–60 (Springer, 2003). & Hudspeth, A. Mechanochemical symmetry breaking during morphogenesis of lateral-line sensory organs. Viader-Llargués, O., Lupperger, V., Pola-Morell, L., Marr, C. & López-Schier, H. Live cell-lineage tracing and machine learning reveal patterns of organ regeneration. A., Linbo, T. H. & Raible, D. W. Spherical harmonics analysis reveals cell shape-fate relationships in zebrafish lateral line neuromasts. Hagberg, A., Swart, P. & Chult, D. S. Exploring network structure, dynamics and function using NetworkX. Pandas—powerful python data analysis toolkit (Pandas, 2015); https://pandas.pydata.org/pandas-docs/stable/index.html von Chamier, L. et al. Democratising deep learning for microscopy with ZeroCostDL4Mic. & Weigert M. Trackastra: Transformer-based cell tracking for live-cell microscopy. In European Conference on Computer Vision 467–484 (Springer, 2024). Lange, M. et al. ZAF, the first open source fully automated feeder for aquatic facilities. Peloggia, J. et al. Adaptive cell invasion maintains lateral line organ homeostasis in response to environmental changes. Dalle Nogare, D. E., Natesh, N., Vishwasrao, H. D., Shroff, H. & Chitnis, A. B. Zebrafish posterior lateral line primordium migration requires interactions between a superficial sheath of motile cells and the skin. Shah, P. et al. Nuclear deformation causes dna damage by increasing replication stress. Harnessing a high cargo-capacity transposon for genetic applications in vertebrates. Nishino, R & Loomis, S. H. C. CuPy: a NumPy-compatible library for nvidia GPU calculations. Huijben, T. A. P. M. et al. inTRACKtive: a web-based tool for interactive cell tracking visualization. Virtanen, P. et al. Scipy 1.0: fundamental algorithms for scientific computing in Python. Van der Walt, S. et al. scikit-image: image processing in Python. & Seibert, S. Numba: a LLVM-based Python JIT compiler. Second Workshop on the LLVM Compiler Infrastructure in HPC 1–6 (Association for Computing Machinery, 2015). Moore, J. et al. OME-Zarr: a cloud-optimized bioimaging file format with international community support. Moore, J. et al. OME-NGFF: a next-generation file format for expanding bioimaging data-access strategies. Playing with kruskal: algorithms for morphological trees in edge-weighted graphs. Tarjan, R. E. Efficiency of a good but not linear set union algorithm. Goyal, V., Schaub, N. J., Voss, T. C. & Hotaling, N. A. Unbiased image segmentation assessment toolkit for quantitative differentiation of state-of-the-art algorithms and pipelines. Fielding, R. T. Architectural Styles and the Design of Network-Based Software Architectures. Qt—cross-platform software libraries and apis (The Qt Company, accessed 9 July 2024); https://www.qt.io/ Ph.D. dissertation, KTH Royal Institute of Technology (2016). Kim, D., Kwon, C. & Hwang, I. Gaussian mixture probability hypothesis density filter against measurement origin uncertainty. Hirsch, P. et al. Tracking by weakly-supervised learning and graph optimization for whole-embryo C. elegans lineages. In International Conference on Medical Image Computing and Computer-Assisted Intervention 25–35 (Springer, 2022). Dzyubachyk, O., Van Cappellen, W. A., Essers, J., Niessen, W. J. & Meijering, E. Advanced level-set-based cell tracking in time-lapse fluorescence microscopy. Stegmaier, J. et al. Fast segmentation of stained nuclei in terabyte-scale, time resolved 3D microscopy image stacks. We extend our gratitude to D. Wagner and J. Huisken for generously sharing their zebrafish lines. We thank A. Abbas and P. Swoboda for their insightful discussions on Integer Linear Programming and optimization. We are grateful to B. Moretti, V. Pfannenstill and J. Ko for their valuable feedback and for being early adopters of Ultrack. was funded by National Institutes of Health awards R01GM137605 and R35GM153257. This work was funded by the Chan Zuckerberg Biohub San Francisco (CZB SF). We are deeply grateful to CZB SF donors P. Chan and M. Zuckerberg for their generous support, which made this research possible. Chan Zuckerberg Biohub, San Francisco, CA, USA Jordão Bragantini, Ilan Theodoro, Xiang Zhao, Teun A.P.M. Huijben, Eduardo Hirata-Miyasaki, Shruthi VijayKumar, Akilandeswari Balasubramanian, Tiger Lao, Sheng Xiao, Shalin Mehta, Adrian Jacobo, Merlin Lange & Loïc A. Royer Weill Institute for Cell and Molecular Biology - Cornell University, Ithaca, NY, USA Meinig School of Biomedical Engineering - Cornell University, Ithaca, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar prepared and imaged the zebrahub embryo. E.H.-M. curated and analyzed the label-free data. R.A. prepared and imaged the multicolor data. All authors contributed to editing the paper. Correspondence to Jordão Bragantini or Loïc A. Royer. The authors declare no competing interests. Nature Methods thanks Nathan Hotaling, Yu Mu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Rita Strack, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Pipeline for intensity-based joint segmentation and tracking from label-free quantitative phase imaging: Quantitative phase image (top left) is processed by VisCy58 to generate virtual staining of nuclei and membrane (second from left). These virtual stains are used to derive contours (third from left) and foreground maps (not shown) through filtering, subtraction, and thresholding. Ultrack uses these inputs to perform tracking (right). b, We compare the above against first applying a Cellpose segmentation to the quantitative phase image (bottom left), then deriving contours (and foreground map, not shown), and then applying Ultrack (right). c, Quantitative comparison of tracking errors using Cellpose59 segmentations (gray) versus virtually stained membranes (orange) as input. The bar chart shows the F1-score of detections, upper y-axis and the number of false negative (FN) edges, false negative (FN) detections, and false positive (FP) detections for each method, lower y-axis, from 247 tracklets spanning 48 frames. a, Consecutive frames, t (magenta) and t + 1 (cyan), highlighting divergent motion along the right boundary. c, Qualitative comparison of cell identity preservation: top row shows a cell identity switch in unregistered frames, identity switch indicated by red arrow, bottom row demonstrates improved consistency with flow registration. d, Comparative analysis of tracking accuracy for registered and unregistered data across different cell velocities (Δt), illustrating enhanced performance with registration. The left y-axis corresponds to the colored lines showing TRA scores, while the right y-axis and boxplots show cell movement distributions between adjacent frames in voxels. At Δt = 10.0, some cells move more than 150 voxels between frames - a substantial distance given the median cell diameter of approximately 19.64 voxels. Boxplots followed Tukey's original definition107 using the median for the center line, first (q1) and third quantile (q3) for the box limits and whiskers at q1 - 1.5IQR and q3 + 1.5IQR, where IQR = q3 - q1. a, Napari plugin interface showcasing multi-color automatic cell tracking results. The interface includes workflow selection, input configuration, image processing results, segmentation and tracking visualization, workflow settings, and execution control. b, Fiji plugin interface guiding users through workflow selection, input specification, and advanced settings. It also provides execution control and displays Ultrack server logs. c, Example of image data overlaid with cell detections and tracks as viewed in the Fiji interface. d, A concise Python API example (18 lines) demonstrating data loading, configuration, tracking execution, and result visualization in napari. e, Sample SLURM submission script for Ultrack's CLI, illustrating its integration with high-performance computing environments for distributed processing. Using multiple segmentation hypotheses with temporal consistency helps reduce the challenges of parameter tuning. Enhanced multicolor cell tracking by integrating segmentation results from multiple algorithms. Improved cell tracking in label-free imaging by leveraging virtual staining. Enhancing tracking accuracy through temporal registration. Sparse fluorescence labeling enables high-fidelity tracking validation over extended time-lapse recordings. Multi-terabyte cell tracking of zebrafish embryo. Near-perfect nuclear-based and membrane-based 3D tracking of zebrafish neuromast cells. Ultrack's guided user interfaces across multiple platforms facilitate cell tracking, catering to diverse user needs and computational environments. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Bragantini, J., Theodoro, I., Zhao, X. et al. Ultrack: pushing the limits of cell tracking across biological scales. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03891-5'>Interplay of genetic predisposition, plasma metabolome and Mediterranean diet in dementia risk and cognitive function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 09:03:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Alzheimer's disease (AD) and AD-related dementias (AD/ADRD) have a substantial genetic basis, with APOE4 homozygotes increasingly recognized as a distinct genetic subtype. To identify genotype-specific metabolic pathways and modifiable risk factors, we integrated genetic, plasma metabolomic and dietary data from 4,215 women and 1,490 men in prospective cohorts. Here we show that the associations of 57 metabolites with dementia risk varied by APOE4 genotype or other AD/ADRD risk variants. For example, cholesteryl esters and sphingomyelins were most strongly associated with increased dementia risk in APOE4 homozygotes, whereas inverse associations with glycerides were specific to this genotype. Dimethylguanidino-valeric acid was more strongly associated with dementia risk among carriers of the rs2154481-C allele (APP). Adherence to the Mediterranean diet more effectively modulated dementia-related metabolites in APOE4 homozygotes, suggesting targeted prevention strategies. Incorporating metabolomic data modestly improved dementia risk prediction, particularly during early follow-up. Mendelian randomization analysis identified 19 putative causal relationships between metabolites and cognitive outcomes, including protective effects of 4-guanidinobutanoate, carotenoids and N6-carbamoylthreonyladenosine. These findings reveal genotype-dependent metabolic profiles of cognitive health and support precision nutrition approaches for ADRD prevention. Alzheimer's disease (AD) and AD-related dementias (AD/ADRD) are neurodegenerative disorders characterized by progressive decline in memory, cognitive function and the ability to perform daily activities1. AD/ADRD has a substantial genetic basis, with heritability estimated at up to 80% from twin studies2. The apolipoprotein E (APOE) gene is the strongest genetic risk factor for sporadic AD3; carrying 1 APOE-ε4 (APOE4) allele increases risk 3–4-fold and 2 alleles increase risk 8–12-fold compared to the common APOE-ε3 allele4. APOE4 exacerbates amyloid-β (Aβ) pathology in the brain5 and is strongly linked to dysregulation in lipid metabolism and impaired cerebral glucose metabolism6, highlighting its multifaceted role in the pathology of ADRD. Recent studies further suggest that APOE4 homozygotes exhibit unique clinical, pathological and biomarker changes that begin at younger ages7. Beyond APOE, genome-wide association studies (GWASs) have identified common genetic variants at >80 loci associated with AD/ADRD risk, including ABCA7, BIN1 and CR1, confirming its polygenic basis and implicating key pathogenic pathways such as immune response and endocytosis8,9. However, previous research has predominantly focused on the genetic effects of a limited set of biomarkers that primarily capture Aβ and tau pathologies, often assessed during the prodromal stages of the disease or with limited duration of prospective follow-up. There is a notable lack of evidence about the impact of genetic factors on early stage biomarkers derived from high-throughput omics approaches, such as metabolomics, and their associations with ADRD risk. Furthermore, the potential to leverage identified gene–biomarker interactions to develop individualized prevention and treatment strategies remains largely unexplored. Genetic variations, particularly in genes involved in enzyme and transporter functions, directly influence metabolite production, degradation and circulation11. Environmental factors, especially diet, interact with genetics to further shape the metabolome by introducing exogenous metabolites and modulating key metabolic processes, such as inflammation, energy production and oxidative stress12,13. Metabolomic profiles across various tissues, including plasma14, brain15 and cerebrospinal fluid16, have shown associations with ADRD risk and cognitive function. Emerging evidence suggests that the APOE4 genotype may modulate associations between plasma metabolites and dementia risk17,18,19,20. For example, recent findings indicate that females carrying the APOE4 allele have distinct metabolomic profiles, reflecting alterations in lipid and amino acid metabolism, potentially increasing susceptibility to AD17,19. Despite these findings, there is a lack of data from large prospective studies investigating the extent to which associations between the metabolome and cognitive outcomes vary by genetic background. In addition, it remains unclear whether certain modifiable risk factors, such as diet, can mitigate ADRD risk and cognitive decline by targeting specific metabolic pathways across different genetic risk groups. To examine the interplay of genetics, the plasma metabolome and diet in relation to dementia risk and cognitive function, we conducted a prospective analysis of 4,215 women during a 34-year follow-up in the Nurses' Health Study (NHS). Notably, we observed widespread variations in associations between metabolites and cognitive outcomes across genotypes, most notably among APOE4 homozygotes. Associations of the Mediterranean diet (MedDiet) with metabolites and dementia risk were also genotype dependent, with metabolites mediating the MedDiet–dementia risk association only among APOE4 carriers. Two-sample Mendelian randomization (MR) further supported causal relationships between plasma metabolites and cognitive outcomes. To our knowledge, this is one of the first studies to demonstrate genotype-dependent associations between metabolites and ADRD risk, with additional findings that inform individualized dietary approaches for ADRD prevention. We prospectively followed 4,215 women in the NHS from 1989 to 2023 (mean age, at baseline, 57 years; Fig. 1a and Supplementary Table 1), during which 485 participants developed dementia. In addition, we longitudinally assessed objective cognitive function using a telephone-based battery, including the Telephone Interview for Cognitive Status (TICS), in a subset of 1,037 participants (1995–2008). In the replication analyses in the HPFS, 1,490 men (mean age 63 years at baseline) were prospectively followed from 1993 to 2023, with 121 dementia cases documented (Extended Data Fig. a, Prospective follow-up of 4,215 women in the NHS from 1989 to 2023. Genetic and metabolomic profiles were generated from blood samples collected at baseline. Detailed demographic, lifestyle, dietary, medical history and medication use data were collected via questionnaires. In addition, a telephone-based neuropsychological assessment battery was administered longitudinally from 1995 to 2008 to assess cognitive function in a subset of 1,037 participants. A total of 1,490 men from the HPFS were included as a replication cohort (Extended Data Fig. b, Distribution of plasma metabolites (n = 401). The inner circle displays the mean relative abundance of each metabolite, shown as a gradient in blue. The innermost circle color codes represent the different HMDB superclasses defined based on chemical structural similarities. c, Overall genetic structure associated with individual metabolites. Each dot represents an individual and is colored by APOE4 genotype, showing no clear pattern between the overall population substructure and APOE4 genotype. The metabolites with the highest Pearson's correlations with the top two genetic PCs from each metabolite superclass are included on the plot as arrows, colored by their superclass (see legend for b). Consistent with the curves, unadjusted hazard ratios (HRs) were estimated using Cox proportional hazards (PH) model; covariate-adjusted HRs with 95% confidence intervals (CIs) are provided in Supplementary Table 4. Person time was accrued from baseline until the earliest occurrence of an incident dementia case, dementia death or the end of follow-up. e, A wide range of adherence to the MedDiet, as assessed by a dietary index and intake levels of food and nutrient components of MedDiet. All analyses and distributions were based on data from 4,215 NHS participants. All statistical tests were two sided. Metabolomic data were generated from plasma samples using a liquid chromatography–mass spectrometry (LC–MS)-based platform in both the NHS and the HPFS; a total of 401 metabolites from 10 Human Metabolome Database (HMDB) superclasses were included in the NHS analyses after quality control (QC), with 254 of the metabolites available in the HPFS (Fig. Genotyping data were generated from blood samples, followed by QC and imputation (Methods). We extracted the two APOE variants along with 73 other common variants identified from AD/ADRD GWASs9,21 and calculated two polygenic risk scores (PRSs) for ADRD; one included the two APOE variants and one excluded the APOE region, using weights from published studies9,21. We first investigated the global influence of genetic variations on the plasma metabolome, using principal components (PCs) to capture genetic structure. Strong correlations were observed between genetic PC1 or PC2 and metabolites previously linked to ADRD risk, such as trimethylamine N-oxide22 and lithocholate23 (Fig. As expected, APOE4 homozygosity was broadly associated with elevated lipid metabolites compared to noncarriers (Extended Data Fig. To validate the dementia outcome, we examined plasma phosphorylated tau 217 (p-tau217), an established biomarker for early AD diagnosis24, in 103 NHS participants and found an approximately 3-fold higher dementia risk comparing the highest and lowest quartiles of p-tau217. We further confirmed that carrying APOE4 alleles or having a higher PRS of ADRD was associated with significantly increased risk of dementia and poorer cognitive function in both cohorts (Fig. We collected long-term dietary data using extensively validated semiquantitative food frequency questionnaires (SFFQs) in both cohorts. To assess dietary quality, we employed the MedDiet index, the only dietary pattern causally linked to delayed cognitive decline in a long-term, randomized controlled trial25. A widespread distribution of MedDiet adherence was observed (Fig. 1e), with higher MedDiet index scores associated with older age, lower body mass index, higher education level and more physical activity (Supplementary Tables 2 and 5). We identified 49 significant interactions of metabolites with APOE4 genotypes in relation to dementia risk at a false discovery rate (FDR) < 0.05 (Fig. All significant interactions were specific to APOE4 homozygotes, aligning with recent findings on this isoform in AD pathology7 and suggesting that it exhibits a distinct plasma metabolomic profile associated with ADRD risk, evident even decades before disease onset. a, Significant variation in the association between metabolites and dementia risk across different genotypes. increment in metabolite levels among individuals with different genetic predispositions, as defined by APOE4 genotype or PRS of ADRD (including the APOE variants), estimated using Cox PH model. Only metabolites with FDR < 0.05 for their interaction terms with genotype are displayed in the heatmaps. In all heatmaps, associations or interactions with FDR < 0.05 are indicated by double asterisks (**) and those with FDR < 0.25 by a single asterisk (*). Metabolites are grouped according to HMDB superclasses. The analyses were conducted among 4,215 NHS participants. b, Gene–metabolite interactions related to dementia risk widely distributed across metabolite superclasses and genotypes. The Manhattan plot displays metabolome-wide interaction results, represented by −log10(FDR) values for interaction terms from Cox PH models. Each dot represents a metabolite colored by the direction of interaction and grouped by HMDB superclass. Top, for the APOE4 genotype, the data point with the lower FDR between heterozygous (diamond) and homozygous (square) APOE4 interactions included for each metabolite. Bottom, for other common AD/ADRD variants, the most significant interaction across all 73 variants shown for each metabolite. The analyses were conducted among 4,215 NHS participants. c, Selected associations between metabolites and dementia risk with FDR for interaction <0.05, stratified by genotype. The first row presents stratified HRs and 95% CIs for dementia risk per 1-s.d. increment in metabolite level, categorized by the APOE4 genotype, with FDR values for interaction terms between APOE4 heterozygosity and homozygosity annotated (using the noncarrier as the reference group). The second row displays stratified results (HR and 95% CIs per 1-s.d. increment in metabolite level) by AD/ADRD variants, with FDR values for interaction terms with the variant effect allele dosage annotated. Genotype groups were defined based on rounded allele dosages. Results for the rs1800978 GG genotype group are excluded due to data sparsity. The analyses were conducted across 4,215 NHS participants. d, Consistency of metabolite–APOE4 interaction results across models with dementia risk and cognitive function as dependent variables. Each dot represents a metabolite with significant APOE4 interactions, colored by the HMDB superclass. Pearson's correlation coefficients in the β coefficients for interaction terms between metabolites and APOE4 carrier status estimated from Cox PH models, with dementia as the dependent variable, and from generalized linear models, with cognitive function scores as the dependent variable, are annotated on each figure. APOE4 carriers were not further divided into heterozygotes and homozygotes due to data sparsity among homozygotes with non-missing values for each metabolite in the cognitive function subset. Dementia risk analyses were conducted among 4,215 NHS participants and cognitive function analyses among 1,037 NHS participants. All statistical tests were two sided. DAG, diacylglycerol; PC, phosphatidylcholine; PE, phosphatidylethanolamine; TAG, triacylglycerol. Subgroup analyses further confirmed distinct association patterns between metabolites and dementia risk among APOE4 homozygotes compared to others, with variations in both the magnitude and, in some cases, the direction of associations (Fig. For example, a significant positive association between betaine and dementia risk was observed only among APOE4 homozygotes (Fig. Plasma betaine levels reflect both dietary intake and one-carbon metabolism activity26. The protective, albeit nonsignificant, association observed among APOE4 noncarriers and heterozygotes may primarily reflect dietary betaine's benefit for cognitive health27, whereas, in APOE4 homozygotes, elevated betaine may indicate methylation imbalance and metabolic dysregulation, contributing to an increased dementia risk26,28. We observed broadly positive associations across cholesteryl esters (CEs), sphingomyelins (SMs) and dementia risk in all genetic risk groups, with the strongest associations in APOE4 homozygotes. The APOE4 allele promotes the accumulation of cholesterol and CEs in the brain29, activating inflammatory pathways that exacerbate neuronal damage and contributing to the formation of amyloid plaques and tau tangles, thereby increasing dementia risk30. We observed inverse associations between glycerides and dementia risk, specifically among APOE4 homozygotes, where elevated glyceride levels likely reflect this reduced delipidation, which may, in turn, limit the aggregation of apolipoprotein E (ApoE) and the formation of amyloid plaques31. Replication of the findings for dementia risk in objective cognitive function yielded broadly consistent results (Pearson's r ranging from −0.78 to −0.63 for the interaction effect estimates; Fig. Sensitivity analyses, excluding family history of dementia from the covariates or modeling dementia case and death separately, showed similar results (Supplementary Figs. Independent replication of the NHS findings in the HPFS yielded broadly consistent results (Pearson's r = 0.40), with 32 out of 38 significant interactions (84.2%) in the same direction. Notably, 4 interactions reached P < 0.05 and 10 reached P < 0.10, including inverse interactions with glycerides, although some variation between women and men remained (Extended Data Fig. Although APOE4 is the major contributor to the genetic risk of dementia, recent GWASs have identified many other common variants linked to AD/ADRD risk8,9. We examined how these risk variants, either aggregated into PRSs or as individual variants, may modify the associations between metabolites and dementia risk (Supplementary Tables 11 and 12). Although no significant interaction between PRSs and metabolites was detected after multiple testing correction (Fig. 2a and Supplementary Tables 13 and 14), we identified eight significant interactions between individual AD/ADRD variants, including those mapped to ABCA1, APP, ADAMTS1, CTSH and USP6NL, and metabolites in relation to dementia risk with an FDR < 0.05 (Fig. We observed that the positive association between 1-methylhistamine, a metabolite involved in immune and inflammatory responses in the brain32 (Fig. 2c), and dementia risk was significantly more pronounced in individuals carrying the rs1800978-G allele mapped to ABCA1, a gene that plays a crucial role in clearing Aβ peptide from the brain33. Dimethylguanidino-valeric acid, a metabolite associated with impaired fatty acid and amino acid metabolism, showed a stronger positive association with dementia risk among individuals carrying the C allele at rs2154481, a variant mapped to the APP gene that encodes Aβ precursor protein (APP), suggesting that dysregulated lipid and glucose metabolism may influence the processing of APP, leading to increased Aβ production34. In addition, lipid metabolites showed significant interactions with AD/ADRD genetic variants. For example, C32:2 phosphatidylcholine exhibited a positive interaction with an ADAMTS1-linked variant (rs2830489-T), implicating extracellular matrix remodeling and neuroinflammation in neurodegeneration35 (Supplementary Text). A key distinction between metabolomics and genetics is that metabolites can be modified by exogenous factors and may serve as targets for intervention; in particular, diet significantly impacts the metabolome36. We thus examined whether diet, specifically the MedDiet, which has been implicated in cognitive health25, could modulate metabolite levels in individuals with different genetic predispositions to AD/ADRD. We found that individuals with greater adherence to the MedDiet had a significantly lower risk of dementia and better cognitive function (Fig. Notably, these protective associations for dementia risk were more pronounced among APOE4 homozygotes compared to noncarriers and heterozygotes (Fig. 3b), although no clear trend was observed when stratifying by ADRD PRS (Extended Data Fig. Next, to evaluate MedDiet's impact on the overall metabolomic profile, we used a random Forest (RF) classifier to distinguish individuals with high versus low MedDiet adherence based on plasma metabolite levels. This classifier demonstrated excellent performance, with an area under the receiver operating characteristic (ROC) curve (AUC) of 0.76 in the NHS (Fig. 3c) and 0.72 when replicating in the HPFS (Extended Data Fig. In addition, individual components of the MedDiet, such as nuts, fruit and monounsaturated fats, were strongly associated with overall metabolomic patterns, as captured by the top two metabolite PCs (Extended Data Fig. a, Higher adherence to the MedDiet prospectively associated with a lower risk of dementia and enhanced cognitive performance, as assessed by the telephone-based neuropsychological assessment battery (TICS). For dementia risk analysis, a restricted cubic spline Cox PH model estimated HRs and 95% CIs across varying levels of the MedDiet index, using 0 as the reference. For the TICS score analysis, a generalized linear model estimated the adjusted TICS score and corresponding 95% CI across MedDiet index levels, with P values annotated. The analyses were conducted among NHS participants with cognitive and dietary data (n = 86,740 for dementia analysis and n = 16,244 for cognitive function analysis). b, The protective association between adherence to the MedDiet and risk of dementia most pronounced among APOE4 homozygotes. Stratified HR and 95% CIs for dementia risk per a 1-unit increment in the MedDiet index score, categorized by the APOE4 genotype, were estimated from Cox PH models, with stratified P values annotated (unadjusted for multiple comparisons in the hypothesis-driven analysis). The analyses were conducted among NHS participants with genetic, dietary and dementia outcome data (n = 16,497). c, Strong association between adherence to MedDiet and the overall plasma metabolome from an RF model to classify individuals in the top versus the bottom quartile of the MedDiet index based on plasma metabolites. For the RF classification, the dataset was randomly divided into training (60%) and test (40%) sets. The analyses were conducted among 4,215 NHS participants. d, Associations between MedDiet adherence and plasma metabolite levels differing by APOE4 genotype. The heatmap shows β coefficients representing a 1-s.d. increment in the MedDiet index from a generalized linear model, with plasma metabolite levels as the dependent variable. The analyses were conducted among 4,215 NHS participants. e, Select associations between the MedDiet index and plasma metabolite levels with P < 0.05 from the likelihood ratio test for the interaction between APOE4 genotype and MedDiet index in relation to metabolites, using a generalized linear model stratified by APOE4 genotype. Covariate-adjusted residuals of metabolites are shown along with fitted linear regression lines, 95% CIs and P values for interaction. These results were not adjusted for multiple testing. The analyses were conducted across 4,215 NHS participants. All statistical tests were two sided. To assess whether the metabolic response to MedDiet is genotype dependent, particularly in relation to APOE4, we examined the association between MedDiet adherence and metabolite levels across APOE4 genotypes. As anticipated, the overall association patterns differed among APOE4 homozygotes and heterozygotes compared to noncarriers (Fig. Similar patterns were observed in the HPFS (Extended Data Fig. Consistent with findings from the PREDIMED trial involving a randomized MedDiet37, greater adherence to the MedDiet was associated with higher levels of unsaturated glycerides and lower levels of saturated glycerides, lipid patterns potentially beneficial for cognitive health, as well as increased levels of established neuroprotective compounds, including piperine, betaine and pantothenic acid38,39 (Extended Data Fig. Among the metabolites showing suggestive APOE4–MedDiet interactions (P < 0.05), we observed an inverse association between MedDiet and asparagine levels exclusively among APOE4 homozygotes (Fig. This may reflect MedDiet-driven changes in amino acid metabolism, including glutaminolysis and the tricarboxylic acid cycle, which are key to asparagine regulation. Given asparagine's role in protein and nucleotide synthesis, its reduction may signal broader metabolic benefits specific to APOE4 homozygotes40. We also identified nominally significant interactions between MedDiet and the APOE4 genotype in relation to 1,7-dimethyluric acid, a derivative of caffeine metabolism with established antioxidant properties and potential neuroprotective effects41 (Supplementary Text). Furthermore, we found that 39.5% of the association between MedDiet adherence and dementia risk was mediated by a set of metabolites among APOE4 carriers (P = 0.05), whereas no mediation effect was observed among noncarriers or in the full dataset (Methods). Besides the APOE4 genotype, broad interactions were also observed for ADRD PRS and individual variants (Supplementary Tables 19–21). These findings collectively indicated that the MedDiet's potential to modulate cognitive health-related metabolites varied by APOE4 genotype, suggesting that this dietary pattern could be an effective strategy to delay dementia onset in APOE4 homozygotes, despite their higher risk profile. Given that genetics, metabolites and MedDiet adherence are all linked to dementia risk, we further examined how incorporating these factors could enhance the prediction of cognitive outcomes. Compared to a baseline model, which included age, family history of dementia, education level, smoking status, history of depression or regular antidepressant drug use and the MedDiet index, adding APOE4 and ADRD PRS to Cox model moderately improved the performance for predicting dementia risk (Fig. This also demonstrated that genetic factors capture additional information beyond family history of dementia and that the ADRD PRS adds modest but incremental predictive value beyond APOE4. Adding metabolites predictive of dementia risk to the model further improved the time-specific model performance, indicating that metabolites provide additional predictive value beyond the MedDiet and other major dementia risk factors (average AUC = 0.78; Supplementary Table 22). As expected, these baseline characteristics were better at predicting short-term dementia risk, that is, 15-year risk, compared to long-term risk, which may be attributed to the inherent within-individual variability of metabolomic measurements over longer follow-up periods, potentially introducing random measurement error. Similar patterns were observed for Harrell's C-index (Extended Data Fig. The performance of different models in APOE4 subgroups did not substantially deviate, likely due to the limited sample size and number of cases within these subgroups, which may have introduced substantial instability. Future studies with larger sample sizes are warranted to more reliably evaluate prediction performance within APOE4 subgroups. a, The inclusion of genetic factors improving dementia risk prediction using Cox PH model, with an additional modest enhancement when plasma metabolites also included. Time-dependent ROC curve analyses were conducted for dementia risk over both the entire follow-up period and the first 15 years of follow-up. The baseline model predictors included age, family history of dementia, education level, smoking status, history of depression or regular antidepressant use and MedDiet index. The PRS of ADRD excluded variants in the APOE region (see Methods for selection of metabolite predictors). b, Plasma metabolites among the top contributors for predicting dementia risk as quantified by the SHAP value. Feature contributions were evaluated for Cox PH model to predict overall and 15-year dementia risk, including the full list of predictors. SHAP values were calculated for each category of predictors by summing the SHAP value of all predictors in that category. c, Integration of genetic and metabolomic data enhancing cognitive status prediction within APOE4 subgroups. In subgroup analyses by APOE4 carrier status, APOE4 genotype was excluded as a predictor. For all analyses, the NHS dataset (n = 4,215) was randomly divided into training (60%) and test (40%) sets; models were fitted on the training set and evaluated on the test set. All results shown are from the test set. We employed the SHapley Additive exPlanation (SHAP) values to quantify the contributions of individual predictors to dementia risk predictions. As anticipated, age, APOE4 and ADRD PRS were among the top contributors (Fig. Plasma metabolites meaningfully contributed to the prediction throughout the entire follow-up period, with their overall contribution ranking just below age and genetic factors. In contrast, for short-term risk prediction, MedDiet emerged as a major predictor. The relatively modest contribution of MedDiet to overall follow-up predictions likely reflects the use of baseline dietary data only, which does not account for changes in dietary behavior over time. We observed similar patterns in the HPFS (Extended Data Fig. 10a,b) and when predicting a dichotomized TICS score (highest versus lowest tertile) within APOE4 subgroups, whereas no improvement was observed in the full dataset (Fig. The unstable results for cognitive score prediction likely reflect limited sample size in the subset. Larger studies are needed to further assess the predictive utility of these models for cognitive function. Last, genetics can offer mechanistic insights by using variants as instrumental variables to test whether associations between metabolites and cognitive outcomes are causal using the MR approach. To maximize the statistical power and mitigate biases, we implemented a two-sample MR design leveraging data from published GWASs and selected genetic instruments for 657 metabolites and 133 ratios of metabolite pairs sharing an enzyme or transporter (reflecting metabolic flux; Supplementary Tables 24 and 25)11. a, Schematic of the two-sample MR and colocalization analyses. Genetic instruments were selected for 657 metabolites and 133 metabolite ratios from a published GWAS. Summary statistics for overall dementia, AD, vascular dementia and cognitive performance were also obtained from published GWASs (Methods). Two-sample MR was performed to identify putative causal relationships between metabolites or ratios and cognitive outcomes, followed by colocalization analysis for the causal associations with an FDR < 0.05. b, Identification of numerous putative causal interrelationships of various metabolites, metabolite ratios and cognitive outcome using genetic instruments. The chord diagram displays causal relationships with an FDR < 0.05 from MR analyses using Wald ratio, inverse variance-weighted or MR Egger methods, where each arc represents an identified link between a metabolite or ratio and a cognitive outcome. Arcs and nodes are color coded by the HMDB superclasses. c, Colocalization analyses strengthening evidence of causality, suggesting that the identified putative causal relationships between metabolites or ratios and cognitive outcomes have potential shared causal variants and biology. Putative causal relationships, represented by the odds ratios (ORs) or β coefficients with 95% CIs, are shown for associations with FDR < 0.05 and colocalization signals. Bayesian colocalization analysis was performed within the ±500-kb region around a genetic instrument (Methods). If multiple instruments were used, a causal association was reported if the metabolite or ratio and cognitive outcomes colocalized at least one genetic locus. Colocalization signals were reported for a locus if the conditional probability of colocalization, PP.H4/(PP.H3 + PP.H4), was >70%, where PP.H3 is posterior probability that the two traits have independent causal variants and PP.H4 is the posterior probability that the two traits share a single causal variant. d, Regional genetic association plots providing evidence of potential shared causal variants affecting both metabolites or ratios and cognitive outcomes at specific genetic loci. The plots display genetic association results for metabolites or ratios and cognitive outcomes at three colocalized loci with PP.H4/(PP.H3 + PP.H4) > 70%. Each plot is annotated with the genetic instrument and dots are color coded according to their linkage disequilibrium with the instrumental variant. The −log10(P) values for both metabolites or ratios and cognitive outcomes were obtained from the original GWASs. The sample sizes for the original GWASs are as follows: metabolites or ratios (n = 8,299), cognitive performance (n = 257,841), dementia (5,933 cases and 166,584 controls), AD (90,338 cases and 1,036,225 controls) and vascular dementia (881 cases and 211,508 controls) (Methods). All statistical tests were two sided. We identified 99 significant causal relationships involving 95 metabolites or ratios across all 4 outcomes at an FDR < 0.05 (Fig. A colocalization analysis was further performed to prioritize metabolite–outcome pairs with potential shared genetic basis and biology; we identified 19 pairs with a conditional posterior probability of colocalization >70% under a single causal variant assumption (Fig. A notable finding was the protective causal effect of 4-guanidinobutanoate (4-GBA) on dementia, supported by colocalization signals (Fig. 5d); 4-GBA is a gamma-aminobutyric acid (GABA)-related metabolite involved in inhibitory neurotransmission42 and may counteract excitotoxicity, a known contributor to dementia pathogenesis43. Evidence suggests that agmatine may have therapeutic potential for AD by modulating Aβ production, aggregation and clearance45. Our findings highlight potential shared pathways involving 4-GBA, GABA and agmatine in dementia pathology. Carotene diol (1) and (2), naturally occurring carotenoids with potent antioxidant properties46, showed protective effects against AD, consistent with randomized controlled trial evidence causally linking carotenoid intake to reduced cognitive decline47. Building on studies showing altered glutamine levels in the brain and cerebrospinal fluid of individuals with AD48, our findings also highlighted glutamine's potentially neuroprotective role in AD (Supplementary Fig. For cognitive function, N6-carbamoylthreonyladenosine exhibited a strong beneficial effect, with colocalization signals allowing for multiple causal variants (Methods and Supplementary Table 28). A number of causal relationships were identified between metabolites involved in ATP metabolism, suggesting a causal role for the interplay between disrupted energy metabolism and oxidative stress in the pathophysiology of cognitive decline (Supplementary Text). Future investigations into these pathways could inform targeted therapeutic strategies. Elucidating the pathogenic mechanisms underlying ADRD and identifying modifiable factors capable of targeting these mechanisms are critical for prevention. Given the strong genetic basis of ADRD, accounting for genetic predisposition is essential when investigating these mechanisms. In this study, we leveraged up to 34 years of follow-up and multiomics data from the NHS, with independent replication in the HPFS, to comprehensively examine gene–metabolite interactions in relation to ADRD risk. We identified distinct plasma metabolomic profiles of dementia risk among APOE4 homozygotes. Beyond APOE4, we found that the metabolite–dementia associations are also modified by other common AD/ADRD risk variants implicated in cholesterol homeostasis, mitochondrial function, APP production and extracellular matrix integrity. Notably, adherence to the MedDiet contributed to cognitive health in an APOE4-dependent manner, potentially mediated by metabolites. We further demonstrated that incorporating metabolites into predictive models alongside traditional and genetic predictors improved the prediction performance. Our two-sample MR analysis further supported the causal effects of specific metabolites, such as 4-guanidinobutanoate, carotene diol and glutamine, on cognitive outcomes. ADRD has long been considered a metabolic disease, largely due to the central role of APOE4 in lipid transport and metabolism6. Emerging evidence indicates that the metabolomic contributions to ADRD risk are modulated by APOE4 status17,18,19,20. A recent study further highlights that APOE4 homozygosity represents a distinct genetic subtype which exhibits significantly higher levels of AD pathological markers at a younger age7. Our study extends those findings, revealing that APOE4 not only distorts lipid metabolism but also affects other pathways, including betaine and the urea cycle (for example, citrulline) metabolism. These findings suggest a broader role for APOE4, implicating its involvement in neuroinflammation, gut-derived choline metabolism and neurotoxic pathways that elevate ADRD risk5,49. Furthermore, our findings, aligned with the recent study7, indicate that APOE4 homozygosity uniquely alters the plasma metabolome and its associations with ADRD risk, underscoring the necessity of distinguishing homozygotes from heterozygotes in investigations of APOE4's role in AD pathology. Although APOE4 homozygotes exhibit a high penetrance of AD biology, not all individuals progress to clinical AD6. Our findings highlight the potential for preventive strategies targeting specific metabolic pathways in this high-risk group. Beyond APOE4, examination of other common risk variants identified additional findings pointing to broader pathogenic mechanisms. Consistent with the results from the PREDIMED trial25, we found that long-term adherence to the MedDiet was strongly associated with lower ADRD risk. We further examined whether this association varied by genetic predisposition. A key distinction of our study, compared to previous studies with mixed findings50,51,52, is that we went beyond the binary classification of APOE4 carrier status and found particularly strong associations only among APOE4 homozygotes. The strongest association in APOE4 homozygotes may reflect more effective modulation of ADRD-related metabolic profiles by the MedDiet in this genotype. Thus, our study not only identifies mechanistic insights, but also proposes actionable prevention strategies targeting these pathways. This has important implications for public health messaging, highlighting the overall benefit of adhering to the MedDiet for ADRD prevention, as well as the potential for targeted interventions in genetically vulnerable populations. In addition, our findings underscore the value of metabolites in predicting ADRD risk decades before disease onset. Last, our two-sample MR analysis corroborated established mechanisms in ADRD pathology, such as the role of GABAergic signaling, reinforcing causality derived from mechanistic studies and randomized controlled trials, including evidence of carotenoids' neuroprotective effects. We also identified new causal biomarkers, including metabolites involved in glutamine, urate and hypotaurine metabolism (Supplementary Text). It was conducted in well-educated individuals of European ancestry, which may limit generalizability but enhances internal validity. Ascertainment bias is possible given the exclusion of participants with dementia at baseline, although the baseline age was relatively young and APOE4 homozygotes did not differ significantly in health status, making major bias unlikely. Dementia outcomes were based on self-reported physician diagnoses and death records, which may introduce misclassification, but validity is supported by participants' professional backgrounds, strong associations with known genetic factors and p-tau217 and consistency across data sources. Although omics-based models are not yet widely implemented clinically, emerging tools such as PRSs and aging clocks show translational promise (Supplementary Text). Despite these limitations, our large, prospective design enabled robust examination of gene–metabolite–diet interactions over decades of follow-up. Importantly, key findings in women were independently validated in men, supporting generalizability across sexes, particularly relevant given the sex-based differences in both metabolite profiles53 and dementia risk54. In conclusion, our study highlights the substantial influence of genetic variants, particularly APOE4 homozygosity, on plasma metabolites and their associations with ADRD risk. These genetic effects are widespread across the plasma metabolome and our findings identify the MedDiet as a promising approach to mitigate genetically dependent ADRD risk by targeting a broad spectrum of metabolic pathways. Moreover, this work provides a uniquely valuable resource for advancing the early prediction of ADRD risk through emerging omics-based biomarkers and identifying causal mechanisms underlying ADRD risk, which hold potential as prevention targets and druggable pathways. This study included de-identified data from participants who had consented to the use of their anonymized information for research purposes. Participants were not financially compensated for their participation. Chan School of Public Health (institutional review board protocol nos. The NHS is a prospective cohort study that enrolled 121,700 US female registered nurses aged 30–55 years in 197655. The participants were followed biennially to collect information on diet, lifestyle, medication use and newly diagnosed diseases through mailed self-administered questionnaires. We included 4,215 women who were aged <75 years and free from dementia, Parkinson's disease, stroke and cancer at baseline when the blood samples were collected (1989–1992); the blood was assayed for genetics and metabolomics, as described in detail below. The HPFS is a prospective cohort study that enrolled 51,529 US male health professionals aged 40–75 years in 1986. We included 1,490 men who met the same inclusion and exclusion criteria at baseline (1993–1996), when the blood samples were collected, as a replication cohort. The participants were followed from the baseline to 2023 for a composite dementia endpoint, which included self-reported dementia and deaths due to dementia. Participants self-reported a physician diagnosis of AD or other forms of dementia (ADRD) every 2 years via questionnaires. Deaths were identified through state vital statistics records, the National Death Index, family reports and the postal system. A study physician reviewed medical records and death certificates to determine whether dementia was listed as the primary or contributing cause of death. Our system for death ascertainment and death cause adjudication is validated and >98% of deaths were identified56. Objective cognitive function was assessed through a telephone cognitive interview in a subset of 1,037 NHS participants aged ≥70 years, with four telephone interviews conducted between 1995 and 2008. Five other tests were added later, including (1) immediate recall of the East Boston Memory Test (EBMT), (2) delayed recall of the EBMT, (3) delayed recall of the TICS ten-word list, (4) a test of verbal fluency and (5) the digit span backward test. We assessed three cognitive measures, including the TICS score and the composite scores for global cognition and verbal memory. Global cognition included all six tests. Verbal memory included four tests: immediate recall of the ten-word list in the TICS, delayed recall of the TICS ten-word list and immediate and delayed recalls of the EBMT. To calculate the composite scores for global cognition and verbal memory, we first calculated z-scores for each test at each measurement, based on the baseline mean and the s.d. We then averaged these z-scores across all relevant tests at each measurement and calculated the overall mean value across all measurements. Blood samples were genotyped using one of six genotyping platforms. We restricted the samples to inferred European ancestry based on genetic PCs. Post-QC data were imputed using the 1000 Genomes Phase 3 v5 reference panel. APOE4 genotype was determined using two SNPs: rs429358 and rs7412. All other combinations of alleles were considered APOE4 noncarriers. We calculated PRSs for ADRD using weights from two published studies9,21,59. The PRS developed by Bellenguez et al. (PGS002280) comprised 83 variants and excluded any variants from the APOE region9. The PRS developed by Zhang et al. (PGS000334) comprised 22 variants and included the two APOE variants21. Only variants available in the imputed genetic data were included in the PRS calculation (Supplementary Table 11). The PRS was calculated as a weighted sum of the effect allele dosage across all included variants for each individual and was standardized to have a mean of 0 and an s.d. of 1 within each genotyping platform to maximize comparability. We further extracted individual variants from these two PRSs, which were identified in previous GWASs of AD/ADRD. Duplicated variants and those with a minor allele frequency <0.01 were removed from the combined list, followed by linkage disequilibrium pruning (r2 < 0.1 with the 1000 Genomes European population as the linkage disequilibrium reference) using the SNPclip function in the LDlinkR package in R, leaving a total of 73 variants for subsequent analyses (Supplementary Table 12). Plasma metabolomic profiling was performed for nested case–control studies within the NHS and HPFS using high-throughput LC–MS techniques at the Broad Institute of MIT and Harvard (Cambridge, MA, USA). Additional details of metabolomic profiling are provided in Supplementary Text. For metabolites with <25% missing data in an individual study, missing values were imputed with half of the minimum measured value for that metabolite in that study; metabolites with <100 samples were removed. A probit transformation was applied to metabolites within each study to correct for batch effects, reduce the impact of skewed distributions and heavy tails on the results and scale the metabolite values to the same range. After merging the metabolite data with other types of data (n = 4,215 for the NHS and n = 1,490 for the HPFS), we excluded metabolites with ≥90% missing values among dementia cases with genetic data. We further excluded metabolites with an intraclass correlation coefficient <0.4 or coefficient of variation in the top 10 percentiles of all remaining metabolites. We created another set of metabolomics data for the NHS by selecting those with ≥100 samples and <25% missing values from the 401 metabolites in the final dataset, followed by RF imputation with 100 trees using the missRanger package in R; 237 metabolites were retained for subsequent analyses of the overall metabolomic profile that required no missing data. Dietary intakes were assessed using SFFQs. The MedDiet was assessed using the Alternate Mediterranean Diet Score, which was calculated based on nine components61. For vegetables, fruit, nuts, whole grains, legumes, fish and the ratio of monounsaturated to saturated fat, a score of 1 was given if the intake was at or above the SFFQ-specific median; otherwise, 0 was given. For red and processed meat consumption, a score of 1 was assigned if the intake was below the SFFQ-specific median; otherwise, a score of 0 was assigned. For alcohol intake, a score of 1 was assigned if the intake was between 5 and 15 g d−1; otherwise, a score of 0 was assigned. The scores of individual components were summed to obtain the overall MedDiet index, which ranged from 0 to 9. Pearson's correlation coefficients were calculated for each of the 401 metabolites with genetic PC1 and PC2. Correlation coefficients were then ranked from highest to lowest for PC1 and PC2, respectively. The overall rank was determined by summing the ranks of PC1 and PC2 and ranking them from lowest to highest. Pearson's correlation coefficients were calculated for the MedDiet index and its individual components (monounsaturated and saturated fat calculated separately) with metabolites PC1 and PC2 calculated from the 237 metabolites. The dataset was first randomly split into training (60%) and test (40%) sets. A dichotomized MedDiet index outcome was derived from the top and bottom quartiles of the continuous score. Cox PH models were fitted to assess the associations between each of the 401 metabolites and the time-to-event outcome of dementia among 4,215 women in the NHS using the coxph function in the survival package in R. Details of covariate assessment and adjustment are provided in Supplementary Text. The same approach and FDR threshold were used for all other analyses involving multiple testing corrections in this study, including all interaction tests. All statistical tests in this study were two sided. For the interaction analysis of metabolites and APOE4 status, interaction terms between APOE4 carrier status (or APOE4 heterozygote and homozygote) and the metabolite were added to the model, along with ADRD PRS (excluding the APOE region) and its interaction term with the metabolite. In addition, likelihood ratio tests were performed comparing the model with and without the APOE4 interaction terms. For the interaction analysis of metabolites with ADRD PRS or other AD/ADRD variants, the relevant gene–metabolite interaction terms (PRS or effect allele dosage) were added to the model, along with APOE4 heterozygote and homozygote. All interaction models were additionally adjusted for the top four genetic PCs and genotyping platforms. For APOE4 homozygotes, the models were adjusted for only continuous covariates, ADRD PRS and the top four genetic PCs due to the limited sample size. For analyses within ADRD PRS tertiles (defined using genetic PC-adjusted scores), no additional covariate was adjusted. Generalized linear models (Gaussian family) were fitted to assess the association between each objective cognitive function score and each of the 401 metabolites among a subset of 1,037 women with cognitive function measurements using the glm function of the stats package in R. The models were adjusted for the same covariates as in the dementia risk analysis. Interaction analysis was performed for APOE4 carrier status with the same additional covariates as in the dementia risk analysis. APOE4 carriers were not further stratified into heterozygotes and homozygotes due to data sparsity among homozygotes in this subset. FDR correction was applied to the interaction P value per cognitive outcome. Interaction analysis with other genetic factors or subgroup analysis was not performed due to the limited power. As a replication, interaction analysis of the 254 available metabolites was conducted in 1,490 men from the HPFS, with additional adjustment for profession. Due to data sparsity in non-missing values for each metabolite, only interactions with APOE4 carrier status were assessed. In the analysis of the associations of the MedDiet index score with dementia risk and objective cognitive function, we leveraged data from the full NHS cohort and excluded participants who had dementia, Parkinson's disease, stroke, cancer or missing components for the MedDiet index at baseline, which was 1980 for the dementia endpoint and 1994 for the objective cognitive function. For the analysis of the objective cognitive function, we further excluded participants aged <70 years at baseline. A total of 86,740 participants were included in the analysis of dementia risk and 16,244 participants in the analysis of objective cognitive function. Cox PH model was fitted to prospectively assess the association between the continuous MedDiet index and the time-to-event outcome of dementia risk. Details of covariate adjustment are provided in Supplementary Text. A cubic spline regression model was fitted to assess the nonlinear trend. Subgroup analyses were performed by APOE4 genotype and tertiles of ADRD PRSs. Subgroup analyses were performed by APOE4 genotype and tertiles of ADRD PRSs. The same Cox PH and cubic spline models were fitted, with additional adjustment for profession. Subgroup analyses were performed by APOE4 genotype. Subgroup analyses were performed by APOE4 genotype and tertiles of ADRD PRS (excluding APOE region). We conducted a mediation analysis to quantify the extent to which metabolites mediate the association between MedDiet adherence and dementia risk in the NHS. Metabolites associated with both dementia risk (P < 0.05) and MedDiet adherence (FDR < 0.05) were selected as candidate mediators, followed by orthogonal filtering using a correlation threshold of r < 0.5 to exclude highly correlated metabolites, resulting in seven metabolites: allantoin, C16:1 CE, C18:0 SM, 1-methylguanine, 1,7-dimethyluric acid, C34:5 phosphatidylcholine plasmalogen and piperine. Regression-based mediation analyses were then conducted by comparing a full model (including both MedDiet and selected metabolites) with a reduced model (excluding metabolites), with dementia risk as the dependent variable to quantify the proportion of the effect of MedDiet adherence on dementia risk, explained by these selected metabolites in the full dataset, as well as stratified by APOE4 carrier status. Cox PH models were built for predicting dementia risk among 4,215 women in the NHS. The dataset was first randomly split into training (60%) and test (40%) sets, including within APOE4 subgroups. APOE4 predictors were not included in the models for APOE4 subgroups. All models were built in the training set and evaluated on the test set. Time-dependent AUC was calculated based on an incident/dynamic version of sensitivity and specificity within distinct risk sets, defined as groups of participants who remained at risk for developing dementia at specified time points, which provides a dynamic view of model performance across the follow-up period62, using the risksetAUC function in the risksetROC package in R. Harrell's C-index63 was calculated to quantify the overall discriminative ability of the model by assessing how well the model can rank individuals by their risk of the event using the survival package in R. Feature contributions were quantified by the SHAP value64 for Cox PH model in predicting overall and 15-year dementia risk, including the full list of predictors using the fastshap package in R. SHAP values were calculated for each category of predictors by summing the SHAP value of all predictors in that category. As a replication, Cox PH models were built to predict dementia risk among 1,490 men in the HPFS. The same modeling approaches were applied, with profession included in the baseline model instead of education level. Due to data sparsity, prediction models were constructed only for the overall dementia risk in the full dataset. The 164 RF-imputed metabolites with no missing values were considered as candidate predictors and 5 metabolites were selected in the final model, following the same procedure used in the NHS (Supplementary Table 22). Two-sample MR analysis65 was conducted to assess the causal relationships between plasma metabolites and cognitive outcomes using published GWAS summary statistics. GWAS summary statistics for the exposures, including plasma metabolites and metabolite ratios, were obtained from ref. 11 and for the cognitive outcomes were obtained from GWASs of overall dementia66, AD8, vascular dementia66 and cognitive performance67. All GWAS populations were of European ancestry. In the original study11, GWASs were performed for 1,091 plasma metabolites and 309 metabolite ratios. In this study, we included all the variant–metabolite and variant–metabolite ratio pairs selected by the original study for MR analyses, with additional selection applied to metabolites and ratios not included in the original set (Supplementary Text). We used three MR methods to assess the causal effect of each metabolite or ratio on each cognitive outcome. The inverse variance-weighted method was used when there were at least two instruments. The MR Egger method was used when there were at least three instruments and potential pleiotropy was detected. MR analysis was performed using the mr function in the TwoSampleMR package in R. Details of the sensitivity analyses are described in Supplementary Text. For each pair, Bayesian colocalization analysis was performed in the ±500-kb region around each genetic instrument, restricting to variants with minor allele frequency <0.01, using the coloc.abf function with default prior probabilities in the coloc package in R. Colocalization signals were reported for a locus if the conditional probability of colocalization, PP.H4/(PP.H3 + PP.H4), was >70%, where PP.H3 is posterior probability that the two traits have independent causal variants and PP.H4 is the posterior probability that the two traits share a single causal variant. These colocalized exposure–outcome pairs were considered to have putative causal relationships. If there were instruments at multiple loci for a pair, the causal relationship was reported if the exposure and outcome were colocalized at at least one genetic locus. The coloc.abf function assumes that there is a single causal variant for each trait. We further conducted a sensitivity analysis that relaxed the single causal variant assumption using the coloc.susie function in the coloc package in R, which calculates posterior probabilities under the assumption of multiple causal variants. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. As informed consent was gained from the participants, all the individual-level data from the NHS and the HPFS are available through a request for external collaboration and upon approval of a letter of intent and a research proposal. Details on how to request external collaborations with the NHS can be found at https://nurseshealthstudy.org/researchers (contact principal investigator: A.H.E., email: nhahe@channing.harvard.edu) and with the HPFS at https://sites.sph.harvard.edu/hpfs/for-collaborators (contact principal investigator: L. Mucci, email: lmucci@hsph.harvard.edu). Source data are provided with this paper. Analysis-specific programs are publicly available via GitHub at https://github.com/DW-Group/Gene_Metabolites_MedDiet_Dementia. Emmady, P. D., Schoo, C. & Tadi, P. Major Neurocognitive Disorder (Dementia) (StatPearls Publishing, 2022). Gatz, M. et al. Role of genes and environments for explaining Alzheimer disease. Liu, C. C., Liu, C. C., Kanekiyo, T., Xu, H. & Bu, G. Apolipoprotein E and Alzheimer disease: risk, mechanisms and therapy. Yamazaki, Y., Zhao, N., Caulfield, T. R., Liu, C. C. & Bu, G. Apolipoprotein E and Alzheimer disease: pathobiology and targeting strategies. Parhizkar, S. & Holtzman, D. M. APOE mediated neuroinflammation and neurodegeneration in Alzheimer's disease. Jackson, R. J., Hyman, B. T. & Serrano-Pozo, A. Multifaceted roles of APOE in Alzheimer disease. Fortea, J. et al. APOE4 homozygozity represents a distinct genetic form of Alzheimer's disease. Wightman, D. P. et al. A genome-wide association study with 1,126,563 individuals identifies new risk loci for Alzheimer's disease. Bellenguez, C. et al. New insights into the genetic etiology of Alzheimer's disease and related dementias. Fuller, H. et al. Metabolomic epidemiology offers insights into disease aetiology. Chen, Y. et al. Genomic atlas of the plasma metabolome prioritizes metabolites implicated in human diseases. Influence of the microbiome, diet and genetics on inter-individual variation in the human plasma metabolome. & Tornero-Aguilera, J. F. Global impacts of western diet and its effects on metabolism and health: a narrative review. Niedzwiecki, M. M. et al. High-resolution metabolomic profiling of Alzheimer's disease in plasma. Brain and blood metabolome for Alzheimer's dementia: findings from a targeted metabolomics analysis. Sex and APOE epsilon4 genotype modify the Alzheimer's disease serum metabolome. Chang, R. et al. Predictive metabolic networks reveal sex- and APOE genotype-specific metabolic signatures and drivers for precision medicine in Alzheimer's disease. Gonzalez-Dominguez, R. et al. Apolipoprotein E and sex modulate fatty acid metabolism in a prospective observational study of cognitive decline. Contrasting association pattern of plasma low-density lipoprotein with white matter integrity in APOE4 carriers versus non-carriers. Zhang, Q. et al. Risk prediction of late-onset Alzheimer's disease implies an oligogenic architecture. The gut microbiota-derived metabolite trimethylamine N-oxide is elevated in Alzheimer's disease. Marksteiner, J., Blasko, I., Kemmler, G., Koal, T. & Humpel, C. Bile acid quantification of 20 plasma metabolites identifies lithocholic acid as a putative biomarker in Alzheimer's disease. Diagnostic accuracy of a plasma phosphorylated Tau 217 immunoassay for Alzheimer disease pathology. Valls-Pedret, C. et al. Mediterranean diet and age-related cognitive decline: a randomized clinical trial. & Hustad, S. Betaine: a key modulator of one-carbon metabolism and homocysteine status. Bhatt, M., Di Iacovo, A., Romanazzi, T., Roseti, C. & Bossi, E. Betaine—the dark knight of the brain. Obeid, R. The metabolic burden of methyl donor deficiency with focus on the betaine homocysteine methyltransferase pathway. Blanchard, J. W. et al. APOE4 impairs myelination via cholesterol dysregulation in oligodendrocytes. van der Kant, R. et al. Cholesterol metabolism Is a druggable axis that independently regulates Tau and amyloid-beta in iPSC derived Alzheimer's disease neurons. Kaji, S. et al. Apolipoprotein E aggregation in microglia initiates Alzheimer's disease pathology by seeding beta-amyloidosis. & Sato, M. N. Role of histamine in modulating the immune response and inflammation. & Rivest, S. The role of ABCB1 and ABCA1 in beta-amyloid clearance at the neurovascular unit in Alzheimer's disease. O'Sullivan, J. F. et al. Dimethylguanidino valeric acid is a marker of liver fat and predicts diabetes. Gibney, M. J. et al. Metabolomics in human nutrition: opportunities and challenges. Plasma lipidomic profiles and cardiovascular events in a randomized intervention trial with the Mediterranean diet. Xu, J. et al. Cerebral deficiency of vitamin B5 (d-pantothenic acid; pantothenate) as a potentially-reversible cause of neurodegeneration and dementia in sporadic Alzheimer's disease. Snowden, S. G. et al. Association between fatty acid metabolism in the brain and Alzheimer disease neuropathology and cognitive performance: a nontargeted metabolomic study. Zhang, J. et al. Asparagine plays a critical role in regulating cellular adaptation to glutamine depletion. Fujimaki, M. et al. Serum caffeine and metabolites are reliable biomarkers of early Parkinson disease. McCormick, D. A. GABA as an inhibitory neurotransmitter in human cerebral cortex. Kim, Y. S. & Yoon, B. E. Altered GABAergic signaling in brain disease at various stages of life. Sastre, M., Regunathan, S., Galea, E. & Reis, D. J. Agmatinase activity in rat brain: a metabolic pathway for the degradation of agmatine. Song, J. et al. Agmatine improves cognitive dysfunction and prevents cell death in a streptozotocin-induced Alzheimer rat model. & Burda, K. Potential role of carotenoids as antioxidants in human health and disease. Grodstein, F., Kang, J. H., Glynn, R. J., Cook, N. R. & Gaziano, J. M. A randomized trial of beta carotene supplementation and cognitive function in men: the Physicians' Health Study II. Madeira, C. et al. Elevated glutamate and glutamine levels in the cerebrospinal fluid of patients with probable Alzheimer's disease and depression. & Narayan, P. S. Apolipoprotein E in lipid metabolism and neurodegenerative disease. van de Rest, O. et al. APOE epsilon4 and the associations of seafood and long-chain omega-3 fatty acids with cognitive decline. Shannon, O. M. et al. Mediterranean diet adherence is associated with lower dementia risk, independent of genetic predisposition: findings from the UK Biobank prospective cohort study. Migliore, L. & Coppede, F. Gene-environment interactions in Alzheimer disease: the emerging role of epigenetics. Lista, S. et al. Integrative metabolomics science in Alzheimer's disease: relevance and future perspectives. Mielke, M. M. Sex and gender differences in Alzheimer's disease dementia. Origin, methods, and evolution of the three Nurses' Health Studies. Stampfer, M. J. et al. Test of the national death index. A practical method for grading the cognitive state of patients for the clinician. Lindstrom, S. et al. A comprehensive survey of genetic variation in 20,691 subjects from four large cohorts. Willett, W. C. et al. Reproducibility and validity of a semiquantitative food frequency questionnaire. Diet quality is associated with the risk of estrogen receptor-negative breast cancer in postmenopausal women. Survival model predictive accuracy and ROC curves. Harrell, F. E., Califf, R. M., Pryor, D. B., Lee, K. L. & Rosati, R. A. Evaluating the yield of medical tests. A unified approach to interpreting model predictions. 31st International Conference on Neural Information Processing Systems (eds Guyon, I. et al.) 4768–4777 (Curran Associates Inc., 2017). Lawlor, D. A. Commentary: two-sample Mendelian randomization: opportunities and challenges. Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. We acknowledge the contribution to this study from central cancer registries supported through the Centers for Disease Control and Prevention's National Program of Cancer Registries and/or the National Cancer Institute's Surveillance, Epidemiology, and End Results (SEER) program. Central registries may also be supported by state agencies, universities and cancer centers. Participating central cancer registries include the following: Alabama, Alaska, Arizona, Arkansas, California, Colorado, Connecticut, Delaware, Florida, Georgia, Hawaii, Idaho, Indiana, Iowa, Kentucky, Louisiana, Massachusetts, Maine, Maryland, Michigan, Mississippi, Montana, Nebraska, Nevada, New Hampshire, New Jersey, New Mexico, New York, North Carolina, North Dakota, Ohio, Oklahoma, Oregon, Pennsylvania, Puerto Rico, Rhode Island, Seattle SEER Registry, South Carolina, Tennessee, Texas, Utah, Virginia, West Virginia and Wyoming. This work was funded by the National Institutes of Health (NIH) (grant nos. The computations in this paper were run in part on the FAS Research Computing System supported by the FAS Division of Science Research Computing Group at Harvard University. Channing Division of Network Medicine, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA Yuxi Liu, Cheng Peng, Danyue Dong, Yuhan Li, Oana A. Zeleznik, Jae H. Kang, Molin Wang, Frank B. Hu, Olivia I. Okereke, A. Heather Eliassen, Meir J. Stampfer & Dong D. Wang Chan School of Public Health, Boston, MA, USA Yuxi Liu, Yu Zhang, Molin Wang, Frank B. Hu, Walter C. Willett, Olivia I. Okereke, A. Heather Eliassen, Peter Kraft & Meir J. Stampfer Broad Institute of MIT and Harvard, Cambridge, MA, USA Chan School of Public Health, Boston, MA, USA Xiao Gu, Yanping Li, Fenglei Wang, Yin Zhang, Frank B. Hu, Walter C. Willett, A. Heather Eliassen, Meir J. Stampfer & Dong D. Wang Department of Psychiatry, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Chan School of Public Health, Boston, MA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors declare no competing interests. Nature Medicine thanks Moonil Kang, Oliver Shannon and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Liam Messin, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) We prospectively followed 1,490 men in Health Professionals Follow-Up Study (HPFS) from 1993 through 2023. Genetic and metabolomic profiles were generated from blood samples collected at baseline. Detailed demographic, lifestyle, dietary, medical history and medication use data were collected via questionnaires. (b) Associations between established genetic risk factors for Alzheimer's disease and related dementias (ADRD) and dementia risk. Lines indicate cumulative incidence across APOE4 genotypes and tertiles of polygenic risk score (PRS) of ADRD (excluding the APOE region) over the follow-up period, with shaded areas representing 95% confidence intervals (CIs) and P from the log-rank test annotated. Consistent with the curves, unadjusted hazard ratios (HRs) were estimated using Cox proportional hazards models; covariate-adjusted HRs with 95% CIs are provided in Supplementary Table 4. Person-time was accrued from baseline until the earliest occurrence of an incident dementia case, dementia death or the end of follow-up. The analyses were conducted among 1,490 HPFS participants. (a) Correlations between genetic principal components (PCs) and metabolites: PC1 (top panel) and PC2 (bottom panel). The dashed lines denote false discovery rate (FDR) = 0.05, adjusted for multiple comparisons, based on Pearson's correlation tests. (b) Associations of APOE4 heterozygosity (top panel) and homozygosity (bottom panel) with metabolites, with APOE4 noncarrier as the reference group from Cox proportional hazards models. Models were adjusted for age, the top 4 genetic PCs, and genotyping platforms. The dashed lines denote nominal P = 0.05 (none with FDR < 0.05). All analyses were conducted among 4,215 NHS participants. Bars indicate the number of associations that reached nominal significance (P < 0.05) based on the Cox proportional hazards models, and those that remained significant after adjustment for multiple comparisons (false discovery rate <0.05). Each dot represents a metabolite with significant APOE4 interactions in Nurses' Health Study (NHS; n = 38 out of 49 available in Health Professionals Follow-Up Study [HPFS]), colored by the Human Metabolome Database superclass (see Fig. Pearson's correlation coefficients in the β coefficients for interaction terms between metabolites and APOE4 carrier status estimated from Cox proportional hazards models with dementia as the dependent variable in NHS and HPFS is annotated on the figure. APOE4 carriers were not further stratified into heterozygotes and homozygotes due to data sparsity among homozygotes with non-missing values for each metabolite in HPFS. Analyses were conducted among 4,215 NHS and 1,490 HPFS participants. (a) Associations between Mediterranean diet (MedDiet) index score and cognitive function scores by APOE4 genotype from generalized linear models. (b) Associations between MedDiet index score and dementia risk by tertiles of ADRD PRSs from Cox proportional hazards models. (c) Associations between MedDiet index score and cognitive function scores by tertiles of ADRD PRSs from generalized linear models. Stratified β coefficients (for cognitive function) and hazard ratios (HRs; for dementia risk), along with 95% confidence intervals (CIs) per one-unit increase in the MedDiet index score, are shown by APOE4 genotype or PRS categories. The analyses were conducted among NHS participants with genetic, dietary, and cognitive outcome data (n = 16,497 for dementia analyses and n = 3,770 for cognitive function analyses). (a) Higher adherence to the Mediterranean diet (MedDiet) was prospectively associated with a lower risk of dementia. A restricted cubic spline Cox proportional hazards (PH) model estimated hazard ratios (HRs) and 95% confidence intervals (CIs) across varying levels of the MedDiet index. This analysis was conducted among Health Professionals Follow-Up Study (HPFS) participants with dementia and dietary data (n = 43,500). (b) The protective association between adherence to the MedDiet and risk of dementia was most pronounced among APOE4 homozygotes. Stratified HR and 95% CIs for dementia risk per one-unit increment in the MedDiet index score, categorized by APOE4 genotype, were estimated from Cox PH models, with stratified P annotated (unadjusted for multiple comparisons in the hypothesis-driven analysis). The analyses were conducted among HPFS participants with genetic, dietary, and dementia outcome data (n = 9,828). (c) Strong association between adherence to MedDiet and the overall plasma metabolome profile identified by a random forest (RF) model to classify individuals in the top versus bottom quartile of the MedDiet index based on plasma metabolites. For the RF classification, the dataset was randomly divided into training (60%) and test (40%) sets. The analyses were conducted among 1,490 HPFS participants. The heatmap shows β coefficients representing a 1-s.d. increment in the MedDiet index from a generalized linear model, with plasma metabolite levels as the dependent variable, restricting to the 254 metabolites available in both cohorts. In the metabolite analysis, APOE4 carriers were not further stratified into heterozygotes and homozygotes due to data sparsity among homozygotes with non-missing values for each metabolite in HPFS. The analyses were conducted among 4,215 NHS and 1,490 HPFS participants. Each dot represents an individual and is colored by the MedDiet score. MedDiet score and its components are shown on the plot as arrows; the coordinates of the arrow heads represent their Pearson's correlation coefficients with metabolite PC1 and PC2. The analyses were conducted among 4,215 participants in Nurses' Health Study. Each dot represents a metabolite (diglycerides on the left panel and triglycerides on the right panel), color-coded by the direction and significance level of its association with the Mediterranean diet index. The analyses were conducted among 4,215 NHS participants. APOE4 genotype was not included as a predictor in the subgroup analysis within APOE4 carrier and noncarrier subgroups. The full NHS dataset (n = 4,215) was randomly split into training (60%) and test (40%) sets. All prediction models were fitted in the training set and were evaluated in the test set. All results shown are from the test set. (a) The inclusion of genetic factors improved dementia risk prediction using Cox proportional hazards (PH) model, with an additional modest enhancement when plasma metabolites were also included. The baseline model predictors included age, family history of dementia, profession, smoking status, history of depression or regular antidepressant use, and Mediterranean diet (MedDiet) index. The polygenic risk score (PRS) of Alzheimer's disease and related dementias excluded variants in the APOE region. (b) Plasma metabolites are among the top contributors for predicting dementia risk as quantified by the Shapley Additive Explanations (SHAP) value. Feature contributions were evaluated for the Cox PH model for predicting dementia risk including the full list of predictors. SHAP values were calculated for each category of predictors by summing the SHAP value of all predictors in that category. We did not assess 15-year dementia risk in the Health Professionals Follow-Up Study (HPFS) due to the limited number of dementia cases. For all analyses, the HPFS dataset (n = 1,490) was randomly divided into training (60%) and test (40%) sets; models were fitted on the training set and evaluated on the test set. All results shown are from the test set. Statistical source data for Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Interplay of genetic predisposition, plasma metabolome and Mediterranean diet in dementia risk and cognitive function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250824031547.htm'>Scientists discover a strange new magnet that bends light like magic</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 05:53:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers have uncovered the magnetic properties and underlying mechanisms of a novel magnet using advanced optical techniques. Unlike conventional ferromagnets and antiferromagnets, altermagnets exhibit unique magnetic behavior. Details of their breakthrough were published recently in the journal Physical Review Research. "Unlike typical magnets that attract each other, altermagnets do not exhibit net magnetization, yet they can still influence the polarization of reflected light," points out Satoshi Iguchi, associate professor at Tohoku University's Institute for Materials Research. "This makes them difficult to study using conventional optical techniques." To overcome this, Iguchi and his colleagues applied a newly derived general formula for light reflection to the organic crystal, successfully clarifying its magnetic properties and origin. The team's newly derived general formula for light reflection was based on Maxwell's equations and is applicable to a wide range of materials, including those with low crystal symmetry, such as the organic compound studied here. This new theoretical framework also allowed the team to develop a precise optical measurement method and apply it to the organic crystal κ-(BEDT-TTF)2Cu[N(CN)2]Cl. The results revealed three key features in the spectrum: (1) edge peaks indicating spin band splitting, (2) a real component associated with crystal distortion and piezomagnetic effects, and (3) an imaginary component linked to rotational currents. "This research opens the door to exploring magnetism in a broader class of materials, including organic compounds, and lays the groundwork for future development of high-performance magnetic devices based on lightweight, flexible materials," adds Iguchi. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250824031544.htm'>Scientists discover flaws that make electronics faster, smarter, and more efficient</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-25 04:11:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Traditional devices rely only on the electric charge of electrons to store and process information. Spintronics takes advantage of two additional quantum properties: spin angular momentum, which can be imagined as a built-in "up" or "down" orientation of the electron, and orbital angular momentum, which describes how electrons move around atomic nuclei. By using these extra degrees of freedom, spintronic devices can store more data in smaller spaces, operate faster, consume less energy, and retain information even when the power is switched off. A longstanding challenge in spintronics has been the role of material defects. Introducing imperfections into a material can sometimes make it easier to "write" data into memory bits by reducing the current needed, but this typically comes at a cost: electrical resistance increases, spin Hall conductivity declines, and overall power consumption goes up. This trade-off has been a major obstacle to developing ultra-low-power spintronic devices. Their study, published in Nature Materials, focused on the orbital Hall effect in strontium ruthenate (SrRuO3), a transition metal oxide whose properties can be finely tuned. This quantum phenomenon causes electrons to move in a way determined by their orbital angular momentum. Using custom-designed devices and precision measurement techniques, the researchers uncovered an unconventional scaling law that achieves a "two birds with one stone" outcome: Defect engineering simultaneously boosts both orbital Hall conductivity and orbital Hall angle, a stark contrast to conventional spin-based systems. "Scattering processes that typically degrade performance actually extend the lifetime of orbital angular momentum, thereby enhancing orbital current," said Dr. Xuan Zheng, a co-first author of the study. Experimental measurements confirm the technology's potential: tailored conductivity modulation yielded a threefold improvement in switching energy efficiency. This study not only provides new insights into orbital transport physics but also redefines design strategies for energy-efficient spintronics. This study received support from the National Key Research and Development Program of China, the National Natural Science Foundation of China, and other funding bodies. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            