
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-10</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09174-w'>Silicate clouds and a circumplanetary disk in the YSES-1 exoplanet system</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 15:30:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Please note there may be errors present which affect the content, and all legal disclaimers apply. Young exoplanets provide a critical link between understanding planet formation and atmospheric evolution1. Direct imaging spectroscopy allows us to infer the properties of young, wide orbit, giant planets with high signal-to-noise. This allows us to compare this young population to exoplanets characterized with transmission spectroscopy, which has indirectly revealed the presence of clouds2-4, photochemistry5, and a diversity of atmospheric compositions6-7. With two exoplanets around a solar type star, the YSES-1 system is an ideal laboratory for studying this early phase of exoplanet evolution. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Space Telescope Science Institute, 3700 San Martin Dr, Baltimore, MD, USA Millennium Nucleus on Young Exoplanets and their Moons (YEMS), Santiago, Chile University of Rome Tor Vergata, Via Cracovia, 50, Roma, RM, Italy University of California Observatories, 550 Red Hill Rd, Santa Cruz, CA, USA Hoch, K.K.W., Rowland, M., Petrus, S. et al. Silicate clouds and a circumplanetary disk in the YSES-1 exoplanet system. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250610074256.htm'>Sun unleashes monster solar storm: Rare G4 alert issued for earth</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 14:02:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Local weather alerts are familiar warnings for potentially dangerous conditions, but an alert that puts all of Earth on warning is rare. Naval Research Laboratory's (NRL) space-based instrumentation captured real-time observations of a powerful Coronal Mass Ejection (CME) that erupted from the Sun initiating a "severe geomagnetic storm" alert for Earth. "Our observations demonstrated that the eruption was a so-called 'halo CME,' meaning it was Earth-directed, with our preliminary analysis of the data showing an apparent velocity of over 1,700 kilometers per second for the event," stated Karl Battams, Ph.D., computational scientist for NRL's Heliospheric Science Division. A geomagnetic storm is a major disturbance of Earth's magnetosphere that's caused by the highly efficient transfer of energy from the solar wind into our planet's surrounding space environment. These disruptions are primarily driven by sustained periods of high-speed solar wind and, crucially, a southward-directed solar wind magnetic field that can peel away Earth's field on the dayside of the magnetosphere. Energy from the solar wind can open Earth's magnetic shield. Powerful storms such as this are typically associated with CMEs. While CMEs generally take several days to reach Earth, the most intense events have been observed to arrive in as little as 18 hours. The energy released blasted a CME directly toward Earth, leading to the geomagnetic storm that has produced auroras as far south as New Mexico. NRL's space-based instrumentation, operating on NASA and NOAA spacecraft, provided vital real-time observations of this event. Notably, NRL's venerable Large Angle Spectrometric Coronagraph (LASCO), which has been in operation since 1996, and the Compact Coronagraph 1 (CCOR-1), launched in 2024, both relayed critical data. While precisely predicting the severity, exact timing, or duration of a geomagnetic storm remains challenging, these advance warnings are vital for enabling the Department of Defense (DoD) and other agencies to prepare. The potential impacts of severe geomagnetic storms on DoD and Department of the Navy missions are significant and far-reaching. "NRL has been a pioneer in heliophysics and space weather research since the very inception of the field, dating back to the first discovery of CMEs through NRL space-based observations in 1971," Battams said. "Since then, NRL has consistently maintained its position at the forefront of coronal imaging with a portfolio of groundbreaking instrumentation that has driven heliospheric and space weather studies." The data collected from events such as this will be instrumental in future research, further enhancing our understanding and predictive capabilities and ultimately bolstering the resilience of national security and critical infrastructure. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250610074301.htm'>Sharper than lightning: Oxford's one-in-6. 7-million quantum breakthrough</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 13:50:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Physicists at the University of Oxford have set a new global benchmark for the accuracy of controlling a single quantum bit, achieving the lowest-ever error rate for a quantum logic operation -- just 0.000015%, or one error in 6.7 million operations. This record-breaking result represents nearly an order of magnitude improvement over the previous benchmark, set by the same research group a decade ago. To put the result in perspective: a person is more likely to be struck by lightning in a given year (1 in 1.2 million) than for one of Oxford's quantum logic gates to make a mistake. The findings, published in Physical Review Letters, are a major advance towards having robust and useful quantum computers. "It is an important step toward building practical quantum computers that can tackle real-world problems." Although error correction can be used to fix mistakes, this comes at the cost of requiring many more qubits. Co-lead author Molly Smith (Graduate Student, Department of Physics, University of Oxford), said: "By drastically reducing the chance of error, this work significantly reduces the infrastructure required for error correction, opening the way for future quantum computers to be smaller, faster, and more efficient. This unprecedented level of precision was achieved using a trapped calcium ion as the qubit (quantum bit). This method offers greater stability than laser control and also has other benefits for building a practical quantum computer. Furthermore, the experiment was conducted at room temperature and without magnetic shielding, thus simplifying the technical requirements for a working quantum computer. Whilst this record-breaking result marks a major milestone, the research team caution that it is part of a larger challenge. Quantum computing requires both single- and two-qubit gates to function together. Currently, two-qubit gates still have significantly higher error rates -- around 1 in 2000 in the best demonstrations to date -- so reducing these will be crucial to building fully fault-tolerant quantum machines. The experiments were carried out at the University of Oxford's Department of Physics by Molly Smith, Aaron Leu, Dr Mario Gely and Professor David Lucas, together with a visiting researcher, Dr Koichiro Miyanishi, from the University of Osaka's Centre for Quantum Information and Quantum Biology. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250610074307.htm'>Scientists uncover why "stealth" volcanoes stay silent until eruption</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 13:43:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When volcanoes are preparing to erupt, scientists rely on typical signs to warn people living nearby: deformation of the ground and earthquakes, caused by underground chambers filling up with magma and volcanic gas. Now scientists studying Veniaminof, Alaska, have developed a model which could explain and predict stealthy eruptions. "Despite major advances in monitoring, some volcanoes erupt with little or no detectable precursors, significantly increasing the risk to nearby populations," said Dr Yuyu Li of the University of Illinois, lead author of the study in Frontiers in Earth Science. "Our work helps explain how this happens, by identifying the key internal conditions -- such as low magma supply and warm host rock -- that make eruptions stealthy." Veniaminof is an ice-clad volcano in the Aleutian Arc of Alaska. It's carefully monitored, but only two of its 13 eruptions since 1993 have been preceded by enough signs to alert observing scientists. In fact, a 2021 eruption wasn't caught until three days after it had started. "It is one of the most active volcanoes in Alaska. In recent decades, it has produced several VEI 3 eruptions -- moderate-sized explosive events that can send ash up to 15 km high, disrupt air traffic, and pose regional hazards to nearby communities and infrastructure -- often without clear warning signs." To understand Veniaminof better, the scientists used monitoring data over three summer seasons immediately before the 2018 stealthy eruption, which produced only ambiguous warning signs immediately before it happened. They then compared the models to the data to see which matched best, and which conditions produced eruptions, stealthy or otherwise. If magma is flowing quickly into a large chamber, an eruption may not occur, but if one does the ground will deform enough to warn scientists first. Magma flowing into larger, flatter chambers may cause minimal earthquakes, while smaller, more elongated chambers may produce little deformation of the ground. "To mitigate the impact of these potential surprise eruptions, we need to integrate high-precision instruments like borehole tiltmeters and strainmeters and fiber optic sensing, as well as newer approaches such as infrasound and gas emission monitoring," said Li. At Veniaminof, taking measures to improve the coverage of satellite monitoring and adding tiltmeters and strainmeters could improve the rate of detection. In the meantime, scientists now know which volcanoes they need to watch most closely: volcanoes with small, warm reservoirs and slow magma flows. "Combining these models with real-time observations represents a promising direction for improving volcano forecasting," said Li. "In the future, this approach can enable improved monitoring for these stealthy systems, ultimately leading to more effective responses to protect nearby communities." Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65013540/confederate-soldier-skeletons/'>Archaeologists Were Digging Up a Hospital—and Found the Skeletons of 4 Confederate Soldiers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The deaths were likely tied to a Civil War battle. “If these men were found in a mass grave on a battlefield, and there was no other information, we probably wouldn't be trying to do this.” The list of the wounded was part of a private family's archival documents which, at some point, were given to nearby William & Mary University. The researchers went through the detailed documentation, which included everything from names and regiments to dates of deaths and amputation information, for over 60 soldiers. They even have some individual names in mind, keeping those close to the vest until they can work with possible descendants using DNA from the soldiers' teeth. Gary said the goal is to ensure an “ironclad” identification. The remains, which were initially discovered in 2023, were carefully buried with their arms crossed, indication they did not die during the battle, as those men were traditionally placed in trenches and then moved to a cemetery, Gary said. The four soldiers were out of uniform, as evidenced by buttons and a buckle, commonly found on more comfortable clothes, that the archaeologists also found. Archaeologist Eric Schweickart told the AP that one soldier had two $5 gold coins from 1852, and they also found a toothbrush made from animal bone and a snuff bottle. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Archaeologists Found an Odd Skull With a Big Wound This Christian Site Is the Tomb of Ancient Royalty Researchers Found Ruins of a Lost Byzantine City The Dead Sea Scrolls Are Older Than We Thought</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250610004101.htm'>This overlooked supplement could help you think sharper and age better</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 10:08:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Creatine, the supplement popular with athletes for its ability to help build strength and power, is increasingly being recognized for its broad health benefits. Kreider has spent more than 30 years investigating the effects of creatine, a naturally occurring compound stored in the muscle that combines with phosphate to form creatine phosphate, which is needed for cellular energy. "When the body is stressed, like in exercise or under metabolic conditions like some diseases, creatine phosphate is needed to maintain energy in the cell, and therefore has a lot of protective and health benefits, in addition to the exercise performance effects that have been seen," Kreider said. Our bodies create about a gram per day, but it's recommended to get two to four grams of creatine per day, depending on muscle mass and activity levels. According to Kreider, most people fall short of getting enough creatine from diet alone. This is why supplementation matters, especially for vegetarians or vegans who do not consume enough creatine in their diet. Supplementation "helps load the muscle up with more energy," which makes for improved high-intensity exercise, recovery and even cognitive function. Beyond boosting athletic performance, creatine is important for everyone as they age throughout their lives, Kreider said. It can help older adults who lose muscle mass and cognitive function as they age, he said, and in adolescents, low dietary creatine intake is associated with slower growth, less muscle mass and higher body fat. In a comprehensive review published in February in the Journal of the International Society of Sports Nutrition, Kreider and colleagues analyzed 685 clinical trials on creatine supplementation to assess its safety and the frequency of reported side effects. The analysis showed there were no significant differences in the rate of side effects for participants taking a placebo and those taking creatine. As for anecdotal concerns like bloating or cramping, Kreider says those claims don't hold up under scrutiny, and studies have shown creatine can actually prevent cramping because it helps the body retain more fluid. "There's absolutely no data supporting any negative side effect anecdotally reported about creatine on the internet and in the media," he said. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41558-025-02354-6'>Recommendations for producing knowledge syntheses to inform climate change assessments</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 09:58:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Climate Change (2025)Cite this article Climate change assessments (CCAs) play a critical role in taking stock of the available science and other forms of knowledge and informing policy processes. As the underlying evidence base increases exponentially, the complexity also increases and challenges CCA author teams to capture all the relevant knowledge. Therefore, CCAs will need to transition from predominantly assessing primary research to focusing on the assessment and critical appraisal of knowledge syntheses of such work, alongside capturing knowledges held outside traditional scientific sources. To support this, a stronger knowledge synthesis culture is needed, and we propose key recommendations and offer guidance for producing robust, transparent, reproducible, inclusive and timely syntheses that can inform CCAs across scales. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Berrang-Ford, L. et al. Editorial: Evidence synthesis for accelerated learning on climate solutions. Haddaway, N. R., Lotfi, T. & Mbuagbaw, L. Systematic reviews: a glossary for public health. Public Health 51, 1–10 (2023). This paper develops a typology for choosing and naming review methods used with a focus on application in a public health context. Google Scholar Haddaway, N. R. & Macura, B. The role of reporting standards in producing robust literature reviews comment. Google Scholar Surkovic, E. & Vigar, D. Scientific advice for policymakers on climate change: the role of evidence synthesis. Donnelly, C. A. et al. Four principles for synthesizing evidence. Google Scholar Nalau, J., Gilmore, E. & Howden, M. Improving adaptation assessment in the IPCC. npj Clim. Hughes, H. Actors, activities, and forms of authority in the IPCC. Google Scholar Slade, R. et al. Back to basics for the IPCC: applying lessons from AR6 to the Seventh Assessment Cycle. npj Clim. Asayama, S. et al. Three institutional pathways to envision the future of the IPCC. Google Scholar Oppenheimer, M. et al. Discerning Experts—The Practices of Scientific Assessment for Environmental Policy (Univ. The authors examine how environmental assessments are shaped by expert judgement, institutional norms and political contexts. Callaghan, M. W., Minx, J. C. & Forster, P. M. A topography of climate change research. This paper documents the massive expansion of scientific literature on climate change and the nature of the challenge facing global environmental assessments. Khojasteh, D. et al. Climate change science is evolving toward adaptation and mitigation solutions. Sietsma, A. J. et al. Machine learning evidence map reveals global differences in adaptation action. Lamb, W. F., Creutzig, F., Callaghan, M. W. & Minx, J. C. Learning about urban climate solutions from case studies. Google Scholar Haddaway, N. R. et al. On the use of computer-assistance to facilitate systematic mapping. Minx, J. C., Callaghan, M., Lamb, W. F., Garard, J. Learning about climate change solutions in the IPCC and beyond. Google Scholar Dicks, L. V. et al. Knowledge Synthesis for Environmental Decisions: An Evaluation of Existing Methods, and Guidance for their Selection, Use and Development (EKLIPSE, 2017); https://researchportal.port.ac.uk/en/publications/knowledge-synthesis-for-environmental-decisions-an-evaluation-of- Alegria, A. et al. Towards an IPCC Atlas for comprehensive climate change risk assessments. npj Clim. Elliott, J. et al. Decision makers need ‘living' evidence synthesis. Google Scholar Pearson, H. Scientists are building giant ‘evidence banks' to create policies that actually work. Google Scholar Berrang-Ford, L., Pearce, T. & Ford, J. D. Systematic review approaches for climate change adaptation research. This study examines and illustrates how to conduct systematic reviews for adaptation-relevant questions. Google Scholar Chigbu, U. E., Atiku, S. O. & Du Plessis, C. C. The science of literature reviews: searching, identifying, selecting, and synthesising. The authors focus in detail on the steps involved in literature reviews. Bednar-Friedl, B. et al. The Next Frontier for Climate Change Science: Insights from the Authors of the IPCC 6th Assessment Report on Knowledge Gaps and Priorities for Research (eds Drabicka, K. et al.) (European Comission Directorate-General for Research and Innovation, Publications Office of the European Union, 202); https://data.europa.eu/doi/10.2777/34601 Sixty-first Session of the IPCC 27 July–2 August 2024, Sofia, Bulgaria: Decisions Adopted by the Panel (IPCC, 2024); https://www.ipcc.ch/site/assets/uploads/2024/08/IPCC-61_decisions-adopted-by-the-Panel.pdf Collins, A., Coughlin, D. & Randall, N. Engaging environmental policy-makers with systematic reviews: challenges, solutions and lessons learned. Haddaway, N. & Crowe, S. Experiences and lessons in stakeholder engagement in environmental evidence synthesis: a truly special series. Pullin, A. S., Frampton, G. K., Livoreil, B. & Petrokofsky, G. (eds) Guidelines and Standards for Evidence Synthesis in Environmental Management Version 5.1 (Collaboration for Environmental Evidence, 2022); https://environmentalevidence.org/information-for-authors/ Kopp, R. et al. Communicating future sea-level rise uncertainty and ambiguity to assessment users. Google Scholar Magnan, A. K. et al. Sea level rise risks and societal adaptation benefits in low-lying coastal areas. Milbank, C. et al. Rethinking hierarchies of evidence for sustainable food systems. Google Scholar Vijayan, D. et al. Indigenous knowledge in food system transformations. Ford, J., Pearce, T., Canosa, I. & Harper, S. The rapidly changing Arctic and its societal implications. Google Scholar Alaskan Inuit Food Security Conceptual Framework: How to Assess the Arctic from an Inuit Perspective (Inuit Circumpolar Council-Alaska, 2015). Haddaway, N. et al. A framework for stakeholder engagement during systematic reviews and maps in environmental management. Google Scholar Methley, A., Campbell, S., Chew-Graham, C., McNally, R. & Cheraghi-Sohi, S. PICO, PICOS and SPIDER: a comparison study of specificity and sensitivity in three search tools for qualitative systematic reviews. BMC Health Serv. Cooke, A., Smith, D. & Booth, A. Beyond PICO: the SPIDER tool for qualitative evidence synthesis. Google Scholar Evidence Synthesis for Policy: Statement of Principles (Royal Society & Academy of Medical Sciences, 2018). Principles Governing IPCC Work (IPCC, 2013). The importance of distinguishing climate science from climate activism. npj Clim. Singh, C., Ford, J., Ley, D., Bazaz, A. & Revi, A. Assessing the feasibility of adaptation options: methodological advancements and directions for climate adaptation research and practice. Google Scholar Haddaway, N. R. et al. Eight problems with literature reviews and how to fix them. This study identifies and characterizes major pitfalls that should be avoided when conducting and reporting systematic reviews. Google Scholar Preferred Reporting Items for Systematic Reviews and Meta Analyses (PRISMA, 2020); https://www.prisma-statement.org/ RepOrting standards for Systematic Evidence Syntheses (ROSES, 2024); https://www.roses-reporting.com/ Tengo, M. et al. Weaving knowledge systems in IPBES, CBD and beyond-lessons learned for sustainability. Google Scholar & Sharp, K. Indigenous climate knowledges. Google Scholar Mahony, M. & Hulme, M. Epistemic geographies of climate change: Science, space and politics. Google Scholar Ford, J. D. et al. Including Indigenous knowledge and experience in IPCC assessment reports. Google Scholar McElwee, P. et al. Working with Indigenous and local knowledge (ILK) in large-scale ecological assessments: reviewing the experience of the IPBES Global Assessment. Google Scholar The Knowledge Dialogues Methodology (Pan American Health Organization, 2022); https://doi.org/10.37774/9789275124703 McDowell, G. et al. The Canadian Mountain Assessment (Univ. This book is an example of a climate change assessment where Indigenous knowledge was incorporated from the beginning of the assessment process. We Rise Together: Achieving Pathway to Canada Target 1 Through the Creation of Indigenous Protected and Conserved Areas in the Spirit and Practice of Reconciliation: Indigenous Circle of Experts Report and Recommendations (Government of Canada, 2018); https://publications.gc.ca/pub?id=9.852966&sl=0 Tengö, M., Brondizio, E., Elmqvist, T., Malmer, P. & Spierenburg, M. Connecting diverse knowledge systems for enhanced ecosystem governance: the multiple evidence base approach. The authors present an approach for connecting multiple forms of knoweldge in environmental assessments. Google Scholar Sherwood, S. et al. An assessment of Earth's climate sensitivity using multiple lines of evidence. A physics-informed Bayesian storyline approach to assess sediment transport in the Mekong. Kopp, R. et al. The Framework for Assessing Changes To Sea-level (FACTS) v1.0: a platform for characterizing parametric and structural uncertainty in future global, relative, and extreme sea-level change. Johnson, E., O'Keefe, H., Sutton, A. & Marshall, C. The systematic review toolbox: keeping up to date with tools to support evidence synthesis. Tsafnat, G. et al. Systematic review automation technologies. Debernardi, C., Seeber, M. & Cattaneo, M. Thirty years of climate change research:a fine-grained analysis of geographical specialization. Sietsma, A., Ford, J. The next generation of machine learning for tracking adaptation texts. This Perspective offers an in-depth examination of how machine learning can be used to track adaptation progress, along with pitfalls and opportunities. Binz, M. et al. How should the advent of large language models affect the practice of science? Preprint at https://arxiv.org/pdf/2312.03759 (2025). Khalil, H. et al. Automation tools to support undertaking scoping reviews. Affengruber, L. et al. Rapid review methods series: guidance on the use of supportive software. Jimenez, R. et al. Machine learning computational tools to assist the performance of systematic reviews: a mapping review. Callaghan, M. & Müller-Hansen, F. Statistical stopping criteria for automated screening in systematic reviews. Muccione, V. et al. Integrating artificial intelligence with expert knowledge in global environmental assessments: opportunities, challenges and the way ahead. This study explores how AI can be used within global environmental assessments, illustrating how some processes can be automated and streamlined. Debnath, R., Creutzig, F., Sovacool, B. K. & Shuckburgh, E. Harnessing human and machine intelligence for planetary-level climate action. npj Clim. Wells, S. Ready or not, AI is coming to science education—and students have opinions. Schulz, K. & Grimes, D. Blinding in randomised trials: hiding who got what. McClenachan, L., Ferretti, F. & Baum, J. From archives to conservation: why historical data are needed to set baselines for marine animals and ecosystems. Ford, J. D. Increasing trust in climate vulnerability projections. Lloyd, E. & Oreskes, N. Climate Change attribution: when is it appropriate to accept new methods. Shea, B. et al. AMSTAR 2: a critical appraisal tool for systematic reviews that include randomised or non-randomised studies of healthcare interventions, or both. Frampton, G. et al. Principles and framework for assessing the risk of bias for studies included in comparative quantitative environmental systematic reviews. David-Chavez, D. M. & Gavin, M. C. A global assessment of Indigenous community engagement in climate research. Perumal, L., New, M., Jonas, M. & Liu, W. The impact of roads on sub-Saharan African ecosystems: a systematic review. Rebelo, A., Holden, P., Esler, K. & New, M. Benefits of water-related ecological infrastructure investments to support sustainable land-use: a review of evidence from critically water-stressed catchments in South Africa. Sprague, N. L. et al. The CHANGE (Climate Health ANalysis Grading Evaluation) tool for weight of evidence reviews on climate change and health research. Jager, N. W., Newig, J., Challies, E., Kochskämper, E. & von Wehrden, H. Case study meta-analysis in the social sciences. Insights on data quality and reliability from a large-N case survey. Eick, S. M., Goin, D. E., Chartres, N., Lam, J. & Woodruff, T. J. Assessing risk of bias in human environmental epidemiology studies using three tools: different conclusions from different tools. Mastrandrea, M. D. et al. Guidance Note for Lead Authors of the IPCC Fifth Assessment Report on Consistent Treatment of Uncertainties (IPCC, 2010). Fox-Kemper, B. et al. in Climate Change 2021: The Physical Science Basis (eds Masson-Delmotte, V. et al.) 1211–1362 (IPCC, Cambridge Univ. Haddaway, N. Open synthesis: on the need for evidence synthesis to embrace Open Science. The author explores key principles needed to underpin open science in the context of evidence synthesis. Nosek, B. et al. Promoting an open research culture. Chin, J., Growns, B., Sebastian, J., Page, M. & Nakagawa, S. The transparency and reproducibility of systematic reviews in forensic science. Forensic Sci. Informing Rainfall Risk Insurance for Farmers in Gujarat, India: Evidence Impact Summaries (3ie, 2021); https://www.3ieimpact.org/evidence-hub/Evidence-impact-summaries/strengthening-rule-law-and-justice-programming-usaid de Bruin, W. et al. Improving figures for climate change communications: insights from interviews with international policymakers and practitioners. Harold, J., Lorenzoni, I., Shipley, T. & Coventry, K. Communication of IPCC visuals: IPCC authors' views and assessments of visual complexity. Haddaway, N. et al. EviAtlas: a tool for visualising evidence synthesis databases. Yoeli, H., Dhital, R., Hermaszewska, S. & Sin, J. A meta-ethnography of participatory health research and co-production in Nepal. Social Sci. IPCC WGI Interactive Atlas (IPCC, 2024); https://interactive-atlas.ipcc.ch/ Campbell, R. et al. Evaluating meta-ethnography: systematic analysis and synthesis of qualitative research. Health Technol. Pawson, R., Greenhalgh, T., Harvey, G. & Walshe, K. Realist review: a new method of systematic review designed for complex policy interventions. Health Serv. Magnan, A. K. et al. The value of structured expert judgment to help assess climate adaptation. Risk Manage. Magnan, A. K. et al. Status of global coastal adaptation. Mitchell, D. et al. Expert judgement reveals current and emerging UK climate- mortality burden. Oxford Martin Restatements (Oxford Martin School, 2024); https://www.oxfordmartin.ox.ac.uk/publications/restatements Jordon, M. W. et al. A restatement of the natural science evidence base concerning grassland management, grazing livestock and soil carbon storage. The White/Wiphala Paper on Indigenous Peoples' Food Systems (FAO, 2021); https://doi.org/10.4060/cb4932eN Petzold, J., Andrews, N., Ford, J. D., Hedemann, C. & Postigo, J. C. Indigenous knowledge on climate change adaptation: a global evidence map of academic literature. Cohen, S. Mackenzie Basin Impact Study 327 (Atmospheric Environment Services, Environment Canada, 1997). Arctic Climate Impact Assessment (Cambridge Univ. Rashidi, P. & Lyons, K. Democratizing global climate governance? The case of Indigenous representation in the Intergovernmental Panel on Climate Change (IPCC). van Bavel, B., Petrasek Macdonald, J. & Sambo Dorough, D. in A Critical Assessment of the Intergovernmental Panel on Climate Change (eds De Pryck, K. & Hulme, M.) Ch. Hill, R. et al. Working with Indigenous, local and scientific knowledge in assessments of nature and nature's linkages with people. First Indigenous Knowledge and Local Knowledge Dialogue to Support the Preparation of the GEO-7 Assessment (UNEP, 2024); https://www.unep.org/events/online-event/first-indigenous-knowledge-and-local-knowledge-dialogue-support-preparation-geo Indigenous and Local Knowledge Dialogue Reports (IPBES, 2024); https://www.ipbes.net/ilk-dialogue-reports We thank B. van den Hurk who commented on earlier versions of the paper. The work of J.D.F. was supported by UK Economic and Social Research Council (grant number ES/Z000114/1) and an ERC Advanced Grant (via the UKRI Horizon Europe guarantee scheme, EPSRC grant number EP/Z533385/1). The work of R.B. was supported through the Dutch Research Council (NWO grant number VI.Vidi.211.132). The work of J.C.M. was supported by the European Research Council (ERC) under the European Union's Horizon 2020 Framework Programme as part of the project ‘GeoEngineering and NegatIve Emissions pathways in Europe' (GENIE) (grant agreement number 951542) as well as the project ‘Pathfinder 2: accelerating climate action for health' funded by the Wellcome Trust (project-ID 227165/Z/23/Z). Priestley Centre for Climate Futures, University of Leeds, Leeds, UK James D. Ford, Lea Berrang Ford & Jan C. Minx Public Administration and Policy, Wageningen University & Research, Wageningen, the Netherlands Robbert Biesbroek Centre for Climate & Health Security, UK Health Security Agency, London, UK Lea Berrang Ford Technische Universität Berlin, Berlin, Germany Science Policy Research Unit (SPRU), University of Sussex Business School, University of Sussex, Brighton, UK Department of Environmental Economics and Policy, Potsdam Institute for Climate Impact Research, Potsdam, Germany Felix Creutzig, Jan C. Minx & Max Callaghan School of Public Health, University of Alberta, Edmonton, Alberta, Canada Department of Environmental and Geographical Science, University of Cape Town, Cape Town, South Africa African Climate and Development Initiative, University of Cape Town, Cape Town, South Africa Climate Policy Radar, London, UK Intercultural Citizenship and Indigenous Health Unit (UCISI), Universidad Peruana Cayetano Heredia, Lima, Peru You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceptualized the Perspective. wrote the original and final draft of the paper. conducted analysis for Fig. All authors provided input on the text. Correspondence to James D. Ford. The authors declare no competing interests. Nature Climate Change thanks Robert Kopp and Debora Ley for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions et al. Recommendations for producing knowledge syntheses to inform climate change assessments. Accepted: 02 May 2025 Published: 10 June 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Climate Change © 2025 Springer Nature Limited Sign up for the Nature Briefing: Anthropocene newsletter — what matters in anthropocene research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59596-3'>Iterative transcription factor screening enables rapid generation of microglia-like cells from human iPSC</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 09:57:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Differentiation of induced pluripotent stem cells (iPSCs) into specialized cell types is essential for uncovering cell-type specific molecular mechanisms and interrogating cellular function. Transcription factor screens have enabled efficient production of a few cell types; however, engineering cell types that require complex transcription factor combinations remains challenging. Here, we report an iterative, high-throughput single-cell transcription factor screening method that enables the identification of transcription factor combinations for specialized cell differentiation, which we validated by differentiating human microglia-like cells. We found that the expression of six transcription factors, SPI1, CEBPA, FLI1, MEF2C, CEBPB, and IRF8, is sufficient to differentiate human iPSC into cells with transcriptional and functional similarity to primary human microglia within 4 days. Through this screening method, we also describe a novel computational method allowing the exploration of single-cell RNA sequencing data derived from transcription factor perturbation assays to construct causal gene regulatory networks for future cell fate engineering. Recent advances and applications of single-cell assays, exemplified by collaborative efforts such as the Human Cell Atlas (HCA)1, have begun to provide a comprehensive view of cell types and cellular states within the human body. Such maps are crucial for understanding tissue heterogeneity during human development and disease. Furthermore, these maps can be mined for promising targets for cell fate engineering, with significant implications for disease modeling, cell therapy, and regenerative medicine. Previously, our lab generated the first comprehensive library of 1732 human transcription factor (TF) isoforms and developed an unbiased screening approach to discover 290 TFs that differentiate human induced pluripotent stem cells (iPSCs) into various functional cell types2. This approach illustrates the power of unbiased screening and led to the discovery of differentiation protocols for several cell types; however, creating a user-specified cell type of interest remains an open area for innovation. To this end, we developed a novel screening platform that enables the identification of TF combinations for differentiating iPSCs into any given cell type using a combination of pooled transfections of a barcoded TF library, single-cell RNA sequencing (scRNA-seq) with TF ranking (Fig. As proof of principle, we sought to determine TF combinations capable of producing microglia in an efficient, rapid, and reproducible fashion and perform a comparative analysis to other differentiation methods. a Workflow of the first pooled TF screen. Two independent transductions were performed for the first pooled TF screen. This panel is generated using BioRender under an academic license. b Flow cytometry analysis of stem cell (TRA-1-60) and microglia (P2RY12, CD11b, CX3CR1) proteins in the PGP1 + 40 TF pool before and after Dox induction from two independent pooled TF transductions. c Representative image of cells with low TRA-1-60 expression in the Dox+ group to be sorted for scRNA-seq. d UMAP clustering of two independently transfected and differentiated PGP1 iPSC pools. Colors represent clusters identified by Seurat at 0.3 resolution. e Expression of microglia (ITGAM, CX3CR1, TMEM119, P2RY12, TREM2) and spiked-in stem cell (POU5F1) genes in scRNA-seq. f Primer designs for co-amplification of TF and cell barcodes in 10x Genomics 3' workflow. This panel is generated using BioRender under an academic license. g Number of TFs per cell counted from normalized and binarized TF expression matrix. h Ranking of the 40 TFs after one-sided Wilcoxon rank sum test with the two tested groups being with or without microglia gene expression. i Flow cytometry validation of a single transduction of top-ranking TFs for inducing microglia protein expression. “Pool” refers to simultaneous transfection of multiple single-TF plasmids, no polycistronic cassette used. Raw and processed data are provided in GEO accession GSE287847. Research on microglia has long relied on rodents, immortalized cell lines, or the isolation of primary microglia from human brain biopsies. However, many critical transcriptomic and functional differences exist between rodent versus human microglia, making them unsuitable for disease modeling and drug testing3,4,5. Using small molecules to differentiate iPSCs into microglia has greatly improved the availability of human microglia for in vitro studies. However, these protocols involve formulating complex cocktails of small molecules and growth factors, require extended differentiation periods, and sometimes necessitate co-culturing with neurons6,7,8,9,10,11,12,13,14. Current TF-based microglia differentiation protocols are largely based on literature search, and thus have been restricted to a limited set of known TFs15,16. In contrast, our method systematically and iteratively tests a wide array of TF candidates to identify an optimal set of TFs that effectively yield a desired cell identity. Additionally, variability in microglia yield and quality across different iPSC lines remains a significant bottleneck. While our previous studies have demonstrated the reproducibility of this approach across three distinct iPSC lines2, we have chosen to use a single founder line for screening and TF identification to enhance efficiency. In this report, we conduct two sequential iterations of pooled TF screening. Each screening round involves creating a barcoded TF library, performing pooled transfection into iPSCs to induce differentiation, and subsequently analyzing the results with scRNAseq. By ranking TFs based on their ability to drive microglial gene expression, we identify and validate a combination of six TFs. Overexpressing these six TFs in iPSCs enables faster generation of microglia-like cells (TFiMGLs) within four days without compromising cell identity. In addition to shared transcriptional and molecular signatures, we confirm that these cells exhibit key functional features of human primary microglia and other TF-based microglia. Notably, we achieve this differentiation process in standard culture media without the need for additional factors, offering a potential avenue for defining media compositions for further microglia specification in the future. Our barcoding and amplification strategy allows for high-throughput, simultaneous detection of single-cell gene expression and TF barcodes, which provides a template for others seeking to discover TFs for cell fate programming. Finally, we use the TF-perturbation data we generated in a stepwise regression model to construct gene regulatory networks (GRNs) from pooled TF screens. This perturbation-based approach to the construction of regulatory relationships offers unique insight from inferential GRNs, thus providing a computational framework for using high-throughput screening data to better understand regulatory relationships for future cell engineering. To identify TFs that differentiate iPSCs to microglia, we first surveyed previous literature on microglial development17,18,19,20,21, epigenetic, and transcriptomic patterns3,22,23,24,25, and gene regulatory networks26 and shortlisted 40 TFs for the first pooled TF screen (Supplementary Data S1). To ensure genomic integration of several TF copies we cloned each TF into the pBAN2 vector for integration with PiggyBac transposase and doxycycline (Dox)-inducible expression. Moreover, to distinguish between exogenous and endogenous TF transcripts, we added a 20-nucleotide (nt) barcode between the stop codon and the poly-A sequence of each TF (Supplementary Fig. We then transfected the 40 TF vectors into 600,000 hiPSCs from a healthy donor (PGP1) in duplicates with mass ratio of 4:1 between TF and transposase DNA. We added different amounts of DNA during TF nucleofection and quantified copy numbers of PiggyBac vector genomic integration. We determined that a DNA dose of 5 µg was most consistent for integrating a single digit copy number of at least 5 TFs per cell (Supplementary Fig. After puromycin selection for TF-integrated cells, we induced differentiation by Dox treatment for four days. Fluorescent activated cell sorting (FACS) of two separate differentiations showed that 0.3-0.5% of the cells expressed the consensus microglial surface proteins CX3CR1, P2RY12, and CD11b (Fig. In addition, we found that 30% of the cells lost expression of the stem cell marker, TRA-1-60 (Fig. To pinpoint which of the 40 TF(s) induced microglial gene expression, we sorted all differentiated cells (TRA-1-60 negative) for scRNA-seq. Then, we performed scRNA-seq on 10,000 cells, reasoning that since not all TF combinations are divergent or functionally unique, this quantity would provide sufficient information for initial screen and TF prioritization (Fig. We performed two independent transfections (Supplementary Fig. S3) and included a 10% spike-in of non-induced hiPSCs in each replicate as an undifferentiated control during scRNA-seq. After observing a consistent differentiation pattern between the two replicates (Fig. 1d), we pooled the data for downstream analysis. We observed expression of microglia genes (ITGAM, P2RY12, CX3CR1, TMEM119, TREM2), separating from a cluster of cells with high expression of POU5F1, a marker for stem cells (Fig. We quantified the expression of exogenous TFs through amplicon sequencing of co-amplified TF and cell barcodes from cDNAs (Fig. An average of 6.9 TFs (median 6, first quantile 4, third quantile 9) were expressed per cell while 877 (8.5%) out of 10285 single cells lacked TF expression, consistent with the 10% stem cell spike-in (Fig. We compared TF expression levels in cells with or without microglial RNA expression and identified the three TFs -SPI1, FLI1, and CEBPA- to most likely induce microglial gene expression (Fig. SPI1, which encodes PU.1 protein, is a known TF required for microglia development20,21. CEBPA is a known critical regulator for myeloid differentiation27. Although FLI1 has not yet been published for microglial differentiation, it has been reported to interact with TFs RUNX128 and SPI129, which are both indispensable for tissue-resident macrophage development30. We further tested whether these TFs could lead to microglial differentiation individually, or in combination. Individual expression of CEBPA and FLI1 in hiPSCs led to almost complete cell death observed by microscopy, while expression of SPI1 alone was insufficient for differentiation, as only 3% of all cells showed CD11b induction (Supplementary Fig. Thus, we tested various combinations of these TFs. Pooled transfection of CEBPA + FLI1 (“C + F pool”) and CEBPA + SPI1 (“C + S pool”) led to improved microglial marker expression, while CEBPA + FLI1 + SPI1 (“C + F + S pool”) produced the most positive cells, reaching 14% CD11b+, 54% P2RY12+ after four days (Fig. However, we did not observe expression of CX3CR1, a chemokine receptor important for microglia activation and migration31,32. We hypothesized that this could be due to incomplete integration of different TFs. Therefore, to ensure integration and expression of all three TFs, we generated polycistronic expression cassettes by linking the TFs with 2 A peptides. Therefore, we arranged the TFs in different orders (Supplementary Fig. Transfection and induction of the MG3.1-CFS and FCS cassettes both led to almost complete cell death by day 4, consistent with the previous observations of sole CEBPA and FLI1 expression causing cell death. The difference in CD11b and P2RY12 expression between cells derived using the MG3.1-SFC cassette and the C + F + S pool is potentially due to different TF dosages. While individual cells within the C + F + S pool may have expressed variable dosage combinations of the three TFs, MG3.1-SFC likely induced a fixed dosage ratio for all cells. Critically, the lack of CX3CR1 expression and the low percentage of CD11b- and P2RY12-positive cells from all 3-TF conditions indicated that additional TFs were needed for efficient microglia differentiation from hiPSCs. To identify TFs to supplement SPI1, FLI1, and CEBPA, we pursued a second iteration of our screen. To determine what TFs should be included in the second pool, we performed bulk RNA-seq analysis of MG3.1-SFC and compared it with published data on human primary microglia (GSE89189, GSE99074)7,22. Based on differential gene expression analysis using DESeq236, we first picked 25 TFs included in the first pool that showed lower expression levels in MG3.1-SFC than primary microglia. We then included six additional TFs that showed significantly higher gene expression in primary microglia. We also included IRF2 and ELF1 since these TFs showed the highest enrichment in downregulated genes in MG3.1-SFC when using Molecular Signatures Database (MSigDB)37 regulatory target gene sets. Additionally, using CellNet38, a computational tool that can classify bulk transcriptomic data and predict missing gene regulators, we added six more TFs to the pool. Lastly, following a recent single-cell study on fetal microglia development39, we added SPIB, ETS1, and ELK3. In summary, the second TF pool contained a total of 42 TFs (Supplementary Data S2). a Workflow of the second pooled TF screen performed in duplicate. This panel is generated using BioRender under an academic license. b Polycistronic cassette design for performing dual-drug selection to achieve 3 + X TF screen. c Normalized mRNA expression from the polycistronic cassette (SPI1, FLI1, CEBPA) and stem cells (POU5F1) from two independent TF transductions. d TF barcode counting enabled the identification of stem cells (“No TF BC”), MG3.1-SFC, and cells with additional TFs (“SFC + X”). e Example histograms of TF barcode raw counts in single cells. f Number of TFs per cell counted from normalized and binarized TF expression matrix. g Ranking of the 42 TFs after one-sided Wilcoxon rank sum test with the two tested groups being with or without microglia gene expression. Grey highlights the SFC polycistronic cassette. h Flow cytometry validation of a single transduction of the top-ranking TFs for improving microglia protein expression. i Polycistronic cassette designs for varying TF orders. This panel is generated using BioRender under an academic license. Raw and processed data are provided in GEO accession GSE287850. To ensure each cell expressed both the SPI1-T2A-FLI1-P2A-CEBPA cassette and additional TFs from the second pool, we cloned the SFC cassette into a bleomycin-resistant vector and the new TF pool into a puromycin-resistant vector (Fig. We transfected 600,000 PGP1 hiPSCs in duplicates and performed dual-drug selection. After four days of TF expression, we performed the same process of scRNA-seq and TF barcode amplicon sequencing as in the first iteration (Supplementary Fig. As controls, we spiked in 5% undifferentiated hiPSCs and 10% MG3.1-SFC during single-cell encapsulation to mark the differentiation starting point of two iterations. We applied this approach to two independent pools of cells. When we analyzed TF barcodes in this experiment, we observed two clusters of cells on UMAP that corresponded to (1) hiPSCs (POU5F1 positive, no TF barcode) and (2) MG3.1-SFC along with other cells expressing additional TFs (Fig. Upon counting TF barcodes, we observed that out of the total 8051 single cells from two independent transfections, 284 (3.5%) cells had no TF barcode, and 613 (7.6%) cells had only the barcode for MG3.1-SFC. S9), with most cells (88.9%) expressing the SFC cassette plus at least one other TF. To determine which of the new TFs lead to improved microglia differentiation, we analyzed their effects on microglial gene expression. We observed significantly higher (p < 0.01) number of MEF2C and KLF6 barcode in cells expressing CX3CR1 (Fig. 2g), suggesting their ability to induce CX3CR1 expression. We also noted that MEF2C was also present in the first screening but failed to reach significance for upregulating CX3CR1, indicating that the use of the SFC cassette as baseline enabled other influential TFs to be found. MEF2C and BHLHE41 also reached high ranking for TMEM119 (Fig. In this round of screening, CEBPB and IRF8 emerged as high-potential TFs for promoting ITGAM or P2RY12 expression (Fig. From this second pooled TF screening, the additional TFs of interest found were MEF2C, CEBPB, IRF8, KLF6, and BHLHE41. To validate that these additional TFs can promote microglial gene expression, we individually expressed them in addition to SFC (SFC + 1). When compared with MG3.1-SFC, SFC + CEBPB increased the percentage of CD11b+ cells from 37% to 98% (Fig. 2h) but led to more cell death at day 4. These results corresponded well with the predictions from single-cell TF barcode analysis, indicating the validity of using pooled TF screening for inferring causality between TF and target gene expression. To test if microglia differentiation could be further promoted by delivering more TFs to each cell, we chose the three TFs from the SFC + 1 experiment that led to the highest increase in percentage of microglial gene-expressing cells, CEBPB, IRF8, and MEF2C, to add to the SFC set. We combined MEF2C, CEBPB and IRF8 into polycistronic cassettes. We also varied the position of FLI1 and CEBPA in the first construct to produce SFC and SCF, keeping SPI1 in the front to avoid excessive cell death during differentiation. We tested all four combinations of the two 3-TF cassettes (SFC-MIC, SFC-MCI, SCF-MIC, SCF-MCI) for their ability to induce microglia differentiation (Fig. Encouragingly, all 6-TF cocktails produced cell pools with increased expression of microglial proteins when compared with MG3.1-SFC (Fig. We observed that the most effective combination was MG6.4-SCF-MCI, resulting in 66% CD11b+, 93% P2RY12+ and 16% CX3CR1+ cells at day 4, compared with 37%, 6%, and 0% respectively for MG3.1-SFC, the baseline of the second iteration. To determine the differentiation dynamics of TFiMGLs, we performed bulk RNA-seq analysis of the cells on days 0, 1, 2, 3, 4, 6 post TF induction. We observed a rapid induction of all six TFs accompanied by a quick downregulation of POU5F1 on day 1 (Fig. TF induction reached a plateau on day 2, aligning with the sustained upregulation of microglial genes from that point onward. Moreover, we assessed how accurately the TFiMGLs transcriptome recapitulates that of human microglia, by comparing TFiMGLs bulk-RNA-seq data to previously published bulk RNA-seq data from human primary microglia, iPSC-derived microglia (GSE89189, GSE99074) and TF-based microglia differentiation protocols3,7,15,16,22. To minimize potential batch effects that might hinder meaningful comparison between datasets before downstream analysis, we aligned all raw FASTQ files to the same reference genome and applied a negative binomial regression-based batch effect correction method, ComBat-seq40. The PCA shows that the TFiMGLs transcriptomes from days 2–6 are more similar to primary microglia from various sources than to either iPSCs or hematopoietic progenitors (HPCs), suggesting successful microglial fate induction (Fig. Additionally, we observed that TFiMGLs were distinct from monocytes or dendritic cells, two related cell types from the myeloid lineage. Importantly, Gene Set Enrichment Analysis (GSEA)41 on TFiMGLs versus iPSCs using two microglial gene collections from the MSigDB database derived from human brain scRNA-seq (M40168; M39077), revealed significant enrichment scores with both gene collections (M40168: score = 0.72, p-value = 9.01e−10, gene set size = 313; M39077: score = 0.66, p-value = 9.01e−10, gene set size = 391), demonstrating successful derivation of microglia-like cells (Fig. To further investigate transcriptomic similarity of TFiMGLs to primary microglia, we assessed gene clustering after adding a previously published collection of 881 microglia-enriched genes3. The transcriptomes of day-1 TFiMGLs clustered closer to iPSCs while from day-2 onwards the upregulation of key microglial genes resulted in closer clustering with primary microglia, distinct from monocytes or dendritic cells (Supplementary Fig. These results demonstrate that induction of SPI1, CEBPA, FLI1, MEF2C, CEBPB, and IRF8 in iPSCs for 2 days is sufficient to obtain cells with a transcriptional profile characteristic of microglia. a Expression of the six induced TFs over time measured by bulk RNA-seq (n = 2 for each day). b Expression of stem cell (POU5F1) and microglia (ITGAM, CX3CR1, TMEM119, P2RY12, TREM2) genes over time (n = 2 for each day) measured by bulk RNA-seq. d PCA of bulk RNA-seq data from multiple sources containing primary microglia. MG Microglia, DC dendritic cell, HPC hematopoietic progenitor, iMGL growth factor-induced microglia-like cell, Mono monocyte. e GSEA of TFiMGLs versus iPS using two microglia marker gene sets from MSigDB: M40168 and M39077. Raw and processed data are provided in GEO accession GSE287851. Further survival and maturation of TFiMGLs might require adapting the culture media composition. Thus, we conducted a time course study to assess transcriptional profiles of TFiMGLs under two different cell culture conditions. In the first condition, TFiMGLs were induced following the conditions of the TF screen—in short, cells were continuously cultured in mTeSR Plus (+MT) media supplemented with Dox for 6 days. Whereas in the second condition, mTeSR Plus media was switched on day 2 to commercially available microglia-specific media (+MG) supplemented with Dox until day 6. Cells grown in both conditions were collected at days 2, 4, and 6 post induction for scRNA-seq, allowing us to explore the impact of the media compositions on TFiMGL development. Visualizing TFiMGLs gene expression patterns using UMAP revealed two distinct clusters. By color coding cells based on the sample condition, we observed that TFiMGLs at 2 days post induction (D2) localized to a single cluster, whereas all other conditions localized to the second cluster (Fig. TFiMGLs at 4- or 6-days post induction grown in either +MG or +MT did not show a discrete separation on the second cluster suggesting that media composition has limited effects on RNA expression patterns at such early induction time points. In addition, plotting a correlation matrix of scRNA-seq data revealed a considerably high correlation between D4 and D6 despite culturing in different media (Supplementary Fig. We compared differentially expressed genes (log2 fold change ≥ 2, p value ≤ 0.05) from each TFiMGL sample versus all other samples and used the Enrichr search engine42,43 to query annotated cell-type gene sets from other RNA-seq or scRNA-seq databases (Fig. Importantly, all TFiMGL samples revealed microglia as the top significantly enriched cell type using the scRNA-seq reference databases Descartes44 and Cell Marker Augmented45, suggesting that these specific media compositions cannot modify TFiMGL identity between days 2 to 6 post induction. Since P2RY12 and CX3CR1 are consensus microglial markers, we further sub-sampled P2YR12 or CX3CR1 positive cells from our scRNA-seq data to determine the proportion of cells expressing these markers and the identity of those cells. Interestingly, we observed an increase in both marker's expressions from D2 to D4 post induction; however, this increase was only maintained until D6 on the +MT condition while the +MG condition reverted to levels comparable to D2 (Fig. When tracing both sub-sampled populations back onto UMAP space, we observed that the P2RY12+ population was evenly dispersed in both clusters, whereas the CX3CR1+ population was slightly more biased towards the D4 and D6 cluster (Fig. We performed gene enrichment analysis for each sub-sampled population using Descartes as a reference dataset which revealed that both sub-sampled populations displayed a significant microglial cell type enrichment from various brain regions (Fig. a UMAP clustering of a single TFiMGL induction at 2-, 4-, and 6-days post induction switching to either stem cell media or microglia media from day 2. Colors represent clusters identified by day and media condition. Enrichment p-value was computed using Fisher's exact test or the hypergeometric test and adjusted p-value was computed using the Benjamini-Hochberg method for correction for multiple hypotheses. d Proportion of cells per TFiMGL condition expressing microglia marker gene P2RY12. e Normalized mRNA expression of P2RY12 on overall TFiMGL population. g Proportion of cells per TFiMGL condition expressing microglia marker gene CX3XR1. h Normalized mRNA expression of CX3CR1 on overall TFiMGL population. j Mean expression levels of known pan-microglial and mature human microglia genes from different TFiMGL condition (rows). Microglial marker genes were obtained from previously published studies. Genes directly overexpressed are outlined in red. k–m Mean expression levels of (k) proliferation-associated microglia, (l) neural-associated microglia, and (m) immune-associated microglia transcriptional signatures found in various regions of the developing human brain. Raw and processed data are provided in GEO accession GSE287852. In addition, we curated gene marker lists from previously published studies involving in vitro microglia cultures3,46 as well as mature human microglia47 and determined gene expression overlap of our transcriptomic dataset for each TFiMGL condition and timepoint (Supplementary Data S8). All TFiMGL conditions exhibited high expression of ‘human microglia' genes, including ENTPD148, a purinergic receptor modulating immune responses and inflammation, ELMO149, a key regulator of microglial function, and NEAT150, an immune response and inflammatory process regulator in microglia. However, low expression was observed for other microglial genes, such as GPR34, a receptor mediating microglial chemotaxis and neuroinflammation, ADRB2, an immune response regulator via adrenergic signaling, and CSF1R, a receptor critical for microglial survival, differentiation, and proliferation. This might suggest conversion to microglia of different activation stages or subtypes of microglia (Fig. Different timepoints and media conditions had moderate effects on expression of B2M and HLA-B but didn't change gene expression patterns drastically. To further validate microglia diversity, we identified and visualized collective gene expression of previously reported, defined microglia signatures (Supplementary Data S9) within our scRNA-seq data set (Fig. The cluster comprising the day 2 TFiMGL subpopulation showed enriched gene expression signatures associated with proliferating microglia (Fig. 4k), while neural associated microglia gene signatures were spread throughout the differentiation time points and media conditions (Fig. Notably, immune associated microglia gene expression patterns are mostly found in TFiMGLs at days 4 and 6 cultured with mTeSR1 media (Fig. Overall, TFiMGLs show clear microglia identity and display a variety of different microglia signatures depending on maturation and irrespective of media composition. Since TFiMGL transcriptomes on day 4 displayed mature microglia identity, we chose this timepoint for functional characterization of TFiMGLs. Additional brightfield microscopy analysis of TFiMGLs confirmed rapid morphological change from day 1 to day 6 (Supplementary Fig. Since microglia respond to signals associated with brain infection and inflammation in vivo, we assessed the ability of TFiMGLs to respond to various stimuli. Specifically, we tested their response to interferon gamma (IFNγ), beta amyloid (Aβ), and TDP-43. IFNγ, secreted by T lymphocytes, is a known microglial activator51. Aβ has been shown to elicit microglial response in the context of Alzheimer's Disease (AD) pathology52, while TDP-43 aggregation, a hallmark of amyotrophic lateral sclerosis (ALS) pathology, has also been shown to activate microglia53. To investigate how TFiMGLs respond to IFNγ, fibrillar Aβ (fAβ), and TDP-43, we treated TFiMGLs in triplicates with each of the three molecules for 24 h and harvested cells for RNA-seq. PCA analysis revealed transcriptomic changes in the IFNγ and TDP-43 treated group, while the fAβ-treated group showed minimal differences (Fig. We confirmed fAβ formation by conducting an in vitro amyloid fibrillation experiment that showed the Aβ peptide could form fibrils after 1 h of incubation (Supplementary Fig. Pathway analysis of differentially expressed genes from the IFNγ treated group included “response to virus” and “response to bacterium” (Fig. Top upregulated genes by IFNγ included CXCL10, CXCL11, IRF1 and IL18BP (Supplementary Fig. S15), aligning with the IFNγ response genes revealed by an independent single-cell level human-derived macrophage stimulation study54. For the TDP-43 treated cells, top differentially regulated pathways included “myeloid leukocyte mediated immunity” and “myeloid cell activation involved in immune response” (Fig. S16), demonstrating that TFiMGLs were activated by the TDP-43 treatment. Collectively, these results suggest that TFiMGLs exhibited microglia-like responses to infection and ALS-related TDP-43 aggregation. a Representative immunofluorescence images of stem cell (OCT4), Dox-induced (PU.1), and microglia (CD11b, P2RY12, CX3CR1) proteins on day 4. b Flow cytometry quantification of microglia protein expression on day 4 (n = 3). c PCA of TFiMGLs transcriptome after 24 h treatment with IFNγ, fAβ, or TDP43. d, e Pathway analysis of significantly differentially expressed genes after treatment with IFNγ or TDP43. One-sided Fisher's exact test was used to identify enriched categories with FDR control using Benjamini–Hochberg procedure. g Microscopy analysis of particle uptake combined with microglia surface protein staining. Graphs shown are representative of three independent experiments with similar results. Images are taken once every 3 s. ADP was added at t0 Graphs shown are representative of two independent experiments with similar results. j Peak dynamics analysis shows a fast rise and slow decay pattern of the intracellular calcium concentration. Raw and processed data are provided in GEO accession GSE287853. To assess this phagocytic activity, we incubated TFiMGLs with pHrodo green labeled S. aureus particles for 0.5, 2, and 4 h in triplicate, and performed flow cytometry and microscopy. While 0.5-h incubation showed minimal phagocytic activity, nearly all cells were positive for pHrodo green at 2 h with fluorescence intensity reaching its peak at 4 h (Fig. Additionally, co-staining with microglial surface proteins CD11b, P2RY12, and CX3CR1 confirmed the intracellular localization of pHrodo green labeled particles (Fig. Injured neurons release adenosine diphosphate (ADP) and stimulate microglial responses55,56. To verify ADP response capabilities, we incubated TFiMGLs with the calcium indicator Fluo-4 AM and then stimulated them with ADP containing media. We imaged the cells at a three-second interval and observed a rapid increase in calcium signal following ADP exposure, suggesting that TFiMGLs are responsive to ADP stimulation (Fig. An important area of continuing progress in cell fate engineering is the derivation of a complete knowledge map for accurately predicting TF expression levels along with their respective downstream gene expression signatures. Currently, insight on this subject has been gained by leveraging computational methods for inferring TF-gene regulatory network (GRN)57,58 and databases based on TF-binding sites59,60, TF-gene co-expression61,62, and protein-protein interaction63,64. However, determining causal TF-gene regulatory relationships would require direct TF perturbations with downstream transcriptomics, and to do this at an omics scale requires the scaling of classic single-gene perturbation methods to a high-throughput one. In this context, the barcoding and amplification strategy presented in this study enables the readout of single-cell gene expression after multiplexed TF perturbation, and the ability to link transcriptomic results with TF barcode quantification of those individual cells. Thus, we utilized TF-perturbation data from the two pooled screens to develop a stepwise regression model to construct gene regulatory networks (GRNs) to better understand TF regulatory relationships for future cell engineering (Fig. In our dataset, each TF transgene was represented by two distinct values: counts from barcode amplicon sequencing, and counts from their RNA molecules in scRNA-seq. Although the counts from scRNA-seq might contain reads from endogenous TF expression, the two measurements correlated well for most TFs (Supplementary Fig. S17), the exception being a few TFs with low expression. We reasoned that TFs that had a higher correlation between their barcode and RNA measurements demonstrated higher consistency between experiments, making them more reliable to produce accurate regression results using the two matrices. Thus, by screening TFs with a correlation coefficient greater than 0.3 between both measurements, we identified 18 TFs from the first iteration and 21 TFs from the second for regression analysis. We observed extensive gene expression changes caused by CEBPA and the triple TF cassette MG3.1_SFC expression (Fig. Among the 672 edges, we observed 76% of them to be positive regulations. Some TFs (CIITA, SPI1, JUN) only showed positive edges in current thresholding conditions (Abs(coefficient) >0.1 & −log10(p-value) >20), indicating they were mostly activating other genes. Other TFs (CEBPB, ZFP36, BHLHE41) showed negative edges, indicating their repressive roles. We also observed several genes simultaneously connected with more than one TF (Supplementary Fig. For example, RAB13, a membrane trafficking regulator, was upregulated by both CEBPA and CEBPB; HMGA1, a master regulator of chromatin structure, was downregulated by both BHLHE41 and CEBPA; FLNC, an actin crosslinking protein, was upregulated by JUN while downregulated by CEBPA. There are many more regulatory relationships we listed in detail from these two pooled screens (Supplementary Data S10, 11). Although we have created a firm foundation, with the aid of larger perturbation libraries, higher-throughput scRNA-seq, and more scalable regression analysis methods, it should be possible to build a complete knowledge map of causal TF-gene regulatory relationships in the near future, greatly facilitating cell fate engineering efforts. Edges were selected with these criteria: Abs(coefficient) >0.1 and −log10(p-value) >20. F-test was used to determine if any of the independent variables were significant in the regression model. a Global network for the first pooled screen. b Global network for the second pooled screen. Sub-network for (c) CIITA, (d) JUN, (e) SPI1, (f) CEBP3, (g) ZFP36, (h) EGR2. The ability to obtain highly specialized cell types that are not routinely accessible through biopsies is essential to understanding basic cellular mechanisms, molecular disease drivers, and therapeutics development. While engineered expression of transcription factors (TFs) identified through TF screens in iPSCs can rapidly and efficiently produce some cell types, the generation of other cell types that require complex combinations of TFs has been difficult. Here, we addressed this challenge by combining an iterative genetic library screen in hiPSCs with high-throughput scRNA-seq for cell fate engineering and validated this approach by generating microglia-like cells. We conducted two iterations of our design-screen-validate workflow and discovered that the combination of TFs SPI1, CEBPA, FLI1, MEF2C, CEBPB, and IRF8 effectively drives microglia differentiation from hiPSCs within 2 days, significantly reducing the standard 35 days through growth factor-based protocols11. Characterizations of TFiMGLs demonstrated transcriptomic and functional resemblance to primary human microglia. We further uncovered TF-gene regulatory relationships in these pooled screens through genome-wide regression analysis between TF barcode counts and gene expression levels. During this study, we noted several technological challenges that could be addressed in future studies. Most current TF-based differentiation protocols rely on a one-time induction of TF expression, while lacking the capability for sequential induction. Despite this challenge, current strategies have successfully generated certain cell types, including multiple types of neurons65, endothelial cells2, and the induction of iPSCs themselves66. However, during development in vivo coordinated gene programs are sequentially activated, as observed in time-resolved transcriptomic analysis of developing tissues67. This feature could be re-created by identifying orthogonal induction system with a comparable strength to the doxycycline-inducible system or developing tunable gene circuits. With these tools, it will be possible to test whether sequential TF expression can lead to improved differentiation accuracy. In addition to temporal control of TF expression, the ability to regulate expression levels of individual TFs could also lead to improvements in differentiation. There were two manifestations of this pattern in the current study. In the first case, although CEBPA and FLI1 expressed individually led to cell death, their presence in the SFC cassette enabled cell survival and differentiation. The reduced expression levels of CEBPA and FLI1 and their potential interactions with other TFs could also explain why we were able to observe an extensive GRN for CEBPA on the first pooled screen, which would not be possible due to toxicity in CEBPA-expressing cells. The second case can be observed in the effect that different sequential arrangements of TFs in the polycistronic cassettes led to different levels of downstream microglial protein expression. The effect of TF stoichiometry on differentiation efficiency has also been observed for cardiac myocyte programming68 and was shown to influence iPSC reprogramming34,35. While the positional effects in a polycistronic cassette offer one way to explore the stoichiometry space, the development of new titratable promoters allowing the turning of individual genes could be an important tool for cell fate engineering. Establishing a specialized cell fate from stem cells requires orchestrated interactions between many genes. Because the combinatorial space of the TFome can quickly become astronomical and cost-prohibitive, especially when scRNA-seq is involved, we decided to prioritize a small subset of the TFome based on existing literature evidence for microglia development and differentiation. It is impossible to pre-determine how many TFs will be necessary to achieve effective differentiation. While having a large number (>10) of TFs overexpressed in a single stem cell could increase the chance of having the correct combination, it also raises the risk of biological noise, making the screening results less interpretable. With this in mind, we aimed for approximately five TFs per cell and adopted an iterative screening strategy. Although using fewer TFs per cell reduces the potential to discover the complete set of necessary TFs in a single screen, this issue can be mitigated by identifying top TFs from each round of screening and keeping them constant in subsequent screen iterations. We believe this stepwise approach gives us a better chance of disentangling the complex gene regulatory network needed for establishing a specific cell fate. Microglia are resident immune cells of the brain which originate from erythro-myeloid progenitors (EMPs) in the yolk sac17,69. Recent studies have demonstrated the link between neuroinflammation and neurodegenerative disease, such as Alzheimer's Disease (AD)73,74, and microglia have been shown to be an important cell type in AD and other neurodegenerative diseases75,76,77,78. However, functional studies to define microglia-targeting therapeutics have been greatly hindered by the limited availability of human brain biopsies79,80. This supply issue cannot be simply mitigated by using murine models given that interspecies differences limit the transferability of knowledge4,5. Various research groups have developed human microglia-like cells from iPSCs in efforts to fill this gap. Several studies have accomplished this goal through a process of embryoid body formation, growth factor treatment, and, in some cases, co-culturing with neurons6,7,8,9,10,11,12,13,14. These protocols draw inspiration from the natural developmental stages of microglia and have timelines ranging from 30–74 days. More recently, a study showed microglia differentiation through forced expression of both SPI1 and CEBPA16. Comparison of RNAseq datasets showed similarities between SPI1/CEBPA16, PU.1/MAFB/CEBPα/CEBPβ/IRF5/IRF815 induced iPSCs and TFiMGLs at early time points, however at later time points TFiMGLs showed closer similarity to microglia derived through longer iPSC differentiation processes involving hematopoiesis and subsequent microglia differentiation7. This suggests that through the induction of additional TFs, we leveraged complex TF-gene regulatory relationships which altered the overall developmental trajectory of TFiMGLs. In the future, the identity of TFiMGLs could be more precisely defined by optimizing culture conditions and co-culturing with neural cells or organoids. This approach would enhance the transcriptional cascades triggered by TF overexpression, enabling a deeper investigation into their similarity to in vivo microglia under various conditions. Differentiating human cell types from stem cells is essential for basic research and therapeutics development, especially when the desired cell types are not easily obtainable from accessible human tissues. Advances in the understanding of developmental biology have fueled the discovery and application of protocols to differentiate specific cell types from iPSCs. Some of this work has been translated into treatment strategies that are now being investigated with clinical trials for devastating diseases like age-related macular degeneration81 and type 1 diabetes82. Differentiated iPSCs have now also become routinely used in laboratories for studying disease mechanisms and testing drugs83. Recent global efforts on building single-cell atlases of cellular development have expanded the knowledge of human development and diseases and also present a key resource for cell fate engineering. Combining technological advancements in genetic library construction with high-throughput screening and sequencing technologies, this work lays the groundwork for the developmental biology field to engineer cell fates in a more systematic and multiplexed fashion. For expression in hiPSCs, a PiggyBac integrating Dox-inducible vector pBAN2 was used. To create barcoded pBAN expression vector (pBAN-BC), the original pBAN was digested with AgeI and KpnI, followed by ligation of a gBlock (IDT DNA) containing the same excised piece with an additional 20-bp random barcode. After bacteria transformation, individual colonies were expanded and extracted for plasmid DNA. Barcode sequence for each TF was confirmed by Sanger sequencing. The human male iPSC line GM23338 was obtained from Coriell. No commonly misidentified cell lines were used in the study. hiPSCs were cultured in mTeSR Plus media (Stemcell Technologies, 100-0276) on multi-well plates coated with Matrigel (Corning, 354277) or Cultrex (Bio-Techne Corporation, 3434-005-02). For passaging, cells were dissociated with TrypLE Express (Life Technologies, 12604013) and seeded into freshly coated plates and media containing 10 µM Y-27632 ROCK inhibitor (Millipore, 688001) for 24 h. Daily media change was performed until cells were ready for another passaging or downstream experiments. TF (pBAN-TF-BC) and Super PiggyBac (SPB) Transposase (System Biosciences, PB210PA-1) expression vectors were mixed at a mass ratio of 4:1 and transfected into hiPSCs using P3 Primary Cell 4D-Nucleofector X Kit L (Lonza, V4XP-3024) on a 4D-Nucleofector X Unit (Lonza, AAF-1002X) following the manufacturer's instructions. For the two pooled TF screenings, 600,000 cells were transfected with 5 µg of DNA and seeded into one well of a 6-well plate. For individual TF combinations, 120,000 cells were transfected with 2.5 µg of DNA and seeded into one well of a 12-well plate. For PiggyBac dose curve of copy number quantification using qPCR, 600,000 cells were nucleofected with vectors containing puromycin- and bleomycin-resistance genes at total DNA concentrations of 1.25 µg, 2.5 µg, 5 µg, and 10 µg and SPB transposase was mixed at a mass ration of 4:1. Nucleofected cells were seeded into one well of a 6-well plate. Program CB150 was used for the nucleofections. Cells were passaged again when reaching 80% confluency. For induction of TF expression, cells were seeded into mTeSR Plus media containing 0.5 µg/mL doxycycline (Sigma-Aldrich, D3072) and 10 µM Y-27632 ROCK inhibitor and were changed into media only containing doxycycline after 24 h. iPSCs integrated with vectors containing Puromycin- and Bleomycin-resistance genes at concentrations of 1.25 µg, 2.5 µg, 5 µg, and 10 µg or with 5 µg of DNA of MG6.4-SCF-MCI TFs were harvested by first washing with PBS (- calcium, - magnesium) and then dissociating them into single cell suspensions with Accutase (StemCell Technologies, 07920) for 10 min at 37 °C. Cultures were then quenched with DMEM/F12 (Gibco, 11320033) and subsequently centrifuged at 300g for 3 min. The cell pellet was rinsed with PBS (- calcium, - magnesium), centrifuged at 300g for 3 min and flash frozen at −110 °C for PiggyBac copy number qPCR quantification. gDNA extraction was conducted using the PiggyBac Copy Number qPCR Kit (System Biosciences, PBC100A-1) following the manufacturer's protocol. The qPCR reaction mixes comprised 1x DyNAmo ColorFlash SYBR Green qPCR Mix (Thermo Scientific, F-416L), 840 nM primers, and 1 µL of gDNA (10-50 ng). Primers targeting rtTA, bleomycin resistance gene, and puromycin resistance gene were employed to amplify PiggyBac integrations, while UCR1 primers (System Biosciences) were utilized for the normalization to the cell genome copies. Quantitative PCR was performed using the QuantStudio 7 Flex system (Applied Biosystems) with thermocycling settings recommended by the PiggyBac Copy Number qPCR Kit. Calculations to determine integration copies were performed in accordance with the manufacturer's protocol. pBAN-Bleo-fwd  CAA GTT GAC CAG TGC CGT TC pBAN-Bleo-rev  ATG AAC AGG GTC ACG TCG TC For cytometry analysis, cells were dissociated with TrypLE Express for 5 min at 37 °C, diluted with twice the volume of Cell Staining Buffer (Biolegend, 420201) and centrifuged at 200 g for 3 min to remove the digesting enzyme. Cells were then incubated with 25 µg/mL of Human Fc Block (BD Biosciences, 564219) diluted in Cell Staining Buffer for 15 min on ice, followed immediately by staining with fluorescently conjugated antibodies or isotype controls at 5 µl per million cells in 100 µl staining volume for 30 min on ice. Antibodies were diluted in Cell Staining Buffer and Human Fc Block was not removed from the mixture. After antibody staining, cells were washed twice with Cell Staining Buffer before being put through 35 µm nylon mesh into a 5 mL round bottom polystyrene tube (Falcon, 352235). Flow cytometry data were acquired on a BD LSRFortessa Cell Analyzer. For cell sorting, the staining protocol was the same except that Cell Staining Buffer was replaced with mTeSR Plus media in order to maintain the best viability of cells. scRNA-seq experiments were performed using 10x Genomics Chromium Single Cell 3' Reagent Kits v3 or v3.1 following the manufacturer's instruction. 5000 single cells were calculated as targeted input for each sample. For the first iteration, 10% of stem cells were spiked in as undifferentiated control. For the second iteration, 5% stem cells and 5% MG3.1-SFC were spiked in as undifferentiated and initial differentiation control. The only modification made to the protocol was at the Sample Index PCR step, where 5 µL of the PCR mix was taken out and mixed with 0.5 µL 1000X SYBR Gold (Invitrogen, S11494) for a qPCR reaction. Final libraries were sequenced on NextSeq 500 or NovaSeq with a goal of at least 30,000 reads per cell. Because after the cDNA amplification step in the 10x scRNA-seq protocol the amplicons contained cell barcodes, UMIs, and TF barcodes, these cDNAs could be used as the template for further amplification of TF-cell barcodes. Two sequential PCR reactions were performed, each was accompanied by a SYBR Gold spike-in qPCR to determine the optimal cycle number as described in “scRNA-seq library preparation”. For PCR1, NGS10x-F-i7-BC-PCR1F and i5000 were used as primers. PCR1 reaction was purified with 1.2X SPRIselect beads (Beckman Coulter, B23318) following standard protocol. The sample was eluted in 20 µL water. For PCR2, i7000, P5, and P7 were used as primers. PCR2 product was purified the same as PCR1. Final libraries were submitted for MiSeq v3 with paired-end reads of 80 cycles from either direction. i5000: AATGATACGGCGACCACCGAGATCTACACaactcgctACACTCTTTCCCTACACGACGCTCTTCCGATCT (lower case region represents a sample-specific barcode) i7000: CAAGCAGAAGACGGCATACGAGATtcgccttaGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT (lower case region represents a sample-specific barcode) For scRNA-seq, raw FASTQ files were aligned to GRCh38 and quantified using Cell Ranger. Detailed information about cell number, read depth and gene detected is visualized in Supplementary Figs. Seurat was used to perform cell filtering, data normalization, and clustering. To remove duplicated reads from the same molecule, duplicated rows that have the same value for all three columns were removed. Then the table was counted and reshaped into a frequency table where the row names represent cell and column names represent TF. This table contains the raw counts of each TF barcode in all single cells. Because the TF barcodes were amplified from the cDNA during library preparation, we normalized the TF barcode count with the number of total RNA UMIs detected in each cell, reasoning that cells with more total UMIs were likely to have more reads for TF barcode. The raw gene expression matrix and normalized TF count matrix were used to identify which TF barcodes were likely to induce microglial gene expression. Specifically, the expression of microglial genes was binarized, with any cell had a non-zero expression being 1. Then between the two groups of cells 0 or 1 microglial gene expression, a one-sided Wilcoxon rank sum test was performed for all barcoded TFs to determine which TF(s) had a higher expression in cells expressing microglial genes. Cultured cells were dissolved directly with TRIzol (Thermo Fisher Scientific, 15596018) for total RNA purification with Direct-zol RNA MiniPrep Kit (Zymo Research, R2050). RNA integrity was confirmed by the presence of 18S and 28S bands on a 2% E-Gel EX Agarose Gel (Thermo Fisher Scientific, G402002). Between 100 ng to 1000 ng total RNA was used as input for mRNA enrichment using NEBNext Poly(A) mRNA Magnetic Isolation Module (New England Biolabs, E7490), followed by library construction with NEBNext Ultra II Directional RNA Library Prep Kit (New England Biolabs, E7760S) following the manufacturer's instructions. Biopolymers Facility at Harvard Medical School performed library QC and sequencing. For both in-house generated sample and datasets downloaded from GEO, raw FASTQ files were aligned to GRCh38 and quantified using STAR (version 2.5.2b). Regularized-logarithm (rlog) transformation was applied to the raw counts before visualization using PCA. For analysis where data from multiple sources were involved, ComBat-seq was used for batch correction before PCA. Differential gene expression analysis was conducted with DESeq236. Pathway enrichment and GSEA analysis were performed with clusterProfiler84. Bulk RNA-seq data generated and analyzed in this study have been deposited in Gene Expression Omnibus (GEO). TFiMGLs were fully cultured in either stem cell media, mTeSR Plus (MT), or switched to commercially available microglia media (MG) (Stemcell Technologies, 100-0020) from day 2 until day 6. Undifferentiated iPSCs and TFiMGLs at days 2-, 4-, and 6-post induction (both MT and MG media conditions for days 4 and 6) were harvested by first washing with PBS (- calcium, - magnesium) followed by dissociating cultures into single cell suspensions with Accutase (StemCell Technologies, 07920) for 10 min at 37 °C, cultures were then quenched with DMEM/F12 (Gibco, 11320033) and subsequently centrifuged at 300 g for 3 min. The cell pellet was rinsed with PBS (- calcium, - magnesium) and centrifuged again at 300 g for 3 min prior to fixation. Single cell suspensions from each sample were processed using the Parse Biosciences Fixation Kit following the manufacturer's instructions (SB1001) and subsequently stored at −80 °C until all samples of the time course were harvested. Libraries for all samples were prepared using the Parse Whole Transcriptome Mini V1 kit following the manufacturer's instructions (EC-W01010) where an average of 450 bp libraries were generated. Sequencing of libraries was carried out on an Illumina NextSeq 500 High Output v2.5 (300 Cycles) (sequenced at Biopolymers Facility - Harvard Medical School) using a single index (6 bp) and 5% PhiX spike-in as per Parse Biosciences sequencing instructions. Sequenced data was processed using Parse Batch Analysis (version 1.0.9) on the DNAnexus platform to demultiplex into library fastq files and count matrices. Processing and analysis of scRNAseq data was carried out using the Python package Scanpy85 (version 1.9.5)—data processing included initial filtering out of cells that had fewer than 200 genes as well as genes found in fewer than 2 cells followed by concatenating all of the samples. Further pre-processing of the data included removing cells with gene counts higher than 7000 in addition to a mitochondrial gene expression higher than 12%. Downstream analyses (e.g. dimensionality reduction, clustering and plotting of graphs) followed standard procedures as per Scanpy documentation (https://scanpy.readthedocs.io/en/stable/). For differential gene expression analysis of samples, we compared each condition to all other conditions and set a minimum threshold of log2 fold change ≥ 2 and minimum p-value of ≤ 0.05. This generated differently sized lists of genes for each condition (Supplementary Data S3). For consistency, the top 122 genes for all conditions were used and cell type annotations were assessed with EnrichR (https://maayanlab.cloud/Enrichr/#libraries) through either the Descartes or Cell Marker Augmented databases. We obtained the top enriched cell marker annotation hits for each database by considering adjusted p-value ≤ 0.05 (Supplementary Data S4–5). An expression threshold method was used when we subsetted cells based on the expression of a specific gene of interest (CX3CR1 or P2RY12). This approach involved selecting cells that exhibited expression levels above a predefined threshold of 0.5. For differential gene expression analysis of the subsetted cluster we used the top 200 genes and input them in the python package Gseapy (version 1.0.5) to analyze GO term pathways in addition to cell type annotation assessment through Descartes cell atlas database (https://maayanlab.cloud/Enrichr/#libraries) (Supplementary Data S6-7). For cell type annotation, we used marker gene sets from the following published datasets as references35 (Supplementary Data S8–9). scRNA-seq data generated and analyzed in this study have been deposited in Gene Expression Omnibus (GEO). IF experiments were performed in µ-Plate 96 Well Black plate (ibidi, 89626). After media removal, cells were fixed with 4% paraformaldehyde (Electron Microscopy Sciences, 15710) in 1x phosphate buffered saline (PBS) (Thermo Fisher Scientific, 10010072) for 15 min at room temperature (RT). Cells were rinsed three times with PBS before proceeding to permeabilization or blocking. Permeabilization was conducted with 0.25% Triton-X-100 (Thermo Fisher Scientific, 85111) in 1x PBS for 15 min at RT followed by three rinses with PBS. Cells were then blocked with 1% bovine serum albumin (BSA) in PBS for 1 h at RT. For primary and secondary antibody staining, antibodies were diluted in PBS with 1% BSA and incubated with cells for 1 h at RT. Three 5-min washes with PBS were used to remove excessive antibodies after staining. Cells were directly imaged in a plate on a Nikon Ti2 Eclipse inverted microscope with a Plan Apo Lambda DM 60× (1.4 NA, Ph3) oil objective and an Andor Zyla sCMOS camera. Images were acquired by NIS-Element AR software. All antibodies were used at 1:200 dilution. Primary IF antibodies used in this study were: Oct-3/4 (Santa Cruz Biotechnology, sc-5279), PU.1 (Thermo Fisher Scientific, PA5-17505), CD11b (BioLegend, 101202), P2RY12 (Thermo Fisher Scientific, 702516), CX3CR1 (Abcam, ab8021). Differentiated cells were incubated with 20 µg/mL of pHrodo Green S. aureus BioParticles (Thermo Fisher Scientific, P35382) for 0-4 h in mTeSR Plus media in the presence of 100 µg/ml Penicillin-Streptomycin (Corning, 30-002-CI). After removal of excessive particles with PBS washes, cells were harvested for antibody (CX3CR1, P2RY12, CD11b) staining and flow cytometry analysis as described in the previous section. Remaining stained cells after flow cytometry were transferred into µ-Plate 96 Well Black plate for fluorescence microscopy to confirm the intracellular localization of the particles. Calcium imaging experiment was conducted in standard 12-well cell culture plates. Differentiated cells were incubated with 1 µg/mL Fluo-4 AM calcium indicator (Thermo Fisher Scientific, F23917) in 1 mL of mTeSR Plus media for 30 min in a cell culture incubator. Excessive dye was washed away with two 1 mL media washes. After adding 1 mL of fresh mTeSR Plus, the cells were put on stage in a microscope inside the incubator. Image acquisition started without stimulation for 90 s to determine baseline signal. After 90 s, 1 mL of media containing 150 µM ADP was added to the cells while imaging was continuing. The total length of imaging was 10 min. The fluorescent signal was quantified and plotted using MATLAB. Aβ fibrillation experiments were performed using SensoLyte Thioflavin T β-Amyloid (1-42) Aggregation Kit (AnaSpec, AS-72214) according to the manufacturer's instruction. The reaction was set up in µ-Plate 96 Well Black plate. Data were acquired on a plate reader with excitation/emission = 440 nm/484 nm at 37 degrees Celsius once every 5 min for 3 h. Day 4 TFiMGLs were plated at a seeding density of 120k cells per well of a 12-well plate in triplicates for each condition. Cells were incubated for 24 h with either 250 μg/mL Aβ (Anaspec, AS-72214), 20 ng/mL INFγ (Novosbio NBP2-34992) or 2.8 μg/mL TDP-43 (RnD systems, AP-190) added to the culture media. After the incubation, cells were harvested for bulk RNA sequencing as previously described. For a given significance level \(\alpha\), the threshold value is \({F}^{1}\). If \({F}_{i1}^{(1)}\, > \,{F}^{1}\), then \({X}_{i1}\) will be included in the regression model and recorded as the set of selected variable indicators as \({I}_{1}\). For a given significance level \(\alpha\), record the corresponding critical value as \({F}^{(2)}\). Otherwise, the variable introduction process is terminated. This step is repeated by selecting an independent variable that is not introduced into the regression model until the test does not introduce any variables. Briefly, source and target TF node interactions were visualized using a node degree sorting scheme. Edge directionality was represented by color (red for positive, blue for negative) and the edge weight was also represented by width. Sample sizes were determined by existing studies in the field to enable statistical analyses and reproducibility. For each pooled screen, 600,000 cells were transfected in order to obtain sufficient numbers of cells post-purification and scRNA-seq processing. We performed scRNA-seq on 10,000 cells reasoning that since not all TF combinations are divergent or functionally unique, this quantity would provide sufficient information for initial screen and TF prioritization. No data was excluded from the relevant analyses. Both pooled screens were performed in duplicate, TFiMGL characterizations were performed in triplicate, PiggyBac dose curve copy number quantification was performed in replicate. TF-induced iPSC differentiation has demonstrated high reproducibility across experiments and studies. Randomization was not utilized in this study since the cell products were derived from the same cell source. The Investigators were not blinded to allocation during experiments and outcome assessment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Bulk RNA-seq data and scRNA-seq data generated and analyzed in this study have been deposited in Gene Expression Omnibus (GEO) with the accession codes: GSE287847, GSE287850, GSE287851, GSE287852, and GSE287853. The raw data generated in this study are provided in the Supplementary Information/Source Data file. Source data are provided with this paper. R script for analyzing gene expression data to identify relationships between target genes and transcription factors (TFs) using linear regression models is available on GitHub87 https://github.com/churchlab/Microglia/tree/main and https://doi.org/10.5281/zenodo.15026232. & Benoist, C. Science forum: the human cell atlas. Ng, A. H. M. et al. A comprehensive library of human transcription factors for cell fate engineering. Gosselin, D. et al. An environment-dependent transcriptional network specifies human microglia identity. Smith, A. M. & Dragunow, M. The human side of microglia. Watkins, L. R. & Hutchinson, M. R. A concern on comparing ‘apples' and ‘oranges' when differences between microglia used in human and rodent studies go far, far beyond simply species: comment on Smith and Dragunow. Abud, E. M. et al. iPSC-derived human microglia-like cells to study neurological diseases. Haenseler, W. et al. A highly efficient human pluripotent stem cell microglia model displays a neuronal-co-culture-specific expression profile and inflammatory response. Directed differentiation of human pluripotent stem cells to microglia. Takata, K. et al. Induced-pluripotent-stem-cell-derived primitive macrophages provide a platform for modeling tissue-resident macrophage differentiation and function. McQuade, A. et al. Development and validation of a simplified method to generate human microglia from pluripotent stem cells. Speicher, A. M., Wiendl, H., Meuth, S. G. & Pawlowski, M. Generating microglia from human pluripotent stem cells: novel in vitro models for the study of neurodegeneration. Xu, R. et al. Human iPSC-derived mature microglia retain their identity and functionally integrate in the chimeric mouse brain. Pandya, H. et al. Differentiation of human and murine induced pluripotent stem cells to microglia-like cells. Dräger, N. M. et al. A CRISPRi/a platform in human iPSC-derived microglia uncovers regulators of disease states. Efficient conversion of human induced pluripotent stem cells into microglia by defined transcription factors. Fate mapping analysis reveals that adult microglia derive from primitive macrophages. Nayak, D., Roth, T. L. & McGavern, D. B. Microglia development and function. Matcovitch-Natan, O. et al. Microglia development follows a stepwise program to regulate brain homeostasis. Kierdorf, K. et al. Microglia emerge from erythromyeloid precursors via Pu.1- and Irf8-dependent pathways. The transcription factor PU.1 is critical for viability and function of human brain microglia. Galatro, T. F. et al. Transcriptomic analysis of purified human cortical microglia reveals age-associated changes. Zhong, S. et al. A single-cell RNA-seq survey of the developmental landscape of the human prefrontal cortex. Olah, M. et al. A transcriptomic atlas of aged human microglia. Identification of a unique TGF-β-dependent molecular and functional signature in microglia. Wehrspaun, C. C., Haerty, W. & Ponting, C. P. Microglia recapitulate a hematopoietic master regulator network in the aging human frontal cortex. Starck, J. et al. Spi-1/PU.1 is a positive regulator of the Fli-1 gene involved in inhibition of erythroid differentiation in friend erythroleukemic cell lines. Hoeffel, G. & Ginhoux, F. Ontogeny of tissue-resident macrophages. & Perry, V. H. Expression of fractalkine (CX3CL1) and its receptor, CX3CR1, during acute and chronic inflammation in the rodent CNS. Systematic comparison of 2A peptides for cloning multi-genes in a polycistronic vector. Okita, K., Nakagawa, M., Hyenjong, H., Ichisaka, T. & Yamanaka, S. Generation of mouse induced pluripotent stem cells without viral vectors. Carey, B. W. et al. Reprogramming factor stoichiometry influences the epigenetic state and biological properties of induced pluripotent stem cells. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cahan, P. et al. CellNet: network biology applied to stem cell engineering. Kracht, L. et al. Human fetal microglia acquire homeostatic immune-sensing properties early in development. Zhang, Y., Parmigiani, G. & Johnson, W. E. ComBat-seq: batch effect adjustment for RNA-seq count data. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Chen, E. Y. et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Cao, J. et al. A human cell atlas of fetal gene expression. Zhang, X. et al. CellMarker: a manually curated resource of cell markers in human and mouse. Exposure of iPSC-derived human microglia to brain substrates enables the generation and manipulation of diverse transcriptional states in vitro. Li, Y. et al. Decoding the temporal and regional specification of microglia in the developing human brain. Induction of NTPDase1/CD39 by reactive microglia and macrophages is associated with the functional state during EAE. Janus-faced microglia: beneficial and detrimental consequences of microglial phagocytosis—PubMed. Guvenek, A. et al. Transcriptional profiling in microglia across physiological and pathological states identifies a transcriptional module associated with neurodegeneration. Zhong, L. et al. Amyloid-beta modulates microglial responses by binding to the triggering receptor expressed on myeloid cells 2 (TREM2). Zhao, W. et al. TDP-43 activates microglia through NF-κB and NLRP3 inflammasome. Zhang, F. et al. IFN-γ and TNF-α drive a CXCL10 + CCL2+ macrophage phenotype expanded in severe COVID-19 lungs and inflammatory diseases with tissue inflammation. Di Virgilio, F., Ceruti, S., Bramanti, P. & Abbracchio, M. P. Purinergic signalling in inflammation of the central nervous system. Van de Sande, B. et al. A scalable SCENIC workflow for single-cell gene regulatory network analysis. Hecker, M., Lambeck, S., Toepfer, S., van Someren, E. & Guthke, R. Gene regulatory network inference: data integration in dynamic models-a review. Matys, V. et al. TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes. Kanehisa, M., Furumichi, M., Sato, Y., Ishiguro-Watanabe, M. & Tanabe, M. KEGG: integrating viruses and cellular organisms. Liu, Z.-P., Wu, C., Miao, H. & Wu, H. RegNetwork: an integrated database of transcriptional and post-transcriptional regulatory networks in human and mouse. The BioGRID database: a comprehensive biomedical resource of curated protein, genetic, and chemical interactions. The STRING database in 2021: customizable protein-protein networks, and functional characterization of user-uploaded gene/measurement sets. Transcription factor-based fate specification and forward programming for neural regeneration. Takahashi, K. & Yamanaka, S. Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. Haniffa, M. et al. A roadmap for the Human Developmental Cell Atlas. Wang, L. et al. Stoichiometry of Gata4, Mef2c, and Tbx5 influences the efficiency and quality of induced cardiac myocyte reprogramming. Hoeffel, G. et al. C-Myb(+) erythro-myeloid progenitor-derived fetal monocytes give rise to adult tissue-resident macrophages. & Ransohoff, R. M. Microglial Physiology and Pathophysiology: Insights from Genome-wide Transcriptional Profiling. Nimmerjahn, A., Kirchhoff, F. & Helmchen, F. Resting microglial cells are highly dynamic surveillants of brain parenchyma in vivo. Colonna, M. & Butovsky, O. Microglia function in the central nervous system during health and neurodegeneration. Salter, M. W. & Stevens, B. Microglia emerge as central players in brain disease. Mathys, H. et al. Single-cell transcriptomic analysis of Alzheimer's disease. Keren-Shaul, H. et al. A Unique Microglia Type Associated with Restricting Development of Alzheimer's Disease. Yeh, F. L., Hansen, D. V. & Sheng, M. TREM2, microglia, and neurodegenerative diseases. Ulland, T. K. et al. TREM2 maintains microglial metabolic fitness in alzheimer's disease. The human microglial HMC3 cell line: where do we stand? An Overview of in vitro Methods to Study Microglia. & Takahashi, M. Trends of stem cell therapies in age-related macular degeneration. de Klerk, E. & Hebrok, M. Stem cell-based clinical trials for diabetes mellitus. Shi, Y., Inoue, H., Wu, J. C. & Yamanaka, S. Induced pluripotent stem cell technology: a decade of progress. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data—PubMed. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. The authors would like to acknowledge Gabriel Filsinger, Yu Wang, Cory Smith, Dima Ter-Ovanesyan, and Michael Chou for helpful discussions. The authors would also like to thank the Biopolymers Facility at HMS, Research Computing at HMS, Harvard Chan Bioinformatics Core, and HMS Immunology Flow Cytometry Core Facility for technical assistance. This work has been supported by the National Human Genome Research Institute of the National Institutes of Health [RM1 HG008525 to G.M.C.] Department of Genetics, Blavatnik Institute, Harvard Medical School, Boston, MA, USA Songlei Liu, Li Li, Mariana Garcia-Corral, Katharina Meyer, Patrick R. J. Fortuna, Björn van Sambeek, Evan Appleton, Alex H. M. Ng, Parastoo Khoshakhlagh, Yuting Chen, Chun-Ting Wu, Jeremy Y. Huang, George Chao, John Aach, Jenny M. Tam & George M. Church Wyss Institute for Biologically Inspired Engineering, Harvard University, Boston, MA, USA Songlei Liu, Li Li, Mariana Garcia-Corral, Katharina Meyer, Patrick R. J. Fortuna, Evan Appleton, Alex H. M. Ng, Parastoo Khoshakhlagh, Chun-Ting Wu, Jeremy Y. Huang, George Chao, Jenny M. Tam & George M. Church Center for Data Sciences, Brigham and Women's Hospital, Boston, MA, USA GC Therapeutics, Inc, Cambridge, MA, USA Whitehead Institute for Biomedical Research, Cambridge, MA, USA Department of Immunology, Blavatnik Institute, Harvard Medical School, Boston, MA, USA Department of Molecular Biology and Genetics, Johns Hopkins University School of Medicine, Baltimore, MD, USA Institute for Cell Engineering, Johns Hopkins University School of Medicine, Baltimore, MD, USA NeuroNexus Institute, University of Massachusetts Chan Medical School, Worcester, MA, USA Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA Broad Institute of MIT and Harvard, Cambridge, MA, USA Centre for Genetics and Genomics Versus Arthritis, Centre for Musculoskeletal Research, The University of Manchester, Manchester, UK You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed overall experimental and analytical strategies. helped with single-cell TF barcode quantification and regression analysis. and J.C. performed PiggyBac copy number quantification. helped with cell atlas integration and mapping. provided significant discussion and input over project design. Correspondence to Jenny M. Tam, Soumya Raychaudhuri or George M. Church. are listed as inventors of a patent related to work on this article. Full disclosure for GMC is available at arep.med.harvard.edu/gmc/tech.html. The remaining authors declare no competing interests. Nature Communications thanks Keisuke Kaji, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Liu, S., Li, L., Zhang, F. et al. Iterative transcription factor screening enables rapid generation of microglia-like cells from human iPSC. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59497-5'>Retinoic acid breakdown is required for proximodistal positional identity during axolotl limb regeneration</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 09:08:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Regenerating limbs retain their proximodistal (PD) positional identity following amputation. This positional identity is genetically encoded by PD patterning genes that instruct blastema cells to regenerate the appropriate PD limb segment. Retinoic acid (RA) is known to specify proximal limb identity, but how RA signaling levels are established in the blastema is unknown. Here, we show that RA breakdown via CYP26B1 is essential for determining RA signaling levels within blastemas. CYP26B1 inhibition molecularly reprograms distal blastemas into a more proximal identity, phenocopying the effects of administering excess RA. We identify Shox as an RA-responsive gene that is differentially expressed between proximally and distally amputated limbs. Ablation of Shox results in shortened limbs with proximal skeletal elements that fail to initiate endochondral ossification. These results suggest that PD positional identity is determined by RA degradation and RA-responsive genes that regulate PD skeletal element formation during limb regeneration. Tissue regeneration requires a complex cellular choreography that results in restoration of missing structures. Salamander limb regeneration is no exception, where mesenchymal cells, including dermal fibroblasts and periskeletal cells, dedifferentiate into a more embryonic-like state and migrate to the tip of the amputated limb to form a blastema1,2. Mesenchymal cells within the blastema contain positional information which coordinates proximodistal (PD) pattern reestablishment in the regenerating limb, enabling autopod-forming blastema cells to distinguish themselves from stylopod-forming blastema cells3,4. It has been proposed that continuous values of positional information exist along the PD axis and that thresholds of these values specify limb segments5,6. These segments are genetically established by combinations of homeobox genes including Hox and Meis genes7,8,9, and each limb segment contains a unique epigenetic profile around these homeobox genes10. However, a mechanistic explanation for how continuous values of positional information are established and differentially interpreted by limb segments during limb regeneration is lacking. Retinoic acid (RA) is a small, pleiotropic molecule that is pervasively involved during vertebrate morphogenesis, including initiating and patterning the developing limb. The prevailing model suggests that RA is synthesized in the lateral plate mesoderm during amniote limb development and diffuses into the limb bud to specify proximal limb identity through activation of Meis genes11,12,13,14,15. An intrinsically timed, antagonizing gradient of fibroblast growth factors (FGFs) emanating from the apical ectodermal ridge (AER) then creates a zone of distal identity marked by Hoxa13 expression15,16,17. This activates CYP26B1, which eliminates RA from the distal limb cells and creates a gradient of RA that patterns the developing limb PD axis18. Perturbing the establishment of this gradient often results in PD patterning defects18,19. Our understanding of the role of RA in urodele limb development is not as comprehensive as in amniotes, but some aspects have been elucidated. Similar to amniotes, a gradient of active RA signaling exists along the developing urodele PD axis20 and disruption results in abnormal skeletal morphologies along the PD axis21,22,23. As in limb development, RA concentration is thought to differentiate the positional identity of upper limb blastemas from lower limb blastemas, thereby ensuring regeneration of the appropriate PD limb structures from disparate amputation planes. RA synthesis occurs endogenously within the blastema in response to injury24,25, and RA signaling is approximately 3.5 times higher in proximal blastemas (PBs) compared to distal blastemas (DBs)26. These endogenous levels of RA are necessary for proper regeneration23,27,28. Furthermore, administering RA to DBs reprograms blastema cells to a proximal identity, resulting in regeneration of more proximal structures in a concentration-dependent manner29,30. This reprogramming is associated with downregulation of distal limb patterning genes like Hoxa13 and upregulation of proximal limb patterning genes like Meis1 and Meis29,22. In agreement with this, perturbing Meis1 and Meis2 in RA treated DBs partially blocks limb duplication31. These studies collectively point to a model whereby heightened RA signaling in PBs specifies proximal positional identity through activation of proximal patterning genes and repression of distal patterning genes. Despite strong evidence that RA regulates positional identity along the regenerating PD axis, the differences in RA signaling levels between PBs and DBs are not fully understood. In this work, we examine the spatiotemporal expression of limb patterning genes in PBs and DBs and compare it to expression of genes related to RA synthesis, degradation, and signaling. We observed that Cyp26b1 was more highly expressed in mesenchymal cells of DBs than PBs, suggesting that differences in RA signaling levels between PBs and DBs are due to RA degradation. Pharmacological inhibition of CYP26B1 in DBs increases RA signaling and results in concentration-dependent duplications of proximal limb segments. These duplications occur by repressing distal limb patterning genes and activating proximal limb patterning genes. Disruption of Shox yields phenotypically normal autopods but shortened stylopods and zeugopods that fail to initiate endochondral ossification. Moreover, we show that Shox is not required for limb regeneration but is crucial for endochondral ossification of stylopodial and zeugopodial skeletal elements during regeneration. Our results collectively suggest that RA breakdown via CYP26B1 is required for establishing positional identity along the regenerating PD axis, enabling the activation of genes such as Shox that confer proximal limb positional identity. To explore how patterning genes regulate RA signaling in PBs and DBs, we examined the expression of known RA-responsive homeobox genes, including Meis1 and Meis215,31. We assessed the expression of these genes in limbs amputated at the upper stylopod (US), lower stylopod (LS), upper zeugopod (UZ), and autopod levels at 10 DPA using qRT-PCR and found that Meis1 and Meis2 expression decreases in progressively distal amputations (Fig. We next reanalyzed blastema scRNA-seq data from proximally amputated limbs (Fig. A Schematic of PD amputation plane qRT-PCR experiment. B, C qRT-PCR of Meis1 (B) and Meis2 (C) at different PD amputation locations (3.5 cm (HT) animals aged 2.5 months, 10 DPA). Expression normalized to Ef1a and analyzed by one-way ANOVA with Tukey-Kramer multiple comparisons. R2 and p values from linear regression analysis are shown. Dashed lines indicate amputation plane. Differences analyzed by two-tailed pairwise clustered Wilcoxon rank-sum tests. F PD intensity plots for mesenchymal Meis1 and Meis2 in PBs and DBs at 10 and 14 DPA (3.5 cm (HT) animals aged 2.5 months). Shaded region represents 99% confidence interval. G–I qRT-PCR of Hoxa9 (G), Hoxa11 (H), and Hoxa13 (I) at different PD amputation locations (3.5 cm (HT) animals aged 2.5 months, 10 DPA). Boxplot and analyses as in (B, C). Dashed lines indicate amputation plane. Axes and analyses as in Fig. Axes and analyses as in (F). Sample size and exact p-values are located within the source data file. We then visualized Meis1 and Meis2 in DBs at 7, 10, and 14 DPA and found Meis1 was primarily expressed in the epithelium while Meis2 was lowly expressed in the mesenchyme and epithelium (Fig. Similarly, PBs at 7 DPA exhibited low mesenchymal Meis1 and Meis2 expression (Fig. At 10 DPA, Meis1 was significantly higher in the mesenchyme of PBs, and Meis2 was elevated in PBs but was generally lowly expressed (Fig. By 14 DPA, Meis1 was significantly higher in PBs and localized to the proximal-most mesenchyme cells, creating a distal zone devoid of Meis1 expression (Fig. A similar Meis1 expression pattern was observed during axolotl (Figs. S3E and S6A), mouse, and chick limb development33,34, suggesting an evolutionarily conserved role reutilized for limb regeneration. Meis2 was significantly higher in PBs at 14 DPA but was generally lower than Meis1 (Fig. Neither Meis1 nor Meis2 expression differed in the epithelium of PBs and DBs at any time point (Fig. S2D), underscoring the importance of mesenchymal cells in conveying PD positional identity within the blastema. We next examined Hoxa9, Hoxa11, and Hoxa13 which have a known role in establishing both the amphibian regenerating and developing PD axis by providing positional identity to each limb segment7,8,9. We observed similar Hoxa9 expression in blastemas at each amputation location (Fig. 1G), while Hoxa11 was elevated in blastemas amputated at the UZ and autopod levels relative to US and LS amputations (Fig. Hoxa13 was significantly more highly expressed in autopod amputations compared to any other amputation level (Fig. 1I), reflecting activation of more 5' Hox genes in increasingly distal limb amputations. Additionally, we found that Hoxa9, Hoxa11, and Hoxa13 were expressed in mesenchymal and epithelial cells (Fig. We observed that Hoxa9, Hoxa11, and Hoxa13 were expressed uniformly in the mesenchyme of DBs at 7, 10, and 14 DPA (Fig. In PBs at 7 DPA, only Hoxa9 was expressed in the mesenchyme while Hoxa11 and Hoxa13 were absent (Fig. By 10 DPA, Hoxa9 and Hoxa11 were expressed in the mesenchyme of PBs, but Hoxa13 remained low (Fig. At 14 DPA, Hoxa9 was expressed throughout the mesenchyme of PBs at similar levels as DBs (Fig. S2J), and Hoxa11 was detected in cells from the mid-blastema to the distal tip (Fig. Hoxa13 was detected in the distal-most mesenchymal cells of PBs, although at lower levels than in DBs (Fig. This colinear activation of 5' Hox genes mirrors Hoxa9, Hoxa11, and Hoxa13 expression during limb development (Figs. S2K and S6B), supporting the hypothesis that progressive specification establishes PD positional identity in both processes7. We hypothesized that RA signaling pathway members regulate RA signaling levels in response to PD patterning genes. We reasoned that a candidate gene would be expressed in the blastema mesenchyme, show graded expression along the PD axis, and complement the spatiotemporal expression of Meis1, Hoxa11, and Hoxa13, which direct limb morphogenesis35. To this end, we focused on RA receptors (Rara, Rarg), genes involved in RA synthesis (Raldh1, Raldh2, Raldh3), and genes involved in RA degradation (Cyp26a1, Cyp26b1). Both Rara and Rarg were expressed in the mesenchyme, but their expression did not match Meis1, Hoxa11, and Hoxa13 (Figs. We next investigated RA degradation in PBs and DBs and found that Cyp26a1 expression was higher in US blastemas and decreased in distal amputations (Fig. Conversely, Cyp26b1 expression was highest in autopod level amputations and decreased in more proximal amputation locations (Fig. Cyp26a1 and Cyp26b1 expression levels appeared to form gradients independent of limb segment, with similar levels in blastemas amputated at the LS and UZ (Fig. ScRNA-seq did not detect Cyp26a1, but Cyp26b1 was highly expressed in the mesenchyme and epidermis of DBs (Fig. Spatially, Cyp26a1 expression was similar in PBs and DBs and showed no bias towards the epithelium or mesenchyme (Fig. This difference in Cyp26a1 detection between our HCR-FISH and qRT-PCR results is likely due to the higher sensitivity of qRT-PCR. In contrast, epithelial Cyp26b1 was observed in PBs and DBs at 7 DPA but was more highly expressed in the mesenchyme of DBs (Fig. By 10 DPA, Cyp26b1 expression was similar in the epithelium of PBs and DBs, but significantly higher in the mesenchyme of DBs (Fig. At 14 DPA, Cyp26b1 was no longer differentially expressed between PBs and DBs (Fig. 2C, D), but was concentrated at the distal tip of PBs, tapering off proximally (Fig. Finally, the spatiotemporal expression of Cyp26b1 within the mesenchyme resembled that of Hoxa11 and Hoxa13 while appearing negatively correlated with Meis1 (Fig. These results indicate that Cyp26b1 expression is graded in mesenchymal cells along the PD axis and its spatiotemporal expression correlates with PD patterning genes. A, B qRT-PCR of Cyp26a1 (A) and Cyp26b1 (B) at different PD amputation locations (3.5 cm (HT) animals aged 2.5 months, 10 DPA). Boxplot and analyses as in Fig. Dashed lines indicate amputation plane. Axes and analyses as in Fig. Shaded region represents 99% confidence interval. Axes and analyses as in Fig. Sample size and exact p-values are located within the source data file. We hypothesized that positional identity along the regenerating PD axis depends on RA degradation in the blastema. To test this, we used talarozole (TAL, or R115866) to inhibit CYP26 during limb regeneration. Three TAL concentrations (0.1 µM, 1 µM, and 5 µM) or DMSO were administered at 4 DPA30 to animals with proximally amputated left limbs and distally amputated right limbs for 7 days (Fig. The skeletal structures of regenerates were then analyzed for abnormalities in morphology (Fig. At 14 DPA, we observed that drug treated blastemas were smaller than DMSO controls (Fig. After fully regenerating, DMSO treated limbs and proximally amputated limbs treated with 0.1 µM or 1 µM TAL exhibited normal skeletal morphology (Fig. At 5 µM TAL, 15.4% of proximally amputated limbs regenerated without skeletal irregularities while limbs in the remaining 84.6% regressed to the scapula and failed to regenerate (Fig. Conversely, 92.3% of distally amputated limbs treated with 0.1 µM TAL exhibited whole (61.2%) or partial (30.1%) zeugopod duplications while 7.7% displayed no limb duplications (Fig. Increasing TAL to 1 µM resulted in full (66.7%) or partial (33.3%) stylopod duplications from distal amputations (Fig. Treatment with 5 µM TAL inhibited regeneration in 92.3% of distally amputated limbs, while 7.7% showed full humerus duplication (Fig. These results mirror both the inhibition of limb regeneration by excess retinoids36 and PD duplications observed in regenerating Xenopus laevis hindlimbs after TAL treatment37. A Timeline of TAL experiments and tissue collection timepoints. B Brightfield images of regenerates and skeletal structures of PBs and DBs treated with DMSO or 0.1, 1, or 5 µM TAL. Dashed lines indicate amputation plane. C 10 DPA DBs from RA reporter animals treated with DMSO or 1 µM TAL (3 cm (HT) animals aged 2 months). Dashed lines indicate amputation plane. D qRT-PCR of Gfp in tissue from RA reporter animals. (3.5 cm (HT) animals aged 2.5 months, 10 DPA). Boxplot and analyses as in Fig. E HCR-FISH for Cyp26a1 and Cyp26b1 in DBs administered 1 µM TAL at 14 DPA (3.5 cm (HT) animals aged 2.5 months). Dashed line indicates amputation plane. F Dot quantification for mesenchymal and epithelial Cyp26a1 and Cyp26b1 in DBs treated with DMSO or 1 µM TAL at 14 DPA (3.5 cm (HT) animals aged 2.5 months). Axes and analyses as in Fig. G 14 DPA DBs from Hoxa13:mCHERRY reporter animals treated with DMSO or 1 µM TAL (7.5 cm (HT) animals aged 6 months). Dashed lines indicate amputation plane. Sample size and exact p-values are located within the source data file. Limb duplications after TAL treatment resembled the effects of administering excess RA to regenerating limbs. To determine if TAL increased RA signaling, we administered 0.1 µM or 1 µM TAL to RA reporter animals20 and assessed GFP expression in DBs at 10 DPA. We observed increased GFP signal and Gfp expression following TAL administration (Fig. 3C- D), indicating TAL increased RA signaling in the blastema. Additionally, mesenchymal Cyp26b1 decreased in DBs treated with 1 µM TAL while Cyp26a1 expression was unaffected (Figs. However, epithelial Cyp26a1 was elevated following TAL treatment (Fig. 3E-F), suggesting that CYP26A1 degrades detrimental levels of RA in the epithelium but is uninvolved in regulating mesenchymal RA levels. Previous studies have shown that excess RA decreases Hoxa137,9,22, leading us to hypothesize that Hoxa13 would similarly decrease following TAL treatment. To address this, we utilized Hoxa13 reporter animals38 to visualize the impact of TAL on HOXA13. DMSO treated DBs demonstrated strong mCHERRY signal at 14 DPA, whereas expression was absent in DBs treated with 1 µM TAL (Fig. We also observed reduced Hoxa13 expression at the transcript level (Fig. S14C, D, F), showing that Hoxa13 decreases following TAL administration. Our results indicate that TAL increases mesenchymal RA signaling in DBs, leading to both concentration-dependent increases in RA signaling and skeletal morphologies that mimic RA-induced limb duplications. 2C, D and 3E, F), the final skeletal morphologies are likely due to inhibition of CYP26B1, not CYP26A1. Moreover, we observed no change in skeletal morphology of proximally amputated limbs treated with the same TAL concentration that caused full limb duplication in distally amputated limbs, indicating RA breakdown is crucial for positional identity in distally, not proximally, amputated limbs. However, most PBs treated with 5 µM TAL failed to regenerate, suggesting that some RA breakdown is necessary for limb regeneration in PBs (Fig. To elucidate the mechanism behind TAL-induced proximalization of DBs, we cotreated animals with TAL and either disulfiram (DIS), a pan RALDH inhibitor, or AGN 193109 (RAA), a pan-RAR antagonist. Treatments were administered as above (Fig. DMSO treated limbs appeared phenotypically normal regardless of treatment condition (Fig. Animals treated with DIS or RAA alone showed either no phenotype or minor skeletal abnormalities from both proximally and distally amputated limbs, but nothing that appeared to affect PD patterning (Fig. In distally amputated limbs treated with 1 µM TAL and 0.1 µM DIS, all limbs were duplicated at the radius/ulna level (Fig. Similarly, 80% of the limbs treated with 1 µM TAL and 1 µM DIS showed either half duplication of the radius/ulna or no duplication (Table S3). These results show that DIS inhibits full proximalization of TAL-treated DBs, indicating that RA synthesis is required for proximalization. None of the distally amputated limbs treated with 1 µM TAL and 0.1 µM RAA showed duplications (Fig. S15D, Table S4), and 83.3% of those treated with 1 µM TAL and 1 µM RAA (Table S4) failed to duplicate. This shows RA signaling through RARs is necessary for limb duplication. We next hypothesized that increasing TAL concentration in DBs would progressively reprogram DBs into a more PB-like identity. Additionally, we hypothesized that reprogramming would be driven by RA-responsive genes that are differentially expressed along the PD axis. We tested these hypotheses using bulk RNA-seq on DBs treated with DMSO, 0.1 µM TAL, 1 µM TAL, and PBs treated with DMSO at 14 DPA. Principal component analysis (PCA) showed segregation between treatment groups including PBs and DBs treated with DMSO (Fig. DBs treated with 0.1 or 1 µM TAL were more transcriptionally similar to each other compared to either DMSO treated PBs or DBs (Fig. TAL-treated DBs exhibited transcriptomes more akin to DMSO treated PBs, suggesting a shift towards a more PB-like positional identity (Fig. However, the transition from DB to PB is likely incomplete, impacting gene expression beyond just positional information. A PCA of bulk transcriptomes from DBs treated with DMSO, 0.1, or 1 µM TAL and PBs treated with DMSO. B Heatmap of the top 371 (padj <0.01, FC = 1.5) genes expressed in each sample type. Cluster numbers are next to the dendrogram. C Bar graphs of significantly upregulated and downregulated genes (padj <0.1) within each comparison. Full gene lists are located in the source data file. E Selected shared DEGs from (D). F HCR-FISH for Meis1 and Meis2 in DBs administered DMSO or 1 µM TAL at 14 DPA. Dashed line indicates amputation plane. G Dot quantification for mesenchymal Meis1 and Meis2 in DBs treated with DMSO or 1 µM TAL at 14 DPA (3.5 cm (HT) animals aged 2.5 months). Axes and analyses as in Fig. Sample size and exact p-values are located within the source data file. We classified these transcriptional differences using hierarchical clustering of the top 371 differentially expressed genes (DEGs) (padj <0.01, FC > 1.5) and observed four clusters (Fig. In cluster one, we observed RA-responsive genes upregulated by 1 µM TAL, including Cyp26a1, Krt15, and Acan which are known to be upregulated in response to RA (Fig. Considering these genes were not differentially expressed in PBs and DBs, it is unlikely that they play a role in PD positional identity. Cluster two consisted of genes highly expressed in DMSO-treated PBs, some of which increased in TAL treated DBs concentrations, while cluster three included genes highly expressed in DMSO-treated DBs. TAL treatment generally decreased cluster three gene expression, with the lowest levels in DMSO-treated PBs (Fig. Cluster four represents a transitional phase between DBs treated with 0.1 µM and 1 µM TAL, containing genes that are highly expressed in DBs treated with DMSO and 0.1 µM TAL but lowly expressed in DBs treated with 1 µM TAL and PBs treated with DMSO (Fig. The number of DEGs relative to DMSO treated DBs increases from 363 to 889 (padj <0.1) as TAL concentration rises from 0.1 or 1 µM, respectively (Fig. In comparison, DMSO treated PBs contained 539 DEGs (padj <0.1) relative to DMSO treated DBs (Fig. The number of DEGs between DMSO treated DBs and 1 µM TAL treated DBs was higher than between DMSO treated DBs and PBs, likely reflecting RA-responsive genes from cluster one not involved in PD patterning. DMSO treated PBs had 425 and 733 DEGs (padj <0.1) compared to 0.1 or 1 µM TAL treated DBs, showing that while TAL treated DBs adopted a more PB-like identity, differences remained (Fig. Comparisons between DMSO treated DBs and either DMSO treated PBs or 1 µM TAL treated DBs enabled identification of RA-responsive genes associated with proximal or distal limb identity. We found that 138 DEGs (FDR < 0.1) were shared between both comparisons (Fig. Among these genes, we identified several that have known roles in patterning the developing or regenerating limb, including Pax940, Evx141, and Alx442 (Fig. Notably absent were Meis1, Meis2, and Cyp26b1 despite our previous results (Figs. 1–2) and their known roles as RA-responsive genes involved in PD patterning18,22,39,43. We suspect these genes were absent from our RNAseq analysis due to equally high levels in the wound epithelium of PBs and DBs (Figs. However, we observed significantly different expression levels of mesenchymal Cyp26b1, Meis1, and Meis2 in DBs treated with 1 µM TAL compared to DMSO treated DBs at 14 DPA (Figs. S16O), indicating that these genes are both involved in patterning and responsive to TAL treatment. Together, our results show that TAL treatment induces DBs to adopt a more proximal positional identity, likely mediated by RA-responsive patterning genes differentially expressed between PBs and DBs. Among RA-responsive genes that were differentially expressed between PBs and DBs were Shox and Shox2 (Fig. Shortened limbs are frequently observed in humans with Shox haploinsufficiency, which is commonly linked to idiopathic short stature, Turner syndrome, and Leri-Weill dyschondrosteosis44,45. While mice lack a functional Shox ortholog, Shox2 mutant mice develop with shortened humeri46. In the axolotl, SHOX and SHOX2 share 73.98% sequence similarity and contain a 100% identical homeodomain (Fig. Shox and Shox2 have previously been noted for their potential role in proximal positional identity during axolotl limb regeneration43, and were more epigenetically accessible in uninjured connective tissue (CT) cells of the stylopod compared to those of the autopod10. These findings suggest that Shox and Shox2 are involved in maintaining and reestablishing proximal limb positional information during limb regeneration. To explore the role of Shox and Shox2 in patterning the PD axis, we visualized Shox and Shox2 expression in developing limb buds from stage 44-47 (Fig. During limb development, we observed high levels of Shox throughout the mesenchyme of stage 44 limb buds with Shox2 localized to the posterior mesenchyme. At stage 47, Shox2 expression remained in the posterior limb mesenchyme but was proximally biased (Fig. Shox was exclusively expressed in the proximal mesenchyme of the stage 47 limb bud, leaving a Hoxa13+ zone of distal mesenchymal cells devoid of Shox expression (Fig. These results suggest that Shox is involved in establishing proximal limb identity during limb development. A Whole mount HCR-FISH for Shox, Shox2, and Hoxa13 in stage 44-47 developing limb buds. B, C qRT-PCR of Shox and Shox2 in DMSO or 1 µM TAL treated DBs (3.5 cm (HT) animals aged 2.5 months, 10 DPA). Each gene was normalized to Ef1a and the groups were analyzed using a two-tailed t-test. D, E qRT-PCR of Shox (D) and Shox2 (E) at different PD amputation locations (3.5 cm (HT) animals aged 2.5 months, 10 DPA). Boxplot and analyses as in Fig. Dashed lines indicate amputation plane. Axes and analyses as in Fig. Shaded region represents 99% confidence interval. Axes and analyses as in Fig. I, J HCR-FISH for Meis1, Shox2, and Shox (I) or Shox, Shox2, and Hoxa13 (J) in a PB at 14 DPA. Dashed line indicates amputation plane. K UMAP of Shox+ and Hoxa13+ cells in blastemas at 7, 14, and 22 DPA from reanalyzed scRNA-seq dataset32. Sample size and exact p-values are located within the source data file. We next examined Shox and Shox2 expression following limb amputation. In agreement with our RNA-seq results, Shox and Shox2 expression increased in DBs upon 1 µM TAL treatment and were more highly expressed in US amputations compared to autopod amputations (Fig. Shox expression appeared to decrease incrementally in more distal amputations whereas Shox2 was expressed at the same level in LS blastemas and autopod blastemas (Fig. We also observed that while Shox2 expression in PBs treated with 1 µM TAL was identical to DMSO treated PBs, Shox expression was significantly lower in PBs treated with 1 µM TAL compared to control PBs (Fig. These results suggest that Shox is more sensitive to heightened RA levels observed in PBs treated with 1 µM TAL than Shox2. Furthermore, Shox and Shox2 were primarily expressed in mesenchymal cells with little expression in any other cell type (Fig. As in limb development, Shox2 expression in PBs and DBs at 7, 10, or 14 DPA was localized proximally and posteriorly in mesenchymal cells (Fig. Shox2 expression levels in PBs and DBs at 10 DPA were similar, but by 14 DPA Shox2 was significantly more highly expressed in the proximal and posterior mesenchyme of PBs (Fig. This may indicate that Shox2 has a role in patterning both the PD and AP axis during limb development and regeneration. Shox was lowly expressed in the mesenchyme of PBs and DBs at 7 DPA (Fig. By 10 DPA, Shox expression was located throughout the mesenchyme of PBs and was significantly more highly expressed than in DBs (Fig. The limited areas of Shox expression in DBs at 10 and 14 DPA seemed to be associated with the uninjured skeletal elements (Fig. At 14 DPA, Shox remained significantly more highly expressed in PBs but was restricted to the Meis1+ proximal mesenchyme, leaving a distal subset of Hoxa13+ cells devoid of Shox expression (Fig. Shox+ and Hoxa13+ cells appeared mutually exclusive (Fig. 5J, K), suggesting that Shox is uninvolved in autopod formation. The spatiotemporal expression patterns of Shox, Meis1, and Hoxa13 (Fig. Given that Meis1 is RA-responsive (Fig. 4F, G), this overlap in expression may indicate that mesenchymal Shox is activated by RA via Meis1. Indeed, the Shox promoter contains several putative Meis1 binding sites (Table S5). Conversely, Hoxa13 is repressed by RA (Figs. 3G and 4E), suggesting that Hoxa13 is involved in creating the distal limit for Shox. Our gene expression results led us to hypothesize that Shox has a role in establishing both stylopod and zeugopod, but not autopod, positional identity during both limb development and regeneration. We targeted Shox with two sgRNAs specifically designed against exons 1 and 2 and simultaneously injected these sgRNAs into axolotl embryos to create mosaic F0 Shox knockout animals (Shox crispants) (Fig. NGS genotyping analyses of 10 Shox crispants indicated that both targeted loci were highly mutated, ranging from 75.15 to 97.62% of all sequenced alleles being mutated across each animal (Fig. Furthermore, these animals developed to adulthood, enabling us to examine the role of Shox during limb development and regeneration. Shox crispants developed significantly smaller limbs than controls with significantly smaller stylopods and zeugopods (Fig. Interestingly, autopod length was unaffected in Shox crispants compared to controls (Fig. 6C), showing that Shox is critical for stylopod and zeugopod development but dispensable for autopod development. To confirm this finding, we generated F1 homozygous Shox knockout axolotls (Shox−/− animals) by crossing two Shox crispants. Like the F0 crispants, Shox−/− animals exhibited shortened stylopodial and zeugopodial elements, while their autopods appeared relatively unaffected (Fig. This indicates that despite mosaicism, F0 crispants display the same limb phenotype as F1 Shox−/− animals. A similar phenotype was observed in the limbs of Shox2 knockout mice, where chondrocytes in shortened stylopods and zeugopods failed to proliferate and mature, preventing endochondral ossification46. A Schematic of the Shox genomic landscape. B Brightfield images of control and Shox crispant limbs (3.5 cm (HT) animals aged 2.5 months). C Skeletal element quantification in control and Shox crispant limbs (7.5 cm (HT) animals aged 6 months). Differences between control and Shox crispants were analyzed using a two-tailed t-test. D Alcian blue and alizarin red stain of adult control and Shox crispant limbs (12 cm (HT) animals aged 10 months). E H&E&A of whole stylopods, proximal epiphyses, and digits from controls and Shox crispants (8 cm (HT) animals aged 7 months). F HCR-FISH for Shox and Sox9 in a whole mount stage 46 developing limb. Sample size and exact p-values are located within the source data file. We observed that while skeletal elements in adult control stylopods and zeugopods were partially calcified, those from Shox crispants appeared to lack calcification (Fig. Additionally, chondrocytes from control stylopods demonstrated clear progression from proliferation to hypertrophy before calcifying (Fig. In contrast, chondrocytes from Shox crispants failed to proliferate, appearing to remain as reserve cartilage through adulthood (Fig. Considering autopod size was not impacted in Shox crispant limbs, we next examined the digits of Shox crispants limbs to determine if endochondral ossification in autopodial elements was disrupted. Chondrocytes from both control and Shox crispant digits underwent phenotypically normal endochondral ossification, suggesting that while essential for more proximal limb segments, Shox is not required for autopodial skeletal maturation (Fig. This indicates that proximal and distal skeletal elements of the limb have distinct transcriptional programs responsible for endochondral ossification. Collectively, our data indicate that Shox is required for chondrocyte maturation within proximal skeletal elements. During limb development, however, many Shox+ cells are Sox9-, suggesting that Shox may also be involved in patterning non-chondrogenic mesenchymal cells (Fig. Finally, we found that Shox was dispensable for limb regeneration, as Shox crispants successfully regenerated limbs and progressed though typical stages, including blastema and palette formation, without obvious abnormalities (Fig. Shox crispant limbs remained shortened after fully regenerating, suggesting that Shox is dispensable for limb regeneration but crucial for patterning the regenerating stylopodial and zeugopodial segments (Fig. This finding was also confirmed in Shox−/− animals, which regenerated similarly sized autopods as controls but significantly smaller stylopods and zeugopods (Fig. We further observed that while many Shox+ cells in PBs at 21 DPA were Sox9+, many other Shox+ cells were Prrx1+ and Sox9- (Fig. Additionally, Sox9 was expressed in the presumptive digits of DBs at 21 DPA (Fig. S22), indicating that Shox is not required to activate Sox9 expression. Together, these results both suggest that chondrocyte formation is independent of Shox and that Shox has an additional role in patterning non-chondrogenic mesenchymal cells during limb regeneration. A Regeneration time course of PBs and DBs in Shox crispants. B UMAP of Shox+ and Sox9+ cells in blastemas at 7, 14, and 22 DPA from reanalyzed scRNA-seq dataset32. C HCR-FISH for Shox and Sox9 in a control PB and DB at 21 DPA. Dashed line indicates amputation plane. Dashed lines indicate amputation plane. Boxplots and analyses as in Fig. I Brightfield images of regenerates and skeletal structures of control or Shox crispant limbs treated with 1 µm TAL. Dashed lines indicate amputation plane. Sample size and exact p-values are located within the source data file. We next assessed Meis1 and Hoxa13 expression in Shox−/− PBs and DBs and found their spatial expression patterns unchanged compared to controls (Fig. S22), consistent with findings from Shox2 KO mice46. Correspondingly, the relative expression of these genes in Shox−/− PBs and DBs was unchanged from controls (Fig. These results suggest that PD positional identity is unaffected by the absence of Shox. Furthermore, while Shox2 was expressed in the posterior mesenchyme of both control and Shox−/− DBs, it was generally more lowly expressed in Shox−/− DBs (Fig. In contrast, Shox2 expression expanded into the anterior mesenchyme of Shox−/− PBs, though its levels remained unchanged compared to controls (Fig. This suggests that Shox2 compensates for the loss of Shox in Shox−/− PBs. The observations that Hoxa13 and Meis1 expression are unaffected by Shox knockout (Fig. 7D–F), along with Meis1 and Shox colocalization (Fig. 5H, I), suggest that Shox acts downstream of Meis1 to regulate endochondral ossification of proximal skeletal elements. Considering Meis1 is RA-responsive and limb-specific Meis KO mice lack proximal skeletal elements14, RA likely directs proximal endochondral ossification through Shox via Meis1. Consistent with this, Cyp26b1 was significantly more highly expressed in Shox−/− DBs than Shox−/− PBs (Fig. In addition, Cyp26b1 expression levels did not vary between control DBs and Shox−/− DBs, as well as between control PBs and Shox−/− PBs (Fig. These observations suggest both that Shox does not determine RA concentration in the blastema and physiological levels of RA are similar in regenerating control and Shox−/− limbs. Further, Shox crispant DBs treated with 1 µM TAL regenerate with shortened duplicated proximal elements (Fig. 7I), showing that Shox crispant limbs respond to RA but cannot properly pattern proximal elements. Our study shows that regulation of endogenous RA levels is required for PD limb patterning during regeneration. We propose that Cyp26b1-mediated RA breakdown, not RA synthesis or RAR expression, determines PD positional identity by setting the RA signaling levels in the blastema, activating or repressing RA signaling (Fig. Moreover, CT cells in the uninjured limb have inherent, epigenetically encoded positional identity10. Upon amputation, these cells dedifferentiate into a limb bud-like state while retaining their PD positional memory2. We propose that dedifferentiated blastema cells modify Cyp26b1 expression depending on their positional memory, which adjusts RA signaling levels to the appropriate PD location and regulates genes that convey PD positional identity. Elevated RA levels in PBs activate Shox, promoting endochondral ossification in proximal skeletal elements. In contrast, reduced RA levels in DBs leads to a Shox-independent mechanism for endochondral ossification of distal skeletal elements. Our results show that RA signaling levels exert segment-specific effects during skeleton regeneration (Fig. Proposed model for how PD patterning occurs during axolotl limb regeneration. The model that we propose explains how positional identity is determined among cells at two spatial scales: the entire limb PD axis and PD axes within different limb segments. Cyp26b1 expression was graded across the limb PD axis, with similar levels of expression observed between blastemas that formed from different, but spatially juxtaposed limb segments (e.g., LS and UZ blastemas). However, within limb segments, Cyp26b1 expression differed considerably between blastemas that formed from spatially disparate locations (e.g., US and LS blastemas). Our model suggests that positional identity is conveyed in a segment specific manner, as Shox expression is mutually exclusive from Hoxa13 and Shox perturbation affects only stylopodial and zeugopodial skeletal elements. Further evidence for this comes from Hoxa13 knockout newts that fail to regenerate autopods8. This may imply that each limb segment requires a specific threshold of positional values determined by RA signaling levels, as posited by the French Flag model5. Once this threshold is met, blastema cells can create intra-segmental positional values based on RA signaling levels, enabling fine-tuning of limb segment morphology. It may also explain why half-segment duplications are observed following treatment with lower concentrations of TAL or RA during limb regeneration29,30. While our model specifically focuses on RA degradation in PBs and DBs, it does not address RA synthesis along the regenerating PD axis. Our results present two possibilities: (1) heightened levels of RA synthesis at more proximal amputations and (2) identical levels of RA synthesis at any amputation level that is fine-tuned at each amputation location by RA degradation. Regarding the former, our qRT-PCR analysis revealed that Raldh3 expression was significantly higher in proximal amputations compared to distal amputations, suggesting that elevated RA levels in PBs are generated primarily through Raldh3 rather than Raldh1 or Raldh2. However, this finding was inconsistent with our FISH and RNAseq results, which showed no differences in expression of Raldh1, Raldh2, and Raldh3 between PBs and DBs. Furthermore, inhibiting RA synthesis in PBs and DBs caused skeletal abnormalities in the final regenerate at each amputation location. These results are more consistent with the possibility that RA synthesis is uniform along the PD axis. Nevertheless, our results do not exclude either scenario, and future studies are needed to ascertain RA synthesis levels along the PD axis. An outstanding question related to this is why RA would be synthesized in the regenerating limb only to be degraded depending on amputation location. While our work does not address this complexity directly, we speculate that RA has several roles during limb regeneration outside of providing proximal limb positional identity. RA is important for directing nerves to their targets, including epithelial and neuromuscular junctions47. Furthermore, RA plays an essential role in replenishing epithelial cells during physiological growth48, which may contribute to the scarless wound healing following injuries in salamanders49. Further studies are needed to identify upstream activators of Cyp26b1 during amphibian limb regeneration. We observed that Cyp26b1 exhibited a similar spatiotemporal expression pattern as Hoxa11 and Hoxa13, providing circumstantial evidence that 5' Hox genes regulate Cyp26b1 expression. Previous studies on mouse limb development have suggested that AER-derived FGFs activate Cyp26b1 instead of 5' Hox genes to create a domain of distal identity while simultaneously interacting with SHH to promote distal outgrowth17. Indeed, Fgf8 expression in Cyp26b1−/− limbs does not appear to be impacted by the loss of CYP26B1 function18. However, this may differ during limb regeneration as the SHH-FGF feedback loop is required for blastema distal outgrowth and proliferation50, and neither Fgf8 nor Shh were differentially expressed between PBs and DBs in our RNA-seq. For these reasons, it seems more likely that Cyp26b1 is regulated by 5' Hox genes including Hoxa11 and Hoxa13 during limb regeneration. How a cell conveys its PD positional identity to neighboring cells in response to RA to coordinate patterning in the blastema remains an unanswered question in the field. Several lines of evidence suggest that differences in adhesivity and cell adhesion molecules (CAMs) differentiate PBs from DBs51. These differences in adhesivity can be modified by RA, suggesting that RA signaling levels are important for generating a gradient of adhesive properties in PBs and DBs52,53. In agreement with these studies, two CAMs, TIG1 and PROD1, have previously been identified that are RA-responsive and modify adhesivity during limb regeneration38,54. Neither of these CAMs appeared to be RA-responsive or differentially expressed in PBs and DBs, despite their importance in directing cell adhesivity during limb regeneration. However, our RNAseq represents only a small portion of gene expression during the entire duration of limb regeneration. Indeed, Tig1 expression appears strongest at earlier timepoints, which may explain its absence from our dataset38. Nonetheless, our RNA-seq dataset should serve as a helpful resource for identifying other RA-responsive CAMs that are differentially expressed in PBs and DBs. Transgenic lines (RA reporter- tgSceI(RARE:GFP)Pmx, Hoxa13 reporter- tm(Hoxa13t/+:Hoxa13-T2A-mCHERRY)Etnka) used in this study were bred at Northeastern University. d/d genotype axolotls (RRID:AGSC 101E; RRID:AGSC101L) were bred at Northeastern University or were obtained from the Ambystoma Genetic Stock Center (RRID: SCR_006372). Crispant lines were generated at both Northeastern University and the University of Kentucky. An equal distribution of males and females were used in this study. All surgeries were performed under anesthesia using 0.01% benzocaine and were approved by the Northeastern University Institutional Animal Care and Use Committee. Talarozole and AGN 193109 were resuspended in dimethyl sulfoxide (DMSO) to 5 mM and stored at −20 °C. Disulfiram was resuspended in DMSO to 20 mM and stored at −20 °C. Animals received amputations through the carpals on one limb and through the upper humerus on the contralateral limb. Four days post amputation (DPA), animals were placed in drug or DMSO. Animals were redosed every other day for 7 days. Following treatment, animals were removed from the drug water and placed into axolotl housing water. Regenerated limbs were collected from the animals 120 days post treatment for analysis of skeletal morphology. Blastema tissue (4–5 blastemas per sample, 3–6 samples per experiment) was collected from 3.5 cm animals head to tail (HT) aged 2.5 months and immediately frozen in liquid nitrogen. RNA isolation and qRT-PCR were performed with standard protocols, using 1 ng of cDNA per reaction. Each qRT-PCR reaction was run in duplicate. Primers for each gene are listed in Table S1. Gene expression was normalized to Ef1α and the Livak 2−ΔΔCq method was used to quantify relative fold change in mRNA abundance. Statistical significance between groups was tested using either a two-tailed Student's t-test or a one-way analysis of variance (ANOVA) with post-hoc Tukey's honestly significant difference (HSD) test on ΔCt values. Linear regression analysis was performed to test for linear relationships across samples. Existing single-cell RNA sequencing (scRNA-seq) data from proximally amputated forelimbs was accessed from NCBI SRA PRJNA58948432. UMI counts for gene expression data were generated using kallisto55 and bustools56 against Ambystoma mexicanum transcriptome v4.757. Isoform-level count matrices were generated using “bustools count” with the --em flag. Counts matrices were imported into Seurat v558 in an RStudio IDE. Cells expressing fewer than 200 features and features expressed in fewer than 3 cells were filtered out; matrices were further filtered to include cells with between 500 and 25,000 counts, <5% red blood cell gene content, and <15% mitochondrial gene content. Counts were normalized using the SCTransform function with regression for mitochondrial content and red blood cell content. The counts layers were flattened with the IntegrateLayers function using RPCAIntegration, after which FindClusters was run with a resolution of 0.4 followed by RunUMAP using the first 30 dimensions, 50 nearest neighbors, and a minimum distance of 0.1. Following cluster annotation, SCTransform-normalized isoform-level expression for genes of interest was summed and log-transformed for plotting with ggplot2 using UMAP coordinates from Seurat59. Whole mount and tissue section version 3 hybridization chain reaction fluorescence in-situ hybridization (HCR-FISH) protocols were performed as previously described60 with slight modifications on the tissue section protocol. Namely, fresh frozen blastema tissue from 3.5 cm animals (HT) aged 2.5 months were sectioned at 10 µm. Slides were then fixed with 4% paraformaldehyde (PFA) for 15 min at room temperature and washed with 1X phosphate buffered saline (PBS) three times for 5 min at room temperature before being stored in 70% ethanol at 4 °C until use. Slides were washed again three times for 5 min with 1X PBS at room temperature before beginning HCR-FISH protocol. Probes for each gene were generated using ProbeGenerator (http://ec2-44-211-232-78.compute-1.amazonaws.com) and can be found in the source data file. All tissue sections were imaged using a Zeiss LSM 880 confocal microscope at 20X magnification with airyscan fast settings. For each image, 4–6 optical sections were captured in the 10 µm tissue section. Following acquisition, images were processed on Zen Black using airyscan processing with the automatic 2D setting, then a maximum intensity projection step was performed. If obvious issues in tile alignment were observed, an additional stitching step was performed prior to maximum intensity projection. Whole mount samples stained using HCR-FISH were mounted in 1.5% low-melt agarose and refractive index matched with EasyIndex (LifeCanvas Technologies) overnight at 4 °C60. Samples were then imaged in EasyIndex using a Zeiss Lightsheet Z1 at 20X magnification. All representative images are a single z-plane from the stack. Dot detection in HCR-FISH images was performed using RS FISH61. For visualization of HCR-FISH dots in figure pictures, RS-FISH dots were flattened atop the DAPI channel and background dots outside tissue sections were removed. For quantification, identified dots from RS-FISH were overlaid on a black background and the image was flattened. The ImageJ function “Find Maxima” with prominence greater than 0 was used to convert dots to single pixels. Cell outlines were obtained using Cellpose segmentation run on the DAPI channel using the “cyto2” model with default settings and a diameter of 6062. A modified version of the region of interest conversion script provided by the Cellpose authors was used to obtain measurements of area and raw integrated density per cell. Pipeline documentation can be accessed at https://github.com/Monaghan-Lab/HCRFISH-DotCounting. All measurements were concatenated and filtered for cells with an area greater than 60, as smaller areas can indicate poor segmentation or cells at a nonoptimal plane. To evaluate the expression levels for each gene between groups, two-tailed clustered Wilcoxon rank-sum tests (n = 2–6 blastemas, sample size for each group located within the source data file) were performed with FDR p-adjustment using the R package “clusrank” with test method “ds”63. Violin plots were generated in RStudio using the ggplot2 package59. An equal rotation was performed on the DAPI channel from the same tissue. The freehand selection tool was used to isolate the blastemal mesenchyme from the DAPI image, then this selection was overlaid onto the rotated maxima image and used to clear any points lying outside of the selection before adding a value of 25 to each pixel within the blastema selection. The value z at every (x,y) pixel in the image was then measured. Measurements for each image were imported into R and filtered for z > 0 to select for points falling only within the blastemal mesenchyme. The y values were then rescaled to generate a pseudo-proximodistal axis with range [0, 1]. Z values were reassigned such that 25 represented a 0 or “negative” pixel and 255 represented a 1 or “positive” pixel, then grouped at each pseudo-y value and averaged to yield a proportion of positive pixels. The square root of these proportions was plotted as a smoothed curve along the pseudo-proximodistal axis using the “stat_smooth()” function in ggplot2 with an n of 200059. A sample script can be accessed at https://github.com/Monaghan-Lab/HCRFISH-DotCounting. Whole mount alcian blue/alizarin red staining was performed using a modified version of a previously published protocol64. Briefly, regenerated limbs were collected and immediately fixed in 4% PFA overnight at 4 °C. The next day, limbs were washed three times for 5 min each with 1X PBS at room temperature and skinned. Limbs were dehydrated in 25%, 50%, and 100% ethanol for 20 min at each concentration before being placed in alcian blue mixture (5 mg alcian blue in 30 mL 100% ethanol, 20 mL acetic acid) and left on a rocker overnight at room temperature. The next day, limbs were rehydrated in 100%, 50%, and 25% ethanol for 20 min at room temperature at each concentration. Limbs were placed into trypsin solution (1% trypsin in 30% borax) and rocked for 45 min at room temperature. Samples were washed twice in 1% KOH for 30 min each before being placed in alizarin red mixture (5 mg alizarin red in 50 mL 1% KOH) and rocked overnight at room temperature. Limbs were again washed twice with 1% KOH for 30 min at room temperature and placed in 25% glycerol/1% KOH solution until samples cleared. The limbs were then dehydrated in 25%, 50%, and 100% ethanol for 20 min at room temperature at each concentration. TAL was administered in the housing water as indicated above. Samples (n = 3 samples per condition, 2-3 blastemas per sample, 5 cm (HT) animals aged 3 months) were collected at 14 DPA and immediately flash frozen in liquid nitrogen before storing at −80 °C. Samples were shipped to Genewiz for RNA sequencing using an Illumina HiSeq platform and 150-bp paired-end reads for an average sequencing depth of roughly 24.1 million reads per sample. Raw sequencing data are available at GEO (accession number GSE272731). Reads were quality trimmed before quasi-mapping to the v.47 axolotl transcriptome65 and quantification. Differential expression analysis was performed on counts matrices with DESeq2 v1.34.066 using the Trinity v2.8.5 script67 “run_DE_analysis.pl” with default parameters. Visualizations were produced with ggplot2, ggvenn, and ComplexHeatmap68 where appropriate. Sample correlation heatmap was produced using Trinity script “PtR” on the gene counts matrix67. Shox crispants were generated using CRISPR/Cas9 according to previous protocols69. The following sgRNAs were used: NGS-based genotyping was conducted on a pool of 10 tail tips from crispant animals and analyzed using CRISPResso270. Samples for hematoxylin, eosin, and alcian blue staining (H&E&A) were collected and placed in 4% PFA overnight at 4 °C. Samples were then washed with 1X PBS three times for 5 min each. Following the third wash, samples were placed in 1 mM ethylenediaminetetraacetic acid (EDTA) for four days at 4 °C. The EDTA solution was changed every other day. After EDTA treatment, the samples were again washed with 1X PBS three times for 5 min each before being cryoprotected in 30% sucrose. Once the samples sunk in the 30% sucrose, the samples were mounted in optimal cutting temperature medium and frozen at −80 °C until use. These blocks were then sectioned at 10 µm, and the resultant slides were baked at 65 °C overnight. To improve the adherence of the skeletal structure to the slide, slides were placed in 4% PFA for 15 min at room temperature. Slides were washed three times for 5 min with 1X PBS, then the slides were placed in alcian blue solution (5 mg of alcian blue in 30 mL 100% ethanol and 20 mL acetic acid) for 10 min at room temperature. The slides were then dehydrated with 100% EtOH for 1 min at room temperature and allowed to air dry. Hematoxylin solution was added to the slides and incubated at room temperature for 7 min. Slides were then dipped into tap water five times, then clean tap water another 15 times. Slides were dipped another 15 times in clean tap water before adding bluing buffer for 2 min at room temperature. Slides were again dipped in clean tap water five times, and eosin solution was pipetted onto the slides for 2 min at room temperature. Residual eosin was removed by dipping slides ten times in clean tap water, and the slides were air dried before imaging. Animals for each experiment were randomly selected. Sample size for each experiment is indicated in source data file. No statistical method was used to predetermine sample size. Statistical analyses are indicated within the figure legends and methods, and exact p-values are indicated in the source data file. Analyses were conducted on Matlab or R. Fluorescent images not compatible with RS-FISH and qRT-PCR samples that demonstrated abnormal amplification were excluded from the analyses. Experiments were replicated at least twice, and all attempts at replication were successful. The experiments were randomized, and the investigators were blinded to allocation during experiments and outcome assessment. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Raw bulk RNAseq data have been deposited in the GEO database under accession code GSE272731. scRNAseq data were accessed at NCBI SRA under accession code PRJNA589484. Source data are provided with this paper. All custom code generated for this manuscript were submitted to GitHub for public access. Code for HCR-FISH dot quantification and HCR-FISH PD intensity measurements can be accessed at https://github.com/Monaghan-Lab/HCRFISH-DotCounting. Currie, J. D. et al. Live imaging of axolotl digit regeneration reveals spatiotemporal choreography of diverse connective tissue progenitor pools. Gerber, T. et al. Single-cell analysis uncovers convergence of cell identities during axolotl limb regeneration. Cells keep a memory of their tissue origin during axolotl limb regeneration. Nacu, E. et al. Connective tissue cells, but not muscle cells, are involved in establishing the proximo-distal outcome of limb regeneration in the axolotl. Wolpert, L. Positional information and the spatial pattern of cellular differentiation. Pescitelli, M. J. Jr. & Stocum, D. L. Nonsegmental organization of positional information in regenerating Ambystoma limbs. & Tanaka, E. M. Progressive specification rather than intercalation of segments during limb regeneration. Takeuchi, T. et al. Newt Hoxa13 has an essential and predominant role in digit formation during development and regeneration. & Bryant, S. V. Regulation of HoxA expression in developing and regenerating axolotl limbs. Kawaguchi, A. et al. A chromatin code for limb segment identity in axolotl limb regeneration. Niederreither, K., Vermot, J., Schuhbaur, B., Chambon, P. & Dolle, P. Embryonic retinoic acid synthesis is required for forelimb growth and anteroposterior patterning in the mouse. Cooper, K. L. et al. Initiation of proximal-distal patterning in the vertebrate limb by signals and growth. & Torres, M. Diffusible signals, not autonomous mechanisms, determine the main proximodistal limb subdivision. Delgado, I. et al. Proximo-distal positional information encoded by an Fgf-regulated gradient of homeodomain transcription factors in the vertebrate limb. Opposing RA and FGF signals control proximodistal vertebrate limb development through regulation of Meis genes. Saiz-Lopez, P. et al. An intrinsic timer specifies distal structures of the vertebrate limb. Probst, S. et al. SHH propagates distal limb bud development by enhancing CYP26B1-mediated retinoic acid clearance via AER-FGF signalling. Yashiro, K. et al. Regulation of retinoic acid distribution is required for proximodistal patterning and outgrowth of the developing mouse limb. Niederreither, K., Subbarayan, V., Dolle, P. & Chambon, P. Embryonic retinoic acid synthesis is essential for early mouse post-implantation development. Monaghan, J. R. & Maden, M. Visualization of retinoic acid signaling in transgenic axolotls during limb development and regeneration. Scadding, S. R. & Maden, M. Comparison of the effects of vitamin A on limb development and regeneration in the axolotl, Ambystoma mexicanum. Nguyen, M. et al. Retinoic acid receptor regulation of epimorphic and homeostatic regeneration in the axolotl. Maden, M. Retinoids as endogenous components of the regenerating limb and tail. Viviano, C. M., Horton, C. E., Maden, M. & Brockes, J. P. Synthesis and release of 9-cis retinoic acid by the urodele wound epidermis. Scadding, S. R. & Maden, M. Retinoic Acid Gradients during Limb Regeneration. Brockes, J. P. Introduction of a retinoid reporter gene into the urodele limb blastema. Scadding, S. R. Citral, an inhibitor of retinoic acid synthesis, modifies pattern formation during limb regeneration in the axolotl Ambystoma mexicanum. Lee, E., Ju, B.-G. & Kim, W.-S. Endogenous retinoic acid mediates the early events in salamander limb regeneration. Maden, M. Vitamin A and pattern formation in the regenerating limb. Thoms, S. D. & Stocum, D. L. Retinoic acid-induced pattern duplication in regenerating urodele limbs. Mercader, N., Tanaka, E. M. & Torres, M. Proximodistal identity during vertebrate limb regeneration is regulated by Meis homeodomain proteins. Dynamic cell transition and immune response landscapes of axolotl limb regeneration revealed by single-cell analysis. Roselló-Díez, A., Arques, C. G., Delgado, I., Giovinazzo, G. & Torres, M. Diffusible signals and epigenetic timing cooperate in late proximo-distal limb patterning. Mercader, N. et al. Ectopic Meis1 expression in the mouse limb bud alters P-D patterning in a Pbx1-independent manner. Uzkudun, M., Marcon, L. & Sharpe, J. Data-driven modelling of a gene regulatory network for cell fate decisions in the growing limb bud. Chemical activation of RARβ induces post-embryonically bilateral limb duplication during Xenopus limb regeneration. Oliveira, C. R. et al. Tig1 regulates proximo-distal identity during salamander limb regeneration. Polvadore, T. & Maden, M. Retinoic acid receptors and the control of positional information in the regenerating axolotl limb. McGlinn, E. et al. Pax9 and Jagged1 act downstream of Gli3 in vertebrate limb development. Niswander, L., Jeffrey, S., Martin, G. R. & Tickle, C. A positive feedback loop coordinates growth and patterning in the vertebrate limb. te Welscher, P., Fernandez-Teran, M., Ros, M. A. & Zeller, R. Mutual genetic antagonism involving GLI3 and dHAND prepatterns the vertebrate limb bud mesenchyme prior to SHH signaling. Bryant, D. M. et al. A tissue-mapped axolotl de novo transcriptome enables identification of limb regeneration factors. Rao, E. et al. Pseudoautosomal deletions encompassing a novel homeobox gene cause growth failure in idiopathic short stature and Turner syndrome. Shears, D. J. et al. Mutation and deletion of the pseudoautosomal gene SHOX cause Leri-Weill dyschondrosteosis. Yu, L. et al. Shox2 is required for chondrocyte proliferation and maturation in proximal limb skeleton. Dmetrichuk, J. M., Spencer, G. E. & Carlone, R. L. Retinoic acid-dependent attraction of adult spinal cord axons towards regenerating newt limb blastemas in vitro. Zasada, M. & Budzisz, E. Retinoids: active molecules influencing skin structure formation in cosmetic and dermatological treatments. Seifert, A. W., Monaghan, J. R., Voss, S. R. & Maden, M. Skin regeneration in adult axolotls: a blueprint for scar-free healing in vertebrates. Nacu, E., Gromberg, E., Oliveira, C. R., Drechsel, D. & Tanaka, E. M. FGF8 and SHH substitute for anterior-posterior tissue interactions to induce limb regeneration. & Stocum, D. L. Surface properties of regenerating limb cells: evidence for gradation along the proximodistal axis. & Scadding, S. R. Effects of tunicamycin on retinoic acid induced respecification of positional values in regenerating limbs of the larval axolotl, Ambystoma mexicanum. Crawford, K. & Stocum, D. L. Retinoic acid coordinately proximalizes regenerate pattern and blastema differential affinity in axolotl limbs. & Brockes, J. P. The newt ortholog of CD59 is implicated in proximodistal identity during amphibian limb regeneration. Bray, N. L., Pimentel, H., Melsted, P. & Pachter, L. Near-optimal probabilistic RNA-seq quantification. Melsted, P., Ntranos, V. & Pachter, L. The barcode, UMI, set format and BUStools. The giant axolotl genome uncovers the evolution, scaling, and transcriptional control of complex gene loci. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer International Publishing, 2016). Lovely, A. M., Duerr, T. J., Stein, D. F., Mun, E. T. & Monaghan, J. R. Hybridization chain reaction fluorescence in situ hybridization (HCR-FISH) in Ambystoma mexicanum tissue. Bahry, E. et al. RS-FISH: precise, interactive, fast, and scalable FISH spot detection. Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Jiang, Y., Lee, M.-L. T., He, X., Rosner, B. & Yan, J. Wilcoxon rank-based tests for clustered data with R package clusrank. Riquelme-Guzmán, C. & Sandoval-Guzmán, T. Methods for studying appendicular skeletal biology in axolotls. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Grabherr, M. G. et al. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Gu, Z., Eils, R. & Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Trofka, A. et al. Genetic basis for an evolutionary shift from ancestral preaxial to postaxial limb polarity in non-urodele vertebrates. Clement, K. et al. CRISPResso2 provides accurate and rapid genome editing sequence analysis. The authors thank Guoxin Rong for his imaging expertise and assistance with microscopy. Additionally, we thank Prayag Murawala for providing transgenic animals and the Ambystoma Genetic Stock Center for non-transgenic animals. We thank Malcolm Maden for his critical analysis of the manuscript. We finally thank the Institute for Chemical Imaging of Living Systems at Northeastern University for consultation and imaging support. Non-transgenic animals were obtained from the Ambystoma Genetic Stock Center funded through NIH grant P40-OD019794 obtained by S.R.V. Northeastern University, Department of Biology, Boston, MA, USA Timothy J. Duerr, Melissa Miller, Dareen Bakr, Jackson R. Griffiths, Aditya K. Gautham, Danielle Douglas & James R. Monaghan Northeastern University, Institute for Chemical Imaging of Living Systems, Boston, MA, USA Timothy J. Duerr, Sage Kumar & James R. Monaghan University of Kentucky, Spinal Cord and Brain Injury Research Center, Department of Neuroscience, Ambystoma Genetic Stock Center, Lexington, KY, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed and conceived the project. obtained funding and supervised the project. All authors contributed to the final draft. The authors declare no competing interests. Nature Communications thanks Maximina Yun, who co-reviewed with Catarina Rodrigues Olivera; Mekayla Storer, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Duerr, T.J., Miller, M., Kumar, S. et al. Retinoic acid breakdown is required for proximodistal positional identity during axolotl limb regeneration. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41562-025-02193-7'>Genetics of monozygotic twins reveals the impact of environmental sensitivity on psychiatric and neurodevelopmental phenotypes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 09:03:57
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Individual sensitivity to environmental exposures may be genetically influenced. This genotype-by-environment interplay implies differences in phenotypic variance across genotypes, but these variants have proven challenging to detect. Genome-wide association studies of monozygotic twin differences are conducted through family-based variance analyses, which are more robust to the systemic biases that impact population-based methods. We combined data from 21,792 monozygotic twins (10,896 pairs) from 11 studies to conduct one of the largest genome-wide association study meta-analyses of monozygotic phenotypic differences, in children, adolescents and adults separately, for seven psychiatric and neurodevelopmental phenotypes: attention deficit hyperactivity disorder symptoms, autistic traits, anxiety and depression symptoms, psychotic-like experiences, neuroticism and wellbeing. The proportions of phenotypic variance explained by single-nucleotide polymorphisms in these phenotypes were estimated (h2 = 0–18%), but were imprecise. This is the largest genetic study of monozygotic twins to date by an order of magnitude, evidencing an alternative method to study the genetic architecture of environmental sensitivity. This is evidenced when a genotype is associated with phenotype levels only under certain environmental conditions. Environmental sensitivity can also increase the variance of a trait if a genotype produces a wide range of phenotypes depending on environmental exposures, which may or may not also affect its population mean. Genetic knowledge of environmental sensitivity is most consistently exploited in bioengineering and evidenced in behavioural ecology, but it has been extremely challenging to evaluate in humans, especially for psychiatric disorders3. Understanding the genetic basis of environmental sensitivity is crucial for improving human health, as it informs on the biological pathways implicated in variations in responses to environmental exposures. In contrast with most GWASs, which estimate associations of genetic variants and phenotypic means, Genome-wide variance quantitative trait locus (vQTL) analysis4,5 aims to discover genetic variants associated with phenotypic variance, which can be prioritized for a statistical test of gene–environment interaction6. It is therefore challenging to robustly determine which potential mechanisms have given rise to the observed trait variance associated with a vQTL. Furthermore, commonly used population-based methods for estimating genetic effects using unrelated individuals suffer from variance inflation9, bias due to insufficient correction of demographic and indirect genetic effects10, and unstable test statistics for vQTLs when tested loci are in linkage disequilibrium with additive effects11. GWASs of monozygotic (MZ) twin differences provide an alternative, family-based approach to estimating vQTLs that is less susceptible to these sources of bias12 and can therefore more reliably identify variants that reflect environmental sensitivity. Therefore, within-pair phenotypic differences are probably due to chance or the environment13. Somatic mutations may also play a role in MZ differences for chromosomal and rare diseases14, but are unlikely to play a systematic role in common and polygenic complex traits15. Although all MZ twin pairs have the same degree of genetic similarity, they have varying degrees of phenotypic similarity. Greater within-pair differences in a population of MZ twins therefore reflect the pairs' greater sensitivity to their non-shared environments. Jinks and Fulker16 proposed a test in MZ twins for which the association between MZ pair differences and MZ pair mean score is obtained, and they provided a full description of its standard biometrical terms. The rationale behind their test of genotype–environment interaction is the same one that underlies tests involving inbred lines of animals that detect genotype–environment interaction through the heterogeneity of within-strain variances. We may now take single-nucleotide polymorphisms (SNPs) as measured genotype indicators and test for associations with within-pair differences. A GWAS of MZ phenotypic differences can identify the loci associated with variations in environmental sensitivity while also controlling for dynastic and epistatic effects, which are difficult to account for in population-based approaches (Fig. Although this approach has been advocated for because it facilitates understanding of the genetics of environmental sensitivity12, the requirement for a large sample of MZ twins has been a major impediment to progress in this field. In this Article, we report the findings from a GWAS meta-analysis of MZ differences for seven psychological phenotypes, using data from up to 21,792 MZ twins (10,896 pairs) from 11 studies. This is the largest GWAS conducted on MZ twin differences to date, representing an order of magnitude more participants than two previous MZ twin differences GWAS17,18. We conducted meta-analyses separately for children, adolescents and adults and identified 13 genome-wide significant associations across the phenotypes studied. This enabled us to estimate the SNP heritability (that is, the proportion of a trait's variance that can be attributed to genetic variation explained by SNPs) of environmental sensitivity to various mental health phenotypes (adolescent attention deficit hyperactivity disorder (ADHD): h2 = 0.18 and s.e. We also found that higher genetic liability to depression was associated with greater environmental sensitivity to depression in adults. These analyses are conducted in three main steps: First, a quantitative phenotype value is obtained for a population of MZ twin pairs. Second, the absolute phenotypic difference score is calculated for each MZ pair. This score reflects phenotypic differences due to environmental effects, as the environment makes genetically identical twins diverge phenotypically. The absolute phenotypic difference is corrected for age, sex, ten genetic principal components and any study-specific covariates. The residuals are standardized and inverse rank transformed. Third, a GWAS of the MZ differences score is conducted, using the phenotypic differences score for one twin from each pair and their genotype data; therefore, the sample comprises unrelated individuals. 2 provides a flow chart for the study). Authors of contributing studies were asked to conduct GWASs of MZ differences separately for samples from child, adolescent and adult twins if repeated measures across lifespan were available. The study-level GWAS results were subjected to quality control and harmonization (Supplementary Information section 2.2) using EasyQC (version 23.8)19, then meta-analysed using the inverse-variance-weighted fixed-effect meta-analysis method in METAL (2011 release)20. GWAS meta-analyses were conducted using the largest available sample from each study and separately in developmentally stratified samples (Methods). GWAS meta-analysis results were quality controlled (Methods), then used for gene-based and gene set analyses using MAGMA (version 1.08) in the FUMA web application (version 1.5.2)21 and SNP heritability analyses using LDSC (version 1.0.1)22. We used Mendelian randomization to estimate the effects of psychological phenotypes (as reflected in GWAS associations with means) on phenotypic variance (Methods). Flow chart genome-wide association meta-analysis of MZ twin differences in the present study. If multiple data points were available during the same developmental stage (for example, ages 7, 8 and 11 years), the largest sample was selected. n values for GWAS analyses were halved as only data from one twin from each pair were included. Table 1 shows descriptive statistics of the GWAS meta-analysis samples per phenotype. The largest sample was for depression symptoms, with 21,792 MZ twins from 11 studies. The smallest was for psychotic-like experiences (PLEs; 3,636 twins from two cohorts). Overall, within-twin MZ correlations tended to be lower in samples from adult twins than those from child twins. Meta-analyses for each phenotype identified two genome-wide significant variants (Table 2): one associated with variability in wellbeing (rs2940988; P = 9.93 × 10−9), located in the intronic region of the protein-coding chromosome 4 open reading frame 19 (C4orf19) gene; and one (rs60358762; P = 5.07 × 10−9) associated with variance in anxiety symptoms in adults, located in the intergenic region of the protein-coding SLC15A1 gene on chromosome 13. Manhattan plots, quantile–quantile (QQ) plots and the genomic regions of these genome-wide significant variants are presented in Fig. No variants were genome-wide significantly associated with variance in the other phenotypes (Supplementary Table 4 provides the top SNPs for each phenotype). 7–11 provide Manhattan and QQ plots for all of the tested phenotypes. a, Manhattan plots based on GWASs of MZ differences for wellbeing (left) and anxiety in samples from adults (right). b, Regional plots of independent significant SNPs for wellbeing (left; rs2940988; β = −0.16; P = 9.93 × 10−9) and anxiety (right; rs60358762; β  = 0.44; P = 5.07 × 10−9). SNPs not in linkage disequilibrium with significant independent lead SNPs in the selected region are coloured grey. The MAGMA gene-based analysis (Methods) found several genes associated with phenotypic variability; however, only two associations passed the threshold for Bonferroni correction for multiple testing (Table 2). The patched 1 (PTCH1) gene was associated with variance in depression (P = 1.80 × 10−6) and the chromosome 15 open reading frame 38 (C15orf38) gene was associated with variance in anxiety symptoms (P = 2.00 × 10−7). 12–16 and Supplementary Table 6 provide the top genes per phenotype. The MAGMA competitive test of gene sets (Methods) identified nine significant associations after Bonferroni correction for multiple testing (Table 2). Two gene sets were significantly associated with variance in depression symptoms, two with neuroticism, three with PLEs and two with autistic traits (one in samples from adults and one in samples from children). Supplementary Table 7 provides the top gene sets per phenotype, Supplementary Tables 8–11 provide details of significant gene sets and Supplementary Table 12 provides biological annotations for all genome-wide significant results. We estimated the SNP heritability of MZ differences using LDSC (version 1.0.1)22 (Methods). The SNP heritability estimate of environmental sensitivity to ADHD symptoms in the samples from adolescents was 0.18 (s.e. The SNP heritability estimate for environmental sensitivity to adult autistic traits was 0.09 (s.e. All estimates, including those from the remaining phenotypes, were not statistically significant (Supplementary Table 13). We could not estimate the genetic correlation (rg) between all phenotypes because the SNP heritabilities were too low and imprecise, except for child and adolescent symptoms of ADHD (rg = 0.82; s.e. Overall, despite this being one of the largest samples of MZ twins to date, low power resulted in large confidence intervals around the heritability estimates. It has previously been speculated that environmental sensitivity might relate to polygenic liability rather than single loci, due to the environment interacting with a polygenic biological component23. We used Mendelian randomization to estimate the influence of the genetic liability of psychological phenotypes on their environmental sensitivity (Methods). We found a strong effect for depression (β = 0.84; s.e. We also ran the analyses separately for our samples from children and those from adults (Fig. The signal for depression was driven by analyses in adults (β = 1.58; s.e. = 0.29; P = 5 × 10−7), with the childhood association attenuated (β = 0.35; s.e. There was little evidence for heterogeneity of effect estimates across depression liability instruments. Other phenotypes did not exhibit an influence of liability on environmental sensitivity after correcting for multiple testing (Supplementary Table 14). We found little evidence that mean body mass indices and years of schooling affected phenotypic variance. Since vQTL effects could be biased by main effects, we conducted simulations to evaluate the sensitivity of the analytical approach to the Mendelian randomization results being driven by this bias (Supplementary Information section 7). We found that the MZ differences model can be liable to this problem under some phenotype normalization approaches, but that this was less likely to be driving the results in this study because we normalized by inverse rank transformation (Supplementary Fig. Genetic liability to depression, based on population studies is shown on the x axes and variance in depression scores (using 102 variants found to associate with depression incidence) is shown on the y axes. Genetic liability to depression increased the variability in depression outcomes and this was attenuated to the null in children (interaction term in linear regression P = 0.008). We used simulations to investigate the properties of the method used to detect MZ twin-based variance loci and compared them with those of population-based vQTL methods. Second, we found that for moderate heritability the MZ differences approach has substantially greater statistical power than the population-based approach when sample sizes are equal (that is, 10,000 MZ twin pairs using the MZ twin-based method versus 20,000 unrelated individuals using the population-based vQTL method). However, in practice, the number of population-based samples generally available drastically outstrips the number of MZ twin samples available. When using a more realistic sample size (for example, 10,000 MZ twin pairs versus 500,000 population samples), the MZ differences approach only achieves similar power to the population approach when narrow-sense heritability values are very high (for example, >0.9). It has recently been shown that a variant tested for interaction can have inflated test statistics when in linkage disequilibrium with a strong additive effect11, and we investigated whether that mechanism can also adversely impact MZ twin-based estimates. Our simulations show that this problem is substantially exacerbated through population-based vQTL methods compared with direct interaction tests, but the MZ twin-based approach is robust to this bias (Fig. Comparisons were made between the MZ pairs approach and two different sample sizes for the population-based approach (one matching (with double the number of unrelated individuals; 20,000) and one with a typical sample size for modern GWASs (500,000)). b, False discovery rates (y axis) due to incomplete linkage disequilibrium (LD) with a tagging causal variant (x axis), compared between the MZ differences approach (left) and population-based approach (right). Several genome-wide significant results were notable, including our finding that the PTCH1 gene was associated with variation in depression symptoms, as this gene has previously been reported to be associated with depression-related phenotypes, including neuroticism24,25, anxiety26, depression symptoms24, feeling emotionally hurt27 and sensitivity to environmental stress and adversity27. The C15orf38 gene (also known as ARPIN-AP3S2) was associated with variance in anxiety symptoms in our samples from children and has previously been associated with type 2 diabetes in adults25,28 and corticotropin-releasing factor protein levels29, which are involved in regulating anxiety, mood, eating and inflammation30. Hypoglycaemia symptoms in type 2 diabetes include a rapid heartbeat, sweating and nervousness, all of which are physical sensations associated with anxiety. It is possible that certain variants in this gene impact sensitivity to the effects of diet and stressors that are involved in the variability in insulin31, unpleasant physical sensations of which may be contextualized and made sense of as worries and anxieties (for example, the two-factor model of emotions32). For autistic traits, the identified gene set included genes involved in tissue morphogenesis and healing that regulate the response to transforming growth factor beta (TGFβ1) levels and are involved in tissue repair pathways33. Growth factors serve important roles in neurodevelopment, immune function and development of the central nervous system and there is evidence that autism is associated with TGFβ1 and other growth factor-encoding genes34,35,36. For PLEs, the identified gene sets were related to the regulation of dopamine and catecholamine uptake. We also estimated the SNP heritability of environmental sensitivity (adolescent ADHD: h2 = 0.18 and s.e. = 0.09), but the estimates were not statistically significant. We also showed that variants that affect mean levels of depression and anxiety influence the variance of these phenotypes. Therefore, our study provides independent evidence that mean effects can influence variance. The findings should therefore be considered in light of our statistical power. Furthermore, as with all empirical analyses, our inferences depend on specific assumptions that may not hold. First, we took the MZ differences score to reflect genuine phenotypic variability. MZ differences, however, also reflect measurement errors that are difficult to separate from genuine phenotypic differences, and it is unclear whether MZ differences are stable across time. Future projects should investigate this approach with other phenotypes with low measurement errors, such as height, and consider a longitudinal design to assess the stability of these differences. Second, it is challenging to determine which mechanisms explain phenotypic variance. Genetic sensitivity to the environment and epigenetic processes, such as DNA methylation, imprinting, chorionicity and skewed X inactivation (for female MZ twins), also contribute to MZ phenotypic differences15,39. Also, the people included in our samples were all of European genetic ancestry since data from twins with DNA in other ancestries were not available in sufficient sample sizes. Our findings may not be generalizable to non-European genetic ancestries. The current study underlines the utility of DNA data from twins and should encourage funding for genetic data collection in multi-ancestry twin cohorts. In summary, we have identified novel genetic factors associated with phenotypic variability. Our study illustrates the importance of large meta-analyses of genotyped MZ twin samples as a method for discovering and understanding the genetics of phenotypic variance and environmental sensitivity. This research complies with all of the ethical regulations related to the secondary analysis of data collected by various cohorts, each of which obtained ethical approval and informed consent. The collection and use of biological material and survey information was approved by the Regional Committees on Health Research Ethics for Southern Denmark. This study is registered in Southern Denmrak University's internal list (notification number 11.059) and complies with the rules of the General Data Protection Regulation. Ethics approval was obtained for multiple studies, the latest of which was related to the transfer of all available DNA samples, genotypes and associated phenotypes to the THL Biobank by the Hospital District of Helsinki and Uusimaa ethics board in 2018 (1799/2017). Ethical approval was provided by the Central Committee on Research Involving Human Subjects of the VU University Medical Center in Amsterdam, an institutional review board certified by the US Office for Human Research Protections (IRB number IRB-2991 under Federal-wide Assurance-3703; IRB/institute codes, NTR 03-180). The procedures of this registry have been approved by the University of Murcia Research Ethics Committee. The Older Australian Twins Study (OATS) was approved by the human research ethics committees of the Australian Twin Registry, University of New South Wales, University of Melbourne, Queensland Institute of Medical Research (QIMR) and South Eastern Sydney and Illawarra Area Health Service. Written informed consent was provided by all participants. The different twin studies received separate approvals from the regional ethical review board in Stockholm (Dnr 80:80, 84:61; 93:226, 98:319 and 2010/657-31/3 (Swedish Adoption/Twin Study of Aging); Dnr 98:380 (OCTO-Twin); and Dnr: 97:051 and 2007/151-31/4 (HARMONY). Ethical approval for the PSYCH, TwinGene and Young Adult Twin Study in Sweden cohorts was given by the Uppsala Ethical Review Authority (2019-06066). Ethical approval for TEDS has been provided by the King's College London Ethics Committee (reference: PNM/09/10–104). Approval was obtained from the TwinsUK BioBank, approved by the North West–Liverpool Central Research Ethics Committee (reference 19/NW/0187; Integrated Research Application System ID = 258513). This approval supersedes earlier approvals granted to TwinsUK by the St Thomas' Hospital Research Ethics Committee and later by the London–Westminster Research Ethics Committee (reference EC04/015), which have now been subsumed within the TwinsUK BioBank. Figure  3 provides a flow chart for the current study. The eligibility criteria for inclusion of a study's data in the present study analyses included: (1) cohorts comprising at least 100 pairs of MZ twins; (2) genotyping data for one or both twins; (3) availability of imputed genotype data (for example, 1000 Genomes or Haplotype Reference Consortium (HRC) data); (4) complete data from both twins for one or more phenotypes (imputation of missing data for incomplete pairs is not recommended); (5) complete covariate data for both twins (that is, age, sex and principal components for the genotyped twin); and (6) samples of European ancestry. The analysis plan and pipeline were written, pre-specified and shared with interested cohorts to enable them to conduct GWASs of MZ differences for their available phenotypes, then the results were uploaded to a designated repository (https://github.com/LaurenceHowe/MZTwins-vQTL). For some participating studies, data were available across the lifespan of participants, thus comprising repeated measures for certain phenotypes. It was therefore possible to explore genetic associations in the context of development by conducting developmentally stratified analyses. GWAS analyses were conducted separately for different cohorts and for each developmental stage if data were available. For anxiety symptoms, depression symptoms and autistic traits, GWAS data were available for children and adults. For wellbeing and neuroticism, the samples comprised adults only, and for PLEs the samples comprised adolescents only. Two sets of meta-analyses were then conducted using the GWAS results: a developmentally informed analysis, whereby GWAS results across studies were grouped according to the developmental stage of the sample and meta-analyses were conducted per phenotype (for example, depression child or depression adult); and a developmentally agnostic analysis, whereby a meta-analysis was conducted for each phenotype using the largest sample from each study, regardless of the developmental stage (for example, depression largest or anxiety largest). This ensured maximum power for meta-analysis per phenotype via the largest n value (Supplementary Information section 1.1 provides more details on study design). The samples analysed in this study included MZ twin pairs from cohort studies or twin registries in Australia and Europe (Supplementary Information Section 1.2 and Supplementary Table 1 provide details of the participating studies). Informed consent and ethical approvals were obtained for all participating cohorts (Supplementary Information). Table 1 shows the total sample sizes per phenotype and across developmental groups. The MZ differences method requires continuous or categorical non-binary phenotypic data to calculate variance. Therefore, we used mean symptom scores instead of case/control diagnosis and the preference was for continuous measures. Various instruments have been developed for the assessment of psychological phenotypes, as was reflected in the participating studies. If multiple rating scales of a phenotype were available, study authors were asked to select the scale with the most items (tapping most symptom domains). Absolute phenotypic differences were obtained for each MZ pair. Using linear regression, absolute phenotypic differences were corrected for age, sex, ten genetic principal components and any study-specific covariates. A brief description of each phenotype is provided below (Supplementary Information Section 1.1 gives details). ADHD is a childhood-onset neurodevelopmental disorder of attention, activity and impulsivity. They are typically measured continuously using rating scales, often with separate scales for attention problems and hyperactivity or impulsivity, which can be summed to give a total score of ADHD symptoms. We were interested in generalized anxiety symptoms, usually measured via self-report, and reflected in a total score of anxiety symptoms. Autism spectrum disorder is a neurodevelopmental disorder broadly reflecting difficulties in social interaction and verbal communication, as well as repetitive behaviours. Symptoms typically emerge in early childhood, and assessment is carried out via questionnaires and/or interviews. The continuous scores reflect the presence or extent of autism traits rather than a diagnosis of autism spectrum disorder. Depression is heterogeneous, with many clinical presentations. A diagnosis requires a distinct change of mood, characterized by sadness or irritability and accompanied by psychophysiological changes, such as disturbances in sleep and appetite or loss of the ability to experience pleasure. Neuroticism is a personality domain and refers to a lack of emotional stability, stress vulnerability, the tendency to experience intense negative emotions, affects and cognitions, and impulsive behaviours under emotional strain. Neuroticism is considered a risk factor for anxiety and depression. PLEs include a sub-clinical threshold of symptoms related to psychosis or schizophrenia disorders, such as persecutory ideation or perceptual abnormalities, that are prevalent in the community and non-clinical samples. PLEs are screened using self- or other-report questionnaires or interviews that cover some or all of these domains: paranoia, hallucinations, cognitive disorganization, anhedonia and negative symptoms. Wellbeing includes both hedonic and eudaimonic wellbeing and is typically assessed via questionnaires that index an individual's subjective sense of wellness, such as reporting satisfaction with one's life or being hopeful and optimistic about it. The data from participating studies mainly related to subjective wellbeing (for example, life satisfaction). We preferred to use data for which wellbeing had been measured using a battery of questions. Participating studies were required to have genotype data from all 22 autosomes imputed to either the 1000 Genomes reference panel (preferably phase 3) or the HRC. Almost all contributing studies had already participated in a project related to the Within Family Consortium10 and used the same protocol in the automated scripts for genetic data preparation and quality control procedures before GWAS analysis. Minimum quality control requirements at the study level included: filtering SNPs for an imputation quality of >0.3 for HapMap imputed data or >0.5 for 1000 Genomes or HRC data; a call rate of >95%; and a minor allele frequency of >1%. Studies also removed one pair randomly when there were two MZ pairs with a kinship of >0.1. Study-level genotyping and quality control information are included in the Supplementary Data. Methods using family-based design, such as the MZ differences method, complement population-based vQTL methods in three ways: (1) statistical power; (2) robustness to bias due to additive effects; and (3) providing an alternative identification strategy for triangulation40. To simulate vQTLs, we used the following data-generating model. where yi,t is the phenotypic value for twin t = {A,B} at MZ pair i = {1,…,n} and α is an intercept term, which is set to 0 for simplicity. zi is the remaining polygenic risk for the MZ pair, which is normally distributed with a mean of 0 for simplicity, and variance is defined as The variance heterogeneity term vi,t is modelled as such that each additional allele at G increases the within-MZ difference by a factor of β2. Finally, ei,t is the independent individual residual variance defined as where \({\sigma }_{{\rm{v}}}^{2}\) is the variance of vi,t. Therefore, for a pair (A and B) of MZs, the additive genetic factors Gi and zi are fixed, but the within-MZ variability is induced from the vi,t + ei,t terms. We estimated vQTL effects from unrelated individuals (choosing one MZ at random) using the deviation regression model from Marderstein and colleagues41. We estimated vQTL effects using MZs and the following MZ difference model: where \({\hat{\beta }}_{2}\) is an estimate of the vQTL effect, and ϵi is the residual error from this regression. We investigated the power of each method by generating vQTL effects (β2) calibrated to have 80% statistical power for the deviation regression model method in 500,000 unrelated individuals (with parameters p = 0.3 and β1 = 0). We then estimated how the power of the MZ differences approach varies for these parameters across a range of narrow-sense heritability values \({{\sigma }^{2}}_{{\rm{g}}}={h}^{2}\). Briefly, the data-generating model described above is simulated with an additive effect of β1 = 0.1 generated, but the vQTL test is performed at a second locus, G*, that is generated to be in linkage disequilibrium with G. The simulations were then performed with no vQTL effect (β1 = 0), variance of the linkage disequilibrium between G and G* (r2 = 0,…,1), h2 = 0.5 and p = 0.1, and 500 repeats were performed for each parameter combination. In our primary analysis, we estimated the association between the absolute phenotypic difference between MZ twins (residualized for age, sex and principal components and then standardized and rank transformed) and the genetic marker using linear regression for each SNP, j: We constructed two further models for sensitivity analyses: model 2, for which the within-twin mean of the phenotype was a covariate in the regression; and model 3, which differed from our primary model by not adjusting for principal components when constructing the phenotype. However, this also risks over-correcting, especially for vQTLs, which affect both the mean and variance of a phenotype, as was indicated here by a small-to-moderate (r = ~0.3–0.6) positive correlation between the MZ phenotypic mean and MZ phenotypic differences in our sample (as was previously proposed42). Model 3 was constructed since some participating studies were likely to be very small (fewer than 300 participants). Including ten principal components for all studies might have been overly conservative, leading to an inflation of P values. The results indicated that models 2 and 3 were not significantly different from model 1 (Supplementary Table 3a–c). Therefore, we considered model 1 to be the most parsimonious, with a lower number of parameters than model 2 but similar results, while also correcting for population stratification confounding. The remaining analyses were therefore conducted using model 1 results only. Variants with missing estimate (β) values, standard error (s.e.) values, statistical significance (P) values or imputation quality score and SNPs with a minor allele frequency of <0.01 or an imputation quality score of <0.5 were removed. Cptid identifiers (chr:bp:A1:A2) were created and alleles, effects and frequencies were checked in all GWAS results and harmonized according to their respective reference panel (1000 Genomes phase 3 version 5 or the HRC). Indels, monomorphic SNPs and duplicate SNPs that could also be tri-allelic (with the same base pair position but different alleles) were removed, retaining only the SNP with the largest sample. Manhattan and QQ plots were obtained and lambda-median values were inspected for P value inflation (Supplementary Figs. METAL (2011 release)20 was used to conduct inverse-variance-weighted fixed-effect meta-analysis across studies, per phenotype. Cptid IDs were mapped into rsIDs from the 1000 Genomes phase 3 version 5 European panel. The SNP2GENE function in the FUMA web application (version 1.5.0)21 was used to annotate GWAS SNPs and identify independent significant SNPs (SNPs in linkage disequilibrium with the lead SNP at r2 = 0.1; lead SNPs are those in linkage disequilibrium with any independent significant SNPs with r2 > 0.6; Supplementary Table 5) and to produce regional, QQ and Manhattan plots (Supplementary Figs. MAGMA (version 1.08) in the FUMA web application (version 1.5.2)21 was used to annotate GWAS SNPs and conduct gene-based and gene set analyses. Meta-analysed GWAS results were filtered to include only SNPs available in at least 50% of studies. SNPs were annotated to Ensembl version 92 protein-coding genes for gene-based analyses using default parameters (a SNP-wise model for gene testing). Curated gene sets contained nine data resources, including the Kyoto Encyclopedia of Genes and Genomes, Reactome and BioCarta. Gene Ontology terms comprised three categories: biological processes, cellular components and molecular functions. The major histocompatibility complex region was excluded from all annotations (Supplementary Table 5 provides details). SNP heritability estimates per phenotype were obtained using LDSC (version 1.0.1)22. The European 1000 Genomes linkage disequilibrium scores generated by the authors of LDSC were used, and SNPs for heritability analyses were merged with a set of ~1.2 million high-quality SNPs defined by the authors of LDSC22. For ADHD symptoms, data were available for children (5–12 years of age) and adolescents (13–18 years of age), whereas for anxiety, depression and autistic traits, data were available for both children and adults (>18 years of age). The stratified analyses included a meta-analysis of GWAS results separately for samples from children, adolescents and adult for these phenotypes, followed by gene-based, gene set and heritability analyses, using the same criteria as the largest non-stratified sample. We used a two-sample summary data Mendelian randomization to estimate the influence of the genetic liability of psychological phenotypes on their environmental sensitivity. Mendelian randomization uses genetic variants as instrumental variables for the exposure of interest. We used the summary data from the MZ twins GWAS as an outcome in our Mendelian randomization analyses and investigated whether differences in the mean of each exposure (for example, depression) affect the variance in each psychiatric outcome. This assumption would be violated if these variants also affected the outcome via their effects on the variance in depression, independent of their effects on the mean liability to depression. Under such a scenario, our estimates may be biased. Theoretically, this bias could be in either direction. More methodological research would be useful to determine the likelihood and magnitude of these potential biases. We selected 102 independent (linkage disequilibrium = 10,000 kilobases; r2 = 0.001) genetic variants associated with depression in Howard et al.43 as instruments for genetic liability to depression. We harmonized the effects by effect allele, chromosome and position on build hg19. The variants were strongly associated with depression and there was minimal overlap between our samples and those used in the GWASs to select variants. We followed the same procedure for the other phenotypes, but with a different number of SNPs (Supplementary Table 14). Because depression is strongly genetically correlated with anxiety but there are no well-powered GWASs for anxiety, we performed a similar analysis for variance in anxiety but using the 102 variants for depression. Here the main effects for anxiety at each of the 102 variants were obtained from a GWAS using UK Biobank data on self-reported anxiety measures46. Finally, we also tested educational attainment47 and body mass index. GWAS summary statistics for main effects were obtained from OpenGWAS48. We reported this analysis using the STROBE-MR checklist49. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Accession codes for the GWAS meta-analyses are as follows: ieu-b-5146 (adolescent) and ieu-b-5147 (child) for MZ twin differences in ADHD symptoms; ieu-b-5148 (largest), ieu-b-5149 (adult) and ieu-b-5150 (child) for MZ twin differences in anxiety symptoms; ieu-b-5151 (largest), ieu-b-5152 (adult) and ieu-b-5153 (child) for MZ twin differences in autistic traits; ieu-b-5154 (largest), ieu-b-5155 (adult) and ieu-b-5156 (child) for MZ twin differences in depression symptoms; ieu-b-5157 for MZ twin differences in neuroticism score; ieu-b-5158 for MZ twin differences in PLEs; and ieu-b-5159 for MZ twin differences in subjective wellbeing. Data from individual studies are not publicly available and are subject to strict access control because the consent given by the participants does not allow for data storage on an individual level in repositories or journals. Access to these data requires specific approval from the relevant data access committees for each cohort. Mapping and allele frequency reference files (all based on National Center for Biotechnology Information build 37) for 1000 Genomes phase 1 version 3, 1000 Genomes phase 3 version 5 and the HRC are available via https://www.uni-regensburg.de/medizin/epidemiologie-praeventivmedizin/genetische-epidemiologie/software/index.html. Scripts for GWAS analyses are available from GitHub at https://github.com/Elham-Assary/MZ-differences-GWAS. Berg, K., Kondo, I., Drayna, D. & Lawn, R. “Variability gene” effect of cholesteryl ester transfer protein (CETP) genes. On the use of variance per genotype as a tool to identify quantitative trait interaction effects: a report from the Women's Genome Health Study. Yang, J. et al. FTO genotype is associated with phenotypic variability of body mass index. Wang, H. et al. Genotype-by-environment interactions inferred from genetic effects on phenotypic variability in the UK Biobank. Selection on noise constrains variation in a eukaryotic promoter. Ek, W. E. et al. Genetic variants influencing phenotypic variance heterogeneity. Keller, M. C. Gene × environment interaction studies have not properly controlled for potential confounders: the problem and the (simple) solution. Howe, L. J. et al. Within-sibship genome-wide association analyses decrease bias in estimates of direct genetic effects. Hemani, G. et al. Phantom epistasis between unlinked loci. Visscher, P. M. & Posthuma, D. Statistical power to detect genetic loci affecting environmental sensitivity. Some causes of genotypic and phenotypic discordance in monozygotic twin pairs. Genetic or epigenetic difference causing discordance between monozygotic twins as a clue to molecular basis of mental disorders. Jinks, J. L. & Fulker, D. W. Comparison of the biometrical genetical, MAVA, and classical approaches to the analysis of human behavior. Surakka, I. et al. A genome-wide association study of monozygotic twin-pairs suggests a locus related to variability of serum high-density lipoprotein cholesterol. Keers, R. et al. A genome-wide test of the differential susceptibility hypothesis reveals a genetic predictor of differential response to psychological treatments for child anxiety disorders. Winkler, T. W. et al. Quality control and conduct of genome-wide association meta-analyses. & Abecasis, G. R. METAL: fast and efficient meta-analysis of genomewide association scans. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. Bulik-Sullivan, B. K. et al. LD score regression distinguishes confounding from polygenicity in genome-wide association studies. Coleman, J. R. I. et al. Genome-wide gene–environment analyses of major depressive disorder and reported lifetime traumatic experiences in UK Biobank. Baselmans, B. M. L. et al. Multivariate genome-wide analyses of the well-being spectrum. Kichaev, G. et al. Leveraging polygenic functional enrichment to improve GWAS power. Thorp, J. G. et al. Symptom-level modelling unravels the shared genetic architecture of anxiety and depression. Nagel, M., Watanabe, K., Stringer, S., Posthuma, D. & van der Sluis, S. Item-level analyses reveal genetic heterogeneity in neuroticism. Mahajan, A. et al. Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps. Pietzner, M. et al. Mapping the proteo-genomic convergence of human diseases. Dautzenberg, F. M. & Hauger, R. L. The CRF peptide family and their receptors: yet more partners discovered. Nolan, C. J., Damm, P. & Prentki, M. Type 2 diabetes across generations: from pathophysiology to prevention and management. Schachter, S., Fau-Singer, J. E. & Singer, J. E. Cognitive, social, and physiological determinants of emotional state. Toyoda, T. et al. SNP analyses of growth factor genes EGF, TGFβ-1, and HGF reveal haplotypic association of EGF with autism. Decreased serum levels of epidermal growth factor in adult subjects with high-functioning autism. Howes, O., McCutcheon, R. & Stone, J. Glutamate and dopamine in schizophrenia: an update for the 21st century. Taylor, M. J., Freeman, D., Lundström, S., Larsson, H. & Ronald, A. Heritability of psychotic experiences in adolescents and interaction with environmental risk. & Wong, A. H. Not really identical: epigenetic differences in monozygotic twins and implications for twin studies in psychiatry. Lawlor, D. A., Tilling, K. & Davey Smith, G. Triangulation in aetiological epidemiology. Marderstein, A. R. et al. Leveraging phenotypic variability to identify genetic interactions in human phenotypes. Howard, D. M. et al. Genome-wide meta-analysis of depression identifies 102 independent variants and highlights the importance of the prefrontal brain regions. Sadreev, I. I. et al. Navigating sample overlap, winner's curse and weak instrument bias in Mendelian randomization studies using the UK Biobank. Dönertaş, H. M., Fabian, D. K., Valenzuela, M. F., Partridge, L. & Thornton, J. M. Common genetic associations between age-related diseases. Okbay, A. et al. Genome-wide association study identifies 74 loci associated with educational attainment. King, T., Butcher, S. & Lukasz, Z. Apocrita—high performance computing cluster for Queen Mary University of London. We dedicate this paper to Robert Keers, a wonderful friend, colleague, mentor and scientist, whom we lost too soon. This project continued in his absence, with contributions from many colleagues who helped to complete the work that he started. for stepping in to lead the project following Rob's passing. This project was funded by: a Wellcome Trust grant (project number 208881/Z/17/Z awarded to R.K.); the National Institute for Health and Care Research (NIHR) Barts Biomedical Research Centre—a delivery partnership of Barts Health NHS Trust, Queen Mary University of London, St George's University Hospitals NHS Foundation Trust and St George's University of London (NIHR203330 awarded to P.B.M. ); the Norwegian Research Council (grant number 295989), Division of Psychiatry at University College London, Medical Research Council (MRC; MR/V002147/1) and National Institute of Mental Health (MH130448) (all awarded to N.M.D. ); the Australian National Health and Medical Research Council (NHMRC) (grant APP1173025 awarded to K.L.G. ); National Institutes of Health (NIH) grants R01AG060470, R01AG059329, R01AG050595 and R01AG046938 (awarded to C.A.R. ); Riksbankens Jubileumsfond (P18-0782:1) and the Swedish Research Council (2019-00244) (both awarded to S.O. ); a King's International Postgraduate Research Scholarship (awarded to K.L. ); the Research Council of Norway (288083) and Jacobs Foundation (2023-1510-00) (both awarded to R.C. ); the European Research Council (ERC-2017-COG 771057 (WELL-BEING) awarded to MPVDW and M.B. ); the Academy of Finland (grants 265240 and 263278) and Sigrid Jusélius Foundation (all awarded to J.K.); Academy of Finland grant 308698 (awarded to A.L. ); and the NIHR Maudsley Biomedical Research Centre (awarded to J.R.I.C.). The Danish Twin Registry has received support from the National Program for Research Infrastructure 2007 (09-063256), Danish Agency for Science, Technology and Innovation, and NIH (P01 AG08761). Genotyping at the Danish Twin Registry was conducted by the SNP&SEQ Technology Platform's Science for Life Laboratory (http://snpseq.medsci.uu.se/genotyping/snp-services/) and supported by NIH grant R01AG037985 (to N.L.P.). The Finnish Twin Cohort's phenotype and genotype data collection has received support from the Wellcome Sanger Institute, Broad Institute, European Network for Genetic and Genomic Epidemiology (FP7-HEALTH-F4-2007; grant agreement 201413), National Institute on Alcohol Abuse and Alcoholism (grants AA-12502, AA-00145 and AA-09203 to R. J. The OATS was facilitated through access to Twins Research Australia, a national resource supported by a Centres of Research Excellence Grant (1079102) from the NHMRC. OATS genotyping was partly funded by a Commonwealth Scientific and Industrial Research Organisation Flagships Collaboration Fund grant. The Swedish Twin Registry is funded by the Swedish Research Council (2017-0064 and 421-2013-1061) and Ragnar Söderberg Foundation (E9/11). PsychChip GWAS genotyping and collaborative work at the Swedish Twin Registry was supported in part by NIH/National Institute on Aging grants R01AG037985, R01AG059329 and R01AG060470; DNA extraction was supported by grants R01AG17561 and R01AG028555; HARMONY was supported by grant R01AG08724; OCTO-Twin was supported by grant R01AG08861; gender was supported by the MacArthur Foundation Research Network on Successful Aging, Axel and Margaret Ax:son Johnson Foundation, Swedish Council for Social Research and Swedish Foundation for Health Care Sciences and Allergy Research; the Swedish Adoption/Twin Study of Aging was supported by grants R01AG04563 and R01AG10175, the MacArthur Foundation Research Network on Successful Aging, the Swedish Council For Working Life and Social Research (97:0147:1B, 2009-0795) and the Swedish Research Council (825-2007-7460 and 825-2009-6141). Research at the Swedish Twin Registry was also supported by Riksbankens Jubileumsfond (P18-0782:1) and the Swedish Research Council (2019-00244). The Twins Early Development Study is supported by a programme grant (MR/V012878/1) to T.C.E. from the UK Medical Research Council (previously MR/M021475/1 awarded to R.P. This research utilized the Queen Mary's Apocrita high-performance computing facility, supported by Queen Mary University of London's Research-IT services (https://doi.org/10.5281/zenodo.438045)50. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. These authors jointly supervised this work: Thalia C. Eley, Neil M. Davies, Patricia B. Munroe, Robert Keers. Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK Elham Assary, Jonathan R. I. Coleman, Kai Lim, Jean-Baptiste Pingault, Robert Plomin & Thalia C. Eley Margot P. van de Weijer, Jouke-Jan Hottenga & Meike Bartels QIMR Berghofer Medical Research Institute, Brisbane, Queensland, Australia Katrina L. Grasby, Lucia Colodro-Conde, Scott Gordon, Sarah E. Medland & Nicholas G. Martin Centre for Healthy Brain Ageing, Discipline of Psychiatry and Mental Health, School of Clinical Medicine, Faculty of Medicine and Health, UNSW Sydney, Sydney, New South Wales, Australia Anbupalam Thalamuthu, Nicola J. Armstrong, Perminder S. Sachdev, Teresa Lee, Henry Brodaty, Julian N. Trollor, Vibeke S. Catts & Karen A. Mather National Centre of Excellence in Intellectual Disability Health, Faculty of Medicine and Health, UNSW Sydney, Sydney, New South Wales, Australia University of Melbourne Academic Unit for Psychiatry of Old Age, St George's Hospital, Melbourne, Victoria, Australia Department of Psychiatry, Harvard Medical School, Boston, MA, USA Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Solna, Sweden Mark Taylor, Henrik Larsson, Robert Karlsson, Nancy L. Pedersen & Paul Lichtenstein Robin G. Walters, Sam Morris, Zhengming Chen, Kuang Lin & Iona Y. Millwood Department of Clinical, Educational and Health Psychology, Division of Psychology and Language Sciences, University College London, London, UK Behavioral Decision Making Area, Anderson School of Management, University of California, Los Angeles, Los Angeles, CA, USA Human Genetics Department, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA, USA National Bureau of Economic Research, Cambridge, MA, USA Department of Precision Medicine, School of Clinical Sciences at Monash Health, Monash University, Melbourne, Victoria, Australia Eco de Geus, Floris Huider & Wonu Akingbuwa Center for Behavioral Science and Mental Health, Istituto Superiore di Sanità, Rome, Italy Joohon Sung, Soo Ji Lee & Hyojin Pyun Institute for Molecular Biosciences, University of Queensland, Brisbane, Australia Centre for Genomc and Experimental Medicine, Institute of Genetics & Cancer, University of Edinburgh, Edinburgh, United Kingdom You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar were involved in conceptualizing and designing the study. were involved in data acquisition and collation. played a key role in supervising the analyses and P.B.M. For the Swedish Twin Registry's ageing study, C.A.R. For the Swedish Twin Registry's PSYCH, TwinGene and Young Adult Twin Study in Sweden studies, R.A. performed the analysis and S.O. All authors contributed to writing and critically reviewing the manuscript. Nature Human Behaviour thanks Pietro Biroli and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Assary, E., Coleman, J.R.I., Hemani, G. et al. Genetics of monozygotic twins reveals the impact of environmental sensitivity on psychiatric and neurodevelopmental phenotypes. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250610004054.htm'>How madagascar's lizards became the island's last hope for reforestation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-10 08:37:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>After the island of Madagascar drifted away from India 88 million years ago, isolating it from all other landmasses, its flora and fauna evolved in seclusion. As these transformed into plants and animals completely unique to their island, Madagascar became a biodiversity hotspot unlike anywhere else on Earth. An important facet of this biodiversity is an ecological process called endozoochory, which is when animals eat plant seeds and then poop them out somewhere else, aiding in the spread of the plants. Most research on endozoochory has focused on the roles of birds and mammals as seed dispersers, but lizards, which are also known play a significant role, have remained largely overlooked. Fewer than 10% of lizard species have been reported to consume fruits, but those that do can play an important role, and some lizards are even known to act as primary seed dispersers for certain plant species. "Lizards are under-appreciated as seed dispersers in many forest ecosystems, but we hypothesized that they may play a more important role across a broader range of regions than previously recognized," says corresponding author Ryobu Fukuyama. These plant species were largely different from those typically consumed by the Common Brown Lemur, a principal seed disperser in Madagascar's forests, indicating the lizards may play a more crucial role than previously thought. Their project isn't only about acknowledging the importance of lizards, however. In recent years, environmental degradation caused by human activities has taken a toll on Malagasy forests, making some uninhabitable for large frugivores like lemurs. As seed dispersers, these lizards could potentially contribute to forest regeneration, but there are still many unknowns. "Although lizards in Madagascar consume fruits from many plant species, other aspects of their role as seed dispersers, such as dispersal distances, remain poorly understood," says Fukuyama. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            