
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-23</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08801-w'>Long-distance coherent quantum communications in deployed telecom networks</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 19:34:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature volume 640, pages 911–917 (2025)Cite this article Recent advances in quantum communications have underscored the crucial role of optical coherence in developing quantum networks. This resource, which is fundamental to the phase-based architecture of the quantum internet1, has enabled the only successful demonstrations of multi-node quantum networks2,3,4 and substantially extended the range of quantum key distribution (QKD)5. However, the scalability of coherence-based quantum protocols remains uncertain owing to the specialized hardware required, such as ultra-stable optical cavities and cryogenic photon detectors. Here we implement the coherence-based twin-field QKD protocol over a 254-kilometre commercial telecom network spanning between Frankfurt and Kehl, Germany, achieving encryption key distribution at 110 bits per second. Our results are enabled by a scalable approach to optical coherence distribution, supported by a practical system architecture and non-cryogenic single-photon detection aided by off-band phase stabilization. Our results demonstrate repeater-like quantum communication in an operational network setting, doubling the distance for practical real-world QKD implementations without cryogenic cooling. In addition, to our knowledge, we realized one of the largest QKD networks featuring measurement-device-independent properties6. Our research aligns the requirements of coherence-based quantum communication with the capabilities of existing telecommunication infrastructure, which is likely to be useful to the future of high-performance quantum networks, including the implementation of advanced quantum communication protocols, quantum repeaters, quantum sensing networks and distributed quantum computing7. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data supporting the findings of this study are available in the paper and its Supplementary Information. Additional details can be obtained from the corresponding author upon request. The quantum internet. Google Scholar Pompili, M. et al. Realization of a multinode quantum network of remote solid-state qubits. Google Scholar Hermans, S. L. N. et al. Qubit teleportation between non-neighbouring nodes in a quantum network. Google Scholar Liu, J.-L. et al. Creation of memory–memory entanglement in a metropolitan quantum network. Google Scholar Lucamarini, M., Yuan, Z. L., Dynes, J. F. & Shields, A. J. Overcoming the rate–distance limit of quantum key distribution without quantum repeaters. Google Scholar Lo, H.-K., Curty, M. & Qi, B. Measurement-device-independent quantum key distribution. Google Scholar Wehner, S., Elkouss, D. & Hanson, R. Quantum internet: a vision for the road ahead. Bennett, C. H. & Brassard, G. Quantum cryptography: public key distribution and coin tossing. Google Scholar Ekert, A. K. Quantum cryptography based on Bell's theorem. Google Scholar Xu, F., Ma, X., Zhang, Q., Lo, H.-K. & Pan, J.-W. Secure quantum key distribution with realistic devices. Google Scholar Pirandola, S. et al. Advances in quantum cryptography. Google Scholar Buhrman, H., Cleve, R., Watrous, J. & de Wolf, R. Quantum fingerprinting. Google Scholar Buhrman, H., Cleve, R., Massar, S. & de Wolf, R. Nonlocality and communication complexity. Google Scholar Arrazola, J. M. & Lütkenhaus, N. Quantum fingerprinting with coherent states and a constant mean number of photons. Google Scholar Gottesman, D., Jennewein, T. & Croke, S. Longer-baseline telescopes using quantum repeaters. Google Scholar Kómár, P. et al. A quantum network of clocks. Google Scholar Degen, C. L., Reinhard, F. & Cappellaro, P. Quantum sensing. Google Scholar Distributed quantum phase estimation with entangled photons. Google Scholar Denchev, V. S. & Pandurangan, G. Distributed quantum computing. ACM SIGACT News 39, 77–95 (2008). Google Scholar Barz, S. et al. Demonstration of blind quantum computing. Google Scholar Ben-Or, M. & Hassidim, A. Fast quantum byzantine agreement. Thirty-seventh Annual ACM Symposium on Theory of Computing (eds Gabow, H. & Fagin, R.) 481–485 (ACM, 2005). Pfaff, W. et al. Quantum information. Unconditional quantum teleportation between distant solid-state quantum bits. Google Scholar Northup, T. E. & Blatt, R. Quantum information transfer using photons. Google Scholar Hensen, B. et al. Loophole-free Bell inequality violation using electron spins separated by 1.3 kilometres. Google Scholar Roberts, G. L. et al. Experimental measurement-device-independent quantum digital signatures. Google Scholar Bose, S., Knight, P. L., Plenio, M. B. & Vedral, V. Proposal for teleportation of an atomic state via cavity decay. Google Scholar Cabrillo, C., Cirac, J. I., García-Fernández, P. & Zoller, P. Creation of entangled states of distant atoms by interference. Google Scholar Duan, L. M., Lukin, M. D., Cirac, J. I. & Zoller, P. Long-distance quantum communication with atomic ensembles and linear optics. Google Scholar Bennett, C. H. et al. Teleporting an unknown quantum state via dual classical and Einstein–Podolsky–Rosen channels. Google Scholar Wang, X.-B., Yu, Z.-W. & Hu, X.-L. Twin-field quantum key distribution with large misalignment error. Curty, M., Azuma, K. & Lo, H.-K. Simple security proof of twin-field type quantum key distribution protocol. npj Quantum Inf. Google Scholar Zeng, P., Zhou, H., Wu, W. & Ma, X. Mode-pairing quantum key distribution. Google Scholar Xie, Y.-M. et al. Breaking the rate-loss bound of quantum key distribution with asynchronous two-photon interference. Pirandola, S., Laurenza, R., Ottaviani, C. & Banchi, L. Fundamental limits of repeaterless quantum communications. Google Scholar Minder, M. et al. Experimental quantum key distribution beyond the repeaterless secret key capacity. Google Scholar Pittaluga, M. et al. 600-km repeater-like quantum communications with dual-band stabilization. Google Scholar Chen, J.-P. et al. Twin-field quantum key distribution over a 511 km optical fibre linking two distant metropolitan areas. Google Scholar Chen, J.-P. et al. Quantum key distribution over 658 km fiber with distributed vibration sensing. Google Scholar Wang, S. et al. Twin-field quantum key distribution over 830-km fibre. Google Scholar Clivati, C. et al. Coherent phase transfer for real-world twin-field quantum key distribution. Google Scholar Liu, Y. et al. Experimental twin-field quantum key distribution over 1000 km fiber distance. Google Scholar Zhou, L., Lin, J., Jing, Y. & Yuan, Z. Twin-field quantum key distribution without optical frequency dissemination. Google Scholar Li, W. et al. Twin-field quantum key distribution without phase locking. Google Scholar Hadfield, R. H. Single-photon detectors for optical quantum information applications. Google Scholar Zhang, J., Itzler, M. A., Zbinden, H. & Pan, J.-W. Advances in InGaAs/InP single-photon detector systems for quantum communication. Humer, G. et al. A simple and robust method for estimating afterpulsing in single photon detectors. Jiang, C., Hu, X.-L., Xu, H., Yu, Z.-W. & Wang, X.-B. Zigzag approach to higher key rate of sending-or-not-sending twin field quantum key distribution with finite-key effects. Braunstein, S. L. & Pirandola, S. Side-channel-free quantum key distribution. Ye, J. et al. Delivery of high-stability optical and microwave frequency standards over an optical fiber network. Comandar, L. C. et al. Near perfect mode overlap between independently seeded, gain-switched lasers. Lo, H.-K., Ma, X. & Chen, K. Decoy state quantum key distribution. Beating the photon-number-splitting attack in practical quantum cryptography. Yuan, Z. L., Dixon, A. R., Dynes, J. F., Sharpe, A. W. & Shields, A. J. Gigahertz quantum key distribution with InGaAs avalanche photodiodes. Zhang, J. et al. 2.23 GHz gating InGaAs/InP single-photon avalanche diode for quantum key distribution. Hu, X.-L., Jiang, C., Yu, Z.-W. & Wang, X.-B. Sending-or-not-sending twin-field protocol for quantum key distribution with asymmetric source parameters. Panayi, C., Razavi, M., Ma, X. & Lütkenhaus, N. Memory-assisted measurement-device-independent quantum key distribution. Azuma, K., Tamaki, K. & Lo, H.-K. All-photonic quantum repeaters. Fröhlich, B. et al. Long-distance quantum key distribution secure against coherent attacks. Boaron, A. et al. Secure quantum key distribution over 421 km of optical fiber. Xu, H., Yu, Z.-W., Jiang, C., Hu, X.-L. & Wang, X.-B. Sending-or-not-sending twin-field quantum key distribution: breaking the direct transmission key rate. Liu, H. et al. Field test of twin-field quantum key distribution through sending-or-not-sending over 428 km. Pirandola, S. End-to-end capacities of a quantum communication network. Takeoka, M., Guha, S. & Wilde, M. M. Fundamental rate-loss tradeoff for optical quantum key distribution. Rohde, P. P. & Ralph, T. C. Modelling photo-detectors in quantum optics. We acknowledge funding from the Ministry of Internal Affairs and Communications, Japan, via the project of ICT priority technology (JPMI00316) ‘Research and Development for Construction of a Global Quantum Cryptography Network'. We acknowledge funding from the European Union's Horizon 2020 research and innovation programme under grant agreement number 857156 ‘OPENQKD', number 101072637 ‘Quantum-Safe Internet (QSI)'. The research leading to these results has received funding from the European Union's Horizon Europe research and innovation programme under grant agreement number 101100680 (GN5-1). Co-funded by the European Union. Views and opinions expressed are however those of the author(s) only and do not necessarily reflect those of the European Union. Neither the European Union nor the granting authority can be held responsible for them. Toshiba Europe Limited, Cambridge, UK Mirko Pittaluga, Yuen San Lo, Adam Brzosko, Robert I. Woodward, Davide Scalcon, Matthew S. Winnel, Thomas Roger, James F. Dynes, Kim A. Owen, Sergio Juárez & Andrew J. Shields Poznan Supercomputing and Networking Center, Poznan, Poland GÉANT Vereniging, Amsterdam, the Netherlands Domenico Vicinanza & Guy Roberts School of Computing and Information Science, Anglia Ruskin University, Cambridge, UK You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the study, developed the experimental system, performed the experiment, provided the simulations, and analysed the data. provided access to the network infrastructure and supported the system installation. supported the experimental work. provided definitions for the secret capacity bounds. guided the work. wrote the paper, with contributions from all the authors. Correspondence to Mirko Pittaluga. The authors declare no competing interests. Nature thanks Avishek Nag and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Pittaluga, M., Lo, Y.S., Brzosko, A. et al. Long-distance coherent quantum communications in deployed telecom networks. Issue Date: 24 April 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250423112135.htm'>Engineers print synthetic 'metamaterials' that are both strong and stretchy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 17:45:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Metamaterials are synthetic materials with microscopic structures that give the overall material exceptional properties. A huge focus has been in designing metamaterials that are stronger and stiffer than their conventional counterparts. The key to the new material's dual properties is a combination of stiff microscopic struts and a softer woven architecture. This microscopic "double network," which is printed using a plexiglass-like polymer, produced a material that could stretch over four times its size without fully breaking. The researchers say the new double-network design can be applied to other materials, for instance to fabricate stretchy ceramics, glass, and metals. Such tough yet bendy materials could be made into tear-resistant textiles, flexible semiconductors, electronic chip packaging, and durable yet compliant scaffolds on which to grow cells for tissue repair. "We are opening up this new territory for metamaterials," says Carlos Portela, the Robert N. Noyce Career Development Associate Professor at MIT. "You could print a double-network metal or ceramic, and you could get a lot of these benefits, in that it would take more energy to break them, and they would be significantly more stretchable." Portela and his colleagues will report their findings in the journal Nature Materials. Along with other research groups, Portela and his colleagues have typically designed metamaterials by printing or nanofabricating microscopic lattices using conventional polymers similar to plexiglass and ceramic. Several years ago, Portela was curious whether a metamaterial could be made from an inherently stiff material, but be patterned in a way that would turn it into a much softer, stretchier version. "We realized that the field of metamaterials has not really tried to make an impact in the soft matter realm," he says. "So far, we've all been looking for the stiffest and strongest materials possible." Instead, he looked for a way to synthesize softer, stretchier metamaterials. Rather than printing microscopic struts and trusses, similar to those of conventional lattice-based metamaterials, he and his team made an architecture of interwoven springs, or coils. They found that, while the material they used was itself stiff like plexiglass, the resulting woven metamaterial was soft and springy, like rubber. "They were stretchy, but too soft and compliant," Portela recalls. In looking for ways to bulk up their softer metamaterial, the team found inspiration in an entirely different material: hydrogel. They do so by combining polymer networks with very different properties, such as a network of molecules that is naturally stiff, which gets chemically cross-linked with another molecular network that is inherently soft. Portela and his colleagues wondered whether such a double-network design could be adapted to metamaterials. For their new study, the team fabricated a metamaterial by combining two microscopic architectures. The first is a rigid, grid-like scaffold of struts and trusses. The second is a pattern of coils that weave around each strut and truss. Both networks are made from the same acrylic plastic and are printed in one go, using a high-precision, laser-based printing technique called two-photon lithography. The researchers printed samples of the new double-network-inspired metamaterial, each measuring in size from several square microns to several square millimeters. They also recorded high-resolution videos to observe the locations and ways in which the material stretched and tore as it was pulled apart. They found their new double-network design was able stretch three times its own length, which also happened to be 10 times farther compared to a conventional lattice-patterned metamaterial printed with the same acrylic plastic. As this stress spreads unevenly through the material, an initial crack is unlikely to go straight through and quickly tear the material. What's more, the team found that if they introduced strategic holes, or "defects," in the metamaterial, they could further dissipate any stress that the material undergoes, making it even stretchier and more resistant to tearing apart. "You might think this makes the material worse," says study co-author Surjadi. The team has developed a computational framework that can help engineers estimate how a metamaterial will perform given the pattern of its stiff and stretchy networks. They envision such a blueprint will be useful in designing tear-proof textiles and fabrics. For that, the two networks could be made from different polymers, that respond to temperature in different ways, so that a fabric can open its pores or become more compliant when it's warm and can be more rigid when it's cold. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250423111753.htm'>Remembering the cold: Scientists discover how memories control metabolism</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 16:35:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New multidisciplinary research led by Prof. Tomás Ryan from Trinity College Dublin shows that the brain forms memories of cold experiences and uses them to control our metabolism. This newly published study is the first to show that cold memories form in the brain -- and map out how they subsequently drive thermoregulation. The discovery may have important applications in therapies designed to treat a range of disorders -- from obesity to cancer -- in which thermoregulation and metabolism (or a lack of control in this area) plays a role, as well as opening the door to more fundamental research, which could help us better understand how memories impact our behaviour and emotions. He showed that dogs could be trained to salivate in hopeful anticipation of food, when an associated bell was rung. Classical or Pavlovian conditioning has since become a core staple of neuroscience and psychology. Long-term memories are stored in the brain as ensembles of inter-connected cells, termed engrams. Increasingly, modern neuroscience is beginning to identify engrams that encode for bodily representations, such as experiences of infection; inflammation; food consumption; and pain. The researchers behind this work hypothesised that the brain may form engrams for temperature representations, and that these would serve to help an organism survive in changing temperatures. While memories are generally measured as changes in animal behaviour, the Ryan Lab collaborated with Prof. Lydia Lynch (then at Trinity College Dublin, now at Princeton University). They focused on metabolism as a first-order readout of cold experience, because mammals are known to increase their metabolism to create heat in the body when the environment is cold, via a process of adaptive thermogenesis. Lead author of the article published today in the journal, Nature,, Dr Andrea Muñoz Zamora, successfully trained mice to associate a cold experience of 4oC with novel visual cues that were only present in designated cold contexts. Remarkably, when these cold engram cells were artificially stimulated (using a technique called optogenetics), the mice increased their metabolism in order to generate heat. Dr Muñoz Zamora, said: "We discovered that when mice are exposed to a cold temperature they form memories that allow them to up-regulate their body's metabolism when they anticipate cold experiences in the future." Prof. Lynch added: "A large part of this learned control of body temperature seems to be due to increased activity of brown adipose tissue -- or brown fat -- which can be controlled by innervations originating in the brain. Our brain must learn from the bodily experiences of cold, but then feeds back to control how our fat cells respond to cold." Dr Aaron Douglas, who was joint lead author on the study, said: "Numerous clinical disorders, ranging from obesity to forms of cancer, may be treated by manipulating thermoregulation through brown adipose tissue. In the future, it will be important to test whether the manipulation of cold memories in humans could provide novel avenues for altering metabolism for therapeutic purposes." Understanding how representations of cold experiences affect broader brain functions such as emotion, decision-making, and social behaviour will provide insights into the embodied nature of the mind, for example. "The sophisticated aspects of our minds evolved from more basic, visceral, bodily representations," said Prof. Ryan. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/firing-science-advisors-will-leave-the-u-s-senseless/'>Here's One Neat Trick for Making the U.S. Government Foolish: Firing Expert Panels</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 15:33:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>From public health to space exploration, advisory panels have helped U.S. agencies make smarter decisions. Keep in touch with the Argonaut: Get email alerts for this weekly column by Dan Vergano In a February 19 executive order, Donald Trump directed his staff to compile a list of “Federal Advisory Committees that should be terminated on grounds that they are unnecessary.” The order directly terminated the HHS Advisory Committee on Long COVID (a syndrome afflicting 23 million people in the U.S. right now) and the Health Equity Advisory Committee, which sought to help underserved people access care like blood pressure medication or postpartum treatment, through Medicare and Medicaid. Since then NOAA has closed several of its advisory panels, NSF closed a dozen, NASA has consolidated its wildly disparate astrophysics, biological and physical sciences, Earth science, heliophysics and planetary sciences panels into one body, and the U.S. Geological Survey closed its new scientific integrity body, alongside five others at the Department of Interior that included a climate adaptation panel. “This means that you, the public, will be more at-risk of being harmed because the scientific integrity and misconduct issues that were prevalent before will continue to persist,” wrote integrity panel member Jacob Carter. He called the committee's cancellation, “an indicator that this administration has no intention to uphold scientific evidence in its decisions.” He's right; contrary to the executive order, these committees matter. Cutting away advisory panels hurts everyone and leaves the U.S. government uninformed when making critical decisions that affect millions of lives, alongside a public left in the dark about what advice agencies do receive. A 2021 Ecology Law Quarterly review found past end runs around advisory committees were linked to lead pollution, fracking contamination of drinking water, and worse air quality. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Federal advisory committees operate under a 1972 law, which governs the roughly 1,000 expert committees advising federal agencies on evidence-based practices on issues like boating safety or railroad retirement benefits at a yearly cost of $400 million. The panels are a bargain, providing by law “fairly balanced” expert advice that includes disclosures of financial conflicts of interest, and providing information openly to the public. History repeats when it comes to attacks on advisory panels. His then EPA chief resorted to stuffing a clean air panel with industry stooges instead, and an antiabortion advocates panel lacking any scientific credibility was whipped up to eliminate fetal tissue research at NIH. All Department of Homeland Security advisory committees members were fired in January, halting a probe into a massive Chinese breach of U.S. telecommunications infrastructure. Perhaps the clearest sign of the scientific advice we can instead expect from the Trump administration comes from RFK, Jr., naming an unqualified antivaccine activist, one who in 2011 was disciplined for practicing medicine without a license, to head a phony study to make autism “preventable” by September. The cruelty of his dishonest sham, founded on disdain for the autistic community and aimed at parents of autistic children, defies decency. It seems squarely aimed at making kids sick by discouraging vaccination. Such secrecy will be indubitably ubiquitous amid the news administration's “completely insane” meetings held on Signal, and tariff decisions surrounded by suspected insider trading. In our unhinged current moment, it's hard to recall that Trump campaign bankroller Elon Musk quit a Trump presidential advisory committee in 2017 over the withdrawal from the Paris climate agreement. One of the trials of the Trump era is that its malignancy comes packaged in a cloud of buffoonery: take your pick from Musk waving a chainsaw around on stage or an education secretary calling artificial intelligence “Ay-One” or any of Trump's inane digressions that should have triggered the 25th Amendment in his first term. But his administration's steps, large and small, from attacking universities to immigrants to expert advisors, aimed at destroying competence and honesty as guiding principles for the U.S., will in the end yield only tears, not laughter. Wrecking the federal advisory committee system is just one more tread on a path to American ruin. This is an opinion and analysis article, and the views expressed by the author or authors are not necessarily those of Scientific American. Dan Vergano is senior opinion editor at Scientific American, where he writes the weekly column Argonaut.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08927-x'>Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 15:25:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Patients with treatment-refractory pancreatic cancer often succumb to systemic metastases1,2,3; however, the transcriptomic heterogeneity that underlies therapeutic recalcitrance remains understudied, particularly in a spatial context. Here we construct high-resolution maps of lineage states, clonal architecture and the tumour microenvironment (TME) using spatially resolved transcriptomics from 55 samples of primary tumour and metastases (liver, lung and peritoneum) collected from rapid autopsies of 13 people. We observe discernible transcriptomic shifts in cancer-cell lineage states as tumours transition from primary sites to organ-specific metastases, with the most pronounced intra-patient distinctions between liver and lung. Phylogenetic trees constructed from inferred copy number variations in primary and metastatic loci in each patient highlight diverse patient-specific evolutionary trajectories and clonal dissemination. We show that multiple tumour lineage states co-exist in each tissue, including concurrent metastatic foci in the same organ. Agnostic to tissue site, lineage states correlate with distinct TME features, such as the spatial proximity of TGFB1-expressing myofibroblastic cancer-associated fibroblasts (myCAFs) to aggressive ‘basal-like' cancer cells, but not to cells in the ‘classical' or ‘intermediate' states. These findings were validated through orthogonal and cross-species analyses using mouse tissues and patient-derived organoids. Notably, basal-like cancer cells aligned with myCAFs correlate with plasma-cell exclusion from the tumour milieu, and neighbouring cell analyses suggest that CXCR4–CXCL12 signalling is the underlying basis for observed immune exclusion. Collectively, our findings underscore the profound transcriptomic heterogeneity and microenvironmental dynamics that characterize treatment-refractory pancreatic cancer. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout The raw and processed spatial transcriptome data and CosMx data generated in this study have been deposited at the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) with accession numbers GSE274557 and GSE277782. The combined datasets can be accessed at GEO with accession number GSE277783. All R and Python scripts supporting the findings of this study are available in the GitHub repository at https://github.com/Coolgenome/PDAC. Halbrook, C. J., Lyssiotis, C. A., Pasca di Magliano, M. & Maitra, A. Pancreatic cancer: advances and challenges. Addressing the challenges of pancreatic cancer: future directions for improving outcomes. Park, W., Chawla, A. & O'Reilly, E. M. Pancreatic cancer: a review. Groot, V. P. et al. Implications of the pattern of disease recurrence on survival following pancreatectomy for pancreatic ductal adenocarcinoma. Lung metastases in patients with stage IV pancreatic cancer: prevalence, risk factors, and survival impact. Sahin, I. H., Elias, H., Chou, J. F., Capanu, M. & O'Reilly, E. M. Pancreatic adenocarcinoma: insights into patterns of recurrence and disease behavior. Distant metastasis occurs late during the genetic evolution of pancreatic cancer. Yachida, S. et al. Clinical significance of the genetic landscape of pancreatic cancer and implications for identification of potential long-term survivors. Makohon-Moore, A. P. et al. Limited heterogeneity of known driver gene mutations among the metastases of individual patients with pancreatic cancer. Reiter, J. G. et al. Minimal functional driver gene heterogeneity among untreated metastases. Notta, F. et al. A renewed model of pancreatic cancer evolution based on genomic rearrangement patterns. McDonald, O. G. et al. Epigenomic reprogramming during pancreatic cancer progression links anabolic glucose metabolism to distant metastasis. Kim, M. S. et al. Heterogeneity of pancreatic cancer metastases in a single patient revealed by quantitative proteomics. Erickson, A. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. O'Kane, G. M. et al. GATA6 expression distinguishes classical and basal-like subtypes in advanced pancreatic cancer. Raghavan, S. et al. Microenvironment drives cell state, plasticity, and drug response in pancreatic cancer. Wormann, S. M. et al. APOBEC3A drives deaminase domain-independent chromosomal instability to promote pancreatic cancer metastasis. Collisson, E. A. et al. Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy. Bailey, P. et al. Genomic analyses identify molecular subtypes of pancreatic cancer. Gavish, A. et al. Hallmarks of transcriptional intratumour heterogeneity across a thousand tumours. Gabitova-Cornell, L. et al. Cholesterol pathway inhibition induces TGF-β signaling to promote basal differentiation in pancreatic cancer. Zhang, D. et al. Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology. Elyada, E. et al. Cross-species single-cell analysis of pancreatic ductal adenocarcinoma reveals antigen-presenting cancer-associated fibroblasts. Sahai, E. et al. A framework for advancing our understanding of cancer-associated fibroblasts. Dilly, J. et al. Mechanisms of resistance to oncogenic KRAS inhibition in pancreatic cancer. Zhang, Y. et al. Macropinocytosis in cancer-associated fibroblasts is dependent on CaMKK2/ARHGEF2 signaling and functions to support tumor and stromal cell fitness. Z. Macrophage diversity in cancer revisited in the era of single-cell omics. Comparison of immune infiltrates in melanoma and pancreatic cancer highlights VISTA as a potential target in pancreatic cancer. Carstens, J. L. et al. Spatial computation of intratumoral T cells correlates with survival of patients with pancreatic cancer. Falcomata, C. et al. Context-specific determinants of the immunosuppressive tumor microenvironment in pancreatic cancer. Biffi, G. et al. IL1-induced JAK/STAT signaling is antagonized by TGFβ to shape CAF heterogeneity in pancreatic ductal adenocarcinoma. Feig, C. et al. Targeting CXCL12 from FAP-expressing carcinoma-associated fibroblasts synergizes with anti-PD-L1 immunotherapy in pancreatic cancer. Jiang, H. et al. Targeting focal adhesion kinase renders pancreatic cancers responsive to checkpoint immunotherapy. He, S. et al. High-plex imaging of RNA and proteins at subcellular resolution in fixed tissue by spatial molecular imaging. Integration of genomic and transcriptional features in pancreatic cancer reveals increased cell cycle progression in metastases. Chan-Seng-Yue, M. et al. Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution. Moffitt, R. A. et al. Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma. Puleo, F. et al. Stratification of pancreatic ductal adenocarcinomas based on tumor and microenvironment features. Lee, J. J. et al. Elucidation of tumor-stromal heterogeneity and the ligand-receptor interactome by single-cell transcriptomics in real-world pancreatic cancer biopsies. Topham, J. T. et al. Subtype-discordant pancreatic ductal adenocarcinoma tumors show intermediate clinical and molecular characteristics. Williams, H. L. et al. Spatially resolved single-cell assessment of pancreatic cancer expression subtypes reveals co-expressor phenotypes and extensive intratumoral heterogeneity. Wang, Z. et al. Carcinomas assemble a filamentous CXCL12-keratin-19 coating that suppresses T cell-mediated immune attack. Yan, R., Moresco, P., Gegenhuber, B. & Fearon, D. T. T cell-mediated development of stromal fibroblasts with an immune-enhancing chemokine profile. Meylan, M. et al. Tertiary lymphoid structures generate and propagate anti-tumor antibody-producing plasma cells in renal cell cancer. The proteomic landscape of pancreatic ductal adenocarcinoma liver metastases identifies molecular subtypes and associations with clinical response. Stuart, T. et al. Comprehensive integration of single-cell data. McInnes, L., Healy, J., Saul, N. & Grossberger, L. UMAP: uniform manifold approximation and projection. Open Source Softw. Chu, Y. et al. Pan-cancer T cell atlas links a cellular stress response state to immunotherapy resistance. Wang, R. et al. Evolution of immune and stromal cell states and ecotypes during gastric adenocarcinoma progression. Wang, R. et al. Single-cell dissection of intratumoral heterogeneity and lineage diversity in metastatic gastric adenocarcinoma. Sans, M. et al. Spatial transcriptomics of intraductal papillary mucinous neoplasms of the pancreas identifies NKX6-2 as a driver of gastric differentiation and indolent biological potential. Yu, G., Smith, D. K., Zhu, H., Guan, Y. & Lam, T. T. Y. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Gao, R. et al. Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes. De Falco, A., Caruso, F., Su, X.-D., Iavarone, A. & Ceccarelli, M. A variational algorithm to detect the clonal copy number substructure of tumors from scRNA-seq data. Schliep, K. phangorn: phylogenetic analysis in R. Bioinformatics 27, 592–593 (2011). The Molecular Signatures Database (MSigDB) hallmark gene set collection. Hao, D. et al. The single-cell immunogenomic landscape of B and plasma cells in early-stage lung adenocarcinoma. Liao, Y., Wang, J., Jaehnig, E. J., Shi, Z. & Zhang, B. WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Ohlund, D. et al. Distinct populations of inflammatory fibroblasts and myofibroblasts in pancreatic cancer. Cable, D. M. et al. Robust decomposition of cell type mixtures in spatial transcriptomics. Wei, R. et al. Spatial charting of single-cell transcriptomes in tissues. Vahid, M. R. et al. High-resolution alignment of single-cell and spatial transcriptomes with CytoSPACE. Pei, G. et al. deCS: a tool for systematic cell type annotations of single-cell RNA sequencing data among human tissues. & DeNardo, D. G. A single-cell window into pancreas cancer fibroblast heterogeneity. Boyd, L. N. C., Andini, K. D., Peters, G. J., Kazemier, G. & Giovannetti, E. Heterogeneity and plasticity of cancer-associated fibroblasts in the pancreatic tumor microenvironment. Hanzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. A.M. was supported by the MD Anderson Pancreatic Cancer Moon Shot Program; the Sheikh Khalifa Bin Zayed Al-Nahyan Foundation; Break Through Cancer; and NIH grants (U54CA274371, U01CA200468 and U24CA274274). was supported by NIH–NCI grants (R01CA266280, U24CA274274, U01CA294518 and U01CA264583); research funding provided by the James P. Allison Institute; the Institute for Data Science in Oncology; the University of Texas MD Anderson Cancer Center; and Break Through Cancer. is an Associate Member of the James P. Allison Institute and an Andrew Sabin Family Foundation Fellow at the MD Anderson Cancer Center. acknowledges support from NIH–NCI grant U01CA294518 and the Program for T Cell-based Therapy at the MD Anderson Cancer Center. A.S. was supported by the the ACCENT (B-487.0012) and BONFOR program (O-112.0070). was supported by a NIH grant (U01CA274295). was funded by the Hale Center for Pancreatic Cancer Research; Break Through Cancer; the Lustgarten Foundation; the Pancreatic Cancer Action Network; NIH–NCI grants (P50CA127003, U01CA274276 and R01CA276268); and the Dana-Farber Cancer Institute Hale Center for Pancreatic Cancer Research. were supported by the Pancreatic Cancer Detection Consortium (U01CA210240); a NCI Cancer Center Support grant (P30CA36727); and a NCI Research Specialist award (R50CA211462). The cyclic IF staining was performed in the Flow Cytometry and Cellular Imaging Core Facility, which is supported in part by the NIH through MD Anderson's Cancer Center Support grant (P30CA016672), the NCI's Research Specialist 1 (R50CA243707-01A1) and a Shared Instrumentation award from the Cancer Prevention Research Institution of Texas (CPRIT). We thank A. S. Multani for the FISH experiments and S. P. So, E. E. Rodriguez, A. T. Reckard, Y. A. Zuberi, A. V. Basi and J. A. Gomez for technical assistance. These authors contributed equally: Guangsheng Pei, Jimin Min Department of Genomic Medicine, University of Texas MD Anderson Cancer Center, Houston, TX, USA Guangsheng Pei, Yunhe Liu, Kyung Serk Cho, Yanshuo Chu, Enyu Dai, Guangchun Han & Linghua Wang Sheikh Ahmed Center for Pancreatic Cancer Research, University of Texas MD Anderson Cancer Center, Houston, TX, USA Jimin Min, Kimal I. Rajapakshe, Vittorio Branchi, Benson Chellakkan Selvanesan, Fredrik Thege, Dorsay Sadeghian, Paola A. Guerrero & Anirban Maitra Department of Translational Molecular Pathology, University of Texas MD Anderson Cancer Center, Houston, TX, USA Jimin Min, Kimal I. Rajapakshe, Vittorio Branchi, Benson Chellakkan Selvanesan, Fredrik Thege, Dorsay Sadeghian, Paola A. Guerrero & Anirban Maitra Department of Biostatistics, Epidemiology and Informatics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Daiwei Zhang & Mingyao Li Department of Biostatistics, University of North Carolina, Chapel Hill,  NC, USA Department of Genetics, University of North Carolina, Chapel Hill,  NC, USA Department of Melanoma Medical Oncology, University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA, USA Kazuki Takahashi & Andrew J. Aguirre Harvard Medical School, Boston, MA, USA Kazuki Takahashi & Andrew J. Aguirre Broad Institute of Harvard and MIT, Cambridge, MA, USA Kazuki Takahashi & Andrew J. Aguirre Department of Surgery, Division of Surgical Oncology, Moores Cancer Center, University of California, San Diego, La Jolla, CA, USA Bharti Garg, Herve Tiriac & Andrew M. Lowy Department of Radiation Oncology, University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Surgery, University Hospital Bonn, University of Bonn, Bonn, Germany Department of Internal Medicine, University of Nebraska Medical Center, Omaha, NE, USA Eppley Institute for Research in Cancer and Allied Diseases, Fred & Pamela Buffett Cancer Center, University of Nebraska Medical Center, Omaha, NE, USA Thomas C. Caffrey, Paul M. Grandgenett & Michael A. Hollingsworth Department of Leukemia and Division of Cancer Medicine, University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Medicine, Brigham and Women's Hospital, Boston, MA, USA University of Texas MD Anderson Cancer Center UTHealth Houston Graduate School of Biomedical Sciences, Houston, TX, USA James P. Allison Institute, University of Texas MD Anderson Cancer Center, Houston, TX, USA Institute for Data Science in Oncology, University of Texas MD Anderson Cancer Center, Houston, TX, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar contributed to sample and patient information collection. processed tissues and prepared libraries for SRT. contributed to pathology review. supervised the processing, analysis and interpretation of bioinformatics data. contributed to tool development. assisted with data analysis. contributed to the processing of sequencing data and integrative analyses. provided mouse tissues, human organoid lines, human CAF lines and relevant resources. provided support and resources for cyclic IF. performed in vitro experiments and data analysis. analysed data and generated figures and tables for the manuscript. and A.M. contributed to data interpretation. and A.M. wrote and revised the manuscript, and all co-authors reviewed the manuscript. Correspondence to Linghua Wang or Anirban Maitra. A.M. is listed as an inventor on a patent that has been licensed by Johns Hopkins University to Thrive Earlier Detection and serves as a consultant for Tezcat Biosciences. has consulted for Anji Pharmaceuticals, Affini-T Therapeutics, Arrakis Therapeutics, AstraZeneca, Boehringer Ingelheim, Kestrel Therapeutics, Merck, Mirati Therapeutics, Nimbus Therapeutics, Oncorus, Plexium, Quanta Therapeutics, Revolution Medicines, Reactive Biosciences, Riva Therapeutics, Servier Pharmaceuticals, Syros Pharmaceuticals, T-knife Therapeutics, Third Rock Ventures and Ventus Therapeutics; holds equity in Riva Therapeutics and Kestrel Therapeutics; and has research funding from Boehringer Ingelheim, Bristol Myers Squibb, Deerfield, Eli Lilly, Mirati Therapeutics, Novartis, Novo Ventures, Revolution Medicines and Syros Pharmaceuticals. The remaining authors declare no competing interests. Nature thanks Jen Jen Yeh, Nancy Zhang and the other, anonymous, reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,b, Each dot represents a single spot coloured by (a) patient and (b) treatment history. c,d, Venn diagrams showing the overlap of significantly upregulated genes in the treated group (c) or untreated group (d) across four different tissue sites. The workflow comprises four key steps: (1) annotation of tumour spots, (2) inference of CNVs at spot level, (3) construction of phylogenetic tree, and (4) spatial mapping of inferred subclones. a,e, Spatial visualization of subclones from Fig. 2, along with fluorescence images for MYC (red) and control (Ctrl, green) probes targeting the centromere of chromosome 8 in Pt-10 (a) and Pt-1 (e). Dotted boxes and numbers indicate the enlarged areas. b,f, MYC CNV scores from the inferCNV analysis are shown in b (Pt-10) and f (Pt-1). Data are presented as mean ± standard deviation (SD). For Pt-10 (b), clone A (n = 2), clone B (n = 3), clone C (n = 5) and clone D (n = 1). For Pt-1 (f), clone A (n = 2), clone B (n = 3) and clone C (n = 5). c,g, Per-cell copy number (CN) ratio for MYC in Pt-10 (c) and Pt-1 (g). d,h, Percentages of cells with MYC amplification (CN ratio over 1) in Pt-10 (d) and Pt-1 (h). a, UMAP plot displaying 67,990 “neoplastic” spots cross lineage states (left), patients (middle) or tissue origin sites (right). b, Bar plots displaying the relative fraction of tumour spots in classical, intermediate, and basal lineage states in each ST sample. The samples were ordered by tissue sites and the fraction of basal lineage. The top pie charts represent the global lineage composition in all Pri, LiM, PerM and LuM sites. c, Co-immunostaining of PanCK (green), S100A2 (red), and GATA6 (blue) in matched liver and lung metastases of three KPCY mice. Dotted boxes indicate the enlarged areas. d, Mean intensity of S100A2 and GATA6 across individual cells in tumour ROIs of liver and lung metastases. Each dot with connected lines represents data from the same mouse (n = 3). P values are indicated above the plot. a, Pearson correlation analysis among classical, basal lineage, mesenchymal, squamous, and 8 core basal gene signature scores across all tumour spots. b, Bar plots comparing the relative fraction of tumour spots with mesenchymal and squamous lineages in each ST sample. c, Overview of 41 MPs among classical, intermediate, mesenchymal, and squamous lineage tumour enriched spots. a, Schematic diagram illustrating the redefinition of tumour regions based on their distance from non-tumour areas, categorizing them into tumour edge, intermediate, and core regions. b,c, Total spot number (left) and relative composition (right) of tumour spots from the three main different lineages (b), with further subtyping of the basal lineage into mesenchymal and squamous lineages (c), among tumour-edge, tumour-intermediate, and tumour-core regions. d, Overview of 41 MPs based on tumour regions. a, Co-immunostaining of PanCK (green), S100A2 (red), and DAPI (blue) in matched liver and lung metastases of three different KPCY mice. White lines denote the tumour bed, and yellow lines denote the juxtalesional areas. b, Mean intensity of α-SMA across individual cells in juxtalesional ROIs between liver (LiM) and lung (LuM) metastases. For mouse 1, n = 86,763 cells (LiM) and n = 4,125 cells (LuM). For mouse 2, n = 21,310 cells (LiM) and n = 5,101 cells (LuM). For mouse 3, n = 194,729 cells (LiM) and n = 11,861 cells (LuM). Each dot with connected lines represents data from the same mouse. ****P < 0.0001. c, A representative image of E-cadherin (white), α-SMA (green), S100A2 (red), and GATA6 (blue) in a liver metastasis containing both classical-like and basal-like tumour cells. White arrows with numbers denote the enlarged areas. d, Phase contrast, H&E staining, and co-immunostaining for S100A2 (red), α-SMA (green), and DAPI (blue) in classical-like and basal-like PDOs cultured with human CAFs. Full PDO IDs are as follows: 185; PANFR0185_T2, 332; PANFR0332_T1, 172; PANFR0172_T4, and 440; PANFR0440_T1. e, Quantification of the percentage of organoids displaying direct attachments of α-SMA+ CAFs in each PDO line. Each dot represents an individual CAF line. Data are presented as mean ± SD (n = 3, independent experiments). Student's unpaired t-test. From top to bottom, the images show tumour lineage distribution, iStar-derived T cell signature, iStar-derived B cell signature, and expression levels of CD3D and MS4A1. This file contains Supplementary Figures 1–12. This file contains Supplementary Tables 1–10. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. et al. Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08898-z'>BMAL1–HIF2A heterodimer modulates circadian variations of myocardial injury</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 15:25:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Acute myocardial infarction is a leading cause of morbidity and mortality worldwide1. Clinical studies have shown that the severity of cardiac injury after myocardial infarction exhibits a circadian pattern, with larger infarcts and poorer outcomes in patients experiencing morning-onset events2,3,4,5,6,7. However, the molecular mechanisms underlying these diurnal variations remain unclear. Here we show that the core circadian transcription factor BMAL17,8,9,10,11 regulates circadian-dependent myocardial injury by forming a transcriptionally active heterodimer with a non-canonical partner—hypoxia-inducible factor 2 alpha (HIF2A)12,13,14,15,16—in a diurnal manner. To substantiate this finding, we determined the cryo-EM structure of the BMAL1–HIF2A–DNA complex, revealing structural rearrangements within BMAL1 that enable cross-talk between circadian rhythms and hypoxia signalling. We further identified amphiregulin (AREG)16,17 as a rhythmic target of the BMAL1–HIF2A complex, critical for regulating daytime variations of myocardial injury. Pharmacologically targeting the BMAL1–HIF2A–AREG pathway provides cardioprotection, with maximum efficacy when aligned with the pathway's circadian phase. These findings identify a mechanism governing circadian variations of myocardial injury and highlight the therapeutic potential of clock-based pharmacological interventions for treating ischaemic heart disease. Cardiac injury after acute myocardial infarction (MI) exhibits pronounced circadian rhythmicity, with severity and clinical outcomes varying on the basis of the time of onset2,3,4,5,6,7. Circadian rhythms, driven by the Earth's day–night cycles, synchronize internal biological functions with environmental changes, enabling organisms to adapt to the daily fluctuations18,19. At the core of these rhythms are transcription factors, notably BMAL17,8,9,10,11, which forms a heterodimer with CLOCK to regulate clock-controlled genes8,10,11,20. Although clinical and preclinical studies have highlighted the importance of circadian rhythms in cardiovascular physiology and diseases6,7,21,22,23,24, the mechanisms underlying circadian variations in myocardial injury remain poorly understood. This knowledge gap poses challenges to therapeutic efficacy and may contribute to suboptimal treatment outcomes21,25. Here we identify a molecular mechanism in which cross-talk between circadian rhythms and hypoxia signalling13,14,15,26,27,28,29,30 underpins circadian-dependent cardioprotection. Given the universal and essential roles of these pathways across nearly all cells and organs, our findings have broad implications for advancing the understanding and treatment of ischaemic diseases influenced by circadian rhythms. Consistent with previous studies27,31, diurnal variations in cardiac injury and long-term outcomes were evident in a mouse myocardial ischaemia and reperfusion injury (IRI) model, with the least injury at zeitgeber time 8 (ZT8; 15:00) and the most severe at ZT20 (03:00) (Supplementary Fig. To identify the molecular factors that drive these circadian variations, RNA-sequencing (RNA-seq) was performed on area-at-risk (AAR) samples collected after 2 h of reperfusion at ZT8 and ZT20. This analysis revealed distinct transcriptional profiles, with 18 genes upregulated and 42 downregulated at ZT8 compared with at ZT20 (Extended Data Fig. Notably, BMAL1 target genes, including Per2, Per3, Nr1d2 and Dbp, were significantly upregulated at ZT8 (Fig. 1a,b and Supplementary Table 1), indicating elevated BMAL1 transcriptional activity. Conversely, Bmal1 transcript levels were downregulated at ZT8, exhibiting an antiphasic expression pattern relative to its targets, a hallmark of circadian transcription–translation feedback loops10,11,20. Analysis using quantitative PCR with reverse transcription (RT–qPCR) confirmed these oscillations (Fig. 1c) pathway analyses identified ‘circadian rhythm' as a highly enriched pathway, while a dysregulation network further emphasized BMAL1's central role in orchestrating diurnal gene expression in ischaemic mouse hearts (Extended Data Fig. a–d, RNA-seq analysis of the AAR from C57BL/6J mice after 2 h reperfusion at ZT8 or ZT20, n = 3 mice per timepoint. b, The fold change (FC) in expression. d, The top ten enriched biological process GO terms. e–i, RNA-seq analysis of post-clamping LV biopsies from patients who had aortic valve replacement (AVR) surgery in the morning (AM; n = 56) or afternoon (PM; n = 17). CRI, chronic renal insufficiency; sCr, serum creatinine. g, The fold change in expression after clamping (morning versus afternoon). i, Normalized read counts for DEGs. j, The experimental setup for cardiac injury and function assessment in Bmal1loxP/loxP myosin-Cre+ and myosin-Cre+ mice subjected to IRI at ZT8 or ZT20. k, Heart slices were stained with Evan's blue and 2,3,5-triphenyltetrazolium chloride (TTC) after 2 h reperfusion; the infarct area (green) and AAR (blue) are shown. n, Serum troponin I levels. o–q, Cardiac function on day 14 after MI by STE. o, The EF, FS and GLS. p, 3D and six-segment longitudinal strain imaging with annotations for reduced contractility (stars), dyskinesis (triangles) and dyssynchrony (circles). Dark blue, anterior base; yellow, anterior mid; magenta, anterior apex; cyan, posterior apex; light pink, posterior mid; green, posterior base. All samples are biologically independent. For c,l–o and q, data are mean ± s.e.m. The diagram in j was created using BioRender. To investigate transcriptional pathways in human hearts experiencing myocardial injury at different times of the day, we examined left ventricle (LV) biopsy samples from 73 patients undergoing elective aortic valve replacement surgery (NCT00281164). Samples from the morning (n = 56, median, 10:32) or afternoon (n = 17, median, 17:15) cohort were collected before and after around 80 min of ischaemia induced by aortic cross-clamping (Fig. Patient demographics and clinical characteristics were comparable between cohorts (Supplementary Table 3). RNA-seq analysis of pre-clamping samples revealed significant circadian modulation of gene expression, with BMAL1 identified as the most downregulated gene in the morning cohort (Extended Data Fig. 1e and Supplementary Table 4), suggesting its role in the time-of-day-dependent regulation of cardiac physiology, such as metabolism and contractility6,21,22,23,24,32,33. Post-clamping RNA-seq analysis revealed distinct transcriptional signatures based on surgery timing (Extended Data Fig. 1f), with timing emerging as the dominant determinant over covariates such as smoking or comorbidities (Fig. BMAL1 remained substantially downregulated in the morning cohort (Fig. 1g and Supplementary Table 6), further implicating it in modulating the heart's ischaemic response. GO and KEGG analyses revealed significant enrichment in the ‘circadian rhythm' pathway (Fig. 1h and Supplementary Table 7), with antiphasic expression patterns of BMAL1 and its targets paralleling murine observations (Fig. These findings suggest that the conserved role of BMAL1 as a transcription factor is involved in regulating circadian variations in myocardial injury across species. To further explore BMAL1's functional role, we generated an inducible cardiomyocyte-specific Bmal1-knockout mouse line (Bmal1loxP/loxPMyh6-cre; hereafter, Bmal1loxP/loxP myosin-Cre+) by crossing floxed Bmal1 mice with myosin-Cre+ mice and inducing Bmal1 ablation by tamoxifen injection at 8 weeks of age. We then subjected Bmal1loxP/loxP myosin-Cre+ mice and control myosin-Cre+ mice to myocardial IRI at ZT8 or ZT20 (Fig. After 2 h of reperfusion, Bmal1-deficient mice showed abolished circadian variability in myocardial injury and diminished endogenous cardioprotection at ZT8 compared with the controls (Fig. This phenotype persisted during extended reperfusion (day 14 after MI), as speckle-tracking echocardiography (STE)34,35 demonstrated that ZT8 Bmal1-deficient mice exhibited significantly reduced systolic function (ejection fraction (EF), fractional shortening (FS), global longitudinal strain (GLS)), greater LV dilation (end-diastolic volume and end-systolic volume) and LV mass (end-diastolic LV mass and end-systolic LV mass) (Fig. Impaired contractility was evident in both infarcted (anterior mid and posterior apex) and non-infarcted (posterior mid and posterior base) heart segments, indicating more extensive myocardial injury (Supplementary Fig. Pronounced intraventricular disparities further suggested heart failure progression35 (Fig. By contrast, IRI at ZT20 showed no significant differences in cardiac function between knockout and control mice. These findings highlight BMAL1's key role in circadian regulation of acute myocardial injury and long-term outcomes. As a key member of the basic helix-loop-helix PER–ARNT–SIM (bHLH-PAS) family, BMAL1 exerts regulatory functions through interactions with specific partners8,10,11,20. Using the Human Reference Interactome (HuRI)36, we identified hypoxia-inducible factor 2 alpha (HIF2A, also known as EPAS1), another bHLH-PAS family member, as the most abundantly expressed BMAL1 interactor in human LV (largest node in Fig. GO analysis highlighted shared roles of BMAL1 and HIF2A in transcriptional regulation and cellular responses to hypoxia and oxidative stress (Extended Data Fig. Given the severe hypoxia during MI13,15,26,28 and HIF2A's established role in oxygen homeostasis16,37, we examined the BMAL1–HIF2A interaction under hypoxic conditions. Co-immunoprecipitation (co-IP) assays in HEK293 cells revealed a robust BMAL1–HIF2A nuclear interaction under ambient hypoxia (1% O2, 4 h), distinct from the oxygen-insensitive BMAL1–CLOCK interaction (Fig. 2b,c and Extended Data Fig. Consistent with previous reports38, binding of BMAL1 to HIF1A is considerably weaker (Fig. We further validated in primary human cardiomyocytes (HCMs) using endogenous co-IP and proximity ligation assay (PLA)39, confirming a robust BMAL1–HIF2A interaction at the single-molecule level under hypoxia, while BMAL1–HIF1A binding was also detected, albeit at significantly lower levels (Fig. The N-terminal regions of BMAL1 and HIF2A, containing conserved bHLH and PAS-A/B domains, mediate partner dimerization8,40 (Fig. 2g), while their C-terminal transactivation domains enable transcriptional activation40. Pull-down assays using recombinant BMAL1 and HIF2A demonstrated direct binding between their N-terminal regions (Fig. Size-exclusion chromatography confirmed the formation of a stable BMAL1–HIF2A heterodimer (Extended Data Fig. Despite a 67% sequence identity between the N-terminal regions of HIF1A and HIF2A, BMAL1 exhibits notably weaker binding to HIF1A (Fig. These results demonstrate a specific and direct interaction between BMAL1 and HIF2A during hypoxia. a, Predicted interactions of BMAL1 with bHLH-PAS transcription factors in the human LV using the HuRI. d, Co-IP analysis of HIF2A in HCMs. g, Schematic of BMAL1 and HIF2A protein domains. i,j, Western blot analysis (i) and quantification (j) of nuclear BMAL1 and HIF2A levels in the AAR of C57BL/6J mice after 2 h reperfusion at ZT8 or ZT20. Owing to similar molecular masses of proteins, the samples were run on separate gels, with TBP as the sample processing control. k, Immunofluorescence analysis of BMAL1 and HIF2A colocalization in the border zone on day 1 after MI, shown in merged images (yellow, white arrows). Evan's blue and TTC-stained heart slices (l; scale bar, 1 mm), AAR as the percentage of LV (m), infarct size as the percentage of the AAR (n) and serum troponin I levels (o) after 2 h reperfusion are shown. p–s, Cardiac function by day 14 after MI. p, The EF and FS. r, LV 3D and six-segment longitudinal strain images. Statistical analysis was performed using one-way ANOVA (c) and two-way ANOVA (f,j,m–q and s). Although HIF2A is well recognized for its cardioprotective role against myocardial IRI16,28,41, its involvement in diurnal cardiac injury remains unclear. After 2 h of reperfusion, HIF2A protein levels were notably stabilized in the nuclear fraction of the AAR at ZT8 but not ZT20, paralleling BMAL1 expression (Fig. 2i,j), suggesting a time-of-day-dependent myocardial hypoxic response. On day 1 after MI, immunofluorescence revealed significantly higher HIF2A levels in the cytoplasm and nuclei of cardiomyocytes in the border zone (ischaemic region) at ZT8 compared with at ZT20, while expression remained low in the remote myocardium (Extended Data Fig. BMAL1 displayed a similar diurnal nuclear localization pattern (Extended Data Fig. 2k), indicating circadian regulation of their interaction. To elucidate the functional role of HIF2A in circadian myocardial injury, we used an inducible cardiomyocyte-specific Hif2a knockout model (Hif2aloxP/loxP myosin-Cre+)16,41. Similar to the Bmal1-deficient mice, these mice exhibited abolished circadian-dependent cardiac injury and diminished endogenous cardioprotection at ZT8 (Fig. On day 14 after MI, STE revealed worsened systolic function, increased LV dilation and mass, and greater wall motion abnormalities and intraventricular disparities in ZT8 Hif2aloxP/loxP myosin-Cre+ mice compared with the controls (Fig. These changes were absent at ZT20. By contrast, cardiomyocyte Hif1a deletion did not affect circadian variations in myocardial injury (Supplementary Fig. These results highlight a selective role for cardiomyocyte-specific HIF2A in regulating diurnal susceptibility to IRI. Next, to investigate how the BMAL1–HIF2A complex influences diurnal variations of myocardial injury, we reanalysed our previously published microarray data from Hif2aloxP/loxP myosin-Cre+ and myosin-Cre+ mice16. Among potential HIF2A target genes that are uniquely upregulated in myosin-Cre+ mice after IRI (Fig. 3a and Supplementary Table 9), Areg—a known HIF2A target and member of the epidermal growth factor family16,17—exhibited the largest diurnal fold change in the AAR at ZT8 compared with at ZT20 (Fig. AREG protein levels in cytosolic extracts mirrored this diurnal pattern, peaking at ZT8 (Fig. Immunofluorescence staining confirmed elevated AREG expression in the cytoplasm of cardiomyocytes within the border zone at ZT8, while infarcted and remote myocardium showed low levels (Fig. 3e,f and Extended Data Fig. Minimal AREG expression was detected in fibroblasts or smooth muscle cells (Extended Data Fig. This spatiotemporal pattern aligns with BMAL1 and HIF2A nuclear co-localization in the ZT8 border zone (Fig. 2k), suggesting that the BMAL1–HIF2A complex drives diurnal Areg induction in ischaemic hearts. a, Heat map of the top 20 potential HIF2A targets. b–d, HIF2A targets (b) and AREG protein (western blot (c) and quantification (d)) in the AAR of C57BL/6J mice after 2 h reperfusion. Owing to similar molecular masses of proteins, the samples were run on separate gels, with α-tubulin as the sample processing control. e,f, AREG immunostaining in the border zone on day 1 after MI (e; scale bars, 25 μm) and quantification (f). g–i, Synchronized HCMs were exposed to hypoxia (1% O2, 4 h) across circadian times (CT0–CT40); AREG mRNA expression (g) and BMAL1–HIF2A/AREG protein (h) and quantification (i) are shown. n = 3. j, Analysis of AREG mRNA in HEK293 cells transfected with various siRNAs and exposed to hypoxia. n = 3. k–m, The fold change in Areg mRNA transcripts (k), and western blot (l) and quantification (m) of cardiomyocyte AREG after 2 h reperfusion in the AAR of myosin-Cre+, Bmal1loxP/loxP myosin-Cre+ and Hif2aloxP/loxP myosin-Cre+ mice. Statistical analysis was performed using unpaired two-tailed t-tests, with Welch's t-tests comparing between myosin-Cre+ and Hif2aloxP/loxP myosin-Cre+ mice (protein). n,o, Surface plasmon resonance analysis of BMAL1–HIF2A binding to HRE (n) or E-box (o). n = 3. p, Conserved BMAL1–HIF2A-binding site (CAGGTG) on the AREG promoter. q, ChIP–qPCR analysis of HIF2A binding in HEK293 cells at this shared site at CT20 or CT32. n = 5. r, Co-IP analysis of HIF2A–BMAL1 interactions in HCMs at CT20/CT32. t, Luciferase assays of hAREG promoter activation in HEK293 cells. All of the samples are biologically independent. Statistical analysis was performed using one-way ANOVA (g,i,j and t), two-way ANOVA (b,d,q), unpaired two-tailed t-tests (f) and two‐sided Mann–Whitney U-test (s). To assess the role of the cell-intrinsic molecular clock in AREG regulation, synchronized HCMs were exposed to hypoxia (1% O2, 4 h) at different circadian times (CT0 to CT40). Under normoxia, AREG transcript levels were non-rhythmic, but hypoxia significantly induced AREG at CT32 (analogous to ZT8), with a much weaker induction at CT20 (Fig. 3h,i), suggesting circadian-dependent, hypoxia-driven regulation of AREG. Silencing HIF2A or BMAL1 with small interfering RNA (siRNA) significantly reduced hypoxia-induced AREG expression, whereas CLOCK, HIF1A or HIF1B knockdown had no effect (Fig. Consistently, myocardial IRI-induced Areg expression was markedly reduced in cardiomyocytes from Bmal1- or Hif2a-deficient mice (Fig. Together, these findings underscore the BMAL1–HIF2A complex as a key regulator of rhythmic AREG induction during both hypoxia and myocardial IRI. To examine how the BMAL1–HIF2A heterodimer regulates AREG, we investigated its DNA-binding properties and potential transcriptional synergy under hypoxia. HIF2A–HIF1B and BMAL1–CLOCK complexes bind to hypoxia response elements (HREs, [A/G]CGTG)40 and E-box motifs (CACGTG)8, respectively, which share notable sequence similarity. Electrophoretic mobility shift assay (Supplementary Fig. 3l) and surface plasmon resonance analysis (Fig. 3n,o) demonstrated robust and comparable binding of the BMAL1–HIF2A heterodimer to both motifs, confirming its ability to recognize DNA elements of target genes. Analysis of the human AREG promoter using JASPAR (https://jaspar.elixir.no/; v.2022) revealed a conserved binding site (CAGGTG) for BMAL1 and HIF2A across species (Fig. Chromatin immunoprecipitation followed by qPCR (ChIP–qPCR) in HCMs confirmed significant binding enrichment of endogenous HIF2A to this site under hypoxia at CT32 but not at CT20 (Fig. 3q), indicating circadian gating of HIF2A transcriptional activity. This finding aligns with circadian-dependent BMAL1–HIF2A interaction, which peaks at CT32 (Fig. 3r) and corresponds to enhanced AREG induction (Fig. Additional ChIP–qPCR confirmed direct BMAL1 binding to the same conserved site during hypoxia (Fig. To evaluate functional cross-talk between BMAL1 and HIF2A at the chromatin level, the conserved sequence was cloned into a luciferase reporter and transfected into HEK293 cells. Both proteins enhanced luciferase activity, with co-transfection producing a synergistic effect and greater activation (Fig. These findings demonstrate that BMAL1 and HIF2A function as an integrated regulatory unit, synergistically driving rhythmic AREG transcription during hypoxia. To determine whether AREG directly contributes to circadian variations in myocardial injury, we analysed its role using genetic deletion models. Similar to Bmal1- and Hif2a-deficient mice, Areg−/− mice exhibited a loss of circadian variations in myocardial injury and diminished endogenous cardioprotection at ZT8 after 2 h of reperfusion (Fig. Diurnal variations in long-term outcomes, including cardiac function, LV size and mass, segmental wall motion abnormalities and intraventricular synchronicity, observed in wild-type (WT) controls, were completely absent in Areg−/− mice by day 14 after MI (Fig. Terminal deoxynucleotidyl-transferase-mediated dUTP-biotin nick-end labelling (TUNEL) staining, which detects double-stranded DNA breaks as a marker of apoptosis, revealed significantly increased cardiomyocyte apoptosis in the border zone of Areg−/− mice at ZT8 compared with in WT mice, with minimal differences observed at ZT20 (Fig. These findings indicate a functional role of AREG in regulating daytime-dependent cardiac injury. a, Schematic of myocardial injury and cardiac function assessment in Areg−/− mice and WT mice subjected to IRI at ZT8 or ZT20. b, Heart slices were stained with Evan's blue and TTC after 2 h reperfusion. e, Serum troponin I levels. f,g, Cardiac function on day 14 after MI was determined using STE. f, The EF and GLS. g, B-mode images with 2D longitudinal strain. h,i, TUNEL staining in the border zone on day 1 after MI (h; scale bar, 25 μm) and quantification (i). White arrows point out TUNEL-positive (green) cardiomyocyte (red) nuclei (blue). j, Schematic of the cardiac injury and function assessment in AREG-treated or vehicle (veh. )-treated mice subjected to IRI at ZT8 or ZT20. Treatment was initiated at reperfusion and continued daily for 3 days. m–p, Evan's blue- and TTC-stained heart slices (m, scale bar, 1 mm), the AAR as a percentage of the LV (n), the infarct size as a percentage of the AAR (o) and serum troponin I levels (p) after 2 h reperfusion. q,r, Cardiac function on day 14 after MI. q, The EF, FS and GLS. r, LV 3D images with six-segment longitudinal strain. s,t, TUNEL staining in the border zone. The percentage of TUNEL-positive cells (s) and representative images (t) are shown. White arrows indicate TUNEL-positive (green) cardiomyocyte (red) nuclei (blue). For c–f,i,l,n–q and s, statistical analysis was performed using two‐way ANOVA. The diagrams in a and j were created using BioRender. Building on AREG's critical role in circadian injury modulation, we assessed its therapeutic potential through exogenous administration and gene delivery. To mimic a clinical scenario of reperfusion treatment, recombinant AREG (10 µg) or vehicle was administered to C57BL/6J mice at the onset of reperfusion. 4j–l) and significantly reduced infarct size and serum troponin I levels after 2 h of reperfusion. Notably, protection was greater at ZT20, potentially compensating for the lower endogenous AREG levels at that time (Fig. Furthermore, AREG (10 µg) administered daily for 3 days after injury at ZT20 improved LV systolic function, reduced cardiac remodelling, restored segmental contractility and re-established normokinesis by day 14 after MI compared with the vehicle controls (Fig. TUNEL staining on day 1 after MI revealed notably reduced cardiomyocyte apoptosis in the border zone in ZT20-treated mice (Fig. To further assess its therapeutic potential, we used adeno-associated virus serotype 9 (AAV9) to deliver the Areg gene under the muscle creatine kinase (MCK) promoter for cardiomyocyte-specific targeting. Tail-vein injection of Areg-AAV (2 × 1011 genome copies (GC) per mouse) significantly increased AREG levels in the heart by day 28, with minimal off-target expression (Supplementary Fig. This intervention reduced the infarct size and improved cardiac function, with greater effects at ZT20 (Supplementary Fig. These findings establish AREG as a promising therapeutic target for time-optimized interventions, offering strategies to harness circadian rhythms for improved cardiac protection. To examine the translational potential of modulating circadian rhythms for cardioprotection, we used nobiletin (NOB), a flavonoid that enhances circadian rhythms by directly activating RORs42, key transcriptional activators of Bmal110,20. NOB treatment in C57BL/6J mice (200 mg per kg, intraperitoneal (i.p.)) significantly upregulated RORα, leading to a 60–80-fold increase in Bmal1 mRNA and a 2–3-fold increase in protein levels at both ZT8 and ZT20, ultimately inducing AREG upregulation in the heart (Extended Data Fig. Immunofluorescence analysis confirmed elevated nuclear BMAL1 and cytoplasmic AREG in cardiomyocytes in the border zone on day 1 after MI (Extended Data Fig. NOB treatment significantly attenuated acute myocardial injury, improved long-term cardiac function and reduced cardiomyocyte apoptosis, with more pronounced protection observed at ZT20 (Extended Data Fig. 5g–n), probably due to its ability to compensate for the natural trough in nuclear BMAL1 expression and transcriptional activity at this time10,11,20. To determine whether NOB's cardioprotective effects depend on the BMAL1–HIF2A–AREG pathway, we evaluated its efficacy in genetic knockout models. Cardioprotection was completely abrogated in Hif2a-deficient mice treated with NOB, as evidenced by larger infarct sizes and significantly impaired cardiac function (Supplementary Fig. Similarly, NOB treatment failed in Bmal1loxP/loxP myosin-Cre+ mice (Supplementary Fig. 6p–u), confirming the essential role of this pathway. BMAL1 overexpression significantly reduced infarct size and improved cardiac function with greater protection observed at ZT20 (Supplementary Fig. These effects mirrored those of NOB treatment, highlighting the therapeutic potential of BMAL1 activation in attenuating myocardial injury and advancing the development of circadian-targeted pharmacological strategies. To investigate the therapeutic potential of HIF2A activation in myocardial injury, C57BL/6J mice were treated with vadadustat, a prolyl hydroxylase domain (PHD) inhibitor15 (50 mg per kg, i.p., daily for 3 days) at ZT8 or ZT20. Vadadustat selectively stabilized HIF2A, particularly at ZT8, without affecting HIF1A levels (Extended Data Fig. 6a–d), potentially due to circadian regulation of PHD enzyme activity31. This stabilization significantly reduced infarct sizes and improved cardiac function in ZT8-treated mice, indicating time-dependent cardioprotection (Extended Data Fig. Analysis using co-IP revealed enhanced BMAL1–HIF2A binding in the nuclear fraction of the AAR at ZT8, accompanied by increased AREG expression in cardiomyocytes in the border zone after 3 h of reperfusion (Extended Data Fig. These findings suggest that BMAL1–HIF2A interaction and synergistic AREG induction underlie the circadian-dependent cardioprotection conferred by vadadustat at ZT8. To confirm this mechanism, vadadustat treatment was evaluated in Bmal1-deficient mice, which showed abolished cardioprotection at ZT8, highlighting a BMAL1-dependent mechanism (Supplementary Fig. A similar loss of protection was observed in Hif2a- and Areg-deficient mice, further supporting the essential role of the BMAL1–HIF2A–AREG pathway (Supplementary Fig. Moreover, cardiomyocyte-specific human-HIF2A knock-in mice (LSL-HIF2dPA myosin-Cre+) exhibited similar circadian-dependent HIF2A stabilization and increased Areg induction at ZT8 in mouse hearts (Supplementary Fig. Consistent with vadadustat treatment, HIF2A overexpression significantly reduced infarct size and improved cardiac function, with the most pronounced benefits at ZT8 (Supplementary Fig. These findings highlight the therapeutic potential of time-of-day-dependent HIF2A stabilization in enhancing cardioprotection, paving the way for new chronotherapy interventions. Circadian-dependent HIF2A stabilization was observed after myocardial IRI, hypoxia, vadadustat administration and in cardiomyocyte-specific HIF2A knock-in mice (Figs. 2i,j, 3h,i, Extended Data Fig. 9a,b), suggesting that clock genes may regulate HIF2A expression. However, RNA-seq and qPCR analyses revealed no circadian variations in Hif2a transcripts after IRI in humans and mice (Extended Data Fig. In Bmal1-deficient mice, nuclear HIF2A protein levels were reduced without changes in Hif2a transcripts, while NOB treatment further stabilized HIF2A protein at both ZT8 and ZT20, again without affecting transcript levels (Extended Data Fig. Gain-of-function and loss-of-function experiments in HCMs corroborated these findings, indicating a post-translational regulatory mechanism (Extended Data Fig. ChIP–qPCR analysis of predicted E-box elements in the HIF2A promoter showed no detectable BMAL1 binding, suggesting that BMAL1 does not directly regulate HIF2A transcriptionally (Extended Data Fig. Instead, cycloheximide (CHX) chase assays demonstrated that BMAL1 prolongs the HIF2A protein half-life by reducing its ubiquitin-mediated degradation (Extended Data Fig. Notably, BMAL1 did not affect HIF1A protein stability. These findings identify a role for BMAL1 in stabilizing HIF2A through post-translational mechanisms, offering insights into how circadian rhythms modulate hypoxic response during myocardial injury. Given BMAL1's interaction with HIF2A, we assessed its influence on the HIF2A–HIF1B complex, which mediates canonical hypoxia signalling. RNA-seq analysis of ischaemic LV biopsy samples from patients and mouse AAR revealed no circadian variation in Hif1b transcript levels (Extended Data Fig. Gain-of-function and loss-of-function experiments in mice and HCMs further confirmed that BMAL1 does not regulate HIF1B transcript or protein levels during IRI or hypoxia (Extended Data Fig. Co-IP analysis showed that BMAL1 overexpression has no impact on HIF2A–HIF1B binding, while ChIP–qPCR revealed no changes in HIF1B binding to the erythropoietin (EPO) promoter43 (Extended Data Fig. Moreover, HIF1B was not observed to bind the AREG promoter, and luciferase reporter assays confirmed that BMAL1 does not affect the transcriptional activity of the HIF2A–HIF1B complex on PGK1 (Extended Data Fig. These findings demonstrate that BMAL1 interacts with HIF2A without disrupting the canonical HIF2A–HIF1B complex, emphasizing its distinct role in circadian regulation of hypoxic responses. Although the interaction between BMAL1 and HIF2A has been reported38,44,45, the structural information of their heterodimerization remains unclear. To determine the structure using single-particle cryo-EM, we expressed and purified the N-terminal regions of BMAL1 and HIF2A, including the bHLH and PAS domains, bound to a 22-bp DNA containing a canonical HRE element (Fig. After two-dimensional (2D) and three-dimensional (3D) classification, the final approximately 43,000 polished particles were processed for 3D refinement, resulting in a density map with an average resolution of 3.6 Å (Fig. In brief, HIF2A exhibited a better overall resolution compared with BMAL1, probably due to its compact structure and stabilization by BMAL1. An atomic model of the DNA-bound heterodimer was built into the density map using the structures of BMAL1 and HIF2A from the BMAL1–CLOCK8 and HIF2A–HIF1B40 complexes as templates, respectively (Extended Data Fig. a, Cryo-EM map of the BMAL1–HIF2A–DNA complex. The map was sharpened using DeepEMhancer. c, Individual structures of HIF2A and BMAL1 within the complex. d, The four major interfaces (I to IV) between HIF2A and BMAL1. Interaction residues in BMAL1 (red) and HIF2A (purple) are shown; the BMAL1 residues mutated for pull-down analysis in e are shown in magenta. e, Pull-down analysis showing impaired interaction between GST–HIF2A and Flag-tagged BMAL1 mutants. f, The relative binding of BMAL1 mutants compared with WT BMAL1, with WT BMAL1 binding to HIF2A set to 1. n = 3 independent experiments. g, HEK293 cells overexpressing WT or mutated Flag-tagged BMAL1 were exposed to ambient hypoxia (1% O2) for 4 h, followed by IP with Flag and blotted with the indicated antibodies. h, Quantification of the relative binding in g. n = 3 independent experiments. i, HEK293 cells were transfected with either WT or mutant BMAL1 along with a HIF2A vector. A luciferase reporter assay was performed to evaluate transcriptional activation by the BMAL1–HIF2A complex of the AREG promoter, which contains a shared binding site. For f,h and i, data are mean ± s.e.m. j, Schematic illustrating the substantial structural rearrangement of BMAL1 (red) when accommodating various partners to be involved in different pathways. The PAS domains of BMAL1 (red) bend towards nearly opposite direction and position separately when intertwining with HIF2A (purple). Statistical analysis was performed using one-way ANOVA (f, h and i). The diagram in j was created using BioRender. In the structure, BMAL1 and HIF2A exhibit distinct architectures (Fig. HIF2A adopts a compact conformation, with the PAS-A domain bridging the bHLH and PAS-B domains, whereas the corresponding domains in BMAL1 are arranged more loosely, lacking interdomain contacts (Fig. BMAL1 uses its bHLH and PAS domains to wrap around their respective counterparts in HIF2A, establishing four major interfaces (I to IV) (Fig. Similar to classical bHLH transcription factors, the BMAL1–HIF2A heterodimer binds to the DNA through two α-helices, H1 and h1, from their bHLH domains, inserting into the major groove46 (Extended Data Fig. Notably, the structure of BMAL1 bound to HIF2A differs significantly from that of its complex with CLOCK8, in which the PAS domains of BMAL1 are tightly packed together (Extended Data Fig. By contrast, in the BMAL1–HIF2A structure, the PAS domains of BMAL1 are positioned separately and arranged in a conformation similar to the HIF1B observed in its complex with HIF2A. The conformational similarity between BMAL1 and HIF1B is probably due to their significant sequence identity (44%) in the N-terminal region, where several residues involved in HIF2A binding are conserved (Extended Data Fig. As both BMAL1–CLOCK and BMAL1–HIF2A complexes bind to DNA through their bHLH domains, we superimposed the bHLH domain of BMAL1 and observed that its PAS domains bend in nearly opposite directions. Similarly, superimposing the PAS-A domains of BMAL1 reveals conformational variability in the bHLH and PAS-B domains (Extended Data Fig. These structural comparisons suggest that BMAL1 undergoes significant conformational changes to accommodate different partners (Supplementary Video 1). Notably, proteins with PAS domains in the bHLH family typically form alpha–beta dimers47. In the BMAL1–HIF2A structure, two PAS-B domains form an open end-to-end alpha–beta dimer, with the conserved Trp427 in BMAL1 interacting with Met250 in HIF2A to stabilize the PAS-B dimer (Extended Data Fig. However, it is unclear whether this interaction allows HIF2A to bind to other factors, similar to the PAS-B interaction between CLOCK and CRY18,48. To strengthen our structural observations, we mutated conserved residues within each domain of BMAL1. These mutations disrupted BMAL1–HIF2A complex formation both in vitro and in HEK293 cells under hypoxia, underscoring the essential role of each domain in heterodimer stability (Fig. Using a luciferase-reporter assay with a human AREG-gluc reporter containing the common binding sequence (CAGGTG), we observed that BMAL1 mutants significantly impaired BMAL1–HIF2A transactivation at the AREG promoter, underscoring the importance of proper heterodimerization for AREG induction (Fig. Collectively, our study provides the structural basis of the DNA-bound BMAL1–HIF2A heterodimer, highlighting its role in regulating target genes, such as AREG, and the capability of BMAL1 in bridging diverse pathways through structural rearrangements for partner binding (Fig. Our study underscores the pivotal role of the BMAL1–HIF2A heterodimer in mediating circadian-dependent cardioprotection. This conclusion is supported by robust evidence, including the cryo-EM structure, the diurnal interaction between BMAL1 and HIF2A under hypoxia and during IRI, their functional roles in the synergistic transactivation of AREG and their interdependence in mediating circadian-dependent cardioprotection. However, we cannot conclusively prove that BMAL1–HIF2A heterodimerization is the bona fide mechanism regulating the time-of-day dependence of myocardial injury. Although our findings indicate that cardiomyocyte HIF1A does not significantly affect circadian variations in myocardial injury, the established interaction between BMAL1 and HIF1A31,49 warrants further investigation into whether BMAL1–HIF1A interactions in other cells contribute to circadian myocardial injury. Moreover, while we demonstrate that BMAL1 enhances HIF2A transcriptional activity and stabilizes it by reducing ubiquitination, the precise mechanisms remain unclear. For example, it is uncertain whether BMAL1 directly regulates HIF hydroxylation or influences HIF2A levels through alternative post-translational modifications. Finally, the contributions of other circadian rhythm molecules, such as PER2, and their interactions with HIFs may also have a role in this phenomenon27. Further elucidation of these pathways could advance chronotherapeutic strategies to optimize HIF-targeted interventions13,14,15. The widespread presence of peripheral molecular clocks across various cardiac cell types21 emphasizes the need to define BMAL1's tissue-specific roles for coordinated cardiac protection. Our BMAL1–HIF2A–DNA structure reveals BMAL1's ability to undergo structural rearrangements to bind a non-canonical partner, enabling cross-talk between signalling pathways. It also provides a molecular basis for developing therapeutics to stabilize this interaction, offering opportunities for treating ischaemic heart disease. Given the feasibility of integrating circadian timing into clinical practice, prospective trials are needed to evaluate whether aligning treatments with circadian phases or directly targeting circadian rhythms can improve outcomes in patients with MI. Beyond MI, circadian modulation of hypoxic responses may have broader implications for other hypoxia- and inflammation-related diseases with circadian characteristics, potentially paving the way for clock-based therapeutic innovations. Detailed information on experimental materials, reagents and mouse lines, including sources and identifiers, is provided in the key resources table in Supplementary Table 12. We examined samples from a prospective study of myocardial injury in humans undergoing cardiac surgery (ClinicalTrials.gov: NCT00281164). The study population consisted of consecutive patients (aged ≥20 years) with aortic stenosis referred to our cardiovascular surgery department at Brigham and Women's Hospital for aortic valve replacement (with or without coronary artery bypass graft) between 1 January 2009 and 31 December 2014. Patients enrolled in a concurrent drug or device trial were excluded. This ongoing study involved 56 patients in the morning (samples collected between 08:00 and 12:00; median time, 10:32) and 17 patients who underwent the same procedure in the afternoon (samples collected between 15:00 and 21:00; median time, 17:15). Patients whose surgery fell outside of these time periods were excluded from the analysis. The ethics committee for the Protection of Human Subjects from Research Risks of Brigham and Women's Hospital approved the protocol, and written informed consent was obtained from all of the patients. Patients underwent aortic valve replacement in the morning or afternoon by the same senior surgeon. Anaesthesia, cardiopulmonary bypass, cardioplegia and surgical procedures were done according to standard guidelines. Anaesthesia was induced with intravenous fentanyl or sufentanil and propofol (0.5–1.5 mg per kg) and maintained with isoflurane. Surgery was done using normothermic cardiopulmonary bypass and repeated antegrade and retrograde cold blood cardioplegia. LV biopsy samples were obtained from both the morning and afternoon patient cohorts before and after approximately 80 min of aortic cross-clamping—a procedure that temporarily halts blood flow to the heart. Animal care was performed according to the guide for the care and use of laboratory animals of the National Institutes of Health. All experimental procedures were approved by the UTHealth Institutional Animal Care and Use Committee. The sample size was estimated based on published literature on previous murine models of myocardial IRI27. All of the mice were housed under a standard 12 h–12 h light–dark photoperiod at 22 °C with ad libitum access to a standard chow diet, and maintained at a humidity level of 40–60%. To address sex as a biological variable, both male and female mice aged 8 to 16 weeks underwent myocardial ischaemia and reperfusion surgery. Our sex-specific analysis, which included assessments of infarct size, troponin levels and cardiac function, revealed no significant differences between sexes29,50. To induce Cre-recombinase activity, mice received an i.p. injection of tamoxifen at a dosage of 1 mg per day for five consecutive days, as previously described29,30,50. A recovery period of 7 days was allowed after the final tamoxifen dose before proceeding with further experimental procedures. In these experiments, myosin-Cre+ mice were used as controls. Moreover, Areg−/− mice (B6;129-Aregtm1Dle/Mmnc, MMRRC, MMRRC_011533-UNC)51 aged 8 weeks were purchased from Mutant Mouse Resources & Research Centers. To circumvent lactation difficulties observed in younger female mice, Areg−/− mice were bred using a heterozygous (Areg+/−) strategy16,41. Control mice for these experiments were C57BL/6J (The Jackson Laboratory, 000664) mice aged 8 weeks, chosen for their same genetic background as the Areg−/− mice. Genotyping for all mouse strains was conducted by GeneTyper (NY). The mouse model used in the study was subjected to a period of entrainment for at least 2 weeks within circadian cabinets (Actimetrics) to establish a stable circadian rhythm42. After this acclimatization period, the mice underwent an in situ procedure for myocardial IRI at different times of day (ZT2, ZT8, ZT14 or ZT20), as described in previous publications27,29,30,50,52. In brief, the mice were anaesthetized using 1–3% isoflurane delivered by a compact small-animal anaesthesia device (RWD), followed by endotracheal intubation and ventilation using a VentStar Small Animal Ventilator (RWD). Once anaesthetized, the mice were placed into a supine position and received pre-incisional analgesia with sustained-release buprenorphine (0.1 mg per kg, subcutaneous; ZooPharm). Throughout the procedure, their body temperature was consistently maintained at 37 °C with a ThermoStar Homeothermic Blanket equipped with rectal feedback control (RWD). The surgical procedure was meticulously performed using a research stereomicroscope system, SZX10 (Olympus). We temporarily ligated the proximal left anterior descending coronary artery approximately 2 mm from its emergence beneath the left atrium, using Surgipro II 7–0 monofilament polypropylene sutures (Covidien). The success of this ligation was confirmed by observing blanching or a pale discolouration of the anterior wall of the heart, a reduction in wall movement and the presence of ST-segment elevation on an electrocardiogram (ECG). After 45 min of induced ischaemia, reperfusion was initiated to restore the blood flow. The effectiveness of reperfusion was verified both by the resolution of ST-segment elevation on the ECG and through direct visual inspection, where a return of colour and movement in the previously affected area of the heart was noted. After surgery, the mice were carefully transferred back to the circadian cabinets for recovery and continued observation. Any mice that did not survive within the first 48 h after MI were considered to have experienced technical complications and were consequently excluded from further analysis. Blinding was implemented for the individuals analysing the data, including those calculating infarct size and measuring serum troponin I levels, to ensure unbiased data interpretation. Despite this, the surgeries were carried out by experienced surgeons, ensuring consistency. To confirm the similarity of conditions, we calculated the AAR/LV, which showed comparable values across ZT and treatment/vehicle groups. This suggests that any differences observed in the study are largely due to experimental conditions rather than variations in the surgical procedure. All cell lines and primary cells used in this study, including HEK293 cells (ATCC, CRL-1573) and human primary cardiomyocytes (ScienCell Research Laboratories, 6200), were authenticated by short-tandem-repeat profiling, as performed by the manufacturers. Moreover, the cell lines were tested for mycoplasma contamination by the manufacturers and were subsequently tested monthly for potential contamination throughout the study. All mycoplasma contamination tests yielded negative results. Furthermore, none of the cell lines used in this study are listed in the database of commonly misidentified cell lines maintained by the International Cell Line Authentication Committee (ICLAC) and NCBI Biosample. To simulate the clinical scenario in which treatment is initiated after the onset of MI, mouse recombinant AREG protein (R&D Systems) was prepared in 0.9% NaCl solution, and a dosage of 10 µg AREG was administered at the start of the reperfusion phase. Furthermore, to assess the long-term effects of timed AREG treatment on cardiac function, a post-injury treatment regimen was implemented. In this protocol, AREG (10 µg) was administered through i.p. injection at either ZT8 or ZT20 daily for the first 3 days after myocardial IRI. For control purposes, an equivalent volume of 0.9% NaCl solution was administered as a vehicle. In the NOB treatment experiments, Mice were administered either DMSO (as a vehicle control) or NOB at a dose of 200 mg per kg body weight. The administration was carried out through i.p. injection and repeated on an every-other-day basis. This treatment protocol was followed for 2 weeks before the surgical procedure and was specifically timed within the ZT14–ZT20 time window. To evaluate the long-term effects of NOB on cardiac function and remodelling, the same dosing regimen was continued after IRI. The chosen dosing regimen was based on several considerations: first, the dosage range was consistent with those used in previous studies (100–200 mg per kg per day)42,53, and it was aligned with the active phase of the mice. Second, daily dosing was intentionally avoided to prevent the entrainment of the experimental mice to the dosing schedule as an artificial zeitgeber. Third, previous single-dose pharmacokinetic assays demonstrated a favourable pharmacokinetic profile for NOB, with significant exposure detected in the serum, brain and liver42. Considering that NOB levels were typically undetectable within 8–24 h after administration42,54, we opted for an every-other-day dosing strategy to prevent incomplete clearance over the 4-week experimental period. The AAR of the heart was collected after 2 h of reperfusion from the C57BL/6J mice subjected to myocardial IRI at either ZT8 or ZT20. Total RNA was extracted using the RNeasy Mini kit (Qiagen) and was used to construct RNA-seq libraries, which were then sequenced using the Illumina 1.9 platform (75 bp paired end). Alignment of RNA sequencing tags was restricted to those mapping to the same DNA strand as annotated in the GRCm38 reference genome, using STAR (v.2.7.10a). Quantification of transcripts was conducted by calculating the fragments per kilobase of transcript per million mapped reads (FPKM) values, along with transcript counts. In the preprocessing step of mRNA analysis, any gene identified as non-expressed, defined as having an FPKM expression level less than 1 in more than 80% of samples, was excluded. The normalized expression profiles of these mRNAs were then subjected to PCA for quality control and to evaluate sample similarity. Differential expression analysis was performed using the DESeq255 pipeline, with DEGs being identified based on a threshold of 1.5-fold change and an adjusted P < 0.05, as determined using the Benjamini–Hochberg method56. For rigorous validation, only pathways and GO terms with an adjusted P < 0.05 were included. In constructing the gene regulation network, we used known transcription factor (TF)–mRNA interactions from the TRRUST_v258 and Chipbase_v259 databases. Moreover, the mouse protein–protein interaction (PPI) network was integrated from the STRING database (v.11.5)60, with a focus on interactions having a combined score above 900. Considering the limited sample size at each timepoint, we merged six samples from both conditions to calculate the Spearman's correlation coefficient (SCC), ensuring robust and meaningful analysis. In finalizing the gene regulation network, we merged the TF–mRNA regulation network with the PPI data, specifically targeting interactions where both genes were differentially expressed. We also selectively incorporated edges with high SCC into our network, tailored to align with the specific conditions of our experimental design. For human RNA-seq and subsequent bioinformatic analysis, we began by excluding outlier samples that failed to cluster appropriately, resulting in a total of 73 samples for analysis, including 56 morning and 17 afternoon samples derived from various sequencing protocols. Quality control of raw sequencing reads identified adaptor sequences from the Illumina Nextera platform in some samples, which were subsequently trimmed using Cutadapt (v.4.1)61. kallisto (v.0.46.1)62 was used to quantify transcript-level expression by mapping to a transcript index built from GENCODE human transcript (v.44)63. We next used tximport to convert these transcript-level quantifications to gene read counts64. Differential expression analysis was performed with DESeq2 (v.1.34.0)55, with particular attention paid to sequencing batch to control for batch effects. Genes displaying a log2-transformed fold change of greater than 0.5 and P < 0.01 were deemed to be significant. For enrichment analysis, we used the GO annotation (v.1.1) for humans, using all genes annotated by at least one GO term65 as the background in Fisher's exact test to identify over-represented biological processes. Moreover, the R package KEGGREST (v.1.36.0) was used to extract KEGG pathway annotations, and Cytoscape (v.3.10.0) was used to construct network plots for the identified KEGG pathways. The microarray assay for gene expression transcript levels of post-ischaemic myocardium from myosin-Cre+ or Hif2aloxP/loxP myosin-Cre+ mice was reanalysed using data obtained from the GEO (GSE67308)16. Differential expression analysis was performed using GEO2R, with limma precision weights applied and the remaining options set to default66. The assessment of myocardial infarct size was conducted by determining the percentage of infarcted myocardium within the AAR using a previously established method16,27,29,30,50. After 2 h or 1 day of reperfusion, the hearts were flushed with PBS and then subjected to permanent occlusion of the left coronary artery. 1 ml of 1% Evans blue dye (Sigma-Aldrich, E2129) was then infused through the carotid artery catheter. After infusion, the hearts were excised and sectioned into 1 mm slices using a microtome (Roboz, SA-4130). These heart sections were subsequently double-stained with 1% TTC (Sigma-Aldrich, T8877) for 10 min at 37 °C and then fixed in 10% formalin overnight. The double-stained heart slices were photographed, and ImageJ software (NIH, Fiji v.2.1.0) was used to calculate the infarct size. Serum samples were collected from mice subjected to myocardial IRI at various ZTs through the inferior vena cava after 2 h of reperfusion. The measurement of serum troponin I, a highly sensitive biomarker for cardiac injury, was conducted using the mouse cardiac troponin-I SPARCL kit (Life Diagnostics, CTNI-SP-1), as we have done previously16,27,29,30,50. Cardiac function and structure were assessed by transthoracic echocardiography using the Vevo3100 Ultrasound system (VisualSonics). Mice were lightly sedated with 0.5–1.0% isoflurane and positioned on a heated platform equipped with ECG monitoring. Validation criteria included consistent, uninterrupted tracking of the endocardium throughout the cardiac cycles, maintaining a heart rate between 450 and 550 beats per minute, and a frame rate exceeding 250 fps. Two-dimensional grey-scale echocardiographic images were obtained from parasternal long-axis and short-axis views. Longitudinal strain represented myocardial shortening at the endocardium, while radial strain indicated shortening at the mesocardium. All image acquisitions and subsequent offline measurements were conducted by a single investigator who was blinded to the grouping of the animals. For strain analysis using Vevo LAB (FUJIFILM VisualSonics, v.5.7.1), we carefully selected suitable B-mode loops, ensuring a clear view of the endocardial border and no image artefacts. Three consecutive cardiac cycles with distinct ECG recordings were chosen for in-depth analysis. Semi-automated tracing of the endocardial and epicardial borders was performed, with adjustments made as necessary to maintain optimal tracking quality throughout each cine loop. The resulting strain values were averaged across these cardiac cycles, providing comprehensive data on the LV systolic function (EF, FS and GLS), LV size (end-diastolic volume and end-systolic volume) and LV mass (end-diastolic LV mass and end-systolic LV mass). In analysing regional and global cardiac dynamics, such as contractility and synchronicity, the LV endocardium was delineated using 48 sampling points34,67. This method divided the chamber into six segments for detailed examination in the long-axis view: basal anterior, mid anterior, apical anterior, basal posterior, mid posterior and apical posterior segments67. In myocardial IRI models, the mid anterior, apical anterior and apical posterior wall segments were designated as the infarct region, and the remaining segments were categorized as the non-infarct region34. Peak systolic strain, indicating the percentage change in length during myocardial contraction and relaxation, was calculated using the formula: ε = (L1 − L0)/L0, where L0 is the initial length and L1 is the final length69. This calculation was applied to each segment to evaluate the peak systolic strain (%) and the time-to-peak systolic strain (ms). Abnormal ventricular contractility patterns were evaluated on the basis of the magnitude of systolic strain (segment peak strain) and its timing (peak of shortening). Dyssynchrony was specifically identified by a pattern of reduced systolic strain magnitude, early opposing deflection and a delayed time-to-peak systole. Akinesis was defined as minimal or no contractility, with peak systolic strain values between −5% and 5%. Dyskinesis was described as systolic motion of the ventricle occurring in the opposite direction of normal contraction. of the time-to-peak strain across segments34,70. HCMs were obtained from ScienCell Research Laboratories (ScienCell, 6200) and isolated from human hearts71. The cells were synchronized using 200 nM dexamethasone, which was replaced with complete media (ScienCell, 6101) after 1 h. After synchronization, we initiated a series of hypoxia treatments, exposing different batches of cells to 1% oxygen for 4 h at systematic 4-h intervals to capture cellular responses at various circadian phases. The first hypoxia session started immediately after synchronization (CT0) and lasted for 4 h (referred to as the CT4 timepoint). Subsequent treatments were applied every 4 h: from CT4 to CT8 (CT8 timepoint), and so on, continuing up to CT40. Immediately after each hypoxia period, cells were collected to evaluate transcript and protein levels. To ensure valid results, we carefully selected timepoints for post-synchronization analysis. After synchronization, cells require a stabilization period to re-establish their circadian rhythms, and early evaluations may capture this adjustment phase rather than true circadian behaviour72. Moreover, dexamethasone induces immediate stress responses and activates signalling pathways unrelated to circadian changes73,74. Administering hypoxia treatments too soon after synchronization could overlap these stress responses, confounding the circadian-related changes we aim to measure. For this reason, early timepoints like CT8 were excluded, and we focused on later timepoints to minimize these effects and ensure that the data reflect circadian-driven changes. HCMs were synchronized by dexamethasone (200 nM) for 1 h and then exposed to normoxia or hypoxia (1% oxygen) for 4 h at CT20 and CT32 in Fig. HEK293 cells were transfected with pcDNA3-mBmal1 (a gift from A. Sancar, Addgene, 31367)75 and treated with normoxia or hypoxia (1% oxygen) for 4 h in Fig. ChIP–qPCR was conducted using the SimpleChIP enzymatic chromatin IP kit (Cell Signaling Technology, 9002). In brief, cells were fixed with 1% formaldehyde, quenched with 125 mM glycine, washed and sonicated to digest the DNA to an average length of 150–500 bp. Lysates were then incubated overnight at 4 °C with 2 µg of ChIP grade anti-HIF2A antibody (Novus Biologicals, NB100-122) and anti-BMAL1 antibody (Cell Signaling Technology, 14020), anti-HIF1B antibody (Cell Signaling Technology, 5537) or IgG as a negative control. The cross-links were then reversed through overnight incubation at 65 °C, followed by DNA purification. For the qPCR analysis, specific primer pairs were designed to amplify the predicted common binding sequence (CAGGTG) on the human AREG promoter region or E-box regions on the human HIF2A promoter region. The enrichment was quantitatively compared against input controls and IgG negative controls to confirm the specificity of the ChIP-qPCR assay. 2b) overexpressing the pcDNA3-mBmal1 (a gift from A. Sancar, Addgene, 31367)75 were exposed to either normoxia or hypoxia (1% oxygen) for 4 h. In Fig. 3r, HCMs were initially synchronized using dexamethasone (200 nM) for 1 h and then exposed to hypoxia (1% oxygen) for 4 h at CT20 and CT32. After exposure, the cytoplasmic and nuclear fractions were separately extracted using the NE-PER Nuclear and Cytoplasmic Extraction Kit (Thermo Fisher Scientific, 78835), according to the manufacturer's instructions. The cell lysates were centrifuged and pre-cleared with protein A/G beads (Thermo Fisher Scientific, 53133) for 1 h with rotation. For the IP, the pre-cleared lysates were incubated overnight at 4 °C with specific antibodies: mouse anti-Flag (Sigma-Aldrich, F1804) or rabbit anti-HIF2A (Novus Biologicals, NB100-122) antibodies. Mouse IgG (Cell Signaling Technology, 5415) and rabbit IgG (Abcam, ab172730) were used as isotype controls for the respective antibodies. After the overnight incubation, 40 μl of protein A/G beads were added to each sample, and the incubation was continued for an additional 4 h at 4 °C. The proteins were then eluted using an SDS sample buffer and subjected to heating at 95 °C for 5 min with vigorous shaking. The eluted proteins were subsequently stored at −80 °C for further analysis. 2d, HCMs were subjected to hypoxic conditions (1% oxygen) for various durations: 0, 0.5, 1, 2, 4 or 8 h. After each exposure period, HCMs from a single 10-cm dish were lysed using 330 µl of HEPES extraction buffer containing 20 mM HEPES pH 7.4, 100 mM NaCl, 1 mM EDTA, 0.1% Triton X-100, 5% glycerol, and a cocktail of protease, phosphatase and RNase inhibitors. Co-IP was then performed according to the previously described protocol. 7w,x, HEK293 cells were transfected with UB–HA, HIF2A–MYC or BMAL1–Flag expression constructs. To assess the ubiquitination of HIF2A, cells were treated with 10 µM MG132 for 16 h. After treatment, the cells were collected and lysed using HEPES extraction buffer. Immunoprecipitation was then performed using rabbit anti-HIF2A antibodies (Novus, NB100-122) as previously described. HCMs were transduced with Bmal1-AAV (105 GC per cell) or shBMAL1-AAV (105 GC per cell) and exposed to 1% O2 for 4 h to stabilize HIF1A and HIF2A. After the return to normoxic conditions, CHX (20 μg ml−1, Sigma-Aldrich, C4859) was added to inhibit de novo protein synthesis. Cells were collected at 15, 30, 60, 90 and 120 min after CHX treatment, washed with cold PBS, and lysed in RIPA buffer containing protease and phosphatase inhibitors. After centrifugation, protein concentrations were determined using a BCA protein assay kit. Equal amounts of protein (10–20 μg) were separated by SDS–PAGE and probed with primary antibodies against HIF2A (Novus Biologicals, NB100-122, 1:1,000), HIF1A (Novus Biologicals, NB100-479, 1:1,000), BMAL1 (Cell Signaling Technology, 14020, 1:1,000) and β-actin (Cell Signaling Technology, 4967, 1:1,000). The relative protein levels at each timepoint were quantified using ImageJ (NIH, v.2.1.0) and normalized to β-actin as a loading control. For each experimental group, normalized protein levels at 15, 30, 60, 90 and 120 min were expressed as the percentage of the 0-min timepoint (set as 100%). Confluent HCMs were subjected to hypoxic conditions (1% oxygen) for various durations (0, 1, 4 or 8 h). After exposure, the cells were fixed using 4% paraformaldehyde (Millipore Sigma, P6148), permeabilized with 0.1% Triton X-100 (Amresco, M143), and subjected to a blocking step to reduce non-specific binding. Rabbit anti-BMAL1 (Abcam, ab3350, 1:100), mouse anti-HIF2A (Novus Biologicals, NB100-132, 1:100) and mouse anti-HIF1A (Novus Biologicals, NB100-105, 1:100) were then applied. For PLA, pairs of primary antibodies raised in different species were used as mentioned. After primary antibody incubation, cells were treated with Duolink In Situ PLA Probes (Millipore Sigma, DUO92002 and DUO92004) conjugated to these antibodies. These probes, when in close proximity of less than 40 nm, facilitated the ligation of adjacent oligonucleotides attached to them. Subsequently, the ligated oligonucleotides were amplified and detected using fluorescently labelled probes. The resulting fluorescence signals, which are indicative of PPIs, were visualized using a Nikon Eclipse Ti2 confocal microscope (Nikon) and analysed with NIS Element AR software (Nikon, v6.10.01). To overexpress proteins in Escherichia coli, the DNA sequences encoding the N-terminal portions of mouse HIF2A (residues 3–361) and mouse HIF1A (residues 13–357), with a 6×His tag at their N terminus, were inserted into the pET15b vector. The DNA encoding the N-terminal region of mouse BMAL1 protein (residues 68–488), with a C-terminal Flag tag, was ligated into the pET24b vector. For GST pull-down, the DNA encoding residues 3–361 of mouse HIF2A was inserted into pGEX-6P-1 to generate GST–HIF2A for expression. The DNA sequences of all of the mutants were verified by DNA sequencing. The recombinant plasmids were individually transformed into E. coli BL21 Rosetta (DE3) or co-transformed together to express either individual proteins or the BMAL1–HIF2A complex. Expression of the proteins or BMAL1–HIF2A complex was carried out at 18 °C by induction with 0.1 mM isopropyl-β-d-thiogalactoside for 16 h. The BMAL1–HIF2A heterodimer (including bHLH, PAS-A and PAS-B domains) was first purified through a Ni-NTA affinity column (Thermo Fisher Scientific, 88222) followed by heparin chromatography. The peak fractions were assessed using SDS–PAGE and Coomassie blue staining. Subsequently, the samples were combined and concentrated. The concentrated protein was then used for cryo-EM sample preparation or stored at −80 °C for further use. His–HIF2A and His–HIF1A were purified individually using Ni-NTA affinity columns, followed by dialysis to eliminate imidazole. The proteins were concentrated and stored at −80 °C for further use. To confirm the interaction between BMAL1 and HIF2A or HIF1A in Fig. After centrifugation, the supernatants were incubated with anti-Flag M2 beads (Millipore Sigma, A2220) for 40 min at 4 °C. After incubation for 40 min at 4 °C, the beads were washed five times using the binding buffer. The proteins bound to beads were then eluted using Flag peptide (0.3 mg ml−1) and further analysed by SDS–PAGE and western blotting using anti-Flag (Sigma-Aldrich, F1804, 1:1,000) and anti-His (Thermo Fisher Scientific, MA1-21315, 1:1,000) antibodies. To compare the interactions between HIF2A and WT BMAL1 or BMAL1 mutants in Fig. 5e, the cell pellets of GST–HIF2A and GST (as a control) were lysed by sonication in binding buffer (20 mM HEPES (pH 7.4), 10% glycerol, 0.05% Triton X-100, 300 mM NaCl, 5 mM MgCl2, 1 mM DTT and protease inhibitors). After centrifugation, the supernatants were incubated with Glutathione magnetic agarose beads (GE Healthcare, 17-0756-01) for 30 min at 4 °C. The beads were subsequently washed three times using the binding buffer, followed by the addition of Flag-tagged WT or mutated BMAL1. After 40 min incubation at 4 °C, the beads were washed five times using the binding buffer. The proteins bound to beads were eluted by heating with SDS–PAGE loading dye at 95 °C. The eluted samples were further analysed by SDS–PAGE and western blotting using anti-GST (Genscript, A00865, 1:1,000) and anti-Flag (Sigma-Aldrich, F1804, 1:1,000) antibodies. Five fmol of 22-bp biotin-labelled dsDNA fragment containing an HRE consensus sequence was incubated with 1, 2, 3, 4 and 6 pmol of purified BMAL1–HIF2A heterodimer in a total volume of 10 µl in 10 mM MES (pH 6.0), 50 mM KCl, 1 mM DTT, 2.5% glycerol, 0.05% NP-40 and 5 mM MgCl2. After incubation at room temperature for 20 min, the binding reactions were directly loaded onto a native 4–20% polyacrylamide gel and electrophoresed in 0.5× TBE buffer. DNA on the gel was transferred onto a positively charged nylon membrane and detected using an HRP-conjugated streptavidin with chemiluminescent substrates. The interaction analysis of the BMAL1–HIF2A heterodimer with 22-bp DNA duplexes (HRE or E-box) was performed using OpenSPR (Nicoya Life Sciences). A 5′-end biotin-labelled dsDNA fragment (2 μM) containing either an HRE or E-box consensus sequence was immobilized in channel 2 of the biotin–streptavidin sensor, while channel 1 remained unmodified and served as a control. The BMAL1–HIF2A heterodimer at various concentrations (from 5 to 50 nM) was injected and flowed slowly at a rate of 20 μl min−1 over the sensor chip for 5 min in a running buffer (20 mM MES pH 5.5, 300 mM NaCl, 0.05% Tween-20, 0.02% BSA). After the injection, a 10-min dissociation phase was collected. The resulting data were analysed using Trace Drawer Kinetic Data Analysis software (v.1.9.2), using a one-to-one model, which assumes one monovalent ligand binding to one target. HEK293 cells were co-transfected with several constructs: pcDNA3-mHif2a-P405A/P530V/N851A (oxygen-regulation insensitive, a gift from C. Simon, Addgene, 44027)76, pcDNA3-mBmal1 (a gift from A. Sancar, Addgene, 31367)75, hAREG-gluc and SEAP vectors (GeneCopoeia, LF031). Transfection was performed using Lipofectamine 3000 reagent (Invitrogen, L3000015). After 48 h, the culture medium was collected, and the transcriptional activity was quantified using the Secrete-pair Dual Luminescence Assay Kit (GeneCopoeia, LF031) on the Cytation5 (Agilent Technologies) device with Gen5 software (v.3.12, Agilent Technologies). To investigate the transcriptional activity of HIF2A and BMAL1 on a general HIF target gene, HEK293 cells were transfected with the hPGK1-luc reporter (a gift from C. V. Dang, Addgene plasmid, 128095) along with HIF2A or BMAL1 expression constructs. After 48 h of transfection, firefly and Renilla luciferase activities were measured using the Dual-Luciferase Reporter Assay System (Promega, E1690). The luciferase activity was detected on the Cytation5 (Agilent Technologies) system and analysed using Gen5 software (v.3.12, Agilent Technologies). On day 1 after MI, collected heart tissues were fixed in 10% neutral-buffered formalin and then dehydrated through a graded alcohol series. After dehydration, the tissues were cleared in xylene and embedded in paraffin blocks. Sections of 5–6-μm thickness were prepared and underwent antigen retrieval. These sections were then blocked with diluted donkey serum, followed by overnight incubation with primary antibodies at 4 °C. In the immunofluorescence staining process, a range of primary antibodies was used to specifically target and identify various proteins within the tissue sections. These primary antibodies included: AREG (Santa Cruz, sc-74501, 1:50), HIF2A (Novus Biologicals, NB100-122, 1:200; Novus Biologicals, NB100-132, 1:200), BMAL1 (Abcam, ab3350, 1:200), α-sarcomeric (Abcam, ab137346, 1:200), vimentin (Cell Signaling Technology, 5741, 1:200), and α-smooth muscle actin (Cell Signaling Technology, 19245, 1:200) antibodies. For antigen visualization, the sections were then incubated with Alexa fluorescence-conjugated secondary antibodies (Invitrogen). Moreover, TUNEL staining was conducted using in situ Click-iT Plus TUNEL assay kits (Thermo Fisher Scientific, C10617) according to the manufacturer's protocol to detect apoptotic cells as previously described77. The samples were counterstained with DAPI (1 μg ml−1, Invitrogen, D3571) and mounted using SlowFade Gold Antifade reagent (Invitrogen, S36936). The stained sections were then imaged with Nikon Eclipse Ti2 confocal microscopy (Nikon) and analysed using ImageJ software (NIH, Fiji v.2.1.0). Total RNA was extracted from both tissue samples and isolated cells using the RNeasy Mini Kit (Qiagen, 74106) according to the manufacturer's guidelines. This procedure is consistent with our established protocols as described in previous publications16,29,50. After the extraction, cDNA was synthesized from the extracted RNA using the High-Capacity cDNA Reverse Transcription Kit (Thermo Fisher Scientific, 4368814). qPCR was then performed using SYBR Green PCR Master Mix (Qiagen, 204145). The reactions were performed on the Bio-Rad CFX384 Touch Real-Time PCR Detection System. For the analysis of gene expression levels, we applied the comparative Ct (ΔΔCt) method. The final data were presented as mean expression ratios relative to β-actin, allowing for the comparison of gene expression across different samples. Mouse tissues and isolated cells were prepared for western blot analysis by lysing in RIPA lysis buffer (Thermo Fisher Scientific, 89900), supplemented with both protease (Thermo Fisher Scientific, 78425) and phosphatase inhibitor cocktails (Thermo Fisher Scientific, 78420). Protein concentrations were quantified, and 10–20 μg of total protein was separated on 4–12% SDS-PAGE gels (Bio-Rad Laboratories). After electrophoresis, the proteins were transferred to membranes for immunoblotting. The membranes were probed with the following primary antibodies: anti-HIF1A (Novus, NB100-105, 1:1,000), anti-HIF2A (Novus, NB100-122, 1:1,000), anti-HIF2A (Bethyl Laboratories, A700-003, 1:1,000), anti-HIF1A (Bethyl Laboratories, A700-001, 1:1,000), anti-HIF1B (Cell Signaling Technology, 5537. After incubation with secondary antibodies, the membranes were developed using appropriate substrates. The signal intensity was detected and quantified using ImageJ software (NIH, Fiji v.2.1.0). The results were normalized to the appropriate internal control, and data were expressed as the relative fold changes in protein levels. The purified BMAL1–HIF2A heterodimer was mixed with a twofold molar excess of the HRE dsDNA, followed by incubation on ice for 20 min. The eluate fractions from the column were analysed by SDS–PAGE and stained with Coomassie blue. The peak fractions corresponding to the BMAL1–HIF2A–DNA complex were combined and concentrated to 0.5 mg ml−1 for preparing cryo-EM specimens. In brief, 3 μl of the purified complex was applied onto freshly glow-discharged R 2/1 holey carbon 300 mesh copper grids (C-flat). The grids were then vitrified in liquid ethane using an FEI Vitrobot (Mark IV). Single-particle cryo-EM images were collected using the FEI Titan Krios (300 KV) electron microscope with a Gatan GIF Quantum K2 direct electron detector. The images were automatically acquired using EPU (Thermo Fisher Scientific, v.2.10) at underfocus values ranging from 1.0 to 2.5 μm with a pixel size of 0.85 Å px−1. Each micrograph was exposed for 9 s with a total dose of about 75 e− Å−2, which was fractionated into 45 frames. The movie frames were aligned using MotionCor2 (v.1.4.0)78, resulting in a total of 18,039 micrographs. Gctf (v.1.18)79 was used to estimate the parameters of contrast transfer function (CTF) for each micrograph. crYOLO (v.1.10)80 was used for reference-free automatic particle picking, yielding a total of approximately 710,000 particles. The particles were extracted with a box size of 352. Multiple rounds of Cryosparc (v.-2.5)81 2D classification were initially performed to remove junk particles. Next, the remaining particles were subjected to an additional round of 2D classification using RELION (v.3.1)82, yielding a stack of 420,000 particles. An ab initio model, showing secondary structural features bound to DNA, was generated using CryoSPARC and then used as a reference for subsequent 3D analysis. The particles were rescaled to a pixel size of 1.02 Å px−1 with a box size of 328 pixels and subsequently subjected to 3D refinement. The reference map for 3D refinement was filtered to 25 Å, and a low-pass filtered mask at 25 Å was generated using a threshold that included the protein and DNA regions, followed by a 4-pixel dilation and 5-pixel soft padding. 3D refinement in RELION produced a 4.7 Å-resolution map of the BMAL1–HIF2A–DNA complex. Next, these particles were further processed by CTF refinement and Bayesian polishing in RELION, resulting in 141,218 shiny particles. These particles were then subjected to 3D refinement, producing a map with a resolution of 4.3 Å. Multiple rounds of 3D classification, with and without alignment, were then conducted. A final set of 43,098 shiny particles was used for 3D refinement, resulting in a map with an average resolution of 3.6 Å. The resolution was calculated using a low-pass filtered mask at 15 Å, created with a threshold of 0.008, a 3-pixel dilation and 3-pixel soft padding. The local resolutions were calculated using CryoSPARC. The resolutions of the 3D maps were estimated using gold-standard Fourier shell correlation (FSC) curves with a 0.143 cut-off criteria84. The final set of 43,098 particles had an Eod value of 0.74 with the best and worst PSF resolutions of 2.8 and 5.3 Å, respectively, as calculated by cryoEF (v.1.1.0)86, indicating minor anisotropy. The cryo-EM data collection and image analysis procedures are shown in Supplementary Table 11 and Extended Data Fig. To build an atomic model of the BMAL1–HIF2A–DNA complex, the crystal structure of HIF2A (Protein Data Bank (PDB): 4ZP4) and a 22-bp B-form DNA duplex were used as templates and fitted into the deepEMhancer map of BMAL1–HIF2A–DNA by rigid-body fitting in Chimera (v.1.15)87. The model was then manually built and adjusted in Coot (v.1.1)88, followed by real-space refinement in Phenix (v.1.21)89 (Supplementary Table 11). In the final BMAL1–HIF2A–DNA atomic model, amino acids for HIF2A (3–12, 76–88, 151–163, 174–185, 201–219 and 356–361) and BMAL1 (68–74, 102–110, 126–144, 164–167, 210–242, 254–279, 291–311, 321–337, 345–349, 407–412 and 441–488) as well as a four DNA bases near the terminal ends were not built because of missing or poor densities. The refined model statics are shown in Supplementary Table 11. All molecular graphic figures were generated by Chimera, ChimeraX (v.1.7)90 and PyMOL (v.2.5.5)91. Unless otherwise specified, all results are presented as the mean ± s.e.m., with the precise number of biological replicates (n) described in the figure legends. All data were plotted from independent biological replicates. The Shapiro–Wilk normality test was used to assess normal distribution. Statistical analysis was performed using GraphPad Prism 10.0, using unpaired t-tests (two-tailed), Welch's t-tests (for unequal variances), Mann–Whitney U-tests (when normality assumptions were not met) or one-way ANOVA with Bonferroni's multiple-comparison analysis, depending on the dataset. Outliers were identified using the ROUT method (Q = 1%) in GraphPad Prism. For ChIP–qPCR data analysis in Fig. 3s, values identified as outliers were excluded from the statistical analysis. Cardiac function assessment in Bmal1loxP/loxP myosin-Cre+, Hif2aloxP/loxP myosin-Cre+ and Hif1aloxP/loxP myosin-Cre+ mice was compared to a shared myosin-Cre+ control group. Cardiac function data from Bmal1loxP/loxP myosin-Cre+ mice were used as the control for comparing NOB-treated Bmal1loxP/loxP myosin-Cre+ mice. Similarly, cardiac function data from Areg−/− mice were used as the control for comparing NOB-treated Areg−/− mice. Cardiac function data from C57BL/6J mice were partially used as the control group for Areg−/− mice. This strategy minimized animal use and ensured a consistent baseline, optimizing research efficiency while enabling reliable comparative analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The mouse myocardial IRI heart bulk RNA-seq data are available at the NCBI GEO database under accession number GSE255307. Analyses of the mouse RNA-seq data were performed using the Mus musculus reference genome assembly GRCm38 (Genome Reference Consortium Mouse Build 38), which is available at the NCBI under accession number GCF_000001635.20. Human surgical LV bulk RNA-seq data are available under controlled access through the NIH database of Genotypes and Phenotypes (dbGaP) under accession number phs001679.v1.p1. Access to these data is restricted due to privacy and ethical considerations. Requests must be submitted through dbGaP's data access request process. Interested researchers should apply for access through dbGaP by contacting the NIH Data Access Committee (DAC) and providing a detailed research proposal outlining the intended use of the data. The DAC typically reviews requests within 2 weeks, and access is granted subject to compliance with data use agreements. For further details on controlled access policies, data use agreements and any additional restrictions, please refer to the dbGaP study page linked above. Questions regarding data access can be directed to the dbGaP helpdesk. The microarray assay for gene expression transcript levels in post-ischaemic myocardium from myosin-Cre+ or Hif2aloxP/loxP myosin-Cre+ mice was re-analysed using data obtained from the GEO (GSE67308). The corresponding atomic model was deposited at the RCSB PDB under accession number 8VHG. Source data are provided with this paper. Suarez-Barrientos, A. et al. Circadian variations of infarct size in acute myocardial infarction. Fournier, S. et al. Circadian variations of ischemic burden among patients with myocardial infarction undergoing primary percutaneous coronary intervention. Reiter, R., Swingen, C., Moore, L., Henry, T. D. & Traverse, J. H. Circadian dependence of infarct size and left ventricular function after ST elevation myocardial infarction. Role of the circadian system in cardiovascular disease. & Eltzschig, H. K. Circadian rhythm as a therapeutic target. Crystal structure of the heterodimeric CLOCK: BMAL1 transcriptional activator complex. Bunger, M. K. et al. Mop3 is an essential component of the master circadian pacemaker in mammals. Ko, C. H. & Takahashi, J. S. Molecular components of the mammalian circadian clock. Takahashi, J. S. Transcriptional architecture of the mammalian circadian clock. Tian, H., McKnight, S. L. & Russell, D. W. Endothelial PAS domain protein 1 (EPAS1), a transcription factor selectively expressed in endothelial cells. Eltzschig, H. K., Bratton, D. L. & Colgan, S. P. Targeting hypoxia signalling for the treatment of ischaemic and inflammatory diseases. Interplay of hypoxia-inducible factors and oxygen therapy in cardiovascular medicine. Yuan, X., Ruan, W., Bobrow, B., Carmeliet, P. & Eltzschig, H. K. Targeting hypoxia-inducible factors: therapeutic opportunities and challenges. Koeppen, M. et al. Hypoxia-inducible factor 2-alpha-dependent induction of amphiregulin dampens myocardial ischemia-reperfusion injury. Shoyab, M., Plowman, G. D., McDonald, V. L., Bradley, J. G. & Todaro, G. J. Structure and function of human amphiregulin: a member of the epidermal growth factor family. Bargiello, T. A., Jackson, F. R. & Young, M. W. Restoration of circadian behavioural rhythms by gene transfer in Drosophila. Hardin, P. E., Hall, J. C. & Rosbash, M. Feedback of the Drosophila period gene product on circadian cycling of its messenger RNA levels. Koike, N. et al. Transcriptional architecture and chromatin landscape of the core circadian clock in mammals. Crnko, S., Du Pré, B. C., Sluijter, J. P. & Van Laake, L. W. Circadian rhythms and the molecular clock in cardiovascular biology and disease. Young, M. E. et al. Cardiomyocyte-specific BMAL1 plays critical roles in metabolism, signaling, and maintenance of contractile function of the heart. Bray, M. S. et al. Disruption of the circadian clock within the cardiomyocyte influences myocardial contractile function, metabolism, and gene expression. Ingle, K. A. et al. Cardiomyocyte-specific Bmal1 deletion in mice triggers diastolic dysfunction, extracellular matrix response, and impaired resolution of inflammation. Lecour, S. et al. Circadian rhythms in ischaemic heart disease: key aspects for preclinical and translational research: position paper of the ESC working group on cellular biology of the heart. Eltzschig, H. K. & Carmeliet, P. Hypoxia and inflammation. Eckle, T. et al. Adora2b-elicited Per2 stabilization promotes a HIF-dependent metabolic switch crucial for myocardial adaptation to ischemia. The hypoxia-adenosine link during myocardial ischemia—reperfusion injury. Targeting myocardial equilibrative nucleoside transporter ENT1 provides cardioprotection by enhancing myeloid Adora2b signaling. Heck-Swain, K. L. et al. Myeloid hypoxia-inducible factor HIF1A provides cardio-protection during ischemia and reperfusion via induction of netrin-1. Reciprocal regulation between the circadian clock and hypoxia signaling at the genome level in mammals. Lefta, M., Campbell, K. S., Feng, H.-Z., Jin, J.-P. & Esser, K. A. Development of dilated cardiomyopathy in Bmal1-deficient mice. Liang, Q. et al. Postnatal deletion of Bmal1 in cardiomyocyte promotes pressure overload induced cardiac remodeling in mice. Bhan, A. et al. High-frequency speckle tracking echocardiography in the assessment of left ventricular function and remodeling after murine myocardial infarction. Mollema, S. A. et al. Left ventricular dyssynchrony acutely after myocardial infarction predicts left ventricular remodeling. Luck, K. et al. A reference map of the human binary protein interactome. The basic-helix–loop–helix-PAS orphan MOP3 forms transcriptionally active complexes with circadian and hypoxia factors. Direct observation of individual endogenous protein complexes in situ by proximity ligation. Wu, D., Potluri, N., Lu, J., Kim, Y. & Rastinejad, F. Structural integration in hypoxia-inducible factors. Lee, J. W. et al. Transcription-independent induction of ERBB1 through hypoxia-inducible factor 2A provides cardioprotection during ischemia and reperfusion. The small molecule nobiletin targets the molecular oscillator to enhance circadian rhythms and protect against metabolic syndrome. Orlando, I. M. C. et al. Distal and proximal hypoxia response elements cooperate to regulate organ-specific erythropoietin gene expression. Saito, T. et al. Transcriptional regulation of endochondral ossification by HIF-2α during skeletal growth and osteoarthritis development. Liu, N. et al. A highland-adaptation mutation of the Epas1 protein increases its stability and disrupts the circadian clock in the plateau pika. de Martin, X., Sodaei, R. & Santpere, G. Mechanisms of binding specificity among bHLH transcription factors. Rojas, B. L., Vazquez-Rivera, E., Partch, C. L. & Bradfield, C. A. Dimerization rules of mammalian PAS proteins. Peek, C. B. et al. Circadian clock interaction with HIF1α mediates oxygenic metabolism and anaerobic glycolysis in skeletal muscle. Li, J. et al. PMN-derived netrin-1 attenuates cardiac ischemia-reperfusion injury via myeloid ADORA2B signaling. Targeted inactivation of the EGF and amphiregulin genes reveals distinct roles for EGF receptor ligands in mouse mammary gland development. DeBerge, M. et al. Hypoxia-inducible factors individually facilitate inflammatory myeloid metabolism and inefficient cardiac repair. Lee, Y. S. et al. Nobiletin improves obesity and insulin resistance in high-fat diet-induced obese mice. The benzenesulfoamide T0901317 [N-(2,2,2-trifluoroethyl)-N-[4-[2,2,2-trifluoro-1-hydroxy-1-(trifluoromethyl)ethyl]phenyl]-benzenesulfonamide] is a novel retinoic acid receptor-related orphan receptor-α/γ inverse agonist. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Yova Benjamini, Y. H. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Liao, Y., Wang, J., Jaehnig, E. J., Shi, Z. & Zhang, B. WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs. Han, H. et al. TRRUST v2: an expanded reference database of human and mouse transcriptional regulatory interactions. Zhou, K. R. et al. ChIPBase v2.0: decoding transcriptional regulatory networks of non-coding RNAs and protein-coding genes from ChIP-seq data. The STRING database in 2017: quality-controlled protein-protein association networks, made broadly accessible. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Bray, N. L., Pimentel, H., Melsted, P. & Pachter, L. Near-optimal probabilistic RNA-seq quantification. & Robinson, M. D. Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. The Gene Ontology knowledgebase in 2023. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Mondillo, S. et al. Speckle‐tracking echocardiography: a new technique for assessing myocardial function. Bauer, M. et al. Echocardiographic speckle-tracking based strain imaging for rapid cardiovascular phenotyping in mice. Heimdal, A., Støylen, A., Torp, H. & Skjærpe, T. Real-time strain rate imaging of the left ventricle by ultrasound. Usefulness of tissue Doppler velocity and strain dyssynchrony for predicting left ventricular reverse remodeling response after cardiac resynchronization therapy. Mukhopadhyay, P. et al. CB1 cannabinoid receptors promote oxidative stress and cell death in murine models of doxorubicin-induced cardiomyopathy and in human cardiomyocytes. Balsalobre, A. et al. Resetting of circadian time in peripheral tissues by glucocorticoid signaling. Almeida, M., Han, L., Ambrogini, E., Weinstein, R. S. & Manolagas, S. C. Glucocorticoids and tumor necrosis factor α increase oxidative stress and suppress Wnt protein signaling in osteoblasts. Ayroldi, E. et al. Mechanisms of the anti‐inflammatory effects of glucocorticoids: genomic and nongenomic interference with MAPK signaling pathways. Ye, R., Selby, C. P., Ozturk, N., Annayev, Y. Hu, C. J., Sataur, A., Wang, L., Chen, H. & Simon, M. C. The N-terminal transactivation domain confers target gene specificity of hypoxia-inducible factors HIF-1α and HIF-2α. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Zhang, K. Gctf: real-time CTF determination and correction. Wagner, T. et al. SPHIRE-crYOLO is a fast and accurate fully automated particle picker for cryo-EM. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Scheres, S. H. RELION: implementation of a Bayesian approach to cryo-EM structure determination. Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. Outcome of the first electron microscopy validation task force meeting. Addressing preferred specimen orientation in single-particle cryo-EM through tilting. Measuring the effects of particle orientation to improve the efficiency of electron cryomicroscopy. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. The PyMOL molecular graphics system, version 1.8 (Schrödinger, 2015). We thank the members of the Structural Biology Imaging Center at UTHealth McGovern Medical School, supported by the CPRIT Core Facility Award RP190602, and the Academia Sinica Cryo-EM facility for their assistance with cryo-EM data collection; and the staff at the Small Animal Cardiovascular Phenotyping Core of the UTHealth McGovern Medical School for help with STE measurement. is supported by National Institutes of Health Grants R35HL177402, R01HL154720, R01HL165748, R01HL119837 and T32GM135118; K.-L.T. by National Institutes of Health Grants R01GM143587 and R01HL165748; W.R. by National Institutes of Health Grant R01HL165748, 2022 International Anaesthesia Research Society Mentored Research Award 984307, National Natural Science Foundation of China 82470305 and the Science and Technology Innovation Programs of Hunan Province 2021RC3034; J. Lee by National Center for General Medical Sciences T32GM086287; X.M. by National Institutes of Health Grant R35GM145232; and X.Y. by National Institutes of Health Grants R01HL155950, R01HL155950-02S1, R01HL169519 and Parker B. Francis Fellowship. Z. Zhou is partially supported by the National Institutes of Health Grant R01LM012806. are partially supported by Cancer Prevention and Research Institute of Texas (CPRIT) grants RP180734 and RP240610. is supported by National Institutes of Health Grants R01HL118266 and R01HL150401. is supported by the National Institutes of Health Grant R01GM144836. Figures 1j, 4a,j, 5j, Extended Data Fig. We acknowledge BioRender.com for providing the illustration tools. These authors contributed equally: Wei Ruan, Tao Li, In Hyuk Bang, Jaewoong Lee Department of Anesthesiology, Critical Care and Pain Medicine, The University of Texas Health Science Center at Houston, McGovern Medical School, Houston, TX, USA Wei Ruan, In Hyuk Bang, Xinxin Ma, Cong Luo, Fang Du, Boyun Kim, Jiwen Li, Xiaoyi Yuan, Katherine Figarella, Yu A. An, Yafen Liang, Matthew DeBerge, Dongze Zhang, Yanyu Wang, Ragini Nair & Holger K. Eltzschig Department of Anesthesiology, Second Xiangya Hospital, Central South University, Changsha, China Department of Biochemistry and Molecular Biology, The University of Texas Health Science Center at Houston, McGovern Medical School, Houston, TX, USA Tao Li, Seung-Hee Yoo & Kuang-Lei Tsai Department of Anesthesiology, Yale University School of Medicine, New Haven, CT, USA Center for Precision Health, McWilliams School of Biomedical Informatics, The University of Texas Health Science Center at Houston, Houston, TX, USA Wankun Deng, Yin-Ying Wang & Zhongming Zhao Department of Anesthesiology, Zhongshan Hospital, Fudan University, Shanghai, China Major in Aquaculture and Applied Life Sciences, College of Fisheries Science, Pukyong National University, Busan, Republic of Korea Department of Cardiac Surgery, Sir Run Run Shaw Hospital, School of Medicine, Zhejiang University, Hangzhou, China Division of Medical Genetics, Department of Internal Medicine, The University of Texas Health Science Center at Houston, McGovern Medical School, Houston, TX, USA Department of Genetics, Harvard Medical School, Boston, MA, USA Department of Surgery, Division of Cardiac Surgery, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA Division of Cardiology, Department of Medicine, The University of Texas Health Science Center at Houston, McGovern Medical School, Memorial Hermann Hospital, Houston, TX, USA Department of Integrative Biology and Pharmacology, The University of Texas Health Science Center at Houston, McGovern Medical School, Houston, TX, USA Department of Anesthesiology and Critical Care Medicine, Johns Hopkins University School of Medicine, Baltimore, MD, USA MD Anderson Cancer Center, UTHealth Houston Graduate School of Biomedical Sciences, Houston, TX, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceptualized and designed the study. and J. Li helped with the animal surgeries. helped with the RNA-seq and microarray data analysis. participated in data analysis and interpretation. reviewed the raw and source data. Correspondence to Wei Ruan, Kuang-Lei Tsai or Holger K. Eltzschig. The authors declare no competing interests. Nature thanks Gary Lopaschuk and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a-d, RNA-seq analysis of the AAR following 2 h of reperfusion in C57BL/6J mice subjected to IRI at ZT8 or ZT20. a, PCA of mRNA expression profiles. b, Differentially expressed genes (DEGs) identified between ZT8 and ZT20 (fold change > 1.5, adjusted p < 0.05); Wald test. c, Top five enriched KEGG pathways for DEGs; two-sided Fisher's exact test. d, Gene dysregulation network showing transcription factors (triangles) and genes (circles), with red and green nodes representing upregulated and downregulated DEGs, respectively, in ZT8 hearts relative to ZT20. Node size indicates degree, and edge width reflects correlation strength. e, Volcano plot of DEGs in pre-clamping LV biopsies from cardiac surgical patients grouped by surgery time (morning vs. afternoon); Wald test. f-h, Human RNA-seq analysis of post-clamping LV biopsies comparing morning (AM) and afternoon (PM) groups. f, PCA showing distinct transcriptional signatures between AM and PM groups. g, DEGs identified between AM and PM groups (log2 fold change > 0.5, p < 0.01); Wald test. h, Top three enriched KEGG pathways for DEGs, with node colour indicating p-value and size representing fold change; two-sided Fisher's exact test. a, GO enrichment analysis was conducted to elucidate the molecular functions (MF), biological processes (BP), and cellular components (CC) associated with proteins predicted by the HuRI to potentially interact with BMAL1; two-sided Fisher's exact test, Benjamini–Hochberg correction applied. b, Western Blot analysis of reciprocal co-IP with HIF2A in hypoxia-treated (1% O2, 4 h) or normoxia-treated HEK293 cells. An IgG control affirmed procedure specificity. c, Size-exclusion chromatography analysis of the BMAL1/HIF2A heterodimer. The purified BMAL1/HIF2A complex was loaded onto a Superdex 200 Increase 10/300 GL column. The molecular weights of makers are as indicated. d, SDS-PAGE analysis of the recombinant BMAL1/HIF2A heterodimer purified by size-exclusion chromatography. a, Representative trichrome staining of heart sections on day 1 post‐MI. Border and remote areas are identified with an arrow and a star, respectively. b and d, Representative immunostaining of HIF2A (red) (b), BMAL1 (red) (d), WGA (green), and nuclei (DAPI; blue) in the border zone and remote area of hearts from C57BL/6J mice subjected to myocardial IRI at ZT8 or ZT20, n = 4 mice/group/time point, scale bar, 20 μm. White arrows indicate HIF2A or BMAL1 within the nuclei and cytoplasm. c and e, Quantification of fluorescence intensity of HIF2A in (b) and BMAL1 in (d). Each quantification value dot represents the average value of three fields in one section. Statistical analysis was performed using two-way ANOVA. a-b, a, Representative immunostaining of AREG (red), α-sarcomeric (cardiomyocyte marker; green), and nuclei (DAPI; blue) on day 1 post-MI in the border zone, infarct area, and remote area of hearts from C57BL/6J mice subjected to myocardial IRI at ZT8 or ZT20, scale bar, 25 μm. Each quantification value dot represents the average value of three fields in one section. Statistical analysis was performed using two-way ANOVA. c-d, Representative immunostaining of AREG (red), vimentin (fibroblast marker; green) (c), α-smooth muscle actin (α-SMA, smooth muscle cell marker; green) (d) and nuclei (DAPI; blue) on day 1 post-MI in the border zone of hearts from C57BL/6J mice subjected to myocardial IRI at ZT8 or ZT20, scale bar, 25 μm. a, Experimental setup for evaluating myocardial injury and cardiac function in NOB-treated (200 mg/kg, i.p., every other day) or Veh-treated C57BL/6J mice subjected to myocardial IRI at ZT8 or ZT20. b-d, BMAL1, RORα, and AREG transcript levels in the AAR after 2 h of reperfusion (b) and protein levels by Western blot (c) with quantification (d). Due to similar molecular weight of proteins, some samples were run on separate gels with equal loading volumes, using α-tubulin as sample processing control. e, BMAL1 immunofluorescence (red) in cell nuclei (DAPI: blue) in the border zone on day 1 post-IRI, with quantification. f, AREG immunofluorescence (red), α-sarcomeric (green), and nuclei (DAPI: blue) in the border zone, with quantification. g–j, Evan's blue and TTC-stained heart slices (g), AAR as % LV (h), infarct size as % AAR (i), and serum troponin I levels (j) after 2 h of reperfusion. k and l, Cardiac function on day 14 post-MI assessed by STE. EF, FS, and GLS (k) and 3D longitudinal strain with six-segment images showing LV motion abnormalities (l). Colour-coded segments: Anterior Base (dark blue), Anterior Mid (yellow), Anterior Apex (magenta), Posterior Apex (cyan), Posterior Mid (light pink), and Posterior Base (green). Symbols indicate reduced contractility (stars), dyskinesis (triangles), and dyssynchrony (circles). m and n, TUNEL staining on day 1 post-MI (m) and quantification of TUNEL-positive cardiomyocytes (n); scale bar, 25 μm. The diagram in a was created using BioRender. a-d, C57BL/6J mice treated with vadadustat (vada, 50 mg/kg, i.p., daily) or Vehicle (Veh) for three days at ZT8 or ZT20. Immunostaining of HIF2A (red), WGA (green), and nuclei (DAPI, blue) in mouse hearts (c; scale bar: 25 μm), with quantification of HIF2A fluorescence intensity (d). e-g, Mice given an additional dose of vadadustat 2 h before IRI at ZT8 or ZT20. Evan's blue and TTC-stained heart slices (e), quantification of AAR as % LV (f), and infarct size as % AAR (g) after 2 h of reperfusion. h-n, Cardiac function on day 14 post-MI assessed by STE. LV systolic function (EF, FS, GLS; h), ESV (i), EDLVM (j), B-mode imaging with 2D longitudinal strain (k), segmental wall contractility (l), 3D longitudinal strain with images showing LV motion abnormalities (m), and mechanical dyssynchrony measured by intra-ventricular delay (n). Symbols in (m) indicate reduced contractility (stars), dyskinesis (triangles), and dyssynchrony (circles). o and p, Mice treated with vadadustat for three days at ZT8 or ZT20, followed by IRI. Nuclear fractions from the AAR 3 h post-reperfusion were immunoprecipitated with HIF2A and blotted for HIF2A, BMAL1, and Lamin B (o) with quantification of BMAL1 and HIF2A protein levels (p). q and r, Immunostaining of AREG (red), α-sarcomeric (green), and nuclei (DAPI: blue) 3 h post-reperfusion (q, scale bar, 25 μm), with quantification of fluorescence intensity (r). a and b, HIF2A transcript levels from RNA-seq of LV biopsy samples after aortic cross-clamping in morning and afternoon cardiac surgery patients (a, n = 56 morning, n = 17 afternoon; boxplots show the 25th and 75th percentiles (box), median (central line), and minimum to maximum values (whiskers)) and in the AAR of mice after 2 h of reperfusion at ZT8 or ZT20 (b, n = 3 mice/time point; unpaired two-tailed t-tests). c, Hif2a transcript levels by real-time PCR after 2 h of reperfusion in the AAR subjected to IRI at ZT8 or ZT20. n = 3 mice/time point; unpaired two-tailed t-tests. d-f, Hif2a transcript levels by real-time PCR in the AAR after 2 h of reperfusion in Bmal1loxP/loxP Myosin Cre+ mice and Myosin Cre+ mice (d, n = 5/Myosin Cre+ and n = 3/Bmal1loxP/loxP Myosin Cre + ; unpaired two-tailed t-tests). HIF2A protein levels by Western blot in the nuclear fractions (e, n = 3/Myosin Cre+ and n = 5/Bmal1loxP/loxP Myosin Cre + ) and quantification (f, Welch's t-tests). g-k, C57BL/6J mice treated with NOB (200 mg/kg, i.p.) Immunostaining of HIF2A in the myocardium (j, n = 4 mice/group/time point, scale bar, 25 μm) and quantification (k, two-way ANOVA). l-q, HCMs transduced with Bmal1-AAV or shBmal1-AAV were treated with 1% O2 for 4 h. BMAL1 and HIF2A transcript levels by real-time PCR (l, o), protein levels by Western blot (m, p), and quantification (n, q). n = 3 independent experiments; Unpaired two-tailed t-tests were used for all comparisons, except for BMAL1 protein levels in (n), which were analysed using Welch's t-tests. The samples used in (m) were also used in Extended Data Fig. The samples used in (p) were also used in Extended Data Fig. r, ChIP-qPCR of BMAL1 binding to E-box elements in the human HIF2A promoter under normoxic and hypoxic conditions. n = 4 independent experiments; one-way ANOVA. s-v, HCMs transduced with Bmal1-AAV (s, t) or shBmal1-AAV (u, v) were exposed to 1% O2 for 4 h and treated with CHX to assess HIF2A degradation. Protein levels were quantified and plotted to determine half-life using exponential decay. w, x, HEK293 cells transfected with HIF2A-Myc, Ub-HA, and BMAL1-Flag, treated with MG132, and subjected to HIF2A immunoprecipitation. Ubiquitination levels were analysed (w) and quantified (x). n = 3 independent experiments; one-way ANOVA. a and b, RNA-seq analysis of HIF1B transcript levels in LV biopsies after aortic cross-clamping in morning vs. afternoon patients (a, n = 56 AM, n = 17 PM; boxplots show the 25th and 75th percentiles (box), median (central line), and minimum to maximum values (whiskers)) and in the AAR after 2 h of reperfusion at ZT8 or ZT20 in mice (b, n = 3 mice/time point; unpaired two-tailed t-tests). c-e, Hif1b transcript levels in the AAR on day 1 post-IRI in Bmal1loxP/loxP Myosin Cre+ and Myosin Cre+ mice (c, n = 5 Myosin Cre+ and n = 3 Bmal1loxP/loxP Myosin Cre+ mice; unpaired two-tailed t-tests), HIF1B protein levels by Western blot (d, n = 4 mice/group) and quantification (e, unpaired two-tailed t-tests). f-h, Hif1b transcript levels in mouse hearts 28 days post-Bmal1-AAV or control AAV injection (f, n = 3 mice/group; unpaired two-tailed t-tests), HIF1B protein levels by Western blot (g, n = 3 mice/group) and quantification (h, unpaired two-tailed t-tests). i-k, HIF1B transcript levels in HCMs transduced with Bmal1-AAV or control AAV under hypoxia (i, n = 3 independent experiments; unpaired two-tailed t-tests), HIF1B protein levels by Western blot (j, n = 3 independent experiments), and quantification (k; unpaired two-tailed t-tests). The same samples used in (j) were also used in Extended Data Fig. l-n, HIF1B transcript levels in HCMs transduced with shBmal1-AAV or shControl AAV (l, n = 3 independent experiments; unpaired two-tailed t-tests), HIF1B protein levels by Western blot (m, n = 3 independent experiments), and protein quantification (n; unpaired two-tailed t-tests). o and p, Immunoprecipitation using HIF1B and HIF2A antibodies in HEK293 cells transfected with BMAL1-Flag or control plasmid under hypoxia or normoxia, analysed by Western blot (o, n = 3 independent experiments) and normalized to α-tubulin (p, one-way ANOVA). q and r, ChIP-qPCR using HIF1B antibody showing binding to the human EPO (q) and AREG (r) promoters under normoxia and hypoxia. n = 8 independent experiments; two-way ANOVA. s, Luciferase assays in HEK293 cells transfected with BMAL1-Flag (0–200 nM), HIF2A-HA, and a luciferase reporter for the human PGK1 promoter. n = 4 independent experiments; one-way ANOVA. b, Representative 2D class averages of the BMAL1/HIF2A/DNA complex. d, FSC curves of the BMAL1/HIF2A/DNA complex. e, Cryo-EM density map of the BMAL1/HIF2A/DNA complex coloured by local resolution. g, Density map of the BMAL1/HIF2A/DNA complex generated by deepEMhancer. h, Histogram and directional FSC plot generated by 3DFSC indicates that 90% of directions achieve a resolution better than 4 Å. i and j, Local structures with their corresponding densities are shown. a, Four domain interfaces (I to IV) between BMAL1 and HIF2A. Each of these is indicated by a dashed ellipse. b, Zoom-in views of the interfaces (I to IV) between HIF2A and BMAL1. c, Comparison of the DNA-binding by two bHLH domains in HIF1B/HIF2A (left, PDB ID 4ZPK), BMAL1/HIF2A (middle), and BMAL1/CLOCK (right, PDB ID 4H10). The PAS domains are omitted for clarity. Upper: Structural comparison of BMAL1/HIF2A with BMAL1/CLOCK and HIF1B/HIF2A by aligning bHLH domains (highlighted in blue dashed frames). For clarity, DNA was omitted. Middle: Structure of BMAL1/CLOCK with its BMAL1 PAS-A domain (dashed circle) aligned to that of BMAL1/HIF2A. The conserved residues in the HI loop and their interacting residues are shown. f, BMAL1 undergoes structural rearrangements upon binding with various partners. Superimposing the BMAL1/HIF2A and BMAL1/CLOCK complexes by aligning their bHLH domains reveals that BMAL1 (red) undergoes a substantial conformational change, with the two PAS domains bending in nearly opposite directions. BMAL1 exhibits a compact overall architecture when bound with CLOCK (green) and a distinctly separated conformation when interacting with HIF2A (purple). Source data for Supplementary Tables 1–9. Structural rearrangement of BMAL1 after binding with various partners. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. BMAL1–HIF2A heterodimer modulates circadian variations of myocardial injury. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08922-2'>Human de novo mutation rates from a four-generation pedigree reference</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 15:06:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Understanding the human de novo mutation (DNM) rate requires complete sequence information1. Here using five complementary short-read and long-read sequencing technologies, we phased and assembled more than 95% of each diploid human genome in a four-generation, twenty-eight-member family (CEPH 1463). We estimate 98–206 DNMs per transmission, including 74.5 de novo single-nucleotide variants, 7.4 non-tandem repeat indels, 65.3 de novo indels or structural variants originating from tandem repeats, and 4.4 centromeric DNMs. Among male individuals, we find 12.4 de novo Y chromosome events per generation. Short tandem repeats and variable-number tandem repeats are the most mutable, with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length and sequence identity. We show a strong paternal bias (75–81%) for all forms of germline DNM, yet we estimate that 16% of de novo single-nucleotide variants are postzygotic in origin with no paternal bias, including early germline mosaic mutations. We place all this variation in the context of a high-resolution recombination map (~3.4 kb breakpoint resolution) and find no correlation between meiotic crossover and de novo structural variants. These near-telomere-to-telomere familial genomes provide a truth set to understand the most fundamental processes underlying human genetic variation. The telomere-to-telomere (T2T) assembly of a human genome1 added an estimated 8% of the most repeat-rich DNA, including regions typically excluded from studies of human genetic variation, such as centromeres2, segmental duplications (SDs)3 and acrocentric regions1,4. The goal of this study was to construct a high-quality human pedigree resource whereby chromosomes were fully assembled and phased, and their transmission was studied intergenerationally to enhance our understanding of both recombination and DNM processes. We sought to eliminate three ascertainment biases with respect to discovery, including biases to specific genomic regions, classes of genetic variation and reference genome effects. To achieve this, we focused on the four-generation, 28-member family CEPH 1463. This family has been intensively studied over the past three decades11, and we sequenced the family members using five sequencing technologies with distinct and complementary error modalities. This particular pedigree has served as a benchmark for early linkage mapping studies11,12 using short-read sequencing (SRS)13 and continues to serve as reference for understanding human variation, including patterns of mosaicism14,15. Just as the initial T2T genome1 served as a reference for understanding all regions of the genome, our objective was to create a reference truth set for both inherited and de novo variation. We generated PacBio high-fidelity (HiFi), ultra-long Oxford Nanopore Technologies (UL-ONT), Strand-seq, Illumina and Element AVITI Biosciences (Element) whole-genome sequencing (WGS) data for most of the 28 members from a four-generation family (CEPH 1463 pedigree) (Fig. Twenty-eight members of the four-generation pedigree CEPH 1463 were sequenced using five orthogonal next-generation and LRS platforms: HiFi sequencing, Illumina and Element sequencing were performed on peripheral blood for G2–G4, and UL-ONT and Strand-seq data were generated on available lymphoblastoid cell lines for G1–G3. For the purpose of variant discovery, we focused on generating long-read PacBio, short-read Illumina and Element data from blood-derived DNA to avoid cell-line-specific artefacts. We also used the corresponding cell lines to generate UL-ONT reads to construct near-T2T assemblies as well as Strand-seq data to detect large polymorphic inversions and evaluate assembly accuracy (Methods and Supplementary Table 2). In brief, we generated deep WGS data from multiple orthogonal sequencing platforms, focusing primarily on the first three generations (G1–G3) (Extended Data Fig. 1a), and used the fourth generation (G4) to validate de novo germline variants. We applied two hybrid genome assembly pipelines, Verkko16 and hifiasm17, to generate highly contiguous, phased genome assemblies for G1–G3, while G4 members were assembled using HiFi data only (Methods). In summary, Verkko assemblies are the most contiguous (AuN (similar to average contig length measure): 102 Mb) (Extended Data Fig. Moreover, 42.3% (213 out of 504) of non-acrocentric chromosomes are spanned in a single contig with canonical telomere repeats at each end (Methods, Extended Data Fig. We sequenced and assembled 288 centromeres (44.7%, 288 out of 644) across G1–G3 and note that different assemblers preferentially assembled different human centromeres (Methods, Extended Data Fig. Both the sequence (QV range, 47–58) and phasing accuracy are high (Methods, Supplementary Figs. This data resource enables us to track the inheritance of any genomic segment and associated variants across all four generations (Extended Data Fig. We identified a total of 5.95 million single-nucleotide variants (SNVs) and indels and 35,662 structural variants (SVs)—all of which are Mendelian consistent across the second and third generations (Methods, Supplementary Table 5, Supplementary Fig. Of the 5.95 million, 77% of small variants are supported by all three technologies, with variant calling from primary material helping to eliminate DNMs arising from cell line artefacts (Supplementary Note 1). LRS provides access to an additional approximately 260 Mb of the human genome (2.77 Gb) in contrast to the Genome in a Bottle (GIAB) (2.51 Gb)19 or Illumina WGS (2.58 Gb)13 data, including 201 Mb not present in either study. Some of the largest gains occur among SDs and their associated genes. We classified 85.5% (6,883 out of 8,048 merged SDs) of the SDs (coverage, >95%) as high confidence in comparison to 25.6% (2,060 out of 8,048 merged SDs) in the previous GIAB analysis, a major improvement for these highly copy-number variable regions20. We find that the majority (>91%) of known copy-number variable regions were stably transmitted in this pedigree, while the remaining 9% were often flagged as potentially misassembled (Supplementary Note 2). Similarly, we provide a comprehensive census of mobile element insertions (Methods, Supplementary Table 6, Supplementary Fig. The latter includes a rare inversion (~703 kb) overlapping a disease-associated copy-number variable region at chromosome 15q25.2 (ref. 19) and an inverted duplication (~295 kb) at chromosome 16q11.2 (Supplementary Fig. Using three different approaches13,22 (Methods and Extended Data Fig. 2b), we identify 539 meiotic breakpoints in G3 (n = 8) with respect to T2T-CHM13, with 99.8% (538 out of 539) supported by more than one approach (Supplementary Table 8 and Supplementary Fig. From an initial resolution of around 3.4 kb, we further refined 90.4% (487 out of 539) of the breakpoints to a median size of about 2.5 kb based on direct genome comparisons between parent and a child (Methods and Supplementary Fig. Notably, 191 breakpoints actually increase in size as a result of reference biases in T2T-CHM13 (Supplementary Fig. We distinguish recombination breakpoints with very sharp transition between parental haplotypes from those with an extended region of homology at both parental haplotypes (Extended Data Fig. We also characterize 78 smaller haplotype segment ‘switches' in G3 (median size of ~1 kb)23,24,25 that would be consistent with either a double crossover or an allelic gene conversion event, although this is probably an underestimate due to our strict filtering criteria (Methods, Supplementary Table 9 and Supplementary Fig. Extending recombination mapping to G4 chromosomes, we add 964 breakpoints for a total of 1,503 meiotic breakpoints across 22 transmissions (Supplementary Fig. This includes 16 recombination hotspots, 11 of which are consistent with previously reported increased recombination rates26 (Supplementary Table 8 and Supplementary Fig. Overall, 15–20% of paternal and maternal homologues are transmitted without a detectable meiotic breakpoint (that is, non-recombinant chromosomes) (Supplementary Fig. We observe a significant excess (Wilcoxon signed-rank test, P = 6.4 × 10−5) of maternal recombination events with expected maternal to paternal breakpoint ratio of 1.4 (ref. Paternal recombination is significantly biased towards the ends of human chromosomes with 55 paternal recombination events mapping to within 2 Mb of the telomere in comparison to 1 event in female individuals27,28,29 (Methods, Extended Data Fig. In G2–G3, we observed a decrease in crossover events with advancing parental age for both male and female germlines (Extended Data Fig. We modelled this observation across G1–G4 using a Poisson generalized linear model (GLM) with a log link and continued to observe a significant decrease in recombination breakpoints as a function of parental age and sex (P = 7.17 × 10−3 and 1.22 × 10−9 for parental age and sex, respectively; Poisson GLM with a log link, AIC = 284.2) (Supplementary Fig. Although there is no known biological mechanism that would lead to a decrease in both parental germlines, this observation runs counter to a population-level analysis based on SRS data25,30,31. We consider this observation to be preliminary until a larger number of families is analysed. To discover small variants, we examined HiFi reads aligned to T2T-CHM13, then used orthogonal ONT and Illumina data to confirm that a variant is in fact present in a sample and absent from parents (Methods). This strategy reduces platform bias but restricts DNM discovery to G2 (n = 2) and G3 (n = 8) individuals, as ONT data were not generated for G4. Our de novo callset included 755 SNVs and 73 indels across the autosomes (Fig. We used flanking SNVs to construct haplotypes, phase variants and trace a mutation back either to a parental gamete or the early embryo. We determined that a mutation occurred somatically, and probably early in embryonic development, if it met one of two criteria: it was incompletely linked to a parental haplotype (n = 122) or, if it could not be phased, it had an allele balance significantly less than 0.5 across all three sequencing platforms (n = 7) (Fig. 2b), which was further confirmed using Element data (Supplementary Fig. Moreover, we validated each postzygotic mutation (PZM) by tracing its haplotype backwards across generations and forwards for the four individuals with sequenced offspring (Supplementary Note 4). a, The number of de novo germline mutations, PZMs and indels (<50 bp) for the parents (G2) and eight children in CEPH 1463. TR DNMs (<50 bp) are shown for G3 only because they have greater parental sequencing depth and we can assess transmission (Methods). b, Germline SNVs (n = 626) have a mean allele balance of near 0.50 across the sequencing platforms, while the mean postzygotic SNV (n = 119) allele balance is less than 0.25. c, A strong paternal age effect is observed for germline de novo SNVs (+1.55 DNMs per year; two-sided t-test, P = 0.013) but not for PZMs (P = 0.72). We observe no significant maternal age effect for DNMs (+0.20 DNMs per year, P = 0.54) or PZMs (P = 0.74). The solid lines are regression lines that were fitted using a linear model function; the surrounding shaded areas represent their 95% confidence intervals. d, The estimated SNV DNM rate by region of the genome shows a significant excess of DNM for large repeat regions, including centromeres and SDs. Assembly-based DNM calls on the centromeres and Y chromosome (chr.) show an excess of DNM in the satellite DNA. A significant difference from the autosomal DNM or PZM rate was determined using two-sided t-tests; *P < 0.05, **P < 0.001. We found that 10 PZMs failed these haplotype-based validations, resulting in a final callset of 119 PZMs, accounting for 16% of total autosomal SNVs (745 de novo SNVs). Previous Illumina-based analysis of this family14 identified 605 de novo SNVs of either germline (G2 and G3) or postzygotic (only G2) origin, 92.4% (n = 559) of which were represented in our final callset, while all but four of the absent variants failed validation with long-read data. We were able to identify an additional 72 PZMs in G3 for the first time, including a total of 186 novel DNMs, a 6.1% and 21% increase in germline SNV and indel discovery, respectively. In total, 81.4% of germline small DNMs originate on paternal haplotypes (4.38:1 paternal:maternal ratio, Wilcoxon signed-rank test, P < 2 × 10−16), with a significant parental age effect of 1.55 germline DNMs per additional year of paternal age when fitting with linear regression (two-sided t-test, P = 0.013). By contrast, PZMs show no significant difference with respect to parental origin (1.38:1 paternal:maternal ratio, Wilcoxon signed-rank test, P = 0.09) and no parental age effects (Fig. Although our small sample size does not provide sufficient power to detect significant differences between the de novo and postzygotic mutational spectra (Supplementary Fig. 33a), we do observe a novel depletion of CpG>TpG PZMs (χ2 test, P = 0.17) and an enrichment of postzygotic T>A substitutions (χ2 test, P = 0.268) that has been previously observed14. We successfully assayed 91.9% of the autosomal genome (2.66 Gb) (Supplementary Fig. Excluding all variants classified as postzygotic, we find that the parental germline contributes 1.17 × 10−8 SNVs per bp per generation (95% confidence interval (CI) = 1.08 × 10−8–1.27 × 10−8). De novo SNVs are significantly enriched in repetitive sequences, as much as 2.8-fold in centromeres (95% CI = 1.79 × 10−8–5.51 × 10−8 SNVs per bp per generation, two-sided t-test, P = 0.017) and 1.9-fold in SDs (95% CI = 1.64 × 10−8–2.88 × 10−8 SNVs per bp per generation, two-sided t-test, P = 0.0066) (Fig. We observed a lower PZM rate of 2.04 × 10−9 SNVs per bp per generation (95% CI = 1.68 × 10−9–2.47 × 10−9) across the autosomes, yet we see a 3.9-fold enrichment of PZMs in SDs (95% CI = 4.84 × 10−9–1.25 × 10−8 SNVs per bp per generation, two-sided t-test, P = 0.049). Among PZMs transmitted to the next generation (n = 33 PZMs across four samples), we observe a 2.69-fold enrichment in SDs (95% CI = 1.15 × 10−9–1.08 × 10−8 SNVs per bp per generation) that does not reach significance owing to the small sample size (two-sided t-test, P = 0.218). We successfully genotyped 7.68 million out of 7.82 million TR loci (Methods) on HiFi data using the Tandem Repeat Genotyping Tool (TRGT)32, across all members of the pedigree. Of those, 7.17 million (93.4%) loci were completely Mendelian concordant across all trios. We used TRGT-denovo to identify candidate DNMs at loci that were covered by at least 10 HiFi reads across all members of a given trio; on average, 6.88 million TR loci met this criterion33. We refined these putative DNMs through orthogonal sequencing and transmission (Methods). Element sequencing, generated from blood DNA, exhibits substantially lower error rates following homopolymer tracts34, so we tested whether it could more accurately measure the length of homopolymers and other TR alleles. We observed low stutter in the Element data at homopolymers; across a random sample of 1,000 homozygous homopolymer loci called by TRGT, an average of 99.5% of Element reads perfectly support the TRGT-genotyped allele size in GRCh38, compared to 93.5% of Illumina sequencing reads (Supplementary Figs. We used the Element data to further validate de novo TR alleles called by TRGT-denovo. We considered a DNM validated if Element reads supported the TRGT allele size in the child and did not support it in either parent (allowing for off-by-one base-pair errors; Methods). Of the 80 de novo STRs that we could assess, 56 (70%) passed our strict consistency criteria. The validation rate was lower at homopolymers (3 out of 20; 15%) than at non-homopolymers (53 out of 60; 88.3%), indicating that our estimates of mutation rates at homopolymers may be less precise. Using pedigree information, we required that candidate de novo TR alleles observed in the two G3 individuals with sequenced children (NA12879 and NA12886) be transmitted to at least one child in the subsequent generation (G4). Of the 128 de novo TR alleles observed in the two G3 individuals, 96 (75%) were transmitted to the next generation, which is significantly lower than de novo SNVs reflecting the challenges that still remain in accurately characterizing de novo TRs. After Element and transmission validation, we found an average of 65.3 TR DNMs (including STRs, VNTRs and complex loci) per sample and estimated a TR DNM rate of 4.74 × 10−6 per locus per haplotype per generation (95% CI = 4.06 × 10−6–5.43 × 10−6), with substantial variation across repeat motif sizes (Fig. Collectively, TR DNMs inserted or deleted a mean of 978 bp per sample or 15.0 bp per event (Supplementary Table 10). An average of 54.9 mutations were expansions or contractions of STR motifs, 2.6 affected VNTR motifs and 7.8 affected ‘complex' loci comprising both STR and VNTR motifs. The VNTR mutation rate was 0.83 × 10−6 (95% CI = 0.51 × 10−6–1.27 × 10−6), predominantly comprising loci that could not be assessed in SRS studies. Several previous estimates of the genome-wide STR mutation rate considered only polymorphic STR loci; when we limited our analysis to STR loci that were polymorphic in the CEPH 1463 pedigree, we found 5.98 × 10−5 de novo STR events per locus per generation (95% CI = 5.43–6.57 × 10−5), which is broadly consistent with previous estimates of 4.95 × 10−5–5.6 × 10−5 (refs. Overall, 75.0% of phased de novo TR alleles were paternal in origin (Fig. The mutation rate for dinucleotide motifs was higher than for homopolymers, and we observed an increasing mutation rate with motif size for motifs greater than 6 bp in length (Fig. As reported in previous studies35, larger TR loci (defined as the total length of the TR locus in the reference genome sequence) exhibited higher mutation rates (Supplementary Fig. We did not observe a significant bias towards expansions or contractions (two-sided binomial test, P = 0.19) (Supplementary Fig. a, TR DNM rates (mutations per haplotype per locus per generation) are displayed for each TR class (STR, VNTR or complex) as a function of the minimum motif size observed at each TR locus (n = 522) in the T2T-CHM13 reference genome (blue; left y axis).The average number of loci of each motif size that passed filtering criteria in each individual are displayed in grey (right y axis). The mutation rates include all non-recurrent calls that pass TRGT-denovo filtering criteria and Element consistency analysis. b, The inferred parent-of-origin for confidently phased TR DNMs in G3. The hatching indicates transmission to at least one G4 child, where available. c, Pedigree overview of a recurrent VNTR locus at chromosome 8: 2376919–2377075 (T2T-CHM13) with motif composition GAGGCGCCAGGAGAGAGCGCT(n)ACGGG(n). Allele colouring indicates inheritance patterns as determined by inheritance vectors, with grey representing unavailable data. The symbols denote inheritance type relative to the inherited parental allele: plus (+) for de novo expansion and minus (−) for de novo contraction, shown only for the mutating alleles; the numbers indicate allele lengths in bp. De novo TR alleles are present in seven out of eight G3 individuals and transmit to four G4 individuals, with two expanding further after transmission. d, Read-level evidence for the recurrent DNM in c, represented as vertical lines, obtained from individual sequencing reads, shown per sample. Where available, both HiFi (top) and ONT (bottom) sequencing reads are displayed. Colouring is consistent with the inheritance patterns in c; the outlined boxes with plus or minus markers highlight DNMs. We identified a subset of TR loci that were recurrently mutated among members of the pedigree. We identified a high-confidence set of 32 loci (Methods and Supplementary Table 11): five showing intragenerational recurrence (observed DNMs in at least two G3 individuals) and 27 loci with intergenerational recurrence (observed DNMs in at least two generations). As they are observed only in a single generation, the five intragenerational DNMs may represent mosaicism in the parental germline, rather than recurrent mutational events. Notably, we observed three or more distinct de novo expansions or contractions at 16 of the loci that exhibited recurrence (Extended Data Table 1). As an example, we highlight an intergenerational recurrently mutated TR locus with ten unique de novo expansions and/or contractions (Fig. All allele transmissions are fully consistent with the inheritance vectors (Supplementary Note 5) and are supported by both HiFi and ONT reads. Among the 288 completely assembled centromeres, we assessed 150 intergenerational transmissions (Fig. Comparing these assembled centromeres between parent and child, we identify 18 (12%) de novo SVs validated by both ONT and HiFi data with roughly equivalent numbers of insertions and deletions (Fig. All de novo SVs (n = 8) that had a child sequenced as part of this study confirmed transmission to the next generation (Supplementary Table 10). We find that 72.2% (13 out of 18) of SVs map to α-satellite higher-order repeat (HOR) arrays (Extended Data Fig. 4a) with the remainder (5 out of 18, 27.8%) corresponding to various pericentromeric flanking sequences but not flanking monomeric α-satellites. All α-satellite HOR de novo SV events involve integer changes in the basic α-satellite HOR cassettes specific to each centromere and range in size from 680 bp (one 4-mer α-satellite HOR on chromosome 9) to 12,228 bp (four 18-mer α-satellite HORs on chromosome 6) (Fig. One transmission from chromosome 9 involves both a gain of 2,052 bp (six dimer α-satellite HOR units) and a loss of 1,710 bp (one 4-mer α-satellite HOR and three α-satellite dimer units) in a single G2-to-G3 transmission (Fig. The chromosome 6 centromere has the most recurrent structural events, with three being observed across three generations (Fig. The chromosome 6 centromere has the greatest number of nearly perfectly identical (>99.9%) α-satellite HORs (Extended Data Fig. a, Summary of the number of correctly assembled centromeres (dark grey) as well as those transmitted to the next generation (light grey). Transmitted centromeres that carry a de novo deletion, insertion or both are coloured. b, The lengths of the de novo SVs within α-satellite HOR arrays and flanking regions. The deleted region is highlighted by a red outline. d, An example of a de novo insertion and deletion in the chromosome 19 α-satellite HOR array of G3-NA12885. We also assessed 18 SV events for their potential effect on the hypomethylation pocket associated with the centromere dip region (CDR)—a marker of the site of kinetochore attachment38,39 (Methods). We find that 11 SVs mapping outside of the CDR have a marginal effect on changing the centre point of the CDR (<100 kb) from one generation to another (Extended Data Fig. 4d,e), while SVs mapping within the CDR have a more marked effect (average shift of around 260 kb) and/or they completely alter the distribution of the CDR (Fig. Although follow-up experiments using CENP-A chromatin immunoprecipitation–sequencing are needed to confirm the actual binding site of the kinetochore, these findings suggest that structural mutations may have epigenetic consequences in changing the position of kinetochore. Finally, using 31 parent–child transmissions of centromeres (150.5 Mb), we identify 16 SNV DNMs in centromeres, including five within the α-satellite HOR arrays, for a DNM rate of 1.01 × 10−7 mutations per bp per generation (95% CI = 5.75 × 10−8–1.63 × 10−7). This rate is comparable to the rate from our read-based mapping approach, which identified 14 centromeric SNVs, albeit over more than 10 times the amount of sequence, resulting in a DNM rate of 3.27 × 10−8 mutations per bp per generation (95% CI = 1.79 × 10−8–5.51 × 10−8) (Fig. By combining the data, we estimate a significantly higher SNV DNM rate for centromeres of 4.94 × 10−8 (two-sided t-test, P = 0.017). We believe that this is a conservative estimate because we required validation of all events by both the ONT and HiFi sequencing platforms. There are nine male members who carry the R1b1a-Z302 Y haplogroup across the four generations (Fig. 1) Y-chromosome assembly as a reference for DNM detection across 48.8 Mb of the male-specific Y-chromosomal region (MSY) (Methods and Supplementary Note 6). The de novo assembly-based approach increases by more than twofold the number of accessible base pairs when compared to HiFi read-based calling but increases by more than sevenfold the discovery of de novo SNVs. In total, we identify 48 de novo SNVs in the MSY across the 5 G2–G3 male individuals, ranging from 7 to 11 SNVs per Y transmission (mean, 9.6; median, 10) (Supplementary Table 13). In total, we estimate a de novo SNV rate of 1.99 × 10−7 mutations per bp per generation (95% CI = 1.59 × 10−7–2.39 × 10−7) for the entire MSY. This estimate is an order of magnitude higher than that previously reported for Y euchromatic regions40 due to access to Yq12 satellite DNA (Supplementary Table 13). We note that 13 out of 45 (29%) of the DNMs had 100% identical matches elsewhere in the Yq12 region (but not at orthologous positions) and probably result from interlocus gene conversion events within the DYZ1/DYZ2 repeats41 (Methods). We also identify a total of nine de novo indels (<50 bp, homopolymers excluded) ranging from 1–3 indels per sample (mean, 1.8 events per Y transmission) and five de novo SVs (≥50 bp) (Fig. The latter range from 2,416 to 4,839 bp in size, each affecting an entire DYZ2 repeat unit(s), with an average of one SV per Y transmission. All applicable DNMs (SNVs, n = 20 out of 48; indels, n = 6 out of 9; SVs, n = 4 out of 5) are concordant with the expected transmission through generations (that is, from G2 to G3–G4 and from G3-NA12866 to his three male descendants in G4) (Fig. Y-chromosomal sequence classes are shown with the pairwise sequence identity between samples in 100 kb bins, with quality-control-passed SVs identified in the pedigree male individuals shown as blue and red outlines. DNMs that show evidence of transmission from G2 to G3–G4, and from G3-NA12886 to his male descendants in G4 are shown in grey. d, HiFi read support for the de novo SVA insertion in G3-NA12887. In total, we validated 41 de novo SVs across eight individuals (G3), including 16 insertions and 25 deletions (Methods) of which 68% (28 out of 41) originate in the paternal germline with a trend towards an increase in SVs with paternal age (Supplementary Fig. Almost all SVs (40 out of 41) correspond to TRs, including mutation in centromeres, Y chromosome satellites and clustered SDs (Supplementary Table 10). We estimate around 5 SVs (95% CI = 3–7) per transmission affecting approximately 4.4 kb of DNA (median, 4,875 bp). If we exclude de novo SVs mapping to the centromere and Y chromosomes (n = 14), the median size of the events drops by an order of magnitude (median, 362 bp). Non-allelic homologous recombination (NAHR) has frequently been invoked as a mechanism to underlie TR expansions and contractions42,43. However, we find that none of the 27 euchromatic de novo SVs coincide with recombination crossovers (Supplementary Fig. This argues against NAHR between homologous chromosomes during meiosis I as the primary mechanism for their origin, although we cannot preclude other mechanisms associated with double-stranded breaks not involving recombination. We identify one retrotransposition event: a full-length (3,407 bp long) de novo insertion of an SVA element (SVAF subfamily) (G3-NA12887)44 with the predicted donor mapping around 23 Mb upstream (Fig. This insertion is present at a low frequency (around 11% of reads) in the parent (G2-NA12878) but not in the grandparental transmitting haplotype, consistent with a germline mosaic event arising in G2 postzygotically (Fig. Most DNM studies40,45,46,47,48,49 are based on SRS data from large groups of trios and converge on around 60–70 DNMs per generation; however, these studies often exclude highly mutable regions of the genome7. Our multiplatform and multigenerational, assembly-based approach provides access to some of the most repetitive regions, such as centromeres and heterochromatic regions on the Y chromosome. The use of parental references in addition to the standard references and the ability to confirm transmissions across subsequent generations improves both sensitivity and specificity. In this multigenerational pedigree, we estimate a range of 98–206 DNMs per transmission (average of 152 per generation) and observe a strong paternal de novo bias (70–80%) and an increase with advancing paternal age, not only for SNVs but also for indels and SVs, including TRs. The rate of de novo SNVs varies by more than an order of magnitude depending on the genomic context, consistent with recent human population-based analyses7,50 and theoretical predictions51. SD regions show an 88% increase (2.2 × 10−8 (95% CI = 1.64 × 10−8–2.88 × 10−8) versus 1.17 × 10−8). This is driven by SDs with >95% identity. We also observe a significant decrease in the de novo transition/transversion ratio compared with the genome (χ2 test, P = 0.0109) as predicted7 (Supplementary Note 7). We estimate that satellite DNA in the Yq12 heterochromatic region41,52 is at least 30 times more mutable than autosomal euchromatin (3.86 × 10−7 mutations per bp per generation). It is composed of thousands of short satellite DNA repeats (DYZ1/Hsat3A6 and DYZ2/Hsat1B) organized into Mb blocks that are >98% identical41,52. This, along with the fact that 29% of mutational changes match to non-orthologous sites in Yq12, is consistent with ‘interlocus gene conversion' driving this >20-fold excess, potentially as a result of increased sister chromatid exchange events41. Previous studies predicted that 6–10% of DNMs are not germline in origin, but instead arise sometime after fertilization, giving rise to a mosaic variant14,53. This distinction has been based on allele balance thresholds53 or incomplete linkage to nearby SNVs across three generations14. LRS increases sensitivity by assigning nearly every de novo SNV to a parental haplotype and define PZM by its incomplete linkage to that haplotype. We classify 16% of de novo SNVs as postzygotic in origin (n = 119 PZMs/745 de novo SNVs). As all sequencing data in this study are derived from blood, we cannot demonstrate that every PZM is present in multiple tissues, but we can use transmission to the next generation as a proxy, as it reveals that the mutation is also present in germ cells. PZMs account for 12% of all SNVs transmitted to the next generation (n = 33 PZMs/275 transmitted SNVs), an increase over previous estimates. Early cell divisions of human embryos are frequently error prone54,55 with an accelerated rate of cell division potentially contributing to the large fraction of PZMs with high (>25%) allele balance. Such events would previously have been classified as germline but, consistent with PZM expectations, we find no paternal bias associated with these DNMs (Fig. TRs are among the most mutable loci of our genome36,56,57, with the number of such de novo events comparable to germline SNVs58 but affecting more than an order of magnitude more base pairs per generation. We find a threefold differential in TR DNM rate with increasing repeat number and motif length generally correlating with mutation rate. However, we observe an apparent mutation rate trough between dinucleotides and larger motif lengths (>10 bp) (Fig. 3b), which may reflect different mutational mechanisms based on locus size, motif length and complexity. For example, larger TR motifs may be more likely to mutate through NAHR, synthesis-dependent strand annealing or interlocus gene conversion while mutational events at STRs may be biased toward traditional replication-based slippage mutational mechanisms42,43. Consistent with some earlier genome-wide analyses of minisatellites59, we did not find evidence that TR changes are mediated by unequal crossover between homologues during meiosis as none of our TR de novo SVs (n = 27) coincided with recombination breakpoints. At five of these recurrent loci, we discovered multiple DNMs within a single generation (G3); these DNMs may be the outcome of germline mosaicism in a G2 parent or the activity of hypermutable TRs. Nearly all of these highly recurrent de novo events produced TR alleles that are significantly longer than the average short-read length and were detectable only using LRS. This includes changes in the length of around 7% of human centromeres in which insertions and deletions all occur as multiples of the predominant HOR unit56. The rate of de novo SVs increased from previous estimates of 0.2–0.3 per generation15,61 to 3–4 de novo SVs per generation reported in this study. There are several limitations to this study. First, homopolymers still remain challenging even with the use of Element data as longer alleles and motifs embedded in larger repeats are still not reliably assayed with short reads. Third, we limited DNM discovery to the first three generations of only one multigenerational family and used G4 for validation purposes of transmitted variants. We acknowledge that familial variation depends on the genetic background14,36,62 and, therefore, many more families will be required to establish a reliable estimate of the mutation rate, especially for complex regions of the genome. In that regard, it is perhaps noteworthy that efforts are underway to characterize additional pedigrees. Notwithstanding, this study highlights that a single sequencing technology and a single human genome reference are insufficient to comprehensively estimate mutation rates. Multigenerational resources such as these will further refine DNM estimates and serve as another useful benchmark63 for new algorithms and new sequencing technologies. This includes informed consent for publication of research data for 23 family members; the remaining 5 provided informed consent for biobanking with controlled access (Data availability). Cell lines for 14 members of the CEPH 1463 family (G1-GM12889, G1-GM12890, G1-GM12891, G1-GM12892, G2-GM12877, G2-GM12878, G3-GM12879, G3-GM12881, G3-GM12882, G3-GM12883, G3-GM12884, G3-GM12885, G3-GM12886 and G3-GM12887) were obtained from Coriell Institute for Medical Research (CEPH collection). Furthermore, we explored whether the obtained sequencing data match the expected inheritance patterns of parents and offspring. To our knowledge, none of the cell lines mentioned above were tested for mycoplasma contamination. Newly enrolled family members underwent informed consent, and blood was obtained for DNA and cell lines. DNA was extracted from whole blood using the Flexigene system (Qiagen 51206). All samples are broadly consented for scientific purposes, which makes this dataset ideal for future tool development and benchmarking studies. Sequencing data from orthogonal short- and long-read platforms were generated as follows: Illumina WGS data for G1–G3 were generated as previously described14. Illumina WGS data for G4 and marry-in spouses for G3 were generated by the Northwest Genomics Center using the PCR-free TruSeq library prep kit and sequenced to approximately 30× on the NovaSeq 6000 with paired-end 150 bp reads. PacBio HiFi data were generated according to the manufacturer's recommendations. In brief, DNA was extracted from blood samples as described or cultured lymphoblasts using the Monarch HMW DNA Extraction Kit for Cells & Blood (New England Biolabs, T3050L). At all steps, quantification was performed with Qubit dsDNA HS (Thermo Fisher Scientific, Q32854) measured on DS-11 FX (Denovix) and the size distribution checked using FEMTO Pulse (Agilent, M5330AA and FP-1002-0275.) HMW DNA was sheared with the Megaruptor 3 (Diagenode, B06010003 & E07010003) system using the settings 28/30, 28/31 or 27/29 based on the initial quality check to target a peak size of ~22 kb. After shearing, the DNA was used to generate PacBio HiFi libraries using the SMRTbell prep kit 3.0 (PacBio, 102-182-700). Size selection was performed either with diluted AMPure PB beads according to the protocol, or with Pippin HT using a high-pass cut-off between 10–17 kb based on shear size (Sage Science, HTP0001 and HPE7510). Libraries were sequenced either on the Sequel II platform on SMRT Cells 8M (PacBio, 101-389-001) using Sequel II sequencing chemistry 3.2 (PacBio,102-333-300) with 2 h pre-extension and 30 h movies on SMRT Link v.11.0 or 11.1, or on the Revio platform on Revio SMRT Cells (PacBio, 102-202-200) and Revio polymerase kit v1 (PacBio, 102-817-600) with 2 h pre-extension and 24 h movies on SMRT Link v.12.0. Ultra-high molecular mass gDNA was extracted from the lymphoblastoid cell lines according to a previously published protocol64. In brief, 3–5 × 107 cells were lysed in a buffer containing 10 mM Tris-Cl (pH 8.0), 0.1 M EDTA (pH 8.0), 0.5% (w/v) SDS, and 20 mg ml−1 RNase A for 1 h at 37 °C. Then, 200 μg ml−1 proteinase K was added, and the solution was incubated at 50 °C for 2 h. DNA was purified through two rounds of 25:24:1 phenol–chloroform–isoamyl alcohol extraction followed by ethanol precipitation. Precipitated DNA was solubilized in 10 mM Tris (pH 8.0) containing 0.02% Triton X-100 at 4 °C for 2 days. Libraries were constructed using the Ultra-Long DNA Sequencing Kit (ONT, SQK-ULK001) with modifications to the manufacturer's protocol: ~40 μg of DNA was mixed with FRA enzyme and FDB buffer as described in the protocol and incubated for 5 min at room temperature, followed by heat inactivation for 5 min at 75 °C. RAP enzyme was mixed with the DNA solution and incubated at room temperature for 1 h before the clean-up step. Clean-up was performed using the Nanobind UL Library Prep Kit (Circulomics, NB-900-601-01) and eluted in 450 μl EB. Then, 75 μl of library was loaded onto a primed FLO-PRO002 R9.4.1 flow cell for sequencing on the PromethION (using MinKNOW software v.21.02.17–23.04.5), with two nuclease washes and reloads after 24 and 48 h of sequencing. All G1–G3 ONT base calling was done with Guppy (v.6.3.7). Element WGS data were generated according to the manufacturer's recommendations. In brief, DNA was extracted from whole blood as described above. PCR-free libraries were prepared using mechanical shearing, yielding ~350 bp fragments, and the Element Elevate library preparation kit (Element Biosciences, 830-00008). Linear libraries were quantified by quantitative PCR and sequenced on AVITI 2 × 150 bp flow cells (Element Biosciences, not yet commercially available). Bases2Fastq Software (Element Biosciences) was used to generate demultiplexed FASTQ files. Single-cell Strand-seq libraries were prepared using a streamlined version of the established OP-Strand-seq protocol65 with the following modifications. In brief, EBV cells from G1–3 were cultured for 24 h in the presence of BrdU and nuclei with BrdU in the G1 phase of the cell cycle were sorted using fluorescence-activated cell sorting as described previously65. Next, single nuclei were dispensed into individual wells of an open 72 × 72 well nanowell array and treated with heat-labile protease, followed by digestion of DNA with the restriction enzymes AluI and HpyCH4V (NEB) instead of micrococcal nuclease (MNase). Next, fragments were A-tailed, ligated to forked adapters, UV-treated and PCR-amplified with index primers. The use of restriction enzymes results in short, reproducible, blunt-ended DNA fragments (>90% smaller than 1 kb) that do not require end-repair before adapter ligation, in contrast to the ends of DNA generated by MNase. The pre-spotted, dried primers survive and do not interfere with the library preparation steps before PCR amplification. Pre-spotting of index primers is more reliable than the transfer of index primers between arrays during library preparation as described previously65. Strand-seq libraries were pooled and cleaned with AMPure XP beads, and library fragments between 300 and 700 bp were gel purified before PE75 sequencing on either the NextSeq 550 or the AVITI (Element Biosciences) system. 40 shows examples of Strand-seq libraries made with restriction enzymes. The demultiplexed FASTQ files were aligned to both GRCh38 and T2T-CHM13 reference assemblies (Supplementary Table 14) using BWA66 (v.0.7.17-r1188) for standard library selection. Aligned reads were sorted by genomic position using SAMtools67 (v.1.10) and duplicate reads were marked using sambamba68 (v.1.0). Libraries passing quality filters were pre-selected using ASHLEYS69 (v.0.2.0). We also evaluated such selected Strand-seq libraries manually and further excluded libraries with an uneven coverage, or an excess of ‘background reads' (reads mapped in opposing orientation for chromosomes expected to inherit only Crick or Watson strands) as previously described70. This is done to ensure accurate inversion detection and phasing. Polymorphic inversions for G1–G3 were detected by mapping Strand-seq read orientation with respect to the reference genome as previously described71,72. For each sample, we selected 60+ Strand-seq libraries (range, 62–90) with a median of around 274,000 reads with mapping quality ≥10 per library, translating to about 0.67% genome (T2T-CHM13) being covered per library (Supplementary Fig. Then we ran breakpointR73 (v.1.15.1) across selected Strand-seq libraries to detect points of strand-state changes73. We used these results to generate sample-specific composite files using breakpointR function ‘synchronizeReadDir' as described previously71. Again, we ran breakpointR on such composite files to detect regions where Strand-seq reads map in reverse orientation and are indicative of an inversion. Lastly, we manually evaluated each reported inverted region by inspection of Strand-seq read mapping in UCSC Genome Browser74 and removed any low-confidence calls. We phased all inversions using Strand-seq data as well and then synchronized the phase with phased genome assemblies based on haplotype concordance. Lastly, we evaluated the Mendelian concordance of detected and fully phased inversions. We mark sites where at least half of the G3 samples were fully phased by Strand-seq and concordant with possible inherited G2 parental alleles as being Mendelian concordant (Supplementary Table 7). Phased genome assemblies were generated using two different algorithms, namely Verkko (v.1.3.1 and v.1.4.1)16 and hifiasm (UL) with ONT support (v.0.19.5)17. Owing to active development of the Verkko and hifiasm algorithms, assemblies were generated with two different versions. Phased assemblies for G2–G3 were generated using a combination of HiFi and ONT reads using parental Illumina k-mers for phasing. To generate phased genome assemblies of G1, we still used a combination of HiFi and ONT reads with the Verkko pipeline and used Strand-seq to phase assembly graphs75. Lastly, G4 samples were assembled using HiFi reads only with hifiasm (v.0.19.5). By contrast, for hifiasm assemblies, we report switched haplotype labelling such that haplotype 1 is paternal and haplotype 2 is maternal to match HPRC standard for hifiasm assemblies. To evaluate the base pair and structural accuracy of each phased assembly, we used a multitude of assembly evaluation tools as well as orthogonal datasets such as PacBio HiFi, ONT, Strand-seq, Illumina and Element data. Known assembly issues are listed in Supplementary Table 4. We note that we fixed four haplotype switch errors in our assembly-based variant callsets to avoid biases in subsequent analysis. We used Strand-seq data to evaluate directional and structural accuracy of each phased assembly. First, we aligned selected Strand-seq libraries for each sample to the phased de novo assembly using BWA66 (v.0.7.17-r1188). We next ran breakpointR73 (v.1.15.1) using aligned BAM files as the input. We then created directional composite files using the breakpointR function createCompositeFiles followed by running breakpointR on such composite files using the runBreakpointR function. This provided us, for any given sample, with regions where strand-state changes across all single-cell Strand-seq libraries. Many such regions point to real heterozygous inversions. However, regions where Strand-seq reads mapped in opposite orientation with respect to surrounding regions are probably caused by misorientation. Moreover, positions where the strand state of Strand-seq reads changes repeatedly in multiple libraries might be a sign of an assembly misjoin and such regions were investigated more closely to rule out any such large structural assembly inconsistencies. To evaluate de novo assembly accuracy, we aligned sample-specific PacBio HiFi reads to their corresponding phased genome assemblies using Winnowmap76 (v.2.03) with the following parameters: A reference-specific BED file (chm13v2.0.sd.bed) was used, setting a maximum read divergence of 2% and specifying reference-biased blocks. These flagged regions were analysed to identify collapses, false duplications, erroneous regions and correctly assembled haploid blocks with the expected read coverage. We used Flagger v.0.3.3 (https://github.com/mobinasri/flagger) to run the flagger_end_to_end WDL. Read-to-contig alignments—Winnowmap alignments of all HiFi reads to the assembly (hap1, hap2 and unassigned.fasta) A combined assembly fasta file with hap1, hap2 and unassigned contigs BAM alignments of assembly to the CHM13v2.0 reference hap1, hap2 and unassigned fasta files of the assembly were aligned to CHM13v2.0 using a pipeline available at GitHub (https://github.com/mrvollger/asm-to-reference-alignment). NucFreq77 (v.0.1) was used to calculate nucleotide frequencies for HiFi reads aligned using Winnowmap76 (v.2.03). This was used to identify regions of collapses, where the second-highest nucleotide count exceeded 5; and misassembly, where all nucleotide counts were zero. The NucFreq analysis pipeline is available at GitHub (https://github.com/mrvollger/NucFreq). These unique k-mers indicate potential base-pair errors. To evaluate the completeness of single-copy genes in our assemblies, we used compleasm79 (v.0.2.4). Further details are available at GitHub (https://github.com/huangnengCSU/compleasm). We ran compleasm with the following parameters: All de novo assemblies were aligned to both GRCh38 as well as to the complete version of the human reference genome T2T-CHM13 (v2) using minimap2 (ref. A complete pipeline for this reference alignment is available at GitHub: (https://github.com/mrvollger/asm-to-reference-alignment). We also generated a trimmed version of these alignments using the rustybam (v.0.1.33) (https://github.com/mrvollger/rustybam) function trim-paf to trim redundant alignments that mostly appear at highly identical SDs. With this, we aim to reduce the effect of multiple alignments of a single contig over these duplicated regions. For this analysis, we use assembly to reference alignments (see the ‘Assembly to reference alignment' section), reported as PAF files. We used trimmed PAF files reported by the rustybam trim-paf function. Any region with two or more alignments per haplotype is assigned as ‘multi' alignment. These reports were generated using the ‘getPloidy' R function (Code availability). The first approach is based on chromosome-length haplotypes extracted from Strand-seq data using R package StrandPhaseR81 (v.0.99). The second approach uses inheritance vectors derived from Mendelian consistency of small variants across the family pedigree13. Our final approach uses trio-based phased genome assemblies followed by small variant calling using PAV and Dipcall to more precisely define the meiotic breakpoints. By contrast, a recombination map of G4 individuals was constructed using a combination of Strand-seq data for G3 spouses and an assembly-based variant callset (Dipcall) of G4 samples. To map meiotic recombination breakpoints using circular binary segmentation, we used two different datasets. The first dataset represents phased small variants (SNVs and indels) as reported by Strand-seq-based (SSQ) phasing22,81. The other is based on small variants reported in trio-based phased assemblies either by PAV8 (v.2.3.4) or Dipcall82 (v.0.3). With this approach, we set to detect recombination breakpoints as positions where a child's haplotype switches from matching H1 to H2 of a given parent or vice versa. To detect these positions, we first established which homologue in a child was inherited from either parent by calculating the level of agreement between child's alleles and homozygous variants in each parent. Next, we compared each child's homologue to both homologues of the corresponding parent and encoded them as 0 or 1 if they match H1 or H2, respectively. We applied a circular binary segmentation algorithm on such binary vectors by using the R function fastseg implemented in the R package fastseg83 (v.1.46.0) with the following parameters: fastseg (binary.vector, minSeg={}, segMedianT=c (0.8, 0.2)). In the case of sparse Strand-seq haplotypes, we set the fastseg parameter minSeg to 20 and, in the case of dense assembly-based haplotypes, we used a larger window of 400 and 500 for Dipcall- and PAV-based variant calls to achieve comparable sensitivity in detecting recombination breakpoints. The regions with a segmentation mean of ≤0.25 are then marked as H1 while regions with a segmentation mean of ≥0.75 are assigned as H2. Regions with a segmentation mean in between these values were deemed to be ambiguous and were excluded. Moreover, we filtered out regions shorter than 500 kb and merged consecutive regions assigned the same haplotype (Code availability). DeepVariant calls (see the ‘Read-based variant calling' section) from HiFi sequencing data from G1, G2 and G3 pedigree members allow us to identify the haplotype of origin for heterozygous loci in G3 and infer the occurrence of a recombination along the chromosome when the haplotype of origin changes between loci. An initial outline of the inheritance vectors was identified by first applying a depth filter to remove variants outside the expected coverage distribution per sample; inheritance was then sketched out using a custom script, requiring a minimum of 10 SNVs supporting a particular haplotype, and manually refined to remove biologically unlikely haplotype blocks, or add additional haplotype blocks, where support existed, and refine haplotype coordinates. We developed a hidden Markov model framework to identify the most probable sequence of inheritance vectors from SNV sites using the Viterbi algorithm. For details including the transition/emission probabilities see ref. The values contained within transition and emission matrices were refined to recapitulate the previously identified inheritance vectors, while correctly identifying missing vectors. The Viterbi algorithm identified 539 recombinations, a maternal recombination rate of 1.29 cM per Mb, and a paternal recombination rate of 0.99 cM per Mb. Maternal bias was observed in the pedigree, with 57% of recombinations identified in G3 of maternal origin. Meiotic recombination breakpoints reported by different orthogonal technologies and algorithms (see the sections ‘Detection of meiotic recombination breakpoints using circular binary segmentation' and ‘Detection of meiotic recombination breakpoints using inheritance vectors') were merged separately for G2 and G3 samples. We started with the G3 recombination map where we used an inheritance-based map as a reference and then looked for support of each reference breakpoint in recombination maps reported based on PAV, Dipcall and Strand-seq (SSQ) phased variants. A recombination breakpoint was supported if for a given sample and homologue an orthogonal technology reported a breakpoint no further than 1 Mb from the reference breakpoint. We repeated this for the G2 recombination map as well. However, in the case of the G2 recombination map, we used a PAV-based map as a reference. This is because inheritance-based approaches need three generations to map recombination breakpoints in G3. We also report a column called ‘best.range', which is the narrowest breakpoint across all orthogonal recombination maps that directly overlaps with a given reference breakpoint. Lastly, we report a ‘min.range' column that represents for any given breakpoint a range with the highest coverage across all orthogonal datasets. Merged recombination breakpoints are reported in Supplementary Table 8. We tested enrichment of all (n = 1,503) recombination breakpoints detected in G2–G4 with respect to T2T-CHM13 if they cluster towards the ends of the chromosomes depending on parental homologue origin. For this, we counted the number of recombination breakpoints in the last 5% of each chromosome end specifically for maternal and paternal breakpoints. We then shuffle detected recombination breakpoints along each chromosome 1,000 times and redo the counts. Up to this point, all meiotic recombination breakpoints were called using variation detected with respect to a single linear reference (GRCh38 or T2T-CHM13). To alleviate any possible biases introduced by comparison to a single reference genome, we set out to refine detected recombination breakpoints for each inherited homologue (in child) directly in comparison to parental haplotypes from whom the homologue was inherited from. We start with a set of merged T2T-CHM13 reference breakpoints for G3 only by selecting the ‘best.range' column (Supplementary Table 8). Then, for each breakpoint, we set a ‘lookup' region to 750 kb on each side from the breakpoint boundaries and used the SVbyEye85 (v.0.99.0) function subsetPafAlignments to subset PAF alignments of a phased assembly to the reference (T2T-CHM13) to a given region. We did this separately for inherited child homologues (recombined) and the corresponding parental haplotypes that belong to a parent from whom the child homologue was inherited from. Next, we created a multiple sequence alignment (MSA) for three sequences (child-inherited homologue, parental homologue 1 and parental homologue 2) using the R package DECIPHER86 (v.2.28.0; with the function AlignSeqs). Fasta sequences of which the size differ by more than 100 kb or their nucleotide frequencies differ by more than 10,000 bases are skipped due to increased computational time needed to align such different sequences optimally using DECIPHER. After MSA construction, we selected positions with at least one mismatch and also removed sites where both parental haplotypes carry the same allele. A recombination breakpoint is a region where the inherited child homologue is partly matching alleles coming from parental homologues 1 and 2. We therefore skipped analysis of MSAs in which a child's alleles are more than 99% identical to a single parental homologue. If this filter is passed, we use the custom R function getAlleleChangepoints (Code availability) to detect changepoints where the child's inherited haplotype switches from matching alleles coming from parental haplotype 1 to alleles coming from parental haplotype 2. Such MSA-specific changepoints are then reported as a new range where a recombination breakpoint probably occurred. Lastly, we attempt to report reference coordinates of such MSA-specific breakpoints by extracting 1 kb long k-mers from the breakpoint boundaries and matching such k-mers against reference sequence (per chromosome) using R package Biostrings (v.2.70.2) with its function ‘matchPattern' and allowing for up to 10 mismatches. A list of refined recombination breakpoints is reported in Supplementary Table 8. We set out to detect smaller localized changes in parental allele inheritance using a previously defined recombination map of this family. We did this analysis for all G3 samples (n = 8) in comparison to G2 parents. For this, we iterated over each child's homologue (in each sample) and compared it to both parental homologues from which the child's homologue was inherited from. We did this by comparing SNV and indel calls obtained from phased genome assemblies between the child and corresponding parent. To consider only reliable variants, we retained only those supported by at least two read-based callers (either DeepVariant-HiFi, Clair3-ONT or dragen-Illumina callset). We further retained only variable sites that are heterozygous in the parent and were also called in the child. After such strict variant filtering, we slide by two consecutive child's variants at a time and compare them to both haplotype 1 and haplotype 2 of the respective parent of origin. For this similarity calculation, we use the custom R function getHaplotypeSimilarity (Code Availability). Then, for each haplotype segment, defined by recombination breakpoints, we report regions where at least two consecutive variants match the opposing parental haplotype in contrast to the expected parental homologue defined by recombination map. We further merge consecutive regions that are ≤5 kb apart. For the list of putative gene conversion events, we retained only regions that have not been reported as problematic by Flagger. We also removed regions that are ≤100 kb from previously defined recombination events and events that overlap centromeric satellite regions and highly identical SDs (≥99% identical). Lastly, we evaluated the list of putative allelic gene conversion events by visual inspection of phased HiFi reads. PacBio HiFi data were processed with the human-WGS-WDL available at GitHub (https://github.com/PacificBiosciences/HiFi-human-WGS-WDL/releases/tag/v1.0.3). We used the aligned haplotype-tagged HiFi BAMs for all downstream PacBio analysis. 88) (v.1.0.7) variant calls were made based on the alignments with default models for PacBio HiFi and ONT (ont_guppy5) data, respectively, with phasing and gVCF generation enabled. Variant calling was conducted on each chromosome individually and concatenated into one VCF. gVCFs were then fed into GLNexus89 with a custom configuration file. We used a previously established framework to define ground truth genetic variation13. Our analysis, in contrast to trio-based filtering, uses all four alleles to detect genotyping errors, whereas, in a trio, only two alleles are transmitted and observed. We establish a map of the haplotypes across the third generation (inheritance vector) from which we can adjudicate variant calls against. These functions are implemented as a single binary tool that requires the inheritance vectors and a standard formatted VCF file, for example: The pedigree filtering and additional steps to build a small variant truth set are available at GitHub (https://github.com/Platinum-Pedigree-Consortium/Platinum-Pedigree-Inheritance). Following the parameters outlined previously10, we called variants in HiFi data aligned to T2T-CHM13 using GATK HaplotypeCaller90 (v.4.3.0.0) and DeepVariant87 (v.1.4.0) and naively identified variants unique to each G2 and G3 sample. We separated out SNV and indel calls and applied basic quality filters, such as removing clusters of three or more SNVs in a 1 kb window. We combined this set of variant calls generated by a secondary calling method (https://github.com/Platinum-Pedigree-Consortium/Platinum-Pedigree-Inheritance/blob/main/analyses/Denovo.md) and subjected all calls to the following validation process. We validated both SNVs and indels by examining them in HiFi, ONT and Illumina read data, excluding reads that failed to reach the mapping quality (59 for long reads, 0 for short reads) thresholds. Reads with high base quality (>20) and low base quality (<20) at the variant site were counted separately. We retained variants that were present in at least two types of sequencing data for the child, and absent from high-base-quality parental reads. We determined an SNV was truly de novo if it was absent from every family member that was not a direct descendant of the de novo sample. Finally, we examined the allele balance of every variant, determined which variants were in TRs and re-evaluated parental read data across all sequencing platforms, removing variants with noisy sequencing data or more than two low-quality parental reads supporting the alternative allele (Supplementary Note 9). First, we used our initial GATK variant calls to identify informative sites in an 80 kb window around the DNM, selecting any single-nucleotide polymorphisms (SNPs) where one allele could be uniquely assigned to one parent (for example, a site that is homozygous reference in a father and heterozygous in a mother). For every DNM, we evaluated every ONT and HiFi read that aligned to the site of the de novo allele and assigned it to either a paternal or maternal haplotype (if informative SNPs were available) by calculating an inheritance score as outlined previously10. DNMs that were exclusively assigned to maternal or paternal haplotypes were successfully phased, whereas DNMs on conflicting haplotypes were excluded from our final callset. Unphased variants were determined to be postzygotic in origin (n = 7) if their allele balance was not significantly different across platforms (by a χ2 test) and if their combined allele balance was significantly different from 0.5. Once we assigned every read to a parental haplotype, we counted the number of maternal and paternal reads that had either the reference or alternative allele. We determined that a DNM was germline in origin if it was present on every read from a given parent's haplotype. Conversely, if a DNM was present on only a fraction of reads from a parental haplotype, we determined that it was postzygotic in origin. To identify DNMs on the X chromosome, we applied the same strategy as autosomal variants, with one exception: we used only variant calls generated by GATK. For male individuals, we reran GATK in haploid mode, such that it would only identify one genotype on the X chromosome. To identify DNMs on the Y chromosome, we aligned male HiFi, ONT and Illumina data to the G1-NA12889 chromosome Y assembly and then called variants using GATK in haploid mode on the aligned HiFi data. We validated SNVs and indels by examining the father's HiFi, ONT and Illumina data and excluded any variants that were present in the parental reads, applying the same logic that we used for autosomal variants. To determine where we were able to identify de novo variation in the genome, we assessed HiFi data for every trio. We first used GATK HaplotypeCaller90 (v.4.3.0.0) with the option ‘ERC BP_RESOLUTION' to generate a genotype call at every site in the genome. Only sites where both parents were genotyped as homozygous reference (0/0) were considered callable, as sites with a parental alternative allele were excluded from our de novo discovery pipeline. We then examined the HiFi reads from a sample and its parents, restricting to only primary alignments with mapping quality of at least 59. For children, we only considered HiFi reads derived from blood, but we considered blood and cell line data for parents. We counted the number of reads with a minimum base quality score of 20 at every site in the genome and then combined this information with our variant calls. A site was deemed to be callable if both parents and the child each had at least one high-quality read with a high-quality base call. Moreover, male sex chromosomes were not restricted to sites where both parents were genotyped as reference—each parent was allowed to carry an alternative allele. We calculated the germline autosomal mutation rate for every sample by dividing the number of germline autosomal DNMs by twice the number of base pairs we determined to be callable. For PZMs, we used the same denominator. For each feature-specific mutation rate (such as SDs), we intersected both a sample's de novo SNVs and the sample's callable regions with coordinates of the relevant feature. Given the challenges associated with assaying mutations in STRs (1–6 bp motifs) and VNTRs (≥7 bp motifs), we applied a targeted HiFi genotyping strategy coupled with validation by transmission and orthogonal sequencing. The command trf-mod -s 20 -l 160 {reference.fasta} was used, resulting in a minimum reference locus size of 10 bp and motif sizes of 1 to 2,000 bp (https://github.com/lh3/TRF-mod)91. The remaining loci were annotated with tr-solve (https://github.com/trgt-paper/tr-solve) to resolve locus structure in compound loci. Only TRs annotated on Chromosomes 1–22, X and Y were considered (Data availability). TRGT32 is a software tool for genotyping TR alleles using PacBio HiFi sequencing reads (https://github.com/PacificBiosciences/trgt). Provided with aligned HiFi sequencing reads (in BAM format) and a file that enumerates the genomic locations and motif structures of a collection of TR loci, TRGT will return a VCF file with inferred genotypes at each TR locus. In this analysis, we ran TRGT (v.0.7.0-493ef25) on each member of the CEPH 1463 pedigree using the TR catalogue defined above. TRGT was run using the default parameters: trgt --threads 32 --genome {in_reference} --repeats {in_bed} --reads {in_bam} --output-prefix {out_prefix} --karyotype {karyotype}` bcftools sort -m 3072M -Ob -o {out_prefix}.sorted.vcf.gz {out_prefix}.vcf.gz bcftools index --threads 4 {out_prefix}.sorted.vcf.gz samtools sort -@ 8 -o {out_prefix}.spanning.sorted.bam {out_prefix}.spanning.bam samtools index -@ 8 {out_prefix}.spanning.sorted.bam To determine the concordant inheritance of TRs, we calculated the possible Manhattan distances derived from all possible combinations of a proband's allele length (AL) from TRGT with both the maternal and paternal AL values. We considered a locus to be concordant if the minimum Manhattan distance from all computed distances was found to be 0, suggesting that a combination of the proband's AL values matched the parental AL values perfectly. By contrast, if the minimum Manhattan distance was greater than 0, suggesting that all combinations of the proband's AL values exhibited some deviation from the parental AL values, we regarded the locus as discordant and recorded it as a potential Mendelian inheritance error. For each TR locus, we calculated the number of concordant trios, the number of MIE trios and the number of trios that had missing values and could not be fully genotyped. Loci with any missing genotypes were excluded when calculating the percent concordance; however, individual complete trios were considered for de novo variant calling below. We focused de novo TR calling on G3 for several reasons. First, their G2 parents (NA12877 and NA12878) were sequenced to 99 and 109 HiFi sequencing depths, resulting in a far lower chance of parental allelic dropout than samples with more modest sequencing depths. Second, G1 DNA was derived from cell lines, increasing the risk of artefacts when calling DNMs in G2. We used TRGT-denovo33 (v.0.1.3), a companion tool to TRGT, to enable in-depth analysis of TR DNMs in family trios using HiFi sequencing data (https://github.com/PacificBiosciences/trgt-denovo). TRGT-denovo uses consensus allele sequences and genotyping data generated by TRGT and also incorporates additional evidence from spanning HiFi reads used to predict these allele sequences. In brief, TRGT-denovo extracts and partitions spanning reads from each family member (mother, father and child) to their most likely alleles. Parental spanning reads are realigned to each of the two consensus allele sequences in the child, and alignment scores (which summarize the difference between a parental read and a consensus allele sequence) are computed for each read. At every TR locus, each of the two child alleles is independently considered as a putative de novo candidate. We measured the sizes of de novo TR alleles with respect to the parental TR allele that most likely experienced a contraction or expansion event. If TRGT-denovo reported a de novo expansion or contraction at a particular locus, we did the following to calculate the size of the event. If we were able to phase the de novo TR allele to a parent of origin, we simply identify the minimum diff among that parent's ALs and treat it as the likely expansion/contraction size. Otherwise, we assume that the smallest diff across all parental ALs represents the likely de novo size. We applied a series of filters to the candidate TR DNMs (identified by TRGT-denovo) to remove likely false positives. For each de novo allele observed in a child, we required the following (Supplementary Notes 9 and 10): HiFi sequencing depth in the child, mother, and father ≥10 reads. The candidate de novo allele must represent an expansion or contraction with respect to the parental allele. At least two HiFi reads supporting the candidate de novo allele (denovo_coverage ≥ 2) in the child, and at least 20% of total reads supporting the candidate de novo allele (child_ratio ≥ 0.2). Fewer than 5% of parental reads likely supporting the candidate de novo AL in the child. To calculate TR DNM rates in a given individual, we first calculated the total number of TR loci (among the ~7.8 million loci genotyped using TRGT) that were covered by at least 10 HiFi sequencing reads in each member of the focal individual's trio (that is, the focal individual and both of their parents). We then divided the total count of de novo TR alleles by the total number of callable loci to obtain an overall DNM rate, expressed per locus per generation. 3a, we also estimated DNM rates as a function of the minimum motif size observed within a locus. We counted the number of TR DNMs that occurred at loci with a minimum motif size of N and divided that count by the total number of TR loci with a minimum motif size of N that passed filtering thresholds. When calculating STR, VNTR and complex mutation rates, we defined STR loci as loci at which all constituent motifs were between 1 and 6 bp; we defined VNTR loci as loci at which all motifs were larger than 6 bp; and we defined complex loci as loci at which there were both STR (1–6 bp) and VNTR (≥7 bp) motifs. Previous studies usually measured STR mutation rates at loci that are polymorphic within the cohort of interest. To generate mutation rate estimates that are more consistent with these previous studies, we also calculated the number of STR loci that were polymorphic within the CEPH 1463 pedigree. Loci were defined as polymorphic if at least two unique ALs were observed among the CEPH 1463 individuals at a given TR locus. We note that this definition of polymorphic STRs is sensitive to both the size of the cohort and the sequencing technology used to genotype STRs. Moreover, by defining loci as polymorphic if we observed more than one unique AL across the cohort, we may erroneously classify loci as polymorphic if HiFi sequencing reads exhibited a substantial amount of stutter at those loci, producing variable estimates of STR ALs across individuals. In total, 1,096,430 STRs were polymorphic within the cohort. To calculate mutation rates in each G3 individual, we applied the same coverage quality thresholds as described above. The STRs genotyped by TRGT were phased using HiPhase92 (v.1.0.0-f1bc7a8). We followed HiPhase's guidelines for jointly phasing small variants, SVs and TRs by inputting the relevant VCF files from DeepVariant, PBSV and TRGT into HiPhase, resulting in three phased VCF files for each analysed sample. We also activated global realignment through the --global-realignment-cputime parameter to improve allele assignment accuracy. Note that HiPhase specifically excludes variants that fall entirely within genotyped STRs from the phasing process. This is motivated because STRs often encompass numerous smaller variants. We used the phased genotypes inferred by HiPhase to determine the likely parent of origin for de novo TR expansions and contractions. For each phased de novo allele that we observed in a child, we examined all informative SNVs in that child's parents ±500 kb from the de novo allele. We defined informative sites using the following criteria: sites must be biallelic SNVs; total read depth in the mother, father and child must be at least 10 reads; Phred-scaled genotype quality in the mother, father and child must be at least 20; the child's genotype must be heterozygous; and the parents' genotypes must not be identical-by-state. We repeat this process for all informative sites within the ±500 kb interval. At each candidate de novo TR allele, we calculated concordance between the de novo ALs estimated by TRGT and the ALs supported by Element, ONT or HiFi reads. We restricted our concordance analyses to autosomal TR loci with a single expansion or contraction (that is, we did not analyse ‘complex' TR loci containing multiple unique expansions and/or contractions). TRGT reports two AL estimates for every member of a trio at an autosomal TR locus, and TRGT-denovo assigns one of these two ALs to be the de novo AL in the child. We refer to the difference between the TRGT AL and the reference locus size as the relative AL. We then queried BAM files containing Element, Illumina, ONT or PacBio HiFi reads at each TR locus. Using the pysam library (https://github.com/pysam-developers/pysam), we iterated over all reads that completely spanned the TR locus and had a mapping quality of 60. For example, an Element read might have the following CIGAR string: 100M2D10M6I32M. For each of the CIGAR operations that overlap the TR locus, we increment a counter by OP * BP, where OP equals 0 for ‘match' CIGAR operations, 1 for ‘insertion' operations, and -1 for ‘deletion' operations, and BP equals the number of base pairs associated with the given CIGAR operation. Thus, at each TR locus, we generated a distribution of net CIGAR operations in each member of the trio. We used these net CIGAR operations to validate candidate de novo TR alleles in each child. We then calculated the number of Element reads in that child's parents supporting the de novo AL (also allowing for off-by-one errors). If at least one Element read supported the de novo TR AL in the child, and zero Element reads supported the de novo TR AL in both parents, we considered the de novo TR to be validated. To assemble a confident list of candidate recurrent de novo TR alleles, we first assembled a list of TR loci where two or more CEPH 1463 individuals (in either G2, G3 or G4) harboured evidence for a de novo TR allele. For each candidate locus, we then required that all members of the CEPH 1463 pedigree were genotyped for a TR allele at the locus and had at least 10 aligned HiFi reads at the locus. These filters produced a list of 49 candidate loci where we observed evidence of either intragenerational or intergenerational recurrence. We visually inspected HiFi read evidence using the Integrated Genomics Viewer (IGV)93, as well as bespoke plots of HiFi CIGAR operations, at each locus to determine whether the candidate de novo TR alleles seemed plausible. We attempted to obtain putative de novo SVs from three different sources. The first one is based on reporting de novo SVs from read-based callsets (PBSV (v.2.9.0), Sniffles94 (v.0.12.0), Sawfish95 (v.2.2)). The second reports putative de novo SVs from variants called in phased genome assemblies. The last used pangenome graphs constructed from phased genome assemblies to report de novo SVs. SVPOP8 (v.3.4.0) (https://github.com/EichlerLab/svpop) was used to produce a merged PAV callset across all samples. The merge definition used was: nr::ro:szro:exact:match. The samples were provided in this order (G1–G2–G3): NA12889, NA12890, NA12891, NA12892, NA12877, NA12878, NA12879, NA12881, NA12882, NA12883, NA12884, NA12885, NA12886, NA12887. To compare variant calls against the previous generation, SVPOP was used again to do a PBSV/PAV intersection. This involved intersecting the PAV calls for G3 with the PBSV calls for G2, comparing each sample in G3 against each sample in G2. The callable BED files from PAV, intersections with G2's PBSV calls, and the list of putative de novo calls went into our validation pipeline. The pipeline (1) checks if the putative de novo variant was called by PBSV in either parent. (2) Checks if the putative de novo variant is seen in HiFi reads in either parent by running subseq (https://github.com/EichlerLab/subseq). Verkko assemblies were partitioned by chromosome by mapping them against the GRCh38, T2T-CHM13 and HG002 (v.1.0.1) human reference genomes using WFMASH (v.0.13.1-251f4e1) pangenome aligner. On each set of contigs, we applied PGGB (v0.6.0-87510bc) to build chromosome-level unbiased pangenome variation graphs96 with the following parameters: -s 20k -p 95 -k 47 -V chm13:100000, grch38:100000. Variants were then decomposed by applying VCFBUB (v.0.1.0-26a1f0c) to retain those found in top-level bubbles that are anchored on the genome used as reference, and VCFWAVE (v.1.0.3) to homogenize SV representation across samples. Subsequently, raw VCF files were used as an input for pedigree-based filtering of putative de novo SVs. Filtering of de novo SVs was done using BCFtools (v.1.17) +fill-tags followed by filtering the joint-called VCF for singleton-derived alleles at sites where all samples had a genotype call. By considering all G2/G3 family members (not just trios), we increased de novo SV specificity. All candidate de novo SVs collected across all regions of the genomes were further evaluated using phased genome assemblies and long-read alignments. Further details are provided in Supplementary Note 10. We first extracted an inserted SVA element in the de novo Verkko assembly of NA12887 (maternal haplotype, haplotype 1). 80) (v.2.24) to align this ~3.4-kb-long piece of DNA to both maternal and paternal Verkko assemblies using the parameters reported below: With these parameters we reported all locations of this DNA segment. We defined a putative donor site as an alignment position in maternal haplotype that has nearly perfect match with SVA de novo insertion. To identify completely and accurately assembled centromeres from each genome assembly, we first aligned the genome assemblies generated via Verkko16 or hifiasm (UL)17 to the T2T-CHM13 reference genome1 using minimap2 (ref. We also aligned native ONT data >30 kb in length from the same source genome to each whole-genome assembly using minimap2 (v.2.28) and assessed the assemblies for uniform read depth across the centromeric regions using IGV browser93. To identify de novo SVs and SNVs within each centromeric region, we first aligned each child's genome assembly to the relevant parent's genome assembly using minimap2 and the following parameters: -a --eqx -x asm20 -s 5000 -I 10G -t {threads}. We then used the resulting PAF file to identify de novo SVs and SNVs using SVbyEye85 (v.0.99.0), filtering our results to only those centromeres that were completely and accurately assembled. We checked each SV and SNV call with NucFreq, Flagger9 and native ONT data to ensure that the underlying data supported each call. Further details are provided in Supplementary Notes 9 and 10. We processed all G1, G2 and G3 assemblies with Tandem Repeats Finder (TRF)91 to determine the existence of the canonical telomeric repeat (p-arm, CCCTAA; q-arm, TTAGGG) within the distal regions of each assembled contig; TRF (v.4.09.1) was run with parameters: '2 7 7 80 10 50 10 -d -h-ngs', recommended for young (in this context, non-deteriorated) repeats as implemented in RepeatMasker (v.4.1.6). 80) (v.2.24) using the asm20 preset to establish the identities of each sequence (that is, whether a given contig represented the whole reference chromosome or a part of it, and whether it should be reverse-complemented to represent it canonically). With identities established, TRF annotations were crawled from the outside in (from the 5′ end on p-arms and from the 3′ end on q-arms, with respect to reverse complementarity as reported by minimap2) until the canonical repeat was encountered; incidences of non-canonical interspersed repeats were also retained. Moreover, PacBio HiFi reads were mapped to the contigs to assess by how many HiFi reads each region of each assembly was supported (coverage depth); distal regions supported by fewer than five HiFi reads were masked. Of the non-acrocentric chromosome ends across all G1, G2 and G3 samples, 74.2% of the Verkko assemblies (893 out of the possible 1,204 across all participants and haplotypes) were found to terminate in a canonical telomeric repeat (either spanning from the very start or end of the contig, or immediately adjacent to the region masked due to low coverage) with the median length of such repeats being 5,608 bp (Supplementary Table 3). Moreover, out of the T2T-CHM13 chromosomes for which both p and q telomeric ends were recovered, 64.6% (221 out of 342) were represented each by a single assembled contig spanning from the p telomere to the q telomere. 3) with a median length of the canonical repeat being 4,674 bp (Supplementary Table 3; same as for G1–G3), and the contiguity was markedly worse: only one chromosome (chromosome 9 in haplotype 1 of individual G4-200101) was verifiably spanned by a single contig (h1tg000017l). To determine the CpG methylation status of each centromere, we first base called raw ONT data with Guppy (https://community.nanoporetech.com; v.6.5.7) using the sup-prom model and the dna_r9.4.1_450bps_modbases_5hmc_5mc_cg_sup_prom.cfg config file. Next, we aligned the ONT data from each sample to the respective genome assembly using minimap2 (ref. 80) (v.2.28) with the following parameters: -ax lr:hq -y -t 4 -I 8 g. We converted the resulting BAM file to a bedMethyl file using modbam2bed (https://github.com/epi2me-labs/modbam2bed) and the following parameters: -e -m 5mC --cpg -t {threads} {input.bam} > {output.bed}. Next, we converted the bedMethyl file into a bedGraph using the following command: awk ‘BEGIN {OFS=“\t”}; {print $1, $2, $3, $11}' {input.bed} | grep -v “nan” | sort -k1,1 -k2,2n > {output.bedgraph} and subsequently converted the bedGraph into a bigwig using bedGraphToBigWig (https://www.encodeproject.org/software/bedgraphtobigwig/) and then visualized the bigwig file using Integrative Genomics Viewer93,98 (v.2.16.0). To determine the size of a hypomethylated region (termed the CDR2,39) in each centromere, we used CDR-Finder (https://github.com/arozanski97/CDR-Finder), which first bins the bedGraph into 5 kb windows, computes the median CpG methylation frequency within windows containing α-satellite (as determined by RepeatMasker99 (v.4.1.0)), selects bins that have a lower CpG methylation frequency than the median frequency in the region, merges consecutive bins into a larger bin, filters for merged bins >50 kb and reports the location of these bins. The construction and dating of Y-chromosomal phylogeny for 58 total samples, combining the 14 pedigree males from the current study with 44 individuals, for which long-read-based Y assemblies have previously been published, was done as described previously in detail52. In brief, all sites were called from the Illumina high-coverage data14 of the 14 pedigree males using the approximately 10.4 Mb of Y-chromosomal sequence previously defined as accessible to SRS100. SNVs within 5 bp of an indel call (SnpGap) and all indels were removed, followed by filtering all calls for a minimum read depth of 3 and a requirement of ≥85% of reads covering the position to support the called genotype. 52 for the 44 individuals using BCFtools, and then sites with ≥5% of missing calls, that is, missing in more than 3 out of 58 samples, were removed using VCFtools103 (v.0.1.16). The Y haplogroups of each sample were predicted as previously described104 and correspond to the International Society of Genetic Genealogy nomenclature (ISOGG; https://isogg.org; v.15.73). A coalescence-based method implemented in BEAST105 (v.1.10.4) was used to estimate the ages of internal nodes. RAxML106 (v.8.2.10) with the GTRGAMMA substitution model was used to construct a starting maximum-likelihood phylogenetic tree for BEAST. A constant-sized coalescent tree prior, the GTR substitution model, accounting for site heterogeneity (gamma), and a strict clock with a normal distribution based on the 95% CI of the substitution rate (0.76 × 10−9 (95% CI = 0.67 × 10−9–0.86 × 10−9) single-nucleotide mutations per base pair per year) was used107. Detailed analysis of Y-chromosomal DNMs focused on seven male individuals (R1b1a-Z302 Y haplogroup, G1-NA12889, G2-NA12877, G3-NA12882, G3-NA12883, G3-NA12884 and G3-NA12886) for whom phased Verkko assemblies were generated. Contigs containing X- and Y-chromosomal sequences were identified and extracted from the whole-genome assemblies as previously described52. The Yq12 repeat annotations were generated using HMMER108 (v.3.3.2dev) with published DYZ1, DYZ2, DYZ18, 2k7bp and 3k1bp sequences52, followed by manual checking of repeat unit orientation and distance from each other. Dot plots to compare Y-chromosomal sequences were generated using Gepard109 (v.2.0). For this reason, the Y assembly of the G1 grandfather NA12889 was used as a reference for DNM detection. Variants were identified from the MSY only, that is, the pseudoautosomal regions were excluded from this analysis. All identified variants were filtered as follows: any variant calls overlapping with regions flagged by Flagger or NucFreq in either reference or query assembly were filtered out. For SNVs, the final filtered calls were supported by 100% of HiFi reads (that is, no reads supported the reference allele in offspring or alternative allele in the father) and ONT reads mapped to both the reference and each individual assembly were checked for support. Individual reads mapped to the reference (G1 NA12889 Y assembly) and covering the indel call plus 150 bp of flanking sequence were extracted from all samples using subseq (https://github.com/EichlerLab/subseq), followed by alignment using MAFFT110,111 (v.7.508) with the default parameters. All alignments were manually checked and any calls where the HiFi data had two or more reads supporting a reference allele and one or more reads supporting an alternative allele were removed. All final SNV and indel calls were additionally supported (if unique mapping to the region was possible) by both Illumina and Element read data mapped to the reference. For all SV calls, HiFi read depth for reference and alternative alleles were visualized and SVs in regions showing high levels of read depth variation coinciding with clusters of SNVs with >10% of reads supporting an alternative allele removed. HiFi and ONT reads mapped to both the reference and individual assemblies were checked for support. For all variants, concordance with the expected transmission through generations was confirmed. Moreover, the HiFi data available for three G4 male individuals (200101, 200102 and 200105) were checked for support of the identified variants. The assembly-based DNM rates were calculated for each of the five male individuals based on the accessible regions of each individual Y assembly (that is, any regions flagged by Flagger and/or NucFreq were removed). Mobile element analysis was performed on PacBio HiFi reads using xTea112 (v.0.1.9). Potential non-reference mobile element insertions (MEI) identified with xTea were visualized using IGV to ensure that the insertions were identifiable in the sequencing reads and to determine whether any of these events were de novo. Using BEDTools113, we intersected the non-reference insertions with introns, exons, 5′-UTRs and 3′-UTRs from T2T-CHM13. If there were multiple matches in the reference genome that had the same score, a source element was not called. MEI sequences representing known Alu, L1 and SVA subclasses were obtained from previous work115, Dfam116 and UCSC Genome Browser74. Reference and novel sequences for each MEI class were combined into class-specific files. MEI sequences were aligned using the MUSCLE117 (v.3.8.31) aligner. Pairwise distances among MEI sequences were calculated using a Kimera two-parameter method and then converted to correlations. Principal components were obtained by eigenvalue decomposition of the pairwise correlation matrix. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All underlying data from 28 members of the family are available as part of the AWS Open Data program, European Nucleotide Archive (ENA) or dbGaP. Variant calls, mapped sequencing data and assemblies for 23 family members (G1-GM12889, G1-GM12890, G1-GM12891, G1-GM12892, G2-GM12877, G2-GM12878, G3-GM12879, G3-GM12881, G3-GM12882, G3-GM12885, G3-GM12886, G3-200080-spouse, G4-200081, G4-200082, G4-200084, G4-200085, G4-200086, G4-200087, G3-200100-spouse, G4-200101, G4-200102, G4-200104 and G4-200106) who provided consent for their data to be publicly accessible similar to the 1000 Genomes Project samples to allow for development of new technologies, study of human variation, research on the biology of DNA and study of health and disease are available via the AWS Open Data program (s3://platinum-pedigree-data/) as well as the European Nucleotide Archive (BioProject: PRJEB86317). Specific details on how to access the data are provided at GitHub (https://github.com/Platinum-Pedigree-Consortium/Platinum-Pedigree-Datasets). Mapped sequencing data and assemblies for five family members (G3-NA12883, G3-NA12884, G3-NA12887, G4-200103 and G4-200105) who did not consent for open access are available at dbGaP (phs003793.v1.p1; Platinum Pedigree Consortium LRS). These also include variant calls for the whole family (28 members). The TR catalogues are available at Zenodo (https://doi.org/10.5281/zenodo.13178746). The Y-chromosomal assembly for a closely related R1b haplogroup sample HG00731 was downloaded from the Human Genome Structural Variation Consortium IGSR site (https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/HGSVC3/working/20230927_verkko_batch2/assemblies/HG00731/). Custom code and pipelines used in this study are publicly available at GitHub (https://github.com/orgs/Platinum-Pedigree-Consortium/repositories). The complete sequence of a human genome. Complete genomic and epigenetic maps of human centromeres. Vollger, M. R. et al. Segmental duplications and their variation in a complete human genome. Guarracino, A. et al. Recombination between heterologous human acrocentric chromosomes. Miga, K. H. & Eichler, E. E. Envisioning a new era: complete genetic information from routine, telomere-to-telomere genomes. Porubsky, D. & Eichler, E. E. A 25-year odyssey of genomic technology advances and structural variant discovery. Increased mutation and gene conversion within human segmental duplications. Ebert, P. et al. Haplotype-resolved diverse human genomes and integrated analysis of structural variation. Liao, W.-W. et al. A draft human pangenome reference. Noyes, M. D. et al. Familial long-read sequencing increases yield of de novo mutations. Dausset, J. et al. Centre d'etude du polymorphisme humain (CEPH): collaborative genetic mapping of the human genome. Eberle, M. A. et al. A reference data set of 5.4 million phased human variants validated by genetic inheritance from sequencing a three-generation 17-member pedigree. Sasani, T. A. et al. Large, three-generation human families reveal post-zygotic mosaicism and variability in germline mutation accumulation. Belyeu, J. R. et al. De novo structural mutation rates and gamete-of-origin biases revealed through genome sequencing of 2,396 families. Rautiainen, M. et al. Telomere-to-telomere assembly of diploid chromosomes with Verkko. Cheng, H., Asri, M., Lucas, J., Koren, S. & Li, H. Scalable telomere-to-telomere assembly for diploid and polyploid genomes with double graph. The Platinum Pedigree: a long-read benchmark for genetic variants. Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. Sudmant, P. H. et al. An integrated map of structural variation in 2,504 human genomes. Cooper, G. M. et al. A copy number variation morbidity map of developmental delay. & Livingston, D. M. Mitotic gene conversion lengths, coconversion patterns, and the incidence of reciprocal recombination in a Saccharomyces cerevisiae plasmid system. Williams, A. L. et al. Non-crossover gene conversions show strong GC bias and unexpected clustering in humans. Halldorsson, B. V. et al. Characterizing mutagenic effects of recombination through a sequence-level genetic map. Kong, A. et al. Fine-scale recombination rate differences between sexes, populations and individuals. Broman, K. W., Murray, J. C., Sheffield, V. C., White, R. L. & Weber, J. L. Comprehensive human genetic maps: individual and sex-specific variation in recombination. Kong, A. et al. A high-resolution recombination map of the human genome. Refined genetic maps reveal sexual dimorphism in human meiotic recombination at multiple scales. Kong, A. et al. Recombination rate and reproductive success in humans. Hussin, J., Roy-Gagnon, M.-H., Gendron, R., Andelfinger, G. & Awadalla, P. Age-dependent recombination rates in human pedigrees. Characterization and visualization of tandem repeats at genome scale. Mokveld, T. et al. TRGT-denovo: accurate detection of de novo tandem repeat mutations. Arslan, S. et al. Sequencing by avidity enables high accuracy with low reagent consumption. Patterns of de novo tandem repeat mutations and their role in autism. The mutational dynamics of short tandem repeats in large, multigenerational families. Kristmundsdottir, S. et al. Sequence variants affecting the genome-wide rate of germline microsatellite mutations. The structure, function and evolution of a complete human chromosome 8. Gershman, A. et al. Epigenetic patterns in a complete human genome. The Y-chromosome point mutation rate in humans. The complete sequence of a human Y chromosome. Richard, G. & Pâques, F. Mini‐ and microsatellite expansions: the recombination connection. Verbiest, M. et al. Mutation and selection processes regulating short tandem repeats give rise to genetic and phenotypic diversity across species. Feusier, J. et al. Pedigree-based estimation of human mobile element retrotransposition rates. & Brunner, H. G. De novo mutations in human genetic disease. Kong, A. et al. Rate of de novo mutations and the importance of father's age to disease risk. Francioli, L. C. et al. Genome-wide patterns and properties of de novo mutations in humans. Goldmann, J. M. et al. Parent-of-origin-specific signatures of de novo mutations. Jónsson, H. et al. Parental influence on human germline de novo mutations in 1,548 trios from Iceland. The variation and evolution of complete human centromeres. Teshima, K. M. & Innan, H. The coalescent with selection on copy number variants. Assembly of 43 human Y chromosomes reveals extensive complexity and variation. Acuna-Hidalgo, R. et al. Post-zygotic point mutations are an underrecognized source of de novo genomic variation. Ju, Y. S. et al. Somatic mutations reveal asymmetric cellular dynamics in the early human embryo. Park, S. et al. Clonal dynamics in early human embryogenesis inferred from somatic mutation. Smith, G. P. Evolution of repeated DNA sequences by unequal crossover. Jeffreys, A. J., Royle, N. J., Wilson, V. & Wong, Z. Spontaneous mutation rates to new length alleles at tandem-repetitive hypervariable loci in human DNA. Willems, T. et al. Population-scale sequencing data enable precise estimates of Y-STR mutation rates. Bois, P. & Jeffreys, A. J. Minisatellite instability and germline mutation. Fu, Y. H. et al. An unstable triplet repeat in a gene related to myotonic muscular dystrophy. Collins, R. L. et al. A structural variation reference for medical and population genetics. Timing, rates and spectra of human germline mutation. Zook, J. M. et al. An open resource for accurately benchmarking small variant and reference calls. Logsdon, G. A. HMW gDNA purification and ONT ultra-long-read data generation v1. Hanlon, V. C. T. et al. Construction of Strand-seq libraries in open nanoliter arrays. Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows–Wheeler transform. & Prins, P. Sambamba: fast processing of NGS alignment formats. Gros, C., Sanders, A. D., Korbel, J. O., Marschall, T. & Ebert, P. ASHLEYS: automated quality control for single-cell Strand-seq data. & Lansdorp, P. M. Single-cell template strand sequencing by Strand-seq enables the characterization of individual homologs. Sanders, A. D. et al. Characterizing polymorphic inversions in human genomes by single-cell sequencing. Porubsky, D. et al. Recurrent inversion polymorphisms in humans associate with genetic instability and genomic disorders. Porubsky, D. et al. breakpointR: an R/Bioconductor package to localize strand state changes in Strand-seq data. Henglin, M. et al. Graphasing: phasing diploid genome assembly graphs with single-cell strand sequencing. Jain, C., Rhie, A., Hansen, N. F., Koren, S. & Phillippy, A. M. Long-read mapping to repetitive reference sequences using Winnowmap2. Vollger, M. R. et al. Long-read sequence and assembly of segmental duplications. Rhie, A., Walenz, B. P., Koren, S. & Phillippy, A. M. Merqury: reference-free quality, completeness, and phasing assessment for genome assemblies. Huang, N. & Li, H. compleasm: a faster and more accurate reimplementation of BUSCO. Dense and accurate whole-chromosome haplotyping of individual genomes. Li, H. et al. A synthetic-diploid benchmark for accurate variant-calling evaluation. Klambauer, G. et al. cn.MOPS: mixture of Poissons for discovering copy number variations in next-generation sequencing data with a low false discovery rate. Gel, B. et al. regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests. Porubsky, D. et al. SVbyEye: a visual tool to characterize structural variation among whole-genome assemblies. Wright, E. S. Using DECIPHER v2.0 to analyze big biological sequence data in R. R J. Poplin, R. et al. A universal SNP and small-indel variant caller using deep neural networks. Zheng, Z. et al. Symphonizing pileup and full-alignment for deep learning-based long-read variant calling. Accurate, scalable cohort variant calls using DeepVariant and GLnexus. Benson, G. Tandem repeats finder: a program to analyze DNA sequences. Holt, J. M. et al. HiPhase: jointly phasing small, structural, and tandem repeat variants from HiFi sequencing. Detection of mosaic and population-level structural variants with Sniffles2. Saunders, C. T. et al. Sawfish: improving long-read structural variant discovery and genotyping with local haplotype modeling. Poznik, G. D. et al. Sequencing Y chromosomes resolves discrepancy in time to common ancestor of males versus females. Li, H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Danecek, P. et al. Twelve years of SAMtools and BCFtools. & Tyler-Smith, C. A Southeast Asian origin for present-day non-African human Y chromosomes. BEAST: Bayesian evolutionary analysis by sampling trees. Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Fu, Q. et al. Genome sequence of a 45,000-year-old modern human from western Siberia. & Punta, M. Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions. Krumsiek, J., Arnold, R. & Rattei, T. Gepard: a rapid and sensitive tool for creating dotplots on genome scale. & Miyata, T. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. Katoh, K. & Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Chu, C. et al. Comprehensive identification of transposable element insertions using multiple sequencing technologies. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Price, A. L., Eskin, E. & Pevzner, P. A. Whole-genome analysis of Alu repeat elements reveals complex evolutionary history. & Smit, A. F. The Dfam community resource of transposable element families, sequence models, and genome annotations. We thank S. Jankauskiene, M. Lee and T. Nguyen for technical assistance with preparation and sequencing of Strand-seq libraries; C. Steidl for use of the NextSeq550 sequence platform; and T. Brown for edits in the preparation of this manuscript. Library pools were also sequenced on the Element AVITI at the University of California Davis DNA Technologies Core. This research was supported in part by funding from the National Institutes of Health (NIH) grants R01HG002385, R01HG010169 and R01MH101221 (to E.E.E.) is an investigator of the Howard Hughes Medical Institute (HHMI). was funded in part by a program project grant (1074) from the Terry Fox Research Foundation and a research grant (159787) from the Canadian Institutes of Health Research. This research was further supported by funding to H.D. HHMI laboratory heads have previously granted a non-exclusive CC BY 4.0 license to the public and a sublicensable license to HHMI in their research articles. Pursuant to those licences, the author-accepted manuscript of this article can be made freely available under a CC BY 4.0 license immediately on publication. Present address: Altos Labs, San Diego, CA, USA These authors contributed equally: Harriet Dashnow, Thomas A. Sasani, Glennis A. Logsdon, Pille Hallast, Michelle D. Noyes, Zev N. Kronenberg, Tom Mokveld David Porubsky, Glennis A. Logsdon, Michelle D. Noyes, Nidhi Koundinya, William T. Harvey, Jiadong Lin, Sean McGee, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Jordan Knuth, Gage H. Garcia, Joshua D. Smith & Evan E. Eichler Harriet Dashnow, Thomas A. Sasani, Cody J. Steely, Thomas J. Nicholas, Michael E. Goldberg, W. Scott Watkins, Brent S. Pedersen, Hannah C. Happ, Deborah W. Neklason, Lynn B. Jorde & Aaron R. Quinlan The Jackson Laboratory for Genomic Medicine, Farmington, CT, USA Zev N. Kronenberg, Tom Mokveld, Cillian Nolan, Egor Dolzhenko, William J. Rowell, Cairbre Fanslow, Christine Lambert & Michael A. Eberle Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN, USA Blue Marble Space Institute of Science, Seattle, WA, USA Center for Digital Medicine, Heinrich Heine University, Düsseldorf, Germany Terry Fox Laboratory, BC Cancer Agency, Vancouver, British Columbia, Canada Tiffany Y. Leung, Vincent C. T. Hanlon, Daniel D. Chan, Yanni Wang & Peter M. Lansdorp Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY, USA Howard Hughes Medical Institute, University of Washington, Seattle, WA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Generation of de novo assemblies and validation: N.K., W.T.H. is a scientific advisory board member of Variant Bio. C. Lee is a scientific advisory board member of Nabsys and Genome Insight. has previously disclosed a patent application (no. is a private shareholder in Phase Genomics. The other authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Scatterplot of sequence read depth and read length N50 for ONT (blue) and PacBio (PB; magenta) with median coverage (dashed line) and different generations indicated (point shape). b) Scatterplot of the assembly contiguity measured in AuN values for Verkko (brown), hifiasm (UL) (light blue), and hifiasm (light grey) assemblies of G1-G4. c) Top: Total number of Verkko contigs whose maximum aligned bases are within +/−5% of the total T2T-CHM13 chromosome length. *Due to substantial size differences between the T2T-CHM13 Y (haplogroup J1a-L816) and the Y chromosome of this pedigree (haplogroup R1b1a-Z302), three contigs are shown that span the entire male-specific Y region without breaks (i.e., excluding the pseudoautosomal regions). Bottom: Each dot represents a single Verkko contig with the highest number of aligned bases in a given chromosome. d) Chromosomes containing complete telomeres and being spanned by a single contig are annotated as solid squares. We mark centromeres assembled by Verkko (brown), hifiasm (UL) (light blue), or both (green). Alignments transmitted between generations that are >99.99% identical (red) are contrasted with non-transmitted with lower sequence identity (grey). b) T2T recombination between child and parental haplotypes for Chromosome 8. Alignments between the parental and child haplotypes are binned into 500 kbp long bins and coloured based on the percentage of matched bases. Inherited maternal (shades of red) and paternal (shades of blue) segments are marked on top. Black tick marks show positions of mismatches between parental and child haplotypes. c) Distribution of distances of maternal (red) and paternal (blue) recombination breakpoints (G2-G4) to chromosome ends with respect to T2T-CHM13 (histogram bin size: 50). d) Significant association between the number of recombination breaks (y-axis) and parental age (x-axis) shown separately for maternal (red) and paternal (blue) recombination breakpoints (G2-G3) detected with respect to T2T-CHM13. Regression lines were fitted using Poisson GLM with a log link (p = 2.02 × 10−3, 7.88 × 10−4 for parental age and sex effects, respectively). a) The fraction of a parent's germline SNVs (green, DNMs) and postzygotic SNVs (purple, PZMs) transferred to each child. b) The mean allele balance (AB) of DNMs (n = 249) and PZMs (n = 55) across HiFi, Illumina, and ONT data plotted against the fraction of children who inherited a variant are significantly correlated for DNMs (two-sided t-test, p = 0.0084) and PZMs (p = 0.00021). Half of PZMs with AB < 0.25 are transmitted to at least one child (n = 18/36). c) On average, DNMs are transmitted to 50% of children, while PZMs are transmitted to less than 25% of children. d) Number of DNMs and PZMs transmitted to each child in the pedigree. Major components and their structures are shown. b) Deletion of an 18-monomer α-satellite HOR within the Chromosome 6 centromere of G2-NA12878 is inherited in G3-NA12887, shortening the length of the α-satellite HOR array by ~3 kbp. Summary of generated sequencing data and phased genome assemblies. Detected assembly errors in phased genome assemblies. List of putative allelic gene conversion events. Recurrent tandem repeat mutations observed at least twice (T2T-CHM13 coordinates). Reference genomes and annotations used in this study. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Human de novo mutation rates from a four-generation pedigree reference. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08913-3'>Punic people were genetically diverse with almost no Levantine ancestors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 15:02:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Here, we generated genome-wide data for 210 individuals, including 196 from 14 sites traditionally identified as Phoenician and Punic in the Levant, North Africa, Iberia, Sicily, Sardinia and Ibiza, and an early Iron Age individual from Algeria. Levantine Phoenicians made little genetic contribution to Punic settlements in the central and western Mediterranean between the sixth and second centuries bce, despite abundant archaeological evidence of cultural, historical, linguistic and religious links4. However, this was a minority contributor of ancestry in all of the sampled sites, including in Carthage itself. Different Punic sites across the central and western Mediterranean show similar patterns of high genetic diversity. We also detect genetic relationships across the Mediterranean, reflecting shared demographic processes that shaped the Punic world. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Open science principles require making all data used to support the conclusions of a study fully available, and we support these principles here by making publicly available not only the digital copies of molecules (the uploaded sequences) but also the molecular copies (the aDNA libraries themselves, which constitute molecular data storage). Researchers who wish to carry out deeper sequencing of libraries published in this study should make a request to the corresponding author D.R. We commit to granting reasonable requests as long as the libraries remain preserved in our laboratories, with no requirement that we be included as collaborators or co-authors on any resulting publications. The raw DNA sequences for individuals newly sequenced in this study are deposited in the European Nucleotide Archive under accession number PRJEB86313. Their processed genotype data in pseudohaploid eigenstrat format can be obtained from the Harvard Dataverse repository (https://doi.org/10.7910/DVN/UPDESR). We include other newly reported data such as radiocarbon dates and archaeological context information in the Article and its Supplementary Information. The land-sea mask, coastline, lake, river and political boundary data are extracted from the GSHHG datasets (v.2.3.6) using GMT (5.x series). Bondì, S. F., Garbati, G., Botto, M. & Oggiano, I. Fenici e Cartaginesi: Una Civiltà Mediterranea (Istituto poligrafico e Zecca dello Stato, Libreria dello Stato, 2009). Regev, D. Painting the Mediterranean Phoenician: On Canaanite-Phoenician Trade-Nets (Equinox, 2021). Roppa, A., Botto, M. & Van Dommelen, P. Il Mediterraneo Occidentale Dalla Fase Fenicia All'egemonia Cartaginese. Dinamiche Insediative, Forme Rituali e Cultura Materiale Nel V Secolo a. Hoyos, D. Hannibal's Dynasty: Power and Politics in the Western Mediterranean, 247-183 BC (Psychology Press, 2005). Miles, R. Carthage Must Be Destroyed: The Rise and Fall of an Ancient Civilization (National Geographic Books, 2012). van Dommelen, P. A. R. & Bellard, C. G. Rural Landscapes of the Punic World (Equinox, 2008). Prag, J. R. W. in The Punic Mediterranean: Identities and Identification from Phoenician Settlement to Roman Rule (eds Quinn, J. C. & Vella, N. C. E.) 11–23 (Cambridge Univ. Matisoo-Smith, E. et al. A European mitochondrial haplotype identified in ancient Phoenician remains from Carthage, North Africa. Insights into Punic genetic signatures in the southern necropolis of Tharros (Sardinia). Haber, M. et al. Continuity and admixture in the last five millennia of Levantine history from ancient Canaanite and present-day Lebanese genome sequences. Haber, M. et al. A genetic history of the Near East from an aDNA time course sampling eight points in the past 4,000 years. Moots, H. M. et al. A genetic history of continuity and mobility in the Iron Age central Mediterranean. Fu, Q. et al. An early modern human from Romania with a recent Neanderthal ancestor. Fu, Q. et al. A revised timescale for human evolution based on ancient mitochondrial genomes. Rodríguez-Varela, R. et al. Genomic analyses of pre-European conquest human remains from the Canary Islands reveal close affinity to modern North Africans. Massive migration from the steppe was a source for Indo-European languages in Europe. Harney, É., Patterson, N., Reich, D. & Wakeley, J. Assessing the performance of qpAdm: a statistical tool for studying population admixture. Zalloua, P. et al. Identifying genetic traces of historical expansions: Phoenician footprints in the Mediterranean. & Steinrücken, M. Parental relatedness through time revealed by runs of homozygosity in ancient DNA. Waldman, S. et al. Genome-wide data from medieval German Jews show that the Ashkenazi founder event pre-dated the 14th century. Ceballos, F. C., Joshi, P. K., Clark, D. W., Ramsay, M. & Wilson, J. F. Runs of homozygosity: windows into population history and trait architecture. Ancient DNA reveals admixture history and endogamy in the prehistoric Aegean. Growing up in Ancient Sardinia: infant-toddler dietary changes revealed by the novel use of hydrogen isotopes (δ2H). Ancient Rome: a genetic crossroads of Europe and the Mediterranean. Olalde, I. et al. A genetic history of the Balkans from Roman frontier to Slavic migrations. Complete mitochondrial genome sequence of a Middle Pleistocene cave bear reconstructed from ultrashort DNA fragments. & Meyer, M. Extraction of highly degraded DNA from ancient bones, teeth and sediments for high-throughput sequencing. Rohland, N., Harney, E., Mallick, S., Nordenfelt, S. & Reich, D. Partial uracil-DNA-glycosylase treatment for screening of ancient DNA. Gansauge, M.-T., Aximu-Petri, A., Nagel, S. & Meyer, M. Manual and automated preparation of single-stranded DNA libraries for the sequencing of DNA from ancient biological remains and other sources of highly degraded DNA. Rohland, N. et al. Three assays for in-solution enrichment of ancient human DNA at more than a million SNPs. Li, H. & Durbin, R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Behar, D. M. et al. A ‘Copernican' reassessment of the human mitochondrial DNA tree from its root. Weissensteiner, H. et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Rasmussen, M. et al. An Aboriginal Australian genome reveals separate human dispersals into Asia. The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP). & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Lazaridis, I. et al. Genetic origins of the Minoans and Mycenaeans. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Evanno, G., Regnaut, S. & Goudet, J. Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. & Mayrose, I. Clumpak: a program for identifying clustering modes and packaging population structure inferences across K. Mol. A., Liu, K. Z., Liu-Fang, G., Nakka, P. & Ramachandran, S. pong: fast analysis and visualization of latent clusters in population genetic data. & Kelso, J. admixr-R package for reproducible analyses using ADMIXTOOLS. Accurate detection of identity-by-descent segments in human ancient DNA. Kennett, D. J. et al. Archaeogenomic evidence reveals prehistoric matrilineal dynasty. van den Brink, E. C. M. et al. A Late Bronze Age II clay coffin from Tel Shaddud in the Central Jezreel Valley, Israel: context and historical implications. Ancient DNA sheds light on the genetic origins of early Iron Age Philistines. I.G., A.S.-M. and D. Regev were supported by ISF grant number 1045/20. were supported by AGED PRIN 2017 project, MUR Italy. D. Reich was supported by National Institutes of Health grant HG012287; by John Templeton Foundation grant 61220; by the Howard Hughes Medical Institute (HHMI), a gift from J.-F. Clin; by the Allen Discovery Center, a Paul G. Allen Frontiers Group advised program of the Paul G. Allen Family Foundation; and by a grant from the Getty Foundation “The Classical World in Context: the Near East”. We thank V. Moses and M. McCormick for their comments. We thank the Musée de l'Homme for providing us access to the human remains from Khenchela cave. These authors jointly supervised this work: Alfredo Coppa, David Caramelli, Ron Pinhasi, Carles Lalueza-Fox, Ilan Gronau, David Reich Department of Human Evolutionary Biology, Harvard University, Cambridge, MA, USA Harald Ringbauer, Iñigo Olalde, Alissa Mittnik, Iosif Lazaridis, Arie Shaus, Nadin Rohland & David Reich Harald Ringbauer, Alissa Mittnik, Francesco Fontani & David Reich Department of Archaeogenetics, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany Efi Arazi School of Computer Science, Reichman University, Herzliya, Israel BIOMICs Research Group, Department of Zoology and Animal Cell Biology, University of the Basque Country UPV/EHU, Vitoria-Gasteiz, Spain Luca Sineo, Francesco La Pastina, Gabriele Lauria & Giulio Catalano Gioacchino Falsone, Francesco La Pastina, Francesca Meli & Paola Sconzo Joukowsky Institute for Archaeology and the Ancient World, Brown University, Providence, RI, USA Department of Genetics, Harvard Medical School, Boston, MA, USA Iosif Lazaridis, Kim Callan, Elizabeth Curtis, Aisling Kearns, Ann Marie Lawson, Matthew Mah, Adam Micco, Jonas Oppenheimer, Lijun Qiu, Kristin Stewardson, J. Noah Workman, Swapan Mallick, Nadin Rohland & David Reich Archaeological Museum of Ibiza and Formentera, Eivissa, Spain Maria Bofill, Ana Mezquida, Benjamí Costa & Helena Jiménez Department of Data Science, Mount Holyoke College, South Hadley, MA, USA Kim Callan, Elizabeth Curtis, Ann Marie Lawson, Matthew Mah, Jonas Oppenheimer, Lijun Qiu, Kristin Stewardson, Swapan Mallick & David Reich Broad Institute of MIT and Harvard, Cambridge, MA, USA Matthew Mah, Swapan Mallick, Nadin Rohland & David Reich Enrique Viguera, José Suárez Padilla & Sonia López Chamizo Institute of Evolutionary Biology (UPF-CSIC), PRBB, Barcelona, Spain Catalan Institution of Research and Advanced Studies (ICREA), Barcelona, Spain CNAG, Centro Nacional de Analisis Genomico, Barcelona, Spain Bar Ilan University, The Azrieli Faculty of Medicine, Safed, Israel Weizmann Institute of Science, Scientific Archaeology Unit, D-REAMS Radiocarbon Dating Laboratory, Rehovot, Israel Elisabetta Cilli, Anna Chiara Fariselli, Francesco Fontani & Donata Luiselli Department of Law and Digital Society, Unitelma Sapienza, Rome, Italy You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar We annotate author contributions using the CRediT Taxonomy labels; where multiple authors serve in the same role, we specify the degree of contribution as lead, equal or support. Data curation (archaeology and bioanthropology): lead, R.P., C.L.-F., D. Regev, A.C., L.S., D.C., P.v.D. Resources: G.F., M.B., A. Mezquida, B.C., H.J., P. Smith, S.V., A. Modi, K.C., E. Curtis, A.K., A.M.L., M.M., A. Micco, J.O., L.Q., K.S., J.N.W., N.M.-G., A.M.S.R., M.L.L.F., J.M.J.-A., I.J.T.M., E.V., J.S.P., S.L.C., T.M.-B., E.L., A.R.R., F.O., P.T., V.G., A.B., L.C., E.B., M.F., M.L., F.L.P., A.N., F.G., C.D.V., G.L., F.M., P. Sconzo, G.C., E. Cilli, A.C.F., F.F., D.L., B.J.C., N.R. Writing (review and editing): all of the authors. Correspondence to Harald Ringbauer, Ilan Gronau or David Reich. Nature thanks Timothy Jull, Josephine Quinn and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. We project individuals sequenced at over 20,000 SNPs onto the same two PCs as in Fig. We split the sample into panels representing our four major geographic regions: Iberia, Sardinia, North Africa, and Sicily. The 122 Phoenician-Punic individuals sequenced for more than 100,000 SNPs were jointly analysed with 24 individuals from related ancient populations across the Mediterranean (Supplementary Table 12). (a) The model with the highest likelihood was obtained for each value of K among 50 replicate runs. Values of the ΔK score of58 are specified for K = 3,4, with a higher score obtained for K = 3, suggesting optimal fit. Individuals are partitioned within each region according to site and time range (see legend). The unsupervised ADMIXTURE model does not adequately differentiate between Levantine ancestry and ancestry found in other Mediterranean locations (e.g., Anatolia and Sicily), unlike the qpAdm models of Extended Data Fig. We partitioned individuals by region: (a) North Africa, (b) Sicily, (c) Sardinia, (d) Iberia, and (e) the Levant. Within each region, we grouped individuals by site, and for sites in Sicily, Sardinia, and Iberia, also by broad date ranges (see legend for colour code). We did not correct these P-values for multiple testing, but this approach is conservative since we report models with comparatively high P-values (those that are not rejected by the test). Individuals with low coverage (fewer than 100,000 SNPs) are indicated by an asterisk (*) next to the sample ID. Eastern ancestry models are indicated by a contribution of the proxy sources Levant MLBA. In contrast, western ancestry models are indicated by contributions from either Greece BA (Myc), Sicily EBA, Sardinia LBA, Iberia LBA, or Steppe MLBA. There are five individuals for whom no valid eastern or western model was inferred. For four of them, we inferred valid models under the broad ancestry scheme (marked by an asterisk above the vertical bar), and for one (I22122 from Tharros, Sardinia), we could not infer any valid model. We exclude from this analysis the Akhziv sample, the three individuals from Sicily and Sardinia that cluster near Levantine individuals in the 2D PCA, and one individual for which we could not fit a qpAdm model. The two approaches yield similar estimates, with qpAdm being more sensitive to low ancestry proportions. Individuals from Kerkouane (depicted as squares) appear to have a broad range of North African ancestry (0 - 94%). Individuals from Sicily typically have lower proportions of North African ancestry (<20%), and we observe no significant shift in time. On the other hand, in Sardinia, none of the 12 individuals for which we inferred more than 10% North African ancestry (according to at least one of the two approaches) dated before 400 BCE, suggesting that North African ancestry was likely introduced around that time (Supplementary Information S3). We see a similar pattern in Iberia, but since we only have one individual from Iberia dating before 400 BCE, we cannot confidently infer the absence of North African ancestry during this time. We inferred the first four characters of the ISOGG 2019 Y haplogroup classification for all Phoenician and Punic males with more than 100,000 autosomal SNPs covered (as those in almost all cases have sufficient coverage on the Y chromosome; see Methods). (a) Pie chart of Y haplogroup frequencies. (b) We visualize the Y haplogroup diversity partitioned per Phoenician or Punic site and denote each individual's haplotype by one circle. (a) Y haplogroup diversity measured using the Inverse Simpson index. This value is computed as in Fig. 3a, excluding the three Punic individuals (from Kerkouane, Villaricos and Selinunte) with distinct North African Y haplogroups E1a and L (see Extended Data Fig. (b) autosomal diversity measured using the first two PCs from Fig. 1 and the mean pairwise distance of those coordinates. This value is computed as in Fig. 3b, excluding individuals with more than 10% North African ancestry based on qpAdm in Phoenician-Punic sites (see Extended Data Fig. Here, we combined individuals from the nearby Sicilian sites of Birgi, Motya, and Lilybaeum into one group (labelled Lilybaeum here). 3 (without any additional filtering), and the dashed horizontal bar in both panels indicates the maximum diversity observed in sites dating before 500 BCE. See Supplementary Information S5 for a more detailed description of this analysis. We reconstructed two pedigrees based on inferring biological relatives with pairwise kinship (using IBD segment sharing) and uniparental haplogroups: (a) A pedigree linking five individuals from Kerkouane, North Africa; (b) A pedigree linking three individuals from Tharros, Sardinia. In the Kerkouane pedigree in (a), individuals I24215 and I24194 are inferred to be 3rd-4th degree relatives of the two siblings I24494 and I24193, but the exact pedigree relationship cannot be resolved. Both pedigrees contain individuals dating to 800–400 calBCE and link several individuals via the maternal lineage: We infer four identical maternal haplogroups in Kerkouane and a maternal grandfather in Tharros–two observations that are inconsistent with strict patrilocality. We computed runs of homozygosity (ROH) in all individuals with more than 400,000 SNPs covered and recorded the total length (in cM) of ROHs binned by length into four categories (see legend). We label individuals with at least 50 and 100 cM of their genome in long ROH (>20 cM) with triangle and square marks as in30 - to indicate offspring of close biological parental relatives. (a) ROH in Phoenician and Punic individuals, grouped by site. (c) Expected ROH for offspring of various cousin matings (according to the degree of relation between parents) and for individuals sampled in populations with small effective size (calculated as described in30). Ancestry models inferred using qpAdm for individuals from Sicily from (a) the indigenous Iron Age sites of Polizzello and Monte Falcone, (b) from Phoenician sites before Roman expansion (as shown in Extended Data Fig. 3b), and (c) from Punic sites after Roman expansion. We did not correct these P-values for multiple testing, but this approach is conservative since we report models with comparatively high P-values (those that are not rejected by the test). Eastern ancestry models are indicated by a contribution of the proxy source Levant MLBA. In contrast, western ancestry models are indicated by contributions from either Greece BA (Myc), Sicily EBA, Sardinia LBA, Iberia LBA, or Steppe MLBA. There are seven individuals for which no valid eastern or western model was inferred. We inferred valid models under the broad ancestry scheme (marked by an asterisk above the vertical bar) for five of them. Two individuals were inferred to be related through IBD-sharing and are indicated in the figure. The analysis suggests that indigenous populations in Sicily have similar ancestry patterns as observed in the Phoenician sites but without North African ancestry. In later periods, we see the introduction of diverse ancestry sources (Levantine and western Mediterranean), likely associated with the Roman expansion into Sicily. We show the same PCA as in Fig. 1 but focus on the ancient reference populations. (b) Zoom in PCA projections of Levantine populations. We also include additional Bronze and Iron Age Levant individuals not included in Fig. Those previously published individuals originate from Sidon in present-day Lebanon17 and various sites in present-day Israel (Megiddo, Yehud, Hazor, Baq'ah25, Tel Shadud64, Ashkelon64,65). All 13 individuals from Akhziv cluster next to other Levantine individuals, together with a single outlier individual from Tharros (I22119) inferred to have Levantine ancestry (Extended Data Fig. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Ringbauer, H., Salman-Minkov, A., Regev, D. et al. Punic people were genetically diverse with almost no Levantine ancestors. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64564055/stonehenge-fingerprint-origin/'>A Fingerprint Taken From Stonehenge Changes Everything We Know About Its Mystical Origins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 13:48:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Here's how scientists unlocked new hidden secrets of the iconic monument. We may earn commission if you buy from a link. Scholars previously believed that they knew most of what there was to be known about the Altar Stone—the largest of the non-sarsen stones on site, which is now partially buried beneath two fallen stones. “Our analysis found specific mineral grains in the Altar Stone are mostly between 1,000 to 2,000 million years old, while other minerals are around 450 million years old,” Anthony Clarke, lead author and Ph.D. student from the Timescales of Mineral Systems Group at Curtin's School of Earth and Planetary Sciences, said in a statement. “This provides a distinct chemical fingerprint suggesting the stone came from rocks in the Orcadian Basin, Scotland, at least 750 kilometers [466 miles] away from Stonehenge.” According to English Heritage, the Altar Stone is a large slab of greenish Old Red Sandstone. Richard Bevins, study co-author and professor at Aberystwyth University, said in a statement that with the chemical fingerprint tracing the iconic rock to Scotland, the hunt for its exact point of origin starts now. It must have required, the authors claim, an unexpectedly advanced transport method and complex societal organization. “Our discovery of the Altar Stone's origins highlights a significant level of societal coordination during the Neolithic period and helps paint a fascinating picture of prehistoric Britain,” Chris Kirkland, study co-author and Curtin professor, said in a statement. “Transporting such massive cargo overland from Scotland to southern England would have been extremely challenging, indicating a likely marine shipping route along the coast of Britain. This implies long-distance trade networks and a higher level of societal organization than is widely understood to have existed during the Neolithic period in Britain.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Messages Found on the Walls of Last Supper Site The Maya Kingdom Collapsed Due to Burning Events Amateurs Found a Hoard of Ancient Silver Treasure</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a64542602/olo-color-oz/'>Scientists Discovered a New Color. It's Been Right in Front of Us All Along.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Described as a kind of blue-green, the new color—named “olo”—can only be seen using lasers to manipulate certain photoreceptors. The human eye is a wonder of evolution, and is our primary window into understanding the known universe that surrounds us. And even in this small visible sliver of the electromagnetic pie, some colors remain hidden from our view—until now. “The ultimate goal is to provide programmable control over every photoreceptor [light-sensing cell] in the retina,” Fong told Live Science. (Some people, known as tetrachromats, actually have a fourth rod that sensitive to orange, but that's a story for another time). The result, according to the researchers, was a color that was a type of blue-green no human had ever seen. “They describe the color as blue-green of unprecedented saturation.” Although some scientists remain unconvinced that “olo” is indeed a new color—one vision scientist told The Guardian that the work has “limited value”—Fong and his team hope that further exploration of the retina using Oz could help treat color blindness and other retinal diseases, including retinitis pigmentosa. The tool could also be used to simulate what it would be like to be one of those genetically mutated women (it's mostly women) that have four distinct cones and can see millions more colors than the average human. For as much as we think we know when it comes to color, evidently there's more than meets the eye. Humans Could Grow New Teeth in Just a Few Years The Secrets of Queen Bees Could Help Humanity We Totally Missed a Big Part of Our Immune System This Is the Secret to Being a Supercentenarian Humans May Be Able to Grow New Teeth in 6 Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250422190639.htm'>Light fields with extraordinary structure: Plasmonic skyrmion bags</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 01:35:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A research group at the University of Stuttgart has manipulated light through its interaction with a metal surface so that it exhibits entirely new properties. The researchers have published their findings in Nature Physics. Skyrmions are a mathematical description of vortex-like structures that help researchers better understand fundamental physical relationships. In recent years, this theoretical concept has been confirmed experimentally across a wide range of areas, including magnetic solids and material surfaces. Giessen's group has now investigated whether light impinging on the structured surface of a thin gold layer can be made to behave like skyrmion bags that follow specific symmetries. These bags consist of skyrmions contained within a larger skyrmion. "We then observed a superposition of two skyrmion light fields, from which the skyrmion bags formed," explains Julian Schwab, lead author of the publication and doctoral student in Giessen's research group. However, these light-field skyrmions exhibit extraordinary properties, thereby sparking researchers' imagination in terms of potential technical applications. Whether the gold surface used by Giessen's team is suitable for this purpose remains to be seen. "If someone finds a suitable material, our concept could be applied in microscopy," states Giessen. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250422155932.htm'>New species of electricity-conducting organism, name it after Tribe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 01:13:42
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists have identified a novel species of bacteria that acts as electrical wiring, potentially ushering in a new era of bioelectronic devices for use in medicine, industry, food safety, and environmental monitoring and cleanup. Findings were published today in Applied and Environmental Microbiology. Cable bacteria consist of rod-shaped cells attached end to end with a shared outer membrane, forming filaments that can reach several centimeters in length. Their electrical conductivity, unusual among bacteria, is an adaptation that optimizes their metabolic processes in the sediment environments in which they live. "This new species seems to be a bridge, an early branch within the Ca. Electrothrix clade, which suggests it could provide new insights into how these bacteria evolved and how they might function in different environments," said Li, who in June will return to Oregon State as an assistant professor in the College of Agricultural Sciences following a stint on the faculty of James Madison University. "It stands out from all other described cable bacteria species in terms of its metabolic potential, and it has distinctive structural features, including pronounced surface ridges, up to three times wider than those seen in other species, that house highly conductive fibers made of unique, nickel-based molecules." The bacteria's ability to participate in reduction-oxidation reactions over significant distances gives it a key role in sediment geochemistry and nutrient cycling. "These bacteria can transfer electrons to clean up pollutants, so they could be used to remove harmful substances from sediments," Li said. "Also, their design of a highly conductive nickel protein can possibly inspire new bioelectronics." Cable bacteria can live under diverse climatic conditions and are found in various environments, including both freshwater and saltwater sediments. Electrothrix yaqonensis draws its name from the Yaqona people, whose ancestral lands encompassed Yaquina Bay. Yaqona referred to the bay and river that made up much of their homeland, as well as to the people themselves. "Naming an ecologically important bacterium after a Tribe recognizes its historical bond with the land and acknowledges its enduring contributions to ecological knowledge and sustainability," Li said. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/04/250422155820.htm'>The most effective prevention method for complications post lung transplant</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-23 00:15:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When receiving a lung transplant, one of the most important complications to look out for is chronic lung allograft dysfunction, known as CLAD. Preventing this complication for those who receive a lung transplant is of the highest priority, as there are no universally effective treatments for CLAD once it is established. Chronic lung allograft dysfunction is a label for a wide range of clinical manifestations that all lead to the transplanted lung losing its normal function. In the absence of effective treatments, strategies to prevent CLAD are crucial. Using the International Society for Heart and Lung Transplantation Thoracic Organ Transplant Registry data, a research team led by Michael Combs, M.D., M.S., an assistant professor of pulmonary diseases and internal medicine at Michigan Medicine, conducted the first study to show a survival benefit between using the treatments tacrolimus versus cyclosporine after lung transplantation. Out of the 22,222 individuals with data for chronic lung allograft dysfunction treatment, 88.6% received tacrolimus immediate release. The participants taking immediate release tacrolimus had a much lower rate of experiencing chronic lung allograft dysfunction than their counterparts that took twice-daily cyclosporine. "This present study should reassure transplant patients and providers twice-daily tacrolimus -- and not only once-daily tacrolimus -- is the superior treatment to cyclosporine." "Importantly," Combs added, "in our study we found that twice-daily tacrolimus not only resulted in lower rates of CLAD relative to cyclosporine, but it was also associated with improved overall survival after lung transplantation. This is an important, patient-centered finding which has not been previously demonstrated." "Given the theoretical benefits of once-daily medication regiments, future research will need to investigate if either tacrolimus XR and tacrolimus IR are superior to each other. However, until then, we can rest assured knowing that tacrolimus -- regardless of its formulation -- is the best option for our lung transplant patients," he stated. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            