
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-11</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41573-025-01273-7'>Synthetic lethality in cancer drug discovery: challenges and opportunities</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 15:33:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Drug Discovery (2025)Cite this article Synthetic lethality, first proposed more than two decades ago, has long held immense promise for targeted cancer therapy. Although the clinical success of PARP inhibition in BRCA-mutant cancers stands as proof of concept, few other synthetic lethal interactions have been translated from preclinical findings into effective therapies. This slow pace of translation stems in part from the difficulty of developing drugs against genetic dependencies, but also reflects the cell- and tissue-specific nature of these interactions. In this Review, we outline recent advances in the discovery and validation of synthetic lethal pairs, from their discovery in large-scale genetic screens to the development of drugs for the clinic. We discuss how alternative CRISPR-based approaches — including combinatorial screens, base editing and saturation mutagenesis — are now being used to discover new tractable interactions. We also examine how machine learning models can enable prioritization of candidate pairs and the identification of biomarkers for patient stratification. Finally, we highlight alternative phenotypic readouts, such as high-content imaging and single-cell profiling, which enable the dissection of phenotypes beyond simple cell growth or fitness. Together, these developments are refining the synthetic lethality paradigm and advancing its potential for cancer therapy. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Hartwell, L. H., Szankasi, P., Roberts, C. J., Murray, A. W. & Friend, S. H. Integrating genetic approaches into the discovery of anticancer drugs. Google Scholar Farmer, H. et al. Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy. Google Scholar Prakash, R., Zhang, Y., Feng, W. & Jasin, M. Homologous recombination and human health: the roles of BRCA1, BRCA2, and associated proteins. Cold Spring Harb. Google Scholar Pettitt, S. J. et al. Clinical BRCA1/2 reversion analysis identifies hotspot mutations and predicted neoantigens associated with therapy resistance. Cancer Discov. Google Scholar Tutt, A. N. J. et al. Adjuvant olaparib for patients with BRCA1- or BRCA2-mutated breast cancer. Google Scholar Langelier, M.-F., Lin, X., Zha, S. & Pascal, J. M. Clinical PARP inhibitors allosterically induce PARP2 retention on DNA. Google Scholar The genotype-tissue expression (GTEx) project. Google Scholar Jiang, L. et al. A quantitative proteome map of the human body. Google Scholar Pan-Cancer Analysis of Whole Genomes Consortium. Pan-cancer analysis of whole genomes. Google Scholar Gonçalves, E. et al. Pan-cancer proteomic map of 949 human cell lines. Google Scholar Chen, S. et al. A genomic mutational constraint map using variation in 76,156 human genomes. Google Scholar Pacini, C. et al. A comprehensive clinically informed map of dependencies in cancer cells and framework for target prioritization. This article provides a recent systematic analysis of genetic dependencies in 930 cancer cell lines, linking biomarkers with potential therapeutic targets and analysing the prevalence of these biomarkers in patient populations. Google Scholar Barretina, J. et al. The cancer cell line encyclopedia enables predictive modelling of anticancer drug sensitivity. Google Scholar Garnett, M. J. et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Google Scholar Ghandi, M. et al. Next-generation characterization of the cancer cell line encyclopedia. Google Scholar Behan, F. M. et al. Prioritization of cancer therapeutic targets using CRISPR–Cas9 screens. Google Scholar Chen, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. References 16 and 17 both show for the first time using CRISPR–Cas9 screens that WRN is synthetically lethal in MSI cancers. Google Scholar Picco, G. et al. Werner helicase is a synthetic-lethal vulnerability in mismatch repair-deficient colorectal cancer refractory to targeted therapies, chemotherapy and immunotherapy. Google Scholar US National Library of Medicine. Adams, D. & McDermott, U. Editorial overview: cancer genomics: kill it. Kill it dead. Google Scholar O'Neil, N. J., Bailey, M. L. & Hieter, P. Synthetic lethality and cancer. Google Scholar & Lord, C. J. Synthetic lethal therapies for cancer: what's next after PARP inhibitors? Google Scholar Huang, A., Garraway, L. A., Ashworth, A. & Weber, B. Synthetic lethality as an engine for cancer drug target discovery. Google Scholar Setton, J. et al. Synthetic lethality in cancer therapeutics: the next generation. Google Scholar Schäffer, A. A., Chung, Y., Kammula, A. V., Ruppin, E. & Lee, J. S. A systematic analysis of the landscape of synthetic lethality-driven precision oncology. This article provides a systematic analysis of SL therapies in clinical trials, revealing that most of these therapies involve genes in the DDR and that success rates for SL trials are generally higher than for non-SL therapies. Google Scholar Ryan, C. J., Mehta, I., Kebabci, N. & Adams, D. J. Targeting synthetic lethal paralogs in cancer. Trends Cancer Res. Google Scholar Ryan, C. J., Devakumar, L. P. S., Pettitt, S. J. & Lord, C. J. Complex synthetic lethality in cancer. Google Scholar Ross, J. S. et al. The HER-2 receptor and breast cancer: ten years of targeted anti-HER-2 therapy and personalized medicine. Google Scholar Chapman, P. B. et al. Improved survival with vemurafenib in melanoma with BRAF V600E mutation. Google Scholar & Joe, A. Oncogene addiction. Cancer Res. Google Scholar Dang, C. V., Reddy, E. P., Shokat, K. M. & Soucek, L. Drugging the ‘undruggable' cancer targets. Google Scholar Wang, J. et al. SynLethDB 2.0: a web-based knowledge graph database on synthetic lethality for novel anticancer drug discovery. Google Scholar Gökbağ, B. et al. SLKB: synthetic lethality knowledge base. Nucleic Acids Res. Google Scholar Zatreanu, D. et al. Polθ inhibitors elicit BRCA-gene synthetic lethality and target PARP inhibitor resistance. Google Scholar Gravells, P., Grant, E., Smith, K. M., James, D. I. & Bryant, H. E. Specific killing of DNA damage-response deficient cells with inhibitors of poly(ADP-ribose) glycohydrolase. DNA Repair. Google Scholar Yap, T. A. et al. First-in-human phase I trial of the oral first-in-class ubiquitin specific peptidase 1 (USP1) inhibitor KSQ-4279 (KSQi), given as single agent (SA) and in combination with olaparib (OLA) or carboplatin (CARBO) in patients (pts) with advanced solid tumors, enriched for deleterious homologous recombination repair (HRR) mutations. Google Scholar Simoneau, A. Abstract B054: TNG348, a selective USP1 inhibitor, shows strong preclinical combination activity with PARP inhibitors and other agents targeting DNA repair. Google Scholar Ryan, C. J., Bajrami, I. & Lord, C. J. Synthetic lethality and cancer - penetrance as the major barrier. Trends Cancer Res. Google Scholar Downward, J. RAS synthetic lethal screens revisited: still seeking the elusive prize? Cancer Res. Google Scholar Ku, A. Integration of multiple biological contexts reveals principles of synthetic lethality that affect reproducibility. Google Scholar Wang, Y. et al. Critical role for transcriptional repressor Snail2 in transformation by oncogenic RAS in colorectal carcinoma cells. Google Scholar Luo, J. et al. A genome-wide RNAi screen identifies multiple synthetic lethal interactions with the Ras oncogene. Google Scholar Lord, C. J., Quinn, N. & Ryan, C. J. Integrative analysis of large-scale loss-of-function screens identifies robust cancer-associated genetic interactions. Google Scholar Thompson, N. A. et al. Combinatorial CRISPR screen identifies fitness effects of gene paralogues. Google Scholar Martin, T. D. et al. A role for mitochondrial translation in promotion of viability in K-Ras mutant cells. Google Scholar Adam, S. et al. The CIP2A-TOPBP1 axis safeguards chromosome stability and is a synthetic lethal target for BRCA-mutated cancer. Google Scholar Noordermeer, S. M. et al. The shieldin complex mediates 53BP1-dependent DNA repair. Google Scholar Tsujino, T. et al. CRISPR screens reveal genetic determinants of PARP inhibitor sensitivity and resistance in prostate cancer. Google Scholar Bouwman, P. et al. 53BP1 loss rescues BRCA1 deficiency and is associated with triple-negative and BRCA-mutated breast cancers. Google Scholar Harvey-Jones, E. et al. Longitudinal profiling identifies co-occurring BRCA1/2 reversions, TP53BP1, RIF1 and PAXIP1 mutations in PARP inhibitor-resistant advanced breast cancer. Google Scholar Wang, Y. et al. The BRCA1-Δ11q alternative splice isoform bypasses germline mutations and promotes therapeutic resistance to PARP inhibition and cisplatin. Cancer Res. Google Scholar Kim, H. S. et al. Systematic identification of molecular subtype-selective vulnerabilities in non-small-cell lung cancer. Google Scholar Herken, B. W., Wong, G. T., Norman, T. M. & Gilbert, L. A. Environmental challenge rewires functional connections among human genes. Preprint at bioRxiv https://doi.org/10.1101/2023.08.09.552346 (2023). Cheung, H. W. et al. Systematic investigation of genetic vulnerabilities across cancer cell lines reveals lineage-specific dependencies in ovarian cancer. Google Scholar Hart, T. et al. High-resolution CRISPR screens reveal fitness genes and genotype-specific cancer liabilities. Google Scholar Tsherniak, A. et al. Defining a cancer dependency map. Seminal paper introducing the concept of a Cancer Dependency Map. Google Scholar Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells. Google Scholar Spradlin, J. N., Zhang, E. & Nomura, D. K. Reimagining druggability using chemoproteomic platforms. Google Scholar Ochoa, D. et al. The next-generation open targets platform: reimagined, redesigned, rebuilt. Nucleic Acids Res. Google Scholar Finan, C. et al. The druggable genome and support for target identification and validation in drug development. Google Scholar Helming, K. C. et al. ARID1B is a specific vulnerability in ARID1A-mutant cancers. Google Scholar Chory, E. J. et al. Chemical inhibitors of a selective SWI/SNF function synergize with ATR inhibition in cancer cell killing. ACS Chem. Google Scholar Békés, M., Langley, D. R. & Crews, C. M. PROTAC targeted protein degraders: the past is prologue. Google Scholar Sasso, J. M. et al. Molecular glues: the adhesive connecting targeted protein degradation to the clinic. Google Scholar Schneider, M. et al. The PROTACtable genome. This Review provides a systematic effort to identify proteins that might be druggable using PROTAC approaches, identifying 1,336 promising candidates. Google Scholar Cantley, J. et al. Selective PROTAC-mediated degradation of SMARCA2 is efficacious in SMARCA4 mutant cancers. Google Scholar Kofink, C. et al. A selective and orally bioavailable VHL-recruiting PROTAC achieves SMARCA2 degradation in vivo. Google Scholar Tejwani, V. et al. PROTAC-mediated conditional degradation of the WRN helicase as a potential strategy for selective killing of cancer cells with microsatellite instability. Google Scholar Chang, Y. et al. Development of an orally bioavailable CDK12/13 degrader and induction of synthetic lethality with AKT pathway inhibition. Cell Rep. Med. Google Scholar Muller, F. L. et al. Passenger deletions generate therapeutic vulnerabilities in cancer. Google Scholar Yoshihama, Y. et al. Potent and selective PTDSS1 inhibitors induce collateral lethality in cancers with PTDSS2 deletion. Cancer Res. Google Scholar van der Lelij, P. et al. Synthetic lethality between the cohesin subunits STAG1 and STAG2 in diverse cancer contexts. Google Scholar Benedetti, L., Cereda, M., Monteverde, L., Desai, N. & Ciccarelli, F. D. Synthetic lethal interaction between the tumour suppressor STAG2 and its paralog STAG1. Google Scholar Leonard, P. G. et al. SF2312 is a natural phosphonate inhibitor of enolase. Google Scholar Lin, Y.-H. et al. An enolase inhibitor for the targeted treatment of ENO1-deleted cancers. This article describes the development of a paralogue-specific inhibitor for ENO2 that exploits a collateral lethality associated with ENO1 deletion. Google Scholar Herencia-Ropero, A. et al. The PARP1 selective inhibitor saruparib (AZD5305) elicits potent and durable antitumor activity in patient-derived BRCA1/2-associated cancer models. Google Scholar Johannes, J. W. et al. Discovery of 5-{4-[(7-Ethyl-6-oxo-5,6-dihydro-1,5-naphthyridin-3-yl)methyl]piperazin-1-yl}-N-methylpyridine-2-carboxamide (AZD5305): a PARP1-DNA trapper with high selectivity for PARP1 over PARP2 and other PARPs. Google Scholar Dvorak, V. et al. Paralog-dependent isogenic cell assay cascade generates highly selective SLC16A3 inhibitors. Google Scholar Gonçalves, E. et al. Drug mechanism-of-action discovery through the integration of pharmacological and CRISPR screens. Google Scholar Vangamudi, B. et al. The SMARCA2/4 ATPase domain surpasses the bromodomain as a drug target in SWI/SNF-mutant cancers: insights from cDNA rescue and PFI-3 inhibitor studies. Cancer Res. Google Scholar Murai, J. et al. Trapping of PARP1 and PARP2 by clinical PARP inhibitors. Cancer Res. Google Scholar Krill-Burger, J. M. et al. Partial gene suppression improves identification of cancer vulnerabilities when CRISPR-Cas9 knockout is pan-lethal. Genome Biol. Google Scholar Steckel, M. et al. Determination of synthetic lethal interactions in KRAS oncogene-dependent cancer cells reveals novel therapeutic targeting strategies. Google Scholar Marjon, K. et al. MTAP deletions in cancer create vulnerability to targeting of the MAT2A/PRMT5/RIOK1 axis. Google Scholar Mavrakis, K. J. et al. Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5. Google Scholar Kryukov, G. V. et al. MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells. References 84, 85 and 86 present a series of studies all demonstrating, using multiple models and gene perturbation approaches, that homozygous deletion of MTAP is associated with increased sensitivity to PRMT5. Google Scholar Cottrell, K. M. et al. MTA-cooperative PRMT5 inhibitors: mechanism switching through structure-based design. Google Scholar Smith, C. R. et al. Fragment-based discovery of MRTX1719, a synthetic lethal inhibitor of the PRMT5•MTA complex for the treatment of MTAP-deleted cancers. Google Scholar Cottrell, K. M. et al. Discovery of TNG908: a selective, brain penetrant, MTA-cooperative PRMT5 inhibitor that is synthetically lethal with MTAP-deleted cancers. Google Scholar Rodon, J. et al. First-in-human study of AMG 193, an MTA-cooperative PRMT5 inhibitor, in patients with MTAP-deleted solid tumors: results from phase I dose exploration. Google Scholar Engstrom, L. D. et al. MRTX1719 is an MTA-cooperative PRMT5 inhibitor that exhibits synthetic lethality in preclinical models and patients with MTAP-deleted cancer. Google Scholar Chang, L., Ruiz, P., Ito, T. & Sellers, W. R. Targeting pan-essential genes in cancer: challenges and opportunities. Google Scholar Cacheiro, P. et al. Human and mouse essentiality screens as a resource for disease gene discovery. Google Scholar Groza, T. et al. The International mouse phenotyping consortium: comprehensive knockout phenotyping underpinning the study of human disease. Nucleic Acids Res. Google Scholar & Clevers, H. Organoids in cancer research. Google Scholar Ringel, T. et al. Genome-scale CRISPR screening in human intestinal organoids identifies drivers of TGF-β resistance. Google Scholar LaFargue, C. J., Dal Molin, G. Z., Sood, A. K. & Coleman, R. L. Exploring and comparing adverse events between PARP inhibitors. Google Scholar Simoneau, A. et al. Characterization of TNG348: a selective, allosteric USP1 inhibitor that synergizes with PARP inhibitors in tumors with homologous recombination deficiency. Google Scholar Burris, H. A. et al. A phase I study of ATR inhibitor gartisertib (M4344) as a single agent and in combination with carboplatin in patients with advanced solid tumours. Google Scholar Ashburn, T. T. & Thor, K. B. Drug repositioning: identifying and developing new uses for existing drugs. Google Scholar Corsello, S. M. et al. Discovering the anticancer potential of non-oncology drugs by systematic viability profiling. Google Scholar Bland, P. et al. SF3B1 hotspot mutations confer sensitivity to PARP inhibition by eliciting a defective replication stress response. Google Scholar Lappin, K. M. et al. Cancer-associated SF3B1 mutations confer a BRCA-like cellular phenotype and synthetic lethality to PARP inhibitors. Cancer Res. Google Scholar Brenner, J. C. et al. PARP-1 inhibition as a targeted strategy to treat Ewing's sarcoma. Cancer Res. Google Scholar Bajrami, I. et al. E-Cadherin/ROS1 inhibitor synthetic lethality in breast cancer. Google Scholar Shaw, A. T. et al. Crizotinib in ROS1-rearranged non-small-cell lung cancer. Google Scholar Han, K. et al. Synergistic drug combinations for cancer identified in a CRISPR screen for pairwise genetic interactions. This article provides an early example of rational selection of gene pairs for screening in a combinatorial CRISPR screen, focusing on known drug targets. Google Scholar Najm, F. J. et al. Orthologous CRISPR-Cas9 enzymes for combinatorial genetic screens. Google Scholar Dede, M., McLaughlin, M., Kim, E. & Hart, T. Multiplex enCas12a screens detect functional buffering among paralogs otherwise masked in monogenic Cas9 knockout screens. Genome Biol. Google Scholar Esmaeili Anvar, N. et al. Efficient gene knockout and genetic interaction screening using the in4mer CRISPR/Cas12a multiplex knockout platform. Google Scholar Yan, W. X. et al. Cas13d is a compact RNA-targeting type VI CRISPR effector positively modulated by a WYL-domain-containing accessory protein. Google Scholar Aregger, M. et al. Systematic mapping of genetic interactions for de novo fatty acid synthesis identifies C12orf49 as a regulator of lipid metabolism. This article reports the results from a CRISPR screen, using Cas12a, to identify synthetic lethality between human paralogue pairs across three cell lines. Google Scholar DeWeirdt, P. C. et al. Optimization of AsCas12a for combinatorial genetic screens in human cells. Google Scholar Hsiung, C. C.-S. et al. Engineered CRISPR-Cas12a for higher-order combinatorial chromatin perturbations. Google Scholar Boettcher, M. et al. Dual gene activation and knockout screen reveals directional dependencies in genetic networks. Google Scholar Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Google Scholar Costanzo, M. et al. A global genetic interaction network maps a wiring diagram of cellular function. Google Scholar Ryan, C. J. et al. Hierarchical modularity and the evolution of genetic interactomes across species. Google Scholar De Kegel, B., Quinn, N., Thompson, N. A., Adams, D. J. Comprehensive prediction of robust synthetic lethality between paralog pairs in cancer cell lines. Cell Syst. This study describes the development of a machine learning approach to predict synthetic lethality between all human paralogue pairs and demonstrates that this model can predict the results of combinatorial CRISPR screens. Google Scholar Schuldiner, M. et al. Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile. Google Scholar Collins, S. R. et al. Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map. Google Scholar Simpson, D., Ling, J., Jing, Y. & Adamson, B. Mapping the genetic interaction network of PARP inhibitor response. Preprint at bioRxiv https://doi.org/10.1101/2023.08.19.553986 (2023). Fielden, J. et al. Comprehensive interrogation of synthetic lethality in the DNA damage response. Google Scholar Wong, S. L. et al. Combining biological networks to predict genetic interactions. Google Scholar Picco, G. et al. Novel WRN helicase inhibitors selectively target microsatellite-unstable cancer cells. Google Scholar Komor, A. C., Kim, Y. B., Packer, M. S., Zuris, J. & Liu, D. R. Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Google Scholar Anzalone, A. V. et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Google Scholar Dace, P. & Findlay, G. M. Reducing uncertainty in genetic testing with saturation genome editing. Google Scholar Waters, A. J. et al. Saturation genome editing of BAP1 functionally classifies somatic and germline variants. Google Scholar Olvera-León, R. et al. High-resolution functional mapping of RAD51C by saturation genome editing. Google Scholar Coelho, M. A. et al. Base editing screens define the genetic landscape of cancer drug resistance mechanisms. Google Scholar Cagiada, M. et al. Discovering functionally important sites in proteins. Google Scholar Kelley, R. & Ideker, T. Systematic interpretation of genetic interactions using protein networks. Google Scholar Tischler, J., Lehner, B. & Fraser, A. G. Evolutionary plasticity of genetic interaction networks. Google Scholar Horn, T. et al. Mapping of signaling networks through synthetic genetic interaction analysis by RNAi. Google Scholar Troyanskaya, O. G., Dolinski, K., Owen, A. B., Altman, R. B. & Botstein, D. A Bayesian framework for combining heterogeneous data sources for gene function prediction (in Saccharomyces cerevisiae). Google Scholar Srivas, R. et al. A network of conserved synthetic lethal interactions for exploration of precision cancer therapy. This article provides a systematic analysis of SL interactions conserved from yeast to humans, demonstrating that this conservation is, at least to some extent, predictable using a machine learning approach. Google Scholar Nilsson, A. & Nielsen, J. Genome scale metabolic modeling of cancer. Google Scholar Zomorrodi, A. R. & Maranas, C. D. Improving the iMM904 S. cerevisiae metabolic model using essentiality and synthetic lethality data. Google Scholar Szappanos, B. et al. An integrated approach to characterize genetic interaction networks in yeast metabolism. Google Scholar van Pel, D. M. et al. An evolutionarily conserved synthetic lethal interaction network identifies FEN1 as a broad-spectrum target for anticancer therapeutic development. PLoS Genet. Google Scholar Tosti, E. et al. Evolutionarily conserved genetic interactions with budding and fission yeast MutS identify orthologous relationships in mismatch repair-deficient cancer cells. Google Scholar Mengwasser, K. E. et al. Genetic screens reveal FEN1 and APEX2 as BRCA2 synthetic lethal targets. Google Scholar Guo, E. et al. FEN1 endonuclease as a therapeutic target for human cancers with defects in homologous recombination. Google Scholar Yu, L. et al. Chemical-genetic profiling of imidazo[1,2-a]pyridines and -pyrimidines reveals target pathways conserved between yeast and human cells. PLoS Genet. Google Scholar Shoemaker, R. H. The NCI60 human tumour cell line anticancer drug screen. Google Scholar Trastulla, L., Noorbakhsh, J., Vazquez, F., McFarland, J. & Iorio, F. Computational estimation of quality and clinical relevance of cancer cell lines. Google Scholar Boehm, J. S. et al. Cancer research needs a better map. Google Scholar Ben-David, U. et al. Genetic and transcriptional evolution alters cancer cell line drug response. Google Scholar Iorio, F. et al. A landscape of pharmacogenomic interactions in cancer. Google Scholar Jerby-Arnon, L. et al. Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. Google Scholar Sinha, S. et al. Systematic discovery of mutation-specific synthetic lethals by mining pan-cancer human primary tumor data. Google Scholar Lee, J. S. et al. Harnessing synthetic lethality to predict the response to cancer treatment. Google Scholar Lee, J. S. et al. Synthetic lethality-mediated precision oncology via the tumor transcriptome. Google Scholar Barrena, N., Valcárcel, L. V., Olaverri-Mendizabal, D., Apaolaza, I. & Planes, F. J. Synthetic lethality in large-scale integrated metabolic and regulatory network models of human cells. NPJ Syst. Google Scholar Zhang, C. & Hua, Q. Applications of genome-scale metabolic models in biotechnology and systems medicine. Google Scholar Folger, O. et al. Predicting selective drug targets in cancer through metabolic networks. Google Scholar Frezza, C. et al. Haem oxygenase is synthetically lethal with the tumour suppressor fumarate hydratase. Reference 157 demonstrates the use of metabolic flux balance modelling to predict new SL targets in cancer, predicting haem oxygenase to be SL with the tumour suppressor fumarate hydratase. These results are subsequently confirmed in reference 158. Google Scholar Robinson, J. L. et al. An atlas of human metabolism. Google Scholar Alzoubi, D., Desouki, A. & Lercher, M. J. Flux balance analysis with or without molecular crowding fails to predict two thirds of experimentally observed epistasis in yeast. Google Scholar Kryazhimskiy, S. Emergence and propagation of epistasis in metabolic networks. Google Scholar Seale, C., Tepeli, Y. & Gonçalves, J. P. Overcoming selection bias in synthetic lethality prediction. Google Scholar Feng, Y. et al. Benchmarking machine learning methods for synthetic lethality prediction in cancer. This article describes a systematic effort to benchmark machine learning methods to predict synthetic lethality, demonstrating that performance is highly dependent on the source of ‘true-positive' SLs. Google Scholar Zamanighomi, M. et al. GEMINI: a variational Bayesian approach to identify genetic interactions from combinatorial CRISPR screens. Genome Biol. Google Scholar Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Google Scholar Kryshtafovych, A., Schwede, T., Topf, M., Fidelis, K. & Moult, J. Critical assessment of methods of protein structure prediction (CASP)-round XIV. Google Scholar Guo, J., Liu, H. & Zheng, J. SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. Nucleic Acids Res. Google Scholar Cell painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes. Google Scholar Norman, T. M. et al. Exploring genetic interaction manifolds constructed from rich single-cell phenotypes. Google Scholar Way, G. P. et al. Predicting cell health phenotypes using image-based morphology profiling. Google Scholar Lotfollahi, M. et al. Predicting cellular responses to complex perturbations in high-throughput screens. Google Scholar Ji, Y. et al. Scalable and universal prediction of cellular phenotypes. Preprint at bioRxiv https://doi.org/10.1101/2024.08.12.607533 (2024). Regev, A. et al. The human cell atlas. Google Scholar Zhang, J. et al. Tahoe-100M: a Giga-scale single-cell perturbation atlas for context-dependent gene function and cellular modeling. Preprint at bioRxiv https://doi.org/10.1101/2025.02.20.639398 (2025). Replogle, J. M. et al. Mapping information-rich genotype-phenotype landscapes with genome-scale Perturb-seq. Google Scholar Huang, A. C. et al. X-atlas/Orion: genome-wide Perturb-seq datasets via a scalable fix-cryopreserve platform for training dose-dependent biological foundation models. Preprint at bioRxiv https://doi.org/10.1101/2025.06.11.659105 (2025). Carpenter, A. E. et al. CellProfiler: image analysis software for identifying and quantifying cell phenotypes. Genome Biol. Google Scholar Caicedo, J. C. et al. Data-analysis strategies for image-based cell profiling. Google Scholar Bock, C. et al. High-content CRISPR screening. Google Scholar Heigwer, F. et al. A global genetic interaction network by single-cell imaging and machine learning. Cell Syst. This article demonstrates in fruit fly cells how high-content imaging can be combined with genetic interaction screening to reveal interactions that impact phenotypes beyond growth. Google Scholar Tunyasuvunakool, K. et al. Highly accurate protein structure prediction for the human proteome. Google Scholar Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Google Scholar Baek, M. et al. Accurate prediction of protein structures and interactions using a three-track neural network. Google Scholar Baek, M. et al. Accurate prediction of protein-nucleic acid complexes using RoseTTAFoldNA. Cagiada, M., Thomasen, F. E., Ovchinnikov, S., Deane, C. M. & Lindorff-Larsen, K. AF2χ: predicting protein side-chain rotamer distributions with AlphaFold2. Preprint at bioRxiv https://doi.org/10.1101/2025.04.16.649219 (2025). Borkakoti, N. & Thornton, J. M. AlphaFold2 protein structure prediction: implications for drug discovery. Terwilliger, T. C. et al. AlphaFold predictions are valuable hypotheses and accelerate but do not replace experimental structure determination. Zhavoronkov, A. et al. Deep learning enables rapid identification of potent DDR1 kinase inhibitors. Munson, B. P. et al. De novo generation of multi-target compounds using deep generative chemistry. Tang, X. et al. A survey of generative AI for de novo drug design: new frontiers in molecule and protein generation. Gangwal, A. & Lavecchia, A. Unleashing the power of generative AI in drug discovery. Olivieri, M. et al. A genetic map of the response to DNA damage in human cells. Davoodi, S., Daryaee, F., Chang, A., Walker, S. G. & Tonge, P. J. Correlating drug-target residence time and post-antibiotic effect: insight into target vulnerability. ACS Infect. Price, S. et al. A suspension technique for efficient large-scale cancer organoid culturing and perturbation screens. Okines, A. et al. Abstract P2-07-24: results from the phase II study of ROS1 targeting with crizotinib in advanced E-cadherin negative lobular breast cancer (ROLo). Cancer Res. Ahn, D. H. et al. Onvansertib in combination with chemotherapy and bevacizumab in second-line treatment of KRAS-mutant metastatic colorectal cancer: a single-arm, phase II trial. Wessels, H.-H. et al. Efficient combinatorial targeting of RNA transcripts in single cells with Cas13 RNA Perturb-seq. & Liu, D. R. Prime editing for precise and highly versatile genome manipulation. Ross, D. T. et al. Systematic variation in gene expression patterns in human cancer cell lines. Bairoch, A. The cellosaurus, a cell-line knowledge resource. Holbeck, S. L., Collins, J. M. & Doroshow, J. H. Analysis of food and drug administration-approved anticancer agents in the NCI60 panel of human tumor cell lines. Yang, W. et al. Genomics of drug sensitivity in cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells. Nucleic Acids Res. Garcia-Alonso, L. et al. Transcription factor activities enhance markers of drug sensitivity in cancer. Cancer Res. Li, J. et al. Characterization of human cancer cell lines reverse-phase protein arrays. Roumeliotis, T. I. et al. Genomic determinants of protein abundance variation in colorectal cancer cells. Nusinow, D. P. et al. Quantitative proteomics of the cancer cell line encyclopedia. Li, H. et al. The landscape of cancer cell line metabolism. Shorthouse, D., Bradley, J., Critchlow, S. E., Bendtsen, C. & Hall, B. Heterogeneity of the cancer cell line metabolic landscape. Cherkaoui, S. et al. A functional analysis of 180 cancer cell lines reveals conserved intrinsic metabolic programs. Seashore-Ludlow, B. et al. Harnessing connectivity in a large-scale small-molecule sensitivity dataset. Rees, M. G. et al. Correlating chemical sensitivity and basal gene expression reveals mechanism of action. Jaaks, P. et al. Effective drug combinations in breast, colon and pancreatic cancer cells. Bashi, A. C. et al. Large-scale pan-cancer cell line screening identifies actionable and effective drug combinations. Whitehurst, A. W. et al. Synthetic lethal screen identification of chemosensitizer loci in cancer cells. Turner, N. C. et al. A synthetic lethal siRNA screen identifying genes mediating sensitivity to a PARP inhibitor. Lord, C. J., McDonald, S., Swift, S., Turner, N. C. & Ashworth, A. A high-throughput RNA interference screen for DNA repair determinants of PARP inhibitor sensitivity. DNA Repair 7, 2010–2019 (2008). Vizeacoumar, F. J. et al. A negative genetic interaction map in isogenic cancer cell lines reveals cancer cell vulnerabilities. McDonald, E. R. 3rd et al. Project DRIVE: a compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening. Shalem, O. et al. Genome-scale CRISPR-Cas9 knockout screening in human cells. Shen, J. P. et al. Combinatorial CRISPR-Cas9 screens for de novo mapping of genetic interactions. Wang, T. et al. Gene essentiality profiling reveals gene networks and synthetic lethal interactions with oncogenic Ras. Horlbeck, M. A. et al. Mapping the genetic landscape of human cells. De Kegel, B. & Ryan, C. J. Paralog buffering contributes to the variable essentiality of genes in cancer cell lines. PLoS Genet. Sharma, S., Dincer, C., Weidemüller, P., Wright, G. J. & Petsalaki, E. CEN-tools: an integrative platform to identify the contexts of essential genes. Madhukar, N. S., Elemento, O. & Pandey, G. Prediction of genetic interactions using machine learning and network properties. Conde-Pueyo, N., Munteanu, A., Solé, R. V. & Rodríguez-Caso, C. Human synthetic lethal inference as potential anti-cancer target gene detection. Lu, X., Kensche, P. R., Huynen, M. A. & Notebaart, R. A. Genome evolution predicts genetic interactions in protein complexes and reveals cancer drug targets. Kim, J. W. et al. Characterizing genomic alterations in cancer by complementary functional associations. Apaolaza, I. et al. An in-silico approach to predict and exploit synthetic lethality in cancer metabolism. Park, S. & Lehner, B. Cancer type-dependent genetic interactions between cancer driver alterations indicate plasticity of epistasis across cell types. Liany, H., Jeyasekharan, A. & Rajan, V. Predicting synthetic lethal interactions using heterogeneous data sources. Liu, X. et al. PiLSL: pairwise interaction learning-based graph neural network for synthetic lethality prediction in human cancers. Srivatsa, S. et al. Discovery of synthetic lethal interactions from large-scale pan-cancer perturbation screens. Menden, M. P. et al. Machine learning prediction of cancer cell sensitivity to drugs based on genomic and chemical properties. Dempster, J. M. et al. Agreement between two large pan-cancer CRISPR-Cas9 gene dependency data sets. Cancer Cell Line Encyclopedia Consortium & Genomics of Drug Sensitivity in Cancer Consortium.Pharmacogenomic agreement between two cancer cell line data sets. Cai, Z. et al. Synthetic augmentation of cancer cell line multi-omic datasets using unsupervised deep learning. Najgebauer, H. et al. CELLector: genomics-guided selection of cancer in vitro models. Cell Syst. Dharia, N. V. et al. A first-generation pediatric cancer dependency map. van de Wetering, M. et al. Prospective derivation of a living organoid biobank of colorectal cancer patients. Bonneville, R. et al. Landscape of microsatellite instability across 39 cancer types. JCO Precis. Lieb, S. et al. Werner syndrome helicase is a selective vulnerability of microsatellite instability-high tumor cells. Kategaya, L., Perumal, S. K., Hager, J. H. & Belmont, L. D. Werner syndrome helicase is required for the survival of cancer cells with microsatellite instability. van Wietmarschen, N. et al. Repeat expansions confer WRN dependence in microsatellite-unstable cancers. Stover, E. H., Fuh, K., Konstantinopoulos, P. A., Matulonis, U. & Liu, J. F. Clinical assays for assessment of homologous recombination DNA repair deficiency. Ferretti, S. et al. Discovery of WRN inhibitor HRO761 with synthetic lethality in MSI cancers. Kikuchi, S. et al. Abstract ND11: chemoproteomic-enabled discovery of VVD-214, a synthetic lethal allosteric inhibitor of WRN helicase. Cancer Res. Download references E.G. work is supported by FCT (Fundação para a Ciência e Tecnologia), under projects UIDB/50021/2020 (DOI:10.54499/UIDB/50021/2020), 2024.07252.IACDC (https://doi.org/10.54499/2024.07252.IACDC, through RE-C05-i08.M04) and grant number 15030 (https://doi.org/10.54499/LISBOA2030-FEDER-00868200). is funded by Research Ireland under grant number 20/FFP‐P/8641. is supported by the Wellcome Trust, CR-UK and the UKRI, Medical Research Council. Instituto Superior Técnico (IST), Universidade de Lisboa, Lisboa, Portugal Emanuel Gonçalves INESC-ID, Lisboa, Portugal Emanuel Gonçalves School of Medicine, University College Dublin, Dublin, Ireland Conway Institute, University College Dublin, Dublin, Ireland Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK David J. Adams Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors contributed equally to all aspects of the article. Correspondence to Emanuel Gonçalves, Colm J. Ryan or David J. Adams. The authors declare no competing interests. Nature Reviews Drug Discovery thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. CellModelPassports: https://cellmodelpassports.sanger.ac.uk/ DepMap Portal: https://depmap.org/portal/ GDSC / CancerRxGenes: https://www.cancerrxgene.org/ Project Score: https://score.depmap.sanger.ac.uk/ The process of reducing the volume of reagents and samples in laboratory assays, enabling high-throughput screening with increased efficiency and reduced costs. Deletion of a gene, for example, a tumour suppressor gene, that inadvertently leads to the loss of neighbouring genes, known as passenger genes, resulting in a therapeutic vulnerability that can be exploited for targeted cancer treatment. A genetic interaction in which the effect of one gene is modified by one or more other genes. A genetically engineered variant of the human HAP1 cell line in which the LIG4 gene, encoding DNA ligase IV, has been knocked out, disrupting the non-homologous end joining DNA repair pathway, promoting DNA repair via homologous recombination, important for the insertion of specific DNA edits. The interdependence of metabolic reaction rates within a network, whereby the flux through one reaction constrains or affects the flux through another. This coupling can arise from stoichiometric relationships, shared intermediates or regulatory mechanisms. A condition characterized by the accumulation of insertion or deletion mutations in microsatellite regions of DNA, typically caused by defects in the mismatch repair system. Genes that arise from a duplication event within the genome of a species. Proteins with features (ubiquitylation site, long half-life) that indicate that they may be suitable for degradation using proteolysis-targeting chimeras (PROTACS). A technique in molecular biology in which specific regions of a gene are systematically mutated to all possible variations. Proteins currently not yet targetable with existing pharmacological strategies. Targets previously deemed undruggable, such as the oncoprotein KRAS, now have available inhibitors and so this category is continually evolving, especially as new modalities, such as proteolysis-targeting chimeras (PROTACs) and molecular glues, expand the targetable proteome. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Synthetic lethality in cancer drug discovery: challenges and opportunities. Nat Rev Drug Discov  (2025). Download citation Accepted: 24 July 2025 DOI: https://doi.org/10.1038/s41573-025-01273-7 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Drug Discovery (Nat Rev Drug Discov) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/magic-mushroom-edibles-found-to-contain-no-psilocybin/'>These Magic Mushroom Edibles Have No Psilocybin—And Potentially Dangerous Undisclosed Ingredients</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Magic Mushroom Edibles Found to Contain Undisclosed Ingredients—And No Psilocybin Researchers tested 12 “magic mushroom” edible products sold in Portland, Ore., and found no trace of psilocybin, the hallucinogenic compound that gives magic mushrooms their name. Instead seven of the products contained at least one undisclosed active ingredient. “We found no evidence of mushroom compounds of any kind, coming from any species,” says the new study's co-author Richard van Breemen, a pharmaceutical sciences professor at Oregon State University. The research was published on Thursday in JAMA Network Open. Psilocybin and other psychedelic drugs have received a lot of attention in recent years as potential treatments for mental health conditions such as depression and post-traumatic stress disorder. “There's a lot of hype around these substances, so people are increasingly trying them outside current legal pathways,” says Lori Bruce, a bioethicist at Yale University, who researches psychedelics. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. In June 2024 the Centers for Disease Control and Prevention announced that the agency was investigating reports of severe illness after people consumed certain purported psilocybin edibles. By last October, 73 hospitalizations and three deaths—all possibly associated with such products—had been reported across 34 states. “The mislabeling highlighted in this [study] causes harms,” Bruce says. Psilocybin comes from several mushroom species, including some in the genus Psilocybe. When consumed, the body breaks down the compound into psilocin, an alkaloid that can cause startling visual hallucinations and psychological effects, often including intense introspection. The U.S. federal government classifies both psilocybin and psilocin as Schedule I drugs, meaning they're deemed to have a high potential for abuse and no currently accepted medical use. Both medical and recreational use are legal in Colorado, and New Mexico passed a law this year that legalizes certain medical uses. In Oregon the regulations are stricter: the drug must be taken in the company of a licensed facilitator at a specific “psilocybin service center.” These centers receive psilocybin from licensed growers, whose products are tested by a licensed laboratory. This dynamic may be leading more people to try purported psilocybin products that are untested and prohibited but nonetheless sold at some retail stores, says legal researcher Mason Marks, a law professor at Florida State University who focused on psychedelics. “A lot of people are very curious about these substances. And if you're in a state, like Oregon, that does not decriminalize them, people might go to these shops and buy these products that are either blatantly illegal or kind of in this gray area.” These products are “being marketed widely in local convenience stores and on the Internet,” van Breemen says. Seven products contained undisclosed active ingredients such as caffeine, kava extract (a legal herbal supplement with antianxiety and hallucinogenic effects) and cannabis extract (including the plant's main psychoactive ingredient, tetrahydrocannabinol, or THC). Four of the gummies contained no active ingredients at all. Their effects on the body have not been studied in humans. "We don't know what harm they might cause,” van Breemen says. “Over the years, we've tested many botanical dietary supplements,” he says. “Psychedelics show initial promise for intractable mental health disorders, but science is still teasing out who will benefit and who will be harmed from their psychedelics experience.” The Trump administration has indicated an interest in deregulating psychedelics for therapeutic purposes. In this context, Bruce adds, “the concerns I raise may become more widespread.” As a multimedia journalist, she contributes to Scientific American's podcast Science Quickly. Parshall's work has also appeared in Quanta Magazine and Inverse. She graduated from New York University's Arthur L. Carter Journalism Institute with a master's degree in science, health and environmental reporting. She has a bachelor's degree in psychology from Georgetown University. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/08/250821094513.htm'>Ancient fossils in Ethiopia rewrite the first chapter of human evolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 11:59:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But in the past, Homo wasn't alone. We coexisted and competed with other branches of the human family tree. Research supported by the National Science Foundation and the Leakey Foundation and published in Nature now fills in a piece of the ongoing evolutionary puzzle, placing two early species of hominin side-by-side. A team working in the Afar Region of Ethiopia, at the site of Ledi-Geraru, reports hominin fossils that date between 2.6 and 3.0 million years old. Lucas Delezene, an associate professor of anthropology at the University of Arkansas, was second author on a paper that incorporated the work of more than 20 researchers from North America, Africa and Europe. Unexpectedly, the team also found evidence that Homo overlapped at the site with a different type of hominin, Australopithecus, at 2.6 million years ago. The famous Australopithecus fossil known as Lucy was found at a nearby site, but her species disappeared from the fossil record at 3 million years ago. "People often think evolution is a linear progression," explains Delezene, "like the March of Progress, but in reality humans are only one species that make up a twig of a bigger family tree -- it's quite bushy and what we found is another twig that was previously unknown. The idea that Homo appears and immediately spreads around the planet and replaces all other hominin species is not accurate. Homo lived side-by-side with many other hominin species throughout Africa. For example, from southern Ethiopia to southern Africa, the earliest species of Homo overlapped with a hominin known as Paranthropus, which is well known for its massive teeth and chewing muscles and a diet reliant on grass in some parts of its range. However, in the Afar Region of Ethiopia, no Paranthropus fossils have ever been found. Instead, the team working at Ledi-Geraru found that Homo overlap with a different type of hominin, Australopithecus. Did Homo eat the same things in Ethiopia where it coexisted with Australopithecus as it did in the south where it coexisted with Paranthropus, or was its diet flexible? We know that Homo eventually becomes a culturally reliant tool user and occasionally consumed meat. But the oldest Homo fossils at Ledi-Geraru predate any evidence of tool manufacture or meat consumption. Did Homo evolve those traits to avoid competing with other hominin species? Delezene, a hominin dental expert, says, "When we get down to the picky details, the teeth of Homo and Australopithecus look different. The multi-national collaboration, done in partnership with the local community of Afar people, will continue its work looking for more fossils, ideally with continued funding. Note: Content may be edited for style and length. NASA Perseverance Rover's Stunning Find May Be Mars' First Sign of Life Scientists Sound Alarm: Why Are Rivers in Alaska Turning Orange? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/early-experiments-show-fast-acting-antidote-targets-carbon-monoxide/'>Early Experiments Show Fast-Acting Antidote Targets Carbon Monoxide Poisoning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 10:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Early Experiments Show Fast-Acting Antidote Targets Carbon Monoxide Poisoning A study in mice and on human blood uses a new protein to snag carbon monoxide before it latches onto blood cells A new antidote is designed to rapidly address carbon monoxide poisoning. Odorless and colorless, it has a uniquely efficient ability to starve the body of oxygen: It acts quickly, building up in the bloodstream and attaching to hemoglobin in oxygen's place. This gas, a common by-product of incomplete fuel combustion, causes 50,000 to 100,000 emergency room visits and 1,500 deaths in the U.S. each year. It's effective, but it's slow—and while only a small percentage of people with carbon monoxide poisoning die, survivors are often left with brain damage, cardiac complications or kidney and liver problems from oxygen deprivation. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. He says RcoM-HBD-CCC could be injected into people on their way to the hospital in an ambulance or to people with low oxygen levels at the site of fires. “This molecule becomes CO-bound pretty much as soon as you inject it,” says study co-author and University of Pittsburgh researcher Jesus Tejero. Because it has a much higher affinity for carbon monoxide than carbon monoxide has to hemoglobin, RcoM-HBD-CCC quickly sponges up the toxin. Lance B. Becker, an emergency medicine researcher at the Donald and Barbara Zucker School of Medicine at Hofstra/Northwell, who was not involved in the study, notes that the new protein binds to carbon monoxide—but not to nitric oxide, a gas molecule that plays a key role in relaxing blood vessels to improve blood flow. Gladwin and his team had previously engineered a protein with an affinity to carbon monoxide—but it also bound to nitric oxide, causing problematic artery stiffening in early mouse tests. Becker hopes this treatment will prove effective in planned studies with larger animals and eventually in human trials, which are likely still a few years off. Sara Novak is a science writer based on Sullivan's Island, S.C. Her work has appeared in Discover, Sierra Magazine, Popular Science, New Scientist, and more. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-63277-6'>Social exposome and brain health outcomes of dementia across Latin America</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 10:14:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. A multidimensional social exposome (MSE)—the combined lifespan measures of education, food insecurity, financial status, access to healthcare, childhood experiences, and more—may shape dementia risk and brain health over the lifespan, particularly in underserved regions like Latin America. However, the MSE effects on brain health and dementia are unknown. We evaluated 2211 individuals (controls, Alzheimer's disease, and frontotemporal lobar degeneration) from a non-representative sample across six Latin American countries. Adverse exposomes associate with poorer cognition in healthy aging. Food insecurity, financial resources, subjective socioeconomic status, and access to healthcare emerge as critical predictors. Cumulative exposome measures surpass isolated factors in predicting clinical-cognitive profiles. Multiple sensitivity analyses confirm our results. Findings highlight the need for personalized approaches integrating MSE across the lifespan, emphasizing prevention and interventions targeting social disparities. Poverty and socioeconomic disparities have multidimensional effects that can harm the brain through multiple pathways. The exposome refers to the totality of environmental, social, and biological exposures accumulated over a lifetime. Within this framework, the social exposome captures the cumulative and multidimensional impact of socially driven exposures that influence health outcomes1. These effects may exert a more substantial influence on aging and dementia than isolated risk factors2,3,4. For instance, individuals growing up in poverty may face limited educational opportunities and food insecurity, relying on low-cost, nutrient-poor diets that impair brain development3. In adulthood, chronic stress from traumatic events and financial insecurity may add to this burden5, while in older age, accumulated adverse exposures throughout life contribute to physiological dysregulation, increasing vulnerability to neurodegeneration. The cumulative burden of socially-related factors5 (e.g., low educational attainment, adverse childhood experiences, and traumatic events), aggravating factors (e.g., limited access to healthcare6), and affected domains (e.g., financial burden and reduced social interaction) may significantly exacerbate dementia phenotypes. These cumulative social burdens constitute the multidimensional social exposome (MSE)1, extending traditional factors such as education level and socioeconomic status (SES). Social determinants of health (SDH), which encompass some proxies of the social exposome, play a crucial role in brain health, healthy aging, and dementia2,7. However, current research fails to characterize the MSE across different life stages10. Factors such as food insecurity, access to healthcare6, childhood experiences, and exposure to traumatic events11 may significantly contribute to dementia, but they have not been systematically addressed. In particular, the impact of the social exposome on core dimensions of brain health in aging and dementia—cognition, functional ability, neuropsychiatric manifestations, brain atrophy, and connectivity—remains unknown. Latin America (LA) presents a wide spectrum of social and environmental factors that may influence late-life brain health outcomes. These factors are linked to historical, sociopolitical, and sociocultural diversity12, as well as political polarization and its potential impact on health. These include financial assets, healthcare access, education, social support, and infrastructure12. Such inequality is reflected by Gini scores frequently surpassing 0.47, indicating considerable wealth inequality14. LA is typified by the intersection of unique educational challenges, food insecurity, childhood difficulties, healthcare inequalities, and adverse environments5,12. Other reports suggest stronger effects of SDH than genetic ancestry for dementia risk in LA16. However, current research is limited to specific factors such as SES, education, or single macrosocial estimates2,7,14,15,16 usually investigated in isolation. Considering multidimensional lifespan measures, a cumulative social exposome assessment may be much more informative in capturing the heterogeneity and diversity of underserved populations facing multifocal adversities. Significant gaps impede an adequate understanding of the MSE. Most evidence is based only on broad measures of education (years of schooling) and SES10, usually factored out as control variables rather than targeted as relevant phenomena8. Although valid, SES measures present multiple limitations10, such as inconsistent definitions (e.g., varying metrics for education, income, and profession), lack of standardization across contexts, and fragmentation of domains precluding the assessment of cumulative burdens. SDH research has neglected multimodal clinical-cognitive measures and their biological embedding of brain health10. The available evidence is biased toward the United States (US) and European data, with an underrepresentation of individuals suffering from larger disparities in countries with more diverse origins. Models based on data from high-income countries fail to capture brain and dementia phenotypes accurately and do not generalize well to more diverse populations, particularly in LA18. Though variously defined, brain health is consistently considered a multilevel construct encompassing brain structural and functional integrity, cognitive health, functional ability, and low levels of neuropsychiatric symptoms. There is a need to explore the various aspects of the social exposome and its impact on brain health outcomes in aging and dementia for underrepresented populations. This may help to develop tailored biopsychosocial models of dementia. We aimed to address these gaps by employing an MSE approach (Fig. 1) to characterize brain health in aging and dementia in terms of cognition, functional ability, neuropsychiatric symptoms, brain volume, and functional connectivity in a non-representative sample of 2211 individuals across 6 LA countries, including healthy controls (HC), patients with Alzheimer's disease (AD) and frontotemporal lobar degeneration (FTLD) (Table 1). These two conditions are among the leading causes of dementia and are critically underrepresented in research from LA. We created a multidimensional score leveraging expert criteria, factorial confirmatory analysis, and structural equation modeling of combined variables in different domains (education, food insecurity, financial status, assets, access to healthcare, childhood labor, subjective SES, childhood experiences, traumatic events, and relationship assessments). Each dimension included a combination of various components across different life stages (from birth to present). Access to healthcare is considered an item evaluating challenges in paying for services from 0 to 10, 35 to 45 years old, and recently, including specific barriers such as economic reasons, medical appointment availability, transportation, distance to medical facilities, lack of accompaniment, the COVID-19 pandemic, or socio-political events. The MSE also included exposure to alcohol and/or drug abuse in the household, experiences of neglect during childhood, and household organization (see Dimensions in MSE assessment). We anticipated that adverse MSE would have an extensive effect on brain health, including (a) poor cognition, reduced functional ability, and increased neuropsychiatric symptoms; (b) a more substantial impact than individual classical predictors (e.g., education, income); and (c) larger brain atrophy and reduced functional connectivity in dementia-sensitive brain regions for AD and FTLD. a Healthy controls (HC) and participants with Alzheimer's disease (AD) and frontotemporal lobar degeneration (FTLD) were recruited from six Latin American countries: Argentina, Brazil, Chile, Colombia, Mexico, and Peru. b The assessment protocol included a comprehensive multidimensional social exposome (MSE) evaluation alongside clinical, cognitive, and neuroimaging measures. For MSE quantification, all items underwent imputation for missing values (NaN) and were then min-max scaled between 0 and 1, with higher values indicating lower levels of social adversity. Composite scores for each participant were calculated by averaging variables within specific categories: education, food insecurity, financial status, assets, access to healthcare, childhood labor, subjective socioeconomic status (SES), childhood experiences, traumatic events, and relationships. c These domain-specific scores, ranging from 0 to 1, served as inputs in structural equation models (SEM) to predict cognitive function, functional ability, and neuropsychiatric symptoms in the whole sample (green), HC (yellow), AD (purple), and FTLD (red). d The global MSE score, extracted from the latent variable, was employed to predict brain structure and functional connectivity. The coverage of regions within the top clusters identified in whole-brain voxel-based morphometry analyses using magnetic resonance imaging (MRI) was examined. Specific regions may include the middle frontal gyrus (MFG), insula (INS), rolandic operculum (ROL), and anterior cingulate cortex (ACG). In resting-state functional magnetic resonance imaging (rsfMRI) analyses, region of interest (ROI)-to-ROI connectivity was assessed by identifying the top regions with the highest number of connections. All analyses were controlled for age, sex, total intracranial volume (TIV, for voxel-based morphometry analyses), and scanner. (2025) https://BioRender.com/rxcmtgq (sample size, assessment protocol, covariate icons and MRI device). The MSE was operationalized by grouping items measuring facets of similar dimensions based on expert consensus agreement. Then, we calculated the weighted averages on these items to create dimension scores for education, food insecurity, financial status, assets, healthcare access, childhood labor, subjective SES, childhood experiences, traumatic events, and relationships. These scores were obtained by scaling between the minimum and maximum values for 319 items and then averaging these to obtain a value for each dimension (Fig. We validated these dimensions via confirmatory analysis. Structural equation modeling was fed with these composite factors to create a global latent MSE score per subject (Fig. To examine whether the combined MSE effects were stronger than, and not driven by, single factors influencing the global score, we tested the contributions of individual predictors with Lasso models. Then, we compared them with the global MSE score via meta-regressions. The structural and functional brain correlates of the MSE scores were tested in a subsample with whole-brain gray matter volume and ROI-to-ROI functional connectivity (Fig. Multiple sensitivity analyses were employed to confirm that the results were not biased by individual dimensions or traditional SES factors, demographic, recruitment bias, disease-related factors (age, sex, disease severity, age at diagnosis, years after diagnosis, and FTLD subtype); or scanner effects (scanner type, signal-to-noise ratio, and motion parameters). The global score of adverse MSE was associated with diminished cognitive function (β = 0.261 [95% CI = 0.18–0.282], p < 0.001), decreased functional ability (β = 0.079 [95% CI = 0.015–0.11], p < 0.001), and heightened neuropsychiatric symptoms (β = 0.105 [95% CI = 0.033–0.132], p < 0.001). SEM between the multidimensional social exposome (MSE) and cognitive status, functional ability and neuropsychiatric symptoms. Top factor loadings (exact p-values: education = 1 × 10−308; food insecurity = 1 × 10−308; financial status =  1 × 10−308; assets = 1 × 10−308; childhood labor = 1 × 10−308; subjective SES = 1 × 10−308; childhood experiences = 1 × 10−308; traumatic events = 1 × 10−15; relations = 1 × 10−8) and distribution of association with outcomes for a the whole sample (green), b healthy controls (HC, yellow), c persons with Alzheimer's disease (AD, purple) and d frontotemporal lobar degeneration (FTLD, red). No adjustments were made for multiple comparisons. The SEM was implemented as a two-sided model by default. In the scatter plots, each point represents an individual observation. In healthy aging, adverse MSE was linked to lower cognitive function (β = 0.610 [95% CI = 0.463–0.737], p < 0.001) but did not predict functional ability (β = −0.047 [95% CI = −0.026 to 0.073], p = 0.101) nor neuropsychiatric symptoms (β = −0.032 [95% CI = −0.114 to −0.002], p = 0.259) (Fig. The model showed lower fit indices with CFI = 0.859, TLI = 0.756, RMSEA = 0.105, and SRMR = 0.092. In individuals with AD, adverse MSE was associated with lower cognitive function (β = 0.229 [95% CI = 0.157–0.309], p < 0.001), reduced functional ability (β = 0.079 [95% CI = 0.004–0.152], p = 0.038), and an increase in neuropsychiatric symptoms (β = 0.104 [95% CI = 0.013–0.193], p = 0.007) (Fig. The model obtained excellent fit indices with CFI = 0.955, TLI = 0.922, RMSEA = 0.058, and SRMR = 0.041. For those with FTLD, adverse MSE predicted decreased cognitive function (β = 0.165 [95% CI = 0.037–0.289], p = 0.017), reduced functional ability (β = 0.140 [95% CI = 0.015–0.288], p = 0.040), and increased neuropsychiatric symptoms (β = 0.141 [95% CI = 0.019–0.281], p = 0.040) (Fig. The model showed excellent fit indices with CFI = 0.985, TLI = 0.973, RMSEA = 0.036, and SRMR = 0.041. The most significant adverse MSE predictors in healthy aging were assets, education, and financial status, associated with lower cognitive function. When individual predictors were assessed for the entire sample using Lasso regression, a reduced effect across outcomes was observed compared to the global MSE score (all p < 0.01, Table 2). Similarly, a reduced effect was noted for functional ability (R2 = 0.04, f2 = 0.04, MnSqErr = 0.05, MAE = 0.18), with subjective SES (coef = 0.16), education (coef = 0.12), and access to healthcare (coef = 0.11) emerging as key predictors. This pattern of reduced effects of individual predictors compared to the global MSE score was consistent between groups (all p < 0.01, Table 2). For healthy aging, reduced effects for cognition were observed (R2 = 0.43, f2 = 0.75, MnSqErr = 0.01, MAE = 0.05), with education (coef = 0.14) and assets (coef = 0.26) being prominent contributors. Regarding functional ability in AD (R2 = 0.01, f2 = 0.01, MnSqErr = 0.05, MAE = 0.18), subjective SES (coef = 0.08) was the primary predictor. In the context of neuropsychiatric symptoms, AD showed a reduced effect (R2 = 0.02, f2 = 0.02, MnSqErr = 0.03, MAE = 0.13), with food insecurity (coef = 0.05), childhood experiences (coef = 0.05), and access to healthcare (coef = 0.01) as leading predictors. Thus, the cumulative MSE effects were neither driven by nor weaker than any independent effects. In individuals with AD, more adverse MSE was linked to reduced gray matter volume in frontal and cerebellar regions, with the highest cluster peak values observed in the superior, middle, and inferior frontal gyrus, vermis (lobules IV–VI), cerebellum (crus I and II, lobule VIIb and VIII), and insula (Fig. A closer inspection of the regions covered in the top clusters revealed the Rolandic operculum, gyrus rectus, orbitofrontal cortex, anterior cingulate gyrus, and the putamen, which showed a stable association between less adverse MSE and higher gray matter volumes of each top cluster (Fig. a Whole-brain analysis for persons with Alzheimer's disease (AD, purple) shows regions with significant associations between the multidimensional social exposome (MSE) and gray matter volume (GMV), highlighted by threshold-free cluster enhancement (TFCE) values. Scatterplots illustrate the relationship between MSE scores and clusters' GMV. Corresponding scatterplots depict the associations between MSE scores and GMV for each identified cluster. In the scatter plots, each point represents an individual observation. For those with FTLD, larger adverse MSE was associated with decreased gray matter volume in fronto-temporo-cerebellar regions. The highest cluster peaks were found in the vermis (lobule IV, V, and VII), superior temporal gyrus, superior and middle frontal gyrus, and anterior cingulate cortex (Fig. Coverage of top cluster regions identified the cerebellum (crus I and II, lobule IV, V, VI, VIII, and IX), Rolandic operculum, insula, Heschl's gyrus, and postcentral gyrus. A consistent link was found between fewer adverse MSE and increased gray matter volumes in each top cluster (Fig. In individuals with AD, more adverse MSE was associated with lower frontotemporal connectivity between the anterior cingulate cortex, hippocampus, parahippocampal gyrus, striatum, insula, and cerebellum (lobule III and vermis lobule III). a Functional connectivity in persons with Alzheimer's disease (AD) displayed through region of interest (ROI)-to-ROI connectivity maps, where color-coded edges represent significant t-values for decreased (red) and increased (blue) connectivity associated with MSE. The connectome ring provides a simplified visualization of interconnections between key regions. b In persons with frontotemporal lobar degeneration (FTLD), ROI-to-ROI connectivity maps similarly provide visualizations of significant connections associated with MSE, with the connectome ring displaying interconnections between key regions and a circular barplot chart showing key regions with the highest number of connections in this group (red). In FTLD, more adverse MSE was associated with lower frontotemporal connectivity primarily between frontal (superior, middle, and inferior, precentral gyrus), temporal (superior, parahippocampal gyrus), limbic (amygdala, hippocampus, and cingulate cortex), and cerebellar regions (crus I, lobule III, IV-V, IX, and X). Greater MSE adversity was associated with higher connectivity in frontal (superior, middle, inferior), temporal (superior, middle, inferior, and parahippocampal gyrus), hippocampus, and cerebellar regions (crus I, lobule IV–V, VI, and X) was observed (Fig. Multiple effects were tested to confirm the robustness of our analysis. The association between adverse MSE with reduced cognition and functional ability and increased neuropsychiatric symptoms was preserved (all p < 0.01) after controlling for the effect of demographics, potential recruitment bias, and dementia-related covariates (age, sex, disease severity, age at diagnosis, years after diagnosis, and FTLD subtype; Supplementary Table 5). Despite an unbalanced distribution of age and sex across groups (Table 1), all effects remained consistent after controlling for these covariates. Moreover, none of these covariates were associated with the MSE score (all p > 0.10), confirming that potential confounders did not drive the main effects (Supplementary Table 5). The associations between adverse MSE with reduced cognition and functional ability and increased neuropsychiatric symptoms were preserved (all p < 0.01) after controlling for country level (Supplementary Information 1). Additional models were run to rule out potential bias from single factors linked to worse clinical-cognitive phenotypes (i.e., education and SES). Excluding education from the global MSE score confirmed that the association between adverse MSE and reduced cognition remained significant (Supplementary Information 2). Furthermore, models considering only objective or subjective SES showed lower fit indices and weaker associations than MSE in predicting cognition, functional ability, and neuropsychiatric symptoms (Supplementary Information 3). In summary, the robustness of our findings was supported through multiple approaches, including the validation of MSE dimensions, controlling for key covariates, and assessing cross-country consistency. MSE models demonstrated better fit and predictive power than SES-only models, and the effects persisted even when education was excluded from the MSE or missing values were not imputed. To control the effects of different scanners, scanner type was included as a dummy covariate of no interest among the predictors in whole-brain gray matter volume and ROI-to-ROI functional connectivity analyses14, thus corroborating that scanner type did not explain findings (Tables 3 and 4, Supplementary Tables 3 and 4). The observed effects in whole-brain gray matter volume and ROI-to-ROI functional connectivity analyses were robust to imbalances in age and sex distribution across groups. Consistent results were observed while controlling for these covariates in the regression models (Tables 3 and 4, Supplementary Tables 3 and 4). We also explored the association between MSE and f/MRI data quality metrics (Methods, Sensitivity analysis). The same null effects were observed across groups (all p > 0.113) (Supplementary Table 6). Further, we tested separate models excluding participants with less than 70% of artifact-free frames19. Results replicated the main functional connectivity effects (Supplementary Tables 7 and 8). In addition to the primary analyses, we ran additional voxel-based morphometry models employing a harmonization strategy that normalized the voxel intensity values for MRI scans2. To provide a framework for replication with previous evidence on functional connectivity in LA populations2,14,15,20,21,22,23,24, we ran additional models employing the automated anatomical labeling (AAL) atlas. Results matched the connectivity patterns described in the main results (Supplementary Information 4, Supplementary Tables 11 and 12). All analyses were controlled by age, sex, and total intracranial volume (for voxel-based morphometry analyses). Multiple comparisons were false discovery rate (FDR)-corrected. This study aimed to identify the association of MSE and brain health outcomes in aging and dementia in LA. The combined effects of MSE components demonstrated stronger associations, even after excluding education. The stronger model fit and greater magnitude of associations for MSE compared to SES alone underscored a cumulative multidimensional burden. The top predictors in healthy aging were composite measures of education and assets, whereas, in dementia, more multidimensional exposomes emerged, such as food insecurity, financial status, subjective SES, and access to healthcare. Potential compensatory functional hyperconnectivity, characterized by higher functional connectivity associated with higher MSE adversity, was observed primarily in regions with reduced volume that were associated with more adverse MSE. Variations in participants' country of origin, demographic and dementia-related factors, image acquisition methods, or signal quality did not account for the results. Results demonstrate that the MSE, as a combined measure, exhibits stronger effects than individual factors and composite SES measures, providing a low-dimensional representation of the cumulative social exposome. While some studies have assessed the influence of SES7, nutrition, relationships7, access to healthcare, and traumatic experiences on functional ability and neuropsychiatric symptoms in aging, we evaluated their complex cumulative effects across the lifespan, ranging from birth to the present time. We expanded the classical assessments of SDH25 by including food insecurity, access to healthcare, subjective SES, and a weighted and lifespan evaluation of these factors. For instance, most previous studies8,9 have used years of education, and we expanded these effects with a combined measure that incorporates multiple domains (school type and location, perceived education quality throughout life, the number of books at home during childhood, parental education levels, and years of formal schooling). This approach allowed us to address complex cumulative effects. In healthy aging, cognition was mainly modulated by traditional factors such as education, assets, and financial status7,26 as assessed with complex measures. Our approach considered the combined influence of traditional (e.g., educational achievement and financial status) and non-traditional factors for each dimension, such as parental education, rurality, education quality, financial stress, food insecurity, difficulties in accessing healthcare6, childhood experiences, exposure to traumatic events11, and relationships7. Each factor has been studied individually, but not combined into a composite exposome measure to assess its association with brain health. Thus, our approach allowed us to examine the cumulative effects of multiple dimensions via the MSE. The impact on functional ability and neuropsychiatric symptoms in aging was null, probably due to the floor effect of these measures (see below). While factors like food insecurity, access to healthcare, childhood experiences, and subjective SES were significant predictors, the stronger contribution of classical factors probably reflects regional inequalities. Formal education is the main modifiable risk factor for dementia in LA27, associated with marked inequalities between the US and low-middle income countries, and critically differentiates aging and dementia brain outcomes in LA vs. the US15. SES is the second most burdensome disparity-related factor across LA regarding cognition7, probably by reducing cognitive reserve in older adults26. Current results support the role of education in preventing cognitive decline and dementia, this time involving intergenerational and quality-based measures. Moreover, results call for brain capital policy initiatives1; without systematically decreasing the structural inequalities14, the adverse individual-level effects on brain and cognitive health in the region cannot be properly addressed. Lifestyle exposomes, including physical activity, smoking, alcohol consumption, and cognitive engagement in late life, are associated with dementia risk3. Specific components related to social exposome—including education, social isolation, socioeconomic status (SES), and structural inequality—have been linked to cognition, functional ability7, and alterations in brain structure and functional connectivity2,14,15. The complex social exposome in AD29 and FTLD4 suggests a lifespan cumulative influence3,30,31,32. These effects can lead to accumulated dysregulation across different life stages, predisposing individuals to neurodegeneration in aging. Components of adverse social exposome in AD are associated with dysregulated stress responses, linked with reduced cognition, impaired functional ability, increased neuropsychiatric symptoms, and brain burden33. Elevated plasma cortisol levels, indicative of HPA axis dysregulation, have been associated with reduced total brain volume, decreased glucose metabolism in the frontal cortex, and increased beta-amyloid load in AD-vulnerable regions34,35. Emerging evidence in FTLD links altered stress responses with neurodegenerative processes via multisystemic dysregulations31,32. Overall, our MSE approach offers a combined and granular characterization of the social exposome in dementia. This work supports current efforts to define the broader dementia exposome4, enabling more precise risk profiling. The brain correlates of adverse MSE in dementia align with disease-specific environmental explanations for pathophysiological mechanisms in AD. These suggest lower resistance to p-tau and Aβ burden across frontotemporal regions associated with lifetime stressful events33, stress-mediated impaired insulin signaling and cerebellar vulnerability30, and inflammatory models of clinical progression33. Regarding brain connectivity, our results showed similar associations with those reported with physical exposome2, education15, and structural inequalities14 in dementia. There is little evidence on environmental stress and FTLD phenotypes, suggesting a substantial genetic contribution to this condition17. However, FTLD brain health outcomes were linked with the social exposome in this study. Evidence of pathophysiological mechanisms suggests that the brain burden observed in FTLD may be driven by the accumulation of physiological stressors over time. Evidence on pathophysiological mechanisms links the brain burden to the accumulation of physiological stressors (i.e., allostatic overload) in FTLD31,32 and also relates it to SDH2,14,15, targeting both disease-specific and non-specific regions. Environmental stressors can impact distributed brain regions while accelerating specific pathophysiological processes related to dementia2,14,15,21. In FTLD, early degeneration targets fronto-temporo-limbic networks, leading to connectivity disruptions underlying socioemotional dysregulation, behavioral changes, and similar clinical outcomes. MSE may primarily exacerbate AD structural vulnerability through cumulative stress and resource depletion, whereas in FTLD, they predominantly could aggravate functional connectivity dysregulation. These neurobiological pathways may suggest that MSE can be linked with partially comparable clinical outcomes despite differing structural and functional correlates. Conversely, greater functional connectivity is associated with greater MSE adversity is aligned with evidence on compensatory effects in AD36 and FTLD37, as well as aversity-related factors such as socioeconomic status38, financial stress39, and discrimination40. This may reflect a compensatory process triggered by brain burdens associated with (a) neurodegenerative processes36,37, (b) disease-specific environmental factors contributing to pathophysiological mechanisms30,31,32, and (c) their interactions. In line with the prolonged accumulation of amyloid and tau pathology and with environmental exposures linked to the later onset of AD33,34,35, stronger compensatory functional connectivity associated with greater MSE adversity was observed in this group compared to FTLD. This may reflect the longer-lasting impact of degenerative and environmental burdens on brain structural signatures (atrophy) in AD, whereas in FTLD, more transient compensatory changes may emerge earlier in the disease course due to its typically younger onset. Although sometimes neglected in the literature41, the cerebellum is a core structure in atrophy related to dementia41 and environmental stressors such as low-SES9, structural inequality14, and poverty42. This may also explain why the cerebellum is part of the allostatic interoceptive network, which is impaired in individuals with dementia21,31,43,44 and those with social disparities32,38. Thus, allostatic interoception provides a framework for understanding the dual impact of dementia pathophysiology and social adversity. Overall, our findings deepen the association between social exposome and brain health outcomes in dementia in LA. We provided comprehensive coverage of social exposome dimensions and extensive brain health phenotyping of aging and dementia beyond current approaches that usually only consider traditional and unidimensional measures (e.g., years of education, basic SES scores), specific life periods (i.e., aging), or unimodal phenotypes. The cumulative lifespan of social exposome provides an understanding of the long-term impacts on brain health and dementia. We leveraged a large and geographically diverse sample across LA encompassing six countries with varying characteristics, surpassing the size of those typically found in current research in a region with marked inequalities14,16. Our protocol was harmonized across settings, and multiple sensitivity analyses confirmed robust results against heterogeneities in demographics (age and sex), country of origin, disease-related factors and recruitment bias (disease severity, age at diagnosis, years after diagnosis, FTLD subtype), and scanner effects (scanner type, signal-to-noise ratio, and motion parameters). However, several reasons suggest this is not the case, as we performed multiple control analyses. These included: (a) accounting for variations in disease severity, age at diagnosis, and time elapsed since diagnosis to mitigate potential overrepresentation of participants from disadvantaged groups in more advanced disease stages; (b) eliminating circularity in the assessment of MSE by excluding variables that may confound socioeconomic disparities with dementia outcomes (e.g., access to specific services such as cell phones could indicate either socioeconomic disparities or the consequences of cognitive decline); and (c) conducting within-group analyses to identify MSE associations through a dimensional approach rather than presuming uniform patterns across groups. However, future studies should incorporate other probabilistic or stratified random sampling strategies in the designs. The findings demonstrate the need for diverse data and tailored modeling to capture brain health outcomes of aging and dementia, crucial for more efficient prevention and interventions. These strengths are accompanied by limitations that invite further research. While our MSE assessment considers different life stages, it relies on participant self-report, which could introduce bias. We partially addressed this issue by omitting items from the evaluation that can be biased by dementia symptomatology (e.g., smartphone use, occupational situation, Supplementary Information 5) and corroborating participant self-reports with information provided by caregivers. This is consistent with standard practices for questionnaire-based assessments in individuals with dementia, such as evaluating depressive symptoms45, quality of life46, and activities of daily living47. Additionally, we controlled the impact of disease severity on these predictors to minimize confounding effects. While our MSE assessment considers different life stages, items were limited to early, middle, and late life (0–10, 35–45 years old, and recently), leaving out other age periods. This approach was taken to ensure a more detailed assessment than traditional SES approaches while avoiding excessive items that could increase participant burden and compromise data quality. However, future studies should incorporate additional life stages to enable a more comprehensive analysis across the lifespan. Factors other than MSE, beyond the social domain, have cumulative effects on brain health48. Potentially modifiable risk factors for dementia may be influenced by disparities3. LA has one of the highest population-attributable fractions for dementia risk factors in the world27, associated with high levels of adversity, inequality5,12, and dementia prevalence12,13. Complex interactions between social exposome, genetics, physical exposome (pollution, pesticides, heavy metals)2,17, and modifiable risk factors are beyond the scope of this work and require additional research. Like previous studies employing social determinants measures15,16, our data were measured at the individual level across countries. Our results do not comprise aggregate-level data. Future studies should incorporate nested designs with aggregated-level analyses and geocoded data relevant to brain health in LA, such as structural inequality14, air pollution2, and access to green spaces49. A further step in future research may involve adjusting individual metrics by nested analysis of spatial buffers at different scales (i.e., block [0–100 m], neighborhood [100–500 m], district/commune [500–5000 m], city [5–50 km]). Our study did not account for (ethnic or racial) discrimination and ancestry/genetic effects in a region with marked ethnic diversity16,17. As other works in the region2,14,15, a limitation of this study is the reliance on clinical criteria for diagnosis without biomarkers like amyloid-β and tau, assessed via PET or plasma. PET is costly, globally inaccessible, and lacks validation in diverse populations such as LA50. Similarly, the feasibility and accuracy of blood-based biomarkers in LA have not yet been developed to validate their use combined with clinical criteria. Sample sizes are relatively modest within each country. However, the ReDLat cohort uses a well-distributed LA clinical sample that includes diverse populations51,52. While we acknowledge that LA is not a homogenous construct, this does not preclude the use of harmonized datasets across diverse countries2,7,14,15,20,53, as has been done in other large-scale initiatives such as ADNI and European cohorts, which also feature significant sociocultural differences. Future work should expand these findings by incorporating community-representative samples to enhance the generalizability of results. Although our recruitment was aligned with current guidelines54, many of the recommended strategies are designed with US research infrastructure in mind. They may not entirely apply to the general LA context, where limited resources and larger socioeconomic disparities often hinder research. Further, our protocol adheres to specific methodological considerations and recommended best practices for cross-national comparisons55: (a) The MSE questionnaire was developed and harmonized by a team of experts from various countries where it was subsequently applied, validated with a diverse sample of Latino Spanish-speaking participants56, and tested to represent the same construct across countries via psychometric validation. (b) The covariables included in structural equation model (SEM) analyses are available and collected under a harmonized protocol across all countries. Our study addresses underrepresented populations from LA, which present unique patterns of brain health2,7 compared to more homogenous populations (e.g., UK Biobank). The assessments of the dimensions related to social exposome are designed with different objectives and utilize distinct methodologies10. The functional ability and neuropsychiatric measures were not sensitive to healthy aging, likely due to a disease-sensitive floor effect of these measures (i.e., healthy controls present floor effects with no impairments). Future studies should consider other assessments tailored to healthy aging. Lastly, our neuroimaging analyses encompassed different scanners and recording parameters. However, sensitivity analyses confirmed that inter-scanner differences and data quality metrics did not compromise the results. In conclusion, this study evidenced an association of the multidimensional social exposome on brain health outcomes in aging and dementia across LA underserved populations. These results call for the development of tailored models incorporating the impact of social environments in characterizing dementia. Prevention initiatives and multicomponent interventions targeting structural inequality should consider lifespan cumulative effects to foster healthier aging trajectories and improve brain health in LA. The study included 2211 participants with a mean age of 64.63 years (SD = 11.26), of whom 67.03% were women. ReDLat collects data from multiple centers across these countries, using a standardized data framework and harmonized diagnostic protocols7,20,53,58,59. Participants were recruited from extensive networks including (a) clinical networks, involving memory clinics, neurology departments, and affiliated hospitals; (b) academic collaborations, leveraging partnerships with universities and research institutions; (c) community outreach programs, engaging with local communities through informational sessions, and culturally tailored materials to encourage participation from rural and urban populations with diverse socioeconomic backgrounds; and (d) public health initiatives and local organizations, integrating recruitment efforts with public health campaigns and community groups to raise awareness and facilitate participation. These efforts allowed us to include individuals from rural and urban settings, focusing on underrepresented groups, as our ReDLat cohort is marked by socioeconomic inequality14 and educational disparities15. Strategies to improve access and recruitment for these groups involve field screenings, community engagement efforts, and the use of mobile units. Exclusion criteria included participants with conditions other than AD or FTLD or those with impairments preventing task completion. Diagnoses were determined by consensus among expert healthcare providers at each site, based on cognitive and neurological exams, clinical interviews, and MRI57. The diagnoses are based on the clinical criteria established by the National Institute of Neurological and Communicative Disorders and Stroke and the Alzheimer's Disease and Related Disorders Association (NINCDS-ADRDA) for AD, as well as on the clinical criteria specified for FTLD. Healthy Controls had preserved cognition and no history of neurological or psychiatric conditions. A standardized battery was used to capture clinician evaluations. ReDLat aligns with current guidelines to increase the representativeness of samples enrolled in Alzheimer's disease research centers54 by implementing inclusive recruitment strategies and strengthened community engagement, geographic decentralization, and unified inclusion and exclusion criteria. All clinicians underwent training and certification by a specialized team and adhered to a quality control protocol57. Data collection and analysis presented no risks related to stigmatization, incrimination, discrimination, animal welfare, environmental or health concerns, safety, security or personal privacy. Additionally, no transfer of biological materials, cultural artifacts or traditional knowledge occurred. All participants signed informed consent in accordance with the 2013 Declaration of Helsinki. An expert team designed a questionnaire that reviewed existing tools and their validity data on various social, economic, familial, and environmental factors that can influence health across the lifespan56. This questionnaire was created by different experts working with Latino participants from Mexico, Central America, and South America. Matching guidelines to increase sample representativeness in Alzheimer's disease research54 and recommended practices for cross-national comparisons55, validity testing was performed with a diverse sample of Latino Spanish-speaking participants, clinicians, and researchers to iteratively improve the questionnaire and standardize the collection of sociodemographic factors. Unlike typical questionnaires that focus on single domains, such as education, childhood experiences, or traumatic experiences, this questionnaire aimed to evaluate multiple domains simultaneously to understand how life experiences and conditions from childhood to the present impact health. Questionnaire construction aligns with current harmonization guidelines60, as (a) items were selected after careful inspection from clinicians and researchers from different cultures (at least one from each country) and comparability across countries; (b) development of psychometric procedures to test if dimension composite score represents a coherent and consistent latent construct of interrelated variables and captures the intended dimension; and (c) transparency and openness by providing the full version of the questionnaire in a repository (https://osf.io/78ng6/). The protocol has been standardized, validated, and subjected to psychometric standardization samples from different countries, including healthy individuals and those with neurocognitive disorders, ensuring its applicability to diverse populations56,61. The questionnaire was administered to 137 participant-caregiver dyads in a more detailed validation. After multivariate assumptions such as normality, linearity, homogeneity, and homoscedasticity, adequacy tests were conducted to ensure data quality (Bartlett's test of sphericity: χ2 = 2191.611, p < 0.001; Kaiser–Meyer–Olkin measure: MSA = 0.75). Lastly, a three-factor SEM was created, including SES, challenging life experiences, and educational environment, which showed good fit indices (TLI = 0.764, RMSEA = 0.062, SRMR = 0.06, CFI = 0.884)56. We reviewed the questionnaire to remove items that might create circular associations with dementia-related outcomes. We excluded dimensions that could be directly influenced by dementia symptoms, thereby risking the measurement of those symptoms instead of the intended MSE construct. Specifically, we removed items related to technology access (smartphone use), employment status (work engagement), and income sources (current salary) (Supplementary Information 5). The MSE questionnaire systematically evaluates a comprehensive range of topics to ensure a thorough assessment of the participant's background and experiences. Education is evaluated through the number of years of formal education, the type of school attended (public or private), the geographical location of the school (rural or urban), and the overall quality of education received (from very bad to excellent), number of books at home during the first 10 years of life (from 0 to more than 200), and the educational achievement of the mother and father (from none to doctoral degree). Food insecurity captures if the participant had to eat less (yes/no) or less healthy (yes/no) due to economic hardship across the lifespan (from 0 to 10, 35 to 45 years old, and recently), capturing the quality and consistency of food insecurity experienced by the participant across the lifespan. Financial status assesses the participant's financial stability and ability to meet basic needs over different periods of their life (from 0 to 10, 35 to 45 years old, and recently). Additionally, it gauges the participant's financial resilience in the event of lost income and health insurance (public, private, or none), offering a comprehensive view of their economic situation and security throughout their lifespan. It assesses whether the participant had access to electricity, radio, television, refrigerator, washing machine, landline telephone, water heater, indoor bathroom, running water, automobile, computer, internet, sound system, smartphone, and private room. Additionally, it measures the quantity of certain items like televisions, bathrooms, cars, computers, and private rooms the participant had during these periods. This assessment provides a detailed view of the participant's material living conditions and how they have evolved throughout their lifespan. It evaluates whether the participant experienced difficulty paying for medical services and explores specific barriers that may have prevented them from seeing a doctor (from not hard to very hard). These barriers include financial constraints, appointment availability, transportation issues, distance, lack of accompaniment, the COVID-19 pandemic, or socio-political events (yes/no). This assessment provides insights into the participant's access to healthcare and the factors influencing it throughout their lifespan. Childhood labor investigates whether the participant worked before the age of 18 (yes/no) and the specific age range during which they started working (from 5 to 18 years old). It also delves into the reasons for working at a young age, including whether to meet personal needs, support family needs, save money, or learn a new skill (yes/no). This assessment provides insight into the participants' early exposure to work and the SES factors that may have influenced their decision to work during childhood or adolescence. Childhood experiences comprise the participant's early childhood environment, focusing on family structure, emotional support, and potential adverse experiences. Additionally, it explores conflicts within the family, such as fights between parents, parents-participant, and siblings. The questions also inquire about exposure to alcohol abuse, the organization of the household, and experiences of neglect during childhood (from never to frequently). This assessment provides a comprehensive view of the participant's early family life and its potential impact on their development. It includes questions about the loss of siblings or children before the age of 18, experiences of political violence and repression, and exposure to theft or robbery both inside and outside the home. It also examines the participants' involvement in or witness to accidents, situations where they feared for their lives, and encounters with dead or injured individuals. Furthermore, the questions explore experiences of physical violence, being attacked with a weapon, humiliation, restrictions on personal freedom, neglect, and financial control. Additionally, the set assesses instances where the participant was pressured to leave their property or coerced into sexual activity (yes/no). These experiences are considered in relation to different individuals (family members, friends, caregivers, strangers) (yes/no). This comprehensive assessment aims to capture the depth and breadth of the participant's exposure to adverse and potentially traumatic experiences throughout their life. Relationships assess the participant's frequency of contact with loved ones and experiences of being treated with less respect across different stages of life and in recent times. It measures how often the participant interacts with close family or friends, ranging from less than once a week to more than five times a week. These experiences are categorized by how often they occur, ranging from almost daily to less than once a year or never. This assessment provides insights into the participant's social connections and the prevalence of negative interpersonal experiences throughout their life. The questionnaire employs various response formats, including Likert scales, multiple-choice questions (Yes/No), and free text fields. Following previous reports62, we scored the questionnaire by creating composite variables by grouping items based on expert consensus on the similarity of the assessed domains. Then, the dimension scores for each participant were obtained by averaging the value of the total number of variables per category (education, food insecurity, financial status, assets, access to healthcare, childhood labor, subjective SES, childhood experiences, traumatic events, and relations) to produce a score between 0 and 1. These values for each domain were used as inputs in the SEM. Confirmatory factor analyses showed a good model fit for the dimensions (Supplementary Table 13). The mini-mental state examination (MMSE) is a widely used cognitive screening tool that evaluates various cognitive functions, including arithmetic, memory, and orientation. The MMSE is scored from 0 to 30, with higher scores indicating better cognitive function. A score of 24 or above is considered normal, while scores below 24 suggest cognitive impairment. Specifically, scores between 19 and 23 indicate mild cognitive impairment, scores between 10 and 18 indicate moderate cognitive impairment, and scores below 10 suggest severe cognitive impairment. The MMSE has demonstrated moderate to high reliability and is frequently used across LA populations. The Pfeffer Functional Activities Questionnaire (PFAQ) is a tool designed to assess the functional ability of older adults, focusing on instrumental activities of daily living. Scoring of the PFAQ ranges from 0 to 30, with higher scores indicating greater levels of dependence or functional impairment. Each item is rated on a scale from 0 (independent) to 3 (dependent), and the total score provides an overview of the individual's functional status. The PFAQ is reliable and employed in studies involving LA populations. The Neuropsychiatric Inventory Questionnaire (NPI-Q) is a streamlined version of the original Neuropsychiatric Inventory (NPI), designed to assess a wide range of neuropsychiatric symptoms commonly found in patients with dementia. The NPI-Q evaluates 12 key domains: delusions, hallucinations, agitation/aggression, depression/dysphoria, anxiety, euphoria/elation, apathy/indifference, disinhibition, irritability/lability, aberrant motor behavior, nighttime behavioral disturbances, and appetite/eating abnormalities. Scoring of the NPI-Q ranges from 0 to 36, with two components for each domain: the frequency and severity of symptoms. Frequency is rated from 1 (occasionally) to 4 (very frequently), and severity is rated from 1 (mild) to 3 (severe). The total score sums these ratings across all domains, providing a comprehensive measure of the patient's neuropsychiatric profile. Higher scores indicate more severe and frequent symptoms. The NPI-Q has adequate test-retest reliability and convergent validity and is commonly applied in research involving LA populations. 3D T1-weighted images were collected for 875 individuals with a mean age of 65.13 years (SD = 10.77), of whom 65.94% were women. Preprocessing and analysis were employed using VBM with the Computational Anatomy (CAT12) toolbox and in Statistical Parametric Mapping software (SPM 12; Wellcome Center for Human Neuroimaging; www.fil.ion.ucl.ac.uk/spm/software/spm12/) in Matlab R2021a. The standard pipeline included bias-field correction, noise reduction, skull stripping, segmentation, and normalization to the Montreal Neurological Institute (MNI) space at a 1.5 mm isotropic resolution. CAT12 also performed intra-subject harmonization by normalizing data to the mean global intensity for each subject, followed by smoothing gray matter images with a 6 × 6 × 6 mm Gaussian kernel. Scanner effects were controlled through two approaches: by including scanner type as a covariate in the VBM regression models and standardizing between the minimum and maximum intensity values of each voxel for all subjects evaluated by each scanner type2. Resting-state sequences were collected for 500 individuals with a mean age of 65.27 years (SD = 12.02), of whom 61.80% were women. Image preprocessing was employed using the fmriprep (version 22.0.2) standard pipeline, encompassing head motion artifacts, slice timing, susceptibility distortion correction, co-registration to the anatomical image, and normalization to standard space, with additional steps in the CONN22.a toolbox63. This involved smoothing with a 6 × 6 × 6 mm Gaussian kernel and denoising through linear regression with nine nuisance regressors applied in a single step: six motion parameters (translation and rotation), white matter, cerebrospinal fluid signals, and scrubbing regressors for high-motion time points, and applying a band-pass filter (0.008-0.09 Hz. A motion correction technique was applied by rigidly aligning fMRI volumes to T1-weighted images, ensuring that the impact of motion artifacts was minimized. Motion scrubbing was then applied using framewise displacement >0.2 mm and temporal derivative variance across apace> 5%, which are stricter than conventional thresholds, to flag and remove high-motion frames. The mean proportion of artifact-free to rejected frames was 0.915 (SD = 0.126), with values ranging from 30% to 100%. This was implemented using the artifact detection tools within the CONN toolbox63, using a conservative setting (FD = 0.2 mm, global signal Z = 5) to remove motion artifacts while preserving the biological signal. Pearson correlation coefficients were computed between the average BOLD time series of each pair of regions of interest (ROIs) from the Brainnetome atlas, a structural and functional connectivity-based parcellation atlas that captures both cortical and subcortical regions, better suited to functional connectivity analysis. AAL atlas cerebellar regions were added, generating a total of 272 × 272 ROIs correlation matrix for each participant. Scanner variability was controlled by including scanner type as a covariate in the ROI-to-ROI connectivity regression models. SEM is a statistical method that facilitates the examination of relationships among observed and latent variables, particularly useful for measuring the effect of each indicator and its cumulative effect through a latent variable by integrating aspects of regression analysis, factor analysis, and simultaneous equation models64. The model's structure was based on ten composite variables derived from the MSE questionnaire, which included education, food insecurity, financial status, assets, access to healthcare (randomly set as the fixed indicator to identify the values for the latent factor), childhood labor, subjective SES, childhood experiences, traumatic events, and relationships. These variables served as indicators of a single MSE latent variable, which was used to predict cognition, functional ability, and neuropsychiatric symptoms. The functional ability and neuropsychiatric symptoms scores were inverted to align their interpretation with other measures. Covariances between variables were theoretically guided by the top modification index score. Model fit was assessed using the fit indices comparative fit index (CFI ≥ 0.9), Tucker–Lewis index (TLI ≥ 0.9), root mean square error of approximation (RMSEA ≤ 0.08), and standardized root mean square residual (SRMR ≤ 0.08). We did not consider χ2 based on its high sensitivity to sample size; as the sample size increases, χ2 values rise, resulting in lower p-values, thus skewing the interpretation of the model's fit. Separate models were run for each group to examine the specific associations with cognition, functional ability, and neuropsychiatric symptoms. In addition to testing group-specific effects, this approach allows us to isolate the effects within each group. This approach abolishes the confounding factors observed in group comparisons due to age and sex differences. The value of the MSE latent variable for each subject was extracted from these models for subsequent analysis of associations with gray matter and resting-state functional connectivity. Even the smallest sample size (FTLD) met the N ratio of 10:1 for statistical accuracy and power, with 225 participants exceeding the number of estimated parameters (q = 13). Missing data (maximum 19.7%) were imputed across the sample using predictive mean matching through the multivariate imputation by chained equations (MICE) package (3.16.0) in R version 4.3.0. The main SEM models were run without imputation for the entire sample and each group separately. Results showed consistent associations across all models, indicating that imputation did not introduce bias. Lasso regression was employed to examine the individual contributions of MSE dimensions while controlling for multicollinearity. This method was chosen for its ability to perform variable selection and regularization, shrinking less relevant coefficients toward zero and effectively excluding them from the model. This comparison assessed the global MSE score's explanatory power versus individual dimensions, highlighting the benefits of using an aggregate social exposure metric for understanding multidimensional outcomes. In parallel, Lasso regression was applied to each bootstrap sample to identify the model's predictive performance using R2, also capturing effect size variability across the bootstrap samples. After obtaining the bootstrap estimates for both SEM and Lasso regression, the results were compared via meta-regressions for each group across outcomes. We conducted regression analyses using parametric tests to examine the associations between MSE and brain volume in SPM12, with age, sex, total intracranial volume (TIV), and scanner effects (see Supplementary Table 17 for acquisition parameters) included as covariates of no interest. To enhance behavioral variance and statistical power, HC were merged with AD patients to run VBM models for the AD group, and the process was repeated by pairing HC with FTLD patients for the FTLD model. This method integrates voxel and cluster thresholds, eliminating the need for arbitrary cluster formation thresholds and enhancing sensitivity to both focal and diffuse effects, thus optimizing the balance between the FDR and replicability65. Statistical significance was assessed through 5000 permutations and set at PFDR < 0.05. Brain imaging results were visualized using MRIcroGL (v1.2.20220720). We conducted regression analyses via parametric tests with the MSE score as a predictor of the whole-brain ROI-to-ROI connectivity. The number of connections of each ROI was calculated to assess its contribution to the broader connectivity patterns associated with MSE. Following alternate methods of calculating degree centrality67,68, we considered edges that were positively or negatively weighted and statistically significant (FDR-corrected). These correspond to values different from zero in the t-statistic matrix, reflecting meaningful ROI-to-ROI relationships. These edges were binarized—assigned a value of one—so that each significant, weighted edge contributes a count of 1 to a node's degree centrality, representing the number of connections each ROI exhibits in relation to MSE. Higher values indicate that a given ROI's connectivity demonstrates more widespread associations with MSE. As for gray matter associations, age, sex, and scanner effect (see Supplementary Table 18 for acquisition parameters) were included as covariates of no interest, and each patient group was analyzed in tandem with HC. To control for the effect of confounding variables, we ran a separate SEM model for the entire sample that included age and sex as predictors of the latent MSE factor, as well as cognition, functional ability, and neuropsychiatric symptoms. To control for disease-related variables, we ran a model with AD and FTLD participants that included disease severity (Clinical Dementia Rating (CDR) total score and CDR-FTLD scale-modified), age at onset, and years since diagnosis, in addition to age and sex as predictors of MSE, as well as cognition, functional ability, and neuropsychiatric symptoms. Additionally, to examine the potential effect of the FTLD subtype, we implemented a model with FTLD participants, including a binary subtype variable (behavioral variant frontotemporal lobar degeneration vs. any other subtype) as a predictor of MSE, as well as cognition, functional ability, and neuropsychiatric symptoms. To investigate whether MSE was influenced by data quality of structural and functional MRI, we conducted sensitivity analyses examining the relationships between MSE and spatial signal-to-noise ratio (SNR) for both MRI and fMRI, temporal SNR for fMRI, and head motion parameters for fMRI. For fMRI data, the quality evaluation involved segmenting each time series into 20 repetition times (TR) intervals to compute the temporal signal-to-noise ratio (tSNR)70. The tSNR was defined as the mean fMRI signal within each segment divided by its standard deviation. To further test the effect of motion correction, we applied two procedures: rigid realignment of fMRI volumes to T1-weighted images and scrubbing of high-motion frames. Scrubbing was based on framewise displacement >0.2 mm and global signal change >5% (Methods, Neuroimaging Acquisition and Preprocessing). Participants with less than 70% of artifact-free frames were excluded (Supplementary Tables 7-8). The minimum and maximum voxel intensity values were extracted from all subjects belonging to each scanner type, which were then scaled between the minimum and maximum intensity values specific to each scanner type. Then, voxel-based morphometry analyses were replicated with the scaled images. This method accounts for differences in location and scale (e.g., mean and variance) between sites, reducing scanner-related variability. This process helps to minimize the variability introduced by different scanners2,20. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The preprocessed behavioral, MRI, and fMRI data generated in this study have been deposited in the OSF database under accession code https://osf.io/78ng6/. The raw data are available under restricted access due to ethical and regulatory constraints. Access can be obtained after IRB approval of the formal data-sharing agreement in a process that can last up to 12 weeks. For questions related to the data request and usage, contact Agustín Ibañez at agustin.ibanez@gbhi.org. Source data are provided with this paper. The code used to preprocess and analyze the data of this work is available in an Open Science Foundation repository at the following address: https://osf.io/78ng6/ Ibanez, A. et al. Neuroecological links of the exposome and One Health. Livingston, G. et al. Dementia prevention, intervention, and care: 2024 report of the Lancet standing Commission. & Feldman, E. L. Role of the exposome in neurodegenerative disease: recent insights and future directions. Migeot, J. et al. Allostasis, health, and development in Latin America. Access to primary care and cognitive impairment: results from a national community study of aging Americans. Santamaria-Garcia, H. et al. Factors associated with healthy aging in Latin American populations. The neuroscience of socioeconomic status: correlates, causes, and consequences. Structural and functional brain correlates of socioeconomic status across the life span: a systematic review. & Ruiz-Adame, M. Addressing the gaps between socioeconomic disparities and biological models of dementia. Severs, E. et al. Traumatic life events and risk for dementia: a systematic review and meta-analysis. Structural inequality linked to brain volume and network dynamics in aging and dementia across the Americas. Gonzalez-Gomez, R. et al. Educational disparities in brain health and dementia across Latin America and the United States. Llibre-Guerra, J. J. et al. Social determinants of health but not global genetic ancestry predict dementia prevalence in Latin America. & Slachevsky, A. Environmental–genetic interactions in ageing and dementia across Latin America. Healthy aging meta-analyses and scoping review of risk factors across Latin America reveal large heterogeneity and weak predictive models. Suarez-Jimenez, B. et al. Intrusive traumatic re-experiencing domain: functional connectivity feature classification by the ENIGMA PTSD Consortium. Fittipaldi, S. et al. Heterogeneous factors influence social cognition across diverse settings in brain health and age-related diseases. Birba, A. et al. Allostatic-interoceptive overload in frontotemporal dementia. Coronel-Oliveros, C. et al. Viscous dynamics associated with hypoexcitation and structural disintegration in neurodegeneration via generative whole-brain modeling. Moguilner, S. et al. Biophysical models applied to dementia patients reveal links between geographical origin, gender, disease duration, and loss of neural inhibition. Hazelton, J. L. et al. Altered spatiotemporal brain dynamics of interoception in behavioural-variant frontotemporal dementia. The impacts of social determinants of health and cardiometabolic factors on cognitive and functional aging in Colombian underserved populations. & Fittipaldi, S. Socioeconomic status impacts cognitive and socioemotional processes in healthy ageing. & Livingston, G. Population attributable fractions for risk factors for dementia in low-income and middle-income countries: an analysis using cross-sectional survey data. The exposome of healthy and accelerated aging across 40 countries. & Ferreira, S. T. Impaired insulin signalling and allostatic load in Alzheimer disease. A predictive coding framework of allostatic-interoceptive overload in frontotemporal dementia. Lifetime stressful events associated with Alzheimer's pathologies, neuroinflammation and brain structure in a risk enriched cohort. Plasma cortisol, brain amyloid-β, and cognitive decline in preclinical Alzheimer's disease: a 6-Year prospective cohort study. Roemer-Cassiano, S. N. et al. Amyloid-associated hyperconnectivity drives tau spread across connected brain regions in Alzheimer's disease. Agosta, F. et al. Functional connectivity from disease epicenters in frontotemporal dementia. Lower socioeconomic position is associated with greater activity in and integration within an allostatic-interoceptive brain network in response to affective stimuli. Suárez-Pellicioni, M. & McDonough, I. M. Separating neurocognitive mechanisms of maintenance and compensation to support financial ability in middle-aged and older adults: The role of language and the inferior frontal gyrus. Han, S. D. et al. Self-reported experiences of discrimination in older black adults are associated with insula functional connectivity. Schmahmann, J. D. Cerebellum in Alzheimer's disease and frontotemporal dementia: not a silent bystander. The ecology of poverty and children's brain development: A systematic review and quantitative meta-analysis of brain imaging studies. Santamaría-García, H. et al. Allostatic interoceptive overload across psychiatric and neurological conditions. Emerging Theories of Allostatic-Interoceptive Overload in Neurodegeneration. Park, S.-H. & Cho, Y. S. Predictive validity of the Cornell Scale for depression in dementia among older adults with and without dementia: a systematic review and meta-analysis. Kahle-Wrobleski, K. et al. Assessing quality of life in Alzheimer's disease: Implications for clinical trials. Fish, J. in Encyclopedia of Clinical Neuropsychology (eds Jeffrey S. Kreutzer, John DeLuca & Bruce Caplan) 111–112 (Springer New York, 2011). Resilience and brain health in global populations. Rojas-Rueda, D., Vaught, E. & Buss, D. Why a new research agenda on green spaces and health is needed in Latin America: results of a systematic review. Parra, M. A. et al. Biomarkers for dementia in Latin American countries: gaps and opportunities. The Multi-Partner Consortium to Expand Dementia Research in Latin America (ReDLat): driving multicentric research and implementation science. Visual deep learning of unprocessed neuroimaging characterises dementia subtypes and generalises across non-stereotypic samples. Classification of Alzheimer's disease and frontotemporal dementia using routine clinical and cognitive measures across multicentric underrepresented samples: A cross sectional observational study. Arce Rentería, M. et al. Representativeness of samples enrolled in Alzheimer's disease research centers. Kobayashi, L. C. et al. Cross-national comparisons of later-life cognitive function using data from the Harmonized Cognitive Assessment Protocol (HCAP): Considerations and recommended best practices. Piña-Escudero, S. D. et al. Social determinants of health questionnaire development for patients with Alzheimer's disease and frontotemporal dementia. The Multi-Partner Consortium to Expand Dementia Research in Latin America (ReDLat): driving multicentric research and implementation science. Dynamic brain fluctuations outperform connectivity measures and mirror pathophysiological profiles across dementia subtypes: a multicenter study. Moguilner, S. et al. Multi-feature computational framework for combined signatures of dementia in underrepresented settings. Gross, A. L. et al. Harmonisation of later-life cognitive function across national contexts: results from the Harmonized Cognitive Assessment Protocols. Andrew, M. K. & Keefe, J. M. Social vulnerability from a social ecology perspective: a cohort study of older adults from the National Population Health Survey of Canada. Nieto-Castanon, A. Handbook of functional connectivity Magnetic Resonance Imaging methods in CONN. (2020). Cause and Correlation in Biology: A User's Guide to Path Analysis, Structural Equations and Causal Inference with R. 2 edn, (Cambridge University Press, 2016). & Yan, C. G. Reproducibility of R-fMRI metrics on the impact of different strategies for multiple comparison correction and sample sizes. Smith, S. M. & Nichols, T. E. Threshold-free cluster enhancement: addressing problems of smoothing, threshold dependence and localisation in cluster inference. Sporns, O. Graph theory methods: applications in brain networks. Complex network measures of brain connectivity: uses and interpretations. The relationship between fMRI temporal signal to noise ratio and necessary scan duration. Data in this manuscript were collected by the MULTI-PARTNER CONSORTIUM TO EXPAND DEMENTIA RESEARCH IN LATIN AMERICA (ReDLat), supported by Fogarty International Center (FIC), National Institutes of Health, National Institutes of Aging (R01 AG057234, R01 AG075775, R01 AG21051, R01 AG083799, CARDS-NIH 75N95022C00031), Alzheimer's Association (SG-20-725707), Rainwater Charitable Foundation – The Bluefield project to cure FTD, and Global Brain Health Institute)]; AI is supported by ANID/FONDECYT Regular (1250091, 1210176, and 1220995) and ANID/FONDAP/15150012. CDA and AI are supported by grant ANID/PIA/ANILLO ACT210096. These authors contributed equally: Joaquin Migeot, Stefanie D. Pina-Escudero, Hernan Hernandez Joaquin Migeot, Hernan Hernandez, Raul Gonzalez-Gomez, Agustina Legaz, Sol Fittipaldi, Claudia Duran-Aniotz, Maria E. Godoy & Agustin Ibanez Joaquin Migeot, Sol Fittipaldi, Brian Lawlor, Sven Vanneste & Agustin Ibanez Global Brain Health Institute, University of California, San Francisco, CA, 94158, USA Stefanie D. Pina-Escudero, Sol Fittipaldi, Elisa de Paula França Resende, Adolfo M. García, Bruce Miller, Maira Okada de Oliveira, Victor Valcour, Jennifer S. Yokoyama, Katherine L. Possin & Agustin Ibanez Stefanie D. Pina-Escudero, Serggio Lanata, Bruce Miller, Victor Valcour, Jennifer S. Yokoyama & Katherine L. Possin Cognitive Neuroscience Center, Universidad de San Andrés, Ciudad Autónoma de Buenos Aires, Buenos Aires, 1644, Argentina Agustina Legaz, Adolfo M. García, Maria E. Godoy, Marcelo Adrian Maito & Agustin Ibanez Unit Cognitive Impairment and Dementia Prevention, Peruvian Institute of Neurosciences, Lima, 15046, Peru Department of Anesthesia, Critical Care and Pain Medicine, Massachusetts General Hospital, Harvard Medical School, Boston, MA, 02114, USA Departamento de Salud Mental, Hospital Universitario Fundación Santa Fe, Bogotá, Colombia Centro de Memoria y Cognición Hospital Universitario San Ignacio, Bogotá, Colombia Hospital Universitario San Ignacio, Center for Memory and Cognition, Intellectus, Bogotá, D.C, 110231, Colombia Memory and Neuropsychiatric Center (CMYN), Neurology Department, Hospital del Salvador & Faculty of Medicine, University of Chile, Santiago de Chile, Metropolitan Region of Santiago, Santiago, 7500921, Chile Neuropsychology and Clinical Neuroscience Laboratory (LANNEC), Physiopathology Program – Institute of Biomedical Sciences, Neuroscience and East Neuroscience Departments, Faculty of Medicine, University of Chile, Santiago de Chile, Metropolitan Region of Santiago, Santiago, 8380453, Chile Lab for Clinical and Integrative Neuroscience, Trinity College Institute for Neuroscience, Trinity College Dublin, Dublin, D02 PN40, Ireland Department of Psychology and Neuroscience, Boston College, Chestnut Hill, MA, 02467, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Joaquin Migeot: Conceptualization, Methodology, Software, Formal analysis, Writing— Original Draft, Visualization. Hernan Hernandez: Writing—Review & Editing, Validation, Formal analysis, Methodology, Software. Raul Gonzalez-Gomez: Data Curation, Writing—Review & Editing, Formal analysis, Software. Elisa de Paula França Resende: Writing—Review & Editing. Bruce Miller: Writing—Review & Editing, Funding acquisition, Project administration. Agustin Ibanez: Conceptualization, Methodology, Writing—Original Draft, Visualization Review & Editing, Supervision, Project administration, Funding acquisition. Nature Communications thanks Raphael Castilhos, Bogdan Draganski, Miguel Arce Rentería and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Migeot, J., Pina-Escudero, S.D., Hernandez, H. et al. Social exposome and brain health outcomes of dementia across Latin America. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41558-025-02428-5'>Neglecting land–atmosphere feedbacks overestimates climate-driven increases in evapotranspiration</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 10:12:47
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Climate Change (2025)Cite this article Accurate projections of evapotranspiration (ET) are crucial for understanding hydrological responses to climate warming, which remain highly uncertain because of complex land–atmosphere interactions. Here we develop a theoretical framework to disentangle these interactions, achieving highly consistent ET projections between offline and coupled models. Our findings show that previous estimates of climate-driven ET increases have been exaggerated, primarily due to a substantial overestimation of atmospheric evaporative demand. Notably, the atmospheric conditions often assumed to drive ET are, in fact, responses to ET changes induced by soil moisture and vegetation dynamics. Neglecting these land–atmosphere feedbacks has led to a 25–39% overestimation of climate-driven global ET increases and a 77–121% exaggeration of the negative contribution from land surface changes. These biases have caused large discrepancies in hydrological projections and attributions between offline and coupled models, underscoring the importance of accurately representing land–atmosphere interactions to improve the reliability and consistency of future hydrological projections. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The Fluxnet2015 dataset is publicly available at https://fluxnet.org/data/fluxnet2015-dataset/, and the CMIP6 model simulations are publicly available at https://esgf-node.llnl.gov/search/cmip6/. The R code (version 3.5.1) used for data analyses is available via Zenodo at https://doi.org/10.5281/zenodo.16730362 (ref. Fischer, E. M. & Knutti, R. Anthropogenic contribution to global occurrence of heavy-precipitation and high-temperature extremes. Google Scholar Zhou, S., Zhang, Y., Williams, A. P. & Gentine, P. Projected increases in intensity, frequency, and terrestrial carbon costs of compound drought and aridity events. Google Scholar Allan, R. P. et al. Advances in understanding large‐scale responses of the water cycle to climate change. Zhang, S. et al. Reconciling disagreement on global river flood changes in a warming climate. Zhou, S., Yu, B. & Zhang, Y. Global concurrent climate extremes exacerbated by anthropogenic climate change. Gu, B., Zhou, S., Yu, B., Findell, K. L. & Lintner, B. R. Multifaceted changes in water availability with a warmer climate. npj Clim. Budyko, M. I. Climate and Life (Academic Press, 1974). Held, I. M. & Soden, B. J. Robust responses of the hydrological cycle to global warming. Milly, P. C. D. & Dunne, K. A. Potential evapotranspiration and continental drying. Greve, P., Roderick, M. L., Ukkola, A. M. & Wada, Y. The aridity index under global warming. Koutsoyiannis, D. Revisiting the global hydrological cycle: is it intensifying? Earth Syst. Milly, P. C. D. & Dunne, K. A. A hydrologic drying bias in water-resource impact analyses of anthropogenic climate change. Water Resour. Zaitchik, B. F., Rodell, M., Biasutti, M. & Seneviratne, S. I. Wetting and drying trends under climate change. Zhou, S., Yu, B., Huang, Y. & Wang, G. The complementary relationship and generation of the Budyko functions. Scheff, J., Coats, S. & Laguë, M. M. Why do the global warming responses of land‐surface models and climatic dryness metrics disagree? Earth's Future 10, e2022EF002814 (2022). Roderick, M. L., Greve, P. & Farquhar, G. D. On the assessment of aridity with changes in atmospheric CO2. Water Resour. Yang, Y., Roderick, M. L., Zhang, S., McVicar, T. R. & Donohue, R. J. Hydrologic implications of vegetation response to elevated CO2 in climate projections. Zhou, S., Yu, B., Lintner, B. R., Findell, K. L. & Zhang, Y. Projected increase in global runoff dominated by land surface changes. Scheff, J., Mankin, J. S., Coats, S. & Liu, H. CO2-plant effects do not account for the gap between dryness indices and projected dryness impacts in CMIP6 or CMIP5. Zhou, S. et al. Soil moisture–atmosphere feedbacks mitigate declining water availability in drylands. Zhou, S. et al. Diminishing seasonality of subtropical water availability in a warmer world dominated by soil moisture–atmosphere feedbacks. Berg, A. et al. Land–atmosphere feedbacks amplify aridity increase over land under global warming. Seneviratne, S. I. et al. Impact of soil moisture–climate feedbacks on CMIP5 projections: first results from the GLACE-CMIP5 experiment. Zhou, S. et al. Land–atmosphere feedbacks exacerbate concurrent soil drought and atmospheric aridity. Natl Acad. Zhao, L. et al. Evapotranspiration estimation methods in hydrological models. Zhang, K., Kimball, J. S. & Running, S. W. A review of remote sensing based actual evapotranspiration estimation. WIREs Water 3, 834–853 (2016). Kim, Y., Garcia, M. & Johnson, M. S. Land–atmosphere coupling constrains increases to potential evaporation in a warming climate: implications at local and global scales. Earth's Future 11, e2022EF002886 (2023). Zhang, Y. et al. Future global streamflow declines are probably more severe than previously estimated. Yang, H. & Yang, D. Derivation of climate elasticity of runoff to assess the effects of climate change on annual runoff: derivation of climate elasticity of runoff. Water Resour. Zhou, S. et al. A new method to partition climate and catchment effect on the mean annual runoff based on the Budyko complementary relationship. Water Resour. In Handbook of Hydrology (McGraw-Hill Education, 1993). Allen, R. G., Pereira, L. S., Raes, D. & Smith, M. Crop Evapotranspiration: Guidelines for Computing Crop Water Requirements (Food and Agriculture Organization of the United Nations, 1998). Zhou, S. & Yu, B. Physical basis of the potential evapotranspiration and its estimation over land. Zhou, S. & Yu, B. Reconciling the discrepancy in projected global dryland expansion in a warming world. Change Biol. Huang, J., Yu, H., Guan, X., Wang, G. & Guo, R. Accelerated dryland expansion under climate change. Mankin, J. S., Seager, R., Smerdon, J. E., Cook, B. I. & Williams, A. P. Mid-latitude freshwater availability reduced by projected vegetation responses to climate change. Brutsaert, W. A generalized complementary principle with physical constraints for land‐surface evaporation. Water Resour. Szilagyi, J. On the inherent asymmetric nature of the complementary relationship of evaporation. Pastorello, G. The FLUXNET2015 dataset and the ONEFlux processing pipeline for eddy covariance data. Eyring, V. et al. Overview of the Coupled Model Intercomparison Project Phase 6 (CMIP6) experimental design and organization. Model Dev. Skinner, C. B., Poulsen, C. J. & Mankin, J. S. Amplification of heat extremes by plant CO2 physiological forcing. Priestley, C. H. B. On the assessment of surface heat flux and evaporation using large-scale parameters. McColl, K. A., Roderick, M. L., Berg, A. & Scheff, J. The terrestrial water cycle in a warming world. Berg, A. Bridging the gap between simple metrics and model simulations of climate change impacts on land hydrology. Earth's Future 10, e2022EF003259 (2022). Sun, R., Hernández, F., Liang, X. & Yuan, H. A calibration framework for high-resolution hydrological models using a multiresolution and heterogeneous strategy. Water Resour. Pandi, D., Kothandaraman, S. & Kuppusamy, M. Hydrological models: a review. Ma, N., Szilagyi, J. & Zhang, Y. Calibration‐free complementary relationship estimates terrestrial evapotranspiration globally. Water Resour. Betts, R. A. et al. Projected increase in continental runoff due to plant responses to increasing carbon dioxide. Wei, H. et al. Direct vegetation response to recent CO2 rise shows limited effect on global streamflow. Zhou, S. et al. Large divergence in tropical hydrological projections caused by model spread in vegetation responses to elevated CO2. Earth's Future 10, e2021EF002457 (2022). Allen, M. R. & Ingram, W. J. Constraints on future changes in climate and the hydrologic cycle. Lian, X. et al. Multifaceted characteristics of dryland aridity changes in a warming world. Earth Environ. Berg, A. & McColl, K. A. No projected global drylands expansion under greenhouse warming. Wells, N., Goddard, S. & Hayes, M. J. A self-calibrating palmer drought severity index. Vicente-Serrano, S. M., Beguería, S. & López-Moreno, J. I. A multiscalar drought index sensitive to global warming: the standardized precipitation evapotranspiration index. Free vector and raster map data. Natural Earth http://www.naturalearthdata.com (2024). Chow, V. T., Maidment, D. R. & Mays, L. W. Applied Hydrology (McGraw-Hill Book Company, 1988). Penman, H. L. Natural evaporation from open water, bare soil and grass. Series A 192, 120–145 (1948). & Crago, R. D. Graphical interpretation of wet surface evaporation equations. Water Resour. & Sun, F. New analytical derivation of the mean annual water-energy balance equation. Water Resour. Padrón, R. S., Gudmundsson, L., Greve, P. & Seneviratne, S. I. Large-scale controls of the surface water balance over land: insights from a systematic review and meta-analysis: review of surface water balance controls. Water Resour. Xu, X., Liu, W., Scanlon, B. R., Zhang, L. & Pan, M. Local and global factors controlling water–energy balances within the Budyko framework. Lemordant, L., Gentine, P., Swann, A. S., Cook, B. I. & Scheff, J. Critical impact of vegetation physiology on the continental hydrologic cycle in response to increasing CO2. Natl Acad. Zhou, S. Neglecting land–atmosphere feedbacks overestimates climate-driven increases in evapotranspiration. Zenodo https://doi.org/10.5281/zenodo.16730362 (2025). Download references We acknowledge all the principal investigators who contributed data to the Fluxnet2015 dataset (Supplementary Table 1). We also acknowledge the World Climate Research Programme's Working Group on Coupled Modelling, which is responsible for CMIP, and we thank the climate modelling groups (listed in Supplementary Table 2) for producing and making available their model output. For CMIP the US Department of Energy's Program for Climate Model Diagnosis and Intercomparison provides coordinating support and led development of software infrastructure in partnership with the Global Organization for Earth System Science Portals. This work was supported by the National Natural Science Foundation of China (grant nos. and the National Key Research and Development Program of China (grant no. State Key Laboratory of Earth Surface Processes and Disaster Risk Reduction, Faculty of Geographical Science, Beijing Normal University, Beijing, China Institute of Land Surface System and Sustainable Development, Faculty of Geographical Science, Beijing Normal University, Beijing, China School of Engineering and Built Environment, Griffith University Nathan Campus, Brisbane, Queensland, Australia Bofu Yu Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived of and designed the study. contributed to theretical and data analysis and interpretation. drafted the paper. revised and finalized the paper. Correspondence to Sha Zhou. The authors declare no competing interests. Nature Climate Change thanks Rene Orth and the other, anonymous, reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, ET is constrained by the net radiation (\({R}_{n}\)) and surface moisture availability over a dry surface. b, PETa is constrained by aerodynamic conditions, assuming unlimited water and energy supply. The dry surface in a is hypothetically converted to a wet surface but remains decoupled from the atmosphere, meaning that surface temperature (\({T}_{s}\)), air temperature (\({T}_{a}\)), and vapor pressure (\({e}_{a}\)) remain unchanged from the dry surface. The only change is that the vapor pressure at the evaporative surface becomes saturated (\({e}_{s}^{* }\) replaces unsaturated \({e}_{s}\)). Consequently, sensible heat (\(H\)) over the hypothetical wet surface remains unchanged compared to the dry surface, but the latent heat (\(\lambda {PE}{T}_{a}\)) is much larger than \(\lambda {ET}\). c, PETe is constrained by the net radiation (\({R}_{n}\)), when the entire dry surface becomes saturated and fully coupled with the atmosphere. Both surface and atmospheric conditions cool down (\({T}_{s}^{{\prime} }\) and \({T}_{a}^{{\prime} }\)) and become more humid (\({e}_{s}^{* {\prime} }\) and \({e}_{a}^{{\prime} }\)) relative to the dry environment. As a result, the latent heat (\(\lambda {PE}{T}_{e}\)) increases, while the sensible heat (\({H}_{w}\)) decreases, all constrained by \({R}_{n}\). a, Illustration of the difference in \({\beta }_{w}\) when an ocean grid cell were artificially dried up completely. The grey curve shows the saturation vapor pressure (\({e}^{* }/\gamma\)) as a function of temperature (\(T\)), while the two dashed lines represent air and surface isenthapic processes with constant enthalpy, \(T+e/\gamma\), from wet to the driest conditions. Temperature and vapor pressure for the air and surface are shown in blue for the wet state, pink for the dry state, and purple for the driest state. b, Illustration of changes in \({\beta }_{w}\) when surface and air temperatures change proportionally from wet to the driest conditions for the ocean grid cell. The x-axis refers to the mean surface and air temperature. The blue line shows \({\beta }_{{pt}}\) inferred from the Priestley-Taylor equation (PETpt) for the same ocean grid cell. c-d, Frequency distribution of the sensitivities of \({\beta }_{w}\) and \({\beta }_{{pt}}\) to mean temperature, that is, the ratio of the differences in \({\beta }_{w}\) and \({\beta }_{{pt}}\) over the difference in mean temperature (\(d{\beta }_{w}/{dT}\) and \(d{\beta }_{{pt}}/{dT}\)) between wet and the driest conditions, for ocean grid cells in 2020 and 2100, based on 32 CMIP6 models. Mean sensitivities of \({\beta }_{w}\) and \({\beta }_{{pt}}\) across all grid cells are shown in parentheses. a,b, Frequency distribution of the relative changes in PETe (\(\Delta\)PETe/PETe, a) and PETa (\(\Delta\)PETa/PETa, b) from wet to the driest conditions in 2020 and 2100, based on 32 CMIP6 models. PETe and PETa under wet conditions are derived from ocean grid cells, while their values under the driest conditions are estimated based on the constraint of moist enthalpy (Methods and Extended Data Fig. Mean relative changes across all grid cells and models are shown in each panel. c,d, Frequency distribution of the relative changes in PETe (\(\Delta\)PETe/PETe, c) and PETa (\(\Delta\)PETa/PETa, d) induced by the vegetation physiological responses to elevated CO2 across land grid cells, based on 7 ESMs. In each ESM, relative changes are computed as the ratio of the mean difference between 1pctCO2 and 1pctCO2-rad to the value in 1pctCO2-rad, averaged over the final 30-year period of their 140-year simulations (1960–1989). a-c, Multi-model mean PETe from 1pctCO2 and 1pctCO2-rad and their differences (\(\Delta\)PETe) induced by vegetation physiological responses to elevated CO2 during the final 30 years of the 140-year simulations (1960–1989). d-f, The same as a-c, but for PETa. Basemap in a-f is from Natural Earth56. a, Relationship between the dryness index (PET/P) and the ET ratio (ET/P), using PET from the energy balance (PETe), the Priestley-Taylor (PETpt), the open-water Penman (PETow), the reference crop Penman-Monteith (PETrc), and the energy-only (PETeo) equations. The circles show global terrestrial mean values from 1850 to 1989 (140 years) averaged over the 7 models in 1pctCO2-rad. b, Distribution of the correlation coefficient (r) between global terrestrial mean PET/P and ET/P over the 140 years across the 7 models. The top and bottom of each box represent the first and third quartiles, the center line the median, the whiskers the data range (excluding outliers, shown as circles), and the cross the mean values across the 7 models. c, Changes in the dryness index (PET/P) and the ET ratio over land relative to the first 30-year period during the 140-year simulations (1850-1989). d, Changes in the global mean ET (\(\Delta\)ET) projected by the 7 models and estimated using the Budyko framework with 30-year mean precipitation and PETe (\(\Delta\)ET_PETe), the three Penman equations (\(\Delta\)ET_PETpt, \(\Delta\)ET_PETow, and \(\Delta\)ET_PETrc), and the energy-only equation (\(\Delta\)ET_PETeo). The first year of each 30-year period (111 moving averages in total) is shown along the horizontal axis. The solid lines show 7-model means and the shading refers to one standard deviation across the 7 models in c and d. a, Relationship between the dryness index (PETLAC/P) and the ET ratio (ET/P) with climate change. The circles represent global terrestrial mean values from 1850 to 1989, averaged across 7 models in 1pctCO2-rad. b, Global pattern of the 7-model mean correlation coefficient (r) between annual ET/P and annual PETLAC/P in 1pctCO2-rad. c, Global pattern of the difference in ET changes projected directly from the 7 models (\(\Delta\)ET) and indirectly using the Budyko framework with PETLAC and precipitation (\(\Delta\)ET_Budyko). The 7-model mean \(\Delta\)ET is computed between the first (1850-1879) and final (1960-1989) 30-year periods of 1pctCO2-rad. d, Climate-driven changes in the global mean \(\Delta\)ET projected by the 7 models and estimated using the Budyko framework (\(\Delta\)ET_Budyko) over the 140-year simulation of 1pctCO2-rad. e, Changes in the global mean \(\Delta\)ET projected by the 7 models in 1pctCO2 and the isolated contributions of climate change (\(\Delta\)ETcc, 1pctCO2-rad) and land surface changes (\(\Delta\)ETlsc, 1pctCO2 – 1pctCO2-rad). The climate change effect (\(\Delta\)ETcc_Budyko) and land surface effect (\(\Delta\)ETlsc_Budyko) estimated using the Budyko framework are also shown. The first year of each 30-year period (111 moving averages in total) is shown along the horizontal axis. The solid lines represent the 7-model means and the shading refers to one standard deviation across the 7 models in d and e. Basemap in b and c made with Natural Earth56. a-c, Multi-model mean differences between the energy-based PETe and ET during the historical period (1980-2009), the future period (2071-2100), and their changes (future minus historical). d-f, The same as a-c, but for the differences between the PET based on land atmosphere coupling (PETLAC) and ET. Grid cells where the absolute difference between PET and ET is less than 0.05 mm/day are shown in grey. Each panel reports the mean and one standard deviation of the PET-ET difference (bias) across all ocean grid cells. Basemap in a-f made with Natural Earth56. Supplementary Texts 1 and 2, Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zhou, S., Yu, B. Neglecting land–atmosphere feedbacks overestimates climate-driven increases in evapotranspiration. Download citation Received: 06 March 2025 Accepted: 15 August 2025 Published: 11 September 2025 DOI: https://doi.org/10.1038/s41558-025-02428-5 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Climate Change ISSN 1758-6798 (online) ISSN 1758-678X (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02118-0'>Phenazines contribute to microbiome dynamics by targeting topoisomerase IV</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 09:29:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Phenazines are highly prevalent, natural bioactive substances secreted by microbes. However, their mode of action and potential involvement in shaping microbiomes remain elusive. Here we performed a comprehensive analysis of over 1.35 million bacterial genomes to identify phenazine-producing bacteria distributed across 193 species in 34 families. Analysis of rhizosphere microbiome and public rhizosphere metagenomic datasets revealed that phenazines could shape the microbial community by inhibiting Gram-positive bacteria, which was verified by pairwise interaction assays using Phenazine-1-carboxamide (PCN)-producing Pseudomonas chlororaphis. PCN induced DNA damage in Bacillus subtilis, a model Gram-positive target, where it directly bound to the bacterial topoisomerase IV, inhibiting its decatenation activity and leading to cell death. A two-species consortium of phenazine-producing Pseudomonas and resistant B. subtilis exhibited superior synergistic activity in preventing Fusarium crown rot in wheat plants. This work advances our understanding of a prevalent microbial interaction and its potential for biocontrol. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Amplicon sequencing data, genome sequences of PCN-resistant strains, and transcriptomic data of PCN-treated B. subtilis 3610 have been deposited in the Genome Sequence Archive of the Beijing Institute of Genomics (BIG) Data Center with accession numbers CRA028325, CRA028335 and CRA019862. The information on the phz+ phenazine biosynthesis gene cluster is available in the Science Data Bank at https://www.scidb.cn/s/EfmaIb (ref. All data supporting the findings of this study are available in this article and its Supplementary Information. Source data are provided with this paper. The scripts employed in the ‘Identifier of Phenazine-Producing Bacteria (IPPB)' pipeline are available in GitHub at https://github.com/xzliu919/Identifier-of-Phenazine-Producing-Bacteria-IPPB- (ref. Lee, J. Y., Tsolis, R. M. & Baumler, A. J. The microbiome and gut homeostasis. Argaw-Denboba, A. et al. Paternal microbiome perturbations impact offspring fitness. Duran, P. et al. Microbial interkingdom interactions in roots promote Arabidopsis survival. Wilde, J., Slack, E. & Foster, K. R. Host control of the microbiome: mechanisms, evolution, and disease. Zhang, J. Y. et al. NRT1.1B is associated with root microbiota composition and nitrogen use in field-grown rice. Cordero, O. X. et al. Ecological populations of bacteria act as socially cohesive units of antibiotic production and resistance. Goodrich, J. K., Davenport, E. R., Waters, J. L., Clark, A. G. & Ley, R. E. Cross-species comparisons of host genetic associations with the microbiome. Lyng, M. & Kovacs, A. T. Frenemies of the soil: Bacillus and Pseudomonas interspecies interactions. Trends Microbiol. Garcia-Bayona, L. & Comstock, L. E. Bacterial antagonism in host-associated microbial communities. Kehe, J. et al. Positive interactions are common among culturable bacteria. Dahlstrom, K. M., McRose, D. L. & Newman, D. K. Keystone metabolites of crop rhizosphere microbiomes. Krespach, M. K. C. et al. Streptomyces polyketides mediate bacteria–fungi interactions across soil environments. Purtschert-Montenegro, G. et al. Pseudomonas putida mediates bacterial killing, biofilm invasion and biocontrol with a type IVB secretion system. Barone, G. D. et al. Implications of bacteria–bacteria interactions within the plant microbiota for plant health and productivity. J. Zhejiang Univ. Article PubMed Mavrodi, D. V., Blankenfeldt, W. & Thomashow, L. S. Phenazine compounds in fluorescent Pseudomonas spp. biosynthesis and regulation. Dar, D., Thomashow, L. S., Weller, D. M. & Newman, D. K. Global landscape of phenazine biosynthesis and biodegradation reveals species-specific colonization patterns in agricultural soils and crop microbiomes. Saunders, S. H. et al. Extracellular DNA promotes efficient extracellular electron transfer by pyocyanin in Pseudomonas aeruginosa biofilms. McRose, D. L. & Newman, D. K. Redox-active antibiotics enhance phosphorus bioavailability. McRose, D. L., Li, J. Y. & Newman, D. K. The chemical ecology of coumarins and phenazines affects iron acquisition by Pseudomonads. & Newman, D. K. Phenazines and toxoflavin act as interspecies modulators of resilience to diverse antibiotics. Moura-Alves, P. et al. AhR sensing of bacterial pigments regulates antibacterial defence. Guo, S. Q. et al. Microbial synthesis of antibacterial phenazine-1,6-dicarboxylic acid and the role of PhzG in Pseudomonas chlororaphis GP72AN. Perry, E. K. & Newman, D. K. Prevalence and correlates of phenazine resistance in culturable bacteria from a dryland wheat field. Article PubMed Kozdrój, J., Trevors, J. T. & van Elsas, J. D. Influence of introduced potential biocontrol agents on maize seedling growth and bacterial community structure in the rhizosphere. & Nielsen, J. Phenazine natural products: biosynthesis, synthetic analogues, and biological activity. Chen, Y. et al. Wheat microbiome bacteria can reduce virulence of a plant pathogenic fungus by altering histone acetylation. Briard, B. et al. Pseudomonas aeruginosa manipulates redox and iron homeostasis of its microbiota partner Aspergillus fumigatus via phenazines. Hotter, V. et al. A polyyne toxin produced by an antagonistic bacterium blinds and lyses a Chlamydomonad alga. Mavrodi, D. V. et al. Diversity and evolution of the phenazine biosynthesis pathway. Imai, Y. et al. Evybactin is a DNA gyrase inhibitor that selectively kills Mycobacterium tuberculosis. Perry, E. K., Meirelles, L. A. & Newman, D. K. From the soil to the clinic: the impact of microbial secondary metabolites on antibiotic tolerance and resistance. Ran, H. M., Hassett, D. J. & Lau, G. W. Human targets of Pseudomonas aeruginosa pyocyanin. Vos, S. M., Tretter, E. M., Schmidt, B. H. & Berger, J. M. All tangled up: how cells direct, manage and exploit topoisomerase function. Laponogov, I. et al. Structural basis of gate-DNA breakage and resealing by Type II topoisomerases. Zhang, G., Wang, C., Sui, Z. H. & Feng, J. Insights into the evolutionary trajectories of fluoroquinolone resistance in Streptococcus pneumoniae. Mostertz, J., Scharf, C., Hecker, M. & Homuth, G. Transcriptome and proteome analysis of Bacillus subtilis gene expression in response to superoxide and peroxide stress. Keren, I., Wu, Y., Inocencio, J., Mulcahy, L. R. & Lewis, K. Killing by bactericidal antibiotics does not depend on reactive oxygen species. Cell death from antibiotics without the involvement of reactive oxygen species. Yan, J. J. et al. Advances in phenazines over the past decade: review of their pharmacological activities, mechanisms of action, biosynthetic pathways and synthetic strategies. Gonçalves, T. & Vasconcelos, U. Colour me blue: the history and the biotechnological potential of pyocyanin. Dey, T. et al. Outcomes of clofazimine for the treatment of drug-resistant tuberculosis: a systematic review and meta-analysis. Appelbaum, P. C. & Hunter, P. A. The fluoroquinolone antibacterials: past, present and future perspectives. Swanson, K. et al. Generative AI for designing and validating easily synthesizable and structurally novel antibiotics. Su, P. et al. Microbiome homeostasis on rice leaves is regulated by a precursor molecule of lignin biosynthesis. Netzker, T., Shepherdson, E. M. F., Zambri, M. P. & Elliot, M. A. Bacterial volatile compounds: functions in communication, cooperation, and competition. He, Y. W. et al. DSF-family quorum sensing signal-mediated intraspecies, interspecies, and inter-kingdom communication. Trends Microbiol. Molina-Santiago, C. et al. The extracellular matrix protects Bacillus subtilis colonies from Pseudomonas invasion and modulates plant co-colonization. Molina-Santiago, C. et al. Chemical interplay and complementary adaptative strategies toggle bacterial antagonism and co-existence. Korgaonkar, A., Trivedi, U., Rumbaugh, K. P. & Whiteley, M. Community surveillance enhances Pseudomonas aeruginosa virulence during polymicrobial infection. Giddens, S. R., Feng, Y. & Mahanty, H. K. Characterization of a novel phenazine antibiotic gene cluster in Erwinia herbicola Eh1087. Pierson, L. S. & Pierson, E. A. Metabolism and function of phenazines in bacteria: impacts on the behavior of bacteria in the environment and biotechnological processes. Mentel, M. et al. Of two make one: the biosynthesis of phenazines. Li, J., Wu, Q. & Zhu, X. Research progress of phenazine-1-carboxylic acid and its analogue. Madeira, F. et al. The EMBL-EBI search and sequence analysis tools APIs in 2019. Nucleic Acids Res. McDonald, M., Mavrodi, D. V., Thomashow, L. S. & Floss, H. G. Phenazine biosynthesis in Pseudomonas fluorescens branchpoint from the primary shikimate biosynthetic pathway and role of phenazine-1,6-dicarboxylic acid. Shemesh, M. & Chai, Y. A combination of glycerol and manganese promotes biofilm formation in Bacillus subtilis via histidine kinase KinD signaling. Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Callahan, B. J. et al. DADA2: high-resolution sample inference from Illumina amplicon data. Quast, C. et al. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Nucleic Acids Res. Rognes, T., Flouri, T., Nichols, B., Quince, C. & Mahé, F. VSEARCH: a versatile open source tool for metagenomics. & Holmes, S. phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. Segata, N. et al. Microbial community function and biomarker discovery in the human microbiome. Article PubMed Central Chen, I. M. A. et al. IMG/M: integrated genome and metagenome comparative data analysis system. Nucleic Acids Res. Wood, D. E., Lu, J. Improved metagenomic analysis with Kraken 2. Buchfink, B., Xie, C. & Huson, D. H. Fast and sensitive protein alignment using DIAMOND. Article PubMed Gryczan, T. J., Contente, S. & Dubnau, D. Characterization of Staphylococcus aureus plasmids introduced by transformation into Bacillus subtilis. Chai, Y., Chu, F., Kolter, R. & Losick, R. Bistability and biofilm formation in Bacillus subtilis. Ben-Yehuda, S., Rudner, D. Z. & Losick, R. RacA, a bacterial protein that anchors chromosomes to the cell poles. Peters, J. M. et al. A comprehensive, CRISPR-based functional analysis of essential genes in bacteria. Methods for Dilution Antimicrobial Susceptibility Test for Bacteria that Grow Aerobically: Approved Standard CLSI Document M07-A9, 19th edn (CLSI, 2012). & Chen, Y. phz+ strains data [DS/OL]. Science Data Bank https://doi.org/10.57760/sciencedb.25989 (2025). Liu, C. Identifier-of-Phenazine-Producing-Bacteria (IPPB). We thank R. Losick (Harvard University, USA) for his suggestions during the revision of the paper; K. Lewis (Northeastern University, USA) for providing the E. coli MG1655, WO153 and ΔtolC mutant strains; and J. Feng (Chinese Academy of Sciences, China) for providing the S. pneumoniae R6WT and R6M-3 strains. This research was supported by the Natural Science Foundation of Zhejiang Province (LZ23C140004) to Y. Chen, National Natural Science Foundation of China U21A20219 to Z.M., National Key R&D Program of China 2022YFD1400100 to Y. Chen, National Natural Science Foundation of China U24A20808 to H.L., and ‘Pioneer' and ‘Leading Goose' R&D Program of Zhejiang 2023C02030 to Y. Chen. State Key Laboratory of Rice Biology and Breeding, Zhejiang Key Laboratory of Biology and Ecological Regulation of Crop Pathogens and Insects, Institute of Biotechnology, Zhejiang University, Hangzhou, China Yaqi Zhou, Hongkai Wang, Jiaxin Sun, Chao Liu, Zhonghua Ma & Yun Chen Institute of Environmental Biotechnology, Graz University of Technology, Graz, Austria Wisnu Adi Wicaksono & Gabriele Berg Department of Biology, Northeastern University, Boston, MA, USA Yinghao He, Yuxuan Qin & Yunrong Chai Leibniz Institute for Agricultural Engineering and Bioeconomy (ATB), Potsdam, Germany Institute for Biochemistry and Biology, University of Potsdam, Potsdam, Germany Marine Drug Integrated Innovation Center, Ren Ji Hospital, School of Medicine, Shanghai Jiao Tong University, Shanghai, China Lei Li & Houwen Lin Peking-Tsinghua Center for Life Sciences, College of Life Sciences, Peking University, Beijing, China School of Biological Sciences, Faculty of Environmental and Life Sciences, University of Southampton, Southampton, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Y. Chen initiated, coordinated and supervised the project. performed and analysed most of the experiments. performed MIC tests. collected and analysed the phz+ strains data. collected and analysed the metagenome and metatranscriptome datasets. performed ROS staining experiments. constructed the KD-ParE/C and KD-GyrA/B strains. performed structural determination of phenazine-1-carboxylic acid. wrote the paper. Y. Chen, Z.M., T.C., Y.B., Y. Chai and G.B. revised the paper. Correspondence to Yun Chen. The authors declare no competing interests. Nature Microbiology thanks Wulf Blankenfeldt, Axel Brakhage and Ákos T Kovács for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic representation of the Identifier of Phenazine-Producing Bacteria (IPPB) pipeline. The pipeline automates the search for homologous proteins of phenazine biosynthetic core proteins (PhzA/B, PhzC, PhzD, PhzE, PhzF, PhzG) within a strain's genome. Protein identity thresholds are applied as indicated, and strains are classified as phz+ if all core proteins are identified and the corresponding genes are located within 20 kb of each other. Strains meeting these criteria are classified as phz+ bacterium, and included in the phylogenetic tree. Abbreviations: NGDC, National Geoscience Data Centre; IMG/M, Integrated Microbial Genomes & Microbiomes system. b, Schematic representation of 23 reference phenazine biosynthetic gene clusters. Core biosynthetic genes (phzA/B, phzC, phzD, phzE, phzF, phzG) are shown in color, while adjacent, non-core genes are depicted in grey. a, Maximum-likelihood phylogenetic tree of the phz+ bacterial genera based full-length 16S rRNA gene sequences. Phylum-level classifications are indicated. Bar plots on the right show the number of species within each phz+ genus and the total number of phz+ strains. b, 1H NMR (nuclear magnetic resonance) spectrum of phenazine-1-carboxylic acid (PCA) produced in S. rosea LHW63015. c, 13C NMR spectrum of PCA. d, DEPT135 NMR spectrum of PCA. e, X-ray ORTEP drawing for PCA. a, Number of ASVs in samples treated with phz+ ZJU60 and its phenazine mutant ΔphzA-H. b, Shannon diversity index in samples treated with phz+ ZJU60 and ΔphzA-H. c, Community composition in samples treated with phz+ ZJU60 and ΔphzA-H. d, Dynamics of wheat rhizosphere microbiota following soil irrigation with ZJU60 and ΔphzA–H. Soil samples were collected on days 3 and 7 after irrigation with ZJU60 and ΔphzA–H in wheat field microcosms. 16S rRNA gene fragment amplicon sequencing revealed distinct temporal changes in the composition and structure of the rhizosphere microbiomes. Data are presented as means ± s.d. of n = 5 biological replicates. For graphs shown in a, b and d, letters that are different from one another indicate that their means are statistically different using a two-sided ANOVA and Tukey's HSD (P < 0.05). a, Correlation analysis between the relative abundances (RA) of phz+ bacteria and Pseudomonas in rhizosphere microbiomes. b-d, Pairwise Spearman correlation between the abundance of the three most prevalent bacterial taxa at each taxonomic level and the abundance of phz+ Pseudomonas. The relative abundances of the analyzed taxa were obtained from shotgun-sequenced metagenome datasets and subsequently log-transformed. The Spearman correlation coefficient (R) and corresponding P value was calculated using the ggscatter function from the ggpubr package in R. The grey shaded area indicates the 95% confidence interval of the regression line, shown in blue. a, Correlation analysis between phzA/BDEFG gene transcripts (log-transformed counts per million, CPM) and the relative abundances (RA) of bacteria assigned to the class Bacilli. b, Correlation analysis between the phzA/BDEFG gene transcripts and the relative abundance of bacteria assigned to the genus Pseudomonas. Metatranscriptomic datasets were used to determine both gene expression and the relative abundances of the target taxa. The Spearman correlation coefficient (R) and corresponding P value (P < 0.05) were calculated using the ggscatter function from the ggpubr package in R. The grey shaded area indicates the 95% confidence interval of the regression line, shown in blue. The antimicrobial activity of ZJU60 was assessed on LBGM agar medium against 70 bacterial species. One representative strain from each species was selected for testing. ZJU60 was inoculated on the top of a lawn of the individual test strains, and inhibition zones were indicated by dashed circles after 72 h of incubation. Scale bar = 5 mm. Detailed information on the tested strains can be found in Supplementary Table 1. a, b, Time-resolved interactions between P. chlororaphis ZJU60 or ΔphzA-H and B. subtilis 3610-GFP over 24, 48, and 72 h in LBGM agar medium (left panel). The log(CFU·mL⁻¹·cm⁻²) of ZJU60 or ΔphzA-H and 3610 in the interaction areas at different time points (24, 48, 72 h) is shown in the right panel. Cells from the three sections of the interaction zone were collected, sonicated, fully re-suspended, diluted, and plated to determine the CFU of each species. Scale bar = 8 mm. c, Time-resolved interactions between ΔphzA-H (supplemented with 50 µg/mL PCN) and B. subtilis 3610 over 24, 48, and 72 h in LBGM medium. d, Growth curves of B. subtilis 3610 with ZJU60 supernatant, ΔphzA-H supernatant, and ΔphzA-H supernatant supplemented with 50 µg/mL PCN. Data for graphs in a, b and d are presented as means ± s.d. For a and b, n = 3 biological replicates. For d, n = 6 biological replicates. Differential interference contrast (DIC) images (left) showing cell morphology of four PCN-sensitive bacterial strains in the absence and presence of PCN. Scale bar = 5 μm. Quantification of cell length from images (right). The graphs represent results from one representative experiment. Statistical differences were analyzed by a two-sided Student's t-test. Data are presented as means ± s.d. of n = 50 individual cells. Experiments were independently repeated three times with similar results. a, Fluorescence staining of reactive oxygen species (ROS) signals following PCN treatment of B. subtilis 3610. Scale bar = 10 μm. b, Quantification of fluorescence intensity from panel a. Data are presented as mean ± s.d. of n = 78 individual cells. Differences between groups were analyzed using a two-sided Student's t-test. c, Analysis of the expression of ROS response-related mutant genes following PCN treatment. d, Growth curves for B. subtilis 3610 and ROS-scavenging mutants, with or without treatment with 15 μg/mL PCN. Data are presented as means ± s.d. of n = 3 biological replicates. Supplementary Figs. Source data for Supplementary Fig. Statistical source data. Statistical source data. Unprocessed western blots and gels. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Zhou, Y., Wang, H., Sun, J. et al. Phenazines contribute to microbiome dynamics by targeting topoisomerase IV. Accepted: 06 August 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250910000242.htm'>Life on Mars? NASA discovers potential biosignatures in Martian mudstones</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 02:59:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Data and images from NASA's Mars Perseverance rover reveals that recently discovered rocks in Jezero crater are organic carbon bearing mudstones. The paper, led by Joel Hurowitz, PhD, Associate Professor in the Department of Geosciences at Stony Brook University, builds upon ongoing research conducted with the rover since it landed in 2021 – work aimed at characterizing early Martian geological processes and collecting samples that may someday be returned to Earth. While the research team is not claiming to have found some form of fossilized life on Mars, they do believe the rocks contain features that could have been formed by life – a potential biosignature. A potential biosignature is any characteristic, element, molecule, substance, or feature that could have been formed by past biological life, but which also could have formed in the absence of life. “These mudstones provide information about Mars' surface environmental conditions at a time hundreds of millions of years after the planet formed, and thus they can be seen as a great record of the planetary environment and habitability during that period,” says Hurowitz, who has been involved with Mars rover research since he was a graduate student at Stony Brook University in 2004. “We will need to conduct broader research into both living and non-living processes that will help us to better understand the conditions under which the collection of minerals and organic phases in the Bright Angel formation were formed,” he explains. Continued research will be done to assess the rocks and mudstone features. Scientists Sound Alarm: Why Are Rivers in Alaska Turning Orange? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250910000309.htm'>Surprising gut discovery reveals a hidden trigger of diabetes and liver disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-11 02:59:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The findings, published in Cell Metabolism on July 29, 2025, could open the door to new therapies to treat metabolic diseases like type 2 diabetes and fatty liver disease. The Canadian team found that obese mice -- and even people with obesity -- have higher levels of a lesser-known molecule, D-lactate, in their blood. Unlike the more familiar L-lactate made by muscles, most of the D-lactate comes from gut microbes and was shown to raise blood sugar and liver fat more aggressively. To stop this, the researchers created a "gut substrate trap" -- a safe, biodegradable polymer that binds to D-lactate in the gut and prevents it from being absorbed. Mice fed this trap had lower blood glucose, less insulin resistance, and reduced liver inflammation and fibrosis -- all without changing their diet or body weight. "This is a completely new way to think about treating metabolic diseases like type 2 diabetes and fatty liver disease. Instead of targeting hormones or the liver directly, we're intercepting a microbial fuel source before it can do harm," says Schertzer, a member of the Centre for Metabolism, Obesity, and Diabetes Research (MODR) and Farncombe Family Digestive Health Research Institute at McMaster. Schertzer holds a Canada Research Chair in Metabolic Inflammation. Note: Content may be edited for style and length. New Smart Pimple Patch Clears Acne in Just 7 Days Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            