
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-06</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/space/deep-space/a69254312/how-and-when-the-universe-will-end/'>Scientists Think They Figured Out How and When the Universe Will End</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists have recently used an updated understanding of the math of our universe to theorize that the universe will end in a crunch rather than entropy into oblivion. The field of cosmology may be very vocabulary-dense, but the bottom line is that one of our key numbers—the cosmological constant—may have switched teams from positive to negative. Those who publish research on cosmology must work in a different way than many other sciences, because key moments like the Big Bang and an eventual Big Crunch are almost completely outside current human understanding. We can wind time backward by studying stars, black holes, exotic structures, and even the shadows we believe may lead to dark matter or dark energy. The cosmological constant is one effort to help answer these huge existential questions. We know that much of the mass and energy in our universe is invisible, or “dark,”  but we don't know the nature of this missing stuff, nor the source of it (though physics indicates that we're likely swimming amid dark matter at all times). The universe was “gaining weight,” so to speak, so it would continue to expand at a faster and faster rate because the “weight” grew at a constant rate. But this changed after recent results from both the Dark Energy Survey (DES) and the Dark Energy Spectroscopic Instrument (DESI) earlier this year. Scientists studied the data and found that the cosmological constant may not be constant at all, but rather changing over time and indicating that its influence may be weakening. Cosmology has been rippling from these DES and DESI revelations for months now, leading to research like this new paper—written by scientists based in the U.S., China, and Spain—which appears now in the peer reviewed journal Journal of Cosmology and Astroparticle Physics. These scientists have built a new, adjusted model for the expected lifespan of the universe by taking a possible negative cosmological constant into account. If the universe will stop expanding at some point and begin to contract, that's a huge change from the understanding we had of the state of things at this same time a few years ago. Their conclusions bring with them some newly opened doors. So, for those hoping to leave a letter for the new universe 18.9 billion years from now, the next paradigm may not have humans, paper, or even stars. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. This Space Object May Be From a Dead Civilization If Aliens Exist, Why Haven't We Heard From Them? The Best Star Projectors Turn Your Ceiling Into a Night Sky A Scientist Says This Object May Be an Alien Craft Aliens May Have Started Life on Earth, Expert Says</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/alarm-grows-over-proposed-giant-mirrors-in-orbit-and-other-commercial-space/'>Giant Mirrors, Orbital Data Centers and Space-Based Advertisements Could Soon Clutter the Night Sky</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Seeking Profits, Private Companies Look to Light up the Night Sky Reflect Orbital's plan to deliver “sunlight on demand” using thousands of giant orbital mirrors is just the latest in a growing list of disruptive commercial activities in space For astronomers, the sky isn't exactly falling—yet the sky-high ambitions of tech companies seeking profits in Earth orbit and beyond are becoming too disruptive to ignore. SpaceX's Starlink Internet service, built with thousands of telescope-photobombing satellites, is the poster child for this problematic trend, but it's not alone. The latest start-up with brash out-of-this-world plans is Reflect Orbital, which has built a business case for beaming sunlight from orbit to power solar farms after dark. Marketed as “sunlight on demand,” Reflect Orbital's high-frontier initiative is just one among many; other companies in the proliferating space industry want to launch space advertisements, human remains and made-to-order artificial meteor showers. Tolkien, would probably make the anti-industrial author roll in his grave. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. “It's simple but not easy,” says Darren McKnight, a systems engineer and senior technical fellow at LeoLabs, a spacecraft- and debris-tracking company based in Menlo Park, Calif. “People look at each individual technology and say, ‘See, it's possible,' but don't put it all together.” Overheating and station-keeping could be big problems for the sprawling, sunbathed satellite, as could the precise control required to pinpoint a reflected beam onto targets far below. Overcoming these overlapping challenges would be a tall order, and the transmission losses alone could be astronomical across such vast distances, McKnight says. This approach is simply a reckless and inefficient use of Earth orbit, a precious and finite resource.” In a statement to Scientific American, Reflect Orbital's chief strategy officer Ally Stone said the company “is committed to protecting dark skies,” and that its first missions would involve “tightly controlled light spots steered well away from observatories and sensitive areas.” If the company's plans come to fruition, following its tests next year it will begin launching more mirror-toting satellites, ultimately building a mega constellation of 4,000 by 2030. Each would be capable of casting a 5 km-wide beam about four times brighter than the full moon down to Earth. But atmospheric scattering would ensure that some light escapes each beam, says John Barentine, a Tucson, Ariz.–based astronomer and executive officer of Dark Sky Consulting, which advises companies and city officials on outdoor lighting use. Large numbers of satellites in low-Earth orbit are crucial to Reflect Orbital's plans because a daisy-chain approach is required to consistently illuminate a target on the ground. A single satellite there could only beam sunlight to a surface target for some four minutes before flying out of range, whereupon another satellite would take over with its own beam. This process could continue for an hour or two during twilight and dawn. Besides Reflect Orbital's planned fleet of satellites, he cites other companies' bright spacecraft—not only SpaceX's Starlink mega constellation, which now includes more than 8,000 among its ranks, but also Amazon's growing Project Kuiper satellite fleet. Reflect Orbital's plan isn't an isolated phenomenon, says Jordan Bimm, a space historian and an assistant professor of science communication at the University of Chicago. “Humans have been sending weird stuff to space for a while,” he says. Numerous times, NASA astronauts have brought up various trinkets or smuggled items to space, including an unauthorized corned beef sandwich taken on the Gemini III mission in 1965 and a gorilla suit brought to the International Space Station in 2016. Furthermore, robotic missions have often included “festooning,” stowing or bolting small, nontechnical elements that don't directly involve the mission on landers and rovers, often for public engagement or other purposes. The past few years have brought multiple contentious space projects and concepts: For the first test flight of its Falcon Heavy rocket in 2018, SpaceX launched an entire Tesla Roadster into space, where the gradual, chaotic effects of orbital mechanics could eventually push it on a collision course with Earth or Mars. And not one but two companies had “lunar memorial services” (that is, cremated human remains) as payloads onboard last year's Astrobotic lunar lander, which failed in Earth orbit. And the Japanese company ALE wants to deploy spacecraft that would generate high-visibility artificial meteor showers on demand. The trend extends into proposals for commercial space infrastructure as well. Tech companies such as Nvidia and Hewlett Packard Enterprise have begun exploring the feasibility of launching solar-powered space-based data centers, and space-tech titans Jeff Bezos and Elon Musk have each recently voiced vigorous support for the idea. And of course, billionaires themselves have traveled to space, including founders of companies that are leading efforts to design the first commercial space stations, which may gain prominence when government facilities such as the International Space Station cease operations. For most of the 20th century, space activities remained largely the purview of powerful nations, with an initial emphasis on cold-war-era military applications, followed by more science-centric “soft power” feats such as moon landings, interplanetary missions and orbital telescopes, Bimm points out. “In general, nation states remain responsible for authorizing and supervising their respective commercial space activities,” but some regulations need updating, he says. For U.S.-based companies, that means oversight is critical from regulatory agencies such as the Federal Aviation Administration and the Federal Communications Commission, and other countries should make similar efforts, he says. Boley and other experts don't oppose companies pursuing far-out proposals per se. Investing “strategic R&D” in an ambitious idea that might not pan out anytime soon sometimes yields unexpected science and technology breakthroughs, McKnight says. But potential innovations can't be the only consideration, Bimm says. “I think being bold is important, but you must also be thoughtful.” Ramin Skibba is a science writer based in San Diego, Calif. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69206614/warrior-grave-satellite/'>A Satellite Spotted a Lump in the Ground. It Was the Grave of an Ancient Warrior.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Robbers may have stolen some exciting artifacts from a seventh-century Avarian warrior's grave, but they didn't take everything. Fortunately, among what they had left behind was a stunningly rare, curved iron saber—a 1,300-year-old find that, after its recent discovery, has gone on display in a Hungarian museum. The find came thanks to the Hungarian National Museum's Cemeteries from Space program, which uses satellite imagery to pinpoint potential archaeological sites worth exploring. In addition to the sword, those looters also left behind arrowheads, a silver belt, earrings with glass pearl studs, decorative metal hair braids, and a knife. “These cemeteries prove that Fejer County was an important center 1,400 years ago,” said historian Gyorgy Szabados in a translated statement from the museum, according to Heritage Daily. “The saber discovery adds weight to that claim, offering both a tangible artifact and a symbol of the region's place in early medieval history.” Just who this particular warrior was, though, remains a mystery. The type of saber suggests that the individual was of a high social rank, and other expensive grave goods back that theory up. With no known written language, the culture's history has been tricky to trace. Roughly 80 potential graves remain unexplored at the site, and from them, experts hope to gain more insight into Avar history. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Are There More Humans on Earth Than We Thought? AUV Reveals Stunning Images of Dotson Ice Shelf Our Universe May Be a Simulation, Scientist Says Roman Military Camp Discovered High in Swiss Alps</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44160-025-00919-z'>Reductive radical chain initiation through the thermal generation of carbon dioxide radical anion</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 10:33:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Synthesis (2025)Cite this article Radical chain initiation strategies are fundamental to the synthesis of small molecule drugs and macromolecular materials. Modern methods for initiation through one-electron reduction are largely dominated by photo- and electrochemistry but the large-scale industrial application of these methods is often hampered by scalability challenges. Here we report a general, thermally driven and scalable method for the reductive initiation of radical chains that involves reacting an inexpensive azo initiator with a formate salt to form a carbon dioxide radical anion. Substoichiometric quantities of this initiator system were used to form C(sp2)–C(sp3), C(sp2)–S, C(sp2)–H, C(sp2)–B and C(sp2)–P bonds from complex (hetero)aryl halides, with high chemoselectivity and under transition-metal-free conditions. The developed initiator system was also used to probe the mechanism of other radical reactions. The controlled initiation of radical chains is a subject of fundamental importance to polymer science1, organic synthesis2,3, atmospheric chemistry4 and biochemistry5. In the context of synthetic organic chemistry, a substantial proportion of radical reactions are driven by chains6,7, but this aspect is often obscured—especially if the length of the chain is short. If the chain length is short, the reaction must be continuously reinitiated for its duration8. This is a major reason why photo- and electrochemistry have emerged as leading methods of initiation: they provide flexible and programmable frameworks to promote electron transfer and generate radicals at a controllable rate9,10,11,12,13. One drawback of photo-/electrochemical initiation strategies is their non-trivial scalability in a process development and manufacturing setting (beyond the milligram to gram scales used in earlier stages of medicinal chemistry)14,15. Indeed, both strategies require specialist reactor technologies, which may increase process development times and costs, limiting applications on the industrial scale when compared to standard manufacturing techniques (Fig. However, for chains driven by one-electron reduction (electron-catalysed and electron-transfer chain processes)16,17 photo- and electrochemical methods still far surpass the general utility of chemical/thermal initiation strategies using ground state electron donors18,19. Considering this current state-of-the-art, we sought to develop a general reductive initiation system with the following characteristics: (1) strongly reducing (\({{E}_{1/2}^{\circ}} < -2\,{\rm{V}}\) versus saturated calomel electrode (SCE)); (2) usable in substoichiometric quantities; (3) thermally controlled; (4) does not form chain-terminating persistent radicals (a traceless one-electron reductant); (5) compatible with a broad range of substrates; (6) applicable to a wide range of reactions; and (7) inexpensive and scalable using standard manufacturing vessels. a, One-electron reduction initiation strategies. b, Proposed thermal strategy using azo initiators to generate carbon dioxide radical anion. PC, photocatalyst; D, donor; \({E}_{1/2}^{\circ}\), half-wave potential. In this study, we describe the realization of this goal by heating inexpensive azo initiators commonly used in the polymer industry20 in the presence of formate salts to generate a strong one-electron reductant, carbon dioxide radical anion (CO2·−, \({{E}_{1/2}^{\circ }}=-{2.22}\,{\rm{V}}\) versus SCE)20,21. Inspired by advances in the photochemical generation of CO2•− (refs. 22,23,24,25,26,27,28,29,30), we hypothesized that electrophilic α-cyano alkyl radicals (derived from the thermal decomposition of an azo initiator) would readily abstract hydrogen atoms from formate salts to from CO2·− in a polarity-matched process (Fig. The proposed combination of an azo initiator and formate salt would provide an operationally simple ‘dump and stir' thermal method of initiation, which would satisfy the aforementioned criteria and be widely applicable across synthetic chemistry. To evaluate the feasibility of our mechanistic proposal, we monitored the reaction of an azo initiator and sodium formate by electron paramagnetic resonance (EPR) spectroscopy in the presence of the spin trap, 5,5-dimethyl-1-pyrroline-N-oxide (DMPO). 4,4-Azobis(4-cyanovaleric acid) (ACVA) was selected as the azo initiator, as unlike azobisisobutyronitrile (AIBN), ACVA is not classified as an explosive. When reacting ACVA, HCO2Na and DMPO, a species matching previously reported data for a DMPO–CO2·− adduct was detected (Fig. To verify if this species was indeed a CO2·− adduct, these experiments were repeated using H13CO2Na and the expected additional 13C hyperfine coupling was clearly observed (aiso13C = 33 MHz). These results strongly suggest that CO2·− is formed from the reaction of ACVA and sodium formate. The feasibility of polarity-matched hydrogen-atom transfer (HAT)32 between the ACVA-derived α-cyano alkyl radical I and formate II was assessed by density functional theory (DFT), which indicated that HAT is both kinetically and thermodynamically viable (ΔG‡ = 14.5 kcal mol−1, ΔG = −3.6 kcal mol−1; Fig. It is possible that the distonic nature of I and interactions between the α-cyano radical and carboxylate anion may play a role in the efficiency of this electron upconversion process (the conversion of a weak/mild reductant into a stronger reductant product)17, but any such interactions are probably weakened by the highly polar dimethylsulfoxide (DMSO) solvent environment33,34,35,36. a, EPR spectroscopy to probe the formation of CO2·−. b, Computational study of HAT conducted at the M062X-D3(0) def2-TZVP SMD(DMSO) level of theory. c, Optimization of the proposed carbonyl α-arylation reaction. giso, isotropic g value; aiso, isotropic a value (the hyperfine coupling constant); Ar, aryl; ACHN, 1,1'-azobis(cyclohexane-1-carbonitrile); AAPH, 2,2'-azobis(2-amidinopropane) dihydrochloride; AIPH, 2,2′-azobis[2-(2-imidazolin-2-yl)propane] dihydrochloride; NA, not applicable. Having validated our key mechanistic hypothesis, model synthetic applications were explored to determine the generality of this initiation strategy. We first targeted C(sp2)–C(sp3) bond-forming carbonyl α-arylation reactions due to the synthetic utility and prevalence of the α-aryl carbonyl motif in biologically active compounds37,38. Moreover, we posited that a general transition-metal-free radical strategy for carbonyl α-arylation would: (1) complement the scope of existing methods37,38 by providing improved compatibility with heteroaromatic systems, which are common poisons for metal catalysts; and (2) avoid issues related to the cost and toxicity of transition metals, as well as the geopolitical, ethical and environmental concerns regarding their supply chains39,40. Here, we hypothesized that CO2·− could initiate the electron-catalysed unimolecular radical-nucleophilic substitution (SRN1)41,42 of aryl halides with enolate nucleophiles43,44. Indeed, CO2·− is known to readily reduce aryl halides to form aryl radicals22,23,45. Thus, aryl halide 1 was reacted with ethyl acetoacetate 2 (4.0 equiv. and potassium formate (0.5 equiv.) in DMSO at 80 °C for 4 h (Fig. Pleasingly, following the addition of ammonium chloride (to promote complete deacetylation of the intermediate 1,3-dicarbonyl 3), this one-pot procedure produced deacetylated α-aryl ester 4 in 79% yield and hydrodehalogenated side product 5 in 10% yield (Fig. Hydrodehalogenated product 5 is proposed to form from a slower competing HAT chain in which the aryl radical abstracts a hydrogen atom from the formate salt (reforming CO2·−). Other commercially available azo initiators were trialled, but none proved superior to ACVA (entries 2–5). No product formation was observed in the absence of ACVA and HCO2K (entry 6). However, some product formation was observed with HCO2K alone (entry 7); we attribute this background reactivity to inefficient spontaneous initiation events—in which the anionic nucleophile itself serves as a weak one-electron reductant45—being amplified by CO2·− formation. Repeating the optimal reaction with ACVA and HCO2K in the presence of air reduced the yield of product 4 to 54% (entry 8). Moreover, all reactivity was completely supressed in the presence of 2,2,6,6-tetramethylpiperidine-1-oxyl (TEMPO) (entry 9). Finally, repeating the reaction in the absence of the nucleophile 2 afforded only the hydrodehalogenated product 5 in 70% yield (entry 10). These findings are all consistent with an SRN1 electron-transfer chain process. Other compatible, but less effective formate salts and reaction conditions are described in Supplementary Tables 1–3. With optimized conditions in hand, the scope and application of this thermally initiated transformation were explored (Fig. First, the importance of the nucleofuge was examined with para-halobenzonitrile derivatives. Pleasingly, the iodo, bromo and chloro derivatives were all converted into α-aryl ester 4 in 54%, 67% and 53% yield, respectively. Conversely, only 7% of 4 was formed from the corresponding fluoride, which indicates that the contribution of a polar SNAr mechanism to these reactions is probably minimal. Moreover, efficient reactivity was observed regardless of the arene substitution pattern because the meta- and ortho-substituted products 6 and 7 were both formed in good yields. Other electron-deficient nitrobenzene and aryl sulfone derivatives were similarly compatible and converted into products 8–13 in 30–85% yield. Interestingly, the reaction of 4-iodonitrobenzene formed product 11 in 66% yield in the absence of ACVA and HCO2K. We attribute this reactivity to spontaneous initiation, which is commonly observed when reacting anionic nucleophiles with easily reduced substrates (strongly electron-deficient aryl iodides)46,47. Carbonyl and ester derivatives were also tolerated and converted into α-aryl esters 14–20 in good to excellent yields. Notably, ester 20 could be formed in 77% yield from a dihalogenated substrate, illustrating the selectivity of this method for heavier halogens. Reactivity was still observed in the absence of strong electron-withdrawing resonance effects: bistrifluoromethyl derivative 21 was prepared in 66% yield. However, less electron-deficient systems displayed lower levels of reactivity as demonstrated by the formation of 22 in 15% yield. Moreover, no α-aryl ester product was formed from electrophiles bearing strong electron-donating groups such as 4-iodoanisole, but hydrodehalogenated product 23 was formed in 42% yield. This important result indicates that while the ACVA–formate initiator system is capable of reducing challenging electron-rich systems, the model SRN1 electron-transfer chain was not viable (potentially due to inefficient radical-anion coupling). Attention then turned towards our original target of heteroaryl halides. Pleasingly, pyridine, pyrazine and pyrimidine azaarenes, including the drug etoricoxib, with varying substitution patterns were all tolerated, furnishing α-aryl esters 24–35 in 26–63% yield. Finally, a variety of fused bicyclic heteroaryl halides, including quinolines, isoquinolines, quinoxalines, quinazolines, benzothiazoles, pyrazolopyrimidines and imidazopyridazines, were also compatible and substituted to form products 36–43 in 13–88% yield. a, Scope of aryl and heteroaryl halides. b, Scale-up at AstraZeneca. aSee Supplementary Information, ‘Experimental Procedures and Characterisation Data', for variations of the standard reaction conditions. bYields determined by 1H NMR spectroscopy against an internal standard (1,3,5-trimethoxybenzene). cNo ACVA or formate added. Next, in collaboration with the process chemistry group at AstraZeneca, we sought to determine if the developed method and initiator system were sufficiently scalable to allow widespread use in the pharmaceutical and related industries. Therefore, the safety of the model reaction system was examined, in particular the use of ACVA, DMSO and base at elevated temperature. Differential scanning calorimetry and fall-hammer testing of ACVA, and high-rate Carius tube testing of the reaction mixture, all confirmed that the reaction can be safely performed at 80 °C. The developed reaction was further optimized via high-throughput experimentation48 and time-course analysis, before being performed on a 50-g scale, which provided product 4 in 61% isolated yield and >95% purity following a simple precipitation and filtration process (Fig. This result was particularly pleasing considering the heterogeneous nature of the reaction mixture (common to many reactions using inorganic bases in organic solvents) because a lack of reaction homogeneity can complicate scale-up49. In addition, the clean reaction profile and lack of side products was notable as azo initiators are known to also react as electrophiles with strong nucleophiles50. Thus, the low-cost ACVA–formate initiator system appears highly amenable to large-scale applications (current prices: ACVA, £1.57 g−1; HCO2K, £0.11 g−1 (Sigma Aldrich)). Inspired by these promising findings, we sought to test the limits of the developed method and initiator system by exploring their compatibility with complex (hetero)aryl drug-like intermediates through microscale parallel screening at AstraZeneca (Fig. 4) (see Supplementary Information for more details). We had confidence in the ultrahigh-performance liquid chromatography (uHPLC)/mass spectrometry (MS)-based analysis because four substrates previously tested showed close agreement between isolated yields and uHPLC/MS area/area%. Pleasingly, 13 of the remaining 20 complex substrates, dense with Lewis basic-nitrogen functionality, afforded the desired products in synthetically useful yields under unoptimized conditions. To further validate these results, two of these ‘hits' (49 and 59) were performed on a preparative scale and isolated in comparable yield. These results demonstrate the broad synthetic utility of the developed initiator system and α-arylation methodology, which both appear ready for immediate application in industrial drug discovery programmes. The deacetylation step with ammonium chloride was not performed to facilitate a workflow without additional solid handling. Yields shown are combined uHPLC/MS area/area% of acetylated and deacetylated products (if applicable) at 220 nm (identified by MS analysis). Screening yields are visually summarized in a heatmap-style graphic (higher yields are brighter in colour). Isolated yields at 0.3-mmol scale for validation are reported in parentheses. aAccurate reporting of the reaction outcome was hampered by poor resolution of the reaction components across multiple analytical methods. Ts, p-toluenesulfonyl; Bn, benzyl; PMB, p-methoxybenzyl; Boc, t-butyloxycarbonyl. Having sufficiently explored the scope with respect to the electrophile, the nucleophile scope was investigated using 1 as a model substrate (Fig. First, closely related methyl and tert-butyl acetoacetates 67 and 68 were reacted to afford the corresponding α-arylated esters 69 and 70 in 77% and 42% yield, respectively. The ester functionality was not essential for reactivity because acetylacetone 71 was also converted into 72 (following a modified deacetylation protocol). Amide 73 was also tolerated and selectively converted into dicarbonyl 74 in 50% yield. More sterically congested nucleophiles such as 75 could be used to form α-methyl ester 76 in 55% yield. This result was particularly pleasing considering the prevalence of the α-methyl carboxylic acid moiety in non-steroidal anti-inflammatory drugs, as exemplified by the conversion of aryl bromide 77 into suprofen ethyl ester 78 in 42% yield. Cyclic ketone 79 was also successfully arylated to form 80 with a quaternary carbon centre in 27% yield. The ketone functionality was not essential for reactivity because ethyl cyanoacetate 81 was arylated to form 82 in 66% yield. In a similar fashion, malonates 83–85 were converted into 86–88 in 24–68% yield. Pleasingly, benzophenone glycine derivative 89 could also be used to form the corresponding N-protected unnatural α-aryl amino acid 90 in 43% yield. Finally, we examined non-anionic nucleophiles such as enamines using modified conditions based on the work of Gianetti and co-workers who arylated enamines under photoredox-catalysed conditions51. Here, the pyrrolidine-derived enamine of cyclohexanone 91 was formed in situ and arylated to afford 92 in 36% yield. This result supports the mechanistic hypothesis of Gianetti and co-workers who proposed that a chain mechanism was potentially operative alongside a photoredox catalytic cycle. In addition to applications in methodology development, the ACVA–formate initiator system may therefore be used as a general mechanistic tool to probe cases in which a radical chain mechanism is suspected. a, Scope of carbon-based nucleophiles and synthesis of suprofen ethyl ester. b, Scope of other nucleophiles and radical traps. aYields determined by 1H NMR spectroscopy against an internal standard (1,3,5-trimethoxybenzene). EWG, electron-withdrawing group; Ac, acetyl; tBu, tert-butyl; Tol, p-tolyl; Pin, pinacol. To conclude our studies, we chose to demonstrate the utility of the developed ACVA–formate initiator system beyond C(sp2)–C(sp3) bond formation. First, simply by exchanging the nucleophile for a thiol, 4-iodobiphenyl 93 was converted into diaryl thioether 94 in 62% yield (Fig. This reaction presumably proceeds via an SRN1 mechanism as thiolates are known to be excellent SRN1 nucleophiles42,52. Next, by omitting any nucleophile and increasing the equivalents of ACVA and HCO2K, aryl bromide 95 was converted into hydrodehalogenated product 96 in 61% yield. The addition of bis(pinacolato)diboron (B2Pin2) enabled aryl boronic ester 97 to be formed in 51% yield. In a similar fashion, phosphonate ester 98 was formed in 50% yield when using triethyl phosphite as a radical trap. These results illustrate that the developed initiator system can be broadly applied for reaction development. We have developed a general, scalable and inexpensive method to initiate radical chains through one-electron reduction by reacting an azo initiator with a formate salt. The power of this initiation approach was demonstrated through the development of a general carbonyl α-(hetero)arylation protocol, which enabled challenging and valuable structural motifs to be formed under remarkably simple reaction conditions. The scalability of this initiation approach and carbonyl α-(hetero)arylation protocol was demonstrated on a 50-g scale in an industrial setting. Moreover, compatibility with a range of complex substrates was illustrated through microscale parallel screening. Finally, in addition to C(sp2)–C(sp3) bond formation, the wider synthetic potential of the developed initiation system was illustrated through the development of C(sp2)–S, C(sp2)–H, C(sp2)–B and C(sp2)–P coupling reactions. Considering the multidisciplinary importance of radical chain chemistry, we anticipate that this inexpensive initiation strategy will find widespread application in both industry and academia. We also speculate that alongside large-scale manufacturing opportunities, the disclosed initiator system may serve as a valuable mechanistic tool to probe the chain character of other radical reactions. An 8-ml screw-cap vial was charged with ACVA (25.6 mg, 75 μmol, 0.25 equiv. ), HCO2K (12.6 mg, 150 μmol, 0.50 equiv. ), Cs2CO3 (440 mg, 1.35 mmol, 4.5 equiv. ), and if solid, the aryl halide coupling partner (1.0 equiv.). To the solids was sequentially added a magnetic stirrer bar, anhydrous DMSO (1.5 ml), ethyl acetoacetate (156 mg, 1.20 mmol, 4.0 equiv. ), and if liquid, the aryl halide coupling partner through the screw-cap septa. The reaction mixture was sparged with N2 for 15 min before being sealed with parafilm. The reaction mixture was then stirred at 80 °C in a metal heating block for 4 h. To promote complete deacetylation of intermediate 1,3-dicarbonyls, NH4Cl (80.3 mg, 1.50 mmol, 5.0 equiv.) was added. The reaction was then stirred at 50 °C in a metal heating block for 1 h. The reaction mixture was cooled to room temperature before being diluted with CH2Cl2 (10 ml) and water (10 ml). The organic phase was collected, and the aqueous phase was extracted with CH2Cl2 (3 × 10 ml). The organics were combined, washed with brine (25 ml), dried (MgSO4) and concentrated under reduced pressure. The crude product was then purified by column chromatography to afford the α-arylated product. The authors declare that the data supporting the findings of this study are available within the paper and its Supplementary Information files. Theriot, J. C. et al. Organocatalyzed atom transfer radical polymerization driven by visible light. Google Scholar Fazekas, T. J. et al. Diversification of aliphatic C–H bonds in small molecules and polyolefins through radical chain transfer. Google Scholar Constantin, T. et al. Halogen-atom and group transfer reactivity enabled by hydrogen tunneling. Google Scholar Johansson, K. O., Head-Gordon, M. P., Schrader, P. E., Wilson, K. R. & Michelsen, H. A. Resonance-stabilized hydrocarbon-radical chain reactions may explain soot inception and growth. Google Scholar Freitas, F. P. et al. 7-Dehydrocholesterol is an endogenous suppressor of ferroptosis. Google Scholar Alabugin, I. V., Eckhardt, P., Christopher, K. M. & Opatz, T. The photoredox paradox: electron and hole upconversion as the hidden secrets of photoredox catalysis. Google Scholar & Crich, D. Free Radical Chain Reactions in Organic Synthesis (Elsevier, 1992). & Curran, D. P. Catalysis of radical reactions: a radical chemistry perspective. Google Scholar Shaw, M. H., Twilton, J. & MacMillan, D. W. C. Photoredox catalysis in organic chemistry. Google Scholar Yan, M., Kawamata, Y. & Baran, P. S. Synthetic organic electrochemical methods since 2000: on the verge of a renaissance. Google Scholar Magenau, A. J. D., Strandwitz, N. C., Gennaro, A. & Matyjaszewski, K. Electrochemically mediated atom transfer radical polymerization. Google Scholar Fu, M.-C., Shang, R., Zhao, B., Wang, B. & Fu, Y. Photocatalytic decarboxylative alkylations mediated by triphenylphosphine and sodium iodide. Google Scholar Constantin, T. et al. Aminoalkyl radicals as halogen-atom transfer agents for activation of alkyl and aryl halides. Google Scholar Lovato, K., Fier, P. S. & Maloney, K. M. The application of modern reactions in large-scale synthesis. Google Scholar Petrović, N., Malviya, B. K., Kappe, C. O. & Cantillo, D. Scaling-up electroorganic synthesis using a spinning electrode electrochemical reactor in batch and flow mode. Process Res. & Curran, D. P. The electron is a catalyst. Syroeshkin, M. A. et al. Upconversion of reductants. Zhou, S. et al. Identifying the roles of amino acids, alcohols and 1,2-diamines as mediators in coupling of haloarenes to arenes. Rohrbach, S., Shah, R. S., Tuttle, T. & Murphy, J. Neutral organic super electron donors made catalytic. Odian, G. Principles of Polymerization (Wiley, 2004). Koppenol, W. H. & Rush, J. D. Reduction potential of the carbon dioxide/carbon dioxide radical anion: a comparison with other C1 radicals. Hendy, C. M., Smith, G. C., Xu, Z., Lian, T. & Jui, N. T. Radical chain reduction via carbon dioxide radical anion (CO2·–). Chmiel, A. F., Williams, O. P., Chernowsky, C. P., Yeung, C. S. & Wickens, Z. K. Non-innocent radical ion intermediates in photoredox catalysis: parallel reduction modes enable coupling of diverse aryl chlorides. Alektiar, S. N. & Wickens, Z. K. Photoinduced hydrocarboxylation via thiol-catalyzed delivery of formate across activated alkenes. Song, L. et al. Visible-light photocatalytic di- and hydro-carboxylation of unactivated alkenes with CO2. Alektiar, S. N., Han, J., Dang, Y., Rubel, C. Z. & Wickens, Z. K. Radical hydrocarboxylation of unactivated alkenes via photocatalytic formate activation. Williams, O. P. et al. Practical and general alcohol deoxygenation protocol. Dang, Y. et al. Alkene carboxy-alkylation via CO2·–. Yu, B. et al. Switchable divergent di- or tricarboxylation of allylic alcohols with CO2. Ghosh, P. et al. Taming CO2·– via synergistic triple catalysis in anti-markovnikov hydrocarboxylation of alkenes. Villamena, F. A., Locigno, E. J., Rockenbauer, A., Hadad, C. M. & Zweier, J. L. Theoretical and experimental studies of the spin trapping of inorganic radicals by 5,5-dimethyl-1-pyrroline N -oxide (DMPO). Carbonate radical anion. Ruffoni, A., Mykura, R. C., Bietti, M. & Leonori, D. The interplay of polar effects in controlling the selectivity of radical reactions. Gryn'ova, G., Marshall, D. L., Blanksby, S. J. & Coote, M. L. Switching radical stability by pH-induced orbital conversion. Article PubMed Zhao, R., Fu, K., Fang, Y., Zhou, J. & Shi, L. Site-specific C(sp3)–H Aminations of imidates and amidines enabled by covalently tethered distonic radical anions. Mondal, T., Shaik, S., Kenttämaa, H. & Stuyver, T. Modulating the radical reactivity of phenyl radicals with the help of distonic charges: it is all about electrostatic catalysis. Gryn'ova, G. & Coote, M. L. Origin and scope of long-range stabilizing interactions and associated SOMO–HOMO conversion in distonic radical anions. Article PubMed Lloyd-Jones, G. C. Palladium-catalyzed α-arylation of esters: ideal new methodology for discovery chemistry. Johansson, C. C. C. & Colacot, T. J. Metal-catalyzed α-arylation of carbonyl and related molecules: novel trends in C–C bond formation by C–H bond functionalization. Economidou, M., Mistry, N., Wheelhouse, K. M. P. & Lindsay, D. M. Palladium extraction following metal-catalyzed reactions: recent advances and applications in the pharmaceutical industry. Process Res. Transition-metal-free coupling reactions. Bunnett, J. F. Aromatic substitution by the SRN1 mechanism. Rossi, R. A., Pierini, A. & Peñéñory, A. B. Nucleophilic substitution reactions by electron transfer. & Bunnett, J. F. Photostimulated aromatic SRN1 reactions. Beugelmans, R., Bois-Choussy, M. & Boudet, B. Studies on SRN1 reactions. Part 8: new and direct arylation and hetarylation of β-dicarbonyl compounds by SRN1. A., Walkingshaw, R. P., Hamud, S. A. S. & White, H. S. Reduction by oxidation: selective hydrodehalogenation of aryl halides by mediated oxalate oxidation. Costentin, C., Hapiot, P., Médebielle, M. & Savéant, J.-M. “Thermal” SRN1 reactions: how do they work? Novel evidence that the driving force controls the transition between stepwise and concerted mechanisms in dissociative electron transfers. Costentin, C., Hapiot, P., Médebielle, M. & Savéant, J.-M. Investigation of dissociative electron transfer mechanisms and reactivity patterns through kinetic amplification by a chain process. Douglas, J. J. et al. The implementation and impact of chemical high-throughput experimentation at AstraZeneca. Meyers, C. et al. Study of a new rate increasing “base effect” in the palladium-catalyzed amination of aryl iodides. & Alabugin, I. V. AIBN as an electrophilic reagent for cyano group transfer. Hossain, M. M., Shaikh, A. C., Moutet, J. & Gianetti, T. L. Photocatalytic α-arylation of cyclic ketones. Bunnett, J. F. & Creary, X. Arylation of arenethiolate ions by the SRN1 mechanism. Convenient synthesis of diaryl sulfides. thanks the EPSRC Centre for Doctoral Training in Integrated Catalysis (EPSRC grant EP/S023755/1) for a PhD Studentship. thank The University of Manchester for PhD studentships. acknowledges the financial support from The University of Manchester and the EPSRC. We thank the EPSRC for funding of the EPR National Facility at Manchester (EP/W014521/1). We thank T. Churchill and P. Gillespie for performing safety studies and A. Ray for high-resolution MS analysis of the product resulting from 59. Department of Chemistry, The University of Manchester, Manchester, UK Ethan R. X. Lim, Bradley D. Cooper, Muralidharan Shanmugam, Jonathan Da Luz, Eric J. L. McInnes, Cristina Trujillo, James J. Douglas & Michael J. James Early Chemical Development, Pharmaceutical Sciences, AstraZeneca R&D, Macclesfield, UK James J. Douglas Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar supervised the project. performed all the synthetic work. performed all the EPR experiments. performed all the computational work. analysed the results and wrote the paper. Correspondence to Michael J. James. The authors declare no competing interests. Nature Synthesis thanks Igor Alabugin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Peter Seavill, in collaboration with the Nature Synthesis team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Experimental details, Supplementary Figs. 1–12 and Tables 1–7. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Lim, E.R.X., Cooper, B.D., Shanmugam, M. et al. Reductive radical chain initiation through the thermal generation of carbon dioxide radical anion. Accepted: 30 September 2025 Published: 06 November 2025 Version of record: 06 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02895-3'>Site-specific DNA insertion into the human genome with engineered recombinases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 10:05:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Insertions of large DNA sequences into the genome are broadly enabling for research and therapeutic applications. Large serine recombinases (LSRs) can mediate direct, site-specific genomic integration of multi-kilobase DNA sequences without a pre-installed landing pad, albeit with low insertion rates and high off-target activity. Here we present an engineering roadmap for jointly optimizing their DNA recombination efficiency and specificity. We combine directed evolution, structural analysis and computational models to rapidly identify additive mutational combinations. We further enhance performance through donor DNA optimization and dCas9 fusions, enabling simultaneous target and donor recruitment. Our top engineered LSR variants, superDn29−dCas9, goldDn29−dCas9 and hifiDn29−dCas9, achieve up to 53% integration efficiency and 97% genome-wide specificity at an endogenous human locus and effectively integrate large DNA cargoes up to 12 kb for stable expression in non-dividing cells, stem cells and primary human T cells. Rational engineering of DNA recombinases enables precise and efficient single-step genome insertion for diverse applications across gene and cell therapies. The ability to insert multi-kilobase DNA sequences efficiently and precisely into specified sites in the human genome in a single-step mechanism would advance both synthetic biology and gene therapy, enabling integration of gene circuits, large-scale pooled libraries and entire gene replacement rather than individual correction of diverse patient mutations1. However, previous approaches have been hampered by semi-random integration2, limited efficiency3,4,5,6, ceilings on donor template size7 or complex multi-component delivery8,9,10,11. DNA recombinases are an emerging class of genome editing systems with important mechanistic advantages for achieving precise DNA insertions into the genome. The LSR enzyme family offers high recombination efficiency and site-specificity, operates independently from host DNA repair machinery and requires only two components: the recombinase and the donor DNA12. Natively, these enzymes facilitate mobile genetic element integration into bacterial genomes by recombining two double-stranded DNA attachment sites (attP and attB). During recombination, serine recombinases simultaneously cleave all four DNA strands, creating a covalent protein−DNA intermediate, followed by controlled strand exchange via subunit rotation and rejoining of the DNA ends13. They are readily adaptable to human cell engineering, integrating at either pre-installed landing pads or endogenous pseudosites (termed attH for the human genome) that closely resemble their native integration sequence12,14,15. Pre-installing landing pads can be particularly challenging in primary cells or in vivo settings. Even when suitable pseudosites exist, ensuring high recombinase specificity remains difficult. Our previous screening of more than 60 diverse LSR orthologs in human cells revealed that many integrated into genomic pseudosites but often with low specificity, targeting hundreds of sites with varying efficiencies12. Attempts to engineer LSRs and other serine recombinase systems to target endogenous sequences through mutagenesis or DNA-binding-domain fusions have historically yielded inefficient or non-specific systems3,16,17,18,19,20. Engineering tyrosine recombinases to target endogenous sequences showed greater success, yet their inherent bidirectionality limits genome integration applications21,22,23. In the present study, we developed LSRs with high efficiency and high specificity for direct human genome integration without pre-engineered landing pads. We reasoned that three key mechanistic limitations impeding efficient and specific endogenous integrations are genome recognition and binding, donor DNA binding and recombination efficiency (Fig. To address these challenges, we developed a framework to guide recombinase engineering, using the genome-targeting LSR Dn29 as a proof of concept. We combined four strategies—directed evolution, machine-learning-guided mutation stacking, dCas9 fusions and donor attachment site sequence optimization—to create recombinases that specifically integrate DNA cargo at a single endogenous locus in the human genome. Furthermore, we demonstrate integration in stem cells and primary T cells and show that the integration site maintains similar transgene expression to validated safe harbors, such as AAVS1, while reducing transcriptome perturbation. a, Overview of engineering strategies to improve integrations into endogenous genomic sites. By improving genome recognition and binding, recombination efficiency and donor DNA binding, we aim to enhance recombination between a plasmid containing the recombinase attachment site attP and a genomic pseudosite, attH1, by maximizing on-target integrations and minimizing off-targets. b, Genome-wide specificity profile of WT Dn29. The yellow dots are off-target sites, ranked by their insertion efficiency. Data shown are the same as Fig. c, Schematic of Dn29 directed evolution scheme in E. coli. An evolution backbone, pEVO, expresses a library of Dn29 variants containing NNK codons across the coding sequence (CDS) and contains attP and attH1 sites. Active variants remove the NdeI site, allowing selective PCR recovery after digestion. The active recombinase library can be shuffled to generate higher-order combinations of beneficial mutations and re-cloned into pEVO for subsequent rounds of evolution. Colonies from the active recombinase library are randomly selected and validated in HEK293FT cells. ddPCR at the on-target (attH1), a single-off target (attH3) and a genomic reference measures the efficiency (attH1/reference) and specificity (attH1/attH3). e, Efficiency and specificity of 247 LSR library members in HEK293FT cells, shown as fold change (FC) to WT. Each dot represents the mean of n = 2 biological replicates. f, Efficiency and specificity of WT Dn29 (blue, n = 16 biological replicates), variant 62 (orange, n = 2 biological replicates) and variant 93 (teal, n = 2 biological replicates) and variant 127 (green, n = 10 biological replicates), generated by stacking all mutations found on variants 62 and 93. Dots and error bars represent the mean ± s.d. of samples with n ≥ 3 biological replicates, shown as FC to WT. g, Efficiency and specificity of variants harboring driver mutations, shown as FC to variant 127. Variants are generated by adding individual mutations from the enhanced variants in e on top of variant 127. Dots represent the mean of n = 2 biological replicates. h, Efficiency and specificity of all variants generated across rounds of mutation stacking, following the path from WT Dn29 (gray, n = 16 biological replicates, same data as f) to superDn29, goldDn29 and hifiDn29 (green, n = 2, 5 and 3 biological replicates, respectively). Variant 127 is shown in blue (n = 12 biological replicates, same data as f); orange dots indicate the addition of efficiency mutations; and teal dots indicate the addition of specificity mutations, with each dot representing 2−7 biological replicates. Dots and error bars represent the mean ± s.d. of samples with n ≥ 3 biological replicates. Gray lines indicate the lineage of mutation addition between variants. Data presented are the same as shown in h. j, Efficiency of integration into a single off-target (attH3) of WT (n = 10 biological replicates), superDn29 (n = 2 biological replicates), goldDn29 (n = 5 biological replicates) and hifiDn29 (n = 3 biological replicates). We previously reported that LSRs catalyze DNA integration directly into endogenous genomic pseudosites12, with the number and identity varying across LSR orthologs based on the sequence similarity between their native attachment site (attB) and the targeted genome (Fig. In the present study, we sought to establish engineering principles for optimizing any LSR for site-specific genome integration, choosing the genome-targeting LSR Dn29 as a proof of concept owing to its favorable specificity and efficiency profile12. Dn29 integrates into the endogenous genome at 5% overall efficiency and directs 12% of insertions into a top site (termed attH1, located within an intron of NEBL), with three prominent off-targets (attH2, attH3 and attH4) each comprising 5−10% of total insertions and approximately 80 other low-frequency off-target sites (Fig. Our framework involves three key steps: increasing on-target integration efficiency at attH1, reducing insertion frequency at prominent off-targets and minimizing the long tail of low-frequency off-target insertions. To measure our progress toward these goals, we developed three key metrics: ‘efficiency' as the percentage of attH1 sites that receive an insertion; ‘specificity' as the ratio of insertions into attH1 versus the prominent off-target attH3; and ‘genome-wide specificity' as the ratio of attH1 insertions relative to all on-target or off-target integration events (Fig. Because the protein−DNA recognition code of LSR enzymes for their DNA target site is unknown but unlikely to be modular like zinc finger or TALE proteins24, we reasoned that Dn29 would need modification via directed evolution to improve on-target integration into attH1. To increase Dn29 insertion efficiency at attH1 and disfavor integration into off-targets, we performed deep scanning mutagenesis of Dn29 at single-site saturation and tested the variant library in an intra-plasmid recombination reporter containing the attH1 and attP sites (Fig. Successful recombinants are positively selected by removing an intervening restriction enzyme site, whereas unproductive variants are eliminated via plasmid digestion23,25. To increase library complexity beyond single mutants, which may not be sufficient for modifying LSR target site preference, we performed DNA fragmentation and reassembly to shuffle successful mutations (Supplementary Fig. After 12 evolution rounds and two DNA shuffles, we sequenced the output library with short-read and long-read sequencing (Supplementary Fig. The resulting library contained a median of three amino acid changes per variant, with 72% of the pool carrying 2−6 mutations (Supplementary Fig. Additionally, the evolution process exhibited robust selection against non-functional variants, with mutation dropout rates increasing from 0.0035% in the input library to 44.9% in the output library (Extended Data Fig. As expected, we saw strong retention of the catalytic serine and conserved zinc-coordinating cysteines27 and depletion of stop codons (Extended Data Fig. 1c,d), and we observed that phylogenetic conservation negatively correlated with mutational tolerance (Pearsonʼs r = −0.3577, two-tailed P < 0.0001) (Extended Data Fig. We observed higher mutational sensitivity in the N-terminal domain (NTD), consistent with its important functional roles in inter-subunit interactions, subunit rotation, catalysis and ligation28. Next, we functionally evaluated variant library members in human cells from three directed evolution timepoints: after five cycles, after seven cycles plus one shuffle and after 12 cycles plus two shuffles (the final output library). We assessed the efficiency (insertion into attH1) and specificity (attH1/attH3 ratio) of each variant (Fig. 1d), observing a shift toward higher on-target efficiency between the first and second timepoints and more variants with improved specificity after the third (Extended Data Fig. Although our selection was based solely on integration efficiency, the emergence of specificity-enhanced variants suggests that some mutations improved target discrimination without compromising recombination activity. Because most variants exhibited either enhanced efficiency or specificity, but not both, we hypothesized that combining mutations across these two classes would achieve both desired properties (Fig. First, we combined the four mutations from variant 62 (2.5-fold wild-type (WT) efficiency) and variant 93 (8.6-fold WT specificity) into variant 127, which demonstrated 13.4-fold specificity and 1.8-fold efficiency (Fig. The ability to simultaneously improve LSR efficiency and specificity motivated systematic exploration of all mutations in our efficiency-enhanced and specificity-enhanced variants. We sought to identify the causal point mutations and remove passenger mutations, thereby enabling higher-order mutation stacking. Individual validation of each mutation in variant 62 identified E70G and A224P as the driving efficiency mutations (Extended Data Fig. The sole amino acid mutation in variant 93, N341K, was further investigated by substituting N341 with every amino acid (Extended Data Fig. Many substitutions increased activity, with N341Q improving efficiency 2.3-fold and positively charged residues robustly improving specificity (N341K: 6.7-fold, N341R: 5.3-fold). We next assessed all point mutations from variants with at least 1.5-fold WT efficiency (n = 47 mutations) and two-fold WT specificity (n = 28 mutations) (Fig. We also included lysine mutations at putative DNA-interfacing residues, chosen based on alignment with the crystal structure of Listeria integrase C-terminal domain bound to attP (Protein Data Bank (PDB): 4KIS)29, hypothesizing that positively charged mutations could modify DNA binding (Extended Data Fig. These mutations were individually installed into variant 127, identifying 12 additional efficiency and seven additional specificity driver mutations, each contributing 1.2-fold to 2.5-fold efficiency and 1.1-fold to 6.8-fold specificity improvements over variant 127 (Fig. A final round of mutation validation (Extended Data Fig. 2f) yielded a final list of 21 efficiency and 12 specificity driver mutations for further rational engineering (Supplementary Table 1). We optimized Dn29 variants through sequential mutation layering, introducing single mutations into the best variant(s) from the previous round. We began with two lineages: an efficiency lineage containing 341Q and a specificity lineage containing 341K (Extended Data Fig. Double mutations showed mostly additive and subadditive effects, with rare antagonistic or synergistic epistasis (Extended Data Fig. This process yielded three key variants: superDn29 (10-fold efficiency and 70-fold specificity), goldDn29 (four-fold efficiency and 44-fold specificity) and hifiDn29 that combined goldDn29 with four additional specificity driver mutations to achieve WT efficiency with high specificity that approached the droplet digital polymerase chain reaction (ddPCR) limit of detection (Fig. These variants demonstrated a substrate preference shift toward attH1 while maintaining or reducing activity at the native attB attachment site, indicating partial substrate reprogramming rather than expansion (Extended Data Fig. Finally, we conducted whole-genome insertion profiling using an unbiased next-generation sequencing (NGS)-based integration site assay that quantifies the relative frequency of on-target versus off-target integrations (Methods). This analysis confirmed the substantial improvement in genome-wide specificity, which increased from 12% on-target integration with WT Dn29 to 40−60% with the engineered variants (Fig. Overall, we identified specific mutations driving efficiency or specificity improvements, enabling rational engineering of optimized recombinases. We demonstrated that LSR-directed evolution with efficiency and specificity classification enables iterative mutation combinations to generate LSRs with desired functional profiles. To expedite experimental testing higher mutational loads, we developed a computational model for predicting combinatorial variant activity from single mutant data. The variants were divided into distinct rounds, with the individual mutation validation in round 1 (Fig. 1g) and the iterative mutation stacking experiments comprising rounds 2−5. We trained two linear models (linear regression and ridge regression) and two gradient boosting models (XGBoost and CatBoost)30,31 on one-hot encoded variant sequences from round 1 and then tested these models on rounds 2−5 (Fig. a, Division of data into training and test sets. Models are trained on the individual driver mutation validation data (round 1) and tested on the iterative rounds of higher-order mutants (rounds 2−5). b, Evaluation metrics (Spearman's ρ, NDCG and RMSE) of linear regression, ridge regression, XGBoost and CatBoost models. c, Predicted versus true efficiency of higher-order mutants, as fold change (FC) to WT. e, Coefficient values of mutations in the efficiency and specificity models. Colored dots indicate mutations identified as efficiency (orange) or specificity (teal) driver mutations from Fig. g, Efficiency (top) and specificity (bottom) of top model-guided combinatorial mutants, generated by predicting the activity of combining two mutations on top of superDn29. Each dot represents the mean of n = 6 biological replicates for WT and superDn29 and n = 2 biological replicates for all model-guided variants. h,i, Efficiency (h) and specificity (i) mutations mapped to the AlphaFold3 structure of Dn29 bound to attB-R. CC, coiled-coil motif; RD, recombinase domain; ZD, zinc-ribbon domain. We evaluated model performance using root mean squared error (RMSE), Spearman's rank correlation and normalized discounted cumulative gain (NDCG). NDCG was prioritized as the key evaluation metric because it emphasizes higher-activity mutants, matching our experimental validation priorities. Although all models performed well, ridge regression excelled for both efficiency and specificity, showing high accuracy (NDCG = 0.970 for efficiency, 0.971 for specificity) in predicting higher-order mutant activities (Fig. The performance of linear models in extrapolating single mutant activity to higher-order mutants supports our previous observations that LSR mutations are largely additive (Extended Data Fig. 2h) and aligns with previous research showing that regularized linear models perform well across mutagenesis datasets of 14 different enzymes32. To identify key mutations, we examined regression coefficients, revealing strong concordance between computationally and experimentally identified impactful mutations (Fig. The model coefficients strongly correlated with experimental fold changes (efficiency: ρ = 0.623, two-tailed P = 0.0025; specificity: ρ = 0.930, two-tailed P < 0.0001; Extended Data Fig. 3c,d), demonstrating that the interpretability of linear models enables the automated identification of impactful mutations. We outline an approach for applying regression models to predict efficiency and specificity of higher-order mutants (Fig. From individual mutations measured for specificity and efficiency by ddPCR, we generated ridge regression models, examined their coefficients to quantify the impact of key mutations and predicted activity of higher-order mutants, enabling prioritization of mutants for experimental testing. To demonstrate this approach, we predicted the activities of all double mutants added to superDn29, skipping round 6 to directly test round 7 of iteration. We tested the top 10 efficiency and top three specificity variants, finding that eight of 10 efficiency variants and three of three specificity variants performed better than superDn29 (Fig. Although the three key variants (superDn29, goldDn29 and hifiDn29) that we experimentally identified served as the primary variants for subsequent characterization studies, these model-guided variants demonstrate the potential for further optimization and represent valuable targets for future investigation. Overall, we demonstrate that model-guided recombinase design across multiple activity axes can further push efficiency and specificity beyond spaces easily reachable by traditional directed evolution. To gain mechanistic insights from our mutational landscape, we generated an AlphaFold3 structural model of Dn29 bound to the attB-R DNA half-site (Fig. 4a)33, which showed high alignment (zinc-ribbon domain RMSD = 1.341, recombinase domain RMSD = 1.707) to the Listeria integrase C-terminal domain bound to attP (PDB: 4KIS)29 (Extended Data Fig. We identified an efficiency mutation hotspot (residues 373−393 and 449) in the coiled-coil motif hinge region, with four of the eight activating mutations converting to proline residues (L388P, Q390P, L393P and L449P), suggesting that destabilization of the helical secondary structure in this region enhances activity, potentially by modifying tetramer stabilization or autoinhibitory control (Extended Data Fig. Despite previous work on a different ortholog that identified mutations in this region as enabling the excision reaction35, our key variants maintained unidirectionality (Extended Data Fig. Another key efficiency mutation (D503N) reduces negative charge in a tri-aspartic acid stretch (503−505), likely enhancing DNA phosphate backbone interactions (Extended Data Fig. Many specificity driver mutations localize near the DNA-binding interface, often replacing neutral amino acids with positively charged ones, potentially strengthening DNA interactions (Extended Data Fig. This combined computational and structural analysis provides a multifaceted understanding of how specific mutations impact LSR function, informing rational engineering to enhance LSR performance. We reasoned that the LSR protein and target DNA interaction could be further enhanced by developing LSR−dCas9 fusions that facilitate LSR recruitment to the genomic target site, as demonstrated in other recombinase systems3,4,6,19,22. We first optimized the fusion design, observing that an N-terminal dCas9 fusion abolished Dn29 activity, likely due to steric hindrance of tetramerization as the N terminus is located at the tetrameric complex core29,36. By contrast, C-terminal fusions supported robust recombination and were used for further experiments (Fig. a, Schematic of LSR−dCas9 fusion orientations and recombination efficiency with various linkers at attH1 with an NTG. b, Schematic of sgRNA targets for the attH1 (chr10:21,130,405) and attH3 (chr1:230,490,334) pseudosites. d,e, Integration efficiencies of Dn29 and Dn29−dCas9 at attH1 (d) and attH3 (e) with sgRNAs targeting proximal to the respective pseudosites. Data are shown as fold change (FC) relative to NTG. of n = 3 biological replicates, shown as dots. f, Biasing Dn29−dCas9 integrations to different pseudosites. Shown is the percent of all genome-wide insertions at attH1 (orange) and attH3 (green), using sgRNAs targeting the pseudosites. of n = 3 biological replicates, shown as dots. Data shown are the same as presented in Fig. Data shown combine replicates transfected with WT attP and e-attP. Data shown for Dn29 are the same as presented in Fig. k, Heatmap showing attH3 integration efficiencies (%) of Dn29−dCas9 using guides targeting upstream and downstream of the pseudosite, individually and multiplexed. Each cell represents the mean of n = 3 biological replicates. m, Integration efficiencies of Dn29−dCas9 using donor plasmids with the H1-g3 sgRNA target sequence adjacent to the attP. of n = 3 biological replicates, shown as dots. Asterisks show t-test significance compared to WT donor plasmid. We next evaluated the impact of dCas9-mediated genomic recruitment using single guide RNAs (sgRNAs) targeting regions near attH1 or attH3 (Fig. This single-guide approach increased integration efficiency 2.7-fold at attH1 and six-fold at attH3 compared to non-targeting guides (NTGs) (Fig. We further improved efficiency by optimizing the ratio of donor, effector and guide components (Extended Data Fig. 5a) and confirmed the importance of direct tethering of Dn29 to the genomic target site, as linker replacement with a P2A peptide to induce ribosomal skipping abolished the effect37,38 (Extended Data Fig. Interestingly, we could manipulate the natural integration preference of Dn29 using dCas9 recruitment. Although WT Dn29 naturally integrates into attH1 with two-fold frequency over attH3, dCas9-based recruitment to attH1 amplified this preference to 14-fold. Conversely, attH3-targeting sgRNAs reversed this bias, resulting in 11-fold higher integration at attH3 compared to attH1 (Fig. Combining dCas9 fusions with our optimized Dn29 variants further improved on-target efficiency. SuperDn29−dCas9 achieved 50.8% integration at attH1, whereas goldDn29−dCas9 and hifiDn29−dCas9 reached 44.5% and 39.1%, respectively, representing up to an 11.8-fold efficiency increase (Fig. Next, we tested if dCas9 fusions enhanced specificity by biasing insertion toward the desired pseudosite. Whole-genome insertion profiling showed that on-target integrations improved from 12% with WT Dn29 to more than 60% with Dn29−dCas9 targeted to attH1, although rare off-target sites persisted. SuperDn29−dCas9 maintained a similar insertion profile but moderately increased rare off-targets due to higher overall activity. Off-target site concordance across variants was low, with only 15 sites appearing in more than one sample (Supplementary Fig. Furthermore, these off-target sites did not overlap with predicted guide RNA off-target sites, indicating that they are not dCas9 mediated (Supplementary Fig. Taken together, these results suggest that specificity engineering has effectively eliminated reproducible off-target activity, with the low reproducibility potentially reflecting both rare genuine events and technical artifacts previously reported for this off-target detection method39. We assessed the generalizability of our fusion approach across three additional LSR orthologs (the genome-targeting LSRs Pf80 and Nm60 and the landing pad LSR Si74), demonstrating up to 10-fold improved efficiency (Extended Data Fig. To elucidate optimal sgRNA design parameters, we analyzed integration efficiencies across all fusion variants and orthologs and determined optimal sgRNA placement to be approximately 40 bp from the attachment site core, agnostic of guide orientation (Extended Data Fig. We next explored three orthogonal strategies to further improve efficiency—using dCas9 variants with alternative PAM recognition40, multiplexing sgRNAs and incorporating sgRNA binding sites on donor plasmids. dCas9 variants with increased PAM flexibility can expand pseudosite-proximal guide options. Using dCas9−HF1−SpG (NGN PAM), we achieved 22% insertion efficiency at the best NGH guide compared to 15% with the best NGG guide (Extended Data Fig. Multiplexed sgRNAs targeting upstream and downstream of the attachment site could further improve genome search and binding (Fig. Finally, we tested the ability of an sgRNA-binding sequence on the donor plasmid to facilitate donor recruitment, across four orientations flanking the minimal attP site (top and bottom strand, upstream and downstream) (Fig. All four configurations improved integration efficiency for Dn29−dCas9, reaching up to 40% integration compared to 23% with the WT donor (Fig. Similarly, incorporating the Nm60-g1 sequence on the donor plasmid improved Nm60 attH2 integration from 61% to 73% (Extended Data Fig. Our results show that LSR−dCas9 fusions can substantially enhance both efficiency and specificity of cargo DNA insertion by improving LSR recruitment to target and donor DNA. The positive correlation between efficiency and specificity (Extended Data Fig. 5o) suggests that optimizing and multiplexing guide RNAs are crucial for maximizing both parameters simultaneously. This approach, combined with engineered LSR variants and dual-targeting guides, achieves up to 97% specificity or over 73% efficiency at a single genomic locus. Beyond dCas9 fusions, we explored enhancing donor DNA recruitment by optimizing the attP sequence. We created two Dn29 attP donor plasmid libraries, each with one constant half-site and the other mutated using custom mixed base oligos for an average of approximately 5.5 mutations per 26-bp half-site (Methods). Transfecting these libraries into cells expressing WT Dn29−dCas9 and H1-g3 guide, we sequenced attH1 integrants and calculated nucleotide enrichment scores (Fig. Although the WT nucleotide was generally preferred, we identified six positions where non-WT nucleotides showed moderately higher enrichment. Combining all six substitutions into an optimized e-attP sequence improved integration efficiency by 1.3-fold (Fig. Applying this strategy to Nm60 with Nm60−dCas9 and Nm60-g2, we identified an e-attP with seven mutations that improved efficiency by 1.1-fold (Fig. a, Schematic of attP optimization screen. b, Design of attP-L and attP-R libraries. Libraries are generated using mixed base oligos containing 79% WT and 7% each other nucleotide for one half-site and constant WT sequence for the other half-site. c, Dn29 attP nucleotide enrichment/depletion heatmap (top) and sequence logo of enriched nucleotides in e-attP (bottom). Data represent average enrichment scores of two biological replicates. d, Dn29 integration efficiency with attP and e-attP donor plasmids. Asterisks show t-test significance compared to WT donor. *one-tailed P = 0.0165. e, Nm60 attP nucleotide enrichment/depletion heatmap (top) and sequence logo of enriched nucleotides in e-attP (bottom). Data represent average enrichment scores of two biological replicates. f, Nm60 integration efficiency with attP and e-attP donor plasmids. of n = 3 biological replicates, shown as dots. Asterisks show t-test significance compared to WT donor. Our library enrichment approach provides a high-resolution view of DNA specificity and recombination efficiency determinants. For both Dn29 and Nm60, native attachment sites appear highly evolutionarily optimized, with only a few positions showing incremental efficiency improvements through mutation. We identified core-distal regions of outsized importance for functional recombination: positions 16−23 and 16'−23' for Dn29 (Fig. These findings are consistent with previous reports on other LSR orthologs that show stronger zinc-ribbon domain binding requirements and more relaxed sequence specificity in the recombinase domain41,42. Dn29's position 17/17' showed particularly high enrichment, strongly preferring A and T, respectively. Furthermore, analysis of depleted nucleotides reveals selection against attB-like sequences, particularly A/T at position 12/12' for Dn29 and A/T at position 14/14' for Nm60, which correspond to critical attB nucleotides (Supplementary Fig. This depletion suggests evolutionary pressure to maintain discrimination between attB and attP sites via previously described ‘discriminator bases'41. By analyzing dinucleotide normalized to single-nucleotide abundance, we identified preferential RY dinucleotides at positions 17/18 and YR dinucleotides at positions 22/23, suggesting that DNA flexibility as well as sequence contributes to protein recognition of the attachment site43,44,45,46 (Supplementary Fig. Given the numerous DNA-binding domain mutations in our variants, we applied this approach to identify optimal attP substrates for each recombinase. The engineered variants preferred attP sequences with 7−9 mutations relative to WT Dn29 e-attP, predominantly favoring guanine/cytosine bases (Supplementary Fig. This G/C bias likely reflects enhanced lysine−guanine interactions resulting from the numerous specificity-enhancing lysine residues incorporated into the DNA-binding regions during protein engineering47. Although variant-specific e-attPs demonstrated modest improvement over the WT e-attP sequence (Supplementary Fig. 7d), we used the WT e-attP for all subsequent studies to maintain a standardized donor template across all variants. Although LSR attachment sites are canonically imperfect inverted repeats, the importance of this asymmetry remains unclear42. Our data reveal that the most strongly enriched nucleotides are symmetrical across the core, aligning with previous Bxb1 studies48,49. However, for Dn29 and Nm60, we observed five and eight nucleotides with subtle preferences for asymmetric nucleotides at corresponding half-site positions. This suggests that slight attachment site asymmetry may be a deliberate feature of the recombination mechanism rather than a consequence of mutations or phage genome sequence constraints. Overall, this attachment site exploration deepens our understanding of LSR target site recognition and advances our ability to design optimal DNA donors. Next, we aimed to create optimal LSR tools for large DNA cargo integration by combining our orthogonal engineering efforts. Armed with directed evolution variants, dCas9 fusions with sgRNA design rules and optimized donor DNA substrates, we assessed combining these features into a single system (Fig. Overall, our combined engineering strategies substantially improved recombination efficiency: the variants fused to dCas9 with the optimized donor achieved 41−53% efficiency, a 9.6-fold to 12.3-fold improvement over the WT enzyme (Fig. a, Schematic overview of developed engineering strategies. b, Integration efficiencies at attH1 of Dn29 with combined engineering strategies. of n = 3 biological replicates, shown as dots. Dots and error bars represent the mean ± s.d. of samples with n ≥ 3 biological replicates. d, Specificity analysis of edited HEK293FT single-cell clones engineered with Dn29 and hifiDn29, with and without dCas9 fusions. Number (n) of clones analyzed per sample is indicated on the x axis. Using hifiDn29−dCas9, our most specific configuration, we measured 97% genome-wide specificity by bulk integration site sequencing (Fig. To better understand the single-cell variation of insertional mutagenesis, including on-target/off-target co-occurrence and integration copy number, we mapped integrations in approximately 50 clonal HEK293FT populations edited with hifiDn29 or WT Dn29, with and without dCas9 fusions (Fig. dCas9 fusion dramatically improved performance for both Dn29 and hifiDn29, resulting in over 95% of clones containing on-target edits. However, the most striking difference was observed in off-target insertions: 91% of Dn29 clones contained off-target insertions compared to 46% of hifiDn29 clones. These single-cell specificity measurements closely mirrored the bulk genome-wide specificity results (Extended Data Fig. Beyond targeting accuracy, measuring the number of integrations per cell is crucial for fully understanding editing outcomes. HifiDn29 showed the highest rate of single on-target insertion events, with half of the clones exhibiting this genotype. By contrast, dCas9 fusions decreased single on-target insertion events to 38% and increased the rate of multiple on-target insertions from 4% to 53% of clones (Extended Data Fig. Due to the pseudo-triploid genome and copy number variation/instability of HEK293FT cells50, on-target insertions ranged from zero to five per cell, with a median of two for both hifiDn29−dCas9 and Dn29−dCas9 clones (Extended Data Fig. Overall, these single-cell results demonstrate the enhanced precision and efficiency of hifiDn29−dCas9, nominate hifiDn29 for generating clonal cell lines containing single on-target integrations and highlight the value of single-cell analysis in evaluating gene editing outcome heterogeneity. Genome editing can pose safety risks through unintended outcomes, including insertion and deletion (indel) formation, cytotoxicity and genomic rearrangements. Because the recombinase mechanism involves coordinated cleavage of all four DNA strands, abortive recombination could potentially lead to double-stranded break (DSB) formation, causing indels at the attachment sites51,52,53. We identified rare but significant indels at attH1 (0.01−1.4%), with indel frequency correlating with recombination efficiency (Extended Data Fig. This trend parallels observations in small serine recombinase systems, where activating mutations that increase synapse formation also demonstrate higher rates of DSB generation54. As a proxy for genome-wide DSB formation or DNA damage response (DDR) activation, we employed phosphorylated-H2AX (γ-H2AX) staining and flow cytometry. Dn29 and superDn29 showed low but significant γ-H2AX, whereas specificity-enhanced variants (goldDn29 and hifiDn29) reduced damage to background levels (Extended Data Fig. Notably, these variants also generated less γ-H2AX than Bxb1, previously reported as the highest-fidelity recombinase51 and commonly used in attachment site prime editing approaches such as PASTE and PASSIGE8,10,39. Interestingly, delivering catalytically dead LSR also produced γ-H2AX, suggesting that non-catalytic mechanisms such as DNA binding may damage DNA (Extended Data Fig. Cytotoxicity can arise from multiple mechanisms, including off-target effects, DDR activation, immune activation, protein burden and genomic instability. Cell viability assays revealed no significant toxicity in HEK293FT cells when transfecting LSR variants and donor plasmid (Extended Data Fig. Measuring genome-wide junctions with attH1 by NGS, we detected genomic rearrangements ranging between 0% and 2% within chromosome 10 and 0−0.3% inter-chromosomal translocations (Extended Data Fig. Most junctions occurred within 50 kb of attH1, potentially reflecting a preference for recombining with nearby sequences or resulting from DSB-induced large deletions (Extended Data Fig. Overall, our comprehensive safety assessment demonstrates that Dn29-mediated editing, particularly with our specificity-enhanced variants, offers a favorable safety profile with minimal cytotoxicity, rare genomic rearrangements and reduced DNA damage compared to existing recombinases, positioning this system as a promising option for precise genomic integration. Next, we sought to benchmark our engineered LSRs in a diverse set of genome insertion tasks across various cell types. LSRs offer an advantage in genome engineering applications, including robust insertion efficiency in non-dividing cells10. We treated HEK293FT cells with aphidicolin to induce cell cycle arrest and observed that Dn29 and key variants showed largely equivalent integration rates to untreated cells. dCas9 fusions had decreased integration efficiency by 35−50%, but still achieved up to 30% integration (Extended Data Fig. We then tested DNA cargo installation in H1 human embryonic stem cells (hESCs) and observed that on-target integration efficiency with engineered recombinases increased up to six-fold relative to WT Dn29, from 4.1% to 24.5% (Fig. These improvements are 25−50% of HEK293FT integration efficiencies, likely due to the 50−80% reduction in plasmid transfection efficiency in stem cells (Extended Data Fig. Insertion specificity also significantly improved, as off-target integration at attH3 for the engineered variants approached the ddPCR detection limit (Extended Data Fig. a, Integration efficiencies of Dn29 variants and dCas9 fusions in hESCs. of n = 3 biological replicates, shown as dots. b, Schematic of hESC engineering with LSRs. A 12-kb CRISPRi donor plasmid and an LSR−dCas9 effector/guide plasmid are transfected into H1 hESCs or WTC-11 iPSCs, selected and differentiated into HPCs. At the stem cell stage or the HPC stage, cells are transduced with lentivirus expressing sgRNAs for CD81, CD63 or CD147 for measuring CRISPRi knockdown. c, Integration efficiency of 12-kb CRISPRi donor by LSR−dCas9s at attH1. d, Genotyping hESC single-cell clones engineered with goldDn29−dCas9. Number (n) of clones analyzed per sample is indicated in the legend. Plots show the knockdown quantification of biological replicates (mean ± s.d. DEG significance thresholded at Benjamini–Hochberg FDR-adjusted P < 0.05 and log2(fold change) > 1. g, Top, schematic of ssAAV or scAAV donor and effector mRNA delivery into primary human T cells. Bottom, integration efficiencies of Dn29 variants and dCas9 fusions at attH1 in primary human T cells using scAAV donor. of n = 4 biological replicates, each originating from a different blood donor. h, Top, schematic of non-viral plasmid delivery of a CD19 CAR, an sgRNA expression plasmid and effector mRNA into primary human T cells. Bottom, integration efficiencies of Dn29 variants and dCas9 fusions at attH1 in primary human T cells using non-viral plasmid and mRNA delivery. of n = 3 biological replicates, each originating from a different blood donor. i, Top, schematic of in vitro cancer target cell-killing assay. Engineered T cells are cultured for 13 days to allow episomal plasmid dilution and then co-cultured with varying concentrations of CD19+ Nalm6 leukemia target cells. After 48 hours of co-culture, remaining Nalm6 cells are quantified. Bottom, percentage of Nalm6 cells killed after 48 hours at various E:T ratios. of samples with n ≥ 3 biological replicates (each an average of two technical replicates), with each originating from a different blood donor. To evaluate larger cargo installation and enable functional genomics applications, we designed a 12-kb CRISPR interference (CRISPRi) construct encoding the ZIM3−dCas9 fusion and multiple regulatory elements and marker genes, including blue fluorescent protein (BFP), neomycin resistance marker, Woodchuck post-transcriptional regulatory element (WPRE) and ubiquitous chromatin opening element (UCOE). We achieved robust insertion efficiencies up to 13% and verified transgene expression after selection in both bulk and clonal lines (Fig. Genotyping of goldDn29−dCas9-edited clones revealed that 95% of clones possessed precise heterozygous attH1 insertions, with only one homozygous clone and one clone showing both on-target and off-target integrations (Fig. These results underscore the high clonal consistency achieved, demonstrating a predominance of accurate, single-copy integrations and strong maintenance of cargo gene expression in H1 stem cells. We tested cells edited at attH1 using either hifiDn29−dCas9 or Cas9 with homology-directed repair (HDR) and at established safe harbors AAVS1 and CLYBL using Cas9 HDR55,56,57,58. CRISPRi-mediated knockdown of target genes demonstrated similar and robust efficiency across all loci (Fig. Bulk RNA sequencing (RNA-seq) analysis revealed that hifiDn29−dCas9 produced the fewest differentially expressed genes (DEGs) compared to Cas9 editing at all loci in both cell types (Fig. The DEGs in hifiDn29−dCas9 samples demonstrated the strongest safety profile, with no disruption of oncogenes, tumor suppressor genes or essential genes (Extended Data Fig. Critically, NEBL expression showed no significant change across all edited cell lines, confirming that attH1 integration does not disrupt the endogenous locus (Supplementary Tables 6 and 7). Transgene silencing is a persistent challenge in stem cell engineering, likely due to extensive chromatin restructuring that occurs during differentiation59. To assess whether attH1 supports stable cargo expression during differentiation, we differentiated the CRISPRi-edited hESCs into hematopoietic progenitor cells (HPCs) under neomycin selection. After differentiation, edited HPCs maintained robust cargo expression (approximately 70% BFP+) with approximately 80% of cells expressing canonical HPC markers, such as CD34 and CD43 (Extended Data Fig. Next, we transduced sgRNAs targeting CD63, CD81 and CD147 into the edited HPCs and observed 82−96% knockdown of these cell surface markers compared to a non-targeting control, with knockdown rates similar to AAVS1 and CLYBL sites (Fig. Taken together, we demonstrate that engineered recombinases efficiently produce bulk hESC lines with near-clonal genotypes, minimal transcriptomic perturbation and stable large cargo expression throughout hESC-to-HPC differentiation, making them suitable for generating stable cell lines for CRISPR screens or potentially introducing therapeutic genetic cargoes. Site-specific transgene insertions show great promise in immune cell engineering, enabling integration of functional cargos such as chimeric antigen receptors (CARs) and additional immune regulators for therapeutic applications. Although primary T cells can be sensitive to high doses of nucleic acid delivery (Extended Data Fig. 9a), we developed multiple effective strategies for LSR-mediated integration. We first explored viral delivery strategies, electroporating T cells with effector mRNA and synthesized sgRNAs and delivering donor templates via single-stranded AAV (ssAAV) or self-complementary AAV (scAAV) (Fig. LSR−dCas9 fusions achieved up to 17% AAV integration efficiency, which maintained high cell viability even at the highest doses. The scAAV donor yielded 2.3-fold to 3.3-fold higher integration rates compared to the ssAAV donor, likely due to double-stranded DNA requirements for LSR-mediated integration. Sequencing confirmed that more than 98% of integrations were genuine LSR-mediated events rather than non-specific AAV capture (Extended Data Fig. Because LSR-mediated integration of linear AAV genomes creates DSBs requiring cellular repair mechanisms to rejoin the inverted terminal repeat (ITR) ends, future applications could benefit from circular AAV donor templates to eliminate this repair requirement60. To expand cargo capacity beyond AAV genomes and circumvent DSB repair requirements, we optimized the electroporation protocol for efficient plasmid integration (up to 14.8%) while balancing reductions in cell viability across variants (Extended Data Fig. These findings suggest that in more sensitive cell types, the most active recombinases (superDn29) may exhibit increased toxicity, whereas the dCas9-fused recombinases showed minimal toxicity. We integrated a 5.8-kb plasmid expressing a CD19-targeted CAR into primary T cells at up to 11% efficiency, which demonstrated robust cytotoxicity against Nalm6 cells in an in vitro cancer target cell-killing assay (Fig. This delivery format versatility provides flexibility for different applications and manufacturing contexts, strongly supporting further development of this approach for engineering primary T cells for therapeutic applications. Finally, we investigated the cross-reactivity of our engineered LSRs across model organisms, including mice and various non-human primates. An attH1-like sequence is present in the NEBL intron in marmosets, rhesus monkeys and cynomolgus monkeys, with 1−2 point mutations compared to attH1, and is located intergenically in the mouse X chromosome with six point mutations. Dn29 and goldDn29 could robustly recombine attP with the model organism pseudosites in a plasmid recombination assay in HEK293FT cells (Extended Data Fig. 9j−l), enabling future advancement in preclinical animal studies that bridge the gap between laboratory research and human clinical trials. In this study, we combine directed evolution, protein engineering and machine learning models to engineer DNA recombinases to efficiently and specifically insert large genetic cargos directly into the human genome, overcoming the need to pre-install attachment site landing pads. As a proof of concept, we integrated into a single genomic locus using optimized Dn29 LSRs, achieving 13-fold to 17-fold improvements in insertion efficiency. Combining mutants with dCas9 fusions and optimized donor sequences (e-attP and sgRNA target sites) yielded recombinases with 40−53% efficiency and 90−97% genome-wide specificity for an endogenous locus. Our engineering efforts provide numerous mechanistic insights into LSR function during genome integration. Notably, our dCas9 fusion experiments demonstrate that improved genome search and DNA binding are crucial areas for increased integration efficiency. To further interrogate DNA binding, we employed structural modeling and attachment site screening to identify specific protein and DNA regions critical for target recognition. Directed evolution revealed a general tradeoff between efficiency-improving and specificity-improving mutations for Dn29, which we overcame by strategically pairing mutations across distinct LSR domains and increasing the protein−DNA interface through fusions with DNA-binding proteins. Our current system incorporates CRISPR components, which include both protein (dCas9) and RNA (sgRNA) elements. Although this design improves efficiency by seven-fold and specificity by five-fold, it also increases the overall size of the system and introduces an additional RNA component, which increases the manufacturing and formulation complexity. In future iterations, these CRISPR components could be replaced with smaller, protein-only DNA-binding domains, such as zinc fingers22. Such modifications would preserve the delivery advantages of these compact recombinases and maintain a streamlined system of a single protein and single DNA donor. Further studies of LSR safety should be conducted to advance the translational potential of these tools. Many off-target sites exhibit minimal sequence similarity to the target site, complicating off-target locus prediction. Our most highly active variant (superDn29) also exhibited higher rates of indels or chromosomal rearrangements, and lower T cell viability, than the more balanced goldDn29. LSR−dCas9 fusions also exhibited reduced efficiency in cell-cycle-arrested cells, suggesting the need for further studies to understand potential cell cycle dependence or improve the nuclear delivery of larger fusion constructs. Finally, improved methods for off-target prediction, detection and validation will be needed to assess and overcome these challenges. These studies focused on improving Bxb1 targeting of its natural attB site to enhance integration rates into landing pads or retarget Bxb1 to endogenous sites. However, these approaches require pre-installation of the landing pad or concurrent delivery of multiple Bxb1 variants, increasing delivery complexity and increasing the space of potential off-target sites. By contrast, our study presents multiple orthogonal engineering strategies to enhance the ability of an LSR to recognize and integrate at endogenous genomic sequences. We further demonstrate the generalizability of these approaches beyond Dn29 to Nm60, improving on-target genomic insertion efficiency to 73%. These advancements have use across diverse research and therapeutic applications of LSRs. Current research uses lentiviral engineering of cell lines and single-copy installation of pooled libraries, which can lead to unpredictable effects on gene expression, potential insertional mutagenesis and silencing of transgenes64. Our engineered recombinases overcome these limitations by targeting a defined integration locus at high efficiencies, which are essential for large-scale and uniform functional genomics studies. In hESCs, we demonstrate that 95% of cells have single-copy, on-target insertions, enabling the generation of homogenous bulk cell populations without single-clone selection. Furthermore, we previously demonstrated the use of LSRs for virus-free library screening in landing pad cell lines12. We extend this capability, showing the feasibility of integrating RNA or protein libraries directly into an endogenous human genomic locus. In hESCs, these integrations occur at copy numbers similar to low multiplicity of infection (MOI) lentivirus (MOI = 0.1), well within the standard guidelines for approximating one integrant per cell65. In the therapeutic space, this approach offers advantages over prevailing CRISPR-based gene therapies, which require a guide RNA to target a distinct disease-causing mutation. By contrast, multi-kilobase insertions enable replacement of entire corrective open reading frames, providing a ‘one-size-fits-all' approach for correcting genetic diseases with mutational heterogeneity across patient populations. Additionally, these corrective transgenes can include critical non-coding regulatory elements for enhanced control of gene expression. Furthermore, Dn29 can cross-reactively integrate into attH1-like sequences in diverse model organisms, an important consideration for future Investigational New Drug (IND)-enabling studies. The strategies outlined in this work can be adapted to target diverse genomic loci beyond Dn29 attH1. To target a different locus, such as validated genomic safe harbors such as AAVS1 or therapeutic targets such as TRAC, LSRs can be mined from the thousands of naturally occurring orthologs to find a recombinase with a closer match to the desired target sequence. These candidate LSRs can then be subjected to our joint optimization approach, combining directed evolution, machine learning predictions and DNA-binding protein fusions to enhance both efficiency and specificity. Our research complies with relevant ethical regulations. Experiments using hESC lines were performed under an allowance granted by the Arc Institute Stem Cell Research Oversight Committee. Experiments were conducted in HEK293FT cells (Thermo Fisher Scientific, R70007, female), H1 hESCs (WiCell Research Institute, male), WTC-11 iPSCs (Coriell Institute for Medical Research, GM25256, male) and primary human T cells (STEMCELL Technologies, 200-0092) from deidentified healthy donors. HEK293FT cells were cultured in DMEM with 10% FBS (Gibco) and 1× penicillin−streptomycin (Thermo Fisher Scientific) and dissociated using TrypLE Express (Gibco). H1 hESCs and WTC-11 iPSCs were maintained in mTeSR Plus (STEMCELL Technologies) supplemented with 1× antibiotic-antimycotic (Thermo Fisher Scientific) and cultured on Cultrex (Bio-Techne) or Matrigel (Corning) coated plates. For routine passaging, hESCs and iPSCs were dissociated with ReLeSR (STEMCELL Technologies). For 96-well plating prior to transfections, single-cell dissociation was performed using Accutase (STEMCELL Technologies). hESCs and iPSCs were supplemented with 10 µM ROCK inhibitor for 24 hours after dissociation. Primary human T cells were cultured in complete X-VIVO 15 (cXVIVO 15) (Lonza Bioscience, 04-418Q), which consists of 5% FCS (R&D Systems, M19187), 5 ng μl−1 IL-7 and 5 ng μl−1 IL-15. For the cell cycle arrest experiment, HEK293FT cells were treated with 5 μM aphidicolin at the time of transfection. An NNK deep mutational scanning library of the entire Dn29 coding sequence (CDS) was generated using NNK oligos and overlap extension PCRs. First, forward and reverse oligos with NNK mixed bases at each codon were designed with a melting temperature of 65 °C. Each NNK forward primer was paired with Dn29 DMS_universal_reverse that binds downstream of the CDS and each NNK reverse primer with DMS_universal_forward primer, generating amplicons flanking the mutated codon. Upstream and downstream amplicons (2.5 µl each) were pooled and cleaned with 2 µl of ExoSAP-IT (Thermo Fisher Scientific) and 0.5 µl of DpnI (NEB), incubating at 37 °C for 30 minutes and then at 80 °C for 15 minutes. For the overlap extension PCR, 1 µl of cleaned PCR pool was mixed with 2.5 µl of Q5 2× Master Mix, 0.025 µl of each universal primer (100 µM) and 1.45 µl of water, using the same cycling conditions. The full mutant pool was created by combining 2.5 µl of each overlap extension PCR. The reaction was split into two 20-µl reactions, ligated for 30 minutes at room temperature, inactivated at 65 °C for 10 minutes and purified (Clean and Concentrator-5 Kit; Zymo Research). The ligation product was electroporated into XL-1 Blue cells (Agilent Technologies) according to the manufacturer's instructions, recovered for 1 hour at 37 °C in 1 ml of SOC medium and plated onto four 245-mm × 245-mm BioAssay dishes. Plasmids were purified using a NucleoBond Xtra Midi EF Kit (Macherey Nagel) and sequenced with an Illumina NextSeq 2000 600-cycle P1 Kit (Supplementary Fig. 1c) using the NextSeq 1000/2000 Control Software Suite version 1.7.1. For library transformation, induction and growth: 4 µl of pEVO plasmid library was electroporated into 50 µl of XL-1 Blue competent cells (Agilent Technologies), recovered in 1 ml of SOC medium (37 °C, 1 hour) and then seeded into 100 ml of LB medium with carbenicillin and L-arabinose (10 µg ml−1 or 0 µg ml−1). Library coverage (>1 million colonies) was confirmed by plating serial dilutions. Plasmids were extracted using a Qiagen Plasmid Midi Kit (0.3 g of wet bacteria pellet per column). Selection of active variants: 500 ng of plasmid was digested with NdeI (NEB) to eliminate inactive variants. Cloning for next evolution cycle: Amplified active variants and pEVO backbone were digested with XbaI and HindIII-HF (NEB) at 37 °C for 30 minutes and then heat inactivated at 80 °C for 20 minutes. Five ligation reactions (20 µl each) were set up using 100 ng of DNA (3:1 ratio of library to backbone) and T4 ligase (NEB). Ligation occurred at room temperature for 30 minutes, followed by heat inactivation at 65 °C for 10 minutes. Pooled reactions were purified (DNA Clean and Concentrator-5 Kit), eluted in 6 µl of water and electroporated into XL-1 Blue cells to start the next evolution cycle. Shuffling the active variants between rounds of cycling involved a uridine exchange PCR to partially exchange thymidines for uridine, USER Enzyme fragmentation at uridine sites, primerless PCR fragment reassembly and PCR for full-length gene recovery. Uridine exchange PCR: Fragment size and yield was optimized by modifying dUTP/dTTP ratio, with the optimal ratio being 3/7. USER Enzyme digestion: 500-ng aliquots were digested with 2 µl of USER Enzyme (NEB) at 37 °C for 3 hours. Gel electrophoresis confirmed fragment distribution (100−1,000 bp). Fragment reassembly: Fragments were purified (DNA Clean and Concentrator-5) and reassembled in a primerless PCR reaction using the following conditions: 25 µl of purified fragments and 25 µl of 2× Q5 High-Fidelity Master Mix (NEB). A final PCR was performed to recover only the full-length Dn29 CDS for further rounds of directed evolution. The following conditions were used for full-length gene recovery: PCR mixture: 25 µl of Platinum SuperFi II 2× Master Mix (Thermo Fisher Scientific), 10 µl of reassembled fragments, 2 µl each of DMS_universal_forward and DMS_universal_reverse primers (10 µM) and 11 µl of water. Six primer sets (DMS_NGS primers; Supplementary Table 8) were designed to amplify approximately 260-bp segments of the Dn29 CDS with Illumina adapter overhangs. Two rounds of PCR were performed to add P5/P7 adapters and i5/i7 indexes (FLAP2 primers). Amplicons were cleaned with AMPure XP beads (Beckman Coulter) between PCR rounds and after the final PCR. Amplicons were pooled in equimolar ratios, quantified using Qubit dsDNA High Sensitivity Kit (Thermo Fisher Scientific) and sequenced on an Illumina NextSeq 2000 (600-cycle kit). Full overlap between read 1 and read 2 was ensured for higher confidence in mutation calling. Paired-end reads were merged using BBMerge (version 39.06) and analyzed with a custom Python script. Reads with a summed error probability greater than 0.5 or containing frameshifts were filtered out. Enrichment for each amino acid (AA) between the input and output libraries was calculated using the following formula: ((%AAoutput) / (1 − %AAoutput)) / ((%AAinput) / (1 − %AAinput)). To distinguish library construction-based dropouts from selection-based dropouts in the enrichment heatmaps, any amino acids with zero reads in the output library were assigned a single read. The plasmid library was linearized by Eco105I digestion. Nanopore libraries were prepared using a barcoded nanopore sequencing kit (SQK-NBD114.24) with 1 µg of linearized plasmid library and sequenced on a MinION flow cell (R10.4.1) for 72 hours using MinKnow UI control software version 6.5.15. Sequencing reads were filtered using nanoq (version 0.9.0) with settings –min_len 4500–max_len 5500–min-qual 10 (that is, minimum q score of 10, a minimum read length equivalent to 90% of the expected read length and a maximum read length equivalent to 110% of the expected read length). The UMI sequence was extracted using Cutadapt (version 1.18) with settings –g ‘GGCGGTCACCATCACCACCACCACGCTACACG;max_error_rate = 0.2…ACTGTAC;max_error_rate = 0.2'–trimmed-only–revcomp–minimum_length 95. All UMI sequences were trimmed to 95 nt using seqkit (version 1.3-r106) with the command seqkit subseq -r 1:95. For each UMI cluster bin with at least 15 reads, a representative cluster sequence was generated by using usearch (version 11) with settings -cluster_fast -id 0.75 -strand both -sizeout -centroids and taking the first representative sequence of the output66. A final consensus sequence was generated by one round of polishing with Medaka (version 1.9.1) with settings -m r1041_e82_260bps_hac_g632. Counts for each unique variant were determined by tallying the total consensus sequences. Primers (DE_mammalian_forward and DE_mammalian_reverse) were designed to amplify the Dn29 CDS from the active variant PCR library, adding overhangs for Esp3I-compatible Golden Gate cloning. A mammalian expression vector was designed with the EF1α promoter upstream of an Esp3I Golden Gate landing pad, used as the destination for the protein variant library. The landing pad was followed by a T2A self-cleaving peptide sequence and an enhanced green fluorescent protein (EGFP) CDS. Golden Gate reaction mixture: 75 ng of mammalian expression vector, amplified variant library (3:1 molar ratio to vector), 1 µl of T4 DNA Ligase Buffer (NEB), 0.5 µl of T4 DNA Ligase (NEB), 0.5 µl of Esp3I (Thermo Fisher Scientific) and up to 10 µl of nuclease-free water. The library was transformed into Mach1 Escherichia coli and plated for overnight growth. Random colonies were picked, grown in 4 ml of TB-Carbenicillin and miniprepped (NucleoSpin Plasmid Transfection Grade Mini Kit; Machery-Nagel). One day before transfection, 12,000−18,000 HEK293FT cells were plated per well of a 96-well plate, aiming for 60–80% confluency at the time of transfection. Standard LSR + donor transfection: for transfections containing an LSR effector plasmid and a donor plasmid, each well was transfected with 725 ng of DNA, containing a 5:1 molar ratio of donor plasmid to effector plasmid, using 0.5 µl of Lipofectamine 2000 (Thermo Fisher Scientific) per well. Standard LSR-dCas9 + donor + guide transfection: LSR−dCas9 effector plasmid, donor plasmid and guide plasmid were transfected with 725 ng of total DNA, containing a 5:1:1 molar ratio of donor:effector:guide plasmid with 0.5 µl of Lipofectamine 2000 per well, unless specified otherwise in the figure legends. Modified transfection conditions: Experiments shown in Fig. In HEK293FT experiments, this consolidated plasmid was transfected at a 5:1 ratio of donor:effector/guide plasmid with 0.585 µl of Lipofectamine 2000 per well. For transfections containing two gRNA plasmids (Fig. The cells were incubated and monitored for 3 days for mCherry (donor plasmid) and GFP (effector plasmid) expression. Cells were then harvested for flow cytometry (Attune NxT Flow Cytometer; Thermo Fisher Scientific) or genomic DNA extraction for downstream analyses. Three days after transfection, cells were trypsinized with 50 µl of TrypLE (Gibco) for 10 minutes and then quenched with 50 µl of Stain Buffer (BD Biosciences). The 100-µl cell suspension was split into two 50-µl aliquots in U-bottom 96-well plates and centrifuged (300g, 5 minutes), and the supernatant was aspirated. One plate was resuspended in 200 µl of Stain Buffer and analyzed with an Attune NxT Flow Cytometer with autosampler (Thermo Fisher Scientific). DNA was cleaned with 0.9× AMPure XP (Beckman Coulter) beads. To assess integration efficiency and specificity, qPCR/ddPCR primers and probes were designed to span the left integration junction of attH1 and attH3, using a constant primer that binds to the donor plasmid sequence (ddPCR_donor_reverse_1), a genome binding primer near the pseudosite (ddPCR_attH1_forward_1, ddPCR_attH3_forward) and a FAM probe within the amplicon (ddPCR_attH1_probe_1, ddPCR_attH3_probe). For attH1, a second set of primers/probes was designed to target the right junction to verify measurement accuracy (ddPCR_attH1_2 primers/probe). Genomic reference primers and probes located nearby each attachment site were designed to measure pseudosite copy number for efficiency percentage calculations. Each reaction contained primers and probes for the target site (FAM probe) and a nearby reference locus (HEX probe). Reactions were run on a QX200 AutoDG Droplet Digital PCR System (Bio-Rad) using Bio-Rad QX Manager Software version 2.1.0, and data were analyzed and visualized using Microsoft Excel (version 16.89.1) and GraphPad Prism (version 10.3.0). For off-target detection or low-concentration samples, primers were increased to 20 µM and volume halved, and gDNA volume was increased to 4.95 µl. The master mix was split into three 10-µl technical replicates in a 384-well plate and run on a LightCycler 480 (Roche) using LightCycler 480 software version 1.5.1.62. Primer pairs for ddPCR and qPCR are provided in Supplementary Table 8. A fluorescent reporter assay was used to assess episomal plasmid recombination in HEK293FT cells. One day before transfection, 12,000−18,000 HEK293FT cells were plated per well of a 96-well plate, aiming for 60−80% confluency at the time of transfection. Upon recombination of the two attachment sequences, the EF1α promoter will drive expression of the mCherry CDS, which is read out by flow cytometry (Extended Data Fig. To assess attP recombination with model organism pseudosites, the attB sequence is replaced with the pseudosite sequences. Mismatching LSR (Bxb1) controls with each donor and acceptor plasmid is used to correct for the leaky mCherry background expression, defining the flow cytometry gating boundaries. Three days after transfection, the cells were trypsinized with 50 µl of TrypLE (Gibco) for 10 minutes, quenched with 50 µl of Stain Buffer, transferred to U-bottom 96-well plates and centrifuged (300g, 5 minutes), and then the supernatant was aspirated. Plates were resuspended in 200 µl of Stain Buffer and analyzed with an Attune NxT Flow Cytometer with autosampler. Site-directed mutagenesis (SDM) primers were designed using the script from Bi et al.67, selecting primers with melting temperature closest to 65 °C. For each mutation, a forward and reverse primer were generated, each containing the desired mutation at the center. PCR was run using the standard Platinum SuperFi II Master Mix protocol with annealing temperature at 65 °C. Products were cleaned with 0.5× AMPure XP beads. For Gibson assembly, 1 µl each of cleaned PCR product, 5 µl of Gibson master mix and 3 µl of water were incubated at 50 °C for 15 minutes and then transformed into Mach1 E. coli and plated. For simultaneous cloning of two or more mutations, universal primers were replaced with other mutationsʼ forward and reverse primers. Two mutations required a two-piece Gibson assembly, three mutations required a three-piece assembly and so forth. A Tn5 tagmentation and PCR amplification-based assay was used to unbiasedly measure the relative efficiency of all integration sites, as described in Durrant et al.12. In brief, extracted genomic DNA is tagmented with Tn5 transposase to randomly add adaptors throughout the genome. Then, two nested PCRs are performed, with primers that bind the donor plasmid and the Tn5 adaptor to amplify the donor−genome junction and add Illumina sequencing adaptors. UMIs on the donor plasmid enable counting of the relative frequencies of integration events at each genomic locus. HEK293FT cells were transfected as previously described, with a non-matching LSR (Bxb1) plasmid replacing the effector plasmid as a control for donor plasmid dilution. Cells were cultured for 2−3 weeks, passaging and analyzing by flow cytometry every 2−3 days at 80% confluency, until the non-matching LSR control was less than 1% mCherry+, indicating that the plasmid had nearly completely diluted out. Genomic DNA was extracted using a Quick-DNA Miniprep Plus Kit (Zymo Research), quantified by Qubit HS dsDNA Assay (Thermo Fisher Scientific), and 1 µg of gDNA per sample was DpnI digested (NEB) to remove residual donor plasmid. Tn5 transposase was purified following the Picelli et al.68 protocol. Tn5 adaptors were prepared by annealing top and bottom oligos (100 μM each) at 95 °C for 2 minutes, followed by slow cooling to 25 °C over 1 hour. The transpososome was assembled by combining 85.7 μl of purified Tn5 with 14.3 μl of pre-annealed oligos and incubating at room temperature for 1 hour. Samples were mixed thoroughly and incubated at 55 °C for 20 minutes. Reactions were placed on ice and purified with Zymo DNA Clean and Concentrate Kit according to the manufacturer's protocol, eluting in 11 μl of nuclease-free water. Sample quality was confirmed by Bioanalyzer to verify fragmentation of approximately 1.5−2.5 kb. Products were purified using 0.9× Agencourt AMPure XP SPRI beads and eluted in 11 μl of water. For size selection, approximately 40 μl of round 2 PCR product was loaded on a 2% agarose gel, and the smear between 300 bp and 800 bp was excised. DNA was extracted using the Monarch Gel Extraction Kit according to the manufacturer's protocol. Purified libraries were quantified using a Qubit dsDNA HS Assay Kit (Thermo Fisher Scientific) and pooled at equimolar ratios. Pooled libraries were sequenced on a NextSeq 2000 or Illumina MiSeq with 2 × 300-bp paired-end reads, using Nextseq 1000/2000 Control Software Suite version 1.7.1 or MiSeq Control Software version 4.1.0 and Illumina BaseSpace Software version 7.38.0. Raw sequencing data were processed using a custom bioinformatics pipeline as described in Durrant et al.12. To reduce occurrence of index hopping, unique dual i7 and i5 barcodes were used for the attH1 targeted samples in Fig. To directly compare specificity of samples with different numbers of measured integration events, samples were downsampled to the same total UMI count. Variable flexible linkers, including (GGS)8, (GGGGS)6 XTEN16, XTEN32-(GGSS)2 and XTEN48-(GGSS)2, were used to link the dCas9 and LSR. Spacers targeting loci proximal to the LSR integration site and non-targeting controls were cloned into an sgRNA-expressing plasmid via oligo ligation and Golden Gate cloning. Spacer selection was based on PAM sequence and pseudosite proximity. Two plasmid libraries (attP-L and attP-R) were constructed to determine nucleotide preference within the attP, with each 26-bp half-site mutagenized separately. Integrated DNA Technologies (IDT)-synthesized oligo pools contained 79% WT base and 7% each of the other bases at each position. Single-stranded oligo pools were subjected to second-strand synthesis. First, an oligo anneal reaction containing 2 µl of Library Oligo (100 µM), 4 µl of Klenow primer (100 µM), 3.4 µl of 10× STE Buffer and 24.6 µl of water was heated at 95 °C for 5 minutes and then cooled to room temperature. Next, a Klenow extension reaction containing 34 µl of annealed libraries, 8 µl of water, 5 µl of 10× NEBuffer2, 2 µl of 10 mM dNTPs (NEB) and 1 µl of DNA Polymerase I, Large (Klenow) Fragment (NEB, 5,000 U ml−1) was incubated at 37 °C for 30 minutes, purified (DNA Clean and Concentrator-5) and eluted in 20 µl of nuclease-free water. The purified product was cloned by Esp3I Golden Gate cloning into the donor plasmid backbone: 75 ng of pre-digested backbone, 3:1 molar ratio of attP library to backbone, 0.5 µl each of T4 DNA ligase (NEB) and Esp3I (Thermo Fisher Scientific), 1 µl of T4 DNA Ligase Buffer (NEB) and water to 10 µl were incubated at 37 °C for 1 hour, purified and eluted in 6 µl of nuclease-free water. Then, 1 µl of purified library was electroporated into Endura Electrocompetent Cells (Biosearch Technologies) at 10 μF, 600 Ω, 1,800 V, recovered in 2 ml of Lucigen Recovery Medium (37 °C, 1 hour), plated on 245-mm × 245-mm BioAssay dishes and incubated at 30 °C overnight. Final libraries were scraped, purified (Nucleobond Xtra Maxi EF Kit) and sequenced with an Illumina NextSeq 2000. Next, 2.2 × 106 HEK293FT cells were plated on 10-cm dishes 1 day before transfection to achieve 70% confluence at transfection. Then, 24 μg of total plasmid DNA was prepared at a 5:1:1 molar ratio (attP library:LSR effector:sgRNA). DNA and 72 μl of Lipofectamine 2000 were separately mixed with 1.5 ml of OMEM, incubated for 5 minutes and then combined and incubated for 10 minutes before adding dropwise to cells. After 3 days, cells were harvested with TrypLE (Gibco), and genomic DNA was extracted using a Quick-DNA Midiprep Plus Kit (Zymo Research). Integration events were amplified by single-step PCR with i5/i7 index-adding primers using all available genomic DNA. Biological replicates had 1-bp staggered amplicons to increase nucleotide diversity. PCR conditions were as follows: 25 μl of NEBNext High Fidelity PCR Master Mix, 2.5 μg of genomic DNA, 1.25 μl each of the attL or attR i5 or i7 primers (Supplementary Table 8) and water to 50 μl. PCR products were pooled and run on 2% agarose gel, and correct size bands were extracted (Monarch DNA Gel Extraction Kit). Libraries were quantified (Qubit 1× dsDNA High Sensitivity Assay; Thermo Fisher Scientific), pooled equimolar with 35% PhiX spike-in and sequenced on an Illumina NextSeq 2000 (150-bp paired-end reads). Libraries were demultiplexed using Illumina BaseSpace automatic demultiplexing workflow. Paired-end reads were merged using BBMerge (version 39.06) and analyzed with a custom Python script. Reads were filtered for exact amplicon length and QScore ≥ 30. Next, percent abundance of each nucleotide at each attP position was calculated for input and output libraries. Enrichment scores were computed using the following equation: \(r=\frac{{A}/1-A}{B/1-B}\), where A and B represent the read counts for selected nucleotides in output and input libraries, respectively, normalized to the total number of reads. Enrichment scores were converted to sequence logos, generated using Logomaker69 and matplotlib packages. Unique library members recovered as integration events were assessed by generating the set of unique reads. The number of unique integration events from NGS analysis was compared to ddPCR analysis of bulk genomic DNA for validation. Dinucleotide enrichment analysis was performed by first counting individual nucleotide frequencies at each position across all reads, followed by counting all possible dinucleotide combinations using a 2-bp sliding window at consecutive position pairs. Raw counts were normalized to total reads to calculate probabilities for both single nucleotides and dinucleotides at each position. To assess deviation from independence, observed dinucleotide probabilities were divided by the product of their constituent single-nucleotide probabilities: P(dinucleotide) / (P(nucleotide1) × P(nucleotide2)). Enrichment scores were calculated by comparing output to input library frequencies using \(r=\frac{{A}/1-A}{B/1-B}\), where A represents output library frequency and B represents input library frequency for each dinucleotide. Final values were log2 transformed and averaged across dinucleotide categories based on purine (R: A,G) and pyrimidine (Y: C,T) classification: RR (purine−purine), YY (pyrimidine−pyrimidine), RY (purine−pyrimidine) and YR (pyrimidine−purine). H1 hESCs and WTC-11 iPSCs were cultured in mTeSR Plus medium (STEMCELL Technologies) on Cultrex-coated (Bio-Techne) or Matrigel-coated (Corning) six-well plates. Cells were routinely subcultured at a 1:12 ratio using ReLeSR Passaging Reagent (STEMCELL Technologies) every 4 days or at 70−80% confluency. Three days after splitting (60% confluency), the cells were dissociated for 10 minutes with Accutase (STEMCELL Technologies) and plated in Cultrex-coated 96-well plates at 25,000−30,000 cells per well with 10 µM ROCK inhibitor. The next day (at 70% confluency), media were changed to include 50 µM ROCK inhibitor 2 hours before transfection. Then, 3 µg of plasmid DNA containing a 1:1 molar ratio of combined effector/guide plasmid to donor plasmid in 10-µl volume was diluted in 81 µl of mTeSR Plus and thoroughly pipette mixed. Next, 9 µl of FuGENE HD Transfection Reagent (Promega) was added to the DNA/mTeSR mix, thoroughly mixed and incubated for 12 minutes. After another thorough pipette mix, 7 µl of the DNA was added dropwise to each well. The cells were incubated at 37 °C, splitting 1:2 if 90% confluency was reached. After 3 days, the cells were dissociated with Accutase and split into two V-bottom plates, one for flow cytometry and one for gDNA harvest with QuickExtract DNA Solution (Biosearch Technologies). hESCs were differentiated into HPCs using the STEMdiff Hematopoietic Kit (STEMCELL Technologies). On day 10 of differentiation, 250 µl of non-adherent cells were collected from the supernatant using wide-bore P1000 tips and transferred to a V-bottom 96-well plate. Next, the cells were pelleted at 400g for 5 minutes, supernatant discarded and resuspended in 95 ml of Stain Buffer containing 1 µl of each antibody with a wide-bore pipette. The following antibodies were used: APC CD81 (BD Biosciences, 551112), APC CD147 (Thermo Fisher Scientific, A15706), Alexa Fluor 647 CD63 (BD Biosciences, 561983), APC/Cyanine7 CD34 (BioLegend, 343514) and PE CD43 (BioLegend, 343204). The cells were incubated in the dark for 20 minutes to 1 hour, washed once with Stain Buffer and flowed on the Attune Flow Cytometer (Thermo Fisher Scientific) using Attune Cytometric Software version 5.3.0 for collection. hESCs were diluted to one cell per 100 µl in mTeSR Plus medium supplemented with 1× CloneR (STEMCELL Technologies) and plated into two 96-well plates per sample. Single colonies were expanded to 24-well dishes when they covered half the surface area of the 96-well plate. At the next split, one quarter of each well was pelleted for gDNA extraction using QuickExtract DNA Solution (Biosearch Technologies). The extracted gDNA was cleaned with 0.9× AMPure XP beads and genotyped by ddPCR. Primers and probes were designed to target the attH1 junction (ddPCR_attH1_1 set), the donor sequence (Amp_forward, Amp_reverse, Amp_probe) and a nearby genomic reference sequence. On-target zygosity was determined by the attH1/reference ratio, and total zygosity was measured by the donor/reference ratio. Stem cells were transfected as previously described. Two days after transfection, cells were selected using Geneticin (Gibco) at 100 μg ml−1 and penicillin−streptomycin (Gibco) at 100 U ml−1. Cells were maintained in culture for 3 weeks until selection was complete and sufficient cell expansion was achieved for downstream applications, including cryopreservation and RNA extraction. Throughout the culture period, cell quality was monitored daily via microscopy to assess morphology and identify spontaneous differentiation events. Culture medium consisting of mTeSR Plus supplemented with penicillin−streptomycin and Geneticin was replaced daily. Upon reaching 70−80% confluency, cells were clump passaged using ReLeSR according to the manufacturer's instructions. If spontaneous differentiation was observed, cells were subjected to a straining protocol to remove differentiated cells and maintain pluripotent populations. In brief, media were aspirated, and four drops of ReLeSR were added to each well and incubated for 10 minutes. Cells were gently dislodged by pipetting or tapping the side of the culture dish to release cell clumps. The cell suspension was passed through a 40-μm cell strainer placed on a 50-ml Falcon tube and rinsed with 6 ml of PBS. The strainer was then inverted onto a fresh Falcon tube, and clumps were collected with 3 ml of culture medium before replating into six-well plates. For RNA-seq, cells from one well of a six-well plate were harvested by adding 1 ml of TRIzol reagent. The lysate was mixed by pipetting until a homogeneous viscosity was achieved and stored at −80 °C until RNA extraction. The upper aqueous phase containing RNA was carefully transferred to a fresh tube, and 0.5 ml of isopropanol was added and mixed. The supernatant was removed, and the RNA pellet was washed with 1 ml of 75% ethanol, mixed and centrifuged at 7,500g for 5 minutes at 4 °C. The RNA pellet was air dried for 5−10 minutes before resuspension. The extracted RNA was analyzed on a High Sensitivity RNA ScreenTape (Agilent Technologies) to measure the RNA integrity number (RIN) score, which was higher than 9 for all samples. mRNA enrichment was performed using the Roche/KAPA mRNA HyperPrep Kit according to the manufacturer's protocol. After mRNA enrichment, sequencing libraries were prepared using the HyperPrep Library Preparation Kit according to the manufacturer's instructions. Final libraries were sequenced on an Illumina NovaSeq X at a depth of at least 20 million reads per sample. Raw paired-end FASTQ files for each stem cell sample were first subjected to adapter and quality trimming using Trim Galore (version 0.6.7)70 with default settings, retaining reads ≥20 nt. Quality of raw and trimmed reads was assessed with FastQC (version 0.11.9)71 and aggregated using MultiQC (version 1.15)72. Trimmed reads were aligned to the GRCh38.p13 reference genome (GENCODE version 46 primary assembly, FASTA and GTF obtained from GENCODE) using STAR (version 2.7.10a)73 in two-pass mode. Concurrently, Salmon (version 1.10.0)78 was used to quantify transcript abundances (–validateMappings), and transcript-to-gene summarization was performed to produce gene-level count and transcripts per million (TPM) matrices. Gene-level count matrices (salmon.merged.gene_counts.tsv) were imported into R (version 4.3.1)82, and DESeq2 (version 1.38.1)83 was used for normalization and differential expression. A sample metadata table containing sample_id, condition and group_id was preprocessed so that identifiers matched the column names of the count matrix. For each stem cell line (group_id), the wild-type (‘WT') condition was identified, and pairwise comparisons were performed between each edited condition and the corresponding wild-type condition. Differential expression was modeled in DESeq2 with the formula ‘~ condition' (R formula syntax), meaning that gene counts were fit as a function of the experimental condition (edited or wild-type). Wald tests were used to estimate log2 fold changes, and P values were adjusted for multiple testing by the Benjamini–Hochberg (false discovery rate (FDR)) method. DEGs were defined as those with adjusted P < 0.05 and |log2 fold change | > 1. HEK293FT cells were transfected as previously described. On day 18, cells were trypsinized and strained through a 35-µm filter, and single mCherry+ cells were sorted into four 96-well plates per sample using a FACSAria Fusion (BD Biosciences). Single-cell colonies were expanded for 2 weeks until more than 50% confluent, with visual inspection to ensure single colony growth. Wells with zero or multiple colonies were excluded from analysis. Amplicons were sequenced via Sanger and/or NGS to determine on-target UMI count (PCR 1) and total UMI count (PCR 2), allowing calculation of on-target and off-target insertion counts per colony. HEK293FT cells were transfected with LSR and donor plasmids at a 1:5 ratio, as described above. After 3 days, cells were passaged into a 24-well dish for expansion. On day 5 after transfection, genomic DNA was harvested using the Zymo Quick-DNA Miniprep Plus Kit according to the manufacturer's instructions. PCR primers with 0−5 stagger base pairs were designed to amplify the attH1 site. Each PCR reaction contained 1 μg of gDNA, 40 μl of Platinum SuperFi II Master Mix, 3.2 μl of forward primer (PR_N284−PR_N289, 10 μM), 3.2 μl of reverse primer (PR_N290−PR_N295, 10 μM) and water to 80 μl total volume. After 25 cycles under standard conditions, products were purified with 0.8× AMPure XP beads. After seven cycles, libraries were purified with 0.7× AMPure XP beads, quantified via Qubit, pooled and sequenced using Illumina chemistry. Indel rates were calculated using Crispresso2 with the following command: CRISPResso -a CATTGGTGAATGTCTCATGTGGGTTTGAAAAGAGTGTGTATTCTGCTGTTGTTGGGTAAAGTAGTCTATACATGTCAATGATATGCTGTTGATTGATGCTGGTGTTGAATTCAACTATGTCCTTGCTGATTTTCTGCCTGCTGGATCTGTCTGAC -g GTCTATACATGTCAATGATA -r1 Read_1.fastq.gz -r2 Read_2.fastq.gz–keep_intermediate -w 20 -q 30–min_bp_quality_or_N 30–exclude_bp_from_left 10–exclude_bp_from_right 10–plot_window_size 20–ignore_substitutions. HEK293FT cells were plated in black-walled, clear-bottom optical plates, excluding edge wells and transfected with LSR and donor plasmids at a 1:5 ratio with four replicates per sample. Two days after transfection, cell viability was assessed using the CellTiter-Glo Assay (Promega). Cells were first refreshed with 100 μl of fresh D-10 medium and then treated with 100 μl of combined room temperature CellTiter-Glo Buffer and Substrate. for 2 minutes on a Tecan Spark Microplate Reader and incubated for an additional 8 minutes, and then luminescence was measured with 1,000-ms integration time. HEK293FT cells were plated into 96-well plates and transfected as described above. Two days after transfection, cells were dissociated with TrypLE and transferred to a V-bottom plate. Cells were centrifuged at 300g for 5 minutes and washed with 200 μl of DPBS. Next, cells were centrifuged again and resuspended in 50 μl of 4% paraformaldehyde (diluted in DPBS) for fixation. Cells were incubated for 10 minutes at room temperature. After fixation, cells were washed three times and stored in PBS overnight. Next, cells were washed twice with DPBS and incubated in blocking buffer composed of the following: 10% goat serum (Sigma-Aldrich, G6767), 0.5% NP-40 (Sigma-Aldrich, I3021) and 5% w/v saponin (Sigma-Aldrich, 84510) diluted in DPBS. Samples were incubated in blocking buffer for 30 minutes at room temperature in the dark. After incubation, samples were centrifuged and resuspended in a 1:1,000 dilution of Alexa Fluor 647-conjugated anti-phospho histone H2A.X (Ser139) antibody (Sigma-Aldrich, cat. 05-636-AF647, clone JBW301, lot 4214083) diluted in blocking buffer. Samples were incubated for 2 hours, washed twice with DPBS and analyzed on the Attune flow cytometer. HEK293FT cells were transfected with LSR and donor plasmids at a 1:5 ratio. After 3 days, cells were passaged into a 24-well dish for expansion. On day 5 after transfection, genomic DNA was harvested using the Quick-DNA Miniprep Plus Kit. For enrichment of translocation junctions, tagmented DNA underwent a two-step nested PCR. Reactions were amplified for 12 cycles (standard three-step protocol, 60 °C annealing, 1-minute extension), purified with 0.9× AMPure XP beads and eluted in 11 μl of water. The second nested PCR added indexes and Illumina adaptors using 10 μl of the first PCR product, 25 μl of Platinum SuperFi II Master Mix, 2.5 μl of inner primer (PR_N298−PR_N327 for upstream samples or PR_N328−PR_N356 for downstream samples), 2.5 μl of PR_N149 and 10 μl of water. After 20 cycles, products were purified with 0.9× AMpure XP beads, quantified by Qubit and pooled equimolarly. Amplicons between 300 bp and 900 bp were selected by gel extraction, quantified using the KAPA Library Quantification Kit and sequenced with Illumina chemistry for 600 cycles. Genomic rearrangements and translocations were identified using a custom pipeline. After merging paired-end reads, the sequence between the inner primer and the attH1 dinucleotide core (‘upstream sequence') was searched for, allowing up to three mismatches to account for sequencing errors. Reads containing the upstream sequence were processed to extract downstream portions (minimum 20-bp length), which were then aligned to both WT and donor insertion references using BWA-MEM (-a -M -k 8 -T 20)79. Reads were classified based on alignment quality (≥80% alignment and mapping quality (MAPQ) ≥ 20) into WT aligned, donor insertion aligned or potential translocations. Potential translocation reads underwent further analysis by BWA alignment to the human reference genome (hg38). The resulting alignments were converted to sorted BAM files using SAMtools (version 1.22) for visualization and BED files using BEDTools (version 2.31.0) for genome browser compatibility. Translocation events were classified into four categories: (1) close to target (within 2 kb of on-target site, reclassified as WT aligned); (2) EF1α promoter aligned (mapping to chr6 region 73,519,610−73,522,070, reclassified as donor insertion aligned); (3) non-chr10 translocations; and (4) chr10 rearrangements. All reads containing the upstream sequence were aligned to WT and donor insertion references using BWA (version 0.7.19). All reads that did not align to these references were then aligned to the human genome. Finally, all reads that did not align to the human genome were aligned to AAV ITR sequences using Geneious Prime (version 11.0.20.1+1). sgRNA spacers targeting cell surface markers CD81, CD147 and CD63 were cloned into the LentiGuide-Puro construct (Addgene, 52963). Lentivirus was generated using the LV-MAX Lentiviral Production Kit (Invitrogen) according to the manufacturer's instructions and concentrated 100× with Lenti-X Concentrator (Takara). HPCs were diluted to 50,000 cells per well in 100 µl of Medium B (STEMCELL Technologies, STEMdiff Hematopoietic Kit) in a 96-well plate. Media were changed the next day. Four days after transduction, a subset of cells was stained for cell surface markers. Remaining cells were treated with 1 μg ml−1 puromycin for 4 days to select for transduced cells, followed by cell surface marker staining. Antibodies used for cell surface staining were as follows: APC CD81 (BD Biosciences, cat. 551112, lot 2061009, clone JS-81, 1:100 dilution); APC CD147 (Thermo Fisher Scientific, cat. A15706, lot 540242, clone 8D12, 1:100 dilution); Alexa Fluor 647 CD63 (BD Biosciences, cat. 561983, lot 2112938, clone H5C6, 1:100 dilution); APC CD63 (BioLegend, cat. 343514, lot B413134, clone 581, 1:100 dilution); and PE CD43 (BioLegend, cat. All antibodies chosen are validated for flow cytometric analysis of human cells according to the manufacturer's website. The full-length WT Dn29 protein sequence and minimal attB-L or attB-R sequence (attB-L: GTAGACAAGGAAGGTAATGA; attB-R: GAAATAAGTTTGATAGATAT) were input into the AlphaFold3 web server with the seed set to ‘auto'. Five models were generated for each query of Dn29 bound to an attB half-site. One model (Dn29 × attB-R) out of the 10 generated models met this criterion and was selected for further analysis. Despite 4KIS being bound to attP instead of attB, domain-wise comparisons showed strong alignment: RMSDs were 1.341 and 1.707 for the zinc-ribbon domain and the recombinase domain, respectively (Extended Data Fig. Protein/DNA interface residues were identified with the InterfaceResidues pymol script using default settings. The efficiency and specificity data of all Dn29 variants were split into a training and test set based on what round of experimentation they were generated in. The training set, called round 1, contained all variants from the two single-mutation validation experiments, where mutations were tested individually on top of variant 127 (Fig. The testing set contained all higher-order combinations from the iterative rounds of driver mutation stacking (rounds 2−5). The efficiency (percent of integrations at attH1) was normalized to WT, and specificity (ratio of attH1/attH3 activity) was log transformed. The full amino acid sequences of the protein variants were one-hot encoded, a technique that transforms each amino acid in the sequence into a binary vector of length 21 (corresponding to the 20 standard amino acids plus a stop codon), where the position corresponding to that amino acid is set to 1 and all others are 0—this encoding is then flattened into a single vector representing the entire sequence. Activity in the training set was modeled using linear regression, ridge regression, XGBoost and CatBoost with the scikit-learn (version 1.0.2), xgboost (version 1.6.2) and catboost (version 1.2.5) Python libraries. For the ridge regression, optimal α was identified through minimization of the testing set R2 (α = 0.8 for efficiency model, α = 1.3 for specificity model). Hyperparameter optimizations were conducted for XGBoost and CatBoost by performing a randomized search, evaluating on negative mean squared error, using the following parameters: XGBoost: ‘n_estimators': [100, 500, 1,000], ‘learning_rate': [0.01, 0.05, 0.1], ‘max_depth': [3, 5, 7], ‘subsample': [0.5, 0.6, 0.7, 0.8, 1.0], ‘colsample_bytree': [0.7, 0.8, 1.0]; CatBoost: ‘iterations': [100, 200, 500, 1,000], ‘learning_rate': [0.01, 0.05, 0.1, 0.2], ‘depth': [4, 6, 8, 10], ‘l2_leaf_reg': [1, 3, 5, 7, 9], ‘bagging_temperature': [0, 1, 2, 3], ‘random_strength': [1, 1.5, 2, 3], ‘border_count': [32, 64, 128], ‘grow_policy': [‘SymmetricTree', ‘Depthwise', ‘Lossguide']. The following parameters were chosen for each model: XGBoost, specificity: ‘subsample' = 0.5, ‘n_estimators' = 1,000, ‘max_depth' = 7, ‘learning_rate' = 0.1, ‘colsample_bytree' = 0.8; XGBoost, efficiency: ‘subsample' = 0.7, ‘n_estimators' = 100, ‘max_depth' = 7, ‘learning_rate' = 0.05, ‘colsample_bytree' = 0.7; CatBoost, specificity: ‘random_strength' = 1.5, ‘learning_rate' = 0.1, ‘l2_leaf_reg' = 1, ‘iterations' = 1,000, ‘grow_policy' = ‘Depthwise', ‘depth' = 4, ‘border_count' = 128, ‘bagging_temperature' = 1; CatBoost; efficiency: ‘random_strength' = 1.5, ‘learning_rate' = 0.1, ‘l2_leaf_reg' = 7, ‘iterations' = 500, ‘grow_policy' = ‘Lossguide', ‘depth' = 4, ‘border_count' = 32, ‘bagging_temperature = 2. Effector constructs were cloned into an in vitro transcription (IVT) plasmid as previously described84. IVT templates were generated by PCR using primers oGX006 and oLGR009, which incorporate a poly(A) tail and correct the T7 promoter mutation. PCR reactions were performed using KAPA HiFi HotStart 2× (Roche) Master Mix with 6.25 ng of plasmid template per 25-µl reaction. The PCR protocol involved annealing at 63 °C, extending for 45 seconds per kilobase and running for 18 cycles. The reactions were purified using 0.8× volume of SPRI beads and eluted into water. The purified PCRs were analyzed by gel electrophoresis and NanoDrop to ensure correct size and determine concentration. The IVT reactions were set up using the HiScribe T7 High-Yield RNA Synthesis Kit (NEB, E2040S), modified with full pseudo-UTP substitution using N1-Methyl-Pseudo-U (TriLink Biotechnologies, N-1081) and co-transcriptionally capped with CleanCap AG (TriLink Biotechnologies, N-7113). Each IVT reaction contained 5 mM ATP, CTP, GTP and pseudo-UTP, 4 mM CleanCAP AG, 1× Transcription Buffer, 3.75 ng µl−1 DNA template, 1 U µl−1 Murine RNAse Inhibitor (NEB, M0314L), 0.002 U µl−1 yeast inorganic pyrophosphatase (NEB, M2403L) and 5 U µl−1 T7 RNA polymerase. Reactions were incubated for 2.5 hours at 37 °C. Next, the mRNA was purified using lithium chloride. To each reaction, 1.5× water and 1.25 × 7.5 M LiCl were added. The supernatant was discarded, and the pellet was rinsed with 70% ice-cold ethanol to remove residual salts. After another maximum speed spin for 10 minutes at 4 °C, the mRNA was resuspended in water and stored at −80 °C. The mRNA was analyzed on an Agilent TapeStation and by Qubit RNA High Sensitivity (Thermo Fisher Scientific) to ensure correct size and determine concentration. Two days before electroporation, T cells were seeded at 1 × 106 fresh cells per milliliter and activated with a 1:1 bead-to-cell ratio with anti-CD3/CD28 Dynabeads (Life Technologies, 40203D). On the day of electroporation, the beads were magnetically removed, and the T cells were electroporated with 2 µg of LSR−dCas9−P2A−EGFP mRNA and 2 µg of sgRNA (Synthego) for LSR−dCas9 samples or 1 µg of LSR−P2A−EGFP mRNA for LSR samples using the Lonza P3 Primary Cell Kit. Immediately after electroporation, 80 µl of pre-warmed culture media was added to the Nucleocuvette strip, which was then incubated at 37 °C for 15−30 minutes. Next, 2 × 105 cells per condition were split into 96-well U-bottom plates in 100 µl of serum-free medium (TheraPEAK X-VIVO-15 Serum-free Hematopoietic Cell Medium, BEBP04-744Q) supplemented with 5 ng μl−1 IL-7 and 5 ng μl−1 IL-15. Cells were then transduced at an MOI of 1 × 105 genome copies per cell with ssAAV or scAAV vectors of serotype 6 (AAV6) containing the e-attP sequence, attH1 sgRNA target sequence and an mCherry expression cassette, which were ordered from VectorBuilder. The next morning, cells were spun down at 300g for 5 minutes; the serum-free medium was removed; and cells were resuspended in 200 µl of fresh cX-VIVO. Cells were maintained and passaged as needed by the addition of cX-VIVO every 2−3 days. Peripheral blood mononuclear cells (PBMCs) from healthy human blood donors were collected under an approved institutional review board protocol by the Stanford Blood Center and used to isolate human T cells. In brief, leukoreduction chambers from processing of platelet donations were used to isolate PBMCs using density centrifugation with Ficoll (Lymphoprep; STEMCELL Technologies) within SepMate tubes (STEMCELL Technologies) according to the manufacturerʼs instructions. Next, primary human CD3+ T cells were isolated by negative selection using a Human CD3 T Cell Enrichment Kit (STEMCELL Technologies) according to the manufacturerʼs instructions. Isolated primary human CD3 T cells were counted using an automated cell counter (Countess; Thermo Fisher Scientific) and activated using anti-human CD3/CD28 Dynabeads (Cell Therapy Systems; Thermo Fisher Scientific) at a 1:1 ratio in X-VIVO 15 medium (Lonza) supplemented with 5% FBS (MilliporeSigma) and 50 IU ml−1 human IL-2 (PeproTech). T cells were activated at a 1:1 ratio of cells to Dynabeads and initially cultured in standard tissue culture incubators at approximately 1 × 106 cells per milliliter of medium. After gene editing/electroporations, T cells were counted and reseeded at approximately 1 × 106 cells per milliliter, with additional IL-2 and X-VIVO 15 complete media added every 2–3 days to maintain a culture density of approximately 1 × 106 cells per milliliter. Forty-eight hours after activation, Dynabeads were magnetically removed from activated T cell cultures by incubating for 2 minutes at room temperature on a magnet (EasySep Magnet; STEMCELL Technologies), and cells were counted using an automated cell counter (Countess; Thermo Fisher Scientific). For electroporations, 1−2 million T cells per editing condition were gently pelleted by centrifugation at 90g for 10 minutes, followed by careful aspiration of the supernatant. T cell pellets were resuspended in 20 μl per editing condition in P3 buffer (Lonza) and then mixed with prepared LSR mRNA and DNA templates. All plasmids were purified using the ZymoPure II Plasmid Midiprep Kit (Zymo Research). The 5.8-kb CD19 CAR-expressing plasmid contains the EF1α promoter, tNGFR EC domain (cell surface reporter), T2A, 1928z CAR and bGH poly(A). Total nucleic acid volume was limited to 5 µl. Immediately after electroporation, 80 μl of pre-warmed X-VIVO 15 media was added to each cuvette, and cells were rested within the cuvettes for 15 minutes in a standard 37 °C tissue culture incubator. The cells were then gently resuspended and transferred to standard 96-well round-bottom plates with 300 µl of total X-VIVO 15 complete medium with 50 IU ml−1 human IL-2. T cells were maintained at 0.5 × 106 to 1 × 106 cells per milliliter, and X-VIVO 15 complete medium with 50 IU ml−1 human IL-2 was refreshed every 2−3 days. Three days after electroporation, 50 µl of T cells was collected for staining and flow cytometry. In brief, cells were centrifuged, washed once with 200 μl of cell staining buffer and stained with Ghost Dye Red 780 at a 1:1,000 dilution (Tonbo, 13-0865-T500) for 20 minutes in the dark at 4 °C. The cells were measured using an Attune NxT Cytometer with a 96-well autosampler (Invitrogen) and analyzed using FlowJo software (version 10.10.0) for viability, mCherry fluorescence (expressed on the AAV) and GFP fluorescence (effector expression). The remaining 150 µl of T cells in culture was centrifuged at 300g for 5 minutes, and the gDNA was harvested using QuickExtract DNA Solution (Biosearch Technologies) and analyzed by ddPCR as described above. At 13 days after non-viral gene editing, T cell editing was assessed by flow cytometry, and cells edited with goldDn29, goldDn29−dCas9 and Dn29−dCas9 were selected for the killing assay. T cells were mixed at indicated effector:target (E:T) ratios with target Nalm6 leukemia cells in 96-well plates, with four different Nalm6 conditions (16,000, 8,000, 4,000 or 2,000 cells per well) and 4,000 T cells per well. Cell killing was assessed by flow cytometry at 48 hours, and the percentage of Nalm6 tumor cell killing was calculated by taking 1 − (no. of Nalm6 cells alive in experimental condition / no. of Nalm6 cells alive in no-T-cell conditions). Effector cells were stained with human NGFR-APC (clone ME20.4, BioLegend, 345108), and target cells were stained with human CD19-PE (clone HIB19, BioLegend, 982402), for flow cytometric analysis. Artificial intelligence language models (ChatGPT and Claude) were used for generating custom Python scripts for data analysis and visualization, assistance with copyediting and infilling preliminary drafts of some sections based on an author-provided outline. All content generated by artificial intelligence was thoroughly reviewed, edited and verified by the authors. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The NGS dataset is available on the National Center for Biotechnology Information Sequence Read Archive at BioProject PRJNA1172311 (ref. RNA-seq analysis scripts and parameter settings are available in our GitHub repository (https://github.com/julianaqmartins/bulkRNAseq) and archived at Zenodo (https://doi.org/10.5281/zenodo.17239032)86. A global reference for human genetic variation. Gogol-Döring, A. et al. Genome-wide profiling reveals remarkable parallels between insertion site selection properties of the MLV retrovirus and the piggyBac transposon in primary human CD4+ T cells. Chaikind, B., Bessen, J. L., Thompson, D. B., Hu, J. H. & Liu, D. R. A programmable Cas9-serine recombinase fusion protein that operates on DNA sequences in mammalian cells. Standage-Beier, K. et al. RNA-guided recombinase-Cas9 fusion targets genomic DNA deletion and integration. Pallarès-Masmitjà, M. et al. Find and cut-and-transfer (FiCAT) mammalian genome engineering. Wu, Z., Yang, H. & Colosi, P. Effect of genome size on AAV vector packaging. Programmable deletion, replacement, integration and inversion of large DNA sequences with twin prime editing. Targeted DNA integration in human cells without double-strand breaks using CRISPR-associated transposases. Yarnall, M. T. N. et al. Drag-and-drop genome insertion of large sequences without double-strand DNA cleavage using CRISPR-directed integrases. & Kleinstiver, B. P. Precise cut-and-paste DNA insertion using engineered type V-K CRISPR-associated transposases. Systematic discovery of recombinases for efficient integration of large DNA sequences into the human genome. Thyagarajan, B., Olivares, E. C., Hollis, R. P., Ginsburg, D. S. & Calos, M. P. Site-specific genomic integration in mammalian cells mediated by phage phiC31 integrase. & Fowler, D. M. A platform for functional assessment of large variant libraries in mammalian cells. & Calos, M. P. Directed evolution of a recombinase for improved genomic integration at a native human sequence. Keravala, A. et al. Mutational derivatives of ɸC31 integrase with increased efficiency and specificity. Expanding the scope of site-specific recombinases for genetic and metabolic engineering. Evolution of programmable zinc finger-recombinases with activity in human cells. Directed evolution of recombinase specificity by split gene reassembly. Correction of a Factor VIII genomic inversion with designer-recombinases. Mukhametzyanova, L. et al. Activation of recombinases at specific DNA loci by zinc-finger domain insertions. Lansing, F. et al. A heterodimer of evolved designer-recombinases precisely excises a human genomic DNA locus. & Barbas3rd, C. F. ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering. Buchholz, F. & Stewart, A. F. Alteration of Cre recombinase site specificity by substrate-linked protein evolution. Müller, K. M. et al. Nucleotide exchange and excision technology (NExT) DNA shuffling: a robust method for DNA fragmentation and directed evolution. & Smith, M. C. M. Zinc is essential for high-affinity DNA binding and recombinase activity of φC31 integrase. Van Duyne, G. D. & Rutherford, K. Large serine recombinase domain structure and attachment site binding. Rutherford, K., Yuan, P., Perry, K., Sharp, R. & Van Duyne, G. D. Attachment site recognition and regulation of directionality by the serine integrases. Chen, T. & Guestrin, C. XGBoost: a scalable tree boosting system. Prokhorenkova, L., Gusev, G., Vorobev, A., Dorogush, A. V. & Gulin, A. CatBoost: unbiased boosting with categorical features. In 32nd Conference on Neural Information Processing Systems (NeurIPS 2018) https://proceedings.neurips.cc/paper_files/paper/2018/file/14491b756b3a51daac41c24863285549-Paper.pdf (2018). Learning protein fitness models from evolutionary and assay-labeled data. Accurate structure prediction of biomolecular interactions with AlphaFold 3. B., Li, H. & Van Duyne, G. D. Coiled-coil interactions mediate serine integrase directionality. Rowley, P. A., Smith, M. C. A., Younger, E. & Smith, M. C. M. A motif in the C-terminal domain of ɸC31 integrase controls the directionality of recombination. Yuan, P., Gupta, K. & Van Duyne, G. D. Tetrameric structure of a serine integrase catalytic domain. Ryan, M. D., King, A. M. & Thomas, G. P. Cleavage of foot-and-mouth disease virus polyprotein is mediated by residues located within a 19 amino acid sequence. The ‘cleavage' activities of foot-and-mouth disease virus 2A site-directed mutants and naturally occurring ‘2A-like' sequences. Efficient site-specific integration of large genes in mammalian cells via continuously evolved recombinases and prime editing. Walton, R. T., Christie, K. A., Whittaker, M. N. & Kleinstiver, B. P. Unconstrained genome targeting with near-PAMless engineered CRISPR−Cas9 variants. Singh, S., Ghosh, P. & Hatfull, G. F. Attachment site selection and identity in Bxb1 serine integrase-mediated site-specific recombination. Li, H., Sharp, R., Rutherford, K., Gupta, K. & Van Duyne, G. D. Serine integrase attP binding and specificity. Schnepf, M., von Reutern, M., Ludwig, C., Jung, C. & Gaul, U. Transcription factor binding affinities and DNA shape readout. Siddharthan, R. Dinucleotide weight matrices for predicting transcription factor binding sites: generalizing the position weight matrix. Heddi, B., Oguey, C., Lavelle, C., Foloppe, N. & Hartmann, B. Intrinsic flexibility of B-DNA: the experimental TRX scale. Packer, M. J., Dauncey, M. P. & Hunter, C. A. Sequence-dependent DNA structure: dinucleotide conformational maps. Bessen, J. L. et al. High-resolution specificity profiling and off-target prediction for site-specific DNA recombinases. Zhang, Q., Azarin, S. M. & Sarkar, C. A. Model-guided engineering of DNA sequences with predictable site-specific recombination rates. Lin, Y.-C. et al. Genome dynamics of the human embryonic kidney 293 lineage in response to cell biology manipulations. Hazelbaker, D. Z. et al. Large serine integrase off-target discovery and validation for therapeutic genome editing. Malla, S., Dafhnis-Calas, F., Brookfield, J. F. Y., Smith, M. C. M. & Brown, W. R. A. Olorunniji, F. J., He, J., Wenwieser, S. V. C. T., Boocock, M. R. & Stark, W. M. Synapsis and catalysis by activated Tn3 resolvase mutants. Hayashi, H., Kubo, Y., Izumida, M. & Matsuyama, T. Efficient viral delivery of Cas9 into human safe harbor. Robust, persistent transgene expression in human embryonic stem cells is achieved with AAVS1-targeted integration. Creation of engineered human embryonic stem cell lines using phiC31 integrase. Transcription activator-like effector nuclease (TALEN)-mediated CLYBL targeting enables enhanced transgene expression and one-step generation of dual reporter human induced pluripotent stem cell (iPSC) and neural stem cell (NSC) lines. The sound of silence: transgene silencing in mammalian cell engineering. Estes, B. J. G. et al. Development of circular AAV cargos for targeted seamless insertion with large serine integrases. Engineered Bxb1 variants improve integrase activity and fidelity. Systematic development of reprogrammed modular integrases enables precise genomic integration of large DNA sequences. & Hawley, R. G. Strategies to insulate lentiviral vector-expressed transgenes. Joung, J. et al. Genome-scale CRISPR−Cas9 knockout and transcriptional activation screening. Edgar, R. C. Search and clustering orders of magnitude faster than BLAST. Bi, C. et al. A python script to design site-directed mutagenesis primers. Picelli, S. et al. Tn5 transposase and tagmentation procedures for massively scaled sequencing projects. B. Logomaker: beautiful sequence logos in Python. Andrews, S. FastQC: a quality control tool for high throughput sequence data. Ewels, P., Magnusson, M., Lundin, S. & Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. & García-Alcalde, F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. Sayols, S., Scherzinger, D. & Klein, H. dupRadar: a Bioconductor package for the assessment of PCR artifacts in RNA-Seq data. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Di Tommaso, P. et al. Nextflow enables reproducible computational workflows. Merkel, D. Docker: lightweight linux containers for consistent development and deployment. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Schmidt, R. et al. Base-editing mutagenesis maps alleles to tune human T cell functions. Fanton, A. et al. Site-specific DNA insertion into the human genome with engineered recombinases. Suehnholz, S. P. et al. Quantifying the expanding landscape of clinical actionability for patients with cancer. Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. Cacheiro, P. et al. Human and mouse essentiality screens as a resource for disease gene discovery. Meehan, T. F. et al. Disease model discovery from 3,328 gene knockouts by The International Mouse Phenotyping Consortium. Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR–Cas9 essentiality screens in cancer cells. Rehm, H. L. et al. ClinGen—the Clinical Genome Resource. We thank A. Natarajan, N. Enright, B. Mijts, M. C. Bassik, H. Nishimasu, C. Ito, M. Fanton, N. T. Perry and all members of the Hsu laboratory for helpful discussions; B. Plosky, C. Ricci-Tam and the Arc Institute Scientific Publications Team for assistance with the manuscript; and the FACS Core and Genomics Platform at the Arc Institute for experimental assistance. are supported by funding from the Arc Institute. was partially supported by the National Science Foundation Graduate Research Fellowship Program (2019284848). was partially supported by the Curci Foundation Ph.D. Scholars Program. Arc Institute, Palo Alto, CA, USA Alison Fanton, Liam J. Bartie, Juliana Q. Martins, Vincent Q. Tran, Laine Goudy, Courtney Kernick, Matthew G. Durrant, Jingyi Wei, April Pawluk, Silvana Konermann, Luke A. Gilbert, Theodore L. Roth & Patrick D. Hsu University of California, Berkeley − University of California, San Francisco, Graduate Program in Bioengineering, Berkeley, CA, USA Gladstone-UCSF Institute of Genomic Immunology, San Francisco, CA, USA Laine Goudy, Zev Armour-Garb & Alexander Marson Helen Diller Family Comprehensive Cancer Center, University of California, San Francisco, San Francisco, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar provided overall supervision of the research, with assistance from S.K., A.M., T.L.R. wrote the manuscript, with input from all authors. acknowledges outside interest as a co-founder of Monet AI, Terrain Biosciences and Stylus Medicine; board of directors at Stylus Medicine; board observer at Terrain Biosciences; scientific advisory board member at Veda Bio; and venture partner at Thrive Capital. acknowledge outside interest in Stylus Medicine. are inventors on patents relating to this work. A.M. is a co-founder of Site Tx, Arsenal Biosciences, Spotlight Therapeutics and Survey Genomics; serves on the boards of directors at Site Tx and Survey Genomics; is a member of the scientific advisory boards of Network.bio, Site Tx, Arsenal Biosciences, Cellanome, Survey Genomics, NewLimit, Amgen and Tenaya; owns stock in Network.bio, Arsenal Biosciences, Site Tx, Cellanome, Spotlight Therapeutics, NewLimit, Survey Genomics, Tenaya and Lightcast; and has received fees from Network.bio, Site Tx, Arsenal Biosciences, Cellanome, Spotlight Therapeutics, NewLimit, AbbVie, Gilead, Pfizer, 23andMe, PACT Pharma, Juno Therapeutics, Tenaya, Lightcast, Trizell, Vertex, Merck, Amgen, Genentech, GLG, ClearView Healthcare, AlphaSights, Rupert Case Management, Bernstein and ALDA. A.M. is an investor in and informal advisor to Offline Ventures and a client of EPIQ. The Marson laboratory has received research support from the Parker Institute for Cancer Immunotherapy, CZI, the Emerson Collective, the Arc Institute, Juno Therapeutics, Epinomics, Sanofi, GlaxoSmithKline, Gilead and Anthem and reagents from Genscript, Illumina, Ultima and 10x Genomics. has filed patents on CRISPR tools and CRISPR functional genomics, is a co-founder of nChroma Bio and is a consultant for nChroma Bio. The Roth laboratory has received research support from the Parker Institute for Cancer Immunotherapy and Northpond Ventures. The other authors declare no competing interests. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. White indicates the WT amino acid, black indicates an amino acid dropout, and red and blue indicate enrichment and depletion, respectively, compared to the expected mutation frequency (0.006%, shown in gray). c, Enrichment score of the catalytic serine (S11) and four conserved zinc-coordinating cysteines throughout various timepoints in the directed evolution campaign. d, Box plot of stop codon enrichment scores across all coding sequence positions at various time points (n = 515). e, Correlation between mutational tolerance (average non-WT residue enrichment) and phylogenetic conservation (% identity from multiple sequence alignment of 106 LSR clusters within 30% identity of Dn29). Each dot represents a position in the CDS, colored by domain. NTD: N-terminal domain, CTDs: C-terminal domains. a, Integration efficiency (left) and specificity (right) of variants from different libraries throughout directed evolution, as fold change to WT Dn29. Each dot represents the mean of n = 2 biological replicates. Dotted line represents the average WT activity. b, Integration efficiency of output library variants, as fold change to WT Dn29. Dots and error bars represent the mean ± s.d. c, Integration efficiency of mutations in variant 127 reverted to WT, shown as fold change to variant 127. of n = 2 biological replicates, shown as dots. d, Integration efficiency (orange, left y axis) and specificity (teal, right y axis) of variant 127 with position 341 saturation mutagenesis, shown as fold change to WT. e, Integration efficiency (orange, left y axis) and specificity (teal, right y axis) of variant 127 with lysine mutations of putative DNA binding residues. f, Integration efficiency (orange, left y axis) and specificity (orange, right y axis) of variant 381 with significant (one-tailed P < 0.05) mutations from second validation round. g, Integration efficiency vs. specificity of 341K and 341Q lineages with driver mutations. h, Epistatic interactions between rounds 2 and 3 mutations. x axis: expected effect (sum of single mutant log2(fold change)); y axis: observed effect (double mutant log2(fold change)). Pearson r = 0.2841. i, Top, Recombination efficiencies of Dn29 variants using the three plasmid recombination assay, shown as percent of mCherry+ cells. of n = 3 biological replicates, shown as dots. mCherry expresses upon recombination between attP and attB/H. a,b, Top 40 coefficients of the (a) efficiency model and (b) specificity model. c,d, Correlation between the (c) efficiency (Spearman's ρ = 0.623, two-tailed P = 0.0025) and (d) specificity (Spearman's ρ = 0.930, two-tailed P = 2.643e-04) models' coefficient values and experimental impact of mutations, shown as fold change to the base mutant (variant 127 from mutations identified in the first round of individual validation (Fig. 2C-E), or variant 381 from mutations identified in the second round of individual mutation validation (Extended Data Fig. e, Efficiency and specificity of model-guided variants, which each contain two mutations on top of superDn29, and were designed to maximize efficiency (orange) or specificity (teal). Each dot represents the mean of n = 2 biological replicates. f,g, Comparison between the model-predicted and true (f) efficiency and (g) specificity of the model-guided variants. Each dot represents the mean of n = 2 biological replicates. a, AlphaFold3 model of Dn29 bound to the attB-R half site, colored by protein domain. b, Alignment of the Dn29 attB-R AlphaFold3 structure to listeria integrase (LI) C-terminal domain bound to attP crystal structure (PDB: 4KIS). Root mean square deviation (RMSD) values provided. c, Coiled-coil hinge region with efficiency mutations (orange). Corresponds to box c in panel a. d, Top, schematic of plasmid recombination assay for attachment site recombination. mCherry expresses upon recombination between attachment sites X and Y. Bottom, recombination of Dn29, key variants, and mismatching LSR control between attP, attB, attL, and attR, measured by mCherry median fluorescence intensity (MFI). of n = 3 biological replicates, shown as dots. e, Efficiency mutation D503N (orange) and neighboring DNA-interfacing residue R502 (magenta). Corresponds to box e in panel a. f, Specificity (teal) and efficiency (orange) mutations near DNA-interfacing residues. N341 (green) is both a specificity and efficiency mutation. a, attH1 integration efficiency of donor:effector:guide plasmid stoichiometries for Dn29 (left, n = 2 biological replicates) and Dn29−dCas9 (right, n = 4 biological replicates). b, attH1 integration efficiency of Dn29−dCas9 with direct fusion or 2A peptide linkers, with H1-g3 and non-targeting sgRNA (NTG). 2A peptides ranked in order from least to most complete ribosomal skipping. c, Schematic of sgRNA targets for Pf80 attH1 pseudosite (chr11:64,243,293). d, Integration efficiency at Pf80 attH1 pseudosite by Pf80 and Pf80−dCas9, shown as fold change to NTG. e, Schematic of sgRNA targets for Si74 attB, pre-inserted at the AAVS1 locus. f, Integration efficiency at AAVS1 Si74 attB with Si74 and Si74−dCas9, shown as fold change to NTG. Dots represent the 3 technical replicates per biological replicate. g, Schematic of sgRNA targets for the Nm60 attH2 pseudosite (chr9:83,308,045). h, Integration efficiency of Nm60−dCas9 at attH2 with guides targeting upstream and downstream of the pseudosite, in single and multiplex. i, Genome-wide specificity of Nm60 and Nm60−dCas9 with Nm60-g2 sgRNA targeting Nm60 attH2. j, Integration efficiency (fold change to NTG) of all LSR−dCas9 fusions (Dn29−dCas9, Pf80−dCas9, Si74−dCas9, Nm60−dCas9) at all pseudosites/gRNAs tested in Fig. k, Schematic of sgRNA targets for Dn29 attH2 pseudosite (chr10:58,514,256). Blue targets: NGG PAMs; purple target: NGH PAM. l, Absolute integration efficiency of Dn29−dCas9 compared to Dn29−SpG−HF1 at attH2. of n = 3 biological replicates, shown as dots. Dots represent the mean of n = 3 biological replicates, with each dot representing a unique sgRNA. n, Integration efficiency of Nm60−dCas9 at attH2 with a donor-binding sgRNA (Nm60-g2) plasmid. *=one-tailed P < 0.05. o, Correlation between integration efficiency (fold change to NTG, n = 3 biological replicates) and genome-wide specificity at attH3 (n = 2 biological replicates) of various attH3-targeting sgRNAs. Dots and error bars represent the mean ± s.d. of samples with n ≥ 3 biological replicates. Data shown is the same as presented in Figs. a, Comparison of hifiDn29−dCas9 on-target and off-target integrations: single cell clonal genotyping (n = 53 clones, left) vs bulk genome-wide integration assay (mean ± s.d., n = 3 biological replicates, right). c, On-target insertion copy number per clone for hifiDn29−dCas9 and Dn29−dCas9. Number (n) of clones is labeled above each bar. d, Correlation of attH1 insertion efficiency by ddPCR and rate of indel formation at attH1 by NGS. e, Genome-wide γ-H2AX staining and flow cytometry, measured 2 days after transfection with LSR and donor plasmids. S11A variants have an alanine mutation in the catalytic serine. 25 μM aphidicolin was included as a positive control. Asterisks and P values show t-test significance compared to no effector control. Exact P values are provided in Table S5. f, Viability of HEK293FT cells, 2 days after transfection with LSR and donor plasmids. Asterisks show t-test significance compared to no effector control. Exact P values are provided in Table S5. g, Schematic of off-target genome rearrangement outcomes. Recombination between attH1 and an attP-like pseudosite on the same chromosome could lead to intrachromosomal rearrangements resulting in either excision or inversion, depending on attachment site orientation. Recombination between attH1 and an attP-like pseudosite on different chromosomes would lead to a translocation. h, Quantification of interchromosomal translocations and intrachromosomal rearrangements after transfection with LSR and donor plasmids, with NGS baited upstream (left panel) or downstream (right panel) of attH1. i, Distance of intrachromosomal rearrangements to attH1, aggregating all rearrangement reads from the downstream-baited samples. Top panel shows distribution of all rearrangements across the entire chromosome. Bottom panel shows a magnified view of rearrangements within 1 Mb upstream and downstream of attH1. a, Integration efficiencies of Dn29 variants and dCas9 fusions at attH1, with and without cell cycle arrest by aphidicolin treatment. b, Donor plasmid transfection efficiency in HEK293FTs and hESCs (% mCherry+ cells). of n = 3 biological replicates, shown as dots. c, Specificity of Dn29 and variants in hESCs, measured as attH3 off-target integration efficiency by ddPCR. of n = 3 biological replicates, shown as dots. d, H1 hESC clones (n = 37) edited with goldDn29−dCas9: BFP expression (top) and genotyping (bottom). Plots show the knockdown quantification of n = 2−8 biological replicates (mean ± s.d. f, CRISPRi−BFP cassette expression in engineered hESCs after selection, pre- and post- differentiation into HPCs. of n = 3 biological replicates, shown as dots. h, Example gating strategy for HPCs. First, unstained cells are used as a negative control to set the Sytox Orange gate, indicating the boundary between live (BL2-A negative) and dead (BL2-A positive) cells (left). Within this population, the median fluorescence intensity (MFI) of the stained cell surface marker is used for determination of knockdown efficiency. a, Volcano plots representing differentially expressed genes compared to WT for engineered hESC and iPSC cell lines. Dotted lines indicate significance thresholds, set at Benjamini–Hochberg FDR–adjusted P value < 0.05 (two-tailed) and log2(fold change) > 1. n.s. c, Number of DEGs classified as essential genes. Essential genes were identified using the IMPC Essential Genes Data Portal, which collates five independent databases: IMPC mouse knockout data, DepMap Achilles CRISPR screens, FUSIL cell culture screens, gnomAD population constraint metrics, and ClinGen haploinsufficiency classifications70,71,89,90,91,92. The following thresholds were applied: IMPC lethal phenotypes, Achilles scores < −0.75, FUSIL lethality, gnomAD pLI > 0.9, and ClinGen sufficient/emerging haploinsufficiency evidence. Genes were classified by the number of databases they appeared essential in, with an additional category for population constraint (gnomAD LoF o/e < 0.35). d, Venn diagram showing overlap between DEGs identified in hifiDn29−dCas9 vs Cas9 editing at attH1 in H1 hESCs. a−c, Viability of primary T cells upon (a) unoptimized electroporation with Lonza pMaxGFP plasmid DNA, (b) transduction with ssAAV, and (c) transduction with scAAV. n = 2 biological replicates from separate blood donors. d, Integration efficiencies of Dn29 variants and dCas9 fusions at attH1 in primary human T cells using ssAAV donor. of n = 4 biological replicates, each originating from a different blood donor. e, Quantification of ITR sequences at attH1-donor junctions, indicating AAV genome capture versus LSR-mediated integration. f, Viability of human primary T cells one day after electroporation of donor and sgRNA plasmids using optimized plasmid electroporation protocol (Methods). of n = 2 biological replicates, each originating from a different blood donor. g, Viability and attH1 integration efficiency of superDn29−dCas9, delivered as mRNA, in primary T cells using standard 4.8-kb donor and sgRNA plasmids. of n = 2 biological replicates, each originating from a different blood donor. h, T cell viability four days after electroporation with all Dn29 variants ± dCas9 fusion, 5.8-kb CAR donor plasmid, and sgRNA plasmid. Samples correspond to those in Fig. i, Example gating strategy for cancer target-cell-killing assay. Nalm6 target cells are identified by CD19 expression and CAR-T cells are identified by tNGFR expression. j, Alignment of attH1-like pseudosites in human, marmoset, rhesus monkey, cynomolgus monkey, mouse and a sequence logo of the top 100 WT Dn29 pseudosites in HEK293FTs. k, Schematic of plasmid recombination assay for testing attH1-like pseudosites in HEK293FTs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Site-specific DNA insertion into the human genome with engineered recombinases. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251105050718.htm'>Nanotech makes cancer drug 20,000x stronger, without side effects</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 04:24:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In a major step toward improving cancer treatment, researchers at Northwestern University have redesigned the molecular structure of a widely used chemotherapy drug, making it far more soluble, potent, and less toxic to the body. The new therapy was tested in animals with acute myeloid leukemia (AML), a fast-growing and hard-to-treat blood cancer. Compared with the standard chemotherapy version, the SNA-based drug entered leukemia cells 12.5 times more efficiently, destroyed them up to 20,000 times more effectively, and slowed cancer progression 59-fold -- all without detectable side effects. With seven SNA-based treatments already in clinical testing, researchers believe this approach could pave the way for new vaccines and therapies for cancers, infections, neurodegenerative disorders, and autoimmune diseases. "If this translates to human patients, it's a really exciting advance. It would mean more effective chemotherapy, better response rates, and fewer side effects. That's always the goal with any sort of cancer treatment." Less than 1% dissolves in many biological fluids, meaning most of it never reaches its intended targets. "We all know that chemotherapy is often horribly toxic," Mirkin said. Cells readily recognize these structures and pull them inside. In this case, Mirkin's team chemically incorporated the chemotherapy molecules into the DNA strands themselves, creating a drug that cancer cells naturally absorb. "Most cells have scavenger receptors on their surfaces," Mirkin explained. "But myeloid cells overexpress these receptors, so there are even more of them. Instead of having to force their way into cells, SNAs are naturally taken up by these receptors." This structural redesign completely changed how 5-Fu interacted with leukemia cells, dramatically increasing its effectiveness. Because the SNAs selectively targeted AML cells, healthy tissues remained unharmed. Our structural nanomedicine preferentially seeks out the myeloid cells. The research team now plans to test the approach in a larger group of small animal models before advancing to larger animals and, ultimately, human clinical trials, once additional funding becomes available. Note: Content may be edited for style and length. Why This New “Super-Earth” Has Scientists So Excited About Alien Life NASA Balloon Detects Strange Signals Coming From Ice in Antarctica A Common Drug Might Help Reverse Genetic Hearing Loss, Scientists Say Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/11/251105050720.htm'>Scientists may have found how to reverse memory loss in aging brains</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-06 03:58:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Memory problems may not be an unavoidable part of getting older. New findings from Virginia Tech reveal that age-related memory loss stems from specific molecular changes in the brain, and that fine-tuning these processes can help restore memory function. In two complementary studies, Timothy Jarome, an associate professor in the College of Agriculture and Life Sciences' School of Animal Sciences, and his graduate students used advanced gene-editing tools to target these molecular changes and improve memory performance in older rats. Rats are commonly used as models for understanding how memory declines with age. "Memory loss affects more than a third of people over 70, and it's a major risk factor for Alzheimer's disease," said Jarome, who also serves in the School of Neuroscience. In the first study, published in Neuroscience and led by Jarome and doctoral student Yeeun Bae, the researchers investigated a molecular process called K63 polyubiquitination. This process acts like a tagging system that directs proteins inside brain cells on how to behave. The researchers discovered that aging alters this process in two important brain regions. In the hippocampus, which is responsible for forming and retrieving memories, K63 polyubiquitination levels rise with age. Using a gene-editing system called CRISPR-dCas13, the team lowered those levels and observed improved memory in older rats. When the researchers further reduced this activity, memory performance also improved. "In both regions, adjusting this one molecular process helped improve memory." The second study, published in the Brain Research Bulletin and led by Jarome and doctoral student Shannon Kincaid, focused on IGF2, a growth-factor gene known to support memory formation. "When that single copy starts to shut down with age, you lose its benefit." Using the CRISPR-dCas9 gene-editing system, they removed these tags and successfully reactivated IGF2. Older rats showed significant memory improvement once the gene was turned back on. Together, these studies reveal that memory loss during aging doesn't result from a single cause. Instead, it involves several molecular systems that change over time. "These projects represent the kind of graduate-led, collaborative research that defines our work," Jarome said. "Our students are deeply involved in designing experiments, analyzing data, and helping shape the scientific questions we pursue." "Everyone has some memory decline as they get older," Jarome added. "But when it becomes abnormal, the risk for Alzheimer's disease rises. What we're learning is that some of those changes happening at a molecular level can be corrected -- and that gives us a path forward to potential treatments." Why This New “Super-Earth” Has Scientists So Excited About Alien Life A Common Drug Might Help Reverse Genetic Hearing Loss, Scientists Say Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            