
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-21</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a64514733/heart-transplant-beating/'>Doctors Completed a Heart Transplant With an Incredible Twist: It Never Stopped Beating</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In the U.S. alone, nearly 3,800 people are on the waiting list for heart transplant, but as Yale Medicine notes, demand for healthy hearts currently outstrips supply by a significant margin, leading many people to suffer for months with failing hearts. To help ease the stress on this biological ‘supply chain,' scientists have turned to new methods to ease the suffering of those waiting for hearts while also ensuring that surgical success rates climb ever upward. And now, a medical team at the National Taiwan University Hospital (NTUH) claims to have performed the very first “beating heart” transplant, meaning that the donor heart never stopped beating during its removal, connection to support system, and surgical transplantation. Previously, teams from Stanford University (the university where the first U.S. heart transplant was performed in 1968) had also published studies claiming that they performed beating heart transplants back in 2023 and 2024. But the Taipei Times reports that those procedures still required very brief ischemic time between removal and being hooked up to a “heart in a box” support system. “We wanted to perform a heart transplant without any ischemic time so that the heart wouldn't have to stop, and we could also avoid injury [to heart tissue] that typically occurs after reperfusion,” Chi Nai-hsin, an attending physician at NTUH, said during a press conference in Taipei. Post-transplant—during which the heart continued to beat during the entire process—a 49-year old who was experiencing end-stage heart failure is alive and well. With this technique, doctors and scientists hope to see less tissue damage during transplantation, which in turn should tick up the success rate of heart transplants overall. Humans Could Grow New Teeth in Just a Few Years We Totally Missed a Big Part of Our Immune System Humans May Be Able to Grow New Teeth in 6 Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/trump-cuts-threaten-key-noaa-work-to-improve-weather-forecasts-and-monitor/'>Proposed Trump Cuts to NOAA Threaten Hurricane Hunters and Toxic Algal Bloom Monitoring</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 17:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Proposed Trump Cuts to NOAA Threaten Hurricane Hunters and Toxic Algal Bloom Monitoring The Trump administration has proposed gutting NOAA's cooperative institutes, which study everything from improving lifesaving weather forecasts to monitoring fish stocks Commander Mark Nelson, with NOAA, climbs the steps to one of the hurricane hunter planes after a press conference at MacDill Air Force base on Thursday, May 22, 2008. CLIMATEWIRE | Researchers in Oklahoma are hard at work on a new lifesaving weather forecasting system. In Michigan, they're keeping tabs on toxic algae blooms. In Florida, they're studying tropical cyclones by flying into the hearts of hurricanes. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. A new proposal from the White House Office of Management and Budget would dramatically reorganize NOAA and gut most of its climate research programs in fiscal 2026. Part of that plan includes terminating funding for NOAA's cooperative institutes and its 10 laboratories, which are heavily staffed by CI researchers. The plan, presented last week in an OMB document known as a “passback” memorandum, is technically still hypothetical. But the bill provides little guidance for agencies on how exactly they must use their funds. “That's the authority Congress afforded them by not articulating more detail in its agency budgets.” And it's unclear whether this direction would legally sidestep Congress' authority to direct the appropriation of funds. Meanwhile, some CIs across the country have not yet received any of their 2025 funds. Meanwhile, Commerce Secretary Howard Lutnick — head of the agency that houses NOAA — is personally reviewing all funding commitments above $100,000. “The money is very slow in coming, and a number of institutes are at great risk of not having the funding after a couple months from now,” Abdalati said. Even if Congress restores funding for 2026, cuts and layoffs in the near term would be devastating, he added. Many staffers likely would seek new jobs, taking their knowledge and experience with them. “Once a certain amount of damage is done, it's not recoverable,” Abdalati said. A harmful algal bloom sparked the Toledo water crisis of 2014, in which 400,000 residents in and around the Ohio city had no safe drinking water for two days. Ty Wright for The Washington Post via Getty Images But because of the ongoing funding delays, “we're looking at having to lay off a substantial number of our workers in the next few months,” said CIGLR director Gregory Dick. And it's possible the institute will have to halt its algal monitoring program. “One of my big fears is that we'll be more vulnerable to such incidents,” Dick said, adding that the program “seems like it's in limbo — it's complete uncertainty.” The cooperative institutes are one part of NOAA's broader research ecosystem and just one of many proposed cuts across the department. “The CIs are 50 percent of everything we do in research,” said Craig McLean, NOAA's former top scientist. “They are of equal vitality and importance to the NOAA mission as every NOAA scientist — many of whom have come from the CIs.” Each agreement is awarded on a five-year basis, with the potential to renew for another five years. After that, universities must compete again for a new award. Still, many cooperative institutes have been around for decades — CIRES, the oldest and largest, was established in 1967. Many involve multiple university partners and employ dozens or hundreds of staff. And many maintain long-standing data collection programs with major impacts on human societies. It's also active in ocean exploration, mapping parts of the seabed where methane reserves or critical minerals may be abundant. [The cuts represent a] "complete sabotaging of American weather forecasting. These studies help keep the U.S. competitive with other global science leaders, said CIMERS director Francis Chan. “There's a new science race going on,” he said. Other CIs help improve the forecasting tools used by NOAA's own National Weather Service. Scientists from the Cooperative Institute for Marine and Atmospheric Studies are key members of NOAA's famed Hurricane Hunter missions, which fly specialized data-collecting aircraft through tropical cyclones. Meanwhile, scientists at the Cooperative Institute for Severe and High-Impact Weather Research and Operations (CIWRO), are developing products to help meteorologists spot dangerous weather events with more advance warning. One of these is Warn-on-Forecast, an experimental system designed to rapidly incorporate radar and satellite observations into a high-resolution model, producing updated forecasts about every 15 minutes. As twisters whirled across the central U.S. last month, amid an outbreak that killed dozens in the Southeast and Midwest, Warn-on-Forecast predictions helped accurately predict a storm track in the Missouri Ozarks with about two hours of lead time, according to CIWRO's director, Greg McFarquhar. The forecast, combined with other data, prompted National Weather Service staff to contact emergency managers on the evening of March 14 and warn them that long-track tornadoes may be forming. NWS followed up shortly afterward with a Special Weather Statement, narrowing down the tornado tracks to nearby Carter and Ripley counties. With funding delays dragging on and existential cuts looming, scientists say these research projects are all in jeopardy. Larger institutes like CIRES said they might continue to exist in a diminished form — but the loss of NOAA resources would take a huge toll. “We wouldn't be as robust,” said Abdalati, the CIRES director. “And honestly it would be, I think, a big loss to the American people — because we do things that matter, that are important.” But CI scientists note their projects delve far beyond climate change research. And many have implications for the economy, national security and competition with countries such as China — priorities the Trump administration has claimed to support. “Are people making decisions because they don't have the full picture of what science is doing? If that's the case, we're open to providing information.” The American Meteorological Society warned in a statement that eliminating NOAA's research arm would have “unknown — yet almost certainly disastrous — consequences for public safety and economic health.” The cuts represent a "complete sabotaging of American weather forecasting,” said Marc Alessi, a science fellow with the nonprofit advocacy organization Union of Concerned Scientists. “It would totally change the game in terms of our prediction.” Nine Democratic representatives from New Jersey submitted a letter last week to Lutnick decrying the proposed cuts, which they argued would endanger their state and its nearly 1,800 miles of coastline. They expressed particular concern about the proposed elimination of NOAA's Geophysical Fluid Dynamics Laboratory in Princeton, New Jersey. “Without their work, Americans will not receive accurate weather or tidal predictions, impacting our safety, economy and national security,” the letter stated. Democratic Sen. Michael Bennet of Colorado said in a statement to E&E News that worsening droughts and wildfires across the western United States mean that the "work our scientists and civil servants do at NOAA is essential to U.S. national security and the personal safety and daily lives of Americans.” Her work has appeared in The Washington Post, Popular Science, Men's Journal and others.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01650-9'>Tumour interstitial fluid-enriched phosphoethanolamine suppresses T cell function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 16:37:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nutrient stress represents an important barrier for anti-tumour immunity, and tumour interstitial fluid often contains metabolites that hinder immune function. However, it is difficult to isolate the effects of tumour nutrient stress from other suppressive factors. Thus, we used a chemically defined cell culture medium based on the metabolomic profile of tumour interstitial fluid: tumour interstitial fluid medium (TIFM). Culture of CD8+ T cells in TIFM limited cell expansion and impaired CD8+ T cell effector functions upon restimulation, suggesting that tumour nutrient stress alone is sufficient to drive T cell dysfunction. We identified phosphoethanolamine (pEtn), a phospholipid intermediate, as a driver of T cell dysfunction. pEtn dampened T cell receptor signalling by depleting T cells of diacylglycerol required for T cell receptor signal transduction. The reduction of pEtn accumulation in tumours improved intratumoural T cell function and tumour control, suggesting that pEtn accumulation plays a dominant role in immunosuppression in the tumour microenvironment. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout RNA sequencing data that support the findings of this study have been deposited in the Gene Expression Omnibus (GEO) under accession code GSE235214. Metabolomics and lipidomics datasets have been deposited and are available via Metabolomics Workbench under study number ST003664 at https://dev.metabolomicsworkbench.org:22222/data/DRCCMetadata.php?Mode=Study&StudyID=ST003664&Access=IqaQ1644. 5a were reanalysed from ref. 6j were reanalysed from ref. Source data are provided with this paper. All other data supporting the findings of this study are available from the corresponding author on reasonable request. Immune infiltration in human tumors: a prognostic factor that should not be ignored. Dahlin, A. M. et al. Colorectal cancer prognosis depends on T-cell infiltration and molecular characteristics of the tumor. & Wolchok, J. D. Cancer immunotherapy using checkpoint blockade. Ai, L. et al. Research status and outlook of PD-1/PD-L1 inhibitors for cancer therapy. Sambi, M., Bagheri, L. & Szewczuk, M. R. Current challenges in cancer immunotherapy: multimodal approaches to improve efficacy and patient response rates. Anderson, K. G., Stromnes, I. M. & Greenberg, P. D. Obstacles posed by the tumor microenvironment to T cell activity: a case for synergistic therapies. Verma, N. K. et al. Obstacles for T-lymphocytes in the tumour microenvironment: therapeutic challenges, advances and opportunities beyond immune checkpoint. Defining ‘T cell exhaustion'. Scharping, N. E. et al. Mitochondrial stress induced by continuous stimulation under hypoxia rapidly drives T cell exhaustion. Xia, A., Zhang, Y., Xu, J., Yin, T. & Lu, X. J. T cell dysfunction in cancer immunity and immunotherapy. Scharping, N. E. & Delgoffe, G. M. Tumor microenvironment metabolism: a new checkpoint for anti-tumor immunity. & Delgoffe, G. M. Fighting in a wasteland: deleterious metabolites and antitumor immunity. Wilfahrt, D. & Delgoffe, G. M. Metabolic waypoints during T cell differentiation. Buck, M. D., O'Sullivan, D. & Pearce, E. L. T cell metabolism drives immunity. Chapman, N. M., Boothby, M. R. & Chi, H. Metabolic coordination of T cell quiescence and activation. The hallmarks of cancer metabolism: still emerging. Cell programmed nutrient partitioning in the tumor microenvironment. Grzywa, T. M. et al. Myeloid cell-derived arginase in cancer immune response. Platten, M., Wick, W. & Van Den Eynde, B. J. Tryptophan catabolism in cancer: beyond IDO and tryptophan depletion. Hezaveh, K. et al. Tryptophan-derived microbial metabolites activate the aryl hydrocarbon receptor in tumor-associated macrophages to suppress anti-tumor immunity. Schaaf, M. B., Garg, A. D. & Agostinis, P. Defining the role of the tumor vasculature in antitumor immunity and immunotherapy. Leone, R. D. et al. Glutamine blockade induces divergent metabolic programs to overcome tumor immune evasion. Targeting metabolism in cancer cells and the tumour microenvironment for cancer therapy. Sullivan, M. R. et al. Quantification of microenvironmental metabolites in murine cancers reveals determinants of tumor nutrient availability. A. et al. Pancreatic tumors exhibit myeloid-driven amino acid stress and upregulate arginine biosynthesis. Raynor, J. L. & Chi, H. Nutrients: signal 4 in T cell immunity. Rodriguez, P. C., Ochoa, A. C. & Al-Khami, A. A. Arginine metabolism in myeloid cells shapes innate and adaptive immunity. Rodriguez, P. C. et al. l-arginine consumption by macrophages modulates the expression of CD3 zeta chain in T lymphocytes. Crump, N. T. et al. Chromatin accessibility governs the differential response of cancer and T cells to arginine starvation. Geiger, R. et al. l-Arginine modulates T cell metabolism and enhances survival and anti-tumor activity. Abbott, K. L. et al. Metabolite profiling of human renal cell carcinoma reveals tissue-origin dominance in nutrient availability. & Rylett, A. Accumulation of phosphoethanolamine in the livers of rats injected with hepatocarcinogens. Osawa, T. et al. Phosphoethanolamine accumulation protects cancer cells under glutamine starvation through downregulation of PCYT2. Courtney, A. H., Lo, W. L. & Weiss, A. TCR signaling: mechanisms of initiation and propagation. Leney-Greene, M. A., Boddapati, A. K., Su, H. C., Cantor, J. R. & Lenardo, M. J. Human plasma-like medium improves T lymphocyte activation. MacPherson, S. et al. Clinically relevant T cell expansion media activate distinct metabolic programs uncoupled from cellular function. Tighanimine, K. et al. A homoeostatic switch causing glycerol-3-phosphate and phosphoethanolamine accumulation triggers senescence by rewiring lipid metabolism. Saab, R. Senescence and pre-malignancy: how do tumors progress? & Demaria, M. Senescent cells in cancer therapy: friends or foes? Deng, Y., Wu, L. Ding, Q. & Yu, H. AGXT2L1 is downregulated in carcinomas of the digestive system. Ding, Q. et al. AGXT2L1 is down-regulated in heptocellular carcinoma and associated with abnormal lipogenesis. Vodnala, S. K. et al. T cell stemness and dysfunction in tumors are triggered by a common mechanism. The costimulatory activity of Tim-3 requires Akt and MAPK signaling and its recruitment to the immune synapse. Matyash, V., Liebisch, G., Kurzchalia, T. V., Shevchenko, A. & Schwudke, D. Lipid extraction by methyl-tert-butyl ether for high-throughput lipidomics. The Skyline ecosystem: Informatics for quantitative mass spectrometry proteomics. Peralta, R. M. et al. Dysfunction of exhausted T cells is enforced by MCT11-mediated lactate metabolism. Workman for the assistance with synapse imaging, S. G. Wendell and S. J. Mullett for the help with lipidomics analysis and L. C. Hermida for the help with RNA sequencing analysis. A.M. was supported by the Brinson Foundation and the Cancer Research Foundation. We thank L. Becker for sharing interstitial fluid samples from B16F10 murine tumours. We also thank M. V. Heiden and M. Sullivan for sharing interstitial fluid samples from BrafCA; PTENfl/fl; Tyr-CreER and MMTV-Cre; Brca1fl/fl; Trp53+/− tumour-bearing mice. We thank funding sources for this work, including the Cancer Research Institute Lloyd J. Old STAR Award (CRI3447) awarded to G.M.D., Mark Foundation for Cancer Research's Emerging Leader Award (19-040-ELA) awarded to G.M.D., The Pittsburgh Foundation (MR2023-134140) awarded to G.M.D., NIAID R01AI171483 awarded to G.M.D., NIAID R01AI166598 awarded to G.M.D., NCI R01CA277473 awarded to G.M.D., T32AI089443 awarded to D.W., L70CA294410 awarded to D.W., Brinson Foundation Junior Investigator Award to A.M. and Cancer Research Foundation Young Investigator Award to A.M. These authors contributed equally: Yupeng Wang, Drew Wilfahrt. These authors jointly supervised this work: Alexander Muir, Greg M. Delgoffe. Tsinghua Medical School, Beijing, China Department of Immunology, University of Pittsburgh, Pittsburgh, PA, USA Drew Wilfahrt, Konstantinos Lontos, Benjamin Cameron, Bingxian Xie, Ronal M. Peralta, Emerson R. Schoedel, William G. Gunn, Dayana B. Rivadeneira & Greg M. Delgoffe Tumor Microenvironment Center, UPMC Hillman Cancer Center, Pittsburgh, PA, USA Drew Wilfahrt, Bingxian Xie, Ronal M. Peralta, Dayana B. Rivadeneira & Greg M. Delgoffe Ben May Department for Cancer Research, University of Chicago, Chicago, IL, USA Patrick Jonker, Chufan Cai & Alexander Muir Metabolomics Platform, Comprehensive Cancer Center, University of Chicago, Chicago, IL, USA Roya AminiTabrizi & Hardik Shah You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed and performed the majority of experiments, analysed data and wrote the manuscript. They are listed alphabetically by last name on the manuscript, and they provided equal contribution. conducted and analysed lipidomics analysis. performed and analysed metabolomics. performed cell culture and flow cytometry experiments. generated overexpressing tumour cells and aided in tumour experiments. performed and analysed microscopy experiments. performed some of the cell culture studies. measured tumours blinded and generated tumour growth curves. collected and processed data. assisted with tumour growth and tumour-lymphocyte analysis. jointly oversaw and directed the research, analysed data, obtained research funding, and wrote and edited the manuscript. Correspondence to Alexander Muir or Greg M. Delgoffe. declares competing financial interests and has submitted patents targeting exhausted T cells that are licensed or pending and is entitled to a share in net income generated from licensing of these patent rights for commercial development. consults for and/or is on the scientific advisory board of BlueSphere Bio, Novasenta, Xyphos and Kalivir Immunotherapeutics; has grants from Novasenta, Astellas, RemplirBio and Kalivir; and owns stock in Novasenta, BlueSphere Bio and RemplirBio. The other authors declare no competing interests. Nature Cell Biology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. A) %PD1hi cells from Fig. Data are ±SEM from four independent experiments (n = 8 mice/group (p-value 0.0018). B-E) Cells from Fig. 1d-f analyzed for B) Normalized IFNγ MFI±SEM C) Normalized TNF MFI ±SEM D) %Granzyme Bhi cells and E) %IL-2hi cells. Statistics were calculated using a two-tailed paired t-test. B) p-value F) Timeline Schematic of experiments in G-J G) Normalized PD-1 MFI (mean fluorescence intensity) ±SEM from four independent experiments (n = 8-9 mice/group, p-value RPMI vs. T3 p = 0.0010, T1 vs. T3 p = 0.0061). H) Day 7 OT-I CD8+ T cells were analyzed for IFNγ+TNF+ co-expression after 6 hours αCD3/αCD28 restimulation. Data quantified show %IFNγ+TNF+ events among Live CD8+ T cells ±SEM from four independent experiments (n = 8 mice/group). I-J) Day 7 OT-I T cells were analyzed for expression of I) IL-2 or J) Granzyme B after 6 hours of αCD3/αCD28 restimulation. Data quantified show normalized MFI ±SEM from four independent experiments (n = 8-9 mice/group, J) p-value RPMI vs. T3 p < 0.0001, RPMI vs. T3, p = 0.0026). K) Day 7 OT-I CD8+ T cells were co-cultured with B16-OVA cells at a 1:1 effector:target ratio in an Agilent xCELLigence RTCA DP system to monitor target cell killing. XY plot on the left shows %Cytolysis ±SEM, and Area under the curve values are quantified from three independent experiments (n = 5-6 mice/group, p values RPMI vs. T1 p = 0.9897, RPMI vs. T3, p = 0.0179, T1 vs. T3, p = 0.0224). Statistical analysis for all indicated comparisons in F-K) was determined using a one-way ANOVA with Tukey's multiple comparisons test. L-O) Cells from Fig. 1h-j were restimulated with αCD3/αCD28 in RPMI or TIFM for 6 hours. After restimulation, cells were analyzed for L) Normalized IFNγ MFI of Live IFN+ CD8+ Cells, M) Normalized TNF MFI of Live TNF+CD8+ cells N) % of Granzyme Bhi cells and J) % of IL-2hi cells. (n = 8 mice/group from three independent experiments). For L-O), statistics were calculated using a two-tailed paired t-test (p-values L) p = 0.0268 M) p < 0.0001, N) p < 0.0001, O) p = 0.0006) Source numerical data are available in source data. Fold change of the metabolite concentrations measured in the TIF relative to those in the plasma of mice bearing PDAC tumor, highlighted metabolites are marked by color to indicate metabolites that are depleted (red) and enriched (blue) in the tumor. Source numerical data are available in source data. A) Data related to Fig. 2c were analyzed for % of PD-1hi cells from three independent experiments (n = 5 mice/group, RPMI vs. TIFM p = 0.0499, RPMI vs. TIFM+Arg p = 0.190, TIFM vs. TIFM+Arg p = 0.0954) B-E) Data from Fig. 2d-f were analyzed for B) Normalized IFNγ MFI of Live IFN+ CD8+ Cells, C) Normalized TNF MFI of Live TNF+CD8+ cells D) % of IL-2hi cells and E) % of Granzyme Bhi cells. F-I) Data from Fig. 3i-k were analyzed for F) Normalized IFNγ MFI of Live IFN+ CD8+ Cells, G) Normalized TNF MFI of Live TNF+CD8+ cells H) % of IL-2hi cells and I) % of Granzyme Bhi cells. Statistical analysis for all indicated comparisons in this figure was determined using a one-way ANOVA with Tukey's multiple comparisons test. Source numerical data are available in source data. A-C) OT-I CD8 + T cells were activated and cultured as in Fig. 1a) in RPMI, TIFM, or TIFM formulated without pEtn and supplemented with RPMI levels of Arginine. At the end of culture, cells were restimulated and analyzed for A) Granzyme B MFI B) IL-2 MFI and C) %IFNγ+TNF+ cells. Bar plots show quantified normalized MFI or %+ ±SEM on the left of each flow cytogram, data are from 3 independent experiments (n = 6 mice/group, p-values A) RPMI vs. TIFM p = 0.0014, TIFM vs. TIFM+Arg-pEtn p = 0.0458, RPMI vs. TIFM-pEtn+Arg p = 0.5720 B) RPMI vs. TIFM p = 0.0389, TIFM vs. TIFM+Arg-pEtn p = 0.0315 RPMI vs. TIFM-pEtn+Arg p = 0.9741 C) RPMI vs. TIFM p = 0.0019, TIFM vs. TIFM+Arg-pEtn p = 0.0210, RPMI vs. TIFM-pEtn+Arg p = 0.1116). Statistical analysis for all indicated comparisons in A-C) was determined using a one-way analysis of variance (ANOVA) with Tukey's multiple comparisons test. Source numerical data are available in source data. A) Data analyzed from previous reports31. Previously generated data pEtn concentration (µM) of human patient plasma or renal cell tumor tumor interstitial fluid (TIF) (n = 27-46 samples/group, p < 0.0001). B) Choline and phosphocholine levels in the plasma and the tumor interstitial fluid of mice bearing PDAC tumors. Statistical significance for indicated comparison quantified using a two-tailed unpaired t-test. Source numerical data are available in source data. Error bars in both graphs represent SEM. A) PCA analysis of lipidomics performed in 4I-O and S6B B) Heatmap showing Log2FC over mean for each lipid measured in lipidomics described in Fig. C-E) Lipidomics analysis of activated OT-I T cells cultured in either RPMI or RPMI supplemented with pEtn. C) Ratio of phosphatidylethanolamine (PE) to Phosphatidylcholine (PC) (p-value p = 0.0142). Boxes range from 25th to 75th percentiles, while the whiskers extend to the minimum and maximum value for each dataset. Fold change of D) PE species and E) PC species in pEtn treated OT-I T F-H) Cells treated with F) pEtn, G) Choline, or H) pCholine at TIFM-levels for 3 or 5 days and analyzed via lipidomics as described in Fig. Each volcano plot shows Log2 fold change of treated cells over RPMI-treated controls on the x axis, and Log10 p-value on the y axis. Red dots indicate DAG species, tan dots are PE species, blue dots are PC species, and purple dots are sphingomyelins. I) DAG levels in OT-I T cells that were cultured as described in 4I-O). Normalized DAG MFI is quantified on the left from six independent experiments (n = 18 mice/group, RPMI vs. pEtn p = 0.0037, RPMI vs. Chol p = 0.8314, RPMI vs. pChol p = 0.9923) and statistical analysis for all indicated comparisons in was determined using a one-way analysis of variance (ANOVA) with Tukey's multiple comparisons test. J) RNA expression of choline and choline-like transporters in PD1+Tim3+ tumor-infiltrating CD8 + T cells. Data retrieved from previous report46. ND = transcripts not detected. K-M) Representative flow plots for quantification shown in Fig. N) Western blot for Pcyt2 and β-Actin on Day 7 WT and Pcyt2 KO CD8 T cells from Fig. 4p-s. (n = 4 mice/group from 3 independent experiments) Statistical significance for indicated comparison in ED6C) quantified using a two-tailed paired t-test (p = 0.0098). Proteins were probed across multiple independent blots that were loaded with the same original proteins. Source numerical data are available in source data. A) Quantification of metabolites in TCA cycle, oxidative phosphorylation, glycolysis, glutaminolysis and pentose phosphate pathway of SIINFEKL-activated OT-I T cells in RPMI+pEtn versus RPMI culture B) Quantification of lipid metabolites of SIINFEKL-activated OT-I T cells in RPMI+pEtn versus RPMI culture as shown in Fig. C) Volcano plot of RNA-Sequencing of activated OT-I T cells cultured in RPMI or RPMI +pEtn. D) Gene set enrichment assay of RNA-Seq data shown in C) of activated OT-I cells cultured in either RPMI or RPMI+pEtn. (E-G) OT-I T cells activated by SIINFEKL O/N were cultured in RPMI for 2 days and then in either RPMI or RPMI+pEtn for 5 days as described in Fig. Then the cells were restimulated by either α-CD3 and α-CD28 or PMA/Ionomycin for 6 hours in RPMI. After restimulation, cells were analyzed for E) IFNγ and TNF co-expression F) Granzyme B expression, and G) IL-2 expression by flow cytometry. Normalized MFI ±SEM is quantified for Granzyme B and IL-2, while %IFNγ+TNF+ cells ±SEM are quantified on the left of each plot from three independent experiments (n = 6 mice/group, p-values left to right E) p = 0.0002, p = 0.6170, F) p = 0.0046, p = 0.0267, G) p = 0.0007, p = 0.2566). A) OT-I CD8+ T cells were treated with RPMI + pEtn for 5 days as described in Fig. 3a then labeled with Cholera Toxin Subunit B-GFP. Sum fluorescent intensity of GFP is quantified on the left. (n = 70-80 cells/group) Statistical significance for indicated comparison was calculated using a two-tailed unpaired t-test (p < 0.0001). B) OT-I CD8+ T cells were treated with RPMI + pEtn for 5 days as described in Fig. On day 7, cell lysates were prepared, and western blot was performed with antibodies specific for Cdk6, Cyclin D3, phosphorylated and total Chk1 (pChk, and tChk, respectively). Representative blots are shown on the left. Quantification on the right is from 3 independent experiments (n = 6 mice/group, p values left to right, CDK6 p = 0.4213, Cyclin D3 p = 0.5531, p-Chk1 p = 0.5491) Proteins were probed across multiple, independent blots that were loaded with the same original proteins. C-E) OT-I T cells were treated as in A), then on Day 7, cells were treated with protein transport inhibitor cocktail for 6 hours without restimulation, and then C) Granzyme B MFI, p = 0.4806, D) IL-2 MFI, p = 0.9536, and E) %IFNγ+TNF+, p = 0.6104, was measured by flow cytometry. All data in C-E) are from 3 independent experiments (n = 6 mice/group). Statistical significance for indicated comparisons was calculated using a two-tailed paired t-test. Source numerical data and unprocessed blots are available in source data. A) DAG expression of the tumor infiltrating transferred Pmel T cells from the tumors in Fig. Bar graph on the left shows normalized DAG MFI ±SEM from three independent experiments (n = 7 mice/group, p = 0.0729) B) Western blot confirmation for Pcty2 and Actin in B16 EV and B16-Pcyt2 melanoma cell lines. (Data show n = 3 lysates from 1 independent experiment) Proteins were probed across multiple independent blots that were loaded with the same original proteins. C) PD1 and Tim3 co-expression of endogenous CD8+ T cells from tumors in Fig. 6i, Data quantified on the left show % PD1hiTim3hi from three independent experiments (n = 14-15 mice/group, p = 0.8162). Statistical significance for all indicated comparisons in this figure was calculated using a two-tailed unpaired t-test. D) Data from Fig. 6k were analyzed for Normalized TNF MFI of Live CD8+ TNF+ cells. Data are quantified from three independent experiments (n = 9-10 mice/group, p = 0.1188) E-F) pmel T cells from tumor-draining lymph nodes were restimulated with gp100 peptide for 1 hour and analyzed by flow cytometry for E) % p-S6+ (n = 8-10 mice/group from 3 independent experiments p = 0.8470) and F) % p-ERK+ (n = 8-10 mice/group from 3 independent experiments p = 0.7162) Statistical significance for all indicated comparisons in this figure was calculated using a two-tailed unpaired t-test. Source numerical data and unprocessed blots are available in source data. Gating strategy for CD8+ T cells isolated from cell culture: cells were collected on day 7 of In vitro RPMI culture. The flow cytograms show the representative gating strategy of CD8+ T cells used throughout all in vitro experiments. Source data for all figures and extended data figures, with each figure as its own tab. Unprocessed blot images for all figures (labelled on each page). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Wang, Y., Wilfahrt, D., Jonker, P. et al. Tumour interstitial fluid-enriched phosphoethanolamine suppresses T cell function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03616-8'>Blood pressure reduction and all-cause dementia in people with uncontrolled hypertension: an open-label, blinded-endpoint, cluster-randomized trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 15:02:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Dementia is a leading cause of death and disability worldwide. Here we tested the effectiveness of blood pressure (BP) reduction on the risk of all-cause dementia among 33,995 individuals aged ≥40 years with uncontrolled hypertension in rural China. We randomly assigned 163 villages to a non-physician community healthcare provider-led intervention and 163 villages to usual care. In the intervention group, trained non-physician community healthcare providers initiated and titrated antihypertensive medications according to a simple stepped-care protocol to achieve a systolic BP goal of <130 mm Hg and a diastolic BP goal of <80 mm Hg, with supervision from primary care physicians. Over 48 months, the net reduction in systolic BP was 22.0 mm Hg (95% confidence interval (CI) 20.6 to 23.4; P < 0.0001) and that in diastolic BP was 9.3 mm Hg (95% CI 8.7 to 10.0; P < 0.0001) in the intervention group compared to usual care. The primary outcome of all-cause dementia was significantly lower in the intervention group than in the usual care group (risk ratio: 0.85; 95% CI 0.76 to 0.95; P = 0.0035). Additionally, serious adverse events occurred less frequently in the intervention group (risk ratio: 0.94; 95% CI 0.91 to 0.98; P = 0.0006). This cluster-randomized trial indicates that intensive BP reduction is effective in lowering the risk of all-cause dementia in patients with hypertension. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Since patients have not explicitly consented to data sharing, we are not allowed to post individual participant data in a public data repository for legal reasons. However, researchers with a valuable research question can request study data from the corresponding authors. If the proposal is approved by the CRHCP Study Steering Committee, deidentified individual data may be shared after consultation with the data protection officers and legal representatives of the participating institutions and after signing a data-sharing agreement. All data sharing will abide by the rules and policies defined by the sponsor, relevant ethics committees, and government laws and regulations. A response to requests for data access can be expected within 4 weeks. Source data are provided with this paper. The modified Poisson regression analysis and generalized estimating equation linear model were implemented using Proc GENMOD in SAS v.9.4. GBD 2019 Dementia Forecasting Collaborators. Estimation of the global prevalence of dementia in 2019 and forecasted prevalence in 2050: an analysis for the Global Burden of Disease Study 2019. Lancet Public Health 7, e105–e125 (2022). Ferri, C. P. & Jacob, K. S. Dementia in low-income and middle-income countries: different realities mandate tailored solutions. GBD 2016 Dementia Collaborators. Global, regional, and national burden of Alzheimer's disease and other dementias, 1990–2016: a systematic analysis for the Global Burden of Disease Study 2016. Livingston, G. et al. Dementia prevention, intervention, and care. Mukadam, N., Sommerlad, A., Huntley, J. & Livingston, G. Population attributable fractions for risk factors for dementia in low-income and middle-income countries: an analysis using cross-sectional survey data. Mills, K. T. et al. Global disparities of hypertension prevalence and control: a systematic analysis of population-based studies from 90 countries. The global epidemiology of hypertension. Lennon, M. J. et al. Use of antihypertensives, blood pressure, and estimated risk of dementia in late life: an individual participant data meta-analysis. van Dalen, J. W. et al. Association of systolic blood pressure with dementia risk and the role of age, u-shaped associations, and mortality. Forette, F. et al. Prevention of dementia in randomised double-blind placebo-controlled Systolic Hypertension in Europe (Syst-Eur) trial. Peters, R. et al. Incident dementia and blood pressure lowering in the Hypertension in the Very Elderly Trial cognitive function assessment (HYVET-COG): a double-blind, placebo controlled trial. Tzourio, C. et al. Effects of blood pressure lowering with perindopril and indapamide therapy on dementia and cognitive decline in participants with cerebrovascular disease. Williamson, J. D. et al. Effect of intensive vs standard blood pressure control on probable dementia: a randomized clinical trial. Peters, R. et al. Blood pressure lowering and prevention of dementia: an individual patient data meta-analysis. Hughes, D. et al. Association of blood pressure lowering with incident dementia or cognitive impairment: a systematic review and meta-analysis. He, J. et al. Effectiveness of a nonphysician community healthcare provider-led intensive blood pressure intervention versus usual care on cardiovascular disease (CRHCP): an open-label, blinded-endpoint, cluster-randomised trial. Sun, Y. et al. A village doctor-led multifaceted intervention for blood pressure control in rural China: an open, cluster randomised trial. Whelton, P. K. et al. ACC/AHA/AAPA/ABC/ACPM/AGS/APhA/ASH/ASPC/NMA/PCNA guideline for the prevention, detection, evaluation, and management of high blood pressure in adults: a report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines. Joint Committee for Guideline Revision. 2018 Chinese guidelines for prevention and treatment of hypertension: a report of the Revision Committee of Chinese guidelines for prevention and treatment of hypertension. Al-Makki, A. et al. Hypertension pharmacological treatment in adults: a world health organization guideline executive summary. Lu, J. et al. Prevalence, awareness, treatment, and control of hypertension in China: data from 1.7 million adults in a population-based screening study (China PEACE Million Persons Project). den Brok, M. et al. Antihypertensive medication classes and the risk of dementia: a systematic review and network meta-analysis. Peters, R. et al. Investigation of antihypertensive class, dementia, and cognitive decline: a meta-analysis. Baggeroer, C. E., Cambronero, F. E., Savan, N. A., Jefferson, A. L. & Santisteban, M. M. Basic mechanisms of brain injury and cognitive decline in hypertension. Goff, D. C. Jr et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. Rationale and design of a cluster randomized trial of a village doctor-led intervention on hypertension control in China. Hu, D. et al. Development of village doctors in China: financial compensation and health system support. A practical method for grading the cognitive state of participants for the clinician. Pfeffer, R. I., Kurosaki, T. T., Harrah, C. H., Chance, J. M. & Filos, S. Measurement of functional activities in older adults in the community. González, D. A., Gonzales, M. M., Resch, Z. J., Sullivan, A. C. & Soble, J. R. Comprehensive evaluation of the Functional Activities Questionnaire (FAQ) and its reliability and validity. Galvin, J. E. The quick dementia rating system (QDRS): a rapid dementia staging tool. Pang, T. et al. Validation of the informant quick dementia rating system (QDRS) among older adults in singapore. Stewart, P. V. et al. Validation and extension of the quick dementia rating system (QDRS). Katzman, R. et al. A Chinese version of the Mini-Mental State Examination; impact of illiteracy in a shanghai dementia survey. Li, H., Jia, J. & Yang, Z. Mini-Mental State Examination in elderly Chinese: a population-based normative study. Yin, L. et al. The power of the functional activities questionnaire for screening dementia in rural-dwelling older adults at high-risk of cognitive impairment. Associations among vascular risk factors, neuroimaging biomarkers, and cognition: preliminary analyses from the Multi-Ethnic Study of Atherosclerosis (MESA). McKhann, G. M. et al. The diagnosis of dementia due to Alzheimer's disease: recommendations from the National Institute on Aging-Alzheimer's Association workgroups on diagnostic guidelines for Alzheimer's disease. The diagnosis of mild cognitive impairment due to Alzheimer's disease: recommendations from the National Institute on Aging-Alzheimer's Association workgroups on diagnostic guidelines for Alzheimer's disease. Prevalence, risk factors, and management of dementia and mild cognitive impairment in adults aged 60 years or older in China: a cross-sectional study. Lancet Public Health 5, e661–e671 (2020). Jia, L. et al. Dementia in China: epidemiology, clinical management, and research advances. & Klar, N. Design and Analysis of Cluster Randomization Trials in Health Research (Arnold, 2000). Zou, G. A modified poisson regression approach to prospective studies with binary data. & Zeger S. L. Analysis of Longitudinal Data 2nd edn (Oxford Univ. Brown, H. & Prescott, R. Applied Mixed Models in Medicine 3rd edn (Wiley, 2015). Missing data analysis using multiple imputation: getting to the heart of the matter. Multiple imputation for missing data in epidemiological and clinical research: potential and pitfalls. Bunouf, P. & Molenberghs, G. Implementation of pattern-mixture models in randomized clinical trials. Wu, S., Crespi, C. M. & Wong, W. K. Comparison of methods for estimating the intraclass correlation coefficient for binary responses in cancer prevention cluster randomized trials. This work was supported by the National Key Research and Development Program of the Ministry of Science and Technology of China (grant no. ), Chinese Society of Cardiology Foundation (grant no. ), and the Science and Technology Program of Liaoning Province, China (grant no. The funder of the study had no role in study design, data collection, data analysis, data interpretation or writing of the report. The US investigators did not receive any financial support from this study. We are grateful to the NPCHPs, primary care physicians, hypertension specialists and research staff at all participating institutes for their support throughout the study. A full list of the committee members and study group members of the China Rural Hypertension Control Project is provided in Supplementary Information. We acknowledge C. Li from the Alzheimerʼs Disease Research Center at the Icahn School of Medicine at Mount Sinai and J. S.-Lemus from the Department of Psychology at Montclair State University for their advice and recommendations on cognitive and functional status screening tools and diagnostic guidelines for dementia and CIND. We thank C. Li and X. Zeng for providing training and certification on cognitive and functional status assessment to the key investigators. Furthermore, we express our gratitude to H. He and S. Geng from the Tulane University Translational Science Institute for their advice and assistance in statistical analyses. These authors contributed equally: Jiang He, Chuansheng Zhao, Shanshan Zhong, Nanxiang Ouyang, Guozhe Sun, Lixia Qiao. Department of Epidemiology, Peter O'Donnell Jr. School of Public Health, UT Southwestern Medical Center, Dallas, TX, USA Department of Internal Medicine, UT Southwestern Medical Center, Dallas, TX, USA Department of Neurology, UT Southwestern Medical Center, Dallas, TX, USA Peter O'Donnell Jr. Brain Institute, UT Southwestern Medical Center, Dallas, TX, USA Department of Neurology, First Hospital of China Medical University, Shenyang, China Chuansheng Zhao, Shanshan Zhong, Huayan Liu, Weiyu Teng & Xu Liu Department of Cardiology, First Hospital of China Medical University, Shenyang, China Nanxiang Ouyang, Guozhe Sun, Lixia Qiao, Chang Wang, Songyue Liu & Yingxian Sun Division of Cardiology, Department of Internal Medicine, Tongji Hospital, Tongji Medical College of Huazhong University of Science and Technology, Wuhan, China Tulane University Translational Science Institute, New Orleans, LA, USA Sticht Center for Healthy Aging and Alzheimer's Prevention, Department of Internal Medicine, Section on Geriatric Medicine, Wake Forest University School of Medicine, Winston-Salem, NC, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived and designed the study. supervised the data collection. analyzed and interpreted the data. All authors revised the paper for important intellectual content and approved the final submitted version. J.H., Chuansheng Zhao, S.Z., N.O., G.S., C.-S.C. and Y.S. accessed and verified the data. had full access to all the data in the study and had final responsibility for the decision to submit for publication. All authors vouch for the completeness and accuracy of the data and for the fidelity of the trial to the protocol. Correspondence to Jiang He or Yingxian Sun. The authors declare no competing interests. Nature Medicine thanks Josef Coresh, Ruth Peters and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. List of CRHCP committee members and study group members, eligibility criteria, definition of study outcomes, and Supplementary Tables 1–8 and Fig. Systolic and diastolic blood pressure in the intervention and usual care groups over the 48 months of follow-up. Forest plot of the primary and main secondary outcomes according to subgroups. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions He, J., Zhao, C., Zhong, S. et al. Blood pressure reduction and all-cause dementia in people with uncontrolled hypertension: an open-label, blinded-endpoint, cluster-randomized trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64524905/secret-universe-qubit/'>Scientists Think the Secrets of Our Universe Could Be Contained in One Tiny Qubit</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“Give me a qubit for long enough and a probe in which to measure it, and I shall extract the geometry of our world.” Is there a chance that the secrets of our universe are contained in some of its smallest, newest components: the quantum bit, or qubit? In a recent paper exploring the spirit of this question, uploaded to the preprint server arXiv, quantum physicists James Fullwood and Vlatko Vedral suggest that spacetime itself may be a large-scale rendering of tiny quantum changes within time that could be found on these microscopic devices. After all, qubits are simple enough to be one logical bit in a quantum computer, but robust enough to embody quantum mechanics principles like superposition. far out, but it joins a growing discussion about the nature of spacetime. As quantum and classical physicists continue to search for the union between their separate sets of ideas, we know after nearly 100 years of work that spacetime is more of a shorthand for something we don't understand yet. Cosmologists continue to wind time backward as best they can, but there's much we will never be able to study or verify about where our universe came from—or what it really is or isn't. “There is a growing consensus in theoretical physics that spacetime is not a primitive notion,” Fullwood and Vedral explain, meaning that the universe is not as intuitive, or apparent, as people have guessed by looking at the sky. A toy model is an extremely simplified version, usually to show one or two key mechanics without other variables. The toy qubit universe “serves as a reservoir from which an observer may extract the information necessary for a geometric structure to emerge,” they concluded. “[T]he question remains as to how the metric of spacetime [emphasis theirs] may emerge in such a framework, and more generally, how gravity may enter the picture.” The researchers hint at this themselves with a concluding thought: “Archimedes is said to have summarized his discovery via the poetic statement ‘Give me a lever long enough and a fulcrum on which to place it, and I shall move the world.' [W]e summarize this Letter with the following statement: ‘Give me a qubit for long enough and a probe in which to measure it, and I shall extract the geometry of our world. Caroline Delbert is a writer, avid reader, and contributing editor at Pop Mech. Humans Could Grow New Teeth in Just a Few Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/cars/hybrid-electric/a64516018/cybertruck-sales-bad/'>1 Million People Reserved a Tesla Cybertruck. In the End, Only 5% Actually Meant It.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Elon Musk is sitting on a lot of inventory. It's no surprise that we've reported on Tesla's current struggles quite frequently in recent weeks. For those not in the know, old inventory is particularly problematic in Tesla's case, as these older vehicles are not eligible for the federal tax credit awarded to many who purchase electric vehicles (EVs). Electrek reports that some of the old stock may even include foundation series trucks reserved for early adopters of the Cybertruck. Regardless, Tesla's recent promotions of improved financing options for the vehicle, along with five-figure discounts, point to sales struggles. Business Insider reports that the Texas Gigafactory is not operating at full capacity anymore, and has pivoted some Cybertruck workers to the Model Y production line. For context, Elon himself said Giga Texas was capable of churning out 200,000 Cybertrucks annually, with the manpower to up that to 250,000 units if needed. So, you might be wondering: how many Cybertrucks has Tesla moved in the first quarter of this year? Thanks to Tesla's most recent recall notice, we also know that they've sold less than 50,000 trucks altogether. This curbs any ideas that Tesla's more affordable rear-wheel-drive (RWD) Cybertruck could save the day. We use “affordable” lightly, as it's slated to start at $72,000—that's $10,000 cheaper than the AWD spec—and features considerably fewer amenities. And it's not looking like current events are bolstering the brand's sales numbers, either. A recent Wired article mentioned that, typically, about 2 to 16 percent of reservations transfer into actual vehicle sales. If we assume Cybertruck was at the top of the range, retaining 16 percent of people who made reservations, that would equate to 160,000 sales. Now, they're merely a fringe mobility solution that you'll occasionally see doing tours of your city. We should clarify that the Cybertruck's waning popularity isn't as dire a scenario as it sounds for Tesla and Giga Texas. Yes, there's no getting around the fact that they've spent billions building the Gigafactory down south, but production won't just stop altogether. They'll likely continue to pivot production towards more profitable vehicles like the Model Y. He was previously a contributing writer for Motor1 following internships at Circuit Of The Americas F1 Track and Speed City, an Austin radio broadcaster focused on the world of motor racing. He earned a bachelor's degree from the University of Arizona School of Journalism, where he raced mountain bikes with the University Club Team. You Can Fix Cybertruck Scratches With a Power Tool A Ford Bronco Was Brought Back to Life as an EV EV Charging Doesn't Really Need to Be That Fast</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/measles-whooping-cough-the-laschamp-event-and-a-colossal-squid/'>Carbon Cycle on Mars, Colossal Squid and Ancient Solutions to Shifting Magnetic Poles</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Public Health Focuses on Childhood, Magnetic Poles Once Wandered, and Colossal Squid Discovered This week's news roundup covers measles and whooping cough cases, evidence of a carbon cycle on Mars and the first glimpse at a colossal squid in its natural habitat. Measles is continuing to spread in the U.S., with 712 cases confirmed so far in 2025 as of April 11, according to the Centers for Disease Control and Prevention. For reference, there were just 285 confirmed cases in all of 2024. The CDC has confirmed two deaths from measles this year and is investigating a third. Last week the CDC's Advisory Committee on Immunization Practices met after its planned meeting in February was postponed. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. But measles isn't the only illness that's increasingly putting kids at risk. Cases of whooping cough, or pertussis, are up more than 1,500 percent nationwide compared with 2021, according to recent reporting by ProPublica. Deaths from whooping cough are also on the rise. Caused by the bacterium Bordetella pertussis, whooping cough spreads easily between humans. Even people with mild symptoms can pass the microbe along, and the resulting illness can be much worse in vulnerable individuals like babies. While some infants will have cold symptoms, others may develop pneumonia and difficulty breathing. Kids should then get a Tdap booster from age 11 to 12. The CDC also recommends that people get a Tdap booster while pregnant—ideally between 27 and 36 weeks—to help confer immunity to their newborns. You can also opt to get a Tdap jab when it's time for a tetanus booster in case your immunity against whooping cough has waned. According to ProPublica, vaccination rates among kindergarten students have fallen for measles, mumps and rubella; pertussis; diphtheria; tetanus; hepatitis B; and polio. The U.S. Department of Health and Human Services, which is helmed by Robert F. Kennedy Jr., put out a statement about the report that claimed it showed “a persistent rise in [autism spectrum disorder] prevalence [and] an alarming escalation in case severity,” but this contradicts the conclusion of the study's own authors. The report suggests that rates of autism spectrum disorder are likely rising because early detection is improving, especially in groups that previously had less access to diagnostics. Looking back to simpler times, a study published last Wednesday in the journal Science Advances explored how ancient humans survived something called the Laschamp event. This incident about 41,000 years ago was a geomagnetic “excursion,” which is where the Earth's magnetic poles move around. The planet's magnetic field was much weaker than usual during this period. Given that Earth's magnetic field helps protect us from cosmic radiation, it's likely that people were exposed to more UV light as a result. The researchers also saw an uptick in humans' use of caves in times and places that solar radiation would have posed more of a threat. Speaking of cosmic happenings: another study from last week's Science Advances describes a planet with an unprecedented orbit. The story starts with a pair of rare “failed stars” about 120 light-years away. They're both brown dwarfs, which sit somewhere between gas giants and small stars on the planet-to-star spectrum. Brown dwarfs interest scientists because they seem to form the way stars do, but they don't actually manage the hydrogen fusion that gives stars their light. Eclipsing brown dwarf pairs are really rare, and the passage of one star in front of the other helps scientists make certain observations to calculate their masses. Now this binary system is proving to be even more rare than we thought: it features a planet that orbits perpendicularly around them, instead of orbiting roughly along the same plane on which the brown dwarfs themselves orbit. That's never been seen in a binary star system before. Still in space, but much closer to home, scientists have found new evidence that Mars once had a carbon cycle. In a study published last Thursday in Science, researchers report that the Curiosity rover dug up a mineral called siderite when drilling the Gale crater. This mineral is made of iron and carbonate, and it indicates that carbon once moved through the Red Planet's environment similarly to how it does on Earth. The species, which was first discovered a century ago, has never been caught on camera in its natural habitat. Now, using a remote-controlled vessel at 1,968 feet [600 meters] below the surface, scientists have finally spotted a colossal squid in the comfort of its deep-sea home. This one is young, so it's only about a foot [30 centimeters] long. You can check out the footage for yourself on our YouTube channel. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper, Naeem Amarsy and Jeff DelViscio. Subscribe to Scientific American for more up-to-date and in-depth science news. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Fonda Mwangi is a multimedia editor at Scientific American. She previously worked as an audio producer at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02169-3'>Transcriptome-wide analysis of differential expression in perturbation atlases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 09:37:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Single-cell CRISPR screens such as Perturb-seq enable transcriptomic profiling of genetic perturbations at scale. However, the data produced by these screens are noisy, and many effects may go undetected. Here we introduce transcriptome-wide analysis of differential expression (TRADE)—a statistical model for the distribution of true differential expression effects that accounts for estimation error appropriately. TRADE estimates the ‘transcriptome-wide impact', which quantifies the total effect of a perturbation across the transcriptome. Analyzing several large Perturb-seq datasets, we show that many transcriptional effects remain undetected in standard analyses but emerge in aggregate using TRADE. A typical gene perturbation affects an estimated 45 genes, whereas a typical essential gene affects over 500. We find moderate consistency of perturbation effects across cell types, identify perturbations where transcriptional responses vary qualitatively across dosage levels and clarify the relationship between genetic and transcriptomic correlations across neuropsychiatric disorders. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Raw sequencing data are deposited on SRA under BioProject PRJNA1100571. Aligned sequencing data and processed single-cell populations are available on GEO at GSE264667. Perturb-seq data from Replogle et al.6 can be accessed at https://gwps.wi.mit.edu/ (raw data available at SRA under BioProject PRJNA831566). dTAG data from Naqvi et al.26 are available at Gene Expression Omnibus (GEO) under accession number GSE205904. dTAG data from Weber et al.27 are available at GEO under accession number GSE145016. Summary statistics from the PsychENCODE consortium are available at https://github.com/mgandal/Shared-molecular-neuropathology-across-major-psychiatric-disorders-parallels-polygenic-overlap; Raw data are all available at Synapse under accession number syn4921369. RNA-seq data from the OneK1K dataset are available at GEO under accession number GSE196830. The TRADE method, with accompanying documentation, is publicly available as an R package at https://github.com/ajaynadig/TRADEtools. The publication version of this package is available as a persistent repository via Zenodo at https://doi.org/10.5281/zenodo.14993815 (ref. Dixit, A. et al. Perturb-seq: dissecting molecular circuits with scalable single-cell RNA profiling of pooled genetic screens. Adamson, B. et al. A multiplexed single-cell CRISPR screening platform enables systematic dissection of the unfolded protein response. Jaitin, D. A. et al. Dissecting immune circuits by linking CRISPR-pooled screens with single-cell RNA-seq. Datlinger, P. et al. Pooled CRISPR screening with single-cell transcriptome readout. Yang, J. et al. Common SNPs explain a large proportion of the heritability for human height. Replogle, J. M. et al. Mapping information-rich genotype-phenotype landscapes with genome-scale Perturb-seq. Peidli, S. et al. scPerturb: harmonized single-cell perturbation data. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Plaisier, S. B., Taschereau, R., Wong, J. & Graeber, T. G. Rank–rank hypergeometric overlap: identification of statistically significant overlap between gene-expression signatures. Nucleic Acids Res. Ma, Y. et al. Integrative differential expression and gene set enrichment analysis using summary statistics for scRNA-seq studies. O'Connor, L. J. The distribution of common-variant effect sizes. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Stephens, M. False discovery rates: a new deal. Law, C. W., Chen, Y., Shi, W. & Smyth, G. K. voom: precision weights unlock linear model analysis tools for RNA-seq read counts. O'Connor, L. J. et al. Extreme polygenicity of complex traits is explained by negative selection. Meyers, R. M. et al. Computational correction of copy number effect improves specificity of CRISPR–Cas9 essentiality screens in cancer cells. Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Binder, J. X. et al. COMPARTMENTS: unification and visualization of protein subcellular localization evidence. Crow, M., Lim, N., Ballouz, S., Pavlidis, P. & Gillis, J. Predictability of human differential gene expression. Lin, Y. et al. Evaluating stably expressed genes in single cells. Weiss, M. J., Keller, G. & Orkin, S. H. Novel insights into erythroid development revealed through in vitro differentiation of GATA-1 embryonic stem cells. Genes Dev. Lacher, S. M. et al. HMG-CoA reductase promotes protein prenylation and therefore is indispensible for T-cell survival. Cell Death Dis. Collins, R. L. et al. A cross-disorder dosage sensitivity map of the human genome. Domingo, J. et al. Non-linear transcriptional responses to gradual modulation of transcription factor dosage. Jost, M. et al. Titrating gene expression using libraries of systematically attenuated CRISPR guide RNAs. Naqvi, S. et al. Precise modulation of transcription factor levels identifies features underlying dosage sensitivity. Weber, C. M. et al. mSWI/SNF promotes Polycomb repression both directly and through genome-wide redistribution. Nabet, B. et al. The dTAG system for immediate and target-specific protein degradation. Gandal, M. J. et al. Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap. Lee, S. H., Yang, J., Goddard, M. E., Visscher, P. M. & Wray, N. R. Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood. A. et al. Co-varying neighborhood analysis identifies cell populations associated with phenotypes of interest from single-cell transcriptomics. Ji, Y. et al. Optimal distance metrics for single-cell RNA-seq populations. Preprint at bioRxiv https://doi.org/10.1101/2023.12.26.572833 (2023). Yazar, S. et al. Single-cell eQTL mapping identifies cell type–specific genetic control of autoimmune disease. Kang, J. B. et al. Mapping the dynamic genetic regulatory architecture of HLA genes at single-cell resolution. Pratapa, A., Jalihal, A. P., Law, J. N., Bharadwaj, A. & Murali, T. M. Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data. Squair, J. W. et al. Confronting false discoveries in single-cell differential expression. Soneson, C. & Robinson, M. D. Bias, robustness and scalability in single-cell differential expression analysis. Lopez, R., Regier, J., Cole, M. B., Jordan, M. I. & Yosef, N. Deep generative modeling for single-cell transcriptomics. Rubin, A. J. et al. Coupled single-cell CRISPR screening and epigenomic profiling reveals causal gene regulatory networks. Feldman, D. et al. Optical pooled screens in human cells. Gu, J. et al. CRISPRmap: sequencing-free optical pooled screens mapping multi-omic phenotypes in cells and tissue. Preprint at bioRxiv https://doi.org/10.1101/2023.12.26.572587 (2023). Binan, L. et al. Simultaneous CRISPR screening and spatial transcriptomics reveals intracellular, intercellular, and functional transcriptional circuits. Preprint at bioRxiv https://doi.org/10.1101/2023.11.30.569494 (2023). Xu, Z., Sziraki, A., Lee, J., Zhou, W. & Cao, J. Dissecting key regulators of transcriptome kinetics through scalable single-cell RNA profiling of pooled CRISPR screens. Kudo, T. et al. Multiplexed, image-based pooled screens in primary cells and tissues with PerturbView. Rood, J. E., Hupalowska, A. & Regev, A. Toward a foundation model of causal cell and tissue biology with a perturbation cell and tissue atlas. A., Sun, J. S. & Sanjana, N. E. Next-generation forward genetic screens: uniting high-throughput perturbations with single-cell analysis. Yao, D. et al. Scalable genetic screening for regulatory circuits using compressed Perturb-seq. Simmons, S. K. et al. Mostly natural sequencing-by-synthesis for scRNA-seq using Ultima sequencing. Lee, S. H. et al. Estimation of pleiotropy between complex diseases using single-nucleotide polymorphism-derived genomic relationships and restricted maximum likelihood. Replogle, J. M. et al. Maximizing CRISPRi efficacy and accessibility with dual-sgRNA libraries and optimal effectors. Torres, S. E. et al. Ceapins block the unfolded protein response sensor ATF6α by inducing a neomorphic inter-organelle tether. Replogle, J. M. et al. Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing. Urbut, S. M., Wang, G., Carbonetto, P. & Stephens, M. Flexible statistical methods for estimating and testing effects in genomic studies with multiple conditions. Patro, R., Duggal, G., Love, M.I., Irizarry, R.A. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Nicolas, L. B., Harold, P. P. & Pachter, L. M. Near-optimal probabilistic RNA-seq quantification. Cross-Disorder Group of the Psychiatric Genomics Consortium. Genomic relationships, novel loci, and pleiotropic mechanisms across eight psychiatric disorders. Article PubMed Central Rumker, L. et al. Identifying genetic variants that influence the abundance of cell states in single-cell data. Download references We thank K. A. Lagattuta, D. J. Weiner, B. Harris, T. Aicher, D. L. Barabasi, K. Maher, T. Kamath, M. T. Tegtmeyer and members of the O'Connor and Robinson laboratories for helpful comments and discussions. is supported by National Institutes of Health (NIH) grant no. is supported by NIH grant nos. This work was supported by a grant from SFARI (704413, E.B.R.). This work was supported by the Stanley Center for Psychiatric Research at the Broad Institute. acknowledges funding from the NIH National Institute of General Medical Sciences, 1R35GM155278, and SFARI, GR0243225. acknowledges funding from the NIH Center of Excellences in Genome Sciences, 2RM1HG009490, the Whitehead Innovation Initiative and The Eric and Wendy Schmidt Center at the Broad Institute. The project described was supported by award no. T32GM007753 and T32GM144273 from the National Institute of General Medical Sciences. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of General Medical Sciences or the NIH. are HHMI investigators. Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA Ajay Nadig & Luke J. O'Connor Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA Ajay Nadig & Elise B. Robinson Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA Ajay Nadig, Mukundh Murthy, Steven A. McCarroll & Elise B. Robinson Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USA Ajay Nadig & Elise B. Robinson Department of Medicine, Massachusetts General Hospital, Boston, MA, USA Joseph M. Replogle Medical Scientist Training Program, University of California San Francisco, San Francisco, CA, USA Joseph M. Replogle Whitehead Institute for Biomedical Research, Massachusetts Institute of Technology, Cambridge, MA, USA Joseph M. Replogle, Angela N. Pogson & Jonathan S. Weissman Department of Genetics, Harvard Medical School, Boston, MA, USA Howard Hughes Medical Institute, Cambridge, MA, USA Steven A. McCarroll & Jonathan S. Weissman David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA, USA Jonathan S. Weissman Department of Biology, Massachusetts Institute of Technology, Cambridge, MA, USA Jonathan S. Weissman Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived the study. collected new data. supervised new data collection. wrote the manuscript. provided additional project supervision. Correspondence to Ajay Nadig, Joseph M. Replogle or Luke J. O'Connor. declares outside interest in 5 AM Venture, Amgen, Chroma Medicine, KSQ Therapeutics, Maze Therapeutics, Tenaya Therapeutics, Tessera Therapeutics, Ziada Therapeutics and Third Rock Ventures. consults for Third Rock Ventures and Maze Therapeutics, and is a consultant for and equity holder in Waypoint Bio. The remaining authors declare no competing interests. Nature Genetics thanks Stefan Peidli, Ying Ma and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. 1–26 and Notes 1–5. Supplementary Tables. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Nadig, A., Replogle, J.M., Pogson, A.N. et al. Transcriptome-wide analysis of differential expression in perturbation atlases. Download citation Accepted: 17 March 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Genetics ISSN 1546-1718 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02641-9'>QBEmax is a sequence-permuted and internally protected base editor</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 09:23:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cytosine base editors (CBEs) show promise for multiplex gene knockout applications, but impure edits, indels and off-targets still frequently occur. We describe here QBEmax, which exhibits high efficiency, low indel and off-targets and high product purity with up to 99.8% of edits comprised of C-to-T. Through molecular dynamic modeling, QBEmax presents as a compact and stable base editor that shields protected bases from undesired repair processes. The two most common classes of base editors are modular fusion proteins comprised of a deaminase and DNA-binding protein to perform targeted and precise C·G-to-T·A (cytosine base editors (CBEs)) or A·T-to-G·C (adenine base editors (ABEs)) base conversions with high efficiency1,2. In addition to the use of base editors to correct disease-causing genetic mutations, base editors have shown promise as an alternative to nucleases for multiplex gene knockout applications. CBEs can precisely edit arginine, glutamine or tryptophan codons to generate premature stop codons; because this process does not undergo the formation of DNA double-strand breaks, base editing for multiplex gene knockouts minimizes the risk of genomic translocations, cell toxicity and DNA chromothripsis3,4. However, unintended indels and impure base editing byproducts (for example, C-to-G and C-to-A) are still frequently observed when using CBEs. The formation of impure edits could be detrimental and act as a missense mutation in a gene otherwise targeted for knockout. In this regard, an ideal CBE for gene knockout should exhibit properties such as high efficiency, low indel formation, low off-target edits, an expanded editing window to reach more potential bases and, importantly, high product purity for safety purposes. Because canonical base editors are comprised of different proteins fused together end-to-end, each protein's orientation may not be best to balance catalytic processes including nontarget strand association/dissociation by the deaminase, target base deamination, base protection/exposure and, ultimately, endogenous cellular DNA repair enzymes. Previous engineering studies have demonstrated that either the use of an inlaid deaminase domain or a circularly permuted Cas protein could affect the on-target or off-target editing efficiencies, product purity or editing windows of base editors5,6,7,8,9. However, the combination of these parameters has not been optimized using any one approach. To obtain a more ideal base editor, we hypothesized that we could treat the deaminase and Cas9 protein together as one complex comprised of different domains. Therefore, by shuffling the orientation of domains from both the deaminase and Cas9 together, which combines concepts of circularly permuted proteins and inlaid deaminases, we hoped to identify a CBE that maximizes the beneficial properties of using CBEs for gene knockout. We first designed four base editor orientations in which a deaminase was internally embedded within a circularly permuted Cas9(D10A) protein10 (Fig. 1a) based on Cas9 positions previously found to be amenable for circular permutation or for inlaying a small peptide10,11. These architectures are hereby designated as Q base editors (QBE1 to QBE4) to reflect the reconstitution process of circularizing domains from the Cas9 protein and inserting a deaminase internally to generate a new start codon (circle with an internal cut resembling the letter ‘Q'). To characterize the properties of different QBE architectures, six different cytidine deaminases (rAPOBEC1 (ref. a, Schematic representations of the BE4max, QBE1, QBE2, QBE3 (QBEmax), QBE4 editors. Numbers under and above nCas9(D10A) represent amino acid positions in reference to wild-type SpCas9. for three independent biological replicates. c, Gray scale heat map showing average cytosine base editing frequencies by mini-Sdd9-BE4max and mini-Sdd9-QBEmax at each protospacer position across 17 endogenous sites tested. Numbers below the heat map indicate protospacer positions. d, AlphaFold3-predicted structures of mini-Sdd9-BE4max and mini-Sdd9-QBEmax bind to a sgRNA and DNA target (site 1). Blue, Cas9 domains; orange, sgRNA; purple, dsDNA target; pink, mini-Sdd9. e–g, Average editing frequencies of C-to-T conversions (e), indels (f) and ratio of base edit-to-indel (g) induced by the mini-Sdd9-BE4max or mini-Sdd9-QBEmax editors across 17 endogenous sites; each dot represents the mean for three independent biological replicates for a specified target site, the violin plot shows the base editing frequency distribution with medians and quartiles, and significances are indicated between the mini-Sdd9-BE4max and mini-Sdd9-QBEmax by exact P value using two-tailed Student's t-test, n = 17. h, Percent of edited reads with C-to-T conversions among edited events at each C1 to C16 base position, cumulated across 17 endogenous sites; values and error bars represent the means and s.e.m. for three independent biological replicates. CMV pro, enhanced cytomegalovirus promoter; DEA, deaminase; NLS, nuclear localization signals; bGH, bovine growth hormone polyadenylation signal. Plasmids encoding corresponding CBEs were transfected in HEK293T cells and compared with the canonical BE4max architecture13 at two endogenous genomic sites. Deep sequencing revealed that QBE3-based editors showed comparable or higher editing frequencies for four of the six deaminases evaluated; in contrast, QBE1-based, QBE2-based and QBE4-based editors exhibited lower editing frequencies (Supplementary Fig. Compared to the BE4max counterparts, the QBE3-based editors showed substantially lower indels, with an average indel reduction of 60.4% for site 1 and 62.6% for site 3 (Supplementary Fig. We next measured the edit-to-indel ratios and found that six of the seven editors at site 1, and five of the seven editors at site 3 showed substantially higher edit/indel ratios (Supplementary Fig. We then analyzed the editing window and product purities of QBE editors. We observed higher editing efficiencies at PAM-proximal Cs for QBE3 editors and greatly improved product purities compared to BE4max editors (Supplementary Fig. Based on these, we hereby refer to QBE3 as QBEmax (Fig. From initial evaluations, we found that mini-Sdd9-based QBE editors exhibited superior editing properties in terms of editing activity, indel formation and product purity. Using mini-Sdd9, we designed seven additional QBEs (QBE5–QBE11; Supplementary Fig. We found that only one editor, QBE6 demonstrated similar performance to QBEmax in terms of editing efficiency and purity; however, its editing window appeared narrow, so we hereby designate it as QBEn (Supplementary Fig. Because of mini-Sdd9-QBEmax's overall superior performance and relatively wide editing window, which is desired for expanding the targeting scope of gene knockout applications with base editors, we selected it for further study. It was reported that fusing the deaminase to the N terminus of a circularly permuted Cas9 (referred to as CP-BE)5 could broaden the editing window, and inlaying the deaminase within the Cas9 protein (referred to as inlaid-BE)6,7,8,9 could affect the editing efficiency or off-target efficiency. We next compared mini-Sdd9-QBEmax with CP-BEs and inlaid-BEs with Cas9 permutation or deaminases inlaid at positions used in the QBE1–QBE11 editors. We found that all CP-BEs and inlaid-BEs induced more impure products than QBEmax. Notably, QBEmax (which uses positions 1,031 and 1,244 for modular assembly) outperforms individual CP-1031 and inlaid-1244 CBEs when comparing the combination of editing frequencies, indel formation and product purities (Extended Data Fig. 1), suggesting that the modularly designed QBEmax architecture enhances desired properties from each functional domain. To further profile editing properties of mini-Sdd9-QBEmax, we compared mini-Sdd9-QBEmax with mini-Sdd9-BE4max across 17 endogenous genomic sites in HEK293T cells. We found that, in contrast to mini-Sdd9-BE4max, which biases editing at the PAM-distal region, mini-Sdd9-QBEmax showed a wider editing window as far as C16 (Fig. Aggregate analyses revealed a ‘forward-shifted' and wider editing window for mini-Sdd9-QBEmax with target Cs between 4 and 14 being favored (Fig. 14) to predict a mini-Sdd9-QBEmax–sgRNA–target DNA ternary structure and compared it with that of the corresponding BE4max architecture (Fig. We found that these structures revealed the deaminase in mini-Sdd9-QBEmax being more closely associated to PAM-proximal Cs, while in BE4max, more closely associated to PAM-distal Cs, which is consistent with experimental results from genomic edits. The average editing frequencies across 17 genomic sites for mini-Sdd9-QBEmax and mini-Sdd9-BE4max were 52.4 ± 2.4% and 54.5 ± 2.2%, respectively (Fig. Mini-Sdd9-QBEmax induced lower indels at 16 of 17 sites tested, with average indel frequencies decreasing by 56.5% from 2.8 ± 0.3% to 1.2 ± 0.2% (Fig. 4), which also substantially increases average edit-to-indel ratios (Fig. We next evaluated cytosine base editing product purities, which is calculated as the proportion of ‘C' edited to ‘T' as opposed to ‘G' or ‘A', for each position within the protospacer and aggregated all sites together. Importantly, mini-Sdd9-QBEmax exhibited superior product purities (99.4% ± 0.4%) at all positions within a C1–C16 editing window compared to that of mini-Sdd9-BE4max (95.5% ± 2.8%; Fig. To test the versatility of the QBEmax, we evaluated mini-Sdd9-QBEmax in additional mammalian cell lines, including A549, HeLa and HCT116. QBEmax exhibited higher or comparable editing efficiencies, improved product purities and decreased indels in all cell lines tested (Extended Data Fig. These results highlight QBEmax in achieving efficient and precise base edits at a flexible editing window with minimal indel and byproducts. Chimeric antigen receptor-T cell (CAR-T) therapy has demonstrated success as a cancer immunotherapy for hematological malignancies. Many clinical trials and research studies have found that multiplex knockout of genes related to immune rejection and graft-versus-host disease (GvHD) would further benefit CAR-T therapy in terms of both durability and potency. Encouraged by the performance of mini-Sdd9-QBEmax, we next sought to perform multiplex gene knockout to simultaneously edit genes that could compromise the efficacy of CAR-Ts. 15), CISH16, Fas17, TGFBR2 (truncating off the endodomain)18,19,20 and TRAC21, were selected, which all previously demonstrated potential in improving CAR-T performance when knocked out or downregulated. We first identified all possible SpCas9 protospacers with an NGG PAM and target C located within codons encoding tryptophan (W), arginine (R) or glutamine (Q), so that a cytosine base edit would generate a stop codon (TAA, TAG or TGA). We obtained 46, 21, 16, 30 and 4 protospacers for PD-1, CISH, Fas, TGFBR2 and TRAC, respectively (Supplementary Table 1). Lastly, we included one additional target for CISH, PD-1 and TRAC, which disrupts a splice site to perform gene knockout as reported previously21,22 (Fig. a, Schematic representations showing PD-1, CISH, Fas, TGFBR2 and TRAC genes. Light blue boxes indicate exons of genes, and short red lines represent the position of selected protospacers. b–d, Average editing frequencies of desired editing efficiencies (b), indels (c) and ratio of base edits to indels (d) induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax editors for PD-1 (n = 17), CISH (n = 10), Fas (n = 10), TGFBR2 (n = 10) and TRAC (n = 4) genes; each dot represents the mean for three independent biological replicates for a specified target site and the violin plot shows base editing frequency distribution with medians and quartiles. e, Percent of edited reads with C-to-T or C-to-R conversions at target genes indicated with values and error bars representing the means and s.e.m. for three independent biological replicates across all target sites for PD-1 (n = 17), CISH (n = 10), Fas (n = 10), TGFBR2 (n = 10) and TRAC (n = 4) genes; significances are indicated between BE4max and QBEmax by exact P value using two-tailed Student's t-test. f, Schematic representations of potential editing outcomes induced by C-to-T, C-to-G and C-to-A conversions of tryptophan (W), arginine (R) and glutamine (Q). g–i, Desired editing efficiencies (g), indels (h) and percent of edited reads with C-to-T or C-to-R conversions at target genes indicated (i) induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax editors during multiplexed base editing of five genes; values and error bars represent the means and s.e.m., respectively, for four independent biological replicates. j, Single-cell colony analysis of multiplex base editing distributions in unsorted and sorted cell populations. l, Frequencies of C-to-T conversions at the dSaCas9-induced R-loop sites; values and error bars represent the means and s.e.m. for four independent biological replicates. m, Number of C-to-U RNA variants induced by the editors indicated; values and error bars represent the means and s.e.m. for two (Cas9(D10A)) or four (BE4max and QBEmax) independent biological replicates, significances are indicated by exact P value using one-way ANOVA Tukey's multiple comparisons. We then analyzed desired editing efficiencies (calculated as percent C-to-T for stop codon creation), indel frequencies, desired edit-to-indel ratios and product purities. We found that mini-Sdd9-QBEmax achieved comparable or slightly higher average desired editing at the target base for all sites aggregated for each of the five genes (Fig. Average indel frequencies for all sites aggregated decreased by 76%, 75%, 59% and 71% for PD-1, CISH, Fas and TGFBR2, respectively (Fig. The average indel frequency at TRAC was 0.8 ± 0.15% for mini-Sdd9-BE4max and 1.0 ± 0.34% for mini-Sdd9-QBEmax due to one outlier at TRAC-site 3 whereby mini-Sdd9-BE4max and mini-Sdd9-QBEmax exhibited 1.4 ± 0.18% and 2.9 ± 0.22%, respectively. Cumulatively, the desired edit-to-indel ratios induced by mini-Sdd9-QBEmax were 3.26, 3.99, 2.02, 9.91 or 1.99-fold higher than that of mini-Sdd9-BE4max (Fig. This increase in product purity minimizes the formation of missense mutations from imprecise C-to-G or C-to-A edits (Fig. When analyzed individually, mini-Sdd9-QBEmax induced lower indels at 47 of the 51 target sites and higher product purities at 45 of the 51 target sites (Extended Data Figs. For each gene, we identified one ideal guide and next edited PD-1, CISH, Fas, TGFBR2 and TRAC simultaneously for multiplex gene knockout. We transformed plasmids for each of the five sgRNAs together with QBEmax or BE4max editors into HEK293T cells. We observed that mini-Sdd9-QBEmax exhibited comparable or higher editing compared to mini-Sdd9-BE4max across all five sites in HEK293T cells in the absence of any selection pressure (Fig. Notably, mini-Sdd9-QBEmax achieved lower indel formations (Fig. 2h) and exhibited superior product purity (Fig. To validate that all five base edits occurred in a single cell, we sequenced 48 and 112 QBEmax-transfected single-cell colonies arising from unsorted or sorted cell populations, respectively. We found that 21 (43.8%) and 100 (89.3%) cell colonies exhibited all five genes edited, respectively, demonstrating successful multiplex base editing by QBEmax in a single cell (Fig. To further evaluate the potential of QBEmax, we co-electroporated QBEmax and all five sgRNA plasmids into an immortal Jurkat T cell line. In these T cells, QBEmax also exhibited superior editing efficiencies, improved product purities and decreased indels, which is similar to its performance in HEK293T cells and further supports the versatility of QBEmax for safe and robust base editing in clinical applications (Extended Data Fig. Because DNA off-targets are a major concern for base editing therapeutic applications, we next evaluated Cas-independent DNA off-target effects of mini-Sdd9-QBEmax using the orthogonal R-loop assay23,24,25. We cotransfected a dead-SaCas9 (dSaCas9) and sgRNA to induce the formation of an orthogonal R-loop simultaneously with the multiplexed gene knockout strategy (Fig. We evaluated five orthogonal sites and deep sequencing at each orthogonal R-loop showed that mini-Sdd9-QBEmax induced lower Cas-independent off-target editing at all five R-loop sites compared to that of mini-Sdd9-BE4max (Fig. We next evaluated the RNA off-target effects of mini-Sdd9-QBEmax. We conducted whole transcriptome sequencing and analyzed the number of C-to-U variants in QBEmax, BE4max and nCas9 (D10A) treated samples together with a sgRNA plasmid targeting CISH. We found that QBEmax induced substantially lower RNA off-target edits on transcriptome-wide RNA transcripts compared to that of the BE4max without compromising DNA on-target editing (Fig. The robust desired editing efficiencies, minimized indels, high product purities and decreased DNA and RNA off-target effects portray QBEmax as an ideal base editor for multiplex gene knockout applications. We next sought to probe the molecular basis by which mini-Sdd9-QBEmax embodies its desired properties. We performed molecular dynamic (MD) simulation analyses based on the AlphaFold3-predicted ternary structures of mini-Sdd9-QBEmax or BE4max with a sgRNA and target DNA (Fig. With these models, all-atom MD simulations of approximately 300 ns were performed (Supplementary Fig. We first investigated the conformational stability of these two systems by projecting their free energy landscapes onto corresponding root mean square deviation (RMSD) and radius of gyration (Rg) components. We found that both the RMSD and Rg of mini-Sdd9-QBEmax were lower than that of mini-Sdd9-BE4max, and only one single stable energy state was observed (Fig. This suggests that the QBEmax architecture better treats the deaminase and Cas protein as one complex so that each domain is oriented compactly within itself. At the minimum energy state, while the deaminase was predicted to be associated with the nontarget strand in both systems (Supplementary Fig. 6c,d), the RMSD of mini-Sdd9-QBEmax system was lower throughout the 300 ns MD process (Supplementary Fig. 3c,d) compared to the BE4max architecture. We speculate a more compact QBEmax architecture that limits the deaminase from sporadically swinging in space, thereby contributing to lower Cas-independent DNA off-target editing, lower indel formation and higher product purity. a,b, The free energy landscape against RMSD and Rg for mini-Sdd9-BE4max (a) and mini-Sdd9-QBEmax (b) during a 300 ns MS simulation. c,d, RMSF plot for mini-Sdd9-BE4max (c) and mini-Sdd9-QBEmax (d) in the MD simulation, systems equilibrated after 150 ns; schematic representations of editors are shown above the plots. e, SASA analysis of Cs within the editing window of site 1 in predicted mini-Sdd9-BE4max and mini-Sdd9-QBEmax ternary structures; each replicate represents the SASA by a 1.0 nm probe during a 1 ns time scale; n = 150 ns following system equilibration, boxes and lines represent the interquartile range (IQR) and median, respectively, and whiskers represent 1.5× IQR. f, Snapshots showing exposed Cs in the editing window. Blue, Cas9 and UGI; yellow, linker; green, mini-Sdd9 deaminase. During cytosine base editing, intermediate uracil cleavage by endogenous uracil DNA glycosylase (UNG) drives the formation of indels and imprecise C-to-G or C-to-A edits1,26. We envisioned that an ideal base editor adopts a compact and protective conformation for the exposed R-loop so that the intermediate uracil base is not excised before cellular mismatch repair resolving a permanent C-to-T conversion. To evaluate R-loop exposure, we performed solvent accessibility analyses using a 1.0 nm probe and found that the solvent-accessible surface area (SASA) was increased for C3 and substantially increased for C7 and C8 in the mini-Sdd9-BE4max compared to the mini-Sdd9-QBEmax architectures (Fig. 3e,f), suggesting that these residues are accessible by UNG and ultimately form indels and byproducts. We evaluated the position and distance of the two UGI domains to the ssDNA target bases in the QBEmax, BE4max or individual CP-1031 and inlaid-1244 CBEs based on molecular dynamic modeling data. We observed that indeed both UGI domains in QBEmax exhibited a relatively shorter distance to the target bases in the ssDNA R-loop region, which suggests an inverse relationship between UGI positioning to product purities and indel formation, as others previously have also identified5 (Supplementary Fig. Based on these results, we propose a model for QBEmax base editing. In cytosine base editing using a canonical BE4max architecture, indel formation and impure C-to-G or C-to-A base edits arise from uracil excision and abasic site formation. Because QBEmax exhibits a more compact architecture, limits deaminase swinging and shields the Cas9-induced R-loop, base editing intermediates are protected from cellular UNG excision before Cas9 detaching from the target DNA and subsequent mismatch repair. Therefore, a protective and compact base editor conformation reduces unintended effects driven by DNA repair processes and further promotes desired base editing events (Extended Data Fig. Taken together, we designed and identified a base editor architecture, QBEmax, which achieves high efficiency on-target editing while decreasing low indel formation, exhibits high edit product purities and minimizes DNA off-targets. The development of QBEmax serves as a promising base editor architecture for developing more efficient and precise base edits toward the use of base editing in multiplex therapeutic applications such as CAR-T immunotherapies. The efficient delivery of QBEmax in vivo will further expand on its use and analysis of desired base editing properties. Advances in protein prediction and MD further help shed light on the molecular basis of genome editors and would greatly aid in future developments of new editing technologies. The construct fragments were PCR amplified using 2× Phanta Max Master Mix (Vazyme Biotech) and 2× KOD one Master Mix (TOYOBO Life Sciences) and cloned into the pCMV24 backbone, using Uniclone One Step Seamless Cloning Kit (Genesand). Plasmids for HEK293T cell line transfection were extracted and purified using EndoFree Plasmid Kits (Qiangen) or FastPure EndoFree Plasmid Mini Kit DC203 (Vazyme Biotech). Primers used for amplicon sequencing were synthesized by the Beijing Genomics Institute and are listed in Supplementary Table 2. HEK293T, A549, HeLa and HCT116 cells are cultured in DMEM (Gibco) supplemented with 10% (vol/vol) FBS (Gibco) and 1% (vol/vol) penicillin–streptomycin (Gibco); Jurkat, Clone E6-1 cells are cultured in Roswell Park Memorial Institute (RPMI, Gibco) 1640 medium supplemented with 10% (vol/vol) FBS (Gibco) in a humidified incubator at 37 °C with 5% CO2. All the cells were routinely tested for Mycoplasma contamination with a mycoplasma detection kit (TransGen Biotech). For HEK293T cells transfection, 6 × 104 cells per well were seeded into 48-well poly-d-lysine-coated plates (Corning) in the absence of antibiotic. After 16–24 h, plasmids were transfected into the cells; for single target base editing, cells were transfected with 1 μl jetPRIME transfection reagent (Polyplus), 375 ng editor plus 125 ng sgRNA plasmids per well, for simultaneous editing of the five CAR-T relevant genes and the R-loop assays, 200 ng of editor plasmid and 66 ng of sgRNA plasmid for each gene was transfected, together with 200 ng of dSaCas9 plasmid and 66 ng of an orthogonal R-loop inducing sgRNA plasmid, at a 60–80% cell confluency. Cells were washed with PBS and followed by DNA extraction 72 h after transfection. To isolate single-cell colonies, transfected cells with or without sorting were diluted and plated into 96-well plates at a density of 0.9 cells per well. The cell colonies were cultured for 2 to 3 weeks and then transferred into 48-well plates to grow for another 5 days before cell lysis, DNA extraction and sequencing of all five targeted genomic loci. Effective editing in a single-cell colony for any individual gene is benchmarked as having an editing efficiency surpassing 50% at the targeted base when analyzed by CRISPResso2 (ref. For the sorted cell population, sgRNA plasmids and a mini-Sdd9-QBEmax-P2A-mScarlet plasmid were transfected and the top 5% of mScarlet-positive cells were collected and diluted into single-cell colonies. For A549, HeLa and HCT116 cells, 5 × 104, 2 × 105 or 4 × 105 cells were seeded into 12-well poly-d-lysine-coated plates (Corning), respectively, in the absence of antibiotic. After 16–24 h, 375 ng of mini-Sdd9-BE4max-P2A-mScarlet or mini-Sdd9-QBEmax-P2A-mScarlet plasmids were transfected into cells together with 125 ng sgRNA plasmid while using 2 μl of jetPRIME transfection reagent (Polyplus). After 48 h, cells were resuspended for fluorescence-activated cell sorting. For Jurkat cells, 1,000 ng of mini-Sdd9-BE4max-P2A-mScarlet or mini-Sdd9-QBEmax-P2A-mScarlet editor plasmids and 250 ng of sgRNA plasmid for each of the five genes were cotransfected into 4 × 105 cells using Entranster-E (Engreen Biosystem) and the 4D-Nucleofector (Lonza Biosciences) with program DS167. mScarlet-positive cells were collected for DNA extraction. Cells were collected and resuspended in a culture medium supplemented with 2% FBS in a 0.5 ml volume. Cells were sorted on a FACSAria III (BD Biosciences) cytometer after gating for the singlet-cell population by the mScarlet signal. For RNA off-target analysis, both the fluorescence-positive and negative cells were collected for further paired analysis. A representative flow cytometry gating strategy can be found in Supplementary Fig. Genomic DNA extraction was performed by the addition of 100 μl freshly prepared lysis buffer (10 mM Tris–HCl (pH8.0), 0.05% SDS and 25 μg ml−1 proteinase K (Thermo Fisher Scientific) directly into the 48-well culture plate after cells were washed once with 1× Dulbecco's PBS (Thermo Fisher Scientific). The mixture was incubated at 37 °C for 60 min and then treated at 80 °C in the thermocycler for 20 min. To profile the transcriptome-wide RNA off-target effects of the QBEmax and BE4max, 2 × 105 HEK293T cells were seeded into 12-well poly-d-lysine-coated plates (Corning) in the absence of antibiotics. After 24 h, 750 ng of mini-Sdd9-BE4max-P2A-mScarlet, mini-Sdd9-QBEmax-P2A-mScarlet or Cas9(D10A)-P2A-mScarlet plasmids were transfected into cells together with 250 ng sgRNA plasmid targeting the CISH-site 1 using 2 μl of jetPRIME transfection reagent (Polyplus). After 48 h, cells were collected for cell sorting. For each treatment, both the mScarlet-positive and negative cells were collected for paired analyses. RNA samples were sequenced using an MGI T7 (2 × 150 PE) platform at the JMDNA, at a depth of ~45 million reads per sample. Raw reads were cleaned by fastp28. After removing duplication, variants were identified using strelka30 (version 2.9.10). Finally, C-to-U edits in the transcribed strand were considered for downstream analyses. Two rounds of PCR were used to amplify a DNA fragment spanning the target site. In the first round PCR, the target region was amplified from genomic DNA with site-specific primers using the DNA template. In the second round, both forward and reverse barcodes were added to the ends of the PCR products for library construction. Equal amounts of PCR product were pooled and purified with a FastPure Gel DNA Extraction Mini Kit (Vazyme Biotech. Inc.) and quantified with a Qubit 4 (Thermo Fisher Scientific). Sequences of the NGS primers and the corresponding amplicons are listed in Supplementary Table 2. Analysis of the base editing outcomes was performed as described previously31. Initial protein structure models of mini-Sdd9-BE4max and mini-Sdd9-QBEmax ternary complex were predicted using AlphaFold3 (ref. MD simulations were conducted using GROMACS32 (version 2023.1, CUDA) with the Amber99BSC1 force field and the TIP3P water model33,34. Na⁺ and Cl− ions were added to the system to achieve charge neutrality with an ion concentration set at 0.15 M. During the simulation, the particle mesh Ewald method35 was employed to calculate electrostatic interactions and a cut-off distance of 14 Å was used for analysis of short-range electrostatic and van der Waals forces. The LINCS algorithm36 was applied to constrain bonds involving hydrogen atoms, and periodic boundary conditions were specified in all three dimensions. To optimize the system, energy minimization was performed using the steepest descent method, continuing until a maximum of 2,500 steps was reached or the maximum force <1,000 kJ mol−1 nm−1. Next, 100 ps of NVT equilibration and 100 ps of NPT equilibration were conducted. During the equilibration process, the V-rescale temperature and Parrinell–Rahman pressure coupling methods were employed, with the system's temperature and pressure set to 310 K and 1.0 bar, respectively. After system equilibration, a 300 ns MD simulation was performed. RMSD and RMSF data were generated using the gmx rmsd and gmx rmsf commands in GROMACS. The SASA data were obtained through the gmx sasa command, with a probe diameter of 1.0 nm, and snapshots of the SASA state were collected every 1 ns within the stable region. All data were statistically analyzed and visualized using R packages ggplot2 (3.5.1) and tidyverse (2.0.0). Visualization of this data was performed using the matplotlib (3.8.2) package in Python37. GraphPad Prism 9 software was used to analyze the data. All numerical values are presented as means ± s.e.m. Significant differences between controls and treatments were tested using the Student's t-test or one-way ANOVA Tukey's multiple comparisons. P < 0.05 was considered statistically significant, and P < 0.01 was considered statistically extremely significant. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The deep amplicon sequencing data have been deposited in the NCBI BioProject database under accession code PRJNA1147008 (ref. Plasmids encoding QBEmax and QBEn are available at Addgene. All other data are available in the main paper or Supplementary Information. & Liu, D. R. Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Highly efficient multiplex human T cell engineering without double-strand breaks using Cas9 base editors. Repair of double-strand breaks induced by CRISPR–Cas9 leads to large deletions and complex rearrangements. Huang, T. P. et al. Circularly permuted and PAM-modified Cas9 variants broaden the targeting scope of base editors. Jiang, L. et al. Internally inlaid SaCas9 base editors enable window specific base editing. Chu, S. H. et al. Rationally designed base editors for precise editing of the sickle cell disease mutation. Nguyen Tran, M. T. et al. Engineering domain-inlaid SaCas9 adenine base editors with reduced RNA off-targets and increased on-target DNA editing. Wang, Y., Zhou, L., Liu, N. & Yao, S. BE-PIGS: a base-editing tool with deaminases inlaid into Cas9 PI domain significantly expanded the editing scope. Oakes, B. L. et al. Profiling of engineering hotspots identifies an allosteric CRISPR–Cas9 switch. Oakes, B. L. et al. CRISPR–Cas9 circular permutants as programmable scaffolds for genome modification. Discovery of deaminase functions by structure-based protein clustering. Improving cytidine and adenine base editors by expression optimization and ancestral reconstruction. Accurate structure prediction of biomolecular interactions with AlphaFold 3. McGowan, E. et al. PD-1 disrupted CAR-T cells in the treatment of solid tumors: promises and challenges. Zhu, H. et al. Metabolic reprograming via deletion of CISH in human iPSC-derived NK cells promotes in vivo persistence and enhances anti-tumor activity. Yamamoto, T. N. et al. T cells genetically engineered to overcome death signaling enhance adoptive cancer immunotherapy. The TGF-β/SMAD pathway is an important mechanism for NK cell immune evasion in childhood B-acute lymphoblastic leukemia. Adapting a transforming growth factor β-related tumor protection strategy to enhance antitumor immunity. Bollard, C. M. et al. Tumor-specific T-cells engineered to overcome tumor immune evasion induce clinical responses in patients with relapsed Hodgkin lymphoma. Diorio, C. et al. Cytosine base editing enables quadruple-edited allogeneic CART cells for T-ALL. Kluesner, M. G. et al. CRISPR–Cas9 cytidine and adenosine base editing of splice-sites mediates highly-efficient disruption of proteins in primary and immortalized cells. & Liu, D. R. Evaluation and minimization of Cas9-independent off-target DNA editing by cytosine base editors. Richter, M. F. et al. Phage-assisted evolution of an adenine base editor with improved Cas domain compatibility and activity. Jin, S. et al. Rationally designed APOBEC3B cytosine base editors with improved specificity. Improved base excision repair inhibition and bacteriophage Mu Gam protein yields C:G-to-T:A base editors with higher efficiency and product purity. Clement, K. et al. CRISPResso2 provides accurate and rapid genome editing sequence analysis. Chen, S. Ultrafast one‐pass FASTQ data preprocessing, quality control, and deduplication using fastp. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Kim, S. et al. Strelka2: fast and accurate calling of germline and somatic variants. & Gao, C. Optimized prime editing in monocot plants using PlantPegDesigner and engineered plant prime editors (ePPEs). Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. Ivani, I. et al. Parmbsc1: a refined force field for DNA simulations. Nayar, D., Agarwal, M. & Chakravarty, C. Comparison of tetrahedral order, liquid state anomalies, and hydration behavior of mTIP3P and TIP4P water models. Darden, T., York, D. & Pedersen, L. Particle mesh Ewald: an N⋅log(N) method for Ewald sums in large systems. Hess, B., Bekker, H., Berendsen, H. J. C. & Fraaije, J. G. E. M. LINCS: a linear constraint solver for molecular simulations. Hunter, J. D. Matplotlib: a 2D graphics environment. Qi Biodesign, Inc. Homo sapiens (human): NGS raw sequencing data. This work was supported by the National Key R&D Program (2023YFF1001600), Beijing Municipal Science & Technology Commission (Z241100009024035) and Beijing Rural Revitalization Agricultural Science and Technology (project NY2401010024). We thank Y. Li at Qi Biodesign for assistance in RNA-sequencing raw data analysis and B. Zhou at Xin Hua Hospital, Shanghai Jiao Tong University School of Medicine for assistance in cell colony isolation. We thank the Flow Cytometry Facility at the National Center for Protein Sciences Beijing, particularly C. Han and Y. Wang, for their assistance. The Jurkat, Clone E6-1 cells were kindly provided by Cell Bank, Chinese Academy of Sciences. Jiacheng Hu, Mengyue Guo, Qiang Gao, He Jia, Mingyang He, Zhiwei Wang, Lina Guo, Guanwen Liu, Quan Gao & Kevin Tianmeng Zhao Center for Genome Editing, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar collected and analyzed amplicon sequencing data. wrote scripts and processed the raw amplicon sequencing data. wrote the manuscript with input from all authors. Correspondence to Kevin Tianmeng Zhao. The authors have submitted a patent application based on the results reported in this paper. is the founder and holds equity at Qi Biodesign. and Quan Gao are employees of Qi Biodesign. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Three-dimensional plots of editing efficiencies (y axis), indels (z axis) and product purities (x axis) induced by BE4max, QBEmax, CP-BEs and inlaid-BEs at site 1 (left) and site 3 (right). A golden cube in the top right corner highlights ideal base editors that exhibit high editing efficiency, high product purity and low indel formation simultaneously (thresholds for efficiency, purity and indel are set to be >70%, >98% and <3%, respectively). Each point represents the mean for three independent biological replicates; the target sites are shown above the plots with targeted Cs in red and PAM in blue. b, Values used for plotting the graph in a. a–c, Desired editing efficiencies (a), indels (b), and percent of edited reads (c) with C-to-T conversions induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax editors at site 1 with C7 in red used to calculate the efficiency and product purity. for three independent biological replicates. a, Frequencies of C-to-T or C-to-R conversions (left y axis) and indels (right y axis) induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax for ten sites targeting CISH knockout. b, Ratio of base edits to indels induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax; significances are indicated between the mini-Sdd9-BE4max and mini-Sdd9-QBEmax by exact P value using two-tailed Student's t test. c, Percent of edited reads with C-to-T or C-to-R conversions at the target sites indicated. for three independent biological replicates. a, Frequencies of C-to-T or C-to-R conversions (left y axis) and indels (right y axis) induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax for ten sites targeting Fas knockout. b, Ratio of base edits to indels induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax; significances are indicated between the mini-Sdd9-BE4max and mini-Sdd9-QBEmax by exact P value using two-tailed Student's t test. c, Percent of edited reads with C-to-T or C-to-R conversions at the target sites indicated. for three independent biological replicates. a, Frequencies of C-to-T or C-to-R conversions (left y axis) and indels (right y axis) induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax for 17 sites targeting PD-1 knockout. b, Ratio of base edits to indels induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax; significances are indicated between the mini-Sdd9-BE4max and mini-Sdd9-QBEmax by exact P value using two-tailed Student's t test. c, Percent of edited reads with C-to-T or C-to-R conversions at the target sites indicated. for three independent biological replicates. a, Frequencies of C-to-T or C-to-R conversions (left y axis) and indels (right y axis) induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax for ten sites targeting TGFBR2 knockout. b, Ratio of base edits to indels induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax; significances are indicated between the mini-Sdd9-BE4max and mini-Sdd9-QBEmax by exact P value using two-tailed Student's t test. c, Percent of edited reads with C-to-T or C-to-R conversions at the target sites indicated. for three independent biological replicates. a, Frequencies of C-to-T or C-to-R conversions (plot on the left y axis) and indels (plot on the right y axis) induced by the mini-Sdd9-BE4max and mini-Sdd9-QBEmax for four sites targeting TRAC knockout. b, Ratio of base edits to indels induced by the mini-Sdd9-BE4max and the mini-Sdd9-QBEmax; significances are indicated between the mini-Sdd9-BE4max and mini-Sdd9-QBEmax by exact P value using two-tailed Student's t test. c, Percent of edited reads with C-to-T or C-to-R conversions at the target sites indicated. for three independent biological replicates. a–c, Desired editing efficiencies (a), indels (b) and percent of edited reads (c) with C-to-T conversions at target genes induced by mini-Sdd9-BE4max and mini-Sdd9-QBEmax editors during multiplexed base editing of five genes; values and error bars represent the means and s.e.m. for three independent biological replicates. UNG, uracil DNA glycosylase; AP lyase, apurinic or apyrimidinic site lyase; NHEJ, nonhomologous end joining. sgRNA list of the 5 CAR-T genes. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Hu, J., Guo, M., Gao, Q. et al. QBEmax is a sequence-permuted and internally protected base editor. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01235-4'>How India rewrote the rules of space travel when it launched its first satellite</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-21 09:19:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Pranav Sharma is a science historian at the Raman Research Institute, Bengaluru, India, and a visiting faculty member of the Druk Gyalpo's Institute in Paro, Bhutan. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. The satellite, Aryabhata, provided a huge boost to India's space programme.Credit: NASA/Alamy The Indian engineers, most of whom were less than 35 years old, had arrived at this remote enclave in what is now southern Russia to hurl their nation's first satellite, named Aryabhata after an ancient Indian astronomer, into space, with the help of a Kosmos-3M launch vehicle. The relatively light satellite — a 358-kilogram payload packed with scientific instruments — had been flown halfway across the continent in a custom-built shockproof container padded with helical springs, designed to shield it from any forces it wouldn't be able to endure. When Aryabhata arrived at the Kapustin Yar cosmodrome, its constituent parts — bottom shell, instrumentation deck and top shell — were reassembled and carefully inspected. Soviet scientists meticulously checked the satellite's shock resistance, thermal cycles and vibration. To their surprise, it passed the tests with flying colours. Although they had previously worked on sounding rockets and small collaborative projects, nothing matched the scale or importance of this mission. The modest polyhedral satellite was about to redefine what a low-income country could accomplish. When the Kosmos-3M rocket roared to life, it carried not just circuitry but also the dreams of a nation not even 30 years free from colonial rule. As Aryabhata hurtled through layers of piercing cold Soviet air, a space programme destined to become the envy of the world, for its ability to operate on a shoestring budget was quietly born. Fifty years on, ISRO provides launch services to other low- and middle-income countries, nurturing the space ambitions of many African and Latin American nations. With private for-profit enterprises increasingly dominating the space industry, Aryabhata's legacy offers a valuable counterpoint. India's space programme showcases the value of public investment in science. Perhaps Aryabhata's more immeasurable impact, however, is the boost it has given to national confidence — inspiring a generation of scientists and engineers. Its purpose was humble but profound: to provide hands-on experience in designing, building and operating a spacecraft to a team of young Indian scientists. U. R. Rao, the project's director, had convinced ISRO's leadership that developing operational communications and remote-sensing satellites was impossible without building experimental ones first. Its main purpose: training a new generation of space technologists and validating home-grown hardware. Gandhi saw scientific modernity as an extension of India's civilizational legacy. The name carried deep symbolism, reaffirming India's rich history of scientific enquiry and linking ancient intellectual achievements to modern technological progress. With symbolism deeply woven into the project, success was crucial. That's why, when Aryabhata started tumbling soon after reaching orbit, the engineers who had clustered around consoles at what was then the Sriharikota Range ground station in India held their breath. The issue was traced to a faulty valve relay that failed to initiate the satellite's spin, which was crucial for orbital stability. Engineers on the ground sent a correction command and, over four tense days, Aryabhata gathered data on X-ray sources and ionospheric electrons. India's space scientists continue to expand their programme and inspire the next generation.Credit: Anshuman Poyrekar/Hindustan Times via Getty By day five of the mission, another snag was detected. A 9-volt power bus, which powered all three scientific experiments (X-ray astronomy, solar γ-rays and aeronomy), had failed. It was then decided to shut down the experiments and operate Aryabhata as a technological test platform. With this, India had not only launched but also controlled its first satellite in orbit. Evidence of star cluster migration and merger in dwarf galaxies Acoustic modes in M67 cluster stars trace deepening convective envelopes Light pollution threatens fleet of world-class telescopes in Atacama Desert How a PhD travel fellowship enriched an international cell-biology meeting CMLR's goal is to advance machine learning-related research across a wide range of disciplines. Westlake University ‘Frontiers of Life Sciences' International Undergraduate Summer School 2025 An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            