
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-16</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-04154-z'>A short-acting psychedelic intervention for major depressive disorder: a phase IIa randomized placebo-controlled trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 16:10:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Major depressive disorder (MDD) is a leading cause of disability worldwide, yet many patients have inadequate responses to current treatments. Dimethyltryptamine (DMT), a serotonergic psychedelic with rapid onset and short duration, shows promise as a potential antidepressant (AD), although clinical evidence in MDD remains limited. We conducted a phase IIa, double-blind, placebo-controlled, randomized clinical trial to evaluate the efficacy and safety of intravenous DMT (SPL026; DMT fumarate) in adults with moderate-to-severe MDD. Participants received a single 21.5-mg dose of DMT or placebo infused over 10 min, along with supportive psychotherapeutic support, followed by a 2-week assessment. At 2 weeks, the DMT group showed a significantly greater reduction in MADRS score than placebo (mean difference = −7.35; 95% CI = −13.62 to −1.08; P = 0.023). In the open-label phase, AD effects persisted up to 3 months, with no significant differences between those who received one versus two doses. Adverse events were mostly mild to moderate, commonly infusion site pain, nausea and transient anxiety. A single dose of DMT with psychotherapeutic support produced a rapid, significant reduction in depressive symptoms, sustained up to 3 months. The treatment was well-tolerated and safe. Major depressive disorder (MDD) is a leading cause of disability worldwide, substantially affecting individuals' quality of life and creating a substantial public health burden1,2. While selective serotonin reuptake inhibitors are commonly used as first-line treatments, many patients experience inadequate responses or unacceptable side effects3,4, emphasizing the need for innovative and more effective treatments. Psychedelics have emerged as a promising treatment for mood disorders5,6, with especially psilocybin gaining attention. Meta-analyses of randomized trials indicate that psilocybin produces moderate-to-large antidepressant (AD) effects compared to control (pooled g ≈ 0.66), with benefits emerging by days 8–15 and an optimal dose around 25 mg (refs. 7,8,9,10); these magnitudes exceed selective serotonin reuptake inhibitor meta-analytic averages (SMD ≈ g ≈ 0.30)11 and approach ketamine (intravenous (IV), SMD ≈ g ≈ 0.70)12, providing the empirical impetus to test whether similar rapid effects generalize across other classic 5-HT2A agonists, like dimethyltryptamine (N,N-DMT or DMT). The serotonin system, and its 5-HT2AR receptors, is implicated in mood regulation, cognition and sensory processing14. Unlike other serotonergic psychedelics, such as oral LSD and psilocybin, IV DMT has a short half life (~5 min) that enables shorter therapeutic sessions, potentially increasing convenience and reducing costs. Recent studies have provided initial evidence for DMT's AD potential. A placebo-controlled study in healthy psychedelic-experienced participants found that IV DMT (20 mg) administration as a bolus injection with psychological support during dosing day significantly reduced low-level subclinical depression severity and trait anxiety scores after 1 week. These outcomes correlated with peak subjective experiences15. Furthermore, an open-label phase I study investigated IV DMT (0.1 mg kg−1 followed by 0.3 mg kg−1) in MDD participants and healthy controls with minimal psychological support, consisting of a 45-min preparatory session and a debriefing session with the study psychiatrist after the drug effects were weaned off. This exploratory dose-escalation study showed that DMT was safe and well-tolerated, suggesting substantial reductions in depression scores observed in MDD participants the day after dosing. The study also confirmed DMT's rapid onset and short duration of subjective psychedelic effects when given as a bolus injection, making it a practical option for clinical use16. Subsequent studies in healthy volunteers have suggested that administering DMT through a slower IV infusion, as opposed to a rapid bolus, may be better tolerated and associated with fewer anxiety-related or adverse subjective effects10,17,18. This supports the rationale for exploring infusion-based approaches in clinical populations. In parallel, a recent open-label trial19 evaluated inhaled DMT with psychological support in individuals with treatment-resistant depression (TRD), further supporting the compound's therapeutic potential. Fourteen patients underwent a dose-escalation protocol (15 mg and 60 mg), showing rapid and sustained AD effects, with a mean Montgomery–Åsberg Depression Rating Scale (MADRS) reduction of 21.14 points by day 7. The study also confirmed the safety and tolerability of vaporized DMT, suggesting it as a promising, noninvasive alternative to IV administration in clinical settings. In this study, the psychological support element appeared to be more structured, conducted by both a psychotherapist and a psychiatrist, and focused on meaning-making, emotional or cognitive challenges from the acute phase, and promoting insights. Building on these findings, we conducted a two-stage phase IIa clinical trial evaluating the safety and efficacy of a 10-min IV DMT infusion in participants with moderate-to-severe MDD. The trial included a double-blind, placebo-controlled phase followed by an open-label phase (see Fig. 1 for schematics of study design and dosing schedule), with DMT (SPL026; DMT fumarate) administered in a controlled therapeutic environment supported by trained therapists. Please note that this trial is the second part of a two-part trial, the first part being a phase I dose-escalation study in healthy psychedelic-naïve participants, with results reported elsewhere10,20. Schematic overview of the two-stage phase IIa randomized, placebo-controlled trial evaluating IV DMT (SPL026) in adults with moderate-to-severe MDD. In stage 1 (double blind), participants were randomly assigned 1:1 to receive a single 21.5-mg DMT infusion (AA arm; n = 17) or placebo (PA arm; n = 17), each combined with supportive psychotherapy. In stage 2 (open label, 2 weeks later), the PA group received their first DMT dose, and the AA group received a second DMT dose. Across both sites, 399 signed a prescreening consent form and were prescreened. At this stage, 312 were excluded for not meeting criteria, 105 passed prescreening and of these 87 attended a formal screening visit. Of these, 47 participants did not meet inclusion criteria, declined to participate or were excluded for other reasons (Supplementary Results (Consort diagram) and Supplementary Fig. Thus, 34 participants were enrolled and underwent randomization. A total of 17 were assigned to the placebo–active (PA) group and 17 to the active–active (AA) group. Two participants withdrew from the trial. Additionally, four participants in AA group did not receive their second doses but remained in the trial (thus referred to as A−). More details can be found in Fig. For explanation, see Supplementary Results (Participant withdrawals). CONSORT diagram showing participant screening, randomization, dosing and follow-up. Four in the AA arm declined a second dose but completed follow-up. The diagram details attrition and numbers analyzed at each study stage. Each participant represents one independent biological replicate. CONSORT, Consolidated Standards of Reporting Trials. The study enrolled 34 participants with a mean age of 32.8 years (±9.1), of whom 29.4% were women, and the majority (88.2%) self-identified as white. Participants had experienced depression for an average of 10.5 years. At baseline, depression severity reflected similar levels of symptom intensity across groups. Eight participants were taking psychiatric medications at the time of screening, which were discontinued for at least 14 days (at least 28 days for monoamine oxidase inhibitors) before their first dosing session. A total of 11 participants reported prior psychedelic use, with all participants except two only experiencing low doses, not reporting any profound or immersive effects from the compounds. Overall, the demographic and clinical characteristics of the groups were comparable, with additional details provided in Table 1. The mean change from baseline in MADRS scores at 2 weeks after the first dose (the primary outcome) was significantly greater in participants receiving DMT fumarate compared to those receiving placebo and displayed a large effect size (mean difference = −7.35 points; 95% CI = −13.62 to −1.08; P = 0.023; d = 0.82). At 1 week (secondary outcome), the reduction in scores between the two groups was also significant (mean difference = −10.75 points; 95% CI = −16.95 to −4.55; P = 0.002; d = 1.09). In the open-label phase (stage 2), mixed model for repeated measures (MMRM) analyses revealed no significant differences in MADRS scores among participants receiving one dose of DMT fumarate (PA group) and those receiving two doses (AA group) at any follow-up time points. At trial week 3 (day 22, 1 week after the second dose), the mean difference between the PA and AA groups was −3.26 points (95% CI = −11.34 to 4.81; P = 0.42). Similar results were observed at week 4 (day 29 = −3.26 points; 95% CI = −11.10 to 4.59; P = 0.41), week 6 (day 45 = 0.16 points; 95% CI = −7.68 to 8.00; P = 0.96) and week 14 (day 105 = 7.03 points; 95% CI = −1.04 to 15.11; P = 0.08; Fig. 3, Supplementary Results (Secondary analysis—MADRS score comparison between PA and AA groups in stage) and Supplementary Table 4). In stage 1 (double blind), participants who received DMT (AA, n = 17) showed a significantly greater reduction in depressive symptoms at 2 weeks compared with placebo (PA, n = 17; mean difference = −7.35 points; 95% CI = −13.62 to −1.08; P = 0.023, two-sample t test). During stage 2 (open label), both groups maintained improved scores up to 3 months. Analyses of the double-blinded phase used an independent samples t test, and analyses of the open-label phase used an MMRM. In participants who received two DMT doses (AA participants in stages 1 and 2), most clinical improvements in MADRS scores were observed within the initial 2 weeks after the first dose, with no significant additional effect observed after the second dose (Supplementary Results (Supplementary analysis—AA group within-participant comparison of two DMT doses) and Supplementary Table 5). Similarly, the timing of the first dose (stage 1 or stage 2) did not significantly influence the reductions in MADRS scores post-DMT (Supplementary Results (Supplementary analysis—timing of the first DMT dose: stage 1 vs stage 2) and Supplementary Table 6). At 1 week, an MADRS response was observed in 6% PA participants and 44% AA participants (difference = 37.5 percentage points; 95% CI = 10.5 to 64.6), while remission was achieved in 13% and 44%, respectively (difference = 31.3 percentage points; 95% CI = 2.0 to 60.5). At 2 weeks, response rates were 12% for PA and 35% for AA (difference = 23.5 percentage points; 95% CI = −3.9 to 50.9), and remission rates were 12% and 29%, respectively (difference = 17.7 percentage points; 95% CI = −8.9 to 44.2). Sustained response and remission rates during the open-label phase remained stable over time, with no statistically significant differences between groups at any stage 2 time point (remission rates can be found in Supplementary Results (Supplementary analysis—MADRS remission rates) and Supplementary Table 7). Remission and response rates at 6 months for all pooled participants (one or two doses) were 40% and 44%, respectively. Descriptive representations of Beck Depression Inventory-II (BDI-II) and State-Trait Anxiety Inventory (STAI-T) outcome data, which trended in line with the MADRS data, can be found in Supplementary Results (Secondary analysis—all BDI and STAI-T data) and Supplementary Table 3. The treatment was overall well-tolerated in male and female participants with MDD who received one or two IV doses of DMT fumarate. Treatment-emergent adverse events (TEAEs) occurred in the majority of participants, with 25 individuals (73.5%) reporting 47 TEAEs deemed possibly related to treatment by the investigator. The severity of these TEAEs was largely mild, observed in 15 participants, while 10 participants experienced moderate TEAEs. They mostly resolved over the dosing visit. The incidence of TEAEs was higher after the first dose of DMT fumarate (64.7% in stage 1 and 62.5% in stage 2) compared to placebo (23.5%). Interestingly, a second dose of DMT fumarate in stage 2 in the AA group was associated with a lower incidence of treatment-related TEAEs (23.1%), a proportion similar to that seen in the placebo group (Table 3). Notably, there were no serious adverse events related to treatment, no deaths or TEAEs that necessitated withdrawal from the study, dose discontinuation or adjustment. A table listing all TEAEs (including those deemed not related to treatment) can be found in Supplementary Results (Safety results) and Supplementary Table 14. Clinical evaluations, including physical examinations, laboratory analyses and electrocardiograms (ECGs), revealed no clinically significant abnormalities throughout the trial. However, transient and discrete increases in blood pressure and heart rate were observed immediately after DMT administration (Supplementary Results (Safety results)). All participants tolerated the dosing sessions well, except for one individual who expressed regret after their second dose (Supplementary Results (Safety results)). Throughout the study, no meaningful or concerning changes in suicidal ideation were recorded based on the Beck Scale for Suicidal Ideation (Supplementary Results (Safety results) and Supplementary Table 18). AD effects 2 weeks after participants' first active DMT dose were observed to be moderated by their ‘Mystical Experience' Questionnaire (MEQ) scores, as well as Ego Dissolution Inventory, feelings of Unity (11 Dimensions Altered States of Consciousness questionnaire) and subject-led visual analog scales (Individual-Rated Visual Analog Scales) general intensity scores (Supplementary Results (Supplementary materials—acute measures and moderation effects of Mystical Experience (MEQ) scores and related measures of the acute psychedelic experience)). We also explored whether AD washout, prior psychedelic use or psychedelic predictor scale subscales (set, setting, rapport, intention) moderated treatment outcomes. None of these factors showed a significant effect on AD response at 2 weeks. Full results and statistical tables are provided in Supplementary Results (Supplementary analyses on possible effects of antidepressant washout, prior experience with psychedelics and preparation (set, setting, therapeutic rapport and intention) on antidepressant effects). Given the small subgroup sizes and limited power, these findings should be interpreted with caution. This phase IIa trial evaluated the efficacy, safety and tolerability of one or two 21.5-mg IV 10-min infusions of DMT fumarate with supportive therapy in participants with moderate-to-severe MDD. Participants were randomized to receive either DMT or placebo in a blinded phase, followed 2 weeks later by an open-label phase where all participants could have received active treatment. The primary endpoint, change in MADRS scores from baseline to 2 weeks, demonstrated a significant improvement after DMT treatment compared with placebo, confirming DMT's superior AD efficacy to placebo in the blinded phase of the trial. Thus, the trial's main endpoint was met. Secondary analyses showed a rapid reduction in MADRS scores by 1 week, and concomitant improvements in response and remission rates generally aligned with the primary outcome. In the open-label phase, the AD effects were durable over 12 weeks (and up to 6 months in some participants), with no notable differences between single-dose and two-dose regimens, suggesting that one dose may suffice for sustained benefits. However, this longer-term difference did not reach statistical significance (P = 0.08). This may be due in part to reduced statistical power in part B, where participant dropouts exceeded 10% (9 of 34 participants), limiting the ability to detect between-group differences at later time points. Extended therapist–participant interaction may have facilitated a stronger therapeutic alliance, which is increasingly recognized as a key contributor to outcomes in psychedelic-assisted therapy21. While this remains speculative, the trend observed here highlights the need for future studies that both minimize attrition and directly measure therapeutic alliance to better understand its role in shaping long-term treatment effects. The absence of formal measures of therapeutic rapport or alliance in the present study also limits our ability to evaluate this hypothesis directly and should be addressed in future research. DMT was well-tolerated, with no serious adverse events or instances of suicidal ideation or suicidal behavior deemed related to the study drug reported. Most adverse events were mild or moderate, and the most common was infusion site pain. A second dose was associated with fewer adverse events than the first, and most resolved during the dosing visit. No substantial safety concerns were identified in laboratory findings, ECG or vital signs, supporting the overall tolerability of the treatment. Unlike psilocybin, which has been associated with postdose headaches in 40–50% participants in previous studies22, DMT did not appear to reliably induce headaches in this study. The between-group effect size for the reduction in depressive symptoms at 2 weeks in the present study (d = 0.82) is comparable to the effect size observed in a previous trial of psilocybin (25 mg versus niacin) with psychological support in MDD participants (d = 0.90)23 and greater than the 3-week between-group effect size of psilocybin (25 mg versus 1 mg) in TRD (d = 0.55)24. Similar to other psychedelic compounds, DMT's AD effects may be linked to its ability to promote neuroplasticity25, its transient—but substantial—impact in brain function in associative regions15,26,27 or (relatedly) to its subjective effects15, with the present results suggesting that reductions in depression were moderated by the intensity of mystical-type experiences induced during treatment (Supplementary Results (Supplementary materials—acute measures and moderation effects of Mystical Experience (MEQ) scores and related measures of the acute psychedelic experience)). Due to its shorter duration of action, IV DMT treatment could offer a more time-efficient, practical and scalable option for psychedelic-based treatments. A recent open-label phase IIa trial using inhaled DMT (escalating 15 mg and then 60 mg) in TRD also reported rapid and sustained AD effects over a 3-month period, with large within-subject effect sizes (for example, g = 2.20 at day 7, g = 1.33 at 3 months)19. While methodological differences make direct comparisons difficult, variations in dose, route of administration and study design, including the use of a placebo control in the present study, may partly account for the somewhat smaller between-group effect size observed at 2 weeks (d = 0.82). Together, these studies highlight the importance of further controlled research to clarify the role of dosing, setting and delivery methods in shaping clinical outcomes. Participants with a history of serious suicide attempts were excluded. Additionally, we did not assess blinding integrity or participant expectancy, which may have influenced outcomes, particularly given the distinct subjective effects of DMT. If blinding is compromised, expectations about improvement may influence outcomes, making it harder to isolate pharmacological effects from psychological or contextual ones (although see ref. Neither factor showed an effect, although the small subgroup sizes limit the strength of these conclusions (Supplementary Results (Supplementary analyses on possible effects of antidepressant washout, prior experience with psychedelics and preparation (set, setting, therapeutic rapport and intention) on antidepressant effects)). The results of this second phase of the study must thus be interpreted as hypothesis-generating. DMT was administered alongside a time-limited, relational psychotherapeutic framework focused on psychological flexibility. Given the hypothesis of synergistic effects between psychedelic compounds and therapeutic support29, it remains unclear whether similar results would be achieved with minimal psychological support. Formal measures on the quality of psychological integration were also not collected, limiting our capacity to draw conclusions on the influence of the psychological intervention on clinical outcomes. In this phase IIa trial, a 21.5-mg dose of DMT administered as a 10-min IV infusion with psychological support resulted in a significantly greater reduction in depression at 2 weeks compared with placebo in participants with moderate-to-severe MDD. The treatment demonstrated a rapid onset of effect, with improvements observed as early as the first assessment time point of 1 week, and AD effects were sustained during the 12-week open-label phase. Longer and larger trials, including comparisons with existing treatments, are needed to further evaluate the efficacy, safety and cost-effectiveness of DMT in the treatment of MDD. This was a phase IIa double-blind, placebo-controlled, randomized clinical trial. The sponsor, Small Pharma (now Cybin UK), designed and supported the trial and provided a proprietary pharmaceutical-grade formulation of DMT fumarate (SPL026), which was analyzed for stability and purity. The trial evaluated the efficacy and safety of DMT, administered as a single or double IV dose, to participants with MDD. The trial was conducted at Hammersmith Medicines Research (HMR), MAC Clinical Research (MAC) and Imperial College London Hammersmith Campus under their respective standard operating procedures. HMR supervised the trial's execution, and independent site raters conducted blinded assessments of participants using validated tools, ensuring raters were unaware of trial-group assignments. Data management and analysis were performed by HMR, reviewed by the sponsor and supplemented with post hoc statistical analyses conducted by the sponsor. Small Pharma, as Sponsor, had full oversight of the clinical trial. Data were collected via an electronic case report form using the Medrio EDC system (latest version at the time of use, 2020–2022). All participants provided written informed consent. The trial is registered on ClinicalTrials.gov (NCT04673383) and ISRCTN (ISRCTN63465876), where the clinical study report synopsis is available. Adults aged 18 years or over with a diagnosis of moderate-to-severe MDD and a history of at least two prior unsuccessful treatment attempts (pharmacological or/and psychotherapeutic) were recruited formally through databases held by the clinical trial sites, informally through social media, and through other sources, which directed participants to a recruitment website. The main exclusion criteria were an immediate family or personal history of psychosis, medically significant health conditions that make a person unsuitable to participate in the trial (as assessed by a physician), a history of serious suicide attempts, a positive pregnancy test, the use of serotonergic psychedelic drugs in the previous 6 months or the suspected or known presence of a pre-existing psychiatric condition (for example, borderline personality disorder) that could jeopardize rapport between the participant and their two trial therapists in light of the protocol-defined limits of available therapist support. Additional details about the trial exclusion criteria are provided in Supplementary Methods (Inclusion and exclusion criteria). Most of the recruited participants referred themselves and some were recruited through patient databases. Confirmation of a diagnosis of depression and medical history were obtained from the participant's general physician. Eligible participants then underwent face-to-face physical and mental health assessments (including the Mini-International Neuropsychiatric Interview) with a trial physician, psychiatrist and therapist (visit 1). Assessment with the 17-item Hamilton Depression Scale (HAM-D-17) was done by the site psychiatrist; a score of at least 17 (indicating moderate-to-severe MDD) on a scale that ranges from 0 to 52, with higher scores indicating greater depression, was required for trial enrollment. Any participants taking AD medication at screening discontinued the use before starting the trial (with the support of a study psychiatrist), with full discontinuation occurring at least 2 weeks before starting a trial medication. Participants were financially compensated for their time. The tables use the term sex rather than gender, but this was determined by self-reporting and may reflect gender identity rather than sex assigned at birth. Sex-based/gender-based analyses have not been performed due to overall low sample size. The race/ethnicity categories used followed the FDA's recommendations which are as follows: American Indian or Alaska Native; Asian; Black or African American; Native Hawaiian or other Pacific Islander; and White. Participants were allowed to self-report and could also designate themselves as multiracial. Ethnicity was separated from race, with the categories—Hispanic/Latino or not Hispanic/Latino. These categories were not used for any inferences or analyses. The sponsor-initiated trial reported here constitutes part B of a larger study investigating the safety, tolerability and efficacy of DMT fumarate in healthy participants (part A) and participants with MDD (part B). Findings from part A, conducted in healthy participants, have been published previously10,20 and the remainder of this manuscript refers to part B only. Enrolled participants with MDD followed a two-stage, double-blind, randomized, placebo-controlled design, with a planned enrollment of 28 to 36 participants (34 were randomized). Participants received up to two single IV doses of DMT fumarate or placebo accompanied by a time-limited, relational psychotherapeutic framework focused on psychological flexibility (Supplementary Methods (Therapeutic support)). In stage 1, participant numbers were allocated to blinded treatments (active or placebo) according to a 1:1 randomization schedule prepared by an independent HMR statistician, using SAS statistical analysis software. Two weeks later, in stage 2, participants received a second, open label, dose of DMT, either as their first exposure for those who received placebo in stage 1, defined as PA arm (PA group) or as a second dose for those who received DMT in stage 1, defined as AA arm (AA group). Participants who declined a second dose or were deemed unsuitable for further dosing by the study psychiatrist proceeded to follow-up assessments after their first dose (defined as A- arm). Due to the noticeable psychedelic effects of DMT fumarate, it was likely that both participants and investigators could determine whether a participant had received the active drug or placebo. Despite this, stage 1 of the study adhered to double-blind principles, while stage 2 was conducted as an open-label phase. The dosing regimen was based on data from part A and selected for safety and tolerability. The dose administered was 21.5-mg DMT fumarate, infused intravenously over 10 min in two phases—6 mg over the first 5 min, followed by 15.5 mg over the subsequent 5 min (see refs. Details of the screening process can be found in Supplementary Methods (Trial visits). Dosing took place in a controlled inpatient setting, with participants admitted to the ward 1 day prior and discharged the following morning. Preparation involved a 90-min session with two therapists the day before dosing, including orientation to the setting, a guided visualization to promote relaxation and interpersonal safety, and discussion of participant expectations. On dosing day, an additional brief session supported emotional readiness and anticipatory anxiety. Participants received IV DMT in a nonclinical, softly lit room designed to promote calm and containment. They wore eyeshades and noise-canceling headphones and listened to a specially curated, nonverbal ambient music playlist produced by M. Cooper. Two therapists were present throughout the session to provide silent, attentive support, while a study psychiatrist remained on site. Integration was delivered using a relational and participant-led framework developed from previous psilocybin trials. This approach encourages open exploration of emotional, embodied and transpersonal elements of the experience, without prematurely fixing meaning. Integration sessions were conducted in person on day 1 and day 2 postdose, and again on day 15, with therapists helping participants reflect on the psychedelic material and explore how insights could be applied to daily life. Additional follow-up check-ins supported ongoing integration and monitored well-being. Full procedural details for trial visits and the therapeutic framework are provided in the Supplementary Methods (Trial visits and Therapeutic support). Remote follow-up assessments were conducted 1-week postdose, an in-person visit at day 29 and remote visits 1 month, and 3 months after the final dose. MADRS was assessed (using the SIGMA guide) by a trained independent site rater who was not present during dosing. An overview of the trial is shown in Fig. In addition to the main trial, an exploratory out-of-study 6-month follow-up visit was performed to monitor long-term outcomes (Supplementary Methods (Trial visits)). The sample size calculation was based on a two-sided, two-sample t test with equal variance at a significance level of 0.05 and a 1:1 allocation ratio, using data from ref. Secondary efficacy measures included MADRS scores assessed at 1 week (day 8; ±1 day) after the first dose and at 1, 2, 4 and 12 weeks after the second dose (day 22; ±1 day, 29; ±2 days, 45; ±2 days and 105; ±5 days). These instruments were used to evaluate broader aspects of mood and anxiety that complement the primary depression-related outcome. Participants who did not receive a second dose but remained in the study completed the same follow-up assessments as other participants. The BDI-II and STAI-T were administered in paper form for self-assessment. MADRS was completed on a paper form by an independent site rater after the SIGMA guide34. The MEQ35, as well as other subjective measures like Ego Dissolution Inventory36, Emotional Breakthrough Inventory37, 11 Dimensions Altered States of Consciousness38, Psychological Insights Scale39, Challenging Experience Questionnaire40 and Individual-Rated Visual Analog Scales, were used to measure the subjective effects of DMT. These subjective scores were also examined to see if they could predict subsequent AD effects. We also examined whether elements of psychedelic preparation (therapeutic rapport, intentions, set and setting) influenced AD response at 2 weeks using the psychedelic predictor scale41. Exploratory analyses also assessed for possible influence of AD washout and prior psychedelic use on AD effects at 2 weeks. A completed list of the measures collected in this trial, including the ones not reported here, can be found in Supplementary Methods (Complete list of collected measures) and Supplementary Table 1. Adverse events were recorded throughout the study and categorized by severity (mild, moderate, severe) and relationship to treatment (possibly or unlikely related). Serious adverse events were defined per regulatory guidelines and managed by the sponsor. Additional details on adverse event collection are provided in the Supplementary Methods (Additional adverse event collection methods). Tolerability was assessed postdose by asking if participants regretted the experience. Injection site assessments recorded pain, tenderness, erythema or induration. Secondary endpoints, including sustained effects of DMT and the potential benefit of an additional dose, were analyzed using an MMRM. The first MMRM analysis compared MADRS score changes at 1 week (day 22), 2 weeks (day 29), 1 month (day 45) and 3 months (day 105) during the open-label phase (stage 2) between participants who received a single DMT dose (PA group) and those who received two doses (AA group). Additional MMRM analyses explored within-participant changes in MADRS scores after each dose in the AA group and assessed whether the timing of the first DMT dose influenced clinical outcomes (Supplementary Results (‘Secondary analysis—MADRS score comparison between PA and AA groups in stage' to ‘Supplementary analysis—MADRS remission rates')). Response and remission rates, defined as >50% reduction in MADRS score and an MADRS score ≤10, respectively, were analyzed using logistic regression adjusted for baseline scores. Changes in BDI-II and STAI-T scores and MADRS scores at the 6-month follow-up (day 224) were summarized descriptively. Moderation effects of the acute psychedelic experience, measured by MEQ scores, on depression severity were explored (Supplementary Results (Supplementary materials—acute measures and moderation effects of Mystical Experience (MEQ) scores and related measures of the acute psychedelic experience) and Supplementary Fig. In response to reviewer feedback, a worst-case scenario sensitivity analysis was conducted to evaluate robustness against potential nonrandom missingness of data in the open-label phase. All analyses, excluding the moderation effects, were done after the statistical analysis plan had been signed and after the database had been locked. The statistical analyses were done by an HMR statistician, using SAS (version 9.4; moderation effects were analyzed by Imperial College using R, the latest version at the time of analysis). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The clinical trial data underlying this article, including protocol and statistical analysis plan, are not publicly available due to privacy concerns and restrictions from the trial sponsor (Cybin IRL). In line with ICMJE guidelines, de-identified participant data will be made available upon reasonable request to the sponsor for the purposes of verifying the results within 1 month of execution of a data-sharing agreement. Access will be granted to qualified researchers following review and subject to a data-sharing agreement to ensure participant confidentiality and compliance with regulatory requirements. Addressing the side effects of contemporary antidepressant drugs: a comprehensive review. Cosci, F., Mansueto, G. & Fava, G. A. Relapse prevention in recurrent major depressive disorder. A comparison of different treatment options based on clinical experience and a critical review of the literature. Nutt, D., Erritzoe, D. & Carhart-Harris, R. Psychedelic psychiatry's brave new world. Erritzoe, D. et al. Effect of psilocybin versus escitalopram on depression symptom severity in patients with moderate-to-severe major depressive disorder: observational 6-month follow-up of a phase 2, double-blind, randomised, controlled trial. Swieczkowski, D. et al. Efficacy and safety of psilocybin in the treatment of major depressive disorder (MDD): a dose–response network meta-analysis of randomized placebo-controlled clinical trials. James, E. et al. Safety, tolerability, pharmacodynamic and wellbeing effects of SPL026 (dimethyltryptamine fumarate) in healthy participants: a randomized, placebo-controlled phase 1 trial. Comparative efficacy and acceptability of 21 antidepressant drugs for the acute treatment of adults with major depressive disorder: a systematic review and network meta-analysis. Carbonaro, T. M. & Gatch, M. B. Neuropharmacology of N,N-dimethyltryptamine. Carhart-Harris, R. L. & Nutt, D. J. Serotonin and brain function: a tale of two receptors. Effects of DMT on mental health outcomes in healthy volunteers. D'Souza, D. C. et al. Exploratory study of the dose-related safety, tolerability, and efficacy of dimethyltryptamine (DMT) in healthy volunteers and major depressive disorder. Acute dose-dependent effects and self-guided titration of continuous N,N-dimethyltryptamine infusions in a double-blind placebo-controlled study in healthy participants. Rapid and sustained antidepressant effects of vaporized N,N-dimethyltryptamine: a phase 2a clinical trial in treatment-resistant depression. Good, M. et al. Pharmacokinetics of N,N-dimethyltryptamine in humans. Therapeutic alliance and rapport modulate responses to psilocybin assisted therapy for depression. Nutt, D. J. et al. Psilocybin and other classic psychedelics in depression. Raison, C. L. et al. Single-dose psilocybin treatment for major depressive disorder: a randomized clinical trial. Goodwin, G. M. et al. Single-dose psilocybin for a treatment-resistant episode of major depression. Ly, C. et al. Psychedelics promote structural and functional neural plasticity. Timmermann, C. et al. Human brain effects of DMT assessed via EEG-fMRI. Alamia, A. et al. DMT alters cortical travelling waves. Szigeti, B. et al. Assessing expectancy and suggestibility in a trial of escitalopram v. psilocybin for depression. Treatment with psychedelics is psychotherapy: beyond reductionism. Rapid antidepressant effects of the psychedelic ayahuasca in treatment-resistant depression: a randomized placebo-controlled trial. & Brown, G. K. Beck Depression Inventory (Psychological Corporation, 1996). Development and reliability of a structured interview guide for the Montgomery–Åsberg Depression Rating Scale (SIGMA). Barrett, F. S., Johnson, M. W. & Griffiths, R. R. Validation of the revised Mystical Experience Questionnaire in experimental sessions with psilocybin. Peill, J. M. et al. Validation of the psychological insight scale: a new scale to assess psychological insight following a psychedelic experience. Angyus, M. et al. Validation of the imperial psychedelic predictor scale. Their time, trust and willingness to engage with the research were essential to its completion. The authors thank F. Devlin, L. MacDonald, L. Pellegrini (University of Trieste) and D. Gregory for their contributions as trial psychiatrists and for their assistance with data collection. The authors also acknowledge the clinical trial teams at HMR in London and MAC in Liverpool for their dedicated operational support throughout the study. The authors are indebted to several consultants whose insights and guidance enriched the therapeutic framework of the trial, including R. Watts, T. Read, L. Luan and M. Cooper, who provided musical contributions to the dosing sessions. Laidler, E. Lou, A. Phillips, R. Boni, J. Corrie, G. von Lob, H. Rimmer, K. Palmer, K. Egerton, S. Sharp, J. Small Pharma acted as trial sponsor, having the overall legal responsibility for trial's initiation, management, financing and quality, ensuring safety and well-being of participants and compliance with all regulatory standards and approvals. These authors jointly supervised this work: Carol Routledge, Ellen James. Tiffanie Benway, Graham Campbell, Peter Rands & Carol Routledge Zelah Joel, Meghan Good, Marie Layzell & Ellen James Center for Psychedelic and Consciousness Research, Department of Psychiatry and Behavioral Sciences, Johns Hopkins School of Medicine, Baltimore, MD, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar (follow-up time points) performed the clinical phase and collected data. Correspondence to David Erritzoe, Tommaso Barba or Ellen James. serves as a paid scientific advisor for Aya Biosciences, Lophora Aps, Clerkenwell Health and Mindstate Design Lab. has no current affiliations with companies developing psychedelic treatments for depression, although he previously advised COMPASS Pathways. He currently advises Algernon Pharmaceuticals on the nonpsychedelic application of DMT for stroke and Neural Therapeutics on mescaline research. was employed by Small Pharma at the time of the trial (fully seconded to HMR). R.C.-H. is a scientific advisor to TRYP Therapeutics, MindState and Otsuka. T. Benway was employed by Small Pharma during the trial and is a shareholder; she is now employed by GH Research. were employed by Small Pharma and are now employed by Cybin UK/IRL, trading as Helus/CybinIRL, and are option holders/shareholders in Cybin. The other authors declare no competing interests. Nature Medicine thanks Hongyuan Cao, Matthias Liechti, Johannes Ramaekers and Joshua Rosenblat for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Erritzoe, D., Barba, T., Benway, T. et al. A short-acting psychedelic intervention for major depressive disorder: a phase IIa randomized placebo-controlled trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00473-4'>‘What are we doing here?' The polymaths who searched for the meaning of life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 13:35:37
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Urmila Chadayammuri is an associate editor at Nature based in Berlin. Chemist Dorothy Hodgkin mapped the structures of molecules using X-ray crystallography. Against the odds: 12 women who beat bias to succeed in science Against the odds: 12 women who beat bias to succeed in science In Traversal, this incredibly interdisciplinary exploration of knowledge and meaning gets to grow across 600 pages. Whether she is writing about colonialist explorer James Cook, crystallographer Dorothy Hodgkin or novelist Mary Shelley, about poetry, abolitionism or paint, Popova has a way of weaving one story into the next as if the boundaries between disciplines, cultures and centuries do not exist. Marie Tharp, who discovered the Mid-Atlantic Ridge, appears as both oceanographer and violinist. Against the odds: 12 women who beat bias to succeed in science How to wow a popular-science writer with your research expertise Why we don't really know what the public thinks about science Tree rings and salt lakes give clues about ancient rainfall Join HZAU's global faculty team to advance research with competitive benefits. Against the odds: 12 women who beat bias to succeed in science An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225602.htm'>One in three people carry this brain parasite but the body has a kill switch</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 13:34:41
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A parasite that may already be living in your brain has the ability to infect the very immune cells designed to eliminate it. New research from UVA Health explains how the body still manages to keep this infection under control. Toxoplasma gondii is a potentially dangerous parasite that infects warm blooded animals. Roughly one third of the global population is believed to carry Toxoplasma, yet most people never develop symptoms. When illness does occur, known as toxoplasmosis, it is most serious in individuals with weakened immune systems. "Understanding how the immune system fights Toxoplasma is important for several reasons. People with compromised immune systems are vulnerable to this infection, and now we have a better understanding of why and how we can help patients fight this infection." Harris and her team discovered that CD8+ T cells rely on a powerful enzyme called caspase-8 to control T. gondii. This happened even though both groups mounted strong immune responses against the infection. Mice with caspase-8 remained healthy, while those without it became severely ill and died. These findings indicate that caspase-8 plays a crucial role in limiting T. gondii inside T cells. "We scoured the scientific literature to find examples of pathogens infecting T cells. The only pathogens that can live in CD8+ T cells have developed ways to mess with Caspase-8 function. Prior to our study, we had no idea that Caspase-8 was so important for protecting the brain from Toxoplasma." The research team included Lydia A. Sibley, Maureen N. Cowan, Abigail G. Kelly, NaaDedee A. Amadi, Isaac W. Babcock, Sydney A. Labuzan, Michael A. Kovacs, Samantha J. Batista, John R. Lukens and Harris. Funding for the research came from the National Institutes of Health, grants R01NS112516, R01NS134747, R21NS12855, T32GM008715, T32AI007496, T32AI007046, T32NS115657, F30AI154740, T32AI007496 and T32GM007267; a University of Virginia Pinn Scholars Award; a UVA Shannon Fellowship; and UVA's Strategic Investment Fund. Intermittent Fasting Fails to Deliver in Major Weight Loss Review Could Sugary Drinks Be Fueling the Rise in Teen Anxiety? Scientists Crack a 50-Year Tissue Mystery With Major Cancer Implications Hard-to-Treat Cancers Have a Secret Trick That Researchers Finally Cracked Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-026-69690-9'>Microbiome-mediated colonization resistance to and countermeasures of Klebsiella pneumoniae</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 13:01:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications , Article number:  (2026) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Klebsiella pneumoniae intestinal colonization contributes to infectious and non-infectious diseases. Recent studies have uncovered the complex interplay between the gut microbiota and K. pneumoniae colonization, highlighting its role in disease development and progression and illustrating the potential of microbiome-based therapies. Naghavi, M. et al. Global burden of bacterial antimicrobial resistance 1990-2021: a systematic analysis with forecasts to 2050. Nguyen, T. N. T., Howells, G. & Short, F. L. How Klebsiella pneumoniae controls its virulence. Federici, S. et al. Targeted suppression of human IBD-associated gut microbiota commensals by phage consortia for treatment of intestinal inflammation. Yuan, J. et al. Fatty liver disease caused by high-alcohol-producing Klebsiella pneumoniae. Wang, X. et al. Gut-liver translocation of pathogen Klebsiella pneumoniae promotes hepatocellular carcinoma in mice. Xue, G. et al. Three Klebsiella species as potential pathobionts generating endogenous ethanol in a clinical cohort of patients with auto-brewery syndrome: a case control study. Tamburini, F. B. et al. Precision identification of diverse bloodstream pathogens in the gut microbiome. Woelfel, S., Silva, M. S. & Stecher, B. Intestinal colonization resistance in the context of environmental, host, and microbial determinants. Cell Host Microbe 32, 820–836 (2024). Spragge, F. et al. Microbiome diversity protects against pathogens by nutrient blocking. Furuichi, M. et al. Commensal consortia decolonize Enterobacteriaceae via ecological control. Yang, J. et al. Microbiome-mediated colonization resistance to carbapenem-resistant Klebsiella pneumoniae in ICU patients. NPJ Biofilms Microbiomes 11, 157 (2025). Kang, J. T. L. et al. Long-term ecological and evolutionary dynamics in the gut microbiomes of carbapenemase-producing Enterobacteriaceae colonized subjects. Almasi, E. D. H. et al. Klebsiella oxytoca facilitates microbiome recovery via antibiotic degradation and restores colonization resistance in a diet-dependent manner. Osbelt, L. et al. Klebsiella oxytoca causes colonization resistance against multidrug-resistant K. pneumoniae in the gut via cooperative carbohydrate competition. Cell Host Microbe 29, 1663–1679 e1667 (2021). Hecht, A. L. et al. Dietary carbohydrates regulate intestinal colonization and dissemination of Klebsiella pneumoniae. Yang, J. et al. Klebsiella oxytoca complex: update on taxonomy, antimicrobial resistance, and virulence. Djukovic, A. et al. Lactobacillus supports Clostridiales to restrict gut colonization by multidrug-resistant Enterobacteriaceae. Mortzfeld, B. M. et al. Microcin MccI47 selectively inhibits enteric bacteria and reduces carbapenem-resistant Klebsiella pneumoniae colonization in vivo when administered via an engineered live biotherapeutic. Gut Microbes 14, 2127633 (2022). Kienesberger, S. et al. Enterotoxin tilimycin from gut-resident Klebsiella promotes mutational evolution and antibiotic resistance in mice. Bray, A. S. et al. Klebsiella pneumoniae employs a type VI secretion system to overcome microbiota-mediated colonization resistance. Bray, A. S. & Zafar, M. A. Deciphering the gastrointestinal carriage of Klebsiella pneumoniae. Li, D. et al. Cpx-mediated amino acid sensing diversifies gastrointestinal colonization of Klebsiella pneumoniae. Teo, T. H. et al. Differential mucosal tropism and dissemination of classical and hypervirulent Klebsiella pneumoniae infection. Russo, T. A. & Marr, C. M. Hypervirulent Klebsiella pneumoniae. Microbiol Rev. Ikhimiukor, O. O. et al. Clonal background and routes of plasmid transmission underlie antimicrobial resistance features of bloodstream Klebsiella pneumoniae. Wang, R. et al. Increase in antioxidant capacity associated with the successful subclone of hypervirulent carbapenem-resistant Klebsiella pneumoniae ST11-KL64. Heinz, E. et al. Longitudinal analysis within one hospital in sub-Saharan Africa over 20 years reveals repeated replacements of dominant clones of Klebsiella pneumoniae and stresses the importance to include temporal patterns for vaccine design considerations. Genome Med. Zaborskyte, G., Hjort, K., Lytsy, B. & Sandegren, L. Parallel within-host evolution alters virulence factors in an opportunistic Klebsiella pneumoniae during a hospital outbreak. Tisza, M. J. et al. Longitudinal phage-bacteria dynamics in the early life gut microbiome. Fang, Q. et al. Safety and efficacy of phage application in bacterial decolonisation: a systematic review. Lancet Microbe 5, e489–e499 (2024). Download references This work is supported by the National Key Research and Development Program of China (grant no. ), the National Natural Science Foundation of China (grant no. ), and the Medical Research Council (grant no. Center of Infectious Diseases, West China Hospital, Sichuan University, Chengdu, China Yongqiang Yang & Zhiyong Zong Department of Microbes, Infection and Microbiomes, School of Infection, Inflammation and Immunology, College of Medicine and Health, University of Birmingham, Birmingham, UK Willem van Schaik & Alan McNally Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Y.Y., Z.Z., W.v.S., and A.M. conceived the comment. drafted the manuscript. and A.M. discussed and revised the manuscript. Correspondence to Zhiyong Zong. The authors declare no competing interests. Nature Communications thanks Éva Almási, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Yang, Y., van Schaik, W., McNally, A. et al. Microbiome-mediated colonization resistance to and countermeasures of Klebsiella pneumoniae. Nat Commun  (2026). Download citation Received: 31 October 2025 Published: 16 February 2026 DOI: https://doi.org/10.1038/s41467-026-69690-9 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Communications (Nat Commun) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/the-mathematical-mystery-inside-the-legendary-90s-shooter-quake-3/'>The mathematical mystery inside the legendary '90s shooter Quake 3</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Deep within the source code of this online multiplayer game lies an enigmatic number that puzzles and inspires experts to this day Game developers didn't have it easy in the 1990s. It had a lasting impact on the industry. This legacy wasn't so much due to the story, but rather because Quake 3 was one of the first multiplayer first-person shooters. Players could connect their computers via network cables or the internet to compete in real time. It included an extremely efficient algorithm that still amazes experts and sparks curiosity among scientists. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. To figure out the orientations of objects, characters or other players in three-dimensional space mathematically, you create a vector, which is essentially an arrow that shows direction. And that's where a tricky calculation comes up: the inverse square root, which is one divided by the square root of a number. If I asked you to calculate the inverse square root of 26 without a calculator, you'd probably be stuck for a while—and honestly, so would I. That's why the Quake 3 programmers hunted for a better way to find this inverse square root. And indeed, their source code revealed an ingenious solution. What's fascinating is that the developers never advertised their trick. But once it was released, curious enthusiasts took notice. But gradually people figured out how the code worked. Today there are many tutorials that guide you step-by-step through the program code. These walkthroughs exploit special features of the C programming language. This is a clever way to avoid computationally intensive operations like division. From a mathematical perspective, the code is easily explained. To determine the inverse square root, you first make a guess at the solution (which is generally incorrect) and then refine that estimate through a set procedure. In this way, it gradually reaches better solutions. This process requires a lot of computing power. This may sound counterintuitive at first, since one wants to calculate the inverse square root and not just any zero. Through Newton-Raphson's method, the error thus becomes progressively smaller, allowing one to get ever closer to the exact solution. The algorithm starts with a certain guess: let's say 3.1. If 3.1 were truly the inverse square root of 2.5, then 1 divided by 3.1 squared would be 2.5. The Newton-Raphson method reduces this difference over each iteration so that you gradually get closer to the exact value. The method's algorithm essentially operates in three steps: Take the given number whose inverse square root is to be calculated and convert it into a corresponding memory address (a location in the computer's stored data). Particularly mysterious is the cryptic string 0x5f3759df, which has since gone down in computer science history as the “magic number.” It is the reason why only one iteration is necessary to obtain an approximate solution for the inverse square root that produces an error of at most 0.175 percent. In a technical paper published in 2003, computer scientist Chris Lomont wrote: “Where does this value come from, and why does the code work?” The hexadecimal number 0x5f3759df corresponds to 1,597,463,007 in decimal notation. By breaking down the individual steps of the program code, Lomont realized that he could obtain 1,597,463,007 through certain calculations. Lomont mathematically investigated which value yields an optimal result for different inputs. In other words: Which starting value best approximates the inverse square root and should therefore lead to the smallest error? He arrived at a value of 1,597,465,647, which is approximately: When Lomont compared his results with those of the original, he encountered a surprise. "Yet surprisingly, after one Newton iteration, it has a higher maximal relative error," Lomont writes. In search of a better constant, Lomont repeated his calculation and optimized for the best possible solution for a single Newton step. He arrived at a value of 1,597,463,174, which is approximately: When he put this result to a practical test, it actually yielded slightly better results than the magic number in the Quake 3 code. Online communities began a relentless search for this mystery person. Particularly dedicated to this effort was computer scientist Rys Sommefeldt, who first contacted John Carmack, the lead developer of Quake 3. Carmack was unsure of who coded this snippet and could only offer guesses, however. Sommefeldt contacted some of the most prominent developers of the 1990s, who each suggested other possible authors without claiming authorship for themselves. It then found its way into the Quake 3 algorithm via several other individuals. But exactly how the magic number was determined remains unclear to this day. It's astonishing how much effort and brainpower went into efficient software programming back then—a trend that's often overlooked today thanks to current computing power. This article originally appeared in Spektrum der Wissenschaft and was reproduced with permission. Manon Bischoff is a theoretical physicist and an editor at Spektrum der Wissenschaft, the German-language sister publication of Scientific American. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225541.htm'>Scientists confirm one-dimensional electron behavior in phosphorus chains</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 11:54:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at BESSY II have, for the first time, experimentally confirmed that a material can exhibit truly one-dimensional electronic properties. The team studied short chains of phosphorus atoms that naturally arrange themselves at specific angles on a silver surface. By applying advanced measurement and analysis techniques, they separated the signals coming from chains aligned in different directions. This careful work showed that each individual chain behaves as a genuine one-dimensional electronic system. If packed tightly together, however, calculations predict it would behave like a metal. All materials are built from atoms that bond together in different patterns. In most solids, atoms connect both within a plane and vertically. Two-dimensional materials have attracted intense interest because of their unusual electronic and optical properties. Under carefully controlled conditions, phosphorus atoms can organize themselves into short, straight lines on a silver substrate. However, neighboring chains may still interact with one another from the side. Those lateral interactions can alter the electronic structure and potentially disrupt true one-dimensional behavior. "Through a very thorough evaluation of measurements at BESSY II, we have now shown that such phosphorus chains really do have a one-dimensional electronic structure," says Professor Oliver Rader, head of the Spin and Topology in Quantum Materials department at HZB. "We achieved very high-quality results, enabling us to observe standing waves of electrons forming between the chains," says Varykhalov. The team then mapped the electronic structure using angle-resolved photoelectron spectroscopy (ARPES) at BESSY II, a technique in which they have extensive expertise. Dr. Maxim Krivenkov and Dr. Maryam Sajedi played a key role in interpreting the data. "We could disentangle the ARPES signals from these domains and thus demonstrate that these 1D phosphorus chains actually possess a very distinct 1D electron structure too," says Krivenkov. "We have entered a new field of research here, uncharted territory where many exciting discoveries are likely to be made," says Varykhalov. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/astronomers-spot-one-of-the-largest-spinning-structures-in-the-universe/'>Astronomers spot one of the largest spinning structures in the universe</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 11:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The first time that University of Oxford astronomer Lyla Jung saw the cosmic configuration on her monitor, she almost didn't believe it was real. But it was—and Jung and her colleagues went on to identify one of the largest rotating structures ever found in space: a chain of galaxies embedded in a spinning cosmic filament 400 million light-years from Earth. The finding, published in Monthly Notices of the Royal Astronomical Society, may give astronomers new insights into galaxies' formation, evolution and diversity, Jung says. Galaxies are not positioned either randomly or uniformly in the universe; instead they are connected in structures called filaments that link them, together with dark matter, across space. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. These filaments are the main channels through which matter flows, feeding galaxies and clusters as structures expand. “By studying filaments, we gain insight into how large-scale structure forms and how galaxies acquire their spins,” says astrophysicist Peng Wang of the Shanghai Astronomical Observatory, who was not involved in the new study. The new study takes a closer look at one of these structures. The researchers observed that many of the individual galaxies MeerKAT detected were spinning—and, to their surprise, they also found that the entire filament, including the rest of its galaxies, appeared to be rotating in sync with that spin at a speed of about 110 kilometers per second, something astronomers hadn't seen before. Detecting this phenomenon “is exceptional,” Wang adds, because the observation signal is faint, and overlapping objects along the line of sight can muddy the picture without very careful data collection and modeling. In later analyses, Jung and her team found that the filament is probably still taking on more material. Many of its galaxies seem to be in the early stages of growth, she says, because they appear rich in the hydrogen that provides fuel for new stars. Studying the rotation of filaments could also reveal how much dark matter is in them, says astronomer Noam Libeskind of the Leibniz Institute for Astrophysics Potsdam in Germany, who was not involved in the study. By revealing what portion of the universe exists in these filaments, Libeskind says, this study and future ones like it offer “a way of measuring the dark matter content of the universe.” Humberto Basilio is a Mexican science journalist covering policy, health, misconduct, archaeology and the environment. He is also a former news intern at Scientific American. His work has been published in the New York Times, National Geographic, Science, Nature, and more. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00435-w'>Why Europe barred China from flagship Horizon research programmes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 11:11:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Chinese research organizations can no longer take part in most of the research grants funded by Horizon Europe, the European Union's €93.5 billion (US$111 billion) research-funding programme. Starting this year, organizations based in or controlled by China cannot apply for grants to fund projects involving artificial intelligence, telecommunications such as 5G, health, semiconductors, biotechnology or quantum technologies. However, Chinese organizations can still apply for or participate in select research projects related to climate, biodiversity, food and agriculture. Beatrice Plazzotta, a business analyst for the Europe Commission, which manages the Horizon Europe programme, in Brussels, Belgium, said at a webinar last month: “We are in an economic and geopolitical conjuncture where we have to pay attention to what we do and who we collaborate with.” Plazzotta said the protection of intellectual property (IP) was among the EU's concerns. The United States has accused China of stealing trade secrets and obtaining IP by forcing foreign companies to join Chinese firms to sell in China, allegations the Chinese government has repeatedly denied. This includes its Made in China 2025 strategy — a ten-year policy aimed at boosting China's technology and manufacturing in areas such as AI, robotics and biotech — which has the stated goal of encouraging cross-border IP licensing. Joy Zhang, a sociologist at the University of Kent, UK, says that Horizon Europe's concerns about IP and information theft are valid, but the response is not. “It sends the wrong signal,” she adds, and does not address the fact that IP theft is more likely to occur among commercial companies, not researchers making cutting-edge discoveries. China leads research in 90% of crucial technologies — a dramatic shift this century China is leading the world on AI governance: other countries must engage US repeals key ‘endangerment finding' that climate change is a public threat US repeals key ‘endangerment finding' that climate change is a public threat Universities in exile: displaced scholars count the costs of starting afresh Join HZAU's global faculty team to advance research with competitive benefits. China leads research in 90% of crucial technologies — a dramatic shift this century China is leading the world on AI governance: other countries must engage An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41590-026-02432-7'>Demographic and genetic factors shape the epitope specificity of the human antibody repertoire against viruses</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 10:49:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite advances in understanding the mechanisms of antibody generation, a comprehensive model of how intrinsic and external factors shape human humoral responses to viruses has been lacking. Here we apply phage immunoprecipitation sequencing to investigate the effects of demographic factors—including 108 lifestyle and health-related variables—and genetic variation on antibody reactivity to over 97,000 viral peptides in 1,212 healthy adults. We demonstrate that age, sex and continent of birth extensively affect not only the viruses but also the specific viral epitopes targeted by the antibody repertoire. Notably, we find that antibodies against rapidly evolving epitopes of influenza A virus decrease with age, whereas immunoreactivity to conserved epitopes increases. Furthermore, we identify strong associations between antibodies against 34 viruses and genetic variants at HLA, FUT2, IGH and IGK loci, some of which increase autoimmune disease risk. These findings offer a valuable resource for understanding the factors affecting antibody-mediated immunity, laying the groundwork for optimizing vaccine strategies. Antibodies are central effectors of adaptive immunity and serve as correlates of protection following natural infection or vaccination. The large inter-individual variability in antibody repertoires indicates that antibody production and maintenance are shaped by multiple factors. Family- and population-based studies have revealed marked differences in antibody titers according to sex and age. For example, women exhibit higher titers against human papillomavirus1 and Epstein–Barr virus (EBV)1,2 and mount stronger vaccine responses than men3. Furthermore, antibodies against persistent herpesviruses, such as herpes simplex virus 1 (HSV-1) and cytomegalovirus (CMV), tend to increase with age, reflecting cumulative exposure1,2,4,5, whereas antibodies against viruses that primarily infect children (for example, respiratory syncytial virus (RSV) and varicella–zoster virus (VZV)) typically persist at high levels into adulthood1,2. Additional non-genetic factors associated with antibody levels include socioeconomic status1,2 and smoking4. Human genetic factors also contribute to variation in antibody responses. Total and virus-specific antibody titers against CMV, EBV and influenza A virus (IAV) have been shown to be heritable2,6. At the genome-wide scale, the HLA locus presents strong associations with antibody titers against numerous viruses, such as EBV, IAV, rubella virus and VZV2,7,8,9,10. Furthermore, although antibodies targeting a single virus can recognize numerous epitopes—the portion of an antigen bound by the immune system—inter-individual variation in epitope reactivity remains poorly characterized, leaving the determinants of viral antigenic specificity largely unknown. In this Resource, we address these questions using phage immunoprecipitation sequencing (PhIP-seq), a high-throughput approach for assessing antibody–epitope interactions12,13. PhIP-seq has been applied to characterize antibody repertoire changes across diseases5,14 and evaluate humoral immunity to bacteria4,15. A virus-focused implementation, VirScan16, which spans the complete peptidome of all known human viruses, has recently allowed investigation of the impact of measles infection on antibody profiles17 and immune development in neonates18. Using the VirScan library, we profiled 97,978 viral peptides in 1,212 healthy adults and integrated these data with extensive demographic and genetic information. This approach enabled us to characterize differences in the viruses, viral proteins and epitopes targeted by individual antibody repertoires and to identify key factors shaping both the breath and epitope specificity of antiviral humoral immunity. To assess the viral peptidome-wide antibody repertoire, we performed PhIP-seq on 900 plasma samples from the Milieu Intérieur cohort19, comprising individuals of European ancestry with balanced sex and age distribution (20–69 years; Fig. To validate our findings and explore population-level differences in humoral immunity, we analyzed an additional 312 samples from the EvoImmunoPop (EIP) cohort20, including 100 and 212 Belgian residents born in Central Africa (AFB) and Europe (EUB), respectively—all male, aged 20–50 years (Fig. For both cohorts, we used the VirScan V3 library, encompassing 115,753 microbial peptides16. After filtering for unique viral sequences, we retained 97,978 peptides, representing a wide range of viral families and species (Extended Data Fig. PhIP-seq read counts for each viral peptide were converted into standardized, batch-corrected Z scores (Extended Data Fig. 2 and Supplementary Note), which have been shown to correlate strongly with antibody titers16. a, Sample sizes and age distribution by sex within the Milieu Intérieur cohort. b, Sample sizes and age distribution by continent of birth within the EIP cohort. d, Number of positive peptides per Milieu Intérieur donor as a function of age and sex. i, Heatmap indicating the predicted infection status of each Milieu Intérieur and EIP donor for the 20 most prevalent viruses, as determined by AVARDA (Padj < 0.05 after Benjamini–Hochberg correction). The total numbers of positive peptides per individual were normally distributed (Fig. 1c,d,f,g), averaging 881 and 1,044 peptides for Milieu Intérieur and EIP individuals, respectively, due to differences in cohort demographics or sampling protocols. Approximately 97% of peptides were positive in <5% of individuals, reflecting individual-specific immunity (denoted private peptides) or false positives4,15 (Fig. We therefore conducted all subsequent analyses on peptides positive in >5% of individuals, requiring at least two peptides to be positive per virus (denoted public peptides). This yielded 2,608 and 3,210 public peptides in Milieu Intérieur and EIP individuals, respectively, originating from 113 viral species, with EBV, IAV and enterovirus B being the most prevalent in both cohorts (Extended Data Fig. Given the high sequence identity among some of the peptides tested, cross-reactivity can lead to false-positive signals. To address this, we used the antiviral antibody response deconvolution algorithm (AVARDA), which estimates the breadth of antibody responses for each viral species—defined as the largest number of reactive viral peptides that show low sequence identity—while accounting for cross-reactivity and the unbalanced peptide representation in the PhIP-seq library21. As expected, seroprevalence determined by AVARDA breadth scores was highest for common viruses such as EBV, HSV-1, CMV, rhinovirus B and adenovirus C (Fig. We validated the resolution, sensitivity and serostatus prediction accuracy of both peptide-level Z scores and virus-level AVARDA breadth scores through extensive comparisons with enzyme-linked immunosorbent assays (ELISA) and Luminex assays (Extended Data Figs. Together, these analyses underscore the specificity and sensitivity of PhIP-seq and reveal the remarkable diversity of human antibody repertoires against viruses causing common infections. We investigated the effects of non-genetic and genetic factors on antiviral humoral immunity by systematically testing their associations with both peptide-level Z scores and virus-level AVARDA breadth scores. This combined approach allowed us to capture inter-individual variation in antigenic specificity and to identify associations that can be missed at the virus level while accounting for between-species cross-reactivity (Supplementary Note). Before conducting the main analyses, we verified whether antibody reactivity scores were associated with the numbers of different B cell subsets in blood or with B cell output from bone marrow (Supplementary Table 3), which could mediate the effects of age or smoking on humoral responses22. No such associations were detected (adjusted P value (Padj) > 0.05; Extended Data Fig. 5 and Supplementary Note), suggesting that the factors examined below act directly on antibody levels. As no significant nonlinear effects of age or age × sex interactions were observed, we focused only on linear and additive effects (Methods). Linear regression modeling revealed that age is strongly associated with antibody reactivity against a broad range of viruses (Fig. Antibodies against 565 peptides increased with age (false-discovery-rate-adjusted P < 0.05; Supplementary Table 4), primarily from the herpesviruses HSV-1, HSV-2 and EBV, which can reactivate throughout life23. These associations were robust to cross-reactivity, as supported by AVARDA (Extended Data Fig. The strongest age effects were observed for antibodies targeting HSV-1 envelope glycoprotein D (Extended Data Fig. Both peptide-level Z scores and AVARDA breadth scores also showed positive associations with age for hepatitis A virus and Aichi virus A (Fig. 6a), the latter of which is a kobuvirus that was initially isolated during a 1989 gastroenteritis outbreak in Japan and has subsequently been detected in Europe24. Conversely, antibodies against 766 peptides decreased with age, primarily from rhinoviruses, enteroviruses and adenoviruses (Padj < 0.05; Fig. After accounting for cross-reactivity, AVARDA confirmed age-related decreases for antibodies against rhinoviruses A and B, enteroviruses B and C and adenovirus D (Extended Data Fig. a, −log10(Padj) values and the directions of associations between all public peptide Z scores and age in the Milieu Intérieur cohort, by viral species (two-sided Wald test). The dotted gray vertical lines indicate viruses for which the AVARDA breadth score is significantly associated with age. b, −log10(Padj) values against effect sizes of associations between IAV peptide Z scores and age in the Milieu Intérieur cohort, colored by viral protein (two-sided Wald test). c, Amino acid positions of the midpoint of public HA peptides associated with age within the full IAV (HA) protein. d, Per-residue evolutionary rates of HA, computed from all H3 viral sequences sampled between 1975 and 2012. e, HA peptide evolutionary rates as a function of −log10(Padj) values for negative associations between antibody reactivity and age (two-sided Wald test). The black line and shaded area indicate the regression line and 95% confidence interval, respectively. g, PhIP-seq-based Z score as a function of age for HA peptides 1 and 3, highlighted in c. h, Luminex-based antibody titers targeting globular head (H3N2 HA1 Beijing/1992) and stalk (H1N1 Puerto Rico/1934) HA domains as a function of age, replicating the results shown in g. The blue line indicates the regression line. In c and i, significance and the directions of associations with age are indicated on the y axis and by the direction of triangles, respectively. The most significant peptides for each epitope are circled and labeled. Notably, antibodies against different IAV peptides either strongly increase or decrease with age (Fig. In younger individuals, antibodies primarily target amino acid positions 1–100 and 300–400 of hemagglutinin (HA) (for example, peptide 1: βVirScan = −0.029 and PVirScan = 1.48 × 10−19; Fig. 2c,f,g), corresponding to the more variable globular head domain25 (Fig. By contrast, in older individuals, antibodies preferentially target positions 450–550 within the more conserved stalk domain (for example, peptide 3: βVirScan = 0.023 and PVirScan = 1.11 × 10−21; Fig. Antibodies showing the greatest decrease with age tend to target the most variable HA peptides (Spearman's ρ = −0.53 and P = 7.4 × 10−5; Fig. 2e), suggesting that antibodies against rapidly evolving viral epitopes wane faster than those targeting more evolutionarily stable epitopes. These observations were validated with Luminex immunoassays (H3N2 head: βLuminex = −0.033 and PLuminex = 4.42 × 10−76; H1N1 stalk: βLuminex = 0.014 and PLuminex = 8.62 × 10−15; Fig. A similar pattern was observed for the IAV matrix protein 1 (M1) (Fig. 2b,i): antibodies against less conserved positions (200–250) decrease with age (βVirScan = −0.018; PVirScan = 5.90 × 10−6; βLuminex = −0.0068; PLuminex = 4.04 × 10−4), whereas antibodies targeting more conserved positions (150–200) increase (βVirScan = 0.02; PVirScan = 3.33 × 10−10). These effects result in a significant relationship between M1 evolutionary rates and age-related changes in immunoreactivity (Spearman's ρ = −0.70; P = 1.8 × 10−4). To determine whether these peptide-specific effects were driven by age-related variations in exposure to different IAV strains, we compared antibodies targeting protein domains from the same viral strain. Furthermore, although past influenza vaccination was associated with higher total anti-IAV antibody titers in the Milieu Intérieur cohort (β = 0.34; P = 2.62 × 10−14), vaccination was only weakly associated with age (odds ratio = 0.012; P = 0.048), supporting the view that vaccination is unlikely to account for the observed age-related patterns. Notably, the AVARDA breadth score for IAV was not associated with age (Extended Data Fig. 6a), as it aggregates peptides with opposing age effects. Together, these findings indicate that epitope specificity of anti-IAV humoral responses varies with age. Sex effects were more modest than age effects: 330 peptides showed higher antibody levels in women and 236 peptides showed higher antibody levels in men (Padj < 0.05; Extended Data Fig. Accounting for cross-reactivity, AVARDA supported higher reactivity in women for antibodies against CMV, human herpesvirus 6A (HHV-6A) and HHV-6B (Extended Data Fig. These results suggest higher exposure and/or stronger humoral responses to herpesviruses in women, in contrast with bacterial antibody responses that show no sex differences4. As with age, we observed sex-specific antibody reactivity to different proteins of influenza viruses (Extended Data Fig. 7c,d): women preferentially target the HA protein (βVirScan = −0.71; PVirScan = 1.26 × 10−13; βLuminex = −0.10; PLuminex = 0.038), whereas men showed higher reactivity to M1 (βVirScan = 0.71; PVirScan = 2.10 × 10−14; βLuminex = 0.092; PLuminex = 0.02) and NP (βVirScan = 0.60; PVirScan = 3.55 × 10−10; βLuminex = 0.10; PLuminex = 0.03) proteins from IAV and IBV, respectively. For comparison, we inferred serostatus for M1 from H3N2 and found that 10.0% of women and 16.1% of men were seropositive (odds ratio = 1.72; 95% confidence interval = 1.16–2.57; PVirScan = 0.0077), consistent with previous findings26. As influenza vaccination rates did not differ between women and men in the Milieu Intérieur cohort (20.2 versus 18.6%, respectively; P = 0.51) and vaccines do not typically target the M1 protein, these findings point to intrinsic sex differences in humoral responses to influenza viruses. To investigate how the antiviral antibody repertoire varies between populations, we leveraged the EIP cohort, comprising individuals born in Central Africa (AFB) or Europe (EUB). Although all samples were collected in Belgium, AFB individuals had relocated to Europe shortly before sample collection (2.45 years before, on average27), implying that differences compared with EUB probably result from previous viral exposures and/or genetic ancestry. Antibody levels against 898 viral peptides were increased in EUB, predominantly from rhinoviruses, adenoviruses and IAV (Padj < 0.05; Fig. 3a), although the significance was weak when considering AVARDA scores (Padj > 0.001). By contrast, higher antibody reactivity in AFB was observed for 647 peptides, of which 61% were derived from herpesviruses. Adjusting the statistical models for genetic determinants of the antiviral antibody repertoire (Supplementary Table 5) had minimal impact on the differences observed between AFB and EUB (Extended Data Fig. 8d), supporting differential viral exposure—rather than genetic ancestry—as the primary driver of population-level variation in antibody repertoires. a, −log10(Padj) values and the directions of associations between all public peptide Z scores and continent of birth in the EIP cohort, separated by viral species (two-sided Wald test). The dashed gray vertical lines indicate viruses for which the AVARDA breadth score is significantly associated with continent of birth. b, −log10(Padj) values against effect sizes of associations between continent of birth and peptide Z scores from the EBV AG876 strain in the EIP cohort (two-sided Wald test). c,d, PhIP-seq reactivity scores (c) and Luminex-based titers (d) for antibodies targeting the most significant EBNA-4 peptide (UniProt ID: Q1HVG4) from the EBV AG876 strain, in AFB and EUB separately. Horizontal lines, box edges and whiskers indicate the median value, interquartile range and 1.5× the interquartile range, respectively. e, Amino acid positions of the midpoint of public EBNA-4 peptides associated with continent of birth within the full EBV EBNA-4 protein for the EIP cohort (two-sided Wald test). f, −log10(Padj) values against effect sizes of associations between continent of birth and peptide Z scores from the EBV B95-8 strain in the EIP cohort (two-sided Wald test). g,h, PhIP-seq reactivity scores (g) and Luminex-based titers (h) for antibodies targeting the most significant EBNA-6 peptide (UniProt ID: P03204) from EBV B95-8, in AFB and EUB separately. Horizontal lines, box edges and whiskers indicate the median value, interquartile range and 1.5× the interquartile range, respectively. i, Amino acid positions of the midpoint of all public EBNA-6 peptides associated with continent of birth within the full EBV EBNA-6 protein for the EIP cohort (two-sided Wald test). Population differences were also evident at the epitope level within individual viral species. Despite similar overall EBV reactivity between populations (Padj > 0.05; Extended Data Fig. 8a), AFB and EUB targeted different EBV peptides (Fig. Antibodies from AFB more frequently targeted the EBNA-4 viral protein, whereas those from EUB preferentially targeted EBNA-6. The four EBNA-4 peptides most associated with African origin clustered between amino acid positions 600 and 800 and derived from the AG876 strain, a type 2 EBV strain that is prevalent in Africa30 (Fig. Conversely, EBNA-6 peptides associated with European origin localized to positions 750–850 and derived from the GD1 and B95-8 cosmopolitan strains (Fig. These findings, validated by Luminex immunoassays for both EBNA-4 (β = −1.28; P = 9.12 × 10−14; Fig. 3h), suggest that population-specific epitope targeting reflects past exposure to distinct EBV strains. Collectively, these results reveal population disparities in antibody reactivity against epitopes of common viruses, highlighting the limitations of single-antigen assays to assess seroprevalence in global epidemiological studies. To identify new non-genetic factors affecting antiviral antibody repertoires, we searched for associations in the Milieu Intérieur cohort with 108 variables related to socioeconomic status, health-related habits, medical history and disease biomarkers (Supplementary Table 3) while controlling for age and sex. Besides weak associations with socioeconomic status and health biomarkers (Fig. 4a, Supplementary Table 4 and Supplementary Note), the only strongly significant associations were found for tobacco smoking, correlating with antibodies against 134 peptides (Padj < 0.05; Fig. AVARDA confirmed the association between cigarette consumption and antibodies targeting rhinoviruses A and B (β = 0.032; Padj = 1.99 × 10−4). Rhinoviruses are a prevalent cause of the common cold, which occurs more frequently and with greater severity in smokers, although the underlying mechanisms remain debated31,32. a, −log10(Padj) values for associations between public peptide Z score and health- and lifestyle-related variables (two-sided Wald test for continuous and binary variables; analysis of covariance for categorical variables). Only variables with an association of Padj < 0.01 are shown. b, −log10(Padj) values and directions of associations between all public peptide Z scores and smoking status in the Milieu Intérieur cohort, separated by viral species (two-sided Wald test). The dotted gray vertical lines indicate viruses for which the AVARDA breadth score is significantly associated with smoking status. c, Antibody reactivity for the rhinovirus B peptide most significantly associated with smoking status, categorized by smoking status. Horizontal lines, box edges and whiskers indicate the median value, interquartile range and 1.5× the interquartile range, respectively. d, Antibody reactivity for the rhinovirus B peptide most significantly associated with smoking status, as a function of years of smoking in active smokers. e, Antibody reactivity for the rhinovirus B peptide most significantly associated with smoking status, as a function of years since last smoking in former smokers. These associations with smoking were not reproduced using immunoassays based on whole-rhinovirus lysates (Supplementary Table 1), suggesting that peptide-specific effects are masked when the full rhinovirus peptidome is assessed. Consistently, antibody reactivity was associated with smoking status for only 29% (63 out of 218) of the rhinovirus peptides included in the VirScan library. However, the five peptides most strongly associated with smoking—derived from a rhinovirus polyprotein containing capsid proteins (β > 0.040; Padj < 1.0 × 10−10; Fig. 4c)—included the peptide showing the strongest smoking association in the LifeLines-DEEP cohort (n = 1,443; β = 1.09; Padj = 3.29 × 10−9) using a custom PhIP-seq library4, validating that these associations are both genuine and peptide specific. We found that anti-rhinovirus B reactivity was not associated with smoking duration in active smokers (β = 0.012; P = 0.383; Fig. Interestingly, ex-smokers exhibited antibody levels comparable to those of people who had never smoked (β = −0.17; P = 0.084; Fig. Accordingly, anti-rhinovirus B antibodies decreased with years after quitting smoking in former smokers (β = −0.022; P = 8.74 × 10−3; Fig. These findings collectively suggest that smoking exerts a strong, yet reversible, effect on the antibody repertoire against rhinoviruses. To identify genetic factors affecting the antiviral antibody repertoire, we conducted a genome-wide association study (GWAS) of 2,608 public peptide Z scores in the Milieu Intérieur cohort, testing associations with 5,699,237 imputed common single-nucleotide polymorphisms (SNPs)22 while controlling for age, sex and genetic structure (Methods). Given the incomplete coverage of B cell receptor loci by the imputed SNPs, we performed next-generation sequencing of the IGH, IGK and IGL genes in all Milieu Intérieur donors at a depth of ~35× coverage, identifying 30,503 additional common variants. In total, we detected strong genome-wide significant associations for 225 viral peptides at four independent loci, including HLA, FUT2, IGH and IGK genes (P < 1.31 × 10−10; Fig. a, Manhattan plot of associations between 2,608 public peptides and common human genetic variants (minor allele frequency > 5%) in the Milieu Intérieur cohort (two-sided Wald test). The top hit of each peak is annotated with the closest gene or gene locus. In b–d, horizontal lines, box edges and whiskers indicate the median value, interquartile range and 1.5× the interquartile range, respectively. e,f, LocusZoom plots for the associations between IGH variants and antibody reactivity against: the rubella virus (UniProt ID: D5KJ87) (e) and the sandfly fever Sicilian virus (UniProt ID: A7KCL0) (f) (two-sided Wald test). The variant most significantly associated with antibody reactivity and the closest gene usage quantitative trait locus variant (rs9671760) are indicated by gray vertical lines. IGHV gene, immunoglobulin heavy chain variable gene; lncRNA, long non-coding RNA; miRNA, microRNA; rRNA, ribosomal RNA. We found associations between HLA variants and antibody reactivity against 112 peptides from 15 viruses, including EBV, HSV-1 and adenoviruses A–F, consistent with previous studies2,4,7,8,9,10 and replicated in the EIP cohort (PEIP < 0.05; Supplementary Table 5). To account for linkage disequilibrium and facilitate comparisons with disease studies, we imputed HLA alleles from genotype data and tested for associations between peptide Z scores and allele dosages. This analysis identified 85 associations (Supplementary Table 6), including HLA-DRB1*04 (β = 0.96; P = 2.0 × 10−16) and HLA-DQA1*03:01 (β = 0.66; P = 2.3 × 10−15) with adenovirus peptides (Fig. These alleles have been associated with an increased risk for type 1 diabetes and rheumatoid arthritis33, potentially explaining the link between these immune-mediated diseases and EBV or adenovirus infections34,35. Variants near FUT2 were associated with antibodies against norovirus peptides (β = 0.31; P = 1.10 × 10−10; Extended Data Fig. Mutations in FUT2 determine the non-secretor phenotype, which confers resistance to norovirus infection and susceptibility to type 1 diabetes and inflammatory bowel disease36,37. We additionally identified a novel association between variants in near-complete linkage disequilibrium (r2 = 0.995) with rs601338 and antibodies against two salivirus strains for both Milieu Intérieur (β = −0.33; P < 1.58 × 10−14) and EIP (PEIP < 1.36 × 10−10; Extended Data Fig. Saliviruses, discovered in 2009 in diarrheal samples, are known to cause gastroenteritis39, although their cellular tropism and entry mechanisms remain unclear. Associations between FUT2 non-secretor status and anti-salivirus antibodies are unlikely to reflect cross-reactivity with norovirus, as corresponding Z scores were uncorrelated (Extended Data Fig. Genetic variation at the IGH locus was associated with 107 peptides from 21 viruses (Fig. This locus encodes the antibody heavy chain and has previously been associated with antibody levels against various bacteria, IAV and norovirus4. Our analyses extended these findings to additional viruses, including herpesviruses (HSV-2, EBV, CMV and HHV-6), RSV, IAV, HBV, coronavirus NL63, rubella virus, sandfly fever Sicilian virus, enteroviruses and rhinoviruses. Several newly identified variants influence IGHV clonal gene usage by V(D)J somatic recombination, as assessed by adaptive immune receptor repertoire sequencing in a previous study40. For example, we found that a variant associated with anti-rubella antibodies (rs1024350; β = 0.33; P = 1.90 × 10−11) and suggestively associated with anti-IAV antibodies (β = −0.28; P = 5.38 × 10−10) affects IGHV1-69 usage40 (P = 1.14 × 10−16). IGHV1-69 gene usage partially determines the quality of anti-influenza antibodies41 and has been associated with lupus and type 1 diabetes42. Another variant, rs9671760, associated with antibodies against rubella virus (β = 0.37; P = 3.34 × 10−14; Fig. 5e) and sandfly fever Sicilian virus (β = −0.38; P = 1.46 × 10−11; Fig. The fourth locus included IGK, encoding the κ light chain of antibodies, and was associated with antibody levels targeting adenovirus B peptides (β = 0.81; P = 1.51 × 10−23; Extended Data Fig. Together, these findings underscore the pervasive impact of host genetic factors, including germline mutations in immunoglobulin genes, on humoral responses to multiple viruses. Finally, to quantify the relative contributions of demographic (non-genetic) and genetic factors to antibody variability, we estimated the proportion of variance explained by age, sex, smoking and GWAS lead variants for each of the 2,608 public peptides of the Milieu Intérieur cohort. Together, these factors explained an average of 7.39% (range = 0.91–25.50%) of inter-individual variation in antibody reactivity (Fig. Demographic factors accounted for 3.81% (range = 0.007–20.68%) of the variance, whereas genetic factors contributed to 3.44% (range = 0.48–23.02%) of the variance. These relative contributions varied substantially across viruses (Extended Data Fig. 10a,b): antibody levels against rhinovirus peptides were dominated by age effects (Fig. 2a), those against CMV were dominated by sex and those against EBV were dominated by genetic variation (Supplementary Table 5). a, Proportion of variance explained by demographic (that is, age, sex and smoking) and genetic factors for antibody reactivity against 2,608 public peptides in the Milieu Intérieur cohort. The peptides are sorted by total variance explained. b, Variance explained by age and IGH genetic variation for RSV protein G peptides in the Milieu Intérieur cohort, colored according to RSV strain, as in c. c, Hierarchical clustering of peptide sequences from RSV protein G, separating peptides affiliated with the RSV-A (green) and RSV-B (blue) strains. d, Amino acid positions of the midpoint of protein G peptides associated with age and IGH genetic variation within the full RSV protein G for the Milieu Intérieur cohort (two-sided Wald test). P values for the association with age (beige) and the most significant IGH variant (black) are indicated, separated by RSV strain. Significance and directions of associations are indicated on the y axis and by the direction of triangles, respectively. For example, antibody reactivity to the HA protein of IAV was predominantly explained by age, whereas anti-M1 antibodies were primarily affected by IGH genetic variation (Extended Data Fig. Similarly, anti-EBV antibodies targeting the EBNA-5 protein were strongly shaped by HLA genotype, whereas those targeting EBNA-4 and tegument proteins varied primarily in an age-dependent manner (Extended Data Fig. A comparable pattern emerged for anti-RSV antibodies, but at the level of a single protein: variance of antibodies against different peptides of the immunogenic glycoprotein G was primarily explained by either age or IGH genetic variants (Fig. Age-associated peptides derived from RSV strain A, whereas IGH-associated peptides originated from strain B—two phylogenetic RSV lineages differing in the protein G sequence (Fig. Specifically, antibodies increasing with age primarily target positions 150–200 of protein G in RSV-A (β = 0.016; P = 4.85 × 10−23; Fig. 6d)—a pattern consistent with a previous study43 and replicated in the EIP cohort for EUB individuals only (βEIP = 0.036; PEIP = 0.015; Extended Data Fig. 10e)—whereas antibodies associated with the IGH variant (rs59595881) target positions 225–275 of protein G in RSV-B (β = 0.60; P = 1.11 × 10−12). The position-specific effects of age and genetics are unlikely to be driven solely by differential exposure to RSV, as Europeans are seasonally exposed to both RSV-A and RSV-B44. Overall, these findings indicate that the effects of demographic and genetic factors largely differ among viruses, viral strains, proteins and epitopes targeted by the antibody repertoire. In this study, we generated a comprehensive dataset of plasma antibody levels against over 97,000 viral peptides, providing a valuable resource to investigate the intrinsic, environmental and genetic factors shaping the antibody repertoire in healthy adults. All of the results can be explored via a dedicated web-based browser (https://mirepertoire.pasteur.cloud/). Among these factors, age had the most profound and widespread effect. Age-related increases in antibody response may reflect higher exposure in older adults (for example, hepatitis A virus and Aichi virus A), reactivation of latent viruses (for example, HSV-1, HSV-2, EBV and CMV) or reinfections by viruses causing recurrent infections (for example, IAV, IBV and RSV). Conversely, age-related decreases probably reflect higher exposure during young adulthood and/or faster antibody waning (for example, rhinoviruses A–C and enteroviruses B and C). Importantly, our study shows that aging is associated with differential epitope recognition within the same viral protein. For example, anti-IAV antibodies of younger and older adults preferentially target different domains of viral proteins, this pattern persisting within the same viral strains and for the M1 protein, which is not a typical vaccine target. Therefore, our results suggest that these differences are not solely due to age-related disparities in natural or vaccine-induced exposure to diverse viral strains, but reflect the waning and recall of antibodies targeting variable versus conserved influenza epitopes45, respectively. Alternatively, the accessibility of certain viral protein domains may require multiple reinfections to elicit antibodies, consistent with proposed mechanisms for age-related differences in neutralizing titers against HA globular head and stalk domains of IAV46,47. We propose that age-dependent antigenic specificity, observed here across several IAV proteins, may therefore be a broader phenomenon than was previously appreciated. Similarly, we show that sex influences immunodominance, with women's antibodies preferentially targeting the HA protein of IAV and IBV, whereas men's antibodies disproportionally target NP and M1. These differences are unlikely to reflect vaccination rates, which were similar between sexes, although we cannot exclude that women received greater numbers or more recent vaccine doses than men. Antibody profiles also vary markedly according to the continent of birth, probably due to differences in viral exposure16. Antibodies from individuals born in Central Africa and Europe preferentially target different EBV proteins, suggesting that regional variation in EBV strains30 contributes to population differences in antibody responses at the epitope level. Among environmental factors affecting the antibody repertoire, we identified a strong association between smoking and anti-rhinovirus antibodies, consistent with previous findings4 and the higher risk of common cold in smokers31. Notably, ex-smokers exhibited antibody levels against rhinoviruses comparable to those of people who had never smoked, suggesting that altered viral clearance and/or heightened exposure in smokers are reversible upon smoking cessation. Finally, our GWAS confirmed that HLA and IGH affect antibody levels against a range of viruses2,4,7,8,9,10 and largely expanded the list of associated viruses by revealing novel associations with herpesviruses 2–6, RSV, HBV, rhinoviruses, enteroviruses, coronavirus NL63 and rubella virus. Sequencing of the immunoglobulin genes was critical in discovering these associations, as well as the association with IGK, since SNP arrays poorly cover these complex regions. We also identified a strong association between antibodies against the recently discovered and poorly understood saliviruses and FUT2, previously linked to norovirus infection, suggesting that saliviruses may exploit similar host infection mechanisms as noroviruses. Several of these associated genetic variants have previously been linked to increased autoimmune disease risk33,36,37. Individuals with these diseases often show higher seroprevalence for common viruses, leading the authors of previous studies to suggest a causal role of these viral infections in autoimmunity34,35. However, our results suggest that associations between autoimmune conditions and antibody levels against viruses may instead result from a shared genetic etiology that affects both traits independently. Our study also supports the hypothesis of antagonistic pleiotropy, whereby variants that once conferred resistance to infection now predispose to non-infectious immune diseases48. Consistently, HLA and FUT2 alleles associated with antiviral humoral responses and autoimmunity have increased in frequency under natural selection in Europe over the past millennia49. First, although the VirScan library offers broad coverage, it is restricted to linear peptides, potentially missing antibodies recognizing conformational epitopes. Second, antibody cross-reactivity between peptides complicates the precise attribution of responses to specific viruses. Although we addressed this risk by using AVARDA, this method may also lead to false negatives. Fourth, PhIP-seq does not determine whether antibodies are protective against viral infection, which requires dedicated experiments. Finally, although we identified robust associations between antibody repertoires and non-genetic and genetic factors, these explain only a fraction of inter-individual variation. Longitudinal studies integrating the human viral exposome and virome, alongside genome-to-genome association studies50, are needed to fully elucidate the determinants of human variation in humoral responses to viruses. Despite these challenges, our study provides high-resolution insights into the widespread effects of age, sex, continent of birth and genetics on the antibody repertoire. Crucially, it reveals that these factors differentially affect antibodies targeting specific epitopes within the same virus or viral protein, deepening our understanding of antibody generation and maintenance. We anticipate that these findings and the accompanying dataset will prompt mechanistic studies of antiviral immunity, with the potential to inform vaccine and therapeutic strategies. The Milieu Intérieur cohort comprises 1,000 healthy adults recruited to investigate genetic and non-genetic determinants of immune response variation19. Recruitment was conducted in Rennes (France) in 2012–2013 and individuals were selected based on a large set of relatively strict inclusion and exclusion criteria, as described elsewhere19. Of the 900 individuals reported in the present study, 453 are female and 447 are male, ranging from 20–69 years of age. The study has been approved by the Comité de Protection des Personnes—Ouest VI and French Agence Nationale de Sécurité du Médicament. The samples and data were formally established as the Milieu Intérieur biocollection (NCT03905993), with approvals by the Comité de Protection des Personnes Sud Méditerranée and Commission Nationale de l'Informatique et des Libertés on 11 April 2018. The EIP cohort comprises 390 healthy adults recruited to investigate human population differences in immune responses. Of the 312 individuals reported in the present study, 100 individuals reported to be born in Central Africa (AFB; age range = 20–50 years) and 212 reported to be born in Europe (EUB; age range = 20–50 years). AFB and EUB individuals presented no evidence of recent genetic admixture with populations originating from another continent, besides two AFB donors who presented 22% Near Eastern and 25% European ancestries, respectively20. All individuals were negative for serological tests against human immunodeficiency virus, hepatitis B or hepatitis C. The study was approved by the Ethics Committee of Ghent University, the Ethics Board of Institut Pasteur (EVOIMMUNOPOP-281297) and the French authorities Comité de Protection des Personnes, Comité Consultatif sur le Traitement de l'Information en Matière de Recherche and Commission Nationale de l'Informatique et des Libertés. To investigate the virus-specific and viral peptide-specific antibody profiles in the Milieu Intérieur and EIP samples, we used PhIP-Seq using the VirScan V3 library, a pathogen-epitope scanning method combining bacteriophage display and immunoprecipitation. The detailed protocol and VirScan library are described elsewhere16,18. In brief, a library of linear peptides of 56 amino acids each was constructed to cover all UniProt protein sequences of viruses known to infect humans. Peptides were staggered along each protein sequence with an overlap of 28 amino acids. The phage library was inactivated and incubated with plasma samples normalized to total IgG concentration and controls (bead samples) to form IgG–phage immunocomplexes. The immunocomplexes were then captured by magnetic beads, lysed and sent for next-generation sequencing. Two replicates were performed for each individual, to assess reproducibility. Sequencing reads were processed as in ref. Subsequently, the positivity of each peptide was determined by a binning strategy whereby read counts from blank controls were first used to group the peptides into hundreds of bins so that the counts formed a uniform distribution within each bin. Then, the peptides from plasma samples were allocated into the pre-defined bins and Z scores were calculated for each peptide from each plasma sample. After generating a matrix of 115,753 peptide Z scores for 900 Milieu Intérieur or 312 EIP samples, we discarded peptides from bacteria, fungi and allergens from the VirScan library, resulting in 99,460 viral peptides. Z score values were inverse hyperbolic sine (arcsinh) transformed. In contrast to log transformation, the arcsinh function is convenient when handling overdispersion due to both outliers and zero values, which were common in the VirScan Z score data. Peptides of poor quality were identified by leveraging discordance across replicates. Z score values missing in only one replicate were set to missing in both replicates. Then, outliers in each replicate were defined as Z scores higher than the 99.5% quantile. The absolute difference in Z score between replicates was calculated for all peptides with an outlier value in at least one replicate. The distribution of absolute differences was bimodal, with the lower peak representing consistent Z scores between replicates and the upper peak representing inconsistent Z scores. The Z score values of both replicates for all outlier peptides were then set to missing. Next, peptides with >50% missing values were removed from the dataset, leaving 98,757 for Milieu Intérieur and 98,697 for EIP. Missing values were imputed by running a principal component analysis on all Z scores using the pca function from the pcaMethods package (nPcs = 10, scale = ‘uv'), followed by imputation using the completeObs function from the same package. As individual samples were processed in batches on cell culture plates, samples were batch corrected using the ComBat53 function from the sva R package, using plates as the batch variable (Supplementary Note). A peptide was considered significantly positive if the Z scores of both replicates were >3.5. To generate the list of public peptides, the datasets were filtered on peptides significantly positive in >5% of tested individuals for at least two peptides per virus. Between-species antibody cross-reactivity, unbalanced representation of viruses in the VirScan library and viral genome size can make peptide-level data challenging to interpret in some cases. To address these limitations and compare antibody profiles at the virus species level, we applied AVARDA21, using the code available at https://github.com/drmonaco/AVARDA. Individual VirScan peptides were aligned to each other and to a master library of all viral genetic sequences translated in all reading frames using BLAST. Evidence peptides were VirScan peptides that aligned to the master library with a bit score of >80. For each virus, AVARDA calculated a maximally independent set of unrelated peptides that explained the total reactivity toward this virus. Finally, cross-reactivity was evaluated by ranking all viruses based on the probability of infection. Pairs of viruses were then iteratively compared, where shared reactive peptides were assigned to the virus with the most substantial evidence of infection based solely on non-shared peptides. Once all peptides were exclusively assigned to a single virus, a final probability of infection for each sample was calculated using the binomial testing procedure described above. Additionally, a breadth score was calculated, defined as the largest number of reactive peptides from a given virus species that did not share any sequence similarities. Blood was collected in ethylenediaminetetraacetic acid (EDTA)-treated tubes, and the plasma was extracted by centrifugation. Total levels of the immunoglobulins IgG, IgM, IgE and IgA were measured with a turbidimetric test on an Olympus AU400 Chemistry Analyzer. The ELISA-based serologies were measured for IgG against the following viruses and antigens: CMV, HSV-1, HSV-2, EBV, VZV, IAV, rubella, mumps and measles (Supplementary Table 1). The data processing steps for the immunoassay-based serology data are described in more detail in ref. The absorbance and emission values collected in each assay were used to call serostatus. The cutoff values used for calling a sample positive or negative were given by the manufacturer and can be found in supplementary table 2 of ref. Milieu Intérieur plasma samples were tested for antibodies to a broad panel of common respiratory pathogens and routine vaccine-preventable diseases using bead-based multiplex assays. Samples were run at a dilution of 1:200. Plates were read using a Luminex INTELLIFLEX system and the median fluorescence intensity was used for analysis. To validate experimental associations between PhIP-seq-based Z scores and age, sex, continent of birth and smoking, the associated peptides were synthesized (Supplementary Table 1) and antibody titers against these peptides were measured by Luminex immunoassay (see next section). Peptide synthesis was performed with automated synthesizers (Genecust) and a tag was added to each peptide according to the standard protocol54, using solid-phase 9-fluorenylmethoxycarbonyl (Fmoc) chemistry. The construct included a fold-on trimerization domain and hexahistidine tag. The plasmid was transiently transfected in Expi293F (Thermo Fisher Scientific) using PEI Max (Polysciences) as a transfection reagent. One day after transfection, the flask was transferred to 32 °C and 6.5 mM sodium propionate and 50 mM glucose were added. Following incubation, the cell culture supernatant was clarified and the recombinant cH6/11 protein was captured on an Ni-NTA column (Ni-advance HiFli; Protein Ark) and stored in small aliquots at −80 °C until further use. We validated associations between PhIP-seq-based Z scores and age, sex, continent of birth and smoking by measuring antibody titers against relevant viral peptides using multiplex Luminex immunoassays. To couple viral peptides to MagPlex microspheres, we adapted the protocol from ‘Modification of microspheres with ADH'56 and Wakeman et al.54. The stock uncoupled microspheres were sonicated and vortexed for 30 s. Subsequently, 2.5 × 106 microspheres (200 µl) were transferred to an Eppendorf tube and washed once with 1 ml 0.1 M 2-(N-morpholino)ethane sulfonic acid (MES) (pH 6.0) using a magnetic separator. The 1-ethyl-3-(3-dimethylaminopropyl) carbodiimide hydrochloride was prepared extemporaneously in 0.1 M MES (pH 6.0) immediately before use. Following activation, the beads were washed three times with 0.1 M MES (pH 4.5) and resuspended in 1 ml of 0.1 M MES (pH 4.5). One day after the ADH modification of microspheres, they were washed once with 1 ml of 0.1 M MES (pH 6.0) using a magnetic separator and resuspended in 350 µl of 0.1 M MES (pH 6.0) with 20 µg of each peptide, 10 µl EDC and 10 µl hydroxysulfosuccinimide sodium salt (50 mg ml−1; Sigma–Aldrich). This suspension was incubated for 2 h 30 m in the dark on a rotator. After incubation, the beads were washed twice with 0.1 M MES (pH 6.0) and blocked with 500 µl of 0.1 M MES (pH 6.0) containing 300 µg of each peptide for 1 h at room temperature in the dark in a rotator. After blocking, the beads were washed twice with 0.1 M MES (pH 6.0) and resuspended in 1 ml PBS-TN. One day after the coupling process, all coupled beads were counted using a TC20 Automated Cell Counter (Bio-Rad). Serum samples were run at a 1:400 dilution. Plates were read using the Intelliflex system at a low detector sensitivity and the median fluorescence intensity was measured. We focused on predicting serostatus for four common viruses for which ELISA data were available: CMV, EBV (EBNA-1 and EA-D), HSV-1 and HSV-2 (Supplementary Note and Supplementary Table 2). We considered four alternative approaches: (1) the hit-based heuristic method, which assigns seropositivity for a given virus when the number of hits is >3 or >5 (as in ref. 16); (2) the hit-based optimized method, which involves searching for the number of positive hits for a given virus that maximizes prediction precision and recall; (3) the AVARDA-based optimized method, which involves searching for the threshold value of the AVARDA breadth score for a given virus that maximizes prediction precision and recall; and (4) an elastic net penalized logistic regression trained from a subset of the VirScan Z score data. We only considered VirScan peptide Z scores for the tested virus as features during feature selection. Two complementary approaches were implemented to reduce overfitting: we discarded features with variance lower than a user-specified threshold, defining a first hyper-parameter, and kept the features with univariate association statistics higher than a user-specified percentile, defining a second hyper-parameter. A grid-based approach was used to optimize the two hyper-parameters and the ratio between elastic net L1 and L2 penalty, performing a fivefold cross-validation for each point of the three-dimensional grid. We visually inspected learning curves to ensure the absence of overfitting. All of the packages were installed in a conda 24.3.0 environment for reproducibility. Ten eight-color flow cytometry panels were previously established22 to count blood cell types, including 78 counts for 27 innate immune cell subtypes and 51 adaptive immune cell subtypes. The protocols, panel design, staining antibodies and gating strategies used to acquire and analyze flow cytometry data are detailed elsewhere22. In brief, cells were acquired using two MACSQuant analyzers calibrated with MACSQuant Calibration Beads (Miltenyi Biotec). Generated MQD files were converted to FCS format and analyzed with FlowJo. The exclusion of problematic and outlier values was described previously22. Some 74 donors failed quality control for the T cell panel and were thus excluded. The remaining missing values were imputed by random forest-based imputation using the missForest R package. To evaluate whether B cell renewal affects antibody levels, we tested the association between all public peptide Z scores and circulating levels of kappa-deleting recombination excision circles (KRECs; that is, excised signal circular DNA segments generated in B cells during their maturation in bone marrow). KRECs serve as surrogates of new B cell output, as they persist in B cells and are diluted with cell division57. KREC quantification was performed as in ref. Whole-blood genomic DNA (1–2 µg) was pre-amplified for 3 min at 95 °C and then 18 cycles of 95 °C for 15 s, 60 °C for 30 s and 68 °C for 30 s, in a 50 µl reaction containing primers, 200 µM of each dNTP, 2.5 mM MgSO4 and 1.25 U Platinum Taq DNA Polymerase, High Fidelity (Thermo Fisher Scientific) in 1× buffer. Rows were loaded with an equal mixture of 2× Assay Loading Reagent and 2× Assay Biomark containing only the two primers and the probe specific for each assay. These columns were subjected to 40 cycles of PCR (95 °C for 15 s and 60 °C for 60 s) in a Biomark HD system (Fluidigm). Coding and signal joint KRECs were normalized to 150,000 cells using quantification of the albumin gene as an endogenous control. Details about SNP array genotyping of the Milieu Intérieur cohort are available elsewhere22. DNA was extracted from whole blood collected on EDTA using the Nucleon BACC3 Genomic DNA Extraction Kit (RPN8512; Cytiva). After applying quality control filters, the SNP array datasets from the two genotyping platforms were merged. SNPs that were discordant in genotypes or position between the two platforms were removed, yielding a final dataset containing 732,341 genotyped SNPs. The dataset was then phased using SHAPEIT2 (ref. We removed 13 individuals based on relatedness and admixture22. Details about SNP array genotyping of the EIP cohort are available elsewhere20. Peripheral blood mononuclear cells were isolated from blood collected into EDTA vacutainers, monocytes were removed with CD14+ microbeads, and DNA was isolated from the monocyte-negative fraction using a standard phenol–chloroform protocol, followed by ethanol precipitation. Genotyping was performed in all individuals using the HumanOmni5-Quad BeadChip (Illumina) and whole-exome sequencing was performed with the Nextera Rapid Capture Expanded Exome kit. For the SNP array data, SNPs were passed through multiple quality control filters, and SNPs originating from the sex chromosomes were removed. For the whole-exome sequencing data, reads were processed according to GATK Best Practices. Discordant variants between the two datasets were removed before merging the SNP array and whole-exome sequencing datasets. After combining the two datasets, the data were phased using SHAPEIT2 and imputed using IMPUTE2, with 1 Mb windows and a buffer region of 1 Mb. After imputation and additional quality control filtering, 19,619,457 SNPs remained. Finally, four individuals were removed based on relatedness and admixture20. Whole-genome sequencing was performed by the Centre National de Recherche en Génomique Humaine at the Institut de Biologie François Jacob. After quality control, 1 µg genomic DNA was used to prepare a library using the Illumina TruSeq DNA PCR-Free Library Preparation Kit, according to the manufacturer's instructions. After normalization and quality control, qualified libraries were sequenced on an Illumina HiSeq X5 platform as paired-end 150 bp reads. One lane of HiSeq X5 flow cell was produced for each sample in order to reach an average sequencing depth of ~30× for each sample. FASTQ files were mapped on the human reference genome version hs37d5, using BWA-MEM with default options62. Reads were realigned and recalibrated with GATK61 version 4.1. Sequencing reads mapping to the HLA, IGH, IGK and IGL loci were extracted from the mapped BAM files. Multi-sample genotype calling was performed jointly on combined GVCF files with GATK GenotypeGVCFs. After variant quality score recalibration, variants that passed the tranche sensitivity threshold of 99.0% were selected. Multiallelic sites were split into several biallelic sites with bcftools norm -m-both and variants spanning deletions were filtered out. Genotypes were set to missing if the depth of coverage was <8× or the genotype quality was <20. Based on kinship coefficients estimated with KING63, ten related individuals and one individual detected as contaminated were excluded. Finally, variants with a minor allele frequency of <0.05, a Hardy–Weinberg equilibrium P value of <10−10 (calculated using the HWExact function from the GWASExactHW R package) or a call rate of <0.95 were discarded, resulting in a total of 30,503 common variants near and within immunoglobulin genes. All statistical associations were tested using multiple regression models. In all models, the dependent variable was either an asinh-transformed VirScan Z score (for a given peptide) or an AVARDA breadth score (for a given virus). The independent variables could be: (1) serological measurements based on ELISA; (2) serological measurements based on Luminex xMAP assays; or (3) age, sex, continent of birth and candidate non-genetic factors, including smoking, diet, past diseases, health biomarkers and anthropometric measures (Supplementary Table 3). The three groups of variables, (1), (2) and (3), were treated as independent families of tests and P values were adjusted for multiple testing accordingly, using the false discovery rate procedure. Tests within the Milieu Intérieur and EIP cohorts were also considered independent. As detailed below, the specific model and complete list of covariates used varied depending on the independent variables being tested. The effect size of each independent continuous or binary variable was estimated and tested for being non-null (that is, the two-sided alternative hypothesis) using the linear regression model implemented in the glm R function. When the independent variable was categorical with more than two levels, an analysis of covariance model was applied using the aov R function. In the association analyses of the Milieu Intérieur cohort, age and sex were systematically included as covariates. We also investigated nonlinear effects of age by testing an analysis of variance model that models age as a factor with five ten-year levels. In addition, we tested for age × sex, age × smoking and sex × smoking interactions by adding an interaction term to the linear model. The only analyzed independent variables for the EIP cohort were age and continent of birth. As all individuals in the EIP cohort were male, sex was not used as a covariate in these analyses. When age was used as the variable of interest, the continent of birth was controlled for, and vice versa. To separate genetic from non-genetic effects of continent of birth, we performed an additional analysis that also included genetic variants that influenced the antibody repertoire (Supplementary Table 7; see the section ‘Estimation of the proportion of variance explained' below). To leverage the high resolution of the VirScan peptide library while accounting for between-species antibody cross-reactivity, we first tested the association between all public peptide Z scores and non-genetic factors and then evaluated whether AVARDA breadth scores for the tested viruses were associated with the corresponding factors. To correct for population stratification, a principal component analysis was run on all SNPs, and the first two principal components were included as covariates. Age was included as a covariate for both cohorts, and sex was included as a covariate for the Milieu Intérieur cohort only. The population of origin was included as an additional binary covariate for the EIP cohort. HLA allele imputation was done using whole-genome sequencing data of the HLA locus (here defined as position 28–35 Mb in GRCh37), using all variants in the region with a minor allele frequency of ≥5%. Imputation was conducted on the Michigan Imputation Server66, using the four-digit multi-ethnic HLA reference panel (version 2). Association testing was conducted similarly to individual SNP analysis but using HLA allele dosages instead of SNP genotypes. Genetic factors were the most associated SNPs identified through conditional GWAS (that is, by testing associations with all variants while controlling for hitherto identified lead SNPs). This process was continued until no more SNPs with a P value below genome-wide significance (P < 1.31 × 10−10) were identified, leaving a total of 17 SNPs (Supplementary Table 7). Age, sex and smoking were included as demographic factors. To estimate evolutionary rates for each residue of the IAV HA and M1 proteins (Fig. 2), HA sequences for all H3 subtypes and M1 sequences for the H5N1 subtype with collection dates between 1 January 1975 and 1 January 2013 were retrieved from the GISAID EpiFlu database on 13 June 2025. The accession number, virus name, collection date, originating laboratory, submitting laboratory and contributors of each individual sequence can be accessed under the accession codes EPI_SET_250807be (https://doi.org/10.55876/gis8.250807be) and EPI_SET_250807kf (https://doi.org/10.55876/gis8.250807kf), respectively. Normalized evolutionary rates were calculated for each residue with the empirical Bayesian inference from ConSurf-DB68. All UniProt amino acid sequences used to build the VirScan peptide library for RSV protein G were aligned with the msa function from the msa package69. Based on this shared region, a distance matrix between all UniProt sequences was computed with the DistanceMatrix function from the DECIPHER package70, and complete-linkage clustering was used to obtain a phylogenetic tree using the hclust R function. Strain annotations were then interpolated for all VirScan peptides using the constructed tree. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The VirScan V3 PhIP-seq raw and processed data generated in this study have been deposited in the Institut Pasteur data repository, OWEY, and can be accessed via https://doi.org/10.48802/owey.84rn-jg72 (Milieu Intérieur) and https://doi.org/10.48802/owey.uCQ5VsxD (EIP). All association statistics obtained in this study can also be explored and downloaded from http://mirepertoire.pasteur.cloud/. All other pseudonymized datasets can be accessed on OWEY by submitting a data access request at https://redcap.pasteur.fr/surveys/?s=ND8TP8MDD3 (Milieu Intérieur) or https://redcap.pasteur.fr/surveys/?s=F3AA7J4M4W8LRNJ4 (EIP). In particular, research on Milieu Intérieur and EIP datasets is restricted to research on the genetic and environmental determinants of human variation in immune responses. Data access is typically granted two months after request submission. The custom code used to generate the results presented in this study was written in R and is available via GitHub at https://github.com/h-e-g/virscan_association. Identification of host–pathogen–disease relationships using a scalable multiplex serology platform in UK Biobank. Scepanovic, P. et al. Human genetic variants and age are the strongest predictors of humoral immune responses to common pathogens and vaccines. Flanagan, K. L., Fink, A. L., Plebanski, M. & Klein, S. L. Sex and gender differences in the outcomes of vaccination over the life course. Andreu-Sánchez, S. et al. Phage display sequencing reveals that genetic, environmental, and intrinsic factors influence variation of human antibody epitope repertoire. Bourgonje, A. R. et al. Phage-display immunoprecipitation sequencing of the antibody epitope repertoire in inflammatory bowel disease reveals distinct antibody signatures. Identification of sequence variants influencing immunoglobulin levels. Hammer, C. et al. Amino acid variation in HLA class II proteins is a major determinant of humoral response to common viruses. The landscape of host genetic factors involved in immune response to common viral infections. Rubicz, R. et al. A genome-wide integrative genomic study localizes genetic factors influencing antibodies against Epstein–Barr virus nuclear antigen 1 (EBNA-1). Analysis of antibody binding specificities in twin and SNP-genotyped cohorts reveals that antiviral antibody epitope selection is a heritable trait. Hodel, F. et al. Human genomics of the humoral immune response against polyomaviruses. Larman, H. B. et al. Autoantigen discovery with a synthetic human peptidome. Larman, H. B. et al. PhIP-Seq characterization of autoantibodies from patients with multiple sclerosis, type 1 diabetes and rheumatoid arthritis. Vogl, T. et al. Population-wide diversity and stability of serum antibody epitope repertoires against human microbiota. Measles virus infection diminishes preexisting antibodies that offer protection from other pathogens. The repertoire of maternal anti-viral antibodies in human newborns. The Milieu Intérieur study—an integrative approach for study of human immunological variance. Quach, H. et al. Genetic adaptation and Neandertal admixture shaped the immune system of human populations. Monaco, D. R. et al. Deconvoluting virome-wide antibody epitope reactivity profiles. Patin, E. et al. Natural variation in the parameters of innate immune cells is preferentially driven by genetic factors. Goyer, M., Aho, L.-S., Bour, J.-B., Ambert-Balay, K. & Pothier, P. Seroprevalence distribution of Aichi virus among a French population in 2006–2007. Cretescu, L., Beare, A. S. & Schild, G. C. Formation of antibody to matrix protein in experimental human influenza A virus infections. Aquino, Y. et al. Dissecting human population variation in single-cell responses to SARS-CoV-2. Chatlynne, L. G. & Ablashi, D. V. Seroepidemiology of Kaposi's sarcoma-associated herpesvirus (KSHV). Zuhair, M. et al. Estimation of the worldwide seroprevalence of cytomegalovirus: a systematic review and meta-analysis. Palser, A. L. et al. Genome diversity of Epstein–Barr virus from multiple tumor types and normal infection. & Smith, A. P. Smoking, alcohol consumption, and susceptibility to the common cold. et al. Cigarette smoke selectively enhances viral PAMP- and virus-induced pulmonary innate immune and remodeling responses in mice. Kurki, M. I. et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Prospective virome analyses in young children at increased genetic risk for type 1 diabetes. Chiou, J. et al. Interpreting type 1 diabetes risk with genetics and single-cell epigenomics. Jostins, L. et al. Host–microbe interactions have shaped the genetic architecture of inflammatory bowel disease. Lindesmith, L. et al. Human susceptibility and resistance to Norwalk virus infection. Reuter, G., Pankovics, P. & Boros, Á Saliviruses—the first knowledge about a newly discovered human picornavirus. Rodriguez, O. L. et al. Genetic variation in the immunoglobulin heavy chain locus shapes the human antibody repertoire. Avnir, Y. et al. IGHV1-69 polymorphism modulates anti-influenza antibody repertoires, correlates with IGHV utilization shifts and varies by ethnicity. Zaslavsky, M. E. et al. Disease diagnostics using machine learning of B cell and T cell receptor sequences. Fuentes, S., Coyle, E. M., Beeler, J., Golding, H. & Khurana, S. Antigenic fingerprinting following primary RSV infection in young children identifies novel antigenic sites and reveals unlinked evolution of human antibody repertoires to fusion and attachment glycoproteins. Zlateva, K. T., Vijgen, L., Dekeersmaeker, N., Naranjo, C. & Van Ranst, M. Subgroup prevalence and genotype circulation patterns of human respiratory syncytial virus in Belgium during ten successive epidemic seasons. Nachbagauer, R. et al. Age dependence and isotype specificity of influenza virus hemagglutinin stalk-reactive antibodies in humans. Miller, M. S. et al. Neutralizing antibodies against previously encountered influenza virus strains increase over time: a longitudinal analysis. The influence of evolutionary history on human health and disease. Kerner, G. et al. Genetic adaptation to pathogens and increased risk of inflammatory disorders in post-Neolithic Europe. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Adjusting batch effects in microarray expression data using empirical Bayes methods. Wakeman, B. S. et al. Development of a new peptide–bead coupling method for an all peptide-based Luminex multiplexing assay for detection of Plasmodium falciparum antibody responses. & Palese, P. Chimeric hemagglutinin influenza virus vaccine constructs elicit broadly protective stalk-specific antibodies. Angeloni, S. D. S., Dunbar, S., Stone, V. & Swift, S. xMAP® Cookbook: a Collection of Methods and Protocols for Developing Multiplex Assays with xMAP® Technology 54–55 (Luminex, 2018). Glauzy, S. et al. Impact of acute and chronic graft-versus-host disease on human B-cell generation and replication. Improved whole-chromosome phasing for disease and population genetic studies. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. Van der Auwera, G. & O'Connor, B. D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra (O'Reilly Media, 2020). Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Robust relationship inference in genome-wide association studies. Gogarten, S. M. et al. GWASTools: an R/Bioconductor package for quality control and analysis of genome-wide association studies. Juliusdottir, T. topr: an R package for viewing and annotating genetic association results. Luo, Y. et al. A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response. Relative importance for linear regression in R: the package relaimpo. & McCauley, J. GISAID: global initiative on sharing all influenza data—from vision to reality. Bodenhofer, U., Bonatesta, E., Horejš-Kainrath, C. & Hochreiter, S. msa: an R package for multiple sequence alignment. Wright, E. S. Using DECIPHER v2.0 to analyze big biological sequence data in R. R J. We thank the HPC Core Facility of Institut Pasteur for supporting this work. Pre-processing of the VirScan data was supported by resources from the National Academic Infrastructure for Supercomputing in Sweden, partially funded by the Swedish Research Council (2022-06725). We thank J. Vahokoski, R. J. Cox Brokstad and F. Krammer for providing the purified, chimeric cH6/1 protein. We gratefully acknowledge all data contributors for generating the genetic sequences and metadata and sharing via the GISAID Initiative, on which some of this research is based. This study is sponsored by the Institut Pasteur (Pasteur ID-RCB number: 2012-A00238-35) and was funded by the French Government's Investissements d'Avenir program, managed by the Agence Nationale de la Recherche (ANR-10-LABX-69-01). The E.S.-L. laboratory is funded by the INCEPTION program (Investissements d'Avenir; ANR-16-CONV-0005), Ixcore Foundation for Research and HERA Project DURABLE (101102733). These authors jointly supervised this work: Lluis Quintana-Murci, Etienne Patin. Axel Olin, Anthony Jaquaniello, Maguelonne Roux, Dang Liu, Maxime Rotival, Lluis Quintana-Murci & Etienne Patin Department of Women's and Children's Health, Karolinska Institutet, Solna, Sweden Data Management Platform, Institut Pasteur, Paris, France Evolutionary Genomics of RNA Viruses unit, Institut Pasteur, Université Paris Cité, CNRS UMR 2000, Paris, France Florian Dubois, Bruno Charbit, Darragh Duffy & Milena Hasan Françoise Donnadieu, Laura Garcia, Camille Lambert, Emma Bloch & Michael White Emmanuel Clave, Itauá Leston Araujo & Antoine Toubert Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM U1163, Necker Hospital for Sick Children, Paris, France St. Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY, USA Unité Biologie Cellulaire des Lymphocytes, Institut Pasteur, Université Paris Cité, INSERM U1224, Ligue Nationale Contre le Cancer, Équipe Labellisée Ligue-2018, Paris, France Discipline of Medical Gerontology, School of Medicine, Trinity Translational Medicine Institute, Trinity College Dublin, Dublin, Ireland Genome Integrity, Immunity and Cancer Unit, Institut Pasteur, Université Paris Cité, INSERM U1223, Équipe Labellisée Ligue Contre Le Cancer, Paris, France Unité Biologie et Génétique de la Paroi Bactérienne, Institut Pasteur, Université Paris Cité, CNRS UMR6047, INSERM U1306, Paris, France Department of Microbiology, Tumor and Cell Biology, Karolinska Institutet, Stockholm, Sweden Nutritional Epidemiology Research Team, Centre for Research in Epidemiology and Statistics, Université Sorbonne Paris Nord and Université Paris Cité, INSERM, INRAE, CNAM, Paris, France Nutrition And Cancer Research Network, Jouy-en-Josas, France Department of Immunology, Institut Pasteur, Paris, France Laboratoire d'Immunologie Clinique, Institut Curie, INSERM U932, Paris, France Centre d'investigation Clinique en Biothérapie Gustave-Roussy Institut Curie (CIC-BT1428), Paris, France Laboratory of Single-Cell Inflammatory Responses and Multi-OMICs Networks, Imagine Institute, Université Paris Cité, INSERM UMR1163, Paris, France Fédération de Génétique et Médecine Génomique, Service de Médecine Génomique des Maladies Rares, Assistance Publique-Hôpitaux de Paris, Necker Hospital for Sick Children, Paris, France Laboratory of Immunogenetics of Pediatric Autoimmune Diseases, Imagine Institute, Université de Paris, INSERM UMR1163, Paris, France Ecology and Emergence of Arthropod-borne Pathogens Unit, Institut Pasteur, Université Paris Cité, CNRS UMR2000, Paris, France ICAReB-Biobank, Centre de Ressources Biologiques, Institut Pasteur, Paris, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar J.C. conducted viral conservation analyses under the supervision of E.S.-L. A.O. performed all of the remaining analyses, with contributions from A.J., M. Roux, M. Rotival and D.L., under the supervision of E.P. Correspondence to Lluis Quintana-Murci or Etienne Patin. Nature Immunology thanks Amalio Telenti and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Laurie A. Dempsey, in collaboration with the Nature Immunology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,b, Number of peptides in the VirScan PhIP-seq library, separated by (a) viral family and (b) species. Only the 50 most covered viruses are shown. c, Percentage of Milieu Intérieur (left) and EIP (right) individuals positive for 2,608 public peptides, separated by virus. Only viruses with at least 10 peptides showing an enrichment of >5% are included. a, Principal component analysis of the Milieu Intérieur dataset, before batch correction (n = 2,608 public peptide Z-scores). b, Principal component analysis of the Milieu Intérieur dataset, after batch correction. c, Variance explained by batches before batch correction against that explained by age after batch correction, in the Milieu Intérieur cohort. d, Principal component analysis of the Milieu Intérieur dataset, after batch correction. e, Variance explained by batches before batch correction against that explained by sex after batch correction, in the Milieu Intérieur cohort. f, Principal component analysis of the EIP dataset, before batch correction (n = 3,210 public peptide Z-scores). g, Principal component analysis of the EIP dataset, after batch correction. Colors indicate continent of birth (AFB: Belgians born in Central Africa; EUB: Belgians born in Europe). h, Variance explained by batches before batch correction against that explained by continent of birth after batch correction, in the EIP cohort. i, Variance explained by batches before batch correction in EUB samples only against that explained by continent of birth after batch correction, in the EIP cohort. c,e,h,i, Colors indicate viral families. a,g, −log10(P-values) for the association between serology determined by (a) ELISA or (g) Luminex xMAP and the AVARDA breadth scores. Serology variables are plotted on the x-axis. The top two AVARDA associations are connected by a black vertical line. Significant associations (FDR < 0.05) are colored in green or red if the association is or is not for the cognate virus, respectively. b-f, −log10(P-values) for the association between the AVARDA breadth scores and ELISA-based serostatus for (b) CMV, (c) EBV VCA antigen, (d) rubella virus, (e) IAV, and (f) hepatitis B. h-l, −log10(P-values) for the association between the AVARDA breadth scores and Luminex-based serology for (h) CMV, (i) EBV VCA antigen, (j) rubella virus, (k) IAV, and (l) hepatitis B. Significant associations (FDR < 0.05) are colored in green or red if the association is or is not for the cognate virus, respectively. a,b, −log10(P-values) for the association between the VirScan Z-scores and serology determined by (a) ELISA or (b) Luminex xMAP. Significant associations (FDR < 0.05) are colored in green or red if the association is or is not for the cognate virus, respectively. c,e, −log10(P-values) for the association between the VirScan Z-scores and ELISA-determined serostatus for (c) CMV and (e) IAV. Significant associations (FDR < 0.05) are colored in green or red if the association is or is not for the cognate virus, respectively. d,f, −log10(P-values) for the association between the VirScan Z-scores and Luminex-determined serology for (d) CMV and (f) IAV. Significant associations (FDR < 0.05) are colored in green or red if the association is or is not for the cognate virus, respectively. g, VirScan Z-score distributions for the three peptides most significantly associated with ELISA-based serostatus for the EBV antigens EA, EBNA and VCA. ELISA-based serostatus for each EBV antigen is shown on the x-axis. h, Luminex-based serologies for the spike and nucleocapsid proteins shown for the four coronaviruses 229E, HKU1, NL63, and OC43. a, −log10(adjusted P-values) and direction of associations between AVARDA breadth scores and immune cell phenotypes in the Milieu Intérieur cohort, by viral species. Each point indicates an association between an immune cell phenotype and the AVARDA antibody breadth score for a given virus, colored according to its viral family. b, −log10(adjusted P-values) of associations between the frequency of CD4+ TEMRA cells and CMV peptide Z-scores, separated by viral protein. c, −log10(adjusted P-values) and direction of associations between all public peptide Z-scores and the HLA-DR MFI in the cDC1 population, by viral species. a-c, The strength and direction of associations are indicated by the size and direction of triangles, respectively. a,b, −log10(adjusted P-values) against effect sizes for associations between the AVARDA breadth score and age in the Milieu Intérieur (a) and EIP (b) cohorts (two-sided Wald test). Each point indicates the AVARDA breadth score for a given a virus, colored according its viral family. c,d, Effect sizes for the associations between age and HSV-1 (c) and EBV (d) peptide Z-scores in the Milieu Intérieur and EIP cohorts. e,f, −log10(adjusted P-values) of associations between age and (e) HSV-1 and (f) EBV peptide Z-scores, separated by viral protein, in the Milieu Intérieur cohort (two-sided Wald test). The strength and direction of associations are indicated by the size and direction of triangles, respectively. a, −log10(adjusted P-values) and direction of associations between all public peptide Z-scores and sex in the Milieu Intérieur cohort (two-sided Wald test). The dashed gray vertical lines indicate viruses for which the AVARDA breadth score is significantly associated with sex. b, −log10(adjusted P-values) against effect sizes for associations between the AVARDA breadth score and sex in the Milieu Intérieur cohort (two-sided Wald test). Each point indicates a virus, colored according to its viral family. c,d, −log10(adjusted P-values) against effect sizes for associations between (c) IAV and (d) IBV peptide Z-scores and sex in the Milieu Intérieur cohort, colored according to the viral protein (two-sided Wald test). a, −log10(adjusted P-values) against effect sizes for associations between the AVARDA breadth score and continent of birth in the EIP cohort (two-sided Wald test). Each point indicates a virus, colored according to its viral family. b,c, −log10(adjusted P-values) of associations between continent of birth and HHV-8 (b) and CMV (c) peptide Z-scores, separated by viral protein, in the EIP cohort (two-sided Wald test). The strength and direction of associations are indicated by the size and direction of triangles, respectively. d, −log10(adjusted P-values) and direction of associations between all public peptide Z-scores and continent of birth in the EIP cohort, separated by viral species, after adjusting on genotypes of the lead variants identified by GWAS (Supplementary Table 7) (two-sided Wald test). The dashed gray vertical lines indicate viruses for which the AVARDA breadth score is significantly associated with continent of birth. e, −log10(adjusted P-values) against effect sizes for associations between IAV peptide Z-scores and continent of birth in the EIP cohort, faceted by main IAV subtypes (two-sided Wald test). f,g, Amino-acid positions of the midpoint of HA (f) and NP (g) peptides associated with continent of birth within the full IAV proteins for the EIP cohort (two-sided Wald test). a,b, LocusZoom plots showing associations between the FUT2 locus and antibody reactivity against (a) norovirus (UniProt ID: Q8V0P3) and (b) salivirus A (UniProt ID: D1L752) (two-sided Wald test). c,d, Scatter plots (c) and correlation matrix (d) for the three norovirus and salivirus peptide Z-scores most significantly associated with FUT2 variants. e, LocusZoom plot showing associations between the IGK locus and antibody reactivity against adenovirus B (UniProt ID: C7SRU6; two-sided Wald test). a, Proportion of variance explained by age, sex, smoking, and genetics for antibody reactivity against public peptides from the six viruses with the largest number of public peptides in the Milieu Intérieur cohort. Peptides are sorted by total variance explained. b, Proportion of variance explained by genetic factors for antibody reactivity against public peptides from the six viruses with the largest number of public peptides in the Milieu Intérieur cohort. Genetic variance is separated by genetic variation in the HLA and IGH loci and variation external to these loci. Peptides are sorted by total variance explained. c,d, Variance explained by (c) age and IGH genetic variation for IAV peptides and (d) age and HLA genetic variation for EBV peptides in the Milieu Intérieur cohort, colored according to viral protein. e, Amino-acid positions of the midpoint of protein G peptides associated with age within the full RSV protein G for the EIP cohort (two-sided Wald test). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Olin, A., Pou, C., Jaquaniello, A. et al. Demographic and genetic factors shape the epitope specificity of the human antibody repertoire against viruses. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41589-025-02137-2'>High-throughput ligand diversification to discover chemical inducers of proximity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 10:30:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Chemical inducers of proximity (CIPs) stabilize biomolecular interactions, often causing an emergent rewiring of cellular biochemistry. While the discovery of heterobifunctional CIPs is expedited by rational design strategies, molecular glues have relied predominantly on serendipity. We hypothesized that preexisting ligands could be systematically decorated with chemical modifications to discover compounds that recruit proteins to a composite protein–ligand interface. Using sulfur(VI) fluoride exchange-based high-throughput chemistry (HTC) to install 3,163 structurally diverse building blocks onto ENL (eleven-nineteen leukemia) and BRD4 (bromodomain-containing protein 4) ligands, we screened each analog for degrader activity. This revealed dHTC1, an ENL degrader that recruits CRL4CRBN complex through an extended interface of protein–protein contacts and only engages CRBN after pre-forming the ENL:dHTC1 complex. We also identified dHTC3, a molecular glue that selectively dimerizes BRD4 bromodomain 1 to SCFFBXO3, an E3 ligase not previously accessible for chemical rewiring. Altogether, this study introduces HTC as a facile tool to discover new CIPs and new effectors for proximity pharmacology. Chemical inducers of proximity (CIPs) include bifunctional small molecules and molecular glues that induce proximity between two targets, rewiring biological circuits in a manner that is difficult to achieve with other pharmacological approaches1,2. The first CIPs were discovered serendipitously by studying the mechanisms of action of natural product immunosuppressants3,4; however, but heterobifunctional chemistries, exemplified by proteolysis-targeting chimeras (PROTACs), have made it possible to design CIPs prospectively5,6. Molecular glues function as CIPs without needing to bind the target and effector independently; instead, they can form a composite protein–ligand surface that cooperatively stabilizes an interfacial protein–protein interaction7. For example, thalidomide analogs (or immunomodulatory imide drugs, IMiDs)—anticancer agents that function through this mechanism—resurface the E3 substrate receptor, Cereblon (CRBN), to degrade unligandable neosubstrates8,9,10,11,12,13,14,15,16,17. These drugs highlight the potential for molecular glues to address difficult classes of drug targets, yet prospective approaches to discover molecular glues remain limited. Lacking modular design principles like bifunctional CIPs, most molecular glues have been discovered through retrospective mechanism-of-action studies1,2. We considered it instructive that several recently discovered molecular glues possess close structural analogs that bind the same target but do not function as glues18,19,20,21,22,23,24,25. In several examples, CIP behavior can be attributed to subtle structural differences that project outward toward the solvent, forming a composite protein–ligand surface that can stabilize an interfacial interaction19,20,25. However, because of the sporadic and primarily serendipitous nature of these discoveries, it is unclear how often structural alterations can confer glue-type activity to a ligand and whether these alterations can be identified prospectively for a preselected target. To address this, we coupled a high-throughput approach for ligand diversification to a phenotypic readout of neomorphic pharmacological behavior—targeted protein degradation—enabling the prospective discovery of structural alterations that convert ligands into CIPs. To establish an initial proof of concept, we selected SR-0813, a ligand we previously discovered that binds to the YEATS domain of the leukemia target and transcriptional coactivator eleven-nineteen leukemia (ENL)26. As SR-0813 does not affect ENL stability (Extended Data Fig. 1a), we reasoned that high-throughput diversification could be coupled to cellular ENL degradation screens to identify structural modifications that convert it into a CIP. To test this, we used sulfur(VI) fluoride exchange (SuFEx)-based ‘click chemistry' transformations, in which an iminosulfur oxydifluoride group reacts with primary and secondary aliphatic amines to completion in DMSO–PBS mixtures, forming sulfamides or sulfuramidimidoyl fluorides (Fig. These mild, biocompatible reaction conditions allow crude products to be tested directly in living cells, thus marrying high-throughput chemistry (HTC) with miniaturized cell-based screens26,29,30. a, TM-7 (1) with an iminosulfur oxydifluoride SuFEx handle (2) was used for the parallel synthesis of 3,163 analogs. b, Crude reaction products were screened for ENL degradation (5 µM, 24 h) in MV4;11 cells expressing HiBiT–ENL. HiBiT luminescence normalized to DMSO–PBS vehicle-control-treated cells (single independent experiment). Dashed line indicates 2 standard deviations below the mean. c, Resynthesized hits were rescreened by HiBiT–ENL and counterscreened against BRD4–HiBiT (n = 3 biological replicates). HiBiT luminescence normalized to DMSO–PBS vehicle-control-treated cells. Dashed lines indicate average of DMSO-treated cells. e, Immunoblot of ENL in AML cell lines treated with purified rac-dHTC1 (10 µM, 24 h) or DMSO vehicle control. Data are representative of two independent experiments. f, Time-dependent and dose-responsive loss of HiBiT–ENL signal in MV4;11 cells treated with rac-dHTC1. Luminescence is normalized to the DMSO vehicle control (mean ± s.e.m. HiBiT–ENL luminescence normalized to DMSO vehicle control (mean ± s.e.m. 2e were taken from the same experiment; DMSO pretreatment was repeated in both panels. h, Quantitative TMT-based expression proteomics analysis of MV4;11 cells treated with rac-dHTC1 (1 µM, 16 h) or DMSO vehicle control (5,513 proteins with >2 peptides). Data were filtered using DTAselect 2.0 within IP2. P values were calculated using a two-tailed Student's t-test (n = 3 biological replicates). Black dots indicate ENL target genes. We first furnished TM-7 (1), a synthetically simplified analog of SR-0813 (Fig. 1b,c), with an iminosulfur oxydifluoride SuFEx hub to afford TM-7 difluoride (2) and sourced a library of 3,163 commercially available amine building blocks encompassing diverse chemical structures, molecular weights and physiochemical properties (Extended Data Fig. We then synthesized more than 3,000 TM-7 analogs in parallel and screened each crude product for degradation of HiBiT–ENL (Fig. A total of 93 hits scoring more than 2 s.d. below the mean were promptly resynthesized, retested for ENL degradation in a newly optimized MV4;11 HiBiT–ENL cell line and counterscreened against MV4;11 cells expressing bromodomain-containing protein 4 (BRD4)–HiBiT to remove false positives, as TM-7 itself does not bind BRD4 (Fig. One hit, the crude product of TM-7 difluoride and a pyrrolidine spirosuccinimide building block JS-1 (3), selectively decreased HiBiT–ENL signal (Fig. It was resynthesized and purified to enable further validation, yielding rac-dHTC1 (4) as a 1:1 mix of enantiomers (Supplementary Methods and Fig. Using a panel of acute myeloid leukemia (AML) cell lines, degradation of native ENL was confirmed by immunoblot (Fig. 1h) and dose-responsive degradation was assessed by HiBiT, determining half-maximal degradation concentration (DC50) values ranging from 50 nM to 1.85 µM and maximal degradation (Dmax) ranging from 91% to 58% (Extended Data Fig. In MV4;11 cells, degradation was observed within 30 min, maximized after 24 h and sustained for at least 5 days (Fig. Pretreatment with a saturating concentration of the ENL/AF9 YEATS ligand, SR-0813, prevented loss of HiBiT–ENL signal upon rac-dHTC1 treatment (Fig. 1g), which established that on-target engagement of ENL is necessary for its activity. Furthermore, loss of HiBiT–ENL signal could also be rescued by the neddylation inhibitor, MLN4924, and the proteasome inhibitor, carfilzomib (Fig. 1g), indicating that rac-dHTC1 prompts proteasome-dependent degradation of ENL through a Cullin–RING ligase (CRL) complex. We used an expression proteomics analysis to evaluate the selectivity of rac-dHTC1 in MV4;11 cells, finding that it exclusively degrades ENL while also decreasing the abundance of proteins encoded by transcriptional ENL target genes (for example, HOXA10 and MYC) (Fig. 1h and Supplementary Table 4), a known effect of ENL degradation in MV4;11 cells26. AF9, a paralog of ENL that is also bound by this chemical series26, was not detected by proteomics; however, an immunoblot analysis confirmed that it can also be degraded by dHTC1 (Extended Data Fig. We next used a forward genetic screen to identify the effectors responsible for dHTC1 activity, which was enabled by a dual fluorescence ENL stability reporter (ENL–TagBFP–P2A–mCherry) expressed in KBM7 cells (Extended Data Fig. A ubiquitin proteasome system (UPS)-focused library of single guide RNAs (sgRNAs) was screened in these cells for the ability to modify the activity of rac-dHTC1 or SR-1114 (Fig. 1a and 2b), a CRBN-based ENL/AF9 PROTAC that we previously disclosed26. Consistent with the known mechanisms of CRBN-based PROTACs37, SR-1114 activity was most impacted by sgRNAs targeting CRL4CRBN, the proteasome and the constitutive photomorphogenesis 9 signalosome (Extended Data Fig. Strikingly, the same machinery was identified for rac-dHTC1, with CRBN being the most substantially enriched hit (Fig. a, FACS-based CRISPR screens for UPS components affecting ENL stability in KBM7 reporter cells (ENL–BFP–P2A–mCherry) treated with DMSO or rac-dHTC1 (10 µM) for 8 h (6 sgRNAs targeting 1,301 UPS-associated genes). Gene-level fold changes and P values were determined by one-sided MAGeCK analysis114. b, Immunoblot analysis of WT treated with rac-dHTC1 (10 µM, 6 h) or DMSO vehicle control. Data are representative of two independent experiments. c, SPR analysis of CRBN–DDB1 binding to immobilized ENL YEATS in the presence of rac-dHTC1 or buffer control (mean; n = 2 technical replicates). The same experiment is presented in Extended Data Fig. 3g; rac-dHTC1 treatment was repeated in both panels. d, Rescue of dBET6-mediated BRD4 degradation to assess CRBN target engagement. MV4;11 cells expressing BRD4–HiBiT were pretreated with compounds in a dose–response series for 2 h before a 1-h treatment with dBET6 (500 nM). HiBiT luminescence signal is normalized to DMSO-treated cells (mean ± s.e.m. 4e, which originated from the same experiment. e, SPR analysis of pomalidomide binding to immobilized CRBNmidi (mean ± s.d. f, Same as e, but for rac-dHTC1. g, Heat map depicting CRBN FP signal (background-subtracted and normalized to DMSO) following treatment with the indicated compounds, in the presence of ENL (3 µM) or buffer control (mean; n = 3 biological replicates). Cooperativity values are calculated from the ratio of IC50 values with and without ENL. h, Structures of dHTC1 enantiomers and their time-dependent and dose-responsive loss of HiBiT–ENL signal in MV4;11 cells treated with rac-dHTC1. The HiBiT luminescence signal was normalized to the DMSO vehicle control (mean ± s.e.m. Using population-level CRBN-knockout (KO) MV4;11 cells30, we confirmed that CRBN is required for the degradation of ENL by rac-dHTC1 (Extended Data Fig. We also tested whether overexpression of CRBN can improve rac-dHTC1 activity in MOLM-13 cells (Extended Data Fig. 1b), where its activity was found to be relatively weak (Fig. Indeed, the potency and depth of degradation were both improved in proportion to the extent of CRBN overexpression (Extended Data Fig. Furthermore, a saturating concentration of pomalidomide rescued ENL degradation (Fig. 2e), pointing toward a mechanism whereby rac-dHTC1 contacts the IMiD-binding site of CRBN. This would presumably rely on its spirosuccinimide moiety, which bears resemblance to both the glutarimide motif present in thalidomide analogs and the asparagine cyclic imide modification proposed to be a natural degron recognized by CRBN38,39,40. In vitro, we could demonstrate that the ENL YEATS domain interacts with CRBN–DDB1 (damage-specific DNA-binding protein 1) and the newly reported CRBNmidi construct41 but only in the presence of the compound (Fig. We also used affinity purification–mass spectrometry (AP–MS) to map the interactions of CRBN globally42, which revealed an exquisitely selective and dHTC1-dependent enrichment of ENL, AF9 and other members of the super elongation complex (SEC) (Extended Data Fig. Interestingly, we found that dHTC1 did not enrich for any common CRBN neosubstrate targets (using a previously collated list43), nor did it degrade any (Extended Data Fig. This contrasts with SR-1114, which induces degradation of Ikaros-family zinc-finger protein 1 (IKZF1), as we previously reported26. Several observations indicated that dHTC1 does not function like a conventional CRBN-based degrader. First, using a CRBN engagement assay based on the competitive displacement of the CRBN-based BRD4 PROTAC dBET6 (refs. 44,45,46), we failed to detect target engagement in cells (Fig. Whereas SR-1114 and the CRBN ligand pomalidomide both blocked dBET6-induced BRD4–HiBiT degradation by engaging CRBN, neither rac-dHTC1 nor its spirosuccinimide building block JS-1 could do so (Fig. Second, in contrast to the submicromolar affinity we observed for pomalidomide binding to CRBNmidi by surface plasmon resonance (SPR) (Fig. 2e), dHTC1 showed weak and nonspecific (that is, unsaturable) binding and an estimated KD of nearly 50 µM (Fig. To further assess target engagement in vitro, we used a competitive fluorescence polarization (FP) assay that reports on the displacement of pomalidomide–BODIPY from purified preparations of recombinant CRBN–DDB1 (ref. Unlike thalidomide, which showed submicromolar potency in dose–response assays, we did not detect inhibition by rac-dHTC1 until it reached a concentration of 50 µM (Fig. Even at this high concentration—orders of magnitude above its DC50—we did not observe complete inhibition of the FP (Fig. Likewise, JS-1 showed no inhibition of the CRBN FP assay, even at 50 µM (Fig. On the basis of these data, we hypothesized that rac-dHTC1 might require a preformed complex with ENL to engage the IMiD-binding site with high affinity. Indeed, addition of purified recombinant ENL YEATS domain to the FP assay shifted the half-maximal inhibitory concentration (IC50) of rac-dHTC1 from 23 µM to 106 nM (Fig. 2g), resulting in a calculated 220-fold cooperativity α-factor47. By comparison, the thalidomide-based ENL PROTAC, SR-1114, also showed a shift in the presence of ENL YEATS but the cooperativity was substantially lower (α = 2.3) and it was fully capable of inhibiting FP signal in the absence of ENL YEATS (Fig. In fact, without ENL YEATS, the activity of SR-1114 (IC50 = 155 nM) closely matched its parent ligand, thalidomide (IC50 = 197 nM), highlighting the difference between dHTC1 and traditional CRBN-based degraders that engage CRBN independently. To evaluate the mechanism of CRBN engagement further, we assessed the dHTC1 structure–activity relationship (SAR) by replacing the sulfamide connection with an amide, yielding rac-dHTC1-amide (5), or with a urea, yielding rac-dHTC1-urea (6). These analogs showed relatively similar binding to ENL in vitro and in cells but neither could provoke ENL degradation (Extended Data Fig. Both analogs showed weak inhibition of the CRBN FP assay—in both the presence and the absence of ENL—and neither compound could mediate ternary complex formation (Extended Data Fig. 3e–g), demonstrating that dHTC1 activity is dependent on its ability to cooperatively engage CRBN. Therefore, we would not expect that its spirosuccinimide modification would yield an active degrader when installed onto ligands for other targets. To test this, we synthesized two bromodomain and extraterminal (BET) ligands and two poly(ADP-ribose) polymerase (PARP1/2) ligands harboring JS-1 at previously established exit vector positions, none of which succeeded in degrading their targets (Extended Data Fig. To further probe dHTC1 SAR, we separated its two enantiomers by preparative supercritical fluid chromatography (SFC) and assigned absolute stereochemical configurations using vibrational circular dichroism (VCD) (Fig. Whereas (S)-dHTC1 (7) was able to elicit potent ENL degradation, (R)-dHTC1 (8) was completely inactive (Fig. Both enantiomers bound to ENL identically in vitro (KD = 49–52 nM) and in cells (half-maximal effective concentration = 8 µM) and both showed the same weak CRBN engagement as the racemate (Extended Data Fig. However, (S)-dHTC1 showed potent and cooperative engagement of CRBN in the presence of ENL (Extended Data Fig. 4g), resulting in the stereoselective formation of a ternary complex (Extended Data Fig. Therefore, although CRBN shows low affinity for dHTC1 and no detectable binding to ENL up to a concentration of 1 µM, the stable, preformed ENL:(S)-dHTC1 complex can cooperatively engage CRBN with high apparent affinity to induce stereochemistry-dependent ENL degradation. Interestingly, we observed a strong hook effect for ternary complex formation in vitro using SPR and time-resolved fluorescence resonance energy transfer (TR-FRET) assays (Extended Data Fig. This might relate to the lack of cooperativity seen for ENL engagement by dHTC1 (Extended Data Fig. 4i), which contrasts with its highly cooperative engagement of CRBN. However, the observation of a hook effect is context dependent and influenced by the relative concentrations of all three members of a ternary complex48, which may differ between native cellular environments and reconstituted biochemical assays. Therefore, it is unclear how the strong hook effect we observed in vitro is relevant to the cooperative formation of a ternary complex in cells. (S)-dHTC1 produces only a weak hook effect in degradation assays (Fig. 4a), suggesting that the formation of a ternary complex is favored over the binary complexes under physiologic conditions. Indeed, in the presence of CRBN overexpression, (S)-dHTC1 induces maximal degradation at concentrations spanning more than three orders of magnitude with no apparent hook effect at any time point (Extended Data Fig. While the racemate produces a stronger hook effect in degradation assays, this occurs only at early time points and high concentrations (Fig. 1f), which could possibly be attributed to competition from the equimolar presence of (R)-dHTC1, which binds to ENL with equal affinity. Given the weak independent CRBN engagement by dHTC1 (Extended Data Fig. 4e), we interpret the totality of data to suggest that degradation by dHTC1 is dependent on the cooperative engagement of CRBN. The ENL-dependent binding of dHTC1 to CRBN invokes a distinct mechanism of action relative to prior CRBN-based degraders. To further characterize this mechanism, we coupled deep mutational scanning (DMS) of CRBN to a fluorescence-activated cell sorting (FACS)-based assessment of ENL degradation in the presence of rac-dHTC1 or the ENL PROTAC, SR-1114 (Extended Data Fig. Using our previously described DMS library that covers all possible point mutations within 10 Å of the IMiD-binding site of CRBN49, we found that rac-dHTC1 and SR-1114 are sensitive to similar CRBN mutations (Extended Data Fig. However, several mutations were suggested by the screen to preferentially confer resistance to either rac-dHTC1 or SR-1114, which we validated by immunoblot and flow cytometry (Extended Data Fig. To investigate how these mutations might impact dHTC1 activity selectively, we determined the cocrystal structure of (S)-dHTC1 bound to CRBN at a resolution of 2.2 Å using CRBNmidi (Extended Data Fig. This showed (S)-dHTC1 occupying the IMiD-binding site of CRBN, positioning its spirosuccinimide moiety within the tritryptophan cage formed by W380, W386 and W400 (Extended Data Fig. However, apart from the sulfamide making hydrogen-bonding contacts with the side chains of N351 and W400, the remainder of dHTC1, which is responsible for binding ENL, makes minimal contacts with CRBN. Focusing on mutations that selectively affected the activity of dHTC1 or SR-1114, we found that these residues generally mapped to distinct hemispheres of the IMiD-binding site (Extended Data Fig. As several of these residues are buried within the IMiD-binding site, we hypothesized that they might reflect differences in the binding mode of the dHTC1 spirosuccinimide compared to traditional glutarimide-based CRBN ligands. To evaluate this, we compared our DMS results to a previously published screen of CC-885, a thalidomide analog that degrades G1 to S phase transition 1 (GSPT1) (Extended Data Fig. This comparison indicated that a W386Y substitution, which impacts the activity of dHTC1 but not SR-1114, also does not impact CC-885 activity (Extended Data Fig. Likewise, several P352 substitutions that affected the activity of SR-1114 but not dHTC1 also affected CC-885 (Extended Data Fig. We also identified compound-selective mutations mapping to regions outside of the direct IMiD-binding site. For example, P54 and G61 substitutions, which affect dHTC1 more broadly than SR-1114 (Extended Data Fig. 5e,f), are located far from the IMiD-binding site on another face of CRBN (Extended Data Fig. Another, H353, which is adjacent to the IMiD-binding site but presents outward into the solvent (Extended Data Fig. 6f), showed varying behavior, with some substitutions impairing dHTC1 but not SR-1114 or CC-885 and others affecting SR-1114 and CC-885 but not dHTC1 (Extended Data Figs. To understand how dHTC1 recruits CRBN when bound to ENL, we determined a ternary complex structure of DDB1∆BPB•CRBN•(S)-dHTC1•ENL YEATS by cryo-electron microscopy (cryo-EM) refined to a global resolution of 2.6 Å (Supplementary Table 10). To obtain insight into the CRBN–ENL interface, we conducted local refinements focusing on CRBN•(S)-dHTC1•ENL YEATS leading to a resolution of 2.9 Å (Fig. 3a,b and Extended Data Fig. The amido-imidazopyridine core of dHTC1 inserts into the acetyllysine-binding channel of ENL and makes canonical contacts with F28, H56, H59, S58 and Y78 (Extended Data Fig. The spirosuccinimide of (S)-dHTC1 engages the IMiD-binding site of CRBN but, interestingly, it adopts a slightly different pose in the ternary complex compared to what was revealed by the cocrystal structure of the binary complex (Extended Data Fig. This suggests that contacts outside of the ligand-binding site contribute to its improved affinity in the presence of ENL and, indeed, our model revealed an extensive interfacial interaction between CRBN and ENL YEATS, with a buried solvent-accessible surface area of ~2,258 Å2 and a high degree of shape complementarity (0.64 out of 1)52. 8a,b)—is likely necessary to accommodate ENL engagement and, in turn, necessitates a rather weak primary affinity for CRBN. a, EM map of DDB1:CRBN bound to (S)-dHTC1 and ENL colored by protein and domain. b, Model of DDB1:CRBN bound to (S)-dHTC1 and ENL complex. c, Superposition of reported ligand-bound CRBN structures (gray) reveals a movement of CRBN loops (red arrows). e–h, Views of CRBN and ENL residues making intermolecular protein–protein contacts, focused on CRBN H353 (e), ENL R16 (f), ENL K72 and CRBN Y355 (g), and ENL Q41 and CRBN Q178 (h). i, Degradation of WT or mutant ENL–BFP reporter in MV4;11 cells. ENL–TagBFP and internal mCherry control were measured by flow cytometry following dHTC1 (1 µM) treatment for 24 h (BFP or mCherry, normalized to DMSO-treated cells; mean ± s.e.m. j, Same as i, but for treatment with SR-1114 (10 µM). k, ENL–TagBFP (WT or Q41A) degradation in MV4;11 cells (BFP or mCherry, normalized to DMSO control; mean ± s.e.m. l, Superposition of the apo ENL YEATS domain crystal structure (PDB 6HQ0 (ref. Interestingly, ENL binds to CRBN primarily along its N-terminal domain, differing from previously reported ternary complex structures of CRBN-based molecular glues, which mostly rely on interactions with the C-terminal domain of CRBN14,15,53. On the basis of this finding, we queried for conformational rearrangements of CRBN that might stabilize its interaction with ENL by superimposing every reported structure of CRBN bound to a molecular glue in the closed conformation (Fig. This revealed that a peripheral loop in the N-terminal domain of CRBN (amino acids 146–153) is brought closer to the ENL YEATS domain than in any other structure (median root-mean-square deviation (r.m.s.d.) = 3.8 Å) and contributes to the high degree of shape complementarity between CRBN and ENL (Fig. Protein–protein contacts are widely distributed across the CRBN–ENL binding interface (Fig. CRBN H353, which is solvent exposed in the binary complex structure (Extended Data Fig. 6f), contacts several ENL residues in the ternary complex structure, most notably making a hydrogen bond with ENL E74 where it shows connected density in the EM map (Fig. Substituting ENL E74 to alanine fully blocked ENL degradation by (S)-dHTC1 while only modestly impacting SR-1114 activity (Fig. The close ENL contacts made by CRBN H353 might explain why the more extended H353R allele was found to disrupt dHTC1 activity in the DMS screen but not SR-1114 (Extended Data Fig. ENL R16 is surrounded by several CRBN residues in the N-terminal domain (F102, F150 and I152) and stacks between CRBN F102 and F150, whose relative positions suggest formation of a cation–π interaction (Fig. Notably, F150 is located within the loop on CRBN (amino acids 146–153) that shows a substantial conformational rearrangement in the ternary complex with ENL (Fig. Consistent with this interaction being essential for CRBN–ENL contacts, we found ENL-R16A to be completely resistant to (S)-dHTC1-mediated degradation but sensitive to SR-1114 (Fig. We also observed a cation–π interaction between CRBN Y355 and ENL K72 (Fig. 3g), with the ENL-K72A mutant showing less sensitivity to (S)-dHTC1 but unchanged sensitivity to SR-1114 (Fig. Altogether, these results demonstrate that ENL–CRBN contacts outside of their ligand-binding sites are critical for dHTC1 activity, consistent with a cooperativity-dependent mechanism of CRBN engagement by dHTC1. We also observed a polar–polar interaction between CRBN Q178 and ENL Q41 (Fig. Surprisingly, we found that ENL-Q41A is more sensitive to (S)-dHTC1 but equivalently sensitive to SR-1114 and unaffected by other YEATS domain ligands or inactive analogs of dHTC1 (Fig. These data suggest that the Q41A substitution may assist ENL in making more favorable contacts with CRBN in the presence of (S)-dHTC1. Interestingly, when we compared our cryo-EM model to the isolated crystal structure of ENL (PDB 6HQ0), we observed an inward movement of the loop containing ENL Q41 (amino acids 37–42; backbone r.m.s.d. = 1.9), which is wedged into a groove formed by the CRBN N-lobe (Extended Data Fig. Compared to other observed CRBN structures, this groove is enlarged by the β-barrel loop of CRBN (amino acids 172–178) moving outward (Fig. 3c), allowing for the formation of a polar–polar interaction between CRBN Q178 and ENL Q41 (Fig. These movements likely avoid clashes between CRBN and ENL Q41 (Fig. 3l), rationalizing our observation that ENL-Q41A is more readily degraded by dHTC1. Altogether, these data demonstrate that HTC afforded the discovery of a highly cooperative molecular glue degrader of ENL. Improved physiochemical and pharmacological properties are often cited as a key motivation for the discovery of molecular glues over heterobifunctionals. Therefore, we evaluated dHTC1 alongside SR-1114 in translationally relevant models of AML that require ENL for survival26,54,55. In cell culture, we found that (S)-dHTC1 potently and stereoselectively suppressed the growth of ENL-dependent MV4;11 cells but did not impact ENL-insensitive HL-60 cells (Fig. Importantly, (R)-dHTC1 did not impact either cell line and KO of CRBN conferred resistance to (S)-dHTC1 in MV4;11 cells, demonstrating on-target antiproliferative activity (Fig. (S)-dHTC1 showed greater antiproliferative effects than ENL/AF9 YEATS domain inhibitors in wild-type (WT) MV4;11 cells but not in CRBN-deficient cells, demonstrating that ENL degradation is more effective than inhibition (Fig. However, as SR-1114 could not sustain ENL degradation beyond 24 h (Extended Data Fig. 8g), it showed weak antiproliferative activity (Fig. As we previously demonstrated that ENL abundance rebounds to normal levels 24 h after washout of SR-1114 (ref. 26), we can conclude that the less durable degradation of ENL by SR-1114 (24 h versus 5 days for dHTC1) is because of its inferior stability in cell culture—a known liability of the glutarimide scaffold in many CRBN degraders56. a, Viability of MV4;11 cells after 12-day treatments with (S)-dHTC1 or (R)-dHTC1. Cells were split 1:10 and fresh drug was added every 3 days; then, viability was measured by ATP-dependent luminescence (normalized to DMSO; n = 3 independent experiments; mean ± s.e.m. b, Mean plasma concentration of (S)-dHTC1 and SR-1114 in male C57BL/6 mice dosed with 50 mg kg−1 through intraperitoneal injection (n = 3 mice). Cellular DC50 (MV4;11 at 8 h) is depicted with and without adjustment for PPB in both plots. c, Diagram representing the timeline of dosing and data collection for in vivo studies. d, Immunoblot analysis of mouse-cell-depleted bone marrow from MV4;11 xenotransplantation following treatment with vehicle, (S)-dHTC1 (50 mg kg−1) or (R)-dHTC1 (50 mg kg−1) through intraperitoneal injection (n = 4 mice per group). e, RNA-seq analysis of mouse-cell-depleted bone marrow lysates from MV4;11 xenograft (n = 3 individual mice per treatment condition from xenograft experiment shown in c). P values were calculated using DESeq2 (two-sided Wald's test). f, Box-and-whisker plot depicting the effect of (S)-dHTC1 on the mRNA expression of ENL target genes compared to vehicle control. ENL target genes, defined by typical (gray; n = 176 genes) and asymmetric (blue; n = 56 genes) ENL ChIP-seq signal strength within the gene promoter region (typical and asymmetric target genes) or by responsiveness to ENL degradation through dTAG-enabled chemically induced degradation (green; n = 150 genes). Nontargets include all other expressed transcripts (n = 14,854 genes). g, Flow-cytometry-based quantification of hCD11b expression in indicated tissues (n = 3 mice; same mice as in d). Statistical analysis was conducted using an ordinary two-way analysis of variance with uncorrected Fisher's least significant difference multiple-comparisons test (two-sided). Next, we evaluated the pharmacokinetic behavior of dHTC1 in mice, finding that it shows a highly favorable profile for in vivo experimentation (Fig. After delivery by intraperitoneal injection, a single 50 mg kg−1 dose of (S)-dHTC1 produced a maximal plasma concentration (Cmax) of nearly 50 µM within 20 min (2.5 µM adjusted for 94.9% plasma protein binding, PPB) (Fig. This exposure of dHTC1 greatly exceeds its DC50 value in MV4;11 cells, whereas the same dose of SR-1114 produces a Cmax that only narrowly reaches its PPB-adjusted DC50 in MV4;11 cells (PPB = 93.9%). These data indicate that dHTC1 afforded a probe with more suitable properties for experimentation in vivo compared to a similarly unoptimized PROTAC. To assess pharmacodynamics, we used an MV4;11 orthotopic xenotransplantation model of AML and observed stereoselective ENL degradation after three administrations of (R)-dHTC1 or (S)-dHTC1 (50 mg kg−1) (Fig. (S)-dHTC1 stereoselectively inhibited the expression of ENL target genes (Fig. 8j), as defined by the binding of ENL to gene promoters in chromatin immunoprecipitation and sequencing (ChIP-seq) experiments or by their responsiveness to dTAG-mediated degradation54. Consistent with the prodifferentiation effects of ENL inhibitors and degraders on leukemia cells54,57,58, we also noted an increase in transcription of the myeloid differentiation marker ITGAM (encoding CD11b), which was confirmed by measuring cell surface expression of CD11b in the spleen and peripheral blood (Fig. Altogether, these data demonstrate that HTC diversification can deliver CIPs with properties that are useful for in vivo studies. To evaluate the extensibility of this approach, we sought to convert JQ1, a potent and selective BET (BRD2, BRD3 and BRD4) bromodomain ligand59, into molecular glue degraders, several of which have been reported recently23,24,25,34,60,61. JQ1 was first converted to an anilino intermediate through Buchwald–Hartwig amination assisted by a newly reported Pd(COD)(DQ) catalyst62, which was then furnished with an iminosulfur oxydifluoride SuFEx hub to yield JQ1-difluoride (9) facilitated by a silver pentafluorooxosulfate salt (Extended Data Fig. Using the same 3,163 amines that were previously used to discover dHTC1, we synthesized more than 3,000 JQ1 analogs and screened each for the ability to induce BRD4–HiBiT degradation (Fig. 5b), which were retested in triplicate and counterscreened against HiBiT–ENL, as JQ1 does not bind to ENL (Extended Data Fig. We prioritized two building blocks for follow-up studies, nipecotamide JS-2 (10) and cyclopropylmethanamine JS-3 (11), the latter of which was unintentionally included in the library twice (Fig. These were used to synthesize and purify dHTC2 (12) and dHTC3 (13), which were confirmed by high-resolution MS and nuclear magnetic resonance (NMR) to be formed by sulfuramidimidoyl fluoride and sulfamide connections, respectively (Fig. Both compounds induced selective and CRL-dependent degradation of BRD4 (Fig. Blockade of BRD4–HiBiT degradation by MLN4924 and carfilzomib was not observed at higher concentrations approaching 10 µM, which we suspect is because of the transcriptional consequences of BET bromodomain inhibition, as JQ1 itself decreases BRD4–HiBiT signal at these high concentrations (Extended Data Fig. We cannot rule out other explanations, including off-target effects at high concentrations of dHTC2 and dHTC3. Nevertheless, the full rescue of BRD4–HiBiT levels at lower concentrations of dHTC2 and dHTC3 demonstrate that their activity is CRL dependent, motivating effector discovery by forward genetics. a, Structure of JQ1 analogs synthesized by HTC. Products were formed as a 9:1 mix of enantiomers (S:R). b, Crude reaction products were screened for BRD4 degradation (2 µM, 16 h) in MV4;11 cells expressing BRD4–HiBiT. HiBiT luminescence is normalized to DMSO:PBS vehicle control (single independent experiment). Molecular weight is shown for the free amine building block. d, Quantitative TMT-based expression proteomics analysis of MV4;11 cells treated with dHTC2 and dHTC3 (1 µM, 16 h) or DMSO vehicle control (5,399 proteins with >2 peptides depicted; data were filtered using DTAselect 2.0 within IP2). P values were calculated using a two-tailed Student's t-test, (n = 3 biological replicates). e, Gene effect scores in CRISPR–Cas9-based KO screen (18,993 genes) in NALM6 cells treated with either dHTC2 (150 nM) or dHTC3 (400 nM). f, FACS-based CRISPR screens for UPS components affecting BRD4 stability in KBM7 reporter cells (KBM7 iCas9 BRD4s–TagBFP–P2A–mCherry) treated with DMSO or dHTC3 at 2 µM for 16 h (six sgRNAs targeting 1,301 UPS-associated genes). Gene-level fold change and P values were determined by one-sided MAGeCK analysis114. Members of the 26S proteasome are indicated in black. g, HEK293T BRD4–HiBiT cells with and without FBXO3 KO treated with dHTC3 for 16 h. HiBiT luminescence normalized to DMSO vehicle control (mean ± s.e.m. i, MV4;11 BRD4–HiBiT cells treated for 16 h. HiBiT luminescence was normalized to the DMSO vehicle control (mean ± s.e.m. j, RPE1 cells ectopically expressing FBXO3–GFP–cpHalo and Hpep6–BRD4s–EF1as–TagBFP constructs were pretreated with carfilzomib and the indicated compounds (20 µM) or DMSO control for 1 h and then treated with dHTC3 for the dose–response assay and with 100 nM TAMRA-CA for 3 h before flow cytometry analysis. Results are presented as the ratio of TAMRA to cpHalo and then normalized to DMSO (mean ± s.e.m. ; n = 3 biological replicates from two independent experiments). k, Degradation of BRD4 with WT and mutant bromodomains. KBM7 iCas9 cells expressing the tandem bromodomains of BRD4 (BD1–BD2) tagged with TagBFP–P2A–mCherry were treated with DMSO, 10 nM dBET6 or 2 µM dHTC3 for 16 h. The signal is BFP–mCherry, normalized to DMSO-treated cells (mean ± s.e.m. ; n = 3 biological replicates from two independent experiments). We performed two orthogonal CRISPR screens to identify the cellular effectors of BRD4 degradation. In the first screen, we leveraged the toxicity of BRD4 degradation to perform a growth-based drug resistance screen in NALM6 cells. As BET degradation elicits stronger antiproliferative effects than bromodomain inhibition46,64, we selected concentrations of each compound that are sufficient to induce BRD4 degradation but show minimal engagement of BRD4 bromodomains (Extended Data Fig. From a genome-scale sgRNA library, we primarily identified members of the UPS, with CRL regulators being shared as hits by both dHTC2 and dHTC3 (Fig. Subunits of CRL4DCAF16 (DDB1-associated and Cullin 4-associated factor 16) were identified only for dHTC2, whereas F-box protein 3 (FBXO3), a substrate receptor of the Skp1–Cullin–F-box (SCF) complex, was identified only for dHTC3 (Fig. The SCF regulators, ubiquitin-conjugating enzyme E2 L3 (UBE2L3) and Ariadne RBR E3 ubiquitin protein ligase 1 (ARIH1)65,66, were also enriched by dHTC3, albeit to a weaker extant than FBXO3. In parallel, we performed a FACS-based CRISPR screen using a library of UPS-focused sgRNAs and a dual fluorescence stability reporter for the short isoform of BRD4, BRD4(S) (KBM7 cells expressing a BRD4(S)–TagBFP–P2A–mCherry construct) (Fig. This experiment independently confirmed CRL4DCAF16 and SCFFBXO3 as being responsible for the activity of dHTC2 and dHTC3, respectively. While FBXO3 was the strongest hit for dHTC3, this screen also identified UBE2L3, ARIH1 and the SCF subunits, Cullin and Skp1. The latter two, which are essential genes, were uniquely discovered by the reporter-based screen, highlighting its ability to query genes that are required for cell survival33,34,35,36. We validated the results for dHTC2 using previously reported DCAF16-KO KBM7 cells (Extended Data Fig. To validate FBXO3 as the effector for dHTC3, we introduced FBXO3 KO in KBM7, MV4;11, NALM6 and HEK293T cells, each of which were resistant to dHTC3 (Fig. As DCAF16 is already reported to support targeted protein degradation67 and several DCAF16-dependent BRD4 degraders are already known23,25,34, we focused our subsequent efforts on dHTC3 and SCFFBXO3. We first synthesized epi-dHTC3 (14) with a stereochemical inversion known to disrupt BET bromodomain engagement59 and confirmed that it could not induce BRD4 degradation (Fig. 5h,i and Extended Data Fig. We also developed a splitHaloTag-based proximity assay for FBXO3 and BRD4 to interrogate ternary complex formation in cells68, observing a stereoselective increase in splitHaloTag signal in the presence of dHTC3 that could be suppressed by excess JQ1 (Fig. To further probe this mechanism of ternary complex formation, we tested whether dHTC3 activity could be blocked by iBET-BD1 or iBET-BD2, bromodomain inhibitors that selectively bind to BD1 or BD2 of BET proteins, respectively69. Only iBET-BD1 could block both dHTC3-induced ternary complex formation and BRD4 degradation (Fig. 10f), despite the ability of dHTC3 to engage both bromodomains, as shown by its blockade of dBET6-induced BRD4 degradation (Extended Data Fig. In agreement with these data, truncated constructs of BRD4 (ref. 34) showed that dHTC3 could only degrade BD1 fusions, whereas the PROTAC dBET6 could degrade BD1 and BD2 fusions (Extended Data Fig. Likewise, dHTC3 failed to induce degradation of a tandem bromodomain construct when it harbored an N140F substitution that blocks binding to BD1 (Fig. 5k)70, further confirming it cannot induce degradation through BD2. To determine whether dHTC3 engagement of BD2 is unable to induce ternary complex formation or, alternatively, leads to the formation of a ternary complex that is nonproductive for targeted protein degradation, we again leveraged the splitHaloTag assay. Using BRD4(S) constructs harboring an N140F or N433F substitutions, we found that dHTC3 could only mediate ternary complex formation with BD1 (Extended Data Fig. Lastly, we note that dHTC3 did not show a hook effect in BRD4 degradation or BRD4–FBXO3 ternary complex formation assays, providing a preliminary suggestion that it may engage FBXO3 cooperatively through a glue-like mechanism of action (Fig. To assess whether this scaffold can engage FBXO3 independently, we tested whether epi-dHTC3, which cannot bind to BRD4, could block dHTC3-induced degradation of BRD4 (Extended Data Fig. Its inability to block dHTC3 suggests that FBXO3 is not independently liganded by dHTC3 and is likely recruited only to the preformed BRD4(BD1)–dHTC3 complex. While we and others have shown that HTC can be used to reshape the neosubstrate scopes of E3 ligase ligands13,30,71,72,73,74,75,76,77,78,79, a generalizable solution for the target-centric discovery of molecular glues has remained elusive. Satisfied that ligand binding can, in principle, create surface alterations that cooperatively stabilize a defined protein–protein interaction, we sought a method to discover molecular glues by systematically altering a predefined protein–ligand surface at scale. This was principally inspired by the report of BCL6 and CDK12/cyclin K molecular glue degraders, which were retrospectively found to stabilize protein–protein interactions through surficial ligand modifications absent in their nonglue analogs18,19,20,21,22,80. However, as BCL6 and CDK12/cyclin K appear uniquely susceptible to molecular glue degraders (nearly half of their ligands are also degraders18,80,81), it was unclear whether these examples would be generally instructive. Previous reports of molecular glues that first bind to a target and then cooperatively engage an E3 ligase have largely relied on serendipity or target-agnostic screens18,19,20,21,22,82,83,84. HTC proved essential for converting ligands into molecular glues, as we found that relatively few modifications could endow ligands with this neomorphic activity. In contrast, previous studies have demonstrated that CIPs can be discovered from a much smaller collection of analogs by appending simple electrophilic handles directly onto a ligand of interest23,24. These covalent handles can be transplanted onto ligands of diverse targets to afford effective degraders, as they possess intrinsic reactivity for the E3 ligase23,24. A differentiating feature of this HTC-based approach is the ability, embodied by dHTC1, to prospectively discover noncovalent molecular glues that rely on interfacial protein–protein contacts that cooperatively stabilize a ternary complex. However, it is also conceptually possible to discover heterobifunctionals using HTC (dHTC2 shows a strong hook effect in degradation assays) and further study is required to determine whether a hit behaves as a molecular glue or heterobifunctional. SuFEx provided a favorable method for high-throughput ligand diversification, given (1) the wide commercial availability of SuFEx-compatible amine building blocks; (2) the facile and scalable synthetic procedures used for large library builds; and (3) the ability to perform an effector-agnostic screen by testing crude reaction products directly in native living systems. However, as the sulfamide of dHTC1 was essential for its activity, other chemistries will likely yield different hits and it is, therefore, encouraging that additional HTCs are available85,86,87,88,89. DNA-encoded library approaches have identified bifunctional compounds with glue-like cooperativity but they require preselection of both target and effector90,91. It is notable that neither the mechanism of dHTC1 nor the effector for dHTC3 would have been anticipated in advance of these screens, highlighting an advantage of effector-agnostic screening in cells. Furthermore, we envision that HTC could be adapted to cell-based screening assays for neomorphic pharmacology well beyond targeted protein degradation. Initially, it seemed improbable to screen over 3,000 analogs only to discover a compound operating through the well-established effector, CRBN, especially one with minimal independent affinity for the ligase. However, this finding may inform whether some ligases are more easily co-opted than others. Genetic screens using fusion protein constructs have evaluated this question en masse, comparing hundreds of proteins for their ability to stimulate protein degradation92,93. Like CRBN, FBXO3 scored highly in these screens, suggesting that it may be similarly well disposed to proximity pharmacology92,93. However, several effectors (for example, CRBN, DCAF11, DCAF16, FBXO22 and tripartite motif-containing protein 21 (TRIM21)) have been overrepresented in reports of small-molecule degraders23,25,34,36,60,67,94,95,96,97,98,99,100,101, despite some, like DCAF16, scoring poorly in previous genetic screens92,93. It is interesting to consider other characteristics that might produce these ‘frequent hitters', such as their intrinsic ligandability, covalent reactivity or their overrepresentation in the cellular repertoire of CRL complexes, among other possibilities102. Furthermore, while we did not observe interactions between ENL and 1 µM CRBN, we cannot rule out the possibility that they interact at concentrations higher than those we were able to assess. If so, dHTC1 may exploit a weak native interaction between ENL and CRBN to form a stable ternary complex, which is a common feature of diverse molecular glues103. One hypothesis is that some proteins may have evolved the capacity to promiscuously engage many interaction partners. CRBN recognizes proteins with C-terminal cyclic imide modifications that are formed from the cyclization of asparagine and glutamine on damaged proteins38. This mechanism of degron recognition might suggest that CRBN is poised to bind a large fraction of the proteome, provided that it is presented with a cyclic imide modification for recognition38—a hypothesis that would be supported by the conditional engagement of CRBN by dHTC1. Our structural studies indicate that CRBN makes concerted movements to accommodate the induced ENL interaction with a high degree of shape complementarity, which is congruent with emerging studies demonstrating that the CRBN surface is highly plastic and capable of accommodating diverse structural degrons42,104,105,106. On the basis of these data, we might envision efforts to decorate ligands with diverse cyclic imide moieties, regardless of their intrinsic affinity for CRBN, with the intent of discovering protein–ligand complexes that are biased for CRBN capture without necessarily being biased for CRBN affinity. As the degradation of CRBN neosubstrates can be a major concern in the development of CRBN-based degraders16,107,108,109, a lack of independent CRBN affinity is likely a benefit that minimizes the chance of eliciting off-target neosubstrate degradation, as we see for dHTC1. This selectivity and the availability of an inactive enantiomeric control position (S)-dHTC1 favorably as a chemical probe for the study of ENL function in cellular and animal model systems. Notably, such a stereoisomeric control is difficult to develop with thalidomide-based degraders, which are often prone to fast racemization in situ110. Unlike CRBN, which has a central role in the development of targeted protein degradation, FBXO3 has not been co-opted by small molecules. Several observations point toward dHTC3 functioning as a molecular glue, most notably the lack of an apparent hook effect by dHTC3 and the inability of epi-dHTC3 to block its activity, as well as its unexpected selectivity for BD1 over BD2. We hypothesize that dHTC3 must first engage BRD4 BD1 before engaging FBXO3. However, structural studies will be required to definitively assess this question and molecular glue activity would not rule out the existence of a ligandable pocket on FBXO3. Prior studies have identified FBXO3 as a regulator of inflammatory signaling111,112,113, suggesting that, in the future, it may perhaps be possible to discover FBXO3-dependent degraders that are conditionally activated by inflammatory cell states. A detailed description of chemical syntheses, spectroscopic characterization and relevant chiral characterization, as well as flow cytometry gating strategies and SPR sensorgrams, is provided in Supplementary Methods. HL-60 cells (gift from the laboratory of J. Bradner), KBM7 cells (gift from the laboratory of T. Brummelkamp) and KBM7 cells harboring a doxycycline-inducible Cas9 (iCas9; gift from the laboratory of J. Zuber) were cultured in 10% FBS in Iscove's modified Dulbecco's medium (IMDM). OCI-AML2 (gift from J. Bradner), OCI-AML3 (German Collection of Microorganisms and Cell Cultures (DSMZ), ACC 582) and Kasumi-1 (American Type Culture Collection (ATCC), CRL-2724) were cultured in 20% FBS in RPMI 1640. HB11;19 (gift from the laboratory of A. Yokoyama), Jurkat (gift of the laboratory of M. Bollong) MV4;11 and MOLM-13 (gifts from J. Bradner), NALM6 (given by ChemoGenix, University of Montreal) and EOL1 (DSMZ, ACC 386) cells were cultured in 10% FBS in RPMI 1640. RPE1 (ATCC, CRL-4000), HEK293T (ATCC, CRL-3216) and Lenti-X 293T (Takara Bio, 632180) cells were cultured in 10% FBS in DMEM. All cell lines were confirmed to be free of Mycoplasma at 1-month intervals and never tested positive. All cell lines were cultured in media using heat-inactivated FBS, supplemented with streptomycin and penicillin, and stored in a 37 °C incubator at 5% CO2. For lentiviral production, Lenti-X 293T cells were seeded in six-well plates or 10-cm dishes and transfected at approximately 80% confluency with 0.6/3.4 μg of target plasmid, 1.7/0.3 μg of psPAX2 (Addgene, 12260) and 0.85/0.15 μg of pMD2.G (Addgene, 12259) using polyethylenimine (PEI; PolySciences). Virus supernatant was collected after 60 h, filtered using a 0.45-μm filter and stored in aliquots for transduction. To assess drug response, 20 × 104 MV4;11 or HL-60 cells were seeded per well in a non-tissue-culture-treated 96-well plate (VWR) and treated with 0.1% DMSO or increasing concentrations of (R)-dHTC1 or (S)-dHTC1 (39.07 nM, 78.13 nM, 156 nM, 312 nM, 625 nM, 1.25 µM, 2.5 µM, 5 µM and 10 µM). Cells were split 1:10 and fresh drug was added every 3 days. Cell proliferation was measured on day 12 using the CellTiterGlo luminescent cell viability assay (Promega). Drug response was calculated on the basis of luminescence relative to the DMSO-treated samples. The experiment was independently repeated three times in technical replicates and, for each, the average luminescence was calculated. The following cell lines were generated bearing an endogenously tagged N-terminal HiBiT–ENL with a GSG tripeptide linker through homology-directed repair: OCI-AML2, OCI-AML3, MV4;11 (WT and CRBN-KO30), MOLM-13, EOL1 and HL-60. All components necessary were ordered from Integrated DNA Technologies (IDT). A Neon transfection system (Thermo Fisher) was used to electroporate cells. Briefly, sgRNA complexes were synthesized by mixing equal volumes of crRNA (2 µl, 160 µM in nuclease-free water, ggcgccagccatggacaatc) and tracrRNA (2 µl, 160 µM) and incubated at 37 °C for 30 min before Cas9 (2.7 µl, 10 µg µl−1, Alt-R SpCas9 nuclease V3, 1081058) and duplex buffer (1.3 µl; 30 mM HEPES pH 7.5 and 100 mM potassium acetate in nuclease-free water) were added. This suspension was incubated at 37 °C for an additional 15 min to form the active RNP complex. The RNP solution (1 µl) was added to single-stranded oligodeoxynucleotide (ssODN; 2 µM in nuclease-free water, gcggcggcggcgagcgacgcggggcccgggggcggggcggggcgccagccatggtgagcggctggcggctgttcaagaagattagcggaagcggagacaatcaagtgaggagcggccgcgccgcccctgcgcagcccgcccggccccct). The resulting mixture is electroporated with 200,000 cells in electroporation buffer and the cells are deposited into antibiotic-free medium, as described above, and allowed to rest in a 37 °C incubator for 24 h before additional antibiotic-containing medium was added. Replicates with the highest overall signal were selected and expanded. 1b) was run in MV4;11 cells bearing an N-terminal tagged ENL–HiBiT tag without a GSG linker. Subsequent assays used a redeveloped cell line bearing this tripeptide linker as we found the original HiBiT reporter signal diminished after serial passaging in cell culture. The resulting HiBiT–GSG–ENL reporter (hereafter referred to as HiBiT–ENL) showed improved durability and overall assay performance and was, therefore, used for all subsequent studies. For MV4;11 cells expressing HiBiT–ENL without a GSG linker, the following ssODN was ordered from IDT and used as received: gcggcggcggcgagcgacgcggggcccgggggcggggcggggcgccagccATGGTGAGCGGCTGGCGGCTGTTCAAGAAGATTAGCGACAccgcccggccctccccggtcccggcctccccgctccgcgcccgcccgccc. The electroporation conditions for the MV4;11, OCI-AML2, OCI-AML3, MV4;11, MOLM-13 and EOL1 cell lines were as follows: pulse voltage, 1,600 V; pulse width, 10 ms; number of pulses, 3. The electroporation conditions for the HL-60 cell line were as follows: pulse voltage, 1,350 V; pulse width, 35 ms; number of pulses, 1. All components necessary were ordered from IDT. A Neon transfection system (Thermo Fisher) was used to electroporate cells. Briefly, sgRNA complexes were synthesized by mixing equal volumes of crRNA (2 µl, 160 µM in nuclease-free water, aatcttttctgagcgcacct) and tracrRNA (2 µl, 160 µM) and incubated at 37 °C for 30 min before Cas9 (2.7 µl, 10 µg µl−1; Alt-R SpCas9 nuclease V3, 1081058) and duplex buffer (1.3 µl; 30 mM HEPES pH 7.5 and 100 mM potassium acetate in nuclease-free water) were added. This suspension was incubated at 37 °C for an additional 15 min to form the active RNP complex. The RNP solution (1 µl) was added to ssODN (2 µM in nuclease-free water, catgaatttccagagtgatctattgtcaatatttgaagaaaatcttttcggtggcggtggctcgggcggtggtgggtcgggtggcggcggatctgtgagcggctggcggctgttcaagaagattagctgacctaggtggcttctgactttgattttctggcaaaacattgactttccata). All oligonucleotides were acquired from IDT and used as received. The resulting mixture was electroporated (pulse voltage, 1,600 V; pulse width, 10 ms; number of pulses, 3) with 200,000 cells in electroporation buffer and the cells were deposited into antibiotic-free medium, as described above, and allowed to rest in a 37 °C incubator for 24 h before additional antibiotic-containing medium was added. Replicates with the highest overall signal were selected and expanded. For MV4;11 and NALM6 cells, the mixture was electroporated with the following conditions: pulse voltage, 1,600 V; pulse width, 10 ms; number of pulses, 3. For HEK293T cells, the mixture was electroporated with the following conditions: pulse voltage, 1,500 V; pulse width, 30 ms; number of pulses, 1. The procedure above was adapted to generate MV4;11 cells with a C-terminal HiBiT-tagged PARP1. The crRNA used was taagacctccctgtggtaat and the ssODN used was tctgaagtatctgctgaaactgaaattcaattttaagacctccctgtggggaagcggagTAAGCGGCTGGCGGCTGTTCAAGAAGATTAGCTAAaattgggagaggtagccgagtcacacccggtggctctggtatgaattcac. All oligonucleotides were acquired from IDT and used as received. All components necessary were ordered from IDT. sgRNA complexes were synthesized by mixing equal volumes of crRNA (2 µl, 160 µM in nuclease-free water) and tracrRNA (2 µl, 160 µM) and incubated at 37 °C for 30 min before Cas9 (2.7 µl, 10 µg µl−1; Alt-R SpCas9 nuclease V3, 1081058) and duplex buffer (1.3 µl; 30 mM HEPES pH 7.5 and 100 mM potassium acetate in nuclease-free water) were added. This suspension was incubated at 37 °C for an additional 15 min to form the active RNP complex. The RNP solution (1 µl) was electroporated with 200,000 cells in electroporation buffer and the cells were deposited into antibiotic-free medium, as described above, and allowed to rest in a 37 °C incubator for 24 h before additional antibiotic-containing medium was added. For MV4;11 and NALM6 cells, the mixture was electroporated with the following conditions: pulse voltage, 1,600 V; pulse width, 10 ms; number of pulses, 3. For HEK293T cells, the mixture was electroporated with the following conditions: pulse voltage, 1,500 V; pulse width, 30 ms; number of pulses, 1. The following FBXO3 crRNA guides were ordered from IDT and used: HiBiT reporter cells, generated as described above, were seeded into 384-well plates at 20,000 cells per well and treated with the indicated compounds and equivalent concentrations of DMSO for the time indicated in the experiment, usually 3, 24 or 72 h. Levels of HiBiT-tagged proteins were detected by luminescence using the Nano-Glo HiBiT lytic detection system (Promega, N3050). Dose–response curves were plotted in GraphPad (log(inhibitor) versus response (three parameters)) and the data shown were from three or four independent repeats (mean ± s.e.m. Intracellular ENL engagement assays were carried out as described previously116. Briefly, an Echo acoustic liquid handler (Labcyte) was used to transfer drug in a ten-point dose–response series (100-nl transfers) to empty wells on a white, tissue-culture-treated 384-well plate. To these wells were added OCI/AML2 cells stably expressing ENL (YEATS)–HiBiT at concentrations of 20,000 cells in 20 µl of medium. The plate was placed on an orbital shaker for 15 s and then placed in a 37 °C incubator for 3 h or a time otherwise indicated, after which each well was treated with 20 µl of Nano-Glo HiBiT lytic detection (Promega, N3050). After this treatment, the plate was mixed with an orbital shaker for 15 s and allowed to rest in the dark for 20 min before luminescence was measured with a Clariostar microplate reader (BMG Labtech). Dose–response curves were plotted in GraphPad (log(inhibitor) versus response (three parameters)) and the data shown are from three or four independent repeats (mean ± s.e.m. An Echo acoustic liquid handler (Labcyte) was used to transfer drug in a ten-point dose–response series (100-nl transfers) to empty wells on a white, tissue-culture-treated 384-well plate. MV4;11 cells stably expressing an endogenous BRD4 with a C-terminal HiBiT30 were added at a concentration of 20,000 cells in 18 µl of medium and the plate was mixed on an orbital shaker for 15 s. The plate was placed into a 37 °C incubator for 2 h before 2 µl of a 5 µM dBET6 solution in RPMI with 10% FBS was added to each well and the plate was again placed on an orbital shaker for 15 s (final concentration of dBET6: 500 nM); control wells were treated with an equal volume of DMSO in RPMI with 10% FBS. After 1 h of incubation at 37 °C, each well was treated with 20 µl of Nano-Glo HiBiT lytic detection (Promega, N3050). After this treatment, the plate was mixed with an orbital shaker for 15 s and allowed to rest in the dark for 20 min before luminescence was measured with a Clariostar microplate reader (BMG Labtech). Dose–response curves were plotted in GraphPad (log(inhibitor) versus response (three parameters)) and the data shown are from three or four independent repeats (mean ± s.e.m. BRD4 BD1 homogeneous time-resolved fluorescence (HTRF) assays were performed as previously described116. Briefly, assays were executed by combining recombinant protein and synthetic histone peptide in assay buffer (25 mM HEPES pH 7, 20 mM NaCl, 0.2% Pluronic F-127 and 0.05% BSA) with 1 nM LanthaScreen Eu anti-His tag antibody (Thermo Fisher, PV5597) and 8.9 nM SureLight allophycocyanin–streptavidin (PerkinElmer, CR130-100). BRD4 BD1 was used at 10 nM and the BRD4 BD1 assay was performed with 13.3 nM tetra-acetylated H4 (BioVision, 7144-01). Next, 5 μl of combined reagents were dispensed into each well of a 1536-well plate (Greiner HiBase) and the drug was added using an Echo 655 acoustic liquid handler (Beckman). Assays were incubated for 2 h before measurement of signal on a PHERAstar plate reader (BMG Labtech; simultaneous dual emission: excitation = 337 nm, emission 1 = 665 nm, emission 2 = 620 nm). HTRF signals (emission ratio at 665 nm to 337 nm) from DMSO vehicle-treated wells (maximum signal) and no-peptide-control wells (minimum signal) were used to calculate the percentage inhibition for drug-treated wells. First, 100 nM biotinylated ENL YEATS domain, 2 nM Tb (CoraFluor-1)-labeled Streptavidin and 100 nM eGFP–CRBN–DDB1ΔBPB were added in assay buffer (25 mM HEPES–NaOH pH 7.4, 150 mM NaCl, 0.5% (w/v) BSA, 0.05% (v/v) Tween-20 and 3 mM TCEP). The reaction mix was added to a 384-well microplate at a total volume of 15 µl per well. Compounds were dispensed using a D300 (Tecan) to the microplate in triplicates of 24 concentrations ranging from 10 µM to 0 µM and normalized with DMSO. Reactions were incubated at room temperature for 1 h and read using a PHERAstar plate reader (BMG Labtech). TR-FRET ratios were calculated as 520 nm/490 nm. Ratios were plotted using GraphPad Prism (version 10.3.1). An Echo acoustic liquid handler (Labcyte) was used to transfer compound in a ten-point dose–response series (100-nl transfers) to empty wells on a white, tissue-culture-treated 384-well plate. MV4;11 cells stably expressing endogenous BRD4 with a C-terminal HiBiT30 were added at a concentration of 20,000 cells in 18 µl of medium and the plate was mixed on an orbital shaker for 15 s. The plate was placed into a 37 °C incubator for 1 h before 2 µl of a 5 µM dBET6 solution in RPMI with 10% FBS was added to each well and the plate was again placed on an orbital shaker for 15 s (final concentration of dBET6: 500 nM); control wells were treated with an equal volume of DMSO in RPMI with 10% FBS. After 1 h of incubation at 37 °C, each well was treated with 20 µl of Nano-Glo HiBiT lytic detection (Promega, N3050). After this treatment, the plate was mixed with an orbital shaker for 15 s and allowed to rest in the dark for 20 min before luminescence was measured with a Clariostar microplate reader (BMG Labtech). Luminescence values were normalized to DMSO-treated wells and dose–response curves were fitted using nonlinear regression. An Echo acoustic liquid handler (Labcyte) was used to transfer compound in a ten-point dose–response series (100-nl transfers) to empty wells on a white, tissue-culture-treated 384-well plate. MV4;11 cells stably expressing endogenous PARP1 with a C-terminal HiBiT were added at a concentration of 20,000 cells in 18 µl of medium and the plate was mixed on an orbital shaker for 15 s. The plate was placed into a 37 °C incubator for 2 h before 2 µl of a 2.5 µM SK-575 (MedChem Express) solution in RPMI with 10% FBS was added to each well and the plate was again placed on an orbital shaker for 15 s (final concentration of SK-575: 250 nM); control wells were treated with an equal volume of DMSO in RPMI with 10% FBS. After 4 h of incubation at 37 °C, each well was treated with 20 µl of Nano-Glo HiBiT lytic detection (Promega, N3050). After this treatment, the plate was mixed with an orbital shaker for 15 s and allowed to rest in the dark for 20 min before luminescence was measured with a Clariostar microplate reader (BMG Labtech). Luminescence values were normalized to DMSO-treated wells and dose–response curves were fitted using nonlinear regression. An Echo acoustic liquid handler (Labcyte) was used to transfer compound in a ten-point dose–response series (100-nl transfers) to empty wells on a white, tissue-culture-treated 384-well plate. HEK293T cells stably expressing endogenous BRD4–HiBiT were added at a concentration of 20,000 cells in 18 µl of medium and the plate was mixed on an orbital shaker for 15 s. The plate was placed into a 37 °C incubator for 1 h before 2 µl of a 10 µM dHTC3 solution in RPMI with 10% FBS was added to each well and the plate was again placed on an orbital shaker for 15 s (final concentration of dHTC3: 1 µM); control wells were treated with an equal volume of DMSO in RPMI with 10% FBS. After 16 h of incubation at 37 °C, each well was treated with 20 µl of Nano-Glo HiBiT lytic detection (Promega, N3050). After this treatment, the plate was mixed with an orbital shaker for 15 s and allowed to rest in the dark for 20 min before luminescence was measured with a Clariostar microplate reader (BMG Labtech). Luminescence values were normalized to DMSO-treated wells and dose–response curves were fitted using nonlinear regression. Lentiviral production was carried out in Lenti-X 293T cells (Takara Biosciences) with cotransfection with pMD2.G and psPAX2 (Addgene, 12259 and 12260) and a lentiviral expression plasmid with PEI. Lentivirus-containing supernatants were collected 48 h and 72 h after transfection, filtered through a 0.45-µm filter and concentrated 20-fold with Lenti-X concentrator (Takara Biosciences). Cell lines were spinfected at 500g for 1 h at room temperature with 8 µg ml−1 polybrene (EMD Millipore). Lentivirus-compatible plasmids with varying CRBN expression promoters (EF-1a, hPGK and UbC) were transduced into HiBiT–ENL MOLM-13 cells. CRBN expressed plasmids by EF-1a and hPGK were selected with puromycin at 2 µg ml−1. CRBN expressed plasmid by UbC was selected with hygromycin at 250 µg ml−1. After selection, WT CRBN, EF-1a CRBN, hPGK CRBN and UbC CRBN were screened in a ten-point dose–response series against rac-dHTC1, (S)-dHTC1, (R)-dHTC1 and SR-1114 (three biological replicates repeated with two independent experiments). HiBiT–ENL was read out after 24 h and HiBiT counts were normalized to DMSO-treated cells. For adherent cell lines, cells were plated in six-well or 12-well plates at approximately 1 × 106 cells per well and allowed to adhere overnight. For suspension lines, cells were plated at 1 × 106 cells per well and used shortly thereafter. Wells were treated with the indicated compounds for the times indicated in each experiment. Cells were then washed with ice-cold PBS and lysed with 1× RIPA buffer supplemented with protease inhibitor and phosphatase inhibitor (Life Technologies). Protein concentration in cell lysates was measured using a BCA assay. Then, 25 µg of denatured total protein from each sample was separated by SDS–PAGE using 4–12% Bis–Tris gels and transferred to nitrocellulose membranes. Membranes were blocked with 5% milk in Tris-buffered saline with Tween-20 (TBST) and then incubated with primary antibodies overnight at 4 °C (all antibodies were 1:1,000 in 5% milk in TBST, while GAPDH primary antibody was diluted 1:5,000 in 5% milk in TBST). Secondary antibodies were incubated at room temperature for 90 min before washing with additional TBST and were visualized on an Odyssey CLx (LI-COR Biosciences). Alternatively, MV4;11 CRBN−/− ENL–TagBFP–P2A–mCherry cells rescued with CRBN WT or mutant were seeded in 12-well plates at 1 × 106 cells per ml and treated with DMSO, 1 μM dHTC1 or 1 μM SR-1114 for 6 h. Cells were washed twice in ice-cold PBS and lysed for 15 min on ice with RIPA buffer (50 mM Tris-HCl pH 8.0, 150 mM NaCl, 1% Triton X-100, 0.5% sodium deoxycholate and 0.1% SDS) supplemented with Benzonase (25 U per ml) and 1× Halt protease inhibitor cocktail. Following clearance by centrifugation, protein concentration of lysates was determined using a Pierce BCA protein assay (23225, Fisher Scientific) and 30 μg of lysate was prepared using 4× LDS sample buffer (Thermo Fisher) and 10% 2-mercaptoethanol and run on NuPAGE 4–15% Bis–Tris gels (Thermo Fisher). Proteins were transferred to nitrocellulose membranes, blocked for 30 min in 5% milk in TBST at room temperature before incubating with primary antibodies for 1 h at room temperature. The following primary antibodies were used: ENL (1:1,000; D9M4B, Cell Signaling Technology) CRBN (1:2,000; gift from R. Eichner and F. Bassermann) and GAPDH (1:2,000; FL-335, Santa Cruz Biotechnology). Membranes were then washed in TBST and incubated with horseradish peroxidase (HRP)-conjugated secondary antibodies for 45 min at room temperature before further washes and developing with chemiluminescence films. The secondary antibody used was HRP-conjugated AffiniPure goat anti-rabbit IgG (1:10,000, Jackson ImmunoResearch, 111-035-003). For western blots of xenografted MV4;11 cells, mouse cell depletion was performed on whole bone marrow cells using the mouse cell depletion kit and LS columns (Miltenyi Biotec). RIPA lysis was performed with purified human cells and 25 µg of protein was loaded on 10% Bis–Tris NuPAGE gels with MOPS SDS running buffer (Life Technologies). Samples were transferred on nitrocellulose iBlot 3 transfer stacks (Life Technologies) using the iBlot 3 gel transfer device (Thermo Fisher). Membranes were washed, followed by incubation with the secondary IRDye 800CW donkey anti-rabbit antibody (Thermo Fisher Scientific; 1:7,000 in 5% milk in TBST) for 1 h at room temperature, and subsequent washes were performed. Membranes were imaged on a LI-COR imager. Primary antibodies used were as follows: ENL: D9M4B (1:1,000; Cell Signaling Technologies) or ABE2596-100UG (1:1,000; Millipore) BRD4: E2A7X-13440 (1:1,000; Cell Signaling Technology) and Ab128874 (1:1,000; Abcam) Secondary antibodies used were as follows: Goat anti-mouse IgG with IRDye 680RD: 925-68070 (1:5,000; LI-COR) Goat anti-rabbit IgG with IRDye 800CW: 926-32211 (1:5,000; LI-COR) Goat anti-rabbit IgG with HRP: 111-035-003 (1:10,000; Jackson ImmunoResearch) Goat anti-mouse IgG with HRP: 115-035-003 (1:10,000; Jackson ImmunoResearch) Donkey anti-rabbit with IRDye 800CW: 926-32213 (1:7,000; LI-COR) The double-stranded sequence for CRBNmidi was amplified from a previously published plasmid (Addgene, 215330)41. The sequence was cloned into a modified expression vector backbone derived from pET21a (Novagen) with an N-terminal 6×His tag and a C-terminal AviTag. The ensuing plasmid was cotransformed in BL21(DE3) cells (New England Biolabs) with biotin ligase expression vector pBirAcm (Avidity). Cultures were grown at 37 °C in Luria–Bertani growth medium to an optical density at 600 nm (OD600) of 0.6. Protein expression was induced at 16 °C with the addition of 100 µM biotin and IPTG to a final concentration of 1 mM. The culture was harvested by centrifugation following a 16-h incubation and resuspended in lysis buffer (50 mM Tris-HCl pH 8.0, 300 mM NaCl, 10% glycerol, 10 mM imidazole and 1 mM DTT containing cOmplete EDTA-free protease inhibitor (Roche)). Cells were lysed by two passes through a MicroFluidizer at 10,000 psi. The supernatant was passed over Ni-NTA agarose (Qiagen) and the protein was eluted in elution buffer (50 mM Tris-HCl pH 8.0, 300 mM NaCl, 10% glycerol, 250 mM imidazole and 1 mM DTT). Elution fractions were buffer-exchanged into 20 mM Tris-HCl at pH 8.0 and 50 mM NaCl using PD10 desalting columns (GE Life Sciences). The protein was further purified by size-exclusion chromatography (SEC) through a HiLoad Superdex column (GE) on an AKTA Pure (GE). SPR experiments were performed on a Biacore 8K (Cytiva). A series S streptavidin sensor chip was used for ligand immobilization (Cytiva). Ligand immobilization was carried out in flow cell 2 with a series of ligand injections at a flow rate of 5 µl min−1 until the desired density was achieved (700 RU for ENL_YEATS ternary complex, 1,800 RU for ENL_YEATS small-molecule binding and 4,430 RU for CRBNmidi). Flow cell 1 was left blank as a reference. HBS-EP+ buffer with 1% DMSO was used as running buffer and for the preparation of all analyte samples. For samples with CRBN–DDB1 in the eluent, CRBN–DDB1 was serially diluted in either HBS-EP+ and 1% DMSO buffer or HBS-EP+ and 1% DMSO buffer with 1 µM compound. Analytes were serially diluted in polystyrene 96-well plates immediately before injection. Before analyte injection, three startup cycles were run with injections of running buffer over all flow cells. Binding data were collected by injecting the analyte in a single-cycle analysis at a flow rate of 30 µl min−1 over two flow cells (reference and immobilized ligand) at a temperature of 25 °C. The association to biotin–ENL_YEATS or CRBNmidi was measured over 120 s and the dissociation was measured over 1,200 s. Duplicate injections were carried out for each sample, including the blank. Affinity curves, association constants, dissociation constants and sensorgrams were determined using Biacore 8K evaluation software. Curves were plotted in GraphPad (one-site-specific binding) and the data shown are from two or three independent repeats. Probe displacement from the CRBN–DDB1 complex (performed by Reaction Biology) in the presence or absence of ENL (YEATS) was assessed through FP detection. First, a 9 µM (final 3 µM) solution of ENL (YEATS) was prepared in fresh assay buffer (50 mM HEPES pH 7.5, 100 mM NaCl, 0.5 mM DTT and 0.01% Tween-20). Then, 5 µl of ENL (YEATS) solution or buffer only (for no-ENL controls) was added to assay wells. Compounds were then added to assay wells using an acoustic liquid handler (Echo, Beckman) and incubated for 20 min at 4 °C. Next, a 150 nM (final 50 nM) solution of CRBN–DDB1, prepared in assay buffer, was added (5 µl) to each reaction well, followed by another 20-min incubation at room temperature. Assay buffer was added to blank wells. Lastly, 30 nM solution of BODIPY–thalidomide probe (final concentration: 10 nM; Tenova Pharma, T52674) was prepared in assay buffer and added (5 µl) to assay wells. After a 90-min incubation at room temperature, FP signal was measured using a PheraStar plate reader (BMG Labtech) with an FP module (excitation: 485 nm, emission: 520 nm (S/P)). The observed signal was blank-corrected and converted to percentage binding relative to DMSO control. IC50 determinations were performed using GraphPad Prism 4.0 software with a sigmoidal dose–response (variable slope) equation or log(inhibitor) versus response (three parameters) curve. Treated cells were resuspended in 200 µl of cold DPBS containing protease inhibitor (cOmplete ULTRA tablets mini EDTA-free; Roche, 05892791001; one tablet was dissolved in 10 ml of DPBS before use). Cells were lysed by probe sonication using a Branson Sonifer 250 ultrasonic cell disruptor (2 × 12 pulses, 10% power output). The protein content of whole-cell lysates was determined using the DC protein assay (Bio-Rad, 5000113 and 5000114) with absorbance at 750 nm measured on a CLARIOstar plate reader. A volume corresponding to 200 µg of protein was transferred to a new low-bind Eppendorf tube containing 48 mg of urea and brought to a total volume of 100 µl. Samples were reduced with DTT by the addition of 5 µl of a 200 mM solution (final concentration: 10 mM) and were incubated at 65 °C for 15 min. Samples were next alkylated with iodoacetamide by the addition of 5 µl of a 400 mM solution (20 mM final concentration) and incubated at 37 °C for 30 min in the dark. Samples were vortexed and centrifuged (16,000g, 10 min, 4 °C). The supernatant was aspirated, leaving behind the protein disc, and an additional 1 ml of cold CH3OH was added. The samples were vortexed again and centrifuged (16,000g, 10 min, 4 °C). The supernatant was aspirated and the protein pellet was briefly allowed to air dry. The protein pellet was then resuspended in 160 µl of EPPS buffer (200 mM, pH 8.0) by probe sonication (ten pulses, 10% power output). Proteomes were digested with LysC by the addition of 4 µl of a 0.5 µg µl−1 LysC solution in high-performance liquid chromatography (HPLC)-grade water and the samples were incubated for 2 h at 37 °C. Proteomes were then digested with trypsin by the addition of 8 µl of a 0.5 µg µl−1 trypsin solution in trypsin resuspension buffer with 20 mM CaCl2 and the samples were incubated at 37 °C overnight. Peptide concentrations were determined using a Micro BCA assay (Thermo Scientific, 23235) and a volume that corresponded to 25 µg was transferred to a new low-bind Eppendorf tube and brought to a total volume of 25 µl with EPPS buffer. To each sample, 9 µl of HPLC-grade acetonitrile was added followed by 3 µl (20 µg µl−1 in dry acetonitrile) of the respective 6plex tandem mass tag (TMT) and the reaction was vortexed and incubated at room temperature for 1 h with additional vortexing every 20 min. The reaction was quenched by the addition of 6 µl of 5% hydroxylamine in H2O, vortexed and incubated at room temperature for 15 min. Samples were combined by taking 25 µl of each sample (~12.5 µg of labeled peptide per channel) and vacuum-centrifuged. The combined samples were desalted with a Sep-Pak C18 cartridge and offline-fractionated by HPLC. Samples were resuspended in 500 µl of buffer A (95% H2O, 5% acetonitrile and 0.1% formic acid), vortexed and sonicated in a water bath for 5 min at room temperature. Before loading the sample onto a Sep-Pak C18 cartridge (Waters, WAT054955), the column was conditioned with acetonitrile (three times, 1 ml each) and equilibrated with buffer A (three times, 1 ml each). The sample was desalted by adding buffer A (three times, 1 ml each) and passing it through and the sample was then eluted by the addition of 1 ml of buffer B (80% acetonitrile, 20% H2O and 0.1% formic acid). The eluted sample was collected into a new low-bind Eppendorf tube and was evaporated to dryness using a SpeedVac vacuum concentrator. The sample was then resuspended in 500 µl of buffer A and fractionated using an Agilent HPLC (Agilent Infinity 1260 II system) into a 96-deep-well plate containing 20 µl of 20% formic acid as previously reported118,119. The eluent collected in the 96-deep-well plate was evaporated to dryness using a SpeedVac vacuum concentrator. The peptides were resuspended in 80% acetonitrile with 0.1% formic acid buffer and combined into 12 fractions (fractions were combined down a column of the plate) by the addition 200 µl of buffer, resuspension by pipetting and combination into the next row. This process was repeated with another 200 µl. Each of the fractions had a final volume of 400 µl and was evaporated to dryness using a SpeedVac vacuum concentrator. Dried fractions were solubilized in 50 ml of 0.1% trifluoroacetic acid and desalted using 2-μg-capacity ZipTips (Millipore) according to the manufacturer's instructions. Peptides were then online eluted into a Fusion Tribrid MS instrument (Thermo Scientific) from an EASY PepMap RSLC C18 column (2 μm, 100 Å, 75 μm × 50 cm; Thermo Scientific), using a hold of 6% solvent B (80:20 acetonitrile and water, 0.1% formic acid) for 15 min, followed by a gradient of 6–28% solvent B in 105 min, then 28–49% solvent B in 15 min, 40–100% solvent B in 10 min, a 10-min hold of 100% solvent B, a return to 5% solvent B in 3 min and finally a 3-min hold of 5% solvent B. The gradient was then extended for the purpose of cleaning the column by increasing solvent B to 100% in 3 min, a 100% solvent B hold for 10 min, a return to 5% solvent B in 3 min, a 5% solvent B hold for 3 min, an increase in solvent B again to 100% in 3 min, then a 100% solvent B hold for 10 min, a return to 5% solvent B in 3 min, a 5% solvent B hold for 3 min, one last increase to 100% solvent B in 3 min and a hold of 100% solvent B for 10 min. All flow rates were 250 nl min−1 delivered using an nEasy-LC1000 nanoLC system (Thermo Scientific). Solvent A consisted of water and 0.1% formic acid. Ions were created at 2.0 or 1.9 kV using an EASY Spray source (Thermo Scientific) held at 50 °C. A synchronous precursor selection MS3 method was selected on the basis of the work by Ting et al.120. Scans were conducted between 380 and 2,000 m/z at a resolution of 120,000 for MS1 in the Orbitrap mass analyzer at a standard automatic gain control (AGC) target (as defined in Thermo Scientific Xcalibur version 4.6.67.17) and a maximum injection of 50 ms. Collision-induced dissociation (CID) was then performed in the linear ion trap of peptide monoisotopic ions with a charge of 2–8 above an intensity threshold of 5 × 103, using a quadrupole isolation of 0.7 m/z and a CID energy of 35%. The ion trap AGC target was also set to standard with a maximum injection time of 50 ms. Dynamic exclusion duration was set at 60 s and ions were excluded after one iteration within the ±10-ppm mass tolerance window. The top ten MS2 ions in the ion trap between 400 and 1,200 m/z were then chosen for higher-energy C-trap dissociation at 65% energy. Detection occurred in the Orbitrap at a resolution of 50,000 and an AGC target of 1 × 105 (or 200% custom setting as defined in Thermo Scientific Xcalibur version 4.6.67.17), with an injection time of 120 ms (MS3). All scan events occurred within a 3-s specified cycle time. The analysis was performed at The Herbert Wertheim UF Scripps Institute for Biomedical Innovation and Technology, MS and Proteomics Core Facility (RRID:SCR_023576). The raw spectrum files were uploaded to the Integrated Proteomics Pipeline (IP2) (http://ip2.scripps.edu/ip2/mainMenu.html) and MS2 and MS3 files were extracted from the raw files using RAW converter and searched using the ProLuCID algorithm against a reverse-concatenated and nonredundant variant of the human UniProt database (2016-07). The precursor ion mass tolerance for a minimum envelop of three isotopic peaks was set to 50 ppm. Cysteine residues were searched with a static modification for carbamidomethylation (+57.02146 Da) and up to two differential modifications were allowed per peptide for methionine oxidation (+15.994915 Da). Lysine and N-terminal residues were also searched with a static modification corresponding to the TMT (+229.1629 Da). Peptides were required to be a minimum of 6 aa long. ProLuCID data were filtered using DTAselect 2.0 within IP2 to allow for a false-positive rate less than 1% at the spectrum level. MS3-based peptide quantification was performed with a reporter ion mass tolerance set to 20 ppm with IP2. Proteins were required to have at least two unique peptides. Peptide-spectrum matches were grouped on the basis of protein identifier and peptides were excluded if (1) the sum of the reporter ion intensities was less than 10,000; (2) the coefficient of variance was greater than 0.5, or (3) they were nonunique or nontryptic peptide sequences. Proteins were quantified by summing the reporter ion intensities across all matching peptide-spectrum matches and normalizing to the highest signal channel per protein. Plots were generated with the log2 fold change of the quotient of DMSO-treated cells with drug-treated cells and the inverse log10 of a two-tailed Student's t-test. Typical and asymmetrical targeted genes were previously determined by ChIP-seq26. dTAG-responsive genes were also previously defined by RNA sequencing (RNA-seq) following dTAG-13-mediated degradation of ENL54. These are categorized separately in Fig. 4e, whereas the combined list is used in Fig. These studies were approved by the Dana-Farber Cancer Institute's Institutional Animal Care and Use Committee (IACUC) and were carried out with compliance to all ethical regulations. Studies were conducted in 6–8-week-old female NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ mice under IACUC protocol number 16-021 at the Dana-Farber Cancer Institute. A total of 1 × 106 MV4;11 cells were intravenously injected per mouse and engraftment was confirmed in the peripheral blood by flow-cytometry-based detection of human CD45+ cells using an LSR Fortessa, BD Biosciences (human CD45–PE, clone HI30, 1:200; mouse CD45–APC–Cy7, clone 30-F11, 1:100; BioLegend). Upon confirmation of engraftment, mice were randomly assigned to treatment groups and received three doses of (R)-dHTC1, (S)-dHTC1 (50 mg kg−1 intraperitoneally every 12 h) or vehicle. The compounds were dissolved in 5% DMAC and 95% PEG-300 (40%) + NaCl (0.9%) (60%). Mice were killed 1 h after the last dosing using CO2 inhalation followed by cervical dislocation. Bone marrow cells were isolated from tibias, femurs, iliac crest and spine by crushing the bones with a mortar and pestle in PBS (Life Technologies) + 2% FBS (Life Technologies) and passing the cell suspension through a 40-µm cell strainer (Falcon). Spleens were smashed using a syringe plunger and passed through a 40-µm cell strainer (Falcon). Blood was collected by cardiac puncture and red blood cell (RBC) lysis was performed using RBC lysis buffer (BioLegend). The differentiation status of bone marrow, spleen and blood cells was assessed by flow cytometry (LSR Fortessa, BD Biosciences) using the following antibodies: human CD45–PE, clone HI30, 1:200; mouse CD45–APC–Cy7, clone 30-F11, 1:100; human CD11b-FITC, clone ICRF44, 1:100 (BioLegend). Mice were housed in the DMPK animal facility in Building A of Shanghai ChemPartner. The room environment was well controlled: ventilating at least 15 times per hour, light 12 h per day, temperature of approximately 20–24 °C and humidity of approximately 40–70%. Animals had access to certified rodent diet (SCXK [SH] 2017-0012, Shanghai Pu Lu Teng Biological Technology) ad libitum. Water was available ad libitum. The nutritional composition and levels of contaminants of the diet and impurities and contaminants of the water were monitored. The health status of the animals was evaluated in accordance with accepted animal husbandry procedures and deemed suitable for experimental use. Pharmacokinetic analyses were performed by Shanghai ChemPartner in male C57BL/6 mice (6–8 weeks old, 19–21 g; n = 27 mice) obtained from Shanghai Jihui Laboratory Animal (qualification no. Animals were housed with free access to food and water. At defined time points, blood was collected under manual restraint either by facial-vein sampling or by terminal cardiac puncture. Approximately 95 μl of blood was drawn into K2EDTA tubes, kept on ice and centrifuged promptly (2,000g, 5 min, 4 °C) to isolate plasma within 15 min. Alternatively, male C57BL/6 mice (6–8 weeks old, 19–21 g; n = 9 mice) obtained from Shanghai Jihui Laboratory Animal (qualification no. SCXK (SH) 2022-0009, 20220009020523) were administered compounds intraperitoneally (25, 50 or 75 mg kg−1, all 10 ml kg−1, injection). At defined time points, mice were anesthetized with isoflurane and approximately 26 µl of blood per time point was collected into K2EDTA tubes through facial or cardiac puncture for terminal bleeding. Tubes were kept on ice and centrifuged promptly (2,000g, 5 min, 4 °C) to isolate plasma within 15 min. Plasma samples were processed by protein precipitation. For undiluted plasma, 20 μl of sample was used. Alternatively, samples were prepared as tenfold dilutions (2 µl of sample diluted into 18 µl of blank plasma) or 20-fold dilutions (2 µl of sample diluted into 38 µl of blank plasma, shaken for 5 min, with 20 µl aliquoted for analysis). These samples were mixed with 200 μl of methanol containing 40 ng ml−1 diclofenac or propranolol as the internal standard (IS). Then, 90 µl of supernatant was transferred to a fresh plate and 3 μl was injected for LC–MS/MS analysis. Concentrations were calculated using a calibration curve using 0.2–1,000 ng ml−1 of the analyte in male C57 plasma. PPB assays were performed by Shanghai ChemPartner. Plasma protein binding was determined using a 96-well equilibrium dialysis format with a sodium phosphate buffer system (0.05 M, pH 7.4). The buffer was prepared by combining monobasic and dibasic sodium phosphate solutions with sodium chloride, followed by dilution and pH adjustment to 7.4. Stock solutions of test and reference compounds were diluted to generate two working solutions: Solution A (0.5 mM): 10 µl of a 10 mM DMSO stock diluted with 190 µl of DMSO Solution B (0.02 mM): 8 µl of solution A added to 192 µl of phosphate buffer (final DMSO 4%) Working solutions were spiked into plasma by adding 20 µl of solution B to 380 µl of blank plasma in each well of a 96-well plate (final compound concentration: 1 µM, 0.2% DMSO). Dialysis chambers were preloaded with 100 µl of blank buffer on the receiver side. An equal volume of spiked plasma was added to the donor side. Chambers were sealed with a plastic lid and incubated on a shaker (60 rpm) for 5 h at 37 °C. For T0 samples, immediately before incubation, 25-µl aliquots of spiked plasma were mixed 1:1 (v/v) with buffer and quenched with 200 µl of acetonitrile containing IS (warfarin or quinidine). For postdialysis samples (5 h), aliquots (25 µl) from both plasma (donor) and buffer (receiver) chambers were transferred to new plates, mixed 1:1 with the opposite matrix and quenched with 200 µl of acetonitrile containing IS. All samples were shaken at 600 rpm for 10 min and centrifuged at 5,594g for 15 min. Supernatants (50 µL) were transferred to new plates containing 50 µl of purified water and stored at −20 °C until LC–MS/MS quantification. The fraction unbound was determined from the ratio of compound concentration in the buffer chamber to that in the plasma chamber at equilibrium, corrected for recovery using T0 samples. The design and construction of the human UPS-focused sgRNA library used for ENL stability screens and the human CRBN DMS library were described previously36,49. The fluorescent protein stability reporter of GSPT1 was generated by inserting the GSPT1 coding sequence into pArtichoke plasmid (Addgene, 73320) by Gibson assembly. All the BRD4-based stability reporters were generated previously34. For splitHalo assays, two separate Gateway-compatible pRRL vectors were generated using cpHalo∆ sequence from Addgene 205703 (pRRL-SFFV-gw-EGFP-(GSG)9-cpHalo∆, where ‘gw' is the Gateway cassette) and the published Hpep6 sequence68 (pRRL-SFFV-Hpep6-GGGS-gw-EF1as-TagBFP), into which FBXO3 and BRD4 short (BRD4s) coding sequences were inserted by Gateway cloning. KBM7 iCas9 cells were a gift from J. Zuber. Lenti-X 293T lentiviral packaging cells (Clontech), HEK293T cells and RPE1 cells were cultured in DMEM (Gibco) supplemented with 10% fetal calf serum (FCS; Thermo Fisher) and 1% penicillin–streptomycin (Thermo Fisher). MV4;11 cells were cultured in RPMI supplemented with 1% FCS and 1% penicillin–streptomycin. KBM7 cells were cultured in IMDM supplemented with 1% FCS and 1% penicillin–streptomycin. All cells were grown in a humidified incubator at 37 °C and 5% CO2 and routinely tested for Mycoplasma contamination. For lentiviral production, Lenti-X 293T cells were seeded in six-well plates or 10-cm dishes and transfected at approximately 80% confluency with 0.6/3.4 μg of target plasmid, 1.7/0.3 μg of psPAX2 (Addgene, 12260) and 0.85/0.15 μg of pMD2.G (Addgene, 12259) using PEI (PolySciences). Virus supernatant was collected after 60 h, filtered using a 0.45-μm filter and stored in aliquots for transduction. The UPS-focused library packaged lentivirus-containing supernatant was used to transduce KBM7 ENL–TagBFP–P2A–mCherry or BRD4s–TagBFP–P2A–mCherry cells harboring iCas9 at a multiplicity of infection (MOI) of 0.19 and 1,000-fold library representation for ENL and an MOI of 0.03 and 300-fold library representation for BRD4s using 8 μg ml−1 polybrene (Szabo Scandic, SACSC-134220). Library-transduced KBM7 cells were selected with 1 mg ml−1 G418 (Gibco) for 14 days and expanded; Cas9 expression was induced with doxycycline at 0.4 μg ml−1 (PanReac AppliChem). Then, 3 days after doxycycline induction, 50 million cells per condition were treated with DMSO, rac-dHTC1 (10 μM) or SR-1114 (10 μM) for 8 h in two biological replicates for ENL or with DMSO, dHTC2 (2 μM) or dHTC3 (2 μM) for 16 h in two biological replicates for BRD4s. Cells were washed with PBS, stained with APC anti-mouse Thy1.1 antibody (1:400; 202526, BioLegend) and Zombie near-infrared (NIR) fixable viability dye (1:1,000; BioLegend) in the presence of Human TruStain FcX Fc receptor blocking solution (1:400; 422302, BioLegend) for 10 min at 4 °C in the dark and fixed with 1 ml of BD CytoFix fixation buffer (BD Biosciences, 554655) for 30 min at 4 °C in the dark. Cells were washed with and stored in FACS buffer (PBS with 5% FCS and 1 mM EDTA) at 4 °C overnight. The next day, cells were strained through a 35-μm nylon mesh and sorted on a BD FACSAria Fusion (BD Biosciences) operated on BD FACSDiva software (version 8.0.2) using a 70-μm nozzle excluding aggregate, dead (Zombie NIR-positive), Cas9-negative (GFP-negative) and sgRNA library-negative (Thy1.1–APC-negative) cells. Cells corresponding to at least 750-fold library representation were sorted per each sample replicate. DNA libraries of sorted fractions for next-generation sequencing (NGS) were prepared as described previously121. In short, genomic DNA was extracted by cell lysis (10 mM Tris-HCl, 10 mM EDTA, 150 mM NaCl and 0.1% SDS), proteinase K treatment (New England Biolabs) and DNAse-free RNAse digest (Thermo Fisher), followed by two rounds of phenol extraction and isopropanol precipitation. Barcoded NGS libraries for each sorted fraction were prepared using a two-step PCR protocol using AmpliTaq Gold polymerase (Invitrogen). Barcodes for each sample were introduced in a first PCR, the product of which were purified using Mag-Bind TotalPure NGS beads (Omega Bio-tek) and amplified in a second PCR introducing the standard Illumina adaptors. The final Illumina libraries were bead-purified, pooled and sequenced on NovaSeq 6000 instrument (Illumina) following a 100-bp, paired-end recipe. Screen analysis was performed as previously described121. The crispr-process-nf Nextflow workflow is available from GitHub (https://github.com/ZuberLab/crispr-process-nf/tree/566f6d46bbcc2a3f49f51bbc96b9820f408ec4a3). For statistical analysis, we used the crispr-mageck-nf Nextflow workflow, also available from GitHub (https://github.com/ZuberLab/crispr-mageck-nf/tree/c75a90f670698bfa78bfd8be786d6e5d6d4fc455). To calculate gene-level enrichment, the sorted populations (ENL/BRD4shigh or ENL/BRD4slow) were compared to the ENL/BRD4smid populations in MAGeCK (version 0.5.9)114 using median-normalized read counts. A total of 18 million MV4;11 CRBN−/− ENL–TagBFP–P2A–mCherry cells were transduced with the CRBN DMS library virus supernatant at an MOI of 0.17, yielding a calculated library representation of 1,760 cells per variant. For the transduction, 1 million cells were seeded in a 24-well plate with 8 μg ml−1 polybrene (Szabo Scandic, SACSC-134220) and filled to 1 ml with the titrated amount of virus and culture medium. The plate was centrifuged at 760g for 45 min at 37 °C and then cells were pooled and expanded. Next, 3 days after transduction, library-transduced cells were selected with 1 μg ml−1 Blasticidin (Gibco, R21001) for 7 days and selected cells were expanded. A total of 50 million cells per condition were treated with rac-dHTC1 (1 μM) or SR-1114 (1 μM) for 8 h in three biological replicates. Cells were washed with PBS, stained with Zombie NIR fixable viability dye (1:1,000; BioLegend) for 10 min at 4 °C in the dark and fixed with 1 ml of BD CytoFix fixation buffer (BD Biosciences, 554655) for 30 min at 4 °C in the dark. Cells were washed with and stored in FACS buffer (PBS with 5% FCS and 1 mM EDTA) at 4 °C overnight. Two sets of 50 million unsorted cells were also harvested as a control, directly washed and frozen. The next day, cells were strained through a 35-μm nylon mesh and sorted on a BD FACSAria Fusion (BD Biosciences) operated on BD FACSDiva software (version 8.0.2) using a 70-μm nozzle excluding aggregate, dead (Zombie NIR-positive) and reporter-negative (BFP-negative and mCherry-negative) cells. The remaining cells were sorted on the basis of their ENL−1BFP and mCherry levels into an ENLhi fraction (30−135% of cells). Cells corresponding to at least 3,000-fold library representation were sorted per each sample replicate. DNA libraries of sorted and unsorted cell pools for NGS were prepared as described previously49,121. In short, genomic DNA was extracted by cell lysis (10 mM Tris-HCl, 10 mM EDTA, 150 mM NaCl and 0.1 % SDS), proteinase K treatment (New England Biolabs) and DNAse-free RNAse digest (Thermo Fisher), followed by two rounds of phenol extraction and isopropanol precipitation. CRBN variant complementary DNA was amplified by PCR from genomic DNA using AmpliTaq Gold polymerase (Invitrogen) and primers CRBN_GA_fwd (AGGTGTCGTGACGTACGGGATCCCAGGACCATGGCCGGCGAAGGAG) and CRBN_GA_rev (GGGGGGGGGGCGGAATTAATTCCTACTACTTACAAGCAAAGTATTACTTTGTCTGGAC). The cycle number for specific amplification of the 1.4-kb CRBN target was confirmed by agarose gel electrophoresis. PCR reactions for each sample were pooled and purified using Mag-Bind TotalPure NGS beads (Omega Bio-tek). The library preparation from the amplified DNA was performed using the Tagment DNA TDE1 enzyme and IDT for Illumina unique dual indices (Illumina). Library concentrations were quantified with the Qubit 2.0 fluorometric quantitation system (Life Technologies) and the size distribution was assessed using the 2100 Bioanalyzer instrument (Agilent). For sequencing, samples were diluted and pooled into NGS libraries and sequenced on NovaSeq 6000 instrument (Illumina) following a 100-bp, paired-end recipe. Raw sequencing reads were converted to fastq format with SAMtools (version 1.15.1) and BEDTools (version 2.30.0) using the bamtofastq function. Sequencing reads were trimmed using Trim Galore (version 0.6.6) using nextera and pair modes. Short reads were aligned to the CRBN cassette and SAM files were generated using mem algorithm from the bwa software package (version 0.7.17). The SAM file was converted to BAM using SAMtools and mutation calling was performed using the AnalyzeSaturationMutagenesis tool from GATK (version 4.1.8.1). Given our sequencing strategy, >98% of reads corresponded to WT sequences and were filtered out during this step. Next, relative frequencies of variants were calculated for each position and variants that were covered by <1 in 30,000 reads were excluded from further analysis. Read counts for each variant were then normalized to total read count of each sample and log2 fold changes of ENLhi over unsorted pools were calculated. To correct for differential drug potency, each variant was then normalized to the maximum log2 fold change. For drug comparisons, log2 fold changes of ENLhi over unsorted pools of each drug were subtracted. Heat maps were generated using the pheatmap (version 1.0.12) package in R (version 4.1.0) or with GraphPad Prism. MV4;11 CRBN−/− ENL–TagBFP–P2A–mCherry or GSPT1–GFP–P2A–mCherry cells were transduced with lentivirus expressing WT or mutant CRBN in pRRL-EF1a-CRBN-IRES-BlastR plasmid to generate stable cell lines. For evaluation of reporter degradation on the different CRBN backgrounds, cells were treated with DMSO or rac-dHTC1, SR-1114 (both 1 μM, 6 h, ENL reporter) or CC-885 (1 μM, 5 h, GSPT1 reporter) before flow cytometry analysis on an LSR Fortessa (BD Biosciences) operated on BD FACSDiva software (version 9.0). KBM7 iCas9 expressing the reporter SFFV–BRD4(s)–mTagBFP–P2A–mCherry or its different truncations were generated previously34. To quantify the influence of genetic perturbation on compound-induced BRD4(s) reporter degradation, the stable cell line was transduced with lentiviral sgRNA (pLentiV2-U6-sgRNA-IT-PGK-iRFP720-P2A-Neo) to 30–50% transduction efficiency. Cas9 expression was induced by treating with doxycycline (0.4 μg ml−1) for 3 days, followed by a 16-h treatment with DMSO, dBET6 (10 nM), IBG3 (0.1 nM), dHTC2 (2 μM) or dHTC3 (2 μM). Cells were stained for sgRNA expression with an APC-conjugated anti-mouse Thy1.1 antibody (202526, BioLegend; 1:400) and human TruStain FcX receptor blocking solution (422302, BioLegend; 1:1,000) for 5 min at 4 °C in FACS buffer (PBS with 5% FBS and 1 mM EDTA). Cells were washed and resuspended in FACS buffer and analyzed on an LSR Fortessa (BD Biosciences) operated on BD FACSDiva software (version 9.0). Flow cytometry data analysis was performed in FlowJo version 10.10.0. mCherry and BFP median fluorescence intensity values were normalized by background subtraction of the respective values from reporter-negative KBM7 cells. BRD4 abundance was calculated as the ration of background-subtracted BFP to mCherry median fluorescence intensity and is displayed normalized to DMSO-treated cells. RPE1 cells stably expressing SFFV–FBXO3–EGFP–cpHalo and SFFV–Hpep6–BRD4s–EF1as–TagBFP were seeded in 96-well plates. After attachment, cells were pretreated with carfilzomib (1 μM) and, when indicated, with DMSO, JQ1, GSK778 (iBET-BD1) or GSK046 (iBET-BD2) (all at 20 μM) for 1 h and then treated with DMSO, dBET6, epi-dHTC3 or dHTC3 at the concentrations indicated for each experiment in combination with 100 nM TAMRA-CA for 3 h. After treatment, cells were washed, trypsinized and resuspended in FACS buffer (PBS with 5% FBS and 1 mM EDTA) and analyzed on an LSR Fortessa (BD Biosciences) with plate reader operated on BD FACSDiva software (version 9.0). Flow cytometry data analysis was performed in FlowJo version 10.10.0. PE and GFP median fluorescence intensity values from GFP+BFP+ cells were normalized by background subtraction of the respective values from reporter-negative cells. SplitHalo signal was calculated as the ratio of background-subtracted PE to GFP median fluorescence intensity and is displayed normalized to DMSO-treated cells. The Genome-wide pooled CRISPR/Cas9 KO screen was performed by the ChemoGenix platform (IRIC, Université de Montréal; https://chemogenix.iric.ca/) as previously described122. Briefly, a NALM6 clone bearing an integrated iCas9 expression cassette generated by lentiviruses made from pCW-Cas9 (Addgene, 50661) was transduced with the genome-wide KO EKO sgRNA library (278,754 different sgRNAs)122. After thawing the library from liquid N2 and letting it recover in 10% FBS RPMI for 1 day, KOs were induced for 7 days of culture with 2 µg ml−1 doxycycline. The pooled library was then split in different T-75 flasks (28 × 106 cells per flask; a representation of 100 cells per sgRNA) in 70 ml at 4 × 105 cells per ml. Cells were treated with dHTC2 (150 nM) or dHTC3 (400 nM), using 1,000× DMSO stock solution for 8 days with monitoring of growth every 2 days, diluting back to 4 × 105 cells per ml and adding more compound to maintain same final concentration whenever cells reached 8 × 105 cells per ml. Over that period, treated cells had 4.64 (dHTC2) or 1.39 (dHTC3) population doublings whereas DMSO-only treated negative controls had a 7.5 population doubling. Cells were collected, genomic DNA was extracted using the Gentra Puregene kit according to the manufacturer's instructions (Qiagen) and sgRNA sequences were PCR-amplified as described122. sgRNA frequencies were obtained by NGS (Illumina NextSeq500). Reads were aligned using Bowtie 2.2.5 in the forward direction only (--norc option) with otherwise default parameters and total read counts per sgRNA were tabulated. Context-dependent chemogenomic interaction scores were calculated using a modified version of the RANKS algorithm122, which uses guides targeting similarly essential genes as controls to distinguish condition-specific chemogenomic interactions from nonspecific fitness and essentiality phenotypes. Raw read counts are available upon request from the ChemoGenix platform. Structural coordinates were deposited to the PDB and are available under accession number 9GY3. His6–CRBNmidi protein (Addgene, 215330) was expressed and purified as previously described41. CRBNmidi at 3.4 mg ml−1 in a buffer containing 20 mM HEPES pH 7.5, 500 mM NaCl and 0.5 mM TCEP was mixed with a 4 M excess of dHTC1 (364 µM, 1.8% (v/v) DMSO final). The complex was subjected to cocrystallization using the sitting-drop vapor diffusion method across several sparse matrix screens at 20 °C by mixing equal volumes of protein solution and reservoir solution. Crystal hits from Morpheus H10 (0.1 M buffer system 3, pH 8.5, 0.12 M alcohol and 30% precipitant mix 2) (Molecular Dimensions) were isolated directly from the drop and flash-cooled in liquid nitrogen. Diffraction data were collected at Diamond Light Source beamline I24 using the Eiger CdTe 9M detector at a wavelength of 0.6199 Å. The 2.2-Å dataset was processed using Dials123, indexed in the P31 space group and the structure was solved by molecular replacement in phenix.phaser124 using the coordinates of PDB 8RQA as a search model. The data discussed in this publication were deposited to the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO)127 under accession number GSE278582. Total RNA was extracted from mouse-cell-depleted bone marrow cells from MV4;11 xenografts using the RNeasy Kit (Qiagen). RNA samples were run on a TapeStation (Agilent) to assure an RNA integrity score > 8 and quantified by Qbit (Thermo Fisher). The NEBNext poly(A) mRNA magnetic isolation module and NEBNext Ultra II RNA library prep kit (NEB BioLabs) were used to prepare RNA libraries. Further analysis was performed on expressed genes (rlog expression < 3). N-terminal His6-tagged hsDDB1ΔBPB (UniProt Q16531, amino acids 396–705 replaced with a GNGNSG linker) and FLAG-tagged hsCRBN (UniProt Q96SW2) were cloned into pAC8-derived vectors128. The N-terminal GST-tagged eGFP–hsCRBN was cloned into a pLIB vector, which was a gift from J.-M. Peters (Addgene, 80610)129. To generate baculovirus from the pAC8-derived constructs, 1 μg of plasmids were cotransfected with linearized baculoviral DNA in Spodoptera frugiperda Sf9 cells grown in ESF 921 medium (Expression Systems) at a density of 1.0 × 106 cells per ml. To generate baculovirus from the pLIB construct, 0.5 µg of the pLIB construct was transformed in DH10EMBacY cells130 and recombinant bacmid was isolated using the Invitrogen PureLink quick plasmid miniprep kit (Invitrogen, K210010). Sf9 cells were transfected at a density of of 1.0 × 106 cells per ml and viral titer was increased by three rounds of amplification. For protein expression, Trichoplusia ni High Five cells, grown in SF-4 Baculo Express ICM (BioConcept) to a density of 2 × 106 cells per ml, were coinfected with 1.5% (v/v) each of DDB1ΔBPB -expressing and CRBN-expressing baculoviruses. After 48 h of incubation, cells were collected (20 min, 3,300g). For His6–hsDDB1ΔBPB/FLAG–hsCRBN-infected cells, the pellet was resuspended and sonicated in FLAG lysis buffer (50 mM HEPES–NaOH pH 7.4, 200 mM NaCl, 0.1% (v/v) Triton X-100 and 1 mM TCEP) supplemented with protease inhibitors. The cell lysate was treated with Benzonase for 10 min and cleared by ultracentrifugation (60 min, 120,000g), the supernatant was incubated with FLAG antibody-coated beads (Genscript, L00432) and washed with wash buffer (50 mM HEPES–NaOH pH 8.0 and 200 mM NaCl). Bound proteins were eluted with 0.15 mg ml−1 1× FLAG (DYKDDDK) peptide and diluted to 50 mM NaCl with anion-exchange chromatography (IEX) buffer A (50 mM HEPES–NaOH pH 7.4 and 1 mM TCEP). Samples were added to Poros 50 HQ resin (Thermo Scientific) and eluted with a linear salt gradient from 50 mM to 1,000 mM. Peak fractions from IEX were concentrated using centrifugal concentrators (Amicon, 30-kDa molecular weight cutoff (MWCO)) and polished by SEC (HiLoad 16/600 Supderdex 200 pg, Cytiva) in CRBN SEC buffer (50 mM HEPES–NaOH pH 7.4, 150 mM NaCl and 1 mM TCEP). Peak fractions were concentrated by centrifugation, flash-frozen in liquid nitrogen and stored at −80 °C. For His6–hsDDB1ΔBPB/GST–eGFP–hsCRBN cells, the pellet was resuspended and sonicated in GST lysis buffer (50 mM Tris pH 8.0, 200 mM NaCl and 2 mM TCEP) supplemented with protease inhibitors. The cell lysate was cleared by ultracentrifugation and treated with Benzonase for 10 min, incubated with glutathione Sepharose 4B (Cytiva, GE17-0756-01) and washed with GST lysis buffer. Elution fractions including the target protein were pooled and concentrated using 30-kDa centrifugal concentrators and cleaved with tobacco etch virus protease at 4 °C overnight. The C-terminal His6-tagged ENL YEATS domain (amino acids 1–148; UniProt Q03111) and C-terminal Strep-Avi-tagged ENL YEATS domain were cloned into a pNIC-Bio2 (SGC Oxford) vector and expressed in a LOBSTR Escherichia coli expression strain (Kerafast, EC1002). After transformation, 2-L cultures in Terrific Broth medium were grown at 37 °C to an OD600 of ~0.6 and induced with 0.35 mM IPTG. Temperature was decreased to 18 °C, proteins were expressed overnight and cultures were harvested by centrifugation (20 min, 3,300g). After clearance by ultracentrifugation (60 min, 120,000g), the supernatant was treated with Benzonase for 10 min and incubated with high-affinity Ni-charged resin (Genscript, L00223) or Strep-Tactin XT 4Flow high-capacity resin (IBA Life Sciences, 2-5030-002). His6-tagged protein elution fractions were concentrated using a centrifugal concentrator (10-kDa MWCO) and polished by SEC (Superdex 75 Increase 10/300 GL, Cytiva) in ENL SEC buffer (30 mM HEPES–NaOH pH 7.4, 300 mM NaCl and 3 mM TCEP). Peak fractions were concentrated by centrifugation, flash-frozen in liquid nitrogen and stored at −80 °C. Strep-tagged protein elution fractions were concentrated using a centrifugal concentrator (10-kDa MWCO) and biotinylated overnight in 4 °C with BirA biotin ligase. Biotinylated ENL YEATS domain was further polished by SEC using ENL SEC buffer. Grids (Quantifoil UltrAuFoil R 0.6/1) were glow-discharged (20 mA, 120 s, 39 Pa) and vitrified in a Leica EM-GP operated at 10 °C and 90% relative humidity. Grids were preincubated with 4 µl of 10 µM CRBN agnostic IKZF1-Q146A;G151N (amino acids 140–196) for 1 min and then blotted from behind for 4 s with NCI Micro Whatman no. Immediately, 4 µl of the sample were applied to the grids before blotting for 4 s and plunging into liquid ethane, kept at −181 °C. Datasets were collected using Smart EPU (version 3.7) on a Thermo Scientific Titan Krios equipped with a Thermo Fisher Scientific Selectris energy filter (10-eV slit width) and Thermo Fisher Scientific Falcon 4 direct electron detector. Videos (49 frames per video, 2.87-s exposure time) were acquired at 300 kV at a nominal magnification of ×165,000 in counting mode with a pixel size of 0.736 Å per pixel. One video was recorded per hole with aberration-free image shift (as implemented in EPU). A total of 9,856 videos for dataset 1 and 13,027 videos for dataset 2 were recorded with defocus varying from −0.8 to −2.0 µm and a total exposure rate of 50.22 e− per A2. cryoSPARC (version 4.5.3)131 was used for all processing steps. All resolutions given are based on the Fourier shell correlation (FSC) = 0.143 threshold criterion132. For dataset 1, 9,856 videos were corrected for beam-induced motion and contrast transfer function (CTF) was estimated on the fly in cryoSPARC live. Low-quality videos were rejected and 1,267,417 particles were extracted (1.31 Å per pixel) from the 6,558 remaining micrographs after TOPAZ (version 0.2.5a)133 particle picking. The particles were cleaned up in three rounds of heterogeneous refinement. Preferred orientation became apparent at this step. Thus, the remaining 994,856 particles were further classified using heterogeneous refinement, which revealed a small number of classes (95,520 particles) representing a more balanced range of views. The selected 95,520 particles were reextracted at 0.92 Å per pixel. For dataset 2, 13,027 videos were corrected from beam-induced motion and CTF was estimated on the fly in cryoSPARC live. Low-quality videos were rejected and the remaining 11,541 micrographs were split into two groups, resulting in 5,173,911 particles extracted (1.31 Å per pixel) after TOPAZ (version 0.2.5a)133 particle picking. The particles were cleaned up in two rounds of heterogeneous refinement. Similarly to dataset 1, the resulting 1,536,327 particles were further classified using heterogeneous refinement, which again revealed a small number of classes (89,792 particles) with more balanced views. The selected 89,792 particles were reextracted at 0.92 Å per pixel and combined with the particles from dataset 1. The pooled particles were used in a final homogeneous refinement, yielding a consensus reconstruction at global resolution of 2.6 Å. Local refinement with a soft mask encompassing CRBN and ENL YEATS yielded a final reconstruction at 2.9 Å (EMD-47174), which was followed by postprocessing using deepEMhancer to improve the density for ENL YEATS were used for model building. CRBN (PDB 8TNQ) and ENL YEATS (PDB 6HPY) were fit into the density in UCSF ChimeraX (version 1.6.1)134,135 and relaxed using ISOLDE (version 1.6.0)136 and Rosetta (version 3.13)137. The relaxed model was manually adjusted in Coot (version 0.9.8.0)126 and prepared for refinement with phenix.ready_set (version 1.19.2-4158)138, followed by refinement using reference restraints in phenix.real_space_refine (version 1.19.2-4158)139 against the unsharpened maps. For figures, DDB1ΔBPB (from PDB 8TNQ) was placed on the basis of a superposition of CRBN. The final model (9DUR) and map (EMD-47174) were deposited to the PDB and EM Data Bank, respectively. Data collection parameters and refinement statistics are available in Supplementary Table 9. To compare the model with other reported CRBN structures, every reported closed conformation CRBN structure in complex with a molecular glue degrader or a PROTAC was superimposed (PDB 4TZ4, 5FQD, 5HXB, 5V3O, 6BN7, 6BN8, 6BN9, 6BOY, 6H0G, 6UML, 6XK9, 7LPS, 8D7U, 8D7V, 8D7W, 8D7Z, 8D80, 8D81, 8DEY, 8G66, 8OIZ, 8OJH, 8TNP, 8TNQ, 8TNR, 8TZX, 8U15, 8U16 and 8U17) with the C-terminal domain (amino acids 318–442) of the CRBN in the model13,14,15,17,53,74,78,140,141,142,143,144,145,146. For comparison with a non-ligand-bound ENL structure, we superimposed PDB 6HQ0 with the ENL YEATS domain in the model115. values were calculated in UCSF ChimeraX (version 1.9)134,135. Structural biology applications used in this project were compiled and configured by SBGrid147. Interface comparison was made with Rosetta (version 3.13) by reordering the jump number of fold tree such that the CRBN and ENL YEATS interface was compared and then the following command was used to obtain buried interface solvent-accessible surface area and shape complementarity score52: sasa_and_sc_calc_scripts.sh. A total of 1 × 107 cells per immunoprecipitation were collected and lysed in lysis buffer (50 mM Tris pH 8, 200 mM NaCl, 2 mM TCEP, 0.1% NP-40, 10 U of turbonuclease per 200 µl of buffer and 1× cOmplete protease inhibitor tablet per 5 ml of buffer) and sonicated on ice for five rounds of 2 s followed by 10-s pauses at 25% amplitude. After centrifugation clarification, the following steps were performed on an opentrons OT2 liquid handler. Cell lysate, 10 µg of FLAG–CRBN–DDB1ΔB, 1 µM of MLN4924 and CSN5i-3 (neddylation trap)148 and 1 µM of selected degraders or DMSO vehicle control were transferred to PCR plate and incubated for 1 h at 4 °C. Then, 3 µl of prewashed and resuspended anti-FLAG magnetic bead 25% slurry (Pierce) were added to the lysates, followed by a second incubation for 1 h at 4 °C. Beads were washed three times with wash buffer (50 mM Tris pH 8, 2 mM TCEP, 0.1% NP-40 and 1× cOmplete protease inhibitor tablet per 5 ml of buffer) containing the required compounds, followed by three additional nondetergent (50 mM Tris pH 8, 2 mM TCEP and 1× cOmplete protease inhibitor tablet per 5 ml of buffer) washes containing the required compounds. After the final wash step, samples were eluted using 0.1 M glycine-HCl pH 2.7. Tris (1 M, pH 8.5) was added to elution to reach a pH of 8. Samples were then reduced with 10 mM TCEP for 30 min at room temperature, followed by alkylation with 15 mM iodoacetamide for 45 min at room temperature in the dark. Alkylation was quenched by the addition of 10 mM DTT. The resuspended protein samples were digested with 2 μg of LysC and 1 μg of trypsin overnight at 37 °C. Sample digests were acidified with formic acid to a pH of 2–3 before desalting using C18 solid-phase extraction plates (SOLA, Thermo Fisher Scientific). Desalted peptides were dried in a vacuum-centrifuged and reconstituted in 0.1% formic acid for LC–MS analysis. Data were collected using a TimsTOF Ultra2 (Bruker Daltonics) coupled to a nanoElute2 LC pump (Bruker Daltonics) through a CaptiveSpray nanoelectrospray source. Peptides were separated on a reversed-phase C18 column (25 cm × 75 μm inner diameter, 1.6 μm; IonOpticks) containing an integrated CaptiveSpray emitter. Peptides were separated using a 30-min gradient of 2–30% buffer B (acetonitrile in 0.1% formic acid) with a flow rate of 250 nl min−1 and column temperature maintained at 50 °C. The thermal ionization MS elution voltages were calibrated linearly with three points (Agilent ESI-L tuning mix ions; 622, 922 and 1,222 m/z) to determine the reduced ion mobility coefficients (1/K0). To perform data-independent acquisition parallel accumulation–serial fragmentation (diaPASEF), we used py_diAID149, a Python package, to assess the precursor distribution in the m/z ion mobility plane to generate a diaPASEF acquisition scheme with variable window isolation widths aligned to the precursor density in m/z. Data were acquired using 20 cycles with three mobility window scans each (creating 60 windows) covering the diagonal scan line for doubly and triply charged precursors, with singly charged precursors able to be excluded by their position in the m/z ion mobility plane. These precursor isolation windows were defined between 350 and 1,250 m/z, with 1/K0 of 0.6–1.45 V s cm−2. The diaPASEF raw file processing and control of peptide-level and protein-level false discovery rates (FDRs), assembly of proteins from peptides and protein quantification from peptides were performed using library-free analysis in DIA-NN (version 1.8)150 searched against a Swiss-Prot human database (January 2021). Database search criteria largely followed the default settings for directDIA including: tryptic with two missed cleavages, carbamidomethylation of cysteine and oxidation of methionine and a precursor Q-value (FDR) cutoff of 0.01. The precursor quantification strategy was set to robust LC (high accuracy) with retention-time-dependent cross-run normalization. Resulting data were filtered to only include proteins that had a minimum of three counts in at least four replicates of each independent comparison of treatment sample to the DMSO control. Protein abundances were globally normalized using in-house scripts in the R framework (R Development Core Team, 2014). Proteins with missing values were imputed by random selection from a Gaussian distribution either with a mean of the nonmissing values for that treatment group or with a mean equal to the median of the background (in cases where all values for a treatment group were missing42). Protein abundances were scaled and significant changes comparing the relative protein abundance of these treatments to DMSO control comparisons were assessed by moderated t-test as implemented in the limma package within the R framework151. Mutations were induced with primers on a lentiviral compatible construct expressing ENL–BFP. In a 1:1 ratio, each mutated plasmid was Gibson-assembled into the other half of the pRRL-SFFV-ENL-GGGS-3×V5-BFP backbone. Lentivirus of each plasmid (pRRL-SFFV-ENL-GGGS-3×V5-BFP, pRRL-SFFV-ENLR16A-GGGS-3×V5-BFP, pRRL-SFFV-ENLD31A-GGGS-3×V5-BFP, pRRL-SFFV-ENLM33A-GGGS-3×V5-BFP, pRRL-SFFV-ENLQ41A-GGGS-3×V5-BFP, pRRL-SFFV-ENLK72A-GGGS-3×V5-BFP, pRRL-SFFV-ENLE74A-GGGS-3×V5-BFP and pRRL-SFFV-ENLR96A-GGGS-3×V5-BFP) was prepared as described above and transduced into MV4;11 cells. Cells were selected for BFP expression. BFP signal was measured by flow cytometry after a 24-h treatment with (S)-dHTC1, (R)-dHTC1 or SR-1114 and normalized to DMSO (n = 3) at the doses indicated. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The sequencing data discussed in this publication were to deposited the NCBI GEO127 under accession number GSE278582. Structural coordinates were deposited to the PDB and are available under accession numbers 9GY3 (CRBNmidi:dHTC1) and 9DUR (DDB1∆BPB•CRBN–dHTC1–ENL YEATS). The cryo-EM data generated by this study were deposited to the EM Data Bank under accession code EMD-47174. Source data are provided with this paper. Schreiber, S. L. The rise of molecular glues. & Crabtree, G. R. Chemically induced proximity in biology and medicine. Brown, E. J. et al. A mammalian protein targeted by G1-arresting rapamycin–receptor complex. Nalawansha, D. A., Mangano, K., den Besten, W. & Potts, P. R. TAC-tics for leveraging proximity biology in drug discovery. Burslem, G. M. & Crews, C. M. Proteolysis-targeting chimeras as therapeutics and tools for biological discovery. & Thomä, N. H. Haven't got a glue: protein surface variation for the design of molecular glue degraders. Lopez-Girona, A. et al. Cereblon is a direct protein target for immunomodulatory and antiproliferative activities of lenalidomide and pomalidomide. Krönke, J. et al. Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells. Krönke, J. et al. Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS. Matyskiela, M. E. et al. A novel Cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase. Petzold, G., Fischer, E. S. & Thomä, N. H. Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase. Watson, E. R. et al. Molecular glue CELMoD compounds are regulators of Cereblon conformation. Donovan, K. A. et al. Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane Radial Ray syndrome. Crystal structure of the SALL4–pomalidomide–Cereblon–DDB1 complex. Chemically induced degradation of the oncogenic transcription factor BCL6. Słabicki, M. et al. Small-molecule-induced polymerization triggers degradation of BCL6. The CDK inhibitor CR8 acts as a molecular glue degrader that depletes cyclin K. Nature 585, 293–297 (2020). Rational discovery of molecular glue degraders via scalable chemical profiling. Discovery of a molecular glue promoting CDK12–DDB1 interaction to trigger cyclin K degradation. Lim, M. et al. DCAF16-based covalent handle for the rational design of monovalent degraders. Rational chemical design of molecular glue degraders. Garnar-Wortzel, L. et al. Chemical inhibition of ENL/AF9 YEATS domains in acute leukemia. Li, S., Wu, P., Moses, J. E. & Sharpless, K. B. Multidimensional SuFEx click chemistry: sequential sulfur(VI) fluoride exchange connections of diverse modules launched from an SOF4 hub. Liu, F. et al. Biocompatible SuFEx Click Chemistry: thionyl tetrafluoride (SOF4)-derived connective hubs for bioconjugation to DNA and proteins. Kitamura, S. et al. Sulfur(VI) fluoride exchange (SuFEx)-enabled high-throughput medicinal chemistry. Carter, T. R. et al. SuFEx-based chemical diversification for the systematic discovery of CRBN molecular glues. Schwinn, M. K. et al. CRISPR-mediated tagging of endogenous proteins with a luminescent peptide. Riching, K. M. et al. Quantitative live-cell kinetic degradation and mechanistic profiling of PROTAC mode of action. Targeted protein degradation via intramolecular bivalent glues. Discovery of a drug-like, natural product-inspired DCAF11 ligand chemotype. Kagiou, C. et al. Alkylamine-tethered molecules recruit FBXO22 for targeted protein degradation. Mayor-Ruiz, C. et al. Plasticity of the Cullin–RING ligase repertoire shapes sensitivity to ligand-induced protein degradation. Heim, C., Spring, A.-K., Kirchgäßner, S., Schwarzer, D. & Hartmann, M. D. Identification and structural basis of C-terminal cyclic imides as natural degrons for Cereblon. & Handa, H. Exploiting ubiquitin ligase Cereblon as a target for small-molecule compounds in medicine and chemical biology. Kroupova, A. et al. Design of a Cereblon construct for crystallographic and biophysical studies of protein degraders. Baek, K. et al. Unveiling the hidden interactome of CRBN molecular glues. Oleinikovas, V., Gainza, P., Ryckmans, T., Fasching, B. & Thomä, N. H. From thalidomide to rational molecular glue design for targeted protein degradation. Donovan, K. A. et al. Mapping the degradable kinome provides a resource for expedited degrader development. Gechijian, L. N. et al. Functional TRIM24 degrader via conjugation of ineffectual bromodomain and VHL ligands. BET bromodomain proteins function as master transcription elongation factors independent of CDK9 recruitment. Structural basis of PROTAC cooperative recognition for selective protein degradation. Douglass, E. F., Miller, C. J., Sparer, G., Shapiro, H. & Spiegel, D. A. A comprehensive mathematical model for three-body binding equilibria. Hanzl, A. et al. Functional E3 ligase hotspots and resistance mechanisms to small-molecule degraders. Structure of the DDB1–CRBN E3 ubiquitin ligase in complex with thalidomide. Erb, M. A. Small-molecule tools for YEATS domain proteins. Lawrence, M. C. & Colman, P. M. Shape complementarity at protein/protein interfaces. Continuous evolution of compact protein degradation tags regulated by selective molecular glues. Transcription control by the ENL YEATS domain in acute leukaemia. Wan, L. et al. ENL links histone acetylation to oncogenic gene expression in acute myeloid leukaemia. Schumacher, H., Smith, R. L. & Williams, R. T. The metabolism of thalidomide: the spontaneous hydrolysis of thalidomide in solution. Liu, Y. et al. Small-molecule inhibition of the acyl-lysine reader ENL as a strategy against acute myeloid leukemia. Xue, Z. et al. A potent and selective ENL degrader suppresses oncogenic gene expression and leukemia progression. Selective inhibition of BET bromodomains. Shergalis, A. G. et al. CRISPR screen reveals BRD2/4 molecular glue-like degrader via recruitment of DCAF16. et al. Pd(COD)(DQ): a stable, versatile, and monometallic palladium(0) source for organometallic synthesis and catalysis. Zogu, A. et al. Perfluorooxosulfate salts as SOF4-gas-free precursors to multidimensional SuFEx electrophiles. Winter, G. E. et al. Phthalimide conjugation as a strategy for in vivo target protein degradation. Horn-Ghetko, D. et al. Ubiquitin ligation to F-box protein targets by SCF–RBR E3–E3 super-assembly. Scott, D. C. et al. Two distinct types of E3 ligases work in unison to regulate substrate ubiquitylation. Zhang, X., Crowley, V. M., Wucherpfennig, T. G., Dix, M. M. & Cravatt, B. F. Electrophilic PROTACs that degrade nuclear proteins by engaging DCAF16. Huppertz, M. C. et al. Recording physiological history of cells with chemical labeling. Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation. Potent and selective bivalent inhibitors of BET bromodomains. Selective degradation of GSPT1 by Cereblon modulators identified via a focused combinatorial library. Identification of potent, selective, and orally bioavailable small-molecule GSPT1/2 degraders from a focused library of Cereblon modulators. Acute pharmacological degradation of Helios destabilizes regulatory T cells. Redirecting the neo-substrate specificity of Cereblon-targeting PROTACs to Helios. Discovery and characterization of a selective IKZF2 glue degrader for cancer immunotherapy. Discovery of molecular glue degraders via isogenic morphological profiling. Wang, Z. et al. Direct-to-biology, automated, nano-scale synthesis, and phenotypic screening-enabled E3 ligase modulator discovery. Ting, P. Y. et al. A molecular glue degrader of the WIZ transcription factor for fetal hemoglobin induction. Kozicka, Z. et al. Design principles for cyclin K molecular glue degraders. Thomas, K. L. et al. Degradation by design: new cyclin K degraders from old CDK inhibitors. Fares, I. et al. Pyrimidoindole derivatives are agonists of human hematopoietic stem cell self-renewal. Yeo, M. J. R. et al. UM171 glues asymmetric CRL3–HDAC1/2 assembly to degrade CoREST corepressors. Xie, X. et al. Converging mechanism of UM171 and KBTBD4 neomorphic cancer mutations. Mondal, M. & Hirsch, A. K. H. Dynamic combinatorial chemistry: a tool to facilitate the identification of inhibitors for protein targets. Accelerating PROTACs discovery through a direct-to-biology platform enabled by modular photoclick chemistry. Integrated direct-to-biology platform for the nanoscale synthesis and biological evaluation of PROTACs. Thomas, R. P. et al. A direct-to-biology high-throughput chemistry approach to reactive fragment screening. Hendrick, C. E. et al. Direct-to-biology accelerates PROTAC synthesis and the evaluation of linker effects on permeability and degradation. Mason, J. W. et al. DNA-encoded library-enabled discovery of proximity-inducing small molecules. Rational screening for cooperativity in small-molecule inducers of protein–protein associations. Poirson, J. et al. Proteome-scale discovery of protein degradation and stabilization effectors. Serebrenik, Y. V., Mani, D., Maujean, T., Burslem, G. M. & Shalem, O. Pooled endogenous protein tagging and recruitment for systematic profiling of protein function. Zhang, X. et al. DCAF11 supports targeted protein degradation by electrophilic proteolysis-targeting chimeras. Sarott, R. C. et al. Chemical specification of E3 ubiquitin ligase engagement by cysteine-reactive chemistry. A. et al. A CRISPR activation screen identifies FBXO22 supporting targeted protein degradation. Chemically induced nuclear pore complex protein degradation via TRIM21. Selective degradation of multimeric proteins by TRIM21-based molecular glue and PROTAC degraders. Scemama de Gialluly, M. A. et al. Elaboration of molecular glues that target TRIM21 into TRIMTACs that degrade protein aggregates. Cheng, Y. et al. TRIM21–NUP98 interface accommodates structurally diverse molecular glue degraders. Zhang, X., Simon, G. M. & Cravatt, B. F. Implications of frequent hitter E3 ligases in targeted protein degradation screens. Defining molecular glues with a dual-nanobody cannabidiol sensor. Annunziato, S. et al. Molecular surface mimicry enables CRBN to target G3BP2 for degradation. Steger, M. et al. Unbiased mapping of Cereblon neosubstrate landscape by high-throughput proteomics. Petzold, G. et al. Mining the CRBN target space redefines rules for molecular glue-induced neosubstrate recognition. Translation termination factor GSPT1 is a phenotypically relevant off-target of heterobifunctional phthalimide degraders. Matyskiela, M. E. et al. SALL4 mediates teratogenicity as a thalidomide-dependent Cereblon substrate. Nguyen, T. M. et al. Proteolysis-targeting chimeras with reduced off-targets. & Blaschke, G. Investigations on the in vitro racemization of thalidomide by high-performance liquid chromatography. B. et al. A combinatorial F box protein directed pathway controls TRAF adaptor stability to regulate inflammation. Targeting F box protein FBXO3 to control cytokine-driven inflammation. Shao, W., Zumer, K., Fujinaga, K. & Peterlin, B. M. FBXO3 protein promotes ubiquitylation and transcriptional activity of AIRE (autoimmune regulator). Li, W. et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Heidenreich, D. et al. Structure-based approach toward identification of inhibitory fragments for eleven-nineteen-leukemia protein (ENL). Asiaban, J. N. et al. Cell-based ligand discovery for the ENL YEATS domain. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. Lazear, M. R. et al. Proteomic discovery of chemical probes that perturb protein complexes in human cells. Vinogradova, E. V. et al. An activity-guided map of electrophile–cysteine interactions in primary human T cells. Ting, L., Rad, R., Gygi, S. P. & Haas, W. MS3 eliminates ratio distortion in isobaric multiplexed quantitative proteomics. de Almeida, M. et al. AKIRIN2 controls the nuclear import of proteasomes in vertebrates. Bertomeu, T. et al. A high-resolution genome-wide CRISPR/Cas9 viability screen reveals structural features and contextual diversity of the human cell-essential proteome. DIALS: implementation and evaluation of a new integration package. McCoy, A. J. et al. Phaser crystallographic software. Towards automated crystallographic structure refinement with phenix.refine. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Edgar, R., Domrachev, M. & Lash, A. E. Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Abdulrahman, W. et al. A set of baculovirus transfer vectors for screening of affinity tags and parallel expression strategies. Weissmann, F. et al. biGBac enables rapid gene assembly for the expression of large multisubunit protein complexes. Trowitzsch, S., Bieniossek, C., Nie, Y., Garzoni, F. & Berger, I. New baculovirus expression tools for recombinant protein complex production. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. & Henderson, R. Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. Croll, T. I. ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps. Leman, J. K. et al. Macromolecular modeling and design in Rosetta: recent methods and frameworks. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Structure of the human Cereblon–DDB1–lenalidomide complex reveals basis for responsiveness to thalidomide analogs. Matyskiela, M. E. et al. A Cereblon modulator (CC-220) with improved degradation of Ikaros and Aiolos. Defining the human C2H2 zinc finger degrome targeted by thalidomide analogs through CRBN. Surka, C. et al. CC-90009, a novel Cereblon E3 ligase modulator, targets acute myeloid leukemia blasts and leukemia stem cells. Bouguenina, H. et al. A degron blocking strategy towards improved CRL4CRBN recruiting PROTAC selectivity. Structural and biophysical comparisons of the pomalidomide- and CC-220-induced interactions of SALL4 with Cereblon. Collaboration gets the most out of software. Reichermeier, K. M. et al. PIKES analysis reveals response to degraders and key regulatory mechanisms of the CRL4 network. Rapid and in-depth coverage of the (phospho-)proteome with deep libraries and optimal window design for dia-PASEF. Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. & Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. This work was supported by the Ono Pharma Foundation Breakthrough Science Initiative Awards Program, the Baxter Foundation Young Investigator Award, the National Institutes of Health (NIH) Office of the Director (DP5-OD26380) and the National Cancer Institute (R01CA280720, M.A.E. ), a PhD Fellowship from the Boehringer Ingelheim Fonds (M.M.O), the German Research Foundation (Deutsche Forschungsgemeinschaft) project number 511811315 (D.V.W. ), United States National Science Foundation (CHE-2046286, K.M.E. ), the NIH Institutional National Research Service Award (T32TR004396, B.D.C.) H.F. received funding from a Japan Society for the Promotion of Science postdoctoral fellowship (23KJ1669). Funding is also gratefully acknowledged from the Innovative Medicines Initiative 2 (IMI2) Joint Undertaking under grant agreement 875510 (EUbOPEN project, A.C.). The IMI2 Joint Undertaking receives support from the European Union's Horizon 2020 research and innovation program, European Federation of Pharmaceutical Industries and Associations companies and associated partners KTH, OICR, Diamond and McGill. lab are supported by the Austrian Academy of Sciences. AITHYRA is supported by the Austrian Academy of Sciences and Boehringer Ingelheim Stiftung. lab is further supported by funding from the European Research Council under the European Union's Horizon 2020 research and innovation program (grant agreement 851478), as well as by funding from the Austrian Science Fund (projects P7909, P36746 and P5918723) and the Vienna Science and Technology Fund (project LS21-015). was supported by grants from the NIH (CA259273 and CA066996), the V Foundation for Cancer Research and Curing Kids Cancer. We thank G. Tsaprailis at the Proteomics Core at The Herbert Wertheim UF Scripps Institute for aid in proteomics experiments. We thank the Core Facility Flow Cytometry of the Medical University of Vienna for access to flow cytometry instruments and assistance with cell sorting, the CeMM Biomedical Sequencing Facility for NGS sample processing, sequencing and data curation and J. Zuber and members of his laboratory for sharing KBM7 iCas9 cell lines and plasmids. We thank the staff of the Harvard Cryo-EM Center for Structural Biology for their technical expertise and support during grid screening and data collection. We acknowledge the SBGrid consortium for assistance with structural biology software packages and high-performance computing. X-ray crystallography was supported by proposal mx35324 at the Diamond Light Source and beamline scientists at I24. Molecular graphics and analyses were performed with UCSF ChimeraX, developed by the Resource for Biocomputing, Visualization and Informatics at the University of California, San Francisco, with support from NIH R01-GM129325 and the Office of Cyber Infrastructure and Computational Biology, National Institute of Allergy and Infectious Diseases. These authors contributed equally: James B. Shaum, Miquel Muñoz i Ordoño. Department of Chemistry, The Scripps Research Institute, La Jolla, CA, USA James B. Shaum, Erica A. Steen, Jordan Janowski, Eric M. Bilotta, Paige A. Barta, Natalia Milosevich, Lauren M. Hargis, Timothy R. Bishop, Trever R. Carter, Bryce da Camara, Wen-Ji He, Benjamin F. Cravatt, Keary M. Engle, Bruno Melillo & Michael A. Erb CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria Miquel Muñoz i Ordoño, Matthias Hinterndorfer, Fabian Offensperger & Georg E. Winter AITHYRA Research Institute for Biomedical Artificial Intelligence of the Austrian Academy of Sciences, Vienna, Austria Miquel Muñoz i Ordoño & Georg E. Winter Division of Hematology/Oncology, Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston Children's Hospital, and Harvard Medical School, Boston, MA, USA Daniela V. Wenge, Sydney R. Schweber, Charlie Hatton, Yanhe Wen & Scott A. Armstrong Hakyung Cheong, Moritz Hunkeler, Abby M. Thornhill, Katherine A. Donovan & Eric S. Fischer Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, MA, USA Moritz Hunkeler, Katherine A. Donovan & Eric S. Fischer Centre for Targeted Protein Degradation, School of Life Sciences, University of Dundee, Dundee, UK Zoe J. Rutter, Hirotake Furihata & Alessio Ciulli Department of Biochemistry, Albert Einstein College of Medicine, Bronx, New York, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the CRISPR screens on UPS components, splitHalo assays and DMS screens with input from M. Hinterndorfer and F.O. performed the SPR and HTRF. helped with the material characterization. performed the animal model studies with S.R.S., C.H. provided access to Pd(COD)(DQ) and established the protocols. Supervision was carried out by K.A.D., B.F.C, S.K., A.C., S.A.A., E.S.F., G.E.W. All authors edited and approved the paper. Correspondence to Georg E. Winter or Michael A. Erb. are inventors on a patent application related to SR-0813 and TM-7. are inventors on a patent application related to dHTC1. holds equity in Nexo Therapeutics and serves on their scientific advisory board (SAB). lab has received research funding from Pfizer. is a founder, SAB member and equity holder of Civetta Therapeutics, Proximity Therapeutics, Neomorph (also on the board of directors), Stelexis Biosciences and Anvia Therapeutics (also on the board of directors). He is an equity holder and SAB member of Avilar Therapeutics, Photys Therapeutics and Ajax Therapeutics and an equity holder in Lighthorse Therapeutics and CPD4 (also on the board of directors). is a consultant to Novartis, EcoR1 Capital and Deerfield. lab receives or has received research funding from Deerfield, Novartis, Ajax, Interline, Bayer and Astellas. A.C. is a cofounder and shareholder of Amphista Therapeutics, a company that develops targeted protein degradation platforms. The A.C. laboratory receives or has received sponsored research support from Almirall, Amgen, Amphista Therapeutics, Boehringer Ingelheim, Eisai, Merck KgaA, Nurix Therapeutics, Ono Pharmaceutical and Tocris-Biotechne. A.C. is on the SAB of ProtOS Bio. are founders and/or scientific advisors to Magnet Biomedicine. receives or has received consulting fees from Neomorph and Kronos Bio. has received research support from Janssen and Syndax. is an inventor on a patent related to MENIN inhibition WO/2017/132398A1. The other authors declare no competing interests. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, MV4;11 cells expressing HiBiT-ENL were treated with the indicated compounds for 24 h. HiBiT luminescence signal is normalized to DMSO-treated cells. b, Intracellular engagement of ENL YEATS by TM-7 and SR-0813. c, ENL YEATS affinity determination using SPR (mean of 2 technical replicates). Same experiment as Extended Data Fig. d, Frequency histograms of key physicochemical properties for the library of amine building blocks used in high-throughput derivatization. e, Schematic of a model reaction run on each library plate, and the corresponding relative yields (measured as percent integration of total). f, BRD4 engagement in vitro, as measured by HTRF competitive histone peptide displacement assay (mean of 3 biological replicates ± S.E.M.). g, MV4;11 cells were engineered to stably express an amino (N) terminal HiBiT ENL tag. Shown is the raw HiBiT luminescence signal for cells (20,000 cells in 20 µL of media) treated with vehicle control or SR-1114 (5 µM) for 24 h, demonstrating signal window. Left demonstrates cells bearing a direct fusion of the HiBiT tag to ENL. Right demonstrates cells with a flexible GSG linker. h, Immunoblot analysis of ENL in a panel of AML cell lines treated with rac-dHTC1 (10 µM) or DMSO control (24 h). i, ENL-HiBiT expressing AML cell lines treated with rac-dHTC1 for 24 hours. Signal is normalized to DMSO treated cells. j, Immunoblot analysis of AF9 levels in Jurkat cells treated with rac-dHTC1 or DMSO control. a,b,i mean, 4 biological replicates ± S.E.M. a, Validation of an ENL reporter construct. KBM7 cells expressing an ENL-TagBFP-P2A-mCherry dual fluorescence stability reporter were treated with 10 µM compound or DMSO control for 8 h (1 independent experiment). b, FACS-based CRISPR screens for UPS components affecting ENL stability in KBM7 reporter cells (ENL-BFP-P2A-mCherry) treated with SR-1114 (10 µM) for 8 h (6 sgRNAs targeting 1301 UPS-associated genes). Gene-level fold change and P values were determined by one-sided MAGeCK114 analysis. c, Top: HiBiT luminescence for MV4;11-HiBiT-ENL cells treated with rac-dHTC1 in dose response (normalized to DMSO, mean of 4 biological replicates ± S.E.M.). Bottom: immunoblot analysis of MV4;11 cells treated with rac-dHTC1 (10 µM, 6 h) or DMSO vehicle control. d, Top: immunoblot analysis of MOLM13-HiBiT-ENL cells confirming CRBN overexpression. Bottom: treatment of MOLM13-HiBiT-ENL cells with rac-dHTC1 in dose response. HiBiT luminescence is normalized to DMSO (mean of 3 biological replicates ± S.E.M.). e, MV4;11-HiBiT-ENL cells pretreated with pomalidomide (25 µM, 1 h) or DMSO control prior to rac-dHTC1 treatment (24 h). Luminescence normalized to DMSO vehicle control (mean of 4 biological replicates ± S.E.M.). 1g are taken from the same experiment (DMSO pre-treatment is repeated in both panels). g, Scatterplot of APMS results showing relative protein abundance by label-free proteomics following enrichment of recombinant Flag-CRBN-DDB1ΔB from MOLT4 cell lysates treated with 1 µM dHTC1 or DMSO. P values were calculated using a moderated two-sided t-test implemented in the limma package (4 independent replicates of each treatment group). Select super elongation complex members are shown in blue. h, Same data presented in Fig. 1h, but with black dots indicating common CRBN neosubstrates43 detected by proteomics mass spectrometry. b, HiBiT-based intracellular ENL YEATS target engagement assay (3 h). MLN4924 pre-treatment (1 µM, 1 h) was used to prevent the potential for compound-induced degradation. Signal is normalized to DMSO-treated cells (mean, 4 biological replicates ± S.E.M). 4d is taken from the same experiment (rac-dHTC1 data are repeated in both panels). c, Binding to immobilized ENL YEATS as measured by SPR (mean of 2 technical replicates.). 4c is taken from the same experiment (rac-dHTC1 data are repeated in both panels). d, HiBiT-based ENL degradation assay (24 h). HiBiT luminescence signal is normalized to DMSO-treated cells (mean, 4 biological replicates ± S.E.M.). e, CRBN FP assay for 7 in the presence of recombinant ENL YEATS domain (3 µM) or buffer (mean of 3 biological replicates ± S.E.M., background subtracted and normalized to DMSO). f, Urea analog 8 (same as panel e). 2c (rac-dHTC1 and buffer are repeated in both panels). h, Structure of two JQ1 analogs functionalized with the spirosuccinimide building block, BRD4 intracellular target engagement in MV4;11-BRD4-HiBiT cells (1 h, HiBiT luminescence normalized to DMSO, 3 biological replicates ± S.E.M. ), and BRD4 degradation assay in MV4;11-BRD4-HiBiT cells (16 h, HiBiT luminescence normalized to DMSO, mean, 4 biological replicates ± S.E.M.). i, Structure of PARP1/2 inhibitor scaffolds functionalized with the spirosuccinimide building block, PARP1 intracellular target engagement in MV4;11-PARP1-HiBiT cells after 4 h treatment (HiBiT luminescence signal normalized to DMSO, mean of 3 biological replicates ± S.E.M. ), and HiBiT degradation assay in MV4;11-PARP1-HiBiT cells after 16 h treatment (HiBiT luminescence signal normalized to DMSO, mean of 4 biological replicates ± S.E.M.). Compounds are compared to SK-575, a known PARP1 PROTAC degrader, or rucaparib and olaparib, which are known PARP1 ligands. a, HiBiT degradation assay in MV4;11-HiBiT-ENL cells (signal normalized to DMSO treated cells, 4 biological replicates). b, (R)-dHTC1, same as panel a. c, SPR analysis of binding to immobilized ENL (mean of 2 technical replicates). 3c is taken from the same experiment (rac-dHTC1 data is repeated in both panels). d, HiBiT-based intracellular ENL YEATS target engagement assay (3 h). OCI-AML2 cells were pretreated with MLN4924 (1 µM, 1 h) to prevent compound-induced degradation. Signal is normalized to DMSO-treated cells (4 biological replicates). 3b is taken from the same experiment (rac-dHTC1 data is repeated in both panels). e, Intracellular CRBN target engagement assessed in MV4;11-BRD4-HiBiT via rescue of dBET6-induced BRD4 degradation. Cells pretreated with compounds (2 h) in dose response before dBET6 (500 nM, 1 h, 3 biological replicates). Data for pomalidomide is repeated in Fig. 2d, which originated from a single experiment. f, SPR analysis of binding to immobilized CRBNmidi (3 technical replicates). g, CRBN FP signal following treatment with the indicated compound in the presence of ENL YEATS (3 µM) or buffer control (FP signal is background subtracted and normalized to DMSO, 3 biological replicates). h, SPR analysis of CRBN–DDB1 binding to immobilized ENL YEATS in the presence of 1 µM (S)- or (R)-dHTC1 (2 technical replicates). i, (S)-dHTC1 binding to ENL measured by SPR, with or without CRBN-DDB1 (500 nM) present (2 technical replicates). Each trace is normalized to response at saturation. (S)-dHTC1 without CRBN-DDB1 data is from panel c. (S)-dHTC1 with CRBN-DDB1 is from panel j. j, SPR analysis of CRBN-DDB1 (500 nM) binding to immobilized ENL YEATS in the presence of an (S)- or (R)-dHTC1 dose response (2 technical replicates). k, HTRF-based assay to measure induced proximity of ENL YEATS and CRBN-DDB1 with (S)- and (R)-dHTC1 (HTRF signal normalized to DMSO, 3 technical replicates). l, HiBiT degradation assay in MOLM13-HiBiT-ENL cells overexpressing CRBN (HiBiT luminescence signal normalized to DMSO, 3 biological replicates). a,b,d-g,k,l, mean ± S.E.M. a, Flow cytometry following treatment in MV4;11 CRBN KO “parental” cells expressing ENL-TagBFP-P2A-mCherry. Histograms for SR-1114, rac-dHTC1 and DMSO treatments overlap as ENL degradation is CRBN-dependent. b, Reintroduction of wild-type CRBN restores ENL degradation. c, Reintroduction of CRBN with inactivating mutation (W380I) fails to mediate ENL degradation. d, Cells transduced with CRBN DMS library regained activity; roughly 70% of cells showing ENL degradation. e, MV4;11 CRBN KO cells expressing ENL-TagBFP-P2A-mCherry were transduced with a DMS library, which reintroduced CRBN with single amino acid mutations within a 10 Å radius of the thalidomide binding site. After these cells were treated with rac-dHTC1 (1 µM, 8 h), endpoint ENL levels were quantified by flow cytometry. Results are depicted indicating log2 fold change of ENLhigh over the unsorted pools. g, Heatmap comparing CRBN DMS results for rac-dHTC1 and SR-1114. Arrows above indicate residues showing selectivity for dHTC1 (blue), SR-1114 (gray), or mixed (purple) that were selected for follow-up assays. h, Validation of altered sensitivity to ENL degraders by individual CRBN mutants. MV4;11 CRBN knockout cells expressing ENL-TagBFP-P2A-mCherry reporter with 1 µM of rac-dHTC1 or 1 µM of SR-1114 (8 h), assessed by flow cytometry (BFP/mCherry normalized to DMSO, 3 technical replicates, 2 independent experiments ± S.D.). i, Immunoblot analyses of ENL levels in MV4;11 CRBN knockout cells expressing ENL-TagBFP-P2A-mCherry. Wild-type or mutant CRBN was reintroduced by lentiviral expression to evaluate mutations that selectively impact the ability of dHTC1 or SR-1114 to degrade ENL (1 µM, 6 h). For ENL, the lower molecular weight band is endogenous protein and higher molecular weight band is reporter construct. a, Co-crystal structure of (S)-dHTC1 bound to CRBNmidi. b, View of the IMiD-binding site occupied by (S)-dHTC1. Residues indicated with an arrow in Extended Data Fig. c, Differential activity between CRBN DMS screen for RKO cells treated with CC-885 (500 nM, 7 d)49 and MV4;11-ENL-TagBFP-P2A-mCherry cells rac-dHTC1 (1 µM, 8 h). d, Validation of DMS screening results, comparing CRBN mutations in MV4;11 CRBN−/− GSPT1-GFP-P2A-mCherry cells treated with CC-885 versus MV4;11 CRBN−/− ENL-TagBFP-P2A-mCherry cells treated with rac-dHTC1. BFP/mCherry signal is normalized to respective DMSO control (3 technical replicates, 2 independent experiments ± S.D.). e, View of P54 and G61 residues distal ( > 10 Å) to the IMiD-binding pocket. a, Overview of processing workflow from raw micrograph (scale bar indicated) to final maps. Particles belonging to colored volumes were taken into subsequent steps. Contour levels of maps after homogeneous refinement, local refinement, and postprocessing with DeepEMhancer are 0.025, 0.03, and 0.10 respectively. b, Local resolution mapped onto final map postprocessed by DeepEMhancer, c, FSC plot, d, Directional FSC plot. f, Model-to-map FSC for 9DUR in EMD-47174. g, h, Cryo-EM density examples (threshold 0.10) for ENL YEATS and dHTC1 (map postprocessed by DeepEMhancer). a, Top, a two-dimensional interaction map of dHTC1 bound to CRBN and ENL and, bottom, dHTC1 bound to ENL YEATS with key interactions highlighted. b, Left, an overlay of dHTC1 binding pose with CRBN in binary and ternary complex structures and, right, binding pose of dHTC1 with CRBN in the ternary complex with key amino acid residues highlighted. c, Peripheral CRBN loop (aa146-153) and its density (silver, threshold). d, MV4;11 ENL-TagBFP-P2A-mCherry cells (wild type or Q41A) treated with 1 µM of the indicated compounds or DMSO. BFP percentage normalized to DMSO (mean, 3 biological replicates ± S.E.M.). e, Superposition of the apo ENL YEATS domain crystal structure (6HQ0115, dark gray) to the dHTC1-bound ENL YEATS-CRBN/DDB1 structure showing ENL density (silver, threshold 0.03). Red arrow indicates movement of the ENL loop relative to the apo structure (aa 37-42). f, Viability of HL60 cells after 12-day treatment (cells were split 1:10 and fresh drug was added every 3 days) with (S)-dHTC1 or (R)-dHTC1 measured by ATP-dependent luminescence (normalized to DMSO, 3 independent experiments, mean of 3 biological replicates ± S.E.M.). g, Time-dependent and dose-responsive loss of HiBiT-ENL signal in MV4;11 cells treated with SR-1114. Luminescence normalized to DMSO vehicle control (mean, 4 biological replicates ± S.E.M.). plasma concentration and pharmacokinetic properties of male C57BL/6 mice dosed with (S)-dHTC1 (25, 50 or 75 mg/kg) via intraperitoneal injection (n = 3 mice). 50 mg/kg data is plotted in Fig. i, Mean ± S.E.M of plasma concentration of male C57BL/6 mice dosed with rac-dHTC1 at 10 mg/kg via intraperitoneal (IP) injection, 2 mg/kg via intravenous (IV) injection or 2 mg/kg per os (PO) (n = 9 mice). j, Scatter plot of RNA-seq analysis of mouse-cell-depleted bone marrow lysates from MV4;11 xenograft (same experiment as data from Fig. a, JQ1 was functionalized with an iminosulfur oxydifluoride SuFEx handle and used for parallel synthesis of 3,163 analogs. Products were formed as a 9:1 mix of enantiomers (S:R). b, Assay performance of MV4;11-BRD4-HiBiT cells for BRD4 degradation. Data is raw luminescence in MV4;11-BRD4-HiBiT cells treated with vehicle or dBET6. 5b hits were rescreened at 2 or 10 µM (16 h) in MV4;11-BRD4-HiBiT cells, counter screened against MV4;11-HiBiT-ENL cells (HiBiT luminescence normalized to DMSO:PBS vehicle control, 3 biological replicates). d, SPR analysis of binding to immobilized ENL-YEATS (mean of 2 technical replicates). Same experiment as Extended Data Fig. Luminescence normalized to DMSO (mean of 4 biological replicates ± S.E.M.). f, Dose-responsive effects of dHTC2, dHTC3, JQ1, and dBET6 (12 h treatment) in MV4;11 BRD4 HiBiT cells following indicated pretreatments (1 h). BRD4-HiBiT luminescence normalized to DMSO vehicle control (mean of 4 biological replicates ± S.E.M.). g-h, BRD4 degradation and target engagement assays were performed to select concentrations for growth-based CRISPR knockout screen presented in Fig. Left, target engagement by rescue of dBET6-induced BRD4-HiBiT degradation. NALM6-BRD4-HiBiT cells were treated with the indicated compounds in dose response for 1 h, prior to dBET6 treatment (500 nM final concentration) for 1 h (mean of 4 biological replicates ± S.E.M., normalized to DMSO treated cells). Right, a 16 h dose-responsive loss of BRD4-HiBiT signal in NALM6 cells treated with 12 and 13 (HiBiT luminescence normalized to DMSO vehicle control, mean of 4 biological replicates ± S.E.M.). Gray bands indicate concentrations selected for genetic screen, which induce near-maximal degradation with minimal BRD4 engagement. i, FACS-based CRISPR screens for UPS components affecting BRD4 stability in KBM7 reporter cells (KBM7 iCas9 BRD4s-TagBFP-P2A-mCherry) treated with DMSO or dHTC2 at 2 µM for 16 h (6 sgRNAs targeting 1301 UPS-associated genes). Gene-level fold change and P values determined by one-sided MAGeCK114 analysis. 26S proteasome members are indicated in black. a, KBM7-iCas9-BRD4(TandemBD)-TagBFP-P2A-mCherry reporter cells transduced with sgRNAs targeting indicated genes and co-expressing iRFP720 to identify sgRNA-positive cells. Cas9 was induced with doxycycline 4 days before treatment. b, CRISPR/Cas9 RNPs containing one of four guides delivered to cells by electroporation. sgFBXO3 #4, was selected for further studies. c, Left, MV4;11-BRD4-HiBiT and, right, NALM6-BRD4-HiBiT cells (parental) or those bearing an FBXO3 knockout treated with dHTC3 (16 h). BRD4-HiBiT luminescence normalized to DMSO-treated cells (mean, 4 biological replicates). HiBiT luminescence normalized to DMSO (mean, 4 biological replicates). e, SplitHalo assays in RPE1 cells ectopically expressing FBXO3-GFP-cpHalo and Hpep6-BRD4s-EF1αs-TagBFP constructs. Cells pre-treated with carfilzomib (1 h), and then treated with indicated compounds (10 nM dBET6, 2 µM dHTC3 and 2 µM epi-dHTC3, 3 h) and 100 nM TAMRA-CA. GFP/BFP double-positive cells were gated, SplitHalo signal (TAMRA, PE channel) was normalized to cpHalo expression (GFP signal) to account for expression-level variation. Data shown as TAMRA/GFP ratios, normalized to DMSO-treated control. f, KBM7-iCas9 cells expressing BRD4short or tandem BD1-BD2 bromodomains (BRD4tandem) in TagBFP-P2A-mCherry vector were pre-treated (1 h) with indicated compounds (20 µM) before dBET6 (10 nM) or dHTC3 (2 µM) (8 h). Signal (BFP/mCherry ratio) is normalized to DMSO-treated cells. g, KBM7-iCas9 cells expressing BRD4 BD1 or BD2 (with BRD4 nuclear localization sequence) fused to TagBFP-P2A-mCherry were treated with DMSO, 10 nM dBET6, or 2 µM dHTC3 (16 h). BFP/mCherry ratios are normalized to DMSO-treated cells. h, SplitHalo assays conducted in RPE1 cells (same as panel e) expressing BRD4(S) with BD1 (N140F) or BD2 (N433F) mutants. Cells pre-treated with carfilzomib and either DMSO or 10 µM JQ1 (1 h), followed by DMSO or 2 µM dHTC3 (3 h) with 100 nM TAMRA-CA. HEK293T-BRD4-HiBiT cells were pre-treated with compounds (1 h) prior to 16 h treatment with dHTC3 (1 µM, mean, 4 biological replicates). e-h mean, 3 biological replicates, 2 independent experiments. 1, NMRs, flow cytometry gating strategies and raw sensorgrams. Key physicochemical properties for the library of amine building blocks used in high-throughput derivatization. Crude reaction products were screened for ENL degradation (5 µM, 24 h) in MV4;11 cells expressing HiBiT–ENL. HiBiT was luminescence normalized to DMSO:PBS vehicle-control-treated cells (single independent experiment). Resynthesized hits were rescreened by HiBiT–ENL and counterscreened against BRD4–HiBiT (three biological replicates). HiBiT luminescence was normalized to DMSO:PBS vehicle-control-treated cells. Quantitative TMT-based expression proteomics analysis of MV4;11 cells treated with rac-dHTC1 (1 µM, 16 h) or DMSO vehicle control (5,513 proteins with >2 peptides; data were filtered using DTAselect 2.0 within IP2). P values were calculated using a two-tailed Student's t-test (three biological replicates). AP–MS results showing relative protein abundance by label-free proteomics following enrichment of recombinant FLAG–CRBN–DDB1ΔB from MOLT4 cell lysates treated with 1 µM dHTC1 or DMSO. P values were calculated using a moderated two-sided t-test implemented in the limma package (four independent replicates of each treatment group). CRBN FP signal (background subtracted and normalized to DMSO) following treatment with the indicated compounds, in the presence of ENL (3 µM) or buffer control (mean; three biological replicates). MV4;11 CRBN-KO cells expressing ENL–TagBFP–P2A–mCherry were transduced with a DMS library, which reintroduced CRBN with single-amino-acid substitutions within a 10-Å radius of the thalidomide-binding site. After these cells were treated with SR-1114 (1 µM, 8 h), endpoint ENL levels were quantified by flow cytometry. Results are depicted indicating log2 fold change of ENLhigh over the unsorted pools. MV4;11 CRBN-KO cells expressing ENL–TagBFP–P2A–mCherry were transduced with a DMS library, which reintroduced CRBN with single-amino-acid substitutions within a 10-Å radius of the thalidomide-binding site. After these cells were treated with dHTC1 (1 µM, 8 h), endpoint ENL levels were quantified by flow cytometry. Results are depicted indicating log2 fold change of ENLhigh over the unsorted pools. Crude reaction products were screened for BRD4 degradation (2 µM, 16 h) in MV4;11 cells expressing BRD4–HiBiT. HiBiT luminescence was normalized to DMSO:PBS vehicle-control-treated cells (single independent experiment). 5b were rescreened at 2 or 10 µM (16 h) in MV4;11 BRD4–HiBiT cells and counter screened against MV4;11 HiBiT–ENL cells. HiBiT luminescence was normalized to DMSO:PBS vehicle-control-treated cells (three biological replicates). Quantitative TMT-based expression proteomics analysis of MV4;11 cells treated with dHTC2 and dHTC3 (1 µM, 16 h) or DMSO vehicle control (5,399 proteins with >2 peptides depicted; data were filtered using DTAselect 2.0 within IP2). P values were calculated using a two-tailed Student's t-test (three biological replicates). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Shaum, J.B., Muñoz i Ordoño, M., Steen, E.A. et al. High-throughput ligand diversification to discover chemical inducers of proximity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225537.htm'>Universe may end in a “big crunch,” new dark energy data suggests</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 08:28:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A Cornell physicist has calculated that the universe may be nearing the halfway point of a total lifespan of about 33 billion years. Using newly released data from major dark energy observatories, he concludes that the cosmos will continue expanding for roughly another 11 billion years before reaching its largest size. After that, it would begin to shrink, eventually collapsing back into a single point, much like a stretched rubber band snapping back. This concept was first introduced more than a century ago by Albert Einstein and has been central to modern predictions about how the universe will evolve. The universe is currently 13.8 billion years old and still expanding. If the cosmological constant is positive, expansion continues indefinitely. If it is negative, the universe would eventually stop growing, reach a maximum size, and then reverse direction, contracting until everything collapses to zero. Based on his calculations, that collapse would occur in about 20 billion years. Key evidence comes from new findings released this year by the Dark Energy Survey (DES) in Chile and the Dark Energy Spectroscopic Instrument (DESI) in Arizona. Tye noted that results from these two observatories, located in opposite hemispheres, closely agree with each other. Both projects aim to better understand dark energy, which makes up about 68% of the mass and energy in the universe. The universe does not appear to be governed solely by a pure cosmological constant. Something additional may be influencing how dark energy behaves. To account for this, Tye and his collaborators proposed a hypothetical particle with extremely low mass. Early in cosmic history, this particle would have acted like a cosmological constant, but over time its effects would have changed. DESI will continue collecting observations for another year. Additional projects are already contributing or preparing to begin, including the Zwicky Transient Facility in San Diego; the European Euclid space telescope; NASA's recently launched SPHEREx mission; and the Vera C. Rubin Observatory (named after Vera Rubin, M.S. Tye says it is encouraging that scientists can attempt to calculate the total lifespan of the universe in measurable terms. In the 1960s, we learned that it has a beginning. Then the next question is, 'Does it have an end?' Tye's co-authors are his former Hong Kong University of Science and Technology doctoral students Hoang Nhan Luu and Yu-Cheng Qiu. Note: Content may be edited for style and length. Intermittent Fasting Fails to Deliver in Major Weight Loss Review Could Sugary Drinks Be Fueling the Rise in Teen Anxiety? Scientists Crack a 50-Year Tissue Mystery With Major Cancer Implications Hard-to-Treat Cancers Have a Secret Trick That Researchers Finally Cracked Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225546.htm'>Scientists discover the enzyme that lets cancer rapidly rewire its DNA</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-16 06:02:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Although chromothripsis was first recognized more than ten years ago as a major force in cancer progression, researchers had not known what actually set it in motion. Cancer cells use several tactics to survive therapy, but chromothripsis stands out because of its sheer scale. Rather than building up mutations gradually over time, this process can create dozens or even hundreds of genetic alterations in a single catastrophic episode. The result is a rapid burst of evolution that can make tumors harder to control. Studies suggest that about one in four cancers shows signs of this type of chromosome damage, and in some cancers the rate is even higher. Nearly all osteosarcomas, an aggressive bone cancer, display evidence of chromothripsis, and many brain cancers show especially elevated levels. Chromothripsis begins when errors during cell division trap individual chromosomes inside small, fragile compartments known as micronuclei. When a micronucleus ruptures, the chromosome inside is left exposed. Until now, researchers did not know which nuclease was responsible for triggering this destructive chain reaction, making it impossible to design therapies aimed at stopping it. They monitored how each enzyme behaved in living cancer cells. It was uniquely able to enter micronuclei and fragment the DNA inside. When they removed the enzyme from brain cancer cells, chromosome shattering dropped dramatically. "These experiments showed us that N4BP2 isn't just correlated with chromothripsis. "This is the first direct molecular explanation for how catastrophic chromosome fragmentation begins." The team also examined more than 10,000 cancer genomes spanning multiple tumor types. Cancers with higher levels of N4BP2 activity showed significantly more chromothripsis and large scale structural rearrangements. These tumors also had increased amounts of extrachromosomal DNA (ecDNA), circular DNA fragments that often carry cancer promoting genes and are closely associated with aggressive growth and resistance to therapy. Tumors rich in ecDNA are among the most difficult to treat. "Understanding what triggers chromothripsis gives us a new way to think about stopping it," said Cleveland. "By targeting N4BP2 or the pathways it activates, we may be able to limit the genomic chaos that allows tumors to adapt, recur and become drug-resistant." Additional coauthors of the study include Alexander Goginashvili, Michael W. Baughn, Stephen Moore, Christopher D. Steele, Amy T. Nguyen, Daniel L. Zhang, Prasad Trivedi, Aarti Malhotra, David Jenkins, Andrew K. Shiau, Yohei Miyake, Tomoyuki Koga, Shunichiro Miki, Frank B. Furnari and Ludmil B. Alexandrov, all at UC San Diego and Jonas Koeppel and Peter J. Campbell of the University of Cambridge and the Wellcome Trust Sanger Institute. This Surprising High-Fat Diet Helped Brains Heal From Stress Before Birth Scientists Finally Solve the Mystery Behind Rare COVID Vaccine Blood Clots Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            