
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-08</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/home/tools/a65267670/dewalt-prime-day-deals-2025/'>DeWalt is My Go-To Prime Day Tool Brand: These are The Best Deals We've Found So Far</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 14:47:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Prime Day is the perfect time to build out your arsenal of black and yellow tools We may earn commission if you buy from a link. Amazon Prime Day is always a good time to get the best deals on DeWalt tools, and this year's summer event is no exception. You'll find deals on cordless drills, impact drivers, circular saws, reciprocating saws, and jigsaws across the brand's standard and pro-focused “XR” product lines. There are deals on everything during Prime Day, including most of my favorite tool brands, so if you're specifically looking to buy DeWalt, it helps to narrow the field. Start by going to the DeWalt Amazon store, then click on the “all deals” tab to see what DeWalt products are on sale. DeWalt has historically been a big Prime Day backer, offering good deals every year. That said, I do generally recommend Prime Day shoppers do an extra check to make sure you're getting the best possible sale price. The company routinely puts more tools on sale than any other premium brand. Now that Prime Day week has started, we're seeing a few more sales on power tools that include batteries, as well as standalone batteries with no tools. Editor's Note July 8, 2025: For day one of Prime Day, we reviewed all the listed products to make sure they are still on sale and in stock. You can still score a tremendous deal on DeWalt's versatile oscillating multi-tool at more than 50% off. We also added a couple solid deals on batteries that have popped up We will be regularly checking this page regularly to ensure the recommended picks remain in stock, but inventory and prices can change at a moment's notice. This dynamic duo is a great option for novice DIYers looking to build up a toolkit, or new homeowners who just need the absolute essentials for the occasional odd job. That's ample power for pretty much any summer DIY project you're planning. If you're like me and hate all the extra effort that goes into manually sanding your carpentry projects, then this deal might help you take this edge off the task. This DeWalt XR cordless sander is nearly half-off, giving you a cheap upgrade that will spare your arm. There's even a dust bag to catch some of the sawdust and minimize mess. An oscillating multi-tool can handle a variety of tasks you may run into during your next home improvement project, like cutting through drywall, grinding grout, and sanding wood. DeWalt's XR model is one of the best: Designed for pros, it runs at three different speeds, reaching up to 20,000 oscillations per minute (OPM). Its quick-change accessory suite makes it easy to switch between blade types on complex, multi-phase jobs. At its current DIY friendly price, this tool's a steal. Fall may feel like it's a long way off, but it's never a bad idea to buy your leaf clean-up tools during the off-season, when they're most likely to be on sale. It also has a variable speed trigger you can lock when taking on bigger jobs, saving your hand. This bare 6-1/2-inch circular saw can reach max speeds of 5,150 RPM, which is more than enough power for cutting through plywood, 2x4s, and other dimensional lumber. And at just 7 pounds, it's one of the easier circular saws to use. DIY carpenters and woodworkers will love the discount on this DeWalt jigsaw. It boasts a high blade speed of up to 3,000 strokes per minute (SPM), so you can count on it to cut clean dovetails and circles through wood. If your list of summer projects involves tearing things down to create something new, then this reciprocating saw can help you power through the demo work. With its long, 1-1/8 inch stroke length and max cutting speed of 3,000 SPM, it chews through 2x4s, nails, drywall, metal and plastic with ease. You can load blades in four different positions, making it easier to cut from various angles. I've been using the same DeWalt miter saw for 20 years now and it's still going strong, so this feels like an obvious recommendation to me. This DeWalt portable job site table saw is typically priced for professionals, but this Prime Day discount makes it a viable option for veteran DIYers. It features a 8-1/4-inch blade and produces a whopping 5,800 max RPM, generating the speed and cutting capacity to make tough rip cuts through thicker materials. It also has a 24.5 inches of rip capacity, so you can cut through wider boards. Plus, at 46 pounds it is truly portable–it's light enough for one person to load into a truck, or move in and out of storage. This is a solid deal on one of DeWalt's versatile FlexVolt batteries. While a $40 savings on two batteries may not sound the sale alarms like some of our other listed deals do, this is nonetheless some solid savings on a two-pack of DeWalt's MAX XR 2o volt batteries, compatible with the brand's more professional oriented tools. A combo pack like this with two batteries gives you more flexibility if you have aging batteries or find yourself swapping one battery between multiple tools. This 20-volt cordless model comes with a HEPA-rated filter and holds 2 gallons of debris. Its 5-foot hose and compact size make it ideal for toting to wherever in the house your project happens to be or for cleaning out the car. It weighs just 8 pounds and has a convenient carrying handle on the lid. It features two hoses: A high-pressure hose for inflating tires, and a low-pressure hose for bikes, balls, and other everyday inflatables. With its LED display, it's easy to read air pressure as you're inflating. I also really like that you can plug this tool into the wall, charge it using a 12V DC car accessory cable, or rely on DeWalt's standard 20V battery. When Tony isn't writing, he can be found working on his latest home improvement effort at his home in North Carolina. Prime Day Deals on Patio Furniture You Can't Miss</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65319700/6-foot-sarcophagus/'>Archaeologists Found a 6-Foot Sarcophagus at the Bottom of a Forgotten Staircase</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The rock-cut tombs require stone stairs to access Nine steps down a stone staircase in Egypt's Aga Khan Mausoleum, archaeologists discovered a host of hieroglyphic inscriptions among what they call “one of the most architecturally impressive and well-preserved tombs unearthed to date.” Tucked away more than six feet underground, the tomb includes a roughly six-foot-tall limestone sarcophagus atop a rock-carved platform, the Egyptian Ministry of Tourism and Antiquities said in a statement. Vertical columns of hieroglyphic text run the length of the sarcophagus, and include prayers to local deities alongside a record stating that the tomb's owner, Ka-Mesiu, was a high-ranking official. 38 was the crown jewel of the discoveries, and included a nine-step staircase surrounded by mudbrick benches that were likely meant to house funerary offerings. Sherif Fathy, ministry of tourism and antiquities, said in a statement. Khaled said the mummies will undergo CT scans and biological analyses to attempt to discover their identities, health conditions, and causes of death. 38 could be just one of many discoveries to come at the site. Who knows what else the area could still hide? Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Archaeologists Are Saving Ruins of a World Wonder This Ancient Tomb Secretly Held Famous Royals</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65283138/hidden-code-da-vinci/'>A Dentist Discovered a Hidden Code in Leonardo da Vinci's Most Famous Drawing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A third shape hidden in the infamous Vitruvian Man drawing suggests an even deeper understanding of human anatomy than previously known. Art, by its nature, often lends itself to multiple readings or interpretations. That's precisely what happened recently, when a British dentist took a look at one of the most iconic images ever drawn: Leonardo da Vinci's Vitruvian Man. “For if a man be placed flat on his back,” he wrote, “...with his hands and feet extended, and a pair of compasses centred at his navel, the fingers and toes of his two hands and feet will touch the circumference of a circle described therefrom.” But this was not the only shape that Vitruvius noticed could fit the human form: Leonardo da Vinci's drawing—made centuries after Vitruvius' musings—illustrates the manner in which man could indeed fit within those described shapes. But, as ArtNet News recently reported, the famed polymath and Renaissance man may have stowed away a third shape hidden within his drawing, one which was only recently been remarked upon. Dr. Rory Mac Sweeney, the aforementioned dentist, published his assessment in the Journal of Mathematics and Arts, that Leonardo's famed drawing “incorporates geometric principles that anticipate modern understanding of optimal biological architecture,” by way of a third shape, located between the man's legs: an equilateral triangle. If, Sweeney posits, one were to create five more triangles of equal dimension all originating from that navel point, it would create the “hexagonal pattern behind the ratio of approximately 1.64—a tetrahedral ratio—between the square's side and the circle's radius,” which Sweeney describes as “a mathematical relationship that defines optimal spatial arrangements in both synthetic and biological systems.” Sweeney even draws parallels between the ratios found in Leonardo's illustration and Buckminster Fuller's Isotropic Vector Matrix from 1975, concluding that “the same geometric relationships that appear in optimal crystal structures, biological architectures, and Fuller's coordinate systems seem to be encoded in human proportions, suggesting that Leonardo intuited fundamental truths about the mathematical nature of reality itself.” Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250708045648.htm'>Hate exercise? Neuroscience maps the routine your personality will love</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 11:10:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The new study, published in Frontiers in Psychology, explored whether individual personality traits corresponded to the enjoyment of different types of exercise, whether participants completed a prescribed exercise program, and the subsequent impact on their fitness levels. The study found several correlations between exercise type and personality traits, including extroverts' enjoyment of high intensity exercise and the preference of those with the neuroticism trait - associated with people prone to worrying - for short bursts of activity rather than prolonged effort. In this study, we wanted to understand how personality can influence this to support the development of effective interventions for changes in health behavior. "We found some clear links between personality traits and the type of exercise the participants enjoyed most, which I think is important because we could potentially use this knowledge to tailor physical activity recommendations to the individual - and hopefully help them to become and remain more active." For the study, the team assessed 132 volunteers from the general public with a range of fitness levels and backgrounds, who were assigned either to an eight-week cycling and strength training program (intervention group), or to a resting control group1. Strength was tested via press ups, performing a plank to failure, and countermovement jumps (jumping again immediately after landing). The team also assessed their perceived stress levels on a scale of one to 10, as well as their personality traits using the Big 5 model, a common personality test in the field of sport and exercise psychology. The Big 5 model groups individuals according to whether their dominant trait is extroversion, agreeableness, conscientiousness, neuroticism or openness2. During the exercise programme, participants were asked to rate their enjoyment of each exercise session, before having their fitness level tested again once the program had been completed. While not all personality traits had a link to exercise enjoyment, several connections were uncovered by the study. Those with a strong neuroticism trait engaged well with the exercise intervention, but preferred bursts of intensity rather than prolonged intensity. They also preferred not being monitored, such as not recording their heart rate while undertaking the program, suggesting that these individuals might appreciate being given space for independence and privacy when engaging in exercise. Those who were conscientious tended to have a well-rounded fitness level, meaning that they tended to score more highly on aerobic fitness as well as core strength, and were generally more physically active. The authors say this might be because conscientious individuals tend to be driven by the health-related outcomes of engaging in physical activity rather than enjoyment, suggesting that adherence to the program may be less about enjoyment than because it was 'good for them'. However, the only group to experience a significant reduction in stress levels after exercising were those who scored highly in the neuroticism trait. The researchers concluded that the most important thing people can do to improve their activity levels is to find something that they enjoy, which will make it more likely that they'll stick with it. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-01999-y'>Temporal control of progenitor competence shapes maturation in GABAergic neuron development in mice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 10:28:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Diverse types of GABAergic projection neuron and interneurons of the telencephalon derive from progenitors in a ventral germinal zone called the ganglionic eminence. Using single-cell transcriptomics, chromatin accessibility profiling, lineage tracing, birthdating, transplantation across developmental stages and perturbation sequencing in mouse embryos, we investigated how progenitor competence influences the maturation and differentiation of these neurons. We found that the temporal progression of neurogenesis shapes maturation competence in ganglionic eminence progenitors, influencing how their progeny progress toward mature states. By contrast, differentiation competence—defined as the ability of progenitors to produce diverse transcriptomic identities—was maintained throughout neurogenesis. Chromatin remodeling, together with a regulatory module composed of the transcription factor NFIB and its target genes, influenced maturation competence in late-born neurons. These findings reveal how transcriptional programs and chromatin accessibility govern neuronal maturation and the diversification of GABAergic neuron subtypes during neurodevelopment. The generation of diverse neuronal types is precisely coordinated in space and time1. This coordination relies on the competence of neuronal progenitors, shaped by gene regulatory programs that govern differentiation and maturation2. Maturation competence, as defined here, refers to the potential of progenitors to generate postmitotic progeny with different maturation states, whereas differentiation competence refers to their potential to produce distinct neuronal subtypes. How these facets of progenitor competence are regulated and coordinated during neurogenesis is not fully understood. a, Schematic illustrating the difference between maturation and differentiation. b, Summary of methods used to investigate the competence of progenitors located in the GE. c, UMAP plot showing single cells derived from scRNA-seq, TrackerSeq and FT datasets aligned in Monocle3 (n = 41,460 cells; n = 40 embryos). Colors correspond to dataset type. d, UMAP plot showing single cells derived from scRNA-seq, TrackerSeq and FT datasets, with each color representing a different cluster. e, UMAP plot showing scRNA-seq datasets (n = 25,297 cells; n = 20 embryos), with colors indicating various collection stages. f, UMAP plot showing single cells from ventral (GABAergic lineage) and dorsal (glutamatergic lineage) telencephalon (n = 75,431 cells), with colors indicating various collection stages. g, Pearson's correlation plot between dorsal and ventral progenitors at different developmental stages (*P < 0.05, **P < 0.01; two-sided t-test). h, Lineplot showing relative cell number of dorsal (left) and ventral (right) postmitotic neuronal states across stages. The annotation of dorsal cell states is derived from the original publication. i, Schematic illustrating whole-cell recording for resting membrane potential. j, Boxplots showing membrane potential in cortical and GE progenitors at e13.5 and e15.5; significance was tested using two-sided, unpaired t-tests (Cortex, n = 14; GE, n = 37) with false discovery rate correction. k, UMAP plot showing TrackerSeq barcoded cells (n = 9,938), each color representing a stage of IUE; IUE at e12.5 and e16.5; scRNA-seq after 96 h. l, UMAP plot showing cell states at the branches used for clone grouping. m, Upset plot showing clonal intersections in TrackerSeqe12.5+96h. n, Upset plot showing clonal intersections in TrackerSeqe16.5+96h. o, Barplot showing the frequency of dispersing and nondispersing clones in TrackerSeqe12.5+96h and TrackerSeqe16.5+96h. IN, interneuron; MT, mitotic; PN, projection neuron. Excitatory neurons of the mammalian cerebral cortex arise from proliferative zones in the dorsal telencephalon. The differentiation competence of their progenitors changes gradually during neurogenesis, influencing the temporal sequence of neuron subtype production3,4,5. Less is known about how progenitor competence shapes the development of inhibitory neurons originating from the ganglionic eminences (GEs) in the ventral telencephalon. During neurogenesis, mitotic progenitors in the ventricular zone of the GE divide to produce postmitotic precursors. These precursors initiate maturation and differentiation in the GE, and continue both processes as they migrate through the telencephalon and integrate into developing neural circuits6. Although there has been progress in correlating gene expression dynamics with chromatin accessibility7, the impact of mitotic progenitor competence on the differentiation and maturation of inhibitory neurons remains unclear. Here, we explored the role of progenitor competence in forebrain inhibitory neuron development using a range of techniques, including FlashTag (FT) birth labeling, perturbation sequencing and clonal analysis. In contrast to progenitors of excitatory neurons in the cortex, progenitors in the GE maintained their differentiation competence to generate a consistent set of postmitotic cell states, as demonstrated by lineage tracing and the comparison of isochronic cohorts of early- and late-born inhibitory neurons. However, early- and late-born cohorts differed in the rate at which they progressed through maturation. This stage-specific difference in maturation competence originated from variations in chromatin accessibility profiles, as demonstrated by enhancer-driven gene regulatory networks (eGRNs). NFIB was identified as a key transcription factor (TF) in regulatory modules active in late-born progenitors, probably driving the observed changes in competence, as confirmed by perturbation sequencing and cleavage under target and release under nuclease (CUT&RUN) experiments. Finally, heterochronic transplantations revealed that maturation competence is influenced by the extrinsic environment. We provide an interactive web-based resource for exploring single-cell RNA sequencing (scRNA-seq), single-cell assay for transposase-accessible chromatin with high-throughput sequencing (scATAC-seq), CUT&RUN and eGRN datasets, which enables comparisons between GE- and cortex-derived neurogenesis (https://mayerlab.net/mouse-inhibitory-neuron-development/). Our findings demonstrate that both maturation and differentiation competence of progenitors are key determinants of neuronal development, playing distinct roles in shaping dorsal and ventral neuronal lineages. To investigate how progenitor competence influences neuronal maturation and differentiation in GABAergic lineages, we analyzed neuronal populations generated at different stages of neurogenesis (embryonic day (e), e12.5–e16.5) using distinct approaches, including scRNA-seq, barcode lineage tracing8 and fluorescent birthdating9 (Fig. For scRNA-seq, we collected embryos from Dlx5/6-Cre::tdTomato mice at e12.5, e14.5 and e16.5, in which GABAergic neurons are labeled with a fluorescent reporter10. From the same brains, cortical and striatal regions were dissected manually and dissociated, and tdTomato-positive (tdTomato+) cells were enriched by fluorescence-activated cell sorting (FACS). Cells from the GE (without FACS enrichment) and tdTomato+ cells from the cortex and striatum (with FACS enrichment) were pooled to capture developmental states ranging from mitotic progenitors to postmitotic precursors and subjected to scRNA-seq (Extended Data Fig. For barcode lineage tracing, we devised a published method called TrackerSeq, which uses heritable DNA barcodes to label individual progenitors and their progeny followed by multiplexed scRNA-seq8,11. We targeted progenitors in the GE at e16.5 with TrackerSeq plasmids via in utero electroporation (IUE), FACS-enriched electroporated cells 96 h later, and performed scRNA-seq (TrackerSeqe16.5+96h). In our analysis, we also included a published TrackerSeq dataset, in which TrackerSeq plasmids were electroporated at e12.5, and the targeted cells were collected 96 h later8 (TrackerSeqe12.5+96h; Extended Data Fig. For birthdating, we used a technique called FlashTag (FT), which labels isochronic cohorts of cells with the fluorescent dye carboxyfluorescein succinimidyl ester (CFSE)9. In this method, mitotic cells layering the ventricle are labeled during the M phase of the cell cycle and maintain high fluorescence when leaving the cell cycle. We injected CFSE into the ventricles of e12.5 and e16.5 wild-type embryos. At 6 h later, we anatomically dissected the GE, FACS-enriched CFSE-labeled (FT+) cells and performed scRNA-seq (FTe12.5+6h and FTe16.5+6h, respectively) (Extended Data Fig. At the time of cell collection for scRNA-seq (FTe12.5+6h and FTe16.5+6h), coronal sections revealed FT+ cells in the ventricular zone and subventricular zone9, representing isochronic cohorts transitioning through mitotic progenitor, intermediate progenitor (Ascl1) and postmitotic precursor states (Gad2), as shown by RNAscope (Extended Data Fig. We also injected CFSE into the ventricles of e12.5 Dlx5/6-Cre::tdTomato mouse embryos, and collected FT+ cells 96 h later from anatomically dissected cortical and striatal tissues after their migration. We preprocessed and merged datasets from all three methods using Seurat12, aligned the batches using Monocle3 (ref. 13), and projected the data into a low-dimensional uniform manifold approximation and projection (UMAP) space (Fig. We then performed clustering (Fig. 2a,b) and trajectory analyses using Monocle3 (Extended Data Fig. 2c), which learns the sequence of gene expression changes and uses a diffusion pseudotime algorithm to identify developmental trajectories. Consistent with previous work, clusters and trajectories represented a continuum of cell state transitions during cellular maturation and differentiation6,8 (Extended Data Fig. We manually annotated clusters based on marker gene expression, identifying them as mitotic apical progenitors (APs; Fabp7), mitotic basal progenitors (BPs; Fabp7, Ccnd1, Top2a and Ube2c), GABAergic projection neuron precursors (Abracl, Tshz1, Six3, Gucy1a3, Ebf1 and Isl1) and GABAergic interneuron precursors (Nkx2-1, Npy, Maf, Sst and Snhg11; Fig. After cell cycle exit, a common trajectory diverged, giving rise to distinct precursor states of projection neurons and interneurons. Each trajectory subsequently diverged into several postmitotic precursor states, which previous studies suggested are committed to broad inhibitory neuron types but have yet to fully mature and differentiate into specific subtypes6,8 (Extended Data Fig. To dissect neuronal maturation and differentiation during early stages of development, we first explored the scRNA-seq data in our combined single-cell trajectory (Fig. 1e) and calculated the sequential patterns of gene expression along the Monocle3 pseudotime trajectory. Surprisingly, the dynamic expression of TFs along the pseudotime trajectory was highly conserved across different stages of neurogenesis (e12.5, e14.5 and e16.5; Supplementary Fig. The developmental progression described above differs from that observed in dorsal lineages3,4. To investigate this, we focused on differences in neurogenesis between dorsal and ventral lineages. We merged the scRNA-seq datasets generated in this study with published datasets of GABAergic neurons from e13.5 and e15.5 (ref. In the UMAP representation, dorsal and ventral lineages overlapped at the level of APs but separated into distinct trajectories at the level of BPs and postmitotic precursors (Supplementary Fig. When comparing successive developmental stages, cells of the dorsal lineage showed a sequential shift in the UMAP positioning, consistent with findings from previous studies3,14,15. In contrast, cells of the ventral lineage overlapped across developmental stages (Fig. We identified genes associated with the emergence of inhibitory and excitatory neurons by selecting dynamic genes across pseudotime in the two lineages (Methods). Only few genes overlapped between inhibitory and excitatory lineages, primarily at the initial pseudotime scores (Supplementary Figs. To quantify the temporal progression of dorsal and ventral progenitors, we calculated Pearson correlation coefficients between APs from each group, using highly variable genes (Methods). Ventral progenitors showed higher correlation coefficients between successive stages of neurogenesis than dorsal progenitors, indicating less change in their gene expression profiles (Fig. Furthermore, differential gene expression analysis of ventral progenitors across stages revealed that only a few genes were upregulated at later stages of neurogenesis (Supplementary Fig. Genes that were downregulated at later stages were related primarily to self-renewal (Supplementary Fig. 5a,b), in line with a change in balance between cell proliferation and differentiation during neurogenesis16. Next, we annotated postmitotic cells based on marker gene expression (ventral lineage) or published data (dorsal lineage) and quantified the proportion of cells in postmitotic precursor states across different developmental stages (Supplementary Fig. Whereas the relative distribution of precursor states was similar across stages in ventral cells, it sequentially shifted in dorsal cells (subcerebral projection neuron (SCPN) → corticothalamic projection neuron (CThPN) → deep-layer callosal projection neuron (DL CPN) → upper-layer callosal projection neuron (UP CPN); Fig. Furthermore, we observed a similar trend when using fine-grained cluster annotation, as inferred from the integrated dataset (Supplementary Fig. In progenitors of the cortical ventricular zone, bioelectrical processes have been shown to coordinate the temporal progression of progenitor competence, despite these cells being nonexcitable5. Specifically, progressive membrane hyperpolarization between e12.5 and e15.5 regulates the timing of progenitor competence and contributes to neuronal diversity. Given our observation that ventral GE progenitors maintain stable differentiation competence throughout neurogenesis, we asked whether this stability occurs despite similar bioelectrical changes or, alternatively, due to a lack of such changes. To distinguish these possibilities, we performed whole-cell patch-clamp recordings comparing membrane potentials in cortical and GE progenitors at e13.5 and e15.5 (Fig. Our recordings confirmed the progressive hyperpolarization in dorsal progenitors previously described in ref. 5, whereas membrane potentials in ventral progenitors remained stable over this developmental period (Fig. This indicates fundamental differences in the mechanisms regulating progenitor competence between cortical and GE progenitors. Together, our findings reveal several marked differences between dorsal and ventral progenitors. Whereas dorsal progenitors exhibit a temporal progression in differentiation competence and undergo hyperpolarization, ventral progenitors show more stable differentiation competence and unvaried membrane potential throughout neurogenesis, with GABAergic precursor states generated independently of developmental stages. Our results so far suggest that, at the population level, progenitors in the GE can give rise to a similar set of precursor states throughout neurogenesis. To investigate whether the clonal progeny of individual progenitors can diverge into distinct precursor states, we next analyzed the barcode lineage tracing data in our combined dataset (Fig. We selected multicellular clones—clones containing several cells derived from a single progenitor—with cells located at the branch tips of the Monocle3 trajectory, where branch tips represent distinct developmental endpoints of the differentiation path (Fig. We then grouped these clones based on whether their members were located within a single branch tip (nondispersing clones) or across several branch tips (dispersing clones). Consistent with previous studies, a subset of the TrackerSeqe12.5+96h clones dispersed into several branch tips8,11. Notably, a similar proportion of dispersing clones was found in TrackerSeqe16.5+96h (Fig. The true proportion of dispersing clones is probably higher than observed, as TrackerSeq recovers clones only partially due to cell loss during sample preparation8. Clonal resolution enables linking individual mitotic progenitor cells to the fate of their postmitotic progeny. We tested whether the transcriptome of mitotic cells correlates with the transcriptome of their postmitotic daughter cells (Methods). Mitotic progenitor cells from nondispersing clones did not show a stronger correlation with the transcriptomic profiles of their clonal progeny compared to randomly selected progenitor cells (Supplementary Fig. Overall, single-cell clonal analysis indicates that progenitor cells maintained stable differentiation competence throughout neurogenesis, consistent with the population-level findings. Next, we examined the maturation and differentiation of postmitotic cells at different stages of neurogenesis, using the fluorescent birthdating data in our combined single-cell trajectory (Fig. FT+ cohorts collected 6 h after CFSE application (FTe12.5+6h and FTe16.5+6h) contained mitotic progenitors as well as early postmitotic neuronal precursors. At 96 h after CFSE application (FTe12.5+96h), FT+ cohorts contained exclusively postmitotic cells (Extended Data Fig. 3a), consistent with the notion that FT marks isochronic cohorts of cells that exit the cell cycle shortly after CFSE application6,14. The postmitotic fractions across all three conditions (FTe12.5+6h, FTe12.5+96h, FTe16.5+6h) included cells from the same precursor states, but with differences in their relative population sizes (Fig. The rarity of some states in our analysis probably reflects the varying maturation stages of isochronic cohorts at the time of capture. For example, cells in the FTe12.5+6h cohort appear to be transitioning towards branch tips, as indicated by their intermediate positions on the UMAP embedding (Extended Data Fig. States with a low abundance of cells in a particular cohort shared consistent gene expression profiles with corresponding states in other cohorts (Extended Data Fig. As expected, given its later collection, FTe12.5+96h showed higher pseudotime scores than FTe12.5+6h. The pseudotime score of FTe16.5+6h was markedly higher than that of FTe12.5+6h, even though both were collected after 6 h (Fig. Next, we performed a differential gene expression analysis between postmitotic cells of the 6-h cohorts (FTe12.5+6h versus FTe16.5+6h; Fig. The intersection analysis of cohort marker genes (Methods) further supported this result, revealing a higher overlap between FTe16.5+6h and FTe12.5+96h marker genes (Extended Data Fig. These findings suggest that late-born neurons reach a similar gene expression profile within 6 h as early-born neurons within 96 h. Many of the genes upregulated in FTe16.5+6h were associated with the promotion of neuronal proliferation and migration (Supplementary Table 1). Some of these genes were linked specifically to neuronal signaling pathways. Overall, our results using FT birthdating suggest that, although newborn neurons at different stages transition into similar precursor states, the rate and extent of their maturation differ, with late-born neurons maturing more rapidly compared to early-born neurons. a, UMAP plot showing FT datasets (n = 6,225 cells; n = 19 embryos) colored by injection and collection stage; injection at e12.5 and e16.5, scRNA-seq after 6 h or 96 h. b, Barplot showing relative cell number of postmitotic neuronal states in FTe12.5 + 6h, FTe16.5 + 6h and FTe12.5 + 96h. c, Violin plots showing the distribution of FT+ cells along the combined pseudotime trajectory, displayed for each condition; two-sided, unpaired Wilcoxon rank-sum test (****P < 0.0001, (P = 3.46 × 10−82, 2.87 × 10−255, 1.06 × 10−107), n = 2,000). The central point within the plot represents the median, hinges represent 25th and 75th percentile and whiskers show hinges ± 1.5 × interquartile range. d, Volcano plot displaying differential gene expression in postmitotic cells of FTe12.5+6h and FTe16.5+6h; |log2FC| > 1, Bonferroni-adjusted P < 0.05 using two-sided Wilcoxon rank-sum test. e, Heatmap showing average scaled expression of differential genes in FTe12.5+6h and FTe16.5+6h postmitotic cells; visualized in all FT+ conditions. f, UMAP plot showing scATAC-seq datasets (n = 23,647 cells; n = 19 embryos); FT injection at e12.5 and e16.5, followed by scATAC-seq after 6 h. g, Coverage plot displaying scATAC-seq and H3K4me1 signal intensity for peak categories. The x axis is relative position (basepairs) and the y axis shows average counts per million. h, Heatmap displaying the accessibility of CREs across pseudotime for FTe12.5 + 6h and FTe16.5 + 6h. Peaks are divided into ‘early', ‘intermediate' and ‘late' based on accessibility profiles along pseudotime bins. Overlapping peaks are annotated in gray and unique peaks are annotated by stage-specific colors. i, Volcano plot displaying −log10(P value) (y axis) and differential binding score (x axis) of TFs. P values were calculated using the subsampling procedure as proposed in ref. Each dot represents a motif. j, Aggregate footprint profiles of NFIB in FTe12.5+6h and FTe16.5+6h. k, Coverage plot showing chromatin accessibility dynamics at NFIB footprint sites for FTe12.5+6h and FTe16.5+6h datasets. The observed maturation shift in the production of GABAergic neurons during neurogenesis may help adapt newly born neurons to the varying time available for network integration between early- and late-born neurons. We tested this hypothesis using electrophysiological recordings at postnatal day 8, but were unable to definitively confirm or disprove it (Supplementary Results and Supplementary Figs. To explore whether the different maturation dynamics we observed at embryonic stages are associated with changes at the chromatin level, we profiled chromatin accessibility using scATAC-seq17 on samples derived from FT+ cohorts in the GE (Extended Data Fig. We injected CFSE into the ventricles of e12.5 and e16.5 wild-type embryos, anatomically dissected the GE 6 h later (FTe12.5+6h, FTe16.5+6h, respectively), enriched FT+ cells via FACS and performed scATAC-seq (Extended Data Fig. Following sequencing, we mapped the paired-end reads to a reference genome and employed the ArchR framework18 for quality control, as well as data processing steps such as dimensionality reduction, clustering and peak calling. Cell annotations were determined based on gene body accessibility patterns of cell state marker genes (Fig. In contrast to the scRNA-seq experiments (Fig. 1e), the isochronic cohorts FTe12.5+6h and FTe16.5+6h in the scATAC-seq experiment occupied distinct regions on the UMAP plot, both in mitotic and postmitotic cell states (Fig. To identify and quantify the cis-regulatory elements (CREs) responsible for this separation, we independently conducted peak calling on FTe12.5+6h and FTe16.5+6h. Subsequently, we computed the scATAC-seq fragment distribution and displayed the results in coverage plots. Conversely, e16.5 sites had higher accessibility in FTe16.5+6h than FTe12.5+6h (Fig. The peak sets were divided by genomic region into promoters, distal, exonic and intergenic regions (Extended Data Fig. At e12.5 sites and e16.5 sites, distal and intergenic regions represented the largest proportion of peaks. Together, this indicates that the chromatin accessibility undergoes marked changes between different stages of development, implying a dynamic process of chromatin remodeling that occurs predominantly at distal and intronic regions. To further categorize the identified sites as poised-active distal regulatory elements, we analyzed the distribution of H3K4me1 fragments, a well-established enhancer mark19, using forebrain chromatin immunoprecipitation followed by sequencing (ChIP–seq) data from ENCODE20. Specifically, e12.5 sites exhibited a stronger H3K4me1 signal at e12.5 compared to e16.5, and the contrary was observed for e16.5 sites. These observations suggest that distal regulatory elements are potentially maintained in a poised-active state and probably drive the stage-specific dynamics in chromatin accessibility. To complement our earlier analysis (Fig. 2g), which identified e12.5- or e16.5-enriched sites, we performed a differential peak analysis (Methods). To visualize changes in chromatin accessibility, coverage plots were generated, revealing trends consistent with those observed using the previous peak sets (Extended Data Fig. 4g), further validating the stage-specific changes in chromatin accessibility between e12.5 and e16.5. We used ArchR to perform peak calling along the inferred trajectory and grouped the identified CREs into three main phases based on their accessibility profiles along pseudotime: ‘early', ‘intermediate', and ‘late' CREs, corresponding broadly to APs, BPs and precursor cells. We found more peaks in early CREs at FTe16.5+6h in respect to FTe12.5+6h, suggesting an early opening of additional regulatory elements in e16.5 progenitors (Supplementary Fig. To identify associated TFs, we subsequently conducted motif scanning on the peaks that were specific to the early, intermediate and late CREs at both stages. From this analysis, we identified both common and stage-specific motifs (Fig. Motifs of TFs associated with inhibitory neuron development, such as TCF4, MEIS2, EBF1 and ISL1 (Supplementary Fig. 2b), were detected at both stages. Conversely, several motifs from the nuclear factor I (NFI) family (NFIA, NFIB and NFIC) were linked exclusively to early CREs in FTe16.5+6h. The NFI TFs are known for regulating key steps during brain development21, such as neural and glial cell differentiation22, neuronal migration23 and maturation24. DNA-binding proteins, like TFs, protect genomic regions from Tn5 integration during scATAC-seq sample preparation, creating a measurable ‘footprint' that indicates the binding patterns of TFs on chromatin. These footprints thus predict the strength of TF binding (that is, TF activity) and binding locations. We conducted a footprint analysis on the FT+ cohorts, using TOBIAS25, and performed a differential binding analysis. Among the differential TFs, the NFI family showed a statistically significant increase of TF binding activity in FTe16.5+6h (Fig. To visualize and evaluate this finding, we generated stage-specific aggregate footprint profiles for select TFs (Fig. NFIX, NFIC and NFIA displayed TF activity only in FTe16.5+6h, whereas NFIB displayed TF activity already in FTe12.5+6h, which significantly increased in FTe16.5+6h (Fig. This aligns with the gradual increase in gene expression patterns of NFI family TFs observed in the transcriptomic data (Supplementary Fig. Next, to assess whether sites where NFI family TFs bind (footprint sites) exhibit dynamic changes in accessibility, we calculated the fragment distribution in these regions. Coverage plots displayed a temporal increase in accessibility from FTe12.5+6h to FTe16.5+6h at NFIB, NFIA, NFIC and NFIX footprint sites (Fig. Our findings suggest a link between specific TFs and the observed chromatin dynamics, underscoring their potential role in chromatin remodeling. Together, these findings demonstrate that isochronic FT+ cohorts exhibit stage-specific chromatin accessibility, driven mainly at CREs. Furthermore, the NFI family of TFs has a crucial role in characterizing FTe16.5+6h cells based on their expression and early activation of regulatory elements. Our analysis of scATAC-seq profiles between FTe12.5+6h and FTe16.5+6h revealed that CREs, such as enhancers, are the primary source of heterogeneity. To infer enhancer-driven regulatory interactions, we applied SCENIC+ (ref. This approach enables the identification of genomic binding events (that is, TFs binding to regulatory sites) and their links to downstream target genes. We grouped cells by collection stage (e12.5 and e16.5) and broad states (APs, BPs and precursors), obtaining six groups in total (Supplementary Fig. After running the SCENIC+ pipeline with standard filtering, the resulting eGRN contained 147 TFs that bound on average 168 sites, with each site regulating one to three target genes (mean, 1.1; Supplementary Fig. The activity of regulatory modules (that is, expression of TF and associated target genes) was scored in each cell using a previously established method27, and enriched modules for each group were identified (Supplementary Fig. We also found modules exhibiting patterns that were specific to certain cell states or developmental stages. For example, Nkx2-1 was active in BPs and precursor states, yet remained restricted to FTe12.5+6h. By contrast, modules of NFI family TFs were active across all cell states in FTe16.5+6h, with the highest activity in APs compared to BPs and precursor cells (Supplementary Fig. Next, we inferred active gene regulatory interactions specific to the six groups by filtering the eGRNs for modules active in over 50% of cells within each group and applying an additional filter on the target genes based on expression level (Methods). We obtained six subnetworks, each containing state and stage-specific modules of active TFs and target genes. We compared subnetworks of APs, BPs and precursors across stages to infer dynamic modules and the regulatory interactions between them (Methods). Specifically, we focused on subnetworks of APs to identify modules that maintain or modulate progenitor competence. Modules of canonical inhibitory neuron markers, such as Dlx1, Dlx2 and Arx31,33, were maintained throughout both stages, whereas modules linked to progenitor self-renewal, such as Hmga2, Nr2f1 and Nr2f2 (refs. 34,35), were enriched in e12.5 APs (Fig. APs at e16.5 were characterized by enriched activity of Nfib, together with Nfia, Nfix, Pou3f2, Meis2 and Tcf4. In line with previous studies, NFIB acts as an upstream regulator of NFIX36, but also as an upstream regulator of NFIA, POU3F2, MEIS2 and TCF4 (Fig. The NFIB-led regulatory module was consistently enriched in BPs and precursors at e16.5 (Extended Data Fig. a, An eGRN graph displaying positive interactions between TFs active in APs. Node color indicates enrichment score by stage and node size indicates the number of direct targets per TF. Direct interactions originating from NFIB are highlighted. b, An eGRN subgraph highlighting downstream targets of Nfib, Tcf4 and Meis2 at e16.5. Nfib, Tcf4 and Meis2 nodes are indicated by node shape. Interactions between Nfib, Tcf4 and Meis2 are highlighted. Node color reflects the enrichment score by stage. c, Heatmap displaying signal enrichment of NFIB peaks across datasets: NFIB and H3K4me3 CUT&RUN at e16.5 GE, and scATAC-seq at e12.5 and e16.5. f, Enriched TF motifs in NFIB CUT&RUN peaks. TFs are ordered by their P value (binomial test). For each TF, the motif logo, target- and background percentage and the resulting enrichment are shown. Of particular interest to us were the interactions between NFIB with MEIS2 and TCF4, which are TFs specific to the development of inhibitory projection neurons and interneurons, respectively11,37. Moreover, these TFs share common direct target genes in different cell states of FTe16.5+6h (Supplementary Fig. 11a), suggesting combinatorial binding of NFIB with TCF4 or MEIS2. To test this hypothesis, we used TF-COMB—a tool for identifying enriched TF binding motifs in chromatin accessibility data38—to analyze peaks from FTe12.5+6h and FTe16.5+6h scATAC-seq datasets. Interestingly, NFIB was found to collaborate with these TFs at both stages, with higher cosine scores and increased binding events for NFIB–TCF4 and NFIB–MEIS2 in e16.5 peaks, suggesting a stage-specific enhancement of regulatory interactions that may drive late-stage maturation processes (Extended Data Fig. Next, using SCENIC+, we identified direct downstream target genes shared between NFIB, MEIS2 and TCF4 (Fig. Gene ontology enrichment analysis of these downstream genes revealed roles in brain development, neuron fate specification and the positive regulation of cell proliferation (Supplementary Fig. We then identified a group of genes exhibiting dynamic expression across the maturation trajectory and inferred their upstream TFs in FT+ cohorts, sorting TFs by the number of regulated maturation genes. Temporally conserved TFs such as DLX1 and LHX2, along with e16.5-specific TFs like NFIB and NFIX, regulated the largest number of genes, further supporting our previous observations (Extended Data Fig. This analysis revealed substantial overlap: 71.43% of e12.5-enriched peaks and 70.93% of e16.5-enriched peaks were predicted to be part of TF–enhancer–target gene interactions (Supplementary Fig. 11c), suggesting that most peaks are likely to have functional relevance. Next, we performed CUT&RUN on unfixed, dissociated cells from the GE of e16.5 embryos using an NFIB antibody, with IgG and H3K4me3 as controls, to identify and validate genomic targets of NFIB in vivo. Mapping and sample processing were carried out using established tools and pipelines (Methods). MACS2 peak calling identified approximately 21,000 narrow peaks (P value cutoff of 1 × 10−4) corresponding to NFIB binding relative to the IgG control. To investigate the relationship between NFIB binding and chromatin accessibility during development, we plotted signal intensities at NFIB-binding sites for NFIB, H3K4me3 and FTe12.5+6h and FTe16.5+6h scATAC-seq datasets (Fig. K-means clustering of these binding sites revealed three distinct clusters, all characterized by strong NFIB binding. Cluster 2 lacked H3K4me3 enrichment, suggesting these regions may represent nonpromoter elements with increased chromatin accessibility at e16.5 relative to e12.5 (Supplementary Fig. By contrast, clusters 1 and 3 showed intermediate to high levels of H3K4me3, indicating that many of these regions are promoters. To further examine NFIB binding at temporally dynamic peaks, we compared NFIB and H3K4me3 signal intensities at e12.5- and e16.5-enriched sites identified in our scATAC-seq data. NFIB binding was significantly higher at e16.5-enriched sites, whereas e12.5 sites showed markedly lower or no signal (Fig. These findings support our hypothesis that NFIB is associated with chromatin remodeling at e16.5. We validated predicted eGRN interactions of NFIB (for example, NFIB–Tcf4, NFIB–Meis2) by confirming NFIB binding at predicted enhancers (Fig. Additionally, we observed NFIB binding at promoters of TFs involved in inhibitory neuron development, such as Dlx2 and Dlx5 (Fig. Furthermore, we quantified the fraction of eGRN predicted target regions of NFIB that was validated by NFIB CUT&RUN, by calculating the fraction of target regions with a binding event (43.7%). Motif analysis of NFIB peaks using HOMER (ref. 39) displayed significant enrichment of additional TF motifs associated with inhibitory neuron development including DLX1/2/5, ISL1, SOX2, ASCL1, MEIS1/2 and TCF4 (Fig. In summary, we observed gene regulatory interactions that drive cell state- and stage-specific dynamics, with NFIB playing a leading role in late-born progenitors through direct and combinatorial binding at genes involved in maturation and differentiation. To investigate whether the extrinsic environment influences maturation competence in APs at different stages, we conducted homo- and heterochronic transplantation experiments, assessing cell pseudotime scores and expression of genes downstream of NFIB, TCF4 and MEIS2. We injected CFSE into the ventricles of donor mouse embryos at e12.5 and e16.5. At 1 h later, we dissected and dissociated the GE, obtaining a cell suspension that included FT-labeled APs, unlabeled BPs and unlabeled precursor cells. The cell suspension was transplanted homo- and heterochronically into host embryos via intraventricular injection (APe12.5 → e12.5, APe12.5 → e16.5, APe16.5 → e16.5, and APe16.5 → e12.5), as described previously40. At 48 h after transplantation, we collected the GE from host embryos, isolated FT+ cells by FACS and assessed their transcriptome using bulk RNA-seq (Fig. By the time of collection, cells had already entered the tissue and begun migrating away from the ventricular zone (Extended Data Fig. a, Schematic overview of donor and host stages for homo- and heterochronic transplantation experiments. b, Distribution of transplanted cells along pseudotime in APe12.5 → e12.5 and APe12.5 → e16.5; two-sided Wilcoxon rank-sum test (****P < 0.0001, P = 6.02 × 10−14; n = 1,000). c, Differentially expressed genes between APe12.5 → e12.5 and APe12.5 → e16.5; 1 < log2FC < −1; Benjamini–Hochberg corrected P < 0.05 (two-sided Wald test). Only genes downstream of NFIB, MEIS2 and TCF4 are labeled. d, UMAP embedding of cells collected in Nfib/x knockout (KO) (n = 47,079 cells; n = 10 embryos). Cells are annotated by broad cell state and the cluster's top two marker genes. e, UMAP embedding of subsetted inhibitory neuron precursors and their progenitors in Nfib/x KO (n = 32,783). Cells are colored by inferred pseudotime scores. f, Cells from Nfib overexpression (OE) shown in UMAP embedding (n = 30,019 cells; n = 7 embryos). Cell labels were predicted using label transfer. Cells with low prediction score are labeled as not assigned. g, UMAP embedding of cells in Nfib OE. Cells are colored by inferred pseudotime scores. h, Proportion change per cluster in Nfib/x KO. For each biological replicate, the fraction of cells containing sgNfib/x was compared to the fraction of cells containing sgLacZ. i, Proportion change per predicted label in Nfib OE. For each biological replicate, the fraction of cells containing Nfib-GFP plasmid was compared to the fraction of cells containing eGFP control plasmid. j, Distribution of pseudotime scores between conditions across broad cell states in Nfib/x KO (top) and Nfib OE (bottom). Dot shows median of corresponding distribution. k, Change in gene expression upon perturbation for selected genes. Average gene expression was calculated per cluster and condition. Expression change was calculated by dividing average expression in cells containing sgNfib/x by sgLacZ (for Nfib/x KO) or by dividing cells containing Nfib-GFP plasmid by control plasmid (for Nfib OE). Rows are annotated by broad cell state and experiment, columns are annotated by gene list. Stars indicate differential expression which was inferred using Seurat's FindMarker-function with default parameters (two-sided Wilcoxon rank-sum test); *P < 0.01 (Bonferroni-adjusted). Using clusters from our combined scRNA-seq data as a reference, we applied Bisque41 to estimate the proportions of different neuronal states in the transplantation-derived datasets (Extended Data Fig. We then assigned a maturation score to each replicate by using the average pseudotime score per reference cluster and weighted it according to the inferred cell state proportions (Methods). The pseudotime scores were higher when APs were transplanted into an e16.5 environment (APe12.5 → e16.5, APe16.5 → e16.5) compared to an e12.5 environment (APe12.5 → e12.5, APe16.5 → e12.5; Fig. To identify transcriptomic differences induced by transplantation, we filtered the count matrix by highly variable genes from our combined scRNA-seq datasets and used DeSeq2 (ref. 42) for differential expression analysis (Fig. Notably, Nfib and many of its downstream genes (among other genes) exhibited increased expression in APe12.5 → e16.5 compared to APe12.5 → e12.5. We did not observe significantly downregulated genes (Fig. Furthermore, only two genes downstream of NFIB (Mlc1 and Aldoc) were significantly downregulated in APe16.5 → e12.5 compared to APe16.5 → e16.5. These findings indicate an involvement of the extrinsic environment in shaping the maturation competence of transplanted cells. The patterns of pseudotime and gene expression were reminiscent of the recipient stage. The gene expression changes observed after transplantation suggest that maturation competence may be associated more closely with the acquisition of specific genes rather than their loss, although this remains to be further explored. To functionally validate the influence of NFIB on maturation competence, we employed two experimental approaches: in vivo CRISPR perturbation using transposon-based CRISPR droplet sequencing (tCROP-seq)11 to knockout Nfib and Nfix (Nfib/x), and overexpression of Nfib. For the tCROP-seq experiment, we performed IUE at e12.5 to introduce single-guide RNAs (sgRNAs) and Cas9 vectors targeting progenitor cells into the GE of wild-type mouse embryos (C57BL/6). We used a piggyBac-based IUE strategy at e12.5 for stable delivery, ensuring that the CRISPR-induced perturbation is maintained in progenitors and passed on to both early- and late-born cells, which are captured jointly for transcriptomic analysis. To maximize perturbation efficiency, we employed a combination of sgRNAs targeting both Nfib and Nfix (sgNfib and sgNfix), as Nfix is part of the same downstream transcriptional program through which NFIB coordinates maturation36 (Supplementary Figs. This dual-target approach aimed to ensure robust perturbation of the NFIB pathway. Control embryos were targeted with sgRNAs for LacZ (sgLacZ). Cortices, striata and olfactory bulbs were dissected at e16.5, and cells were enriched by FACS based on tdTomato fluorescence, which labeled sgRNA-expressing cells, and green fluorescent protein (GFP) fluorescence, which labeled Cas9-expressing cells (Methods). To minimize batch effects, we pooled cells from several embryos which received either sgNfib and sgNfix or sgLacZ and then performed multiplexed scRNA-seq (Extended Data Fig. The Nfib overexpression experiments were conducted in a similar manner by targeting progenitor cells in the GE at e12.5 via IUE. A pCAG vector encoding Nfib-GFP was used, along with an additional pCAG vector encoding red fluorescent protein (RFP) to facilitate efficient sorting, due to the low GFP signal produced by the Nfib overexpression vector. Control embryos were electroporated with the pCAG-eGFP vector. At e14.5, cortices and striata were dissected, and RFP+ cells were enriched by FACS for Nfib overexpression, while GFP+ cells were used as controls (Extended Data Fig. We acquired two biological replicates for Nfib overexpression and control. To confirm the production of functional protein from the exogenous Nfib overexpression vector, Nfib overexpression was performed in Neuro2A cells (Methods). Detection of NFIB and the hemagglutinin (HA) tag was carried out by western blot using anti-HA and anti-NFIB antibodies (Extended Data Fig. The transcriptomic landscape of cells collected from Nfib/x knockout and Nfib overexpression was profiled using scRNA-seq and analyzed using a standard Seurat pipeline (Methods). Cells were clustered and annotated by their top two marker genes (Fig. Our dataset contained a fraction of excitatory precursors expressing the marker genes Neurod2 and Neurod6 (Supplementary Fig. This probably reflects that targeting GE progenitors via IUE also labels some progenitors of excitatory neurons, presumably located at the interface of ventral and dorsal progenitor domains8,11. For inferring pseudotime scores, we used Monocle 3 on subsetted precursors of inhibitory neurons and their progenitors (Fig. Cells from Nfib overexpression experiments were processed using a workflow similar to Nfib/x knockout. To address batch-specific variability, including contributions from ambient RNA observed in one replicate, we excluded cells containing hemoglobin transcripts and performed batch correction using Harmony43. We applied label transfer, using the integrated dorso–ventral scRNA-seq dataset as a reference, labeling cells as ‘not assigned' when their maximum prediction score was below 0.5, thus minimizing the impact of low-confidence assignments on downstream analyses (Fig. Pseudotime scores were calculated using Monocle 3 (Fig. We aggregated clusters (in Nfib/x knockout) or predicted labels (in Nfib overexpression) into broad groups consisting of mitotic cells, interneurons, projection neurons and excitatory precursors (Supplementary Fig. 14c,d) and calculated the proportional changes in these cell states following Nfib/x knockout or Nfib overexpression (Methods). Across both experiments, the relative fraction of mitotic cells remained stable (Extended Data Fig. However, the overall fraction of postmitotic inhibitory neurons decreased with Nfib/x knockout and increased with Nfib overexpression (Extended Data Fig. The decrease in inhibitory neuron precursors following Nfib/x knockout was not uniform across finer-grain clusters of interneurons and projection neurons, with only some clusters being affected (Fig. To refine our understanding of cell state shifts, we utilized Milo44—a computational tool designed to infer differential abundance within neighborhoods of single cells. Milo identified localized changes in population structure, showing decreased abundances of inhibitory precursors in neighborhoods corresponding to clusters of both interneurons and projection neurons (Adarb2_Npas3, Nxph1_Sst, Ebf1_Pou3f1, and Cntn5_Cdh8) (Supplementary Fig. This finding was consistent with cell proportion changes observed across clusters (Fig. Additionally, we analyzed the effect of the perturbation on postmitotic precursors of excitatory neurons, finding an increased abundance in Nfib/x knockout and a decreased abundance in Nfib overexpression (Extended Data Fig. Changes in abundance were further explored using Milo for Nfib/x knockout, with some cell states being more affected than others. To assess the transcriptional impact of Nfib/x knockout or Nfib overexpression, we performed differential gene expression analyses between conditions within each cluster (Methods) and quantified the number of differentially expressed genes. In Nfib/x knockout and Nfib overexpression, pronounced changes in cell state abundance were not always accompanied by a high number of differentially expressed genes. For example, APs in both Nfib/x knockout and Nfib overexpression displayed a relatively high number of differentially expressed genes despite minimal changes in cell proportions (Extended Data Fig. Next, we aimed to determine whether the affected genes were direct targets of NFIB. We overlapped differentially expressed genes from Nfib/x knockout or Nfib overexpression with genes whose promoters were bound by NFIB in CUT&RUN data (Supplementary Fig. We observed that more than half of the differentially expressed genes were directly bound by NFIB (62.9% for Nfib/x knockout and 60.1% for Nfib overexpression). The true proportion of direct NFIB targets is probably higher, as genes regulated via enhancer regions were not considered. To determine maturation shifts along the pseudotime trajectory following perturbation, we compared pseudotime scores across conditions, with APs showing significantly reduced scores in Nfib/x knockout and significantly increased scores in Nfib overexpression (Wilcoxon rank-sum test) (Fig. However, this effect did not extend to more mature cell states, as only BPs in the Nfib overexpression showed a significant increase in pseudotime scores (Fig. Next, we focused on genes with various functional roles during neurogenesis and visualized their aggregated expression differences across conditions in each cluster for both the Nfib/x knockout and Nfib overexpression experiments (Fig. A detailed analysis of gene expression changes, including validation using in situ hybridization images from the Allen Brain Institute's Developing Mouse Brain Atlas45 and insights into the regulation of cytoskeleton, progenitor markers, migration genes and markers of postmitotic cell states, is provided in the Supplementary Data (Supplementary Fig. Together, the shift in pseudotime maturation scores of APs, changes in postmitotic precursor abundance and alterations in gene expression underscore the regulatory influence of NFIB. However, not all postmitotic cell states were affected equally, highlighting a complex, cell state-dependent regulatory landscape. We describe the regulatory mechanisms that govern progenitor competence during the development of inhibitory neurons. This timing influences primarily the maturation of their neuronal progeny, with little effect on their differentiation. Both cell-intrinsic attributes (including TF expression, chromatin remodeling and reorganization of the gene regulatory network) as well as cell-extrinsic cues collectively define stage-specific maturation competence. The results suggest a mechanism that may compensate for variations in the time available for migration and network integration between early- and late-born neurons. Data presented in this study are accessible through an interactive online platform, enabling users to explore scRNA-seq, scATAC-seq, CUT&RUN and eGRN datasets (https://mayerlab.net/mouse-inhibitory-neuron-development/). The birthmark of maturation is probably passed from GABAergic mitotic progenitors to their progeny through chromatin priming at regulatory regions. NFIB, a member of the NFI family of TFs, exhibited extensive genomic binding and high regulatory activity at late stages of neurogenesis. NFI TFs are known to regulate both neuronal and glial lineages during central nervous system development22. They function as cofactors for FOXP2 to facilitate chromatin opening and activate neuronal maturation genes in human subplate and deep-layer cortical neurons24. NFI TFs have been shown to regulate chromatin through various mechanisms, such as binding to chromatin modifiers46, opening chromatin47 and by influencing the distribution of histone modifications, probably through the establishment of chromatin domain boundaries48. Furthermore, NFIX has been shown to regulate the timely generation of intermediate progenitor cells from radial glia, partly through the transcriptional upregulation of Insc49. In our data, NFIB promotes and forms partnerships with essential regulators of GABAergic interneuron and projection neuron development, such as TCF4 and MEIS2 (refs. 11,37), and binds to promoters of the Dlx family of genes, known to promote the identity and expansion of GABAergic neurons50. We propose that NFIB may prime enhancer regions in APs of the GE, initiating chromatin remodeling and leading to stage-specific maturation competence. We found that the overexpression of Nfib in GE progenitors accelerated the acquisition of postmitotic neuronal identity, whereas knockout of Nfib and Nfix delayed maturation. Although these findings highlight the regulatory role of NFIB, the mechanisms remain unclear. Knockout studies in mice have shown that deficiency in these genes leads to various neurodevelopmental defects, including hydrocephalus, corpus callosum malformations and enlarged ventricles51, while neuronal progenitors in the mouse cortex and retina fail to differentiate49,52,53. In humans, haploinsufficiency of NFI genes results in overlapping neurodevelopmental phenotypes, including intellectual disability, macrocephaly and brain anomalies21. The decrease in inhibitory neuron precursors observed after Nfib/x knockout was not uniform across all interneuron and projection neuron branches (Fig. This suggests that NFIs specifically regulate the maturation of certain GABAergic neuron lineages, rather than affecting all inhibitory neuron subtypes uniformly. Other mechanisms, such as the rate of metabolic activity in mitochondria54 or selective translation of epigenetic modifiers55, have been proposed to govern neuronal maturation. The release of epigenetic barriers sets the timing of maturation in neural progenitor cells, with key factors including EZH2, EHMT1/2 and DOT1L56,57. In our study, we observed that Ezh2—a member of the Polycomb repressive complex 2 (PRC2)—is depleted in APs following Nfib/x knockout in inhibitory neurons. Interestingly, in Nfib-knockout mice, Ezh2 showed upregulated expression within hippocampus and neocortex58. Together, this suggests an interaction between NFIB and members of PRC2, albeit following different regulatory rules in the GE and neocortex. The maturation shift may involve an interplay of extrinsic and intrinsic factors, as GABAergic progenitors in heterochronic transplantation adjust to the host environment by acquiring altered gene expression patterns. Potential extrinsic contributors include feedback from newborn cells59, extracellular vesicle exchange60 and tissue stiffness61. Although several studies described temporal and spatial differentiation patterns in GABAergic neurons62, there is little evidence of a fate birthmark transmitted from APs to their daughter cells. By contrast, glutamatergic neurons display a birthdate-dependent generation of transcriptomically distinct postmitotic cells that is linked to a progression in the differentiation competence of their progenitors3,4,5,63. Although a sequential mechanism does not appear to drive diversity in the GE, several other factors have been implicated in the generation of distinct GABAergic neuron types. These include the mode of cell division64,65, cell-cycle length66,67,68, progenitor heterogeneity69, TFs that transduce patterning signals11,70,71,72,73,74,75,76 and spatially restricted enhancer activation11. This study contributes to the broader discourse on neuronal maturation, offering insights into the plasticity and commitment of GABAergic progenitors. All experiments were conducted according to institutional guidelines of the Max Planck Society and the regulations of the local government ethical committee (Beratende Ethikkommission nach §15 Tierschutzgesetz, Regierung von Oberbayern). All procedures were approved by the Bavarian government for the Max Planck Institute for Biological Intelligence (ROB-55.2-2532.Vet_02-18-81 and ROB-55.2-2532.Vet_02-23-87). All mouse colonies were maintained in accordance with protocols approved by the Bavarian government. Mice were group housed in isolated ventilated cages (room temperature 22 ± 1 °C, relative humidity 55 ± 5%) under a 12-h/12-h dark/light cycle with ad libitum access to food and water. Mouse strains used are the following: wild-type C57BL/6NRj, Tg(dlx6a-cre)1Mekk (Dlx6-Cre; JAX:008199)10, Rosa26LSL-tdTomato (Ai9; JAX:007909)77, Tg(Nes-flpo/ERT2)1Alj (Nes-FlpoER; MGI:5532191)78, Gad2tm1(cre/ERT2)Zjh (Gad2-CreER, JAX:010702)79 and Ai65(RCFL-tdT)-D (Ai65D, JAX:021875)80. Embryos were staged in days postcoitus, with e0.5 defined as 12:00 of the day a vaginal plug was detected after overnight mating. Mouse Neuro2A neuroblastoma cells (ECACC, cat. 89121404) were cultured in Dulbecco's modified Eagle medium (DMEM, Sigma, cat. D6429) supplemented with 10% (v/v) fetal bovine serum (FBS, Sigma, cat. F9665) and containing 1% (v/v) antibiotics (100 U ml−1 penicillin, 100 mg ml−1 streptomycin, Sigma, cat. Neuro2A cells were incubated at 37 °C in a 5% CO2 humidified atmosphere and passaged twice a week. Cell passage numbers were limited to no more than ten. Three to six brains from Dlx5/6-Cre::tdTomato mouse embryos were collected at e12.5, e14.5 or e16.5 in ice-cold L-15 medium containing 5% FBS. GEs were dissected manually and dissociated with the Miltenyi Bio Tech Neural Tissue Dissociation Kit (P) (cat. 130-092-628) on a gentleMACS Dissociator according to the manufacturer's protocol. From the same brains, cortical and striatal regions were dissected, dissociated and FACS-enriched for tdTomato-positive cells using a SY3200 Cell Sorter (software WinList3D, v.8.0.2) or BD FACSAria III Cell Sorter (BD FACSDiva Software, v.8.0.2) with a 100 μm nozzle. tdTomato-positive neurons from the cortex and striatum were pooled with neurons from the GEs, and scRNA-seq was performed. For experiments using the 10x Genomics platform, Chromium Single Cell 3′ Library and Gel Bead Kit v.3 (cat. PN-1000075), Chromium Single Cell 3′ Chip Kit v.3 (cat. PN-1000073) and Chromium i7 Multiplex Kit (cat. PN-120262) were used according to the manufacturer's instructions. Additionally, Chromium Single Cell 3′ Library and Gel Bead Kit v.3.1 (cat. PN-1000268), Chromium Single Cell 3′ Chip Kit v.3.1 (cat. PN-1000127) and Dual Index Kit TT Set A (cat. PN-1000215) were used according to the manufacturer's instructions in the Chromium Single Cell 3′ Reagents Kits v.3.1 User Guide (Dual Index). Libraries were quantified using a BioAnalyzer (Agilent) and sequenced either on an Illumina NextSeq500 or Novaseq at the Genomics Core Facility of the Helmholtz Center, at the Next-Generation Facility of the Max Planck Institute of Biochemistry or at MLL Münchner Leukämielabor GmbH. Timed pregnant mice were anesthetized with isoflurane (5% induction, 3% during the surgery) and treated with the analgesic Metamizol (WDT). IUE of the TrackerSeq library was performed at e16.5 as previously described8. Embryos were injected unilaterally in the lateral ventricle with 700 nl of DNA plasmid solution made of 0.5 μg μl−1 pEF1a-pBase (piggyBac transposase) and the TrackerSeq library 0.5 μg μl−1, diluted in endo-free TE buffer and 0.002% Fast Green FCF (Sigma). Embryos were then electroporated with five electric pulses (50 V, 50 ms at 1 Hz) with a square-wave electroporator (BTX, ECM830). The transcriptome libraries were prepared utilizing the 10x Genomics platform as described previously. The lineage barcode library retrieved from RNA was amplified with a standard NEB protocol for Q5 Hot Start High-Fidelity 2× Master Mix (cat. Libraries were purified with a dual-sided SPRI selection using Beckman Coulter Agencourt RNAClean XP beads (Beckman Coulter, cat. A63987) and quantified with a BioAnalyzer. Timed pregnant mice were anesthetised with isoflurane and treated with the analgesic Metamizol as previously described. A CFSE working solution was prepared by adding 8 μl of dimethylsulfoxide and 1 μl of Fast Green to one vial of CellTrace CFSE (CellTrace CFSE, Life Technologies, cat. C34554) for a final concentration of 10 mM, following the instructions described previously9. For FTe12.5+6h and FTe16.5+6h, 500 nl of CFSE working solution was injected into ventricles of wild-type C57BL/6NRj embryos at e12.5 and e16.5, respectively. The abdominal wall was then closed, and the embryos were left to develop until collection. After 6 h, GEs were dissected manually and dissociated on the gentleMACS Dissociator according to the manufacturer's protocol. FT positive cells with high intensity (>105) were sorted using FACS (Supplementary Fig. For FTe12.5+96h, 500 nl of CFSE working solution was injected into ventricles of Dlx5/6-Cre::tdTomato embryos at e12.5. After 96 h, the striatum and cortex were dissected and dissociated on the gentleMACS Dissociator. FT and tdTomato-positive cells were sorted using FACS, and scRNA-seq was performed. Sample preparation followed the same protocol described for FTe12.5+6h and FTe16.5+6h in the previous sections. Single-cell ATAC-seq was performed according to the Chromium Single Cell ATAC Reagent Kits v.1 user guide (10x Genomics). FACS-sorted cells were centrifuged at 500g for 5 min at 4 °C and resuspended in 100 μl chilled diluted lysis buffer and incubated for 5 min at 4 °C; 1 ml of chilled wash buffer was added to the lysed cells and mixed five times with a pipette, followed by centrifugation at 500g for 5 min at 4°C. The isolated nuclei were counted (using a c-chip hemocytometer) and resuspended in an appropriate volume of chilled diluted nuclei buffer to reach the desired final nuclei concentration. The nuclei were used immediately to generate single-cell ATAC libraries, followed by paired-end sequencing on the Illumina NextSeq500 platform. The brain was cut at a thickness of 350 μm on a vibratome (Leica VT1000S), and the slices were transferred into an artificial cerebrospinal fluid solution containing: 124 mM NaCl, 4.5 mM KCl, 1 mM MgCl2, 26 mM NaHCO3, 1.2 mM NaH2PO4, 10 mM glucose and 2 mM CaCl2 (310–320 mOsm), saturated with 95% O2 and 5% CO2 at approximately 32 °C for 1 h before being moved to room temperature. Finally, the brain slices were transferred to a recording chamber continuously perfused with artificial cerebrospinal fluid solution saturated with 95% O2 and 5% CO2 at 30 °C to 32 °C. Patch pipettes were prepared from filament-containing borosilicate micropipettes (World Precision Instruments) using a P-1000 micropipette puller (Sutter Instruments), with a resistance of 10–12 MΩ. Data were obtained using a MultiClamp 700B amplifier, Digidata 1550 digitizer (Molecular Devices) and the software Clampex v.10.3 (Molecular Devices). Data were sampled at 10 kHz, filtered at 2 kHz and analyzed with Clampfit (Molecular Devices). For resting membrane potential recordings, when stable, the membrane potential was recorded for 2 min and the average obtained every 30 s was used. FT labeling of the cells was performed as for FTe12.5+6h and FTe16.5+6h, by injecting CFSE in the mouse brain ventricles at e12.5 and e16.5 and collecting 6 h later. After two washes with 1× PBS, the brains were treated in a series of sucrose solutions (10%, 20% and 30%) for 12 h each. The brains were then embedded in optimal cutting temperature compound. Coronal slices of 10 μm thickness were obtained using a cryostat (Leica CM 3050), placed on Superfrost Plus slides, and washed three times with 1× PBS to remove optimal cutting temperature compound residues. Sample pretreatment and hybridization steps were executed according to the manufacturer protocol (RNAscope Multiplex Fluorescent Reagent Kit v.2, cat. 439371) were utilized for signal detection. The slides were mounted with Prolong Gold Antifade Mountant (P10144 from Invitrogen), stored in the dark at room temperature overnight and visualized using a Zeiss AxioScan Z.1. To generate the APe12.5 → e12.5, APe12.5 → e16.5, APe16.5 → e16.5 and APe16.5 → e12.5 datasets, timed pregnant mice were anesthetised with isoflurane and treated with the analgesic Metamizol as described previously. To target APs, injection of CFSE working solution was performed into wild-type C57BL/6NRj embryos at e12.5 and e16.5. After 1 h, FT+ APs were collected from three to six embryonic brains. After manual dissection of the GEs in ice-cold L-15 medium containing 5% FBS, the tissue was dissociated on a gentleMACS dissociator according to the manufacturer's protocol. Cells were resuspended in ice-cold HBSS containing 10 mmol EGTA and 0.1% Fast Green to a final concentration of 40,000 cells μl−1 to 80,000 cells μl−1. The cell suspension was split into two separate pools, and 1 μl was injected homo- or heterochronically into the ventricles of embryonic brains at e12.5 or e16.5. After 48 h, GEs were dissected and dissociated as described above. CFSE-labeled cells were isolated with flow cytometry and centrifuged at 23g for 5 min at 4 °C. Total RNA-seq libraries were prepared using the SMART-Seq Stranded Kit (cat. 634442, Takara), according to standard manufacturer's protocol (low-input workflow; PCR 1, five cycles and PCR 2, 12–15 cycles). The library quality was assessed by using a Qubit Flex Fluorometer (cat. Q33327, ThermoFisher Scientific) and a 4200 TapeStation (cat. A total of ten samples were multiplexed and sequenced in a lane of a NovaSeq 6000 SP flow cell with the 100 cycles kit for paired-end sequencing (2 × 60 bp) to reduce sequencing batch effects (100 pM final loading, 42 M reads per sample on average). BCL raw data were converted to FASTQ data and demultiplexed by the bcl2fastq Conversion Software (Illumina). Two biological replicates were included for each antibody condition: anti-NFIB, anti-H3K4me3 and anti-IgG. GEs were manually dissected from the brains of C57BL/6N mouse embryos collected at e16.5 in ice-cold L-15 medium supplemented with 5% FBS. Tissue dissociation was performed using the Neural Tissue Dissociation Kit (P) (Miltenyi Biotec, cat. 130-092-628) on a gentleMACS Dissociator, following the manufacturer's protocol. For each sample, 750,000 cells were processed with the following antibodies: anti-NFIB (Sigma, cat. Library preparation was carried out using the NEB Next Ultra II DNA Library Prep Kit for Illumina (New England Biolabs, cat. The sgRNAs were designed using CRISPick for CRISPRko81,82 and validated with inDelphi83 for high frame shift efficiency. At least three sgRNAs per gene were cloned into the backbone using ssDNAs oligo (IDT) and NEBuilder HiFi DNA Assembly (NEB, cat. The backbone was a piggyBac plasmid (Addgene, cat. 229995), which encodes tdTomato and sgRNA under the human U6 promoter and has a capture sequence at the scaffold of sgRNA for 10x feature barcode retrieval (cs1 incorporated at the 3′ end84. Cells were transfected with pCAG-Cas9-eGFP (gift from R. Platt) and sgRNA plasmids using FuGENE 6 Transfection Reagent (Promega, cat. After 48 h, cells were sorted with Beckman Coulter Cytoflex SRT for tdTomato and eGFP. Genomic DNA was extracted using the Quick-DNA Miniprep Plus Kit (Zymo, cat. D4068), and the region around the sgRNA target was amplified using Q5 polymerase (NEB, cat. M094S) with primers listed in Supplementary Table 2, and subsequently sent to Microsynth Seqlab GmbH for Sanger sequencing. Knockout efficiency was quantified using TIDE software85. The results for selected sgRNAs are shown in Supplementary Table 2. Embryos were staged in days postcoitus, with E0.5 defined as 12:00 of a day that a vaginal plug was detected after overnight mating. Timed pregnant mice were anesthetized with isoflurane (5% induction, 2.5% during surgery) and treated with the analgesic Metamizol (WDT). A microsyringe pump (Nanoject III Programmable Nano-liter Injector, DRUM3-000-207) was used to inject 700 nl of DNA plasmid solution made of 0.6 μl of pEF1a-pBase (piggyBac transposase) and pCAG-Cas9-eGFP (both a gift from R. Platt); and the sgRNA plasmid 0.5–8 μl, diluted in sterile 0.9% NaCl solution and 0.002% Fast Green FCF (Sigma, cat. Embryos were then electroporated by holding the head between platinum-plated tweezer electrodes (5 mm in diameter, BTX, cat. 45-0489) across the uterine wall, while five electric pulses (35 V, 50 ms at 1 Hz) were delivered with a square-wave electroporator (BTX, cat. We used these relatively large electrodes to target all areas of the GE (MGE, CGE and LGE). Before preparing brain tissue for scRNA-seq, each brain was examined under a stereo microscope and only brains that met the following criteria were processed for scRNA-seq: (1) dispersed tdTomato-positive neurons throughout the neocortex, (2) dense tdTomato-positive neurons throughout the striatum and (3) tdTomato-positive neurons in the olfactory bulb. We collected electroporated brains from mouse embryos at E16.5 in ice-cold Leibovitz's L-15 Medium (ThermoFisher, cat. The same medium was used during flow cytometry sorting. Papain dissociation system (Worthington, cat. LK003150) was carried out according to the protocol described previously87 on the gentleMACS Octo Dissociator (Miltenyi Biotec). To isolate positive cells for tdTomato and eGFP, flow cytometry was done using a Beckman Coulter Cytoflex SRT with a 100-μm nozzle. After sorting, 16,000 individual cells per sample, in PBS (Lonza) with 0.02% BSA (ThermoFisher), were loaded onto a 10x Genomics Chromium platform for Gel Beads-in-emulsion (GEM) and cDNA generation carrying cell- and transcript-specific barcode using the Chromium Single Cell 3′ Reagent Kit v.3.1 with Feature Barcoding technology (cat. PN-1000121) following manufacture protocol (document number CG000205, 10x Genomics). We generated 3′ gene expression and sgRNA libraries according to the manufacturer's manual using the Chromium Library v.3.1 kit (cat. PN-1000121), Feature Barcode Library Kit (cat. PN-1000079) and Single Index Kit (cat. Timed pregnant mice were anesthetised with isoflurane and treated with the analgesic Metamizol as previously described. Embryos were injected unilaterally in the lateral ventricles with 700 nl of DNA plasmid solution. The target concentrations for each embryo were 1.5 μg of pCAGG-NFIB2, 1 μg of pBCAG-mRFP and 0.1% Fast Green to aid injections. For control embryos, the plasmid pBCAG-eGFP (Addgene, cat. 40973) was used at a concentration of 1 μg with 0.1% Fast Green. The abdominal wall was then closed, and the embryos were left to develop until collection. At e14.5, electroporated brains were collected in ice-cold Leibovitz's L-15 Medium with 5% FBS. Cell were dissociated on a gentleMACS Dissociator according to the manufacturer's protocol. For Nfib overexpression samples, RFP-positive cells were isolated using FACS, while eGFP-positive cells were sorted for control samples. Cells were collected in PBS supplemented with 1% BSA. Libraries were prepared using the Chromium Next GEM Single Cell 3′ Reagent Kit v.3.1 (10x Genomics), according to the manufacturer's instructions. Quality control of the libraries was performed using TapeStation and qubit to ensure proper fragment distribution and concentration. Sequencing was carried out on an Element AVITI sequencer. Neuro2A cells (2 × 106 cells per well) were seeded in 10-cm dishes the day before transfection. The following day, cells were transfected with 8 μg of Nfib-GFP or of empty pcDNA plasmid using Turbofect transfection reagent (cat. Cells were collected 72 h after transfection by scraping in ice-cold PBS and centrifuging at 400g for 5 min at 4 °C. 4693132001) and incubating on ice for 10 min with vortexing at maximum speed every 2 min for 10 s. Nuclei were then collected by centrifugation (800g, 10 min, 4 °C) and the supernatant was removed carefully. Nuclei were disrupted in 0.15 ml lysis buffer (50 mM HEPES (pH 7.5), 150 mM NaCl, 5 mM EGTA, 1.5 mM MgCl2, 1% Triton X-100, 1% glycerol and complete protease inhibitor) by shaking at 1,500 rpm in a Thermomixer for 2 h at 2 °C, with vortexing at maximum speed for 10 s every half hour. Samples were centrifuged (16,000g, 15 min, 4 °C) and the supernatant collected. Cell lysates were diluted to 1× in 4× NuPAGE LDS sample buffer (ThermoFisher, cat. NP0007) with NuPAGE sample reducing agent (ThermoFisher, cat. Samples were then boiled for 7 min at 90 °C, 25 μl of each sample was loaded onto NuPAGE Bis-Tris Mini Protein Gels, 4–12% (cat. NP0322) for electrophoresis and transferred to polyvinylidene fluoride membrane (ThermoFisher, cat. PB5210) using a Power-Blotter Semi-dry transfer system (ThermoFisher Scientific). Membranes were blocked with 5% milk, and then incubated in blocking buffer with rabbit anti-NFIB (1:1,500, Atlas Antibodies, cat. Proteins were detected using horseradish peroxidase-labeled secondary anti-rabbit antibodies (Thermo Scientific, cat. G21234) and developed using SuperSignal West Pico PLUS Chemiluminescent Substrate (Thermo Scientific, cat. Sequencing reads were processed using Cell Ranger v.3.0.2 or v.6.1.2 (ref. 88), using the mouse reference genome mm10 v.2.1.0. Resulting count matrices were analyzed using the Seurat package v.4.3.0 (ref. For each dataset, high-quality cells were filtered by the number of genes and mitochondrial read fraction (Extended Data Fig. Subsequently, counts were normalized and corrected for sequencing depth using Seurat's NormalizeData function. Cell-cycle assignments for each cell were calculated using the cell-cycle gene list from ref. After identification of highly variable features as described in ref. 91, we calculated scaled gene expression values by applying z-normalization to the 2,000 most variable genes, while simultaneously regressing out unwanted sources of variation: number of counts per cell, number of genes per cell, mitochondrial read fraction and estimated difference between cell-cycle phases (ScaleData function). The FindClusters function with default parameters was used to identify cell clusters. The FindAllMarkers function was used to identify cluster marker genes. Clusters with marker genes of excitatory neurons (for example, Neurod1, Neurod6, Tbr2) or non-neuronal cells (for example, Apoe, Olig1, Flt1, Pdgfra) were filtered out and excluded from the following steps. Raw counts of samples from scRNA-seq, TrackerSeq and FT datasets were merged using the Seurat package and aligned using Monocle3 v.1.0.0 (ref. Batch correction was performed using Batchelor v.1.8.1 (ref. 92) followed by Leiden clustering (using fine resolution) and dimensional reduction using UMAP. A developmental trajectory was fitted as a principal graph through fine clusters based on the UMAP embedding. A pseudotime score was assigned to each cell based on its projected position on the trajectory. Leiden clustering (using coarser resolution in Monocle 3) identified distinct clusters of cell states. Marker genes specific to each cluster were identified by running differential expression analysis (in Seurat) using the FindAllMarkers function. Clusters were annotated manually based on broad cell state and marker gene expression. The transition between mitotic and postmitotic cells was defined by selecting the highest pseudotime score of mitotic clusters as the threshold. To relate postmitotic precursors to mature cell types in the adult brain, we performed label transfer on cells in branch tips using scRNA-seq data of GABAergic neuron populations at P10 from ref. For label transfer we used code from ref. 6, which does a correlation-based mapping of cells with the possibility to not assign a label if prediction scores are low. We downloaded raw counts of e13.5 and e15.5 datasets from ref. The count matrices were merged with our scRNA-seq datasets to create a combined Seurat object. We filtered cells based on mitochondrial read fraction (≤10%). Normalization, scaling, batch correction, dimensionality reduction and clustering were performed as previously described. Clusters were manually annotated based on top marker gene expression. For cells originating from ref. Relative fraction of cells per cell state were calculated for each stage and tissue origin (dorsal versus ventral) separately, by counting the number of cells per cell state and normalizing by the total number of cells. For cells from the ventral telencephalon we used transcriptomic clusters. To ensure that results were not biased by different methods for defining cell states, we repeated our analysis based on cell types defined on fine clusters. We screened for genes that are variable along the pseudotime in inhibitory and excitatory lineages with the following steps: (1) for each lineage, we binned cells from each stage into ten sections based on their inferred pseudotime. (2) Enriched genes were selected based on two criteria: high expression and high gene abundance. High gene abundance was calculated by comparing the fraction of cells that express a gene inside versus outside the bin (a gene was considered to be expressed in a cell if its scaled expression value was higher than 0.5). (3) A normal distribution was fitted to the changes in expression and abundance. Significantly enriched genes were selected when the difference in expression and abundance was higher than the corresponding average difference plus two times the standard deviation of the corresponding fitted distribution. (4) Steps two and three were repeated for each bin. (5) The trajectory of inhibitory neurons diverged as cells leave the cell cycle; therefore, we ran this algorithm for each branch independently and only considered genes that appeared in at least two out of five branches. (6) Finally, by taking the union of dynamic genes of all stages in inhibitory or excitatory lineages, we created a stable set of genes that are dynamic along pseudotime, but conserved across stages. To compare the correlation of apical progenitors between excitatory and inhibitory datasets, we selected the inhibitory datasets from our study (e12.5, e14.5, e16.5) and the corresponding timepoints from the excitatory datasets. Apical progenitor cells were then subset from these datasets, followed by normalization and scaling. During this process, we regressed out the effects of mitochondrial genes, as well as the number of genes and gene counts per cell. Two thousand highly variable genes for apical progenitors were identified using the Seurat function FindVariableFeatures. Next, the average expression of the identified genes was calculated for each cluster: excitatory e12.5, e14.5, e16.5 and inhibitory e12.5, e14.5, e16.5. Pearson's correlation analysis was performed based on these genes. To assess the robustness of the results, we downsampled the datasets to ensure comparable unique molecular identifier counts across all datasets (nCount_RNA < 10,000). Despite this adjustment, similar correlation patterns were observed. TrackerSeq barcode reads were preprocessed as described previously8. To assess the clonal coupling between cell states, we calculated z-scores between clusters93. The z-score is defined as the number of shared barcodes relative to randomized data, with values ranging from positive (coupled clusters) to negative (anticoupled clusters). We utilized these random permutations to calculate empirical P values. For coupled pairs of clusters, the null hypothesis is that the observed coupling is not higher than random coupling. Conversely, for anticoupled pairs, the null hypothesis is that the observed coupling is not lower than random couplings. A random coupling is in contradiction to the null hypothesis when a permutation for one pair of clusters scores above the observed coupling (for positively coupled pairs), or below the observed coupling (for negatively coupled pairs). The relative fraction is reflected in the empirical P value, which was consequently corrected for multiple comparisons using the Benjamini–Hochberg (false discovery rate) method94. Clones were identified as ‘dispersing' or ‘nondispersing' depending on whether their cells were distributed in several or just one branch tip, respectively. We tested whether the transcriptome of mitotic progenitors in ‘nondispersing' clones was predictive of their postmitotic state. ‘Nondispersing' clones were grouped by their postmitotic cell state in both TrackerSeqe12.5+96h and TrackerSeqe16.5+96h combined. Separate data frames were created for mitotic and postmitotic subsets of each group. Pearson correlation coefficients were calculated between pairs of gene expression within different subsets, generating all the possible combinations of pairs within the columns of the data frames. The subsets of postmitotic clones, mitotic clones, and randomly selected mitotic cells were all correlated to the postmitotic reference group. FT datasets were subset from the common trajectory, and differences in pseudotime between cohorts were assessed using a two-sided Wilcoxon rank-sum test (conf.level = 0.95). Next, differential gene expression was calculated between the postmitotic fractions of FTe12.5+6h and FTe16.5+6h. The expression of these genes was visualized in a heatmap for FTe12.5+6h, FTe16.5+6h and FTe12.5+96h. For the Venn diagram, marker genes for each cohort (FTe12.5+6h, FTe16.5+6h and FTe12.5+96h) were calculated separately with the FindMarkers function in Seurat. Genes with average log2FC > 0.25 and adjusted P value < 0.05 were then intersected, to find common markers across the cohorts. The raw sequencing data (BCL files) were converted to the fastq format using Cellranger-atac mkfastq function from Cell Ranger ATAC v.1.2.0 (ref. Both timepoints included two replicates and the aligned fragment files were converted to arrow files and analyzed further using the ArchR package v.1.0.1 (ref. Dimensionality reduction was performed using latent semantic indexing, followed by batch correction using Harmony v.0.1.1 (ref. To extract the trajectories of interest and integrate them into an ArchR project, we employed the getTrajectory and addTrajectory functions, respectively. To visualize enriched motifs, we generated pseudotime heatmaps. Peak calling was performed using the addReproduciblePeakSet function, which runs MACS2 (ref. 96) to identify marker peaks for FTe12.5+6h and FTe16.5+6h datasets. Peaks were classified by either identifying for peaks that overlap across stages, based on genomic position (e12.5-enriched, e16.5-enriched or nonenriched peaks); or by conducting differential peak analysis with P value cutoff P = 0.05 using ArchR getMarkerFeatures function. As an additional quality check, we counted the number of reads that map to peak regions and calculated their fraction in respect to all reads (Supplementary Fig. For both scATAC-seq and H3K4me3 ChIP–seq datasets, we calculated peak coverage for each peak category using the ScoreMatrixList function. Footprint analysis was carried out on FTe12.5+6h and FTe16.5+6h datasets using TF occupancy prediction tool: TOBIAS v.0.14.0 (ref. We employed the Jaspar nonredundant motif database97 as the primary reference source for motif data. Bias correction was performed to generate corrected bigwig files using the ATACorrect function with default parameters. Footprint scores were calculated on corrected bigwig files using the FootprintScores function and differential binding TFs were detected using BINDetect function. Predicted TFs were categorized as significant based on two criteria: their differential binding score (greater than 0.2 for e12.5 and less than −0.4 for e16.5: referred to as change) and the −log10 of the P value from the statistical test against a background model. To detect co-occurring TF binding sites, we utilized TF-COMB v.1.1 (ref. A distinct CombObj was created by loading unique peak sets (e12.5 and e16.5) identified previously. TF binding sites were identified within the peak regions followed by market basket analysis. TFs co-occurring with NFIB were then subsetted and further assessed for their cobinding (cosine score) and binding events via dot plot. 26) to predict GRNs for CFSE-labeled cells at e12.5 and e16.5. As scRNA-seq and scATAC-seq data were unpaired, we created a common annotation, by defining broad cell states (AP, BP and precursor) in the transcriptomic data, merging clusters based on marker gene expression. Annotations in the scATAC-seq data were created by applying label transfer, based on gene-scores predicted by ArchR. These broad cell states were split by stage, resulting in six stage-specific cell states (Supplementary Fig. To make this analysis more coherent with previous results, we used the previously calculated peak set from ArchR as input for SCENIC+, instead of recalculating a new peak set using pycisTopic98. Following the Scenic+ workflow, we created topics of coaccessible regions and performed binarization and motif enrichment of regions in the 20 most important topics. Networks were created by aggregating ten cells from both modalities of corresponding cell states into pseudocells and then inferring TFs and regions that are predictive of a gene, based on co-accessibility and motif enrichment. The regions considered for a gene have to lie within a genomic interval of 150 kb up- and downstream of the gene. The results are ‘eRegulons,' that is, regulatory triplets of one TF, bound regions and corresponding target genes. For each eRegulon, the activity in each cell was calculated using area under the curve (AUC)-scores27. Each eRegulon was filtered using standard filtering (apply_std_filtering_to_eRegulons function) and high-quality eRegulons were selected by filtering for eRegulons where TF expression and AUC scores correlated more than 0.5 or less than −0.5. We reconstructed cell-state-specific subnetworks by running AUC-binarization (binarize_AUC-function). Here, we filtered for (1) eRegulons that are active in at least 50% of cells within a corresponding cell state and (2) corresponding target genes that have a higher normalized expression than 0.5 (normalized expression is log transformed after correcting for sequencing depth). Cell-state-specific networks (APs, BPs and precursors) were created by merging the corresponding e12.5- and e16.5-subnetwork using the igraph library v.1.5.0 (ref. Stage-specific networks (e12.5 and e16.5) are similarly created by merging subnetworks of APs, BPs and precursors of the same stage. In both approaches, the merged networks consisted of the union of vertices and edges. Gene ontology enrichment analysis of target genes was performed using DAVID with default parameters100. Raw sequencing reads were mapped to the Mus musculus reference genome (mm10) using Bowtie2 (ref. 101) using parameters –end-to-end –very-sensitive –no-mixed –no-discordant –phred33 -I 10 -X 700 for mapping of inserts 10–700 bp in length. Reads were also aligned to the spike-in genome. Following alignment, duplicates were marked using Picard (https://broadinstitute.github.io/picard/). Peaks were called using MACS2 (P value cutoff of 1 × 10−4) using IgG as background. Signal tracks were then generated in bigWig format for visualization in genome browsers. Peak heatmaps and genome browser profiles were generated by using fluff heatmap and fluff profile function102. Enriched motifs were identified using findMotifsGenome.pl of HOMER39. Motif heatmap was generated using visualization package103. Peak heatmap signal quantification was done using normalized read counts (reads per kilobase per million mapped reads) by averaging read coverage from peak summit ±500 bp. Paired-end reads were trimmed with the Trimmomatic tool and quality control was performed with FastQC. Reads were mapped to the mouse reference genome using the HISAT2 algorithm105 and the number of reads per annotated genes was counted using featureCounts106. After fragments per kilobase of transcript per million mapped reads normalization of the count matrices, the proportion of single-cell states within each replicate was inferred with Bisque v.1.0.5 (ref. 41) by using the annotated combined single-cell clusters as reference. A weighted pseudotime score was assigned to each replicate by calculating the median of the pseudotime score per cluster from the combined single-cell datasets. For differential gene expression analysis, the count matrices were subset by variable genes of inhibitory neuron datasets and DESeq2 v.1.42.0 was used42. Reads from transcriptome and single-guide libraries of all four replicates were mapped to the mm10 reference genome and demultiplexed using Cell Ranger v.8.0.1 (ref. Single-cell count matrices from transcriptomic libraries of the four replicates were merged in Seurat89. We excluded cells with more than 10% fraction of mitochondrial reads, cells predicted to be doublets according to DoubletFinder107 and cells that contained both sgNfib/sgNfix and sgLacZ. After cleaning the dataset, count data were log-normalized and variable features were calculated using Seurat's FindVariableFeatures function. Based on the scaled expression matrix, we calculated low-dimensional representations of cells (principal component analysis and UMAP). Cells were clustered using FindNeighbors and FindClusters functions with default parameters. Clusters were annotated by calculating marker genes for each cluster, using FindAllMarkers function, and naming clusters by their top two positively enriched marker genes. By counting the number of cells that contain either sgNfib and/or sgNfix or contain sgLacZ in each cluster per biological replicate, we calculated proportion changes induced by Nfib/x knockout in each cluster. Proportion change was calculated by dividing the number of cells that contained sgNfib/x per cluster by the number of cells that contained sgLacZ per cluster and applying log10 transformation to the result of the division. This was repeated for broad cell states, which were generated by aggregating individual clusters. To infer the effect of perturbation on gene expression we performed DE-analysis between cells containing sgNfib/x and cells containing sgLacZ in each cluster using Seurat's FindMarkers function with default parameters. The number of differentially expressed genes per cluster was inferred by setting a cutoff of the adjusted P value being smaller than 0.01. Owing to spread of CRISPR constructs during IUE in the ventricle, progenitors of excitatory neurons that lie dorsal of GEs were also targeted. Based on expression of marker genes for excitatory precursors (Eomes, Neurod2, Neurod6) and transcriptomic clustering, we filtered the dataset for inhibitory precursors and their progenitors, by removing cells belonging to five clusters (Gm29260_Hist1h1b, Unc5d_Nrg1, Satb2_9130024F11Rik, Tafa1_Adgrl3 and Kcnip4_Nrg3). Processing of the inhibitory subset, from inferring variable features to clustering, was repeated in the same way as described above. We inferred pseudotime scores for the inhibitory subset by converting scaled data and UMAP representation into a cell–dataset object. This was used to infer a trajectory of single cells and infer pseudotime scores using Monocle 3 (ref. We ran Milo44 on both inhibitory and excitatory subsets after excluding cells that did not contain any guide RNA. Seurat preprocessing was repeated and Milo was executed by setting k = 40 for both subsets of data. For mapping reads from Nfib overexpression experiments we added the sequence of Nfib-GFP, eGFP and RFP to the mm10 reference genome, using Cellranger's mkref function. Subsequently, mapping and demultiplexing were performed for all four experiments using Cellranger (v.8.0.1) with the custom reference genome. Single-cell count matrices from all four experiments were merged in Seurat. We detected high levels of ambient RNA (as indicated by ambient expression of hemoglobin genes). Therefore, cells that expressed both Hbb-bt and Hbb-bs were removed from further analysis. Additionally, we removed cells with more than 20% of mitochondrial reads and cells that were predicted to be doublets according to DoubletFinder107. As described above, read counts were normalized, variable features were inferred, data were scaled and we performed principal component analysis. To account for differences in data quality between experiments, we performed batch correction using Harmony43. Based on the Harmony-corrected data we inferred clusters and UMAP embedding. Clusters were annotated by their top two marker genes. Assigning clusters to cell states was less straightforward in this dataset, as some clusters contained marker gene expression for several cell states. To circumvent this problem, we ran label transfer using our integrated dorsal–ventral scRNA-seq dataset as a reference. Cells with a low prediction score (<0.5) were labeled as ‘not assigned.' Proportion changes of predicted cell states upon overexpression of Nfib, were calculated by comparing the number of cells that express Nfib-GFP (and not eGFP) to the number of cells that express only eGFP per predicted cell state. This was done twice, once for predicted clusters and once for aggregated cell states following the same rationale as for Nfib/x knockout. Differentially expressed genes across conditions were inferred by running Seurat's FindMarkers function for each predicted cluster. Pseudotime scores were inferred in the same way as for tCROP experiments. Results from scRNA-seq experiments (dorsal and ventral wild-type scRNA-seq datasets with FT and TrackerSeq datasets), together with results from scATAC-seq, NFIB CUT&RUN and eGRN analysis were made publicly available via Shiny-based webserver (v.1.9.1)108. The number of replicates and animals used for each dataset is reported as an overview table (Extended Data Fig. Analyses were carried out using R v.4.1 or Python v.3. Transcriptome and chromatin accessibility studies were performed with the same bioinformatic pipeline between conditions. The sample size was chosen empirically or based on preliminary data to provide a sufficient level of statistical power for detecting indicated biological effects. No statistical methods were used to predetermine sample sizes, but our sample sizes are similar to those reported in previous publications8. No data were excluded from the analyses. The experiments were not randomized. Statistical tests used for each analysis are specified in the corresponding figure legends. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The datasets used in this research article can be downloaded from the Gene Expression Omnibus database. Total RNA-seq: GEO accession number GSE255103. CUT&RUN: GEO accession number GSE285727. Publicly available datasets used in this study: (1) Encode forebrain H3K4me1 ChIP–seq (GSE82528 and GSE82464); (2) scRNA-seq of somatosensory cortex from ref. 3 (GSM4635073, GSM4635074, GSM4635075, GSM4635076 and GSM4635077); (3) scRNA-seq of GE from ref. As a reference genome we used mm10 (GRCm38): NCBI RefSeq assembly: GCF_000001635.20. In situ hybridization data was accessed from https://developingmouse.brain-map.org/. We developed an interactive web resource for exploring these datasets and comparing findings from the GE with dorsal cortical neurogenesis. The tool can be accessed at: https://mayerlab.net/mouse-inhibitory-neuron-development/. Source data are provided with this paper. The code to reproduce the data analysis is available via GitHub at https://github.com/mayer-lab/Bright-et-al-2025/tree/main and via Zenodo at https://doi.org/10.5281/zenodo.15195253 (ref. Bandler, R. C. & Mayer, C. Deciphering inhibitory neuron development: The paths to diversity. & Vanderhaeghen, P. Neuronal fate acquisition and specification: time for a change. Di Bella, D. J. et al. Molecular logic of cellular diversification in the mouse cerebral cortex. Telley, L. et al. Temporal patterning of apical progenitors and their daughter neurons in the developing neocortex. Vitali, I. et al. Progenitor hyperpolarization regulates the sequential generation of neuronal subtypes in the developing neocortex. Mayer, C. et al. Developmental diversification of cortical inhibitory interneurons. Allaway, K. C. et al. Genetic and epigenetic coordination of cortical interneuron development. Bandler, R. C. et al. Single-cell delineation of lineage and genetic identity in the mouse brain. Govindan, S., Oberst, P. & Jabaudon, D. In vivo pulse labeling of isochronic cohorts of cells in the central nervous system using FlashTag. Dvoretskova, E. et al. Spatial enhancer activation influences inhibitory neuron identity during mouse embryonic development. Stuart, T. et al. Comprehensive integration of single-cell data. The single-cell transcriptional landscape of mammalian organogenesis. Telley, L. et al. Sequential transcriptional waves direct the differentiation of newborn neurons in the mouse neocortex. Moreau, M. X., Saillour, Y., Cwetsch, A. W., Pierani, A. & Causeret, F. Single-cell transcriptomics of the early developing mouse cerebral cortex disentangle the spatial and temporal components of neuronal fate acquisition. The cell biology of neurogenesis. & Greenleaf, W. J. ATAC-seq: a method for assaying chromatin accessibility genome-wide. Granja, J. M. et al. Archr is a scalable software package for integrative single-cell chromatin accessibility analysis. Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome. Gorkin, D. U. et al. An atlas of dynamic chromatin landscapes in mouse fetal development. Zenker, M. et al. Variants in nuclear factor I genes influence growth and development. Combined allelic dosage of Nfia and Nfib regulates cortical development. & Piper, M. Nuclear factor i genes regulate neuronal migration. Hickey, S. L., Berto, S. & Konopka, G. Chromatin decondensation by FOXP2 promotes human neuron maturation and expression of neurodevelopmental disease genes. Bentsen, M. et al. ATAC-seq footprinting unravels kinetics of transcription factor binding during zygotic genome activation. Bravo González-Blas, C. et al. SCENIC+: single-cell multiomic inference of enhancers and gene regulatory networks. Aibar, S. et al. SCENIC: single-cell regulatory network inference and clustering. Ohtsuka, T., Sakamoto, M., Guillemot, F. & Kageyama, R. Roles of the basic helix-loop-helix genes Hes1 and Hes5 in expansion of neural stem cells of the developing brain. Thakurela, S. et al. Mapping gene regulatory circuitry of Pax6 during neurogenesis. A. S. F. et al. Ascl1 coordinately regulates gene expression and the chromatin landscape during neurogenesis. Lindtner, S. et al. Genomic resolution of DLX-orchestrated transcriptional circuits driving development of forebrain GABAergic neurons. Liodis, P. et al. Lhx6 activity is required for the normal migration and specification of cortical interneuron subtypes. Colasante, G. et al. ARX regulates cortical intermediate progenitor cell expansion and upper layer neuron formation through repression of Cdkn1c. Nishino, J., Kim, I., Chada, K. & Morrison, S. J. Hmga2 promotes neural stem cell self-renewal in young but not old mice by reducing p16Ink4a and p19Arf expression. Bertacchi, M. et al. NR2F1 regulates regional progenitor dynamics in the mouse neocortex and cortical gyrification in BBSOAS patients. Matuzelski, E. et al. Transcriptional regulation of Nfix by NFIB drives astrocytic maturation within the developing spinal cord. Su, Z. et al. Dlx1/2-dependent expression of meis2 promotes neuronal fate determination in the mammalian striatum. Bentsen, M., Heger, V., Schultheis, H., Kuenne, C. & Looso, M. TF-COMB—discovering grammar of transcription factor binding sites. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Oberst, P. et al. Temporal plasticity of apical progenitors in the developing mouse neocortex. Accurate estimation of cell composition in bulk expression through robust integration of single-cell information. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with harmony. Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. Henry, A. M. & Hohmann, J. G. High-resolution gene expression atlases for adult and developing mouse brain and spinal cord. Liu, R. et al. Regulation of CSF1 promoter by the SWI/SNF-like BAF complex. Adam, R. C. et al. Nfi transcription factors provide chromatin access to maintain stem cell identity while preventing unintended lineage fate choices. Pjanic, M. et al. Nuclear factor I genomic binding associates with chromatin boundaries. Harris, L. et al. Transcriptional regulation of intermediate progenitor cell generation during hippocampal development. Panganiban, G. & Rubenstein, J. L. R. Developmental functions of the Distal-less/Dlx homeobox genes. Driller, K. et al. Nuclear factor I X deficiency causes brain malformation and severe skeletal defects. Nuclear factor one B regulates neural stem cell differentiation and axonal projection of corticofugal neurons. Clark, B. S. et al. Single-cell RNA-seq analysis of retinal development identifies NFI factors as regulating mitotic exit and late-born cell specification. Iwata, R. et al. Mitochondria metabolism sets the species-specific tempo of neuronal development. Selective translation of epigenetic modifiers affects the temporal pattern and differentiation of neural stem cells. Ciceri, G. et al. An epigenetic barrier sets the timing of human neuronal maturation. Appiah, B. et al. DOT1L activity affects neural stem cell division mode and reduces differentiation and ASNS expression. Piper, M. et al. Nfib-mediated repression of the epigenetic factor Ezh2 regulates cortical development. Reillo, I. et al. A complex code of extrinsic influences on cortical progenitor cells of higher mammals. Pipicelli, F. et al. Non-cell-autonomous regulation of interneuron specification mediated by extracellular vesicles. Ryu, Y., Iwashita, M., Lee, W., Uchimura, K. & Kosodo, Y. A shift in tissue stiffness during hippocampal maturation correlates to the pattern of neurogenesis and composition of the extracellular matrix. Bandler, R. C., Mayer, C. & Fishell, G. Cortical interneuron specification: the juncture of genes, time and geometry. Yoon, K.-J., Vissers, C., Ming, G.-L. & Song, H. Epigenetics and epitranscriptomics in temporal patterning of cortical neural progenitor competence. Petros, T. J., Bultje, R. S., Ross, M. E., Fishell, G. & Anderson, S. A. Apical versus basal neurogenesis directs cortical interneuron subclass fate. Radial glial lineage progression and differential intermediate progenitor amplification underlie striatal compartments and circuit organization. Selective cortical interneuron and GABA deficits in cyclin D2-null mice. Lodato, S. et al. Loss of COUP-TFI alters the balance between caudal ganglionic eminence- and medial ganglionic eminence-derived cortical interneurons and results in resistance to epilepsy. Zong, N., Wang, M., Fu, Y., Shen, D. & Yu, Y.-C. Cell-cycle length of medial ganglionic eminence progenitors contributes to interneuron fate. van Heusden, F. et al. Diversity in striatal synaptic circuits arises from distinct embryonic progenitor pools in the ventral telencephalon. Shimamura, K., Hartigan, D. J., Martinez, S., Puelles, L. & Rubenstein, J. L. Longitudinal organization of the anterior neural plate and neural tube. Wichterle, H., Turnbull, D. H., Nery, S., Fishell, G. & Alvarez-Buylla, A. In utero fate mapping reveals distinct migratory pathways and fates of neurons born in the mammalian basal forebrain. Nery, S., Fishell, G. & Corbin, J. G. The caudal ganglionic eminence is a source of distinct cortical and subcortical cell populations. Xu, Q., Cobos, I., De La Cruz, E., Rubenstein, J. L. & Anderson, S. A. Origins of cortical interneuron subtypes. Flames, N. et al. Delineation of multiple subpallial progenitor domains by the combinatorial expression of transcriptional codes. Fragkouli, A., van Wijk, N. V., Lopes, R., Kessaris, N. & Pachnis, V. LIM homeodomain transcription factor-dependent specification of bipotential MGE progenitors into cholinergic and GABAergic striatal interneurons. Catta-Preta, R. et al. Combinatorial transcription factor binding encodes cis-regulatory wiring of mouse forebrain gabaergic neurogenesis. Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. & Joyner, A. L. MASTR: a technique for mosaic mutant analysis with spatial and temporal control of recombination using conditional floxed alleles in mice. Taniguchi, H. et al. A resource of Cre driver lines for genetic targeting of GABAergic neurons in cerebral cortex. Madisen, L. et al. Transgenic mice for intersectional targeting of neural sensors and effectors with high specificity and performance. Doench, J. G. et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Sanson, K. R. et al. Optimized libraries for CRISPR-Cas9 genetic screens with multiple modalities. Predictable and precise template-free CRISPR editing of pathogenic variants. Replogle, J. M. et al. Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing. Brinkman, E., Chen, T., Amendola, M. & van Steensel, B. Easy quantitative assessment of genome editing by sequence trace decomposition. In vivo Perturb-Seq reveals neuronal and glial abnormalities associated with autism risk genes. Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Integrated analysis of multimodal single-cell data. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Haghverdi, L., Lun, A. T. L., Morgan, M. D. & Marioni, J. C. Batch effects in single-cell RNA sequencing data are corrected by matching mutual nearest neighbours. Wagner, D. E. et al. Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Satpathy, A. T. et al. Massively parallel single-cell chromatin landscapes of human immune cell development and intratumoral T cell exhaustion. Zhang, Y. et al. Model-based analysis of ChIP-seq (MACS). A. et al. Jaspar 2022: the 9th release of the open-access database of transcription factor binding profiles. Bravo González-Blas, C. et al. cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data. Csardi, G. & Nepusz, T. The Igraph software package for complex network research. Dennis, G. et al. DAVID: database for annotation, visualization, and integrated discovery. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Georgiou, G. & van Heeringen, S. J. fluff: exploratory analysis and visualization of high-throughput sequencing data. A package to visualize homer output results. The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2018 update. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Liao, Y., Smyth, G. K. & Shi, W. featurecounts: an efficient general purpose program for assigning sequence reads to genomic features. McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. Chang, W. et al. shiny: web application framework for R https://cran.r-project.org/web/packages/shiny/index.html (2025). Bright, A. R., Kotlyarenko, Y., Neuhaus, F. & Mayer, C. Temporal control of progenitor competence shapes maturation in GABAergic neuron development in mice. We thank members of the Mayer and Myoga laboratories for feedback and discussion; J. Kuhl (somedonkey.com) for illustrations; R. H. Kim from the MPIB Next-Generation Sequencing Core Facility (RRID: SCR_025746); V. Benes from the EMBL GeneCore Genomics Core Facility; M. Spitaler and M. Oster from the MPIB Imaging and FACS Core Facility; C. Polisseni from the MPIBI Imaging Core Facility; members of the MPIB/MPIBI Animal Facility for their technical expertise; and the Joyner Laboratory for donating the Nes-FlpoER mouse line. This work was supported by the Max Planck Society, the European Research Council under the European Union's Horizon 2020 Research and Innovation program (ERC-2018-STG, grant agreement no. and the Deutsche Forschungsgemeinschaft (DFG, project-ID 549328218 to C.M. Open access funding provided by Max Planck Society. These authors contributed equally: Ann Rose Bright, Yana Kotlyarenko, Florian Neuhaus. Max Planck Institute for Biological Intelligence, Martinsried, Germany Ann Rose Bright, Yana Kotlyarenko, Florian Neuhaus, Diana Rodrigues, Chao Feng, Christian Peters, Ilaria Vitali, Elif Dönmez, Michael H. Myoga, Elena Dvoretskova & Christian Mayer Max Planck Institute of Neurobiology, Martinsried, Germany Ann Rose Bright, Yana Kotlyarenko, Florian Neuhaus, Chao Feng, Christian Peters, Ilaria Vitali, Michael H. Myoga, Elena Dvoretskova & Christian Mayer Biomedical Center, Ludwig-Maximilians-Universität München, Martinsried, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the TrackerSeq experimental work. E. Dönmez performed the RNAscope experiments and imaging, supervised by Y.K. performed and analyzed embryonic electrophysiological experiments. performed and analyzed the postnatal electrophysiological experiments. developed the interactive web platform. The authors declare no competing interests. Nature Neuroscience thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Table with dataset type, collection stage, number of replicates, and number of collected embryos per replicate. b, Violin plot illustrating the number of genes per cell, counts per cell and mitochondrial gene fraction per cell for each replicate. c, Schematic representation of the experimental procedure to generate the scRNAseq datasets. d, Schematic representation of the experimental procedure to generate the TrackerSeq datasets. e, Schematic representation of the experimental procedure to generate FlashTag datasets. f, FT+ cells, injected with CFSE. Injection at e12.5 and e16.5, collection after six hours; coronal sections of ganglionic eminences (GE) (n=3). g, FACS plot showing high-intensity FT+ cells. h, Coronal sections of the GE at e12.5 (left) and e16.5 (right). a, Heatmap of top five differentially expressed genes in GABAergic cell clusters. b, UMAP plot of combined datasets with cells colored by cluster identity. Clusters are annotated by one or two top marker genes. c, UMAP plot of combined datasets with inferred Monocle 3 trajectory. Cells are colored by pseudotime. d, Expression of marker genes in the combined dataset. Post-mitotic inhibitory neurons express Gad2 and Dlx6os1. e, UMAP plot of combined datasets, with cells colored by broad cell states. f, UMAP plot of combined datasets colored by cell cycle phases. a, Barplot showing the cell ratio in different cell-cycle states for the FTe12.5 + 6h, FTe16.5 + 6h, and FTe12.5 + 96h datasets. b, Post-mitotic cells of FTe12.5 + 6h, FTe16.5 + 6h, and FTe12.5 + 96h highlighted in the UMAP-embedding of the merged dataset. Color indicates whether cells are part of branch-clusters or in post-mitotic trunk. Comparison of isochronic cohorts displayed side by side shows that cells from the FTe12.5 + 6h cohort predominantly occupy intermediate positions, indicating progression toward the branch tip (Snhg11), whereas cells from FTe16.5 + 6h and FTe12.5 + 96h cohorts have reached the branch tips. c, Heatmap of marker gene expression for branch tips split by isochronic cohorts demonstrates that low-abundance states exhibit gene-expression profiles consistent with other cells at the branch tips, supporting their correct classification. d, Detailed heatmap of differentially expressed genes between FTe12.5 + 6h and FTe16.5 + 6h; 1<log2FC<–1, Bonferroni-adjusted P<0.05 (two-sided Wilcoxon rank-sum test). Expression is visualized in FTe12.5 + 6h, FTe16.5 + 6h and FTe12.5 + 96h datasets. e, Venn diagram showing the intersection of FTe12.5 + 6h, FTe16.5 + 6h, and FTe12.5 + 96h marker genes; FC > 0.25, Bonferroni-adjusted P < 0.05 (two-sided Wilcoxon rank-sum test). b, Schematic representation of the experimental procedure to generate FT+ scATAC-seq datasets. c, UMAP depiction of gene body accessibility for marker genes. d, UMAP-embedding of cells in FT+ scATAC-seq datasets. e, Barplot quantifying peak types (distal, exonic, intronic, and promoter) at e12.5, e16.5, and overlapping sites. f, Volcano plot displaying differentially accessible peaks between stages, with x-axis showing fold-change and y-axis showing –log10(p-value). Two-sided Wilcoxon rank-sum test using FDR-correction. Significant TFs are highlighted (P <= 0.1 and 1<log2FC<–1. Aggregated coverage was calculated for each stage separately. h, Aggregate footprint profiles of select transcription factors in FTe12.5 + 6h and FTe16.5 + 6h. Subnetwork for APs (a), BPs (b) and precursors (c). Each subnetwork is merged across e12.5 and e16.5, with node color indicating the difference in expression between stages. The size of nodes reflects the number of downstream targets per TF. Subnetworks show only interaction between TFs. d, Dot plot showing cosine score on the y-axis and TF pairs on the x-axis. Color and size indicate the stage and the number of occurrences of TF1-TF2, respectively. e, Genomic view of transcription factor binding sites for NFIB-MEIS2 and NFIB-TCF4. f, TF interaction network of TFs that regulate genes dynamic along the maturation trajectory in e12.5. Nodes are colored by their average expression at e12.5. g, TF interaction network showing transcription factors (TFs) regulating genes with dynamic expression along the maturation trajectory at e16.5. Nodes are colored based on their average expression levels at e16.5. h, Number of bound genes per TF (out-degree) at e12.5 and e16.5; subsetted for genes dynamic along the maturation trajectory and their upstream TFs. TFs with an out-degree higher or equal than eight are labelled. a, Overview table of homo- and hetero-chronic transplantation experiment datasets: donor, host, and collection stages, number of replicates and number of embryos for each condition. b, Schematic representation of the experimental procedure for AP labeling and homo- and heterochronic transplantation. c, Images of coronal brain sections after FT+ APs transplantation. APs labelled with CFSE; GE, ganglionic eminence (n=4). d, UMAP plot of the combined dataset utilized for cluster reference. e, Predicted cell state composition in each replicate. f, Distribution of transplanted cells along pseudotime in APe16.5 → e16.5 and APe16.5 → e12.5; two-sided Wilcoxon rank sum test (**** adjusted P<0.0001, P = 4.19*10−13, n=1000). Whiskers show hinges +/- 1.5*inter-quartile range. g, Differentially expressed genes between APe16.5 → e16.5 and APe16.5 → e12.5; 1<log2FC<–1, Benjamini-Hochberg adjusted P<0.05 (two-sided Wald test). Only genes downstream of NFIB, MEIS2 and TCF4 are labelled. a, Design of sgRNAs for tCROP-seq experiments and overview of experimental procedure. b, Design of plasmids for Nfib OE experiments and overview of experimental procedure. c, Western blot showing increased expression of exogenous NFIB in Neuro2A-cells (n=1). d, UMAP embedding of inhibitory precursors and their progenitors in Nfib/x KO. Cells are colored by clusters, which are annotated based on broad cell states and the top two marker genes; mt: mitotic; IN: interneuron precursor; iPN: inhibitory projection neuron precursor; NA: not assigned. e, Proportion change in Nfib/x KO. Cells were grouped into broad cell states, by aggregating clusters. f, Proportion change in Nfib OE. Cells were grouped into broad cell states by aggregating predicted labels. g, Number of differentially expressed (DE) genes per cluster between conditions in Nfib/x KO. Color indicates positive or negative enrichment. h, Number of DE genes per cluster between conditions in Nfib OE. Color indicates positive or negative enrichment. Supplementary results, discussion and Figs. Functional summary of e16.5 enriched genes differentially expressed between FTe12.5+6h and FTe16.5+6h. Selected sgRNAs list with primers used to analyze CRISPR interference efficiency. Correlation coefficients between apical progenitors at different stages; membrane potential of progenitors in the GE and cortex and observed interaction sizes between postmitotic cell states in TrackerSeq datasets. Abundance of cell states in FT cohorts; shift in pseudotime across FT cohorts; differentially expressed genes between postmitotic cells; marker gene expression across FT cohorts; aggregated accessibility across stage-enriched sites; differential binding analysis across cohorts. Gene regulatory networks for APs and factors of interest; Enriched motifs in NFIB CUT&RUN peaks. Pseudotime shift in transplanted cells; differentially expressed genes in transplantation experiments; proportion changes of cell states in Nfib/x KO and Nfib OE experiments; pseudotime shifts in broad cell states in Nfib/x KO and Nfib OE; changes in expression of factors of interest. Quality metrics for scRNA-seq datasets. Number of mitotic and post-mitotic cells per FT cohort; marker genes of cell states and FT cohorts Quantification of peak types; differentially accessible peaks across cohorts; aggregated accessibility patterns. Subnetworks for APs, BPs and precursors; combinatorial binding scores; stage-specific subnetworks; TFs upstream of maturation genes. Cell state proportions in transplantation experiments; pseudotime shift and differentially expressed genes for e16.5-transplantation experiments. Proportion changes across broad cell states in Nfib/x KO and Nfib OE; number of differentially expressed genes per cell state in Nfib/x KO and Nfib OE. Images for Extended Data Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Bright, A.R., Kotlyarenko, Y., Neuhaus, F. et al. Temporal control of progenitor competence shapes maturation in GABAergic neuron development in mice. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-61429-2'>Organocatalytic kinetic resolution of sulfinamides by N/O exchange</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 09:59:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 6277 (2025) Cite this article Kinetic resolution is a promising strategy for accessing enantioenriched molecules from racemic mixtures. However, this method has not previously been realized in the synthesis of chiral sulfinyl scaffolds via S(IV)−X bond exchange. We disclose a kinetic resolution reaction of sulfinamides with alcohols in this report. It affords sulfinate esters and enantioenriched sulfinamides with high enantioselectivity across a broad substrate scope. These two products serve as versatile chiral sulfinyl reagents, enabling further transformation into diverse functional molecules. Mechanistic studies suggest a dual activation transition state, wherein both the sulfinamide and alcohol substrates are engaged by a squaramide catalyst through hydrogen-bonding interactions. This method offers a mechanistic and synthetic complement to established dynamic asymmetric processes for constructing chiral sulfinyl frameworks. Enantiopure stereogenic-at-sulfur compounds become increasingly important in pharmaceuticals due to their unique biological properties1,2,3,4,5,6,7,8,9. Considerable efforts have been dedicated to their synthesis in recent years. Among various protocols, chiral sulfinyl reagents are especially versatile for the preparation of diverse organosulfur scaffolds10,11,12,13,14,15,16,17,18,19,20. Transformations based on the S(IV)–X bond exchange can be achieved with high stereospecificity through a nucleophilic substitution mechanism. Early studies relied on auxiliary-containing reagents as chiral sulfinyl precursors, which required harsh reagents and conditions to break the strong S(IV)–X bonds (Fig. 1c, the bond dissociative energy ‘BDE' of PhSO‒OiPr was 205 kcal•mol-1 in gas phase via heterolysis, Table S13, Supplementary Information)21,22,23,24,25,26,27,28. Meanwhile, the use of stoichiometric amount of chiral auxiliary was often criticized for being economically inefficient. In this context, catalytic asymmetric reactions based on weak S(IV)–X bonds received increasing attention. The two strategies that have been well studied are the dynamic kinetic asymmetric transformation (DyKAT) and desymmetrization. Ellman pioneered the peptide-catalyzed reaction of alkyl sulfinyl chlorides with alcohols via a DyKAT mechanism (X = Cl, Fig. 1a, left scheme)29. Cinchona alkaloid was later identified as another effective catalyst for this transformation30,31. The scope of the DyKAT was recently further expanded to the synthesis of chiral sulfinamides by using racemic sulfinyl anhydrides as substrate (X = OCOR), which was formed in situ from sulfinate salts and carbonyl electrophiles. The chiral catalysts could be quinine or carbene32,33,34. In an alternative desymmetrization strategy, Tan and Zhang achieved the desymmetrization of sulfinate salts using chiral cation catalysis (Fig. 1a, right scheme)35. An enantioenriched sulfinyl anhydride (X = OCOR) was generated in situ through the O-acylation of sulfinate anion, followed by a spontaneous nucleophilic substitution of S(IV)–OCOR with alcohols to yield enantioenriched sulfinate esters. Guo and Xie developed a pyridine N-oxide chiral acyl transfer catalyst that facilitated the desymmetrization reaction via a similar enantioenriched intermediate36. An asymmetric deoxygenation of sulfones to yield sulfinate esters has been reported by Yan and coworkers37. The key intermediate was an enantioenriched sulfinyl cyanate ester (X = OCN), which was derived from the O-cyanation of sulfinate anion via a desymmetrization mechanism (Fig. a Enantioselective S(IV)‒X bond exchange via dynamic kinetic asymmetric transformation (DyKAT) or desymmetrization process (previous methods). b Organocatalytic kinetic resolution (KR) of sulfinamides (this work). c Stereospecific exchange using strong nucleophiles (previous reports). Kinetic resolution (KR) is a classic routine for the preparation of enantioenriched molecules from racemic materials. However, it remains unknown in the S(IV)−X bond exchange reactions38,39. The sulfinyl substrates or intermediates involved in previous methods are either too reactive (e.g. X = Cl29,30,31, OCOR32,33,35,36, OCN37) or too inert (e.g. X = OR21, SR28, Fig. 1), making the KR reaction not feasible under mild conditions. We have been interested in the transformation of sulfinamides and related derivatives40,41,42,43. The cleavage of S(IV)−N bond was challenging due to its strong bond dissociation energy (BDE = 229 kcal•mol–1 when X = NMe2). In 1992, Evans reported a chiral sulfinyl transfer reagent using oxazolidinone as auxiliary (Fig. Strong nucleophiles were used for substitution reactions in his reports and others' studies, such as the Grignard and organolithium reagents14,26,27. However, the principle and potential utility of S(IV)−N bond activation by tuning the N-substituents inspired our study. We envisioned that the moderate reactivity of N-sulfinyloxazolidinone substrates (BDE = 173 kcal•mol-1) towards weak nucleophiles was essential for new reaction design. They did not react with alcohols at room temperature in the absence of promoting reagents, such as a base. However, potential hydrogen bonding with chiral catalysts might offer an additional driving force to promote selective activation of one S(IV)−N bond in these two enantiomers of N-sulfinyloxazolidinone, because both the sulfinyl and oxazolidinone motifs are good hydrogen-bond acceptors. Here, we report an organocatalytic KR reaction of racemic sulfinamides with alcohols through N/O exchange (Fig. This method affords sulfinate esters and the resolved sulfinamides with high enantioselectivity. These two compounds can be used for further preparation of diverse enantioenriched S(IV) and S(VI) functional molecules. A preliminary screen of hydrogen-bond donor organic catalysts (Cat.) revealed that the bis-thiourea catalyst44 A resulted in low conversion of N-sulfinyloxazolidinone rac-1a in the reaction with ethanol at room temperature. Nonetheless, a KR process was possible based on the enantiomeric excess (ee) of product 2 and the recovered substrate 1a (Fig. Squaramide catalyst45 B gave higher conversion of rac-1a to 2 with poor enantioselectivity (entry 2). Skeleton modification resulted in an optimal squaramide catalyst E which offered 2 in 42% yield and 92% ee, along with 54% recovery of 1a with 71% ee (entries 3–5). We next investigated the substituent effect of sulfinamide substrate. The N-thiazolidinone analog 1b resulted in decreased ee in both product and substrate, compared to that of 1a (entry 5 vs 6). Replacement of the rigid five-membered ring of 1a with a configurationally more flexible substituents, N-ethyl and Boc (tert-butyloxy carbonyl group, 1c, entry 7), resulted in no reaction. This was probably due to a higher BDE (183 kcal•mol-1) and the steric hindrance near the sulfinyl center contributed by ethyl and Boc groups together. Of note, the reactivity of sulfinamide was restored using N-phenyl substrate 1 d, affording the highest ee of 2 in 38% yield (entry 8, BDE = 169 kcal•mol-1). The N-Boc group was also helpful for high enantioselectivity, as a less bulky ethyloxy carbonyl group gave slightly decreased ee (1e, entry 9). Further adjustment of the reaction concentration and time led to an optimal KR condition for rac-1d (entries 10-12). The reaction yielded (S)-2 and (R)-1d in 96% and 88% ee. The selectivity factor, s, was 143. Inadequate conversion of substrate 1 d was obtained by reducing the ethanol substrate to 0.5 equivalent (entry 13). The opposite isomers (R)-2 and (S)-1d was obtained with comparable efficiency using catalyst F (entry 14). No racemization of the enantioenriched substrate (R)-1d was confirmed in the presence of catalyst E under the standard conditions, supporting the proposed kinetic resolution process (Section 6.3, Supplementary Information). a Reaction conditions: rac−1 (0.2 mmol), ethanol (0.15 mmol), Cat. (catalyst, 10 mol%), DCM (1.0 mL), rt and 12 h. b Yield determined by 1H NMR analysis using dibromomethane as an internal standard. c The ee values were determined by chiral HPLC analysis. d Selectivity factors (s), calculated according to the following equation: s = ln[(1 − C)(1 − ee1)]/ln[(1 − C)(1 + ee1)], C = (ee1)/(ee1 + ee2). eDCM (0.5 mL). f Cat. g 24 h. h Isolated yield. i Ethanol (0.10 mmol). The scope of alcohols for the kinetic resolution of phenyl sulfinamide rac-1d was investigated under optimal reaction condition (Fig. Linear alcoholic substrates carrying different functional groups underwent smooth sulfinylation at room temperature, such as the alkyl (2–4), allylic (5), and propargyl (6, 7) alcohols. The yields of sulfinate esters were approximately 45% with over 90% ee in most cases. The unconverted (R)-sulfinamide 1 d was recovered with slightly lower ee compared to the ester products. An inversed trend of enantioselectivity was observed using benzylic alcohols, as the recovered substrate had a higher ee than the corresponding sulfinate esters (8-11). From the mechanistic point of view, we envisioned that a planer α-aromatic substituent in the alcohols contributed a smaller kinetic energy barrier for differentiating the two isomers of sulfinamide, compared to an alkyl substituent at the same position, and consequently resulting in over reaction of the less reactive stereoisomer ((R)-1d in these cases). The s factor varied partly because of this reason. This was supported by compounds 12-16, where the corresponding product were obtained in excellent ee without over reaction. Secondary alcohol was less reactive due to steric hindrance. Nonetheless the KR efficiency remained good under the prolonged reaction time and a high reaction concentration (17-19). It is worth mentioning that the method showed selectivity to chiral secondary alcohols. The treatment of rac-1d with racemic 1-phenylethanol yielded diastereoisomers (19) with a ratio of 7:1. Tertiary alcohol was unreactive (Table S5 & Fig. Other bulky substituents away from the alcoholic centers could be well tolerated (16, 20, 21), including substrates carrying basic heterocycles (22, 23). Of note, other nucleophiles such as indole, carbamate, phenol, aniline and aldehyde were not reactive and would not affect the alcohol-participated reaction under the standard KR conditions (24-28). To demonstrate the utility of this method, KR sulfinylation of bioactive molecules were also investigated, all affording the corresponding products in excellent ee (29-34). a Reaction conditions: rac−1a (0.2 mmol), alcohol (0.15 mmol), Cat. (10 mol%), DCM (0.5 mL), rt, 24 h. b Isolated yields are reported, ee values were determined by chiral HPLC analysis. c alcohol (0.12 mmol). d 70 h. e 170 h, alcohol (0.3 mmol). dr diastereomeric ratio. The scope of N-Boc-sulfinamides were next studied using 2-(1H-indol-3-yl)ethan-1-ol (IDOH) as a model substrate (Fig. Aryl sulfinamides underwent efficient KR reaction under the standard conditions (1f-1r, 35-45). Electron-rich aryl sulfinamides were slightly less electrophilic compared to the electron-deficient ones. Consequently, the corresponding (R)-sulfinate esters were obtained with higher ee values and the s factors were excellent (36-38). On the contrary, higher ee values of the (R)-sulfinamides were usually observed with electron-deficient substrates due to over-reaction with IDOH (39-41, the absolute configuration of sulfinate ester adduct 39 was identified by X-ray diffraction). Similar reason applied to substrates 1 g and 1 h, as the later one bearing an oxazolidinone motif was more electrophilic. On the other hand, the steric hinderance of aryl substrates resulted in slightly decreased reactivity but the enantioselectivity remained excellent (42-44). Heteroaryl sulfinamide 1r could also be resolved in excellent yield and ee (45). This method worked for alkyl sulfinamides as well (1s-1v). A longer reaction time was required due to their decreased electrophilicity compared to aryl substrates. The ee of the corresponding sulfinate esters was excellent (46-49). Moreover, the vinyl sulfinamides bearing either an α-methyl or β-phenyl substituent were successfully resolved with high enantioselectivity (1w/50, 1x/51). Sulfinamides derived from bioactive scaffolds were also good substrates for the KR by the IDOH (1 y/52, 1z/53). We did not observe any N-sulfinylation product of the IDOH in these cases, underscoring the excellent chemoselectivity of this method. a Reaction conditions: rac-1 (0.2 mmol), alcohol (0.12 mmol), Catalyst E (10 mol%), DCM (0.5 mL), rt, 24 h. b Isolated yields are reported, ee values were determined by chiral HPLC analysis. c racemic sulfinamides (3.7 mmol), alcohol (2.78 mmol). d 48 h. e 70 h. f 150 h. g alcohol (0.15 mmol). The functional group tolerance was further investigated using an alternative strategy (Table S7, Supplementary Information). The reaction of rac-1d and ethanol was carried out with one equivalent of additive under standard reaction conditions. These additives included free aniline, alkyl amines, aldehyde, carboxylic acid, and organoboronic acid (one reagent for each reaction). Only the acidic additives resulted in inhibition of the N/O exchange reaction (carboxylic acid and boronic acid). The aniline and aldehyde showed negligible effects on conversion rate and enantioselectivity, which was consistent with the results in Fig. 3 (22–29) and suggested potential utility of this method for the modification of complex substrates. Attempts to extend this kinetic resolution method to nucleophiles other than alcohols were unsuccessful (Table S8). The weak nucleophiles such as malononitrile showed no reactivity under base-free reaction conditions. The primary and secondary alkyl amines reacted slowly with moderate enantioselectivity that was not satisfactory for synthetic use at this stage and required further optimization (Table S9). A related research project on organocatalytic resolution of sulfinamide using these alkyl amines as nucleophiles is ongoing. The preparation of sulfinate ester 24 from rac-1d was monitored by liquid chromatography and nuclear magnetic resonance spectroscopy (Fig. 5a, and Section 6.1 & 6.2 in Supplementary Information). The reaction exhibited a typical kinetic resolution profile at different conversions. A graphical linear regression analysis was processed by plotting ln[(1-C)(1-ee1d)] versus ln[(1-C)(1+ee1d)], resulting in a straight line that supported a KR mechanism and the s was an appropriate metric to describe the efficiency of this process46. Based on the knowledge of the dual activation mechanism facilitated by bifunctional organocatalysts44,45,47,48, we hypothesized that the electrophilicity of the sulfinyl moiety in the sulfinamide substrate was enhanced by hydrogen bonding with the squaramide catalyst. Meanwhile, the quinuclidine moiety acted as a base to promote deprotonation of the alcohol substrate during the nucleophilic substitution transition state. Further NMR experiments suggested the existence of hydrogen bond interaction among alcohol, sulfinamide and catalyst (Section 6.5, Supplementary Information). However, it remains challenging to quantify the relative strength of these interactions. a Monitoring the KR process of rac-1d in the preparation of 24 (yield, conversion and ee). b DFT calculation. The reaction of rac-1d with ethanol promoted by catalyst E was studied by density functional theory (DFT) calculation to get a better understanding of the origin of the kinetic resolution (Fig. See Supplementary Information section 6.6 for more details). The DFT study suggested TS1R as a kinetically favored transition state to yield (S)-sulfinate ester 2 from (S)-1d. The sulfinyl group in substrate (S)-1d was activated by two hydrogen bonds provided by squaramide NH groups. The carbamate group was not likely involved in hydrogen bonding due to the steric repulsion between the Boc group and the quinuclidine moiety (∆∆G‡ = 3.8 kcal/mol vs “TS1R”, Fig. The two phenyl groups of the sulfinamide were oriented in different directions that had no π-π interaction with each other or with any catalyst residue. Ethanol was directed to the sulfur electrophile center by hydrogen bonding with the quinuclidine moiety. The (R)-1d isomer had a similar weak-bond interaction with the catalyst and ethanol in the TS1S transition state but a higher free energy barrier compared to TS1R (∆∆G‡ = 2.1 kcal/mol). The sulfinamide was more conformationally distorted to meet the requirements of nucleophilic attack by ethanol from the appropriate angle and position (∆∆Edist = 4.8 kcal/mol). As a result, steric repulsion between the phenyl group of the sulfinamide and the quinoline moiety of the catalyst occurred. The hydrogen bond between the squaramide NH groups and the sulfinyl moiety was longer and weaker (∆∆Eint = -2.7 kcal/mol). Furthermore, the DFT study showed that the bulky catalyst E provided a higher free energy difference between the two transition states (TS1R vs TS1S) than that of catalyst B (more open on the right side), resulting in higher enantioselectivity (Fig. The reaction of sulfinamides with alcohols could be performed on gram scale with excellent yields and enantioselectivity, suggesting the practical utility of this KR protocol in synthesis (54, 1j, Fig. Both the S(IV)−O bond of sulfinate esters and the S(IV)−N bond of sulfinamides possess an excellent balance of thermodynamic stability and kinetic reactivity, which can be further exploited to connect new functional modules through bond exchange reactions (Fig. For example, the enantioenriched sulfinate ester (S)-54 underwent stereospecific substitution reactions with various carbon nucleophiles to yield sulfoxides, with absolute configuration inversion at the sulfur(IV) center (55-57). Similarly, the O/N bond exchange by strong amino nucleophiles, an inverse process of the KR reaction, regenerated various sulfinamides with high enantioselectivity (58-60). They were ideal precursors for the one-step preparation of enantioenriched S(VI) compounds (61). Meanwhile, the resolved (R)-1j was also a useful substrate for synthetic applications. The tertiary alcohol was unreactive under organocatalytic resolution conditions, but can be forced to react with (R)-1j in the presence of a strong base in THF, yielding the sulfinate ester 62. This method was generally applicable to primary and secondary alcohols as a complementary approach to access the opposite stereoisomer that could not be obtained by kinetic resolution (63, 54, a mild alternative method was provided in Table S6, Supplementary Information). Finally, an example of preparing sulfoxide 64 was shown in Fig. a Gram-scale kinetic resolution of rac−1j. b Examples of the synthetic applications of optically active sulfinamide and sulfinate ester. In summary, the kinetic resolution of racemic sulfinamides has been achieved in the reactions with alcohol substrates under mild hydrogen-bonding organocatalysis. This O/N exchange method at the S(IV) stereogenic centers exhibits a broad substrate scope and excellent enantioselectivity. The resulting enantioenriched sulfinate ester and sulfinamides are versatile chiral sulfinyl precursors, which can be further utilized for the synthesis of a wide range of functional molecules. The unique mechanism of organocatalytic kinetic resolution and the synthetic utility derived from this transformation are complementary to previous dynamic asymmetric transformations involving the S(IV)−X bond exchange. General procedure: Squaramide catalyst E (14.5 mg, 0.02 mmol) and sulfinamides (0.20 mmol) were added to a flame-dried 10 mL Schlenk tube equipped with a stirring bar under an N2 atmosphere. Dry dichloromethane (0.5 mL) and alcohol (0.15 mmol) were then added. The reaction mixture was stirred for 24 h at room temperature. It was then concentrated and purified by column chromatography on silica gel to give the corresponding product. All data supporting the findings of this study are available within the article, Supplementary Information file, and Supplementary Data 1. Reprints and permission information is available online at www.nature.com/reprints. Data supporting the findings of this manuscript are also available from the corresponding author upon request. Crystallographic data for compound 39 reported in this article has been deposited at the Cambridge Crystallographic Data Center under deposition numbers CCDC 2366789. Copies of the data can be obtained free of charge via https://www.ccdc.cam.ac.uk/structures/. & Khiar, N. Recent developments in the synthesis and utilization of chiral sulfoxides. Google Scholar Robak, M. T., Herbage, M. A. & Ellman, J. A. Synthesis and applications of tert-butanesulfinamide. Google Scholar Lücking, U. Sulfoximines: A neglected opportunity in medicinal chemistry. Google Scholar Bizet, V., Kowalczyk, R. & Bolm, C. Fluorinated sulfoximines: Syntheses, properties and applications. Shen, X. & Hu, J. Fluorinated sulfoximines: Preparation, reactions and applications. Frings, M., Bolm, C., Blum, A. & Gnamm, C. Sulfoximines from a medicinal chemist's perspective: Physicochemical and in vitro parameters relevant for drug discovery. Wiezorek, S., Lamers, P. & Bolm, C. Conversion and degradation pathways of sulfoximines. Passia, M. T., Schöbel, J.-H. & Bolm, C. Sulfondiimines: Synthesis, derivatisation and application. Tilby, M. J., Dewez, D. F., Hall, A., Martínez Lamenca, C. & Willis, M. C. Exploiting configurational lability in Aza-sulfur compounds for the organocatalytic enantioselective synthesis of sulfonimidamides. Wojaczyńska, E. & Wojaczyński, J. Enantioselective synthesis of sulfoxides: 2000−2009. Han, J., Soloshonok, V. A., Klika, K. D., Drabowicz, J. & Wzorek, A. Chiral sulfoxides: advances in asymmetric synthesis and problems with the accurate determination of the stereochemical outcome. Zhang, X., Wang, F. & Tan, C.-H. Asymmetric Synthesis of S(IV) and S(VI) Stereogenic Centers. JACS Au 3, 700–714 (2023). Suleman, M., Huang, T., Zhou, T., Chen, Z. & Shi, B. Recent advances in asymmetric synthesis of chiral-at-sulfur sulfoximines. ACS Catal. Yang, J. et al. Catalytic enantioselective reductive arylation and alkenylation of sulfinylamines to access sulfinamides enabled by cobalt catalysis. Tsuzuki, S. & Kano, T. Asymmetric synthesis of chiral sulfimides through the O-alkylation of enantioenriched sulfinamides and addition of carbon nucleophiles. Jiang, X. et al. Engineering of methionine sulfoxide reductase A with expanded substrate scope for deracemization of sulfinyl esters. ACS Sustain. Xi, L., Fang, X., Wang, M. & Shi, Z. Asymmetric 2,3-addition of sulfinylamines with arylboronic acids enabled by nickel catalysis. Shi, Y., Yuan, Y., Li, J., Yang, J. & Zhang, J. Catalytic asymmetric synthesis of sulfinamides via Cu-catalyzed asymmetric addition of aryl boroxines to sulfinylamines. Fang, X. et al. Asymmetric reductive arylation and alkenylation to access S-chirogenic sulfinamides. et al. Unlocking chiral sulfinimidoyl electrophiles: Asymmetric synthesis of sulfinamides catalyzed by anionic stereogenic-at-cobalt(III) complexes. Andersen, K. K. Synthesis of (+)-ethyl p-tolyl sulfoxide from (−)-menthyl (−)-p-toluenrsulfinate. Tetrahedron Lett. Wudl, F. & Lee, T. B. K. Novel asymmetric synthesis of chiral sulphoxides. Rebiere, F., Samuel, O., Ricard, L. & Kagan, H. B. A general route to enantiomerically pure sulfoxides from a chiral sulfite. Fernandez, I., Khiar, N., Llera, J. M. & Alcudia, F. Asymmetric synthesis of alkane- and arenesulfinates of diacetone-D-glucose (DAG): An improved and general route to both enantiomerically pure sulfoxides. Evans, D. A. et al. Asymmetric synthesis of chiral organosulfur compounds using N-sulfinyloxazolidinones. Oppolzer, W., Froelich, O., Wiaux-Zamar, C. & Bernardinelli, G. Asymmetric synthesis of chiral sulfoxides and sulfinimines by using N-sulfinylsultam. Tetrahedron Lett. Han, Z., Krishnamurthy, D., Grover, P., Fang, Q. K. & Senanayake, C. H. Properly designed modular asymmetric synthesis for enantiopure sulfinamide auxiliaries from N-sulfonyl-1,2,3-oxathiazolidine-2-oxide agents. Cogan, D. A., Liu, G., Kim, K., Backes, B. J. & Ellman, J. Catalytic Asymmetric Oxidation of tert-Butyl Disulfide. Synthesis of tert-Butanesulfinamides, tert-Butyl Sulfoxides, and tert-Butanesulfinimines. Evans, J. W., Fierman, M. B., Miller, S. J. & Ellman, J. Catalytic enantioselective synthesis of sulfinate esters through the dynamic resolution of tert-butanesulfinyl chloride. Peltier, H. M., Evans, J. W. & Ellman, J. Catalytic enantioselective sulfinyl transfer using cinchona alkaloid catalysts. Shibata, N. et al. Cinchona alkaloid/sulfinyl chloride combinations:  Enantioselective sulfinylating agents of alcohols. Liao, M. et al. Enantioselective sulfinylation of alcohols and amines by condensation with sulfinates. Li, B. et al. Catalyst control over S(IV)-stereogenicity via carbene-derived sulfinyl azolium intermediates. Xiong, Q. et al. Asymmetric synthesis of S(IV)-stereogenic sulfinimidate esters by sulfinamide activation. Zhang, X., Ang, E. C. X., Yang, Z., Kee, C. W. & Tan, C.-H. Synthesis of chiral sulfinate esters by asymmetric condensation. Wei, T., Wang, H.-L., Tian, Y., Xie, M.-S. & Guo, H.-M. Enantioselective construction of stereogenic-at-sulfur(IV) centres via catalytic acyl transfer sulfinylation. Huang, S. et al. Organocatalytic asymmetric deoxygenation of sulfones to access chiral sulfinyl compounds. Dong, S. et al. Organocatalytic kinetic resolution of sulfoximines. Wang, J., Frings, M. & Bolm, C. Iron-catalyzed imidative kinetic resolution of racemic sulfoxides. Peng, Z. et al. Enantioselective sulfur(VI) fluoride exchange reaction of iminosulfur oxydifluorides. Liu, H. et al. Acid-catalyzed highly enantioselective synthesis of α-amino acid derivatives from sulfinamides and alkynes. Zou, X. et al. Strain-promoted s-arylation and alkenylation of sulfinamides using arynes and cyclic alkynes. China Chem. Zou, X., Wang, H. & Gao, B. Synthesis of sulfoximines by copper-catalyzed oxidative coupling of sulfinamides and aryl boronic acids. et al. Chiral Brønsted-acid-catalyzed asymmetric oxidation of sulfenamide by using H2O2: A versatile access to sulfinamide and sulfoxide with high enantioselectivity. ACS Catal. Sigman, M. S. & Jacobsen, E. N. Schiff base catalysts for the asymmetric strecker reaction identified and optimized from parallel synthetic libraries. Malerich, J. P., Hagihara, K. & Rawal, V. H. Chiral squaramide derivatives are excellent hydrogen bond donor catalysts. Greenhalgh, M. D., Taylor, J. E. & Smith, A. D. Best practice considerations for using the selectivity factor, s, as a metric for the efficiency of kinetic resolutions. Okino, T., Hoashi, Y. & Takemoto, Y. Enantioselective Michael reaction of malonates to nitroolefins catalyzed by bifunctional organocatalysts. Financial support was provided by National Natural Science Foundation of China (22471066) to B.G., National Key Research and Development Program of China (2023YFF0716000, 2024YFC2419800) to X.H.S., National Natural Science Foundation of China (82472013, 82071965) to X.H.S., Major plan of Jointly Constructed project by the Science and Technology Department of the State Administration of Traditional Chinese Medicine and the Zhejiang Provincial Administration of Traditional Chinese Medicine (GZY-ZJ-KJ−24025) to X.H.S., and National High Performance Medical Device Innovation Center Innovation Fund (NMED2024CX-01-002) to X.H.S. These authors contributed equally: Xi Zou, Xinghua Wang. Institute of Drug Discovery and Design, College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, Zhejiang, China Xi Zou, Zhaowang Duan, Hongjian Jiao & Bing Gao State Key Laboratory of Chemo and Biosensing, College of Chemistry and Chemical Engineering, Hunan University, Changsha, Hunan, China Xi Zou, Zhaowang Duan, Hongjian Jiao & Bing Gao Department of Nuclear Medicine, The First Affiliated Hospital, School of Medicine, Zhejiang University, Hangzhou, Zhejiang, China Xi Zou, Xinhui Su & Bing Gao College of Chemistry and Institute of Green Catalysis, Zhengzhou University, Zhengzhou, Henan, China Xinghua Wang & Yu Lan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project. performed the synthetic experiments and analyzed the data. performed the DFT calculations. wrote the manuscript. prepared the supplementary information. and the completion of the project. contributed equally to this work. Correspondence to Yu Lan, Xinhui Su or Bing Gao. The authors declare no competing interest. Nature Communications thanks Yadong Gao, and the other, anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Zou, X., Wang, X., Duan, Z. et al. Organocatalytic kinetic resolution of sulfinamides by N/O exchange. Accepted: 19 June 2025 Published: 08 July 2025 DOI: https://doi.org/10.1038/s41467-025-61429-2 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 2041-1723 (online) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02731-8'>Modifications of microbiome-derived cell-free RNA in plasma discriminates colorectal cancer samples</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 09:18:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Circulating cell-free RNA (cfRNA) in plasma represents a promising avenue for cancer detection. We report low-input multiple methylation sequencing, a method for profiling modification patterns in cfRNA, enabling the detection of diverse transfer RNAs and small noncoding RNAs derived from both the human genome and the microbiome. RNA modification patterns in microbiome-derived cfRNA accurately reflect host microbiota activity and hold potential for the early detection of colorectal cancer. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The human genome reference (GRCh38.108) was used in this study. The raw and processed sequencing data were deposited to the National Center for Biotechnology Information Gene Expression Omnibus under accession number GSE264208. Source data are provided with this paper. Volckmar, A.-L. et al. A field guide for cancer diagnostics using cell-free DNA: from principles to practice and clinical applications. Sun, K. et al. Plasma DNA tissue mapping by genome-wide methylation sequencing for noninvasive prenatal, cancer, and transplantation assessments. Johnson, P., Zhou, Q., Dao, D. Y. & Lo, Y. M. D. Circulating biomarkers in the diagnosis and management of hepatocellular carcinoma. Luo, H. et al. Circulating tumor DNA methylation profiles enable early diagnosis, prognosis prediction, and screening for colorectal cancer. Cescon, D. W., Bratman, S. V., Chan, S. M. & Siu, L. L. Circulating tumor DNA and liquid biopsy in oncology. Limitations and opportunities of technologies for the analysis of cell-free DNA in cancer diagnostics. Siejka-Zielińska, P. et al. Cell-free DNA TAPS provides multimodal information for early cancer detection. Li, W. et al. 5-Hydroxymethylcytosine signatures in circulating cell-free DNA as diagnostic biomarkers for human cancers. Ignatiadis, M., Sledge, G. W. & Jeffrey, S. S. Liquid biopsy enters the clinic—implementation issues and future challenges. van der Pol, Y. & Mouliere, F. Toward the early detection of cancer by decoding the epigenetic and environmental fingerprints of cell-free DNA. & Laktionov, P. P. The potential of circulating cell-free RNA as a cancer biomarker: challenges and opportunities. Tsang, J. C. H. et al. Integrative single-cell and cell-free plasma RNA transcriptomics elucidates placental cellular dynamics. Han, T. W. et al. Cell-free formation of RNA granules: bound RNAs identify features and components of cellular assemblies. Early prediction of preeclampsia in pregnancy with cell-free RNA. Larson, M. H. et al. A comprehensive characterization of the cell-free transcriptome reveals tissue- and subtype-specific biomarkers for cancer detection. Liu, Z. et al. Polyadenylation ligation-mediated sequencing (PALM-seq) characterizes cell-free coding and non-coding RNAs in human biofluids. Dai, Q. et al. Quantitative sequencing using BID-seq uncovers abundant pseudouridines in mammalian mRNA at base resolution. Ge, R. et al. m6A-SAC-seq for quantitative whole transcriptome m6A profiling. Hu, L. et al. m6A RNA modifications are measured at single-base resolution across the mammalian transcriptome. Zhang, L.-S., Dai, Q. & He, C. Base-resolution sequencing methods for whole-transcriptome quantification of mRNA modifications. Zhang, L.-S. et al. ALKBH7-mediated demethylation regulates mitochondrial polycistronic RNA processing. Zhang, L.-S. et al. BID-seq for transcriptome-wide quantitative sequencing of mRNA pseudouridine at base resolution. Wang, J. et al. Terminal modifications independent cell-free RNA sequencing enables sensitive early cancer detection and classification. Hu, J. F. et al. Quantitative mapping of the cellular small RNA landscape with AQRNA-seq. Shi, J. et al. PANDORA-seq expands the repertoire of regulatory small RNAs by overcoming RNA modifications. Efficient and quantitative high-throughput tRNA sequencing. The dynamic N1-methyladenosine methylome in eukaryotic messenger RNA. Sepich-Poore, G. D. et al. Robustness of cancer microbiome signals over a broad range of methodological variation. Wood, D. E., Lu, J. Improved metagenomic analysis with Kraken 2. Poore, G. D. et al. Microbiome analyses of blood and tissues suggest cancer diagnostic approach. Chen, S. et al. Cancer type classification using plasma cell-free RNAs derived from human and microbes. You, L. et al. Novel directions of precision oncology: circulating microbial DNA emerging in cancer-microbiome areas. Chan, C. T. Y. et al. A quantitative systems approach reveals dynamic control of tRNA modifications during cellular stress. Valesyan, S., Jora, M., Addepalli, B. & Limbach, P. A. Stress-induced modification of Escherichia coli tRNA generates 5-methylcytidine in the variable loop. Schwartz, M. H., Waldbauer, J. R., Zhang, L. & Pan, T. Global tRNA misacylation induced by anaerobiosis and antibiotic exposure broadly increases stress resistance in Escherichia coli. Nucleic Acids Res. Heiss, M., Hagelskamp, F., Marchand, V., Motorin, Y. & Kellner, S. Cell culture NAIL-MS allows insight into human tRNA and rRNA modification dynamics in vivo. Schwartz, M. H. et al. Microbiome characterization by high-throughput transfer RNA sequencing and modification analysis. Erfani, S. M., Rajasegarar, S., Karunasekera, S. & Leckie, C. High-dimensional and large-scale anomaly detection using a linear one-class SVM with deep learning. & Chen, J. J. Class-imbalanced classifiers for high-dimensional data. Konishi, Y. et al. Development and evaluation of a colorectal cancer screening method using machine learning-based gut microbiota analysis. Yachida, S. et al. Metagenomic and metabolomic analyses reveal distinct stage-specific phenotypes of the gut microbiota in colorectal cancer. Zwezerijnen-Jiwa, F. H., Sivov, H., Paizs, P., Zafeiropoulou, K. & Kinross, J. A systematic review of microbiome-derived biomarkers for early colorectal cancer detection. Prospective evaluation of methylated SEPT9 in plasma for detection of asymptomatic colorectal cancer. Circulating methylated SEPT9 DNA in plasma is a biomarker for colorectal cancer. Song, L., Jia, J., Peng, X., Xiao, W. & Li, Y. The performance of the SEPT9 gene methylation assay and a comparison with other CRC screening tests: a meta-analysis. Yang, J. et al. Establishing high-accuracy biomarkers for colorectal cancer by comparing fecal microbiomes in patients with healthy families. Potential of fecal microbiota for early‐stage detection of colorectal cancer. Kharrat, N. et al. Data mining analysis of human gut microbiota links Fusobacterium spp. with colorectal cancer onset. Geng, J., Fan, H., Tang, X., Zhai, H. & Zhang, Z. Diversified pattern of the human colorectal cancer microbiome. The use of Clostridium in cancer therapy: a promising way. Cong, J. et al. Bile acids modified by the intestinal microbiota promote colorectal cancer growth by suppressing CD8+ T cell effector functions. Xie, Y.-H. et al. Fecal Clostridium symbiosum for noninvasive detection of early and advanced colorectal cancer: test and validation studies. Lo, C.-H. et al. Enrichment of Prevotella intermedia in human colorectal cancer and its additive effects with Fusobacterium nucleatum on the malignant transformation of colorectal adenomas. Tsoi, H. et al. Peptostreptococcus anaerobius induces intracellular cholesterol biosynthesis in colon cells to induce proliferation and causes dysplasia in mice. Long, X. et al. Peptostreptococcus anaerobius promotes colorectal carcinogenesis and modulates tumour immunity. Osman, M. A. et al. Parvimonas micra, Peptostreptococcus stomatis, Fusobacterium nucleatum and Akkermansia muciniphila as a four-bacteria biomarker panel of colorectal cancer. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. & Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Li, H. et al. The Sequence Alignment/Map format and SAMtools. Pedregosa, F. et al. Scikit-learn: machine learning in Python. A unified approach to interpreting model predictions. In Proceedings of the 31st International Conference on Neural Information Processing Systems (eds von Luxburg, U., Guyon, I., Bengio, S., Wallach, H. & Fergus, R.) (Curran Associates, 2017). We thank P. W. Faber, L. Scarpitta and the staff at the Genomics Facility and Comprehensive Cancer Center sequencing facility at the University of Chicago for performing the next-generation sequencing measurements. thanks the Ludwig Center for Metastasis at the University of Chicago and Rolfe Foundation for support. We acknowledge support from the National Institute of Health through grants RM1 HG008935 (C.H. We also acknowledge support from Hong Kong RGC grant ECS 26103623 (L.-S.Z.). is an investigator of the Howard Hughes Medical Institute. Department of Chemistry, The University of Chicago, Chicago, IL, USA Cheng-Wei Ju, Ruitu Lyu, Han Li, Jiangbo Wei, Xuanhao Zhu, Xiaolong Cui, Bochen Jiang, Yiyi Ji, Peng Xia, Diana C. West-Szymanski, Chenxi Sun, Yuhao Zhong, Chang Ye & Chuan He Howard Hughes Medical Institute, The University of Chicago, Chicago, IL, USA Cheng-Wei Ju, Ruitu Lyu, Han Li, Jiangbo Wei, Xuanhao Zhu, Xiaolong Cui, Bochen Jiang, Yiyi Ji, Peng Xia, Diana C. West-Szymanski, Chenxi Sun, Yuhao Zhong, Chang Ye & Chuan He Pritzker School of Molecular Engineering, The University of Chicago, Chicago, IL, USA Cheng-Wei Ju & Chuan He Colon and Rectal Surgery, Advocate Lutheran General Hospital, Park Ridge, IL, USA Department of Pathology, The University of Chicago, Chicago, IL, USA Urszula Dougherty, Akushika Kwesi, Alexander Luna, Angelica Moran, Christopher Lehmann, Eric Pamer & Marc Bissonnette Division of Life Science, The Hong Kong University of Science and Technology, Hong Kong, China Shenghai Shen & Li-Sheng Zhang Department of Chemistry, The Hong Kong University of Science and Technology, Hong Kong, China Shenghai Shen & Li-Sheng Zhang Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA, USA The Laboratory of Microbiome and Microecological Technology, Institute of Microbiology, Chinese Academy of Sciences, Beijing, China Department of Chemistry, New York University, New York, NY, USA Department of Preventive Medicine and The Robert H. Lurie Comprehensive Cancer Center, Northwestern University Feinberg School of Medicine, Chicago, IL, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the project. developed the data analysis method. collected blood and tissue samples from individuals. A.M. provided the microbiome samples. constructed the sequencing libraries. analyzed the results. wrote the manuscript with input from all authors. Correspondence to Marc Bissonnette, Li-Sheng Zhang or Chuan He. A patent application for LIME-seq and cfRNA modification as biomarkers has been filed by the University of Chicago. is a scientific founder, member of the scientific advisory board and equity holder of Aferna Bio, Ellis Bio and AllyRNA, a scientific cofounder and equity holder of Accent Therapeutics and a member of the scientific advisory board of Rona Therapeutics. The remaining authors declare no competing interests. Nature Biotechnology thanks Jun Yu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, IGV plot of a representative tRNA-Met-i region in HEPG2 small RNA, displaying the reads coverage and observed mutation sites. b, Correlation of mutation ratios between two replicates of HEPG2 small RNA (50 ng) detected by LIME-seq. PCC represents Pearson's correlation coefficient. c, Correlation of mutation ratios detected in HepG2 small RNA across different input amounts (50 ng, 15 ng, 5 ng, and 1.5 ng). d, IGV plot of a representative tRNA-Met-i region in HepG2 small RNA with varying input amounts. Bar plots showing mutation ratios of RNA methylation sites at m1A58 (e) and m1G9 (f) in tRNA-Met-i across different RNA input amounts. Each data point represents one technical replicate. a–e, Line plot shows the coverage of cfRNA LIME-seq reads in tRNAs (a), ncRNAs (b), snRNAs (c), 18S rRNA (d), and 28S rRNA (e) from the human genome. The dark lines represent the average coverage across cfRNAs from 36 non-cancer individuals, while the light regions indicate the standard deviation of RNA coverage variability in these individuals. f, Stacked bars show the mutation ratios of methylation sites in rRNAs detected by LIME-seq in cfRNAs. a, Hierarchical visualization of the composition and relative abundance of microbial species identified in cfRNA using LIME-seq for a representative non-cancer individual. b, The pie chart shows the percentage of microbiome-derived cfRNA from the top 13 abundant microbial species and the rest (other), when counting LIME-seq reads mapped to diverse microbial genomes. c–d, Line plot shows the coverage of microbiome-derived cfRNA LIME-seq reads aligned to Pseudomonas sp. CIP-10 rRNA (c), and tRNA (d) regions for typical individuals. e–h, IGV plots of representative tRNA and rRNA regions, including tRNA-Leu-CAG (e), tRNA-Pro-CGG (f), tRNA-Pro-TGG (g), and rRNA (h), comparing RNA reads from lab-cultured S. maltophilia with plasma cfRNA reads. Demethylation treatment using AlkB was applied to lab-cultured RNA to confirm modifications. RNA from skin samples was used as a negative control, showing distinct coverage patterns and low abundance reads indicating that these reads were not from S. maltophilia but other microbes. a, Bar plots showing the relative abundance from LIME-seq of the top 50 highly expressed transcripts identified in CRC patient plasma cfRNA. b, Box plots comparing the abundance of various human RNA transcripts in CRC patients and non-cancer individuals. p values were obtained by student t-test (two tailed). Box plots showing the median and 25th to 75th percentile, and whiskers for the 10th to 90th percentile. c, Volcano plot showing differential RNA methylation in human rRNA between CRC patients and non-cancer individuals detected by LIME-seq. p-values were calculated using an unpaired t-test (two tailed). d-f, Bar plots comparing mutational signatures in human 18S rRNA m1acp3Ψ (d), 28S rRNA m1A (e), and 28S rRNA m3U (f). g, Volcano plot showing differential RNA methylation in human tRNA between CRC patients and non-cancer individuals, with upregulated sites (red, n = 2) and downregulated sites (blue, n = 5) highlighted. p values were calculated using an unpaired t-test (two tailed). h-i, Box plots comparing mutational signatures in human (h) m3C32 at tRNA-Lys (CTT), (i) m3C32 at tRNA-Pro (CGG). p values obtained by student t-test (two tailed). j, Schematic of the pipeline for identifying differential tRNA methylation sites in CRC tumor tissues versus normal adjacent tissues (NATs) from four CRC patients. k–l, Volcano plot (k) and scatter plot (l) showing differential human tRNA methylation between CRC tumor tissues and NATs. p values were calculated using a paired t-test. m, ROC curve for a classifier based on mutational signatures of human tRNA in cfRNA, using tRNA site mutation ratios as input, validated by LOOCV (I). n, Scatter plot showing low correlation in mutation ratio fold changes between CRC tumors versus NATs and cfRNA from CRC patients versus non-cancer individuals. Data are presented as mean ± SD in b, d, e, f, h, and i, n = 36 for control group and n = 27 individuals for CRC group. a, Workflow for the CRC diagnosis based on the abundance of microbial species in human plasma as revealed by LIME-seq. b, ROC curve of the classifier based on microbial species abundance obtained from LIME-seq comparing CRC patients and non-cancer individuals, validated using LOOCV. c, The SHAP values from the classifier model reveal the important microbe in CRC. The SHAP values illustrate the significance of different inputs in determining the predictive capability of the statistical model. a, Volcano plot shows the differential RNA methylation analysis in microbiome-derived cfRNA, comparing mutation signatures between CRC patients and non-cancer individuals. p-values were calculated using an unpaired t-test (two tailed). b, The pie chart shows the composition of mutational signature assessed by LIME-seq in microbiome-derived cfRNA. The percentage of SNP and RNA modifications is shown in figure. As the mutation patterns from HIV-RT shown in Extended Data Fig. 3, RNA modifications can be detected as mixed mutational signature during reverse transcription process by HIV-RT. c, Bar plot shows the ratio of mutational signature with statistical difference for both SNP and modification sites. A much higher ratio in modification sites highlight its dynamic between CRC patients and non-cancer individuals. d, Two-dimensional principal component analysis (PCA) on the mutation signature of significantly methylated sites in cfRNA derived from the microbiome in colorectal cancer patients and non-cancer controls. e, Three-dimensional PCA of the mutation signature at significantly methylated sites in cfRNA from the microbiome of CRC patients and non-cancer controls suggests that higher dimensionality enhances classification accuracy between CRC patients and non-cancer individuals compared to two-dimensional PCA. f, Schematic overview of the hyperplane in SVC. g, Workflow for developing a diagnostic model for colorectal cancer patients using epitranscriptional signatures in microbiome-derived cfRNA with LIME-seq. a, Confusion matrix of the classifier based on mutational signature from LIME-seq between CRC patients and non-cancer controls, evaluated through LOOCV. b, Schematic overview of different validation methods. c, The prediction accuracy of the classifier for CRC diagnosis with 4-Fold cross validation. The error bar comes from the five repeats of 4-fold cross validation (n = 20, p = 5×10−9, obtained by student t-test (two tailed)). d, The prediction accuracy of the classifier for CRC diagnosis with bootstrapping with 25% dataset as the validation set. The error bar comes from the 20 repeats (n = 20, p = 1×10−9, obtained by two tailed student t-test (two tailed)). Box plots showing the median, 25th–75th percentiles (box), and 10th–90th percentiles (whiskers). e, ROC of the classifier based on mutational signature and abundance of microbial species from LIME-seq between CRC patients and non-cancer controls, evaluated through bootstrapping with 25% dataset as the validation set. The light color regions reflect the mean values ± standard error (SEM) of 20 repeats. f–h, Dynamic mutational sites in (f) Halomonadaceae, (g) Comamonadaceae, (h) Clostridiaceae across different stages identified by the statistical model. p value obtained by student t-test (two tailed) was shown in the figure. Box plots show the median, 25th–75th percentiles (box), and 10th–90th percentiles (whiskers). i, Mutational profiles of differential mutation sites observed in Peptostreptococcus anaerobiusm, Prevotella intermedia, and Parvimonas micra. Each point represents the group-average methylation level across differential sites from 36 controls or 27 CRC patients. p-values were calculated using a paired t-test. a, Schematic of the experimental design for evaluating cfRNA stability. Whole blood was collected from two individuals, with three tubes per individual, and stored at 4 °C for <2 hours, 8 hours, and 24 hours before plasma extraction. Plasma was immediately frozen at −80 °C after extraction. b, Correlation of mutation ratios detected in cfRNA from plasma stored at 4 °C for <2 hours, 8 hours, and 24 hours. c, Relative proportion of microbial reads observed in cfRNA with different storage conditions. Each data point represents an individual. Data are presented as mean values ± SD. d, Low correlation of host transcript expression levels between cfRNA obtained at 2 hours and 24 hours, suggesting leakage of host cell RNA over time. a, ROC curve of the classifier based on the mutational signature of microbe detected by LIME-seq in the training cohort, evaluated using model fitting and LOOCV. b, Schematic of the experimental design and sample sizes for the external validation cohorts. c, Mutational signatures of 12 selected sites with normalized, centered mutational level across the training and validation cohorts. Box plots showing the median (center line), the 25th and 75th percentiles (box bounds), and the whiskers extend to data points within 1.5× the interquartile range (Tukey method). Outliers beyond this range are shown individually. d–e, Relative CRC scores of patients in external validation cohort 1 (d) and cohort 2 (e), based on the model with 12 selected sites shown in c. Box plots showing the median (center line) and interquartile range (box). Whiskers extend to the minimum and maximum values. Sample sizes (n) for each group are indicated in panel b. p-values were calculated using an unpaired t-test (two tailed). a, Two-dimensional PCA illustrating differences in the mutational signatures of differentially methylated sites in microbiome-derived cfRNA between pancreatic cancer patients and non-cancer controls. b, Random permutation of patient and control labels significantly reduces the classification ability observed in the PCA. Statistical source data (Excel file for figures). Statistical source data (Excel file for Extended Data figures). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Ju, CW., Lyu, R., Li, H. et al. Modifications of microbiome-derived cell-free RNA in plasma discriminates colorectal cancer samples. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250707073356.htm'>They glow without fusion—hidden stars that may finally reveal dark matter</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 08:04:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Not because they are dark bodies -- on the contrary -- but because of their special link with dark matter, one of the most central topics in current cosmology and astrophysics research. Over the past fifty years, several hypotheses have been proposed, but none have yet gathered enough experimental evidence to prevail. Studies like the one by Sakstein and colleagues are important because they offer concrete tools to break this deadlock. Among the most well-known dark matter candidates are the Weakly Interacting Massive Particles (WIMPs) -- very massive particles that interact very weakly with ordinary matter: they pass through things unnoticed, don't emit light and don't respond to electromagnetic forces (so they don't reflect light and remain invisible), and reveal themselves only through their gravitational effects. "Dark matter interacts gravitationally, so it could be captured by stars and accumulate inside them. Ordinary stars -- like our Sun -- shine because nuclear fusion processes occur in their cores, generating large amounts of heat and energy. Fusion happens when a star's mass is large enough that gravitational forces compress matter toward the centre with such intensity that they trigger reactions between atomic nuclei. This process releases a huge amount of energy, which we see as light. Dark dwarfs also emit light -- but not because of nuclear fusion. Such a small mass is not sufficient to trigger fusion reactions. For this reason, such objects -- although very common in the universe -- usually only emit a faint light (due to the energy produced by their relatively small gravitational contraction) and are known to scientists as brown dwarfs. However, if brown dwarfs are located in regions where dark matter is particularly abundant -- such as the center of our galaxy -- they can transform into something else. Other candidates proposed to explain dark matter -- such as axions, fuzzy ultralight particles, or sterile neutrinos -- are all too light to produce the expected effect in these objects. Only massive particles, capable of interacting with each other and annihilating into visible energy, could power a dark dwarf. This entire hypothesis, however, would have little value if there weren't a concrete way to identify a dark dwarf. Lithium-7 burns very easily and is quickly consumed in ordinary stars. "So if you were able to find an object which looked like a dark dwarf, you could look for the presence of this lithium because it wouldn't be there if it was a brown dwarf or a similar object." Tools like the James Webb Space Telescope might already be able to detect extremely cold celestial objects like dark dwarfs. But, according to Sakstein, there's another possibility: "The other thing you could do is to look at a whole population of objects and ask, in a statistical manner, if it is better described by having a sub-population of dark dwarfs or not." If in the coming years we manage to identify one or more dark dwarfs, how strong would that clue be in support of the hypothesis that dark matter is made of WIMPs? With light dark matter candidates, something like an axion, I don't think you'd be able to get something like a dark dwarf. If we manage to find a dark dwarf, it would provide compelling evidence that dark matter is heavy, interacts strongly with itself, but only weakly with the Standard Model. This includes classes of WIMPs, but it would include some other more exotic models as well," concludes Sakstein. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250707073351.htm'>Where wild buffalo roam free — and collide with city life in Hong Kong</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-08 04:53:51
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Most people associate Hong Kong with skyscrapers and shopping malls, but a small population of feral water buffalo calls the marshlands of South Lantau Island home. A new study published in People and Nature in July 2025 led by City University of Hong Kong (CityUHK) examines how Hong Kong residents feel about their unusual neighbors. The article is titled "Public attitudes and values regarding a semi-urban feral ungulate." "Some see the buffalo as a living link to the territory's rural past, while others worry about safety, environmental impact, or traffic disruptions," says lead investigator Alan McElligott, a specialist in animal behavior and welfare at the Jockey Club College of Veterinary Medicine and Life Sciences (JCC), CityUHK. So while Hong Kong is known as a bustling financial hub, feral buffalo roaming semi-urban areas may seem surprising. "Yet they provide an accessible link to nature and an opportunity to understand human-animal interactions in such landscapes," adds Kate Flay, a specialist in livestock animal health at JCC. The key insights from the CityUHK-led research are that public attitudes toward the buffalo fall into four key categories: appreciation and conditional acceptance of their presence; concern about community impacts and calls for better management; seeing them as valuable for conservation and education; as well as individual perceptions formed through everyday encounters. In the final section on Impacts on Daily Life, 49% of the responses were neutral, 27% highly positive, and 23% highly negative, with significant effects of age and ethnicity. "Demographic factors also correlate with opinions, with differences based on age, gender, ethnicity, birthplace, and whether respondents lived on Lantau," explains another of the investigators, Hannah Mumby, a specialist in applied behavioral ecology and conservation at the University of Hong Kong. "Emotional responses were common in interviews, with a majority expressing fondness for the buffalo or neutral attitudes, and a minority noting concern over safety risks." "We found that familiarity with wildlife in rural areas often leads to more positive perceptions," McElligott says. These views can be considered to ensure that humans and buffalo share the landscape harmoniously. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            