
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-22</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a64852050/science-behind-runners-high/'>This Skill Can Help You Unlock an Altered State of Consciousness—And Mimic the Way Marijuana Feels</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 16:43:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>It may be one of evolution's greatest tricks yet. Indeed, the approximately three-quarters of athletes who say they've experienced “runner's high,” report that as pain, doubt, and a concrete sense of time slip away, they're replaced by bliss, confidence, and a trance-like focus. In other words, an altered state of consciousness. For decades, scientists chalked up this heady buzz to endorphins, those natural painkillers that the body releases when we laugh, eat dark chocolate, or work up a sweat. But in the last few years, research has begun to point to a different set of biochemicals as the cause of runner's high. While a pleasant physiological response may have evolved to allow our ancestors to chase prey over long distances, there's evidence that runner's high can help contemporary humans feel happier, less stressed, and more motivated to exercise. In the 1980s, exercise scientists started to attribute the groove you feel on a really good run to endorphins, after finding high levels of the natural painkillers in people's bloodstreams during vigorous exercise. Researchers assumed these chemicals must be responsible for the ease and euphoria that sometimes can arise during an intense workout. But there was a problem with this explanation. Endorphin molecules are too large to cross the blood-brain barrier. Even if a runner's blood was swimming with endorphins, there was no way for them to reach the brain and unlock an altered state of consciousness. A 2012 study was among the first to establish that mammals adapted for running—humans and dogs, specifically—produce extra cannabinoids while hoofing it on a treadmill. But the real breakthrough came in 2015, when researchers at the University Medical Center Hamburg-Eppendorf in Germany showed that mice whose endorphins were blocked ended their laboratory runs calmer and less stressed than when they started. Later that year, the researchers reproduced these results in human runners. After a 45-minute run, participants whose endorphin system had been deactivated by the drug naloxone still reported that euphoric runner's high, and their blood showed increased levels of endocannabinoids. While different chemicals are at play, you might be hard-pressed to tell the two apart. “Endocannabinoid receptors are concentrated in the central nervous system and designed for the chemicals our bodies make,” says Angela Bryan, Ph.D., an avid runner and neuroscientist who studies how cannabis affects the human body. The reason why an epic run can feel so much like eating a potent gummy is that THC is an exogenous cannabinoid, in other words a cannabinoid that comes from outside the body, meaning “it fits into those receptors like a lock in a key,” according to Bryan. For modern humans, running is an ideal way to stay in shape or simply get some alone time outdoors. But back when our ancestors were chasing game across vast savannas, running was the way we found dinner. Activating the endocannabinoid system, which fires up to regulate stress and reduce pain, may have allowed us to push through the burning lungs and biting cramps of a long-distance hunt. “If that was horrible and felt awful and [we] didn't want to do it, we might not have survived.” Some athletes report clearer thinking and sharper vision during their long-haul jaunts, two sensations that would come in handy while stalking a large, wily animal. But vanishingly few workouts result in a runner's high, and there is evidence that some people can't achieve one no matter how many miles they run. For example, past studies have found that as many as a quarter of endurance runners have reported never experiencing a runner's high. While the bad news is it's unlikely you'll unlock a new level of consciousness on a light, 20-minute run, the better news is you probably don't need to push yourself to extremes. David Raichlen, the evolutionary biologist behind that 2012 study about runner's high in mammals, has suggested that working out at about 70 to 85 percent of your maximum heart rate (there's a calculator for that based on your age) for at least 45 minutes is ideal for boosting endocannabinoid production. To find out what works for you, the scientist says, “start low and go slow. To get a runner's high the old-fashioned way, Bryan's advice is to lace up your running shoes often and consistently. “One thing that we know is the better we get at it, the better it feels,” she says. For a safe, expert-backed introduction to running, check out the beginner-friendly training plans that our friends at Runner's World have developed. Ashley Stimpson is a freelance journalist who writes most often about science, conservation, and the outdoors. Her work has appeared in the Guardian, WIRED, Nat Geo, Atlas Obscura, and elsewhere. She lives in Columbia, Maryland, with her partner, their greyhound, and a very bad cat. Lucid Dreaming Is a New State of Consciousness Conscious ‘Alien Minds' Could Be Living Among Us Is AI Actually a Form of Alien Intelligence?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01630-x'>These contact lenses give people infrared vision — even with their eyes shut</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 15:18:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. People who tested a new type of designer contact lens could see flashing infrared signals from a light source.Credit: Yuqian Ma, Yunuo Chen, Hang Zhao (CC BY-SA) Humans have a new way of seeing infrared light, without the need for clunky night-vision goggles. The technology, which was detailed in Cell on 22 May1, “is incredibly cool, just like something out of a science-fiction movie”, says Xiaomin Li, a chemist at Fudan University in Shanghai, China. It opens up “new possibilities for understanding the world around us”, he adds. Near-infrared light sits just outside the range of wavelengths that humans can normally detect. Some animals can sense infrared light, although probably not well enough to form images. The new lenses avoid these limitations while also offering richer, multi-coloured infrared images that night-vision goggles, which operate on a monochrome green scale, typically do not. Moreover, unlike night-vision goggles, which amplify light to detect low-level infrared signals, the lenses also allow users to see only intense infrared signals, such as those emitted by light-emitting diodes (LEDs). For these reasons, some critics don't think the lenses will prove useful. “I cannot think of any application that would not be fundamentally simpler with infrared goggles,” says Glen Jeffery, a neuroscientist at University College London who specializes in eye health. For instance, wearers would be able to read anti-counterfeit marks that emit infrared wavelengths but are otherwise invisible to the human eye, says co-author Yuqian Ma, a neuroscientist at the USTC. Li, who was not involved in the work, offers another possibility: the lenses might be worn by doctors conducting near-infrared fluorescence surgery, to directly detect and remove cancerous lesions “without relying on bulky traditional equipment”. The structure of liquid carbon elucidated by in situ X-ray diffraction Biogen is seeking a strategic partner providing expert medical leadership to guide development, launch planning, and support for our FA program. Biogen is seeking an experienced Bioinformatics professional to lead the design, implementation, and growth of our information architecture. Biogen is seeking a Development Asset Lead to serve as the full-time leader of one or more clinical stage portfolio asset(s) within Development. An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/health/a64830375/human-brain-cells-computer/'>This $35,000 Computer Runs on Human Brain Cells. Scientists Say It's Not Conscious—Yet.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Two very different types of “computers” dominate the world today. These are two wildly different architectures with totally different capabilities—the human brain somehow achieves a level of consciousness unseen elsewhere in nature (or the observable universe, for that matter), but computers can do calculating tasks many times faster than our minds could ever fathom. So, it should come as no surprise that since the very origins of computing, scientists and engineers have pondered whether these two disparate “computers” could one day merge together. Cortical Labs, an Australia-based company, announced earlier this year that it had successfully developed the world's first “biological computer,” which it calls the CL-1. Fusing human brain cells with silicon hardware, Cortical Labs says the CL-1 is an ideal tool for science and medical research, meaning that this thing isn't going to be siding onto your desktop and delivering a 120 frames-per-second gaming experience. “The large majority of drugs for neurological and psychiatric diseases that enter clinical trial testing fail, because there's so much more nuance when it comes to the brain—but you can actually see that nuance when you test with these tools,” Brett Kagan, Cortical Labs' Chief Scientific Officer, told New Atlas back in March. “Our hope is that we're able to replace significant areas of animal testing with this. Animal testing is unfortunately still necessary, but I think there are a lot of cases where it can be replaced and that's an ethically good thing.” Kagan was directly involved in the development of CL-1's predecessor, DishBrain, which grabbed headlines back in 2022 for successfully playing a game of Pong (the results of which were published in the journal Neuron). As Kagan explained at the time, DishBrain played Pong quite unlike a human brain, and instead described the neurons as sort of experiencing its surroundings as if it was the Pong paddle itself. Back in 2022, Kagan said that his team was working closely with bioethicists to answer this question, and clearly came to the conclusion that this wasn't a concern—at least, not yet. Speaking with Live Science, a stem cell research unaffiliated with Cortical Labs came to a similar conclusion, and Suhas Kumar from Sandia National Laboratories also told Popular Mechanics back in 2022 that the simplicity of this setup means the neurons are simply responding to a stimulus. However, seeing as this is just a first step into the broader world of neuromorphic computing, the relative simplicity of CL-1 could get dizzyingly complex pretty quick. The Cortical Labs team argues that Synthetic Biological Intelligence, or SBI, is “inherently more natural than AI” because it uses materials more akin to the human brain. Is the Key to Human Immortality This Sea Creature? A Surprising Reason Why Your Pee May Turn Red</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64838465/homo-erectus-java/'>Scientists Found Evidence of Our Ancient Human Ancestors Underwater, Suggesting a Sunken World</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Some of the early hominins evidently went island hopping. We may earn commission if you buy from a link. During the glacial period that chilled the Earth 140,000 years ago, sea levels in the Indonesian region of Sundaland were low enough for present-day islands to tower like mountain ranges with a lowland savannah stretching between them. It was an expanse of mostly dry grasslands with strips of forest edging the rivers, and animals like crocodiles, river sharks, elephants, hippos, rhinos, and carnivorous lizards flourished in the region. Long thought to have been isolated on the island of Java, two fossil fragments of a Homo erectus skull—which surfaced with recent ocean dredging in preparation for the construction of an artificial island—revealed that this hominin species migrated and spread throughout the islands when they could still walk over bridges of land. Homo erectus was first discovered in Java (and was known as “Java Man” until the species was officially renamed), but sossilized remains had never before been found on the seafloor between what are now the islands of Java, Bali, Sumatra and Borneo. “Under the relatively dry Middle Pleistocene climate of eastern Java, herds of herbivores and groups of hominins on the lowland plains were probably dependent on large perennial rivers, providing drinking water and terrestrial as well as aquatic food sources,” Berghuis said in a study recently published in Quaternary Environments and Humans. Trees bore fruit all year, and the ancient hominins would have been able to gather edible plants in addition to catching fish and shellfish. They may have even used mussel shells as tools—the oldest known evidence of them being used for that purpose—and engraved some of them (the most ancient human engravings have been found on shells that previously turned up in Java). The new findings show that they also hunted river turtles and terrestrial animals. More modern human species on the Asian mainland (such as Denisovans and Neanderthals) were already known to have hunted bovids, and while no evidence for this had been found on Java, the presence of these seafloor fossils could mean that hunting methods were transferred from one species to the other. Homo erectus marked a significant shift in human evolution—they were the earliest hominids to bear more of a resemblance to modern humans, with larger bodies, longer legs, and shorter arms relative to their torso. More muscle mass meant that they could walk and run faster than earlier hominins, and were likely more adept hunters. An increase in body size is also associated with an increase in brain size, and skulls tell us that their brains were over 50% larger than those of early Australopithecus species (though the human brain would eventually evolve to be 40% larger than that by the time Homo sapiens appeared). Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Explorers Found Maya Sacrifices in a ‘Blood Cave'</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-much-ultraprocessed-food-do-you-eat-blood-and-urine-record-it/'>How Much Ultraprocessed Food Do You Eat? Blood and Urine Record It</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Diets rich in industrially manufactured food have been associated with several health conditions. Molecules in urine and blood can reveal how much of a person's diet comes from ultra-processed foods, according to a study published in PLOS Medicine today. The paper suggests that these measurements provide an objective way to track consumption of ultra-processed food — and would be useful for investigating links to diseases such as diabetes and cancer. Ultra-processed foods are industrially manufactured and often contain ingredients, such as additives and emulsifiers, that are not typically found in home-cooked food. From sweetened yogurts to factory-made bread and packaged snacks, “it's a really wide range of foods”, says study co-author Erikka Loftfield, an epidemiologist at the US National Cancer Institute in Rockville, Maryland. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Loftfield and her colleagues have now expanded that analysis to include more than 1,000 metabolites, which are produced when the body converts food into energy. “This work is important,” says Oliver Robinson, a molecular epidemiologist at Imperial College London. “There's a lot of measurement error in traditional assessment methods for diet.” Loftfield and her colleagues studied samples collected from 718 healthy individuals aged 50–74 in 2012–13. The participants' urine and blood samples had been collected twice, six months apart. Loftfield's team labelled each food item, from the bun in a burger to the cheese slice, meat patty and ketchup, either ultra-processed or not. The researchers then used a machine-learning technique to give each participant a score for how much of their daily energy intake came from ultra-processed foods, says Loftfield. Individuals who consumed the most ultra-processed foods generally got more of their energy from carbohydrates, added sugars and saturated fats — and less from proteins and fibre — than did those who consumed lower levels of ultra-processed foods. Samples from people with diets rich in ultra-processed foods were more likely to contain a metabolite linked to an increased risk of type-2 diabetes — and some of these people's urine samples contained a molecule produced by certain food packaging. They also contained fewer metabolites derived from fresh fruits and vegetables. To test whether the metabolite scoring could be used to predict the presence of a lot of ultra-processed food in a person's diet, the researchers used data from a randomized controlled diet study of 20 individuals aged 18–50 conducted between 2018 and 2020. Loftfield wants to test the method on populations with more varied diets, and on younger people, who tend to eat more ultra-processed foods. Robinson wonders whether the tool could be used to address big unanswered questions, including what it is about ultra-processed food that is bad for you. Understanding this difference could help companies to improve their products. “We're sort of trapped in this industrial food-production system where we all eat ultra-processed food, and it's quite hard for most people to go back to fresh food, because our food systems are now set up like this,” adds Robinson. This article is reproduced with permission and was first published on May 20, 2025. First published in 1869, Nature is the world's leading multidisciplinary science journal.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64809816/orange-cat-x-chromosome/'>Orange Cats Have Long Been a Genetic Mystery. Scientists Have Finally Solved It.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The answer might even explain why they're... like that. We may earn commission if you buy from a link. Garfield might be the most iconic orange tabby around, but Hollywood has seen quite a few leading cats of the same color—Heathcliff, Oliver, Hiyao Miyazaki's adorably terrifying Catbus from My Neighbor Totoro, and Morris from those retro cat food commercials all come to mind. Like most celebrities, they have so far refused to give up their most guarded beauty secret—how did they get those fabulous golden auburn coats? The location of the mutation on this particular gene also explains why there are so many more male orange cats than female ones. In female cats (and all female mammals), one of the two X chromosomes in each cell is randomly switched off in a process known as X chromosome inactivation, so even if the mutation is present, it is unlikely that it will be expressed by every cell and appear as an even (or even semi-even) orange coat. Male cats, on the other hand, only have one X chromosome, and are therefore much more likely to evenly express that mutation. Whether orange or not, all fur pigmentation genes are X-linked. Calico and tortoiseshell coats also come from different combinations of activated X chromosomes—both with and without the deletion that results in orange—which explains why most of them are female. Genes promoting melanogenesis, or the production of melanin in melanocytes, suppress ARHGAP36 and are upregulated in brown, black, and gray patches. These colors are associated with the black or brown pigment known as eumelanin, which is also the most common form of melanin. Sasaki believes that when a mutated ARHGAP36 is expressed as orange fur in cats, the missing part of the gene would have suppressed orange coloration had it been present. In cats, mutated ARHGAP36 was shown to suppress other genes involved in the production of eumelanin so that it could instead produce a different type of melanin called pheomelanin, which is the reddish-yellow pigment in orange fur. It seems that high ARHGAP36 activity is, in general, associated with reduced activity in genes involved with the production of eumelanin. Sasaki is convinced that this gene's takeover may somehow shift pigment production to pheomelanin instead—though, how exactly it pulls this off is still unknown. And because ARHGAP36 also has significant importance in the brain, there is even speculation as to whether there are true associations between fur color and personality (the confirmation of which could either prove or silence all the orange cat memes out there). Somewhere, Garfield is smugly beaming next to a tray of lasagna. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. World's First Patient Saved by Custom Gene Therapy Experts May Have Found a Way to Simplify Gravity</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/saturn-has-274-known-moons-thanks-in-large-part-to-this-astronomer/'>How One Astronomer Uncovered Nearly 200 Moons—Around Just One Planet</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>How One Astronomer Helped to Discover Nearly 200 Moons of Saturn Scientific American spoke with the astronomer who has contributed to the discovery of two thirds of Saturn's known moons NASA, ESA, John T. Clarke (Boston University), Zolt G. Levay (STScI) A mere decade ago, astronomers knew of just 62 moons around Saturn. Today the ringed planet boasts a staggering 274 official satellites. That's more than any other world in the solar system—and far too many for most people to keep track of. Astronomer Edward Ashton is no exception, even though he has helped to discover 192 of them—he thinks that's the total, anyway, after pausing to do some mental math. Ashton is now a postdoctoral fellow at the Academia Sinica Institute of Astronomy and Astrophysics in Taiwan. He fell into hunting for Saturn's moons in 2018, when his then academic adviser suggested the project for his Ph.D. at the University of British Columbia. Most recently, in March, Ashton and his colleagues announced a batch of 128 newfound Saturnian satellites. Scientific American spoke with Ashton about the science of discovering so many relatively tiny moons—most of them just a few kilometers wide—using vast amounts of data gathered by the Canada-France-Hawaii Telescope (CFHT), located in Hawaii. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. To detect the moons, we use a technique known as shifting and stacking. We take 44 sequential images of the same patch of sky over a three-hour period because, in that time frame, the moons move relative to the stars at a rate similar to Saturn. So what we do is: we shift the images relative to one another at multiple different rates near that of Saturn, and then we basically blink between the different shift rates. And then, as you get faster than the moon's rate, it expands again. But just seeing an object moving at a Saturn-like rate near Saturn doesn't guarantee that it is a moon. So what we need to do is track the objects to show that they are in orbit around the planet. Did you need new techniques and observatories to do this work? The technique and the technology have been there for a while—the same technique has been used to find moons of Neptune and Uranus. But the sky area around those planets where moons can exist is a lot smaller, so it takes less time to search through the data. One of the reasons why this hadn't been done for Saturn is because it's very time-consuming. Why do those other planets have less space where moons could be than Saturn does? I had been wondering if this technique works for other planets, and clearly the answer is yes. But do you think there are other moons that have yet to be found around Saturn or other planets with the method? So if you redo this technique again, you will be able to find more moons around Saturn, but this is a case of diminishing returns. At the moment, if you use the same technique for Jupiter, you will be able to find fainter moons. And Jupiter is much brighter than Saturn and the other planets, so there's a lot of scattered light that makes it harder to see the moons. So it's even harder to find satellites around Jupiter, and as you mentioned, other groups have already done this work for Uranus and Neptune. Yeah, you probably have to wait until better technology comes along. But there is a telescope that's set to launch pretty soon, the Nancy Grace Roman Space Telescope, that has quite a large field of view. So that'll be a good telescope to use for hunting more moons. What do we know about these new moons? You basically can only get the moons' orbits and approximate sizes. Moons that are sort of clumped together in orbital space are most likely the result of a collision, so you can see what moons come from the same parent object. Is seeing so many moons around Saturn unusual? But when you get down to the smaller ones that we're discovering, Saturn seems to shoot up in terms of the numbers. This could just be because there was a recent collision within the Saturnian system that produced a large number of fragments. Some of these new moons, they've been linked back to observations by a different group from more than 10 years ago. For the rest, we get full discovery credit, which, I think, means we get the right to name them. Do you have more moon-hunting observations to analyze? No, I'm taking a little break from moons! There are some mysteries about them at the moment. Meghan Bartels is a science journalist based in New York City. She joined Scientific American in 2023 and is now a senior news reporter there. Her writing has also appeared in Audubon, Nautilus, Astronomy and Smithsonian, among other publications. She attended Georgetown University and earned a master's degree in journalism at New York University's Science, Health and Environmental Reporting Program. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41556-025-01671-4'>Neutrophils secrete exosome-associated DNA to resolve sterile acute inflammation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-22 10:40:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Cell Biology (2025)Cite this article Metrics details Acute inflammation, characterized by a rapid influx of neutrophils, is a protective response that can lead to chronic inflammatory diseases when left unresolved. We previously showed that secretion of LTB4-containing exosomes via nuclear envelope-derived multivesicular bodies is required for effective neutrophil infiltration during inflammation. Here we report that the co-secretion of these exosomes with nuclear DNA facilitates the resolution of the neutrophil infiltrate in a mouse skin model of sterile inflammation. Activated neutrophils exhibit rapid and repetitive DNA secretion as they migrate directionally using a mechanism distinct from suicidal neutrophil extracellular trap release and cell death. Packaging of DNA in the lumen of nuclear envelope-multivesicular bodies is mediated by lamin B receptor and chromatin decondensation. These findings advance our understanding of neutrophil functions during inflammation and the physiological relevance of DNA secretion. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $209.00 per year only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All the raw data and associated statistical analysis presented have been provided as ‘source data' and ‘unprocessed western blots' for the respective figures. Owing to the large size of high-resolution z-stack and time-lapse microscopy images, the raw microscopy images are available from the corresponding author upon reasonable request. The mass spectrometry proteomics data are available as Supplementary Table 1 and have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD061995. Source data are provided with this paper. All CellProfiler pipelines used in this study are made available on the CellProfiler website linked to the publication weblink and accession information, to ensure transparency and reproducibility of the analysis. The MATLAB code used to analyse under-agarose migration of neutrophils has been uploaded to a publicly available repository and can be accessed on Zenodo at https://doi.org/10.5281/zenodo.15080547 (ref. Requests for reagents and resources should be directed to the corresponding author C.A.P. Németh, T., Sperandio, M. & Mócsai, A. Neutrophils as emerging therapeutic targets. Drug Discov. Article PubMed Google Scholar Serhan, C. N. et al. Resolution of inflammation: state of the art, definitions and terms. FASEB J. Google Scholar Fine, N., Tasevski, N., McCulloch, C. A., Tenenbaum, H. C. & Glogauer, M. The neutrophil: constant defender and first responder. Google Scholar Peiseler, M. & Kubes, P. More friend than foe: the emerging role of neutrophils in tissue repair. Google Scholar Netea, M. G. et al. A guiding map for inflammation. Google Scholar Furman, D. et al. Chronic inflammation in the etiology of disease across the life span. Google Scholar Castanheira, F. V. S. & Kubes, P. Neutrophils and NETs in modulating acute and chronic inflammation. Google Scholar Wigerblad, G. & Kaplan, M. J. Neutrophil extracellular traps in systemic autoimmune and autoinflammatory diseases. Google Scholar Yipp, B. G. & Kubes, P. NETosis: how vital is it? Burn, G. L., Foti, A., Marsman, G., Patel, D. F. & Zychlinsky, A. The neutrophil. Pilsczek, F. H. et al. A novel mechanism of rapid nuclear neutrophil extracellular trap formation in response to Staphylococcus aureus. Yipp, B. G. et al. Infection-induced NETosis is a dynamic process involving neutrophil multitasking in vivo. Eghbalzadeh, K. et al. Compromised anti-inflammatory action of neutrophil extracellular traps in PAD4-deficient mice contributes to aggravated acute inflammation after myocardial infarction. Schauer, C. et al. Aggregated neutrophil extracellular traps limit inflammation by degrading cytokines and chemokines. Knopf, J., Leppkes, M., Schett, G., Herrmann, M. & Muñoz, L. E. Aggregated NETs sequester and detoxify extracellular histones. Hahn, J. et al. Aggregated neutrophil extracellular traps resolve inflammation by proteolysis of cytokines and chemokines and protection from antiproteases. FASEB J. Rodríguez-Morales, P. & Franklin, R. A. Macrophage phenotypes and functions: resolving inflammation and restoring homeostasis. Trends Immunol. Devchand, P. R. et al. The PPARα–leukotriene B4 pathway to inflammation control. de Oliveira, S., Rosowski, E. E. & Huttenlocher, A. Neutrophil migration in infection and wound repair: going forward in reverse. Xu, Q., Zhao, W., Yan, M. & Mei, H. Neutrophil reverse migration. Lämmermann, T. et al. Neutrophil swarms require LTB4 and integrins at sites of cell death in vivo. Article PubMed Afonso, P. V. et al. LTB4 is a signal-relay molecule during neutrophil chemotaxis. Mandal, A. K. et al. The nuclear membrane organization of leukotriene synthesis. Natl Acad. Arya, S. B., Chen, S., Jordan-Javed, F. & Parent, C. A. Ceramide-rich microdomains facilitate nuclear envelope budding for non-conventional exosome formation. Cell Biol. Majumdar, R., Tameh, A. T., Arya, S. B. & Parent, C. A. Exosomes mediate LTB4 release during neutrophil chemotaxis. PLoS Biol. Solovei, I. et al. LBR and lamin A/C sequentially tether peripheral heterochromatin and inversely regulate differentiation. Liokatis, S. et al. Solution structure and molecular interactions of lamin B receptor tudor domain. Tsai, P.-L., Zhao, C., Turner, E. & Schlieker, C. The lamin B receptor is essential for cholesterol synthesis and perturbed by disease-causing mutations. Polioudaki, H. et al. Histones H3/H4 form a tight complex with the inner nuclear membrane protein LBR and heterochromatin protein 1. Gaines, P., Gotur, D., Halene, S., Olins, A. L. & Olins, D. E. Dynamic expression of the lamin B receptor and associated proteins during myeloid differentiation: insight into mechanisms of neutrophil vs. macrophage morphogenesis. Manley, H. R., Keightley, M. C. & Lieschke, G. J. The neutrophil nucleus: an important influence on neutrophil migration and function. Gaines, P. et al. Mouse neutrophils lacking lamin B-receptor expression exhibit aberrant development and lack critical functional responses. Toda, A., Yokomizo, T. & Shimizu, T. Leukotriene B4 receptors. Prostaglandins Other Lipid Mediat. Article PubMed Narala, V. R. et al. Leukotriene B4 is a physiologically relevant endogenous peroxisome proliferator-activated receptor-α agonist. Esposito, E. et al. PPAR-α contributes to the anti-inflammatory activity of verbascoside in a model of inflammatory bowel disease in mice. PPAR Res. Dubrac, S. & Schmuth, M. PPAR-α in cutaneous inflammation. Delayre-Orthez, C. et al. PPARα downregulates airway inflammation induced by lipopolysaccharide in the mouse. Lu, P. et al. Chemical screening identifies ROCK1 as a regulator of migrasome formation. Cell Discov. Saunders, C. A., Majumdar, R., Molina, Y., Subramanian, B. C. & Parent, C. A. Genetic manipulation of PLB-985 cells and quantification of chemotaxis using the underagarose assay. Methods Cell Biol. Heit, B. & Kubes, P. Measuring chemotaxis and chemokinesis: the under-agarose cell migration assay. Deniset, J. F. & Kubes, P. Neutrophil heterogeneity: bona fide subsets or polarization states? Gupta, S. et al. Sex differences in neutrophil biology modulate response to type I interferons and immunometabolism. Natl Acad. Blazkova, J. et al. Multicenter systems analysis of human blood reveals immature neutrophils in males and during pregnancy. McQuin, C. et al. CellProfiler 3.0: next-generation image processing for biology. PLoS Biol. Stirling, D. R. et al. CellProfiler 4: improvements in speed, utility and usability. BMC Bioinform. Yousefi, S., Mihalache, C., Kozlowski, E., Schmid, I. & Simon, H. U. Viable neutrophils release mitochondrial DNA to form neutrophil extracellular traps. Cell Death Differ. Lood, C. et al. Neutrophil extracellular traps enriched in oxidized mitochondrial DNA are interferogenic and contribute to lupus-like disease. Pendergrass, W., Wolf, N. & Poot, M. Efficacy of MitoTracker GreenTM and CMXrosamine to measure changes in mitochondrial membrane potentials in living cells and tissues. Part A 61A, 162–169 (2004). Neubert, E. et al. Chromatin swelling drives neutrophil extracellular trap release. Fuchs, T. A. et al. Novel cell death program leads to neutrophil extracellular traps. Cell Biol. Rohrbach, A. S., Slade, D. J., Thompson, P. R. & Mowen, K. A. Activation of PAD4 in NET formation. Sollberger, G. et al. Gasdermin D plays a vital role in the generation of neutrophil extracellular traps. Article PubMed Desai, J. et al. PMA and crystal‐induced neutrophil extracellular trap formation involves RIPK1‐RIPK3‐MLKL signaling. Stankova, J., Turcotte, S., Harris, J. & Rola-Pleszczynski, M. Modulation of leukotriene B4 receptor-1 expression by dexamethasone: potential mechanism for enhanced neutrophil survival. Pétrin, D., Turcotte, S., Gilbert, A.-K., Rola-Pleszczynski, M. & Stankova, J. The anti-apoptotic effect of leukotriene B4 in neutrophils: a role for phosphatidylinositol 3-kinase, extracellular signal-regulated kinase and Mcl-1. Cell Signal. Article PubMed Gee, D. J. et al. Dimethylsulfoxide exposure modulates HL-60 cell rolling interactions. Breitman, T. R., Collins, S. J. & Keene, B. R. Replacement of serum by insulin and transferrin supports growth and differentiation of the human promyelocytic cell line, HL-60. Rowat, A. C. et al. Nuclear envelope composition determines the ability of neutrophil-type cells to passage through micron-scale constrictions. Olins, A. L., Ernst, A., Zwerger, M., Herrmann, H. & Olins, D. E. An in vitro model for Pelger-Huët anomaly. Dusse, L. M. S. et al. Acquired Pelger–Huët: what does it really mean? de Oliveira, S., Boudinot, P., Calado, Â. & Mulero, V. Duox1-derived H2O2 modulates Cxcl8 expression and neutrophil recruitment via JNK/c-JUN/AP-1 signaling and chromatin modifications. Article PubMed Hamam, H., Khan, M. & Palaniyar, N. Histone acetylation promotes neutrophil extracellular trap formation. Zhang, C. et al. Histone acetylation: novel target for the treatment of acute lymphoblastic leukemia. Ferguson, A. D. et al. Crystal structure of inhibitor-bound human 5-lipoxygenase-activating protein. Mancini, J. A. et al. 5‐lipoxygenase‐activating protein is an arachidonate binding protein. FEBS Lett. Gábor, M. Models of acute inflammation in the ear. Methods Mol. Xu, Y., Cohen, E., Johnson, C. N., Parent, C. A. & Coulombe, P. A. Repeated stress to the skin amplifies neutrophil infiltration in a keratin 17- and PKCα-dependent manner. PLoS Biol. Bernut, A., Loynes, C. A., Floto, R. A. & Renshaw, S. A. Deletion of cftr leads to an excessive neutrophilic response and defective tissue repair in a zebrafish model of sterile inflammation. Wang, J. et al. Visualizing the function and fate of neutrophils in sterile injury and repair. Hoste, E. et al. Epithelial HMGB1 delays skin wound healing and drives tumor initiation by priming neutrophils for NET formation. Wong, S. L. et al. Diabetes primes neutrophils to undergo NETosis, which impairs wound healing. Serhan, C. N. & Levy, B. D. Resolvins in inflammation: emergence of the pro-resolving superfamily of mediators. Colom, B. et al. Leukotriene B4-neutrophil elastase axis drives neutrophil reverse transendothelial cell migration in vivo. Temme, S. et al. Technical advance: monitoring the trafficking of neutrophil granulocytes and monocytes during the course of tissue inflammation by noninvasive 19F MRI. Article PubMed Soehnlein, O., Lindbom, L. & Weber, C. Mechanisms underlying neutrophil-mediated monocyte recruitment. Jiménez‐Alcázar, M. et al. Impaired DNase1‐mediated degradation of neutrophil extracellular traps is associated with acute thrombotic microangiopathies. Article PubMed Li, T. et al. TBK1 recruitment to STING mediates autoinflammatory arthritis caused by defective DNA clearance. Malíčková, K. et al. Impaired deoxyribonuclease I activity in patients with inflammatory bowel diseases. Autoimmune Dis. Yasutomo, K. et al. Mutation of DNASE1 in people with systemic lupus erythematosus. Garcia, G. et al. Impaired balance between neutrophil extracellular trap formation and degradation by DNases in COVID-19 disease. Jiménez-Alcázar, M. et al. Host DNases prevent vascular occlusion by neutrophil extracellular traps. Article PubMed Oved, J. H. et al. Neutrophils promote clearance of nuclear debris following acid-induced lung injury. Shimada, O. et al. Detection of deoxyribonuclease I along the secretory pathway in Paneth cells of human small intestine. Subramanian, B. C., Moissoglu, K. & Parent, C. A. The LTB4–BLT1 axis regulates the polarized trafficking of chemoattractant GPCRs during neutrophil chemotaxis. Cell Sci. Iizuka, Y. et al. Protective role of the leukotriene B4 receptor BLT2 in murine inflammatory colitis. FASEB J. Zinn, S. et al. The leukotriene B4 receptors BLT1 and BLT2 form an antagonistic sensitizing system in peripheral sensory neurons. Li, G. et al. Hepatic peroxisome proliferator‐activated receptor α mediates the major metabolic effects of Wy‐14643. Stienstra, R. et al. The interleukin-1 receptor antagonist is a direct target gene of PPARα in liver. Ghosh, A. et al. Activation of peroxisome proliferator-activated receptor α induces lysosomal biogenesis in brain cells. Saika, A. et al. Mead acid inhibits retinol-induced irritant contact dermatitis via peroxisome proliferator-activated receptor alpha. Xu, H. E. et al. Structural basis for antagonist-mediated recruitment of nuclear co-repressors by PPARα. Briguglio, E. et al. WY-14643, a potent peroxisome proliferator activator receptor-α PPAR-α agonist ameliorates the inflammatory process associated to experimental periodontitis. PPAR Res 2010, 1–13 (2010). D'Cruz, A. A. et al. The pseudokinase MLKL activates PAD4-dependent NET formation in necroptotic neutrophils. Sedej, I. et al. Extracellular vesicle‐bound DNA in urine is indicative of kidney allograft injury. Wortzel, I. et al. Unique structural configuration of EV-DNA primes Kupffer cell-mediated antitumor immunity to prevent metastatic progression. Yokoi, A. et al. Mechanisms of nuclear content loading to exosomes. Kwon, M., Leibowitz, M. L. & Lee, J.-H. Small but mighty: the causes and consequences of micronucleus rupture. Lusk, C. P. & King, M. C. Rotten to the core: why micronuclei rupture. Guo, X. et al. Understanding the birth of rupture-prone and irreparable micronuclei. Raab, M. et al. ESCRT III repairs nuclear envelope ruptures during cell migration to limit DNA damage and cell death. Mackenzie, K. J. et al. cGAS surveillance of micronuclei links genome instability to innate immunity. Sato, Y. & Hayashi, M. T. Micronucleus is not a potent inducer of the cGAS/STING pathway. Life Sci. Tsering, T., Nadeau, A., Wu, T., Dickinson, K. & Burnier, J. V. Extracellular vesicle-associated DNA: ten years since its discovery in human blood. Cell Death Dis. Jacobson, E. C. et al. Migration through a small pore disrupts inactive chromatin organization in neutrophil-like cells. BMC Biol. et al. Nuclear softening expedites interstitial cell migration in fibrous networks and dense connective tissues. Mehl, J. L. et al. Blockage of lamin-A/C loss diminishes the pro-inflammatory macrophage response. Denholtz, M. et al. Upon microbial challenge, human neutrophils undergo rapid changes in nuclear architecture and chromatin folding to orchestrate an immediate inflammatory gene program. Genes Dev. Herrera-Uribe, J. et al. Changes in H3K27ac at gene regulatory regions in porcine alveolar macrophages following LPS or PolyIC exposure. Stephens, A. D. et al. Chromatin histone modifications and rigidity affect nuclear morphology independent of lamins. Popova, E. Y., Claxton, D. F., Lukasova, E., Bird, P. I. & Grigoryev, S. A. Epigenetic heterochromatin markers distinguish terminally differentiated leukocytes from incompletely differentiated leukemia cells in human blood. Barth, T. K. & Imhof, A. Fast signals and slow marks: the dynamics of histone modifications. Trends Biochem. Hsu, B. E. et al. Immature low-density neutrophils exhibit metabolic flexibility that facilitates breast cancer liver metastasis. Sun, R. et al. Dysfunction of low-density neutrophils in peripheral circulation in patients with sepsis. van Grinsven, E. et al. Immature neutrophils released in acute inflammation exhibit efficient migration despite incomplete segmentation of the nucleus. Article PubMed Miotto, G. et al. Insight into the mechanism of ferroptosis inhibition by ferrostatin-1. Redox Biol. Sun, Q. et al. Cholesterol mediated ferroptosis suppression reveals essential roles of coenzyme Q and squalene. Poplimont, H. et al. Neutrophil swarming in damaged tissue is orchestrated by connexins and cooperative calcium alarm signals. Al-Shabanah, O. A., Mansour, M. A. & Elmazar, M. M. Enhanced generation of leukotriene B4 and superoxide radical from calcium ionophore (A23187) stimulated human neutrophils after priming with interferon-α. Kenny, E. F. et al. Diverse stimuli engage different neutrophil extracellular trap pathways. Pedrera, L. et al. Ferroptotic pores induce Ca2+ fluxes and ESCRT-III activation to modulate cell death kinetics. Cell Death Differ. Galluzzi, L. et al. Molecular mechanisms of cell death: recommendations of the Nomenclature Committee on Cell Death 2018. Cell Death Differ. Pedrera, L., Ros, U. & García-Sáez, A. J. Calcium as a master regulator of ferroptosis and other types of regulated necrosis. Cell Calcium 114, 102778 (2023). Isles, H. M. et al. Pioneer neutrophils release chromatin within in vivo swarms. Saffarzadeh, M. et al. Neutrophil extracellular traps directly induce epithelial and endothelial cell death: a predominant role of histones. PLoS ONE 7, e32366 (2012). Cahilog, Z. et al. The role of neutrophil NETosis in organ injury: novel inflammatory cell death mechanisms. Article PubMed Mathis, S. P., Jala, V. R., Lee, D. M. & Haribabu, B. Nonredundant roles for leukotriene B4 Receptors BLT1 and BLT2 in inflammatory arthritis. Tatsumi, R. et al. Stepwise phosphorylation of BLT1 defines complex assemblies with β-arrestin serving distinct functions. FASEB J. Xu, Y., Denning, K. L. & Lu, Y. PPARα agonist WY-14,643 induces the PLA2/COX-2/ACOX1 pathway to enhance peroxisomal lipid metabolism and ameliorate alcoholic fatty liver in mice. Paloschi, M. V. et al. Cytosolic phospholipase A2-α participates in lipid body formation and PGE2 release in human neutrophils stimulated with an l-amino acid oxidase from Calloselasma rhodostoma venom. Norris, P. C., Gosselin, D., Reichart, D., Glass, C. K. & Dennis, E. A. Phospholipase A α regulates eicosanoid class switching during inflammasome activation. Natl Acad. Babatunde, K. A., Ayuso, J. M., Kerr, S. C., Huttenlocher, A. & Beebe, D. J. Microfluidic systems to study neutrophil forward and reverse migration. Loynes, C. A. et al. PGE 2 production at sites of tissue injury promotes an anti-inflammatory neutrophil phenotype and determines the outcome of inflammation resolution in vivo. Levy, B. D., Clish, C. B., Schmidt, B., Gronert, K. & Serhan, C. N. Lipid mediator class switching during acute inflammation: signals in resolution. Baig, S. et al. Baseline elevations of leukotriene metabolites and altered plasmalogens are prognostic biomarkers of plaque progression in systemic lupus erythematosus. Bonyek-Silva, I. et al. LTB4-driven inflammation and increased expression of ALOX5/ACE2 during severe COVID-19 in individuals with diabetes. van der Meer, A. J. et al. Systemic inflammation induces release of cell-free DNA from hematopoietic and parenchymal cells in mice and humans. Blood Adv. Cheng, S., Davis, S. & Sharma, S. Maternal‐fetal cross talk through cell‐free fetal DNA, telomere shortening, microchimerism, and inflammation. Linney, M. D. et al. Microbial sources of exocellular DNA in the ocean. Wen, F., Curlango‐Rivera, G., Huskey, D. A., Xiong, Z. & Hawes, M. C. Visualization of extracellular DNA released during border cell separation from the root cap. Zhang, X., Zhuchenko, O., Kuspa, A. & Soldati, T. Social amoebae trap and kill bacteria by casting DNA nets. Liu, X. et al. PAD4 takes charge during neutrophil activation: Impact of PAD4 mediated NET formation on immune‐mediated disease. Cantin, A. M. DNase I acutely increases cystic fibrosis sputum elastase activity and its potential to induce lung hemorrhage in mice. Care Med 157, 464–469 (1998). Metersky, M. & Chalmers, J. Bronchiectasis insanity: doing the same thing over and over again and expecting different results? O'Donnell, A. E., Barker, A. F., Ilowite, J. S. & Fick, R. B. Treatment of idiopathic bronchiectasis with aerosolized recombinant human DNase I. Article PubMed Ngo, A. T. P. et al. Platelet factor 4 limits neutrophil extracellular trap- and cell-free DNA-induced thrombogenicity and endothelial injury. JCI Insight 8, e171054 (2023). Mai, S. H. C. et al. Delayed but not early treatment with DNase reduces organ damage and improves outcome in a murine model of sepsis. Kremserova, S. & Nauseef, W. M. in Methods in Molecular Biology, Vol. 2087 (eds Quinn, M. & DeLeo, F.) 33–42 (Humana, 2020). & Lee, H.-H. A new chemical complex can rapidly concentrate lentivirus and significantly enhance gene transduction. Shechter, D., Dormann, H. L., Allis, C. D. & Hake, S. B. Extraction, purification and analysis of histones. Graham, J. M. Fractionation of subcellular organelles. Cell Biol. Théry, C. et al. Minimal information for studies of extracellular vesicles 2018 (MISEV2018): a position statement of the International Society for Extracellular Vesicles and update of the MISEV2014 guidelines. Rider, M. A., Hurwitz, S. N. & Meckes, D. G. ExtraPEG: a polyethylene glycol-based method for enrichment of extracellular vesicles. Truckenbrodt, S., Sommer, C., Rizzoli, S. O. & Danzl, J. G. A practical guide to optimization in X10 expansion microscopy. Gambarotto, D. et al. Imaging cellular ultrastructures using expansion microscopy (U-ExM). Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. Article PubMed Mastronarde, D. N. & Held, S. R. Automated tilt series alignment and tomographic reconstruction in IMOD. Article PubMed Chen, M. et al. Convolutional neural networks for automated annotation of cellular cryo-electron tomograms. McAlister, G. C. et al. MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes. Collie, S. et al. underagarose-analysis. Zenodo https://doi.org/10.5281/zenodo.15080547 (2025). Download references We thank the past and present members of the Parent and Coulombe laboratories for advice and support. We thank the Platelet Pharmacology and Physiology Core at the University of Michigan for providing human blood from healthy volunteers, the proteomics resource facility and V. Basrur for assistance with mass spectrometry data acquisition and analysis, and the microscopy core facility for assistance with TEM sample processing. We also thank P. Hanson (University of Michigan) for valuable suggestions and D. Wang and D. Sinha from the Coulombe and Parent laboratories for their help with animal experiments. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. This work was supported by funding from the University of Michigan School of Medicine (C.A.P. and predoctoral (AWD025905) (S.P.C.) fellowship awards from an American Heart Association, a Life Sciences Institute cubed award, the Arnold and Mabel Beckmann Foundation award to the University of Michigan Cryo-EM facility and by several grant awards from the National Institutes of Health, namely T32 training program in cell and molecular biology GM145470 (S.P.C. ), T32 training program in translational research GM141840 (M.F. These authors contributed equally: Samuel P. Collie, Yang Xu. Life Sciences Institute, University of Michigan, Ann Arbor, MI, USA Subhash B. Arya, Samuel P. Collie, Yang Xu, Martin Fernandez, Shyamal Mosalaganti & Carole A. Cellular and Molecular Biology Graduate Program, University of Michigan Medical School, Ann Arbor, MI, USA Samuel P. Collie Department of Pharmacology, University of Michigan Medical School, Ann Arbor, MI, USA Yang Xu & Carole A. Department of Biophysics, University of Michigan, Ann Arbor, MI, USA Martin Fernandez & Shyamal Mosalaganti Department of Internal Medicine, University of Michigan Medical School, Ann Arbor, MI, USA Jonathan Z. Sexton Department of Medicinal Chemistry, University of Michigan, Ann Arbor, MI, USA Jonathan Z. Sexton Department of Cell and Developmental Biology, University of Michigan Medical School, Ann Arbor, MI, USA Shyamal Mosalaganti, Pierre A. Coulombe & Carole A. Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI, USA Pierre A. Coulombe Rogel Cancer Center, University of Michigan Medical School, Ann Arbor, MI, USA Pierre A. Coulombe & Carole A. You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Carole A. The authors declare no competing interests. Nature Cell Biology thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Fourfold expansion microscopy images, representative of three independent experiments, showing fixed PMNs chemotaxing towards 100 nM LTB4, immunostained post-expansion for 5LO (yellow) and FLAP (magenta), and co-stained with Vybrant™ Dil (cyan). Orange dashed line outlines cell boundary, and red inset is zoomed and presented as individual channels (grayscale invert) in panel b. c. 3D-volume rendering of the cropped NE-MVB from panel a, sectioned through the centre. The scale is 5 µm, and 1 µm in the inset. Representative of 10 images from three independent experiments. d. Histogram depicting the intensity profile of the objects along the white dashed arrow in the merged image in panel b. e. Representative immuno-TEM images of LTB4-activated PMNs stained using anti-5LO antibody, showing the presence of 5LO (electron-dense dots, gold particles) within ~200 nm ILVs present inside the MVBs. The scale is 5 µm and 200 nm in the inset. N denotes nucleus. Representative of 7 images from two independent experiments. f. Representative TEM images of PMNs migrating towards LTB4 showing the presence of chromatin-like structures in cytoplasmic MVBs alongside ILVs. The scale is 5 µm, 200 nm in the inset. N denotes nucleus. Representative of 15 images from three independent experiments. Source numerical data is available in the source data file. Source data An overlay image of PMNs on fibrinogen-fibronectin coated Quantifoil grids stimulated with 100 nM LTB4 for 15 min in the presence of SYTOXgreen, plunge-frozen, and imaged on Leica Stellaris-5 cryo-confocal, showing the reflected light (grey) and extracellular DNA (green). Dashed yellow lines highlight PMNs on the grid square. Representative of two biological replicates. The scale bar is 10 μm. A low-magnification TEM image (6500x) of the highlighted region (red box) in panel a, is overlaid with the SYTOXgreen fluorescence signal. The cyan box indicates the area where cryo-ET data was acquired. The scale bar is 500 nm. c. A slice through the tomogram of the region highlighted (cyan box) in panel b, with insets showing bead-on-string-like structures (yellow arrowheads). The scale bar is 100 nm, and in the inset, it is 10 nm. d. Airyscan microscopy image of Hoechst 33342 (grey) stained PMN showing the extracellular distribution (white inset, zoomed below) of TSPAN4 (magenta) and 5LO (cyan). Cell is outlined in yellow, and scale is 5 µm, in the inset it is 1 µm. Representative of two biological replicates. Source data Time-lapse of PMNs stained with Mitotracker red CMXros (red, mitochondria), Hoechst (blue, nuclei), and SYTOXgreen (green, extracellular DNA) (a) migrating towards LTB4 or (b) treated with 100 nM PMA. Insets in panel b show cell outline (red), nuclei (blue), and NETs (green) generated by CellProfiler. Scale is 5 μm. Images are representative of three independent experiments. See associated Supplementary Movies 3 and 5. c. Before-after aligned dot plot showing Mitotracker Red intensity in migrating PMNs before, during, and after DNA secretion within 1 hr. Total 30 data points (red circles) pooled from three independent experiments are presented as mean ± s.e.m. (black lines), with multiplicity-adjusted P values from ordinary one-way ANOVA. d. Scatter dot plot showing percentage of DNA-secreting PMNs chemotaxis towards LTB4 with or without MitoTEMPO (10 μM). Data points (red dots) from three independent experiments are presented as mean ± s.e.m. (black lines), with P value from two-tailed ratio paired t-test. e. Airyscan microscopy image of Mitotracker Red CMXros (magenta) stained PMNs chemotaxing towards LTB4, fixed and immunostained for FLAP (yellow) and Hoechst 33342 (cyan). Orange dashed line outlines cell and inset is zoomed as individual channels (invert grayscale). The 3D-volume is shown below. Images are representative of three independent experiments and scale is 5 μm. f. Graph showing nuclei area extent over time in PMNs treated with PMA or migrating towards LTB4. Dots represent mean nuclear area extent per 40,000 μm2. Blue/red dots indicate percentage of PMNs without/with SYTOXgreen staining. Thick black line represents non-linear regression. Data is representative of three independent experiments. g. Graph showing the percentage of PMNs lysed over time. Data points from three independent experiments are plotted as mean (red/blue dots) ± s.e.m. (black lines). h. Graph showing the percentage of maximum lactate dehydrogenase (LDH) activity in supernatants of PMNs treated with either DMSO, PMA (20 nM), or LTB4 (100 nM) for 2 hrs. Data points from three independent experiments (similarly coloured circles) are presented as mean ± s.e.m. (black lines), with multiplicity-adjusted P values from RM one-way ANOVA. Scatter dot plot showing percentage of DNA-secreting PMNs migrating towards LTB4 in a 40,000 μm2 observation window for 1 hr either (i) with or without PAD4 and NOX2 inhibitors, GSK484 (2 µM) and GSK2795039 (10 µM), respectively, or (j) cell death pathway inhibitors namely, Z-DEVD-FMK (apoptosis, 10 μM), ferrostatin-1 (ferroptosis, 5 μM), and GSK872 (necroptosis, 10 μM). Data points from three (or more) independent experiments (similarly coloured circles) are presented as mean ± s.e.m., with multiplicity-adjusted P values from RM one-way ANOVA (i) and mixed-effect analysis (j). See associated Supplementary Movie 6 and 7. Source numerical data is available in the source data file. Source data Scatter dot plots showing the change in the (a) nuclear form factor and (b) cell eccentricity during DNA secretion compared to PMNs without DNA secretion. Data points (red circles) pooled from five independent experiments are plotted as mean ± s.e.m. (black lines). Multiplicity-adjusted P values obtained using ordinary one-way ANOVA for (before vs. during vs. after comparison) DNA-secreting cells and the Mann-Whitney test (before DNA secretion vs. no DNA secretion comparison) are shown. Source numerical data is available in the source data file. Source data Western blot images representative of three independent experiments, showing the levels of lamin A/C, lamin B1, lamin B2, and LBR in SCR, LMNA KO, and LMNA/LBR KO dHL60 cell lysates. GAPDH is loading control. The molecular weights (kilodaltons, kDa) are indicated on left. b. Airyscan microscopy images of PMNs, SCR, LMNA KO, and LMNA/LBR KO dHL60 cells migrating towards fMLF, fixed and stained for LBR (magenta, nuclear envelope) and Hoechst (cyan, nucleus). Presented “sum of slices” projections are representative of three independent experiments. Dashed white/black outlines indicate cell shape. Scale is 5 μm. c-f. Scatter dot plots showing (c) NE to cytoplasm LBR intensity ratio, (d) nuclei form factor, (e) NE invaginations, and (f) heterochromatin spots in dHL60 neutrophils. Data points (red circles) pooled from three independent experiments are presented as mean ± s.e.m., with multiplicity-adjusted P values from ordinary one-way ANOVA. g. Scatter plot (left) and histogram (right) showing the gating strategy used for the analysis of flow cytometry data plotted in Fig. 4e-g. Graphs are representative of DMSO-treated SCR dHL60. Source numerical data and unprocessed western blots are available in the source data file. Source data Graphs showing the fold change in the gene ontology (GO, biological process) profile of the associated (a) downregulated and (b) upregulated proteins in LMNA KO dHL60 cells relative to SCR dHL60 cells, quantified using tandem mass tagging (TMT)-based mass spectroscopy analysis. c-f. Scatter dot plots showing (c) the number of cells migrating towards fMLF for 1 hr, (d) median speed, (e) median directionality, and (f) cell eccentricity. Data is plotted as mean ± s.e.m. of 9, 8, and 5 independent experiments (red circles) for SCR, LMNA KO, and LMNA/LBR KO samples, respectively. For cell eccentricity, a total of 29, 45, and 16 data points for SCR, LMNA KO, and LMNA/LBR KO, were pooled from three independent experiments. The multiplicity-adjusted P values calculated using mixed-effect analysis (c-e) and ordinary one-way ANOVA (f) are shown. Source numerical data is available in the source data file. Source data Cell tracks of PMNs pretreated with either DMSO (vehicle) or MK886 (1 μM), migrating towards LTB4 (x-axis) in the presence or absence of rDNase I (10 U/mL) for 1 h. Graphs display the migration of 100 randomly selected tracks from each condition. The colour-coded time scale is shown on the right. Presented tracks are representative of five independent experiments. See associated Supplementary Movie 10. b-d. Bar graphs showing the (b) number of PMNs migrated towards LTB4 within 1 hr, (c) average directionality, and (d) average speed in the presence or absence of rDNase I. Data points (black circles) from five independent experiments are presented as mean ± s.e.m., with multiplicity-adjusted P values from RM one-way ANOVA. e. Bar graph showing the average directionality and speed of PMNs chemotaxing towards LTB4 in the presence or absence of RNase (10 u/mL). Data points (black dots) from five independent experiments are presented as mean ± s.e.m., with P values calculated using two-tailed student's t-test. Source numerical data are available in the source data file. Source data Airyscan microscopy images of ear cryosections (20 µm thick) treated with acetone/TPA for the indicated duration in mice injected with either rDNase I or PBS showing the temporal distribution of citrullinated histone H3 (magenta) and DAPI (grey, nuclei). Presented ‘sum of slices' projection is representative of three independent experiments. Scale is 100 µm, and 20 µm in the inset. Source data a. Schematic illustrating the antibody binding sites on FLAP and the principle of the PLA used to assess the presence of SEADs in inflamed ears. b. Airyscan microscopy images of ear cryosections (20 μm) from mice treated with TPA for 12 hrs, showing the status of PLA dots (magenta) in either FLAP- or dsDNA-only antibody conditions, used to test the nonspecific PLA signal. Sections were co-stained with DAPI (nucleus). Presented ‘sum of slices' is representative of three independent experiments, and Scale is 100 μm. c. Airyscan microscopy images of ear cryosections (20 μm) from a mouse injected with either PBS or rDNase I and treated with TPA for 12 h, showing the status of CD63-citH3 PLA dots (magenta) in indicated conditions. Sections were co-stained with DAPI (nucleus). Zoomed insets are presented as inverted grayscale above respective conditions. Presented ‘sum of slices' is representative of two independent experiments, and scale is 100 μm, in the inset it is 20 µm. Source data a. Scatter dot plots showing the gating strategy used for quantifying Ly6G+ ICAM1high CXCR1low reverse-transmigrated neutrophils as shown in Fig. b-c. Airyscan microscopy images of ear cryosections (20 µm thick) treated with (b) either acetone/TPA for the indicated duration in mice injected with either rDNase I or PBS or (c) added topical treatment of PPARα agonist/antagonist, showing the temporal distribution of F4/80 (magenta, macrophages) and DAPI (grey, nuclei). Presented ‘sum of slices' projection is representative of three independent experiments. Scale is 100 μm. Source data List of genes with significantly altered expression in LMNA KO dHL60 relative to SCR dHL60. Key resources. Tomogram and segmentation of data shown in Fig. Nonlytic, repetitive and rapid secretion of DNA from chemotaxing PMNs. PMNs stained with CellMask Orange (PM) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen imaged using Airyscan microscopy. Images acquired at 30 s intervals are presented as three frames per second. Scale bar, 5 µm. Phenotypes of DNA secreted from chemotaxing PMNs. PMNs stained with MitoTracker Red CMXros (mitochondria) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen imaged using confocal microscopy. Images acquired at 30 s intervals are presented as three frames per second. Scale bar, 5 µm. Cyan arrow marks DNA trails, white triangle marks ‘DNA blobs', and hollow white triangles mark ‘attached-DNA blobs'. Right panel shows the outline of the neutrophils (red, attached to DNA; white, no DNA within 1 µm of cell membrane) as generated by object segmentation using CellProfiler. Cyan and magenta outlines denote the attached and released DNA, respectively. The secretion of DNA from chemotaxing PMNs is dependent on SMase activity. PMNs stained with CellMask Orange (PM) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen and either DMSO (vehicle control) or GW4869 (nSMase inhibitor) imaged using confocal microscopy. Images acquired at 30 s intervals are presented as three frames per second. Scale bar, 5 µm. Cyan arrow marks DNA trails, white triangle marks ‘DNA blobs', and hollow white triangles mark ‘attached-DNA blobs'. PMA-induced suicidal NET release in PMNs. PMNs stained with CellMask Orange (PM) and Hoechst 33342 (nuclei) stimulated with PMA (100 nM) in the presence of SYTOXgreen and imaged using Airyscan microscopy. Images acquired at 18 s intervals are presented as three frames per second. Scale bar, 5 µm. The secretion of DNA from chemotaxing PMNs is independent of PAD4 and NOX2 activity. PMNs stained with CellTrackerCMPTX (cell) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen, and either DMSO, PAD4 inhibitor and NOX2 inhibitor imaged using confocal microscopy. Images acquired at 45 s intervals are presented as one frame per second. White triangle marks ‘DNA blobs' and hollow white triangles mark ‘attached-DNA blobs'. Effect of various cell death pathway inhibitors on the secretion of DNA from chemotaxing PMNs. PMNs stained with CellMask Orange (PM) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen, and either DMSO, apoptosis, ferroptosis or necroptosis inhibitors imaged using confocal microscopy. Images acquired at 30 s intervals are presented as three frames per second. Cyan arrow marks DNA trails, white triangle marks ‘DNA blobs', and hollow white triangles mark ‘attached-DNA blobs'. LBR loss inhibits DNA secretion in chemotaxing dHL60 cells. SCR, LMNA KO and LMNA/LBR KO dHL60 cells stained with CellMask Orange (PM) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen imaged using confocal microscopy. Images acquired at 30 s intervals are presented as three frames per second. Hollow white triangles mark ‘attached-DNA blobs'. Histone acetylation mediates the secretion of DNA from chemotaxing PMNs. PMNs stained with CellMask Orange (PM) and Hoechst 33342 (nuclei) migrating towards LTB4 in the presence of SYTOXgreen, and either DMSO, HAT or HDAC inhibitors imaged using confocal microscopy. Images acquired at 30 s intervals are presented as three frames per second. White triangle marks ‘DNA blobs', and hollow white triangles mark ‘attached-DNA blobs'. Effects of MK886 and rDNase I treatment on PMN chemotaxis. PMNs stained with Hoechst 33342 migrating towards LTB4 in the presence of either DMSO, rDNase I, MK886, or MK886 + rDNase I, were imaged using fluorescence microscopy and analysed using Trackmate on ImageJ. Images acquired every 45 s are presented at a rate of five frames per second. Circles indicate the individual cells as identified by Trackmate, and lines are colour-coded for the duration of migration from blue (earlier) to red (later). Scale bar, 200 µm. Statistical source data. Unprocessed western blots. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Arya, S.B., Collie, S.P., Xu, Y. et al. Neutrophils secrete exosome-associated DNA to resolve sterile acute inflammation. Nat Cell Biol  (2025). Download citation Received: 22 May 2024 Accepted: 09 April 2025 Published: 22 May 2025 DOI: https://doi.org/10.1038/s41556-025-01671-4 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Cell Biology (Nat Cell Biol) ISSN 1476-4679 (online) ISSN 1465-7392 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            