
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-26</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09774-6'>Thalamocortical transcriptional gates coordinate memory stabilization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 16:14:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The molecular mechanisms that enable memories to persist over long timescales from days to weeks and months are still poorly understood1. Here, to develop insights into this process, we created a behavioural task in which mice formed multiple memories but only consolidated some, while forgetting others, over the span of weeks. We then monitored circuit-specific molecular programs that diverged between consolidated and forgotten memories. We identified multiple distinct waves of transcription, that is, cellular macrostates, in the thalamocortical circuit that defined memory persistence. Of note, a small set of transcriptional regulators orchestrated broad molecular programs that enabled entry into these macrostates. Targeted CRISPR-knockout studies revealed that although these transcriptional regulators had no effects on memory formation, they had prominent, causal and strikingly time-dependent roles in memory stabilization. In particular, the calmodulin-dependent transcription factor CAMTA1 was required for initial memory maintenance over days, whereas the transcription factor TCF4 and the histone methyltransferase ASH1L were required later to maintain memory over weeks. These results identify a critical CAMTA1–TCF4–ASH1L thalamocortical transcriptional cascade that is required for memory stabilization and put forth a model in which the sequential recruitment of circuit-specific transcriptional programs enables memory maintenance over progressively longer timescales. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Raw and processed scRNA-seq data from the mouse (accession GSE300871), raw and aligned ATAC-seq data (accession GSE304095), and raw and aligned ChIP–seq data (accession GSE304099) are available from the Gene Expression Omnibus. No new algorithms were developed for this paper. The analysis code will be available on RajasethupathyLab GitHub. Yadav, N., Toader, A. & Rajasethupathy, P. Beyond hippocampus: thalamic and prefrontal contributions to an evolving memory. Agranoff, B. W. & Klinger, P. D. Puromycin effect on memory fixation in the goldfish. Effect of acetoxycycloheximide and of an acetoxycycloheximide-puromycin mixture on cerebral protein synthesis and memory in mice. Squire, L. R. & Barondes, S. H. Actinomycin-D: effects on memory at different times after training. Igaz, L. M., Vianna, M. R. M., Medina, J. H. & Izquierdo, I. Two time periods of hippocampal mRNA synthesis are required for memory consolidation of fear-motivated learning. Kandel, E. R. The molecular biology of memory storage: a dialogue between genes and synapses. & Kandel, E. R. Injection of the cAMP-responsive element into the nucleus of aplysia sensory neurons blocks long-term facilitation. Alberini, C. M., Ghirardi, M., Metz, R. & Kandel, E. R. C/EBP is an immediate-early gene required for the consolidation of long-term facilitation in aplysia. Induction of a dominant negative CREB transgene specifically blocks long-term memory in Drosophila. Bourtchuladze, R. et al. Deficient long-term memory in mice with a targeted mutation of the cAMP-responsive element-binding protein. Silva, A. J., Kogan, J. H., Frankland, P. W. & Kida, S. CREB and memory. Yin, J. C., Del Vecchio, M., Zhou, H. & Tully, T. CREB as a memory modulator: induced expression of a dCREB2 activator isoform enhances long-term memory in Drosophila. Bartsch, D., Casadio, A., Karl, K. A., Serodio, P. & Kandel, E. R. CREB1 encodes a nuclear activator, a repressor, and a cytoplasmic modulator that form a regulatory unit critical for long-term facilitation. Long-term memory is facilitated by cAMP response element-binding protein overexpression in the amygdala. Barco, A., Alarcon, J. M. & Kandel, E. R. Expression of constitutively active CREB protein facilitates the late phase of long-term potentiation by enhancing synaptic capture. Lin, H.-W., Chen, C.-C., de Belle, J. S., Tully, T. & Chiang, A.-S. CREBA and CREBB in two identified neurons gate long-term memory formation in Drosophila. Structural plasticity and memory. Carlezon, W. A. Jr, Duman, R. S. & Nestler, E. J. The many faces of CREB. Kandel, E. R. The molecular biology of memory: cAMP, PKA, CRE, CREB-1, CREB-2, and CPEB. Nader, K. Memory traces unbound. Alberini, C. M. Mechanisms of memory stabilization: are consolidation and reconsolidation similar or distinct processes? Dudai, Y. Reconsolidation: the advantage of being refocused. Wood, M. A., Hawk, J. D. & Abel, T. Combinatorial chromatin modifications and memory storage: a code for memory? From cellular to fear memory: an epigenetic toolbox to remember. & Mayford, M. R. The molecular and systems biology of memory. Holt, C. E., Martin, K. C. & Schuman, E. M. Local translation in neurons: visualization and function. The organization of recent and remote memories. Toader, A. C. et al. Anteromedial thalamus gates the selection and stabilization of long-term memories. Zeisel, A. et al. Molecular architecture of the mouse nervous system. Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. Heumos, L. et al. Pertpy: an end-to-end framework for perturbation analysis. Preprint at bioRxiv https://doi.org/10.1101/2024.08.04.606516 (2024). Characterization of cell fate probabilities in single-cell data with Palantir. Weiler, P., Lange, M., Klein, M., Pe'er, D. & Theis, F. CellRank 2: unified fate mapping in multiview single-cell data. Platt, R. J. et al. CRISPR-Cas9 knockin mice for genome editing and cancer modeling. Dudai, Y., Jan, Y. N., Byers, D., Quinn, W. G. & Benzer, S. dunce, A mutant of Drosophila deficient in learning. Davis, R. L. Physiology and biochemistry of Drosophila learning mutants. Quinn, W. G., Sziber, P. P. & Booker, R. The Drosophila memory mutant amnesiac. & Abbott, L. F. Cascade models of synaptically stored memories. Marco, A. et al. Mapping the epigenomic and transcriptomic interplay during memory formation and recall in the hippocampal engram ensemble. Santoni, G. et al. Chromatin plasticity predetermines neuronal eligibility for memory trace formation. Huentelman, M. J. et al. Calmodulin-binding transcription activator 1 (CAMTA1) alleles predispose human episodic memory performance. Teixeira, J. R., Szeto, R. A., Carvalho, V. M. A., Muotri, A. R. & Papes, F. Transcription factor 4 and its association with psychiatric disorders. Liu, H. et al. ASH1L mutation caused seizures and intellectual disability in twin sisters. Bintu, L. et al. Dynamics of epigenetic regulation at the single-cell level. Establishment, maintenance, and recall of inflammatory memory. Stern, S., Kirst, C. & Bargmann, C. I. Neuromodulatory control of long-term behavioral patterns and individuality across development. Combined small-molecule treatment accelerates maturation of human pluripotent stem cell-derived neurons. How the epigenome integrates information and reshapes the synapse. Mews, P. et al. From circuits to chromatin: the emerging role of epigenetics in mental health. Grandi, F. C., Modi, H., Kampman, L. & Corces, M. R. Chromatin accessibility profiling by ATAC-seq. Hrvatin, S. et al. Single-cell analysis of experience-dependent transcriptomic states in the mouse visual cortex. Nott, A., Schlachetzki, J. C. M., Fixsen, B. R. & Glass, C. K. Nuclei isolation of multiple brain cell types for omics interrogation. Azizi, E. et al. Single-cell map of diverse immune phenotypes in the breast tumor microenvironment. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Wolock, S. L., Lopez, R. & Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. Levine, J. H. et al. Data-driven phenotypic dissection of AML reveals progenitor-like cells that correlate with prognosis. Phillips, J. W. et al. A repeated molecular architecture across thalamic pathways. Finak, G. et al. MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. Fang, Z., Liu, X. & Peltz, G. GSEApy: a comprehensive package for performing gene set enrichment analysis in Python. Glasner, A. et al. Conserved transcriptional connectivity of regulatory T cells in the tumor microenvironment informs new combination cancer therapy strategies. Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. B. et al. ChEA3: transcription factor enrichment analysis by orthogonal omics integration. Whole mouse brain transcriptomic cell type atlas — 10x scRNAseq whole brain [dataset]. We thank the Rockefeller Flow Cytometry Resource Center; R. Chaligné at the Single Cell Analytics Innovation Lab (SAIL) at the MSKCC for helping to optimize the sorting and sequencing of scRNA-seq samples; H. Tan in the laboratory of J. Friedman for generously providing Rosa26-LSL-spCas9-eGFP mice; Z. Gershon for troubleshooting single-cell dissociations; E. Azizi, O. V. Goldman and A. Sziraki for helpful discussions related to scRNA-seq analysis; N. Blobel and S. Nakandakari for assistance with in vitro CRISPR manipulations and sharing reagents; Y. Kishi for sharing the ATAC-seq protocol; and N. Heintz, E. Azizi and members of the Rajasethupathy laboratory for comments on the manuscript. This work is supported by a Kavli Pilot Grant (to A.T.), the NIH Medical Scientist Training Program T32GM152349 (to C.C. ), a Cycle for Survival and an NCI grant P30 CA008748 (to the MSKCC Innovations Labs), funding from the Alan and Sandra Gerry Metastasis and Tumor Ecosystems Center (to SAIL, MSKCC), and grants from Irma T. Hirschl/Weill Caulier Trust, the Pershing Square Foundation, and the US National Institutes of Health under award numbers DP2AG058487 and RF1NS132047 (to P.R.). Figures 1a,d, 2a,b, 3a,d,h,k and 4a,g,i,j and Extended Data Figs. 8a and 9b were created using BioRender (https://biorender.com). All Allen Mouse Brain Atlas figures included in this work are adapted in accordance with the Allen Institute citation and rights policies, and used under their non-commercial open-access license. These authors contributed equally: Andrea Terceros, Celine Chen Laboratory of Neural Dynamics and Cognition, The Rockefeller University, New York, NY, USA Andrea Terceros, Celine Chen, Yujin Harada, Tim Eilers, Millennium Gebremedhin & Priya Rajasethupathy Epigenetics Research Innovation Lab, Memorial Sloan Kettering Cancer Center, New York, NY, USA Pierre-Jacques Hamard & Richard Koche Single-Cell Analytics Innovation Lab, Memorial Sloan Kettering Cancer Center, New York, NY, USA Computational and Systems Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar A.T. performed the optogenetic experiments, collected scRNA-seq samples with assistance from M.G. and developed the analysis pipeline for scRNA-seq. developed a pipeline for in vitro and in vivo validation of sgRNAs, performed the in vivo CRISPR manipulations and conducted the behavioural testing. performed the photometry experiments and analysis. performed qPCR and immunohistochemistry or western blot validations. collected samples for ATAC-seq. R.K. analysed the ATAC-seq data. collected samples for ChIP–seq. A.T. performed the downstream analysis. wrote the manuscript with input from all authors. supervised all aspects of the work. Correspondence to Priya Rajasethupathy. The authors declare no competing interests. Nature thanks Jian Zhou and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Discrimination indices (DI) of learning and retrieval performance of mice exposed to shuffled cue-outcomes, n = 8 mice, DIs from individual mice shown (faded lines), with mean ± SEM (solid line). Dashed black line indicates DI = 0 (at chance). b, Optogenetic inhibition during recent and remote retrieval, n = 7 HPC mCherry control, n = 8 HPC stGtACR2 (inhibitory opsin), n = 7 ACC mCherry control, n = 7 ACC stGtACR2, individual animals shown (faded lines), with mean ± SEM (solid line). Light delivered during cue periods of each trial. Quantification of discrimination indices between HR and aversive lick rates, *** P < 0.0001 between HPC mCherry and HPC stGtACR2 at recent and remote retrieval, ***P = 0.0001 between ACC mCherry and ACC stGtACR2 at remote retrieval, One-way ANOVA with Bonferroni correction. c, Discrimination indices of learning performance in HR or LR for mice receiving inhibition during retrieval days, n = 7-8 mice per cohort, individual data points shown (faded lines), with mean ± SEM (solid line). Dashed red line represents learning criteria set as discrimination index ≥ 0.3. d, Discrimination indices of learning performance in HR or LR for mice receiving inhibition during training days, n = 7-9 mice per cohort, individual animals shown (faded lines), with mean ± SEM (solid line). ACC, anterior cingulate cortex. BLA, basolateral amygdala. EC, entorhinal cortex. HR, high repetition. LR, low repetition. RSC, retrosplenial cortex. SSFO, stabilized step-function opsin. stGtACR2, soma-targeted Guillardia theta anion channelrhodopsin-2. a, Discrimination indices of learning performance in HR and LR contexts of mice used for scRNA sequencing, n = 23 mice for ANT and 24 mice for ACC, individual animals shown (faded lines), with mean (solid line). b, Representative lick traces from one mouse showing trial averages of lick rate (Hz) in HR and LR at recent, mid and remote retrieval, data are mean (solid line) ± SEM (shaded area), n = 30-40 trials. c, Memory performance in HR and LR contexts of mice used for scRNA sequencing, n = 42 mice, data are mean ± SEM (error bars). d, f, Library size of ANT (d) or ACC (f) samples, n = 9 samples each region, median and interquartile range depicted (lower bound = 25th percentile, upper bound = 75th percentile, lower whisker = smallest data point ≥ (Q1 − 1.5 × IQR), upper whisker = largest data point ≤ (Q3 + 1.5 × IQR)). e, g, UMAP visualization of all cells from ANT (e) (n = 176566 cells) or ACC (g) (n = 145327 cells), clustered based on transcriptional profile and colored by cluster number. h, m, UMAP sub-clustering of cells identified as neurons in ANT (n = 5535 cells) (h) or ACC, (n = 5671 cells) (m) colored by time point. i, n, UMAP of ANT (i) or ACC (n) neurons colored by feature barcode of biological replicates. j, o, UMAP of ANT (j) or ACC (o) neurons colored by neurotransmitter class. k, Proportion of ANT neurons assigned to each neurotransmitter class. l, Breakdown of neuron classes across time points and conditions by proportion. p, UMAP of ACC neurons colored by cortical layer assignment. q, Proportion of ACC neurons assigned to each cortical layer. r, Breakdown of cortical layer assignment across time points and conditions by proportion. ACC, anterior cingulate cortex. HR, high repetition. LR, low repetition. a, Line plots of Wasserstein global transcriptional distance, using the whole transcriptome (solid colors) or a random set of genes (faded). Distances are to early-training in ANT (left) and to recent-retrieval in ACC (right), ***P < 0.0001 for global distances at recent, mid and remote retrieval HR vs LR in ANT, ***P < 0.0001 for global distances at mid, remote and late-remote retrieval HR vs LR in ACC, One-way ANOVA with Bonferroni correction. b, c, Gene ontology (GO) of ANT HR vs LR DEGs (b) or ACC HR vs LR DEGs (c) obtained using pseudo-bulk differential gene expression analysis. GO enrichment was performed using the one-tailed hypergeometric test (over-representation analysis), with multiple comparisons correction. Color gradient represents the nominal −log10(p-value), and circle size indicates percentage of genes within a GO term overlapping with total DEGs. d, GO analysis of genes upregulated in HR neurons at remote retrieval in ANT (left) and late-remote retrieval in ACC (right) using the same statistical parameters as (b), (c). e, Overlap of genes belonging to the histone methylation module across retrieval time points in ACC. f, Overlap between HR vs LR DEGs obtained using all ANT neurons, or Vglut2 neurons, at recent and remote retrieval. g, Overlap between HR vs LR DEGs obtained using all ACC neurons, or Vglut1 neurons, at recent and remote retrieval. h, GO analysis of DEGs from ACC neurons classified as cortical layer 2/3 or layer 6, using the same statistical parameters as (b), (c). ACC, anterior cingulate cortex. DEGs, differentially expressed genes. LR, low repetition. a, b, Kernel density estimate tSNE plots of ANT (a) or ACC (b) neurons for each biological replicate (n = 2-3 mice per time point) across all time points. c, d, tSNE visualization of ANT (c) or ACC (d) pseudo-time trajectory colored by neuronal class in ANT and anatomical layer in ACC. e, Bar plot of percent Fos+ neurons across time points in ANT (left) or ACC (right) neurons used to construct pseudotime trajectories. f, g, tSNE visualization of ANT (f) or ACC (g) trajectories, colored by Fos+ neurons with scored density >= 0.6. h, Overlap between DEGs obtained from either all neurons or only Fos + cells. Shown for mid-retrieval in ANT (left) and mid and remote retrieval in ACC (right). i, Density plots for HR and LR ACC neurons on the tSNE pseudotime space across retrieval days. ACC, anterior cingulate cortex. DEGs, differentially expressed genes. LR, low repetition. a, tSNE visualizations of ANT (left) and ACC (right) pseudotime trajectories, colored by neighborhood abundance of cells from HR or LR conditions. b, Pseudotime trajectories colored by average expression of DEGs from early-training and remote retrieval in ANT (left); recent-retrieval and late-remote retrieval in ACC (right), units are log2CPM + 1. c, ANT pseudotime trajectory colored by average expression of learning Immediate Early Genes (IEGs, units log2CPM + 1). d, Pseudotime trajectories of ANT (left) or ACC (right) obtained using the CellRank algorithm. Stars highlight apex points. e, Expression of genes associated with the synaptic plasticity GO module in ANT (left) at mid-retrieval, and the histone methylation GO module in ACC (right) at late-remote retrieval, each data point depicts an individual mouse, average expression represented in units log2CPM + 1, data are mean expression ± SEM (error bars). f, Expression of genes correlated with the late macrostate in ANT (left) or late-remote macrostate in ACC (right) across HR and LR, each data point depicts an individual mouse, data are mean ± SEM (error bars). g, Pearson correlation between behavioral performance during retrieval (measured as lick rate Hz) and average expression of genes shown in (e) and (f), each dot is a mouse. Mice tested on the HR condition in magenta and mice tested on the LR condition in green, r = 0.27. h, Volcano plots of DEGs between HR and LR at remote retrieval in ANT (left) or in ACC (right). Labeled genes indicate TRs, with TR modules highlighted by color. DEGs determined through a two-part generalized linear hurdle model, p-values adjusted for multiple comparisons using the Benjamini-Hochberg method. i, Left: pseudotime trajectory of V1 visual cortex neurons colored by time point. j, Kernel density estimate tSNE plots of V1 visual cortex neurons across early-training, remote retrieval HR and remote retrieval LR. ACC, anterior cingulate cortex. DEGs, differentially expressed genes. LR, low repetition. a, Relative expression of pseudotime signature genes (assessed by qPCR) in ANT at early (top) and remote retrieval (bottom) between HR and LR mice, normalized to Map2. Data points represent biological replicates (n = 3 or n = 6), each corresponding to one animal and quantified as the mean of three technical replicates (Ct values), *P = 0.0211 for Camta1; *P = 0.0495 for Gria2; *P = 0.0286 for Clu, *P = 0.0384 for Cadm1, unpaired one-tailed Student's t-test, mean ± SEM shown. b, Average expression of all transcriptional regulators in ANT neurons (n = 5535 neurons, top) or ACC neurons (n = 5671 neurons, bottom) across conditions and time points (units log2CPM + 1), median and interquartile range depicted. c, Average expression of all transcriptional regulators in thalamus neurons (n = 131212 neurons, top) or ACC neuron (n = 22385 neurons, bottom) from the Allen Mouse Brain Atlas dataset (units log2 CPM + 1), median and interquartile range depicted. Reproduced from The Neuroscience Multi-Omic Archive under a Creative Commons licence CC BY 4.0. d, Line plots of average expression of ANT (left) and ACC (right) transcriptional regulators in HR neurons only across behavioral time points (mean ± SEM shown, units log2CPM + 1). ACC, anterior cingulate cortex. LR, low repetition. a, Boxplot of normalized read counts for ANT (left) and ACC (right) ATAC-seq samples, n = 4 mice per time point in ANT samples; n = 3 mice in recent retrieval, n = 4 mice in homecage and remote retrieval, n = 5 mice in mid retrieval in ACC samples. Median and interquartile range depicted (lower bound = 25th percentile, upper bound = 75th percentile, lower whisker = smallest data point ≥ (Q1 − 1.5 × IQR), upper whisker = largest data point ≤ (Q3 + 1.5 × IQR)). b, Principal component analysis (PCA) of samples from ANT (left) or ACC (right). Each point is an individual mouse, colored by the behavioral time point. c, Bar plot of significant peaks in HR condition versus home-cage (HC) comparisons across time points in ANT (top) or ACC (bottom) neurons. d, Sorted rank plot of transcription factor motifs for clustered peak modules at recent retrieval (left) or at mid retrieval (right) in ANT neurons, p-values for motifs are unadjusted and are derived from a one-sided binomial test against a nucleotide composition-matched background. e, Z-scored accessibility of ATAC peaks of ANT (left) or ACC (right) TRs, across time points. Columns are biological replicates (individual animals), n = 3-5 mice per time point, and rows are ATAC peaks. ACC, anterior cingulate cortex. HC, home cage. LR, low repetition. a, Schematic of in vitro CRISPR sgRNA screen. b, Neuro2A cells transfected with Px458-sgRNA-Creb1-spCas9-GFP, scale: 200 µm. Transfection efficacy was independently confirmed for each sgRNA by immunofluorescence, representative image shown. c, Percentage of WT vs mutant Sanger sequencing reads from genomic DNA isolated from Neuro2a cells transfected with Px458-sgRNA-Cas9-GFP after FACS sorting for GFP+ cells. For FACS gating strategy, see Supplementary Fig. n = 1 biological replicate per gene, showing the most efficient guide from the screen. d, Pie chart illustrating the breakdown of specific indel mutations identified by NGS of the PCR amplicon spanning the cut site from Neuro2a cells in (c) for select guides. e, Top left: coronal slice from a control Rosa26-Cas9 knock-in mouse stained for Creb1 and DAPI in HPC. Bottom left: coronal slice from a control Rosa26-Cas9 knock-in mouse stained for Camta1 and DAPI in ANT. Scale bar = 1 mm in 4x, scale bar = 0.5 mm in 10x. Top right: coronal slice from a Rosa26-Cas9 knock-in mouse injected with sgRNA-Creb1-GFP stained for Creb1 and DAPI in HPC, with inserts showing 10x and 40x field of view. Bottom right: coronal slice from a Rosa26-Cas9 knock-in mouse injected with sgRNA-Camta1-GFP stained for Camta1 and DAPI in ANT, with inserts showing 10x and 40x field of view. Scale bar = 0.5 mm in 10x, scale bar = 0.1 mm in 40x. Representative images from one animal used in our behavioral experiments. f, Immunofluorescent staining of NeuN or Cleaved Caspase 3 in Rosa26-Cas9 knock-in mouse injected with sgRNA-Creb1-GFP to evaluate extent of neuronal toxicity from AAV injection. Representative image shown from one mouse. Scale bar 0.5 mm in 10x. g, Western blot validation of Tcf4-KO from ANT tissue compared to control tissue, blotting performed on one animal (for uncropped blots, see Supplementary Fig. h, Discrimination indices of learning and recall performances in HR of Rosa26-Cas9 knock-in mice expressing sgRNA targeting Myt1l, Mef2c or Kmt2a. n = 6 sgRNA-Myt1l or sgRNA-Mef2c, n = 8 sgRNA-Kmt2a mice, individual data points shown (faded lines), with mean ± SEM (solid line). ACC, anterior cingulate cortex. The schematic in a was created using BioRender (https://biorender.com). a, Example traces of z-scored fluorescence intensity or F(z) from one control mouse from late training from ACC and ANT (AM) aligned to lick rate and task zones, three trials shown. b, Schematic of ChIP-sequencing workflow: selection and validation of antibodies via generation of knockout N2A cell lines, followed by ChIP-sequencing of samples collected at recent, mid and remote retrieval for WT and Ash1l KO (H3K4me3 antibody) or at mid-retrieval for WT and Camta1/Tcf4 KO. Prior to ChIP-seq, dissected tissue was dissociated and FACS sorted for NeuN+ (WT controls) or GFP+ (KO) cells (For FACS gating strategy, refer to Supplementary Fig. 2e).c, Heatmaps of normalized ChIP-sequencing read densities centered at the midpoints (±2 kb) for WT controls, Camta1 KO and Tcf4 KO at mid retrieval. d, Top: Camta1 ChIP-seq signal at Snap25, Arpp21 gene loci for WT vs. Camta1-KO mice at mid retrieval. Bottom: Tcf4 ChIP-seq signal at Rbfox1, C1ql3 gene loci for WT vs. Tcf4-KO mice at mid retrieval. Tracks show normalized read density (RPKM) and are colored as darker shade for WT, lighter shade for KO. e, Gene ontology analysis of genes derived from ChIP-seq differential peak analysis of WT vs. Camta1-KO (top) or WT vs. Tcf4-KO (bottom) at mid retrieval, one-tailed hypergeometric test (over-representation analysis), with multiple comparisons correction. f, Left: bar plot of normalized counts in peak regions associated with Camta1 target genes in control versus Camta1 KO; n = 1 replicate sequenced for WT controls after pooling together NeuN+ sorted cells (neurons) from n = 4 animals; n = 1 sample sequenced for Camta1 KO after pooling together GFP+ sorted cells (CRISPR/Cas9 edited) from n = 9 animals. Right: bar plot of normalized counts in peak regions associated with Tcf4 target genes in control versus Tcf4 KO, n = 1 sample sequenced, same number of animals used as for Camta1. g, qPCR validation of Camta1 target gene expression in Camta1-KO mice. Bar graph showing the relative mRNA expression of plasticity-related genes (Arp21, Rasgrp1, and Snap25), previously validated by Camta1 ChIP, in Camta1-KO) animals compared to WT controls. Expression levels are normalized to Map2. All animals used were run on VR behavior and dissected at the endpoint (Methods). Data are presented as mean ± SEM (n = 4 independent biological replicates/animals per cohort). *P = 0.00159 for Arp21; **P = 0.0018 for Rasgrp1; *P = 0.0231 for Snap25, unpaired one-tailed Student's t-test with Bonferroni-Dunn correction. AM/ANT, anteromedial thalamus, Chr, chromosome. F(z), z-scored fluorescence intensity, ITI, intertrial interval. The schematic in a was created using BioRender (https://biorender.com). a, Heatmaps of normalized H3K4me3 ChIP-sequencing read densities centered at the midpoints (±2 kb) for WT controls and Ash1l KO at mid retrieval. b, Heatmap of H3K4me3 marks in ACC Ash1l KO across mid and remote retrieval. c, Gene ontology analysis of genes associated with H3K4me3 peaks in WT controls at mid or remote retrieval, one-tailed hypergeometric test (over-representation analysis), with multiple comparisons correction. d, ChIP-sequencing signal at gene loci for H3K4me3 in WT or Ash1l-KO mice at remote retrieval. Tracks show normalized read density (RPKM) and are colored by condition (darker shade for WT, lighter shade for Ash1l-KO). e, Gene ontology analysis of genes associated with H3K4me3 peaks from Ash1l-KO mice at mid and remote retrieval, one-tailed hypergeometric test (over-representation analysis), with multiple comparisons correction. f, Bar plot of normalized counts in peak regions associated with Ash1l target genes in control versus Ash1l-KO samples, n = 2 independent biological replicates (each replicate is pooled dissected and GFP+ or NeuN+ FACS sorted tissue from 2 mice), mean of replicates depicted as bar. Every animal used underwent VR behavior and dissected at the endpoint (Methods). g, Left: Z-scored accessibility of ATAC peaks of Camta1 and Tcf4 modules derived from ChIP-sequencing in ANT neurons. Right: Z-scored accessibility of ATAC peaks of H3K4me3 module derived from ChIP-sequencing in ACC neurons. Columns are biological replicates, n = 3-5 mice per time point, and rows are ATAC peaks. HC, home cage. 1 contains the unedited original western blots used for the figs and Supplementary Fig. 2 contains the gating strategies use for cell sorting. Marker genes used for cell typing of ANT and ACC cells. Marker genes used to identify anterior/posterior thalamic nuclei, layers from ACC, and neuron classes. Number of cells across clusters and time point/condition in all ANT and ACC cells. Number of neurons across clusters and time point/condition; and number of neuron subtype across time points/condition in ANT and ACC. DEGs across time points in ANT and ACC neurons. Correlated genes with the early or late consolidation branch trajectories for ANT and ACC neurons. Information related to ChIP-sequencing experiments. sgRNAs and corresponding primers, and qPCR primers. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Terceros, A., Chen, C., Harada, Y. et al. Thalamocortical transcriptional gates coordinate memory stabilization. Version of record: 26 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a69525264/highly-energetic-water/'>Scientists Discovered ‘Highly Energetic' Water With Massive Potential</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Whether in the form of flowing rivers, undulating waves, or raindrops streaking down a car window, water almost always seems to be in motion. When confined to the especially tight spaces between molecules, it can start behaving strangely, and how exactly it interacts with those molecules its trapped between was unknown. In an attempt to end the debate over whether trapped water coexists with its surrounding molecules or is involved in their interactions, he and his research team studied the behavior of trapped water through computer simulations—and found something surprising. Water trapped between molecules is highly energetic despite not being able to move, but that changes as soon as another molecule enters the scene and the water is displaced. Think of this phenomenon like a crowded subway car at rush hour, when every seat and space to stand is taken. Passengers are going to be eager to get out as soon as humanly possible, and an onrush of people will pour out onto the platform when the doors finally open, allowing for different people to claim every available seat or place to stand before the train takes off again. Once they escape, they leave an empty car for the second group to squeeze themselves into, and seeing how much free space exists on the platform may have them frantically pushing past everyone else. While no physical transfer of energy happens on a subway, it could be argued that people streaming out the doors could be incentive for those waiting to board to rush in and grab a seat as soon as possible before only standing room is left. And the whole thing gets kicked up to 11 at rush hour. Likewise, highly energetic water can release more or less energy depending on how much energy it has to begin with and what kind of molecule is incoming, acting as a binding force for the new molecule. Water that has more energy to release and interacts with the right molecule acts like those rush hour crowds. Biedermann's discovery could have applications far beyond figuring out the physics and chemistry of trapped water behavior. Strengthening bonds between molecules this way could ultimately create stronger medications and materials. If water is found to be trapped between proteins in a medication, scientists can develop a formula that allows for specific active molecules to displace that water and take advantage of the binding force its energy offers. Enhanced bonds would then make the treatment more effective. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. These Letters Don't Belong to Any Known Language</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03885-w'>‘They don't have symptoms': CAR-T therapies send autoimmune diseases into remission</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 11:50:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Rachel Fieldhouse is a reporter for Nature in Sydney, Australia. CAR T cells (pictured) are being used to treat autoimmune diseases.Credit: Eye Of Science/SPL Engineered immune cells are being used to successfully treat people with a range of debilitating autoimmune conditions, such as ulcerative colitis, rheumatoid arthritis and lupus. CAR-T-cell therapy exploits the immune system's T cells that fight off infection: these cells are collected from a person and tweaked to produce proteins called chimeric antigen receptors. They are then reintroduced into the body to target antigens expressed by B cells, another type of immune cell. In autoimmune disorders, these B cells make antibodies that attack the body's own healthy tissues. Clinician–researcher David Simon, who was involved in that treatment2, says the CAR-T therapies have since entered phase I and II clinical trials for autoimmune conditions including systemic sclerosis, myositis and rheumatoid arthritis. Phase III trials are also under way for lupus and myasthenia gravis, a condition that causes weakness in the muscles that are used to breathe, swallow and see. Simon, who focuses on rheumatoid arthritis at the Charité University Hospital in Berlin, says people in CAR-T-cell therapy trials for rheumatoid arthritis and lupus seem to be ‘cured'. “This is something totally new which we didn't observe before.” CAR-T-cell therapy has become a routine treatment for several types of cancer since it was first approved to treat blood cancers in 2017. Ulcerative colitis, a condition that affects the colon and causes stomach pain, ulcers and bloody diarrhoea, is one of the latest diseases in which CAR-T-cell therapy has shown promise. In September, gastroenterologist Markus Neurath and his colleagues at the University Hospital Erlangen in Germany, reported using CAR T cells to treat a 21-year-old woman with ulcerative colitis3. After the treatment, she showed signs of remission that persisted for 14 weeks, no longer needed medication and could return to work. His team plans to test the treatment in a few more people before proceeding to a controlled clinical trial involving many individuals. Earlier this year, Bing Du, an immunologist at East China Normal University in Shanghai, and his team published the results of a CAR-T pilot study using immune cells from a donor to treat drug-resistant lupus4. Last year, Du was also involved in a world-first study of bioengineered and modified immune cells from a donor for the treatment of two rare and severe autoimmune diseases6. The other three women received low-dose steroids as a maintenance therapy. This kids' brain cancer is incurable — but immune therapy holds promise Woman in cancer remission for record 19 years after CAR-T immune treatment Cutting-edge CAR-T cancer therapy is now made in India — at one-tenth the cost World-first therapy using donor cells sends autoimmune diseases into remission Waste not: how researchers harness pee and poo for science Mind-reading devices can now predict preconscious thoughts: is it time to worry? NSD2 targeting reverses plasticity and drug resistance in prostate cancer We are all mosaics: vast genetic diversity found between cells in a single person What we are looking for The BioMed X Institute in New Haven, Connecticut, USA is establishing a new, fully funded research group in the following f... Join HZAU's global faculty team to advance research with competitive benefits. This kids' brain cancer is incurable — but immune therapy holds promise Woman in cancer remission for record 19 years after CAR-T immune treatment Cutting-edge CAR-T cancer therapy is now made in India — at one-tenth the cost World-first therapy using donor cells sends autoimmune diseases into remission An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02182-6'>Industrial and agricultural chemicals exhibit antimicrobial activity against human gut bacteria in vitro</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 10:45:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Industrial and agricultural chemicals such as pesticides are often considered to have restricted biological activity. Yet, there are concerns regarding their broader toxicity range and impact on human gut microbiota. Here we report a systematic in vitro screening to assess the impact of 1,076 pollutants, spanning diverse chemistries and indicated applications, on 22 prevalent gut bacteria. Our investigation uncovered 588 inhibitory interactions involving 168 chemicals, the majority of which were not previously reported to have antibacterial properties. Fungicides and industrial chemicals showed the largest impact, with around 30% exhibiting anti-gut-bacterial properties. We demonstrate that the scale of our dataset enables a machine learning approach for predicting the antibacterial activity of pesticides. Mechanistically, chemical–genetic screens using transposon mutant libraries of Parabacteroides merdae and Bacteroides thetaiotaomicron implicated genes involved in conserved efflux pathways, including the acrR locus, as mediators of pollutant resistance. We also found that loss-of-function mutations in genes coding for metabolic enzymes were selected under pollutant exposure, including those for branched short-chain fatty acid biosynthesis under tetrabromobisphenol A, a flame retardant. Taken together, our results suggest that the antibacterial activity of chemical pollutants should be considered in future studies on the microbiome and the emergence of antimicrobial resistance, as well as in toxicological assessments. Synthetic chemicals have become indispensable for agriculture and industry. Their pervasive use and environmental persistence have led to pollution levels exceeding the planetary boundary for stable and resilient Earth systems1,2,3. Pesticide pollutants are of particular concern, with the Food and Drug Administration and European Food Safety Authority documenting hundreds of different pesticide residues in food4,5, as well as broader water contamination6,7. Industrial chemicals, such as per- and polyfluoroalkyl substances (PFASs), plasticizers, surface coatings and dyes, also routinely enter food and water via environmental pollution, agricultural application or industrial processing8. The contamination of food and water by chemical pollutants potentially exposes the human gastrointestinal tract, through passage and enterohepatic circulation, to thousands of xenobiotic compounds9. Over 95% of Americans and nearly the entire Dutch population are estimated to have detectable levels of PFASs in their blood10,11. Similarly, a German mother cohort study found hundreds of chemicals derived from consumer goods, cosmetics and food in plasma, reaching up to micromolar concentrations13. These studies indicate that the gut microbiota—a community of hundreds of microbial species—is probably exposed to numerous chemicals. Yet, little is known about the impact of agricultural and industrial chemical contaminants on gut bacteria. Chemicals such as pesticides are usually marketed with a narrow definition of target organisms (for example, as insecticides, herbicides or fungicides). Yet, many chemicals have been reported to show off-purpose activity on non-target organisms, including soil bacteria14 and pathogens15. A small number of pesticides and industrial chemicals have also been shown to impact human and animal gut microbiota12,16,17,18,19,20,21,22,23,24. More broadly, human-targeted drugs have also been shown to impact gut bacterial growth and metabolism25,26. Collectively, these studies indicate that many pesticides and other chemical pollutants might exhibit inhibitory activity against gut bacteria. However, safety assessments for these chemicals currently do not consider the human gut microbiome. In vivo assessment of the impact of chemicals on gut bacteria is currently not possible due to the lack of systematic and quantitative data on chemical exposure. Furthermore, disentangling the effects of chemicals from other factors such as diet and medications is not possible without mechanistic knowledge of direct chemical–bacteria interactions27,28. We therefore conducted a large-scale in vitro screen assessing the anti-gut-bacterial effect of 1,076 chemical pollutants, including pesticides, pesticide metabolites and industrial chemicals. The screen uncovered numerous growth inhibitory interactions, particularly with fungicides and industrial chemicals. We further used chemical–genetic screens to identify mechanisms of bacterial susceptibility to chemicals that overlap with antibiotic resistance. Our study expands the knowledgebase on xenobiotic toxicity in bacteria and demonstrates the feasibility of computational prediction using machine learning on chemical structures. To systematically assess the impact of chemical contaminants, we used a large library of 1,076 compounds that are likely to enter food and water (Extended Data Fig. The library includes 829 pesticides (mainly herbicides, insecticides and fungicides), along with compounds targeting other organisms, including spiders, nematodes, bacteria and rodents. It also contains 119 known pesticide metabolites formed by biotransformation29 and 75 pesticide-related compounds (precursors, breakdown products, chemical isomers or formulation products). In addition, the library contains 48 industrial chemicals, such as the widespread contaminants bisphenols and nitrosamines, and five mycotoxins found in mould-contaminated foods. Several compounds are considered to be persistent organic pollutants, such as PFASs and the insecticides dichlorodiphenyltrichloroethane (DDT, also known as clofenotane) and chlordecone. Overall, the library spans 291 out of 352 pesticide classes in the British Crop Production Council Pesticide Compendium and 87% of compounds reported in a study of pesticide discharge from rivers into oceans6. The library, together with extensive metadata (Supplementary Data 1), constitutes a resource for systematic investigation of the impact of chemical pollutants on biological systems. We performed the chemical–bacteria screen at 20 μM concentration for all compounds. This dose was chosen for consistency with previous xenobiotic screens25,30,31, thus enabling direct comparison. To contextualize this dose, we analysed xenobiotic plasma concentrations from a recent study13. In 305 of 592 samples, at least one xenobiotic was observed at a concentration of 10 μM or higher. Among the measured xenobiotics, spinosyn A, melamine, mepiquat, acrylamide and amitrole, all of which are present in our compound library, most frequently surpassed 2 μM in plasma (45, 40, 39, 34 and 33 samples, respectively). Pharmacokinetic studies suggest that colonic concentrations tend to be in the same range or higher than in the plasma25,32. The relevance of micromolar-range doses is further supported by the food regulatory limits of some chemicals33. Furthermore, chemicals such as the antiparasitic closantel have been found in dairy products at micromolar levels34,35. Next, we assessed the impact of all 1,076 compounds in the library on the growth of 22 commensal gut bacterial strains belonging to 21 species (Fig. These strains were selected based on their abundance and prevalence in the healthy human gut microbiota, as well as phylogenetic and metabolic representation (Supplementary Data 2)36,37. All strains were grown anaerobically at 37 °C in mGAM broth, a rich medium designed to support the growth of a maximum number of strains. Bacterial growth was monitored for 24 h and quantified as the area under the growth curve (AUC). We defined growth inhibition hits as chemical–bacteria interactions featuring a growth decrease of >20% and a false-discovery-rate adjusted P value (Padj) < 0.05. Only hits identified in at least two out of three biological replicates are considered (Supplementary Data 3 and Extended Data Fig. a, Workflow of high-throughput screening to test the susceptibility of gut bacteria against industrial and agricultural chemicals. Monocultures were grown in mGAM medium with 20 μM of each compound (n = 3 experiments on independent days). Growth curves were recorded and the AUC relative to DMSO controls was used as a fitness proxy. b, High-throughput growth screening results for 22 gut bacterial strains. Significant pollutant chemical–strain interactions, where the growth of a specific strain was inhibited versus the DMSO control, are highlighted with a coloured bar (Padj < 0.05; >20% reduction in AUC; two out of three independent replicates significant). The data are grouped and coloured by compound class. Note that in rare cases a single compound can belong to more than one compound class. The phylum, abundance and prevalence of strains are displayed in coloured and greyscale bars, and the Gram stain and number of compounds that inhibit growth are shown in parentheses after each strain name. c–e, Distributions of the number of inhibitory bacterial interactions (hits) per compound (c) and bacterial strain (d), and the strength of growth inhibition, shown as mean normalized AUC values of hits(e). The colours indicate mean rAUC values normalized to DMSO controls. Around one-sixth of the tested chemicals (168) were inhibitory against at least one bacterial strain (Fig. The most sensitive taxa were Bacteroidales, especially Parabacteroides distasonis, whereas Escherichia coli, Fusobacterium nucleatum subspecies animalis and Akkermansia muciniphila were the least sensitive species (Fig. The chemical categories featuring prevalent anti-gut-bacterial activity (at least one in five compounds) were fungicides, industrial chemicals and acaricides (Fig. Although most compounds were inhibitory against a few strains, indicating a narrow spectrum of activity, 24 chemicals showed broad toxicity, inhibiting more than one-third of the species tested (Extended Data Fig. Around 150 chemical–bacteria interactions featured >90% growth reduction, uncovering strong anti-gut-bacterial activity for 33 compounds (Fig. To test whether the inhibitory activity was retained at lower concentrations, we tested 11 pesticides against eight bacteria across multiple concentrations (Fig. Independently purchased compounds were used to account for any errors associated with batch variations and large-scale library preparations. Most interactions from the main screen were also observed in this independent experiment (72%; 23 out of 32; Extended Data Fig. Many compounds show strong inhibitory effects at substantially lower concentrations than those tested in the main screen. For example, imazalil sulfate, closantel and prochloraz inhibit Eubacterium rectale, a prevalent species that produces the health-associated metabolite butyrate39, at a concentration as low as 2.5 µM. Together, our systematic assessment revealed hundreds of chemical–bacteria interactions, including contaminants with broad and potent anti-gut-bacterial activity. The off-target activity of chemicals against gut bacteria has been reported for >1,000 human-targeted drugs and antibiotics by Maier et al.25,30. As in the case of therapeutic drugs, we observed a positive correlation between the number of compounds affecting a species and its relative abundance in human microbiomes (Spearman's rank correlation (ρ) = 0.57; P = 0.005) (Fig. Thus, chemicals with broad- as well as narrow-spectrum activity are likely to influence microbiome composition due to their impact on abundant taxa. Strains sensitive to many chemical pollutants were also sensitive to many therapeutic drugs tested by Maier et al. (ρ = 0.63; P = 0.009) (Fig. Pollutant sensitivity correlated more strongly with human-targeted drugs (ρ = 0.66; P = 0.004) than with antibiotics (⍴ = 0.33; P = 0.19) (Fig. 2d), which is consistent with the broader-spectrum activity of antibiotics. The correlation between chemical pollutants in our screen and human-targeted drugs supports that gut bacteria interact with diverse chemicals via common and non-specific mechanisms25,28,40. a,b, Correlation between average species abundance (a) or prevalence (b), observed in human cohort studies, and compound sensitivity across gut bacterial strains (n = 22 strains). ρ is the Spearman's rank correlation coefficient (two sided). c, Correlation between the number of pollutant compounds inhibiting a strain (this study) and the number of human-targeted drugs inhibiting the same strain in the study by Maier et al.25 (n = 17 strains). d, As in c, but stratified by drug type. e, Multi-dimensional scaling (MDS) of the strain sensitivity profiles of strains (phylum colour coded as in a). MDS was performed with scikit-learn using default settings. f, Relative abundances of synthetic community Com20 species after exposure to BPAF and TBBPA, coloured by phylum (n = 3 biological replicates). The legend identifies individual species using colour intensity. g, Comparison of monoculture versus community susceptibility to BPAF. The data represent ratios of absolute abundance (16S sequencing read counts normalized to the community optical density) between cultures treated with BPAF and the DMSO control. The points are the average of three Com20 replicates, corresponding to species overlapping between monoculture and synthetic community experiments. h, LC–MS/MS measurements of BPAF in synthetic communities (Com20) show bioaccumulation (that is, depletion of BPAF in supernatant compared with whole culture) (n = 6 biological replicates; two-sided t-test). The data are presented as means ± s.d. Chemical susceptibility profiles of bacterial species grouped clearly by evolutionary relatedness (that is, by phylum; Fig. 2e), with most compounds showing phylum-level differences in toxicity (Extended Data Fig. For example, some Firmicutes were uniquely sensitive to conazole fungicides, including the widely used imazalil and prochloraz. This indicates a shared genetic basis for toxicity and suggests that chemical susceptibility data could help to predict effects on related bacterial species for in silico toxicological assessments. We next investigated how species-level chemical effects translate in bacterial communities using a synthetic community of 20 gut bacteria (Com20)38 (Methods and Supplementary Data 5). Com20 is a phylogenetically and functionally diverse community, spanning six phyla, 11 families and 17 genera, collectively encoding >60% of the metabolic pathways present in healthy human gut microbiomes. Members of the community grow together stably and reproducibly in mGAM38—the growth medium used in our screen. The Com20 was transferred twice in fresh growth medium before being challenged with BPAF or TBBPA, two broad-spectrum anti-gut-bacterial chemicals. In the case of TBBPA, the community was dominated by Bacteroides thetaiotaomicron, despite its susceptibility in monoculture. BPAF induced compositional changes concordant with monoculture effects (Fig. 2f), including decreased relative abundances of B. thetaiotaomicron, Bacteroides uniformis, Parabacteroides merdae and Roseburia intestinalis, and increased abundances of insensitive species (that is, Collinsella aerofaciens, Coprococcus comes and Streptococcus salivarius). Yet, F. nucleatum and E. rectale were protected against BPAF in the community (Fig. Community-level effects, such as cross-protection, have been described in gut bacterial communities exposed to therapeutic drugs and could partly be attributed to compound bioaccumulation31,41. We hypothesized that cross-protection against BPAF in Com20 is due to bioaccumulation, as this has been observed in Bacteroidales monocultures42. We measured BPAF concentrations in whole culture, supernatant and pellets of Com20 cultures exposed to 20 µM BPAF using liquid chromatography–tandem mass spectrometry (LC–MS/MS). 2h), indicating that most of BPAF in Com20 is sequestered within cells, explaining cross-protection of some community members. Large-scale surveys of xenobiotic–bacteria interactions, combined with computational analyses, could enable in silico toxicity predictions43. However, accurate prediction requires sufficient data spanning the relevant chemical space. To assess this, we compared the extended-connectivity fingerprints (ECFPs) of our pollutant library with those of previously characterized pharmaceutical drugs25 and 250,000 random PubChem compounds. Pollutants and pharmaceutical drugs were widely dispersed in uniform manifold approximation and projection (UMAP) space (Fig. 3a), indicating large chemical diversity for both groups. Notably, pollutants and drugs did not form clear clusters, and only six pollutants had a Tanimoto similarity to drugs of >0.75, highlighting limited molecular fingerprint overlap between the two libraries44,45 (Fig. This suggested that labels such as ‘pesticide' or ‘pharmaceutical drugs' do not capture the molecular basis of anti-gut-bacterial toxicity. To quantify how well pollutant and drug libraries jointly span the chemical space, we noted the Tanimoto similarity of each PubChem compound to its closest match in the libraries. This was significantly increased in the combined library (P < 2.2 × 10−16; Kolmogorov–Smirnov test; Extended Data Fig. 3a), indicating broader chemical-space coverage towards predicting the impact of novel compounds. a, UMAP for ECFPs of pollutant library compounds from this study, previously screened pharmaceutical drugs25 and a random selection of 250,000 compounds from PubChem. b, Distribution of similarities between the closest-matching pollutants from this study and drugs used by Maier et al.25. The two compound sets are structurally diverse and distinct, as only very few (six) compounds from the library used in this study had a similarity to a drug from ref. Compounds shared between the two studies (n = 32; mostly veterinary drugs) were excluded from this analysis. c, Machine learning workflow to predict species-specific xenobiotic toxicity from molecular descriptors and embedded features. Random forest classifier models were trained and evaluated using 20× cross-validation, using performance metrics appropriate for imbalanced datasets. Left, models trained on pesticides only (n = 801) using different feature sets (x axes). Middle, model trained on pesticides and pharmaceutical drugs from Maier et al.25 (n = 1,940). Right, models trained on pesticides (this study) cannot predict the toxicity of pharmaceutical drugs25 and vice versa. Each dot represents a species (mean of 20× cross-validation splits). Median random model performances are shown as overlayed bars. To probe the utility of increased chemical-space coverage, we employed machine learning for predicting the anti-gut-bacterial activity of chemicals. We used two datasets: pesticides from our pollutant screen and the drug screen by Maier et al. (Fig. For each compound, we obtained a comprehensive set of structural features, including molecular fingerprints (Chemistry Development Kit) and deep-learning-based feature embeddings from MoLFormer46. The latter is a deep learning model trained on 1.1 billion molecules to predict a wide range of chemical properties, such as substructures, from simplified molecular input line entry system (SMILES) strings. To avoid excessively imbalanced groups for machine learning modelling, we chose seven species that showed >20 hits in both pollutant and drug datasets. Random forest classifier models were trained and evaluated using cross-validation with stratified splits. Classifications were obtained by thresholding probability outputs at the hit frequency of the training dataset—a strategy suitable for imbalanced datasets47. We evaluated models using two metrics appropriate for imbalanced datasets: balanced accuracy (the mean of sensitivity and specificity) and average precision (the area under the precision-recall curve)48. Models trained and tested on pesticides alone achieved good performance when predicting pesticide toxicity (average balanced accuracy = 0.77; average precision = 0.65; 20× cross-validation; Fig. Performance varied slightly between species, with the best results obtained for E. rectale (balanced accuracy = 0.8; average precision = 0.73) and the weakest predictions for B. thetaiotaomicron (balanced accuracy = 0.73; average precision = 0.52). Training models on only molecular descriptor features (that is, excluding MoLFormer embeddings) slightly improved the performance (balanced accuracy = 0.81; average precision = 0.72), indicating that this smaller feature set is sufficient for predictions. Analysis of feature importance (Extended Data Fig. 3) identified molecular features implicated in antibacterial activity. Although a model trained and tested on the combined pesticide–drug dataset showed similar performance, cross-predictions from one dataset to another showed poorer performance (approximate balanced accuracy = 0.7; average precision = 0.2). This indicates that pesticides and drugs are overall different in both chemical and activity space (Fig. The pesticide toxicity against gut bacteria thus could not have been predicted based on previously available data. Given the high sensitivity of Bacteroidales to pollutant compounds (Fig. We used a pooled transposon mutant library of P. merdae49 to identify genes that modulate the impact of xenobiotics on bacterial fitness. The library encompasses barcoded transposon insertion mutants (Tn mutants) of >3,000 non-essential genes, enabling a genome-wide functional analysis. We conducted the competition assay against a panel of ten chemicals. TBBPA, BPAF, imazalil sulfate, fluazinam, closantel and emamectin benzoate were tested at concentrations of ≤20 μM, whereas glyphosate, propiconazole, perfluorooctanoic acid and perfluorononanoic acid (PFNA) were tested at ≥20 μM to ensure sufficient selective pressure (Methods). Cultures inoculated with the pooled library were grown to the early stationary phase, and Tn mutant selection was quantified using barcoded transposon sequencing (TnBarSeq) (Supplementary Data 6). Gene hits were defined by an absolute log2[fold change] in mutant abundance of >0.25 and Padj < 0.05, relative to end-point vehicle dimethyl sulfoxide (DMSO) or media controls. The abundance changes were not attributed to global differences in cell doublings (Extended Data Fig. Among the compounds tested at ≤20 μM, TBBPA, BPAF and closantel exhibited the strongest effects in library selection, whereas 500 μM PFNA showed the most hits overall. In contrast, 20 μM PFNA, 20 μM perfluorooctanoic acid and 50 μM glyphosate did not show significant hits, consistent with the lack of effect on growth at those concentrations (Extended Data Fig. Closantel, an anthelmintic, caused the strongest selection, with >90% of Tn mutants carrying insertions across over 20 different positions of the coding and promoter regions of the gene NQ542_01170 (Fig. This gene encodes a transcriptional regulator homologous to the efflux repressor acrR from B. uniformis and is co-located with tolC and RND-efflux genes, with conserved synteny51. Notably, acrR Tn mutants were also the top hit under TBBPA, a flame retardant, with >100-fold enrichment compared with the DMSO control, but were not enriched under any other pollutants (Fig. a, Overview of the closantel experiment and a representative plot of TnBarSeq insertion at the P. merdae acrR efflux regulator locus (NQ542_01170). b, Relative abundances of acrR locus mutants under 2.5 μM closantel and 20 μM TBBPA, compared with the DMSO control. The bars represent mean values and data points are also shown (n = 2 for closantel and TBBPA; n = 3 for DMSO). c, Growth curves under 2 μM closantel of acrR coding and promoter region mutants (n = 7 independent insertions), the wild type (WT; n = 6) and a transposon insertion control (n = 2). The original isolation conditions of acrR mutants are shown as dashed (TBBPA) and solid lines (closantel). Results for the DMSO control are provided in Extended Data Fig. The points represent the OD600 value at 24 h, taken from three independent experiments with technical duplicates. The lines connect median OD600 values per genotype. The highest ciprofloxacin dose of 603.6 μM corresponds to 200 μg ml−1. e, Heatmap of the top TnBarSeq gene hits. The red and blue scale indicates the log2[fold change] in mutant abundance relative to that of the end-point control (red and blue represent enrichment and depletion, respectively). Statistical significance was assessed using TRANSIT resampling (permutation based, two sided and Benjamini–Hochberg adjusted). Values of Padj < 0.05 are marked with an asterisk. The functional categories are also colour coded. Hits are defined by an absolute effect on growth of >20% and P < 0.05. g, Representative plot of P. merdae TnBarSeq insertion in the inositol lipid biosynthesis gene cluster under 500 μM PFNA or DMSO control. Homology with B. thetaiotaomicron is shown using clinker. h, Growth curves of isolated P. merdae transposon mutants with mutation of a putative phosphatidylinositol phosphate phosphatase (PIPPh)-encoding gene (NQ542_07540), Results are shown for the wild type and transposon insertion control (Tn::acrR) under 250 μM PFNA. To further characterize acrR Tn mutant phenotypes, we isolated strains with transposon insertions at seven distinct positions within acrR coding and promoter regions after growth on closantel or TBBPA. All Tn::acrR isolates showed resistance to both chemicals, regardless of the compound they were isolated on (Fig. When further evaluating two Tn::acrR strains, we observed an eightfold increase in the minimum inhibitory concentration (MIC) for closantel compared with the wild-type strain. Consistent with previous reports linking loss of acrR function to ciprofloxacin resistance in diverse bacteria52, we observed an up to 16-fold increase in the ciprofloxacin MIC for P. merdae Tn::acrR mutants compared with the wild type (Fig. The differential enrichment of acrR Tn mutants suggests a degree of specificity in the efflux of chemicals and/or that efflux alone is insufficient in tackling the other chemicals. However, some transporter Tn mutants showed broad pollutant sensitivity, indicating common pollutant tolerance mechanisms in P. merdae. For example, strains with insertions in genes of the fadL family of hydrophobic compound transporters (NQ542_07320 and NQ542_08455) were prevalent negative fitness hits across seven xenobiotics, whereas insertion mutants of the RND multidrug efflux locus NQ542_06240–50 were a significant hit across five chemicals (Extended Data Fig. Other common negative fitness hits were Tn mutants of the genes encoding components of the F0F1 ATP synthase complex (Fig. 4e), the loss of function of which has been associated with increased sensitivity to polymyxin antibiotics under anaerobic conditions53. This is consistent with bisphenol exposure leading to overexpression of F0F1 ATP synthase subunits in B. thetaiotaomicron23. This enzyme class often mediates polymyxin resistance via modifications in membrane polarity54. Overall, genes implicated in efflux and membrane homeostasis modulated P. merdae survival following diverse xenobiotic challenges. To assess whether the involvement of transporters in xenobiotic response is broadly conserved in the order Bacteroidales, we evaluated mutants of B. thetaiotaomicron, a member of a family distant to that of P. merdae55. We tested closantel, TBBPA, PFNA and BPAF on a curated subset of 185 transporter insertion mutants from the B. thetaiotaomicron arrayed transposon loss-of-function mutant library56. Growth was monitored over 24 h and we evaluated the response of each Tn insertion mutant by calculating the normalized relative AUC (nrAUC), comparing the xenobiotic treatment with the DMSO control (Extended Data Fig. The strongest growth reduction for closantel was observed at 2 and 10 μM for the Tn mutant of BT2687 encoding an RND-efflux pump (Fig. This gene is part of an acrR-RND-tolC locus homologous to the P. merdae Tn::acrR closantel hit, suggesting a conserved efflux response (Extended Data Fig. Additional Tn mutants with conditional growth reduction on closantel included the co-localized genes BT0562 and BT0563, which encode multidrug ABC transporter genes. Loss-of-function Tn mutants of BT2687, BT0562 and BT0563 have previously been linked to increased susceptibility to antibiotics and biocides in B. thetaiotaomicron57. The top homology and synteny matches for BT3337 and BT3338 in P. merdae are the RND-encoding loci NQ542_09525–35, which were a specific negative fitness hit for PFNA in the pooled screen (Extended Data Fig. As PFNA can be bioaccumulated by several Bacteroidales42, we assessed whether the phenotypes of Tn::BT3337 and Tn::BT3338 could be caused by increased compound uptake. Testing PFNA bioaccumulation across concentrations with LC–MS/MS showed no differences (Extended Data Fig. 6e,f and Supplementary Data 11), indicating other membrane-related mechanisms. Overall, the shared responses between P. merdae and B. thetaiotaomicron support conserved mechanisms of pollutant and antibiotic tolerance across Bacteroidales. P. merdae TnBarSeq insertion mutant gene hits showed enrichment in various Kyoto Encyclopedia of Genes and Genomes (KEGG) metabolic pathways for most (seven) of the compounds tested. TBBPA 20-μM selection showed a significant enrichment of Tn mutants mapping in the branched-chain amino acid (BCAA) degradation pathway (Extended Data Fig. The second top locus for positive selection under TBBPA was the porA catabolic gene cluster, containing four genes with Tn insertions with >30-fold enrichment (Extended Data Fig. This pathway is involved in the degradation of BCAAs into branched short-chain fatty acids59. Qiao et al.59 reported on a P. merdae porA mutant (with deletion of the NQ542_07740 gene, encoding pyruvate:ferredoxin oxidoreductase) with loss of its beneficial probiotic property of preventing atherosclerosis in mice. PorA is present in several gut bacteria and conserved across Gram-positive and -negative bacteria, including Clostridium sporogenes, where it contributes to immunomodulation60,61. The next top hit for positive selection under TBBPA was a gene encoding a putative BCAA aminotransferase (greater than tenfold enrichment; NQ542_03725). This gene encodes a close homologue of B. fragilis BF9343-3671 (74% identity; 100% coverage), whose knockout mutant stops producing immunomodulatory branched-chain α-galactosylceramide lipids, impacting colonic natural killer T cell regulation in monocolonized mice62. If these xenobiotics select for the loss of bacterial biosynthesis of BCAA-derived beneficial metabolites such as branched short-chain fatty acids and α-galactosylceramides in vivo, they could further impact human health via gut–heart or gut–immune axes. Loss-of-function Tn insertion mutants of the secondary metabolism genes NQ542_07535–55 were positively selected under 500 μM PFNA challenge (~16-fold enrichment), but were negative fitness hits under 20 μM TBBPA (Fig. These genes were identified as being part of a putative inositol lipid biosynthetic gene cluster by homology to B. thetaiotaomicron genes (BT_1522–6) (Fig. This gene cluster is widespread in Bacteroidetes, and loss of inositol lipid production is associated with changes in the cell capsule and increased resistance to antimicrobial peptides63. To validate the phenotype, we isolated two P. merdae mutants (with transposon insertions at two locations of the gene NQ542_07540, which encodes putative phosphatidylinositol phosphate phosphatase) after enrichment in 500 μM PFNA. These strains showed increased resistance to PFNA and hypersensitivity to TBBPA at a range of concentrations, in agreement with pooled fitness (Fig. To evaluate whether the increased resistance to PFNA was due to differences in compound uptake, we tested PFNA bioaccumulation, but no growth-independent differences were seen (Extended Data Fig. 5f), indicating the accumulation-independent function of inositol lipid metabolism in PFNA resistance. In summary, our genetic screen suggests that loss-of-function mutations affecting accessory metabolic pathways might be selected for under xenobiotic exposure, potentially altering the metabolic output of the gut microbiota. Our study uncovered 588 inhibitory interactions between 168 chemicals of concern and common human gut bacteria. Most of these chemicals had not been previously reported to have antibacterial properties. Fungicides and industrial chemicals showed the largest impact, with around 30% exhibiting anti-gut-bacterial activity. For example, although conazole fungicides inhibit ergosterol synthesis, a biochemical pathway unique to fungi, they inhibited some Firmicutes—prominent members of the human gut microbiota. The extent of interactions identified in our study, together with the immense genetic diversity of the gut microbiome64, suggest that the microbiome may harbour vastly more potential xenobiotic targets than was previously appreciated. The prevalence of the inhibitory activities of chemical pollutants revealed by our screen, together with those of human-targeted drugs25,40, challenge the categorical labelling of chemicals into antibiotics, human-targeted drugs, industrial chemicals and pesticides. The antibacterial activity of many xenobiotics is currently not accounted for in assessments of their potential adverse impacts, and toxicity studies are typically set up in ways that are incompatible with capturing gut microbiota effects (Extended Data Fig. To address this gap, toxicology assessment could incorporate tests in single gut bacteria, synthetic communities and gut-on-chip models65. Nevertheless, the vast numbers of chemicals in use and the fast pace at which new molecules are brought to market will make it difficult to implement meaningful experimental assessment for all compounds. Towards tackling this, our study demonstrates the feasibility of computational toxicity predictions that can be used at scale and for safe-by-design product development. Our genetic screens using non-model, yet prevalent, gut bacteria provide mechanistic insights into how bacteria maintain fitness under chemical stressors and indicate potential molecular targets. We observe compound-dependent reliance on particular efflux pumps and their regulators for survival in the presence of pollutants. Regulation of efflux pumps stands out as a mechanism conserved between P. merdae and B. thetaiotaomicron, spanning different Bacteroidales families. This remarkable conservation, together with the previous observation from E. coli genetic screens showing the importance of efflux against human-targeted drugs25,40,66, reinforces the key role of efflux as a general chemical defence mechanism. This mechanistically supports the idea of a general resistant phenotype and that xenobiotics could lead to collateral resistance to antibiotics67. In our study, genetic selection against two persistent pollutants—the flame retardant TBBPA and the anthelmintic closantel—resulted in resistance to the antibiotic ciprofloxacin. Genetic selection in P. merdae is enriched for catabolic and biosynthetic genes, suggesting that metabolism plays a fundamental role in response to xenobiotic stress. This enrichment is reminiscent of the prevalence of metabolism-related gene hits observed for non-antibiotic drug screens in E. coli 40. In our study, loss-of-function mutations in genes coding for enzymes involved in producing secondary metabolites with known impact on human health provided a growth advantage under several xenobiotics. This raises the possibility that exposure to chemical pollutants could impact the selection landscape in the gut, leading to metabolic alterations in microbiome–host interaction pathways. Are our results relevant for real-world exposure scenarios in humans? The concentrations for individuals living in polluted areas and with high occupational exposure (for example, from unprotected pesticide spraying, which is common in much of the developing world), are likely to be much higher. Yet, there are no cohort data available that track both chemical exposure and microbiome dynamics. To enable a large screen and comparison with previous datasets, we tested only a single concentration for most compounds.Although these results clearly highlight compound classes with antibacterial activity, such as fungicides, testing at a single concentration limits the assessment of hazard and risk levels for individual chemicals. Furthermore, the strains used in our study do not span the full diversity of gut bacterial strains and additional screening will be necessary to gain insights into species- and strain-level variation in chemical sensitivity and resistance. Our results therefore currently cannot be directly translated to exposure effects on the gut microbiota in vivo. Going forward, we envisage that our pollutant–bacteria interaction data will encourage the collection of exposure data in microbiome studies, similar to how in vitro drug–bacteria screens accelerated the recording of drug usage in cohort studies25,31. Nevertheless, in vivo assessment of the impact of chemicals exposure is likely to be difficult due to complex interactions between the microbiota and diet, medications and other lifestyle factors. Genetic and physiological insights gained from our study will therefore help to bridge in vitro and in vivo datasets. The chemical–bacteria interaction map and identified genetic determinants of resistance could be used to guide the selection of compounds and molecular effects to monitor in future microbiome studies. An arrayed library of pesticides, pesticide metabolites and pesticide-related compounds was obtained from the Chemical Biology Core Facility at the European Molecular Biology Laboratory (EMBL; Heidelberg, Germany). Original compound stocks were ordered from the collection of PESTANAL standards by Merck/Sigma–Aldrich, with catalogue numbers and Chemical Abstracts Service identifiers listed in Supplementary Data 1. An additional plate containing industrial chemicals and mycotoxins was prepared in house. Compounds were dissolved in DMSO at a concentration of 10 mM. The combined library was arranged on 12 96-well plates, which included eight DMSO controls interspersed within the plate. One unique well position of each plate contained a dye (Trypan Blue), serving as a footprint to enable the detection and prevention of plate mix ups and rotation. A working stock concentrated at 2 mM in DMSO was prepared and aliquoted. Extensive metadata were compiled from multiple sources. PubChem Compound Identifiers (CIDs) were primarily retrieved automatically with compound name searches using the PUG–REST interface, and missing CIDs were added manually. PubChem metadata were then retrieved based on CIDs. PubMed hit counts were retrieved in April 2022 using the National Center for Biotechnology Information E-Utilities application programming interface. Compound names were semi-automatically matched to the Pesticide Compendium of the British Crop Production Council to map hierarchical target classifications, and unmatched entries (for example, pesticide metabolites and non-pesticides) were manually curated. Food and Drug Administration and European Food Safety Authority 2018 pesticide monitoring reports were used to annotate detection in common foods and maximum residue level exceedances. Details of bacterial strains, including their accession numbers in common culture collections and alternative species names, are listed in Supplementary Data 2. All bacteria were grown under anaerobic conditions in a polyvinyl chamber (Coy Laboratory Products) filled with 2% hydrogen and 12% carbon dioxide in nitrogen. Bacteria were grown at 37 °C in modified Gifu anaerobic medium (mGAM; HyServe, Germany, produced by Nissui Pharmaceuticals), prepared according to the manufacturer's instructions and sterilized by autoclaving. Bacterial strains were chosen based on their abundance and prevalence in the healthy human gut33. Assay plates were prepared by first diluting the frozen pesticide library aliquots (2 mM in DMSO) in mGAM within a master deep-well plate from which 50 μl measures were aliquoted into clear, round-bottom, polystyrene untreated microplates (3795; Corning). Assay plates were then placed in the anaerobic chamber overnight to become anoxic. All liquid handling was performed using a Biomek i7 Automated Workstation (Beckman Coulter). Each strain was measured in biological triplicates from independent cultures inoculated on separate days. Bacteria were grown for one or two days (depending on the growth rate) in 10 ml media, inoculated directly from frozen glycerol stocks. Cultures were then diluted 100-fold and incubated again for the same amount of time. Optical densities measured at a wavelength of 600 nm (OD600) were determined and a dilution with an OD600 of 0.1 was prepared. Assay plates were sealed with a gas-permeable membrane (Breathe-Easy; Z380059; Merck), which was additionally pierced with a syringe to prevent the build-up of gas, which otherwise occurs for some species. Plates were stacked without lids and the OD600 was recorded every hour for 24 h using a stacker (BioStack 4; Agilent BioTek) and plate reader (Epoch 2; Agilent BioTek). First, for each growth curve, the minimum OD600 value was set to 0. Then, the raw AUC was calculated for each well using the bayestestR package and area_under_curve function. Further processing of growth curves was done by plate. For the industrial chemicals and mycotoxin library, AUCs were normalized by the median row and column values (to correct for edge effects). Then, for both screens, a z score was calculated as follows: (raw AUC − control median)/control s.d. Each z score was converted to its respective P value and the P value was false discovery rate corrected for the number of compounds tested (n = 1,076). Significance analyses were performed separately for each replicate. A replicate was considered to be significant for growth inhibition if Padj < 0.05 and there was a >20% reduction in the normalized AUC. A compound was considered to significantly inhibit a specific strain if at least two out of three replicates were significant. 38) was inoculated from a frozen ready-to-use stock (equal member ratio; 10% glycerol containing palladium black; 520810; Sigma–Aldrich) into 5 ml mGAM under anaerobic conditions for two 1:100 overnight passages. Com20 was dispensed into wells of 96-well plates containing 500 µl media with xenobiotic compounds, at a starting OD578 of 0.01, sealed with AeraSeal membranes (A9224; Sigma–Aldrich) and anaerobically incubated at 37 °C for 24 h. Pellets from 300 µl of culture were frozen for 16S ribosomal RNA analysis and the OD578 was measured as a biomass proxy. DNA was extracted using a DNeasy UltraClean 96 Microbial Kit (10196-4; Qiagen) and quantified with a Qubit dsDNA BR/HS Assay Kit (Thermo Fisher Scientific). Library preparation and 16S sequencing were conducted at the Next Generation Sequencing Competence Center (Tübingen, Germany) using a two-step PCR protocol. The V4 region was amplified using 515F/806R primers with KAPA HiFi HotStart ReadyMix (Roche) and indexed with unique dual-index barcodes (IDT for Illumina DNA/RNA UD Indexes (Tagmentation)). Libraries were pooled equimolarly, normalized to 10 pM and sequenced on an Illumina MiSeq system (v2 kit; 2 × 250 bp; 20% PhiX). Reads were processed with the DADA2 package (version 1.21.053) in R (version 4.2.0), using the standard operating procedure (https://benjjneb.github.io/dada2/bigdata.html) with the parameters trimLeft: 23, 24; truncLen: 240, 200; maxEE: 2, 2; truncQ: 11, and only retaining merged reads of 250–256 base pairs. Genus-level taxonomy was first assigned against a curated DADA2-formatted database derived from the genome taxonomy database (release R06-RS20253)68 and amplicon sequence variants from expected Com20 genera were then further classified at the species level using full-length 16S ribosomal RNA sequences of all 20 members. Classification was performed with the R package DECIPHER (version 2.24.054) at ≥98% identity. To assess the effects of chemical pollutants on gut microbiota and extrapolate in vitro bioactivity data to potential human health risks, we focused on 168 active chemicals. In vitro growth screening concentrations were used to compute estimated administered doses via the IVIVE tool from the Integrated Chemical Environment (https://ice.ntp.niehs.nih.gov/), leveraging the Environmental Protection Agency's httk R package69 and in-house NICEATM code70. IVIVE calculations determined gut concentrations corresponding to bioactivity levels (MICs from growth screening assays). Essential parameters included the toxicokinetic model type, chemical-specific parameters (log[P], molecular weight, pKa, intrinsic clearance, fraction unbound and tissue partition coefficient), dosing scenarios and exposure routes. The physiologically based toxicokinetic model utilizes multiple compartments for various organs and tissue with perfusion-rate-limited kinetics, simulating elimination primarily via hepatic and renal routes, under a 30-day oral exposure scenario with 24-h intervals. The US Environmental Protection Agency's Toxicity Values Database served as the primary source for condensed in vivo point-of-departure (POD) data, encompassing human health references and animal toxicity values from various studies via the CompTox Chemicals Dashboard (https://comptox.epa.gov/dashboard/). Filters applied to the data available for 126 chemicals included: exposures in mg kg−1 bodyweight or mg kg−1 bodyweight per day (or convertible units); species-specific conversion factors for rats, mice, dogs and rabbits; the study types sub-chronic, subacute and subchronic; and the POD types lowest observed effect level, lowest adverse effect level, no observed effect level and no observed adverse effect level. The SMILES strings for all compounds in PubChem were downloaded, from which 250,000 were selected at random. ECFPs were generated from SMILES strings for all library compounds, previously screened pharmaceutical drugs73,74 and PubChem compounds using ChemPlot75. The ECFPs were then embedded in a two-dimensional representation using UMAP76, with the number of neighbours and minimum distance set to 80 and 0.35, respectively. The distributions for the maximum Tanimoto similarity index for each PubChem compound for the pharmaceutical drugs and combined pharmaceutical drugs and pollutants libraries were compared using a two-sided Kolmogorov–Smirnov test. Molecular descriptor features were generated with the rdck R package. Embedded features were obtained from a pre-trained version of MoLFormer46, which takes the SMILES strings as input and outputs a vector with 768 dimensions (https://huggingface.co/ibm-research/MoLFormer-XL-both-10pct). Machine learning was implemented using scikit-learn version 1.2.2. The data were split into train and test sets in an 8:2 ratio with stratification to ensure equal hit frequencies in both sets. The feature_importances_ attribute was used to access feature importance values. The dense transposon mutant library of P. merdae ATCC 43184 (ref. 49) was used for TnBarSeq mapped to the annotated genome CP102286.1 (GenBank). Two vials of the pooled stock transposon library were used to inoculate mGAM with 30 μg ml−1 erythromycin and grown until the mid-exponential phase. This culture was used to inoculate 1 ml cultures at OD600 = 0.02 in a 2-ml deep-well plate, with duplicates for each xenobiotic and triplicates for the respective control media with 1% DMSO, 0.2% DMSO or plain media (xenobiotic concentrations are provided in Supplementary Data 6). The transposon library cultures were grown to the early stationary phase (16 h for all samples except 2.5 μM closantel, which was grown to 48 h). Genomic DNA was extracted using a MagMAX Microbiome Ultra Nucleic Acid Isolation Kit. The transposon barcodes were amplified by PCR with Q5 Hot Start High-Fidelity 2X (New England Biolabs) using indexed primers (the sequences are provided in Supplementary Data 10), and sequenced with an Illumina NextSeq (single-end sequencing with a read length of 75 nucleotides). Barcodes were counted with 2FAST2Q79 and transposon insertion counts per sample were inputted as wiggle files to TRANSIT, where they were analysed against their respective control using reads in the central 80% of coding regions and normalized using the trimmed total read count, followed by resampling of 20,000 permutations80 and two-sided statistical testing with Benjamini–Hochberg correction (Supplementary Data 6). Gene hits were considered significant at Padj < 0.05 and |log2[fold change]| > 0.25. Post-hoc doublings bias correction was performed in R using the EEL_correction script50. KEGG terms were assigned with EggNOG Mapper81 and KEGG term enrichment was performed using a Fisher's exact test in R. Protein homology was mapped with clinker to B. thetaiotaomicron82; membrane protein families were annotated using TransAAP (hosted by TransportDB)83 and the Conserved Domains Database at the National Center for Biotechnology Information; and additional relevant homologues were identified using PaperBLAST84. To enrich for positively selected mutants, the P. merdae transposon mutant library was cultured in 20 μM TBBPA, 500 μM PFNA or 2 μM closantel for two passages. Cultures were inoculated at an OD600 of 0.02 in 5 ml media in 15-ml tubes and incubated anaerobically for 24 h of growth, then passaged one in 50 into 5 ml fresh media and incubated for an additional 24 h. Following enrichment, cultures were streaked to isolation on anaerobic mGAM agar supplemented with 10 μg ml−1 erythromycin, and individual colonies were picked and glycerol stocked. Transposon barcodes of the isolated colonies were identified by Sanger sequencing (Supplementary Data 7) and validated using locus-specific diagnostic PCR (Extended Data Fig. 5a,b; the oligonucleotides are provided in Supplementary Data 10). For MIC testing, mutant strains were passaged twice for 24 h in anaerobic mGAM before being used to inoculate, at an initial OD600 of 0.025, 100-μl cultures with twofold dilution series of ciprofloxacin, closantel or DMSO control. All plates contained empty-well controls and no compound control. Growth was tracked for 24 h, and the OD600 at 24 h was used to determine the MIC. Three independent replicates were performed on different days, with two technical replicates per strain. The results are provided in Supplementary Data 8. We assembled a subset of the B. thetaiotaomicron arrayed mutant library56, selecting 185 gene mutants annotated as putative transporters. The genes were selected based on annotations from TransAAP83 or if they included in their functional gene annotation the terms pump, efflux or transporter (Supplementary Data 9). Mutants were re-arrayed in three 96-well plates, each containing empty-well controls. Inoculum library plates were grown anaerobically in mGAM from glycerol stocks or a first-passage plate for 24 h. Test plates of each condition were prepared at 2× concentration for 20 μM BPAF, 20 μM TBBPA, 2 and 10 μM closantel, 250 μM PFNA and their respective vehicle controls (0.4 or 1.0% DMSO) and left overnight inside the anaerobic chamber to equilibrate. Plates were inoculated by passaging library inoculum at 1:100 for each test condition in two technical replicates. Plates were sealed with gas-permeable membranes and growth was monitored over 24 h, as described in the section ‘High-throughput screening and MIC determination'. Growth curves were fitted using the Growthcurver package in R78 and the AUC fit was calculated (Supplementary Data 9). One mutant strain that failed growth in the DMSO control was removed from further analysis. To identify mutants whose growth was differentially impacted by xenobiotic treatment, relative AUC values (rAUC) of each mutant were calculated as rAUC = (AUC)/(median AUC of mutants of the plate), assuming a neutral phenotype for most mutants. The average rAUC per technical replicate of mutant strains was calculated. A Welch's t-test was performed and P values were corrected for multiple testing using the Benjamini–Hochberg method. Gene mutant hits with conditional growth effects were defined as those with P < 0.05 and an nrAUC value corresponding to a 20% increase or decrease. Bioaccumulation of BPAF was assessed by reverse-phase LC–MS/MS, as described previously42. Com20 cultures were grown anaerobically at 37 °C in mGAM with 1% DMSO and 20 μM BPAF, with controls lacking either BPAF or bacterial cells. After 24 h, at the stationary phase, 800-μl whole-culture samples were mixed and taken, then the cultures were centrifuged (10 min at 3,200g and room temperature) and 800 μl of the supernatant was collected (the supernatant sample). The rest of the supernatant was discarded, the pellet was resuspended in 8 ml water and 800 μl transferred to a fresh tube (cell sample). For LC–MS/MS, samples were thawed and mixed, then 160 μl extracted with 240 μl organic solvent (1:1 MeOH:ACN + 0.1% formic acid with 20 μM amoxicillin, caffeine, ibuprofen and donepezil as internal standards), incubated for 30 min at −20 °C and cleared by centrifugation (5 min at 10,000g and 4 °C). The supernatants were then transferred to fresh vials. A standard curve was prepared by serially diluting BPAF in the appropriate matrix (untreated supernatant or cell sample extracts of Com20 cultures from the same experiment). Samples (0.5 μl) were analysed on an Agilent 1290 Infinity II chromatography system coupled to an Agilent 6470 Triple Quadrupole (QQQ) system with a Jet Stream ion source operated in dynamic multiple reaction monitoring mode with a cycle time of 500 ms. The analytical column (Agilent Poroshell 120 EC-C18; 2.1 × 100 mm; 1.9 μm) was operated at 40 °C and a constant flow rate of 0.3 ml min−1. Initial conditions were 65% buffer A (water + 0.1% formic acid + 10 mM ammonium formate) and 35% buffer B (9:1 acetonitrile:water + 0.1% formic acid + 10 mM ammonium formate). The source parameters were as follows: gas temperature: 300 °C; gas flow: 10 l min−1; nebulizer: 50 psi; sheath gas temperature: 300 °C; sheath gas flow: 11 l min−1; capillary: 3,500 V (positive) and 3,000 V (negative); nozzle voltage: 2,000 V (positive) and 500 V (negative). BPAF was monitored in negative mode, using a precursor m/z of 335 and fragment m/z of 265 and 69 (500-ms cycle time). The data were analysed using MassHunter Workstation Quantitative Analysis for QQQ version 10.1. Bioaccumulation of PFNA was performed as previously described42. Strains were inoculated at an OD600 of 0.025 in 1-ml cultures with varying PFNA concentrations, plus DMSO and empty-well controls. A 100-μl sample was taken to track growth dynamics in parallel. Samples were extracted with solvent mix (1:1 MeOH:ACN + 0.1% formic acid with 20 µM caffeine, ibuprofen and donepezil as the internal standard), followed by 2 h incubation at 4 °C and 30 min incubation at −20 °C, then centrifuged as above. Samples were injected at a 0.3-μl volume in the LC–MS/MS instrument described above. The analytical column (Agilent Poroshell 120 EC-C18; 2.1 × 50 mm; 1.9 μm) was operated at 40 °C with a constant flow rate of 0.8 ml min−1. The initial buffer composition was 70% A (water + 0.1% formic acid) and 30% B (acetonitrile + 0.1% formic acid). The source parameters were identical to those described above. PFNA was monitored in negative mode, using a precursor m/z of 463 and fragment m/z of 417.9 and 219 (200 ms cycle time). The data were analysed using MassHunter Workstation Quantitative Analysis for QQQ version 10.1 and a custom R script, and the results are provided in Supplementary Data 11. No randomization was used in the microbiological experiments, and data collection and analysis were not performed blind to the conditions of the experiments, as no group allocation was needed. No statistical methods were used to determine samples sizes; instead, they were chosen based on previous experience and according to those in the literature25,56,57. The data distribution was assumed to be normal, but this was not formally tested. Further statistics and reproducibility information are described in each section. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Growth curves, LC–MS/MS raw data and TnBarSeq mapped files used for TRANSIT, as well as the associated processing scripts and intermediate files are available via Mendeley Data (https://doi.org/10.17632/g7hy84t2r6.1). Raw sequencing reads of 16S sequencing of synthetic communities and TnBarSeq data are available from the European Nucleotide Archive (PRJEB97051 and PRJEB76605). Source data are provided with this paper. Code for the generation and comparison of ECFPs is available from GitHub at https://github.com/MRCToxBioinformatics/Compound_bacteria_screen_chemical_space. Code for machine learning is available from GitHub at https://github.com/yeastbeast/PesticideToxicityPrediction. Code for the chemical–genetic experiments is available from GitHub at https://github.com/I-Roux/Pollutant-chemical-genetics. Steffen, W. et al. Planetary boundaries: guiding human development on a changing planet. Persson, L. et al. Outside the safe operating space of the planetary boundary for novel entities. The EAT–Lancet Commission on healthy, sustainable, and just food systems. Medina-Pastor, P. & Triacchini, G. The 2018 European Union report on pesticide residues in food. Maggi, F., Tang, F. H. M. & Tubiello, F. N. Agricultural pesticide land budget and river discharge to oceans. Egli, M. et al. A One-Health environmental risk assessment of contaminants of emerging concern in London's waterways throughout the SARS-CoV-2 pandemic. Groh, K. J., Geueke, B., Martin, O., Maffini, M. & Muncke, J. Overview of intentionally used food contact chemicals and their hazards. Lindell, A. E., Zimmermann-Kogadeeva, M. & Patil, K. R. Multimodal interactions of drugs, natural compounds and pollutants with the gut microbiota. Smalling, K. L. et al. Per- and polyfluoroalkyl substances (PFAS) in United States tapwater: comparison of underserved private-well and public-supply exposures and associated health implications. Mesnage, R. et al. Impacts of dietary exposure to pesticides on faecal microbiome metabolism in adult twins. Clarke, A. et al. Tolfenpyrad displays Francisella-targeted antibiotic activity that requires an oxidative stress response regulator for sensitivity. Utembe, W. & Kamng'ona, A. W. Gut microbiota-mediated pesticide toxicity in humans: methodological issues and challenges in the risk assessment of pesticides. Gois, M. F. B. et al. Impact of occupational pesticide exposure on the human gut microbiome. Hu, J. et al. Low-dose exposure of glyphosate-based herbicides disrupt the urine metabolome and its interaction with gut microbiota. Mesnage, R. et al. Use of shotgun metagenomics and metabolomics to evaluate the impact of glyphosate or roundup MON 52276 on the gut microbiota and serum metabolome of Sprague-Dawley rats. Pesticides: unintended impact on the hidden world of gut microbiota. Defois, C. et al. Food chemicals disrupt human gut microbiota activity and impact intestinal homeostasis as revealed by in vitro systems. Chen, L. et al. Mapping pesticide-induced metabolic alterations in human gut bacteria. Maier, L. et al. Extensive impact of non-antibiotic drugs on human gut bacteria. & Fu, J. Interaction between drugs and the gut microbiome. Host variables confound gut microbiota studies of human disease. Nagata, N. et al. Population-level metagenomics uncovers distinct effects of multiple medications on the human gut microbiome. Koppel, N., Rekdal, V. M. & Balskus, E. P. Chemical transformation of xenobiotics by the human gut microbiota. Maier, L. et al. Unravelling the collateral damage of antibiotics on gut bacteria. Klünemann, M. et al. Bioaccumulation of therapeutic drugs by human gut bacteria. Hens, B., Brouwers, J., Corsetti, M. & Augustijns, P. Supersaturation and precipitation of posaconazole upon entry in the upper small intestine in humans. Maximum Residue Limits in Great Britain (Veterinary Medicines Directorate, 2022); https://assets.publishing.service.gov.uk/media/66ab59a5ab418ab05559315d/_2097921-v2-MRLs_in_GB.pdf Power, C. et al. Investigation of the persistence of closantel residues in bovine milk following lactating-cow and dry-cow treatments and its migration into dairy products. Iezzi, S. et al. Closantel plasma and milk disposition in dairy goats: assessment of drug residues in cheese and ricotta. Tramontano, M. et al. Nutritional preferences of human gut bacteria reveal their metabolic idiosyncrasies. Dai, D. et al. GMrepo v2: a curated human gut microbiome database with special focus on disease markers and cross-dataset comparison. Müller, P. et al. High-throughput anaerobic screening for identifying compounds acting against gut bacteria in monocultures or communities. Formation of propionate and butyrate by the human colonic microbiota. & Mitchell, A. Antibacterial activity of nonantibiotics is orthogonal to standard antibiotics. Emergence of community behaviors in the gut microbiota upon drug treatment. Lindell, A. E. et al. Human gut bacteria bioaccumulate per- and polyfluoroalkyl substances. McCoubrey, L. E., Elbadawi, M., Orlu, M., Gaisford, S. & Basit, A. W. Machine learning uncovers adverse drug effects on intestinal bacteria. Maggiora, G., Vogt, M., Stumpfe, D. & Bajorath, J. Molecular similarity in medicinal chemistry. & Héberger, K. Why is Tanimoto index an appropriate choice for fingerprint-based similarity calculations? Large-scale chemical language representations capture molecular structure and properties. Collell, G., Prelec, D. & Patil, K. R. A simple plug-in bagging ensemble based on threshold-moving for classifying binary and multiclass imbalanced data. Saito, T. & Rehmsmeier, M. The precision-recall plot is more informative than the ROC plot when evaluating binary classifiers on imbalanced datasets. Voogdt, C. G. P. et al. A toolkit for transposon libraries and functional genomics in intestinal Bacteroidales. Correction of a widespread bias in pooled chemical genomics screens improves their interpretability. Shared and unique evolutionary trajectories to ciprofloxacin resistance in Gram-negative bacterial pathogens. Vestergaard, M. et al. Inhibition of the ATP synthase eliminates the intrinsic resistance of Staphylococcus aureus towards polymyxins. & Vrielink, A. Lipid A phosphoethanolamine transferase: regulation, structure and immune response. Zhang, Z. J. et al. Comprehensive analyses of a large human gut Bacteroidales culture collection reveal species- and strain-level diversity and evolution. Arjes, H. A. et al. Construction and characterization of a genome-scale ordered mutant collection of Bacteroides thetaiotaomicron. Liu, H. et al. Functional genetics of human gut commensal Bacteroides thetaiotaomicron reveals metabolic requirements for growth across environments. A. et al. Bile-induced biofilm formation in Bacteroides thetaiotaomicron requires magnesium efflux by an RND pump. Gut Parabacteroides merdae protects against cardiovascular damage by enhancing branched-chain amino acid catabolism. Dodd, D. et al. A gut bacterial pathway metabolizes aromatic amino acids into nine circulating metabolites. Guo, C. J. et al. Depletion of microbiome-derived molecules in the host using Clostridium genetics. Host immunomodulatory lipids created by symbionts from dietary amino acids. Characterization of inositol lipid metabolism in gut-associated Bacteroidetes. Almeida, A. et al. A unified catalog of 204,938 reference genomes from the human gut microbiome. & Bishehsari, F. Gut-on-chip for ecological and causal human gut microbiome research. Systematic screen uncovers regulator contributions to chemical cues in Escherichia coli. Murray, L. M. et al. Co-selection for antibiotic resistance by environmental contaminants. Lundin, D. & Andersson, A. SBDI Sativa curated 16S GTDB database. Pearce, R. G., Setzer, R. W., Strope, C. L., Sipes, N. S. & Wambaugh, J. F. httk: R package for high-throughput toxicokinetics. OPERA models for predicting physicochemical properties and environmental fate endpoints. Chang, X. et al. IVIVE: facilitating the use of in vitro toxicity data in risk assessment and decision making. Periwal, V. et al. Bioactivity assessment of natural compounds using machine learning models trained on target similarity between drugs. & Er, S. ChemPlot, a Python library for chemical space visualization. & Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Sprouffske, K. & Wagner, A. Growthcurver: an R package for obtaining interpretable metrics from microbial growth curves. & Veening, J. W. 2FAST2Q: a general-purpose sequence search and counting program for FASTQ files. DeJesus, M. A., Ambadipudi, C., Baker, R., Sassetti, C. & Ioerger, T. R. TRANSIT—a software tool for Himar1 TnSeq analysis. Cantalapiedra, C. P., Hern̗andez-Plaza, A., Letunic, I., Bork, P. & Huerta-Cepas, J. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Gilchrist, C. L. M. & Chooi, Y.-H. Clinker & Clustermap.Js: automatic generation of gene cluster comparison figures. Elbourne, L. D. H. et al. Transaap: an automated annotation pipeline for membrane transporter prediction in bacterial genomes. Price, M. N. & Arkin, A. P. PaperBLAST: text mining papers for information about homologs. We thank O. Lemke (Charité – Universitätsmedizin Berlin) for advice on machine learning. We thank C. G. P. Voogdt and A. Typas (EMBL) for providing materials and guidance for the TnBarSeq library experiments. We thank A. Shiver and K. C. Huang (Stanford University, USA) for the B. thetaiotaomicron mutant library, and T. Davis (Medical Research Council Toxicology Unit, University of Cambridge, UK) for help with mutant library re-arraying. received funding from the European Research Council under the European Union's Horizon 2020 research and innovation programme (grant 866028). were supported by the UK Medical Research Council (project MC_UU_00025/11). was supported by the German Research Foundation (Cluster of Excellence EXC 2124; grant MA 8164/1-2) and European Research Council (gutMAP project; grant 101076967). and S. Krishna were supported in part by the Intramural Research Program of the National Institutes of Health, National Institute of Environmental Health Sciences. Present address: Cambiotics ApS, Copenhagen, Denmark Present address: Medical Research Council Laboratory of Molecular Biology, Cambridge, UK Present address: Physicians Committee for Responsible Medicine, Washington, DC, USA Present address: Quadram Institute, Norwich, United Kingdom These authors contributed equally: Indra Roux, Anna E. Lindell. Indra Roux, Anna E. Lindell, Tom Smith, Rui Guan, Deniz Rad, Luisa Faria, Sonja Blasche, Stephan Kamrad & Kiran R. Patil Excellence Cluster ‘Controlling Microbes to Fight Infections', Interfaculty Institute of Microbiology and Infection Medicine, M3 Research Center, University of Tübingen, Tübingen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Kaustubh R. Patil and S. Kamrad performed machine learning analysis. planned the microbial community experiments and A.G. performed them. designed the toxicokinetic modelling and S. Krishna performed it. S. Kamrad and Kiran R. Patil supervised the overall study. I.R., A.E.L., S. Kamrad and Kiran R. Patil wrote the paper. All authors commented on the paper. Correspondence to Stephan Kamrad or Kiran R. Patil. The remaining authors declare no competing interests. Nature Microbiology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. A) Network illustrating the number of library compounds targeted at different organisms and their chemical subclasses, based on the BCPC Pesticide Compendium. B) Venn diagram illustrating overlaps of significant interactions between n=3 biological replicates of the main screen. A compound was considered an overall hit if two out of three biological replicates were statistically significant (padj <0.05) and the area under the curve (AUC) was at least 20% lower than for the DMSO control. C) Overlap of hits between main and independent validation screen. D) Normalised AUC values in main screen and validation experiment. A) Chemical susceptibility profiles grouped by phylum. compounds (x-axis) and phyla (y-axis) were clustered by profile similarity, using Euclidean distance and Ward clustering. B) Heatmap showing growth inhibition by compounds (20 μM) which were identified to have broad-spectrum activity, inhibiting more than a third of the tested species. Several of these compounds are widely used and/or persistent organic pollutants, raising concerns about their potential anti-gut-bacteria effects. A) Coverage of chemical space is increased by our new, complementary xenobiotic compound library. The inset focuses on a Tanimoto threshold of 0.75, where significantly increased coverage of the chemical space is observed (p< 2.2e-16, Kolmogorov-Smirnov test). B) Heatmap including all features which belonged to the top10 features in at least one species (ranked by mean across 20 cross-validation sets). Feature importance was accessed using the feature_importances_ attribute of the RandomForestClassifier, which relies on decrease in mean impurity to quantify feature importance. Feature based on molecular distance edge descriptors of carbon atoms (MDEC) (S. Liu et al., 1998) stood out as particularly useful for predicting anti-gut-bacterial activity across different species. C) Correlation analysis of the features with each other. D) Comparison of feature importances between pesticides and pharmaceutical drugs indicates that similar features are important for both compound classes. A) Post hoc correction for possible growth generation bias in log2FC (EEL method, Kim et al. 2024) show main TnBarSeq hits cannot be explained by differences library growth. Values per gene and condition are found in Supplementary Data 6. B) Area under the curve (AUC) for P. merdae Tn mutant library under the compounds tested normalised to control, plotted against total TnBarseq hits. Point shape indicates whether the compound was a growth hit for wild type P. merdae in the previous growth screen. C) Putative membrane transporters and other membrane proteins showing negative fitness in response to more than one xenobiotic. Protein families are indicated in colours. D) Enriched KEGG Pathway categories among all TnBarSeq hits for compounds tested. Number of gene hits are indicated by circle size, P-value is derived from Fisher's test. E) Representative TnBarSeq insertion plot for the porA branched short-chained fatty acid gene cluster (orange) in TBBPA 20 μM and DMSO control. A) Diagnostic PCR genotyping of acrR locus transposon mutants (Tn::NQ542_01170). Transposon insertions are validated as a larger amplicon size compared to the wild type strain (WT). Mutant insertion location information in Supplementary Data 7. B) Genotyping of transposon mutants located at a putative phosphatidylinositol phosphate phosphatase (Tn::NQ542_07540) using primers targeting the junction between transposon and insertion locus confirms transposon location and orientation predicted by barcode sequencing. C) Growth curves of isolate transposon mutants under xenobiotics. Curves represent strains with independent insertions on acrR coding region and promoter regions (purple n=7), Tn::NQ542_07540 from inositol lipid gene cluster (light blue, n=2) and WT strains for comparison (black, n=6). E) Fitness of isolated P. merdae mutants (strain 9 and 10 corresponding to independent insertions at NQ542_07540) and WT control across a range of concentrations of PFNA (0–100 μM). NrAUC is normalised to the median AUC of the WT value per concentration, and the median of DMSO control (0 μM) per genotype. Replicates are shown as points and P-adjusted values were obtained comparing mutants to WT at each concentration with Welch's T-test corrected by multiplicity with BH. F) PFNA bioaccumulation in isolated P. merdae strains of different genotypes grown on a range of PFNA concentrations, as analysed by liquid chromatography coupled to mass spectrometry (LC-MS), see Methods. End OD before compound extraction is indicated in coloured scale. Replicate values are shown as points. Adjusted P-values were calculated compared to WT using two-sided Welch's T-test corrected by multiplicity with BH. A) Scatterplot of relative area under the curve for B. thetaiotaomicron arrayed transporter mutants (rAUC= AUC/(median AUC of the plate)) across two technical replicate plates per condition. Gene name of main hits in agreement are indicated. C) Conservation of the acrABR-tolC locus between B. thetaiotaomicron and P. merdae. Genes in red represent top candidates for positive selection under closantel in P. merdae (repressor of efflux acrR), while blue shows the top hit for sensitivity in the arrayed transporter panel of B. thetaiotaomicron (RND transporter BT687). Protein homology is shown as gray lines connecting orthologs using Clinker. D) Homology and synteny between top RND-efflux genes negative fitness hits in P. merdae and B. thetaiotaomicron. E) Growth of B. thetaiotaomicron PFNA hypersensitive mutants (Tn::BT3337 and Tn::BT3338) and wild type (WT) across a range of concentrations of PFNA (0–100 µM). Replicate values are shown as shapes per strains and P-adjusted values were obtained comparing mutants to WT at each concentration with two-sided Welch's T-test corrected by multiplicity with BH. F) PFNA bioaccumulation profiles of strains Tn::BT3337 and Tn::BT3338 and WT control on a range of PFNA concentrations. Non-significant differences were observed when comparing mutants to WT using two-sided Welch's T-test corrected by multiplicity with BH. A) Analysis of minimum equivalent administered oral doses in humans EAD (green triangle), Point of departure - lowest adverse effect level (POD-LOAEL, orange circle) and Point of departure - no adverse effect level (POD-NOAEL, orange circle), for 149 chemicals. Boxes highlight chemicals with negative and positive POD ratios, enlarged in panel B and C respectively. To contrast the observed sensitivity of gut bacteria to chemicals against animal toxicity estimates, we applied physiologically based toxicokinetic (PBTK) modelling to 168 chemicals. We compared these EADs to the doses at which toxicity was observed in a curated database of animal experiments (ToxValDB). This comparison is captured as the PODratio, which is the ratio of lowest dose at which an adverse effect was observed in mice (Point of departure - lowest adverse effect level: POD-LOAEL) to the previously determined EAD. We detected a positive log10(PODratio) for the vast majority of compounds in our analysis (56 out of 68 compounds for which animal data was available). A positive log10(PODratio) indicates that our in vitro screen picked up effects at lower equivalent doses than required to elicit toxicity in animals. While this is not unexpected, as animal toxicology does not usually consider the microbiome, this indicates that in vitro screening can be a sensitive method to assess potential adverse effects on microbiota not typically picked up in animal toxicity studies. More consideration and innovative study designs are required to better capture the microbiota-damaging toxic effects of xenobiotics in animal studies. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Roux, I., Lindell, A.E., Grießhammer, A. et al. Industrial and agricultural chemicals exhibit antimicrobial activity against human gut bacteria in vitro. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41567-025-03102-5'>Fault-tolerant quantum computation with polylogarithmic time and constant space overheads</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 10:41:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. A major challenge in fault-tolerant quantum computation is to reduce both the space overhead, that is, the large number of physical qubits per logical qubit, and the time overhead, that is, the long physical gate sequences needed to implement a logical gate. Here we prove that a protocol using non-vanishing-rate quantum low-density parity-check (QLDPC) codes, combined with concatenated Steane codes, achieves constant space overhead and polylogarithmic time overhead, even when accounting for the required classical processing. This protocol offers an improvement over existing constant-space-overhead protocols. To prove our result, we develop a technique that we call partial circuit reduction, which enables error analysis for the entire fault-tolerant circuit by examining smaller parts composed of a few gadgets. With this approach, we resolve a logical gap in the existing arguments for the threshold theorem for the constant-space-overhead protocol with QLDPC codes and complete its proof. Our work establishes that the QLDPC-code-based approach can realize fault-tolerant quantum computation with a negligibly small slowdown and a bounded overhead of physical qubits. Quantum computation has promising potential for accelerating the solving of certain classes of computational problems compared with classical computation1,2. However, the inherent fragility of quantum systems poses a major challenge in implementing quantum computations on quantum devices. Fault-tolerant quantum computation (FTQC)3,4 uses quantum error-correcting codes to encode logical qubits and suppress errors, typically incurring a space overhead (the number of physical qubits per logical qubit) and a time overhead (the ratio of physical to logical circuit depth). Currently, two prominent schemes have been proposed for FTQC: one is a concatenated code scheme3,4,5,6, and the other is a quantum low-density parity-check (QLDPC) code scheme7,8,9,10,11,12,13,14,15,16,17. For both schemes, threshold theorems4,5,7,18,19,20,21,22,23,24,25 guarantee that the failure probability of the fault-tolerant simulation can be arbitrarily suppressed, provided that the physical error rate is below a certain threshold. Conventional FTQC schemes, using vanishing-rate codes such as those using surface codes9,10 or concatenated Steane codes5, incur polylogarithmic overheads in both space and time with respect to the size of the original quantum circuit. In recent years, there have been notable advances towards achieving a constant space overhead by using non-vanishing-rate codes. 11 clarified the properties that non-vanishing-rate QLDPC codes must retain to achieve FTQC with constant space overhead, in combination with concatenated Steane codes. 26,27 showed that quantum expander codes16,26,28 can serve as suitable QLDPC codes for this protocol. In this protocol, logical gates are implemented via gate teleportation using auxiliary states encoded in QLDPC codes3,29,30. The fault-tolerant preparation of these auxiliary states has been challenging without relying on the conventional protocol using concatenated Steane codes3,5. However, the conventional protocol incurs a growing space overhead3,5, which undermines complete gate parallelism when we need to maintain a constant space overhead. As a result, the existing analyses in refs. 11,26,27 require sequential gate implementation, leading to a polynomial increase in the time overhead of this protocol. 6 resolved this bottleneck by developing a new constant-space-overhead protocol with complete gate parallelism based on concatenated quantum Hamming codes. However, even with this fully parallel execution of gates, the protocol only achieves quasi-polylogarithmic time overhead. Thus, it remains an open question whether it is possible to design an even faster constant-space-overhead protocol that achieves the polylogarithmic time overhead. From a more fundamental perspective, this question involves a trade-off relation between space and time overhead in FTQC, originally raised in ref. 11 and partially addressed in ref. On the one hand, conventional protocols that use vanishing-rate codes3,4,5,14, whether they are QLDPC codes or concatenated codes, incur polylogarithmic overheads in both space and time. On the other hand, the newer protocols using non-vanishing-rate codes achieve constant space overhead, but this improvement comes at the expense of increased time overhead. Remarkably, both QLDPC-code-based and concatenated-code-based approaches exhibit a similar pattern here as well, where reductions in space overhead have so far led to increases in time overheads beyond the conventional polylogarithmic scaling6,27. The essential focus here is not to propose a marginally improved protocol that outperforms existing ones in certain respects, but rather to address a more fundamental question: whether it is possible to overcome the barrier of super-polylogarithmic time overhead while maintaining constant space overhead. Another fundamental issue is that the existing analyses of threshold theorems for protocols with QLDPC codes11,26,27,31 assume that classical computations are performed instantaneously on arbitrarily large scales, and before our work, the non-zero runtime of classical computations in implementing FTQC was explicitly taken into account only in the analysis of the threshold theorem for concatenated codes6. To fully analyse the overhead achievable in FTQC, it is essential to incorporate the waiting time for noninstantaneous classical computations into the analysis. Analysing overhead in FTQC must consider all possible bottlenecks, including the runtime of classical computation in executing FTQC (for example, decoding algorithms for error correction and gate teleportation), because the decoding runtime, which has often been considered negligible in theoretical analyses, can dominate the asymptotic scaling32. In fact, accounting for decoding delays in surface-code-based protocols not only increases the exponent in the polylogarithmic time overhead, but may potentially make FTQC infeasible due to severe slowdowns—an issue known as the backlog problem32. Such asymptotically non-negligible classical computations can also render the existence of a threshold non-trivial. In practice, even at current scales, classical computations have already become non-negligible bottlenecks, further underscoring the practical necessity of such comprehensive analyses33,34. In this Article, we address these challenges and rigorously prove that a polylogarithmic time overhead can be achieved while maintaining a constant space overhead, even when fully accounting for non-zero classical computation times, using the non-vanishing-rate QLDPC codes. Our main theorem is informally stated as follows. Then, for all target error ε > 0, there exists a threshold pth > 0 such that, if the physical error rate p is below the threshold, that is, p < pth, then there exists a sequence of fault-tolerant circuits \(\{{C}_{n}^{{\rm{FT}}}\}\) such that the classical output of \({C}_{n}^{{\rm{FT}}}\) is sampled from a probability distribution close to that of \({C}_{n}^{{\rm{org}}}\) in terms of total variation distance at most ε, and \({C}_{n}^{{\rm{FT}}}\) has a width WFT(n) and a depth DFT(n) satisfying The formal statement and proof of this theorem, along with the necessary definitions, are provided in Supplementary Sections II and V, in particular in theorem 11 therein. Our analysis is based on standard assumptions in the previous studies on FTQC5,6,7,11,26 (Supplementary Section II). In particular, we assume no geometric locality constraint on two-qubit gates; a CNOT gate can be applied to an arbitrary pair of physical qubits in a single time step. This is motivated by platforms such as neutral atoms, trapped ions and optics, which may offer pathways to realizing such non-local interactions35,36,37,38,39,40,41,42,43,44. To prove theorem 1, we present and analyse a hybrid fault-tolerant protocol that combines concatenated Steane codes5 and non-vanishing-rate QLDPC codes with an efficient decoding algorithm, specifically quantum expander codes16, refining the previous analyses in refs. Our analysis enables the polylogarithmic time overhead by increasing gate parallelism in the gate operations using gate teleportation, substantially improving the polynomial time overhead of the previous analyses in refs. 11,26,27, while still maintaining a constant space overhead. Our analysis addresses and fixes a previously overlooked problem in the existing analyses11,26,27 of the threshold theorem for the QLDPC codes. In the analyses of FTQC, it is conventional to consider the local stochastic Pauli error model, where the correlated errors may occur across the entire fault-tolerant quantum circuit, as in refs. For each code block of the QLDPC codes, the existing analyses in refs. 11,26,27 argue that the physical error rate of the codeword is suppressed after noisy syndrome measurements followed by quantum error correction using decoding algorithms, but this codeword state is on specific code blocks, that is, only in a small part of the entire fault-tolerant circuit on which the local stochastic Pauli error model is defined. To address this issue, we introduce a new method called partial circuit reduction. This method enables error analysis of the entire fault-tolerant circuit through the examination of individual gadgets on the code blocks of QLDPC codes. Our approach allows us to leverage the existing results from the decoding algorithms11,26,27 as a black box, so that we fully complete the proof of the threshold theorem for the constant-space-overhead protocol with the QLDPC codes. Furthermore, by combining this method with theoretical advances of the decoding algorithms in refs. 26,27, we show that it is indeed possible to achieve higher parallelization of the logical gates compared with the existing analyses in refs. 11,26,27, resulting in a polylogarithmic time overhead. These advances contribute to a fundamental understanding of the overhead and its space-time trade-off in FTQC and provide a robust foundation for realizing quantum computation. We note that, in a concurrent work, another fault-tolerant protocol with constant space overhead and polylogarithmic time overhead45 has been proposed, relying on an approach substantially different from ours. The primary contribution of our work lies in the proof of the threshold theorem, which takes into account the non-zero runtime of classical computation. In our protocol, quantum expander codes, which are Calderbank–Shor–Steane low-density parity-check codes with a non-vanishing rate, serve as the registers that store and protect logical qubits46,47. Using the conventional notation [[N, K, D]] for the code parameters (N is the number of physical qubits, K is the number of logical qubits and D is the code distance), quantum expander codes16 exhibit parameters of \([[N,K={{\varTheta}}(N),D={{\varTheta}}(\sqrt{N})]]\), so the rate K/N remains positive as N → ∞. Together with the small-set-flip decoder, these codes suppress the probability of decoding failure, which may lead to logical error, exponentially in the code distance, as established in refs. These properties make quantum expander codes well suited for high-capacity, robust quantum memory. To realize a universal gate set that manipulates this QLDPC-encoded quantum memory, we use a hybrid approach involving concatenated Steane codes5,47,48 to prepare auxiliary states encoded by quantum expander codes in a fault-tolerant manner, enabling the logical gates via gate teleportation. The fault-tolerant protocol with concatenated Steane codes can be used to simulate arbitrary original quantum circuits, albeit with polylogarithmic overheads in both time and space5. In our hybrid approach, this concatenated Steane code protocol is crucially used to prepare logical auxiliary states that are encoded in the QLDPC code. The fault-tolerant preparation of an auxiliary state, denoted by \(\left\vert \psi \right\rangle\), which is a physical state representing a codeword of a QLDPC code that encodes a target logical auxiliary state, can be understood intuitively as follows. While \(\left\vert \psi \right\rangle\) can be prepared by some non-fault-tolerant unitary circuit, such a circuit cannot be directly used for its fault-tolerant preparation. Instead, the idea is to first prepare \(\left\vert \psi \right\rangle\) as a logical state encoded in the concatenated Steane code. Because the well-established fault-tolerant protocol with the concatenated Steane code supports simulation of universal quantum computation, this Steane-encoded version of \(\left\vert \psi \right\rangle\) can be prepared with a sufficiently low logical error rate. We then convert this logical state into the physical state \(\left\vert \psi \right\rangle\) over the physical qubits, by decoding the concatenated Steane code. Roughly speaking, this step introduces errors on each physical qubit at the physical error rate, up to constant factors. This fact was implicitly used in the previous analysis of the constant-space-overhead protocol based on QLDPC codes11, and we provide a rigorous proof and explicit bound on this error rate in Supplementary Appendix A. Although each physical qubit may now suffer errors at the physical error rate, it is important to recall that \(\left\vert \psi \right\rangle\) is already encoded as a codeword of the QLDPC code. Therefore, we can transition to using the decoder of the QLDPC code to continue suppressing its decoding failure probability and protect the encoded logical state in \(\left\vert \psi \right\rangle\). Once these QLDPC-encoded auxiliary states are fault-tolerantly prepared, they enable the implementation of logical Clifford and non-Clifford gates on the QLDPC logical qubits via fault-tolerant gate teleportation3,29,30, thereby equipping our protocol with a universal set of logical operations. The compilation process in our hybrid fault-tolerant protocol consists of two stages (see Fig. 1 and Supplementary Section III for details). The process involves two main stages. The aim of the first stage is to convert an original quantum circuit \({C}_{n}^{{\rm{org}}}\) to an intermediate circuit acting on registers, where each register is a collection of at most K qubits. The intermediate circuit is composed of a predetermined set of elementary operations, which includes a \({\left\vert 0\right\rangle }^{\otimes K}\)-state preparation, Clifford-state preparations, magic-state preparations, Pauli-gate operations, a CNOT-gate operation, a ZK-measurement operation, a Bell-measurement operation and a wait operation. Then, \(\left\vert 0\right\rangle\)-state preparations in the original circuit are replaced by \({\left\vert 0\right\rangle }^{\otimes K}\)-state preparations, and Z-basis measurements are replaced by ZK-measurement operations. Clifford gates (for example, a tensor product of the Hadamard H gate) and non-Clifford gates (for example, a tensor product of T gates) in the original circuit are converted into a Clifford unitary UC acting on two registers and a non-Clifford unitary UT on a single register, respectively. To maintain constant space overhead, the number of the \({\left\vert 0\right\rangle }^{\otimes K}\)-state preparations, the UC gates, the UT gates and the ZK-measurement operations performed at a single time step is limited to \({\varTheta}(W(n)/{\rm{polylog}}(n/{\varepsilon}))\), where W(n) is the original circuit width. Following this, each of the UC gates and the UT gates is replaced by a sequence of elementary operations performing the corresponding teleportation with the aid of auxiliary registers, which is denoted as a UC-gate or a UT-gate abbreviation. The next stage is the compilation of this intermediate circuit into the fault-tolerant circuit \({C}_{n}^{{\rm{FT}}}\). This stage is achieved by replacing each elementary operation with a corresponding gadget, and an EC gadget is inserted between each pair of adjacent gadgets. First, an original quantum circuit is compiled into an intermediate circuit described as follows. An original circuit consists of initial state preparation, Clifford gates, non-Clifford gates and measurements. The qubits of the original circuit are partitioned into collections of at most K qubits each, called registers6. Here, K is chosen to match the number of logical qubits of the QLDPC code to be used, because each qubit in the intermediate circuit will eventually be regarded as a logical qubit of the code. For the compilation, we rewrite the original circuit using the following operations acting on registers: a \({\left\vert 0\right\rangle }^{\otimes K}\) -state preparation, Clifford unitaries UC acting on two registers, non-Clifford unitaries UT on a single register and a ZK measurement (Fig. To achieve constant overhead, we restrict the number of the preparations, the measurements and the unitary gates executed in parallel, which is done by inserting the wait operations to delay part of the operations. Then, we regard each of the unitary gates as an abbreviation of a gate-teleportation circuit composed of elementary operations and equipped with ancillary registers. Here, the set of the elementary operations is composed of a \({\left\vert 0\right\rangle }^{\otimes K}\)-state preparation, Clifford-state preparations, magic-state preparations, Pauli-gate operations, a CNOT-gate operation, a ZK-measurement operation, a Bell-measurement operation and a wait operation. At this point, the original circuit has been converted to a circuit composed of the elementary operations, which we call the intermediate circuit (Fig. Next, this intermediate circuit is compiled into a fault-tolerant circuit. In this stage, each elementary operation within the intermediate circuit is replaced with its corresponding gadget, and each register is replaced with the QLDPC code block. Each gadget is a physical circuit designed to perform the logical operation acting on the logical qubits within a code block of the QLDPC code in a fault-tolerant way. As described above, the state-preparation gadget is fault-tolerantly constructed using the concatenated Steane code, while the other gadgets can be implemented transversally for quantum expander codes. Furthermore, an error-correction (EC) gadget acting on each code block is inserted between each pair of adjacent gadgets. Through this process, the final fault-tolerant circuit is constructed. A crucial feature of our protocol is its ability to achieve higher parallelism in executing logical gates compared with existing constant-space-overhead fault-tolerant protocols with QLDPC codes11,26,27, making it possible to achieve polylogarithmic time and constant space overhead simultaneously. In previous analyses11,26,27 for constant-overhead FTQC with QLDPC codes, logical gates could act simultaneously act on only O(W(n)/poly(n/ε)) logical qubits. This limitation naturally led to a polynomial time overhead. Our protocol overcomes this bottleneck by providing a refined analysis of the required code size for quantum expander codes. These codes exhibit an exponential suppression of decoding failure probability as we increase the code distance. Indeed, this error suppression enables the use of a very small QLDPC code block size, scaling only polylogarithmically. The use of such small code blocks is important for our protocol; it reduces the resources required for the auxiliary states needed for gate teleportation. The QLDPC-encoded auxiliary states required for gate teleportation are prepared using a concatenated Steane code protocol. The fault-tolerant preparation using the concatenated Steane code itself incurs polylogarithmic space and time overheads. Implementation of each logical qubit of the QLDPC code thus requires a polylogarithmic number of physical qubits. Furthermore, each logical gate, implemented via these auxiliary logical states prepared using concatenated Steane codes, also introduces a polylogarithmic time overhead. This polylogarithmic requirement for the QLDPC code block size, in turn, enables the performance of more logical operations, that is, those on \(O(W(n)/{\rm{polylog}}(n/\varepsilon ))\) logical qubits, in parallel at each time step while maintaining the constant space overhead. This substantially increased parallelism in logical gate execution contributes to achieving a polylogarithmic time overhead. The detailed overhead analysis is presented in Supplementary Section V. A key challenge in proving the threshold theorem for FTQC with QLDPC codes lies in consistently analysing errors across the entire fault-tolerant circuit, especially when individual code blocks are decoded locally. Existing analyses often focus on error suppression within a single QLDPC code block, overlooking correlations between the block and the rest of the circuit11,26,27, which is globally subject to a local stochastic Pauli error model11,26,27. To address this fundamental issue, we introduce and utilize a technique called partial circuit reduction. The core motivation for this technique is to provide a systematic framework for analysing errors in a modular fashion. In our fault-tolerant circuit, each operation gadget is followed by EC gadgets for the associated code blocks, forming a circuit segment, which we call a rectangle. Instead of tackling the entire fault-tolerant circuit at once, we sequentially analyse errors in each rectangle. Partial circuit reduction allows us to formally replace a noisy rectangle in the fault-tolerant circuit with its ideally behaving, noiseless version, while appropriately updating the probability distribution of faulty locations in the subsequent, not-yet-analysed parts of the circuit. This step-by-step reduction is easy to use because it only requires analysis of a local segment at a time, while ensuring that the global statistics of fault occurrences continue to follow the same model with updated parameters. This method enables us to systematically manage error correlations and integrate established results from decoding algorithms26,27 as black-box components within a complete and rigorous proof of threshold existence. The detailed mathematical formulation of partial circuit reduction and its application in proving the threshold theorem are presented in Supplementary Section V. With all techniques described above, we prove the threshold theorem for our protocol. A fundamental challenge in theoretical FTQC analysis, often leading to oversimplification, is the reliance on the assumption that classical computation required in error correction and gate teleportation can be performed instantaneously, that is, in zero time26,27,49. In reality, these computations have a non-zero runtime that may grow as the size of original circuits becomes larger. The impact of this additional runtime is not limited to the time overhead. Because errors accumulate during the wait time for classical computation, they can potentially affect space overheads and even the existence of a fault-tolerant threshold. Our analysis explicitly incorporates this non-zero runtime by inserting appropriate wait periods in the fault-tolerant circuit such that outcomes of classical computation can be used to choose on-demand operations. To establish a threshold under this realistic condition, we identify crucial properties for the decoding algorithm of the QLDPC codes to satisfy. These include: (1) the single-shot property, enabling error correction from a single round of potentially noisy syndrome measurements; (2) iterative execution, where each internal loop of the decoder completes in O(1) time with O(N) classical parallel processors; (3) exponential suppression of decoding failure probability with code distance; and (4) monotonic reduction of residual physical error rate with the number T of the internal loops when physical and syndrome error rates are below certain thresholds. We confirm that the small-set-flip decoder for the quantum expander code26,27 fulfils these requirements. These conditions ensure that the execution of the decoder within the EC gadget (that is, the wait time) can always be kept to a constant time, even if we carefully account for classical processing time. This has enabled us to rigorously prove the existence of a fault-tolerance threshold for our protocol without compromising overhead scalings. The detailed conditions for the decoder and the proof of the threshold theorem are provided in Supplementary Sections IV and V, respectively. In this Article, we have presented a hybrid fault-tolerant protocol that combines concatenated codes for gate operations and non-vanishing-rate QLDPC codes (in particular, the quantum expander codes16,26,28) for quantum memory. These results contribute to a fundamental understanding of the overhead and its space-time trade-off required to realize FTQC. Unlike previous analyses that assumed instantaneous classical computations, our work explicitly accounts for non-zero-time classical computation times and their associated error accumulation. Incorporating classical runtime into the overhead analysis and establishing a threshold theorem under this setting not only offers a comprehensive understanding of overheads accounting for every possible bottleneck, but also provides a solid foundation for realizing FTQC. An encouraging aspect of our results is that polylogarithmic time overhead and constant space overhead can be achieved using well-established components: quantum expander codes and concatenated Steane codes. Both of these have been the focus of active experimental research in various physical architectures. Recent theoretical proposals indicate the feasibility of quantum expander codes in scalable systems such as neutral atoms and superconducting circuits42,43,44, while small-scale protocols with the Steane code have already begun to be demonstrated experimentally37,50,51. Our findings indicate a promising potential for low-overhead FTQC using the hybrid approach that combines non-vanishing-rate QLDPC codes and concatenated codes, in addition to relying solely on the code-concatenation approach as proposed in refs. With regard to practical implementation, our results clarify the competing time overheads of these two constant-space-overhead approaches, highlighting the importance of a comprehensive investigation into the physical realizability of both approaches. This work does not have any associated data. Montanaro, A. Quantum algorithms: an overview. Dalzell, A. M. et al. Quantum Algorithms: A Survey of Applications and End-to-End Complexities (Cambridge Univ. Gottesman, D. An introduction to quantum error correction and fault-tolerant quantum computation. Symposia in Applied Mathematics (ed. Lomonaco, S. J. Jr) 13–58 (American Mathematical Society, 2010). & Chuang, I. L. Quantum Computation and Quantum Information: 10th Anniversary Edition (Cambridge Univ. Aliferis, P., Gottesman, D. & Preskill, J. Quantum accuracy threshold for concatenated distance-3 codes. Yamasaki, H. & Koashi, M. Time-efficient constant-space-overhead fault-tolerant quantum computation. Kitaev, A. Y. Quantum computations: algorithms and error correction. Kitaev, A. Fault-tolerant quantum computation by anyons. & Kitaev, A. Y. Quantum codes on a lattice with boundary. Litinski, D. A game of surface codes: large-scale quantum computing with lattice surgery. Gottesman, D. Fault-tolerant quantum computation with constant overhead. Bombín, H. Single-shot fault-tolerant quantum error correction. Bravyi, S. et al. High-threshold and low-overhead fault-tolerant quantum memory. Fowler, A. G., Mariantoni, M., Martinis, J. M. & Cleland, A. N. Surface codes: towards practical large-scale quantum computation. Breuckmann, N. P. & Eberhardt, J. N. Quantum low-density parity-check codes. Leverrier, A., Tillich, J.-P. & Zémor, G. Quantum expander codes. In 2015 IEEE 56th Annual Symposium on Foundations of Computer Science (ed. & Zémor, G. Quantum LDPC codes with positive rate and minimum distance proportional to the square root of the blocklength. Aharonov, D. & Ben-Or, M. Fault-tolerant quantum computation with constant error rate. Knill, E., Laflamme, R. & Zurek, W. H. Resilient quantum computation: error models and thresholds. Reichardt, B. W. Fault-tolerance threshold for a distance-three quantum code. 33rd International Conference on Automata, Languages and Programming (eds Bugliesi, M. et al.) 50–61 (Springer-Verlag, 2006). Terhal, B. M. & Burkard, G. Fault-tolerant quantum computation for local non-Markovian noise. Aliferis, P. & Preskill, J. Fault-tolerant quantum computation against biased noise. Aliferis, P. & Terhal, B. M. Fault-tolerant quantum computation for local leakage faults. Accuracy threshold for postselected quantum computation. 2018 IEEE 59th Annual Symposium on Foundations of Computer Science (ed. Constant Time Decoding of Quantum Expander Codes and Application to Fault-Tolerant Quantum Computation. Efficient decoding of random errors for quantum expander codes. 50th Annual ACM SIGACT Symposium on Theory of Computing (eds Diakonikolas, I. et al.) 521–534 (ACM, 2018). Knill, E. Scalable quantum computing in the presence of large detected-error rates. Knill, E. Quantum computing with realistically noisy devices. & Preskill, J. Topological quantum memory. Demonstrating real-time and low-latency quantum error correction with superconducting qubits. Acharya, R. et al. Quantum error correction below the surface code threshold. Logical quantum processor based on reconfigurable atom arrays. Moses, S. A. et al. A race-track trapped-ion quantum processor. Ryan-Anderson, C. et al. Realization of real-time fault-tolerant quantum error correction. Egan, L. et al. Fault-tolerant control of an error-corrected qubit. Blueprint for a scalable photonic fault-tolerant quantum computer. Litinski, D. & Nickerson, N. Active volume: an architecture for efficient fault-tolerant quantum computers with limited non-local connections. Yamasaki, H., Fukui, K., Takeuchi, Y., Tani, S. & Koashi, M. Polylog-overhead highly fault-tolerant measurement-based quantum computation: all-gaussian implementation with gottesman-kitaev-preskill code. Sunami, S., Tamiya, S., Inoue, R., Yamasaki, H. & Goban, A. Scalable networking of neutral-atom qubits: Nanofiber-based approach for multiprocessor fault-tolerant quantum computer. Tremblay, M. A., Delfosse, N. & Beverland, M. E. Constant-overhead quantum error correction with thin planar connectivity. Xu, Q. et al. Constant-overhead fault-tolerant quantum computation with reconfigurable atom arrays. Nguyen, Q. T. & Pattison, C. A. Quantum fault tolerance with constant-space and logarithmic-time overheads. 57th Annual ACM Symposium on Theory of Computing (eds Koucký, M. & Bansal, N.) 730–737 (ACM, 2025). Calderbank, A. R. & Shor, P. W. Good quantum error-correcting codes exist. Multiple particle interference and quantum error correction. Knill, E. & Laflamme, R. Concatenated quantum codes. Gottesman, D. & Chuang, I. L. Demonstrating the viability of universal quantum computation using teleportation and single-qubit operations. Demonstration of fault-tolerant universal quantum gate operations. Demonstration of fault-tolerant Steane quantum error correction. Yoshida, S., Tamiya, S. & Yamasaki, H. Concatenate codes, save qubits. This work was supported by JST (Moonshot R&D) under grant no. was supported by JSPS KAKENHI (grant no. 23KJ0521) and FoPM, WINGS Program, the University of Tokyo. acknowledges JST PRESTO (grant nos. JPMJPR201A and JPMJPR23FC), JSPS KAKENHI (grant no. JP23K19970) and MEXT Quantum Leap Flagship Program (MEXT QLEAP) (grant nos. The quantum circuits shown in this Article were drawn using QPIC53. Nanofiber Quantum Technologies Inc., Tokyo, Japan Photon Science Center, Graduate School of Engineering, The University of Tokyo, Tokyo, Japan Department of Computer Science, Graduate School of Information Science and Technology, The University of Tokyo, Tokyo, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Shiro Tamiya, Masato Koashi or Hayata Yamasaki. The authors declare no competing interests. Nature Physics thanks Benjamin Brown and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Tamiya, S., Koashi, M. & Yamasaki, H. Fault-tolerant quantum computation with polylogarithmic time and constant space overheads. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41566-025-01780-4'>Electrical control of photon spin angular momentum in organic electroluminescent materials</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 10:20:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The photon spin information encoded in circularly polarized (CP) light is of high interest for current and future technologies, including low-power displays, encrypted communications and high-performance quantum applications. Engineering organic light-emitting diodes (LED) to emit oppositely handed electroluminescent CP light typically requires access to left- and right-handed chiral molecules. In conjugated polymer LEDs, the handedness of CP electroluminescence also depends on the active-layer thickness or direction of current flow. For a given active-layer thickness, it remains unknown whether a single-handed chiral material can emit CP light with opposite handedness in the same LED architecture. Here we demonstrate organic LEDs in which the handedness of the emitted CP electroluminescence can be controlled electrically, solely by using specific interlayers with no change in the emissive material composition or thickness. We reveal that this occurs due to a change in mechanism for the generation of CP electroluminescence, as a function of the recombination zone position within the device. This result provides a paradigm shift in the realization of organic CP-LEDs with controllable spin angular momentum information and further contributes to ongoing discussions relating the fundamental physics of chiral optoelectronics. Chirality is the property of an object with broken inversion and mirror symmetry, resulting in a pair of non-superimposable mirror image forms. In chemistry, each non-superimposable mirror image of a chiral molecule is called an enantiomer1. Chiral materials can absorb, emit and refract circularly polarized light (CPL), which is light with a defined spin angular momentum (SAM). SAM is the intrinsic angular momentum associated to a polarized photon, with a quantized value of ±ℏ per photon, corresponding to left- or right-handed circular polarization. Chiral organic semiconductors have recently attracted much interest as the emissive layer in circularly polarized light-emitting diodes (CP-LEDs)2,3. Partly due to the encoded SAM, CP-LEDs will enable multiple technological advances, from the generation of more energy-efficient display technologies to highly precise sensors to devices for quantum information processing and holography4,5. Regardless of which of these mechanisms is operative, the handedness of the emitted CPL from organic chiral materials correlates to the handedness of the enantiomer of the emissive layer: one enantiomeric emitter gives a CPL of one preferred handedness, whereas the other enantiomer gives equal-magnitude CPL of the opposite handedness13. When referring to the emission from a CP-LED, we term this as normal circularly polarized electroluminescence (NCPE). One of the most successful approaches to realize high circularly polarized electroluminescence (CP-EL) in CP organic light-emitting diodes (OLEDs) has been the use of polymeric thin-film blend materials, where an achiral conjugated polymer is blended with a chiral small-molecule additive14,15. We have found the helical small-molecule (P)-aza[6]helicene (aza[6]H; Fig. 1a) to be a particularly useful additive to produce films with very high chiroptical activity16. While optimizing aza[6]H:polymer blends to improve CP-OLED device efficiencies16,17, we discovered a distinct CP-EL mechanism, where the handedness of the CPL emitted inverts as a function of the device architecture (conventional versus inverted) or the emission direction (front or back of the device). This anomalous CP-EL (ACPE)18 was found to correlate to the directionality of current flow in the device, and was assigned to induced orbital angular momentum (OAM) in the charge carriers, with subsequent transfer to the SAM of the emitted photons. Further details on these mechanisms are provided in Supplementary Note 1. a, Structure and energy levels of polymer system (F8BT) and chiral additives (aza[6]H or monoF[6]H). b, Dissymmetry factor of F8BT:(P)-aza[6]H and F8BT:(P)-monoF[6]H thin films (160 nm), annealed at 140 °C. c, J–V–L characteristics of conventional CP-OLEDs based on F8BT:(P)-aza[6]H/(P)-monoF[6]H, with or without TFB as the HTL. d, J–V–L characteristics of inverted CP-OLEDs based on F8BT:(P)-aza[6]H/(P)-monoF[6]H, with or without TCTA as the HTL. Voltage–current density curves are shown as solid lines, and voltage–luminance curves are shown as dashed lines. The interplay between NCPE and ACPE in chiral polymer LEDs remains unclear. From a practical standpoint, exploiting NCPE requires the use of both enantiomers in separate devices to access opposite-handed CPL. Conversely, ACPE requires the use of two different device architectures. It would be a notable advance to be able to access both handedness of CPL from the same enantiomer material in the same device architecture. Here we realize such a goal, demonstrating how CPL handedness can be electrically controlled in a chiral polymer-based CP-OLED. Switching between NCPE and ACPE is achieved by controlling the charge injection and charge balance through the use of interlayers, and is enabled by a newly developed chiral additive, a monofluorinated [6]helicene ((P)-monoF[6]helicene (monoF[6]H); Fig. NCPE occurs with balanced charge transport in the emissive layer when the recombination zone (RZ) is located in the centre of the device and is independent of the device architecture (that is, conventional or inverted). ACPE occurs with unbalanced charge transport in the emissive layer when the RZ is located close to one of the transport layers (Fig. To the best of our knowledge, such electrical control of CP-EL has only previously been observed in inorganic materials19 and in different device configurations (namely, light-emitting transistors). As such, we believe the learnings in this work will enable wholly new directions and progress of CP-OLED device technologies with controllable SAM and contributing to the ongoing studies of chirality-dependent optoelectronic physics20. Although aza[6]H has proved to be a useful additive to generate blend polymer films with large chiroptical activity, the optoelectronic performance of poly(9,9-dioctylfluorene-alt-benzothiadiazole) (F8BT):aza[6]H materials are hindered by the shallower highest occupied molecular orbital of aza[6]H (5.4 eV) compared with the highest occupied molecular orbital of F8BT (5.8 eV; Supplementary Fig. This results in aza[6]H acting as a hole trap in the blend, which decreases the device performance and limits the type of device configurations that can be used for working OLEDs. To address this issue, we developed fluorine-substituted monoF[6]H, a chiral additive with a deeper highest occupied molecular orbital (5.8 eV) than its aza[6]H counterpart (Supplementary Figs. 1–8), to reduce hole trapping in polymer blends (Fig. When blended with F8BT at the same concentration (10 wt%) and processing conditions (160 nm thickness), both aza[6]H and monoF[6]H induce a chiral phase in the polymeric film. The CD of these films contains negligible contributions from linear dichroism and linear birefringence, with no CD sign inversion on rotation or flipping of the samples (Supplementary Figs. This was further confirmed by Mueller matrix polarimetry (MMP) data showing that the CD elements m30 and m03 are symmetric whereas circular birefringence elements m21 and m12 are antisymmetric (Supplementary Figs. 15 and 16), which is consistent with prior studies on aza[6]H (ref. This excludes any structural contributions to CPL, with the high chiroptical activity being attributed to the strong excitonic coupling between polymer-chain chromophores21. When optically excited, no change in the emitted CPL handedness is observed for a given enantiomer, regardless of the excitation geometry (Supplementary Figs. Conventional and inverted devices were fabricated with the same emissive layer thickness (160 nm). 1c) of F8BT:(P)-aza[6]H blends with poly(9,9-dioctylfluorene-alt-N-(4-sec-butylphenyl)-diphenylamine) (TFB) as an additional hole transport layer (HTL) had lower efficiencies and higher turn-on voltages compared with F8BT:(P)-monoF[6]H-based devices (Supplementary Figs. Furthermore, although the HTL is required to realize working CP-OLEDs for F8BT:(P)-aza[6]H blends, devices using F8BT:(P)-monoF[6]H can be fabricated HTL-free and maintain lower turn-on voltages compared with CP-OLEDs with aza[6]H (Supplementary Table 2). Inverted CP-OLEDs for all conditions (Fig. 1d) show higher device efficiencies compared with the conventional architecture (Supplementary Figs. Consistent with conventional devices, inverted devices using F8BT:(P)-aza[6]H require (tris(4-carbazoyl-9-ylphenyl)amine) (TCTA) as an HTL, whereas F8BT:(P)-monoF[6]H-based CP-OLEDs can operate without an HTL. Inverted monoF[6]H-based CP-OLEDs demonstrate high efficiencies and, for HTL-free devices, very low efficiency roll-off (Supplementary Fig. The superior performance of CP-OLEDs incorporating F8BT:(P)-monoF[6]H confirms that the new chiral additive reduces the trapping caused by aza[6]H. The lower density of trap states in F8BT:monoF[6]H blends is also supported by hole transport measurements via the space-charge-limited current method (Supplementary Figs. Almost three orders of magnitude difference in the hole mobility was observed in comparing blends of monoF[6]H and aza[6]H, with higher hole trap densities in aza[6]H blends. Mobility measurements of bilayers including an HTL reveal that for aza[6]H blends, the HTL improves hole mobility, whereas for monoF[6]H, it limits the injection of holes, pinning the RZ close to the anode in both cases. No differences in electron transport properties of the two additives were observed, even when measured in bilayers with an electron transport layer (Supplementary Fig. These measurements, performed using the same device stack as the respective CP-OLEDs, reveal that (1) in devices (conventional and inverted) with an HTL, the RZ is pinned near the HTL interface and (2) in HTL-free devices, the RZ is located closer to the centre of the stack due to balanced charge carrier injection and transport. This is consistent with the measured efficiency roll-off: efficiency roll-off is minimized in HTL-free devices due to the more balanced charge recombination and reduced quenching (Supplementary Fig. Stability measurements further support this analysis, with conventional and inverted HTL-free devices showing improved stability compared with counterparts with an HTL (Supplementary Fig. Capacitance–voltage (CV) and capacitance–frequency (CF) measurements indicate that devices with HTLs suffer from charge accumulation and capacitance build-up due to the unbalanced charge carrier mobility and the pinned RZ close to the HTL. Specifically, HTL-free devices feature a sharp drop in capacitance after turning on in CV measurements, whereas devices with an HTL feature a capacitance build-up (Supplementary Fig. CF measurements, carried out after device turn on, also feature a higher capacitance at low frequencies in HTL-free devices (Supplementary Fig. Both these characterizations demonstrate charge accumulation and capacitance build-up due to unbalanced charge carrier mobility and support the location of the RZ near the interface with the HTL. Enabled by the discovery of a chiral polymeric material with improved hole mobility, it was found that the handedness of CP-EL could be controlled through the use of conventional and inverted F8BT:(P)-monoF[6]H CP-OLEDs, with or without HTLs (Fig. The measured CP-EL retains the same handedness irrespective of the device architecture (preferentially left handed for the P enantiomer of monoF[6]H, gEL CONV = +0.10 and gEL INV = +0.13). This is the same handedness as optically pumped CP photoluminescence (gPL = +0.24). Together with the fact that the handedness of CPL cannot be inverted by changing the current flow direction, this suggests NCPE for F8BT:(P)-monoF[6]H CP-OLEDs without an HTL (Supplementary Figs. Consistent with this assignment, measurements on semitransparent devices show the same magnitude and handedness of CP-EL regardless of the measurement direction: gEL,ITO = +0.12 and gEL,Al = +0.10 in the case of conventional semitransparent devices (Supplementary Fig. 33) and gEL,ITO = +0.12 and gEL,Au = +0.10 for inverted semitransparent devices (Supplementary Fig. This is also consistent with the handedness of CPL emitted from thin films on optical excitation at different geometries. The small differences in gEL between conventional and inverted devices can be attributed to optical losses due to the attenuation of CPL propagating inside the device (Supplementary Note 3). For the same reason, the sum of gEL obtained from semitransparent devices does not match the gEL measured in devices with highly reflective electrodes. a, Device architecture (ITO/PEDOT:PSS/F8BT:monoF[6]H/TPBi/Ca/Al) and CP-EL characteristic of conventional HTL-free CP-OLEDs. b, Device architecture (ITO/ZnO/PEIE/F8BT:monoF[6]H/MoOx/Au) and CP-EL characteristic of HTL-free inverted CP-OLEDs. c, Device architecture (ITO/PEDOT:PSS/TFB/F8BT:monoF[6]H/TPBi/Ca/Al) and CP-EL characteristics of conventional devices, with TFB as the HTL d, Device architecture (ITO/ZnO/PEIE/F8BT:monoF[6]H/TCTA/MoOx/Au) and CP-EL characteristics of conventional devices, with TCTA as the HTL. The thickness of all emissive layers is 160 nm. IL, intensity of left-handed (LH) light; IR, intensity of right-handed (RH) light. In conventional devices with an HTL, the handedness of CP-EL inverts compared with devices without the HTL (Fig. 2c) and right-handed emission is preferred for the P enantiomer of monoF[6]H (gEL CONV = –0.40). By contrast, left-handed emission is still preferred for the P enantiomer of monoF[6]H when used in inverted devices with an HTL (gEL INV = +0.38; Fig. As such, the handedness of CPL is clearly inverted by changing the current flow direction for F8BT:(P)-monoF[6]H CP-OLEDs with an HTL, suggesting ACPE. Measurements using semitransparent devices showed that conventional devices with a TFB HTL gave opposite-handed CP-EL as a function of emission direction (gEL,ITO = −0.22 and gEL,Al = +0.23; Supplementary Fig. Inverting the device structure when using TCTA as an HTL also results in opposite-handed CP-EL, but with the absolute handedness inverted relative to conventional devices with an HTL (gEL,ITO = 0.33 and gEL,Au = −0.27; Supplementary Fig. The emission of CPL is non-reciprocal: CPL emitted towards the top electrode inverts handedness, partially combining with CPL emitted towards the ITO and increasing the measured gEL. This overcomes the theoretical limit of devices with only NCPE (gEL > gPL). 37 evaluates the impact of HTL thickness in inverted devices, where the HTL is evaporated rather than solution processed. It is clear that the HTL thickness does not impact |gEL|, but it does impact the EL lineshape due to cavity effects22. In summary, the electrical control of CP-EL handedness is observed simply through the addition or omission of an HTL within the device stack. Devices without an HTL show NCPE, whereas devices with an HTL show APCE. In all the devices (that is, with ACPE and NCPE), gEL is independent of current density, which is consistent with previous results in chiral polymer blends (Supplementary Fig. This impressive performance is accompanied by the first demonstration of electrical control over the emission mechanism (Supplementary Table 5), achieved through careful material and device design. We suggest that the measured CP-EL from CP-OLEDs using our chiral blend materials can be described with contributions from both ACPE and NCPE18: According to our previous theory and quantitative analysis of ACPE18, directional charge transport through chiral emissive layers induces OAM in CP-OLEDs23,24. We ignore the SAM of charge carriers because of negligible spin–orbital coupling in these organic polymers. There is a finite OAM transfer (\({\Delta l=l}_{{\rm{e}}}-{l}_{{\rm{h}}}\)) from charge carriers to photons during the electron–hole recombination process (Supplementary Note 2) and a given photon with finite Δl exhibits opposite handedness if it is emitted in opposite directions. Reversing the current directionality changes the sign of le (lh) due to the orbital-momentum locking in chiral materials18,23, and therefore, conventional and inverted devices emit CP-EL with opposite handedness. Through this mechanism, ACPE rationalizes the device-architecture-dependent CP-EL handedness in our devices that contain an HTL. When an HTL is used in conventional (HTL = TFB) or inverted (HTL = TCTA) devices, charge transport in the emissive layer is unbalanced towards electrons, pinning the RZ of the device to the HTL interface near the anode. As such, in the presence of an HTL, electrons travel through the chiral active layer and gain OAM le, whereas holes travel through the achiral HTL and develop zero OAM (lh = 0). In devices without an HTL, charge injection and the transport of electrons and holes are balanced, and the RZ is in the centre of the active layer (Fig. Inversion of CP-EL handedness is not observed when switching the current direction or as a function of emission direction. The handedness of emitted light is the same as optically pumped CPL emission. In this balanced scenario, electrons and holes carry similar finite OAM (\({l}_{{\rm{e}}}\approx {l}_{{\rm{h}}}\)); therefore, \(\Delta l\approx 0\), which suppresses ACPE. As such, only NCPE is observed. Thus, through adjusting the position of the RZ within a CP-OLED device using these materials, it is possible to tune the contribution of ACPE versus NCPE and, in turn, control the handedness of CP-EL for a fixed enantiomer of the emissive material. a, NCPE occurs when there is balanced charge transport in the device through the omission of an HTL. b,c, ACPE occurs when charge transport is unbalanced towards electron transport through the addition of an HTL. For conventional devices, the RZ is close to the front electrode (b), whereas for inverted devices, the RZ is close to the back electrode (c). The control of ACPE and NCPE in OLEDs requires a careful design of materials and devices. Materials capable of ACPE demonstrate a helical electron distribution such that the movement of charge carriers induces an OAM that is transferred to SAM in the emitted light. In the achiral luminescent polymer–chiral additive blend system considered here, twisted polymer backbones generate the appropriate topological electronic landscape. The additive should be designed such that it can induce a chiral phase and maintain the charge carrier properties of the achiral luminescent polymer. ACPE requires unbalanced, ambipolar charge transport in the emissive layer, which confines the RZ to a narrow region and pins the RZ position close to one of the electrodes. The RZ position is best controlled when the energy levels of the transport layer match those of the emissive layer, and requires selecting transport layers with a tuneable charge mobility. We have demonstrated that CP-EL handedness from a chiral polymer LED can be controlled electrically by tuning the charge carrier mobility and the RZ position, without changing the handedness or composition of the active layer. CPL emission in CP-OLEDs does not simply originate from the same mechanism of optically excited transitions (NCPE). The origin of this effect appears to be adjusting the interplay of NCPE versus ACPE through adjusting the balance of induced OAM in electrons versus holes. On a practical level, this strategy provides a simple means to access CP-OLEDs with high levels of CP-EL, where left- or right-handed CP-EL is available from a single emissive enantiomer. Notable advances in the asymmetric synthesis of helicenes25 means that it is increasingly simple to synthesize a single-enantiomer chiral additive. If a single-enantiomer additive can be used to generate CP-OLEDs of either CP-EL handedness, this new CP-OLED strategy avoids the need for laborious and poorly scalable separation of racemic chiral additives into their separate enantiomers for CP-OLED usage. Thus, this approach provides a paradigm shift in the realization of organic CP-LEDs with controllable SAM information, which should enable many of the promised applications, from display technologies to quantum information systems. On a fundamental level, this study provides further evidence of the rich physics at play in chiral optoelectronic systems and suggests a means to control induced OAM in charge carriers within chiral semiconducting materials. Combining ACPE with other strategies, such as multilayer OLEDs or chiral microcavities26, could also enable new ways to boost the dissymmetric response of CP-LEDs. There continues to be high interest in the interplay of chirality, charge and spin, including through the expanding studies concerning chiral-induced spin selectivity20, and this study provides further materials and device frameworks for the characterization of these phenomena. Aza[6]H was synthesized according to a previous report14, and monoF[6]H according to the synthesis procedure in the Supplementary Information and separated using preparative chiral high-performance liquid chromatography. F8BT and TFB were provided by Cambridge Display Technology. 2,2′,2″-(1,3,5-Benzinetriyl)-tris(1-phenyl-1H-benzimidazole) (TPBi) and TCTA were provided by Ossila. All the substrates were cleaned via subsequent sonication for 10 min in deionized water, acetone and isopropanol, followed by 7 min of oxygen-plasma treatment. F8BT and (P)-aza[6]helicene (aza[6]H) or (P)-4-fluoro[6]helicene (monoF[6]H) were dissolved in toluene at 30 mg ml−1 and blended to form a solution with 10% of chiral dopant. F8BT–chiral additive active layer blends were spin coated at different spin speeds (1,500–3,000 rpm) for 1 min to control the active-layer film thickness and annealed for 10 min in a nitrogen atmosphere (<0.1-ppm H2O and <0.1-ppm O2) at 140 °C. The film thickness was measured using a Dektak profilometer. Hole-only devices for charge carrier mobilities were built on prepatterned ITO glass (120 nm, 10 Ω). Poly(3,4-ethylenedioxythiophene) polystyrene sulfonate (PEDOT:PSS) was spin coated to achieve a 50-nm-thick film. If used, TFB was spin coated at 1,000 rpm for 1 min and annealed for crosslinking. The F8BT–chiral inducer active layer blend was then spin coated following the same procedure for thin films. If included, TCTA was evaporated with a thickness of 25 nm. Electron-only devices for charge carrier mobilities were built on Al-prepatterned glass. The F8BT–chiral inducer active layer blend was then spin coated at 2,000 rpm following the same procedure for thin films. Further, if included in the device, 10 nm of TPBi was thermally evaporated. Conventional OLEDs were fabricated on prepatterned ITO glass (120 nm, 10 Ω). PEDOT:PSS was spin coated to achieve a 50-nm-thick film. If included in the device structure, TFB was spin coated at 1,000 rpm for 1 min and annealed at 180 °C for crosslinking. The active layer was then spin coated at 2,000 rpm following the same procedure for thin films. Inverted OLEDs were fabricated on prepatterned ITO glass (120 nm, 10 Ω). ZnO and ethoxylated polyethylenimine (PEIE) were spin coated following the already reported procedure. The active layer was then spin coated at 2,000 rpm following the same procedure for thin films. CD measurements of thin films were carried out with a Chirascan (Applied Photophysics) spectrophotometer. CP photoluminescence was measured with a custom-built setup, using an achromatic quarter-wave plate (Thorlabs) and a linear polarizer with a 180° configuration (transmittance). Atomic force microscopy was measured using an Asylum MFP 3D device. Current–voltage–luminance (J–V–L) characterizations were carried out using a Keithley 2400 device and a Konica Minolta LS-160 luminance meter, with the emission profile assumed to be Lambertian. The CP-EL spectra were measured using a super-achromatic quarter-wave plate (Thorlabs SAQWP05M-700) and a visible wire grid linear polarizer (Thorlabs WP25M-VIS) placed in front of an Ocean Optics FL-4000 charge-coupled spectrophotometer, as previously reported28, and driving the device at a constant current (1 mA). The setup is automated via the integration of a rotation mount (Thorlabs ELL14K), ensuring the acquisition of left- and right-handed CP emission spectra in a time interval of <0.5 s to avoid potential gEL artefacts due to device degradation. This methodology ensures that the quarter-wave plate and polarizer are always correctly positioned, preventing unintentional birefringence or polarization artefacts that would otherwise need angle-dependent measurements and depolarization corrections. The dissymmetry factor (gEL) was calculated using the equation gEL = 2(IL − IR)/(IL + IR) at the maximum of the emission, with IL and IR being the left-handed and right-handed emission intensities, respectively. Any additional information can be obtained from the corresponding authors upon reasonable request. Source data are provided with this paper. The added value of small-molecule chirality in technological applications. Furlan, F. et al. Chiral materials and mechanisms for circularly polarized light-emitting diodes. Zhang, D. W., Li, M. & Chen, C. F. Recent advances in circularly polarized electroluminescence based on organic light-emitting diodes. & Takanashi, K. Future perspectives for spintronic devices. Controlling the helicity of light by electrical magnetization switching. Albano, G., Pescitelli, G. & Di Bari, L. Chiroptical properties in thin films of π-conjugated systems. Pathways to increase the dissymmetry in the interaction of chiral light and chiral molecules. Wade, J. et al. Natural optical activity as the origin of the large chiroptical properties in π-conjugated polymer thin films. Di Nuzzo, D. et al. High circular polarization of electroluminescence achieved via self-assembly of a light-emitting chiral conjugated polymer into multidomain cholesteric films. Jung, J. H., Lee, D. M., Kim, J. H. & Yu, C. J. Circularly polarized electroluminescence by controlling the emission zone in a twisted mesogenic conjugate polymer. Effects of molecular ordering on circularly polarized emission from a twisted mesogenic conjugated polymer. Simultaneous emission of orthogonal handedness in circular polarization from a single luminophore. Albano, G., Pescitelli, G. & Di Bari, L. Reciprocal and non-reciprocal chiroptical features in thin films of organic dyes. Yang, Y., Da Costa, R. C., Smilgies, D. M., Campbell, A. J. Induction of circularly polarized electroluminescence from an achiral light-emitting polymer via a chiral small-molecule dopant. Control of circularly polarized electroluminescence in induced twist structure of conjugate polymer. Wan, L. et al. Inverting the handedness of circularly polarized luminescence from light-emitting polymers using film thickness. Wan, L., Wade, J., Wang, X., Campbell, A. J. Engineering the sign of circularly polarized emission in achiral polymer—chiral small molecule blends as a function of blend ratio. & Yan, B. Anomalous circularly polarized light emission in organic light-emitting diodes caused by orbital–momentum locking. Zhang, Y. J., Oka, T., Suzuki, R., Ye, J. T. & Iwasa, Y. Electrically switchable chiral light-emitting transistor. & Waldeck, D. H. Chiral molecules and the electron spin. Laidlaw, B. et al. On the factors influencing the chiroptical response of conjugated polymer thin films. Wang, M., Lin, J., Hsiao, Y. C., Liu, X. Investigating underlying mechanism in spectral narrowing phenomenon induced by microcavity in organic light emitting diodes. & Yan, B. Chirality-driven topological electronic structure of DNA-like materials. Gingras, M., Félix, G. & Peresutti, R. One hundred years of helicene chemistry. Part 2: stereoselective syntheses and chiral separations of carbohelicenes. Chen, T. L. et al. A 2D chiral microcavity based on apparent circular dichroism. Hussain, R., Jávorfi, T. & Siligardi, G. CD imaging at high spatial resolution at diamond B23 beamline: evolution and applications. Enhancing circularly polarized electroluminescence through energy transfer within a chiral polymer host. This work was supported by European Union's Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement number 859752 (M.J.F., J.W., N.G., I.G.S. This material was based on work supported by the Air Force Office of Scientific Research under award number FA9550-23-1-0633 (M.J.F. This work was supported by a grant from the Weizmann–UK Joint Research Program, award number 723018 (M.J.F. MMP measurements were carried out at Diamond Light Source B23 beamline (SM32632 beamtime; J.W., M.J.F. This work has received funding from the Czech Science Foundation (registration number 24-10787S, I.G.S.) acknowledges the National Science Foundation through the Penn State Materials Research Science and Engineering Center (MRSEC) DMR 2011839. We would like to thank L. Bednárová for measuring the optical and chiroptical spectra of monoF[6]H. Department of Chemistry and Molecular Sciences Research Hub, Imperial College London, London, UK Francesco Furlan, Davide Nodari, Nicola Gasparini & Matthew J. Fuchter Francesco Furlan, Jessica Wade, Nicola Gasparini & Matthew J. Fuchter Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Prague, Czech Republic Michal Šámal, Jiří Rybáček & Irena G. Stará Marta Di Girolamo & Jessica Wade Diamond Light Source, Harwell Science and Innovation Campus, Didcot, UK Department of Chemistry, Chemistry Research Laboratory, University of Oxford, Oxford, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to the fabrication and measurements of semitransparent devices. designed and synthesized the new helicene additive. carried out the thin-film measurements at beamline B32, with A.T. and G.S. measured and analysed the atomic force microscopy data. provided guidance for the experiment design and data analysis. supervised the project, assisted by J.W. The other authors declare no competing interests. Nature Photonics thanks Weibo Gao and Haipeng Lu for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Source data for the circular dichroism and J–V–L characteristics of conventional and inverted devices. Source data of the CP-EL for Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Furlan, F., Šámal, M., Rybáček, J. et al. Electrical control of photon spin angular momentum in organic electroluminescent materials. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02928-x'>Antimicrobial peptide delivery to lung as peptibody mRNA in anti-inflammatory lipids treats multidrug-resistant bacterial pneumonia</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-26 10:06:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article The efficacy of antimicrobial peptides (AMPs) is limited by challenges of delivery and potency. We enhance AMP performance in the lung by converting AMPs to a peptibody format that fuses AMPs with fragment crystallizable domains to activate innate immunity and cathelin domains for infection-responsive activation, with their mRNA constructs delivered by anti-inflammatory lipid nanoparticles. The highest-scoring design outperforms antibiotic therapy approved by the US Food and Drug Administration in multidrug-resistant pneumonia models, eradicating representative MDR bacteria while mitigating inflammation. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are available within the article and its Supplementary Information. Source data are provided with this paper. Metlay, J. P. et al. Diagnosis and treatment of adults with community-acquired pneumonia. An official clinical practice guideline of the American Thoracic Society and Infectious Diseases Society of America. Care Med. Pragman, A. A., Berger, J. P. & Williams, B. J. Understanding persistent bacterial lung infections: clinical implications informed by the biology of the microbiota and biofilms. Mettelman, R. C., Allen, E. K. & Thomas, P. G. Mucosal immune responses to infection and vaccination in the respiratory tract. Hewitt, R. J. & Lloyd, C. M. Regulation of immune responses by the airway epithelial cell landscape. Cookson, W. O., Cox, M. J. & Moffatt, M. F. New opportunities for managing acute and chronic lung infections. Parmanik, A. et al. Current treatment strategies against multidrug-resistant bacteria: a review. Grief, S. N. & Loza, J. K. Guidelines for the evaluation and treatment of pneumonia. Kellum, J. A. et al. Understanding the inflammatory cytokine response in pneumonia and sepsis: results of the genetic and inflammatory markers of sepsis (GenIMS) study. Kumar, V. Pulmonary innate immune response determines the outcome of inflammation during pneumonia and sepsis-associated acute lung injury. Mookherjee, N., Anderson, M. A., Haagsman, H. P. & Davidson, D. J. Antimicrobial host defence peptides: functions and clinical potential. Drug Discov. Dijksteel, G. S., Ulrich, M. M., Middelkoop, E. & Boekema, B. K. Lessons learned from clinical trials using antimicrobial peptides (AMPs). Izadpanah, A. & Gallo, R. L. Antimicrobial peptides. & Gallo, R. L. Antimicrobial peptides. Di, Y. P., Kuhn, J. M. & Mangoni, M. L. Lung antimicrobial proteins and peptides: from host defense to therapeutic strategies. Mahlapuu, M., Björn, C. & Ekblom, J. Antimicrobial peptides as therapeutic agents: opportunities and challenges. Hou, X. et al. Vitamin lipid nanoparticles enable adoptive macrophage transfer for the treatment of multidrug-resistant bacterial sepsis. Steinstraesser, L. et al. Skin electroporation of a plasmid encoding hCAP-18/LL-37 host defense peptide promotes wound healing. Pyzik, M., Kozicky, L. K., Gandhi, A. K. & Blumberg, R. S. The therapeutic age of the neonatal Fc receptor. Czajkowsky, D. M., Hu, J., Shao, Z. & Pleass, R. J. Fc-fusion proteins: new developments and future perspectives. Levin, D., Golding, B., Strome, S. E. & Sauna, Z. E. Fc fusion as a platform technology: potential for modulating immunogenicity. Dürr, U. H., Sudheendra, U. S. & Ramamoorthy, A. LL-37, the only human member of the cathelicidin family of antimicrobial peptides. Sørensen, O. E. et al. Human cathelicidin, hCAP-18, is processed to the antimicrobial peptide LL-37 by extracellular cleavage with proteinase 3. Park, M. D., Silvin, A., Ginhoux, F. & Merad, M. Macrophages in health and disease. Rosales, C. & Uribe-Querol, E. Phagocytosis: a fundamental process in immunity. Nimmerjahn, F. & Ravetch, J. V. Fcγ receptors as regulators of immune responses. Uribe-Querol, E. & Rosales, C. Phagocytosis: our current understanding of a universal biological process. Herrero-Cervera, A., Soehnlein, O. & Kenne, E. Neutrophils in chronic inflammatory diseases. Mittal, M., Siddiqui, M. R., Tran, K., Reddy, S. P. & Malik, A. B. Reactive oxygen species in inflammation and tissue injury. Saxton, R. A., Glassman, C. R. & Garcia, K. C. Emerging principles of cytokine pharmacology and therapeutics. Drug Discov. Huang, X., He, D., Pan, Z., Luo, G. & Deng, J. Reactive-oxygen-species-scavenging nanomaterials for resolving inflammation. Wang, S. et al. Accelerating diabetic wound healing by ROS-scavenging lipid nanoparticle–mRNA formulation. Zhang, F. et al. Nanoparticle-modified microrobots for in vivo antibiotic delivery to treat acute bacterial pneumonia. Maciá, M. D. et al. Efficacy and potential for resistance selection of antipseudomonal treatments in a mouse model of lung infection by hypermutable Pseudomonas aeruginosa. Agents Chemother. Lee, A. S. et al. Methicillin-resistant Staphylococcus aureus. Zeng, C. et al. Leveraging mRNA sequences and nanoparticles to deliver SARS-CoV-2 antigens in vivo. Chen, R. et al. Engineering circular RNA for enhanced protein production. Wang, C. et al. Blood–brain-barrier-crossing lipid nanoparticles for mRNA delivery to the central nervous system. Kim, M. et al. Dual SORT LNPs for multi-organ base editing. Devkota, S. P., Onah, C., Joshi, P. R., Adhikari, S. & Baral, P. Optimized method for higher yield of alveolar macrophage isolation for ex vivo studies. Xue, Y. et al. LNP–RNA-mediated antigen presentation leverages SARS-CoV-2-specific immunity for cancer treatment. Chen, S. et al. Nanotechnology-based mRNA vaccines. Mukherjee, A. et al. Engineered mutant α-ENaC subunit mRNA delivered by lipid nanoparticles reduces amiloride currents in cystic fibrosis-based cell and mice models. Witten, J. et al. Artificial intelligence-guided design of lipid nanoparticles for pulmonary gene therapy. Kim, J. et al. Engineering lipid nanoparticles for enhanced intracellular delivery of mrna through inhalation. Sun, Y. et al. In vivo editing of lung stem cells for durable gene correction in mice. Facchini, M., De Fino, I., Riva, C. & Bragonzi, A. Long term chronic Pseudomonas aeruginosa airway infection in mice. van den Berg, E. et al. Role of the Fas/FasL system in a model of RSV infection in mechanically ventilated mice. Lung Cell. Download references acknowledges support from the Maximizing Investigators' Research Award (R35GM144117) from the National Institute of General Medical Sciences and institutional funds from the Icahn School of Medicine at Mount Sinai. We thank the Biorepository and Pathology CoRE Laboratory at the Icahn School of Medicine at Mount Sinai for their support. Cryo-electron microscopy was performed at the Center for Electron Microscopy and Analysis at The Ohio State University. We are grateful to E. Purisic and J. Dai for sharing the vibratome and providing instructions. Certain figures were created with BioRender.com. These authors contributed equally: Yonger Xue, Xucheng Hou, Siyu Wang. Icahn Genomics Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Yonger Xue, Xucheng Hou, Siyu Wang, Yichen Zhong, Diana D. Kang, Chang Wang, Haoyuan Li, Changyue Yu, Zhengwei Liu, Meng Tian, Dinglingge Cao, Ya Ying Zheng & Yizhou Dong Marc and Jennifer Lipschultz Precision Immunology Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Yonger Xue, Xucheng Hou, Siyu Wang, Yichen Zhong, Diana D. Kang, Chang Wang, Haoyuan Li, Changyue Yu, Zhengwei Liu, Meng Tian, Dinglingge Cao, Ya Ying Zheng, Pauline Hamon, Miriam Merad & Yizhou Dong Department of Immunology and Immunotherapy, Icahn School of Medicine at Mount Sinai, New York, NY, USA Yonger Xue, Xucheng Hou, Siyu Wang, Yichen Zhong, Diana D. Kang, Chang Wang, Haoyuan Li, Changyue Yu, Zhengwei Liu, Meng Tian, Dinglingge Cao, Ya Ying Zheng, Pauline Hamon, Miriam Merad & Yizhou Dong Division of Pharmaceutics & Pharmacology, College of Pharmacy, The Ohio State University, Columbus, OH, USA Yuebao Zhang & Diana D. Kang Center for Electron Microscopy and Analysis, The Ohio State University, Columbus, OH, USA Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Pauline Hamon, Miriam Merad & Yizhou Dong Division of Hematology/Oncology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Miriam Merad Human Immune Monitoring Center, Icahn School of Medicine at Mount Sinai, New York, NY, USA Center for Thoracic Oncology, Icahn School of Medicine at Mount Sinai, New York, NY, USA Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Yizhou Dong Department of Oncological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA Yizhou Dong Biomedical Engineering and Imaging Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Yizhou Dong Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the work, performed the experiments, analyzed the data and wrote the paper. contributed to the lipid synthesis. contributed to the mRNA synthesis. and D.C. contributed to the animal studies. contributed to the cryo-electron microscopy imaging. contributed to the collection of human lung tissues. conceptualized and supervised the project and wrote the paper. The final paper was edited and approved by all authors. Correspondence to Xucheng Hou or Yizhou Dong. is a cofounder and holds equity in Immunanoengineering Therapeutics. The other authors declare no competing interests. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Relative luminescence intensity of TS LNP-FLuc mRNA in MLG cells. Intensity was normalized to SM-102 LNP group. b, Relative luminescence intensity of TS LNP-FLuc mRNA in A549 cells. Intensity was normalized to SM-102 LNP group. c, Luminescence intensity in the lungs of mice intratracheally injected with TS LNPs-FLuc mRNA. d, Representative IVIS images of lung tissues from c. e, Luminescence intensity in the lungs of mice following IT injection of SM-102 LNP-FLuc mRNA, TS41 LNP-FLuc mRNA, or TS41S LNP-FLuc mRNA. f, Representative IVIS images of lung tissues from e. g, Schematic illustration showing that delivery of Cre recombinase mRNA induces tdTomato expression in Ai14 reporter mice through Cre-mediated recombination. h, Flow cytometry analysis of tdTomato expression in various lung cell types following IT injection of SM-102 LNP-Cre mRNA or TS41S LNP-Cre mRNA. All data are from n = 3 biologically independent samples and are presented as mean values ± s.d. Statistical significance was determined by one-way ANOVA followed by Dunnett's multiple comparisons test. a, Table for two rounds of TS41 LNP optimization. b, Orthogonal assay to evaluate the impact trends of individual lipid component in TS41 LNP formulation at four levels in MLG cells. c, Luminescence intensity fold changes of the two rounds of optimization in MLG cells. d, Orthogonal assay to evaluate the impact trends of individual lipid component in TS41 LNP formulation at four levels in A549 cells. e, Luminescence intensity fold changes of the two rounds of optimization in A549 cells. f, Hydrodynamic diameter and PDI of TS41S LNP. PDI, polydispersity index. g, Zeta potential and encapsulation efficiency of TS41S LNP. h, Representative Cryo-TEM image of TS41S LNP. Scale bar = 100 nm. Data in b-g are from n = 3 biologically independent samples. Data in b and d are plotted as floating bar charts, where each bar represents the range of luminescence intensity from a start value (minimum intensity) to an end value (maximum intensity), with a line indicating the mean intensity. Data in c and e-g are presented as mean values ± SD. Statistical significance was determined by one-way ANOVA followed by Dunnett's multiple comparison test. a, Flow cytometry analysis of ROS levels in neutrophils, alveolar macrophages and monocyte-derived macrophages in bronchoalveolar lavage fluid (BALF) from pneumonia lung treated with different LNPs. b, Percentage and cell number of infiltrating neutrophils among total immune cells in BALF. c, Expression level of inducible nitric oxide synthase (iNOS) in neutrophils. d, e, f, g, Expression level of IL-1β (d), IL-6 (e), TNF-α (f), and IFN-γ (g) in BALF. All data are from n = 5 biologically independent samples and are presented as mean values ± SD. Statistical significance was determined by one-way ANOVA followed by Dunnett's multiple comparison test. a, Treatment scheme for chronic pneumonia models induced by S. aureus infection or co-infection of S. aureus and P. aeruginosa. b, Relative body weight for each group in chronic pneumonia model of S. aureus infection. c, Quantification of bacteria load in the lung tissues of each group in chronic pneumonia model of S. aureus infection. d, Relative body weight for each group in chronic pneumonia model of S. aureus and P. aeruginosa co-infection. e, Quantification of bacteria load in the lung tissues of each group in chronic pneumonia model of S. aureus and P. aeruginosa co-infection. All data are from n = 5 (b and c) or 6 (d and e) biologically independent samples and are presented as mean values ± SD. Statistical significance was determined by one-way ANOVA followed by Dunnett's multiple comparison test. Panel a created using BioRender.com. Unprocessed western blots and statistical source data. Unprocessed western blots and statistical source data. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Xue, Y., Hou, X., Wang, S. et al. Antimicrobial peptide delivery to lung as peptibody mRNA in anti-inflammatory lipids treats multidrug-resistant bacterial pneumonia. Accepted: 27 October 2025 Version of record: 26 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1087-0156 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            