
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-26</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/animals/a70081203/walking-shark-reproduction/'>Scientists Found a Creature That's Breaking the Rules of Reproduction</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The epaulette shark (Hemiscyllium ocellatum), which lives off the coast of northern Australia and New Guinea, is a strange creature. For one, it's the youngest known shark in the world, with genetic evidence suggesting their emergence only nine million years ago. But a new study, published in the journal Biology Open, makes an even more puzzling discovery about this fascinating shallow-sea creature. Energy is king in the natural world, and a large driver of evolution is developing strategies to reduce energy expenditure by any means necessary. “We expected that when sharks make this complex egg, their energy use would shoot up.” The assumption is that the reproduction cycle of sharks—along with other chondrichthyan fishes such as rays, skates, and chimaeras—are energy-intensive due to small litter sizes and their long gestation periods. To get more data on the energy requirements for shark reproduction, Rummer and her team recorded metabolic and physiological changes in five epaulette sharks during oviparous (egg-laying) reproduction by tracking oxygen uptake and measuring reproductive hormones. This good news expands beyond just epaulette sharks themselves. When animals are under stress, reproduction is often impacted as a means to conserve energy—and there's no greater stressor than climate change. Luckily, the amazing “walking shark” doesn't require an immense energy spike for reproduction, suggesting that the shark could continue exploring Australia's coral reefs even as its waters warm. “Under environmental stress, many species will choose between survival and reproduction, but the epaulette shark might be able to continue to produce eggs, even under such stressors,” Carolyn Wheeler, the lead author and a recent JCU Ph.D. graduate, said in a press statement. We May Have the Evolutionary Timeline All Wrong A Colossal Squid Might Have Shown Its Secret Face These Birds Are Migrating in the Wrong Direction This Creature Exists at the Fringes of Life A Creature With 24 Eyes Can See Everywhere At Once</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a70062673/roman-tablets-well/'>These Ancient Roman Tablets Were Supposed to be Unreadable. Scholars Just Read Them.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The tablets were purposefully thrown in a well to obscure what was written on them, but there were still impressions on the wood scholars painstakingly deciphered. Roman officials in modern-day Belgium once tossed old wooden frames used for wax writing tablets into a well to make sure nobody could read what was written. Now, a group of scholars has figured out why. The team published its findings online, showing off their discovery of traces of government documents and offering historical context that was previously unknown. These were likely official government documents purposefully throw in into the well to destroy them. Think of it as an ancient paper shredder. These appeared to have pupils' writing exercises on them. “The wood, with its natural grain, was completely dried out,” Markus Scholz, a specialist in provincial Roman archaeology and ancient inscriptions who led the team in deciphering the writing, said in a translated statement from Goethe University Frankfurt. “Distinguishing between grooves that formed part of a letter and those caused by cracks, damage, or the drying process itself was extremely challenging.” The remnants reveal evidence that high political offices were held in the Roman provinces, including mentions of a senior magistrate known as a decemvir. It included paleography, philology, onomastics, wax analysis, wood species identification, and script visualization by multi-light reflectance imaging. When discovered nearly 100 years ago, scholars thought the wood pieces were planks or boxes. It wasn't until much later that they were identified as wooden frames of Roman wax tablets. So think about that the next time you're deleting a text or shredding some paperwork. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A New Theory Says Gravity May Come From Entropy Fire Patrol Finds Ancient Face Carving in Boulder Your Teeth Could Be the Key to a Longer Life Scientists Found Evidence of Unknown Life in Rocks The First Law of Thermodynamics Has Been Rewritten</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a70095694/golden-armor-tang-dynasty/'>Archaeologists Found a Legendary Set of Golden Armor Many Thought Didn't Exist</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The ancient Chinese legend that Tang dynasty rulers donned gold-plated armor in battle had long been mere myth—until now. Researchers not only discovered a suit of gilded armor in a Tibetan plateau tomb of a Tuyuhun king, but they've now restored the find to digitally recreate what was likely its original look. Archaeologists first discovered the roughly 1,200-year-old suspected suit of armor in a Qinghai Province tomb in 2018, according to the Chinese Academy of Social Sciences (CASS), but they didn't quite know just what they had discovered. 1 Tomb in Dulan Country were first believed to be bronze armor plates, many rectangular with a semicircular lower edge. “We adopted a strategy of disassembling the whole into parts and reassembling the parts into a whole, conducting layered cleaning, extraction, and protection while meticulously cataloging each armor plate,” Guo Zhengchen, a conservationist working on the armor, said during a press conference according to the South China Morning Post. China's Tang dynasty celebrated its illustrious golden armor in art and poetry. For example a notable work by Tang poet Wang Changling read: “We will not leave the desert till we beat the foe, although in war our golden armor be outworn 100 times,” according to the SCMP. Confirmed as the tomb of a Tuyuhun king, the armor of gold was likely a precious possession, helping bring to life another Tang dynasty line of poetry: “Like golden scale on water, sunrays strike our coats of mail.” The region was once home to a powerful kingdom on China's western frontier. “These technologies have provided solutions to the challenges of artifact degradation, allowing for more accurate restoration, interactive displays, and global accessibility to cultural heritage,” archaeologists previously wrote about the process, according to CASS. CASS said in a statement that the “approach enabled the scientific restoration of several sets of precious cultural relics that were severely damaged and structurally complex,” including the only physical example of a Tang dynasty golden armor. As evidenced by the unique coat of armor, the Tuyuhun king seemed to enjoy a little panache—he also had quite a fanciful covering for his horse. Additionally found in the tomb was a lacquered tray designed to hold grapes featuring intricate Tang dynasty gold-and-silver inlay designs. All these finds help breathe new life into pieces of Chinese history once thought to only be the stuff of legend. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A New Theory Says Gravity May Come From Entropy Your Teeth Could Be the Key to a Longer Life</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00246-z'>As we breach 1.5 °C, we must replace temperature limits with clean-energy targets</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 11:21:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Kwesi A. Quagraine is an Advanced Study Program postdoctoral fellow at the National Science Foundation National Center for Atmospheric Research, Boulder, Colorado, USA. Mark Lynas is scientific adviser to the Office of the Secretary-General, Climate Vulnerable Forum, Accra, Ghana, and head of policy at WePlanet International, Ghent, Belgium. Erle C. Ellis is a professor in the Department of Geography and Environmental Systems, University of Maryland, Baltimore County, Baltimore, Maryland, USA. Keeping to the Paris target now looks impossible by any realistic measure. Instead, it demands an urgent reframing of how climate progress is measured and mobilized. Although emissions are still rising and global actions on climate change are slow, a lot of progress has been made. To move forwards, climate scientists and policymakers must first accept that the Paris 1.5 °C target has outlived its usefulness. Although initially valuable as a unifying focus for international efforts to increase mitigation, continuing to emphasize a failed temperature target might produce more harm than good. Forecasts suggest that Earth is likely to exceed the 1.5 °C threshold around 2028, for instance (see go.nature.com/4pf95x6). Confirmation would therefore not come until a decade after the fact. Environmental treaties are paralysed — here's how we can do better Environmental treaties are paralysed — here's how we can do better Such a target, requiring years of expert interpretation to assess, could never have great salience for decision makers. And it challenges public understanding because, by definition, no person experiences global average temperatures. Moving to a higher number when 1.5 °C is crossed, such as 1.6 °C or 1.7 °C, would only make climate target setting seem arbitrary and unrooted in scientific evidence. For example, injecting vast amounts of sulfate aerosols into the stratosphere through ‘solar radiation modification' could be one of the few remaining ways to bring 1.5 °C back into reach. But it might also alter precipitation patterns or result in a burst of warming if the programme were suddenly halted. Discussions have begun around the concept of ‘overshoot', in which the planet's average temperature would be brought back to 1.5 °C by 2100 after a period exceeding that level4. But its achievement would be even harder to pin down. While in overshoot, no one can be sure whether it is permanent or temporary. Any approach based on projections to 2100 is unlikely to inspire public interest or political action because the goal is so far off. It is hubristic to think that researchers can predict accurately how emerging technologies such as artificial intelligence will affect the climate, or how Earth's climate system itself will respond to unprecedented conditions. For example, in 2023 at the UN Conference of the Parties (COP) 28 climate summit in Dubai, United Arab Emirates, countries called for a tripling of renewable-energy capacity globally by 2030 and the transition away from fossil fuels to reach net zero by 2050. Whereas delegates at COP30 in Belém, Brazil, in 2025, struggled to agree to phase out fossil fuels, support for accelerating clean energy is almost universal. To fulfil this mandate, the world needs one clear number with which to measure climate progress during a transition that ends the use of fossil fuels. Building on a concept initially proposed by Bloomberg New Energy Finance founder Michael Liebreich (see go.nature.com/3zr5y1), we define it as the growth rate in clean-energy generation minus the growth rate in total energy demand for a given time interval. This metric emphasizes that clean-energy supply must expand faster than overall energy demand for decarbonization. By contrast, simply measuring clean energy share is insufficient, because fossil fuels might also rise overall to meet extra demand. Clouds of steam emanate from a geothermal plant in Iceland.Credit: Getty Chasing the number zero will never motivate politicians concerned about economic development. Framing decarbonization as a story of building clean-energy industries and jobs is more appealing. The metric points to policies that are politically feasible, economically desirable and already advancing rapidly technologically, including solar, wind, batteries, geothermal, hydro and nuclear — without picking winners. The power of pursuing this approach is evident in recent energy trends. Between 2018 and 2020, the annual percentage growth of clean-energy generation rose sharply, driven by rapid additions of solar, wind and storage capacities. Total global energy demand also grew, but at a slower rate, implying that growth in clean-energy production began eating into the share of fossil fuels. If this clean-energy shift can be increased further, the world would see a peak and then a steady decline in fossil-fuel use and related emissions. This means that clean-energy generation must continue its current rapid growth, rising several percentage points faster than total energy demand each year. Although challenging, trends in solar, wind and storage deployment suggest that this acceleration is achievable — although it will require expanded manufacturing capacity, grids built to integrate renewable technologies, continued reductions in battery costs and political will. To achieve global decarbonization by 2050, policymakers need short-term milestones. We suggest they might set global targets of clean-energy shifts in five-year intervals — like rungs on a ladder, each climbing closer to a safe climate. It should be noted that the intervals are non-linear in terms of added capacity: because clean-energy shift is a percentage-growth metric, the higher rungs reflect increasingly rapid rates of clean-energy expansion, rather than a constant yearly addition. Lower numbers would mean fossil fuels staying longer as part of the energy mix. Climate change is devastating mining of minerals needed to fight it Climate change is devastating mining of minerals needed to fight it These intervals can be aligned with the five-year timelines used by the UN Framework Convention on Climate Change (UNFCCC) policy process, such as major COP agreements, reviews of ‘nationally determined contributions' and ‘global stocktakes' of progress towards the Paris goals for emissions reductions. The clean-energy shift metric also respects long-established UN principles of equity and accountability, including the concept of ‘common but differentiated responsibilities and respective capabilities' in climate policy. Individual countries can use the metric to track their own progress. And several major emitters already include clean-energy targets in their Paris commitments. China has more than doubled its wind and solar capacity over the past three years. The European Union currently has about one-quarter of its energy consumption supplied by renewable sources, which it has mandated should rise to 42.5% within five years. And the clean-energy shift excludes emissions from sources other than fossil fuels, such as greenhouse gases resulting from deforestation, soil cultivation or wildfires. But the climate damage caused by fossil fuels is unique in its scale — comprising 90% of the carbon dioxide problem (38 of 42 gigatonnes in 2024)7. Rescuing 1.5 ºC: New Evidence on Highest Possible Ambition to Deliver the Paris Agreement (Climate Analytics & PIK, 2025). China's Climate Transition: Outlook 2025 (Centre for Research on Energy and Clean Air, 2025). Bhandary, R. R., Maris, N. & Zucker-Marques, M. V20 Debt Review: An Account of Debt in the Vulnerable Group of Twenty (Climate Vulnerable Forum and Global Policy Development Centre, 2025). Climate change is devastating mining of minerals needed to fight it Environmental treaties are paralysed — here's how we can do better Chinese nuclear fusion reactor pushes plasma past crucial limit: what happens next Science in 2050: the future breakthroughs that will shape our world — and beyond Application closing date: 01.02.2026 Join a place where ambitious science thrives Human T... The University of Hong Kong Vice-Presidents and Pro-Vice-Chancellors (various positions) The University of Hong Kong (HKU), a leading research-led ... Climate change is devastating mining of minerals needed to fight it Environmental treaties are paralysed — here's how we can do better An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41563-025-02464-8'>Achieving high tensile strength and ductility in refractory alloys by tuning electronic structure</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 11:01:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Materials (2026)Cite this article The energy efficiency of heat engines (gas and steam turbines) for electricity production and propulsion is determined by the Carnot cycle and scales with operating temperature. Commercial nickel- and cobalt-based superalloys melt near 1,500 °C and rapidly lose mechanical strength beyond 1,000 °C. Refractory metals melt well above 2,000 °C but have inherent manufacturability challenges that are barriers to adoption, such as high ductile-to-brittle transition temperatures. Using density functional theory-guided design, we demonstrate tailored local lattice distortions that promote phase-stable, non-equiatomic refractory concentrated solid solutions with both high ductility and strength. We exemplify this for single-phase, body-centred cubic Nb4Ta4V3Ti that exhibits castability, excellent room-temperature tensile yield strength (∼1 GPa) and ductility (approaching 20% uniform strain), and exceptional high-temperature tensile strength (500 MPa at 1,000 °C). These findings illustrate a path for designing materials that hold great potential for advancing next-generation technologies such as Generation IV fission reactors, first-generation fusion-plasma reactors, and more efficient gas turbines for electricity generation and propulsion. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All relevant data are provided in the Supplementary Information. Raw data can be made available upon request. No unique codes were developed for the purpose of this work; all codes used are documented in publications cited in this paper. Computational output results are provided in the Supplementary Information. Sohn, S. S. et al. Ultrastrong medium-entropy single-phase alloys designed via severe lattice distortion. Google Scholar Zhao, Y. Y. & Nieh, T. G. Correlation between lattice distortion and friction stress in Ni-based equiatomic alloys. Google Scholar He, Q. On lattice distortion in high entropy alloys. Google Scholar Cantor, B., Chang, I. T. H., Knight, P. & Vincent, A. J. B. Microstructural development in equiatomic multicomponent alloys. Google Scholar Gludovatz, B. et al. A fracture-resistant high-entropy alloy for cryogenic applications. Google Scholar Li, Z., Pradeep, K. G., Deng, Y., Raabe, D. & Tasan, C. C. Metastable high-entropy dual-phase alloys overcome the strength–ductility trade-off. Google Scholar Wei, S. et al. Natural-mixing guided design of refractory high-entropy alloys with as-cast tensile ductility. Google Scholar Shi, P. et al. Hierarchical crack buffering triples ductility in eutectic herringbone high-entropy alloys. Google Scholar Google Scholar The conflicts between strength and toughness. Google Scholar Senkov, O. N., Wilks, G. B., Scott, J. M. & Miracle, D. B. Mechanical properties of Nb25Mo25Ta25W25 and V20Nb20Mo20Ta20W20 refractory high entropy alloys. Google Scholar Cook, D. H. et al. Kink bands promote exceptional fracture resistance in a NbTaTiHf refractory medium-entropy alloy. Google Scholar Miracle, D. B., Senkov, O. N., Frey, C., Rao, S. & Pollock, T. M. Strength vs temperature for refractory complex concentrated alloys (RCCAs): a critical comparison with refractory BCC elements and dilute alloys. Acta Mater. Google Scholar El-Genk, M. S. & Tournier, J.-M. A review of refractory metal alloys and mechanically alloyed-oxide dispersion strengthened steels for space nuclear power systems. Google Scholar & DiStefano, J. R. The role of oxygen uptake and scale formation on the embrittlement of vanadium alloys. Google Scholar Lei, Z. et al. Enhanced strength and ductility in a high-entropy alloy via ordered oxygen complexes. Google Scholar Jiao, M. et al. Manipulating the ordered oxygen complexes to achieve high strength and ductility in medium-entropy alloys. Google Scholar Wang, S. et al. The effects of interstitial oxygen on mechanical properties of TiZrNb medium-entropy alloy over a wide temperature range. Google Scholar Smith, T. M. et al. A 3D printable alloy designed for extreme environments. Nature https://doi.org/10.1038/s41586-023-05893-0 (2023). Wilkinson, W. D. Properties of Refractory Metals (Gordon & Breach, 1969). Singh, P. et al. A ductility metric for refractory-based multi-principal-element alloys. Acta Mater. Google Scholar Pugh, S. F. XCII. Relations between the elastic moduli and the plastic properties of polycrystalline pure metals. Google Scholar Rice, J. R. Dislocation nucleation from a crack tip: an analysis based on the Peierls concept. Google Scholar Winter, I. S., De Jong, M., Montoya, J., Rothchild, E. & Chrzan, D. C. Intrinsic ductility of random substitutional alloys from nonlinear elasticity theory. Google Scholar Hu, Y.-J., Sundar, A., Ogata, S. & Qi, L. Screening of generalized stacking fault energies, surface energies and intrinsic ductile potency of refractory multicomponent alloys. Acta Mater. Google Scholar Johnson, D. D., Singh, P., Smirnov, A. V. & Argibay, N. Universal maximum strength of solid metals and alloys. Google Scholar Wang, F. et al. Multiplicity of dislocation pathways in a refractory multiprincipal element alloy. Google Scholar Borges, P. P. P. O., Ritchie, R. O. & Asta, M. Electronic descriptors for dislocation deformation behavior and intrinsic ductility in bcc high-entropy alloys. Google Scholar An, Z. et al. A novel HfNbTaTiV high-entropy alloy of superior mechanical properties designed on the principle of maximum lattice distortion. Google Scholar Lee, C. et al. Lattice-distortion-enhanced yield strength in a refractory high-entropy alloy. Nguyen, V. T. et al. Compositional design of strong and ductile (tensile) Ti–Zr–Nb–Ta medium entropy alloys (MEAs) using the atomic mismatch approach. Lai, W. et al. Design of BCC refractory multi-principal element alloys with superior mechanical properties. Yin, X. et al. Effects of Nb addition on Charpy impact properties of TiVTa refractory high-entropy alloy. Acta Metall. Beniwal, D. & Ray, P. K. Learning phase selection and assemblages in high-entropy alloys through a stochastic ensemble-averaging model. Singh, P. et al. Theory-guided design of duplex-phase multi-principal-element alloys. Acta Mater. Singh, P., Smirnov, A. V. & Johnson, D. D. Atomic short-range order and incipient long-range order in high-entropy alloys. Weinberger, C. R., Tucker, G. J. & Foiles, S. M. Peierls potential of screw dislocations in bcc transition metals: predictions from density functional theory. Trinkle, D. R. & Woodward, C. The chemistry of deformation: how solutes soften pure metals. Gröger, R. & Vitek, V. Multiscale modeling of plastic deformation of molybdenum and tungsten. Effects of temperature and plastic strain rate. Acta Mater. Vitek, V. Core structure of screw dislocations in body-centred cubic metals: relation to symmetry and interatomic bonding. Yang, D., Chen, B., Li, S., Ding, X. Effect of local lattice distortion on the core structure of edge dislocation in NbMoTaW multi-principal element alloys and the subsystems. Natarajan, A. R. & Van Der Ven, A. Linking electronic structure calculations to generalized stacking fault energies in multicomponent alloys. Singh, P., Smirnov, A. V. & Johnson, D. D. Ta–Nb–Mo–W refractory high-entropy alloys: anomalous ordering behavior and its intriguing electronic origin. & Pack, J. D. Special points for Brillouin-zone integrations. Zhang, R. et al. Short-range order and its impact on the CrCoNi medium-entropy alloy. Wu, Y. et al. Short-range ordering and its effects on mechanical properties of high-entropy alloys. Wang, S. D. et al. Chemical short-range ordering and its strengthening effect in refractory high-entropy alloys. Wang, L. et al. Tailoring planar slip to achieve pure metal-like ductility in body-centred-cubic multi-principal element alloys. Juan, C.-C. et al. Simultaneously increasing the strength and ductility of a refractory high-entropy alloy via grain refining. He, Q., Yoshida, S. & Tsuji, N. Characteristic strengthening mechanisms in body-centered cubic refractory high/medium entropy alloys. Wang, Z. et al. Solving oxygen embrittlement of refractory high-entropy alloy via grain boundary engineering. Frank, F. C. & Stroh, A. N. On the theory of kinking. Yamasaki, M., Hagihara, K., Inoue, S., Hadorn, J. P. & Kawamura, Y. Crystallographic classification of kink bands in an extruded Mg–Zn–Y alloy using intragranular misorientation axis analysis. Acta Mater. Wang, S., Wu, M., Shu, D. & Sun, B. Kinking in a refractory TiZrHfNb0.7 medium-entropy alloy. Wang, S. et al. Mechanical instability and tensile properties of TiZrHfNbTa high entropy alloy at cryogenic temperatures. Acta Mater. Hyde, J. M., Marquis, E. A., Wilford, K. B. A sensitivity analysis of the maximum separation method for the characterisation of solute clusters. Stephenson, L. T., Moody, M. P., Liddicoat, P. V. & Ringer, S. P. New techniques for the analysis of fine-scaled clustering phenomena within atom probe tomography (APT) data. Chen, S. et al. Short-range ordering alters the dislocation nucleation and propagation in refractory high-entropy alloys. Maresca, F. & Curtin, W. A. Mechanistic origin of high strength in refractory BCC high entropy alloys up to 1900 K. Acta Mater. Kubilay, R. E., Ghafarollahi, A., Maresca, F. & Curtin, W. A. High energy barriers for edge dislocation motion in body-centered cubic high entropy alloys. Lee, C. et al. Strength can be controlled by edge dislocations in refractory high-entropy alloys. Fleischer, R. L. Effects of non-uniformities on the hardening of crystals. Acta Metall. Labusch, R. Statistical theories of solid solution hardening. Acta Metall. Senkov, O. N., Scott, J. M., Senkova, S. V., Miracle, D. B. & Woodward, C. F. Microstructure and room temperature properties of a high-entropy TaNbHfZrTi alloy. & Rivera-Díaz-del-Castillo, P. E. J. Modelling solid solution hardening in high entropy alloys. Acta Mater. Coury, F. G., Kaufman, M. & Clarke, A. J. Solid-solution strengthening in refractory high entropy alloys. Acta Mater. Maresca, F. & Curtin, W. A. Theory of screw dislocation strengthening in random BCC alloys from dilute to ‘high-entropy' alloys. Acta Mater. Chen, B. et al. Trade-off between local chemical order and lattice distortion in affecting dislocation motion in NbTiZr multi-principal element alloys. Acta Mater. Antillon, E., Woodward, C., Rao, S. I. Chemical short range order strengthening in BCC complex concentrated alloys. Acta Mater. Baruffi, C., Maresca, F. & Curtin, W. A. Screw vs. edge dislocation strengthening in body-centered-cubic high entropy alloys and implications for guided alloy design. MRS Commun. Lee, C. et al. Temperature dependence of elastic and plastic deformation behavior of a refractory high-entropy alloy. Pollock, T. M. Alloy design for aircraft engines. Tabor, D. A simple theory of static and dynamic hardness. Hsu, W.-C., Shen, T.-E., Liang, Y.-C., Yeh, J.-W. & Tsai, C.-W. In situ analysis of the Portevin–Le Chatelier effect from low to high-entropy alloy in equal HfNbTaTiZr system. Acta Mater. Wang, S.-P., Ma, E. & Xu, J. New ternary equi-atomic refractory medium-entropy alloys with tensile ductility: hafnium versus titanium into NbTa-based solution. Pang, J. et al. Simultaneous enhancement of strength and ductility of body-centered cubic TiZrNb multi-principal element alloys via boron-doping. Schmidt, F. F. & Ogden, H. R. The Engineering Properties of Columbium and Columbium Alloys (Defense Metals Information Center, Battelle Memorial Institute, 1963). Jalali, S. I. A., Patullo, M. S., Philips, N. & Hemker, K. J. Capturing the ultrahigh temperature response of materials with sub-scale tensile testing. Leonard, K. J., Busby, J. T., Hoelzer, D. T. & Zinkle, S. J. Nb-base FS-85 alloy as a candidate structural material for space reactor applications: effects of thermal aging. Molybdenum: Properties (Plansee, 2025). Blöchl, P. E., Jepsen, O. & Andersen, O. K. Improved tetrahedron method for Brillouin-zone integrations. Kresse, G. & Joubert, D. From ultrasoft pseudopotentials to the projector augmented-wave method. Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Singh, R., Sharma, A., Singh, P., Balasubramanian, G. & Johnson, D. D. Accelerating computational modeling and design of high-entropy alloys. Zhang, X.-C., Cao, S., Zhang, L.-J., Yang, R. & Hu, Q.-M. Unstable stacking fault energy and Peierls stress for evaluating slip system competition in body-centered cubic metals. Singh, P., Smirnov, A. V., Alam, A. & Johnson, D. D. First-principles prediction of incipient order in arbitrary high-entropy alloys: exemplified in Ti0.25CrFeNiAlx. Acta Mater. Ouyang, G. et al. Design of refractory multi-principal-element alloys for high-temperature applications. Download references were supported by the Laboratory Directed Research and Development (LDRD) programme at Ames National Laboratory (theory and computation) and the US Department of Energy (DOE) Energy Efficiency and Renewable Energy (EERE) office (experiments). The majority of the research was performed at the Ames National Laboratory, which is operated for the US DOE by the Iowa State University under contract DE-AC02-07CH11358. were supported by the US Air Force Office of Scientific Research (AFOSR) under grant FA9550-21-1-0304. were also supported by a DOE FES RENEW award DE-SC0024541. This work was in part performed at the Center for Integrated Nanotechnologies, an Office of Science User Facility operated for the US DOE Office of Science by Los Alamos National Laboratory and Sandia National Laboratories. Sandia National Laboratories is a multiprogramme laboratory managed and operated by National Technology and Engineering Solutions of Sandia, LLC, a wholly owned subsidiary of Honeywell International, Inc., for the US DOE's National Nuclear Security Administration under contract DE-NA0003525. Financial support for ultrahigh-temperature tensile testing was provided by D. Shifler and the US Office of Naval Research (ONR) under grant number N000141812180. This paper describes objective technical results and analysis. Any subjective views or opinions that might be expressed in the paper do not necessarily represent the views of the US DOE or the US Government. Present address: X-ray Science Division, Argonne National Laboratory, Lemont, IL, USA Division of Materials Science and Engineering, Ames National Laboratory, Ames, IA, USA Hailong Huang, Prashant Singh, Duane D. Johnson, Gaoyuan Ouyang, Luke Gaydos, Trevor Riedemann, Andrew B. Kustas, Ryan T. Ott & Nicolas Argibay Department of Materials Science and Engineering, Iowa State University, Ames, IA, USA Duane D. Johnson, Luke Gaydos & Nicolas Argibay Metallurgical and Materials Engineering, Indian Institute of Technology Ropar, Rupnagar, India Dishant Beniwal & Pratik K. Ray Department of Materials Science and Engineering, University of North Texas, Denton, TX, USA Tirthesh Ingale, Vishal Soni, Rajarshi Banerjee & Thomas W. Scharf Material, Physical, and Chemical Science Center, Sandia National Laboratories, Albuquerque, NM, USA Ping Lu, Frank W. DelRio & Andrew B. Kustas RTX Technology Research Center, East Hartford, CT, USA John A. Sharon & Ryan Deacon Department of Mechanical Engineering, Johns Hopkins University, Baltimore, MD, USA A. Jalali, Michael Patullo & Kevin J. Hemker Department of Materials Science & Engineering, Johns Hopkins University, Baltimore, MD, USA Sharon Park & Kevin J. Hemker Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to all aspects of the experimental work, data analysis, and interpretation, writing, and editing. contributed to all aspects of the theory and computation work, conception, data analysis and interpretation, writing, and editing. contributed to data curation and visualization. performed SEM and EBSD analysis of deformed microstructures and APT, related data analysis and interpretation, and writing. contributed to materials synthesis and processing, mechanical testing, data analysis, and writing. contributed to materials synthesis and processing, mechanical testing, and analysis, and writing. performed TEM analysis and contributed to data interpretation and writing. performed nanoindentation testing and contributed to writing. contributed to data analysis and interpretation, editing and project management. performed tensile sample preparation for high-temperature testing, contributed to analysis and interpretation, and writing. performed high-temperature tensile testing and contributed to data analysis, interpretation and writing. contributed to conception, analysis and interpretation, writing, editing and project management. Correspondence to Nicolas Argibay. The authors declare no competing interests. Nature Materials thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. DFT results spreadsheet. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Huang, H., Singh, P., Johnson, D.D. et al. Achieving high tensile strength and ductility in refractory alloys by tuning electronic structure. Download citation Received: 23 April 2025 Accepted: 05 December 2025 Published: 26 January 2026 Version of record: 26 January 2026 DOI: https://doi.org/10.1038/s41563-025-02464-8 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/nasa-readies-for-artemis-ii-mission-ai-powered-speech-gives-stroke-patients/'>NASA gears up for Artemis II, scientists use AI to give stroke patients their voice back, and cave art discovery breaks record</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>NASA gears up for a historic lunar flyby, AI gives stroke patients hope, and researchers discover the oldest known cave art What's on the road to the launch of NASA's Artemis II, how scientists are using artificial intelligence to help stroke patients speak, and what an Indonesian cave art discovery says about early human migration By Kendra Pierre-Louis, Lee Billings, Sushmita Pathak, Kelso Harper & Alex Sugiura You're listening to our weekly science news roundup. First, we have an update on humans going back to the moon. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. To learn more about it we chatted with Lee Billings, SciAm's senior desk editor for physical science. It was just meant to show that the key hardware components work properly, that they can get into space and go to the moon and come back. And now we are doing that with humans on board, so it's much higher stakes. Artemis II is not going to land on the moon. It's not even going to orbit the moon. And so that means there will be some interesting spaceflight records being broken. The four astronauts that will be on board this mission, looping around the moon, will be instrumented and sensored all up. They'll have all kinds of biometrics coming off them. And we'll be doing that to have a better idea of how humans respond to the deep-space environment for notional future missions that will go to the surface of the moon, with Artemis III and onwards. And, and so where we are now with Artemis II is that on January 17 it rolled out, in this very prestigious and ceremonial proceeding, rolled out from the Vehicle Assembly Building, this giant building that NASA has at Kennedy Space Center in Cape Canaveral, Florida. And it was loaded on this giant diesel-electric tractor, essentially, to slowly, glacially go at about a pace of a mile an hour or so, from, the Vehicle Assembly Building to the actual launch pad where it will launch from. The next big step is going to be something called the “wet dress rehearsal”; this is slated for February 2. And hopefully, we won't see any leaks because if we do see a bunch of leaks, then it'll probably delay what is supposed to be the onset of the launch window, which is February 6. And each month there's about five days that the moon and the Earth are aligned so that, you know, we can pull this launch off, so if they miss that kind of five-day window in early February, well, we're looking to March. And why do we wanna go back to the moon? Well, a big part of it is geopolitics. And a big question now is whether or not we can beat them back to the moon, even though we already did it more than 50 years ago. There are extremely interesting scientific questions as well. For instance, the places that people wanna go on the moon for this new generation of missions, it's largely concentrated around the lunar south pole, which is where we know there are deposits of water ice and other types of volatiles. This is a very special region that has near-constant illumination from the sun but also permanently shadowed craters. And you can do that there and be totally shielded from the Earth-based radio interference you would otherwise receive that would scuttle all your measurements. Pierre-Louis: For more on NASA's lunar mission go to ScientificAmerican.com. The key, researchers say, is a new device called Revoice. You see, roughly half of all patients who experience a stroke also develop dysarthria, which weakens the muscles used for speech and breath control. The condition can cause slurred, slow or strained speech. The good news is that with rehabilitation many patients regain their speech, but the process can take anywhere from months to years. Given that recovery is possible for patients, the scientists behind the new study wanted to help patients communicate faster than existing technologies that require letter-by-letter input. The Revoice device the scientists developed consists of a soft collar embedded with sensors that track throat movement and heart rate and provide that information to two AI agents. Both of these agents process the data using a large language model. The other then expands those words into full sentences by using the wearer's pulse to analyze their emotional state and detecting broader ambient conditions, including the weather and time of day. Combined, the system can anticipate what the person is trying to say and, with just two nods of their head, speak for them. If the results hold, Revoice could be a useful tool not only for stroke patients but also for those with other neurological conditions, including Parkinson's disease. In other news about communication a study published on Wednesday in the journal Nature reveals the oldest cave art reportedly ever found. Previously, the oldest-known cave art were depictions of a pig and three humanlike figures thought to be over 51,000 years old. That art was found on the Indonesian island of Sulawesi. This new discovery was found on the same island but in a different cave. But the limestone caves of Sulawesi are easier to work with. A chemical analysis found that the stencil dated back some 67,800 years at least, making it roughly 15,000 years older than the previously discovered cave art. This discovery could help us pinpoint when humans first settled in Australia. Archaeologists suspect that humans migrated there through Indonesia but have been unable to determine the exact time frame. Franco Viviani, a physical anthropologist who was not involved in the new study, told SciAm that the findings also offer new insight into ancient societies, saying, quote, “They confirm what is known today: that art is positively correlated to critical thinking and creative problem-solving skills.” And speaking of creative problem solvers a new study on bats sheds some light on how these winged mammals get around. Every school-age kid at some point learns that bats are able to navigate in darkness using echolocation—that is, they send out a call and based on how the sound bounces back they can tell where an object is. But scientists have long wondered how bats navigate in object-rich environments. A single bat call will send back echoes ricocheting off multiple objects from various directions and distances. In complex situations scientists figured it wasn't really possible for a bat to analyze each individual echo, so they must be relying on an alternative strategy. To study this the research team constructed what they called a “bat accelerator” machine lined with 8,000 movable acoustic reflectors, or fake leaves. Tune in on Wednesday, when we'll dig into the nascent science of what foods make people stink. Record a voice memo on your phone or computer, and send it over to ScienceQuickly@sciam.com. Be sure to include your name and where you're from. Science Quickly is produced by me, Kendra Pierre-Louis, along with Fonda Mwangi, Sushmita Pathak and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. She has worked for Gimlet, Bloomberg News and Popular Science. He is author of a critically acclaimed book, Five Billion Years of Solitude: The Search for Life Among the Stars, which in 2014 won a Science Communication Award from the American Institute of Physics. In addition to his work for Scientific American, Billings's writing has appeared in the New York Times, the Wall Street Journal, the Boston Globe, Wired, New Scientist, Popular Science and many other publications. Billings joined Scientific American in 2014 and previously worked as a staff editor at SEED magazine. She previously worked at NPR and was a regular contributor to The World from PRX and The Christian Science Monitor. Kelso Harper is an award-winning senior multimedia editor at Scientific American. As a producer, editor and host, they work on short documentaries, social videos and Scientific American's podcast Science Quickly. They have a bachelor's degree in chemistry from Johns Hopkins University and a master's degree in science writing from the Massachusetts Institute of Technology. Previously, they worked with Wired, Science, Popular Mechanics, and MIT News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00245-0'>Technology is changing how we write — and how we think about writing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 10:55:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Chinese calligraphy can be traced back more than 3,000 years to the carving of characters onto animal bones for divination. Tools of the Scribe: How Writing Systems, Technology, and Human Factors Interact to Affect the Act of Writing Brian Roark et al. Springer (2025) The world's oldest writing system still in use, that of Chinese characters, dates from about 1200 bc. How the world's largest language family spread — and why others go extinct How the world's largest language family spread — and why others go extinct Yet today, millions of people who speak and read Chinese have forgotten how to compose many of the traditional characters by hand, relying instead on simpler phonetic and digital equivalents. This controversial trend in China, generally known as character amnesia, opens Tools of the Scribe, a stimulating and original, if technical, book by computational linguists Brian Roark, Richard Sproat and Su-Youn Yoon. Although their main focus is on technology, the authors draw on research in many fields, including psychology, neuroscience, linguistics, ergonomics, anthropology and speech and language pathology. Chinese characters illustrate this interaction between implement, medium, system and writer vividly. Notches were drilled and chiselled into the surfaces of turtle shells and ox scapulae, such that when heat was applied, cracks would appear. Then, during the latter half of the first millennium bc, the characters evolved into a more complex calligraphic form, painted in ink with a brush or pen on bamboo or paper: an art form still practised today. Today, almost 100,000 characters are listed in Unicode — the international character-encoding standard. The first commercial typewriter, designed for simple Western alphabets, was launched in 1874. By contrast, several attempts to commercialize a Chinese typewriter, beginning in 1919, were unsuccessful owing to the quantity and complexity of Chinese characters. Many university students use chatbots powered by large language models, such as OpenAI's ChatGPT, to help write assignments.Credit: Alamy In 1936, Mao Zedong, leader of the Chinese Communist Party and a devoted calligrapher, announced in his first interview with a Western journalist that “sooner or later, we believe, we will have to abandon the Chinese character altogether if we are to create a new social culture in which the masses fully participate”. After Mao's government came to power in 1949, it began to simplify characters by eliminating some variants and reducing the number of strokes in many of those remaining. Many people now rely on Pinyin to input Chinese characters into digital devices. Nevertheless, Chinese characters continue to dominate China's non-digital communication as a symbol of national identity. By contrast, computers can now suggest alternative spellings or synonyms, automatically correct errors and — with the advance of artificial intelligence — even write a whole text. In a 2024 survey, 96% of US university students reported having used OpenAI's ChatGPT chatbot for at least one project in that academic year, and 69% had done so for writing assignments (see go.nature.com/4pyzbyv). In another study, US secondary-school students used ChatGPT mainly to plan essays (S. Levine et al. J. Adolesc. However, they tended to accept or reject the tool's suggested edits as a whole, instead of examining each suggestion separately “to learn from their mistakes”. How the world's largest language family spread — and why others go extinct Shared and language-specific phonological processing in the human temporal lobe AI and nuclear energy feature strongly in agenda-setting technologies for 2026 Humanoid robots step up their game: how useful are the latest droids? AI tools boost individual scientists but could limit research as a whole Application closing date: 01.02.2026 Join a place where ambitious science thrives Human T... How the world's largest language family spread — and why others go extinct An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02241-y'>Effector–host interactome map links type III secretion systems in healthy gut microbiomes to immune modulation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 10:30:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Pseudomonadota (formerly Proteobacteria) are prevalent in the commensal human gut microbiota, but also include many pathogens that rely on secretion systems to support pathogenicity by injecting proteins into host cells. Here we show that 80% of Pseudomonadota from healthy gut microbiomes also have intact type III secretion systems (T3SS). Candidate effectors predicted by machine learning display sequence and structural features that are distinct from those of pathogen effectors. Towards a systems-level functional understanding, we experimentally constructed a protein–protein meta-interactome map between human proteins and commensal effectors. Network analyses uncovered that effector-targeted neighbourhoods are enriched for genetic variation linked to microbiome-associated conditions, including autoimmune and metabolic diseases. Functionally, commensal effectors can translocate into human cells and modulate NF-κB signalling and cytokine secretion in vitro. Our findings indicate that T3SS contribute to microorganism–host cohabitation and that effector–host protein interactions may represent an underappreciated route by which commensal gut microbiota influences health. Host-associated microbiota influences human health in complex, genotype-dependent ways. Especially the human gut microbiome, which is dominated by Firmicutes, Bacteroidetes and Pseudomonadota1 (formerly Proteobacteria2), can alter the risk of diverse conditions, including metabolic disorders, and autoimmune and neurodegenerative diseases3. However, the underlying molecular mechanisms are incompletely understood. Most studies have focused on metabolites, extracellular microorganism-associated molecular patterns, or community-level microbiome properties4. However, the role of intracellular bacteria–host protein interactions is largely unexplored. The potential impact of such interkingdom protein interactions is illustrated by viral proteins in asymptomatic or non-acute infections, which can influence cellular signalling and cell physiology and thereby contribute to complex diseases in a likely host-genetics dependent manner5,6,7,8. In bacteria, the type III secretion system (T3SS) is a well-characterized apparatus for delivering proteins into eukaryotic cells. The T3SS is a highly conserved ‘needle and syringe'-like machinery found in Pseusomonadota to inject bacterial proteins into host cells9. T3SS and their substrate effectors have been studied almost exclusively in human and plant pathogens such as Yersinia, Pseudomonas or Salmonella, for which protein translocation is a key pathogenic strategy. In the host, translocated effectors manipulate cellular processes including cytoskeletal dynamics or immune signalling to ensure bacterial survival and promote transmission9. Thus, the T3SS has traditionally been framed as a virulence determinant. Insights from plant and insect systems challenge this pathogen-centric view of the T3SS. Many commensal and beneficial microorganisms for these hosts deploy T3SS or analogous machinery to translocate proteins that promote symbiosis or fine-tune host immunity10,11,12. Effector–host protein–protein interaction maps in plants further reveal that both pathogenic and mutualistic microorganisms converge on central host signalling nodes, suggesting conserved principles by which injected proteins can modulate eukaryotic biology across diverse symbioses12,13,14. Despite the emerging broader conceptual importance of protein injection and cross-kingdom protein interactions in diverse microorganism–host systems, it is unknown whether analogous mechanisms operate in the healthy human gut. Here we investigate the distribution, diversity and host interactions of T3SS and their effectors in commensal Pseudomonadota from human guts. By integrating comparative genomics, structural prediction, functional assays and host protein interaction networks, we uncover an underappreciated layer of direct, protein-mediated communication between commensal microorganisms and the human host, with implications for immune modulation, microbial competition and complex disease biology. We first analysed reference genomes of Pseudomonadota strains from healthy gut and stool samples isolated, for example, by the human microbiome project15. Using EffectiveDB16, a widely used tool for secretion system identification, we detected complete T3SS in 44 of the 77 genomes (Supplementary Data 1). To expand the scope, we analysed genomes of 4,752 phylogenetically diverse strains of the human intestinal bacteria collection (HiBC)17, Broad Institute–OpenBiome Microbiome Library (BIO–ML)18 and Global Microbiome Conservancy (GMC)19. Of the 568 Pseudomonadota genomes, 449 (79%) have complete T3SS (Extended Data Fig. 1); similar proportions have T4SS (315) and T6SS (474), which may also inject effectors into host cells among other functions20 (Extended Data Fig. Together, 527 of the 568 Pseudomonatoda genomes (92%) have at least one host-directed secretion system. Because culturing can bias taxon representations, we also screened 16,179 high- and intermediate-quality Pseudomonadota metagenome-assembled genomes (MAGs)21,22,23, finding complete T3SS in 770 (5%) MAGs (Extended Data Fig. Among the T3SS-positive (T3SS+) species, 24 matched representatives in two cohorts of the Weizmann Institute of Science24: 59.4% of individuals in the Israeli and 47.1% in the Dutch cohort harboured potentially T3SS+ species in their gut microbiome at 0.80% and 0.48% relative abundance, respectively, with Escherichia coli being the most common. These observations indicate that T3SS+ strains are common members of the human gut microbiota and motivated our further investigation. a, Most abundant genera, species and genomes encoding complete T3SS among reference strains and MAGs from the human gut. b, Sequence similarity of 3,002 candidate commensal T3SS effectors with 1,195 effectors from pathogenic bacteria across alignment coverages (bottom left). Each dot represents a pairwise sequence comparison. Dot colour indicates effectors with significant and non-significant Jackhmmer results (inset legend) indicating homology to pathogen effectors. Marginal histograms display the aggregated distribution of alignment coverage (top) and aggregated sequence similarity (right), with colour indicating Jackhmmer outcome. c, Left, number of the structure clusters observed in FoldSeek analysis (red arrow) compared to random expectation for that group (homogeneous or mixed) in grey (exp. empirical P < 0.0001, n = 10,000, two-sided label permutation test). Middle, example structures for one cluster in the group; small networks are representative structure clusters for the group with an anchor structure in the centre and similar structures connected by links. Donut plots: proportion of proteins with origins indicated by colour in all clusters of (homogeneous or mixed) the structure-cluster group. d, Selection of 18 commensal Pseudomonadota strains for subsequent functional analyses. e, Injection of indicated effectors by wild-type and ΔsctV (T3SS-defective) Salmonella Typhimurium into HeLa cells detected by luminescence of reconstituted nano-luciferase (y axis). SipA is an assay control used as reference. Asterisks denote statistically significant differences between the wild-type and ΔsctV-negative strains (two-sided Wilcoxon test; five biological repeats with four technical repeats each). f, Injection of effectors from gut commensal Edwardsiella tarda into HeLa cells. SipA tested in wild-type and ΔsctV Salmonella Typhimurium were used as positive and negative controls, respectively (two-sided Wilcoxon test, *P < 0.05, **P < 0.001; NS, not significant; seven biological repeats with four technical repeats each). Raw data and precise P values for all panels are found in Supplementary Data 1, 3, 5, 6 and 10 as described in Supplementary Information. Boxplots (e,f) show the median (centre line) and the interquartile range (IQR, box), with whiskers extending to minimum and maximum values within 1.5× IQR. Using three complementary machine-learning models25,26,27, 3,002 effector candidates were confidently predicted in the T3SS+ reference genomes (hereafter: strain effectors) (Supplementary Data 4) and 182 in the 770 T3SS+ MAGs (meta-effectors) (Supplementary Data 4). To find weak similarities, we performed iterative jackhmmer29 searches against ~124 M non-redundant bacterial sequences from UniRef90. Yet, even with this sensitive approach, significant similarity to pathogen effectors was found only for 155 commensal strain effectors (~5%) and 42 meta-effectors (22.5%) (Fig. As effectors can be structurally related despite sequence divergence, we clustered AlphaFold30-predicted tertiary structures using FoldSeek31 for a structural comparison. Surprisingly, homogeneous clusters with effectors from only commensal strains or pathogens were highly overrepresented, whereas mixed clusters II and III, reflecting common structures of effectors from pathogens and commensal strains, were depleted (Fig. Meta-effectors clustered exclusively with strain effectors, albeit close to random expectation. All results were robust over varying FoldSeek parameters and when considering only vertebrate or human pathogens (Supplementary Data 6). Thus, candidate effectors in T3SS+ strains from healthy human guts markedly differ from pathogen effectors in both sequence and structure. We analysed all candidate effectors from the strains for annotated domains. Besides 860 proteins without any identifiable domain, among the most common finds were the diguanylate cyclase, GGDEF domain (PF00990) (58 effectors), and EAL domain (PF00563) (50 effectors), none of which was found in pathogen effectors (Supplementary Data 5). Cyclic diguanylate is a known second messenger in bacterial signal transduction, and the EAL domain is thought to be a diguanylate phosphodiesterase, thus opposing the effect of the cyclase32. Furthermore, we observed a PAS-fold domain (PF08447) in 32 effectors, which can function as a ligand-binding sensor32 and in some effectors co-occurs with a guanylate cyclase domain. As cyclic dinucleotides recently emerged as important immune regulators in all kingdoms of life33, the observation that two domains acting on the same second messenger occur at high frequency among the commensal effector candidates makes a role for this signalling molecule in interkingdom communication plausible. A key question is whether commensal candidate effectors get injected into human cells by T3SS. To enable functional studies, we cloned open reading frames (ORFs) encoding effectors from 18 bacterial strains (Fig. The generated human microbiome effector ORFeome v1 (HuMEOme_v1) contains 910 sequence-verified, full-length ORFs representing 746 strain effectors and 164 meta-effectors (Supplementary Data 7). Cloning failure mainly resulted from failed PCR amplification without indications of toxicity. Typhimurium (S. Typhimurium) as a model, we established a nano-luciferase-based injection assay34 fusing an 11-amino acid Nano-Luc HiBiT tag to the C terminus of candidate effectors expressed in bacteria. HeLa cells stably expressed the complementary LgBiT fragment, so that effector injection reconstitutes functional nano-luciferase. Specificity was ensured by inclusion of the T3SS-defective ΔsctV mutant for all tests. Benchmarking with six pathogen effectors demonstrated effective translocation of four. Among 97 tested candidate effectors from 11 strains, 32 were specifically and significantly injected (Fig. The slightly higher success rate for the positive controls probably reflects phylogenetic diversity and missing chaperones and cofactors. Thus, although some false effector identifications cannot be excluded, overall, our pipeline reliably identified bona fide T3SS substrate effectors from commensal strains in healthy human guts. Of the 11 strains with at least one T3SS-injectable effector, 6 could not be tested due to antibiotic resistance or transformation failure. Whereas the two E. coli strains yielded no signals, Citrobacter pasteurii and Phytobacter massiliensis showed occasional signals suggesting sporadic activation of the T3SS. By contrast, Edwardsiella tarda reproducibly and significantly injected three out of four tested effectors into HeLa cells (Fig. Notably, only one, Eta_3, was also positive in the Salmonella system, supporting the notion that missing cofactors may have caused false negatives in the first experiment. Overall, these data demonstrate that functional T3SS are present in strains from healthy human guts and can deliver identified effectors into human cells. Next, we explored possible functions of the commensal effectors by systematically mapping their physical interactions with host proteins using our multi-assay mapping pipeline35 (Extended Data Fig. Screening all cloned effectors against the full human ORFeome9.136 identified 1,067 interactions constituting the human-microbiome meta-interactome (HuMMI) main dataset (HuMMIMAIN) (Fig. Three repeat screens with 290 effectors and 1,440 human proteins yielded 39 interactions (HuMMIRPT) and indicated a sampling sensitivity of ~32% for the main screen (Fig. Lastly, we addressed how sequence similarity affects effector interaction profiles. We grouped effectors with ≥30% sequence identity (Supplementary Data 11) and experimentally tested them against the union of their interactors from the main screen. The resulting HuMMIHOM dataset contains 394 interactions, of which 181 are non-redundant. Altogether, HuMMI contains 1,255 unique verified interactions between 286 effectors and 426 human proteins (Fig. a, Verified human microbiome meta-interactome (HuMMI) map; coloured nodes indicate effectors from strains according to colour legend in f. Grey nodes represent human proteins; outer layer human proteins are targeted only by the nearest strain, central human proteins by effectors from multiple strains. b, Sampling sensitivity: saturation curve calculated from HuMMIRPT. Red dots represent the average of verified interactions found in any combination of indicated number of repeat screens; diamonds denote interaction counts per experiment over all sequential experiment combinations; error bars indicate standard deviation; black dots and line represent calculated saturation curve. e, Co-immunoprecipitation of MYC-tagged human proteins by Flag-tagged effectors or Flag–GFP as negative control. Input, cell lysates; green dots, successful co-immunoprecipitation; red dot, no co-immunoprecipitation; effector espG of Escherichia coli (Q7DB50) as positive control (one biological replicate). Molecular mass markers are given in kilodaltons. f, Most-targeted human proteins interacting with the indicated number of effectors from different strains. Colours represent strains according to indicated legend (full statistics in Supplementary Data 11). g, Most highly connected effectors interacting with the indicated number of human proteins (Supplementary Data 11). h, Observed number of effector-interacting human proteins compared to random expectation (two-sided permutation test, P < 0.0001; n = 10,000). i, Frequency distribution of human proteins targeted by effectors from the indicated number of different strains (red) compared to random expectation (two-sided permutation test, P = 0.004; n = 10,000). To experimentally assess data quality, we assembled a positive control set of 67 well-documented binary interactions of pathogen effectors with human proteins (bacterial human literature binary multiple (bhLit_BM-v1)) and a negative control set of random effector–human protein pairs (bacterial host random reference set (bhRRS-v1)) (Supplementary Data 12). Benchmarking our yeast-two-hybrid (Y2H) assay with these, alongside established human reference sets (hsPRS-v2 and hsRRS-v2)38, indicated an assay sensitivity of 13% and 17.5%, respectively, matching previous observations35,37,38 (Fig. No negative control scored positive, demonstrating the reliability of our system. Across thresholds, sets with bacterial proteins yielded fewer positive-scoring pairs than the human sets (Extended Data Fig. As this included the negative controls and no effector toxicity was observed, prokaryotic proteins appear harder to test in this assay system, reinforcing the need for tailored reference sets. Thus, the biophysical quality of HuMMI is on par with well-documented literature interactions. We aimed to demonstrate that interactions can occur within the human cell environment. We performed immunoprecipitation experiments in HEK293 cells (RRID: CVCL_0045, DSMZ) using Flag-tagged effectors and negative control Flag–GFP as baits and detecting the MYC-tagged human interaction partner by western blot. Of 32 pairs including 4 positive controls, 18 pairs and 3 controls yielded meaningful data, while 10 could not be evaluated due to unspecific binding of the human protein (3) or poor expression (7). Together, these results demonstrate that HuMMI contains biophysically reliable interactions that are robustly detectable in different assays and occur in human cellular environments. Importantly, functional effects may go in both directions and while in most cases effectors probably perturb the host cell, intracellular immune receptors may also recognize effectors to then initiate defence responses. The degree distribution of HuMMIMAIN shows that numerous human proteins interact with multiple effectors, often from different species (Fig. Random sampling demonstrates highly significant effector convergence on few host interactors (Fig. 2h), a phenomenon linked to the functional importance of the targeted host proteins as shown in plant–pathogen systems13. Moreover, interactions of human proteins with effectors from four bacterial strains are highly significant and unlikely to result from random processes (Fig. Thus, 60 human proteins are subject to effector convergence, highlighting their potential importance for microbiome–host interactions. To explore overlap with pathogen effectors, we extracted 265 high-quality binary interactions between 217 human proteins and 80 effectors from 17 pathogenic strains from IntAct39 (Supplementary Data 15). We found a numerically low, albeit significant, number of 12 human proteins targeted by both groups (P = 0.014, Fisher's exact test, odds ratio = 2.26), of which 3 are subject to convergence by commensal effectors (P = 0.067, Fisher's exact test, odds ratio = 3.37, Supplementary Data 11). Although limited by sample size, experimental differences and the non-systematic nature of the pathogen data, these findings support both overlap and lifestyle-dependent specificity in commensal and pathogenic effector targeting40. For instance, cluster 3 contains 7 effectors sharing >90% sequence similarity, yet their interaction profile similarities range from identical to complementary (Fig. Conversely, clustering effectors unrelated in sequence and structure by their pairwise interaction similarity in HuMMIMAIN identified substantial overlap outside homology clusters (Extended Data Fig. 3), suggesting that dissimilar effectors can have similar functions in the host. Thus, host effector function as measured by protein-interaction profiles is largely independent of overall sequence similarity. b, Scatterplot of mutual sequence similarity and Jaccard interaction similarity for all effector pairs in the indicated homology groups. The union of human proteins targeted by each effector pair is indicated by node size as denoted in the legend. Individual data in Supplementary Data 16. c, Yeast growth in one representative of four repeats of all effector–human interactions tested for homology cluster 3. d, Left, proportions of human-protein targets interacting with the same bacterial effector, grouped by interface similarity on the basis of the Jaccard index (JI) of target-contacting residues, categorized as: distinct (JI ≤ 0.1), overlapping (0.1 < JI < 0.6) and same (JI ≥ 0.6). Right, Mmo_5 interacts with example human proteins via the same interface (top, 66% overlap), whereas Pfa_4 uses distinct interfaces. e, Proportions of different effectors interacting with the same human protein, grouped by interface similarity as in d. Example interface models (right) show effector binding to the same (Pfa_9, Pse_2) or distinct (Pfa_9, Yen_2) interfaces on human TCF4. f, Count of domain–motif interfaces identified in HuMMIMAIN matching at least one stringency criterion (arrow) compared to random expectation (one-sided permutation test, P = 0.0137; n = 10,000). All data related to c–f are available in Supplementary Data 17. g, Results of holdup assay and comparison with Y2H results. Indicated PDZ domains of human proteins shown on y axis were tested against 10-amino-acid C-terminal peptides of the effectors indicated on top. Calculated dissociation constant (Kd) values as indicated. Overlap between holdup (HU) and Y2H on protein level is indicated by coloured frames. Precise P values and n for each test are shown in Supplementary Data 19. To gain structural insights and potential functional leads, we modelled effector–host protein interactions using AlphaFold-Multimer, obtaining predictions for 123 pairs (10%). For proteins with multiple interactors, we classified interfaces as ‘same' (≥60% shared contacting residues), ‘different' (<10% overlap) or ‘overlapping' (Fig. For instance, Mmo_5 binds to TPD52L1 and BORCS6 via the same interface, whereas Pfa_4 interacts with NOTO and LBX1 with different interfaces, possibly enabling simultaneous interactions with both (Fig. Analogously, Pse_2 and Pfa_9 bind to the same interface of TCF4, whereas Yen_2 targets a different part of the protein (Fig. Identical interface binding was more frequent on human proteins than on effectors, suggesting the importance of targeting functions linked to specific domains. Mapping the binding interfaces to domain annotations strengthened this hypothesis, as even effectors binding via different interfaces may target the same domain (for example, the DNA-binding domain of LBX1). More commonly, however, effectors with different interfaces bind to distinguishable parts of the host protein. Beyond large interfaces, many interactions are mediated by short linear motifs (SLiMs) in intrinsically disordered regions that bind to specific protein pocket-forming domains41. As AlphaFold often misses such interactions42, we used the orthogonal mimicINT approach to identify SLiM–domain interactions, which matches interaction pairs to known SLiM–domain templates43 (Fig. This identified putative interfaces for 54 HuMMIMAIN interactions involving bacterial host-like SLiMs binding to human domains (Supplementary Data 18), of which 51 passed at least one (Fig. P value) and 22 passed two stringency criteria (Extended Data Fig. Some of the matched motifs encompass phosphorylation sites that interact with kinases or phosphorylation-dependent binding domains such as SH2 domains. Conversely, although several commensal effectors encode predicted enzymatic domains (Supplementary Data 5), using an analogous approach we found no case in which these engage cognate substrate motifs on host proteins, and only a single effector-domain–SLiM match consistent with known docking specificity: the calcineurin-like phosphoesterase domain (PF00149) of Efe_1 and the canonical LxVP docking motif in VAC14. The largest group of 23 interactions involved PDZ domains in human proteins binding PDZ-binding motifs (PBM) in the C terminus of the bacterial interaction partners. PDZ domain-containing proteins commonly mediate functions important for microorganism–host interactions including cell–cell adhesion, protein trafficking and immune signalling44. To experimentally validate these interfaces, individual and tandem PDZ domains from 13 human proteins and C-terminal peptides from 16 interacting bacterial effectors were tested via the quantitative in vitro interaction holdup assay 45. In three instances, two PDZ domains arranged in tandem were required for the interaction, suggesting that some Y2H pairs might have been missed by the holdup method due to untested combinations. As for the predicted globular interfaces, for human proteins with multiple PDZ domains, different effectors often target different domains, demonstrating specificity and functional specialization (Fig. Thus, while overall effector sequence similarity does not correlate with interaction profiles, structural modelling showed that some effectors target similar interfaces and domains, suggesting shared functions, whereas others bind distinct domains pointing to functional specialization. We explored effector target functions using Gene Ontology (GO) enrichment (Fig. Among the most enriched functions was ‘response to muramyl dipeptide (MDP)', a bacterial cell wall-derived peptide. Central immune signalling pathways are also enriched, namely, the NF-κB and the stress-activated protein kinase and Jun-N-terminal kinase (SAPK/JNK) pathways. Remarkably, five significantly targeted convergence proteins belong to the NF-κB module (Extended Data Fig. 3), one of the evolutionarily oldest immune pathways in animals47. Using the Recon3D human genome-scale metabolic model48, we further found significant enrichment for metabolic enzymes among the human interactors (P = 0.0001, Fisher's exact test); however, beyond glycerophospholipid metabolism, no metabolic subsystem stood out (Supplementary Data 20). Finally, we compared commensal-targeted functions to those of pathogens (Supplementary Data 20). Some pathways were common to targets of both groups, such as ‘NF-κB signalling', whereas others are specific to commensals including ‘collagen biosynthesis' and ‘response to muramyl dipeptide'. a, Odds ratios of representative functional annotations enriched among effector-targeted human proteins (FDR < 0.05, Fisher's exact test with Bonferroni FDR correction). The lowest and highest odds ratios observed for the represented group are indicated by light shaded areas of bars. Black line indicates odds ratios for shown representative terms. White triangles indicate functions also enriched in pathogen targets. b, Genetic predisposition for traits and diseases enriched among human genes encoding effector-interacting proteins in HuRI (α = 0.05, Fisher's exact test; n = 349). The odds ratio in a and b estimates the effect size of significant function/trait (two-sided Fisher's exact test FDR < 0.05) and is calculated as the odds of function-annotated/trait-associated human genes encoding effector targets to function-unannotated/trait-unassociated human genes encoding effector targets in the target set, divided by the same ratio in the HuRI set (see Methods). c, Disease groups for which genetic predisposition proteins are enriched in network neighbourhoods of effectors of the indicated strains. Trait node size corresponds to number of significantly targeted traits in that group as indicated in the legend. Thickness of strain–group edges reflects the number of underlying significant effector–trait links (α < 0.01 and odds ratio > 3, two-sided Fisher's exact test). Precise P values and n for all tests are provided in Supplementary Data 23. We wondered whether perturbations by commensal effectors could influence non-infectious human diseases, starting our analysis at the network level. Genetic variants, but also viruses, contribute to complex diseases by often subtly altering intracellular networks and disease-relevant functions. We first explored whether commensal effectors target proteins that are genetically relevant for diseases and other traits. We used ‘causal genes' identified from genome-wide-association studies (GWAS) by the Open Targets initiative49 to identify the encoded ‘disease proteins', and unified traits by the Experimental Factor Ontology (EFO)50 (Fig. Effector targets are also associated with cancers and immune diseases, such as psoriasis, asthma, allergies and systemic lupus erythematosus, although none of these predominantly affect the gut. Given the abundance of immune-related measurement traits, it is possible that the effectors systemically perturb immune signalling and thereby contribute to lung and skin diseases. Alternatively, convergence proteins such as REL or TCF4 (Fig. 2f,h) may also be targeted by local microbiota in skin or lung tissues. Supporting this, 26% of HuMMI effectors are detectable in skin microbiome samples, suggesting that commensal effectors are shared across ecological niches (Supplementary Data 22). In addition to disease proteins being direct targets, we previously discovered that relevant genetic variation often resides in their protein interaction neighbourhood13,35. To explore these, we performed short random walks in the binary human reference interactome (HuRI)36 and defined ‘neighbourhood' as proteins visited significantly more often in HuRI than in degree-preserved randomly rewired control networks. In these neighbourhoods, we assessed disease-protein enrichment using Open Targets, aggregated nominally significant associations at the strain level, and summarized them by disease group (Fig. Most disease groups we found are known to be affected by the gut microbiome3. Among immune diseases, inflammatory bowel disease (IBD) was enriched (nominal P = 0.0008, Fisher's exact test), particularly Crohn's disease(nominal P = 8.5 × 10−5, Fisher's exact test) but not ulcerative colitis (Fig. As for direct targets, neighbourhoods also harboured susceptibility for skin and lung diseases such as asthma and psoriasis. Considering the microbiota's relevance for metabolic disorders, effector targeting of neighbourhoods affecting high- and low-density lipoprotein (HDL and LDL, respectively) cholesterol levels (nominal P = 0.006 and P = 0.008, respectively, Fisher's exact test) and several diabetes-related traits is notable (Supplementary Data 23). Together, these findings suggest that commensal effectors modulate host immune signalling and the local metabolic and structural microenvironment. As the targeted proteins and neighbourhoods are genetically associated with several diseases, modulation of their functions by effectors may contribute to disease aetiology. We sought to experimentally verify that commensal effectors perturb some of the identified pathways and functions. We focused on NF-κB signalling, which is central to many diseases and emerged repeatedly in our study. Using a dual-luciferase assay35 in HEK293 cells (RRID: CVCL_0045, DSMZ), 5 out of 26 commensal effectors significantly activated NF-κB activity in the absence of stimulation (Fig. 5a,b and Supplementary Data 24), while 3 effectors reduced NF-κB activity under strong TNF stimulation (Fig. Next, we assessed whether the effectors modulate NF-κB signalling in unstimulated Caco-2 cells and after pro-inflammatory stimulation (Extended Data Fig. Consistently, Cpa_12, an ABC domain-containing effector, reduced secretion of several cytokines with and without stimulation (Fig. Other effectors enhanced cytokine responses, particularly IL-6 and IL-8, only after Pam3CSK4 stimulation, but not after TNF or flagellin stimulation (Fig. Pam3CSK4 mimics TLR1/2 activation by triacylated lipopeptides abundant in Gram-positive Bacteroidetes, while flagellin mimics TLR5 activation by Gram-negative Pseudomonadota52. Thus, commensal effectors exert complex effects on intracellular immune signalling. a, Relative NF-κB transcription reporter activity in HEK293 cells expressing the indicated effectors at baseline conditions (no TNF) (Kruskal–Wallis test with Dunn's post hoc comparisons, *P < 0.05, **P < 0.01; n = 4 biological replicates). Boxes represent IQR, black line indicates the mean, whiskers indicate highest and lowest data point within 1.5× IQR. b, Summary of significant influence of effectors on normalized NF-κB transcriptional reporter activity at baseline conditions (−TNF) and after TNF stimulation (+TNF) (Kruskal–Wallis test with Dunn's post hoc comparisons, *P < 0.05, **P < 0.01; n = 4 biological replicates). c,d, Concentration of cytokines secreted by Caco-2 cells transfected with indicated effectors at basal conditions (unstim) or after stimulation with a proinflammatory cocktail (stim) (c) or with Pam3CSK4 (d). Numbers above brackets indicate P values calculated by Kruskal–Wallis test with Dunn's post hoc comparisons; n = 3 (c) and n =  5 (d) biological replicates. Boxes represent IQR, black line indicates the mean, circles indicate individual data points. e, Radial barplot showing fold change in prevalence of 122 bacterial effectors in metagenomes of individuals with Crohn's disease (CD; n = 504 patient samples, orange) or ulcerative colitis (UC; n = 302 patient samples, purple) relative to healthy controls (n = 334 samples). Fold changes were calculated using pseudo-counts from healthy controls (Supplementary Data 26). Labels indicate effectors with significant prevalence in either Crohn's disease or ulcerative colitis (FDR < 0.01, Fisher's exact test with BH correction). Black asterisks mark statistical significance for individual bars. f, Prevalence of indicated effectors in metagenomes of individuals with Crohn's disease (n = 504) and ulcerative colitis (n = 302) compared to healthy controls (FDR < 0.01, two-sided Fisher's exact test, BH correction). g, HuMMI subnetworks showing human proteins (grey) associated with Crohn's disease (orange border) or ulcerative colitis (purple border) interacting with effectors (coloured nodes) from strains enriched and depleted in patient metagenomes. Given the genetic and functional links between commensal effectors and IBD, we wondered whether clinical data support a potential role of effectors in the disease. Hypothesizing that causal involvement in IBD aetiology may be reflected in altered effector prevalence, we analysed a metagenome study with over 800 individuals with IBD (504 Crohn's disease, 302 ulcerative colitis) and 334 healthy controls53. Focusing on effectors with physical interactions in HuMMI, 64 effectors were significantly more prevalent in individuals with Crohn's disease compared with healthy controls, whereas effectors were less common in individuals with ulcerative colitis (Fig. These opposing trends were unexpected as Pseudomonadota abundance reportedly increases in both IBDs46,54. Some hypotheses for mechanisms underlying this observation emerged from HuMMI: effectors from K. pneumonia, E. coli and E. fergusonii, all highly prevalent in Crohn's disease, interact with the Crohn's disease susceptibility protein COG6, which directly interacts with the ulcerative colitis susceptibility gene RTP5 (Fig. Similarly, Efe_13 of E. fergusonii binds the Crohn's disease susceptibility protein TNIP1, which functions in NF-κB signalling and interacts with two genes associated with ulcerative colitis. Other enriched effectors show indirect links to Crohn's disease and ulcerative colitis proteins via shared interaction partners (Fig. While the mechanistic relevance of these interactions requires future studies, these direct and indirect connections to IBD disease proteins invite speculation that they may cause a homeostatic shift that increase the risk for Crohn's disease while decreasing the same for ulcerative colitis. T3SS are traditionally viewed as hallmarks of pathogenicity, yet in plants and insects they mediate a wider range of functions including beneficial interactions55. Our findings extend this observation to the human gut, revealing that T3SS are unexpectedly common among commensal Pseudomonadota. In particular, E. coli, which resides close to the intestinal epithelium56, frequently encodes complete systems. Although not detected in commensal beta- or delta-Pseudomonadota, divergent systems may have escaped current detection tools, thus underestimating their true distribution. Functional assays validated our predictions and revealed regulatory complexity: C. pasteurii and P. massiliensis showed inconsistent T3SS activation, whereas E. tarda reliably injected effectors into human cells. Using S. Typhimurium as a heterologous host that robustly initiates the T3SS, we confirmed translocation of 32 effectors from 11 species, indicating that many commensals harbour host-directed secretion capability. The regulatory diversity of T3SS activation is consistent with the idea that, in contrast to pathogens such as S. Typhimurium, commensals may require highly specific host or environmental cues to activate secretion. Whether human epithelial or immune cells, akin to plant hosts57, can actively signal to commensals to induce T3SS, or whether secretion primarily reflects stress responses of potential pathobionts, remains an important area for future investigation. Interpreting T3SS functionality in the human gut requires moving beyond species-level labels such as ‘commensal' or ‘pathogenic', which often obscure substantial within-species diversity. As observed in other host kingdoms58, these categories are fluid: E. coli includes both highly pathogenic lineages (for example, EPEC or EHEC) and harmless or beneficial ones, such as the probiotic E. coli Nissle 191759. In our analyses, strains isolated from apparently healthy individuals were considered commensals, whereas strains encoding known virulence effectors28, including P. aeruginosa and Salmonella spp., were designated pathogens. Importantly, between these poles lie opportunistic pathogens whose infectious potential emerges only in specific environmental or host-related conditions. A key question is therefore whether commensal T3SS primarily support opportunistic pathogenicity, or whether they have adaptive functions in the non-pathogenic lifestyle. Multiple lines of evidence from our study support the latter. Comparative sequence, structure and host-target analyses revealed that commensal and pathogen effector repertoires are largely distinct, supporting a model in which commensal T3SS are adapted for cooperative rather than pathogenic interactions. Homotypic clustering of effector structures and depletion of mixed commensal–pathogen clusters suggest that commensal effectors follow separate selective trajectories. The domain analysis supports this, revealing many domains found only in commensal effectors possibly to support a non-pathogenic lifestyle. Notably, numerous effectors involved in cyclic diguanylate synthesis or degradation were identified, often paired with PAS sensor domains suggesting environmentally responsive functions. Intriguingly, several effectors from Gram-negative commensal Pseudomonadota potentiated Pam3CSK4-induced TLR1/2 signalling, suggesting that T3SS may modulate host responses to Gram-positive Bacteroidetes and thereby influence interphyla competition within the gut ecosystem. Despite substantial divergence in effector structures, commensal and pathogen effectors exhibited both shared and unique host interactions within the meta-interactome. Although these comparisons are limited by the availability of hypothesis-driven interaction datasets for pathogen effectors, the observed patterns parallel findings in plant systems, where effectors from mutualists and pathogens converge on some common host targets while also interacting with proteins critical for distinct outcomes40. Across systems, convergence on a subset of host proteins emerges as a signature with biological importance13,40. These convergence proteins therefore emerge as key nodes in host–microorganism interactions and understanding their role in commensal versus pathogenic contexts is a promising entry point for understanding how pathogenicity emerges and how balanced immune responses are ensured. The interaction–structure models provide leads for dissecting effector mechanisms. Targeted host protein domains can indicate which processes an effector may perturb and, when mediated by a corresponding motif, whether the effector may get post-translationally modified, as seen for H. pylori CagA60. A manual analysis matching the mimicINT workflow, however, revealed no clear cases in which these engage cognate host substrate motifs and only a single example consistent with known docking specificity. Whether this reflects differences between commensal and pathogen effectors, or the prevalence of functional mimicry without sequence similarity61, or merely limitations of our approach, remain to be clarified. Post-translational modification of effectors by host enzymes may either enhance effector function or act in host defence, such as by targeting foreign proteins for degradation. The latter would, however, be expected to select against motif retention. Thus, while biochemical directionality from host domain to effector SLiM is plausible, the available evidence suggests that such modifications predominantly support the lifestyle of the injecting bacterium. When commensals act as pathobionts and contribute to non-communicable diseases, such interactions may become intervention targets. Analysis of host pathways targeted by commensal effectors indeed revealed enrichment for proteins and genetic variation implicated in immune disorders, cancers and metabolic traits. Notably, commensal effectors target network neighbourhoods associated with Crohn's disease but not ulcerative colitis, and physically and functionally interact with key members of the TNF–NF-κB signalling axis. Consistent with these molecular data, T3SS effectors were enriched in the microbiomes of individuals with Crohn's disease while being depleted in ulcerative colitis. This pattern mirrors the differential clinical response to anti-TNF therapy, which is highly effective in Crohn's disease but not in ulcerative colitis. Understanding whether and how commensal effectors directly contribute to Crohn's disease risk or flares, and whether they may even confer protection in ulcerative colitis, are compelling questions for future mechanistic studies with potential therapeutic implications. Together, our data position host-directed secretion as an underappreciated mode of communication between the human microbiota and its host. By integrating genomic, structural, functional and systems-level analyses, we provide an initial map of the commensal T3SS meta-interactome and establish a framework for exploring its roles across microbial niches, host genotypes and disease states. These findings broaden the conceptual boundaries of T3SS biology and highlight the need to examine secretion systems not only as virulence factors but also as potential modulators of mutualism, competition and host physiology within the human gut. Reference genomes for Pseudomonadota strains isolated by the human microbiome project from human guts and available from DSMZ (via BacDive), ATCC (atcc.org) or BEI (beiresources.org) were identified and cross-referenced with GenBank (release 229), yielding 77 matches, and subjected to T3SS identification, along with 92,143 and 9,367 MAGs, respectively, from two different meta-studies21,22 that were at least 50% complete and less than 5% contaminated. Prediction performance of EffectiveDB16 was evaluated by fivefold cross-validation with five repeats using simulated MAGs of 0–100% completeness and 0–50% contamination (in 5% steps) by random sampling genes from the test set. A performance-improved re-implementation of the EffectiveDB classifier (https://github.com/univieCUBE/phenotrex, trained on EggNOG 4.0 annotations62) was used with a positive prediction threshold of >0.7. For 770 T3SS+, MAGs protein coding sequences for 474,871 representative proteins were identified using prodigal (v.2.6.3)63 and CD-HIT (v.4.8.1, parameters: ‘-c 1.0')64. A total of 61,115 proteins were encoded by 44 T3SS+ culture collection genomes. Three machine-learning tools (EffectiveT3 v.2.0.1, DeepT3 v.2.025 and pEffect27) were used to predict T3SS signal or effector homology. Predictions were integrated using a 0–2 scoring scheme: 2 for perfect score (pEffect >90, EffectiveT3 >0.9999, DeepT3: both classifiers positive prediction); 1 for positive prediction at default thresholds (pEffect >50, EffectiveT3 >0.95, DeepT3: one classifier); 0 for negatives. Sequences with a sum score above 4 were regarded as potential effectors. Sequences lacking start/stop codons or containing transmembrane regions (TMHMM 2.0) were excluded. Proteins were clustered using 90% sequence identity (CD-HIT parameters: ‘-c 0.9 -s 0.9') to reduce redundancy. T3SS were analogously predicted for 4,753 strains from the human gastrointestinal bacteria genome collection (HBC)17, Broad Institute–Open Biome Microbiome Library (BIO–ML) and Global Microbiome Conservancy (GMC)18,19. To obtain phylogenetic relationships for T3SS+ strains, concatenated bac120 marker proteins from GTDB-Tk (v.2.1)65 were used. T3SS+ genomes were matched to Weizmann Institute of Science representative genomes of the human gut24 with FastANI v.1.0 using average nucleotide identity (ANI) values >95% (ref. Effectors were aligned using the Needleman Wunsch algorithm and were considered ‘homologous' in HuMMIHOM using mutual sequence identity of ≥30% over 90% of the common sequence length (Supplementary Data 11). Sequences of 1,195 known pathogenic T3 effectors were obtained from BastionHub28 (29 August 2022), and sequence similarity between commensal and pathogenic effector sequences was assessed using BLAST (v.2.10)67. For each commensal effector, the pathogen effector with the highest sequence similarity was considered as the best match and used to calculate alignment coverage. Additional significant similarities were identified using iterative sequence searches against ~124 M non-redundant bacterial sequences from UniRef90 (January 2024) with Jackhmmer29. For each commensal effector, we ran five iterations using inclusion and comparison E-value thresholds of <10−5 (Supplementary Data 5). Protein domain annotation for effectors was carried out using the standalone version of InterProScan (v.5.75-106.0), using Pfam v.37.4 as reference. Amino acid sequences in FASTA format were used as input across three datasets: effector proteins from commensal bacterial strains (n = 3,002) and MAGs (n = 186), human and vertebrate pathogen effectors obtained from BastionHub, and all reviewed human proteins from the UniProtKB/Swiss-Prot reference proteome. InterProScan used (translated) protein sequences (Supplementary Data 4) with default parameters. Domain hits with an E-value < 10−5 were considered significant. Domains identified as significant in commensal effectors were used as reference for comparative analysis and evaluated for their presence in pathogen effectors and human proteins, applying the same annotation criteria and significance threshold. All domain annotation results, including individual hits across datasets and the comparative summary, are provided in Supplementary Data 5. Structures of pathogen and commensal effectors were compared using FoldSeek70. Effector structures were downloaded from the AlphaFold DB when available; otherwise, a model with >95% sequence identity and >90% sequence coverage was selected as representative. The clustering analysis was run for all commensals against three sets of pathogen effectors: all pathogens (895 structures from human, vertebrate and plant pathogens), human and vertebrate pathogens (536 structures) and human pathogen effectors only (488 structures) (Supplementary Data 6). For PCR cloning, genomic DNA or bacterial stocks were obtained from culture collections: ATCC (via LGC Standards, Wesel, Germany, or ATCC, Manassas, VA, USA), DSMZ (Leibniz Institute DSMZ, Braunschweig, Germany) and BEI resources (Manassas, VA, USA) (Supplementary Data 9). Live strains were cultured according to supplier protocols and DNA was extracted using the NucleosSpin Plasmid mini kit. Effectors were cloned into pENTR223.1 by nested PCR to add Sfi sites and by restriction enzyme-based cloning using standard protocols, and verified by Sanger sequencing. For experiments, effectors were moved into pDEST-DB (pPC97, Cen origin), the pDEST-N2H-N1 and -N2 and pMH-Flag-HA by Gateway LR reactions. For bacterial injection assays, effector ORFs were cloned into a modified bacterial expression plasmid based on the pEYFP backbone (BD Biosciences, 6004-1). The EYFP sequence (positions 217–1,407) was removed and replaced with (1) SfiI and XbaI restriction sites for directional cloning of effector ORFs, (2) a 3× Flag epitope tag, (3) the HiBiT tag coding sequence VSGWRLFKKIS (Promega), and (4) the E. coli rrnB transcriptional terminator (pLac_FL_HiBiT). PCR-amplified effectors were ligated into pLac_FL_HiBiT at SfiI and XbaI restriction sites (primers in Supplementary Data 7). The positive control SipA was amplified from a pT10-based plasmid (pMIB6433)34. Cloning was verified by analytical PCR. Typhimurium (wild-type SB300 and ΔsctV mutant SB1751)34, E. tarda (ATCC 23685), C. pasteurii (DSM 28879) and P. massiliensis (DSM 26120) were generated in-house and electroporated with effector encoding plasmids using a Gene Pulser Xcell Electroporation System (Bio-Rad) at 2.5 kV and 200 Ω for ~5 ms. Transformed strains were cultured overnight in LB medium with ampicillin for subsequent use in injection assays. The injection assay was adapted from ref. HeLa cells stably expressing LgBiT (HeLa-LgBiT) were grown using standard conditions (DMEM, 10% fetal bovine serum (FBS), 37 °C, 5% CO2) for 24 h before infection. S. Typhimurium strains carrying pLac_FL_HiBiT effector constructs were cultured overnight in LB supplemented with 0.3 M NaCl and ampicillin. Edwardsiella tarda, C. pasteurii and P. massiliensis strains were cultured with 200 μM IPTG to induce effector expression. Overnight bacterial cultures were added to the HeLa-LgBiT cells at a multiplicity of infection of 50 and jointly incubated for 1 h (Salmonella) or 1.5 h. After media replacement, extracellular luminescence was quenched by addition of 1× DarkBiT peptide (Promega, CS3002A02) for 50 min. Luminescence was measured after addition of 25 μl fresh Nano-Glo reagent (Promega, N2012) using a SpectraMax ID3 microplate reader (1,000 ms). Each strain was tested with four technical replicates and five biological replicates performed on separate days. Luminescence values from technical replicates were averaged to obtain a single value for each biological replicate. Luminescence fold-change was calculated by dividing the average signal from the effector-expressing strain by that of the mock control separately for wild-type and ΔsctV strains. To assess effector translocation, fold-change values were statistically compared between wild-type and mutant strains for Salmonella and against the negative control (SipA in ΔsctV) for E. tarda, using Wilcoxon rank-sum test (Supplementary Data 10). Proteins were separated by 10% or 15% SDS–PAGE, transferred to PVDF membranes (Bio-Rad, 1620177) and blocked in blocking solution (5% non-fat dry milk in 1× PBS) for 1 h at room temperature or overnight at 4 °C. Blots were done with mouse anti-Flag M2 monoclonal antibody (Sigma-Aldrich, F1804, 1:5,000), rabbit anti-Myc (Abcam, ab9106, 1:1,000), followed by HRP-conjugated secondary antibody (Santa Cruz Biotechnology; anti-mouse: sc-516102; anti-rabbit: sc-2357, both 1:5,000) for 1 h each with three washes with blocking solution or PBST, respectively. Signal was detected with SuperSignal West Femto Substrate (Thermo Scientific, 34094) according to manufacturer instructions. Blots were imaged using the Intas ChemoStar imaging system. A multi-assay interactome mapping pipeline was used37 (Extended Data Fig. In the initial screening by Y2H, candidate effectors fused to the Gal4 DNA-binding domain (DB-X) were screened against 17,472 human proteins fused to the Gal4 activation domain (AD-Y). Before screening, DB-X ORFs were tested for autoactivation by mating against AD-empty plasmids. In the primary screen, DB-X strains in Y8930 (MATα) were mated on yeast extract peptone dextrose (YEPD) agar (1%) plates against minipools of ~188 AD-Y in Y8800 (MATa) representing the human ORFeome collection (v.9.1)36. After 24 h, yeasts were replica-plated onto selective media lacking leucine, tryptophan and histidine (SC-Leu-Trp-His), containing 1 mM 3-AT (3-amino-1,2,4-triazole) (3-AT plates) and replica-cleaned after 24 h. After 48 h, colonies were picked and then grown for 72 h in SC-Leu-Trp liquid medium for secondary phenotyping using the same selective +SC-Leu-His + 1 mM 3-AT + 1 mg l−1 cycloheximide plates to identify spontaneous autoactivators. Clones growing on 3-AT plates but not on cycloheximide plates were processed for sequence identification using a modified Kilo-seq procedure35: ORFs were amplified and tagged by PCR using a universal ‘term' reverse primer (5'-GGAGACTTGACCAAACCTCTGGC) and Gal4-AD and -DB specific forward primers with position barcodes (Supplementary Table 11) and a TruSeq P7 sequence (0.2 µl DreamTaq DNA polymerase (ThermoFisher, EP0702), 3 µl 2 µM term primer, 3 µl forward primer, 2 µl yeast lysis). The DNA concentration of each pool was quantified by the QuantiT PicoGreen dsDNA Assay kit (ThermoFisher, P7589) using a lambda DNA dilution series (50–0.390625 ng μl−1), then diluted to 1–2 ng μl−1 and tagmented with 0.25 µl TDE enzyme (Illumina Tagment DNA TDE1 Enzyme and Buffer kit, 20034197). Finally, haploid yeasts of the DB-X and AD-Y candidate interaction pairs were mated individually and tested four times on selective plates using empty AD and DB plasmids as negative controls. Growth scoring was performed using a custom dilated convolutional neural network35. Pairs scoring positive in at least three out of four repeats qualified as bona fide Y2H interactors. The AD-Y and DB-X constructs were again identified by Illumina sequencing. To identify reliably documented interactions between bacterial effectors and human proteins for our control set, we queried the IMEx consortium protein interaction databases71 for pairs supported by multiple evidence and at least one experiment detecting direct interactions. We manually recurated the corresponding publications and identified 67 well-documented direct interactions between 29 T3 effectors and 64 human proteins, described in 38 distinct publications constituting bhLit_BM-v1. To assemble bhRRS-v1, we randomly paired T3 effectors from bhLit_BM-v1 with human proteins in HuRI (Supplementary Data 12). Effector ORFs were cloned into Entry and experimental plasmids as described above. Human hsPRS/RRS-v2 ORFs were taken from hORFeome9.1 (ref. 36) and verified by end-read Sanger sequencing. A total of 200 interaction pairs were randomly picked from HuMMI; all ORFs (Supplementary Data 14) were transferred by Gateway LR reactions into pDEST-N2H-N1 and pDEST-N2H-N2, and transformed into haploid Saccharomyces cerevisiae Y8800 (MATa) and Y8930 (MATα) strains. Protein pairs from all datasets were randomly distributed across matching 96-well plates. Luminescence from reconstituted NanoLuc for each sample was measured on a SpectraMax ID3 (Molecular Devices) with a 2-s integration time. All obtained NLR values were log2 transformed and the positive fraction for each dataset was determined at log2 NLR thresholds between –2 and 2, in 0.01 increments. Statistical results were robust across a wide range of stringency thresholds. Reported P values were calculated by Fisher's exact test. We evaluated whether N-terminally Flag-HA-tagged effector, or negative control Flag-GFP, co-immunoprecipitated the human proteins carrying an N-terminal MYC tag. Transfections for test and control pairs were always processed in parallel. HEK293 cells (RRID: CVCL_0045, DSMZ) were seeded in 10-cm dishes at a density yielding 60–70% confluency on the day of transfection. Plasmid DNA and X-tremeGENE transfection reagent (Roche) were mixed at a ratio of 1:2 (µg DNA:µl reagent) in serum-free DMEM. Per dish, 10 µg plasmid DNA (consisting of 3 µg effector- or GFP-encoding plasmid, 3 µg plasmid encoding the human protein and 4 µg empty vector) was diluted in 500 µl serum-free medium, followed by the addition of 20 µl X-tremeGENE reagent. The mixture was inverted twice, incubated for 15 min at room temperature and then added dropwise to the culture dish containing cells in complete growth medium. Cells were incubated under standard culture conditions (37 °C, 5% CO2) for 24 h before downstream analysis. For cell lysate preparation, all steps were performed on ice. Culture medium was aspirated, and cells were washed three times with ice-cold 1× PBS by rinsing and aspirating sequentially. Cells were lysed directly on the plate by adding 1 ml NP-40 lysis buffer per plate (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1% (v/v) NP-40 and 2.5 mM EDTA, with Roche complete protease inhibitor). Lysates were incubated on ice for 30 min and cleared by centrifugation at 30,000g for 15 min at 4 °C. For immunoprecipitation experiments, 1 mg of cleared lysates of each sample was diluted into a final volume of 750 µl, and then 50 µl of an NP-40 buffer equilibrated with 20% anti-Flag M2 affinity gel (Sigma-Aldrich, A2220) slurry was added. Samples were rotated at 4 °C for 1 h. For washing, the tube was centrifuged at maximum speed for 30 s, the supernatant aspirated and 1 ml NP-40 wash buffer added, followed by a brief inversion. After three washes, the beads were resuspended in 50 µl NP-40 buffer, 50 µl Laemmli loading buffer was added, and the beads were heated at 98 °C for 10 min and briefly centrifuged before analysis. For analysis, 10 µl of cleared lysates and 15 µl of all immunoprecipitates were loaded on SDS–PAGE and processed through western blots as described above. The completeness of an interactome map is an important parameter that enables assessment of overlap and how complete a given biology is covered by the map. All reference sets were tested four times using the Y2H screening pipeline (Supplementary Data 13). To assess sampling sensitivity (Ss), a repeat screen was conducted. A total of 288 bacterial effectors were screened 4 times against 5 pools comprising 1,475 human proteins. A saturation curve was calculated as described37. Reciprocal parameters were calculated and the Michaelis Menten equation was used with modified variables: analogous to increasing substrate concentrations in enzyme reactions, repeat screens progressively drive the screen to saturation91. Hence a saturation curve was predicted using Ni(R) = Nimax × R/Km + R, with Ni representing the interaction count after R repeats, Nimax the saturation limit and Km the Michaelis constant. Overall sensitivity emerges from both sampling and assay limitations and was calculated as So = Sa × Ss. To estimate the significance of effector convergence, we performed a permutation test by randomly sampling with replacement 979 target nodes from HuRI36 (n = 8,274). A P value was obtained from the z-scores using the ‘pnorm()' R command and multiplied by 2 for a two-tailed test. To avoid overestimation and increase stringency, we restricted the analysis to Y2H positive proteins in HuMMIMAIN and HuRI. To assess interspecies convergence, we used a conditional permutation test that preserves the strain contribution. Each iteration generated 18 samples corresponding to the observed number of targets for each strain (Supplementary Data 11). For every protein, the frequency of selection across all strains was recorded as its convergence value. Significance was observed from four strains onward (P < 0.004), and proteins targeted by at least four strains were considered to show interspecies convergence. Pairs with fewer than three targets were excluded. To analyse the interfaces of effector–host interaction pairs, all identified pairs were subjected to structural prediction using AlphaFold v.2.3.1 with the following options: –model_preset=multimer, –db_preset=full_dbs, –max_template_date=2023-12-19, –num_multimer_predictions_per_model=1, –enable_cpu_relax and –use_precomputed_msas. Predictions were not generated for pairs whose combined length exceeded 2,500 residues. The predicted aligned error (PAE) matrix was extracted from the AlphaFold pickle output using alphapickle v.1.4.1 (https://github.com/mattarnoldbio/alphapickle, https://doi.org/10.5281/zenodo.5752375). To assess confidence, we used the confident contacts count (CCC), which is the number of residue–residue contacts72 with PAE < 4 Å. Each putative interface residue was assigned a PAE value. When a residue was in contact with multiple residues on the partner protein, the minimum PAE value among those contacts was used. Structure predictions were considered confident when the CCC was ≥5. Protein sequences (Supplementary Data 17) were converted from single-letter aa notation to three-letter residue annotation, and residue identifiers were assigned to match their positions in the AlphaFold PAE matrix. Only human proteins targeted by at least two bacterial effectors were retained. Contacts with PAE values equal or below the threshold were retained, and the corresponding human and bacterial residues and total retained contacts were recorded. Interface similarity between bacterial effectors targeting the same human protein was assessed using the Jaccard index across all PAE thresholds. For each targeted human protein, all interacting bacterial effectors were identified, and all possible effector–effector combinations were generated. At each threshold, the Jaccard index was calculated as the number of overlapping human interface residues divided by the total number of unique residues in both interfaces. Analogous calculations were performed to analyse interfaces of human proteins targeted by the same bacterial effector. Domains were assigned to the interacting human proteins using InterProScan v.5.75 with InterPro release 106.0, run through the EBI web server. Domain coordinates, descriptions and confidence scores were retrieved. We used as mimicINT43 input, a representative set of effectors identified in isolated strains (2,300 sequences clustered at 90% identity) and all effectors identified in MAGs (186). mimicINT detects domains in effector sequences using the signatures from the InterPro v.81.0 database73, retaining matches with an E-value < 10−5. For motif detection, mimicINT uses definitions available in the ELM database74. The IUPred 1.0 algorithm75 was employed to detect motifs in disordered regions with both short and long models (motif disorder propensity = 0.2 (ref. First, domain binding specificity within the same family was accounted for by computing a profile HMM-based domain score41 (stringency threshold = 0.3). Second, given the degenerate nature of motifs41, mimicINT uses Monte Carlo simulations to estimate the probability of a SLiM occurring by chance, by shuffling disordered regions of the input sequences to generate N randomized proteins. Disordered regions were shuffled 100,000 times using two backgrounds: same-strain effectors (within-strain shuffling) and full effector set (interstrain shuffling). To assess whether the number of inferred interface-resolved interactions exceeded random expectation, the analysis was controlled using 10,000 degree-controlled random networks generated from the human interaction search space (Supplementary Data 18). Holdup is a biochemical assay used to validate the interface predictions involving PDZ domains. A total of 54 human PDZ domains and 11 tandem constructs were recombinantly expressed as His6-MBP-PDZ constructs in E. coli BL21(DE3) pLysS and purified by Ni2+-affinity columns using 800 µl of beads (Chelating Sepharose Fast Flow immobilized metal affinity chromatography, Cytiva) per target. Protein concentrations were determined using A280 nm on a PHERAstar FSX plate reader (BMG LABTECH), and purity assessed by SDS–PAGE and capillary electrophoresis; 4 µM stocks were stored at −20 °C. Biotinylated peptides (10-mer) corresponding to the C-terminal sequences of effectors were synthesized by GenicBio Limited; the N-terminal biotin was attached via a 6-aminohexanoic acid linker, and all peptides were >95% pure (HPLC and MS). For the assay, 2.5 µl of streptavidin resin (Cytiva, 17511301) was incubated in a 384-well filter plate (Millipore, MZHVN0W10) for 15 min with 20 µl of a 42 µM peptide solution. Individual PDZ domains were added to wells, incubated for 15 min, and unbound PDZ recovered by centrifugation into 384-well black assay plates for fluorescence readout. Concentrations were quantified by intrinsic Trp fluorescence, and fluorescein/mCherry was used for peak normalization. Binding affinities and equilibrium dissociation constants were calculated as previously described45, using the mean PBM concentration. Raw values and statistical analysis are provided in Supplementary Data 19. Functional enrichment of effector targets was assessed using the ‘gost()' function in the ‘gprofiler2' R package (v.0.2.1)78 with HuRI as the background (custom_bg), excluding electronic annotations (exclude_iea = TRUE), with Benjamini–Hochberg correction (correction_method = ‘fdr'). Functional categories were drawn from Gene Ontology biological process terms (GO:BP), Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways, and the Reactome pathways database (sources = c(‘GO:BP', ‘KEGG', ‘ REAC')). Odds ratios and fold enrichments were calculated to estimate effect sizes, where the odds ratios was the ratio of odds in the target set to those in the HuRI background, and fold enrichment compared observed to expected annotated targets. Expected values were based on random sampling from the HuRI background (GO:BP = 6,988; KEGG = 3,250; Reactome = 4,592) (Supplementary Data 20). Similar analyses were performed for functional enrichment analysis of human proteins targeted by pathogens (Supplementary Data 20). We assessed enrichment of targeted enzymes across metabolic subsystems using the human genome-scale model Recon3D48. Recon3D is a curated static model of human metabolism that lacks post-translational and allosteric regulation. Ligases and kinases were excluded to focus on metabolic enzymes. For each of the 95 Recon3D subsystems, enrichment was tested using the ‘phyper()' R function, with inputs corresponding to annotated and unannotated targeted enzymes and BH false discovery rate (FDR) correction. OR and fold enrichment were calculated as described for functional analyses (Supplementary Data 20). Associations of effector targets and convergence proteins with human disease genetics were tested using a two-sided Fisher's exact test. Disease-causal genes were obtained from the Open Targets genetic portal (access date 23 August 2022), which integrates variant-to-gene distance, quantitative trait loci co-localization, chromatin interactions and variant pathogenicity79. The portal's machine-learning model assigns each locus-to-gene (L2G) score to genes in loci identified in GWAS to identify the most probable causal gene. Genes with L2G ≥ 0.5 were considered causal as recommended80. Ensembl identifiers were converted to gene symbols using the biomaRt R package (v.2.60.1, Bioconductor 3.19), and Fisher's exact test was implemented in R (fisher.test), stats v.4.2.2 using default parameters on 2 × 2 contingency tables comparing causal gene presence in query and background sets. HuRI protein encoding genes were used as the background, and targets or convergence proteins as the query sets. FDR correction and OR and fold enrichments were calculated as done for functional enrichment (Supplementary Data 21). We implemented a random walk with restart (RWR) algorithm, RWR-MH81, to explore the network neighbourhood of 338 human proteins targeted by 243 commensal effectors in HuRI36 (HuMMIMAIN). Human targets were used as seeds, with the restart probability of 0.7 generating a ranked list of proteins. Statistical significance was assessed by random walks in degree-preserved randomized networks. We generated 1,000 random networks from HuRI and computed RWR scores for each protein, retaining as network neighbour only those with empirical P < 0.01. For each set of significant neighbourhood proteins, we tested for enrichment of Open Targets causal genes (L2G ≥ 0.5) linked to traits supported by at least three causal genes. Enrichment in each strain neighbourhood was assessed using two-sided Fisher's exact test with BH correction. We therefore focused on 400 associations with nominal P < 0.01 and odds ratio > 3. Disease categorizations were refined to reflect aetiology; Sjogren syndrome, eczema and psoriasis were grouped as immunological rather than eye or skin traits, and osteoarthritis as musculoskeletal/connective tissue rather than metabolic traits. 4d, related asthma and psoriasis terms were merged (Supplementary Data 23). HEK293 cells (RRID: CVCL_0045, DSMZ) were maintained in DMEM, 10% FBS, 100 U ml−1 penicillin–streptomycin at 37 °C and 5% CO2. Cells (1 × 106 per 60 mm dish) were transfected with 10 ng NF-κB reporter plasmid (6× NF-κB firefly luciferase pGL2), 50 ng pTK reporter (Renilla luciferase) and 2 µg bacterial ORF in pMH-Flag-HA using the calcium phosphate method. To assess NF-κB inhibition, cells were treated for 4 h with 20 ng ml−1 TNF (Sigma-Aldrich, SRP3177) at 24 h post transfection. Lysates were analysed using the dual-luciferase reporter kit (Promega, E1980) with a luminometer (Berthold Centro LB960 microplate reader, software: MikroWin 2010). NF-κB induction was determined as firefly/Renilla luminescence. P values were calculated using Kruskal–Wallis test with Dunn's post hoc comparisons followed by FDR correction. Raw values and statistical analysis are provided in Supplementary Data 24. Protein expression was analysed by western blot as described above with following modifications: blocking solution contained 0.1% Tween-20. Membranes were incubated overnight at 4 °C with primary antibodies in 2.5% BSA in PBST, washed and probed with anti-mouse secondary antibody in PBST for 1 h at room temperature (1:10,000; Jackson ImmunoResearch Labs, RRID:AB_2340770). Signals were detected using LumiGlo reagent (CST, 7003S) and chemiluminescence film (Sigma-Aldrich, GE28-9068-36). 5c were performed by transfecting Caco-2 cells using 40,000 MW linear polyethylenimine (PEI MAX) (Polysciences) at a ratio of 1:5 pDNA:PEI. Cells were exposed to the transfection mixture for 16 h, washed, recovered for 6 h and then sorted (BD FACSAria III cell sorter, BD Biosciences). During activation, proliferation was monitored in the Incucyte S3 Live Cell Analysis system (Essen BioScience). Cytokine levels were determined using the human inflammation panel 1 LEGENDplex kit (Biolegend). Statistical significance was tested on the average of the technical replicates using Kruskal–Wallis test with Dunn's post hoc comparisons. 4d were performed by transfecting cells using the 4D-Nucleofector system (Lonza). Collected cells were resuspended in SF nucleofector solution, added with 0.6 µg plasmid, and pulsed (code DG-113) and plated in DMEM + 5% FBS. Cells were allowed to recover overnight and then rested in culture medium for 24 h. Cells were stimulated with 10 µg ml−1 Pam3CSK4 (tlrl-pms, Invivogen), 1 µg ml−1 flagellin (tlrl-stfla, Invivogen) or 100 ng ml−1 TNF (130-094-014, Miltenyi Biotec) for 24 h. We performed five biological repeat experiments with three technical repeats each. For each experiment, pooled supernatants were analysed using the Human Anti-virus Response Panel V02 (BioLegend). The data were analysed using Kruskal–Wallis test with Dunn's post hoc comparisons. Metagenomic assemblies from the Inflammatory Bowel Disease Multiomics DataBases (IBDMBD)53 and from the skin metagenome82 were downloaded, and protein repertoires predicted using Prodigal (option: -p meta)83. Effectors were compared to the metagenomic protein repertoires using DIAMOND 0.9.24 (options: >90% query length, >80% identity). Binary presence and absence vectors for each effector across the sample were generated and the prevalence of each effector in patients compared to the controls was assessed using Fisher's exact test, implemented within the SciPy 1.9.3 Python 3.10.12 module, and FDR corrected using BH correction. Differences in prevalence distributions between healthy and either patient cohort were estimated using the Wilcoxon rank-sum test, implemented in the ‘wilcox.test()' R function. We used fold change as the measure of effect size in Fig. 5e, calculated as prevalence in the test group divided by prevalence in the healthy group. To avoid division by zero, we applied a small pseudo-count to the healthy cohort data for individuals with 0% prevalence. The pseudo-count was equivalent to half a case in the healthy cohort (n = 334 individuals), ensuring minimal influence on results while enabling calculation of fold change. Statistical details are provided in Supplementary Data 26. Data were subjected to statistical analysis and plotted in Microsoft Excel 2010 or Python or R scripts. For comparison of values not passing the normality tests, we used either Kruskal–Wallis test with Dunn's correction for multiple-group comparisons or Wilcoxon rank-sum test for two-group comparisons as indicated. Enrichments were calculated using Fisher's exact test with Bonferroni FDR correction. All statistical evaluations were done as two-sided tests. Generally, a corrected P < 0.05 was considered significant. GO, KEGG and Reactome functional enrichments were calculated using the gprofiler2 R package with the indicated background sets. For the disease target enrichments and neighbourhood associations, no associations were significant after multiple hypothesis correction, which is why nominally significant associations calculated by Fisher's exact tests were used for Fig. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All sequence, interaction and functional data generated in this study are available as supplementary information. The effectors identified and cloned for interactome mapping are presented in Supplementary Data 7. All protein interaction data have been deposited to the IMEx consortium (http://www.imexconsortium.org) through IntAct and assigned the identifier IM-29849. New effector sequences have been submitted to GenBank: BankIt2727690: OR372873–OR373035 and OR509516–OR509528. Source data are provided with this paper. Data and scripts related to the prediction of T3SS-positive reference strains and metagenomes are available on Zenodo at https://doi.org/10.5281/zenodo.17825584 (ref. All data and scripts generated to perform the structural comparison between commensal and pathogen effectors are available on Zenodo at https://doi.org/10.5281/zenodo.11951539 (ref. The full set of inferred SLiM–domain and domain–domain interactions and the randomly generated networks are available on Zenodo at https://doi.org/10.5281/zenodo.11400863 (ref. The mimicINT43 code can be found on GitHub at https://github.com/TAGC-NetworkBiology/mimicINT/releases/tag/v1 (ref. The 1,000 randomized control networks for the random walk analysis are available on Zenodo at https://doi.org/10.5281/zenodo.12743976 (ref. The AlphaFold predictions of effector–host interaction pairs along with all confidently predicted structures are available on Zenodo at https://doi.org/10.5281/zenodo.16816224 (ref. Structure, function and diversity of the healthy human microbiome. Examining the healthy human microbiome concept. Lietzen, N. et al. Coxsackievirus B persistence modifies the proteome and the secretome of pancreatic ductal cells. Rozenblatt-Rosen, O. et al. Interpreting cancer genomes using systematic host network perturbations by tumour virus proteins. Assembly, structure, function and regulation of type III secretion systems. & Bailey, C. M. Bioinformatics, genomics and evolution of non-flagellar type-III secretion systems: a Darwinian perspective. Miwa, H. & Okazaki, S. How effectors promote beneficial interactions. The evolution of effectome-based strategies to establish beneficial symbiosis. Wessling, R. et al. Convergent targeting of a common host protein-network by pathogen effectors from three kingdoms of life. Independently evolved virulence effectors converge onto hubs in a plant immune system network. Nelson, K. E. et al. A catalog of reference genomes from the human microbiome. Eichinger, V. et al. EffectiveDB—updates and novel features for a better annotation of bacterial secreted proteins and Type III, IV, VI secretion systems. Hitch, T. C. A. et al. HiBC: a publicly available collection of bacterial strains isolated from the human gut. Poyet, M. et al. A library of human gut bacterial isolates paired with longitudinal multiomics data enables mechanistic microbiome research. Groussin, M. et al. Elevated rates of horizontal gene transfer in the industrialized human microbiome. & Shen, X. H. Type VI secretion systems present new insights on pathogenic Yersinia. Almeida, A. et al. A new genomic blueprint of the human gut microbiota. Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Leviatan, S., Shoer, S., Rothschild, D., Gorodetski, M. & Segal, E. An expanded reference map of the human gut microbiome reveals hundreds of previously unknown species. Jing, R. et al. DeepT3 2.0: improving type III secreted effector predictions by an integrative deep learning framework. Arnold, R. et al. Sequence-based prediction of type III secreted proteins. & Bromberg, Y. Computational prediction shines light on type III secretion origins. Wang, J. W. et al. BastionHub: a universal platform for integrating and analyzing substrates secreted by Gram-negative bacteria. Varadi, M. et al. AlphaFold Protein Structure Database in 2024: providing structure coverage for over 214 million protein sequences. Galperin, M. Y., Nikolskaya, A. N. & Koonin, E. V. Novel domains of the prokaryotic two-component signal transduction systems. & Woodward, J. J. Cyclic dinucleotides at the forefront of innate immunity. Westerhausen, S. et al. A NanoLuc luciferase-based assay enabling the real-time analysis of protein secretion and injection by bacterial type III secretion systems. Kim, D. K. et al. A proteome-scale map of the SARS-CoV-2-human contactome. Luck, K. et al. A reference map of the human binary protein interactome. Altmann, M. et al. Extensive signal integration by the phytohormone protein network. Maximizing binary interactome mapping with a minimal number of assays. The IntAct database: efficient access to fine-grained molecular interaction data. Osborne, R. et al. Symbiont–host interactome mapping reveals effector-targeted modulation of hormone networks and activation of growth promotion. Towards a structurally resolved human protein interaction network. Choteau, S. A. et al. mimicINT: A workflow for microbe–host protein interaction inference. Gutierrez-Gonzalez, L. H. et al. Peptide targeting of PDZ-dependent interactions as pharmacological intervention in immune-related diseases. Gogl, G. et al. Quantitative fragmentomics allow affinity mapping of interactomes. The gut microbiome and inflammatory bowel diseases. & Gilmore, T. D. Evolutionary origins of Toll-like receptor signaling. Brunk, E. et al. Recon3D enables a three-dimensional view of gene variation in human metabolism. Ochoa, D. et al. Open Targets Platform: supporting systematic drug-target identification and prioritisation. Modeling sample variables with an Experimental Factor Ontology. Bunker, J. J. et al. Natural polyreactive IgA antibodies coat the intestinal microbiota. & van Putten, J. P. M. Reptile Toll-like receptor 5 unveils adaptive evolution of bacterial flagellin recognition. Lloyd-Price, J. et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. Gut microbiome structure and metabolic activity in inflammatory bowel disease. Egan, F., Barret, M. & O'Gara, F. The SPI-1-like type III secretion system: more roles than you think. Shalon, D. et al. Profiling the human intestinal environment under physiological conditions. & Filion, M. Bridging the gap: type III secretion systems in plant-beneficial bacteria. Rodriguez, P. A. et al. Systems biology of plant–microbiome interactions. Oral administration of the probiotic strain Escherichia coli Nissle 1917 reduces susceptibility to neuroinflammation and repairs experimental autoimmune encephalomyelitis-induced intestinal barrier dysfunction. Backert, S., Tegtmeyer, N. & Selbach, M. The versatility of Helicobacter pylori CagA effector protein functions: the master key hypothesis. Scott, N. E. & Hartland, E. L. Post-translational mechanisms of host subversion by bacterial effectors. Huerta-Cepas, J. et al. eggNOG 4.5: a hierarchical orthology framework with improved functional annotations for eukaryotic, prokaryotic and viral sequences. Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. & Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Fu, L., Niu, B., Zhu, Z., Wu, S. & Li, W. CD-HIT: accelerated for clustering the next-generation sequencing data. Chaumeil, P. A., Mussig, A. J., Hugenholtz, P. & Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. Jain, C., Rodriguez, R. L., Phillippy, A. M., Konstantinidis, K. T. & Aluru, S. High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. Camacho, C. et al. BLAST+: architecture and applications. Urban, M. et al. PHI-base in 2022: a multi-species phenotype database for pathogen–host interactions. Introducing the Bacterial and Viral Bioinformatics Resource Center (BV-BRC): a resource combining PATRIC, IRD and ViPR. van Kempen, M. et al. Fast and accurate protein structure search with Foldseek. Protein interaction data curation: the International Molecular Exchange (IMEx) consortium. & Aloy, P. Interactome3D: adding structural details to protein networks. The InterPro protein families and domains database: 20 years on. Kumar, M. et al. ELM—the eukaryotic linear motif resource in 2020. Prediction of protein disorder based on IUPred. Edwards, R. J., Paulsen, K., Aguilar Gomez, C. M. & Perez-Bercoff, A. Computational prediction of disordered protein motifs using SLiMSuite. & Peterson, H. gprofiler2—an R package for gene list functional enrichment analysis and namespace conversion toolset g:Profiler. Mountjoy, E. et al. An open approach to systematically prioritize causal variants and genes at all published human GWAS trait-associated loci. Barrio-Hernandez, I. et al. Network expansion of genetic associations defines a pleiotropy map of human cell biology. Valdeolivas, A. et al. Random walk with restart on multiplex and heterogeneous biological networks. Oh, J. et al. Biogeography and individuality shape function in the human skin metagenome. Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. Hyden, P. Prediction of T3SS-positive reference strains and metagenomes. Choteau, S., Brun, C. & Zanzoni, A. Inferred protein interactions between candidate type 3 effectors and human proteins. Choteau, S. et al. mimicINT: a computational workflow to infer protein–protein interactions. Saha, D., Perrin, J., Brun, C. & Zanzoni, A. RWR neighborhood analysis on HuRI. Lambourne, L. AlphaFold-multimer predictions for meta-interactome PPIs. Dohai, B. Metainteractome functional and disease enrichment analysis. Evidence for network evolution in an Arabidopsis interactome map. This work was supported by HDHL-INTIMIC ‘Interrelation of the Intestinal Microbiome, Diet and Health' (BMBF 01EA1803 to P.F.-B., ANR ANR-17-HDIM-0001 to C.B. ), the European Union's Horizon 2020 Research and Innovation Programme (Project ID 101003633, RiPCoN; P.F.-B., C.B. ); the Free State of Bavaria's AI for Therapy (AI4T) Initiative through the Institute of AI for Drug Discovery (AID) (P.F.-B. Centre de Calcul Intensif d'Aix-Marseille is acknowledged for granting access to its high-performance computing resources. received funding from the ‘Espoirs de la recherche' programme managed by the French Fondation pour la Recherche Médicale (FRM, FDT202106013072). J.F.-M. was funded by Consejo Nacional de Humanidades Ciencias y Tecnologias (CONAHCYT) Becas al Extranjero Convenios GOBIERNO FRANCES 2021 – 1 grant 795494, and received support from the ‘Espoirs de la Recherche' programme managed by the French Fondation pour la Recherche Medicale (FRM, FDT202404018637). D. Krappmann was supported by Deutsche Forschungsgemeinschaft (ID 210592381 – SFB 1054 A04). was supported by the Deutsche Forschungsgemeinschaft (project 403224013 – SFB 1382 Q02). was supported by an ERS Long-Term Research Fellowship (LTRF2024-01131). Open access funding provided by Helmholtz Zentrum München - Deutsches Forschungszentrum für Gesundheit und Umwelt (GmbH). These authors contributed equally: Veronika Young, Bushra Dohai. Institute of Network Biology (INET), Molecular Targets and Therapeutics Center (MTTC), Helmholtz Munich, German Research Center for Environmental Health, Neuherberg, Germany Veronika Young, Bushra Dohai, Hridi Halder, Benjamin Weller, Chung-Wen Lin, Ye Min Tun, Luisa Pletschacher, Patrick Schwehn, Simin T. Rothballer, Melina Altmann, Stefan Altmann, Alexandra Strobel, Michael Rothballer, Claudia Falter & Pascal Falter-Braun Aix-Marseille University, INSERM, TAGC, UMR_S1090, Turing Center for Living Systems, Marseille, France Jaime Fernandez-Macgregor, Deeya Saha, Mégane Boujeant, Sébastien A. Choteau, Lou Bergogne, Jérémie Perrin, Christine Brun & Andreas Zanzoni Niels S. van Heusden, Daan K. J. Pieren & Sonja Rittchen Functional Microbiome Research Group, Institute of Medical Microbiology, University Hospital of RWTH Aachen, Aachen, Germany Department of Microbiology and Ecosystem Science, Research Network: Chemistry meets Microbiology, University of Vienna, Vienna, Austria Luke Lambourne, Michael A. Calderwood & Marc Vidal Department of Genetics, Blavatnik Institute, Harvard Medical School, Boston, MA, USA Luke Lambourne, Michael A. Calderwood & Marc Vidal Luke Lambourne, Michael A. Calderwood & Marc Vidal Research Unit Signaling and Translation, Molecular Targets and Therapeutics Center (MTTC), Helmholtz Munich, German Research Center for Environmental Health, Neuherberg, Germany Franziska Ober, Marie Tofaute & Daniel Krappmann Institute of Translational Genomics, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany Institute of Computational Biology (ICB), Computational Health Center, Helmholtz Munich, German Research Center for Environmental Health, Neuherberg, Germany TERRA Teaching and Research Centre, University of Liège, Gembloux, Belgium Laboratory of Algal Synthetic and Systems Biology, Division of Science, New York University Abu Dhabi, Abu Dhabi, United Arab Emirates Microbe-Host Interactions, Faculty of Biology, Ludwig-Maximilians-Universität (LMU) München, Planegg-Martinsried, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the holdup and peptide assays. generated visualization and developed the figures. Correspondence to Andreas Zanzoni or Pascal Falter-Braun. The authors declare no competing interests. Nature Microbiology thanks Kenichi Tsuda, Tamas Korcsmaros and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Effector complements comparison of the 44 T3SS+ Pseudomonadota reference strains. Numbers indicate the count of shared effectors at > 90% mutual sequence similarity across 90% common sequence length among the indicated strains (Supplementary Data 1). b, Abundance of secretion systems in Pseudomonadota genomes among the 77 reference strains of human intestinal and stool samples, in a collection of 4,475 strains isolated from normal human guts (HBC/BIO-ML/GMC strains) and in meta-assembled genomes (MAG) of normal human guts. c, Similarity of identified 182 candidate effectors from the 770 T3SS+ MAGs with 1,195 effectors from pathogenic microbes across the range of alignment coverages. Full data for all panels in Supplementary Data 4. d, Cloning success: success rates of effector open reading frame (ORF) cloning for the indicated reference strains, and the number of obtained and sequence verified ORFs (on top of bars) (Supplementary Data 7). e, Luminescence from injection assays with Salmonella Typhimurium wt and ΔsctV strains expressing SipA, and Citrobacter pasteurii and Phytobacter massiliensis expressing the indicated effectors. Each data point represents a single technical replicate. f. Western blots showing expression of FLAG-tagged effectors in wt and ΔsctV S. Typhimurium in the indicated strains. a, Schematic of the multi-assay screening pipeline based on initial screening of bacterial ORFs against the human ORFeome 9.1. The primary screening involved screening against human protein pools, followed by retesting of positives, identification of candidate pairs by sequencing and final, independent four-fold verification. Full data in Supplementary Data 14. c, Co-immunoprecipitation of indicated Myc-tagged human proteins by indicated FLAG-tagged effectors or FLAG-GFP as negative control. Molecular weight markers are given in kDa. Light green dots: successful co-immunoprecipitation, but weak or no effector detection in lysate. a, Jaccard-interaction similarity of all interacting effector-pairs with at least three shared human interactors. Effector pairs marked with “H” share the same homology cluster. Clusters are color-coded according to the legend. b, Actual count of motif-domain pairs matching at least two stringency criteria identified in HuMMIMAIN (arrow) compared to n = 10,000 randomized control networks (empirical P = 0.0003). c, (Top) Distribution of residue–residue contacts across predicted aligned error (PAE) scores for interfaces between bacterial effectors and their human targets. (Bottom) Proportions of human–human protein pairs targeted by the same bacterial effector, grouped by interface similarity at different PAE thresholds (Jaccard Index (JI) categories: Distinct ≤ 0.1, Overlapping 0.1–0.6, Same ≥ 0.6). d, Top: Distribution of residue–residue contacts across predicted aligned error (PAE) scores for interfaces between human proteins and their bacterial effectors. Bottom: Proportions of bacterial effector–effector pairs targeting the same human protein, grouped by interface similarity at different PAE thresholds (JI categories: Distinct ≤ 0.1, Overlapping 0.1 – 0.6, Same ≥ 0.6). e, GO enrichment for convergence proteins. OR of functional annotations enriched among effector-targeted human proteins that are subject of convergence (FDR < 0.05, Fisher's exact test with Bonferroni FDR correction). Relative NF-κB transcriptional reporter activity of HEK293 cells expressing the indicated effectors under TNF-stimulated conditions (Kruskal-Wallis test with Dunn's post-hoc comparisons, * P < 0.05, ** P = 0.01, n = 4 biological replicates). Boxes represent IQR, with the bold black line representing the mean; whiskers indicate highest and lowest data point within 1.5 IQR. b, Representative anti-Hemagglutinin (HA) and anti-Flag (FLAG) Western blots showing expression of transfected effector proteins relative to actin control (ACT), which was run on a different blot. c. Representative proliferation curves of Caco-2 cells transfected with empty vector (EV) or Cpa_12 in basal conditions (unstim) or following pro-inflammatory stimulation (stim) over 72 h after sorting. Error bars indicate one standard deviation above and below the mean. d, Concentration of cytokines secreted by Caco-2 cells transfected with the indicated effectors in basal conditions (US) or following stimulation with the indicated elicitors. EV indicates empty vector mock control. Indicated P values calculated by Kruskal-Wallis test with Dunn's post-hoc comparisons (n = 5). Boxes represent IQR, with the bold black line representing the mean; whiskers indicate highest and lowest data point. Guide to Supplementary Data providing an overview of 26 thematically organized datasets, each introduced by a summary sheet describing the contents of every worksheet. Structural effector analysis and Foldseek clustering. Phylogenetic assignments of strains used in this study. Effector identifiers and abbreviations used in this study. Injection assays raw data and statistics. Interactions between pathogens effectors and human proteins downloaded from IMEXs. Holdup assay and validation of PBM–PDZ interface predictions. Genetic predisposition enrichment of effector targets. Prevalence of HuMMI effector proteins across the OhJ_2014 skin cohort. Genetic predisposition trait enrichment in effector target neighbourhoods identified by RWR. Prevalence of HuMMI effector proteins in IBD versus healthy patients. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Young, V., Dohai, B., Halder, H. et al. Effector–host interactome map links type III secretion systems in healthy gut microbiomes to immune modulation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41566-025-01833-8'>Optical coherent perfect absorption and amplification in a time-varying medium</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 10:28:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Photonics (2026)Cite this article Time-invariant photonic structures amplify or absorb light on the basis of their intrinsic material gain or loss. The coherent interference of multiple beams in space, for example, in a resonator, can be exploited to tailor the wave interaction with material gain or loss, respectively maximizing lasing or coherent perfect absorption. By contrast, a time-varying system is not bound to conserve energy, even in the absence of material gain or loss, and can support amplification or absorption of a probe wave through parametric phenomena. Here we demonstrate theoretically and experimentally how a subwavelength film of indium tin oxide, whose bulk permittivity is homogeneously and periodically modulated via optical pumping, can be dynamically tuned to act both as a non-resonant amplifier and as a perfect absorber, by manipulating the relative phase of two counterpropagating probe beams. This extends the concept of coherent perfect absorption to the temporal domain. We interpret this result as selective switching between the gain and loss modes present in the momentum bandgap of a periodically modulated medium. By tailoring the relative intensity of the two probes, high-contrast modulation can be achieved with up to 80% absorption and 400% amplification. Our results demonstrate control of gain and loss in time-varying media at optical frequencies and pave the way towards coherent manipulation of light in Floquet-engineered complex photonic systems. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Data used in this work are available from the corresponding authors upon reasonable request. Codes used in this work are available from the corresponding authors upon reasonable request. Yu, Z. & Fan, S. Complete optical isolation created by indirect interband photonic transitions. Google Scholar Fleury, R., Sounas, D. L., Sieck, C. F., Haberman, M. R. & Alù, A. Sound isolation and giant linear nonreciprocity in a compact acoustic circulator. Sounas, D. L. & Alù, A. Non-reciprocal photonics based on time modulation. Firestein, C., Shlivinski, A. & Hadad, Y. Absorption and scattering by a temporally switched lossy layer: going beyond the Rozanov bound. Solís, D. M. & Engheta, N. Functional analysis of the polarization response in linear time-varying media: a generalization of the Kramers–Kronig relations. Koutserimpas, T. T. & Monticone, F. Time-varying media, dispersion, and the principle of causality. Sisler, J. et al. Electrically tunable space–time metasurfaces at optical frequencies. Bruno, V. et al. Negative refraction in time-varying strongly coupled plasmonic-antenna–epsilon-near-zero systems. Lyubarov, M. et al. Amplified emission and lasing in photonic time crystals. Article MathSciNet Galiffi, E. et al. Photonics of time-varying media. Kishino, K. et al. Resonant cavity-enhanced (RCE) photodetectors. IEEE J. Quantum Electron. Chong, Y. D., Ge, L., Cao, H. & Stone, A. D. Coherent perfect absorbers: time-reversed lasers. Sweeney, W. R., Hsu, C. W., Rotter, S. & Stone, A. D. Perfectly absorbing exceptional points and chiral absorbers. Wang, C., Sweeney, W. R., Stone, A. D. & Yang, L. Coherent perfect absorption at an exceptional point. Roger, T. et al. Coherent perfect absorption in deeply subwavelength films in the single-photon regime. Hörner, H. et al. Coherent perfect absorption of arbitrary wavefronts at an exceptional point. Slobodkin, Y. et al. Massively degenerate coherent perfect absorber for arbitrary wavefronts. Baranov, D. G., Krasnok, A., Shegai, T., Alù, A. & Chong, Y. Coherent perfect absorbers: linear control of light with light. Siegman, A. E. Lasers (Univ. Science Books, 1986). Nikzamir, A., Rouhi, K., Figotin, A. & Capolino, F. Time modulation to manage and increase the power harvested from external vibrations. Galiffi, E. et al. Broadband coherent wave control through photonic collisions at time interfaces. Galiffi, E., Yin, S. & Alú, A. Tapered photonic switching. Ortega-Gomez, A., Lobet, M., Vázquez-Lozano, J. E. & Liberal, I. Tutorial on the conservation of momentum in photonic time-varying media. Morgenthaler, F. R. Velocity modulation of electromagnetic waves. Microwave Theory Tech. Mendonça, J. T. & Shukla, P. K. Time refraction and time reflection: two basic concepts. Bacot, V., Labousse, M., Eddi, A., Fink, M. & Fort, E. Time reversal and holography with spacetime transformations. Moussa, H. et al. Observation of temporal reflection and broadband frequency translation at photonic time interfaces. Lustig, E. et al. Time-refraction optics with single cycle modulation. Reyes-Ayona, J. R. & Halevi, P. Observation of genuine wave vector (k or β) gap in a dynamic transmission line and temporal photonic crystals. Lustig, E., Sharabi, Y. & Segev, M. Topological aspects of photonic time crystals. Cui, W. Y., Zhang, J., Luo, Y., Gao, X. & Cui, T. J. Dynamic switching from coherent perfect absorption to parametric amplification in a nonlinear spoof plasmonic waveguide. Joannopoulos, J. D., Villeneuve, P. R. & Fan, S. Photonic crystals. Solid State Commun. Khurgin, J. B. Photonic time crystals and parametric amplification: similarity and distinction. ACS Photonics 11, 2150–2159 (2023). Boyd, R. W., Gaeta, A. L., & Giese, E. in Springer Handbook of Atomic, Molecular, and Optical Physics 1097–1110 (Springer, 2008). Alam, M. Z., De Leon, I. & Boyd, R. W. Large optical nonlinearity of indium tin oxide in its epsilon-near-zero region. Bohn, J. et al. All-optical switching of an epsilon-near-zero plasmon resonance in indium tin oxide. Vezzoli, S. et al. Optical time reversal from time-dependent epsilon-near-zero media. Tirole, R. et al. Double-slit time diffraction at optical frequencies. Un, I. W., Sarkar, S. & Sivan, Y. Electronic-based model of the optical nonlinearity of low-electron-density Drude materials. Narimanov, E. E. Ultrafast optical modulation by virtual interband transitions. ACS Photonics 12, 402–408 (2024). Tirole, R. et al. Saturable time-varying mirror based on an epsilon-near-zero material. Lobet, M. et al. New horizons in near-zero refractive index photonics and hyperbolic metamaterials. ACS Photonics 10, 3805–-3820 (2023). Pendry, J. An avalanche model for femtosecond optical response. Preprint at https://arxiv.org/abs/2407.08391 (2024). Pendry, J. Time reversal and negative refraction. Article MathSciNet Miyamaru, F. et al. Ultrafast frequency-shift dynamics at temporal boundary induced by structural-dispersion switching of waveguides. Khurgin, J. B., Clerici, M. & Kinsey, N. Fast and slow nonlinearities in epsilon-near-zero materials. Laser Photonics Rev. Horsley, S. A. R., Galiffi, E. & Wang, Y. T. Eigenpulses of dispersive time-varying media. Article MathSciNet Cotrufo, M., Cordaro, A., Sounas, D. L., Polman, A. & Alù, A. Passive bias-free non-reciprocal metasurfaces based on thermally nonlinear quasi-bound states in the continuum. Download references A.A., R.T. and E.G. were partially supported by the US Department of Defense and the Simons Collaboration on Extreme Wave Phenomena. E.G. acknowledges funding from the Simons Foundation through a Junior Fellowship of the Simons Society of Fellows (855344/EG). acknowledges support from the Val O'Donoghue Scholarship in Natural Sciences. acknowledge support from UKRI (EP/Y015673). are supported by the European Research Council (ERC) (Grant agreement No. These authors contributed equally: Emanuele Galiffi, Anthony C. Harwood. Chandra Family Department of Electrical and Computer Engineering, University of Texas at Austin, Austin, TX, USA Emanuele Galiffi Photonics Initiative, Advanced Science Research Center, City University of New York, New York, NY, USA Emanuele Galiffi, Romain Tirole & Andrea Alù The Blackett Laboratory, Imperial College London, South Kensington Campus, London, UK Anthony C. Harwood, Stefano Vezzoli, Romain Tirole & Riccardo Sapienza Physics Program, Graduate Center, City University of New York, New York, NY, USA Andrea Alù Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar E.G. devised the concept and developed the theoretical model. and R.T. built the experimental set-up and carried out experimental measurements. and E.G. carried out data analysis and model fitting. and A.A. supervised the project. All authors contributed to scientific discussions and preparation of the manuscript. Correspondence to Emanuele Galiffi, Andrea Alù or Riccardo Sapienza. The authors declare no competing interests. Nature Photonics thanks Stefan Rotter and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. Supplementary discussions of (1) theory for temporal coherent control in ITO films; (2) coherent modulation for different wavelengths and fluctuation sources; (3) pump power dependence; and (4) transmission and negative refraction from the back of the sample. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Galiffi, E., Harwood, A.C., Vezzoli, S. et al. Optical coherent perfect absorption and amplification in a time-varying medium. Download citation Accepted: 11 December 2025 Published: 26 January 2026 Version of record: 26 January 2026 DOI: https://doi.org/10.1038/s41566-025-01833-8 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Photonics ISSN 1749-4893 (online) ISSN 1749-4885 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41893-025-01754-y'>Global gridded dataset of heating and cooling degree days under climate change scenarios</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-26 10:05:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Accurate projections of heating and cooling demands are crucial for advancing towards the sustainable development goals. Here we present a global dataset of heating degree days (HDDs) and cooling degree days (CDDs) for 3 levels of global mean temperature rise above pre-industrial conditions—1.0 °C (2006–2016), 1.5 °C and 2.0 °C—regardless of the pathways leading to these warming scenarios. The dataset comprises 30 gridded maps (0.883° × 0.556° resolution) characterizing climate variability through 5 statistical metrics per variable and scenario over a representative 10-year period. The dataset reveals a widespread decline in HDDs and a pronounced, nonlinear increase in CDDs, with the most significant shifts in climate intensity and adaptation needs emerging early in the warming trajectory. Furthermore, using the ‘middle-of-the-road' Shared Socioeconomic Pathway 2-4.5 as a reference, the dataset indicates that the population experiencing extreme heat conditions (exceeding 3,000 CDDs) is projected to nearly double if the 2.0 °C threshold is reached, increasing from 23% (1.54 billion people) in 2010 to 41% (3.79 billion) by 2050, with the largest projected populations affected in India, Nigeria, Indonesia, Bangladesh, Pakistan and the Philippines. This HDD–CDD dataset provides a robust foundation for integrating climate information into sustainability planning and development policy. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The global gridded dataset of HDDs and CDDs under the three climate change scenarios (1 °C, 1.5 °C and 2 °C) is available in the Oxford University Research Archive (ORA) repository at https://doi.org/10.5287/ora-w4qpqy522. Five maps are available for HDDs and CDDs per scenario: mean, median, 10th percentile, 90th percentile and standard deviation. The complete list of maps for each global warming level is provided in Extended Data Table 1. The spatial resolution is 0.833° × 0.556° (longitude latitude) over the land surface. Further data are available from the authors on request. The code to calculate HDDs and CDDs from the temperature ensemble is available via Github at https://github.com/lizanafj/cdd_hdd_mapping. The code for data visualization and statistical analysis is available via Github at https://github.com/lizanafj/python_examples_with_CDDandHDD_files. Examples of how to use the Python code are provided in Supplementary Note 7. IPCC Climate Change 2022: Mitigation of Climate Change (eds Shukla, P. R. et al.) 953–1048 (Cambridge Univ. Khosla, R. et al. Cooling for sustainable development. Abergel, T. & Delmastro, C. Is cooling the future of heating? IEA The Future of Cooling. Opportunities for Energy-Efficient Air Conditioning (IEA Publications, 2018). Reinhart, C. Linking energy use to local climate. Global Status Report for Buildings and Construction: Beyond Foundations: Mainstreaming Sustainable Solutions to Cut Emissions from the Buildings Sector (UNEP, 2024); https://doi.org/10.59117/20.500.11822/45095 Staffell, I., Pfenninger, S. & Johnson, N. A global model of hourly space heating and cooling demand at multiple spatial scales. Degree-Days: Theory and Application - TM41: 2006 (CIBSE, 2006). Biardeau, L. T., Davis, L. W., Gertler, P. & Wolfram, C. Heat exposure and global air conditioning. Mistry, M. N. Historical global gridded degree-days: a high-spatial resolution database of CDD and HDD. & Caldeira, K. Impacts of global warming on residential heating and cooling degree-days in the United States. Spinoni, J. et al. Changes of heating and cooling degree-days in Europe from 1981 to 2100. Almazroui, M., Saeed, S., Saeed, F., Islam, M. N. & Ismail, M. Projections of precipitation and temperature over the South Asian countries in CMIP6. Almazroui, M. et al. Projected change in temperature and precipitation over Africa from CMIP6. Deroubaix, A. et al. Large uncertainties in trends of energy demand for heating and cooling under climate change. Spinoni, J. et al. Global population-weighted degree-day projections for a combination of climate and socio-economic scenarios. Lizana, J. et al. Ensemble of global climate simulations for temperature in historical, 1.5 °C and 2.0 °C scenarios from HadAM4. Lizana, J. et al. Large ensemble of global mean temperatures: 6-hourly HadAM4 model run data using the Climateprediction.net platform. CEDA Archive https://doi.org/10.5285/9c41e3aa67024bbdad796290a861e968 (2023). Watson, P. et al. Multi-thousand member ensemble atmospheric simulations with global 60km resolution using climateprediction.net. Copernicus Meetings https://doi.org/10.5194/egusphere-egu2020-10895 (2023). Leach, N. J., Watson, P. A. G., Sparrow, S. N., Wallom, D. C. H. & Sexton, D. M. H. Generating samples of extreme winters to support climate adaptation. Mitchell, D. et al. Half a degree additional warming, prognosis and projected impacts (HAPPI): background and experimental design. Miranda, N. D. et al. Change in cooling degree days with global mean temperature rise increasing from 1.5 °C to 2.0 °C. Lizana, J. et al. Global land surface dataset of heating and cooling degree days from a bias-corrected HadAM4-based temperature ensemble under 1.0 °C, 1.5 °C, and 2.0 °C climate scenarios. IPCC Climate Change 2022: Impacts, Adaptation, and Vulnerability (Cambridge Univ. KC, S. & Lutz, W. The human core of the shared socioeconomic pathways: population scenarios by age, sex and level of education for all countries to 2100. Mehmood, S., Lizana, J., Núñez-Peiró, M., Maximov, S. A. & Friedrich, D. Resilient cooling pathway for extremely hot climates in southern Asia. Lizana, J. et al. Overcoming the incumbency and barriers to sustainable cooling. Franken, L. et al. Power system benefits of simultaneous domestic transport and heating demand flexibility in Great Britain's energy transition. Halloran, C., Lizana, J., Fele, F. & Mcculloch, M. Data-based, high spatiotemporal resolution heat pump demand for power system planning. Lizana, J., Ortiz, C., Soltero, V. M. & Chacartegui, R. District heating systems based on low-carbon energy technologies in Mediterranean areas. Ballester, J. et al. Heat-related mortality in Europe during the summer of 2022. Webb, M., Senior, C., Bony, S. & Morcrette, J.-J. Combining ERBE and ISCCP data to assess clouds in the Hadley Centre, ECMWF and LMD atmospheric climate models. Williams, K. D., Ringer, M. A. Evaluating the cloud response to climate change and current climate variability. Hersbach, H. et al. ERA5 hourly data on single levels from 1940 to present. Copernicus Climate Change Service (C3S) Climate Data Store https://doi.org/10.24381/cds.adbb2d47 (2023). Wehner, M. et al. Changes in extremely hot days under stabilized 1.5 and 2.0 °C global warming scenarios as simulated by the HAPPI multi-model ensemble. Wang, X., Meng, X. Projecting 1 km-grid population distributions from 2020 to 2100 globally under shared socioeconomic pathways. Wang, X., Meng, X. Projecting 1 km-grid population distributions from 2020 to 2100 globally under shared socioeconomic pathways. Labriet, M. et al. Worldwide impacts of climate change on energy for heating and cooling. Isaac, M. & van Vuuren, D. P. Modeling global residential sector energy demand for heating and air conditioning in the context of climate change. Andrijevic, M., Byers, E., Mastrucci, A., Smits, J. & Fuss, S. Future cooling gap in shared socioeconomic pathways. Sivak, M. Where to live in the United States: combined energy demand for heating and cooling in the 50 largest metropolitan areas. This research received support from the University of Oxford Strategic Research Fund and the Oxford Martin School's Future of Cooling Programme. acknowledges funding from the European Union's Horizon 2020 research and innovation programme (Marie Skłodowska-Curie grant agreement no. was funded by UK Research and Innovation (UKRI, NE/P002099/1). To ensure open access, we have granted a CC BY licence to any Author Accepted Manuscript resulting from this work. We are also grateful to M. Z. Walker and R. Renaldi for their valuable contributions to the early stages of the research concept. ZERO Institute, University of Oxford, Oxford, UK Jesus Lizana, David C. H. Wallom, Radhika Khosla & Malcolm McCulloch Future of Cooling Programme, Oxford Martin School, University of Oxford, Oxford, UK Jesus Lizana, Nicole D. Miranda, Radhika Khosla & Malcolm McCulloch Energy and Power Group, Department of Engineering Science, University of Oxford, Oxford, UK Jesus Lizana, Nicole D. Miranda & Malcolm McCulloch Oxford e-Research Centre, University of Oxford, Oxford, UK Sarah N. Sparrow & David C. H. Wallom Smith School of Enterprise and the Environment, School of Geography and the Environment, University of Oxford, Oxford, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceptualized the work proposed. coordinated the study. performed the data extraction and data management. performed the data pre-processing of the climate model. calculated and generated the CDD and HDD datasets. provided the statistics and visualizations available in the paper. wrote the paper draft. led the interpretation and analysis of the data. provided expertise in data analytics. All authors reviewed the paper. Correspondence to Jesus Lizana. The authors declare no competing interests. Nature Sustainability thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Absolute change in HDD (Delta HDD) between the 1.0 °C and 1.5 °C scenario. b, Absolute change in HDD (Delta HDD) between 1.5 °C and 2.0 °C. c, Absolute change in HDD (Delta HDD) between 1.0 °C and 2.0 °C. Delta HDD (ΔHDD) refers to the incremental/decremental change in mean annual HDD per grid. Administrative boundary data © EuroGeographics 2025. a, Absolute change in CDD (Delta CDD) from 1.0 °C to 1.5 °C scenario. b, Absolute change in CDD (Delta CDD) from 1.5 °C to 2.0 °C scenario. c, Absolute change in CDD (Delta CDD) between 1.0 °C and 2.0 °C. Delta CDD (ΔCDD) refers to the incremental/decremental change in mean annual CDD per grid. Administrative boundary data © EuroGeographics 2025. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Lizana, J., Miranda, N.D., Sparrow, S.N. Global gridded dataset of heating and cooling degree days under climate change scenarios. Version of record: 26 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            