
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-02</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-04181-w'>Time-of-day immunochemotherapy in nonsmall cell lung cancer: a randomized phase 3 trial</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 16:13:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Retrospective studies suggest that early time-of-day (ToD) infusions of immunochemotherapy may improve efficacy. However, prospective randomized controlled trials are needed to validate it. In this randomized phase 3 LungTIME-C01 trial, 210 patients with treatment naive stage IIIC–IV nonsmall cell lung cancer (NSCLC) lacking driver mutations were randomly assigned in a 1:1 ratio to either an early or late ToD group, defined by the administration of the first four cycles of an anti-PD-1 agent before or after 15:00 h. The primary endpoint was progression-free survival (PFS), while secondary endpoints included overall survival (OS) and objective response rate (ORR). After a median follow-up of 28.7 months, the median PFS was 11.3 months (95% confidence interval (CI) = 9.2–13.4) in the early ToD group and 5.7 months (95% CI = 5.2–6.2) in the late ToD group, corresponding to a hazard ratio (HR) for earlier disease progression of 0.40 (95% CI = 0.29–0.55; P < 0.001). The median OS was 28.0 months (95% CI = not estimable (NE)–NE) in the early ToD group and 16.8 months (95% CI = 13.7–19.9) in the late ToD group, corresponding to an HR of an earlier death of 0.42 (95% CI = 0.29–0.60; P < 0.001). Treatment-related adverse events were consistent with the established safety profile, with no new safety signals observed. No significant differences in immune-related adverse events were observed between the two groups. Over the first four cycles, morning circulating CD8+ T cells increased in the early ToD group, whereas they declined in the late ToD group (P < 0.001). Furthermore, the ratio of activated (CD38+ HLA-DR+) versus exhausted (TIM-3+PD-1+) CD8+ T cells was higher in the early ToD group (P < 0.001) compared with the late ToD group (P < 0.001). In summary, our study indicates that early ToD immunochemotherapy substantially improves PFS and OS and is associated with enhanced antitumor CD8+ T cell characteristics compared with late ToD treatment. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Due to concerns regarding patient privacy and institutional data governance, the clinical datasets generated or used in this study are not publicly accessible. To protect the confidentiality of patients, de-identified individual-level data may be made available upon reasonable request. Researchers interested in accessing the data should contact Y.Z. All inquiries will be addressed within approximately 10 weeks. Each request will undergo evaluation by the data oversight committee of Hunan Cancer Hospital to assess compliance with confidentiality policies and potential intellectual property constraints. The role of chemotherapy plus immune checkpoint inhibitors in oncogenic-driven NSCLC: a University of California lung cancer consortium retrospective study. & Hellmann, M. D. First-line immunotherapy for non-small-cell lung cancer. Martinez, P., Peters, S., Stammers, T. & Soria, J. C. Immunotherapy for the first-line treatment of patients with metastatic non-small cell lung cancer. Five-year outcomes with pembrolizumab versus chemotherapy for metastatic non-small-cell lung cancer with PD-L1 tumor proportion score >/= 50. Huang, M. Y., Jiang, X. M., Wang, B. L., Sun, Y. Combination therapy with PD-1/PD-L1 blockade in non-small cell lung cancer: strategies and mechanisms. Meyer, M. L. et al. New promises and challenges in the treatment of advanced non-small-cell lung cancer. Kisamore, C. O., Elliott, B. D., DeVries, A. C., Nelson, R. J. & Walker, W. H. II Chronotherapeutics for solid tumors. Cermakian, N. & Labrecque, N. Regulation of cytotoxic CD8+ T cells by the circadian clock. & Blanco, J. R. Chronotherapy: circadian rhythms and their influence in cancer therapy. Influence of circadian clocks on adaptive immunity and vaccination responses. & Dang, C. V. Clocking cancer immunotherapy responses. Qian, D. C. et al. Effect of immunotherapy time-of-day infusion on overall survival among patients with advanced melanoma in the USA (MEMOIR): a propensity score-matched analysis of a single-centre, longitudinal study. Patel, J. S. et al. Impact of immunotherapy time-of-day infusion on survival and immunologic correlates in patients with metastatic renal cell carcinoma: a multicenter cohort analysis. Ruiz-Torres, D. A. et al. Immunotherapy time of infusion impacts survival in head and neck cancer: a propensity score matched analysis. Timing of the infusion of nivolumab for patients with recurrent or metastatic squamous cell carcinoma of the esophagus influences its efficacy. Hirata, T. et al. Brief report: clinical outcomes by infusion timing of immune checkpoint inhibitors in patients with locally advanced NSCLC. Dizman, N. et al. Association between time-of-day of immune checkpoint blockade administration and outcomes in metastatic renal cell carcinoma. Landré, T. et al. Effect of immunotherapy-infusion time of day on survival of patients with advanced cancers: a study-level meta-analysis. Karaboué, A. et al. Why does circadian timing of administration matter for immune checkpoint inhibitors' efficacy? Cortellini, A. et al. A multicentre study of pembrolizumab time-of-day infusion patterns and clinical outcomes in non-small-cell lung cancer: too soon to promote morning infusions. Rousseau, A. et al. Clinical outcomes by infusion timing of immune checkpoint inhibitors in patients with advanced non-small cell lung cancer. Goss, G. et al. LBA48 CCTG BR.31: a global, double-blind placebo-controlled, randomized phase III study of adjuvant durvalumab in completely resected non-small cell lung cancer (NSCLC). Adjuvant atezolizumab after adjuvant chemotherapy in resected stage IB–IIIA non-small-cell lung cancer (IMpower010): a randomised, multicentre, open-label, phase 3 trial. O'Brien, M. et al. Pembrolizumab versus placebo as adjuvant therapy for completely resected stage IB–IIIA non-small-cell lung cancer (PEARLS/KEYNOTE–091): an interim analysis of a randomised, triple-blind, phase 3 trial. Overall survival according to time-of-day of combined immuno-chemotherapy for advanced non-small cell lung cancer: a bicentric bicontinental study. Peripheral blood lymphocyte subsets predict the efficacy of immune checkpoint inhibitors in non-small cell lung cancer. Changes in T lymphocyte subsets predict the efficacy of atezolizumab in advanced non-small cell lung cancer: a retrospective study. Yost, K. E. et al. Clonal replacement of tumor-specific T cells following PD-1 blockade. Zhang, J. et al. Compartmental analysis of T-cell clonal dynamics as a function of pathologic response to neoadjuvant PD-1 blockade in resectable non-small cell lung cancer. Marcos Rubio, A., Everaert, C., Van Damme, E., De Preter, K. & Vermaelen, K. Circulating immune cell dynamics as outcome predictors for immunotherapy in non-small cell lung cancer. Systemic immunity is required for effective cancer immunotherapy. Wang, C. et al. Circadian tumor infiltration and function of CD8+ T cells dictate immunotherapy efficacy. Fortin, B. M. et al. Circadian control of tumor immunosuppression affects efficacy of immune checkpoint blockade. Riely, G. J. et al. Non-small cell lung cancer, version 4.2024, NCCN clinical practice guidelines in oncology. Kazandjian, D., Gong, Y., Keegan, P., Pazdur, R. & Blumenthal, G. M. Prognostic value of the lung immune prognostic index for patients treated for metastatic non-small cell lung cancer. Yeung, C., Kartolo, A., Tong, J., Hopman, W. & Baetz, T. Association of circadian timing of initial infusions of immune checkpoint inhibitors with survival in advanced melanoma. Vilalta, A. et al. 967P The time of anti-PD-1 infusion improves survival outcomes by fasting conditions simulation in non-small cell lung cancer. Seymour, L. et al. iRECIST: guidelines for response criteria for use in trials testing immunotherapeutics. Grambsch, P. M. & Therneau, T. M. Proportional hazards tests and diagnostics based on weighted residuals. Steinitz, O., Heller, J., Tsoar, A., Rotem, D. & Kadmon, R. Predicting regional patterns of similarity in species composition for conservation planning. Nekola, J. C. & White, P. S. The distance decay of similarity in biogeography and ecology. We are deeply grateful to all the patients and their families who participated in this study. This work received financial support from the National Natural Science Foundation of China (grants 82222048 and 82173338 to Y.Z. These authors contributed equally: Zhe Huang, Liang Zeng, Zhaohui Ruan, Qun Zeng. Early Phase Clinical Trial Center, Department of Investigational Cancer Therapeutics, Hunan Cancer Hospital/The Affiliated Cancer Hospital of Xiangya School of Medicine, Central South University, Changsha, China Zhe Huang, Liang Zeng, Zhaohui Ruan, Huan Yan, Jiacheng Dai, Nachuan Zou, Shidong Xu, Jun Deng, Xue Chen, Jing Wang & Yongchang Zhang Department of Pathology, School of Basic Medical Science, Central South University, Changsha, China Zhe Huang, Shidong Xu & Yongchang Zhang Third Xiangya Hospital, Central South University, Changsha, China Department of Medical Oncology, Lung Cancer and Gastrointestinal Unit, Hunan Cancer Hospital/The Affiliated Cancer Hospital of Xiangya School of Medicine, Central South University, Changsha, China Wenjuan Jiang, Yi Xiong, Chunhua Zhou, Haiyan Yang, Li Liu, Ya Wang, Zhan Wang, Nong Yang & Yongchang Zhang Department of Interventional Vascular Surgery, Hunan Cancer Hospital/The Affiliated Cancer Hospital of Xiangya School of Medicine, Central South University, Changsha, China Research Unit ‘Chronotherapy, Cancer, Transplantation', Faculty of Medicine, Paris-Saclay University, Hospital Paul Brousse, Villejuif, France Xiaomei Li, Boris Duchemann & Francis Lévi Thoracic and Medical Oncology Unit, Avicenne Hospital, Assistance Publique–Hôpitaux de Paris, Bobigny, France Department of Clinical Oncology, State Key Laboratory of Translational Oncology Hong Kong, Hong Kong, China Translational Research Centre in Onco-Hematology (CRTOH), Geneva, Switzerland Biomedical Center (BMC), Institute for Cardiovascular Physiology and Pathophysiology, Walter Brendel Center for Experimental Medicine (WBex), Faculty of Medicine, Ludwig-Maximilians-Universität (LMU) Munich, Planegg-Martinsried, Germany Gastro-Intestinal and Medical Oncology Department, Paul Brousse Hospital, Assistance Publique-Hôpitaux de Paris, Villejuif, France Hunan Second People's Hospital, Changsha, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the study, contributed to the analysis and interpretation of the data, manuscript writing and development of figures and tables. Yan., W.J., Y. Xiong, C.Z., H. Yang, L.L., J. Dai, N.Z., S.X., Y.W., Z.W., J. Deng and X.C. collected the data, contributed to the analysis and interpretation of the data and to manuscript review and revision. conceived the study, contributed to all study progress and development, contributed to methods, results, interpretation and manuscript writing. conceived the study, contributed to all study progress and development, contributed to methods, results, interpretation and manuscript writing. codirected this study, including conception, organization, data collection, auditing, supervision, project management, funding acquisition, writing and editing the manuscript. All authors approved the current manuscript. Correspondence to Tony Mok, Christoph Scheiermann, Francis Lévi, Nong Yang or Yongchang Zhang. The authors declare no competing interests. Nature Medicine thanks Benjamin Creelan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ulrike Harjes, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Distribution of first 4 infusion times among 210 patients, who were divided into early time-of-day (ToD) infusion group and late ToD group. (b) Histogram of median times of the first 4 infusions per patient (n = 210). (a) Forest plots of the univariate and multivariate Cox regression results for progression-free survival (PFS) (n = 210). (b) Forest plots of the univariate and multivariate Cox regression results for overall survival (OS) (n = 210). P values (two-sided), hazard ratios (HRs), and 95% confidence intervals of HRs were estimated using univariable or multivariable Cox proportional hazards models, and P values were not adjusted for multiple comparisons. Data are presented as HR (points) with 95% CIs (horizontal lines). LUSC, lung squamous cell carcinoma. ECOG PS, Eastern Cooperative Oncology Group Performance Status. LIPI, Lung Immune Prognostic Index. The tumor response was assessed by a blinded independent review committee (BIRC) (n = 210). P values were determined using a two-sided chi-square test. Patient values are normalized to individual baseline levels and assessed after 2 cycles (prior to cycle 3) and 4 cycles (prior to cycle 5) of treatment. Line-point graphs depict dynamic changes of CD3+ T cell proportion (a), CD8+ T cell proportion (b), CD4+ T cell proportion (f), B cell proportion (g), NK cell proportion (h) and CD8+/CD4+ T cell ratio (i) in individual patients from the early and late time-of-day (ToD) groups. Colored lines link sequential measurements from individual patients. Linear regressions (solid lines) with shaded 95% confidence intervals illustrate changes in CD4+ T cell proportions (c), B cell proportions (d) and NK cell proportions (e) over time in patients from the early and late time-of-day (ToD) groups. Data are presented as mean ± s.e. Dotted horizontal lines indicate the normalized baseline (ratio = 1.0). P values were determined using a permutation test (two-sided) and two-way repeated-measures ANOVA (two-sided), without adjustment for multiple comparisons. Flow cytometric analyses of CD4+, B and NK cells were performed on paired blood samples collected at baseline, after 2 cycles and after 4 cycles from 61 patients in the early ToD group and 44 patients in the late ToD group (n = 105 total patients; n = 315 total samples). Representative flow cytometry gating strategy used to identify CD38+HLA-DR+CD8+ T cells and TIM-3+PD-1+CD8+ T cells from peripheral blood mononuclear cells (PBMCs) (a). Linear regressions (solid lines) with shaded 95% confidence intervals illustrate changes in CD38+ HLA-DR+ CD8+ T cell proportions (b). Data are presented as mean ± s.e.m. Dotted horizontal lines indicate the normalized baseline (ratio = 1.0). P values were determined using a permutation test (two-sided) and two-way repeated-measures ANOVA (two-sided), without adjustment for multiple comparisons. Line-point graphs depict dynamic changes of CD38+ HLA-DR+ CD8+ T cell (c), TIM-3+ PD-1+ CD8+ T cell proportion (d) and CD38+ HLA-DR+/ TIM-3+ PD-1+ CD8+ T cell ratio (e) in individual patients from the early and late time-of-day (ToD) groups. Colored lines connect serial measurements from the same patient. Dotted horizontal lines indicate the normalized baseline (ratio = 1.0). PBMCs, peripheral blood mononuclear cells. Flow cytometric analyses of CD3+, CD4+, CD8+ T, B and NK cells were performed on paired blood samples collected at baseline, after 2 cycles and after 4 cycles from 61 patients in the early ToD group and 44 patients in the late ToD group (n = 105 total patients; n = 315 total samples). CD38+ HLA-DR+ CD8+ T cells and TIM-3+ PD-1+ CD8+ T cells were assessed in paired cryopreserved PBMCs collected at baseline, after 2 cycles and after 4 cycles from 14 patients in the early ToD group and 25 patients in the late ToD group (n = 39 total patients; n = 117 total samples). Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Huang, Z., Zeng, L., Ruan, Z. et al. Time-of-day immunochemotherapy in nonsmall cell lung cancer: a randomized phase 3 trial. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260201231230.htm'>Hundreds of new species found in a hidden world beneath the Pacific</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 15:25:04
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new international study has revealed that this push toward deep-sea mining may have less overall environmental impact than scientists once feared. Hundreds of previously unknown species were found living nearly 4,000 meters below the ocean's surface, highlighting just how little is known about these remote environments. Several of these metals are found in large quantities on the deep-sea floor, but until now, no one has shown how they can be extracted or what environmental impact this would have," says marine biologist Thomas Dahlgren, that together with Helena Wiklund, also at the University of Gothenburg, have participated in the research project. Over five years, researchers cataloged marine life and tested mining impacts in the Clarion-Clipperton Zone, a vast region of the Pacific Ocean located between Mexico and Hawaii. The results showed that areas directly disturbed by mining equipment experienced a 37 percent decline in animal numbers and a 32 percent reduction in species diversity. "The research required 160 days at sea and five years of work. Our study will be important for the International Seabed Authority (ISA), which regulates mineral mining in international waters," says Thomas Dahlgren. To put the scarcity of life into perspective, a single seafloor sample from the North Sea can contain up to 20,000 animals. A similar sample from the deep Pacific seabed contains roughly the same number of species, but only about 200 individual animals. Researchers collected 4,350 animals larger than 0.3 mm living in and on the seabed. The team also identified a new solitaire coral, that are described in another study. "I have been working in the Clarion-Clipperton Zone for over 13 years, and this is by far the largest study that has been conducted. In Gothenburg, we led the identification of marine polychaete worms. Since most species have not been described previously, molecular (DNA) data was crucial in facilitating studies of biodiversity and ecology on the seabed," says Thomas Dahlgren. As the inventory progressed, scientists noticed that deep-sea communities changed naturally over time, likely in response to shifts in how much food reached the ocean floor. However, researchers still do not know how widely these species are distributed across the Pacific's deep-sea regions. At present, we have virtually no idea what lives there," says Adrian Glover, senior author from the Natural History Museum of London. Note: Content may be edited for style and length. Here's What Experts Want You To Know About Tinnitus Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a70177459/secret-maps-lost-trade-route/'>Scholars Were Analyzing Secret Army Maps—And Found a Lost Trade Route</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These maps detailed the Great Mongolian Road, which included key settlements and important water sources. Now, a secret set of century-old Japanese military maps has revealed the route once more—and in striking detail. Between 1873 and 1945, the Japanese Imperial Army created gaihōzu, which roughly translates to “maps of outer lands.” Originally devised for military planning during Japan's imperial expansion efforts, they still collectively serve as one of the most comprehensive cartographic records of east and inner Asia. In a new study published in the Journal of Historical Geography, a team of researchers said the gaihōzu display rich detail of many territories beyond Japanese control and “now serve as geographical time capsules, preserving landscapes since transformed by modernization.” The research team had four map sheets from the Toa Yochizu—the “Maps of East Asia,” which were drawn between 1901 and 1922—highlighting a region starting near Mongolia's eastern border with China and moving west. The painstaking detail in the maps highlights everything from road locations and settlements right down to oases and water wells. “The gaihōzu capture not merely routes but complete support systems, including water sources, terrain features, and settlements vital for navigation and survival in these harsh arid environments,” the study authors wrote. “By mapping this historical corridor, these once-secret military documents provide valuable baseline data for historical geography, cultural heritage preservation, and environmental change assessment across the landscapes of Asia.” Field research showed that the route connecting northern China with Central Asia included water sources spaced roughly 15 miles apart, which was the typical distance a camel caravan would have traveled in a day. “Although they were created to serve imperial ambitions, these maps have transcended their original purpose to become irreplaceable records of human geographic knowledge.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. This Set of Fossils Could Be Our Oldest Ancestor</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a70165451/life-fukushima-reactors/'>Something Strange Is Living—and Thriving—Below Fukushima's Dead Reactors</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ionizing radiation apparently didn't prevent some types of bacteria from breeding in the water, but astonishingly, they are not the radiation-resistant types expected in an environment like this. We may earn commission if you buy from a link. Following the 2011 tsunami that caused massive meltdowns in the Fukushima Daiichi Nuclear Power Station, Japan shut down all of its nuclear operations. Soon, an environment assumed to be uninhabitable was actually crawling with microbes. Microbes can be a major obstacle during the cleanup that comes with decommissioning nuclear power plants, since many species corrode metal, and swarms of them can turn water murky and reduce visibility. Recently, biologists Tomoro Warashina and Akio Kanai from Keio University in Tokyo made an extraordinary discovery when they analyzed samples of microbes taken from the highly radioactive water inside the power plant's torus room, below the reactor building. In the face of a nuclear disaster, organisms either perish or evolve. Mutations have made it possible for everything from wolves to a nearly indestructible black mold to thrive in the otherwise hostile environment of Chernobyl even decades later. This is why the scientists expected to find radiation-resistant microbial species such as Deinococcus radiodurans or Methylobacterium radiotolerans in their samples. As the team explained in a study recently published in Applied and Environmental Microbiology, the microbiome they analyzed had been exposed to persistent radiation, and gathering information about such microorganisms is vital for understanding how to deal sustainably with stagnant, radioactive water environments during decommissioning work. After testing their water samples for genetic markers of different microbes, Warashina and Kanai discovered that it was teeming with bacteria from the Limnobacter and Brevirhabdus genera. These chemolithotrophic bacteria make their living by oxidizing inorganic compounds such as manganese or sulfides. Sulfur oxidizers benefit from the sulfite oxidase enzyme, which is secreted by their mitochondria and detoxes sulfides by breaking down amino acids containing sulfur. The scientists also found a smaller amount of iron oxidizers from the Hoeflea and Sphinopyxis genera, which live by converting one form of iron to another. None of the species found by Kanai and Warashina have superpowers for radiation resistance. Yet despite high levels of ionizing radiation that would have been toxic to many other forms of life, these bacteria were able to thrive. Metals are commonly oxidized and corroded by these bacteria, and the scientists reasoned that it's possible the slime covering such bacterial masses may have actually given them extra protection from radiation. “In contrast, [most] of the bacterial genera in the torus room water were associated with metal corrosion, indicating that the impact of bacteria on metal corrosion must be considered in long-term decommissioning work.” It is possible that these bacteria crashed in with the waves of the tsunami or that something about their adaptations to a marine environment also helps them stay alive in the bowels of a dead nuclear reactor. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Are There More Humans on Earth Than We Thought? Archaeologists Found a Set of Legendary Gold Armor Chinese Tomb Mural from 8th Century Has Blonde Man</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00285-6'>Does AI already have human-level intelligence? The evidence is clear</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 12:04:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Eddy Keming Chen is an associate professor of philosophy at the University of California, San Diego, San Diego, California, USA. Mikhail Belkin is a professor of artificial intelligence, data science, and computer science at the University of California, San Diego, San Diego, California, USA. Leon Bergen is an associate professor of linguistics and computer science at the University of California, San Diego, USA, and a member of technical staff at Goodfire AI in San Francisco, California, USA. David Danks is a professor of data science, philosophy and policy at the University of California, San Diego, San Diego, California, USA. Now known as the Turing test, it addressed a question that seemed purely hypothetical: could machines display the kind of flexible, general cognitive competence that is characteristic of human thought, such that they could pass themselves off as humans to unaware humans? Three-quarters of a century later, the answer looks like ‘yes'. In March 2025, the large language model (LLM) GPT-4.5, developed by OpenAI in San Francisco, California, was judged by humans in a Turing test to be human 73% of the time — more often than actual humans were2. LLMs have achieved gold-medal performance at the International Mathematical Olympiad, collaborated with leading mathematicians to prove theorems4, generated scientific hypotheses that have been validated in experiments5, solved problems from PhD exams, assisted professional programmers in writing code, composed poetry and much more — including chatting 24/7 with hundreds of millions of people around the world. In other words, LLMs have shown many signs of the sort of broad, flexible cognitive competence that was Turing's focus — what we now call ‘general intelligence', although Turing did not use the term. Yet many experts baulk at saying that current AI models display artificial general intelligence (AGI) — and some doubt that they ever will. A March 2025 survey by the Association for the Advancement of Artificial Intelligence in Washington DC found that 76% of leading researchers thought that scaling up current AI approaches would be ‘unlikely' or ‘very unlikely' to yield AGI (see go.nature.com/4smn16b). We suggest that the problem is part conceptual, because definitions of AGI are ambiguous and inconsistent; part emotional, because AGI raises fear of displacement and disruption; and part practical, as the term is entangled with commercial interests that can distort assessments. In writing this Comment, we approached this question from different perspectives — philosophy, machine learning, linguistics and cognitive science — and reached a consensus after extensive discussion. In what follows, we set out why we think that, once you clear away certain confusions, and strive to make fair comparisons and avoid anthropocentric biases, the conclusion is straightforward: by reasonable standards, including Turing's own, we have artificial systems that are generally intelligent. The long-standing problem of creating AGI has been solved. Some think that general intelligence does not exist at all, even in humans. Although this view is coherent and philosophically interesting, we set it aside here as being too disconnected from most AI discourse. But having made this assumption, how should we characterize general intelligence? Does it mean a top human expert for each task? Then no individual qualifies — Marie Curie won Nobel prizes in chemistry and physics but was not an expert in number theory. Does it mean a composite human with competence across the board? This, too, seems a high bar — Albert Einstein revolutionized physics, but he couldn't speak Mandarin. A definition that excludes essentially all humans is not a definition of general intelligence; it is about something else, perhaps ideal expertise or collective intelligence. Rather, general intelligence is about having sufficient breadth and depth of cognitive abilities, with ‘sufficient' anchored by paradigm cases. Breadth means abilities across multiple domains — mathematics, language, science, practical reasoning, creative tasks — in contrast to ‘narrow' intelligences, such as a calculator or a chess-playing program. Depth means strong performance within those domains, not merely superficial engagement. Children, average adults and an acknowledged genius such as Einstein all have general intelligence of varying level and profile. Individual humans excel or fall short in different domains. The same flexibility should apply to artificial systems: we should ask whether they have the core cognitive abilities at levels comparable to human-level general intelligence. Rather than stipulating a definition, we draw on both actual and hypothetical cases of general intelligence — from Einstein to aliens to oracles — to triangulate the contours of the concept and refine it more systematically. Our conclusion: insofar as individual humans have general intelligence, current LLMs do, too. We can start by identifying four features that are not required for general intelligence. We don't expect a physicist to match Einstein's insights, or a biologist to replicate Charles Darwin's breakthroughs. Few, if any, humans have perfect depth even within specialist areas of competence. Human general intelligence does not require perfection; neither should AGI. No individual human can do every cognitive task, and other species have abilities that exceed our own: an octopus can control its eight arms independently; many insects can see parts of the electromagnetic spectrum that are invisible to humans. Intelligence is a functional property that can be realized in different substrates — a point Turing embraced in 1950 by setting aside human biology1. We would not demand these things of intelligent aliens; the same applies to machines. This is generally used to indicate any system that greatly exceeds the cognitive performance of humans in almost all areas. Superintelligence and AGI are often conflated, particularly in business contexts, in which ‘superintelligence' often signals economic disruption. No human meets this standard; it should not be a requirement for AGI, either. Concepts such as ‘life' and ‘health' resist sharp definition yet remain useful; we recognize paradigm cases without needing exact boundaries. Humans are paradigm examples of general intelligence; a pocket calculator lacks it, despite superhuman ability at calculations. When we assess general intelligence or ability in other humans, we do not attempt to peer inside their heads to verify understanding — we infer it from behaviour, conversation and problem-solving. No single test is definitive, but evidence accumulates. Just as we assess human general intelligence through progressively demanding tests, from basic literacy to PhD examinations, we can consider a cascade of increasingly demanding evidence that warrants progressively higher confidence in the presence of AGI. A decade ago, meeting these might have been widely accepted as sufficiently strong evidence for AGI. Current AIs are more broadly capable than the science-fiction supercomputer HAL 9000 was.Credit: Hethers/Shutterstock Here, the demands escalate: gold-medal performance at international competitions, solving problems on PhD exams across multiple fields, writing and debugging complex code, fluency in dozens of languages, useful frontier research assistance as well as competent creative and practical problem-solving, from essay writing to trip planning. These achievements exceed many depictions of AGI in science fiction. The sentient supercomputer HAL 9000, from director Stanley Kubrick's 1968 film 2001: A Space Odyssey, exhibited less breadth than current LLMs do. And current LLMs even exceed what we demand of humans: we credit individual people with general intelligence on the basis of much weaker evidence. Revolutionary scientific discoveries and consistent superiority over leading human experts across a range of domains. Such evidence would surely allow no reasonable debate about the presence of general intelligence in a machine — but it is not required evidence for its presence, because no human shows this. Current LLMs already cover the first two levels. As LLMs tackle progressively more difficult problems, alternative explanations for their capabilities — for instance, that they are gigantic ‘lookup tables'8 that retrieve pre-computed answers or ‘stochastic parrots'9 that regurgitate shallow regularities without grasping meaning or structure — become increasingly disconfirmed. Often, however, such claims just reappear with different predictions. Hypotheses that retreat before each new success, always predicting failure just beyond current achievements, are not compelling scientific theories, but a dogmatic commitment to perpetual scepticism. AI language models killed the Turing test: do we even need a replacement? AI language models killed the Turing test: do we even need a replacement? By inference to the best explanation — the same reasoning we use in attributing general intelligence to other people — we are observing AGI of a high degree. Machines such as those envisioned by Turing have arrived. Our argument benefits from substantial advances and extra time. As of early 2026, the case for AGI is considerably more clear-cut. We now examine ten common objections to the idea that current LLMs display general intelligence. Several of them echo objections that Turing himself considered in 1950. Each, we suggest, either conflates general intelligence with non-essential aspects of intelligence or applies standards that individual humans fail to meet. The stochastic parrot objection says that LLMs merely interpolate training data. This echoes ‘Lady Lovelace's Objection', inspired by Ada Lovelace's 1843 remark and formulated by Turing as the claim that machines can “never do anything really new”1. Early LLMs certainly made mistakes on problems requiring reasoning and generalization beyond surface patterns in training data. But current LLMs can solve new, unpublished maths problems, perform near-optimal in-context statistical inference on scientific data11 and exhibit cross-domain transfer, in that training on code improves general reasoning across non-coding domains12. If critics demand revolutionary discoveries such as Einstein's relativity, they are setting the bar too high, because very few humans make such discoveries either. Furthermore, there is no guarantee that human intelligence is not itself a sophisticated version of a stochastic parrot. All intelligence, human or artificial, must extract structure from correlational data; the question is how deep the extraction goes. LLMs supposedly lack representations of their physical environment that are necessary for genuine understanding. But having a world model requires only the ability to predict what would happen if circumstances differed — to answer counterfactual questions. The ability of LLMs to solve olympiad mathematics and physics problems and assist with engineering design suggests that they possess functional models of physical principles. By these standards, LLMs already have world models. Furthermore, neural networks developed for specialized domains such as autonomous driving are already learning predictive models of physical scenes that support counterfactual reasoning and sophisticated physical awareness13. This objection centres on the fact that LLMs are trained only on text, and so must be fundamentally limited to text-based tasks. Frontier models are now trained on images and other multimodal data, making this objection somewhat obsolete. Moreover, language is humanity's most powerful tool for compressing and capturing knowledge about reality. LLMs can extract this compressed knowledge and apply it to distinctly non-linguistic tasks: helping researchers to design experiments — for example, suggesting what to test next in biology and materials science4 — goes beyond merely linguistic performance. Without embodiment, critics argue, there can be no general intelligence. This reflects an anthropocentric bias that seems to be wielded only against AI. An entity that responds accurately to any question, but never moves or acts physically, would be regarded as profoundly intelligent. Physicist Stephen Hawking interacted with the world almost entirely through text and synthesized speech, yet his physical limitations in no way diminished his intelligence. Even ‘agentic' AI systems — such as frontier coding agents — typically act only when a user triggers a task, even if they can then automatically draft features and fix bugs. Like the Oracle of Delphi — understood as a system that produces accurate answers only when queried — current LLMs need not initiate goals to count as intelligent. Humans typically have both general intelligence and autonomy, but we should not thereby conclude that one requires the other. Autonomy matters for moral responsibility, but it is not constitutive of intelligence. Dai, Y., Gao, Z., Sattar, Y., Dean, S. & Sun, J. Preprint at arXiv https://doi.org/10.48550/arXiv.2506.07298 (2025). Goodfire had no role in the conceptualization, writing or decision to publish this work AI language models killed the Turing test: do we even need a replacement? How to detect consciousness in people, animals and maybe even AI Science and the new age of AI: a Nature special ArXiv says submissions must be in English: are AI translators up for the job? Technologies to give a clearer view of the lungs OpenAI's brain implant would use ultrasound to read minds. ArXiv says submissions must be in English: are AI translators up for the job? How DeepMind's genome AI could help solve rare-disease mysteries The International Center for Young Scientists (ICYS) of the National Institute for Materials Science (NIMS) invites outstanding researchers to appl... Applicants should have a good educational background in their field of study，different qualifications for different positions 379, North Section of Mingli Road, Zhengzhou City, Henan Province, China No.1 Shizishan Street, Hongshan District, Wuhan, Hubei Province, China AI language models killed the Turing test: do we even need a replacement? How to detect consciousness in people, animals and maybe even AI Science and the new age of AI: a Nature special An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/celebrate-groundhog-day-with-6-bizarre-groundhog-facts/'>Happy Groundhog Day! Celebrate with 6 startling facts about these rodents</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 11:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Groundhogs don't really forecast the weather, but there are plenty of other strange things about these rodents Groundhogs are unique, and it's not just because of their annual weather forecasts Other groundhogs (Marmota monax) around the country have joined him in this climatological soothsaying. Folklore, likely stemming from Celtic mid-season festivals, dictates that if Phil sees his shadow, winter will continue for six more weeks, whereas if he doesn't, spring will come early. (Many people around the country are probably hoping that shadow doesn't appear after recent winter weather.) Of course, Phil's “forecasts” are actually no better than chance, but thousands, drawn by the allure of an unusual event for an unusual creature, still gather to see him emerge every February 2. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. When groundhogs emerge this time of year, they are actually more interested in finding mates than making forecasts. The first ones to go aboveground are usually males that take a few days to mark their territory and gauge potential partners before they head back into their burrows for another month of hibernation. Groundhogs' teeth never stop growing, so gnawing on things to keep them from getting too long is essential. The animals usually use their powerfully regenerative teeth to break off pieces of food, defend themselves and gather materials they might use to build their nests. As groundhogs dig deep to create their burrows, they have sometimes stumbled upon historical relics. In at least three cases, groundhogs have helped locate new archeological sites—including one of the oldest known sites of human habitation in North America: Pennsylvania's Meadowcroft Rockshelter. Archeology isn't the only science groundhogs help us with—they also help medical researchers better understand the connection between hepatitis B (HBV) and liver cancer. Because the groundhog equivalent of HBV is so similar to that in humans, researchers can use them as a stand-in for people to better understand how HBV causes liver issues. Their ubiquity means lots of groups, including Indigenous Americans, have had a chance to name them. Besides “groundhog,” a few of the more common names include woodchuck and whistle-pig, referring to the “chuck” and “whistle” sounds they make or their stocky build. K. R. Callaway is a freelance journalist specializing in science, health, history and policy. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02995-0'>Customizing CRISPR–Cas PAM specificity with protein language models</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 11:08:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article CRISPR–Cas enzymes must recognize a protospacer-adjacent motif (PAM) to edit a genomic site, greatly limiting the range of targetable sequences in a genome. Although engineering strategies to alter PAM specificity exist, they typically require labor-intensive, iterative experimentation. We introduce an evolution-informed deep learning model, Protein2PAM, to efficiently guide the design of Cas protein variants tailored to recognize specific PAMs. Trained on a dataset of over 45,000 CRISPR–Cas PAMs, Protein2PAM rapidly and accurately predicts PAM specificity directly from Cas proteins across type I, II and V CRISPR–Cas systems. Using in silico mutagenesis, the model identifies residues critical for PAM recognition in Cas9 without using structural information. We use Protein2PAM to computationally evolve Nme1Cas9, generating variants with broadened PAM recognition and up to a 50-fold increase in PAM cleavage rates compared to the wild type in vitro. Our machine learning approach allows Cas enzymes to target sequences that were previously inaccessible because of PAM constraints, potentially increasing target flexibility in personalized genome editing. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Training data and model weights are available from Hugging Face (https://huggingface.co/collections/Profluent-Bio/protein2pam). Sequencing results from HT-PAMDA screening are available through the National Center for Biotechnology Information Sequence Read Archive through BioProject PRJNA1378997. An open-source Protein2PAM Python API is available from GitHub (https://github.com/Profluent-AI/protein2pam). Our webserver, which provides interactive access to Protein2PAM predictions and confidence scores without the need for local installation, can be accessed at https://protein2pam.profluent.bio. Collias, D. & Beisel, C. L. CRISPR technologies and the search for the PAM-free nuclease. Karvelis, T. et al. Rapid characterization of CRISPR–Cas9 protospacer adjacent motif sequence elements. Genome Biol. Gasiunas, G. et al. A catalogue of biochemically diverse CRISPR–Cas9 orthologs. Yan, W. X. et al. Functionally diverse type V CRISPR–Cas systems. Anzalone, A. V., Koblan, L. W. & Liu, D. R. Genome editing with CRISPR–Cas nucleases, base editors, transposases and prime editors. Christie, K. A. et al. Towards personalised allele-specific CRISPR gene editing to treat autosomal dominant disorders. Nishimasu, H. et al. Engineered CRISPR–Cas9 nuclease with expanded targeting space. Walton, R. T., Christie, K. A., Whittaker, M. N. & Kleinstiver, B. P. Unconstrained genome targeting with near-PAMless engineered CRISPR–Cas9 variants. Kleinstiver, B. P. et al. Engineered CRISPR–Cas12a variants with increased activities and improved targeting ranges for gene, epigenetic and base editing. Kleinstiver, B. P. et al. Broadening the targeting range of Staphylococcus aureus CRISPR–Cas9 by modifying PAM recognition. Kleinstiver, B. P. et al. Engineered CRISPR–Cas9 nucleases with altered PAM specificities. Hu, J. H. et al. Evolved Cas9 variants with broad PAM compatibility and high DNA specificity. Huang, T. P. et al. High-throughput continuous evolution of compact Cas9 variants targeting single-nucleotide-pyrimidine PAMs. Miller, S. M. et al. Continuous evolution of SpCas9 variants compatible with non-G PAMs. Nijkamp, E., Ruffolo, J. A., Weinstein, E. N., Naik, N. & Madani, A. ProGen2: exploring the boundaries of protein language models. Cell Syst. Lin, Z. et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. Ruffolo, J. A. et al. Design of highly functional genome editors by modelling CRISPR–Cas sequences. & Marraffini, L. A. RNA guide complementarity prevents self-targeting in type VI CRISPR Systems. Marraffini, L. A. & Sontheimer, E. J. CRISPR interference: RNA-directed adaptive immunity in bacteria and archaea. Camargo, A. P. et al. IMG/VR v4: an expanded database of uncultivated virus genomes within a framework of extensive functional, taxonomic, and ecological metadata. Nucleic Acids Res. Camargo, A. P. et al. IMG/PR: a database of plasmids from genomes and metagenomes with rich annotations and metadata. Nucleic Acids Res. Ciciani, M. et al. Automated identification of sequence-tailored Cas9 proteins using massive metagenomic data. Adler, B. A. et al. CasPEDIA Database: a functional classification system for class 2 CRISPR–Cas enzymes. Nucleic Acids Res. Gleditzsch, D. et al. PAM identification by CRISPR–Cas effector complexes: diversified mechanisms and structures. RNA Biol. Anders, C., Niewoehner, O., Duerst, A. & Jinek, M. Structural basis of PAM-dependent target DNA recognition by the Cas9 endonuclease. Ruffolo, J. Adapting protein language models for structure-conditioned design. Preprint at bioRxiv https://doi.org/10.1101/2024.08.03.606485 (2024). Wei, J. et al. Closely related type II-C Cas9 orthologs recognize diverse PAMs. Wimmer, F., Mougiakos, I., Englert, F. & Beisel, C. L. Rapid cell-free characterization of multi-subunit CRISPR effectors and transposons. Sun, W. et al. Structures of Neisseria meningitidis Cas9 complexes in catalytically poised and anti-CRISPR-inhibited states. Huang, X. et al. Decoding CRISPR-Cas PAM recognition with UniDesign. Hirano, S., Nishimasu, H., Ishitani, R. & Nureki, O. Structural basis for the altered PAM specificities of engineered CRISPR–Cas9. Anders, C., Bargsten, K. & Jinek, M. Structural plasticity of PAM recognition by engineered variants of the RNA-guided endonuclease Cas9. Schmidheini, L. et al. Continuous directed evolution of a compact CjCas9 variant with broad PAM compatibility. Amrani, N. et al. NmeCas9 is an intrinsically high-fidelity genome-editing platform. Genome Biol. Tsui, T. K. M., Hand, T. H., Duboy, E. C. & Li, H. The impact of DNA topology and guide length on target selection by a cytosine-specific Cas9. ACS Synth. Luscombe, N. M., Laskowski, R. A. & Thornton, J. M. Amino acid-base interactions: a three-dimensional analysis of protein–DNA interactions at an atomic level. Nucleic Acids Res. Walton, R. T., Hsu, J. Y., Joung, J. K. & Kleinstiver, B. P. Scalable characterization of the PAM requirements of CRISPR–Cas enzymes using HT-PAMDA. Grathwohl, W., Swersky, K., Hashemi, M., Duvenaud, D. & Maddison, C. Oops I took a gradient: scalable sampling for discrete distributions. In Proceedings of the 38th International Conference on Machine Learning (eds Meila, M. & Zhang, T.) (PMLR, 2021). Li, W. & Godzik, A. CD-HIT: a fast program for clustering and comparing large sets of protein or nucleotide sequences. Buchfink, B., Xie, C. & Huson, D. H. Fast and sensitive protein alignment using DIAMOND. van Dongen, S. Graph clustering via a discrete uncoupling process. Deorowicz, S., Debudaj-Grabysz, A. & Gudyś, A. FAMSA: fast and accurate multiple sequence alignment of huge protein families. Eddy, S. R. Accelerated profile HMM searches. Steinegger, M. & Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. Letunic, I. & Bork, P. Interactive Tree of Life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. Liu, Z. et al. Versatile and efficient genome editing with Neisseria cinerea Cas9. Hand, T. H., Das, A. & Li, H. Directed evolution studies of a thermophilic type II-C Cas9. Hirano, H. et al. Structure and engineering of Francisella novicida Cas9. Cui, Z. et al. FrCas9 is a CRISPR/Cas9 system with high editing efficiency and fidelity. Kim, E. et al. In vivo genome editing with a small Cas9 orthologue derived from Campylobacter jejuni. Hirano, S. et al. Structural basis for the promiscuous PAM recognition by Corynebacterium diphtheriae Cas9. Zetsche, B. et al. Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR–Cas system. Zetsche, B., Abudayyeh, O. O., Gootenberg, J. S., Scott, D. A. & Zhang, F. A survey of genome editing activity for 16 Cas12a orthologs. Strecker, J. et al. Engineering of CRISPR–Cas12b for human genome editing. Harrington, L. B. et al. A scoutRNA is required for some type V CRISPR–Cas systems. Burstein, D. et al. New CRISPR–Cas systems from uncultivated microbes. Karvelis, T. et al. PAM recognition by miniature CRISPR–Cas12f nucleases triggers programmable double-stranded DNA target cleavage. Nucleic Acids Res. Wang, Y. et al. A highly specific CRISPR-Cas12j nuclease enables allele-specific genome editing. Strecker, J. et al. RNA-guided DNA insertion with CRISPR-associated transposases. Urbaitis, T. et al. A new family of CRISPR-type V nucleases with C-rich PAM recognition. Wu, W. Y. et al. The miniature CRISPR–Cas12m effector binds DNA to block transcription. Al-Shayeb, B. et al. Diverse virus-encoded CRISPR–Cas systems include streamlined genome editors. Zhang, Y. et al. Catalytic-state structure and engineering of Streptococcus thermophilus Cas9. Tran, M. H. et al. A more efficient CRISPR–Cas12a variant derived from MA2020. Gao, L. et al. Engineered Cpf1 variants with altered PAM specificities. Russel, J., Pinilla-Redondo, R., Mayo-Mun˜oz, D., Shah, S. A. & Sørensen, S. J. CRISPRCasTyper: automated identification, annotation, and classification of CRISPR–Cas loci. CRISPR J. Chen, Z. & Zhao, H. A highly sensitive selection method for directed evolution of homing endonucleases. Nucleic Acids Res. Rohland, N. & Reich, D. Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture. Gooden, A. A., Evans, C. N., Sheets, T. P., Clapp, M. E. & Chari, R. dbGuide: a database of functionally validated guide RNAs for genome editing in human and mouse cells. Nucleic Acids Res. Article PubMed Central & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Clement, K. et al. CRISPResso2 provides accurate and rapid genome editing sequence analysis. We acknowledge funding from the Natural Sciences and Engineering Research Council of Canada doctoral postgraduate scholarship (PGS-D 567791 to R.A.S. ), the Kayden-Lambert Massachusetts General Hospital (MGH) Research Scholar Award 2023–2028 (B.P.K.) and National Institutes of Health grants DP2CA281401 (B.P.K.) Profluent Bio, Emeryville, CA, USA Stephen Nayfach, Aadyot Bhatnagar, Andrey Novichkov, Alexander M. Hoffnagle, Riffat Hussain, Gabriella O. Estevam, Emily Hill, Jeffrey A. Ruffolo, Joseph Gallagher, Alexander J. Meeske, Peter Cameron & Ali Madani Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USA Nahye Kim, Rachel A. Silverstein & Benjamin P. Kleinstiver Department of Pathology, Massachusetts General Hospital, Boston, MA, USA Nahye Kim, Rachel A. Silverstein & Benjamin P. Kleinstiver Department of Pathology, Harvard Medical School, Boston, MA, USA Nahye Kim, Rachel A. Silverstein & Benjamin P. Kleinstiver Biological and Biomedical Sciences Program, Harvard University, Boston, MA, USA Rachel A. Silverstein Department of Microbiology, University of Washington, Seattle, WA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and A.M. conceptualized the project. built the training dataset. trained the Protein2PAM models. performed the computational experiments. developed the webserver. assisted with structural analysis. and E.H. performed wet-lab experiments with oversight from J.G. performed HT-PAMDA experiments and R.A.S. with HT-PAMDA data analysis with oversight from B.P.K. provided critical feedback. prepared the manuscript with input and contributions from A.B. All authors contributed to writing and/or reviewing the final draft of the manuscript. Correspondence to Stephen Nayfach or Ali Madani. and A.M. are current or former employees, contractors or executives of Profluent Bio and may hold shares in Profluent Bio. are inventors on patents or patent applications filed by Mass General Brigham (MGB) that describe HT-PAMDA or genome-engineering technologies related to the current study. is a consultant for Novartis Venture Fund, Foresite Labs, Generation Bio and Jumble Therapeutics and is on the scientific advisory boards of Life Edit Therapeutics and Prime Medicine. has a financial interest in Prime Medicine. 's interests were reviewed and are managed by MGH and MGB in accordance with their conflict-of-interest policies. The remaining authors declare no competing interests. Nature Biotechnology thanks Wei Li and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Figs. Supplementary Tables 1–10. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Nayfach, S., Bhatnagar, A., Novichkov, A. et al. Customizing CRISPR–Cas PAM specificity with protein language models. Accepted: 19 December 2025 Published: 02 February 2026 Version of record: 02 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative ISSN 1546-1696 (online) ISSN 1087-0156 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02496-5'>Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signaling</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 10:59:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. CTNNB1, the gene encoding β-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signaling pathway, typically through missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signaling phenotypes for all 342 possible missense mutations in the mutation hotspot. Our data define the genetic requirements for β-catenin degron function, refine the consensus motif for substrate recognition by β-TRCP and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of predicted activity. In hepatocellular carcinoma, mutation effect scores distinguish two tumor subclasses with different levels of β-catenin signaling, and weaker mutations predict greater immune cell infiltration in the tumor microenvironment. Our work provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy. The canonical Wnt pathway is essential for normal development and is erroneously activated in many cancers1. Its main intracellular effector, β-catenin (encoded by CTNNB1), is normally degraded by a destruction complex consisting of APC, AXIN2, CK1 and GSK3β. Wnt ligand binding or mutations that impair this complex lead to β-catenin accumulation, nuclear translocation and activation of T cell Factor (TCF)/lymphoid enhancer-binding factor (LEF) transcription factor target genes (Supplementary Fig. The tumor suppressor adenomatous polyposis coli (APC) serves as a scaffold within the destruction complex. APC loss-of-function mutations activate β-catenin and are common in colorectal cancers, in which tumors often select for mutations with a ‘just-right' level of β-catenin signaling2,3,4,5. Gain-of-function changes within the CTNNB1 exon 3 degron motif are among the most common mutations in several tumor types6,7. These mutations act in a dominant manner by disrupting the interaction between the phosphorylated DpSGXXpS motif (residues 32–37) and the E3 ubiquitin ligase receptor β-TRCP, preventing β-catenin ubiquitylation and degradation7. The phospho-regulatory cascade involves CK1 priming at S45, followed by GSK3β phosphorylation at T41, S37 and S33, enabling β-TRCP docking. HCC has been classified into subtypes with distinct molecular and immune features10,11,12,13. The inflamed class is characterized by interferon signaling, enrichment of cytolytic T cells and M1 macrophages, whereas the non-inflamed class is largely devoid of immune infiltrate. β-catenin signaling and CTNNB1 mutations are associated with immune exclusion and promote resistance to immunotherapy in mouse models of HCC9,14,15,16. However, a substantial fraction of CTNNB1-mutant HCCs presents an inflamed phenotype (30%), highlighting the need for studies to elucidate the underlying mechanisms14. A small subset of exon 3 genotypes observed in human cancer has been directly compared using cellular assays; however, the functional impact of most variants remains unknown. Here, we develop a saturation genome editing (SGE) assay to systemically quantify the relative activity of all possible missense substitutions within the exon 3 hotspot, assigning functional scores to >80% of CTNNB1 missense mutations observed in cancer. We analyzed 9,248 tumors with CTNNB1 mutations in the COSMIC database (Fig. As previously reported6, the majority of mutations (86%) were missense substitutions. Of these, 88% occurred in the L31–G48 hotspot region encoded by exon 3 (Fig. a, Histogram showing the frequency of mutations at each amino acid position of human CTNNB1 across all tumors present in the COSMIC database. The consensus docking site for the β-TRCP E3 ligase substrate receptor is shown in a yellow box; ϕ indicates any hydrophobic amino acid, and X indicates any amino acid. Phosphorylation sites known to be critical for β-catenin turnover are highlighted in red boxes. c, Histograms show distinct distributions of mutations within the CTNNB1 mutation hotspot for COSMIC tumors of the adrenal gland, central nervous system (CNS) and skin. Data from other primary tissue sites are shown in Supplementary Fig. 2. d, Schematic overview of the mutational scanning assay. A β-catenin signaling activity reporter construct was integrated randomly into the genome to generate transgenic mice23. Primary ES cells were derived, and a region spanning exons 2–6 of Ctnnb1 was replaced with a puro∆TK selection cassette on one of two alleles. A plasmid library was generated for homology-directed repair, encoding each of 342 possible single amino acid substitutions spanning codons 31 to 48 of β-catenin, together with silent mutations to allow selective PCR amplification of edited alleles from genomic DNA. This was transfected into multiplex-ready mouse ES cells together with sgRNAs that cut in an allele-specific manner on either side of the selection cassette. After recovery, transfected cells were cultured in the presence of FIAU to kill those in which the selection cassette had not been removed. Cells were then subject to fluorescence-activated cell sorting based on the level of GFP reporter expression, before extraction of genomic DNA, amplification of Ctnnb1 exon 3 by PCR and Illumina sequencing. Further methodological information is detailed in the Methods. e, The frequency of individual missense mutations across each position in the mutation hotspot in cell populations sorted according to the scheme shown in d, expressed relative to their frequency in the unsorted pool sample. The color scheme used to represent different missense mutations is shown to the right. Analysis across tissues revealed tissue-specific mutation preferences (Fig. S45 mutations are greatly enriched in adrenal and kidney tumors, whereas mutations within the β-TCRP docking motif predominated in the central nervous system (Fig. Liver and skin showed a broader profile, encompassing all six of the most frequently mutated positions (Fig. Specific amino acid substitutions at each residue also varied by tissue. Overall, CTNNB1 hotspot mutational patterns are highly diverse and tissue-dependent. To systematically measure the consequences of CTNNB1 hotspot mutations on β-catenin signaling, we devised a multiplexed CRISPR homology-directed repair (HDR) assay covering all 342 possible single amino acid substitutions across positions 31–48 (Fig. This assay enables phenotypic analysis of variants expressed from the endogenous mouse Ctnnb1 locus, which is 100% conserved with human exon 3 at the amino acid level. Variant function was measured in parallel under normal regulatory control20,21,22, providing a sensitive and comprehensive assessment of hotspot mutant phenotypes. A library of HDR templates was synthesized, each encoding a single amino acid substitution and flanking synonymous changes to enable selective PCR amplification of edited alleles20,21 (Fig. To analyze individual hotspot mutations at the single-cell level, we derived embryonic stem (ES) cells from the Tcf/Lef-H2B–GFP transgenic mouse line23,24, then replaced exons 2–6 of Ctnnb1 with a negative counter-selection cassette on one allele (Fig. Specifically targeting and replacing the selection cassette with a multiplexed HDR template library, combined with fluorescence-activated cell sorting (Supplementary Fig. ES cells were chosen for their high HDR efficiency and intact canonical WNT signaling, in contrast to most tumor lines. After editing and selection, a subset of cells displayed elevated Tcf/Lef-H2B–GFP reporter expression (Supplementary Fig. 3a), consistent with canonical Wnt pathway activation. Cells were sorted into six equally log-spaced bins (P1–P6) based on increasing reporter GFP signal (Supplementary Fig. 3a), and genomic DNA was subjected to amplicon deep sequencing across the Ctnnb1 hotspot region using primers specific for HDR-edited alleles (Fig. Amplicon sequencing was also performed on the untransfected HDR donor library (‘plasmid') and edited but unsorted cells (‘pool'; Supplementary Fig. 3b), enabling quantification of Ctnnb1 variant frequencies across reporter activity bins relative to their frequency in the total cell population (Fig. Normalized mutant allele frequencies showed high correlations between biological replicates (Pearson's r, range 0.54–0.89; Supplementary Fig. 3c) and were merged for downstream analysis. Frequencies also correlated well between plasmid and pool samples (Pearson's r, 0.63), indicating consistent and unbiased HDR efficiency throughout the targeted region (Supplementary Fig. The mutation frequency was relatively constant across positions in the unsorted cell population (Supplementary Fig. 3b), whereas it varied substantially across cells with different GFP levels (Fig. Substitutions at codon positions that are frequently mutated in tumors (D32, S33, G34, S37, T41 and S45) were rarely observed in low activity bins but frequently observed in bins of higher activity (Fig. 1e), validating this system's ability to functionally classify CTNNB1 mutations. For each variant, we calculated a mutational effect score (MES) by integrating allele frequencies and reporter activation levels across bins25 (Supplementary Methods). This procedure provided a metric to compare the phenotypic consequences of each substitution and gain insight into β-catenin regulation (Fig. a, Heatmap representation of MESs, showing the activity profile for every possible amino acid substitution at the CTNNB1 mutation hotspot, reconstructed from data shown in Fig. Individual scores are provided in Supplementary Table 2. The consensus motif for β-TRCP docking is shown in a beige rectangle above the heatmap, with known phosphorylation sites highlighted in red. b, Schematic to illustrate the derivation of clonal mouse ES cell lines genome-edited to express individual Ctnnb1 exon 3 mutations. c, Unsupervised clustering of Ctnnb-mutant clones based on global RNA-seq expression profiling shows co-clustering of clones with mutations of similar MES values (purple bar). d,e, Ten endogenous β-catenin target genes were selected based on the high correlation (>0.85) between transcript expression and dose of the GSK3β inhibitor CHIR99021 (d); transcript expression also correlated with MES values of the relevant exon 3 mutations (e). We first compared the MES values with scores produced by 50 computational variant effect predictors (VEPs) for the same substitutions26 (Supplementary Table 3). The strongest correlation was observed for EVE, which uses deep generative models to calculate variant effect scores from evolutionary sequence information27. The correlation between EVE and MES scores for the CTNNB1 hotspot (Spearman's ρ, 0.662) was higher than any other EVE–deep mutational scanning (DMS) combination and all but three of 1,430 VEP–DMS comparisons in a prior benchmarking study26. Although correlation with VEP scores is not a definitive measure of DMS reliability, the strong concordance supports the high quality of the current dataset. We next tested the ability of MES scores to predict endogenous β-catenin target activation. Using multiplex-ready ES cells, we generated 15 clonal cell lines (Fig. 2b), each heterozygous for one of 11 mutations spanning a range of MES values, focusing on substitutions at the three most frequently mutated positions in human cancer. Transcriptional responses were measured using bulk 3′ RNA sequencing (RNA-seq), with GSK3β inhibition by CHIR99021 serving as a positive control. Unsupervised hierarchical clustering classified cell lines into low, medium and high MES categories (Fig. Ten β-catenin targets that showed dose-responsive activation by CHIR99021 (Pearson r > 0.85; Fig. 2d) also displayed positive correlations between transcript levels and the relevant MES value (Fig. 2e; median Pearson r = 0.57; range, 0.02–0.87; P < 0.05 for seven out of ten target genes). MES values also predicted endogenous β-catenin target gene expression in human fetal liver organoids. A previous study introduced one of four exon 3 mutations (D32G, S33F, T41A or S45P) homozygously at the endogenous CTNNB1 locus (Supplementary Fig. RNA-seq data from these organoids showed positive correlations between MES values and β-catenin target genes in nine out of ten cases (median Pearson r = 0.65; range, −0.5 to 0.96; Supplementary Fig. As outlined further below, MES scores also predicted the strength of β-catenin target activation in two independent human HCC cohorts. Altogether, these data show that MES values predict the degree to which endogenous β-catenin target genes are activated by different exon 3 mutations in both mouse ES cells and hepatocytes. The current model of genotype–phenotype relationships within the exon 3 hotspot proposes that missense mutations exert position-dependent effects on β-catenin signaling: S45 mutations activate weakly, T41 mutations activate moderately and mutations within the core β-TRCP docking motif (positions 32–37) activate strongly19. Our derivation of MES values for all 19 possible missense mutations across the hotspot region enabled testing and refinement of this model. Notably, of the 342 possible amino acid changes, only 105 can arise by a single nucleotide substitution (median of six changes per site; Fig. As expected, this group accounts for >98% of the single amino acid substitutions in the COSMIC database (Supplementary Table 1). a, The distribution of MES values is shown by codon position for all 19 missense substitutions (top) or just the subset that can be reached by a single nucleotide mutation in the human genome (bottom). The core β-TRCP docking motif is highlighted in yellow. For boxplots in a and b, horizontal lines show the median value, boxes show the second and third quartiles and whiskers show the range. b, The distribution of all MES values (top) or just the single nucleotide subset (bottom) for invariable positions within the β-TRCP docking motif (combined for D32, S33, G34, S37: ‘Docking'), T41 and the CK1 target site at position 45. The two most common T41 mutations in human cancer are labeled in the bottom panel. P values show one-way ANOVA with post hoc Tukey's honestly significant difference test. c, The distribution of MES values for all amino acid substitutions is shown for individual phosphosites, with serine–threonine substitutions highlighted. d, Structure of human β-TRCP (shown in surface mode) in complex with the phosphorylated degron peptide of β-catenin (amino acids 30–40 are shown) based on PDB 1P22. Individual β-catenin residues are colored according to the mean MES value across all substitutions at that position. e, Correlation between MES values for individual substitutions at position I35 with the Kyte–Doolittle hydrophobicity scale34. f, Correlation between MES values for individual substitutions at position I35 with the Chou–Fasman β-sheet scale36. P values in e and f show Spearman's rank tests (two-tailed). g, Amino acid logo generated from n = 29 high-confidence β-TRCP docking sites containing the ‘DSGX' motif37. Position 4, equivalent to I35 in β-catenin, is highlighted with an arrow. h, Stacked bar chart showing the proportion of high-confidence β-TRCP motifs (n = 29), with position 4 residues that rank in the top 25% most disruptive based on the hydrophobicity and β-sheet indices detailed above. P value shows Fisher's exact test (two-tailed). Consistent with the current model, we found that S45 mutations generally produced weaker activation than mutations in the docking motif, both among single-nucleotide variants and across all substitutions (Fig. Nonetheless, MES values at the same position varied substantially (Fig. For example, S45T and T41S were well tolerated (Fig. 3c), reflecting the ability of CK1α and GSK3β to phosphorylate either serine or threonine29. Substitution of S45 for small amino acids (alanine, glycine) was also well tolerated (Fig. It has previously been reported that in-frame deletions at S45 still permit phosphorylation at T41, S37 and S33 in colon cancer cells30,31. Collectively, the data thus suggest that the absence of a large side chain at S45 enables β-catenin degron function without CK1α priming32. Although the average effect of all T41 substitutions was intermediate, >98% in the COSMIC database are substitutions for alanine or isoleucine, which were the two strongest activating substitutions identified at this position. Both substitutions can be reached by one nucleotide change (Fig. This contrasts with S45, for which all eight strongest amino acid substitutions require at least two nucleotide changes (Fig. Therefore, strong activating S45 mutations are theoretically possible (for example, S45M, S45K) but are rarely observed in cancer, probably because they require at least two nucleotide changes. Extending the current model19, our data show that T41A and T41I activate β-catenin more than many commonly observed docking-motif variants (for example, D32V, D32Y, D32H, S33P, G34R). Other mutations at T41 (T41N, T41P), which are rare but still recurrent in human cancer (Supplementary Table 1), elicit weaker activation in the lower range of S45 mutations. It is therefore evident that different substitutions at the same position can elicit markedly different effects on β-catenin signaling, refining our understanding of CTNNB1 mutational diversity. I35 lies at the center of the docking motif, directly contacting β-TRCP, and is thought to require a hydrophobic amino acid side chain6,33. MES values for I35 substitutions were broadly distributed (Figs. However, substitution of isoleucine for several polar amino acids (for example, threonine, asparagine, tyrosine) yielded low MES values (Fig. 2a), indicating that hydrophobicity is not essential. By screening 566 amino acid property indices35, we observed that the strongest correlations with I35 MES values were nearly all related to secondary structure propensities. Although residue I35 does not form a β-sheet in any available crystal structure, its dihedral angles place it clearly in the β region of a Ramachandran plot. To ask whether this secondary structure requirement extended to other β-TRCP docking sites, we analyzed 28 high-confidence β-TRCP-dependent degrons containing the ‘DSGX' motif37 (Supplementary Table 4). Position 4 residues in this motif (corresponding to I35 in β-catenin) were variable across substrates (Fig. Numerous substrates (eight out of 28) had position 4 amino acids that ranked in the 25% most disruptive on the hydrophobicity scale, whereas only one out of 28 featured in the same bracket of the β-sheet scale (Fisher's exact test, P = 0.0248). Therefore, an extended backbone conformation, rather than side chain hydrophobicity, better explains the effects of I35 substitutions and potentially analogous positions in other β-TRCP docking motifs. The frequency of CTNNB1 hotspot mutations varies across tumor types (Fig. Stem cells in different tissues experience different genotoxic insults, which affect overall mutational spectra38,39 as well as the probability that specific CTNNB1 missense mutations become available for selection. Alternatively, or in addition, different tissue environments might favor selection for missense mutations causing levels of activation that are optimal, or ‘just-right', for their spatio-temporal context2. The current data provide an opportunity to distinguish these possibilities. Mutational probability, calculated from background nucleotide substitution rates, is a poor predictor of CTNNB1 mutation patterns in cancer40, suggesting a strong influence of selection. We computed ‘mutational likelihood scores' (MLSs) for each of 342 hotspot missense mutations, using the background rates of nucleotide substitution in HCC and endometrial carcinoma whole-exome sequencing data (Supplementary Fig. These scores represent the probability of an amino acid substitution, given the dominant mutational biases seen genome-wide in coding sequences of CTNNB1-mutant tumors. MLS values correlated positively across tissues (Pearson r = 0.920 for all mutations, r = 0.655 for the single-nucleotide group; Fig. 4a) but did not predict observed mutation frequencies (Fig. a, Relationship between MLSs (see Supplementary Fig. 5) in exomes of HCC (LIHC) and endometrial carcinoma (UCEC) tumors from TCGA. Scores are shown for all 342 possible amino acid changes, colored according to the minimum number of nucleotide substitutions required. b, Relationship between the observed frequency of specific CTNNB1 hotspot amino acid substitutions in the COSMIC database and MLSs calculated from LIHC and UCEC exomes from TCGA (see Supplementary Fig. Only amino acid substitutions that can be reached by a single nucleotide mutation (>98% of observed mutations) are shown. c, Violin plots show the distribution of MES values of CTNNB1 exon 3 hotspot mutations in LIHC and UCEC tumors from TCGA. P value shows two-tailed Mann–Whitney test without adjustment for MLS. d, The distribution of MES values in tumors from different primary sites in the COSMIC dataset. All tissue sites with >100 CTNNB1 exon 3 mutations are shown. We next tested whether tumors arising in different tissues select for hotspot mutations that activate β-catenin signaling to different degrees. Mutations in The Cancer Genome Atlas (TCGA) endometrial carcinoma cohort (Uterine Corpus Endometrial Carcinoma, UCEC) were enriched for higher MES values (Mann–Whitney test, P = 2.2 × 10−5), whereas in TCGA HCC cohort (Liver Hepatocellular Carcinoma, LIHC), mutations spanned a broader range (Fig. This difference persisted following adjustment for tissue-specific background rates of mutation within coding sequences (P = 3.2 × 10−5). Given that mutation bias does not explain the tissue-specific difference in observed mutation frequencies, we postulate that they arise through natural selection for different optimal β-catenin signaling2. Even greater variation in MES values was observed in the larger COSMIC dataset (Supplementary Table 1) across CTNNB1-mutant tumors from diverse tissue origins (Fig. Using MES distributions, tissues could be categorized as favoring high-effect mutations (for example, central nervous system), low-effect mutations (for example, kidney) or a broad or bimodal distribution (Fig. Bimodal distributions, observed in tissues such as large intestine and liver, comprised a lower-effect group, with mutations at S45 and weaker mutations within the β-TRCP docking motif (for example, H36P, D32Y, S33P), and a higher-effect group containing S33, G34, S37 and T41 mutations. These distributions highlight the potential for phenotypic variation arising from different CTNNB1 mutations among tumors with the same site of origin. To investigate how CTNNB1 mutation strength influences signaling and immune exclusion in human cancer, we focused on HCC, which shows frequent exon 3 mutations. Patients with single exon 3 missense mutations and available RNA-seq data in TCGA HCC cohort8 (n = 80) were initially stratified into low and high groups based solely on the MES value of their mutation (Supplementary Fig. However, copy number gains spanning CTNNB1 enhance signaling, especially for S45 mutant alleles19, so six patients with copy number gains spanning mutant alleles in the low MES group were reassigned (Supplementary Fig. This produced 53 patients predicted to have strong and 27 with predicted weak β-catenin pathway activation (Fig. a, Proportion of HCC samples from TCGA cohort with CTNNB1 missense or AXIN1 coding mutations (left). The CTNNB1 missense mutations are then further divided into three categories (right): strong and weak mutations within the exon 3 hotspot (Supplementary Fig. 6b) and mutations that occurred elsewhere in the gene (other). A total of 18 further samples fell into more than one category or had deletions within CTNNB1; therefore, they were excluded. TCGA sample IDs and classifications are listed in Supplementary Table 9. b, Stacked histograms show the frequency and proportion of CTNNB1 exon 3 mutations from TCGA HCC cohort classified as weak or strong that lie in the docking site for β-TRCP (positions 32–37) versus elsewhere in the mutation hotspot. P value shows Fisher's exact test (two-sided). c, Expression of n = 10 β-catenin target genes in HCC stratified by β-catenin pathway mutation status. Solid horizontal lines show the median percentage value across all ten genes, boxes show the upper and lower quartiles and whiskers show the range. *P < 4 × 10−3, **P < 1 × 10−4 from negative binomial generalized linear model (two-sided), with Tukey's adjustment for a family of five estimates. Gene-level data are shown in Supplementary Fig. d, Heatmap representation of β-catenin pathway activation for TCGA HCC samples with exon 3 hotspot mutations separated into weak and strong categories. LGR5 and GLUL are individual HCC targets. ‘Hallmark' indicates a multi-gene score calculated across 42 genes known to be activated by the accumulation of β-catenin (Hallmark wnt_b-Catenin gene set, MSigDB). e, Boxplots show Hallmark β-catenin target gene set activation in TCGA tumors with strong or weak mutations in the docking motif, compared to those with mutations at T41 or S45. Samples with copy number gain spanning lower-effect mutations (n = 6) were excluded. Differences between groups were not significant in a one-way ANOVA (P = 0.098). f, Enrichment of Gene Ontology terms in transcripts ranked among the most upregulated in pairwise comparisons between strong and weak CTNNB1-mutant HCC samples from TCGA. Terms were selected from the full list shown in Supplementary Table 6. Normalized enrichment scores (NES) show enrichment scores normalized to the size of the gene set. P value estimation is based on an adaptive multi-level split Monte Carlo scheme, adjusted for multiple testing using Benjamini–Hochberg correction. g, H&E tumor sections from TCGA were scored using an ordinal scale according to the level of inflammatory infiltrate from zero (no visible immune cells) to three (diffuse or nodular aggregates of immune cells), then scores were compared across patients based on β-catenin pathway mutation status. P value shows Fisher's exact test (two-way) for differences between weak and strong groups in the fraction of patients with an immune score of zero versus one or greater. Statistical analysis was not performed on other groups. To assess the effects of other mutations in the β-catenin pathway, we compared transcript levels for ten liver β-catenin target genes in HCC tumors with hotspot missense mutations versus no CTNNB1 pathway mutation. We included missense mutations in CTNNB1 outside the hotspot region (‘other'; n = 19; Fig. 5a), including positions 335, 383 and 387, which disrupt interaction with the destruction complex subunit APC41, giving a total of 99 CTNNB1 missense mutation cases in the TCGA HCC cohort (n = 80 hotspot, n = 19 other). A further 10% of the cohort had coding mutations in AXIN1 (n = 36), another destruction complex subunit and known HCC driver8,42. Tumors in either the weak or strong hotspot class showed markedly higher β-catenin target gene expression compared to the ‘no mutation' group (Fig. Tumors with ‘other' mutations in CTNNB1, or AXIN1 mutations, expressed these targets at levels comparable to the ‘no mutation' samples (Fig. Therefore, although AXIN1 and non-hotspot CTNNB1 mutations contribute to HCC development, their effects on β-catenin target gene expression differ from both weak and strong hotspot mutations16,42,43,44. As predicted by our screen, β-catenin targets were expressed at significantly higher levels in tumors from the strong versus weak patient group (Fig. This was observed across all ten liver target genes in the curated set (median 49% reduction in transcript levels for the weak group (Fig. Tumors with weak exon 3 mutations also displayed significantly lower expression scores for the Hallmark CTNNB1 target gene set from MSigDB (37% median reduction, P < 0.01; Fig. The same trend was confirmed in a second independent HCC cohort (Montironi cohort)14 (Supplementary Fig. β-catenin pathway activation in tumors with T41 mutations was comparable to that of strong β-TRCP docking motif variants, with both groups showing non-significant trends towards higher activation than tumors with the weak docking motif or S45 mutations (one-way ANOVA, P = 0.098; Fig. Overall, weak mutations still activated β-catenin targets, but at intermediate levels compared to strong mutations and non-mutated tumors (Fig. Transcriptome-wide, genes upregulated in strong versus weak HCC samples were significantly enriched for the Gene Ontology term ‘canonical Wnt signaling pathway' as well as terms associated with proliferation and stem cell function (Fig. TERT may be a direct transcriptional target of β-catenin regulation45, and activating mutations in the TERT promoter are among the most frequent HCC driver mutations8,46. Genes upregulated in tumors with weak versus strong exon 3 mutations were significantly enriched for terms associated with immune cell infiltration (Fig. Given that β-catenin signaling activation is a major immune escape pathway in several tumor types, including HCC15,47, weaker pathway activation may permit greater immune engagement, with potential implications for patient stratification and targeted therapy15,48,49. Consistent with this idea, tumors with weak CTNNB1 mutations showed upregulation of canonical T cell transcripts (Supplementary Fig. In addition, histology confirmed more frequent immune cell infiltration in weak versus strong mutant tumors (65 vs 38% with a score of >1, Fisher's exact test, P = 0.0245; Fig. Altogether, CTNNB1 exon 3 hotspot MES scores derived from a cell-autonomous reporter assay predict not only β-catenin signaling strength in HCC, but also clinically relevant phenotypes associated with the tumor microenvironment (Fig. SGE enables genetic variants to be functionally assessed in their native chromosomal context using scalable, multiplexed assays. We developed an SGE screening assay to quantify the impact of all amino acid substitutions in the β-catenin degron on signaling activation, spanning a region that covers >80% of cancer-associated missense mutations. The data explain why particular CTNNB1 mutations are observed in cancer and others are not, despite disrupting residues known to be critical for degron function. Our experimental design has three advantages over previous studies that have compared the phenotype of CTNNB1 mutations17,18,19. First, we tested the function of all 19 alternative amino acids at each position, providing a complete understanding of genotype–phenotype relationships. Second, variants were introduced by genome editing at the endogenous locus, avoiding artifacts associated with ectopic overexpression. Third, the function of each variant was evaluated in parallel, in the same population of primary stem cells with a normal functioning Wnt pathway, providing sensitivity to distinguish subtle phenotypic differences. Previous studies introduced the concept that distinct CTNNB1 hotspot mutations activate the canonical Wnt pathway to varying degrees17,18,19. One study19 analyzed β-catenin target gene activation in benign and malignant liver tumors and proposed that signaling strength depends on mutation position: S45 mutations activate weakly, T41 mutations activate moderately and mutations within the docking site (32–37) activate strongly. Our dataset supports certain aspects of this model, including weaker activity of S45 variants, but also revealed that different substitutions at the same position can have markedly different effects on signaling (Fig. Small side chains at S45 are well tolerated, whereas amino acids at position I35 must support an extended β-sheet-like conformation (Fig. 3f), which is probably required for β-TRCP docking because the same requirement is observed at other β-TRCP-dependent degrons (Fig. Our data thus redefine the consensus motif for β-TRCP docking as DpSGβXpS, where β indicates any residue that supports an extended backbone conformation. At the adjacent position H36, proline is most disruptive (Fig. 2a), probably because of conformational rigidity, explaining why H36P is the only H36 mutation commonly observed in human cancer (Supplementary Table 1). Our dataset enables functional interpretation for variants of unknown significance within the CTNNB1 hotspot (Supplementary Table 1) and supports reclassification of variants with known relevance in HCC. We find that common T41 mutations (T41A, T41I) are strong activators, with MES values exceeding the average of docking site mutants (Figs. Moreover, about 25% of docking motif mutations in HCC showed relatively low activation, comparable to S45 mutations, which we classify as weak (Fig. This nearly doubles the proportion of TCGA patients with weak CTNNB1 mutations. On average, tumors driven by weak mutations express liver β-catenin target genes at ~50% levels relative to the strong group (Fig. Importantly, this group is also less likely to be immune-excluded (Fig. Weakly activating exon 3 mutations could provide both a novel mechanism and biomarker for this patient group and help to guide strategies for personalized combination-based therapies. Our COSMIC database analysis showed that tumors arising in different tissues harbor exon 3 mutations of different predicted strength (Fig. However, MES values were measured using a reporter gene in mouse ES cells. MES can predict endogenous β-catenin signaling outputs in mouse ES cells (Fig. However, further work is required to show this empirically; other β-catenin-dependent tumor types in TCGA were not informative either because of low sample numbers (for example, adrenocortical carcinoma), low frequency of exon 3 mutant tumors (for example, colorectal adenocarcinoma, melanoma) or low diversity of MES values (for example, endometrial carcinoma). The relationship between exon 3 mutations and signaling outputs in advanced tumors may not reflect earlier stages owing to epistatic effects of co-occurring mutations, together with environmental and metabolic changes that accompany tumor progression. This concern motivated us to conduct the SGE screen in non-transformed primary cells rather than a cancer cell line. We also leave open the possibility that MES values measured in mouse ES cells may lack physiological relevance in certain contexts; for example, where β-catenin is degraded by SCFFBXW11 rather than SCFβTRCP (ref. This work complied with all relevant ethical regulations. Animal work was approved by Memorial Sloan Kettering Cancer Center's Institutional Animal Care and Use Committee (protocol 03-12-017; principal investigator A.-K. Hadjantonakis). No human data were generated specifically for this study. All guide RNAs (gRNAs) were designed using the optimized CRISPR design webtool previously hosted by the Feng Zhang laboratory, Benchling (https://benchling.com) and Wellcome Sanger Institute Genome Editing (http://www.sanger.ac.uk/htgt/wge). The gRNAs were cloned into either pSpCas9(BB)-2A-GFP (Addgene plasmid 48138) or pSpCas9(BB)-2A-mCherry52 and verified by Sanger sequencing. A backbone vector (wild-type β-catenin vector) was generated by amplifying a 5.4 kb region of Ctnnb1 intron1–6 using primers 5′-GGTTGATACTACCTTGAGTACTC-3′ and 5′-GATTCACAGGGCTGCTAGTG-3′. The amplicon was cloned into the PCR4 TOPO vector using a TOPO TA cloning Kit (Invitrogen). Then, a Gibson cloning reaction was set up, including the wild-type β-catenin vector (amplified using primers 5′-GTGAGGCTTTCTTTGTTGGC-3′ and 5′-GTCAAAAGGCAGAATGAAAACAG-3′), PuroΔTKamplicon (5′-CTGTTTTCATTCTGCCTTTTGACCATAGAGCCCACCGCATCC-3′ and 5′-GCCAACAAAGAAAGCCTCACTACC GGGTAGGGGAGGCG-3′) and Gibson Assembly master mix (NEB) following the manufacturer's guidelines. After Puromycin (1 µg ml−1) selection, individual clones were picked, grown and validated as heterozygously targeted by Sanger sequencing. Cells were then routinely maintained on gelatin-coated flasks in Knockout DMEM (Gibco) supplemented with 10% FBS (GE Healthcare, HyClone), 2 mM L-glutamine (Gibco), 0.1 mM MEM non-essential amino acids, 0.1 mM β-mercaptoethanol (Gibco), 3 µM CHIR99021 (Axon Medchem), 1 µM PD0325901 (Axon Medchem) and leukemia inhibitory factor (hereafter referred to as R2i media). A double-stranded DNA library was synthesized by Twist Biosciences, with each 200 bp fragment encoding a single distinct amino acid (n = 20 per site) spanning codons 31 to 48 of β-catenin. Each fragment was flanked by BbsI recognition sites for cloning and included synonymous mutations to enable specific amplification of HDR-edited alleles. Each fragment was then cloned individually into a β-catenin destination vector containing a 5.5 kb β-catenin intron1–6 sequence (5′-GGTTGATACTACCTTGAGTACTC-3′ and 5′-GATTCACAGGGCTGCTAGTG-3′) with two BbsI sites flanking the target region. After ligation, the reactions were transformed into Stbl3 competent cells and incubated overnight on a shaker at 37 °C. An equal volume of inoculum from each pool was then combined and used as a starter culture for a single maxiprep plasmid isolation of the pooled HDR template library (Qiagen Maxiprep kit). TCF/Lef-H2B–GFP reporter cells with heterozygous PuroΔTK knockin at the endogenous Ctnnb1 locus were maintained in R2i media. A total of 200 × 106 cells were transfected with the pooled HDR template library and PuroΔTK sgRNAs 1–4 (spacer sequences: 5′-TGGGGATGCGGTGGGCTCTA-3′, 5′-CCACCGCATCCCCAGCATGC-3′, 5′-GCCTCCCCTACCCGGTAGTG-3′, 5′-GCCTCACTACCGGGTAGGGG-3′) in 26 six-well plates, using Lipofectamine 2000 (Invitrogen) following the manufacturer's protocol. Small-molecule enhancer L755507 was used at a concentration of 5 µM. On day 3, R2i media was replaced with GMEM (Gibco) supplemented with 10% FBS (GE Healthcare, HyClone), 2 mM L-glutamine (Gibco), 1 mM sodium pyruvate (Gibco), 0.1 mM MEM non-essential amino acids, 0.1 mM β-mercaptoethanol (Gibco) and leukemia inhibitory factor, including FIAU at 0.2 µM to selectively kill cells that had not deleted the TK cassette. On day 5, a single-cell suspension was generated using trypsin, and the cells were sorted based on GFP intensity into six equally logged bins using BD FACS Aria III (BD Biosciences). In a parallel control condition, cells underwent an identical genome editing and selection procedure but were collected without flow sorting (the ‘pool' sample). Each set of cells went through the same editing, drug selection and flow sorting procedure described above and in Fig. The number of cells sorted from each bin is listed in Supplementary Table 7. Genomic DNA was isolated using the DNeasy Blood and Tissue kit (Qiagen) according to the manufacturer's protocol. A first round of PCR was performed using a forward primer that annealed upstream of the homology arm (5′-GTGGACATCAGAGGACAACTTG-3′) and a reverse primer that annealed to a region containing the HDR-edited allele (5′-TGTCAACATCTTCTTCTTCGGGA-3′). The entire DNA sample was amplified in several 30-cycle reactions using Q5 Hot Start High-Fidelity 2× Master Mix (NEB), then the amplicons were digested with DpnI (Thermo Scientific), gel-purified and pooled. For library preparation for the Illumina sequencing platform, a second round of PCR was performed to incorporate Illumina-specific barcode, primer pad, linker and adaptors. The reactions were performed in triplicate for 14 cycles using Q5 Hot Start High-Fidelity 2× Master Mix (NEB), then pooled and purified using AMPure XP beads (Beckman Coulter). The mouse embryonic feeder-free stem cell line E14IVtg2a (E14) was maintained on gelatin-coated flasks in GMEM (Gibco) supplemented with 10% FBS (GE Healthcare, HyClone), 2 mM L-glutamine (Gibco), 1 mM sodium pyruvate (Gibco), 0.1 mM MEM non-essential amino acids, 0.1 mM β-mercaptoethanol (Gibco) and leukemia inhibitory factor. E14 cells were targeted with the puroΔTK vector as described above for the Tcf/Lef-H2B–GFP reporter cells, then the puroΔTK cassette was replaced by HDR donor templates containing individual missense mutations at positions S37, T41 and S45, as shown in Fig. ES cell colonies were picked and expanded, and clonal lines with heterozygous knockin were identified by Sanger sequencing of PCR amplicons. Total RNA was isolated from approximately one million cells per sample using the RNAeasy mini kit (Qiagen) according to the manufacturer's instructions. For the generation of cDNA and the RNA-seq library, we adapted the mcSCRB-seq method53 using previously described modifications54, starting with 100 ng of total RNA per sample; Illumina paired-end sequencing of the library was then performed on a NovaSeq 6000 (Illumina) following the manufacturer's instructions. After sequencing, Read 1 contained the cDNA information, and read 2 only contained the unique molecular identifier. A Galaxy workflow consisting of the following tools was used to perform (reads to count) transcriptome analysis. Cutadapt (Galaxy version 4.0+galaxy1; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/4.0+galaxy1) was first used to remove adapter sequences from fastq files. FastQC (Galaxy version 0.73+galaxy0; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.73+galaxy0) was then used to generate the Read Quality reports. Filter with SortMeRNA (Galaxy version 2.1b.6; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/sortmerna/bg_sortmerna/2.1b.6) was used to filter reads for ribosomal RNAs in metatranscriptomic data. The reads from this filtered fastq files were aligned to mm10 build of mouse genome using RNA STAR the Gapped-read mapper for RNA-seq data (Galaxy version 2.7.8a+galaxy0; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a+galaxy0). UMI-tools deduplicate (Galaxy version 1.1.2+galaxy2; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/umi_tools_dedup/umi_tools_dedup/1.1.2+galaxy2) was used for deduplication of UMIs. MarkDuplicates (Galaxy version 2.18.2.3; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/2.18.2.3) was used to examine aligned records in BAM datasets to locate and filter duplicate molecules. Bedtools Genome Coverage (Galaxy version 2.30.0; https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_genomecoveragebed/2.30.0) was used to record the genome coverage in bedgraph file format from the BED files with the alignment data. CONVERTER_bedgraph_to_bigwig (Galaxy version 1.0.1; https://usegalaxy.eu/root?tool_id=CONVERTER_bedgraph_to_bigwig) was used to convert the bedgraph file to bigwig file format for upload and visualization of the expression data on the UCSC or IGV genome browser. The counts table was converted into counts per million for all samples before normalization to z-score values, which were then used to generate a heatmap using the pHeatmap package (v.1.0.12) in RStudio. Unsupervised hierarchical clustering resulted in three main clusters. Differential gene expression analysis was then performed between clusters using the DESeq2 (Galaxy version 2.11.40.8+galaxy0) pipeline, and a final heatmap was generated using the pHeatmap package (v.1.0.12) in RStudio, limited to the list of genes that were differentially expressed between the three clusters. Targeted and genome-wide mutations were downloaded from https://cancer.sanger.ac.uk/cosmic/download, filtered by gene CTNNB1, from COSMIC release v.94 (28 May 2021). Mutations labeled ‘Substitution – Missense' were extracted, excluding one multiple nucleotide polymorphism (p.I35_H36delinsSN). Adaptors were trimmed and paired ends merged with NGmerge (https://github.com/harvardinformatics/NGmerge) to produce single-end reads. Mean sequence quality (Phred) scores were high (>30) across all base positions targeted for mutagenesis, so low-quality reads were not filtered out. Single reads were then aligned with bwa mem (v.0.7.17)55 to the 162 bp CTNNB1 reference sequence. A set of reads with a single missense on-target mutation, and no other mutations, was generated. Reads that did not fully cover the region targeted for mutagenesis (58–111 bp) were excluded, as were alignments with insertions and deletions anywhere, no mutations in the target region, mutations only outside the target region, multiple mutations in the target region, synonymous mutations only or mutations resulting in a codon that was not in the repair template library. The remaining reads had precisely one missense mutation specified from the HDR template library. Read counts were normalized within each of the six experimental bins and two control conditions by dividing the number of reads for each mutation by the total number of filtered reads in that bin, such that the sum of normalized counts for all mutations in the bin was equal to 1. To calculate enrichment values relative to the starting population, the normalized count for each mutation in each bin was then divided by the normalized count for the same mutation in cells that had undergone the same genome editing and selection procedure but had not undergone flow sorting based on GFP. Replicates had high Pearson correlation (0.54–0.89 across GFP bins; Supplementary Fig. 3), so reads were merged for downstream analysis. Plasmid and unselected pool codon frequencies had a Pearson correlation of 0.63. These mutations are marked as ‘lower confidence' in Supplementary Table 2. None of the lower confidence mutations were observed among patients in TCGA or Montironi HCC cohorts. VEP scores were obtained for all β-catenin single amino acid substitutions spanning amino acids 31–48 from 50 different methods, using a published pipeline26. Note that some predictors only output scores for missense variants possible by single nucleotide changes, meaning that the correlations were calculated from fewer mutations; however, this has been shown to have little effect on overall correlations or relative predictor rankings56. To investigate amino acid properties potentially related to the effects of mutations at I35, we downloaded all 566 indices from the AAIndex database35,37. These correlations are provided in Supplementary Table 8. Peptide sequences for previously reported βTrCP-dependent degrons were extracted from Table S1 of a previous publication37, then filtered for a precise match to the DSGX motif spanning positions 32–35 of β-catenin (n = 28 peptides, detailed in Supplementary Table 4). Samples from TCGA-LIHC cohort (n = 370) were filtered into one of six groups (Supplementary Table 9): (1) ‘strong': those with a hotspot missense mutation with a MES value of >18,000 or MES < 18,000 plus copy number gain defined in cBioportal (n = 53); (2) ‘weak': those with a hotspot missense mutation with a MES value of <18,000 and no copy number gain (n = 27); (3) ‘other': those with a CTNNB1 missense mutation outside the hotspot region (n = 19); (4) ‘AXIN1': those with any coding mutation in AXIN1 (n = 36); (5) ‘no mutation': those without any coding mutation in CTNNB1 or AXIN1 (n = 217); and (6) ‘exclude': those with deletions and/or complex mutations in CTNNB1, those falling into more than one of groups (1) to (4) and those lacking survival and/or RNA-seq data (n = 18). Group (6) was excluded from further analysis. For the Montironi cohort, we considered only patients with single missense mutations in the exon 3 hotspot (n = 44). Of those, 31 had mutations classified as strong based on the same classification system described above, including n = 2 with MES < 18,000 plus copy number gain, and 13 had mutations classified as weak. Whole-slide images of haematoxylin and eosin (H&E)-stained, formalin-fixed, paraffin-embedded sections (‘diagnostic slide') of TCGA-LIHC cohort were viewed using the NCI GDC Data Portal slide viewer; cases for which only a frozen section image (‘tissue slide') was available were not evaluated. Each case was scored by an expert consultant liver histopathologist and National Liver Pathology External Quality Assurance scheme member working at the national liver transplant center (T.K. Intratumoral inflammation was scored using H&E morphology alone using a four-point ordinal scale57: 0, no inflammation; 1, scattered inflammatory cells; 2, focal aggregates of inflammatory cells; and 3, diffuse or nodular aggregates of inflammatory cells. To compare transcriptomes of HCC samples from the categories shown in Fig. 5b, STAR transcript quantification for TCGA HCC samples (n = 361) was downloaded from the GDC Data Portal. Differential expression of samples in each category compared to the weak MES missense mutation was calculated with DeSeq2 (v.1.34)58 using shrunken log-fold changes. Gene set enrichment analysis was performed to investigate the functional enrichment of the differentially expressed genes identified from RNA-seq data. The ranked list of differentially expressed genes was generated based on the DESeq2 statistics, which takes into account both fold-change and P value information. Gene set enrichment analysis was performed on the Gene Ontology database, which consists of three structured, controlled vocabularies. The full output of this analysis is shown in Supplementary Table 6. For the analysis of β-catenin target genes, ten known targets (based on published literature) were identified among the 50 most upregulated transcripts in strong versus no mutation tumor groups from TCGA HCC dataset. We modeled expression as predicted by gene identity, HCC subgroup and their interaction using negative binomial models (R package MASS). We then obtained P values comparing expression in each HCC subgroup to ‘strong' as the baseline using marginal means testing implemented in the R package emmeans. RNA-seq data from the Montironi cohort were generated and processed as previously described14. Hallmark expression scores were generated using the Wnt β-catenin Hallmark signature, using the ssGSEA pipeline implemented in Genepattern60,61. For human liver organoid data, normalized RNA-seq counts (fragments per kilobase per million) were downloaded from the Gene Expression Omnibus (Accession ID GSE236490). Expression values were averaged for n = 2 biological replicates corresponding to each mutation. Exome sequence variants from TCGA-LIHC (n = 375) and TCGA-UCEC (n = 404) were downloaded from the GDC Data Portal (https://portal.gdc.cancer.gov). Variants called by at least two of the four provided workflows (MuSE, MuTect2, SomaticSniper, VarScan2) were retained. Cases with one or more missense mutations in the 31–48 amino acid target region were extracted (TCGA-LIHC, n = 82; TCGA-UCEC, n = 104), and the single-nucleotide polymorphisms from these were used to generate tri-nucleotide mutation frequencies with SomaticSignatures62. The results shown are based upon data generated by the TCGA Research Network (https://www.cancer.gov/tcga). All possible mutational paths from one codon to another, one nucleotide change at a time, were generated for all codon pairs. The MLS of each codon change in every possible tri-nucleotide context is the sum of all paths between the two codons; similarly, the MLS for each amino acid change in its tri-nucleotide context is the sum of its codon change scores. A worked example is shown in Supplementary Fig. For statistical testing, we modeled observed mutation frequencies as the result of MLS in the same tumor type, using a negative binomial generalized linear model implemented in the R package MASS. We compared these fits to null models fitting only the mean using likelihood ratio tests. Statistical tests were performed in R (v.4.2.2) using RStudio (v.2022.12.0). All tests were two-sided unless stated otherwise. Sample size for the SGE screen was determined by the number of possible missense mutations between positions 31 and 48 of β-catenin, and all 342 variants were included. Sample sizes for clinical studies were determined by the number of patient samples available in relevant cohorts that could be unambiguously assigned to one category based on CTNNB1 pathway mutation status. No blinding or randomization was performed. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. RNA-seq data from genome-edited mouse ES cell lines with individual CTNNB1 mutations are available from the Gene Expression Omnibus under accession code GSE299075. TCGA data are accessible through cBioportal. RNA-seq and whole-exome sequencing data for the Montironi cohort have been deposited at the European Genome–Phenome Archive using accession code EGAS00001005364. Source data are provided with this paper. Custom scripts used for data analysis in this manuscript are available at https://doi.org/10.5281/zenodo.17940132 (ref. The ‘just-right' signaling model: APC somatic mutations are selected based on a specific level of activation of the β-catenin signaling cascade. The Apc 1322T mouse develops severe polyposis associated with submaximal nuclear β-catenin expression. Feng, Y. et al. Tissue-specific effects of reduced β-catenin expression on adenomatous polyposis coli mutation-instigated tumorigenesis in mouse colon and ovarian epithelium. Kielman, M. F. et al. Apc modulates embryonic stem-cell differentiation by controlling the dosage of β-catenin signaling. Celen, I., Ross, K. E., Arighi, C. N. & Wu, C. H. Bioinformatics knowledge map for analysis of β-catenin function in cancer. Gao, C. et al. Exon 3 mutations of CTNNB1 drive tumorigenesis: a review. Cancer Genome Atlas Research Network.Comprehensive and integrative genomic characterization of hepatocellular carcinoma. Lachenmayer, A. et al. Wnt-pathway activation in two molecular classes of hepatocellular carcinoma and experimental modulation by sorafenib. Hoshida, Y. et al. Integrative transcriptome analysis reveals common molecular subclasses of human hepatocellular carcinoma. Montironi, C. et al. Inflamed and non-inflamed classes of HCC: a revised immunogenomic classification. Ruiz de Galarreta, M. et al. β-Catenin activation promotes immune escape and resistance to anti-PD-1 therapy in hepatocellular carcinoma. Muller, M. et al. Human-correlated genetic models identify precision therapy for liver cancer. Provost, E. et al. Functional correlates of mutations in β-catenin exon 3 phosphorylation sites. Correlation between β-catenin mutations and expression of Wnt-signaling target genes in hepatocellular carcinoma. Rebouissou, S. et al. Genotype-phenotype correlation of CTNNB1 mutations reveals different β-catenin activity associated with liver tumor progression. Findlay, G. M., Boyle, E. A., Hause, R. J., Klein, J. C. & Shendure, J. Saturation editing of genomic regions by multiplex homology-directed repair. Accurate classification of BRCA1 variants with saturation genome editing. Saturation genome editing maps the functional spectrum of pathogenic VHL alleles. Ferrer-Vaquer, A. et al. A sensitive and bright single-cell resolution live imaging reporter of Wnt/β-catenin signaling in the mouse. Czechanski, A. et al. Derivation and characterization of mouse embryonic stem cells from permissive and nonpermissive strains. Kosuri, S. et al. Composability of regulatory sequences controlling transcription and translation in Escherichia coli. Updated benchmarking of variant effect predictors using deep mutational scanning. Frazer, J. et al. Disease variant prediction with deep generative models of evolutionary data. Geurts, M. H. et al. One-step generation of tumor models by base editor multiplexing in adult stem cell-derived organoids. Johnson, J. L. et al. An atlas of substrate specificities for the human serine/threonine kinome. & Kinzler, K. W. Phosphorylation of β-catenin at S33, S37, or T41 can occur in the absence of phosphorylation at T45 in colon cancer cells. & Neufeld, K. L. Oncogenic serine 45-deleted β-catenin remains susceptible to Wnt stimulation and APC regulation in human colonocytes. Amit, S. et al. Axin-mediated CKI phosphorylation of β-catenin at Ser 45: a molecular switch for the Wnt pathway. Kawashima, S. et al. AAindex: amino acid index database, progress report 2008. Low, T. Y. et al. A systems-wide screen identifies substrates of the SCFβTrCP ubiquitin ligase. Blokzijl, F. et al. Tissue-specific mutation accumulation in human adult stem cells during life. Alexandrov, L. B. et al. Signatures of mutational processes in human cancer. & Lopez-Bigas, N. In silico saturation mutagenesis of cancer genes. Liu, P. et al. Oncogenic mutations in armadillo repeats 5 and 6 of β-catenin reduce binding to APC, increasing signaling and transcription of target genes. Differential effects of inactivated Axin1 and activated β-catenin mutations in human hepatocellular carcinomas. Abitbol, S. et al. AXIN deficiency in human and mouse hepatocytes induces hepatocellular carcinoma in the absence of β-catenin activation. Nault, J. C. et al. Telomerase reverse transcriptase promoter mutation is an early somatic genetic alteration in the transformation of premalignant nodules in hepatocellular carcinoma on cirrhosis. Spranger, S., Bao, R. & Gajewski, T. F. Melanoma-intrinsic β-catenin signalling prevents anti-tumour immunity. von Felden, J. et al. Mutations in circulating tumor DNA predict primary resistance to systemic therapies in advanced hepatocellular carcinoma. Prospective genotyping of hepatocellular carcinoma: clinical implications of next-generation sequencing for matching patients to targeted and immune therapies. Wong, K. et al. Wnt/β-catenin activation by mutually exclusive FBXW11 and CTNNB1 hotspot mutations drives salivary basal cell adenoma. The ground state of embryonic stem cell self-renewal. Ran, F. A. et al. Genome engineering using the CRISPR–Cas9 system. Sensitive and powerful single-cell RNA sequencing using mcSCRB-seq. Kenkhuis, B. et al. Iron accumulation induces oxidative stress, while depressing inflammatory polarization in human iPSC-derived microglia. Vasimuddin, M., Misra, S., Li, H. & Aluru, S. Efficient architecture-aware acceleration of BWA-MEM for multicore systems. Using deep mutational scanning to benchmark variant effect predictors and identify disease mutations. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Gehring, J. S., Fischer, B., Lawrence, M. & Huber, W. SomaticSignatures: inferring mutational signatures from single-nucleotide variants. Meynert, A.M. & Wood, A. Mutational scanning reveals diverse effects of oncogeneic CTNNB1 mutations on signalling. and an MRC Unit Award to the MRC Human Genetics Unit); by the BBSRC, UK (BB/P013732/1) to P.H. ; by a Wellcome Trust Sir Henry Dale Fellowship (102560/Z/13/Z) to A.J.W. ; and by the National Institutes of Health (R01DK12782, R01HD035455, P30CA008748) to A.K.H. J.M.L is supported by grants from European Commission (Horizon Europe-Mission Cancer, THRIVE, Ref. 101136622), the National Institutes of Health (R01-CA273932-01, R01DK56621 and R01DK128289); Samuel Waxman Cancer Research Foundation; the Spanish National Health Institute (MICINN, PID2022-139365OB-I00, funded by MICIU/AEI/10.13039/501100011033 and FEDER); Cancer Research UK (CRUK), Fondazione AIRC per la Ricerca sul Cancro and Fundación Científica de la Asociación Española Contra el Cáncer (FAECC) (Accelerator Award, HUNTER, Ref. C9380/A26813); “la Caixa” Foundation (Agreement LCF/PR/SP23/52950009); Fundación Científica de la Asociación Española Contra el Cáncer (FAECC; Proyectos Generales, Ref. was supported by the Fundació de Recerca Clínic Barcelona–IDIBAPS and by a grant from the Spanish National Health Institute (MICINN, PID2022-139365OB-I00). Anagha Krishna, Shahida Sheraz, Peter Hohenstein & Derya D. Ozdemir MRC Human Genetics Unit, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK Alison Meynert, Martijn Kelder, Ailith Ewing, Gillian CA Taylor, Philippe Gautier, Graeme Grimes, Hannes Becher, Ryan Silk, Colin A. Semple, Joseph A. Marsh & Andrew J. Liver Cancer Translational Research Laboratory, Institut d'Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Hospital Clínic, Universitat de Barcelona, Barcelona, Spain Agavni Mesropian, Albert Gris-Oliver, Roser Pinyol & Josep M. Llovet Agavni Mesropian, Albert Gris-Oliver, Roser Pinyol & Josep M. Llovet Conny Brouwers, Jill WC Claassens, Margot M. Linssen & Peter Hohenstein Developmental Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA Centre for Inflammation Research, Institute for Regeneration and Repair, University of Edinburgh, Edinburgh, UK Cancer Research Scotland Centre, Edinburgh, UK Cancer Research Scotland Centre, Glasgow, UK Mount Sinai Liver Cancer Program, Divisions of Liver Diseases, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain Koç University Research Center for Translational Medicine (KUTTAM), Istanbul, Turkey Koç University School of Medicine, Istanbul, Turkey Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and S.S. performed the wet lab experiments. Correspondence to Peter Hohenstein, Andrew J. has received sponsored lecture fees from Altos Labs and serves as a consultant for Gemini Law. has served as a consultant or advisory board member for Resolution Therapeutics, Clinnovate Health, HistoIndex, Fibrofind, Kynos Therapeutics, Perspectum, Concept Life Sciences, Servier Laboratories, Taiho Oncology and Jazz Pharmaceuticals, and has received speakers' fees from Servier Laboratories, Jazz Pharmaceuticals, AstraZeneca, HistoIndex and Incyte Corporation. has received research support from Genentech & Roche, consulting and sponsored lecture fees from Eisai, Merck, Roche, Genentech, AstraZeneca, Bayer Pharmaceuticals, AbbVie, Sanofi, Moderna, Glycotest, Exelixis and Boehringer Ingelheim, and is on the Data Safety Monitoring Board for Bristol Myers Squibb. All other authors declare no competing interests. Nature Genetics thanks Lea Starita and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signaling. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01742-y'>The filamentous ultrastructure of the PopZ condensate is required for its cellular function</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 10:54:27
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Biomolecular condensates have key roles in regulating cellular processes. Yet, the relationship between atomic features and condensate function remains poorly understood. We studied this relationship using the polar organizing protein Z (PopZ). Here, we revealed hierarchical assembly of PopZ into a filamentous condensate by integrating cryo-electron tomography, biochemistry, single-molecule techniques and molecular dynamics simulations. The PopZ helical domain drives filamentation and condensation, while the disordered region inhibits them. Phase-dependent conformational changes prevent interfilament contacts in the dilute phase and expose client-binding sites in the dense phase. Perturbing filament formation in vitro alters the dynamics of scaffold and client proteins and the condensate's wetting behavior. In cells, perturbing either filament formation or the ability of filaments to condense impairs PopZ function and leads to growth phenotypes. These findings establish a multiscale framework linking molecular interactions and condensate ultrastructure to cellular function. Biomolecular condensates are nonstoichiometric assemblies of macromolecules that concentrate specific components while excluding others1,2,3. Their formation is driven by multivalent interactions implemented through intrinsically disordered regions (IDRs), repeated oligomerization motifs or nucleic acid chains4,5. These dynamic assemblies are pervasive across biology, serving as a fundamental mechanism of cellular organization and mediating an array of cellular functions, including transcription regulation6, signal transduction7 and protein quality control8,9. These emergent properties—arising from the collective behavior of many molecules—include viscoelasticity13, surface tension14 and electrochemical potential15. Despite their importance, it remains a major challenge to link condensate-level properties to the molecular-level features from which they originate. A particularly understudied question is how condensate function relates to its internal structure4,16. By internal structure, we refer to stable structures on the molecular and supramolecular level, larger than single molecules but smaller than the condensate itself. A notable supramolecular structure that has been observed for several condensates across different cellular pathways is the filament17. Unlike cytoskeletal filaments, which are long and designed for mechanical stability18, these condensate-associated filaments are typically short and flexible. The prevalence of filamentous condensates is illustrated by systems such as p62/SQSTM1 (hereafter p62)19, DAXX/SPOP20 and F-actin21, underscoring how useful a detailed characterization of this class of condensates could be. To determine whether and how filamentation impacts condensate properties and function, it is essential to examine the individual and collective behavior of macromolecules across all relevant length scales—from molecular to filament to condensate level—in both dispersed and condensed states. Here, we elucidated the structure–function relationship of a condensate using in vivo and in vitro techniques, including cryo-electron tomography (cryo-ET), biochemical assays, single-molecule Förster resonance energy transfer (smFRET) and computational simulations. The polar organizing protein Z (PopZ)22,23,24, a conserved protein within α-proteobacteria25, has a crucial role in cytosol organization. First identified in Caulobacter crescentus26,27,28, a well-studied model system for asymmetric cell division, PopZ forms condensates at the ends (poles) of the cell. These condensates selectively recruit cell-cycle-regulated proteins, establishing signaling gradients essential for driving asymmetric cell division29. Deletion of the popZ gene in C. crescentus results in abnormal cell division, disrupted chromosome segregation and loss of cellular asymmetry23,24. Furthermore, aberrations in the condensate's material properties (too fluid or too solid) lead to severe growth defects30. Beyond its role in C. crescentus, PopZ forms condensates across various systems, including in vitro, multiple bacterial species and human cells; it is also amenable to genetic engineering30. In this study, we used an integrative approach to decode the structural and chemical determinants of PopZ condensate formation. We uncovered a hierarchical pathway in which PopZ assembles into trimers, hexamers, filaments and condensates. We characterized the conformational dynamics of individual PopZ molecules in both dilute and dense phases and observed phase-dependent conformational changes. Our studies indicate that filamentation promotes condensate sphericity and slows down movements of PopZ and its clients in vitro and is tightly linked to PopZ function in vivo. While other filamentous condensates rely on separate proteins for filament formation, capping and crosslinking, PopZ integrates all three functions into a single 77-residue helical domain at its C terminus. Our work establishes a mechanistic analysis framework to understand how molecular-level features propagate across length scales and establishes a foundation for characterizing key attributes of filamentous condensates. To investigate the supramolecular structure of PopZ within condensates, we purified recombinant wild-type (WT) PopZ, triggered condensation by adding 50 mM MgCl2 and visualized the resulting condensates by cryo-ET (Fig. PopZ condensates were composed of filaments (Fig. We quantified filament properties by segmenting tomograms with the software DragonFly (Comet Technologies)31 and tracing individual filaments using Amira (Thermo Fisher) (Supplementary Video 1). Filament lengths followed an exponential distribution (R2 = 0.99) with median and mean lengths of 34 nm and 38 nm, respectively (Fig. This distribution indicates isodesmic polymerization, a noncooperative filament formation where the association constant of building blocks is independent of filament length. In isodesmic polymers, there are no distinct nucleation and elongation phases, as the bonds between building blocks are identical and the affinity between them does not depend on their position within the filament32,33. Therefore, characterizing the PopZ filament reduces to identifying its building blocks and the bonds connecting them. a, Phase-contrast imaging and cryo-ET of in vitro PopZ condensates. Top left, phase-contrast image of PopZ condensates formed by adding 50 mM MgCl2 to 5 µM of purified PopZ in 10 mM sodium phosphate (pH 7.4) with 150 mM NaCl. A zoomed-in view is provided for illustrative purposes. Top middle, cryo-ET tomogram of 2 µM purified WT PopZ in the same buffer, captured with ×53,000 magnification. Top right, a 3D rendering of segmented tomograms. Bottom, six zoomed-in regions illustrating the flexibility of filaments in PopZ condensates. Yellow arrows highlight filamentous structures. b, Filament length distribution fits an isodesmic model. Length distribution of over 8,000 filaments from three independent condensates (Extended Data Fig. The turquoise line indicates a fit to an isodesmic model at the mean. c, Cryo-ET reveals filaments within PopZ condensates in C. crescentus. Representative cryo-ET images of the PopZ microdomain in ΔpopZ C. crescentus cells expressing mCherry-tagged WT PopZ. Filaments can be seen in the zoomed-in panel. Additional reconstructions and representative filament images are shown in Extended Data Fig. To investigate the relationship between filamentation and condensation, we analyzed cryo-ET tomograms of WT PopZ in the dilute phase (regions without condensates; Extended Data Fig. In the dilute phase, we observed fewer and shorter filaments, along with numerous speckles, which likely correspond to PopZ hexamers. These observations confirm that the increased concentration in the dense phase promotes filamentation (Extended Data Fig. To confirm that WT PopZ forms filaments in vivo, we vitrified C. crescentus cells expressing mCherry-tagged WT PopZ and used cryo-focused ion beam (cryo-FIB) milling to create lamellae thinner than 200 nm. This reduction in specimen thickness, combined with high-magnification cryo-ET, yielded a detailed visualization of C. crescentus cells. We observed the previously described PopZ microdomains at the cell poles, which were devoid of large macromolecules, such as ribosomes (Fig. Within microdomains, we identified filamentous structures consistent with WT PopZ filaments we observed in vitro and in agreement with findings by Toro-Nahuelpan et al., who visualized PopZ filaments in Magnetospirillum gryphiswaldense and C. crescentus34. Taken together, our structural characterization reveals that the underlying substructure of the PopZ condensate in vivo and in vitro consists of a meshwork of interwoven filaments, which themselves have a broad length distribution. To characterize the driving forces of PopZ condensation, we screened different salt and pH conditions for their ability to trigger droplet formation. Divalent cations such as MgCl2 at physiological concentrations (2–5 mM) are sufficient for condensation, with higher concentrations accelerating assembly30. Low pH (<6.0) or increased concentrations of salt (>300 mM NaCl) also promote condensation (Extended Data Fig. We tested three buffering agents—sodium phosphate, HEPES and Tris-HCl—all of which supported droplet formation. PopZ formed smaller droplets in Tris-HCl buffer and occasionally formed nonspherical, aggregate-like clusters in a buffer lacking salt and divalent cations (Extended Data Fig. Using a physiologically relevant buffer with lower magnesium concentration and a crowding agent (10 mM sodium phosphate pH 7.4 with 150 mM NaCl, 1 mM MgCl2 and 5% PEG), we observed robust droplet formation. PopZ's modular structure includes a C-terminal helical OD necessary for condensation29,35,36, a negatively charged, proline-rich IDR that imparts fluidity to the condensates30 and an N-terminal helical region (H1) that recruits binding partners into the microdomain37,38 (Fig. We reasoned that conditions promoting condensation screen repulsive interactions driven by the IDR. We previously showed that the length and charge distribution of the IDR affect the material properties of PopZ condensates in vivo30, indicating that the IDR influences condensate function. To test whether the IDR also affects its formation, we engineered a tobacco etch virus (TEV)-cleavable construct (H1–IDR–TEV–OD). Before cleavage, this construct mirrored WT PopZ's behavior, forming droplets only under condensing conditions. TEV cleavage separated the IDR from the OD, resulting in droplets made up entirely of OD and supernatant containing H1–IDR (Fig. Fluorescent labeling confirmed that OD droplets excluded the H1–IDR (Fig. Thus, when covalently attached to the OD, H1–IDR inhibits condensation through a mechanism that can be overcome using low pH, addition of divalent cations or high salt concentrations (more information in the Supplementary Text). a, Schematic of the PopZ construct. The construct includes an N-terminal 6×His fusion (brown) fused to PopZ from C. crescentus, which codes for a short helical N-terminal region, H1 (pink box), a 78-aa IDR (gray curly line) and a C-terminal region comprising three predicted helices: H2, H3 and H4 (blue boxes). Positions −20 and 132 (indicated by asterisks) were used to covalently attach fluorescent dyes for imaging and smFRET studies. b, AlphaFold 3 model of the PopZ monomer shown for illustrative purposes. A TEV cleavage site, introduced between residues 99 and 100, is present in our PopZ construct only when specified. c, Representative SDS–PAGE gel with Coomassie staining. TEV protease was added to TEV-cleavable PopZ in vitro and incubated for 1 h before pelleting down the dense phase (2,000g, 3 min). MW, molecular weight; P, pellet fraction, which contains the OD; SN, supernatant, which contains the IDR. The IDR migrates as a larger species because of its stretches of densely packed negative charges. The experiment was repeated twice with similar results. d, Phase and fluorescence images 30 min after incubation with TEV protease. The formed droplets (phase) are composed of cleaved OD (OD–AF488) and exclude cleaved H1–IDR (H1–IDR–AF647). Experiments were repeated twice with similar results. e, OD condensates are made up of filaments. Left, representative cryo-ET tomogram of 5 µM H1–IDR–TEV–OD cleaved for 3 min with TEV protease in 20 mM Tris-HCl pH 8.0, 0.5 mM EDTA and 2 mM DTT. A total of three tomograms were recorded (Extended Data Fig. To determine whether condensates formed by isolated OD are also composed of filaments, we inspected them using cryo-ET. Without MgCl2, these OD condensates exhibited an internal structure resembling that of WT PopZ condensates (Fig. 2e), indicating that the OD alone is sufficient for filament formation. When we analyzed the length distribution of OD filaments, we found that it closely matched that of WT PopZ, with both median and average lengths only 3–5 nm shorter (Extended Data Fig. Because this difference is smaller than the size of the building block itself, as discussed later, we conclude that the IDR has minimal influence on filament length. The mechanism of condensation through interactions between structured filaments and their inhibition through an IDR remains largely underexplored, although similar mechanisms have been documented in a few systems17,19,21,39 (Discussion). Indeed, current phase-separation predictors did not capture PopZ's ability to condense (Supplementary Data 2). Thus, there is a need for predictors that can account for condensation driven by mechanisms involving polymerization, as used by PopZ. To study the assembly pathway leading to PopZ filaments, we used mass photometry40,41, where WT PopZ at 250 nM concentrations adopted trimers and hexamers (Fig. Buffer conditions and salt concentrations modulate the populations of these oligomers but both species were consistently present (Supplementary Data 3). The addition of 50 mM MgCl2 promoted the formation of higher-order oligomers such as 12-mers and 18-mers (Fig. 3a, middle), suggesting that PopZ forms filaments by stacking hexamers before condensation. Consistent with this interpretation, low pH and increased salt concentrations also promoted hexamer stacking (Supplementary Data 3). These conditions likely screen the repulsive negative charges of the IDR and thereby allow ODs of different hexamers to approach each other and interact. a, Mass photometry analysis of purified PopZ variants at 250 nM. Left, WT PopZ (gray) forms trimers and hexamers, while the P146A variant (magenta) is trapped in the trimeric state and the ΔL6 variant (tan) is trapped in the hexameric state. Middle, comparison of WT PopZ in the presence (turquoise) and absence (gray) of MgCl2 in 10 mM sodium phosphate pH 7.4 + 150 mM NaCl, showing that MgCl2 promotes higher-order oligomerization. Right, relative populations of monomer, trimer and hexamer across several PopZ variants, including WT (gray), ΔL6 (tan), P146A (magenta), Δloop (light gray), ΔH2 (green) and OD* (P146A;W151G;L152S, blue). Dots show the population of each oligomer for each of two biological replicates. b, AUC of PopZ variants at approximately 100 µM in 10 mM sodium phosphate pH 7.4 + 150 mM NaCl shows that ΔL6 (tan) remains hexameric even at high concentrations, whereas P146A (magenta) and WT (gray) form higher-order oligomers, with WT assembling into the largest species. c, ΔL6-PopZ forms condensates made up of hexamers. Left, phase-contrast image of ΔL6-PopZ condensates in solution. Middle, cryo-ET tomogram of a ΔL6-PopZ condensate formed by adding 50 mM MgCl2 to 50 µM of purified ΔL6-PopZ in 10 mM sodium phosphate pH 7.4 + 150 mM NaCl. Right, a 3D rendering of the traced ΔL6-PopZ tomograms. d, Phase-contrast (top) and fluorescence (middle) microscopy show that ΔL6-PopZ condensates wet the surface of glass coverslips, whereas WT PopZ condensates (bottom) largely remain spherical over a period of 4 h. Scale bars, 5 µm. e, Left, FRAP experiments showing the recovery profiles of AF647-labeled PopZ for different condensate compositions. is shown for two biological replicates, each averaged over at least nine condensates. The solid lines indicate fits to a simple exponential model. Right, representative examples of droplets used for FRAP experiments. f, Deletion of the L6 region impacts client protein (ChpT) diffusivity inside the condensate. Left, FRAP recovery curves of ChpT inside WT PopZ condensates (gray) and ΔL6 condensates (tan). is shown for two biological replicates, each averaged over at least 12 condensates. The solid lines indicate fits to a two-phase exponential model. Right, representative examples of droplets used for FRAP experiments. To further elucidate the determinants of oligomeric assembly, we examined two PopZ variants shown to disrupt oligomerization in native gel electrophoresis and multiangle light scattering35: P146A and Δ172–177 (hereafter referred to as ΔL6, indicating the deletion of the last six residues). At 250 nM, P146A formed only trimers, whereas ΔL6 assembled exclusively into hexamers (Fig. Addition of MgCl2 did not induce higher-order oligomerization in either variant (Supplementary Data 3). Surprisingly, using light microscopy, we observed that both variants still formed condensates, albeit at higher saturation concentration (csat) than the WT, approximately 2 µM for P146A and 20 µM for ΔL6 versus ~0.5 µM for WT PopZ (Extended Data Table 1). We, therefore, studied the relationship between filamentation and condensate formation. Mass photometry is unsuitable for monitoring oligomerization at micromolar concentrations, as it can no longer distinguish between a single large particle and two smaller particles landing close to each other. To study the oligomerization of dispersed PopZ in the micromolar range, we used analytical ultracentrifugation (AUC)42. AUC revealed distinct oligomerization patterns for P146A, ΔL6 and WT PopZ (Fig. WT PopZ (gray) formed the largest species, which we attributed to filaments of varying lengths. P146A (magenta) also formed higher-order oligomers, albeit smaller ones than WT PopZ, while ΔL6 (tan) did not exceed a sedimentation coefficient of 3, consistent with hexamers. Unexpectedly, WT PopZ populated a single species with a large s value rather than a distribution similar to that observed within the condensate by cryo-ET (Fig. This may result from higher concentrations required for AUC (100 µM) compared to cryo-ET and physiological conditions (5 µM), possibly leading to the formation of supramolecular clusters that limit filament growth. Because all three variants could form condensates, these data suggest that either filamentation is not strictly required for condensation or that ΔL6 requires magnesium, salt or low pH to form filaments. To determine whether ΔL6-PopZ condensates are made up of filaments, we determined their ultrastructure using cryo-ET. Unlike WT PopZ and OD, ΔL6-PopZ condensates were composed of discrete hexamers (Fig. 3c), indicating that L6 is required for stacking the hexamers into filaments. This suggests that, while ΔL6-PopZ can condense without forming filaments, the reduced multivalency of hexamers compared to filaments likely causes the observed increase in csat. The tendency of a liquid to minimize its surface area by forming a sphere is related to its surface tension14,43. 1a) appeared spherical with clear boundaries, indicative of high surface tension. In contrast, ΔL6 droplets had irregular boundaries and were deformed upon contact with the glass surface, displaying enhanced wetting behavior (Fig. 3d; comparison of WT, OD and ΔL6 in Extended Data Fig. These observations indicate that loss of filamentation drastically affected surface tension. Viscosity, another biologically important material property, relates to the diffusivity of molecules within condensates, which is often probed using fluorescence recovery after photobleaching (FRAP)44. To test whether filamentation affects the diffusivity of PopZ, we incubated fluorescently labeled PopZ with WT, ΔL6 or a mix of both and performed FRAP (fitting details in Supplementary Data 4). In WT PopZ condensates (that is, filaments), recovery was negligible after 20 min. Adding hexameric ΔL6 enabled partial recovery, with a 40% mobile fraction and a 27-min half-time. This effect was amplified in ΔL6-only condensates, which had a 90% mobile fraction and 10-min recovery half-time (Fig. Thus, PopZ diffuses more freely in hexamer-based condensates than in filamentous ones. These results provide a structural basis for emergent properties of the condensate. Next, we addressed whether filament loss affects the diffusivity of condensate clients, defined as components that partition into and reside within preformed condensates45. In C. crescentus, PopZ interacts with clients through its H1 (refs. 37,46), which is distant in sequence from the OD and the last six residues. We examined the recruitment of the client ChpT47, an essential mediator within C. crescentus's cell-cycle circuit that directly binds to PopZ29. ChpT labeled with AF488 showed recovery after photobleaching in WT, mixed and ΔL6 droplets (Fig. Recovery kinetics required a two-phase model, combining diffusion and binding/unbinding dynamics between ChpT and PopZ. Increasing the hexamer fraction (and, thus, decreasing the filament fraction) again sped up recovery and increased the mobile fraction (Supplementary Data 4). These results indicate that filamentous WT slows down ChpT movement to a greater extent than hexameric ΔL6-PopZ. Because WT PopZ can form filaments without condensing and ΔL6-PopZ can condense without forming filaments, we conclude that filamentation and condensation, while strongly coupled, are two distinct processes governed by different residues or driving forces. Above, we found that PopZ forms trimers, hexamers and filaments. A structural description of these oligomeric states and how they interact to form condensates would provide deeper insights into PopZ biology and allow targeted sequence modifications for engineering purposes. To identify the determinants of PopZ oligomerization, we combined mass photometry and mutagenesis with structural models computed by AlphaFold 3 (ref. AlphaFold 3 predicted that the trimeric form of PopZ features a hydrophobic core formed by a coiled coil of three H3H4 helices (residues 135–177) and stabilized by the hydrophobic effect (Extended Data Fig. 4a; summary of all structural models in Supplementary Data 5). This core remained stable when we included the H2 helix (residues 102–128) and the flexible linker connecting H2 to H3H4 (residues 129–135, which we refer to as ‘the loop'). In these predictions, the three H2 helices adopted variable orientations relative to H3H4, likely because of the flexibility of the loop connecting these segments (Extended Data Fig. To validate these predictions, we destabilized the putative hydrophobic core by substituting W151 and L152 (highlighted in Extended Data Fig. During purification, this construct was prone to aggregation and mass photometry confirmed that it existed primarily as a monomer (Fig. 3a, right, OD* in blue), confirming that W151 and L152 are crucial for the stability of the trimer. We hypothesized that the trimeric core is a conserved feature across PopZ orthologs in α-proteobacteria. 49), we predicted the structures of PopZ trimers from orthologs spanning hundreds of species across α-proteobacteria. In all cases, the predicted structures closely resembled the trimeric coiled coil of C. crescentus PopZ (Extended Data Fig. 4a), suggesting that it is a central feature of PopZ. To characterize PopZ hexamers, we combined AlphaFold 3 structural models with experimental validation. We tested the models through mutagenesis and mass photometry to determine oligomeric states and cryo-electron microscopy (cryo-EM) techniques, including subtomogram averaging and single-particle cryo-EM two-dimensional (2D) class averages, to resolve the hexamer shape and dimensions. To pinpoint the minimal hexamer-forming segment, we designed a construct (H1–IDR–TEV–OD-ΔL6) missing the last six residues and separated H1–IDR from OD-ΔL6 after TEV cleavage (Extended Data Fig. Isolated OD-ΔL6 (residues 102–171) remained soluble at concentrations below 5 µM but formed droplets at higher concentrations, with mass photometry confirming hexamer formation (Extended Data Fig. 4d), indicating that neither L6 nor H1–IDR is necessary for hexamer assembly. Mass photometry of ΔH2-PopZ (lacking residues 102–128) revealed trimer and hexamer populations comparable to those of full-length PopZ (FL-PopZ) (Fig. 3a, right, gray and green), indicating H2 is also dispensable for hexamer formation. Lastly, mass photometry of Δloop-PopZ (lacking residues 129–134) showed a complete shift of the population to trimers (Fig. 3a, right, light gray), indicating that the loop connecting H2 to H3H4 is essential for hexamer formation (summary of sequence determinants in Fig. Thus, we identified residues 129–171 as the core hexamer-forming segment. To define the hexamer shape and dimensions, we performed subtomogram averaging on hexameric ΔL6-PopZ. It resolved two predominant hexamer shapes: a straight one, measuring 100 Å in length, and a curved one, extending to 130 Å (Extended Data Fig. Disordered segments cannot usually be resolved in cryo-ET because of conformational variability, meaning the resolved shapes represent the structured OD. Modeling the hexamer-forming segment with AlphaFold 3 predicted head-to-head interactions between trimers mediated by the loop and H3 helices, with C termini projecting outward (Extended Data Fig. Comparing the hexamer and trimer models also suggests that H3H4 helices can slide with respect to each other (Extended Data Fig. Additional support for the AlphaFold 3 model came from single-particle cryo-EM 2D class averages of ΔH2ΔL6-PopZ (Extended Data Fig. 6b), although the complex's inherent flexibility limited high-resolution structural determination. Including H2 in AlphaFold 3 yielded an S-shaped model with H2 bundles replacing H3 contacts, compatible with the curved 2D classes obtained in subtomogram averaging of ΔL6-PopZ (Extended Data Fig. Together, these results indicate that H3H4 helices form trimers that interact through the loop to assemble hexamers approximately 100–130 Å in length. Furthermore, the model suggests that the hydrophobic H2 helices project outward from the hexamer center, potentially forming interoligomer contacts with H2 bundles from other PopZ particles. To gain high-resolution insights into WT PopZ filaments, we used subtomogram averaging on the segmented WT PopZ filaments. This approach proved challenging because of the heterogeneity and pronounced curvature of the filaments (Fig. To address this, we focused our analysis on a subset of straight filaments, which we isolated using a segmentation-based particle picking method. The class averages revealed that the filaments have the same width as individual hexamers, suggesting that filaments are formed through the longitudinal stacking of hexameric subunits (Extended Data Fig. Given that H2 is not required for hexamer formation (Fig. 3a), we explored its potential role in filament assembly. AUC showed that ΔH2-PopZ did not sediment beyond a coefficient of approximately 3, indicating that it remained hexameric, similar to ΔL6-PopZ (Extended Data Fig. This finding suggests that H2 contributes to filamentation either directly or indirectly. To identify the specific residues within H2 involved in filament formation, we generated partial truncations of H2, deleting either residues 102–109 or residues 110–122. Surprisingly, neither construct restored filamentation (Extended Data Fig. 4e, coral and magenta), indicating that the entire H2 is necessary for this process. Therefore, both the H2 region and the last six residues are required for filamentation, either because they directly participate in the filament-forming interactions or because they enable PopZ to adopt a conformation needed for filamentation. The terminal position of the last six residues makes them more likely to facilitate the longitudinal stacking of hexamers, while the central position of H2 suggests it may stabilize a filament-favoring conformation. Taken together, our data indicate that PopZ filaments form through longitudinal stacking of hexamers, can bend considerably and require both the H2 region and the last six residues for proper assembly. Condensation requires intermolecular interactions to become thermodynamically favorable over intramolecular interactions and solvent interactions50. Because PopZ forms supramolecular assemblies (that is, oligomers and filaments), the term intermolecular can be ambiguous. Therefore, we use the term interoligomer contacts to describe interactions between distinct PopZ particles, whether they are hexamers or filaments. To identify the regions responsible for interoligomer contacts, we performed a series of deletions along the PopZ sequence and assessed their impact on droplet formation using light microscopy. Deleting the last six residues abolished filament formation and significantly increased csat (Extended Data Table 1), likely because of reducing multivalency rather than compromising interoligomer contacts. Similarly, deletion of the loop increased csat. In contrast, deleting the H2 region (102–128) completely prevented condensation in our standard buffer (10 mM sodium phosphate pH 7.4, 150 mM NaCl and 50 mM MgCl2), even at high protein concentration (100 µM). While ΔH2-PopZ formed droplets at elevated salt or magnesium concentrations (Extended Data Table 1), these droplets dissolved within ~20 min, suggesting that H2 stabilizes interoligomer contacts critical for condensation. Furthermore, partial deletion of H2 (Δ102–109 and Δ110–122) did not restore condensation, possibly because of destabilization of the H2 helix, suggesting that H2 promotes condensation not only through specific residues but also by maintaining its helical structure. In summary, while deletions within the OD that disrupt oligomerization or filamentation also increased csat, the removal of H2 precluded condensation under standard conditions altogether. As an orthogonal approach to identify the contacts between trimers, hexamers and/or filaments that drive condensation, we conducted coarse-grained molecular dynamics (MD) simulations51,52 of PopZ self-assembly (Methods). We validated our model by comparing the simulated propensity of PopZ to undergo phase separation across several solution conditions and mutations, showing qualitative agreement with our experimental data (Extended Data Fig. While we primarily focus on the condensation of trimers, hexamer simulations gave similar results (Extended Data Fig. To better understand the molecular interactions that drive condensation, we examined the intertrimer contact map of WT PopZ during cluster formation induced by 600 mM monovalent salt (Extended Data Fig. Thus, the simulations indicate that H2 and H3H4 drive trimer–trimer interactions. It is likely that similar interactions among trimers, hexamers and/or filaments drive condensation in vivo. To experimentally validate these computational predictions, we created a library of PopZ mutants, altering in the range of 2–11 residues predicted to be solvent accessible and interaction prone according to AlphaFold 3 and our MD simulations. We substituted these residues either from sticky to inert (for example, arginine to alanine) or from sticky to repulsive (for example, arginine to glutamic acid) (Extended Data Fig. We assessed the ability of these mutants to form condensates in Escherichia coli (Extended Data Fig. 7c), finding that condensation could be disrupted by mutations in either H2 or H3H4 alone. We purified four of these constructs and confirmed by mass photometry that they could still form trimers and hexamers, suggesting that their structural integrity was preserved (Extended Data Fig. In vitro, three of four constructs retained the ability to form condensates, although their csat was increased by an order of magnitude. These results suggest that condensation does not depend exclusively on any single residue or module; rather, an ensemble of sticky residues is distributed across the H2, loop and H3H4 regions. Each of these residues contributes to lowering csat and collectively they promote condensation. In summary, both experimental and computational data suggest that interoligomer contacts are formed by multiple residues across H2, the loop and H3H4. We showed above that the H1–IDR segment inhibits OD condensation unless divalent cations, high salt or low pH are present (Fig. We labeled the same cysteine positions as in Fig. 2d (C−20 and C132) within the same molecule and verified that labeling did not disrupt oligomer or condensate formation (Extended Data Fig. For smFRET experiments, we supplemented this PopZ FRET reporter (at ~300 pM) with a 16,667-fold excess of unlabeled WT PopZ (5 µM) to prevent formation of oligomers with more than one FRET reporter, which could lead to artifacts in FRET parameters. With 151 disordered residues between the two dyes, we expected a low FRET efficiency (EFRET < 0.1) on the basis of mean distance predictions of 91 Å for a random chain55 and >100 Å for the actual sequence according to ALBATROSS, a deep learning algorithm56,57. We indeed observed a population at low EFRET ~ 0.1 (Fig. However, we also observed an intermediate EFRET state (mean EFRET > 0.5, average interdye distance of ~54 Å), suggesting that the N terminus and the OD can come much closer to one another than expected in the absence of intra-PopZ interactions. Photon distribution analysis (PDA)58 showed that 67% of PopZ molecules adopt the compacted state (intermediate EFRET). Denaturing and weakening PopZ intermolecular interactions with 6 M Gdn-HCl significantly reduced the population of the compacted state, confirming an interaction (Fig. a, smFRET reveals that PopZ adopts extended and compacted conformations in the dilute phase. FRET efficiency distribution of the smFRET reporter labeled at C−20 and C132 shows two predominant conformations in the dilute phase (light gray): extended and compacted. b, H1 drives the compacted conformation. Left, schematic illustrating two experimental conditions. Top, in the first condition, labeled FL-PopZ was mixed with an excess of unlabeled WT PopZ. For each labeled PopZ monomer (fluorophores indicated by green and red circles), two unlabeled PopZ monomers are shown, where the H1 of the unlabeled monomer competes with the H1 of the labeled monomer for a binding site on the OD (excess WT). Bottom, in the second condition, labeled FL-PopZ was mixed with an excess of unlabeled ΔH1-PopZ (missing residues 1–23). In this case, only H1 of the labeled PopZ monomer can bind the OD (excess ΔH1). Right, FRET efficiency plot shows that excess WT promotes the extended conformation (gray), similar to the denaturing condition (tan). In contrast, excess ΔH1-PopZ promotes the compacted conformation (blue). c, ChpT does not interact with PopZ in the dilute phase. Left, FRET efficiency histograms indicate that ChpT at 5 µM (tan) only marginally increased the population of the extended conformation of PopZ (gray). Right, mass photometry and sedimentation coefficient distribution from AUC for individual samples of PopZ (gray) and ChpT (magenta), as well as combined samples (tan). The combination did not indicate the formation of a new PopZ:ChpT species. Error bars in the inset represent 95% confidence interval estimated by a Jacobian in PDAFit (nWT = 3,955 bursts; nChpT = 3,162 bursts). d, H1 recruits ChpT to PopZ condensates. Fluorescence microscopy images show 1 nM ChpT labeled with AF488 recruited by FL-PopZ condensates (top), whereas ChpT remains diffuse in the presence of condensates of isolated OD (bottom). In both conditions, PopZ concentration was 5 μM. Top left, violin plots of recorded burst durations for dispersed WT PopZ (gray) and WT PopZ with 50 mM MgCl2 (turquoise). Dispersed PopZ particles do not exhibit burst durations above 15 ms. Bottom left, the 2D histograms of FRET efficiency versus burst duration. Durations greater than 15 ms (turquoise box) were selected to isolate fluorescence bursts coming from within condensates. Right, condensed PopZ exclusively adopts the extended conformation. FRET efficiency histogram of condensed and dispersed (same condition as in a) PopZ. Error bars in the inset represent 95% confidence interval estimated by a Jacobian in PDAFit (nWT = 3,955 bursts; ncondensed = 1,049 bursts). f, A model of phase-dependent interactions. In the dilute phase, H1 is predominantly bound to the OD, preventing ChpT (green) from interacting with H1. To reduce such competition, we supplemented the FRET reporter with unlabeled ΔH1-PopZ instead of unlabeled WT (Fig. This interaction is also predicted by AlphaFold 3, although the predicted local distance difference test (pLDDT) and overall model scores are low (Extended Data Fig. Given that H1 is responsible for client binding, we expected clients such as ChpT to stabilize the extended state of PopZ. Surprisingly, adding 5 µM ChpT had little effect on the extended population (Fig. 4c) despite this being a csat according to previous studies of the ChpT–PopZ interaction29,37. These two studies, however, may have altered PopZ behavior; Lasker et al.29 immobilized PopZ on an SPR surface, possibly affecting its fold and oligomerization, while Holmes et al.37 removed the OD to make PopZ amenable for NMR, thereby eliminating the compacted state entirely. To test whether ChpT binds to dispersed PopZ with orthogonal techniques that do not compromise its structural integrity, we turned to mass photometry and AUC at physiological and supraphysiological concentrations. Neither technique revealed any detectable ChpT binding to dispersed PopZ (Fig. In contrast, condensed PopZ robustly recruited AF488-labeled ChpT (Fig. Recruitment was lost in droplets of isolated OD where H1 is absent. These results suggest condensation substantially enhances ChpT–PopZ interactions. To rationalize the lack of robust ChpT–PopZ interactions in the dilute phase, we hypothesized that the H1–OD interactions in the dilute phase prevent H1–ChpT interactions, whereas condensation suppresses these interactions, enabling client recruitment. To test this, we repeated the smFRET experiment on condensed WT PopZ by adding 50 mM MgCl2. The addition of magnesium led to a drastic increase in the observed burst durations (the time a labeled molecule spends in the detection volume) compared to dispersed PopZ (Fig. We plotted the duration of fluorescence bursts versus EFRET to select only bursts belonging to slow-moving species (that is, droplets; Fig. The plot shows that condensation shifted the entire population to the extended state, where H1 and OD are far from each other (Fig. Together, these findings support a model where, in the dilute phase, H1 interacts with the OD, serving as a competitive inhibitor of ChpT binding, while condensation relieves this inhibition, allowing ChpT recruitment. We note that, even though one third of the dispersed population featured an extended H1, ChpT was unable to bind PopZ until it condensed, suggesting that even the extended H1 fraction in dispersed PopZ may not be as accessible as H1 within condensates. This mechanism combines avidity with a condensation-dependent conformational switch (Fig. Our model implies that removing H1 would promote the extended state and might allow condensation without divalent cations, salt or low pH. However, purified ΔH1-PopZ did not form spherical droplets. Instead, addition of magnesium led to aggregate-like clusters (Extended Data Fig. 8c, left), similar to those at low pH without salt or magnesium. In AUC, ΔH1-PopZ formed nonnative species that grew much larger than WT PopZ (Extended Data Fig. We propose that, without the tethering of H1 to the OD, PopZ can get trapped in aggregation-prone intermediates. The IDR likely has an important role in the formation of these intermediates, as isolated OD, which lacks both H1 and the IDR, is well behaved in vitro (Fig. To assess the role of PopZ filamentation in vivo, we expressed filament-deficient mutants in C. crescentus cells. Replacing WT PopZ, which forms polar foci, with hexameric PopZ (ΔL6-PopZ) led to a uniform cytoplasmic distribution of PopZ35, indicating that filaments are important for condensation by increasing multivalency, consistent with our in vitro findings. To determine whether filamentation confers functional benefits, we expressed low levels of mCherry–ΔL6-PopZ in otherwise WT cells, mimicking our in vitro mixing experiments (Fig. Even at low expression (Methods), mCherry–ΔL6-PopZ was toxic, completely inhibiting cell growth (Fig. Cells expressing mCherry–PopZ exhibit normal growth (top), whereas cells expressing mCherry–ΔL6-PopZ (bottom) do not grow. During chromosome replication, one copy of the ParB:parS complex remains anchored at the old pole through ParB interactions, while the other copy migrates to the new pole, associating with PopZ with assistance from a ParA gradient to facilitate rapid migration61,62. c, The effect of PopZ mutants on DNA segregation. Top, schematic illustrating the oligomerization pattern and localization of different mCherry–PopZ variants (red) expressed in WT C. crescentus cells expressing CFP-labeled ParB and endogenous WT PopZ (black). Middle, representative time-lapse images of synchronized cells at three time points, starting from swarmer cells. The cells express mCherry–PopZ variants from a plasmid, while CFP–ParB (blue) replaces WT ParB to track DNA segregation. Checkmarks or crosses indicate whether chromosome segregation proceeded properly, that is, whether ParB foci were anchored and the old pole, replicated, migrated to the new pole and were captured at the new pole within expected time frames. Bottom, kymographs displaying the cell body (black) and CFP–ParB (cyan) over time. Time progresses from left to right. To investigate this toxicity, we monitored DNA segregation using the ParABS partitioning system59,60 (Fig. Labeling the centromere-binding protein ParB with cyan fluorescent protein (CFP) allowed us to track its motion throughout the cell cycle as a probe for PopZ's ability to anchor and capture ParB foci. Low-level expression of mCherry–WT PopZ alongside native PopZ did not disrupt ParB anchoring at the old pole, migration or capture at the new pole. In contrast, low expression levels of mCherry–ΔL6-PopZ caused severe phenotypes, localizing partially at the pole and dispersed throughout the cytosol (Fig. We also observed increased dynamics, frequently exchanging between the pole and cytosol and impaired the ability to capture and anchor ParB:parS complexes at the poles (kymographs in Fig. To assess whether the observed aberrations in ParB:parS anchoring were because of impaired PopZ structure at the pole, its presence in the cytosol or both, we examined two additional mutants: Δ133–177, lacking the H3H4 region and incapable of condensation and polar localization, and ΔH1Δ133–177, which is unable to bind clients (Fig. While Δ133–177 drastically slowed down ParB motions, deletion of H1 restored normal function, suggesting that cytoplasmic PopZ that is unable to interact with clients is not harmful. However, when we deleted H1 from ΔL6-PopZ, cells were still unable to anchor or capture ParB:parS foci, indicating that ΔL6-PopZ impairs in vivo function by interfering with the WT PopZ condensate rather than through aberrant client interactions. In summary, our results indicate that filamentation is crucial for the function of PopZ condensates in vivo, not merely by enhancing multivalency but by maintaining condensate integrity essential for proper DNA segregation. In the first section, we showed that condensation promotes filamentation and, here, we demonstrated that filamentation is required for PopZ's biological function. We next asked whether PopZ filaments must reside within condensates to carry out their function or whether dispersed filaments are sufficient. Therefore, we searched within our mutant library of PopZ variants with reduced stickiness (Extended Data Fig. 7) for mutants that form filaments but fail to condense. We used cryo-ET on the mutants named Loki and Ganesha to assess whether they form filaments. Both mutants successfully formed filaments; however, at physiological PopZ concentration (5 µM), only Loki formed condensates (Extended Data Fig. 9a,b), which resembled the WT. With these and other constructs in hand, we next asked whether reducing condensation in favor of dilute-phase filaments could recover PopZ function in ΔpopZ Caulobacter cells (Extended Data Fig. In Caulobacter, dispersed PopZ is distributed throughout the cytoplasm, whereas condensed PopZ forms distinct polar foci. Deletion of popZ results in pronounced cell elongation and abnormal morphology22,23. Therefore, to evaluate the condensate-forming capability of constructs from our mutant library, we expressed each variant from an inducible promoter in ΔpopZ cells for 4 h. We then measured polar enrichment as a proxy for condensation (Extended Data Fig. 9d, left) and cell length as an indicator of restored function (Extended Data Fig. We observed a clear inverse correlation between polar enrichment (condensation) and cell length (functional rescue), indicating that reduced condensation compromises the biological activity of PopZ filaments (Extended Data Fig. Taken together, these data make a case that neither condensates made up of speckles (for example, ΔL6-PopZ) nor dispersed filaments (for example, construct Ganesha) fully support PopZ's biological function. Rather, this function requires condensates made up of filaments. The objective of this study was to characterize the relationship between the primary sequence of PopZ and its supramolecular assembly into condensates. We found that PopZ monomers form hexamers through trimerization mediated by helices H3 and H4, followed by head-to-head dimerization of trimers involving loop and helix residues, giving rise to a structure with outwardly directed C termini. The IDR and H1 helices extend from this core, modulating condensation by creating electrostatic repulsion that prevents premature interoligomer contacts. Upon charge neutralization, this repulsion is relieved, promoting filamentation and enabling condensation. Polymerization greatly enhances multivalency, while condensation further promotes polymer assembly through increased local concentrations, thereby defining the condensate's ultrastructural and physical properties, including increased viscosity and surface tension. A more detailed discussion of these findings is provided in the Supplementary Text and Table 1. a, Schematic representation of the PopZ sequence highlighting its subdomains and their roles in assembling PopZ into oligomers, filaments and condensates. (i)–(iii) PopZ oligomers in the dilute phase (dashed gray circles). (ii) AlphaFold 3 model of the PopZ trimer, showcasing the formation of a triple coiled coil through the H3H4 region (blue), with H1 helices (pink) interacting proximally to these coiled regions. Conformational flexibility is illustrated by depicting two H1 helices close to H3H4 and one further away. (iii) AlphaFold 3 hexamer model displaying interactions between two trimers (colored purple and green) mediated by intertrimer H2 region interactions. Four of the six H1 helices are shown proximal to their corresponding H3H4 regions. (iv)–(vi) PopZ filament structure in the dense phase (dashed brown circles). (iv) Fitting of three hexamers (purple and green) into a density extracted from the condensate tomogram (yellow), demonstrating a tail-to-tail organization between hexamers with IDRs extending from the center of each hexamer. In this configuration, the IDRs are positioned away from H3H4 and are involved in binding client molecules. (v) A larger segment of the tomogram (yellow) with fitted hexamers, illustrating variability in filament lengths. Unused portions of the tomogram are outlined in black. Here, only the OD of each hexamer is displayed. A scale bar is shown for each circle. Left, trimer and hexamer interactions in the dilute phase, with the gray ellipse representing a cloud of negative charge because of negatively charged residues and the flexibility of the IDR. The hierarchical and modular nature of PopZ condensation parallels other filament-driven systems such as p62, F-actin and SPOP/DAXX, highlighting common principles such as regulated polymerization, length control and multivalent crosslinking. Interestingly, PopZ filaments exhibit an intrinsic regulatory mechanism that allows condensate formation independent of external cellular factors, as demonstrated by the self-contained interplay between filament length, density and client recruitment. In α-proteobacteria, the evolutionarily tunable IDR tail provides an elegant means to regulate condensate properties crucial for cellular fitness, including viscosity for client retention and surface tension for compartmental integrity. These findings suggest broader principles for filament-driven condensates, emphasizing intrinsic polymerization control as a versatile strategy across biological systems. E. coli MG1655 strain was grown in M9G liquid medium supplemented with kanamycin (5 µg ml−1). Overnight cultures were diluted 100-fold in the same medium until they reached log phase (optical density at 600 nm (OD600) ≈ 0.5) at which point they were pelleted and resuspended to an OD600 of 0.12 in LB supplemented with kanamycin and with 0.02% arabinose to induce expression of PopZ variants from a pBAD vector. After 1 h of induction, bacteria were transferred to a pad of M9 with 1.2% agarose and 0.02% arabinose and imaged. Caulobacter strains were grown at 30 °C in peptone yeast extract (PYE) or M2G liquid medium supplemented with kanamycin (5 µg ml−1). When indicated, cells were synchronized as follows63. First, 30 ml of bacteria in log phase were pelleted, resuspended in 1 ml of cold M2, pelleted again and resuspended in 900 µl of M2 and 900 µl of Percoll (Cytiva). They were then pelleted for 20 min at 15,000g and 4 °C to separate swarmer and stalked cells. Swarmer cells were transferred into M2 and washed three times. Finally, the bacteria were resuspended in PYE and diluted to an OD of 0.12 and transferred to a pad of PYE and 1.2% agarose. Expression of PopZ variants was induced indirectly through the leaky expression from PYE medium35. PopZ was expressed and purified from E. coli strain BL21. Cells were grown in 0.5 L of Luria–Bertani medium supplemented with kanamycin (50 µg ml−1) to an OD of 0.4 at 37 °C and then switched to an induction temperature of 18 °C for 20 min before induction. PopZ expression was induced overnight with 0.1 mM IPTG. Cell pellets were collected by centrifugation and stored at −80 °C. The entire purification was conducted under denaturing conditions of 8 M urea to prevent aggregation and condensation of PopZ. The frozen cell pellet was resuspended in approximately 100 ml of lysis buffer containing 10 mM sodium phosphate pH 7.2, 10 mM Tris-HCl pH 8.0, 300 mM NaCl, 8 M urea, 20 mM imidazole and one tablet of EDTA-free protease inhibitors (Pierce) for every 50 ml of lysis buffer. Cells were lysed by sonication and insoluble material was removed by centrifugation at 12,000g for 30 min at 12 °C. The supernatant was injected onto a 5-ml HisTrap FF (Cytiva) using an AKTA Go system (Cytiva). The supernatant was reinjected once. PopZ was eluted through a step gradient of imidazole. The eluted PopZ was diluted fourfold in ion-exchange buffer A, containing 10 mM sodium phosphate pH 7.2, 10 mM Tris-HCl pH 8.0, 50 mM NaCl and 8 M urea. The diluted sample was then loaded onto a 1-ml HiTrap Q FF (Cytiva) ion-exchange column and eluted with a salt gradient. The purified protein was flash-frozen and stored at −80 °C. It was buffer-exchanged into the desired buffer immediately before experiments. Purity of PopZ variants was assessed using SDS–PAGE and mass spectrometry. First, 50–200 µl of 10-nm gold fiducials (Aurion) were spun down in a benchtop centrifuge for 20 min at 15,000 rpm. The supernatant was removed and the beads were washed twice with standard buffer (10 mM sodium phosphate pH 7.4, 150 mM NaCl and 50 mM MgCl2). They were then resuspended in an appropriate amount of buffer, resulting in a 12.5–20% higher bead concentration than the original gold fiducial concentration. Quantifoil R 2/1 copper 200-mesh grids were glow-discharged with a Pelco easiGlow (Ted Pella, 25 s of glow at 15 mA). Then, 2.3 µl of gold fiducial beads were deposited onto the grid. The beads were given 30 s to spread out. Next, 4 µl of the PopZ sample was deposited onto the grid, backblotted and plunged into a mixture of propane and ethane using either a Vitrobot Mark IV (Thermo Fisher Scientific; 2.5 s of blot time, 0 s of wait time, 0.5 s of drain time and blot force of 0) at 22 °C, 90% humidity or a homemade gravity-driven plunging device. We used 2 µM as the PopZ concentration to obtain droplets that were small enough to fit into the field of view. After freezing, the vitrified grids were clipped with autogrids (Thermo Fisher Scientific). The ΔpopZ C. crescentus strain expressing mCherry–WT PopZ through a pXMCS-2 plasmid was grown at 30 °C in PYE supplemented with kanamycin until an OD600 of 0.2– 0.5 and then diluted 100-fold and grown until an OD600 of 0.3. Cells were then pelleted and resuspended in PYE to an OD600 of 3. Next, 5 µl of the sample was deposited on glow-discharged (Pelco easiGlow; 25 s of glow at 15 mA) Quantifoil R 2/1 copper 200-mesh grids. The grids were then backblotted for 7–12 s before plunge freezing with a Vitrobot Mark IV into a mixture of 37% propane and ethane. After freezing, the vitrified grids were clipped with autogrids. C. crescentus samples were subjected to cryo-FIB milling to reduce sample thickness and increase image contrast. Cryo-FIB milling was performed using the Aquilos 2 dual-beam cryo-FIB–scanning EM instrument (Thermo Fisher Scientific). The sample was sputtered with metallic platinum for 15 s, followed by a layer of organometallic platinum for 40 s, and then sputtered again with metallic platinum for 15 s. The metallic platinum helps to reduce charging and drifting while milling and the organometallic platinum helps to protect the sample from the FIB. Targeted regions on the grid were milled to produce lamellae with a thickness of approximately 150–200 nm. Finally, the grid was sputtered with metallic platinum for 15–20 s. The final sputter coat adds bead-like fiducial inclusions that are used for tilt-series alignment. Both in vitro PopZ samples and cryo-lamellae of Caulobacter samples were imaged with a Titan Krios 300-keV microscope (Thermo Fisher Scientific) equipped with a field-emission gun, a direct detection device (Gatan K3) and an energy filter (operated with a slit width of 20 eV). The SerialEM64 package with PACEtomo65 scripts was used to collect image stacks at tilt angles ranging from +51° to −51° (3° increment) using a dose-symmetric scheme with a cumulative dose of ~106 e− per Å2. Images were taken at a magnification resulting in 1.663 Å per pixel and a nominal defocus of −6 to −5 µm. Image stacks containing ~10 frames were motion-corrected with MotionCor2 (ref. All tomograms used in the figures were denoised with CryoCARE67 followed by Isonet68. For in vitro PopZ data, 4× binned tomograms were imported into EMAN2 (ref. Straight segments of PopZ filaments were selected using a segmentation-based particle picking function. Particle positions were translated into the IMOD coordinate system to be imported into the i3 (ref. Before averaging, the contrast transfer function (CTF) of tilted images was estimated using gCTF71, CTF correction was performed using the IMOD ctfphaseflip function and CTF-corrected tomograms were created using IMOD. An initial reference was created by manually picking ~100 straight segments of filaments with predefined orientations using Tomopick72 software and averaging them using i3. Using this initial reference, the remaining particles were aligned and classified. For dense-phase characterization, denoised in vitro PopZ tomograms were segmented using the deep learning tool in Dragonfly (Comet Technologies)31. For dilute-phase characterization, undenoised tomograms were segmented. The island removal feature in Dragonfly was used to eliminate independent densities smaller than the size of a PopZ hexamer, thereby reducing noise from the segmentation results. In vitro PopZ segmentations were used in Amira (Thermo Fisher Scientific) for filament tracing. Filaments shorter than 250 Å were excluded because filaments shorter than 250 Å tend to be fragmented parts of longer filaments. Tracing results were exported into xml files containing the filament tracing metadata table. Parameters such as curved lengths and chord lengths were imported into Prism for further analysis. Statistical significance was determined using the Mann–Whitney U-test and the histogram of curved lengths was analyzed using a nonlinear regression (curve fit) approach. Samples were kept at 4 °C throughout the entire preparation process. ΔH2ΔL6-PopZ was prepared at 2 mg ml−1 in a buffer containing 10 mM sodium phosphate pH 7.4 and 150 mM NaCl. Quantifoil 300-mesh R0.6/1 UltrAuFoil holey gold films were glow-discharged under vacuum for 30 s at 15 mA in a Pelco easiGlow 91000 glow discharge cleaning system (Ted Pella). Then, 3 μl of sample was applied to the surface of the grid and blot-plunged using a manual plunger in a 4 °C cold room with ~95% humidity. Grids were manually blotted with Whatman 1 filter paper for 10 s and immediately plunged into a liquid ethane pool cooled by liquid nitrogen. Grids were then clipped into AutoGrids (Thermo Fisher Scientific) under liquid nitrogen vapor and stored in liquid nitrogen until the day of imaging. Cryo-EM data were collected on a Thermo Fisher Talos Arctica transmission EM instrument operating at 200 keV, which was aligned for parallel illumination settings with a 30-µm condenser 2 aperture. Micrographs were collected using a Falcon 4i electron detector, with an applied total electron exposure of 50 e− per Å2. The EPU data collection software was used to collect micrographs at ×190,000 nominal magnification (0.74 Å per pixel at the specimen level) with a nominal defocus range set to 0.8 to 1.4 µm under focus. A maximum image shift of 8 µm with aberration-free image shift was used to obtain a single exposure in the center of the 0.6-µm holes. A total of 8,762 micrographs were collected and processed in real-time using cryoSPARC Live (version 4.5.3)73. Patch motion correction was used to mitigate motions and radiation damage that occur during imaging. Motion-corrected images were processed collectively at the end of data collection, beginning with CTF estimation using CTFFIND4 (ref. The 7,042 micrographs with reported CTF resolutions better than 8 Å were selected for further processing and 5,899,512 particle selections were identified using a blob picker with default parameters, a minimum particle diameter of 70 Å and a maximum particle diameter of 100 Å. Particle selections that had local power between 426.0 and 967.0 were used for subsequent processing, leaving a total of 3,595,338 particles that were extracted using a box size of 194 pixels. A 2D classification of this extracted stack into 75 classes was performed using default parameters with the exception of 40 online EM iterations and a batch size of 400 per class. Epifluorescence microscopy was performed on a Nikon Ti2-E automated inverted microscope with a ×100 oil-immersion objective (CFI60 plan apochromat lens, numerical aperture: 1.45; Nikon) and a Kinetix scientific complementary metal–oxide–semiconductor camera (Teledyne Photometrics). Images were acquired with the NIS Elements software version 5.42.03 and further processed in ImageJ or MicrobeJ. Bacteria were grown in liquid medium to an OD of 0.1–0.3 and put on pads (1.2% agarose in PYE) secured between coverglass and No. 1.5 coverslips by silicone gaskets (VWR). mCherry was excited with the 594-nm epi line at 12% intensity and imaged with 100-ms exposure time using a 603–800-nm emission filter. CFP was excited with the 440-nm line at 15% intensity and imaged with 800-ms exposure time using a 464–486 nm emission filter. For FRAP experiments, the same instrument was used but images were acquired using a total internal reflection fluorescence iLAS2 module (Gataca Systems). Before the experiments, unlabeled WT PopZ and ΔL6-PopZ were mixed with either labeled ChpT or labeled PopZ and incubated for 12 h before condensation was triggered to allow the sample to equilibrate. After droplets formed, they were allowed to settle until the intensity of the whole image was stable, usually 2 h. Droplets were partially bleached in their center using a point stimulation. Data were corrected for photobleaching and time-dependent changes in intensity by including unbleached reference droplets and then normalized. The data were fitted in GraphPad Prism to a simple exponential for labeled PopZ and a two-phase model for labeled ChpT. For measurement of PopZ polar enrichment and C. crescentus cell length, MicrobeJ and imageJ were used. To obtain polar enrichment, the background was first subtracted from images using a rolling ball (radius: 50 pixels) in imageJ75. Then, a circle with a diameter of 5 pixels was used to calculate mean intensity of mCherry at whichever pole showed the highest intensity divided by the mean intensity across the cytoplasm. To obtain cell length, we used MicrobeJ76 to detect bacteria whenever possible and imageJ to manually draw cells with abnormal morphologies, which could not be detected by MicrobeJ. Polar enrichment was measured after 1 h in liquid PYE medium, as soon as the bacteria were deposited on agar pads. Length was measured after 4 h of incubation on agar pads, to give strains with a ΔpopZ phenotype time to elongate. In vitro TEV cleavage was initiated by adding 100 nM TEV protease (purified in-house) to unlabeled WT PopZ at 5 µM in 50 mM Tris-HCl pH 8.0 with 0.5 mM EDTA and 1 mM DTT. When fluorescently labeled PopZ was also present, it was added at low nanomolar concentrations. Cleavage was allowed to proceed for 10 min unless otherwise specified. For gel analysis, droplets were pelleted (2,000g for 10 min) and separated from the supernatant. Both fractions were run on a 4–20% Mini-PROTEAN TGX gradient gel (BioRad) and protein was revealed by a Coomassie blue type LC6060 SimplyBlue stain (Invitrogen). Mass photometry experiments were performed on a Refeyn TwoMP calibrated with a mix of BSA (Sigma-Aldrich) and thyroglobulin (Sigma-Aldrich). Coverslips (WillCo Wells) and gaskets (Grace Bio Labs) were prepared by washing with double-distilled water (ddH2O) followed by isopropanol and again with ddH2O, repeated three times, before drying. Next, 20 μl of buffer was added to each well to focus the instrument and then 15 μl of buffer was removed and replaced with 15 μl of sample, before mixing by pipette for 2 s before frame acquisition. Frames were acquired over 60 s using AcquireMP (version 2022 R1; Refeyn) using standard settings. Data were processed and analyzed by fitting a Gaussian distribution to the data using DiscoverMP (version 2022 R1; Refeyn). Sedimentation velocity measurements were conducted with the indicated protein concentrations (between 30 and 140 µM) in a solution of 10 mM sodium phosphate pH 7.4 + 150 mM NaCl at 25 °C on a Beckman Optima XL-I instrument equipped with a AN-60 Ti rotor and at 40,000 rpm. Samples were monitored at 280 nm for 150 scans with 8-min intervals for a total of 20 h. Scans were processed using SEDFIT software77. Fitting parameters such as the buffer density (1.00555 g ml−1), buffer viscosity (0.010016 poise) and partial specific volume (varied across PopZ constructs) were calculated by SEDNTERP78. Sedimentation velocity profiles are shown at a confidence level of 95%. To generate structural models, we ran both AlphaFold-Multimer (version 2.3.2, AlphaFold 2, from https://github.com/google-deepmind/alphafold)49 and AlphaFold 3 (from alphafoldserver.com)48 on five sequences (Supplementary Data 5). For each PopZ sequence of interest, we created an initial atomic structural model of an oligomer using AlphaFold-Multimer. This initial model served as a template for computing structure-based potentials79, which restrained the backbone and tertiary structure (default parameter values from the maximum entropy optimized force field80 were used). For trimers, tertiary interactions were considered only within each H1 helix (residues 10–24 in WT PopZ), within each H2 helix (residues 102–128 in WT PopZ) or to stabilize the trimer formed by H3 and H4 (residues 136–171 in WT PopZ). For hexamers, we similarly considered interactions within each H1 helix (residues 10–24 in WT PopZ) but the entire H2 through H4 (residues 102–171 in WT PopZ) interface was stabilized. Intermolecular interactions were scored by the hydrophobicity scale model parameterized with the Urry hydrophobicity scale (HPS-Urry)81. All simulations were conducted using Debye–Huckël electrostatics with 150 mM salt, unless otherwise noted. In the low-pH (pH 4) simulation, changes in pH were approximated by modulating amino acid residue charges. Slab simulations were performed for each PopZ variant and solution condition84. For each simulation, 100 PopZ oligomers were initially placed in a simulation box with dimensions of 100 × 100 × 100 nm3. We then performed steepest descent energy minimization, followed by a 0.1-µs constant-temperature, constant-pressure simulation at 150 K and 1 bar using a Langevin integrator with a time coupling constant of 1 ps and a Monte Carlo barostat. This simulation resulted in a single dense phase. Then, the z dimension of the simulation box was expanded by approximately 20 times the original size to 500 nm, resulting in a droplet with a dilute phase on either side. With this modified simulation box, we performed a constant-temperature, constant-volume simulation for 0.1 µs with a time coupling constant of 100 ps. The resulting equilibrated system was simulated for an additional 5 µs at a constant temperature of 300 K and constant volume. To analyze the protein density from slab simulations, we first computed a molecular contact matrix. Two PopZ oligomers were defined to be in contact if any of their α-carbons were within 1 nm. Residue-by-residue contact maps were calculated with a cutoff distance of 1 nm. To simulate coexistence of the dense and dilute phases, we combined this HPS model with slab simulations87, placing 100 PopZ oligomers in a simulation box. To increase our confidence in the simulation results, we assessed whether the simulations could reproduce key experimental findings, including the conditions that promote PopZ condensation and the effect of two mutations. The final frame of the slab simulations (left) and density profiles (right) for each of the four different conditions was consistent with our experimental observations (Extended Data Fig. WT PopZ did not form condensates at neutral pH and low salt concentrations. Lowering the pH or adding salt drove the formation of large clusters, indicative of condensation. Additionally, removing the last six residues destabilized the condensate at high salt, resulting in a larger protein concentration in the dilute phase. Meanwhile, removing H1–IDR produced a stable condensate, even at physiological salt concentrations. The input files for all simulations and code for analysis can be found on GitHub (https://github.com/alatham13/Open_ABC_PopZ). Single-cysteine or double-cysteine constructs of PopZ were labeled with Alexa Fluor 488 (AF488) and AF647 maleimide dyes (Thermo Fisher Scientific). Before labeling, IMAC-pure PopZ at 60–90 µM was buffer-exchanged into labeling buffer (50 mM HEPES pH 7.2 and 1 mM TCEP). To avoid preferential labeling by one dye over the other, substoichiometric additions of the dye mixture were made to a purified protein construct over 3 h to a final sixfold molar excess of each dye. Then the labeling was allowed to proceed overnight. To quench the labeling reaction and exchange the sample back into denaturing buffer before ion-exchange cleanup, we buffer-exchanged it into10 mM sodium phosphate pH 7.2, 10 mM Tris-HCl pH 8.0, 50 mM NaCl, 8 M urea and 2 mM DTT. It was then purified by ion exchange, flash-frozen and stored at −80 °C. To assess the purity, successful labeling and integrity of labeled PopZ, it was subjected to mass photometry and light microscopy, where it behaved like unlabeled PopZ (Extended Data Fig. For each measurement, PopZ labeled at positions C−20 and C132 with AF488 and AF647 was diluted to 300 pM and supplemented with 5 µM unlabeled PopZ. Background or scatter samples were prepared similarly but without labeled protein. For each experiment, 1–4-h datasets were recorded at 22 °C on a homebuilt multiparameter fluorescence detection microscope with pulsed interleaved excitation (MFD-PIE). Emission from a pulsed 483-nm laser diode (LDH-D-C-485, PicoQuant) was cleaned up (Semrock, FF01-482/25-25), emission from a 635-nm laser diode (LDH-D-C-640, PicoQuant) was cleaned up (Semrock, FF01-635/ 18-25) and both lasers were alternated at 30 MHz using a waveform generator (Keysight), a picosecond delayer (Micro Photon Devices) connected to the laser drivers (PDL 800-D). The red laser was delayed by ~20 ns with respect to the blue laser. Linear polarization was cleaned up (Glan-Taylor Polarizer; Thorlabs, GT10A); then, the red light and blue light were combined into a single mode optical fiber (kineFlex, Excelitas) before the light (90 μW of 483-nm light and 70 μW of 635-nm light) was reflected into the back port of the microscope (Axiovert 200, Zeiss) and to the objective (C-apochromat, ×40/1.2 W; Zeiss). Sample emission was transmitted through a polychroic mirror (Chroma, ZT488/640rpc), focused through a 75-μm pinhole and spectrally split (Semrock, FF593-Di03-25×36). The blue range was filtered (Semrock, FF03-525/50-25) and polarization was split (PBS101, Thorlabs) into parallel and perpendicular channels. The red range was also filtered (Semrock, FF01-698/70- 25), and polarization was split (PBS101, Thorlabs). Photons were detected on four avalanche photodiodes (SPCM-AQR-14, PerkinElmer, for the green parallel and perpendicular channels and for the red parallel channel and SPCM-AQRH-14, Excelitas, for the red perpendicular channel), which were connected to a time-correlated single-photon counting (TCSPC) device (Multi-Harp 150 N, PicoQuant). Signals were stored in 32-bit first-in-first-out files. Microscope alignment was verified using fluorescence correlation spectroscopy on freely diffusing AF488 and AF647 (Thermo Fisher Scientific). Instrument response functions (IRFs) were recorded one detector at a time in a solution of ATTO 488-CA or ATTO 655-CA in near-saturated centrifuged potassium iodide at a 25-kHz average count rate for a total of 25 × 106 photons. Macrotime-dependent microtime shifting was corrected for two (blue/parallel and red/perpendicular) of four avalanche photodiodes using the IRF data as input. Data were analyzed with the PAM software88 using standard procedures for MFD-PIE smFRET burst analysis89,90. Signals from each TCSPC routing channel (corresponding to the individual detectors) were divided in time gates to discern 483-nm excited FRET photons from 635-nm excited acceptor photons. A two-color MFD all-photon burst search algorithm using a 500-µs sliding time window (minimum of 100 photons per burst, minimum of 5 photons per time window) was used to identify single donor-labeled and/or acceptor-labeled molecules in the fluorescence traces. Double-labeled single molecules were selected from the raw burst data using a kernel density estimator (ALEX-2CDE < 12)91 that also excluded other artifacts. Sparse slow-diffusing aggregates were removed from the data by excluding bursts exhibiting a burst duration > 12 ms except for conditions with condensates. Data were corrected in this order to obtain the absolute stoichiometry parameter S and absolute FRET efficiency E: background subtraction, donor emission crosstalk correction, acceptor direct excitation correction and relative detection efficiency correction. By making histograms of E versus measurement time, we corroborated that the distribution of E was invariant over the duration of the measurement. Static PDA was carried out to obtain the absolute interdye distance distribution assuming two Gaussian-distributed states58. For each FRET dataset, raw bursts were rebinned in 1-ms time bins and histograms were constructed and analyzed. Only bins with at least 20 and maximally 200 photons (to reduce calculation time) were used for PDA. A two-state model for a Gaussian distance distribution was used to generate a library of simulated EFRET values, which was fitted to the experimental EFRET histogram using a reduced c2-guided simplex search algorithm to obtain the amplitude, mean distance R and width s of all Gaussian-distributed substates and the area fraction of each state A (%). A probability density function (PDF) was calculated per state using the R and σ parameters that describe the underlying Gaussian-distributed states. The summed PDF was scaled to a total area of unity, with the PDF area of each state scaled to the corresponding fraction of molecules. A Jacobian was used to estimate confidence intervals (95%) of the fit parameters. Primers were purchased from IDT for site-directed mutagenesis by Q5 polymerase. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Representative tomograms generated in this study are available from the Electron Microscopy Data Bank (EMDB) under the following accession codes: EMD-47539 for an in vitro WT PopZ condensate, EMD-47540 for an in vitro OD-PopZ condensate, EMD-47542 for an in vitro ΔL6-PopZ condensate and EMD-47557 for a PopZ condensate at the pole of a C. crescentus cell. Unprocessed tilt series are available from the EM Public Image Archive (EMPIAR) under accession codes EMPIAR-12429 for PopZ condensates in ΔpopZ C. crescentus cells expressing mCherry-tagged WT PopZ and EMPIAR-12430 for all in vitro PopZ datasets. Data and materials can be obtained from the corresponding authors upon request. Source data are provided with this paper. The software, instructions and code generated by this study used for slab simulations are available from GitHub (https://github.com/alatham13/Open_ABC_PopZ). & Rosen, M. K. Biomolecular condensates: organizers of cellular biochemistry. & Brangwynne, C. P. Liquid phase condensation in cell physiology and disease. & Rosen, M. K. A framework for understanding the functions of biomolecular condensates across scales. Choi, J. M., Holehouse, A. S. & Pappu, R. V. Physical principles underlying the complex biology of intracellular phase transitions. Pei, G., Lyons, H., Li, P. & Sabari, B. R. Transcription regulation by biomolecular condensates. Phase separation of signaling molecules promotes T cell receptor signal transduction. A. Biomolecular condensates at the nexus of cellular stress, protein aggregation disease and ageing. Visser, B. S., Lipinski, W. P. & Spruijt, E. The role of biomolecular condensates in protein aggregation. Pappu, R. V., Cohen, S. R., Dar, F., Farag, M. & Kar, M. Phase transitions of associative biomacromolecules. Mittag, T. & Pappu, R. V. A conceptual framework for understanding phase separation and addressing open questions and challenges. Gouveia, B. et al. Capillary forces generated by biomolecular condensates. Alshareedah, I. et al. Sequence-specific interactions determine viscoelasticity and ageing dynamics of protein condensates. Feric, M. et al. Coexisting liquid phases underlie nucleolar subcompartments. Posey, A. E. et al. Biomolecular condensates are characterized by interphase electric potentials. & Mittag, T. Molecular structure in biomolecular condensates. Bienz, M. Head-to-tail polymerization in the assembly of biomolecular condensates. & Mullins, R. D. Cell mechanics and the cytoskeleton. Berkamp, S., Mostafavi, S. & Sachse, C. Structure and function of p62/SQSTM1 in the emerging framework of phase separation. Bouchard, J. J. et al. Cancer mutations of the tumor suppressor SPOP disrupt the formation of active, phase-separated compartments. Liquid behavior of cross-linked actin bundles. Bowman, G. R. et al. A polymeric protein anchors the chromosomal origin/ParB complex at a bacterial cell pole. & Jacobs-Wagner, C. A self-associating protein critical for chromosome attachment, division, and polar organization in Caulobacter. Bowman, G. R. et al. Caulobacter PopZ forms a polar subdomain dictating sequential changes in pole composition and function. Tsokos, C. G. & Laub, M. T. Polarity and cell fate asymmetry in Caulobacter crescentus. Govers, S. K. & Jacobs-Wagner, C. Caulobacter crescentus: model system extraordinaire. Lasker, K., Mann, T. H. & Shapiro, L. An intracellular compass spatially coordinates cell cycle modules in Caulobacter crescentus. Selective sequestration of signalling proteins in a membraneless organelle reinforces the spatial regulation of asymmetry in Caulobacter crescentus. The material properties of a bacterial-derived biomolecular condensate tune biological function in natural and synthetic systems. Deep learning-based segmentation of cryo-electron tomograms. Toro-Nahuelpan, M., Plitzko, J. M., Schüler, D. & Pfeiffer, D. In vivo architecture of the polar organizing protein Z (PopZ) meshwork in the alphaproteobacteria Magnetospirillum gryphiswaldense and Caulobacter crescentus. Bowman, G. R. et al. Oligomerization and higher-order assembly contribute to sub-cellular localization of a bacterial scaffold. Laloux, G. & Jacobs-Wagner, C. Spatiotemporal control of PopZ localization through cell cycle-coupled multimerization. A. et al. Caulobacter PopZ forms an intrinsically disordered hub in organizing bacterial cell poles. Ptacin, J. L. et al. Bacterial scaffold directs pole-specific centromere segregation. Marzahn, M. R. et al. Higher-order oligomerization promotes localization of SPOP to liquid nuclear speckles. Soltermann, F. et al. Quantifying protein–protein interactions by molecular counting with mass photometry. Young, G. et al. Quantitative mass imaging of single biological macromolecules. Howlett, G. J., Minton, A. P. & Rivas, G. Analytical ultracentrifugation for the study of protein association and assembly. Agudo-Canalejo, J. et al. Wetting regulates autophagy of phase-separated compartments and the cytosol. Alshareedah, I., Kaur, T. & Banerjee, P. R. Methods for characterizing the material properties of biomolecular condensates. Nordyke, C. T., Ahmed, Y., Puterbaugh, R. Z., Bowman, G. R. & Varga, K. Intrinsically disordered bacterial polar organizing protein Z, PopZ, interacts with protein binding partners through an N-terminal molecular recognition feature. Biondi, E. G. et al. Regulation of the bacterial cell cycle by an integrated genetic circuit. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Highly accurate protein structure prediction with AlphaFold. A. Conformational freedom and topological confinement of proteins in biomolecular condensates. Theoretical and data-driven approaches for biomolecular condensates. Latham, A. P. & Zhang, B. Unifying coarse-grained force fields for folded and disordered proteins. Bentley, E. P., Scholl, D., Wright, P. E. & Deniz, A. A. Coupling of binding and differential subdomain folding of the intrinsically disordered transcription factor CREB. Lerner, E. et al. FRET-based dynamic structural biology: challenges, perspectives and an appeal for open-science practices. Fitzkee, N. C. & Rose, G. D. Reassessing random-coil statistics in unfolded proteins. Lotthammer, J. M., Ginell, G. M., Griffith, D., Emenecker, R. J. & Holehouse, A. S. Direct prediction of intrinsically disordered protein conformational properties from sequence. Das, R. K., Ruff, K. M. & Pappu, R. V. Relating sequence encoded information to form and function of intrinsically disordered proteins. Separating structural heterogeneities from stochastic variations in fluorescence resonance energy transfer distributions via photon distribution analysis. & Waldor, M. K. Distribution of centromere-like parS sites in bacteria: insights from comparative genomics. Ptacin, J. L. et al. A spindle-like apparatus guides bacterial chromosome segregation. Shebelut, C. W., Guberman, J. M., van Teeffelen, S., Yakhnina, A. & Gitai, Z. Caulobacter chromosome segregation is an ordered multistep process. Puentes-Rodriguez, S. G., Norcross, J. D. & Mera, P. E. To let go or not to let go: how ParA can impact the release of the chromosomal anchoring in Caulobacter crescentus. Tsai, J. W. & Alley, M. R. Proteolysis of the Caulobacter McpA chemoreceptor is cell cycle regulated by a ClpX-dependent pathway. Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. Parallel cryo electron tomography on in situ lamellae. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Buchholz, T. O. et al. Content-aware image restoration for electron microscopy. Liu, Y. T. et al. Isotropic reconstruction for electron tomography with deep learning. Chen, M. et al. A complete data processing workflow for cryo-ET and subtomogram averaging. Winkler, H. 3D reconstruction and processing of volumetric data in cryo-electron tomography. Using Tomoauto: a protocol for high-throughput automated cryo-electron tomography. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Ducret, A., Quardokus, E. M. & Brun, Y. V. MicrobeJ, a tool for high throughput bacterial cell detection and quantitative analysis. Schuck, P. Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and lamm equation modeling. Philo, J. S. SEDNTERP: a calculation and database utility to aid interpretation of analytical ultracentrifugation and light scattering data. Clementi, C., Nymeyer, H. & Onuchic, J. N. Topological and energetic factors: what determines the structural details of the transition state ensemble and ‘en-route' intermediates for protein folding? An investigation for small globular proteins. Consistent force field captures homologue-resolved HP1 phase separation. Regy, R. M., Thompson, J., Kim, Y. C. & Mittal, J. Improved coarse-grained model for studying sequence dependent phase separation of disordered proteins. Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics. Liu, S., Wang, C., Latham, A. P., Ding, X. & Zhang, B. OpenABC enables flexible, simplified, and efficient GPU accelerated simulations of biomolecular condensates. Dignon, G. L., Zheng, W., Best, R. B., Kim, Y. C. & Mittal, J. Relation between single-molecule properties and phase behavior of intrinsically disordered proteins. & Beckstein, O. MDAnalysis: a toolkit for the analysis of molecular dynamics simulations. Gowers, R. J. et al. MDAnalysis: a Python package for the rapid analysis of molecular dynamics simulations. In Proceedings of the 15th Python in Science Conference (eds Benthall, S. & Rostrup, S.) (SciPy, 2016). Dignon, G. L., Zheng, W., Kim, Y. C., Best, R. B. & Mittal, J. Sequence determinants of protein phase behavior from a coarse-grained model. & Lamb, D. C. PAM: a framework for integrated analysis of imaging, single-molecule, and ensemble fluorescence data. Precision and accuracy of single-molecule FRET measurements—a multi-laboratory benchmark study. Combining MFD and PIE for accurate single-pair Förster resonance energy transfer measurements. Tomov, T. E. et al. Disentangling subpopulations in single-molecule FRET and ALEX experiments with photon distribution analysis. We thank members of the K.L., A.A.D. and L. Racki labs for insightful discussions. We thank D. Grotjahn and her lab members for their expert advice and assistance with cryo-ET data analysis. We also thank J. Hammond, P. Ordoukhanian and the Biophysics and Biochemistry Core at Scripps Research for their support of biochemistry experiments. We gratefully acknowledge support from the National Institutes of Health (National Institute of Neurological Disorders and Stroke (NINDS) DP2 NS142714 to S.B., National Institute of General Medical Sciences (NIGMS) F32 GM150243 to A.P.L., NIGMS R01 GM083960 to A. Sali, NINDS R01 NS095892 and NIGMS RO1 GM14305 to G.C.L., NIGMS R35 GM130375 to A.A.D. We also gratefully acknowledge support from the National Science Foundation (2235200 to A. Salazar and DBI 2213983, Water and Life Interface Institute, to S.B. acknowledges the Gordon and Betty Moore Foundation for support of this work through the Moore Inventor Fellowship number 579361. is a Cancer Prevention and Research Institute of Texas (CPRIT) scholar in Cancer Research and work in his lab is supported by CPRIT (RR220094). Present address: Department of Radiology and Imaging Sciences, National Institutes of Health, Bethesda, MD, USA Department of Integrative Structural and Computational Biology, The Scripps Research Institute, La Jolla, CA, USA Daniel Scholl, Tumara Boyd, Alexandra Salazar, Asma M. A. M. Khan, Gabriel C. Lander, Donghyun Park, Ashok A. Deniz & Keren Lasker Quantitative Biosciences Institute, University of California, San Francisco, San Francisco, CA, USA Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, San Francisco, CA, USA Department of Pharmaceutical Chemistry, University of California, San Francisco, San Francisco, CA, USA Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX, USA Jan and Dan Duncan Neurological Research Institute, Texas Children's Hospital, Houston, TX, USA Therapeutic Innovation Center (THINC), Baylor College of Medicine, Houston, TX, USA Center for Alzheimer's and Neurodegenerative Diseases (CAND), Texas Children's Hospital, Houston, TX, USA Dan L Duncan Comprehensive Cancer Center (DLDCCC), Baylor College of Medicine, Houston, TX, USA Department of Biochemistry and Molecular Biophysics, Washington University School of Medicine, St. Louis, MO, USA Center for Biomolecular Condensates (CBC), Washington University in St. Louis, St. Louis, MO, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Mass photometry, smFRET and AUC, D.S. Cryo-ET data processing and analysis, T.B. Writing (review and editing), all authors. Correspondence to Donghyun Park, Ashok A. Deniz or Keren Lasker. are coinventors on a patent (US20230044825A1) covering the use of some of the protein sequences described in this work. The remaining authors declare no competing interests. Nature Structural & Molecular Biology thanks Lars Schäfer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editor: Katarzyna Ciazynska, in collaboration with the Nature Structural & Molecular Biology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) (left) Representative cryo-ET image of WT-PopZ on a z-plane where the filaments outside the condensate are visible. (right) Segmented WT-PopZ filaments are shown in yellow inside the condensate, while pink represents the filaments outside of the condensates. (b) Representative cryo-ET images of WT PopZ. Three distinct regions are shown, each represented by two z-planes: the top panels show dispersed filaments in the dilute phase, while the bottom panels show planes intersecting PopZ condensates. A two-tailed Mann-Whitney test was used to calculate statistical significance. ROUT (Q = 1%) was used to remove outliers. (a) Cryo-ET of Caulobacter cells reveals filaments within PopZ microdomains. Shown are three representative microdomains in ΔpopZ Caulobacter cells expressing mCherry-tagged WT-PopZ. The purple squares indicate zoomed-in regions. Yellow arrows highlight filamentous structures. (b) PopZ condensation requires MgCl2, high salt, or low pH. Phase contrast images of 5 µM purified PopZ in the annotated buffer conditions. (c) Fluorescence microscopy of PopZ constructs labeled with AF647 and AF488 attached either to Cys-20 (N-term) or Cys132 (OD). (upper two panels) Both uncleaved constructs readily partition into WT-PopZ droplets formed using 50 mM MgCl2, regardless of the dye positions. (d) Filament length distribution comparison between isolated OD and FL PopZ. The turquoise line indicates a fit to an isodesmic model at the mean. Dotted lines represent the median and quartiles. A two-tailed Mann-Whitney U test was used to evaluate statistical significance; exact p = 1.043 ×10−13. (a) Representative cryo-ET images at 3600X magnification of WT, OD-only, and ΔL6 PopZ condensates. The purple boxes indicate zoomed-in regions. For each variant, three tomograms from distinct grid regions were analyzed (b-d) Upper panels: Three representative cryo-ET images of WT, OD-only, and ΔL6 PopZ condensates (Scale bar: 100 nm). Lower panels, zoomed-in images of the condensates. (a) AlphaFold 3 prediction for the H3H4 trimer, colored by pLDDT. (b) AF3 prediction for the H2-loop-H3H4 trimer, colored by pLDDT (c) (left) Ion-exchange chromatogram of H1-IDR-TEV-OD-ΔL6 after TEV cleavage. (right) SDS-PAGE gel showing that OD-ΔL6 stayed in the ion-exchange flow-through, whereas H1-IDR attached to the column and was concentrated in the elution fraction. Experiments were repeated twice with similar results. (d) Mass photometry confirms that isolated OD-ΔL6 forms hexamers. (e) Deletion of H2 or its fragments traps PopZ as a hexamer. Analytical ultracentrifugation of H2 deletion variants at approximately 100 µM in 10 mM sodium phosphate pH 7.4 + 150 mM NaCl. (a) 2D cross-sections of six class averages for ΔL6-PopZ. (b) 2D cross-sections of class average structures for WT-PopZ. (c) Simulation results agree with our experimental observation and point to interaction regions between PopZ trimers. (right) Visualization of the corresponding slab density profiles using the same color code. (d) The last frames of slab simulations for the indicated conditions using hexamers. (e) Inter-trimer contact map of FL-PopZ at 600 mM NaCl with color coding indicating contact frequency. Residue-by-residue contact maps were calculated with a cutoff distance of 1 nm in MDAnalysis. (a) (top) AlphaFold 3 hexamer model using six loop-H3H4ΔL6 chains, colored by chain (left) and pLDDT score (right). The AlphaFold prediction was converted to a molecular envelope using the ChimeraX molmap command, and the resulting density was shown using volumetric rendering. (bottom) Cryo-EM 2D class averages of ΔH2ΔL6-PopZ show a similar elongated organization. (c) AF3 model of six H2-loop-H3H4ΔL6 chains, colored by chain (top) and pLDDT score (bottom). The structural model fits into the low-resolution envelope of the curved hexamer 2D class obtained via cryo-ET sub-tomogram averaging (bottom right). (a) PopZ variants tested in this study, with the number of mutated residues indicated in parentheses. Yellow and red bands represent mutations that reduce stickiness or introduce a repulsive charge, respectively. A green checkmark indicates variants that form condensates in E. coli. See Supplementary Data 5 for a complete list of mutations. (b) AF3 model of a PopZ hexamer with the four residues highlighted that are mutated in construct Loki. (c) Representative images of E. coli MG1655 cells transformed with mCherry-fusions of the PopZ variants shown in (a). A single screening experiment was performed to identify variants whose condensation is likely impaired. Scale bar: 5 µm (d) Mass photometry histograms annotated with the observed oligomeric species and the saturation concentration (csat) obtained by light microscopy. (a) Labeled PopZ behaves like WT-PopZ. (left) Mass photometry of the smFRET reporter labeled at C-20 and C132 (illustrated in Fig. 2a) shows that the labeled reporter adopts the same oligomeric species as WT-PopZ upon MgCl2 addition. (right) Fluorescence microscopy shows that the labeled FRET reporter forms spherical droplets, akin to WT-PopZ. Experiments were repeated twice with similar results. (b) AF3 model of three FL-PopZ molecules. (right) AUC shows that in the absence of MgCl2, ΔH1-PopZ forms larger species than the WT filaments. The experiment was repeated twice with similar results. (a) Representative cryo-ET images of PopZ variants Loki and Ganesha. (b) Zoomed-in cryo-ET images of Loki and Ganesha. (c) (top) Representative fluorescence and phase images of ΔpopZ C. crescentus also expressing CFP-ParB and mCherry-PopZ fusions. Images were taken after 2 hours incubation on PYE agarose pads. (bottom) Single-cell images of a cell representing the median polar enrichment of PopZ observed for each variant when the cells were first transferred to the pad. (d) Violin plots of polar enrichment (left) and cell length after 4 hours (right). Dotted lines represent the median and quartiles. (right) Median of polar enrichment and cell length plotted for each variant across the same number of cells as indicated in the left and middle panels, respectively. Horizontal and vertical error bars represent half of the interquartile range. A 3D visualization of WT PopZ condensate observed by cryo-ET. Slices from a representative tomogram of WT PopZ condensate. Segmented 3D filaments are shown in yellow. Statistical source data for Supplementary Fig. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. The filamentous ultrastructure of the PopZ condensate is required for its cellular function. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/01/260131084555.htm'>NASA's Perseverance rover completes the first AI-planned drive on Mars</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-02 09:20:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The team behind NASA's six-wheeled Mars explorer tested a vision-enabled artificial intelligence system to map a safe route across the Martian surface without relying on human route planners. NASA's Perseverance rover has now completed the first drives on another planet that were planned by artificial intelligence. During the test, generative AI was used to select waypoints for the rover, a complex planning task that is normally handled by human experts on Earth. "Autonomous technologies like this can help missions to operate more efficiently, respond to challenging terrain, and increase science return as distance from Earth grows. It's a strong example of teams applying new technology carefully and responsibly in real operations." For the demonstration, engineers used a form of generative AI known as vision-language models to examine existing data from JPL's surface mission dataset. The system analyzed the same images and information that human planners typically use, then identified waypoint locations so Perseverance could travel safely across difficult Martian terrain. The work was coordinated from JPL's Rover Operations Center (ROC) and carried out in collaboration with Anthropic, using the company's Claude AI models. That distance creates long communication delays, making real-time control of a rover impossible. For nearly three decades, rover navigation has depended on human drivers who carefully study terrain data and plan routes in advance. The system examined high-resolution orbital images captured by the HiRISE (High Resolution Imaging Science Experiment) camera aboard NASA's Mars Reconnaissance Orbiter, along with terrain slope data from digital elevation models. Using this information, the AI identified important surface features such as bedrock, outcrops, boulder fields, and sand ripples. It then produced a continuous driving path that included all necessary waypoints. This step checked more than 500,000 telemetry variables to ensure the plan would work safely with Perseverance's flight software. "We are moving towards a day where generative AI and other smart tools will help our surface rovers handle kilometer-scale drives while minimizing operator workload, and flag interesting surface features for our science team by scouring huge volumes of rover images." "Imagine intelligent systems not only on the ground at Earth, but also in edge applications in our rovers, helicopters, drones, and other surface elements trained with the collective wisdom of our NASA engineers, scientists, and astronauts," said Matt Wallace, manager of JPL's Exploration Systems Office. Managed for NASA by Caltech, JPL is home to the Rover Operations Center (ROC). New Research Reveals Humans Have a Remote Touch “Seventh Sense” Melting Antarctic Ice Did the Opposite of What Scientists Expected Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            