
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-24</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09302-6'>Structural and functional characterization of human sweet taste receptor</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 15:17:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Please note there may be errors present which affect the content, and all legal disclaimers apply. Here, we present cryo-EM structures of the full-length human sweet taste receptor in apo- and sucralose-bound states. These structures reveal a distinct asymmetric heterodimer architecture, with sucralose binding exclusively to the Venus flytrap domain of TAS1R2. Combining mutagenesis and molecular dynamics simulations, this work delineates the sweeteners recognition modes in TAS1R2. Structural comparisons further uncover the conformational changes upon ligand binding and unique activation mechanism. These findings illuminate the signal transduction mechanisms of chemosensory receptors in class C GPCRs and provide molecular basis for new-generation sweetener design. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout These authors contributed equally: Zongjun Shi, Weixiu Xu, Lijie Wu, Xiaolei Yue School of Life Science and Technology, ShanghaiTech University, Shanghai, China The conformational changes of VFTDs from apo- to sucralose-bound states in sweet taste receptor. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250624044328.htm'>Brain reboot: Gene therapy reverses Alzheimer's memory loss in mice</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 14:59:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at University of California San Diego School of Medicine have developed a gene therapy for Alzheimer's disease that could help protect the brain from damage and preserve cognitive function. While current treatments can manage symptoms of Alzheimer's, the new gene therapy aims to halt or even reverse disease progression. While further studies will be required to translate these findings into human clinical trials, the gene therapy offers a unique and promising approach to mitigating cognitive decline and promoting brain health. The study, published in Signal Transduction and Targeted Therapy, was led by senior author Brian Head, Ph.D., professor of anesthesiology at UC San Diego School of Medicine and Veterans Affairs research career scientist, and co-senior author Shanshan Wang, M.D. The gene therapy technology was licensed by UC San Diego to Eikonoklastes Therapeutics in 2021. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250624044326.htm'>Ancient carbon ‘burps' caused ocean oxygen crashes — and we're repeating the mistake</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 14:40:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research from the University of California, Davis, the Chinese Academy of Sciences and Texas A&M University reveals that massive emissions, or burps, of carbon dioxide from natural earth systems led to significant decreases in ocean oxygen concentrations some 300 million years ago. Combining geochemical analyses of sediment cores and advanced climate modeling, the study, published June 23 in Proceedings of the National Academy of Sciences, highlights five periods when significant decreases in ocean oxygen levels (by 4% to 12%) coincided with significant increases in levels of carbon dioxide in the atmosphere. Such oxygen-less, or anoxic, events are known for their detrimental effects on marine life and biodiversity. If events of a similar scale were to happen today, they would likely affect coastal areas that are important for fisheries and marine biodiversity. "This is our only analog for big changes in carbon dioxide at levels comparable to what we're living in today, where we see doublings and triplings of the levels," said senior author Isabel P. Montañez, a Distinguished Professor in the Department of Earth and Planetary Sciences at UC Davis. While carbon dioxide levels of long-past climates were influenced by natural systems like volcanic eruptions, human-produced and human-related carbon dioxide emissions strongly influence today's levels. "We're creating a burp now and at a rate two, maybe three, orders of magnitude faster than in the past," Montañez said. By analyzing the geochemical makeup of these deep-water cores, specifically carbonate uranium isotopes, the team chronicled Earth's environmental conditions from 310 to 290 million years ago. "Through that analysis, we see these 'burps' not just in carbon dioxide but in the ocean's uranium isotope signature too," Montañez said. The team then used that information to inform leading-edge climate models, developed by the authors of this study, that are used to better understand ancient climates. "It's a mathematical framework in which we put in all our proxy information and we run it hundreds of thousands of times on a supercomputer," Montañez said. While the decrease in ocean oxygen doesn't appear to correlate to any known mass extinctions, it does align with pauses in biodiversity that can be seen in the geological record. For one, oxygen in the atmosphere was 40% to 50% higher than it is today. That could be interpreted as a warning, according to Montañez. "This is a huge discovery because how do you take an ocean sitting under an atmosphere with much more oxygen than today and permit this?" "The message for us is, 'Don't be so sure that we can't do this again with our current human-driven release of carbon dioxide.'" Materials provided by University of California - Davis. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250624044322.htm'>Killer whales use seaweed tools in never-before-seen grooming behavior</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 14:03:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Reporting in the Cell Press journal Current Biology on June 23, a team of whale experts has found widespread tool manufacture and usage in an endangered population of resident killer whales living in the Salish Sea -- part of the Pacific Ocean between British Columbia and Washington. The whales fashion tools from kelp and use them for grooming purposes. "We found that southern resident killer whales regularly use lengths of bull kelp during social interactions, apparently as a tool to groom one another," says Michael Weiss (@CetaceanMike) of the Center for Whale Research in Friday Harbor, WA. "To find that the whales were not just using but also manufacturing tools, and that these objects were being used in a way never before reported in marine mammals, was incredibly exciting." Weiss and his team discovered this unexpected whale activity while conducting aerial observations of southern resident killer whales, which are a critically endangered whale population with fewer than 80 individuals left found in the Pacific Northwest. They also saw some evidence that whales with more molting or dead skin were more likely to engage in grooming, suggesting it may have a hygienic function. "What I find most remarkable is that despite this apparently being a common behavior -- we see it most days we fly our drone over these whales -- it hadn't yet been discovered in this population despite nearly 50 years of dedicated observation," says Weiss. "To me, this demonstrates not just the power of new observation methods but also how much we still have to learn about these animals." Regardless, the discovery opens new avenues for understanding tool use in marine mammals and demonstrates that tools can be used in a wide array of contexts, say the researchers. "This finding highlights yet another way these whales' society and culture is unique and the importance of recovering the southern resident killer whale population," says Weiss. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/to-make-better-choices-understand-how-your-brain-processes-values/'>To Make Better Choices, Understand How Your Brain Processes Values</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>To Make Better Choices, Understand How Your Brain Processes Values The brain weighs factors based on their importance to oneself and one's social world as part of a complex calculation that shapes behavior But what about our more intentional, conscious day-to-day choices? As the book explores, many people feel frustrated when they find themselves making choices that run counter to their big-picture goals—things like exercising more, visiting older relatives or cutting back screen time. In the process, she offers a compassionate framework for recognizing how we make choices and why change can be so challenging. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Scientific American spoke with Falk about how the brain sifts through options and how reflecting on our priorities can help align our behaviors with long-term aims. When we're making a decision, the brain first identifies the different things that we might be choosing between.Then the brain's valuation system takes input from many other systems, including the ones that help us process emotions, think about ourselves, and help us understand other people's thoughts and feelings. It integrates that information and assigns a subjective value to each option. Ultimately, that valuation system provides a final common pathway in arriving at our subjective evaluations across very different things—whether relationship partners, food choices or financial decisions. As just one example of factors that influence these value calculations, in January my colleagues and I published findings that showed people are more likely to share news stories with others when they reflect on why that article is relevant to themselves and their social worlds. How might understanding these systems change the way people think about their choices or behavior? We know that one important input for the value system is self-relevance, or how much something feels psychologically meaningful or close to oneself. Neuroscientific research shows a deep intertwining of these self and values systems. As another example, people prefer rewards that are close in time or instantly gratifying. Scientists have studied how the medial prefrontal cortex, which is important to both the value and self-relevance systems, handles multiple kinds of psychological distance in a similar way. For instance, people are more motivated to choose something when you expect the rewards will come sooner or involve something geographically close or people who feel socially close. Scientists have done that by asking people to imagine otherwise distant scenarios in concrete ways. You can vividly imagine how your future self will spend money in retirement as an incentive to save now or envision the transformative consequences for a specific child of donating to UNICEF to increase your giving. In your research, you've looked at another strategy, what psychologists call values affirmation, and found it can help people make better decisions for themselves in the long term. For example, in studies that our team has run, sedentary people are more likely to become physically active after being asked to affirm their values by identifying the most important ones on a list and then envisioning themselves in various scenarios that tap into those values. When we look at brain activity, we see that when thinking about what matters most, core parts of the value and self-relevance systems become active. We also find people are less defensive and more open to change after reflecting on their most important values. So if we have people do this type of values affirmation before we give them messages intended to coach them toward healthier choices—such as reminders to exercise—we see that the people who affirmed their values show more activity in the value and self-relevance systems in the brain than others who did not do that affirmation. And then when we follow our participants over the next month—sending out more value-reflection prompts and reminders to be active—the ones who showed that greater brain activation were ultimately less sedentary. Why would thinking about family and friends change someone's inclination to exercise? Instead the idea is that thinking about what really matters to you helps you zoom out and consider the big picture. Affirming your values—by writing about what you care about most, choosing values from a list or imagining situations that involve those values—can ground you. In addition, self-transcendent values, which connect you to something outside of yourself, such as friends and family or spirituality, are more effective than purely self-focused values at helping people make changes. People who connect with those self-transcendent values on a regular basis, and people who have a stronger sense of purpose, get health and well-being benefits that don't accrue when we focus on more self-focused values such as personal money, fame and power. Feeling purposeful means having a clear picture of what you're trying to do and why. It can be like a North Star that helps us think about the path from where we are now to where we want to be. People's sense of purpose fluctuates, on a day-to-day basis, around whatever their baseline is. This is helpful to keep in mind because it highlights that we can do things that might change how purposeful we feel on a given day and what benefits we get as a result. For example, day-to-day choices and physical health relate to purpose. Are you a scientist who specializes in neuroscience, cognitive science or psychology? Please send suggestions to Scientific American's Mind Matters editor Daisy Yuhas dyuhas@sciam.com. Daisy Yuhas edits the Scientific American column Mind Matters. She is a freelance science journalist and editor based in Austin, Tex. She is author of the Kids Field Guide to Birds.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65141200/seismic-waves-core-mantle-boundary/'>Something Strange Is Happening 1,700 Miles Beneath Your Feet. Now We Know Why.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Deep beneath Earth's surface are layers of soil, rock strata often embedded with fossils, gurgling magma, and—back up. This is in part because if you were to plunge down 2,700 kilometers (1,700 miles), you would be jump-scared by seismic waves that accelerate when they hit the threshold of the D” layer. But that still wasn't enough to explain the phenomenon. Geoscientist Motohiko Murakami wanted to investigate what could possibly be going on to cause the strange seismic wave acceleration known as the D” discontinuity. Because trekking to the core-mantle boundary (CMB) where the D” layer lies is obviously not an option, he led a team of researchers from Switzerland and Japan in running lab tests and computer simulations to find out what post-perovskite had to do with he unusual increase in seismic waves. “The deformation-induced texture forms when crystals undergo plastic deformation, causing their orientations to align in specific directions. It is mainly produced by dislocation slip or creep,” Murakami said in a study recently published in the journal Communications Earth & Environment. How post-perovskite crystals are aligned determines their hardness, and the speed at which seismic waves move through them depends on how hard they are. Materials called perovskites can be created from any substances capable of being arranged into the same cubic crystal structure. Perovskite is a calcium titanium oxide mineral (CaTiO3), while post-perovskite is a form of magnesium silicate (MgSiO3) achieved at extremely high pressures. Like perovskite, MgGeO3 crystals deform easily when pressure is applied, so how they behaved would reflect was is going on over a thousand miles underground. They were then exposed to high-pressure sound waves, and the wave velocity was measured once those waves passed through the crystals. It turned out that sound waves can experience a substantial increase in velocity when passing through aligned post-perovskite crystals. This is the first time any evidence—even lab-based evidence—has been found for Earth's innards moving. “While previous theoretical work has suggested that anisotropy could explain the observed seismic discontinuities,” he said. “Our results, obtained through in situ measurements of post-perovskite velocities under high pressure, represent the experimental verification of this hypothesis, bridging the gap between theory and observation.” Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. Tons of Coal Ash Holds a Vast Treasure in the U.S. Past CO2 Rise Can't Even Compare to Climate Change Are There More Humans on Earth Than We Thought? One Single Ancestor Gave Rise to All Life on Earth</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02710-z'>Prime editor-based high-throughput screening reveals functional synonymous mutations in human cells</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 10:57:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Synonymous mutations are generally considered neutral, while their roles in the human genome remain largely unexplored. Here we use the PEmax system to create a library of 297,900 engineered prime-editing guide RNAs and perform extensive screening to identify synonymous mutations affecting cell fitness. Unlike recent findings in yeast, group-level analyses show that synonymous mutations diverge from nonsynonymous mutations in fitness effects yet exhibit similar phenotypic distributions relative to negative controls. Following rigorous quality control, only a small subset demonstrated measurable effects. For these functional mutations, we develop a specialized machine learning tool and uncover their impact on various biological processes such as messenger RNA splicing and transcription, supported by multifaceted experimental evidence. We find that synonymous mutations can alter RNA folding and affect translation, as demonstrated by PLK1_S2. By integrating screening data with our model, we predict clinically deleterious synonymous mutations. This research deepens our understanding of synonymous mutations, providing insights for clinical disease studies. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout The plasmids used in this study were deposited to Addgene or are available upon request. All raw sequencing data were deposited to the Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation and Beijing Institute of Genomics, Chinese Academy of Sciences under accession number HRA007615. The human reference genome used in this study is GRCh38.p14 from the NCBI (GCF_000001405.40). Databases involved in this study included ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), SynMICdb (https://synmicdb.dkfz.de/rsynmicdb/), DepMap (https://depmap.org/portal/), GWAS Catalog (https://www.ebi.ac.uk/gwas/) and the Medical Research Council Integrative Epidemiology Unit OpenGWAS (https://gwas.mrcieu.ac.uk/). Source data are provided with this paper. The ZFC-eBAR (version 0.2.0) algorithm and DS Finder (version 0.1.0), implemented in Python 3, can be downloaded from GitHub (https://github.com/UronicAcid/ZFC-eBAR and https://github.com/UronicAcid/DS-Finder). Other processed data and code can be found on Zenodo (https://doi.org/10.5281/zenodo.14639522)69. Kimura, M. Preponderance of synonymous changes as evidence for the neutral theory of molecular evolution. Shen, X., Song, S., Li, C. & Zhang, J. Synonymous mutations in representative yeast genes are mostly strongly non-neutral. Insufficient evidence for non-neutrality of synonymous mutations. Cuevas, J. M., Domingo-Calap, P. & Sanjuan, R. The fitness effects of synonymous mutations in DNA and RNA viruses. Kristofich, J. et al. Synonymous mutations make dramatic contributions to fitness when growth is limited by a weak-link enzyme. Walsh, I. M., Bowman, M. A., Soto Santarriaga, I. F., Rodriguez, A. & Clark, P. L. Synonymous codon substitutions perturb cotranslational protein folding in vivo and impair cell fitness. Lebeuf-Taylor, E., McCloskey, N., Bailey, S. F., Hinz, A. & Kassen, R. The distribution of fitness effects among synonymous mutations in a gene under directional selection. Sauna, Z. E. & Kimchi-Sarfaty, C. Understanding the contribution of synonymous mutations to human disease. Supek, F., Minana, B., Valcarcel, J., Gabaldon, T. & Lehner, B. Synonymous mutations frequently act as driver mutations in human cancers. Buske, O. J., Manickaraj, A., Mital, S., Ray, P. N. & Brudno, M. Identification of deleterious synonymous variants in human genomes. Rentzsch, P., Witten, D., Cooper, G. M., Shendure, J. & Kircher, M. CADD: predicting the deleteriousness of variants throughout the human genome. Jinek, M. et al. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Cong, L. et al. Multiplex genome engineering using CRISPR/Cas systems. Anzalone, A. V. et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Engineered pegRNAs improve prime editing efficiency. Enhanced prime editing systems by manipulating cellular determinants of editing outcomes. Zhu, S. et al. Guide RNAs with embedded barcodes boost CRISPR-pooled screens. Zhu, S. et al. Genome-wide CRISPR activation screen identifies candidate receptors for SARS-CoV-2 entry. Sharma, Y. et al. A pan-cancer analysis of synonymous mutations. Kolde, R., Laur, S., Adler, P. & Vilo, J. Robust rank aggregation for gene list integration and meta-analysis. Accurate proteome-wide missense variant effect prediction with AlphaMissense. Mathy, C. J. P. et al. A complete allosteric map of a GTPase switch in its native cellular network. He, Y. et al. Near-atomic resolution visualization of human transcription promoter opening. Predicting the efficiency of prime editing guide RNAs in human cells. Predicting prime editing efficiency and product purity by deep learning. Prokhorenkova, L., Gusev, G., Vorobev, A., Dorogush, A. V. & Gulin, A. CatBoost: unbiased boosting with categorical features. In Proc 32nd International Conference on Neural Information Processing Systems (eds Bengio, S. et al.) 6639–6649 (NIPS, 2018). Predicting splicing from primary sequence with deep learning. Wagner, N. et al. Aberrant splicing prediction across human tissues. Barry, T., Wang, X., Morris, J. A., Roeder, K. & Katsevich, E. SCEPTRE improves calibration and sensitivity in single-cell CRISPR screen analysis. Nackley, A. G. et al. Human catechol-O-methyltransferase haplotypes modulate protein expression by altering mRNA secondary structure. Tuller, T. et al. An evolutionarily conserved mechanism for controlling the efficiency of protein translation. Bae, H. & Coller, J. Codon optimality-mediated mRNA degradation: linking translational elongation to mRNA stability. Dhindsa, R. S. et al. A minimal role for synonymous variation in human disease. Cuella-Martin, R. et al. Functional interrogation of DNA damage response variants with base editing screens. Hanna, R. E. et al. Massively parallel assessment of human variants with base editor screens. Basic principles of yeast genomics, a personal recollection. Chamary, J. V., Parmley, J. L. & Hurst, L. D. Hearing silence: non-neutral evolution at synonymous sites in mammals. Liu, Y. et al. Genome-wide screening for functional long noncoding RNAs in human cells by Cas9 targeting of splice sites. Radrizzani, S., Kudla, G., Izsvak, Z. & Hurst, L. D. Selection on synonymous sites: the unwanted transcript hypothesis. Kudla, G., Murray, A. W., Tollervey, D. & Plotkin, J. B. Coding-sequence determinants of gene expression in Escherichia coli. Gu, W., Zhou, T. & Wilke, C. O. A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes. Improving prime editing with an endogenous small RNA-binding protein. Ren, X. et al. High-throughput PRIME-editing screens identify functional DNA variants in the human genome. Gould, S. I. et al. High-throughput evaluation of genetic variants with prime editing sensor libraries. Cirincione, A. et al. A benchmarked, high-efficiency prime editing platform for multiplexed dropout screening. Belli, O., Karava, K., Farouni, R. & Platt, R. J. Multimodal scanning of genetic variants with base and prime editing. Kim, Y., Oh, H. C., Lee, S. & Kim, H. H. Saturation profiling of drug-resistant genetic variants using prime editing. Herger, M. et al. High-throughput screening of human genetic variants by pooled prime editing. Zuker, M. Mfold web server for nucleic acid folding and hybridization prediction. Morales, J. et al. A joint NCBI and EMBL-EBI transcript set for clinical genomics and research. Langmead, B., Trapnell, C., Pop, M. & Salzberg, S. L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Masella, A. P., Bartram, A. K., Truszkowski, J. M., Brown, D. G. & Neufeld, J. D. PANDAseq: paired-end assembler for Illumina sequences. Camacho, C. et al. BLAST+: architecture and applications. Xu, P. et al. Genome-wide interrogation of gene functions through base editor screens empowered by barcoded sgRNAs. Cleveland, W. S. Robust locally weighted regression and smoothing scatterplots. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Crooks, G. E., Hon, G., Chandonia, J. M. & Brenner, S. E. WebLogo: a sequence logo generator. The UCSC Genome Browser Database: update 2006. Ding, B. et al. Noncoding loci without epigenomic signals can be essential for maintaining global chromatin organization and cell viability. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. & Dewey, C. N. RSEM: accurate transcript quantification from RNA-seq data with or without a reference genome. The Sequence Alignment/Map format and SAMtools. Replogle, J. M. et al. Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing. Integrated analysis of multimodal single-cell data. Anders, S., Pyl, P. T. & Huber, W. HTSeq—a Python framework to work with high-throughput sequencing data. & Smyth, G. K.edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Nakamura, Y., Gojobori, T. & Ikemura, T. Codon usage tabulated from international DNA sequence databases: status for the year 2000. Gruber, A. R., Lorenz, R., Bernhart, S. H., Neubock, R. & Hofacker, I. L. The Vienna RNA websuite. Tang, W. et al. Data and code related to ‘Prime editor-based high-throughput screening reveals functional synonymous mutations in human cells'. We thank the National Center for Protein Sciences at Peking University for their assistance with fluorescence-activated cell sorting and analysis, especially H. Lv and H. Yang for their technical support. We thank the High-Performance Computing Platform of Peking University for enabling our data analysis. We thank the National Center for Protein Sciences PKU-EdiGene High-Throughput Screening Core at Peking University. We thank Tsingke Biotech for providing primer synthesis and sequencing services. We thank Cloud-Seq Biotech for providing the Ribo-seq service and Emei Tongde Technology Development for their technical support in single-cell RNA library preparation. This research was funded by the National Natural Science Foundation of China (31930016, to W.W.), the Peking-Tsinghua Center for Life Sciences (to W.W.), Changping Laboratory (to W.W.) and the Taishan Scholarship (tsqn202312362, to Yongshuo Liu). These authors contributed equally: Xuran Niu, Wei Tang, Yongshuo Liu. Biomedical Pioneering Innovation Center, Beijing Advanced Innovation Center for Genomics, Peking-Tsinghua Center for Life Sciences, Peking University Genome Editing Research Center, State Key Laboratory of Gene Function and Modulation Research, School of Life Sciences, Peking University, Beijing, China Xuran Niu, Wei Tang, Yongshuo Liu, Binrui Mo, Ying Yu, Ying Liu & Wensheng Wei Academy for Advanced Interdisciplinary Studies, Peking University, Beijing, China Department of Clinical Laboratory, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Sciences, Jinan, China Changping Laboratory, Beijing, China Ying Liu & Wensheng Wei Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar and Ying Liu conceptualized the project, with W.W. supervising it. W.W., X.N., W.T., Ying Liu and Yongshuo Liu designed the experiments. and Yongshuo Liu conducted the library screens. performed the single-cell screens, subsequent validations and experimental data analysis with assistance from B.M. handled all bioinformatics analysis. constructed the NGS library. developed the machine learning model and designed the web page. and Ying Liu wrote the paper, which W.W. revised. Correspondence to Ying Liu or Wensheng Wei. W.W. is a scientific advisor and founder of EdiGene and Therorna. The other authors declare no competing interests. Nature Biotechnology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Diagram of the epegRNAeBAR library structure. RTT: reverse transcription template, PBS: primer binding site. b, Principles and workflow for designing epegRNAs within the library. Searches for NGG PAMs are conducted on the representative transcript of each gene to determine the spacer sequence, PBS sequence, and RTT, followed by a first filtering step. Comprehensive searches along the coding region are then performed to achieve saturation mutagenesis of synonymous mutations, with linker sequences subsequently designed and a secondary filter applied, resulting in the final epegRNA sequences. c, Distribution of expression levels and essentiality of genes within the library in HCT116 cells. Red dots represent the 11 genes targeted for complete saturation mutagenesis, blue dots represent the 56 genes targeted for synonymous saturation mutagenesis, and gray dots represent all other genes. d, Histogram of the number of epegRNA designed for mutations in the library. e, Statistics of the types of amino acid substitutions for the 11 genes targeted for saturation mutagenesis. The y-axis represents the original amino acid, the x-axis represents the amino acid post-PE editing, and the values represent the number of epegRNAs designed for each substitution. The histogram on the right illustrates the coverage of the corresponding amino acids, indicating the proportion of amino acids that can be targeted among all amino acids in the 11 genes. f, Clustered heatmap displaying Pearson correlation coefficients of the whole transcriptome between HCT116-PEmax cells and wild-type (WT) HCT116 cells, as well as after the addition of nontargeting (NT) epegRNA and AAVS1-targeting epegRNA. g, Graph showing the changes in editing efficiency over a 28-day period of continuous culture in HCT116-PEmax cells with the addition of two test epegRNAs targeting different sites. a, Diagram illustrating the genomic PCR process performed on the cell library post-screening. b, Algorithm description of ZFC-eBAR (see Methods for details). c, The x-axis represents the number of genes corresponding to the positive controls (epegRNAs that introduces nonsense and frameshift mutations on essential genes), while the y-axis represents the AUC calculated using these positive controls as well as negative controls (AAVS1-targeting and nontargeting epegRNA). The score used to calculate AUC is the screen score. d, Similar to c, but nonsense/frameshift mutations at the same site are regarded as the same mutation. The score used to calculate AUC is the Top score. e, ROC analysis of epegRNAs in highly essential genes (Chronos score < −2.5) based on different scores. The red point represents the threshold we selected. The red line represents the ROC curve for the top1 score of epegRNA, the dark blue line represents the ROC curve for the top2 mean score of epegRNA, and the green line represents the ROC curve for the all mean score of epegRNA. f, Spearman correlation of zLFC for the top 2 ranked epegRNAs in highly essential genes (Chronos score < −2.5). a, Correlation of zLFC values between two eBARs replicates for epegRNA with the third eBAR's |LFC| ≥ 0. b, Correlation of zLFC values between two eBARs for epegRNAs with |AGCA LFC| ≥ 1 and |CACT LFC| ≥ 1. c, Pearson correlation analysis of LFC values between two eBARs across varying thresholds of absolute LFC values from the third eBAR. The x-axis indicates the absolute LFC threshold applied to each eBAR, while the y-axis represents the Pearson correlation coefficient between the remaining two eBARs. d, Correlation of LFC values between two eBARs for epegRNAs with the third eBAR's |LFC| ≥ 1. e, Correlation of LFC values between two eBARs for epegRNAs with the third eBAR's |LFC| ≥ 0. For panels a, b, d, and e, Pearson correlation coefficient (r) is labeled. N indicates the number of epegRNAs shown in these figures (for panels a and e, excluding those with zero sequencing read counts and black lines indicate density-based contours). f–g, Correlation analysis of enriched mutations in the HCT116 screen across three eBARs, based on LFC values (f) or zLFC values (g). Diagonal panels show histograms of value distributions, lower left panels show scatter plots of pairwise correlations, and upper right panels present Pearson correlation coefficient (r). The dataset includes 1,717 epegRNAs with nonsynonymous mutations and 417 epegRNAs with synonymous mutations. h, Boxplot of the relationship between the degree of reduction in counts of simulated deleterious variants in the experimental group (x-axis) and the calculated |screen score| (y-axis). n = 492 per group (number of epegRNAs). The gray dashed lines correspond to thresholds of 3 and 2.2. Boxplots are depicted as follows: the center line represents the median, the box limits denote the upper and lower quartiles, and whiskers extend to 1.5 times the interquartile range. i, Corresponding to h, the x-axis represents the average counts of these simulated deleterious variants in day 0, and the y-axis indicates the percentage reduction in day 35. Green labels highlight points where the |screen score| after reduction exceeds the selection threshold (3). a, Comparison of fitness score distributions for synonymous mutations across three groups: 67 saturation-mutated genes (synonymous mutations only), 11 saturation-mutated genes (including all mutation types), and human homologous genes of the yeast (excluding RPL39 due to limited data). P values were calculated using a two-sided Wilcoxon test. b, Distribution of essentiality for human homologous genes of yeast in HCT116 cells, data of RPL29 are missing in DepMap. The corresponding positions are labeled in purple. c, Shown in order as in b, distribution of fitness scores for synonymous, nonsense and frameshift mutations for each gene. From left to right, the mutation counts for each group are 46/5 (RPS7), 269/6 (TSR2), 459/6 (BUD23), 167/6 (ATP6V1F), 791/6 (PAF1), 233/6 (GET1), 131/6 (VMA21), 162/6 (LSM1), 193/6 (UBE2B), 323/6 (INO80C), 487/6 (PRPS2) and 80/6 (RPL29), with n = 494 for AAVS1 and n = 981 for Nontargeting. Boxplots are depicted as follows: the center line represents the median, the box limits denote the upper and lower quartiles, and whiskers extend to 1.5 times the interquartile range. a, Correlation of BLOSUM62 scores with the screening results for 11 genes. Mean scores were compared to nontargeting controls, with mutations showing adjusted P < 0.05 (two-tailed Student's t test with Benjamini–Hochberg correction) marked in black. Only mutations with counts ≥10 were included. b, Upper panel displays a heatmap of screen scores for all mutations in RAN. Light gray indicates amino acids not included in the library. Synonymous mutations are marked with dots, and dashed lines denote domain boundaries. The lower panel illustrates domain annotations for RAN. c, Top scores for missense and synonymous mutations are compared across all regions of RAN, including G3 Switch II and Q69, n indicates the number of mutations. d, Comparison of top screen scores from our study with scores from ref. Spearman correlation coefficient = 0.3076786. e, Detailed Mutation Analysis in POLR2L. Middle panel: Heatmap of screening scores for all mutations in POLR2L. Light gray indicates amino acids not designed in the screening library. Dots indicate synonymous mutations. Dashed lines represent four Zn2+ binding sites. Lower panel: Average top scores across each position, with three designed Zn2+ binding sites highlighted in red. Left panel: Average top scores by amino acid, with the two most depleted ones shown in orange. f, Cell fitness effects of different amino acid (aa) substitutions (statistical methods as a). g, The distributions of fitness scores for mutation types with a significant impact on cell fitness. From left to right, the mutation counts are 1,475, 182, 258, 113, 233, 230, and 850 respectively. h–i, Distribution of enriched mutations for various amino acid substitutions (h) and among different types of base pair substitutions (i) in both saturation synonymous mutations and clinical synonymous mutations. Top score represents the maximum |screen score| value for different epegRNAs corresponding to each mutation in the screen. For c and g, boxplots are depicted as follows: the center line represents the median, the box limits denote the upper and lower quartiles, and whiskers extend to 1.5 times the interquartile range. a, Distribution of SpliceAI scores for synonymous mutations. b, Distribution of RNA folding energy changes for synonymous mutations. c, Distribution of codon usage frequency changes for synonymous mutations. d, Distribution of conservation scores for synonymous mutations. e, Distribution of gene essentiality (Chronos score) for synonymous mutations in HCT116 cells. f, Distribution of expression levels for synonymous mutations in HCT116 cells. a, Schematic depiction of the splicing alterations caused by the RPL11_V132 (GTG > GTC) mutation. The transcript sequence information was obtained by sequencing the cDNA from the experimental and control groups. b, Validation of the effect of the RPL11_V132 (GTG > GTC) mutation on cell proliferation in HCT116 cells. c, Relative mRNA expression levels of RPL11 in the experimental and control groups. The mRNA level of each sample was quantified by real-time qPCR and normalized by GAPDH, and the indicated relative mRNA level was normalized to that of AAVS1-targeting control cells. d, Analysis of editing outcomes for epegRNA targeting RPL11_V132 and controls via genome sequence amplification and NGS. e, Schematic depiction of the splicing alterations caused by the KRAS_G75 (GGG > GGT) mutation. The transcript sequence information was obtained by sequencing the cDNA from the experimental and control groups. f, Validation of the effect of the KRAS_G75 (GGG > GGT) mutation on cell proliferation in HCT116 cells. g, Relative mRNA expression levels of KRAS in the experimental and control groups. The mRNA level of each sample was quantified by real-time qPCR and normalized by GAPDH, and the indicated relative mRNA level was normalized to that of AAVS1-targeting control cells. h, Analysis of editing outcomes for epegRNA targeting KRAS_G75 and controls via genome sequence amplification and NGS. All data are presented as mean ± s.d. (n = 3 biological replicates for cell proliferation assay, n = 3 technical replicates for real-time qPCR). P values were calculated using two-tailed Student's t test, *P < 0.05, ****P < 0.0001; n.s., not significant. a, Analysis of editing outcomes for epegRNA targeting PLK1_S2 and controls via genome sequence amplification and NGS. b, Relative mRNA expression levels of PLK1 in the experimental and control groups. The mRNA level of each sample was quantified by real-time qPCR and normalized by GAPDH, and the indicated relative mRNA level was normalized to that of AAVS1-targeting control cells. Data are presented as mean ± s.d. P values were calculated using two-tailed Student's t test, n.s., not significant. a, Overview of the process for capturing epegRNA and single-cell transcriptomes. The cDNA of epegRNA was obtained by reverse transcription using primers targeting the constant secondary structure of evopreQ1, and the cDNA of the entire transcriptome within the cells was obtained by reverse transcription using oligo-dT primers. The figure was adapted from ref. 63. b, Sequencing depth impact epegRNA detection probability and observed gene expression levels. The blue dashed line represents the linear regression line correlating total epegRNAs per cell (y) with total UMIs per cell (x). c–d, Quantile–quantile plots comparing 15 nontargeting epegRNAs (negative control) with 395 epegRNAs across 213 genes (discovery). Genes for the nontargeting tests were randomly selected from the entire gene set. The left-hand panel shows the QQ plot of raw P values, and the right-hand panel shows the QQ plot of –log10 transformed P values. In both panels, the black solid line denotes the theoretical y = x reference under the null hypothesis and the surrounding gray band represents the pointwise 95% confidence interval of that regression. a–b, ROC analysis of DS Finder, SilVA, and CADD on 45 confirmed pathogenic mutations (positive controls) and 1,439 synonymous mutations without phenotypes from our screening (negative controls). The x-axis represents the false positive rate, and the y-axis represents the true positive rate. The red, dark blue, and green curves represent DS Finder, SilVA, and CADD, respectively. The test set in a contains 20 positive controls, excluding the mutations used in the SilVA training set and the test set in b consists of the 25 mutations used in the SilVA training set. c–f, Screening results analysis of three cell lines. Volcano plot illustrating the results of screening for functional synonymous and nonsynonymous mutations affecting cell fitness in A549 (c) and KYSE-30 (e). Blue and red dots denote depleted and enriched epegRNAs, respectively. Analyzing the correlation between three eBARs using zLFC in A549 (d) and KYSE-30 (f), following the same method as in Fig. The data were filtered according to the log2 fold change of a specific eBAR and the Pearson correlation of the zLFC of another pair of eBARs was investigated. Red, green, and blue denote three individual eBARs: AGCA, CACT, and GCAG, respectively. g, Performance of DS Finder in three cell lines compared with CADD and SilVA. Each point on the graph represents a different dataset, totaling 10. 1–5 and captions for Tables 1–10. The genes involved in the epegRNA library used for this screening. This table provides detailed information on these 3,644 genes and our selection criteria. Information about the epegRNA library. Sheet 1 provides an overall summary of the library. Sheet 2 contains detailed information, including the target gene, mutation details and sequence information for each epegRNA. Screening results in HCT116 cells. Single-cell screening results in HCT116 cells. Features of synonymous mutations. This table details 75 features used in the synonymous mutation analysis, with comprehensive presentation of the HCT116 screening results. Clinical prediction results of the DS Finder model and mutation information used for model analysis. Sheet 1 contains prediction scores of colonic disease-associated synonymous mutations by DS Finder. Sheet 2 includes 45 known clinically pathogenic synonymous mutations used for evaluating model prediction performance. Sheet 3 provides same-site controls for these 45 mutations. Sheet 4 contains negative control mutations obtained from screening. Screening results in A549 cells. Screening results in KYSE-30 cells. Sequence information of all epegRNAs used in validation experiments. All primers used in this study. Each sheet sequentially includes library PCR primers, genomic PCR primers for each validation site, cDNA PCR primers and reverse transcription–qPCR primers. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Niu, X., Tang, W., Liu, Y. et al. Prime editor-based high-throughput screening reveals functional synonymous mutations in human cells. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02746-z'>Threat reduction must be coupled with targeted recovery programmes to conserve global bird diversity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 10:20:06
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Ambitious international commitments have been made to preserve biodiversity, with the goal of preventing extinctions and maintaining ecosystem resilience, yet the efficacy of large-scale protection for preventing near-term extinctions remains unclear. Here, we used a trait-based approach to show that global actions—such as the immediate abatement of all threats across at least half of species ranges for ~10,000 bird species—will only prevent half of the projected species extinctions and functional diversity loss attributable to current and future threats in the next 100 years. Nonetheless, targeted recovery programmes prioritizing the protection of the 100 most functionally unique threatened birds could avoid 68% of projected functional diversity loss. Actions targeting ‘habitat loss and degradation' will prevent the greatest number of species extinctions and proportion of functional diversity loss relative to other drivers of extinction, whereas control of ‘hunting and collection' and ‘disturbance and accidental mortality' would save fewer species but disproportionately boost functional richness. These findings show that conservation of avian diversity requires action partitioned across all drivers of decline and highlight the importance of understanding and mitigating the ecological impacts of species extinctions that are predicted to occur even under optimistic levels of conservation action. Biodiversity is declining at an unprecedented rate, with implications for ecosystem functioning and the delivery of ecosystem services1,2. Human activity has led to widespread decline in the extent and structural condition of ecosystems and changes in community trait composition3. High functional diversity—the diversity of traits that describe an organism's ecological niche—has been associated with greater ecosystem functioning4,5, more reliable ecosystem service delivery6 and greater ecosystem resilience7,8. Therefore, changes in community composition could undermine the persistence of natural communities. Owing to the potential importance of functional diversity in supporting ecosystem function and resilience9, identifying effective measures for conserving functional diversity alongside species richness is paramount10. Ambitious policies and substantial conservation resources have been dedicated to halting and reversing biodiversity loss by reducing the impact of threats11. Programmes designed to alleviate threats at a large scale, a strategy referred to as threat abatement12, are essential for the long-term persistence of species; however, it remains unclear to what extent they can avert imminent extinctions and functional diversity loss. Previous studies have rarely extended past isolated analyses of single threats and their impacts on species richness or broader syntheses of the coverage of conservation targets13,14. The main alternative to threat abatement strategies is direct management interventions such as breeding programmes and translocations. These measures can be effective15,16, particularly for rare species or those that are vulnerable to human pressures17. However, targeted recovery programmes, including ex-situ conservation and in-situ measures to boost species survival and success, are often prohibitively expensive18, limiting their application as a global conservation strategy17. Therefore, conserving bird diversity can probably only be achieved with a combination of large-scale protection through threat abatement coupled with targeted species recovery programmes15,19. However, the extent to which abatement can reduce the need for intensive management to boost species population and reproductive success remains unclear. Here, we used a trait-based approach to evaluate how much biodiversity and associated ecological function could be protected in the near term, defined as the next 100 years, under different global conservation strategies. We assessed the probable success of strategies focusing on the abatement of current and future drivers of extinction and estimated whether shortfalls in efficacy can be countered through targeted species recovery programmes. We used a phylogenetic generalized linear mixed model (PGLMM) to predict species extinction risk based on threats listed by the International Union for the Conservation of Nature (IUCN), accounting for non-independence geographically and across the avian tree of life. We quantified the importance of conserving unique species, which provide a disproportionate contribution to the global diversity of form and function in birds. We projected expected bird extinctions for the next 100 years based on IUCN Red List threat data20. We fitted a PGLMM implemented in a Bayesian Markov chain Monte Carlo framework that predicted species assignment to Red List category with 86.8% accuracy (Supplementary Analyses), using data on threat scope and severity and including random effects describing the spatial and phylogenetic relationships among species. We then stochastically projected species extinctions based on expected probabilities of extinction for each Red List category (see Methods). In the baseline extinction scenario, we assumed that human activity and natural threats would continue to impact bird populations as currently listed. In this scenario, we predicted that 5.2 ± 0.2% (mean ± s.d.) of the 9,873 extant birds studied would go extinct in the next 100 years (517 ± 19 species) (Fig. 1); more than three times the recorded number of bird extinctions since 1500. This figure falls within the range of previously predicted bird extinctions, ranging from 226–589 species extinctions in the next 500 years21 to 669–738 species extinctions in the next 100 years22. Extinctions on this scale are expected to fundamentally alter the global bird assemblage, potentially reducing functional diversity23,24. a, Loss in species richness and functional richness under three scenarios: baseline extinction, partial abatement of all drivers of extinction and complete abatement of all drivers of extinction. Black points show mean loss across 1,000 iterations for each scenario, with variation in those points shown by their distribution (violin plots) and the individual values (grey dots). b, Diversity loss avoided under driver-specific complete abatement of six major drivers of extinction (circles represent the mean; error bars, 0.5 s.d.). Note that in some iterations, loss avoided could be negative, as more diversity was lost with driver-specific abatement than under the baseline scenario. The dotted diagonal line shows mean functional richness loss per species richness loss under complete abatement of all threats. Drivers above this line show greater avoidance of functional richness loss per species richness loss avoided relative to the mean across all drivers of extinction. Hunting, hunting and collection; climate, climate change and severe weather; invasive, invasive species, genes and disease; disturbance, disturbance and accidental mortality. Analyses based on 9,873 species (of which 2,087 species currently listed as Near Threatened or in threatened categories were modelled and could have reduced extinction risk in the abatement scenarios). A total of 1,000 iterations were run for each extinction scenario. To quantify projected change in functional diversity in the world's avifauna (n = 9,873), we used published data on 11 continuous morphological traits that collectively capture bird ecological niches through their well-established association with diet, dispersal and habitat25,26,27. These traits were summarized using the first three axes produced by phylogenetic principal component analysis (pPCA), which explained 87.2% of variance in the dataset (Extended Data Fig. 1), providing an overview of global avian functional diversity (Extended Data Fig. We estimated functional diversity using probabilistic hypervolumes28, which can be applied to multidimensional data and have been shown to be less sensitive to extreme trait values than other methods, such as convex hulls29. We quantified the volume of trait space occupied by the current global avian assemblage (n = 9,873) as well as under future extinction scenarios (Extended Data Fig. Under the baseline extinction scenario, functional diversity was projected to decrease by 3.2 ± 0.4% in the next 100 years relative to present-day functional diversity. This is probably a conservative estimate that only reflects loss in three-dimensional functional space and ignores internal erosion of the space23,30. Projected functional diversity loss varied between 2.4 ± 0.9% and 3.8 ± 0.4% when measured with two dimensions and four dimensions, respectively (Supplementary Fig. Threat abatement could prevent species extinctions and reduce functional diversity loss. However, it is unclear to what extent imminent biodiversity loss can be avoided, and what scale of action is required to prevent species extinctions and functional diversity loss altogether. Using our PGLMM model, we predicted how extinction risk would change under three management scenarios that reflect varying levels of threat abatement (see Methods and Fig. Complete abatement involved removal of all direct drivers of extinction across the entirety of all species ranges; partial abatement involved removal of all direct drivers of extinction across at least half of all species ranges (threat spatial scopes downgraded to ‘Minority < 50%'); and minimal abatement involved the removal of all direct drivers of extinction across at least 10% of all species ranges (threat spatial scopes downgraded to ‘Majority 50–90%'). a, IUCN Red List data20 on threat scope and severity were used to assign projected population decline over a 10-year period or three generations, according to previous publications83,43. Data on projected population decline for all species and all threats were used in an MCMCglmm to predict IUCN extinction risk category, using phylogenetic and spatial variables as random effects. NT, Near Threatened; VU, Vulnerable; EN, Endangered; CR, Critically Endangered. b, Four extinction scenarios were used: baseline, in which current, future and likely-to-return threats remained as listed by the IUCN20; complete abatement, in which threats were removed across the entirety of the species range; partial abatement, in which threats were removed from at least 50% of the species range; and minimal abatement, in which threats were removed from at least 10% of the species range. Rattus fuscipes (Rachel T Mason, CC0 1.0) and Quercus robur silhouettes from Phylopic. Globe silhouette from ClipSafari (Sev, CC0 1.0). Under the complete abatement scenario, half of the biodiversity loss predicted under the baseline scenario could be prevented (Fig. 1 and Extended Data Table 1). Given that our model did not include Least Concern species (for which threat data are scarce), it could not predict assignment to the Least Concern category. However, in reality, threat abatement could result in full recovery to Least Concern. To evaluate the impact of this on projected diversity loss, we tested the effect of assuming a low extinction probability of 0.0001 for the Near Threatened category, equal to the expected for the Least Concern category (1 × 10−6 extinctions per species per year). We obtained similar estimates (241 ± 18 species extinctions; 1.4 ± 0.3% functional diversity loss; Supplementary Analyses), showing that lack of assignment of species to the Least Concern category under abatement scenarios did not notably affect our results. Some extinctions were not preventable even with complete abatement; therefore, they were not attributable to current and future drivers of extinction. These extinctions could reflect particularly vulnerable species that have high extinction risk despite being affected by few threats, as well as species that were severely affected by past threats that can no longer be managed or abated. The model captured variation in species vulnerability to extinction that was not described by threats through spatial and phylogenetic random effects. The relevance of spatial and phylogenetic variables was supported by the fact that background extinction rates and extinctions owing to stochastic events varied by taxonomic group31 and across space32. Deviations from expected extinction risk captured by spatial and phylogenetic variables could reflect fast or slow life history33, variation in overlap with areas of high human influence34 and isolation and connectivity35. These factors are expected to be important for explaining variation in both extinction risk and species recovery36,37,38 and often exhibit high degrees of spatial or phylogenetic correlation39,40,41. For example, island endemics are particularly sensitive to extinction because of their small and isolated ranges35, which may be captured through spatial random effects, and evolution of traits associated with increased extinction risk, such as flightlessness, which may be captured by phylogenetic random effects35. The Cebu flowerpecker (Dicaeum quadricolor) is a Critically Endangered species that was predicted to be at risk of extinction even under complete abatement. Like many island species, it has a very small remaining population (60–70 individuals)42, and our analysis suggests that it is likely to go extinct without complementary measures such as habitat restoration or ex-situ conservation. Our finding that even large-scale and ambitious actions leading to the removal of all present, future and likely-to-return threats will fail to prevent almost half of projected species extinctions challenges some of the key assumptions of global metrics used to track conservation progress. For example, the Species Threat Abatement and Restoration (STAR) metric43 is based on IUCN Red List data on threat scope and severity but assumes that complete threat abatement will allow the vast majority of species to be downgraded to Least Concern, an assumption that our findings did not support. Although those authors43 acknowledge that some species may require restoration to be downgraded to Least Concern, our results suggested that many species will require conservation measures in addition to threat abatement. Even when species are not affected by current or future threats, they may still be threatened with extinction. Although we did not explicitly test the reasons for ongoing declines, they could occur because of continued population decline (particularly in populations which are no longer self-sustaining), high vulnerability to stochastic events because of small population or range size, or reduced fitness as a result of severe population decline in the past44. Partial abatement was somewhat effective at reducing avian diversity loss, preventing about one-quarter of projected losses (26 ± 4% of projected species extinctions and 26 ± 13% of projected functional diversity loss; Fig. Species that are experiencing severe declines but are affected by few threats showed the greatest reduction in extinction risk under partial abatement. The green-faced parrot finch (Erythrura viridifacies) and the Saint Vincent parrot (Amazona guildingii) responded particularly well to partial abatement, with a reduction in extinction risk that was almost as great as the reduction in extinction risk under complete abatement. Although partial abatement prevented some losses, there was still a 3.9% decrease in species richness (385 ± 18 species) and a 2.3 ± 0.3% decrease in functional richness. Of the diversity loss that was attributable to the drivers of extinction (diversity loss under complete abatement), approximately half was prevented through partial abatement (50 ± 7% of species extinctions and 49 ± 26% of functional diversity loss). The minimal abatement scenario prevented only a small proportion of biodiversity loss (Extended Data Table 1). Using different traits to quantify functional diversity did not affect our conclusions (Supplementary Analyses), as we obtained similar results when we used pPCs constructed from three-dimensional scans of beak morphology45 and when the first pPC (largely describing variation in body size) was removed. Protecting species from the drivers of extinction does not provide a comprehensive solution to biodiversity loss in the near future without additional measures such as targeted species recovery programmes, habitat restoration and prioritization of protection in important areas46. This finding is consistent with previous studies assessing biodiversity impacts of future conservation and mitigation scenarios. One study47 found that although it was possible to bend the curve of biodiversity loss with an integrated strategy, protected area management and expansion to avert the impact of habitat loss and degradation were insufficient to avoid more than 50% of projected biodiversity loss on average in biodiversity-rich regions. Similarly, another study48 predicted that in a 2015–2050 scenario of strong land use and climate change mitigation globally, rates of biodiversity loss would decrease but biodiversity would continue to decline. Threat reduction is an essential component of tackling the biodiversity crisis and is necessary to ensure that species with healthy, stable populations do not go into decline in the future15. However, it is not enough. The abatement scenarios explored here represented significant management efforts with optimistic assumptions about their impact and uptake. We assumed that the drivers of extinction, and the species declines caused by these drivers, could be halted immediately and that all drivers of extinction could be alleviated, including climate change, which arguably may be difficult to mitigate with site-based protection. Even in these ambitious and optimistic scenarios, we predicted that over half of the projected species extinctions and loss of functional diversity in the next 100 years would occur anyway. Projected loss of functional diversity was not evenly distributed across functional space (Figs. Areas of trait space with high pPC1 values (generally larger birds) were predicted to show the greatest proportional losses under the baseline extinction scenario. Complete abatement was predicted to reduce loss across functional space (Fig. 3 and Extended Data Fig. 4) but was less effective in a region of high pPC1 and pPC2 values, predominantly occupied by large aquatic predators (Extended Data Fig. pPC1 is a descriptor of body size, pPC2 is associated with wing morphology and pPC3 is associated with beak and tail morphology (for trait loadings, see Supplementary Table 3). a,b, Predicted proportional decline in functional trait space occupation in the next 100 years under the baseline extinction scenario with respect to pPC1 and pPC2 (a) and pPC3 and pPC2 (b). c,d, Averted proportional decline under the complete abatement scenario for pPC1 and pPC2 (c) and pPC2 and pPC3 (d). In all panels, grey colour shows areas where no functional diversity loss was projected or where no functional diversity loss was avoided under complete abatement (fewer than five pixels in all panels). Analyses based on 9,873 species (of which 2,087 species currently listed as Near Threatened or in threatened categories were modelled and could have reduced extinction risk in the abatement scenarios). A total of 1,000 iterations were run for each extinction scenario. All silhouettes are from Phylopic. In a and c (left to right): Apteryx (Ferran Sayol, CC0 1.0), Mellisuga helenae (Steven Traver, CC0 1.0), Troglodytes hiemalis (Andy Wilson, CC0 1.0), Pteroptochos castaneus (Ferran Sayol, CCO 1.0), Atlantisia rogersi (there was no silhouette of A. rogersi so a silhouette of Gallirallus australis was used instead; T. Michael Keesey and HuttyMcphoo, CC BY-SA 3.0), Pelecanoides urinatrix (Louis Ranjard, CC BY 3.0), Spheniscus humboldti (Juan Carlos Jerí, CC0 1.0), Larus (Ferran Sayol, CC0 1.0), Diomedeidae (Ferran Sayol, CC0 1.0), Struthio camelus (Darren Naish and T. Michael Keesey, CC BY 3.0), Buceros (Ferran Sayol, CC0 1.0) and Leptoptilos javanicus (T. Michael Keesey and Vaibhavcho, CC BY-SA 3.0). In b and d (left to right): Apteryx (Ferran Sayol, CC0 1.0), Pelecanus (Ferran Sayol, CC0 1.0), Ramphastidae (Federico Degrange, CC0 1.0), S. humboldti (Juan Carlos Jerí, CC0 1.0), M. helenae (Steven Traver, CC0 1.0), Buceros (Ferran Sayol, CC0 1.0), Apus apus (Ferran Sayol, CC0 1.0), Phasianus colchicus (Mattia Menchetti, CC0 1.0), Menura (T. Michael Keesey, CC0 1.0) and S. camelus (Darren Naish and T. Michael Keesey, CC BY 3.0). a, Posterior values from a multi-response MCMCglmm showing the relationships between pPC values and the frequency (from 1,000 iterations across 9,873 species) in which extinction was avoided under driver-specific complete abatement scenarios. pPC1 is a descriptor of body size, pPC2 is associated with wing morphology and pPC3 is associated with beak and tail morphology (Supplementary Table 3). Least Concern species were not included in the extinction risk model, as improvements under driver-specific complete abatement could not occur by definition. b, Distribution of drivers of extinction with respect to phylogeny, shown by family (9,873 species across 194 families, of which threat information was included for 2,087 Near Threatened and threatened species), with the intensity of colour reflecting the proportion of species in a family affected by each driver (families including only Least Concern or Data Deficient species are shaded white). All silhouettes are from Phylopic. In a (left to right): T. hiemalis (Andy Wilson, CC0 1.0), S. camelus (Darren Naish and T. Michael Keesey, CC BY 3.0), Apteryx (Ferran Sayol, CC0 1.0), A. apus (Ferran Sayol, CC0 1.0), Pelecanus (Ferran Sayol, CC0 1.0), Menura (T. Michael Keesey, CC0 1.0). In b (left to right): Falconiformes (Kai Caspar, CC0 1.0), Coraciiformes (Estelle Bourdon, CC0 1.0), Piciformes (Federico Degrange, CC0 1.0), Bucerotiformes (Ferran Sayol, CC0 1.0), Charadriiformes (Auckland Museum, CC BY 3.0), Apodiformes (Andy Wilson, CC0 1.0), Passeriformes (Andy Wilson, CC0 1.0), Eurypygiformes (Ferran Sayol, CC0 1.0), Pelecaniformes (Ferran Sayol, CC0 1.0), Suliformes (Juan Carlos Jerí, CC0 1.0), Procellariiformes (Louis Ranjard, CC BY 3.0), Musophagiformes (Ferran Sayol, CC0 1.0), Gruiformes (Ferran Sayol, CC0 1.0), Phoenicopteriformes (T. Michael Keesey, PDM 1.0), Mesitornithiformes (Ferran Sayol, CC0 1.0), Galliformes (Elisabeth Östman, PDM 1.0), Anseriformes (Rebecca Groom, CC BY 3.0), Apterygiformes (Ferran Sayol, CC0 1.0) and Tinamiformes (Darren Naish and T. Michael Keesey, CC BY 3.0). Species' traits shape their vulnerability to human activity, but different areas of trait space are affected by different threats49,50. As such, abatement of drivers of extinction could have differential outcomes for functional diversity. To test this concept, we focused on six drivers of extinction (Supplementary Dataset 1) and quantified the ‘maximum avoidable contribution' from each driver, describing the species and functional diversity loss avoided when the impact of current and future threats within each driver of extinction were completely removed, relative to diversity loss in the baseline scenario (see Methods). ‘Habitat loss and degradation' had the highest maximum avoidable contribution, as driver-specific complete abatement was projected to avoid 1.4 ± 0.2% species richness loss (141 ± 24 species extinctions) and 0.9 ± 0.5% functional diversity loss (Fig. Driver-specific complete abatement of ‘hunting and collection' was projected to avoid 0.4 ± 0.5% functional diversity loss (Fig. 1), almost half that of ‘habitat loss and degradation' despite requiring action for 591 species rather than 1,658 species. Other drivers of extinction had smaller maximum avoidable contributions (Fig. The relative magnitude of maximum avoidable contributions among drivers was comparable when simulating driver-specific partial abatement and driver-specific minimal abatement rather than driver-specific complete abatement. Although assessments of individual drivers on avian functional diversity exist13,14,51, assessments of multiple drivers simultaneously are rare and are important for quantifying the relative impact of different drivers of extinction on avian functional diversity. We found that driver-specific complete abatement of ‘hunting and collection' and ‘disturbance and accidental mortality' was projected to result in disproportionately high avoidance of functional diversity loss for the number of species extinctions avoided (Extended Data Fig. As abatement of different drivers of extinction had different value for the preservation of functional diversity, we argue that it is necessary to consider functional diversity in conservation planning and prioritization. To assess which species traits were vulnerable to drivers of extinction, we used a mixed-effects multi-response regression model of reduction in extinction risk under driver-specific complete abatement scenarios against values of three pPCs (see Methods). Reduction in extinction risk was quantified as the number of iterations in which species extinction was avoided in driver-specific complete abatement scenarios relative to the baseline scenario. A significant positive relationship was detected when abatement of a given driver of extinction reduced extinction risk in species with high values of a given pPC, and a significant negative relationship occurred when abatement of a given driver of extinction reduced extinction risk in species with low values of a given pPC (Fig. We found that the abatement of different drivers of extinction would benefit distinct morphologies. Birds with large body size (pPC1) were more likely to experience a reduction in extinction risk when abating ‘hunting and collection' or ‘climate change and severe weather' (Fig. Although extinction risk bias towards species with large body size is widely reported52,53, we found that this was not the case for all threats, as there was no evidence of bias with respect to pPC1 for other drivers of extinction49. The bias in extinction avoidance with respect to wing morphology (pPC2) was variable across drivers of extinction. Birds with broader wings (those with low pPC2 values) were more likely to experience a reduction in extinction risk under abatement of ‘habitat loss and degradation' (PMCMC < 0.01). By contrast, birds with slender wings (high pPC2 values) were more likely to experience a reduction in extinction risk under abatement of ‘hunting and collection' (PMCMC = 0.08; values under 0.1 are treated as significant to give a one-tailed significance test of overlap with zero rather than the default two-tailed test). Our finding that extinction avoidance was more likely for species with broad wings (low values of pPC2) when abating habitat loss and degradation is consistent with recent studies that show that birds with a low hand-wing index (described by pPC2) are more sensitive to fragmentation54 and deforestation55. The bias in extinction avoidance with respect to tail and beak morphology (pPC3) was also variable across drivers of extinction. Species with long tails and short beaks (low pPC3 values) were more likely to experience a reduction in extinction risk following abatement of both ‘hunting and collection' and ‘invasive species, and disease' (PMCMC = 0.96 and PMCMC < 0.05, respectively (hunting and collection was not significant when insignificant variables were removed)), whereas species with short tails and long beaks (high pPC3 values) were more likely to experience a reduction in extinction risk following abatement of ‘climate change and severe weather' (PMCMC = 0.08). Reduced extinction risk following abatement of climate change and severe weather was associated with traits involved in thermoregulation56,57,58,59. Birds with large body size (pPC1) but also large beaks (pPC3) were more likely to avoid extinction when climate change and severe weather was abated. As a bird's beak also influences its trophic niche60, failing to mitigate species decline caused by climate change could have knock-on effects for trophic interactions. Variable extinction avoidance across functional trait space suggests that prioritizing threat abatement based on the magnitude of projected biodiversity loss alone is inappropriate. Reducing the impact of multiple drivers of extinction is necessary to ensure that diverse functional morphologies are conserved. Even with ambitious action, large-scale threat abatement will not prevent all species extinctions and functional richness loss in the next 100 years. As such, targeted species recovery programmes will be needed, which we defined as in-situ and ex-situ measures to boost species survival and reproductive success that do not involve threat reduction. Here, we explored one possible approach, quantifying the benefits of preventing a small number of species extinctions targeted to reduce the loss of global functional richness. Using a metric of functional uniqueness that describes the probability of functional richness loss as a result of species extinction (Extended Data Fig. 7), we identified the most unique threatened species among the 9,873 bird species studied. Preventing the extinction of the most unique species (we tested scenarios protecting between 40 and 200 species; Fig. 5) was effective at reducing projected functional diversity loss. We found that preventing the extinction of the top 100 most unique threatened species avoided 68 ± 5% of projected functional diversity loss under the baseline scenario compared to the 26 ± 13% avoided by partial abatement of all threats for all species. By preventing the extinction of 100 species (1% of species), 2.2 ± 0.32% of functional diversity could be conserved if the most functionally unique threatened species were prioritized. This approach would require the avoidance of 37 ± 25 projected extinctions in the next 100 years. A previous publication17 reported that 21–32 bird species have been saved from extinction by conservation efforts since 1993, suggesting that this could be an achievable goal (although ten extinctions occurred despite management). Functional diversity loss avoided (as a percentage of projected functional diversity loss under the baseline scenario) from 1,000 iterations. Black points show mean loss avoided; violin plots show the distribution; grey points show individual values of loss avoided under each iteration. The number of unique threatened species that were prevented from going extinct (‘protected') varied between 40 and 200 unique threatened species at intervals of 20 species. The most functionally unique birds spanned taxonomic and ecological groups, from the Sulu hornbill (Anthracoceros montani) of the southernmost Philippine islands61 to the Ascension frigatebird (Fregata aquila) that patrols the Atlantic Ocean. Some were wide-ranging, like the southern royal albatross (Diomedea epomophora), and others are thought to only survive in one location, like Stresemann's bristlefront (Merulaxis stresemanni). Unique species included scavengers, such as the Andean condor (Vultur gryphus), nectarivores, such as the yellow-bellied sunbird-asity (Neodrepanis hypoxantha), vertivores, such as the Madagascar serpent-eagle (Eutriorchis astur) and frugivores, such as the bare-necked umbrellabird (Cephalopterus glabricollis). A full list of the top 200 most unique threatened birds is given in Supplementary Dataset 2. Previous studies have found that functionally unique species are more likely to be threatened with extinction than less functionally unique species30,62. We provide evidence that conservation strategies for birds should prioritize functionally unique species, as has been proposed for other taxonomic groups63,64. In addition to their inherent value, functionally unique species are more likely to be used by humans for food, material and medicine; therefore, preventing the extinction of functionally unique species could be important for the delivery of ecosystem services65. Effective targeted recovery programmes that explicitly consider species uniqueness hold great potential for conserving global functional diversity as a complementary strategy to threat abatement. Both large-scale protection from the drivers of extinction and targeted species recovery programmes will be needed to prevent avian extinctions and functional diversity loss in the next 100 years. Although not effective at preventing all biodiversity loss, threat abatement is essential for ensuring that species that currently have healthy, stable populations do not fall into decline15. Nevertheless, our findings suggest that conservation policy should not focus solely on large-scale protection from the drivers of extinction, given that even in ambitious scenarios, only half of the projected species extinctions and functional diversity loss attributable to these drivers of extinction could be avoided. Abatement of ‘habitat loss and degradation' made the greatest overall contribution to avoided species extinctions and functional diversity loss, but management of ‘hunting and collection' and ‘disturbance and accidental mortality' prevented greater functional diversity loss proportional to the number of species projected to become extinct. Given that different areas of functional trait space were impacted by different drivers, consideration and abatement of all drivers of extinction is necessary to conserve functional diversity. When completely or partially abating the drivers of extinction, functional richness loss was correlated with species extinctions, so reducing species extinctions is projected to reduce functional diversity loss. However, targeted species recovery programmes that focus on functionally unique species hold great potential for the conservation of functional diversity, while requiring conservation of relatively few species. By conserving the top 100 most unique threatened species, it may be possible to prevent more than two-thirds of the projected functional diversity loss through avoiding ~37 species extinctions. Although prioritization of recovery programmes offers great potential for protecting functional diversity, the ethical questions about prioritizing some species over others and the risks of overlooking ecosystem functions and services provided by other species, whether known or unknown, must be considered. If human activity continues to affect biodiversity as it is today, we project that in the next 100 years, we will lose more than three times the number of bird species as have been lost since 1500. It is, therefore, urgent that we decide which dimensions of biodiversity we wish to protect, consolidate their measurement and include them in every stage of conservation planning, monitoring and impact assessment. We used data on species morphological and geographical traits, threats and phylogenetic relationships to conduct this study. Trait data for 11,003 extant bird species66 were obtained from AVONET26. Under BirdLife taxonomy, only 8% of species in AVONET have imputed data for one or more traits, and <5% of species have imputed data for more than one trait. For all study species, data on threats were obtained in June 2022 from the IUCN Red List20 using the function rl_threats in the package rredlist67 in R68. Bird species are reassessed every 4 years, causing a slight possible delay between species decline or recovery and reported change in extinction risk category69. Taxonomic discrepancies between AVONET and IUCN (n = 141 species) were reconciled using the function rl_synonym. One-to-one matches were found for all species; therefore, these taxonomic differences did not impact the results. A maximum clade credibility phylogenetic tree was constructed from the first 1,000 trees in a previous publication70 based on the Hackett backbone70,71. The authors70 included 9,993 species in their analysis; we refer to the species nomenclature and taxonomic treatments adopted in this study as ‘BirdTree taxonomy'. To enable analysis of functional diversity loss while accounting for phylogenetic covariance between species, differences between the BirdLife66 and BirdTree70 taxonomies were reconciled using the crosswalk provided with AVONET26 (Supplementary Analyses). This gave 9,879 selected synonym matches between BirdLife and BirdTree (89.9% of BirdLife synonyms and 98.9% of BirdTree synonyms). Repeating analyses with all BirdLife synonyms and non-pPCA had a small impact on the percentage of projected species extinctions and functional diversity loss but did not affect our conclusions (Supplementary Fig. Five species treated as Extinct in the Wild and one species listed as Extinct by IUCN20 but not listed as extinct in the AVONET crosswalk (Zosterops conspicillatus) were removed from the analysis, giving a total of 9,873 species. Functional diversity quantifies the diversity of functional traits within an assemblage, defined as the measurable characteristics of an organism that influence its ecological niche72,73. We used 11 continuous morphological traits extracted from AVONET26, including body mass and linear measurements of beak, wing, tail and tarsus. These traits collectively capture bird ecological niches through their association with diet, dispersal and habitat25,26,27. Using continuous morphological traits enables more fine-grained discrimination between species sharing the same ecological groups, thus providing more in-depth information about ecological variation between species than categorical traits60. As life history traits are more useful for explaining variation in species response to human activity rather than the ecological impacts of decline26, they were not included in functional diversity estimations. Trait data were log10 transformed and scaled to unit variance. We used pPCA to reduce dimensionality. PCA produces axes that are mathematically uncorrelated but may be phylogenetically correlated if species trait data are non-independent owing to shared evolutionary history74,75. pPCA accounts for phylogenetic correlation between axes by removing phylogenetic covariance and calculating major axes of non-phylogenetic residual variation75. pPCA was carried out using the phyl.pca function in phytools76 based on covariance and using lambda to obtain the correlation structure, which was optimized using restricted maximum likelihood. Adding more pPCs described comparatively less variation (see scree plots, Extended Data Fig. We therefore used the first three pPCs to summarize variation in the dataset. Using the first two or four pPCs instead, or using alternative ordination methods, did not affect our conclusions (see Supplementary Tables 4 and 5 and Supplementary Figs. We calculated global functional richness for the whole assemblage (9,873 species) using trait probability densities28. Firstly, a multivariate Gaussian probability distribution was fitted for each species (Extended Data Fig. 3), in which means were provided by pPC values derived from functional trait data26 and standard deviations were estimated using a bandwidth selector (Hpi.diag function from package ks77). Next, we took the sum of species probability distributions to obtain the community trait probability density (Extended Data Fig. This was implemented through the TPDsMean and TPDc functions in package TPD78, with 50 divisions for each pPC. We calculated functional richness using the REND function in package TPD. To predict how threat reduction affected projected avian diversity loss, we constructed a model of species extinction risk (IUCN Red List category20) with threats as explanatory variables and accounting for spatial and phylogenetic covariance (referred to as ‘the extinction risk model'). The extinction risk model allowed us to quantify the independent contribution of each threat to extinction risk, while considering that many species were affected by multiple threats (1,978 species out of 2,104 Near Threatened and threatened species with threats listed) and comparatively few were affected by only one (126 out of 2,104 Near Threatened and threatened species with threats listed). Overlooking non-independence between threats can result in misleading findings about the relationship between threats and extinction risk as well as patterns of bias in the impacts of these drivers across species assemblages79. Species vulnerability to extinction, and threat prevalence (Fig. 1), may be affected by where species live or their evolutionary history, so species data are not independent of one another80. We included phylogeny and spatial variables as random effects to account for non-independence among species owing to non-random baseline extinction rates80, as well as other factors influencing extinction risk and recovery, such as small population size32, that are not caused by population decline resulting from listed past, ongoing or future threats. The extinction risk model was fitted using a Markov chain Monte Carlo multivariate generalized linear mixed model (MCMCglmm), predicting extinction risk for species that were listed by the IUCN Red List as Near Threatened, Vulnerable, Endangered and Critically Endangered. Threats have been described for 99% of species in these categories. MCMCglmms were fitted using the R package MCMCglmm81. We used 39 pseudo-continuous fixed effects, describing the expected percentage population decline over a 10-year period or three generations (from threat scope and severity data; see details below and Extended Data Table 3) caused by each threat under the second-level classification described by the IUCN82 (for example, ‘1.1 Housing & Urban Areas' and ‘1.2 Commercial and Industrial Areas'). Threats that affected ten or fewer species were grouped with other threats (Supplementary Dataset 1). Threats were assigned an expected population decline (over a 10-year period or three generations) based on their scope (percentage of species range affected by a threat) and severity, following previous publications43,83 (Extended Data Table 3). If multiple threats were listed for the same threat category under the second-order classification listed by the IUCN82, the maximum expected population decline was used. For example, Acrocephalus familiaris is experiencing slow, continuous declines owing to the invasive species Schistocerca nitens across most of its range, but it is also experiencing rapid declines caused by Oryctolagus cuniculus across the whole of its range. For ‘8.1 Invasive non-native/alien species/diseases', A. familiaris was assigned an expected population decline of 24%, associated with rapid declines across the whole of its range (Extended Data Table 3). We took the maximum expected population decline for second-order threats where multiple third-order threats were listed, as not all second-order threats had information on third-order threats, and without further information, it was difficult to estimate the expected population decline from multiple third-order threats. Only 16% of species–threat combinations had more than one third-order threat listed, and when running the extinction risk model using the sum of expected population decline rather than the maximum, we found that this had minimal impacts on projected species extinctions. Threats expected to cause no decline or negligible declines across a majority or minority of a species' range had an expected population decline of zero and were effectively discarded. Across all species, 11.58% of threat data were missing scope or severity values. Missing scope and severity data were imputed with missForest imputation (implemented through the R package missForest84) from threat type, scope, severity and timing, and incorporating phylogenetic data through eigenvectors85. Removing a similar proportion of values from complete data on threat timing, scope and severity to test imputation accuracy gave a mean accuracy of 82.5% (Supplementary Analyses). A total of 48 rows were also missing data on timing (needed for creating extinction scenarios; see ‘Extinction scenarios' section of the Methods), and these data were imputed in the same way as scope and severity. Phylogeny, minimum latitude, maximum latitude and centroid longitude were included in the model as random effects (Supplementary Analyses and Supplementary Fig. 1), where centroid longitude describes the longitude of the midpoint of species ranges. Spatial variables were obtained from AVONET and had been calculated from species' breeding and resident ranges, including areas where the species was coded as extant and either native or reintroduced26. Centroid latitude was not informative in explaining variation in extinction risk86 (Supplementary Fig. 1) and therefore was not included as a random effect. In total, 17 species with incomplete spatial information were excluded from the extinction risk model. Species with missing spatial information were either Possibly Extinct, had no known breeding or resident range, or their range data had been redacted to protect them from trafficking risk26. A total of 22 species had no threat data listed and were not included in the extinction risk model. The final model was fitted for the remaining threatened and Near Threatened species (n = 2,087). For fixed effects, Cauchy-scaled Gelman priors were used (with an expected value of zero), as is recommended for ordinal regressions87. For phylogenetic random effects, we used a chi-squared prior (expected covariance of 1, degree of belief of 1,000, mean vector of 1 and covariance matrix of 1), as this best approximates a uniform distribution, giving an uninformative prior81,88,89. For spatial random effects, we used parameter-expanded priors (expected covariance of 1, degree of belief of 1, mean vector of 0 and covariance matrix of 625), as they are often less informative than the default inverse-Wishart prior90. As it is not possible to estimate the residual variance with an ordinal response variable (extinction risk), the residual variance was fixed to 1 following previous work90. The model was insensitive to alternative prior specification (Supplementary Figs. Model convergence was assessed by parameter traces produced through the plot function in package MCMCglmm81. Non-significant fixed effects were removed iteratively, removing the least significant fixed effect, rerunning the model and repeating until only significant fixed effects remained. Significance was assessed using the pseudo P value (PMCMC), estimated by MCMCglmm81. The pseudo P value is calculated as the probability that the posterior is greater or less than zero, whichever is smaller, multiplied by two91. A significance threshold of 0.1 was used, giving a one-tailed significance test that the posterior distribution overlaps with zero, rather than the default two-tailed test. The final model structure was: where X1.2 is the expected percentage population decline (Extended Data Table 3) owing to IUCN second-order threat ‘1.2 Commercial and Industrial Areas' and so on. X10.1 grouped threat impacts from X10.1, X10.2 and X10.3, as each threat affected fewer than ten species (see Supplementary Dataset 1 for threat codes, threat descriptions and model parameter estimates). We expected most threats to have a positive posterior mean, indicating that species affected by these threats had higher extinction risk; however, ‘2.2 Wood and pulp plantations' and ‘10 Geological events' (including X10.1, X10.2 and X10.3) had a small, negative posterior mean. Our model estimates the independent contribution of each threat to extinction risk; therefore, although it may appear that wood and pulp plantations and geological events are contributing to decline when combined with other threats, our model suggests that, in general, species threatened by wood and pulp plantations and geological events have slightly lower extinction risk. Model accuracy was assessed as the proportion of species for which the category listed by the IUCN20 matched the category that most frequently (across iterations) had the highest probability. We used the extinction risk model to predict the probability that species belonged to each Red List category; then, using the expected probability of extinction for each Red List category, simulated extinctions that are likely to occur in the next 100 years. We used an approach explicitly incorporating uncertainty in model estimates and stochasticity in realized extinctions given extinction probabilities. The extinction risk model returned 1,000 posterior estimates (1,000 iterations) of the probability that each species belonged to each extinction risk category (Near Threatened, Vulnerable, Endangered and Critically Endangered). All Least Concern species and 39 Near Threatened or threatened species that were missing spatial or threat data (species not included in the extinction risk model) were assigned a probability of 1 of belonging to their Red List category as currently listed by the IUCN. Species classified as Data Deficient (n = 41) were conservatively assigned to the Least Concern category. For many classes, Data Deficient species are likely to be at higher risk of extinction than data-sufficient species93; however, this is not the case for birds94. Given that Data Deficient species make up a very small percentage of total species (0.4%), uncertainty over their extinction risk was expected to have a negligible impact on projected diversity loss. An overall probability of extinction was then calculated for each species (equation 1): where \({{ex}}_{p,s}\) is the probability of extinction in the next 100 years for species s according to the posterior estimation p; ex100c is the assigned probability of extinction in the next 100 years of a species in IUCN extinction risk category c; and \({{cat}}_{p,c,s}\) is the probability that species s belonged to IUCN extinction risk category c according to posterior estimation p or, for species not included in the extinction risk model, a probability of 1 for their Red List category as currently listed by the IUCN and a probability of 0 for all other Red List categories. Estimates of \({{ex}}_{p,s}\) were converted to a binary outcome of extinct or extant using the R function sample, in which the probability of being assigned extinct was \({{ex}}_{p,s}\). For each scenario, we report the mean and standard deviation in the number of extinctions across 1,000 iterations as a percentage of the total number of species included in the study (9,873 species). Functional diversity loss was estimated by removing species projected to go extinct, calculating functional diversity across species predicted to be extant and comparing to the functional diversity of the full assemblage (9,873 species). We estimated projected loss in species and functional diversity in the next 100 years under a baseline extinction scenario and three threat reduction scenarios: complete abatement, partial abatement and minimal abatement. Under the baseline scenario, we used the extinction risk model to predict the probability that species belonged to Red List categories, assuming that the impact of all threats remained as currently listed by IUCN20, following the method for predicting extinctions outlined above. Under the complete abatement scenario, predictions were obtained after setting the expected population decline to zero for all threats with a timing of ‘Ongoing', ‘Past, Likely to Return' and ‘Future'. The expected population decline of threats with a timing of ‘Past, Unlikely to Return' was retained as they cannot be prevented but could still contribute to extinction risk through extinction lags (although predictions were similar if expected population decline for these threats was set to zero; Supplementary Table 6). Under the partial abatement scenario, threat impacts were altered to simulate removal of threats across at least 50% of species ranges by reassigning expected population declines for threats with a scope of ‘Whole (>90%)' or ‘Majority (50–90%)' to the decline expected for a scope of ‘Minority (<50%)' (Extended Data Table 3). Under the minimal abatement scenario, threat impacts were altered to simulate the removal of threats across at least 10% of species ranges by reassigning expected population declines of threats with a scope of ‘Whole (>90%)' to expected decline for a scope of ‘Majority (50–90%)'. Least Concern species were not included in the extinction risk model, given that this is the lowest risk category and, by definition, could not show a reduction in extinction risk under abatement scenarios. We used Cohen's D to quantify the effect size of the difference in means of diversity loss under each threat reduction scenario, divided by their pooled standard error. We do not report P values, as the sample size (number of iterations) could be increased easily, reducing standard error and giving significance even with very small differences in means, leading to type 1 errors. To determine the bird morphologies with the greatest extinction risk, we plotted the loss in density of trait space occupation under the baseline scenario using trait probability densities. We constructed a community probability distribution by taking the sum of species probability distributions, in which each species probability distribution was given a weight between 0 and 1,000, and describing the number of iterations in which they did not go extinct under the baseline scenario. The resulting community probability distribution was rescaled to show absolute rather than relative change in the density of occupation of trait space. For each cell, we compared the density of occupation of the whole assemblage to the density of occupation in iterations of the baseline extinction scenario. The loss in density was calculated as a percentage of the density of trait space occupation in the full assemblage for which all species had a value of 1,000, indicating no extinction (Extended Data Fig. Areas of trait space with high loss in density (approaching 100%) were predicted to have a high risk of extinction. To determine the bird morphologies with the greatest reduction in extinction risk under complete abatement of all threats (protection in Extended Data Fig. 4), we plotted the density of trait space occupation under the complete abatement scenario using trait probability densities, whereby all species were given a weight according to the number of iterations in which extinction was avoided in the complete abatement relative to the baseline scenarios. We also plotted the loss in density of trait space occupation that was not avoided, as a proportion of density loss under complete abatement. For plotting loss in density under the baseline scenario, loss in density averted under the complete abatement scenario and loss in density not averted under the complete abatement scenario, pPCs were divided into 100 bins to provide high plot resolution. To facilitate visualization, this process was carried out in two dimensions (two pPCs at a time). We aimed to assess the independent contribution of six major drivers of extinction to projected avian diversity loss. Threats in the extinction risk model were grouped into six ‘drivers of extinction': ‘habitat loss and degradation', ‘hunting and collection', ‘climate change and severe weather', ‘disturbance and accidental mortality', ‘invasive species and disease' and ‘pollution' (see Supplementary Dataset 1). Geological events and threats described as ‘other' were grouped into an ‘Other' category. Although these other threats were included in the model of extinction risk and their impact was accounted for when assessing the impact of all drivers of extinction together, we did not assess their impact individually. We projected species and functional diversity loss under driver-specific complete abatement scenarios in which the impact of threats in a given driver of extinction with a timing of ‘Ongoing', ‘Past, Likely to Return' and ‘Future' were removed by setting their expected population decline to zero. The ‘maximum avoidable contribution' was calculated as the difference in predicted species and functional diversity loss between the baseline scenario and the driver-specific complete abatement scenario. We used Cohen's D to quantify the effect size of the difference in means of diversity loss under the baseline and driver-specific complete abatement scenarios, divided by their pooled standard error. As before, P values were not reported. To determine the severity of functional richness loss under a given driver of extinction in relation to the number of species projected to become extinct, we used a linear mixed-effects model to describe the functional richness loss avoided, using the number of species extinctions avoided and the driver of extinction (categorical) as explanatory variables. Model iteration was used as a random effect to account for non-independence in calculated differences in functional diversity loss and species extinctions from baseline to threat reduction scenarios. The dredge function from the MuMin package96 was used to identify the best model from all combinations of explanatory variables and an interaction between species extinctions and driver of extinction. The best model included the number of species extinctions avoided and the driver of extinction but not the interaction between species extinctions avoided and the driver of extinction. We aimed to find whether abatement of different drivers of extinction could avoid extinction in different regions of trait space. We constructed a multi-response MCMCglmm with the three pPC values for each species as response variables (including all 9,873 species studied) and the number of iterations in which extinction was avoided under driver-specific complete abatement for each driver of extinction as explanatory variables, accounting for phylogenetic covariance. Pollution was not included, as a driver-specific complete abatement of pollution had a negligible impact on functional richness loss. The residual structure was allowed to vary for each response variable. Random effect priors were provided as a diagonal matrix, with an expected covariance between response variables of zero and expected variance within response variables of 1. The degree of belief parameter for the random effect prior was 2 (ref. The expected value of fixed effects and the theta-scale parameter were zero, with a covariance matrix in which the expected covariance between fixed effects was zero and the expected variance within fixed effects was 1 × 1010. Posterior distributions were plotted, and a significance threshold of 5% overlap with zero was used (PMCMC = 0.1). To identify the potential value of preventing the extinction of the most functionally unique threatened species, we calculated functional uniqueness for all species (Extended Data Fig. For each species, we calculated the proportion of the density of the community probability distribution that was occupied by the species probability distribution for each grid cell in which the species probability density was greater than zero. We calculated the mean proportion for each species across grid cells in which the species probability distribution was greater than zero, giving greater weight to cells in which the species distribution had greater probability (higher density). The maximum uniqueness possible was 1, indicating that a particular species' probability distribution had no overlap with the probability distribution of any other species included in the analysis. Uniqueness tended towards a lower limit of zero, indicating that the species probability distribution had high overlap with many other species probability distributions, and many other species occupied the same area of trait space. We then identified the most unique and threatened (listed as Vulnerable, Endangered or Critically Endangered) species as potential targets for action. We compared functional richness loss in the baseline scenario with functional richness loss when extinctions of the most unique threatened species were prevented. We explored the consequences of avoiding extinction for the top 40–200 most unique threatened species (in intervals of 20 species). A total of 1,000 posterior estimates were obtained for each extinction scenario. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. AVONET data on morphological, ecological and geographical traits for all birds26 is available for use under a Creative Commons licence (CC BY 4.0): https://doi.org/10.6084/m9.figshare.16586228.v7. Data on IUCN extinction risk categories and threats affecting each species are available from the IUCN Red List20 and can be accessed through the package rredlist67. Information on the terms of use of IUCN Red List data can be found at https://www.iucnredlist.org/terms/terms-of-use. Supplementary datasets are provided at https://doi.org/10.6084/m9.figshare.26067970 under a Creative Commons license (CC BY 4.0). The code used for figures and analyses is provided at https://doi.org/10.6084/m9.figshare.26067970 under a Creative Commons license (CC BY 4.0). Rockström, J. et al. A safe operating space for humanity. Richardson, K. et al. Earth beyond six of nine planetary boundaries. In The Global Assessment Report on Biodiversity and Ecosystem Services (eds Brondízio, E. S. et al.) Chapter 2.2 (IPBES Secretariat, 2019); https://doi.org/10.5281/zenodo.5517457 Cadotte, M. W., Carscadden, K. & Mirotchnick, N. Beyond species: functional diversity and the maintenance of ecological processes and services. The influence of functional diversity and composition on ecosystem processes. Martínez-Salinas, A. et al. Bird functional diversity supports pest control services in a Costa Rican coffee farm. Weeks, B. C., Naeem, S., Lasky, J. R. & Tobias, J. A. Diversity and extinction risk are inversely related at a global scale. Wojcik, L. A., Ceulemans, R. & Gaedke, U. Functional diversity buffers the effects of a pulse perturbation on the dynamics of tritrophic food webs. Approaches to defining a planetary boundary for biodiversity. The decline of mammal functional and evolutionary diversity worldwide. Kunming–Montreal Global Biodiversity Framework CBD/COP/15/L.25 (United Nations Environment Programme, 2022); https://www.cbd.int/doc/c/e6d3/cd1d/daf663719a03902a9b116c34/cop-15-l-25-en.pdf Measuring Contributions Towards Terrestrial and Marine Biodiversity Targets (International Union for Conservation of Nature and Natural Resources, 2024); https://www.iucn.org/sites/default/files/2024-02/star-issues-brief-february-2024-update.pdf Etard, A., Pigot, A. L. & Newbold, T. Intensive human land uses negatively affect vertebrate functional diversity. Global hotspots of traded phylogenetic and functional diversity. Bolam, F. et al. How many bird and mammal extinctions has recent conservation action prevented? Bolam, F. et al. Over half of threatened species require targeted recovery actions to avert human-induced extinction. & Tella, J. L. How much does it cost to save a species from extinction? Costs and rewards of conserving the Lear's macaw. Kearney, S. et al. Threat-abatement framework confirms habitat retention and invasive species management are critical to conserve Australia's threatened species. The IUCN Red List of Threatened Species, version 2022-2 (International Union for the Conservation of Nature, 2022); https://www.iucnredlist.org & Bokma, F. The dynamics underlying avian extinction trajectories forecast a wave of extinctions. Andermann, T., Faurby, S., Cooke, R., Silvestro, D. & Antonelli, A. iucn_sim: a new program to simulate future extinctions based on IUCN threat status. Cooke, R. S. C., Eigenbrod, F. & Bates, A. E. Projected losses of global mammal and bird ecological strategies. Erosion of global functional diversity across the tree of life. Sheard, C. et al. Ecological drivers of global gradients in avian dispersal inferred from wing morphology. A. et al. AVONET: morphological, ecological and geographical data for all birds. Zeffer, A., Johansson, L. C. & Marmebro, Å. Functional correlation between habitat use and leg morphology in birds (Aves). Carmona, C. P., de Bello, F., Mason, N. W. H. & Lepš, J. Traits without borders: integrating functional diversity across scales. Mammola, S. & Cardoso, P. Functional diversity metrics using kernel density n-dimensional hypervolumes. Ali, J. R., Blonder, B. W., Pigot, A. L. & Tobias, J. A. Bird extinctions threaten to cause disproportionate reductions of functional diversity and uniqueness. Accelerated modern human-induced species losses: entering the sixth mass extinction. The biodiversity of species and their rates of extinction, distribution, and protection. Purvis, A., Gittleman, J. L., Cowlishaw, G. & Mace, G. M. Predicting extinction risk in declining species. Di Marco, M., Venter, O., Possingham, H. P. & Watson, J. E. M. Changes in human footprint drive changes in species extinction risk. The global loss of avian functional and phylogenetic diversity from anthropogenic extinctions. Population resilience to an extreme drought is influenced by habitat area and fragmentation in the local landscape. Capdevila, P. et al. Life history mediates the trade-offs among different components of demographic resilience. Improving protected area effectiveness through consideration of different human-pressure baselines. Böhning-Gaese, K. & Oberrath, R. Phylogenetic effects on morphological, life-history, behavioural and ecological traits of birds. People, energy and avian species richness. Global patterns of geographic range size in birds. Mair, L. et al. A metric for spatially explicit contributions to science-based species targets. IUCN Red List Categories and Criteria, version 3.1 (IUCN Species Survival Commission, 2001); https://portals.iucn.org/library/sites/library/files/documents/RL-2001-001.pdf The signature of competition in ecomorphological traits across the avian radiation. Protecting important sites for biodiversity contributes to meeting global conservation targets. Leclère, D. et al. Bending the curve of terrestrial biodiversity needs an integrated strategy. Global trends and scenarios for terrestrial biodiversity and ecosystem services from 1900 to 2050. & Revilla, E. Which intrinsic traits predict vulnerability to extinction depends on the actual threatening processes. Rodríguez-Caro, R. C. et al. Anthropogenic impacts on threatened species erode functional diversity in chelonians and crocodilians. Global impacts of climate change on avian functional diversity. Gaston, K. & Blackburn, T. M. Birds, body size and the threat of extinction. Owens, I. P. F. & Bennett, P. M. Ecological basis of extinction risk in birds: habitat loss versus human persecution and introduced predators. Hua, F. et al. Ecological filtering shapes the impacts of agricultural deforestation on biodiversity. Weeks, T. L. et al. Climate-driven variation in dispersal ability predicts responses to forest fragmentation in birds. Campbell-Tennant, D. J. E., Gardner, J. L., Kearney, M. R. & Symonds, M. R. Climate-related spatial and temporal variation in bill morphology over the past century in Australian parrots. & Symonds, M. R. E. Bill size mediates behavioural thermoregulation in birds. Frӧhlich, A., Kotowska, D., Martyka, R. & Symonds, M. R. E. Allometry reveals trade-offs between Bergmann's and Allen's rules, and different avian adaptive strategies for thermoregulation. Subasinghe, K. et al. Spatial variation in avian bill size is associated with temperature extremes in a major radiation of Australian passerines. Pigot, A. L. et al. Macroevolutionary convergence connects morphological form to ecological function in birds. Species factsheet: Sulu hornbill Anthracoceros montani (Birdlife International, 2024); https://datazone.birdlife.org/species/factsheet/sulu-hornbill-anthracoceros-montani Toussaint, A., Pärtel, M. & Carmona, C. P. Contrasting impacts of non-native and threatened species on morphological, life history, and phylogenetic diversity in bird assemblages. Griffith, P., Lang, J. W., Turvey, S. T. & Gumbs, R. Using functional traits to identify conservation priorities for the world's crocodylians. Pimiento, C. et al. Functional diversity of sharks and rays is highly vulnerable and supported by unique species and locations worldwide. Gumbs, R. et al. Conserving avian evolutionary history can effectively safeguard future benefits for people. Handbook of the Birds of the World and BirdLife International. Digital checklist of the birds of the world, version 3 (Birdlife International, 2018); http://datazone.birdlife.org/userfiles/file/Species/Taxonomy/HBW-BirdLife_Checklist_v3_Nov18.zip Chamberlain, S. rredlist: ‘IUCN' Red List Client. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2022). Global biodiversity: indicators of recent declines. Jetz, W., Thomas, G., Joy, J., Hartmann, K. & Mooers, A. O. The global diversity of birds in space and time. Hackett, S. J. et al. A phylogenomic study of birds reveals their evolutionary history. Violle, C. et al. Let the concept of trait be functional! Díaz, S. et al. Functional traits, the phylogeny of function, and ecosystem service vulnerability. Revell, L. J. Size-correction and principal components for interspecific comparative studies. & Goswami, A. Phylogenetic principal components analysis and geometric morphometrics. Revell, L. J. phytools: an R package for phylogenetic comparative biology (and other things). Carmona C. P. TPD: methods for measuring functional diversity based on trait probability density. González-Suárez, M. & Revilla, E. Generalized drivers in the mammalian endangerment process. Purvis, A. Phylogenetic approaches to the study of extinction. Hadfield, J. D. MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. Unified Classification of Direct Threats, version 3.3 (IUCN, 2022); https://www.iucnredlist.org/resources/threat-classification-scheme Garnett, S. T. et al. Metrics of progress in the understanding and management of threats to Australian birds. Stekhoven, D. J. missForest: nonparametric missing value imputation using random forest. Debastiani, V. J., Bastazini, V. A. G. & Pillar, V. D. Using phylogenetic information to impute missing functional trait values in ecological databases. Faraway, J. Inferential methods for linear mixed models: single random effect https://people.bath.ac.uk/jjf23/mixchange/onewayanova.html (2015). & Su, Y.-S. A weakly informative default prior distribution for logistic and other regression models. Comparing parent–offspring regression with frequentist and Bayesian animal models to estimate heritability in wild populations: a simulation study for Gaussian and binary traits. Capellini, I., Baker, J., Allen, W. L., Street, S. E. & Venditti, C. The role of life history traits in mammalian invasion success. Hadfield, J. D. MCMCglmm Course Notes http://cran.nexr.com/web/packages/MCMCglmm/vignettes/CourseNotes.pdf (2017). Hadfield, J. D., Heap, E. A., Bayer, F., Mittell, E. A. & Crouch, N. M. Intraclutch differences in egg characteristics mitigate the consequences of age-related hierarchies in a wild passerine. Wiley, J. postMCMCglmm: average marginal predicted probabilities from Bayesian ordered probit models. & Verones, F. More than half of data deficient species predicted to be threatened by extinction. Butchart, S. H. M. & Bird, J. P. Data Deficient birds on the IUCN Red List: What don't we know and why does it matter? Mooers, A. Ø., Faith, D. P. & Maddison, W. P. Converting endangered species categories to probabilities of extinction for phylogenetic conservation prioritization. de Villemereuil P. Estimation of a biological trait heritability using the animal model and MCMCglmm, version 2 https://devillemereuil.legtux.org/wp-content/uploads/2021/09/tuto_en.pdf (2023). Aarts, S., van den Akker, M. & Winkens, B. The importance of effect sizes. acknowledges PhD studentship funding from the SCENARIO NERC Doctoral Training Partnership grant NE/S007261/1. was funded by the Natural Environment Research Council (grants NE/Z000130/1 and NE/T006579/1). was funded by the Estonian Research Council (grants PRG2142 and MOBERC100) and the European Union (ERC, PLECTRUM, 101126117). Ecology and Evolutionary Biology, School of Biological Sciences, University of Reading, Reading, UK Kerry Stewart, Chris Venditti, Joanna Baker & Manuela González-Suárez Institute of Ecology and Earth Sciences, University of Tartu, Tartu, Estonia Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and designed the analyses. wrote and edited the manuscript. The authors declare no competing interests. Nature Ecology & Evolution thanks Michelle Ward and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Red dotted line indicates elbow after which adding additional phylogenetic principal components would explain little additional variance (n=9873 species). Shown on 2-dimensional plane with respect to a, pPC1 and pPC2, and b, pPC2 and pPC3 with trait loadings (also see Supplementary Table 3). BD = beak depth, BLC = beak length (culmen), BLN = beak length (nares), BW =beak width, HWI = hand-wing index, KD = Kipp's distance, LL = tarsus length, M = body mass, S = first secondary length, TL = tail length, WL = wing length (n=9873 species). All silhouettes from Phylopic. Panel a left to right: Apteryx (Ferran Sayol, CC0 1.0), Mellisuga helenae (Steven Traver, CC0 1.0), Troglodytes hiemalis (Andy Wilson, CC0 1.0), Pteroptochos castaneus (Ferran Sayol, CCO 1.0), Pelecanoides urinatrix (Louis Ranjard, CC BY 3.0), Gallirallus australis (there was no silhouette of Atlantsia rogersi so a silhouette of Gallirallus australis was used instead, T. Michael Keesey, CC BY-SA 3.0), Spheniscus humboldti (Juan Carlos Jerí, CC0 1.0), Larus (Ferran Sayol, CC0 1.0), Diomedeidae (Ferran Sayol, CC0 1.0), Struthio camelus (Darren Naish and T. Michael Keesey, CC BY 3.0), Buceros (Ferran Sayol, CC0 1.0), Leptoptilos javanicus (T. Michael Keesey, CC BY-SA 3.0). Panel b left to right: Apteryx (Ferran Sayol, CC0 1.0), Pelecanus (Ferran Sayol, CC0 1.0), Ramphastidae (FJDegrange, CC0 1.0), Spheniscus humboldti (Juan Carlos Jerí, CC0 1.0), Mellisuga helenae (Steven Traver, CC0 1.0), Buceros (Ferran Sayol, CC0 1.0), Apus apus (Ferran Sayol, CC0 1.0), Struthio camelus (Darren Naish and T. Michael Keesey, CC BY 3.0), Phasianus colchicus (Mattia Menchetti, CC0 1.0), Menura (T. Michael Keesey, CC0 1.0). When species removal is biased with respect to species traits (principal component values) functional richness loss is greater. Shown in one dimension for simplicity, functional richness was calculated in three-dimensional trait space composed of the first three phylogenetic principal components. Loss in density of morphospace was calculated using the baseline extinction scenario, and averted loss in density of morphospace was calculated using the complete abatement scenario. Grey shows areas where no functional diversity loss was projected, or where no functional diversity loss was avoided under complete abatement (only three pixels in panel a and two pixels in panel b). Analyses based on 9873 species (of which 2087 species currently listed as Near Threatened or in threatened categories were modelled and could have reduced extinction risk in the abatement scenarios). 1000 iterations were run for each extinction scenario. All silhouettes from Phylopic. Panel a left to right: Apteryx (Ferran Sayol, CC0 1.0), Mellisuga helenae (Steven Traver, CC0 1.0), Troglodytes hiemalis (Andy Wilson, CC0 1.0), Pteroptochos castaneus (Ferran Sayol, CCO 1.0), Atlantisia rogersi (there was no silhouette of Atlantsia rogersi so a silhouette of Gallirallus australis was used instead, T. Michael Keesey and HuttyMcphoo, CC BY-SA 3.0), Pelecanoides urinatrix (Louis Ranjard, CC BY 3.0), Spheniscus humboldti (Juan Carlos Jerí, CC0 1.0), Larus (Ferran Sayol, CC0 1.0), Diomedeidae (Ferran Sayol, CC0 1.0), Struthio camelus (Darren Naish and T. Michael Keesey, CC BY 3.0), Buceros (Ferran Sayol, CC0 1.0), Leptoptilos javanicus (T. Michael Keesey and Vaibhavcho, CC BY-SA 3.0). Panel b left to right: Apteryx (Ferran Sayol, CC0 1.0), Pelecanus (Ferran Sayol, CC0 1.0), Ramphastidae (Federico Degrange, CC0 1.0), Spheniscus humboldti (Juan Carlos Jerí, CC0 1.0), Mellisuga helenae (Steven Traver, CC0 1.0), Buceros (Ferran Sayol, CC0 1.0), Apus apus (Ferran Sayol, CC0 1.0), Phasianus colchicus (Mattia Menchetti, CC0 1.0), Menura (T. Michael Keesey, CC0 1.0), Struthio camelus (Darren Naish and T. Michael Keesey, CC BY 3.0). a, Number of species extinctions avoided under driver-specific complete abatement against functional richness loss avoided (% of functional richness of full assemblage) as described by a linear mixed effects model including number of species extinctions avoided and driver of extinction as fixed effects, and iteration number as a random effect. Pollution was not included as it made a negligible contribution to functional richness loss (see Extended Data Table 3) (n = 5000, 1000 iterations for each extinction scenario). Functional uniqueness describes the proportion of the community probability distribution that was composed of the species probability distribution. Proportions were summed across cells in which the species probability distribution was greater than 0, with a weight proportional to the probability of species occurrence in that cell, indicated by the height of the species probability distribution. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Threat reduction must be coupled with targeted recovery programmes to conserve global bird diversity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41589-025-01936-x'>A small-molecule VHL molecular glue degrader for cysteine dioxygenase 1</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 09:58:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Chemical Biology (2025)Cite this article The von Hippel–Lindau tumor suppressor gene product (pVHL) is an E3 ligase substrate receptor that binds proline-hydroxylated hypoxia-inducible factor HIF1α, leading to its ubiquitin-dependent degradation. By using protein arrays, we identified a small molecule that binds the HIF1α-binding pocket on pVHL and functions as a molecular glue degrader of the neosubstrate cysteine dioxygenase (CDO1) by recruiting it into the VHL–Cullin–RING E3 ligase complex and leading to its selective degradation. The CDO1-binding region involved in VHL recruitment was characterized through a combination of mutagenesis and protein–protein docking coupled with molecular-dynamics-based solvation analysis. The X-ray structure of the ternary complexes of VHL, CDO1 and degrader molecules confirms the binding region prediction and provides atomic insights into key molecular glue interactions. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data generated or analyzed during this study are included in this published article and its Supplementary Information. The atomic models were deposited to the PDB under accession codes 8VLB and 8VL9. Source data are provided with this paper. Maxwell, P. H. et al. The tumour suppressor protein VHL targets hypoxia-inducible factors for oxygen-dependent proteolysis. Buckley, D. L. et al. Targeting the von Hippel–Lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1α interaction. Van Molle, I. et al. Dissecting fragment-based lead discovery at the von Hippel–Lindau protein:hypoxia inducible factor 1α protein–protein interface. Buckley, D. L. et al. Small-molecule inhibitors of the interaction between the E3 ligase VHL and HIF1α. Sakamoto, K. M. et al. Protacs: chimeric molecules that target proteins to the Skp1–Cullin–F box complex for ubiquitination and degradation. Bondeson, D. P. et al. Catalytic in vivo protein knockdown by small-molecule PROTACs. Diehl, C. J. & Ciulli, A. Discovery of small molecule ligands for the von Hippel–Lindau (VHL) E3 ligase and their use as inhibitors and PROTAC degraders. Zengerle, M., Chan, K. H. & Ciulli, A. Selective small molecule induced degradation of the BET bromodomain protein BRD4. ACS Chem. Bond, M. J., Chu, L., Nalawansha, D. A., Li, K. & Crews, C. M. Targeted degradation of oncogenic KRASG12C by VHL-recruiting PROTACs. ACS Cent. Hallin, J. et al. The KRASG12C inhibitor MRTX849 provides insight toward therapeutic susceptibility of KRAS-mutant cancers in mouse models and patients. Cancer Discov. Casement, R., Bond, A., Craigon, C. & Ciulli, A. Mechanistic and structural features of PROTAC ternary complexes. Methods Mol. Jin, F. et al. A pooling–deconvolution strategy for biological network elucidation. Frost, J. et al. Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition. Buckley, D. L. et al. HaloPROTACS: use of small molecule PROTACs to induce degradation of HaloTag fusion proteins. ACS Chem. Galdeano, C. et al. Structure-guided design and optimization of small molecules targeting the protein–protein interaction between the von Hippel–Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities. Gnarra, J. R. et al. Mutations of the VHL tumour suppressor gene in renal carcinoma. Iliopoulos, O., Kibel, A., Gray, S. & Kaelin, W. G. Jr. Tumour suppression by the human von Hippel–Lindau gene product. Soucy, T. A. et al. An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer. Case, D. A. et al. AMBER 2018. University of California, San Francisco (2018). Velez-Vega, C., McKay, D. J., Aravamuthan, V., Pearlstein, R. & Duca, J. S. Time-averaged distributions of solute and solvent motions: exploring proton wires of GFP and PfM2DH. Velez-Vega, C. et al. Estimation of solvation entropy and enthalpy via analysis of water oxygen–hydrogen correlations. Ye, S. et al. An insight into the mechanism of human cysteine dioxygenase. Key roles of the thioether-bonded tyrosine–cysteine cofactor. Bogan, A. & Thorn, K. S. Anatomy of hot spots in protein interfaces. Stipanuk, M. H., Hirschberger, L. L., Londono, M. P., Cresenzi, C. L. & Yu, A. F. The ubiquitin–proteasome system is responsible for cysteine-responsive regulation of cysteine dioxygenase concentration in liver. Lu, G. et al. The myeloma drug lenalidomide promotes the cereblon-dependent destruction of Ikaros proteins. Kronke, J. et al. Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells. Article PubMed Matyskiela, M. E. et al. A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase. Bonazzi, S. et al. Discovery and characterization of a selective IKZF2 glue degrader for cancer immunotherapy. Cell Chem. Petzold, G., Fischer, E. S. & Thoma, N. H. Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase. Bussiere, D. E. et al. Structural basis of indisulam-mediated RBM39 recruitment to DCAF15 E3 ligase complex. Du, X. et al. Structural basis and kinetic pathway of RBM39 recruitment to DCAF15 by a sulfonamide molecular glue E7820. Faust, T. B. et al. Structural complementarity facilitates E7820-mediated degradation of RBM39 by DCAF15. Ting, T. C. et al. Aryl sulfonamides degrade RBM39 and RBM23 by recruitment to CRL4–DCAF15. Tong, J. et al. Hypoxia-induced long non-coding RNA DARS-AS1 regulates RBM39 stability to promote myeloma malignancy. Shergalis, A. G. et al. CRISPR screen reveals BRD2/4 molecular glue-like degrader via recruitment of DCAF16. ACS Chem. Lv, L. et al. Discovery of a molecular glue promoting CDK12–DDB1 interaction to trigger cyclin K degradation. Stipanuk, M. H., Ueki, I., Dominy, J. E. Jr., Simmons, C. R. & Hirschberger, L. L. Cysteine dioxygenase: a robust system for regulation of cellular cysteine levels. Amino Acids 37, 55–63 (2009). Parsons, R. B., Waring, R. H., Ramsden, D. B. & Williams, A. C. Toxicity of cysteine and cysteine sulphinic acid to human neuronal cell-lines. Yamashita, K., Hosoda, K., Nishizawa, N., Katoh, H. & Watanabe, M. Epigenetic biomarkers of promoter DNA methylation in the new era of cancer treatment. Cancer Sci. Mason, J. W. et al. DNA-encoded library-enabled discovery of proximity-inducing small molecules. Liu, S. et al. Rational screening for cooperativity in small-molecule inducers of protein–protein associations. Proudfoot, A., Frank, A. O., Frommlet, A. Selective methyl labeling of proteins: enabling structural and mechanistic studies as well as drug discovery applications by solution-state NMR. Methods Enzymol. Schanda, P., Kupce, E. & Brutscher, B. SOFAST-HMQC experiments for recording two-dimensional heteronuclear correlation spectra of proteins within a few seconds. Zaro, B. W. et al. Proteomic analysis of young and old mouse hematopoietic stem cells and their progenitors reveals post-transcriptional regulation in stem cells. van Bergen, W., Heck, A. J. R. & Baggelaar, M. P. Recent advancements in mass spectrometry-based tools to investigate newly synthesized proteins. Article PubMed Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. Vonrhein, C. et al. Data processing and analysis with the autoPROC toolbox. Acta Crystallogr. D Biol. Acta Crystallogr. D Biol. Evans, P. R. & Murshudov, G. N. How good are my data and what is the resolution? Acta Crystallogr. McCoy, A. J. et al. Phaser crystallographic software. Stebbins, C. E., Kaelin, W. G. Jr. & Pavletich, N. P. Structure of the VHL–ElonginC–ElonginB complex: implications for VHL tumor suppressor function. Li, J. et al. Cleavage of a carbon–fluorine bond by an engineered cysteine dioxygenase. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr. Chaudhury, S. et al. Benchmarking and analysis of protein docking performance in Rosetta v3.2. Gray, J. J. et al. Protein–protein docking with simultaneous optimization of rigid-body displacement and side-chain conformations. Download references We would like to thank M. Salcius for his expertise in protein microarray methodology, which influenced the design of protein array assays that led to the identification of CDO1. Discovery Sciences, Novartis Biomedical Research, Cambridge, MA, USA Antonin Tutter, Dennis Buckley, Veronique Darsigny, Rohan Beckwith, Jonathan Solomon, Pasupuleti Rao, Lei Xu, Aleem Fazal, Jennifer S. Cobb, Amanda Hachey, Jennifer Tullai & Gregory A. Michaud Global Discovery Chemistry, Novartis Biomedical Research, Cambridge, MA, USA Andrei A. Golosov & Daniel J. J. McKay Global Discovery Chemistry, Novartis Biomedical Research, Emeryville, CA, USA Xiaolei Ma & Wei Shu Discovery Sciences, Novartis Biomedical Research, Emeryville, CA, USA Dustin Dovala & Charles Wartchow Global Discovery Chemistry, Novartis Biomedical Research, Basel, Switzerland Andreas Lingel Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar A.T. performed the protein array experiments that led to the identification of CDO1, expressed and purified the wild-type, Q99N mutant, species-specific and chimeric CDO1 proteins and performed the TR-FRET, NanoBiT, HiBiT and CTG assays. designed and synthesized the VHL ligands. solved the X-ray ternary complex structures. A.G. performed the protein–protein docking analysis and DM performed the WATMD analysis for ternary complex interface predictions which were used in chimera designs for interface mapping. RB proposed the idea of using protein arrays to find molecular glue targets for VHL. conducted the compound 4 experiment in 786-0CDO1+/VHL− and 786-0CDO1+/VHL+ cell lines. ran the MLN4924 and MG132 experiment in 786-0CDO1+/VHL− and 786-0CDO1+/VHL+ cell lines. generated the 786-0CDO1+/VHL− and 786-0CDO1+/VHL+ cell lines. performed the NMR experiments. performed the SPR assays to measure compound affinities with VHL. J.C. and A.H. conducted the proteomics experiments. expressed and purified biotinylated and unbiotinylated pVHL complexes. performed the VHL western blots and CDO1 degradation time-course experiment in HUH-7 and VCAP cells. performed the SPR assays for ternary complex formation. Correspondence to Gregory A. Michaud. All authors are or were employees of Novartis Biomedical Research and may have held stock in Novartis when the research for the manuscript was performed. Nature Chemical Biology thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–11, Table 1 and Note (synthetic procedures). Numerical source data for Supplementary Figs. Tabs are named according to the supplementary figure panels. Numerical source data. Unprocessed western blot. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Tutter, A., Buckley, D., Golosov, A.A. et al. A small-molecule VHL molecular glue degrader for cysteine dioxygenase 1. Nat Chem Biol  (2025). Download citation Accepted: 11 May 2025 DOI: https://doi.org/10.1038/s41589-025-01936-x Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Chemical Biology (Nat Chem Biol) ISSN 1552-4469 (online) ISSN 1552-4450 (print) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/06/250623233320.htm'>No kings buried here: DNA unravels the myth of incestuous elites in ancient Ireland</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-24 04:32:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers examined the evidence from burials of individuals at the Newgrange prehistoric monument in Ireland -- a large circular mound containing a chamber where cremated and unburnt human remains were placed, the discovery of which led to speculation that those buried there must have been from an elite class or a royal family. A research team, including the University of York and University College Dublin, however, have now shown that there is no corroborating evidence in nearby settlements, dietary practices, or trade, that an elite existed either economically or socially. "There are no large settlement systems or trade mechanisms, and we also don't see production of craft on the scale that we see in other ancient societies such as in Ancient Egypt, where incest was thought to be practiced by the ruling elite. "We can only begin to understand these monuments and tombs if we examine the social lives or the communities that built and use them, and when we start to do this, this idea of a 'social elite' or a form of royalty starts to appear less likely." Older than Stonehenge and the Pyramids of Giza, Newgrange is believed to have been built by a farming community that prospered in the Boyne Valley, County Meath, some 5,000 years ago. Newgrange was rediscovered in AD 1699 and its interior had been heavily disturbed prior to its modern excavation in the 1960s, and so researchers question whether this was even the original burial place of the skull fragment, referred to as NG10, dated to 3340 -- 3020 BC. The genetic clustering in passage tombs, such as at Newgrange, typically reflects very distant biological relationships -- like second cousins or great-great-great-grandparents -- rather than close familial ties. To establish if the tombs were reserved for an elite, researchers say they would expect much closer genetic ties than they have seen at Newgrange. It is more likely that these tombs were places that reflect family, work, and other social relationships, rather than the mark of an elite family group. Associate Professor Jessica Smyth, from University College Dublin, said: "People were definitely being selected for burial in passage tombs -- the whole community does not end up in these monuments. However, we don't know the reasons behind this selection, and why they were thought to be special. "Unlike today, bodies don't tend to be buried 'whole' or 'intact' in this time period. Before they end up in megalithic monuments, bodies are broken down, sometimes cremated and even circulated around their communities." As so many of the dead found in these passage tombs were broken down and mixed with cremated remains, and possibly moved around to different places in the landscape, it is not clear the people who placed NG10 fragment in Newgrange knew who they were in life or who the parents of this individual were, casting further doubt on how 'special' the individual might have been. Professor Penny Bickle said: "It is by no means clear that the monument was the first burial site of NG10 and the tomb grew in stages, so tracing who this individual was is a very difficult task indeed. "There are still many questions to solve here, but building this picture means looking at the monument together with the society that was built up around it, and from this, we get a step closer to understanding a community that was much more inclusive and equal than previously thought." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            