
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-17</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a66107800/roman-tombstone-inscription/'>A Newly Discovered Tomb Inscription Rewrites What We Know About the Roman Military</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The deceased Roman citizen was a soldier in his nation's army, but his role wasn't what researchers expected. Researchers found more than aged vintage in a Spanish wine cellar, instead discovering the tombstone of a Roman cavalry rider that rewrites what we know about military movements during the Julio-Claudian dynasty from the first century C.E. and may provide a look at Roman citizens included in the empire's Spanish-based auxiliary military. In a new study published in Gladius, a pair of researchers from the University of Salamanca and the University of Leon translated a quartzite funerary stele inscription written in Latin, which they discovered being reused as construction material in Fuente Encalada, Spain. Placing a Roman citizen within an auxiliary unit upends common thought about the makeup of the Roman army: legions were made up of citizens, while auxiliary units only comprised non-citizens. Prior research had shown that Roman citizens were actually members of auxiliary units more commonly than had been believed, but this example discovered in Spain indicates that was the case much earlier than experts had realized. The inscription also allows researchers to better understand Roman military movements at that time. “Detailed analysis allows us to approach when this unit arrived in Iberian territories and where it might have been stationed with greater accuracy.” Even though scholars know legions were stationed on Iberian soil, understanding the auxiliary troop dispersions is less understood. The authors believe this find could be a previously unheard-of example of the auxiliary troops sharing a camp with the main legion, though they also state it could simply have been stationed close by. to help establish a presence for the Romans. This inscription is a major piece of evidence in rewiring our understanding of Roman military history, so the discovery is no small feat. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Treasure May Be Hidden Amongst Terracotta Army A College Student Broke the Laws of Thermodynamics Ancient Fossils Found Behind a New Jersey Lowe's Information Could Be the Fifth State of Matter</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09522-w'>CRISPR activation for SCN2A -related neurodevelopmental disorders</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 16:17:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Most neurodevelopmental disorders with single gene diagnoses act via haploinsufficiency, in which only one of the two gene copies is functional1. SCN2A haploinsufficiency is one of the most frequent causes of neurodevelopmental disorder, often presenting with autism spectrum disorder, intellectual disability and, in a subset of children, refractory epilepsy2. Here, using SCN2A haploinsufficiency as a proof-of-concept, we show that upregulation of the existing functional gene copy through CRISPR activation (CRISPRa) can rescue neurological-associated phenotypes in Scn2a haploinsufficient mice. We first show that restoring Scn2a expression in adolescent heterozygous Scn2a conditional knock-in mice rescues electrophysiological deficits associated with Scn2a haploinsufficiency (Scn2a+/−). Next, using an adeno-associated virus CRISPRa-based treatment in adolescent mice, we show that we can correct intrinsic and synaptic deficits in neocortical pyramidal cells, a major cell type that contributes to neurodevelopmental disorders and seizure aetiology in SCN2A haploinsufficiency. Furthermore, we find that systemic delivery of CRISPRa protects Scn2a+/− mice against chemoconvulsant-induced seizures. Finally, we also show that adeno-associated virus CRISPRa treatment rescues excitability in SCN2A haploinsufficient human stem-cell-derived neurons. Our results showcase the potential of this therapeutic approach to rescue SCN2A haploinsufficiency and demonstrates that rescue even at adolescent stages can ameliorate neurodevelopmental phenotypes. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All RNA-seq data are available at the NCBI Gene Expression Omnibus as Bioproject GSE302897. Requests for materials can be directed to N.A. (CRT-based approaches) or K.J.B. Source data are provided with this paper. No custom code was generated for this work. Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Sanders, S. J. et al. Progress in understanding and treating SCN2A-mediated disorders. Maenner, M. J. et al. Prevalence of autism spectrum disorder among children aged 8 years – Autism and Developmental Disabilities Monitoring Network, 11 sites, United States, 2016. Kaplanis, J. et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Fu, J. M. et al. Rare coding variation provides insight into the genetic architecture and phenotypic context of autism. Epi25 Collaborative Exome sequencing of 20,979 individuals with epilepsy reveals shared and distinct ultra-rare genetic risk across disorder subtypes. Wang, D., Tai, P. W. L. & Gao, G. Adeno-associated virus vector as a platform for gene therapy delivery. Wu, Z., Yang, H. & Colosi, P. Effect of genome size on AAV vector packaging. Howe, K. L. et al. Ensembl 2021. Nucleic Acids Res. Matharu, N. & Ahituv, N. Modulating gene regulation to treat genetic disorders. Werling, D. M. et al. Whole-genome and RNA sequencing reveal variation and transcriptomic coordination in the developing human prefrontal cortex. Jenkins, P. M. & Bender, K. J. Axon initial segment structure and function in health and disease. Spratt, P. W. E. et al. The autism-associated gene Scn2a contributes to dendritic excitability and synaptic function in the prefrontal cortex. Nelson, A. D. et al. Physical and functional convergence of the autism risk genes Scn2a and Ank2 in neocortical pyramidal cell dendrites. Lu, C. et al. Overexpression of NEUROG2 and NEUROG1 in human embryonic stem cells produces a network of excitatory and inhibitory neurons. Li, T. et al. Action potential initiation in neocortical inhibitory interneurons. Matharu, N. et al. CRISPR-mediated activation of a promoter or enhancer rescues obesity caused by haploinsufficiency. Grimm, D. et al. In vitro and in vivo gene therapy vector evolution via multispecies interbreeding and retargeting of adeno-associated viruses. Bae, S., Park, J. & Kim, J.-S. Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Wang, W. et al. PTPN14 is required for the density-dependent control of YAP1. Correia, J. C. et al. Zfp697 is an RNA-binding protein that regulates skeletal muscle inflammation and remodeling. Busse, D. C. et al. Interferon-induced protein 44 and interferon-induced protein 44-like restrict replication of respiratory syncytial virus. Baum, M. L. et al. CSMD1 regulates brain complement activity and circuit development. Spratt, P. W. E. et al. Paradoxical hyperexcitability from NaV1.2 sodium channel loss in neocortical pyramidal cells. Chung, J. H., Larsen, A. R., Chen, E. & Bunz, F. A PTCH1 homolog transcriptionally activated by p53 suppresses hedgehog signaling. Liang, L. et al. Developmental dynamics of voltage-gated sodium channel isoform expression in the human and mouse brain. Yuan, Y. et al. Antisense oligonucleotides restore excitability, GABA signalling and sodium current density in a Dravet syndrome model. Zhang, J. et al. Severe deficiency of the voltage-gated sodium channel NaV1.2 elevates neuronal excitability in adult mice. Miyamoto, H. et al. Impaired cortico-striatal excitatory transmission triggers epilepsy. Reynolds, C., King, M. D. & Gorman, K. M. The phenotypic spectrum of SCN2A-related epilepsy. Colasante, G. et al. dCas9-based Scn1a gene activation restores inhibitory interneuron excitability and attenuates seizures in Dravet syndrome mice. Colasante, G. et al. In vivo CRISPRa decreases seizures and rescues cognitive deficits in a rodent model of epilepsy. Yamagata, T. et al. CRISPR/dCas9-based Scn1a gene activation in inhibitory neurons ameliorates epileptic and behavioral phenotypes of Dravet syndrome model mice. Chang, H.-C. et al. rAAV-CRISPRa therapy corrects Rai1 haploinsufficiency and rescues selective disease features in Smith–Magenis syndrome mice. Wang, G. et al. Multiplexed activation of endogenous genes by CRISPRa elicits potent antitumor immunity. Liao, H.-K. et al. In vivo target gene activation via CRISPR/Cas9-mediated trans-epigenetic modulation. Böhm, S. et al. A gene therapy for inherited blindness using dCas9-VPR-mediated transcriptional activation. Kemaladewi, D. U. et al. A mutation-independent approach for muscular dystrophy via upregulation of a modifier gene. Mich, J. K. et al. Interneuron-specific dual-AAV SCN1A gene replacement corrects epileptic phenotypes in mouse models of Dravet syndrome. Waszkielewicz, A. M. et al. Ion channels as drug targets in central nervous system disorders. Johnson, J. P. et al. NBI-921352, a first-in-class, NaV1.6 selective, sodium channel inhibitor that prevents seizures in Scn8a gain-of-function mice, and wild-type mice and rats. Ferdosi, S. R. et al. Multifunctional CRISPR-Cas9 with engineered immunosilenced human T cell epitopes. & Merkel, O. M. Immunogenicity of Cas9 protein. Gaj, T., Sirk, S. J., Shui, S.-L. & Liu, J. Genome-editing technologies: principles and applications. Cold Spring Harb. Levy, G. & Barak, B. Postnatal therapeutic approaches in genetic neurodevelopmental disorders. Markati, T., Duis, J. & Servais, L. Therapies in preclinical and clinical development for Angelman syndrome. Silva-Santos, S. et al. Ube3a reinstatement identifies distinct developmental windows in a murine Angelman syndrome model. Wolter, J. M. et al. Cas9 gene therapy for Angelman syndrome traps Ube3a-ATS long non-coding RNA. Berg, A. T. et al. Expanded clinical phenotype spectrum correlates with variant function in SCN2A-related disorders. Eaton, M. et al. Generation and basic characterization of a gene-trap knockout mouse model of Scn2a with a substantial reduction of voltage-gated sodium channel Nav 1.2 expression. Tatsukawa, T. et al. Scn2a haploinsufficient mice display a spectrum of phenotypes affecting anxiety, sociability, memory flexibility and ampakine CX516 rescues their hyperactivity. Shin, W. et al. Scn2a haploinsufficiency in mice suppresses hippocampal neuronal excitability, excitatory synaptic drive, and long-term potentiation, and spatial learning and memory. & Mantegazza, M. NaV1.2 haploinsufficiency in Scn2a knock-out mice causes an autistic-like phenotype attenuated with age. Schamiloglu, S., Wu, H., Zhou, M., Kwan, A. C. & Bender, K. J. Dynamic foraging behavior performance is not affected by Scn2a haploinsufficiency. Goertsen, D. et al. AAV capsid variants with brain-wide transgene expression and decreased liver targeting after intravenous delivery in mouse and marmoset. Keiser, M. S. et al. Toxicity after AAV delivery of RNAi expression constructs into nonhuman primate brain. Hordeaux, J. et al. The GPI-linked protein LY6A drives AAV-PHP.B transport across the blood-brain barrier. Yao, Y. et al. Variants of the adeno-associated virus serotype 9 with enhanced penetration of the blood-brain barrier in rodents and primates. Blesa, J. et al. BBB opening with focused ultrasound in nonhuman primates and Parkinson's disease patients: targeted AAV vector delivery and PET imaging. Ewels, P. A. et al. The nf-core framework for community-curated bioinformatics pipelines. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Fang, Z., Liu, X. & Peltz, G. GSEApy: a comprehensive package for performing gene set enrichment analysis in Python. Planells-Cases, R. et al. Neuronal death and perinatal lethality in voltage-gated sodium channel αII-deficient mice. Van Erum, J., Van Dam, D. & De Deyn, P. P. PTZ-induced seizures in mice require a revised Racine scale. Markram, H. et al. Reconstruction and simulation of neocortical microcircuitry. Ben-Shalom, R. et al. Opposing effects on NaV1.2 function underlie differences between SCN2A variants observed in individuals with autism spectrum disorder or infantile seizures. B., Dixit, M., Zepeda, J. C., Grueter, B. & Dugan, L. L. Robust expression of functional NMDA receptors in human induced pluripotent stem cell-derived neuronal cultures using an accelerated protocol. We thank the Bender and Ahituv laboratory members for comments and discussions on this manuscript, and members of the FamilieSCN2A Foundation who provided the core motivation and inspiration for this work. We acknowledge the UCSF Parnassus Flow CoLab (RRID:SCR_018206) supported in part by NIH grant no. P30 DK063720 and by the NIH S10 Instrumentation Grant no. This work was supported by grants from SFARI (grant no. ), the Broad Institute Target Practice Initiative (K.J.B. ), the Autism Science Foundation (S.T. ), the Weill Neurohub Investigator Program (K.J.B., N.A. ), an NSERC Predoctoral Fellowship (P.W.E.S. ), the Ford Foundation Dissertation Fellowship (S.S.H. ), the Weill Foundation Graduate Student Fellowship (S.S.H. ), an NSF Graduate Fellowship (S.T.) and the NIH (grant no. ), the Medical Research Council Centre of Research Excellence in Therapeutic Genomics (MR/Z504725/1: S.J.S., N.A. ), the UCSF Bakar Aging Research Institute postdoctoral fellowship (C.A.) and the Bettencourt Schueller foundation (C.A. These authors contributed equally: Serena Tamura, Andrew D. Nelson, Perry W. E. Spratt Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA, USA Serena Tamura, Xujia Zhou, Zizheng Li, Coline Arnould, Vladyslav Barskyi, Beniamin Krupkin, Kiana Young, Jingjing Zhao, Navneet Matharu & Nadav Ahituv Institute for Human Genetics, University of California San Francisco, San Francisco, CA, USA Serena Tamura, Xujia Zhou, Zizheng Li, Coline Arnould, Vladyslav Barskyi, Beniamin Krupkin, Kiana Young, Jingjing Zhao, Navneet Matharu & Nadav Ahituv Weill Institute for Neurosciences, University of California San Francisco, San Francisco, CA, USA Andrew D. Nelson, Perry W. E. Spratt, Elizabeth C. Hamada, Henry Kyoung, Atehsa Sahagun, Caroline M. Keeshen, Roy Ben-Shalom, Sunrae E. Taloma, Selin Schamiloglu, Ying C. Li, Stephan J. Sanders & Kevin J. Bender Department of Neurology, University of California San Francisco, San Francisco, CA, USA Andrew D. Nelson, Perry W. E. Spratt, Elizabeth C. Hamada, Henry Kyoung, Stephanie S. Holden, Atehsa Sahagun, Caroline M. Keeshen, Roy Ben-Shalom, Sunrae E. Taloma, Selin Schamiloglu, Ying C. Li, Jeanne T. Paz & Kevin J. Bender Gladstone Institute of Neurological Disease, University of California San Francisco, San Francisco, CA, USA Stephanie S. Holden & Jeanne T. Paz Stanley Center for Psychiatric Research, Broad Institute, Cambridge, MA, USA Congyi Lu & Jen Q. Pan Department of Pharmacology, University of Michigan Medical School, Ann Arbor, MI, USA Lia Min & Paul M. Jenkins Department of Psychiatry, University of Michigan Medical School, Ann Arbor, MI, USA Department of Psychiatry, University of California San Francisco, San Francisco, CA, USA Institute of Developmental and Regenerative Medicine, Department of Paediatrics, University of Oxford, Oxford, UK Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Writing—review and editing: all authors. Correspondence to Nadav Ahituv or Kevin J. Bender. N.M. is the cofounder and former board member and CSO of Regel Therapeutics, N.A. is the cofounder of Regel Therapeutics and both N.A. are on the scientific advisory board of Regel Therapeutics. is a Program Director at Regel Therapeutics. are the inventors on patent ‘Gene therapy for haploinsufficiency' WO2018148256A9. receive funding from BioMarin Pharmaceutical Incorporated. The other authors declare no competing interests. Nature thanks Elvir Becirovic, James Trimmer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. Coronal brain sections from P60 Scn2a+/KI mouse (Cre-) immunostained with anti-GFP and anti-parvalbumin (PV). b. Zoom of area highlighted by dashed box in panel a, with GFP and Parvalbumin channels separated then merged at bottom. Parv+ somata are circled in the GFP panel. c. Further zoom of region in layer 5b in panel b. Parv+ somata are circled as in panel c. d. Quantification of mean fluorescence intensify of GFP in PV-negative cells, PV-positive cells, and neuropil (area without somata as a measure of background fluorescence). Data are from 2 mice. Circles are mean GFP intensity values in ROIs of Parv- somata, Parv+ somata, or neighboring neuropil in single optical sections. Box plots are medians, quartiles, and 100% tails. n = 67 Parv- and 37 Parv+ cells analyzed; Parv- vs. Parv + : ****p < 0.0001. Parv- vs. neuropil: ****p < 0.0001. Holm-Šídák multiple comparisons test. Fold change of Scn2a expression in Neuro-2a cells transfected with plasmids containing sgRNAs targeting the promoter of mouse Scn2a compared to a no-sgRNA VP64 control. Blue bars represent plasmids with largest increase in Scn2a expression. Circles are replicates, overlaid on mean ± SD. Fold change of Scn2a transduced with rAAV-DJ virus in Neuro-2a cells. Circles are replicates, overlaid on mean ± SD. c. Total animal weight at time of 8 mg/kg 4-AP administration (animals aged 69–119 days). Circles are animals. Box plots are medians, quartiles and 100% tails. d. RT-qPCR analysis of dCas9 and mCherry mRNA within the mPFC of tail vein injected Scn2a+/+ + CRISPRa (light gray) and Scn2a+/− + CRISPRa (purple) versus uninjected controls Scn2a+/+ (dark gray) or Scn2a+/− (cyan). Injected animals with at least a 10-fold increase in expression levels of both dCas9 and mCherry to the average Scn2a+/+ uninjected controls were included in EEG datasets in Fig. a. RNA-seq expression levels of Scn2a TAD-domain genes from Scn2a-rAAV-CRISPRa treated Neuro-2a cells compared to VP64-only. Circles are individual replicates, box plots are medians, quartiles and 100% tails. p-values noted above data. b. RNA-seq expression levels of sodium channels from Scn2a-rAAV-CRISPRa treated Neuro-2a cells compared to VP64-only. Circles are individual replicates, box plots are medians, quartiles and 100% tails. p-values noted above data. c. Volcano plot representing Log2 fold-change in expression levels for each gene. P-values noted above each comparison. Grey dots represent no significant DEGs for CRISPRa, purple dots signify Scn2a and nearby genes, and orange dots denote upregulated genes. d. Gene ontology (GO) analysis showing enrichment of molecular functions. e. Gene ontology (GO) analysis showing enrichment of biological processes. f. Analysis of sgRNA sequence off-targeting using Cas-OFFinder. a. AP threshold from P57-85 Scn2a-rAAV-empty vector Scn2a+/+ (light gray) and Scn2a+/− (magenta) neurons, Scn2a-rAAV-CRISPRa treated Scn2a+/+ (dark gray) and Scn2a+/− (purple) neurons, and Scn2a+/KI Cre- (green) and Scn2a+/KI Cre+ (gray) neurons. Threshold of the first AP evoked by a near-rheobase current. No significant differences. Holm-Šídák multiple comparisons test. Circles are individual cells, box plots are medians, quartiles and 100% tails. Input resistance (MΩ). No significant differences. Holm-Šídák multiple comparisons test. Circles are individual cells, box plots are medians, quartiles and 100% tails. c. Rheobase current (pA) to generate first spike. No significant differences. Holm-Šídák multiple comparisons test. Circles are individual cells, box plots are medians, quartiles and 100% tails. d. APs per 300 ms stimulation epoch for each current amplitude. Left: Quantification of firing rate slope of data on Right. No significant differences. Holm-Šídák multiple comparisons test. Circles are individual cells, box plots are medians, quartiles and 100% tails.Right: Number of APs versus current amplitude injected. Circles and bars are mean ± SEM at each stimulus intensity. a. RNA-seq expression levels of Scn2a TAD-domain genes from Scn2a-rAAV-CRISPRa treated Neuro-2a cells compared to VP64-only. Circles are individual mice, box plots are medians, quartiles and 100% tails. b. RNA-seq expression levels of sodium channels from Scn2a-rAAV-CRISPRa treated Neuro-2a cells compared to VP64-only. Circles are individual mice, box plots are medians, quartiles and 100% tails. c. Volcano plot representing Log2 fold-change in expression levels for each gene for WT vs Scn2a+/− “Het”. P-values noted above each comparison. Grey dots represent no significant DEGs for CRISPRa, purple dots signify Scn2a and nearby genes, and orange dots denote upregulated genes. d. Same as c, but for CRISPRa-treated Scn2a+/− “CRISPRa” vs WT. e. Same as c, but for CRISPRa-treated Scn2a+/− “CRISPRa” vs Scn2a+/− “Het”. f. Analysis of sgRNA sequence off-targeting using Cas-OFFinder. g. Data representation of NeuN-positive neuronal nuclei isolated from cortical tissue for FACS sorting. Representative FACS plots visualized with FlowJo V10 with percentage of parent gates for each population. Singlets were selected based on FSC-H versus FSC-A. h. From singlets, DAPI-positive events were gated to identify nuclei. i. NeuN-positive neuronal nuclei were selected using an anti-NeuN antibody conjugated to Alexa Fluor 488. Peak AP dV/dt (top) at 30-, 56-, 107-, and 210-days following tail-vein injection of Scn2a-rAAV-CRISPRa-PHP.eB in Scn2a+/−mice (purple). Circles represent individual neurons. Lines are linear regression. qPCR of mCherry (middle) or dCas9 (bottom) mRNA from neocortical samples collected from tail-vein or retro-orbital Scn2a-rAAV-CRISPRa-PHP.eB injected Scn2a+/+ (gray) or Scn2a+/− mice (purple) across time. Circles represent single mice. Lines are linear regression. Example PFC EEGs from animals receiving 8 and 15 mg/kg 4-AP. 4-AP administered at onset of all recordings. Dashed box denotes a tonic clonic seizure and subsequent mortality occurring with 15 mg/kg dosing in Scn2a+/− animal. EEG data within boxed area is shown at higher resolution on right. Survival curves for all WT and Scn2a+/− animals for 6, 8 and 15 mg/kg 4-AP. WT: black, Scn2a+/−: cyan. Fold change of SCN2A expression in SH-SY5Y cells transfected with plasmids containing sgRNAs targeting the promoter of human SCN2A compared to a no sgRNA VP64 control. Fold change of SCN2A transduced with rAAV-DJ virus in human SH-SY5Y cells. c. SCN2A mRNA expression from SCN2A+/+ (black), SCN2A+/− (cyan), and SCN2A-rAAV-CRISPRa treated SCN2A+/− (purple) hESC-derived neurons normalized to wild type average. Holm-Šídák multiple comparisons test. n = 3 replicates for all conditions. circles are individual replicates. Bars are mean ± SD. Representative images of SCN2A+/+ (black) and SCN2A+/− (cyan) human stem-cell-derived neurons immunostained with antibodies against ankyrin-G (green) and MAP2 (magenta). Arrowheads denote start and end points used to quantify AIS length. b. Quantification of AIS length. SCN2A+/+: n = 56 cells, 3 dishes. SCN2A+/−: n = 56 cells, 3 dishes. c. Representative images of SCN2A+/− neurons expressing Scn2a-rAAV-CRISPRa-mCherry (purple) and mCherry-negative internal SCN2A+/− controls (cyan). Immunostaining against ankyrin-G and MAP2. d. Quantification of AIS length. SCN2A+/−: n = 122 cells, 3 dishes. SCN2A+/− + CRISPRa: n = 23 cells, 3 dishes. Circles are individual AIS calculations. Box plots are medians, quartiles, and 100% tails. Full descriptions for Supplementary Tables 1–3. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Tamura, S., Nelson, A.D., Spratt, P.W.E. et al. CRISPR activation for SCN2A-related neurodevelopmental disorders. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09529-3'>Learning the natural history of human disease with generative transformers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 15:25:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Decision-making in healthcare relies on understanding patients' past and current health states to predict and, ultimately, change their future course1,2,3. Artificial intelligence (AI) methods promise to aid this task by learning patterns of disease progression from large corpora of health records4,5. However, their potential has not been fully investigated at scale. Here we modify the GPT6 (generative pretrained transformer) architecture to model the progression and competing nature of human diseases. We train this model, Delphi-2M, on data from 0.4 million UK Biobank participants and validate it using external data from 1.9 million Danish individuals with no change in parameters. Delphi-2M predicts the rates of more than 1,000 diseases, conditional on each individual's past disease history, with accuracy comparable to that of existing single-disease models. Delphi-2M's generative nature also enables sampling of synthetic future health trajectories, providing meaningful estimates of potential disease burden for up to 20 years, and enabling the training of AI models that have never seen actual data. Explainable AI methods7 provide insights into Delphi-2M's predictions, revealing clusters of co-morbidities within and across disease chapters and their time-dependent consequences on future health, but also highlight biases learnt from training data. In summary, transformer-based models appear to be well suited for predictive and generative health-related tasks, are applicable to population-scale datasets and provide insights into temporal dependencies between disease events, potentially improving the understanding of personalized health risks and informing precision medicine approaches. Patterns of multimorbidity affect individuals unevenly and have been associated with lifestyle, heritable traits and socioeconomic status1,2,3. Understanding each individual's multi-morbidity risks is important to tailor healthcare decisions, motivate lifestyle changes or direct entrance into screening programs, as is the case for cancer8,9. Critically, health cannot only be understood by the presentation of individual diagnoses but, rather, in the context of an individual's co-morbidities and their evolution over time. While a wide range of prediction algorithms exist for specific diseases, from cardiovascular disease to cancer10,11,12, few algorithms are capable of predicting the full spectrum of human disease, which recognizes more than 1,000 diagnoses at the top level of the International Classification of Diseases, Tenth Revision (ICD-10) coding system. Learning and predicting patterns of disease progression is also important in populations that are ageing and that exhibit shifts in their underlying demographic's morbidities. For example, it has been predicted that, globally, the number of cancer diagnoses will increase 77% by 2050 (ref. 13) or that, in the UK, the number of working-age individuals with major illnesses, including depression, asthma, diabetes, cardiovascular disease, cancer or dementia, will increase from 3 to 3.7 million by 2040 (ref. Modelling the expected burden of disease is therefore critical for healthcare and economic planning and, moreover, the continual tracking of disease occurrence along with its likely future prevalence within population groups promotes a more informed healthcare system. Recent developments in AI may help to address some methodological limitations of multi-morbidity modelling, which have so far proved difficult to overcome15. Aside from the great number of diagnoses, these include challenges in modelling temporal dependencies among previous events, the integration of potentially diverse prognostically relevant data and the statistical calibration of predictions. Large language models (LLMs)16,17,18,19—a subfield of AI that enables chatbots such as ChatGPT20,21—model language as a sequence of word fragments (tokens). Generated token by token, the new text is based on all preceding text and, with enough training, the statistical dependencies among these tokens prove sufficient to produce context-aware and even conversational text, which is often indistinguishable from that of a human counterpart. The analogy between LLMs and disease progression modelling, which also entails recognizing past events and exploiting their mutual dependencies to predict the future sequence of morbidity, has recently inspired a series of new AI models. However, despite promising proofs of concept4,28,29, the potential for comprehensive and generative multi-morbidity modelling has not yet been fully assessed. Here we demonstrate that attention-based transformer models, similar to LLMs, can be extended to learn lifetime health trajectories and accurately predict future disease rates for more than 1,000 diseases simultaneously on the basis of previous health diagnoses, lifestyle factors and further informative data. Our extended model, termed Delphi-2M, was trained on data from the UK Biobank, a population-scale research cohort, and validated on Danish population registries. The vocabulary of the model includes ICD-10 top-level diagnostic codes, as well as sex, body mass, smoking, alcohol consumption and death. Delphi provides individual-level predictions of multi-disease incidences and models future health trajectories at any point throughout an individual's life course. Moreover, the internal model of Delphi offers insights into how past data influence the rates of subsequent diseases. We further assess biases and fairness across demographic subgroups and discuss Delphi's potential as a framework for healthcare modelling. A person's health trajectory can be represented by a sequence of diagnoses using top-level ICD-10 codes recorded at the age of first diagnosis as well as death. Furthermore, ‘no event' padding tokens were randomly added at an average rate of 1 per 5 years to eliminate long intervals without other inputs, which are especially frequent for younger ages and during which the baseline disease risk can change substantially (Extended Data Fig. Together, these data comprise 1,258 distinct states—tokens in LLM terminology. Additional information includes sex, body mass index (BMI) and indicators of smoking and alcohol consumption, which are used as input information but not predicted by the model (Fig. a, Schematic of health trajectories based on ICD-10 diagnoses, lifestyle and healthy padding tokens, each recorded at a distinct age. b, Training, validation and testing data derived from the UK Biobank (left) and Danish disease registries (right). d, Example model input (prompt) and output (samples) comprising (age:token) pairs. e, Scaling laws of Delphi, showing the optimal validation loss as a function of model parameters for different training data sizes. f, Ablation results measured by the cross-entropy differences relative to an age- and sex-based baseline (y axis) for different ages (x axis). g, The accuracy of predicted time to event. The observed (y axis) and expected (x axis) time to events are shown for each next token prediction (grey dots). Additional external testing was conducted on the Danish disease registry data, which covered 1.93 million Danish nationals and spanned the period from 1978 to 2018. To model disease history data, which, in contrast to text, occurs on a continuous time axis, we extended the GPT-2 architecture6 (Fig. Transformer models map their inputs into an embedding space, where information is successively aggregated to enable autoregressive predictions. The first change therefore replaces GPT's positional encoding, a mapping that identifies each text token's discrete position, with an encoding of continuous age using sine and cosine basis functions16. Standard GPT models only predict the next token using a multinomial probability model. Third, GPT's causal attention masks, which ensure that the model accesses only information from past events, are amended to additionally mask tokens recorded at the same time. Padding, lifestyle and sex tokens use a similar encoding but do not enter the likelihoods, as the model is deliberately not trained to predict them. This architecture enables one to provide the model with a partial health trajectory (prompt in LLM terminology) to calculate the subsequent rate (per day) for each of the 1,256 disease tokens plus death. Iteratively, this procedure samples entire health trajectories (Fig. A systematic screen of architecture hyperparameters (embedding dimensionality, number of layers, heads) confirms the reported empirical scaling laws30, which state that model performance increases with the number of datapoints and, up to a limit defined by the available data, as the number of parameters increases (Fig. The screen indicates that, for the UK Biobank dataset, optimal Delphi models have around 2 million parameters. One of the models within the optimal range has an internal embedding dimensionality of 120, 12 layers and 12 heads, amounting to a total of 2.2 million parameters. Results based on this model parameterization are discussed throughout the rest of the paper. We note that qualitatively similar results are obtained from other parameter choices (Extended Data Fig. An ablation analysis shows how Delphi-2M architectural modifications contribute to a better age- and sex-stratified cross-entropy compared with a standard GPT model (Fig. A good, albeit slightly inferior, classification performance at different ages may already be achieved by adding regular ‘no event' padding tokens to the input data with GPT models alone. However, a key distinguishing feature of Delphi compared with basic GPT models is its ability to calculate the absolute rates of tokens, which provide consistent estimates of inter-event times (Fig. This property also implies that the rates may be interpreted as the incidences of tokens. Delphi-2M's accuracy in predicting diverse disease outcomes in the validation cohort is compared to the sex and age-stratified incidence as an epidemiological baseline. 2a, the incidence curves are very varied, with some diseases, such as chickenpox, peaking in infancy, while others, such as asthma or depression, are relatively flat and with most rising exponentially in old age. Moreover, there are noticeable differences between the sexes, which are obvious for breast cancer but also pronounced for diabetes, depression, acute myocardial infarction and death. For some diseases, such as asthma or arthrosis, the spread is narrow, indicating a limited ability to predict beyond the sex- and age-incidence trend. Yet for other diseases, including septicaemia, and also death, the spread is wide, indicating predictable inter-individual differences in disease rates. a, The predicted rates for nine exemplary diagnoses and death (y axis) as a function of age (x axis). The points show predictions at each recorded input token. Colours separate biological sex; the darker colours indicate predictions immediately before the diagnosis in question. The purple and turquoise lines are disease rates observed for each yearly age bin in the training data. The solid black line connects consecutive predictions for one randomly selected case throughout age. b, Average age–sex-stratified AUC values (y axis) as a function of training occurrences (x axis). Shown are data for n = 906 diagnoses for male individuals and n = 957 diagnoses for female individuals for which a sufficient number of events was recorded in the validation data to evaluate AUC values. d, The same as b, aggregated by sex. e, AUC values of all diagnoses in b for different time gaps between prediction and diagnoses (x axis). f, ROC curves for Delphi and other clinical or machine learning methods for three selected end points evaluated on the internal longitudinal testing set. g, AUC values of MILTON31, a biomarker-based machine learning model (x axis), in prognostic mode, compared with Delphi-2M AUC values from the UK Biobank validation set (y axis) for n = 410 diagnoses. Delphi's ability to predict the next diagnosis token across the spectrum of human disease is confirmed by the average age-stratified area under the receiver operating characteristic curve (AUC), which averages at values of approximately 0.76 in the internal validation data (Fig. For 97% of diagnoses, the AUC was greater than 0.5, indicating that the vast majority followed patterns with at least partial predictability. These patterns were found to be true across the different chapters of the ICD-10 spectrum, which define broad groups of disease for both sexes (Fig. Among the most confidently predicted next events is death, with an age-stratified AUC of 0.97 in both sexes. Importantly, calibration analyses in 5-year age brackets show that the predicted rates closely match the observed number of cases, showing that the models' rates of the next tokens are consistently estimated (Extended Data Fig. Next-event predictions are often the consequence of acute illness or diagnostic refinements that accrue over the course of a few weeks or months, which may be undesirable for prognostication. Delphi-2M's average AUC values decrease from an average of 0.76 to 0.70 after 10 years, indicating that its predictions are also relevant for long-term prognostication (Fig. Similar results were observed in longitudinal test data, which also show no substantial shift in diagnostic patterns throughout the Biobank's follow-up (Supplementary Fig. The performance of Delphi was similar to routinely used clinical risk scores for cardiovascular disease and dementia, and better than those used for death. For diabetes, the performance of Delphi was worse compared with the use of a single marker, HbA1c, which is used clinically for risk prediction and diagnosis of diabetes (Fig. This was the case for next-event predictions, as well as prediction horizons up to 24 months. Delphi-2M's AUC values were also generally higher than those of a recent machine learning algorithm that calculates the risks of a similarly broad spectrum of ICD-10 diagnoses using 67 different biomarkers available through the UK Biobank31, even though for many diagnoses, such as diabetes, biomarkers remain indispensable (Fig. 4), marking potential for future modifications of Delphi that additionally use data beyond health records (Extended Data Fig. For most cases, Delphi-2M's multi-disease predictions match or exceed current risk models for individual disease outcomes and offer the great advantage of enabling the simultaneous assessment of more than 1,000 diseases and their timing at any given time, while also surpassing multi-disease models in quality. One of the most promising features of generative models is the ability to sample disease trajectories, conditional on data recorded up to a certain point. This is a property that few conventional epidemiological models possess. To systematically assess the influence of medical histories on future health, we sampled health trajectories for each participant from the UK Biobank validation cohort on the basis of data available until the age of 60 years (Fig. This provides the opportunity to compare 63,662 sampled and observed trajectories. When evaluated at the population level, the disease incidences at ages 70–75 years are well recapitulated, showing that the overall distributions are well preserved by iterative sampling (Fig. This is further confirmed by the cross-entropy loss of sampled trajectories, which is, on average, indistinguishable from the observed data but drops when the preceding disease histories are shuffled between participants (Supplementary Fig. a, Schematic of the experiment design. Delphi-2M is used to simulate health trajectories using validation data (n = 63,622 individuals with disease records both before and after 60 years of age) observed until the age of 60. A single trajectory is simulated per individual. b, Delphi-2M-modelled disease rates at ages 70–75 years (x axis) compared with observed rates at the same ages (y axis). c, The fraction of correctly predicted diagnoses (y axis) per 1-year age bin as a function of the years after simulation started at age 60 years (x axis). The blue curve uses age and sex as a prediction baseline. d, Simulated (x axis) and observed (y axis) fold changes of disease rates for high versus low smoking, alcohol consumption and BMI groups. The evaluation period included ages 70–75 years and used simulations from the age of 60 years. e, The same as b, evaluated for simulations from birth. f, The AUC values of disease risk prediction (n = 1,334 disease–sex pairs) for Delphi when trained on UKB and Delphi-2M-sampled synthetic data (Methods). In the first year of sampling, there are on average 17% disease tokens that are correctly predicted, and this drops to less than 14% 20 years later. These figures compare to values of 12–13% of correctly predicted disease tokens using sex and age alone, confirming that the conditional generation helps to make more accurate predictions of future events (Fig. Delphi-2M's ability to simulate differential health outcomes over a decade or more, on the basis of each individual's health history, manifests in a multitude of ways. For example, the changes in disease burden in different population subsets defined by smoking, alcohol consumption or BMI are well predicted (Fig. Similar findings are observed when the population is stratified by the presence of previous diseases or by estimated disease risks (Extended Data Fig. Together, these analyses show that Delphi-2M's conditional samples provide meaningful extrapolations for future health courses, which reflect the influence of past health events. The use of synthetic data has been proposed to help overcome issues with privacy in biomedical modelling if such datasets do not reveal characteristics specific to any one person. Fully synthetic data, which are sampled from birth with randomly assigned sex, reproduce the observed age and sex-specific incidence patterns throughout life (Fig. Further assessment shows that the generated trajectories do not exhibit any greater similarity to the training data than those from the validation cohort (Supplementary Fig. While partially overlapping disease trajectories may be found in terms of absolute disease tokens, the extent of overlap appears as expected on the basis of the observed incidences and co-morbidity patterns. To illustrate the use of synthetic data, we trained a version of Delphi-2M exclusively on synthetic data. Notably, when evaluated on the observed validation data, the fully synthetically trained model achieves an age–sex-stratified average AUC of 0.74, which is only three percentage points lower than that of the original Delphi-2M model (Fig. This confirms that synthetic data preserve much of the information relevant to training Delphi models and may serve as a less privacy-sensitive alternative to personal data. Insights into how Delphi-2M uses past information to predict future disease rates can be obtained by assessing the structure of the disease embeddings. GPT models linearly map inputs into a lower-dimensional embedding space, in which the temporal sequence of events is iteratively aggregated to produce a state from which predictions of each next token are derived (Fig. As shown by the uniform manifold approximation and projection (UMAP) representation of Delphi-2M's embedding matrix in Fig. 4a, disease codes cluster closely by the underlying chapter, a property that the model has no direct knowledge of, and that purely reflects co-occurrence patterns in the data. Yet there are also noticeable exceptions, for example, cancers and precancers of the female reproductive tract. Another noteworthy cluster involves the two types of diabetes, retinal disorders and the neuropathies caused by them. Diseases with high acute mortality, such as myocardial infarction or septicaemia, are clustering with death. a, UMAP projection of token embeddings. Selected diseases are shown in the magnified areas. b, SHAP-explained token risk contributions for individual trajectories. Top, the risk of pancreatic cancer immediately before diagnosis at age 68.2 years, which was found to be 19× increased. Bottom, the SHAP estimates of contributions to estimated mortality at age 63.5 years, which was greatly increased, in large part due to the preceding diagnosis of pancreatic cancer. c, The average SHAP effect of each of n = 778 disease tokens with more than 5 occurrences and grouped by chapter (y axis) on the same set of tokens plus death (x axis). d, Rate change (SHAP value) of mortality (y axis) as a function of time after diagnosis (x axis) for selected diseases. To gain further insights into how individual tokens influence future risks, shapley additive explanations (SHAP) values measure the influence of each token from the input trajectory on model predictions by a systematic assessment of subsampled data for individual prediction. 4b, this analysis reveals that a series of disease diagnoses of the digestive tract (ICD-10 chapter XI) elevated their pancreatic cancer risk 19-fold. The subsequent pancreatic cancer diagnosis in turn increased the rate of mortality almost ten thousandfold. SHAP analysis of data from 100,639 individuals of the validation cohort reveals the mutual dependencies by which each disease, sex and lifestyle token influences the rate of subsequent disease tokens, similar to hazard ratios in conventional statistical models (Fig. Effects mostly increase the rates of other diseases and are usually found among diseases of the same ICD-10 chapters, underscoring that the recorded patterns of co-morbidities often cluster within specific ICD-10 disease chapters. Particular clusters spanning entire disease chapters are visible for ICD-10 chapters V (mental disorders) and XV (pregnancy and childbirth). Notably, such patterns often appeared symmetrical, indicating similar predicted effect sizes of one disease token influencing another and vice versa (Supplementary Fig. This behaviour can be attributed to the structure of the embedding space, which places temporarily co-occurring diagnoses in local proximity (Extended Data Fig. The patterns of modelled influences after 10 years are similar to short-term effects, even though the strength of associations is greatly attenuated (Fig. The cluster relating to diseases of ICD-10 chapter XV (pregnancy and childbirth) within 5 years is entirely absent after 10 years, which is expected given that pregnancy-associated diseases occur within a finite period. However, dependencies among mental disorders remain apparent, similar to the effects of neoplasms on mortality. These observations are noteworthy as the quantification of temporal dependencies on past events poses a particular challenge for conventional epidemiological models, whereas Delphi-2M's GPT model uses attention-based weights, which are updated with every new input, including the ‘no event' paddings. To further illustrate Delphi-2M's capabilities of modelling temporal dependencies, we note that, for some diseases, such as cancers, the influence on mortality decays with a half-life of several years, reflecting the sustained risks of recurrence or impacts of treatment (Fig. However, for septicaemia, the influence on mortality is much more short-lived and drops sharply, effectively recovering to values close to the population average. This inference agrees with traditional Nelson–Aalen analyses of the hazard rates (Supplementary Fig. This behaviour is also reflected by Delphi-2M's attention maps, which show that cancer tokens are attended to for long periods, while those of septicaemia, myocardial infarction and many other diseases tend to be short lived (Supplementary Fig. To assess whether Delphi-2M's inference generalizes to unseen cohorts, we performed external testing using Danish population registry data. For this purpose, we transferred Delphi-2M with the weights learned from the UK Biobank training and evaluated predictions on the Danish data; no retraining or adjustments have been made. The average AUC when Delphi-2M is applied to Danish data was 0.67 (s.d. 0.09), which is lower than for longitudinal testing on UKB data (0.69, s.d. Predictions for different diseases were highly correlated across the datasets (Pearson correlation coefficient 0.76, 95% confidence interval (CI): 0.72–0.80) (Fig. The fact that Delphi-2M can be applied to Danish population data with slightly reduced accuracy indicates that many patterns learned by the model accurately reflect the true evolution of multi-morbidity, while also highlighting the existence of differences within each cohort. a, Comparison between AUC values in the UKB longitudinal testing and the external testing using Danish data. b, Yearly mortality estimates by Delphi-2M (UK validation cohort), observed rates in the UK Biobank and Office for National Statistics national estimates across the entire British population. As only living individuals between 40 and 70 years of age (black line) were recruited to the UK Biobank, many deaths are missing compared with the Office for National Statistics population estimate (grey shaded area). c, UpSet plot of disease data availability in the UK Biobank validation cohort (n = 100,639) (top). e, Hospital-record missingness bias (relative rate after first hospital token; y axis) as a function of token exclusivity to hospital records (x axis) for each relevant diagnosis (points). Points are coloured by the ICD-10 chapter, the overall trend is shown in black using a nonparametric (loess) curve with 95% confidence intervals shown in grey (UK validation cohort). f, Primary care missingness bias (y axis) as a function of primary care token exclusivity, coloured by the ICD-10 chapter. The trend is shown in black using a nonparametric (loess) curve with 95% confidence intervals shown in grey (UK validation cohort). g, SHAP value matrix, similar to Fig. The columns and rows correspond to different diseases and are sorted by the dominating source, then ICD-10 chapter. A dominating source is defined as the origin of more than 65% of records for a given disease; diseases without a dominating source are not shown. SHAP values indicate the greater influence of diseases on other diseases from the same group. A further question is to what extent a model trained on the UK Biobank, which is an epidemiological cohort, generalizes to the general population. The UK Biobank comprises more white British citizens than the general population, and the participants tend to be on average more affluent and educated32. Lower rates of diseases are reported among participants of white ethnicity, and the rate of diagnoses increases with deprivation—trends that are reproduced by Delphi-2M (Extended Data Fig. Further assessments of Delphi-2M's performance in demographic subgroups are provided in the Supplementary Discussion, Supplementary Figs. In the UK Biobank, most individuals have been recruited between the ages of 40 and 70. This creates a selection bias, as no deaths are recorded before recruitment, which has direct implications for the estimated mortality (Fig. This immortality bias also indirectly affects the incidence of diseases associated with high mortality, such as cancers, as only survivors are included in the UK Biobank. For time-dependent analyses, the jump of mortality to non-zero values at recruitment can also lead to false attribution of the apparent increase to unrelated variables recorded at the time of recruitment. Furthermore, limited follow-up data are currently available for individuals 80 years of age and older. This period is therefore not reliably modelled by Delphi-2M. UK Biobank's disease data have been collated from self-reports, primary care, hospital admissions, cancer and death registries, each of which contributes characteristic disease tokens: self-reporting and GP records contain mostly common diseases, while data from hospital records include more-aggressive disease tokens, such as myocardial infarction or septicaemia (Fig. However, the underlying sources were not always available for each participant and time period (Fig. Such patterns, which reflect only the data-collection process, are also learned by Delphi-2M. The predicted rates of diseases exclusive to hospital records are, on average, ten times higher in individuals with a disease history that includes other hospital records (Fig. Septicaemia, for example, is diagnosed in 93% of cases in a hospital setting and is predicted to occur at 8× greater rates in individuals with any other hospital data. These source effects also explain some of the substructures visible in the UMAP representation of disease embeddings (Fig. 10a–d) and also in the matrix of SHAP effects (Figs. While some of these associations may reflect true diagnostic pathways or disease clusters diagnosed in a distinct care setting, it nevertheless appears that some of these associations are artefacts stemming from the incomplete aggregated nature of the UK Biobank's data. Here we present Delphi-2M—a GPT-based model of multi-disease progression. Delphi-2M extends the GPT large language model to account for the temporal nature of health trajectories. Analogous to LLMs, which learn the grammar and contextual logic of language from large bodies of text, Delphi-2M inferred the patterns of multi-disease progression when trained on data for more than 1,000 diseases and baseline health information recorded in 402,799 UK Biobank participants. A detailed assessment of Delphi-2M's predictions showed that they consistently recapitulate the patterns of disease occurrence at the population scale as recorded in the UK Biobank. For the majority of diseases, Delphi-2M's multi-disease, continuous-time model predicted future rates at comparable or better accuracy than established single-disease risk models, alternative machine learning frameworks and blood-biomarker-based models. Only a small performance drop was observed when applied to data from Danish disease registries, demonstrating that models are—even without additional finetuning—largely applicable across national healthcare systems. Delphi-2M is uniquely capable of sampling future disease trajectories, which enables the estimation of cumulative disease burdens over periods of up to 20 years, conditional on previous health information. Note that Delphi's predictions are generally strongly influenced by statistical chance and compatible with a range of outcomes for a given individual. The ability to generate synthetic data may also help create datasets that preserve the statistical co-occurrence patterns without revealing any specific data, which could facilitate the development of further AI models with a decreased risk of revealing personal information. Delphi-2M offers insights into the modes of disease progression. The ability to cluster disease risks may be useful for genomic association studies that focus on comorbidities or are stratified by the risks derived from health trajectories. Delphi-2M's ability to quantify the temporal influence of previous health data revealed that cancers increase mortality in a sustained manner, while the effects of myocardial infarction or septicaemia regress within 5 years. Similar analyses also revealed clusters of persisting comorbidities, such as mental health conditions. Although Delphi-2M appears to be capable of modelling temporally directed dependencies, we caution against interpreting these as causal relationships that could be exploited to modify future health courses. There are also several limitations that need to be considered. A detailed analysis of UK Biobank's first occurrence data revealed several biases, reflected by Delphi-2M. In addition to a healthy volunteer bias and participant selection bias before recruitment, the diverse nature of health data sources impacted Delphi's prediction, as UK-Biobank-specific patterns of missingness were exploited to infer disease rates. Furthermore, Delphi-2M predicts different disease rates in subgroups based on ancestry background and deprivation indices, but no observable trend between lifestyle measures and birth year. These findings underscore the need for caution when using AI models for inference and prediction in heterogeneous healthcare datasets, potentially marking them as useful additions to currently used diagnostic pipelines, rather than replacements. A promising feature of Delphi-2M's implementation is the relative simplicity of incorporating additional data layers, rendered possible by the transformer-based architecture. Immediate refinements of Delphi-2M may incorporate additional lifestyle data, self-reported health status, prescription records and blood tests, all of which are usually available in a general healthcare setting. Further multimodal extensions could include genomic data, richer metabolomic information, diagnostic imaging data or data from wearables that can be added to Delphi-2M's embedding layer, similar to how lifestyle tokens are currently incorporated (Extended Data Fig. Furthermore, while ICD-10 provides a predefined tokenization of diseases, LLMs have been shown to also conceptualize natural language, making it plausible to expect that future models may derive similar meaning directly from free text records, enabling the application of Delphi-like models to unstructured data. Lastly, Delphi-2M itself could serve as an extension to LLMs. Similarly to systems that provide LLMs with query-relevant web search results to reduce hallucinations33, a future healthcare-oriented LLM could invoke a Delphi-based model to improve the numerical accuracy of the generated replies34. An evident application of Delphi-type models is to support medical decision-making by rationally integrating information from various data modalities, which can be a challenge for healthcare professionals. Potential use cases could include identifying individuals who would benefit most from diagnostic tests or finding individuals with disease risk high enough to include them in screening programs, even if they have not yet met conventional age-based criteria. However, deploying clinical decision support systems requires a regulatory framework, which is still in its infancy for AI in healthcare. An alternative use of Delphi models would be to inform healthcare providers, insurers and policymakers. Such applications, which inform the provision of healthcare rather than treatment decisions, require less regulation. Delphi's modelling has a resolution of an individual; when such predictions are aggregated, Delphi-type models may provide substantial system-wide modelling benefits, particularly in projecting the expected disease burden at local, regional and national levels, or for specific demographic groups of interest. This could reveal communities with unmet future healthcare needs over the next 1–2 decades and provide an opportunity to adjust the provision of healthcare. Such capabilities appear especially valuable in ageing populations in which healthcare needs are becoming increasingly complex and resource intensive. These considerations illustrate the wide range of applications of generative models for biomedical research and, ultimately, also for healthcare. With appropriate training and evaluation, future multimodal model extensions may be used for preventive medicine, clinical decision support and healthcare planning. Our model and analyses present a further step towards unlocking the considerable healthcare benefits of the era of AI. The UK Biobank is a cohort-based prospective study comprising approximately 500,000 individuals from various demographic backgrounds recruited across the UK between 2006 and 2010. These data include ICD-10 level 3 codes (for example, E11: type 2 diabetes mellitus) for diseases in chapters I–XVII, excluding chapter II (neoplasms), plus death. The data are pre-assembled by UK Biobank and include the first reported occurrence of a disease in the linked primary care data (cat. 2000), death registry (fields 40001 and 40002) or self-reported data through questionnaires (field 20002). Information on neoplasms was not included in category 1712 by the UK Biobank, and we therefore included the data ourselves through the addition of the linked cancer registry data in fields 40005 and 40006 (subset to the first occurrence and mapped to ICD-10 level 3 codes), which, combined with first occurrence data, gives in total 1,256 distinct diagnoses. A list of all codes used is provided in Supplementary Table 5. We extract information on the self-reported sex of participants as recorded in field 31 (indicators for female and male), a physical assessment of body mass index at recruitment from field 21001, which we split into three indicators encoding BMI < 22 kg m−2, BMI > 28 kg m−2 and otherwise, as well as smoking behaviour from field 1239 with indicators for smoker (UKB coding: 1), occasionally smoking (2) and never smoker (0) and alcohol intake frequency from field 1558 with indicators for daily (1), moderately (2, 3) and limited (4, 5, 6). Furthermore, information that we extracted and used for stratification to assess model performance in subgroups but were not part of the data for model training include self-reported ethnic background (field 21000), with participants grouped into five level groups (white, mixed, Asian or Asian British, Black or Black British, and Chinese) and an index of multiple deprivation as available in field 26410. A list of the variables and their codes can be found in Supplementary Table 5. Exploring comorbidities and health-related factors is uniquely facilitated by Denmark's comprehensive registries, which gather up to 40 years of interconnected data from across the entire population. All used registries are linkable through a unique personal identification number provided in the Central Person Registry along with information on sex and date of birth. Furthermore, we used the Danish National Patient Registry36 (LPR), a nationwide longitudinal register with data on hospital admissions across all of Denmark since 1977, along with the Danish Register of Causes of Death37 since 1970 to extract information on an individual's acquired diagnoses throughout their lifetime. Our current data extract covers information up until around 2019 when reporting to the LPR was updated to LPR3. Furthermore, we restrict our cohort to individuals 50–80 years of age on 1 January 2016, to obtain a similar age range as in the UK Biobank. To obtain a dataset that resembles the UK Biobank data, we retain only the first occurrence of an individual's diagnosis and transform all codes to ICD-10 level 3 codes. Diagnoses before 1995 are reported in ICD-8 and have been converted to ICD-10 codes using published mappings38. Codes that may be present in the Danish register but were not in the UK Biobank are removed. Information on lifestyle is not available, and indicators for BMI, smoking and alcohol intake have therefore been treated as absent. The models were trained on UK Biobank data for 402,799 (80%) individuals using data from birth until 30 June 2020. Internal longitudinal testing was carried out using data for all individuals still alive by the cut-off date (471,057) and evaluated on incidence from 1 July 2021 to 1 July 2022, therefore enforcing a 1 year data gap between predictions and evaluation. Validation assesses how well the model generalized to different individuals from the same cohort. Longitudinal testing investigates whether the model's performance changes over time and if it can be used for prognostic purposes. External longitudinal testing was conducted on the Danish registries. All individuals residing in Denmark 50–80 years of age on the 1 January 2016 were included. Predictions are based on the available data up to this point and were subsequently evaluated on incidence from 1 January 2017 to 1 of January 2018, similar to the internal longitudinal testing. Data were collected for 1.93 million individuals (51% female and 49% male), with 11.51 million disease tokens recorded between 1978 and 2016. Predictions were evaluated on 0.96 million disease tokens across 796 ICD-10 codes (each with at least 25 cases). Delphi's architecture is based on GPT-2 (ref. The basic GPT model uses standard transformer blocks with causal self-attention. A standard lookup table embedding layer with positional encoding was used to obtain the embeddings. The embedding and casual self-attention layers are followed by layer normalization and a fully connected feedforward network. Transformer layers, consisting of causal self-attention and feedforward blocks, are repeated multiple times before the final linear projection that yields the logits of the token predictions. The residual connections within a transformer layer are identical to those in the original GPT implementation. Here we also use weight tying of the token embeddings and final layer weights, which has the advantage of reducing the number of parameters and allowing input and output embeddings to be similarly interpreted. Each datapoint consists of pairs (token, age) recording the token value and the proband's age, measured in days from birth, at which the token was recorded. The token vocabulary consists of n = 1,257 different ICD-10 level 3 disease tokens, plus n = 9 tokens for alcohol, smoking and BMI, each represented by three different levels, as well as n = 2 tokens for sex and n = 1 no-event padding token as well as n = 1 additional, non-informative padding token at the beginning or end of the input sequences. No-event padding tokens were added to the data with a constant rate of 1 per 5 years by uniformly sampling 20 tokens from the range of (0, 36525) and interleaving those with the data tokens after intersecting with the data range for each person. No-event tokens eliminate long time intervals without tokens, which are typical for younger ages, when people generally have fewer diseases and therefore less medical records. Transformers predict the text token probability distribution only at the time of currently observed tokens; thus, no-event tokens can also be inserted during inference to obtain the predicted disease risk at any given time of interest. Sex tokens were presented at birth. As this specific time also coincided with the end of immortal time bias (probands had to be alive when they were recruited), smoking, alcohol and BMI tokens times were randomized by −20 to +40 years from this point in time to break an otherwise confounding correlation leading to a sudden jump in mortality rates (and possibly other diseases with high mortality, such as cancers) associated with the recording of these tokens. This probably also diminishes the true effect of these tokens. Following the logic frequently used for positional encodings, age is represented by sine and cosine functions of different frequencies, where the lowest frequency is given by 1/365. These functions are subsequently linearly combined by a trainable linear transformation, which enables the model to share the same basis function across multiple encoding dimensions. Another advantage of using age encoding is that Delphi can handle token inputs of arbitrary length, as no parameters are associated with token positions. Yet in the case of time-dependent data, tokens can be recorded at the same time with no specified order. Thus, attention masks were amended to mask positions that occurred at the same time as the predicted token. Delphi is motivated by the theory of competing exponentials. Let Ti be the waiting times from the current event to one of i = 1, …, n competing events, where n is the number of predictable tokens. Assuming the Ti are each exponentially distributed waiting times with rates λi = exp(logitsi), the next event being j is equivalent to Tj being the first of the competing waiting times, that is, Tj = min Ti, or equivalently j = argmin Ti. It can be shown that the corresponding probability is P(j = argmin Ti) = λj/Σiλi, which is the softmax function over the vector of logits. Conveniently, this definition corresponds to the classical cross-entropy model for classification with λ = exp(logits). Thus, Delphi uses a conventional loss term for token classification: Furthermore, in the competing exponential model the time to the next event T* = min Tj is also exponentially distributed with rate λ* = Σiλi = Σi exp(logitsi). The loss function of exponential waiting times T between tokens is simply a log-likelihood of the exponential distribution for T*: These approximations hold as long as the rates λi are constant in time, which is a reasonable assumption over short periods. For this reason, padding tokens were introduced to ensure that waiting times are modelled over a relatively short period, which does not exceed 5 years in expectation. In line with the tie-braking logic used for causal self-attention, co-occurring events were predicted from the last non-co-occurring token each. Non-informative padding, as well as sex, alcohol, smoking and BMI, were considered mere input tokens and therefore removed from the loss terms above. This was achieved by setting their logits to -Inf and by evaluating the loss terms only on disease and ‘no event' padding tokens. For the time, samples from all exponential distributions with rates λi are taken, and the minimum is retained. Logits of non-disease tokens (sex, lifestyle) are discarded from the procedure to sample disease events only. Models were trained by stochastic gradient optimization using the Adam optimizer with standard parameters for 200,000 iterations. After 1,000 iterations of warmup, the learning rate was decayed using a cosine scheduler from 6 × 10−4 to 6 × 10−5. For example, probability of an event occurring within a year is given by P(T < 365.25) = 1 − exp(−exp(logit) × 365.25). For the training set, age- and sex-stratified incidences were calculated in annual age brackets. The observed counts were divided by the number of individuals at risk in each age and sex bracket, which was given by the number of probands for each sex minus the cumulative number of deaths to account for censoring. Calibration curves were calculated on the basis of predicted incidences. To this end, all cases of a given token accruing in five year age bins were identified. Subsequently, for all other probands, a control datapoint was randomly selected in the same age band. Predictions were evaluated at the preceding token given that the time difference was less than a year. The predicted incidences were then further grouped log-linearly into risk bins from 10–6 to 1, with multiplicative increments of log10(5). The observed annual incidence was then calculated as the average of cases and control in age bins, divided by 5 years. The procedure was separately executed for each sex. To account for baseline disease risk changes over time, trajectories with disease of interest were stratified into 5 year age brackets from 50 to 80 years, on the basis of the occurrence of the disease of interest. To each bracket, control trajectories of matching age were added. Predicted disease rates were used within each bracket to calculate the AUC, which was then averaged across all brackets with more than two trajectories with the disease of interest. The evaluation was performed separately for different sexes. To estimate the CI for AUC for individual age and sex brackets, we use DeLong's method, which provides CI mean and variance under the assumption that AUC is normally distributed. As AUC for diseases is calculated as an average of AUC for all brackets, as a linear combination of normal distributions it also is normally distributed with parameters: For each disease, we estimated the mean AUC \({\mu }_{s}\) and variance \({\sigma }_{s}^{2}\) for each subgroup using DeLong39 method. Under the null hypothesis that all subgroups have the same true AUC (no bias), any observed differences would be attributable to statistical variance. We use a two-level testing approach: (1) individual subgroup testing: for each disease–subgroup combination, we calculate standardized residuals by subtracting the weighted mean AUC across all subgroups from the subgroup-specific AUC and dividing by the s.d. Under the null hypothesis, these standardized residuals should follow a standard normal distribution. We identify outliers using a two-sided Bonferroni-corrected significance threshold. (2) Disease-level testing: for each disease, we sum the squared standardized residuals across all subgroups: Under the null hypothesis, this sum follows a χ2 distribution with degrees of freedom equal to (n − 1), where n is the number of subgroups. We identify diseases with excessive between-subgroup variance using a one-sided Bonferroni-corrected significance threshold. Owing to limitations of DeLong's method with small sample sizes, in each disease–subgroup combination, we filtered age and sex brackets with fewer than six cases and diseases with less that two brackets remaining after filtering. We also excluded diseases that had fewer than two subgroups presented. To compare the distribution of annual incidences of model and observed data, a cross-entropy metric was used. Let pi be the annual occurrence of token i in each year. For low incidences p and q, the latter term is usually small. The cross-entropy is evaluated across all age groups and sexes. To evaluate the potential of generating disease trajectories, two experiments were conducted using data from the validation cohort. First, trajectories were generated from birth using only sex tokens. This was used to assess whether Delphi-2M recapitulates the overall sex-specific incidence patterns. Second, all available data until the age of 60 were used to simulate subsequent trajectories conditional on the previous health information. A single trajectory was evaluated per proband. Trajectories were truncated after the age of 80 as currently little training data were available beyond this point. Incidence patterns were evaluated as described above. We trained a family of linear regression models on the task of predicting 5-year disease occurrence. Biomarkers (as used in the MILTON paper, biomarkers with more than 100,000 missing values in UKB excluded, imputed with MICE) Delphi logits for the disease of interest Moreover, all models had sex and age information included. To evaluate the performance of the models, we used the same age- and sex-stratified AUC calculation that we used for Delphi performance evaluation. For breast cancer, only female participants were included. For E10 insulin-dependent diabetes mellitus, we masked all other diabetic diseases (E11–E14) from Delphi inputs when computing logits. This has two advantages: (1) we can enforce an explicit cut-off and separate data to avoid any potential time-leakage; and (2) we obtain insights into Delphi-2M prognostic capabilities and generalization. However, as mentioned in the data splits, we use two different cut-off dates between the two data sources, mainly due to differential data availability, the principal setup applies to both in the exact same way. We collate data up to a specific cut-off date for each individual and use Delphi-2M to predict an individual's future rate across all disease tokens. Building on the exponential waiting time representation, we obtain rates over a 1 year time frame. The preceding year after the cut-off date is discarded to introduce a data gap. Predictions are made for individuals 50–80 years of age. We build a standard epidemiological baseline based on the sex- and age-stratified population rates. These are based on the Nelson–Aalen estimator40,41, a nonparametric estimator of the cumulative hazard rate, across all diseases. As the UK Biobank contains a wide range of phenotypic measures, we also estimate clinically established models and other machine learning algorithms for comparison. We evaluate the models on cardiovascular disease (CVD) (ICD-10: I20–25 I63, I64, G45), dementia (ICD-10: F00, F01, F03, G30, G31) and death. For CVD we compare against: QRisk3 (ref. For dementia we compare against: UKBDRS51, Transformer and LLama3.1(8B)50. We collect a total of 60 covariates that are used to varying degrees across the algorithms. A summary description of the covariates, as well as their corresponding UKB codes, can be found in Supplementary Table 5. For missing data, we perform multivariate imputation by chained equations55,56 (R:mice)57. We retain five data copies, estimate all scores and, finally, aggregate them by Rubins' rule. Results are reported based on the aggregated scores. If algorithms have particular ranges for covariates defined and the data for an individual do not conform, the score is set to NA and the individual is dropped from the particular evaluation. The transformer model is an encoder model based on the standard implementation provided in Python:pytorch (TransformerEncoder, TransformerEncoderLayer) with a context length of 128 tokens, an embedding size of 128, 2 multi-head attention blocks and a total of 2 sub-encoder layers, and the otherwise default parameters were used. A linear layer is used to obtain the final prediction score. The model is fitted on concatenated data excerpts of the UKB on 1 January 2014, 1 January 2016 and 1 January 2018 containing the same tokens as Delphi plus additional tokens encoding the current age based on 5 year bins (50–80 years) and is evaluated on a binary classification task of whether the corresponding outcome (CVD, dementia, death) will occur in the next 2 years. AutoPrognosis is fitted in a similar manner with data extracted on 1 January 2014; however, we use the covariates defined previously34. This is for research purposes only. Pretend you are a healthcare risk assessment tool. You will be given some basic information about an individual e.g. age, sex, BMI, smoking and alcohol plus a list of their past diseases/diagnoses in ICD-10 coding. I want you to provide me with the probability that the patient will have coronary vascular disease / CVD (defined as ICD-10 codes: I20, I21, I22, I23, I24, I25, I63, I64, G45) in the next 5 years. Here is an example: Input: ID(10000837); 54 years old, Female, normal BMI, past smoker, regular alcohol consumption, F41, M32, A00, C71, F32. Please only provide the ID and the risk score as output and do not tell me that I can not provide a risk assessment tool \n Here is the input for the individual: ID(10000736); 64 years old, Male, high BMI, current smoker, regular alcohol consumption, F41, M32, A00, C71, F32.” Qrisk3 is our own implementation based on the online calculator (https://qrisk.org/). The UKBDRS risk score for dementia is based on our own implementation as reported in the original paper51. For the comparison to MILTON, we obtained the reported AUC measures for all ICD-10 codes reported for diagnostic, prognostic and time-agnostic MILTON models from the articles supplementary material31. Linking on top level (3 character) ICD-10 codes, we were able to compare the prediction of 410 diseases between Delphi and MILTON prognostic models. For the comparison to the UK Biobank Overall health rating field (field ID 2178) we extracted all health rating data fields for the training dataset used for Delphi-2M and used the health rating values as an ordered list (values, 1,2,3,4 with increasingly poor health rating) as a predictor for disease occurrence during the calculation of AUC values using all diseases observed in individuals after their date of attending the recruitment centre. All other models are based on publicly available implementations. Only for the evaluation on chronic liver disease (K70–77), we used the predictions from a logistic regression model with alkaline phosphatase, alanine aminotransferase, gamma-glutamyltransferase, total protein, albumin, bilirubin and glucose as covariates. Evaluations are based on a 5-year time window after an individual's recruitment data. The Charlson and Elixhauser comorbidity index is based on the same data as Delphi; however, the Charlson comorbidity index is originally based on ICD-10 level 4 codes. We therefore estimate based on a version that maps the level 3 codes to all possible level 4 codes. We estimated a version with the level 3 codes as well and this did perform marginally worse. Overall, we tried to model the data as close as possible to the originally used covariates; however, in some places, small adjustments were made. All diseases with at least 25 cases were assessed. Delphi-2M predicted rates are split into deciles, and for each bin, we compare Delphi-2M's average rate against the observed rate within the bin. We include all diseases with at least 25 cases. Each trajectory from the validation cohort was augmented by masking one or several tokens and then used for prediction. The change of logits after many such augmentations was aggregated by a PartitionExplainer from the SHAP Python package. The number of augmentations for each trajectory was determined using the PartitionExplainer masking algorithm. When masked, tokens were replaced by a ‘no event' placeholder that was also used during training. Sex tokens, when masked, were replaced with the corresponding token of the opposite sex. To assess the interpretation of the SHAP values, we use a penalized time-dependent Cox model, developed for use with EHR data15, and compare the corresponding hazard estimates to our averaged SHAP values. For a given token, we identify individuals with the token and estimate their corresponding cumulative hazard from the occurrence of the token onwards. Moreover, we randomly select five age–sex-matched individuals for each case and estimate the cumulative hazard in this comparison group. The model was trained on simulated trajectories sampled from Delphi-2M. The trajectories were samples from birth; sex was assigned randomly. No training hyperparameters were changed compared to Delphi-2M. No novel data were generated for this study. Reproducibility of the method has been confirmed by retraining Delphi-2M using different train-validation splits (n = 4 independent experiments; Supplementary Fig. 1) and testing the trained model using longitudinal UK Biobank data and external data from the Danish National Patient Registry. The UK Biobank has received approval from the National Information Governance Board for Health and Social Care and the National Health Service North West Centre for Research Ethics Committee 532 (ref: 11/NW/0382). All UK Biobank participants gave written informed consent and were free to withdraw at any time. This research was conducted using the UK Biobank Resource under project 49978. All investigations were conducted in accordance with the tenets of the Declaration of Helsinki. The Danish Act on Ethics Review of Health Research Projects and Health Data Research Projects (the Committee Act) do not apply to the type of secondary analysis of administrative data reported in this paper. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. UK Biobank data are available under restricted access through a procedure described online (http://www.ukbiobank.ac.uk/using-the-resource/). Danish registry data are available for use in secure, dedicated environments through application to the Danish Patient Safety Authority and the Danish Health Data Authority at https://sundhedsdatastyrelsen.dk/da/english/health_data_and_registers/research_services/apply. Source data are provided with this paper. Code for Delphi and accompanying scripts and Jupyter notebooks are available at GitHub (https://github.com/gerstung-lab/delphi). The model's checkpoint is available according to UK Biobank's controlled access procedures with upload ID 7318. Zhu, Z. et al. Causal associations between risk factors and common diseases inferred from GWAS summary data. Social conditions as fundamental causes of disease. Nyberg, S. T. et al. Association of healthy lifestyle with years lived without major chronic diseases. Advancing multimodal medical capabilities of Gemini. Language models are unsupervised multitask learners. A unified approach to interpreting model predictions. Mukama, T. et al. Risk-adapted starting age of screening for relatives of patients with breast cancer. Calculating the starting age for screening in relatives of patients with colorectal cancer based on data from large nationwide data sets. Quan, H. et al. Updating and validating the Charlson comorbidity index and score for risk adjustment in hospital discharge abstracts using data from 6 countries. D'Agostino, R. B. Sr et al. General cardiovascular risk profile for use in primary care: the Framingham Heart Study. & Coupland, C. Development and validation of risk prediction algorithms to estimate future risk of common cancers in men and women: prospective cohort study. Global cancer statistics 2022: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. Watt, T. et al. Health in 2040: Projected Patterns of Illness in England (The Health Foundation, 2023). Jung, A. W. & Gerstung, M. Bayesian Cox regression for large-scale inference with applications to electronic health records. Touvron, H. et al. LLaMA: open and efficient foundation language models. Ouyang, L. et al. Training language models to follow instructions with human feedback. Rasmy, L., Xiang, Y., Xie, Z., Tao, C. & Zhi, D. Med-BERT: pretrained contextualized embeddings on large-scale structured electronic health records for disease prediction. Li, Y. et al. BEHRT: transformer for electronic health records. Li, Y. et al. Hi-BEHRT: Hierarchical Transformer-based model for accurate prediction of clinical events using multimodal longitudinal electronic health records. Yang, Z., Mitra, A., Liu, W., Berlowitz, D. & Yu, H. TransformEHR: transformer-based encoder-decoder generative model to enhance prediction of disease outcomes using electronic health records. Placido, D. et al. A deep learning algorithm to predict risk of pancreatic cancer from disease trajectories. Using sequences of life-events to predict human lives. Kraljevic, Z. et al. Foresight—a generative pretrained transformer for modelling of patient timelines using electronic health records: a retrospective modelling study. & Shah, N. H. MOTOR: a time-to-event foundation model for structured medical records. Hoffmann, J. et al. Training compute-optimal large language models. Garg, M. et al. Disease prediction with multi-omics and biomarkers empowers case-control genetic discoveries in the UK Biobank. Shuster, K., Poff, S., Chen, M., Kiela, D. & Weston, J. Retrieval augmentation reduces hallucination in conversation. Findings of the Association for Computational Linguistics: EMNLP 2021 3784–3803 (2021); https://aclanthology.org/2021.findings-emnlp.320/. Imrie, F., Rauba, P. & van der Schaar, M. Redefining digital health interfaces with large language models. Sudlow, C. et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. The Danish National Patient Registry: a review of content, data quality, and research potential. Pedersen, M. K. et al. A unidirectional mapping of ICD-8 to ICD-10 codes, for harmonized longitudinal analysis of diseases. DeLong, E. R., DeLong, D. M. & Clarke-Pearson, D. L. Comparing the areas under two or more correlated receiver operating characteristic curves: a nonparametric approach. Aalen, O. Nonparametric inference for a family of counting processes. Nelson, W. Theory and applications of hazard plotting for censored failure data. Hippisley-Cox, J., Coupland, C. & Brindle, P. Development and validation of QRISK3 risk prediction algorithms to estimate future risk of cardiovascular disease: prospective cohort study. Including measures of chronic kidney disease to improve cardiovascular risk prediction by SCORE2 and SCORE2-OP. Perez-Vicencio, D., Doudesis, D., Thurston, A. J. F. & Selva, J. R package RiskScorescvd version 0.2.0: cardiovascular risk scores calculator. Khan, S. S. et al. Development and validation of the American Heart Association's PREVENT equations. Goff, D. C. Jr et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. Mayer, M. R package preventr version 0.11.0: an implementation of the PREVENT and pooled cohort equations. McClelland, R. L. et al. 10-year coronary heart disease risk prediction using coronary artery calcium and traditional risk factors: derivation in the MESA (Multi-Ethnic Study of Atherosclerosis) with validation in the HNR (Heinz Nixdorf Recall) study and the DHS (Dallas Heart Study). Imrie, F., Cebere, B., McKinney, E. F. & van der Schaar, M. AutoPrognosis 2.0: democratizing diagnostic and prognostic modeling in healthcare with automated machine learning. Anatürk, M. et al. Development and validation of a dementia risk score in the UK Biobank and Whitehall II cohorts. Elixhauser, A., Steiner, C., Harris, D. R. & Coffey, R. M. Comorbidity measures for use with administrative data. Adaptation of the Charlson comorbidity index for register-based research in Sweden. Multiple Imputation for Nonresponse in Surveys (John Wiley & Sons, 1987). van Buuren, S. & Groothuis-Oudshoorn, K. mice: multivariate imputation by chained equations in R. J. Stat. van Buuren, S. et al. R package mice version 3.18.0: multivariate imputation by chained equations CRAN https://CRAN.R-project.org/package=mice (2025). & Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. We thank the participants of the UK Biobank for providing their medical data and enabling this research. We acknowledge the following sources of funding: Novo Nordisk Foundation grants NNF17OC0027594 (K.G., A.W.J., E.B., M.G., S.B. ); the EMBL European Bioinformatics Institute (EMBL-EBI) (E.B., T.F. We thank C. Relton and G. D. Smith for their detailed comments on the manuscript. Open access funding provided by Deutsches Krebsforschungszentrum (DKFZ). These authors contributed equally: Artem Shmatko, Alexander Wolfgang Jung, Kumar Gaurav Division of AI in Oncology, German Cancer Research Centre DKFZ, Heidelberg, Germany Artem Shmatko, Alexander Wolfgang Jung, Kumar Gaurav, Ewan Birney, Tom Fitzgerald & Moritz Gerstung Novo Nordisk Foundation Center for Protein Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark Alexander Wolfgang Jung & Laust Hvas Mortensen Department of Biosystems Science and Engineering, ETH Zurich, Basel, Switzerland Robert Bosch Center for Tumor Diseases, Stuttgart, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar prepared datasets and assessed the model parameterization. A.S. prepared all of the figures with contributions by A.W.J., K.G., T.F. Correspondence to Ewan Birney, Tom Fitzgerald or Moritz Gerstung. A patent has been filed for the use of generative transformer architectures to model competing risk and timings of diseases (application number: PCT/EP2025/065771; applicants: DKFZ, EMBL), with M.G., A.S., T.F., E.B., K.G. has ownership interests in Hoba Therapeutics Aps, Novo Nordisk, Lundbeck and Eli Lilly. is a consultant and shareholder of Oxford Nanopore. The other authors declare no competing interests. Nature thanks Na Cai, who co-reviewed with Francesco Paolo Casale; Yonghui Wu; and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Boxplots (n = 3 model replicates trained with different seeds) of the average loss (y-axis; lower is better) for Delphi-2M trained with different “no event” padding rates (inverse scale, x-axis). UK Biobank validation data was used to calculate the reported losses. b, Average cross-entropy loss, aggregated over 5-year age bins. A higher rate of “no event” tokens lowers the loss, especially for younger ages, during which generally few disease tokens are recorded, prohibiting the model from adjusting predictions for advancing age. c, “No event” token rate estimated by Delphi (y-axis) vs the true rate at which tokens were added to the training data. n = 4000 random timepoints from the validation dataset trajectories, selected for “no event” token rate evaluation. a, Validation cross-entropy (rightmost axis) for models trained with different architectural hyperparameter values (other axes). b, Same data as a, showing validation loss (y-axis) against each model parameter (x-axis). c, Random-forest-based importance of different hyperparameters and their correlation with validation loss. Shown are results for 9 selected diseases and death on validation data for age groups of 5 years and both sexes. Predictions in each age-sex stratum are grouped into bins of powers of 10 (x-axis, average within each bin, and observed rates are calculated from validation data for predictions falling into each bin (y-axis). b, Calibration plots on the Danish longitudinal testing data. Each line represents an ICD-10 disease evaluated for each decile of the Delphi rate and compared against the observed rate in the population. a, Comparison of Delphi-2M against clinical biomarkers for selected diseases performed using the UKB validation dataset. Predictions are based on the information available at recruitment and evaluated over the subsequent 5 years. Mod: Logistic regression model of several clinical markers. b, AUC results comparing Delphi-2M to a simple disease predictor of Overall health rating UKB data field 2178. AUC values for field 2178 as a predictor for future health events (after the date of recruitment) (x-axis) against the AUC values from Delphi using the UKB validation data. c. Boxplot, showing the prediction AUCs for Delphi, split over sex, disease chapter and lifestyle factors, such as alcohol consumption, smoking and BMI. Results of a linear regression model that uses Delphi logits and additional features to predict 5-year disease occurrence for selected diseases. Shown is the average validation AUC across 5-year age groups ranging from 40 to 80 years of age, additionally stratified by sex. All models use sex and age as additional covariates. For prediction, only data before recruitment was used. As additional features, models use polygenic risk scores (PRS, a), 57 biomarkers used in the MILTON study (b) and UKB field 2178 Overall health rating status (c). All simulations are from the age of 60 onwards and use validation data. b, Simulated and observed incidences for selected prior diseases. Same data as in a, but grouped for different prior diseases. c, Fold changes for the groups with and without prior diseases shown in b. d, Delphi accurately stratifies trajectories into low-, mid- and high-risk groups for selected diagnoses and death. Cumulative incidence (y-axis) as a function of age (x-axis). Risk groups are based on the top 1% and bottom 5% risk at the age of 60 years when simulations started. The low-risk group percentile was chosen to be larger to include sufficient cases for evaluation. Orange curves denote Delphi-2M simulations, blue observed data. Shown are analyses for 10 selected diseases, as stated in the titles. Cox proportional hazard coefficients (y-axis) are estimated using a proportional hazards model with parameter regularization, resulting in a high number of zero coefficients. The non-zero Cox coefficients and SHAP show a high correlation. b. UMAP scatter plot, coloured by the SHAP disease rate change for the disease of interest, denoted by a cross marker. According to the SHAP analysis, diseases with similar embeddings tend to have a greater effect on the predicted rate of each other. Non-random missingness may cause biases in predictions even when sources are not explicitly provided to the model. Disease embedding UMAP for a Delphi model with explicit token sources (e.g. “Common cold (self-reported)” and “Common cold (hospital records)” are separate tokens), tokens coloured by ICD-10 chapters. Same as a, coloured by token source. c. Same as a, but for the standard Delphi-2M model. Only tokens with more than 75% of all entries from one source are shown. d. Same as c, coloured by primary token source. e. SHAP value matrix (similar to Fig. 4c), with tokens grouped by chapter and primary source. a, Modelled rate per year separated by sex and ethnic background. b, Modelled rate per year separated by sex and Townsend deprivation index bins (increasing for greater deprivation index values). c-d, Average number of disease tokens per year, shown for different ethnicities (c) and deprivation indices (d). e-f, Age and sex stratified AUCs for 10 selected diseases. AUCs for individual age and sex brackets are shown as grey dots. 95% confidence intervals are calculated using DeLong's method. g-h, Width of DeLong's 95% confidence intervals for AUC vs number of cases, shown for different ethnicities and deprivation strata. For rare diseases, AUC estimates have high variance. i, Standard deviation between AUC estimates for different strata vs number of cases of this disease for the training dataset. j, Average validation AUC across 5-year age groups ranging from 40 to 80 years of age, aggregated by the corresponding ICD chapters. Difference between average AUCs calculated for participants with birth years before 1944 and after 1960. Dataset information and AUC values for individual diseases. Top, change in cross-entropy of disease token predictions relative to a baseline derived from age (in years) and sex incidence. This table contains classification metrics (ROC-AUC, average precision score and Brier score), evaluated for the tasks of predicting death, cardiovascular diseases and dementia for Delphi and other methods, including clinical risk scores and machine learning models. This table contains list of diseases for which Delphi may be biased towards certain population subgroups, with their AUC values for all subgroups (Methods). This table contains UKB field IDs for all fields used in the study, including general participant information (sex, lifestyle), first disease occurrences and additional information, used for comparisons to other methods. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Shmatko, A., Jung, A.W., Gaurav, K. et al. Learning the natural history of human disease with generative transformers. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09561-3'>A room temperature rechargeable all-solid-state hydride ion battery</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 15:17:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article As a negative charge carrier, the hydride ion (H−) is more energetic, polarizable and reactive than cations1. An H−-mediated electrochemical process is fundamentally different from existing systems and enables the development of innovative electrochemical devices, such as rechargeable batteries, fuel cells, electrolysis cells and gas separation membranes2. Here we developed a core-shell hydride 3CeH3@BaH2, which exhibits fast H− conduction at ambient temperature and becomes a superionic conductor above 60 °C. This hydride allows us to construct an all-solid-state rechargeable H− battery CeH2|3CeH3@BaH2|NaAlH4, which operates at ambient conditions using NaAlH4 and CeH2 as cathode and anode materials, respectively. This battery has an initial specific capacity of 984 mAh g−1 and retains 402 mAh g−1 after 20 cycles. Using hydrogen as charge carriers can avoid the formation of detrimental metal dendrites, in principle, which creates new research avenues for clean energy storage and conversion. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout Source data are provided with this paper. Yamaguchi, S. Large, soft, and polarizable hydride ions sneak around in an oxyhydride. Zhang, W., Cao, H. & Chen, P. Hydride ion conductor: a key material for innovative energy storage and conversion. Verbraeken, M. C., Cheung, C., Suard, E. & Irvine, J. T. S. High H− ionic conductivity in barium hydride. Zhang, W. et al. Deforming lanthanum trihydride for superionic conduction. Irvine, G. J., Smith, R. I., Jones, M. O. & Irvine, J. T. S. Order–disorder and ionic conductivity in calcium nitride-hydride. Kobayashi, G. et al. Pure H− conduction in oxyhydrides. Fukui, K., Iimura, S., Iskandarov, A., Tada, T. & Hosono, H. Room-temperature fast H− conduction in oxygen-substituted lanthanum hydride. Takeiri, F. et al. Hydride-ion-conducting K2NiF4-type Ba–Li oxyhydride solid electrolyte. Izumi, Y. et al. Electropositive metal doping into lanthanum hydride for H− conducting solid electrolyte use at room temperature. Ubukata, H. et al. Anion ordering enables fast H− conduction at low temperatures. Huiberts, J. N. et al. Yttrium and lanthanum hydride films with switchable optical properties. Vajda, P. in Handbook on the Physics and Chemistry of Rare Earths Vol. 207–292 (eds Gschneidner Jr, K. A. Fukui, K. et al. Characteristic fast H− ion conduction in oxygen-substituted lanthanum hydride. Wang, H. L. et al. Semiconductor heterojunction photocatalysts: design, construction, and photocatalytic performances. Capasso, F. Band-gap engineering: from physics and materials to new semiconductor devices. Verbraeken, M. C., Suard, E. & Irvine, J. T. S. Structural and electrical properties of calcium and strontium hydrides. Rowberg, A. J. E., Weston, L. & Van de Walle, C. G. Ion-transport engineering of alkaline-earth hydrides for hydride electrolyte applications. Jain, A. et al. Commentary: the Materials Project: a materials genome approach to accelerating materials innovation. Li, Y. et al. Core–shell nanostructured magnesium-based hydrogen storage materials: a critical review. Gao, Y. et al. Amorphous dual-layer coating: enabling high Li-ion conductivity of non-sintered garnet-type solid electrolyte. He, X., Zhu, Y. Origin of fast ion diffusion in super-ionic conductors. & Klebanoff, L. in Hydrogen Storage Technology: Materials and Applications (ed. Orimo, S., Nakamori, Y., Eliseo, J. R., Züttel, A. & Jensen, C. M. Complex hydrides for hydrogen storage. Grochala, W. & Edwards, P. P. Thermal decomposition of the non-interstitial hydrides for the storage and production of hydrogen. Bogdanović, B. et al. Investigation of hydrogen discharging and recharging processes of Ti-doped NaAlH4 by X-ray diffraction analysis (XRD) and solid-state NMR spectroscopy. & Schwickardi, M. Ti-doped alkali metal aluminium hydrides as potential novel reversible hydrogen storage materials. Ren, Z.-H. et al. Single Ti atoms coupled with Ti–O clusters enable low temperature hydrogen cycling by sodium alanate. Liu, W., Liu, P. & Mitlin, D. Tutorial review on structure – dendrite growth relations in metal battery anode supports. Conder, K. & Kaldis, E. High accuracy volumetric determination of hydrogen in rare-earth hydrides. Blöchl, P. E. Projector augmented-wave method. Efficient iterative schemes for ab initio total-energy calculations using a plane-wave basis set. Efficiency of ab-initio total energy calculations for metals and semiconductors using a plane-wave basis set. Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Anisimov, V. I., Aryasetiawan, F. & Lichtenstein, A. I. First-principles calculations of the electronic structure and spectra of strongly correlated systems: the LDA + U method. Dudarev, S. L., Botton, G. A., Savrasov, S. Y., Humphreys, C. J. & Sutton, A. P. Electron-energy-loss spectra and the structural stability of nickel oxide: an LSDA + U study. Augustyn, V. et al. High-rate electrochemical energy storage through Li+ intercalation pseudocapacitance. Binnewies, M. & Milke, E. Thermochemical Data of Elements and Compounds 2nd edn (Wiley, 2002). Rowberg, A. J., Weston, L. & Van de Walle, C. G. Ion-transport engineering of alkaline-earth hydrides for hydride electrolyte applications. Priyanga, G. S., Rajeswarapalanichamy, R. & Iyakutti, K. First principles study of structural, electronic, elastic and magnetic properties of cerium and praseodymium hydrogen system REHx (RE: Ce, Pr and x = 2, 3). & Tugushev, V. V. An electronic band structure model for the metal-semiconductor transition in cerium-group hydrides. We thank R. Wu for TOF-SIMS support; Q. Jiang and Y. Zhao for HRTEM analysis; and C. Xue for S-XRD assistance. This work was jointly supported by the Liaoning Binhai Laboratory (grant no. LBLE-2023-01), the Science and Technology Major Project of Liaoning Province (grant no. 2024JH1/11700013), the National Natural Science Foundation of China (grant nos. 22309175, 21988101, 22279130 and 21633011), the Dalian Institute of Chemical Physics (DICP I202330) and the Dalian Science and Technology Innovation Fund (2023RJ016). These authors contributed equally: Jirong Cui, Ren Zou Dalian Institute of Chemical Physics, Chinese Academy of Sciences, Dalian, People's Republic of China Jirong Cui, Ren Zou, Weijin Zhang, Hong Wen, Shangshang Wang, Shukun Liu, Hetong Chen, Wei Liu, Xiaohua Ju, Jianping Guo, Teng He, Hujun Cao & Ping Chen Center of Materials Science and Optoelectronics Engineering, University of Chinese Academy of Sciences, Beijing, People's Republic of China Jirong Cui, Shangshang Wang, Shukun Liu & Hetong Chen Department of Chemical Physics, University of Science and Technology of China (USTC), Hefei, People's Republic of China Institute of Theoretical Chemistry, College of Chemistry, Jilin University, Changchun, People's Republic of China Shanghai Advanced Research Institute, Chinese Academy of Sciences, Shanghai, People's Republic of China Weiwei Wang, Tao Gan & Jiong Li State Key Laboratory of Catalysis, Dalian, People's Republic of China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project. J.C. synthesized the hydride ion conductor, P.C., J.C. and R.Z. performed the structural characterization and electrochemical testing. performed the DFT calculations. and W.W. conducted the HRTEM, NMR and S-XRD measurements, respectively. conducted the XAS measurements. repeated material synthesis, battery assembly and electrochemical testing. discussed and analysed the results. Correspondence to Weijin Zhang, Hujun Cao or Ping Chen. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Hydrogen contents of the samples. *The hydrogen contents were determined by acidolysis of MHx using a hydrochloric acid solution. b, XRD patterns of the as-prepared BaH2 and CeH3. c, XRD patterns of the nCeH3-BaH2 (n = 2, 3, 4, 5) samples. A certain amount (50% (V/V)) of silicon powder was added into the samples as an internal standard to accurately identify the phases of the mixtures. The main phase of all the nCeH3-BaH2 samples is the cubic CeH3 \(({Fm}\overline{3}m)\). The phase of BaH2 is not obvious even in the samples with higher BaH2 contents. d, XRD patterns of the 3CeH3-BaH2 samples ball milled for different time. As the milling time increases, the BaH2 phase weakens and disappears, while the crystalline phase of CeH3 retains. e, Diffuse reflectance infrared Fourier transform (DRIFT) spectra of BaH2, CeH3 and 3CeH3-BaH2 samples. The vibration bands centred at 1170 and 915 cm−1 are corresponding to Ce-H bonds and the bands centred at 1075 and 775 cm−1 belong to BaH2. After ball milling, the signals of BaH2 preserve. Noted that DRIFT is a surface sensitive technique. Although the molar ratio of BaH2 accounts only for 1/4 of the sample, the signal of Ba-H is relatively strong, indicating that BaH2 could be enriched on the surface of the material. a,b, SEM images of the ball milled 3CeH3-BaH2 and CeH3 samples, respectively. The 3CeH3-BaH2 sample has a larger particle size, mainly in the rage from 0.5 to 1 μm. The particle size of the ball milled CeH3 is smaller, mainly in 0.2 to 0.6 μm. c, SEM image and corresponding EDS mapping of the 3CeH3-BaH2 sample. The green and red dots represent cerium and barium, respectively. In the micrometre scale, Ba and Ce are uniformly distributed. d, HRTEM images of the 3CeH3-BaH2 sample. Red parallel lines mark the observed crystal plane spacings. These crystal plane data match well with CeH3 crystals \(({Fm}\overline{3}m)\), no crystallines related to BaH2 (Pnma) can been found. e, TEM image of the 3CeH3@BaH2 sample and its corresponding SAED pattern. No crystalline BaH2 was detectable (if there exists BaH2 crystal in the particle, the innermost ring radius of the diffraction pattern should be much smaller because the crystal plane spacing of orthorhombic BaH2 is larger than 0.32 nm). f, TEM and EELS mapping of the 3CeH3@BaH2 sample. Other figures show the corresponding distribution of Ba and Ce elements. It is evident that particles with core-shell structures are clustered together. a, Structure model of CeH3 (111)/BaH2 (010) heterojunction and the partial density of states (PDOS) of BaH2 (010) and CeH3 (111). The average bond lengths of Ba-H in BaH2, Ce-H in CeH3, and the Ba-H and Ce-H bonds at the interface are 2.77, 2.36, 2.61 and 2.49 Å, respectively. b, Charge density difference (Δρ) and planar-averaged along the z-axis for the CeH3-BaH2 heterojunctions, showing electron accumulation (yellow) and depletion (blue) across the interface. c, PDOS of the CeH3 (111) slab calculated with PBE + U, showing states near the Fermi level with a small band gap of ~0.12 eV. d, PDOS of the BaH2 (010) slab calculated with HSE06. This slab has a band gap of 3.27 eV that is slightly lower than the value (3.85 eV) of bulk material. These findings indicate an interfacial electron transfer from BaH2 to CeH3. a, XPS spectra of the Ce 3 d for various samples. b, XPS spectra of the Ba 3d5/2 for various samples. Ba of the sample prepared by hand mixing (denoted by 3CeH3-BaH2-HM) and the neat BaH2 sample have a lower binding energy, while the binding energies of post-ball milled samples are higher. This implies that Ba atoms transfer more electrons to H atoms at the interface of the BaH2/CeH3 heterojunction of the ball milled sample. Little variation was observed for Ce among all the samples. c, Ce L3-edge XANES spectra of various samples. d, Ba L3-edge XANES spectra of various samples. Ba of 3CeH3@BaH2 has a higher peak intensity than neat BaH2 and 3CeH3-BaH2-HM samples, showing it is in a higher oxidative state. This is consistent with the XPS results. Ce, on the other hand, does not show remarked difference among the samples. a, The EIS plots at 180, 200 and 220 °C. Rbulk, resistance of bulk; Rgb, resistance of grain boundary. b, σi, σe, and ti of the nCeH3-BaH2 samples with different ratios at 20 °C. The 3:1 sample has a high ionic conductivity and the highest transfer number. c, The LSV curve of the 3CeH3@BaH2 sample in H2 (black line) and Ar (blue line) atmospheres (1.0 bar) at room temperature. Its electrochemical window is around 1.0 V. d, Arrhenius plot for the ionic conductivity of the 3CeH3@BaH2. Diffusion barrier was derived from the fitted line. The tests were conducted at current densities of 1.3 and 2.7 mA cm−2, respectively. a, The rate capability of the hydride ion battery. Because of substantial capacity fading during the first few cycles, data collection commenced from the tenth cycle. b, Cyclic voltammetry (CV) curves of the CeH2 | 3CeH3@BaH2 | NaAlH4 battery at different sweep rates. There are obvious oxidation and reduction peaks in the CV curves, and the peak positions correspond to the highest rates of charge and discharge processes. The areas belonging to the positive and negative currents are nearly the same, indicating reversibility of the electrode reactions. c, Logarithm peak current (i) versus logarithm sweep rate (v) plots to determine the b-value of redox peaks. The b values of three redox peaks were calculated to be 0.610, 0.533, and 0.508, respectively, suggesting that this battery mainly follows a diffusion-dominated process36. The peaks of the anode (CeH2) shifted significantly to higher angles after discharge, indicating an increase in hydrogen content in Ce hydride; after recharge, asymmetric peaks with a trend of moving towards smaller angles were observed, indicating a decrease in hydrogen content. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Cui, J., Zou, R., Zhang, W. et al. A room temperature rechargeable all-solid-state hydride ion battery. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09534-6'>Repeated head trauma causes neuron loss and inflammation in young athletes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 15:06:20
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Repetitive head impacts (RHIs) sustained from contact sports are the largest risk factor for chronic traumatic encephalopathy (CTE)1,2,3,4. Currently, CTE can only be diagnosed after death and the events that trigger initial hyperphosphorylated tau (p-tau) deposition remain unclear2. Furthermore, the symptoms endorsed by young individuals are not fully explained by the extent of p-tau deposition2, severely hampering therapeutic interventions. Here we observed a multicellular response prior to the onset of CTE p-tau pathology that correlates with number of years of RHI exposure in young people (less than 51 years of age) with RHI exposure, the majority of whom played American football. Leveraging single-nucleus RNA sequencing of tissue from 8 control individuals, 9 RHI-exposed individuals and 11 individuals with low-stage CTE, we identify SPP1-expressing inflammatory microglia, angiogenic and inflamed endothelial cells, astrocytosis and altered synaptic gene expression in those exposed to RHI. We also observe a significant loss of cortical sulcus layer 2/3 neurons independent of p-tau pathology. Finally, we identify TGFβ1 as a potential signal that mediates microglia–endothelial cell cross talk. These results provide robust evidence that multiple years of RHI is sufficient to induce lasting cellular alterations that may underlie p-tau deposition and help explain the early pathogenesis in young former contact sport athletes. Furthermore, these data identify specific cellular responses to RHI that may direct future identification of diagnostic and therapeutic strategies for CTE. Each year, millions of individuals are exposed to RHIs through contact sports, military service and domestic violence. These RHIs are often non-symptomatic and non-concussive, and can occur thousands of times per year, over the course of decades in some cases. CTE, a progressive tauopathy caused by exposure to RHI1,2, is observed in individuals as young as 17. Risk for CTE in exposed individuals is associated with the number of years of exposure to RHI and the cumulative force of the hits endured3,4. Although much of the current research is focused on severe CTE in older individuals, a recent case series of 152 brains from donors under the age of 30 identified 63 brains with CTE, highlighting that RHI-driven disease is a pressing concern in the young population2. Currently, CTE can only be diagnosed post-mortem through identification of p-tau aggregates in neurons around blood vessels at the depth of the cortical sulcus. Our previous research suggests that microglia-mediated neuroinflammation occurs prior to the deposition of p-tau5. Other work has demonstrated that RHI exposure is associated with astrocytic activation, white matter inflammation and damage, blood–brain barrier (BBB) breakdown, serum protein leakage and increases in vascular density in the CTE brain5,6,7,8,9. These cellular changes occur prior to overt neurodegeneration and are likely to drive many of the early clinical impairments that are not explained by the occurrence and extent of p-tau pathology. However, studies examining the full extent of these cellular phenotypes have been limited. A detailed characterization of the early cellular changes in young RHI-exposed athletes is necessary to understand the pathogenic mechanisms in CTE and to identify novel biomarkers or therapeutic targets relevant to early disease stages. To identify the earliest RHI-driven changes, we performed single-nucleus RNA sequencing (snRNA-seq) using autopsy-confirmed frozen human brain tissue from 28 young individuals. Eight non-RHI-exposed controls, 9 RHI-exposed individuals without CTE pathology (RHI-only) and 11 RHI-exposed individuals with diagnosed CTE stage 1 or 2 (CTE) were included (Fig. Of the individuals with RHI exposure, all but one individual played American football (hereafter referred to as football). The remaining individual played soccer (Supplementary Table 2). CTE diagnosis was performed by a neuropathologist and based on the presence of CTE pathognomonic p-tau lesions10 (Fig. Grey matter sulcus from the dorsolateral frontal cortex, one of the first brain regions affected in CTE, was processed for snRNA-seq (Fig. After quality control and filtering, 170,717 nuclei of sufficient quality were clustered into 31 initial clusters and labelled on the basis of their expression of known cell-type markers11,12 (Fig. All major cell types were identified. Compositional analysis with scCODA demonstrated no significant differences in cell-type abundance across pathological groups13 (Fig. Out of all major cell types, minimal RHI-associated changes were observed in oligodendrocytes and oligodendrocytes precursor cells (Extended Data Fig. We thus elected to focus further analyses on microglia, astrocytes, endothelial cells and neurons, consistent with previous studies5,7,8,14. The workflow was run once for each sample. FACS, fluorescence-activated cell sorting; GEM, gel bead in emulsion. c, Uniform manifold approximation and projection (UMAP) analysis of nuclei from all donors labelled for cell type based on cell-type marker expression. d, Expression of cell-type markers across cell-type clusters in c. Astro, astrocytes; Endo, endothelial cells; Exc, excitatory neurons; Inh, inhibitory neurons; Micro, microglia; Oligo, oligodendrocytes. e, Stacked bar plot of pathological group fractions within cell-type clusters. f, Stacked bar plot of cell-type counts coloured by pathological group. On the basis of previously demonstrated involvement of microglial inflammation in CTE and its important role in neurodegeneration, we examined changes in microglial gene expression5. Analysis of 6,863 microglial cells revealed 11 unique clusters (Fig. The microglia cluster size is consistent with other published studies and believed to be appropriately powered11. Cluster 10 contained 263 cells and expressed the perivascular macrophage genes CD163, F13A1 and LYVE1, and cluster 6 was composed of 108 cells that expressed the peripheral monocyte genes PTPRC, LYZ and CR1 as previously observed11,15,16 (Fig. b, Heat map of selected cluster DEGs annotated by function. c, Proportion of microglial subtypes per pathological group. Statistical analysis was performed using a chi-squared test. d, UMAP of each pathologic group. The dotted line depicts RHIM2/3 subtypes. e, hdWGCNA module analysis showing the Turquoise module localization to the RHIM2/3 subtype. f, GO analysis of the hdWGCNA Turquoise module. Statistics generated using gene set enrichment analyses (GSEA) and single-tailed hypergeometric test with Benjamini–Hochberg multiple hypothesis correction. ER, endoplasmic reticulum; miRNA, micro RNA; PID, Pathway Interaction Database. g–l, Violin plots of the Celda gene modules homeostasis (g), complement response (h), inflammation (i), hypoxia response (j), hypoxia response (k) and metabolic process (l). m, Quantification of grey matter sulcal homeostatic microglia (P2RY12hiIBA1+) with years of football play, coloured by pathological group. Statistical analysis performed by linear regression with age as a covariate. Each dot represents an individual donor. n, Representative image of P2RY12 immunofluorescent labelling. Open arrowheads depict cells exhibiting high P2RY12 expression. Solid arrowheads depict cells exhibiting low P2RY12 expression. Solid arrowheads indicate triple-positive cells. p,q, Quantification of the SPP1+HIF1A+ microglial fraction (p) and microglial SPP1 expression (q) in the grey matter sulcus with years of football play. Each dot represents an individual donor. Coloured by pathological group status. Statistical analysis performed by linear regression. Homeostatic clusters were significantly enriched for nuclei from control individuals compared with RHI-only or CTE individuals, but there was no significant difference between RHI-only and CTE individuals (control versus RHI-only: P = 0.048; control versus CTE: P = 0.047; RHI-only versus CTE: P > 0.99; Extended Data Fig. Homeostatic microglial proportion decreased with increasing years of football play (P = 0.004, β = −12.79). Cluster 7 highly expressed CD83, CCL3 and HSP90AA1, reminiscent of a possible pro-resolving phenotype that was recently identified in Alzheimer's disease17. Cluster 4 had the highest differential gene expression of AIF1 (encoding IBA1) across clusters and was characterized by expression of the iron-associated genes FTL and FTH (also known as FTH1), and ribosome-associated genes such as RPS24 and RPS11 (Fig. The proportion of microglial subpopulations found in RHI-only and CTE individuals were significantly different from controls, with the emergence of clusters 1, 5 and 8 in RHI-only and CTE individuals (Fig. For simplicity, these clusters were labelled RHI microglia (RHIM) 1 to 3. We performed gene module analysis with hdWGCNA and Celda to identify co-expression of possible cellular pathways across subclusters and linear mixed modelling statistical analysis to compare Celda gene module expression (Fig. The full list of differentially expressed genes (DEGs) and gene modules can be found in Supplementary Tables 9 and 17. The expression of homeostasis-associated gene modules was significantly decreased in RHIM2 and RHIM3 (Fig. Cluster 5, RHIM1, expressed neuronal-associated genes such as GRID2, GRIK2 and GRIA4, with top identified Gene Ontology (GO) terms including ‘synapse organization' (Fig. Previous work has found that satellite microglia (microglia that closely contact neurons) increase in number following TBI and modulate neuronal firing activity18. Cluster 1, RHIM2, was nearly evenly enriched for RHI and CTE samples (50% versus 46%, respectively), whereas cluster 8, RHIM3, were mostly enriched for CTE samples (83%). Transcriptionally, RHIM2 and RHIM3 were similar, displaying features of an inflammatory microglial phenotype with expression of SPP1, HIF1A, TLR2, IL1B and CTSB (Fig. SPP1 has been described as a general marker of inflammatory or activated microglia, and has a potential role in synaptic engulfment in Alzheimer's disease models19. SPP1 has also been described as an opsin for extracellular debris20,21. hdWGCNA module analyses identified gene sets that were enriched in RHIM2 and RHIM3 that were strongly associated with immune signalling (Fig. Further GO analysis of RHIM2 and RHIM3 DEGs identified ‘cytokine signalling in the immune system', ‘positive regulation of immune response' and ‘vesicle mediated transport' (Extended Data Fig. Celda gene module analysis demonstrated an increase in inflammation, hypoxia and metabolic response in RHIM2 and RHIM3 compared with homeostasis clusters (Fig. 2h–l), providing orthogonal validation of GO and DEG analyses. Some key differences were noted between RHIM2 and RHIM3. RHIM2 expressed C1QA, C1QB, C1QC and CAMK2D, which encode the components and downstream effector of the C1q complement cascade that is known to drive aberrant synaptic engulfment in the neurodegenerative brain22 (Fig. Gene module analysis further highlighted an increase in complement response in RHIM2 compared with homeostatic microglia (Fig. RHIM3 was characterized by upregulation of HIF1A and VEGFA, two central mediators of hypoxia, suggesting a potential response to or initiation of hypoxic conditions following RHI (Extended Data Fig. HIF1A also acts as a transcriptional regulator of many downstream inflammatory genes, and analysis of the transcriptional regulatory networks enriched in each cluster showed that RHIM3 expressed many genes regulated by HIF1A23 (Extended Data Fig. To validate the reduction in the homeostatic microglial population, IBA1 and P2RY12 were co-immunolabelled and quantified in the sulcus of 35 individuals with 0 to 25 years of football play with or without CTE. Microglia were divided according to high or low P2RY12 expression. Homeostatic microglial densities (P2RY12hiIBA1+) were significantly decreased with increasing years of football play (P < 0.001; Fig. Concurrently, non-homeostatic microglia (P2YR12lowIBA1+) cells were positively correlated with increasing years of football play (P < 0.001; Extended Data Fig. Mirroring the snRNA-seq results, CTE status was not significantly associated with homeostatic microglial densities when years of exposure were accounted for. To verify the presence of RHIM2 and RHIM3 cells and their relationship to pathology, we performed in situ hybridization to label microglia that express the RHIM2 and RHIM3 marker genes SPP1 and HIF1A (Fig. P2RY12 was used as a marker for microglia as AIF1 (encoding IBA1) exhibits low expression at the mRNA level, as evidenced in a previous study23 and the present snRNA-seq data. We quantified SPP1-expressing and HIF1A-expressing (SPP1+HIF1A+) microglia across 21 individuals with 2–25 years of football play with and without CTE (Fig. The number of SPP1+HIF1A+ microglia in the cortical sulcus increased significantly with increasing years of football play (P = 0.028; Fig. There was no association between SPP1+HIF1A+ microglia in the nearby cortical crest, suggesting a regional specificity of this inflammatory phenotype (P = 0.53; Extended Data Fig. We determined the layer specificity of SPP1+HIF1A+ microglia, separating superficial and deep layers of the cortical sulcus. The number of SPP1+HIF1A+ microglia increased in both superficial layers 2–3 (P = 0.039) and deeper layers 4–6 (P = 0.026) (Extended Data Fig. This suggests that although the microglial inflammation is specific to the sulcus, there was no layer-wise specificity of this phenotype. Additionally, microglia increased expression of SPP1 with increasing years of football play (P = 0.035; Fig. CTE status (P = 0.34) and tau burden (P = 0.12) did not associate with the prevalence of SPP1+HIF1A+ microglia. Next, we sought to compare our microglial populations to those described in published datasets; notably, Sun et al.23 published a dataset with more than 100,000 microglia from more than 400 individuals. First, we performed Jaccard similarity scoring and confirmed significant correlation of the RHIM2 and RHIM3 clusters with inflammatory, stress, phagocytic and glycolysis-associated gene sets23 (Extended Data Fig. Next, we projected our data onto the dataset and performed bootstrapping to predict labels and confidence scores to better analyse the consistency of subtypes (Extended Data Fig. Overall, we observed significant similarity between select subpopulations, further validating our subtypes. Overall, these results suggest that RHI exposure induces an increase in neuronal surveillance and inflammatory microglial transcriptomic states before the onset of CTE. Inflammatory microglia are localized specifically at the sulcus in RHI-exposed individuals. These microglia may be involved in the initiation and maintenance of neuronal dysfunction, inflammation and angiogenic processes that are present in CTE. Astrocytes have a key role in brain homeostasis in tasks such as neuronal and BBB maintenance and become reactive following RHI exposure and in neurodegenerative disease14,24. We identified four subtypes of astrocytes, Astro1–Astro4, based on stratification of pathological group identity, DEG analysis and gene module analysis (Extended Data Fig. Each subtype was also confirmed to be expressed in the cortical grey matter (Extended Data Fig. Although past work has suggested the importance of astrocytes in CTE, we observed a limited astrocytic response, with only one subtype being enriched for individuals with RHI (Astro3). These findings suggest that although there is pronounced astrogliosis associated with end-stage CTE pathology, astrocytes might have a more subtle role in early disease. Next, owing to the key involvement of vascular dysfunction in CTE, we characterized the vascular response to RHI exposure8,9 (Extended Data Fig. We used known cell-type markers and comparison with published dataset markers to identify 1,762 endothelial cells, 913 pericytes, 487 fibroblasts and 651 vascular smooth muscle cells25 (Extended Data Fig. Of these cell types, only fibroblasts displayed significant changes in total proportion across pathological groups, decreasing from controls to RHI (P = 0.048) and CTE (P = 0.027), with loss associated with years of football play (Extended Data Fig. Endothelial cells were further labelled for arterial, venous and capillary cells by comparing expressed genes with published datasets25 (Extended Data Fig. Capillary cells were then labelled Cap1–Cap4. Cap3 and Cap4 (Seurat Cluster 5; Extended Data Fig. 7b,c) exhibited a slightly different transcriptomic profile with higher expression of collagen-associated genes and showed overlap in expression of pericyte genes, thereby representing a potential transitional cell state but with greatest fidelity to endothelial cell expression (Extended Data Fig. The proportion of endothelial cell subtypes differed significantly between RHI and control individuals and trended towards a difference between control and CTE individuals (Fig. No difference was observed between RHI and CTE individuals (Fig. Two populations of capillary cells, Cap2 (P = 0.0043) and Cap4 (P = 0.0046) were enriched for RHI and CTE samples (Extended Data Fig. The Cap2 cell fraction also increased with increasing years of football play (P = 0.014; Extended Data Fig. No differences were observed in total capillary cells in RHI and CTE compared with controls (Extended Data Fig. Several canonical angiogenesis-associated genes such as HIF1A, ANGPT2, ANGPTL4, STAT3, CAMK2D and NFKBID were significantly upregulated in Cap2 and Cap4, suggesting that capillary cells in RHI-exposed groups may be responding to a local hypoxic environment (Fig. Three major complement regulatory genes—CD59, CD55 and CD46—which inhibit complement-mediated cell lysis, were upregulated indicating a potential response to locally increased levels of complement (Fig. Expression of the vascular adhesion and transmigration-associated genes ICAM1, ICAM2, PECAM1 and CD99 was increased in Cap2 and Cap4, indicating an increased potential for entry of monocytes, T cells, neutrophils or other peripheral cell types across the endothelium (Fig. Cap4 also exhibited high expression of collagen genes (Fig. We performed module co-expression analysis using Celda and hdWGCNA to identify co-expressed genes and possible cellular pathways across endothelial subsets (Supplementary Figs. Statistical linear mixed modelling demonstrated that modules related to immune signalling, angiogenesis, response to growth factors and collagen were significantly upregulated in Cap2 and Cap4 subsets (Fig. hdWGCNA also identified enrichment of collagen-associated genes in Cap4 as additional validation (Supplementary Table 18c and Supplementary Fig. GO analysis identified VEGFA signalling, cytokine signalling and vasculature development as significantly upregulated terms in RHI-exposed endothelial cells (Extended Data Fig. We identified ITGAV as an endothelial gene that was significantly increased in Cap2 cells and in RHI compared with control and CTE samples (Fig. To confirm its expression in the tissue, we performed in situ hybridization paired with GLUT1 immunohistochemistry to label vessels, and found an increase in the fraction of vessels expressing ITGAV with increasing years of football play (P = 0.027; Fig. Together, these data show that capillary cells undergo significant upregulation of angiogenesis and inflammation-associated genes along with an increase in basement membrane components, identifying pathways that may underlie the known microvascular dysfunction after RHI and in CTE8,9. Solid arrowheads indicate clusters that are enriched in RHI and CTE. b, Stacked bar plots of capillary subtype abundance across pathological groups. Statistical analysis performed using a chi-squared test. c, Dot plot of selected DEGs that are upregulated in RHI and CTE across endothelial subtypes, annotated for function. d, Violin plots of Celda module expression across capillary subtypes. Black bars indicate the median statistic from ggsignif. Statistical analysis performed with linear mixed effects model, accounting for sample variability and comparing Cap2 and Cap3 with Cap1. e, Violin plot of ITGAV expression across pathological groups. Each dot represents a cell. Statistical analysis performed by Wilcoxon test from ggsignif. f, Scatter plot of the ITGAV+ vessel fraction in the grey matter sulcus with years of football play, coloured by pathological group status using in situ hybridization. Each dot represents an individual donor. Statistical analysis performed by linear regression. g, Representative image of an ITGAV+ vessel. Each individual sample was stained and analysed once. Next, owing to the known dysfunction and degeneration of neurons and synaptic dysfunction following head trauma and in neurodegenerative disease, we examined neurons, labelling subclusters using known layer-specific markers12,26,27,28,29,30,31 (Fig. Forty-seven per cent of excitatory neuron DEGs were shared across RHI and CTE when compared with control, and only 6% were different between RHI and CTE, suggesting that the greatest changes in excitatory neuronal transcriptional profiles occur with initial exposure to RHI (Fig. Comparison of gene expression in RHI and CTE with controls and GO analysis of total neuronal population and layer-specific DEGs showed enrichment of ‘modulation of chemical synapses' and ‘cell–cell adhesion' processes in both analyses (Fig. Genes associated with synaptic transmission such as SYN3, SNAP91, NRG1, HSP12A1 (a member of the Hsp70 gene family), and genes encoding matrix binding proteins such as CNTN5 and CLSTN2 were upregulated across several excitatory neuron layers. There were 40% fewer layer-specific DEGs in inhibitory neurons than in excitatory neurons, with only 184 DEGs specific to RHI-exposed groups compared with controls. GO analysis of inhibitory neuron layer-specific DEGs showed common upregulation of synapse-associated genes such as SYN3 and SYN2 across layers and downregulation of GABA receptor gene GABRA1 (Extended Data Fig. a, UMAP analysis of excitatory neurons coloured and labelled by layer subtype. b, Venn diagram depicting the overlap between DEGs from RHI versus control, CTE versus control and RHI versus CTE comparisons. Statistics generated using GSEA and single-tailed hypergeometric test with Benjamini–Hochberg multiple hypothesis correction. d, Bar plot representing cell counts per pathological group for each excitatory neuron layer subtype. Statistical analysis performed by ordinary one-way analysis of variance (ANOVA) with Bonferroni correction. e, Representative density heat map of CUX2+LAMP5+ cells from in situ hybridization. Solid arrowheads indicate depth of the cortical sulcus. f, Scatter plot showing the fraction of CUX2+LAMP5+ neurons among total excitatory neurons in snRNA-seq data with total years of football play, coloured by pathological group identity. g, Scatter plot showing cell density of CUX2+LAMP5+ neurons in sulcal layer 2/3 from in situ hybridization coloured by pathological group identity, with years of football play. Statistics performed by general linear regression. h, Representative images of Nissl-stained neurons in superficial cortical layer 2/3. i, Scatter plot showing Nissl-stained neuronal densities across football exposure groups. Since neurodegenerative processes and exposure to head trauma can be associated with neuronal loss, we investigated layer-specific cell composition in individuals with RHI and CTE compared with controls. No pathological group enrichment was found in inhibitory neurons (Extended Data Fig. However, differential abundance analysis of excitatory layer 2/3 CUX2+LAMP5+ neurons demonstrated a significant decrease in individuals with a history of RHI, regardless of CTE status (Fig. These results were confirmed via multinomial Dirichlet multinomial regression to account for the compositional nature of snRNA-seq data13. RHI exposure individuals had an average of 56% fewer CUX2+LAMP5+ neurons than age-matched unexposed controls. When measured by proportion of total neurons, loss of CUX2+LAMP5+ neurons were also observed when comparing RHI (P = 0.0045) and CTE (P = 0.0269) individuals with controls (Extended Data Fig. Neuronal loss was associated with the number of years of playing football, or in the few cases with other types of contact sports play, total years of RHI exposure, independent of age at death (P < 0.001; Fig. To determine the spatial localization of the neuronal loss and validate the snRNA-seq results, we performed quantitative histology with RNAScope in situ hybridization using the excitatory layer 2/3 neuron markers CUX2 and LAMP5. CUX2+LAMP5+ neuronal density at the sulcus was negatively associated with years of football play (P = 0.007, β = −4.92) and highest level of football played (P = 0.033, β = −25.34) (Fig. CUX2+LAMP5+ cell density was significantly lower at the depth of the cortical sulcus compared with the nearby gyral crest, consistent with RHI-specific damage and CTE pathology32 (Fig. CUX2+LAMP5+ cell densities at the crest were not associated with years of play, demonstrating a regional specificity of neuronal cell loss to the sulcus (P = 0.686; Extended Data Fig. CUX2+LAMP5− (putatively CUX2+COL5A2+) neurons were found intermixed with CUX2+LAMP5+ neurons throughout layers 2–4 and are putatively exposed to similar levels of mechanical forces due to adjacent anatomical location. However, neuronal loss was observed to be specific to CUX2+LAMP5+ excitatory neurons across in situ and snRNA-seq experiments, suggesting a specific susceptibility of this population to RHI exposure (Extended Data Fig. To further validate the association between years of football play and neuronal loss, we determined total neuronal densities using Nissl staining of 86 young individuals with 0–28 years of football play. Individuals were grouped by 0, 1–4, 5–14 and 15 or more years of football play based on previously defined thresholds for CTE risk3. Layer 2/3 sulcal neuronal density significantly decreased with increased binned years of football play, independent of age at death (P = 0.028, β = −13.09; Fig. No association was found between years of football play and neuronal densities in deeper layers 4–6 (P = 0.554) or in layer 2/3 (0.571) in the crest (Extended Data Fig. As p-tau deposition has been shown to associate with neuronal loss in neurodegenerative disease, we compared neuronal densities with p-tau pathology in adjacent sections. We found no association between neuronal loss and p-tau deposition, suggesting that neuronal loss occurs prior to and independent of pathologic protein deposition in early stages of disease (in situ hybridization: P = 0.387; Nissl staining: P = 0.825; Extended Data Fig. Microglia contribute and respond to neuronal loss33. To investigate potential relationships between the observed neuronal loss and loss of microglial homeostasis, layer-wise homeostatic microglial populations (P2RY12hiIBA1+) from adjacent histological sections were compared to neuronal densities. Neuronal densities were significantly positively associated with homeostatic microglial populations in layers 2/3 (P = 0.047, β = 0.126; Extended Data Fig. By contrast, in layer 4–6 neuronal densities were not associated with homeostatic microglia populations (P = 0.105; Extended Data Fig. 3p), suggesting that loss of microglial homeostasis may be specifically localized to regions of neuronal loss. Overall, these results provide evidence that exposure to RHI alone may drive significant neuronal loss and dysfunction, which may help explain early symptom onset in young athletes without the presence of significant p-tau pathology. Additionally, the relationship between neuronal loss and loss of microglial homeostasis point to potential mechanisms of or responses to neuronal loss. To identify signalling pathways that may be involved in the cellular response to RHI exposure and CTE pathology, we performed ligand–receptor pair analysis using MultiNicheNet34. Two comparisons were run: RHI was compared with control (Fig. 5a, labelled RHI) to examine signalling occurring in the context of head trauma; and CTE was compared with RHI (Fig. 5a, labelled CTE) to investigate what signalling might be involved in the deposition of p-tau. In RHI-exposed individuals, microglial TGFB1 was identified as an important ligand, which signals to endothelial cells, astrocytes, neurons and other microglia through the TGFB1 receptors ITGAV, TGFBR2, TGFBR3 and TGFBR1. SIGLEC9 and SPP1 were also identified as major signalling hubs in RHI compared with control, implicating the RHIM2/3 phenotype in RHI-associated signalling. In CTE compared with RHI, the top microglial signalling pathways identified also included TGFB1 signalling and WNT2B and HLA-DRA signalling to astrocyte, microglia, endothelial cells and excitatory neurons. TGFB1 signalling has previously been implicated in the activation of neuroinflammation and induction of neuronal cell death in mild TBI35. Additionally, TGFB1 is involved with the fibrogenic response to mechanical stretch stimulus through ITGAV activation on endothelial cells and angiogenic responses through TGFBR2 signalling36,37. a, Circos plots from MultiNicheNet analysis depicting microglia as sender cells. b, RNAScope in situ hybridization depicting a TGFB1+ microglia (P2RY12; solid arrowheads) contacting a ITGAV+TGFBR2+ vessel (GLUT1; open arrowheads). c,d, Quantification of in situ hybridization of TGFB1+ microglia (c) and ITGAV+TGFBR2+ vessels (d) in the grey matter sulcus with years of football play, colour-coded by pathological group. Each dot represents an individual donor. Statistical analysis performed by simple linear regression. e, Bar plot representing ITGAV+TGFBR2+ vessels with CTE status. Statistical analysis performed using a two-tailed t-test. f, The proportion of TGFB1+ microglia within 25 μm of a ITGAV+TGFBR2+ vessel with CTE status. Statistical analysis performed using a two-tailed t-test. g, Scatter plots depicting ITGAV+TGFBR2+ vessels in the grey matter sulcus with the fraction of CUX2+LAMP5+ neurons colour-coded by pathological group. Each dot represents an individual donor. Statistical analysis performed by simple linear regression. We performed in situ hybridization analysis to label TGFB1 in microglia, as well as TGFBR2 and ITGAV, which encode two of the receptors identified in endothelial cells ligand–receptor pair analysis (Fig. We hypothesized that TGFB1+ microglia would increase in proximity to ITGAV+TGFBR2+ vessels to facilitate signalling. The prevalence of TGFB1-expressing microglia did not increase with the level of exposure (P = 0.493) or with CTE status (P = 0.493), however, we found an increase in ITGAV+TGFBR2+ vessels with increasing years of football play and with CTE status (P = 0.047) (Fig. Additionally, there was an increase in TGFB1+ microglia within 25 μm of ITGAV+TGFBR2+ vessels in CTE individuals compared with RHI-exposed individuals without CTE (Fig. The increase in microglia–endothelial cell pairs is likely to be driven by an increase in endothelial ITGAV and TGFBR2 expression as opposed to an increase in TGFB1+ microglia, concurring with data from the microglia and endothelial sections. We then compared the prevalence of ITGAV+TGFBR2+ vessels to CUX2+LAMP5+ neuronal populations in adjacent sections to identify potential relationships between the identified signalling pathway and the observed neuronal loss. We found that with decreasing neuronal populations in the grey matter sulcus, there was an increase in ITGAV+TGFBR2+ vessels (P = 0.010) (Fig. Overall, these findings identify a possible signalling pathway in the microglia–endothelial cell cross talk that may be implicated in the early pathological cascade of CTE pathology. Here we utilize a combination of snRNA-seq, multiplex in situ hybridization and immunohistological analyses to describe and validate a unique dataset of young individuals with exposure to RHI. We describe distinct microglial and endothelial subsets that emerge following RHI and persist with CTE, correlate with years of contact sport play and associate with neuronal loss. Additionally, we observed a sulcus-specific loss of cortical layer 2/3 neurons that correlated with exposure to RHI prior to p-tau deposition. Finally, we identify a possible TGFβ signalling cascade between microglia and endothelial that might drive early pathogenesis. Microglial, astrocytic and endothelial cell transcriptomic subtypes have been described in several neurodegenerative diseases and in severe traumatic brain injury, however to our knowledge, this study is the first to demonstrate such changes in a cohort of young individuals with exposure to repeated non-concussive head impacts. Of note, hypoxia-associated changes are present across these three cell types, suggesting an important role for vascular dysfunction and bolstering previous evidence of vascular remodelling in CTE. Forces from head trauma disproportionately affect blood vessels, causing a lasting endothelial response that affects BBB integrity and oxygen delivery in affected regions32. Activated endothelium, local hypoxia and a breached BBB may trigger a feedback loop, activating astrocytes and microglia with each head impact. Repeated blows to the head in short succession are likely to reactivate an already inflamed system, disallowing sufficient time for full repair, and preventing a return to homeostasis. This is substantiated by the increased microglial activation that is observed decades after retirement from contact sports and is found to correlate with the number of years of RHI exposure in this study and others5. Through this repetitive reactivation, the inflammatory response becomes self-sustaining and chronic, the mechanisms of which remain unclear. Another potential mechanism identified in this study included increased collagen expression in endothelial cells, a potential indicator of an early endothelial fibrotic response. The TGFB1–ITGAV/TGFBR2 signalling between microglia and endothelial cells that we identified through ligand–receptor analysis may represent a potential signalling pathway for the observed endothelial activation. We observed an approximately 56% decrease in superficial layer 2/3 excitatory neurons in RHI-exposed individuals at the depths of the cortical sulci, the region known to sustain the most mechanical force upon head trauma, and the region of initial p-tau accumulation in CTE32. To our knowledge, this study is the first to demonstrate such a substantial loss of a specific neuronal subtype in young individuals driven solely by RHI exposure. This is especially concerning considering that several of the observed individuals had no neuropathologic protein deposition, suggesting that neurodegeneration might begin sooner than CTE onset. Recent studies have demonstrated cortical thinning in frontal regions of high school football players, and cortical thinning and neuronal loss in post-mortem individuals with CTE38,39. Neuronal loss might explain symptoms of traumatic encephalopathy syndrome, the clinical criteria for antemortem CTE diagnosis, in young athletes2,40,41. Layer 2/3 neurons make cortico-cortical connections and, in the frontal cortex, are associated with depressive behaviours and moderation of stress42. Notably, layer 2/3 neurons have been shown to be vulnerable in other neurodegenerative and psychiatric disorders and are susceptible to p-tau accumulation in Alzheimer's disease12,27. Therefore, one may speculate that superficial layer 2/3 excitatory neurons are highly susceptible to damage regardless of source and our data captured the loss across a range of RHI doses. Vulnerability has been hypothesized to be caused by the longer projections being more susceptible to trauma-related diffuse axonal injury or the overall higher metabolic demand of these cells, however the exact mechanisms driving this susceptibility have yet to be explained. Although RHI damage is driving the early neuronal loss, it is likely that as p-tau deposition becomes more severe, neuronal death and dysfunction will become more related to pathogenic protein accumulation. One limitation of our study is the small amount of tissue in each sample. CTE is an inherently patchy disease, which is diagnosed on the basis of the presence of a pathognomonic CTE lesion consisting of a focus of perivascular p-tau accumulation at the depth of the cortical sulcus. It is therefore possible that sampling may have missed regions of important cellular responses. Future studies of RHI-exposed individuals should aim to sample from multiple areas of the brain to improve detection of cellular responses. Similarly, owing to the limited availability of donated samples from individuals under the age of 50, the current sample size is not as large as those in other recent snRNA-seq studies that might have 100 or more cases. However, as long as expression differences are robust and consistent, as observed in this study, disease-specific effects can be identified with smaller sample sizes43. However, follow-up studies to increase the sample size could help identify additional cellular heterogeneity and enhance the resolution of more subtle changes. Finally, owing to the inherent difficulties in acquiring non-disease, non-RHI-exposed post-mortem samples from young humans, some control cases included in the Nissl quantification were from female donors, which may complicate direct comparisons with samples from male athletes. However, we found no statistical correlation between sex and neuronal density. These results highlight the growing concerns linked to long-term RHI exposure from contact sports. The data presented here provide direct evidence demonstrating that RHI-driven cellular perturbations occur prior to the development of CTE and can be observed in young individuals, many of whom exhibit no obvious brain pathology. Novel biomarkers and therapeutic interventions will be vital in identifying the early changes observed in contact sport athletes prior to developing neurodegeneration. Brain tissue was obtained from the CTE and the National Center for PTSD Brain Banks. Identical intake and tissue processing procedures occur with both brain banks. Four controls included in Nissl quantification were provided by the Iowa Neuropathology Resource Laboratory. Neuropathological examination was performed by board certified neuropathologists as described previously10,44. Diagnosis of CTE was determined using published consensus criteria10,44. Institutional review board approval for brain donation and informed consent for research was obtained through the Boston University Alzheimer's Disease and CTE Center, Human Subjects Institutional Review Board of the Boston University School of Medicine, and VA Boston Healthcare System (Boston, MA). Individuals were included in the snRNA-seq and single-molecule fluorescence in situ hybridization (smFISH) experiments based on frozen tissue availability, quality (RNA integrity number (RIN) > 4) and diagnosis. Those used for immunohistochemistry were included based on the same criteria except frozen tissue availability and RIN. Exclusion criteria included neuropathological diagnosis other than CTE, moderate to severe traumatic brain injury directly prior to death, age of death greater than 51 or less than 25. Control cases did not have exposure to any RHI, were negative for any neurodegenerative disease, and did not carry any diagnosis of a neuropsychological disorder. Fresh frozen brain tissue was collected from the dorsolateral frontal cortex of each donor at the depth of the cortical sulcus. Visual delineation of grey and white matter was used to collect 50 mg of grey matter tissue. Tissue was processed and cleaned of white matter prior to homogenization at two levels. First, when removing samples from frozen coronal slabs, the unbiased technician visually inspected and avoided white matter that could be adjacent to target grey matter. Second, immediately before tissue homogenization, a second technician inspects the tissue and removes any remaining white matter. This preparation allows for a highly specific grey matter enrichment. Nuclei isolation and sorting were performed on two donor samples per day, randomizing for diagnosis and age. Tissue was kept on ice throughout nuclei isolation. Tissue was homogenized and lysed in NST Buffer with DAPI (146 mM NaCl, 10 mM Tris, 1 mM CaCl2, 21 mM MgCl2, 0.1% BSA, 0.1% NP-40, 40 U ml−1 Protector RNase Inhibitor and DAPI) and snipped with scissors on ice for 10 min. Debris was removed using a 70-μm filter. Cells were spun down and resuspended in nuclei storage buffer (2% BSA, 400 U ml−1 Protector RNase Inhibitor) to reach a concentration of 500–1,000 nuclei per μl. Nuclei were purified for DAPI-positive cells with a FACS Aria flow cytometer to remove debris and processed using the Chromium Next GEM Single Cell 3′ Reagents Kit V2 (10x Genomics) to create cDNA libraries. Samples were pooled in two batches sequenced with Azenta to a read depth of 30,000 reads per cell on an Illumina NovaSeq. CellRanger v.6.0.1 was used to align reads to the GRCH38 reference and generate filtered count matrices containing 233,555 cells across all samples. The runCellQC function in the singleCellTK R package was used to generate quality control metrics and doublet calls45,46. Contamination from ambient RNA was identified using decontx using the full raw matrix as the ‘background' for each sample47. Nuclei were removed if they had ambient RNA contamination fraction greater than 0.3, mitochondrial or ribosomal percentage greater than 5%, total counts less than 750, or genes detected less than 500. The data were not down sampled to maximize capture of rare populations. Highly variable genes were identified using runSeuratFindHVG with the method vst. Principal components were determined using runSeuratPCA. UMAP dimensionality reduction was calculated using runSeuratUMAP. Clusters across all cell types were identified using the runSeuratFindClusters function at a resolution of 0.3. After initial clustering all the cells, clusters that were predominantly doublets (>50%) were removed and produced the final dataset of 170,717 nuclei (Extended Data Fig. Associations with post-mortem interval (PMI), age at death and sequencing batch were performed using Pearson's correlation analysis in R (Supplementary Fig. Age at death was associated with only excitatory neuron L5_FEZF2_PCP4_RPRM and inhibitory neuron PVALB_SCUBE_PTPRK proportions. Therefore, age was not included in regressions performed with sequencing data. PMI correlated with only one microglial subtype (RHIM1), perivascular macrophages, an excitatory neuron subtype (L2_4CUX2_COL5A2) and several oligodendrocyte subtypes. Sequencing batch was associated with one cluster of OPCs and was therefore not included in analyses. All GO analysis was performed using MetaScape default settings49. DEG lists for all comparisons available in Supplementary Tables 6–16. Cell-type markers verified by previous human snRNA-seq studies were used to identify clusters that belonged to individual cell types (Extended Data Fig. Cell types were subsetted out using subsetSCEColData and reclustered by the same Seurat method described above with the addition of running Harmony to account for sample-to-sample variability50. Clusters expressing high levels of >1 cell-type marker were removed. Excitatory and inhibitory neurons identified from the full dataset were clustered together to determine neuronal subtypes. Four clusters (1, 2, 19 and 21) were found to express low levels of neuronal genes and astrocytic genes (SLC1A2 and SLC1A3), and were single-batch enriched (80–90%) therefore these clusters were not included in downstream analysis (Extended Data Fig. Gene co-expression modules were identified using Celda51. Celda utilized Bayesian hierarchical linear mixed effects models to identify modules of genes that are expressed together. A workflow overview can be found in Extended Data Fig. Celda was run on cellular subtypes to determine module scores on a cell-wise basis and plotted across cellular subtypes. Statistical analysis of module enrichment was performed using a linear mixed effects model using sample ID as a covariate. For microglia, cell subtypes were compared to homeostatic microglia as a baseline, for endothelial cells Cap1 was used, for astrocytes Astro1 (homeostatic astrocytes) were used as a baseline. Celda module analysis was plotted as Violin plots as these types of plots demonstrate statistical differences and also allow for visualization of the variance within the data (Supplementary Figs. Additionally, to help further validate findings, radar plots for each Celda module were also provided to help visualize module distribution among all groups (Supplementary Figs. hdWGCNA (v.0.4.5) was also run to validate gene co-expression modules in astrocytes, microglia, and endothelial cells. The hdWGCNA workflow was run with default parameters except min_cells was set to 25 and k was set to 10 for the metacells analysis performed by the MetacellsByGroups function. Additionally, minModuleSize was set to 25 in the ConstructNetwork function for astrocytes and microglia. Radar plots were provided to demonstrate cell-type distribution. Metascape49 was used to generate GO analyses for Fig. Statistics for GO were generated with GSEA and single-tailed hypergeometric test with Benjamini–Hochberg multiple hypothesis correction. hdWGCNA and Celda Modules were compared against each other for further validation. All major modules of interest could be observed in both module analyses (Supplementary Figs. The discrepancy between module numbers with hdWGCNA and Celda was the result of how each technique process data. Celda clusters every gene into a module, in contrast to hdWGCNA that does not. Celda also captures modules that are broadly expressed across many clusters rather than modules only expressed in small numbers of clusters. Biological function of each module was assessed with the EnrichR package to validate functional significance. Finally, in order to efficiently run hdWCGNA on single cell data, a prior step must be performed that reduces the cells to ‘metacells'. According the hdWGCNA tutorial, “metacell aggregation approach does not yield good results for extremely underrepresented cell types”, which probably also contributes to the reduced module number. Although module numbers may differ, important modules of interest were preserved through both datasets. All module genes and statistical analysis can be viewed in Supplementary Tables 17–19, analysis code is available on GitHub at www.github.com/morganebutler/singleCellScripts. The Sun et al. dataset23 was downloaded from https://compbio.mit.edu/microglia_states/. Another Sun et al.25 dataset was downloaded at http://compbio.mit.edu/scADbbb/. For the microglia, bootstrapping was performed by randomly sampling 80% of the Sun dataset with replacement for 50 iterations. For each iteration, FindTransferAnchors from the Seurat package was used to project the current microglia dataset onto the Sun UMAP space, and MapQuery to predict microglia labels. Label calls were recorded for each iteration and the label consistency was reported as the percentage of iterations the same label was called in Extended Data Fig. For Visium projection of astrocyte subtype genes, publicly available Visium data from human cortex (Adult Human Brain 1) were downloaded from the 10x Genomics website. The Seurat function AddModuleScore was used to create a per-spot score for astrocyte subtype gene expression (significantly upregulated genes in each subtype). Plots were created with SpatialFeaturePlot and displayed in Extended Data Fig. Ligand–receptor pair analysis was performed using MultiNicheNet, an adaptation of nichenet that allows for comparison across more than two condition groups. In brief, this method uses known datasets of ligand–receptor pairs and their downstream targets to identify potentially upregulated cell signalling pathways across cell types accounting for differential expression of genes across groups. MultiNicheNet also uses prioritization of top ligand–receptor pairs to help identify signalling pathways of interest. Contrasts for differential gene expression were set as RHI versus control, and CTE versus RHI to determine RHI and CTE-specific signalling pathways. Finalized cell-type objects were combined and run through the MultiNicheNet pipeline with the exclusion of T cells due to low cell numbers. Analysis was performed without alteration to publicly available code, save for the contrasts used. Formalin-fixed, paraffin-embedded tissue was sectioned and labelled as described52. In brief, 10-μm sections were allowed to dry, baked, dewaxed and rehydrated prior to antibody labelling. For immunofluorescent staining, epitope retrieval was performed using a pH 6 or pH 9 buffer and boiling for 15 min in the microwave. Sections were blocked for 30 min at room temperature with 3% donkey serum and primary antibodies (Supplementary Table 4) were conjugated for 1 h at room temperature. Secondary antibodies were conjugated for 30 min, and Opal TSA dyes were incubated for 10 min. Slides were coverslipped with ProLong Gold Antifade mounting medium (Invitrogen) and imaged at 20× or 40× on a Vectra Polaris whole-slide scanner with the appropriate filters. Images were spectrally unmixed using inForm software prior to image analysis. For Nissl staining, sections were hydrated and stained in 0.01% thionin for 20–40 s and dehydrated back to xylene before coverslipping in Permount mounting media and imaging on an Aperio GT450 scanner at 40×. As formalin-fixed histologic tissue was more readily available than frozen samples, more samples could be utilized for immunohistochemistry and in situ hybridization experiments. A full list of samples that were included in each immunohistochemistry experiment is shown in Supplementary Tables 2 and 3. Tissue was embedded in Optimal Cutting Temperature medium (Sakura Tissue-Tek) and was brought to cryostat temperature (−20 °C) before cutting. Chuck temperature was raised to −12°/−10 °C for optimal cutting conditions. Tissue was sectioned at 16 µm thickness onto Fisher SuperFrost slides. Direction of tissue orientation relative to the depth of the cortical sulcus was randomized across samples. Sections were fixed in cold 4 °C 10% neutral buffered formalin for 60 min and dehydrated in 50%, 70%, 100% and 100% ethanol for 5 min each at room temperature. Fluorescent in situ hybridization was performed using RNAScope kits (Advanced Cell Diagnostics) (Supplementary Table 5) optimized on the Leica BOND Rx automated slide staining system. Slides were pretreated with protease for 15 min. Opal TSA dyes were used for visualization at a concentration of 1:300–1:500. A positive and negative control probe was run for each block before staining with targeted probes. For immunohistochemical codetection of p-tau and GLUT1, sections were run through the RNAScope protocol as described and then manually stained with the AT8 or GLUT1 antibody (Supplementary Table 4) with the immunohistochemical protocol described in ‘Histological analysis' without the antigen retrieval. Not all samples were used across every smFISH experiment due to exhaustion of sample blocks. Analysis of fluorescent RNAScope fluorescence in situ hybridization (FISH) was performed in Indica Labs HALO using the FISH v.3.2.3 algorithm or the FISH-IF v.2.2.5 algorithm. Thresholds for FISH probe positivity for was set manually for each probe (HIF1A, SPP1, P2RY12, ITGAV, TGFB1, TGFBR2, LAMP5 and CUX2) and kept consistent across samples. It should be noted that SPP1 is not exclusively expressed by microglia, and DEG analysis demonstrated that only oligodendrocytes showed elevated expression of SPP1 in our dataset (Supplementary Table 6b). However, colocalization with microglia markers allows for a microglia-specific count of SPP1 activity. Gene expression was determined by the ‘probe cell intensity' in HALO because this measure is agnostic to manual single copy intensity settings. The background on GLUT1 staining in FISH sections was variable due to protease treatment from RNAScope and thresholds were manually adjusted to remove background staining. Vessel proximity analysis was performed by evaluating TGFB1+P2RY12+ cells and GLUT1+ITGAV+TGFBR2+ cells and using the ‘proximity analysis' algorithm in the HALO spatial analysis settings. The number of unique marker-positive microglia/vessel pairs within 25 µm were evaluated. Density heat maps for CUX2+LAMP5+ staining were created using the ‘density heatmap' function within HALO spatial analysis. Depiction of how the sulcus and crest were annotated can be found in Extended Data Fig. To validate consistency between image analyses methods and snRNA-seq results, seven samples that were included in both RNAScope and snRNA-seq methods were compared and cellular proportions of CUX2+LAMP5+ neurons significantly correlated (P = 0.02; Extended Data Fig. Analysis of immunohistochemistry protein staining was performed using the HALO Object Colocalization v.2.1.4 and HighPlex v.4.3.2 algorithm. Microglial P2RY12 was assessed by DAPI+IBA1+ nuclei and P2RY12hi/low thresholds were set manually. High P2RY12 was defined as having at least 70% of the nucleus stained, low P2RY12 was defined as less than 70% of the nucleus stained as demonstrated visually in Fig. Only 5.4% of all IBA1+ or P2RY12+ cells were P2RY12+IBA1−, suggesting that 94.6% of labelled microglia were assessed. IBA1+P2RY12− cells may have been captured in our P2RY12low categorization, however previous studies have shown that these cells are low in abundance and likely represent infiltrating macrophages which have been shown to be present mainly at lesioned vessels in CTE which are also sparse in our cohort53,54. Analysis of Nissl staining was performed using the HALO Nuclei Segmentation AI algorithm. Neurons were selected for training based on previously published criteria55. In brief, the classifier was given examples of brain parenchyma annotated for neurons which were considered cells with a Nissl-positive cytoplasm and a visible nucleus (Extended Data Fig. Nissl+ densities across batches were not significantly different and statistical tests of Nissl densities were corrected for staining batch. For FISH and Nissl sections, the depth of the cortical sulcus was defined and annotated as the bottom third of a gyral crest and sulcus pair. Layer 2/3 and layers 4–6 were annotated using layer-specific FISH markers or for Nissl by an expert observer. The following code and software was used for the analyses: CellRanger v.6.0.1 was used to align reads to the GRCH38 reference and generate filtered count matrices. All other analyses were performed in R v.4.2.1 and Python v.3.10.12 using standard functions unless otherwise stated. Specific versions of packages used are listed in available GitHub code. The following packages were used: CellRanger v.6.0.1, singleCellTK v.2.8.0, Seurat v.4.3.0, scater v.1.24.0, harmony v.0.1.1, RColorBrewer v.1.1.3, ComplexHeatmap v.2.14.0, ArchR v.1.0.2, muscat v.1.12.1, readr v.2.1.4, ggplot2 v.3.4.2, ggsignif v.0.6.4, ggpubr v.0.6.0, magrittr v.2.0.3, scCoda v.0.1.9 Python package, celda v.1.19.1 and hdWGCNA v.0.4.5. HALO v.3.6.4134.193, HALO AI v.3.6.4134, HALO Object Colocalization v.2.1.4 algorithm and FISH v.3.2.3 algorithm were used to analyse the histological and Nissl images. InForm v.2.5.1 was used to spectrally unmix fluorescent in situ hybridization images. The research has included local researchers through the research process and is locally relevant with collaborators. All roles and responsibilities were agreed amongst collaborators ahead of the research. The study was approved by the Institutional review board through the Boston University Alzheimer's Disease and CTE Center, Human Subjects Institutional Review Board of the Boston University School of Medicine, VA Bedford Healthcare System, VA Boston Healthcare System, and Iowa Neuropathology Resource Laboratory. The research did not result in stigmatization, incrimination, discrimination, or risk to donors or research staff. No materials have been transferred out of the country. Local and regional research relevant to the study has been included in the citations. Analyses were performed using GraphPad Prism 10, SPSS v.29 and R (v.4.2.1) packages ggsignif, muscat, scater, and the Python (v.3.10.12) package scCoda. Dirichlet multinomial regression was used to test for cell type and excitatory neuron cell-type enrichment using the scCoda v.0.1.9 Python package13. Celda module expression was evaluated using linear mixed effects modelling, accounting for individual sample differences. Comparisons of cell-type proportions across the three pathological groups were performed using ANOVA with Bonferroni correction, Brown Forsyth with Dunnett post-hoc test, or chi-squared test as indicated in figure legends. Comparison across control and RHI-exposed groups was performed with a t-test with Welch correction or Mann–Whitney U-test, as indicated in the figure legends. Bar plots denote error with s.e.m. Scatter plots denote error with a grey outline of the 95% confidence interval. Evaluation of in situ hybridization analysis was performed using linear regression. P-tau burden was normalized using log10 transformation of positive area density. Nissl+ neuron count comparisons to years of exposure were assessed using linear regression and correcting for age at death and staining batch. Jaccard similarity scoring was performed using the GeneOverlap package by comparing lists of DEGs. All DEGs were filtered by a log2-transformed fold change of 0.15 and false discovery rate (FDR) of <0.05. Chi-squared tests for cellular abundance were performed using the base R chisq.tst function. GO statistics were calculated with GSEA and single-tailed hypergeometric test with Benjamini–Hochberg multiple hypothesis correction. Years of football play was used as a variable for exposure throughout the text instead of total years of play (which includes exposure from all sports) played because it was a more consistent predictor of cellular changes. snRNA-seq tissue isolation was performed once per each individual. Reproducibility was assessed through comparison to other published datasets23,25. As detailed in Extended Data Figs. 3, 5 and 7, there was significant overlap between our subtypes and other previously published subtypes, highlighting that our results are highly reproducible. For all histological antibody, Nissl and in situ hybridization staining, individual cases were stained and analysed once per each experiment. Histologic methods were validated and optimized prior to the start of the experiment to ensure proper labelling and accurate downstream analyses as discussed in the previous sections. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Code can be found at www.github.com/morganebutler/singleCellScripts. Data are available at GEO accession number GSE261807. All other data supporting the findings of this study and unprocessed images are available upon request. Request for tissue or digital images can be made by emailing the corresponding author or through the Boston University ADRC request portal at https://wwwru.bumc.bu.edu/BUADC/RequestBrainTissue.aspx. The following public databases were used for the study: GRCH38 reference GCF_000001405.26, Sun et al. microglia dataset (https://compbio.mit.edu/microglia_states/), Visium spatial expression (https://www.10xgenomics.com/datasets/adult-human-brain-1-cerebral-cortex-unknown-orientation-stains-anti-gfap-anti-nfh-1-standard-1-1-0) and Sun et al. vascular dataset (http://compbio.mit.edu/scADbbb/). Applying the Bradford Hill criteria for causation to repetitive head impacts and chronic traumatic encephalopathy. McKee, A. C. et al. Neuropathologic and clinical findings in young contact sport athletes exposed to repetitive head impacts. Duration of American football play and chronic traumatic encephalopathy. Daneshvar, D. H. et al. Leveraging football accelerometer data to quantify associations between repetitive head impacts and chronic traumatic encephalopathy in males. Cherry, J. D. et al. Microglial neuroinflammation contributes to tau accumulation in chronic traumatic encephalopathy. Alosco, M. L. et al. White matter signal abnormalities in former National Football League players. Chancellor, K. B. et al. Altered oligodendroglia and astroglia in chronic traumatic encephalopathy. Kirsch, D. et al. Vascular injury is associated with repetitive head impacts and tau pathology in chronic traumatic encephalopathy. Rosen, G. et al. Three dimensional evaluation of cerebrovascular density and branching in chronic traumatic encephalopathy. The second NINDS/NIBIB consensus meeting to define neuropathological criteria for the diagnosis of chronic traumatic encephalopathy. Single cell RNA sequencing of human microglia uncovers a subset associated with Alzheimer's disease. Otero-Garcia, M. et al. Molecular signatures underlying neurofibrillary tangle susceptibility in Alzheimer's disease. Büttner, M., Ostner, J., Müller, C. L., Theis, F. J. & Schubert, B. scCODA is a Bayesian model for compositional single-cell data analysis. Babcock, K. J. et al. Interface astrogliosis in contact sport head impacts and military blast exposure. Fabriek, B. O. et al. CD163-positive perivascular macrophages in the human CNS express molecules for antigen recognition and presentation. Distinct amyloid-β and tau-associated microglia profiles in Alzheimer's disease. Sinner, P. et al. Microglial expression of CD83 governs cellular activation and restrains neuroinflammation in experimental autoimmune encephalomyelitis. Novel microglia-mediated mechanisms underlying synaptic loss and cognitive impairment after traumatic brain injury. De Schepper, S. et al. Perivascular cells induce microglial phagocytic states and synaptic engulfment via SPP1 in mouse models of Alzheimer's disease. Shin, Y. J. et al. Osteopontin: correlation with phagocytosis by brain macrophages in a rat model of stroke. Riew, T. R. et al. Osteopontin and its spatiotemporal relationship with glial cells in the striatum of rats treated with mitochondrial toxin 3-nitropropionic acid: possible involvement in phagocytosis. Hong, S. et al. Complement and microglia mediate early synapse loss in Alzheimer mouse models. Sun, N. et al. Human microglial state dynamics in Alzheimer's disease progression. Sadick, J. S. et al. Astrocytes and oligodendrocytes undergo subtype-specific transcriptional changes in Alzheimer's disease. Sun, N. et al. Single-nucleus multiregion transcriptomic analysis of brain vasculature in Alzheimer's disease. Hodge, R. D. et al. Conserved cell types with divergent features in human versus mouse cortex. Identification of epilepsy-associated neuronal subtypes and gene expression underlying epileptogenesis. Gómez-Isla, T. et al. Neuronal loss correlates with but exceeds neurofibrillary tangles in Alzheimer's disease. Yousef, A. et al. Neuron loss and degeneration in the progression of TDP-43 in frontotemporal lobar degeneration. Mckee, A. C. & Daneshvar, D. H. in Handbook of Clinical Neurology, Vol. Cain, A. et al. Multicellular communities are perturbed in the aging human brain and Alzheimer's disease. Tagge, C. A. et al. Concussion, microvascular injury, and early tauopathy in young athletes after impact head injury and an impact concussion mouse model. Neher, J. J. et al. Inhibition of microglial phagocytosis is sufficient to prevent inflammatory neuronal death. Browaeys, R. et al. MultiNicheNet: a flexible framework for differential cell-cell communication analysis from multi-sample multi-condition single-cell transcriptomics data. Patel, R. K., Prasad, N., Kuwar, R., Haldar, D. & Abdul-Muneer, P. M. Transforming growth factor-beta 1 signaling regulates neuroinflammation and apoptosis in mild traumatic brain injury. Hellbach, N. et al. Neural deletion of Tgfbr2 impairs angiogenesis through an altered secretome. Zuidema, T. R. et al. Cerebral cortical surface structure and neural activation pattern among adolescent football players. Nicks, R. et al. Neurodegeneration in the cortical sulcus is a feature of chronic traumatic encephalopathy and associated with repetitive head impacts. Katz, D. I. et al. National Institute of Neurological Disorders and Stroke consensus diagnostic criteria for traumatic encephalopathy syndrome. Clinical presentation of chronic traumatic encephalopathy. & Heintz, N. Layer 2/3 pyramidal cells in the medial prefrontal cortex moderate stress induced depressive behaviors. Shalek, A. K. et al. Single-cell transcriptomics reveals bimodality in expression and splicing in immune cells. The first NINDS/NIBIB consensus meeting to define neuropathological criteria for the diagnosis of chronic traumatic encephalopathy. Hong, R. et al. Comprehensive generation, visualization, and reporting of quality control metrics for single-cell RNA sequencing data. Germain, P. L., Lun, A., Macnair, W. & Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. Decontamination of ambient RNA in single-cell RNA-seq with DecontX. Integrated analysis of multimodal single-cell data. Zhou, Y. et al. Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Wang, Z. et al. Celda: a Bayesian model to perform co-clustering of genes into modules and cells into subpopulations using single-cell RNA-seq data. Butler, M. L. M. D. et al. Tau pathology in chronic traumatic encephalopathy is primarily neuronal. Cherry, J. D. et al. CCL2 is associated with microglia and macrophage recruitment in chronic traumatic encephalopathy. Kenkhuis, B. et al. Co-expression patterns of microglia markers Iba1, TMEM119 and P2RY12 in Alzheimer's disease. García-Cabezas, M., John, Y. J., Barbas, H. & Zikopoulos, B. Distinction of neurons, glia and endothelial cells in the cerebral cortex: an algorithm based on cytological features. This work was supported by grant funding from: NINDS (F31NS132407), NIH (U19-AG068753), NIA (AG057902, AG062348 and AG090553), NINDS (U54NS115266 and RF1NS143015), National Institute of Aging Boston University Alzheimer's Disease Research Center (P30AG072978), Department of Veterans Affairs Biorepository (BX002466), the Department of Veterans Affairs Career Development Award (BX004349) and BLRD Merit Award (I01BX005933). The views, opinions, and/or findings contained in this article are those of the authors and should not be construed as an official Veterans Affairs or Department of Defense position, policy, or decision, unless so designated by other official documentation. Funders did not have a role in the design and conduct of the study; collection, management, analysis, and interpretation of the data; preparation, review, or approval of the manuscript; or decision to submit the manuscript for publication. We would like to extend our gratitude to the brain donors and their families without whom this work would be impossible. The authors thank the clinical and neuropathology teams at the BU CTE Center who perform clinical interviews with next of kin and routine tissue processing and D. Rosene, B. Conner and S. Horowitz for methodological support, and the Boston University Single Cell Sequencing Core for their assistance in performing the experiments. Department of Anatomy and Neurobiology, Boston University Chobanian and Avedisian School of Medicine, Boston, MA, USA Morgane L. M. D. Butler, Kerry Breen, Samantha Calderazzo, Thor D. Stein, Ann C. McKee & Jonathan D. Cherry Nida Pervaiz, Yichen Wang, Julia Cammasola Breda, Sarah Mazzilli & Joshua D. Campbell Department of Pharmacology, Physiology, and Biophysics, Boston University Chobanian and Avedisian School of Medicine, Boston, MA, USA Department of Pathology and Laboratory Medicine, Boston University Chobanian and Avedisian School of Medicine, Boston, MA, USA Samantha Calderazzo, Thor D. Stein, Ann C. McKee & Jonathan D. Cherry VA Boston Healthcare System, Jamaica Plain, MA, USA Petra Ypsilantis, Bertrand R. Huber, Victor E. Alvarez, Thor D. Stein, Ann C. McKee & Jonathan D. Cherry Raymond Nicks, Elizabeth Spurlock, Victor E. Alvarez & Thor D. Stein Department of Neurology, Boston University Chobanian and Avedisian School of Medicine, Boston, MA, USA Bertrand R. Huber, Victor E. Alvarez, Ann C. McKee & Jonathan D. Cherry Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Article conceptualization and design was performed by M.L.M.D.B. Tissue processing, dissection and sample collection were performed by R.N. snRNA-seq tissue processing and data analyses were carried out by M.L.M.D.B., N.P., J.C.B. Histologic antibody-based tissue staining experiments were performed by M.L.M.D.B., P.Y., K.B. performed and analysed the in situ hybridization experiments. Computational support was provided by S.M. Neuropathologic diagnosis of post-mortem human samples was performed by M.M.H., K.L.F., V.E.A., B.R.H., T.D.S. Campbell provided intellectual input throughout the study. All authors read and approved the final manuscript. The authors declare no competing interests. Nature thanks Josh Morganti and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. Fluorescence activated cell sorting gating strategy of DAPI-positive nuclei. b. Stacked bar plot representing the proportion of cell type per donor. c. Stacked bar plot representing the cell type counts per donor. d-e. Violin plots for each donor of (d) total gene counts per cell, (e) unique genes detected per cell, (f) percent of mitochondrial genes detected per cell, and (g) percent ribosomal genes detected per cell. h, i UMAP of full dataset before cleaning colored by (h) doublet or singlets or (i) mitochondrial contamination. j, k. UMAP of full dataset after cleaning colored by (j) doublets or singlets or (k) mitochondrial contamination. l. UMAP of full dataset colored by Seurat clusters. m. Dot plot of cell type marker expression across Seurat clusters depicted in (l). n. Dot plot of cell type marker expression in annotated cell type clusters. Bar plots of overall cell type proportions across pathological groups with each dot representing a sample, bars represent the mean, error bars represent standard error of the mean. Statistical analysis performed by ANOVA with Bonferroni correction. b. UMAP depicting OPCs colored by Seurat clustering, solid arrow indicating RHI/CTE depleted cluster. c. Stacked bar plot showing OPC Seurat cluster distribution across pathological groups. d. Bar plots showing OPC cluster distribution across control and pathological group or control and RHI-exposed samples, bar represents mean, error bars show standard error of the mean. Statistical analysis performed by ANOVA with Bonferroni correction (left) and two-tailed Mann-Whitney U test. e. Heatmap showing GO analysis of OPC cluster DEGs. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. f. UMAP showing oligodendrocytes colored by Seurat cluster, solid arrow indicates RHI and CTE depleted cluster. g. Stacked bar plot showing oligodendrocyte pathological group distribution per Seurat cluster. h. Bar plots representing cluster distribution across pathological groups or control and RHI-exposed samples. Bar represents mean, error bar represents standard error of the mean. Statistical analysis performed by ANOVA with Bonferroni correction (left) or two-tailed t-test (right). i. Heatmap showing GO analysis of oligodendrocyte cluster DEGs. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. j. UMAP showing T cells colored by Seurat cluster. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. snRNAseq fraction of (a) homeostatic and (b) RHI microglia. Statistics performed by ANOVA with Bonferroni correction. Bar represents mean, error bars show SEM. c. Microglial subtypes across control compared to RHI-exposed individuals. Statistical analysis performed by two-tailed t-test or Mann Whitney U test with Welch correction. Bar represents mean, error bars show SEM. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. i. UMAPs depicting microglia module scores of microglial subtypes from Sun et al. j. Heatmap depicting Jaccard score similarity analysis between Sun et al. and current study microglial DEGs. l, m, n. SPP1 + /HIF1A+ microglial fraction in the grey matter (l) crest, (m) L2/3 Sulcus (n) layers 4–6 sulcus colored by pathological group compared to years of football play. Statistical analysis performed by linear regression. Black line represents linear regression, grey shows 95% confidence interval. o. P2RY12 low/Iba1+ microglial densities in the grey matter sulcus compared to years of football play. p. Homeostatic microglial densities compared to Nissl+ neuronal densities in layers 2/3 (left) and layers 4–6 (right). Each dot represents a single individual. Statistical analysis performed by linear regression with age as a covariate. Black line represents linear regression, grey shows 95% confidence interval. Examples of Celda module expression in microglia. UMAPs show module expression, heatmaps show per-cell expression with genes listed on the right. c. Examples of Celda module expression in endothelial cells. UMAPs show module expression, heatmaps show per-cell expression with genes listed on the right. Genes can be viewed in Supplementary Tables 17–19. a. UMAP depicting microglia from Sun et al. dataset with original labels. b. Microglia from Butler et al. dataset projected onto Sun dataset UMAP space with original labels. c. Microglia from Butler et al. dataset projected onto Sun dataset UMAP space with predicted Sun labels. d. Bar plot depicting result of boostrapping, with number of predicted microglial labels and error bars depicting bootstrap confidence. e. UMAP depicting Butler microglia projected onto Sun dataset UMAP space colored by label consistency throughout bootstrapping. f. Stacked barplot depicting proportion of Butler microglia with Sun dataset labels across pathological groups. b. UMAP from (a) colored by pathological group. c. Stacked bar plots showing astrocyte subtype distribution across pathological groups, statistics performed by chi-squared test. d. Bar plots showing astrocyte subcluster distribution in control and RHI-exposed samples, dots represent individual donors colored by pathological group identity. Bars represent mean, error bars represent standard error of the mean. Statistical analysis was performed using two-tailed Mann Whitney U-test. e. Stacked bar plots showing pathological distribution across astrocyte subtypes. f. Violin plots showing Celda module expression across astrocyte subtypes. Black bar showing median statistic. Colored by astrocyte subtype most associated with specific module expression. Statistical analysis performed by linear mixed effects model. g. Gene ontology analysis of astrocytic subtypes performed by Metascape. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. h. Dot plot representing expression of selected DEGs across astrocytic subtype and annotated by function. Projection of current astrocytic modules on to Visium spatial transcriptomic data. Top row of heatmaps show expression of white matter (PLP1, MBP) and grey matter (SLC27A7, SNAP25) genes. Dotted line indicates separation of grey and white matter. Heatmaps on the bottom row show expression of astrocyte subtype modules based on significantly upregulated genes in each subtype. a. UMAP showing all vascular cells colored by Seurat clustering. b. Heatmap depicting vascular cell marker expression. c. Heatmap depicting Jaccard scoring of vascular cell Seurat cluster DEGs compared to Sun and Akay et al. vascular subtype DEGs. Significance denoted by **p < 0.01, ***p < 0.001. d. Bar plots depicting pathological group proportions of vascular subtypes, bar represents mean, error bar represents SEM, dots represent individual samples. Statistical analysis performed by ANOVA with Bonferroni correction. e. Heatmap depicting Jaccard scoring of vascular cell subtype DEGs compared to Sun and Akay et al. vascular subtype DEGs. Significance denoted by ** p < 0.01, *** p < 0.001. f. Scatter plot of fibroblast proportion or Cap2 proportion compared to years of football play from snRNAseq dataset, colored by pathological group status. Statistical analysis performed by linear regression with age as a covariate. Black line represents linear model regression, grey shows 95% confidence interval. g, h. Bar plots of total capillary and relative endothelial cell subtype distribution across control and RHI-exposed samples, dots represent individual donors and are colored by pathological group identity. Statistical analysis was performed by two-tailed Mann-Whitney U test. GO enrichment analysis of DEGs from depicted comparisons. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. Statistics generated with linear regression correcting for age. Black line represents linear model regression, grey shows 95% confidence interval. Each dot represents a single individual. a. UMAP depicting all neurons clustered together colored by Seurat cluster. Dot plot of gene expression of inhibitory and excitatory neuron and astrocyte marker genes Seurat clusters from (a). c. UMAP from (a) colored by cell type determination. e. UMAP showing excitatory neurons colored by Seurat cluster. f. UMAP showing excitatory neurons colored by later subtype. g. Dot plot showing expression of excitatory neuron layer subtype genes in excitatory neuron Seurat clusters from (e). h. UMAP showing inhibitory neurons colored by Seurat cluster. i. UMAP showing inhibitory neurons colored by layer subtype. Dot plot showing expression of inhibitory neuron layer subtype genes across inhibitory neuron Seurat clusters from (h). a. UMAP depicting excitatory neurons colored by layer subtype. Heatmap showing GO analysis of (b) excitatory layer and (c) inhibitory layer up and downregulated DEGs. Statistics generated with GSEA and single-tailed hypergeometric test with Benjamini-Hochberg multiple hypothesis correction. d. Bar plots of excitatory neuron layer proportions by pathological group. Bar represents mean, dots represent individual samples, error bars show standard error of the mean. Statistical analysis performed by ANOVA with Bonferroni correction. e. UMAP showing inhibitory neurons colored by layer subtype. f. Bar plots of inhibitory neuron layer proportions by pathological group. Bar represents mean, dots represent individual samples, error bars show standard error of the mean. Statistical analysis performed by ANOVA with Bonferroni correction. g. Representative image showing RNAScope in situ hybridization of CUX2/LAMP5 image analysis with correct anatomical layer-wise distribution. White squares showing HALO identification of double-positive cells. h. Representative image of Nissl+ staining and neuronal masking using HALO AI. Top box is the raw image and bottom box is AI generated mask over neurons. a. CUX2/LAMP5 proportion from snRNAseq against total years of football play colored by pathological group. Statistical analysis performed by linear regression, depicted as black line. b. CUX2/LAMP5 density from in situ hybridization compared to highest level of football played. Statistical analysis performed by linear regression. Dots represent single individuals; line shows linear regression. c. CUX2/LAMP5 cells identified by in situ experiment compared to proportion of CUX2/LAMP5 neurons from snRNAseq experiment. Statistical analysis performed by linear regression, depicted as black line with 95% confidence intervals in grey. d. Representative image showing the annotation strategy used to identify the sulcus (yellow line) and crest (red line) layer 2/3. e. (left) Layer 2/3 CUX2+/LAMP5+ neuronal density in the sulcus and crest. Statistical analysis performed by paired t-test. (right) Layer 2/3 CUX2+/LAMP5+ neuronal density in the crest compared to years of football play, colored by pathological group. Statistical analysis performed by linear regression. Black line represents general linear model regression, grey shows 95% confidence interval. f. Total CUX2 populations and subpopulations in snRNAseq (right panel) and in situ hybridization (left panel) experiments compared to years of football play. g, h, i. Scatter plots depicting (g) Sulcus layers 4–6 Nissl+ density compared to binned years of football play, (h) Crest layer 2/3 Nissl+ density compared to binned years of football play, (i) L2/3 Nissl+ density compared to log tau+ density, Statistical analysis performed by linear regression, (g, h) corrected for age and staining batch. Statistical analysis performed by linear regression. Black line represents general linear model regression, grey shows 95% confidence interval. k, l. Representative images of Nissl staining across cortical layers depicting neuronal loss in superficial layers in RHI and CTE individuals. This file contains Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Butler, M.L.M.D., Pervaiz, N., Breen, K. et al. Repeated head trauma causes neuron loss and inflammation in young athletes. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09422-z'>DeepSeek-R1 incentivizes reasoning in LLMs through reinforcement learning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 15:06:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. General reasoning represents a long-standing and formidable challenge in artificial intelligence (AI). Recent breakthroughs, exemplified by large language models (LLMs)1,2 and chain-of-thought (CoT) prompting3, have achieved considerable success on foundational reasoning tasks. However, this success is heavily contingent on extensive human-annotated demonstrations and the capabilities of models are still insufficient for more complex problems. Here we show that the reasoning abilities of LLMs can be incentivized through pure reinforcement learning (RL), obviating the need for human-labelled reasoning trajectories. The proposed RL framework facilitates the emergent development of advanced reasoning patterns, such as self-reflection, verification and dynamic strategy adaptation. Consequently, the trained model achieves superior performance on verifiable tasks such as mathematics, coding competitions and STEM fields, surpassing its counterparts trained through conventional supervised learning on human demonstrations. Moreover, the emergent reasoning patterns exhibited by these large-scale models can be systematically used to guide and enhance the reasoning capabilities of smaller models. Reasoning capability, the cornerstone of human intelligence, enables complex cognitive tasks ranging from mathematical problem-solving to logical deduction and programming. Recent advances in AI have demonstrated that LLMs can exhibit emergent behaviours, including reasoning abilities, when scaled to a sufficient size4,5. However, achieving such capabilities in pre-training typically demands substantial computational resources. In parallel, a complementary line of research has demonstrated that LLMs can be effectively augmented through CoT prompting. This technique, which involves either providing carefully designed few-shot examples or using minimalistic prompts such as “Let's think step by step”3,6, enables models to produce intermediate reasoning steps, thereby substantially enhancing their performance on complex tasks. Similarly, further performance gains have been observed when models learn high-quality, multistep reasoning trajectories during the post-training phase2,7. Despite their effectiveness, these approaches exhibit notable limitations. Their dependence on human-annotated reasoning traces slows scalability and introduces cognitive biases. Furthermore, by constraining models to replicate human thought processes, their performance is inherently capped by the human-provided exemplars, which prevents the exploration of superior, non-human-like reasoning pathways. To tackle these issues, we aim to explore the potential of LLMs for developing reasoning abilities through self-evolution in a RL framework, with minimal reliance on human labelling efforts. Specifically, we build on DeepSeek-V3 Base8 and use Group Relative Policy Optimization (GRPO)9 as our RL framework. Notably, we bypass the conventional supervised fine-tuning (SFT) phase before RL training. This design choice originates from our hypothesis that human-defined reasoning patterns may limit model exploration, whereas unrestricted RL training can better incentivize the emergence of new reasoning capabilities in LLMs. Through this process, detailed in the next section, our model (referred to as DeepSeek-R1-Zero) naturally developed diverse and sophisticated reasoning behaviours. To solve reasoning problems, the model exhibits a tendency to generate longer responses, incorporating verification, reflection and the exploration of alternative approaches within each response. Although we do not explicitly teach the model how to reason, it successfully learns improved reasoning strategies through RL. Although DeepSeek-R1-Zero demonstrates excellent reasoning capabilities, it faces challenges such as poor readability and language mixing, occasionally combining English and Chinese in a single CoT response. Furthermore, the rule-based RL training stage of DeepSeek-R1-Zero is narrowly focused on reasoning tasks, resulting in limited performance in broader areas such as writing and open-domain question answering. To address these challenges, we introduce DeepSeek-R1, a model trained through a multistage learning framework that integrates rejection sampling, RL and supervised fine-tuning, detailed in the ‘DeepSeek-R1' section. This training pipeline enables DeepSeek-R1 to inherit the reasoning capabilities of its predecessor, DeepSeek-R1-Zero, while aligning model behaviour with human preferences through further non-reasoning data. To enable broader access to powerful AI at a lower energy cost, we have distilled several smaller models and made them publicly available. These distilled models exhibit strong reasoning capabilities, surpassing the performance of their original instruction-tuned counterparts. We believe that these instruction-tuned versions will also greatly contribute to the research community by providing a valuable resource for understanding the mechanisms underlying long CoT reasoning models and for promoting the development of more powerful reasoning models. We release DeepSeek-R1-Zero, DeepSeek-R1, data samples and distilled models to the public as described in the ‘Code availability' section. Specifically, we use GRPO9 as our RL algorithm, described in Methods section ‘GRPO'. Furthermore, we use a rule-based reward system to compute accuracy and format rewards, with detailed methodologies outlined in Methods section ‘Reward design'. Furthermore, our high-performance RL infrastructure is described in Supplementary Information, section 2.1, ensuring scalable and efficient training. During training, we design a straightforward template to require DeepSeek-R1-Zero to first produce a reasoning process, followed by the final answer. The prompt template is written as below. We intentionally limit our constraints to this structural format, avoiding any content-specific biases to ensure that we can accurately observe the natural progression of the model during the RL process. Figure 1a shows the performance trajectory of DeepSeek-R1-Zero on the American Invitational Mathematics Examination (AIME) 2024 benchmark throughout the RL training process, in which the average pass@1 score on AIME 2024 shows a marked increase, jumping from an initial value of 15.6% to 77.9%. Besides the maths competitions, as shown in Supplementary Fig. AIME takes a mathematical problem as input and a number as output, illustrated in Extended Data Table 1. pass@1 and cons@16 are described in Supplementary Information, section 4.1. DeepSeek-R1-Zero naturally learns to solve reasoning tasks with more thinking time. As well as the progressive enhancement of reasoning capabilities during training, DeepSeek-R1-Zero also demonstrates self-evolutionary behaviour with RL training. 1b, DeepSeek-R1-Zero exhibits a steady increase in thinking time throughout training, driven only by intrinsic adaptation rather than external modifications. Making use of long CoT, the model progressively refines its reasoning, generating hundreds to thousands of tokens to explore and improve its problem-solving strategies. Specifically, DeepSeek-R1-Zero increasingly exhibits advanced reasoning strategies such as reflective reasoning and systematic exploration of alternative solutions provided in Extended Data Fig. 1a, substantially boosting its performance on verifiable tasks such as maths and coding. Notably, during training, DeepSeek-R1-Zero exhibits an ‘aha moment', shown in Table 1, characterized by a sudden increase in the use of the word ‘wait' during reflections, provided in Extended Data Fig. This moment marks a distinct change in reasoning patterns and clearly shows the self-evolution process of DeepSeek-R1-Zero. The self-evolution of DeepSeek-R1-Zero underscores the power and beauty of RL: rather than explicitly teaching the model how to solve a problem, we simply provide it with the right incentives and it autonomously develops advanced problem-solving strategies. This serves as a reminder of the potential of RL to unlock higher levels of capabilities in LLMs, paving the way for more autonomous and adaptive models in the future. Although DeepSeek-R1-Zero exhibits strong reasoning capabilities, it faces several issues. DeepSeek-R1-Zero struggles with challenges such as poor readability and language mixing, as DeepSeek-V3 Base is trained on several languages, especially English and Chinese. To address these issues, we develop DeepSeek-R1, whose pipeline is illustrated in Fig. In the initial stage, we collect thousands of cold-start data that exhibit a conversational, human-aligned thinking process, as detailed in Supplementary Information, section 2.3.2. RL training is then applied with hyperparameters in Methods section ‘Training details of the first RL stage', data details in Supplementary Information, section 2.3.1, to improve the model performance with the conversational thinking process and language consistency. Subsequently, we apply rejection sampling and SFT once more. This stage incorporates both reasoning and non-reasoning datasets into the SFT process, as detailed in Supplementary Information, section 2.3.3, enabling the model to not only excel in reasoning tasks but also demonstrate advanced writing capabilities. To further align the model with human preferences, we implement a secondary RL stage designed to enhance the helpfulness and harmlessness of the model while simultaneously refining its reasoning capabilities. The reward model is described in Methods section ‘Reward design' and RL hyperparameters are in Methods section ‘Training details of the second RL stage'. The total training cost is listed in Supplementary Information, section 2.4.4. A detailed background on DeepSeek-V3 Base and DeepSeek-V3 is provided in Supplementary Information, section 1.1. The models DeepSeek-R1 Dev1, Dev2 and Dev3 represent intermediate checkpoints in this pipeline. We evaluate our models on MMLU11, MMLU-Redux12, MMLU-Pro13, DROP14, C-Eval15, IFEval16, FRAMES17, GPQA Diamond18, SimpleQA19, C-SimpleQA20, CLUEWSC21, AlpacaEval 2.0 (ref. 22), Arena-Hard23, SWE-bench Verified24, Aider-Polyglot25, LiveCodeBench26 (2024-08–2025-01), Codeforces27, Chinese National High School Mathematics Olympiad (CNMO 2024)28 and AIME 2024 (ref. 29). The details of these benchmarks are provided in Supplementary Tables 15–29. Table 2 summarizes the performance of DeepSeek-R1 across several developmental stages, as outlined in Fig. A comparison between DeepSeek-R1-Zero and DeepSeek-R1 Dev1 reveals substantial improvements in instruction-following, as evidenced by higher scores on the IF-Eval and Arena-Hard benchmarks. By contrast, DeepSeek-R1 Dev2 demonstrates marked performance enhancements on benchmarks that require advanced reasoning skills, including those focused on code generation, mathematical problem solving and STEM-related tasks. Benchmarks targeting general-purpose tasks, such as AlpacaEval 2.0, show marginal improvement. These results indicate that reasoning-oriented RL considerably enhances reasoning capabilities while exerting limited influence on user-preference-oriented benchmarks. DeepSeek-R1 Dev3 integrates both reasoning and non-reasoning datasets into the SFT pipeline, thereby enhancing the proficiency of the model in both reasoning and general language-generation tasks. Compared with Dev2, DeepSeek-R1 Dev3 achieves notable performance improvements on AlpacaEval 2.0 and Aider-Polyglot, attributable to the inclusion of large-scale non-reasoning corpora and code-engineering datasets. Finally, comprehensive RL training on DeepSeek-R1 Dev3 using mixed reasoning-focused and general-purpose data produced the final DeepSeek-R1. Marginal improvements occurred in code and mathematics benchmarks, as substantial reasoning-specific RL was done in previous stages. The primary advancements in the final DeepSeek-R1 were in general instruction-following and user-preference benchmarks, with AlpacaEval 2.0 improving by 25% and Arena-Hard by 17%. We also compare DeepSeek-R1 with other models in Supplementary Information, section 4.2. Model safety evaluations are provided in Supplementary Information, section 4.3. A comprehensive analysis of evaluation is provided in Supplementary Information, section 5, including a comparison with DeepSeek-V3, performance evaluations on both fresh test sets, a breakdown of mathematical capabilities by category and an investigation of test-time scaling behaviour. Supplementary Information, section 6 shows that the strong reasoning capability can be transferred to smaller models. For example, R1 can be subject to jailbreak attacks, leading to the generation of dangerous content such as explosive manufacturing plans, whereas the enhanced reasoning capabilities enable the model to provide plans with better operational feasibility and executability. Besides, a public model is also vulnerable to further fine-tuning that could compromise inherent safety protections. In Supplementary Information, section 4.3, we present a comprehensive safety report from several perspectives, including performance on open-source and in-house safety evaluation benchmarks, and safety levels across several languages and against jailbreak attacks. These comprehensive safety analyses conclude that the inherent safety level of the DeepSeek-R1 model, compared with other state-of-the-art models, is generally at a moderate level (comparable with GPT-4o (2024-05-13)30). We present DeepSeek-R1-Zero and DeepSeek-R1, which rely on large-scale RL to incentivize model reasoning behaviours. Our results demonstrate that pre-trained checkpoints inherently have substantial potential for complex reasoning tasks. We believe that the key to unlocking this potential lies not in large-scale human annotation but in the provision of hard reasoning questions, a reliable verifier and sufficient computational resources for RL. Sophisticated reasoning behaviours, such as self-verification and reflection, seemed to emerge organically during the RL process. Even if DeepSeek-R1 achieves frontier results on reasoning benchmarks, it still faces several capability limitations, as outlined below. At present, the structural output capabilities of DeepSeek-R1 remain suboptimal compared with existing models. Moreover, DeepSeek-R1 cannot make use of tools, such as search engines and calculators, to improve the performance of output. However, as it is not hard to build a RL environment for structure output and tool use, we believe that the issue will be addressed in the next version. Unlike conventional test-time computation scaling approaches, such as majority voting or Monte Carlo tree search (MCTS), DeepSeek-R1 dynamically allocates computational resources during inference according to the complexity of the problem at hand. Specifically, it uses fewer tokens to solve simple tasks but generating more tokens for complex tasks. Nevertheless, there remains room for further optimization in terms of token efficiency, as instances of excessive reasoning—manifested as overthinking—are still observed in response to simpler questions. DeepSeek-R1 is at present optimized for Chinese and English, which may result in language-mixing issues when handling queries in other languages. For instance, DeepSeek-R1 might use English for reasoning and responses, even if the query is in a language other than English or Chinese. We aim to address this limitation in future updates. The limitation may be related to the base checkpoint, DeepSeek-V3 Base, which mainly uses Chinese and English, so that it can achieve better results with the two languages in reasoning. When evaluating DeepSeek-R1, we observe that it is sensitive to prompts. Therefore, we recommend that users directly describe the problem and specify the output format using a zero-shot setting for optimal results. Future versions will address this by implementing rejection sampling on software-engineering data or incorporating asynchronous evaluations during the RL process to improve efficiency. Beyond specific capability limitations, the pure RL methodology itself also presents inherent challenges: The success of pure RL depends on reliable reward signals. If the reward signal is assigned by a model instead of predefined rules, it becomes more susceptible to exploitation as training progresses, which means that the policy model may find shortcuts to hack the reward model. Consequently, for complex tasks that cannot be effectively evaluated by a reliable reward model, scaling up pure RL methods remains an open challenge. In this work, for tasks that cannot obtain a reliable signal, DeepSeek-R1 uses human annotation to create supervised data and only conducts RL for hundreds of steps. We hope that, in the future, a robust reward model can be obtained to address such issues. With the advent of pure RL methods such as DeepSeek-R1, the future holds immense potential for solving any task that can be effectively evaluated by a verifier, regardless of its complexity for humans. Machines equipped with such advanced RL techniques are poised to surpass human capabilities in these domains, driven by their ability to optimize performance iteratively through trial and error. In such cases, the lack of a robust feedback mechanism may slow progress, suggesting that future research should focus on developing innovative approaches to define and refine reward structures for these complex, less verifiable problems. Furthermore, making use of tools during the reasoning process holds notable promise. Whether it is using tools such as compilers or search engines to retrieve or compute necessary information or using external tools such as biological or chemical reagents to validate final results in the real world, this integration of tool-augmented reasoning could greatly enhance the scope and accuracy of machine-driven solutions. GRPO9 is the RL algorithm that we use to train DeepSeek-R1-Zero and DeepSeek-R1. It was originally proposed to simplify the training process and reduce the resource consumption of proximal policy optimization (PPO)31, which is widely used in the RL stage of LLMs32. The pipeline of GRPO is shown in Extended Data Fig. For each question q, GRPO samples a group of outputs {o1, o2,…, oG} from the old policy \({\pi }_{{\theta }_{{\rm{old}}}}\) and then optimizes the policy model πθ by maximizing the following objective: in which πref is a reference policy, ϵ and β are hyperparameters and Ai is the advantage, computed using a group of rewards {r1, r2,…, rG} corresponding to the outputs in each group: We give a comparison of GRPO and PPO in Supplementary Information, section 1.3. For DeepSeek-R1-Zero, we use rule-based rewards to deliver precise feedback for data in mathematical, coding and logical reasoning domains. For DeepSeek-R1, we extend this approach by incorporating both rule-based rewards for reasoning-oriented data and model-based rewards for general data, thereby enhancing the adaptability of the learning process across diverse domains. Our rule-based reward system mainly consists of two types of reward: accuracy rewards and format rewards. Accuracy rewards evaluate whether the response is correct. For example, in the case of maths problems with deterministic results, the model is required to provide the final answer in a specified format (for example, within a box), enabling reliable rule-based verification of correctness. Similarly, for code competition prompts, a compiler can be used to evaluate the responses of the model against a suite of predefined test cases, thereby generating objective feedback on correctness. Format rewards complement the accuracy reward model by enforcing specific formatting requirements. In particular, the model is incentivized to encapsulate its reasoning process within designated tags, specifically <think> and </think>. This ensures that the thought process of the model is explicitly delineated, enhancing interpretability and facilitating subsequent analysis. Notably, we abstain from applying neural reward models—whether outcome-based or process-based-to reasoning tasks. This decision is predicated on our observation that neural reward models are susceptible to reward hacking during large-scale RL. Moreover, retraining such models necessitates substantial computational resources and introduces further complexity into the training pipeline, thereby complicating the overall optimization process. For general data, we resort to reward models to capture human preferences in complex and nuanced scenarios. We build on the DeepSeek-V3 pipeline and use a similar distribution of preference pairs and training prompts. For helpful reward model training, we first generate preference pairs by prompting DeepSeek-V3 using the Arena-Hard prompt format, listed in Supplementary Information, section 2.2, for which each pair consists of a user query along with two candidate responses. For each preference pair, we query DeepSeek-V3 four times, randomly assigning the responses as either Response A or Response B to mitigate positional bias. The final preference score is determined by averaging the four independent judgments, retaining only those pairs for which the score difference (Δ) exceeds 1 to ensure meaningful distinctions. Furthermore, to minimize length-related biases, we ensure that the chosen and rejected responses of the whole dataset have comparable lengths. In total, we curated 66,000 data pairs for training the reward model. The prompts used in this dataset are all non-reasoning questions and are sourced either from publicly available open-source datasets or from users who have explicitly consented to share their data for the purpose of model improvement. The maximum sequence length during training is set to 8,192 tokens, whereas no explicit limit is imposed during reward model inference. To assess and improve model safety, we curated a dataset of 106,000 prompts with model-generated responses annotated as ‘safe' or ‘unsafe' according to predefined safety guidelines. Unlike the pairwise loss used in the helpfulness reward model, the safety reward model was trained using a pointwise methodology to distinguish between safe and unsafe responses. For general queries, each instance is categorized as belonging to either the safety dataset or the helpfulness dataset. For each question, we sample 16 outputs with a maximum length of 32,768 tokens before the 8.2k step and 65,536 tokens afterward. As a result, both the performance and response length of DeepSeek-R1-Zero exhibit a substantial jump at the 8.2k step, with training continuing for a total of 10,400 steps, corresponding to 1.6 training epochs. Each training step consists of 32 unique questions, resulting in a training batch size of 512 per step. To accelerate training, each rollout generates 8,192 outputs, which are randomly split into 16 minibatches and trained for only a single inner epoch. For each question, we sample 16 outputs with a maximum length of 32,768. Each training step consists of 32 unique questions, resulting in a training batch size of 512 per step. To accelerate training, each rollout generates 8,192 outputs, which are randomly split into 16 minibatches and trained for only a single inner epoch. However, to mitigate the issue of language mixing, we introduce a language consistency reward during RL training, which is calculated as the proportion of target language words in the CoT. Although ablation experiments in Supplementary Information, sction 2.6 show that such alignment results in a slight degradation in the performance of the model, this reward aligns with human preferences, making it more readable. We apply the language consistency reward to both reasoning and non-reasoning data by directly adding it to the final reward. Note that the clip ratio plays a crucial role in training. A lower value can lead to the truncation of gradients for a large number of tokens, thereby degrading the performance of the model, whereas a higher value may cause instability during training. Specifically, we train the model using a combination of reward signals and diverse prompt distributions. For reasoning data, we follow the methodology outlined in DeepSeek-R1-Zero, which uses rule-based rewards to guide learning in mathematical, coding and logical reasoning domains. During the training process, we observe that CoT often exhibits language mixing, particularly when RL prompts involve several languages. For general data, we use reward models to guide training. Ultimately, the integration of reward signals with diverse data distributions enables us to develop a model that not only excels in reasoning but also assigns priority to helpfulness and harmlessness. Given a batch of data, the reward can be formulated as The second stage of RL retains most of the parameters from the first stage, with the key difference being a reduced temperature of 0.7, as we find that higher temperatures in this stage lead to incoherent generation. The stage comprises a total of 1,700 training steps, during which general instruction data and preference-based rewards are incorporated exclusively in the final 400 steps. We find that more training steps with the model-based preference reward signal may lead to reward hacking, which is documented in Supplementary Information, section 2.5. We provide the data samples used in our rejection sampling and RL prompts at https://github.com/deepseek-ai/DeepSeek-R1 (https://doi.org/10.5281/zenodo.15753193)33. Comprehensive statistics and details of our complete data-generation methodology are presented in Supplementary Information, section 2.3. Neural networks were developed with PyTorch35 and the distributed framework is based on our internal framework HAI-LLM (https://www.high-flyer.cn/en/blog/hai-llm). Wei, J. et al. Chain-of-thought prompting elicits reasoning in large language models. Wei, J. et al. Emergent abilities of large language models. In Transactions on Machine Learning Research (eds Kamath, G. et al.) (2022). Shao, Z. et al. DeepSeekMath: pushing the limits of mathematical reasoning in open language models. Wang, X. et al. Self-consistency improves chain of thought reasoning in language models. In 11th International Conference on Learning Representations (ICLR, 2023). In 9th International Conference on Learning Representations (ICLR, 2021). Gema, A. P. et al. Are we done with MMLU? Wang, Y. et al. MMLU-Pro: a more robust and challenging multi-task language understanding benchmark. Dua, D. et al. DROP: a reading comprehension benchmark requiring discrete reasoning over paragraphs. 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies Vol. 1 (Long and Short Papers) (eds Burstein, J. et al.) 2368–2378 (ACL, 2019). Huang, Y. et al. C-EVAL: a multi-level multi-discipline Chinese evaluation suite for foundation models. Zhou, J. et al. Instruction-following evaluation for large language models. Fact, fetch, and reason: a unified evaluation of retrieval-augmented generation. 63rd Annual Meeting of the Association for Computational Linguistics Vol. Xu, L. et al. CLUE: a Chinese Language Understanding Evaluation benchmark. 28th International Conference on Computational Linguistics (eds Scott, D. et al.) 4762–4772 (International Committee on Computational Linguistics, 2020). Dubois, Y., Galambosi, B., Liang, P. & Hashimoto, T. B. Length-controlled AlpacaEval: a simple way to debias automatic evaluators. Li, T. et al. From crowdsourced data to high-quality benchmarks: Arena-Hard and BenchBuilder pipeline. Jain, N. et al. LiveCodeBench: holistic and contamination free evaluation of large language models for code. In 13th International Conference on Learning Representations (ICLR, 2024). Chinese National High School Mathematics Olympiad; https://www.cms.org.cn/Home/comp/comp/cid/12.html (2024). Ouyang, L. et al. Training language models to follow instructions with human feedback. Paszke, A. et al. PyTorch: an imperative style, high-performance deep learning library. Efficient memory management for large language model serving with PagedAttention. Present address: Jianping High School, Shanghai, China Kexin Huang, Wenqin Yu, Xinyuan Li, Yue Gong & Yujia He Present address: Citadel Securities, Hong Kong SAR, China Daya Guo, Dejian Yang, Haowei Zhang, Junxiao Song, Peiyi Wang, Qihao Zhu, Runxin Xu, Ruoyu Zhang, Shirong Ma, Xiao Bi, Xiaokang Zhang, Xingkai Yu, Yu Wu, Z. F. Wu, Zhibin Gou, Zhihong Shao, Zhuoshu Li, Ziyi Gao, Aixin Liu, Bing Xue, Bingxuan Wang, Bochao Wu, Bei Feng, Chengda Lu, Chenggang Zhao, Chengqi Deng, Chong Ruan, Damai Dai, Deli Chen, Dongjie Ji, Erhang Li, Fangyun Lin, Fucong Dai, Fuli Luo, Guangbo Hao, Guanting Chen, Guowei Li, H. Zhang, Hanwei Xu, Honghui Ding, Huazuo Gao, Hui Qu, Hui Li, Jianzhong Guo, Jiashi Li, Jingchang Chen, Jingyang Yuan, Jinhao Tu, Junjie Qiu, Junlong Li, J. L. Cai, Jiaqi Ni, Jian Liang, Jin Chen, Kai Dong, Kai Hu, Kaichao You, Kaige Gao, Kang Guan, Kexin Huang, Kuai Yu, Lean Wang, Lecong Zhang, Liang Zhao, Litong Wang, Liyue Zhang, Lei Xu, Leyi Xia, Mingchuan Zhang, Minghua Zhang, Minghui Tang, Mingxu Zhou, Meng Li, Miaojun Wang, Mingming Li, Ning Tian, Panpan Huang, Peng Zhang, Qiancheng Wang, Qinyu Chen, Qiushi Du, Ruiqi Ge, Ruisong Zhang, Ruizhe Pan, Runji Wang, R. J. Chen, R. L. Jin, Ruyi Chen, Shanghao Lu, Shangyan Zhou, Shanhuang Chen, Shengfeng Ye, Shiyu Wang, Shuiping Yu, Shunfeng Zhou, Shuting Pan, S. S. Li, Shuang Zhou, Shaoqing Wu, Tao Yun, Tian Pei, Tianyu Sun, T. Wang, Wangding Zeng, Wen Liu, Wenfeng Liang, Wenjun Gao, Wenqin Yu, Wentao Zhang, W. L. Xiao, Wei An, Xiaodong Liu, Xiaohan Wang, Xiaokang Chen, Xiaotao Nie, Xin Cheng, Xin Liu, Xin Xie, Xingchao Liu, Xinyu Yang, Xinyuan Li, Xuecheng Su, Xuheng Lin, X. Q. Li, Xiangyue Jin, Xiaojin Shen, Xiaosha Chen, Xiaowen Sun, Xiaoxiang Wang, Xinnan Song, Xinyi Zhou, Xianzu Wang, Xinxia Shan, Y. K. Li, Y. Q. Wang, Y. X. Wei, Yang Zhang, Yanhong Xu, Yao Li, Yao Zhao, Yaofeng Sun, Yaohui Wang, Yi Yu, Yichao Zhang, Yifan Shi, Yiliang Xiong, Ying He, Yishi Piao, Yisong Wang, Yixuan Tan, Yiyang Ma, Yiyuan Liu, Yongqiang Guo, Yuan Ou, Yuduan Wang, Yue Gong, Yuheng Zou, Yujia He, Yunfan Xiong, Yuxiang Luo, Yuxiang You, Yuxuan Liu, Yuyang Zhou, Y. X. Zhu, Yanping Huang, Yaohui Li, Yi Zheng, Yuchen Zhu, Yunxian Ma, Ying Tang, Yukun Zha, Yuting Yan, Z. Z. Ren, Zehui Ren, Zhangli Sha, Zhe Fu, Zhean Xu, Zhenda Xie, Zhengyan Zhang, Zhewen Hao, Zhicheng Ma, Zhigang Yan, Zhiyu Wu, Zihui Gu, Zijia Zhu, Zijun Liu, Zilin Li, Ziwei Xie, Ziyang Song, Zizheng Pan, Zhen Huang, Zhipeng Xu, Zhongyu Zhang & Zhen Zhang The scientific findings were discussed and approved by all contributors. This article was written by a subgroup of authors designated by the collaboration and it underwent an internal collective review process. These authors, designated as core contributors, contributed equally to this work, and are presented in alphabetical order. The remaining authors have also contributed meaningfully to the study, and their names are likewise presented in alphabetical order. The authors declare no competing interests and will not file patents related to the content of this manuscript. Nature thanks Edward Beeching, Yarin Gal, José Hernández-Orallo, Daphne Ippolito, Subbarao Kambhampati, Lewis Tunstall, Yiming Zhang and Lexin Zhou for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Frequency of representative reflective terms in model-generated outputs throughout the training process. Reflective terms—including ‘wait', ‘mistake', ‘however', ‘but', ‘retry', ‘error', ‘verify', ‘wrong', ‘evaluate' and ‘check'—were identified and curated by a panel of three human experts. Each expert independently proposed a set of words indicative of reflective reasoning, which were subsequently consolidated through consensus into a final vocabulary list. This term was virtually absent during the initial training stages, appeared sporadically between steps 4,000 and 7,000 and exhibited a marked increase in frequency after step 8,000. These trends suggest the emergence of temporal reasoning or self-monitoring behaviour as training progresses. Each response within the group is evaluated by a reward model—either learned (model-based) or manually specified (rule-based)—to assign a scalar reward signal. Subsequently, GRPO computes the relative advantages of each group member based on their assigned rewards. Rather than relying on an explicit value function, as in PPO, GRPO directly estimates advantages from the intra-group reward distribution. The policy parameters are then updated to maximize the expected reward while simultaneously minimizing divergence from a reference policy, typically quantified through the KL divergence. By eliminating the need for a separate value network, GRPO offers a simplified yet effective alternative to traditional actor-critic methods such as PPO. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Guo, D., Yang, D., Zhang, H. et al. DeepSeek-R1 incentivizes reasoning in LLMs through reinforcement learning. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/new-research-shows-gut-cells-communicate-directly-with-the-brain/'>The Gut-Brain Connection Is Real—And It's Changing How We Think about Health</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-17 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>By Rachel Feltman, Fonda Mwangi & Alex Sugiura An illustration of a man revealing his brain and stomach with an arrow drawn between them on a purple background This episode was made possible by the support of Yakult and produced independently by Scientific American's board of editors. People often talk about having “gut feelings,” but new research suggests there may be more to the idiom than we thought. Scientists are finding that specialized cells in our intestines can send signals directly to the brain, potentially influencing appetite and even mood. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Recent studies hint that our microbiomes could play a role in this communication system, though researchers are still trying to understand exactly how these interactions work and what they mean for our health. Thanks so much for coming on to chat with us today. Could you tell us a little bit about why scientists are interested in that and what we know about it so far? Kaelberer: Yeah, I mean, I think more than just scientists are interested in it; we have our gut feelings all the time. And so my work is really focused on understanding the biology behind those gut feelings and how is it that our gut can communicate to us. 'Cause we know, right, things like hangry exist. Feltman: Well, and beyond, you know, hanger, which is obviously a great example, what are some conditions that have been connected to the gut that might surprise people? Kaelberer: So when I was in my postdoc we discovered that there was this direct connection between these cells in the surface of the gut, we call them neuropod cells, and neurons that communicate directly—they reach directly into the brain. And so we call this as—our “gut sense,” and the number-one question I would always get was: Who cares? Like, what [laughs], you know, what is our gut possibly telling us that our mouth and our nose did not already tell us about the food that we ate? And so we delved into this a little bit more in some previous publications with regards to sugar sensing, and I use this example 'cause it's really salient in my own life, which is that I like artificial sweetener in my coffee. Kaelberer: And I don't like regular sugar because regular sugar just feels heavy to me, and I want that, like, artificial sweetener. I can be, like, on the go. I'm not gonna, like, sit down and take a nap afterwards. And so we know that these two stimuli feel different in our gut. And what we found is that these neuropod cells are actually distinguishing between the two stimuli, between real sugar and artificial sweetener. So now we take it back to my coffee preference, and suddenly, I'm like, “Well, this makes sense. And so then this is now this communication system of, like, “Oh, our gut sense is telling us something about the food we eat past whether or not it tastes good. It's telling us a little bit about the value of what we're consuming.” Feltman: So let's get into the latest study. Kaelberer: Yeah, so the microbiome is everywhere. We're always hearing about how the microbiome affects a lot of our behaviors, and there's all this really cool work that's coming out showing that there are these different species, they can affect your mood, but we don't really know what that connection is. And so when we found that these neuropod cells are responding to nutrients, we thought, “Oh, well, in the gut there's this microbiome population. I wonder if the microbiome is actually signaling via these neuropod cells.” And so it turns out what we found is they do, hence the study. And they're doing it in this really interesting way. So what they're doing is they're sensing this protein that's on the tail of the bacteria, and this is any bacteria that has a tail—it has to swim around, has a tail—and this protein, so it's pretty widely expressed across all these different microbes. And what [a neuropod cell is] sensing is: it's sensing that protein, and that protein tells the animal when it's being sensed that the animal should eat a little bit less and when the protein isn't there that the animal should eat a little bit more. And you can think, like, this might be counterintuitive because aren't we against bacteria [laughs], right? And so what we found is that this is actually a circuit, via these neuropod cells, and it's a direct connection that allows the microbiome and the host to somehow communicate about what its food needs are. So what do you think the implications of those findings are? If we know—it's, like, figuring out: What's the language they're using to communicate? And now, once we know what the language they're using is, now you can imagine: “Okay, are there nutrients that affect the signaling pathway? Are there probiotics that influence it more or less? Are there other signals, for instance, from the microbes, not just this tail protein but other things that are actually influencing different types of behavior?” And so this is just kind of the beginning of, like, well, what is this, this “neurobiotic sense” is what we're calling it, because it's this whole new sense that's all about: How do we communicate with our microbiome, and how does our microbiome communicate with us? Feltman: So what are your next steps research-wise? Kaelberer: So research-wise the next steps are—I actually already mentioned them a little bit. So you could imagine, like, a high-fat or a high-sugar diet—I have this pet theory: I think that people with a sweet tooth actually have microbes with a sweet tooth and that there's some kind of communication going on there that's saying, like, “Hey, maybe eat a little more sugar. We want a little more sugar down here.” Or, like, a high-fat diet, this is another, like, thing that we study a lot is: How is it that our diet, or highly processed food, how is this actually influencing that ecosystem that we contain in our gut? Feltman: How far out do you think we are from being able to take advantage of some of these gut-brain connections by manipulating our microbiome? Kaelberer: Rachel, that is a great question because we talk a lot about the science, we talk about the initial discoveries. I'm gonna give a small example, which is Ozempic. This satiety hormone was discovered in animals. And then the first Ozempic drugs, if I'm getting my dates right, were in the teens, was when they came out. And so now we're talking about, like, 30 years of research that had to go into this really key discovery that was made before we had the implications in human health. Kaelberer: I don't think it's gonna take that long because there's a lot of other stuff that we've built on. Kaelberer: But I think it's important to know, like, all right, you've gotta test the mechanism. We don't just give people anything [laughs], right? We have to go through the process of, like, testing it out, testing what we know about it and then testing, like, are there other drugs, say—so this is another common way to do it—are there other drugs that are already approved in humans that are influencing this new pathway that we discovered and are having this positive effect? They grow up without bacteria at all. And these mice are a little weird; turns out we need them. But this finding, I think, is actually really interesting because what it implies is that the mouse did not need prior experience in order for this circuit to actually exist. And when that happens, usually what that means is this is because the circuit evolved; we evolved with our microbiomes in order to communicate. This isn't just some kind of reaction that we're learning based on what microbes come into our body. And so I think understanding that kind of aspect of it puts a different spin on the research 'cause now we're talking about, okay, this is a lifelong interaction that we've had, so now we know we can't just get rid of it, right? You can't just take the microbes out. Kaelberer: As you can probably tell I'm very passionate about the work that I'm doing, and I just think that we're at the forefront of so many really cool discoveries. So I, I always ask, Rachel, and I'm gonna ask you this question: Now that you know what a gut sense is, think about your last meal. How did your gut feel about what you last ate? So I don't think my gut felt bad about it [laughs]. Kaelberer: But now you're gonna be thinking about it. Kaelberer: [Laughs] Let's all be honest about that. But I'm like, “Mm.” I feel the satisfaction that my gut is like, “You made a good decision.” We'll be back on Friday to talk about how social media algorithms are shaping the future of language. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi and Jeff DelViscio. This episode was edited by Alex Sugiura. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. For Scientific American, this is Rachel Feltman. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Fonda Mwangi is a multimedia editor at Scientific American and producer of Science Quickly. She previously worked at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. You can even gift someone a subscription. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. David M. Ewalt, Editor in Chief, Scientific American Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            