
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-17</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00439-6'>Nanoscience is latest discipline to embrace large-scale replication efforts</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 16:53:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. The project — named after bubbles of misinformation that can form when bad science is left uncorrected — made its call for help on 11 February. Last year, for example, a large-scale reproducibility project in Brazil tried and failed to validate dozens of biomedical studies. Engineered nanoparticles such as these can be used to deliver drugs into the body.Credit: David McCarthy/SPL Lévy, one of four researchers who lead the NanoBubbles project, which is backed by an €8-million (US$9.5-million) grant from the European Research Council, wants laboratories to revisit a 2012 study that suggested tiny, fluorescent carbon nanoparticles can detect copper ions inside living cells. This could have medical relevance because high copper levels are linked to diseases such as cancer and neurodegenerative disorders. There are many possible reasons why work in this field might not replicate, says Wolfgang Parak, a physicist at the University of Hamburg, Germany, who is a member of the NanoBubbles advisory board but was not involved in Gharib's experiments. As an example, Parak notes that one of his synthesis reactions became hard to reproduce after he moved his lab from the United States to Europe. “Surface chemistry is extremely sensitive to small impurities,” he says, and reagents in different countries could have varying amounts of contamination. Another problem is that experimental protocols sometimes don't describe steps in sufficient detail. So, any study that identifies crucial points in protocols and how to improve them is helpful. “They really measured everything very carefully and used state-of-the-art techniques,” he says. ‘Doing good science is hard': retraction of high-profile reproducibility study prompts soul-searching How to make research reproducible: psychology protocol gives 86% success rate NIH rolls back red tape on some experiments — spurring excitement and concern Street-dog policy in India is barking up the wrong tree The science influencers going viral on TikTok to fight misinformation How to wow a popular-science writer with your research expertise How AI slop is causing a crisis in computer science Imaging the sub-moiré potential using an atomic single electron transistor Join HZAU's global faculty team to advance research with competitive benefits. ‘Doing good science is hard': retraction of high-profile reproducibility study prompts soul-searching How to make research reproducible: psychology protocol gives 86% success rate An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-026-01454-7'>Ectopic expression of cytosolic DHODH uncouples de novo pyrimidine biosynthesis from mitochondrial electron transport</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 16:36:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Dihydroorotate dehydrogenase is a rate-limiting enzyme of de novo pyrimidine synthesis. In most eukaryotes, this enzyme is bound to the inner mitochondrial membrane, where it couples orotate synthesis to ubiquinone reduction. As ubiquinone must be regenerated by respiratory complex III, pyrimidine biosynthesis and cellular respiration are tightly coupled. Consequently, inhibition of respiration suppresses DNA synthesis and cell proliferation. Here we show that expression of the Saccharomyces cerevisiae URA1 gene (ScURA) in mammalian cells uncouples pyrimidine biosynthesis from mitochondrial electron transport. ScURA forms a homodimer in the cytosol that uses fumarate as an electron acceptor instead of ubiquinone, enabling respiration-independent pyrimidine biosynthesis. Cells expressing ScURA are resistant to drugs that inhibit complex III and the mitochondrial ribosome. Overall, this genetic tool uncovers the contribution of pyrimidine biosynthesis to the phenotypes arising from electron transport chain defects. Cells obtain pyrimidine nucleotides for DNA and RNA synthesis either by de novo biosynthesis or by salvage of preformed bases1. Dihydroorotate dehydrogenase (DHODH) catalyses the fourth step of the biosynthetic pathway: oxidation of dihydroorotate (DHO) to orotate. In most eukaryotes, DHODH is bound to the external surface of the inner mitochondrial membrane, where it uses ubiquinone (UQ) as an electron acceptor. Once UQ is reduced by DHODH, it is re-oxidized by respiratory complex III. Therefore, DHODH activity couples pyrimidine biosynthesis with the mitochondrial electron transport chain (mETC)2. The biochemical consequence of this coupling is that de novo pyrimidine biosynthesis is not possible in the absence of mitochondrial electron transport: the failure to re-oxidize UQH2 to UQ causes a build-up in the UQH2/UQ ratio, depleting the pool of UQ that DHODH requires to function. This explains why cells lacking mitochondrial DNA (mtDNA) or treated with mitochondrial inhibitors affecting complexes III or IV benefit from exogenous uridine supplementation to grow3,4,5,6. Organisms that are capable of growing in the absence of oxygen must produce orotate through a different mechanism. Here, DHODH is either linked to an alternative ETC or is a cytosolic enzyme that uses either NAD+ or fumarate as electron acceptors7,8,9. In these organisms, pyrimidine biosynthesis can thus proceed independently of cellular respiration. In yeast, the distribution of DHODH isoenzymes across various species appears to mirror their growth preferences: aerobic species harbour a mitochondrial DHODH, whereas facultative anaerobes possess a cytosolic DHODH10. Anaerobic yeasts seem to have acquired the cytosolic DHODH gene by horizontal transfer from a bacterial source and to subsequently have lost their eukaryotic gene11. Oxygen-independent pyrimidine biosynthesis is therefore one of the main adaptations that enabled anaerobic growth12. In this work, we asked whether it is possible to uncouple these two processes in mammalian cells by expressing the gene encoding cytosolic DHODH. In aerobic eukaryotes, DHODH shuttles electrons through UQ to complex III and functions as a respiratory enzyme13. Other unicellular organisms have evolved different versions of DHODH, which use either fumarate or NAD+ as electron acceptors to oxidize DHO. These isoenzymes typically lack a mitochondrial import signal and a transmembrane domain, residing instead in the cytosol (Fig. a, Phylogenetic tree (left) and corresponding sequence alignment (right) built from the amino acid sequences of DHODH isoforms from various phyla. H. sapiens and S. cerevisiae are highlighted in red. b, Scheme showing the uridine monophosphate (UMP) production pathways by either mitochondrial DHODH, cytosolic ScURA or exogenous uridine. c, Immunoblot showing the expression of the myc-tagged ScURA protein in ScURA-expressing 143B cells. d, The same sample as in c was run in native conditions, without SDS. Data in c and d are representative of two independent experiments each. e, DHODH-dependent DHO oxidation rate, measured as the reduction of DCPIP, of cell homogenates from ScURA-expressing or control 143B cells. The DCPIP reduction rates per minute (left; Methods) and linear slopes of light absorbance at 600 nm from the first 200 s (right panel) are shown. Data are represented as means; error bars, s.d. Panel b created in BioRender; Enriquez, J. https://biorender.com/l7jbv83 (2026). To enable mETC-independent de novo pyrimidine biosynthesis, we ectopically expressed Saccharomyces cerevisiae URA1 gene (ScURA) in mammalian cells (Fig. The yeast gene's cDNA was codon-optimized for mammalian expression, and a myc tag was added at the carboxyl terminus through a flexible (GGS)3 linker to prevent steric interference between the tag and the enzyme (Supplementary Data). The gene was cloned into a lentiviral vector, and an empty vector was used as the control for all subsequent experiments. In human 143B cells, we were able to detect ScURA expression by western blot (Fig. Accordingly, when 143B homogenates were separated by native, non-denaturing polyacrylamide gel electrophoresis (PAGE), we observed that ScURA migrated more slowly than expected for its monomeric mass (36 kDa) and toward the expected size of the homodimer (Fig. When we tested DHODH enzymatic activity in cell homogenates, we found that ScURA-expressing cells possessed 2.25 times more activity than controls (Fig. Approximately two-thirds of this activity was insensitive to brequinar inhibition, as brequinar targets the UQ binding site of human DHODH, which is not present in ScURA15. Therefore, ScURA expression provides an alternative pathway for DHO dehydrogenation in human cells. Under normal cell culture conditions, DHODH activity is not a rate-limiting factor for cell proliferation, and exogenous uridine supplementation is unnecessary (Fig. Consistently, expression of ScURA did not affect cell proliferation or mitochondrial function and morphology in 143B cells (Fig. To test whether ScURA could replace the endogenous DHODH activity in proliferating cells, we treated 143B cells with brequinar and rescued pyrimidine biosynthesis inhibition by supplementing uridine in the growth medium. Brequinar suppressed cell proliferation in control cells, which was fully restored upon uridine supplementation (Fig. On the other hand, in ScURA-expressing cells, brequinar had no discernible effect on cell proliferation. We next used CRISPR–Cas9 to generate DHODH knockout (KO) 143B cells (Fig. As expected, these cells required exogenous uridine to proliferate (Fig. We then reintroduced ScURA, human DHODH (hDHODH) or hDHODH lacking its mitochondrial targeting sequence (MTS) and the transmembrane domain (hDHODH-ΔMTS) using a lentiviral vector. Re-expression of either hDHODH or ScURA was sufficient to restore uridine-independent cell proliferation, while the hDHODH-ΔMTS variant failed to rescue the phenotype. Furthermore, uridine supplementation offered no additional growth advantage to cells expressing ScURA, demonstrating that it fully compensates for the loss of mitochondrial DHODH. To directly measure the flux through de novo pyrimidine synthesis, we cultured wild-type (WT) and ScURA-expressing cells with [α-15N]glutamine (Fig. 2h), whose nitrogen specifically labels the pyrimidine rings on pyrimidine nucleotides such as uridine triphosphate (UTP) and cytidine triphosphate (CTP). We observed similar rates of UTP and CTP labelling in both cell lines under basal conditions (Fig. Chemical inhibition of DHODH (with brequinar) or of the mETC (with antimycin and myxothiazol) ablated labelling in WT cells. By contrast, ScURA-expressing cells maintained UTP and CTP synthesis in both conditions. These results demonstrate that ScURA enables brequinar-insensitive de novo pyrimidine biosynthesis in human cells by bypassing mitochondrial DHODH. a, Cell doublings of ScURA-expressing and control 143B cells grown in normal culture media; n = 6 independent biological replicates, two-sided Welch's t-test; ns, not significant. b, Oxygen consumption rate (OCR) of 143B cells expressing ScURA permeabilized with digitonin and incubated with ADP (basal) plus substrates for complex I (G/M, glutamate/malate) and complex II (Succ, succinate)-dependent respiration; n = 8 (WT) and n = 7 (ScURA) independent biological replicates. c, ATP production rate in isolated mitochondria determined by a luciferase-dependent assay; data are normalized on total protein. d, Confocal microscopy analysis of total mitochondria count in 143B cells; each dot represents an individual cell. f, Immunoblot showing DHODH KO in 143B cells and re-expression of DHODH or ScURA. g, Cell doublings of 143B-DHODHKO cells cultured with or without exogenous uridine (DHODHKO −uridine versus +uridine and DHODHKO + hDHODH-ΔMTS −uridine versus +uridine, P < 0.0001). The heavy nitrogen atom from the α-amine of glutamine is passed to aspartate via glutamate. Some intermediate reactions have been omitted for simplicity. j, Mass isotopologue distribution of aspartate, UTP and CTP from [α-15N]glutamine tracing. The total levels of aspartate for each condition are shown in Extended Data Fig. Data are means of n = 3 independent biological replicates per condition; error bars, s.d. For c, e, g and i, ordinary one-way ANOVA, Šidák's multiple comparison test, ****P < 0.0001; *P < 0.05. The dependency of pyrimidine biosynthesis on the mETC was first discovered in chick embryo cells treated with chloramphenicol, which were found to be auxotrophic for uridine5. Therefore, we investigated whether ScURA expression could restore pyrimidine biosynthesis and rescue cell proliferation in cells treated with chloramphenicol. To this end, we cultured 143B cells in chloramphenicol and uridine for 1 week before assessing their ability to grow without uridine. Although uridine supplementation was essential to maintain proliferation in WT cells, ScURA-expressing cells were able to grow at the same rate in the absence of uridine (Fig. We next sought to investigate whether ScURA could rescue cell proliferation when ubiquinol oxidation by complex III was disrupted. We treated cells with both antimycin A and myxothiazol to completely block electron flow through complex III16,17,18, rendering cells auxotrophic for uridine3 (Fig. Under these conditions, ScURA expression was able to maintain cell proliferation at the same level as uridine supplementation (Fig. WT cells treated with antimycin A and myxothiazol could not be maintained past a single passage, in contrast to cells supplemented with uridine or carrying ScURA, which could be kept under complex III inhibition indefinitely. We next used semi-targeted metabolomics to address how ScURA-driven metabolic changes may support proliferation in 143B cells (Fig. ScURA-expressing cells displayed a decrease in DHO and an increase in orotate, which are the substrate and the product of the ScURA reaction, respectively (Fig. Although mETC inhibition in WT cells caused a massive (>600-fold) accumulation of DHO, this metabolite remained below the limit of detection in ScURA-expressing cells under the same conditions (Fig. The pools of free pyrimidines were decreased when cells expressed ScURA in basal conditions; however, ScURA expression was able to rescue the loss of UTP and CTP caused by mETC inhibition (Fig. Therefore, ScURA allows mETC-independent de novo pyrimidine biosynthesis in human cells. We also observed accumulation of flavin mononucleotide, the cofactor of ScURA, and a shift in urea cycle intermediates towards arginine synthesis, which is coupled with the production of fumarate, the substrate of ScURA (Fig. In conclusion, these results support the notion that ScURA restores proliferation upon ETC inhibition largely by supplying pyrimidines. a, Cell doublings of ScURA-expressing and control 143B cells treated with chloramphenicol (chlor) and rescued with exogenous uridine supplementation (all significant comparisons, P < 0.0001). b, Cell doublings of ScURA-expressing and control 143B cells treated with antimycin and myxothiazol and rescued with exogenous uridine supplementation (all significant comparisons, P < 0.0001). c, Scheme showing the UMP production pathways upon respiratory complex III inhibition by antimycin A and myxothiazol. d, Volcano plots showing differential metabolite abundance in 143B cells expressing ScURA treated with either antimycin/myxothiazol or vehicle (ethanol). Adjusted P values were calculated using two-sided Student's t-test. e, Heatmap showing unsupervised sample clustering based on z-score-scaled intensity values of de novo pyrimidine biosynthesis metabolites. Data are presented as means of n = 3 independent biological replicates; error bars, s.d. Panel c created in BioRender; Enriquez, J. https://biorender.com/yxlqkqw (2026). ScURA takes reducing equivalents from DHO and deposits them onto fumarate instead of UQ, generating succinate, which in turn can then feed the tricarboxylic acid (TCA) cycle. Therefore, we investigated the impact of this metabolic shift by stable isotope tracing of [U-13C]glutamine (Fig. 4a), which allowed us to track the fate of TCA cycle metabolites. In control conditions, we did not observe any accumulation of succinate in ScURA-expressing cells, probably because succinate can re-enter the TCA cycle and be converted to fumarate by succinate dehydrogenase (SDH) (Fig. However, the pools of other TCA cycle intermediates changed: α-ketoglutarate, malate and acetyl-CoA increased while citrate decreased, with a consequent increase of the α-ketoglutarate/citrate ratio (Fig. 2a,b), a known determinant of reductive carboxylation19. Accordingly, the labelling pattern of citrate, aspartate and malate changed when ScURA was expressed, indicating reductive carboxylation metabolism (Fig. The above observations fit a model in which de novo pyrimidine biosynthesis is sustained by ScURA by creating a cycle of fumarate and succinate between the cytosol and the mitochondria (Fig. Given that the ScURA reaction causes a substantial flux of succinate into the TCA cycle, α-ketoglutarate accumulates, increasing the α-ketoglutarate/citrate ratio and driving reductive carboxylation. a, Model for [U-13C]glutamine oxidation in WT cells (top), in cells expressing ScURA (middle) and in cells expressing ScURA with acute inhibition of the mETC (bottom). Some intermediate reactions have been omitted from the scheme for simplicity. b, Normalized intensity values of succinate and α-ketoglutarate relative to 143B WT cells (ordinary one-way ANOVA, Sidak's multiple comparison test. d, Mass isotopologue distribution of citrate, aspartate, malate, S-(2-succino)-cysteine (2-SC) and succinate from [U-13C]glutamine tracing in 143B cells. f, M+3 and m+4 labelling intensity of succinate in 143B WT and ScURA-expressing cells treated with anti/myxo (P = 0.0003). h, M+3 and m+4 labelling intensity of 2-SC in 143B WT and ScURA-expressing cells treated with anti/myxo (P < 0.0001). In e, f and h: ordinary two-way ANOVA, Šidák's multiple comparison test. Data are presented as means of n = 3 independent biological replicates; error bars, s.d. Blocking UQH2 oxidation prevents cells from oxidizing succinate, causing it to accumulate. This accumulation may be further increased by SDH reverse activity20,21,22. Here, ScURA expression modified the way succinate is produced: instead of being derived from α-ketoglutarate oxidation as in WT cells ([13C4]succinate or (m+4)), an important fraction of succinate was labelled as [13C3]succinate (m+3), with the labelling characteristic of reductive carboxylation (Fig. Citrate, malate and aspartate labelling also followed the same pattern. Therefore, the rate of oxidative carboxylation is enhanced in cells expressing ScURA and provides fumarate as an electron acceptor for DHO oxidation. We could not directly quantify fumarate with our metabolomic experiment. Therefore, we used S-(2-succino)-cysteine (2-SC), the product of the reaction between fumarate and cysteine, as a proxy for fumarate labelling (Fig. The m+3 labelling of 2-SC is consistent with fumarate being produced by reductive carboxylation and becoming the electron acceptor of the reaction catalysed by ScURA (Fig. ScURA was also able to restore the growth defect caused by mETC inhibition in two cell lines lacking SDH, excluding that ScURA activity depends on fumarate reduction through reverse SDH reaction20 (143B and U2OS SDHAKO; Extended Data Fig. These results demonstrate that cells adjust their TCA flux to compensate for succinate production by ScURA. We next asked whether ScURA expression leads to changes in gene expression by RNA sequencing. We found that 113 genes were differentially expressed between ScURA and WT cells in basal conditions (adjusted P < 0.05 and absolute log2(fold change) > 1; Extended Data Fig. Gene set enrichment analysis revealed an enrichment for pathways related to interferon production, hypoxia and cellular responses to oxygen levels (Extended Data Fig. To identify the regulators driving these changes, we inferred transcription factor activity using decoupleR23. This analysis revealed HIF1α as the most differentially active transcription factor, with a corresponding elevation in a group of its known target genes (Extended Data Fig. This observation is consistent with the finding that cytosolic succinate can stabilize HIF1α24. The transcriptional impact of ScURA was far more pronounced upon mETC inhibition. In both 143B WT and ScURA-expressing cells, the integrated stress response was activated, with ATF4 being the top induced transcription factor, consistent with previous reports25 (Extended Data Fig. 5a–c), and ScURA-expressing cells activated genes involved in the integrated stress response similarly to WT cells (Extended Data Fig. We also noted that although the inferred transcriptional activity of MYC was blunted in both lines, the effect was less severe in ScURA-expressing cells, reflecting their sustained proliferation during mETC inhibition. When we analysed differentially expressed genes by gene set enrichment analysis in these conditions, the gene set that clearly stood out was cytoplasmic translation (Extended Data Fig. Indeed, complex III inhibition led to a profound downregulation of ribosomal protein mRNAs in WT cells. ScURA expression completely rescued this phenotype (Extended Data Fig. Ribosomal protein downregulation is a recognized consequence of mitochondrial dysfunction25. In the absence of a functional mETC, the shortage of CTP and UTP limits the transcription of genes encoding ribosomal proteins, while ScURA allows ribosomal protein mRNA synthesis by providing an alternative source of pyrimidines. We next sought to use ScURA in cellular models of mitochondrial diseases. Cells depleted of mtDNA, known as ρ0 cells, are dependent on uridine and pyruvate for growth4. We therefore asked whether we could revert the uridine auxotrophy in ρ0 cells with ScURA. Transduction of two ρ0 cell lines, mouse L929 ρ0 and human 143B ρ0, with the ScURA gene allowed growth in the absence of uridine, while maintaining their pyruvate dependence (Fig. In 143B ρ0 cells, ScURA was able to drive DHO oxidation and synthesis of UTP and CTP (Fig. Mitochondrial diseases are a heterogeneous group of maladies with extremely variable clinical manifestations; they are poorly predictable and often lack an effective treatment. Expression of the alternative oxidase (AOX) from Ciona intestinalis in human 293T cells confers cyanide resistance to cells27. Equally, expression of AOX from Emericella nidulans in mouse L929 ρ0 cells rescues uridine auxotrophy28. In mouse and human cells deficient for complex III or complex IV, AOX can also rescue uridine auxotrophy, ameliorating their metabolic phenotype29,30. In these scenarios, AOX restores the electron flux through the mETC, bypassing respiratory complex III and complex IV, allowing pyrimidine biosynthesis. To examine the implications of ScURA expression in cellular models of mitochondrial diseases, we expressed ScURA in Mt-Cybmut cells, which lack respiratory complex III because of a point mutation in the mtDNA cyb gene3, and in Cox10KO cells, which cannot assemble respiratory complex IV because of the absence of the COX10 assembly factor31 (Extended Data Fig. ScURA expression enabled the growth of both cell lines in the absence of uridine (Fig. We concluded that the growth defects caused by pathological mutations in mitochondrial genes can be alleviated by ScURA. a, Cell doublings of ScURA-expressing and control 143B ρ0 cells. Exogenous uridine and/or pyruvate were added to the growth media as indicated. b, Same experiment as in a, but with ScURA-expressing and control mouse L929 ρ0 cells. c, Volcano plot showing differential metabolite abundance in 143B ρ0 cells expressing either ScURA or an empty construct. Adjusted P values were calculated using two-sided Student's t-test. d, Normalized intensity values of dihydroorotate relative to 143B WT mtDNA-positive cells (P < 0.0001). e, Normalized intensity values of metabolites relative to 143B WT mtDNA-positive cells (UTP: ρ0 versus ρ0 + ScURA, P = 0.0006; all other significant comparisons, P < 0.0001). f, Cell doublings of ScURA-expressing, AOX-expressing and control Mt-Cybmut cells (+ScURA versus +AOX, P = 0.0001; all other significant comparisons, P < 0.0001). g, Cell doublings of ScURA-expressing, AOX-expressing and control Cox10KO cells (−uridine versus +uridine, P = 0.0007; −uridine versus +ScURA, P < 0.0001). Panels a, b and d are all significant comparisons (P < 0.0001). Ordinary one-way ANOVA, Tukey's multiple comparison test. Data are presented as means of n = 3 independent biological replicates; error bars, s.d. The ectopic expression of enzyme orthologs from other phyla has been a valuable tool for investigation of the mETC. For example, Lactobacillus brevis NADH oxidase generates NAD+, bypassing complex I, and has been used extensively both in cellular and animal models32,33. C. intestinalis AOX, which substitutes complex III and IV27, and S. cerevisiae NDI1, which substitutes complex I34, are being explored as therapeutic approaches for mitochondrial diseases5,35. We have extended this genetic toolkit with ScURA, which provides a mETC-independent pathway for orotate production. The reason why most eukaryotes rely on a mitochondrial DHODH is unknown. The connection between DHODH and mitochondrial electron transport could help to fine-tune key metabolic processes such as DNA and RNA synthesis with oxygen and nutrient availability. When oxygen supply is limited, a reduction in complex III and IV activity would deplete the UQ pool, slowing down pyrimidine biosynthesis and cell growth36. Decreased pyrimidine levels might, in turn, stimulate respiratory supercomplex assembly by ether phospholipid accumulation37. A mitochondrial DHODH might therefore support proliferation only in favourable environments. Oxidation of DHO is a fundamental reaction for living beings and requires a suitable electron acceptor. In eukaryotes, this process is accomplished mainly by three alternative strategies. Aerobes use oxygen as the terminal electron acceptor, which is reduced by DHODH through the mETC9. In anaerobic eukaryotes, there are two alternative modes of DHO oxidation, both of which use fumarate as the terminal electron acceptor. Many multicellular organisms that inhabit hypoxic environments oxidize DHO by a membrane DHODH, which transfers electrons to rhodoquinone and a membrane-associated fumarate reductase8. Some yeasts, instead, acquired a bacterial DHODH through horizontal gene transfer, which can directly reduce fumarate10. Recently, rhodoquinone has been found in some mammalian tissues, where it can allow mitochondrial electron transport from ubiquinol to fumarate independently of oxygen38. Experiments that combine the expression of ScURA and the induction of rhodoquinone synthesis could shed light on why eukaryotes have maintained membrane-associated reactions throughout evolution, instead of adopting soluble and more direct enzymatic workarounds. It has been shown that during hypoxia, ubiquinol can be oxidized by reverse SDH activity, allowing DHODH catalysis independently of complex III function20. This pathway can contribute to pyrimidine biosynthesis when complex III is inhibited if exogenous aspartate (or an exogenous electron acceptor such as pyruvate) is given to overcome the deficit in aspartate biosynthesis20,39,40. Given that our experiments were conducted in media complemented with dialysed serum and lacking exogenous aspartate or pyruvate, aspartate deficiency may explain why neither ScURA expression nor uridine supplementation could fully restore cell proliferation to control levels. Further investigation is required to address how nutrient availability constrains the efficacy of metabolic adaptations to respiratory deficiency. ScURA allowed us to evaluate the contribution of de novo pyrimidine biosynthesis to cellular growth independently of cellular respiration. In recent years, genetically engineered mouse models have enabled the study of a wide spectrum of cell-type-specific functions. With this approach, the integrity of the mETC has been shown to be essential for haematopoiesis41, angiogenesis42, regulatory and natural killer T cells functions43 and many more. Such dependence is often assayed by genetic disruption and pharmacologic inhibition of single mitochondrial respiratory complexes. By this approach, the activity of all UQ-dependent dehydrogenases is blocked, raising the question of which mETC-dependent process is responsible for the observed phenotype. The relative contribution of each UQ-dependent pathway can be assessed only by ectopic expression of UQ-independent dehydrogenases in vivo33,42. In this way, the development of effective and precise pharmacological intervention strategies can be undertaken following a precise mechanistic rationale. We designed ScURA following this same logic. One key issue that we think ScURA will help to disentangle is the metabolic requirements of tumour growth. Human tumour cells in which the integrity of mETC is disrupted cannot form tumours when injected subcutaneously into immunocompromised mice44. However, it has been shown in syngeneic models that tumour cells are able to uptake mtDNA from the host, and pyrimidine biosynthesis has been suggested to be the main limiting factor for their growth45. Interestingly, as it happened with the expression of AOX25, ρ0 cells expressing ScURA still needed pyruvate supplementation to grow, stressing that some activity of the mETC, probably respiratory complex I, is essential. ScURA expression in respiratory-deficient tumour cells will provide a straightforward model to directly test this and other hypotheses in vivo. A promising future area for the application of ScURA and other ectopic enzymes is in the treatment of mitochondrial diseases through adeno-associated virus gene therapy. For example, expression of NDI1 and AOX has been shown to improve the survival of pre-clinical models of complex I and III deficiencies, respectively35,46. ScURA gene delivery could also be beneficial in patients suffering from Miller's syndrome, a rare genetic disease characterized by mutations in DHODH47. Cells were grown in complete DMEM medium (D5796, containing 4,500 mg l−1 glucose, 2 mM L-glutamine) supplemented with dialysed 10% FBS (Sigma-Aldrich, F7524), 1% penicillin–streptomycin (PenStrep, Lonza), at 37 °C in an atmosphere of 5% CO2 and 95% air. Where indicated, 50 μg ml−1 uridine and/or 1 mM sodium pyruvate (Sigma-Aldrich) were added to the growing media. Cells were pre-treated with chloramphenicol for 1 week in the presence of uridine to deplete them of respiratory complexes48. Control cells were treated with the appropriate vehicle (ethanol or dimethylsulfoxide (DMSO)). The origin of cell lines was: ρ0 cells49, Mt-Cybmut and Cox10KO AOX-expressing cells29. The coding sequence of ScURA was obtained from a previous publication50 and modified as follows. A myc tag was added at the C terminus through a (GGS)3 linker, and at both ends, restriction enzyme sites were inserted. Codon-optimization and oligonucleotide synthesis were performed by Genewiz. The gene was cloned into the pWXLd lentiviral vector (modified from Trono's lab with a puromycin resistance gene, Addgene). Lentivirus production was carried out by the Viral Vector unit at Centro Nacional de Investigaciones Cardiovasculares (CNIC). Viral supernatants were filtered and added to cells with 8 μg ml−1 polybrene overnight. Then, 2 days after transduction, ScURA-expressing cells were selected with puromycin 0.5 μg ml−1 for approximately 1 week until non-transduced cells in a control plate were dead. For each cell line, a cell line transduced with an empty vector was used as a control. On day 0, a total of 40,000 cells were plated into six-well plates. For each condition, three wells were incubated for 6 h (to allow the cells adhere to the plate), collected and counted. This count was recorded as the number of cells at day 0. For 3–4 days, fresh media and the appropriate treatments were added to each well every day. On the last day, three wells per condition were collected and counted. The number of doublings pre day was calculated as 24 / ((T × (ln2)) / (ln(nf / n0))), where T is time in hours, nf is the number of cells on the final day and n0 is the number of cells at day 0. Cells from two to ten 150 mm plates were collected with trypsin, washed in PBS and resuspended in cold sucrose buffer (0.32 M sucrose, 1 mM EDTA, 10 mM Tris-HCl pH 7.4). Cells were then homogenized in a Teflon potter-type tissue homogenizer with 20–40 ‘pops'. The homogenate was centrifuged at 1,000g for 5 min, the nuclear pellet was discarded and the post-nuclear whole-cell homogenate was stored at −80 °C. When mitochondria were isolated, the whole-cell homogenate obtained was further centrifuged at 10,000g for 10 min. The pellet obtained was resuspended in sucrose buffer and stored at −80 °C. DHODH activity was measured following a previously published method51, with some modifications: 100 µg of whole-cell homogenates were used for each reaction in a final volume of 200 µl. Each biological replicate was split into three technical replicates. DHODH-dependent reduction of 2,6-dichloroindophenol (DCPIP) was measured with a UV-visible spectrophotometer equipped with a 96-well plate reader. The rate of background (DHODH-independent) DCPIP reduction was measured in parallel by omitting DHO from the reaction mixture and was subtracted from the DHO-dependent rate for each sample. Brequinar was added where indicated at 0.05 mM; controls were treated with the same volume of DMSO. All gels were prepared in-house with Tris-HCl buffer and acrylamide/bis. Protein samples were quantified and loaded on the gel, and electrophoresis was performed in Tris-glycine solution. For denaturing electrophoresis, 0.1% SDS was added to the gels and protein samples were incubated for 1 min at 95 °C in loading buffer (50 mM Tris-HCl, pH 6.8, 2% SDS, 10% glycerol, 1% β-mercaptoethanol, 0.02% bromophenol blue) before loading. For non-denaturing native gels, SDS was not added to the gel, whole-cell homogenates were not boiled and the loading buffer used did not contain SDS or β-mercaptoethanol. Proteins were transferred to PVDF membrane (Immobilon-FL, 0.45 μm) by transfer in BioRad Mini Trans-Blot Cell or Trans-Blot Cell systems, in 48 mM Tris, 39 mM glycine and 20% methanol transfer solution overnight at 30 V. The membrane was then blocked in 0.1% PBS-Tween, 5% BSA solution for 1 h at room temperature (22 °C) and incubated with primary antibody overnight at 4 °C. After that, the membrane was incubated with secondary antibody for 1 h at room temperature. Membrane development was performed using fluorescent secondary antibody and revealed with the Odyssey imaging system (LI-COR Biosciences). Antibodies used included anti-hemoagglutinin-tag (1:5,000; Sigma-Aldrich, sAB 4300603), anti-betaActin (1:10,000; Sigma-Aldrich, A2066), anti-GAPDH (1:5,000; Abcam, AB8245) and anti-SDHA (1:5,000; Invitrogen, 459200 or Proteintech, 14865-1-AP). The procedure was carried out as detailed previously48. In brief, 100 μg of mitochondria were incubated with 400 μg of digitonin for 5 min in 50 mM NaCl buffer, 50 mM imidazole, 5 mM aminocaproic acid at a final concentration of 10 μg μl−1. Electrophoresis was performed in a cold chamber and developed for 30 min at 90 V with cathode buffer A. Then, the cathode buffer was exchanged for cathode buffer B, and electrophoresis continued for approximately 1 h more at 300 V. Measurement of NADH dehydrogenase activity of complex I was determined on the same gel after blue native–PAGE electrophoresis by incubating the gel in 0.1 M Tris-HCl, pH 7.4, 0.14 mM NADH and 1 mg ml−1 NitroBlue tetrazolium solution at room temperature. ATP synthesis was assessed as previously described52. In brief, 1.5 billion cells were permeabilized with digitonin and ATP synthesis was assayed using glutamate plus malate or succinate as fuels in the presence of adenosine 5′-diphosphate and diadenosine pentaphosphate (to inhibit the adenylyl kinase) by a kinetic luminescence assay using the luciferin–luciferase reaction. A total of 10,000 cells were plated in a Seahorse XF96 microplate the day before the assay. These conditions allow for the determination of the respiratory capacity of mitochondria through complex I and complex II. For measuring activities of mitochondrial complexes, frozen–thawed cells were used as previously detailed53. The amino acid sequence of DHODH from various species was retrieved from UniProt. The R packages msa54, ape55 and ggmsa56 were used for multiple sequence alignment, phylogenetic tree construction and generation. In total, 20,000 cells were seeded per well on a 12-well plate containing previously sterilized precision cover glasses (Marienfeld, 0117520) and left incubating at 37 °C and 5% CO2 in their corresponding medium. After 48 h, cells were fixed for 15 min at 37 °C in pre-warmed 4% paraformaldehyde (ThermoFisher, 043368.9M) diluted in 1× PBS, followed by three washes in 1× PBS. Fixed cells were permeabilized with 0.1% Triton-X100 in 1× PBS for 10 min at room temperature. Cells were then incubated in Alexa Fluor 405 N-hydroxysuccinimide (NHS) ester (1:5,000) (ThermoFisher, A30000) diluted in 1× PBS. All incubation steps were performed for 40 min at room temperature and followed by three 5 min washes in 1× PBS-Tween 0.1%. Samples were finally mounted on microscopy slides using ProLong Gold Antifade Reagent (Invitrogen, P36934). All microscopy images were collected on a Leica SP5 inverted confocal microscope equipped with a HCX PL APO λ blue ×63, 1.40 NA oil objective. Z-stacks containing 10 planes (with a step size of 0.34 µm) were acquired in a sequential line scanning mode using LAS-AF (v.2.6.0) software (Leica Microsystems) at a digital zoom of ×2, pixel size of 60 nm, scanner speed of 200 Hz and 3 line averages, with a pinhole size of 1 AU. The Tomm20 signal was excited using the Argon 488 laser line at 10% intensity, and the corresponding 505–555 nm emission was collected using a PMT detector set at 800 V gain and −0.1% offset. The NHS405 signal was excited using the UV diode 405 laser line at 2% intensity, and 420–485 nm emission was collected using a PMT detector at 900 V gain and −1% offset. Image analysis was mainly performed in FIJI 1.53t. Z-stacks were first projected into a single plane using the maximum intensity method. For mitochondria, the signal of Ch2 (NHS405) was subtracted to that of Ch1 (Tomm20-CL488) and the resulting image was then thresholded with the Mitochondria Analyzer plugin57 using the following parameters: remove particles (size, ≤0.1 µm2), subtract background (rolling, 2.0), sigma filter plus (radius, 2.0), enhance local contrast (max slope, 3.0), weighted mean threshold (block size, 1.45; C-value, 5.0) and despeckle and remove outliers (pixels, 4.0). Individual networks were isolated from whole thresholded images, and morphology parameters of mitochondria were calculated on a per-cell basis. For analysis of cell morphology, the NHS405 channel was first pre-processed in FIJI by applying CLAHE (block, 500; bin, 256; slope, 10) and a Gaussian blur (radius, 0.3 µm). The objects were then thresholded and their morphology analysed in Cellprofiler (v.4.2.8)58 using an adaptive Otsu method with three classes. U2OS (ATCC, HTB-96) and 143B (ATCC, CRL-8303) SDHAKO clonal cell lines were generated through nucleofection (Neon transfection system, Invitrogen) of ribonucleoprotein complexes prepared by incubating 2 µg of TrueCut Cas9 Protein v2 (Invitrogen, A36499) and 2 µg of a pair of single guide RNAs (sgRNAs) (sgRNA 1 target sequence at intron 1, AACCCTGAAGGCAGCCCAAG; sgRNA 2 target sequence at exon 4, GGGCCTGCTCCGTCATGTAG). DNA oligonucleotides used for sgRNA synthesis were designed using the CRISPOR algorithm (http://crispor.tefor.net), and sgRNAs were synthesized using the GeneArt Precision gRNA Synthesis Kit (Invitrogen, A29377). Nucleofection was carried out at 1230 V or 1300 V (four pulses of 10 ms) for U2OS or 143B cells, respectively. Following nucleofection, cells were seeded into six-well plates and allowed to recover for 48 h. For single-cell clone isolation, cells were sorted using a spectral cell sorter (Cytek Aurora CS). Genomic DNA was extracted, and a pair of primers (primer 1A, GAGTGTGCATCCGACATCCTC; primer 2A, CTCATCACCATTCTTTTGGCTG) flanking SDHA-targeted regions was used to perform PCR screening. Clones were considered positive for gene editing when the amplified product was 551 bp, indicating deletion of the whole DNA fragment between targeted regions. The absence of SDHA protein was confirmed by western blot against SDHA (Santa Cruz, sc-166909), performed following separation of total cell protein extracts on 10% SDS–PAGE gels. Positive clones were further validated by Sanger sequencing of the SDHA gene. A clonal 143B-DHODHKO cell line was generated by following a previously published protocol59, with slight modifications. Each pair of gene-specific sgRNAs (pair 1, CCGCGGAGGACTACGCAGAA; pair 2, ATAGAAACGCTCATCTCCCG) targeting different regions of the gene was cloned into a TCLV2 plasmid (Addgene, 87360)60. Lentiviral particles were made in the Viral Vectors Unit at CNIC. All 143B cells were seeded on six-well plates, and at 80% confluency, they were transduced with lentiviral particles in DMEM supplemented with only 2% FBS, which was changed the following day for fresh complete medium. After 48 h, infected cells were selected by treatment with 1 µg ml−1 puromycin in complete DMEM. After 48 h, surviving cells were treated with 2 µg ml−1 doxycycline to induce Cas9-2A-eGFP expression and were then sorted into a 96-well plate at one cell per well using a FACSAria cell sorter. Final clones were selected by observing cell death in medium lacking uridine and confirmed by running total cell protein extracts on a 12.5% SDS–PAGE gel and performing a western blot against DHODH (1:1,000) (Santa Cruz, sc166348). The 143B-DHODHKO cells were used to re-express a WT version of hDHODH, a version of the protein lacking the mitochondria peptide signal and the transmembrane alpha helix (hDHODHΔMTS) and codon-optimized ScURA protein fused to a c-Myc tag (ScURA-myc). The vectors used were constructed and packaged by VectorBuilder, and the lentiviral particles were generated by the Viral Vectors Unit at CNIC. 143B-DHODHKO cells were seeded on six-well plates, and at 80% confluency, they were transduced with lentiviral particles in DMEM supplemented with 2% FBS, which was changed the following day for fresh complete medium containing uridine. Infected cells were selected by treatment with 400 µg ml−1 hygromycin B for at least 4 days. Re-expression of the proteins was confirmed by running total cell protein extracts on a 12.5% SDS–PAGE gel and performing a western blot against DHODH (1:1,000) (Santa Cruz, sc166348) and c-myc (1:5,000). For each condition, cells were seeded in six-well plates (with one plate kept aside for cell counting) in DMEM + 10% dialysed FBS, no uridine. The day after seeding, media containing inhibitors (1 μM each antimycin and myxothiazol, 2 μM brequinar) and radio-labelled tracers (2 mM [α-15N]glutamine or [U-13C]glutamine; Cambridge Isotope Laboratories) were added. After 24 h of incubation, metabolites were extracted in metabolite extraction buffer (50% methanol, 30% acetonitrile, 20% water (all liquid chromatography–mass spectrometry grade), 5 µM valine-d8, CK isotopes and DLM-488). Cells were washed with PBS, and 500 μl of cold metabolite extraction buffer per million cells was added to the wells; the plate was then immediately placed at −80 °C for 20 min. Metabolites were separated on a Millipore SeQuant ZIC-pHILIC analytical column (5 µm, 2.1 × 150 mm) coupled to a guard column (2.1 × 20 mm, 5 µm). The mobile phase consisted of solvent A (20 mM ammonium carbonate with 2.5 µM medronic acid, adjusted to pH 9.7 with ammonium hydroxide) and solvent B (acetonitrile in water, 95:5, v/v). The column oven was maintained at 40 °C, and the autosampler was kept at 4 °C. Metabolite quantification was performed on a Vanquish Horizon UHPLC system coupled to an Orbitrap Exploris 120/240 mass spectrometer (Thermo Fisher Scientific) equipped with a heated electrospray ionization source. Ionization parameters were set to +3.5 kV for positive mode and −2.8 kV for negative mode, with an RF lens setting of 70, a heated capillary temperature of 320 °C and an auxiliary gas heater temperature of 280 °C. For MS1 acquisition, data were collected in full scan mode over a mass range of m/z = 70–900, using a standard automatic gain control target with an automatically determined maximum injection time. Data were acquired with polarity switching at an Orbitrap resolution of 120,000. Untargeted metabolite profiling was performed using the AcquireX Deep Scan workflow with iterative data-dependent acquisition on pooled samples. Full scans were acquired at a resolution of 60,000, with tandem mass spectrometry fragmentation at 30,000 resolution and a minimum intensity threshold of 5.0 × 103. Dynamic exclusion was triggered after a single event (10 s, ±5 ppm), and precursors were isolated with a 1.2 m/z window. Stepped higher-energy collisional dissociation energies of 30, 50 and 150 were applied, with mild trapping enabled to improve signal quality. Metabolite identification was performed using Compound Discoverer (v.3.2; Thermo Fisher Scientific). Compounds were assigned based on precursor m/z values within 5 ppm of the theoretical mass, fragment ion matches within 5 ppm to an internal spectral library of authentic standards analysed under identical data-dependent tandem mass spectrometry conditions (minimum match score of ≥70), and retention times within 5% of those of purified standards under the same chromatographic conditions. Peak area integration and chromatogram evaluation were carried out in TraceFinder (v.5.1; Thermo Fisher Scientific), and isotopologue distributions were corrected for natural isotope abundance using the AccuCor package61. These filtering steps retain stable metabolites consistently detected in the samples, thereby preserving only robust and reproducible features. Intensities were normalized to the valine-d8 internal standard to correct for sample-to-sample technical variations, followed by LOESS signal correction to account for a systematic run order-dependent drift. Missing values were imputed with the group mean, while features entirely absent in a group were substituted with half of the minimum detected intensity in the dataset. Subsequently, data were log2-transformed to stabilize variance and improve normality before statistical analysis. Principal component analysis was performed using the prcomp function from the R stats package64, and Hotelling's T2 test was applied to identify sample outliers using the qcc package65. Sample group differential abundance was assessed using two-sided Student's t-tests, and P values were adjusted for multiple testing using the Benjamini–Hochberg method from the R base stats package. Finally, readxl66 and tidyverse were used for general scripting67. RNA sequencing was performed by the CNIC Genomics Unit team. RNA libraries were produced using the Illumina TruSeq RNASeq kit and sequenced in an Illumina HiSeq 2500 Sequencer. In brief, following quality control with FastQC, adaptor sequences were removed using Trim Galore! Transcript-level abundances were quantified with Salmon, and gene-level read counts were generated using featureCounts. Raw counts were normalized using the variance stabilizing transformation function, and differential gene expression analysis was performed within the DESeq2 package69. P values were adjusted for multiple testing using the Benjamini–Hochberg procedure. Pathway enrichment analysis was performed using clusterProfiler70 with statistics obtained from DESeq2. Generation of heatmaps and figures was done in R using the pheatmap package. Statistical analyses were performed using GraphPad Prism 10. The statistical tests used, sample size and post hoc corrections are indicated in the figure legends. All statistical tests were two-tailed and had an alpha of 0.05 as the significance threshold. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. RNA-seq data have been deposited in the Gene Expression Omnibus database under accession number GSE314127. Source data are provided with this paper. Regulation of nucleotide metabolism in cancers and immune disorders. Acı́n-Pérez, R. et al. Respiratory complex III is required to maintain complex I in mammalian mitochondria. King, M. P. & Attardi, G. Human cells lacking mtDNA: repopulation with exogenous mitochondria by complementation. & Gravel, D. On auxotrophy for pyrimidines of respiration-deficient chick embryo cells. Björnberg, O., Rowland, P., Larsen, S. & Jensen, K. F. Active site of dihydroorotate dehydrogenase A from Lactococcus lactis investigated by chemical modification and mutagenesis. Müller, M. et al. Biochemistry and evolution of anaerobic energy metabolism in eukaryotes. Nagy, M., Lacroute, F. & Thomas, D. Divergent evolution of pyrimidine biosynthesis between anaerobic and aerobic yeasts. Gojković, Z. et al. Horizontal gene transfer promoted evolution of the ability to propagate under anaerobic conditions in yeasts. Hall, C., Brachat, S. & Dietrich, F. S. Contribution of horizontal gene transfer to the evolution of Saccharomyces cerevisiae. Ishtar Snoek, I. S. & Yde Steensma, H. Factors involved in anaerobic growth of Saccharomyces cerevisiae. & Kennedy, J. Superoxide production and electron transport in mitochondrial oxidation of dihydroorotic acid. Zameitat, E., Pierik, A. J., Zocher, K. & Löffler, M. Dihydroorotate dehydrogenase from Saccharomyces cerevisiae: spectroscopic investigations with the recombinant enzyme throw light on catalytic properties and metabolism of fumarate analogues. McLean, J. E., Neidhardt, E. A., Grossman, T. H. & Hedstrom, L. Multiple inhibitor analysis of the brequinar and leflunomide binding sites on human dihydroorotate dehydrogenase. Jagow, G. V. & Engel, W. D. Complete inhibition of electron transfer from ubiquinol to cytochrome b by the combined action of antimycin and myxothiazol. von Jagow, G., Ljungdahl, P. O., Graf, P., Ohnishi, T. & Trumpower, B. L. An inhibitor of mitochondrial respiration which binds to cytochrome b and displaces quinone from the iron-sulfur protein of the cytochrome bc1 complex. & Lehninger, A. L. Ubisemiquinone is the electron donor for superoxide formation by complex III of heart mitochondria. Fendt, S.-M. et al. Reductive glutamine metabolism is a function of the α-ketoglutarate to citrate ratio in cells. B. et al. Fumarate is a terminal electron acceptor in the mammalian electron transport chain. Mullen, A. R. et al. Reductive carboxylation supports growth in tumour cells with defective mitochondria. Chouchani, E. T. et al. Ischaemic accumulation of succinate controls reperfusion injury through mitochondrial ROS. Badia-I-Mompel, P. et al. decoupleR: ensemble of computational methods to infer biological activities from omics data. Selak, M. A. et al. Succinate links TCA cycle dysfunction to oncogenesis by inhibiting HIF-α prolyl hydroxylase. Quirós, P. M. et al. Multi-omics analysis identifies ATF4 as a key regulator of the mitochondrial stress response in mammals. Adant, I. et al. Pyruvate and uridine rescue the metabolic profile of OXPHOS dysfunction. Hakkaart, G. A. J., Dassa, E. P., Jacobs, H. T. & Rustin, P. Allotopic expression of a mitochondrial alternative oxidase confers cyanide resistance to human cell respiration. Restoration of electron transport without proton pumping in mammalian mitochondria. The CoQH2/CoQ ratio serves as a sensor of respiratory chain efficiency. Expression of the alternative oxidase complements cytochrome c oxidase deficiency in human cells. Diaz, F., Thomas, C. K., Garcia, S., Hernandez, D. & Moraes, C. T. Mice lacking COX10 in skeletal muscle recapitulate the phenotype of progressive mitochondrial myopathies associated with cytochrome c oxidase deficiency. Titov, D. V. et al. Complementation of mitochondrial electron transport chain by manipulation of the NAD+/NADH ratio. Goodman, R. P. et al. Hepatic NADH reductive stress underlies common variation in metabolic traits. Alternative oxidase-mediated respiration prevents lethal mitochondrial cardiomyopathy. Löffler, M. The biosynthetic pathway of pyrimidine (deoxy)nucleotides: a sensor of oxygen tension necessary for maintaining cell proliferation? Bennett, C. F. et al. Peroxisomal-derived ether phospholipids link nucleotides to respirasome assembly. Valeros, J. et al. Rhodoquinone carries electrons in the mammalian electron transport chain. Supporting aspartate biosynthesis is an essential function of respiration in proliferating cells. Birsoy, K. et al. An essential role of the mitochondrial electron transport chain in cell proliferation is to enable aspartate synthesis. The mitochondrial respiratory chain is essential for haematopoietic stem cell function. Diebold, L. P. et al. Mitochondrial complex III is necessary for endothelial cell proliferation during angiogenesis. Weng, X. et al. Mitochondrial metabolism is essential for invariant natural killer T cell development and function. Martínez-Reyes, I. et al. Mitochondrial ubiquinol oxidation is necessary for tumor growth. Bajzikova, M. et al. Reactivation of dihydroorotate dehydrogenase-driven pyrimidine biosynthesis restores tumor growth of respiration-deficient cancer cells. McElroy, G. S. et al. NAD+ regeneration rescues lifespan but not ataxia in a mouse model of brain mitochondrial complex I dysfunction. Ng, S. B. et al. Exome sequencing identifies the cause of a Mendelian disorder. Curtabbi, A. et al. Regulation of respiratory complex I assembly by FMN cofactor targeting. Knecht, W., Altekruse, D., Rotgeri, A., Gonski, S. & Löffler, M. Rat dihydroorotate dehydrogenase: isolation of the recombinant enzyme from mitochondria of insect cells. Vives-Bauza, C., Yang, L. & Manfredi, G. Assay of mitochondrial ATP synthesis in animal cells and tissues. Acin-Perez, R. et al. A novel approach to measure mitochondrial respiration in frozen biological samples. Bodenhofer, U., Bonatesta, E., Horejš-Kainrath, C. & Hochreiter, S. msa: an R package for multiple sequence alignment. Paradis, E. & Schliep, K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics 35, 526–528 (2019). Zhou, L. et al. ggmsa: a visual exploration tool for multiple sequence alignment and associated data. Chaudhry, A., Shi, R. & Luciani, D. S. A pipeline for multidimensional confocal analysis of mitochondrial morphology, function, and dynamics in pancreatic β-cells. Stirling, D. R. et al. CellProfiler 4: improvements in speed, utility and usability. Ran, F. A. et al. Genome engineering using the CRISPR–Cas9 system. Barger, C. J., Branick, C., Chee, L. & Karpf, A. R. Pan-cancer analyses reveal genomic features of FOXM1 overexpression in cancer. Su, X., Lu, W. & Rabinowitz, J. D. Metabolite spectral accuracy on orbitraps. & Xu, G. A data preprocessing strategy for metabolomics to reduce the mask effect in data analysis. Wen, B., Mei, Z., Zeng, C. & Liu, S. metaX: a flexible and comprehensive software for processing metabolomics data. Scrucca, L. qcc: an R package for quality control charting and statistical process control. Wickham, H. & Bryan, J. Readxl: read Excel files. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Using clusterProfiler to characterize multiomics data. Garcia-Alonso, L., Holland, C. H., Ibrahim, M. M., Turei, D. & Saez-Rodriguez, J. Benchmark and integration of resources for the estimation of human transcription factor activities. We are thankful to all GENOXPHOS group members for their scientific discussions contributing to this paper. We thank members of the CNIC facilities (Microscopy, Cell culture and Viral vectors) for their technical assistance. We thank A. Sokol for help with the metabolomics analyses. This study was supported by grants from Ministerio de Ciencia e Innovación (grants PID2021-127988OB-I00 and TED2021-131611B-100), Human Frontier Science Program (grant RGP0016/2018), Fundación Leducq (17CVD04) and Instituto de Salud Carlos III CIBERFES (CB16/10/00282) to J.A.E., as well as the European Union's Horizon 2020 research and innovation program under the Marie Skłodowska-Curie grant agreement no. The CNIC is supported by the Instituto de Salud Carlos III (ISCIII), the Ministerio de Ciencia, Innovación y Universidades (MICIU) and the Pro CNIC Foundation and is a Severo Ochoa Center of Excellence (grant CEX2020-001041-S funded by MICIU/AEI/10.13039/501100011033). is funded by the Alexander-von-Humboldt professorship awarded to C. Frezza and by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy—EXC 2030—390661388. are supported by Agencia Estatal de Investigación (grant no. Centro Nacional de Investigaciones Cardiovasculares Carlos III, Madrid, Spain Andrea Curtabbi, Sara Natalia Jaroszewicz, Rocío Sanz-Cortés, Rebeca Acín-Pérez, Raquel Martínez-de-Mena & José Antonio Enríquez CIBER de Fragilidad y Envejecimiento Saludable (CIBERFES), Madrid, Spain Rebeca Acín-Pérez, Raquel Martínez-de-Mena & José Antonio Enríquez University of Cologne, Faculty of Medicine and University Hospital Cologne, Institute for Metabolomics in Ageing, Cluster of Excellence Cellular Stress Responses in Aging-associated Diseases (CECAD), Cologne, Germany Dimitrios Prymidis, Maksym Cherevatenko & Christian Frezza María Jesús Esteban-Amo & Miguel A. de la Fuente Unit of Excellence Institute of Biomedicine and Molecular Genetics (IBGM), University of Valladolid and Spanish National Research Council (CSIC), Valladolid, Spain María Jesús Esteban-Amo & Miguel A. de la Fuente University of Cologne, Faculty of Mathematics and Natural Sciences, Institute for Genetics, Cluster of Excellence Cellular Stress Responses in Aging-associated Diseases (CECAD), Cologne, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar collected data and performed analysis for cell culture experiments. conducted data collection and analysis for metabolomics experiments. The authors declare no competing interests. Nature Metabolism thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Alfredo Gimenez-Cassina, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Coupled oxygen consumption rate of 143B cells expressing ScURA permeabilized with digitonin and incubated with ADP (basal) plus substrates for complex I (glutamate/malate, G/M) and complex II (succinate, Succ)-dependent respiration. For each replicate only the last measurement is shown. Ordinary one-way ANOVA, Sidak's multiple comparison test, n = 8(WT), 7(ScURA) biological replicates per condition. b, Respiratory complexes activities measured in freeze-thawed mitochondrial membranes with a Seahorse analyzer, n = 8 independent biological replicates per condition. c, Blue-Native gel electrophoresis and immunoblot analysis of respiratory complexes assembly in 143B cells, antibodies used are SDHA for complex II and UQCRC2 for complex III and supercomplex I + III. d, Blue-Native gel electrophoresis and in-gel complex I activity in 143B cells. Data in panels c and d are representative of two independent experiments each. e, Representative confocal microscopy pictures of 143B cells stained with an antibody against TOMM20 and 405 NHS ester fluorescent dye. g, Violin plots showing confocal microscopy analysis of mitochondria size and morphology parameters in 143B cells. In f and g, each dot represents an individual cell. Data are mean ± SD, two-sided Welch's t test, n = 3 independent biological replicates per condition, unless otherwise noted, *P < 0.05. a, Heatmap showing unsupervised sample clustering based on z-score scaled intensity values of TCA cycle metabolites and aspartate. Missing TCA cycle intermediates were not detected, n = 3 independent biological replicates. a, Left, immunoblot showing SDHA knock out in 143B WT cells and re-expression of ScURA. Right, average number of doublings per day of 143B SDHAKO cells treated with antimycin/myxothiazol or vehicle, with or without exogenous uridine supplementation. b, Left, immunoblot showing SDHA knock out in U2OS cells and re-expression of ScURA. Right, average number of doublings per day of U2OS SDHAKO cells treated with antimycin/myxothiazol or vehicle, with or without exogenous uridine supplementation. Data are mean ± SD, n = 3 independent biological replicates. a, Principal component analysis of gene expression data from 143B cells treated with ethanol vehicle or antimycin/myxothiazol (anti/myxo). b, Volcano plots showing differentially expressed genes from ScURA vs. WT 143B cells, antimycin/myxothiazol vs. vehicle in WT cells, and antimycin/myxothiazol vs. vehicle in ScURA-expressing cells; n = 4 independent biological replicates. c, Dot plot showing the top 10 significantly enriched GO biological process terms from a GSEA of ScURA vs. WT cells. Significance was determined using a permutation-based weighted Kolmogorov-Smirnov test (one-sided). P-values were adjusted using the Benjamini-Hochberg method. d, Bar plot of the top 5 most significant upregulated and downregulated transcription factors. The activity of each transcription factor was inferred using decoupleR based on gene expression data from ScURA vs. WT cells. e, Volcano plot showing differentially expressed genes from 143B ScURA vs. WT cells. Dots corresponding to genes whose transcription is activated by HIF1α are highlighted in red (log2FC > 0) or blue (log2FC < 0). Statistical analysis was performed using the DESeq2 R package. For panels b and e significance was determined using the Wald test (two-sided). P-values were adjusted for multiple comparisons using the Benjamini-Hochberg procedure to calculate adjusted p-values. a, As in Extended Data Fig. 4d, but transcription factor activity was inferred from gene expression data from WT cells treated with antimycin/myxothiazol vs. vehicle. b, Fold change in gene expression of integrated stress response genes, relative to 143B WT cells treated with vehicle. Ordinary one-way ANOVA, Sidak's multiple comparison test; P-values refer to the comparison to WT vehicle-treated cells. c, As in (a), but transcription factor activity was inferred from gene expression data from ScURA-expressing cells treated with antimycin/myxothiazol vs. vehicle. d, Dot plot showing the top 10 significantly enriched GO biological process terms from a GSEA of ScURA vs. WT cells, both treated with antimycin/myxothiazol. Significance was determined using a permutation-based weighted Kolmogorov-Smirnov test (one-sided). P-values were adjusted using the Benjamini-Hochberg method. e, Heatmap showing unsupervised sample clustering based on z-score scaled gene expression data of ribosomal protein and elongation factor genes. For panels a and c, scores represent the t-statistic from the uniariate linear model (Wald t-test, two-sided). a, Growth curves of ScURA-expressing and control 143 ρ0 cells. b, Same experiment as panel (a) but with mouse L929 ρ0 cells. c, Immunoblot showing ScURA (c-myc tagged) and E. nidulans AOX (hemagglutinin-tagged, HA) expression in Mt-Cybmut and Cox10KO cells. Antibodies against the two tags were used, estimated molecular weight is indicated next to the expressed proteins. d, Growth curves of ScURA-expressing, AOX-expressing and control Mt-Cybmut cells. e, Same experiment as panel (d) but with mouse Cox10KO cells. Data are mean ± SD, n = 3 independent biological replicates (last day counts). Quantified metabolites from semi-targeted metabolomics analysis. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Curtabbi, A., Jaroszewicz, S.N., Sanz-Cortés, R. et al. Ectopic expression of cytosolic DHODH uncouples de novo pyrimidine biosynthesis from mitochondrial electron transport. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a70260118/1d-anyons/'>Scientists Spotted Particles in Another Dimension. They Could Change Fundamental Physics.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>In our three-dimensional space, elementary particles neatly filter into either bosons or fermions. But in lower dimensions, that distinction gets a bit murky. When you drill down into the very fabric of reality—where elementary particles make up the matter that is you and me and everything around us in three-dimensional space—things get divided nicely into two categories: fermions and bosons. These two categories of particles are defined primarily by their atomic spin (in quantum mechanical terms), with bosons (photons, gluons, Higgs boson, W bosons, and Z bosons) having integer spin values and fermions (protons, neutrons, electrons, neutrinos) having half-integer spin values. This quasiparticle is essentially anything between a boson and a fermion, which explains why American physicist Frank Wilczek named them “any-ons,” or simply anyons. However, it wasn't until 2020 that these weird particles were observed experimentally in one-atom thick (two-dimensional) semiconductors. In two separate papers—both published in the journal Physical Review A—the researchers describe the recipe for adjustable anyons and their theoretical properties, exploring a new corner of this “third kingdom” of elementary particles. “Every particle in our universe seems to fit strictly into two categories: bosonic or fermionic. “With these works, we've now opened the door to improving our understanding of the fundamental properties of the quantum world and it's very exciting to see where theoretical and experimental physics take us from here.” Anyons only exist in lower dimensions because, to put it very simply, particles in those dimensions have fewer options for moving around. Raúl Hidalgo-Sacoton (a Ph.D. student at OIST) describes the “exchange factor” that exists in 3D space—the property demanding that when two electrons swap places, they must obey a simple rule governing the mathematical statistics of the event: Its square must be equal to 1, meaning that all particles are either -1 (fermions) or 1 (bosons). In this study, Busch, Hidalgo-Sacoto, and their colleague Doerte Blume (from the University of Oklahoma) found that this boson-fermion binary remains broken in one dimension, and they even discovered a method to specifically tune the exchange factor. Because of their limited movement in one dimension, particles must pass through each other, and this causes the exchange factor to differ from higher dimensions. “We've identified not only the possibility of existence of one-dimensional anyons, but we've also shown how their exchange statistics can be mapped,” Busch said in a press statement. “We're thrilled to see what future discoveries are made in this area, and what it can tell us about the fundamental physics of our universe.” Just Before Death, the Brain May Try to Stay Alive</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a70277820/architectural-marvel-worlds-driest-cities/'>Archaeologists Found an Architectural Marvel in One of the Ancient World's Driest Cities</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The ‘Ain Braq aqueduct, which runs through the Jabal al-Madhbah massif into Petra, was even more sophisticated than scientists previously believed. But Petra already had aqueducts and water systems thousands of years ago that were far ahead of any on fictional Arrakis—so no one in Petra ever needed a stillsuit to survive. Petra was an urban hub of the ancient world despite its location in a semiarid climate that brought hardly any rain. As an important administrative and trading center, Petra needed water for purposes beyond keeping its people alive. In the course of their research, Jungmann and his team discovered something that the aqueduct had been hiding for millennia. It featured not just one, but two different main conduits that kept water flowing into Petra. What the researchers didn't expect was to unearth the imprint of a second conduit that once held a pressurized lead pipeline supplying water to basins and reservoirs on az-Zantur hill. Measuring roughly 380 feet (116 meters) in length, the newly-discovered pipeline used a welded metal pipe that was made of lead and encased in mortar. Jungmann concluded that such a long and narrow lead pipe was probably used as an inverted siphon within the ‘Ain Braq aqueduct. Previously, one theory posited that the conduit could have taken a detour to supply water to another part of the city before arriving at its destination. A detour would have not even been viable, since only an uninterrupted pipeline that generates enough internal pressure can allow water in a siphon to run from the header tank to the basin or reservoir it was meant to fill. “Overall, this study underscores the potential for further research into Nabataean water management,” said Jungmann. “While some hypotheses about the aqueduct's two branches into the city have been proposed above, more fieldwork is required in order to provide a fuller understanding of the distribution system and its evolution over time.” Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. A Man and His Daughter Discovered Ancient Rock Art Newly Decoded Tablet Warns, 'A King Will Die' Messages Found on the Walls of Last Supper Site This Colossal ‘Hypergravity Machine' Can Compress Space and Time</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00472-5'>The science influencers going viral on TikTok to fight misinformation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 11:11:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Catriona Clarke is an engagement editor in Nature's London office. Video images courtesy of (clockwise from top left): Adam Levy, DM Operations,Simon Clark and The Gut Health Doctor. Adopting the overconfident tone that is common among social-media influencers, he proclaims: “Renewables are a scam!” The ‘PhD influencers' logging lab life on TikTok and Instagram The ‘PhD influencers' logging lab life on TikTok and Instagram The video, with almost 180,000 views, is an effort to fight misinformation by meeting people where they are, he says. Clark started making YouTube videos more than 15 years ago as a master's student in physics at the University of Oxford, UK. He wanted to help others learn about getting into elite universities and navigating the collegiate system. Clark is now also on Instagram, Facebook and the live-streaming service Twitch, where he leans on his scientific credentials to both communicate science and combat misinformation. According to a 2025 report by the Reuters Institute in Oxford and the University of Oxford, 65% of people worldwide now consume video on social media1. Increasingly, many individuals, especially young people, get their news from these platforms. But a lot of that ‘news' is created by anti-science influencers who build loyal followings, using their position as opinion leaders to promote climate denialism, conspiracy theories, vaccine scepticism, autism myths, sham treatments and other pseudoscience. A study last year, for example, analysed nearly 1,000 Instagram and TikTok posts about controversial medical screening tests. Another study analysed the 100 most popular TikTok videos on attention deficit hyperactivity disorder and found more than half to be misleading3. Social scientist Louisa Ha at Bowling Green State University in Ohio says that content creators who intend to mislead use clear tactics: “They pretend to be authentic, trustworthy and knowledgeable,” she says. Whether a video provokes people to engage with it matters more than the video's accuracy, or even whether people agree with it, says Clark. Can giant surveys of scientists fight misinformation on COVID, climate change and more? Can giant surveys of scientists fight misinformation on COVID, climate change and more? Some creators do simple things, such as wearing a shirt backwards, so that viewers will post a comment pointing it out, says Clark. As a registered dietician, Megan Rossi was struck by how many of her patients took diet advice from blogs and other online content. Rossi leans into her credibility as a research fellow in nutritional sciences at King's College London, often referring to herself as a “gut specialist/dietician”. What's important, says Burke-Garcia, who has been working with influencers in the public-health space since 2008 (“although we didn't call them influencers then”), is to support credible voices and evidence in these communities. In a qualitative study published last year, Burke-Garcia, public-health researcher Amy Leader at Thomas Jefferson University in Philadelphia, Pennsylvania, and their colleagues asked ten influencers of colour who had children aged 9 to 14 to spread the word about vaccination against human papillomavirus4. The influencers, whom the researchers provided with a fact sheet, created emotionally charged posts that acknowledged parents' fears and used storytelling to describe their personal journeys towards a vaccination decision. Surveys of people who follow these influencers, Burke-Garcia says, revealed that the followers were more likely to consider vaccinating their children after hearing from someone to whom they could relate. This relatability can help content creators to grow a loyal following. His videos show him doing science demonstrations ranging from standard classroom ones — what happens when you burn magnesium — to those requiring sophisticated set-ups and equipment. “This chicken breast represents you,” he says, before it bursts into flames. His expertise as a former secondary-school science technician comes through, even with his casual approach: “I don't really look like a typical scientist. And I use colloquial terms in my speech,” he says. When it comes to fighting misinformation, content creators take various approaches. Clark favours “pre-bunking” rather than debunking, which he finds less effective. He tries to reach a broad audience with solid facts that leave no room for ambiguity: “This is like vaccinating them against misinformation”. The ‘PhD influencers' logging lab life on TikTok and Instagram Can giant surveys of scientists fight misinformation on COVID, climate change and more? How to avoid nuclear war in an era of AI and misinformation How to wow a popular-science writer with your research expertise Four ways I ensured my research brought about real-world change How to wow a popular-science writer with your research expertise ‘I was nearly arrested': escaping Myanmar's military leadership for a PhD abroad The ‘astounding' rise of semaglutide — and what's next for weight-loss drugs Join HZAU's global faculty team to advance research with competitive benefits. The ‘PhD influencers' logging lab life on TikTok and Instagram Can giant surveys of scientists fight misinformation on COVID, climate change and more? How to avoid nuclear war in an era of AI and misinformation An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/photographer-finds-thousands-of-triassic-dinosaur-prints-on-sheer-mountain/'>These jaw-dropping photographs show a new ‘Triassic Park' of dinosaur prints in the Italian Alps</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A newfound site in the Italian Alps holds one of the largest collections of Triassic dinosaur footprints ever seen About 2,000 fossil footprints appear on this part of the mountain site's walls, researchers say. Last September photographer Elio Della Ferrera spotted thousands of dinosaur tracks traversing vertical rock faces in the Fraele Valley of Stelvio National Park, high in the Italian Alps. In a preliminary study, Dal Sasso and his team deduced that the prints were made by herds of large, herbivorous dinosaurs, probably prosauropods, ancestors of Jurassic sauropods such as Brontosaurus. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Studying this newly named “Triassic Park” will be challenging because it is so difficult to access, Dal Sasso says—researchers will have to rely on drones and remote sensing to study and digitally preserve the footprints. Humberto Basilio is a Mexican science journalist covering policy, health, misconduct, archaeology and the environment. He is also a former news intern at Scientific American. His work has been published in the New York Times, National Geographic, Science, Nature, and more. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/ai-is-entering-health-care-and-nurses-are-being-asked-to-trust-it/'>AI enters the exam room, and nurses are left to manage the fallout</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>When alerts misfire or can't explain themselves, nurses still carry the risk Adam Hart has been a nurse at St. Rose Dominican Hospital in Henderson, Nev., for 14 years. A few years ago, while assigned to help out in the emergency department, he was listening to the ambulance report on a patient who'd just arrived—an elderly woman with dangerously low blood pressure—when a sepsis flag flashed in the hospital's electronic system. It was protocol; in an emergency room, that often means speed. But when Hart examined the woman, he saw that she had a dialysis catheter below her collarbone. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. A physician overheard the escalating conversation and stepped in. Instead of fluids, the doctor ordered dopamine to raise the patient's blood pressure without adding volume—averting what Hart believed could have led to a life-threatening complication. (A hospital spokesperson said that they could not comment on a specific case but that the hospital views AI as “one of the many tools that supports, not supersedes, the expertise and judgment of our care teams.”) Over the past several years hospitals have woven algorithmic models into routine practice. Clinical care often relies on matching a patient's symptoms against rigid protocols—an environment ideal for automation. For an exhausted workforce, the appeal of handing off routine tasks such as documentation to AI is undeniable. The technologies already implemented span a spectrum from predictive models that calculate simple risk scores to agentic AI that promises autonomous decision-making—enabling systems to titrate a patient's oxygen flow or reprioritize an ER triage queue with little human input. A pilot project launched in Utah a few months ago uses chatbot technology with agentic capabilities to renew prescriptions, a move proponents say gives providers more time, although physician associations have opposed the removal of human oversight. Across the country, health systems are using similar tools to flag risks, ambiently listen to visits with patients, generate clinical notes, monitor patients via wearable devices, match participants to clinical trials, and even manage the logistics of operating rooms and intensive care unit transfers. The industry is chasing a vision of truly continuous care: a decision-making infrastructure that keeps tabs on patients between appointments by combining what's in the medical record—laboratory test results, imaging, notes, meds—with population data and with the data people generate on their own by using, for instance, wearables and food logs. It watches for meaningful changes, sends guidance or prompts, and flags cases that need human input. Others say clinicians must stay in the loop, using AI not as autopilot but as a tool to help them make sense of vast troves of data. At the same time, on many hospital floors these digital promises often fracture, according to Anaeze Offodile, chief strategy officer at Memorial Sloan Kettering Cancer Center in New York City. He notes that faulty algorithms, poor implementation and low return on investment have caused some projects to stall. In the past two years nurses in California and New York City have staged demonstrations to draw attention to unregulated algorithmic tools entering the health-care system, arguing that while hospitals invest in AI the bedside remains dangerously short-staffed. Hospitals across the U.S. widely adopted information health technology company Epic's sepsis-prediction algorithm. Later evaluations found it substantially less accurate than marketed. Epic says that studies in clinical settings have found its sepsis model improved outcomes and that it has since released a second version it claims performs better. Those pressures spilled onto the steps of New York City Hall last November, when members of the New York State Nurses Association rallied and then testified before the City Council's hospitals committee. They argued that some of the city's biggest private systems are pouring money into executives and AI projects while hospital units remain understaffed and nurses face escalating safety risks. As this story was going to press in mid-January, 15,000 nurses at hospital systems in New York City were on strike, demanding safer staffing levels and workplace protections. New AI-enabled monitoring models often arrive in hospitals with the same kind of hype that has accompanied AI in other industries. In 2023 UC Davis Health rolled out BioButton in its oncology bone marrow transplant unit, calling it “transformational.” The device, a small, hexagonal silicone sensor worn on a patient's chest, continuously tracked vital signs such as heart rate, temperature and breathing patterns. On the floor it frequently generated alerts that were difficult for nurses to interpret. For Melissa Beebe, a registered nurse who has worked at UC Davis Health for 17 years, the pings offered little actionable data. “This is where it became really problematic,” she says. “It was vague.” The notifications flagged changes in vital signs without specifics. Beebe says she often followed alarms that led nowhere. “I have my own internal alerts—‘something's wrong with this patient, I want to keep an eye on them'—and then the BioButton would have its own thing going on. As a union representative for the California Nurses Association at UC Davis Health, Beebe requested a formal discussion with hospital leadership before the devices were rolled out, as allowed by the union's contract. Among the nurses' concerns were how the device would work on different body types and how quickly they were expected to respond to alerts. Instead nurses were told the device could help with early detection of hemorrhagic strokes, which patients were particularly at risk for on her floor. “But the problem is that heart rate, temperature and respiratory rate, for a stroke, would be some pretty late signs of an issue,” she says. “You'd be kind of dying at that point.” Earlier signs of a hemorrhagic stroke may be difficulty rousing the patient, slurred speech or balance problems. (UC Davis Health said in a statement that it piloted BioButton alongside existing monitors and ultimately chose not to adopt it because its alerts did not offer a clear advantage over current monitoring.) It could be just the way their skin looks or feels to me.” Elven Mitchell, an intensive care nurse of 13 years now at Kaiser Permanente Hospital in Modesto, Calif., echoes that view. “Sometimes you can see a patient and, just looking at them, [know they're] not doing well. “We have five senses, and computers only get input.” Clinical care often relies on matching a patient's symptoms against rigid protocols—an environment ideal for automation. “And that turns out to be a bunch of really important stuff like, How are they answering questions? He estimates that roughly half of the alerts generated by a centralized monitoring team are false positives, yet hospital policy requires bedside staff to evaluate each one, pulling nurses away from patients already flagged as high risk. (Kaiser Permanente said in a statement that its AI monitoring tools are meant to support clinicians, with decisions remaining with care teams, and that the systems are rigorously tested and continuously monitored.) He wishes there were more regulation in the space because health-care decisions can, in extreme cases, be about life or death. Across interviews for this article, nurses consistently emphasized that they are not opposed to technology in the hospital. Many said they welcome tools that are carefully validated and demonstrably improve care. What has made them wary, they argue, is the rapid rollout of heavily marketed AI models whose performance in real-world settings falls short of promises. “You are creating mistrust in a generation of clinicians and providers,” warns one expert, who requested anonymity out of concern about professional repercussions. Hospitals themselves are sometimes bypassing safeguards that once governed the introduction of new medical technologies, says Nancy Hagans, nurse and president of the New York State Nurses Association. Obermeyer, the professor at Berkeley's School of Public Health, found that some algorithms used in patient care turned out to be racist. “They're being used to screen about 100 million to 150 million people every year for these kinds of decisions, so it's very widespread,” he says. “It does bring up the question of why we don't have a system for catching those things before they are deployed and start affecting all these important decisions,” he adds, comparing the introduction of AI tools in health care to medical drug development. Unlike with drugs, there is no single gatekeeper for AI; hospitals are often left to validate tools on their own. If a device performs differently across patients—missing some, overflagging others—the clinician inherits that, too. Over the past couple of years AI-based listening tools that record doctor-patient interactions and generate a clinical note to document the visit spread quickly through health care. Many institutions bought them hoping they'd save clinicians time. Many providers appreciate being freed from taking notes while talking to patients, but emerging evidence suggests the efficiency gains may be modest. “Everybody rushed in saying these things are magical; they're gonna save us hours. Those savings did not materialize,” says Nigam Shah, a professor of medicine at Stanford University and chief data scientist for Stanford Health Care. “What's the return on investment of saving six minutes per day?” Similar experiences have made some elite institutions wary of relying only on outside companies for algorithmic tools. A few years back Stanford Health Care, Mount Sinai Health System in New York City, and others brought AI development in-house so they could develop their own tools, test tools from vendors, tune them and defend them to clinicians. AI tools also need to say why they're recommending something and identify the specific signals that triggered the alert, not just present a score. Hospitals need to pay attention to human-machine interactions, says Suchi Saria, John C. Malone Associate Professor of Computer Science at Johns Hopkins University and director of the school's Machine Learning and Healthcare Lab. AI models, she argues, should function more like well-trained team members. “It's not gonna work if this new team member is disruptive. People aren't gonna use it,” Saria says. “If this new member is unintelligible, people aren't gonna use it.” Yet many institutions do not consult or co-create with their nurses and other frontline staff when considering or building new AI tools that will be used in patient care. “Ask nurses first, doctors second, and if the doctor and nurse disagree, believe the nurse, because they know what's really happening,” he says. To include more staff members in the process of developing AI tools, some institutions have implemented a bottom-up approach in addition to a top-down approach. “Many of the best ideas come from people closest to the work, so we created a process where anyone in the company can submit an idea,” says Robbie Freeman, a former bedside nurse and now chief digital transformation officer at Mount Sinai. The program has a high adoption rate, Freeman says, partly because that nurse is enthusiastically training her peers. Freeman says the goal is not to replace clinical judgment but to build tools clinicians will use—tools that can explain themselves. The next frontier arrived at Mount Sinai's cardiac-catheterization lab last year with a new agentic AI system called Sofiya. Instead of nurses calling patients ahead of a stenting procedure to provide instructions and answer questions, Sofiya now gives them a ring. The AI agent, designed with a “soft-spoken, calming” voice and depicted as a female model in scrubs on life-size promotional cutouts, saved Mount Sinai more than 200 nursing hours in five months, according to Annapoorna Kini, director of the cath lab. Last November, at a New York City Council meeting, Denash Forbes, a nurse at Mount Sinai for 37 years, testified that Sofiya's work must still be checked by nurses to ensure accuracy. Even Freeman admits there is a “ways to go” until this agentic AI will provide an integrated and seamless experience. Or maybe it will join the ranks of failed AI pilots. Hilke Schellmann is an Emmy Award-winning reporter and as associate professor of journalism at New York University. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00474-3'>Why sky-high pay for AI researchers is bad for the future of science</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 10:52:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Nathan E. Sanders is an affiliate of the Berkman Klein Center for Internet & Society at Harvard University in Cambridge, Massachusetts, USA. Bruce Schneier is a lecturer in public policy at the Harvard Kennedy School in Cambridge, Massachusetts, USA, and at the Munk School of Global Affairs & Public Policy at the University of Toronto, Canada. Young, successful AI researchers are increasingly choosing to leave academia for industry.Credit: Costfoto/NurPhoto via Getty In 2025, Google, Amazon, Microsoft and Meta collectively spent US$380 billion on building artificial-intelligence tools. Moreover, these firms are spending lavishly on one particular segment: top technical talent. Eyeing these generous payouts, technical experts earning more modest-salaries might well reconsider their career choices (see ‘Academic brain drain'). A 2025 paper reported that this was especially true for young, highly cited scholars: researchers who were about five years into their careers and whose work ranked among the most cited were 100 times more likely to move to industry the following year than were ten-year veterans whose work received an average number of citations, according to a model based on data from nearly seven million papers1. This outflow threatens the distinct roles of academic research in the scientific enterprise: innovation driven by curiosity rather than profit, as well as providing independent critique and ethical scrutiny. Why hire and manage an entire group of scientists or software engineers when one genius — or an AI agent — can outperform them? It's no coincidence that Google's Gemini 3 Pro AI model was launched with boasts of ‘PhD-level reasoning', a marketing strategy that is appealing to executives seeking to replace people with AI. But the lone-genius narrative is increasingly out of step with reality. A large-scale study of scientific publishing from 1900 to 2011 found that papers produced by larger collaborations consistently have greater impact than do those of smaller teams, even after accounting for self-citation2. Analyses of the most highly cited scientists show a similar pattern: their highest-impact works tend to be those papers with many authors3. A 2020 study of Nobel laureates reinforces this trend, revealing that — much like the wider scientific community — the average size of the teams that they publish with has steadily increased over time as scientific problems increase in scope and complexity4. Why an overreliance on AI-driven modelling is bad for science Why an overreliance on AI-driven modelling is bad for science From the detection of gravitational waves, which are ripples in space-time caused by massive cosmic events, to CRISPR-based gene editing, a precise method for cutting and modifying DNA, to recent AI breakthroughs in protein-structure prediction, the most consequential advances in modern science have been collective achievements. Although these successes are often associated with prominent individuals — senior scientists, Nobel laureates, patent holders — the work itself was driven by teams ranging from dozens to thousands of people and was built on decades of open science: shared data, methods, software and accumulated insight. Examples demonstrating this include the LIGO Scientific Collaboration, the global team that first detected gravitational waves; the Broad Institute of MIT and Harvard in Cambridge, Massachusetts, a leading genomics and biomedical-research centre behind many CRISPR advances; and even for-profit laboratories such as Google DeepMind in London, which drove advances in protein-structure prediction with its AlphaFold tool. By contrast, well-designed institutions amplify individual ability, sustain productivity beyond any one person's career and endure long after any single contributor is gone. Equally important, effective institutions distribute power in beneficial ways. Rather than vesting decision-making authority in the hands of one person, they have mechanisms for sharing control. Allocation committees decide how resources are used, scientific advisory boards set collective research priorities, and peer review determines which ideas enter the scientific record. This is why scientists, academics and policymakers should pay more attention to how AI research is organized and led, especially as the technology becomes essential across scientific disciplines. Used well, AI can support a more equitable scientific enterprise by empowering junior researchers who currently have access to few resources. Instead, some of today's wealthiest scientific institutions might think that they can deploy the same strategies as the tech industry uses and compete for top talent on financial terms — perhaps by getting funding from the same billionaires who back big tech. Indeed, wage inequality has been steadily growing within academia for decades6. But this is not a path that science should follow. If the AI bubble bursts, what will it mean for research? If the AI bubble bursts, what will it mean for research? If the AI bubble bursts, what will it mean for research? Why an overreliance on AI-driven modelling is bad for science Universities in exile: displaced scholars count the costs of starting afresh Epstein files reveal deeper ties to scientists than previously known Grant proposals drafted with AI help more likely to win NIH funding The funding system needs fixing — but it's not a ‘waste of time and money' The Department of Molecular Biology and Genetics in the College of Sciences at Koç University invites applications for full-time tenure-track facul... Laboratory of Regenerative Biology and Medicine (Dr. Yuval Rinkevich), CIMR Join HZAU's global faculty team to advance research with competitive benefits. If the AI bubble bursts, what will it mean for research? Why an overreliance on AI-driven modelling is bad for science An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225555.htm'>Scientists discover brain switches that clear Alzheimer's plaques</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 10:31:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists at Karolinska Institutet in Sweden and the RIKEN Center for Brain Science in Japan have identified two brain receptors that help regulate the breakdown of amyloid beta, the protein that builds up in Alzheimer's disease. Their findings suggest it may be possible to develop future medications that are both safer and more affordable than today's antibody based treatments. Alzheimer's disease is the leading cause of dementia and is marked by sticky clumps of amyloid beta (Aβ) forming plaques in the brain. The research team discovered that two somatostatin receptors, SST1 and SST4, work together to control neprilysin levels in the hippocampus, a region essential for memory. The researchers conducted experiments using genetically modified mice and laboratory grown cells. As a result, amyloid beta accumulated and the mice showed memory problems. The team also tested a compound designed to activate these two receptors. "Our findings show that the brain's own defence against amyloid beta can be strengthened by stimulating these receptors," says Per Nilsson, docent at the Department of Neurobiology, Care Sciences and Society, Karolinska Institutet. Many of the most advanced Alzheimer's therapies currently rely on antibodies. While these treatments can target amyloid, they are extremely expensive and may trigger significant side effects in some patients. "If we can instead develop small molecules that pass the blood-brain barrier, our hope is that we will be able to treat the disease at a significantly lower cost and without serious side effects," says Per Nilsson. SST1 and SST4 belong to a large family of proteins known as G protein-coupled receptors. These receptors are common drug targets because they are well understood and often respond to medications that can be produced at lower cost and taken in pill form. Prehistoric Victory Celebrations Were Far More Brutal Than We Thought Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00496-x'>What's behind ‘teensplaining'? Scientists should study this adolescent behaviour</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 10:19:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>One aspect of their unique communication style has attracted attention in popular culture: ‘teensplaining'. Prices may be subject to local taxes which are calculated during checkout Social-media bans won't work — there are better ways to keep kids safe The great rewiring: is social media really behind an epidemic of teenage mental illness? Why we don't really know what the public thinks about science NIH rolls back red tape on some experiments — spurring excitement and concern Join HZAU's global faculty team to advance research with competitive benefits. Social-media bans won't work — there are better ways to keep kids safe The great rewiring: is social media really behind an epidemic of teenage mental illness? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260215225602.htm'>One in three people carry this brain parasite but the body has a kill switch</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 05:31:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A parasite that may already be living in your brain has the ability to infect the very immune cells designed to eliminate it. New research from UVA Health explains how the body still manages to keep this infection under control. Toxoplasma gondii is a potentially dangerous parasite that infects warm blooded animals. Roughly one third of the global population is believed to carry Toxoplasma, yet most people never develop symptoms. When illness does occur, known as toxoplasmosis, it is most serious in individuals with weakened immune systems. "Understanding how the immune system fights Toxoplasma is important for several reasons. People with compromised immune systems are vulnerable to this infection, and now we have a better understanding of why and how we can help patients fight this infection." Harris and her team discovered that CD8+ T cells rely on a powerful enzyme called caspase-8 to control T. gondii. This happened even though both groups mounted strong immune responses against the infection. Mice with caspase-8 remained healthy, while those without it became severely ill and died. These findings indicate that caspase-8 plays a crucial role in limiting T. gondii inside T cells. The results also add to growing evidence that this enzyme is broadly important in helping the body control infectious threats. "We scoured the scientific literature to find examples of pathogens infecting T cells. The only pathogens that can live in CD8+ T cells have developed ways to mess with Caspase-8 function. Prior to our study, we had no idea that Caspase-8 was so important for protecting the brain from Toxoplasma." The research team included Lydia A. Sibley, Maureen N. Cowan, Abigail G. Kelly, NaaDedee A. Amadi, Isaac W. Babcock, Sydney A. Labuzan, Michael A. Kovacs, Samantha J. Batista, John R. Lukens and Harris. Funding for the research came from the National Institutes of Health, grants R01NS112516, R01NS134747, R21NS12855, T32GM008715, T32AI007496, T32AI007046, T32NS115657, F30AI154740, T32AI007496 and T32GM007267; a University of Virginia Pinn Scholars Award; a UVA Shannon Fellowship; and UVA's Strategic Investment Fund. Note: Content may be edited for style and length. Prehistoric Victory Celebrations Were Far More Brutal Than We Thought Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260216044002.htm'>This new blood test could detect cancer before it shows up on scans</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-17 01:50:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The innovation could eventually allow doctors to identify early warning signs of cancer and other diseases through a routine blood draw. Biomarkers such as proteins, fragments of DNA, and other molecules can signal whether cancer is present, how it is progressing, or a person's risk of developing it. The difficulty is that in the earliest stages of disease, these markers exist in extremely low concentrations, making them hard to measure with conventional tools. "Our sensor combines nanostructures made of DNA with quantum dots and CRISPR gene editing technology to detect faint biomarker signals using a light-based approach known as second harmonic generation (SHG)," said research team leader Han Zhang from Shenzhen University in China. "If successful, this approach could help make disease treatments simpler, potentially improve survival rates and lower overall healthcare costs." In Optica, Optica Publishing Group's journal for high-impact research, Zhang and his team reported that the device detected lung cancer biomarkers in patient samples at sub-attomolar levels. "It could also help advance personalized treatment options by allowing doctors to monitor a patient's biomarker levels daily or weekly to assess drug efficacy, rather than waiting months for imaging results." Most current biomarker tests require chemical amplification to increase tiny molecular signals, which adds time, complexity, and expense. The researchers aimed to create a direct detection strategy that eliminates those additional steps. In this design, SHG takes place on the surface of a two dimensional semiconductor called molybdenum disulfide (MoS₂). These structures hold quantum dots at carefully controlled distances from the MoS₂ surface. CRISPR-Cas gene editing technology was then incorporated to recognize specific biomarkers. Because SHG produces very little background noise, the system can detect extremely low biomarker concentrations with high sensitivity. "Instead of viewing DNA only as a biological substance, we use it as programmable building blocks, allowing us to assemble the components of our sensor with nanometer-level precision," said Zhang. "By combining optical nonlinear sensing, which effectively minimizes background noise, with an amplification-free design, our method offers a distinct balance of speed and precision." To evaluate real world performance, the researchers focused on miR-21, a microRNA biomarker associated with lung cancer. After confirming that the device could detect miR-21 in a controlled buffer solution, they tested it using human serum from lung cancer patients to simulate an actual blood test. "The sensor worked exceptionally well, showing that integrating optics, nanomaterials and biology can be an effective strategy to optimize a device," said Zhang. The researchers aim to develop a portable version that could be used at the bedside, in outpatient clinics, or in remote areas with limited medical resources. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            