
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-11-11</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03584-6'>Be rational: chimp decision-making is guided by the strongest evidence</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 14:15:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Charles Darwin studied apes in the zoo to observe their behaviour and emotions. Prices may be subject to local taxes which are calculated during checkout By the time you hear these bats, it's too late Exceptional young scholars worldwide with strong research achievements in relevant fields Institute of Nanotechnology and Intelligence (inAI), Jinan University, China. Seeking exceptional Senior/Junior PIs, Postdocs, and Core Specialists globally year-round An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69289923/ancient-maya-map-of-the-universe/'>Archaeologists Found an Ancient Maya Map of the Universe Carved Into Bedrock</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 14:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Among the early Maya, the site of Aguada Fénix would have been a place for everyone to gather for trade, socialization, and ritual ceremonies. We may earn commission if you buy from a link. When archaeologist Takeshi Inomata and his team began excavating for Maya ruins in the lowlands of what is now Tabasco on the Gulf Coast of Mexico, they had no idea what was buried just beneath their feet. They would go on to unearth a monumental structure that would change the way history sees the ancient Maya. It was also about 3,000 years old, and brought forth questions about the supposed lack of monumental architecture in early Maya civilization. Aguada Fénix shows that architecture could be built on a monumental scale without the feather-crowned kings or social hierarchies that would arise later in the history of the Maya—its impressive size rivals those of later Mesoamerican monuments. It also did not appear to be a settlement, since there was no evidence of permanent residences at the site, but is instead thought to have been a ritual gathering place. While the population of Aguada Fénix at the time this structure was used is unknown, Inomata thinks that the construction of a monument this size would have required at least a thousand people working to carve it out of the bedrock for several years. Azurite pigments had not previously been seen in Mesoamerican art produced before 100 B.C.E., and malachite pigments in cave murals appeared still later than those at the Aguada Fénix. Since the only locally sourced pigment in the Maya Lowlands was yellow ochre, malachite and azurite were probably brought by traders from copper deposits in the highlands and western parts of Mexico. Seashells likely represented water, aligning with the ritual importance of water for the Maya (water was seen as synonymous with life), which may also explain why canals were built at the site. It is possible that seashells held this additional significance for the later Maya. While the pit indicates that grand constructions could be (and were) produced prior to the implementation of strict social structures, there may have still been leaders among the people who flocked to Aguada Fénix. Inomata thinks that the site was probably designed by community leaders whose knowledge of the calendar and astronomical observations were seen as valuable. “Unlike Olmec sculptures that emphasized the images of rulers and supernatural beings tied to elite power, a stone sculpture and jade objects found at Aguada Fénix showed naturalistic representations of animals and a woman, which were images rooted in everyday experiences of people,” Inomata said. “Thus, Aguada Fénix adds to the observations that large-scale hydraulic features and monumental constructions could be built by groups without prominent social inequality.” Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. A New Predator Emerged From the Deepest Ocean A Carnivorous ‘Death Ball' Emerged from the Deep This Animal Could Help Humans Live for 200 Years Are There More Humans on Earth Than We Thought?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69277945/mysterious-holes-andes-mountains/'>These Mysterious Holes Could Unlock Clues About an Ancient Civilization</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>New research suggests that the holes were the site of an ancient marketplace, and Inca rulers may have used them as a fascinating accounting system. A team of researchers hailing from Australia to Florida think they have answer. “Why would ancient peoples make over 5,000 holes in the foothills of southern Peru? We don't know why they are here, but we have produced some promising new data that yield important clues and support novel theories about the site's use,” Jacob Bongers, digital archaeologist at the University of Sydney and lead author of a new study published in the journal Antiquity, said in a statement. The site, officially Monte Sierpe, which translates as “Serpent Mountain,” but is more commonly known as the “Band of Holes,” features roughly 5,200 aligned holes stretching nearly one mile in length, all about three-to-six-feet wide and up to three feet deep. A 1933 aerial photograph published in National Geographic set the research community on notice, and since then hypotheses surrounding the origins have included a defense system, storage, accounting system, water collection, fog capture, and gardening. Microbotanical analysis of sediment samples show the presence of maize (corn) and ancient wild plants commonly used to make baskets, which were likely stored in the holes. We know the pre-Hispanic population here was around 100,000 people. That would have just been the start, though. “Fundamentally, I view these holes as a type of social technology that brought people together and later became a large-scale accounting system under the Inca Empire,” Bongers said. and mapping it on the ground wasn't productive. “Once we had precision, low-altitude images it was immediately clear that this site was profoundly important and had to be scientifically studied,” Charles Stanish, University of South Florida researcher and senior co-author on the research, said in a statement. Positioned between two Inca administrative sites, near the intersection of a network of pre-Hispanic roads, and between the highlands and lower coast plain, the site could have served as a key gathering point. “This is an extraordinary discovery that expands understandings about the origins and diversity of Indigenous accounting practices within and beyond the Andes,” Bongers said. “This research contributes an important Andean case study on how past communities modified landscapes to bring people together and promote interaction.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Found an Ancient Maya Map of the Universe Archaeologists Found a Cult in an Ancient City Lost Revolutionary War Barracks Found in Virginia</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-03657-6'>China's new scientist visa is a ‘serious bid' for the world's top talent</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 11:59:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. The Chinese government hopes a new visa will encourage talented young researchers to move there.Credit: VCG via Getty China has introduced a visa that will allow young foreign researchers in science, technology, engineering and mathematics (STEM) to move there without having to secure a job first. South Korea, Singapore and several other countries have also launched STEM-oriented visa programmes. The K visa was officially rolled out on 1 October, but Nature understands that applications are yet to open. Few details about eligibility have been released, except that restrictions will apply on the basis of an applicant's age, education and work experience. That also includes people who teach or research STEM topics in such organizations. The visa is designed to help the country boost its competitiveness in areas that it deems important for its future, such as artificial intelligence (AI), robotics and new materials, says Yuen Yuen Ang, a political economist at Johns Hopkins University in in Baltimore, Maryland. As the US government cuts back on its support for science, Chinese leaders see an opportunity to attract the best foreign scientists, particularly those working in fundamental research, Ang says. Sponsorship is needed for most other Chinese visa types, Lu says. The visa is a smart move given China's desire to attract global talent, he adds. The country's immigration figures are relatively low compared with those of other leading science nations. It is unclear how many of them work in STEM, but some estimates show that China has 5,000–10,000 foreign researchers with a PhD, compared with at least 100,000 foreign scholars at this level in the United States. In late September, the administration of President Donald Trump imposed a US$100,000 fee on applications for an H-1B visa, which many foreign STEM researchers rely on to live in the country. “While the US seemed to be raising barriers, China sent a strong counter-signal that it was opening its doors,” says Changhee Kim, a human-resource researcher at Xi'an Jiaotong-Liverpool University in Suzhou, China. US and China sign new science pact — but with severe restrictions China made waves with Deepseek, but its real ambition is AI-driven industrial innovation Kenya's black rhino revival is a story of hope for conservationists Failure is not an option for Africa's newly launched medicines agency Exceptional young scholars worldwide with strong research achievements in relevant fields China made waves with Deepseek, but its real ambition is AI-driven industrial innovation An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02104-z'>Estrogen modulates reward prediction errors and reinforcement learning</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 10:33:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Gonadal hormones act throughout the brain and modulate psychiatric symptoms. Yet how hormones influence cognitive processes is unclear. Exogenous 17β-estradiol, the most potent estrogen, modulates dopamine in the nucleus accumbens core, which instantiates reward prediction errors (RPEs), the difference between received and expected reward. Here we show that following endogenous increases in 17β-estradiol, dopamine RPEs and behavioral sensitivity to previous rewards are enhanced, and nucleus accumbens core dopamine reuptake proteins are reduced. Rats adjusted how quickly they initiated trials in a task with varying reward states, balancing effort against expected rewards. Nucleus accumbens core dopamine reflected RPEs that influenced rats' initiation times. Higher 17β-estradiol predicted greater sensitivity to reward states and larger RPEs. Proteomics revealed reduced dopamine transporter expression following 17β-estradiol increases. Finally, knockdown of midbrain estrogen receptors suppressed sensitivity to reward states. Therefore, endogenous 17β-estradiol predicts dopamine reuptake and RPE signaling, and causally dictates the impact of previous rewards on behavior. Estrogenic hormones, including 17β-estradiol, rhythmically fluctuate over the reproductive cycle in mammalian females and act on estrogen receptors, which can regulate protein expression via transcription and activation of signal transduction pathways1,2,3. These hormonal fluctuations support physiological imperatives such as homeostasis, fertility and sexual behavior2, but, due to broad and diffuse mechanisms of action, also bind to receptors in brain regions that support cognitive behaviors, including decision-making. Dopamine signaling in the nucleus accumbens core (NAcc) is modulated by application of exogenous 17β-estradiol in slices of ovariectomized animals4,5. The NAcc is a part of the ventral striatum that is innervated by midbrain dopaminergic neurons in the ventral tegmental area (VTA), and is required for learning cue–outcome associations6 and determining the vigor of motivated behaviors7. Dopamine release in the NAcc is thought to instantiate reward prediction errors (RPEs) for reinforcement learning, which is a powerful theoretical framework for describing how animals learn through interacting with the environment8. In reinforcement learning, agents learn the value of taking actions in different states; those state–action values are iteratively updated based on RPEs, or the difference between received and expected rewards. A wealth of evidence suggests that dopamine release in the NAcc encodes RPEs9,10,11,12,13,14. Critically, manipulating dopamine release in this circuit has provided causal evidence that it acts as an RPE for reinforcement learning15,16,17. More precisely, converging synaptic inputs (for example, from cortex, hippocampus) onto medium spiny neurons, the principal cells of the striatum, are thought to convey the animal's state. Dopamine is thought to mediate plasticity at these synapses, such that animals will be more (or less) likely to take actions in a state that produced positive (or negative) RPEs. In actor–critic models of the basal ganglia, the NAcc is thought to act as a ‘critic' that learns the expected future reward in a given state (that is, state value)18,19, which is known to influence response vigor20,21,22,23. Therefore, RPEs in the NAcc, in addition to training animals to take reward-maximizing actions, likely influence whether actions in a state will be more or less vigorous7. The fact that exogenous 17β-estradiol influences dopamine release in the NAcc4,24 suggests that endogenous fluctuations could influence RPEs for reinforcement learning. However, how behaviors unrelated to reproduction, such as reward-driven learning, respond to endogenous fluctuations in 17β-estradiol is unresolved. 17β-estradiol is expressed at low levels in brain tissue25, and spatiotemporal dynamics of endogenous release are unknown. This makes it difficult to interpret exogenous application of synthetic estrogens as reflecting endogenous dynamics. To relate physiological fluctuations of 17β-estradiol to behavior and neural dynamics, we measured behavior and dopamine release in the NAcc over the reproductive cycle and suppressed 17β-estradiol's function in the midbrain by knocking down estrogen receptors. Challenges of this approach include the fact that certain reproductive stages are only expressed briefly (for example, proestrus lasts for 12 h once every 4–5 days), rats enter reproductive senescence unpredictably, and the likelihood of senescence increases with age, which becomes limiting for complex tasks with long training times. We overcame these challenges using a brute-force high-throughput behavioral training approach to train hundreds of rats on a rich decision-making task. We find that rats update their response vigor on a trial-by-trial basis based on dopamine RPEs in the NAcc. Rats exhibit enhanced learning when endogenous 17β-estradiol is high, and estrogen receptors in the midbrain causally promote reinforcement learning. We show that following endogenous increases in 17β-estradiol, dopamine transporters are expressed at lower levels in the NAcc, and dopamine RPEs are enhanced. This reveals new roles for both 17β-estradiol and dopamine reuptake mechanisms in shaping RPEs for reinforcement learning. We trained rats on a self-paced temporal wagering task that manipulated reward expectations by varying reward magnitudes over blocks of trials23 (Fig. Rats initiated trials by poking their nose in a center port. This triggered an auditory cue, the frequency of which indicated the volume of water reward offered on that trial (4, 8, 16, 32 or 64 μl). The reward was assigned to one of two side ports, indicated by a light-emitting diode (LED). The rat could wait for an uncued and unpredictable delay to obtain the reward, or at any time could opt out by poking in the other side port to start a new trial. On 15−25% of trials, rewards were withheld to assess how long rats were willing to wait for them (catch trials). a, Schematic of behavioral paradigm. b, Block structure of task with an example session (top) and reward distributions in each block (bottom). c, Mean detrended trial initiation times across blocks for one example rat (106 sessions). times in low and high blocks are significantly different; P <<< 0.0001, two-sided Wilcoxon rank-sum test; error bars are confidence intervals. d, Sensitivity of detrended initiation times to blocks (low–high block) across the population is significantly different from zero; one-sided Wilcoxon signed-rank test, N = 303. e, Schematic depicting reinforcement learning model. Initiation times are inversely proportional to state value (Vt) on trial (t), which gets updated by an RPE (δ) when a reward is offered on that trial (Rt). A learning rate (α) determines how much to weigh the previous rewards (Rt-n) in the estimation of state values. f, Median regression coefficients of detrended trial initiation times as a function of rewards on previous trials during mixed blocks for all rats. g, Median detrended initiation times for low and high blocks, N = 118 female rats in proestrus and diestrus and N = 185 male rats; Kruskal–Wallis test P = 1.04 × 10−6 for low blocks and P = 3.96 × 10−14 for high blocks, two-sided Wilcoxon signed-rank test P = 1.38 × 10−5 and d = 0.38 for low blocks proestrus versus diestrus, P = 5.94 × 10−8 and d = 0.48 for high blocks, two-sided Wilcoxon rank-sum test P = 2.91 × 10−7 and P = 1.57 × 10−14 for low and high blocks proestrus versus males, and P = 0.02 and P = 1.54 × 10−4 for low and high blocks diestrus versus males. h, Median block sensitivity (low–high) of detrended trial initiation times. Two-sided Wilcoxon signed-rank test for proestrus versus diestrus P = 3.50 × 10−8 and d = 0.51, two-sided Wilcoxon rank-sum test for proestrus versus males P = 1.99 × 10−12 and for diestrus versus males P = 1.27 × 10−3. i, Median detrended initiation times at transitions from relatively high-to-low value (blue) or low-to-high value (red) blocks have greater sensitivity to the change in value in proestrus than diestrus, N = 118. Causal filter spanning 15 trials; bars represent P < 0.05 using a two-sided Wilcoxon signed-rank test (actual P values listed in Supplementary Table 1). j, Mean regression coefficients are stronger in proestrus, one-way ANOVA P = 2.42 × 10−18 for one trial back (d = 0.47 for proestrus versus diestrus), P = 0.85 for two trials back, P = 1.07 × 10−7 for three trials back, P = 9.23 × 10−9 for four trials back, P = 0.02 for five trials back and P = 0.04 for six trials back). k, The distribution of the differences between regression coefficients of one trial back for proestrus and diestrus is significantly different from zero; one-sided Wilcoxon signed-rank test P = 3.91 × 10−9. Coefficients are more (less) negative in proestrus (diestrus). l, Block sensitivity is significantly correlated with serum 17β-estradiol levels across the cycle with N = 17 rats and 116 sessions; two-sided Pearson correlation P = 0.018. m, The differences in block sensitivity for proestrus and diestrus are significantly correlated with the change in 17β-estradiol levels on sampled sessions; two-sided Pearson correlation P = 0.013. Data in j and k only include rats with greater initiation times in low compared to high blocks (N = 114 female rats in proestrus and diestrus and N = 179 male rats). To manipulate reward expectations, rats experienced uncued blocks of trials in which they were presented with low (4, 8 or 16 μl) or high (16, 32 or 64 μl) reward volumes (16 μl was in both). These were interleaved with mixed blocks that offered all rewards (Fig. To control for potential trends due to satiety across the session, we regressed out initiation times against trial number. Detrended initiation times were faster in high blocks compared to low blocks (block sensitivity; Fig. Previous work has suggested that this pattern optimally balances the costs of vigor against the benefits of harvesting reward26. The sensitivity of initiation times to the reward blocks was robust across 303 male and female rats (Fig. We previously found that the decision of when to initiate a trial was governed by a reinforcement learning algorithm that updated the value of the environment or state using RPEs23,27. In actor–critic models, the critic learns the value of the state (V), or the discounted sum of future expected rewards (R) in that state, and updates state values by the following equation: Vt+1 = Vt + α(Rt − Vt), where Rt − Vt is the RPE (δ), and α is the learning rate (Fig. This algorithm, also called a delta rule, computes the average of previous rewards, with most recent rewards having the strongest impact. Indeed, regressing trial initiation times against previous reward offers yielded negative coefficients that declined exponentially11, with more recent rewards having a stronger influence on behavior (Fig. When a rat received a large reward, it initiated the next trial more quickly (that is, rewards and initiation times were inversely proportional), consistent with positive RPEs increasing state values. The exponentially decaying influence of previous rewards on value estimates is a hallmark of reinforcement learning algorithms, including the delta rule. Collectively, this suggests that rats estimate their future expected rewards by integrating over past rewards via a delta rule, and initiate trials more quickly when they anticipate higher future expected rewards. To determine whether hormonal fluctuations relate to reinforcement learning, we tracked female rats' reproductive or estrous cycles with vaginal cytology (Extended Data Fig. There are four stages of estrous, each lasting ~12–48 h. 17β-Estradiol is low during metestrus and diestrus and peaks during proestrus, which is followed by estrus, when ovulation occurs (Extended Data Fig. Serum 17β-estradiol expression patterns measured with an enzyme-linked immunosorbent assay (ELISA) validated stages identified by vaginal cytology (Extended Data Fig. When female rats were in proestrus, the stage where 17β-estradiol and progesterone peak, their initiation times were more sensitive to the reward blocks compared to when those same rats were in diestrus, when 17β-estradiol and progesterone are low (Fig. 1g,h and Extended Data Fig. Detrended initiation times showed stronger (weaker) modulation of initiation times following block transitions in proestrus (diestrus; Fig. The difference between trial initiation times in low and high blocks was enhanced in proestrus and estrus, when ovulation occurs (Extended Data Fig. In diestrus, females showed block sensitivity that was closer to males (Fig. Rats also adjusted how long they were willing to wait for rewards in each block23. There was no effect of the estrous cycle on how long they were willing to wait for rewards (Extended Data Fig. Moreover, we previously found that the wait times are governed by a different behavioral strategy that uses state inference, and that is distinct from the trial-by-trial reinforcement learning strategy that governs initiation times23. Because the initiation times reflect a reinforcement learning process that we hypothesize relies on dopaminergic RPEs, and they are hormonally modulated, we focused on that aspect of behavior in this study. To assess whether hormonal fluctuations modulate trial-by-trial learning at a finer timescale than reward blocks, we regressed detrended trial initiation times in mixed blocks in proestrus and diestrus against previous rewards. In proestrus, detrended initiation times were more strongly influenced by the reward on the most recent trial compared to diestrus and to males (Fig. The increased weight of the previous reward correlated with increased behavioral sensitivity to blocks, suggesting that these metrics of learning, although at different timescales, are related (Extended Data Fig. We next regressed detrended initiation times against the previous reward, but this time included stage as a categorical variable. Across rats, we found that there was a significant interaction between stage and the previous reward that increased the influence of the previous reward on behavior in proestrus (P = 8.43 × 10−5). We measured serum levels of 17β-estradiol with an ELISA in 17 rats performing the task across the estrous cycle. We first validated that this subset exhibited the behavioral effects we identified in the larger population (Extended Data Fig. We observed a significant correlation between serum 17β-estradiol and session-by-session learning across all stages of the cycle (Fig. We next tested whether variability in circulating 17β-estradiol could explain variability in the degree to which the estrous cycle modulated behavior across rats (change in block sensitivity between proestrus and diestrus; Fig. Using 11 rats, we first validated that 17β-estradiol was indeed higher in proestrus during these training sessions (Extended Data Fig. We uncovered a strong and significant correlation between changes in circulating levels of 17β-estradiol and hormonal modulation of block sensitivity, where rats with greater changes in 17β-estradiol had greater hormonal modulation of behavior (Fig. This suggests that the degree to which 17β-estradiol fluctuates over the cycle dictates whether the cycle influences learning and can explain why some rats show stronger or weaker modulation. Collectively, these data show significant correlations between reinforcement learning and 17β-estradiol as it fluctuates over the reproductive cycle. 1c), but still enhanced session-by-session learning in that stage (Extended Data Fig. 1e), this could reflect longer-lasting genomic effects following the transient 17β-estradiol surge in proestrus. We refrain from making claims about whether hormonal modulation of behavior, including enhanced sensitivity to previous rewards, is better or worse for performance, because the exact benefits (costs) that are maximized (minimized) are unobservable. Instead, we used a reinforcement learning framework to make precise descriptive statements about changes in learning over hormonal states. We next sought to characterize the neural correlates of enhanced reinforcement learning in proestrus. To test whether physiological patterns of dopamine signaling encode an RPE (Fig. 2a) that is modulated by the estrous cycle, we virally expressed the optical dopamine sensor GRABDA in the NAcc, and implanted a fiber optic over the injection site for fiber photometry (Fig. We used a static fluorophore (mCherry) to correct for brain motion (Methods and Extended Data Fig. We observed a strong phasic dopamine response when the rat heard the reward offer cue. Dopamine fluorescence at the offer cue scaled with the offered reward volume: there were positive phasic responses to larger rewards and negative responses to smaller rewards, consistent with an RPE (Fig. 2c,d and Extended Data Fig. We interpret this result as being consistent with a temporal-difference learning algorithm8,9, in which the RPE would propagate back to the earliest predictor of the value of the environment on each trial (here, the cue). a, Schematic depicting delta rule with RPE highlighted. b, NAcc GRABDA and mCherry virus injections in female rats, with fibers. Top left: schematic and representative example (top right) of GRABDA expression in green and mCherry in red. Bottom: fiber locations in the NAcc for nine female rats. c, Heat map of GRABDA signal in response to reward offer cue across >16,000 trials recorded from an example rat during mixed blocks and sorted by reward volume and peak response in the first 500 ms following offer cue. Time bins are 100 ms. d, Average response to offer cue during mixed blocks, separated by reward volume. e, Heat map of GRABDA signal in response to reward offer cue for 16 μl across >3,000 trials recorded from an example rat sorted by reward block and peak response in the first 500 ms following offer cue. f, Average response to 16 μl offer cue across the population, separated by reward block. g, Response to offer cue, separated by reward block and stage group for an example rat. Gray box represents the window used to calculate the change in area under the curve (ΔAUC) 500 ms after the offer cue in high–low blocks in h. h, Histogram of stage effect (proestrus–diestrus) on ΔAUC for offer cue response over rats shows stronger block encoding in proestrus; two-sided Wilcoxon signed-rank test P = 3.91 × 10−3. i, Dopamine ΔAUC in high–low blocks, as a function of circulating levels of 17β-estradiol across 41 sessions; Pearson correlation coefficient P = 0.019, N = 5. j, Response to offer cue, separated by reward volume and stage group for an example rat. k, Left: median population response to each reward volume, separated by reward block and min–max normalized. Right: median AUC of individual rat responses for the largest reward volume for high and mixed blocks. AUC is larger in proestrus than diestrus; two-sided Wilcoxon signed-rank test during mixed blocks at 16 μl (P = 0.039, d = 0.19), 32 μl (P = 0.02, d = 0.49) and 64 μl (P = 0.0039, d = 0.89) and during high blocks at 32 μl (P = 0.02, d = 0.53) and 64 μl (P = 0.0039, d = 1.64). *P < 0.05, all error bars are ± s.e.m. In g–k, data are baseline-corrected using 50 ms before the offer cue. We next examined the 16 μl reward offers, which appeared in all reward blocks, but with a different value relative to the average reward in each block. In reinforcement learning algorithms, the value of the state (against which rewards are compared) is the discounted sum of future expected rewards in that state. The response to the auditory cue predicting 16 μl was positive in low blocks, when average reward was low, and negative in high blocks, when average reward was high, consistent with an RPE (Fig. We found that phasic dopamine release was more strongly modulated by reward states in proestrus compared to diestrus (Fig. Similar to what was observed behaviorally, in proestrus, the phasic dopamine response to the offer cue was more sensitive to the reward blocks (Fig. For all rats, there was a larger difference in these values in proestrus (Fig. Furthermore, serum 17β-estradiol predicted variability in dopamine block sensitivity across sessions (Fig. 2i), establishing a significant correlation between 17β-estradiol and dopamine signaling. Enhanced dopamine sensitivity to reward states was driven by enhanced dopamine transients for cues predicting the largest rewards (32 μl and 64 μl) in proestrus (Fig. 2j,k and Extended Data Fig. Dopamine release at recording sites dorsal to the NAcc in the caudate putamen also exhibited hallmarks of RPE encoding during the offer cue (Extended Data Fig. However, dopamine at these sites was unaffected by the estrous cycle, suggesting that hormonal modulation of dopamine release may be specific to the NAcc. Consistent with previous studies28, we recently found that dopamine after the delay period exhibited dynamics that may reflect beliefs about hidden task states (whether the state was a rewarded or unrewarded trial)27. Dopamine aligned to the delay start exhibited negative ramps that have been interpreted as moment-by-moment negative RPEs as the animals waited without receiving a reward14 (Extended Data Fig. This pattern requires additional knowledge about the task structure as the probability of being in an unrewarded trial increases over time, so the reward cue is more unexpected given beliefs about the trial28. This RPE was also larger in proestrus compared to diestrus (Extended Data Fig. 7c,d), especially at longer delays, suggesting that RPEs reflecting state inference were also enhanced during proestrus. Notably, these RPEs cannot account for the modulation of initiation times by block over estrous as the delays did not change over blocks. To test whether enhanced RPEs at the offer cue could account for enhanced reinforcement learning in proestrus, we used a reinforcement learning model that predicts rats' trial initiation times23,27. In the model, initiation times were inversely related to state value, which was estimated with a delta rule (Fig. The model captured qualitative aspects of rats' behavior, including sensitivity of initiation times to the reward blocks and previous reward offers (Fig. To determine when RPEs were most strongly encoded, we fit the model to each rat's initiation times. Then we used the best-fitting parameters for each rat to estimate their trial-by-trial RPEs (Methods). We regressed RPEs from the behavioral model against the dopamine AUC in sliding windows around each event (Fig. We found the strongest encoding of RPEs (that is, largest regression coefficients) at the time of the offer cue, when rats hear the tone indicating the reward offer, consistent with a temporal-difference learning algorithm. a, Simulation of the value of the state across reward blocks (blue, low; purple, mixed; red, high) using a reinforcement learning model and predicted initiation times for one session. b, Model-predicted initiation times by block. c, Regression of model-predicted initiation times against rewards on previous trials during mixed blocks. d, Regression coefficients of model-predicted RPEs (estimated from each rat's best-fitting model parameters fit to the last ten trials of the block) as a function of dopamine in each time bin (100 ms) aligned to task events, with the strongest encoding at the offer cue. Error bars are the mean ± s.e.m. e, Left: median dopamine AUC at the offer cue as a function of model-predicted RPEs is greater for proestrus than diestrus for the two largest positive RPE bins; Wilcoxon signed-rank test P = 0.012, d = 0.57 and P = 0.012, d = 0.55. Right: difference in AUC for the largest RPE bin by rat. h–j, Increasing ϕ on large RPEs captured enhanced learning seen in proestrus, including greater block sensitivity (h), more pronounced behavioral changes at block transitions (i) and greater impact of previous rewards on initiation time (j). Error bars in i are the median ± s.e.m. k, Rewards on the current trial have a significantly larger impact on dopamine in proestrus compared to diestrus; Wilcoxon signed-rank test P = 0.02, d = 0.21. Error bars are the median ± s.e.m. l,m, Enhancing ϕ captured the increased regression coefficient on the current trial (l), while enhancing the learning rate, α, did not (m). To quantify encoding of RPEs in proestrus and diestrus, we used the behavioral model—fit to all stages of the estrous cycle—to generate trial-by-trial RPE estimates for each rat. We then separately computed the dopamine AUC for different RPE values in proestrus versus diestrus. We found that in proestrus, dopamine encodes RPEs with a greater dynamic range. Moreover, there is a significant increase in the amplitude of the phasic dopamine response for large, positive RPEs (Fig. 2g,h,j,k and Extended Data Fig. To account for these apparent effects on large positive RPEs, we modified the behavioral model to include a multiplicative gain (ϕ) on positive RPEs above the 90th percentile (Fig. This explicitly enhanced large positive RPEs; however, because these larger RPEs increased state values, this manipulation also produced a modest increase in the dynamic range of negative RPEs (Fig. 3f,g), consistent with dopamine measurements (Fig. Adding this ϕ parameter recapitulated the enhanced behavioral sensitivity to reward blocks in proestrus (Fig. Moreover, it changed the impact and time constant with which previous rewards influenced the model's behavior, and increased behavioral modulation at block transitions, consistent with behavior in proestrus compared to diestrus (Fig. Alternatively, an enhanced learning rate parameter could in principle increase the impact of previous rewards on behavior. However, an increased learning rate does not predict any change in the relationship between the current reward offer and the RPE. In proestrus, dopamine showed enhanced responses to the current reward offer, which was captured by changing the ϕ parameter in the model, but not the learning rate, suggesting that rats in proestrus specifically exhibit larger positive RPEs (Fig. Collectively, these data show that dopamine dynamics at the time of the offer cue encoded an RPE. Dopamine responses in the NAcc at this time point scaled with the offered reward (Fig. 2c,d), were inversely proportional to reward expectations (Fig. 2e,f), correlated with RPEs estimated from the behavioral model (Fig. 3d) and exhibited a greater dynamic range in proestrus compared to diestrus (Figs. Hormonal modulation of female rats' reinforcement learning could be accounted for by a multiplicative gain on large positive RPEs, which was consistent with dopamine measurements in the NAcc. To test this hypothesis, we evaluated dopamine on trials with different subsequent initiation times. In previous work, we found that initiation times were most strongly governed by RPEs following unrewarded trials23, so we restricted this analysis to those trials. Indeed, trials in which subsequent initiation times were fast showed larger phasic dopamine responses, and trials with long subsequent initiation times showed apparent dips in dopamine (Fig. There was a significant negative correlation between the subsequent initiation time on trial t + 1, and the amplitude of the dopamine on trial t, in all rats (Fig. a, Dopamine signal aligned to the offer cue split by the subsequent trial initiation time (<0.2, 0.4, 0.9, 2.5, 12 and >12 s), N = 9 rats. Inset histogram shows how log2(initiation times) were binned. b, Mean AUC of curves in a 500-ms window across nine rats shown in a. c, Two-sided Pearson correlation of the AUC and subsequent initiation time over rats; all P < 0.01. Actual P values are available in Supplementary Table 1. d, Schematic of the strategy used to specifically stimulate TH+ VTA terminals expressing ChR2 in the NAcc. e, ChR2-YFP colocalization with TH in VTA cell bodies and NAcc axons (×20 and ×63). Arrowheads point to labeled NAcc axons or VTA cell bodies. Example image is one of 90. f, TH+ axon terminals in the NAcc were stimulated with 465-nm light during the offer cue on 30% of trials. g, Simulated initiation times with and without an additive RPE gain on 30% of trials, using the behavioral model. h, Mean trial initiation times for an example TH-Cre rat for 10,892 trials with optogenetic stimulation and 17,206 trials without (control). i, Mean trial initiation times across the population normalized to high blocks in control sessions for rats expressing ChR2 in TH+ axons (left) and sham controls (right); two-sided Wilcoxon signed-rank test P = 0.02, d = 1.47 for low blocks and P = 0.03, d = 1.35 for high blocks. All error bars are the mean ± s.e.m., *P < 0.05. To test whether dopamine at this time point causally influenced initiation times, we used optogenetics to activate dopamine terminals in the NAcc. We expressed Cre-dependent excitatory opsins (adeno-associated virus 9 (AAV9)-DIO-ChR2.0) bilaterally in the VTA and substantia nigra of tyrosine hydroxylase (TH)-Cre rats and rats injected with a TH-Cre virus, and bilaterally implanted fiber optics in the NAcc (Fig. We histologically verified that ChR2 expression was largely restricted to neurons that were TH immunoreactive (Fig. On 30% of trials, we optogenetically stimulated midbrain dopamine terminals in the NAcc triggered when rats heard the tone indicating the reward offer. We reasoned that this manipulation was similar to providing an additive gain to RPEs because it would add an (approximately) constant amount of dopamine on stimulation trials. Simulating an additive RPE gain in the behavioral model predicted that rats should exhibit faster trial initiation times because positive prediction errors would increase state values (Fig. The model predicted a larger effect in low blocks, because it implemented a floor effect to capture the fact that rats can only initiate trials so quickly in high blocks. Indeed, we found that dopamine terminal stimulation decreased initiation times (Fig. There was no effect on trial initiation times in sham animals that did not express light-sensitive opsins (Fig. There was also no effect on wait times (two-way ANOVA, F = 0.24 and P = 0.91 for an interaction between reward and session type for TH-Cre rats and F = 0.207 and P = 0.103 for sham rats), further showing that dopamine at reward offer cue encodes an RPE that is selectively used to guide initiation times. Collectively, these results support the hypothesis that dopamine in the NAcc at the time of the offer cue acts as an RPE to update state values for subsequent initiation times. We note that while we used continuous light stimulation, our results are consistent with optogenetic studies that used pulse-based stimulation17,29,30. Estrogen receptors can produce longer-term genomic changes (that is, synthesis of new proteins that can last for hours to days), or activate rapid signaling cascades whose effects persist on shorter timescales (that is, for minutes or hours)3. To test for changes in protein expression, we performed quantitative mass spectrometry of the NAcc in diestrus, proestrus and estrus (N = 5–6 rats in each group). A total of 238 proteins were significantly decreased in expression in proestrus or estrus compared to diestrus, and 258 were increased. Gene Ontology analysis identified several proteins involved in dopamine signaling that were differentially expressed in these stages (Methods and Extended Data Fig. However, the only differentially expressed proteins that could account for the observed increase in phasic dopamine in proestrus and estrus were proteins involved in dopamine reuptake, which were expressed at lower levels in these stages (Fig. The dopamine transporter (DAT or Slc6a3) controls dopamine reuptake in the NAcc, and was expressed at significantly lower levels in proestrus compared to diestrus. The serotonin transporter (SERT or Slc6a4) is also capable of robust dopamine reuptake31, and was expressed at lower levels in proestrus and estrus compared to diestrus. Smpd3 (sphingomyelin phosphodiesterase 3) triggers fusion of DAT to the plasma membrane, and has been shown to make dopamine reuptake more efficacious32,33. Smpd3 was decreased in expression in estrus, suggesting a lower density of DAT in the plasma membrane during estrus, consistent with effects of 17β-estradiol applications on DAT localization in vitro33. Reduced plasmalemmal DAT and SERT should, in principle, lead to higher levels of extracellular dopamine. a, Volcano plot depicting significantly differentially expressed proteins (black), t-test and a Benjamini–Hochberg false discovery rate of 5%. b, Example images from immunohistochemical quantification of SERT (36–54 micrographs per stage) and DAT expression (50–54 micrographs per stage) at ×63 with overlaid mask identifying fluorescence pixels above a threshold. c, Area of mask segmentation for SERT was highest in diestrus micrographs; Kruskal–Wallis test for group effect P = 4.89 × 10−3 and Wilcoxon rank-sum post hoc tests P = 5.17 × 10−3, d = 2.2 for proestrus versus diestrus, P = 0.052, d = 1.13 for estrus versus diestrus and P = 0.024, d = 0.64 for proestrus versus estrus. Example micrographs from each stage. d, DAT was also reduced in proestrus compared to diestrus micrographs; two-sided Wilcoxon rank-sum test P = 3.66 × 10−5, d = 0.92. e, Example of DAT labeled with SIG and imaged with electron microscopy. Arrowheads point to putative DAT proteins. Minimum distances between plasma membrane and center of DAT particle are shown with dashed lines. Axons are highlighted in yellow. f, Proportion of 895 membranous and intracellular DAT particles (left) and their distance from the membrane, expressed as a cumulative probability (right), quantified from electron microscopy images does not change by stage; Wilcoxon rank-sum P = 1 for membranous versus intracellular and Kolmogorov–Smirnov test P = 0.91. g,h, Simulation of dopamine concentrations and fluorescence following different evoked firing rates (FRs), with fast (g) or slow (h) reuptake dynamics. i, Quantification of AUC in g and h. All error bars are the mean ± s.e.m., *P < 0.05, #P < 0.1. We verified that rats exhibited reduced levels of DAT and SERT in proestrus and estrus, using a different assay, immunohistochemistry, and a separate cohort of rats in diestrus, proestrus and estrus (N = 3 per group). All sections were co-incubated in the same solutions simultaneously, to ensure that differences in fluorescence did not reflect subtle differences in tissue processing. We quantified fluorescently labeled pixel area from confocal photomicrographs, where fluorescence indicated DAT or SERT immunoreactivity (Fig. Consistent with the proteomics results of reduced DAT and SERT expression in these stages, we found that significantly fewer pixels showed DAT and SERT immunoreactivity in proestrus and estrus compared to diestrus (Fig. These results, therefore, held across different assays (proteomics and immunohistochemistry) and cohorts of rats. To test whether there was also a change in subcellular localization of DAT in estrus, we characterized DAT localization at the ultrastructural level. Individual DAT particles were labeled with silver-intensified gold (SIG) and visualized with electron microscopy. There was no significant difference in the fraction of DAT particles that were membranous versus intracellular, nor was there a difference in DAT particles' proximity to the membrane between estrus and diestrus (Fig. Therefore, reduced levels of DAT and SERT expression, but not differences in subcellular localization of DAT, likely account for enhanced dopamine RPEs and learning in proestrus and estrus. How could changes in reuptake, which ostensibly would result in elevated levels of extracellular dopamine, account for the enhanced phasic responses we observed? We created a computational model in which action potentials of dopamine neurons caused instantaneous increases in dopamine concentration, which decayed with a rate that reflected reuptake dynamics, unless another action potential caused another instantaneous increase. The fluorescence signal at each time step was treated as a noisy observation of the underlying dopamine concentration34. The model assumed that the Poisson spike rate of the dopamine neurons increased or decreased (to zero) for positive or negative RPEs, respectively. When the dopamine concentration decayed more slowly, to simulate reduced reuptake, this increased the likelihood that dopamine from consecutive action potentials would temporally summate before decaying back to baseline. Averaged over trials, this temporal summation mechanism produced larger positive phasic responses (Fig. 5g–i), consistent with what we observed by measuring dopamine in proestrus at the offer cue (Figs. Moreover, increasing temporal summation modestly elevated baseline levels of dopamine, producing a greater dynamic range for negative RPEs, which is normally thought to be limited by the low background firing rates of dopamine neurons11. These results are consistent with previous studies showing that inhibition of DAT increases both tonic and phasic dopamine35,36. In our simulation, reduced reuptake produced the strongest enhancement of the largest phasic DA responses, similar to our empirical observation that large positive RPEs were most strongly enhanced in proestrus. Therefore, decreased expression of DAT and SERT could produce a greater dynamic range of phasic dopaminergic RPEs by promoting temporal summation of dopamine from consecutive action potentials. Many hormones fluctuate over the estrous cycle. We next sought to test whether estrogenic signaling in the midbrain, specifically, influenced reinforcement learning. Expression of DAT in the NAcc is restricted to dopaminergic axons from the midbrain. Therefore, we hypothesized that estrogen receptors, which can regulate protein expression via transcription and other mechanisms when bound to 17β-estradiol1,2,3, might modulate learning by influencing expression of DAT in dopaminergic neurons. We virally expressed short-hairpin RNA (shRNA), labeled with eYFP, to knockdown expression of the estrogen receptor ERα bilaterally in the VTA (shEsr1-eYFP; Fig. We used either a virus that was inducible by doxycycline (dox) in food and water (U6/tetO-shEsr1) or one that was constitutively active (lenti-shEsr1; Methods). For each virus, we used fluorescence in situ hybridization (FISH) of fixed tissue sections to measure levels of labeled Esr1 (ERα transcript). Quantification of labeled probes showed a significant reduction in Esr1 in rats virally expressing shRNAs (Fig. a, shEsr1-eYFP was injected bilaterally in the VTA. b, Expression of eYFP-labeled shEsr1 virus in the VTA (×20, top) after RNAscope. Expression of Esr1 in eYFP-labeled cells compared to cells not expressing eYFP (×63, bottom). Arrowheads point to lack of Esr1 expression in eYFP-labeled cells. c,d, Number of Esr1-labeled spots per photomicrograph of a z-stack taken from a rat injected with U6/tetO-shEsr1 (c) and lenti-shEsr1 (d) compared to not injected control tissue, expressed as a percentage of the control means, N = 14–15 micrographs. Esr1 is suppressed with shEsr1 compared to controls; two-sided Wilcoxon rank-sum tests for U6/tetO-shEsr1 P = 0.014, d = 1.0 and lenti-shEsr1 P = 2.41 × 10−3, d = 1.15. e, Detrended initiation time across the population before and after shEsr1 injection for low and high blocks. Block sensitivity (low–high) was significantly reduced after shEsr1; two-sided Wilcoxon signed-rank test P = 1.95 × 10−3, d = 0.75, N = 11 rats. f, Median change in block sensitivity is comparable to proestrus compared to diestrus and males in hormonally modulated rats; N = 120 rats. g, Reducing ϕ to below 1 replicates suppression of behavioral sensitivity with shEsr1. h, Serum osmolality is significantly correlated with serum 17β-estradiol; two-sided Pearson correlation P = 0.013. Line of best fit overlaid (N = 18). i, Water consumed per session, controlling for the total amount of time the rat was allotted in the task, was still reduced in proestrus compared to diestrus, and this reduction was not affected by shEsr1; N = 11 rats, two-sided Wilcoxon signed-rank test comparing difference in water consumption in proestrus and diestrus before injection to after shEsr1 P = 0.23 and d = −0.6. 17β-Estradiol reduces expression of dopamine transporters. This increases the amount of dopamine in the extracellular space, allowing for a greater dynamic range of RPEs and enhanced learning from previous rewards in proestrus. All error bars are the median ± s.e.m., unless otherwise noted. Genetic suppression of estrogen receptors in the midbrain significantly reduced rats' behavioral sensitivity to the reward blocks (Fig. This is consistent with the finding that block sensitivity was correlated with 17β-estradiol over the estrous cycle. We conclude that midbrain estrogen receptors promote reinforcement learning by increasing the impact of previous rewards on behavior. We note that the effects of shEsr1 on RPEs are correlatively drawn through these effects on behavior. In proestrus, rats also appeared to be less thirsty: their initiation times were significantly slower, they performed fewer behavioral trials and drank less water, compared to diestrus (Extended Data Fig. Measurements of serum osmolality indicated reduced thirst in proestrus, and were strongly correlated with serum levels of 17β-estradiol (Extended Data Fig. Notably, detrending the initiation times minimized satiety effects, and allowed us to resolve estrogenic influences on trial-by-trial reinforcement learning. However, because the ERα knockdown was specific to the VTA, this experiment provided an opportunity to directly test whether estrogenic modulation of reinforcement learning occurs independent of systemic effects on fluid balance. We compared rats' water consumption before and after knockdown of ERα. Despite reducing behavioral sensitivity to the reward blocks, ERα knockdown had no effect on estrous-dependent water consumption (Fig. These results indicate that endogenous 17β-estradiol impacts reinforcement learning and fluid balance through dissociable mechanisms: the former relies on ERα in the VTA, but the latter does not. Using a task that provided a continuous behavioral measure of state values for reinforcement learning, we found that rats learn more from previous rewards following increases of endogenous 17β-estradiol, building on previous studies that found sex-dependent effects of reward on motivation37. We propose a computational and circuit-level mechanism that could result in midbrain 17β-estradiol-dependent enhanced learning: 17β-estradiol suppresses expression of DAT and SERT in the NAcc, leading to reduced reuptake of dopamine and a greater dynamic range of dopamine that enhances RPEs, which causally influence initiation times on subsequent trials (Fig. The enhancement is most pronounced for large positive RPEs, possibly due to increased occupancy of pools of reuptake proteins at high dopamine concentrations. Expression of DAT and SERT in the striatum is highly dynamic38,39, with protein turnover occurring over hours to days39,40,41, suggesting that plasmalemmal transporter expression could plausibly be reduced due to hormonal fluctuations in proestrus and remain reduced through estrus. One key future question is whether 17β-estradiol modulates the firing rates of dopaminergic cells in the VTA, in addition to dopamine dynamics in the NAcc. While previous studies have examined estrous-dependent VTA dynamics in the context of drug exposure42,43, it is unclear whether 17β-estradiol affects encoding of RPEs in the VTA during reinforcement learning. 17β-Estradiol could, in principle, act directly on VTA neurons44,45, or have indirect effects by modulating afferent inputs46. Medium spiny neurons in the NAcc project back to the VTA47, so it is possible that hormonal modulation of dopamine dynamics in the NAcc described here could indirectly influence VTA activity, via this projection. Here, we find a new role for dopamine transporters in shaping RPEs for reinforcement learning. Drugs of abuse such as cocaine and methamphetamine act on DAT48, and both DAT and SERT are the targets of pharmacological treatments for neuropsychiatric disorders including schizophrenia, depression and anxiety49. Computational psychiatry models have examined these disorders through the lens of reinforcement learning50,51,52. Our results provide a potential biological mechanism that bridges dopamine reuptake and reinforcement learning in health and disease, and might also underlie hormonal modulation of symptom severity in females53,54,55. The mechanisms by which ERα activation leads to reduced expression of dopamine reuptake proteins are important future directions. These could involve ERα-induced signal transduction pathways that control transcription, translation or protein degradation, or a mechanism involving other hormone receptors such as the progesterone receptor, a canonical target of ERα56. We used a reinforcement learning model to characterize dopamine RPEs over the reproductive cycle. However, recent findings that dopamine neurons exhibit heterogeneous responses16,57,58 and respond to unexpected events independent of reward59,60,61 have challenged the RPE hypothesis of dopamine. One proposal is that dopamine encodes not RPEs but generalized sensory prediction errors, and these prediction errors are vector-valued as opposed to scalar59,60,61. Sensory prediction errors would support learning about sensory associations (that is, for successor representation), with different prediction errors broadcast to distinct downstream circuits. In this scenario, the RPE would be one element of the vectorized prediction error60,61. Based on previous studies, it seems likely that the NAcc would receive the RPE from that vectorized signal9,10,11,12,13,14,15,16,17. An alternative proposal has argued that dopamine neurons encode RPEs (as opposed to sensory prediction errors), but different neurons encode RPEs for distinct features of the state representation, producing heterogeneity across neurons that can include reward-independent responses62. In this framework, as in theories of distributional reinforcement learning63, pooling the activity of many dopamine neurons by measuring release would approximate scalar RPE62. Therefore, our interpretation of dopamine release in the NAcc as encoding RPEs is consistent with any of these theories. While our data are also consistent with a proposal that dopamine signals prediction errors related to causal inference64, this new theory is still being tested65. A deeper challenge to the RPE hypothesis has come from studies showing that optogenetically perturbing dopamine can influence learning of neutral sensory associations66,67 without affecting conditioned responding67, suggesting that dopamine might support valueless associative learning rather than value learning. However, we recently found that cholinergic dynamics determine whether dopamine supports striatal-dependent value learning68. Therefore, cholinergic dynamics in the striatum could have mitigated value learning in these studies, whereas sensory associations may reflect dopamine-dependent learning in other circuits. Other findings, including valence-independent dopamine responses69, may be difficult to interpret without knowledge of concurrent striatal cholinergic dynamics, which may gate whether those signals act as RPEs68. There is a major gap in our understanding of how endogenous hormones interact with neuromodulatory systems during rich, quantifiable behaviors. We studied physiological hormonal fluctuations, as opposed to exogenous applications following ovariectomy, because the latter produces compensatory changes in the brain and body70, and likely only crudely approximates endogenous dynamics. Our computational approach provided a principled framework for interpreting behavioral, physiological, optogenetic and molecular profiling data, and represents a new approach for studying endocrine systems in the brain. This approach allowed us to characterize hormonal modulation of dopamine at broad levels of description, and opens up new avenues of research of hormone–neuromodulatory interactions during complex cognitive behaviors. A total of 369 Long–Evans rats (190 male, 179 female) between the ages of 6 and 24 months were used for this study (Rattus norvegicus), including TH-Cre (N = 23), ADORA2A-Cre (N = 10) and DRD1-Cre (N = 3). Animal use procedures were approved by the New York University Animal Welfare Committee (2021-1120) and carried out in accordance with National Institutes of Health (NIH) standards. Rats were housed in pairs when possible, and water restricted to motivate them to perform behavioral trials. From Monday to Friday, they obtained water during behavioral training sessions, which were typically 90 min per day, and a subsequent ad libitum period of ~20 min. Following training on Friday until midday on Sunday, they received ad libitum water. We have previously published a detailed description of behavioral training for this task23. Reward blocks were 40 completed trials, in which the rat remained nose poking in the center port for a variable period randomly drawn from a uniform distribution from (0.8, 1.2) seconds. On each trial, an auditory tone (1, 2, 4, 8 and 16 kHz) signaled one of the five different rewards. There was a one-to-one mapping between tones and reward volumes, from smallest to largest. On each trial, the reward delay period was randomly drawn from an exponential distribution with a mean of 2.5 s; the catch probability was 15−35% until the rat matched the catch rate with opt-out rate, and then lowered to 15−25%. Early cohorts of female rats experienced the same reward set as the males (5, 10, 20, 40 and 80 μl). However, female rats consumed less water due to their smaller body size and performed substantially fewer trials than the males. Reward offers for female rats were slightly reduced while maintaining the logarithmic spacing (4, 8, 16, 32 and 64 μl) to obtain a sufficient number of behavioral trials. For analysis, reward volumes were treated as equivalent to the corresponding volume for the male rats. To determine when rats were sufficiently trained to understand the mapping between the auditory cues and water rewards, we evaluated their wait times on catch trials as a function of offered rewards. Only sessions with a running average slope (using a window of five sessions) above 0.01 (the average slope of shuffled data) were included to accommodate cycle-dependent changes in task performance. If fewer than three sessions per stage were included in the analysis, then these criteria were not imposed. For the comparison of water consumed over the estrous cycle (Extended Data Fig. 10), we wanted to use the same sessions for all analyses, so we imposed these criteria for behavioral performance. For the shEsr1 experiment, screening for behavioral performance did not provide sufficient sessions for the analysis of water consumption (Fig. Therefore, for this analysis, we included all sessions when analyzing water consumption. We emphasize that the criteria for including sessions did not evaluate rats' sensitivity to the reward blocks, or their trial initiation times. We tracked the estrous cycle for all females included in this study using vaginal cytology, with vaginal swabs collected immediately after each session using a cotton-tipped applicator first dipped in deionized water. Samples were smeared onto a clean glass slide and visually classified under bright-field illumination on a Leica ICC50 E microscope at ×10. The estrous cycle stage on each day was manually determined on the basis of the proportion of leukocytes, cornified epithelial cells and nucleated epithelial cells according to established protocols using unstained smears71. Given that experiments were performed over the course of months, days to weeks of irregular cycling were captured and included in the data presented. Once the rat entered reproductive senescence, determined by a continuous estrus phase lasting longer than 2 weeks, all sessions during that time were excluded from the analyses. To validate the accuracy of our visual classification, we measured 17β-estradiol in serum from trunk blood, collected in serum-separating tubes and spun at 3,000 rpm at 4 °C for 10 min, with the mouse/rat estradiol ELISA kit from Calbiotech (ES180S-100) after determining the stage using the method described above in 18 rats (4 in proestrus >5 h before lights out; 4 in proestrus within 3 h before lights out; 5 in estrus; 5 in diestrus), run in triplicates. Behavioral data were collected from proestrus rats during the window when their 17β-estradiol was elevated (that is, within 3 h before lights out). All other endogenous steroids tested were <0.0001% or undetectable. We measured serum 17β-estradiol in behaving rats using the Rat Estradiol ELISA kit from Cusabio (CSB-E05110r). For this kit, the minimum detectable amount of rat estradiol was typically less than 40 pg ml−1. No significant cross-reactivity or interference between rat estradiol and analogs was observed. The intra-assay precision was CV% < 15% and the inter-assay precision was CV% < 15%. For each rat, ~200 μl blood was drawn from the lateral saphenous vein twice a week and each stage was sampled at least twice. When fitting the model to rats' initiation times (for example, to estimate RPEs), we modeled the trial initiation times as previously described (see ref. Briefly, we modeled initiation times as being negatively correlated to state value according to equation (1)26,27: where Vt represents the recency-weighted average of all previous rewards and D is a scaling factor. The estimate of average reward depends on the learning rate parameter α with the recursive equation as given by equation (2): where Vt is the value of the state on trial t, rt is the reward on trial t, δt = rt − Vt is the RPE and α is a learning rate between 0 and 1. Because we previously found that rats exhibited a dynamic learning rate27, we restricted model fitting to the later portion of mixed blocks, when learning rates were stable, and only related dopamine to RPEs on late mixed-block trials. For this model, the activation function that converts values into a behavioral policy was a reciprocal function, which was chosen based on previous literature23,26, and our finding that we could reliably recover generative parameters from this model27. However, when simulating this model, because the reciprocal function is nonlinear, the results were sensitive to choices of other parameters (for example, the scaling factor). Therefore, to improve the interpretability of the model simulations and make them more robust to choices about parameter values, model simulations used a linear activation function, in which the initiation times were negatively correlated with state values, parameterized with an offset and slope parameter. All model simulations used a learning rate parameter of 0.6. We modeled the activity of dopamine neurons as Poisson spikes with a background firing rate that increased or decreased (to zero) to reflect positive or negative RPEs, respectively. The dopamine concentration exhibited instantaneous increases at action potentials, which decayed over time to reflect reuptake, according to equation (3): where ct is the dopamine concentration, γ determines the rate of decay or reuptake and zt = 1 indicates the presence of a spike at time t, or 0 otherwise. The measured dopamine fluorescence at each time point, yt, was treated as an observation of the true concentration with Gaussian noise, according to equation (4): We simulated 500 trials for each condition (positive and negative RPEs), and averaged the measured dopamine signal over those trials. We performed this simulation for γ = 0.65, corresponding to faster reuptake, and γ = 0.85, corresponding to slower reuptake. All surgeries were performed in female rats after 4 months of age with a Neurostar Robot Stereotaxic, using 3% isoflurane in oxygen at a flow rate of 2.5 l min−1 for induction and 2% isoflurane in oxygen at a flow rate of 1.75 l min−1 as maintenance for the duration of the procedure. VTA injections were conducted at a 10° angle from the midline and targeted AP −5.8; ML 0.4–0.6; DV −7.8–8.3. NAcc injections and implants targeted AP 1.3; ML 1.65; DV −6.9 with an 8–10° angle from the midline for bilateral implants. We used fiber photometry and the GRABDA dopamine sensor (AddGene, 140554) to measure dopamine activity during the task. AAV9-hsyn-GRAB_DA2h was injected into the NAcc. To control for motion artifacts, mCherry (AAV1-CB7-CI-mCherry-WPRE-RBG, AddGene, 105-544) was also expressed because it is constitutively active. In total, 60 nl of equal parts GRABDA and mCherry were delivered in each injection. Chronically implantable optic fibers (Thor labs) with a 400-μm core and 0.5-NA fiber optics were implanted unilaterally over the injection site (DV −6.7). Doric Lenses hardware and software (Doric Neuroscience Studio) were used to record fluorescence. Two-channel motion artifact correction was used to correct for movement artifacts, with mCherry as the activity-independent channel73. We photoactivated VTA axons in the NAcc by bilaterally expressing AAV9-EF1a-double floxed-hChR2 (AddGene, 20298), 500 nl per injection, in the VTA of TH-Cre rats (N = 5) or simultaneously expressing AAV9-rTH-PI-Cre (AddGene, 107788, N = 2,250 nl ChR2 and 250 nl TH-Cre) to restrict expression to dopaminergic projections. Two tapered fibers (0.66 NA, 1-mm emitting length, 11-mm implant length, 200-μm core diameter, Optogenix) were implanted in stimulation and sham rats as controls (N = 5). Continuous blue light (465-nm PlexBright LEDs, Plexon) was delivered randomly on 30% of trials for 500–700 ms during the center poke, reaching 7.5 mA. Sixteen rats (five in diestrus, five in proestrus and six in estrus), anesthetized with isoflurane, were decapitated and a brain matrix (Braintree Scientific) was used to make 1.0-mm coronal brain slices containing the central aspects of the NAcc (~0–1.6 anterior to bregma), kept on wet and dry ice. Bilateral 15-gauge punches of the NAcc were obtained and immediately placed on dry ice. Samples were reduced with dithiothreitol at 57 °C for 1 h (5 μl of 0.2 M) and alkylated with iodoacetamide at room temperature in the dark for 45 min (5 μl of 0.5 M). Urea concentration was lowered to <2 M by addition of 100 mM ammonium bicarbonate and sequencing-grade modified trypsin (1.5 μg; Promega) was added to each sample. Digestion proceeded overnight on a shaker at room temperature. To extract proteins, samples were de-salted with Sep-Pak C18 Cartridges. C18 Cartridges were conditioned by passing 1 ml of 90% acetonitrile in 0.5% acetic acid solution three times. Then they were equilibrated by passing 3 ml of 0.1% TFA solution through them two times. Acidified samples were slowly loaded onto corresponding C18 cartridge packing. Samples were de-salted by passing 1 ml of 0.1% TFA solution through the cartridge three times. Then the cartridge was washed with 1 ml of 0.5% acetic acid. Peptides were eluted into clean tubes by slowly passing 500 μl of 40% acetonitrile in 0.5% acetic acid, followed by the addition of 80% acetonitrile in 0.5% acetic acid. Samples were reconstituted in 30 μl of 100 mM HEPES solution, pH ~8.5, and labeled with TMT dissolved in 90 μl of 100% anhydrous ethanol for 5 min. Each sample was combined with its corresponding TMT label and incubated for 1 h. The reaction was quenched by adding 50 μl of 100 mM ammonium bicarbonate solution and incubating for 15 min. Samples were dried in a SpeedVac concentrator. The sample was then fractionated into 45 fractions using basic high-performance liquid chromatography fractionation, and the fractions were concatenated into 15 final fractions for liquid chromatography coupled with mass spectrometry (LC–MS) analysis where 1/10th of the fractions were analyzed individually. Liquid chromatography separation online with a mass spectrometer was performed using the autosampler of an EASY-nLC 1000 (Thermo Scientific). Peptides were gradient eluted from the column directly to an Eclipse mass spectrometer using a 1-h gradient (Thermo Scientific) with 2% acetonitrile and 0.5% acetic acid for solvent A and 80% acetonitrile and 0.5% acetic acid for solvent B. High-resolution full mass spectrometry spectra were acquired with a resolution of 60,000, an automatic gain control target of 4 × 105, with a maximum ion time of 50 ms, and scan range of 400 to 1,500 m/z. Following each full mass spectrometry scan, the top ten data-dependent higher-energy collisional dissociation tandem mass spectrometry (MS/MS) spectra were acquired. All MS/MS spectra were collected using the following instrument parameters: resolution of 60,000, automatic gain control target of 1 × 105, maximum ion time of 60 ms, one microscan, 0.7-m/z isolation window and normalized collision energy of 30. MS/MS spectra were searched against the rat database using MaxQuant. Only proteins that had at least two peptides detected and that were identified in at least three samples of one estrous stage were included in the analysis. Reporter ion intensity values were normalized and all protein intensities were log2 transformed. The mass spectrometric raw files are accessible at https://massive.ucsd.edu/ under accession MassIVE MSV00098185. Rats (N = 12) were anesthetized with isoflurane and perfused through the ascending aorta with 50 ml of phosphate buffered saline (PBS) containing 0.2 ml of heparin and then 4% paraformaldehyde dissolved in 0.1 M phosphate buffer, pH 7.4. The brains were removed from the calvariae and post-fixed for one week in 4% paraformaldehyde. Sections through the NAcc (50 μm) were cut with a vibratome, and stored in 0.01 M PBS and 0.05% azide. To enhance the antibody's penetration of tissue, free-floating vibratome sections underwent a freeze/thaw protocol, for which tissue was cryoprotected in increasing concentrations of dimethyl sulfoxide (DMSO; 5%, 10% and 20%) for 10 min each before rapid freezing in a beaker of 4-methylbutane chilled using dry ice and 100% ethanol as the heat conduit. After rapid freezing, the tissue was thawed by immersing in a bath of 20% DMSO at room temperature, before another bout of rapid freezing for a total of eight freeze/thaw sequences. Tissue was then incubated in decreasing concentrations of DMSO (10% and 5%) for 5 min each before returning to PBS74. Tissue was stained for DAT using the protocol described below, except with an anti-DAT concentration of 1:250 dilution that the tissue was treated with for 48 h. Secondary antibody incubation was overnight in PBS-BSA-azide containing ultra-small colloidal gold-conjugated goat anti-rabbit secondary antibody (1:100 dilution; EMSciences, 25181, lot no. Sections were rinsed in PBS and post-fixed in 2% glutaraldehyde in PBS for 10 min, and then the steps for silver intensification (KPL Silver Enhancer Kit, 5520-0021) were carried out to enlarge the ultra-small colloidal gold particles. The electron microscopy tissue processing was similar to the procedure performed before75, which consisted of osmium-free tissue processing in 1% tannic acid, 1% uranyl acetate and 0.2% iridium tetrabromide, each dissolved in maleate buffer (pH 6.0, 0.1 M), post-fixation in 1% uranyl acetate in 70% ethanol overnight, dehydration in 70%, 90% and 100% ethanol, followed by 100% acetone, infiltration in EPON 812 (EMSciences), flat-embed between Aclar plastic sheets (EMSciences) and capsule-embedded in BEEM capsules (EMSciences). Ultrathin sections were prepared using the Ultracut E ultratomicrotome and collected on nickel grids (EMSciences). Sections were counter-stained with lead citrate to improve contrast. Images were captured at a magnification of ×50,000 using the Hamamatsu CCD camera (1.2 megapixels) and software from AMT. For each animal, approximately 50 axons were sampled and quantified. Axons were considered to be DAT positive when one or more SIG particles could be detected along its plasma membrane or within its cytosol. Axons were identified to be forming symmetrical or asymmetrical synapses. SIG particles were categorized as either membranous (center of particle was perfectly aligned to plasma membrane) or intracellular (center of particle was within the boundaries of the axon). A total of 897 SIG particles were included in the analysis. AAV2-U6/TO-shEsr1-CMV-TetR-P2A-eGFP-KASH-pA was custom constructed from AAV backbone (AddGene, 82706)76 with the insertion of the shRNA sequence of Esr1 (ref. 77) and custom prepped by Wz Biosciences. pJEP11-AAV-U6/TO-sgRNA(Tet2)-CMV-TetR-P2A-eGFP-KASH-pA was a gift from J. Ploski (AddGene plasmid no. shEsr1 was also cloned into the lentiviral vector pLL3.7 as previously described78,79 (AddGene plasmid, 120720). Either AAV-U6TO-shEsr1-CMV-TetR-eGFP (U6/tetO-shEsr1, N = 5) or pLL3.7-shEsr1 (lenti-shEsr1, N = 4), which use the same shEsr1 sequence77, was injected bilaterally into the VTA in eight rats to knock down Esr1 expression. Dox was used to induce viral expression of U6/tetO-shEsr1, delivered via both ad lib food (Bio-Serv diet) and 10–20 min of ad lib dox-containing water (~3 mg ml−1; Sigma-Aldrich dox powder) such that each rat received 20 mg per kg body weight per day (below toxic levels)80. All behavioral sessions after the first day of the dox regimen were included in the analysis for U6/tetO-shEsr1-injected rats and after 21 days of expression for lenti-shEsr1-injected rats and compared to sessions that preceded surgery. Osmolality was measured in the same serum samples used to detect 17β-estradiol expression. The extracted serum was sent by refrigerated transportation to a commercial laboratory for osmolality determination using a freezing-point method (Cornell University Clinical Pathology Laboratory). Rats were euthanized following completion of behavioral testing for photometry, optogenetics and shRNA experiments and when they reached adulthood (10–11 weeks) for electron microscopy and immunohistochemistry quantification experiments. Sections (50 μm) of extracted brain tissue were sliced on a Leica VT1000 vibratome. Sections were treated with 1% hydrogen peroxide in 0.01 M PBS for 30 min, washed with 0.01 M PBS and blocked in 0.01 M PBS containing 0.05% sodium azide and 1% bovine serum albumin (PBS-BSA-azide) for 30 min before being treated with primary antibody overnight at room temperature. Primary antibody was removed with PBS rinses and then sections were treated with secondary antibodies for 1 h away from light. Primary and secondary antibodies were made in PBS-BSA-azide and primary antibodies additionally contained 0.2% Triton-X. To verify virus expression and implant location, endogenous fluorescent markers were amplified with rabbit anti-GFP (Thermo Fisher Scientific A11122, 1:2,000 dilution, lot no. 2083201) and imaged with an Olympus VS120 Virtual Slide Microscope. For light microscopy experiments, DAT was labeled with a 1:500 dilution of Sigma-Aldrich D6944 antibody (lot no. 0000124874), SERT was labeled with a 1:2,000 dilution of Chemicon AB1594P antibody (lot no. 18020403), and TH was labeled with a 1:400 dilution of Millipore MAB318 antibody (lot no. The secondary antibodies used were as follows (all 1:200 dilution and from Thermo Fisher Scientific): anti-rabbit Alexa Fluor 488 IgG A11008 (lot no. 2147635) for DAT, SERT and GFP, and goat anti-mouse Alexa Fluor 555 for TH (A21424, lot no. All slices were mounted on glass slides using Prolong Gold (Vector Laboratories). For verification of Esr1 knockdown by U6/tetO-shEsr1 and lenti-shEsr1 viruses, RNA FISH was performed using the ACD Bio RNAscope Multiplex Fluorescent v2 assay. Brains were cryoprotected in a solution of 10% sucrose/PBS and stored at 4 °C until the brain sank. Cryoprotection was repeated with 20% and 30% sucrose/PBS solutions. The brains were embedded in optimal cutting temperature solution and flash frozen with 2-methylbutane cooled to −80 °C. For tissue sectioning, a cryostat was used to collect 10-μm sections stamped onto superfrost plus slides. Sections were stained with target probes EGFP-O4-C1 (ACD Bio 538851, lot no. 24117A) to label GFP and Rn-Esr1-C2 (ACD Bio 317151-C2, lot no. RNAscope Multiplex Fluorescent Reagent Kit v2 (ACD Bio 323270) was used for the FISH assay. Control tissue was labeled with 3-Plex Positive Control Probe (ACD Bio 320891, lot no. 24121A) and 3-Plex Negative Control Probe (ACD Bio, 320871, lot no. DAPI was used as a counterstain. Photomicrographs were taken at a magnification of ×20 and ×63 with a Leica DM6 CS Confocal using standardized parameters developed to maximize signal-to-noise ratios. z-stacks were taken for Esr1 quantification at a step size of 0.3 μm. Photomicrographs were analyzed in ImageJ to quantify protein expression. A mask was generated to identify the location of fluorescing pixels. The total area of the mask (μm2) was used to determine expression levels of DAT and SERT. Detection settings were set based on filtering of background noise and accurate identification of spots on internal control sections, and standardized across all sections. The total number of spots in each image of a z-stack was used for analysis. No statistical method was used to predetermine sample size. Effect sizes (Cohen's d) for effects across the estrous cycle, which had matching sample sizes across groups, were calculated as given by equation (5): For independent samples, Cohen's d was calculated as given by equation (6): We excluded outlier trial initiation times that were above the 98th percentile of the rat's cumulative trial initiation time distribution pooled over sessions, except when less than three sessions per stage were used, then the cutoff was the 99th percentile. Sensitivity of trial initiation times to blocks was defined as the change in initiation time (mean in low–mean in high blocks). Where noted, the effect of satiety over the session on trial initiation times was accounted for by subtracting the regression of mean trial initiation time as a function of trial number. To assess trial initiation time sensitivity to previous offers, we regressed trial initiation time against previous rewards. We focused only on mixed blocks. Reward offers from a different block (for example, a previous high block) were given not a number (NaN) values. Then, we regressed against the previous nine reward offers, not including the current trial, along with a constant offset. We set the reward for violation, catch and opt-out trials to zero, because rats do not receive a reward on these trials. We used MATLAB's built-in regress function to perform the regression. With the coefficients, we found the first nonsignificant coefficient (coefficient with a 95% confidence interval containing zero), and set that coefficient and all following coefficients to zero, as these coefficients cannot meaningfully be interpreted as being different from zero. To determine the relationship between the estrous cycle and the effect of trial history on initiation times, we created a multiple linear regression model that included cycle stage in addition to reward history. When only two stages were considered, diestrus was set to 1 and proestrus was set to 2. When all four stages were considered, diestrus was set to 1, metestrus was set to 2, estrus was set to 3, and proestrus was set to 4 to parallel the cycle of 17β-estradiol and the distance from its peak. Wait times on catch trials were included in the analysis. Wait times greater than three times the standard deviation above the mean were considered outliers and excluded. Trials that followed violations were excluded from the data presented in Fig. Only control sessions that occurred before the first day of optogenetic stimulation were included in the analysis because optogenetic activation of dopamine terminals had effects on behavior that lasted beyond the day of stimulation. Initiation times were normalized to high blocks in control sessions for both optogenetic and sham groups. Proteins were determined to be significantly differentially expressed with a t-test and a Benjamini–Hochberg false discovery rate of 5% to correct for multiple comparisons. Significantly differentially expressed protein signatures identified with mass spectrometry were subjected to functional annotation. STRING database v12 was used to assess enrichment of Gene Ontology terms for Rattus norvegicus using a hypergeometric test and all identified proteins as background. According to https://geneontology.org/, 246 genes and their proteins are associated with dopamine; 92 of these proteins were identified by mass spectrometry, and 3 that were significantly decreased in expression are related to dopamine uptake and transport (Slc6a3, Slc6a4, and Smpd3). Osmolality measurements greater than 2.5 times the standard deviation above the mean were considered outliers and excluded. 17β-Estradiol expression for one rat in estrus was excluded when correlated with osmolality due to it being an outlier (less than one standard deviation below the mean of all samples). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The mass spectrometric raw files are accessible at https://massive.ucsd.edu/ under accession MassIVE MSV00098185. The rest of the data used in this study are deposited in Zenodo via https://doi.org/10.5281/zenodo.16903822 (ref. 88) were used for Extended Data. Code used to analyze the data and generate the figures is available at https://github.com/constantinoplelab/published/tree/main/EstrousRPEPaper/. Marino, M., Galluzzo, P. & Ascenzi, P. Estrogen signaling multiple pathways to impact gene transcription. Understanding the broad influence of sex hormones and sex differences in the brain. Vasudevan, N. & Pfaff, D. W. Non-genomic actions of estrogens and their interaction with genomic actions in the brain. Di Paolo, T. Modulation of brain dopamine transmission by sex steroids. Rapid effects of ovarian hormones in dorsal striatum and nucleus accumbens. B. Striatal circuits for reward learning and decision-making. The nucleus accumbens: an interface between cognition, emotion, and action. Sutton, R. S. & Barto, A. G. Reinforcement Learning: An Introduction (MIT Press, 2018). Schultz, W., Dayan, P. & Montague, P. R. A neural substrate of prediction and reward. & Schultz, W. Dopamine responses comply with basic assumptions of formal learning theory. Bayer, H. M. & Glimcher, P. W. Midbrain dopamine neurons encode a quantitative reward prediction error signal. Cohen, J. Y., Haesler, S., Vong, L., Lowell, B. & Uchida, N. Neuron-type-specific signals for reward and punishment in the ventral tegmental area. Day, J. J., Roitman, M. F., Wightman, R. M. & Carelli, R. M. Associative learning mediates dynamic shifts in dopamine signaling in the nucleus accumbens. Kim, H. R. et al. A unified framework for dopamine signals across timescales. Steinberg, E. E. et al. A causal link between prediction errors, dopamine neurons and learning. Reward and choice encoding in terminals of midbrain dopamine neurons depends on striatal target. A. et al. Mesolimbic dopamine signals the value of work. & Ruppin, E. Actor–critic models of the basal ganglia: New anatomical and computational perspectives. Chen, R. & Goldberg, J. H. Actor–critic reinforcement learning in the songbird. Niv, Y., Daw, N. D., Joel, D. & Dayan, P. Tonic dopamine: opportunity costs and the control of response vigor. Xu-Wilson, M., Zee, D. S. & Shadmehr, R. The intrinsic value of visual information affects saccade velocities. Wang, A. Y., Miura, K. & Uchida, N. The dorsomedial striatum encodes net expected return, critical for energizing performance vigor. Mah, A., Schiereck, S. S., Bossio, V. & Constantinople, C. M. Distinct value computations support rapid sequential decisions. Detection of estradiol in rat brain tissues: Contribution of local versus systemic production. Niv, Y., Joel, D. & Dayan, P. A normative perspective on motivation. Mah, A., Golden, C. & Constantinople, C. Mesolimbic dopamine encodes reward prediction errors independent of learning rates. Starkweather, C. K., Babayan, B. M., Uchida, N. & Gershman, S. J. Dopamine reward prediction errors reflect hidden-state inference across time. B. et al. Recombinase-driver rat lines: tools, techniques, and optogenetic application to dopamine-mediated reinforcement. Millard, S. J. et al. Cognitive representations of intracranial self-stimulation of midbrain dopamine neurons depend on stimulation frequency. Larsen, M. B. et al. Dopamine transport by the serotonin transporter: a mechanistically distinct mode of substrate translocation. Neutral sphingomyelinase 2 induces dopamine uptake through regulation of intracellular calcium. Watson, C. S. et al. Estradiol effects on the dopamine transporter–protein levels, subcellular location, and function. B. Inferring spikes from calcium imaging in dopamine neurons. Atcherley, C. W., Wood, K. M., Parent, K. L., Hashemi, P. & Heien, M. L. The coaction of tonic and phasic dopamine dynamics. Zhang, L., Doyon, W. M., Clark, J. J., Phillips, P. E. & Dani, J. Controls of tonic and phasic dopamine transmission in the dorsal and ventral striatum. Melikian, H. E. Neurotransmitter transporter trafficking: endocytosis, recycling, and regulation. & Gnegy, M. E. Dopamine transporter trafficking: rapid response on demand. Miranda, M., Wu, C. C., Sorkina, T., Korstjens, D. R. & Sorkin, A. Enhanced ubiquitylation and accelerated degradation of the dopamine transporter mediated by protein kinase c. J. Biol. & Kuhar, M. J. Serotonin transporter production and degradation rates: studies with RTI-76. Calipari, E. S. et al. Dopaminergic dynamics underlying sex-specific cocaine reward. Zhang, D., Yang, S., Yang, C., Jin, G. & Zhen, X. Estrogen regulates responses of dopamine neurons in the ventral tegmental area to cocaine. Comparative distribution of estrogen receptor-α and-β mRNA in the rat central nervous system. Creutz, L. M. & Kritzer, M. F. Estrogen receptor-β immunoreactivity in the midbrain of adult rats: regional, subregional, and cellular localization in the a10, a9, and a8 dopamine cell groups. A. et al. Hormonal gain control of a medial preoptic area social reward circuit. Cocaine disinhibits dopamine neurons by potentiation of gaba transmission in the ventral tegmental area. -M., Gainetdinov, R. R. & Caron, M. G. Molecular biology, pharmacology and functional role of the plasma membrane dopamine transporter. Redish, A. D. Addiction as a computational process gone awry. State representation in mental illness. Psychiatric symptoms across the menstrual cycle in adult women: a comprehensive review. Sex and gender differences in schizophrenic psychoses-a critical review. Burt, V. K., Altshuler, L. L. & Rasgon, N. Depressive symptoms in the perimenopause: prevalence, assessment, and guidelines for treatment. Kastner, P. et al. Two distinct estrogen-regulated promoters generate transcripts encoding the two functionally different human progesterone receptor forms a and b. EMBO J. Menegas, W., Akiti, K., Amo, R., Uchida, N. & Watabe-Uchida, M. Dopamine neurons projecting to the posterior striatum reinforce avoidance of threatening stimuli. Specialized coding of sensory, motor and cognitive variables in vta dopamine neurons. Takahashi, Y. K. et al. Dopamine neurons respond to errors in the prediction of sensory features of expected rewards. Rethinking dopamine as generalized prediction error. Explaining dopamine through prediction errors and beyond. Lee, R. S., Sagiv, Y., Engelhard, B., Witten, I. & Daw, N. D. A feature-specific prediction error model explains dopaminergic heterogeneity. Dabney, W. et al. A distributional code for value in dopamine-based reinforcement learning. Jeong, H. et al. Mesolimbic dopamine release conveys causal associations. Prospective contingency explains behavior and dopamine signals during associative learning. Sharpe, M. J. et al. Dopamine transients are sufficient and necessary for acquisition of model-based associations. Sharpe, M. J. et al. Dopamine transients do not act as model-free prediction errors during associative learning. Jang, H. J., McMahon Ward, R., Golden, C. E. & Constantinople, C. M. Acetylcholine demixes heterogeneous dopamine signals for learning and moving. Kutlu, M. G. et al. Dopamine release in the nucleus accumbens core signals perceived saliency. & Son, C. -G. Linking alterations in estrogen receptor expression to memory deficits and depressive behavior in an ovariectomy mouse model. Haisenleder, D. J., Schoenfelder, A. H., Marcinko, E. S., Geddis, L. M. & Marshall, J. C. Estimation of estradiol in mouse serum samples: evaluation of commercial estradiol immunoassays. Creamer, M. S., Chen, K. S., Leifer, A. M. & Pillow, J. W. Correcting motion induced fluorescence artifacts in two-channel neural imaging. Santiago, A. N., Lim, K. Y., Opendak, M., Sullivan, R. M. & Aoki, C. Early life trauma increases threat response of peri-weaning rats, reduction of axo-somatic synapses formed by parvalbumin cells and perineuronal net in the basolateral nucleus of amygdala. -W., Wable, G. S., Chowdhury, T. G. & Aoki, C. Enlargement of axo-somatic contacts formed by gad-immunoreactive axon terminals onto layer V pyramidal neurons in the medial prefrontal cortex of adolescent female mice is associated with suppression of food restriction-evoked hyperactivity and resilience to activity-based anorexia. De Solis, C. A., Ho, A., Holehonnur, R. & Ploski, J. E. The development of a viral mediated CRISPR/Cas9 system with doxycycline dependent grna expression for inducible in vitro and in vivo genome editing. Musatov, S., Chen, W., Pfaff, D. W., Kaplitt, M. G. & Ogawa, S. Rnai-mediated silencing of estrogen receptor α in the ventromedial nucleus of hypothalamus abolishes female sexual behaviors. Lasek, A., Janak, P., He, L., Whistler, J. & Heberlein, U. Downregulation of mu opioid receptor by RNA interference in the ventral tegmental area reduces ethanol consumption in mice. Satta, R., Certa, B., He, D. & Lasek, A. W. Estrogen receptor β in the nucleus accumbens regulates the rewarding properties of cocaine in female mice. El-Neweshy, M. S. Experimental doxycycline overdose in rats causes cardiomyopathy. Mueller, F. et al. Fish-quant: automatic counting of transcripts in 3D fish images. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. Golden, C. E. M. et al. Estrogen modulates reward prediction errors and reinforcement learning. We thank research technicians in the lab of C.M.C. We thank C. Farb for assistance with histology and electron microscopy and NYU Langone's Proteomics Laboratory (SCR_017926) for the mass spectrometry. B. Bando and G. Canino for their assistance with histology processing, imaging and quantifying. This work was supported by a K99/R00 Pathway to Independence Award (R00MH11-1926), a Klingenstein-Simons Fellowship in Neuroscience and an NIH Director's New Innovator Award (DP2MH126376) to C.M.C. was supported by a grant from the Simons Foundation (855332) and NIH grants F32MH125448 and 5T32MH019524. The mass spectrometric experiments were supported in part by NYU Langone Health, the Laura and Isaac Perlmutter Cancer Center support grant P30CA016087 from the National Cancer Institute and the NIH Shared Instrumentation Grant 1S10OD010582-01A1 for the purchase of an Orbitrap Fusion Lumos Tribrid mass spectrometer. Carla E. M. Golden, Audrey C. Martin, Daljit Kaur, Andrew Mah, Diana H. Levy, Chiye Aoki & Christine M. Constantinople Neuroscience Institute, New York University Grossman School of Medicine, New York, NY, USA Department of Pharmacology and Toxicology, Virginia Commonwealth University, Richmond, VA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar collected and analyzed the data. A.M. developed the behavioral model and contributed to data analysis. created and shared the U6/tetO-shEsr1 virus. created and shared the lenti-shEsr1 virus. The authors declare no competing interests. Nature Neuroscience thanks Melissa Sharpe and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Example images of vaginal cytology by estrous stage. b-c. Expected (b) and observed (c) median 17β-estradiol, measured with an ELISA from serum, N = 18 rats (4 in proestrus > 5 hours before lights out; 4 in proestrus within 3 hours before lights out; 5 in estrus; 5 in diestrus), Kruskal-Wallis test p = 6.15 × 10−3 for group effect, two-sided Wilcoxon rank sum test for late proestrus vs. diestrus p = 0.016 and d = 3.74, early proestrus vs. late proestrus p = 0.029, estrus vs. early proestrus p = 0.032, estrus vs. late proestrus p = 0.016, diestrus vs. estrus p = 1, diestrus vs. early proestrus p = 0.063. d. Serum 17β-estradiol is significantly higher in proestrus than diestrus during training sessions, N = 17 rats, two-sided Wilcoxon signed rank test p = 9.77 × 10−4, d = 1.79. e. Median sensitivity of detrended trial initiation times to blocks for each estrous stage, N = 120 female rats, Kruskal-Wallis test p = 5.00 × 10−5 for group effect, two-sided Wilcoxon signed-rank tests for post-hoc analyses: p = 0.002 and d = 0.17 for proestrus vs. metestrus, p = 2.67 × 10−8 and d = 0.41 for proestrus vs. diestrus, p = 0.09 and d = 0.17 for proestrus vs. estrus, p = 9.06 × 10−11 and d = 0.32 for estrus vs. metestrus, p = 2.67 × 10−15 and d = 0.53 for estrus vs. diestrus, and p = 9.34 × 10−4 and d = 0.20 for metestrus vs diestrus). f. The differences in regression coefficients of one trial back for proestrus and diestrus are significantly correlated with the differences in block sensitivity for proestrus and diestrus, two-sided Pearson correlation p = 4.17 × 10−6. Mean wait time on catch trials by reward in each block (blue = low, red = high) averaged across rats. Error bars are mean ± SEM. Median adaptation of wait times to reward blocks (low - high block) does not change across the cycle, N = 118 rats, Kruskal-Wallis test p = 0.69 for group effect and two-sided Wilcoxon signed-rank test p = 0.771 and d = 0.06 for proestrus vs. diestrus. Error bars are median ± SEM. Rats used in serum 17β-estradiol experiments have (a) median regression coefficients of detrended trial initiation times plotted against rewards on previous trials during mixed blocks that decline as an exponential function of time, (b) enhanced block sensitivity in proestrus, and (c) greater regression coefficients for the previous reward in proestrus because the distribution is significantly different from zero using a two-sided Wilcoxon signed-rank test p = 4.88 × 10−3. Rats used in photometry experiments have (d) median regression coefficients of detrended trial initiation times plotted against rewards on previous trials during mixed blocks that decline as an exponential function of time, (e) enhanced block sensitivity in proestrus, and (f) greater regression coefficients for the previous reward in proestrus. For each rat, histological verification of fiber implant targeting and mCherry (red) and ChR2 (green) expression (left) and task event-aligned dopamine responses, split by reward offer cue volume (right). All error bars are mean ± SEM. Normalized AUC of mean dopamine response across the cycle to largest reward offer cue in mixed blocks (a), Kruskal-Wallis test p = 0.048 for group effect and two-sided Wilcoxon signed-rank test p = 3.91 × 10−3 for proestrus vs. diestrus, (the rest of the pairwise comparisons were not significant), and high blocks (b), Kruskal-Wallis test p = 0.15 for group effect, and largest model-predicted RPE bin (c), Kruskal-Wallis test p = 0.015 for group effect and two-sided Wilcoxon signed-rank test p = 0.039 for metestrus vs. proestrus, p = 0.027 for diestrus vs. proestrus, and p = 0.023 for proestrus vs. estrus (the rest of the pairwise comparisons were not significant). While motion-corrected GRABDA encodes reward volume, mCherry, used to control for motion artifacts, does not. All error bars are mean ± SEM. c. Example histology of blue circle in b. d. Task event-aligned dopamine signaling, separated by reward volume. Response to offer cue, separated by reward volume (e) and block (f) and stage group for example rat, G027. Gray boxes represent the window used to calculate change in area under curve, 0 to 0.5 s (ΔAUC) in (g) and (h). Data is baseline-corrected using the 0.05 to 0 s before offer cue. g. No significant difference in median population response to each reward volume, separated by reward block, min-max normalized, and baseline-corrected using the 50 ms before offer cue. h. Histogram of stage effect (proestrus-diestrus) on ΔAUC during offer cue for all NAcc and CPu rats (*p = 3.9 × 10−3 for NAcc using a two-sided Wilcoxon signed-rank test, p = 0.875 for CPu). All error bars are mean ± SEM. Mean dopamine response aligned to beginning of delay period for rewarded mixed block trials by reward delay length. Mean dopamine response aligned to the end of the delay period for rewarded mixed block trials by reward delay length. c. Dopamine response aligned to the end of the delay period when reward was cued for mixed block trials by reward delay length and stage. d. Median AUC from c is greater in proestrus than diestrus, two-sided Wilcoxon signed-rank test (from low to high delay) p = 7.80 × 10−3, 0.02, 0.01, p = 7.80 × 10−3, p = 3.90 × 10−3. Example images from immunohistochemical quantification of ChR2-eYFP (top) and TH (bottom) co-labeling at 63x with mask (right) identifying fluorescent pixels above a threshold, generated in ImageJ, overlaid. Performed on 90 regions of interest. Image created in ImageJ that depicts pixels that are fluorescent in both ChR2-eYFP and TH images (overlap) with mask. c. Metric of co-labeling across all images: area of mask segmentation for ChR2-eYFP image divided by area of mask segmentation for TH image. The strongest gene ontology terms that decreased (a-b) and increased (c) proteins were significantly enriched for in proestrus (a) and estrus (b-c) compared to diestrus. Increased proteins in proestrus compared to diestrus were not significantly enriched for any known biological functions. d. 92 differentially expressed proteins related to dopamine sorted by fold change, with proteins involved in dopamine reuptake identified. Median initiation time across all trials is significantly higher in proestrus, N = 118 rats, two-sided Wilcoxon signed rank test p = 1.60 × 10−14 and d = 0.49. b-c. Median trials performed (b), p = 8.30 × 10−16 and d = 0.48, and total water consumed (c), p = 1.34 × 10−15 and d = 0.51, controlling for the total amount of time the rat was allotted in the behavioral rig, N = 118 rats. d. Median serum osmolality was lower for stages around ovulation (proestrus and estrus, N = 5 males, N = 7 diestrus, N = 4 early proestrus, N = 4 late proestrus, and N = 5 estrus), Kruskal-Wallis test p = 0.01 for the group effect and d = 1.61 for proestrus vs diestrus. All error bars are median ± SEM. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Golden, C.E.M., Martin, A.C., Kaur, D. et al. Estrogen modulates reward prediction errors and reinforcement learning. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02094-y'>APOE4 to APOE2 allelic switching in mice improves Alzheimer's disease-related metabolic signatures, neuropathology and cognition</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 10:33:29
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Compared to individuals carrying two copies of the ε4 allele of apolipoprotein E (APOE), ε2 homozygotes have an approximate 99% reduction in late-onset Alzheimer's disease (AD) risk. Here we develop a knock-in model that allows for an inducible ‘switch' between risk and protective alleles (APOE4s2). Gene expression and proteomic analyses confirm that APOE4s2 mice synthesize E4 at baseline and E2 after tamoxifen administration. A whole-body allelic switch results in a metabolic profile resembling E2/E2 humans and drives AD-relevant alterations in the lipidome and single-cell transcriptome, particularly in astrocytes. Finally, when crossed to the 5xFAD background, astrocyte-specific E4 to E2 switching improves cognition, decreases amyloid pathology, lowers gliosis and reduces plaque-associated apolipoprotein E. Together, these data show that a short-term transition from APOE4 to APOE2 can broadly affect the cerebral transcriptome and lipidome, and that astrocyte-specific APOE replacement may be a viable strategy for future gene editing approaches to simultaneously reduce multiple AD-associated pathologies. The strongest genetic risk factor for late-onset Alzheimer's disease (AD) is apolipoprotein E (APOE). In humans, APOE has three common polymorphic alleles that encode three protein isoforms of apolipoprotein E (ApoE): E2, E3 and E4. Compared with E3, E4 confers up to a 13-fold increase in risk for AD in homozygous carriers1. Conversely, E2 is strongly neuroprotective, reducing AD risk by up to 40%, slowing cognitive decline and increasing longevity2,3,4. These striking differences in AD risk have been attributed to several possible pathogenic mechanisms, including E4-driven exacerbation of amyloid and tau pathology, synaptic dysfunction, network connectivity, cerebrovascular impairments, a heightened neuroinflammatory state and alterations in glial glucose and lipid metabolism5. Given this strong risk profile and multitude of biological effects, targeting APOE4 and ApoE4 presents as a promising therapeutic approach to simultaneously recalibrate multiple AD-associated pathways6. Studies that have successfully targeted APOE using gene editing in induced pluripotent stem cells (iPSCs) provide strong proof of concept that in vitro editing of ε4 to ε3 or ε2 simultaneously ‘corrects' several phenotypes, including glial transcriptomic profiles, lipid metabolism and other functional measures7,8. Moving in vivo, other studies have explored vector-mediated delivery to supplement the E4 brain with neuroprotective E2 protein, showing that exogenous E2 markedly decreases amyloid burden in mice9,10,11,12. Efficacy of a similar viral-based methodology was validated in nonhuman primates13 and is currently undergoing clinical trials14. Conversely, decreasing E4 protein levels is another promising strategy several groups have demonstrated via genetic, antibody-based or antisense oligonucleotide-based knockdown methods15,16,17, or indirect modulation of ApoE levels by targeting its primary cell surface receptors18. Together, these studies suggest that an approach where E4 is eliminated and replaced with E2 could be a highly effective therapeutic strategy. Importantly, recent advances in CRISPR–Cas9 and related base-editing technologies make this type of precision medicine a conceivable approach for E4 carriers in the coming years19. To generate a preclinical model in which we could investigate the feasibility and therapeutic potential of a full APOE4 to APOE2 transition in vivo, we developed a knock-in model coined the APOE ‘switch' mouse (APOE4s2). Using a ubiquitously expressed inducible Cre, we drove a highly efficient transition from E4 to E2 in both the periphery and brain, as confirmed at both the mRNA and protein level. The metabolic profile of post-switch APOE4s2 mice resembled that of humanized APOE2 ‘targeted replacement' (TR) mice and E2/E2 humans, with elevated ApoE concentrations and increased plasma lipids. Importantly, replacement of E4 with E2 in adult mice resulted in transcriptomic changes across multiple cerebral cell types, particularly within astrocytes and in pathways related to the immune response and metabolism. Interestingly, an astrocyte-only APOE4 to APOE2 switch produced very similar AD-relevant transcriptomic changes as a full-body transition. Finally, when crossed with 5xFAD mice, astrocyte-specific APOE allele switching from E4 to E2 was sufficient to decrease total amyloid levels, reduce gliosis and plaque-associated ApoE, and improve cognition. Collectively, these data suggest that even a short-term, postdevelopmental transition from E4 to E2 broadly affects the cerebral transcriptome and lipidome, and can simultaneously decrease multiple AD pathologies. These findings highlight this inducible system as a promising preclinical model in which to assess the timing, cell-specific contributions and overall therapeutic potential of APOE gene editing. To investigate the in vivo effects of an APOE4 to APOE2 allelic switch, we generated APOE4s2loxP/loxP (APOE4s2) mice that express a floxed coding region (exon 4) of human APOE4 followed by exon 4 of human APOE2 (Fig. APOE4s2 mice were crossed to a tamoxifen (TAM)-activable strain ROSA26-CreERT1 Cre to generate floxed ROSA26-CreERT1(APOE4s2G), allowing for temporal induction of global (‘G') APOE allele switching. To confirm that TAM successfully induces a whole-body transition from expression of E4 to E2, 2-month-old APOE4s2G mice were compared to TAM-treated Cre− APOE4s2 (APOE4s2) littermates and human APOE2-targeted and APOE4 TR mice20,21. As brain and liver are the two major sources of ApoE, efficiency of the E4 to E2 switch was assessed using quantitative PCR (qPCR) to determine APOE mRNA expression in these two tissues. Allelic discrimination plots demonstrated an efficient transition from APOE4 to APOE2 mRNA in both the liver and the brain (Fig. Targeted mass spectrometry (MS)-based proteomic analysis confirmed the E4 to E2 transition at the protein level, where 84–93% and 92–99% of the ApoE peptides detected in the brain and plasma respectively from APOE4s2G mice were unique to E2 (Fig. 1e,f, Extended Data Fig. Together, these data demonstrate that APOE4s2G mice synthesize human ApoE4 at baseline, and that TAM induces a highly efficient transition to APOE2 and ApoE2 at the gene and protein level. a, Schematic depicting the genetic construct for the APOE4 to APOE2 ‘switch' mouse (APOE4s2) and the experimental design for the experiments performed in APOE4s2 mice crossed to a globally (‘G') expressed ROSA26-CreERT1 recombinase (APOE4s2G). Homozygous floxed Cre− littermates (APOE4s2) were used as controls. b,c, Allelic discrimination plots depicting a shift from APOE4 to APOE2 mRNA expression (b) in APOE4s2G mice in both the brain (b) and liver (c) compared to Cre− APOE4s2 littermates and ApoE2 and ApoE4 TR controls injected with TAM. d, Methodology used to semiquantitatively detect the ratio of peptides unique to E2 or E4 in the brain and plasma using liquid chromatography (LC)–MS/MS-based proteomic analysis. e,f, Ratio of unique E2 versus E4 peptides detected using LC–MS/MS-based proteomic analysis showing an efficient transition to E2 in both the brain (f) and plasma (g) of APOE4s2G (n = 4), APOE4s2 (n = 4), E2-TR (n = 2) and E4-TR (n = 2) mice. Data are represented as mean values ± s.e.m. RFU, relative fluorescence unit. Given ApoE's central role in lipid metabolism, we first asked if the global E4 to E2 switch could affect peripheral and cerebral metabolic pathways. E2-TR mice display a plasma lipid profile similar to the type III hyperlipoproteinemia found in ~5–10% of E2/E2 humans, particularly when challenged with a high-fat diet (HFD)20. To determine if APOE4s2G mice phenocopy this hyperlipidemic profile after the E4 to E2 transition, mice were fed a Western diet for 4 months. Blood was collected 3 months into the diet before TAM treatment, and 1 month after the allelic switch. Before TAM, total plasma triglyceride (TG) levels in APOE4s2 and APOE4s2G mice were indistinguishable from the E4-TR controls (Fig. However, 1 month after TAM, APOE4s2G mice fed a Western diet displayed a substantial increase in very-low-density lipoprotein (VLDL) TGs and plasma ApoE levels, which rose to E2-TR levels (Fig. APOE4s2G mice fed a normal chow diet also showed elevations in TGs, cholesterol and phospholipids relative to the APOE4s2 controls, although the increases were more modest (Extended Data Fig. Brain ApoE levels were also elevated in APOE4s2G mice 1 month after switching (Fig. Colocalization of ApoE with cell-specific markers for astrocytes, microglia and neurons showed that like the E2-TR, E4-TR and APOE4s2 controls, ApoE primarily colocalized with astrocytes (glial fibrillary acidic protein (GFAP)+) in APOE4s2G mice, with measurable, but much lower, colocalization with the microglial marker IBA1 (Fig. 2g,h and Extended Data Fig. While neurons synthesize ApoE under certain conditions, we observed negligible colocalization of ApoE with NeuN (Extended Data Fig. d,e, ELISA quantification of ApoE in plasma before TAM (d) and after the TAM-induced switch (e) (E2-TR, n = 5; E4-TR, n = 5; APOE4s2, n = 6; APOE4s2G, n = 7; one-way ANOVA with Tukey's multiple comparisons test). g,h, Representative images showing GFAP (green) (g) or IBA1 (green) (h) and ApoE (red) in APOE4s2 controls (top) or APOE4s2G brains (bottom). i, Volcano plot comparing brain lipids altered in APOE4s2G mice versus Cre− APOE4s2 controls (APOE4s2, n = 8; APOE4s2, n = 8; all female). The cutoff value for significance is log10(P) > 1.3. j, Heatmap depicting the top ten most significantly changed lipids between APOE4s2G and APOE4s2 controls. k,l, WGCNA analysis identified lipid modules associated with the switch from E4 to E2. k, Module-trait correlations between modules and APOE expression indicated by the numbers in the table, with P values shown in parentheses. l, Network plot of lipids identified in the blue module. a–f, For detailed statistical information, see Supplementary Table 3. ***P < 0.001, ****P < 0.0001. i–l, Twenty-two classes of lipids were detected: CER; cardiolipin (CI); coenzyme (CO); diglyceride (DG); dimethyl phosphotidylehtanolamine (DMEPE); fatty acid (FA); ganglioside (GM); lysodimethylphosphatidylethanolamine (LDMEPE); LPC; lysophosphatidylethanolamine (LPE); lysophosphatidylglycerol (LPG); lysophosphatidylinositol (LPI); lysophosphatidylserine (LPS); (O-acyl)-ω-hydroxy FA (OAHFA); phosphatidic acid (PA); PC; PE; phosphatidylglycerol (PG); phosphatidylinositol (PI); phosphatidylserine (PS); sphingomyelin (SM); and TG. We next examined possible changes to the cerebral lipidome using an untargeted LC–MS/MS approach. A comparison of APOE4s2G mice versus Cre− APOE4s2 controls identified 14 lipid species that were significantly different in abundance between the two groups (Fig. In agreement with previous postmortem AD studies22,23, many of these lipids were glycerophospholipids, including many phosphatidylcholine (PC) and ceramide (CER) species. Interestingly, there were nuanced changes within these lipid classes; in both PC and CER classes, some lipid species increased in abundance while others decreased. For reference, a comparison of E2-TR and E4-TR control mice identified 57 lipids of differing abundance, again most of which were glycerophospholipids (Extended Data Fig. We then performed a weighted gene coexpression network analysis (WGCNA) on the cerebral lipidome, where we identified five coexpression modules, including one network that was significantly correlated with the E4 to E2 switch (blue module) and consisted primarily of PC and phosphatidylethanolamine (PE) species (Fig. Together, these data demonstrate that even a short-term transition from E4 to E2 in the adult mouse is sufficient to drive alterations in ApoE concentrations and lipid metabolism, most notably increases in multiple PC species within the brain. To unbiasedly examine how switching from APOE4 to APOE2 globally influences cell-type-specific changes to the brain transcriptome, we performed single-cell RNA sequencing (scRNA-seq) on whole-brain tissue from 6-month-old APOE4s2G mice, Cre− APOE4s2 controls and age-matched and sex-matched E2-TR and E4-TR mice. Dimensionality reduction identified 41 clusters, which were assigned to one of 12 unique cell types, with cell and cluster proportions similarly distributed across all groups (Fig. Analysis of differentially expressed genes (DEGs) revealed astrocytes, oligodendrocytes, microglia and endothelial cells as the cell types most affected by a 1-month-long APOE4 to APOE2 switch (Fig. Across these cell types, there was notable overlap of DEGs from APOE4s2G mice versus APOE4s2 controls with DEGs from E2-TR versus E4-TR mice (Fig. 3c), suggesting that many genes that distinguish ‘lifetime' expression of E4 are acutely malleable. Gene Ontology (GO) analysis of switch-related DEGs highlighted a number of pathways related to metabolism, including substrate transport, redox management and cellular respiration (Fig. Furthermore, many of the genes altered by the E4 to E2 switch fell under one of six primary AD ‘umbrella' terms—pathways implicated across multiple postmortem AD studies24—in particular those related to cerebral metabolism, inflammation and cytoskeleton/vascular integrity (Fig. Finally, the DEGs associated with the E4 to E2 switch also strongly reflected several APOE-associated and AD-associated gene lists, particularly the late-onset AD (LOAD) genes identified in and merged from publicly available human transcriptomic datasets (Fig. Collectively, these results demonstrate that, even in the absence of AD pathology, a postdevelopmental transition from expression of E4 to E2 is sufficient to drive transcriptomic changes in pathways previously implicated in AD. a, Uniform manifold approximation and projection (UMAP) of all 85,701 cells analyzed, classified based on canonical gene expression markers into 12 cell types (APOE4s2G, n = 3; APOE4s2, n = 3; E2-TR, n = 3, E4-TR, n = 2; all females). b, Number of DEGs between APOE4s2G and Cre− APOE4s2 controls according to cell type (Padj = <0.001). c, Venn diagrams revealing commonality of DEGs in the APOE4s2G versus APOE4s2 and E2-TR versus E4-TR datasets. d–g, Upregulated and downregulated GO terms associated with DEGs from astrocytes (d), oligodendrocytes (e), microglia (f) and endothelial cells (g). The cutoff value for significance was set at a false discovery rate (FDR)-adjusted P value of 0.05. h, Pie charts visualizing the categorical distribution of APOE switch-related GO terms into AD-related pathways. Upregulated (left) AD pathways account for 24%, while downregulated (right) AD pathways account for 21% of total downregulated GO terms. i, Overlap of the APOE switch transcriptomics dataset with five relevant gene lists from AD-related transcriptomic studies. DAA, disease-associated astrocyte; OPC, oligodendrocyte precursor cell; Transcription corep. Considering the potential translational concerns associated with germline, whole-body APOE editing, we reasoned that a central nervous system (CNS)-selective APOE4 to APOE2 switch could afford the neuroprotection of E2 while minimizing off-target effects. Because astrocytes synthesize most of CNS ApoE and were the cell type most affected by the global E4 to E2 transition, we generated APOE4s2loxP/loxP/Aldh1l1-CreERT2 APOE switch mice (APOE4s2A) that would express APOE4 in all CNS cell types except for astrocytes, which would selectively transition to APOE2 upon TAM injection29,30 (Fig. Cell-type specificity was confirmed via colocalization of GFAP with tdTomato expression in APOE4s2A mice crossed to an Ai9 Cre reporter strain (Fig. Leveraging the Ai9 cross, we then used cell sorting and qPCR to validate the high efficiency and cell specificity of the astrocyte-specific E4 to E2 switch (Fig. A similar, albeit more modest, shift toward APOE2 expression was observed in whole-brain tissue, suggesting that most of total brain ApoE is derived from astrocytes, which were now synthesizing E2 (Fig. While the Aldh1l1-CreERT2 showed high specificity for astrocytes in the brain; plasma lipid profiling and peripheral ApoE measures indicated that there is also notable E2 expression in the periphery (Extended Data Fig. a, Graphic depicting astrocyte-specific APOE4s2A mice with Aldh1l1-CreERT2. b, Representative images showing colocalization of tdTomato Ai9 Cre reporter (red) and GFAP (green). c,d, Allelic discrimination plots measuring APOE4 versus APOE2 mRNA expression in astrocytes (c) sorted from other cell types in the brain, or from whole-brain homogenates (d), of APOE4s2A mice versus APOE4s2 controls (E2-TR, n = 2; E4-TR, n = 2; APOE4s2A, n = 6; APOE4s2, n = 6). e, Number of DEGs between APOE4s2A and Cre− APOE4s2 controls according to cell type (Padj ≤ 0.001) (APOE4s2A, n = 3; APOE4s2, n = 3). f, Overlap of the APOE switch transcriptomic dataset with five other gene lists from AD-related transcriptomic studies. g, Venn diagrams revealing the commonality of DEGs in the APOE4s2A versus APOE4s2 and APOE4s2G versus APOE4s2 datasets. h–k, GO analysis for downregulated and upregulated terms associated with DEGs from microglia (h), oligodendrocytes (i), astrocytes (j) and endothelial cells (k). The cutoff value for significance was set at an FDR-adjusted P < 0.05 and fold change ≥ 1.3. We next asked if an astrocyte-specific switch could induce similar transcriptomic changes as the full-body APOE allele switch. Analysis of DEGs showed that the astrocyte-specific APOE4 to APOE2 transition primarily affected similar cell types as observed in the global model (Fig. Additionally, as seen in the whole-body model, the DEGs associated with an astrocyte-specific E4 to E2 switch also strongly mirrored the LOAD gene lists (Fig. Interestingly, several cell types that continued expressing E4 in APOE4s2A mice (that is, microglia, oligodendrocytes) still showed a high number of DEGs, suggesting non-cell-autonomous effects of astrocyte-derived E2. There was also substantial overlap (39–60% depending on the cell type) between the specific DEGs associated with a global versus astrocyte-specific E4 to E2 switch (Fig. Across all cell types, pathway analyses of astrocyte switch DEGs revealed similar terms to those seen in whole-body APOE4s2G mice, particularly in pathways related to neurotransmission, redox status and metabolite transport (Fig. Together, these data suggest that astrocyte-exclusive expression of E2 is sufficient to drive AD-relevant alterations in the glial transcriptome, even when all other cell types continue to express E4 and even in a model that lacks overt AD pathology. We then tested if an astrocyte-specific E4 to E2 switch could directly affect AD pathology. To this end, we crossed APOE4s2A mice to the 5xFAD model of AD (4s2A/FAD) and administered TAM at 6 months of age. At 8 months of age, associative fear conditioning and the Morris water maze were used to assess contextual, cued and spatial memory; brains were subsequently collected for histopathological analyses (Fig. While the Morris water maze did not show striking differences in spatial learning or memory, 4s2A/FAD mice showed improvements in associative learning and more robust contextual and cued memory, an effect that was strongest in females (Fig. b–d, Percentage time freezing measured during fear conditioning (learning) (b), and the contextual (memory) (c) and cued memory test (d) tests, in 4s2A/FAD mice compared to Cre− controls 2 months after TAM (4s2A/FAD, n = 16, 4s2/FAD, n = 16; **P < 0.01,***P < 0.001, ****P < 0.0001, using a repeated measures ANOVA). e, Representative images of Amylo-Glo (blue) and nucleus (red) staining in 4s2A/FAD and 4s2/FAD mice. f, Quantification of the total percentage amyloid+ area in 4s2A/FAD mice showed significant decreases in total amyloid measured according to the percentage of the amyloid+ area. The dashed line represents the mean amyloid percentage area in 4s2/FAD mice at the 6-month pre-switch time point (y = 0.118). **P < 0.01, using a two-tailed unpaired t-test. g,h, Heatmap (g) and quantification (h) of the percentage of amyloid+ area according to region in 4s2A/FAD versus 4s2/FAD mice (4s2A/FAD, n = 10; 4s2/FAD, n = 10; two-way ANOVA with multiple comparisons). i–l, ELISA quantification of Aβ40 (i,j) or Aβ42 (k,l) in soluble (PBS) or insoluble (5 M GuHCl) whole-brain fractions from 4s2A/FAD mice compared to 4s2/FAD controls (4s2A/FAD, n = 14; 4s2/FAD, n = 14; *P < 0.05, **P < 0.01, ****P < 0.0001, using a two-tailed unpaired t-test). m, Representative images of mOC31 anti-CAA (green) and CD31 (red) staining in the cortex of 8-month-old 4s2A/FAD and 4s2/FAD mice. n, Quantification of CAA+ vessels normalized to total CD31+ area in 8-month-old 4s2A/FAD mice and 4s2/FAD controls after the switch compared with 6-month-old mice before the switch (4s2A/FAD 8 months, n = 8; 4s2/FAD 8 months, n = 8; 4s2/FAD 6 months, n = 8; *P < 0.05, **P < 0.01, using a two-way ANOVA with multiple comparisons). a–n, For detailed statistical information, see Supplementary Table 3. Individual data points have been plotted. Sex is denoted by the shape of the symbol: females (circle), males (triangle). Previous work established that human APOE affects amyloid plaque deposition in the 5xFAD brain in a genotype-dependent fashion (E4 > E2)31. To test if a late-stage, astrocyte-only replacement of APOE4 with APOE2 could decrease plaque load, we performed immunohistochemical analyses of amyloid accumulation. Strikingly, after a 2-month astrocyte-specific allelic switch, 4s2A/FAD animals showed significant reductions in total amyloid levels (Fig. 5e,f and Extended Data Fig. When compared to ‘baseline' amyloid measures at 6 months of age, the reduced burden in 4s2A/FAD mice was due to slowed or delayed amyloid accumulation (Fig. Anatomical analyses revealed decreases in many regions, with significant reductions in the hippocampus, olfactory area and thalamus primarily driving the decrease in plaque load (Fig. Enzyme-linked immunosorbent assay (ELISA) quantification of Aβ42 and Aβ40 in whole-brain soluble and insoluble fractions revealed reductions in both soluble and insoluble Aβ42 and Aβ40 observed in 4s2A/FAD brains (Fig. Overall, there were very few sex differences in AD-related pathology. While higher amyloid plaque loads were observed in female compared to male 4s2/FAD Cre− controls, we did not detect any other amyloid-dependent sex differences, nor was this difference reflected in the soluble and insoluble Aβ42 and Aβ40 measures (Extended Data Fig. We did note subtle sex differences in the expression of the postsynaptic density marker PSD95 in the cortex, although this was limited to plaque-distal regions and was not seen in the hippocampus (Extended Data Fig. Because APOE4 is also a risk factor for cerebral amyloid angiopathy (CAA)31, we measured CAA before and after the inducible, astrocyte-specific transition to APOE2. Interestingly, we observed no differences in CAA between 4s2A/FAD and 4s2/FAD controls at 8 months of age (Fig. Finally, we measured the expression of the tight junction protein ZO1 and found no change after the switch to astrocyte-derived E2 (Extended Data Fig. Together, these results suggest that a relatively late-stage switch to astrocytic expression of E2 is sufficient to substantially lower parenchymal plaque burden, but not CAA, and improve some, but not all, of the cognitive deficits associated with E4. In addition to the reduction in parenchymal amyloid plaque load, we also observed significantly lower levels of GFAP in 4s2A/FAD mice compared to controls (Fig. Interestingly, although only astrocytes transitioned to the expression of E2 in the 4s2A/FAD mice, we noted a dramatic reduction in microglial IBA1 positivity (Fig. Like the regional reductions in plaque load, GFAP and IBA1 levels were most substantially decreased in the hippocampus, thalamus and cortical areas (Fig. 6d,e and Extended Data Fig. a, Representative images of Amylo-Glo (blue), IBA1 (green), GFAP (orange) and nucleus (red) staining in 4s2A/FAD mice and 4s2/FAD controls. b,c, Quantification of the percentage GFAP+ (b) or IBA1+ (c) area in 4s2A/FAD mice versus 4s2FAD controls (4s2A/FAD, n = 16; 4s2/FAD, n = 16; ***P < 0.001, ****P < 0.000, using a two-tailed unpaired t-test). d,e, Quantification of GFAP+ (d) or IBA1+ (e) percentage area according to region in 4s2A/FAD and 4s2/FAD mice (4s2A/FAD n = 10; 4s2/FAD n = 10; *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001, using a two-way ANOVA with multiple comparisons). f, Representative images of Amylo-Glo (blue), IBA1 (green), GFAP (orange) and nucleus (red) proximity and colocalization. g–i, Percentage area of colocalization of Amylo-Glo and GFAP (g), Amylo-Glo and IBA1 (h) or Amylo-Glo with GFAP and IBA1 (i) in 4s2A/FAD and 4s2/FAD mice (4s2A/FAD, n = 16, 4s2/FAD, n = 16; **P < 0.01, ***P < 0.001, using a two-tailed unpaired t-test). j–m, Histograms (j,l) and area under the curve (AUC) quantification of histograms (k,m) from plaque proximity analyses measuring the percentage area of GFA+ (j,k) and IBA1+ (l,m) cells from the plaque center in 4s2A/FAD mice compared to 4s2/FAD controls (4s2A/FAD, n = 8; 4s2/FAD, n = 8). a–m, For detailed statistical information, see Supplementary Table 3. Individual data points have been plotted. Sex is denoted by the shape of the symbol: females (circle), males (triangle). Reductions in total GFAP and IBA1 levels were especially pronounced in close proximity to amyloid plaques. 4s2A/FAD mice showed decreased colocalization of amyloid with GFAP+ and IBA1+ puncta (Fig. 8d–g); plaque proximity analyses showed more pronounced decreases in GFAP and IBA1 in 4s2A/FAD mice when close to the plaque center, with distribution of the GFAP+ and IBA1+ signal becoming more similar to controls as the distance from the center increased (Fig. Together, these findings highlight a strong decrease in plaque-associated astro-gliosis and micro-gliosis after an astrocytic E4 to E2 switch. We next investigated more nuanced changes to microglia by measuring major histocompatibility complex class II (MHC-II), a marker of activated-response microglia (ARM), an Apoe-dependent subpopulation32. 4s2A/FAD mice showed a decrease in total MHC-II+ area and a decrease in colocalization between MHC-II + IBA1+ cells and amyloid plaques (Fig. 9a–f), indicating a reduction in ARMs after the 2-month astrocyte-specific switch to E2. Notably, sex-based analysis showed significantly more cortical plaques with MHC-II+ microglia in female mice than males (Extended Data Fig. Although there was no change in homeostatic gene expression, decreased expression of several disease-associated microglia (DAM) genes, including Trem2, Clec7a and APOE, was also noted after the astrocyte-specific transition to E2 (Fig. 7e,f and Extended Data Fig. a, Representative images of Amylo-Glo (blue), IBA1 (green), MHC-II (orange) and nuclei (red) staining. b–d, Quantification of total MHC-II+ percentage area (b), colocalized MHC-II+ and IBA1+ area (c), and colocalization of MHC-II+, IBA1+ and Amylo-Glo+ area (d) in 4s2A/FAD and 4s2/FAD mice in the cortex and hippocampus (4s2A/FAD, n = 16; 4s2/FAD, n = 16; *P < 0.0, **P < 0.01, ***P < 0.001, ****P < 0.0001 using a two-way ANOVA with multiple comparisons). e,f, Fold change of Tmem119, P2ry12 and Cx3cr1 (e) or Clec7a, Trem2 and APOE (f) gene expression normalized to 18S in the whole brain from 4s2A/FAD mice compared to 4s2/FAD controls (4s2A/FAD, n = 14; 4s2/FAD, n = 14; *P < 0.0, **P < 0.01 using a two-tailed unpaired t-test). g, Representative images of Amylo-Glo (blue), ApoE (green) and GFAP (orange) in 4s2A/FAD mice and 4s2/FAD controls. h, Quantification of plaque-associated ApoE in 4s2A/FAD mice relative to 4s2/FAD controls in both the cortex and hippocampus (4s2A/FAD, n = 16, 4s2/FAD, n = 16; *P < 0.05, **P < 0.01, using a two-way ANOVA with multiple comparisons). i,j, ELISA quantification of ApoE within the soluble (PBS) (i) and insoluble (5 M GuHCl) (j) brain fractions of 4s2/FAD and 4s2A/FAD whole-brain tissue (4s2A/FAD, n = 14; 4s2/FAD, n = 14; *P < 0.05, using a two-tailed unpaired t-test). k, Correlation of cognitive measures and neuropathological analyses. The red squares indicate a positive correlation; the blue squares indicate a negative correlation. a–k, For detailed statistical information, see Supplementary Table 3. Individual data points have been plotted. Sex is denoted by the shape of the symbol: females (circle), males (triangle). While there was no difference in total ApoE protein levels between the 4s2A/FAD and 4s2/FAD mice, we observed a striking difference in plaque-associated ApoE. Control mice expressing E4 in all cell types displayed many bright halos of ApoE staining surrounding the cores of plaques (Fig. 7g,h and Extended Data Fig. However, these intense bursts of plaque-associated ApoE signal were largely absent in 4s2A/FAD mice, highlighting potential differential effects of the E2 versus E4 isoforms on plaque expansion or compaction. ELISA quantification of ApoE supported these findings, with significant reductions of ApoE within the insoluble (plaque-associated) brain fractions from 4s2A/FAD mice, but no differences in soluble ApoE (Fig. 7i,j and Extended Data Fig. Finally, we linked behavioral and pathological measures from each individual mouse to identify potential significant correlations between cognition and AD-associated pathologies. Several measures were significantly correlated but, paradoxically, positively correlated in 4s2/FAD mice yet negatively correlated in 4s2A/FAD mice (Fig. Taken together, these results suggest that astrocyte-selective expression of ApoE2 leads to downstream reductions in microglial activation and lower levels of plaque-associated ApoE, and that these various measures of gliosis differentially correlate with memory. In the current study, we describe a new APOE ‘switch' model (APOE4s2) that allows for inducible, cell-specific, in vivo replacement of the high-risk APOE4 allele with the protective APOE2 variant. We characterize how a global E4 to E2 replacement drives peripheral metabolic changes and AD-relevant alterations in the cerebral transcriptome and lipidome. We further show that an astrocyte-selective replacement of E4 with E2 significantly alters the transcriptomic profiles of multiple CNS cell types, reduces amyloid plaque load, dampens gliosis and improves cognitive outcomes in an amyloidosis model of AD. These findings underscore the dynamic and reversible nature of APOE's contribution to AD pathology and suggest that targeted APOE allele replacement, particularly in astrocytes, may offer a promising therapeutic avenue to modify multiple AD-relevant pathways simultaneously. APOE4s2G mice, in which all cell types transition from APOE4 to APOE2, reliably phenocopy the humanized E4-TR mice before the switch and E2-TR mice within 1 month after the switch. Specifically, untargeted lipidomic profiling of the brain revealed that the allelic switch altered multiple PC, CER and lyso-PC (LPC) species, a class of pro-inflammatory lipids that act as ‘find me' signals for myeloid cells. These results align with our previous work showing decreased PC levels in the aged E4 and E4 5xFAD mouse brain, albeit relative to E3 (ref. They also closely parallel the lipidomic profiling of postmortem AD brains, where several PC species show E4-associated decreases relative to E2 (ref. Interestingly, the most significantly upregulated lipid in our dataset is a CER (Cer(d40:2)) involved in glucosylceramide synthesis, a process downregulated in AD brains34. Alterations to lipid species such as CERs or PCs could have important implications for intracellular and cell–cell communication, membrane composition and apoptosis35. Across multiple studies, glial cells show APOE4-associated transcriptomic changes in both human E4+ brains and E4-TR mice5. Interestingly, these are the same primary cell types affected by a short-term E4 to E2 transition. Our results showing changes in microglia, oligodendrocyte and endothelial cell transcriptomes after the astrocyte-selective switch also suggest strong non-cell-autonomous effects of astrocyte-derived E2. These findings were mirrored by histopathological reductions in microglial reactivity in 4s2A/FAD mice and are in line with previous studies showing profound downstream effects on microglia after an astrocyte-specific knockout of APOE4 (refs. Direct comparison of the DEGs distinguishing E2-TR from E4-TR mice to the DEGs of APOE4s2G versus APOE4s2 mice showed some commonality (~20%) across multiple cell types, suggesting that E4's transcriptional signature is not permanently established during development or early life. Rather, many of these DEGs are acutely malleable, at least within the specific parameters tested in this study. Further analyses of these DEGs revealed that the pathways most affected by the E4 to E2 switch—both whole-body and astrocyte-specific—were pathways centered on cerebral metabolism, inflammation and cytoskeleton/vascular integrity, which are strongly reminiscent of the lipid metabolism, immune and matrisome findings from in vitro comparisons of isogenic APOE glia8,37. Furthermore, with the caveat that comparisons to late-disease postmortem tissue is inherently limiting, several pathways altered by a short-term E4 to E2 transition were cellular pathways that are also perturbed across 17 large-scale human AD snRNA-seq studies24. Importantly, this overlap with AD scRNA-seq datasets occurred in both the global and astrocyte-specific switch models that lack overt AD pathology. Excitingly, this may suggest that the overarching cellular pathways that define AD in the human brain could be altered by transitioning from E4 to E2. While fully ‘correcting' the E4 risk allele to the neuroprotective E2 allele could, in principle, simultaneously improve cerebral function across multiple fronts and greatly decrease AD risk, this approach is not without its own risks. An estimated 5–10% of E2/E2 individuals present with type III hyperlipoproteinemia (HLP-III), and E2 carriage has been linked to a higher risk of melanoma38, age-related macular degeneration39 and even some neurological disorders such as posttraumatic stress disorder40 and progressive supranuclear palsy41. In fact, peripheral phenotyping of APOE4s2G mice showed that the in vivo E4 to E2 switch shifted animals to an HLP-III-like plasma lipid profile. Because Sullivan et al.20 reported HLP-III-like plasma lipids in E2-TR mice, the increases in plasma TGs and ApoE observed in APOE4s2G mice ‘after the switch' were exciting confirmatory results regarding the effectiveness of our model design. On the other hand, the potential of inducing HLP-III or increasing the risk of other disorders after either a germline or universal (that is, ‘whole-body') allelic switch to E2 raises important translational concerns. To this end, we reasoned that CNS APOE targeting would hold the greatest therapeutic promise for AD. Strikingly, we show that a 2-month-long astrocyte-specific E4 to E2 switch after pathology onset is sufficient to decrease plaque load, reduce markers of glial reactivity and improve associative, but not spatial, memory in the 5xFAD model. It is possible that reductions in microglial reactivity are merely a consequence of decreased amyloid burden. However, the astrocyte-driven, APOE-dependent alterations to the microglial transcriptome found in our scRNA-seq data, coupled with glia–plaque colocalization data showing less IBA1+ and ARM microglia around plaques, suggest an alternative scenario where astrocyte-derived E2 could be having non-cell-autonomous effects on microglial activation and response to plaque (possibly via isoform-specific differences in ApoE–low-density lipoprotein receptor family interactions)42,43. While these findings may appear in contrast to studies indicating a cell-autonomous pathogenic role of ApoE in microglia, macrophages and neurons, these phenomena are not necessarily mutually exclusive15,36,44,45. Similarly, findings from a study by Mahan et al.36 suggested a role for astrocyte-derived ApoE in amyloid reduction and associated gliosis. While that study did not report decreases in IBA1 as those seen in this study, they observed a similar decrease in reactive microglia using Clec7a, and a reduction in plaque-associated ApoE. Using an inverse approach, Liu et al.46 showed that astrocyte-specific expression of E4 suppressed amyloid clearance, with particularly strong effects during the amyloid seeding phase. In this study, we observed that expression of astrocytic E2 slowed amyloid deposition during the 2-month period after the allelic switch in 4s2A/FAD mice. To fully tease out the exact mechanisms at play, more nuanced experiments and timing paradigms will be required to examine how cell-specific ApoE contributes to amyloid deposition and the glial response to plaque. Despite reductions in parenchymal plaque load, we observed similar levels of CAA in the 4s2A/FAD and 4s2/FAD mice. These findings may reflect human AD studies in which both E4 and E2 carriage were associated with higher CAA compared to E3/E3 individuals47,48, and could have translational implications for amyloid-related imaging abnormalities and other APOE-associated vascular disorders. Additionally, microglia cluster around Aβ deposits in CAA49. In this study, we show that short-term expression of astrocyte-derived E2 reduces proximity and colocalization of reactive microglia to plaques. Perhaps, the lack of observed effects on CAA after the astrocytic-E2 switch can be attributed to a dampened microglial response. While the specific role of microglia in CAA still requires more investigation, decreased microglial reactivity could also reduce or impair recruitment of other immune cells, such as perivascular or border-associated macrophages50. Lastly, we show that a late disease stage ‘switch' to expression of astrocytic E2 reduces plaque-associated ApoE. ApoE is a major component of plaques51. The literature suggests that there are isoform-dependent differences in ApoE–plaque associations but that several factors, such as ApoE lipidation state, compartment or cell type of origin, and Aβ conformational state, also have important roles in ApoE–Aβ interactions52,53,54,55,56. Our findings, taken together with the literature, suggest that astrocyte-derived E4's ability to bind Aβ increases Aβ accumulation and ApoE protein aggregation within or surrounding the plaque. These results are consistent with studies implicating ApoE in plaque compaction and reflect a similar loss of function as seen in Apoe knockout mice, which also feature reductions in plaque-associated microgliosis57. Also in line with these results is a recent study, which showed that plaque-associated ApoE attracts microglia to the plaque and stimulates their activation via a vascular cell adhesion molecule 1-based mechanism58. In summary, the current study leverages targeted APOE allele replacement as a promising therapeutic intervention to mitigate AD risk and describes a model in which we can recapitulate potentially beneficial or deleterious effects of APOE replacement. However, several limitations should be addressed. First, the degree of risk and protection afforded by E4 and E2 respectively vary according to ancestry1,59,60,61,62. Thus, some populations may not appreciably benefit from E4-E2 replacement strategies; future APOE-targeted therapies should take ancestry into account when conducting risk–benefit analyses. Second, the 5xFAD model used in this study provided a model with well-characterized APOE-dependent changes in amyloid pathology55,63,64. However, future studies should consider the roles of other facets of the disease, including tauopathies. Finally, Aldh1l1-CreERT2 is highly selective for astrocytes in the CNS, but Aldh1l1 is also expressed in the liver65, as evidenced by the plasma lipid profile of APOE4s2A mice. Our study suggests that astrocyte-exclusive ‘switching' from E4 to E2 in the CNS can mitigate AD pathologies; however, future studies should specifically dissect out the role of the peripheral ApoE on the CNS. Ultimately, we hope that this and other inducible APOE models will provide clean experimental systems to test the efficacy of several APOE gene editing strategies. In combination with select Cre strains and other molecular biology approaches, next-generation ‘switch' models can provide cell-specific and temporally controlled editing among APOE alleles. Using this framework, future studies can help determine optimal cell-specific contributions of ApoE to AD pathogenesis, test for potential off-target effects of APOE manipulation and establish optimal therapeutic windows for APOE-directed clinical strategies. All animal procedures and protocols were approved by the Institutional Animal Care and Use Committee and the University of Kentucky. APOE switch mice (APOE4s2) were generated on a C57BL/6N background at TransViragen. The coding region of APOE (exon 4) was followed by a stop cassette (3× SV40 poly(A)) and flanked with loxP. After the coding region for APOE4, an additional exon 4 contained the sequence for APOE2. 004847, research resource identifier (RRID): IMSR_JAX:004847) and Aldh1l1-Cre/ERT2 mice (strain no. 029655, RRID: IMSR_JAX: 029655) were purchased from The Jackson Laboratories and crossed with homozygous APOE4s2loxP/loxP mice to generate ROSA26Cre/+/APOE4s2loxP/loxP (APOE4s2G) or Aldh1l1Cre/+/APOE4s2loxP/loxP (APOE4s2A) mice. APOE4s2A mice were further crossed to 5xFAD mice to generate Aldh1l1Cre/+/APOE4s2loxP/loxP hemizygous for the familial AD mutation (4s2A/FAD). Human APOE TR mice homozygous for APOE2 or APOE4 and Cre− littermates were used as controls37. Animals were genotyped at 21 days for the APOE switch (APOE4s2) transgene using PCR. Animals were bred to homozygosity for the APOE4s2 allele. Primers specific for the humanized target allele were used to identify whether animals were homozygous for the APOE4s2 allele. T5648, Sigma-Aldrich) was dissolved overnight at 37 C° with gentle rocking in sterile sunflower seed oil (cat. S5007, Sigma-Aldrich) at a concentration of 20 mg ml−1 and protected from light. TAM was administered via intraperitoneal (i.p.) injection once every 24 h for five consecutive days. The injection dose of 75 mg kg−1 was determined according to animal body weight. Brain and liver were collected from APOE4s2G, APOE4s2A, APOE4s2, E2-TR and E4-TR mice switched between 4–6 months of age 72 h, 1 month, 3 months, 6 months or 12 months after TAM administration. RNA was extracted from whole-brain and liver tissue using the RNeasy Mini Kit (cat. 74104, QIAGEN) and converted to complementary DNA (cDNA) (cat. The APOE genotype was determined using PCR with TaqMan assay primers for the two allele-determining single-nucleotide polymorphisms of APOE: rs7412 and rs429358 (cat. Positive human DNA controls for the APOE genotypes E2/E2 and E4/E4 were included with each assay. To quantify expression of homeostatic and DAM genes in 4s2/FAD and 4s2A/FAD mice, brains were collected 2 months after TAM. RNA was extracted and converted to cDNA as described above. TaqMan Gene Expression Assays (cat. 4331182, Thermo Fisher Scientific) were used to measure Cx3cr1 (Mm02620111_s1), Tmem119 (Mm00525305_m1), P2ry12 (Mm01950543_s1), Trem2 (Mm04209422_m1), Clec7a (Mm01183349_m1) and APOE (Hs00171168_m1). Data were normalized to endogenous 18S control (Hs99999901_s1) and the fold change was calculated between groups using \({2}^{-\varDelta \varDelta \,{{\rm{C}}}_{{\rm{t}}}}\) values. Brains were collected from mice who were transcardially perfused with sterile 0.9% saline (cat. S9625, Sigma-Aldrich) unless stated otherwise. The brain was dissected into two hemispheres. The left hemisphere was flash-frozen in liquid nitrogen for biochemical analyses while the right hemisphere was drop-fixed in 4% paraformaldehyde for histology. For the biochemical analyses, the left hemisphere was cryopulverized and 25 mg of tissue was homogenized in radioimmunoprecipitation assay buffer supplemented with a cOmplete protease inhibitor (cat. 11836153001, Sigma-Aldrich) and PhosSTOP phosphatase inhibitor (cat. 4906837001, Sigma-Aldrich) unless specified otherwise. The homogenate was passed through a 20-g needle, agitated for 15 min using a Disrupter Genie (cat. To extract the soluble and insoluble brain fractions, 25 mg of liquid-nitrogen-pulverized whole-brain tissue was sequentially homogenized with PBS and then 5 M guanidine buffer with protease and phosphatase inhibitors. PBS buffer was added at 20 μl mg−1; then, the homogenate was passed through a 20-g needle, agitated for 15 min and centrifuged for 30 min at 15,000 rpm at 4 °C. The supernatant was collected and saved as a PBS-soluble fraction. The same amount of 5 M guanidine buffer was added to the pellet and agitated for 15 min followed by 1-h rotation at room temperature. The supernatant was collected as an insoluble fraction. Plasma ApoE was concentrated using the AlbuVoid Albumin Depletion Kit (cat. AVK10, Biotech Support Group) and filter centrifugation (cat. Brain homogenates were immunoprecipitated for ApoE using magnetic Dynabeads (cat. 10018D, Thermo Fisher Scientific) and incubated with 100 μg of sample for 1 h at room temperature. Samples were eluted from the beads using a denaturing elution with elution buffer and 2× Laemmli loading buffer (cat. Equal amounts of protein were loaded (40 μg) with 2× Laemmli loading buffer and run on a 4–20% gel (cat. The gel was stained with SimplyBlue SafeStain (cat. LC6060, Thermo Scientific) for 1 h at room temperature. Protein bands corresponding to ApoE (~36 kDa) were excised from the gel and subjected to a dithiothreitol reduction, iodoacetamide alkylation and in-gel trypsin digestion. Peptides were extracted and concentrated using filtration with a 0.22-μm polyvinylidene fluoride filter and then subjected to LC–MS/MS analysis using the Q Exactive Orbitrap Mass Spectrometer (Thermo Fisher Scientific). The LC–MS/MS datasets were searched in MASCOT against a custom database of exclusively human ApoE2 and ApoE4 peptides, including dynamic modifications of carbamidomethyl (C) and oxidation (M). The overall ratio of E2 to E4 peptides detected in each sample was calculated as the total intensity for specific E4 or E2 peptides. Experimental and control animals were fed either normal chow or a Western diet (cat. TD.88137, Envigo) for 4 months to exacerbate hyperlipidemic phenotypes. Total plasma TG, cholesterol and phospholipid levels were quantified using colorimetric assays (Fujifilm Wako Chemicals). Plasma lipoprotein fractions were separated using fast protein liquid chromatography. Plasma samples were pooled to a final volume of 100 μl (n = 4 per group) and separations were performed on an ÄKTA pure instrument equipped with Superose 6 (GE Healthcare) columns arranged in series. Plasma was run over the columns at a flow rate of 0.5 ml min−1 in PBS buffer and 0.5-ml fractions were collected. TG, cholesterol and phospholipid levels were quantified in each fraction using colorimetric assays (Fujifilm Wako Chemicals). A commercially available sandwich ELISA kit (cat. ab108813, Abcam) was used to measure total ApoE in the periphery and brain, and ApoE in soluble and insoluble whole-brain fractions. PBS-soluble brain fractions were diluted 1:10 and insoluble fractions were diluted 1:50. The average absorbance of each standard and sample were calculated and corrected by the dilution factor. ApoE concentration was calculated using the equation of the standard curve and normalized by total tissue weight. Untargeted lipidomic analysis was performed on whole-brain homogenate from 6–8-month-old TAM-treated APOE4s2G, APO4s2, E2-TR and E4-TR mice by a commercial company (BGI) using LC–MS/MS. A high-resolution Q Exactive mass spectrometer (Thermo Fisher Scientific) was used for data acquisition in positive-ion and negative-ion mode, respectively, to improve lipid coverage. Data were processed using LipidSearch v.4.1 and the BGI's statistical software package, metaX. Metabolite abundance averages were calculated within groups and multiple group t-tests were performed on the entire dataset. Volcano plots were made for each condition in the dataset and also according to lipid class. Mice were anesthetized using 5.0% isoflurane before exsanguination and transcardial perfusion with ice-cold Dulbecco's PBS (DPBS) (cat. After perfusion, brains were quickly removed and whole right hemispheres sans brainstem and cerebellum were quickly minced using forceps on top of an ice-chilled Petri dish. 130-093-237, Miltenyi Biotec) containing Adult Brain Dissociation Kit (ABDK) enzymatic digest reagents (cat. 130-107-677, Miltenyi Biotec) prepared according to the manufacturer's protocol. Tissues were dissociated using the ‘37C_ABDK' protocol on the gentleMACS Octo Dissociator instrument (cat. 130-095-937, Miltenyi Biotec) with heaters attached. After tissue digestion, cell suspensions were filtered through 70-μm mesh cell filters to remove debris according to the manufacturer's suggested ABDK protocol. The resultant suspension was sequentially filtered (×2) using fresh 30-μm mesh filters. Cell viability was checked using the Acridine Orange/Propidium Iodide viability kit (cat. After viability and counting, cells were diluted to achieve a concentration of ~1,000 cells per 100 μl. The diluted cell suspensions were loaded onto the 10x Chromium instrument. Each sample was loaded into a separate channel on the Single Cell 3′ Chip and libraries were prepared using the Chromium v3 Single Cell 3′ Library and Gel Bead Kit (10x Genomics). Final library quantification and quality checking were performed using a Bioanalyzer instrument (Agilent Technologies); 150-bp paired-end sequencing (Novogene) was performed on a NovaSeq 6000 S4 flow cell. After libraries were sequenced and quality control was performed, samples were aligned with STAR aligner to the mm10 mouse reference genome using the Cell Ranger v.6.1.1 pipeline. The expression matrix was loaded into R for further analysis and visualization using Seurat (v.5.0.3). Ambient RNA contamination was removed using the SoupX R package. Doublets were removed using the scDblFinder package. Cells were then filtered to reduce the potential of including low-quality cells with the following criteria: 200 < nGene < 2000; 200 < nCount; and percent.mito <20%. Feature counts were normalized using sctransform and the effects of percent.mito were regressed out. Datasets were integrated using the Seurat standard protocol. Principal component analysis was performed and nearest neighbor analysis was done using the FindNeighbors function. FindClusters with a Louvain algorithm was used to identify clusters, and clustering was performed using a resolution of 1.0. Dimension reduction was performed using UMAP on the top 50 principal components. Forty-five clusters were identified. Four mixed clusters with high hemoglobin were removed along with several clusters with pituitary contamination (high expression of Gh, Prl and Lhx3). To assign glial cell type identity to each cluster, we manually examined the expression levels of cell-type-specific markers across each cluster using the Partek Flow software to identify clusters containing unique populations of different cell types. Canonical CNS cell type markers were compiled from previous studies66,67,68,69. This process resulted in stringent filtering of cells with ambiguous assignments, leaving a total of 85,701 cells in 41 carefully assigned clusters. The Seurat function FindMarkers with the MAST differential expression test were used to conduct the DEG analysis via grouping for comparison according to clusters and treatment group. DEGs were selected if the Padj < 0.001 and the absolute value of the log fold change was higher than 1.0. Based on the identified DEGs, enrichment analyses of GO terms (Biological Process (BP2021)) were performed via Enrichr or the rWikiPathways R software package70, with cutoff represented by an FDR-adjusted P < 0.05. Bar plots of the top five most significant downregulated and upregulated pathways were generated for the visualizations. Mice were anesthetized using 5.0% isoflurane before exsanguination and transcardial perfusion with ice-cold DPBS. After perfusion, brains were quickly removed and whole right hemispheres sans brainstem and cerebellum were quickly minced using forceps on top of an ice-chilled Petri dish. Minced tissue from the hemispheres was immediately transferred into a gentleMACS C-tube containing ABDK enzymatic digest reagents prepared according to the manufacturer's protocol. Tissues were dissociated using the ‘37C_ABDK' protocol on the gentleMACS Octo Dissociator instrument with heaters attached. After tissue digestion, cell suspensions were filtered through 70-μm mesh cell filters to remove debris according to the manufacturer's suggested ABDK protocol. The resultant suspension was sequentially filtered (×2) using fresh 30-μm mesh filters. Cell viability was checked using the Acridine Orange/Propidium Iodide viability kit. Cell suspensions were then subjected to fluorescence-activated cell sorting to sort tdTomato expression astrocytes from non-fluorescent Cre− cells in the brain. A total of ~110,000 cells were sorted and collected in DPBS. Cells were subsequently prepped for RNA isolation. Brains were post-fixed in 4% paraformaldehyde (cat. 043368.9M, Thermo Fisher Scientific) for 24 h at 4 °C then stored in a 30% sucrose solution with 0.05% sodium azide (cat. Coronal cryosections (30 μM) were obtained using a Leica Cryostat (CM1520) and serially collected in a cryoprotectant solution (50% 0.1 M PB, 30% ethylene glycol, 20% glycerol). Hippocampus-containing sections were selected for all stains, which were performed free-floating unless stated otherwise. For immunofluorescence detection of ApoE (1:500 dilution, cat. 13366S, Cell Signaling Technologies), GFAP (1:500 dilution, cat. 3NB100-1028, Novus Biologicals; 1:500 dilution, cat. ab178846, Abcam), MHC-II (1:200 dilution, cat. 14-5321-82, Thermo Fisher Scientific), PSD95 (1:500 dilution, cat. 51-6900, Invitrogen), anti-CAA mOC31 (1:100 dilution, cat. ab56299, Abcam), sections were washed three times for 5 min in 1× PBS and permeabilized in 1× PBS + 0.2% Triton X-100 for 10 min. Sections were blocked in 10% goat serum in 1× PBS + 0.2% Triton X-100 (ApoE + GFAP) or 10% donkey serum in 1× PBS + 0.2% Triton X-100 (ApoE + IBA1) for 1 h at room temperature. Primary antibody cocktails were diluted in 3% goat or donkey serum and sections were incubated overnight at 4 °C. Sections were washed three times for 10 min in 1× PBS before applying the corresponding secondary antibodies (Goat anti-Rabbit Alexa Fluor 488, cat. A32731, Invitrogen; Goat anti-Rat Alexa Fluor 568, cat. A11077, Invitrogen; Donkey anti-Rabbit Alexa Fluor 488, cat. A32790, Invitrogen; Donkey anti-Goat Alexa Fluor 555, cat. A32816, Invitrogen) at a dilution of 1:250. Sections were incubated in secondary antibodies for 2 h at room temperature, protected from light. Sections were briefly washed in 1× PBS and mounted and coverslipped using a mounting medium with DAPI (cat. H-1500, Vector Laboratories) or NucSpot nuclear stain (cat. For immunofluorescence detection of amyloid, sections were mounted onto slides after secondary incubation and allowed to dry. Slides were stained for amyloid plaques using Amylo-Glo (cat. TR-300-AG, Biosensis) and coverslipped with a mounting medium containing a far-red nuclear stain (cat. Imaging of GFAP, Iba1 and Amylo-Glo tissue-mounted slides was performed using the ZEISS AxioScan 7 to capture full mouse brain coronal sections. Images were uploaded to the Indica Labs HALO image analysis platform. All other tissue was imaged using a Nikon AX/AXR confocal microscope to capture the superior half (including the hippocampus and isocortex) of whole mouse brain coronal sections. All images were uploaded and analyzed with the Indica Labs HALO image analysis platform using the following algorithms: Area Quantification FL (v.2.3.3); Co-localization FL (v.2.1.4); and Proximity analysis (v.2.1.4). Associative learning and memory were assessed using the cued and contextual fear conditioning test over the course of 2 days. Mice were placed in a sound-attenuated chamber with grid flooring capable of delivering an electric shock. Freezing was measured with an overhead camera and the EthoVision XT software as a fear response output measure. On day 1, each chamber was prepared with context no. 1 consisting of white walls, gray metal grid flooring and MB-10 cleaning solution scent. Mice were placed in the chamber and were undisturbed for 3 min as baseline freezing was recording. On the third minute, a paired light and tone cue was played for 15 s; this was immediately followed by delivery of a 0.35-mA shock stimulus that lasted for 1 s. This cue and stimulus combination was repeated four times on every 60-s increment. On day 2, 24 h after day 1, mice were again placed in a chamber with context no. Mice were recorded for 5 min with no cues or stimulus to assess their ability to associate the context of the chamber from the conditioned fear response. After recording, mice were removed from the chambers. All chambers were thoroughly cleaned and prepared with context no. 2, which consisted of vertical black and white striped walls, white floors and a vanilla scent. Two hours later, mice performed the cued test, in which they were placed in chambers with context no. Mice were undisturbed for 3 min, after which three paired light and tone cues were played for 15 s each on every 60-s interval. Percentage freezing was recorded in 60-s intervals. The water maze consisted of a circular pool, filled with opaque water (white paint added, 24 °C) divided conceptually into four quadrants. Mice were trained to locate an ‘escape' platform (Plexiglas circle, 6-cm radius) submerged 2 cm below the surface of the water for five consecutive days during the ‘hidden platform' trails, which required mice to rely on extra-maze cues for spatial reference and orientation. Extra-maze cues consisted of four large (50 × 50 cm) cues of different shapes and color combinations, positioned at the borders of the four quadrants. However, the location in which mice were dropped into the pool differed for each session between the four quadrants to avoid procedural bias in task learning. Mice were given two sessions per day (separated by 3 h) consisting of two trials each (separated by 10 min). Spatial memory retention was assessed 72 h after the final ‘hidden platform' training session. The submerged platform was removed and spatial memory retention assessed during these ‘probe' trials by analyzing the number of times mice crossed over the location of the platform and the time spent swimming in the target quadrant compared to the time spent in the three non-target quadrants. All values are reported as ± s.e.m. All statistical analyses were conducted in Prism 10 (GraphPad Software) unless noted otherwise; t-tests were used for assessing significance between two groups, while two-way ANOVAs were used to assess significance between more than two groups. Significance was determined as P < 0.05 for all tests unless stated otherwise. Significant P values, the value of n per group and experimental groups can be found in the figure legends. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The scRNA-seq data were deposited into the Gene Expression Omnibus under accession no. We reanalyzed data from the following existing sources: Murdock & Tsai (syn18485175GSE138852, GSE147528; syn21125841, GSE157827, GSE129308, GSE146639, GSE163577, GSE148822; and syn3219045, GSE181279, GSE134578, GSE160936); De Bastiani et al. (GSE149661, GSE145907, GSE28146, GSE29378, GSE36980, GSE48350, GSE84422 and GSE123496); Zhao et al. (syn22307008); Habib et al. (GSE143758); Keren-Shaul et al. (GSE98971); and Krasemann et al. (GSE101689). All other data supporting the findings of this study are available within the paper and its Supplementary Information. Belloy, M. E. et al. APOE genotype and Alzheimer disease risk across age, sex, and population ancestry. Protective effect of apolipoproteinE type 2 allele for late onset Alzheimer disease. Garatachea, N. et al. ApoE gene and exceptional longevity: insights from three independent cohorts. Wilson, R., Bienias, J., Berry-Kravis, E., Evans, D. & Bennett, D. The apolipoprotein E ε2 allele and decline in episodic memory. A. et al. ApoE cascade hypothesis in the pathogenesis of Alzheimer's disease and related dementias. Raulin, A.-C. et al. ApoE in Alzheimer's disease: pathophysiology and therapeutic strategies. Brookhouser, N., Raman, S., Frisch, C., Srinivasan, G. & Brafman, D. A. APOE2 mitigates disease-related phenotypes in an isogenic hiPSC-based model of Alzheimer's disease. Lin, Y. T. et al. APOE4 causes widespread molecular and cellular alterations associated with Alzheimer's disease phenotypes in human iPSC-derived brain cell types. Dodart, J.-C. et al. Gene delivery of human apolipoprotein E alters brain Aβ burden in a mouse model of Alzheimer's disease. Hudry, E. et al. Gene transfer of human Apoe isoforms results in differential modulation of amyloid deposition and neurotoxicity in mouse brain. Zhao, L. et al. Intracerebral adeno-associated virus gene delivery of apolipoprotein E2 markedly reduces brain amyloid pathology in Alzheimer's disease mouse models. Jackson, R. J. APOE2 gene therapy reduces amyloid deposition and improves markers of neuroinflammation and neurodegeneration in a mouse model of Alzheimer disease. Rosenburg, J. et al. AAVrh.10-mediated APOE2 central nervous system gene therapy for APOE4-associated Alzheimer's disease. Gene therapy for APOE4 homozygote of Alzheimer's disease https://clinicaltrials.gov/study/NCT03634007 (ClinicalTrials.gov, 2024). Selective removal of astrocytic APOE4 strongly protects against tau-mediated neurodegeneration and decreases synaptic phagocytosis by microglia. Huynh, T. V. et al. Age-dependent effects of apoE reduction using antisense oligonucleotides in a model of β-amyloidosis. Targeting of nonlipidated, aggregated apoE with antibodies inhibits amyloid accumulation. Shi, Y. et al. Overexpressing low-density lipoprotein receptor reduces tau-associated neurodegeneration in relation to apoE-linked mechanisms. Salomonsson, S. E. & Clelland, C. D. Building CRISPR gene therapies for the central nervous system: a review. Sullivan, P., Mezdour, H., Quarfordt, S. & Maeda, N. Type III hyperlipoproteinemia and spontaneous atherosclerosis in mice resulting from gene replacement of mouse Apoe with human Apoe*2. Sullivan, P. M., Mace, B. E., Maeda, N. & Schmechel, D. E. Marked regional differences of brain human apolipoprotein E expression in targeted replacement mice. Lefterov, I. et al. APOE2 orchestrated differences in transcriptomic and lipidomic profiles of postmortem AD brain. Fitz, N. F. et al. Phospholipids of APOE lipoproteins activate microglia in an isoform-specific manner in preclinical models of Alzheimer's disease. Insights into Alzheimer's disease from single-cell genomic approaches. De Bastiani, M. A. et al. Cross-species evidence of differential expression of S100A6 and SLC11A1 in the hippocampus of Alzheimer's disease patients and mouse models. Zhao, N. et al. Alzheimer's risk factors age, APOE genotype, and sex drive distinct molecular pathways. Habib, N. et al. Disease-associated astrocytes in Alzheimer's disease and aging. Keren-Shaul, H. et al. A unique microglia type associated with restricting development of Alzheimer's disease. Profile and regulation of apolipoprotein E (ApoE) expression in the CNS in mice with targeting of green fluorescent protein gene to the ApoE locus. Srinivasan, R. et al. New transgenic mouse lines for selectively targeting astrocytes and studying calcium signals in astrocyte processes in situ and in vivo. Castellano, J. M. et al. Human apoE isoforms differentially regulate brain amyloid-β peptide clearance. The major risk factors for Alzheimer's disease: age, sex, and genes modulate the microglia response to Aβ plaques. Lee, S. et al. APOE modulates microglial immunometabolism in response to age, amyloid pathology, and inflammatory challenge. Increased ceramide in brains with Alzheimer's and other neurodegenerative diseases. Exton, J. H. Signaling through phosphatidylcholine breakdown. Selective reduction of astrocyte apoE3 and apoE4 strongly reduces Aβ accumulation and plaque-related pathology in a mouse model of amyloidosis. Tcw, J. et al. Cholesterol and matrisome pathways dysregulated in astrocytes and microglia. Ostendorf, B. N. et al. Common germline variants of the human APOE gene modulate melanoma progression and survival. Adams, M. K. M. et al. Apolipoprotein E gene associations in age-related macular degeneration: the Melbourne Collaborative Cohort Study. Johnson, L. A. et al. ApoE2 exaggerates PTSD-related behavioral, cognitive, and neuroendocrine alterations. Wang, H. Whole-genome sequencing analysis reveals new susceptibility loci and structural variants associated with progressive supranuclear palsy. Johnson, L. A. et al. Apolipoprotein E-low density lipoprotein receptor interaction affects spatial memory retention and brain ApoE levels in an isoform-dependent manner. Decreased lipidated ApoE-receptor interactions confer protection against pathogenicity of ApoE and its lipid cargoes in lysosomes. Koutsodendris, N. et al. Neuronal APOE4 removal protects against tau-mediated gliosis, neurodegeneration and myelin deficits. Anfray, A. et al. A cell-autonomous role for border-associated macrophages in ApoE4 neurovascular dysfunction and susceptibility to white matter injury. Liu, C.-C.et al. ApoE4 accelerates early seeding of amyloid pathology. Boyle, P. A. et al. Cerebral amyloid angiopathy and cognitive outcomes in community-based older persons. Nelson, P. T. et al. APOE-ε2 and APOE-ε4 correlate with increased amyloid accumulation in cerebral vasculature. Zabel, M. et al. A shift in microglial β-amyloid binding in Alzheimer's disease is associated with cerebral amyloid angiopathy. Munsterman, D. et al. Cerebral amyloid angiopathy and the immune system. Gal, J. et al. Apolipoprotein E proteinopathy is a major dementia-associated pathologic biomarker in individuals with or without the APOE Epsilon 4 Allele. Binding of human apolipoprotein E to synthetic amyloid beta peptide: isoform-specific effects and implications for late-onset Alzheimer disease. LaDu, M. J. et al. Purification of apolipoprotein E attenuates isoform-specific binding to β-amyloid. LaDu, M. J. et al. Isoform-specific binding of apolipoprotein E to β-amyloid. Tai, L. M. et al. Levels of soluble apolipoprotein E/amyloid-β (Aβ) complex are reduced and oligomeric Aβ increased with APOE4 and Alzheimer disease in a transgenic mouse model and human samples. Hashimoto, T. et al. Apolipoprotein E, especially apolipoprotein E4, increases the oligomerization of amyloid β peptide. Ulrich, J. D. et al. ApoE facilitates the microglial response to amyloid plaque pathology. The VCAM1–ApoE pathway directs microglial chemotaxis and alleviates Alzheimer's disease pathology. Increased APOE ε4 expression is associated with the difference in Alzheimer's disease risk from diverse ancestral backgrounds. Huggins, L. K. L. et al. Meta-analysis of variations in association between APOE ɛ4 and Alzheimer's disease and related dementias across Hispanic regions of origin. & Choi, K. Y. APOE promoter polymorphism-219T/G is an effect modifier of the influence of APOE ε4 on Alzheimer's disease risk in a multiracial sample. Eisenberg, D. T., Kuzawa, C. W. & Hayes, M. G. Worldwide allele frequencies of the human apolipoprotein E gene: climate, local adaptations, and evolutionary history. Tai, L. M. et al. EFAD transgenic mice as a human APOE relevant preclinical model of Alzheimer's disease. The role of APOE in transgenic mouse models of AD. Nemeth, D., Luqman, N., Chen, L. & Quan, N. Aldh1l1-Cre/ER T2 is expressed in unintended cell types of the salivary gland, pancreas, and spleen. Brain cell type specific gene expression and co-expression network architectures. Zeisel, A. et al. Molecular architecture of the mouse nervous system. Ximerakis, M. et al. Single-cell transcriptomic profiling of the aging mouse brain. Han, X. et al. Mapping the mouse cell atlas by Microwell-seq. Slenter, D. N. et al. WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research. We thank D. Adreon, C. Friday and H. Williams for their invaluable feedback. We thank the University of Kentucky Proteomics Core for their help with proteomics analysis and the Animal Behavior Core for their assistance with behavioral testing. This work was supported by the National Institute of Health (grant nos. ), Coins for Alzheimer's Research Trust Grant (S.L.M.) Lesley R. Golden, Dahlia S. Siano, Isaiah O. Stephens, Steven M. MacLean, Georgia L. Nolt, Jessica L. Funnell, Akhil V. Pallerla, Sangderk Lee, Cathryn Smith, Clairity Voy, Callie M. Whitus, Gabriela Hernandez, Brandon C. Farmer, Shannon L. Macauley, Scott M. Gordon & Lance A. Johnson Sanders-Brown Center on Aging, University of Kentucky, Lexington, KY, USA Lesley R. Golden, Steven M. MacLean, Kai Saito, Sangderk Lee, Shannon L. Macauley, Josh M. Morganti & Lance A. Johnson Department of Molecular and Cellular Biochemistry, University of Kentucky, Lexington, KY, USA Jing Chen & Haining Zhu Department of Pharmacology and Toxicology, University of Arizona, Tucson, AZ, USA TransViragen Inc., Chapel Hill, NC, USA Kumar Pandya & Dale O. Cowley Animal Models Core Facility, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Shannon L. Macauley & Josh M. Morganti Saha Cardiovascular Research Center, University of Kentucky, Lexington, KY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar supervised the scRNA-seq workflow, with data analysis by K.S, S.L. contributed to the lipidomics analyses. assisted with the biochemical and immunohistochemical analyses. assisted with experimental design, organization and implementation of mouse studies. supervised the plasma lipid analyses with technical assistance from C.M.W. designed and generated the transgenic animals with additional design input from L.A.J. All authors read the paper and provided edits. is employed by, has equity ownership in and serves on the board of directors of TransViragen, the company which has been contracted by UNC-Chapel Hill to manage its Animal Models Core Facility. The other authors declare no competing interests. Nature Neuroscience thanks Guojun Bu, Hui Zheng and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a,d,g,j, Extracted-ion chromatogram (XIC) of peaks from unique E4 peptide LGADMEDVR in an E4-TR control sample (a) and an APOE4s2 sample expressing E4 protein (d) or peaks from unique E2 peptide LGADMEDVRGC in and E2-TR control sample (g) or an APOE4s2G sample (j) expressing E2 protein. b,e,h,k, Mass spectra (MS) showing peaks with similar mass to charge ratios of unique E4 peptide LGADMEDVR in an E4-TR control sample (b) and an APOE4s2 sample expressing E4 protein (e) or peaks from unique E2 peptide LGADMEDVRGC in and E2-TR control sample (h) or an APOE4s2G sample (k) expressing E2 protein. c,f,i,l, Tandem mass spectra (MS/MS) showing ion fragment peaks with similar mass to charge ratios of unique E4 peptide LGADMEDVR in an E4-TR control sample (c) and an APOE4s2 sample expressing E4 protein (f) or peaks from unique E2 peptide LGADMEDVRGC in and E2-TR control sample (i) or an APOE4s2G sample (l) expressing E2 protein. m, Peptide sequences unique to ApoE2 or ApoE4 at amino acid residues 112 and 159 following trypsin digestion. Sequences were used to create a custom database of hApoE2 and hApoE4 peptides. LC-MS/MS data sets generated from plasma and brain samples were searched in against these sequences at both amino acid residues to generate a semi-quantitative ratio of E2:E4 in APOE4s2 and APOE4s2G samples. a, Allelic discrimination plot of mRNA analysis showing efficient and sustained expression of APOE2 in APOE4s2G mice at 72 h, 3 months, 6 months, and 12 months post switch compared to E2-TR, E4-TR, and APOE4s2 controls (nE2-TR = 6, nE4-TR = 6, nAPOE4s2 = 6, nAPOE4s2G = 6 for all timepoints).). c, ApoE quantified in SEC fractions corresponding to VLDL, IDL/LDL and HDL lipoprotein classes. d,f,h, Triglycerides (d), total cholesterol (f), and phospholipids (h) quantified in plasma from E2-TR, E4-TR, APOE4s2, and APOE4s2G mice on normal chow (nE2-TR = 3, nE4-TR = 3, nAPOE4s2 = 5, nAPOE4s2G = 5; *p < 0.05, **p < 0.01, ****p < 0.0001; 2way ANOVA with multiple comparisons). e,g,i, Triglycerides (e), total cholesterol (g), and phospholipids (i) quantified in FPLC eluants corresponding to VLDL, LDL and HDL (nE2-TR = 3 pooled, nE4-TR = 3 pooled, nAPOE4s2 = 4 pooled, nAPOE4s2G = 4 pooled). j, Representative images of ApoE (red), NeuN (green), and DAPI (blue) staining in APOE4s2G and APOE4s2 controls. k, Representative images of ApoE (red) and NeuN (green) with DAPI nuclear stain (blue) in E4-TR and E2-TR brains. l, Representative images of ApoE (red), IBA1 (green), and DAPI (blue) in E4-TR and E2-TR mice. m, Representative images of ApoE (red), GFAP (green), and DAPI (blue) in E4-TR and E2-TR brains. n, Quantification of ApoE with astrocytes, microglia, and neurons in the hippocampus. o, Heatmap depicting top 10 most significantly changed lipids between E2-TR and E4-TR controls. p, Volcano plot showing differentially expressed lipids betweenE4-TR vs. E2-TR mice (nE2-TR = 8, nE4-TR = 8; all females). For detailed statistical information, see supplementary table 3. Individual data points plotted. Box plots show center line, median; box limits, upper and lower quartiles; whiskers, minimum and maximum. a, UMAP identifying a total of 41 unique clusters before cell-type assignment. b, UMAP showing relatively even distribution of clusters across genotypes. c, After cell-type assignment, plots show relatively even distribution of cell type across different genotypes. d, Violin plots showing expression of gene markers in each cell type. e, Gene expression in each cell-type cluster of cell-specific gene markers. a, Volcano plots depicting differentially expressed genes in astrocytes identified between APOE4s2A vs. APOE4s2, APOE4s2G vs. APOE4s2, and E2-TR vs. E4-TR comparisons. p = 0.001, log2FC = 1.0. b, Volcano plots depicting differentially expressed genes in oligodendrocytes identified between APOE4s2A vs. APOE4s2, APOE4s2G vs. APOE4s2, and E2-TR vs. E4-TR comparisons. p = 0.001, log2FC = 1.0. c, Volcano plots depicting differentially expressed genes in microglia identified between , APOE4s2A vs. APOE4s2, APOE4s2G vs. APOE4s2, and E2-TR vs. E4-TR comparisons. p = 0.001, log2FC = 1.0. d, Volcano plots depicting differentially expressed genes in endothelial cells identified between APOE4s2A vs. APOE4s2, APOE4s2G vs. APOE4s2, and E2-TR vs. E4-TR comparisons. a, Allelic discrimination plot of mRNA analysis from whole brain tissue showing efficient and sustained expression of APOE2 in APOE4s2A mice at 72 h, 3 months, 6 months, and 12 months post switch compared to E2-TR, E4-TR, and APOE4s2 controls (nE2-TR = 6, nE4-TR = 6, nAPOE4s2 = 6, nAPOE4s2A = 6 for all timepoints) b, Ratio of unique E2 vs. E4 peptides detected using mass spec based proteomic analysis (nE2-TR = 2, nE4-TR = 2, nAPOE4s2 = 3, nAPOE4s2A = 3). c, Percent td-tomato Aldh1l1-Cre reporter positive area vs. tdTomato Aldh1l1-Cre reporter colocalized with GFAP staining in APOE4s2A (65%) mice vs. APOE4s2 Cre-negative controls (4%%) (APOE4s2A = 3, nAPOE4s2 = 3). d-e, Allelic discrimination plot from liver tissue (d) with ratios of relative fluorescence from SNPs associated with APOE2 vs. APOE4 showing an average of 81% APOE2 mRNA expression in the liver (e) (nE2-TR = 6, nE4-TR = 6, nAPOE4s2 = 6, nAPOE4s2A = 6). f, ApoE quantified by ELISA in SEC fractions corresponding to VLDL, IDL/LDL and HDL lipoprotein classes. g-i, Triglycerides (g), total cholesterol (h), and phospholipids (i) quantified in plasma from E2-TR, E4-TR, APOE4s2, and APOE4s2A mice on normal chow (nE2-TR = 3, nE4-TR = 3, nAPOE4s2 = 5, nAPOE4s2G = 5; *p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001; 2way ANOVA with multiple comparisons). j-l Triglycerides (j), total cholesterol (k), and phospholipids (l) quantified in FPLC eluants corresponding to VLDL, LDL and HDL lipoprotein classes (nE2-TR = 3 pooled, nE4-TR = 3 pooled, nAPOE4s2 = 4 pooled, nAPOE4s2G = 4 pooled). a-i, For detailed statistical information, see supplementary table 3. Individual data points plotted. Box plots show center line, median; box limits, upper and lower quartiles; whiskers, minimum and maximum. a-d, Percent time freezing measured during conditioning in 4s2A/FAD vs. 4s2/FAD stratified by sex. Analyses showed only one significant sex difference where 4s2/FAD males froze more than 4s2/FAD females. e, Latency to platform measured in 4s2A/FAD compared to tamoxifen treated, Cre-negative 4s2/FAD controls over the 5-day hidden platform training trials (n4s2A/FAD = 10, n4s2/FAD = 10; two-tailed, paired t-test). f, Thigmotaxis (defined by 30 s of swimming around perimeter) in 4s2A/FAD and 4s2/FAD controls during hidden platform trials (n4s2A/FAD = 10, n4s2/FAD = 10; two-tailed, paired t-test). g, Latency to platform by sex showing no sex differences in 4s2A/FAD or 4s2/FAD controls. (n4s2A/FAD = 5 females; 5 males, n4s2/FAD = 5 females; 5 males; 2way ANOVA with repeated measures and multiple comparisons). h, There were no significant sex-based differences in thigmotaxis (n4s2A/FAD = 5 females; 5 males, n4s2/FAD = 5 females; 5 males 2way ANOVA with repeated measures and multiple comparisons). i-k, Number of platform crosses (i), occupancy of target quadrant (j), and occupancy of target quadrant compared to other quadrants (k) in the 72-hour memory probe. (n4s2A/FAD = 5 females; 5 males, n4s2/FAD = 5 females; 5 males; **p < 0.01, ***p < 0.001, ****p < 0.0001; two-tailed, unpaired t-test (I,k); 2way ANOVA with multiple comparisons (k)). l-n, Sex-based analyses of platform crosses (l), time in target quadrant (m), and frequency in quadrants relative to other quadrants (n) showed no sex differences (n4s2A/FAD = 5 females; 5 males, n4s2/FAD = 5 females; 5 males; *p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001; 2way ANOVA with multiple comparisons. a-n. For detailed statistical information, see supplementary table 3. Individual data points plotted. Sex is denoted by symbol shape: females (circle), males (triangle). Box plots show center line, median; box limits, upper and lower quartiles; whiskers, minimum and maximum. a, Quantification of total amylo-glo+ area by sex showing significantly higher amyloid+ area in 4s2/FAD females than 4s2/FAD males (n4s2A/FAD = 8 females; 8 males, n4s2/FAD = 8 females; 8 males; **p < 0.01; 2way ANOVA with multiple comparisons). b, Representative image of amylo-Glo (blue) and nuclei (red) staining in 6-month-old (pre-switch) 4s2/FAD mice. c, Percent amyloid+ area quantified over the 2-month period of astrocytic E2 expression showing a significant increase in amyloid+ percent area in8-month-old 4s2/FAD mice. = 16; **p < 0.01; 2way ANOVA with multiple comparisons). d, Average plaque size at 8 mo. between 4s2/FAD controls and 4s2A/FAD mice (n4s2/FAD = 7, n4s2A/FAD = 7; two-tailed, unpaired t-test). e-h, Sex based analyses ELISA quantification of soluble and insoluble Ab42 and Ab40 species show no sex differences but suggest decreases in amyloid could be driven by trends within one sex (n4s2A/FAD = 7 females; 7 males, n4s2/FAD = 7 females; 7 males; *p < 0.05, **p < 0.01, ***p < 0.001, 2way ANOVA with multiple comparisons). i, Representative image of proximity analysis used to quantify PSD-95 puncta in 3um increments up to 15um from the plaque. PSD-95 (green), Amylo-Glo (blue), nuclei (red). j, l, PSD-95 quantification in the cortex showing decreases in PSD-95 in 4s2A/FAD mice starting at 12um from plaque center (j). Sex based analyses revealed that 4s2/FAD females had significantly more PSD-95 puncta than 4s2/FAD males farther from the plaque (l) (n4s2A/FAD = 6 females; 6 males, n4s2/FAD = 6 females; 6 males; **p < 0.01, ***p < 0.001, ****p < 0.0001; 2way ANOVA with multiple comparisons). k, m, PSD-95 quantification in the hippocampus (n4s2A/FAD = 6 females; 6 males, n4s2/FAD = 6 females; 6 males; 2way ANOVA with multiple comparisons). n, Representative images of CD31 (red), ZO1 (green), and DAPI (blue) staining. o, q, Quantification of tight junction protein ZO1 (o) with sex stratified analyses showing that 4s2A/FAD females have higher ZO1 than 4s2A/FAD males (q) (n4s2A/FAD = 8 females; 8 males, n4s2/FAD = 8 females; 8 males; **p < 0.01; 2way ANOVA with multiple comparisons). p, r, Quantification of total % area of CD31+ vessels (n4s2A/FAD = 8 females; 8 males, n4s2/FAD = 8 females; 8 males; 2way ANOVA with multiple comparisons). a-r, Detailed statistical information in supplementary table 3. Individual data points plotted. Sex is denoted by symbol shape: females (circle), males (triangle). Box plots show center line, median; box limits, upper and lower quartiles; whiskers, minimum and maximum. a,c, Quantification of total percent GFAP+ (a) or IBA1+ (c) area in 4s2A/FAD and 4s2/FAD controls at 8 months compared to baseline measures of gliosis at 6 months (nAPOE4s2-6mo = 8, nAPOE4s2-8mo. b,d, Quantification by sex of total percent GFAP+ (a) or IBA1+ (c) area in 4s2A/FAD and 4s2/FAD controls at 8 months compared to baseline measures of gliosis at 6 months (nAPOE4s2-6mo = 8, nAPOE4s2-8mo. e-g, Sex stratified analyses for colocalization of GFAP+ astrocytes and plaque (e), IBA1+ microglia and plaque (f), and plaque with both GFAP+ astrocytes and IBA1+ microglia (g). h,j, Representative images exemplifying plaque and glial cell proximity analyses in HALO software for proximity analysis of amyloid plaques and GFAP positive cells (h), or amyloid plaques and IBA1 positive cells (j). i-k, Plaque proximity analyses by sex (nAPOE4s2-6mo = 4, nAPOE4s2-8mo. = 4 females; 4 males; *p < 0.05; 2way ANOVA with multiple comparisons). a-k, Detailed statistical information in supplementary table 3. Individual data points plotted. Sex is denoted by symbol shape: females (circle), males (triangle). Box plots show center line, median; box limits, upper and lower quartiles; whiskers, minimum and maximum. a-c, Sex stratified quantification of cortical MHCII+ percent area (a) MHCII+ and IBA1+ percent area (b) or MHCII + , IBA1 + , and amylo-glo+ percent area (c) (nAPOE4s2-6mo = 8, nAPOE4s2-8mo. 2way ANOVA with multiple comparisons). d-f, Sex stratified quantification of hippocampal MHCII+ percent area (d) MHCII+ and IBA1+ percent area (e) or MHCII + , IBA1 + , and amylo-glo+ percent area (f) (nAPOE4s2-6mo = 8, nAPOE4s2-8mo. g-h, Sex stratified analysis of homeostatic (g) and DAM (h) gene expression showed no sex differences between groups. i, Representative images of ApoE (green), amylo-glo (blue), and IBA1 (orange) staining. j-k, Quantification of percent area of total ApoE in the cortex (j) or cortical plaque-associated ApoE (k) with significant reduction in total cortical ApoE, as well as plaque-associated ApoE, appearing to be driven by 4s2A/FAD females (nAPOE4s2-6mo = 8, nAPOE4s2-8mo. 2way ANOVA with multiple comparisons. l-m, Quantification of percent area of total ApoE in the hippocampus (l) or plaque-associated ApoE (m) (nAPOE4s2-6mo = 8, nAPOE4s2-8mo. = 8 females; 8 males; 2way ANOVA with multiple comparisons). n-o, ELISA quantification of total brain soluble ApoE (n) and insoluble ApoE (o) by sex 4s2/FAD females had higher insoluble ApoE than males. a-o, Detailed statistical information in supplementary table 3. Individual data points plotted. Sex is denoted by symbol shape: females (circle), males (triangle). Box plots show center line, median; box limits, upper and lower quartiles; whiskers, minimum and maximum. Differentially expressed genes according to cell type for APOE4s2G versus APOE4s2, APOE4s2A versus APOE4s2, and E2-TR and E4-TR comparisons. Detailed statistical test information according to the figures. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. et al. APOE4 to APOE2 allelic switching in mice improves Alzheimer's disease-related metabolic signatures, neuropathology and cognition. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02857-9'>Multimodal learning enables chat-based exploration of single-cell data</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 10:30:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Single-cell sequencing characterizes biological samples at unprecedented scale and detail, but data interpretation remains challenging. Here, we present CellWhisperer, an artificial intelligence (AI) model and software tool for chat-based interrogation of gene expression. We establish a multimodal embedding of transcriptomes and their textual annotations, using contrastive learning on 1 million RNA sequencing profiles with AI-curated descriptions. This embedding informs a large language model that answers user-provided questions about cells and genes in natural-language chats. We benchmark CellWhisperer's performance for zero-shot prediction of cell types and other biological annotations and demonstrate its use for biological discovery in a meta-analysis of human embryonic development. We integrate a CellWhisperer chat box with the CELLxGENE browser, allowing users to interactively explore gene expression through a combined graphical and chat interface. In summary, CellWhisperer leverages large community-scale data repositories to connect transcriptomes and text, thereby enabling interactive exploration of single-cell RNA-sequencing data with natural-language chats. Gene expression profiling is widely used for the characterization of cells and tissues1,2. Bulk RNA sequencing (RNA-seq) provides a detailed assessment of cell states and biological functions through a straightforward and cost-effective assay3. Moreover, with single-cell RNA sequencing (scRNA-seq), researchers can disentangle the cell composition and the biological heterogeneity of tissues, organs and diseases4. Large-scale scRNA-seq is also at the heart of the Human Cell Atlas and its mission to create reference maps of all cell types in the human body5. A typical scRNA-seq dataset can be represented by a count matrix with ~20,000 genes and thousands or millions of single cells. Analyzing and interpreting such datasets are complex tasks that require both bioinformatic skills and application-specific biological domain knowledge. To facilitate scRNA-seq data analysis, software tools have been developed for a wide range of tasks including data visualization, cell clustering, cell type annotation, differential expression and gene set analysis6. Moreover, deep-learning-based ‘single-cell foundation models' (scFMs) have been trained on large scRNA-seq datasets, with the promise of going beyond specialized tools and supporting a wide range of analysis tasks that they were not explicitly optimized for7,8. Here, we demonstrate scRNA-seq data exploration with natural language, allowing the user to interrogate cells in English, with no need to adhere to any particular format or syntax. Our CellWhisperer framework supports free-text search (such as ‘Show me tissue-resident T cells in the intestine') and answers a broad range of questions about cells (for example, ‘What are these selected cells? ', ‘Which genes are highly expressed in these cells? ', ‘What is the role of KLRD1 in natural killer (NK) cells?'). The model's responses are based on the combination of selected scRNA-seq data and the biological knowledge of a large language model (LLM), resulting in answers such as ‘The selected cells appear to be CD16+ NK cells, which are a subset of NK cells that have a crucial role in the innate immune response […]', ‘The top expressed genes in these cells include NKG7, KLRD1, GNLY, GZMA, PRF1 […]', ‘KLRD1 (CD94) is a receptor that has a role in NK cell activation and cytotoxicity. It can recognize MHC class I molecules on target cells and trigger NK-cell-mediated cytotoxicity'. CellWhisperer implements this functionality with two intertwined artificial intelligence (AI) models. First, the CellWhisperer embedding model integrates RNA profiles and their metadata-derived textual annotations through multimodal contrastive learning9, creating a joint multimodal embedding of transcriptomes and text. CellWhisperer's training data comprise over a million transcriptomes and their natural-language descriptions, prepared by AI-assisted curation from two large repositories: Gene Expression Omnibus (GEO)10,11 and CELLxGENE Census12. Second, the CellWhisperer chat model adapts an open-weights LLM13,14 to answer free-text questions about cell states while considering user-provided transcriptome profiles as multimodal input. Combining these two models, CellWhisperer enables interactive chat-based exploration of scRNA-seq data, which we integrated into the widely used CELLxGENE Explorer15. The CellWhisperer software, models, training data and source code are available online (https://cellwhisperer.bocklab.org) and usage examples are provided in Fig. In summary, we developed CellWhisperer as a proof of concept for natural language as an intuitive channel to interact with scRNA-seq datasets (Supplementary Video 1). It is enabled by a multimodal AI model of transcriptomes and text, combined with the biological knowledge of an integrated chat model. We envision the interrogation of data through natural language as a key element of future AI-based bioinformatics research assistants. We present CellWhisperer, a multimodal AI that enables interactive scRNA-seq data exploration with natural-language conversations. Our method was created in three steps (Fig. 1a): (1) LLM-assisted curation of multimodal training data, resulting in 1,082,413 pairs of human RNA-seq profiles and matched textual annotations; (2) training of the CellWhisperer embedding model, which places the transcriptomes and their AI-curated textual descriptions into a joint embedding space for cell search and annotation; and (3) development of the CellWhisperer chat model for transcriptome-aware question answering and natural-language chats. This section summarizes each of these three steps, while further technical details are provided in the Methods, Supplementary Notes 2 and 3 and Extended Data Fig. a, Conceptual outline of CellWhisperer training dataset generation (left), model training and inference (center) and applications in scRNA-seq data analysis (right). b, UMAP visualization of CellWhisperer embeddings for human transcriptomes from the GEO repository. c, CellWhisperer scores for the free-text query term ‘infection' projected on the UMAP of transcriptome embeddings from b. d, Retrieval of sample metadata (here: GEO submission date) for transcriptomes selected by CellWhisperer-generated cluster labels. First, we created a large training dataset of transcriptomes (including bulk RNA-seq profiles and scRNA-seq derived pseudo-bulk profiles) with concise textual annotations (such as ‘Renal cell carcinoma tissue sample taken from a male individual at stage 2, with no metastasis, preserved in formalin-fixed paraffin-embedded blocks') across the wide range of cell types and conditions captured by GEO and CELLxGENE Census. GEO comprises human RNA-seq data from more than 20,000 individual studies based on researcher submissions, which provides tremendous thematic breadth but also a need for data harmonization. We used the ARCHS4 uniform reprocessing of GEO data16 and developed an LLM-assisted curation procedure to create concise, coherent and biologically informative textual annotations for each sample based on sample-specific metadata provided by GEO (which includes cell types, organs, tissues, diseases, experimental methods and scientific project abstracts). LLM prompts and illustrative results are shown in Supplementary Note 2. This AI-assisted data curation yielded a standardized dataset of 705,430 human transcriptomes with matched textual annotations. We also derived pseudo-bulk transcriptomes from several hundred scRNA-seq datasets in the CELLxGENE Census, including reference maps from the Human Cell Atlas. We grouped the cells in each dataset on the basis of the provided metadata and calculated pseudo-bulk transcriptomes by averaging across all scRNA-seq profiles per group. We then applied our LLM-assisted curation procedure to condense the metadata for each group into concise biological descriptions, resulting in 376,983 human transcriptomes with matched textual annotations. Second, we used the combined set of 1,082,413 annotated transcriptomes to train the multimodal CellWhisperer embedding model, which integrates the two data modalities into a joint embedding space (Extended Data Fig. The two resulting vectors were mapped into a 2,048-dimensional multimodal embedding space using conventional feed-forward neural network layers. We then trained this model to place the two modality-specific embeddings in close proximity within the joint embedding space. We validated that the resulting CellWhisperer embedding model was capable of retrieving the transcriptome corresponding to a given textual annotation and vice versa (a standard metric of CLIP model performance9) observing a mean area under the receiver operating characteristic curve (AUROC) value of 0.927 (Extended Data Fig. The trained CellWhisperer embedding model can be prompted with free-text queries to find matching transcriptomes. Third, to enable natural-language chats that take the transcriptome information into account, we customized and fine-tuned the Mistral 7B open-weights LLM13 to incorporate CellWhisperer transcriptome embeddings in addition to text queries. Our approach is inspired by multimodal LLMs that can interpret and converse about images, such as GPT-4, Gemini and LLaVA14. We generated a training dataset of 106,610 conversations including simple rule-based question–answer pairs (for example, ‘What does the sample represent? ', with the sample's textual annotation as the designated answer) and more complex LLM-generated conversations about transcriptomes and cells (technical details are provided in the Methods, examples in Supplementary Note 2). The resulting fine-tuned LLM responds to free-text questions and engages in natural-language chats about cells and their biological functions, gene-regulatory mechanisms and other biological processes that can be linked to transcriptional cell states. To illustrate CellWhisperer's ability to process, organize and annotate large transcriptome datasets, we clustered the CellWhisperer embeddings for 705,430 GEO-derived human transcriptomes and used the CellWhisperer chat model to textually annotate these clusters (Fig. The CellWhisperer embeddings successfully captured cell types, developmental stages, tissues, diseases and other cell characteristics. As each data point in this UMAP connects back to a sample in the GEO database, we can retrieve the corresponding metadata and for example assess the popularity of RNA-seq analysis for certain cell clusters and biological functions over the last decade (Fig. In summary, we built a multimodal AI that facilitates the seamless transition from transcriptomes to text and vice versa and enables the chat-based analysis of bulk and scRNA-seq data in English language. To assess how well the multimodal CellWhisperer embedding model has learned relevant aspects of human biology, we tested its ability to predict cell characteristics such as cell types, diseases, tissues and organs on the basis of cell transcriptomes in a zero-shot manner (that is, without task-specific fine-tuning or reference data). To that end, we selected expert-annotated transcriptome datasets that were not included in CellWhisperer's training data and we used CellWhisperer to assign scores for each potential cell type label to each transcriptome (Fig. We then calculated the coherence between the correct cell type labels (as annotated in the dataset) and the computed CellWhisperer scores to quantify CellWhisperer's ability to correctly annotate and identify cells and transcriptomes. We provide detailed evaluation results for this analysis in Supplementary Table 1. a, Conceptual outline of the performance evaluation for zero-shot prediction of cell types in a zero-shot manner. Left, UMAP visualization of CellWhisperer embeddings for all cells in the Tabula Sapiens dataset, colored by the dataset's expert-annotated cell types (as a ground truth). Right, CellWhisperer scores for the free-text query ‘erythrocytes' projected on the UMAP (top) and as histograms for expert-annotated erythrocytes versus other cell types (bottom). b, Comparison of expert-annotated cell types (UMAP, top left) and CellWhisperer predictions (UMAP, bottom left) for the Tabula Sapiens (20 common cell types) dataset, with the confusion matrix shown as a heat map (right). c, Bar plots of cell type prediction performance across multiple datasets and prediction methods. CellWhisperer's zero-shot prediction performance (orange) is compared to the Geneformer scFM fine-tuned for cell type prediction (purple) and with the CellAssign method for marker-based cell type prediction (gray). The plots show values that were macro-averaged across classes. d, Bar plots of CellWhisperer's zero-shot performance for predicting different cell characteristics (organ, tissue and disease), shown for selected individual classes and as macro-averages across all classes in the corresponding dataset. Dotted black lines denote random baseline performance (AUROC: 0.5; accuracy: 1/number of classes). In the Tabula Sapiens dataset, which comprises scRNA-seq profiles for 483,152 cells from 24 organs19, CellWhisperer distinguished 20 common cell types with an AUROC value of 0.94 (Fig. Mix-ups were mainly between closely related cell types, such as ‘monocytes' versus ‘classical monocytes' and between subgroups of T cells (Fig. Across all 177 annotated cell types, we obtained an AUROC value of 0.91, but with a lower accuracy value given many highly similar cell types (Fig. For bulk RNA-seq profiles of immune cells from the ImmGen consortium20 (GSE227743) and for a recently published scRNA-seq dataset of immune cells from Asian individuals21, we obtained AUROC values above 0.99; for a challenging scRNA-seq meta-analysis of human pancreas with closely related cell types and pronounced batch effects22, the AUROC value was 0.89 (Fig. These results support the robustness of our model. Although the CellWhisperer embedding model was never specifically trained to predict cell types (this capability emerged from the more general task of learning connections between transcriptomes and their textual annotations), its zero-shot predictions performed better than a widely used marker-based method23 and on par with three scFMs17,24,25 that were fine-tuned for cell type prediction (Fig. We also assessed our use of Geneformer17 as the scFM in the CellWhisperer embedding model relative to two alternative scFMs (scGPT24 and UCE25) and we observed comparable performance trends (Extended Data Fig. To test whether CellWhisperer can also predict other cell characteristics, we assessed its zero-shot prediction performance for sample annotations of diseases, tissues and organs. To that end, we assembled a collection of 14,112 disease-associated transcriptomes from GEO that were excluded from our training data. Predicting 229 disease subtypes represented in this Human Diseases dataset, CellWhisperer achieved an AUROC value of 0.82 (Fig. 2d), indicating that disease prediction is harder than cell type prediction but possible with a performance that is substantially better than a random baseline. Similarly, CellWhisperer was able to predict the tissue-of-origin of bulk and single-cell transcriptomes with better-than-random prediction performance both in the Tabula Sapiens dataset (AUROC: 0.75) and in the Human Diseases dataset (AUROC: 0.87) (Fig. To gauge the breadth of biological processes captured by our model, we investigated its recognition of expert-curated gene sets spanning diverse areas of biology. For each of 8,812 gene sets, we used the gene set label (such as ‘colorectal cancer') as a query text to CellWhisperer and determined how well each sample in our Human Diseases dataset matched the query. In other words, we tested whether CellWhisperer had implicitly learned an understanding of the genes that matter for established biological concepts, represented here by gene sets and their labels. We found a clear positive association between CellWhisperer scores for these labels and the expression of their corresponding gene sets (Extended Data Fig. 2c,d and Supplementary Table 2), indicating that our model has learned (albeit imperfectly) many of the tested biological concepts. Importantly, CellWhisperer achieved this by training on transcriptomes and their textual annotations, without having seen any expert-curated gene sets during model training. For further evaluation, we tested how well our model can distinguish between biological signal and technical noise in the Tabula Sapiens dataset, based on an established benchmark for dataset integration and batch effect correction22. We observed improved performance of the CellWhisperer multimodal embeddings compared to transcriptome-only scFMs, for both Geneformer and scGPT, whereas UCE did not profit from the multimodal CellWhisperer training (Extended Data Fig. Lastly, we assessed how well the CellWhisperer embedding model handles complex prompts and variations within them, based on a scRNA-seq dataset of human embryonic development (described in detail below). We systematically compared different wordings of the same queries and observed strong concordance between their CellWhisperer scores (Extended Data Fig. Nevertheless, CLIP-based models are known to be sensitive to prompt variations9 and we caution that different query wordings may result in different results. In summary, multiple lines of evidence (including zero-shot prediction of cell types, diseases, tissues and organs, a data integration task, gene set prediction from their labels and evaluation of prompt variations) support our conclusion that the CellWhisperer embedding model has learned a meaningful representation of cell states and biological processes, based on training data of transcriptomes and matched textual annotations. To illustrate CellWhisperer's utility in a more complex biological application, we performed a meta-analysis of embryonic development on the basis of scRNA-seq data of human embryos that we curated from the literature26,27,28,29,30,31. We identified and integrated six separate datasets with 95,092 scRNA-seq profiles of human embryos collected 3–38 days after fertilization. These data, which were not part of our training dataset, were processed and annotated with CellWhisperer (Fig. a, Overview of the Human Development scRNA-seq dataset with CellWhisperer scores for four key developmental stages. Queries were derived from Carnegie stage annotations using GPT-4o. The heat map shows mean CellWhisperer scores calculated across all cells for each time point. b, CellWhisperer scores for query ‘heart'. The average score across all cells was calculated for each time point and then standardized across time points (as z scores). c, Overlap between CellWhisperer-identified marker genes of heart development (brown), previously reported heart-specific markers derived from an atlas of fetal gene expression32 (dark gray) and randomly selected genes as controls (light gray). Left, Venn diagram with odds ratio and P value (two-sided Fisher's exact test). Right, number of papers per gene that co-mentioned the gene name and the term ‘heart' (two-sided Mann–Whitney U-test). d, Number of papers per gene that co-mentioned the gene name and the term ‘heart', stratified by gene expression enrichment in CellWhisperer-identified heart cells (x axis). P values are based on two-sided Mann–Whitney U-tests. Genes with strongly enriched expression in CellWhisperer-identified heart cells (rightmost plot) but no associated papers (red box) were analyzed further. e, Spatial gene expression in a Carnegie stage 8 (CS8) human embryo for two CellWhisperer-identified marker genes of the developing heart (left); for an established heart marker gene (ISL1) and a gene set related to heart development (center); and for a widely expressed gene (GAPDH) and a notochord-specific gene (NOTO) shown for comparison (right). The notochord as a reference region is marked in orange and gene expression is denoted by colored points. To investigate whether CellWhisperer can identify temporal dynamics in embryonic development, we prepared queries corresponding to four key developmental stages using LLM-based aggregation of vertebrate embryology descriptions. The CellWhisperer scores for these queries matched the expected timing for these stages (Fig. We next used a similar approach to identify phases of organ development, querying CellWhisperer with the names of ten organs (Extended Data Fig. These basic text queries implicitly captured a gradual activation of genes important for organ development, which we validated against the expression of organ-specific marker genes derived from an atlas of fetal gene expression32 (Extended Data Fig. CellWhisperer embeddings are biologically interpretable not only through their link to descriptive text but also by examining genes associated with high CellWhisperer scores. We determined CellWhisperer-identified marker genes for each of the ten investigated organs (Supplementary Table 3) and indeed observed strong overlap with previously reported organ marker genes32 (median odds ratio: 3.3) (Fig. For further validation, we investigated how frequently the CellWhisperer-specific marker genes were co-mentioned with the corresponding organ in publications from the PubMed database of biomedical literature. We found that these genes were co-mentioned with the organ much more frequently than a random set of genes and comparably often as the previously reported organ marker genes32 Genes that were shared between both analyses had the highest frequency of co-mentioning (Extended Data Fig. For each organ, the CellWhisperer analysis identified at least ten new marker genes beyond the previously reported organ marker genes32 (Supplementary Table 3). These genes had strong support from our analysis of co-mentioning in the biomedical literature (Fig. In addition, we observed gene set enrichments for biological functions that are characteristic for the corresponding organs (shown for heart in Extended Data Fig. 3d) and a strong spatial expression correspondence with established and widely used organ marker genes, as validated using a 3D atlas of a gastrulating human embryo33 (Fig. In summary, we applied CellWhisperer to the common and nontrivial task of marker gene discovery across multiple user-provided scRNA-seq datasets, which was achieved using simple text queries (comprising only the organ name) and yielded results that complement previously reported organ marker genes32 at comparable precision. To make CellWhisperer broadly accessible for chat-based analysis of transcriptome data, we integrated it with the CELLxGENE Explorer by adding a CellWhisperer-powered chat box (Fig. CELLxGENE Explorer is an interactive web tool for analyzing scRNA-seq profiles through visual inspection, filtering and differential analysis of cells and samples. CellWhisperer complements CELLxGENE Explorer's functionality for visual analysis by providing natural-language data exploration capabilities including (1) free-text search for cells with user-specified properties; (2) automatic textual annotation of cell clusters; and (3) chat-based investigation of interactively selected cells. More generally, CellWhisperer enables the discussion of cells and genes in natural language through a chat box integrated with the visual features of a single-cell browser. We provide a list of usage examples in Supplementary Note 1. a, Schematized screenshot of the CellWhisperer web tool, showing the Tabula Sapiens dataset with CellWhisperer scores for the free-text query ‘Show me structural cells with immune functions'. b, Zoomed-in view of the UMAP of CellWhisperer embeddings with overlaid CellWhisperer scores for the free-text query from a. Clusters of cells with high CellWhisperer scores were interactively selected in the web tool and examined by prompting the CellWhisperer chat model for a natural-language description (chat request: ‘Describe the selected cells'). Responses were trimmed to the most relevant parts (as indicated by ellipses). Annotations in gray font were manually added. c, Screenshot of a CellWhisperer conversation about the interactively selected cells (marked in b). Here, we illustrate CellWhisperer's functionality on the Tabula Sapiens dataset of human organs19 (Fig. In previous work, we described widespread immune gene activity in nonhematopoietic, structural cells of the mouse34, prompting us to explore this phenomenon in a large multi-organ human scRNA-seq dataset. We, thus, entered ‘structural cells with immune functions' into the CellWhisperer chat box and obtained the corresponding CellWhisperer score as a color-coded overlay to the UMAP visualization of the Tabula Sapiens dataset (Fig. Among the cells that scored highly for this query were endothelial and epithelial cells, fibroblasts and pericytes (Fig. 4b), which are all known or suspected to have important immune-regulatory roles35,36,37. To investigate these cells in more detail, we sequentially selected cell clusters with high CellWhisperer scores (by drawing a circle around the cells of interest) and prompted CellWhisperer by entering ‘Describe these cells in detail' into the chat box (Fig. The resulting descriptions contained information about cell types, organs and developmental stages and, less frequently, details about potential sample donors (such as male or female), highly expressed genes (such as genes encoding collagens and matrix metalloproteinases in fibroblasts), biological functions (such as stress response) and other annotations. We found that the generated descriptions frequently referred to potential immune functions of the selected cells, consistent with our initial search query. To obtain additional information about these cells, we interactively selected one of the cell clusters and asked two follow-up questions: ‘What is the potential relevance of these immune functions?' and ‘How can the genes and pathways that are upregulated in these cells mechanistically contribute to these immune functions?'. This resulted in a coherent conversation with CellWhisperer, providing further characterization with highlighted genes and biological functions that are relevant in the selected cells (Fig. As a plausibility check, we confirmed the expression of those genes by projecting them on the UMAP (Fig. Lastly, we benchmarked the CellWhisperer chat model using the perplexity metric38, which is a common evaluation criterion for LLMs. We assessed how well each question-answer pair fits with the matched transcriptome in two test sets of biologically meaningful conversations (Methods). In our Evaluation Conversations dataset with 200 question-answer pairs, we observed a 90% preference for matched over unmatched transcriptomes (Extended Data Fig. 4a), which confirms that our LLM meaningfully interpreted the transcriptome embedding for its response generation. Furthermore, in the Cell Type Conversations dataset, we found that most cell type labels showed a preferential association with their matched transcriptomes (Extended Data Fig. CellWhisperer achieved best results (lowest perplexity values), even on the out-of-distribution Cell Type Conversations dataset, further supporting that our chat model effectively incorporates the CellWhisperer transcriptome embeddings. We also assessed whether the CellWhisperer chat model may benefit from explicitly providing a list of highly expressed genes as part of the prompt (as commonly done when analyzing transcriptomes with text-only LLMs39,40), in addition to the transcriptome embedding. We observed a mild beneficial effect (Extended Data Fig. 4c) and implemented this hybrid approach in the CellWhisperer web tool. In summary, the integration of a CellWhisperer chat box in the CELLxGENE Explorer software provides user-friendly access to CellWhisperer's AI features and demonstrates the complementarity of visual inspection and natural-language chats for the interactive exploration of scRNA-seq data. To analyze user-provided transcriptome datasets with CellWhisperer, we developed a data-processing pipeline that computes CellWhisperer embeddings and annotations on the basis of the read count matrices from bulk RNA-seq or scRNA-seq (details are provided in the source code repository: https://github.com/epigen/cellwhisperer). The processed data are stored in a single file for dynamic loading into a user-hosted instance of CellWhisperer, while also facilitating reproducibility and sharing of CellWhisperer analyses. Here, we describe a typical CellWhisperer data analysis, investigating stem and progenitor cells in human colon and their response to inflammation (Fig. 5a–f); and we compare it to conventional bioinformatics analysis (Fig. Our analyses are based on scRNA-seq data of pathogenic and adjacent normal biopsies of persons with inflammatory bowel disease and healthy controls41. a, Import and exploration of user-provided scRNA-seq data in CellWhisperer. b, UMAP of the CellWhisperer transcriptome embeddings for the imported Colonic Epithelium dataset41 comprising scRNA-seq profiles of inflamed and noninflamed tissue biopsies of individuals with inflammatory bowel disease and healthy individuals. Cluster labels were generated by CellWhisperer and clusters were repositioned for compact visualization (interactive version: https://cellwhisperer.bocklab.org/colonic_epithelium). c, Zoomed-in view of the cluster labeled ‘Cycling ileal epithelial precursor cells', colored by CellWhisperer scores for the free-text query: ‘Show me stem cells'. e, Expression levels of the LGR5 gene mentioned in the CellWhisperer response (in d), plotted for the cell cluster from c. f, Histogram of CellWhisperer scores (query from c) for cells derived from inflamed versus noninflamed tissue. g, Outline of a conventional bioinformatics analysis that produces similar results as the interactive CellWhisperer analysis (a–f). h, UMAPs before and after batch effect correction using scVI. i, Cell type annotation using CellTypist with cluster-level majority voting. j, Identification of a cell subset labeled ‘Stem cells' using CellTypist without cluster-level majority voting, plotted on top of the UMAP from i. k, Differentially expressed genes between putative stem cells (from j) and all other cells, ranked by log2-transformed fold change and colored by statistical significance (two-sided Wilcoxon test threshold: 0.0001) with a log2-transformed fold change of at least 1 (gray line). l, Differential expression of a generic stemness gene signature among the putative stem cells (from j) for cells from inflamed versus noninflamed tissue. Violin plots are shown, with inner box plots corresponding to the interquartile range and whiskers extending to the farthest data point within 1.5 times the interquartile range. The cluster labels generated by CellWhisperer (Fig. 5a) provide an initial overview of the dataset (Fig. Among the ‘Cycling ileal epithelial precursor cells', we searched for cells with stem cell characteristics using the CellWhisperer query ‘Show me stem cells' and identified a subset of cells within this cluster that scored highly for this query (Fig. Further investigation of these putative stem cells in a follow-up conversation with CellWhisperer (Fig. 5d) suggested that this cell cluster includes LGR5-expressing epithelial stem cells, which constitute well-established stem cells of the gut42. 5e) was highly correlated with the CellWhisperer score for the ‘Show me stem cells' query (Fig. We further compared the prevalence of the CellWhisperer-annotated epithelial stem cells between inflamed and noninflamed colon samples and we observed higher CellWhisperer scores for the ‘stem cells' query among the noninflamed samples (Fig. These results suggest that chronic gut inflammation in persons with inflammatory bowel disease has a negative effect on LGR5-expressing epithelial stem cells, matching the conclusions of the study from which the dataset was obtained41 and previous in vitro experiments43. Importantly, these analyses were performed swiftly and interactively with CellWhisperer. 5b–f) were taken from the web tool as screenshots (https://cellwhisperer.bocklab.org/colonic_epithelium). For comparison, we sought to reproduce these results with a conventional bioinformatics analysis using custom Python code (Fig. We downloaded and preprocessed the gene expression profiles from GEO and visualized them as a UMAP (Fig. We observed substantial batch effects (which was less of an issue in the CellWhisperer analysis because the embedding model intrinsically adjusts for batch effects, as illustrated in Fig. 2e); hence, we corrected for batch effects using the scVI method44 (Fig. Next, we performed cell type annotation using the CellTypist software tool45. With CellTypist's recommended parameters, no cell cluster was annotated as stem cells (Fig. These cells were characterized by high levels of LGR5 expression (Fig. 5k), confirming that these are indeed epithelial stem cells. Lastly, we calculated a general ‘stemness score' on the basis of a previously reported gene set46 and observed higher values in inflamed than in noninflamed colon samples (Fig. This conventional bioinformatics analysis reproduced the conclusions of the interactive CellWhisperer analysis but it was much more complex and time-consuming. Overall, it took 400 lines of custom Python code, calls to five specialized software tools and the expertise of an experienced bioinformatician to plan and conduct the analysis. In summary, CellWhisperer offers a rapid initial assessment of scRNA-seq datasets and an interactive approach to data exploration and hypothesis generation. In contrast, conventional bioinformatics analysis provides more fine-grained control and better traceability. Given the complementary strengths of these two approaches, we envision that chat-based analysis will guide rather than replace sophisticated code-based analyses. Transcriptome profiling is widely used for characterizing biological states of cells and tissues, but data analysis and biological interpretation remain challenging. Here, we provide a proof of concept for scRNA-seq data exploration with natural language, using a multimodal AI model that combines transcriptome profiles with an understanding of biological text and an LLM-powered chat interface for interactive investigation of cell states. Our performance evaluations and usage examples illustrate how multimodal models of transcriptomes and text facilitate exploratory analysis of biomedical data. CellWhisperer is most useful for exploratory analysis and for generating ideas and hypotheses in the early stages of data analysis, while key results should be reconfirmed with conventional bioinformatics approaches. We expect natural language to evolve into a widely used channel for interactive analysis of biomedical data, complementing visual data inspection and programming-based data analysis. We also envision natural language as a human-interpretable integration layer through which AI models of different scales (for example, FMs of molecules, cells, organs and individuals) will share and integrate their perspectives on a shared question, thereby facilitating multiscale and multimodal data analysis. Methods such as CellWhisperer make data exploration more fluid, as users are unburdened by complex syntax and can interrogate biological knowledge within the interactive analysis. It also reduces barriers to entry, for example for biologists with no programming experience and strong preference for human language over computer code. Moreover, by connecting the chat functionality to voice recognition, it will be possible to interact verbally with the AI,for example in the context of virtual reality data analysis software or for researchers with vision impairment. Given the multi-language capabilities of many LLMs, it is technically feasible to support languages other than English. Data analysis in natural language may thus contribute to making bioinformatics more accessible, user-friendly and efficient. CellWhisperer builds on recent advances in AI methodology. First, to establish our coherently annotated training dataset comprising a million bulk and pseudo-bulk transcriptomes, we used general-purpose LLMs for AI-assisted data curation of community-scale data repositories. Second, CellWhisperer uses powerful modality-specific embedding models to process transcriptomes (with Geneformer) and text (with BioBERT). Third, we adapted elements of the CLIP9 and LiT47 architectures for learning multimodal embeddings of transcriptomes and their textual annotations, which constitutes the foundation of the CellWhisperer model and software tool. Fourth, inspired by image-recognizing chat bots and other multimodal applications of LLMs48, we modified a general-purpose LLM to support chat-based analysis of scRNA-seq data by fine-tuning with 106,610 AI-generated transcriptome-centric conversations about cells and biological processes. Fifth, CellWhisperer follows the paradigm of FMs in the sense that it was trained once on large datasets covering a broad spectrum of biology and it handles diverse queries across biological domains without further training. First, like other LLMs, CellWhisperer does not understand the user questions and its own responses in a human sense; rather, it has learned to continue the conversation on the basis of large amounts of training data on how transcriptome-centric question answering usually unfolds. We thus consider CellWhisperer a tool for exploratory analysis that should not be trusted blindly and without validation. Second, CellWhisperer relies on domain-specific models for its embedding of transcriptomes and text and on an LLM for text generation, thus inheriting their current limitations. To let CellWhisperer profit from progress with these models, we implemented a modular software architecture that makes it easy to swap the underlying models. Third, the CellWhisperer chat model occasionally ‘hallucinates', most frequently by providing overly specific information about potential sample origins (such as ‘T cells from an 85-year-old male'). This behavior likely reflects the high abundance of such text in our training data, which could be addressed by fine-tuning with human feedback and/or data curation to remove spurious information from the training data. Fourth, CellWhisperer can only be as good as the available training data; hence, areas of biology that are not well represented in public databases are unlikely to be modeled well by CellWhisperer or similar models. Considering concerns about the risks of modern AI49, we concluded that CellWhisperer can be considered of low risk, enabling us to make all aspects of the method and data openly accessible to the general public. We identified as the most relevant risk of CellWhisperer that incorrect answers may be left unchecked, thereby leading to wasted resources for validation experiments or, worse, the uncritical incorporation of falsehoods into scientific research. To mitigate this risk, we designate CellWhisperer as a tool for exploratory data analysis that should be used with a critical mind and we emphasize that key results should be validated with alternative methods (as illustrated in Fig. In contrast, we did not identify any particular risks to humans or to the environment. Given the complexity and research-centric character of scRNA-seq profiling, it is highly unlikely that CellWhisperer results will be uncritically relied upon in clinical diagnostics and thereby harm persons. It has also been discussed whether AI tools facilitate the development of biological threats and bioweapons50. Given that CellWhisperer does not incorporate any biological data or knowledge that is not already in the public domain and does not provide any dedicated functionality for the design of chemicals, viruses or cells, we consider it highly unlikely that CellWhisperer could constitute a meaningful contribution to the toolbox of adversarial actors. Since the initial description of CellWhisperer in a conference paper51 and bioRxiv preprint52, several methods have been released that share CellWhisperer's ambition of making text-based approaches broadly useful for scRNA-seq data analysis. Most notably, LangCell53 uses a transcriptome–text contrastive learning approach similar to the CellWhisperer embedding model, C2S-Scale54 repurposes LLMs for cell-level interpretation by fine-tuning them on top expressed gene lists and BioDiscoveryAgent55 and BioChatter56 describe LLM-based agentic workflows for transcriptome data analysis. While these studies underline the general interest in chat-based transcriptome analysis, several key elements continue to be unique to CellWhisperer. First, CellWhisperer enables the annotation of millions of single cells using a computationally efficient multimodal embedding model (in contrast, Cell2Sentence54,57 runs an expensive LLM for each single cell) and supports conversations about interactively selected cells (LangCell implements CLIP-style embedding but lacks the ability to chat about cells). Second, CellWhisperer was trained on a massive dataset from GEO and CELLxGENE Census curated with semantic LLM processing, covering diverse human biology from over 20,000 individual studies. In contrast, most related studies exclusively relied on CELLxGENE Census (which only covers hundreds of studies and about 1,000 different samples) and did not incorporate detailed textual metadata beyond aggregating predefined columns. Third, CellWhisperer implements a directly usable workflow and a web-based user interface that showcase a novel analysis paradigm for scRNA-seq analysis, with integration into the widely used CELLxGENE Explorer. In conclusion, CellWhisperer establishes a user-friendly approach for exploring scRNA-seq data, driven by chat-based analysis with natural language. Our method uses AI models to emulate data-centric conversations between biologists and bioinformaticians. We anticipate that natural language will become a broadly useful channel for biological data analysis and it constitutes a key building block of future AI-based bioinformatics research assistants. To establish a large training dataset of transcriptomes and their matched textual annotations, we processed two community-scale repositories: GEO10,11 and CELLxGENE Census (version 2023-12-15)12. From GEO we obtained RNA-seq count matrices of 722,425 human transcriptomes that were uniformly processed by the ARCHS4 project (version 2.2, 30 May 2023)16. We removed 7,049 samples with fewer than 250 expressed genes and 9,946 samples that overlapped with our Human Diseases dataset (described below), resulting in 705,430 transcriptomes. For each transcriptome, we obtained the associated metadata using the Entrez API with either the sample's experiment accession, Bio-Sample accession or GEO accession (in this order of priority) and we removed binary data and special characters using the unidecode package. We included metadata fields describing study-level descriptions such as series design, growth protocol and series summary, as well as sample-level fields including sample title, treatment, and other fields that varied across studies. From CELLxGENE Census we obtained scRNA-seq count matrices of 257 studies that were conducted on human samples using one of four assays with comparable data types (10x Genomics, Seq-Well, Drop-seq and CEL-seq2). We excluded cells with fewer than 100 expressed genes, resulting in a total of 19,663,838 scRNA-seq profiles. Within each sample, we grouped cells on the basis of their cell-level metadata, only considering metadata fields with fewer than 500 distinct values across all cells in the corresponding study, and we calculated pseudo-bulk transcriptomes by taking the mean of the scRNA-seq count values across all cells with identical metadata. Each of these 376,983 pseudo-bulk transcriptomes was linked to its cell-level metadata such as the cell type and to study-level metadata such as a study title and study abstract. For each sample (from GEO) or pseudo-bulk transcriptome (from CELLxGENE Census), we generated a concise natural-language summary from the metadata using Mixtral 8x7b (Q5_K_M quantized version)58, using the llama.cpp Python bindings with a sampling temperature of 0.2, nucleus sampling (top_p) of 0.9 and top probability sampling (top_k) of 50. This LLM-based generation of concise textual annotations was guided by a prompt that we engineered on the basis of established practices such as pre-action reasoning59, role playing60 and few-shot learning61 with a manually curated set of examples. The prompt and illustrative examples of generated textual annotations are shown in Supplementary Note 2. Data processing was performed on compute nodes with eight A100 80-GB GPUs. To enable transcriptome data analysis using natural language, we pursued a multimodal contrastive learning approach, with a neural network architecture that integrates matched pairs of transcriptomes and text into the same embedding space. Specifically, we adapted the CLIP method9, originally developed for joint multimodal embedding of images and text, and implemented it using pytorch62 with the lightning63 and the transformers64 libraries. The outputs of these two models are transformed into two 2,048-dimensional vectors using separate adapter modules, each consisting of two learnable linear layers connected by a rectified linear unit nonlinearity (ReLU) and followed by batch layer normalization. The Geneformer model for transcriptome embedding uses 12 transformer encoder layers to process transcriptomes as ‘sentences of genes' ranked by their expression; it was trained on ~30 million scRNA-seq profiles17. We also tested alternative models (scGPT24 and UCE25 for transcriptome embedding; BioGPT65 for text embedding), which led to similar results (Supplementary Note 3). We trained the CellWhisperer embedding model on the 1,082,413 matched pairs of transcriptomes and textual annotations that we curated from GEO and CELLxGENE Census. Specifically, the transcriptomes were sorted by gene expression levels and the top 2,048 most highly expressed genes were tokenized with a dictionary of human gene symbols17. The textual annotations were tokenized using WordPiece18,66 and trimmed to a maximum of 128 tokens for training efficiency (the vast majority of textual annotations were shorter and, thus, remained untrimmed). We trained the multimodal embedding model with a mini-batch size of 512 and InfoNCE-based loss, which maximizes the cosine similarity between matched pairs of transcriptomes and textual annotations while minimizing the cosine similarity between all other (unmatched) pairs in a given training batch. Training was scheduled for 16 epochs at a maximum learning rate of 0.00001. For the first 3% of all training steps, we froze the Geneformer and BioBERT models to only train the embedding adapters and we linearly increased the learning rate from 0 to its maximum value (warmup). We then unfroze the BioBERT model and continued training with a second learning rate warmup for an additional 3% of the total number of training steps, followed by a learning rate cosine schedule over the remaining 94% of steps of the 16 epochs. The outputs of the consistently frozen Geneformer model were cached to decrease computational complexity during training. Optimal hyperparameters, such as the maximum learning rate, were determined by stochastic grid search. As the performance metric for this optimization procedure, we tested the model's ability to retrieve the correct textual annotation for a given transcriptome in our Human Diseases dataset. We used a deduplicated version of this dataset to increase the robustness of retrieval scoring, thereby reducing the impact of data points with very similar or identical textual annotations. We also used this metric to control for overfitting during model training. The corresponding validation scores of our final model are shown in Extended Data Fig. A full training run (16 epochs) was completed in less than 24 h on an A100 GPU. An ablation study providing a technical evaluation of the final model is described in Supplementary Note 3. To assess CellWhisperer's performance and to demonstrate its functionality, we prepared the following bulk RNA-seq and scRNA-seq datasets and made sure to exclude them from all training data. We obtained 14,112 disease-annotated tissue samples from GEO, by querying the MetaSRA database for the terms ‘primary tissue' and ‘disease state', followed by manual curation based on metadata obtained from SRA, GEO and PubMed. To emulate a realistic application scenario with differences in the initial data processing, we processed these data with a different bioinformatics pipeline and a different LLM than what was used for the training dataset. Because this dataset contains many samples with identical or highly similar textual annotations, we also derived a Human Diseases (deduplicated) dataset for use in retrieval scoring. To that end, we processed all 14,112 textual annotations with BioBERT, performed hierarchical clustering on the embedding vectors (metric: cosine, linkage: average), retained the top 100 clusters and selected the transcriptome that was closest to the cluster center for each cluster, resulting in a total of 100 deduplicated transcriptomes with their nonredundant textual annotations. From the Tabula Sapiens atlas of scRNA-seq profiles19, we obtained transcriptomes for 483,152 single cells across 15 individuals, 24 organs and 177 annotated cell types, with their cell_ontology_class annotation as cell type annotations (we standardized spelling and capitalization). Because of many infrequent cell types in the Tabula Sapiens dataset, we also derived a Tabula Sapiens (20 common cell types) dataset by retaining only the 20 most common cell types in liver, lung and blood (184,450 single cells). From the RNA-seq profiles and manually curated cell types provided by the ImmGen consortium20 (GSE227743), we established a dataset comprising 42 bulk transcriptome profiles across five human immune cell types. From the Asian Immune Diversity Atlas21, we obtained scRNA-seq profiles for peripheral blood mononuclear cells from 619 healthy individuals spanning seven population groups in five countries across Asia. We randomly selected up to 1,000 cells per annotated cell type (including all cells if fewer than 1,000 were available), resulting in a total of 7,842 cells across nine cell types. We obtained scRNA-seq data of human embryonic development, spanning 3–38 days after fertilization (European Nucleotide Archive, ArrayExpress and GEO accessions: PRJEB30442 (ref. Raw sequencing data were uniformly processed and aligned to the human genome (assembly GRCh38) using Cell Ranger70 (E-MTAB-8060, GSE232861 and GSE155121) or using the rnaseq pipeline67,69 (all other datasets). The count matrices were corrected for batch effects with scANVI71, resulting in an integrated dataset of 95,092 scRNA-seq profiles. This dataset was previously assembled from individual studies that used different transcriptome profiling technologies and was used for benchmarking of methods for single-cell data processing and dataset integration22. To showcase CellWhisperer's analysis of user-provided datasets, we obtained scRNA-seq data for epithelial and other cells from the colon, starting from a normalized read count matrix retrieved from GEO (GSE116222). The dataset consists of 11,175 cells from three healthy individuals and three patients with inflammatory bowel disease. For these patients, samples were taken from inflamed and noninflamed regions. We evaluated the trained multimodal embedding model in four complementary ways. First, we assessed CellWhisperer's capability to predict cell characteristics such as cell types, tissues, organs and diseases in a zero-shot manner, by comparing CellWhisperer's transcriptome embeddings to the text embeddings of the corresponding metadata-provided cell characteristic from the evaluation datasets. To that end, we embedded the transcriptomes and the corresponding cell characteristics (as natural-language statements, for example in the form ‘A sample of <cell type> from a healthy individual') for each of the evaluation datasets. For every combination of a transcriptome and a text label, we quantified the agreement using the dot product, which constitutes the ‘CellWhisperer score'. We softmax-transformed the resulting scores for each given transcriptome to obtain probabilities across all labels in the dataset, and we calculated the mean of AUROC scores for these labels as a metric for the model's zero-shot prediction performance. Second, we evaluated how well the CellWhisperer embeddings capture biological (rather than technical) differences between cells and compared this to scFMs (Geneformer, scGPT and UCE). To that end, we embedded the Tabula Sapiens transcriptomes using either the CellWhisperer embedding model or any of the three scFMs and then compared batch effect correction and cell type clustering for the embeddings following a previously described workflow72. For cell type clustering, we used the average bio score24, which is the arithmetic mean of three metrics: average silhouette width22, normalized mutual information and adjusted rank index. For batch effect correction, we used a variant of average silhouette width as implemented in the silhouette_batch function22. Third, we leveraged the broad catalog of biological phenomena provided by gene set libraries to assess which aspects of molecular biology were implicitly learnt by the CellWhisperer embedding model. To that end, we obtained 8,812 gene sets from gene set libraries of cell types19,73,74, diseases75 and Gene Ontology (GO) terms76 and we performed Gene Set Variation Analysis (GSVA) using the GSVA package77 with the ssgsea enrichment function78. GSVA supports gene set enrichment analysis based on read count matrices, providing quantitative results for individual samples against a background of unrelated samples. We ran GSVA across all 8,812 gene sets on the Human Diseases dataset, resulting in an 8,812-dimensional vector of gene set enrichments for each transcriptome. For the same transcriptomes, we also obtained CellWhisperer scores by embedding the names of the gene sets (such as ‘colorectal cancer' from the OMIM_Extended disease library or ‘response to type I interferon' corresponding to GO:0071357 from GO_Biological_Process_2023). Fourth, we evaluated complex queries and their variations using the Human Development dataset. To that end, we prepared queries for four key embryonic developmental stages (zygote, blastula, gastrula and organogenesis) and compared CellWhisperer scores for variations of these queries. We condensed their annotations using GPT-4o with the following prompt: ‘Please, summarize the following sentences in just two sentences (max 500 characters)'. We visualized CellWhisperer scores for the four queries in a time-resolved manner (Fig. 2f), we generated five variants per query using GPT-4o with the following prompt: ‘Rewrite the provided text in five different variants. We benchmarked CellWhisperer's performance in zero-shot cell type prediction against a widely used marker-based method and against scFMs that we fine-tuned for cell type prediction. Marker-based prediction was performed with CellAssign23 and the CellMarker 2.0 cell type marker database79, enabling reference-free cell type prediction akin to CellWhisperer's zero-shot predictions. To resolve cell type naming mismatches between the CellMarker 2.0 database and our evaluation datasets, we mapped them using GPT-4o with the following prompt applied to each cell type in each evaluation dataset: ‘Assign the cell type <cell_type> to one of the following candidates: <candidates>. <newline> Only print the name of a single cell type, nothing else'. We only included cell types from the marker databases that matched the tissue(s) of the respective evaluation dataset and we excluded 66 marker genes from the CellMarker 2.0 database that were originally derived based on scRNA-seq data included in our Pancreas evaluation dataset. For scFM-based cell type predictions, we fine-tuned three scFMs (Geneformer, scGPT and UCE) using 376,983 pseudo-bulk transcriptomes from CELLxGENE Census. To optimize performance, we tested three configurations for each scFM: (1) freeze the scFM and train only its classification head; (2) unfreeze the scFM and fine-tune the entire model; and (3) augment the pseudo-bulk data with single-cell data (at a 1:4 ratio) during training. <newline> Only print the name of a single cell type, nothing else, and don't just repeat the query cell type. Make sure to return one of the candidates'. We allowed multiple classifier outputs to be assigned to the same cell type class and summed up their predicted probabilities to avoid unfair penalization of the scFM classifiers, which predict a larger number of cell types. For comparison, we obtained a list of marker genes that was based on an atlas of fetal gene expression32. To facilitate a fair comparison to these marker genes, we selected matching numbers of CellWhisperer-identified organ marker genes (using a flexible log2 fold-change threshold). We then compared the number of papers in PubMed that co-mentioned each gene name and the corresponding organ in titles, keywords or abstracts using the following PubMed query (run with Biopython's entrez.esearch function): ‘<organ> AND <gene>'. Gene set enrichment was assessed using gseapy's enrichr method80 for the gene set libraries GO_Biological_Process_2023, GO_Molecular_Function_2023, GO_Cellular_Component_2023 and KEGG_2021_Human. Spatial expression patterns for specific genes and lineage gene sets were derived from a corresponding Carnegie stage 8 embryo dataset33 that is accessible online (https://cs8.3dembryo.com). To enable natural-language chats with CellWhisperer, we fine-tuned the Mistral 7B LLM with conversations about transcriptomes and cell states. We generated 106,610 such conversations for transcriptomes from our training dataset of 1,082,413 GEO and CELLxGENE Census data points. To mitigate abundance biases in these repositories, we took a weighted subsample with sampling probabilities that were inversely proportional to the local point density calculated with densMAP81, such that transcriptome–text pairs in lowly covered regions were preferentially picked. For each of these subsampled transcriptome–text pairs, we generated a chat-like conversation using one of two LLMs (GPT-4 or Mixtral) on the basis of (1) the transcriptome's top 50 most highly expressed genes (based on normalized expression across all transcriptomes in the dataset to improve the representation of lowly expressed but important genes such as certain transcription factors); (2) the top 50 GSVA-derived gene sets; and (3) the transcriptome's textual annotation. We prepared conversations in four different ways, resulting in ‘simple', ‘detailed', ‘complex' and ‘conversational' chats. Simple chats were generated for 10,000 transcriptomes by using a generic question (randomly selected from a list of ten manually prepared candidates: ‘What does the sample represent? Detailed chats were generated for 10,000 transcriptomes using an LLM (GPT-4 through the OpenAI API) with zero-shot prompting and the same questions as in the simple chats. GPT-4 produced much more extensive answers compared to the transcriptome's textual annotations used in the simple chats. Complex chats were generated for 5,000 transcriptomes using an LLM (GPT-4 through the OpenAI API) with a few-shot prompt using pre-action reasoning59 to produce more profound question-answer pairs. Conversational chats for 81,610 transcriptomes were generated using an LLM (Mixtral 8x7b) with a few-shot prompt that instructed a natural conversation with multiple questions and answers between a researcher and an AI assistant about the biological state of the corresponding sample or cell cluster. We initialized a two-layer adapter module that transforms the 2,048-dimensional CellWhisperer multimodal embedding for a bulk or single-cell transcriptome into eight 4,096-dimensional embeddings, corresponding to eight token embeddings in the Mistral 7B LLM13, which is the basis for the CellWhisperer chat model. In a first training step, we kept the LLM frozen and trained the adapter layers with supervision on an extended version of the simple chats that included our entire training dataset (1,082,213 question-answer pairs) for one epoch. LLM fine-tuning was performed on four A100 80-GB GPUs with a total runtime of 3 hours. The generated conversation datasets and model checkpoints are available for download on the project website. To evaluate the CellWhisperer chat model, we created two question-answer conversation datasets and assessed CellWhisperer's propensity for the correct answers, quantified on the basis of its predicted output token probabilities using the perplexity metric. In other words, it measures the degree of surprise for the model when confronted with an answer to a given pair of a question and the corresponding transcriptome embedding. The first dataset (termed Evaluation Conversations) provides a general evaluation of CellWhisperer's chat functionality by randomly selecting 200 of the 106,610 training conversations, half based on GEO (bulk transcriptomes) and half based on CELLxGENE Census (pseudo-bulk transcriptomes from scRNA-seq data), with proportional representation of simple, complex and conversational chats. These conversations were trimmed to the first question-answer pairs, text that did not refer to biological insights was manually removed, and the data points associated with these conversations were excluded from CellWhisperer embedding and chat model training. To assess how well CellWhisperer took the transcriptome data into account when generating its chat response, we computed the perplexity for each test conversation not only with the matched transcriptome embedding but also with 30 mismatched transcriptome embeddings, randomly sampled from the dataset. For each of the 177 cell types in Tabula Sapiens, we sampled up to 100 pseudo-bulk transcriptomes, resulting in a total of 15,158 individual transcriptomes. Conversations in this dataset all contain the same question (‘Which cell type is this cell?') and use the annotated cell type label as a natural-language answer by prefixing them with the following text: ‘This cell is a …'. To evaluate the preference of a given transcriptome for its annotated cell type, we calculated the perplexity against two backgrounds: (1) by randomly swapping the transcriptome embeddings (30 times, as above) and (2) by randomly swapping the cell type answer texts (176 times, to all other possible cell types). For both datasets we compared the perplexity quantile scores of the CellWhisperer chat model with two text-only LLMs: Mistral 7B (without fine-tuning) and the much larger Llama 3.3 with 70 billion parameters, representing state-of-the-art LLMs (we could not compute the perplexity for closed models such as GPT-4 or Gemini as they do not provide output token probabilities). We provided the transcriptome context to these text-only LLMs through the following prepended prompt, containing the list of dataset-normalized top 50 genes: ‘USER: Respond to my request regarding a sample of cells characterized by its top expressed genes being <top 50 genes>. We integrated CellWhisperer into CELLxGENE Explorer for web-based analysis of scRNA-seq data15 (version 1.2.0; https://docs.cellxgene.cziscience.com). To that end, we added a chat box to the CELLxGENE Explorer user interface and implemented two API endpoints: (1) natural-language chat functionality to retrieve information about a selected group of cells and (2) search interface to obtain CellWhisperer scores for user-provided free-text queries, which are displayed as cell-level color maps. User requests that start with ‘show', ‘show me', ‘search' or ‘search for' are always routed to the second endpoint. Other requests are routed to the second endpoint only if the user has not selected any cells. The CellWhisperer chats inside CELLxGENE Explorer are primed with a user-hidden prompt (Supplementary Note 2) designed to reduce the prevalence of donor-specific metadata in the responses. This prompt also includes the top 50 expressed genes for the selected cells, as we observed better biological interpretability of responses and a slight performance improvement when providing the CellWhisperer chat model with these gene names in addition to the transcriptome embedding (Extended Data Fig. We host the CellWhisperer-augmented version of CELLxGENE Explorer as a web tool using docker/podman/docker-compose. Each dataset receives its dedicated server job and these jobs are jointly exposed through an nginx web server. CellWhisperer's embedding and chat capabilities are hosted through independent API server jobs, which are accessible online to enable running the software tool locally on computers without a GPU. User-provided datasets have to be prepared for CellWhisperer analysis using an automated pipeline (instructions are provided in the source code repository; https://github.com/epigen/cellwhisperer). Their interactive analysis requires a local installation of the CellWhisperer web tool, as it is currently not supported to upload user-provided datasets to our publicly accessible CellWhisperer instance (https://cellwhisperer.bocklab.org). The pipeline first processes all transcriptome measurements using the CellWhisperer embedding model, facilitating efficient CellWhisperer scoring. Next, a UMAP is calculated for the embeddings, followed by clustering with the Leiden algorithm and generation of a brief textual annotation for each cluster. Then, the CellWhisperer chat model processes the cluster-averaged transcriptome embeddings to generate detailed textual descriptions, which are subsequently condensed into brief textual annotations with an LLM such as GPT-4 (through the OpenAI API, used in this study) or Mixtral 8x7B (which can be run locally and is also supported by our source code). All prompts are provided in Supplementary Note 2. The user-provided dataset with all preprocessed elements is stored as a single h5ad object for use with the CellWhisperer web tool. This data-processing pipeline is run with a single shell command and was used to process the datasets for Figs. Data processing has a runtime on the order of minutes for typical datasets on a compute node with one A100 GPU or on the order of hours when relying on a standard laptop without a GPU. To compare the CellWhisperer analysis of the Colonic Epithelium dataset to a conventional bioinformatics analysis of the same data, we wrote custom Python code following established practices for scRNA-seq analysis. We retrieved the total-read-normalized and log1p-transformed scRNA-seq read count matrix from GEO (GSE116222) and used the inverse transform (exp1m) to obtain gene expression profiles that formed the basis for our analysis. We used scVI44 with sampleID as a batch variable to remove batch effects (parameters: 4,000 highly variable genes, n_layers = 2, n_latent = 30, gene_likelihood = ‘nb'). Next, we computed and plotted a UMAP based on scVI's latent space using scanpy82. We used CellTypist45 with the ‘Cells_Intestinal_Tract.pkl' model83 to automatically annotate cell types, first with majority voting (as recommended by the CellTypist instructions) and then without (which allowed us to identify transcriptomes that were annotated as stem cells). On the basis of these annotations, we determined differentially expressed genes for selected cell types in a one-versus-all manner using scanpy and the Wilcoxon rank sum test. Lastly, to obtain a measure of stemness of inflamed and noninflamed cells, we computed a gene module score based on a corresponding gene set46. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. CellWhisperer was trained on publicly available datasets obtained from GEO and CELLxGENE Census. All training data are available from their original sources. Full details are provided in the Methods and source code (https://github.com/epigen/cellwhisperer), which implements automatic data download. The model weights and LLM-curated datasets for training and model evaluation are available from the project website (https://cellwhisperer.bocklab.org). The source code underlying this project is available on GitHub (https://github.com/epigen/cellwhisperer), which includes the CellWhisperer model and dataset processing code and our adapted version of CELLxGENE Explorer with integrated CellWhisperer functionality. These components enable the analysis of user-supplied datasets for interactive analysis on Linux-based compute infrastructure by running a local version of the CellWhisperer web tool. We further provide automatic start-to-finish pipelines to reproduce all analyses described in this paper. Where we build on external codebases (Geneformer, LLaVA and CELLxGENE Explorer), we forked, adapted and integrated them as submodules in the CellWhisperer GitHub repository. In addition to the CellWhisperer source code, we provide the trained CellWhisperer model as a PyTorch Lightning checkpoint for download on the project website (https://cellwhisperer.bocklab.org). Detailed instructions on installing CellWhisperer and reproducing this study are provided in the GitHub repository. Quake, S. R. The cell as a bag of RNA. Stark, R. et al. RNA sequencing: the teenage years. Aldridge, S. Single cell transcriptomics comes of age. The Human Cell Atlas White Paper. Zappia, L. Over 1000 tools reveal trends in the single-cell RNA-seq analysis landscape. Language models for biological research: a primer. Szałata, A. et al. Transformers in single-cell omics: a review and new perspectives. Radford, A. et al. Learning transferable visual models from natural language supervision. 38th International Conference on Machine Learning (eds Meila, M. & Zhang, T.) (2021). Clough, E. et al. NCBI GEO: archive for gene expression and epigenomics data sets: 23-year update. Edgar, R. et al. Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. CZI Single-Cell Biology Program et al. CZ CELLxGENE Discover: a single-cell data platform for scalable exploration, analysis and modeling of aggregated data. 37th International Conference on NeurIPS (eds Oh, A. et al.) (2023). Megill, C. et al. cellxgene: a performant, scalable exploration platform for high dimensional sparse matrices. Massive mining of publicly available RNA-seq data from human and mouse. Transfer learning enables predictions in network biology. Lee, J. et al. BioBERT: a pre-trained biomedical language representation model for biomedical text mining. The Tabula Sapiens: a multiple-organ, single-cell transcriptomic atlas of humans. The Immunological Genome Project: networks of gene expression in immune cells. Kock, K. H. et al. Asian diversity in human immune cells. Luecken, M. D. et al. Benchmarking atlas-level data integration in single-cell genomics. Zhang, A. W. et al. Probabilistic cell-type assignment of single-cell RNA-seq for tumor microenvironment profiling. Cui, H. et al. scGPT: toward building a foundation model for single-cell multi-omics using generative AI. Petropoulos, S. et al. Single-cell RNA-seq reveals lineage and X chromosome dynamics in human preimplantation embryos. Molè, M. A. et al. A single cell characterisation of human embryogenesis identifies pluripotency transitions and putative anterior hypoblast centre. Integrated pseudotime analysis of human pre-implantation embryo single-cell transcriptomes reveals the dynamics of lineage specification. Tyser, R. C. V. et al. Single-cell transcriptomic characterization of a gastrulating human embryo. Modeling post-implantation stages of human development into early organogenesis with stem-cell-derived peri-gastruloids. Cao, J., et al. A human cell atlas of fetal gene expression. Xiao, Z. et al. 3D reconstruction of a gastrulating human embryo. Structural cells are key regulators of organ-specific immune responses. Amersfoort, J. et al. Immunomodulation by endothelial cells - partnering up with the immune system? Davidson, S. et al. Fibroblasts as immune regulators in infection, inflammation and cancer. Larsen, S. B. et al. Epithelial cells: liaisons of immunity. Rosenfeld, R. Two decades of statistical language modeling: where do we go from here? Crowley, G. et al. Benchmarking cell type annotation by large language models with AnnDictionary. Hou, W. Assessing GPT-4 for cell type annotation in single-cell RNA-seq analysis. Parikh, K. et al. Colonic epithelial cell diversity in health and inflammatory bowel disease. Clevers, H. The intestinal crypt, a prototype stem cell compartment. Long-term culture captures injury-repair cycles of colonic stem cells. Deep generative modeling for single-cell transcriptomics. Conde, D. et al. Cross-tissue immune cell analysis reveals tissue-specific features in humans. Malta, T. M. et al. Machine learning identifies stemness features associated with oncogenic dedifferentiation. Zhai, X. et al. LiT: zero-shot transfer with locked-image text tuning. 2022 IEEE/CVF Conference on Computer Vision and Pattern Recognition (ed. Yin, S., et al. A survey on multimodal large language models. Bengio, Y. et al. International Scientific Report on the Safety of Advanced AI (Department for Science, Innovation and Technology and AI Safety Institute, 2024). Urbina, F. et al. Dual use of artificial intelligence-powered drug discovery. Schaefer, M. et al. Joint embedding of transcriptomes and text enables interactive single-cell RNA-seq data exploration via natural language. ICLR 2024 Workshop on Machine Learning for Genomics Explorations (ed. Schaefer, M. et al. Multimodal learning of transcriptomes and text enables interactive single-cell RNA-seq data exploration with natural-language chats. Zhao, S. et al. LangCell: language-cell pre-training for cell identity understanding. 41st International Conference on Machine Learning (eds Salakhutdinov, R. et al.) (2024). Scaling large language models for next-generation single-cell analysis. Roohani, Y. et al. BioDiscoveryAgent: An AI Agent for Designing Genetic Perturbation Experiments. Lobentanzer, S. et al. A platform for the biomedical application of large language models. 41st International Conference on Machine Learning (eds Salakhutdinov, R. et al.) (2024). Jiang, A. Q, et al. Mixtral of experts. Yao, S. et al. ReAct: synergizing reasoning and acting in language models. In Proc.11th International Conference on Learning Representations (ICLR, 2023). Better zero-shot reasoning with role-play prompting. 2024 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies (Vol. 1: Long Papers) (eds Duh, K. et al.) (ACL, 2024). 34th International Conference on NeurIPS (eds Larochelle, H. et al.) (2020). Paszke, A. et al. Automatic differentiation in PyTorch. 31st Conference on NeurIPS Autodiff Workshop (NIPS, 2017). Wolf, T. et al. Transformers: State-of-the-Art Natural Language Processing. 2020 Conference on Empirical Methods in Natural Language Processing (eds Liu, Q. Luo, R., et al. BioGPT: generative pre-trained transformer for biomedical text generation and mining. Wu, Y. et al. Google's neural machine translation system: bridging the gap between human and machine translation. The nf-core framework for community-curated bioinformatics pipelines. Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Xu, C. et al. Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. Kedzierska, K. Z. et al. Zero-shot evaluation reveals limitations of single-cell foundation models. Franzén, O. et al. PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data. Integrated analysis of multimodal single-cell data. Amberger, J. S., Bocchini, C. A., Schiettecatte, F., Scott, A. F. & Hamosh, A. OMIM.org: Online Mendelian Inheritance in Man (OMIM®), an online catalog of human genes and genetic disorders. Ashburner, M. et al. Gene Ontology: tool for the unification of biology. Hänzelmann, S. et al. GSVA: gene set variation analysis for microarray and RNA-seq data. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Hu, C. et al. CellMarker 2.0: an updated database of manually curated cell markers in human/mouse and web tools based on scRNA-seq data. Chen, E. Y., et al. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. Narayan, A. et al. Assessing single-cell transcriptomic variability through density-preserving data visualization. Wolf, F. A. et al. SCANPY: large-scale single-cell gene expression data analysis. Cells of the human intestinal tract mapped across space and time. The CellWhisperer web tool is hosted at CeMM with infrastructure and maintenance supported by CeMM's IT Services team. is supported by a European Research Council (ERC) Consolidator Grant (101001971). This publication is part of the Human Cell Atlas (www.humancellatlas.org/publications). Open access funding provided by Medical University of Vienna. These authors contributed equally: Moritz Schaefer, Peter Peneder. Medical University of Vienna, Institute of Artificial Intelligence, Center for Medical Data Science (CEDAS), Vienna, Austria Moritz Schaefer, Jake Burton, Varun Sharma, Thomas Krausgruber & Christoph Bock Moritz Schaefer, Daniel Malzl, Salvo Danilo Lombardo, Mihaela Peycheva, Jake Burton, Varun Sharma, Thomas Krausgruber, Celine Sin, Jörg Menche & Christoph Bock St. Anna Children's Cancer Research Institute (CCRI), Vienna, Austria Peter Peneder, Anna Hakobyan & Eleni M. Tomazou Daniel Malzl, Salvo Danilo Lombardo, Varun Sharma, Celine Sin & Jörg Menche Department of Structural and Computational Biology, Center for Molecular Biology, University of Vienna, Vienna, Austria Daniel Malzl, Salvo Danilo Lombardo, Varun Sharma, Celine Sin & Jörg Menche Salvo Danilo Lombardo, Celine Sin & Jörg Menche Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The remaining authors declare no competing interests. Nature Biotechnology thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. b) Cross-modality retrieval performance of the CellWhisperer embedding model, evaluated on the deduplicated Human Diseases dataset for the trained model (left) and during training (right). c) Performance comparison of alternative architectures and hyperparameter choices in an ablation study of the CellWhisperer embedding model. Barplots show the mean performance for five metrics and three replicate models trained with different seeds. The values for each metric (shown as dots) were min-max normalized across models. The bar plots indicate means across all data points, with error bars corresponding to 95% confidence intervals across all 15 data points. The final CellWhisperer embedding model (marked in bold) was trained with batch size 512, no gradient accumulation, frozen Geneformer transcriptome model, InfoNCE loss, GEO and CELLxGENE Census datasets, density-based sample weighting and no data augmentation. JSD: Jensen-Shannon divergence information maximization loss. a) Cell type prediction performance across multiple datasets for two variants of the CellWhisperer embedding model and their matched scFMs fine-tuned for cell type prediction (left: scGPT; right: UCE; same panel format as in Fig. b) CellWhisperer evaluation based on gene sets, comparing CellWhisperer scores computed for gene set labels to the enrichment or depletion of the corresponding genes, calculated across the Human Diseases dataset. KS: Kolmogorov-Smirnoff, GSVA: gene set variation analysis. c) Correspondence (x-axis, measured by the signed KS statistic as in panel b) between the CellWhisperer score and the GSVA score, averaged across all gene sets in each of six gene set libraries (y-axis). Violin plots are shown with inner boxplots corresponding to the interquartile range and whiskers extending to the farthest data point within 1.5 times the interquartile range. Selected gene set labels are shown for illustration. e) Conceptual outline and results of the CellWhisperer performance evaluation based on batch effect correction (left) and cell type clustering (right), comparing three variants of the CellWhisperer embedding model (orange) to their matched scFMs. Scores were computed for the Tabula Sapiens dataset, assessing how well the technical variation was removed (batch integration score) and how well the biological differences between cell types were retained (cell type clustering score). f) Analysis of CellWhisperer embeddings and corresponding CellWhisperer scores for textual variations of four queries. Left: CellWhisperer queries applied to the Human Development dataset and overview of LLM-generated query variants. Center: densMAP visualization of CellWhisperer text embeddings for all query variants. Right: Pearson correlation of CellWhisperer scores on the Human Development dataset for pairs of all query variants. Row and column sorting of the heat map was determined by clustering of the correlation coefficients. a) Correlation between CellWhisperer scores (y-axis) and the expression of literature-reported organ marker genes (x-axis) for each organ across time points (dots). For both metrics, the mean across all cells at each time point was calculated and then standardized across all time points (as z-scores). Pearson correlation coefficients are reported together with their associated p-values (two-sided t-test for correlation significance). b) Number of papers per gene that co-mentioned the gene name and the organ name (as in Fig. 3c; results for heart are shown in both panels). Shared marker genes (light brown) indicate the overlap between CellWhisperer-identified (brown) and literature-reported (dark grey) organ marker genes, quantified by odds ratio and p-value (two-sided Fisher's exact test). A size-matched set of random genes is plotted for comparison (light grey). Violin plots are shown with inner boxplots corresponding to the interquartile range and whiskers extending to the farthest data point within 1.5 times the interquartile range. c) Number of papers per gene that co-mention the gene name and the organ name (as in Fig. 3d; results for heart are shown in both panels), stratified by gene expression enrichment in CellWhisperer-identified organ-specific cells (x-axis). P-values correspond to two-sided Mann-Whitney U tests comparing CellWhisperer-identified marker genes (rightmost violin with a log2 fold-change above 1.5) and non-marker genes (leftmost violin with log2 fold-change below zero). Genes with strongly enriched expression in CellWhisperer-identified organ-specific cells but no associated papers are marked with red boxes. d) Gene set enrichment analysis for CellWhisperer-identified heart marker genes. The combined score (x-axis; c = log(p) × z) integrates the Fisher's exact test p-value (two-sided) with the z-score of rank deviation to capture both statistical significance and effect size. e) Spatial gene expression in a Carnegie stage 8 (CS8) human embryo for two CellWhisperer-identified marker genes of the developing liver (bottom) and for two established liver marker genes (top). The notochord as a reference region is marked in orange, and gene expression is denoted by red points. a) Left: Conceptual outline of the perplexity-based evaluation of the CellWhisperer chat model across 200 question-answer pairs from the Evaluation Conversations dataset with matched versus mismatched transcriptome embeddings. Center: Perplexity values for five question-answer pairs with one matched and 30 mismatched embeddings. b) Perplexity-based performance evaluation of cell type prediction by the CellWhisperer chat model in the Cell Type Conversations dataset. Quantiles were computed by comparing the perplexity for the matched transcriptome against 30 mismatched transcriptomes (left) or for the matched cell type label against all 176 mismatched cell type responses (right) for each transcriptome. Bars indicate the mean across all data points; error bars correspond to the 95% confidence interval. Violin plots are shown with inner boxplots corresponding to the interquartile range and whiskers extending to the farthest data point within 1.5 times the interquartile range. c) Perplexity values for conversation responses using text-only LLMs (Mistral, Llama 3.3) and the CellWhisperer chat model. Violin plots are shown with inner boxplots corresponding to the interquartile range and whiskers extending to the farthest data point within 1.5 times the interquartile range. Performance evaluation of the CellWhisperer embedding model based on the prediction of cell characteristics. Performance evaluation of the CellWhisperer embedding model based on gene sets. CellWhisperer-identified marker genes of human organ development. Interactive single-cell RNA-seq data analysis with the CellWhisperer web tool. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Schaefer, M., Peneder, P., Malzl, D. et al. Multimodal learning enables chat-based exploration of single-cell data. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-65486-5'>Structural basis of cyclobutane pyrimidine dimer recognition by UV-DDB in the nucleosome</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-11-11 10:19:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In mammalian global genomic nucleotide excision repair, UV-DDB plays a central role in recognizing DNA lesions, such as 6-4 photoproducts and cyclobutane pyrimidine dimers, within chromatin. In the present study, we perform cryo-electron microscopy analyses coupled with chromatin-immunoprecipitation to reveal that the cellular UV-DDB binds to UV-damaged DNA lesions in a chromatin unit, the nucleosome, at a position approximately 20 base-pairs from the nucleosomal dyad in human cells. An alternative analysis of the in vitro reconstituted UV-DDB-cyclobutane pyrimidine dimer nucleosome structure demonstrates that the DDB2 subunit of UV-DDB specifically recognizes the cyclobutane pyrimidine dimer lesion at this position on the nucleosome. We also determine the structures of UV-DDB bound to DNA lesions at other positions in purified cellular human nucleosomes. These cellular and reconstituted UV-DDB-nucleosome complex structures provide important evidence for understanding the mechanism by which UV lesions in chromatin are recognized and repaired in mammalian cells. In the eukaryotic genome, 145-147 base-pair DNA segments are wrapped around histone octamers, consisting of two molecules each of the four types of histone proteins, H2A, H2B, H3, and H4, forming the basic unit called the nucleosome1,2. The chromatin structure consists of sequential nucleosomes and regulates various nuclear events, including the DNA damage response3. However, it is unclear how DNA binding proteins overcome this limitation for their activity4. The DNA damage response is an important mechanism to maintain genome integrity across generations. DNA damage generates chemical conformational changes, forming obstacles that block transcription and induce mutations to the genome during DNA replication. Ultraviolet (UV) light causes DNA damage and generates mainly two types of DNA lesions, the (6-4)photoproduct (6-4PP) and the cyclobutane pyrimidine dimer (CPD)5. Nucleotide excision repair (NER) is an important cellular system that eliminates a variety of DNA lesions. Defects in NER cause severe diseases, such as xeroderma pigmentosum (XP) and Cockayne syndrome (CS), which are characterized by sensitivity to sunlight, and XP is associated with a high frequency of skin cancer6,7. In mammalian GG-NER, it is important for XPC to recognize DNA lesions. Although XPC binds to 6-4PP efficiently and proceeds NER, the binding affinity of XPC against CPD is shown to be quite low in vitro8. Therefore, UV-DDB (DDB1-DDB2/XPE heterodimer complex) is thought to be responsible for recognizing CPD in the genome, and then handover the lesion to XPC. In this process, while XPC is monoubiquitinated, the DDB2 subunit is polyubiquitinated, dissociated from the damaged site by the p97/VCP segregase, and subsequently degraded by the 26S proteasome9,10,11,12,13. The XPC-RAD23B-centrin2 (XPC complex) bound to the DNA lesion then recruits other downstream factors to complete the entire NER process14,15. However, the access of NER factors to DNA lesions is limited by the chromatin structures in cells16,17. The mechanism by which UV-DDB gains access to DNA damage has been studied in vitro. DDB2/XPE is the DNA-binding subunit of UV-DDB and has high affinity to the major DNA lesions, 6-4PP and CPD18. In addition to bulky lesions, DDB2 can also bind to abasic sites in DNA18. UV-DDB reportedly functions as a damage sensor in the base excision repair (BER) pathway in chromatin19,20,21. Previous X-ray crystallographic analyses revealed the structures of UV-DDB-damaged DNA complexes, in which DDB2 binds to UV lesions using its positively charged surface22,23,24. The β-hairpin loop of DDB2 is responsible for stabilizing the interactions between the amino acid residues (F-Q-H) and the two orphan bases opposite a UV DNA lesion site. DDB2 binding to UV DNA lesions mainly occurs when the lesion is induced on the strand facing toward the solvent, and this orientation preference is also conserved in the recognition of nucleosomal 6-4PP and abasic sites25,26. In contrast, to repair the 6-4PP and abasic DNA lesions facing toward the histone surface in the nucleosome, DDB2 binding induces register shifting of the nucleosomal DNA to expose the DNA lesion toward the accessible surface26. However, the mechanism by which the UV-DDB complex recognizes CPD in nucleosomes remains enigmatic, although it is the major lesion caused by UV light. To gain insight into UV-DDB binding to UV DNA lesions in cells, in the present study, we performed chromatin-immunoprecipitation coupled cryo-electron microscopy (ChIP-CryoEM)27, and visualized UV-DDB binding to damaged nucleosomal DNA in cells. We also performed cryo-electron microscopy (cryo-EM) analyses with reconstituted and native human nucleosomes containing UV DNA lesions, as complexes with purified UV-DDB. These native and reconstituted structures revealed the mechanism by which UV-DDB recognizes the UV lesion in chromatin. To determine how UV-DDB recognizes DNA damage in cellular chromatin, we established a cell line (U2OS/FLAG-DDB2) in which FLAG-tagged DDB2, the DNA-binding subunit of UV-DDB, is stably produced in the DDB2-deficient cells (U2OS/DDB2 KO)28. To test the localization of FLAG-DDB2, we irradiated UV-C, which is reported to generate mainly CPD within the genome. Immunostaining showed that without UV damage, FLAG-DDB2 was evenly distributed in the nucleus (Fig. In contrast, upon local UV irradiation, FLAG-DDB2 was co-localized with CPD damage sites (Fig. These results suggested that, in cells, FLAG-DDB2 recognizes UV damage and binds to UV lesions in chromatin. FLAG-DDB2 was accumulated in chromatin (insoluble fraction) just after UV irradiation (0.5 h) and released after 1 h (Supplementary Fig. Therefore, UV-DDB may transiently associate with chromatin in a UV damage-dependent manner. a U2OS/KO/FLAG-DDB2 cells were immunostained by an anti-DDB2 antibody (green) or an anti-CPD antibody (red). DNA was stained by DAPI (cyan). b Workflow of the ChIP-CryoEM method. c Soluble (Input: lanes 2,3) after MNase treatment, Unbound (Unb. : lanes 4,5), and bound (Eluate: lanes 6,7) fractions after an incubation with anti-FLAG beads were subjected to SDS-PAGE and stained with Oriole (Bio-Rad). Non-treated cells or UV-irradiated cells are indicated as UV- (lanes 2,4,6) or UV+ (lanes 3,5,7). d Amounts of CPDs were measured by ELISA, using an anti-CPD antibody. Non-treated cells and UV-irradiated cells were used as control NCPs and damaged NCPs, respectively. The Y-axis represents the absorbance at 490 nm, normalized to the minimum value across all measurements, which was set to zero. The graph was generated using Prism 10 software, and all data are based on three technical replicates and are presented as mean ± s.d. A lysate was prepared from U2OS/FLAG-DDB2 cells after UV exposure. The resulting chromatin sample was crosslinked with formaldehyde and digested by sonication and micrococcal nuclease treatment. The cellular UV-DDB bound to chromatin was then immunopurified by agarose beads conjugated with the anti-FLAG M2 antibody (Fig. After UV-DDB immunopurification, we detected DDB1 and FLAG-DDB2 together with the four types of histones (Fig. These proteins were observed by the same chromatin isolation method without crosslinking and confirmed by CBB staining and a mass spectrometry analysis (Supplementary Fig. Our enzyme-linked immunosorbent assay (ELISA) and DNA purification assay confirmed that the eluted fraction contains about 100-150 base-pair DNA fragments bearing CPD lesions (Fig. It should be noted that our mass spectrometric analysis revealed that the pull-down fraction with FLAG-DDB2 contained the peptides from CUL4A, CUL4B, and RBX1, which belong to the ubiquitin-proteasome pathway (Supplementary Fig. However, their target, ubiquitinated DDB2 (Ub-DDB2), was not observed in the fraction. We surmised that the efficient ubiquitination of DDB2 may decrease its chromatin binding affinity, leading to the dissociation of Ub-DDB2 from the nucleosome. We conducted a ChIP-CryoEM single particle analysis with the immunopurified UV-DDB-nucleosome complexes27, and visualized the structure of UV-DDB bound to the cellular nucleosome (Fig. Since only 7339 particles of the UV-DDB-nucleosome complex were obtained, its resolution was limited to 12 Å, which precluded the identification of the amino acid residues (12 Å) (Fig. However, the 12 Å resolution could reveal the approximate binding position of UV-DDB on the nucleosomal DNA. a Cryo-EM map of cellular UV-DDB bound to nucleosomes from cells. The nucleosome, DDB1, and DDB2 are colored blue, yellow, and green, respectively. b The 3D structure model of the nucleosome (blue)(PDB ID: 3AFA) is superimposed on the cryo-EM maps. The dyad and each super helical location (SHL) are indicated. The binding site of DDB2 was predicted to be the SHL ± 2 position. In this complex, UV-DDB gained access to the nucleosomal DNA. Superimposition of a high-resolution 3D structure model of the nucleosome (PDB ID: 3AFA) on the cryo-EM map corresponding to the nucleosome suggested that the UV-DDB may bind to the superhelical location (SHL) ± 2 position, which is approximately 20 base-pairs away from the nucleosomal dyad (SHL 0) (Fig. The DDB1 and DDB2 subunits fit well in the cryo-EM map of the UV-DDB-nucleosome complex (Supplementary Fig. This structure of the cellular UV-DDB-nucleosome complex suggested that the UV-induced DNA damage at the SHL ± 2 position can be efficiently recognized by UV-DDB in cells. We next tested whether the UV-DDB binding to the SHL ± 2 position is a consequence of its coupling with the ATP hydrolysis-dependent nucleosome remodeling activity or its preferential interaction with the SHL ± 2 position. To do so, we repeated the ChIP-CryoEM analysis in the presence of ATPγS, which is a slowly hydrolyzable analog of ATP (often referred to as a non-hydrolyzable analog). Our pull-down assay revealed that, in the presence of ATPγS, only minor amounts of UV-DDB and histones were captured by agarose beads conjugated with the anti-FLAG M2 antibody (Supplementary Fig. In addition, the immunopurified UV-DDB-nucleosome complex was not observed in the presence of ATPγS (Supplementary Fig. These results suggest that the preferential UV-DDB binding to the nucleosomal SHL ± 2 position may be induced by its coupling with the ATP hydrolysis-dependent nucleosome remodeling. The UV-C-generated CPD is a major UV lesion in nucleosomes16,29,30, suggesting that the cellular UV-DDB-nucleosome complex observed by the ChIP-CryoEM method may contain the CPD as UV-induced DNA damage. However, the structure of UV-DDB bound to a nucleosome containing a CPD has not been reported so far. We prepared the nucleosome core particle (NCP) containing a CPD at a position 22-23 base-pairs from the nucleosomal dyad, corresponding to the SHL−2 position (NCPCPD) (Fig. The UV-DDB-NCPCPD complex was reconstituted in vitro and subjected to the cryo-EM single particle analysis. We then determined the cryo-EM structure of the UV-DDB-NCPCPD complex at 3.4 Å resolution (Fig. Interestingly, the UV-DDB-NCPCPD structure fits very well into the cryo-EM map of the cellular UV-DDB-NCP complex (Fig. Therefore, the cellular UV-DDB-NCP complex obtained from human cells may be a form in which UV-DDB binds to the UV-induced DNA lesion at the SHL ± 2 position. b The structural model of the UV-DDB-NCPCPD complex (PDB ID: 9J8W) was determined, corresponding to the cryo-EM map of UV-DDB-NCPCPD complex at 3.4 Å resolution. DDB1 (yellow), DDB2 (green), CPD (red), damaged DNA strand (light gray), and undamaged DNA strand (dark gray) are highlighted. DDB1 (PDB ID: 6R8Y) was superimposed on the cryo-EM map filtered by a 5 Å low-pass filter. c Structural models of the UV-DDB-NCPCPD architecture (PDB ID: 9J8W) and DDB1 (PDB ID: 6R8Y) were superimposed on cryo-EM maps of the cellular UV-DDB-NCP complex shown in Fig. Among these residues, the DDB2 K244E mutation has been found in XP-E patients31. The DDB2 K244E mutant is reportedly unable to bind to the CPD32. This fact suggested that the nucleosomal DNA binding near the CPD lesion by the positively charged DDB2 surface plays an important role in the NER pathway. In the CPD nucleosome complexed with UV-DDB, the bending angle of the nucleosomal DNA at the CPD site was 53°, while it was 49° for undamaged nucleosomal DNA (Fig. In contrast, the bending angle of non-nucleosomal DNA was reported to be ~45° at the CPD site upon DDB2 binding23. Therefore, UV-DDB induces DNA bending at the CPD site, even when the CPD site is already bent by the histones in the nucleosome. b Schematic representation of DDB2 binding to CPD in nucleosomes. Distances between CPD and DDB2 residues were measured by the Pymol software. Bending angle of DNA in the nucleosome at the SHL ± 2 position in the absence (d) or presence (e) of UV-DDB. DNA in the nucleosome without UV-DDB was obtained from the previously determined structural model (PDB ID: 3AFA). Bending angles were measured by generating helical axes of DNA with the 3DNA software53. We next tested whether UV-DDB binds to other damaged DNA positions in the nucleosome. To do so, we prepared native nucleosomes from Expi293 cells, which produce FLAG-tagged H2A (Fig. The cells were lysed, and the chromatin was digested by an MNase treatment. The resulting mononucleosomes were isolated by immunopurification using agarose beads conjugated with the anti-FLAG M2 antibody, and UV irradiated in vitro (UV-irradiated NCP) (Fig. We confirmed that the nucleosome prepared from the H2A-FLAG producing Expi293 cells contains histones H2A-FLAG, H2A, H2B, H3, and H4, as expected, and the histone composition was not affected by UV irradiation (Fig. Since the purified native nucleosomes contain various genomic DNA sequences, the UV damage is potentially introduced at various nucleosome positions. a Workflow of native nucleosome isolation from expi293 cells and production of native UV-irradiated nucleosomes. b Isolated nucleosomes from expi293 cells with or without UV irradiation (500 J/m2) were subjected to Native-PAGE or SDS-PAGE, and then stained with SYBR Gold (left panel) or Oriole (right panel). c Gel shift assay, performed using 40 nM of nucleosomes and UV-DDB (0–160 nM). All reactions were subjected to 6% Native-PAGE and DNA was stained by SYBR Gold. d Amounts of CPDs in nucleosomes with or without UV irradiation (500 J/m2) were measured by ELISA, using an anti-CPD antibody. The Y-axis represents the absorbance at 490 nm, normalized to the minimum value across all measurements, which was set to zero. The graph was generated using Prism 10 software, and all data are based on three technical replicates and are presented as mean ± s.d. We then tested UV-DDB binding to UV-damaged nucleosomes with native genomic DNA sequences. UV-DDB specifically bound to the UV-damaged nucleosomes and formed two distinct complexes, probably containing one and two UV-DDB complexes (Fig. These complexes were scarcely detected with the native nucleosomes without UV damage (Fig. Consistently, the UV-irradiated nucleosomes were confirmed to contain CPDs, although 6-4PP lesions may also have been present (Fig. Our cryo-EM single-particle analysis revealed that UV-DDB bound to multiple sites in the nucleosome, primarily at three nucleosomal DNA positions, SHL ± 2, SHL ± 3, and SHL ± 6. For improved resolution, the DDB1 subunit was masked out, enabling visualization of the DDB2 subunit in the nucleosomes (Fig. Superimposition of the UV-DDB and nucleosome structures in these SHL ± 2, SHL ± 3, and SHL ± 6 complexes suggested that the UV-DDB may bind to the positions 22-23 base-pairs (SHL ± 2), 32-33 base-pairs (SHL ± 3), and 63-64 base-pairs (SHL ± 6) from the nucleosomal dyad. The SHL ± 2 complex is similar to the cellular UV-DDB-nucleosome complex observed by the ChIP-CryoEM analysis (Fig. In the SHL ± 3 and SHL ± 6 complexes, the UV lesions may be exposed to the solvent for efficient recognition by UV-DDB (Fig. These results suggested that UV-DDB has the potential to recognize UV-induced DNA lesions at multiple positions in the nucleosome. The DNA lesion may relocate to the SHL ± 2 position, which is more accessible for the nucleosome-binding proteins, including UV-DDB. In this process, the DNA register shifting activity of UV-DDB may also contribute to lesion relocation26. a Cryo-EM maps of three conformations of UV-DDB bound to damaged nucleosomes. 3D structures of the nucleosome (dark gray) and DDB2 (green, orange or purple)(PDB ID 9J8W) were superimposed on cryo-EM maps with masked out DDB1. Damage locations and DDB2 binding sites of three conformations were predicted as the 22-23 base-pair (SHL ± 2)(green), the 32-33 base-pair (SHL ± 3)(orange), and the 63-64 base-pair (SHL ± 6)(purple). UV-DDB functions in the initial recognition step of UV-damaged DNA in cells10. A previous structural study reported that purified UV-DDB binds to the 6-4PP or abasic damage at the SHL ± 2 position of the reconstituted nucleosome26. In the present study, our ChIP-CryoEM technique revealed the structure of the cellular UV-DDB-nucleosome complex in human cells (Fig. Interestingly, in this complex, UV-DDB binds to the SHL ± 2 position of the nucleosome. The cryo-EM map of this cellular UV-DDB-nucleosome complex fits very well with that of the reconstituted UV-DDB-NCPCPD complex. These results indicate that UV-DDB binds to the UV-damaged nucleosome at the SHL ± 2 position in human cells. Importantly, our cryo-EM analysis of UV-DDB binding to the cellular UV-damaged nucleosomes revealed that UV-DDB also binds to the nucleosomal SHL ± 3 and SHL ± 6 positions, in addition to the SHL ± 2 position. Therefore, UV-DDB may recognize UV-induced DNA damage in the nucleosome when the damaged site is located on an accessible surface, possibly facilitated by nucleosome remodeling activities and/or its own DNA register shifting activity26. It should be noted that, in our ChIP-CryoEM analysis, we could not clearly identify the type of DNA lesion because of the limited resolution. The presence of the CPD lesion in the UV-DDB-nucleosome complex may be verified by further immunoprecipitation with anti-CPD antibodies. Further studies will be awaited. Our pull-down assay in the presence of ATPγS suggested that the damage recognition by UV-DDB in chromatin may depend on ATP hydrolysis. This implies that the ATP-dependent nucleosome remodeling activity may be required for the lesion relocation toward the exposed regions in the nucleosomes, such as the SHL ± 2, ±3, and ±6 positions. Then, UV-DDB may bind to the DNA lesion exposed on the nucleosome surface in cells. UV-DDB reportedly interacts with the INO80 complex, a chromatin remodeler that functions in nucleosome remodeling and/or removal around DNA lesions35. This nucleosome remodeling activity may be important in the DNA-lesion relocation. Further study is required to understand how UV-DDB effectively recognizes the UV lesion by the lesion relocation mechanism in chromatin. The Spodoptera frugiperda Sf9 (Expression Systems, catalog# 94-001F) and Trichoplusia ni High Five (Expression Systems, catalog# 94-002F) cell lines were respectively cultured in Sf-900III (Themo Fisher) and ESF 921 (Expression Systems) media containing 1 × Penicillin-Streptomycin (Nacalai Tesque), in suspension at 27 °C. The U2OS adherent cell line was cultured in RPMI1640 medium (Nacalai Tesque) containing 10% FBS and 1 × Penicillin-Streptomycin, at 37 °C in a 5% CO2 atmosphere. The U2OS/DDB2 KO cell line was established by K.S. The cDNA encoding N-terminally FLAG-tagged human DDB2 was subcloned into the pAAVS1-DNR vector with the AAVS1 Transgene Knock-in vector kit (ORIGENE). Upon co-transfection with pCas-Guide-AAVS1, the U2OS/DDB2 KO/FLAG-DDB2 cell line was selected by 1 μg/ml puromycin. HEK293F (Thermo Fisher, catalog# R79007) expressing FLAG-DDB2 cell line was also established as the same methods using the AAVS1 Transgene Knock-in vector kit (ORIGENE). The Expi293 (Thermo Fisher, catalog# A14635) suspension cell line was purchased from Thermo Fisher and cultured in Expi293 Expression Medium (Thermo Fisher) containing 0.5 × Penicillin-Streptomycin, at 37 °C in an 8% CO2 atmosphere. Expi293/H2A-FLAG was established with the same methods described above, by subcloning the cDNA encoding C-terminally FLAG-tagged human H2A into the pAAVS1-DNR vector. The U2OS cell line was authenticated by short tandem repeat (STR) profiling at Promega. The HEK293F and Expi293 cell lines were purchased from Thermo Fisher as pre-validated cell lines. Cells at 100% confluency (>1 × 108 cells in total) in twenty 150 mm dishes were irradiated with 20 J/m2 doses of UV-C under GL-15 germicidal lamps (ActiveRay Scientific) with a 254-nm peak, and then, incubated for 15 min at 37 °C. For the experiments with the ATP analogue treatment, cells were incubated for 8 h in presence of 20 μM Adenosine 5'-O-(3-thiotriphosphate) (ATPγS; Roche) before UV-C irradiation. After two washes with PBS, the cells were treated with 2.5 mg/ml of Trypsin-EDTA for 5 min and then collected with medium. Cells were fixed with 0.75% formaldehyde by an incubation for 10 min at RT, and then the formaldehyde was quenched with a glycine solution (125 μM final concentration). After two washes with PBS, the cells were lysed with CSK buffer (10 mM Pipes-NaOH (pH 6.8), 3 mM MgCl2, 1 mM EGTA, 33.3% sucrose, 0.1% Triton X-100) containing 0.3 M NaCl and protease inhibitor cocktail (SIGMA). After 1 h incubation, the lysates were centrifuged for 10 min at 20,000 × g to separate the soluble and insoluble fractions. The pellets in the insoluble fraction were washed twice with GF buffer, and then the washed pellets were resuspended by an ultra-sonic homogenizer VP050 (TAITEC) for 3 s in GF buffer containing 1 mM CaCl2 and protease inhibitor cocktail. To cleave the chromatin in the insoluble fraction, 1 unit of Micrococcal Nuclease (New England Biolabs) was added per 1 × 103 cells, and the fraction was incubated for 20 min at 37 °C. After stopping the treatment by adding 1 μl of 0.5 M EDTA per 1 × 106 cells, the fraction was centrifuged for 10 min at 20,000 × g to obtain the supernatant. The resultant supernatant was loaded onto an affinity column packed with FLAG affinity M2 agarose beads (SIGMA). After washing with GF buffer, the bound proteins were eluted with GF buffer containing 100 μg/ml FLAG-peptide. The eluate was concentrated, and the FLAG-peptide was removed by two rounds of dilution with GF buffer and concentration. The following primary antibodies were used in this study with the indicated dilutions: Anti-DDB2 antibody AF3297 (R&D Systems, catalog# AF3297; 1:500 for immunostaining, 1:2000 for immunoblotting), anti-beta-actin MAB8929 (R&D Systems, catalog# MAB8929, 1:10000), anti-histone H3 D1H2 (Cell Signaling Technology, catalog# 4499, 1:2000), and anti-CPD TDM-2 (Cosmo Bio, catalog# NM-DND-001, 1:1000). Secondary antibodies were used at the following dilutions: Anti-Rabbit IgG (SIGMA, catalog# A9919) and Anti-Goat IgG (SIGMA, catalog# A4062) at 1:10,000; anti-Mouse IgG Alexa Fluor 594 (SIGMA, catalog# A-11032) and anti-Rabbit IgG Alexa Fluor 488 (SIGMA, catalog# A-11034) at 1:500; and anti-Mouse IgG antibody (Rockland, catalog# 710-1602) at 1:2000. For immunofluorescence staining, cells were irradiated with 100 J/m2 doses of UV-C by an LED lamp (AItec) with a 265-nm peak through 5 μm pore size hydrophilic polycarbonate membrane filter (Merck) and incubated for 10 min at 37 °C. Cells were fixed with 4% paraformaldehyde for 10 min at RT and permeabilized by 0.5% Triton X-100 for 10 min at RT. Anti-DDB2 and anti-CPD antibodies were applied and incubated for 1 h at 37 °C. After extensive wash, fluorescently labeled secondary antibodies were added and incubated for 1 h at 37 °C. Following further extensive washing, Antifade Mounting Medium with DAPI (Vector Laboratories) were applied, and cells were visualized using a DeltaVision Elite microscope (GE Healthcare) and pseudocoloring of fluorescence images was performed using Fiji software. For immunoblotting, cells were irradiated with 20 J/m² UV-C (254 nm peak) using GL-15 germicidal lamps (ActiveRay Scientific). After incubation for 30 min, 1 h, or 3 h, cells were lysed in CSK buffer supplemented with 0.3 M NaCl and protease inhibitors. Lysates were fractionated into soluble and insoluble fractions by centrifugation for 10 min at 20,000 × g. The resulting pellet was washed twice with CSK buffer, resuspended, and solubilized by sonication using an ultrasonic homogenizer VP050 (TAITEC) for 10 s to obtain the insoluble fraction. Each fraction was separated by SDS-PAGE and transferred to PVDF membranes using the iBlot 2 Gel Transfer Device (Thermo Fisher) according to the manufacturer's P0 program. Membranes were blocked with 10 ml blocking buffer (50 mM Tris-HCl (pH 7.6), 150 mM NaCl, 0.1% Tween-20, 0.1% iBlock (Thermo Fisher)) for 30 min at RT. Primary antibodies (anti-DDB2 AF3297, anti-β-actin MAB8929, anti-histone H3 D1H2) were diluted to the indicated concentrations in 10 ml of the same blocking buffer and incubated with the membranes overnight at 4 °C. After two brief rinses with TBS-T, membranes were washed four times for 10 min each with 10 ml TBS-T. Secondary antibodies diluted in 10 ml blocking buffer were applied and incubated for 1 h RT. Membranes were then washed again four times for 10 min each with 10 mL TBS-T, incubated with CDP-Star substrate (Thermo Fisher) for 5 min at RT, and imaged using an Amersham Imager 680 (Cytiva). Nucleosome core particles (NCPs) were digested with RNase (NIPPON GENE) and Proteinase K (Roche), and DNA was purified using the QIAquick PCR Purification Kit (QIAGEN). Ten nanograms of heat-denatured DNA were applied to protamine sulfate-coated 96-well ELISA plates (Cosmo Bio, catalog# NM-MA-P003) and air-dried. Wells were washed three times with PBS containing 0.05% Tween-20 (PBS-T), blocked with 2% FBS in PBS at 37 °C for 30 min, and washed again with PBS-T. Samples were incubated sequentially at 37 °C for 30 min each with anti-CPD antibody TDM-2 (Cosmo Bio), anti-mouse IgG secondary antibody (Rockland), and HRP-conjugated streptavidin (Thermo Fisher), with PBS-T washes between steps. Color development was achieved by incubation with citrate–phosphate buffer (24.3 mM citric acid monohydrate, 51.4 mM Na₂HPO₄, pH 5.0) for 5 min at RT, followed by addition of o-phenylenediamine (3.7 mM; Fujifilm) and 70 ppm H₂O₂ in the same buffer for 5–30 min at 37 °C. Reactions were stopped with 2 M H₂SO₄ (final one-third of the reaction volume), and absorbance was measured at 490 nm using a ClarioStar microplate reader (BMG Labtech)36. For the preparation of α-satellite DNA sequence, following oligonucleotides were prepared. While Oligos 1, 3, and 4 were purchased from FASMAC, the oligonucleotide containing cyclobutane pyrimidine dimers (CPD)(Oligo 2) was synthesized on an Applied Biosystem 3400 DNA synthesizer, using the CPD building blocks (Glen Research, catalog#11-1330-90)37, a chemical phosphorylation reagent (Glen Research, catalog#10-1900-90), and deoxyribonucleoside phosphoramidite building blocks (Glen Research, catalog#10-1000-05, #10-1029-05, #10-1015-05, and #10-1030-05), which were assembled on controlled-pore glass supports (Glen Research, catalog#20-2031-10). The synthesized oligonucleotides were cleaved and deprotected according to the manufacturer's instructions, and the products were analyzed and purified by HPLC. A Waters µBondasphere C18 5 µm 300 Å column (7.8 mm × 300 mm) was used for purification with a linear gradient of acetonitrile in 0.1 M triethylammonium acetate (pH 7.0). These oligonucleotides were annealed in the combinations of oligos 1 and 3, and oligos 2 and 4, at a 1:1 ratio in annealing buffer (10 mM Tris-HCl pH 8.0, 10 mM NaCl, and 1 mM EDTA). Annealing was performed by heating at 95 °C for 5 min, followed by gradual cooling to 24 °C over 4 h. The resulting duplexes were then ligated at RT for 1 h using T4 DNA ligase (Takara) in 1× ligation buffer (Takara) supplemented with 0.5 mM ATP (Roche). The ligated DNA was further purified by native polyacrylamide gel electrophoresis, using a Prep Cell apparatus (Bio-Rad) in TE buffer (10 mM Tris-HCl (pH 7.5) and 100 μM EDTA). DNA sequences used in this study and CPD locations are indicated in Fig. Human histone proteins were purified from E. coli. Equimolar amounts of lyophilized histones were mixed in Denaturing buffer (20 mM Tris-HCl (pH 7.5), containing 7 M guanidine hydrochloride and 20 mM 2-mercaptoethanol). After dialysis against Refolding buffer (2 M NaCl, 10 mM Tris-HCl, pH 7.5, 1 mM EDTA, 2 mM 2-mercaptoethanol), histone complexes were purified by size exclusion column chromatography (HiLoad 16/600 Superdex 200; Cytiva). Fractions containing histone octamers were collected and concentrated. The nucleosome was reconstituted by salt dialysis and purified using 6% non-denaturing polyacrylamide gel electrophoresis, using a Prep Cell apparatus (Bio-Rad). After co-transfection of pAC8 and viral DNA to Spodoptera frugiperda Sf9 cells, the cells were incubated for 5 days. Further amplification of the viruses by infecting Sf9 cells for 3 days was repeated until the P3 virus was obtained. DDB1 and DDB2 were co-expressed in 4 L cultures of Trichoplusia ni High Five cells, using the Bac-to-Bac system (Thermo Fisher). Cells were maintained at 27 °C and collected three days post-infection. The pellets were resuspended in lysis buffer (50 mM Tris-HCl pH 8.0, 500 mM NaCl, protease inhibitor cocktail (SIGMA), and 250 μM TCEP) and disrupted by sonication. Following centrifugation, the clarified supernatant was subjected to Streptactin affinity chromatography (IBA) to isolate the UV-DDB complex via the N-terminal Strep-tags on both DDB1 and DDB2. The eluate was subsequently purified using Mono Q ion-exchange chromatography (Cytiva), followed by size-exclusion chromatography (HiLoad 16/600 Superdex 200; Cytiva) in GF buffer (50 mM HEPES pH 7.4, 150 mM NaCl, 250 μM TCEP). The final purified protein was concentrated and stored at −80 °C. Nucleosomes (40 nM) were mixed with UV-DDB (0, 10, 20, 40, 80, 160 nM) in Binding buffer (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, 3.4 mM MgCl2, 1.4 mM EDTA, 0.014% Triton X-100, 0.1 mg/ml BSA, and 1 mM DTT) and incubated on ice for 30 min. The mixtures were fractionated on a 6% non-denaturing polyacrylamide gel in 0.5 × TGE buffer (12.5 mM Tris base, 96 mM glycine, and 0.5 mM EDTA). The gel was stained with SYBR Gold (Thermo Fisher) and visualized with a Typhoon biomolecular imaging analyzer (Cytiva). Droplets of purified native nucleosomes (10−15 μl) were spotted in 35 mm glass-bottom dishes (Matsunami) and irradiated with 500 J/m2 doses of UV-C by an LED lamp (AItec) with a 265-nm peak. The droplets were then collected and combined in a microtube. Elute fractions were prepared in the same method described in Fig. 1b, c. HEK293F expressing FLAG-DDB2 cells was used in this experiment. Eluted proteins were separated by SDS-PAGE and stained by Oriole Fluorescent (Bio-rad). Target proteins in the excised gels were in-gel digested with trypsin and identified by nano LC-MS/MS analysis according to a modified procedure of references 39,40,41. By shotgun method, elute fraction was digested with trypsin and identified by nano LC-MS/MS analysis. LC-MS/MS analysis was conducted by Orbitrap Fusion mass spectrometer equipped with Ultimate3000 nano LC containing an autosampler (Thermo Fisher). The nano LC gradient was delivered at 300 nl/min and consisted of a linear gradient from 2 to 40% acetonitrile in 60 min. Most 10 intense precursor ions were selected in MS1 scans and sequentially fragmented by higher-energy collisional dissociation in MS2 scans. The MS/MS data were searched against the Homo sapiens protein sequence database from UniProt using Proteome Discoverer 2.1 (Thermo Fisher) for protein identification. Trypsin cleavage specificity was restricted to lysine residues only, with a minimum peptide length of 6 amino acids and up to two missed cleavage sites allowed. Spectra were searched with a precursor mass tolerance of 10 ppm and a fragment mass tolerance of 0.6 Da. The target false discovery rate (FDR) was set to 0.01 (strict) and 0.05 (relaxed) for peptide-spectrum matches (PSMs) and peptides, and to 0.01 (strict) and 0.05 (relaxed) for proteins. Carbamidomethylation of cysteine was specified as a fixed modification, whereas oxidation of methionine and phosphorylation of serine, threonine, and tyrosine were set as variable modifications. Graphene-coated EM grids were prepared following a modified PMMA-assisted transfer protocol. Briefly, graphene films supported on copper foil were coated with a thin layer of PMMA (poly(methyl methacrylate), average MW 15,000; 1% w/v in anisole) and baked on a hot plate at 100 °C for 15 min. The PMMA/graphene/copper samples were then plasma cleaned using a SOLARUS II plasma cleaner (Gatan) under default O₂/H₂ conditions for 2 min. Copper foils were subsequently removed by floating the samples on 1 M ammonium persulfate (APS) solution at RT with repeated exchanges of fresh solution until complete dissolution of the copper substrate. The PMMA/graphene films were then washed several times with MilliQ water before being transferred onto holey gold EM grids (R1.2/1.3, 200 mesh, Au; Quantifoil Micro Tools GmbH). After drying on filter paper, grids were heated on a hot plate at 130 °C for 20 min. To remove the PMMA support, grids were sequentially immersed in acetone, dichloromethane, and 2-propanol at RT, with gentle agitation and incubation steps to ensure complete polymer removal. Finally, grids were dried on filter paper and annealed on a hot plate at 100 °C for 20 min. Grids were stored under ambient conditions until use. Only grids with continuous graphene coverage were selected for subsequent cryo-EM experiments42. Glow discharging was performed with ozone generated by a UV253MINI cleaner for 10 min (Filgen). After 8 s blotting, the grid was vitrified using a Vitrobot Mark IV (Thermo Fisher) at 4 °C and 100% humidity. For the structure determination of recombinant UV-DDB bound to NCPCPD or UV-damaged cellular NCP, UV-DDB was mixed with NCPs, in a 1:2 molar ratio for NCPCPD or a 1:1 molar ratio for UV-damaged cellular NCP, in GF buffer (50 mM HEPES, pH 7.4, 50 mM NaCl, and 250 μM TCEP). After 30 min incubation on ice, 2.5 μl portions of the samples were applied to R 2/1 Quantifoil grids for NCPCPD, and R 1.2/1.3 Quantifoil grids for other NCPs. Glow discharging was performed with a PIB-10 ION Bombarder (Vacuum Device) for 60 s under soft conditions. After 6 s of blotting, the grid was vitrified using a Vitrobot Mark IV (Thermo Fisher) at 4 °C and 100% humidity. Micrographs were captured by a Krios G4 transmission electron microscope (Thermo Fisher) with a K3 direct electron detector and a BioQuantum energy filter (Gatan). Approximately 10,000–20,000 movies were captured for each cryo-EM sample by the EPU software (Thermo Fisher). Data acquisition was performed with a 1.06 Å pixel size and −1.0 to −2.5 μm defocus values at 0.25 μm steps. Data collection was performed by 40 frames. NCP-UV-DDB particles were picked by RELION autopick using a Laplacian of Gaussian filter, and particles were extracted with a binning factor of 2. After obtaining 2D classification images, five images were used as a reference and then reference autopicking and extraction were performed. After junk particles were removed by 2D classification, 3D classification was performed. NCP-UV-DDB classes were selected by several rounds of 3D classification with or without masking, and then possible classes were processed with 3D autorefinement. After re-extraction with a binning factor of 1, 3D autorefinement, Bayesian polishing, and CTF estimation were further applied to obtain high-resolution cryo-EM maps. Final particle sets were analyzed by 3D autorefinement, postprocessing, and the DeepEMhancer software46. The initial model of NCP-UV-DDB was built based on the structures previously determined (PDB IDs: 6R8Y, 6R8Z, 5B24, 4A0K). Cryo-EM maps were sharpened after processing with the DeepEMhancer or Relion software as a final map. Models of amino acids were modified by real space refinement with the Phenix software, and adjusted by COOT47,48. Structural figures were drawn with the Pymol and ChimeraX software49,50. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The cryo-EM maps and coordinates of structures, cellular NCP-UV-DDB, NCP-UV-DDB complex containing CPD, UV-DDB bound to native NCP at SHL  ±2, ±3, and ±6 have been deposited in the Electron Microscopy Data Bank (EMDB) and Protein Data Bank (PDB), under the accession codes EMD-61242, EMD-61243 (PDB: 9J8W [https://doi.org/10.2210/pdb9J8W/pdb]), EMD-61246, EMD-61247, and EMD-61248, respectively. The cryo-EM and X-ray crystallographic structures used in this study, NCP-6-4PP(−1)-UV-DDB, NCP_THF2(−1)-UV-DDB, NCP-CPD, DDB1-DDB2-CUL4A-RBX1 bound to a 12 bp abasic site containing DNA-duplex, human nucleosome with the α-satellite DNA sequence, hsDDB1-hsDDB2 complex, and human nucleosome with the Widom 601 L DNA sequence are available in the PDB, under the accession codes 6R8Y [https://doi.org/10.2210/pdb6R8Y/pdb], 6R8Z [https://doi.org/10.2210/pdb6R8Z/pdb], 5B24 [https://doi.org/10.2210/pdb5B24/pdb], 4A0K [https://doi.org/10.2210/pdb4A0K/pdb], 3AFA [https://doi.org/10.2210/pdb3AFA/pdb], 3EI4 [https://doi.org/10.2210/pdb3EI4/pdb], 7VZ4 [https://doi.org/10.2210/pdb7VZ4/pdb]. LC-MS/MS analysis data were deposited in JPOST (ID: JPST003722) and PXD (ID: PXD062246)51,52. Source data are provided with this paper. Crystal structure of the nucleosome core particle at 2.8 A resolution. McGinty, R. K. & Tan, S. Nucleosome structure and function. Zhou, K., Gaullier, G. & Luger, K. Nucleosome structure and dynamics are coming of age. Li, G., Levitus, M., Bustamante, C. & Widom, J. Rapid spontaneous accessibility of nucleosomal DNA. Hanawalt, P. C. & Spivak, G. Transcription-coupled DNA repair: two decades of progress and surprises. The bases for Cockayne syndrome. Lehmann, A. R. DNA repair-deficient diseases, xeroderma pigmentosum, Cockayne syndrome and trichothiodystrophy. Sugasawa, K. et al. A multistep damage recognition mechanism for global genomic nucleotide excision repair. Sugasawa, K. et al. Xeroderma pigmentosum group C protein complex is the initiator of global genome nucleotide excision repair. Sugasawa, K. et al. UV-induced ubiquitylation of XPC protein mediated by UV-DDB-ubiquitin ligase complex. Puumalainen, M. R. et al. Chromatin retention of DNA damage sensors DDB2 and XPC through loss of p97 segregase causes genotoxicity. Matsumoto, S. et al. Functional regulation of the DNA damage-recognition factor DDB2 by ubiquitination and interaction with xeroderma pigmentosum group C protein. Rüthemann, P., Balbo Pogliano, C. & Naegeli, H. Global-genome nucleotide excision repair controlled by ubiquitin/sumo modifiers. Schärer, O. D. Nucleotide excision repair in eukaryotes. Matsumoto, S., Horikoshi, N., Takizawa, Y. & Kurumizaka, H. Chromatin structure related to oncogenesis. Thoma, F. Repair of UV lesions in nucleosomes–intrinsic properties and remodeling. Rodriguez, Y., Hinz, J. M. & Smerdon, M. J. Accessing DNA damage in chromatin: preparing the chromatin landscape for base excision repair. Wittschieben, B., Iwai, S. & Wood, R. D. DDB1-DDB2 (xeroderma pigmentosum group E) protein complex recognizes a cyclobutane pyrimidine dimer, mismatches, apurinic/apyrimidinic sites, and compound lesions in DNA. Jang, S. et al. Damage sensor role of UV-DDB during base excision repair. Cooperative interaction between AAG and UV-DDB in the removal of modified bases. & Van Houten, B. UV-DDB as a general sensor of DNA damage in chromatin: multifaceted approaches to assess its direct role in base excision repair. Structural basis of UV DNA-damage recognition by the DDB1-DDB2 complex. The molecular basis of CRL4DDB2/CSA ubiquitin ligase architecture, targeting, and activation. Yeh, J. I. et al. Damaged DNA induced UV-damaged DNA-binding protein (UV-DDB) dimerization and its roles in chromatinized DNA repair. Structural basis of pyrimidine-pyrimidone (6-4) photoproduct recognition by UV-DDB in the nucleosome. Matsumoto, S. et al. DNA damage detection in nucleosomes involves DNA register shifting. Multiple structures of RNA polymerase II isolated from human nuclei by ChIP-CryoEM analysis. Sakai, W. et al. Functional impacts of the ubiquitin-proteasome system on DNA damage recognition in global genome nucleotide excision repair. Gale, J. M. & Smerdon, M. J. UV-induced (6-4) photoproducts are distributed differently than cyclobutane dimers in nucleosomes. You, Y. H. et al. Cyclobutane pyrimidine dimers are responsible for the vast majority of mutations induced by UVB irradiation in mammalian cells. True XP group E patients have a defective UV-damaged DNA binding protein complex and mutations in DDB2, which reveal the functional domains of its p48 product. Ghodke, H. et al. Single-molecule analysis reveals human UV-damaged DNA-binding protein (UV-DDB) dimerizes on DNA via multiple kinetic intermediates. Yan, L., Wu, H., Li, X., Gao, N. & Chen, Z. Structures of the ISWI-nucleosome complex reveal a conserved mechanism of chromatin remodeling. A., Malik, S. & He, Y. Cryo-EM structure of SWI/SNF complex bound to a nucleosome. Jiang, Y. et al. INO80 chromatin remodeling complex promotes the removal of UV lesions by the nucleotide excision repair pathway. Simultaneous establishment of monoclonal antibodies specific for either cyclobutane pyrimidine dimer or (6-4)photoproduct from the same mouse immunized with ultraviolet-irradiated DNA. Iwai, S. Preparation of oligodeoxyribonucleotides containing the pyrimidine(6-4)pyrimidone photoproduct by using a dinucleotide building block. Abdulrahman, W. et al. A set of baculovirus transfer vectors for screening of affinity tags and parallel expression strategies. Shevchenko, A., Wilm, M., Vorm, O. & Mann, M. Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels. Funato, K. et al. SIRT2-mediated inactivation of p73 is required for glioblastoma tumorigenicity. Structural basis for binding diversity of acetyltransferase p300 to the nucleosome. High-yield monolayer graphene grids for near-atomic resolution cryoelectron microscopy. Kimanius, D., Dong, L., Sharov, G., Nakane, T. & Scheres, S. H. W. New tools for automated cryo-EM single-particle analysis in RELION-4.0. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Alexander, N., Woetzel, N. & Meiler, J. Bcl::Cluster : a method for clustering biological molecules coupled with visualization in the PyMOL molecular graphics system. IEEE International Conference on Computational Advances in Bio and Medical Sciences, 13–18. Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in visualization and analysis. Okuda, S. et al. jPOSTrepo: an international standard data repository for proteomes. Okuda, S. et al. jPOST environment accelerates the reuse and reanalysis of public proteome mass spectrometry data. & Olson, W. K. 3DNA: a versatile, integrated software system for the analysis, rebuilding and visualization of three-dimensional nucleic-acid structures. We thank Y. Iikura, Y. Takeda, and M. Dacher (University of Tokyo) for their assistance, and N. Horikoshi (University of Tokyo) for helping to build 3D structure models. This work was supported in part by JSPS KAKENHI Grant Numbers JP22K18034 [to S.M. ]; JST PRESTO Grant Number JPMJPR2288 [to S.M. ], JST ERATO Grant Number JPMJER1901 [to H.K. ], JST CREST Grant Number JPMJCR24T3 [to H.K. ], Novartis Research Grants from The NOVARTIS Foundation for the Promotion of Science [to S.M. ], Basic Science Research Projects from The Sumitomo Foundation [to S.M. ], and Research Support Project for Life Science and Drug Discovery (BINDS) from AMED under Grant Numbers JP24ama121002 [to Y.T.] Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Bunkyo-ku, Tokyo, Japan Syota Matsumoto, Yoshimasa Takizawa, Mitsuo Ogasawara, Kana Hashimoto, Lumi Negishi, Wenjie Xu, Haruna Tachibana & Hitoshi Kurumizaka Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Bunkyo-ku, Tokyo, Japan Department of Biological Sciences, Graduate School of Science, The University of Tokyo, Bunkyo-ku, Tokyo, Japan Wenjie Xu, Haruna Tachibana & Hitoshi Kurumizaka Division of Chemistry, Graduate School of Engineering Science, Osaka University, Toyonaka-shi, Osaka, Japan Junpei Yamamoto & Shigenori Iwai Biosignal Research Center, Organization of Advanced Science and Technology, Kobe University, Kobe-shi, Hyogo, Japan RIKEN Center for Biosystems Dynamics Research, Yokohama, Japan Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived and planned the experiments. established the cell lines used in this study and prepared samples for western blots, immunostaining, ChIP-CryoEM, ELISA, EMSA, and cryo-EM. performed mass spectrometry analyses. prepared the native nucleosomes used in this study. performed cryo-EM microscopy and analysis. established the U2OS/DDB2 KO cell line. performed biochemical and cryo-EM experiments, with guidance from H.K. All authors contributed to writing the manuscript. Correspondence to Hitoshi Kurumizaka. The authors declare no competing interests. Nature Communications thanks Bennett Van Houten and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Matsumoto, S., Takizawa, Y., Ogasawara, M. et al. Structural basis of cyclobutane pyrimidine dimer recognition by UV-DDB in the nucleosome. Version of record: 11 November 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            