
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-08-14</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a65641480/testing-unthinkable-technologies/'>Scientists Are Secretly Testing Unthinkable Technologies ... Years Before They Exist</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 12:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“Science fiction science” or “sci-fi-sci” is an idea put together by researchers Iyad Rahwan (from the Max Planck Institute for Human Development in Germany), Azim Shariff (from the University of British Columbia in Canada), and Jean-Francois Bonnefon (from the Toulouse School of Economics in France). They  describe it as a new process that attempts to apply the scientific method to technologies that are either being planned or are in the early phases of development. Such predictions have been made in science fiction before, but outside of the genre, they have never been fully explored from a scientific perspective. “Predicting the social and behavioral impact of future technologies, before they are achieved, would allow us to guide their development and regulation before these impacts get entrenched,” the researchers said in a study posted to the preprint server arXiv. Rahwan, Shariff, and Bonnefon suggest that using the scientific method to predict the effects of technologies that will likely surface in the near future (though, what exactly “near future” means can be hazy) will make their potential effects more clear to developers, consumers, and policymakers. This unconventional approach has been met with skepticism, as experimental scientists understandably tend to question its validity. Using social media as an example, they suggest that in retrospect, running simulations of how the technology might have operated and having participants virtually interact with it could have helped predict the aftermath of its widespread use, from self-esteem issues to ethics being seriously questioned. Predicting the effects of social media before everyone was constantly checking socials on their smartphones might have led to a more cautious outlook. For instance, effects of the technology gone to extremes become a dystopian reality on the Black Mirror episode “Nosedive,” where social media not only broadcasts people's lives, but social scores that help gauge their popularity and use that data to rank them among their peers. Gage is an app that keeps track of how employees are rated by coworkers. YouTube creator Joshua Fluke criticized Gage as “an algorithmic reputation system” that could be extremely problematic. When employee evaluation relies more heavily on social scores than the quality of actual work done, entire groups could suffer. Those who are neurodivergent and may not communicate in the expected neurotypical way could face difficulty being hired as a result of codified negative social feedback. The researchers describe social credit systems on an even larger scale than Black Mirror, and list them among other types of what they refer to as “nascent or speculative technologies” that could potentially spark policy debates. Hypothetical systems that use AI to monitor every behavior in real time before publicly releasing social credit scores have sparked so much controversy that the European Union is leaning toward preemptively banning them. Will virtual reality experiments that introduce people to technologies that do not yet exist be able to accurately predict their impact on society? For now, that remains in the realm of science fiction. Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. An AI System Found a New Kind of Plasma Physics Why One Side of Earth Is Rapidly Getting Colder</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-61998-2'>A reconceptualized framework for human microbiome transmission in early life</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 12:11:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Human development and physiology are fundamentally linked with the microbiome. This is particularly true during early life, a critical period for microbiome assembly and its impact on the host. Understanding microbial acquisition in early life is thus central to both our basic understanding of the human microbiome and strategies for disease prevention and treatment. Here, we review the historical approaches to categorize microbial transmission originating from the fields of infectious disease epidemiology and evolutionary biology and discuss how this lexicon has influenced our approach to studying the early-life microbiome, often leading to confusion and misinterpretation. We then present a conceptual framework to capture the multifaceted nature of human microbiome acquisition based on four key components: what, where, who, and when. We present ways these parameters may be assigned, with a particular focus on the ‘transmitted strain' through metagenomics to capture these elements. We end with a discussion of approaches for implementing this framework toward defining each component of microbiome acquisition. The human microbiome, the assembly of microorganisms living in and on the human body, and the genes and products of these microbes, has emerged as a pivotal determinant of host physiology and disease, influencing multiple tissues and organ systems. Early life, defined here as spanning from pregnancy to infancy, is a critical period of host-microbe interactions. The interactions range from the impact of maternally derived microbial metabolites on fetal development in utero to serving as a blueprint of current and future health1. Disturbances to the source, order of arrival, and succession of microbes during early life have been linked to infections and different physiological disorders, including cancer2. The terminologies describing microbiome acquisition, mainly that of vertical and horizontal transmission, have roots in infectious disease epidemiology and evolutionary biology. The origin of using “transmission” to describe the transfer of microorganisms from one host to another originates from Koch's works3,4 and studies of invertebrates causing “vector-born” infectious diseases of animals and plants5,6,7,8,9. Applied to mother-to-child transmission, the term “vertical” can be traced to the early 1950s and the transmission of microbial pathogens10,11,12,13,14,15. In the coming decades, microbial transmission between parent and offspring remained in the realm of infectious diseases, being referred to as “inheritance of infection” or “hereditary transmission” (Box 1). Although the used definitions consider single infectious agents and may give information on cross-generation microbial transfer, the microbiome field has adapted the terminology for mother-to-child microbial transmission without major modifications. Compared to the infectious disease lens, in which the term vertical transmission focuses on generational inheritance (namely from a parent), we see an added focus from the fields of ecology and evolutionary biology. In addition to generational inheritance, these scientific fields have emphasized the mechanism and, relatedly, timing of transmission. Here, the added distinction between vertical and horizontal transmission is based on when and, by extension, where transmission occurs (Box 1). A prime example of vertical transmission is the transovarial transmission of the intracellular bacteria Wolbachia in the fruit fly Drosophila16. In comparison, horizontal transmission of microbes, in essence, is everything else, with a focus on routes such as sexual, vector-borne, and attendant-borne transmission17. Well-studied examples focus on environmental sources of microbes in open ecosystems, such as the colonization of the light organ by the marine free-living symbiont Vibrio fischeri by the Hawaiian bobtail squid Euprymna scolopes, and the acquisition of the rhizosphere microbiome from soil. Yet, within studies of the microbiome acquisition along animal and plant life cycles, there are gray areas between vertical and horizontal transmission. For example, during trophallaxis in termites, offspring may acquire the maternal fecal microbiome after hatching. In stinkbugs of the family Plataspidae, endocellular γ-Proteobacteria are transmitted to the offspring via symbiont capsules that females produce upon oviposition18. In this Perspective, we discuss how the above-described historical lexicon is often ambiguous when used in the context of human microbiome acquisition and limited in capturing the multidimensional features of microbial transmission. Given the imprecision and limitations of this status quo lexicon, we propose a conceptual framework termed 4 W to describe microbiome transmission in early life, centered on assigning four critical features: what, where, who, and when (Box 1). We then follow with a discussion of how we can capture these features in a human microbiome cohort design. Central to an accurate description of early-life microbiome acquisition is the ability to define from ‘who', ‘where', and ‘when' transmission occurs by methods enabling tracking of the ‘what'. We define the “transmitted microbial strain” based on metagenomic resolution as currently the most precise unit to determine the transmission of microbes over space and time, and discuss how it can be used to assign the parameters of microbiome acquisition. We then introduce a fifth question, ‘why', discussing how a 4 W framework can address both mechanisms of early-life acquisition, and while not the focus of this Perspective, aspects of microbial assembly, such as succession and colonization. We end by providing recommendations for the design of studies aiming to capture the 4Ws of microbiome acquisition. “Vertical transmission” - The direct transfer of infection from a parent organism to progeny. “Horizontal transmission” - Any transfer of infection between host individuals, except that which occurs directly from parent to progeny. “Horizontal transmission” - From an environmental, often “free-living” symbiont source, anew by each host generation. Proposed 4 W framework for microbial transmission in early life “What” - Microbes with replicative potential, microbial structural elements, and products, such as metabolites. Parents, household members, pets/other animals, and the abiotic environment. “When” - The timing of the event of transmission, such as during pregnancy, at birth, or at what age. The term “vertical” has been widely used for human microbiome acquisition. In similar contexts, “vertical transmission” is broadly and ambiguously defined as 1) transmission from the mother or both parents, 2) from and to different body sites (in contact or not with the open environment), and 3) transmission during or also after birth. Although a commonly used term originating from infectious disease epidemiology, the description of “vertical transmission” in scientific reports often lacks important information, including simultaneous reporting of timing, source of transmission, and the microbial commodity being transmitted. With a few exceptions, the term “horizontal transmission” is rarely used in the early-life microbiome field. Instead, other terms such as “microbial taxa dispersal from different sources” and “horizontal dispersal of microbes” have been applied. The few studies that used the term “horizontal transmission” generally indicated that it is not mother-to-infant microbial transmission; however, it is unclear whether the transmission of the microbiome is from other family members, the community, or the environment. That this term is not commonly used in the context of early-life microbiome acquisition and the absence of its definition shows that transmissions of microbes from the environment and others than the mother remain understudied. What terms, if any, should we use to describe early-life microbiome acquisition? If based on an evolutionary biology definition focused on the timing of transmission, the use of “vertical” and “horizontal” transmission in reference to early-life microbiome transmission may similarly be imprecise and potentially inappropriate. If defined as transmission before birth, the acquisition of microbes with replicative potential is limited to pathogens in an uncomplicated pregnancy, therefore excluding “vertical transmission” from an evolutionary framework. Similarly, as we will describe below, when considering microbial transmission from a broader perspective, including microbe-derived factors such as metabolites across the placenta, indeed, “vertical transmission” from an evolutionary definition may apply, yet this situation is seldom considered in the common use of the term. For vaginal delivery, do we consider this vertical, horizontal, or rather use quasi-terms as in the evolutionary literature? As such, one of the fundamental features distinguishing vertical and horizontal transmission is whether transmission occurs in a closed (transplacental/transovarial) or open ecosystem (consider whale vaginal birth) and thus varies in ecological competition and fidelity of transmission19. We are faced with a common problem in lexicology: the misuse of commonly accepted terms or their acceptance, explicit statements of definition, and even redefinition of terms. As reviewed above, precise terminology for different mechanisms of transmission is lacking, and the existing terms referring to the transmission of microbial strains from mother to infant fail to capture the multifaceted nature of microbial acquisition. Ultimately, what do we want to know about early-life microbiome acquisition? Here, we provide a conceptual and operational framework of early-life microbial transmission structured around four central components (4 W): what, where, who, and when. Characterizing transmission events according to each of these components is critical to our understanding of the assembly of the human microbiome and the mechanisms by which the microbiome impacts human physiology and disease. This precise characterization should inform study design, methodology, and results interpretation when studying the early-life microbiome. Here, we lay out this conceptual framework and provide examples of how these parameters may characterize new and unanticipated transmission events. When considering transmission of the microbiome, we must consider “what” is the transmitted commodity. These might include cells (or, in the case of viruses, virions, viroids or even viroid-like) that have replicative potential (inclusive of microbes in dormant phases such as spores), microbially derived components, such as different structural elements of the cells (proteins/peptides, nucleic acids, mobile genetic elements, lipids and sugars), and their metabolites (Fig. Classically, when thinking about the assembly of the infant microbiome, the operative “what” are microbial cells that can “seed” and subsequently replicate for “colonization”. Importantly, the workhorse of identifying the microbial “what” of the microbiome is next-generation sequencing (NGS), such as amplicon (i.e., bacterial 16S rRNA gene or fungal internal transcribed spacer region) and shotgun metagenomic sequencing, which is based on DNA present in the sample and thus not indicative of “colonization” per se. Advances in single-cell microbial genomics hold the potential for bridging this discrepancy. At present, metagenome-defined strains are the units most accessible to infer transmission and thus assess microbial acquisition in early life (Box 2; Fig. Different microbes colonize the mother at multiple body sites, although the focus is on gut bacteria here. Microbially derived metabolites (what) can translocate from the mother gut (from who) during pregnancy (when) into the intestinal lamina propria and blood circulatory system. Microbially derived DNA from the parent gut can translocate into circulation, cross via the placenta, and may enter different sites in the fetus, including the gut. Sequencing-based techniques may detect the presence of microbial DNA, even when live microbes are not occurring in the system. Live organisms are not expected to translocate outside the gut or cross the placental barrier into the fetus. Pathogenic microorganisms like L. monocytogenes can translocate from the gut and infect the placenta and fetal brain tissue. Strain boundaries should be identified on a species-by-species basis and based on a comparison of (phylo-) genetic distance distributions of strains detected in longitudinal samples from the same individual (same strain; green distribution) to those between unrelated individuals who have never been in contact (different strain; orange distribution). While some strain replacement events might occur within an individual's microbiome even without any intervention (e.g., antibiotic treatment, diet changes), these are a limited minority in samples taken less than six months apart36. Once such thresholds are established, the origin of a strain in the infant can be inferred (maternal: pink distribution; from an unknown source: gray distribution). Sampling of more environments, individuals, and body sites thus adds to the “from where/who” and “to where/who” dimensions, while the collection of samples from multiple time points allows to establish “when” the transmission event took place. A weak signal of the nucleic acid translocation might be detected by NGS and considered as a fetal microbiome. Relatedly, with the exception of colonization of the placenta and fetus with pathogens (such as Group B streptococci; GBS, see “when” below20), contamination with microbial DNA during post-fetal sampling has also been misinterpreted as evidence of placental or fetal colonization21. Because of the latter challenges, one must control for contamination, which is omnipresent in microbial studies and can happen at any stage of sample collection and processing, with well-to-well contamination being a major culprit in microbiome studies. Stringent negative and positive controls need to be included, especially in the case of low microbial biomass samples (e.g., human milk, skin). Several approaches are now available to prevent and detect contamination, including sterile sampling and spike-in quantitative approaches for low biomass communities20,22. Similarly, a variety of open-source tools exists, from lists of common contaminants and guidelines on how to reduce well-to-well effects to reports on technical biases in general. In addition to using NGS approaches for defining the “what” of transmission, advances in analytical techniques such as metabolomics and metaproteomics have enabled the characterization of other essential units of the microbiome (e.g., proteins and metabolites) that may be transmitted in early life. Metaproteomics seeks to comprehensively define the proteins in a given sample. Thus, proteins or shorter peptides detected in infant samples and assigned as of microbial origin may serve both as evidence of microbiome transmission and the potential for function. At the same time, the use of metaproteomics for the study of microbial transmission in early life has been limited to a few pioneering studies so far. For example, bacterial peptides have been detected in the amniotic fluid derived from uncomplicated pregnancies, as well as within extracellular vesicles isolated from amniotic fluid23 and human milk24. The metabolome, the collection of small molecules within a given sample, can also be used to define microbiome transmission. Microbial-derived metabolites, such as short-chain fatty acids or 4-ethylphenylsulfate produced by bacteria in the maternal gut during gestation, may enter circulation and cross the placenta into fetal circulation25. Microbially-derived metabolites have been reported so far in human amniotic fluid26, fetal intestine27, and breast milk28. In addition, free nucleic acids derived from maternal microbes (most notably pathogens) may interact with the placenta, acting as agonizts of innate immunity. Effects of such microbial products can be entry into fetal blood/tissues, and subsequent activation of inflammatory programs impacting the fetus or stimulation of fetal intestinal lymphoid system (Fig. Critically, while some proteins and metabolites can be clearly defined as of microbial origin [such as distinct primary (e.g., acetate) or secondary metabolites (e.g., lantibiotics)], other proteins or metabolites can be synthesized by both host and microbes (e.g., acetate or secondary amino acid metabolites such as serotonin and polyamines). Thus, ascribing such proteins or metabolites as of microbial origin and transmitted by the microbiome requires specialized approaches to trace the source of such molecules. This first has been using 13 C or 15 N labeled substrates, such as dietary fibers or proteins, respectively, in mice with (conventionally reared) or without (germ-free) a resident microbiome. This strategy is followed by sampling of mouse tissues and detection of labeled proteins and/or metabolites, whereby comparison of labels between conventional and germ-free mice defines microbial origin. To date, such an approach has not been used to define early-life transmission. Alternatively, bacteria can be labeled isotopically in vitro, before introduction to mice, followed by sampling of mouse tissues, in essence a ‘pulse-chase' experiment. Such an approach using auxotrophic bacteria to eliminate colonization of labeled bacteria at different timepoints has been used to define ‘when' (gestational, postnatal) and ‘what' (amino acid metabolite, nucleic acid, protein) of early-life transmission of microbial products in mouse models. The “where” of microbial transmission can be thought of as the route of transmission of the “what”, i.e., from where/to where, often occurring “via” conduit(s). During gestation, pathogens, classically such as Listeria monocytogenes, which originate “from” the maternal gut, translocate via the epithelial barrier to the maternal blood and the placenta, eventually disseminating “to” fetal systemic and neurovascular circulation29,30. Another example is the intracellular protozoan Toxoplasma gondii, a parasite of cats with a range of intermediate hosts (e.g., rodents) that transmits from mother to fetus at the placenta, resulting in congenital infection. In vaginal delivery, the birth canal is the main route of microbial transmission, with microbes “from” the maternal vaginal and fecal microbiomes, skin, and nearby environment, being transmitted “to” the newborn's skin, oral cavity, and gut, the latter “via” oral ingestion (Fig. During delivery by Cesarean section, skin microbes from the mother are transferred to the infant, in addition to microbes transferred from the medical staff or operating room (Fig. Breast milk and transmission of breast milk microbiota to the infant during lactation (Fig. 4A) is another example of “where”, such as maternal bacteria transmission via the oral-entero-mammary route31,32. A During vaginal birth (when), microbes (what) are transferred from the mother's (who) vaginal and fecal communities (where from) to the child's oral, gastrointestinal, and skin communities (to where). B During Cesarean section, skin microbes are transferred from the skin of the mother and health personnel to the infant. AThe mother (who) transfers microbes and metabolites (what) to the child via breastfeeding (when) from breast milk (from where) to the mouth, gut, or skin of the infant (to where). B Transfer of microbes from the dog's skin and mouth to the baby's skin and mouth. Related to where is the “who”. The “from who” can vary from both human (mother, father, parent, siblings, health-care providers, etc.) and non-human (pets) and sources from the abiotic environment (food, air, built environment, such as a newborn nursery). “Who” can also involve secondary actors or routes, for example, microbes transmitted from the environment via a secondary carrier. As a hypothetical example, imagine a family dog that carries goat stool microbes on its mouth and transfers them by licking an infant's mouth or skin (Fig. This can be broken down into discrete, operational periods, such as at which gestational week, at which stage of delivery (e.g., premature rupture of membranes), and post-delivery, and including important transitions in diet (suckling and weaning). This final parameter defining early-life microbial transmission emphasizes that the specific timing of when microbes and their products are transmitted is critical to ecological succession, microbial competition, and immune tolerance windows33. An example where the timing of microbial transmission/ colonization has been shown to be important is microbiome disturbance by antibiotics in infancy, which is linked to an increased disease risk later in life34. Since exposures to antibiotics might have a different effect if occurring exclusively during pregnancy or after delivery, this points to distinct time windows by which microbiome transmission impacts infant health. The importance of microbial transmission timing can also be derived from infectious diseases, with examples such as congenital, perinatal, and postnatal acquisition of pathogens, such as cytomegalovirus, human immunodeficiency virus, and herpes simplex35. There, the timing of pathogenic microbe transmission affects the disease risk, with multiple sources having an additive effect on the rate of transmission. The “when” parameter of microbial acquisition also illustrates the multidimensional nature of the early-life microbiome since determining only “when” without any information on the microbial sources (“where” and “who”) is bound to give incomplete information. For example, GBS, which may cause severe neonatal infections such as sepsis and meningitis, is considered of maternal origin (acquired during birth) if it causes early onset infection (< 72 h after delivery). Yet for late-onset sepsis, GBS could be acquired from the mother via various routes, including breast milk or other sources. Hence, to identify the GBS source (“what”) with high certainty, samples from the infant, mother, and their environment (“where” and “who”) would have to be collected during pregnancy and the first month of life (“when”). The 4 W framework provides a conceptual approach to deconstruct microbial transmission, whether in early life or throughout the lifespan. Operationally, defining the 4 W parameters of transmission requires both a toolkit and methods to identify specific microorganisms at the strain level, the lineages they share, the source, and the timing of transmission. First, using observational design, we can identify whether the ‘same' strain is present in samples across the 4Ws (i.e., present in maternal vagina, absent in maternal stool, and detected at specific timepoints after birth in infant stool). A second method of microbial tracking is interventional. A defined strain is introduced as a ‘pre-source' to populate a defined index case ‘source' for both ‘who' and ‘where'. The strain can be traced over time in the infant samples by metagenomic sequencing, genome tag-based identification, or whole genome sequencing upon isolation. Central to this is our ability to identify a ‘strain' using metagenomics readouts and computational approaches. In the following, we discuss the conceptual basis (referring the reader to other papers outlining computational methods)71,72 of defining the “transmitted strain” as a tool to capture the early-life microbial transmission (Fig. Thanks to developments in metagenomics, researchers now have the throughput and ability to accurately profile hundreds of the microbial inhabitants of the human body, together with the genomic resolution to infer transmission between individuals. The human microbiome is person-specific, and multiple tools are now available for profiling genomic variants of microorganisms with strain-level resolution73,74. When applying these tools to infer microbial transmission, common questions remain: “How similar should such variants be to consider that a variant in an infant is the same as that in the mother/ other individual and thus presumably of their origin?”. An overly stringent threshold means missing the detection of some of such events, while one that is too lenient causes false positives, i.e., assuming transmission when that is not the case. Transmitted strain - key unit to determine microbial transmission When considering microbial transmission among individuals, the term “strain” can be used to describe genomic variants of microorganisms that are, in principle, unique to individuals, so that when sharing of a strain between individuals is detected, one can infer a transmission event36. This use of “strain” differs between fields, for example, in classic microbiology, “strain“ represents “a set of genetically similar descendants of a single colony or cell”, while in phylogenetics, the term is sometimes used for leaf nodes71. In metagenomics, metagenome-assembled genomes (MAGs) are also sometimes called “strains”4,72,75. Similarly, to the earlier discussed terms for microbial transmission, the above definitions of “strain” are not suitable for describing transmission in the context of the human microbiome. Therefore, we propose an operational definition of a “transmitted strain” as a “person-specific genetic cluster within a species that allows transmission among individuals to be inferred” (Fig. Beyond arbitrary: Refining microbial strain definitions for transmission While we acknowledge that different methods can give different results (e.g., profiling marker genes or comparing assembled genomes, building phylogenetic trees, or analyzing a multiple sequence alignment), the suggested operational definition of transmitted strain serves the purpose of inferring microbial transmission. Still, the field is moving forward to identify thresholds for genomic similarity that allow for the inference of transmission more accurately36. In addition, as the multiple species in the microbiome evolve at different rates, the thresholds for strain identity are best defined on a species-by-species basis36. For non-bacterial members of the microbiome, other specific thresholds are being defined65. For ecology and evolution studies of the early-life microbiome, the same definition of “strain“ holds; however, more detailed genomic knowledge is necessary than for transmission studies76. Experimental methods for microbial source tracking In addition to advancements in bioinformatic approaches for microbial source tracking, definitive approaches to define sources of microbial transmission have been applied, such as controlled transmission experiments and strain-tagging. When ethical and feasible to conduct, these studies provide a distinct advantage of minimizing ambiguity in inferring sources of transmission. Administration of specific strains (e.g., probiotic) to focal people (such as mothers) and tracking in offspring, for example, by strain-specific PCR or genome sequencing of cultured isolates. Examples include evaluating the transmission of Bifidobacterium probiotic candidates (Probio-M8) ingested by lactating mothers, after which detection was assayed in maternal fecal and breast milk samples and feces of infants31. Genetic tagging of bacteria, such as integrated random barcodes [sequence tag-based analysis of microbial population dynamics (STAMP) and similar approaches77] and wild-type isogenic standardized hybrid (WISH) tags, has been used to label pathogens and trace infections in animal models. Such an approach, combined with the controlled introduction of tagged strains, can be used to define both the sources of transmission and additionally determine the population bottlenecks during early-life transmission. Unlike the movie, it is not feasible nor are agencies likely to fund a prospective study encompassing the multiverse of samples and methods of analysis, powered to capture the 4Ws in the scope and depth required to answer the who (from and to), what, where, and when of each microbe and microbial product of early-life transmission. Ultimately, the design of a prospective study of microbiome acquisition (although not limited to early life period) will depend on a balance of the constraints of a study (e.g., budget, sample collection infrastructure, storage, and technical/analytic capacity), the primary and secondary aims of the study, and which and how many of the 4Ws should be captured to answer specific questions. In Table 1, we highlight a selection of representative studies of early-life microbiome acquisition, presenting these studies through the lens of the 4 W framework, defining which aspects of 4 W were captured, focused on, and how these allowed for the definition of early-life microbial transmission. In Box 3, we provide case studies in which specific questions can be answered and prioritized (and ‘future-proofed' for the potential to address additional and forthcoming questions and aspects of transmission) through weighing sample collection and analysis for specific aspects of the 4Ws. Below, we discuss theoretical approaches to capture the 4Ws in a prospective study of early-life microbial transmission. Although we focus our discussion of the 4 W framework on microbial acquisition at early life, the lens of 4 W can be applied for microbial transmission, colonization, and succession throughout a person's lifetime36, such as defining the 4Ws of microbiome ecology during travel, hospital admissions, and microbiome repopulation through and after antibiotic use. As most (but not all and not all the time22) human microbiomes are dominated by bacteria, metagenomics will bias to a bacterial ‘what'. However, it is crucial to understand that microbial transmission is not restricted to bacteria, which have been the most studied to date. Viruses, fungi, and protozoa, as well as different microbial metabolites and structural components, play significant roles in early life development and deserve sufficient attention from the research community. Current and future studies need to expand the description of “what” in terms of whether targeting microorganisms or metabolites, including the resolution level of the identified organism or metabolite. This will require improved sequencing techniques, such as the use of long-read sequencing to provide better strain-specific identification, tools like Hi-C cross-linkage to map mobile elements and viruses to their host organisms, deep sequencing to allow full coverage of rare community members, and targeted enrichment of hard-to-capture organisms and clades37. Both NGS workflows and omics-based measurements (such as metabolomics and proteomics to a certain extent) are semi-quantitative, providing the relative abundance of the ‘what'. First, our ability to determine microbial growth and colonization versus passing through without replication is limited when we measure the relative abundance of DNA through NGS. Second, measurement of one domain of life (as above), let alone the relative abundance of this domain, curtails our ability to determine the role of interkingdom interactions in early-life transmission. Relatedly, and third, approaches have been proposed to utilize absolute quantification from meta-omics data38, such as metabolomics and proteomics, that can aid in deciphering the mechanisms of microbe-microbe and microbe-host interactions. Future studies focusing on technological and analytical quantitation of the ‘how much' of the microbiome22,39,40,41 during early life will allow us to turn the ‘what' of microbial transmission into the ‘how'. The role of the “where” factor has largely been recognized as a critical influence on early microbiome development. Babies delivered vaginally vs Cesarean delivery show pronounced differences in their microbiome compositions during the first year of life42. To address the “where” aspect comprehensively, it is essential to collect multiple potential sources such as feces, vaginal fluid, skin, saliva, and breast milk and detailed information on delivery and feeding, including specifics such as the time the amniotic sac was broken, the use of a vacuum during delivery, the use of a breast pump, intermittent formula feeding, and finally, multiple samples from the environment. For the “who” aspect, sampling should not be restricted to babies and mothers. Samples from family members and other proximate individuals, including pets, can also carry valuable information. Specific questions around “who” could also be addressed through study designs encompassing the diversity of familial relationships: studies of microbial acquisition in children born by surrogacy or children breastfed by parents who did not give birth to them may provide insight into differences in microbial source and transmission timing. “Who” should also include health care providers, especially in the newborn nursery and neonatal intensive care units, in such critical early life periods. Compared to a rather stable microbiome of a healthy adult, the infant microbiome experiences dynamic changes during its establishment. In contrast, the mother's microbiome gradually changes during pregnancy and returns to a pre-pregnancy state after delivery44,45. Therefore, although challenging, samples should be collected at least once from the mother before and after birth, from all individuals involved in birth, and more frequently from the infant in the first weeks of life. Such a sampling strategy will offer the most detailed insights into the establishment of the infant microbial ecosystem. After this initial stage, monthly sampling from the infant, family members, and environment would be ideal for capturing the different sources contributing to the microbiome development. The frequency of sampling should also increase during life events that alter the microbiome composition, such as the introduction of new food and weaning in general, vaccination, or medical treatment. Finally, while some reports suggest that the infant gut microbiome starts to resemble an adult one after the age of two to three years, others show that a child's microbiome does not reach an adult-like state until later46. These discrepancies underline the need for more frequent and longer follow-ups on the child's microbiome development. In theory, a prospective study of early-life transmission would collect (as we joke) everything (‘what') from everyone and every place (‘who'), everywhere (‘where', duh), all the time (‘when'). In theory, unless you know from whom, to where, and when transmission occurs, then anything is possible, and the sample collection and analysis strategy needs to be nearly open-ended. But in practice, (1) not all possibilities are likely (yes, the Yoga instructor or kitchen sink could be a source to the baby), (2) not all data is relevant, and (3) most importantly, there are limitations. These range from budget and sample volume to various logistical, technical, and computational capacity constraints. Ultimately, one must ask what aspect(s) of early-life acquisition one wants to know. From there, based on balancing the limitations outlined above and with a certain degree of future proofing, i.e., sample collection to address alternative hypotheses/ additional questions, one can design generalist (capture all 4Ws broadly) or specialist (capture one or two 4Ws with breadth and depth) studies. Below, we describe four study designs emphasizing distinct questions and focusing on the capture of distinct 4 W. Data suggests that a majority of the strains of the gut bacterial microbiome of infants come from origins other than the mother. What are the sources of these microbes? This study will focus on the parameter space of ‘who', sampling potential sources of microbial transmission to the infant: the birthing parent, non-birthing parent, individuals present during labor and delivery, home environment (including pets), built environments like hospitals, and dietary sources. From there, one would decide the restricted parameters of the other 4Ws to address from where the infant gut microbes derive: ‘where' (feces, vaginal, skin, or mouth) and which microbial commodity (‘what'). Or one can flip this to address which oral sources across ‘who's' are the source for the infant gut microbiome. In the restricted parameter space of ‘when' would be a decision of the time period and frequency of sampling the defined ‘what'. ‘Who' focused studies will benefit by defining a contact index of the time, distance, and nature of ‘who' - infant interactions (touch, kiss, hold) to associate the strength of such indices with acquisition. This study would focus on a restricted ‘who' (mother and infant), ‘when' (cross-sectional or minimal time series from birth), ‘what' (microbial focus of choice), while emphasizing multiple ‘where's' (oral, multiple skin sites, mother's milk, gut, vagina, from one or both the mother and the infant). If one finds strain-matching from mother's milk or breast skin to the infant gut, one can infer transmission from these origins. Couple this by expanding the ‘when' component, one can find strains that first are detected in the infant (presumably from a non-maternal source) and then in the mother, for reverse acquisition. A quantitative ‘what' (absolute abundance of such a strain) can weigh the probability of transmission of one possible source from the other (i.e., higher abundance of the same strain in the maternal gut versus vagina). Sampling of intermediate route ‘where's' such as detection of strains in an oral and/or gastric samples (i.e., pre-term infants in intensive care units) can define routes of transmission based on absence or presence at each site, such as direct maternal to infant skin without oral-gut seeding versus a route from maternal skin to infant mouth and subsequently the gut via the stomach. What is transmitted at birth? Here, the emphasis is placed on defining as many what's simultaneously as possible, critical to defining ecological mechanisms (‘why') of early-life transmission, across a restricted space of ‘who', where', and ‘when'. For example, a study could seek to define as many ‘what's' as feasible for a given mass/volume of a sample. How many ‘what's' can one ascertain from the same sample will depend on microbial load (bacteria vs. fungi) and detection limits of different methods (extracted DNA amount, cultivation). Key is the choice of the ‘where' (feces vs skin swab) and the technical capacities such as the ability to perform semi-quantitative measurements across the ‘what's' (cross-kingdom) and capabilities to build a metagenomic library from low amounts of DNA. When does transmission occur in early life? The dominant parameter is ‘when' optimizing dense and/or extended periods and focusing on a restricted (or relaxed, such as to capture more ‘who's‘ at focal time points) sampling space for the other 4Ws. A dense time series of a maternal and infant dyad (‘who') of a relaxed ‘where' (either oral, vaginal, or fecal), with focal ‘what' (metagenomics on strain-level) can define when acquisition events from distinct sites may occur. Similarly, coupled with a relaxed ‘who' parameter, with a dominant yet focused ‘when' parameter space, such as by collecting samples at birth and over time from multiple ‘who's' can enable determination of specific windows. In practical terms, conducting large cohorts with extensive metadata collection, comprehensively sampling infants and their household members, plus the environment in a longitudinal design, and finally, sample characterization via multi-omics, is currently unrealistic. This issue is challenging for cohort studies in general, but is pronounced for early-life microbiome studies due to the dynamic nature of microbial communities, which are continuously adapting to the fast-developing human physiology and display a large inter-individual variability. In addition, the specifics of the country where the research takes place will have a major influence on the study design, as one must account for differences in ethical norms, jurisdiction, and geographical proximity/ remoteness that affect logistics. Because of the economic, logistic, and other practical challenges, here we present examples of how researchers have been employing strategies to describe, at least partly, the 4 W parameters (Table 1) and a theoretical approach to study design along a 4 W framework (Box 3). Firstly, one can design nested zoom-in studies within cohorts that have a large number of participants and employ extensive longitudinal sampling47. Depending on the research question, one may apply a cross-sectional, nested case-control, or matched cohort design within a larger population study, using samples from selected timepoints (defining “when”), sources (“who” and “where”), and focusing on particular “what” (e.g., type of microorganisms or their products). From an epidemiological perspective, there will be a difference if the research aim is to study how early life microbial acquisition and transmission are affected by a rare exposure (e.g., formula contamination) or a rare outcome (e.g., necrotizing enterocolitis in premature infants). In the first case, large cohorts are necessary, yet one can opt to characterize only those with the exposure and several controls (matched cohort design). In the case of a rare outcome, one can choose to select and characterize all samples of those with the outcome, matched to one or more controls without the outcome (nested case-control design). The advantages of both approaches compared to characterizing all collected samples include lower cost and comparator groups that are more equal in size, facilitating more statistically robust bioinformatics analyses, as some analytic measures do not perform well if group sizes are unbalanced. Finally, clinical intervention studies, ideally designed as randomized control trials (RCT), can confirm associations between microbial transmission and selected early life factors, as shown in a study using maternal fecal microbiota transplantation to restore the intestinal microbiota of Cesarean section-born infants48. However, RCTs performed in the infant population require extensive ethical approvals compared to adults, which significantly extends the project planning period. Current examples of large population studies that have investigated specific questions related to early life microbial transmission include the Canadian CHILD, Swedish SweMaMi, Danish COPSAC, Dutch Lifelines-NEXT, and Finnish FinnBrain and HELMi studies. Many of these studies are still ongoing and including more 4 W aspects into their designs would help obtain novel insight into early-life microbial transmission and assembly. For empirical evidence, see Table 1 with several case studies that illustrate how the 4 W framework can guide research on human microbiome acquisition and development, when used to identify which of the components of microbial transmission have been closely investigated and which remain largely uncharacterized. Secondly, another study design is to focus only on some of the 4 W parameters, but study them in greater depth. For instance, the BabyBiome project from Oslo sampled 12 infants daily throughout the first year of life, providing high resolution into the dynamics of gut bacterial community over time and describing a strong temporal structure and specific developmental stages of the community maturation49. More of such studies with densely sampled mother-infant pairs would give insight into the “when” of microbial transfer and the effect of the transferred microbes on the colonization dynamics of the infant microbiome. However, this is not feasible for large cohorts. Examples of other focused cohorts include the Diabimmune and DIPP studies50, designed to pinpoint specific microbial profiles in individuals predisposed to immune diseases, and the MicrobeMom study, which studies the transmission of specific bacteria from mother to infant51. Lastly, robust findings on early life microbiome can and will be derived from joint analysis of results from various studies with differing designs52,53,54. This is indeed a good example of the way forward, which lies in collaboration, allowing researchers to connect resources (e.g., economic, know-how, personnel) and combined datasets to tackle research questions. To this end, ensuring the transparent sharing of metagenomic data and metadata is crucial55. Defining the 4Ws of microbiome transmission is crucial to understanding microbiome acquisition in early life. Still, defining ‘what', ‘when', from ‘who' and ‘where' microbes are transmitted does not allow for full comprehension of the acquisition process and aspects of microbial ecology outside of early-life acquisition, such as primary (assembly) and secondary (after perturbation) succession and colonization over time. Indeed, a fifth question on ‘why' is fundamental to understanding which members of the microbiome are acquired and successfully colonize specific habitats in the infant (and beyond). Ultimately, our understanding of ‘why' requires ecological and evolutionary frameworks, and for which we can turn to the four tenets of community ecology and assembly: stochastic processes (dispersal, drift, diversification) and selection (i.e., the niche56,57). To date, most studies have focused on the latter, defining the roles of selection on microbial acquisition and colonization in early life. Many of these selective factors are intimately related to the 4Ws. For example, dietary factors such as milk-derived oligosaccharides, which may selectively feed transmitted microbes in the gut, or the developmental expression of host secreted innate (antimicrobial peptides) and adaptive (antibodies in milk or produced in the gut over time) factors specific for distinct microbes, are examples whereby ‘when' combine with ‘why' to determine transmission. Microbe-microbe interactions, spanning negative (e.g., antibiotics) and positive (e.g., cross-feeding), provide examples where two distinct ‘whats' (i.e., microbial strains) act as selective factors for (co-) acquisition. While selective (or deterministic) pressures define the microbial niche and potential for success of acquisition, stochastic (or probabilistic) processes, i.e., neutral processes, are likely to play instrumental roles in early-life microbiome transmission and assembly. At present, however, as the default or null hypothesis of transmission, dispersal (random movement of microbes across space), drift (random changes in fitness of microbial populations), and diversification (genomic changes), all defined as ‘random', are difficult to assign a weight in early-life microbial transmission. How can the 4 W framework offer new insights into defining stochastic versus selective aspects of microbial ecology, assembly, and succession? Indeed, combining the power of population genetic approaches with microbial source tracking will allow us to test the null hypothesis and the roles of sources and sinks by modeling neutral processes shaping the microbiota acquisition58. Operationally, a broad sampling and quantification across the domains of life of the ‘who' and ‘where' will allow definition of the ‘sources' and ‘sink'. Here also lies a research potential in the use and integration of strain-tagging and controlled transmission experiments31,59. For example, source tracking has been used to identify and categorize transmission events60. The algorithms assume a somewhat unidirectional transmission, where the infant community is the ‘sink' and it acquires microbiomes for a set of ‘source' reference communities. The algorithm can then either predict similarity to a source state or predict the portion of the microbial community contributed by each source61. Modeling of neutral processes during microbiota assembly can reveal the contribution of dispersal and demographic stochasticity, which has been found to explain the prevalence of a majority of infant-colonizing microbes58. With comprehensive sampling across the 4 W and population genetic modeling approaches, we can define if the transmission of a microbe obeys the features of a stochastic process. When this null hypothesis is not met, we can begin to define the contribution of selection (and then what features of microbes are selected) in the assembly and definition of the microbial niche of transmission and successful colonization. For example, by quantifying absolute abundances of a specific microbe (‘what') in a source (e.g., vagina at delivery) and in infant feces (‘who' and ‘where') at birth and over time (‘when'), we could determine if such transmission was related to the microbial abundance in that particular source (stochastic) versus the microbial properties, which facilitate successful acquisition and colonization (selection). Such a 4 W framework can be readily applied to aspects of microbial acquisition outside of the early-life period, such as repopulation of the gut microbiome after antibiotic perturbation. By capturing the 4 W of potential ‘who's' (the person taking the antibiotics and their contacts), ‘where's' (the sources of the microbes to repopulate the gut, i.e., their own oral microbiome or the gut microbiome of their household partner), ‘when' (the timing and relation to the abundance and composition of the focal person's gut microbiome before, during and after use of antibiotics), and ‘what' (strain-level tracking via metagenomics or whole-genome sequencing of cultivated isolates), we can use the 4 W framework to capture the parameters needed to test the null hypothesis and by extension define role of the ecological niche in transmission. Ultimately, a combination of both deterministic and probabilistic modeling will allow a quantitative and predictive understanding of the ‘why' of early-life microbiome transmission. The current lexicon of early-life microbiome acquisition originating from the fields of infectious disease epidemiology and evolutionary biology, and namely vertical and horizontal transmission, often shows limited resolution. To achieve a deep ecological understanding of the early life microbial dynamics, we propose that efforts should be centered on deciphering the what-where-who-when aspects of microbial acquisition. By transitioning from the vertical/ horizontal transmission language toward using precise terminology of the 4 W framework, researchers can systematically examine different components of microbiome acquisition. Currently, metagenomics is the workhorse for describing microbial transmission. But also wider adoption of other methods for tracking microbial cells and their components, such as single cell-based microbial genomics, metabolomics, metaproteomics, and high-throughput culturomics, has the potential to significantly contribute to these goals, when affordable and accessible to non-specialists. Importantly, the interpretation of transmission events carries a large degree of uncertainty and necessitates considering alternative microbial sources. A broader implementation of computational approaches, which can resolve microbial patterns and minimize degrees of freedom in interpreting transmission events, is thus a prerequisite for the study of human microbiome acquisition. Defining which aspects of the 4Ws and how we can feasibly capture these in studies of the early-life human microbiome, and human microbiome at-large, will be instrumental to future-proofing and comprehensively understanding the dynamics and significance of early-life microbial colonization and when, where, and how we should and can intervene for human health. Gensollen, T., Iyer, S. S., Kasper, D. L. & Blumberg, R. S. How colonization by microbiota in early life shapes the immune system. Peppas, I., Ford, A. M., Furness, C. L. & Greaves, M. F. Gut microbiome immaturity and childhood acute lymphoblastic leukaemia. Koch, R. Vorläufige Mitteilungen über die Ergebnisse einer Forschungsreise nach Ostafrika. Berg, G. et al. Microbiome definition re-visited: old concepts and new challenges. Shope, R. E. THE SWINE LUNGWORM AS A RESERVOIR AND INTERMEDIATE HOST FOR SWINE INFLUENZA VIRUS: II. Smith, T. & Kilborne, F. L. Investigations into the Nature, Causation, and Prevention of Texas or Southern Cattle Fever. Cordy, D. R. & Gorham, J. R. The pathology and etiology of salmon disease. Montgomery, E. On a Tickborne Gastroenteritis of sheep and goats occurring in British East Africa. Gross, L. The vertical epidemic of mammary carcinoma in mice; its possible implications for the problem of cancer in general. Gross, L. The “vertical” transmission of mouse mammary carcinoma and chicken leukemia; its possible implications for human pathology. Bevilacqua, G. The Viral Origin of Human Breast Cancer: From the Mouse Mammary Tumor Virus (MMTV) to the Human Betaretrovirus (HBRV). Burmester, B. R. The vertical and horizontal transmission of avian visceral lymphomatosis. Mims, C. A. Immunofluorescence study of the carrier state and mechanism of vertical transmission in lymphocytic choriomeningitis virus infection in mice. Andrewes, C. H. Factors in Virus Evolution. & Axton, J. M. Germ cells colonized by endosymbiotic bacteria. Ebert, D. The Epidemiology and Evolution of Symbionts with Mixed-Mode Transmission. Hosokawa, T., Kikuchi, Y., Nikoh, N., Shimada, M. & Fukatsu, T. Strict host-symbiont cospeciation and reductive genome evolution in insect gut bacteria. The Hologenome Concept: Helpful or Hollow?. de Goffau, M. C. et al. Human placenta has no microbiome but can contain potential pathogens. Questioning the fetal microbiome illustrates pitfalls of low-biomass microbial studies. Rao, C. et al. Multi-kingdom ecological drivers of microbiota assembly in preterm infants. Kaisanlahti, A. et al. Maternal microbiota communicates with the fetus through microbiota-derived extracellular vesicles. Analysis of the human breast milk microbiome and bacterial extracellular vesicles in healthy mothers. Pessa-Morikawa, T. et al. Maternal microbiota-derived metabolic profile in fetal murine intestine, brain and placenta. Parfieniuk, E. et al. Amniotic fluid metabolic fingerprinting indicated metabolites which may play a role in the pathogenesis of foetal Down syndrome - a preliminary report. In utero human intestine harbors unique metabolome, including bacterial metabolites. Stinson, L. F. et al. Human Milk From Atopic Mothers Has Lower Levels of Short Chain Fatty Acids. Wang, Z., Tao, X., Liu, S., Zhao, Y. An Update Review on Listeria Infection in Pregnancy. Luca, C. D. et al. Listeria infection in pregnancy: A review of literature. Zhong, Z. et al. Bifidobacterium animalis subsp. lactis Probio-M8 undergoes host adaptive evolution by glcU mutation and translocates to the infant's gut via oral-/entero-mammary routes through lactation. Influence of maternal oral microbiome on newborn oral microbiome in healthy pregnancies. Patterns of Early-Life Gut Microbial Colonization during Human Immune Development: An Ecological Perspective. Aversa, Z. et al. Association of Infant Antibiotic Exposure With Childhood Health Outcomes. & Bassat, Q. Congenital and perinatally-acquired infections in resource-constrained settings. & Arrieta, M.-C. Inferring early-life host and microbiome functions by mass spectrometry-based metaproteomics and metabolomics. Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. Tkacz, A., Hortala, M. & Poole, P. S. Absolute quantitation of microbiota abundance in environmental samples. Maghini, D. G. et al. Quantifying bias introduced by sample collection in relative and absolute microbiome measurements. Roach, J. et al. Microbiome metabolite quantification methods enabling insights into human health and disease. Busi, S. B. et al. Persistence of birth mode-dependent effects on gut microbiome composition, immune system stimulation and antimicrobial resistance during the first year of life. van den Elsen, L. W. J., Garssen, J., Burcelin, R. & Verhasselt, V. Shaping the Gut Microbiota by Breastfeeding: The Gateway to Allergy Prevention?. Host remodeling of the gut microbiome and metabolic changes during pregnancy. DiGiulio, D. B. et al. Temporal and spatial variation of the human microbiota during pregnancy. Roswall, J. et al. Developmental trajectory of the healthy human gut microbiota during the first 5 years of life. Korpela, K. et al. Maternal Fecal Microbiota Transplantation in Cesarean-Born Infants Rapidly Restores Normal Gut Microbial Development: A Proof-of-Concept Study. & Trosvik, P. Individuality and convergence of the infant gut microbiota during the first year of life. Lamichhane, S. et al. Dynamics of Plasma Lipidome in Progression to Islet Autoimmunity and Type 1 Diabetes - Type 1 Diabetes Prediction and Prevention Study (DIPP). Detailed mapping of Bifidobacterium strain transmission from mother to infant via a dual culture-based and metagenomic approach. Podlesny, D. & Fricke, W. F. Strain inheritance and neonatal gut microbiota development: A meta-analysis. Mancabelli, L. et al. Multi-population cohort meta-analysis of human intestinal microbiota in early life reveals the existence of infant community state types (ICSTs). Ho, N. T. et al. Meta-analysis of effects of exclusive breastfeeding on infant gut microbiota across populations. Reporting guidelines for human microbiome research: the STORMS checklist. Patterns and processes of microbial community assembly. Sprockett, D. D. et al. Microbiota assembly, structure, and dynamics among Tsimane horticulturalists of the Bolivian Amazon. Moore, R. L. et al. Ability of Bifidobacterium breve 702258 to transfer from mother to infant: the MicrobeMom randomized controlled trial. Song, S. J. et al. Naturalization of the microbiota developmental trajectory of Cesarean-born neonates after vaginal seeding. Shenhav, L. et al. FEAST: fast expectation-maximization for microbial source tracking. Ferretti, P. et al. Mother-to-Infant Microbial Transmission from Different Body Sites Shapes the Developing Infant Gut Microbiome. Manara, S. et al. Maternal and food microbial sources shape the infant microbiome of a rural Ethiopian population. Delivery Mode Affects Stability of Early Infant Gut Microbiota. Transmission and dynamics of mother-infant gut viruses during pregnancy and early life. Dubois, L. et al. Paternal and induced gut microbiota seeding complement mother-to-infant transmission. Selma-Royo, M. et al. Birthmode and environment-dependent microbiota transmission dynamics are complemented by breastfeeding during the first year. Early gut mycobiota and mother-offspring transfer. Mobile genetic elements from the maternal microbiome shape infant gut microbial assembly and metabolism. Bright, M. & Bulgheresi, S. A complex journey: transmission of microbial symbionts. & Huttenhower, C. Strain-level epidemiology of microbial communities and the human microbiome. Van Rossum, T., Ferretti, P., Maistrenko, O. M. & Bork, P. Diversity within species: interpreting strains in microbiomes. Ghazi, A. R., Münch, P. C., Chen, D., Jensen, J. & Huttenhower, C. Strain Identification and Quantitative Analysis in Microbial Communities. Altermann, E., Tegetmeyer, H. E. & Chanyi, R. M. The evolution of bacterial genome assemblies - where do we need to go next?. Anyansi, C., Straub, T. J., Manson, A. L., Earl, A. M. & Abeel, T. Computational Methods for Strain-Level Microbial Detection in Colony and Metagenome Sequencing Data. Garud, N. R. & Pollard, K. S. Population Genetics in the Human Microbiome. Infection leaves a genetic and functional mark on the gut population of a commensal bacterium. This project was possible through a grant awarded to V.K.P. by the Center for Advanced Study (CAS) in Oslo, Norway, which funded and hosted the Young CAS research project (Infant Gut Microbiome Acquisition: Off to a Healthy Start) during the 2022/23 academic year. CAS is an independent research foundation funded by the Norwegian Ministry of Education and Research. We also acknowledge and appreciate our partners and children who enabled and enlivened our discussions during the summer of 2023 in Tromsø, Norway. S.R.-N. acknowledges support from the Mathers Foundation, a Career Award for Medical Scientists from the Burroughs Wellcome Fund, a Pew Biomedical Scholarship, and National Institutes of Health (R01DK138023, R01AI158814, R01AI171100, R01AI126915). M.V.-C. acknowledges funding from the Spanish Ministry of Science and Innovation (Knowledge Generation Project PID2022-139328OA-I00), the Spanish Ministry of Universities (Beatriz Galindo Junior fellowship BG22/00172), and from La Caixa Foundation (Junior Leader fellowship 100010434, code LCF/BQ/PI24/12040001). acknowledges funding from the European Union's Horizon 2020 (MSCA-IF postdoctoral grant MicroMi-898088), and Ramón y Cajal grant RYC2023-045839-I, Ministry of Science, Innovation and Universities, the State Research Agency MCIU/AEI/10.13039/501100011033 and FSE+. The Institute of Agrochemistry and Food Technology (IATA-CSIC) is a Center of Excellence Severo Ochoa (CEX2021- 001189-S MCIN/AEI/10.13039/501100011033). is supported by the Netherlands Organization for Scientific Research NWO-VICI grant VI.C.232.074, the NWO Gravitation grant Exposome-NL(024.004.017), the NWO KIC grant KICH1.LWV04.21.01, the ZonMW ME/CFS grant 10091012110015, EU Horizon Europe Program grant INITIALIZE (101094099), and EU Horizon Europe Program grant DarkMatter (ID-DarkMatter-NCD, 101136582). Divisions of Infectious Diseases and Gastroenterology, Department of Pediatrics, Boston Children's Hospital, Boston, MA, USA Department of Microbiology, Harvard Medical School, Boston, MA, USA Broad Institute, Cambridge, MA, USA Department of Epidemiology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA Department of Medical and Life Sciences, Universitat Pompeu Fabra, Barcelona, Spain Centre for Ecological and Evolutionary Synthesis, Department of Biosciences, University of Oslo, Oslo, Norway Department of Biotechnology, Institute of Agrochemistry and Food Technology- Spanish National Research Council (IATA-CSIC), Valencia, Spain Laboratory of Food Chemistry and Toxicology, Faculty of Pharmacy and Food Sciences, University of Valencia, Valencia, Spain Global Health Institute, Department of Family Medicine and Population Health, University of Antwerp, Antwerp, Belgium Centre for Translational Microbiome Research, Department of Microbiology, Tumour and Cell Biology, Karolinska Institutet, Stockholm, Sweden Department of Medical Biology, UiT The Arctic University of Norway, Tromsø, Norway Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar This perspective was conceived during a research sabbatical, with contributions from S.R.N., J.D., M.V.C., H.T.N., M.E.T., A.Z., N.B., and V.K.P. All of these authors contributed to the writing, editing, and critical review of the manuscript. finalized all figures for publication. used his institution's professional BioRender account (Boston Children's Hospital) to illustrate the 4 W study designs in Box 3. 1, 3b were obtained from BioArt, an NIH-funded public repository accessible to J.D. The bacteria's shapes and colors were created by J.D. Correspondence to Seth Rakoff-Nahoum or Veronika Kuchařová Pettersen. The authors declare no competing interests. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Rakoff-Nahoum, S., Debelius, J., Valles-Colomer, M. et al. A reconceptualized framework for human microbiome transmission in early life. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250813083646.htm'>Scientists stunned by colossal formations hidden under the North Sea</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 11:53:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists have discovered hundreds of giant sand bodies beneath the North Sea that appear to defy fundamental geological principles and could have important implications for energy and carbon storage. Using high-resolution 3D seismic (sound wave) imaging, combined with data and rock samples from hundreds of wells, researchers from The University of Manchester in collaboration with industry, identified vast mounds of sand -- some several kilometers wide -- that appear to have sunk downward, displacing older, lighter and softer materials from beneath them. The result is stratigraphic inversion -- a reversal of the usual geological order in which younger rocks are typically deposited on top of older ones on a previously unseen scale. Lead author Professor Mads Huuse from The University of Manchester, said: "This discovery reveals a geological process we haven't seen before on this scale. This displaced the underlying, more porous but rigid, ooze rafts -- composed largely of microscopic marine fossils -- bound by shrinkage cracks, sending them floating upwards. Understanding how these sinkites formed could significantly change how we assess underground reservoirs, sealing, and fluid migration -- all of which are vital for carbon capture and storage." Now the team are busy documenting other examples of this process and assessing how exactly it impacts our understanding of subsurface reservoirs and sealing intervals. Time and yet more research will tell just how widely applicable the model is." This research has been published in the journal Communications Earth & Environment. Scientists Stunned by Alien Mineral That Breaks the Rules of Heat This Tiny Brain Grown in a Lab Could Revolutionize Mental Health Research Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/08/250813083616.htm'>Unprecedented climate shocks are changing the Great Lakes forever</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 10:13:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Heat waves and cold spells are part of life on the Great Lakes. But new research from the University of Michigan shows that is true today in a fundamentally different way than it was even 30 years ago. "The appearance of these extreme temperatures is increasing," said Hazem Abdelhady, a postdoctoral research fellow in the U-M School for Environment and Sustainability, or SEAS. "For most lakes, the appearance is up more than 100% compared with before 1998." That timing is significant because it coincides with the 1997-1998 El Niño, which is one of the strongest on record, he added. "These types of events can have huge impacts on the fishing industry, which is a billion-dollar industry, for example," Abdelhady said. Tribal, recreational and commercial fishing in the Great Lakes account for a total value of more than $7 billion annually, according to the Great Lakes Fishery Commission. While fish can swim to cooler or warmer waters to tolerate gradual temperature changes, the same isn't always true for sudden jumps in either direction, Abdelhady said. Fish eggs are particularly susceptible to abnormal temperature spikes or drops. In studying those events and their connections with global climate phenomena, such as El Niños and La Niñas, we can better prepare to brace for their impact, Abdelhady said. For starters, they're an interconnected system of five lakes. They also contain more than a fifth of the world's fresh surface water. "We can't use the traditional, simpler models for the Great Lakes because they really don't do well," Fujisaki-Manome said. Satellites have enabled routine direct observations of the Great Lakes starting about 45 years ago, Fujisaki-Manome said. But when talking about climate trends and epochs, researchers need to work with longer time periods. "The great thing with this study is we were able to extend that historical period by almost double," Fujisaki-Manome said. "That's why we use modeling a lot of the time. We want to know about the past or the future or a point in space we can't necessarily get to," said coauthor Drew Groneworld, an associate professor in SEAS and a leader of the Global Center for Climate Change and Transboundary Waters. "With the Great Lakes, we have all three of those." David Cannon, an assistant research scientist with CIGRL, and Jia Wang, a climatologist and oceanographer with NOAA's Great Lakes Environmental Research Laboratory, also contributed to the study. The study is a perfect example of how collaborations between universities and government science agencies can create a flow of knowledge that benefits the public and the broader research community, Gronewold said. Note: Content may be edited for style and length. Scientists Stunned by Alien Mineral That Breaks the Rules of Heat This Tiny Brain Grown in a Lab Could Revolutionize Mental Health Research Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41928-025-01422-1'>An integrated microwave neural network for broadband computation and communication</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 09:31:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The development of high-bandwidth applications, including multi-gigabit communication and radar imaging, demands faster processing. However, in the microwave regime, where frequencies exceed clock rates, sampling and computation become challenging. Here we report an integrated microwave neural network for broadband computation and communication. Our microwave neural network operates across tens of gigahertz but is reprogrammed with slow megabits per second control bitstreams. By exploiting strong nonlinearity in coupled microwave oscillations, it expresses its computation in a narrower spectrum, enabling easy read-out. The system searches bit sequences in multi-gigabits per second data and emulates digital functions without custom circuits. It accelerates radio-frequency machine learning by classifying encoding schemes and detecting frequency shifts to track flight trajectories from radar. The microwave neural network is fabricated with standard complementary metal–oxide–semiconductor technology. It occupies a sub-wavelength footprint of 0.088 mm2 on chip and has a sub-200-mW power consumption, supporting integration in a general-purpose analogue processor. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Prices may be subject to local taxes which are calculated during checkout The measured spectral data used for backend training in digital emulation and radar tracking tasks are available via Zenodo at https://zenodo.org/records/14188849 (ref. Code for backend training, radar flight scenario simulation and coupled-mode-theory-based numerical simulations that support the conclusions in this article are available via Zenodo at https://doi.org/10.5281/zenodo.14188849 (ref. Wang, P. et al. Multi-gigabit millimeter wave wireless communications for 5G: from fixed access to cellular networks. Li, K. et al. An integrated CMOS–silicon photonics transmitter with a 112 gigabaud transmission and picojoule per bit energy efficiency. Zhou, Z. et al. Communications with guaranteed bandwidth and low latency using frequency-referenced multiplexing. Li, A. et al. Evaluating modern GPU interconnect: PCIe, NVLink, NV-SLI, NVSwitch and GPUDirect. Gomes, W. et al. Ponte Vecchio: a multi-tile 3D stacked processor for exascale computing. In 2022 IEEE International Solid-State Circuits Conference (ISSCC) 42–44 (IEEE, 2022). Esmaeilzadeh, H. et al. Dark silicon and the end of multicore scaling. In 2011 38th Annual International Symposium on Computer Architecture (ISCA) 365–376 (IEEE, 2011). Gupta, U. et al. Chasing carbon: the elusive environmental footprint of computing. In 2021 IEEE International Symposium on High-Performance Computer Architecture (HPCA) 854–867 (IEEE, 2021). Rupp, K. Microprocessor trend data. Desislavov, R. et al. Trends in AI inference energy consumption: beyond the performance-vs-parameter laws of deep learning. Clark, K. A. et al. Synchronous subnanosecond clock and data recovery for optically switched data centres using clock phase caching. Zhang, W. et al. Edge learning using a fully integrated neuro-inspired memristor chip. Zhou, Y. et al. Computational event-driven vision sensors for in-sensor spiking neural networks. Dong, B. et al. Higher-dimensional processing using a photonic tensor core with continuous-time data. Cai, H. et al. Brain organoid reservoir computing for artificial intelligence. Sludds, A. et al. Delocalized photonic deep learning on the internet's edge. Wright, L. G. et al. Deep physical neural networks trained with backpropagation. Senanian, A. et al. Microwave signal processing using an analogue quantum reservoir computer. Gao, X. et al. Programmable surface plasmonic neural networks for microwave detection and processing. Ross, A. et al. Multilayer spintronic neural networks with radiofrequency connections. Zhu, M. et al. A reconfigurable linear RF analog processor for realizing microwave artificial neural network. Microwave Theory Techn. Feng, H. et al. Integrated lithium niobate microwave photonic processing engine. Xu, Y. et al. Bidirectional interconversion of microwave and light with thin-film lithium niobate. Moy, W. et al. A 1,968-node coupled ring oscillator circuit for combinatorial optimization problem solving. Dutta, S. et al. An Ising Hamiltonian solver based on coupled stochastic phase-transition nano-oscillators. Jouppi, N. et al. In-datacenter performance analysis of a tensor processing unit. 44th Annual International Symposium on Computer Architecture (ISCA) 1–12 (ACM, 2017). Sounas, D. L. et al. Broadband passive isolators based on coupled nonlinear resonances. Govind, B. et al. Ultra-compact quasi-true time delay for boosting wireless channel capacity. The cross-coupled pair—part I [a circuit for all seasons]. IEEE Solid-State Circuits Mag. Babaie, M. et al. A class-F CMOS oscillator. IEEE J. Solid-State Circuits 48, 3120–3133 (2013). Jamali, B. et al. A fully integrated 50-280-GHz frequency comb detector for coherent broadband sensing. Hussein, H. M. E. et al. Passive frequency comb generation at radiofrequency for ranging applications. Shu, H. et al. Microcomb-driven silicon photonic systems. Zhang, M. et al. Broadband electro-optic frequency comb generation in a lithium niobate microring resonator. Haus, H. A. et al. Coupled-mode theory. Pereira, U. et al. Attractor dynamics in networks with learning rules inferred from in vivo data. Parker, T. S. et al. Practical Numerical Algorithms for Chaotic Systems (Springer, 1989). Sarkar, T. et al. An organic artificial spiking neuron for in situ neuromorphic sensing and biointerfacing. Implicit generation and modeling with energy‑based models. Dorkenwald, S. et al. Neuronal wiring diagram of an adult brain. Frans, Y. et al. A 56-Gb/s PAM4 wireline transceiver using a 32-way time-interleaved SAR ADC in 16-nm FinFET. IEEE J. Solid-State Circuits 52, 1101–1110 (2017). Agazzi, O. E. et al. A 90 nm CMOS DSP MLSD transceiver with integrated AFE for electronic dispersion compensation of multimode optical fibers at 10 Gb/s. IEEE J. Solid-State Circuits 43, 2939–2957 (2008). Al-Eryani, J. et al. Fully integrated single-chip 305–375-GHz transceiver with on-chip antennas in SiGe BiCMOS. Lin, Y. et al. An improved neural network pruning technology for automatic modulation classification in edge devices. Wang, Y. et al. MsmcNet: a modular few-shot learning framework for signal modulation classification. Zhang, H. et al. A data preprocessing method for automatic modulation classification based on CNN. Davis III, R. et al. RF-photonic deep learning processor with Shannon-limited data movement. He, K. et al. Deep residual learning for image recognition. IEEE Conference on Computer Vision and Pattern Recognition 770–778 (IEEE, 2016). Pedregosa, F. et al. Scikit-learn: machine learning in Python. Paszke, A. et al. PyTorch: an imperative style, high-performance deep learning library. Loshchilov, I. et al. Decoupled weight decay regularization. In International Conference on Learning Representations 1–10 (ICLR, 2019). Govind, B., Anderson, M., Wu, F., McMahon, P. & Apsel, A. Code and data for ‘An integrated microwave neural network for broadband computation and communication' (data set). and A.A. acknowledge GlobalFoundries and the Defense Advanced Research Projects Agency (DARPA), through its Wideband Adaptive RF Protection (WARP) program, for providing the chip fabrication facility. and A.A. also thank the Cornell NanoScale Facility, a member of the National Nanotechnology Coordinated Infrastructure (NNCI), which is supported by the National Science Foundation (grant no. NNCI-2025233) and where the work was done in part. additionally acknowledge funding from the National Science Foundation (award no. acknowledges support from the Eric and Wendy Schmidt AI in Science Postdoctoral Fellowship, a program of Schmidt Sciences, LLC. We thank A. Aksaray, S. Huang, T. Tapen, F. Monticone, A. Senanian, S. Prabhu and V. Krementski for useful discussions. School of Electrical and Computer Engineering, Cornell University, Ithaca, NY, USA Bala Govind & Alyssa Apsel Kavli Institute at Cornell for Nanoscale Science, Cornell University, Ithaca, NY, USA Bala Govind & Peter L. McMahon School of Applied and Engineering Physics, Cornell University, Ithaca, NY, USA Maxwell G. Anderson, Fan O. Wu & Peter L. McMahon Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the structure of the integrated microwave neural network, designed the CMOS chips and set up the experiment. automated the experiment and performed the machine learning. formulated the coupled model of the parametrically driven oscillators, with F.O.W. producing the numerical results. supervised the machine learning. wrote the paper and produced the figures, with input from all authors. Correspondence to Bala Govind. The authors have filed a provisional US patent application (no. 63/742,208) based on the neural processor presented in this article. Nature Electronics thanks Bowei Dong, Xingjun Wang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, The nonlinear transmission line comprises cascaded π sections of fixed inductive waveguide segments and drive-sensitive nonlinear capacitors. Each resonance absorbs part of the incoming microwave signal and transmits the rest to subsequent segments. The overall nonlinear response results from cumulative reflections back to the RFin port. b, Top view of the layout of the nonlinear waveguide. Nonlinear capacitors are inserted periodically along the pretzel-shaped trace. Their nominal bias points are set through analog voltages. c, In the Silicon-on-Insulator process used here, a single polynomial nonlinear capacitor consists of two pairs of antiparallel diodes. c.i, Compact layout of this component. c.ii, The schematic, wherein the diode pairs are biased at their mid-point. c.iii, SPICE / Spectre-model simulated characterization of the nonlinear capacitor across variation in bias voltages and input RF power shows that by injecting a constant RF input power of -10 dBm, the effective capacitance decreases with increasing bias voltage. c.iv, The effective capacitance is highly sensitive to the input RF drive's power. The capacitance reduces with increasing field strength. c.v, The same nonlinearity, when considering its variance for different frequencies. Here, the capacitance varies with RF voltage as \({{\rm{C}}}_{{\rm{eff}}}({{\rm{V}}}_{{\rm{NL}}},{{\rm{V}}}_{{\rm{in}}})={\rm{a}}-{\rm{b}}{{\rm{V}}}_{{\rm{in}}}+{\rm{c}}{{{\rm{V}}}_{{\rm{in}}}}^{2}-{\rm{d}}{{{\rm{V}}}_{{\rm{in}}}}^{3}+\ldots {\rm{fF}}\) which is well-suited to generating expansive functions for a neural pre-processor. a, Top view of the each tunable linear waveguide (B, C and D) that produces linear modes. Switches S1,2….6 are inserted periodically along the length of the transmission line and make it tunable in length, with subsegments TLine1,2…6. Shorting the line through these switches alters the effective length of the waveguide to support different fundamental frequencies. b, Cross-section of RF-optimized and digital metal layers in the 45 nm Silicon-on-Insulator CMOS metal stack. Here, the top three layers are via'ed together for low-loss transmission. The five metal layers below are used for routing control signals from a Serial-to-Parallel Interface, to the switches, and also via'ed together to form a low-loss return path to the power supply. c, A schematic of the linear waveguide resonators, with options to lengthen or shorten the return path of the microwave signal. In the experiments, however, for simplicity, only the shortest path (configuration with all switches turned on) was used for training in the machine learning tasks. a, The integrated microwave neural network consists of interconnected linear and nonlinear resonators. The linear resonator is designed as a single waveguide with an adjustable length, implemented through a cascade of sub-segments, each referred to as Lin. These sub-segments can be grounded via switches (S1, S2, S3, S4 and S5), which immediately terminate the microwave signal's return path at the first switch that is shorted to ground. In contrast, the nonlinear resonator features a transmission line loaded with polynomially nonlinear capacitors. These capacitors form C-L-C π sections that are coupled by delays. Microwave power from input pads is distributed to these resonators through symmetrically arranged couplers (whose equivalent circuits are marked in purple). The left coupler divides power into two linear waveguides, while the right coupler feeds into a linear waveguide and into a nonlinear waveguide. Saturable gain elements, implemented as cross-coupled transistor pairs, connect the waveguides on opposite sides of the circuit, compensating for losses within the electromagnetic structures. Additionally, a pair of capacitor banks provides a small degree of tunability to the modes supported by the waveguides. Critically, there is parametric coupling between the circuit's upper and lower halves through a pair of slow bitstream-driven switches. b, To simplify the complex circuit, we recognize that since the linear resonators support only a single natural frequency, they can be represented as tank circuits composed of Lj, and Cj. The symmetry in the bottom half of the circuit allows us to approximate the capacitor banks as two evenly split capacitors, contributing to the overall capacitance of the tank circuits. However, the asymmetry in the configuration of the resonators in the upper half does not permit such a simplification. c, To focus on the primary mechanism by which the system's sensitivity to incoming signals is enhanced, we can largely ignore the left half of the circuit and concentrate on the interaction between the nonlinear distributed resonances and the linear resonator on the right half. These components interact only through the inductive path via a coupler and a coupling capacitor between the turns of the coupler. The source of regenerative gain through the cross-coupled pair is retained. For ease of analysis, we represent the parametrically driven switch as a tunable capacitor, which can be toggled between a very small value (open circuit) and a very large value (short circuit). d, The reduced circuit can be represented as an ensemble of coupled modes–a cascade of nonlinear resonators connected to a linear resonator via a parametrically varied switched coupling and a fixed phase delay (through the coupler). These modes interact with the incoming drive (fast Gigabit/sec microwave-speed data), with internal losses being compensated by saturable gain. a, The integrated circuit in the main Article is reduced to a coupled-mode model. This comprises a cascade of nonlinear resonances (waveguide A) coupled both passively and through parametric switching to a linear resonance. The power spectrum and Poincaré map of modes represented by the first nonlinear resonator (vNL,1) are observed. b, The MNN is driven with an external signal at 0.5ωNL,1. For a first set of initial conditions imposed on individual resonators' responses, there exists a regime in which (b.i), pure harmonic oscillation is seen and, (b.ii), for another set of initial conditions, comb-like behavior is produced. These correspond to situations where the Poincaré map shows sparse and organized points. This suggests a quasi-stable dynamic state (b.iii), and another where the two islands reflect more unstable, dynamic responses (b.iv), indicating locally chaotic solutions. c, If, instead, the drive was fed to the MNN at 0.65ωNL,1, different working regimes are triggered by different initial conditions. True comb-like behavior (c.i), can be produced, as evidenced by isolated, longer-memory, coherent solutions on the Poincaré map (c.ii). In another instance, divergent chaotic behavior shown by dense, scattered points (indicative of chaotic dynamics) (c.iv) manifests itself by a less structured spectrum emitted by the nonlinear resonator(s) (c.iii). For all simulations, nonlinear coupling coefficients, gain and decay rates are normalized with respect to ωNL,1. Here, βi,i+1 = 0.02 and \({{{\boldsymbol{\gamma }}}_{{\bf{i}}}}^{{\bf{int}}}=0.03\), with i = 1, 2…7 and G equals 0.2. Since parametric coupling is absent, βpar = 0. In experiments involving Gigabit/sec data and radar returns, the accessible phase space of MNN is constrained by the configurations of incoming drive signals and the parametric bitstreams applied on the switch between the first non-linear mode and a linear mode (a). As a result, the output spectrum is shaped by the constrained, unstable nonlinear dynamics of the system, indicating extreme sensitivity to the modulation of GHz-speed microwave data (1-20 GHz) by slow MHz-speed parameters. For Drive 1, the Poincaré map for slow parameter bitstreams 1 and 2 shows clustered points with a few points that are more spread out (b.iv and b.v), suggesting quasi-periodic dynamics while Bitstream 3 gives tightly clustered points (b.vi), indicating stable, periodic behavior. For Drive 2, under action of all three parametric bitstreams, the formation of ring-like structures indicates quasi-periodic behavior with non-linear dynamics (c.iv, c.v and c.vi). Finally, under Drive 3, bitstreams 1 and 2 induce a combination of ring-like patterns in the Poincaré maps (d.iv and d.v), revealing a tendency towards chaotic behavior, while Bitstream 3 exhibits multiple, structured rings (d.vi), indicating quasi-periodic behavior with complex and unstable dynamics. For all simulations, nonlinear coupling coefficients, gain and decay rates are normalized with respect to ωNL,1. Here, βi,i+1 = 0.02 and \({{{\boldsymbol{\gamma }}}_{{\bf{i}}}}^{{\bf{int}}}=0.03\), with i = 1, 2…7 and G equals 0.2. The CMOS microwave neuron chip is wire-bonded to a Printed Circuit Board, which connects it to external power supplies and bias voltages for the oscillators' core, logic and drivers and Kerr-capacitors' bias. a, A low-speed bit pattern (clocked under 200 MHz) drives switches that establish parametric coupling between oscillators, imparting a shape to the microwave comb neuron. A first pair of probes, forming a Ground-Signal-Ground-Signal-Ground (GSGSG) configuration cyclically transfer high bandwidth 0-12 gigabits per second bitstreams into the chip to interact with the comb. The resulting microwave computations are manifested as steady-state responses in the frequency domain and produce new combs, each unique to the combination of inputs, that is, the fast input bitstreams and slow parameter bitstreams. Output spectra from two ports are read off spectrum analyzers in a small band between 10 and 14 gigahertz, through a second pair of probes and Ground-Signal-Ground-Signal-Ground waveguides. b, The probe-station assembly consists of the CMOS die attached to a breakout PCB, interfaced with millimeter wave probes for input and output data. It also includes a low-speed BNC cable interface for feeding in the parametric bitstream drive. It has a nominal value of 0.5 V, below which it collapses. b, The polynomial nonlinearity is tuned using bias voltages that set the sensitivity of the transmission line's capacitances to incoming microwaves. c, The initial profile of the 12.5 GHz comb can be altered by feeding parameters through a 32-bit sequence, run cyclically at slow speeds under 150 MHz. This parameterized comb is exposed to incoming drive signals and performs computation on them. a, Under nominal biasing conditions (power supply of 0.6 V and VNL = 0.6 V) and without the influence of parametric bits and incoming microwave signals, the measured undisturbed frequency comb is centered at 12.47 GHz and has a constant line spacing of 80 MHz. b, The phase noise is measured for various offsets from the central component. While this version of the chip has slightly higher close-in phase noise than conventional CMOS oscillators, microwave neurons built from this modality are stable and highly input sensitive to drive signals. 1–7, Tables 1–4 and Notes 1–9. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Govind, B., Anderson, M.G., Wu, F.O. et al. An integrated microwave neural network for broadband computation and communication. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03888-0'>A personal health large language model for sleep and fitness coaching</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 09:28:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Although large language models (LLMs) show promise for clinical healthcare applications, their utility for personalized health monitoring using wearable device data remains underexplored. Here we introduce the Personal Health Large Language Model (PH-LLM), designed for applications in sleep and fitness. PH-LLM is a version of the Gemini LLM that was finetuned for text understanding and reasoning when applied to aggregated daily-resolution numerical sensor data. We created three benchmark datasets to assess multiple complementary aspects of sleep and fitness: expert domain knowledge, generation of personalized insights and recommendations and prediction of self-reported sleep quality from longitudinal data. PH-LLM achieved scores that exceeded a sample of human experts on multiple-choice examinations in sleep medicine (79% versus 76%) and fitness (88% versus 71%). In a comprehensive evaluation involving 857 real-world case studies, PH-LLM performed similarly to human experts for fitness-related tasks and improved over the base Gemini model in providing personalized sleep insights. Finally, PH-LLM effectively predicted self-reported sleep quality using a multimodal encoding of wearable sensor data, further demonstrating its ability to effectively contextualize wearable modalities. This work highlights the potential of LLMs to revolutionize personal health monitoring via tailored insights and predictions from wearable data and provides datasets, rubrics and benchmark performance to further accelerate personal health-related LLM research. In medicine specifically, natural language generation may influence clinical practice5, education and research6. When enriched with healthcare-specific data, LLMs achieve impressive performance in medical question answering2,4, electronic health record analysis7, differential diagnosis from medical images8, psychiatric functional assessment9 and delivery of psychological interventions10,11,12. Conventional clinical visits provide invaluable information but offer only periodic assessments of lifestyle features, including sleep, physical activity, stress and cardiometabolic health. These features, which have profound impact on adverse health outcomes, morbidity, disability-adjusted life years and overall health13,14,15,16,17,18, can be measured passively and continuously by wearable devices19. However, these features are not yet widely used in clinical practice nor are incorporated into standard datasets used for medical question answering20,21, likely because these data are typically captured without context, are computationally demanding to store and analyze and can be difficult to interpret. Consequently, general foundation LLMs and even medically tuned LLMs may be unable to use these data effectively to reason about and recommend interventions based on personalized individual health behaviors. Despite these hurdles, initial efforts to integrate sensor data with LLMs have occurred. Personal Health Insights Agent investigated tool use and code generation within an agent framework to accurately compute summary metrics of raw sensor data for use in question answering22. Similarly, Health-LLM evaluated LLMs on rating scale questions in consumer health framed as classification and regression problems23. Although valuable in demonstrating sensor data integration, these approaches do not assess the capability of LLMs to generate the long-form, nuanced and actionable advice characteristic of effective personal health coaching, leaving this critical application underexplored. Here we provide new datasets, evaluation rubrics and benchmarks to assess LLM performance in the sleep and fitness domains. Although wearable devices measure more than just sleep and fitness, we focus on these domains specifically because the accuracy and suitability of commercial wearable measurements for these domains are well studied24,25; they have high engagement among device wearers; and practical recommendations can be offered without giving clinical advice. Additionally, the datasets introduced here can be used to improve performance of other LLMs, and the approach and rubrics are extensible to additional personal health domains. Our main contributions are summarized in Fig. We created Personal Health Large Language Model (PH-LLM), a version of Gemini finetuned to generate both insights about and recommendations to improve sleep and fitness patterns. We evaluated the performance of PH-LLM across three tasks in the sleep and fitness domains that span a breadth of abilities useful for a personal health coach. First, we assessed the amount of expert domain knowledge present in PH-LLM using multiple-choice examinations. Second, we evaluated the model's ability to apply its expert knowledge and interpret aggregated sensor data to provide coaching recommendations from long-form case studies. Finally, for sleep only, we assessed the ability of PH-LLM to predict subjective patient-reported outcomes (PROs) in order to function as a coach that can incorporate individual assessments of sleep quality when generating recommendations. PH-LLM was evaluated on three aspects of personal health: (1) assessing its level of expert knowledge based on certification examination-style MCQs; (2) generating personalized insights and recommendations for user goals in the sleep and fitness domains; and (3) predicting PROs for sleep quality from aggregated daily-resolution numerical sensor data. b, Performance of PH-LLM, as contextualized with the responses of human experts, for professional examinations, coaching recommendations and PROs for sleep quality. ‘Naive performance' is that achieved by a random classifier HR, heart rate; HRV, heart rate variability. The PH-LLM was developed upon Gemini Ultra 1.0 (ref. First, we finetuned the entire model on the task of generating long-form case study responses for sleep and fitness. The training and evaluation data consist of textual representations of demographic data, up to 30 days of daily metrics, aggregated metrics and, for fitness case studies only, individual exercise logs and synthetically generated subjective assessments of workout readiness that mimic a user training log. The target case study responses were created by domain experts in sleep and fitness working in concert with Gemini Ultra 1.0 (Methods and Extended Data Fig. After finetuning on case studies, we finetuned a multimodal adapter for PH-LLM prediction of PROs in sleep disruption and sleep impairment27 from longitudinal passive sensor data containing daily sleep and activity metrics for at least 15 days28 (Methods). After both training stages were completed, we evaluated the performance of PH-LLM across three tasks in the sleep and fitness domains: multiple-choice examinations assessing expert knowledge, coaching recommendations in case study responses and classification performance on PRO prediction. To measure case study response quality, we designed a custom evaluation rubric that quantifies incorporation of user data, appropriate personalization based on user data, use of expert domain knowledge, evidence of confabulations or unwarranted assumptions, potential for harm, readability and overall quality. We compiled multiple-choice examinations in the style of the American Board of Internal Medicine (ABIM) Sleep Medicine Certification examination and the National Strength and Conditioning Association (NSCA) Certified Strength and Conditioning Specialists (CSCS) examination (Methods). PH-LLM correctly answered 79% of sleep medicine and 88% of fitness board examination questions tested (Table 1 and Fig. 1b), comfortably exceeding the approximate grade (70%) to either receive continuing medical education (CME) credit for sleep (https://www.boardvitals.com/sleep-medicine-moc-recertification) or pass the practice examination for fitness (https://www.nsca.com/certification/cscs/certified-strength-and-conditioning-specialist-exam-description), and was competitive with leading external models, slightly worse for sleep but equivalent for fitness questions (Extended Data Fig. In fitness, both PH-LLM and Gemini Ultra 1.0 scored 88%. Receiver operating characteristic (ROC) and precision-recall curves confirm strong model performance in both examinations (Supplementary Fig. Furthermore, despite finetuning PH-LLM for sleep and fitness tasks, its performance on the general medical benchmarks PubMedQA29 and MedQA30 does not degrade (Supplementary Table 1). The sleep medicine question bank contained additional metadata for each question including the distribution of responses from human test takers, enabling performance comparisons stratified by empirical question difficulty. PH-LLM consistently improved slightly over Gemini Ultra 1.0, with larger performance improvement on more difficult questions, suggesting that finetuning on sleep case studies was relevant for these questions (Table 2). Notably, although this is not able to be confirmed, this trend of stronger performance for easier questions suggests that the overall strong performance is not due to memorization or presence of the questions in the model pretraining corpus. To contextualize the performance of PH-LLM, we recruited five sleep medicine experts (average experience: 25 years) with advanced degrees and five professional athletic trainers (average experience: 13.8 years) to take the respective exams. PH-LLM outperformed these expert graders on both question banks (Table 1). In sleep, when stratifying performance by empirical question difficulty for human test takers, we observed that PH-LLM performed similarly to both those test takers and the set of sleep medicine experts whom we recruited (Table 2). We additionally performed ablation studies on the use of self-consistency31 (N = 5 rounds) and chain-of-thought (CoT) prompting32 (prompts in Supplementary Tables 2–4). Self-consistency improved performance on fitness questions for both CoT and non-CoT prompting techniques, whereas the performance from including CoT was mixed (Supplementary Table 5). Few-shot prompting with self-consistency improved performance further in both sleep and fitness examinations (Supplementary Table 6). We next sought to evaluate the model's ability to apply its expert knowledge and interpret sensor data. To do so, we created the first dataset of detailed personal health case studies in sleep and fitness (857 case studies containing 3,271 question–answer pairs) curated by multiple experts in the associated domains. The dataset contains individual wearable sensor data over a multi-week timeframe and corresponding long-form insights and recommendations (Fig. To assess model performance, we compiled responses to each case study by Gemini Ultra 1.0, PH-LLM and a human expert and had all responses rated by human experts. For each case study, the experts considered demographic information and wearable sensor data for a period of up to 30 days over which the device was worn (sections (i)–(iii) of a and b). Both human experts and PH-LLM used these inputs to create domain-specific responses: sleep case studies contained subsections for insights, possible etiologies and recommendations, and fitness case studies contained subsections for training load, sleep, health metrics, readiness assessment and recommendations. Abridged examples of human expert responses are shown in sections (iv) of a and b. c,d, Mean expert ratings across all subsection principles for case study responses generated by Gemini Ultra 1.0, PH-LLM and human experts in the sleep (c) and fitness (d) domains. Error bars represent 95% confidence intervals bootstrapped over 1,000 iterations. ‘*' indicates a statistically significant difference (P < 0.05) between two response types using the two-sided Wilcoxon rank-sum test and multiple hypothesis testing correction. Within each bar, n denotes the number of principle ratings per conversation source, and circles show the proportion of scores at a given Likert rating. bpm, beats per minute; brpm, breaths per minute; hh, hours; HR, heart rate; HRV RMSSD, heart rate variability root mean square of successive differences; mm, minutes. We evaluated the aggregated performance of PH-LLM and human experts on the long-form case study responses, rated by human experts using 15 questions with grading scale 1 (worst) through 5 (best), spanning topics such as using important domain knowledge, correctly referencing relevant user data and avoiding confabulations (Supplementary Table 9). Our rubric closely tracks prior human evaluation of LLMs in healthcare33. To enable inter-rater reliability assessments, we had multiple human experts rate responses (Methods). We conclude moderate inter-rater reliability, based on Gwet's AC2 (refs. 3) and contingency tables of inter-rater agreement (Extended Data Fig. For sleep case studies, PH-LLM responses received an average rating of 4.61 versus 4.75 for human expert responses (P = 3.3 × 10−11, N ≥ 2,606, Z (test statistic) = −6.63, r (effect size) = −0.08; Fig. Finetuning PH-LLM on sleep case studies significantly improved its overall performance in this task as compared to Gemini Ultra 1.0 (average rating of 4.51 for Gemini Ultra 1.0 versus 4.61 for PH-LLM, P = 4.0 × 10−6, N ≥ 2,603, Z = 4.63, r = 0.06). For fitness case studies, PH-LLM aggregate performance was not statistically different from expert performance (P = 0.48, N ≥ 3,335, Z = −0.70, r = −0.01; Fig. Gemini Ultra 1.0 responses also were not statistically different from human expert responses (P = 0.92, N ≥ 3,161, Z = −0.10, r = −0.00). Because the case studies contain multiple sections, we also analyzed ratings for each section separately (Fig. For sleep case studies, finetuning PH-LLM improved its ability to provide insights and etiologies (P = 6.65 × 10−7, N ≥ 800, Z = 5.18, r = 0.11 and P = 2.46 × 10−3, N ≥ 801, Z = 3.15, r = 0.07, respectively). Recommendations showed no statistically significant difference between Gemini Ultra 1.0 and PH-LLM (P = 0.45, N ≥ 801, Z = 0.76, r = 0.02). We further analyzed ratings by various rubric questions (Extended Data Fig. Finetuning PH-LLM improved its ability to reference important domain knowledge (P = 4.47 × 10−5, N ≥ 201, Z = 4.44, r = 0.19), important interpretations (P = 4.47 × 10−5, N ≥ 201, Z = 4.48, r = 0.19), important user data (P = 5.21 × 10−8, N ≥ 201, Z = 5.91, r = 0.26) and no unimportant interpretations (P = 4.31 × 10−2, N ≥ 201, Z = 2.53, r = 0.11). For fitness case studies, PH-LLM had similar performance as both human experts and Gemini Ultra 1.0 (no statistically significant difference observed, N ≥ 768) on three out of four sections (Fig. Training load was the only section in which responses from human experts and Gemini Ultra 1.0 were rated higher than those from PH-LLM (P = 0.01, N ≥ 768, Z = 3.02, r = 0.07). When analyzing ratings by rubric questions, we observed no statistically significant differences in ratings between PH-LLM and human experts (Extended Data Fig. We present examples of PH-LLM model responses for sleep (Supplementary Tables 10–12) and fitness (Supplementary Tables 13–17). To assess robustness to missing data, we examined performance on case studies stratified by data availability. We observed no bias in rating distributions based on data availability (Supplementary Fig. We observed that PH-LLM appropriately adapted its responses depending on the length of the context available (Supplementary Fig. Additionally, PH-LLM effectively used different parts of the input data for distinct aspects of its response. For example, modifying training load statistics while keeping workout logs constant led the model to adjust its response accordingly (Supplementary Table 19). Similarly, the model adapted its output when faced with missing information, such as the absence of today's sleep metrics (Supplementary Table 20). Finally, we note that some individuals were the subject of multiple fitness case studies from distinct days. Expert assessments of workout readiness can appropriately vary from one day to the next (Supplementary Fig. 5), and the relative performance of PH-LLM and human experts is consistent at a variety of target date overlaps allowed in the holdout set (Supplementary Fig. Expert human grading of case studies is laborious and time-consuming. Our best AutoEval models ranked case study response sources more accurately than an LLM not explicitly finetuned for AutoEval tasks (Supplementary Table 21) and similarly to human experts (Extended Data Fig. 6) in a fraction of the time (Supplementary Table 22). When evaluated using the best AutoEval model, PH-LLM scored significantly higher than leading external models on sleep insights and etiologies sections and competitively on the other sections (Extended Data Fig. Additionally, we used our AutoEval raters to explore the effect of finetuning data size on model performance. We observed that sleep insights and etiologies sections showed statistically significant rating improvements when using the whole dataset compared to training with only 25% of samples (Extended Data Fig. To assess whether PH-LLM could infer subjective user experience to better tailor coaching advice, we evaluated its ability to predict PROs in sleep disturbance and sleep impairment from daily-resolution numerical sensor data. To do so, we used the Google Digital Wellbeing (DWB) study dataset, which recruited thousands of individuals to use wearable devices and take validated survey instruments that measure wellbeing over a 4-week span28. We filtered the dataset to 4,163 individuals who had both daily measurements of 20 wearable sensor features for at least 15 days and complete responses for the sleep disturbance and impairment surveys at study completion (Methods). The 16 questions measure related but distinct sleep outcomes (Fig. We then investigated sensor features for possible confounding and observed similar distributions of sensor readings across devices and participant compliance (Supplementary Fig. Next, we computed the importance of individual sensor features for predicting PROs by fitting linear models that used all sensor features as input to predict each survey question independently. We observed that no single feature dominated predictive power across all PROs and that the overall predictive signal was spread across many sensors (Fig. However, we note that interpreting the relationships between sensor features and outcomes is complicated because sensor features may capture demographic effects. b, Feature importance for sensor features predicting survey responses in a linear regression model. c, AUROC for the performance of PH-LLM with adapter, zero-shot and few-shot prompting approaches when predicting binary outcomes derived from survey responses in the test set (n = 833). Data are presented as mean AUROC over 100 bootstrapping iterations, and error bars show 95% confidence intervals. Outcomes for which the confidence intervals of the difference in AUROC between PH-LLM with adapter and both zero- and few-shot both exclude 0 in 100 paired bootstrapping iterations are annotated with ‘*'. d, AUPRC for the performance of PH-LLM with adapter, zero-shot and few-shot prompting approaches when predicting binary outcomes derived from survey responses in the test set (n = 833). Survey response names are mapped to their corresponding questions in Supplementary Tables 39 and 40. Data are presented as mean AUPRC over 100 bootstrapping iterations, and error bars show 95% confidence intervals. Outcomes for which the confidence intervals of the difference in AUPRC between PH-LLM with adapter and both zero-shot and few-shot both exclude 0 in 100 paired bootstrapping iterations are annotated with ‘*'. To enable PH-LLM to predict PROs from sensor features, we trained a multilayer perceptron (MLP) adapter to map summary statistics of the 20 sensor features into PH-LLM's latent token space (Methods). We compared the area under the receiver operating characteristic curve (AUROC) and area under the precision-recall curve (AUPRC) for each binary trait with zero-shot and few-shot text approaches (Fig. However, PH-LLM using the adapter significantly outperformed both prompt-based approaches in terms of AUROC and AUPRC (Extended Data Fig. This performance increase is likely due to adapter-enabled LLMs being able to capture more signal from the training set than zero-shot and few-shot prompting, which see a very limited amount of training data37. To assess how well PH-LLM performed compared to traditional machine learning approaches, we fit independent logistic regression models for each binary trait using the same encoded vector input. We found no statistically significant differences in performance between PH-LLM and logistic regression models for AUROC and only one statistically significant difference out of 16 outcomes for AUPRC (Extended Data Fig. Furthermore, we hypothesized that, in a low-data regime, LLMs might leverage their prior knowledge to outperform logistic regression. However, our comparison of PH-LLM and logistic regression revealed no such advantage (Supplementary Fig. 8), likely due to the limited representation of wearable sensor data in LLM training corpora. Separately, to model nonlinear effects not captured by these logistic regression models, we also trained a convolutional neural network (CNN) on the full time-series sensor data to predict the same outcomes. We performed multiple ablation analyses to assess the effects of the sensor data context window length (number of days of sensor data), number of training data examples and data augmentation on PRO prediction performance. We observed that using only 5 days of sensor data led to lower AUROC and AUPRC metrics, whereas using 10 days or 15 days gave similar performance (Supplementary Fig. Increasing the context window beyond 10 days is likely to provide little value because Patient-Reported Outcomes Measurement Information System (PROMIS) sleep survey questions refer specifically to the past 7 days. We also observed a significant performance improvement when increasing the number of training samples from 25% to 50% and a smaller improvement when increasing the number of training samples from 50% to 100% (Supplementary Fig. 10), indicating that PH-LLM's performance may continue to improve with additional training data. Finally, we retrained PH-LLM after augmenting the adapter inputs with various levels of zero-mean Gaussian noise. Although the AUROC scores do not show consistent trends, the AUPRC scores are generally higher—albeit with wide error bar estimates—from the model augmented with Gaussian noise with zero mean and 0.001 s.d. 11), suggesting that data augmentation may modestly improve performance further. We developed an LLM finetuned from Gemini (PH-LLM) to perform a variety of tasks relevant to setting and achieving personal health goals in sleep and fitness. Our study showed that PH-LLM can answer technical multiple-choice questions (MCQs) in these domains with accuracy exceeding multiple experts. Furthermore, we showed that PH-LLM can integrate passively acquired objective data from wearable devices into personalized insights, potential causes for observed behaviors and recommendations to improve sleep hygiene and fitness outcomes. After finetuning from Gemini Ultra 1.0, which itself displays aggregate performance approaching that of experts in fitness, PH-LLM demonstrated significantly improved use of domain knowledge and personalization of relevant user information for sleep insights. Finally, we trained PH-LLM to use a multimodal encoder that natively integrates daily summaries of time-series health sensor data to predict subjective self-reported outcomes in sleep and achieved performance on par with specialized models to predict the same outcomes. Multiple-choice examinations are commonly used benchmarks to assess general domain knowledge of LLMs1,2,3,4,26. Strong performance on MCQs can be considered a necessary, but not sufficient38, requirement for practical utility of LLMs in realistic settings. Through exhaustive evaluation of PH-LLM and comparison to multiple human experts and other models, we showed that, although leading external general-purpose LLMs exhibited slightly stronger performance on general sleep knowledge, PH-LLM comfortably demonstrates expert-level performance in both sleep and fitness MCQs, and finetuning toward these tasks did not degrade its performance on general medical benchmarks. Open-ended long-form case studies represent key use cases for applications of LLMs to personal health features on wearable devices. In the present study, we created 857 case studies to assess sleep quality and fitness readiness for a workout and coupled the case studies with rigorous evaluation rubrics. We observed that the average performance over all case study responses was very high for all human experts, Gemini Ultra 1.0 and PH-LLM, underscoring the strong knowledge and reasoning capabilities of the Gemini model family. However, finetuning Gemini Ultra 1.0 to create PH-LLM produced significant improvements in the model's ability to provide insights and etiologies for sleep. For example, PH-LLM is better at using relevant domain knowledge, providing important interpretations of data and using important user data. The relatively lower performance gain observed for sleep recommendations may be driven in part by both the increased ambiguity of the task, which led to wider variety in training example content, and the increased challenge of requiring consistency across multiple model generations to provide an accurate recommendation that referenced real data. In contrast, for fitness, both PH-LLM and Gemini Ultra 1.0 performed similarly. This pattern may be attributable to several factors. First, the base model's stronger performance in fitness could reflect well-optimized prompting, minimizing potential gains from finetuning. Second, differences in training data could have influenced the performance. For example, our use of a standardized format for sleep analysis may have resulted in more specific training data, whereas the absence of a standardized guideline for fitness may have led to more generalized data that are more difficult to differentiate. Third, Gemini Ultra 1.0 may possess stronger baseline knowledge in fitness compared to sleep medicine. PH-LLM was outperformed by Gemini Ultra 1.0 (and human experts) for a single section: the training load section of fitness case studies. This may be partially explained by the data generation process, in which multiple models were used to create candidate case study responses. Expert rater comments suggest that some lower-quality statements with incorrect use of domain and user data may be present in the training data, particularly with respect to rest periods. Moreover, because the fitness case studies incorporate sleep quality as one input, there is potential for further improvements in fitness by integrating full sleep case studies into the sleep section of fitness for a more detailed view of an individual's rest status. Additionally, we developed methods for automated evaluation of case studies and demonstrated their ability to be used as scalable proxy measures for expert human evaluation of LLM performance. Our best AutoEval models ranked study responses similarly to human experts, and these models obtained similar agreement measures with expert raters when compared using inter-rater concordance metrics. We used the best AutoEval models to compare PH-LLM to public external models and perform dataset ablation analyses. The analyses underscored the value of our expert-curated case studies, as PH-LLM significantly outperformed on sleep insights and etiologies, and performance degraded when trained on subsets of the data. Beyond expert interpretation of wearable features, understanding the subjective experience of an individual can be useful for determining a comprehensive and personalized action plan. Indeed, subjective PROs are receiving greater attention in health management39,40,41. However, predicting subjective PROs from sensor information alone is difficult, with specialized models developed specifically for those tasks only reaching AUCs in the 0.55–0.75 range. We trained a multimodal adapter to enable PH-LLM to predict subjective PROs from aggregates of daily sensor information and showed that the native multimodal integration was necessary and sufficient to achieve performance on par with specialized discriminative models. Although for the standalone task of PRO prediction the simpler logistic regression model is much more efficient and interpretable, enabling PH-LLM to predict subjective wellbeing allows for integrating this capability directly into user-facing interactions. Additionally, by building the capability directly into PH-LLM, it may benefit from positive transfer learning toward additional out-of-distribution outcomes for which it was not trained, as shown previously8. First, expert evaluation of long-form text is inherently subjective for open-ended tasks such as coaching42, and the observed distribution of case study rubric ratings was skewed high, making differentiation across models and expert responses challenging. Although some case study sections and evaluation rubric principles showed significant differentiation, further training of expert raters to increase inter-rater reliability or adjudicating existing responses could increase signal strength of model performance. Second, to minimize inter-rater variability, most case studies had a single expert rate all candidate responses. Although this made direct comparison of candidate responses straightforward, it introduced the potential for experts to identify expert versus model responses based on style or other non-material factors and, thus, introduce conscious or unconscious biases into ratings. Third, we observed that, despite improvements in referencing and integrating user data into insights, confabulations and incorrect referencing of user data still occasionally occurred. Preventing these issues will be critical to ensure the safe and effective deployment of these technologies into user-facing features. Fourth, the case studies were sampled broadly across demographics (sleep) or to identify common patterns in active individuals (fitness), and the DWB study recruited samples by self-selection into the study, so they may not be representative samples of the population (Supplementary Table 25) nor exhaustively explore the sleep and fitness concerns and unmeasured contextual factors affecting individuals. In particular, the case studies and broad Fitbit research population do not acquire race or ethnicity information, so these samples may be a biased subset of the US population. Furthermore, we sampled to enrich patterns of activity in fitness case studies that enriched those case studies for male participants and individuals aged 30–59 years. Each of these distributional biases may affect generalizability of the results presented here for the entire US or global population. Fifth, we only considered textual representations of case study data input, owing to the absence of persisted raw time-series in the dataset. Exploring the potentially richer temporal signals encoded in raw sensor data is an important area for future work. Sixth, our exploration of multimodal encoding of sensor data similarly explored a small fraction of modeling possibilities owing to the relatively small dataset with nearly complete sensor data and paired outcome data. Further exploration of self-supervised pretraining on raw waveforms and granularly aggregated sensor features may yield richer representations of individuals useful for personal health outcome predictions43 that expand beyond sleep metrics. Seventh, by using daily sensor measurements and imposing selection criteria for case studies, we largely avoided challenges of missing data that wearable sensor studies frequently face. Although our analyses stratified by data missingness and sensor data ablations suggest that PH-LLM is robust to missingness, further exploration is warranted. Eighth, a primary overarching goal for developing models specific to personal health is to improve long-term health outcomes through effective behavior change and maintenance of healthy habits. Blinded user preference studies could provide important evidence of the model's effectiveness and performance relative to expert coaching in the studied domains. Furthermore, we selected the three categories of tasks described herein as they represent important capabilities, but we recognize that there are many other dimensions that one could evaluate, and these remain important areas for future work. Finally, although the performance of PH-LLM on the tasks presented here is encouraging, we caution that much work remains to be done to ensure that LLMs are reliable, safe and equitable in personal health applications. Further reducing confabulations, considering an individual's unique health circumstances not captured by sensor information alone and ensuring alignment of the training data with real-world distributions are a subset of important research areas that warrants further attention. Despite these limitations, we have demonstrated here that Gemini-based models contain substantial health knowledge, and we can effectively finetune Gemini Ultra 1.0 to improve performance across multiple outcomes relevant for personal health. We provide a suite of benchmark tasks relevant for personal health coaching, contextualize performance through exhaustive comparison to human experts and highlight strengths and gaps in current LLMs for acting as coaching agents. The results from this study represent, in our estimation, an important step toward LLMs that deliver personalized information and recommendations that support individuals to achieve their health goals. Owing to the absence of clearly defined language and multimodal datasets in the domain of personal health, we created datasets and associated tasks to evaluate different capabilities of PH-LLM. Sleep medicine examinations: we compiled 629 MCQs from BoardVitals (https://www.boardvitals.com/) sleep medicine board review question banks. We used the American Medical Association (AMA) Physician's Recognition Award (PRA) ‘Category 1 - Sleep Medicine' question bank, which emulates examination content for the American Board of Internal Medicine (ABIM) Sleep Medicine Certification Exam. We also used the Sleep Medicine Maintenance of Certification (MOC) Exam and Longitudinal Knowledge Assessment Review question bank, which emulate examination content for the ABIM Sleep Medicine MOC Exam and the ABIM Longitudinal Knowledge Assessment. A subset of 204 questions was selected for evaluation by sleep experts. Sampling was performed randomly to ensure similar distributions to the entire set of MCQs when stratified based on medical content categories (https://www.abim.org/Media/aypkdxpi/sleep-medicine.pdf) and their difficulty levels. Fitness examinations: we compiled 99 MCQs sourced from multiple question banks that emulate examination content for the CSCS examination preparation book that the NSCA provides (https://www.nsca.com/certification/cscs/certified-strength-and-conditioning-specialist-exam-description). We used the test examination questions from the NSCA-CSCS textbook Essentials of Strength Training and Conditioning. Accuracy was used as the metric to evaluate the performance of our model in professional examinations, in line with previous work evaluating MedMCQA44. Each question presents up to five possible answers, with a single correct answer. MCQs were not used for model training. Each of the 857 case studies was sampled from anonymized Fitbit production data from individuals who provided consent for research purposes. Case studies were designed to interpret a range of physiological sensor information toward deriving insights, potential causes or recommendations for future behaviors. Creation of case studies consisted of four main parts, each developed in close collaboration with domain experts: identifying the primary goals of the case studies, selecting case study input features, sampling individuals for inclusion in case studies and creating gold standard case study responses. This categorization was based on each expert's general availability; ‘primary' contributors had more involvement and higher workload volumes. The sleep and fitness verticals were each led by a clinical expert in their respective fields. The clinical lead oversaw case study development and provided feedback and quality control to the set of domain experts. We recruited six domain experts, all of whom possessed advanced degrees (MD, DO or PsyD) in sleep medicine and professional sleep medicine work experience ranging from 4 years to 46 years. All experts were trained to read and interpret wearable data and map outputs to their corresponding sleep medicine literature counterparts. Primary goals: the sleep case studies aimed to enhance understanding of sleep patterns, identify causes of irregular sleep and offer actionable recommendations based on these findings, with the ultimate objective of enhancing the sleep quality of the individual. Input feature selection: each sleep case study incorporated demographic information (age and self-reported gender), daily sleep metrics (for example, bedtimes, wake times and sleep stage durations) and aggregated sleep statistics (for example, average bedtime) (Fig. For some metrics, such as bedtime, the metrics were computed separately for all days, weekdays only and weekends only to understand weekday versus weekend patterns (Supplementary Table 27). Sample selection: the dataset was sampled to enrich for non-missingness and achieve a representative group across age and self-reported gender (Extended Data Fig. Each case study (N = 507) was created from a unique individual. Gold standard response creation: experts were instructed to interpret the case study input data to craft guidance in the second-person narrative, fostering a direct and personalized dialogue with the user. The case study input data were presented to the experts in both graphical and tabular formats for ease of analysis (Fig. Insights: implicitly, this section answered the question of ‘What are some sleep-related insights based on my data?' The sleep medicine expert examined the data and provided an interpretation of whether a data point might represent an atypical sleep pattern. The experts were asked to systematically review each case to provide a holistic assessment of the user's sleep patterns. To do so, Fitbit sleep metrics were assessed according to the validated Routine, Sleep Quality, Alertness, Timing, Efficiency and Duration (RU-SATED) framework to generate sleep insights45. The experts generally considered the contribution of circadian rhythm, homeostatic drive, psychophysiologic hyperarousal and extrinsic factors and indicated their likelihood. Recommendations: implicitly, this section answered the question of ‘What can I do to improve my sleep?' The experts were asked to provide personalized recommendations to the individual that can help them improve their sleep by addressing potential causes identified in the etiology section. The experts were instructed to create recommendations that are Specific, Measurable, Achievable, Relevant and Time-bound (SMART)46. The seven fitness experts all possessed advanced degrees (MS, MA, MEd or DAT) related to the athletic training field and professional athletic training work experience ranging from 4 years to 25 years. Primary goals: the fitness case studies were designed to provide a comprehensive analysis of an individual's training load, sleep patterns and health metrics, with the ultimate objective of synthesizing the metrics into a data-driven assessment of the extent to which the individual is prepared for physical activity today, and to provide associated recommendations. Input feature selection: similar to the sleep case studies, fitness case study inputs spanned demographic information (age, self-reported gender, height, weight and body mass index (BMI)), wearable sensor daily values for cardiovascular training load, sleep and health metrics and aggregated statistics (Fig. Sample selection: we enriched for samples containing sufficient activity for interesting training readiness analysis by considering individuals who logged more than 30 lifetime runs and had, within a contiguous 21-day window, mean active zone minutes of at least 45 minutes and, at most, 5 days of missing data for step counts, sleep, heart rate variability and resting heart rates. We further filtered to person-days with at least two logged exercises in the past 28 days and that contained noticeable recent changes in heart rate variability, resting heart rate, sleep, active zone minutes or run activity. Each case study (N = 350) targets a unique person-day combination, with N = 58 unique individuals in the dataset. The distribution of the number of available days for fitness case studies is presented in Supplementary Fig. Gold standard response creation: experts were instructed to formulate insights, assessments and recommendations in the second-person narrative. The case study input data were presented to the experts in tabular, text and graphical formats. Demographics: the experts considered the demographics data (age, self-reported gender, height, weight and BMI) and commented on whether any precautions should be taken when recommending a fitness program. Training load: the experts were provided with a detailed table capturing daily metrics over the past 30 days, including day of the week, date, minutes spent in fat-burn, cardio and peak zones, training impulse (TRIMP), which is a training load measure derived from heart rate and exercise duration, and number of steps (Supplementary Table 28). Daily TRIMP values were additionally visualized in a bar plot (Fig. Aggregate statistics were also provided, including means; ranges; acute TRIMP (7-day total training load), a measure of acute training load; chronic TRIMP (28-day average acute training load), a measure of chronic training load; and acute:chronic workload ratio (ACWR), aiding the assessment of training stress. Experts were also given aggregated metrics, including means, standard deviations and z-scores, indicating the difference in metrics between the most recent 3 days and the past 28 days to identify recent trends (Supplementary Table 31). Select sleep metrics were additionally visualized (Fig. Health metrics: daily data on day of the week, date, resting heart rate, heart rate variability and respiratory rate were provided to assess recovery and stress (Supplementary Tables 32 and 33), along with graphical representations (Fig. The experts were also given aggregate metrics, such as means, standard deviations, ranges and z-scores, indicating the difference in metrics between the most recent day and the past 28 days (Supplementary Table 34), to gauge changes over time and assess recovery and stress levels effectively. Assessment and recommendation: the experts integrated multiple data sources to evaluate workout readiness and provide personalized recommendations. They synthesized the most important insights from the previous sections along with simulated user feedback about their subjective state. The synthetically generated user feedback included a qualitative description of subjective readiness (for example, ‘feeling fatigued') and muscle soreness (for example, ‘manageable soreness') (Supplementary Tables 35 and 36). The synthetically generated user feedback was generated using Gemini 1.0 Pro (prompts are shown in Supplementary Tables 37 and 38). Based on all available information, the experts assessed the individual's readiness to work out on a scale of 1 to 5 and provided specific fitness recommendations (Fig. Sleep and fitness case studies were collected from distinct sets of individuals. For both the sleep and fitness verticals, we generated two sets of data: a dataset used for model training, validation and testing and a holdout dataset that was used only for final evaluation of the model by experts (Extended Data Fig. To generate the dataset used for training, validation and testing, we first prompted the Gemini family of models with the data for each section in order to generate baseline model (Gemini Ultra 1.0) responses (Extended Data Fig. This was done to ensure a clearer comparison between experts and the model during final evaluation. In total, we created 507 case studies for sleep (457 case studies for the training, validation and test set and 50 case studies for the holdout set, all drawn from unique individuals) and 350 case studies for fitness (300 case studies drawn from 48 individuals for the training, validation and test set and 50 case studies drawn from 11 individuals for the holdout set). Data splits for fitness, which sample multiple case studies from single individuals, were created to ensure that person-months were unique between the train and holdout test set. Although a single individual contributes case studies with unique person-months to both the train and holdout splits (N = 5 and N = 3, respectively), excluding these holdout samples did not substantially impact holdout performance (Supplementary Fig. Within each fitness split, case studies from a given individual were selected from distinct days such that target person-days were unique across samples. To evaluate the ability of PH-LLM to predict PROs from longitudinal passive sensor data, we used a large institutional review board (IRB)-approved study in which wearable data were collected for a population of 10,099 consented individuals for a 4-week period28. At both intake and completion, participants were asked to complete two surveys: the PROMIS short-form Sleep Disruption and Sleep Impairment surveys27, each containing eight questions with answers on a five-point Likert scale (Supplementary Tables 39 and 40). The study thus linked individuals' perceived sleep quality and its impact on their functioning with longitudinal observed physiological (for example, heart rate and sleep duration) and behavioral (activity) measurements. We used the completion survey responses for all 16 questions as the basis for prediction and, for each question, defined a binary outcome that compared the highest answer (for example, ‘strongly agree') against all others (Supplementary Fig. Features used to predict each binary outcome include 20 time-varying wearable measurements (Supplementary Table 41), each of which was collected from study participants over a 4-week span. Although most individuals have sensor data for over 21 days, distributions are heavily left-skewed (Supplementary Fig. To obtain data points with complete labels and standardized input features, we filtered the dataset to retain only individuals who have all completion survey responses and at least 15 days of sensor data (N = 4,163 individuals). For each individual, we kept only the last 15 (possibly non-contiguous) days of sensor data, resulting in a 20 × 15 matrix that represents the wearable sensor data for each research participant over 15 days as well as a vector of 16 binary values as the prediction targets. We performed sensor data quality filtering by replacing values more than 4 s.d. from the population median with their nearest non-outlier values. Each sample corresponds to a unique individual and contains labels for 16 binary outcomes, resulting in 46,624 (=2,914 × 16) training, 6,656 validation and 13,328 testing data points with binary labels. To train PH-LLM from the most capable base model, we performed automated evaluation of several Gemini candidate model sizes and a medical LLM on the professional examination questions. Because Gemini Ultra 1.0 was the most accurate Gemini family base model on professional examinations, suggesting that it has appropriate domain knowledge in the areas of sleep and fitness, we explored the performance of this model on case studies. We prompted Gemini Ultra 1.0 by summarizing guidelines given to the experts for dataset creation. For example, the sleep experts generally were asked to follow the RU-SATED format45 to generate sleep insights. To encourage Gemini Ultra 1.0 to generate high-quality case study responses, we similarly prompted it to follow the RU-SATED format and provided an explanation of what metrics should be used to assess each dimension (see Supplementary Tables 42–49 for details). We note that each case study consisted of multiple sections containing textual representations of aggregated sensor data that represent different queries and responses: three sections for sleep case studies (insights, etiology and recommendations) and five sections for fitness case studies (demographics, training load, sleep, health metrics and the assessment). Because each section represented a different aspect of the case study, we developed prompts specifically for each section (Supplementary Tables 42–44 for sleep; Supplementary Tables 45–49 for fitness). We represented aggregated sensor data as text to mimic observations of how the experts used and interpreted the case study data while creating gold standard responses. For sections that synthesized results from previous sections—that is, the etiology and recommendation sections in sleep case studies and the assessment section in fitness case studies—we substituted the model answers from previous sections into the prompt (Supplementary Table 49). We finetuned Gemini Ultra 1.0 on the case studies to create PH-LLM. We used the case studies from the training, validation and test sets for model training and selection (457 case studies from 457 individuals for sleep and 300 case studies from 48 individuals for fitness). For each of the sleep and fitness domains, we randomly split the case studies into separate training, validation and test splits using a 70:15:15 ratio. We used the same text prompts that were given to the baseline model to form prompt–response pairs for model tuning. Because each section was treated as a separate example, this resulted in 1,371 prompt–response pairs for sleep and 1,500 prompt–response pairs for fitness across the training, validation and test sets (Extended Data Fig. Typically, LLMs are trained on mixtures of tasks47. Here, we finetuned the model on a 1:1 mixture of sleep and fitness prompt–response pairs. Within the fitness prompt–response pairs, we upsampled higher-quality case studies by a 2:1 ratio, where higher-quality case studies were defined as those that underwent additional rounds of quality control by the fitness experts. The Gemini Ultra 1.0 model, which is based on a transformer decoder26, was finetuned for 1,500 steps with a global batch size of four using linear warmup over 50 steps and cosine decay. Finetuning was performed by minimizing the cross-entropy loss between predicted and target token distributions. For our final model candidate, we chose the first checkpoint (step = 550) after the model had been trained for at least one epoch. At this point, the validation loss had started to plateau and stabilize (Supplementary Fig. Using PH-LLM finetuned on case studies, we further finetuned an adapter for PH-LLM to predict PROs from wearable data. To do so, we used a subset of the Google DWB study dataset28, which collected both wearable data and self-reported outcomes, and we trained the model following the methodology developed in HeLM37. The wearable data of each individual consisted of 20 device measurements over 15 days and were passed to the model through both textual prompt and the adapter. Then, for each individual, we computed the mean and variance of z-scored sensor values across 15 days for each measurement, yielding a standardized and aggregated 20 × 2 sensor data matrix. This matrix was projected into four ‘soft tokens' in the PH-LLM token embedding space by an MLP adapter with an input layer of size 40; three hidden layers of sizes 1,024, 4,096 and 1,024, each followed by rectified linear unit (ReLU) activation function; and an output layer of size 4 × 14,336 = 57,344 where 14,336 is the token embedding size of PH-LLM. Finally, these four ‘soft tokens' were passed to PH-LLM as a prefix to the textual prompt. The adapter was trained to minimize cross-entropy loss between the predicted and actual binary outcome value while keeping PH-LLM weights frozen. The adapter was trained for, at most, 30,000 steps using a global batch size of eight, a learning rate of 1 × 10−4 with linear warmup over 50 steps and cosine decay. We compared the predictions on the test dataset from PH-LLM with adapter to those from text-only PH-LLM using either zero-shot or few-shot prompting. For zero-shot, the prompt was almost the same as that for PH-LLM with adapter except that there was no adapter-based soft token prefix and the prompt also listed the variance across 15 days for each raw measurement value. For all three models, positive and negative outcomes were scored by computing the log likelihood for each outcome. We also trained a CNN using the z-scored wearable data of shape 20 × 15 (whose sensor-based means and variances are the input to the PH-LLM adapter and logistic regression models) as input. The CNN sequentially consists of two convolutional layers with kernel shapes (15, 1) and (1, 1), respectively, each followed by a batch normalization layer and a dropout layer with dropout probability 0.2, two feedforward layers with sizes 64 and 1 and a sigmoid activation at the end. Both convolutional layers have 16 filters, one-step strides and a ReLU activation. We evaluated model performance on the professional examination dataset using a combination of CoT prompting32 and self-consistency31. For our primary results, we generated five reasoning paths per question and selected the most common response as the final answer. For questions that did not have a unique modal response, we selected the answer option with highest log probability as the final answer. To assess the ability of PH-LLM to differentiate correct from incorrect answers based on confidence, we used a log probability analysis. For each multiple-choice examination question, we computed the model's log probability for every answer option and applied softmax normalization to obtain a probability distribution over all answer options. For metric calculation, each answer option was treated as an individual prediction, comparing the model's normalized log probability against a binary label (1 for a correct response, 0 otherwise). We then computed micro-averaged AUROC and micro-averaged AUPRC across the sleep and fitness examinations independently. Although evaluation against MCQs and PROs can be performed by comparing model predictions to gold standard structured responses and numerical values, respectively, the case studies involve longer-form outputs. To evaluate these model responses, the domain experts (including all experts involved in creating the case study responses) were asked to evaluate three responses written to each case study: one by Gemini Ultra 1.0, one by PH-LLM and one by a domain expert (Extended Data Fig. Of the 100 holdout case studies, 94 had at least one single expert rate all three responses (45/50 in sleep and 49/50 in fitness); the remainder had multiple experts contribute ratings. Each domain expert was assigned evaluations randomly to case studies for which they did not write the expert response and were blinded to the source of each response. The domain experts evaluated each case study response based on a custom rubric that quantifies incorporation of user data, appropriate personalization based on user data, use of expert domain knowledge, evidence of confabulations or unwarranted assumptions, potential for harm, readability and overall quality (Supplementary Table 9). Holdout case studies were fully distributed across the primary group of experts. To assess inter-rater agreement and analyze differences between primary and secondary rater groups, we introduced overlap across case study rating assignments within each vertical, resulting in 78–1,428 paired ratings within a subset of raters. To evaluate inter-rater reliability, we employed several established metrics: raw counts of agreement, pairwise Spearman's rank correlation, Kendall's coefficient of concordance (Kendall's W), weighted Cohen's kappa and Gwet's AC2. Spearman's correlation and Kendall's coefficient of concordance are metrics based on comparing ranks of expert ratings. These may be more conservative in the presence of many ties, as in our data with many ratings clustered around 4 and 5 (Extended Data Fig. The metric ranges from −1 to 1 with 0 indicating that the agreement among raters is similar to chance. Weighted Cohen's kappa can exhibit paradoxical behavior (‘Kappa Paradox'), underestimating the true extent of agreement between raters34 in imbalanced datasets such as ours. Gwet's AC1, and its weighted extension designed for ordinal data, Gwet's AC2, were designed to deal with class imbalance while adjusting for chance agreement34,35. Qualitative trends for all metrics were consistent across rater pairs in both verticals (Supplementary Figs. Although expert grading of case study responses was our primary mechanism for assessing model performance, AutoEval allows us to obtain a quick, although potentially less accurate, signal by using secondary models to perform this grading task48. AutoEval dataset: while exploring different modeling mechanisms, we performed an initial round of expert grading using the rubrics (Supplementary Table 9) and procedure described above for 50 expert-generated case studies from each vertical across three response sources: experts, an untuned Gemini Ultra 1.0 model and a finetuned Gemini Pro 1.0 model. We split these studies into vertical-specific training and validation splits containing approximately 80% (N = 38) and 20% (N = 12) of case studies, respectively. Splits were structured such that samples rated by a given expert were evenly distributed between sets. AutoEval examples: using the rated case studies, we constructed LLM prompts and targets. Each target was the expert-generated rating followed by the rating's Likert option text description (for example, for a ‘No incorrect domain knowledge' principle rating of 5, the target is ‘5. AutoEval model training: we created AutoEval rater models by finetuning the smaller Gemini Pro 1.0 model26 using LoRA49 across a variety of vertical-specific data mixtures, including all ratings for a vertical and all ratings from a single rater. All AutoEval modeling experiments used a fixed set of hyperparameters, varying only the training data mixture: a LoRA rank of 4 on attention heads, random Gaussian initialization for LoRA's A matrix and zero initialization for LoRA's B matrix. Models were tuned to minimize the cross-entropy loss between predicted and target token distributions using the Adafactor optimizer50 with a constant learning rate of 2 × 10−5, a β1 of 0.95, a decay of 0.995 and a gradient norm clipping threshold of 1.0. None of warmup, weight decay or quantization was used. Checkpoints were taken four times per epoch, and the best checkpoint was selected by minimizing validation set log perplexity. The total number of steps varied per model because each training mixture contained a different number of samples depending on the vertical and rater (Supplementary Table 52). An untuned Gemini Pro 1.0 model served as a baseline. The other three AutoEval candidates were trained identically using different data mixtures: ‘All' contained all ratings in the vertical; ‘High variance' contained ratings from only the expert rater with highest rating variance (raters ‘Sleep Primary C' and ‘Fitness Primary C'); and ‘Low variance' contained ratings from only the expert rater with lowest rating variance (raters ‘Sleep Primary D' and ‘Fitness Primary B'). We selected the best AutoEval model for each vertical using a combination of log perplexity loss and Spearman's rank correlation between predictions and the ground truth ratings in the validation dataset (referred to as ‘best AutoEval' models). AutoEval inference: given case study candidate responses, we used the Likert scoring procedure described above to automatically rate outputs across case study sections and evaluation principles with our best AutoEval models. Confidence intervals (95%) were determined via bootstrapping with 1,000 iterations. Statistical significance of expert ratings was determined using a two-sided Wilcoxon rank-sum test with false discovery rate (FDR) (Benjamini–Hochberg) correction when multiple sections or multiple evaluation principles were analyzed together. All individuals who fulfilled the criteria for authorship required by Nature Portfolio journals have been included as authors. This work includes findings that are locally relevant, which have been determined in collaboration with local partners. Roles and responsibilities were agreed among collaborators ahead of the research. This research does not result in stigmatization, incrimination, discrimination or personal risk to participants. Local and regional research relevant to our study was taken into account in citations. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All input data and associated expert responses for all 857 case studies in sleep and fitness used for training and evaluation, along with all model responses to the 100 holdout case studies, are stored in JSON format and are publicly available at https://github.com/google-health/consumer-health-research/blob/main/phllm/data/sleep_case_studies.all.jsonl and https://github.com/google-health/consumer-health-research/blob/main/phllm/data/fitness_case_studies.all.jsonl, respectively. All input and target prediction data, which include bucketized age, daily sensor readings for 20 sensors over 15 days and 16 binary survey responses, for all 4,163 samples in the PRO task are stored in JSON format. To ensure privacy of participants, data used for the PRO task are available to approved researchers for non-commercial research after registration and attestation of a data use agreement. Source data are provided with this paper. All code provided with this paper is available in the phllm folder of https://github.com/google-health/consumer-health-research. This repository includes code for interaction with the case study datasets, automatic evaluation of model responses to case study inputs, query and evaluation of MCQ examinations, a reference implementation of the multimodal adapter for predicting PROs and analysis and data visualization tools. The text LLM code underlying PH-LLM is Gemini Ultra 1.0. Katz, D. M., Bommarito, M. J., Gao, S. & Arredondo, P. GPT-4 passes the bar exam. Singhal, K. et al. Toward expert-level medical question answering with large language models. Nori, H. et al. Can generalist foundation models outcompete special-purpose tuning? Towards accurate differential diagnosis with large language models. Galatzer-Levy, I., McDuff, D. & Malgaroli, M. The capability of large language models to measure and differentiate psychiatric conditions through O-shot learning. Sharma, A. et al. Cognitive reframing of negative thoughts through human-language model interaction. 61st Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers) (eds Rogers, A. et al.) 9977–10000 (Association for Computational Linguistics, 2023). Sharma, A., Rushton, K., Lin, I. W., Nguyen, T. & Althoff, T. Facilitating self-guided mental health interventions through human-language model interaction: a case study of cognitive restructuring. Lin, I. et al. IMBUE: Improving interpersonal effectiveness through simulation and just-in-time feedback with human-language model interaction. Wannamethee, S. G., Shaper, A. G. & Walker, M. Changes in physical activity, mortality, and incidence of coronary heart disease in older men. Öhlin, B., Nilsson, P., Nilsson, J.-Å & Berglund, G. Chronic psychosocial stress predicts long-term cardiovascular morbidity and mortality in middle-aged men. Zheng, N. S. et al. Sleep patterns and risk of chronic disease as measured by long-term monitoring with commercial wearable devices in the All of Us Research Program. Kao, C.-K. & Liebovitz, D. M. Consumer mobile health apps: current state, barriers, and future directions. Gordon, W. J., Landman, A., Zhang, H. & Bates, D. W. Beyond validation: getting health apps into clinical practice. Transforming wearable data into health insights using large language model agents. Kim, Y., Xu, X., McDuff, D., Breazeal, C. & Park, H. W. Health-LLM: large language models for health prediction via wearable sensor data. De Zambotti, M. et al. State of the science and recommendations for using wearable technology in sleep and circadian research. Evaluating the validity of current mainstream wearable devices in fitness tracking under various physical activities: comparative study. Gemini: a family of highly capable multimodal models. Yu, L. et al. Development of short forms from the PROMIS™ sleep disturbance and sleep-related impairment item banks. The Google Health Digital Well-Being Study: protocol for a digital device use and well-being study. Jin, Q., Dhingra, B., Liu, Z., Cohen, W. & Lu, X. PubMedQA: a dataset for biomedical research question answering. 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP) (eds Inui, K. et al.) 2567–2577 (Association for Computational Linguistics, 2019). Jin, D. et al. What disease does this patient have? A large-scale open domain question answering dataset from medical exams. Wang, X. et al. Self-consistency improves chain of thought reasoning in language models. In Eleventh International Conference on Learning Representations (ICLR, 2023); https://openreview.net/forum?id=1PL1NIMMrw Wei, J. et al. Chain-of-thought prompting elicits reasoning in large language models. 36th International Conference on Neural Information Processing Systems 24824–24837 (Curran Associates, 2022). Tam, T. Y. C. et al. A framework for human evaluation of large language models in healthcare derived from literature review. Gwet, K. L. Computing inter-rater reliability and its variance in the presence of high agreement. Gwet, K. L. Handbook of Inter-Rater Reliability: The Definitive Guide to Measuring the Extent of Agreement Among Raters (Advanced Analytics, LLC, 2014). Zheng, L. et al. Judging LLM-as-a-judge with MT-Bench and Chatbot Arena. Belyaeva, A. et al. Multimodal LLMs for health grounded in individual-specific data. In Machine Learning for Multimodal Healthcare Data: First International Workshop https://doi.org/10.1007/978-3-031-47679-2_7 (Springer, 2023). Fleming, S. L. et al. MedAlign: a clinician-generated dataset for instruction following with electronic medical records. Marshall, S., Haywood, K. & Fitzpatrick, R. Impact of patient-reported outcome measures on routine practice: a structured review. Dobrozsi, S. & Panepinto, J. Patient-reported outcomes in clinical practice. Liang, P. et al. Holistic evaluation of language models. In Transactions on Machine Learning Research (TMLR, 2023); https://openreview.net/forum?id=iO4LZibEqW Large-scale training of foundation models for wearable biosignals. In Twelfth International Conference on Learning Representations (ICLR, 2024); https://openreview.net/forum?id=pC3WJHf51j Singhal, K. et al. Large language models encode clinical knowledge. way to write managements's goals and objectives. Wei, J. et al. Finetuned language models are zero-shot learners. In International Conference on Learning Representations (ICLR, 2022); https://openreview.net/forum?id=gEZrGCozdqR Can large language models be an alternative to human evaluations? 61st Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers) (eds Rogers, A. et al.) 15607–15631 (Association for Computational Linguistics, 2023). Hu, E. J. et al. LoRA: low-rank adaptation of large language models. In International Conference on Learning Representations (ICLR, 2022); https://openreview.net/forum?id=nZeVKeeFYf9 Shazeer, N. & Stern, M. Adafactor: adaptive learning rates with sublinear memory cost. 35th International Conference on Machine Learning (eds Dy, J. We thank the sleep and fitness experts who developed case study responses and evaluated candidate model responses for their dedication, effort and detailed feedback on multiple model iterations. Contributing sleep experts include B. Graef, T. Wong, T. Dang, S. Gorovoy, N. Krishnamurthy and M. Jonelis. Contributing fitness experts include J. Spraggins, A. Hetrick, J. Hannon, M. Knight, N. Dozier, L. Grissom and J. Leach. We also thank S. Lachgar, L. Winer, M. Shiels, L. Gardner, N. Tal, A. Um'rani, O. Adewunmi and A. Mathur for their valuable insights, technical support and feedback during our research. These authors contributed equally: Justin Khasentino, Anastasiya Belyaeva, Xin Liu, Zhun Yang, Nicholas A. Furlotte. These authors jointly supervised this work: Daniel McDuff, Cory Y. McLean. Justin Khasentino, Anastasiya Belyaeva, Xin Liu, Zhun Yang, Nicholas A. Furlotte, Chace Lee, Erik Schenck, Yojan Patel, Jian Cui, Logan Douglas Schneider, Robby Bryant, Ryan G. Gomes, Allen Jiang, Roy Lee, Yun Liu, Javier Perez, Jameson K. Rogers, Cathy Speed, Shyam Tailor, Megan Walker, Jeffrey Yu, Tim Althoff, Conor Heneghan, John Hernandez, Mark Malhotra, Leor Stern, Yossi Matias, Greg S. Corrado, Shwetak Patel, Shravya Shetty, Jiening Zhan, Shruthi Prabhakara, Daniel McDuff & Cory Y. McLean had major involvement in designing various aspects of the study. had major involvement in data acquisition and ingestion. generated synthetic data and initial case studies. and S. Prabhakara provided organizational support for the project. All authors accept responsibility for the accuracy and integrity of all aspects of the research. Correspondence to Anastasiya Belyaeva, Xin Liu, Daniel McDuff or Cory Y. McLean. This study was funded by Google LLC. All authors are employees of Alphabet and may own stock as part of the standard compensation package. Nature Medicine thanks Jessilyn Dunn, Yonghui Wu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Michael Basson, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Case studies were selected from anonymized Fitbit production data from individuals who provided consent for research purposes. Two sets of case studies were generated: one set for model training, validation, and testing and a separate holdout set for final evaluation. To facilitate rapid development of high-quality answers, the train/validation/test set of case studies had candidate responses generated by Gemini, which were then edited and rewritten by domain experts. To enable comparison of human and model-derived responses, the holdout set had responses written solely by the domain experts. b, For model training, each case study was split into multiple prompt/answer pairs based on how many sections the case study had: N=3 for sleep with insights, etiology, and recommendations sections, N=5 for fitness with demographics, training load, sleep metrics, health metrics, and assessment sections (Methods). Gemini Ultra 1.0 underwent full fine-tuning using those examples to create PH-LLM. For each case study in the holdout set, one or more experts who did not write the corresponding expert response graded the candidate responses (expert-written response, Gemini Ultra 1.0 response, and PH-LLM response, with 94 of the 100 case studies having all three candidate responses graded by a single expert. Data are presented as mean categorical accuracy over 1,000 bootstrapping iterations and error bars show 95% confidence intervals. Metrics were computed using all ratings for each principle and section across case studies rated by more than one rater in the sleep (a) and fitness (b) domains. The number of overlapping ratings is denoted by n. Mean metrics and 95% confidence intervals derived from 1,000 bootstrapping iterations are reported for each pair. Counts are aggregated across all case studies, sections, and principles for each case study for which multiple ratings are available in the sleep (a) and fitness (b) domains. Mean ratings given by experts for different case study evaluation principles across all sections in the sleep (a) and fitness (b) domains. ‘*' indicates a statistically significant difference (p < 0.05) using the two-sided Wilcoxon rank-sum test and multiple hypothesis testing correction. Error bars represent 95% confidence intervals bootstrapped over 1,000 iterations. Within each bar, n denotes the number of principle ratings per conversation source and circles show the proportion of scores at a given Likert rating. Counts are aggregated across all case studies, sections, and principles for each case study for which at least one rating from the AutoEval training rater is available in the sleep (a) and fitness (b) domains. Blue, the primary expert rater versus other raters. Green, the AutoEval model trained on primary expert ratings versus other raters. Mean ratings were generated using our best AutoEval models for the holdout case study subsections in the sleep (a) and fitness (b) domains. Within each section, a ‘*' indicates a statistically significant difference (p <0.05) from the top rated response type using the two-sided Wilcoxon rank-sum test and multiple hypothesis testing correction. Error bars represent 95% confidence intervals bootstrapped over 1,000 iterations. Within each bar, n denotes the number of principle ratings per conversation source and circles show the proportion of scores at a given Likert rating. Mean ratings were generated using our best AutoEval models for the holdout case study subsections in the sleep (a) and fitness (b) domains. ‘PH-LLM' denotes standard performance while ‘Subsampled 25%' and ‘Subsampled 50%' denote responses from models trained on 25% and 50% of the training dataset, respectively. ‘Gemini Ultra' denotes untuned baseline performance (that is, Gemini Ultra 1.0 trained on 0% of the training dataset). Within each section, a ‘*' indicates a statistically significant difference (p <0.05) from the top rated response type using the two-sided Wilcoxon rank-sum test and multiple hypothesis testing correction. Error bars represent 95% confidence intervals bootstrapped over 1,000 iterations. Within each bar, n denotes the number of principle ratings per conversation source and circles show the proportion of scores at a given Likert rating. We compared the ability of PH-LLM with and without a multimodal adapter, logistic regression, and a convolutional neural network (CNN) to infer subjective patient-reported outcomes in the test set (n=833). a, Area under the receiver operating characteristic curve (AUROC). b, Area under the precision-recall curve (AUPRC). Data are presented as mean performance measures over 100 bootstrapping iterations and error bars show 95% confidence intervals. Statistical source data for panels c and d: human expert ratings across conversation sources. Figure panels a and b demonstrate case study data and do not include any statistical source data. Statistical source data: human expert ratings across conversation sources. Statistical source data: human expert ratings across conversation sources. Statistical source data: human expert ratings across conversation sources. Statistical source data: human expert and AutoEval ratings across conversation sources. Statistical source data: AutoEval ratings across conversation sources. Statistical source data: AutoEval ratings across subsampled PH-LLM conversation sources. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Khasentino, J., Belyaeva, A., Liu, X. et al. A personal health large language model for sleep and fitness coaching. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/08/250813083614.htm'>Ancient predators and giant amphibians found in african fossil treasure trove</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-08-14 06:56:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An international team of paleontologists has spent more than 15 years excavating and studying fossils from Africa to expand our understanding of the Permian, a period of Earth's history that began 299 million years ago and ended 252 million years ago with our planet's largest and most devastating mass extinction. The fossils we have collected in Tanzania and Zambia will give us a more global perspective on this unprecedented period in our planet's natural history." These include saber-toothed predators, burrowing foragers and a large, salamander-like creature. Most were discovered by team members on multiple, month-long excavation trips to the region over the past 17 years. Others were analyses of specimens dug up decades prior that had been stored in museum collections. "These parts of Zambia and Tanzania contain absolutely beautiful fossils from the Permian," said Sidor. "They are giving us an unprecedented view of life on land leading up to the mass extinction." The researchers trekked between field sites miles apart to collect fossils. All fossils collected by the team will be returned to Tanzania and Zambia after researchers have completed their analyses. During this time, animal life -- which evolved first in Earth's oceans -- began to colonize land and complex terrestrial ecosystems developed. By the Permian, a diverse array of amphibian and reptile-like creatures roamed environments ranging from early forests to arid valleys. But beginning in the 1930s, paleontologists realized that basins in Tanzania and Zambia contain fossil records of this time range that are almost as pristine as the Karoo's. These new papers look further back into the Permian. "The number of specimens we've found in Zambia and Tanzania is so high and their condition is so exquisite that we can make species-level comparisons to what paleontologists have found in South Africa," said Sidor. "I know of no better place on Earth for getting sufficient detail of this time period to make such detailed conclusions and comparisons." These small, burrowing, reptile-like herbivores first evolved in the mid-Permian. The team's findings also include several new species of large, saber-toothed predators called gorgonopsians, as well as a new species of temnospondyl, a large salamander-like amphibian. "We can now compare two different geographic regions of Pangea and see what was going on both before and after the end-Permian mass extinction," said Sidor. Note: Content may be edited for style and length. Scientists Stunned by Alien Mineral That Breaks the Rules of Heat This Slick New Material Could Change Cookware Forever This Tiny Brain Grown in a Lab Could Revolutionize Mental Health Research Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            