
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-03</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00333-1'>Over one-third of cancer cases are preventable, massive study finds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 16:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Gemma Conroy is a freelance science journalist based in Mexico City. Tobacco smoking is a leading cause of preventable cancer cases.Credit: Jorge Sanz/SOPA Images/LightRocket/Getty Overall, tobacco smoking was the leading contributor to worldwide cancer cases, followed by infections and drinking alcohol. Cancer continues to be a leading cause of illness and death worldwide, with cases expected to rise over the next decades if current trends continue. Previous studies2 have estimated that around 44% of global cancer deaths can be attributed to avoidable or controllable causes. Estimates of preventability have mainly focused on the number of deaths rather than cases and have mostly investigated a single risk factor, says Fink. To address this gap, Fink and her colleagues examined global case data from 2022 for 36 different cancers across 185 countries. The study included 30 modifiable risk factors that are well-established causes of cancer — such as tobacco smoking, alcohol consumption and infections. The researchers combined this information with data from 2012 that captured people's exposure to each risk factor. Fink and her colleagues then estimated the proportion of cases that were directly linked to each risk factor. In 2022, there were a total of 18.7 million new cancer cases worldwide. Roughly 38% — or 7.1 million — of these cases could be attributed to avoidable causes. Globally, tobacco smoking was the leading contributor, accounting for around 15% of preventable cases. Lung, stomach and cervical cancers made up nearly half of all preventable cancer cases. Cancer might protect against Alzheimer's — this protein helps explain why Critical role for a high-plasticity cell state in lung cancer The surprisingly big health benefits of just a little exercise 48 hours without lungs: artificial organ kept man alive until transplant An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-026-04219-7'>Global and regional cancer burden attributable to modifiable risk factors to inform prevention</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 16:07:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Cancer remains a leading cause of morbidity globally, largely attributable to modifiable risks. We estimated the 2022 global and national cancer burden attributable to 30 such factors, including tobacco smoking, alcohol consumption, high body mass index, insufficient physical activity, smokeless tobacco and areca nut, suboptimal breastfeeding, air pollution, ultraviolet radiation, 9 infectious agents and 13 occupational exposures, to inform prevention efforts. Using GLOBOCAN data for 36 cancer sites in 185 countries, we applied prevalence data from around 2012 to reflect exposure−cancer latency and estimated Levin-based or Miettinen-based population-attributable fractions (PAFs) or direct estimates where applicable. Combined PAFs accounting for overlapping exposures were derived by cancer, sex, country and region. In 2022, an estimated 7.1 million of 18.7 million new cancer cases (37.8%) were attributable to 30 modifiable risk factors—2.7 million (29.7%) in women and 4.3 million (45.4%) in men. The proportion of preventable cancers ranged from 24.6% to 38.2% in women and from 28.1% to 57.2% in men across regions. Smoking (15.1%), infections (10.2%) and alcohol consumption (3.2%) were the leading contributors to cancer burden. Lung, stomach and cervical cancers represented nearly half of preventable cancers. Strengthening efforts to reduce modifiable exposures remains central to global cancer prevention. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout All data used in this analysis are publicly available. GLOBOCAN estimates for cancer incidence data are available from the International Agency for Research on Cancer's Global Cancer Observatory (https://gco.iarc.fr/today/), and CI5 population-based cancer registry data are available from the International Agency for Research on Cancer's Cancer Incidence in Five Continents Volume XII: https://ci5.iarc.fr/ci5-xii/. Tobacco smoking prevalence is available on the Institute for Health Metrics and Evaluation Global Burden of Disease Study 2019 Data Input Sources Tool: https://ghdx.healthdata.org/gbd-2019/data-input-sources. Alcohol consumption prevalence data are available on the World Health Organization's Global Health Observatory: https://www.who.int/data/gho/data/indicators/indicator-details/GHO/average-daily-intake-in-grams-of-alcohol–population-15. High-BMI prevalence data are available on the Noncommunicable Disease Risk Factor Collaboration's website: https://www.ncdrisc.org/data-downloads-adiposity.html. Insufficient physical activity prevalence data are available on the World Health Organization's Global Health Observatory: https://www.who.int/data/gho/data/indicators/indicator-details/GHO/prevalence-of-insufficient-physical-activity-among-adults-aged-18-years-(crude-estimate)-(-). Suboptimal breastfeeding prevalence data are available on the UNICEF website: https://data.unicef.org/topic/nutrition/breastfeeding/#data. Concentrations of fine particulate matter (PM2.5) are available on the World Health Organization's Global Health Observatory: https://www.who.int/data/gho/data/indicators/indicator-details/GHO/concentrations-of-fine-particulate-matter-(pm2-5). Prevalence data for occupational exposures are available from the World Health Organization and International Labour Organization's report: https://www.ilo.org/sites/default/files/wcmsp5/groups/public/@ed_dialogue/@lab_admin/documents/publication/wcms_819788.pdf. A sample code for PAF estimation of high BMI as well as the code used for the combination and harmonization of all PAF files of the individual risk factors are available at https://github.com/hfink1/Causes_of_cancer.git. Bray, F. et al. Global cancer statistics 2022: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. Tran, K. B. et al. The global burden of cancer attributable to risk factors, 2010−19: a systematic analysis for the Global Burden of Disease Study 2019. & Colditz, G. A. Modifiable risk factors for cancer. B. et al. Estimation of cancer deaths averted from prevention, screening, and treatment efforts, 1975−2020. GBD 2023 Cancer Collaborators. The global, regional, and national burden of cancer, 1990−2023, with forecasts to 2050: a systematic analysis for the Global Burden of Disease Study 2023. de Martel, C., Georges, D., Bray, F., Ferlay, J. & Clifford, G. M. Global burden of cancer attributable to infections in 2018: a worldwide incidence analysis. Plummer, M. et al. Global burden of cancers attributable to infections in 2012: a synthetic analysis. Whiteman, D. C. & Wilson, L. F. The fractions of cancer attributable to modifiable factors: a global review. Azevedo, E. S. G. et al. The fraction of cancer attributable to ways of life, infections, occupation, and environmental agents in Brazil in 2020. Brown, K. F. et al. The fraction of cancer attributable to modifiable risk factors in England, Wales, Scotland, Northern Ireland, and the United Kingdom in 2015. Islami, F. et al. Cancer deaths and cases attributable to lifestyle factors and infections in China, 2013. Islami, F. et al. Proportion and number of cancer cases and deaths attributable to potentially modifiable risk factors in the United States, 2019. CA Cancer J. Clin. Kulhánová, I. et al. Proportion of cancers attributable to major lifestyle and environmental risk factors in the Eastern Mediterranean region. Nemati, S. et al. Population attributable proportion and number of cancer cases attributed to potentially modifiable risk factors in Iran in 2020. Soerjomataram, I. et al. Cancers related to lifestyle and environmental factors in France in 2015. Teh, H. S. & Woon, Y. L. Burden of cancers attributable to modifiable risk factors in Malaysia. BMC Public Health 21, 410 (2021). Arnold, M. et al. Progress in cancer survival, mortality, and incidence in seven high-income countries 1995−2014 (ICBP SURVMARK-2): a population-based study. Shelton, J. et al. 25 year trends in cancer incidence and mortality among adults aged 35−69 years in the UK, 1993−2018: retrospective secondary analysis. WHO global report on trends in prevalence of tobacco use 2000–2030. https://www.who.int/publications/i/item/9789240088283 (World Health Organization, 2024). Manthey, J. et al. Global alcohol exposure between 1990 and 2017 and forecasts until 2030: a modelling study. Arndt, M. B. et al. Global, regional, and national progress towards the 2030 global nutrition targets and forecasts to 2050: a systematic analysis for the Global Burden of Disease Study 2021. WHO Report on the Global Tobacco Epidemic, 2023: Protect People From Tobacco Smoke (World Health Organization, 2023); https://www.who.int/publications/i/item/9789240077164 Bauman, A. et al. Impact of the first year of the ‘This girl can' physical activity and sport mass media campaign in Australia. BMC Public Health 23, 333 (2023). & Horton, R. A Lancet Commission on women and cancer. WHO Framework Convention on Tobacco Control. Gender-responsive Tobacco Control: Evidence and Options for Policies and Programmes (World Health Organization, 2018); https://fctc.who.int/resources/publications/m/item/gender-responsive-tobacco-control-evidence-and-options-for-policies-and-programmes Forman, D. et al. Global burden of human papillomavirus and related diseases. Lei, J. et al. HPV vaccination and the risk of invasive cervical cancer. Chen, Y. C. et al. Global prevalence of Helicobacter pylori Infection and incidence of gastric cancer between 1980 and 2022. Malaty, H. M. et al. Age at acquisition of Helicobacter pylori infection: a follow-up study from infancy to adulthood. Bellack, N. R., Koehoorn, M. W., MacNab, Y. C. & Morshed, M. G. A conceptual model of water's role as a reservoir in Helicobacter pylori transmission: a review of the evidence. Malfertheiner, P. et al. Helicobacter pylori infection. Chiang, T. H. et al. Mass eradication of Helicobacter pylori to reduce gastric cancer incidence and mortality: a long-term cohort study on Matsu Islands. Dorji, T. et al. Population-level cancer screening and cancer care in Bhutan, 2020−2023: a review. Health Southeast Asia 24, 100370 (2024). Tsuda, M. et al. Effect on Helicobacter pylori eradication therapy against gastric cancer in Japan. Maucort-Boulch, D., de Martel, C., Franceschi, S. & Plummer, M. Fraction and incidence of liver cancer attributable to hepatitis B and C viruses worldwide. Schweitzer, A., Horn, J., Mikolajczyk, R. T., Krause, G. & Ott, J. J. Estimations of worldwide prevalence of chronic hepatitis B virus infection: a systematic review of data published between 1965 and 2013. Kandeel, A. et al. Evidence for the elimination of viral hepatitis B and C in Egypt: results of a nationwide survey in 2022. Salama, I. I. et al. Effectiveness of hepatitis B virus vaccination program in Egypt: multicenter national project. Thursz, M., Njie, R. & Lemoine, M. Global eradication of hepatitis B—feasible or fallacy? Clougherty, J. E. A growing role for gender analysis in air pollution epidemiology. Okello, G., Devereux, G. & Semple, S. Women and girls in resource poor countries experience much greater exposure to household air pollutants than men: results from Uganda and Ethiopia. Shupler, M. et al. Global estimation of exposure to fine particulate matter (PM2.5) from household air pollution. Raimondi, S., Suppa, M. & Gandini, S. Melanoma epidemiology and sun exposure. Iannacone, M. R. & Green, A. C. Towards skin cancer prevention and early detection: evolution of skin cancer awareness campaigns in Australia. Cumberbatch, M. G., Cox, A., Teare, D. & Catto, J. W. Contemporary occupational carcinogen exposure and bladder cancer: a systematic review and meta-analysis. Cislaghi, B. et al. Gender norms and gender equality in full-time employment and health: a 97-country analysis of the world values survey. Santana, V. S. & Ribeiro, F. S. Occupational cancer burden in developing countries and the problem of informal workers. Bray, F., Soerjomataram, I., Mery, L. & Ferlay, J. Improving the quality and coverage of cancer registries globally. Bray, F. et al. Cancer Incidence in Five Continents: inclusion criteria, highlights from Volume X and the global status of cancer registration. Moore, S. C. et al. Endogenous estrogens, estrogen metabolites, and breast cancer risk in postmenopausal Chinese women. J. Natl Cancer Inst. Freedman, N. D. et al. Impact of changing US cigarette smoking patterns on incident cancer: risks of 20 smoking-related cancers among the women and men of the NIH-AARP cohort. Lee, C. H. et al. Independent and combined effects of alcohol intake, tobacco smoking and betel quid chewing on the risk of esophageal cancer in Taiwan. Ferlay, J. et al. Global Cancer Observatory: Cancer Today (version 1.1). International Agency for Research on Cancer https://gco.iarc.fr/today/en (2024). International Statistical Classification of Diseases and Related Health Problems, 10th revision, Fifth edition, 2016 (World Health Organization, 2015); https://iris.who.int/handle/10665/246208 Fritz, A. et al. International Classification of Diseases for Oncology 3rd edn (World Health Organization, 2000); https://www.who.int/standards/classifications/other-classifications/international-classification-of-diseases-for-oncology Bray, F. et al. Cancer Incidence in Five Continents, Vol. XII (IARC CancerBase No. International Agency for Research on Cancer https://ci5.iarc.fr/ci5-xii/ (2023). Crump, K. S. Risk of benzene-induced leukemia: a sensitivity analysis of the pliofilm cohort with additional follow-up and new exposure estimates. Elliott, A. M., Aucott, L. S., Hannaford, P. C. & Smith, W. C. Weight change in adult life and health outcomes. Grundy, A. et al. Cancer incidence attributable to alcohol consumption in Alberta in 2012. Lanphear, B. P. & Buncher, C. R. Latent period for malignant mesothelioma of occupational origin. O'Reilly, K. M., McLaughlin, A. M., Beckett, W. S. & Sime, P. J. Asbestos-related lung disease. Parker, E. D. & Folsom, A. R. Intentional weight loss and incidence of obesity-related cancers: the Iowa Women's Health Study. Steenland, K., Stayner, L. & Deddens, J. Mortality analyses in a cohort of 18 235 ethylene oxide exposed workers: follow up extended from 1987 to 1998. Weiss, W. Cigarette smoking and lung cancer trends. A light at the end of the tunnel? GBD 2019 Tobacco Collaborators. Spatial, temporal, and demographic patterns in prevalence of smoking tobacco use and attributable disease burden in 204 countries and territories, 1990−2019: a systematic analysis from the Global Burden of Disease Study 2019. World Health Organization. Global Information System on Alcohol and Health. NCD Risk Factor Collaboration. National Adult Body-Mass Index – Data Downloads. World Health Organization. Insufficient physical activity – indicator group. Rumgay, H. et al. Global burden of oral cancer in 2022 attributable to smokeless tobacco and areca nut consumption: a population attributable fraction analysis. United Nations Children's Fund (UNICEF). https://data.unicef.org/topic/nutrition/breastfeeding/ (UNICEF Data, 2025). Air pollution: concentrations of fine particulate matter (PM2.5), SDG 11.6.2. World Health Organization https://www.who.int/data/gho/data/indicators/indicator-details/GHO/concentrations-of-fine-particulate-matter-(pm2-5) (2025). Langselius, O. et al. Global burden of cutaneous melanoma incidence attributable to ultraviolet radiation in 2022. WHO/ILO joint estimates of the work-related burden of disease and injury. World Health Organization https://www.who.int/teams/environment-climate-change-and-health/monitoring/who-ilo-joint-estimates (2021). Kehoe, T., Gmel, G., Shield, K. D., Gmel, G. & Rehm, J. Determining the best population-level alcohol consumption model and its impact on estimates of alcohol-attributable harms. Rehm, J. et al. Statistical modeling of volume of alcohol exposure for epidemiological studies of population health: the US example. Shield, K. D. & Rehm, J. Difficulties with telephone-based surveys on alcohol consumption in high-income countries: the Canadian example. NCD Risk Factor Collaboration (NCD-RisC). Worldwide trends in underweight and obesity from 1990 to 2022: a pooled analysis of 3663 population-representative studies with 222 million children, adolescents, and adults. Arnold, M. et al. Global burden of cancer attributable to high body-mass index in 2012: a population-based study. Prevalence of insufficient physical activity among adults. World Health Organization https://www.who.int/data/gho/data/indicators/indicator-details/GHO/prevalence-of-insufficient-physical-activity-among-adults-aged-18-years-(age-standardized-estimate)-(-) (2017). WHO/ILO joint estimates of the work-related burden of disease and injury, 2000−2016: global monitoring report (World Health Organization, 2021); https://www.who.int/publications/i/item/9789240034945 Pega, F., Hamzaoui, H., Náfrádi, B. & Momen, N. C. Global, regional and national burden of disease attributable to 19 selected occupational risk factors for 183 countries, 2000−2016: a systematic analysis from the WHO/ILO Joint Estimates of the Work-related Burden of Disease and Injury. Reitsma, M. B. et al. Spatial, temporal, and demographic patterns in prevalence of smoking tobacco use and initiation among young people in 204 countries and territories, 1990–2019. Lancet Public Health 6, e472–e481 (2021). Jordan, S. J., Whiteman, D. C., Purdie, D. M., Green, A. C. & Webb, P. M. Does smoking increase risk of ovarian cancer? A systematic review. World Cancer Research Fund/American Institute for Cancer Research. Diet, Nutrition, Physical Activity and Cancer: a Global Perspective (World Cancer Research Fund, 2018); https://www.wcrf.org/wp-content/uploads/2024/11/Summary-of-Third-Expert-Report-2018.pdf Shield, K. et al. National, regional, and global burdens of disease from 2000 to 2016 attributable to alcohol use: a comparative risk assessment study. Lancet Public Health 5, e51–e61 (2020). Bagnardi, V. et al. Alcohol consumption and site-specific cancer risk: a comprehensive dose−response meta-analysis. Abar, L. et al. Body size and obesity during adulthood, and risk of lympho-haematopoietic cancers: an update of the WCRF-AICR systematic review of published prospective studies. Wolin, K. Y., Yan, Y., Colditz, G. A. & Lee, I. M. Physical activity and colon cancer prevention: a meta-analysis. Friedenreich, C. M. & Cust, A. E. Physical activity and breast cancer risk: impact of timing, type and dose of activity and population subgroup effects. Schmid, D. et al. A systematic review and meta-analysis of physical activity and endometrial cancer risk. Renfrew, M. J., McCormick, F. M., Wade, A., Quinn, B. & Dowswell, T. Support for healthy breastfeeding mothers with healthy term babies. Cochrane Database Syst. Hamra, G. B. et al. Outdoor particulate matter exposure and lung cancer: a systematic review and meta-analysis. Plummer, M., Franceschi, S., Vignat, J., Forman, D. & de Martel, C. Global burden of gastric cancer attributable to Helicobacter pylori. de Martel, C., Plummer, M., Vignat, J. & Franceschi, S. Worldwide burden of cancer attributable to HPV by site, country and HPV type. GBD 2017 Risk Factor Collaborators. Global, regional, and national comparative risk assessment of 84 behavioural, environmental and occupational, and metabolic risks or clusters of risks for 195 countries and territories, 1990−2017: a systematic analysis for the Global Burden of Disease Study 2017. Levin, M. L. The occurrence of lung cancer in man. Acta Unio Int. Rumgay, H. et al. Global burden of cancer in 2020 attributable to alcohol consumption: a population-based study. Victora, C. G. et al. Breastfeeding in the 21st century: epidemiology, mechanisms, and lifelong effect. WHO global air quality guidelines: particulate matter (PM2.5 and PM10), ozone, nitrogen dioxide, sulfur dioxide and carbon monoxide (World Health Organization, 2021); https://www.who.int/publications/i/item/9789240034228 Arnold, M. et al. Global burden of cutaneous melanoma attributable to ultraviolet radiation in 2012. The authors received no specific funding for this work. Where authors are identified as personnel of the International Agency for Research on Cancer/World Health Organization (IARC/WHO), the authors alone are responsible for the views expressed in this article, and they do not necessarily represent the decisions, policies or views of the IARC/WHO. and his team at Octoma—L.L.-P., L. Jimenez-Perez and M. Erandi—for their assistance with the PAF calculations for insufficient physical activity, suboptimal breastfeeding, air pollution and occupational exposures. Cancer Surveillance Branch, International Agency for Research on Cancer (IARC/WHO), Lyon, France Hanna Fink, Oliver Langselius, Jérôme Vignat, Harriet Rumgay, Eileen Morgan & Isabelle Soerjomataram Institute for Mental Health Policy Research, Centre for Addiction and Mental Health, Toronto, Ontario, Canada Jürgen Rehm & Kevin Shield Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, Toronto, Ontario, Canada Jürgen Rehm & Kevin Shield Dalla Lana School of Public Health, University of Toronto, Toronto, Ontario, Canada Jürgen Rehm & Kevin Shield Department of Psychiatry, University of Toronto, Toronto, Ontario, Canada Public Health Institute of Catalonia, Barcelona, Spain Zentrum für Interdisziplinäre Suchtforschung der Universität Hamburg (ZIS), Universitätsklinikum Hamburg-Eppendorf, Hamburg, Germany Department of Communicable and Noncommunicable Diseases, World Health Organization, Geneva, Switzerland Ricardo X. Martinez, Marilina Santero, Lucero Lopez-Perez & André Ilbawi Division of Prevention, Center for Public Health Sciences, National Cancer Center, Tokyo, Japan National Central Cancer Registry & State Key Laboratory of Molecular Oncology, National Cancer Center/National Clinical Research Center for Cancer/Cancer Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing, China Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The study was conceptualized by I.S. Analyses were conducted by H.F., O.L., J.V., H.R., R.M., M.S. and L.L.-P., under the supervision of and with input from I.S. The first manuscript draft was prepared by H.F. All authors contributed to the editing of the manuscript. Correspondence to Hanna Fink. The authors declare no competing interests. Nature Medicine thanks Shilpa Murthy, Md. Mijan Rahman and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ming Yang, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Cancer cases attributable to modifiable risk factors in 2022 for a) all new cases in women, b) attributable-only in women, c) all new cases in men and d) attributable-only in men. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Fink, H., Langselius, O., Vignat, J. et al. Global and regional cancer burden attributable to modifiable risk factors to inform prevention. Version of record: 03 February 2026 DOI: https://doi.org/10.1038/s41591-026-04219-7 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00311-7'>Many people have no mental imagery. What's going on in their brains?</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 16:07:30
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Elizabeth Quill is an editor for Nature in Washington DC. Can you imagine the pattern on your coffee mug? Most of us can call up such pictures in our minds. But for an estimated 4% of people, this mental imagery is weak or absent. When researchers ask them to imagine something familiar, they might have a concept of what it is, and words and associations might come to mind, but they describe their mind's eye as dark or even blank. Systems neuroscientist Mac Shine at the University of Sydney, Australia, first realized that his mental experience differed in this way in 2013. He and his colleagues were trying to understand how certain types of hallucination come about1, and were discussing the vividness of mental imagery. They immediately asked him what he was talking about. Neither he nor his colleagues had realized how much variation there is in the experiences people have when they close their eyes. This moment of revelation is common to many people who don't form mental images. They report that they might never have thought about this aspect of their inner life if not for a chance conversation, a high-school psychology class or an article they stumbled across (see ‘How do you imagine? Although scientists have known for more than a century that mental imagery varies between people, the topic received a surge of attention when, a decade ago, an influential paper coined the term aphantasia to describe the experience of people with no mental imagery2. Since then, aphantasia has shot into the canon of unusual phenomena that are invaluable for studying how the mind works. Like synaesthesia (in which people's senses are connected in exceptional ways, so they hear colours, for example) and prosopagnosia (also known as face blindness), aphantasia has opened many new research avenues. Much of the early work sought to describe the trait and assess how it affected behaviour. But over the past five years, studies have begun to explore what's different about the brains of people with this form of inner life3. The findings have led to a flurry of discussions about how mental imagery forms, what it is good for and what it might reveal about the puzzle of consciousness: researchers tend to define mental imagery as a conscious experience, and some are now excited to study aphantasia as a way to probe imagery's potentially unconscious forms. Cognitive neuroscientist Giulia Cabbai at University College London is among the researchers interested in these questions. She was shocked to learn about aphantasia in 2015. Her own intensely vivid mental imagery is at the other extreme of the distribution — she has hyperphantasia. The fact that there are people with a complete lack of mental imagery brings fresh ways to study this internal experience, she says. Neurologist Adam Zeman at the University of Edinburgh and the University of Exeter, UK, began studying aphantasia in 2003. He met a man who, after a minimally invasive heart procedure, complained that although his visual perception remained normal, his mind's eye had vanished. Scans of his brain showed the expected activity when he looked at images of famous faces, but notable differences from control individuals when he tried to imagine the faces. After Zeman's team published a case study4 in 2010, Zeman heard from more than 20 people who said that they, too, lacked mental imagery, but they had lacked it their entire lives. Zeman says that he was inundated with messages after an article about the paper appeared in The New York Times. Since that article, some 20,000 people have contacted him with their own stories of mental imagery. The 2015 paper and subsequent research has revealed how much aphantasia can vary. For instance, people with aphantasia often, but not always, lack the ability to imagine in sensory modalities besides vision — having no ‘mind's ear', for example. Some people with aphantasia report dreaming in pictures, but others don't. Researchers have also found that aphantasia seems to have a genetic component, with the likelihood of having aphantasia increasing tenfold if you have a sibling who has a weak or absent mind's eye. Zeman and others say that aphantasia doesn't seem to make much difference to behaviour, and although it might influence creativity, it by no means precludes it. Instead of calling it a disorder or condition, Zeman describes it as an “intriguing variation” — one extreme on a distribution of mental-imagery capabilities. Much of the work characterizing mental imagery relies on asking participants to describe their experience. But such methods are subjective, and they can't separate true variations in experience from variations in how people describe or interpret that experience. So some researchers have been attempting to come up with other techniques. Neuroscientist Joel Pearson at the University of New South Wales in Sydney and his colleagues developed an approach that takes advantage of a perceptual phenomenon called binocular rivalry. When a different visual is presented to each eye simultaneously, for example, a pattern of green lines to the left eye and red lines to the right, a person's perception toggles between the two instead of blending them. Nearly two decades ago, Pearson decided to see what happened if he imagined one of the visuals in his mind's eye — in this case, only the green lines or only the red — before the test began. In a person with typical mental imagery, imagining the red pattern results in the person being more likely to see that red pattern during binocular rivalry. But a person with no visual imagery will not show this same bias. Pearson has been studying mental imagery ever since developing this method7. A person's emotional reaction to scary stories, measured by how much they sweat, can be a good proxy for how vividly they imagine what's happening in the story. And when researchers ask someone to imagine a bright light, the extent to which their pupils constrict correlates with the vividness of their mental imagery. A decade of work has left researchers convinced that aphantasia is a real phenomenon, but many are puzzled by how little it seems to affect behaviour. Behavioural tasks that are thought to depend on mental imagery don't seem to be a problem for people with aphantasia. They perform relatively well on standard memory assessments and they seem to be able to rotate objects in their mind, to determine whether an object in one picture matches another presented from a different angle. I can draw diagrams of the brain — whatever you want. But for me, when I try to imagine a purple dinosaur juggling on top of a bouncy ball, nothing shows up,” says Shine. He says the big mystery is how the brain can function typically in all these ways but at the same time lack this one specific ability. When scientists started looking for the brain signatures of aphantasia, they expected to see differences in the visual cortex. This is the area that receives and processes visual information during perception, and it's known to be active when a person pictures something in their mind. Most researchers have thought of mental imagery as vision in reverse, with higher-level brain areas sending signals to lower visual areas to generate a conscious image. But studies have suggested that when people with aphantasia attempt to imagine something, they activate the visual cortex in ways similar to that seen in control individuals. Perhaps, some researchers say, that brain area is forming visual representations in people with aphantasia, but the conscious mind can't access what is there. During her PhD, Cabbai decided to study the relationship between activity in the visual cortex and a person's experience of imagery. She wanted to find out whether what she and others call ‘sensory representations' in the visual cortex must always come with the experience of imagery. But she also wondered whether aphantasia was a problem with voluntarily conjuring up images on demand, so she and her colleagues took an approach that didn't require participants to do that. The team scanned the brains of people with and without aphantasia while they listened to sounds that should spontaneously trigger a sensory representation of whatever made the sound. Barking, for example, should trigger representations of a dog in the primary visual cortex. Hearing a dog bark did generate activity that represented a dog in both typical imagers and people with aphantasia. But, despite the existence of that activity, people with aphantasia reported seeing nothing in their mind's eye8. The finding suggests that these sensory representations — thought to underlie mental imagery — can remain unconscious and so are not sufficient to trigger imagery on their own. Insights into embodied cognition and mental imagery from aphantasia Insights into embodied cognition and mental imagery from aphantasia Cabbai calls the findings “puzzling”, and says that aphantasia might be a twofold problem involving not experiencing imagery consciously and not being able to voluntarily generate it. Insights into embodied cognition and mental imagery from aphantasia Brain marker signals when anaesthesia takes hold Developmental convergence and divergence in human stem cell models of autism Cholinergic modulation of dopamine release drives effortful behaviour Brain marker signals when anaesthesia takes hold of Computational and Systems Biology at Pitt is hiring two positions, a full-time Assistant Professor and a full-time Associate Professor. The University of Hong Kong We are seeking a motivated Tenure-Track Professor/Associate Professor/Assistant Professor to join our Department of Pha... Insights into embodied cognition and mental imagery from aphantasia An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a70190789/island-fossils-new-zealand/'>Scientists Explored an Island Cave and Found 1 Million-Year-Old Remnants of a Lost World</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 15:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Up to 50 percent of species—which included 12 bird species and 4 frog species—on Aotearoa's North Island went extinct before humans arrived Taken together, the fossils paint a picture of an ancient world that looks drastically different than it does today. “This is a newly recognized avifauna for New Zealand, one that was replaced by the one humans encountered a million years later,” Trevor Worthy, lead study author and associate professor at Flinders University, said in a statement. “This remarkable find suggests our ancient forests were once home to a diverse group of birds that did not survive the next million years.” By extrapolation, the research team estimated that between 33 and 50 percent of all species on the island went extinct during the million years before humans arrived in New Zealand, likely thanks to rapid climate shifts and cataclysmic volcanic eruptions, said Paul Scofield, study co-author and Canterbury Museum senior curator of natural history. Worthy said that “for decades, the extinction of New Zealand's birds was viewed primarily through the lens of human arrival 750 years ago. This study proves that natural forces like super-volcanoes and dramatic climate shifts were already sculpting the unique identity of our wildlife over a million years ago.” Of all the finds, scientists might be the giddiest about discovering a new species of parrot, Strigops insulaborealis, an ancient relative of the iconic Kākāpō, a large flightless parrot known for its heft. Still, the team said more research is needed to confirm that this ancient bird could fly. “We believe this was a major driver for the evolutionary diversification of birds and other fauna in the North Island.” Scofield said that previous excavations showed life in New Zealand between 20 and 16 million years ago and that by contrast, the new discoveries offer the first evidence of life from 15 million years ago to one million years ago. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. This Ancient Stick Could Be Humans' Earliest Tool</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/military/research/a70175245/elephant-bone-hannibal-army/'>Archaeologists Found One Elephant Bone at an Ancient Battlefield. It May Belong to Hannibal's Army.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The most famous four-legged weapons in warfare may have left a trace after all. Hannibal's use of elephants as a weapon of war wielded by his Carthaginian forces is etched into history, depicted in art and literature for more than a millennia. A recent discovery in Córdoba, Spain, however, offers compelling material evidence. Colina de los Quemados in Córdoba was once an Iberian Iron Age fortification known as Corduba, which Arkeonews describes as “a major protohistoric settlement occupying a strategic terrace above the Guadalquivir River.” Remnants of that settlement were uncovered in 2020 during an “emergency excavation” ahead of an expansion of the nearby Provincial Hospital of Córdoba. The archaeologists at the dig discovered “stone artillery projectiles, heavy arrowheads associated with scorpio-type siege weapons, and Carthaginian coinage minted in Cartagena between 237 and 206 B.C.E,” all indicative of military conflict roughly corresponding with the dates of the Second Punic War (218 to 201 BCE). Having these artifacts provide a historical context for the dig site is what made the discovery a single bone mean something far greater. The roughly 10-centimeter bone, Arkeonews reports, was “identified as the third carpal bone (os magnum) of the right forelimb of an elephant” after comparative anatomical analysis with comparable bones of both Asian elephants and steppe mammoths. While the bone's comparative size exceeds that of a Asian female elephant, it doesn't conclusively prove what species of elephant the bone came from. But it does lend credence to the possibility that it once belonged to an African elephant, which Hannibal is documented as using during the Second Punic War. But why is there only a single bone at the site? Researchers speculate that the elephant died amidst a military conflict “with most of its remains later destroyed, reused, or scattered” or that this bone may have been separated from the others during post-conflict activity. Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood. His stories for the Enthusiast Group have involved coordinating with organizations like the National Parks Service and the Secret Service, and travelling to notable historical sites and archaeological digs, from excavations of America' earliest colonies to the former homes of Edgar Allan Poe. Is the U.S. Ready to Fight a New Kind of War? Is One of the Pentagon's UFOs a Party Balloon?</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/unsinkable-metal-discovery-could-build-safer-ships-and-harvest-wave-energy/'>Unsinkable metal discovery could build safer ships and harvest wave energy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Spiders taught scientists how to make unsinkable metal Researchers mimicked the air-trapping tricks of diving bell spiders to create aluminum that stays afloat—even when punctured An “unsinkable” metal tube floats in distilled water in the lab of University of Rochester professor Chunlei Guo on January 9, 2026. Being denser than water, the metal sinks—ask any child. But new research has challenged centuries of certainty. A team at the University of Rochester has etched aluminum tubes so that they won't sink, even when damaged—a trick the scientists borrowed from spiders. “You can poke big holes in them,” said Chunlei Guo, a professor of optics and physics at the University of Rochester and senior author of the research, in a press release. “We showed that even if you severely damage the tubes with as many holes as you can punch, they still float.” If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Guo's team uses lasers to carve microscopic valleys into the aluminum that capture air: picture corduroy fabric shrunk down until it requires an electron microscope to see the ridges. The metal tubes mimic those fine hairs, trapping their own air bubbles. An “unsinkable” metal tube damaged with holes floats in distilled water in the lab of University of Rochester professor Chunlei Guo on January 9, 2026. In nature, superhydrophobicity isn't a new trick. The eyes of mosquitoes have water-repellent nanostructures that keep them clear, for example. As for humans, this isn't our first attempt at floating metal. In 2015 researchers at New York University embedded hollow silicon carbide spheres in a magnesium alloy to create a metal matrix composite lighter than water. Linked tubes could create weight-bearing rafts or ships. Engineers might be closing in on the dream of ships that stay afloat even as water pours into their hulls. One surprising application involves energy: Guo's team demonstrated that rafts made of the tubes could harvest waves to generate electricity. The tubes are currently nearly half a meter long, but Guo sees no barrier to increasing their size. The lasers are now seven times more powerful than they were in 2019, when Guo first attempted laser-etched metal disks. Deni Ellis Béchard is Scientific American's senior writer for technology. His most recent novel, We Are Dreams in the Eternal Machine, explores the ways that artificial intelligence could transform humanity. You can follow him on X, Instagram and Bluesky @denibechard If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-026-68769-7'>Predicting missing links in food webs using stacked models and species traits</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 11:12:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. , Article number:  (2026) Cite this article We are providing an unedited version of this manuscript to give early access to its findings. Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Networks are a powerful way to represent the complexity of large ecological systems. However, most ecological networks, such as food webs, contain only partial lists of species interactions. Computational methods for inferring missing links can facilitate field work and investigations of ecological processes. Here, we describe a stacked generalization approach to predict missing links in food webs that accounts for ecological assumptions including link direction. Tests of this method on synthetic food webs show that it can learn to optimally combine structural and trait-based predictions. On a global database of 290 food webs, the method often achieves near-perfect performance, performs better when it can exploit both species traits and network structure, and is principally driven by a subset of ecologically-interpretable predictors. Furthermore, we find that link predictability varies with ecosystem and network characteristics. These results show broad applicability of stacked generalization for predicting and understanding ecological interactions. The empirical food-web datasets used in this study are available in the GATEWAy database (GlobAl daTabasE of traits and food Web Architecture33,37, version 3, accessible at https://doi.org/10.25829/idiv.283-3-756). The data was pre-processed as described in Supplementary Note 2. The pre-processed data are available at https://doi.org/10.5281/zenodo.18026669. Supplementary Data 1 includes food web features across the datasets and summary statistics for food web features split by ecosystem type. Supplementary Data 2 includes details of Beta regression results. The code for this paper is available at https://doi.org/10.5281/zenodo.18026669 ref. Data pre-processing and missing link prediction scripts were run with Python 3.6.3 on CentOS Linux 7 using High Performance Computing resources supported by BioFrontiers IT. Results visualization scripts were run on Windows 11 with Python 3.12.4. A configuration file is provided in the code repository for reproducing a Python environment with the necessary packages for running the scripts. Clauset, A., Moore, C. & Newman, M. E. J. Hierarchical structure and the prediction of missing links in networks. Google Scholar Ghasemian, A., Hosseinmardi, H., Galstyan, A., Airoldi, E. M. & Clauset, A. Stacking models for nearly optimal link prediction in complex networks. Google Scholar Guimerà, R. & Sales-Pardo, M. Missing and spurious interactions and the reconstruction of complex networks. Google Scholar Zhou, T. Progresses and challenges in link prediction. Google Scholar Wolpert, D. H. Stacked generalization. Google Scholar A., Brose, U., Williams, R. J. & Martinez, N. D. Modeling food-web dynamics: complexity-stability implications. In Aquatic food webs: an ecosystem approach, 117–129 (Oxford, 2005). McDonald-Madden, E. et al. Using food-web theory to conserve ecosystems. Google Scholar Kamenova, S. et al. Invasions Toolkit. In Advances in Ecological Research, vol. Van De Walle, R. et al. Arthropod food webs predicted from body size ratios are improved by incorporating prey defensive properties. Google Scholar Jordano, P. Sampling networks of ecological interactions. Google Scholar Terry, J. C. D. & Lewis, O. T. Finding missing links in interaction networks. Google Scholar Novak, M. et al. Predicting community responses to perturbations in the face of imperfect knowledge and network complexity. Google Scholar Eklöf, A. et al. The dimensionality of ecological networks. Google Scholar Morales-Castilla, I., Matias, M. G., Gravel, D. & Araújo, M. B. Inferring biotic interactions from proxies. Google Scholar Desjardins-Proulx, P., Laigle, I., Poisot, T. & Gravel, D. Ecological interactions and the Netflix problem. Google Scholar Laigle, I. et al. Species traits as drivers of food web structure. Google Scholar Pomeranz, J. P. F., Thompson, R. M., Poisot, T. & Harding, J. S. Inferring predator-prey interactions in food webs. Google Scholar Gravel, D., Poisot, T., Albouy, C., Velez, L. & Mouillot, D. Inferring food web structure from predator-prey body size relationships. Google Scholar Nagelkerke, L. A. J. & Rossberg, A. G. Trophic niche-space imaging, using resource and consumer traits. Google Scholar Wootton, K. L. et al. Layer-specific imprints of traits within a plant-herbivore-predator network - complementary insights from complementary methods. Google Scholar Digel, C., Riede, J. O. Body sizes, cumulative and allometric degree distributions across natural food webs. Google Scholar Allesina, S. & Pascual, M. Food web models: a plea for groups. Google Scholar & Martinez, N. D. Simple rules yield complex food webs. Google Scholar Dalla Riva, G. V. & Stouffer, D. B. Exploring the evolutionary signature of food webs' backbones using functional traits. Google Scholar Rohr, R. P., Naisbit, R. E., Mazza, C. & Bersier, L.-F. Matching-centrality decomposition and the forecasting of new links in networks. B Biol. Google Scholar Rohr, R., Scherer, H., Kehrli, P., Mazza, C. & Bersier, L. Modeling food webs: exploring unexplained structure using latent traits. Google Scholar Williams, R. J., Anandanadesan, A. & Purves, D. The probabilistic niche model reveals the niche structure and role of body size in a complex food web. PLoS ONE 5, e12092 (2010). Google Scholar Pearse, I. S. & Altermatt, F. Predicting novel trophic interactions in a non-native world. Google Scholar Stock, M., Poisot, T., Waegeman, W. & De Baets, B. Linear filtering reveals false negatives in species interaction data. Google Scholar Papadogeorgou, G., Bello, C., Ovaskainen, O. & Dunson, D. B. Covariate-informed latent interaction models: addressing geographic & taxonomic bias in predicting bird-plant interactions. Google Scholar Poisot, T. Guidelines for the prediction of species interactions through binary classification. Google Scholar Guimerà, R. One model to rule them all in network science? Google Scholar GlobAL daTabasE of traits and food Web Architecture (GATEWAy) version 1.0 (2018). Holland, P. W., Laskey, K. B. & Leinhardt, S. Stochastic blockmodels: first steps. Google Scholar Penrose, M. Random geometric graphs (Oxford University Press, 2003). Ling, C. X., Huang, J. & Zhang, H. AUC: A Better Measure than Accuracy in Comparing Learning Algorithms. In Xiang, Y. Advances in Artificial Intelligence, vol. 2671, 329–341 (Springer Berlin Heidelberg, Berlin, Heidelberg, 2003). Series Title: Lecture Notes in Computer Science. Brose, U. et al. Predator traits determine food-web architecture across ecosystems. Google Scholar Brimacombe, C., Bodner, K., Michalska-Smith, M., Poisot, T. & Fortin, M.-J. Shortcomings of reusing species interaction networks created by different sets of researchers. Google Scholar Bohan, D. A., Caron-Lormier, G., Muggleton, S., Raybould, A. Automated discovery of food webs from ecological data using logic-based machine learning. PLoS ONE 6, e29028 (2011). Google Scholar Stouffer, D. B., Rezende, E. L. & Amaral, L. A. N. The role of body mass in diet contiguity and food-web structure: body mass and food-web structure. Google Scholar Huxham, M., Beaney, S. & Raffaelli, D. Do parasites reduce the chances of triangulation in a real food web? Google Scholar Brose, U. et al. Consumer-resource body size relationships in natural food webs. Google Scholar Pichler, M., Boreux, V., Klein, A., Schleuning, M. & Hartig, F. Machine learning algorithms to infer trait-matching and predict species interactions in ecological networks. Google Scholar Caron, D. et al. Trait-matching models predict pairwise interactions across regions, not food web properties. Google Scholar Strydom, T. et al. A roadmap towards predicting species interaction networks (across space and time). B Biol. Google Scholar Barner, A. K., Coblentz, K. E., Hacker, S. D. & Menge, B. Fundamental contradictions among observational and experimental estimates of non-trophic species interactions. Google Scholar Blanchet, F. G., Cazelles, K. & Gravel, D. Co-occurrence is not evidence of ecological interactions. Google Scholar Li, J. et al. A size-constrained feeding-niche model distinguishes predation patterns between aquatic and terrestrial food webs. Google Scholar Cattin, M.-F., Bersier, L.-F., Banašek-Richter, C., Baltensperger, R. & Gabriel, J.-P. Phylogenetic constraints and adaptation explain food-web structure. & Godfray, H. Phylogenetic analysis of trophic associations. Petchey, O. L., Beckerman, A. P., Riede, J. O. & Warren, P. H. Size, foraging, and food web structure. & Leskovec, J. node2vec: Scalable Feature Learning for Networks. In Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, 855–864 (ACM, 2016). Strydom, T. et al. Graph embedding and transfer learning can help predict potential species interaction networks despite data limitations. Lafferty, K. D. et al. Parasites in food webs: the ultimate missing links. & Allesina, S. Telling ecological networks apart by their structure: a computational challenge. Bar-Hen, A., Barbillon, P. & Donnet, S. Block models for generalized multipartite networks: applications in ecology and ethnobiology. A., Moore, C. & Clauset, A. Untangling the roles of parasites in food webs with generative network models (2015). Pringle, R. M. & Hutchinson, M. C. Resolving food-web structure. Tabouy, T., Barbillon, P. & Chiquet, J. Variational inference for stochastic block models from sampled data. He, X. et al. Link prediction accuracy on real-world networks under non-uniform missing-edge patterns. From theories to queries: Active learning in practice. In Guyon, I., Cawley, G., Dror, G., Lemaire, V. & Statnikov, A. Active Learning and Experimental Design workshop In conjunction with AISTATS 2010, vol. 16 of Proceedings of Machine Learning Research, 1–18 (PMLR, 2011). Biton, B., Puzis, R. & Pilosof, S. Inductive link prediction facilitates the discovery of missing links and enables cross-community inference in ecological networks. Nature Ecology & Evolution 1–10 (Springer, 2025). Botella, C., Dray, S., Matias, C., Miele, V. & Thuiller, W. An appraisal of graph embeddings for comparing trophic network architectures. He, X., Ghasemian, A., Lee, E., Clauset, A. & Mucha, P. J. Sequential stacking link prediction algorithms for temporal networks. Breiman, L. Random Forests. Species loss and secondary extinctions in simple and complex model communities. Liben-Nowell, D. & Kleinberg, J. The link-prediction problem for social networks. Cirtwill, A. R. & Wootton, K. L. Stable motifs delay species loss in simulated food webs. Structure and inference in annotated networks. Cribari-Neto, F. & Zeileis, A. Beta regression in r. J. Stat. Van Kleunen, L., Dee, L. E., Wootton, K. L., Massol, F. & Clauset, A. Predicting missing links in food webs using stacked models and species traits (2025). The authors thank the Brose lab for help with the empirical data, A. Ghasemian for helpful conversations about stacking models for link prediction, and B. Singh, D.B. Larremore and E. Bradley for helpful discussions and feedback. This work was supported in part by the National Science Foundation Division of Ocean Sciences (NSF OCE 2049360), the Eppley Foundation for Research, and the Chateaubriand Fellowship of the Office for Science & Technology of the Embassy of France in the United States. The stacking model code used here is adapted for use in directed, attributed, hierarchical networks from Ghasemian et al. (2020). The authors acknowledge the BioFrontiers Computing Core at the University of Colorado Boulder for providing High Performance Computing resources supported by BioFrontiers IT. Department of Computer Science, University of Colorado, Boulder, CO, USA Van Kleunen & Aaron Clauset Department of Public Health and Primary Care, KU Leuven Campus Kulak, Kortrijk, Belgium Itec, imec research group at KU Leuven, Kortrijk, Belgium Department of Ecology and Evolutionary Biology, University of Colorado, Boulder, CO, USA School of Biological Sciences, University of Canterbury, Christchurch, New Zealand University of Lille, CNRS, Inserm, CHU Lille, Institut Pasteur de Lille, U1019 - UMR 9017 - CIIL - Center for Infection and Immunity of Lille, F-59000, Lille, France Sorbonne Université, Université Paris Cité, Université Paris Est Créteil, CNRS, INRAE, IRD, Institut d'Ecologie et des Sciences de l'Environnement de Paris (UMR7618), 75005, Paris, France BioFrontiers Institute, University of Colorado, Boulder, CO, USA Santa Fe Institute, Santa Fe, NM, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar L.V., L.D., and A.C. conceptualized the study. and A.C. supervised the study. and A.C. designed the methodology and performed the mathematical analysis. planned and validated the data pre-processing steps. wrote the code to pre-process the data, performed the computational analysis, and visualized results. L.D., K.W., F.M., and A.C. provided feedback to validate and improve the computational analysis and visualizations. and A.C. wrote an initial paper draft and all authors contributed to final paper writing, review, and editing. Correspondence to Lucy B. Van Kleunen or Aaron Clauset. The authors declare no competing interests. Communications Materials thanks Chencheng Cai, Virginia Dominguez-Garcia, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Reprints and permissions Van Kleunen, L.B., Dee, L.E., Wootton, K.L. Predicting missing links in food webs using stacked models and species traits. Nat Commun  (2026). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative (Nat Commun) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43587-026-01067-5'>Aging clocks delineate neuron types vulnerable or resilient to neurodegeneration and identify neuroprotective interventions</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 11:02:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Different neuron types show distinct susceptibility to age-dependent degeneration, yet the underlying mechanisms are poorly understood. Here we applied aging clocks to single neuron types in Caenorhabditis elegans and found that distinct neurons differ in their biological age. Ciliated sensory neurons with high neuropeptide and protein biosynthesis gene expression show accelerated aging and degeneration, correlating with loss of function, which could be prevented by pharmacological inhibition of translation. We show that the C. elegans neuronal aging transcriptomes correlate with human brain aging patterns and anticorrelate with geroprotective interventions. We performed an in silico drug screen to identify potentially neuroprotective small molecules. We show that the natural occurring plant metabolite syringic acid and the piperazine derivative vanoxerine delay neuronal degeneration, and propose these compounds as neuroprotective interventions. Furthermore, we identify neurotoxins that accelerate neurodegeneration, indicating that distinguishing aging trajectories between neuron types can inform on protective interventions as well as risk factors. Aging is the highest risk factor for neurodegenerative diseases such as Alzheimer's disease or Parkinson's disease that are triggered by the functional decline of distinct neuron types. In mice, distinct classes of brain cells show differences in age-related decline6, and single-cell transcriptomics has identified pro-aging and pro-rejuvenating proximity effects of distinct cell populations7. The extent to which intrinsic aging susceptibility differs among individual neuron types has, however, remained largely unknown. To elucidate whether different neuron types might differ in their biological aging process, we used C. elegans as an experimental model with a well-characterized neuronal system of 302 neurons of ≥128 different neuron types8 that allows assessing the integrity of individual neurons during the normal aging process in vivo. Aging clocks provide predictive models of molecular signatures (for example, DNA methylation) that estimate an individual's chronological age, and could identify biological age differences resulting from genetic or pharmacological age acceleration or deceleration9,10,11. Based on the known effects of many genetic and pharmacologic interventions on lifespan and thus biological age in C. elegans, we recently developed the highly accurate binarized transcriptomic (Bit) age clock. BitAge predicts already in young adult animals quantitatively how genetic, environmental or pharmacological treatments affect lifespan10. We previously established that also the simulation of increasing noise allows the prediction of biological age regardless of organism or data type, allowing us to design the Stochastic Clock to detect age accelerating and decelerating interventions12. Here, we asked whether in chronologically young animals, distinct neuron types might exhibit distinct biological ages in C. elegans. We show that individual neuron types in the late larval L4 stage, which directly precedes adulthood, show a distinct offset of biological age prediction despite sharing the same chronological age. We demonstrate that the age prediction corresponds to neuron-type-specific degeneration and the decline of their specific function. Neuron types with an older predicted age show early neurodegeneration, while those with a younger predicted age are preserved. The transcriptomic differences between these biologically ‘older' and ‘younger' neuron types indicate translation as a crucial driver of neuronal aging and pharmacologically reducing protein synthesis prevented the degeneration of the fast-aging neurons. We determined that the transcriptional patterns of the biological age differences are correlated with the transcriptional patterns of human and mouse brain aging, suggesting conserved neuronal aging mechanisms. The neuronal aging patterns anticorrelated with geroprotective interventions enabling the identification of neuronal aging modulators. Using a transcriptomic resource incorporating thousands of different compounds in human cell lines (CMAP)13, we identified and validated pharmacological compounds that protect the integrity of neurons in vivo. We demonstrate that the naturally occurring plant metabolite syringic acid and the piperazine derivative vanoxerine prevent neurodegeneration and propose that they could serve as neuroprotective interventions. In reverse, our approach can also identify neurotoxins that accelerate neurodegeneration. Our data suggest that the mechanisms underlying neuron-type-specific aging rates allow the identification of therapeutic interventions that could slow down neuronal aging and prevent neurodegeneration. To investigate whether different neuron types age differently within an organism, we applied our previously developed BitAge biological age predictor10 and, as independent age predictor, our previously developed Stochastic Clock12 on neuron-type-specific RNA-sequencing data. We used the single-neuron-type data from the C. elegans Neuronal Gene Expression Map & Network (CeNGEN) dataset14, which comprises 128 distinct neuron types from late L4-stage larvae. The 128 neuron age predictions range from ≈98 h in FLP neurons to ≈177 h in ADL neurons (Fig. 1a; BitAge) suggesting that different neurons might show an almost twofold biological age difference in chronologically young nematodes. a, Distribution of transcriptomic age predictions. The 128 neurons of the CeNGEN dataset were predicted with BitAge (upper) or a stochastic noise-based clock (lower) and sorted by their predicted age. The 20% youngest neurons and their respective age predictions are outlined in blue; the 20% oldest neurons and their respective age predictions are displayed in orange. Neuron types appearing on the top/bottom of both prediction lists are indicated in bold letters. b, Predictions with BitAge and the Stochastic Clock on the CeNGEN dataset are highly correlated (Pearson correlation 0.65, P value 5.5 × 10−17). The regression model fit with a 95% confidence interval (dashed, black lines) is shown. c, Neuron-type-specific genes in the CeNGEN dataset are skewed depending on gene length and age prediction. The log10 gene length (x axis) of the specific genes from the 20% youngest neurons (blue), and of the 20% oldest neurons (orange) are compared to random permutation of all neuron-type-specific genes with the same number of genes (gray). d, Swarm plot of the median log10 length of neuron-type-specific genes expressed in the youngest 20% (blue) and oldest 20% (orange) neurons according to BitAge predictions. Each point represents a single neuron type. Neuron types were grouped into the youngest 20% (blue, n = 17) and oldest 20% (orange, n = 21) according to BitAge-predicted biological age. Data shown are the mean ± s.d. Welch's two-tailed t-test was used to test for significant differences. We independently confirmed the different age predictions of the neuron types in the single-neuron-type RNA-sequencing data from young adults in a recent cell atlas of C. elegans aging (Calico)15, which contains 67 of the 128 neuron types from the CeNGEN dataset. Also, the Calico dataset of day 1 adults exhibits the same predicted age distribution (Extended Data Fig. 1a), and the BitAge predictions on both datasets are significantly correlated with each other (Extended Data Fig. To independently assess each neuron's biological age, we applied a Stochastic Clock that does not rely on specific transcriptomic signatures but instead simulates how random changes accumulate over time12. Despite using a fundamentally different strategy, the Stochastic Clock showed substantial overlap with BitAge (Fig. 1a) and predicts an almost fourfold age difference between the youngest and oldest neurons. 1b; Pearson correlation r = 0.65, P < 0.0001), which was even stronger within the very young or very old predicted group of neurons (Extended Data Fig. 1g; Pearson correlation r = 0.78, P < 0.0001; for the top and bottom 20%). The Stochastic Clock predictions showed a significant correlation between the CeNGEN and Calico datasets (Extended Data Fig. Similarly, the Stochastic Clock predictions correlated significantly with BitAge in the Calico dataset (Extended Data Fig. Taken together, two independent aging clock paradigms using two independent single-neuron transcriptome datasets consistently predict specific neuron types as younger and others as older in chronologically young animals. To address potential limitations of single-cell sequencing coverage, we performed the same analyses on bulk RNA-sequencing data from CeNGEN, which offers improved transcriptomic coverage for a subset of 37 neurons16. BitAge predictions remained consistent between the pseudobulk CeNGEN dataset and the bulk RNA-sequencing dataset (Extended Data Fig. Similarly, the Stochastic Clock predictions on pseudobulk and bulk CeNGEN datasets were also significantly correlated (Extended Data Fig. 1f; Pearson correlation 0.7, P < 0.0001), reinforcing that the predicted biological age differences among neurons are robust across different datasets and sequencing approaches. While the predictions of BitAge and the Stochastic Clock are highly consistent, there are also some outliers that deviate more strongly from the correlation (Extended Data Fig. As additional validation, we used bulk neuron transcriptomes from young and aged animals that were not included in the training data of the clocks17. We found that both clocks could differentiate between the young and old samples (Extended Data Fig. 1h); note that differences between predicted biological age and indicated chronological age could arise from two effects: first, the sample age is reported in days, giving a ±12-h uncertainty; second, the associated reported median lifespan of 22 days is unexpectedly high (median lifespan is typically between 14 and 18 days)17. The aging transcriptome in species ranging from C. elegans to humans was recently shown to exhibit a gene length-dependent transcriptional decline (GLTD): the longer the gene, the more sharply its expression tends to decrease with age18,19. GLTD is thought to result from the age-dependent accumulation of transcription-blocking DNA damage that is more likely to occur in larger genes. To test whether neurons predicted to be biologically older showed GLTD, we analyzed neuron-type-specific genes, which are defined as genes that are expressed in at least 90% of the specific neurons and in not more than 10% of other cells. In line with GLTD, the neuron-type-specific genes of the 20% oldest predicted neurons are significantly shorter than expected by chance (P = 3.6 × 10−4), while the neuron-type-specific genes of the 20% youngest neurons are significantly longer (P = 9.28 × 10−3; Fig. 1c) and show a significantly longer median gene length (Fig. The median length of the neuron-type-specific genes is negatively correlated with the prediction age of BitAge or the Stochastic Clock on either CeNGEN or Calico data (Extended Data Fig. These results indicate that the clock-predicted biological age is reflected in the age-associated GLTD that is most prevalent in the biologically old predicted neurons. Taken together, the alignment of aging clock predictions with the presence of GLTD in neurons predicted to be biologically old in chronologically young animals provides compelling evidence that specific neuron types undergo accelerated aging. To assess whether the predicted age differences are associated with different degrees of neuron-specific degeneration, we next chose three young (I2, OLL, PHC) and three old (ASI, ASJ, ASK) predicted neurons (Fig. 2a) and scored their degeneration over the chronological age starting from the larval L4 stage onward to day 7 of adulthood. To assess the morphological integrity of specific neurons in vivo, we used neuron-type-specific transcriptional green fluorescent protein (GFP) reporters (Fig. Macroscopic aberrations in the neurites were classified as ‘healthy', ‘damaged' or ‘severely damaged' (Fig. We excluded that GFP expression levels might influence neuron integrity, as GFP expression levels and neuronal integrity showed no correlation as tested in ASK and ASJ neurons that exhibit the strongest variance in GFP expression levels among the employed nematode strains (Extended Data Fig. a, Representative fluorescence images (z-stack maximum projections) of the analyzed neurons grouped by prediction age—I2, OLL and PHC as representatives of the young neurons (blue); ASI, ASJ and ASK as representatives of the old neurons (orange). b, Representative fluorescence images (z-stack maximum projections) of nematodes expressing neuronal GFP markers, classified according to the severity of observed degeneration. Classification criteria are indicated below the images. Nematode heads are outlined by a dashed line. c, Fraction plots displaying the fraction of nematodes expressing neuronal markers in different neurons categorized as ‘healthy', ‘mildly damaged' and ‘severely damaged'. Three to four cohorts were analyzed, comprising 10–30 individual nematodes, for every time point indicated. Data shown are the mean ± s.d. Statistical analysis was performed using a cumulative link model (CLM) with a logit link to account for the ordered categorical nature of neuronal damage scores (healthy < mild < severe). P values were adjusted for multiple testing using the Benjamini–Hochberg method. In accordance with our predictions, the three young predicted neuron types exhibited significantly less degeneration than the old predicted neuron types at all analyzed time points (Fig. At L4 stage and on the first day of adulthood, I2, OLL and PHC neurons exhibited a minimal degeneration only affecting 10–20% of the animals, which increased to ≈35% during aging. URY neurons, which were described previously to exhibit approximately 35% degeneration at day 7 of adulthood23, displayed ≈20% degeneration in L4 and young adults, which increased up to ≈35% on day 7 of adulthood (Extended Data Fig. All analyzed neuron types except I2 showed a significant age-dependent deterioration (Fig. The predicted biologically older ASI, ASJ and ASK neurons were damaged in >45% of L4 larvae reaching up to 90% at day 7 of adulthood, indicative of rapid aging. To determine at which stage the rapidly aging neurons start to degenerate, we also examined the ASI, ASJ and ASK neurons in the larval L3 stage. Consistent with a fully developed neuronal system, those neuron types were intact in more than 90% of the L3 animals, suggesting that the first burst of neuronal aging occurs already between the L3 and L4 stage (Extended Data Fig. To discern whether the progressive degenerative process reflects an age-dependent stochastic process or instead a deterministic developmental program, we analyzed whether the bleb patterns appeared randomly along the neurites. The positions of blebs in ASJ, ASK and OLL neurons in adults indeed correlated with a random distribution of damage sites along neurites. Only one site in OLL neurons exhibited nonrandom bleb clustering; it was located at the proximal pharyngeal bulb and prone to swelling (Extended Data Fig. To address whether we might have underestimated the neurodegeneration because only the morphology of neurons that express GFP could be analyzed, and hence are present, we assessed neuronal survival in individual nematodes over a time course of 12 days during adulthood. During aging most animals retained the young predicted I2, OLL and URY neurons, while the old predicted ASK and ASE neurons were progressively lost (Extended Data Fig. 2f–h), showing that early biological age predictions are associated with age-dependent neuronal attrition. Taken together, these results validate our neuron-type-specific biological age predictions for identifying neurons at higher risk of degeneration, with potential implications for understanding selective neuronal vulnerability in aging and neurodegeneration. To test whether the morphological neurite degeneration corresponds with functional impairment, we monitored behaviors that depend specifically on a single neuron type. The requirement of a highly neuron-type-specific phenotypic outcome excluded the functional analysis of the ASI, ASJ, ASK, I2, OLL and PHC neurons for any of the following reasons: (1) some neurons, such as I2, lack well-defined functional roles; (2) some functions are executed through multiple redundantly operating neurons thus precluding assessment of single-neuron function; (3) some neurons are negative regulators of other neurons leading to indirect effects. In contrast, URY neurons (representing young predicted neurons) are strictly required for selected pathogen avoidance24 and ASE neurons (representing old predicted neurons) mediate salt sensation and memory formation25 allowing their functional analysis by behavioral testing. We tested whether the degeneration of URY neurons correlates with impaired avoidance behavior from pathogenic Serratia marcescens Db11 bacteria, which is primarily mediated by the TOL-1 receptor expressed in URY neurons24. URY-marked strains (GFP is expressed specifically in URY neurons) were exposed to either pathogenic Db11 or nonpathogenic OP50 control bacteria. To test whether failure to display avoidance behavior correlated with neurodegeneration, we assessed the integrity of the URY neurons in animals that either avoided or failed to avoid the bacterial lawn (Fig. a, Schematic depiction of the experimental design to examine the correlation between pathogen avoidance from S. marcescens Db11 bacteria and URY neuron integrity. Nematodes were synchronized by L4 picking and transferred to Db11 or OP50 bacteria. After 2 days, avoidance behavior was scored and nematodes classified as avoiding/not-avoiding and subjected to microscopy to analyze URY neuron neurite health. b, Swarm plot of the results from the pathogen avoidance assay from Db11 bacteria. URY neuron marker strain (URYp::GFP, JKM10), I2 neuron marker strain (I2p::GFP, MT21910, control neuron unrelated to avoidance reaction) and TOL-1 receptor mutant strain (∆tol-1, IG10, no avoidance from Db11) were used. Data shown are the mean ± s.d. from four independent cohorts of 50–75 nematodes. Two-way analysis of variance (ANOVA) + Tukey post hoc test were used. c, Swarm plot displaying the healthy neuron fraction size from nematodes subjected to Db11 pathogen or OP50 control bacteria and classified for evasion behavior. URY neuron marker strain (URYp::GFP, JKM10, mediating avoidance from S. marcescens) and I2 neuron marker strain (I2p::GFP, MT21910, control neuron unrelated to avoidance reaction) were used. Statistical analysis was performed using a CLM. P values were adjusted for multiple testing using the Benjamini–Hochberg method. d, Schematic depiction of the experimental design to correlate salt aversive conditioning and neurodegeneration of ASE neurons. Adult 1-day-old nematodes were conditioned by starvation in the presence or absence of NaCl, and thereafter subjected to a chemotaxis assay on an NaCl gradient. Subsequently, nematodes were classified (dwelling on high/low salt) and neurite integrity was analyzed using fluorescence microscopy. Orange asterisks indicate nematodes that failed at learning. e, Swarm plot showing the chemotaxis index of nematodes conditioned at 0 mM (light blue) or 20 mM (dark blue) NaCl. ASE neuron marker strain (ASEp::GFP, NY2067, mediating salt aversive conditioning) and URY neuron marker strain (URYp::GFP, JKM10, control neuron unrelated to salt aversive conditioning) were used. Data shown are the mean ± s.d. from five independent cohorts of 80–150 nematodes. Statistical analysis was performed using a CLM. P values were adjusted for multiple testing using the Benjamini–Hochberg method. f, Swarm plot displaying the healthy neuron fraction size from nematodes after salt aversive conditioning. Nematodes were grouped depending on the conditioning and their subsequent preference for a high (dark blue) or low (light blue) salt concentration. The groups that failed to connect the starvation stimulus and salt concentration are indicated by orange asterisks. ASE neuron marker strain (ASEp::GFP, NY2067, mediating salt aversive conditioning) and URY neuron marker strain (URYp::GFP, JKM10, control neuron unrelated to salt aversive conditioning) were used. Data shown are the mean ± s.d. from five independent cohorts of 10–20 nematodes. Two-way ANOVA + Tukey post hoc test were used. g, We adapted a previously published connectome of C. elegans37. Only neuronal cells are shown in a largely directional information flow on the vertical axis, with sensory neurons (triangles) on top, interneurons (hexagons) in the middle and motor neurons (circles) on the bottom. Chemical synapses and gap junctions are indicated as faded gray lines. The predicted age (BitAge based on the CeNGEN dataset) is color coded from blue (young) to orange (old). The oldest neurons cluster in the middle top part and are largely sensory neurons. h, Swarm plot showing BitAge predictions grouped by amphid neurons/non-amphid neurons and ciliated neurons/non-ciliated neurons. Data shown are the mean ± s.d. A two-sided t-test was used. i, Swarm plot showing ciliated neurons' age predictions divided into five classes depending on where their cilia terminate. Data shown are the mean ± s.d. ANOVA + Tukey post hoc test were used. Approximately 80% of the URY neuron-marked animals avoided the toxic Db11 bacterial lawn thus showing a typical pathogen avoidance behavior (Fig. To ascertain the specificity of the assay, we tested a tol-1 mutant control strain that failed to show any avoidance. A baseline avoidance of ≈20% was observed on nonpathogenic OP50, consistent with natural exploratory behavior. Nematodes that failed to avoid pathogenic Db11 showed significantly increased degeneration of URY neurons compared to those that successfully avoided the bacteria (Fig. As a control, we ascertained that degeneration of I2 neurons, which are also predicted to be biologically young (Figs. 1a and 2c) but are not involved in pathogen avoidance, was uncorrelated with the avoidance behavior, confirming that the loss of avoidance behavior was dependent on URY neuron degeneration (Fig. To corroborate our findings, we tested salt aversive memory formation that is specifically mediated by ASE neurons25, which are predicted to be biologically old (Fig. Nematodes were conditioned to associate either low (0 mM) or high (20 mM) sodium chloride concentrations with starvation. Thereafter, the behavioral responses were tested by using a chemotaxis preference assay on a sodium chloride gradient and subjected to neurite imaging of either salt-sensing ASE neurons or salt-sensing unrelated URY neurons (Fig. The strains expressing GFP in either ASE or URY neurons similarly learned to move toward higher salt concentrations when a low salt concentration was associated with starvation (Fig. 3e), while nematodes conditioned to associate a high salt concentration with starvation moved toward lower salt concentrations on the salt gradient (Fig. Nevertheless, some individual nematodes failed to show the expected avoidance behavior and moved toward the salt concentration they should have learned to avoid (Fig. The neuronal health was assessed as described above (Fig. 2b), and the healthy neuron fraction was quantified across four behavioral groups: Nematodes (1) trained to avoid low salt (left) and found on low salt (light blue); (2) trained to avoid low salt (left) and found on high salt (dark blue); (3) trained to avoid high salt (right) and found on low salt (light blue); and (4) trained to avoid high salt (right) and found on high salt (dark blue). There was no significant difference in the healthy neuron fraction among animals conditioned on 0 mM NaCl, regardless of their behavioral outcome (Fig. In contrast, the ASE neuron-marked animals conditioned to avoid 20 mM NaCl, exhibited a significant increase in damaged neurons in the group that was moving toward the high salt concentration. As expected, URY neuron-marked nematodes did not show an accumulation of neurodegeneration in any of the groups (Fig. In conclusion, these results establish a direct link between neuronal degeneration and functional impairment, demonstrating the power of transcriptome clock-predicting biologically older neurons (henceforth, referred to as ‘biologically old' or ‘biologically aged'). To understand commonalities among the biologically young, as well as among the rapidly aging neuron types, we adapted a hierarchical whole-animal connectome for C. elegans26 with a rough anatomical correspondence on the x axis and directional flow of neuronal signaling on the y axis and color coded it with the predicted biological age (Fig. Of the 10 oldest neurons, 6 are amphid neurons (ADL, ASJ, ASK, ASG, ADF, ASI), the primary chemosensory organ, which is mostly ciliated27. 3h), which can be replicated in the Calico dataset (Extended Data Fig. 3a), and with the Stochastic Clock (Extended Data Fig. The sensory amphid neurons express a variety of neuropeptides, neurotransmitters, receptors and innexins to transmit the sensed cues. The numbers of expressed neuropeptides and receptors were significantly higher in amphid neurons (Extended Data Fig. 3c,d), while the number of neurotransmitter or innexins was not significantly changed (Extended Data Fig. Moreover, the numbers of neuropeptides and receptors per neuron were significantly positively correlated with the predicted biological age in the CeNGEN dataset (Extended Data Fig. 3g,h), the Calico dataset (Extended Data Fig. 3i,j) and the Stochastic Clock predictions (Extended Data Fig. To test whether neuropeptide release promotes neuronal aging, we used unc-31 mutants defective in neuropeptide release and observed a mild, yet nonsignificant, reduction in ASI degeneration (Extended Data Fig. The number of innexins and number of total synapses per neuron did not show a significant correlation with biological age (Extended Data Fig. Amphid neurons are part of the ciliated neuron classes; comparing all 28 ciliated neurons with the remaining 100 neurons also shows a significantly increased biological age (Fig. 3h), which can be replicated in the Calico dataset (Extended Data Fig. 3t) and with the Stochastic Clock (Extended Data Fig. The ciliated neurons can be further divided into five distinct classes depending on where their cilia terminate27. Neurons with cilia exposed to the environment show the highest biological age (Fig. 3i), while the other ciliated neuron classes are not significantly different from not-ciliated neurons. A similar effect can be observed in the Calico dataset (Extended Data Fig. 3v) and with the Stochastic Clock (Extended Data Fig. These results indicate that the oldest neurons are functionally related and mostly consist of ciliated sensory neurons that have contact with the environment and produce neuropeptides. To identify the transcriptional patterns and signatures underlying the biological age distinctions, we initially categorized the 128 neuron types into five distinct groups based on their biological age ranking (for example, the 25 youngest in group 1, position 26 to 50 in group 2). A fuzzy clustering analysis of 4 transcriptional clusters (determined by the standard elbow criterion to provide the best balance between cluster resolution and interpretability) identified neuronal age-dependent gene expression trends (Fig. a, Fuzzy clustering on z-score-normalized gene expression values (y axis) over the predicted aging course of the 128 CeNGEN neurons identified four clusters. The number of genes within each cluster are annotated. Dashed black lines show the central trend. b, Pathway overrepresentation analysis of the four clusters shows age-related pathways. The x axis shows pathway terms, and the y axis indicates clusters. Circle size reflects the enrichment score, defined as the log2 ratio of observed-to-expected genes per pathway (log2 enrichment ratio). Circle color represents statistical significance as –log10(false discovery rate (FDR)). Only representative nonredundant pathways are shown for visualization (Methods). c, Fraction plots displaying the fraction of nematodes at day 1 of adulthood expressing GFP in different neurons categorized as ‘healthy', ‘mildly damaged' and ‘severely damaged' that were treated with 2 mM CHX for 24 h. Three to four cohorts were analyzed, consisting of 10–30 individual nematodes. Ordinal regression (CLM) was used to test for significant differences. d, Representative fluorescence microscopy images (z-stack maximum projections) of ASE neurons in 1-day-old adult nematodes of strain NY2067 after 24-h treatment with 2 mM CHX or dimethylsulfoxide (DMSO) control. Gray arrows indicate morphologic aberrations along the neurites. e, Fraction plots displaying the fraction of nematodes expressing GFP in ASE neurons categorized as ‘healthy', ‘mildly damaged' and ‘severely damaged' that were treated with 2 mM CHX for 24 h at day 1 of adulthood. Four cohorts were analyzed, consisting of 10–30 individual nematodes. Ordinal regression (CLM) was used to test for significant differences. f, Swarm plot showing the chemotaxis index of nematodes conditioned at 0 mM (light blue) or 20 mM (dark blue) NaCl. The ASE neuronal marker strain (NY2067, mediating salt aversive conditioning) was incubated with 2 mM CHX or DMSO control for 24 h at day 1 of adulthood. from eight independent cohorts of 20–100 nematodes. A one-tailed t-test was used to test for significant differences. Cluster A is composed of genes that generally increased over biological age, and is enriched for genes involved in nucleolar functions such as rRNA and tRNA processing, ribosome biogenesis and pre-ribosomes, as well as DNA repair (Fig. Cluster B, composed of genes with highest expression in the youngest age group, an overall decline over the biological age time course including some variation and a peak in age group 4, is enriched in mRNA processing and ribosomal genes. Cluster C, comprising genes increasingly expressed over the predicted age and then sharply declining in the oldest age group, is enriched for cell junctions and synapse-related genes. Cluster D contains genes that only strongly increase in the oldest age group with functions in neuropeptide signaling and immune response. In sum, these four clusters suggest roles for translation, ribosome biogenesis and synaptic function in neuronal aging. Based on the enrichment of protein biosynthesis processes in the accelerated aging neurons, we tested whether translational activity contributes to neurodegeneration. To reduce protein biosynthesis, we treated nematodes with the translation inhibitor cycloheximide (CHX) at a dose that induces translation reduction but not full inhibition29. Here, we scored neurite degeneration in biologically older ASK and ASJ neurons, as well as in biologically younger I2 and OLL neurons. To confirm reduction of translation, we used a SUnSET assay and observed that CHX treatment significantly reduced overall protein synthesis (Extended Data Fig. Additionally, we observed reduced translation in individual neurons as assessed by the reduction of GFP expression in the presence of CHX in three of the four tested neuron types. These results further corroborate our approach using CHX treatment to reduce translation also in the analyzed neurons (Extended Data Fig. In the biologically young neurons, no effect of CHX treatment was observed (Fig. In contrast, biologically old neurons exhibited significantly less neurite deterioration upon CHX treatment (Fig. Consistently, ASE neurons that were CHX treated showed significantly less blebs than solvent control-treated neurons (Fig. 4d,e), and the nematodes showed an improved salt evasion response after they were conditioned to associate high salt concentrations with starvation (Fig. 4f), indicative of preserved neuronal health. Furthermore, we tested whether CHX treatment would have a lasting effect on neuronal integrity (Extended Data Fig. Even at adult day 4, ASJ and ASE neurons showed significantly improved health when treated with CHX for only 24 h beginning at the L4 stage. Chronic treatment for the whole duration until neurite damage scoring conferred even stronger neuroprotection. Taken together, these results indicate that translational activity promotes neurodegeneration and that reduced translation could restore neuronal integrity. Next, we addressed whether the biological age-related transcriptional patterns of neurons in young adult C. elegans (NeuronAge) are conserved in higher organisms. We then compared the conserved Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichments of NeuronAge with mouse and human brain aging datasets. We computed age correlations of z-score-normalized gene counts for all human brain regions during aging in the GTEx dataset, the Tabula Muris Senis (TMS) dataset and an additional mouse hypothalamus aging cohort (GSE157025). Similarly, we calculated the enriched pathways in brain datasets for several geroprotective treatments such as young serum injections, the platelet factor PF4, sport in humans and krill oil in C. elegans (for references, see the Methods). An unbiased clustering analysis revealed that the transcriptomic NeuronAge pattern of C. elegans and the chronological brain aging trajectories of mouse and humans cluster together (Fig. Both NeuronAge and its Calico validation showed significant positive correlations with all individual brain aging trajectories of humans and mice after multiple testing correction. To address potential methodological differences between aging and geroprotective datasets, that is, that aging patterns were derived using Pearson correlations with age, whereas geroprotective effects were analyzed using log-fold changes (logFC), we additionally computed logFC-based pathway enrichments for all aging datasets by comparing the oldest and youngest sample groups within each dataset. The resulting clustering (Extended Data Fig. 5b) confirmed that aging-related patterns remain distinct from geroprotective interventions, regardless of whether Pearson correlation or logFC-based methods were applied. These results indicate that neuronal transcriptomic aging patterns are conserved from nematodes to humans and that known geroprotective treatments anticorrelate with the aging datasets supporting their geroprotective effectiveness. The normalized enrichment scores of the conserved KEGG pathways for the indicated aging pattern or treatment effects were used for an unbiased clustering analysis. As the NeuronAge predictions cluster together with human neuronal chronological aging trajectories, we sought to identify small-molecule compounds that could delay neuronal aging. We used the 3,566 transcriptomes from a terminally differentiated neuronal cell line of the CMAP resource, consisting of 2,467 different molecules13 (Fig. Based on their correlation with the NeuronAge signature, we identified both negatively correlated (potentially neuroprotective/‘anti-NeuronAge') and positively correlated (potentially neurotoxic/‘pro-NeuronAge') compounds (Fig. Consistent with our experimental results, the transcriptome changes induced by CHX were inversely related to the NeuronAge signature, indicating that the beneficial effect of CHX that we observed above is mirrored in the transcriptome. After applying multiple computational filtering steps (Methods), we ranked the remaining candidate compounds by their correlation with NeuronAge (Fig. The top anti-NeuronAge hits contain several compounds (9 of 16) for which a protective effect in neurons has been previously documented, thus independently validating our approach (Fig. Two of 15 ‘pro-NeuronAge' compounds were shown to be detrimental, while one is potentially protective, and one was described as both harmful and protective. More than half of the top hits have, however, not been tested in neurons. a, Flowchart depicting the in silico drug screening. We computed and correlated the conserved KEGG pathway enrichments for NeuronAge and all compounds from the CMAP dataset that are measured on the neuronal cell line NEU. To obtain a manageable list, we filtered for compounds that were measured at least twice, show consistent correlations in all measurements, have a stronger correlation at the 24-h time point compared to the 6-h time point, have information in PubCHEM and have at least an absolute correlation value of 0.25. b, The top anti-NeuronAge and pro-NeuronAge compounds after the filtering steps ranked according to their Pearson correlation. Previously published neuroprotective (blue) or neurotoxic (orange) compounds are indicated. Resveratrol, which has mixed evidence in the literature, is shown in both colors. Four compounds were selected for experimental validation of our predictions and are highlighted by black arrows and their structural formula is given. c, Fraction plots displaying the fraction of nematodes at day 1 of adulthood expressing neuronal volume markers in different neurons categorized as ‘healthy', ‘mildly damaged' and ‘severely damaged' that were treated with 2.5 mM syringic acid (SA), 10 nM vanoxerine (VX), 25 nM WAY-100635 (WAY) or 250 µM (S)-(−)-Bay K8644 for 24 h. Three to four cohorts were analyzed, consisting of 10–25 individual nematodes. Statistical analysis was performed using a CLM with a logit link to account for the ordered categorical nature of neuronal damage scores (healthy < mild < severe). P values were adjusted for multiple testing using the Benjamini–Hochberg method. d, Fraction plot displaying the fraction size of healthy (blue), mildly damaged (orange) and severely damaged (red) URY neurons (JKM10 strain) on day 2 of adulthood after 48-h compound treatment with 25 nM WAY-100635 (WAY), 2.5 mM SA or water control. Statistical analysis was performed using a CLM with a logit link to account for the ordered categorical nature of neuronal damage scores (healthy < mild < severe). Pairwise comparisons between conditions were performed using the ‘emmeans' package with Tukey's adjustment for multiple testing. e, Swarm plot displaying the avoidance index of compound-treated JKM10 (URYp::GFP) nematodes (25 nM WAY-100635 (WAY), 2.5 mM SA or water control for 48 h) on day 2 of adulthood. from five independent cohorts of 50–75 nematodes. Two-way ANOVA + Tukey post hoc test were used to test for significant differences. In summary, 11 of 31 compounds have documented neuroprotective effects, of which 9 are predicted to revert NeuronAge, that is ‘anti-NeuronAge', with our in silico approach. Similarly, 2 of 31 compounds are known to be neurotoxic, and both are predicted correctly to be ‘pro-aging', giving weight to the potential that an in silico screen can identify compounds that could be repurposed as either neuroprotective or neurotoxic agents. Next, we tested whether several compounds that we predicted to be anti-NeuronAge, that is, neuroprotective, could prevent the age-related functional decline of neurons. We chose two compounds, which were among the most strongly anticorrelated with NeuronAge patterns, BRD-K13195996 and vanoxerine (Fig. The chemical identity of the phenolic compound BRD-K13195996 is 3-hydroxy-4,5-dimethoxybenzoic acid, which is related to 4-hydroxy-3,5-dimethoxybenzoic acid, which is also known as syringic acid. Syringic acid is a naturally occurring secondary compound derived from edible plants and fruits, for example, olives, walnuts and grapes30. A correlation between the antioxidative properties of syringic acid and reduced neurotoxicity following bisphenol A insult has recently been shown31, yet no clear mechanism was reported so far32. Vanoxerine is a potent dopamine uptake inhibitor and was tested as a supportive agent in cocaine-abuse medications33; moreover, vanoxerine was observed to impede colorectal cancer stem cell functions by repressing G9a expression34. Vanoxerine is so far not reported to exhibit neuroprotective effects. We applied both compounds to L4 stage nematodes for a 24-h short-term treatment. We assessed neurite degeneration in the biologically old ASJ and ASK neurons and observed a significantly reduced deterioration for both compounds (Fig. In contrast, the biologically young OLL neurons, serving as controls, were not significantly affected by either of the compounds (Extended Data Fig. This indicates that both compounds interfere with the physiological degeneration process of the rapidly aging neurons and can restore a healthy neuron state. Next, we assessed whether the NeuronAge compound predictions could also identify neurotoxic compounds and hence serve for pharmacological risk assessment. We tested the compounds WAY-100635, for which so far no adverse effects on neuron health have been reported, and Bay K8644 on the health of the young predicted I2 and OLL neurons. We observed that WAY-100635 induced significant neurite deterioration in I2 but not in OLL neurons, while Bay K8644 induced significant degeneration in both I2 and OLL neurons (Fig. ASK neurons that already display neurodegeneration early in life were significantly further compromised by Bay K8644 treatment (Extended Data Fig. 6e), confirming the neurotoxic effect of Bay K8644. Additionally, resveratrol, which was reported to have ambiguous effects, was positively correlated with NeuronAge (Fig. 6b), and we observed significantly increased neuronal damage in I2, ASJ and ASK neurons, with a similar but nonsignificant trend observed in OLL neurons (Extended Data Fig. 6f,g), supporting our predicted detrimental impact. Taken together, we validated the anti-NeuronAge compound prediction method by identifying known neuroprotectors as well as discovering previously unknown neuroprotective molecules. In reverse, the prediction of a positive correlation with NeuronAge revealed neurotoxic compound properties. Finally, we assessed whether drug-induced neuroprotection or degeneration preserved or compromised neuronal function, respectively. We treated URY neuron-marked animals with either WAY-100635 or syringic acid and analyzed neurodegeneration and pathogen avoidance that depends on the integrity of URY neurons (Fig. First, we confirmed that WAY-100635 exerts a significant neurotoxic effect in URY neurons (Fig. As expected from the already large healthy fraction of URY neurons, the beneficial effect of syringic acid was slight but not significant (Extended Data Fig. To exclude the possibility that bacterial metabolism of the compounds might confound the analysis, we pre-incubated WAY-100635 and syringic acid with either live or dead Db11 bacteria before nematode exposure (Extended Data Fig. 6h–j) and observed no significant differences thus validating this approach. WAY-100635 treatment severely impaired pathogen avoidance from Db11 bacteria compared to solvent control treatment (Fig. 6e), while syringic acid treatment led to a small, nonsignificant increase in avoidance reaction, in line with the slight improvement in the preservation of URY neurons (Fig. This observation further corroborates the functional outcome of the morphological degeneration and substantiates that our prediction approach can be used to identify neuroprotective and neurotoxic compounds (Fig. In summary, our compound validation demonstrates that predicted anti-NeuronAge compounds significantly reduce neurite degeneration and preserve neuronal function, while predicted neurotoxic compounds exacerbate neuronal damage and disturb neuronal function. These results establish our in silico screening approach as an effective approach for identifying interventions to prevent neurodegeneration, as well as for identifying potential neurotoxic agents. Why distinct neuron types exhibit different susceptibilities to age-dependent degeneration and the associated neurodegenerative diseases has remained largely unclear. While differences in interindividual aging are commonly known, differential aging of organs within the same organism, and aging variance between cells of the same tissue have only recently been observed35,36. Using the two distinct aging clock paradigms, we found a substantial diversity of biological age predictions with specific neuron types displaying accelerated or delayed aging in chronologically young animals. Similar to such early detection of aging, epigenetic clocks in mice start advancing right after their resetting at the ground-zero stage during gastrulation37 and in humans they detected aging during childhood38. The substantial heterogeneity in neuronal aging was further supported by the age-dependent decline of long gene expression. Particularly ciliated sensory neurons show an increased biological age, which may result from their exposure to the environment or could indicate that their utility is restricted to sensing optimal environmental conditions before entering reproductive age. Similarly, degeneration of the olfactory bulb, which is enriched by environmentally exposed, ciliated neurons, and loss of olfaction are among the first symptoms of Alzheimer's disease-related degeneration in humans39. The biological age-correlated transcriptional patterns associated high translational loads with accelerated aging. Age-related changes in the translational machinery have been observed across various species, and downregulation of translation has been shown to extend lifespan upon dietary restriction, downregulation of mTOR or CHX treatment40,41. In C. elegans, nucleolar size that is linked to ribosome biogenesis inversely correlates with lifespan42, while aging is marked by declining protein synthesis43,44 and increased translation errors45. Stochiometric changes of the ribosome lead to accumulation of protein aggregates in the brain of old Nothobranchius furzeri46, which is in line with the enrichment of ribosomal proteins in the insoluble protein fraction of old C. elegans47. Consistently, we observed that transient translation inhibition by CHX treatment is sufficient to prevent the degeneration of the fast-aging neuron types. The transcriptional patterns of nematode neuronal biological age differences are significantly correlated with mouse and human brain chronological aging trajectories, and anticorrelated with known geroprotective interventions such as young plasma treatment or sport. This conservation shows the potential of identifying conserved mechanisms that underlie the biological aging differences and might determine the susceptibility of specific neuron types to undergo degeneration and potentially contribute to specific neurodegenerative diseases. The conservation of transcriptome age trajectories allowed us to design an in silico compound screen using the human CMAP dataset13. Such approaches have previously identified geroprotective compounds that either induce a ‘youthful' state as predicted through an age-classification approach leveraging the GTEx48 transcriptomic dataset49, by identifying compounds that counteract age-associated transcriptomic shifts using conserved aging signatures50, by mimicking longevity FOXO3 overexpression51, or induce a ‘youthful' matreotype52. Comparing transcriptomic data from C. elegans neurons to the CMAP resource, we identified known and previously unexplored neuroprotective small-molecule compounds. Syringic acid was described to reduce oxidative stress and neuroinflammation, potentially by enhancing mitochondrial function and attenuation of acetylcholinesterase activity53,54,55. The 1,4-dialkylpiperazine derivative vanoxerine is a potent dopamine uptake inhibitor and was tested as a supportive agent in cocaine-abuse medications33. Vanoxerine was observed to impede colorectal cancer stem cell functions by repressing G9a expression34, and it was tested in clinical trials for use as an anti-arrhythmic drug56, but a phase III study was terminated due to the occurrence of ventricular proarrhythmia57. The development of analogs overcoming vanoxerine's limitations are a work in progress58. By testing syringic acid and vanoxerine as examples of the top-scoring compounds, we indeed found that they extend the integrity of fast-aging neurons indicating a biological age deceleration. Conversely, the pro-aging prediction revealed neurotoxic effects of compounds and could thus be highly valuable in risk assessment. WAY-100635 is an antagonist of the serotonin 5-HT1A receptor, acting both at presynaptic autoreceptors and postsynaptic sites to modulate serotonergic signaling and increase extracellular 5-HT levels59,60,61. In addition, WAY-100635 has an affinity for the dopamine D4 receptor, where it acts as a potent partial agonist at higher concentrations62,63. The carbon-11-labeled form of WAY-100635 ([carbonyl-¹¹C]WAY-100635) is used as a radioligand for in vivo positron emission tomography imaging of 5-HT1A receptors in the human brain64,65,66. While often considered safe, behavioral studies suggest context-specific effects: for example, it not only exacerbates depression-like phenotypes following mild traumatic brain injury in mice67, but also prevents selective serotonin reuptake inhibitor-induced sexual dysfunction when coadministered with fluoxetine68. WAY-100635 induced significant neurite deterioration in I2 and URY neurons, underscoring the power of our approach for predicting neurotoxicity. Bay K8644 is a potent L-type calcium channel agonist, structurally related to dihydropyridines such as nifedipine. By promoting prolonged opening of L-type voltage-gated calcium channels, Bay K8644 enhances calcium influx into neurons and has been used as a research tool to study calcium-dependent neuronal processes69. In animal models, Bay K8644 has been shown to induce seizures, oxidative stress and excitotoxic neuronal damage70,71,72. Consistently, we validated the predicted neurodegenerative effect of Bay K8644 across multiple neuron types. Notably, our analysis predicted resveratrol to be pro-aging, whereas the published effects have been controversial. Our in vivo validation demonstrates that resveratrol does not have a protective, but rather detrimental impact on neuronal health. Corroborating our findings, a recent study found resveratrol-induced brain atrophy in lemurs73. These results further validate our in silico drug screen for both identifying neuroprotective compounds that could preserve neuronal function and determining the risk of neurotoxic compounds that accelerate neurodegeneration. Taken together, we here define the biological basis for the distinct susceptibility of neurons to undergo age-dependent degeneration. We establish the utility of using aging clocks to identify neuron-type-specific biological aging differences and based on their transcriptome profiles reveal conserved aging patterns also present in human brain aging. We show that this approach is suitable for identifying neuroprotective molecules and propose that they could be useful in delaying neuronal aging and protect from age-associated degeneration. Nematodes were cultured on nematode growth medium (NGM) agar plates at 20 °C under standard conditions unless stated otherwise. All age statements given in this publication consider the first day of adulthood as day 1. A complete strain list can be found in the Supplementary Information. Special care was exerted for maintaining the cultures (specifically for maintaining the strains carrying extrachromosomal arrays): To maintain good expression levels in each population, healthy-looking adults with strong, but not extreme, fluorescence were passaged. Without this careful selection, the fluorescence signals were lost over the course of a few generations. Age synchronization for experiments was achieved by L4 picking. For all experiments, individuals displaying a medium to strong fluorescence were selected (this group consists of about 85% of the fluorescent nematodes); individuals with neither exceptionally strong fluorescence (≈3–5% of the fluorescent nematodes) nor weak fluorescence (≈5–10% of the fluorescent nematodes) were selected for experiments or passaging. For imaging, nematodes were placed in a drop of 250 mM NaN3 on a 2% agarose pad. z-stacks of nematode heads/tails were acquired using 2-µm step width. Acquisition time was set between 100 ms to 3 s per plane to achieve a good signal-to-noise ratio. Recorded z-stack images of neurons were analyzed manually, counting blebs, large spherical outgrowths, branching, breaks and necrosis on the dendrites of the analyzed neurons. Images were classified according to the degree of aberration: necrotic neurons, broken or truncated neurons, neurons with ≥10 blebs or ≥3 outgrowths were scored as ‘severely damaged'; neurons with 5–9 blebs, or 2 outgrowths were classified as ‘mildly damaged'; and neurons with <5 blebs or <2 outgrowths were classified as ‘healthy'. To statistically assess neuronal damage across conditions and time points, we used CLMs using the ordinal package in R. A logit link function was applied to estimate the cumulative probabilities of increasing damage severity. For neuron-specific analyses of damage progression over time, the time point was modeled as a numeric predictor (L4 = 0, day (D) 1 = 1, D4 = 4, D7 = 7), and separate CLMs were fit for each neuron type. Observed neuron counts per category were incorporated as weights using the weights argument. P values for the slope term were adjusted for multiple testing across neurons using the Benjamini–Hochberg method. To compare damage susceptibility between neuron types, a full model including neuron identity, time point and their interaction was fit (score ~ neuron × age). Estimated marginal means were computed using the emmeans package, and Tukey-adjusted P values were reported for all pairwise neuron contrasts. For comparisons between two conditions (for example, treatment versus control), we modeled the score as a function of the condition only (score ~ condition), again using CLMs with logit link and frequency weights. Multiple-testing correction across neurons was performed using the Benjamini–Hochberg method, where applicable. Standard NGM plates, seeded with OP50, were inactivated with 500 mJ/cm2 of 251-nm UV-C light (Stratalinker 2400) and afterward were subjected to compound coating by dropwise addition of compounds directly to the agar surface. Plates were dried for at least 1 h before transferring L4-stage nematodes onto them. Nematodes were incubated with the compounds for 24 h and then used for neurite imaging. The final concentration of compounds was: CHX, 2 mM; syringic acid, 2.5 mM; vanoxerine, 10 nM; WAY-100635, 25 nM; (S)-(−)-Bay K8644, 250 µM; resveratrol, 200 µM. Control nematodes were incubated on appropriate solvent control-coated plates (either water, for syringic acid and WAY-100635, ≤5‰ DMSO, for CHX and vanoxerine, or 1‰ ethanol, for resveratrol). S. marcescens Db11 bacteria were inoculated into LB medium with kanamycin, tetracycline and streptomycin from glycerol stock and incubated overnight with shaking at 150 rpm at 25 °C. Control E. coli OP50 bacteria were grown in LB medium without antibiotics and incubated overnight with shaking at 150 rpm at 37 °C. NGM plates (6 cm) were cast 3 days before seeding. The plates were seeded with 350 µl bacterial solution, generating a perfect round spot in the middle of the plate and allowed to dry for 2 h. For compound treatment, 2.5 mM syringic acid or 25 nM WAY-100635 was added before seeding and allowed to dry. Around 15–30 L4-stage nematodes of strains JKM10, IG10 and MT21910 were added directly into the bacterial lawn and incubated for 48 h. Nematodes were scored for their presence on or avoidance of the lawn. Five independent cohorts of 4–5 plates each were analyzed, and the position of the nematodes was checked four times with a 5-min interval time. The aversion index was calculated as the number of nematodes outside the lawn divided by total number of nematodes. Potential compound metabolization by the living Db11 bacteria has been tested in prior experiments. For testing compounds and pathogen avoidance from S. marcescens Db11, bacteria could not be inactivated with UV irradiation because it would negate the aversion behavior. To test whether Db11 bacteria would metabolize the compounds, a test treatment was performed: 2.5 mM syringic acid or 25 µM WAY-100635 was incubated with living or dead Db11 bacteria for 2 days; afterward, bacteria were removed by sterile filtration and the compound-containing flowthrough was applied on UV-inactivated OP50. L4-stage JKM10 nematodes were placed on these plates, and neurodegeneration was assessed 24 h later. There was no significant difference between the compounds that were pretreated with living or dead Db11 bacteria (Extended Data Fig. Adapted from Lim et al.25, nematodes were synchronized by L4 picking and grown for an additional day to their first day of adulthood, then nematodes were rinsed off the plates with M9 medium and washed twice with M9. To establish the NaCl gradient on the assay plates, agar plugs containing 100 mM NaCl (2% (wt/vol) agar, 100 mM NaCl, 1 mM CaCl2, 1 mM MgSO4, 5 mM KPO42− pH 6.0) were placed on one side on the plate and incubated at 23 °C for 3 h. After the gradient was established, salt-agar plugs were removed and 3 µl 500 mM NaN3 was added to the spot where the plug was placed. After 3 h of conditioning, excessive medium was removed, and nematodes were transferred onto the center of the prepared salt-gradient assay plates. Nematodes were allowed to crawl freely on the plate according to their preferences for 30 min and at 20 °C. Afterward, nematodes that remained on the origin, that moved toward high salt and those that moved toward low salt were counted, and the chemotaxis index was calculated as: ci = (number on high salt – number on low salt) / (total number – number on origin). For CHX treatment, nematodes were synchronized by L4 picking and placed on an OP50-seeded NGM plate, coated with 2 mM CHX, for 24 h. After compound incubation, the nematodes were processed as described above. The BitAge clock10 was used as described previously. Briefly, each sample was binarized, that is, genes higher than the median expression value within each sample after removing genes with zero counts, were set to 1, and the remaining genes to 0. The BitAge coefficients for the 576 clock genes are added up for all genes in a given sample that is 1 after binarization. For the Calico dataset, only single-neuron types were considered, that is, cells annotated as multiple neuron types were excluded. For neuron types with two biological replicates, the predicted biological ages of the replicates were averaged to obtain a single age prediction per neuron type. The Stochastic Clock was used as described previously12. Briefly, each sample was log10-transformed after the addition of one pseudo-count. Subsequently, the samples were min–max normalized to bring each sample within the range (0, 1), and then binarized as described above. The normalized counts were then added up for all 1,010 stochastic data-based clock genes. For the Calico dataset, where some neuron types had two biological replicates, the predicted biological ages of the replicates were averaged to obtain a single age prediction per neuron type. Note that the Stochastic Clock might result in slightly different genes every time a clock is trained. First, we downloaded the differentially expressed genes for each neuron type and all other cells in the CeNGEN14 dataset (https://cengen.shinyapps.io/CengenApp/) with the statistical test ‘Wilcoxon on single cells'. This gives a list of genes with logFC, and percentage expression in the specific neuron type and all other cells. We further filtered this list of significant genes by requiring that the gene is expressed in at least 90% of cells of the specific neuron type and at most 10% in all other cell types. In total, 39 neuron types had no genes with these requirements, that is, 89 neuron types were used for further analysis. To calculate a two-sided permutation test, we calculated the median log10 gene length of the genes and compared it to the 100,000-permutation median. In a complementary analysis, we calculated the median log10 gene length of the marker genes for each of the neuron types individually. These median values were correlated with the predicted biological age using a Pearson correlation and visualized with a linear regression plot. To quantify transcriptomic aging patterns across neuronal cell types, we defined NeuronAge as the correlation between gene expression and predicted biological age with BitAge. TPM values for all 128 neuron types were obtained from the CeNGEN transcriptomic dataset, and each of the 18,475 genes was z-score normalized across all 128 neuron types. For every gene, we calculated the Pearson correlation between its z-scored TPM values and the predicted ages across all 128 neuronal cell types. We downloaded and adapted the Cytoscape file from Cook et al.26 by adding head neurons, deleting nonneuronal cell types and color coding neurons by their predicted Age with BitAge on the CeNGEN dataset. The NeuroType.xlsx file was downloaded from https://www.wormatlas.org/neuronalwiring.html. For each of the 128 neuron types, we summed the median total number of synapses in the head, tail and mid-body. To identify transcriptional patterns associated with the predicted neuronal age, we ranked the 128 neuron types based on their predicted age and grouped them into five bins: (1) [97, 110], (2) [110, 120], (3) [120, 130], (4) [130, 140] and (5) [140, 180]. Next, we used fuzzy clustering with Mfuzz (v2.58.0)74 to identify transcriptional clusters across the predicted age bins. The elbow method was computed with the Dmin function of Mfuzz and indicated an optimal number of four clusters. The genes belonging to each cluster were subsequently used for a pathway enrichment analysis with clusterprofiler (v4.9.2.002)75, with maxGSSize = 500 and the list of all 9,950 genes as the background gene list. All significantly enriched pathways (adjusted P < 0.05) were identified, and redundant pathways were collapsed using semantic similarity filtering. Although this procedure substantially reduced redundancy, the remaining number of nonoverlapping pathways was still too large to display comprehensively in a single figure. For visualization purposes, we therefore selected a representative subset of pathways from each cluster based on (i) statistical significance, (ii) effect size and (iii) biological interpretability. We processed several public datasets (GTEx dataset48, the TMS dataset76 and an additional mouse hypothalamus aging cohort (GSE157025), young serum injections77, platelet factor PF4 (ref. TPM-normalized gene expression values for human brain tissues from the GTEx v8 data81 release (that is, amygdala, anterior cingulate cortex, caudate basal ganglia, cerebellar hemisphere, cerebellum, cortex, frontal cortex, hippocampus, hypothalamus, nuclear accumbens basal, putamen basal ganglia, substantia nigra) were correlated with the chronological age (the midpoints of the publicly available age bins). The mouse aging time course for hypothalamus data from GSE157025 was downloaded and a gene-wise correlation with the chronological age calculated. Whole-brain data from the TMS (GSE132040) was downloaded and edgeR (v3.40.2)82 was used to calculate normalized expression values. Differentially expressed genes for mouse hippocampus data treated with the platelet factor PF4 or saline control were downloaded from GSE173254. We multiplied the logFCs by −1 to always compare treatment versus control, instead of control versus treatment. Differentially expressed genes for mouse hippocampus data treated with young serum or sham were downloaded from GSE234667. C. elegans whole-nematode data treated with krill oil from GSE207152. We used edgeR (v3.40.2)82 to calculate normalized expression values and calculated z-scores for each gene over all samples. Differential expressed genes upon physical activity were downloaded from the supplementary data (Supplemental Table 2) from Berchtold et al.79. The Pearson correlation values of all genes of (1)–(3), the logFC of all genes for (4) and (5) and the \({\beta }_{3}\) coefficients for (6) were used to calculate enriched pathways analysis with fgseaMultilevel from the fgsea R package83 and with nPermSimple = 1,000 for all conserved KEGG pathways between C. elegans, mouse and humans. For (7), the ‘anti-aging/Alzheimer's disease' genes were split into genes that are upregulated and, respectively, downregulated upon exercise. Both gene sets were used for an enrichment analysis with enricher from enrichplot v1.18.0 with maxGSSize = 500 and minGSSize = 5, and all genes were quantified in the supplementary data from Berchtold et al.79 as the background gene list. For both enrichment analyses, the enrichment fold change, that is, the number of observed genes per pathway divided by the number of expected genes per pathway, was calculated. Finally, the fold changes were combined, that is, pathways with a bigger fold-change enrichment in the downregulated genes were multiplied by −1. CeNGENApp (https://cengen.shinyapps.io/CengenApp/) was used to find differentially expressed genes between the ten oldest predicted CeNGEN neurons (according to BitAge) and the ten youngest predicted neurons. Raw gene expression values for human brain tissues from the GTEx v8 data81 release (that is, amygdala, anterior cingulate cortex, caudate basal ganglia, cerebellar hemisphere, cerebellum, cortex, frontal cortex, hippocampus, hypothalamus, nuclear accumbens basal, putamen basal ganglia, substantia nigra) were normalized with edgeR and differentially expressed genes calculated between old (≥70 years) and young (≤40 years). The whole-brain data from the TMS (GSE132040) were normalized with edgeR (v3.40.2)82, and differentially expressed genes were calculated between old (≥21 months) and young (1 month) mice. The mouse aging time course data for the hypothalamus from GSE157025 were normalized with edgeR (v3.40.2)82, and differentially expressed genes were calculated between old (24 months) and young (5 day) mice. The enriched pathways were computed as described above. The CMAP resource uses the L1000 array, which measures 978 landmark transcripts, which can be used to infer most of the remaining transcriptome with high accuracy13. Here we used all available L1000 datasets for a human differentiated neuronal cell line. We downloaded the aggregated level 5 L1000 Connectivity Map13 data from GSE92742 and performed pathway enrichment analysis with fgseaMultilevel from the fgsea R package83 with nPermSimple = 1,000 and the conserved KEGG pathways between human and C. elegans for each of the samples in the level 5 dataset. To compare it to the NeuronAge pattern, we first calculated the z-score of each gene that has at least some gene counts across the 128 neurons of the CeNGEN dataset. We correlated these z-score-normalized genes with the biological age prediction from BitAge. The resulting Pearson correlation values for all genes were used for a pathway enrichment analysis with fgseaMultilevel and the conserved KEGG pathways between human and C. elegans. To identify whether any compound might revert the NeuronAge gene expression pattern on the pathway level, we correlated the normalized enrichment scores of the NeuronAge KEGG pathway enrichment analysis with all the normalized enrichment scores that we calculated from the CMAP dataset. Next, we filtered for only compounds that were tested in the neuronal cell line (‘NEU'). Compounds had to be tested at least twice, with all measurements resulting in correlations in the same direction. Additionally, we filtered for compounds that were measured at 24 h and 6 h and took only those compounds that showed a stronger correlation into the same direction at the 24-h time point compared to the 6-h time point. Lastly, we filtered out those compounds that had no information available at PubCHEM and used a correlation threshold of 0.25 (respective −0.25). The assignment of known protective or detrimental compounds is based on literature reports: The glycogen synthase kinase 3 (GSK3) inhibitor AR-A014418 was shown to inhibit beta-amyloid-induced neurodegeneration84; the selective serotonin reuptake inhibitor fluoxetine protects against neurotoxicity and neurodegeneration85; the PPAR-alpha activator gemfibrozil exhibits neuroprotective effects via upregulating pro-survival factors and suppressing inflammation86; the kinase inhibitor sorafenib protects against neurodegeneration in C. elegans87; the selective aryl hydrocarbon receptor modulator 3,3'-diindolylmethane is neuroprotective and promotes brain-derived neurotrophic factor88; the insulin-sensitizing agent rosiglitazone exhibits neuroprotective effects in the eye and the brain89; the p38 MAPK inhibitor SB202190 was shown to reduce hippocampal apoptosis and rescue spatial learning as well as memory deficits in rats90; dibutyryl-cAMP-Na (dBcAMP) elevates cAMP levels and protects against neurodegeneration in stab wound or kainic acid injuries91; and the catecholamine-O-methyltransferase inhibitor tolcapone was shown to improve cognitive function92. Bay K8644 is described to be neurotoxic70; and amiodarone induces neuronal apoptosis93 and is known to induce adverse neurological effects94. Tacedinaline/CI-994 is a class I histone deacetylase inhibitor, correlates positively with NeuronAge, and was shown to promote functional recovery following spinal cord injury95, and to enhance synaptic and structural neuroplasticity96. Likewise, resveratrol is potentially neuroprotective due to a hormetic response97, while there is evidence that it could lead to impaired brain integrity73. To test whether GFP expression induces neuronal aberrations, we correlated GFP expression intensity and blebbing. In brief, fluorescence images of PY6457 (ASK neurons) and OE3010 (ASJ neurons) nematodes at the first day of adulthood were recorded using an epifluorescence microscope. PY6457 and OE3010 were selected because they carry the neuron-type-specific GFP marker as an extrachromosomal array that has a strongly varying expressivity between individuals. Blebbing was observed as the major degeneration hallmark in ASJ and ASK neurons and, therefore, used as single correlation criterion. Four cohorts of 10–20 nematodes per strain were accumulated in single plots and the correlation was analyzed by Pearson's correlation. To test if bleb appearance is fully random or linked to development of conserved structures along neurites, bleb distance from neurite start (at the site of the mouth) was measured using Fiji for every bleb on each neurite analyzed from three cohorts of 10–20 individual nematodes each. Relative positions of blebs were calculated by dividing the observed bleb's position (for example, 55 µm) by the total length of the analyzed neurite (for example, 98 µm). Relative positions were displayed as a histogram using 12 bins, and data from three cohorts were aggregated. A Pearson correlation versus 100 random data tracks was done using GraphPad Prism 10. To calculate 100 tracks of a random damage distribution, a Python script was used with the following assumptions: (1) a random number of blebs (in 8–20) can appear, matching observed biological variability; (2) new blebs appear primarily at random but are slightly more likely to occur near previously formed blebs, reflecting spatial clustering seen in vivo; (3) already established blebs increase chances of blebs in their vicinity; (4) blebs cannot appear at the same position; (5) the proximal 10% of the neurite corresponds to the ciliary region, where structural complexity and imaging artifacts reduce the detectability of blebs, and this was modeled by suppressing bleb placement in this region; (5) neurite lengths were randomly sampled within the empirically observed range (80–100 µm). All simulation parameters, including neurite length distributions, bleb counts, clustering strength and the extent of the protected proximal region, were derived directly from empirical measurements of ASJ, ASK and OLL neurons imaged in vivo. For each simulated neurite, bleb positions were generated iteratively and normalized to neurite length to match the experimental measurements. Only nematodes that showed a easily visible fluorescence under a dissecting microscope were selected and singled onto OP50-seeded NGM plates. Nematodes were passaged every second day to a fresh plate. Individuals were tracked over the course of 12 days or until they died, or until all GFP-marked neurons disappeared. Neuron loss was analyzed according to Kaplan–Meier using the accumulated data from three cohorts of each of the 12 individuals (totaling 36 nematodes per strain), and significant differences were assessed by a log-rank test. Additionally, the first day a neuron degenerated in every single individual was scored and plotted as a histogram showing the incidence of first neuron death. Wild-type nematodes were synchronized by L4 picking (≈75 nematodes per cohort and condition) and placed on OP50-seeded NGM plates coated with 2 mM CHX or DMSO as control. After 24 h, plates were rinsed with S-Basal, and nematodes were collected in 1.5-ml tubes. Nematodes were washed once with S-Basal. Freshly grown OP50 bacteria were pelleted by centrifugation (20 min at 4,000 rpm at 12 °C) and resuspended in 0.1 Vol. Nematodes were resuspended with 200 µl of the concentrated bacteria. Puromycin was added to a final concentration of 500 µg ml−1 and topped up to 1 ml with S-Basal. CHX-treated samples were supplemented with 2 mM CHX additionally. Puromycin labeling was performed for 3 h, and nematodes were rotating at 20 rpm on a spin wheel at 20 °C. Afterward, nematodes were washed thrice with S-Basal to remove residual puromycin. Finally, nematodes were resuspended in 30 µl 4xSDS-Sample buffer, boiled at 95 °C for 5 min and then stored at −20 °C. Frozen samples were thawed and boiled at 95 °C for 5 min. In total, 15 µl of lysate was loaded on a 12% SDS–PAGE and run at 30 mA for 2 h. Western blot transfer onto a nitrocellulose membrane was performed using Trans-blot Turbo (Bio-Rad) according to the manufacturer's specifications (running at 2.0 A for 10 min). Ponceau staining was washed out three times with TBS-T. Membrane was blocked for 2 h in TBS-T + 5% (wt/vol) milk powder. Primary antibody incubation was done with 4G11 anti-puromycin antibody from mouse (1:1,000 dilution) in TBS-T + 2.5% (wt/vol) milk powder at 4 °C with shaking overnight. Afterward, membrane was washed three times with TBS-T and incubated with horseradish peroxidase-conjugated anti-mouse antibody (1:5,000 dilution) for 2 h. Blot was developed using ECL solution (Pierce) in an Amersham Imager 600. The acquisition time was ≈2 min. Three cohorts per strain and condition (totaling 28–52 nematodes) were used for epifluorescence microscopy. Nematodes were prescreened to find the highest fluorescence intensity per strain; imaging settings were adjusted to not overexpose the nematode with the strongest fluorescence per strain. Imaging was performed on L4 larvae; the remaining L4 larvae from synchronization were subjected onto OP50-seeded NGM plates coated with 2 mM CHX or DMSO control and imaged after 24 h. Imaging settings were kept constant within strains. Imaging settings were kept constant between cohorts. was displayed as a swarm plot, and ANOVA + Tukey post hoc test were performed. Three cohorts of nematodes expressing GFP in ASE, ASJ, ASK or OLL neurons have been treated with 2 mM CHX for either 24 h or 96 h starting from stage L4 onward. Neurite imaging was performed on day 4 of adulthood. Damage classification in ‘healthy', ‘damaged' and ‘severely damaged' neurons was done as described in the main text. Ordinal regression (CLM) was used to test for significant differences (using Python 3.12 library: statsmodels: 0.14.3). All data are presented as the mean ± s.d. Number of cohorts (ℕ), individuals (n) and technical replicates (N) are stated in the figures and their respective figure legends. All statistics are two sided unless stated otherwise. Data normality was assessed by the Shapiro–Wilk test. Independent t-tests were calculated with Python's SciPy98 v1.5.1 stats.ttest_ind function. Cohen's h100 as a measure of effect size was calculated by hand with Python's NumPy101 v1.18.5. Plots were generated with Python's Seaborn102 v.0.11.0, matplotlib103 v.3.3.0 or GraphPad Prism 10. Scatterplots showing a linear regression model fit are shown with a 95% confidence interval. No statistical method was used to predetermine sample size. Sample sizes were determined based on established practices in the field and on empirical considerations. Group sizes of 10–30 animals were used for neurite degeneration scoring, and for chemotaxis assays 50–100 animals were used to balance practical feasibility and the ability to detect meaningful biological effects. Researchers were blinded during data analysis (folders/images were listed and assigned random alphanumeric IDs before analysis; after analysis, files were named back accordingly). If nematodes were damaged during handling, results from them were censored; whole-nematode cohorts were excluded if less than 80% of the desired animals survived until analysis day, or if nematodes were growing slowly (delayed by at least a day compared to usual growth rate of the respective strain) or were apparently sick (slow movements, begging, lethargic; this affected one cohort of MT21910 L4s in Fig. 2c and one subgroup of JKM10 in Fig. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The bulk CeNGEN dataset was downloaded via https://cengen.org/storage/Barrett_et_al_2022_CeNGEN_bulk_RNAseq_data.tsv. The Calico dataset was downloaded from https://c.elegans.aging.atlas.research.calicolabs.com/data/. The neuron-specific information was assessed at https://www.wormatlas.org/. The gene length information was downloaded from https://wormbase.org/. The CMAP data were downloaded from GSE92742. Data for the heat map were downloaded from the Gene Expression Omnibus under accession numbers GSE157025, GSE132040, GSE173254, GSE234667 and GSE207152, from the supplementary data from ref. 79 or the GTEx v8 database via https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression/. Raw values and summaries of all statistics used are included in the Extended Data. Source data are provided with this paper. All scripts and code used are available on GitHub via https://github.com/Meyer-DH/NeuronAging/. Echávarri, C. et al. Atrophy in the parahippocampal gyrus as an early biomarker of Alzheimer's disease. Braak, H. & Braak, E. Neurofibrillary changes confined to the entorhinal region and an abundance of cortical amyloid in cases of presenile and senile dementia. Murphy, C. Olfactory and other sensory impairments in Alzheimer disease. Linking the cerebellum to Parkinson disease: an update. Surmeier, D. J. Determinants of dopaminergic neuron loss in Parkinson's disease. Jin, K. et al. Brain-wide cell-type-specific transcriptomic signatures of healthy ageing in mice. Sun, E. D. et al. Spatial transcriptomic clocks reveal cell proximity effects in brain ageing. Lu, A. T. et al. Universal DNA methylation age across mammalian tissues. BiT age: a transcriptome-based aging clock near the theoretical limit of accuracy. Organ aging signatures in the plasma proteome track health and disease. Aging clocks based on accumulating stochastic variation. Subramanian, A. et al. A next generation connectivity map: L1000 platform and the first 1,000,000 profiles. Taylor, S. R. et al. Molecular topography of an entire nervous system. Individual cell types in C. elegans age differently and activate distinct cell-protective responses. Integrating bulk and single cell RNA-seq refines transcriptomic profiles of individual C. elegans neurons. Wang, X. et al. Ageing induces tissue-specific transcriptomic changes in Caenorhabditis elegans. Aging is associated with a systemic length-associated transcriptome imbalance. Gyenis, A. et al. Genome-wide RNA polymerase stalling shapes the transcriptome during aging. Differential impact of diesel exhaust particles on glutamatergic and dopaminergic neurons in Caenorhabditis elegans: a neurodegenerative perspective. Murray, S. M., Waddell, B. M. & Wu, C. -W. Neuron-specific toxicity of chronic acrylamide exposure in C. elegans. Lian Chew, Y., Fan, X., Götz, J. & Nicholas, H. R. Aging in the nervous system of Caenorhabditis elegans. Novel amyloid-beta pathology C. elegans model reveals distinct neurons as seeds of pathogenicity. Detection and avoidance of a natural product from the pathogenic bacterium Serratia marcescens by Caenorhabditis elegans. Lim, J. P. et al. Loss of CaMKI function disrupts salt aversive learning in C. elegans. Cook, S. J. et al. Whole-animal connectomes of both Caenorhabditis elegans sexes. & Burnell, A. M. Dauer larva recovery in the nematode Caenorhabditis elegans—II. The effect of inhibitors of protein synthesis on recovery, growth and pharyngeal pumping. Srinivasulu, C., Ramgopal, M., Ramanjaneyulu, G., Anuradha, C. M. & Suresh Kumar, C. Syringic acid (SA) ‒ a review of its occurrence, biosynthesis, pharmacological and industrial importance. The protective effects of syringic acid on bisphenol A-induced neurotoxicity possibly through AMPK/PGC-1α/Fndc5 and CREB/BDNF signaling pathways. The role of syringic acid as a neuroprotective agent for neurodegenerative disorders and future expectations. Rothman, R. B., Baumann, M. H., Prisinzano, T. E. & Newman, A. H. Dopamine transport inhibitors based on GBR12909 and benztropine as potential medications to treat cocaine addiction. The dopamine transporter antagonist vanoxerine inhibits G9a and suppresses cancer stem cell functions in colon tumors. Schaum, N. et al. Ageing hallmarks exhibit organ-specific temporal signatures. Pálovics, R. et al. Molecular hallmarks of heterochronic parabiosis at single-cell resolution. Kerepesi, C., Zhang, B., Lee, S. -G., Trapp, A. & Gladyshev, V. N. Epigenetic clocks reveal a rejuvenation event during embryogenesis followed by aging. Horvath, S. DNA methylation age of human tissues and cell types. Injected amyloid beta in the olfactory bulb transfers to other brain regions via neural connections in mice. Restriction of protein synthesis abolishes senescence features at cellular and organismal levels. Wang, S., Meyer, D. H. & Schumacher, B. H3K4me2 regulates the recovery of protein biosynthesis and homeostasis following DNA damage. Tiku, V. et al. Small nucleoli are a cellular hallmark of longevity. Kim, H. S. & Pickering, A. M. Protein translation paradox: implications in translational regulation of aging. Kirstein-Miles, J., Scior, A., Deuerling, E. & Morimoto, R. I. The nascent polypeptide-associated complex is a key regulator of proteostasis. Anisimova, A. S., Alexandrov, A. I., Makarova, N. E., Gladyshev, V. N. & Dmitriev, S. E. Protein synthesis and quality control in aging. Reduced proteasome activity in the aging brain results in ribosome stoichiometry loss and aggregation. Xie, X. et al. Quantification of insoluble protein aggregation in Caenorhabditis elegans during aging with a novel data-independent acquisition workflow. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Janssens, G. E. et al. Transcriptomics-based screening identifies pharmacological inhibition of Hsp90 as a means to defer aging. Tarkhov, A. E. et al. A universal transcriptomic signature of age reveals the temporal scaling of Caenorhabditis elegans aging trajectories. McIntyre, R. L. et al. Inhibition of the neuromuscular acetylcholine receptor with atracurium activates FOXO/DAF-16-induced longevity. Youthful and age-related matreotypes predict drugs promoting longevity. Banderwal, R. Neuroprotective mechanism of syringic acid against traumatic brain injury induced cognitive dysfunction. & Calvo, M. I. Anticholinesterase activity of selected medicinal plants from Navarra region of Spain and a detailed phytochemical investigation of Origanum vulgare L. ssp. Cao, Y. et al. Neuroprotective effects of syringic acid against OGD/R-induced injury in cultured hippocampal neuronal cells. A., Yan, G., Waldo, A. L. & Brown, A. M. Vanoxerine: cellular mechanism of a new antiarrhythmic. Piccini, J. P. et al. Randomized, double-blind, placebo-controlled study to evaluate the safety and efficacy of a single oral dose of vanoxerine for the conversion of subjects with recent onset atrial fibrillation or flutter to normal sinus rhythm: RESTORE SR. Heart Rhythm 13, 1777–1783 (2016). Rothman, R. B. et al. Development of a rationally designed, low abuse potential, biogenic amine releaser that suppresses cocaine self-administration. Forster, E. A. et al. A pharmacological profile of the selective silent 5-HT1A receptor antagonist, WAY-100635. Fletcher, A. et al. Electrophysiological, biochemical, neurohormonal and behavioural studies with WAY-100635, a potent, selective and silent 5-HT1A receptor antagonist. Calcagno, E. et al. Enhancement of cortical extracellular 5-HT by 5-HT1A and 5-HT2C receptor blockade restores the antidepressant-like effect of citalopram in non-responder mice. Chemel, B. R., Roth, B. L., Armbruster, B., Watts, V. J. & Nichols, D. E. WAY-100635 is a potent dopamine D4 receptor agonist. -C. et al. WAY-100635 has high selectivity for serotonin 5-HT1A versus dopamine D4 receptors. & van Waarde, A. Visualisation of serotonin-1A (5-HT1A) receptors in the central nervous system. Hillmer, A. T. et al. First-in-human evaluation of 18F-mefway, a PET radioligand specific to serotonin-1A receptors. Exquisite delineation of 5-HT1A receptors in human brain with PET and [carbonyl-11 C]WAY-100635. Kosari-Nasab, M. et al. Serotonin 5-HT1A receptors modulate depression-related symptoms following mild traumatic brain injury in male adult mice. Sukoff Rizzo, S. J. et al. 5-HT1A receptor antagonism reverses and prevents fluoxetine-induced sexual dysfunction in rats. A dihydropyridine (Bay k 8644) that enhances calcium currents in guinea pig and calf myocardial cells. A new type of positive inotropic agent. Tran, N. K. C. et al. Ginsenoside Re blocks Bay k-8644-induced neurotoxicity via attenuating mitochondrial dysfunction and PKCδ activation in the hippocampus of mice: Involvement of antioxidant potential. Shin, E. -J. et al. Dimemorfan prevents seizures induced by the L-type calcium channel activator BAY k-8644 in mice. Effects of dextromethorphan on the seizures induced by kainate and the calcium channel agonist BAY k-8644: comparison with the effects of dextrorphan. Pifferi, F. et al. Resveratrol does not mimic the positive effects of calorie restriction on lifespan in Microcebus murinus. Kumar, L. & Futschik, E. M. Mfuzz: a software package for soft clustering of microarray data. Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Muris Consortium, T. abula A single-cell transcriptomic atlas characterizes ageing tissues in the mouse. Fitz, N. F. et al. Extracellular vesicles in young serum contribute to the restoration of age-related brain transcriptomes and cognition in old mice. B. et al. Platelet factors attenuate inflammation and rescue cognition in ageing. Berchtold, N. C. et al. Hippocampal gene expression patterns linked to late-life physical activity oppose age and AD-related transcriptional decline. SenGupta, T. et al. Krill oil protects dopaminergic neurons from age-related degeneration through temporal transcriptome rewiring and suppression of several hallmarks of aging. & Smyth, G. K. edgeR: a bioconductor package for differential expression analysis of digital gene expression data. Korotkevich, G. et al. Fast gene set enrichment analysis. Structural insights and biological effects of glycogen synthase kinase 3-specific inhibitor AR-A014418. Chung, E. S. et al. Fluoxetine prevents LPS-induced degeneration of nigral dopaminergic neurons by inhibiting microglia-mediated oxidative stress. Ivraghi, M. S. et al. Neuroprotective effects of gemfibrozil in neurological disorders: focus on inflammation and molecular mechanisms. Liu, Z. et al. Inhibitors of LRRK2 kinase attenuate neurodegeneration and Parkinson-like phenotypes in Caenorhabditis elegans and Drosophila Parkinson's disease models. Rzemieniec, J., Wnuk, A., Lasoń, W., Bilecki, W. & Kajta, M. The neuroprotective action of 3,3′-diindolylmethane against ischemia involves an inhibition of apoptosis and autophagy that depends on HDAC and AhR/CYP1A1 but not ERα/CYP19A1 signaling. Mishra, J., Chaudhary, T. & Kumar, A. Rosiglitazone synergizes the neuroprotective effects of valproic acid against quinolinic acid-induced neurotoxicity in rats: targeting PPARγ and HDAC pathways. Protective effects of p38 MAPK inhibitor SB202190 against hippocampal apoptosis and spatial learning and memory deficits in a rat model of vascular dementia. Rao, M. S. & Abd-El-Basset, E. M. dBcAMP rescues the neurons from degeneration in kainic acid-injured hippocampus, enhances neurogenesis, learning, and memory. A. et al. Tolcapone improves cognition and cortical information processing in normal human subjects. Liao, R. et al. Amiodarone-induced retinal neuronal cell apoptosis attenuated by IGF-1 via counter regulation of the PI3k/Akt/FoxO3a pathway. Algharably, E. A. E., Di Consiglio, E., Testai, E., Kreutz, R. & Gundert-Remy, U. Prediction of the dose range for adverse neurological effects of amiodarone in patients from an in vitro toxicity test by in vitro–in vivo extrapolation. Zhang, S., Fujita, Y., Matsuzaki, R. & Yamashita, T. Class I histone deacetylase (HDAC) inhibitor CI-994 promotes functional recovery following spinal cord injury. Gräff, J. et al. Epigenetic priming of memory updating during reconsolidation to attenuate remote fear memories. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Vallat, R. Pingouin: statistics in Python. Statistical Power Analysis for the Behavioral Sciences (Elsevier, 1977); https://doi.org/10.1016/C2013-0-10517-X Waskom, M. L. seaborn: statistical data visualization. Hunter, J. D. Matplotlib: a 2D graphics environment. We thank the Regional Computing Center of the University of Cologne (RRZK) for providing computing time on the Deutsche Forschungsgemeinschaft (DFG)-funded High Performance Computing (HPC) system CHEOPS as well as support. Nematode strains were provided by the National Bioresource Project (supported by The Ministry of Education, Culture, Sports, Science and Technology, Japan), the Caenorhabditis Genetics Center (funded by the National Institutes of Health National Center for Research Resources) and the C. elegans Gene Knockout Project at the Oklahoma Medical Research Foundation (part of the International C. elegans Gene Knockout Consortium). We express our gratitude to P. Sengupta for sharing the PY6457 strain. was supported by the Peter and Traudl Engelhorn Foundation. is supported by DFG FOR 5504 project 496650118. Open access funding provided by Universität zu Köln. These authors contributed equally: Christian Gallrein, David H. Meyer. Institute for Genome Stability in Aging and Disease, Medical Faculty, University and University Hospital of Cologne, Cologne, Germany Christian Gallrein, David H. Meyer, Valeria Ramirez-Ramirez, Thanh Vuong-Brender & Björn Schumacher Christian Gallrein, David H. Meyer, Valeria Ramirez-Ramirez, Thanh Vuong-Brender & Björn Schumacher Leibniz Institute on Aging – Fritz-Lipmann-Institute (FLI), Jena, Germany Christian Gallrein, Yvonne Woitzat & Janine Kirstein Friedrich Schiller University, Institute for Biochemistry and Biophysics, Jena, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Christian Gallrein, David H. Meyer or Björn Schumacher. The authors declare no competing interests. Nature Aging thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) Distribution of transcriptomic age predictions. The 67 neurons of the Calico day 1 dataset were predicted with BitAge and sorted by their predicted age. b) BitAge predictions on the CeNGEN and the Calico day 1 dataset are highly correlated (Pearson correlation 0.66, p-value 1.2e-09). The regression model fit with a 95% confidence interval (shadowed area) is shown. c) Stochastic Clock predictions on the CeNGEN and the Calico day 1 dataset are highly correlated (Pearson correlation 0.82, p-value 3.2e-17). The regression model fit with a 95% confidence interval (shadowed area) is shown. d) Age prediction of BitAge and the Stochastic Clock on Calico Day 1 data. f) Age prediction of the Stochastic Clock on CeNGEN bulk data vs CeNGEN pseudobulk. g) Predictions with BitAge and the Stochastic Clock on the CeNGEN dataset are highly correlated. All 128 neurons are plotted, the 20% youngest and oldest according to BitAge are coloured in blue, the 20% youngest and oldest according to the StochasticClock are coloured in red, overlap between both violet. The remaining neurons are indicated in grey. The regression model fit with a 95% confidence interval (dashed, black lines) is shown. h) BitAge (blue) and StochasticClock (red) predictions on an independent dataset of neuronal RNA from adult day 1 and adult day 8 old nematodes. i) Correlation of median gene length and predicted age of individual neuron types. BitAge clock used on the CeNGEN dataset. Exemplary, young predicted neurons I2, OLL, and PHC (blue); and old predicted neurons ASI, ASJ, and ASK (orange) are indicated. j) Correlation of median gene length and predicted age of individual neuron types. Stochastic clock used on the CeNGEN dataset. k) Correlation of median gene length and predicted age of individual neuron types. BitAge clock used on the day 1 Calico dataset. l) Correlation of median gene length and predicted age of individual neuron types. Stochastic clock used on the day 1 Calico dataset. a) Correlation of neuronal GFP expression in ASK neurons (left) or ASJ neurons (right) and scored blebs (major hallmark of neurodegeneration in these strains). Classification to the fraction of ‘healthy' (blue), ‘mildly damaged' (orange), or ‘severely damaged' (red) neurons is indicated. Linear regression (black line) and confidence band (grey lines) are displayed, and no significant correlation was found. Displayed are the pooled data of four independent cohorts of 12 – 25 adult day 1 old nematodes each. Four to five cohorts of 12 – 25 JKM10 nematodes expressing GFP in URY neurons have been recorded and neurodegeneration was assessed and neurons categorized as ‘healthy' (blue), ‘mildly damaged' (orange), and ‘severely damaged' (red). c) Z-Stack Maximum projection after background subtraction (Fiji – rolling ball radius 25) of fluorescence microscopic recording of representative JKM10 nematode expressing GFP in URY neurons. d) Fraction plot of neuronal integrity of ASI, ASJ, and ASK neurons at young ages (L3 and L4 stage). Three to four cohorts of 8 – 27 JKM10 nematodes expressing GFP in either ASI, ASJ, or ASK neurons have been recorded, neurodegeneration was assessed, and neurons categorized as ‘healthy' (blue), ‘mildly damaged' (orange), and ‘severely damaged' (red). Ordinal regression (cumulative link model) has been used to test for significant differences between ages. e) Stochastic appearance of neuronal damage. The top panel shows a representative epi fluorescent image of ASK neurons. Blebs that were counted as damaged sites are indicated by violet arrows, which also depict how damage position is mirrored in the histograms below. The three bottom panels show histograms of the neuronal damage distribution at day 1 (green line) and day 4 (blue line) of the indicated neurons (ASK, ASJ, and OLL). Three cohorts of 10 – 25 individuals have been analysed per strain and age. Grey lines are 100 examples of randomly distributed damage along neurites. Damage site was measured as distance from the mouth and binned into 12 damage-site-bins. Pearson correlation has been calculated for every pair of “neuron vs. random distribution” and the average is indicated. f) Z-Stack Maximum projection after background subtraction (Fiji – rolling ball radius 25) of fluorescence microscopic recording of representative NY2067 nematode expressing GFP in ASE neurons. g) Neuron survival was assessed using Kaplan-Mayer. Presence of green fluorescent neuron bodies was scored daily. Shown are the aggregated plots of three independent cohorts of each 12 nematodes per strain. h) Histogram showing the relative frequency of the first day of neuron loss. Three cohorts of each 12 nematodes per strain. a) Amphid neurons are predicted to be significantly older than non-amphid neurons in the Calico day 1 dataset. b) Amphid neurons are predicted to be significantly older than non-amphid neurons in the CeNGEN dataset with a stochastic data-based clock. e) The number of neurotransmitters is not significantly different in amphid and non-amphid neurons. Two-sided t-test p-value: 0.94. f) The number of innexins is not significantly different in amphid and non-amphid neurons. Two-sided t-test p-value: 0.05. g) The number of expressed neuropeptides (y-axis) is significantly correlated (Pearson correlation 0.26, p-value 3e-03) with the predicted age by BitAge in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. h) The number of expressed receptor genes (y-axis) is significantly correlated (Pearson correlation 0.22, p-value 1.2e-02) with the predicted age by BitAge in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. i) The number of expressed neuropeptides (y-axis) is significantly correlated (Pearson correlation 0.36, p-value 2.9e-03) with the predicted age by BitAge in the Calico day 1 dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. j) The number of expressed receptor genes (y-axis) is significantly correlated (Pearson correlation 0.44, p-value 2.4e-04) with the predicted age by BitAge in the Calico day 1 dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. k) The number of expressed neuropeptides (y-axis) is significantly correlated (Pearson correlation 0.43, p-value 4.8e-07) with the predicted age by a stochastic data-based clock in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. l) The number of expressed receptor genes (y-axis) is significantly correlated (Pearson correlation 0.39, p-value 6.9e-06) with the predicted age by a stochastic data-based clock in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. m) Fraction-plot displaying the fraction of nematodes expressing neuronal volume markers in the ASI neuron categorized as ‘healthy', ‘mildly damaged', and ‘severely damaged'. Three to four cohorts were analysed, comprised of 10 – 30 individual nematodes, for every timepoint indicated. Kruskal-Wallis-test was employed to test for significant differences. n) The number of expressed innexin genes (y-axis) is significantly anti-correlated (Pearson correlation -0.19, p-value 3.7e-02) with the predicted age by BitAge in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. o) The number of expressed innexin genes (y-axis) is not-significantly anti-correlated (Pearson correlation -0.2, p-value 0.9e-01) with the predicted age by BitAge in the Calico day 1 dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. p) The number of expressed innexin genes (y-axis) is not-significantly anti-correlated (Pearson correlation -0.12, p-value 1.9e-01) with the predicted age by a stochastic data-based clock in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. q) The number of total synapses (y-axis) is not-significantly anti-correlated (Pearson correlation -0.04, p-value 6.6e-01) with the predicted age by BitAge in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. r) The number of total synapses (y-axis) is not-significantly anti-correlated (Pearson correlation -0.21, p-value 1.0e-01) with the predicted age by BitAge in the Calico day 1 dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. s) The number of total synapses (y-axis) is not-significantly anti-correlated (Pearson correlation -0.07, p-value 4.6e-01) with the predicted age by a stochastic data-based clock in the CeNGEN dataset. The regression model fit with a 95% confidence interval (shadowed area) is shown. t) Ciliated neurons are predicted to be significantly older than non-ciliated neurons in the Calico day 1 dataset. u) Ciliated neurons are predicted to be significantly older than non-ciliated neurons in the CeNGEN dataset with a stochastic data-based clock. v) Ciliated neurons are divided into 5 classes depending on where its cilia terminate. Neurons with exposed cilia are significantly older than non-ciliated neurons or neurons which cilia terminate in the cuticle or behind the cuticle in the Calico day 1 dataset (one-way ANOVA p-value: 2.68e-08, with a post-hoc Tukey test). w) Ciliated neurons are divided into 5 classes depending on where its cilia terminate. Neurons with exposed cilia are significantly older than non-ciliated neurons or neurons which cilia terminate in the cuticle or behind the cuticle in the CeNGEN dataset with a stochastic data-based clock (one-way ANOVA p-value: 6.6e-18, with a post-hoc Tukey test). Soft clustering for cluster numbers ranging from 2-12 (x-axis) were calculated with Dmin. b) SUnSET assay to test translation efficacy upon cycloheximide treatment. Left side: Westernblot detecting puromycin labelled proteins, ponceau staining as loading control, left side: densitometric quantification normalized to total protein according to ponceau signal. Four independent cohorts of each ≈ 35 nematodes tested. One-tailed t-test to test for significance. Lysed nematodes were day 1 of adulthood, 2 mM CHX treatment since L4 stage and while puromycin labelling. c) Fluorescence quantification to assess GFP synthesis in selected neurons as proxy for translation. Three independent cohorts, total 28 – 52 nematodes analysed per strain and condition. One-way ANOVA + Tukey post hoc test. d) Schematic depiction of experimental procedure to test the effect of either short-term or chronic CHX treatment on the neuron integrity. e) Fraction plots of neuronal integrity of ASK, ASJ, ASE, and OLL neurons treated with 2 mM CHX for either 24 h or 96 h starting from L4 stage on. Three cohorts of 4 – 20 nematodes expressing GFP in either ASK, ASJ, ASE, or OLL neurons have been recorded, neurodegeneration was assessed, and neurons categorized as ‘healthy' (blue), ‘mildly damaged' (orange), and ‘severely damaged' (red). Ordinal regression (cumulative link model) has been used to test for significant differences between CHX and solvent control treatment. To test for significant differences a cumulative link model was built, pairwise comparison was corrected according to Bonferroni. 5, but focusing on the aging pattern). b) Unbiased clustering analysis of conserved KEGG pathway enrichments for aging patterns (C. elegans NeuronAge, mouse, human) and geroprotective interventions. Both Pearson correlation-based and log-fold change (logFC)-based pathway enrichments are shown for each aging dataset to assess the impact of preprocessing methodology. Despite differences in computational approach, aging datasets cluster together, while geroprotective interventions form a separate cluster, reinforcing the robustness of the observed transcriptomic relationships. a) Z-stack fluorescence images of GFP expression in ASK, OLL, ASJ neurons after 24 h treatment with water control, 2.5 mM syringic acid (SA). Maximum projections after background subtraction (Fiji – rolling ball radius 80). b) Z-stack fluorescence images of GFP expression in ASK, OLL, ASJ neurons after 24 h treatment with 10 nM vanoxerine (VX). Maximum projections after background subtraction (Fiji – rolling ball radius 80). c) Fraction-plots displaying the fraction of nematodes expressing GFP in the OLL or ASK neurons categorized as ‘healthy', ‘mildly damaged', and ‘severely damaged' that were treated with 2.5 mM syringic acid (SA), 10 nM vanoxerine (VX), or 250 µM Bay K8644 for 24 h. Three to four cohorts were analysed, consisting of 10 – 25 individual nematodes each. Statistical analysis was performed using a cumulative link model with a logit link to account for the ordered categorical nature of neuronal damage scores (Healthy < Mild < Severe). Multiple test adjusted (Benjamini Hochberg) p-values are shown. d) Z-stack fluorescence images of GFP expression in I2 neurons after 24 h treatment with 25 nM WAY-100635 (WAY). Maximum projections after background subtraction (Fiji – rolling ball radius 80). e) Z-stack fluorescence images of GFP expression in ASK, OLL, and I2 neurons after 24 h treatment with 250 µM Bay K8644. Maximum projections after background subtraction (Fiji – rolling ball radius 80). f) Z-stack fluorescence images of GFP expression in ASK, OLL, ASJ, and I2 neurons after 24 h treatment with 1‰ ethanol, or 200 µM resveratrol. Maximum projections after background subtraction (Fiji – rolling ball radius 80). g) Fraction-plots displaying the fraction of nematodes expressing GFP in the OLL, I2, ASJ, or ASK neurons categorized as ‘healthy', ‘mildly damaged', and ‘severely damaged' that were treated with 200 µM resveratrol for 24 h. Four cohorts were analysed, consisting of 12 – 25 individual nematodes. Statistical analysis was performed using a cumulative link model with a logit link to account for the ordered categorical nature of neuronal damage scores (Healthy < Mild < Severe). Multiple test adjusted (Benjamini Hochberg) p-values are shown. h) Schema of experimental strategy to test whether Serratia marcescens Db11 bacteria metabolise syringic acid or WAY-100635. Db11 bacteria are grown and on half is inactivated by UV-C irradiation (500 mJ/cm^2), the other half is still alive. Compounds are added separately and incubated 48 h with dead or alive bacteria. Subsequently, solution is passed trough a 22 µm syringe filter to remove bacteria. The compound containing solution is afterwards added onto a lawn of UV-C inactivated OP50 bacteria. After drying, L4 stage nematodes are added and after 24 h neurodegeneration is assessed using fluorescence microscopy. i) Scan of UV-C inactivation test. Serratia marcescens Db11 bacteria were inactivated by UV-C irradiation (500 mJ/cm^2), streaked-out onto an NGM plate, and growth was documented after 24 h. Left side shows bacteria before inactivation, right side is bacteria after inactivation. j) Fraction-plot displaying the fraction of nematodes expressing GFP in the URY neurons categorized as ‘healthy', ‘mildly damaged', and ‘severely damaged' that were treated with 2.5 mM syringic acid (SA) or 25 nM WAY-100635 (WAY), after compounds were pretreated with living or dead Db11 bacteria for 48 h, for 24 h. Three cohorts were analysed, consisting of 10 – 25 individual nematodes. Statistical analysis was performed using a cumulative link model with a logit link to account for the ordered categorical nature of neuronal damage scores (Healthy < Mild < Severe). Pairwise comparisons between conditions were performed using estimated marginal means (emmeans) with Tukey's adjustment for multiple testing. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Gallrein, C., Meyer, D.H., Woitzat, Y. et al. Aging clocks delineate neuron types vulnerable or resilient to neurodegeneration and identify neuroprotective interventions. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260203020205.htm'>Scientists just mapped the hidden structure holding the Universe together</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 08:49:48
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The findings are based on new observations from NASA's James Webb Space Telescope (Webb) and are published in the journal Nature Astronomy. The international study was led jointly by Durham University, NASA's Jet Propulsion Laboratory (JPL), and the École Polytechnique Fédéral de Lausanne (EPFL), Switzerland. How Dark Matter Shaped the Universe We See Today Its gravity then pulled in normal matter, creating dense regions where stars and galaxies could begin to form. By allowing galaxies and stars to form earlier than they otherwise would have, dark matter also helped create the conditions needed for planets to develop. Without this early influence, the elements required for life may never have formed within our galaxy. Research co-lead author Dr. Gavin Leroy, in the Institute for Computational Cosmology, Department of Physics, Durham University, said: "By revealing dark matter with unprecedented precision, our map shows how an invisible component of the Universe has structured visible matter to the point of enabling the emergence of galaxies, stars, and ultimately life itself. The new map shows this effect with greater clarity than ever before. According to the researchers, Webb's observations show that this alignment is not accidental. Research co-author Professor Richard Massey, in the Institute for Computational Cosmology, Department of Physics, Durham University, said: "Wherever you find normal matter in the Universe today, you also find dark matter. "Billions of dark matter particles pass through your body every second. There's no harm, they don't notice us and just keep going. Without dark matter, the Milky Way would spin itself apart." To locate dark matter, the team measured how its mass bends space, which in turn bends the light traveling to Earth from distant galaxies -- as if that light had passed through a warped windowpane. Research co-lead author Dr. Diana Scognamiglio, of NASA's Jet Propulsion Laboratory, said: "This is the largest dark matter map we've made with Webb, and it's twice as sharp as any dark matter map made by other observatories. "Previously, we were looking at a blurry picture of dark matter. Durham University's Centre for Extragalactic Astronomy contributed to the development of MIRI, which was designed and managed through launch by JPL. The instrument is especially effective at detecting galaxies hidden behind thick clouds of cosmic dust. These future observations will help scientists better understand dark matter's basic properties and how it may have evolved over cosmic time. The region of sky analyzed in this study will serve as a reference point, allowing future dark matter maps to be compared and refined with greater precision. Note: Content may be edited for style and length. Scientists Identify Key Protein That Could Reverse Brain Aging A Hidden Cellular Defense May Protect the Brain From Alzheimer's This One Body Measurement Could Reveal Heart Disease Risk Years Before Symptoms Appear Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260201231224.htm'>A record breaking gravitational wave is helping test Einstein's theory of general relativity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 08:23:19
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For scientists who track faint ripples traveling through the cosmos, GW250114 stands out. It is the clearest gravitational wave signal ever recorded from a pair of merging black holes, giving researchers an unusually sharp tool for testing Albert Einstein's theory of gravity, called general relativity. Mitman is a co-author of the study examining this signal, titled "Black Hole Spectroscopy and Tests of General Relativity with GW250114," which was published in Physical Review Letters on Jan. 29. The gravitational wave known as GW250114 was created when two black holes collided, sending ripples through space-time. That signal reached the U.S.-based Laser Interferometer Gravitational-Wave Observatories (LIGO) on Jan. 14, 2025. At the same time, researchers believe not every black hole merger will follow Einstein's rules so closely, which could open new doors in fundamental physics. When two black holes merge, the newly formed object vibrates, much like a struck bell. These vibrations produce distinct tones defined by two measurements, Mitman explained: an oscillation frequency and a damping time. Detecting two or more tones makes it possible to perform multiple, independent checks of those same properties, as predicted by general relativity. "If those two measurements agree with one another, you are effectively verifying general relativity," Mitman said. He and his collaborators think it is possible that future gravitational wave signals will not fully align with general relativity, offering clues to long-standing mysteries. Physicists already suspect that general relativity cannot be the final word on gravity. "Along those lines, we expect there to be some deviation from Einstein's classical prediction, where you might see signatures of quantum gravity imprinting themselves on these gravitational wave signals. "The hope is that we'll see these deviations one day and that will help guide us along what the true theory of quantum gravity might be." Note: Content may be edited for style and length. A Hidden Cellular Defense May Protect the Brain From Alzheimer's This One Body Measurement Could Reveal Heart Disease Risk Years Before Symptoms Appear Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2026/02/260203020203.htm'>This brain discovery is forcing scientists to rethink how memory works</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-03 07:20:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Instead of using separate neural pathways to retrieve different types of information, the brain appears to activate overlapping areas, a finding that could change how memory is defined and studied. By combining task based experiments with fMRI data, the team found no measurable difference in brain activity between successful episodic and semantic memory retrieval. Episodic memory allows people to recall specific past experiences that happened at a particular place and time. This form of memory enables individuals to mentally revisit moments from their lives, often described as "mental time travel." Semantic memory, by contrast, involves recalling facts and general knowledge about the world. To directly compare how these two types of memory operate, the researchers designed tasks that were carefully aligned. Forty participants were asked to remember pairings between logos and brand names. Some pairings reflected real-world knowledge and formed the semantic task, while others were learned during an earlier study phase and served as the episodic task. During these memory tasks, participants underwent fMRI (Functional Magnetic Resonance Imaging) scanning. In the semantic task, they recalled brand details based on prior knowledge. fMRI is a non-invasive brain imaging technique that measures activity by tracking changes in blood flow. When specific brain regions become active during tasks such as thinking, speaking, or remembering, they receive increased amounts of oxygen-rich blood. "We were very surprised by the results of this study as a long-standing research tradition suggested there would be differences in brain activity with episodic and semantic retrieval. She also noted that the findings could offer new insights into memory related illnesses. For many years, episodic and semantic memory have been treated as separate systems, leading researchers to investigate them independently. This approach has resulted in relatively few studies that examine both memory types within the same experimental framework. "Based on what we already knew from previous research in this area we really expected to see stark differences in brain activity but any difference we did see was very subtle, I think these results should change the direction of travel for this area of research and hopefully open up new interest in looking at both sides of memory and how they work together." Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            