
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-15</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01496-z'>World's first personalized CRISPR therapy given to baby with genetic disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 17:25:49
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You can also search for this author in PubMed Google Scholar Baby KJ Muldoon, who was born with a genetic disease that affected his ability to metabolize proteins, is the first person to receive a bespoke CRISPR treatment. Little KJ Muldoon, now nearly ten months old, is doing well after receiving three doses of a gene-editing treatment to mend a mutation that had impaired his body's ability to process protein, his parents told reporters this week. But it is too soon to use the word “cure”, says Rebecca Ahrens-Nicklas, a paediatrician at Children's Hospital of Philadelphia in Pennsylvania, and one of Muldoon's physicians. CRISPR treatment inserted directly into the body for first time CRISPR treatment inserted directly into the body for first time Yet, the drug that it developed, described in the New England Journal of Medicine on 15 May, is specific to Muldoon's genetic sequence and will probably never be used for another person, says Ahrens-Nicklas. “This truly is the future for all of these gene and cell therapies,” says Arkasubhra Ghosh, who studies gene therapy at Narayana Nethralaya Eye Hospital in Bengaluru, India, and who was not involved in the study. Dozens of people have received CRISPR-based therapies for genetic conditions such as sickle-cell anaemia, but those treatments were designed to be used in many people with the same disorder, regardless of the underlying mutations that caused it. By contrast, researchers tailored Muldoon's therapy to correct a specific genetic sequence in his genome. Muldoon had inherited two mutations, one from each parent, that meant that he did not produce the normal form of a crucial enzyme called carbamoyl phosphate synthetase 1 (CPS-1). Hope, despair and CRISPR — the race to save one woman's life Hope, despair and CRISPR — the race to save one woman's life Meanwhile, each day brought added risk of brain damage or death: only about half of babies with severe CPS-1 deficiency survive long enough to receive a transplant. She and her colleagues had been working with a CRISPR-based technique called base editing, which can make targeted, single-letter changes to DNA sequences. Perhaps now it was time to try the approach in humans, she thought. CRISPR genome-editing grows up: advanced therapies head for the clinic Super-precise CRISPR tool enters US clinical trials for the first time Google AI better than human doctors at diagnosing rashes from pictures Join the International Institute for Intelligent Nanorobots and Nanosystems (IIINN) at Fudan University. Super-precise CRISPR tool enters US clinical trials for the first time An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09104-w'>Ku limits RNA-induced innate immunity to allow Alu-expansion in primates</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 15:33:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. Please note there may be errors present which affect the content, and all legal disclaimers apply. Ku70 and Ku80 form Ku, a ring-shaped complex that initiates the non-homologous end-joining (NHEJ) DNA repair pathway.1 Ku binds to double-stranded DNA (dsDNA) ends and recruits other NHEJ factors (e.g., LIG4, DNA-PKcs). While Ku can bind to double-stranded RNA (dsRNA)2 and trap mutated-DNA-PKcs on ribosomal RNA (rRNA),3,4 the physiological role on Ku-RNA interaction in otherwise wildtype cells remains unclear. Intriguingly, Ku is dispensable for murine development5,6 but essential in human cells.7 Despite similar genome sizes, human cells express ~100-fold more Ku than mouse cells, implying functions beyond NHEJ - possibly through a dose-sensitive interaction with dsRNA, which binds Ku 10~100 times weaker than dsDNA.2,8 Investigating Ku's essentiality in human cells, we found that Ku-depletion - unlike LIG4 - induces profound interferon (IFN) and NF-kB signaling via dsRNA-sensor MDA5/RIG-I and MAVS. Ku expression rose sharply in higher primates, correlating tightly with Alu-expansion (r = 0.94/0.95). Prices may be subject to local taxes which are calculated during checkout Institute for Cancer Genetics, Vagelos College of Physicians and Surgeons, Columbia University, New York, NY, USA Yimeng Zhu, Angelina Li, Brian J. Lee, Kyle J. Wang, Antony B. Holmes & Shan Zha Herbert Irving Comprehensive Cancer Center, Vagelos College of Physicians and Surgeons, Columbia University, New York, NY, USA Medical Scientist Training Program, Columbia University Vagelos College of Physicians and Surgeons, New York, NY, USA Zhu, Y., Li, A., Maji, S. et al. Ku limits RNA-induced innate immunity to allow Alu-expansion in primates. Anyone you share the following link with will be able to read this content: Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59468-w'>Rare earth metals production using alternative feedstock that eliminates HF</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 15:03:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. This work reports the successful production of rare earth (RE) metal using Na-RE-F. Presently, RE metals are primarily produced using RE-fluoride due to its higher air and moisture stability compared to RE-chloride. However, its preparation requires the use of corrosive and hazardous chemicals, such as hydrofluoric acid (HF) or ammonium bifluoride (NH4HF2). The present study demonstrates that Na-RE-F is an alternative salt to the typically used RE-fluoride. The Na-RE-F for this work is produced via a scalable hydrometallurgical approach using three different RE salts as feedstock, including acetate, nitrate, and chloride. HF is neither used nor generated during the salt preparation process. Furthermore, the Na-RE-F powder dries in air (without dry HF), and only water evolves during the drying process. Analyses of the Na-RE-F show that NaF liberates as a flux during the heating process, which lowers the salt reduction temperature to <900 °C, thus minimizing or eliminating the need for additional flux. Calciothermic reduction of the Na-RE-F salt is successfully employed to obtain RE metal. This work represents a safer, greener, and more widely deployable approach for producing the RE metals needed for permanent magnets which support the transition to a cleaner society through the decarbonization of the transportation industry. Rare earth elements (REEs) are increasingly susceptible to supply risks due to their limited geographical availability and the fact that they are subject to rising geopolitical tensions. Some REEs, such as neodymium (Nd), praseodymium (Pr), dysprosium (Dy), and terbium (Tb), are used for permanent magnets that support many clean energy and renewable energy applications, e.g., electric vehicles, defense systems, electronics, etc. Hence, REEs have been termed “the vitamins of modern industry”1,2,3,4,5. Reduction of rare earth (RE) salts to metals is typically accomplished through molten salt electrolytic or metallothermic methods. The molten salt electrolytic method is the widely used approach because of its relatively simple process-flow and better suitability for continuous production6. The electrolyte for the molten-salt bath can be chloride- or fluoride-based, and the feedstock can be RE-chloride or RE-oxides, respectively. The molten salts for both the chloride and fluoride electrolytic routes require heating a mixture of RE-halides and alkali or alkaline earth salts to 800–1000 °C. Unlike the metallothermic methods, the electrolytic methods use consumable graphite electrodes which results in CO2 emissions7,8,9,10,11. Fluoride-based baths have higher current efficiency (<87%), compared to chloride-based baths (<50%)12 because the hygroscopic nature of RE-chlorides and the formation of oxychlorides limit yield and contaminate the obtained RE-metal product. In the metallothermic methods, the RE-feedstock materials are exothermically reduced to metals using more reactive metals like calcium (calciothermic reduction)13. This method can use RE-oxides, chlorides, or fluorides as feedstock. RE-fluoride is less hygroscopic than RE-chloride and both have lower melting temperatures than RE-oxide. The energy cost for direct metallothermic reduction of RE-oxides, as well as the potential for oxidation of the obtained RE-metals due to the lack of a protective flux, makes it less practical. Although calciothermic reduction of RE-chloride has been attempted, it was limited by low yield, and the difficulties with handling the hygroscopic and high vapor pressure RE-chlorides14. From the foregoing discussion, the practical (hence preferred) application of both the electrolytic and metallothermic approaches requires RE-fluoride, which can be prepared via wet or dry processing techniques. The former involves the wet chemical reaction of RE chloride or nitrate salts and hydrofluoric acid (HF), followed by drying at 600–700 °C in a dry hydrogen fluoride environment15. In the dry processing method, RE2O3 is directly reacted with an excess of dry hydrogen fluoride (200%) at ≥700 °C to obtain anhydrous RE-fluoride16,17. The health hazards18,19,20,21,22,23,24 associated with using or generating HF create a considerable technical gap in the anhydrous RE-fluorides preparation for RE metal supply chain. Furthermore, the high melting point of RE-fluorides (>1300 °C) necessitates fluxing with LiF, NaF, KF, etc. Thus, the need for an alternate feedstock to generate RE metals is timely. In this work, we report the use of NaNdF4 (NNF) to overcome most of the challenges faced in the RE metal production using the calciothermic reduction process. NNF has the following advantages (a) in-situ formation of NaF as a fluxing agent during reduction, (b) lower liquidus temperatures (~1100 °C)25, (c) facile room temperature synthesis by co-precipitation26, and (d) no requirement for toxic HF or NH4HF2. Furthermore, reports on using NNF in human cancer pathological studies indicate that it would be a non-toxic feedstock27,28. We have demonstrated the use of NNF for Nd metal production using the calciothermic process. Unlike conventional RE-fluoride reduction, NNF not only ensures fluoride ion availability but also represents a strategic shift toward safer, more sustainable, and regulation-compliant rare earth metal production. The reaction between Nd(CH3COO)3 · 2H2O (neodymium acetate hydrate) and aqueous sodium fluoride solution yields single phase precipitate of α-NaNdF4 (α-NNF) [space group: \({Fm}\bar{3}m\) (no. 1), in agreement with a previous report [Powder Diffraction File (PDF) No. The diffraction pattern for the sample synthesized using the acetate precursor, i.e., NNF-A, comprised of a single α-NaNdF4 phase (denoted with a diamond symbol). The NNF-C and NNF-N samples (synthesized using the chloride and nitrate precursors, respectively) had two phases corresponding to α-NaNdF4 and NdF3 (denoted with a solid square symbol) phases. The reaction of NdCl3 · 6H2O (neodymium chloride hydrate) and Nd(NO3)3 · 6H2O (neodymium nitrate hydrate) with sodium fluoride both yielded two phases (Fig. 1): α-NNF [space group: \({Fm}\bar{3}m\) (no. 225)] and NdF3 [space group: \(P{6}_{3}/{mcm}\) (no. 193)] [Powder Diffraction File(PDF) No. The contents of the NdF3 phases were 10% for neodymium chloride hydrate and 5% for neodymium nitrate hydrate (Table 1). For example, both NdCl3 and NdOCl would be present for the chloride salt, as further explained in the supplementary information. A recent study by Gibson et al. reported a negative Gibbs' energy for NNF, indicating its thermodynamic stability at 298.15 K29. Additional phase analysis information is provided in Supplementary Figs. The scanning electron microscopy (SEM) analysis of the fluoride samples in the present study was used to confirm that they have particle sizes of <100 nm (Supplementary Figs. Hereinafter, the NaNdF4 produced from acetate, chloride and nitrate will be referred to as NNF-A, NNF-C, and NNF-N, respectively. The differential scanning calorimetry (DSC) data (Fig. 2a) depicts one exothermic and two endothermic peaks for all three samples. The NNF-A and NNF-N exhibit the first exothermic peaks at T = 394 °C and T = 355 °C, respectively. The first exothermic peak is not pronounced for NNF-C. This first peak for all the samples is a concomitant occurrence of two thermal events, namely, the volatilization of the water of crystallization (endothermic) and the conversion of α-NaNdF4 to β-NaNdF4 phase (exothermic)26, with the latter being more prominent. a The differential scanning calorimetry (DSC) plot of the three fluoride samples. b thermogravimetric analysis (TGA) patterns of the fluoride samples obtained from the different precursor salts. The higher mass loss for NNF-A is due to the higher α-NaNdF4 content. The endothermic peaks at 642 °C, 750 °C and 700 °C (broad peak) represent the melting of the eutectic mixture comprising NaF and β-NaNdF425 for NNF-N, NNF-C, and NNF-A, respectively. This is possibly due to the higher phase fraction of NaNdF4 in the NNF-A sample which, in turn, contributed to higher water of crystallization content. The TGA profile of NNF-A depicts two mass loss regions: (a) 150 °C to 300 °C for volatilization of adsorbed water and (b) >350 °C for volatilization of water of crystallization. Mass spectrometric analysis was performed in the quasi-multiple ion detection (QMID) mode during the TG-DSC experiments and are depicted in Fig. a There are two peaks in the NNF-A sample corresponding to water at 220 °C and 400 °C, b The NNF-C has a sharp peak at ~100 °C and two less prominent peaks at 300 °C and 400 °C, and c The NNF-N has a sharp peak at ~100 °C and another prominent peak at 470 °C. All three fluorides did not exhibit the evolution of any hydrogen fluoride. NNF-A sample synthesized using the acetate precursor, NNF-C sample synthesized using the chloride precursor, NNF-N sample synthesized using the nitrate precursor. The peaks from the QMID plots help determine the chemical species with a certain mass number that evolves during thermal treatment. There are two notable peaks of mass number 18 (due to water) in all three fluorides. These results corroborate the findings of the DSC studies that show a thermal event at ~400–500 °C (Fig. 2a), along with a mass loss observed in the TGA plot (Fig. The absence of HF evolution was confirmed by the flat line corresponding to mass number 20 in all the three fluorides. The absence of HF was further validated by heating the sample to 600 °C for 3 h (to ensure complete drying), while testing with a HF gas detector. To investigate the phases in the NNF powder samples after the volatilization of water of crystallization, the as-synthesized samples were heat treated at 500 °C for 3 h and the powder X-ray diffraction (XRD) patterns were recorded (Fig. The X-ray diffraction (XRD) pattern obtained by heating as-synthesized powders of NNF-A (black plot), NNF-C (red plot), and NNF-N (blue plot) at 500 °C for 3 h shows three phases for each of the samples. The open circle, solid square, and open square symbols correspond to β-NaNdF4, NdF3, and NaF phases, respectively. NNF-A, sample synthesized using the acetate precursor; NNF-C, sample synthesized using the chloride precursor; NNF-N, sample synthesized using the nitrate precursor. Three phases corresponding to β-NaNdF4 (β-NNF) [space group: \(P{6}_{3}/m\) (no. 176)], NdF3 [space group: \(P{6}_{3}/{mcm}\) (no. 193)], and NaF [space group: \({Fm}\bar{3}m\) (no. Below this temperature, the composition corresponding to the α-NaNdF4 phase falls in a two-phase region of β-NaNdF4 and NdF326. Furthermore, below 732 °C, the β-NNF phase exists as a line compound, and any deviation in stoichiometry leads to a two-phase region of β-NaNdF4 and NaF phases (Supplementary Fig. This explains the formation of three phases upon sample heating and is consistent with previous reports26. All the samples formed significant amounts of NdF3 phase, in addition to the β-NaNdF4 phase. NNF-A had the least amount of NaF while the amounts in NNA-C and NNA-N are comparable. The collected metal after calciothermic reduction is shown in Fig. 5b shows peaks matching with Nd metal [space group: \(P{6}_{3}/{mmc}\) (no. It is common for metals produced by calciothermic or electrolytic methods to undergo further purification which helps to reduce the impurities30,31, such as those from the slag or the unreacted salts, etc. In the present work, the metal production yield was 70–80%, which is expected to increase at higher-scale operations. a Nd metal seated on tantalum cap after calciothermic reduction. The purple surface color is due to the unreacted material and slag on the metal, which was removed by mechanical polishing. The portions of the image corresponding to Nd metal and Ta cap are identified in the picture; b X-ray diffraction (XRD) pattern from the samples reduced to metal. All the peaks correspond to Nd metal [space group: \(P{6}_{3}/{mmc}\) (no. TG-DSC measurement was performed on a sample of the metal, and the results are plotted in Fig. The DSC plot depicts two endotherms at 852 °C and 1016 °C corresponding to a transformation of the unreacted fluoride sample from β-NaNdF4 to α-NaNdF4 and melting of metallic Nd, respectively. The TGA plot depicts little to no mass change, as expected for Nd metal. The inset is a spherical ball of Nd metal obtained after the TG-DSC experiment. Neodymium acetate (99%) from Sigma Aldrich was used for this work. Neodymium chloride and neodymium nitrate were synthesized by dissolving industrial grade of neodymium hydroxide in hydrochloric and nitric acids, respectively, and drying at 80 °C. Neodymium chloride hexahydrate (10 mmol, 3.6 g) and neodymium nitrate hexahydrate (10 mmol, 4.4 g) were dissolved in 50 mL of water each. To this Nd salt solution, 2.1 g NaF in 50 ml water (50 mmol) were added, and the contents were stirred at 25 °C for 6 h. We found that a slightly lower ratio yielded no NaNdF4, and a slightly higher ratio resulted in almost 100% NaNdF4 but required even more excess NaF. The resulting NaNdF4 precipitate was filtered, washed, and dried overnight at 80 °C in the air. To obtain dehydrated NaNdF4 product, the sample was heat treated at 500 °C for 3 h in air. Structural analysis of the NaNdF4 product was performed by powder XRD using a Bruker D-8 X-ray diffractometer with Cu-Kα radiation. Software (Version 3.15), and with Corundum as the reference. Simultaneous thermogravimetric analysis and differential scanning calorimetry (TGA-DSC) were performed using a STA449F1 system (Netzsch, Selb, Germany) with Al2O3 crucibles from 30 °C to 1050 °C. The temperature was scanned at 10 °C/min in ultra-high purity nitrogen gas flow (60 mL/min). Evolved gas analysis was performed using an Aeolos QMS 403D quadrupole mass spectrometer connected to the TGA furnace via a heated transfer line. Resublimed calcium prepared at Materials Preparation Center, Ames National Laboratory was used at a Ca: NaNdF4 ratio of ~2.6 to reduce the NaNdF4 to metals. The reduction process was performed in a 0.75 × 5 × 1.2 (width × length × thickness) cubic inch tantalum crucible. A spun cap was welded at the bottom, and another loose spun cap was used as a lid. The tantalum crucible was placed in a quartz tube with a secondary quartz tube to retain the tantalum cap. The assembly was then suspended into an induction coil (Supplementary Fig. 1a), which was then heated from room temperature to 880 °C in 15 min. The reduction set-up was maintained under vacuum till 700 °C, and then backfilled with Ar to −100 mmHg (Supplementary Fig. The run continued for 23 min, following which an eruption of vapor and an induction heated plasma was observed. The induction power was terminated at this stage. The nitrogen and oxygen were analyzed with LECO-ON836 using 0.1 g of sample in a graphite crucible under helium atmosphere. The carbon and sulfur were analyzed using LECO-CS844 using 0.5 g of sample in an alumina crucible under an O2 atmosphere. The morphology of the fluoride samples was recorded using a FEI Teneo SEM. The present study demonstrates the metallothermic reduction of fluoride salts to Nd metal, using NaNdF4 synthesized via a facile route that excludes the use of HF. The fluoride salts were synthesized through room-temperature reactions involving only an aqueous solvent. Three different rare earth salts have been utilized as starting materials to demonstrate the versatility of the process. Results show that only water evolved from the salts during the heating process. Data sets generated during the current study are available from the corresponding author upon request. Determination of an acceptable request and subsequent data provision are subject to Ames National Laboratory approval. Balaram, V. Rare earth elements: a review of applications, occurrence, exploration, analysis, recycling, and environmental impact. Elshkaki, A. Sustainability of emerging energy and transportation technologies is impacted by the coexistence of minerals in nature. Filho, W. L. et al. Understanding rare earth elements as critical raw materials. Pawar, G. & Ewing, R. C. Recent advances in the global rare-earth supply chain. Ormerod, J., Karati, A., Baghel, A. P. S., Prodius, D. & Nlebedim, I. C. Sourcing, refining and recycling of rare-earth magnets. Liao, C. et al. Research status of electrolytic preparation of rare earth metals and alloys in fluoride molten salt system: a mini review of China. Vogel, H., Flerus, B., Stoffner, F. & Friedrich, B. Reducing greenhouse gas emission from the neodymium oxide electrolysis. Part I: analysis of the anodic gas formation. Reducing greenhouse gas emission from the neodymium oxide electrolysis. Part II: basics of a process control avoiding PFC emission. Cvetković, V. S. et al. Investigation on the electrochemical behaviour and deposition mechanism of neodymium in NdF3–LiF–Nd2O3 melt on Mo electrode. Vogel, H., Friedrich, B., Scholz, H. & Franz, H. Development and research trends of the neodymium electrolysis—A literature review. In Production of non-ferrous metals – indispensable foundation for the industry and our prosperity; Proceedings of the 8th European Metallurgical Conference, Düsseldorf - Vol. Vahidi, E. & Zhao, F. Assessing the environmental footprint of the production of rare earth metals and alloys via molten salt electrolysis. Zhu, H. Rare earth metal production by molten salt electrolysis. In Encyclopedia of Applied Electrochemistry (eds Kreysa, G., Ota, K.-i. In Proceedings of the Eighth International Workshop on Rare-earth Magnets and Their Applications and the Fourth International Symposium on Magnetic Anisotropy and Coercivity in Rare Earth-Transition Metal Alloys 407–422 (University of Dayton, School of Engineering, 1985). Fabrication of rare-earth metals by metallothermic reduction: thermodynamic modeling and practical realization. & Spedding, F. H. Preparation of yttrium metal by reduction of yttrium trifluoride with calcium (Ames Laboratory, 1956). Spedding, F. H. & Daane, A. H. The Rare Earths (Wiley, 1961). Preparation of enriched rare-earth metals by hydrofluorination and reduction. Braun, J., Stöß, H. & Zober, A. Intoxication following the inhalation of hydrogen fluoride. Bajraktarova-Valjakova, E. et al. Hydrofluoric acid: burns and systemic toxicity, protective measures, immediate and hospital medical treatment. Sanz-Gallén, P., Nogué, S., Munné, P. & Faraldo, A. Hypocalcaemia and hypomagnesaemia due to hydrofluoric acid. Stuke, L. E., Arnoldo, B. D., Hunt, J. L. & Purdue, G. F. Hydrofluoric acid burns: a 15-year experience. Peters, D. & Miethchen, R. Symptoms and treatment of hydrogen fluoride injuries. Zierold, C. D. & Chauviere, M. Hydrogen fluoride inhalation injury because of a fire suppression system. Lund, K., Ekstrand, J., Boe, J., Søstrand, P. & Kongerud, J. Exposure to hydrogen fluoride: an experimental study in humans of concentrations of fluoride in plasma, symptoms, and lung function. Thoma, R. E., Insley, H. & Hebert, G. M. The sodium fluoride-lanthanide trifluoride systems. Yang, S., Jayanthi, K., Anderko, A., Riman, R. E. & Navrotsky, A. Thermochemical investigation of the stability and conversion of nanocrystalline and high-temperature phases in sodium neodymium fluorides. Single ultrasmall Mn2+-doped NaNdF4 nanocrystals as multimodal nanoprobes for magnetic resonance and second near-infrared fluorescence imaging. Lu, S. et al. Multifunctional nano-bioprobes based on rattle-structured upconverting luminescent nanoparticles. Gibson, A., Yang, S., Riman, R. E., Navrotsky, A. & Woodfield, B. F. Heat capacity and thermodynamic functions of sodium rare earth ternary fluorides. Huffine, C. L. & Williams, J. M. Refining and purification of rare-earth metals. Technical Report AECU-4426 (General Electric Co., Aircraft Nuclear Propulsion Dept., Cincinnati, OH, 1959). Spedding, F. H. & Daane, A. H. Production of rare earth metals in quantity allows testing of physical properties. This work was supported by the Critical Materials Innovation Hub funded by the U.S. Department of Energy, Office of Energy Efficiency and Renewable Energy, Advanced Materials and Manufacturing Technologies Office. The work was performed at Ames National Laboratory, operated for the U.S. Department of Energy by Iowa State University of Science and Technology under Contract No. These authors contributed equally: Denis Prodius, Ikenna C. Nlebedim. Anirudha Karati, Harshida Parmar, Denis Prodius & Ikenna C. Nlebedim Division of Materials Science and Engineering, The Materials Preparation Center, Ames National Laboratory, Ames, IA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar : Data analysis, Writing: review and editing, Supervision, Acquiring of funding. All authors are co-inventors on a related pending patent application entitled “preparation of rare earth metals with double salts”. Applicant: Iowa State University Research Foundation, Inc. (Ames, IA). Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Karati, A., Parmar, H., Riedemann, T. et al. Rare earth metals production using alternative feedstock that eliminates HF. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64768114/metal-detector-roman-coins/'>A Metal Detectorist Stumbled Upon a Hoard of Nearly 1,500 Roman Coins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Metal detectors work by transmitting an electromagnetic field from the search coin (that round part that detectorists sweep back and forth) into the ground. The detector can be set off by discarded bottle caps, modern coins, or even chunks of lead in the ground, but sometimes the treasure proves to be more exciting. The machines prove so effective that researchers often include metal detector surveys in their archaeological studies. Even amateur detectorists have found incredible artifacts with their machines. Just recently, one found a golden raven head and another found a hoard of Iron Age artifacts. And that list of big discoveries is still growing, as one metal detectorist scanning a field unearthed nearly 1,500 Romanian coins. The man who discovered the hoard, Bebe Mangeac, went out for a typical day with his trusty metal detector—a model he described to What's The Jam as “average and neither cheap nor top end.” He explained that he goes out for the exercise, never knowing what he's going to find. The area he picked for this particular hunt was in Letca Veche, a hilly area with soft soil and little vegetation southwest of Bucharest, Romania. Mangeac began scouting the area, and his machine soon picked up a signal not far beneath the surface. They appeared oxidized yet still well defined. I didn't want to leave any behind. The other, more instinctive part tells you to hurry up. Well, more like ceramic shards of what once was a storage vessel, likely an amphora or a dolium. “I didn't count them all there and then, but it felt like there were a lot,” he remarked. Some depict Emperor Trajan and others elephant iconography. The coins appear to be from several different periods, likely a result of accumulating the collection over time. Perhaps Mangeac is the perfect example of why metal detecting should be the next hobby you pick up—and maybe you and your “average” machine will make the next monumental archeological discovery in your area. Her work appears in several publications including Biography.com and Popular Mechanics. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. A Collapsing Church Wall Revealed Hidden Skeletons Explorers Found Maya Sacrifices in a ‘Blood Cave' Detectorists Find a Dazzling Golden Raven Head Ancient Temple Found Buried in Peruvian Sand Chinese Tomb Mural from 8th Century Has Blonde Man Archaeologists Find Third Ancient Fort on One Site</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/icelands-orca-pods-mysteriously-include-baby-pilot-whales/'>Baby Pilot Whales Are Swimming with Iceland's Orcas. No One Knows Why</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Friend or Food: Why Are Iceland's Orcas Taking in Pilot Whales? Newborn pilot whales have been spotted mysteriously swimming among pods of orcas. “Interactions between Killer Whales (Orcinus orca) and Neonate Long-Finned Pilot Whales (Globicephala melas) off South Iceland,” by Chérine D. Baumgartner et al., in Ecology and Evolution, Vol. One day in June 2022 Chérine Baumgartner, a researcher at the Icelandic Orca Project, was watching from a dinghy as a pod of killer whales fed on herring—when she noticed something very odd about what seemed to be a young member of the pod. “At first, we were like, ‘Oh my god, this killer whale calf has a problem,'” she says. It was far tinier than normal and lacked an infant orca's characteristic black-and-pale-orange coloration. Baumgartner, now a Ph.D. student at the Swiss Federal Institute of Technology Zurich, suddenly realized she was seeing an entirely different species: a baby pilot whale. She and her team observed the pod for nearly three hours before weather conditions forced them back to land. Scientists noticed orcas interacting with baby pilot whales off Iceland every year from 2021 to 2023. Each instance was short-lived and featured different individual pilot whales (dark-gray members of the dolphin family with a bulbous forehead) and different pods of orcas. A neonate long-finned pilot whale swims among killer whales on June 23, 2022. “Interactions between Killer Whales (Orcinus orca) and Neonate Long-Finned Pilot Whales (Globicephala melas) off South Iceland,” by Chérine D. Baumgartner et al., in Ecology and Evolution, Vol. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. In all the sightings, a weeks-old pilot whale swam by a female killer whale in what scientists call an echelon position, with the young whale located beside and slightly behind the adult orca. In 2023 a calf was seen swimming ahead of the group, possibly as if to run away—and at one point it was lifted, belly-up, out of the water on the back of an orca. With the first possible explanation, the killer whales could have been keeping the young pilot whales around like a living lunch box; some orcas in Iceland are known to eat harbor seals and porpoises. But Baumgartner notes that these Icelandic killer whales are predominantly fish eaters and that they didn't display overtly aggressive behavior toward the pilot whale calves. So predation is less likely, though not impossible, she and her colleagues say. Iceland's orcas often herd herring, and they could have been incorporating the pilot whale calves in their hunting games. Whales and dolphins in the wild often care for the young of other members of their pod, and although it's rare, dolphins have adopted calves from different species. These three possibilities also aren't mutually exclusive, she says. A neonate long-finned pilot whale is lifted out of the water on the back of an unidentified killer whale on June 20, 2023. “Interactions between Killer Whales (Orcinus orca) and Neonate Long-Finned Pilot Whales (Globicephala melas) off South Iceland,” by Chérine D. Baumgartner et al., in Ecology and Evolution, Vol. Teman previously studied southern resident killer whales interacting with porpoises in the Salish Sea off British Columbia and Washington State. In that research, she observed interactions that may have been motivated by nurturing, hunting practice or “play”—and often ended up killing the porpoises. “It was fascinating to see such similar behaviors” in the Icelandic orcas, she says, adding that the animals' interactions with the pilot whales seemed to be largely driven by nurturing or play behavior, just as had been seen in the southern resident killer whales' interactions with porpoises. Samarra also speculates that the unusual interactions off Iceland could be a result of climate change because pilot whales increasingly follow schools of mackerel moving into warmer waters that overlap with the killer whales' range. She hopes that, next summer, her group will finally observe how the young pilot whales get entangled with the orcas and what happens to them next. She was previously an editor at Hakai Magazine and is based in Victoria, British Columbia. Visit Wang's website at www.marina-wang.com or follow her on Bluesky @marinacoladas.bsky.social Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41564-025-02015-6'>A shared alarmone–GTP switch controls persister formation in bacteria</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 09:55:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Persisters are phenotypically switched bacteria that survive antibiotic exposure despite being genetically susceptible. Three pathways to persistence—triggered, spontaneous and antibiotic-induced—have been described, but the underlying molecular mechanisms are poorly understood. Here, we used antibiotic time–kill assays as well as single-cell approaches to show that all of the pathways depend on a common switch involving the alarmone guanosine tetra/penta-phosphate ((p)ppGpp) in Bacillus subtilis, each stemming from different alarmone synthetase(s). We developed a fluorescent GTP reporter to visualize rare events of persister formation in wild-type bacteria, revealing a rapid switch from growth to dormancy in single cells as their GTP levels drop beneath a threshold. While a decrease in GTP in the bulk population slows growth and promotes antibiotic tolerance, (p)ppGpp drives persistence by driving rapid, switch-like decreases in GTP levels beneath the persister threshold in single cells. Persistence through alarmone–GTP antagonism is probably a widespread mechanism to survive antibiotics in B. subtilis and potentially beyond. Antibiotic persisters are individual bacterial cells that survive prolonged bactericidal antibiotic treatment despite being genetically sensitive to antibiotics1,2,3. Joseph Bigger originally proposed that persisters can be generated before or after treatment with antibiotics2. Later, persisters were categorized by their pathways of generation—starvation triggered, spontaneously generated and antibiotic induced1—yet their molecular mechanisms remain unclear. Although persistence can be triggered by nutrient deprivations4 through the intracellular accumulation of the alarmone (p)ppGpp5,6,7,8,9, it is also argued that slow growth or a reduction in ATP levels, rather than (p)ppGpp, is responsible for triggering persistence10,11. In addition to starvation, persisters can form spontaneously3. However, spontaneous persisters exist in natural populations at a low frequency, making their identification challenging and making it difficult to establish the mechanisms of their formation. While toxin–antitoxin (TA) modules are strongly implicated in persistence3,5,8,12,13, their roles in spontaneous antibiotic persistence of wild-type bacteria remain unclear14. Finally, persisters were observed both before and after treatment with antibiotics15, and bacteriostatic antibiotics are known to induce persisters16. However, it is difficult to differentiate antibiotic-induced persistence from antibiotic tolerance1, spontaneous persistence, transcriptional response to specific drugs or by-products of cellular damage from drug treatment1. The mechanisms of antibiotic-induced persistence remain poorly understood. Here we directly visualize the formation of starvation-triggered, spontaneous and antibiotic-induced persisters in the Gram-positive bacterium B. subtilis belonging to the phylum Bacillota and reveal that multiple cellular mechanisms of persistence can converge on a common pathway of alarmone–GTP antagonism. To quantify persistence in B. subtilis, we began by treating cells with the cell-wall-targeting antibiotic vancomycin at concentrations of 20 times higher than its minimal inhibitory concentration (MIC) (Fig. The results revealed biphasic killing kinetics, with a rapid elimination of the majority of cells followed by the prolonged survival of a small subpopulation. This surviving fraction (approximately 0.1%) remained consistent across varying drug concentrations and, after regrowth, became resensitized to antibiotics (Fig. 1a), confirming that these are phenotypically switched persisters rather than genetically altered resisters1,2,17. We also tested the effect of prophages on persistence, given that persistence against the DNA-damaging antibiotic ciprofloxacin has been shown to be complicated by prophage activation in Escherichia coli14. We confirmed that, similar to E. coli, persistence against ciprofloxacin is weakened by prophages (Supplementary Fig. 2), but persistence against vancomycin remains unaffected regardless of the presence or absence of resident prophages SPβ and PBSX within the genome. Furthermore, the levels of persisters in prophage-cured cells were consistent across different classes of antibiotics, including vancomycin, ciprofloxacin and kanamycin, that target translation (Fig. This suggests that the persisters originate from a common subpopulation that is refractory to multiple antibiotics1. a, Survival curves of B. subtilis wild-type (WT) and (p)ppGpp0 after treatment with lethal concentrations of vancomycin (VAN). Rekill indicates that cells that survived 20× MIC vancomycin treatment were regrown and treated with vancomycin again. c, Starvation-triggered persistence of WT B. subtilis depends on (p)ppGpp. Biphasic kill curve assays of vancomycin treatment were performed after treatment with several starvation-inducing or (p)ppGpp-inducing conditions. Stationary, stationary phase; RHX, pretreated with the amino acid starvation inducer arginine hydroxamate; CCCP, pretreated with ATP synthesis inhibitor CCCP; p-sasA or p-sasASyn, (p)ppGpp0 strains overexpressing the (p)ppGpp synthetase SasA or its synthetase-dead variant SasASyn (sasAD87G). Plots shown on a log scale are provided in Extended Data Fig. 2. d, The nucleotide levels in cells pretreated with arginine hydroxamate or ATP synthesis inhibitor CCCP or arsenate (As) for 30 min (top). The nucleotide levels were measured using thin-layer chromatography (TLC) and normalized to their levels before induction. Bottom, persistence of the pretreated cells to vancomycin treatment. e,f, Schematic (e) and quantification (f) of the survival of exponential phase (Exp), serial-passaged populations after vancomycin treatment for 5 h. Persisters that remained through the passage were defined as spontaneous persisters. g, In B. subtilis and many Gram-positive bacteria, (p)ppGpp is synthesized by three (p)ppGpp synthetases. Rel is the bifunctional (p)ppGpp synthetase and hydrolase, whereas SasB (also known as SAS1, RelQ or YjbM) and SasA (also known as SAS2, RelP or YwaC) are monofunctional synthetases. h, Persister levels in starvation-induced (arginine hydroxamate; black) WT and (p)ppGpp mutants. i, The levels of spontaneous persisters (green, second passage) in WT and (p)ppGpp mutants. relSyn (relD264G) is a synthetase-inactive variant of Rel that retains its hydrolase activity essential for viability in the presence of sasB and sasA. sasBF42A encodes an allosteric pppGpp-binding-site mutant of SasB. For a–d, f, h and i, data are mean ± s.d. from three or more biological replicates. For d, f, h and i, P values were calculated using unpaired two-tailed t-tests. We therefore examined the effect of (p)ppGpp on antibiotic resistance, tolerance and persistence using mutant B. subtilis lacking all three (p)ppGpp synthetases (hereafter, (p)ppGpp0)18,19. Compared with the wild-type strain, the (p)ppGpp0 strain exhibited similar resistance to a variety of antibiotics (MIC assay; Supplementary Fig. 1), a modestly reduced antibiotic tolerance (approximately 5% reduction in MDK99, where MDK99 is the minimum duration of killing for 99% of the population; Extended Data Fig. 1a,b) and strongly reduced (around tenfold) antibiotic persistence (Fig. These results indicate that (p)ppGpp predominantly promotes antibiotic persistence. Antibiotic persistence can be triggered by starvation or occur spontaneously1. To investigate the role of (p)ppGpp in starvation-triggered persistence, we starved cells using several methods: allowing growth into stationary phase, inducing amino acid starvation, or depleting ATP with arsenate10 or CCCP20. We then performed kill curves with vancomycin under these conditions or after starvation was removed by back-diluting in rich medium. 2a) showed a high level of triggered persisters (around 500-fold increase from 0.1% to about 50%) after all starvation treatments and even after starvation removal, consistent with the established definition of triggered persistence1. Notably, all starvation conditions also induced (p)ppGpp accumulation. We observed (p)ppGpp buildup not only after amino acid starvation but also during ATP depletion (Fig. Furthermore, starvation-triggered persistence was abolished in the (p)ppGpp0 mutant, indicating that (p)ppGpp induction is necessary for triggered persistence (Extended Data Fig. Finally, we found that, even in the absence of starvation, gratuitous induction of (p)ppGpp by overexpressing its synthetase21 was sufficient to produce high levels of persisters (Fig. These findings demonstrate that (p)ppGpp accumulation is both necessary and sufficient to trigger persistence. We next investigated the occurrence of spontaneous persistence. To differentiate spontaneous persisters from those triggered by starvation, we performed a classical serial passage experiment1,3 (Fig. In this method, cells are repeatedly diluted into fresh medium during exponential growth, which removes starvation-triggered persisters from the lag-phase culture while leaving a consistent, basal fraction of spontaneously generated persisters. This approach enabled us to identify a rare population of spontaneous persisters (around 0.05%) after the starvation-triggered persisters have been eliminated (Fig. Notably, the spontaneous persisters were absent in the (p)ppGpp0 mutant, indicating that (p)ppGpp has a key role in both spontaneous and starvation-triggered persistence. In B. subtilis, (p)ppGpp is synthesized by three enzymes: Rel, SasA and SasB (Fig. To assess their roles in persister formation, we analysed single, double and triple mutants of these synthetases (Fig. Our findings show that only the starvation-responsive synthetase, Rel, is essential for starvation-triggered persistence (Fig. By contrast, spontaneous persistence is controlled by both Rel and SasB, as mutations in either rel or sasB significantly reduce spontaneous persister levels and, in combination, they completely abolish this phenotype (Fig. Rel can be activated by pppGpp (Supplementary Fig. SasB, a tetrameric enzyme, enables allosteric activation of (p)ppGpp synthesis by pppGpp, with a Hill coefficient of around 3.0 (ref. A mutation that disrupts the allosteric pppGpp-binding site in SasB (sasBF42A) leads to a similar decrease in spontaneous persistence as observed in the ΔsasB mutant (Fig. These results suggest that self-amplification of (p)ppGpp synthesis is crucial for spontaneous persister formation. We next searched for the downstream effector of (p)ppGpp that mediates antibiotic persistence. (p)ppGpp is a bacterial alarmone with pleiotropic functions including upregulating competence and sporulation24. We found that disruption of competence, sporulation or three known TA systems in B. subtilis has little impact on persistence (Supplementary Fig. (p)ppGpp also antagonizes the essential nucleotide GTP in B. subtilis and downregulates macromolecular biosynthesis18. (p)ppGpp production consumes GTP, and (p)ppGpp directly inhibits multiple enzymes along the pathway of GTP production18,25,26,27 (Fig. Moreover, (p)ppGpp binds to the purine transcription regulator PurR to inhibit transcription of the pur operon, encoding de novo purine biosynthesis genes upstream of GTP synthesis28. Furthermore, (p)ppGpp competes with GTP to regulate macromolecule synthesis enzymes, including the DNA replication enzyme primase29,30, and GTPases for ribosome biogenesis, assembly and protein translation31,32,33,34. Under starvation conditions that trigger persisters, GTP levels were significantly depleted, whereas ATP levels varied (Fig. 1d), suggesting that GTP depletion may be a common effector leading to persistence. a, (p)ppGpp robustly inhibits de novo GTP biosynthesis by inhibiting the guanylate kinase Gmk and the IMP dehydrogenase GuaB, as well as inhibiting expression of the pur operon through the PurR repressor. (p)ppGpp also inhibits GTP synthesis by purine salvage, which is not illustrated here. b,c, Vancomycin-survival curves of WT, purine-biosynthesis mutants ∆purB, ∆purL and ∆purF, the GTP-synthesis mutant ∆guaA and the guaB-repressed mutant (guaBDown) (b); the slow-growth mutants ∆pyk (pyruvate kinase) and ∆ctaA (haem synthase) (c); and the (p)ppGpp0 and (p)ppGpp0 gmkQ110R strains (b). d, Measurement of low-GTP cells in the bacterial population. The PlowGTP reporter is activated when cellular GTP levels are low. A representative image of cells with low or high PlowGTP fluorescence is shown. e–j, The correlation between the growth rate (e,g,h,j), tolerance (e,f), persistence (h,i), population-averaged PlowGTP signals (j) and frequency of low-GTP cells (f,g,i; high PlowGTP fluorescence cells) in WT cells under different growth conditions. WT cells containing the PlowGTP reporter were grown in minimal medium with different carbon sources. Populations were measured for their growth rate, PlowGTP fluorescence and levels of high PlowGTP fluorescence cells using flow cytometry (around 1.5 × 106 cells), and their tolerance and persistence were determined on the basis of vancomycin-survival curves as shown in Extended Data Fig. Each value represents a single replicate. Three replicates for each condition are shown. r represents the Pearson correlation coefficient. P values were calculated using two-tailed Pearson correlation tests. To test whether GTP depletion mediates antibiotic persistence even in the absence of (p)ppGpp induction, we used our previously published strain with inducible downregulated expression of the guanine nucleotide synthesis gene guaB35 (guaBDown), lowering GTP without inducing (p)ppGpp (Supplementary Fig. We observed an increase in persistence by about tenfold (Fig. 2b), indicating that GTP depletion is sufficient to generate persisters. Moreover, we performed a non-biased transposon-insertion-sequencing (Tn-seq) analysis in B. subtilis to identify genes of which the disruption increased cell survival after prolonged antibiotic treatment (Extended Data Fig. The strongest hits mapped to 12 de novo purine biosynthesis genes in the pur operon and guaA, encoding GMP synthetase (Extended Data Fig. We verified that disruption of the pur operon or guaA reduces purine levels, including GTP (Extended Data Fig. 3d), and the corresponding kill curves confirmed increased antibiotic persistence (Fig. We next wanted to determine whether reducing GTP in the complete absence of (p)ppGpp can still promote persistence. Using a genetic screen, we identified a mutation in gmk (encoding guanylate kinase; Methods) that reduces the GTP levels of the (p)ppGpp0 strain back to the wild-type level (Extended Data Fig. This mutant (hereafter, (p)ppGpp0 gmkQ110R) restored persistence back to wild-type levels (Fig. 4), confirming that GTP depletion is sufficient for antibiotic persistence. 5), consistent with our previous findings that GTP depletion leads to slower growth35. This raises questions of whether increased antibiotic persistence is simply a consequence of slow growth, and whether it is always correlated with tolerance. Genetic screens in B. subtilis have identified slow-growing mutants, some of which deplete GTP while others do not. We tested a couple of slow-growing mutants that do not rely on GTP depletion. Notably, although these mutants exhibited increased tolerance, they do not show an increase in persistence; in fact, some showed decreased persistence (Fig. This suggests that, although tolerance and persistence are correlated in certain mutants or conditions11, they can be genetically uncoupled. To further distinguish between tolerance and persistence, we grew wild-type cells in different carbon sources, measuring their growth rates and antibiotic killing dynamics (Extended Data Fig. Our results revealed a strong correlation between slower growth rates and increased tolerance (Fig. 2e), but no significant correlation between growth rates and persistence (Fig. This demonstrates that, although tolerance and persistence can share overlapping characteristics, they represent distinct biological phenomena that can be independently regulated. Our findings, while unexpected, are explainable: although both slow growth and antibiotic tolerance are population-level phenomena, persistence reflects the behaviour of a phenotypically heterogeneous subpopulation of cells functioning as outliers. To differentiate population-based tolerance from the phenotypically heterogeneous subpopulation of persisters, it is critical to measure GTP levels in single cells. To achieve this, we constructed a fluorescent reporter for GTP levels in single cells. Building on our previous result that the ilvB gene is induced by approximately 20-fold after GTP depletion through the CodY repressor—which uses GTP as a co-repressor37,38—we cloned the CodY operator from ilvB to control fluorescent protein expression ectopically (Extended Data Fig. 2d), is fully repressed in (p)ppGpp0 cells, with fluorescence indistinguishable from background autofluorescence (Extended Data Fig. In (p)ppGpp+ cells, the reporter displays a dose-dependent increase in fluorescence across a physiological range of (p)ppGpp induction and GTP depletion (Extended Data Fig. This response to amino acid or ATP starvation is abolished in (p)ppGpp0 mutants (Supplementary Fig. Moreover, neither the PlowGTP reporter nor the CodY regulator interferes with antibiotic survival (Supplementary Fig. The PlowGTP reporter activity is inversely related to the ribosomal P1 promoter activity (Supplementary Fig. Although increased fluorescence could theoretically result from a failure to dilute the fluorescent protein during slow growth, our data show that PlowGTP fluorescence responds to changes in GTP levels rather than slow growth (Supplementary Fig. After calibrating the PlowGTP reporter in the bulk population, we examined whether it reflects GTP levels in individual cells. Fluorescence-activated cell sorting (FACS) of cells with high PlowGTP fluorescence, followed by analysis using liquid chromatography coupled with mass spectrometry (LC–MS), confirmed that the bright cells had notably lower GTP levels (Supplementary Fig. This demonstrates that the reporter accurately reflects GTP levels at both the population and single-cell levels. Together, these results confirm that our reporter reliably tracks GTP depletion rather than slow growth, enabling us to distinguish between slow-growing tolerant cells and GTP-depleted persister subpopulations. Using this reporter, we examined GTP distributions in single cells using flow cytometry (Fig. Most cells are dark during growth in rich medium, consistent with bulk measurements (Extended Data Fig. However, we identified a heavy-tailed subpopulation of rare, bright cells that depended on the presence of the reporter (Fig. This subpopulation was absent in (p)ppGpp0 cells but reappeared in (p)ppGpp0 gmkQ110R cells with rescued persistence (Supplementary Fig. The fractions of bright, low-GTP cells varied when cells were grown in medium with different carbon sources (Extended Data Fig. Notably, the fraction of low-GTP cells was strongly correlated with the fraction of persisters, measured using kill curves (Fig. 7a,b; 5 h survivors), but was not correlated with the growth rate (Fig. 2g,h,j) or antibiotic tolerance (Fig. These results distinguish persistence from tolerance, demonstrating that persistence under various growth conditions is driven by the fraction of low-GTP outliers, not by the population's average growth rate. Despite the strong correlation between persisters and low-GTP cells, it is essential to determine whether these two subpopulations were indeed the same. To explore this, we used two methods. First, we used FACS to sort exponentially growing wild-type cells containing the PlowGTP reporter (Fig. After 5 h of treatment and subsequent antibiotic removal, cells were plated. Around 80% of the sorted low-GTP cells formed colonies compared with only about 0.1% survival in the low-fluorescence population (Fig. This demonstrates that nearly all low-GTP cells are antibiotic-refractory persisters capable of regrowth after antibiotic removal. We confirmed that this persistence was specifically dependent on PlowGTP, using a differentially labelled Pveg control that measured the effect of protein dilution (Supplementary Fig. 15a), or through fluorescent protein swapping (Supplementary Fig. a, Schematic of the FACS sorting and antibiotic survival assay using the PlowGTP reporter. The bacterial population containing the reporter was sorted based on PlowGTP fluorescence and was immediately treated with antibiotic. b, WT cells containing the PlowGTP–GFP (green fluorescent protein) reporter were FACS-sorted into low or high PlowGTP fluorescence fractions. The fractions were treated with vancomycin and measured for survival over time. c, Representative time-lapse micrograph from three biological replicates of WT cells containing the PlowGTP reporter in response to antibiotic treatment. Exponentially growing WT cells containing the reporter were patched onto agarose pads made with growth medium. High PlowGTP fluorescence cells were identified (arrow) and monitored for antibiotic survival using carbenicillin and Sytox Blue staining (cyan) for viability. The numbers indicate time in minutes. d, The survival of low-GTP cells in biofilm. Representative time-lapse micrograph of WT cells containing both PlowGTP (for low GTP, green) and PrrnBP1 (for high GTP, yellow) reporters were grown into colony biofilm and then treated with vancomycin for 6 h. Scale bar, 50 µm. The bar graphs indicate changes in the high-GTP and low-GTP populations before and after treatment. P values were calculated using unpaired two-tailed t-tests. e, Representative single-cell dynamic trace of persister formation. A total of around 30,000 cell growth and division events was recorded to reveal about 10 persistence entrance events. The numbers indicate time in minutes. The arrows indicate a cell with increased PlowGTP fluorescence followed by growth attenuation. f, Quantified changes in cell length (left) versus PlowGTP fluorescence (right) from the single trace in e. The dotted lines indicate persistence entrance. g, Summary data of the specific growth rate and reporter fluorescence from about 100 independent single-cell traces, similar to e (ten entrance traces). In addition to FACS sorting, we used time-lapse microscopy to monitor cell fate during antibiotic treatment. We found that only bright cells were non-growing before antibiotic treatment and remained viable during the treatment, while the rest of the cells, which elongated and divided, were rapidly killed (Fig. To further investigate persisters in biofilms, where they are commonly found39, we developed a microfluidic biofilm system with confocal microscopy (Supplementary Fig. Using dual-labelled cells (PrrnBP1 and PlowGTP), we identified a mixture of persisters and growing cells in microfluidic biofilms (Fig. After prolonged (6 h) antibiotic treatment, the low-GTP cells survived within the biofilm (Fig. 3d), confirming that the majority of persisters in biofilms are low-GTP cells. After confirming that PlowGTP-bright cells are persisters, we investigated how these persisters form before antibiotic treatment. Our goal was to observe how rare, spontaneous persisters emerge from actively growing wild-type cells, and to examine how this process relates to GTP levels. We seeded highly diluted, growing cells onto solid growth medium and monitored around 30,000 growth and division events (Supplementary Fig. From this, we identified ten cells that entered a persister state before reaching a high cell density (Fig. We retrospectively analysed the dynamics of their transition into persistence by quantifying their growth, division and GTP reporter activity. This revealed distinct, switch-like behaviours at the single-cell level (Fig. Initially, each pre-persister cell maintained the same elongation rate as actively growing cells, while its PlowGTP fluorescence gradually increased, often over more than one cell division (Supplementary Fig. This was followed by a sudden stop in elongation, marking entry into dormancy when the PlowGTP fluorescence reached a threshold (Fig. In all cases, the reporter fluorescence increased before growth inhibition occurred (Fig. 8a–i), indicating that GTP depletion triggers growth inhibition rather than resulting from it. The fluorescence thresholds were highly consistent across different persister entry events (Fig. 3g), suggesting that the switch to persistence is activated when intracellular GTP levels fall to a specific threshold. This threshold aligns with values seen in our microscopy and flow cytometry experiments, corresponding to an estimated GTP concentration of around 0.1–0.2 mM (ref. This range is comparable to what is observed in wild-type cells experiencing strong (p)ppGpp accumulation (Extended Data Fig. 6c,d), where most cells became starvation-triggered persisters. While GTP depletion leads to persistence, we next examined the role of (p)ppGpp in persister dynamics. Notably, the (p)ppGpp0 gmkQ110R mutant, which has GTP levels similar to wild-type cells but lacks (p)ppGpp (Fig. 2c), can still form spontaneous persisters (Fig. However, these mutants do not exhibit the switch-like dynamics characteristic of persister entry in wild-type cells. Instead, their entry into dormancy is gradual, with elongation slowing as reporter activity increases (Fig. Moreover, in these mutants, GTP levels and growth rates show a linear relationship (Fig. 3g), contrasting with the switch-like relationship seen in wild-type cells. Thus, (p)ppGpp promotes GTP depletion, enabling the rapid switch-like entry into persistence observed in wild-type cells. Finally, we used our established system to investigate the poorly understood phenomenon of antibiotic-induced persistence1. We found that PlowGTP fluorescence increased in cells exposed to lethal concentrations of cell wall antibiotics, including carbenicillin, vancomycin and bacitracin (Fig. While most cells were killed by irreversible damage, a few PlowGTP-bright cells survived and resumed growth after the antibiotics were removed (Extended Data Fig. LC–MS analysis revealed that exposure to both lethal and sublethal doses of bacitracin led to the accumulation of alarmones, such as ppGpp and ppApp, and a decrease in GTP levels (Fig. These results suggest that cell wall antibiotic treatment induces alarmone production and depletes GTP. Using mutants of (p)ppGpp synthetases, we found that a single alarmone synthetase, SasA21, is responsible for (p)ppGpp accumulation and GTP depletion during cell wall antibiotic treatment (Fig. a, Cell wall antibiotics induce PlowGTP reporter fluorescence. b–d, The levels of ppGpp (b), ppApp (c) and GTP (d) in WT cells treated with sublethal or lethal concentrations of bacitracin for 0.5 h. Sublethal (SL), 0.5× MIC bacitracin; lethal (L), 3× MIC bacitracin. e,f, The levels of cell wall antibiotic-induced persisters in WT and (p)ppGpp mutants. e, Strains containing the PlowGTP reporter before and after 0.5× MIC bacitracin treatment for 0.5 h were measured for generation of low-GTP cells using flow cytometry (around 1 × 106 cells per sample per replicate, three replicates). The levels of induced low-GTP cells were determined by subtracting the levels in uninduced populations. f, The survival of WT or (p)ppGpp mutants pretreated with 0.5× MIC for 0.5 h, followed by lethal (3× MIC) bacitracin treatment for up to 5 h. The levels of induced persistence were determined by subtracting the survival of uninduced populations at 5 h. g, Survival curve of WT or ∆sasA cells pretreated with sublethal (0.5× MIC) bacitracin for 0.5 h, followed by lethal (3× MIC) bacitracin treatment for up to 5 h. For a–g, data are mean ± s.d. from three or more biological replicates. Statistical comparisons were performed between WT and mutant pairs or untreated and treated pairs using unpaired two-tailed t-tests. h, Model for persistence pathways mediated by (p)ppGpp-GTP antagonism. Starvation-triggered persistence (purple) is mediated primarily through the (p)ppGpp synthetase Rel. Spontaneous persistence is mediated through the activities of two (p)ppGpp synthetases, Rel and SasB (green). Persisters can also be induced by cell wall antibiotics through SasA (orange). Thus, (p)ppGpp enables integration of different signals to trigger persistence through GTP depletion. To assess whether this alarmone response provides a protective effect, we measured persistence due to treatment with sublethal concentrations (below MIC) of bacitracin (Fig. While sublethal bacitracin did not impact cell viability, it did induce SasA expression41,42 (Supplementary Fig. Notably, when followed by a lethal dose of bacitracin (3× MIC), cells pretreated with sublethal antibiotics showed approximately five times more persisters than non-pretreated cells, with minimal differences in tolerance (Fig. This survival mechanism, dependent on SasA but not on other alarmone synthetases (Fig. 4f), enables pre-exposed cells to better resist lethal antibiotic exposure. Thus, antibiotic-induced persistence may serve as an adaptive strategy, substantially enhancing cell survival against a later lethal antibiotic challenge. Our data highlight the pivotal role of (p)ppGpp in persistence. We demonstrate that conditions triggering persistence10 also induce (p)ppGpp accumulation, that persistence depends on (p)ppGpp accumulation, and that three (p)ppGpp synthetases are differentially responsible for starvation-triggered, spontaneous and antibiotic-induced persistence (Fig. While (p)ppGpp senses environmental signals to initiate persistence, GTP antagonism functions as a critical effector, probably through coordinated suspension of multiple GTP-dependent processes such as protein translation and DNA synthesis, as well as potentiating competitive inhibition by (p)ppGpp (Extended Data Fig. Our research also distinguishes persisters from tolerators, revealing that persisters arise through specific genetic pathways that generate individual cells with low GTP, independent of population-wide growth rates or bulk GTP levels. Our results suggest a metabolic model for spontaneous persistence. Phenotypic switch into persistence can be mediated by competition between toxins and antitoxins in TA modules to provide a threshold-like mechanism3,12,43. Our findings suggest that cells can also use an alternative mechanism for the spontaneous entrance, through allosteric enzyme cooperativity and self-amplification of (p)ppGpp synthesis (Supplementary Fig. This enables rapid amplification of a small signal, such as from noise in (p)ppGpp synthesis, to allow spontaneous persister formation in rare individual cells. Our study also provides strong evidence for bactericidal antibiotic-induced persistence. While previous research suggested that antibiotics induce persistence through cellular damage15 and stress responses such as the SOS pathway44, our results show that antibiotics, particularly cell-wall-targeting ones, induce persistence through the accumulation of alarmones like (p)ppGpp, driven by SasA expression in response to cell envelope damage41,42,45. In the soil bacterium B. subtilis, the evolution of this regulation may be shaped by a fitness advantage from its habitat to enhance survival to diffusible antimicrobials from neighbouring microorganisms. For example, bacitracin, which we show to induce persistence, is a diffusible natural antibiotic produced by other soil Bacillus species46. Clinically, this model may explain why (p)ppGpp mutants in pathogens in the Bacillota phylum, such as Staphylococcus aureus, show reduced survival against cell-wall-targeting drugs47 and diminished clearance under combination antibiotic therapies48. Finally, persistence may have a key role in the emergence of genetic resistance by expanding the reservoir of surviving bacteria during antibiotic treatment. Given that antibiotic overuse increases bacterial exposure to these drugs, driving the evolution of resistance49, targeting persister formation could improve treatment outcomes and help curb the rise of antibiotic-resistant infections. A list of all of the bacterial strains, plasmids and oligonucleotides used in this study is provided in Supplementary Tables 1–4. The combination of lincomycin (12.5 µg ml−1) and erythromycin (0.5 µg ml−1) was used to select for macrolide–lincosamide–streptogramin (MLS) resistance. Carbenicillin (100 µg ml−1) was used for selection in E. coli. B. subtilis (p)ppGpp biosynthesis mutants were constructed by transformations of integration plasmids containing an I-sceI endonuclease cut site and regions of homology upstream and downstream of specific synthetase genes (pJW300 for ∆sasB; pJW370 for sasBF42A; pJW306 for ∆sasA; and pJW371 for relD264G) followed by transformation of pSS4332 for marker-less recombination50. Successful recombination was verified by PCR. For the construction of the (p)ppGpp0 mutant, ∆sasA ∆sasB cells were transformed with ∆rel::mls PCR product from genomic DNA using the oligos oJW902 and oJW903 followed by MLS resistance selection26. Construction of integration plasmid for sasBF42A was performed using site-directed mutagenesis of pJW370 by PCR using the oligos oJW2309 and oJW2310. The (p)ppGpp0 gmkQ110R mutant was obtained from isolating suppressor mutants from (p)ppGpp0 cells by plating on S7 minimal medium plates containing 1% glucose. The surviving colonies were plated on S7 minimal medium plates containing 0.5% casamino acids and 0.5 mM 8-azaguanine, or S7 minimal medium plates containing 0.5% casamino acids and 0.1 mM guanosine26 to differentiate between mutants with mutations in hprT or gmk. Colonies that can grow on guanosine but not 8-azaguanine were sequenced to identify the mutant gmk allele. Whole-genome sequencing was performed to confirm that gmkQ110R is the only mutation in the strain. The guaBdown mutant in the (p)ppGpp+ background was constructed by transformation of pJW30526, which replaces the chromosomal copy of guaB with an isopropyl-β-d-1-thiogalactopyranoside (IPTG)-inducible copy of guaB (Pspac-guaB). This enables controllable guaB expression using IPTG during strain construction and growth to avoid generation of suppressors. B. subtilis deletion mutants were constructed by serial transformation of PCR products from the B. subtilis knockout collection (BGSC, Gross laboratory)51. Where required, the lox-site-flanked ermR or kanR cassette was removed using pDR244-cre followed by selection for the loss of MLS or Kan resistance. Construction of PlowGTP fluorescence reporters was done by fusion of PCR products containing the PlowGTP promoter (primers oJW1935 and oJW1936) with coding regions of fluorescence proteins (primers oJW1995 and oJW1996 (for GFP) or oJW2805 and oJW2806 (for mCherry)) using ligase cycling reaction (LCR)52. The resulting PCR products were fused by LCR followed by amplification using PCR to generate the linear recombination fragment of lacA::PrrnBP1-GFPns-lox-ermR-lox for transformation. The resulting PCR products were fused by LCR followed by amplification using PCR to generate the linear recombination fragment of lacA::PsasA-mCh-lox-ermR-lox for transformation. The resulting PCR products were fused by LCR followed by amplification using PCR to generate the linear recombination fragment of lacA:: Pveg-GFP-lox-ermR-lox for transformation. Removal of the lox-site flanked ermR cassette was done by transformation with pDR244-cre and selecting for the loss of MLS resistance. All of the mutants and constructs were verified by DNA sequencing. B. subtilis strains were grown in S7 defined medium54; MOPS was used at 50 mM rather than 100 mM, supplemented with 0.1% glutamate, 1% glucose and 0.5% casamino acids. Growth of the YB886 strain background was supplemented with 20 µg ml−1 tryptophan and 50 µg ml−1 methionine. For growth in minimal medium, both glutamate and casamino acids were replaced with 200 µg ml−1 l-isoleucine, 200 µg ml−1 l-leucine and 200 µg ml−1 l-valine, and 1% carbon sources were used as indicated. Routinely, cells from young colonies on overnight LB-agar plates at 37 °C (<12 h) were inoculated into growth medium and then grown at 37 °C with 250 rpm shaking. Cultures in logarithmic phase (optical density at 600 nm (OD600) of around 0.1–0.3) were treated with antibiotics or inducers, including arginine hydroxamate (RHX, 0.5 mg ml−1), carbonyl cyanide m-chlorophenyl hydrazone (CCCP, 5 µM), sodium azide (NaN3, 4 mM) or arsenate (2.5 mM). IPTG was added to a final concentration of 0.5 mM to induce guaB expression from an IPTG-inducible promoter (Pspac), while depletion of guaB expression was done by omitting IPTG in the growth medium. MICs for chloramphenicol, tetracycline, kanamycin, ciprofloxacin, norfloxacin, rifampicin, bacitracin and vancomycin were determined using the microdilution method55. Logarithmic-phase cells were back-diluted to a final titre of around 5 × 105 colony-forming units (CFU) per ml into 96-well plates containing twofold serial dilutions of respective antibiotics in S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids. After incubation for 16–20 h at 37 °C with 250 rpm shaking, the MIC was determined as the lowest drug concentration that prevented visible growth. For growth measurement, fresh colonies of B. subtilis strains on LB agar were resuspended into different growth media as specified and diluted to OD600 ≈ 0.005 in 96-well plates. Growth was monitored by OD600 at 37 °C under shaking in the Synergy2 microplate reader (BioTek). Doubling times were estimated by fitting the growth data to the exponential growth curve using a custom Python script. To prepare exponentially growing B. subtilis populations, cells from young colonies on overnight LB-agar plates at 37 °C (<12 h) were inoculated into S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids, and grown to OD600 ≈ 0.1–0.3 at 37 °C, 250 rpm. For growth in minimal medium, both glutamate and casamino acids were replaced with 200 µg ml−1 l-isoleucine, 200 µg ml−1 l-leucine and 200 µg ml−1 l-valine, and 1% carbon sources were used as indicated. To determine cell viability, culture aliquots were taken at T = 0 and at designated times after treatment, serially diluted and plated onto LB agar. The viability at different timepoints, determined as the CFU per ml and relative survival (versus T0), was calculated. For experiments involving pre-induction of cells with (p)ppGpp-inducing agents, cells were grown to OD600 ≈ 0.1 and divided into two cultures: one containing the inducing agent (RHX, 0.5 mg ml−1; bacitracin, 64 μg ml−1; CCCP, 5 μM; NaN3, 4 mM; arsenate, 12.5 mM) and other as a non-induction control. The cultures were grown for an additional 30 min under the same conditions (T = 0.5 h) and subjected to the persister assay, as described above. In the case of measuring spontaneous persistence, we defined spontaneous persisters as those that are generated during growth in a non-stressed condition1,3. To achieve this, early-exponentially growing cultures were 1:100 back-diluted into fresh medium and regrown to exponential phase (OD600 ≈ 0.1–0.3) for one or two rounds followed by antibiotic treatment1. To exclude the contribution of persisters to population tolerance, we determined the level of persisters in the biphasic killing curve and subtracted this subpopulation from the bulk population. To measure intracellular nucleotides, cells were first collected from overnight plates, back-diluted to OD600 = 0.005 and grown in low-phosphate (0.1× phosphate, 0.5 mM) S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids. Once cultures reached OD600 ≈ 0.05, 1 ml cells was labelled with 50 µCi of 32P orthophosphate (900 mCi mmol−1; Perkin Elmer) for 2–3 generations before treatment or sampling. At OD600 ≈ 0.15, RHX, CCCP or arsenate were added to the cultures and the samples were collected at regular timepoints for nucleotide extraction. Nucleotides were extracted by incubating 100 µl cells with 20 µl of 2 M formic acid on ice for at least 20 min. The samples were spotted onto PEI cellulose TLC plates (Millipore, 1055790001) and resolved in 1.5 M or 0.85 M potassium phosphate monobasic (KH2PO4, pH 3.4) buffer to separate (p)ppGpp or GTP, respectively. TLC plates were exposed on storage phosphor screens (GE Healthcare) and scanned on the Typhoon imager (GE Healthcare). LC–MS quantification of nucleotides was performed as described previously21. Cells were grown in S750 medium supplemented with 20 amino acids21 to OD600 ≈ 0.3 at 37 °C, 250 rpm before collection. Then, 25 ml of cultures was sampled and filtered through PTFE membrane (Sartorius, 14555419). For experiments involving bacitracin induction, cells were collected before and after 0.25× MIC (sublethal) or 1.25× MIC (lethal) bacitracin treatment for 30 min. Membranes with cell pellets were submerged in 3 ml extraction solvent mix (on ice 50:50 (v/v), chloroform:water) to quench metabolism, lyse the cells and extract metabolites. Mixtures of cell extracts were centrifuged at 5,000g for 10 min to remove organic phase and then centrifuged at 20,000g for 10 min to remove cell debris. The samples were analysed using an HPLC–MS system, consisting of a Vanquish UHPLC system linked through electrospray ionization (ESI, negative mode) to the Q Exactive Orbitrap mass spectrometer (Thermo Fisher Scientific) operated in full-scan mode to detect targeted metabolites based on their accurate masses. LC was performed on the Acquity UPLC BEH C18 column (1.7 μm, 2.1 × 100 mm; Waters). The total run time was 30 min with a flow rate of 0.2 ml min−1, using solvent A as denoted above and acetonitrile as solvent B. The gradient was as follows: 0 min, 5% solvent B; 2.5 min, 5% solvent B; 19 min, 100% solvent B; 23.5 min, 100% solvent B; 24 min, 5% solvent B; 30 min, 5% solvent B. Quantification of metabolites was performed using MAVEN software56 and normalized to the OD600 at the time of cell collection. For LC–MS analysis of FACS-sorted cells, membranes with filtered cells (about 8 × 106 cells) were submerged in 1.5 ml extraction solvent mix (methanol:acetonitrile:H2O, 40:40:20) to quench metabolism, lyse the cells and extract metabolites. The cell extract was centrifuged at 21,000g for 10 min at 4 °C, and 1 ml of supernatant was then transferred to a new microcentrifuge tube and dried completely with the SpeedVac. Dried metabolites were then resuspended in 100 μl solvent A (97:3 (v/v) water:methanol, 10 mM tributylamine ~pH 8.2–8.5 adjusted with ~9 mM acetic acid). The samples were analysed using an HPLC-MS system consisting of a Vanquish UHPLC system linked through heated electrospray ionization (ESI, negative mode) to a hybrid quadrupole high-resolution mass spectrometer (Q-Exactive Orbitrap, Thermo Fisher Scientific) operated in full-scan selected ion monitoring (MS-SIM) mode to detect targeted metabolites on the basis of their accurate masses. Only ions with a retention time of 10–15 min were scanned by MS. LC was performed on the Acquity UPLC BEH C18 column (1.7 μm, 2.1 × 100 mm; Waters). The total run time was 30 min with a flow rate of 0.2 ml min−1, using solvent A and 100% acetonitrile as solvent B. Raw output data from the MS was converted to mzXML format using custom software. Quantification of metabolites was performed using MAVEN software56 and normalized to an internal standard of six most represented nucleotides detected in the sample. Metamorph Advanced (v.7.8.3.0) (Molecular Devices) was used for microscopy data collection. Single-cell time-lapse imaging was performed at 15 min intervals for each field at 37 °C using a temperature-controlled imaging chamber (Tokai Hit) coupled to an automatic stage and the microscope control as described previously57. When comparing phenotypes between strains, both strains were imaged in parallel on the same imaging dish using the same microscope with same settings. For imaging persister survival in time-lapse experiments, final concentrations of 5 μg ml−1 carbenicillin (10× MIC) and 200 nM Sytox Blue or propidium iodide (Molecular Probes) were applied to the agarose pads at designated times. To remove carbenicillin after treatment, 5 U ml−1 final concentration of penicillinase (Sigma-Aldrich, P0389) was applied. Strains without the fluorescence reporters were used for autofluorescence measurement. B. subtilis biofilms were grown on a custom microfluidic device fabricated with polydimethylsiloxane. The device contains a central chamber connected to inlet and outlet media channels, allowing for constant medium flow through the central chamber. A semipermeable dialysis membrane was fixed on top of the central chamber to provide a platform for biofilm growth. This setup allows diffusion of nutrients or small molecules from the medium flow underneath to support biofilm growth on the membrane and allows subsequent treatment with antibiotics. To grow B. subtilis biofilms, 1 µl of early exponential phase culture (OD600 ≈ 0.05) was applied onto the membrane and grown at room temperature (25 °C) for 24 h under a constant flow of S750 medium supplemented with 0.5% glutamate and 0.5% glycerol. Imaging of biofilms was performed with the IXplore SpinSR confocal imaging microscope (Olympus) using a ×20 phase-contrast objective with fluorescence filters (488 nm laser with 510–550 nm emission filter for GFP; 561 nm laser with 575–625 nm emission filter for mCherry). Biofilms at 24 h after inoculation were imaged before and after switching to the same growth medium containing 4 µg ml−1 vancomycin. In total, 51 stacks at 1 µm intervals were taken for each timepoint. For quantitation of reporter signals within the biofilm, we sampled around four random regions within the centre of the biofilm and measured their GFP and mCherry intensities. The background fluorescence was measured from regions without biofilm and used for background subtraction. To prepare samples for flow cytometry, cells from young colonies on overnight LB-agar plates at 37 °C (<12 h) were inoculated into and grown in S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids to OD600 ≈ 0.1–0.3 at 37 °C, 250 rpm. For growth in minimal medium, both glutamate and casamino acids were omitted and 1% carbon sources were used as indicated. Cells were immediately fixed with 0.4% paraformaldehyde for 15 min at room temperature, washed three times with 1× PBS and kept at 4 °C until analysis. Fixation was verified by viability plating and microscopy. Flow cytometry analysis was performed using the BD LSRFortessa flow cytometer (BD Biosciences) with a 70 µm nozzle. BD FACSDiva v.8.0.2 was used for data collection. Cell populations were detected using both forward and side scatter (FSC and SSC). Single-cell fluorescence was measured using the 488 nm laser and detection filters for GFP (530/30 nm, 505LP dichroic filter). Autofluorescence was measured by analysing parental strains without the fluorescence reporter and subtracted from the raw reporter fluorescence. Approximately 1.5 million events were measured for each sample. For the determination of antibiotic-induced persistence, the frequency of low-GTP cells after antibiotic induction was subtracted from the frequencies before induction. FACS was performed at the UWCCC flow cytometry core. To prepare samples for cell sorting, cells were collected from young colonies on overnight LB-agar plates at 37 °C (<12 h) and grown in S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids to OD600 ≈ 0.3 at 37 °C, 250 rpm. FACS analysis was performed using the BD FACSAria cell sorter (BD Biosciences) with a 70 µm nozzle at room temperature using the 488 nm laser and 530/30 nm detection filters for GFP, and the 561 nm laser and 610/20 nm detection filters for mCherry. BD FACSDiva v.8.0.2 was used for data collection. Autofluorescent cells were eliminated by gating using an isogenic strain without the fluorescent reporters. At least 1,000 cells were obtained from the rarest gate for each sample. Cell recovery rate was estimated to be >90% based on viability counting on LB plates. For antibiotic treatment, cells were directly sorted into tubes containing 4× MIC of vancomycin followed by shaking at 37 °C. Aliquots were taken at different times for serial dilution and plating to measure survival by colony counting. For FACS-sorting of cells for LC–MS analysis, cells containing the PlowGTP reporter were grown in S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids to either exponential (OD600 ≈ 0.2) or stationary (OD600 ≈ 4.0) phase. Both populations were mixed at a 10:1 ratio, and immediately FACS-sorted into low-fluorescence or high-fluorescence fractions. Autofluorescent cells were eliminated by gating using an isogenic strain without the fluorescent reporters. Approximately 8 × 106 cells were sorted and filtered on the PTFE membrane (Sartorius, 14555419) to remove the sheath fluid. Membranes with filtered cells were then subjected to metabolite extraction and LC–MS analysis. CFU count analysis was performed from a small aliquot of the sorted fractions to account for potential variations in the number of cells sorted between fractions. Construction of the library was performed as follows. In brief, in vitro transposition of B. subtilis 168 genomic DNA (gDNA) with magellen6x transposon was performed by mixing 1.3 µg pCJ41 (containing magellen6x transposon), 34 ng purified MarC9 transposase, 5 µg B. subtilis gDNA, 10 µl 2× buffer A (41 mM HEPES pH 7.9, 19% glycerol, 187 mM NaCl, 19 mM MgCl2, 476 µg ml−1 BSA and 3.8 mM DTT) into a 20 µl reaction in vitro and incubated overnight at 30 °C. The transposed DNA was precipitated and resuspended in 2 µl 10× buffer B (500 mM Tris-Cl pH 7.8, 100 mM MgCl2, 10 mM DTT), 2 µl 1 mg ml−1 BSA and 11 µl H2O followed by 4 h incubation at 37 °C. After incubation, 4 µl of 2.5 mM dNTPs and 1 µl of 3U µl−1 T4 DNA polymerase were added to the DNA and further incubated for 20 min at 12 °C, followed by heat-inactivation at 75 °C for 15 min. Next, 0.2 µl 2.6 mM NAD and 1 µl of 10 U µl−1 E. coli DNA ligase were added and the reaction was incubated overnight at 16 °C. The resulting in vitro transposed and repaired gDNA was transformed into B. subtilis 168 and plated onto LB agar containing spectinomycin and incubated overnight. Colonies containing the transposon were washed off and pooled into a single library. The library was estimated to contain around 50,000 unique transposon inserts across the genome. For the selection experiment with the transposon library, an aliquot of the library was inoculated and grown in S750 medium with 0.1% glutamate, 1% glucose and 0.5% casamino acids supplemented with tryptophan (20 µg ml−1) at 37 °C, 250 rpm. At OD600 ≈ 0.1–0.3, the cultures were treated with 20× MIC vancomycin or ciprofloxacin. Cultures before and after antibiotic treatment were plated onto LB plates and recovered after incubation for around 14 h at 37 °C. Around 650,000 colonies from each sample were pooled and snap-frozen for gDNA extraction and sequencing library preparation. Preparation of the sequencing library was performed as previously described59. Frozen cell pellets were resuspended in 500 µl lysis buffer with lysozyme and RNase A (20 mM Tris-HCl pH 7.5, 50 mM EDTA, 100 mM NaCl, 2 mg ml−1 lysozyme, 120 µg ml−1 RNase A) and incubated at 37 °C for 20–30 min. Next, the incubated cell lysate was mixed with 60 µl 10% N-lauroylsarkosine and further incubated at 37 °C for 15 min. gDNA was purified using 600 μl phenol, then 600 µl phenol:chloroform:isoamyl alcohol (25:24:1) and finally 600 µl pure chloroform. DNA in the aqueous phase was precipitated using 1/10 volumes of 3 M NaOAc and 2 volumes of 100% ethanol. The DNA pellet was washed with 70% ethanol, air-dried on the bench and resuspended in 10 mM Tris-HCl, pH 8.5 and stored at 4 °C. DNA was digested for 2.5 h at 37 °C, after which 2 µl calf intestinal phosphatase (10,000 U ml−1, NEB) was added to the digest and the sample was incubated for 1 h at 37 °C. Digested gDNA was extracted with 200 µl phenol:chloroform:isoamyl alcohol (25:24:1) followed by 200 µl of pure chloroform. DNA in the aqueous phase was first mixed with 1/10 volume of 3 M NaOAc and 67 ng ml−1 glycogen, and then 2.5 volumes of 100% ethanol. Precipitated DNA was washed with 150 µl 70% ethanol twice at room temperature, air-dried and resuspended in 15 µl of double-distilled H2O. For annealing of the DNA adaptor, 20 µl of 100 µM synthesized oligos (IDT) were mixed with 1 µl of 41 mM Tris-HCl pH 8.0 (final concentration of adaptor, 50 µM in 1 mM Tris-HCl pH 8.0). Oligos were annealed by heat denaturation (95 °C for 5 min) and stepwise cool-down (94 °C for 45 s then repeat with −0.3 °C per cycle for 250 cycles, then hold at 15 °C) using a PCR machine. Annealed adaptors were diluted to 3.3 µM in double-distilled H2O and stored at −20 °C. The ligation mix was incubated overnight at 16 °C in a PCR machine. Amplification of the adaptor-ligated DNA library was performed using barcoded primers and Phusion high-fidelity DNA polymerase (NEB) for 18 cycles according to the provided instructions. The PCR products were mixed in equal amounts, purified by size exclusion and submitted for sequencing using the Illumina sequencing primer (5′-ACACTCTTTCCCTACACGACGCTCTTCCGATCT-3′). Deep sequencing was performed on the Illumina HiSeq 2500 (Illumina) system by the University of Michigan DNA Sequencing Core. Analysis of sequencing data was performed using a custom Python script and mapped to the B. subtilis 168 reference genome (NCBI: NC_000964.3). Visual inspection of transposon insertion profiles was done using GenomeBrowse (Golden Helix). The plasmid for Rel purification was constructed as follows. The B. subtilis rel coding sequence was PCR amplified from NCIB3610 genomic DNA using the primers oJW3196 and oJW3197. The PCR products were assembled to generate pJW753 by Golden Gate assembly (New England BioLabs). Plasmids were verified by DNA sequencing. To express His6–SUMO–Rel, fresh transformants of E. coli BL21 carrying pE-SUMO-rel were grown in LB at 37 °C to OD600 ≈ 0.5, followed by 1:50 dilution into Terrific Broth and grown at 30 °C until OD600 ≈ 1.5. His6–SUMO–Rel expression was induced with 1 mM IPTG for 4 h at 30 °C. Cells were pelleted and stored at −80 °C until use. Frozen cell pellets were thawed on ice, resuspended in ice-cold lysis buffer (50 mM Tris-HCl, pH 8, 1 M NaCl, 10 mM imidazole, DNase and cOmplete protease inhibitor (Roche)), and lysed using a French press at 4 °C. The filtered supernatant was injected into the ӒKTA FPLC system (GE Healthcare) and passed through the HisTrap FF column (GE Healthcare). His6–SUMO–Rel was eluted with a gradient of buffer A (50 mM Tris-HCl, pH 8, 1 M NaCl, 5% glycerol, 10 mM imidazole) and buffer C (50 mM Tris-HCl, pH 8, 1 M NaCl, 5% glycerol, 500 mM imidazole). The fractions containing the protein were pooled with 300 µl SUMO protease into Spectra/Por dialysis tubing (Spectrum), and dialysed into 50 mM Tris-HCl, 1 M NaCl, 1 mM β-mercaptoethanol and 5% glycerol overnight. Rel without the His6–SUMO tag was passed through the HisTrap FF column and then purified by size exclusion using the Superose 12 10/300 GL column (GE Healthcare) with the ӒKTA FPLC system. Aliquots were snap-frozen using liquid nitrogen and stored at −80 °C. In vitro pppGpp synthesis by Rel was monitored by measuring synthesis of radiolabelled pppGpp over time. The reaction lacked manganese to avoid a potential effect of pppGpp hydrolysis. At the indicated times, 10 µl of the reaction was mixed with 2 µl of 2 M formic acid and chilled on ice for 20 min to quench the reaction. Then, 1 µl samples were spotted onto PEI cellulose TLC plates (Millipore, 1055790001) and resolved in 1.5 M potassium phosphate monobasic (KH2PO4, pH 3.4) buffer to separate pppGpp. TLC plates were dried and exposed on storage phosphor screens (GE Healthcare) and scanned on the Typhoon imager (GE Healthcare). For TLC experiments, nucleotide spots were quantified using ImageQuant v.5.0 (Molecular Dynamics). The raw intensities were corrected to the number of phosphates in the corresponding nucleotide and normalized to OD600 or ATP level before treatment (ATPT=0) for comparison between samples. For in vitro pppGpp synthesis assays, changes in pppGpp levels were normalized to T = 0. Microscopy image analysis and cell parameters (cell area and fluorescence intensity) measurements were performed using Metamorph Advanced (v.7.8.3.0) (Molecular Devices). Background and autofluorescence were subtracted by comparing images obtained from identical strains without the fluorescence reporter. The single-cell specific growth rate (µ) at each frame was calculated using the equation \(\mu =\frac{1}{l}\frac{\Delta l}{\Delta t}\), where l is the cell length, Δl is the change in cell length and Δt is the change in time (in min). Flow-cytometry data were analysed using FlowJo X (FlowJo); an example gating strategy is shown in Supplementary Fig. Cells within a narrow range of cell sizes were gated, subgated to filter cell aggregates and then measured for their fluorescence distribution. Autofluorescence was measured using isogenic strains without the fluorescent reporters and subtracted from raw reporter signals. Gating for PlowGTP-high cells was set at fivefold or higher above mean population fluorescence. This cut-off agrees with our FACS sorting experiment in which cells with reporter fluorescence above this threshold are predominantly persisters. For the determination of antibiotic-induced persistence, the frequency of low-GTP cells after antibiotic induction was subtracted from the frequencies before induction. Statistical information of individual experiments is included in the figure legends. n represents the number of biological replicates or number of cells for experiments involving single-cell measurements as indicated in the legends. Significance was tested using unpaired two-tailed t-test or as specified in the figure legend. Confidence intervals were calculated based on binomial distribution. Prism 7 (GraphPad) was used for statistical analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data generated in this study are included in the figures and Supplementary Information as much as possible. The sequencing data used for Tn-seq analysis are available at SRA under BioProject under accession number PRJNA1229644. Any other relevant data supporting the findings of this study are available from the corresponding author on request. Balaban, N. Q. et al. Definitions and guidelines for research on antibiotic persistence. Treatment of staphylococcal infections with penicillin by intermittent sterilisation. Balaban, N. Q., Merrin, J., Chait, R., Kowalik, L. & Leibler, S. Bacterial persistence as a phenotypic switch. Fung, D. K., Chan, E. W., Chin, M. L. & Chan, R. C. Delineation of a bacterial starvation stress response network which can mediate antibiotic tolerance development. Kaspy, I. et al. HipA-mediated antibiotic persistence via phosphorylation of the glutamyl-tRNA-synthetase. Active starvation responses mediate antibiotic tolerance in biofilms and nutrient-limited bacteria. & Hill, T. M. Characterization of the hipA7 allele of Escherichia coli and evidence that high persistence is governed by (p)ppGpp synthesis. Helaine, S. et al. Internalization of Salmonella by macrophages induces formation of nonreplicating persisters. Pacios, O. et al. (p)ppGpp and its role in bacterial persistence: new challenges. Conlon, B. P. et al. Persister formation in Staphylococcus aureus is associated with ATP depletion. Slow growth determines nonheritable antibiotic resistance in Salmonella enterica. Rotem, E. et al. Regulation of phenotypic variability by a threshold-based mechanism underlies bacterial persistence. Harms, A., Fino, C., Sorensen, M. A., Semsey, S. & Gerdes, K. Prophages and growth dynamics confound experimental results with antibiotic-tolerant persister cells. & Levin, B. R. Pharmacodynamics, population dynamics, and the evolution of persistence in Staphylococcus aureus. The widely used antimicrobial triclosan induces high levels of antibiotic tolerance in vitro and reduces antibiotic efficacy up to 100-fold in vivo. El-Halfawy, O. M. & Valvano, M. A. Antimicrobial heteroresistance: an emerging field in need of clarity. Anderson, B. W., Fung, D. K. & Wang, J. D. Regulatory themes and variations by the stress-signaling nucleotide alarmones (p)ppGpp in bacteria. Atkinson, G. C., Tenson, T. & Hauryliuk, V. The RelA/SpoT homolog (RSH) superfamily: distribution and functional evolution of ppGpp synthetases and hydrolases across the tree of life. & Wood, T. K. Arrested protein synthesis increases persister-like cell formation. Fung, D. K., Yang, J., Stevenson, D. M., Amador-Noguez, D. & Wang, J. D. Small alarmone synthetase SasA expression leads to concomitant accumulation of pGpp, ppApp, and AppppA in Bacillus subtilis. Roghanian, M. et al. (p)ppGpp controls stringent factors by exploiting antagonistic allosteric coupling between catalytic domains. Catalytic mechanism and allosteric regulation of an oligomeric (p)ppGpp synthetase by an alarmone. Potrykus, K. & Cashel, M. (p)ppGpp: still magical. Anderson, B. W., Hao, A., Satyshur, K. A., Keck, J. L. & Wang, J. D. Molecular mechanism of regulation of the purine salvage enzyme XPRT by the alarmones pppGpp, ppGpp, and pGpp. Direct regulation of GTP homeostasis by (p)ppGpp: a critical component of viability and stress resistance. Wang, B. et al. Affinity-based capture and identification of protein effectors of the growth regulator ppGpp. The nucleotide messenger (p)ppGpp is an anti-inducer of the purine synthesis transcription regulator PurR in Bacillus. Wang, J. D., Sanders, G. M. & Grossman, A. D. Nutritional control of elongation of DNA replication by (p)ppGpp. & Wang, J. D. The alarmone (p)ppGpp regulates primer extension by bacterial primase. & Gründling, A. ppGpp negatively impacts ribosome assembly affecting growth and antimicrobial tolerance in Gram-positive bacteria. The nucleotide pGpp acts as a third alarmone in Bacillus, with functions distinct from those of (p)ppGpp. Irving, S. E., Choudhury, N. R. & Corrigan, R. M. The stringent response and physiological roles of (pp)pGpp in bacteria. Diez, S., Ryu, J., Caban, K., Gonzalez, R. L. Jr. & Dworkin, J. The alarmones (p)ppGpp directly regulate translation initiation during entry into quiescence. & Wang, J. D. Lowering GTP level increases survival of amino acid starvation but slows growth rate for Bacillus subtilis cells lacking (p)ppGpp. Kriel, A. et al. GTP dysregulation in Bacillus subtilis cells lacking (p)ppGpp results in phenotypic amino acid auxotrophy and failure to adapt to nutrient downshift and regulate biosynthesis genes. Handke, L. D., Shivers, R. P. & Sonenshein, A. L. Interaction of Bacillus subtilis CodY with GTP. Lewis, K. Multidrug tolerance of biofilms and persister cells. Liu, K. et al. Molecular mechanism and evolution of guanylate kinase regulation by (p)ppGpp. Libby, E. A., Reuveni, S. & Dworkin, J. Multisite phosphorylation drives phenotypic variation in (p)ppGpp synthetase-dependent antibiotic tolerance. D'Elia, M. A. et al. Probing teichoic acid genetics with bioactive molecules reveals new interactions among diverse processes in bacterial cell wall biogenesis. Veening, J. W., Smits, W. K. & Kuipers, O. P. Bistability, epigenetics, and bet-hedging in bacteria. Miller, C. et al. SOS response induction by β-lactams and bacterial defense against antibiotic lethality. Eiamphungporn, W. & Helmann, J. D. The Bacillus subtilis σM regulon and its contribution to cell envelope stress responses. Katz, E. & Demain, A. L. The peptide antibiotics of Bacillus: chemistry, biogenesis, and possible functions. Geiger, T., Kastle, B., Gratani, F. L., Goerke, C. & Wolz, C. Two small (p)ppGpp synthases in Staphylococcus aureus mediate tolerance against cell envelope stress conditions. Lazar, V., Snitser, O., Barkan, D. & Kishony, R. Antibiotic combinations reduce Staphylococcus aureus clearance. Janes, B. K. & Stibitz, S. Routine markerless gene replacement in Bacillus anthracis. Koo, B. M. et al. Construction and analysis of two genome-scale deletion libraries for Bacillus subtilis. Rapid and reliable DNA assembly via ligase cycling reaction. Griffith, K. L. & Grossman, A. D. Inducible protein degradation in Bacillus subtilis using heterologous peptide tags and adaptor proteins to target substrates to the protease ClpXP. Harwood, C. R. & Cutting, S. M. Molecular Biological Methods for Bacillus (Wiley, 1990). Adimpong, D. B. et al. Antimicrobial susceptibility of Bacillus strains isolated from primary starters for African traditional bread production and characterization of the bacitracin operon and bacitracin biosynthesis. Clasquin, M. F., Melamud, E. & Rabinowitz, J. D. LC-MS data processing with MAVEN: a metabolomic analysis and visualization engine. Measuring single-cell gene expression dynamics in bacteria using fluorescence time-lapse microscopy. Johnson, C. M. & Grossman, A. D. Identification of host genes that affect acquisition of an integrative and conjugative element in Bacillus subtilis. van Opijnen, T., Lazinski, D. W. & Camilli, A. Genome-wide fitness and genetic interactions determined by Tn-seq, a high-throughput massively parallel sequencing method for microorganisms. This study is supported by the National Institute of Health R35 GM127088, Howard Hughes Medical Institute Emerging Pathogens Initiative, a USDA Hatch Formula Award (to J.D.W. ), Louis and Elsa Thomsen Wisconsin Distinguished Graduate Fellowships (to J.Y.) and National Science Foundation (NSF) grant award no. These authors contributed equally: Danny K. Fung, Jessica T. Barra. Danny K. Fung, Jessica T. Barra, Jin Yang, Jeremy W. Schroeder, Fukang She, Megan Young, David Ying, David M. Stevenson, Daniel Amador-Noguez & Jue D. Wang You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar The authors declare no competing interests. Nature Microbiology thanks Tanel Tenson and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Measuring tolerance from kill curve. The biphasic killing curve is composed of two phases: an initial killing phase (red) where the non-persister population dies rapidly upon antibiotic treatment, followed by an antibiotic-refractory phase (black) where persisters survive prolonged treatment. To measure tolerance of the bulk population, we subtracted the persister fraction from the population and estimated its MDK99 (minimum duration of killing for 99% cells) from the killing phase. (b-c) Tolerance (b) and persistence (c) of exponentially growing WT or (p)ppGpp0 mutant after 5 h vancomycin treatment. p values were derived from unpaired two-tailed t-test. (a) Vancomycin survival curves of WT cells after starvation or (p)ppGpp induction. stationary: stationary phase, RHX: pre-treated with amino acid starvation inducer arginine hydroxamate, +CCCP: pre-treated with ATP synthesis inhibitor CCCP, +As: pre-treated with ATP synthesis inhibitor arsenate. RHX: pre-treated with amino acid starvation inducer arginine hydroxamate, +CCCP: pre-treated with ATP synthesis inhibitor CCCP, p-sasA or p-sasASyn: (p)ppGpp0 strains overexpressing the (p)ppGpp synthetase SasA or its synthetase-dead variant SasASyn (SasAD87G). See also Supplemental Data Table 5 for a summary of resistance, tolerance and persistence in starvation-induced wild type and (p)ppGpp0 mutant. Transposon mutant library was grown to log phase followed by treatment with Vancomycin or Ciprofloxacin. Populations before and after treatment were sampled for sequencing library preparation and deep sequencing. In this assay, only insertions in non-essential or conditionally essential genes (due to availability of casamino acids) will be selected. (b) Genome-wide insertion profile from the experiment in (a), red box indicates enrichment of the pur operon disruptions after antibiotic treatment. (c) Pathway of de novo purine and GTP biosynthesis. Enriched gene disruptions in the pur operon and guaA from the Tn-Seq library screen were highlighted in cyan. Numbers in parentheses indicate relative enrichment of disruptions in log2 (vs before Vancomycin treatment). Overall, the insertions before the IMP step would all lower GTP and ATP synthesis, while guaA insertion would lower GTP but not ATP synthesis. guaB and gmk have been reported as essential genes and were not present in the library. Nucleotide levels were measured by thin layer chromatography (TLC) and normalized to OD600. Values represent mean and error bars indicate s.d. p values were derived from unpaired two-tailed t-test between WT and mutant pairs. (e) Vancomycin survival curves of WT, ∆purB, ∆purL, ∆purF, and ∆guaA. Importantly, guaA deletion which only lower GTP but not ATP showed the same increase in persistence as the pur mutants. This implies that GTP depletion is critical to persistence. (a-b) Vancomycin survival curves of WT, gmkQ110R,(p)ppGpp0, (p)ppGpp0 gmkQ110R, and guaB repressed mutant (guaBDown). (c-d) GTP levels (c) and persister levels (d) of WT, gmkQ110R,(p)ppGpp0, (p)ppGpp0 gmkQ110R, and guaB repressed mutant (guaBDown). Nucleotide levels were measured by thin layer chromatography (TLC) and normalized to OD600. p values were derived from unpaired two-tailed t-test. (a) Wild type cells containing the PlowGTP reporter were grown in minimal media with different carbon sources as indicated. Each curve represents a single replicate. (b) Survival curves of the populations in (a) to vancomycin treatment. (c) Distribution of cells with high PlowGTP fluorescence analysed using flow cytometry ( ~ 1.5×106 cells per sample per replicate, three biological replicates). Numbers indicate the fraction (%) of GFP-high cells. The fluorescence cut-off for GFP high cells was selected based on FACS and antibiotic treatment experiment that these high fluorescence cells display persistence phenotype (see Fig. (d) Panel of growth rates, tolerance, population PlowGTP fluorescence, frequency of high PlowGTP-GFP-high cells, and persistence to vancomycin treatment from the data in a-c. Values represent mean ± s.d., three biological replicates. Low GTP inactivates CodY binding thus de-repressing the promoter (that is reporter ON) and vice versa. It has been demonstrated that ilvB promoter activity is proportional to GTP over a physiologically relevant range in vitro38. Our previous transcriptome work showed that the ilvB promoter is responding almost exclusively to GTP levels in a variety of conditions and mutants in vivo37. Sequence of PlowGTP is included in Supplemental Table 4. (b) PlowGTP fluorescence in wild type or (p)ppGpp0 population during exponential growth in rich media with or without (p)ppGpp induction using arginine hydroxamate (RHX). Background fluorescence was subtracted using strains without the reporter. (c-d) Correlation of PlowGTP fluorescence versus (p)ppGpp (blue) and GTP (red) in wild type upon (p)ppGpp induction by amino acids starvation. PlowGTP fluorescence was determined using microscopy from averages of > 200 cells each from three biological replicates. (c) Nucleotide levels were measured by thin layer chromatography (TLC) and normalized to their levels before induction. R square was derived from linear regression from three biological replicates for each data point. (d) Nucleotide levels were measured by LC-MS and normalized to OD600. Values represent mean and error bars indicate s.d. from three biological replicates for each data point. (a) Flow cytometry analysis of low GTP persisters in exponentially growing populations of wild-type and (p)ppGpp biosynthesis mutants containing the PlowGTP reporter ( ~ 1.5×106 cells each in triplicate). Numbers indicate the fraction (%) of GFP-high cells. The fluorescence cut-off was chosen based on FACS and antibiotic treatment experiment that these high fluorescence cells display persistence phenotype (see Fig. (b) Correlation between persisters and GFP-high cells. Persister levels were determined from vancomycin kill curves in Supplemental Data Fig 4. Error bars indicate s.e.m., three biological replicates. (a-l) Single cell traces of spontaneous persistence entrance plotted as growth rate against time in (a-i) wild type and (j-l) (p)ppGpp0 gmkQ110R mutant. Changes in PlowGTP reporter fluorescence (green) and growth rate (black) are shown. Persistence entrance in each trace is highlighted by the shaded area. Overall, we observed that, the first event in pre-persister is that PlowGTP reporter fluorescence (reflecting drop in GTP) increases, sometimes over several cell cycle (up to 120 min) while cells are growing at a normal rate without slowing down. Only as the fluorescence reaches a common threshold (on our microscope ~600 fluorescence unit), growth suddenly stopped within our 15 min observation window. If bacteria stop growth first, then we would see fluorescence increase only after growth cessation. This conclusion applies to all our cell trajectory measurements. The results suggest that GTP drop occurs before growth cessation, rather than GTP drops once bacteria stop growing. (a) (Left) Example images of wild type cells containing the PlowGTP reporter before and after treatment with 200x MIC carbenicillin (CARB) and cell death dye propidium iodide (PI, magenta) for 2 h. (Right) Quantitation of PlowGTP fluorescence in survivors (PI-negative cells) before and after treatment. (b-c) Representative time-lapse images from three biological replicates of antibiotic-induced survival upon antibiotic treatment. Exponentially growing wild type cells containing the PlowGTP-GFP reporter (green) were allowed to grow in the absence of antibiotic followed by carbenicillin treatment with the cell death dye propidium iodide (PI, magenta). Antibiotic was removed by penicillinase treatment. Arrow indicates a cell that regrew after drug removal. (c) Same experiment as (b) but with PlowGTP-mCherry reporter (red) and cell death dye sytox blue (cyan). How does low GTP lead to growth suspension and persistence? Although reduction of GTP levels elicits a fundamental change in transcription, the effect of depleting GTP is far beyond transcription. Low GTP is sufficient to broadly suspend diverse growth-determining cellular processes that utilize GTP, including DNA replication, rRNA synthesis, translation initiation and elongation, ribosome assembly, 100S ribosome dissociation, and potentially cell wall synthesis. In addition, since the majority of (p)ppGpp regulation on targets such as protein synthesis, rRNA maturation via ribosome-dependent GTPases, or DNA synthesis all stem from competition between (p)ppGpp and GTP binding to the enzyme active site, low GTP would exacerbate the inhibitory effect of (p)ppGpp on most (p)ppGpp targets. Since lethality of conventional antibiotics stems from deleterious disruption of these growth-associated processes, low GTP bacteria will be able to better survive antibiotic treatment. Therefore, persistence can be due to a global metabolic brake of multiple cellular processes triggered by low GTP. (p)ppGpp-GTP may also facilitate coordinated shut down of macromolecular synthesis, such as transcription and replication, to minimize detrimental conflicts. Finally, the growth suspension may also reduce generation of reactive oxygen species which have been implicated in antibiotic lethality. Contributions of these downstream effects require further investigation. Spontaneous persistence entrance in wild-type B. subtilis. Spontaneous persistence entrance in (p)ppGpp0 gmkQ110R mutant. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Fung, D.K., Barra, J.T., Yang, J. et al. A shared alarmone–GTP switch controls persister formation in bacteria. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02701-7'>Ultrafast and accurate sequence alignment and clustering of viral genomes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 09:55:28
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Viromics produces millions of viral genomes and fragments annually, overwhelming traditional sequence comparison methods. Here we introduce Vclust, an approach that determines average nucleotide identity by Lempel–Ziv parsing and clusters viral genomes with thresholds endorsed by authoritative viral genomics and taxonomy consortia. Vclust demonstrates superior accuracy and efficiency compared to existing tools, clustering millions of genomes in a few hours on a mid-range workstation. Metagenomics and viromics are identifying new viruses at an unprecedented rate, but recognizing which sequences were seen before remains challenging1,2. Calculating average nucleotide identity (ANI), essential for classification, is limited by the scalability of alignment tools like anicalc3, commonly used to cluster viruses into virus operational taxonomic units (vOTUs), or VIRIDIC4, recommended by the International Committee on Taxonomy of Viruses (ICTV) to delineate bacteriophage species and genera. Large-scale sequence comparisons rely on efficient, albeit less accurate, k-mer approaches such as sketching (FastANI5) or sparse approximate alignments (skani6). Moreover, most tools lack clustering functionality or do not scale to large metagenomic datasets (Extended Data Table 1). Vclust is a fast alignment-based method that calculates ANI measures for complete and fragmented viral genomes and clusters them according to ICTV and Minimum Information about an Uncultivated Virus Genome (MIUViG) standards1,4 (Extended Data Table 1). First, Kmer-db 2, a successor of Kmer-db7, rapidly determines related genomes using either all k-mers or a predefined fraction. Second, LZ-ANI, a Lempel–Ziv parsing-based algorithm (Fig. Third, Clusty efficiently implements six clustering algorithms suited for sparse distance matrices with millions of genomes (Fig. a, Vclust's workflow: (1) prefilter similar genome sequence pairs with sufficient k-mer-based identity estimated using Kmer-db 2; (2) align similar genome pairs and calculate ANI using LZ-ANI; and (3) cluster genomes based on defined cutoffs using Clusty. b, Sequence alignment using Lempel–Ziv parsing (Methods). Vertex size represents genome sequence length, and edge lengths indicate the distance (1 − ANI) between genomes. A more detailed depiction of the clustering algorithms is shown in Extended Data Fig. We first tested Vclust's accuracy of total average nucleotide identity (tANI) estimation (Fig. 1d) among 10,000 pairs of phage genomes containing simulated mutations, including substitutions, deletions, insertions, inversions, duplications and translocations (Methods and Supplementary Table 1). Vclust and VIRIDIC, both alignment-based tools, provided tANI values close to the expected ones, with mean absolute error (MAE) values of 0.3% and 0.7%, respectively, outperforming FastANI (6.8%) and skani (21.2%; Fig. Among genome pairs above the ICTV's species threshold (tANI ≥ 95%4, n = 1,188), Vclust reported only 22 pairs below the threshold, whereas VIRIDIC underestimated nearly 10× more (n = 210; Supplementary Table 2). a, Difference between predicted and expected tANI values for 10,000 bacteriophage genome pairs with simulated mutation events. b, Correlations with VIRIDIC tANI values for 22,607 complete bacteriophage genome pairs. c, Wall time and peak memory usage for processing 4,244 bacteriophage genomes (32 threads). d, Venn diagrams comparing numbers of contig pairs meeting MIUViG thresholds (ANI ≥ 95% and AF ≥ 85%) predicted by BLASTn (purple) and other tools (red). The boxen plot shows the error distribution of predicted ANI and AF values relative to corresponding BLASTn-based reference values for 4,361,743 contig pairs meeting MiUVUG thresholds. e, Wall time and peak memory usage for calculating ANI and AF among 15,677,623 IMG/VR contigs (64 threads). BLASTn values were estimated from a random sample of 1,000 query contigs. Vclust was tested in its default setting and with a 0.2 fraction of k-mers used at the ‘prefilter' step. f, Wall time and peak memory usage of Vclust's clustering algorithms for grouping IMG/VR contigs into vOTUs. Next, we determined tANI using VIRIDIC in an all-to-all comparison of 4,244 bacteriophage genomes. Vclust had a higher correlation with VIRIDIC tANI (Pearson's r = 0.983) than skani (r = 0.902) and FastANI (r = 0.671) across the entire tANI range ≥ 70% (22,606 genome pairs; Fig. 2b) and outperformed both tools within their reliability range ≥ 80%5,6. Upon examining genome pairs where both Vclust and VIRIDIC diverged from the ICTV's classification, we found inconsistencies in 50 ICTV taxonomic proposals (Supplementary Tables 3 and 4). Excluding these cases improved the agreement of both tools with ICTV taxonomy, with Vclust retaining superiority (95%) over VIRIDIC (90%) and the other tools (Supplementary Table 5). For genus groupings (tANI ≥ 70%), Vclust achieves 92% agreement with ICTV taxonomy, comparable to VIRIDIC's 93%, despite inconsistent application of the threshold we found across ICTV genera (Supplementary Tables 6 and 7 and Extended Data Fig. Given Vclust's high agreement with ICTV taxonomy, accurate tANI determination and processing speed >40,000× faster than VIRIDIC (Fig. 2c and Supplementary Table 8), it emerges as the prime tool for bacteriophage classification. We then assessed Vclust's accuracy in matching contig pairs that satisfy MIUViG thresholds (ANI ≥ 95% and aligned fraction (AF) ≥ 85%; Fig. We subsampled over 90,000 metagenomic contigs from the IMG/VR database and used BLASTn8+ anicalc3 (most accurate alignment-based method) to identify over 4 million sequence pairs that met MIUViG thresholds. Vclust recovered the highest number of pairs (99%), followed by MegaBLAST + anicalc (97%), skani (96%, or 86% in the fastest mode), FastANI (96%) and MMseqs2 (ref. Both Vclust and MegaBLAST produced ANI and AF estimates consistently with the BLASTn values (Pearson r > 0.96), outperforming the other tools (r = 0.2–0.8). On average, ANI and AF values obtained by Vclust and MegaBLAST showed minimal deviation from the expected values (MAE < 1%; Supplementary Table 9), with Vclust having the narrowest error range among all the tools (Fig. This trend is consistent across varying contig sizes, from smallest (<5 kb) to largest (>100 kb; Supplementary Table 10). Vclust performed sequence identity estimations for ~123 trillion contig pairs and alignments for ~800 million pairs, resulting in 5–8 million vOTUs depending on the clustering algorithm (Supplementary Table 11 and Supplementary Fig. These vOTUs are generally consistent with those identified by MegaBLAST, with Vclust clustering approximately 75,000 more contigs on average, indicating higher sensitivity (Supplementary Table 11). Although skani in its fastest mode was 7× faster than Vclust (Supplementary Table 12), it was substantially less accurate (Supplementary Table 9). In addition, Vclust's runtime and memory usage can be further reduced by ~40% and ~60%, respectively, by analyzing 20% of the k-mers in each genome during prefiltering (Fig. 2e), with negligible impact on sensitivity and specificity (Extended Data Fig. It provides a complete solution for calculating intergenomic similarities and clustering complete, partial and circularly permuted (Extended Data Fig. 5) virus genomes using various ANI measures and clustering algorithms. Given the astonishing diversity of viruses in metagenomic data, we believe that Vclust will be essential for large-scale dereplication and taxonomic classification of viral sequences. It is freely available on GitHub, with a web service option for smaller projects (https://www.vclust.org/), and its core components—Kmer-db, LZ-ANI and Clusty—are available as stand-alone tools for broader applications in sequence comparison and general clustering tasks. Similar to other tools6, Vclust's performance may decrease with large datasets of highly similar genomes owing to the high number of sequence pairs requiring alignment and clustering after prefiltering (Methods). Future work will focus on improving scalability for large homogeneous datasets, including bacterial genomes, and implementing amino acid-based computations (for example, average amino acid identity). Vclust is a workflow that introduces and integrates three tools: Kmer-db 2: performs the initial k-mer-based estimation of sequence identity of all genome pairs (‘Sequence identity estimation: Kmer-db 2'). We implemented Kmer-db 2, LZ-ANI and Clusty in C++20 as stand-alone tools, adaptable for various sequence comparison and clustering tasks (‘Code availability'). Vclust, a Python script, integrates these tools to calculate and cluster viral genomic sequences (‘Vclust implementation'). Kmer-db 2 is an updated tool for k-mer-based estimation of pairwise similarities among nucleotide sequences, using either all or a selected fraction of k-mers. Unlike fixed-sized sketching (used, for example, by Mash10), Kmer-db 2 retains a proportional fraction of k-mers per genome, preserving the relationship between sequence lengths. First, unlike its predecessor, which stored similarity values in RAM as a dense matrix7, Kmer-db 2 uses sparse matrices that retain only nonzero elements in all-to-all pairwise genome comparison mode (‘all2all-sp'), allowing it to handle large and diverse genome sets. Second, Kmer-db 2 supports genome datasets partitioned into multiple input files, each generating a separate Kmer-db database. A new mode, ‘all2all-parts', calculates shared k-mers within and across databases, optimizing memory by loading one or two databases into RAM sequentially, although at the expense of additional computational time from repeated database loading. Third, Kmer-db 2 further minimizes RAM usage by storing only genome pairs that meet a minimum threshold of shared k-mers and sequence identity. Finally, all modes in Kmer-db 2 support multithreading, except for the distance calculation step, which is sufficiently fast without parallelization. 2 shows the computational performance improvements of Kmer-db 2 over Kmer-db 1, with runtime reductions of 3× to 100× across modes and substantially lower RAM requirements. First, the algorithm constructs two indices (dictionaries): for anchors and seeds. Upon finding an exact match, the algorithm extends it in both directions (Fig. Then, the extensions of terminal windows are trimmed to remove poorly aligned ends until they have at least ar exactly matched nucleotides. This extended anchor initiates the first ‘region', which corresponds to a local alignment, and is constructed as described below. No anchor or seed is found: shift by one position in the query and repeat the process of finding a new anchor or a seed match. Only a seed match is found: extend the seed similarly to the initial anchor match, append it to the region, and continue the search for a new anchor or seed match (Fig. Only an anchor match is found: close the current region and extend the anchor match to initiate a new region (Fig. Both anchor and seed matches are found: select the match less likely to occur by chance, based on their lengths, seed proximity (r nucleotides) and the reference sequence length, leading to either scenario 2 or 3 (Fig. This realignment aims to maximize the number of matching nucleotides between neighboring extended matches by allowing a single multi-symbol insertion in the reference or query sequence. As a result, the region represents a local alignment containing both matched and mismatched nucleotides, along with approximated indel fragments. To remove spurious alignments, regions shorter than g nucleotides are excluded from further analysis. The LZ-ANI tool reads input sequences and stores them in RAM in a compact format with three nucleotides per byte. The tool processes sequences in parallel, with each thread comparing a reference sequence to all other sequences. LZ-ANI parameters are adjustable and were optimized for virus genome sequences (Extended Data Table 2). The default anchor length was set to 11 nucleotides, matching the BLASTn default word size, which provides greater sensitivity than MegaBLAST's 28-nucleotide word size. The remaining LZ-ANI parameters were optimized using Bayesian optimization with Gaussian process minimization. This optimization involved 100 evaluations on a dataset of 10,000 pairs of complete genomes with simulated mutations (that is, substitutions, insertions, deletions, duplications, inversions and translocations) and known expected ANI values of ≥70% (Supplementary Table 13). 3 compares the length, number and identity of alignments generated by Vclust (using default parameters), BLASTn and MegaBLAST. Similarly to BLAST-based ANI methods3,4, LZ-ANI alignment between query (A) and reference (B) encompasses ‘regions', analogous to BLAST's high-scoring segment pairs. This alignment allows direct calculation of: L(A, B)—the total length (sum) of all regions when aligning query A to reference B, in nucleotides M(A, B)—the total number of matching nucleotides in all regions These values are used to compute seven sequence similarity measures as follows: Clusty is a versatile package facilitating rapid clustering across diverse data types, using six algorithms: single linkage, complete linkage, UCLUST12, greedy set cover9, CD-HIT13 and Leiden14. Our implementations of these algorithms were optimized for sparse distance matrices. Clusty uses threshold-based clustering, assigning an object to a cluster if its distance from the cluster does not exceed a user-defined threshold. Depending on the algorithm, this distance can refer to the closest, furthest or centroid member. While UCLUST, greedy set cover and CD-HIT are inherently threshold-based algorithms, single and complete linkage algorithms construct dendrograms that can be pruned at customizable distance thresholds. Clusty's sparse data representation assumes all input values to meet the distance or similarity threshold. However, the tool allows clustering data at more stringent thresholds through additional filtering of any combinations of distance/similarity values (for example, tANI, ANI and AF) and/or other measure values (for example, minimum/maximum number of alignments, minimum/maximum number of matched nucleotides). Consequently, the matrix provided to Clusty does not need to be sparse; the tool can handle dense matrices and apply filtering at the loading stage. Clusty interprets input data as a graph, where vertices represent objects and edges represent connections. 1 shows details of the clustering algorithms and their time complexities. Vclust is a Python tool integrating Kmer-db 2, LZ-ANI and Clusty for streamlined computation of intergenomic sequence similarities and clustering of viral genomes. Vclust provides three commands: ‘prefilter', ‘align' and ‘cluster' (Fig. ‘prefilter' and ‘align' accept a single FASTA file containing viral genomic sequences or a directory of FASTA files (one genome per file), with support for gzipped inputs and outputs. The ‘prefilter' command uses Kmer-db 2 to screen out dissimilar genome pairs before alignment, reducing the number of genome pairs to only those with sufficient k-mer-based sequence similarity (that is, minimum number of common k-mers and/or the minimum sequence identity. Sequence identity in Kmer-db 2 is calculated similarly to ANI in Mash (1 − Mash distance) but uses the overlap coefficient15 instead of the Jaccard index. As a result, sequence identity values in the prefiltering step are generally higher than ANI from the alignment step. This allows users to set the minimum sequence identity in prefiltering close to the final ANI threshold without risking the exclusion of relevant genome pairs; for example, if targeting an ANI threshold of 95% or higher, the minimum sequence identity can be set to approximately 0.95 (Supplementary Fig. The ‘align' command uses LZ-ANI to perform pairwise sequence alignments and compute ANI and AF measures between genome pairs identified by the pre-alignment filter. If the filter is not provided, Vclust aligns all possible genome pairs. The output includes two TSV files that are used for clustering: one containing ANI measures for genome pairs and the other listing genome identifiers sorted by decreasing sequence length. Optionally, Vclust can output detailed alignment results in a TSV format similar to BLASTn/MegaBLAST, with coordinates, strand orientation, matched/mismatched nucleotides and sequence identity for each local alignment. The ‘cluster' command uses Clusty for genome clustering, allowing users to specify a similarity measure (for example, tANI, ANI) and its threshold for clustering genomes, with optional additional filtering thresholds for other similarity measures, including AF. Alternatively, Vclust can output representative genomes instead of numerical cluster identifiers, which is particularly useful for dereplication tasks. Vclust is designed for dereplication and clustering of viral sequences across a range of identity values. Computational performance may decline with datasets of highly redundant genome sequences (for example, tens of thousands of sequences from the same species; Supplementary Fig. In all-versus-all pairwise genome comparisons in the ‘prefilter' step, the high frequency of similar sequences expands the similarity matrix, increasing memory consumption and the number of pairs to align, which in turn raises computational demands for alignment and clustering. Vclust has three additional techniques to optimize performance and mitigate excessive resource consumption. First, it partitions a dataset into smaller, equally sized batches of genome sequences using the built-in multi-fasta-split C++ tool. This option considerably reduces memory requirements of the ‘prefilter' step without altering results, although it may slightly increase runtime (Extended Data Fig. Second, Vclust can limit the number of k-mers analyzed from each genome sequence, reducing memory usage and runtime with minimal impact on sensitivity (Extended Data Fig. Third, similarly to MMseqs2 and BLAST-based methods, Vclust's ‘prefilter' can restrict the number of sequences reported per query genome by selecting those with the highest sequence identity, reducing the overall number of genome pairs passing initial similarity assessment. All runtimes were benchmarked on a workstation equipped with an AMD Epyc 9554 CPU (64 cores clocked at 3.1 GHz) and 1,152 GiB (approximately 1,237 GB) RAM. Unless otherwise specified, all tools were run using 64 threads. The exact commands are shown in Supplementary Tables 8, 9 and 12. The tANI accuracy of Vclust v1.2.8, FastANI (v1.33)5, skani (v0.2.1)6 and VIRIDIC (v1.1)4 was assessed using two reference sets. In both reference datasets, VIRIDIC was run with default parameters (--word_size 7, --reward 2, --penalty 3, --gapopen 5, --gapextend 2) for highly sensitive BLASTn alignments. Similarly, skani was run in its most accurate mode optimized for small sequences (--slow, --s 0, --m 200). FastANI and Vclust were run with default parameters. The first reference dataset comprised 22,606 tANI values ranging from 70% to 100%, as determined by VIRIDIC across 4,244 complete genomes of bacteriophages affiliated with the ICTV using ICTV's Virus Metadata Resource (VMR v38.3). Since FastANI and skani do not directly report tANI, their values were calculated from ANI, AF and genome lengths: tANI = (ANI1 × AF1 × LEN1 + ANI2 × AF2 × LEN2)/(LEN1 + LEN2). The second reference set contained expected (true) tANI values in the 70–100% range, derived from 10,000 pairs of bacteriophage genomes subjected to simulated mutations, including different levels of substitution, insertion, deletion, duplication, inversion and translocation events. Specifically, we randomly selected 100 genomes from the bacteriophage dataset and generated 100 copies of each genome. For each genome copy, we introduced mutations using Mutation-Simulator (v3.0.2)16 by randomly selecting a combination of mutation events and their corresponding frequencies (Supplementary Table 1). The expected (true) tANI value between each copy and reference genome was determined based on the variant call format produced by Mutation-Simulator, describing the exact locations of introduced mutations and the number of altered nucleotides. 9) were compared to reference ANI and AF values determined by BLASTn (v2.13.0+)8. Since running BLASTn on the entire IMG/VR v4.1 database was not feasible, we subsampled 94,225 viral contigs and performed an all-to-all BLASTn search to identify 4,361,743 contig pairs meeting the MIUViG thresholds (ANI ≥ 95% and AF ≥ 85%). MegaBLAST, MMseqs and BLASTn outputs were used by the anicalc script from CheckV (v1.0.3)3 to compute ANI and AF values. Given the high level of sequence identity of the reference contig pairs, if a tool did not return a result for a given contig pair, the ANI and AF values were set to zero for that pair. ARI assesses clustering similarity by comparing the number of correct clustering overlaps and disagreements17 against those expected by chance. An ARI of 0 indicates random assignment, while a score of 1 indicates a perfect match. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The datasets generated in this study have been deposited in in Figshare (https://doi.org/10.6084/m9.figshare.28294805)19 and include complete RefSeq and GenBank genomes of 4,244 bacteriophages classified by ICTV, RefSeq and GenBank genome sequences of 10,000 bacteriophages with simulated mutations and corresponding expected total ANI values, and 94,225 metagenomic viral contigs sampled from IMG/VR v4.1 with expected BLASTn-based ANI and AF values. Other databases used in the study include IMG/VR v.4.1 (https://genome.jgi.doe.gov/portal/IMG_VR/) and Virus Metadata Resource v38.3 from ICTV (https://ictv.global/vmr/). Source data are provided with this paper. 20) and as a web service at https://www.vclust.org/. Minimum Information about an Uncultivated Virus Genome (MIUViG). Camargo, A. P. et al. IMG/VR v4: an expanded database of uncultivated virus genomes within a framework of extensive functional, taxonomic, and ecological metadata. Nayfach, S. et al. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. & Kropinski, A. M. VIRIDIC-a novel tool to calculate the intergenomic similarities of prokaryote-infecting viruses. Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. & Aluru, S. High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. & Yu, Y. W. Fast and robust metagenomic sequence comparison through sparse chaining with skani. Deorowicz, S., Gudys, A., Dlugosz, M., Kokot, M. & Danek, A. Kmer-db: instant evolutionary distance estimation. Altschul, S. F. et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Steinegger, M. & Söding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Salomon, D. & Motta, G. Handbook of Data Compression (Springer, 2009). Edgar, R. C. Search and clustering orders of magnitude faster than BLAST. Li, W. & Godzik, A. Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. From Louvain to Leiden: guaranteeing well-connected communities. McGill, M., Koll, M. & Noreault, T. An Evaluation of Factors Affecting Document Ranking by Information Retrieval Systems (School of Information Studies, Syracuse University, 1979). & Ries, D. C. Mutation-Simulator: fine-grained simulation of random mutations in any genome. Rand, W. M. Objective criteria for the evaluation of clustering methods. Pedregosa, F. et al. Scikit-learn: machine learning in Python. & Gudyś, A. Vclust: source code and precompiled binaries (1.2.8). This work is supported by the National Science Centre, Poland, project DEC-2022/45/B/ST6/03032 (to A.G. and S.D. ), the European Research Council (ERC) Consolidator grant 865694: DiversiPHI, the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy—EXC 2051—Project-ID 39071386 (to B.E.D. ), the European Union's Horizon 2020 research and innovation program, under the Marie Skłodowska-Curie Actions Innovative Training Networks grant agreement no. ), the Alexander von Humboldt Foundation in the context of an Alexander von Humboldt-Professorship founded by German Federal Ministry of Education and Research (to B.E.D. and the Polish Ministry of Science and Higher Education under the program ‘Perły Nauki', project number PN/01/0063/2022 (to P.R.). The computations were partially performed at the Poznan Supercomputing and Networking Center (grant numbers pl0243-01 and pl0074-02). Open access funding provided by Friedrich-Schiller-Universität Jena. These authors contributed equally: Andrzej Zielezinski, Adam Gudyś. Adam Gudyś, Krzysztof Siminski & Sebastian Deorowicz You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar A.G. designed and developed Clusty with input from K.S. wrote the manuscript with substantial contributions from B.E.D., J.B. and P.R. All authors reviewed and approved the manuscript. Correspondence to Bas E. Dutilh or Sebastian Deorowicz. Nature Methods thanks Eugene Koonin, Stephen Nayfach and Carson Miller for their contribution to the peer review of this work. Primary Handling Editor: Lin Tang, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The dataset is represented as a graph: objects (genomes) are shown as vertices with sizes proportional to user-assigned weights (for example, sequence length) and edges indicate distances (for example, 1 – tANI). In the consecutive steps of each algorithm objects are assigned to clusters marked with distinct colors. Distance values for this example are available in the source data provided with the paper. The subtree of Bastillevirinae branch from the maximum-likelihood tree based on a concatenated alignment of 10 marker proteins in Herelleviridae family. Leaves are labeled with species names and four species are represented by more than one genome (for example, multiple strains). Snippets of the total heatmap on the right highlight instances where Vclust clustering algorithms and/or ICTV disagree. Numbers in these boxes show tANI (only values higher than 60% - Vclust reliability threshold are shown). Despite recommended rank-demarcation criteria, the classification of viruses is based on multiple forms of evidence, ranging from calculating tANI between pairs to protein content and phylogenetic analyses. Therefore, inconsistencies between Vclust clustering and the official ICTV taxonomy may arise, especially if the tANI oscillates around the demarcation criterion, such as between Bastillevirus hoodyT and Bastillevirus evoli, where the tANI is 95.2%. Vclust was tested with all k-mers (default) and 0.2 fraction of k-mers retained at the prefilter step. The runtimes of all tools decrease linearly with up to eight threads, but beyond this, FastANI, MegaBLAST, and Vclust showed reduced speed-up while skani and MMseqs2 maintained linear scaling. It was followed by the default Vclust setting with the exception of 64 threads, where slow-down caused by the task fragmentation overhead at the prefilter step made it inferior by a small margin to less accurate MMseqs2. In the right panel, memory usage was constant across thread counts for all algorithms except FastANI, which showed a linear increase in memory consumption. a, Wall time and peak memory usage for prefiltering genomes in a varying number of equally-sized sequence batches. Increasing the number of batches slightly increased runtime, significantly reducing memory usage for both k-mer fractions. b, Wall time and peak memory usage for different combinations of k-mers fractions (1 to 0.001) and minimum shared k-mers (20 to 1) during prefiltering. Below each combination, the outputs of prefilter and align steps are compared to the default settings (k-mers fraction 1 and minimum shared k-mers 20). The numbers represent: contig pairs passing the prefilter step (≥95% sequence identity; blue), pairs passing the align step (ANI ≥ 95%, AF ≥ 85%) common to the pairs obtained using all k-mers (green), contig pairs missed compared to using all k-mers (red), and additional contig pairs detected at the specified k-mers fraction but missing compared to using all k-mers (magenta). For instance, a 0.2 k-mers fraction reduces time and memory usage threefold, with a minimal impact on sensitivity and specificity. Comparison of a, total average nucleotide identity (tANI) and b, aligned fraction (AF) values on a reference dataset of 4,244 complete bacteriophage genomes and the same dataset where two regions in each genome were swapped at a random breakpoint position. Both plots cover 40,618 genome pairs with tANI ≥ 70%. The expected (true) tANI values in the 70–100% range, derived from 10,000 pairs of bacteriophage genomes subjected to simulated mutations, including different levels of substitution, insertion, deletion, duplication, inversion and translocation events. Mutation-type frequencies and corresponding altered nucleotide counts are detailed in the table. True tANI values were determined among 1,188 genome pairs with simulated mutations, including substitutions, insertions, deletions, inversions and translocations (Methods). Vclust shows superior accuracy in determining tANI, with a maximum difference (error) from the true tANI of 0.6%, whereas VIRIDIC's maximum error in tANI values reaches up to 4.7%. List of bacteriophage genome pairs showing tANI ≥ 95% as predicted by both Vclust and VIRIDIC, yet classified into different species by the ICTV. For each such pair, the history of all involved taxa was reviewed using the ICTV Taxonomy Browser and taxonomic proposals were checked for adopted demarcation criteria and presented. List of bacteriophage genome pairs showing tANI ≥ 95% as predicted by both Vclust and VIRIDIC, yet classified into different species by the ICTV. For each such pair, the history of all involved taxa was reviewed using the ICTV Taxonomy Browser (https://ictv.global/taxonomy/taxondetails/) and taxonomic proposals were checked for adopted demarcation criteria and presented evidence of genome-genome similarity. List of bacteriophage sequence pairs showing tANI < 95% as predicted by both Vclust and VIRIDIC, yet classified into same species by the ICTV. The identity of such records was reviewed by manual inspection of the GenBank records and associated literature. This fragmented genome was flagged, as the sequences belong to the same species, but show almost no sequence similarity. Comparison of species-level agreement between Vclust and other tools with the ICTV taxonomy (Virus Metadata Resource v38.3) across six different clustering algorithms. Clustering for the evaluated tools was conducted using Clusty based on a tANI threshold of ≥95%. Agreement between the tool clusterings and ICTV species-level clusters was assessed using the ARI, where an ARI of 0 represents no agreement beyond random chance, and an ARI of 1 indicates identical clustering results. List of 614 ICTV bacteriophage genera containing more than one species or genome. For each genus, the mean and maximum tANI values were calculated using VIRIDIC, the recommended and most widely used tool for genus delineation in ICTV taxonomic proposals. Notably, in nearly 10% of genera (n = 57), no genome pair exceeds the genus threshold of 70% tANI, and an additional 8% of genera (n = 43) have a mean tANI below 70%. Wall time and peak memory usage for all-to-all comparisons on the 4,244 bacteriophage genomes in Fig. Comparison of ANI and AF values predicted by the tools in reference to the expected ANI and AF values obtained from BLASTn. The ANI and AF values were calculated among 94,225 viral metagenomic contigs subsampled from IMG/VR v4.1. Comparison of ANI and AF values predicted by the tools in reference to the expected ANI and AF values obtained from BLASTn. Clustering 15,677,623 virus contig sequences from IMG/VR v4.1 into vOTUs based on the MIUViG thresholds (ANI ≥ 95% and AF ≥ 85%). Additionally, comparisons were made with the original IMG/VR vOTU assignments (11th column). Notably, vOTU clusters based on ANI values from Vclust and MegaBLAST showed high ARI values across different clustering algorithms. Vclust with the ‘--kmers-fraction' option set to 0.2, using 20% of k-mers during prefiltering, has minimal impact on the number of clusters compared to the default setting, which uses all k-mers. vOTU clusters generated by each clustering algorithm were compared based on the distribution of intra-cluster and inter-cluster ANI values, as presented in Supplementary Fig. Wall time and peak memory usage for calculating ANI and AF among 15,677,623 IMG/VR contigs, with 64 threads. Vclust performed k-mer-based estimation of sequence identity for 122,893,931,465,064 contig pairs (in the prefilter step), and aligned 811,099,152 contig pairs (in the ANI calculation step). ANI and AF values from MegaBLAST and BLASTn outputs were calculated using the anicalc tool from CheckV. Vclust with the ‘--kmers-fraction' option set to 0.2, using 20% of k-mers during prefiltering, reduces runtime and RAM usage threefold compared to the default setting, which uses all k-mers. Selection of default LZ-ANI parameter values based on Bayesian optimization. Each row represents one of the 100 evaluation runs, with different sets of LZ-ANI parameter combinations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Zielezinski, A., Gudyś, A., Barylski, J. et al. Ultrafast and accurate sequence alignment and clustering of viral genomes. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: AI and Robotics newsletter — what matters in AI and robotics research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41587-025-02664-2'>Self-assembling protein nanoparticles for cytosolic delivery of nucleic acids and proteins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-15 09:37:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Biotechnology (2025)Cite this article Metrics details Intracellular delivery of biomacromolecules is hampered by low efficiency and cytotoxicity. Here we report the development of elastin-based nanoparticles for therapeutic delivery (ENTER), a recombinant elastin-like polypeptide (ELP)-based delivery system for effective cytosolic delivery of biomacromolecules in vitro and in vivo. Through iterative design, we developed fourth-generation ELPs fused to cationic endosomal escape peptides (EEPs) that self-assemble into pH-responsive micellar nanoparticles and enable cytosolic entry of cargo following endocytic uptake. In silico screening of α-helical peptide libraries led to the discovery of an EEP (EEP13) with 48% improved protein delivery efficiency versus a benchmark peptide. Our lead ELP–EEP13 showed similar or superior performance compared to lipid-based transfection reagents in the delivery of mRNA-encoded, DNA-encoded and protein-form Cre recombinase and CRISPR gene editors as well as short interfering RNAs to multiple cell lines and primary cell types. Intranasal administration of ELP–EEP13 combined with Cre protein achieved efficient editing of lung epithelial cells in reporter mice. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $209.00 per year only $17.42 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Data used for training EEP predictive models and the compiled α-helical peptide database are available on GitHub (https://github.com/sayoeweje/elp-eep-discovery)89. Source data are provided with this paper. All additional data are available upon request. All scripts and data used for EEP design can be found on GitHub at https://github.com/sayoeweje/elp-eep-discovery (ref. Baden, L. R. et al. Efficacy and safety of the mRNA-1273 SARS-CoV-2 vaccine. Google Scholar Polack, F. P. et al. Safety and efficacy of the BNT162b2 mRNA COVID-19 vaccine. Google Scholar Adams, D. et al. Patisiran, an RNAi therapeutic, for hereditary transthyretin amyloidosis. Google Scholar Gillmore, J. D. et al. CRISPR–Cas9 in vivo gene editing for transthyretin amyloidosis. Google Scholar Frangoul, H. et al. CRISPR–Cas9 gene editing for sickle cell disease and β-thalassemia. Google Scholar Ramaswamy, S. et al. Systemic delivery of factor IX messenger RNA for protein replacement therapy. Natl Acad. Google Scholar Rurik, J. G. et al. CAR T cells produced in vivo to treat cardiac injury. Google Scholar Parayath, N. N., Stephan, S. B., Koehne, A. L., Nelson, P. S. & Stephan, M. T. In vitro-transcribed antigen receptor mRNA nanocarriers for transient expression in circulating T cells in vivo. Google Scholar B. et al. Nanodelivery of nucleic acids. Methods Primers 2, 24 (2022). Google Scholar Tian, Y., Tirrell, M. V. & LaBelle, J. L. Harnessing the therapeutic potential of biomacromolecules through intracellular delivery of nucleic acids, peptides, and proteins. Google Scholar Bulcha, J. T., Wang, Y., Ma, H., Tai, P. W. L. & Gao, G. Viral vector platforms within the gene therapy landscape. Google Scholar Zhao, Z., Anselmo, A. C. & Mitragotri, S. Viral vector‐based gene therapies in the clinic. Google Scholar Hou, X., Zaks, T., Langer, R. & Dong, Y. Lipid nanoparticles for mRNA delivery. Google Scholar Das, R., Kanjilal, P., Medeiros, J. & Thayumanavan, S. What's next after lipid nanoparticles? A perspective on enablers of nucleic acid therapeutics. Google Scholar Koren, E., Apte, A., Sawant, R. R., Grunwald, J. & Torchilin, V. P. Cell-penetrating TAT peptide in drug delivery systems: proteolytic stability requirements. Google Scholar Schmidt, N., Mishra, A., Lai, G. H. & Wong, G. C. L. Arginine-rich cell-penetrating peptides. Google Scholar Derossi, D., Joliot, A. H., Chassaing, G. & Prochiantz, A. The third helix of the Antennapedia homeodomain translocates through biological membranes. Google Scholar Krishnamurthy, S. et al. Engineered amphiphilic peptides enable delivery of proteins and CRISPR-associated nucleases to airway epithelia. Google Scholar Sánchez-Navarro, M. Advances in peptide-mediated cytosolic delivery of proteins. Google Scholar Boisguérin, P., Konate, K., Josse, E., Vivès, E. & Deshayes, S. Peptide-based nanoparticles for therapeutic nucleic acid delivery. Google Scholar Kim, W. & Chaikof, E. L. Recombinant elastin-mimetic biomaterials: emerging applications in medicine. Google Scholar Gagner, J. E., Kim, W. & Chaikof, E. L. Designing protein-based biomaterials for medical applications. Google Scholar Jenkins, I. C., Milligan, J. J. Genetically encoded elastin-like polypeptides for drug delivery. Google Scholar Urry, D. W. Physical chemistry of biological free energy transduction as demonstrated by elastic protein-based polymers. Google Scholar Dreher, M. R. et al. Temperature triggered self-assembly of polypeptides into multivalent spherical micelles. Google Scholar Li, N. K., García Quiroz, F., Hall, C. K., Chilkoti, A. & Yingling, Y. G. Molecular description of the LCST behavior of an elastin-like polypeptide. Google Scholar Kim, W., Xiao, J. & Chaikof, E. L. Recombinant amphiphilic protein micelles for drug delivery. Google Scholar Kim, W., Brady, C. & Chaikof, E. L. Amphiphilic protein micelles for targeted in vivo imaging. Google Scholar Kim, W. et al. Targeted antithrombotic protein micelles. Google Scholar Yi, A., Sim, D., Lee, Y.-J., Sarangthem, V. & Park, R.-W. Development of elastin-like polypeptide for targeted specific gene delivery in vivo. Google Scholar Yi, A., Sim, D., Lee, S.-B., Sarangthem, V. & Park, R.-W. Application of bioengineered elastin-like polypeptide-based system for targeted gene delivery in tumor cells. Google Scholar Eweje, F. et al. Protein-based nanoparticles for therapeutic nucleic acid delivery. Google Scholar Kim, W., Thévenot, J., Ibarboure, E., Lecommandoux, S. & Chaikof, E. L. Self-assembly of thermally responsive amphiphilic diblock copolypeptides into spherical micellar nanoparticles. Google Scholar Zuris, J. A. et al. Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo. Google Scholar Dorr, B. M., Ham, H. O., An, C., Chaikof, E. L. & Liu, D. R. Reprogramming the specificity of sortase enzymes. Natl Acad. Google Scholar Kwon, H.-S. et al. Anti-human CD117 antibody-mediated bone marrow niche clearance in nonhuman primates and humanized NSG mice. Google Scholar Czechowicz, A. et al. Selective hematopoietic stem cell ablation using CD117-antibody-drug-conjugates enables safe and effective transplantation with immunity preservation. Google Scholar Freeman, E. C., Weiland, L. M. & Meng, W. S. Modeling the proton sponge hypothesis: examining proton sponge effectiveness for enhancing intracellular gene delivery through multiscale modeling. Google Scholar Hajimolaali, M. et al. Application of chloroquine as an endosomal escape enhancing agent: new frontiers for an old drug. Expert Opin. Nielsen, E. J. B. et al. In vivo proof of concept of oral insulin delivery based on a co-administration strategy with the cell-penetrating peptide penetratin. El-Sayed, A., Masuda, T., Khalil, I., Akita, H. & Harashima, H. Enhanced gene expression by a novel stearylated INF7 peptide derivative through fusion independent endosomal escape. Rai, D. K. & Qian, S. Interaction of the antimicrobial peptide aurein 1.2 and charged lipid bilayer. Peterson, J. J. & Meares, C. F. Cathepsin substrates as cleavable peptide linkers in bioconjugates, selected from a fluorescence quench combinatorial library. Biniossek, M. L., Nägler, D. K., Becker-Pauly, C. & Schilling, O. Proteomic identification of protease cleavage sites characterizes prime and non-prime specificity of cysteine cathepsins B, L, and S. J. Proteome Res. Abboud-Jarrous, G. et al. Cathepsin L is responsible for processing and activation of proheparanase through multiple cleavages of a linker segment. Biswas, A. et al. Endoprotease-mediated intracellular protein delivery using nanocapsules. ACS Nano 5, 1385–1394 (2011). Xie, J. et al. Cell-penetrating peptides in diagnosis and treatment of human diseases: from preclinical research to clinical application. Guterstam, P. et al. Elucidating cell-penetrating peptide mechanisms of action for membrane interaction, cellular uptake, and translocation utilizing the hydrophobic counter-anion pyrenebutyrate. Islam, M. Z., Ariyama, H., Alam, J. M. & Yamazaki, M. Entry of cell-penetrating peptide transportan 10 into a single vesicle by translocating across lipid membrane and its induced pores. Akishiba, M. et al. Cytosolic antibody delivery by lipid-sensitive endosomolytic peptide. Sato, H. & Feix, J. B. Peptide–membrane interactions and mechanisms of membrane destruction by amphipathic α-helical antimicrobial peptides. Ho, A., Schwarze, S. R., Mermelstein, S. J., Waksman, G. & Dowdy, S. F. Synthetic protein transduction domains: enhanced transduction potential in vitro and in vivo. Cancer Res. & Vácha, R. Advances in molecular understanding of α-helical membrane-active peptides. Brock, D. J. et al. Mechanism of cell penetration by permeabilization of late endosomes: interplay between a multivalent TAT peptide and bis(monoacylglycero)phosphate. Guay, D., Del'Guidice, T. & Lepetit-Stoffaes, J.-P. Polypeptide-based shuttle agents for improving the transduction efficiency of polypeptide cargos to the cytosol of target eukaryotic cells, uses thereof, methods and kits relating to same. US patent US9738687B2 (2016). Wang, G., Li, X. & Wang, Z. APD3: the antimicrobial peptide database as a tool for research and education. Nucleic Acids Res. Pirtskhalava, M. et al. DBAASP v3: database of antimicrobial/cytotoxic activity and structure of peptides as a resource for development of new therapeutics. Nucleic Acids Res. et al. Helical structure motifs made searchable for functional peptide design. Machida, S., Niimi, S., Shi, X., Ando, Y. & Yu, Y. Design of a novel membrane-destabilizing peptide selectively acting on acidic liposomes. Slaninová, J. et al. Toxicity study of antimicrobial peptides from wild bee venom and their analogs toward mammalian normal and cancer cells. Oh, J. H. et al. Multimeric amphipathic α-helical sequences for rapid and efficient intracellular protein transport at nanomolar concentrations. Chong, S.-E. et al. Intracellular delivery of immunoglobulin G at nanomolar concentrations with domain Z-fused multimeric α-helical cell penetrating peptides. Kobayashi, T. et al. Separation and characterization of late endosomal membrane domains. Zhang, S., Shen, J., Li, D. & Cheng, Y. Strategies in the delivery of Cas9 ribonucleoprotein for CRISPR/Cas9 genome editing. Rilo-Alvarez, H., Ledo, A. M., Vidal, A. & Garcia-Fuentes, M. Delivery of transcription factors as modulators of cell differentiation. Zhang, F. et al. Genetic programming of macrophages to perform anti-tumor functions using targeted mRNA nanocarriers. Krausgruber, T. et al. IRF5 promotes inflammatory macrophage polarization and TH1–TH17 responses. Rennick, J. J., Johnston, A. P. R. & Parton, R. G. Key principles and methods for studying the endocytosis of biological and nanoparticle therapeutics. Hołubowicz, R. et al. Safer and efficient base editing and prime editing via ribonucleoproteins delivered through optimized lipid-nanoparticle formulations. Chen, K. et al. Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. Mónica Bravo-Anaya, L. et al. Coupling of RAFT polymerization and chemoselective post-modifications of elastin-like polypeptides for the synthesis of gene delivery hybrid vectors. Bravo-Anaya, L. M. et al. Nucleic acids complexation with cationic elastin-like polypeptides: stoichiometry and stability of nano-assemblies. J. Colloid Interface Sci. Kelly, G. et al. Intratumoral delivery of brachytherapy and immunotherapy by a thermally triggered polypeptide depot. Lee, C. H., Ingrole, R. S. J. & Gill, H. S. Generation of induced pluripotent stem cells using elastin like polypeptides as a non-viral gene delivery system. Basis Dis. Piña, M. J. et al. A double safety lock tumor-specific device for suicide gene therapy in breast cancer. Cancer Lett. Del'Guidice, T. et al. Membrane permeabilizing amphiphilic peptide delivers recombinant transcription factor and CRISPR–Cas9/Cpf1 ribonucleoproteins in hard-to-modify cells. PLoS ONE 13, e0195558 (2018). Chow, D. C., Dreher, M. R., Trabbic-Carlson, K. & Chilkoti, A. Ultra-high expression of a thermally responsive recombinant fusion protein in E. coli. Tripathi, N. K. Production and purification of recombinant proteins from Escherichia coli. ChemBioEng Rev. Schneier, M., Razdan, S., Miller, A. M., Briceno, M. E. & Barua, S. Current technologies to endotoxin detection and removal for biopharmaceutical purification. Zhang, S., Sun, Y., Zhang, L., Zhang, F. & Gao, W. Thermoresponsive polypeptide fused l-asparaginase with mitigated immunogenicity and enhanced efficacy in treating hematologic malignancies. Cheng, T.-F. et al. Differential activation of IFN regulatory factor (IRF)-3 and IRF-5 transcription factors during viral infection. Brinkman, E. K., Chen, T., Amendola, M. & van Steensel, B. Easy quantitative assessment of genome editing by sequence trace decomposition. Nucleic Acids Res. Xu, L., Liu, Y. & Han, R. BEAT: a Python program to quantify base editing from Sanger sequencing. CRISPR J. Seluanov, A., Vaidya, A. & Gorbunova, V. Establishing primary adult fibroblast cultures from rodents. Angsana, J. et al. Syndecan-1 modulates the motility and resolution responses of macrophages. Chen, J. et al. Modulation of lymphocyte-mediated tissue repair by rational design of heterocyclic aryl hydrocarbon receptor agonists. Tran, N. T. et al. Efficient CRISPR/Cas9-mediated gene knockin in mouse hematopoietic stem and progenitor cells. Bayes, H. K., Ritchie, N., Irvine, S. & Evans, T. J. A murine model of early Pseudomonas aeruginosa lung disease with transition to chronic infection. Eweje, F. et al. Self-assembling protein nanoparticles for cytosolic delivery of nucleic acids and proteins. We thank members of the laboratory of E.L.C. for helpful discussions; B. Pinckney, G. Haskett and J. Tigges (BIDMC Flow Cytometry Core); A. Pauer and T. Ferrante (Wyss Institute for Biologically Inspired Engineering); S. White (BIDMC Histology Core); A. Black (BIDMC Precision RNA Medicine Core); R. Nair (Harvard Molecular Electron Microscopy Suite) and A. Berger and P. Hammond for HeLa-d2eGFP cells (via P. Jain, University of Florida). Research in the laboratory of E.L.C. was supported by the NIH (UG3AI15055 and UH3AI150551) as part of the Somatic Cell Genome Editing consortium. Research in the laboratory of D.R.L. was additionally supported by HHMI. is supported by the Harvard/MIT MD–PhD program (5T32GM007753-42) and a Ruth L. Kirschstein NRSA F31 Fellowship (F31HL167533). and A.A. acknowledge research fellowship support from the Harvard College Research Program. is supported by the Harvard/MIT MD–PhD program (5T32GM144273-02). Images in Figs. 1e were created with BioRender.com. These authors contributed equally: Feyisayo Eweje, Jiaxuan Chen. Department of Surgery, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA, USA Feyisayo Eweje, Vanessa Ibrahim, Aram Shajii, Michelle L. Walsh, Kiran Ahmad, Assma Alrefai, Dominie Miyasato, Hyunok Ham, Kaicheng Li, Carolyn A. Haller, Jiaxuan Chen & Elliot L. Chaikof Harvard and MIT Division of Health Science and Technology, Massachusetts Institute of Technology, Cambridge, MA, USA Feyisayo Eweje & Michelle L. Walsh Harvard/MIT MD–PhD Program, Boston, MA, USA Feyisayo Eweje & Michelle L. Walsh Wyss Institute of Biologically Inspired Engineering, Harvard University, Boston, MA, USA Feyisayo Eweje, Hyunok Ham, Kaicheng Li, Carolyn A. Haller, Jiaxuan Chen & Elliot L. Chaikof Department of Chemistry and Chemical Biology, Harvard University, Cambridge, MA, USA Jessie R. Davis & David R. Liu Howard Hughes Medical Institute, Harvard University, Cambridge, MA, USA Jessie R. Davis & David R. Liu Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of MIT and Harvard, Cambridge, MA, USA Jessie R. Davis & David R. Liu Department of Pathology, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar designed the experiments and analyzed the results. and A.A. expressed and purified ELPs and cargo proteins. J.C. performed sortase conjugation studies. designed and synthesized linkers for monoclonal antibody–ELP sortase conjugation. designed and cloned protein cargo plasmids. F.E., V.I., A.S. and J.C. performed nanoparticle size and zeta potential characterization. conducted in vitro delivery studies. conducted predictive model development and EEP design. and J.C. performed in vivo delivery studies and analyzed efficacy. conducted histological evaluation of lung tissues. wrote the paper with input from coauthors. Correspondence to Jiaxuan Chen or Elliot L. Chaikof. are inventors on a pending patent related to this work filed by the Beth Israel Deaconess Medical Center (PCT/US2024/038614). is a consultant and equity holder of Nvelop Medicine, Prime Medicine, Beam Therapeutics, Pairwise Plants and Chroma Medicine, companies that use or deliver gene editing or genome engineering agents. The other authors declare no competing interests. Nature Biotechnology thanks Joao Conde, Michael Mitchell and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a) PAGE gel image depicting result of sortase-mediated Cre-LPETG conjugation to V1-ELP at varying ratios. The reaction solution underwent centrifugal filtration against a 100 kDa filter to remove unreacted Cre. b) Quantification of Cre-V1-ELP conjugation efficiency via densitometry analysis, measured as percentage of total ELP conjugated to Cre. c) Diameter of Cre-V1-ELP nanoparticles (n = 3 technical replicates). d) PAGE gel image depicting result of sortase-mediated SpCas9-LPETG conjugation to V1-ELP at varying ratios. e) Diameter and f) Zeta potential of SpCas9-V1-ELP nanoparticles (n = 3 technical replicates). g) PAGE gel image and h) Densitometry analysis of CD117 mAb-V1-ELP conjugation efficiency. i) MFI and j) Fluorescent images of CD117 + P815 mast cells treated with CD117 mAb-functionalized V1-ELP nanoparticles compared to unfunctionalized controls (n = 3). For all experiments, [V1-ELP] = 8 μM unless otherwise noted. Significance assessed via one-way ANOVA with Dunnett's correction for multiple comparisons. Data represented by mean ± SEM. Supplementary Methods, Figs. 1–16, Tables 1 and 2 and uncropped supplementary gel images. Source data for supplementary figures. Sequences and catalog numbers of the oligonucleotides used. Statistical source data. Uncropped gel images. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Eweje, F., Ibrahim, V., Shajii, A. et al. Self-assembling protein nanoparticles for cytosolic delivery of nucleic acids and proteins. Nat Biotechnol  (2025). Received: 14 August 2024 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Biotechnology (Nat Biotechnol) © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            