
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-09-15</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a66102466/scientific-proof-of-precognition/'>Scientists Tried to See If Gut Feelings Could Predict the Future. What They Found Was Haunting.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 14:10:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The eerie concept may sound far-fetched, but many studies suggest there's statistical evidence to back up precognition's existence. Dive deeper into the science behind gut feelings and your mind's brilliant capabilities. While working at the University of Nevada, Radin, who had studied precognition for years, hypothesized that if consciousness transcended time, reactions to a stimulus would occur before the stimulus actually appeared. So, he put that hypothesis to the test, using an EEG machine to record the brain activity of study participants. Hooked up to the EEG, the subjects reacted to prompts and images, including positive photos (like a sunrise) and negative pictures (such as a car crash). Radin's findings were statistically significant, and since then, researchers have replicated his study successfully nearly three dozen times. His work proved influential enough that the CIA declassified its pre-sentiment research in 1995, and statisticians confirmed the effect was statistically reliable. Still, questions remain about the science behind precognition. While Radin's work provided a runway for substantiating precognition, the next step is finding evidence of its mechanism—and perhaps convincing others to let go of their fears of the unknown. Still mind-boggled by the future-predicting capabilities of your consciousness? Ashley is Editor of Content Hype at Hearst's Enthusiast & Wellness Group. Ashley loves all things running; she has raced two marathons, plus has covered some of the sport's top events in her career, including the Paris Olympics, U.S. Olympic Trials and multiple World Marathon Majors. A New Formula for Pi Is Here A College Student Broke the Laws of Thermodynamics The Maya Kingdom Collapsed Due to Burning Events Ancient Clay Tablet Holds Bronze Age Shopping List Ancient Greek Blood Money Discovered in Turkey</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250914205853.htm'>Who are the Papua New Guineans? New DNA study reveals stunning origins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 13:26:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>They share a common ancestry from the same 'Out of Africa' event that also gave rise to other non-African groups. Papua New Guineans look strikingly different from other Asian groups and share some features with Sub-Saharan African populations, which fueled theories that they might have descended separately from other non-African populations. More studies are needed to uncover how evolution shaped this remarkable population." Scientists generally agree that modern humans left Africa about 50,000 to 70,000 years ago, spreading into Europe, Asia, and beyond. Early archaeological studies have suggested that the ancestors of Papua New Guineans came from a separate, earlier migration (also known as the 'First Out of Africa' hypothesis), taking a coastal route through India and Southeast Asia. This is because the earliest human site in Oceania dates back to around 50,000 -- 60,000 years ago, older than Europe's oldest sites. Instead, analyses suggest that their lineages connect to other non-African populations. This unique inheritance likely came from mixing with Denisovans in Southeast Asia or Oceania -- another element that confirms the complexity of Papua New Guinean ancestry. Despite these researches, the genetic origin of Papuan New Guineans remains unresolved. Did they carry ancestry from the enigmatic 'First out of Africa' population? Or are they part of the same family tree as other Asians living in proximity? Their results suggest that Papua New Guineans are a sister group to other Asian populations. Contribution from a 'First Out of Africa' migration might not be needed to explain their origins. The researchers found that the ancestors of Papuan New Guineans went through a dramatic population bottleneck -- most likely their numbers dropped sharply after reaching Papua New Guinea and stayed low for thousands of years. Unlike other non-African groups, they did not experience the farming-driven population boom that reshaped Europe and Asia. This unique demographic history left genetic signatures that, if misunderstood, could look like evidence of a contribution from an unknown population. Our Galaxy's Sweet Spot for Life Is Bigger Than We Thought Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a66053963/roman-villa-warship-replica/'>Archaeologists Found an Ancient Roman Villa—with the Front of a Warship in the Dining Room</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>But it was still an example of master craftsmanship. In what was one of the greatest decoration flexes in ancient Roman design, a wealthy resident of Iuvavum—now modern-day Salzburg, Austria—in the second or third century C.E. That three-pound gold-colored bronze piece was recently found during an excavation ahead of a development in Salzburg's Old Town. “Since the mid-20th century, we had not recovered in the urban area of Salzburg-Iuvavum a bronze artifact of these dimensions and characteristics,” said Ulli Hampel, director of the Archaeology Collection at the Salzburg Museum, in a museum statement according to translated reports from local media. “Bronze objects of antiquity of this caliber are extremely rare, since the vast majority were melted down to reuse the valuable metal. The piece was found in a precarious position, broken in multiple locations under the rubble of a once-luxurious villa. It took the museum team over a year, not only to excavate all the pieces while operating under a microscope, using scalpels and an ultrasonic chisel, but then to recreate the form with the help of acrylic and preserve it with a protective layer of microcrystalline wax. “In our work, I was able to take precise impressions of the individual parts and, from the plastic copies created from them, fabricate a reformulation and reconstruction of the spectacular object,” said Maximilian Bertet, archaeological restorer at the museum. It showed not only wealth, but allegiance to the Roman Empire and appreciation for the empire's naval superiority. Originally meant as a decorative symbol to show off status, the bronze prow and rostrum will once again fulfill its original intention by highlighting an exhibit of the Salzburg branch of the new Iovavum Archaeological Museum in Salzburg when it opens in 2028. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A New Formula for Pi Is Here A College Student Broke the Laws of Thermodynamics The Maya Kingdom Collapsed Due to Burning Events Ancient Clay Tablet Holds Bronze Age Shopping List Ancient Greek Blood Money Discovered in Turkey</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a66068517/new-jersey-fossil-discoveries/'>100,000 Fossils in a New Jersey Quarry? Here's Why Paleontologists Say It's a Prehistoric Hotspot.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Some of the world's most important fossil discoveries are hiding in New Jersey's suburbs. Kenneth Lacovara, Ph.D., has spent years traveling the world to uncover prehistoric fossils, but it's the quarry near where he grew up in southern New Jersey that has yielded more than 100,000 fossils representing more than 100 species over the past 18 years. Popular Mechanics spoke with Lacovara to learn more about why New Jersey provides such a rich look at our planet's past. Curious about dinosaurs, fossils, and other ancient finds—from a rare “Book of the Dead” scroll in Egypt to 300,000-year-old fossilized footprints in Germany—that shape what we know about our world's history? Join Pop Mech Pro to unlock all of our stories on archaeological breakthroughs, paleontology, and more. Ashley is Editor of Content Hype at Hearst's Enthusiast & Wellness Group. Ashley loves all things running; she has raced two marathons, plus has covered some of the sport's top events in her career, including the Paris Olympics, U.S. Olympic Trials and multiple World Marathon Majors. A New Kind of Dinosaur is Discovered in Germany Tiny Fossil May Hold Origin and Evolution Secrets Bedbugs Are Much Older Than We Originally Thought</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41592-025-02770-8'>Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 11:29:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Recent advances in spatial transcriptomics (ST) technologies have transformed our ability to profile gene expression while preserving crucial spatial context within tissues. However, existing ST platforms are constrained by high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas, which hinder their broad applications. Here we present iSCALE, a method that reconstructs large-scale, super-resolution gene expression landscapes and automatically annotates cellular-level tissue architecture in samples exceeding capture areas of current ST platforms. The performance of iSCALE was assessed by comprehensive evaluations involving benchmarking experiments, immunohistochemistry staining and manual annotations by pathologists. When applied to multiple sclerosis human brain samples, iSCALE uncovered lesion-associated cellular characteristics undetectable by conventional ST experiments. Our results demonstrate the utility of iSCALE in analyzing large tissues by enabling unbiased annotation, resolving cell type composition, mapping cellular microenvironments and revealing spatial features beyond the reach of standard ST analysis or routine histopathological assessment. Understanding spatial gene expression distributions across tissues is essential for answering crucial biological questions and uncovering cellular functions, disease mechanisms and tissue development. ST technologies have transformed our ability to profile gene expression across tissues while retaining crucial spatial context1,2,3. However, major barriers to the widespread adoption of commercial ST platforms in biomedical research include high costs, long turnaround times, low resolution, limited gene coverage and inherently small tissue capture areas offered by current ST platforms. Sequencing-based ST platforms, such as Visium4, can sequence the whole transcriptome but lack single-cell resolution and come with a limited standard tissue capture area of 6.5 mm × 6.5 mm. Although Visium offers an extended version with a larger capture area of 11 mm × 11 mm, it is more expensive than the standard version. Additionally, many tissue samples still surpass this size limitation, restricting spatial profiling to a small portion of a biopsy or organ and often missing key biological regions. The recently released Visium HD5 offers subcellular resolution but at considerably higher cost than Visium, and its tissue capture area is only 6.5 mm × 6.5 mm. Imaging-based ST platforms such as MERSCOPE6, CosMx7 and Xenium8 provide subcellular resolution and can handle moderately larger tissues, but the number of genes is limited, and image scanning time is typically extensive. These limitations make commercial ST platforms impractical for large-scale investigations, particularly when studying sizable tissues common in human studies. A recent approach, ‘visiumStitched'9, attempts to extend the spatial profiling window by manually stitching a few Visium captures; however, it requires manual alignment and supports only low-resolution analysis of directly measured regions. On the other hand, hematoxylin and eosin (H&E)-stained histology images, routinely used by clinical pathology laboratories, are considerably more cost-effective than ST. Additionally, the physical size of a standard whole-slide H&E image can be as large as 25 mm × 75 mm, greatly exceeding the capture area of all ST platforms. Previous studies showed correlations between gene expression profiles and histological image characteristics10,11,12,13,14,15, suggesting the feasibility of inferring spatial gene expression from these histology images16,17,18,19,20. Because obtaining histology images from large-sized tissues is easily achievable, this opens the possibility of virtually predicting gene expression directly from histology, overcoming the tissue size limitations of conventional ST platforms. Here we introduce iSCALE (inferring Spatially resolved Cellular Architectures in Large-sized tissue Environments), a novel machine learning framework designed to predict gene expression for large-sized tissues with cellular-level resolution. By leveraging the gene expression–histological feature relationship, learned from a small set of training ST captures, iSCALE enables comprehensive gene expression prediction and tissue annotation across entire large tissue sections, including regions without direct gene expression measurements. Unlike existing resolution enhancement methods such as iStar16 and RedeHist19, which rely on a single ST capture for prediction, iSCALE aligns and combines information from distinct ST sections and leverages large-sized H&E images to infer gene expression across extensive tissue areas. This is achieved by mapping individual sections onto the full tissue image and integrating information across multiple ST captures to enable spatially continuous gene expression prediction. This flexible framework allows iSCALE to provide a more comprehensive and scalable approach for spatial gene expression analysis compared to current state-of-the-art methods. Our results demonstrate that iSCALE can provide critical insights into disease by generating high-resolution spatial gene expression maps for large tissue sections. An overview of the iSCALE workflow is shown in Fig. Our goal is to predict gene expression in an H&E image generated from a large-sized tissue section, referred to as the ‘mother image'. The pipeline begins by selecting regions from the same tissue block, potentially spanning adjacent sections, with regions fitting standard ST platform capture areas (Extended Data Fig. This process generates a small set of training ST captures, referred to as ‘daughter captures'. Next, iSCALE harmoniously integrates the gene expression and spatial information across the aligned daughter captures. To construct a gene expression prediction model, iSCALE extracts both global and local tissue structure information from the mother image. Subsequently, a feedforward neural network is employed to learn the relationship between histological image features and gene expression transferred from the aligned daughter captures. Leveraging these predictions, iSCALE further annotates each superpixel in the mother image with cell types and identifies enriched cell types in each tissue region, providing a detailed characterization of tissue architecture at cellular resolution. iSCALE enables ST analysis in large-sized tissues using a multi-capture study design. A high-resolution mother H&E image is obtained from a large tissue slice while multiple standard-sized daughter ST captures containing both gene expression and spatial information are collected from adjacent slices. iSCALE aligns and integrates the daughter captures based on gene expression, corrects for batch effects and smooths across spatial neighborhoods. Histological features are hierarchically extracted from regions of the mother image and combined with integrated ST data to predict super-resolution gene expression across the entire large tissue section. These predictions are then used for automatic cell type annotation and tissue segmentation. iSCALE also supports out-of-sample predictions for additional tissue sections using only H&E images. To evaluate the effectiveness of iSCALE, we conducted a benchmarking experiment using a ground truth single-cell gene expression dataset derived from a large gastric cancer tissue section profiled with 10x Xenium (https://doi.org/10.5281/zenodo.15164980)21. The section contains 377 genes and spans the full Xenium slide (12 mm × 24 mm), making it ideal for benchmarking. In this experiment, we simulated a scenario where gene expression data were available only from a set of daughter captures, each with size of 3.2 mm × 3.2 mm, mimicking the conditions typically observed in real large tissue studies. Within each daughter capture, we simulated pseudo-Visium data, following the spot size and layout of the Visium platform (Fig. First, to assess the accuracy of iSCALE's semiautomatic alignment algorithm, we aligned daughter captures using the proposed algorithm described in the Methods section and achieved 99% alignment accuracy (Extended Data Fig. For comparison, we also predicted gene expression using iStar16 and RedeHist19. Because both iStar and RedeHist can process only one daughter capture at a time to build their prediction models, we applied these algorithms for each daughter capture. As RedeHist requires a reference single-cell RNA sequencing (scRNA-seq) dataset, we used a gastric tumor scRNA-seq dataset from Cheng et al.22. Ground truth single-cell gene expression data were obtained using 10x Xenium. Daughter captures were aligned and integrated to generate a training dataset for the iSCALE model. b, Pathologist manual annotation and tissue segmentation across entire tissue sample using the predicted super-resolution gene expressions from iSCALE, iStar and RedeHist. Figure 2b shows the tissue segmentation results obtained from iSCALE, iStar and RedeHist. iSCALE's segmentation closely resembles the pathologist's manual annotation and successfully identified key tissue structures, including tumor, tumor-infiltrated stroma, mucosa, submucosa, muscle and tertiary lymphoid structure (TLS). In contrast, iStar and RedeHist segmentations varied noticeably across the five models, depending on the daughter capture used for training. For instance, iStar struggled to distinguish between tumor and mucosa when using daughter captures D1 and D5 as the training data. RedeHist performed even more poorly, failing to identify tissue structures regardless of which daughter capture was used as the training data. This is likely due to its inability to correctly detect a large portion of the nuclei (Supplementary Fig. 2c), which are associated with aggressive gastric cancer and poor prognosis and play a critical role in guiding treatment23. However, both iStar and RedeHist failed to detect this boundary when using daughter capture D1, which covers that boundary for model training (Fig. As another demonstration, we examined the TLSs in this sample. Detecting TLSs in cancer is crucial because they are associated with improved immune responses, enhanced antitumor activity and better patient prognosis, making them key indicators of the tumor microenvironment's immune dynamics24,25,26. 2d, cluster 11 (peach color) identified by iSCALE closely aligns with the manually annotated TLSs. In contrast, iStar tends to detect false-positive TLSs, whereas RedeHist exhibited substantially lower detection accuracy (Supplementary Fig. Notably, the daughter capture models that performed well in certain regions often performed poorly for other critical regions, demonstrating considerable variability among predictions. These results underscore the importance of integrating multiple daughter captures to accurately detect fine-grained tissue structures, such as TLSs. The availability of the ground truth single-cell gene expression data in this benchmarking experiment enables a quantitative evaluation of iSCALE's gene expression prediction accuracy. Although iSCALE was designed for Visium (spot-level) ST data, its prediction model can be readily adapted to use Xenium (cellular-level) ST data for training. In the following evaluations, we compared two versions of iSCALE: iSCALE-Seq, which used pseudo-Visium data for training, and iSCALE-Img, which used Xenium data for training. We computed the following metrics: root mean squared error (RMSE), structural similarity index measure (SSIM) and Pearson correlation. 3a,b, iSCALE-Seq outperformed iStar across all evaluation metrics, achieving performance similar to iSCALE-Img, despite being trained on spot-level resolution ST data. RedeHist was excluded from this comparison due to its unsatisfactory performance (Supplementary Fig. Approximately 50% of the genes achieved correlation coefficients greater than 0.45 at a spatial resolution of 32 µm × 32 µm for iSCALE-Seq. Moreover, we observed that low correlation values at the superpixel level, such as r = 0.27 for ACTA2, still resulted in spatial distributions that almost perfectly resembled the ground truth predictions (Fig. Additionally, the top 100 spatially variable genes, identified by Moran's I, revealed even stronger performance by iSCALE (Extended Data Fig. Results across all genes in the dataset (Extended Data Fig. 3b) confirm iSCALE's robustness and consistent superior performance compared to iStar. a,b, Quantitative comparison of iSCALE and iStar predictions using RMSE (lower is better), SSIM (higher is better) and Pearson correlation (higher is better) for the top 100 HVGs, evaluated separately for each iStar model (D1–D5), resulting in 500 gene-wise comparisons. RMSE and SSIM were evaluated at 8-µm spatial resolution, whereas Pearson correlation was assessed at multiple spatial resolutions. iSCALE-Seq: iSCALE trained using pseudo-Visium data daughter captures. iSCALE-Img: iSCALE trained using Xenium data daughter captures. Whiskers extend to the minimum and maximum values within 1.5× IQR from the lower and upper quartiles, respectively. b, Direct comparison of iSCALE-Seq versus iStar predictions at the per-gene level; the blue diagonal line indicates parity between methods. c, Visualization of ground truth gene expression measured by Xenium and predicted super-resolution gene expression by iSCALE-Seq and iStar at the 8-µm spatial resolution. For illustration purposes, we visualized gene expression predictions by iSCALE and iStar compared to the ground truth single-cell expression measured using Xenium (Fig. Across all examined genes, iSCALE's predictions showed the closest resemblance to the ground truth gene expression. For genes with distinct spatial patterns—for example, MYH11, TFF2, ACTA2 and CAPN8—iStar's predictions were highly dependent on the daughter capture used for training. Notably, the optimal capture varied by gene (Fig. This highlights the importance of integrating data from all daughter captures when constructing the prediction model, a strength clearly demonstrated by iSCALE. Next, we evaluated iSCALE's performance for out-of-sample prediction, where the test sample was from a different subject and included only an H&E image with no available gene expression data. For this benchmarking experiment, we used two normal gastric tissue sections from different subjects (https://doi.org/10.5281/zenodo.15164978 and https://doi.org/10.5281/zenodo.15164947)27,28, both profiled using the 10x Xenium platform with 377 genes. Each tissue section was relatively large, spanning the entire Xenium slide. For this scenario, we assumed that gene expression data were available from only 10 daughter captures (each 2 mm × 2 mm) in the training sample (Normal 1) (Fig. 4a), whereas the test sample (Normal 2) had no gene expression data. 4a) to compare iSCALE's out-of-sample and in-sample predictions. Ground truth gene expression data were obtained using 10x Xenium. Ten daughter captures for Normal 1 (N1) and eight daughter captures for Normal 2 (N2), each measuring 2 mm × 2 mm, were generated with pseudo-Visium gene expression data. Daughter captures were aligned and integrated to generate a training dataset for the iSCALE model. b, Pathologist manual annotation and tissue segmentations using the predicted gene expressions from the in-sample and out-of-sample analyses for Normal 2. c, Quantitative comparison of iSCALE in-sample and out-of-sample predictions by chi-squared statistic and Spearman correlation for the top 100 HVGs, with statistics calculated across n = 1,352,947 superpixels. Gene expression predictions were binarized using the threshold 0.025. Chi-squared statistic was calculated at a spatial resolution of 8 µm, whereas Spearman correlation was evaluated across multiple spatial resolutions. Whiskers extend to the minimum and maximum values within 1.5× IQR from the lower and upper quartiles, respectively. d, Visualization of ground truth gene expression directly measured by Xenium and predicted super-resolution gene expression by iSCALE in-sample and out-of-sample predictions, shown at the 8-µm spatial resolution. Because the range of gene expression is unobserved for out-of-sample predictions, evaluation metrics such as RMSE, SSIM and Pearson correlation, which rely on the magnitude of gene expression, are not suitable. Instead, we used Spearman correlation, which is rank based, and introduced a chi-squared statistic to assess overall concordance between binarized (that is, expressed or not expressed) gene expression patterns from the ground truth and predictions. Although the correlations for out-of-sample predictions were lower than those for in-sample predictions, at the spatial resolution of 64 µm, approximately 50% of the genes achieved correlation coefficients greater than 0.45. Although predicting gene expression magnitude is challenging, out-of-sample predictions yielded correlation coefficients similar to in-sample performance. 5 present a few representative genes with varying spatial patterns, demonstrating iSCALE's strong concordance between both in-sample and out-of-sample predictions and the ground truth expression patterns. Encouraged by the promising performance of iSCALE in benchmark evaluations, we next applied the method to data generated from a postmortem human brain of a patient with multiple sclerosis (MS). MS is a chronic inflammatory disorder that affects the central nervous system by mistakenly attacking the myelin sheath of nerve fibers, leading to inflammation and damage, which disrupts electrical impulse transmission along nerves29,30,31. Our focus is on cellular alterations in MS, specifically changes in abundance, distribution and activity of different cell types within and near MS lesions. Identifying distinct cellular profiles around MS lesions can offer insights into the disease's pathology, the immune response it provokes and potential therapeutic targets. To characterize cell attributes associated with MS lesions, we analyzed a tissue block (Sample 1) containing an active core and complete white matter chronic active lesion from a patient with MS (https://doi.org/10.5281/zenodo.15151354)32. 6), we obtained 11 smaller daughter captures from the same tissue block following the large-tissue multi-capture study design in Fig. These captures underwent gene expression analysis using the standard 10x Visium protocol with tissue capture area of 6.5 mm × 6.5 mm (Supplementary Table 1). Traditional ST analysis, which relies solely on observed spot-level gene expression from directly measured daughter captures, would be inadequate due to its low-resolution, spot-level segmentation and limited coverage of critical regions, such as the lesion rim and core. In contrast, iSCALE integrates gene expression information from 11 daughter captures across different regions to train a super-resolution gene expression prediction model for the entire large tissue section. For this analysis, iSCALE predicted the expression of the top 3,000 HVGs, along with 145 additional key cell type markers selected based on in-house single-nucleus RNA sequencing data generated from the same brain. 6, the iSCALE pipeline successfully removed gene expression batch effects across all daughter captures. In total, iSCALE predicted super-resolution gene expression for the entire mother image, consisting of 2,013,871 superpixels, after downsampling for computational purposes. These super-resolution gene expression data allowed iSCALE to segment the tissue and identify regions closely matching pathologist manual annotation (Fig. iSCALE successfully detected the white matter chronic lesion core (purple cluster), the rim with increased microglial activation (red cluster), normal-appearing white matter (orange cluster) and normal-appearing gray matter (blue and green clusters) continuously across the whole mother image. a, Mother H&E image with pathologist manual annotation, observed Visium data from 11 daughter captures obtained from adjacent tissue sections containing 36,912 spatial observations, alignment of daughter captures onto the mother image and segmentation using iSCALE's predicted gene expressions for 2,013,871 superpixels. b, Experimental validation of iSCALE's predicted gene expression using IHC stains obtained for critical markers MOG and CD68 on adjacent tissue sections. Binarized gene expression is shown for iSCALE and iStar, where a superpixel is considered expressed if its value exceeds the 25th percentile of all MOG predictions or the 25th percentile among those with expression >0.05 for CD68. Different thresholds were used because the expression range of MOG is much larger than that of CD68. Because the expression values were standardized to the range [0, 1], using a lower threshold for MOG ensures that the absolute thresholds are more comparable between MOG and CD68. c, Cell type annotation of Sample 1 using the predicted super-resolution gene expression from iSCALE across the whole mother capture. d, Cell type enrichment analysis for the identified clusters from iSCALE. e, Display of histological heterogeneity across distinct anatomical regions. An overlay of iSCALE segmentation on the mother H&E image (left), followed by close-up views of high-resolution H&E for four biological regions corresponding to iSCALE-identified clusters. ALR, active lesion rim; NAGM, normal-appearing gray matter; NAWM, normal-appearing white matter; WMILC, white matter inactive lesion core; E-neuron, excitatory neuron; IFNγ, interferon gamma; I-neuron, inhibitory neuron; Micro iron, iron-rich microglia; OL, oligodendrocyte; OPC, oligodendrocyte precursor cell; PVM, perivascular macrophage; Vascu, vascular. Next, we assessed iSCALE's prediction accuracy using immunohistochemistry (IHC) staining. This technique was used to validate iSCALE's predictions for critical markers essential for identifying and classifying MS lesions, including MOG (used to discern MS lesion types based on reduced MOG immunoreactivity, indicative of demyelination) and CD68 (an activation marker of monocyte lineage). 5b, the iSCALE gene expression predictions agreed with the IHC staining patterns obtained from adjacent tissue sections. Specifically, consistent with the demyelinated nature of the MS lesion, iSCALE predicted negligible expression of MOG within the white matter chronic active lesion core. For an in-depth analysis, we compared the observed spot-level gene expression and the predicted super-resolution gene expression for key marker genes. We further show that the super-resolution gene expression predictions by iSCALE can provide deeper insights into the cellular attributes associated with MS lesions. Using the super-resolution gene expression prediction, iSCALE automatically annotates cell types for each superpixel (Fig. We observed distinct spatial distributions of different cell types across the normal and diseased portions of the tissue, including a high density of oligodendrocytes in the normal-appearing white matter, inhibitory neurons and excitatory neurons in the gray matter, fibrous astrocytes subpopulation C (Astro fibro C) within the inactive lesion core and a mixture of iron-rich microglial and T cells in the lesion rim (Fig. By inferring cell types within these superpixels, we gained insights into the spatial distribution and diversity of cell populations within the tissue. This is particularly valuable in regions of tissue that are challenging to differentiate through histological inspection or gene expression profiling alone. For instance, the red cluster, corresponding to the active lesion rim, is enriched with immune response-related cell types, including T cells, B cells and several microglia subpopulations, such as iron-rich microglia and foamy microglia, along with infiltrated T cells within the lesion (Fig. These subpopulations are challenging for pathologists to discern from H&E images. As a comparison, we performed cell type annotation using iStar-predicted gene expressions. Unsurprisingly, the annotations were highly variable depending on which daughter capture was used for model training, and the cells were frequently annotated in locations that did not align with biological expectations. For instance, when daughter captures 612-1, 612-4 and 695-4 were used for model training, iStar predicted a high volume of inhibitory neurons in the white matter lesion core, a result that contradicts established knowledge of MS lesion biology (Extended Data Fig. To further validate the biological relevance of the iSCALE segmentation results, we compared key tissue regions: normal-appearing gray matter (clusters 1 and 2, region a), normal-appearing white matter (cluster 3, region b), active lesion rim (cluster 5, region c) and inactive lesion core (cluster 4, region d). 5e, these regions reveal distinct histological features in the H&E image. Although the inactive core exhibits fibrous structures, the active rim is denser and more heterogeneous with a mixture of immune cells. confirmed that cluster 5 aligns with the hypercellular rim containing reactive microglia and that cluster 3 reflects white matter with less smoldering microglial neuroinflammation. In this section, we explore the feasibility of using the prediction model trained on Sample 1 to predict gene expression in another MS sample. The test sample, Sample 2, was obtained from a different region of the same MS brain. Sample 2 contains a complete white matter chronic active lesion and regionally distributed active and chronic subpial cortical demyelination, as identified by manual pathologist annotation (Fig. This sample measures approximately 22.3 mm × 23.6 mm (567.7 mm2) and consists of 2,042,781 superpixels, after downsampling for computational purposes, each measuring 8 µm × 8 µm. Similar to Sample 1, gene expression in Sample 2 was originally measured using six daughter captures from the 10x Visium platform, representing only a limited area of the tissue (Supplementary Fig. To demonstrate iSCALE's out-of-sample prediction capability, we applied the model trained on Sample 1's daughter captures to predict super-resolution gene expression and performed tissue segmentation in Sample 2. For comparison, we also trained iSCALE using the six daughter captures from Sample 2 for in-sample predictions. We further validated iSCALE's out-of-sample predictions using IHC staining for MOG and CD68 obtained from adjacent tissue sections and observed that iSCALE accurately inferred the expected general locations for both markers (Fig. The subpanels include the mother H&E image with pathologist manual annotation, alignment of six daughter captures onto the mother image and segmentations using iSCALE's predicted gene expressions. In-sample prediction was based on a prediction model trained using the Visium data generated from the six daughter captures for Sample 2. Out-of-sample prediction was based on a prediction model trained using the Visium data generated from the 11 daughter captures for Sample 1. b, Experimental validation of iSCALE's in-sample and out-of-sample predictions using IHC stains obtained for critical markers MOG and CD68 on adjacent tissue sections. c, Comparison of cell type annotations for in-sample and out-of-sample predictions. Displayed are the annotations for I-neurons, Astro fibro C and T cells. d, Left, segmentation based on iSCALE's in-sample predictions, overlaid with the pathologist's annotations. Right, cell type annotations derived from iSCALE's in-sample predictions. e, Cell type enrichment analysis by cluster using iSCALE's in-sample predictions. Next, we evaluated iSCALE's performance in cell typing for out-of-sample predictions (Fig. This is more challenging than the benchmarking experiment, as the two normal samples, although from different patients, are structurally similar. In contrast, the two MS samples, although originating from the same brain, exhibit distinct lesion characteristics. Despite these marked differences between the MS samples, the out-of-sample predictions in Sample 2 successfully inferred inhibitory neurons in the gray matter cortical regions, with the locations closely matching those inferred from in-sample predictions (Fig. In addition, the out-of-sample predictions in Sample 2 mapped T cells to the expected lesion rim and astrocytes to the expected normal-appearing white matter. Although iSCALE's out-of-sample prediction is currently limited to revealing broad tissue architecture through segmentation and identifying a limited number of cell types, these promising results were achieved even across samples with distinct pathology. Notably, iSCALE's in-sample segmentation for Sample 2 closely aligned with the pathologist's manual annotation and detected the expected cortical layer structure when the number of clusters increased (Extended Data Fig. We also observed strong alignment between the identified clusters and the iSCALE cell type annotation (Fig. Notably, too, cluster 3 (green) corresponded well with oligodendrocytes and the normal-appearing white matter in the pathologist's annotation. Additionally, astrocytes (Astro fibro C), T cells and oligodendrocytes were identified throughout cluster 4 (red) corresponding to the white matter chronic active lesion (Fig. Furthermore, predicted biomarkers using in-sample predictions revealed tissue regions that posed challenges for manual annotation. This introduced uncertainty in defining the borders of subpial demyelination, highlighting the challenges that pathologists face when relying on IHC staining from different adjacent slices. In contrast, iSCALE provides simultaneous predictions for all markers and enables automatic annotation on the same large tissue sample, offering a more integrated and consistent approach for tissue analysis. In summary, we presented iSCALE, an innovative framework for rapid gene expression prediction, cell type annotation and tissue segmentation for comprehensive ST analysis of large-sized tissues, extending beyond the capabilities of conventional ST platforms. By integrating high-resolution large-scale histology images with ST data from a small set of daughter captures, iSCALE effectively processes large tissue samples. Using benchmarking data from gastric cancer and normal gastric tissues, we demonstrated iSCALE's effectiveness in semiautomatic daughter capture alignment and its ability to accurately predict super-resolution gene expression across the entire large tissue. When applied to MS brain samples, iSCALE further demonstrated its utility in assisting scientists in high-throughput and unbiased tissue annotation of entire lesions, inferring cell type compositions and pinpointing regions of interest with features not easily discernible through human visual assessment alone. Although we demonstrated the utility of iSCALE primarily using Visium, the framework is versatile enough to be applied to other ST platforms34,35,36 and helps improve our ability to uncover meaningful functions and insights from genomic data. For example, iSCALE can employ a set of selected fields of view (FOVs) in CosMx to train the gene expression prediction model, which will enable the prediction of single-cell resolution gene expression in tissue regions outside those FOVs, thereby expanding gene expression coverage to the entire tissue section. Despite this flexibility, iSCALE does not directly account for experimental design considerations, such as the strategic and optimized selection of daughter capture regions. Future research focused on systematic approaches for region-of-interest (ROI) selection, and determining the optimal number of captures could further improve data acquisition strategies and enhance the accuracy and efficiency of downstream analyses. Furthermore, this paper also highlights the ability of iSCALE to perform out-of-sample prediction—a highly challenging task due to the absence of observed gene expression and heterogeneity between the training and test data, especially in disease contexts. Although our results remain preliminary, given the limited training data in both the benchmarking and real-world scenarios, they carry major practical implications. H&E-stained histology images are widely accessible, cost-effective to generate and well suited for large tissue sections. We anticipate that, as more ST data are available, iSCALE will benefit from the increased training data. iSCALE's capability for out-of-sample prediction could substantially reduce experimental costs, making large-scale ST studies more feasible. In summary, iSCALE is a computational tool that can help enhance understanding of spatial tissue architecture at the cellular level, offering valuable insights for a wide range of human diseases. This capability has the potential to make substantial contributions to future advancements in human healthcare. With the increasing adoption of ST in biomedical research, we anticipate that iSCALE will have broad applications, particularly in human studies involving large-sized tissues. All research was conducted in accordance with relevant ethical regulations. The human gastric tissue study was approved by the institutional review board (IRB) of the Seoul Metropolitan Government–Seoul National University Boramae Medical Center (IRB no. The human MS brain tissue study was approved by the Multicentre Research Ethics Committee (08/MRE09/31+5) through the Multiple Sclerosis Society Tissue Bank at Imperial College London. In the gastric tissue study, samples were obtained from a 63-year-old female (BS06-9313; Patient 1, Tumor and Normal 1) and a 61-year-old male (AMS-47004; Patient 2, Normal 2). The gastric cancer sample was previously profiled using the 10x Xenium platform as described in Coleman et al.15. In the present study, we generated two additional Xenium datasets from normal gastric tissues. Formalin-fixed, paraffin-embedded sections (5 μm) from two normal blocks were mounted on Xenium slides. After a 20-second water rinse, RNA decrosslinking was performed at 80 °C for 30 minutes. Probe hybridization using the 10x Human Multi-Tissue and Cancer Gene Panel V1 (377 genes) was carried out at 50 °C for 18 hours. Background autofluorescence was quenched, and nuclei were fluorescently stained. Imaging was performed on the Xenium Analyzer (protocol CG000584, revision C), which conducted 15 automated cycles of probe hybridization, imaging and removal (firmware version 1.7.6.0). Image processing, including co-registration, probe filtering and nuclei/cell segmentation, was performed using xenium-1.7.1.0 software. Postmortem human brain tissue was obtained from a 59-year-old female patient with secondary progressive MS. Two regions were sampled: one containing an active core and a chronic active lesion (Sample 1) and another containing a chronic active lesion and regionally distributed active and chronic subpial cortical demyelination (Sample 2). Fresh frozen tissue blocks were provided by Imperial College London. Blocks were equilibrated to −23 °C in a cryostat for at least 30 minutes using minimal optimal cutting temperature (OCT) compound, gently faced and sectioned to obtain full-face slices. Before and after Visium slide collection, four 10-μm-thick serial sections were mounted on Superfrost Plus Gold glass slides, fixed in 10% neutral buffered formalin for 15 minutes and washed twice in PBS (10 minutes each). CD68 and MOG staining were performed on a Leica BOND RXm or a Ventana Discovery Ultra automated stainer using standard chromogenic protocols. For antigen retrieval and permeabilization, slides were heated in EDTA buffer (pH 9) at 94 °C for 25 minutes, followed by 30-minute antibody incubation (CD68: 1:2,250, Cell Signaling Technology, D4B9C, no. Bound antibodies were detected using an HRP-conjugated anti-rabbit secondary polymer and visualized with DAB and hematoxylin counterstaining. Oil Red O (ORO) staining, previously performed by Imperial College London, was used to identify lipid-laden microglia/macrophages. Whole-slide images were acquired on a Pannoramic 250 digital scanner (PerkinElmer). MS lesion types were characterized by the relative absence of MOG immunoreactivity, indicating demyelinating regions. The reactivity of lesion types was further characterized by the activity of CD68 and ORO stain. White matter lesion types were classified into active, chronic active and chronic lesion according to Kuhlmann et al.37. In brief, white matter active lesion was characterized by the loss of MOG reactivity and the presence of large numbers of lipid-laden macrophage/microglia cells with CD68 reactivity and positive ORO stain. The active rim was annotated surrounding the white matter chronic active lesion with the presence of an extensive CD68-positive cellular infiltrate, corresponding to macrophages/activated microglia as described by Absinta et al.38. Cortical gray matter lesion was characterized by the loss of MOG reactivity at gray matter regions and further subtyping into subpial, leukocortical and intracortical lesions39,40. Lesions showing loss of myelin indicated by MOG staining from the outer layer into the cortex were classified as subpial lesions. To collect tissues onto Visium Spatial Gene Expression slides (10x Genomics Slide Kit, ref. 1000185), a pre-chilled razor blade was used to gently score the surface of the frozen tissue block in a grid format to subdivide the tissue for placement on individual capture areas. Slides were pre-chilled prior to collection of tissue sections, and individual sections were collected onto slides by placing a fingertip on the back of the slide to promote tissue capture. Frozen slides were then processed for ST analysis according to the manufacturer's instructions (10x Visium ‘Tissue Staining & Imaging'). Samples were then stained with H&E. Brightfield images were collected on an Olympus VS120 Virtual Slide Microscope equipped with a VC50 camera using a UPLSAPO ×20 objective. After imaging, tissue permeabilization was carried out using the supplied permeabilization buffer at 37 °C for 6 minutes. Permeabilization time was selected based on a prior optimization experiment in mouse tissues using Visium Spatial Tissue Optimization slides (10x Genomics, ref. After permeabilization, reverse transcription was carried out on-slide according to Visium guidelines. A critical step in iSCALE is aligning daughter captures onto the mother image. Given that the daughter captures are often derived from different tissue sections and exhibit drastically different shapes, existing alignment methods fall short of adequately addressing the complexity of these data. To overcome these challenges, we developed a semiautomatic, human-in-the-loop alignment approach. This method streamlines the alignment process while maintaining the flexibility needed to handle complex data structures. Step 1: Clustering spatial gene expression data in daughter captures. Apply spatial clustering algorithms—for example, BayesSpace41—to cluster gene expression data in daughter captures. We recommend generating 2–4 clusters to minimize noisy clustering patterns. The resulting clusters can then be used to facilitate the identification of key points in Step 2. Step 2: Identify key point pairs between a daughter capture and the mother image. Identify at least four key point pairs between a daughter capture and the mother image. Select a new daughter capture and identify key point pairs between this capture and the mother image. Step 5: Repeat alignment of remaining daughter captures. Continue the above alignment process until no additional daughter captures can be aligned. The semiautomatic procedure described above requires minimal manual work and is relatively straightforward to implement. 2 provides an example to illustrate how this alignment algorithm works. To construct the iSCALE training dataset, we integrate location and gene expression information from individual ST experiments. Let qs denote the number of spatial observations in daughter capture s and define the total number of observations as \(Q={\sum }_{s=1}^{S}{q}_{s}\). For each daughter capture \(s\in \{1,\ldots ,S\}\), we define the spatial coordinate matrix \({\phi }_{s}\in {{\mathbb{R}}}^{{q}_{s}\times2}\), where each row corresponds to the (x, y) coordinates of a spot. To combine the spatial locations across all captures, we concatenate the aligned coordinate matrices \({\phi }_{1},\ldots ,{\phi }_{S}\) into a single matrix \(L\in {{\mathbb{R}}}^{Q\times 2}\). Next, we combine gene expression data from multiple daughter captures. Let τs be the set of all genes measured in capture s with length ws. We define \({G}_{s}\in {{\mathbb{R}}}^{{q}_{s}{\times w}_{s}}\) as the gene expression matrix for daughter capture s, where each row corresponds to a spatial measurement (that is, spot or cell), and each column corresponds to a gene. Let \(\gamma =\mathop{\bigcap }\nolimits_{s=1}^{S}{\tau }_{s}\) represent the intersection of genes shared across all captures with length |γ| = φ. To ensure comparability, we subset each expression matrix to include only genes in γ defining the reduced matrices as \({G}_{s}^{* }={G}_{s}(:,\gamma )\in {{\mathbb{R}}}^{{q}_{s}\times \varphi }.\) We concatenate the reduced matrices along the spatial axis to form a unified expression matrix \(G^{*} \in {{\mathbb{R}}}^{Q\times \varphi }\). This matrix serves as a large training dataset covering spatially distinct regions of the tissue section. To further integrate information across partially overlapping daughter captures, iSCALE applies a smoothing step based on dense spatial neighborhoods defined by a distance threshold d. Within each dense neighborhood, both spatial coordinates and gene expression values are averaged to propagate information across adjacent slices. This integration ensures consistency across slices by effectively smoothing spatially overlapping regions. After constructing the gene expression training dataset, iSCALE automatically selects the top HVGs across all spatial locations. The user specifies the desired number of HVGs to be selected and can optionally modify this list to include additional genes of biological or clinical interest. Integrating gene expression data from multiple ST captures can introduce batch effects, potentially obscuring biological signals. 6a, this issue was evident in MS Sample 1, where daughter captures formed distinct clusters in the uniform manifold approximation and projection (UMAP) space. To address this, iSCALE applies a minimum–maximum normalization across all daughter captures, scaling each gene to the [0, 1] range. Let \({\mathcal{G}}\) denote the normalized gene expression matrix, where each gene ‘k' is scaled as: using the minimum and maximum expression across all spatial locations. 6b, this procedure effectively mitigates batch effects, resulting in mixed expression profiles from different captures. iSCALE also generates pre-normalization and post-normalization UMAP plots to visualize batch effects and assess correction. If residual effects remain, users may consider alternative methods such as z-score normalization or batch correction tools such as CarDEC42 or Scanorama43 prior to model training. Although histology images were leveraged to analyze ST data in previous works12,18,44,45, their use of histology information does not fully leverage the rich cellular information provided by high-resolution histology images. In practice, a pathologist examines histology images hierarchically—first identifying ROI by assessing high-level image features that capture global tissue structure and then examining low-level features reflecting local cellular structures. To standardize resolution, each histology image is first rescaled so that the size of each pixel is 0.5 × 0.5 μm, making a 16 × 16-pixel tile correspond to an 8 × 8-μm2 area—approximately the size of a single cell. Next, mimicking this hierarchical review process computationally, we implement a multiscale image feature extraction strategy that captures both global and local tissue structures. This histology image feature extraction approach employs the Vision Transformer (ViT)46 in a hierarchical manner and extracts essential tissue details by breaking down the tissue image processing into stages. It starts by analyzing smaller tissue sections for the fine details, subsequently scaling up to grasp broader tissue patterns. Details of this image feature extraction were described in iStar16. Previously, the method iStar16 was developed to enhance spatial resolution using a single ST sample as both input and training data. In our setting, however, there are multiple ST samples obtained from adjacent tissue slices, and the large target mother image lacks direct gene expression measurements. To address this, we incorporate multiple training ST samples and perform out-of-sample super-resolution gene expression prediction for the missing tissue regions. Histology features from the large-scale mother image are extracted as \(H={\left[{h}_{{mn}}\right]}_{m=1,n=1}^{M^{\prime} ,N^{\prime} }\), and gene expression prediction is performed by training on the integrated daughter data, \({\mathcal{G}}\), to minimize the specified objective function. Specifically, when the daughter capture is from a sequencing-based ST platform, such as 10x Visium, then iSCALE integrates histology features and ST data using a spot-level weakly supervised loss function, allowing differences in spatial resolution between training and prediction. The loss, \({{\mathcal{L}}}_{\rm{{Seq}}}\), is computed over all observed spots \(q\in Q\) where \({g}_{k}\) is the prediction model for gene k; \({{\mathcal{M}}}_{q}\) is the spot mask of q (the collection of superpixels covered by spot q); and hmn is the histology feature at superpixel (m, n): When the daughter captures are from an imaging-based ST platform, such as 10x Xenium or CosMx, iSCALE employs a directly supervised loss function using gene expression labels available at the superpixel level during training: where P denotes the set of superpixels across all daughter captures. After model training, gene expression at each superpixel (m, n) in the mother image is predicted as \({\hat{y}}_{{kmn}}={g}_{k}({h}_{{mn}})\), yielding the continuous super-resolution gene expression image \({\hat{Y}}_{k}=[\;{\hat{y}}_{{kmn}}]_{m=1,n=1}^{{M}^{{\prime} },{N}^{{\prime} }}\in {{\mathbb{R}}}^{{M}^{{\prime} }{N}^{{\prime} }}\) across the entire large-scale mother image for each gene k. A multilayer perceptron architecture was used, consisting of a feedforward neural network with four hidden layers, each containing 256 neurons and employing a leaky rectified linear unit (ReLU) activation function (α = 0.1). The model was trained with a batch size of 100 using the Adam optimizer (learning rate 1 × 10−3) for 1,000 epochs across three random initialization states. No formal hyperparameter optimization was conducted; instead, architecture and training parameters were chosen based on prior experience and performance evaluation using RMSE loss. Training and validation were conducted on an NVIDIA graphics processing unit (GPU) with 64 GB of memory to accommodate the largest tissue sample. We ran iSCALE on a server equipped with an NVIDIA GPU (driver version 550.54.15, CUDA 12.4). The process of extracting histology image features, smoothing and exporting data took approximately 13 minutes for Sample 1 and 15 minutes for Sample 2. Integrating data across daughter captures, training the gene expression prediction model (1,000 epochs, repeated for three random states), predicting gene expression and exporting super-resolution gene expression data for 1,000 genes required 1 hour and 35 minutes for Sample 1 and 54 minutes for Sample 2. The predicted gene expression can be used for tissue segmentation through clustering analysis with the k-means algorithm. iSCALE provides two options for this clustering analysis. By the end of the clustering analysis, iSCALE partitions the tissue into functionally distinct regions in an unsupervised manner based on their gene expression profiles. To support interpretation of segmented tissue regions and reveal cell population heterogeneity, iSCALE performs cell type inference at the superpixel level across the full tissue section. Each superpixel is treated as an artificial cell, with its cell type label inferred using iSCALE-predicted gene expression and a marker gene reference panel. The traditional score-based approach from iStar16 often yields overdominant predictions, assigning a single cell type broadly—especially when marker genes have outlier or inflated expression, which is more likely when training data come from multiple ST captures. To address this, iSCALE introduces key modifications for more robust cell type annotation. Let V be the total number of candidate cell types in the marker reference set. For each cell type \(v\in \{1,\ldots ,V\;\}\), suppose that we have a list of marker gene names ψv. In iSCALE, we apply a filtering criterion based on prediction uncertainty for genes in the marker set. Specifically, we compute the training RMSE for each gene and remove those exceeding the 95th percentile across all genes, yielding a refined marker set \({\psi }_{v}^{* }\). This step eliminates genes that are poorly predicted in the training set from being selected as a candidate for cell typing. For each gene in \({\psi }_{v}^{* }\), we then compute its percentile expression across all superpixels in the tissue. To infer the cell type of superpixel (m, n), let \({t}_{{mn}}^{\max}=\mathop{{\rm{arg}}}\limits_{1\le {\rm{v}}\le {\rm{V}}}\max {u}_{{tmn}}\) be the cell type with the maximal score and \({u}_{{mn}}^{\max }=\mathop{\max }\limits_{1\le {\rm{v}}\le {\rm{V}}}{u}_{{tmn}}\) be the score of this cell type. Given a predetermined threshold \({u}_{{{\rm{threshold}}}}\in \left[0.0,1.0\right]\), if \({{u}_{{mn}}^{\max } > u}_{{{\rm{threshold}}}}\), then the cell type of superpixel (m, n) is predicted to be \({v}_{{mn}}^{\max }\); otherwise, the cell type of this superpixel is unclassified. Overall, the adjusted score-based approach sets thresholds more consistently across datasets with varying magnitudes of imputed expression and reduces reliance on arbitrary cutoffs typical of standard methods. The percentile adjustment mitigates the influence of outlier genes with high predicted expression that might otherwise skew scores and lead to a dominant cell type label, improving robustness to marker gene selection. Although our pipeline uses this percentile-based scoring, iSCALE also allows users to apply any preferred cell type annotation method to the predicted super-resolution gene expression. After cell type inference, iSCALE performs enrichment analysis by identifying abundant cell types within each tissue cluster to reveal biological activity for different spatial regions. For in-sample predictions, we evaluated model performance using three metrics per gene. First, RMSE was calculated as \(1/\sqrt{n}\) times the Euclidean distance between observed and predicted expression across n superpixels. Second, we used the SSIM, which assesses both intensity and spatial structural similarity between expression images; SSIM scores range from 0 to 1, with 1 indicating perfect similarity. Third, Pearson correlation was used to assess the linear relationship between observed and predicted values. For out-of-sample predictions, where absolute gene expression magnitudes in the test data are unobserved, we instead applied Spearman correlation, which compares the ranks of expression values. We also evaluated concordance between binarized ground truth and predicted expression using a chi-squared test, categorizing gene expression as ‘expressed' or ‘not expressed' based on a predefined threshold. This test follows a chi-squared distribution with 1 degree of freedom under the null hypothesis of no association. Resulting P values were corrected for multiple testing across genes using the Bonferroni method. To account for extremely high values, the \({-\log }_{10}P\) metric was capped at a maximum of 10; values exceeding this threshold were set to 10. Complete information about the notation described in the Methods section is available in Supplementary Table 2. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. We analyzed the following datasets in this study: (1) 10x Xenium spatial transcriptomics data from a human patient with gastric cancer, tumor sample (Gastric Patient 1, Tumor) (IRB no. Five pseudo-Visium captures derived from the whole-slide sample were generated for benchmarking. (2) 10x Xenium spatial transcriptomics data from a human patient with gastric cancer, normal sample (Gastric Patient 1, Normal 1) (IRB no. Eleven pseudo-Visium captures derived from the whole-slide sample were generated for benchmarking. (3) 10x Xenium spatial transcriptomics data from a healthy human gastric patient, normal sample (Gastric Patient 2, Normal 2) (https://doi.org/10.5281/zenodo.15164947). Eight pseudo-Visium captures derived from the whole-slide sample were generated for benchmarking. Eleven 10x Visium spatial transcriptomics samples corresponding to partial tissue regions from adjacent slices and high-resolution H&E image of the full large tissue section (https://doi.org/10.5281/zenodo.15151354). Six 10x Visium ST samples corresponding to partial tissue regions from adjacent slices and high-resolution H&E image of the full large tissue section (https://doi.org/10.5281/zenodo.15151517). The benchmarking data generated from the gastric tumor sample are available as a demo for our method on GitHub (https://github.com/amesch441/iSCALE). All benchmarking datasets generated can be publicly accessed at https://upenn.box.com/s/cburekr425ibu276wyxki09q35z2o3x0. Details of the real-world large-tissue MS datasets analyzed in this study are provided in Supplementary Table 1. All datasets referenced in this paper have been made publicly available via Zenodo. Source data are provided with this paper. The iSCALE algorithm was implemented in Python and is available on GitHub at https://github.com/amesch441/iSCALE. Method of the year 2020: spatially resolved transcriptomics. & Lundeberg, J. Spatially resolved transcriptomics adds a new dimension to genomics. Stahl, P. L. et al. Visualization and analysis of gene expression in tissue sections by spatial transcriptomics. Oliveira, M. F. et al. High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer. Moffitt, J. R. et al. Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region. He, S. et al. High-plex imaging of RNA and proteins at subcellular resolution in fixed tissue by spatial molecular imaging. Janesick, A. et al. High resolution mapping of the tumor microenvironment using integrated single-cell, spatial and in situ analysis. Integrating gene expression and imaging data across Visium capture areas with visiumStitched. Schmauch, B. et al. A deep learning model to predict RNA-Seq expression of tumours from whole slide images. Integrating spatial gene expression and breast tumour morphology via deep learning. Bergenstrahle, L. et al. Super-resolved spatial transcriptomics by deep data fusion. Jia, Y., Liu, J., Chen, L., Zhao, T. & Wang, Y. THItoGene: a deep learning method for predicting spatial transcriptomics from histological images. Lin, Y. et al. A contrastive learning approach to integrate spatial transcriptomics and histological images. Resolving tissue complexity by multimodal spatial omics modeling with MISO. Zhang, D. et al. Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology. Innovative super-resolution in spatial transcriptomics: a transformer model exploiting histology images and spatial gene expression. Hu, J. et al. Deciphering tumor ecosystems at super resolution from spatial transcriptomics with TESLA. & Ren, X. Spatial transcriptomics prediction from histology images at single-cell resolution using RedeHist. & Li, M. Unlocking the power of spatial omics with AI. Cheng, X. et al. Atlas of metastatic gastric cancer links ferroptosis to disease progression and immunotherapy response. Pernot, S. et al. Signet-ring cell carcinoma of the stomach: impact on prognosis and specific therapeutic challenge. The immune contexture in human tumours: impact on clinical outcome. Dieu-Nosjean, M. C., Goc, J., Giraldo, N. A., Sautes-Fridman, C. & Fridman, W. H. Tertiary lymphoid structures in cancer and beyond. Tertiary lymphoid structures improve immunotherapy and survival in melanoma. Dendrou, C. A., Fugger, L. & Friese, M. A. Immunopathology of multiple sclerosis. Multiple sclerosis large human brain tissue (MS Sample 1-White Matter Chronic Active Lesion and Active Core) with 11 spatial transcriptomics captures using 10x Genomics Visium. Multiple sclerosis large human brain tissue (MS Sample 2-White Matter Chronic Active Lesion) with 6 spatial transcriptomics captures using 10x Genomics Visium. Liu, Y. et al. High-spatial-resolution multi-omics sequencing via deterministic barcoding in tissue. Bai, Z. et al. Spatially exploring RNA biology in archival formalin-fixed paraffin-embedded tissues. Rodriques, S. G. et al. Slide-seq: a scalable technology for measuring genome-wide expression at high spatial resolution. Kuhlmann, T. et al. An updated histological classification system for multiple sclerosis lesions. Persistent 7-tesla phase rim predicts poor outcome in new multiple sclerosis patient lesions. Bo, L., Vedeler, C. A., Nyland, H. I., Trapp, B. D. & Mork, S. J. Subpial demyelination in the cerebral cortex of multiple sclerosis patients. & Trapp, B. D. Transected neurites, apoptotic neurons, and reduced inflammation in cortical multiple sclerosis lesions. Zhao, E. et al. Spatial transcriptomics at subspot resolution with BayesSpace. Lakkis, J. et al. A joint deep learning model enables simultaneous batch effect correction, denoising, and clustering in single-cell transcriptomics. Efficient integration of heterogeneous single-cell transcriptomes using Scanorama. Hu, J. et al. SpaGCN: integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network. Tan, X., Su, A., Tran, M. & Nguyen, Q. SpaCell: integrating tissue morphology and spatial gene expression to predict disease cells. was partly supported by National Institutes of Health (NIH) grants R01HG013185, R01LM014592, R01HL171595 and U01CA294518 and a sponsored research agreement from Biogen, Inc. T.H.H. was also partly supported by the Eric and Wendy Schmidt Foundation's AI Innovation Award through the Mayo Clinic Foundation and the AACR Innovation and Discovery Grant. The MS tissue samples and associated clinical and neuropathological data were supplied by the Multiple Sclerosis Society Tissue Bank, funded by the Multiple Sclerosis Society of Great Britain and Northern Ireland (registered charity 207495). Amelia Schroeder, Melanie L. Loth, Chunyu Luo, Sicong Yao, Hanying Yan, Daiwei Zhang & Mingyao Li Sarbottam Piya, Edward Plowey, Wenxing Hu, Su Jing Chan, Taylor L. Reynolds, Thomas Carlile, Patrick Cullen, Hui-Hsin Tsai & Baohong Zhang Department of Surgery, Vanderbilt University Medical Center, Nashville, TN, USA Jean R. Clemenceau, Inyeop Jang, Minji Kim, Isabel Barnfather & Tae Hyun Hwang Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar This study was conceived of and led by M.L., B.Z. A.S. designed the model and algorithm, with input from M.L. A.S. implemented the iSCALE software and led data analyses, with input from all other co-authors. developed the semiautomatic daughter capture alignment algorithm. generated and processed the Xenium data for a gastric cancer sample and two normal gastric samples. provided biological interpretation of the MS results. Correspondence to Tae Hyun Hwang, Baohong Zhang or Mingyao Li. receives research funding from Biogen, Inc., which partly supports the present work. is a co-founder of Kure.ai therapeutics and Kure.s and has received consulting fees from IQVIA; these affiliations and financial compensations are independent of the research described in this paper. The other authors declare no competing interests. Nature Methods thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Madhura Mukhopadhyay, in collaboration with the Nature Methods team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Ideal grid-based layout design: The tissue is divided into a grid, and each section (daughter capture) is profiled using an ST platform (for example 10x Visium). This layout allows for systematic and contiguous spatial coverage of the tissue. b, Real-world adjacent tissue slice design: Daughter captures are obtained from adjacent sections of the same tissue block. Due to technical and experimental challenges of tissue processing, these captures often only partially overlap in the X-Y plane, and gaps may arise, leaving regions of the mother tissue unmeasured. a, Key point detection step showing spatial coordinates from a pseudo-Visium daughter capture (left) and corresponding landmarks in the H&E-stained mother image (right). b, Example alignment of a daughter capture (D6) to the mother image using detected key points. d, iSCALE alignment accuracy across ten daughter captures. For each daughter capture, the percentage of overlapping pixels between the aligned and true capture location is reported. RMSE and SSIM were evaluated at 8 µm spatial resolution, while Pearson correlation was assessed at multiple spatial resolutions. iSCALE-Seq: iSCALE trained using pseudo-Visium daughter captures. iSCALE-Img: iSCALE trained using Xenium daughter captures. a,b, Comparisons for top 100 SVGs, identified by Moran's I, evaluated separately for each iStar model (D1-D5), resulting in 500 gene-wise comparisons. c,d, Comparisons for all genes captured in the Xenium data (377 genes), evaluated separately for each iStar model (D1-D5), resulting in 1885 gene-wise comparisons. Whiskers extend to the minimum and maximum values within 1.5 x IQR from the lower and upper quartiles, respectively. b,d, Direct comparison of iSCALE-Seq versus iStar predictions at the per-gene level; the blue diagonal line indicates parity between methods. Visualization of predicted super-resolution gene expression in the Tumor gastric sample for additional genes with distinct spatial patterns (MYH11, DES, DMBT1, MYLK). Each row displays spatial expression at 8 μm resolution across the full tissue for a given model. The top row shows Xenium-measured ground truth, followed by predictions from iSCALE (Img and Seq). iSCALE-Seq: iSCALE trained using pseudo-Visium daughter captures. iSCALE-Img: iSCALE trained using Xenium daughter captures. The bottom five rows show iStar predictions trained separately on daughter captures D1–D5. Visualization of predicted super-resolution gene expression in the Normal 2 gastric sample for additional genes with distinct spatial patterns (DES, EPCAM, DST, MYLK, OGN). Each row displays spatial expression at 8 μm resolution across the full tissue for a given model. The top row shows Xenium-measured ground truth, followed by in-sample predictions from iSCALE trained on eight pseudo-Visium daughter captures from Normal 2. The bottom row shows out-of-sample predictions from iSCALE trained on eleven pseudo-Visium daughter captures from Normal 1. Region-specific annotations and spatial mapping of the large tissue sections from a, MS Sample 1 and b, MS Sample 2. Boxed outlines indicate distinct lesion subtypes and anatomical tissue regions. The blue dashed box denotes the approximate region sampled on adjacent tissue slices for 10x Visium daughter captures. a, Key point detection step showing spatial coordinates from a Visium daughter capture (left) and corresponding landmarks in the H&E-stained mother image (right). b, Slignment of a daughter capture (612-1) to the mother image using detected key points. c, Complete alignment of all eleven Visium daughter captures for MS Sample 1, overlaid on the mother image. d, Complete alignment of all six Visium daughter captures for MS Sample 2, overlaid on the mother image. Visualization of predicted super-resolution gene expression in the MS Sample 1 for additional genes with distinct spatial patterns (SLC17A7, HLA.DRA, MBP, MAP2, PLP1). Each row displays spatial expression at 8 μm resolution across the full tissue for a given model. The top row shows the observed spot-level gene expression aligned to the mother image, followed by super-resolution predictions from iSCALE trained using eleven Visium daughter captures. The bottom five rows show iStar predictions trained individually from a set of representative daughter captures (612-1, 612-4, 613-2, 695-1, 695-4). Cell type annotations in a large postmortem MS brain section using super-resolution gene expression predictions from iSCALE and iStar. The iStar models were trained independently on representative daughter captures (612-1, 612-4, 613-2, 695-1, and 695-4). Each panel shows predicted cell types for the same tissue region using a consistent reference marker panel. a, Pathologist-defined tissue annotations on the mother H&E image for MS Sample 2, including normal-appearing white matter (NAWM), hypercellular active lesion rim, white matter (WM) chronic lesion, cortex-normal appearing gray matter (Cortex-NAGM), and cortex-subpial demyelination. b, Superpixel-level segmentation results from iSCALE, where each panel displays the inferred tissue architecture across increasing numbers of clusters including 4, 5, and 7. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Scaling up spatial transcriptomics for large-sized tissues: uncovering cellular-level tissue architecture beyond conventional platforms with iSCALE. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02921-z'>GPS timekeeping is increasingly vulnerable: here's how to deliver future-proofed time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 10:42:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Aviation and road transport are among the sectors that currently depend on increasingly vulnerable GPS timekeeping. It might not be apparent, but almost everything people do in this digitally connected world, from sending a text or e-mail to doing an Internet search or paying for something online, depends on accurate, resilient timing. Most electronic communications rely on digital infrastructure that is synchronized to a common time reference, enabling information to flow at a known rate. Today, the necessary timing signals come overwhelmingly from Global Navigation Satellite Systems (GNSS). These deliver signals from space, with only a radio receiver needed to receive and interpret them. Chief among them is the US-developed Global Positioning System (GPS). GPS is free to use, and the technology is mature: miniature GPS receivers are now embedded in every smartphone, at a cost of a few dollars per unit. This is an easy, low-cost way to reliably synchronize time, certainly compared with the old method of physically calibrating your clock to a reference clock at a national standards agency. Best ever clocks: breakthrough paves way for ultra-precise ‘nuclear' timekeepers Best ever clocks: breakthrough paves way for ultra-precise ‘nuclear' timekeepers But this success has bred complacency — and led to underappreciated risk. In many parts of the world, organized-crime syndicates and military authorities are among those increasingly jamming GNSS signals (blocking them entirely) or spoofing them (transmitting false signals to provide misleading time and position coordinates). But there is low awareness of the risks — and protections are hugely variable between sectors. Here we set out three key steps that we think would help to facilitate the development of alternative time sources worldwide: increasing buy-in from business; linking up existing, heterogeneous local timing systems; and creating entirely new uses for time. Each GPS satellite carries multiple atomic clocks, which are ultimately synchronized with a representation of the UTC, known as UTC(USNO), generated by an ensemble of atomic clocks at the US Naval Observatory in Washington DC. Other GNSS systems exist, such as GLONASS (operated by Russia), Galileo (the European Union) and BeiDou (China). But GPS remains dominant owing to widespread public confidence in a US-provided infrastructure and to the system's earlier availability for general use, which means a large number of older receivers are only GPS-compliant. There is also a need to support the development of alternative assured ‘holdover time' capabilities: accurate, locally based clocks that can continue to provide time for a while if the main GNSS signals are lost. Such a public utility could instil trust and confidence in users, and cover its costs by offering tiered, paid-for levels of assurance of time. In the United Kingdom, a supporting terrestrial clock system is being designed and developed as a nationwide time infrastructure through the National Timing Centre research and development programme (of which one of us, L.L., is head) at the National Physical Laboratory (NPL) at Teddington. But limited understanding of the need for alternative assured-time capabilities is a barrier to achieving business and political buy-in. Crucial infrastructures, from financial services to transport and energy systems, are often interdependent. Yet they tend to operate as independent ecosystems, subject to their own legislation and with their own regulatory standards, policies, supply chains and digital ecosystems. Time's intangible nature means that it often falls in the gap between cybersecurity and physical-security measures designed to protect digital and physical assets, respectively. When mitigating GNSS loss or disruption is considered at all, it tends to be by individual firms on an ad hoc basis. For example, telecommunications network providers often have atomic clocks distributed across their fibre exchanges to enable synchronization to be maintained for the Internet backbone even if GNSS signals are disrupted. Such mitigation measures are rarely tested robustly for the possibility of GNSS outages across all interconnected sectors. Atomic clock keeps ultra-precise time aboard a rocking naval ship Atomic clock keeps ultra-precise time aboard a rocking naval ship This underlines the desirability of government intervention to provide a base capability that is accessible to all users wherever and whenever they need it. This might take the form of new legislation or regulation and standards to allow interoperability between essential infrastructures. A further consideration is that the global supply chain for timing systems — from atomic clocks, time distribution and monitoring systems to user equipment — is currently sparse and lacks diversity. This needs to be complemented by an upgrade of the skills pipeline. Private-sector investments in building such a resilient supply chain to provide reliable timing will depend on effective, long-term commitment from governments1. This could be achieved by regulating the minimum holdover time — the length of time any backup system can provide accurate timekeeping — across industries to ensure commitment to build and develop such time-based resilience. A focus solely on managing risk and enhancing the resilience of time systems might not, therefore, be enough to convince firms of the need to invest in supporting the development of terrestrial holdover timing systems. Financial markets require precise, trustable time for transactions to be recorded accurately.Caption: Michael M. Santiago/Getty Best ever clocks: breakthrough paves way for ultra-precise ‘nuclear' timekeepers Atomic clock keeps ultra-precise time aboard a rocking naval ship Network of world's most accurate clocks paves way to redefine time The chatbots claiming to be Jesus: spreading gospel or heresy? Why we launched Denmark's second Young Academy (and what's different about it) How to help refugees thrive: have local families host them Scientists take on Trump: these researchers are fighting back Data anomalies and the economic commitment of climate change State Key Laboratory of Materials-Oriented Chemical Engineering, Nanjing Tech University Join the International Institute for Intelligent Nanorobots and Nanosystems (IIINN) at Fudan University. Exceptional young scholars worldwide with strong research achievements in relevant fields Institute of Nanotechnology and Intelligence (inAI), Jinan University, China. Best ever clocks: breakthrough paves way for ultra-precise ‘nuclear' timekeepers Atomic clock keeps ultra-precise time aboard a rocking naval ship Network of world's most accurate clocks paves way to redefine time An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41565-025-02009-9'>The rising danger of AI-generated images in nanomaterials science and what we can do about it</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 10:33:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Generative AI has made it trivial to generate fake microscopy images that are indistinguishable from real images, even for experts. As researchers in nanoscience, it is time for us to face this reality and discuss strategies to conserve the integrity of our discipline. This is a preview of subscription content, access via your institution Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Wilson, E. A. K., Kumar, P., Montjoy, D. G. & Kotov, N. A. ACS Nano 16, 13942–13948 (2022). The authors sincerely thank V. Foliush for initially shocking us with AI replication of experimental images, P. Formanek, K. Werner Stöckelhuber and E. Bittrich for discussions, and all anonymous participants in our online survey. acknowledge support from the Deutsche Forschungsgemeinschaft (DFG) with project IDs 496201730 (Q.A.B), 451785257 (A.F. S.B., N.A.K., and L.M.L.-M. acknowledge funding from the European Research Council through ERC Synergy Grant Chiral-Pro (project ID 101166855). acknowledges funding by the Federal Ministry of Education and Research of Germany (BMBF) in the programme NanoMatFutur (grant no. Nadiia Davydiuk, Elisha Krieg, Jens Gaitzsch, Patrick M. McCall, Günter K. Auernhammer, Andreas Fery & Quinn A. Besford Institute of Comparative Medicine, Columbia University Irving Medical Center, New York, NY, USA Department of Materials Science and Engineering, North Carolina State University, Raleigh, NC, USA Biointerfaces Institute and Department of Materials Science and Engineering, University of Michigan, Ann Arbor, MI, USA Center for Complex Particle Systems (COMPASS), Ann Arbor, MI, USA CIC biomaGUNE, Basque Research and Technology Alliance (BRTA), Donostia-San Sebastián, Spain Ikerbasque, Basque Foundation for Science, Bilbao, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Matthew Faria or Quinn A. Besford. The authors declare no competing interests. The rising danger of AI-generated images in nanomaterials science and what we can do about it. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41594-025-01666-7'>GluA4 AMPA receptor gating mechanisms and modulation by auxiliary proteins</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 10:16:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Structural & Molecular Biology (2025)Cite this article AMPA-type glutamate receptors, fundamental ion channels for fast excitatory neurotransmission and synaptic plasticity, contain a GluA tetrameric core surrounded by auxiliary proteins such as transmembrane AMPA receptor regulatory proteins (TARPs) or Cornichons. Their exact composition and stoichiometry govern functional properties, including kinetics, calcium permeability and trafficking. The GluA1–GluA3 subunits predominate in the adult forebrain and are well characterized. However, we lack structural information on full-length GluA4-containing AMPARs, a subtype that has specific roles in brain development and specific cell types in mammals. Here we present the cryo-electron microscopy structures of rat GluA4:TARP-γ2 trapped in active, resting and desensitized states, covering a full gating cycle. Additionally, we describe the structure of GluA4 alone, which displays a classical Y-shaped conformation. In resting conditions, GluA4:TARP-γ2 adopts two conformations, one resembling the desensitized states of other GluA subunits. Moreover, we identify a regulatory site for TARP-γ2 in the ligand-binding domain that modulates gating kinetics. Our findings uncover distinct features of GluA4, highlighting how subunit composition and auxiliary proteins shape receptor structure and dynamics, expanding glutamatergic signaling diversity. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Data and materials supporting the insights within this paper can be obtained from the corresponding author upon reasonable request. Coordinates and maps were deposited to the PDB and EM Data Bank under the following accession codes: GluA4rest TMD:LBD, 9QPW and EMD-53285; GluA4:γ2rest class I TMD:LBD, 9QDN and EMD-53031; GluA4:γ2rest class I NTD, 9IGZ and EMD-52862; GluA4:γ2rest class II TMD, 9RNH and EMD-54092; GluA4:γ2rest class II TMD:LBD, 9RMS and EMD-54065; GluA4:γ2open TMD:LBD, 9RMW and EMD-54069; GluA4:γ2des TMD, 9RN7 and EMD-54083; GluA4:γ2des TMD:LBD, 9RN4 and EMD-54080. Source data are provided with this paper. Hansen, K. B. et al. Structure, function, and pharmacology of glutamate receptor ion channels. Sobolevsky, A. I., Rosconi, M. P. & Gouaux, E. X-ray structure, symmetry and mechanism of an AMPA-subtype glutamate receptor. García-Nafría, J., Herguedas, B., Watson, J. F. & Greger, I. H. The dynamic AMPA receptor extracellular region: a platform for synaptic protein interactions. Sun, Y. et al. Mechanism of glutamate receptor desensitization. Armstrong, N. & Gouaux, E. Mechanisms for activation and antagonism of an AMPA-sensitive glutamate receptor: crystal structures of the GluR2 ligand binding core. Frydenvang, K., Pickering, D. S. & Kastrup, J. S. Structural basis for positive allosteric modulation of AMPA and kainate receptors. Chen, L., Durr, K. L. & Gouaux, E. X-ray structures of AMPA receptor–cone snail toxin complexes illuminate activation mechanism. Kazi, R., Dai, J., Sweeney, C., Zhou, H.-X. & Wollmuth, L. P. Mechanical coupling maintains the fidelity of NMDA receptor-mediated currents. Bowie, D. Redefining the classification of AMPA-selective ionotropic glutamate receptors. Cull-Candy, S., Kelly, L. & Farrant, M. Regulation of Ca2+-permeable AMPA receptors: synaptic plasticity and beyond. Zhu, J. J., Esteban, J. & Malinow, R. Postnatal synaptic potentiation: delivery of GluR4-containing AMPA receptors by spontaneous activity. Schwenk, J. et al. Regional diversity and developmental dynamics of the AMPA-receptor proteome in the mammalian brain. Mosbacher, J. et al. A molecular determinant for submillisecond desensitization in glutamate receptors. Burnashev, N. et al. Calcium-permeable AMPA-kainate receptors in fusiform cerebellar glial cells. Kita, K. et al. Glua4 facilitates cerebellar expansion coding and enables associative memory formation. Pelkey, K. A. et al. Pentraxins coordinate excitatory synapse maturation and circuit integration of parvalbumin interneurons. Firth, S. I., Li, W., Massey, S. C. & Marshak, D. W. AMPA receptors mediate acetylcholine release from starburst amacrine cells in the rabbit retina. Seol, M. & Kuner, T. Ionotropic glutamate receptor GluA4 and T-type calcium channel Cav3.1 subunits control key aspects of synaptic transmission at the mouse L5B-POm giant synapse. Yang, Y. M. et al. GluA4 is indispensable for driving fast neurotransmission across a high-fidelity central synapse. Sia, G. M. et al. Interaction of the N-terminal domain of the AMPA receptor GluR4 subunit with the neuronal pentraxin NP1 mediates GluR4 synaptic recruitment. Boudkkazi, S. et al. A Noelin-organized extracellular network of proteins required for constitutive and context-dependent anchoring of AMPA-receptors. Soto, D. et al. Selective regulation of long-form calcium-permeable AMPA receptors by an atypical TARP, γ-5. Tomita, S. et al. Functional studies and distribution define a family of transmembrane AMPA receptor regulatory proteins. Miguez-Cabello, F. et al. AMPAR/TARP stoichiometry differentially modulates channel properties. Barad, Z., Shevtsova, O., Arbuthnott, G. W. & Leitch, B. Selective loss of AMPA receptors at corticothalamic synapses in the epileptic stargazer mouse. Zhang, D. et al. Structural mobility tunes signalling of the GluA1 AMPA glutamate receptor. Fukaya, M., Yamazaki, M., Sakimura, K. & Watanabe, M. Spatial diversity in gene expression for VDCCγ subunit family in developing and adult mouse brains. Herguedas, B. et al. Structure and organization of heteromeric AMPA-type glutamate receptors. Devi, S. P. S., Cheng, Y., Tomita, S., Howe, J. R. & Zhang, W. TARPs modulate receptor-mediated paired-pulse depression and recovery from desensitization. Tomita, S. et al. Stargazin modulates AMPA receptor gating and trafficking by distinct domains. Cais, O. et al. Mapping the interaction sites between AMPA receptors and TARPs reveals a role for the receptor N-terminal domain in channel gating. Klykov, O., Gangwar, S. P., Yelshanskaya, M. V., Yen, L. & Sobolevsky, A. I. Structure and desensitization of AMPA receptor complexes with type II TARP γ5 and GSG1L. Yelshanskaya, M. V., Li, M. & Sobolevsky, A. I. Structure of an agonist-bound ionotropic glutamate receptor. Meyerson, J. R. et al. Structural mechanism of glutamate receptor activation and desensitization. Sheardown, M. J., Nielsen, E., Hansen, A. J., Jacobsen, P. & Honor, T. 2,3-Dihydroxy-6-nitro-7-sulfamoyl-benzo(F)quinoxaline: a neuroprotectant for cerebral ischemia. Coombs, I. D. et al. Homomeric GluA2(R) AMPA receptors can conduct when desensitized. Herguedas, B. et al. Architecture of the heteromeric GluA1/2 AMPA receptor in complex with the auxiliary subunit TARP g8. Larsen, A. H., Perozzo, A. M., Biggin, P. C., Bowie, D. & Kastrup, J. S. Recovery from desensitization in GluA2 AMPA receptors is affected by a single mutation in the N-terminal domain interface. Ivica, J. et al. Proton-triggered rearrangement of the AMPA receptor N-terminal domains impacts receptor kinetics and synaptic localization. Möykkynen, T., Coleman, S. K., Semenov, A. & Keinänen, K. N-terminal domain modulates α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptor desensitization. Lomeli, H. et al. Control of kinetic properties of AMPA receptor channels by nuclear RNA editing. Herbrechter, R., Hube, N., Buchholz, R. & Reiner, A. Splicing and editing of ionotropic glutamate receptors: a comprehensive analysis based on human RNA-seq data. Greger, I. H., Akamine, P., Khatri, L. & Ziff, E. B. Developmentally regulated, combinatorial RNA processing modulates AMPA receptor biogenesis. Nakagawa, T., Wang, X. T., Miguez-Cabello, F. J. & Bowie, D. The open gate of the AMPA receptor forms a Ca2+ binding site critical in regulating ion transport. Yu, J. et al. Hippocampal AMPA receptor assemblies and mechanism of allosteric inhibition. Gangwar, S. P. et al. Modulation of GluA2–γ5 synaptic complex desensitization, polyamine block and antiepileptic perampanel inhibition by auxiliary subunit Cornichon-2. Zhao, Y., Chen, S., Yoshioka, C., Baconguis, I. & Gouaux, E. Architecture of fully occupied GluA2 AMPA receptor–TARP complex elucidated by cryo-EM. Twomey, E. C., Yelshanskaya, M. V., Grassucci, R. A., Frank, J. & Sobolevsky, A. I. Elucidation of AMPA receptor–Stargazin complexes by cryo-electron microscopy. Riva, I., Eibl, C., Volkmer, R., Carbone, A. L. & Plested, A. J. R. Control of AMPA receptor activity by the extracellular loops of auxiliary proteins. Dawe, G. B. et al. Distinct structural pathways coordinate the activation of AMPA receptor–auxiliary subunit complexes. Gratacos-Batlle, E., Yefimenko, N., Cascos-GarcÃa, H. & Soto, D. AMPAR interacting protein CPT1C enhances surface expression of GluA1-containing receptors. Cell Neurosci. Dürr, K. L. et al. Structure and dynamics of AMPA receptor GluA2 in resting, pre-open, and desensitized states. Robert, A. & Howe, J. R. How AMPA receptor desensitization depends on receptor occupancy. Coombs, I. D., MacLean, D. M., Jayaraman, V., Farrant, M. & Cull-Candy, S. G. Dual effects of TARP γ-2 on glutamate efficacy can account for AMPA receptor autoinactivation. Dawe, G. B. et al. Nanoscale mobility of the apo state and TARP stoichiometry dictate the gating behavior of alternatively spliced AMPA receptors. Baranovic, J. & Plested, A. J. R. Auxiliary subunits keep AMPA receptors compact during activation and desensitization. Partin, K. M., Patneau, D. K. & Mayer, M. L. Cyclothiazide differentially modulates desensitization of α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptor splice variants. Zhang, W., Devi, S. P. S., Tomita, S. & Howe, J. R. Auxiliary proteins promote modal gating of AMPA- and kainate-type glutamate receptors. Chen, S. et al. Activation and desensitization mechanism of AMPA receptor–TARP complex by cryo-EM. Herguedas, B. et al. Mechanisms underlying TARP modulation of the GluA1/2–γ8 AMPA receptor. Kumari, J. et al. Structural dynamics of the GluK3-kainate receptor neurotransmitter binding domains revealed by cryo-EM. Gangwar, S. P. et al. Kainate receptor channel opening and gating mechanism. Khanra, N., Brown, P. M. G. E., Perozzo, A. M., Bowie, D. & Meyerson, J. R. Architecture and structural dynamics of the heteromeric GluK2/K5 kainate receptor. Twomey, E. C., Yelshanskaya, M. V., Grassucci, R. A., Frank, J. & Sobolevsky, A. I. Channel opening and gating mechanism in AMPA-subtype glutamate receptors. Zhang, D., Watson, J. F., Matthews, P. M., Cais, O. & Greger, I. H. Gating and modulation of a hetero-octameric AMPA glutamate receptor. Carbone, A. L. & Plested, A. J. R. Coupled control of desensitization and gating by the ligand binding domain of glutamate receptors. Hawken, N. M., Zaika, E. I. & Nakagawa, T. Engineering defined membrane-embedded elements of AMPA receptor induces opposing gating modulation by Cornichon 3 and Stargazin. Twomey, E. C., Yelshanskaya, M. V., Grassucci, R. A., Frank, J. & Sobolevsky, A. I. Structural bases of desensitization in AMPA receptor–auxiliary subunit complexes. Nakagawa, T. Structures of the AMPA receptor in complex with its auxiliary subunit Cornichon. Hale, W. D. et al. Allosteric competition and inhibition in AMPA receptors. Twomey, E. C., Yelshanskaya, M. V., Vassilevski, A. & Sobolevsky, A. I. Mechanisms of channel block in calcium-permeable AMPA receptors. Yelshanskaya, M. V., Patel, D. S., Kottke, C. M., Kurnikova, M. G. & Sobolevsky, A. I. Opening of glutamate receptor channel to subconductance levels. Krintel, C. et al. Binding of a negative allosteric modulator and competitive antagonist can occur simultaneously at the ionotropic glutamate receptor GluA2. Salazar, H., Mischke, S. & Plested, A. J. R. Measurements of the timescale and conformational space of AMPA receptor desensitization. Ben-Yaacov, A. et al. Molecular mechanism of AMPA receptor modulation by TARP/Stargazin. Zhao, Y., Chen, S., Swensen, A. C., Qian, W.-J. & Gouaux, E. Architecture and subunit arrangement of native AMPA receptors elucidated by cryo-EM. Fuchs, E. C. et al. Recruitment of parvalbumin-positive interneurons determines hippocampal function and associated behavior. Geiger, J. R. P. et al. Relative abundance of subunit mRNAs determines gating and Ca2+ permeability of AMPA receptors in principal neurons and interneurons in rat CNS. Matsui, K., Jahr, C. E. & Rubio, M. E. High-concentration rapid transients of glutamate mediate neural-glial communication via ectopic release. Smart, O. S., Neduvelil, J. G., Wang, X., Wallace, B. & Sansom, M. S. P. HOLE: a program for the analysis of the pore dimensions of ion channel structural models. García-Nafría, J., Watson, J. F. & Greger, I. H. IVA cloning: a single-tube universal cloning system exploiting bacterial in vivo assembly. Leonte, R. C., Fischer, P. D., Herguedas, B., García-Nafría, J. & Hiller, S. IVA Prime: automated primer design for in vivo assembly cloning. Nucleic Acids Res. Soto, D., Coombs, I. D., Kelly, L., Farrant, M. & Cull-Candy, S. G. Stargazin attenuates intracellular polyamine block of calcium-permeable AMPA receptors. Goehring, A. et al. Screening and large-scale expression of membrane proteins in mammalian cells for structural studies. Production of recombinant transmembrane proteins from mammalian cells for biochemical and structural analyses. Aricescu, A. R., Lu, W. & Jones, E. Y. A time- and cost-efficient system for high-level protein production in mammalian cells. Article PubMed Kimanius, D., Dong, L., Sharov, G., Nakane, T. & Scheres, S. H. W. New tools for automated cryo-EM single-particle analysis in RELION-4.0. Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Rohou, A. & Grigorieff, N. CTFFIND4: fast and accurate defocus estimation from electron micrographs. Bepler, T. et al. Positive-unlabeled convolutional neural networks for particle picking in cryo-electron micrographs. Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A.cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Orain, D. et al. 6-Amino quinazolinedione sulfonamides as orally active competitive AMPA receptor antagonists. Burnley, T., Palmer, C. M. & Winn, M. Recent developments in the CCP-EM software suite. Yamashita, K., Palmer, C. M., Burnley, T. & Murshudov, G. N. Cryo-EM single-particle structure refinement and map calculation using Servalcat. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. A. et al. REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use. Article PubMed Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Chen, V. B. et al. MolProbity: all-atom structure validation for macromolecular crystallography. Dutta, A., Shrivastava, I. H., Sukumaran, M., Greger, I. H. & Bahar, I. Comparative dynamics of NMDA- and AMPA-glutamate receptor N-terminal domains. Krintel, C. et al. Thermodynamics and structural analysis of positive allosteric modulation of the ionotropic glutamate receptor GluA2. Krintel, C. et al. l-Asp is a useful tool in the purification of the ionotropic glutamate receptor A2 ligand-binding domain. Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. C.V.-G. is funded by grant PRE2020-092922, MICIN/AEI 0.13039/501100011033 and ‘ESF Investing in your future'. is funded by grants PID2019-106284GA-I00, PID2022-140185NB-I00 and RYC2018-025720-I, from MICIN/AEI 0.13039/501100011033 and ‘ESF Investing in your future' and ‘ERDF/EU'. J.P.-P. is funded by FPU22/02071 predoctoral fellow by the Spanish Ministry of Universities. is supported by grant PID2020-119932GB-I00 funded by MCIN/AEI/10.13039/501100011033 (MINECO). We thank J. García-Nafría and R. Fernández-Leiro for help with establishing the cryo-EM software at BIFI cluster and J. Diaz and D. Martínez for IT support. We thank J. F. Watson and J. García-Nafría for helpful comments on the paper. We thank I. Greger for the GluA4 PRK5 plasmid and for helpful discussions during project development. This work benefited from access to the Netherlands Center for Electron Nanoscopy (NeCEN) at Leiden University and was supported by iNEXT-Discovery project number 871037, funded by the Horizon 2020 program of the European Commission. We thank L. Renault and W. Noteborn for assistance during cryo-EM data collection at NeCEN. We thank the Basque Resource for EM (BREM) supported by the IKUR 2023 strategy of the Department of Science, Universities and Innovation and the Innovation Fund of the Basque Government, with additional support from MCIN (Recovery, Transformation and Resilience Plan) and the Basque Government ‘Biotechnology Complementary Plan Applied to Health' with funding from European Union NextGenerationEU (PRTR-C17.I1; PRTR-C17.I01.P01.S13; AAAA_ACG_AY_2539/22_05). We thank D. Gil and K. Barragan for assistance during cryo-EM data collection at BREM. We thank P. Guerra from the IBMB-CSIC cryo-EM Platform for assistance during cryo-EM data collection and funding from grant IU16-014045 (cryo-TEM) from Generalitat de Catalunya and ICTS ALBA for funding proposal EM 2023076001. We thank the ESRF and R. Linares for assistance during data collection (proposal MX2443 on CM01 beamline) and J. Martín Benito for ESRF BAG coordination. We thank R. Núñez from the CIB Microscopy facility, J. Boskovic from the CNIO Microscopy facility and the ‘servicio de apoyo a la investigación' from the University of Zaragoza for support in cryo-EM facilities. Institute for Biocomputation and Physics of Complex Systems (BIFI), Department of Biochemistry and Molecular and Cell Biology, University of Zaragoza, Zaragoza, Spain Carlos Vega-Gutiérrez, Irene Sánchez-Valls, Victoria del Pilar Ribón-Fuster & Beatriz Herguedas Laboratory of Neurophysiology, Department of Biomedicine, Faculty of Medicine and Health Sciences, Institute of Neurosciences, University of Barcelona, Barcelona, Spain Javier Picañol-Párraga & David Soto August Pi i Sunyer Biomedical Research Institute (IDIBAPS), Barcelona, Spain Laboratorio de Microscopías Avanzadas (LMA), University of Zaragoza, Zaragoza, Spain Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar C.V.-G. designed the protein constructs and mutants, and conducted the protein purification, cryo-EM data collection, structure determination, structure analysis and crosslinking experiments. designed, performed and analyzed the electrophysiological recordings. I.S.-V designed the baculovirus constructs and optimized the baculovirus production and protein expression methods. V.d.P.R.-F. cloned the mutants and optimized the cysteine crosslinking protocols. C.V.-G prepared the figures with contributions from all authors. conceptualized and supervised the study, processed and analyzed the structural data, designed the mutants, and wrote the paper with contributions from all authors. Correspondence to Beatriz Herguedas. The authors declare no competing interests. Nature Structural & Molecular Biology thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Primary Handling Editor: Katarzyna Ciazynska, in collaboration with the Nature Structural & Molecular Biology team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a-h, Box-and-whisker plots showing: the 25–75th percentiles represented by boxes, the minimum/maximum values specified by whiskers, median values indicated as a line inside the box and mean average values denoted as crosses. Circles, squares and triangles represent the individual value experiments for GluA4 (box-and-whisker shown in white), GluA4:γ2 (box-and-whisker in dark green) and the structural variant GluA4:γ2st (box-and-whisker in light green), respectively. All data comes from at least three independent transfections. Kinetic parameters as deactivation time constant to a 1 ms pulse (a) (p = 0.0094; p = 0.0482 and p = 0.1504 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; all two-tailed Student' t-test), desensitization time constant to a 100 ms pulse (b) (p < 0.0001; p < 0.0001 and p = 0.3547 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; two-tailed Student' t-test), percentage of current at steady state measured at 90 ms time after desensitization (c) (p = 0.0048; p = 0.0018 and p = 0.5287 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; all two-tailed Mann-Whitney U-test), recovery from desensitization time constant calculated from paired pulses of 20 ms (d) (p = 0.0177; p = 0.0010 and p = 0.3701 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively) or time to peak (e) (p = 0.5441; p = 0.0764 and p = 0.2435 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; all two-tailed Student' t-test) where calculated from 4 to 25 responses in out-side out patches from tsA201 cells expressing the different conditions activated by 10 mM glutamate rapid applications at -60 mV holding potential. Rectification index ratio calculated from currents at +60 mV divided currents at -60mV (f) in the same conditions (p = 0.0022; p = 0.0012 and p = 0.7985 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; all two-tailed Mann-Whitney U-test). Single channel conductance (g) (p = 0.0004; p = 0.0013 and p = 0.3066 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; all two-tailed Student' t-test) and peak open probability (h) (p = 0.3848; p = 0.5424 and p = 0.9868 for GluA4 vs. GluA4:γ2, GluA4 vs. GluA4:γ2st and GluA4:γ2 vs. GluA4:γ2 st, respectively; all two-tailed Student' t-test) calculated from non-stationary fluctuation analysis from typically 40 to 150 responses to 10 mM glutamate of patches in the same conditions as previously stated. Student' t-test was performed for parametric data while Mann Whitney U-test were done for non-parametric data. a, Cryo-EM data processing workflow of GluA4rest, from movie processing to 3D maps. Processing steps were carried out with Relion 4.1 (black) or CryoSPARC (blue). Squared maps correspond to the final maps used for modeling, obtained with Relion PostProcess. b, Left: A stain-free SDS-PAGE gel displaying a band corresponding to purified GluA4 sample at approximately 100 kDa. Right: Motion-corrected micrograph (defocus value -1.7 μm). For the selection of the micrograph, we automatically selected all the micrographs with defocus values between -1.7 to -2.1 μm and selected a micrograph with intermediate density of particles, as depending on ice thickness we obtain some micrographs with denser or more sparse particle distribution. The density of particles on micrographs varied with ice thickness, with micrographs in thin ice presenting a denser particle distribution and micrographs in thick ice presenting less particles. Scale bar corresponds to 50 nm. c, Representative 2D class averages obtained with the particles of the final refined model (59,714 particles). The 2D classes excluded correspond to noisy classes due to a reduced number of particles (the selected 16 classes correspond to 56,319 particles). d, Particle distribution of the TMD/LBD final map. a, Full-length models of GluA4rest and GluA2cryst (PDB 4U4G, GluA2 bound to antagonist ZK 200775)33. NTD and LBD layers are more separated in GluA4 than in GluA2cryst. b, Superposition of GluA4 (cylindrical helices) on GluA2cryst (represented as surface in red). The TMD M1-M3 helices have been used for alignment. c, Top view onto the NTD layer of GluA4 (cylindrical helices) on GluA2cryst (represented as surface in red) featuring the displacement and rotation of the entire NTD layer with respect to GluA2. d, Comparison of the LBD “gating ring” in GluA4rest, native AMPARs (PDB 7LEP)45, GluA2cryst (PDB 4U4G) and WT GluA2:γ2 (PDB 8FPK)44. In yellow, the distance between opposing Ala687 residues is highlighted (Ala665 in GluA2). a, Left: Motion-corrected micrograph (defocus value -1.8 μm). For the selection of the micrograph, we automatically selected all the micrographs with defocus values between -1.7 to -2.1 μm and selected a micrograph with intermediate density of particles, as depending on ice thickness we obtain some micrographs with denser or more sparse particle distribution. Scale bar corresponds to 50 nm. Right: A stain-free SDS-PAGE gel displaying a band corresponding to purified GluA4:γ2 at approximately 130 kDa. b, 2D class averages obtained with the subset of particles from GluA4:γ2rest class I (top) and class II (bottom). For Class I, the 296,802 particles in class I were imported into Relion for 2D classification. All the main 2D orientations are included, and excluded classes correspond to more top views or particles with a reduced number of particles. For Class II, the 266,467 particles of Class II were imported into relion for 2D classification. The classes represent all the views, with aditional top views excluded. c, Particle distribution of the final TMD/LBD maps corresponding to class I and class II. d, Cryo-EM data processing workflow of GluA4:γ2rest, from movie processing to 3D maps. Processing steps were carried out with Relion 5.0 (black) or CryoSPARC (blue). Squared maps correspond to the final maps used for modeling, obtained with CryoSPARC or Relion PostProcessing. a, Top view onto the LBD gating ring of GluA4 (pink) and GluA4:γ2rest (purple). The TMD helices M1, M2 and M3 have been selected for alignment. A clockwise 16° rotation of the entire LBD gating ring around the C2 axis is observed. b, Zoom into the LBD dimer showing the extracellular TARP elements involved in the LBD-TARP interaction. c, Comparison of the TMD domain featuring the differences in M4 helix as well as the palmitoylation modification at Cys837. d, Comparison of the palmitate binding site in different AMPAR complexes. GluA4:γ2 and WT GluA2:γ2 are palmitoylated (or their cryo-EM maps contain densities compatible with palmitoylation) and, consequently, the C-terminal of helix M4 is displaced towards TARP. a, b, Comparison of currents elicited by Glutamate (black), CNQX (blue) and NBQX (cyan). As expected, NBQX does not elicit currents. c, Average of representative peak responses to rapid 1 ms applications of 10 mM glutamate recorded from outside-out patches of HEK293T cells expressing GluA2:γ2, in the absence (left trace) or presence (right trace) of 50 μM cyclothiazide (CTZ). Note that, in the CTZ condition, the patches were jumped from a control solution containing CTZ to a glutamate solution containing the same concentration of CTZ. d, Same as in c, but for GluA4:γ2, where an enhancement of the response is clearly observed (124% of the amplitude measured in the presence of CTZ in this example). e, Pooled data showing the effect of CTZ on peak response amplitude. Box-and-whisker plots indicate the median (line), 25–75th percentiles (box), and minimum–maximum values (whiskers); crosses denote the mean. Individual values are shown as symbols; n = 7 (GluA2:γ2) and n = 8 (GluA4:γ2). *p = 0.0368 (two-tailed Student's t-test). f, Western Blot of GluA4 WT and the variants lacking exposed cysteines: A4 1Cys(C213S), A2 2Cys (C185S/C213S), A4 3Cys (C153S/C185/C213S), A4 4Cys (C153S/C185/C213S/C837S) and A4 5Cys (C153S/C185/C213S/C837S/C878S). Although a reduction of protein expression is observed, the GluA4 5Cys variant expression is adequate for crosslinking experiments. The expression of these variants was evaluated in two independent experiments. g, top view of the LBD layer with the position of the interface cysteines S751C and S698C, represented as yellow spheres. Note that the native cysteines involved in disulfide bonds are also presented as spheres. h-j, Western blots of the cysteine mutants obtained in non-reducing conditions in resting (NBQX), desensitized (L-Glu) and open (CTZ + L-Glu) conditions. Note that S751C forms dimers in all conditions, as a result of its position in the LBD dimer interface. The introduction of two cysteines generates high molecular weight oligomers, which are enhanced in desensitizing conditions. For the cross-linking experiments conducted in the presence of NBQX or CTZ and L-Glutamate, the amount sample loaded for the double mutant was twice that of the single mutants and wild-type. In contrast, for the cross-linking performed in the presence of L-Glutamate alone, equal amounts of all samples were loaded onto the gel. Each crosslinking experiment was performed at least three times. Below each gel, the relevant 2D class averages representing classes with two dimeric LBDs and classes with one ruptured LBD are shown. a, Cryo-EM data processing workflow of GluA4:γ2des, from movie processing to 3D maps. Processing steps were carried out with Relion 5.0 (black) or CryoSPARC (blue). Squared maps correspond to the final maps used for modeling, obtained with CryoSPARC. b, Motion-corrected micrograph (defocus value -1,7 μm). To select the micrograph, an automatic selection was performed based on defocus values (-1.7 μm to -2.1 μm) followed by manual selection of a micrograph with an intermediate density of particles. Scale bar corresponds to 50 nm. c, 2D class averages, obtained after import into Relion of the 137,802 particles from the consensus TMD/LBD map. d, Particle distribution of the final TMD/LBD map. a, Fitting of GluA4:γ2des (cartoon model in red) onto the un-sharpened map of GluA4:γ2rest class II (light grey). GluA4:γ2rest class II adopts an open-cleft conformation with bound NBQX, while of GluA4:γ2des cleft is closed. b, Superposition of an LBD monomer in GluA4:γ2des and GluA4:γ2rest class II, highlighting the cleft closure difference. c, Superposition of M3 helices in GluA4:γ2des and GluA4:γ2rest, featuring the similarities in M3 helices and M3-S2 linkers between desensitized and class II states. d, Surface representation of the LBD gating ring in various desensitized state structures. GluA2 receptors present stable dimer interfaces in homomers and in heteromers with the GluA1 subunit. GluA1 LBD dimers are fully ruptured under desensitizing conditions. GluA4-containing AMPARs show a partial rupture of LBD layer, with a dimer in a GluA2-like conformation and two monomers. a, Cryo-EM data processing workflow of GluA4:γ2open, from movie processing to 3D maps. Processing steps were carried out with Relion 5.0 (black) or CryoSPARC (blue). Squared maps correspond to the final maps used for modeling, obtained with CryoSPARC or Relion PostProcess. b, Motion-corrected micrograph (scale bar 50 nm, defocus -2 μm). To select the micrograph, an automatic selection was performed based on defocus values (-1.7 μm to -2.1 μm) followed by manual selection based on the number of particles. Scale bar corresponds to 50 nm. c, 2D class averages obtained after importing into Relion of the 117,558 particles from the consensus TMD/LBD map. d, Particle distribution of the final TMD/LBD map. Local resolution of the different maps of GluA4rest, GluA4:γ2rest class I, GluA4:γ2rest class II, GluA4:γ2des and GluA4:γ2open and Fourier Shell Correlation of the half maps plotted as a function of spatial frequencies. Resolution estimates based on the FSC = 0.143 threshold criteria are indicated. Oligonucleotides used for site-directed mutagenesis. Comparison of the TMD:LBD models of GluA4:TARP-γ2 complexes in different functional states. Numerical source data files for electrophysiological recordings. Numerical source data files for electrophysiological recordings. Numerical source data files for electrophysiological recordings. Numerical source data files for electrophysiological recordings. Unprocessed western blots. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Vega-Gutiérrez, C., Picañol-Párraga, J., Sánchez-Valls, I. et al. GluA4 AMPA receptor gating mechanisms and modulation by auxiliary proteins. Nat Struct Mol Biol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Structural & Molecular Biology (Nat Struct Mol Biol) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03965-4'>Primary care detection of Alzheimer's disease using a self-administered digital cognitive test and blood biomarkers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 10:07:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. After the clinical implementation of amyloid-β-targeting therapies for people with cognitive impairment due to Alzheimer's disease (AD), there is an urgent need to efficiently identify this patient population in primary care. Therefore, we created a brief and self-administered digital cognitive test battery (BioCog). Based on its sub-scores, a logistic regression model was developed in a secondary care cohort (n = 223) and then evaluated in an independent primary care cohort comprising 19 primary care centers (n = 403). In primary care, BioCog had an accuracy of 85% when using a single cutoff to define cognitive impairment, which was significantly better than the assessment of primary care physicians (accuracy 73%). The accuracy increased to 90% when using a two-cutoff approach. BioCog had significantly higher accuracy than standard paper-and-pencil tests (that is, Mini-Mental State Examination, Montreal Cognitive Assessment, Mini-Cog) and another digital cognitive test. Furthermore, BioCog combined with a blood test could detect clinical, biomarker-verified AD with an accuracy of 90% (one cutoff), significantly better than standard-of-care (accuracy 70%) or when using the blood test alone (accuracy 80%). In conclusion, this proof-of-concept study shows that a brief, self-administered digital cognitive test battery can detect cognitive impairment and, when combined with a blood test, accurately identify clinical AD in primary care. AD is the most common cause of dementia, characterized by depositions of amyloid-β (Aβ), aggregated tau and progressive neurodegeneration1. The clinical syndrome of AD typically starts with subjective cognitive decline (SCD), in which individuals report memory problems and/or other cognitive difficulties but perform normally on cognitive tests. Diagnosing AD is challenging, especially in the early stages of the disease and in primary care. The prevalence of both underdiagnosis and misdiagnosis is high with 20–30% of people being misdiagnosed in specialized care and approximately 40% in primary care when the AD diagnosis is not supported by biomarkers3,4,5,6. The implementation of biomarker testing such as positron emission tomography (PET) with Aβ aggregate-binding tracer or by lumbar puncture measuring concentrations of Aβ and tau in the cerebrospinal fluid (CSF) has improved diagnostic precision in specialized clinics7,8,9. Nevertheless, these methods are either costly or invasive, and the availability is very limited globally. Recently, blood tests based on tau phosphorylated at threonine 217 (phosphorylated-tau217) have been shown to detect AD pathology with accuracies of around 90% in both primary and secondary care6, demonstrating performances comparable to clinically used CSF tests10. Therefore, blood tests for AD have the potential to revolutionize the diagnostic work-up of AD in people with cognitive symptoms, especially in primary care centers9,11. The prevalence of AD brain pathology in the population varies by age and apolipoprotein E (APOE) status, but even more so by the severity of cognitive symptoms11. SCD can arise from a variety of causes, resulting in approximately only 20–25% of individuals with SCD exhibiting AD pathology. As a result, the pretest probabilities vary by cognitive stage, resulting in lower positive predictive values (PPV) of AD biomarkers in people with SCD compared to those with dementia11,12. Consequently, a positive AD blood test in an individual with SCD carries a higher risk of being a false-positive than a positive result for an individual with cognitive impairment (MCI or dementia)6. The implications of such a false-positive result are substantial, because a person may be incorrectly informed of having AD associated with progressive cognitive decline in the coming years. Furthermore, only individuals with AD with cognitive impairment (but not SCD) are eligible for treatment with the newly implemented Aβ-targeting immunotherapies1,13,14,15. Biomarker testing in clinical practice is currently only recommended in patients with cognitive impairment1,3,13,16. This underscores the importance of identifying objective impairment (MCI and dementia) in individuals with cognitive symptoms, which in international guidelines is recommended to always precede AD biomarker testing5,11,17,18. The administration and interpretation of cognitive tests vary substantially among primary care providers, both in and between countries. In addition, primary care providers often lack the time, expertise or resources to conduct assisted, in-clinic, cognitive tests19. Brief yet accurate digital cognitive testing, performed either remotely or unsupervised at the primary care center, may offer a viable solution to overcome this gap. We therefore developed a digital, self-administered, cognitive test battery (BioCog) and examined its ability to accurately identify cognitive impairment among people presenting with cognitive symptoms in primary care in a proof-of-concept study. Furthermore, we explored its utility when combined with a blood test to identify individuals with cognitive impairment due to AD. We included 223 participants with cognitive symptoms from a secondary care cohort: the ongoing BioFINDER-2 study (NCT03174938) (Table 1)20. age was 73 (8.8) years, 128 (57%) were male and 119 (53%) had objectively verified cognitive impairment. In addition, 403 participants were included from a primary care cohort, BioFINDER-Primary Care study (NCT06120361) (Table 1 and Supplementary Fig. 1), which recruits people seeking help for cognitive symptoms at 19 primary care units6. Only people for whom the primary care physician (PCP) thought neurodegenerative disease was a reasonably possible cause of the symptomatology were invited to participate in the study. age of the primary care cohort was 77 (8.0) years, 194 (48%) were male and 229 (57%) had objectively verified cognitive impairment. Compared with secondary care, participants in the primary care cohort were older, had fewer years of education, a higher prevalence of cardiovascular disease and hyperlipidemia, and a larger proportion were female. A slightly smaller proportion of participants in primary care were Aβ-positive and APOE ε4 carriers. Similar proportions were cognitively impaired (Table 1). All participants performed the self-administered digital test battery, BioCog, comprising: (1) a word list test with immediate recall, delayed recall and recognition of ten words; (2) a cognitive processing speed task; and (3) questions about orientation to time (Supplementary Fig. The study's primary outcome was objectively verified cognitive impairment determined on the basis of a standard battery of cognitive tests, the Repeatable Battery for the Assessment of Neuropsychological Status (RBANS), administered by neuropsychologists at a memory clinic. The secondary care cohort was used for training to establish cognitive test models and cutoffs. The primary care cohort was used for validation where the models and cutoffs were applied. The subtests of BioCog demonstrated moderate negative Spearman's correlations with age (−0.31 to −0.58), and weak positive correlations with education (0.13 to 0.33). Performance across subtests did not differ by sex in either cohort, except for immediate recall and recognition in the primary care cohort, where female participants performed slightly better (Supplementary Tables 1 and 2). Strong correlations (0.57 to 0.83) were observed between BioCog subtests and their corresponding paper-and-pencil analogs (for example, BioCog processing speed test and Symbol Digit Modalities Test), supporting convergent validity. Correlations between individual BioCog subtests and a short-term memory task were weak (0.11 to 0.29). Similarly, only weak correlations were observed between BioCog subtests and a measure of visuoperceptual or spatial functioning (0.13 to 0.27), indicative of divergent validity (Supplementary Tables 3 and 4). Participants with cognitive impairment scored significantly lower on all BioCog measures compared to cognitively unimpaired individuals (Supplementary Tables 5 and 6). Internal consistency of the subtests, estimated by McDonald's omega (ω), a reliability coefficient, ranged from acceptable to excellent (0.70 to 0.90) (Supplementary Table 7). Self-reported difficulty in understanding test instructions was very low (2%), and most participants reported that the tasks were engaging or were neutral (word list test 80%; processing speed test 91%) (Supplementary Fig. Average completion time for BioCog was 11.2 min (s.d. Overall, these results suggest that BioCog is a valid, reliable and feasible tool for assessing cognitive function. In the secondary care cohort, multiple logistic regression models that could predict the presence or absence of cognitive impairment were established using basic demographics (three variables: age, sex and education level) and BioCog (six variables from the subtests), in total nine variables (Supplementary Table 9). When creating models, recursive variable selection was performed to decrease the risk of overfitting. The model with lowest AIC included six variables (BioCog6): delayed ten-word recall (number of correct answers), cognitive processing speed task (number of correct answers), three repetitions of immediate ten-word recall (number of correct answers), age, delayed ten-word recognition (number of correct answers) and three repetitions of immediate ten-word recall (total time); listed in the order of feature importance. For comparison, receiver operating characteristic (ROC) curves from BioCog6 (area under the curve (AUC) = 0.96) together with models containing all variables (BioCog9, AUC = 0.96, in addition including orientation to time (correct answers), education level and sex), or only the best performing single subtest (BioCog1, AUC = 0.91, only delayed ten-word recall), are provided in Fig. Comparisons of true versus predicted classes are shown in a cross-tabulation analysis (Fig. Model performances in the secondary care cohort (which was also used to fit the models and to establish the cutoff), n = 223. a, ROC curves, AUCs and AICs for the three BioCog models using one (BioCog1), six (BioCog6) or nine (BioCog9) input variables. b, Evaluation of BioCog6 selecting an optimal cutoff point based on the maximum value of Youden's index (probability = 0.575). Error bars indicate 95% CI, with the center point corresponding to the mean value. To enhance diagnostic performance, a two-cutoff approach (as previously proposed3,6,9,21,22,23) was applied using BioCog6. This way, participants with predicted probabilities close to the one-cutoff decision boundary formed an intermediate group and were not considered when evaluating performance (representing individuals who would need further evaluation to determine their cognitive status). The two-cutoff probabilities of 0.332 and 0.769 were chosen to achieve 95% sensitivity and 95% specificity, respectively. Next, we evaluated the predictive ability of the previously established BioCog models and cutoffs in the independent primary care cohort. Of the 96 cognitively impaired participants without AD, 82 were correctly identified as impaired by the BioCog6 model (85% accuracy), suggesting similar performance in this subpopulation. The BioCog6 model was compared to standard assessment of PCPs (PCPcog), which included brief cognitive testing with Mini-Mental State Examination (MMSE) and The Montreal Cognitive Assessment (MoCA), computed tomography scan of the brain and a clinical assessment of the participant. BioCog6 had significantly higher performance across all evaluation metrics (Fig. Comparisons of true versus predicted classes for BioCog6 and PCPcog assessments can be seen in cross-tabulation analyses (Fig. Model performances in the primary care cohort (which was not used to fit the models or to establish the cutoff). a, ROC curves, AUCs and AICs for the three BioCog models using one (BioCog1), six (BioCog6) or nine (BioCog9) input variables, n = 403. b, Evaluation of BioCog6 based on the optimal cutoff point established in the secondary care cohort (probability = 0.575), n = 365. Compared to a PCP diagnosis of SCD versus MCI or dementia (PCPcog). Error bars indicate 95% CI, with the center point corresponding to the mean value. CI and two-sided P values were computed using bootstrapping (n = 5,000 resamples with replacement) and adjusted for multiple comparisons (Benjamini–Hochberg). d, Cross-tabulation analysis using PCPcog diagnoses. Comparisons in b–d were made on a subset of individuals with existing PCPcog evaluation data (n = 365). Exact P values in b: PAccuracy = 0.0007, PPPV = 0.001, PNPV = 0.0007, PSpecificity = 0.005, PSensitivity = 0.0007. Nominal significance <0.0001 was set to 0.0001 during FDR correction, yielding a lowest adjusted P value of 0.0007. Error bars indicate 95% CI, with the center point corresponding to the mean value. b, Cross-tabulation analysis using the BioCog6 two-cutoff model. c, Histogram of the predicted probabilities from the BioCog6 model (x axis) colored by the true binary RBANS composite score (outcome variable). Next, a head-to-head comparison was performed between the BioCog6 model and several widely used standard paper-and-pencil tests (MMSE, MoCA, Mini-Cog) and another digital test (Cambridge Neuropsychological Test Automated Battery (CANTAB)). Using one cutoff, the BioCog6 model significantly outperformed all the other cognitive tests on accuracy (Table 2). This result remained consistent when cognitive tests were treated as continuous variables and adjusted for demographics (Supplementary Fig. It also remained when removing age as an input in the BioCog6 model (Supplementary Table 10). For the two-cutoff approach, BioCog6 demonstrated significantly higher accuracy than MoCA, the only other test adapted for a two-cutoff approach (90% versus 76%, P = 0.001), and with a smaller, but not significantly different intermediate group size (18% versus 23%, P = 0.07). Finally, we simulated a two-step diagnostic workflow in primary care that can be performed in addition to the standard clinical evaluation made by a physician to identify biomarker-verified clinical (symptomatic) AD: step 1, detection of objective cognitive impairment using BioCog (BioCog6); and step 2, blood biomarker testing (PrecivityAD2; Amyloid Probability Score-2 (APS2)) in those who are cognitively impaired according to BioCog (Fig. We compared this workflow in the primary care cohort against current standard clinical evaluation by PCPs, where the physician first assessed whether the patient had cognitive impairment and second, whether the impairment was caused by AD (PCPAD). Comparisons were made on a subset of individuals with all existing data available (n = 365; see Supplementary Table 17 for detailed population characteristics). a, Our proposed primary care two-step workflow consisting of step 1, detection of cognitive impairment using the BioCog, followed by step 2 in cognitively impaired individuals, a blood biomarker assessment to evaluate whether AD pathology is present. b, Evaluation of the workflow using our BioCog6 model for step 1 and the plasma biomarker APS2 for step 2 (green). The workflow was compared against a standard clinical evaluation by PCPs in which the physician assesses both whether the patient had cognitive impairment (MCI or dementia) and whether the impairment was caused by AD (without any biomarkers) (PCPAD, orange), and against a workflow using only the plasma biomarker APS2 without any cognitive assessment (red). Error bars indicate 95% CI, with the center point corresponding to the mean value. Significance comparisons between all workflows are given in Supplementary Table 11. c–e, Corresponding cross-tabulation analyses for BioCog and blood marker workflow (c), PCP workflow (d) and blood biomarker only workflow (e). Credit: Images in a adapted from NIAID NIH BIOART (https://bioart.niaid.nih.gov). When cognitive impairment (based on neuropsychological assessment) and a clinical AD diagnosis (based on a consensus diagnosis done by dementia experts including CSF analysis or Aβ-PET assessments) was used as outcome, the two-step diagnostic workflow of BioCog (one cutoff) and blood biomarker achieved an overall clinical AD diagnostic accuracy of 90% (95% CI 86–92%), PPV of 83% (95% CI 75–89%), NPV of 93% (95% CI 89–95%), specificity of 92% (95% CI 88–95%) and sensitivity of 84% (95% CI 77–90%) (Fig. By contrast, PCPs had an overall accuracy of 70% (95% CI 65–74%), PPV of 52% (95% CI 43–61%), NPV of 79% (95% CI 73–84%), specificity of 76% (95% CI 71–81%) and sensitivity of 56% (95% CI 47–66%), all significantly lower than the new digital test and blood biomarker workflow (Fig. In comparison, relying solely on the blood biomarker to identify clinical AD without using BioCog maintained a high sensitivity of 92% (95% CI 86–96%) and NPV of 95% (95% CI 92–98%), but showed significantly lower accuracy of 80% (95% CI 76–84%), PPV of 63% (95% CI 55–69%) and specificity of 75% (95% CI 69–80%) compared to the combined BioCog and blood biomarker workflow (Fig. Notably, a high proportion (17%) of individuals were classified as false positives when only using the blood test (Fig. Using the BioCog6 and APS2 workflow resulted in significantly higher accuracy than combining APS2 with the traditional cognitive assessments MMSE and MoCA (Supplementary Table 12). If using a two-cutoff approach for both BioCog and the blood biomarker, the accuracy increased to 95% (95% CI 91–97%), with 30% (95% CI 25–34%) of individuals in the intermediate group (Supplementary Fig. All results were similar when using cognitive impairment and an abnormal Aβ CSF or PET biomarker as outcome instead of a clinical AD diagnosis (Supplementary Fig. 5), and when applying the clinically used APS2 cutoff (Supplementary Fig. A Clinical Dementia Rating (CDR) global score of ≥0.5 was evaluated as the reference standard for cognitive impairment instead of RBANS. These results highlight the robustness and generalizability of the BioCog6 model in predicting cognitive impairment, also when using a different reference standard. Additional evaluation metrics for the main analyses in the primary care cohort (including balanced accuracy, harmonic mean of precision and recall (F1 score), and positive and negative likelihood ratios) (Supplementary Tables 14 and 15) supported our main findings. This study demonstrated that a brief digital cognitive assessment battery detects cognitive impairment with an accuracy of 85% when validated in an elderly primary care population presenting with cognitive symptoms. The accuracy of the self-administered BioCog test was significantly higher than that of standard-of-care methods, including paper-and-pencil tests MMSE, MoCA and Mini-Cog, which require the presence of healthcare personnel when administered. The digital test on its own also outperformed clinical evaluation by the PCP, which included patient history and standard cognitive tests. Notably, higher performance was consistent whether cognitive impairment was defined using RBANS or CDR, underscoring BioCog's robustness and generalizability in detecting objective cognitive impairment. Furthermore, in a two-step workflow combining BioCog with a blood biomarker test, clinical AD (that is, cognitive impairment due to AD) could be identified with significantly higher accuracy than the standard PCP assessment (90% versus 70%), or using a blood biomarker test without previous digital cognitive assessment (90% versus 80%). Beyond high accuracy, digital tests like BioCog offer distinct advantages over paper-and-pencil assessments. BioCog enables time-efficient cognitive evaluations and eliminates variability introduced by differences in scoring or interpretation, which are critical features in primary care19. Furthermore, it facilitates the simultaneous collection of different variable categories, such as time, alongside correct answers, allowing for more comprehensive data. Digital platforms also make it easier to utilize advanced statistical models, moving beyond simple threshold-based scoring of sums (used in MMSE, MoCA and Mini-Cog) to leverage more nuanced, data-driven insights without adding significant processing time or requiring advanced interpretation skills. Compared to another digital test that has been CE marked and US Food and Drug Administration approved and is currently used in clinical practice for dementia evaluations (CANTAB), BioCog demonstrated significantly higher accuracy in identifying cognitive impairment (85% versus 76%) (Table 2). Our study did not evaluate all available digital tablet solutions, such as Cognivue24, NIH Toolbox25, cCOG26, Cogstate27 or TabCAT28, considering the load this would impose on participants, and to avoid learning biases. To our knowledge, none of these digital tests have been evaluated in primary care. Future research comparing various brief and self-administered digital tools in primary care will be essential to ensure that clinicians have access to the most effective options for detecting cognitive impairment. Importantly, a central conclusion of this study is that combining digital cognitive testing and a blood biomarker can enhance not only the efficiency of the diagnostic process in primary care, but also the diagnostic certainty of AD. 4) combining a positive BioCog assessment with an accurate blood biomarker test offers an efficient and reliable way to diagnose clinical AD with high certainty (accuracy of 90% compared to 70% for current standard-of-care without biomarkers and 80% for blood biomarkers only). These results suggest that digital cognitive tests, when combined with blood biomarker tests, have potential to significantly enhance the clinical work-up of AD and can become highly valuable to efficiently align clinical AD diagnostics with global recommendations. The World Health Organization's preferred product profile for AD blood tests emphasizes that such tests should be used exclusively in individuals with cognitive impairment17, where the pretest probability is high enough to achieve a PPV of >90% without the need for confirmatory CSF or PET biomarkers. This aligns with other international recommendations11,16. In addition, only individuals with cognitive impairment are eligible for treatment with acetylcholinesterase inhibitors or recently developed Aβ-targeting immunotherapies1,13,14,15. This also underscores the importance of identifying cognitive impairment in primary care with efficient and accurate tools like BioCog, to enable referral of as many eligible patients as possible to specialist clinics for treatment with Aβ immunotherapies without substantial burden on the primary care system. Further evaluation (for example magnetic resonance imaging scans, APOE, contraindicated medications) of potential treatment candidates is required at specialty clinics before initiating treatment29. For digital assessments and blood tests to be effective in diagnosing clinical AD, it is essential that they demonstrate very high PPVs, particularly before initiating expensive treatments. In addition, it is of high importance that their predictive properties are robust, with strong generalization to new patients. Blood biomarker tests have shown high performance across many secondary care cohorts9,30, as well as in primary care11, but their PPV varies by age and cognitive status because of differences in pretest probabilities31, so their use should be adapted to the clinical context. In this study, using BioCog to assess cognitive impairment expectedly showed a slight decrease in performance between the secondary care cohort (used to fit the model, AUC = 0.96 and PPV = 91%) and the more clinically diverse primary care cohort (only used for model evaluation, AUC = 0.93 and PPV = 87%). To develop a robust BioCog model, we applied recursive feature selection with AIC penalization and focused on external model validation in the independent primary care cohort. We also investigated the ability of BioCog6 to identify cognitive impairment with CDR as outcome instead of RBANS. Whereas RBANS offers a more objective measure of cognitive impairment, CDR captures functional decline32,33. When using a CDR global score ≥0.5 as outcome, performance remained high, with only a slight decrease (for example, accuracy from 85% to 82%, and PPV from 87% to 86%). Because the BioCog6 model was optimized to detect cognitive impairment (RBANS) and not functional impairment (CDR), a drop in performance is expected. Nevertheless, this decrease was minimal, and BioCog6 still outperformed both the clinical judgment of PCPs and other cognitive tests. To further increase diagnostic performance of the BioCog6 model, we applied a two-cutoff approach, which resulted in a PPV that remained >90% even in primary care and with 18% of individuals in the intermediate group. These results are encouraging, with BioCog demonstrating a high level of robustness and adaptability, even when applied to a cohort with differing demographic characteristics and a higher prevalence of comorbidities (Table 1), and when defining cognitive impairment using a different reference standard. For the two-cutoff approach, people with intermediate results would need more comprehensive cognitive evaluations. Alternatively, repeated cognitive testing incorporating changes in test scores may help identify cognitive impairment in those individuals36. The findings of our study highlight the key role digital cognitive tools can play in optimizing diagnostic workflows, particularly when combined with blood biomarker tests for a more precise evaluation of clinical AD. Beyond individual test performance, the broader impact of digital assessments lies in their ability to enhance efficiency, standardization and scalability in cognitive assessments37. Importantly, their greatest value emerges when integrated into a broader clinical evaluation, in which results can be interpreted alongside other clinical information to support well-informed diagnostic decisions and strengthen the healthcare professional's confidence in making a correct diagnosis38. Even though this study focuses on highlighting the clinical value of BioCog by showing its independent contribution in identifying objective cognitive impairment and clinical AD, our proposal is not to use BioCog or other digital tests as stand-alone tools. BioCog is aimed to complement, not replace, clinical judgment, and further validation in additional research studies and integration into structured care pathways is essential before deployment. In addition, future implementation-based research in different healthcare systems is needed to evaluate the impact on clinical management and to establish clinical practice guidelines before digital assessments combined with blood biomarkers can be used as part of everyday primary care in real-world settings. There are limitations to this study. First, we established the BioCog model in a secondary care cohort, which differed on demographic factors like age and education level, and on comorbidity prevalence compared to the primary care cohort, representative of the intended use population. Second, although BioCog performed well in identifying cognitive impairment, further research is needed to evaluate its utility in longitudinal settings and its ability to monitor disease progression. Third, BioCog was developed and evaluated in Swedish cohorts. It has been translated into several languages, including English, Finnish, Dutch and Spanish, and validation in these languages and in more diverse populations is important to further assess the generalizability. Such validation will be crucial before clinical implementation. Notably, paper-and-pencil versions of similar types of tests have performed very well across different languages and cultures. Fourth, this study did not directly assess how the use of a digital tool influences the clinical work-up in primary care, or whether physicians find it beneficial in practice. These questions will be addressed in future work. Fifth, BioCog was not evaluated as a screening tool for the general population. Instead, it was assessed in individuals who presented with cognitive complaints in primary care, where a physician considered a neurodegenerative disease to be a possible cause. People whose symptoms were clearly attributable to other causes (for example, depression or sleep disorders) were treated or referred accordingly and were not included in the study. This limits our ability to assess BioCog's utility for screening of the general population. However, such screening currently lacks evidence for improving decision-making or other important outcomes39. We therefore argue that the current study, limited to participants with cognitive symptoms possibly caused by an underlying neurodegenerative disease, is clinically more relevant and reflects a realistic scenario for how digital cognitive tools may be integrated to support physicians in primary care. In summary, this proof-of-concept study demonstrates that a self-administered digital test is effective for identifying objective cognitive impairment in a population with cognitive symptoms for whom the PCP thought a neurodegenerative disease was a possible cause of the symptomatology. Furthermore, when combined with a blood biomarker test, the digital cognitive test could reliably diagnose clinical AD with high accuracy, outperforming the current standard-of-care in primary care. These findings have the potential to transform clinical diagnostics of AD in primary care, enabling detection and treatment in a resource efficient manner. All participants provided written informed consent before enrollment in the study, and ethical approval for the study was obtained from the Swedish Ethical Review Authority. Data for this study were collected between February 2022 and December 2024. The secondary care cohort was used as the training cohort to establish cognitive test models and cutoffs. Secondary care participants were recruited from the ongoing BioFINDER-2 study (NCT03174938), described in detail elsewhere20. Briefly, the study consecutively enrolls participants at the secondary care memory clinic of Skåne University Hospital and the memory clinic of Ängelholm Hospital, and includes a diverse population of study participants. For the current study, patients with either SCD, MCI or dementia were included. Further details are given in the Supplementary Methods. The BioFINDER-Primary Care study (NCT06120361), described previously6, is an ongoing study that recruits patients from primary care centers in southern Sweden. The study consecutively includes patients seeking medical help due to cognitive symptoms either self-reported or reported by a close relative or spouse. In addition, inclusion may also be initiated based on the PCP's suspicion of a neurocognitive disorder. To ensure a realistic and ethical study design aligned with standard care, the primary care cohort included only individuals for whom the treating PCPs, after the first assessment, considered a potential neurodegenerative disease. People whose symptoms were assessed to be clearly attributable to other causes (for example, depression, sleep disorders) were not enrolled and instead treated at the primary center or referred for appropriate care elsewhere. Patients already diagnosed with dementia or suffering from an unstable severe systemic illness as well as those with ongoing significant alcohol or substance abuse were excluded. At their respective primary care center, patients underwent standard clinical evaluation (‘standard-of-care'), including medical examination by the PCP, cognitive testing, standard blood assessments to rule out other causes and structural brain imaging. Based on the assessments of PCPs, only 29.1% (106 of 364) of the participants would have been referred to secondary care if not included in this study, underscoring that the primary and secondary care cohorts represented different patient populations. For this study patients from 19 primary care centers were included. Further details of the cohort, cognitive testing and inclusion and exclusion criteria are provided in Supplementary Methods. The digital cognitive battery, BioCog, was developed using Python and Kivy to run on an Android tablet. The battery was designed to include a primary part and an optional secondary part that contained more demanding tasks (and was therefore not used in this study). During development of the battery, it was tested by cognitively unimpaired volunteers to improve the user experience. Following this pilot phase, participants in the BioFINDER-Primary Care study completed the battery and provided feedback on how they perceived the tasks and instruction comprehensibility. After additional adjustments, digital testing was introduced in the study design and data collection started. None of the participants in this study had previous experience with the BioCog test battery. The first part of the self-administered test battery, which was used in the current study, had an average completion time of 11.2 min, and is comprised of a word list test with immediate and delayed recall as well as a delayed recognition part, a cognitive processing speed task and questions about orientation to the weekday, date, month and year. To evaluate the performance of the digital test to detect cognitive impairment, it was compared to paper-and-pencil tests frequently used in primary care settings (which require healthcare personal to perform the tests) and to a well-known, CE marked and US Food and Drug Administration approved digital test. The brief paper-and-pencil tests used, were: (1) the MMSE40, the world's most used cognitive test, which consists of several shorter tasks and has a maximum score of 30 points; (2) MoCA41, a widely used screening tool with a maximum of 30 points, which evaluates a broader range of cognitive domains than the MMSE; (3), Mini-Cog42, which was developed as a very brief screening test that includes a three-word delayed recall task and clock drawing and has a maximum of 5 points; and (4) the digital, tablet-based paired associates learning task (PAL)43 from the CANTAB, which measures visual associative memory. For the main analyses, externally established and validated cutoffs were used for all measures: MMSE <27 points44, MoCA <26 points41, Mini-Cog <4 points45 and for PAL >41 adjusted errors in total46. For MoCA a two-cutoff approach was also included using <24 points as the lower cutoff and 26 as the upper cutoff21. For the other tests no validated two-cutoff approaches were available. As sensitivity analyses, the cognitive tests were treated as continuous variables and adjusted for demographics. The standard clinical evaluation at the primary care units included brief cognitive testing (MMSE and MoCA), computed tomography scan of the brain, and a clinical assessment of the patient. After completing this evaluation, PCPs were asked to report the cognitive stage (SCD, MCI or dementia) and if assessed as MCI or dementia, the most likely cause of the cognitive impairment. The latter was binarized as AD or non-AD. Blood was collected in EDTA tubes and centrifuged within 1 h. Plasma was then pipetted to 1-ml LoBind tubes and shipped the same day to be frozen at −80 °C. Plasma samples were then shipped to and analyzed by C2N Diagnostics in the United States continuously during the study period (47 occasions). Mass spectrometry assays were used to analyze Aβ42, Aβ40, phosphorylated-tau217 and nonphosphorylated-tau217, as previously described6. These biomarkers were combined in a logistic regression model (PrecivityAD2 test) to yield a score from 0 to 100 called the APS2 (ref. An APS2 cutoff value of 36 was used, previously defined in a clinical practice research study established at 90% specificity for AD pathology6. In addition, a two-cutoff approach with values corresponding to 95% sensitivity and 95% specificity (31 lower, 62 upper) was used, also previously defined6. The primary outcome was objectively verified cognitive impairment. In the primary care cohort, this was established using the RBANS48, administered by neuropsychologists at the memory clinic of Skåne University Hospital. The RBANS is a widely used cognitive test battery that has shown high accuracy in detecting and characterizing cognitive deficits associated with AD49 and MCI in general50. Using normative data from the Swedish manual generated an age-corrected standard (M = 100, s.d. = 15) total scale score for each participant. Based on the widely accepted neuropsychological criteria for defining cognitive impairment51,52, participants with an RBANS total scale score of 1.5 s.d. below the normative mean or less were considered cognitively impaired. In the secondary care cohort, RBANS was not available. Instead, an RBANS proxy variable was created, based on similar paper-and-pencil neuropsychological tests (Supplementary Methods and Supplementary Table 18). The secondary outcome ‘clinical AD' was defined as having cognitive impairment (as described above) where AD was determined as the primary etiology. The latter was determined in consensus discussions including dementia specialists and neuropsychologists. The consensus discussions included access to medical examination by a dementia specialist at the memory clinic, a computed tomography or magnetic resonance imaging scan of the brain, CSF AD biomarkers or Aβ-PET results as well as neuropsychological assessment (Supplementary Methods). The diagnosis of clinical AD was defined in accordance with criteria from the International Working Group13, which includes a typical clinical presentation of AD in patients with objective cognitive impairment and confirmation with AD biomarkers. Participants fulfilling the criteria for clinical AD but also having signs of other concomitant diseases (for example, AD mixed with vascular disease) were labeled as having clinical AD in the analyses. Individuals that were negative according to this outcome were either cognitively unimpaired or cognitively impaired not due to AD (see Supplementary Table 16 for details of comorbidities and diagnoses). The CDR scale is a semiobjective, qualitative staging instrument for the assessment dementia severity32. It assesses the cognitive and functional performance of everyday activities in six categories: memory, orientation, judgment and problem-solving, community affairs, home and hobbies, as well as personal affairs. Each item in the respective subscales is rated and a global CDR score is derived. A score of 0 is considered normal, with a score of 0.5, 1, 2 or 3 indicating questionable, mild, moderate and severe dementia33. Participants with a CDR global score of 0.5 and above were considered cognitively impaired and this was used as an outcome in the sensitivity analysis. In this study, CDR was performed by a dementia specialist at the memory clinic blinded to BioCog. The analyses were performed in Python v.3.9. Multiple logistic regression models were created, and cutoffs were established in the secondary care cohort. Model performance was then evaluated in the same cohort, and more importantly, in the external and completely independent primary care cohort. To decrease the risk of overfitting when creating multiple logistic regression models, the number of input variables was reduced with recursive feature elimination based on the standardized logistic regression model coefficients. The best model was selected using the AIC, requiring a decrease of 2 or more in AIC to indicate a better model fit and justifying the addition of another input variable53,54. Model performance was evaluated from the ROC AUC. An optimal cutoff point was selected based on the maximum value of Youden's index and the corresponding model was evaluated on accuracy, PPV, NPV, specificity and sensitivity. For the two-cutoff approach, cutoffs were selected based on 95% specificity and 95% sensitivity in the independent secondary care cohort. Participants in the intermediate group were not considered when evaluating model performance. The 95% CI and two-sided P values were computed using bootstrapping (n = 5,000 resamples with replacement), with a P value <0.05 indicating statistical significance. P values were adjusted for multiple comparisons by the Benjamini–Hochberg method. Differences between cognitively unimpaired and cognitively impaired on the various subtests of BioCog were assessed using the Mann–Whitney U-test. Spearman's rank correlation coefficients were used to evaluate associations between BioCog subtests and their paper-and-pencil counterparts, as well as with age and education. Internal consistency was assessed using McDonald's ω. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Anonymized data can be shared by request from qualified academic investigators for the purpose of replicating procedures and results presented in the article. Data transfer is required to be in agreement with EU legislation on the general data protection regulation and decisions by the Ethical Review Board of Sweden and Region Skåne, which should be regulated in a material transfer agreement. No custom code or mathematical algorithm that was central to the conclusions was used in this study. Jack, C. R. Jr et al. Revised criteria for diagnosis and staging of Alzheimer's disease: Alzheimer's Association Workgroup. Petersen, R. C. Mild cognitive impairment as a diagnostic entity. Mattke, S. et al. Expected and diagnosed rates of mild cognitive impairment and dementia in the U.S. Medicare population: observational analysis. Rabinovici, G. D. et al. Association of amyloid positron emission tomography with subsequent change in clinical management among Medicare beneficiaries with mild cognitive impairment or dementia. Hansson, O. Biomarkers for neurodegenerative diseases. Hansson, O., Blennow, K., Zetterberg, H. & Dage, J. Blood biomarkers for Alzheimer's disease in clinical practice and trials. Highly accurate blood test for Alzheimer's disease is similar or superior to clinical cerebrospinal fluid tests. Acceptable performance of blood biomarker tests of amyloid pathology — recommendations from the Global CEO Initiative on Alzheimer's Disease. Prevalence estimates of amyloid abnormality across the Alzheimer disease clinical spectrum. Dubois, B. et al. Clinical diagnosis of Alzheimer's disease: recommendations of the International Working Group. Sims, J. R. et al. Donanemab in early symptomatic Alzheimer disease: the TRAILBLAZER-ALZ 2 randomized clinical trial. van Dyck, C. H. et al. Lecanemab in early Alzheimer's disease. & Jack, C. R. Jr. A clinical perspective on the revised criteria for diagnosis and staging of Alzheimer's disease. Preferred Product Characteristics of Blood-Based Biomarker Diagnostics for Alzheimer Disease (2024). Palmqvist, S. et al. Alzheimer's Association Clinical Practice Guideline on the use of blood-based biomarkers in the diagnostic workup of suspected Alzheimer's disease within specialized care settings. Expanding the use of brief cognitive assessments to detect suspected early-stage cognitive impairment in primary care. Palmqvist, S. et al. Discriminative accuracy of plasma phospho-tau217 for Alzheimer disease vs other neurodegenerative disorders. Enhanced diagnostic accuracy for neurocognitive disorders: a revised cut-off approach for the montreal cognitive assessment. Brum, W. S. et al. A two-step workflow based on plasma p-tau217 to screen for amyloid beta positivity with further confirmatory testing only in uncertain cases. Cahn-Hidalgo, D., Estes, P. W. & Benabou, R. Validity, reliability, and psychometric properties of a computerized, cognitive assessment test (Cognivue®). Weintraub, S. et al. Cognition assessment using the NIH Toolbox. Rhodius-Meester, H. F. M. et al. cCOG: a web-based cognitive test tool for detecting neurodegenerative disorders. Maruff, P. et al. Validity of the CogState brief battery: relationship to standardized tests and sensitivity to cognitive impairment in mild traumatic brain injury, schizophrenia, and AIDS dementia complex. The brain health assessment for detecting and diagnosing neurocognitive disorders. Cummings, J. et al. Lecanemab: appropriate use recommendations. Hampel, H. et al. Blood-based biomarkers for Alzheimer's disease: current state and future use in a transformed global healthcare landscape. Diagnosis of Alzheimer's disease using plasma biomarkers adjusted to clinical probability. Hughes, C. P., Berg, L., Danziger, W., Coben, L. A. & Martin, R. L. A new clinical scale for the staging of dementia. Morris, J. C. Clinical dementia rating: a reliable and valid diagnostic and staging measure for dementia of the Alzheimer type. Stability in test-usage practices of clinical neuropsychologists in the United States and Canada over a 10-year period: a follow-up survey of INS and NAN members. Wessels, A. M. et al. Cognitive outcomes in trials of two BACE inhibitors in Alzheimer's disease. Borland, E. et al. Individualized, cross‐validated prediction of future dementia using cognitive assessments in people with mild cognitive symptoms. Effects of the Davos Alzheimer's Collaborative early detection of cognitive impairment program on clinician attitudes, engagement, and confidence. & Eckstrom, E. Screening for cognitive impairment in older adults: a systematic review for the US preventive services task force. Folstein, M. F., Folstein, S. E. & McHugh, P. R. ‘Mini-Mental State': a practical method for grading the cognitive state of patients for the clinician. The Montreal Cognitive Assessment, MoCA: a brief screening tool for mild cognitive impairment. Borson, S., Scanlan, J., Brush, M., Vitaliano, P. & Dokmak, A. The Mini-Cog: a cognitive ‘vital signs' measure for dementia screening in multi-lingual elderly. Sahakian, B. J. et al. A comparative study of visuospatial memory and learning in Alzheimer-type dementia and Parkinson's disease. O'Bryant, S. E. et al. Detecting dementia with the Mini-Mental State Examination in highly educated individuals. McCarten, J. R. et al. Finding dementia in primary care: the results of a clinical demonstration project. Junkkila, J., Oja, S., Laine, M. & Karrasch, M. Applicability of the CANTAB-PAL computerized memory test in identifying amnestic mild cognitive impairment and Alzheimer's disease. Meyer, M. R. et al. Clinical validation of the PrecivityAD2™ blood test: a mass spectrometry-based test with algorithm combining %p-tau217 and Ab42/40 ratio to identify presence of brain amyloid. Randolph, C. Repeatable Battery for the Assessment of Neuropsychological Status (RBANS) (Psychological Corporation, 1998). Duff, K. et al. Utility of the RBANS in detecting cognitive impairment associated with Alzheimer's disease: sensitivity, specificity, and positive and negative predictive powers. Karantzoulis, S., Novitski, J., Gold, M. & Randolph, C. The Repeatable Battery for the Assessment of Neuropsychological Status (RBANS): utility in detection and characterization of mild cognitive impairment due to Alzheimer's disease. Mild cognitive impairment: clinical characterization and outcome. Jak, A. J. et al. Quantification of five neuropsychological approaches to defining mild cognitive impairment. Using Akaike's information theoretic criterion in mixed-effects modeling of pharmacokinetic data: a simulation study. Burnham, K. P. & Anderson, D. R. Multimodel inference: understanding AIC and BIC in model selection. We thank the research volunteers who participated in the studies and their families. Work at the authors' research center was supported by the National Institute of Aging (grant no. R01AG083740), European Research Council (grant no. ZEN24-1069572 and SG-23-1061717), GHR Foundation, Michael J. MJFF-025507), Lilly Research Award Program, WASP and DDLS Joint call for research projects (grant no. 2021-02219, 2022-00775 and 2018-02052), ERA PerMed (grant no. ERAPERMED2021-184), Knut and Alice Wallenberg Foundation (grant no. 2022-0231), Strategic Research Area MultiPark (Multidisciplinary Research in Parkinson's disease) at Lund University, Swedish Alzheimer Foundation (grant nos. AF-1011949, AF-994229 and AF-980907), Swedish Brain Foundation (grant nos. FO2023-0163, FO2024-0284 and FO2021-0293), Parkinson foundation of Sweden (grant no. 1412/22), Cure Alzheimer's fund, Rönström Family Foundation (grant nos. 20243058), Avid Pharmaceuticals, Bundy Academy, Konung Gustaf V:s och Drottning Victorias Frimurarestiftelse, Skåne University Hospital Foundation (grant no. Open access funding provided by Lund University. These authors contributed equally: Pontus Tideman, Linda Karlsson. Clinical Memory Research Unit, Department of Clinical Sciences Malmö, Lund University, Lund, Sweden Pontus Tideman, Linda Karlsson, Olof Strandberg, Ruben Smith, Niklas Mattsson-Carlgren, Erik Stomrud, Sebastian Palmqvist & Oskar Hansson Memory Clinic, Skåne University Hospital, Malmö, Sweden Pontus Tideman, Ruben Smith, Niklas Mattsson-Carlgren, Erik Stomrud & Sebastian Palmqvist Center for Primary Health Care Research, Department of Clinical Sciences Malmö, Lund University, Malmö, Sweden Office for Primary Care, Skåne University Hospital, Lund, Sweden C2N Diagnostics LLC, St Louis, MO, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived of and designed the study. contributed equally as first authors. supervised and was principal investigator for the study. performed statistical analyses and prepared the figures. Correspondence to Pontus Tideman or Oskar Hansson. has acquired research support (for the institution) from Avid and ki elements through ADDF. In the past 2 years, he has received consultancy and/or speaker fees from Bioartic, Biogen, Esai, Eli Lilly, Novo Nordisk and Roche. has received consultancy and/or speaker fees from Lilly, Novo Nordisk, Roche and Triolab. N.M.-C. received consultancy and/or speaker fees from Biogen, Merck and Owkin. is an employee of Eli Lilly and Lund University. are salaried employees for C2N Diagnostics and receiving compensation from the company in the form of a salary or equity. The other authors declare no competing interests. Nature Medicine thanks Vijaya Kolachalama and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Jerome Staal in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. BioCog6 two-cutoff model performance in the secondary care cohort (which was also used to fit the model and to establish the cutoffs). The two cutoffs were set at probabilities 0.332 and 0.769, corresponding to 95% sensitivity and 95% specificity respectively. a) Evaluation of the BioCog6 two-cutoff model. Error bars indicate 95% CIs, with the centre point corresponding to the mean value. b) Cross-tabulation analysis the BioCog6 two-cutoff model. c) Histogram of the predicted probabilities from the BioCog6 model (x-axis) colored by the true binary RBANS proxy composite score (outcome variable). Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Tideman, P., Karlsson, L., Strandberg, O. et al. Primary care detection of Alzheimer's disease using a self-administered digital cognitive test and blood biomarkers. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41593-025-02055-5'>Recurrent pattern completion drives the neocortical representation of sensory inference</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 09:58:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. When sensory information is incomplete, the brain relies on prior expectations to infer perceptual objects. Despite the centrality of this process to perception, the neural mechanisms of sensory inference are not understood. Here we used illusory contours (ICs), multi-Neuropixels measurements, mesoscale two-photon (2p) calcium imaging and 2p holographic optogenetics in mice to reveal the neural codes and circuits of sensory inference. We discovered a specialized subset of neurons in primary visual cortex (V1) that respond emergently to illusory bars but not to component image segments. These data imply that neurons that encode sensory inference are specialized for receiving and locally broadcasting top-down information. More generally, pattern completion circuits in lower cortical areas may selectively reinforce activity patterns that match prior expectations, constituting an integral step in perceptual inference. Illusions arise from rational mistakes in sensory inference. For example, a rational perceptual interpretation of the classic Kanizsa illusion is that of a white triangle in front of three black circles (Fig. The nervous system likely evolved to rely on inference because we must frequently infer objects from partial information, such as during occlusion2. Many species, including humans, nonhuman primates, mice, fish and even insects, perceive illusory contours (ICs), implying that sensory inference is fundamental to perception3,4,5,6,7,8. For example, mice trained to perform an orientation discrimination task on real edges can generalize their performance to ICs and vice versa6,7. b, Example visual stimuli used in this experiment. c, Receptive fields were mapped using 16° patches of circular drifting gratings appearing in one of the nine positions depicted (left). Note that 16° is slightly larger than the typical size of receptive fields in mouse V1. d, Schematic of six Neuropixels probes insertion into V1, LM, RL, AL, PM and AM (12 sessions from 12 mice). e, Receptive field map for V1 regular-spiking (RS) units that exclusively responded to grating patches in position 1, corresponding to the illusory gap region (n = 18 exclusively center-responsive neurons; 12 sessions from 12 mice). f, PSTHs of exclusively center-responsive V1 RS units on IC and IRE trials, for preferred IRE orientation and corresponding IC trials. Bottom, individual neurons with substantially IC-responsive (left; 66.7%) and IRE-responsive (right; 77.8%) neurons marked in magenta on the y axis. g, Comparison of the preferred orientation between illusory bars (IC stimuli) and real bars (IRE stimuli) for IRE-responsive and exclusively center-responsive V1 RS units, showing a 78.6% match. i–k, Same as e–g, but for the 2p imaging dataset (n = 310 exclusively center-responsive V1L2/3 excitatory neurons; 29 sessions from five mice). across exclusively center-responsive excitatory neurons in V1L2/3 (j). Bottom, individual neurons with substantially IC-responsive (left; 21.3%) and IRE-responsive (right; 43.2%) neurons marked in magenta on the y axis. Illusions are uniquely suited for dissociating faithfulness from inference-based sensory representations in the brain. Pioneering work in the primate visual cortex (V1) identified neurons that responded to ICs as if they were real edges, despite the lack of any actual contrast in their receptive fields9,10,11. Such responses may represent a neural correlate of emergent, or Gestalt, perception, in which the perceived whole is greater than the sum of its parts (or sensory inputs). While lower visual cortical areas are primarily faithful to image segments, higher visual areas show stronger emergent representations of the ICs12,13,14. Furthermore, recent studies in mice showed that optogenetic silencing of a higher visual area (lateromedial area, LM) during IC presentation reduced IC-evoked responses in V1 (refs. These data support a model where circuits in higher cortical areas first compute the presence of ICs and then feed back these representations as predictive inferences to lower cortical areas16,17. However, it remains unclear why sensory predictions are fed back to V1 at all. To address the neocortical mechanism of sensory inference, we used large-scale neurophysiological methods to record and manipulate neural activity from awake mice viewing images that contain ICs. We used mesoscale two-photon (2p) calcium imaging, multi-Neuropixels extracellular electrophysiology recordings, 2p holographic optogenetics and a new 2p holographic mesoscope18. With large-scale neural recordings, we investigated how neural decoders trained to discriminate between IC stimuli generalized their performance to the discrimination of real edge stimuli. In V1L2/3, a selected subset of neurons that responded emergently to the IC (‘IC-encoders') mediated the representation of IC inference. To test whether these IC-encoders could drive IC representations within V1L2/3 on their own, we holographically photoactivated them in the absence of any visual stimulus. We found that photoactivating IC-encoders recreated IC representations within V1L2/3 through pattern completion. These results imply that neurons that encode top-down inference signals locally broadcast those signals through pattern completion. On the other hand, neurons that respond to the individual inducing segments of the IC (‘segment responders') drove IC representations in higher visual areas but not within V1L2/3. Taken together, recurrent activity through feedforward, feedback and local pattern completion circuits may facilitate sensory inference by reinforcing activity patterns that match the brain's prior expectations about the sensory world. Using two high-throughput neural recording techniques with complementary strengths, multi-Neuropixels and large-scale 2p imaging, we recorded the neural activity of awake, head-fixed mice while presenting the animals with various images containing real edges or ICs (IC denotes ‘I-shaped combination', IRE denotes ‘I-shaped real edge'; Fig. First, we asked if we could identify visual cortical neurons that respond to ICs even in the absence of any contrast within their receptive fields, as previously shown in primates9,10,14. We focused on V1 neurons whose receptive fields are contained entirely within the gap region. If these neurons are driven by ICs, then their visually evoked activity is, by definition, not bottom up7,9,10,14. Thus, we selected only those neurons that responded exclusively to the illusory gap region, that is, the central 16 visual degree region of the visual presentation monitor (Fig. We then examined the response of these exclusively center-responsive V1 neurons to IC-containing images (IC), in comparison to nearly identical images where the ICs are replaced with real edges (‘IRE images'; note that the thickness of the outline edge was ≥2°, which is detectable by mice19). As expected, these neurons responded more strongly to IRE trials because there was actual contrast within their receptive fields. Notably, a substantial portion of these neurons also responded on IC trials, despite having no contrast within their receptive fields (Fig. Furthermore, there was a strong degree of overlap in the orientation preference for illusory bars in IC images and real bars in IRE images, where the preferred IC and IRE orientation was defined as the orientation of the illusory bar in IC images and the real bar in IRE images that evoked the strongest mean response (Fig. These results demonstrate that mouse V1 contains neurons that respond to illusory bars as if they were real bars, in an orientation selective manner, despite the lack of contrast in their receptive field. Moreover, these results did not change when limiting the analyses to trials where the mouse's eye position stayed within eight visual degrees of the modal eye position, indicating that eye movements did not affect this conclusion (Extended Data Fig. We further verified the illusory bar responsiveness and orientation selectivity for exclusively center-responsive V1 neurons using larger illusory gap sizes (Extended Data Fig. 3a–d), denser receptive field mappings (Extended Data Fig. Taken together, V1 neurons with receptive fields within the illusory gap region can respond to illusory bars with the same orientation selectivity as to real bars. If this were true, these neurons would be sensitive to the global arrangement of inducer elements that are outside their receptive fields. Indeed, several recent studies show that mouse V1 neurons are highly sensitive to contextual information outside their classical receptive fields, suggesting that neurons that are exclusively responsive to visual information within their classical receptive field may be more of an exception than the norm20,21,22,23. Thus, we sought to identify neurons that selectively encode ICs as a global emergent feature of the image, irrespective of their receptive fields. To test this notion, we needed to design a set of visual stimuli that would distinguish between cells that simply responded to low-level features of the image per se (that is, one of the inducer segments), as opposed to the emergent IC inference. It is essential to rule out low-level image differences when identifying IC encoding neurons, such that the neural representation can be interpreted appropriately as representing sensory inference. To address this challenge, we devised a new set of four stimuli that could disambiguate low-level from emergent responses (Fig. This design ensured that response selectivity for one of the four images could not be explained by sensitivity to the component inducer segments; instead, it can only be explained by sensitivity to a global feature of the image. Using these stimuli, we identified approximately 4.3% of neurons in V1L2/3 with selectivity for one of the two IC images, and no substantial responses to either LC image (P < 0.05 Kruskal–Wallis test followed by Tukey–Kramer correction for multiple comparisons; 8.5% for both configurations of white-circles-on-black-background, that is, four IC orientations; Fig. Segment images contain individual segments of the IC and LC images (BR, BL, TL and TR). b, IC-encoders are defined as neurons that respond to one of the IC images and to neither of the LC images (left; two panels). Segment responders are defined as neurons that respond more to the inward than to the outward segment in each position (right; two panels). Average evoked responses of IC-encoders and segment responders to IC and LC images are shown as heatmaps (2p imaging dataset with V1L2/3 field-of-view, 24 sessions from four mice). c, Orientation tuning of IC-encoders, measured with single long bars (left), averaged over each IC-encoder subgroup (P = 1.8 × 10−5 Wilcoxon signed-rank test between the orientation of the illusory bar vs. the orthogonal orientation; 2p imaging dataset with V1L2/3 field-of-view, 11 sessions from five mice). d, Putative excitatory connections are determined as suprathreshold peaks in the cross-correlograms between RS units (example on the left). V1 IC-encoders (n = 79) are substantially more likely than V1 segment responders (n = 526) to receive putative excitatory connections from higher visual areas (LM, RL, AL, PM and AM). All box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5× interquartile range; points, outliers (Wilcoxon rank-sum test P = 7.0 × 10−6; 12 sessions from 12 mice). The defining feature of IC-encoders is that they respond to specific combinations of inducing segments. Conversely, neurons that respond to the individual segments per se are defined as ‘segment responders'; these neurons are substantially more responsive to inward than to outward orientation of each segment (P < 0.05, Wilcoxon rank-sum test; Fig. We identified segment responders in each quadrant—bottom-right (BR), bottom-left (BL), top-left (TL) and top-right (TR). Because each quadrant was shared by one of the IC images and one of the LC images, segment responders responded equally to an IC image and an LC image (Fig. To test whether IC-encoders are selective to the illusory bar in the IC image, we examined the orientation tuning of these neurons using single line images (Fig. Indeed, IC-encoders in V1L2/3 were tuned to the orientation of the illusory bar in the IC image (IC1-encoders were tuned to 135°, and IC2-encoders were tuned to 45°). Interestingly, although some of these neurons had receptive fields that mapped to the gap region of the IC, many did not (Extended Data Fig. This indicates that emergent responses to the illusory bar are not constrained to neurons with receptive fields on the illusory gap region. Furthermore, IC-encoders did not have substantially larger receptive fields compared to other neurons in V1L2/3, arguing that IC-encoders' selectivity to ICs cannot be attributed to feedforward convergence of sensory information across the illusory gap region (Extended Data Fig. In addition, eye movement was largely constrained within the gap region and was not biased across images, suggesting that eye movements did not influence IC-encoders' specificity to ICs (Extended Data Fig. Together, these results imply that bottom-up visual inputs cannot fully account for IC-encoders' responses to ICs in V1L2/3. This leads to the prediction that IC-encoders receive IC inference signals through top-down feedback from higher visual areas. To test this, we leveraged the large-scale nature of the Neuropixels dataset to perform a cross-correlogram analysis as an estimate of connectivity between higher visual areas and V1 (12 sessions from 12 mice; mean ± s.e.m. Under this analysis, we found that V1 IC-encoders received substantially more putative excitatory inputs from higher visual areas, compared to V1 segment responders (Fig. To understand how IC inference is computed at the microcircuit level, we then sought to determine where in the visual hierarchy this computation occurs. In primates, this top-down information is thought to come from V2 and the ventral stream13,14. In mice, the functional specialization of higher visual areas is less clear, although a recent optogenetics study has shown that LM contributes to IC responses in V1 (ref. Perhaps more notably, in both primates and mice, it is unclear whether V1 is the lowest region in the visual hierarchy where IC responses emerge. To address these questions, we recorded from six visual cortical areas simultaneously (V1, LM, anterolateral (AL), rostrolateral (RL), anteromedial (AM) and posteromedial (PM)), by targeting six Neuropixels probes to each area (n = 12 sessions from 12 mice), or by using a 2p mesoscope for calcium imaging in a separate cohort of mice (n = 19 sessions from 5 CaMKII-tTA;tetO-GcaMP6s mice; Supplementary Fig. We also compared V1L2/3 (n = 11 sessions from five CaMKII-tTA;tetO-GcaMP6s mice) and V1L4 (n = 8 sessions from two Scnn1a-Tg3-Cre;TIT2L-GC6s-ICL-tTA2 mice) using multiplane imaging in a standard 2p microscope. Using the same approach as above for V1, we could identify IC-encoders throughout these visual areas both in the electrophysiology and in the calcium imaging datasets (Extended Data Fig. Owing to the high-throughput nature of our recording techniques, we could apply powerful multivariate approaches that can test for the representation of IC inference comprehensively. We trained neural decoders (linear support vector machines, SVMs) to classify visual evoked neural activity patterns into corresponding visual labels for the four trial types described above (IC1, LC1, LC2 and IC2 trials; chance performance is 0.25). The tenfold cross-validation performance was substantially above chance, indicating that neural activity patterns evoked by these four images were highly distinct at the multivariate level (Fig. a, Tenfold cross-validation performance of a linear SVM trained on neural activity evoked by IC1 vs. LC1 vs. LC2 vs. IC2 images. The confusion matrix shows the average decoding performance of V1 neurons in the Neuropixels dataset (n = 12 sessions from 12 mice; average 150 V1 RS units per session, 400 repetitions per trial type). b, The stimuli used for assessing inference decoding. c, Inference decoding performance, obtained by probing the decoder trained on IC and LC trials with TRE trials (same decoders as a). d, Inference decoding of each visual cortical area in the Neuropixels dataset (mean ± s.e.m. across sessions, *P < 0.05, Wilcoxon signed-rank test; for each visual area, of 12 sessions, only sessions with ≥50 RS units are included, which leaves n = 12, 8, 11, 10, 12, 11 sessions with P = 0.0132, 0.0156, 0.0244, 0.0039, 0.0342, 0.1748 for V1, LM, RL, AL, PM and AM, respectively). e, Inference decoding of the 2p dataset. The proportion of TRE trials that are decoded as the corresponding IC (green) and LC (orange) images were averaged across the two image sets (mean ± s.e.m. To test whether IC inference is represented in the visually evoked neural activity patterns, we designed two new images to probe the neural decoder with, called ‘TRE images' (Fig. TRE images contain explicit real edges, unlike any of the four stimuli used to train the decoder (IC1, LC1, LC2 and IC2). Thus, the decoder would be equally likely to classify TRE-evoked neural activity patterns as IC or LC. Thus, if a brain area contained IC inference signals, the TRE-response in that area would be more similar to the IC-response than to the LC-response. We termed this analysis ‘inference decoding'. Applying inference decoding to Neuropixels V1 recordings revealed that TRE was substantially more likely to be decoded as IC than as LC (Fig. 3c; decoding probability for IC vs. LC was 0.51 vs. 0.22, P < 0.05 Wilcoxon signed-rank test). This result indicates that IC inference is represented in mouse V1. Whereas V1L2/3 showed substantial inference decoding performance, V1L4 did not (Fig. This result is consistent with the notion that V1L4, as the primary thalamo-recipient layer, is more faithful to sensory information than V1L2/3. Critically, it implies that V1L2/3 is the lowest region in the visual hierarchy where IC inference is represented. Next, we applied TRE inference decoding to identify higher visual areas that participate in IC encoding (Fig. In the Neuropixels dataset, almost every visual area was substantially biased toward decoding TRE as IC than LC, with V1, LM and AL showing the strongest representations of the IC (Fig. The difference in effect sizes between the 2p imaging and the Neuropixels data likely stems from inherent differences in signal-to-noise ratio between calcium imaging and extracellular electrophysiology. Prior behavioral work has shown that mice trained on real edges could generalize to ICs6, and, conversely, mice trained on ICs could generalize to real edges7. Analogously, we developed a complementary inference decoding approach where the neural decoder was trained on real edges and then probed with ICs. To this end, we designed a new set of images that have equal pixel overlap with IC1 and IC2 images; referred to as ‘XRE images' (Extended Data Fig. We trained the neural decoders to discriminate between neural activity patterns evoked by XRE1 vs. XRE2 images and used those decoders to classify IC1-evoked and IC2-evoked neural activity patterns. Then we computed the XRE inference score as the average of the probability that IC1 was decoded as XRE1, and IC2 as XRE2, and tested whether the XRE inference score was substantially different from chance of 0.5. When applying XRE inference decoding to different visual areas, we found consistent results as TRE inference decoding—IC inference was most strongly represented in L2/3 of V1 and LM (Extended Data Fig. As a further test of the robustness of inference decoding, we applied this analysis to black-and-white inverted images. We found that inference decoding indicated the representation of IC inference in V1L2/3 regardless of black–white inversion (Extended Data Fig. In addition, different types of decoders yielded highly consistent results (Supplementary Fig. Together, our data support a model in which the neocortical representation of IC inference arises through corticocortical feedforward and feedback pathways between V1 and higher visual areas, especially LM. Furthermore, IC inference is encoded in V1L2/3 and not V1L4, indicating that V1L2/3 is the lowest region in the visual hierarchy where IC inference signals emerge. Notably, none of the higher visual areas showed a substantially larger inference performance compared to V1 (Extended Data Fig. This suggests that V1L2/3 may have an active role in shaping IC inference, rather than passively reflecting the inference signals computed in higher visual areas. Thus, we sought to elucidate the role of V1L2/3 in IC encoding. So far, our data have established mechanisms of IC encoding at two levels. At the univariate level, we identified IC-encoders as neurons that respond to the whole illusory bar, but not its individual segments (Fig. We focused on V1L2/3 because it is the lowest region in the visual hierarchy where IC inference is represented. We used the 2p imaging data for this analysis since it afforded the necessary dense sampling of V1L2/3. Moreover, we wanted to eventually test the causal role of functionally identified neural ensembles, and 2p imaging is readily compatible with 2p holographic optogenetics experiments. 4a; V1L2/3 field of view, 24 sessions from four mice; mean ± s.e.m. Indeed, this abolished the IC inference performance; TRE-responses were no longer biased toward being classified as IC compared to LC (Fig. IC-encoders are defined only by their selective visual responses to IC compared to LC, independent of any analysis of decoder weights. Thus, zeroing them out in the decoder can be meaningfully interpreted as being essential for decoding. across n = 24 sessions from four mice; left—cross-validation accuracy Friedman test across subsets P = 1.1 × 10−5, *P < 0.05 Wilcoxon signed-rank test compared to chance performance of 0.25 with P = 1.8 × 10−5, 1.8 × 10−5, 1.8 × 10−5 for each subset; right—TRE inference performance Friedman test across subsets P = 0.0019, *P < 0.05 Wilcoxon signed-rank test comparing fraction of IC vs. LC trials decoded as the corresponding TRE with P = 0.0177, 0.1228, 0.0089 for each subset). b, Decoder performance when decoding only from the indicated subsets of neurons (same subsets as a; cross-validation accuracy Friedman test P = 0.0025 and Wilcoxon signed-rank test P = 1.8 × 10−5, 1.8 × 10−5, 5.6 × 10−5; TRE inference score Friedman test P = 0.0208 and Wilcoxon signed-rank test P = 0.0177, 0.0004, 0.2472; mean ± s.e.m. across n = 24 sessions from four mice, *P < 0.05 Wilcoxon signed-rank test). Next, we took the converse approach and trained a new decoder only on IC-encoders (Fig. Despite training this decoder on the activity of drastically fewer cells, it showed high levels of IC inference performance (Fig. These results demonstrate that the neural representation of IC inference is almost entirely mediated by the activity of IC-encoders. IC encoding is first initiated by bottom-up sensory drive from the inducing segments of the IC16,17. Thus, neurons that respond to the inducing segments, that is, segment responders, likely have an important role in feedforward sensory processing. As expected, segment responders contained sufficient visual information to discriminate between IC and LC images on their own (Fig. However, in contrast to IC-encoders, zeroing out segment responders' activity in the original decoder did not interfere with IC inference performance (Fig. 4a, right), and training a new decoder only on these neurons did not yield a substantial IC inference performance (Fig. These results suggest that segment responders faithfully encode sensory information contained in the image, rather than global emergent features of the image. Furthermore, these results implicate a functional dichotomy between segment responders and IC-encoders, where the former receive bottom-up sensory signals and the latter receive top-down inference signals (Supplementary Fig. Taken together, we found that IC-encoders are disproportionately important for the neural representation of IC inference in V1L2/3. Furthermore, these results imply a segregation of inputs; segment responders receive bottom-up sensory signals and IC-encoders receive top-down inference signals. Next, we investigated the output connectivity of these V1L2/3 ensembles using 2p holographic optogenetics. Prior studies have shown that IC encoding in V1 arises through top-down feedback7,15,16,17. However, prior work does not explain why IC encoding should exist in V1 in the first place, nor whether it might actively contribute to IC encoding. To address these questions, we used 2p holographic optogenetics to examine the direct causal role of distinct V1L2/3 ensembles in shaping IC encoding. One possibility is that V1 IC-encoders broadcast sensory inference signals through neural pattern completion within V1L2/3. This could promote sensory inference by selectively strengthening IC representations. Another possibility is that the segment responders of the inducing segments directly drive IC-encoders, resulting in neural pattern completion. To test these ideas, we selectively photoactivated ensembles of IC-encoders and segment responders using 2p holographic optogenetics (n = 24 sessions from four mice; same dataset as Fig. We developed an ‘all-optical read-write' experimental pipeline consisting of three stages (Fig. In stage 2 (the ‘online analysis' stage), we analyzed the visual response properties of every neuron in the field-of-view to identify IC-encoders and segment responders. Holographically driven activity of four holographic ensembles is shown in Fig. 5b; IC1-encoders, IC2-encoders, BR and TL segment responders (that is, segments in IC1), BL and TR segment responders (that is, segments in IC2). With this approach, we could test whether selectively photoactivating ensembles of IC-encoders or segment responders was sufficient to drive IC representation on their own. If so, this would imply that during normal visual activity, pattern completion circuits in V1 facilitate IC inference. a, ‘All-optical read-write' experimental pipeline (24 sessions from four mice expressing GCaMP6m–ChRmine in V1 neurons). In stage 3 (‘write' stage), each holographic ensemble photoactivation was repeated 50 times in a randomized order. b, PSTHs of holographic evoked activity in directly photoactivated neurons. c, PSTHs of holographic evoked responses in nonphotoactivated neurons (>50 μm from all targets in each ensemble), for neurons that are visually responsive to the corresponding IC image vs. nonresponsive neurons (24 sessions from 4 mice; mean ± s.e.m. Baseline activity, defined as activity −1 to 0 s relative to holography onset, was subtracted for each neuron. d,e, Decoders trained on visual evoked activity (0–1 s relative to visual onset) were used to classify holography-evoked activity (0–1 s relative to holography onset). Only the holographic ensembles with ≥10 targets were analyzed. Box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5× interquartile range; points, outliers. Holography trials were decoded as expected when decoding all imaged neurons (left; P = 7.9 × 10−7 one-tailed Wilcoxon signed-rank test compared to chance across n = 32 IC-encoder ensembles). In addition, nonphotoactivated neurons were decoded as corresponding visual labels substantially above chance (right; P = 4.3 × 10−5; median performance = 0.3720, Q1–Q3 = 0.2870–0.5750). *P < 0.05. e, Holography trials are expected to be decoded as IC1 when targeting BR + TL segment responders, and IC2 when targeting BL + TR segment responders. Holography trials were decoded as expected when decoding all imaged neurons (left; P = 0.0029 one-tailed Wilcoxon signed-rank test compared to chance across n = 37 segment responder ensembles). However, nonphotoactivated neurons were not decoded as the corresponding visual label (right; P = 0.3254; median performance = 0.2280, Q1–Q3 = 0.1680–0.3535). IC-responsive neurons respond to IC images, and they include both IC-encoders (which respond to IC but not LC images) and segment responders (which respond to both IC and LC images). If any of the holographic ensembles strengthened IC representations through neural pattern completion, we would expect them to synaptically drive neurons that are visually responsive to the IC image differentially from those that are not IC responsive. Consistent with prior reports, photoactivating a sparse ensemble of excitatory neurons in V1L2/3 led to a prolonged suppression of the nonphotoactivated population on average (Fig. When targeting IC-encoders, this suppression was preceded by a selective activation of IC-responsive neurons during the photoactivation window (0–1 s relative to holography onset). Inspecting individual neurons revealed that the biphasic response can be decomposed into a relatively fast activation of a subset of neurons, and a slower suppression of a different subset of neurons (Extended Data Fig. This indicates substantial heterogeneity among the holographic evoked activity of nonphotoactivated neurons. Despite this heterogeneity, targeting IC-encoders evoked substantially larger activity in IC-responsive neurons compared to non-IC-responsive neurons during the photoactivation window (P = 0.0023 one-tailed Wilcoxon signed-rank test across n = 32 holographic ensembles). On the contrary, targeting segment responders did not evoke differential activity in IC-responsive vs. non-IC-responsive neurons (P = 0.9660 across n = 37 holographic ensembles). Taken together, photoactivation of IC-encoders, but not segment responders, drove mean activity differences between IC-responsive and non-IC-responsive neurons among nonphotoactivated neurons (Fig. These results suggest that IC-encoders, but not segment responders, might drive neural pattern completion. Next, we used decoding to assess neural pattern completion more holistically. Thus, we first sought to establish that the photoactivation constituted a distinctive partial pattern of the visual evoked activity. To this end, we trained the neural decoder to classify visually evoked neural activity patterns with all neurons in the field-of-view (IC1 vs. LC1 vs. LC2 vs. IC2, as in Fig. Then, we asked how this decoder, trained only on visual evoked activity, would classify the holographically evoked neural activity in the absence of any visual inputs. These results demonstrate that the holographic optogenetic activation of these ensembles successfully emulated partial patterns of visual stimulus-specific neural activity (Fig. = 18 ± 2 IC-encoders and 22 ± 3 segment responders were targeted; analysis was restricted to holographic ensembles with n ≥ 10 targets). Equipped with this approach, we asked if selective photoactivation of these ensembles could drive pattern completion within the local V1L2/3 network. If IC-encoders have a privileged role in amplifying sensory inferences within V1, they may drive decodable representations of IC in the nonphotoactivated population. Indeed, the decoder constructed with nonphotoactivated neurons was able to classify IC-encoder photoactivation trials as the corresponding IC image (Fig. In fact, decoding holographically driven activity with a decoder trained on visual trials showed a performance that approached the cross-validation accuracy of held-out visual trials for the same decoder (mean ± s.e.m. performance for decoding holography trials as visual image labels was 0.4029 ± 0.0296, whereas cross-validation accuracy was 0.4818 ± 0.0165; Extended Data Fig. In contrast, photoactivating a similar number of segment responders did not evoke decodable activity patterns (Fig. These effects were consistent regardless of whether the decoder was constructed from nonphotoactivated neurons for each holographic ensemble (Fig. 5d,e, right, and Extended Data Fig. 9a–d), or from a common pool of nonphotoactivated neurons (Extended Data Fig. Thus, photoactivating IC-encoders, but not segment responders, was sufficient to drive V1 activity patterns resembling those visually evoked by the corresponding IC image even in the absence of any visual stimulus. Of note, in these experiments, we photoactivated nearly every IC-encoder within the imaged population. As a further test of the distinct capacity of IC-encoders to drive pattern completion, we photoactivated ‘LC-encoders', which we defined as neurons that selectively respond to one of the LC images but not to any of the IC images (Extended Data Fig. LC-encoders were substantially less potent at driving pattern completion of LC representations compared to IC-encoders, despite being matched in targeted numbers, visual selectivity and contribution to decoding performance (Extended Data Fig. Moreover, the optogenetically driven recurrent activity of nonphotoactivated neurons was substantially more likely to be decoded as the corresponding IC/LC image when photoactivating IC-encoders compared to LC-encoders and segment responders, even when the amount of visual information contained in the holographic ensembles were matched (Extended Data Fig. Together, these results indicate that IC-encoders are distinctively potent at driving pattern completion of IC representations within V1L2/3. The 2p holographic optogenetics experiments described above suggest a key new component to the conventional model of IC encoding (Fig. Consistent with the conventional model, our conceptual model proposes that segment responders receive bottom-up sensory signals and IC-encoders receive top-down inference signals16,17. Our data suggest a critical addition to this conceptual model—IC-encoders locally broadcast the top-down inference signal within the V1L2/3 network. Taken together, this leads to the prediction that segment responders have a primary role in feedforward propagation, while IC-encoders have a primary role in local reinforcement of inferential signals (Fig. If this is true, photoactivating segment responders should promote IC representations in higher visual areas, while IC-encoders should not. To test this hypothesis, we needed the technical ability to read out not just the local effects of holographic stimulation, but also the downstream effects. This platform allowed us to holographically stimulate V1 neurons while observing network effects in several downstream higher visual areas simultaneously (Fig. V1 segment responders relay feedforward information to higher visual areas; V1 IC-encoders receive feedback from higher visual areas and locally broadcast the top-down information to the rest of the V1L2/3 network. b, Left, mesoscope field-of-view (2.4 × 2.4 mm2; Vglut1-CreXAi203 mouse; 11 sessions across two mice). Right, ‘influence map' of holography-evoked responses in an example 2p holographic mesoscope experiment (targets are marked with green ‘x'). Holographic field-of-view (dotted rectangle) was positioned over V1. Putative visual area borders are demarcated. d,e, Decoders trained on visual evoked activity (0–1 s relative to visual onset) were used to classify holography-evoked activity (0–1 s relative to holography offset; 11 sessions across two mice expressing GCaMP7s–ChroME in excitatory neurons). For each holographically targeted ensemble, only the sessions with ≥10 targets were analyzed. Box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5× interquartile range; points, outliers. d, V1 IC-encoder ensembles were targeted for photoactivation. Left, when decoding all neurons in the field-of-view, holography-evoked activity patterns were decoded as the corresponding visual images (n = 11 IC-encoder ensembles; mean ± s.e.m. = 16 ± 1 targets, P = 0.0010 one-tailed Wilcoxon signed-rank test compared to chance). When decoding nonphotoactivated neurons in higher visual areas (all neurons in the field-of-view outside of V1), holography-evoked activity patterns were not decoded as corresponding visual images (P = 0.3501; median performance = 0.2750, Q1–Q3 = 0.2169–0.2893). e, Same as in d, when targeting V1 segment responder ensembles for photoactivation (n = 22 segment responder ensembles, mean ± s.e.m. = 21 ± 1 targets; left decode all neurons, P = 0.0047, right decode nonphotoactivated neurons in higher visual areas, P = 0.0457; median performance = 0.2780, Q1–Q3 = 0.2340–0.3060). First, as above, decoding from all neurons in the mesoscopic field-of-view confirmed that photoactivation of V1 IC-encoder and segment responder ensembles evoked activity patterns that partially emulated those evoked by the corresponding images (analysis was restricted to holographic ensembles with n ≥ 10 targets; Fig. 6d, left; n = 11 IC-encoder ensembles with mean ± s.e.m. 6e, left, n = 22 segment responder ensembles with 21 ± 1 targets). Next, we decoded only from nonphotoactivated neurons in higher visual areas to test whether photoactivating IC-encoders or segment responders recreated IC representations in downstream regions. In stark contrast to pattern completion effects within V1L2/3 (Fig. 5d,e, right), V1 segment responders, but not V1 IC-encoders, drove IC representation in higher visual areas (Fig. These data demonstrate that V1 segment responders drive feedforward communication of the bottom-up sensory information to higher visual areas, while V1 IC-encoders drive pattern completion within V1L2/3 that can reinforce this process. Taken together, our data establish recurrent looped connectivity among V1 IC-encoders, V1 segment responders and higher visual areas. We combined large-scale electrophysiology, 2p imaging, 2p holographic optogenetics and 2p holographic mesoscope to establish a direct, causal link between neural pattern completion in the primary V1 and sensory inference. We identified a subpopulation of visual cortical neurons that specifically encode ICs as emergent features of the image. Next, by leveraging the large numbers of simultaneously recorded neurons, we developed a multivariate approach to identify patterns of neural activity that encode sensory inferences. Using this new ‘inference decoding' paradigm, we found strong representations of IC inference in L2/3 of V1 and LM. Therefore, V1L2/3 is the lowest area in the visual hierarchy to be involved in IC inference computation. Further analysis of inference decoding in V1L2/3 revealed that IC-encoders mediate the representation of the inferred illusory contour. Strikingly, we found that 2p holographic stimulation of IC-encoders drove neural pattern completion within V1L2/3, in the absence of visual inputs. While segment responders did not induce IC representation within V1L2/3, further examination of the visual cortical hierarchy using the 2p holographic mesoscope revealed that segment responders drove IC representations in higher visual areas. Prior literature emphasized the role of top-down feedback in generating IC inference signals in V1 (refs. Although this top-down feedback explains why IC responses emerge in V1, it does not elucidate their potential role, if any. Our findings fill in this gap by highlighting pattern completion within V1 as a critical missing step in existing models of IC encoding16,17. This pattern completion in V1 may help to ensure that only visual signals that match prior expectations are selectively reinforced during recurrent activity between V1 and higher visual areas. Our experimental demonstration of recurrent looped connectivity in the visual hierarchy confirms a critical prediction from computational studies of IC inference. Previous studies have found that conventional convolutional neural networks (CNNs) with purely feedforward architectures fail to ‘perceive' ICs, but adding in recurrent looped connectivity leads to the emergence of IC inference27,28,29. Thus, our results provide definitive biological evidence for a key prediction from artificial neural network (ANN) studies—recurrent looped connectivity in the visual hierarchy is essential for IC inference. Such structured patterns can even emerge during spontaneous activity34,35. These studies revealed that cortical circuits are primed for pattern completion, likely due to structured connectivity. However, no prior work has studied neocortical pattern completion, nor its underlying connectivity, in the context of sensory inference. Our study suggests a vital function for neocortical pattern completion in sensory inference. We propose that the recurrent pattern completion underlying IC encoding is ubiquitous in active perceptual processing of the external world. In these situations, successful perception depends on sensory inference. For example, humans are capable of fast and accurate object recognition even when objects are occluded41. Similar to ICs, top-down predictions about occluded objects are thought to be implemented by feedback connections from higher sensory areas21,23,42. Reinforcement of top-down predictions in lower sensory areas through recurrent pattern completion likely also has a critical role in visual object recognition during occlusion41. More generally, feedback-driven pattern completion in lower cortical areas may be a universal mechanism of sensory inference. This pattern completion may generate recurrent activity spanning lower and higher sensory cortices, iteratively refining inference computation by selectively reinforcing neural activity patterns that match prior expectations. Taken together, holistic percepts can emerge despite sensory ambiguity through recurrent pattern completion in the neocortical sensory hierarchy. All experiments were performed in C57/B6 mice of both sexes, aged 6 weeks and older. The 2p holographic optogenetics experiments in Figs. 4 and 5 were conducted with AAV8-hSyn-GCaMP6m-p2A-ChRmine-Kv2.1-WPRE injected in PV-IRES-Cre;RCL-tdTomato mice (n = 2) and AAV9-2YF-hSyn-DIO-GcaMP6m-P2A-ChRmine-Kv2.1-WPRE in Emx1-Cre mice (n = 2). Furthermore, 2p holographic mesoscope experiments in Fig. 6 were conducted with Vglut1-Cre;Ai203 mice (Ai203 refers to a transgenic line expressing the transgene TITL-st-ChroME-GCaMP7s-ICL-nls-mRuby3-IRES2-tTA2; n = 2). All experiments on animals were conducted with approval of the Animal Care and Use Committee of the University of California, Berkeley (imaging and 2p holographic optogenetics experiments) and the Allen Institute's Institutional Animal Care and Use Committee (Neuropixels experiments). Mice were anesthetized with isoflurane (2%) and mounted on a stereotaxic apparatus. Glass coverslips of 3–5 mm diameter were placed on top of the craniotomy and sealed with dental cement (C&B Metabond). These glass coverslips, or ‘windows', were made beforehand by gluing two layers of round cover glass (for example, 1 × 3 mm + 1 × 5 mm, Warner Instruments, 1 thickness) with Norland Optical Adhesive 71 (UV-cured). Finally, a titanium headplate was fixed to the skull with dental cement (C&B Metabond). The dental cement was coated in a layer of black oxide to mitigate light leakage during 2p imaging. For 2p holographic optogenetics experiments, we injected the GCaMP6m-p2A-ChRmine-Kv2.1 virus during the window implant surgery (500 nl per two depths and two sites). Injection speed was 100–250 nl min−1, and after injection in each site, pipette was held in place for 5 min. Injection sites were centered at 2.7 mm lateral, 0.2 mm anterior relative to lambda and injection depths were 120 µm and 250 µm. Briefly, the mouse underwent headplate implantation and craniotomy surgery, in a procedure similar to the one described above. The 5-mm craniotomy was sealed with a glass coverslip, the bottom of which was coated with silicone to reduce adhesion to the brain surface. Intrinsic signal imaging was performed to identify visual area boundaries46. Based on the identified visual areas, an insertion window was designed for each mouse. Mice were habituated to head fixation and visual stimulation over a period of 2 weeks. On the day of recording, the cranial coverslip was removed and replaced with the insertion window containing holes aligned to six cortical visual areas. Mice were allowed to recover for 1–2 h after insertion window placement, before being head-fixed in the recording rig. Six Neuropixels probes were targeted to each of the six visual cortical areas (V1, LM, RL, AL, PM and AM). Probes were doused with CM-DiI (1 mM in ethanol; Thermo Fisher Scientific, V22888) for post hoc ex vivo probe localization. Each probe was mounted on a 3-axis micromanipulator (New Scale Technologies). The tip of each probe was aligned to its associated opening in the insertion window guided by an Indian Statistical Institute map of the visual system. After the probes penetrated the brain to a depth of around 100 μm, they were inserted automatically at a rate of 200 μm min−1 (total of 3.5 mm or less in the brain). After the probes reached their targets, they were allowed to settle for 5–10 min. Neuropixels data were acquired at 30 kHz (spike band, 500-Hz high-pass filter) and 2.5 kHz (LFP band, 1,000-Hz low-pass filter) using the Open Ephys GUI43. Videos of the eye and body were acquired at 30 Hz. The spike-band data are median-subtracted, first within-channel to center the signal around zero, then across channels to remove common-mode noise. The median-subtracted data file is sent to the Kilosort2 MATLAB package (https://github.com/mouseland/Kilosort2)47, which applies a 150-Hz high-pass filter, followed by whitening in blocks of 32 channels. Kilosort2 models this filtered, whitened data as a sum of spike ‘templates'. The shape and locations of each template are iteratively refined until the data can be accurately reconstructed from a set of N templates at M spike times. Finally, Kilosort2 output is curated to remove putative double-counted spikes and units with artefactual waveforms. All units not classified as noise were packaged into Neurodata Without Borders files and analyzed in this paper. As our focus was on excitatory neurons, we analyzed regular-spiking (RS) units in the Neuropixels dataset, defined as spike-sorted single units with extracellular spike trough-to-peak width of ≥0.4 ms. For brevity, we referred to RS units as ‘neurons'. The 2p imaging experiments presented in Figs. V1L2/3 and V1L4 imaging was done on a 2p microscope (Neurolabware with Scanbox GUI) with a Ti:sapphire laser (Chameleon Ultra II, Coherent), an electrotunable lens for multiplane imaging (Optotune), 8 kHz galvo-resonant scanner and a Nixon ×16 magnification 12.5-mm focal length 0.8 NA water immersion objective. Typical image settings were as follows: field-of-view size 1,223 × 675 μm2 (796 × 512 pixels), four planes spaced 50 μm apart sampled at volumetric frame rate 7.715 Hz, imaging wavelength 930 nm. Mesoscope 2p imaging was done on 2p random access mesoscope (Thorlabs with ScanImage GUI), equipped with a Ti:sapphire laser (Mai Tai, Spectra Physics), remote-focusing unit consisting of an objective and a mirror mounted on a voice coil, 12 kHz galvo-resonant scanner and Jenoptiks/Thorlabs water immersion objective with 21 mm focal length and 0.6 excitation NA and 1.0 collection NA. Typical image settings were as follows: field-of-view size 3,031 × 2,965 μm2 (3,040 × 1,484 pixels), frame rate 3.139 Hz, imaging wavelength 920 nm. During post hoc analysis, motion correction and source extraction were done using Suite2p48. Suite2p output putative regions of interest (ROIs) that correspond to neuronal somas, and these ROIs were manually inspected to keep only those that looked like cell bodies. Neuropil signal was subtracted from each ROI with a coefficient of 0.7 (FC = FROI − 0.7 × Fneu). For brevity, we referred to manually curated ROIs in the 2p data as ‘neurons'. For 2p holographic optogenetics experiments presented in Fig. 5, three-dimensional (3D) scanless holographic optogenetics with temporal focusing (3D-SHOT) path was implemented in a Sutter MOM (Movable Objective Microscope, Sutter Instrument Co., with ScanImage GUI), as described previously26,49,50,51,52. Ti:sapphire laser (Chameleon Ultra II, Coherent) was used for 2p imaging, and femtosecond fiber laser (Satsuma HP2, 1030 nm, 2 MHz, 350 fs; Amplitude Systems) was used for 2p holographic stimulation. The holography path included a blazed diffraction grating for temporal focusing (600 l mm−1, 1,000 nm blaze; Edmund Optics, 49-570 or 33010FL01-520R; Newport), and a rotating diffuser to randomize the phase pattern and expand the beam. Both the imaging path and the holography path had spatial light modulators (SLM; HSP1920, 1,920 × 1,152 pixels; Meadowlark Optics). The imaging path SLM enabled optical axial focusing. The SLM was conjugated to the X resonant galvo and optical axial focusing was achieved by loading Fresnel-lens-like phase patterns on the SLM for each desired axial depth. The holography path SLM was used to display the holographic phase mask, calculated using the Gerchberg–Saxton algorithm to place several cell-sized diffraction-limited spots in 3D target positions50. A zero-order block was placed in the holography path. Precise alignment of the imaging path and the holography path is achieved by a calibration procedure described previously26,49,50. Briefly, we place a thinly coated fluorescent slide on a substage camera to image both the imaging planes and the holograms at parametrically varied SLM coordinates. Then, the 3D coordinate transformation from imaging coordinates on each plane to SLM coordinates is computed through the ‘polyfitn' function in Matlab. Before each experiment, the alignment is confirmed for each remote-focused plane by ‘burning holes' on a fluorescent slide (photobleaching). The intended imaging coordinates are displayed on ScanImage (Vidrio) as Integration ROIs. As described in the ‘Results', the ‘all-optical read-write' 2p holographic optogenetic experiments were conducted in three stages. First was the ‘read' stage, where we image visual responses of neurons in the V1L2/3 field-of-view. Second was the online analysis stage, where we ran Suite2p on the imaging data collected in the first stage and identified functional groups of neurons (IC-encoders, segment responders). Third was the ‘write' stage, where we holographically stimulated these functionally identified neuron groups. Throughout all stages of the experiment, the mouse was head-fixed under the objective, and the imaging conditions were kept constant (same field-of-view of size 980 × 980 μm2, same set of four planes spaced 30 μm apart, with each plane being imaged every four frames at 7.525 Hz volumetric frame rate, imaging wavelength of 920 nm). For fast online analysis, we converted ScanImage Tiff files into four h5 files, generating one h5 file per plane. On an eight-core CPU (Intel Core i7 10700K 3.8 GHz) with 128 GB RAM, online Suite2p of a 25 min imaging data (512 × 512 pixels) took eight minutes. Then, all ROIs identified by the Suite2p were analyzed in Matlab to identify functional groups of neurons (IC-encoders and segment responders). Because IC-encoders are sparse, we targeted every neuron identified as IC-encoders during the online analysis step. For segment responders, we balanced the number of segment responders in the four groups (BR, BL, TL and TR). Responsiveness was quantified as the area under receiver operating curve in the ideal observer analysis discriminating between neural responses on inward vs. outward segment image trials. Mean number of targets in each holographic ensemble were as follows: 17 IC1-encoders, 18 IC2-encoders, 22 BR + TL segment responders 22 BL + TR segment responders. After determining the holography groups with online analysis, holograms containing all targets in each group were calculated using the Gerchberg–Saxton algorithm50. The holograms were diffraction efficiency corrected to achieve homogeneous power delivery of 7 mW to all targeted cells. Holographic stimulation on each trial consisted of ten pulses at 10 Hz with 10 ms pulse width, and the ITI was 4.5 s. Stimulation of each holography group was repeated 50 times. On average, 1,968 ROIs were classified as putative neurons in the four-plane volumetric field-of-view. Targets were identified as the neuron closest to each target coordinate. Because online analysis does not entail manual curation of the ROIs, a target may not correspond to a neuron. Thus, a target was considered valid only if the closest neuron was within 10 µm, which left 81.5% of targets (mean number of valid targets for each holography group was 15 IC1-encoders, 16 IC2-encoders, 18 BR + TL segment responders and 17 BL + TR segment responders). Of these, holographic stimulation evoked substantial activation in 91.6% of valid targets (mean number of stimulated targets for each holography group was 14 IC1-encoders, 14 IC2-encoders, 16 BR + TL segment responders and 16 BL + TR segment responders). Nonphotoactivated neurons were defined as neurons >50μm from all holographic targets in each ensemble; distance threshold informed by the axial physiological point spread function in Extended Data Fig. Full details are reported in a separate paper18. The holographic field-of-view was placed over V1 (gray-dotted rectangle in Fig. The imaging field-of-view encompassed V1, LM, RL, AL, PM and AM (2,400 × 2,400 m2). The imaging and holographic field-of-view was fixed throughout each recording session. During holographic ensemble stimulation, each target received 50 mW, and each stimulation consisted of ten pulses of 10 ms duration at 16 Hz. Each trial type was repeated 100 times with an ITI of 5 s. Visual stimuli were generated using custom scripts based on PsychoPy62 and were displayed using an ASUS PA248Q LCD monitor, with 1,920 × 1,200 pixels (55.7-cm wide, 60-Hz refresh rate). For all 2p experiments, visual stimuli were generated using custom Matlab scripts based on Psychtoolbox-3 (ref. In the standard 2p imaging setup, a 19-inch Samsung monitor was placed 15 cm from the mouse's right eye (1,280 × 1,024 pixels, 37.6 × 30.1 cm width and height, 60-Hz refresh rate). In addition, in all 2p experiments, electrical tape was applied between the objective and the mouse's headplate to prevent monitor light from contaminating 2p imaging. In the mesoscope 2p imaging, retinotopic mapping was additionally used to determine visual area boundaries using visual field sign maps54,55. After settling on the imaging field-of-view at the beginning of each 2p experiment, retinotopic mapping was conducted with 16° rectangular drifting grating patches appearing in one of 25 positions tiling an 80° × 80° visual degree grid. These patches had spatial frequency of 0.04 cycles per degree and a temporal frequency of 2 Hz, cycling through eight directions (0°/45°/90°/135°/180°/225°/270°/315°) over the course of 2 s, followed by a 1 s gray-screen intertrial interval (ITI). Trial order was randomized within each block. In Neuropixels experiments and a subset of 2p imaging experiments, these images were also shown rotated 315° clockwise in a separate block (IC configuration 0; Extended Data Fig. The black bar length on each inducer segment was 16 visual degrees, such that the IC support ratio in IC images would be 2/3. Edges delimiting the real bar in the real edge images (IRE/TRE/XRE) were at least 2° thick, which is well over the grating acuity threshold (0.55 cycles per degree) and, in fact, close to the maximum sensitivity of mouse vision (0.2 cycles per degree)19. Throughout each IC block, the four white circles stayed in place even during the ITI. In IC configuration 1, each image was repeated at least 50 times, and IC and LC images were repeated 400 times; in IC configuration 0, each image was repeated 30 times. Each image was repeated at least 50 times, and IC and LC images were repeated 400–500 times. 1, each image was presented for 1 s followed by a variable ITI of 1–2 s. Each image was repeated 10 times in 9 sessions and 50 times in 20 sessions. In the receptive field mapping block, circular patches of drifting gratings with a fixed diameter of 16 visual degrees were presented in nine different locations, as shown in Extended Data Fig. Drifting gratings had a spatial frequency of 0.04 cycles per degree and a temporal frequency of 2 Hz. The drifting gratings had 100% contrast and were presented on a gray background. On each trial, a circular patch of drifting grating appeared in one of nine positions and cycled through several directions spaced 45° apart, drifting in each direction for 0.25 s (4/8 directions lasting 1/2 s in Neuropixels/2p experiments, respectively). Presentation at each location was repeated ten times, with 0 s ITI. In the size tuning block, circular patches of drifting gratings were presented with a fixed position at the center of the monitor, in different sizes of 0/4/8/16/32/64 visual degrees. Spatial frequency, temporal frequency and contrast of drifting gratings were identical to the receptive field mapping block. On each trial, a circular patch of drifting grating at a given size appeared in one of eight directions (0°/45°/90°/135°/180°/225°/270°/315°). In Neuropixels experiments, the presentation lasted 0.25 s followed by 0.5 s ITI, and each combination of size and direction was repeated 10 times. In 2p imaging experiments, the presentation lasted 1 s followed by 0 s ITI, and each combination of size and direction was repeated 10 times. The size tuning block was skipped in 2p holographic optogenetics experiments. For results presented in Extended Data Fig. Fixed-gaze trials were defined as trials where pupil position was within eight visual degrees of the resting pupil position throughout the duration of the trial. Exclusively center-responsive neurons were defined as neurons that exclusively responded to circular grating patches in the center position, corresponding to the illusory gap region, and not to any of the other adjacent positions (Fig. 1; Wilcoxon rank-sum test between trials with circular grating patches in each position and gray-screen trials, Bonferroni–Holm correction for multiple comparisons). In the Neuropixels data, there were n = 18 exclusively center-responsive neurons of 1,804 V1 RS units. IC-encoders were defined as neurons that respond to one of the IC images, and to neither of the LC stimuli (Kruskal–Wallis test across responses to ‘blank' (four white circles) vs. IC1 vs. LC1 vs. LC2 vs. IC2 with Tukey–Kramer correction for multiple comparisons). Across ICwcfg0 and ICwcfg1, 159 of 1,804 V1 RS units were identified as IC-encoders in the Neuropixels dataset; 1,569 of 18,576 V1L2/3 excitatory neurons were identified as IC-encoders in the 2p dataset. Segment responders were defined as neurons that selectively responded to inward-pointing inducer segments in each inducer position. For example, BR-segment responders are defined as neurons that respond substantially more to InBR than OutBR (Wilcoxon rank-sum test; Fig. For all decoding analyses presented in the main figures (Figs. First, we constructed the trial-by-trial response matrix (trials x neurons) as the response averaged over the visual stimulus presentation period (firing rate for Neuropixels data, z-F for 2p data). We divided the trials with the training trial types (IC1/LC1/LC2/IC2) into a training set (90%) and a testing set (10%). For tenfold cross-validation, there were ten splits of training/testing sets, and the ten sets of test trials were nonoverlapping. On each iteration, SVM was trained using the ‘fitcecoc' function in MATLAB. Cross-validation accuracy was computed as the SVM's prediction accuracy on the testing set, averaged across ten iterations. As shown by the physiological point spread function (PPSF; Extended Data Fig. 8a), all direct influence of holographic light dissipates by 50 μm. The holographic mesoscope data were analyzed similarly, but with two adjustments. The 2p random access mesoscope patches together multiple imaging regions, and holographic artifacts appear on either side of each imaging region. Thus, we did not analyze the holographic stimulation period. Second, nonphotoactivated neurons were defined as neurons >25 μm from targets in each holographic ensemble; distance threshold was informed by the lateral physiological point spread function in this case because the mesoscopic field-of-view was imaged at a single depth. 5 and 6, nonphotoactivated neurons are defined for each holographic ensemble. Hence, the decoders for nonphotoactivated neurons consist of different sets of neurons for each ensemble. To assess whether these differences in decoder population affected the results, we also tried decoding off of a common pool of nonphotoactivated neurons (Extended Data Fig. The results remain the same as when decoding different sets of nonphotoactivated neurons for each holographic ensemble (Extended Data Fig. SVMs were trained with different kernels—linear vs. quadratic polynomial vs. radial basis function. In addition, a fully connected artificial neural network (ANN) was trained using Pytorch, similar to the decoding described in ref. The ANN had an input layer with number of nodes matching the number of neurons, two hidden layers each with 100 nodes, and one output layer with number of nodes matching the number of training trial types (that is, 4 nodes, each corresponding to IC1/LC1/LC2/IC2). Softmax function was applied to the output layer. All decoders underwent tenfold cross-validation, as described in detail above. To account for cofluctuations in rate, smoothed cross-correlogram was calculated from spike trains smoothed with a 25-ms window. To determine whether there was a putative excitatory connection between a pair of units, peaks in corrected cross-correlogram were found in 1–10-ms window. Nonparametric two-sided tests are used unless otherwise noted. Adjustments for multiple comparisons have been noted wherever they were applied. When applicable, we used cross-validation to avoid overfitting. No statistical method was used to predetermine sample size, but we used an equivalent or larger sample size than similar studies18,26,58,59. Trial order within each block was randomized. Analysis of Neuropixels data was performed blind to collection and spike sorting. Manual curation of 2p imaging data (Suite2p) was performed blind to visual and holography-evoked responses. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Additional postprocessed data are available at https://doi.org/10.24433/CO.6659254.v2. Analysis codes for generating all figures are available at https://doi.org/10.24433/CO.6659254.v2. Kanizsa, G., Renzi, P., Conte, S., Compostela, C. & Guerani, L. Amodal completion in mouse vision. Seeing more than meets the eye: processing of illusory contours in animals. & Wagner, H. Perception and neuronal coding of subjective contours in the owl. Fuss, T., Bleckmann, H. & Schluessel, V. The brain creates illusions not just for us: sharks (Chiloscyllium griseum) can ‘see the magic' as well. Okuyama-Uchimura, F. & Komai, S. Mouse ability to perceive subjective contours. A. Top-down feedback controls the cortical representation of illusory contours in mouse primary visual cortex. Gatto, E., Loukola, O. J., Petrazzini, M. E. M., Agrillo, C. & Cutini, S. Illusional perspective across humans and bees.Vision (Basel) 6, 28 (2022). Von der Heydt, R., Peterhans, E. & Baumgartner, G. Illusory contours and cortical neuron responses. Peterhans, E. & von der Heydt, R. Mechanisms of contour perception in monkey visual cortex. Grosof, D. H., Shapley, R. M. & Hawken, M. J. Macaque V1 neurons can signal ‘illusory' contours. Mendola, J. D., Dale, A. M., Fischl, B., Liu, A. K. & Tootell, R. B. The representation of illusory and real contours in human cortical visual areas revealed by functional magnetic resonance imaging. Murray, M. M. & Herrmann, C. S. Illusory contours: a window onto the neurophysiology of constructing perception. Lee, T. S. & Nguyen, M. Dynamics of subjective contour formation in the early visual cortex. Saeedi, A. et al. Brightness illusions drive a neuronal response in the primary visual cortex under top-down modulation. & O'Reilly, R. C. Early recurrent feedback facilitates visual object recognition under challenging conditions. Lee, T. S. Computations in the early visual cortex. & Adesnik, H. Probing inter-areal computations with a cellular resolution two-photon holographic mesoscope. Prusky, G. T. & Douglas, R. M. Characterization of mouse cortical spatial vision. Lyall, E. H. et al. Synthesis of a comprehensive population code for contextual features in the awake sensory cortex. Keller, A. J., Roth, M. M. & Scanziani, M. Feedback generates a second receptive field in neurons of the visual cortex. Inception loops discover what excites neurons most using deep predictive models. Siegle, J. H. et al. Survey of spiking in the mouse visual system reveals functional hierarchy. Fujisawa, S., Amarasingham, A., Harrison, M. T. & Buzsaki, G. Behavior-dependent short-term assembly dynamics in the medial prefrontal cortex. Baker, N., Erlikhman, G., Kellman, P. & Lu, H. Deep convolutional networks do not perceive illusory contours. Dileep, G. et al. A detailed theory of thalamic and cortical microcircuits for predictive visual inference. & Yuste, R. Controlling visually guided behavior by holographic recalling of cortical ensembles. Marshel, J. H. et al. Cortical layer-specific critical dynamics triggering perception. Burst spiking of a single cortical neuron modifies global brain state. Houweling, A. R. & Brecht, M. Behavioural report of single neuron stimulation in somatosensory cortex. Spontaneously emerging cortical representations of visual attributes. Sugden, A. U. et al. Cortical reactivations of recent sensory experiences predict bidirectional network changes during learning. Ko, H. et al. Functional specificity of local synaptic connections in neocortical networks. Rossi, L. F., Harris, K. D. & Carandini, M. Spatial connectivity matches direction selectivity in visual cortex. Lee, W. C. et al. Anatomy and function of an excitatory network in the visual cortex. Fisek, M. et al. Cortico-cortical feedback engages active dendrites in visual cortex. Ding, Z. et al. Functional connectomics reveals general wiring rule in mouse visual cortex. Tang, H. et al. Recurrent computations for visual pattern completion. Kirchberger, L., Mukherjee, S., Self, M. W. & Roelfsema, P. R. Contextual drive of neuronal responses in mouse V1 in the absence of feedforward input. Siegle, J. H. et al. Open Ephys: an open-source, plugin-based platform for multichannel electrophysiology. Acute head-fixed recordings in awake mice with multiple Neuropixels probes. & Callaway, E. M. Automated identification of mouse visual areas with intrinsic signal imaging. Pachitariu, M. et al. Suite2p: beyond 10,000 neurons with standard two-photon microscopy. Pegard, N. C. et al. Three-dimensional scanless holographic optogenetics with temporal focusing (3D-SHOT). Precise multimodal optical control of neural ensemble activity. High-performance microbial opsins for spatially and temporally precise perturbations of large neuronal networks. Bounds, H. A. et al. All-optical recreation of naturalistic neural activity with a multifunctional transgenic reporter mouse. Garrett, M. E., Nauhaus, I., Marshel, J. H. & Callaway, E. M. Topography and areal organization of mouse visual cortex. Zhuang, J. et al. An extended retinotopic map of mouse cortex. Mathis, A. et al. DeepLabCut: markerless pose estimation of user-defined body parts with deep learning. Stringer, C., Michaelos, M., Tsyboulski, D., Lindo, S. E. & Pachitariu, M. High-precision coding in visual cortex. LaFosse, P. K. et al. Cellular-resolution optogenetics reveals attenuation-by-suppression in visual cortical neurons. Adaptation, not prediction, drives neuronal spiking responses in mammalian sensory cortex. The authors thank J. Beyer and K. Gopakumar for technical suppornt, and M. Li (UC Berkeley Vision Science Core Gene Delivery Module Facility) for viral preparations. This work was funded by the National Institutes of Health (NIH; U19NS107613, R01EY023756 and RF1MH120680) and Chan Zuckerberg Biohub Investigator Award to H.A. ; the Weill Neurohub Fellowship, Seoul National University New Faculty Startup Fund, the National Research Foundation of Korea (NRF; RS-2023-00301976, RS-2024-00358070, RS-2024-00413689) and Samsung Science and Technology Foundation (SSTF-BA2302-07 to H.S. The Neuropixels dataset was obtained as part of the OpenScope Brain Observatory, which is operated by the Allen Institute for Neural Dynamics, with funding provided by NIH under grant U24NS113646. The authors also thank the Allen Institute founder, P. G. Allen, as well as C. Koch and K. Svoboda, for their vision, encouragement and support. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Hyeyoung Shin, Mora B. Ogando, Lamiae Abdeladim, Uday K. Jagadisan & Hillel Adesnik Allen Institute, Neural Dynamics Program, Seattle, WA, USA Severine Durand, Ben Hardcastle, Hannah Belski, Hannah Cabasco, Henry Loefler, Ahad Bawany, Josh Wilkes, Katrina Nguyen, Lucas Suarez, Tye Johnson, Warren Han, Ben Ouellette, Conor Grasso, Jackie Swapp, Vivian Ha, Ahrial Young, Shiella Caldejon, Ali Williford, Peter A. Groblewski, Shawn Olsen, Carly Kiselycznyk & Jerome Lecoq Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed all 2p experiments, including 2p imaging, 2p mesoscope imaging, 2p holographic optogenetics and 2p holographic mesoscope experiments. assisted with the design and execution of 2p holographic optogenetic experiments. L.A. designed and built the 2p holographic remote-focusing microscope, with assistance from M.B.O. assisted with recordings and technical support. performed behavioral habituation for Neuropixels experiments. and S.C. performed Intrinsic signal imaging for Neuropixels experiments. managed various aspects of the Neuropixels experiments. managed all aspects of the OpenScope collaboration. Correspondence to Hyeyoung Shin or Hillel Adesnik. has a patent related to this work—3D sparse holographic temporal focusing, 2016, L.W., N.P. Nature Neuroscience thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic of visual stimulus sequences during the experiment. Throughout each IC visual block, the four circles were present in all stimulus frames. The default block, ICwcfg1, has four white circles in each quadrant on black background. In the ICwcfg0 block, images are rotated 315° clockwise relative to ICwcfg1. c, Images shown in a ICwcfg1 block. a, Top, pupil position on an example session. IC1 vs. LC1 vs. LC2 vs. IC2 images are shown in the background for relative scale. Mouse eye gaze is confined to a small region that fits within the illusory gap region. There are no systematic differences in pupil position among IC1 vs. LC1 vs. LC2 vs. IC2 presentations. When analyzing only fixed-gaze trials, there were n = 11 exclusively center-responsive V1 RS units; 10 sessions had eye tracking data and of these, 9 sessions had a valid receptive field mapping block where the fixed-gaze constraint left at least one trial in each position. c, PSTH of exclusively center-responsive V1 RS units on IC and IRE trials, for preferred IRE orientation and corresponding IC and segment image trials (mean ± s.e.m. These neurons respond to IRE images and less strongly to IC images, but not to the individual segments on either side of the illusory contour. d, PSTH on IC and IRE trials for preferred IRE orientation and corresponding IC trials (top: mean ± s.e.m. e, Top, comparison of the preferred orientation between ICs (IC stimuli) and real edges (IRE stimuli) for exclusively center-responsive V1 RS units. IRE orientations among exclusively center-responsive V1 RS units. Actual match proportion is indicated with a red dotted line. Match proportion is 0.81 among 398 V1 RS units with significant receptive fields (p < 0.05 Kruskal–Wallis test across responses to 9 positions in the receptive field mapping block). g, Pupil area, z-scored across each recording session, during visual presentation of various images in the IC block (n = 10 sessions with eye tracking data; p = 0.7220 Friedman test). Box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5× interquartile range; circles, outliers. a, 2p imaging sessions with IC block images that had larger gaps between white circles; the gap size between diagonal white circles was 24 visual degree gaps (4 sessions from 2 mice). b, Receptive fields were mapped with 16 visual degree circular patches in 9 positions, spaced 16 visual degrees apart, same as Extended Data Fig. There were n = 32 exclusively center-responsive V1 layer 2/3 excitatory neurons. c, PSTH on IC and IRE trials for preferred IRE orientation and corresponding IC trials (top: mean ± s.e.m. d, Top, comparison of the preferred orientation between ICs (IC stimuli) and real edges (IRE stimuli) for exclusively center-responsive V1 layer 2/3 excitatory neurons. Bottom, CDF for match proportions in shuffled distributions is shown as a black solid curve, actual match proportion in red dotted line. e, 2p imaging sessions with dense receptive field mapping and standard IC block images (30 visual degree white circles and 16 visual degree gaps; 5 sessions from 2 mice). f, Receptive fields were mapped with 16 visual degree circular patches in 17 positions, spaced 8 visual degrees apart (dense mapping, as in Extended Data Fig. Of note, evoked response heatmaps are not drawn to proportion (heatmap circles have half the diameter of the actual circular patches used for mapping). Receptive field positions were determined for neurons with p < 0.05 Kruskal–Wallis test for responses across circular patch positions. There were n = 71 V1 layer 2/3 excitatory neurons with receptive fields in the center position. g,h, Same as c and d, but for V1 layer 2/3 excitatory neurons with receptive fields in the center position in dense mapping sessions. i, 2p imaging sessions with IC blocks that had gray background (50% contrast), with 30 visual degree white circles and 16 visual degree gaps (7 sessions from 3 mice). Top, these blocks contained contrast real edge images (CRE) in addition to outline real edge images (IRE). Bottom, PSTHs of exclusively center-responsive neurons show similar evoked responses to CRE and IRE, and smaller evoked responses to IC (mean ± s.e.m. across n = 38 exclusively center-responsive V1 layer 2/3 excitatory neurons). j, Receptive fields were mapped with 16 visual degree circular patches in 9 positions, spaced 16 visual degrees apart, same as b. k,l, Same as c and d, but for CRE trials instead of IRE trials in 50% contrast sessions. m,n, Same as c and d, but for 50% contrast sessions. a, Receptive fields (RF) of all neurons were mapped using 16-degree patches of circular drifting gratings appearing in one of the nine positions depicted (top). Note, 16 degrees is slightly larger than the typical size of receptive fields in mouse V1. b, Example IC1-encoder (top) and IC2-encoder (bottom), showing its receptive field (left), PSTHs to IC and LC images (center), and PSTHs to inward segment images (right). PSTHs are calculated in 1 ms bins and smoothed with a Gaussian kernel that has an s.d. c, Receptive fields percentage in each position for IC-encoders (left, two panels) and segment responders (right, four panels). IC-encoders' receptive fields are not biased toward the illusory gap region (n = 367/386 IC1/IC2-encoders in 24 sessions from 4 mice). On the contrary, segment responders' receptive fields are biased toward the image segments that are used to define them (n = 552/482/409/410 BR/BL/TL/TR segment responders in 24 sessions from 4 mice). d, Positions of circular patches used for dense receptive field mapping shown in cyan and yellow lines, relative to the IC block ‘blank' image (19 sessions from 5 mice). e, IC-encoders' receptive fields percentage in each position (n = 458/578 IC1/IC2-encoders). First and third panels show circular patches and their relative positions in actual proportions; second and fourth panels show the same data with smaller circular patches for clarity. f, IC block images were scaled up by 1.5× (10 sessions from 5 mice). Receptive fields were mapped with 16 visual degree square patches positioned in a 5 × 5 grid. g, IC-encoders' receptive fields percentage in each position (n = 129/126 IC1/IC2-encoders). Second and fourth panels show the receptive field distribution in red, and 95% bootstrapped confidence intervals of IC-responsive/LC-responsive neurons in black. h–i, Decoder performance for images scaled up by 1.5× (n = 10 sessions from 5 mice). Box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; circles, outliers. a, Illusory bars are perceived as dark bars on white-circles blocks (n = 11 sessions from 5 mice) and light bars on black-circles blocks (n = 9 sessions from 6 mice). b, IC-encoders' orientation tuning was measured using single lines that span the entire screen; dark bars were used for IC-encoders identified in white-circles blocks and light bars for black-circles blocks. For both white-circles blocks and black-circles blocks, IC-encoders are tuned to the orientation of the illusory bar that they are selective for (mean ± s.e.m. across neurons; white-circles block n = 93 IC1-encoders and n = 159 IC2-encoders, black-circles block n = 409 IC1-encoders and n = 482 IC2-encoders; Wilcoxon signed-rank test for orientation of the illusory bar vs. orthogonal orientation p = 0.0017 in the white-circles block, p = 2.6 × 10−8 in the black-circles block). c, Size tuning was measured using circular patches of drifting gratings in various sizes, positioned at the center of the monitor. Size tuning for neurons with receptive fields in the center position is shown in black (mean ± s.e.m. across n = 1638 neurons with p < 0.05 Kruskal–Wallis test across responses to 9 positions, maximal activity in the center); size tuning for IC-encoders with receptive fields in the center is shown in green (n = 12 center-receptive field IC-encoders in the white-circles block, n = 48 in the black-circles block). Receptive field size measured in this way is not distinct for IC-encoders compared to other visually responsive neurons. d, Circular patches of drifting gratings with 16 visual degree diameters were shown in 17 different positions spaced 8 visual degrees apart, as shown in Extended Data Fig. Number of patches that IC-encoders respond to is not different from IC-responsive neurons (error bars indicate 95% bootstrapped confidence interval). e, When decoding across all neurons recorded in each V1 layer 2/3 imaging session, TRE-evoked neural activity patterns are more likely to be decoded as IC than LC on both white-circles blocks (Wilcoxon signed-rank test p = 0.0020 across n = 11 sessions) and black-circles blocks (Wilcoxon signed-rank test p = 0.0273 across n = 9 sessions). across sessions, and gray lines indicate paired TRE inference decoding performance for each session. a, IC/LC images used to define IC-encoders and segment responders in this figure. b, For the Neuropixels dataset, proportion of V1 IC-encoders (left, n = 87) and V1 segment responders (right, n = 636) in each layer. 3d,e (left: of 12 sessions in Neuropixels dataset, sessions with <50 RS units are excluded, which leaves n = 12, 8, 11, 10, 12 and 11 sessions for V1, LM, RL, AL, PM and AM, respectively; right: n = 8/11/19 sessions for V1L4 FOV, V1L2/3 FOV, and mesoscope FOV, respectively. For each visual area, sessions with <100 neurons in that area are excluded, which leaves n = 8, 11, 19, 8, 15, 2, 6 and 1 sessions for V1L4, V1L2/3, mesoscope V1, LM, RL, AL, PM and AM, respectively). In c,d, box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; circles, outliers. Visually responsive neurons are defined as neurons that respond to at least one of IC/LC images (p < 0.05 Kruskal–Wallis test across responses to blank vs. IC1 vs. LC1 vs. LC2 vs. IC2). In e,f, response on IC vs. LC trials, averaged across all neurons (e) or visually responsive neurons (f) for each session (mean ± s.e.m. across sessions, paired data for each session indicated with gray lines). IC images do not evoke greater responses than LC images (*p < 0.05 one-tailed Wilcoxon signed-rank test for testing IC > LC; note that none of the visual areas showed significance). e, Response on IC vs. LC trials, averaged across all neurons. In Neuropixels dataset, one-tailed Wilcoxon signed-rank test for testing IC > LC returned p = 0.9614, 0.9805, 0.7676, 0.7217, 0.9895 and 0.8398 for V1, LM, RL, AL, PM and AM, respectively. f, Response on IC vs. LC trials, averaged across visually responsive neurons. In Neuropixels dataset, one-tailed Wilcoxon signed-rank test for testing IC > LC returned p = 0.9788, 0.9805, 0.7114, 0.7842, 0.9866 and 0.6499 for V1, LM, RL, AL, PM and AM, respectively. In a–c, box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; circles, outliers. 2, but showing statistical comparisons between areas (12 sessions from 12 mice; for each area in the Neuropixels dataset, sessions with <50 RS units are excluded, which leaves n = 12, 8, 11, 10, 12 and 11 sessions for V1, LM, RL, AL, PM and AM, respectively; Kruskal–Wallis test p = 0.0019 for cross-validation accuracy across areas, p = 0.0249 for TRE inference score). b, XRE inference decoding entails probing IC-evoked neural activity patterns with decoders trained to discriminate between XRE-evoked neural activity patterns. As with TRE inference decoding (Fig. 2), layer 2/3 of V1 and LM consistently show high inference scores (*p < 0.05, Wilcoxon signed-rank test compared to chance of 0.5; left: of 12 sessions in Neuropixels dataset, sessions with <50 RS units are excluded, which leaves n = 12, 8, 11, 10, 12 and 11 sessions with p = 0.0005, 0.0078, 0.0049, 0.0020, 0.0005 and 0.0137 for V1, LM, RL, AL, PM and AM, respectively; right: n = 8/11/19 sessions for V1L4 FOV, V1L2/3 FOV, and mesoscope FOV, respectively. For each visual area, sessions with <100 neurons in that area are excluded, which leaves n = 8, 11, 19, 8, 15, 2, 6 and 1 sessions with p = 0.4609, 0.0020, 0.0007, 0.0078, 0.0054, 0.5000, 0.4375 and 1.0000 for V1L4, V1L2/3, mesoscope V1, LM, RL, AL, PM, and AM, respectively). c, Same data as b, but showing statistical comparisons between areas (Kruskal–Wallis test p = 0.0005 for cross-validation accuracy across areas, p = 0.0022 for XRE inference score). a, Lateral (left) and axial (right) physiological point spread functions (PPSF) of the two-photon holographic photo-stimulation system, measured in GCaMP6m–ChRmine expressing neurons (mean ± s.e.m. b, Holography evoked responses in nonphotoactivated neurons (>50 μm from all targets in each ensemble), for neurons that are visually responsive to the corresponding IC image (x axis) vs. nonresponsive neurons (y axis) (24 sessions from 4 mice). Holography-evoked response was measured during the photoactivation window (0–1 s relative to holography onset), and baseline activity was subtracted (−1 to 0 s relative to holography onset). Left, targeting IC-encoders evoked significantly larger activity in IC-responsive neurons than nonresponsive neurons (p = 0.0023 one-tailed Wilcoxon signed-rank test across n = 32 holographic ensembles). Right, targeting segment responders did not evoke a larger activity in IC-responsive neurons compared to nonresponsive neurons (p = 0.9660 across n = 37 holographic ensembles). c, PSTH of holography evoked responses. Baseline activity was subtracted as in b. Top, holography-driven PSTH averaged across all neurons in each group (mean ± s.e.m. d, Same as top panels in c (mean ± s.e.m. Activity in directly photoactivated neurons is an order of magnitude higher than the activity in nonphotoactivated neurons. e, PSTH of visual evoked responses, for the same group of neurons shown in d (mean ± s.e.m. In a–d, decoders were constructed with nonphotoactivated neurons, separately for each holographic ensemble (>50 μm from all targets in each ensemble; 24 sessions from 4 mice expressing GCaMP6m–ChRmine in V1 neurons). a, Holography-evoked response patterns are classified into the image that evoked the most similar visual response patterns. b, The proportion of holography trials decoded as the expected visual label (y axis), as a function of the number of targets in each holographic ensemble (left); tenfold cross-validation accuracy of the decoder trained on visual responses (middle); and the proportion of holography trials decoded as the expected visual label when decoder was trained on all neurons (right) (ρ denotes the Spearman correlation across holographic ensembles; n = 24 IC1-encoder ensembles, n = 24 IC2-encoder ensembles). d, Same as b, but for segment responder ensembles (n = 24 BR + TL segment responder ensembles, n = 24 BL + TR segment responder ensembles). e–h, Decoders were trained on neurons that were commonly nonphotoactivated across four holographic ensembles; IC1-encoders, IC2-encoders, BR + TL segment responders and BL + TR segment responders (>50 μm from all targets across these four ensembles). a, LC-encoders are defined as neurons that respond to either LC1 or LC2, but not to either IC1 or IC2 (p < 0.05 Kruskal–Wallis test across responses to blank vs. IC1 vs. LC1 vs. LC2 vs. IC2, post hoc Tukey–Kramer test). In b–g, LC-encoders were targeted in two distinct holographic ensembles; LC1-encoder photoactivation is expected to be decoded as LC1, and LC2-encoder photoactivation is expected to be decoded as LC2 (n = 24 sessions from 4 mice expressing GCaMP6m–ChRmine in V1 neurons). b, Visual responses of neurons in each holographic ensemble. In c–e, only the holographic ensembles with ≥10 targets were analyzed (n = 35, 32 and 37 ensembles for LC-encoders, IC-encoders and segment responders, respectively). Box-and-whisker plots are formatted as follows: centerline, median; box limits, upper and lower quartiles; whiskers, 1.5x interquartile range; circles, outliers. c, To compare visual information contained in holographic ensembles, we compared the tenfold cross-validation accuracy of visual responses for targeted neurons on corresponding visual trials; LC-encoder ensembles on LC trials (LC1/LC2-encoders on LC1/LC2 trials, respectively), IC-encoder ensembles on IC trials (IC1/IC2-encoders on IC1/IC2 trials, respectively) and segment responder ensembles on IC trials (BR + TL/BL + TR segment responders on IC1/IC2 trials, respectively; Kruskal–Wallis test p = 2.0 × 10−8, followed by Bonferroni–Holm corrected Wilcoxon rank-sum tests *p < 0.05). d, Decoders constructed with all neurons were trained on visual evoked activity patterns and used to classify holography evoked activity patterns. Only the holographic ensembles with ≥10 targets were analyzed (n = 35, 32 and 37 ensembles for LC-encoders, IC-encoders and segment responders, respectively; Kruskal-Wallis test p = 2.4 × 10−11, followed by Bonferroni–Holm corrected Wilcoxon rank-sum tests *p < 0.05). e, Decoders were constructed with nonphotoactivated neurons for each holographic ensemble (Kruskal–Wallis test p = 0.0012, followed by Bonferroni–Holm corrected Wilcoxon rank-sum tests *p < 0.05). Cross-validation accuracy of holographic ensembles (x axis) was divided into 6 equidistant bins between 0 and 1, and mean ± s.e.m. g, For the three groups of holographic ensembles, cross-validation accuracies of targets were matched by matching the number of samples in each x-axis bin shown in f. Specifically, in each of the three bins where all three groups had data, we chose the group with the highest number of samples, and bootstrapped data in the other two groups to match the sample number. As a result, the bootstrapped distributions were matched in cross-validation accuracies of targets (left; n = 48 ensembles in each group, Kruskal–Wallis test p = 0.5811). When comparing the decoding performance of holographically driven activity in nonphotoactivated neurons among these bootstrapped distributions, we again see the highest performance when targeting IC-encoders (right; n = 48 ensembles in each group, Kruskal–Wallis test p = 0.0022, followed by Bonferroni–Holm corrected Wilcoxon rank-sum tests *p < 0.05). Box-and-whisker plots are formatted as in c–e. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Shin, H., Ogando, M.B., Abdeladim, L. et al. Recurrent pattern completion drives the neocortical representation of sensory inference. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250914205835.htm'>Scientists just found hidden parasitic wasps spreading across the U. S.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 07:29:43
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A research team including faculty at Binghamton University, State University of New York have identified two previously unknown species of parasitic wasps living in the United States.A research team including facutly at Binghamton University, State University of New York have identified two previously unknown species of parasitic wasps living in the United States. Oak gall wasps and their predators don't have the panache of butterflies, but they're attracting growing interest among both scientists and naturalists. Only 1 to 8 millimeters long, these small insects create the tumor-like plant growths known as "galls." Small as a pinhead or large as an apple, galls can take striking shapes, with some resembling sea urchins or saucers, explained Binghamton University Associate Professor of Biological Sciences Kirsten Prior, who also co-leads Binghamton's Natural Global Environmental Change Center. And if these wasps are a mascot for anything, it's biodiversity. But how many species of parasitoid wasps are out there? In addition to Prior, co-authors include current graduate student Kathy Fridrich and former graduate student Dylan G. Jones, as well as Guerin Brown, Corey Lewis, Christian Weinrich, MaKella Steffensen and Andrew Forbes of the University of Iowa, and Elijah Goodwin of the Stone Barns Center for Food and Agriculture in Tarrytown, N.Y. In 2024, the National Science Foundation awarded a $305,209 grant to Binghamton University for research into the diversity of oak gall wasps and parasitoids throughout North America. The NSF grant investigates a core question: How do gall-forming insects escape diverse and evolving clades of parasitic wasps -- and how do parasites catch up? To answer that question, researchers are collecting oak gall wasps around North America and using genetic sequencing to determine which parasitic wasps emerge from the galls. Among them are Fridrich and fellow Binghamton graduate student Zachary Prete, who spent the summer on a gall- and parasitoid-collection trip from New York to Florida. "We are interested in how oak gall characteristics act as defenses against parasites and affect the evolutionary trajectories of both oak gall wasps and the parasites they host. Over the past several years, researchers with Prior's lab traveled the West Coast from California to British Columbia, collecting approximately 25 oak gall wasp species and rearing tens of thousands of parasitic wasps, which were ultimately identified as more than 100 different species. Researchers at the University of Iowa identified a similar wasp from collections they made in New York state. "Finding this putative European species on the two coasts of North America inspired our group to confirm this parasitic species' identity and whether it was, in fact, an introduced parasite from Europe," Prior explained. Parasitic wasps are small and challenging to identify based on features alone. Because of this, researchers use genetic tools to confirm a species' identity, sequencing "the universal barcoding gene," Cytochrome Oxidase Subunit I (mtCOI), and comparing their results to reference libraries. What they discovered is that the European species B. dorsalis came in two separate varieties, or clades: the New York samples were related to species in Portugal, Iran and Italy, while the Pacific coast wasps were related to those from Spain, Hungary and Iran. And while they were found in at least four different oak gall wasp species from Oregon to British Columbia, all the West Coast B. dorsalis wasps were genetically identical, which means that their introduction was small and localized. The East Coast wasps had slightly more genetic diversity, which could indicate that there was less of a population bottleneck, or that the species was introduced more than once. One possibility is that non-native oak species were intentionally introduced to North America. English oak, or Quercus robur, was widely planted for wood since the 17th century, and is found in British Columbia as well as several northeastern states and provinces. Turkey oak, Q. cerris, is an ornamental tree now found along the East Coast -- including a spot near where B. dorsalis was discovered in New York. Adult parasitic wasps can live for 27 days, so they could have hitchhiked on a plane, Prior said. Other introduced parasite species are known to impact populations of native insects, she acknowledged. "Parasitic wasps are likely the most diverse group of animals on the planet and are extremely important in ecological systems, acting as biological control agents to keep insects in check, including those that are crop or forest pests," Prior explained. Scientists Unveil Potential First-of-Its-Kind Treatment for Deadly Fatty Liver Disease Our Galaxy's Sweet Spot for Life Is Bigger Than We Thought Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/09/250914205759.htm'>Being too thin can be deadlier than being overweight, Danish study reveals</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-09-15 01:56:52
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Individuals with a BMI in the middle and lower parts of the normal weight range 18.5 to <22.5kg/m2, were also more likely to die. As were individuals with a BMI in the underweight range. "Obesity may disrupt the body's metabolism, weaken the immune system and lead to diseases like type 2 diabetes, cardiovascular diseases and up to 15 different cancers, while underweight is tied to malnutrition, weakened immunity and nutrient deficiencies. "There are conflicting findings about the BMI range linked to lowest mortality. To provide some clarity, Dr Gribsholt, Professor Jens Meldgaard Bruun, also of the Steno Diabetes Center Aarhus, and colleagues used health data to examine the relationship between BMI and mortality in 85,761 individuals (81.4% female, median age at baseline 66.4 years). Similarly individuals with BMI of 40 kg/m2 and above (categorized as severe obesity) were more than twice as likely (2.1 times) to have died compared with the reference population. However, higher mortality rates were also found for BMIs that are considered healthy. A similar pattern was obtained when the researchers looked at the relationship between BMI and obesity in participants of different ages, sexes and levels of education. Dr Gribsholt says: "One possible reason for the results is reverse causation: some people may lose weight because of an underlying illness. "Since our data came from people who were having scans for health reasons, we cannot completely rule this out. "It is also possible that people with higher BMI who live longer - most of the people we studied were elderly - may have certain protective traits that influence the results. "Still, in line with earlier research, we found that people who are in the underweight range face a much higher risk of death." He explains: "Other important factors include how the fat is distributed. Visceral fat - fat that is very metabolically active and stored deep within the abdomen, wrapped around the organs - secretes compounds that adversely affect metabolic health. "As a result, an individual who has a BMI of 35 and is apple-shaped - the excess fat is around their abdomen - may have type 2 diabetes or high blood pressure, while another individual with the same BMI may free of these problems because the excess fat is on their hips, buttocks and thighs. "It is clear that the treatment of obesity should be personalized to take into account factors such as fat distribution and the presence of conditions such as type 2 diabetes when setting a target weight." Materials provided by European Association for the Study of Diabetes. Note: Content may be edited for style and length. Scientists Unveil Potential First-of-Its-Kind Treatment for Deadly Fatty Liver Disease Our Galaxy's Sweet Spot for Life Is Bigger Than We Thought Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            