
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-16</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08842-1'>Re-adenylation by TENT5A enhances efficacy of SARS-CoV-2 mRNA vaccines</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-16 15:44:57
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Despite the widespread use of mRNA vaccines against COVID-19, little is known about the metabolism of therapeutic RNAs. Here we use nanopore sequencing1,2,3 to analyse individual therapeutic mRNA molecules, focusing on their poly(A) tails. We show that the Moderna mRNA-1273 vaccine4 has a poly(A) tail of around 100 nucleotides, followed by an mΨCmΨAG sequence. However, in medically relevant preclinical models, particularly in macrophages, mRNA-1273 poly(A) tails are extended to up to 200 nucleotides by the TENT5A poly(A) polymerase5,6,7, which is induced by the vaccine. Re-adenylation, which stabilizes target mRNAs, is consistently observed in synthetic mRNAs that encode proteins targeted to the endoplasmic reticulum, such as ovalbumin or antigens from Zika virus8 or the malaria parasite9. The extent of re-adenylation varies: the BioNTech–Pfizer BNT162b2 vaccine10 shows less potent re-adenylation than mRNA-1273, which correlates with a smaller proportion of membrane-associated BNT162b2. This highlights the crucial role of spatial accessibility to ER-resident TENT5A in determining re-adenylation efficiency. In vivo, TENT5A is expressed in immune cells that take up mRNA vaccine, and TENT5A deficiency reduces specific immunoglobulin production for mRNA vaccines after immunization in mice. Overall, our findings reveal a principle for enhancing the efficacy of therapeutic mRNAs, paving the way for improvement. During the COVID-19 pandemic, mRNA vaccines were deployed on a global scale with unprecedented speed. Despite the need for immediate action, the mRNA vaccine field is still in its early stages, because this method of vaccine design was entirely new. The two anti-COVID-19 mRNA vaccines that were first approved—BNT162b2 (Comirnaty) and mRNA-1273 (Spikevax)—are in vitro transcribed (IVT) mRNAs4,10 that encode the coronaviral spike protein. These therapeutic mRNAs have all of their uridines replaced by N1-methyl-pseudouridines (mΨ), and are overall very similar. However, they differ in their cap structures, untranslated region (UTR) sequences and 3′ tails11. The BNT162b2 vaccine is capped cotranscriptionally with a trinucleotide Cap1 analogue (CleanCap AG, with 3′-O-methylation on m7G) and a composite poly(A) tail of 30 adenosines followed by 10 other nucleotides and then 70 extra adenosines (ref. The mRNA-1273 vaccine is capped post-transcriptionally, creating natural Cap1 through vaccinia virus capping enzyme (VCE) and mRNA Cap 2′-O-methyltransferase (MTA), and has a poly(A) tail of undisclosed length. Both vaccines are encapsulated in lipid nanoparticles (LNPs) of slightly different compositions and enter the cells through endocytic pathways13,14. Intramuscularly administered mRNA vaccines can locally transfect resident and infiltrating immune cells, such as antigen-presenting dendritic cells (DCs) and macrophages. These cells become activated and can migrate to neighbouring lymph nodes15,16,17,18. LNPs, as well as secreted antigens produced in muscle, are transported to local lymph nodes, where they are taken up by macrophages and resident DCs18,19. DCs, which have a central role in adaptive immunity, endocytose antigens produced by other cells and are thought to translate vaccine mRNAs19. Peptide antigens are presented by class I and class II major histocompatibility complex (MHC) molecules to induce an immune response18. Studies on mRNA vaccines have focused on DCs20,21; macrophages and monocytes—despite being the main cell types that take up LNP-encapsulated RNA—have received less attention15,18,22. Moreover, the distribution of mRNA vaccines has been studied in local lymph nodes, rather than in muscles, where it is administered. Thus, the role of macrophages and local antigen production in mRNA vaccine efficacy is not well defined. Notably, almost nothing is known about the metabolism of vaccine poly(A) tails, but the paradigm extrapolated from data on endogenous transcripts (or reporters) asserts that the stability of therapeutic mRNAs is determined by the rate of poly(A) tail shortening (deadenylation). Here we use nanopore RNA sequencing (RNA-seq) to analyse mRNA therapeutics, revealing the complexity of their metabolism. Specifically, we show that the endoplasmic reticulum (ER)-associated TENT5A poly(A) polymerase stabilizes IVT mRNAs encoding secreted proteins. In vivo, TENT5A enhances the immunogenicity of the anti-COVID-19 mRNA-based vaccines mRNA-1273 and BNT162b2, but not that of Nuvaxovid, which contains purified spike protein as an antigen. To our knowledge, the effect of endogenous polyadenylation machinery on mRNA therapeutics has not been previously reported or even considered. Owing to methodological challenges, the in vivo metabolism of therapeutic mRNAs is poorly understood. Commonly used sequencing-by-synthesis methods rely on the analysis of PCR-amplified cDNA, but these approaches are indirect and error-prone, particularly when studying poly(A) tails, because of polymerase slippages on homopolymers1. Quality control of mRNA therapeutics, especially their poly(A) tails, presents similar difficulties. Direct RNA sequencing (DRS), developed by Oxford Nanopore Technologies (ONT)2, is a promising solution to these challenges. DRS records electric currents during the passage of individual single-stranded RNA molecules through a protein pore, providing reliable information about RNA composition with single-molecule resolution (Fig. a, Overview of the eDRS approach for studying mRNA therapeutics. b, Representative raw signals from eDRS showing mRNA-1273 with (bottom) and without (top) a 3′ terminal mΨCmΨAG pentamer. The signal from poly(A) is zoomed in for visualization purposes. c, mRNA-1273 poly(A) length distribution in HEK293T cells treated with the vaccine for up to 72 h. d, Western blot analysis of spike protein expression from mRNA-1273 and BNT162b2 in HEK293T cells up to 96 h after the delivery of mRNA LNPs. Levels of α-actinin, GAPDH and α-tubulin served as loading controls. Asterisk indicates unspecific signal from untreated cells. Bottom, quantification of spike protein levels (five independent biological experiments); mean ± s.e.m. e, mRNA-1273 poly(A) length distribution for HEK293T cells with tetracycline-induced depletion of CNOT1 (CNOT1 knockdown (KD)) or controls (wild type (WT), without tetracycline), treated with the vaccine for up to 72 h. For c,e, eDRS-measured poly(A) tail lengths are shown with reads containing (blue) or lacking (green) an mΨCmΨAG sequence, with median values and the fraction of shortened tails shown. Analysis of intact mRNA-1273 using a standard DRS pipeline (Fig. 1a) revealed that a substantial proportion of aligned reads covered the full-length vaccine (Extended Data Fig. 1a, blue), indicating the potential for comprehensive analysis of mRNA therapeutics. However, uridine-to-mΨ substitution in mRNA therapeutics affects the current signal recorded, resulting in imprecise translation into the sequence (basecalling) (Extended Data Fig. Therefore, we developed a subsequence dynamic time warping (sDTW) approach (Supplementary Note 1 and Extended Data Fig. 1c), which enables the identification of twice as many sequences compared with the standard DRS pipeline (Extended Data Fig. Next, we focused on analysing vaccine poly(A) tails, which are crucial for mRNA stability and translation, using nanopolish polya23 (Extended Data Fig. However, when we visually inspected individual mRNA-1273 nanopore signals, we observed an unexpected perturbation between the poly(A) tail and the adaptor used for library preparation (Fig. To determine the specificity of the signature for the 3′ end of mRNA-1273, we used vaccine RNA with an enzymatically extended 3′ end with inosine nucleotides (I-tailing), followed by DRS using a custom protocol. Finally, the presence of a TCTAG (mΨCmΨAG in the vaccine mRNA) was revealed through rapid amplification of cDNA 3′ end (3′-RACE) Illumina sequencing. To study mRNA vaccines quantitatively, we developed an enhanced DRS (eDRS) pipeline that includes sDTW and a modified algorithm for detecting poly(A) tails with mΨCmΨAG (Supplementary Note 2 and Supplementary Fig. Our analysis of mRNA-1273 revealed that most vaccine RNA molecules had an intact 3′ end, with approximately 20% of reads lacking the mΨCmΨAG sequence (Extended Data Fig. Of note, the mΨCmΨAG pentamer enables easy discrimination between intact vaccine mRNA, defined as possessing a 3′-terminal mΨCmΨAG sequence, and processed RNA, which ends with a pure poly(A) sequence; this facilitates detailed analysis of poly(A) metabolism (Supplementary Note 2). A time-efficient computational pipeline that allows the analysis of poly(A) lengths from nanopore cDNA libraries has recently been released by ONT, enabling us to obtain similar poly(A) lengths and to confirm the presence of the pentamer with the orthogonal method (Extended Data Fig. In summary, both eDRS and cDNA nanopore sequencing are well-suited for analysing mRNA vaccines. Next, we used nanopore sequencing to monitor mRNA-1273 stability and poly(A) tail status in cells that are commonly used in preclinical analyses. The optimal amount of vaccine RNA was determined to be non-toxic to cells (Supplementary Note 4 and Supplementary Fig. 2), allowing efficient protein production and detection by eDRS (Extended Data Fig. This deadenylation occurred exclusively in reads lacking mΨCmΨAG. Notably, BNT162b2 exhibited slower antigen production kinetics than mRNA-1273 (Fig. 1d), even when the vaccine mRNA was delivered into cells using lipofectamine reagent (Extended Data Fig. This was accompanied by a slower rate of deadenylation and greater stability in HEK293T cells, as determined by nanopore cDNA sequencing (Extended Data Fig. Finally, to investigate the role of deadenylation in mRNA-1273 decay, we generated a HEK293T cell line with tetracycline-inducible depletion of the scaffold subunit (CNOT1) of the primary deadenylase complex CCR4–NOT. 2m) and reduced poly(A) tail shortening in mRNA-1273, compared with the uninduced state (Fig. Significant changes in poly(A) length were observed only in reads that lacked the terminal mΨCmΨAG pentamer. The ratio of transcripts lacking the pentamer remained constant under both conditions, indicating that CCR4–NOT does not participate in mΨCmΨAG removal. These observations suggest that deadenylation restricts the lifespan of therapeutic mRNAs in model cell lines. The fate of mRNA therapeutics depends on various factors that are difficult to replicate in model cell lines. To study vaccine mRNA metabolism, we delivered mRNA-1273 intramuscularly in a mouse model. Quantitative PCR with reverse transcription (qRT–PCR) revealed that the abundance of vaccine mRNA is orders of magnitude higher at the injection sites than in the draining lymph nodes (Extended Data Fig. 3a), in which the levels of mRNA-1273 were insufficient for effective eDRS analysis. eDRS on samples from the injection sites revealed that most mRNA-1273 poly(A) tails were unprocessed and contained a terminal mΨCmΨAG sequence even 24 h after injection, although mild deadenylation of processed tails was noticeable at all time points analysed (Fig. Of note, we found that at the 24-h time point, the median poly(A) tail length of mRNA-1273 increased from 100 to 114 nucleotides, and that 49% of the reads lacking mΨCmΨAG pentamer had lengths greater than 115 nucleotides. We confirmed mRNA-1273 poly(A) tail extension with eDRS on RNA isolated from immune cells infiltrating vaccine injection sites (Extended Data Fig. a, mRNA-1273 poly(A) length distribution in tissues surrounding vaccine injection sites up to 24 h after intramuscular administration. b,c, mRNA-1273 poly(A) length distribution in mBMDMs (b) or hMDMs (c) incubated with mRNA-1273 LNPs for up to 96 h. d, BNT162b2 poly(A) length distribution in mBMDMs incubated with BNT162b2 LNPs for up to 72 h. e, In vitro translation assay (3 h) using reticulocyte lysate for mRNA-1273 and BNT162b2. For a–d, eDRS-measured poly(A) tail lengths with median values and fractions of shortened or elongated tails are shown. For a–c, reads containing (blue) or lacking (green) an mΨCmΨAG sequence are shown separately. Data represent two (a,c,d) or three (b) independent experiments. To gain further insight into infiltrating immune cells, which presumably take up mRNA-1273, we performed a flow-cytometry analysis in the time course after immunization (Supplementary Note 5 and Supplementary Figs. We observed that 24 h and 48 h after immunization, the muscles were populated almost exclusively by macrophage-type cells (F4/80+CD64+) also expressing CD11c+ (Extended Data Fig. At the same time, cells with markers attributed to tissue-resident DCs (CD45+, I-A/I-E (MHCII)high, CD11c+, F4/80− and CD64−) represented a minor population (around 20 times smaller than the macrophage population), the abundance of which remained stable over time. We then collected these two populations in a post-injection time course. Notably, cDNA nanopore sequencing indicated that mRNA poly(A) tails can be extended in vivo at 24 h when compared with 12 h for both macrophages and DCs (Extended Data Fig. In parallel with cDNA sequencing, we assessed the absolute abundance of mRNA-1273 using qRT–PCR. Both DCs and macrophages efficiently take up the vaccine, with an average of 82 and 152 molecules per cell, respectively (Extended Data Fig. However, macrophages exhibited a greater stability of mRNA-1273 than DCs (Extended Data Fig. 3d,e), which correlates with the more potent re-adenylation. Given the abundance, macrophages represent the main cell population responsible for the local metabolism of vaccine mRNA, and we therefore shifted to in vitro cultures of mouse bone marrow-derived macrophages (mBMDMs) and human monocyte-derived macrophages (hMDMs) as experimental models. Both were efficiently transfected with mRNA-1273 LNPs without visible toxic effects (Supplementary Note 4), allowing antigen detection for up to 96 h (Extended Data Fig. In mBMDMs, we observed a median elongation of around 24 adenosines in the poly(A) tails of vaccine mRNAs 24 h after transfection, with a fraction of tails reaching up to 200 adenosines. In hMDMs, re-adenylation of mRNA-1273 was observed 48 h after vaccine delivery, although to a lesser extent (Fig. All of this indicates that re-adenylation of mRNA-1273 vaccine observed in vivo can be recapitulated with cultured macrophages as models. We speculated that a similar phenomenon would be observed for BNT162b2, but in fact we observed only a minor extension of poly(A) tails throughout the time course after administration to mBMDMs (Fig. This could be related in part to an inability to distinguish between processed and intact tails, which is not the case for mRNA-1273 (Supplementary Note 2). In addition, dose-dependent assays using mBMDMs showed that mRNA-1273 was more effectively translated at all dose ranges—compared with BNT162b2, which produced less antigen (Extended Data Fig. These slower kinetics of protein production seem to be directly related to the translation efficiency, because the effect is also seen in an vitro translation system based on reticulocyte lysate (Fig. Whole-transcriptome analysis of mBMDMs treated with mRNA-1273 revealed significant changes in poly(A) length not only for mRNA-1273 but also for 124 endogenous transcripts (Kruskall–Wallis test, P < 0.001). The transcripts associated with immune response showed the most pronounced tail extension, with an average of 20 adenosines at 24 h (Fig. Out of the 92 transcripts with a significant change in poly(A) tail length in hMDMs (Extended Data Fig. 4a and Supplementary Table 3), 38 were orthologues of those identified in mBMDMs, encoding mainly proteins that are involved in antigen presentation. Major functionalities enriched in a given cluster are shown on the right. b,c, Differential expression analysis after administration of mRNA-1273 to mBMDMs. b, Number of upregulated or downregulated genes at time points (compared with 0 h). c, MA plot for the 24-h time point; genes with P < 0.05, |log2(fold change)| ≥ 1.5 are marked with red circles and immune response genes are marked in blue. d, Expression (DESeq2-normalized counts) of non-canonical poly(A) and poly(U) polymerases in mBMDMs treated with mRNA-1273 for up to 72 h. Gene names are on the left, adjusted P values from the DESeq2 LRT test are on the right, with bar height indicating significance. Western blot for HA (to detect tagged TENT5A) and spike; GAPDH, β-actin and Ponceau staining (bottom) served as loading controls. Three independent replicates (mice) per time point. Values shown as fold change relative to the 6-h time point. g, Tent5a expression (from single-cell RNA-seq) in lymph node macrophages (top) and DCs (bottom) of naive mice or mice immunized with BNT162b2, at days (D) 1, 3 or 7 after immunization. a–d show results of the poly(A) lengths (a) or differential expression analysis (DESeq2, b–d) on eDRS sequencing data from three independent biological experiments. Because poly(A) extension of mRNA-1273 and endogenous transcripts occurred simultaneously in both cell types (after 24 h and 48 h in mBMDMs and hMDMs, respectively), we hypothesized that a common factor—probably transcriptionally induced after vaccine delivery—mediates this process. Indeed, the changes in poly(A) tails were accompanied by complex transcriptomic responses. At the injection sites, upregulated genes (Extended Data Fig. 4b, clusters 2 and 3 and Supplementary Table 4) were enriched in functions related to the immune response and cell activation (Supplementary Table 5), suggesting the infiltration of immune cells, which is consistent with previous reports19. Differentially expressed genes were observed starting at 24 h after mRNA-1273 administration in mBMDMs (Fig. Of note, the observed responses resembled those previously reported for the BNT162b2 vaccine15, with increased expression of immune response genes (Fig. Among the genes upregulated after mRNA-1273 administration in mBMDMs, we observed Tent5a, the only non-canonical cytoplasmic poly(A) polymerase exhibiting such a pattern (Fig. Previously, we showed that Tent5a and Tent5c were induced in mBMDMs after activation with lipopolysaccharide (LPS)7, and we observed an elongation of poly(A) tails in a group of mRNAs; this elongation was very similar to that seen after mRNA-1273 delivery. It suggests that TENT5A is indeed a factor responsible for vaccine re-adenylation. Moreover, Tent5a is the only cytoplasmic poly(A) polymerase that is significantly induced at vaccine injection sites (Extended Data Fig. Tent5c is also expressed in all analysed cells but at a much lower level (Fig. Finally, the induction of TENT5A after mRNA-1273 administration is also visible at the protein level, as revealed by an analysis of mBMDMs derived from a knock-in mouse line (FKBP12F36V-HA-Tent5a) expressing TENT5A fused with a dTAG degrader (FKBP12(F36V)–HA tag) (Fig. Accordingly, we observed increased expression of Tent5a in cells sorted from the injection sites 12 h after immunization (Fig. One day after immunization with BNT162b2, some induction of Tent5a was also observed in single-cell RNA-seq data from a previous study15, which we reanalysed focusing on the expression of TENT genes (Fig. Overall, these experiments suggest a possible role for TENT5A and, to a lesser extent, TENT5C, in the re-adenylation of mRNA-1273 observed in macrophages, in both mouse and human models. To evaluate the role of TENT5 proteins in vaccine mRNA re-adenylation, we studied mRNA-1273 metabolism in mBMDMs lacking TENT5A and TENT5C (double-knockout Tent5aflox/floxTent5c−/−). Cells were treated with mRNA-1273 for up to 72 h and RNA was isolated and subjected to eDRS. In comparison with wild-type mBMDMs, we observed relatively fast mRNA-1273 decay and deadenylation in the absence of TENT5A and TENT5C (Fig. 4a), with median poly(A) tails shortened to around 83 As after 24 h, despite the high ratio of RNAs lacking the 3′ pentamer. Rapid depletion of TENT5A protein in mBMDMs using the degron-inducible FKBP12F36V-HA-Tent5a model (Fig. 5a) led to the same effect, with mRNA-1273 poly(A) tails shortened 24 h after vaccine administration, when compared with controls. The presence of the FKBP12(F36V) tag itself caused only a minor reduction in the activity of TENT5A, which was evident at both 24 h and 72 h after mRNA administration. Small interfering RNA (siRNA) knockdown of TENT5A in hMDMs also caused mRNA-1273 poly(A) shortening and decreased antigen production, when compared with mock-siRNA-transfected cells, and this effect was observed for three independent siRNAs (Fig. All of these data indicate that TENT5 proteins have a crucial role in the regulation of mRNA-1273, with TENT5A being particularly key. c, mRNA-1273 poly(A) length distribution in hMDMs transfected with either mock (left) or TENT5A-specific (right) siRNAs, treated with mRNA-1273 LNPs for 48 h. Average distributions from three different siRNAs in two biological replicates. d, Stability of mRNA-1273 in wild-type (black) or Tent5aflox/floxTent5c−/− (red) mBMDMs, calculated based on data from a and Fig. Abundance of mRNA-1273 normalized to 24 h time point. e, mRNA-1273 poly(A) length distribution in mBMDMs transfected with either mock or Tent2-, Tent4a/Tent4b- or Tent3a/Tent3b-specific siRNAs, treated with mRNA-1273 LNPs for 24 h. Average distributions from three different siRNAs in two independent biological replicates. For a–c,e, eDRS-measured poly(A) tail lengths are shown with reads containing (blue) or lacking (green) an mΨCmΨAG sequence, with median values and fractions of elongated or shortened tails shown. Data in a,b represent two independent experiments. Notably, dysfunction of TENT5-mediated cytoplasmic polyadenylation had no effect on the transcriptional response, which was similar in both wild-type and Tent5aflox/floxTent5c−/− mBMDMs (Extended Data Fig. A negative effect of the knockout was observed only for poly(A) tails (Extended Data Fig. Furthermore, TENT5A dysfunction does not affect the global translation efficiency, as revealed by high-content microscopy fluorescence quantifications of EGFP-encoding reporter mRNAs (Extended Data Fig. Finally, to look at the potential effect of other non-canonical poly(A)/poly(U) polymerases on mRNA-1273 poly(A) tails, we depleted Tent2, Tent4a, Tent4b, Tent3a and Tent3b in mBMDMs and subsequently administered mRNA-1273. In all cases, we observed efficient re-adenylation of mRNA-1273 (Fig. 5f), which was lost only in Tent5aflox/floxTent5c−/− cells (Fig. We conclude that TENT5A is responsible for mRNA-1273 re-adenylation in macrophages. As a next step, we focused on factors determining TENT5A substrate specificity. Most of the mRNAs with poly(A) tails that were extended after mRNA-1273 vaccination in wild-type but not in Tent5aflox/floxTent5c−/− mBMDMs (Extended Data Fig. 5d) encode proteins that are translated at the ER. Notably, mRNA-1273 also encodes an ER-targeted protein, which, as we show, hijacks TENT5 proteins to extend its poly(A) tails. Massive translation of spike (and other proteins) at the ER, after mRNA-1273 administration, led to increased ER stress in hMDMs (Extended Data Fig. However, siRNA depletion of TENT5A resulted in decreased spike protein translation (Extended Data Fig. 5b) and reduced the levels of unfolded protein response markers (Extended Data Fig. 6b), consistent with the role of TENT5A in promoting vaccine antigen translation. Depleting Fndc3a and Fndc3b in mBMDMs led to the abrogation of mRNA-1273 re-adenylation (Fig. 5a), indicating that, in the case of TENT5A, its cooperation with FNDC3A and FNDC3B is essential for the re-adenylation and stabilization of mRNA-1273. Average distributions from three different siRNAs in two replicates. b, cDNA-seq-measured poly(A) length distribution for mRNA-1273 (top) and BNT162b2 (bottom) co-transfected to wild-type mBMDMs in equal amounts, showing cytoplasmic (C) and membrane (M) fractions over time (up to 36 h). c, cDNA-seq-measured OVA reporter poly(A) length distribution in mBMDMs transfected with mRNAs that have BNT162b2 or mRNA-1273 UTRs and BNT162b2-like poly(A) tails (with terminal mΨCmΨAG) for up to 48 h. The type of UTRs in the reporter is indicated at the top. d,e, PfCSP reporter poly(A) length distribution in mBMDMs transfected with mRNAs that have BNT162b2-like poly(A) tails with a terminal mΨCmΨAG for up to 24 h, analysed with either cDNA-seq (d) or eDRS (e). For all panels, reads are divided into those with (blue) or without (green or grey) an mΨCmΨAG sequence, with median values and fractions of elongated or shortened tails shown. Because our previous studies in different cell types revealed that the specificity of TENT5 proteins is not determined by sequence motifs5,6,7,24, and that the presence of an ER targeting signal is sufficient to drive TENT5-mediated regulation6, the efficient re-adenylation of mRNA-1273 but not BNT162b2 was puzzling. Given that coronavirus spike proteins are co-translationally translocated to the ER26, we sought to further investigate the spatial determinants of vaccine mRNA metabolism. 6c), we co-transfected mRNA-1273 and BNT162b2 to mBMDMs with lipofectamine-based transfection reagent, to exclude the differential effects of each vaccine on transcriptomic response. Next, we fractionated cells after vaccine administration in a time-course manner and performed nanopore cDNA sequencing of isolated RNA. This analysis revealed robust re-adenylation of not only mRNA-1273 but also BNT162b2, specifically in the membrane fraction (Fig. 6d), confirming that ER association is a prerequisite for efficient re-adenylation. The lower load of BNT162b2 on ER (Extended Data Fig. 6e) and the inability to distinguish between intact and processed vaccines—unlike mRNA-1273, for which reads ending in mΨCmΨAG can be excluded—are likely to explain why re-adenylation of BNT162b2 is less pronounced in the total fraction (Extended Data Fig. Next, we administered both vaccines to mBMDMs separately and analysed the dynamics of antigen production. The spike protein was present in the membrane fraction, with a trend towards lower levels of BNT162b2-encoded than mRNA-1273-encoded spike protein on the ER, mainly at early time points (Extended Data Fig. This aligns with the results from our analysis of whole-cell lysates and culture medium, and with the in vitro translation assay presented before (Figs. It could be caused by a combination of effects of differences in the cap structure (Extended Data Fig. Nevertheless, despite relatively inefficient re-adenylation, Tent5aflox/floxTent5c−/− mBMDMs produce less spike protein, compared with wild-type mBMDMs, after BNT162b2 administration over a time course (Extended Data Fig. 6h), indicating that TENT5A also has a beneficial effect on antigen production for this vaccine. Next, to systematically evaluate the influence of the capping strategy, UTRs and poly(A) tail composition (including the presence of mΨCmΨAG) on mRNA re-adenylation in mBMDMs, we generated a series of mΨ-containing reporter mRNAs (Supplementary Note 6 and Extended Data Fig. First, 16 variants of ER-targeted codon-optimized ovalbumin (OVA)-coding mRNAs (Extended Data Fig. 8c,d) were used to evaluate protein production using an enzyme-linked immunosorbent assay (ELISA). Notably, we observed that mBMDMs transfected with the enzymatically VCE + MTA-capped mRNA (Cap 1) produced several folds more OVA than did mBMDMs that were transfected with mRNA cotranscriptionally equipped with CleanCap (Extended Data Fig. Besides the capping method, mRNA-1273 UTRs had a beneficial effect on OVA production, compared with those derived from BNT162b2 (Extended Data Fig. Poly(A) tail and mΨCmΨAG sequence seemed to have a minor role despite being synthesized from different DNA templates. For the analysis of poly(A) lengths, we focused on mRNAs ending with mΨCmΨAG, which, as shown before, does not affect expression and is usually removed before re-adenylation (Extended Data Fig. Consistent with the correlation of protein expression and translation efficiency, re-adenylation was completely absent in OVA mRNAs equipped with CleanCap, as revealed by nanopore cDNA sequencing (Extended Data Fig. At the same time, a certain degree of re-adenylation was observed in all variants of enzymatically capped RNAs (Fig. 9d), despite CDS optimization being significantly different from that of SARS-CoV-2 vaccines (Extended Data Fig. Re-adenylation was slightly more pronounced in reporters with mRNA-1273 UTRs and BNT162b2 poly(A) tails. Finally, to check the re-adenylation potency of other CDSs, we designed reporters encoding an ER-targeted circumsporozoite protein from Plasmodium falciparum (PfCSP; Extended Data Fig. 7h) and the Zika virus envelope structural protein (ZikVE), considered antigens in the mRNA vaccines against malaria9 and Zika virus8, respectively. Again, cDNA sequencing of RNA isolated from mBMDMs 24 h after transfection revealed that all enzymatically capped variants were re-adenylated, with no clear trend with regard to the superiority of particular UTRs or poly(A) tail sequences (Fig. Notably, reanalysis of the PfCSP reporter with BNT162b2 UTRs and BNT162b2 tail using eDRS, which is more accurate than cDNA sequencing on long poly(A) tails, revealed massive re-adenylation in mBMDMs (Fig. Overall, we show that TENT5A-mediated re-adenylation stabilizes a broad range of enzymatically capped ER-targeted IVT mRNAs, including therapeutic ones (Extended Data Fig. This further indicates that the potency of re-adenylation is affected by several factors, such as the capping method and the coding and UTR sequences, whereas poly(A) tail composition has a relatively minor role. DCs are major antigen-presenting cells and are expected to have a key role in the development of immunity after vaccination. However, DCs express low levels of TENT5A (Fig. Tent5a-knockout mice, despite having substantially lower body mass and serious skeletal malformations5, do not exhibit major immune defects, which might have an effect on immune response, as revealed by data from the International Mouse Phenotypic Consortium27 (www.mousephenotype.org). Specifically, they show no phenotypes related to T cells, B cells or DCs, only slightly elevated monocyte, neutrophil and granulocyte counts in females. Thus, we immunized wild-type and Tent5a−/− mice with mRNA-1273, BNT162b2 or a Nuvaxovid (Novavax) protein-based COVID-19 vaccine, and measured antibody production (14 days after vaccination). The level of serum anti-spike IgG 14 days after immunization with mRNA-1273 was significantly lower in Tent5a−/− than in wild-type mice (Extended Data Fig. 10a), with no significant change in the total level of IgGs in the sera (Extended Data Fig. Of note, no negative effect of Tent5a−/− on serum anti-spike IgG levels was observed for protein-based Nuvaxovid vaccine (Extended Data Fig. To more specifically address the role of vaccine uptake by macrophages in muscle, we immunized conditional knockout mice with Cre-induced ablation of Tent5a in CD11c+ cells. Similar to what was seen in Tent5a−/− mice, the level of anti-spike IgG was lower at 14 days after immunization in Tent5aflox/flox mice than in controls (Extended Data Fig. Next, we measured the levels of spike protein in the serum at 48 h and 96 h after immunization (Extended Data Fig. 10i), and found that the levels were significantly lower in Tent5aflox/flox mice than in control mice (Extended Data Fig. Such an effect confirms the major role of CDC11c+ cells expressing Tent5a in antigen production. Thus, TENT5A-mediated re-adenylation, which occurs mainly in macrophages, has a direct effect on antigen production and vaccine efficacy. This report provides evidence that therapeutic mRNAs can be modified in cells, such that their poly(A) tails are extended by the cellular machinery. Re-adenylation leads to increased stability of targeted mRNAs and increased production of coded antigens, providing a possible explanation for the efficacy of existing mRNA vaccines (mRNA-1273 and BNT162b2 and their further generations). Our findings could have profound implications for the development of next-generation mRNA therapeutics, including some that are already in preclinical studies (for instance, mRNA-1325 and mRNA-1895 for Zika, encoding the premembrane and envelope proteins8). Moreover, our work highlights and reinforces the physiological role of the ER in cellular RNA metabolism, showing that the ER acts as a central hub not only for cellular mRNA translation but also for its post-transcriptional regulation. Mechanistically, the cytoplasmic poly(A) polymerase TENT5A, acting in concert with ER-associated FNDC3A and FNDC3B, is responsible for the re-adenylation of ER-targeted mRNA therapeutics. The exact mechanism of substrate selection by TENT5A has yet to be established, but the re-adenylation efficiency is affected by several factors. Notably, the capping strategy considerably influences re-adenylation. In primary macrophages, among OVA-coding mRNAs, only those with enzymatically incorporated Cap 1 underwent efficient poly(A) tail extension, whereas those cotranscriptionally capped with CleanCap did not. This correlated with significantly higher production of the encoded protein. Although the precise mechanism has yet to be elucidated, enzymatically capped mRNA-1273 was expressed at a higher level and underwent more robust adenylation than BNT162b2, which used CleanCap. However, the observed effect was less pronounced than it was for OVA mRNAs, suggesting that factors beyond the chemical cap structure are involved. Furthermore, UTR sequences can affect re-adenylation efficiency, although they do not have a dominant role. This observation aligns with the fact that TENT5A substrates lack distinct sequence motifs identifiable within the non-protein-coding regions of mRNAs5,6,7. Further research is needed to uncover all the details of TENT5-mediated re-adenylation. Once administered, mRNA vaccines generate complex immune responses. Although mRNA design has typically been optimized in cell lines, the aim has been to target DCs, which were thought to produce and present the antigen to generate both humoral and cellular adaptive immune responses. Here we show that macrophages are responsible for antigen production and thus have a crucial role in the development of immunity. Macrophages are not only the major cell population that takes up the vaccine mRNA15,19, but they also express TENT5A, which is induced after immunization and re-adenylates them, increasing stability and protein output. The development of therapeutic mRNAs extends beyond vaccines, and intensive efforts are being made to use mRNA to treat Mendelian diseases or to induce the production of therapeutic proteins. In these cases, the main target is the liver, because transfecting hepatocytes after intravenous administration of LNP-encapsulated mRNAs is very efficient. There are also attempts to target other organs and cell types. Here we show that different cell types metabolize mRNA vaccines differently. We expect that only cells that express high levels of TENT5s stabilize the mRNA by re-adenylation and are thus desirable targets for therapeutic mRNAs that encode secreted proteins. In addition, a better understanding of substrate recognition will enable the rational design of mRNAs that are more efficiently polyadenylated by TENT5 and are, therefore, more stable. In the future, other tissue-specific mechanisms of mRNA stabilization might be uncovered. All mouse lines were generated by CRISPR–Cas9 at the Genome Engineering Unit (https://crisprmice.eu/) as described previously3,5,7. A conditional Tent5aflox/flox and double-knockout Tent5aflox/floxTent5c−/− have been described previously7. B6CBAF1 zygotes were microinjected with a mixture containing Cas9 mRNA, gRNA targeting Tent5a (AAAAGTATCTCTGATGCATC) and double-stranded repair template (TAGAAGGGGCGGCCGCCTCCAGGTGACACAGACGGGACTCTCGCTTGTGCTTTCCAGATGGGAGTGCAGGTGGAAACCATCTCCCCAGGAGACGGGCGCACCTTCCCCAAGCGCGGCCAGACCTGCGTGGTGCACTACACCGGGATGCTTGAAGATGGAAAGAAAGTTGATTCCTCCCGGGACAGAAACAAGCCCTTTAAGTTTATGCTAGGCAAGCAGGAGGTGATCCGAGGCTGGGAAGAAGGGGTTGCCCAGATGAGTGTGGGTCAGAGAGCCAAACTGACTATATCTCCAGATTATGCCTATGGTGCCACTGGGCACCCAGGCATCATCCCACCACATGCCACTCTCGTCTTCGATGTGGAGCTTCTAAAACTGGAAGGCGGCTACCCCTACGACGTGCCCGACTACGCCGGCTATCCGTATGATGTCCCGGACTATGCAGGCGGACATCAGAGATACTTTTGGTGCGGGGCTGCTCTGCGGGGCGCGGCGCGCGGCTGGGCATTG). Zygotes were transferred to surrogate mothers and pups were screened for the presence of insertion using a 3-primer approach (Fw: GCAGCTCCTGCGAAGTGTG, Rv1: TGCCATGGCAAAGTACCCTTC, Rv2: CCCACACTCATCTGGGCAAC). Potential founders were sequenced by Sanger sequencing. Mice were bred in the animal facility of the Faculty of Biology, University of Warsaw, and maintained under conventional conditions3,5,7 in open polypropylene cages filled with woodchip bedding enriched with nesting material and paper tubes. Mice were fed ad libitum with a standard laboratory diet (Labofeed B, Morawski). Regular health monitoring was performed at the IDEXX laboratory. Mice of different genotypes were assigned with individual numerical tags in the database and they were used for tissue collection and throughout the subsequent processing as the only identifiers. Discarded residual vaccine material: mRNA-1273 (Moderna, monovalent or bivalent original/Omicron BA.4-5 spike), BNT162b2 (Pfizer monovalent or bivalent original/Omicron BA.4-5), Nuvaxovid (Novavax, recombinant spike protein, adjuvanted) were used within the manufacturer's stability guidelines. Because this was not available for purchase at the time and because only residual (otherwise discarded) material could be used, we obtained approval from the Polish Ministry of Health (MMI.454.1.2021.TM). All animal experiments were approved by the II Local Ethical Committee in Warsaw (approval numbers: WAW2/71/2021, WAW2/129/2021, WAW2/95/2022, WAW2/127/2022 and WAW2/007/2023) and were performed according to Polish law (act number 653 266/15.01.2015) and in agreement with the corresponding European Union directive. Six-to-fourteen-week-old mice were immunized by intramuscular (Vastus lateralis region) injection with 0.9% NaCl (controls), 1 µg mRNA-1273 (Moderna) or BNT162b2 mRNA (Pfizer) or 50 µl Nuvaxovid vaccine (Novavax) at 40 ng µl−1 protein concentration. Tissues were collected at set time points for subsequent analyses. Mice of both sexes were used, with equal or similar ratios of both sexes in directly compared groups. No formal sample-size calculations were performed a priori. Sample sizes were determined on the basis of established practices in similar published studies and the 3Rs principles (replacement, reduction and refinement) for ethical animal research. We used the minimum numbers necessary to achieve statistical significance while maintaining scientific rigour, on the basis of our previous experience and comparable studies. Details on sample sizes are provided for each experiment. Mice with different genotypes could not be distinguished by the experimenter (except for Tent5a−/− mice, which have a distinguishable phenotype). However, vaccination is a routine procedure and therefore knowledge about the genotype of mice at the time of vaccination was highly unlikely to affect the results of immunization. The level of OVA was measured using an ELISA kit from MyBioSource (MBS2000240; L240916286). Sample dilution was determined experimentally to fit within the standard curve of each ELISA. Owing to intervals between immunizations, different versions and lots of both vaccines and ELISA kits were used, depending on availability. Therefore, ELISA results should be interpreted as semi-quantitative between experiments. A single experiment was always performed simultaneously for all conditions studied, and can therefore be interpreted as quantitative, allowing direct comparisons between experimental conditions. Muscles from anterior thigh were isolated 6 h, 12 h, 24 h or 48 h after vaccine injection, finely minced using surgical scissors and resuspended in 2 ml of serum-free RPMI supplemented with 1,000 U ml−1 collagenase from Clostridium histolyticum (Sigma-Aldrich, C9407), 250 µg ml−1 DNAse I (Sigma-Aldrich, 10104159001) and 8 µg ml−1 dispase I (Sigma-Aldrich, D4693). Samples were incubated for 30 min at 37 °C with 140-rpm agitation. Ten millilitres of ice-cold FACS buffer (0.2% BSA in PBS) was added to muscle suspensions, followed by straining with a 70-µm nylon strainer (Greiner, 542070). The crude pellet was treated twice with ACK (150 mM ammonium chloride, 1 mM potassium bicarbonate and 0.1 mM disodium EDTA, pH 7.2) for 2 min on ice, followed by washing with 10 ml FACS buffer; each time, centrifugation was performed at 300g for 7 min. The crude pellet was then resuspended in 1 ml FACS buffer and strained through the 30-μm cap strainer of a FACS tube. With caps on, the samples were centrifuged at 800g for 3 min at 4 °C and the supernatant was discarded, followed by resuspension in 2 ml FACS buffer. Equal volumes of samples were mixed thoroughly with 33% Percoll (GE17-0891-01) in PBS and spun at 800g for 30 min at 4 °C. Cells were washed once with FACS buffer and incubated with Fc block anti-CD16/32 (clone 2.4G2) (BD Biosciences, 553142) for 5 min on ice, washed with FACS buffer and incubated with anti-CD45–PerCP–Cy5.5, anti-CD11b–BV786, anti-F4/80–PE–Cy7, anti-I-A/I-E–BV605, anti-CD11c–PE and anti-CD64–APC–Fire750 for 40 min in 4 °C, protected from light (detailed information on antibodies and dilutions is provided in Supplementary Table 11). After staining, cells were washed with FACS buffer and then stained with the LIVE/DEAD Fixable Violet Dead Cell Stain Kit (Thermo Fisher Scientific, L34964) for 15 min at 4 °C, protected from light. Cell sorting was performed in a CytoFlex SRT cell sorter (Beckman Coulter) operated by CytExpert SRT v.1.1 software. A restrictive gating strategy was used to sort M0 macrophages and DCs (Supplementary Fig. The gating strategy was based on gating live singlets, next cells identified as CD45+, CD11c+ and I-A/I-E (MHCII)+, with the exclusion of cells that were double positive for CD64 and F4/80, were sorted as DCs. For post-sort analysis, cells were sorted to 100 µl of PBS. For RNA isolation, cells were sorted directly into 100 μl or 250 µl of extraction buffer (Applied Biosystems, KIT0204), depending on the cell amount, followed by centrifugation at 14,000g for 5 min and freezing until further processing. A549 and HEK293 Flp-In T-REx cell lines and their derivatives were cultured in Dulbecco's modified Eagle's medium (DMEM; Gibco) supplemented with 10% FBS (Gibco) and penicillin–streptomycin (Sigma-Aldrich) at 37 °C in a 5% CO2 atmosphere until 80% confluency. To produce the HEK293 Flp-In T-REx (R78007, Thermo Fisher Scientific) cell line with conditional knockdown of CNOT1, we used a strategy previously described by us29. A tri-miRNA construct (listed below; stem–loop sequences ATGGAAGAGCTTGGATTTGAT, CTCCCTCAATTCGCCAACTTA and AGGACTTGAAGGCCTTGTCAA) was designed, and cloned at the BspTI (AflII) + NotI restriction sites in a pKK-RNAi vector (pKK-BI16 nucCherry EGFP-TEV). A total of 1 × 106 HEK293 Flp-In T-REx cells, grown in DMEM high-glucose medium (Gibco) supplemented with 10% FBS on six-well plates, were transfected with 300 ng of the CNOT1-bearing construct mixed with 1 μg of pOG44 plasmid and 250 μl of Opti-MEM medium, supplemented with 2 µl TransIT-2020 transfection reagent (Mirus, MIR5400). The cells were seeded in a 60-mm plate and cultured in DMEM high-glucose medium supplemented with 40 µg ml−1 hygromycin and 8 µg ml−1 blasticidin for the first six to seven days. Cells were grown until single colonies appeared four weeks later. Cell enumeration was performed by crystal violet (Sigma, C3886) staining described by the laboratory of X. Chen (UCSF). In brief, cell medium was aspirated (or collected for subsequent ELISAs if necessary) and cells were washed with 1 ml PBS. Staining solution (0.25% crystal violet in 20% methanol) was then added to each well and cells were incubated for 10 min at room temperature. The primary mBMDM cultures were established from the bone-marrow monocytes isolated from Tent5aflox/floxTent5c−/−, FKBP12F36V-HA-Tent5a and wild-type mice. Young adult mice (12–25 weeks) of both sexes were euthanized by cervical dislocation. Material was mixed from several individual mice (siblings of the same sex) to obtain sufficient cells for subsequent analyses. The sex of the mice used as a source of bone marrow cells was not considered in further analyses. Bone-marrow cells were plated in IMDM medium (Thermo Fisher Scientific, 21980065) supplemented with 10% FBS (Gibco), 100 U ml−1 penicillin and 0.1 mg ml−1 streptomycin solution (Sigma-Aldrich) and 10 ng ml−1 macrophage colony-stimulating factor (M-CSF; PeproTech, 315-02), and cultured at 37 °C in 5% CO2. For conditional Tent5a gene targeting, mBMDMs were transduced on the eighth day after isolation with lentivirus carrying Cre recombinase (pCAG-Cre-IRES2-GFP). Lentivirus production, cell transduction and genotyping were performed as described previously3,7. Cells were used for experiments on the fourteenth day after isolation. To induce depletion of TENT5A, 0.7 × 106 mBMDMs obtained from wild-type and FKBP12F36V-HA-Tent5a mice were seeded in six-well plates and 24 h later, degron was induced by the addition of 1 µM dTAGv-1 or dimethyl sulfoxide (DMSO) as a control31. 32) using MessengerMAX (Invitrogen, LMRNA001) according to the manufacturer's instructions in five technical replicates. Diluted mRNA was mixed with diluted transfection reagent on 96-well plates, left for 40 min and then transferred to 384-well plates (Perkin Elmer, 6057300). Finally, cells were diluted to 5,000 cells per well and supplemented with dTAG-V1 (Bio-Techne, 7374/5) to a final concentration of 2 µM. Control cells were seeded without degron inductor. Cells were then seeded at a volume of 100 µl onto prepared plates with transfection mixes using the Multipette E3x (Eppendorf, 4987000029) and Combitip 5 ml Biopur (Eppendorf, 0030089669). The imaging was started 4 h after transfection in Opera Phenix from Perkin Elmer using three channels: 488 nm (time: 50 ms, power: 20%, height: −8.0 µm); bright-field (time: 20 ms, power: 20%, height: −0.0 µm); digital phase contrast (time: 20 ms, power: 20 %, height: −1.0 µm); water objective 20× non-confocal; binning 2 × 2. Buffy coats were obtained commercially from the Regional Blood Centre in Warsaw. Peripheral blood mononuclear cells (PBMCs) were isolated within 1–2 h after donation by density-gradient centrifugation using Lymphoprep (STEMCELL Technologies). Next, CD14+ PBMCs (monocytes) were isolated immunomagnetically with anti-human CD14 antibody-coated microbeads using LS columns (all from Miltenyi Biotec), strictly following the manufacturer's protocol. CD14+ cells were counted, and viability was checked in 0.1% trypan blue solution (Sigma-Aldrich) and used for subsequent hMDM differentiation. Each isolation resulted in a viability of over 95%. CD14+ PBMCs were seeded in a 10-cm tissue-culture-treated Petri dish at a density of 1 × 106–2 × 106 cells per ml in 10 ml of RPMI-1640 medium (Sigma-Aldrich) supplemented with 10% heat-inactivated FBS (HyClone), 2 mM l-glutamine (Sigma-Aldrich) 100 U ml−1 penicillin and 100 μg ml−1 streptomycin (both Sigma-Aldrich), 1% (v/v) MEM non-essential amino acids solution (Thermo Fisher Scientific) and 1 mM sodium pyruvate (Sigma-Aldrich) supplemented with 50 ng ml−1 of recombinant human M-CSF (for macrophage differentiation). Adherent cells collected from the M-CSF-supplemented culture were checked for markers of macrophages. hMDMs were stained with Zombie UV or Zombie Aqua Fixable Viability Kit (BioLegend) and antibodies detecting membrane markers CD14, CD83, CD86, CD163, CD206 and HLA-DR (listed in Supplementary Table 11). The antibody panel was optimized using isotype and fluorescence minus one (FMO) controls and a built-in compensation algorithm. Flow cytometry was performed using a Fortessa X-20 Analyzer (BD Biosciences) operated by FACSDiva 8.3 software. FlowJo v.10.6.1 software (BD Biosciences) was used for data analysis. The 0.5 × 106–1 × 106 cells were seeded the day before on a six-well plate in medium as described above. The required amount of vaccine mRNA in original LNP formulation was diluted in 150 µl of Opti-MEM medium (Thermo Fisher Scientific) at room temperature, and after 10 min was added to the cells in a drop-wise manner and gently mixed. Cells were collected at time points specified for the individual experiments. For the control cells (those not treated with the specified agent), an appropriate volume of OPTI-MEM was added. All transfections of purified vaccine mRNAs were performed with Lipofectamine MessengerMAX (Thermo Fisher Scientific), according to the manufacturer's instructions. Macrophages were fractionated using the Subcellular Protein Fractionation Kit for Cultured Cells (Thermo Fisher Scientific, 78840), according to the manufacturer's instructions. All buffers used were supplemented with RiboLock (Thermo Fisher Scientific, EO0382) to a final concentration of 1 U µl−1. The collected fractions for RNA isolation were immediately combined with TRIzol LS, incubated for five minutes at room temperature and subsequently frozen at –80 °C. For western blot analyses, samples were frozen in liquid nitrogen. siRNA-mediated knockdowns in hMDMs were performed using validated stealth siRNAs: HSS124645, HSS124646 and HSS183139 (all for TENT5A). Transfections were performed with Lipofectamine RNAiMAX (Invitrogen) according to the manufacturer's instructions, as previously described24,33. An equal number of cells were lysed in PBS supplemented with 0.1% NP40, protease inhibitors and viscolase (final concentration 0.1 U ml−1; A&A Biotechnology, 1010-100) for 30 min at 37 °C with shaking at 1,200 rpm. Then 3× SDS sample buffer (187.5 mM Tris-HCl pH 6.8, 6% SDS, 150 mM DTT, 0.02% bromophenol blue, 30% glycerol and 3% 2-mercaptoethanol) was added and samples were boiled for 10 min. Samples were resolved on 12–15% SDS–PAGE gels and then proteins were wet-transferred to Protran nitrocellulose membranes (GE Healthcare) at 400 mA at 4 °C for 1.5 h in 1× transfer buffer (25 mM Tris base, 192 mM glycine and 20% methanol (v/v). Next, the proteins were visualized by staining with 0.3% w/v Ponceau S in 3% v/v acetic acid and digitalized. Membranes were blocked by incubation in 5% milk in TBST buffer for 1 h followed by overnight incubation at 4 °C with specific primary antibodies (listed in Supplementary Table 11) diluted 1:2,500 (spike protein, CD80, GRP94, HA, SSR1, PERK, calreticulin and PDI) or 1:5,000 (α-tubulin, β-actin, actinin, eIF2a and GAPDH). Membranes were washed three times in TBST buffer for 10 min each and incubated with HRP-conjugated secondary antibodies (anti-mouse (Millipore, 401215) diluted 1:5,000 and anti-rabbit (Millipore, 401393) diluted 1:5,000) for 2 h at room temperature. Membranes were washed three times in TBST buffer and proteins were visualized using X-ray films or ChemiDoc (BioRad) or iBright (Thermo Fisher Scientific) imaging systems. Blots were quantified using Multi Gauge v.3.0 (Fujifilm Life Sciences). Unprocessed scans are shown as source data. Total RNA was isolated from cells or vaccine samples with TRIzol reagent or TRIzol LS reagent (both from Thermo Fisher Scientific), respectively, according to the manufacturer's instructions, dissolved in nuclease-free water and stored at −20 °C (short term) or −80 °C (long term). RNA from frozen muscles and lymph nodes was isolated by tissue homogenization in TRI reagent (Sigma, T9424) pre-heated to 60 °C, using Omni Tissue Homogenizer equipped with a 7 × 115-mm Saw Tooth (Fine) Generator Probe. Homogenous mixtures were further processed according to the manufacturer's instructions. For qRT–PCR, RNA-seq and DRS library preparation, the RNA was treated with TURBO DNase (Thermo Fisher Scientific, AM1907; or Invitrogen, AM2238). To assess the integrity of isolates, each RNA sample after DNAse treatment was analysed with the Agilent 2200 TapeStation system, using Agilent High Senitivity RNA ScreenTape (Agilent, 5067-5579). RNA from cells acquired by fluorescence-activated cell sorting (FACS) was isolated using the Arcturus PicoPure RNA Isolation Kit (Applied Biosystems, KIT0204) following guidelines for RNA isolation from cell pellets with on-column DNAse treatment (Qiagen, 79256), with some modifications. For qRT–PCR analyses, we sorted 1,000 cells of each type, from separate mice, directly to 100 µl extraction buffer. In the case of cDNA sequencing, we sorted 20,000 cells of each type, pooled from three mice, directly to 250 µl extraction buffer. cDNA was synthesized using 500 ng of DNAsed total RNA as a template with SuperScript III Reverse Transcriptase (Invitrogen, 18080093) according to the manufacturer's instructions. The final cDNA concentration was kept at 2.5 ng µl−1 (converted from total RNA). In the case of RNA from FACS-sorted cells, 11 µl of the entire eluate was used for cDNA synthesis. To determine the concentration of Moderna's mRNA-1273 in mouse tissues or FACS-sorted cells, a custom-made TaqMan probe together with TaqMan Gene Expression Master Mix (Applied Biosystems, 4369016) was used and normalized with a pre-designed gene-expression assay for β-actin (Applied Biosystems, Mm01205647_g1). The reaction mix was contained in a total volume of 10 µl with 1× concentration of master mix and gene-expression assay; 5 ng cDNA was used per reaction (converted from total RNA). A thermal cycling program for TaqMan Gene Expression Master Mix was used as instructed by the manufacturer. To estimate the relative expression of Tent5a and Tent5c at injection sites or in FACS-sorted cells, and the levels of mRNA-1273 in HEK293T and A549 cells with varying amounts of vaccine, we used Platinum SYBR Green qPCR SuperMix-UDG (Invitrogen, 11733046) following the general protocol for ABI instruments recommended by the manufacturer. For all qRT–PCR analyses, the QuantStudio 5 Real-Time PCR System, 384-well (Applied Biosystems, A28140) was used. The primer blast algorithm was used to find a unique amplicon for mRNA-1273 in the mouse transcriptome (Refseq mRNA). Within the amplified region, a 15-mer of optimal GC content (53.3%) was selected as the target site for probe hybridization. To test the specificity and sensitivity of the designed assay, we performed a serial dilution experiment in which we spiked 500 ng of DNAsed mouse total RNA with 50 ng of DNAsed mRNA-1273 and performed cDNA synthesis. Then we performed 10× serial dilutions of that mix with unspiked mouse cDNA of the same concentration. qRT–PCR analysis showed that the designed assay is specific and can detect up to 10 ag of mRNA-1273 per ng of total RNA, which is equivalent to roughly 25 molecules. Similarly, we prepared a standard curve with 10× serial dilutions of mRNA-1273 to estimate the absolute concentration of vaccine in FACS-acquired cells. The concentration of vaccine before cDNA synthesis was determined with the Agilent 2200 TapeStation system using Agilent High Sensitivity RNA tape. Next, we performed eight serial dilutions (10×) using yeast cDNA of 2.5 ng µl−1 concentration. Recorded Ct values for each concentration served as data points for plotting an exponential curve, and we used this equation to convert Ct values to molecules of mRNA-1273, assuming its molecular mass is 1321.81 kDa. Next, the calculated number of molecules was normalized to the number of cells in a 384-well plate (translated from the % of eluate used for cDNA synthesis, assuming 100% recovery of cells during sorting and RNA isolation as well as cDNA synthesis efficiency). The relative expression of genes associated with ER stress, we assessed CHOP (also known as DDIT3), GRP94 (HSP90B1), PERK (EIF2AK3) and XBP1 mRNA splicing in mRNA-1273-transfected hMDMs (wild-type and TENT5A knockdown) using the Platinum SYBR Green qPCR SuperMix-UDG (Invitrogen, 11733046), following the general protocol for ABI instruments recommended by the manufacturer. Gene expression for each sample was normalized to ACTB. Primers and probe sequences are listed in Supplementary Table 12. Then, 1 µg of vaccine mRNA was ligated to 20 pmol RA3_7N adaptor: 5rApp/CTGACNNNNNNNTGGAATTCTCGGGTGCCAAGG/3ddC with 10 U of T4 KQ227 RNA ligase 1 (NEB, M0204S) in the presence of 20 U RNase OUT (Thermo Fisher Scientific, 10777019), 1× T4 RNA Ligase Reaction Buffer (NEB, M0204S), 1 mM ATP and 20% PEG8000 in a total reaction volume of 20 µl at 25 °C for 4 h. Ligase was inactivated at 65 °C for 20 min. The ligation product was purified with a 0.8× ratio of KAPA Pure Beads and eluted with 15 µl RNase-free water, according to the manufacturer's protocol. The cleaned ligation product was subjected to reverse transcription with 40 pmol of Illumina index adapter: 5′-CAAGCAGAAGACGGCATACGAGATATCAGTGTGACTGGAGTTCCTTGGCACCCGAGAATTCCA-3′ with SuperScript III (Thermo Fisher Scientific, 18080093), 1× First Strand Buffer (Thermo Fisher Scientific, 18080093), 0.25 mM dNTP mix, 5 mM DTT and 20 U RNAse OUT (Thermo Fisher Scientific, 10777019). The reverse transcription product was cleaned with a 1× ratio of KAPA Pure Beads and eluted with 19 µl RNase-free water. Prepared cDNA was diluted in a 1:3 proportion. In the library amplification step, 0.5 µl of cDNA was mixed with 0.4 pmol of gene-specific starter 5′-AATGATACGGCGACCACCGAGATCTACACGTTCAGAGTTCTACAGTCCGACGATCAGAAGGAGATCGATCGGCTG-3′, 0.2 pmol of RP universal starter 5′-CAAGCAGAAGACGGCATACGAGAT-3′, 0.25 mM dNTP, Phusion High-Fidelity DNA Polymerase (Thermo Fisher Scientific, F530S), 1× Phusion HF Buffer (Thermo Fisher Scientific, F530S) and 3% DMSO. The 50 µl of PCR product was separated in 2.5% agarose gel. The library was cleaned twice with a 1.0× ratio of KAPA Pure Beads. TapeStation analysis of the sample was performed as a quality control. The library was sequenced on an Illumina NovaSeq 6000 sequencer. RA37_N adapter sequence was trimmed from the R2 read (containing poly(A) tail) with cutadapt34 (options -g CCTTGGCACCCGAGAATTCCANNNNNNNGTCAG –discard-untrimmed). Then only reads containing poly(A) tail were identified with cutadapt (options -a TTTTTT --discard-untrimmed --fasta). Obtained sequences were reverse complemented using fastx_reverse_complement from the fastx toolkit (0.0.13) and loaded into R with the BioStrings package. To get rid of unwanted trimming artefacts, sequences with length between 0 and 10 were chosen, four A letters (representing the poly(A) tail) were pasted before each sequence for visualization purposes and a sequence logo representing the nucleotide composition of the 3′ end of mRNA-1273 was produced with the ggseqlogo package35. For in vitro translation, the Retic Lysate IVT Kit (Invitrogen, AM1200M) was used. The translation mix (–met) was supplemented with unlabelled methionine to a final concentration of 50 μM. Spike protein was detected in the reaction mixture using ELISA. Spike-in RNAs were IVT from a set of double-stranded DNA fragments. Templates for transcription were prepared in two consecutive PCR reactions. First, a desired fragment of Renilla luciferase from pRL5Box plasmid was amplified using RLuc_F1/R1-specific primers containing an overhang common for all primers used in the second round of PCR (Supplementary Table 12). PCR products were verified through gel electrophoresis. Correct amplicons were used as templates in the second PCR reaction with RLuc_T7_F2 primer, hybridizing to the overhang sequence from RLuc_F1 and containing the T7 promoter, and backward primer RLuc_Ax_R2, hybridizing to the overhang sequence from RLuc_R1 oligo and introducing a poly(A) tail of a defined length (from 10 to 120 A). The resulting PCR products were assessed and purified by gel electrophoresis. Then, DNA template was removed with TURBO DNase (Ambion) for the next 15 min. Spike-in RNAs were phenol–chloroform extracted, precipitated, visually assessed by denaturing electrophoresis, purified on RNA purification beads and used as controls in DRS runs. Template for transcription of EGFP mRNA was prepared as follows: the high-RSCU EGFP32 CDS was purchased from Invitrogen and cloned into the plasmid vector pJAZZ (Lucigen) using BigEasy-TSA competent bacteria (Lucigen), providing it with the 5′-UTR and 3′-UTR of the mRNA-1273, a poly(A) tail of 90 A and the T7 promoter Φ6.5 (TAATACGACTCACTATAGGG). The plasmid was digested at the poly(A) tail terminus (PaqCI, NEB) and was verified to be completely digested through agarose gel electrophoresis. DNA template was purified by buffered phenol–chloroform extraction and ethanol precipitation. Templates for transcription of reporter mRNAs were prepared in a PCR reaction (Supplementary Note 6 and Extended Data Fig. 8a,b) as follows: full-length CDSs of Plasmodium falciparum circumsporozoite (CS) protein (PfCSP_3D7; NCBI reference sequence: XP_001351122.1; Extended Data Fig. 7h), OVA (sequence available at https://www.trilinkbiotech.com/) and Zika virus protein E (ZIKVE; NCBI reference sequence: YP_002790881.1) with UTRs from either mRNA-1273 or BNT162b2 were synthesized using the GeneArt service (Thermo Fisher Scientific). For OVA and ZIKVE constructs, an N-terminal ER signal peptide sequence was added (MFVFLVLLPLVSSQCV). All CDSs were optimized for expression in mouse cells using software from Benchling (https://www.benchling.com/). DNAs were PCR-amplified using primers complementary to UTRs with overhangs introducing the RNA T7 promoter Φ6.5 (TAATACGACTCACTATAGGG) at the 5′ end and a poly(A) tail from either mRNA-1273 or Pfizer BNT162b2 and either with or without a TCTAG pentamer at the 3′ end (Supplementary Table 12). In the case of OVA DNAs, an additional set was PCR-amplified as described above, but with primers with a point mutation in the T7 promoter sequence, allowing for efficient cotranscriptional RNA capping (TAATACGACTCACTATAAGG). PCR products were verified through agarose gel electrophoresis and purified with KAPA Pure Beads (KAPA Biosystems). The reaction mixtures were supplemented with 1 U µl−1 RiboLock RNase inhibitor (Thermo Fisher Scientific), 0.002 U µl−1 inorganic pyrophosphatase (Thermo Fisher Scientific) and, in the case of EGFP and OVA reporter mRNAs intended for cotranscriptional capping, 10 mM of the trinucleotide Cap1 analogue m7GpppAmpG37 (in-house prepared). The reactions were performed for 120 min at 37 °C. In vitro transcription products were verified through denaturing agarose gel electrophoresis and FPLC-purified using a 0.2 ml GoPure column with POROS Oligo (dT)25 Affinity Resin (Thermo Fisher Scientific) according to the manufacturer's protocol. The concentration of purified mRNAs was determined using UV absorbance measurements. The purity and integrity of the mRNA preparations were determined by automated electrophoresis (TapeStation 2200, Agilent Technologies; Extended Data Fig. Two micrograms of RNA isolated from a sample of mRNA-1273 vaccine was denatured in the presence of 20 U RNAse OUT (10777019, Thermo Fisher Scientific) at 65 °C for 3 min, and immediately placed on ice. An inosine (I)-tailing reaction was performed with 0.5 mM ITP (inosine triphosphate), 1× NEB 2.0 buffer (B7002, NEB) and 2 U poly(U) polymerase (M0337S, NEB) at 37 °C for 45 min, and was terminated by snap-freezing in liquid nitrogen. I-tailed RNA was cleaned twice on KAPA Pure Beads (7983298001, Roche) in a 1× ratio and used for DRS library preparation. I-tailed samples require ligation of a special adaptor RTA_C10, which contains 10 cytosines at the 3′ end: 5′-GAGGCGAGCGGTCAATTTTCCTA AGAGCAAGAAGAAGCCCCCCCCCCCC-3′. To make the I-tailing-specific RTA adaptor, we followed the ONT Direct RNA Sequencing—Sequence-Specific protocol; 0.5 µg of I-tailed mRNA-1273 was taken to the first ligation step of DRS library preparation. Library preparation was performed as described in the Direct RNA Sequencing (ONT, SQK-RNA002) protocol. For RNA isolates obtained from cell cultures or mouse tissues, 3.5–5 µg of total mRNA was mixed with 50–200 ng oligo-(dT)25-enriched mRNA from Saccharomyces cerevisiae and standards with predefined poly(A) lengths, and was processed with a Direct RNA Sequencing Kit (SQK-RNA002, ONT) according to the manufacturer's instructions. For cDNA sequencing, 400 ng of total RNA was used for library preparation using the cDNA-PCR Sequencing Kit (SQK-PCS111) or PCR-cDNA Barcoding Kit (SQK-PCB111.24 or SQK-PCB114.24) according to the manufacturer's instructions. Libraries were amplified with 14 cycles of PCR reaction, and the DNA concentration was measured using the Qubit 1× dsDNA High Sensitivity Kit (Invitrogen, Q33231). Then, if multiplexed, they were diluted and mixed equally to reach a final concentration of 60–100 fmol. cDNA libraries for sequencing with an additional amplification step of mRNA-1273 (applicable for samples with an expected low prevalence of vaccine mRNA, such as cells sorted from vaccine injection sites) were prepared using the standard SQK-PCS111 protocol with the following modification. After synthesis of the first cDNA strand, samples from the reverse transcription reaction were pre-amplified in a 12-cycle PCR reaction with RTP and SSPII_Mod_2 (specific for mRNA-1273, with UMI sequence TTTCTGTTGGTGCTGATATTGCTTTVVVVTTVVVVTTVVVVTTVVVVTTTCCACCGACAACACCTTCGTGAGCGG) primers using the LongAmp Hot Start Taq Master Mix. The reaction products were then purified on Kappa beads, washed with SFB buffer, eluted with H2O and used for the subsequent PCR reaction with barcoded primers, performed according to the SQK-PCB111-24 protocol (14 cycles, 5 min amplification). Finally, cDNA concentrations were measured with Qubit and 12 libraries containing a total of around 60 ng cDNA were used for ligation with the RAPT adapter and sequenced. Sequencing was performed using R9.4 (for DRS and v.11 DNA chemistry) or R10.4 (for v.14 DNA chemistry) flow cells on a MinION device (ONT) and with the Flow Cell Priming Kit (EXP-FLP002). Raw data were basecalled using Guppy (ONT) in the case of DRS, or dorado (ONT) in the case of cDNA sequencing. Raw sequencing data (fast5 or pod5 files) were deposited at the European Nucleotide Archive (ENA, project PRJEB53190; Supplementary Table 13). Basecalled nanopore reads were mapped to the respective transcriptome references (Gencode 26 or Gencode 38 for mouse and human samples, respectively) using Minimap2 2.17 with options -k 14 -ax map-ont –secondary=no, and processed with Samtools 1.9 to filter out supplementary alignments and read mapping to reverse strand (Samtools view -b -F 2320). For the analysis of BNT162b2-originating reads, the nanopolish polya algorithm was modified analogously (Supplementary Note 3) to (1) include unmapped reads enabling the analysis of poly(A) lengths for sDTW-identified reads and (2) detect two poly(A) segments interleaved with a 10-nt linker and report their lengths in the output. For cDNA barcoded libraries (SQK-PCB111-24), raw sequencing data were basecalled with either dorado 0.5.3 or dorado 0.7.0 with parallel mapping to relevant reference (with option –secondary=no to exclude secondary alignments) and poly(A) detection turned on with the option –estimate-poly-a. Poly(A) lengths for each sequencing read were extracted from the pt:i tag (in the basecalled bam file) with a Python script. P values for each transcript were estimated using the Kruskal–Wallis test and adjusted for multiple comparisons using the Benjamini–Hochberg method. Transcripts were considered as having a significant change in poly(A) tail length if the adjusted P value was less than 0.05 and if there were at least 20 supporting reads for each condition. Illumina RNA-seq reads were mapped to the mouse reference genome (GRCm38, ENSEMBL, release 94) using the STAR aligner (v.2.7.6a)39. Read counts were assigned to genes using featureCounts from the Subread package (v.2.0.1) with options -Q 10 -p -B -C -s 2 -g gene_id -t exon and the respective annotation file (Gencode v.M25). Multimappers and reads overlapping multiple features were not counted. Nanopore read counts were derived from the nanopolish polya output files, used for the analyses of poly(A) tail lengths. Reads assigned for each gene were summarized and obtained counts were used for subsequent analyses. Differential expression analysis was performed with the DESeq2 (v.1.22) Bioconductor package40, using a likelihood ratio test for data from time-course experiments. Pair-wise comparisons of each time point were calculated with the Wald test, with log fold change values shrunk using the apeglm method. Genes with similar expression patterns were clustered with hierarchical clustering using the ward.D method and z-scored expression values for each gene. Heat maps were drawn with the ComplexHeatmap package41. g:Profiler42 or ClusterProfiler were used for Gene Ontology (GO) enrichment analysis. To compute the codon adaptation index, a reference set of 500 highly expressed genes in mBMDMs, based on a previous study7, was provided. For calculating the effective number of codons, codon bias or preference within the vertebrate context was used. The mRNA vaccines BNT162b2 and mRNA-1273 were compared against the SARS-CoV-2 spike protein reference sequence from GenBank (accession: NC_045512.2:21563-25384), mouse endogenous transcripts (protein-coding), transcript groups selected by GO terms (for example, associated with the membrane or endoplasmic reticulum) and potential substrates of TENT5A and TENT5C, selected as per a previous study7, as well as IVT reporter sequences for OVA, PfCSP and ZikVE. GC content and other sequence features were assessed using functions available from Bioconductor and the GenRCA Rare Codon Analysis Tool (https://www.genscript.com/tools/rare-codon-analysis), providing mouse as the host organism and the Kazusa database as the codon usage reference. No statistical methods were used to predetermine sample sizes. Statistical analysis was performed on data from two or more biologically independent experimental replicates using the R environment or Prism 6 software unless otherwise stated. Data from the analyses of poly(A) tail length are presented as scatter dot plots, as indicated in the figure legends, and individual data points are shown. The median of poly(A) length is shown on the plots (as a horizontal bar and a numeric value) for mRNA populations with processed poly(A) tails (that is, without the terminal pentamer in the case of mRNA-1273, OVA, PfCSP and ZikVE reporters, or for all the reads in the case of BNT162b2); the mean value would be biased because poly(A) lengths do not follow a normal distribution. Plots indicate fractions of reads with elongated (blue-shaded) or shortened (red-shaded) tails. Classification thresholds (deadenylation and re-adenylation) were estimated as values close to the 0.2 and 0.8 quantiles (respectively) from crude vaccine RNA (with pentamer in the case of mRNA-1273). For cDNA sequencing analysis with dorado, which usually gives shorter estimates of poly(A) tail length, the thresholds were set at 80 and 110 for mRNA-1273 tails and 50 and 80 for BNT162b2 tails (analysis was performed without additional segmentation, so the length of the last poly(A) segment was reported). Samples with clear technical failures during tissue collection, cell isolation, processing or data collection were excluded from the analyses. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Nanopore direct RNA sequences have been deposited at the European Nucleotide Archive (ENA) under accession number PRJEB53190 (all sequencing datasets with corresponding ENA accession numbers are also listed in Supplementary Table 13). Illumina sequencing data have been deposited at the Gene Expression Omnibus (GEO) under accession number GSE233059. Raw data underlying the figures are provided as source data or supplementary files or are available from the corresponding authors upon reasonable request. Source data are provided with this paper. The dynamic time warping script is available at https://github.com/LRB-IIMCB/DTW_mRNA-1273. Nanopolish polya for identification of mΨCmΨAG is available at https://github.com/LRB-IIMCB/nanopolish_mRNA-1273_BNT162b2. Highly parallel direct RNA sequencing on an array of nanopores. Bilska, A. et al. Immunoglobulin expression and the humoral immune response is regulated by the non-canonical poly(A) polymerase TENT5C. Corbett, K. S. et al. SARS-CoV-2 mRNA vaccine design enabled by prototype pathogen preparedness. Gewartowska, O. et al. Cytoplasmic polyadenylation by TENT5A is required for proper bone formation. Brouze, M. et al. TENT5-mediated polyadenylation of mRNAs encoding secreted proteins is essential for gametogenesis in mice. Liudkovska, V. et al. TENT5 cytoplasmic noncanonical poly(A) polymerases regulate the innate immune response in animals. The safety and immunogenicity of two Zika virus mRNA vaccine candidates in healthy flavivirus baseline seropositive and seronegative adults: the results of two randomised, placebo-controlled, dose-ranging, phase 1 clinical trials. Hayashi, C. T. H. et al. mRNA-LNP expressing PfCSP and Pfs25 vaccine candidates targeting infection and transmission of Plasmodium falciparum. Sahin, U. et al. BNT162b2 vaccine induces neutralizing antibodies and poly-specific T cells in humans. B. et al. BNT162b vaccines protect rhesus macaques from SARS-CoV-2. Sahin, U., Karikó, K. & Türeci, Ö. mRNA-based therapeutics—developing a new class of drugs. & Vlatkovic, I. COVID-19 mRNA vaccines: platforms and current developments. Li, C. et al. Mechanisms of innate and adaptive immunity to the Pfizer-BioNTech BNT162b2 vaccine. Lindsay, K. E. et al. Visualization of early events in mRNA vaccine delivery in non-human primates via PET–CT and near-infrared imaging. Ols, S. et al. Route of vaccine administration alters antigen trafficking but not innate or adaptive immunity. Verbeke, R., Hogan, M. J., Loré, K. & Pardi, N. Innate immune mechanisms of mRNA vaccines. Efficient targeting and activation of antigen-presenting cells in vivo after modified mRNA vaccine administration in rhesus macaques. Selective uptake of naked vaccine RNA by dendritic cells is driven by macropinocytosis and abrogated upon DC maturation. Systemic RNA delivery to dendritic cells exploits antiviral defence for cancer immunotherapy. Hassett, K. J. et al. mRNA vaccine trafficking and resulting protein expression after intramuscular administration. The non-canonical poly(A) polymerase FAM46C acts as an onco-suppressor in multiple myeloma. & Spaan, W. J. M. The coronavirus spike protein induces endoplasmic reticulum stress and upregulation of intracellular chemokine mRNA concentrations. Beck, J. D. et al. mRNA therapeutics in cancer immunotherapy. Szczesny, R. J. et al. Versatile approach for functional analysis of human proteins and efficient stable cell line generation using FLP-mediated recombination system. The dTAG system for immediate and target-specific protein degradation. Mauger, D. M. et al. mRNA structure regulates protein expression through changes in functional half-life. Mroczek, S. et al. C16orf57, a gene mutated in poikiloderma with neutropenia, encodes a putative phosphodiesterase responsible for the U6 snRNA 3′ end modification. Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Wagih, O. ggseqlogo: a versatile R package for drawing sequence logos. & Rabideau, A. E. An engineered T7 RNA polymerase that produces mRNA free of immunostimulatory byproducts. The identity and methylation status of the first transcribed nucleotide in eukaryotic mRNA 5′ cap modulates protein expression in living cells. & Sprangers, R. A general method for rapid and cost-efficient large-scale production of 5′ capped RNA. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Gu, Z., Eils, R. & Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Raudvere, U. et al. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). We thank all members of the Laboratory of RNA Biology and the Genome Engineering Facility (GEF) at the International Institute of Molecular and Cell Biology (IIMCB) for their expertise and help with selected experiments; M. Brouze for help with analysing microscopy data; E. Nowak for help with protein purification; M. Kuczmierowski and M. Piotrowska for providing remnant vaccines; and Z. Pilch and K. Czubak for preliminary tests of macrophage isolation from lymphoid organs. Illumina sequencing was performed by the Genomics Core Facility, Centre of New Technologies, University of Warsaw (RRID:SCR_022718). These authors contributed equally: Michał Mazur, Wiktoria Orzeł, Olga Gewartowska Paweł S. Krawczyk, Michał Mazur, Wiktoria Orzeł, Sebastian Jeleń, Wiktor Antczak, Karolina Kasztelan, Aleksandra Brouze, Katarzyna Matylla-Kulińska, Natalia Gumińska, Bartosz Tarkowski, Ewelina P. Owczarek, Kamila Affek, Monika Kusio-Kobiałka, Andrzej Dziembowski & Seweryn Mroczek Wiktoria Orzeł, Sebastian Jeleń, Wiktor Antczak, Aleksandra Brouze, Katarzyna Matylla-Kulińska & Seweryn Mroczek Genome Engineering Facility, International Institute of Molecular and Cell Biology, Warsaw, Poland Institute of Biochemistry and Biophysics, Warsaw, Poland You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar established most of the mBMDM cultures, performed all cell-line-based experiments (except the Fndc3 knockdown), performed all western blots and fractionations, isolated most of the RNA, participated in library preparation for DRS and cDNA sequencing and designed constructs for antigen expression. performed mouse immunizations, serum collection and some of the ELISA measurements, and designed the FKBP12F36V-HA-Tent5a mouse line. established the protocol for cell isolation from injection sites, performed mouse immunizations, RNA isolation and qPCR on mouse tissues and FACS-acquired cells and performed cytometry analysis and FACS of cells infiltrating injection sites (together with M.K.-K.). K.M.-K. prepared poly(A) length standards and participated in the I-tailing experiment (together with W.O.). reanalysed single-cell RNA-seq data and took part in immunization experiments. analysed reporter mRNA translation dynamics in FKBP12F36V-HA-Tent5a mBMDMs. participated in the first steps of IVT reporter RNA synthesis. and A.D. wrote the manuscript, with contributions from other authors. All authors read and approved the manuscript. Correspondence to Andrzej Dziembowski or Seweryn Mroczek. The remaining authors declare no competing interests. Nature thanks Congcong Xu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1. a, Read length distribution of eDRS data. Blue – reads mapped to mRNA-1273 after standard basecalling by Guppy. Orange – reads identified as mRNA-1273 using an sDTW approach. Grey – reads not attributed to mRNA-1273 (too low quality). Mismatches are visible in positions where mΨ was incorporated. Reference signal (on the left, with the part of poly(A) tail marked in green) is compared with the query signal (on top), and a subsequence match location is found (marked in red). Heat map shows the similarities between different parts of reference and query signals, used for drawing the warping path (in red). d, Distribution of sDTW distances (compared to reference signal) of crude mRNA-1273 sample, and HEK293T samples incubated with mRNA-1273 (replicate 1, for clarity only time points 4 h, 12 h, and 24 h are shown). e, Comparison of mRNA-1273 recovery using standard basecalling and sDTW approach. f, Comparison of mRNA-1273 reads recovery from sequencing data of HEK293 + mRNA-1273 cells, using standard basecalling and sDTW approach. g, Distribution of read lengths from mRNA-1273 eDRS, for reads passing quality filter (mean qscore ≥7, bottom), or failing (mean qscore <7, top). In blue - reads basecalled using Guppy and aligned to the reference sequence. In yellow- reads identified as mRNA-1273 using sDTW approach. Signal from poly(A) is zoomed in the bottom panel. c, Distribution of poly(A) lengths for BNT162b2, separately for first (green, 30 adenines) and second poly(A) segment (in khaki, 70 adenines) and whole poly(A) tail (110 nt, in light blue). d, Distribution of poly(A) lengths for mRNA-1273 containing (blue) and lacking (green) 3′ terminal mΨCmΨAG. Bottom – fraction of reads in each group is indicated. e, Distribution of poly(A) lengths for mRNA-1273 eDRS reads, for reads passing quality filter (mean qscore >=7, bottom), or failing (mean qscore <7, top). In blue- reads basecalled using Guppy and aligned to reference sequence. In yellow - reads identified as mRNA-1273 using sDTW approach. f, Representative raw eDRS signal from I-tailed mRNA-1273. Signal from poly(A) is zoomed in the bottom panel. g, Distribution of poly(A) lengths for mRNA-1273 (left) or BNT162b2 (right, second segment, 70 adenines), obtained with nanopore cDNA sequencing. h, Western blots for spike protein showing efficient translation of mRNA-1273 in HEK293T and A549 cells 24 h after treatment with varying amounts of vaccine LNPs. Asterisk indicates unspecific signal from untreated cells. Representative picture from 2 independent biological experiments shown. i, Amounts of vaccine RNA in HEK293T or A549 cells incubated with 0.2-10 µg of mRNA-1273 LNPs per well of 6-well plate for 24 h, assessed with qPCR (n = 2 independent biological experiments). j, mRNA-1273 poly(A) length distribution in A549 cells treated with the vaccine LNPs for up to 72 h. k, Western blot analysis of spike protein expression from mRNA-1273 and BNT162b2 in HEK293T cells up to 96 h after RNA transfection with lipofectamine-based reagent. Western Blot on spike protein; α-tubulin, GAPDH and Ponceau staining served as loading controls. Asterisk indicates unspecific signal from untreated cells. Bottom, quantification of spike protein levels (2 independent biological experiments). l, Poly(A) length distribution for mRNA-1273 (top panel) or BNT162b2 (lower panel) in HEK293T cells treated with the vaccine LNPs for up to 96 h, obtained with nanopore cDNA sequencing. m, Distribution of poly(A) lengths of endogenous transcripts in HEK293T cells with tetracycline-induced depletion of CNOT1 (CNOT1 KD, shown in black) or controls (WT, without tetracycline, shown in orange) in time course up to 72 h. Boxes show 25th-75th percentiles with median line; whiskers extend to 1.5×interquartile range (IQR). Data for a,b,c,d,e,f,j and m were obtained with eDRS, for g and l with cDNA sequencing. For d,g,j, and l (top) poly(A) tail lengths are shown with reads containing (blue) or lacking (green) 3′ terminal mΨCmΨAG, with median values and fraction of shortened tails shown. Data represent 2 independent biological experiments (l and m) or single experiment (j). 2. a, Abundance of mRNA-1273 in tissues surrounding injection sites or in the dLNs, 2 h, 8 h, or 24 h following immunization, expressed as ΔCt (cycle threshold) mRNA-1273 Ct related to ActB Ct values. N = 5 independent biological replicates (mice) for each time point. Values from the same mouse at given time point indicated with matching shapes. b, eDRS-based mRNA-1273 poly(A) lengths distribution in muscle-resident cells collected from tissues surrounding injection sites 24 h after immunization with mRNA-1273 vaccine. N = 6 independent biological experiments (mice) for each time point. 5. d, cDNA-seq measured mRNA-1273 poly(A) lengths distribution in muscle-resident DCs (CD45+, I-A/I-E (MHCII)high, CD11c+, F4/80− and CD64−) and macrophages (CD45+, CD11c+, F4/80+, CD64+), sorted from vaccine injection sites 12 h and 24 h after administration. Values are shown as fold change of vaccine molecules per cell, relative to 6 h time point (top) or ΔCt (cycle threshold) mRNA-1273 Ct related to ActB Ct values (bottom). N = 3 independent biological experiments (mice) for each time point. f,g, Western blot analysis of spike protein expression showing efficient translation of mRNA-1273 in mBMDMs (f) or hMDMs (g) up to 96 h (f) or 72 h (g) after mRNA LNPs delivery. Western blot on spike protein, CD80 as the activation marker (g), α-tubulin and GAPDH levels and Ponceau S staining (bottom) served as loading controls. h, Representative raw DRS currents of mRNA-1273 poly(A) tails detected in samples 24 h after administration to mBMDMs. Unprocessed poly(A) tail with intact pentamer at the poly(A) 3'end (first panel), re-adenylated poly(A) tail with intact pentamer inside poly(A) (second panel), and re-adenylated poly(A) tails without signs of pentamer signature (third, fourth and fifth panel). Expected position of the mΨCmΨAG pentamer signature is marked with grey shading. Out of 100 randomly selected raw nanopore currents, only 2 contained pentamer remnants inside poly(A) stretch. i, eDRS-based BNT162b2 poly(A) lengths distribution in mBMDMs 12 h and 24 h after vaccine administration (related to Fig. Density plot, with values scaled in range 0–1, is shown. j, Spike protein production in mBMDMs up to 72 h after mRNA-1273 (left) or BNT162b2 (right) delivery. Western blot on spike protein, GAPDH, β-actin, and Ponceau staining (bottom) served as loading controls. k, Amount of spike protein produced 24 h and 48 h after a dose-dependent administration (0.2-10 µg of mRNA LNPs per well in 6-well plate) of mRNA-1273 (left) or BNT162b2 (right), assessed by ELISA from culture medium. For b and d, reads are divided into containing (blue) or lacking (green) mΨCmΨAG sequence, with median values and fractions of elongated or shortened tails shown. For d, sequencing of material pooled from 6 mice was done. 3. a, Transcripts with changed poly(A) lengths (revealed with Kruskal–Wallis test) in the time course (up to 72 h) upon mRNA-1273 LNPs administration to hMDMs. b, Genes with changed expression in tissues surrounding injection sites upon mRNA-1273 administration. Only genes with significant change in expression (DESeq2 LRT test P < 0.05) are shown. c, Genes with changed expression in mBMDMs upon mRNA-1273 LNPs administration. Only genes with significant change in expression (DESeq2 LRT test P < 0.05) are shown. d, Differentially expressed genes following administration of mRNA-1273 in hMDMs. Numbers of genes up- and downregulated at each time point (in comparison to time 0 h) are shown. Differential expression analysis done with DESeq2 R package. e,f, GO (Biological Process) terms significantly enriched for genes upregulated 24 h (e) or 72 h (f) after mRNA-1273 administration in mBMDMs (e) or hMDMs (f). g, Expression of Tent5a (left panel) or Tent5c (right panel) at mRNA-1273 injection sites measured with qPCR, presented as fold change (relative to 2 h). P values were calculated with two-sided Wilcoxon test. Group sizes (number of mice included) provided on the plot. h, Expression (DESeq2-normalized counts) of deadenylase complexes subunits in mBMDMs treated with mRNA-1273 LNPs for up to 72 h. Gene names are on the left, adjusted P values from the DESeq2 LRT test are shown on the right, with bar height indicating significance. i, Cells with detected Tent5a, BNT162b2, both, or none of them, in macrophages (top) of DCs (bottom) in the single-cell RNA-seq data from a previous study15. For a–c: Clustering done using hclust, with ward.D method. Descriptions of major functionalities enriched in a given cluster are shown on the right. For e and f: on the x-axis the percentage of all upregulated genes assigned to given GO term. Data from 2 (a,b,d,f) or 3 (c,e,h) sequencing datasets from independent biological experiments. 4. a, Expression of TENT5A (FKBP12F36V-HA-Tent5a) after mRNA-1273 administration (for up to 72 h) to mBMDMs derived from wild-type or FKBP12F36V-HA-Tent5a knock-in mouse line. Both lines were treated with dTAG-V1 inducer or untreated. Western blot on HA (to detect TENT5A) and spike; GAPDH and Ponceau (bottom) staining served as loading controls. GAPDH and Ponceau (bottom) staining served as loading controls. c, Expression of genes induced in mBMDMs upon mRNA-1273 addition, shown for wild-type (left) and (Tent5aflox/flox/Tent5c−/−) cells (right). Clusters and genes order as in Extended Data Fig. d, Changes in poly(A) tails lengths for mBMDMs wild-type (left) and (Tent5aflox/flox/Tent5c−/−) (right) in time course after addition of mRNA-1273. Z-score normalized poly(A) lengths are represented in colour scale. Clusters and gene order as in Fig. e, EGFP translation efficiency in mBMDMs derived from FKBP12F36V-HA-Tent5a knock-in mouse line. Cells were treated with dTAG-V1 inducer or untreated. EGFP intensity values are shown (normalized to cell number) for untreated (bottom left) or transfected with either uridine- (top left) or mΨ-mRNA (top right) cells in a time course after transfection. Area under curve was calculated for each condition (bottom right) and P values calculated using Wilcoxon test. Boxes show 25th-75th percentiles with median line; whiskers extend to 1.5×interquartile range (IQR). f, Expression (DESeq2-normalized counts) of non-canonical poly(A)/poly(U) polymerases in mBMDMs transfected with either mock or Tent2-, Tent4a- and Tent4b-, or Tent3a- and Tent3b-specific siRNAs, treated with mRNA-1273 for 24 h. Gene names are on the left. Red squares indicate genes silenced in a given condition. 5. a,b, ER stress marker expression in wild-type hMDMs after administration of mRNA-1273 for up to 72 h (a) and in hMDMs with siRNA-mediated TENT5A KD, 48 h after mRNA-1273 delivery (b). Data from 2 (a) and 3 (b) independent biological experiments. P values calculated for fold changes using two-sided Wilcoxon test, Benjamini–Hochberg correction, time points (without dividing by transcripts) considered as individual groups. c, Expression (DESeq2-normalized counts) of non-canonical poly(A)/poly(U) polymerases in mBMDMs treated with BNT162b2 for up to 72 h. Gene names on the left. d, Spike protein in cytoplasmic (C) or membrane (M) fractions of mBMDMs after cotransfection of mRNA-1273 and BNT162b2 (for up to 36 h). Western blot for spike; calreticulin, PERK and EIF2a served as a control of purity of fractions; β-actin and Ponceau S staining (bottom) served as loading controls. Representative picture from 2 independent biological experiments shown. f, DRS measured poly(A) length distribution for mRNA-1273 (left) and BNT162b2 (right) co-transfected (with lipofectamine reagent) to wild-type mBMDMs. For the left panel reads are divided into containing (blue) or lacking (green) mΨCmΨAG sequence, with median values and fractions of elongated or shortened tails shown. Data represent 2 independent biological experiments, g, Spike protein in the cytoplasmic (C) or membrane (M) fractions of mBMDMs after mRNA-1273 (green) or BNT162b2 (blue) administration (for up to 36 h). Western blot for spike; controls for fraction purity include SSR1 (TRAP1) and loading controls such as β-actin and α-tubulin. Bottom, quantification of spike protein in membrane fraction (3 biological and 1 technical replicate, normalized to TRAP1 levels), mean values ± s.e.m. f, h, Amount of spike protein (assessed with ELISA) in the culture medium, up to 72 h after administration of BNT162b2 to either wild-type (black) or Tent5aflox/flox/Tent5c−/− (red) mBMDMs. The average increase in spike protein amount per time point (top) or absolute measured concentrations (bottom) are shown. CleanCap used for the synthesis of reporters presented in Extended Data Figs. 8c and 9a–c is the same as in b, but with A nucleotide in the first position. c–e, Effective number of codons (ENC, c), codon adaptation index (CAI, d), and frequency of optimal codons (FOP, e) values for vaccine and reporter mRNAs CDSs used in this study, compared to average distributions for various groups of endogenous mouse transcripts. Shared legend for all the panels is presented on panel c. f, Sliding-window analysis of CAI of reporter mRNAs CDSs used in this study. g, Frequency of individual codons in mRNA-1273 and BNT162b2 CDSs. b, Illustration of IVT mRNA reporters synthesis. Templates for IVT reaction are prepared with the PCR, adding T7 promoter on one of the primers and desired poly(A) tail on the second primer. c,d, Integrity and purity of synthesized mRNAs, assessed with TapeStation 2200. Separate pictures are shown for mRNAs with cotranscriptionally added CleanCap (c) or with post-transcriptionally enzymatically (VCE + MTA) added cap (d). 5. a, Amount of OVA (assessed with ELISA) in the culture medium, 24 h after administration of OVA reporter mRNA to mBMDMs. Type of capping, UTRs, poly(A) tails and presence (or lack) of terminal pentamer are indicated. b,c, CleanCap-equipped OVA reporter poly(A) length distribution in mBMDMs transfected with mRNAs possessing mRNA-1273- (b) or BNT16b2-like (c) poly(A) tail (both with terminal mΨCmΨAG) for up to 48 h. The type of UTRs in the reporter is indicated at the top. d, VCE-capped OVA reporter poly(A) length distribution in mBMDMs transfected with mRNAs possessing mRNA-1273-like poly(A) tail (with terminal mΨCmΨAG) for up to 48 h. The type of UTRs in the reporter is indicated at the top. e, VCE-capped PfCSP reporter poly(A) length distribution in mBMDMs transfected with mRNAs possessing mRNA-1273-like poly(A) tail (with terminal mΨCmΨAG) for up to 24 h. The type of UTRs in the reporter is indicated at the top. f,g, VCE-capped ZikVE reporters poly(A) length distribution in mBMDMs transfected with mRNAs possessing mRNA-1273- (f) or BNT16b2-like (g) poly(A) tail (both with terminal mΨCmΨAG) for up to 24 h. The type of UTRs and poly(A) tails in the reporter are indicated at the top. h, Model of of TENT5A poly(A) polymerase's action, indicating ER-localization as a prerequisite for efficient mRNA adenylation by TENT5A. For b–g, poly(A) tail lengths measured with nanopore cDNA sequencing are shown with reads containing (blue) or lacking (grey) mΨCmΨAG sequence, with median values and fractions of elongated or shortened tails shown. c,d, Anti-spike IgG levels in serum of wild-type and Tent5a−/− mice 14 (c) or 21 (d) days after immunization with Nuvaxovid (WT n = 9, KO n = 6), e–g, Total IgG levels in serum of mRNA-1273 (e), BNT162b2 (f), or Nuvaxovid (g) immunized mice, wild-type and Tent5a−/−, measured 14 days after immunization. n = 5. j,k, Spike protein levels in serum of immunized mice, CD11c-CRE(−); Tent5a and CD11c-CRE(+);Tent5aflox/flox, measured 48 h (j) or 96 h (k) after immunization. For all panels values were obtained with the ELISA assay. P values were calculated using two-sided Wilcoxon test. Differential expression of genes in hMDMs upon mRNA-1273 delivery GO (Biological Process) terms for WT mBMDMs upon mRNA-1273 delivery. GO (Biological Process) terms for hMDMs upon mRNA-1273 delivery. List of antibodies used in this work Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Krawczyk, P.S., Mazur, M., Orzeł, W. et al. Re-adenylation by TENT5A enhances efficacy of SARS-CoV-2 mRNA vaccines. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-00688-x'>Clinical trials test the safety of stem-cell therapy for Parkinson's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-16 15:26:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Hideyuki Okano is in the Keio University Regenerative Medicine Research Center, Kawasaki, Kanagawa 210-0821, Japan. You can also search for this author in PubMed Google Scholar Pluripotent stem cells have special properties: they can proliferate almost indefinitely and have the capacity to differentiate into any cell that makes up an individual. Under the right conditions, both embryonic stem cells and induced pluripotent stem cells can be driven to differentiate into various types of cell, and efforts to produce differentiated cells from human pluripotent stem cells for clinical use have gathered pace over the past few decades. Writing in Nature, Sawamoto et al.1 and Tabar et al.2 report on clinical trials that harness pluripotent stem cells for the possible treatment of Parkinson's disease, the second most common neurodegenerative disease worldwide. Prices may be subject to local taxes which are calculated during checkout Read the paper: Phase I/II trial of iPS-cell-derived dopaminergic cells for Parkinson's disease Read the paper: Phase I trial of hES-cell-derived dopaminergic neurons for Parkinson's disease ‘Big leap' for Parkinson's treatment: symptoms improve in stem-cells trials Regulatory T cells aid stem-cell therapy for Parkinson's disease Gliomagenesis mimics an injury response orchestrated by neural crest-like cells Japan's big bet on stem-cell therapies might soon pay off with medical breakthroughs The Medical Faculty Mannheim of Heidelberg University offers the position of a   Full Professorship (W3) for Special Hematology (f/m/d)     to be f... Full Professorship (W3) for “Dermatology and Venereology” (f/m/d)     to be filled as of 01.10.2027. Job Title: Senior Publisher, Applied Science Journals  Location(s): Paris or London  Application Deadline: 1st May    About Springer Nature Group  ... The Hong Kong Univeristy of Science and Technology (Guangzhou) Read the paper: Phase I/II trial of iPS-cell-derived dopaminergic cells for Parkinson's disease Read the paper: Phase I trial of hES-cell-derived dopaminergic neurons for Parkinson's disease ‘Big leap' for Parkinson's treatment: symptoms improve in stem-cells trials Regulatory T cells aid stem-cell therapy for Parkinson's disease An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08845-y'>Phase I trial of hES cell-derived dopaminergic neurons for Parkinson's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-16 15:05:51
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Parkinson's disease is a progressive neurodegenerative condition with a considerable health and economic burden1. It is characterized by the loss of midbrain dopaminergic neurons and a diminished response to symptomatic medical or surgical therapy as the disease progresses2. Cell therapy aims to replenish lost dopaminergic neurons and their striatal projections by intrastriatal grafting. Here, we report the results of an open-label phase I clinical trial (NCT04802733) of an investigational cryopreserved, off-the-shelf dopaminergic neuron progenitor cell product (bemdaneprocel) derived from human embryonic stem (hES) cells and grafted bilaterally into the putamen of patients with Parkinson's disease. Twelve patients were enrolled sequentially in two cohorts—a low-dose (0.9 million cells, n = 5) and a high-dose (2.7 million cells, n = 7) cohort—and all of the participants received one year of immunosuppression. The trial achieved its primary objectives of safety and tolerability one year after transplantation, with no adverse events related to the cell product. At 18 months after grafting, putaminal 18Fluoro-DOPA positron emission tomography uptake increased, indicating graft survival. Secondary and exploratory clinical outcomes showed improvement or stability, including improvement in the Movement Disorder Society Unified Parkinson's Disease Rating Scale (MDS-UPDRS) Part III OFF scores by an average of 23 points in the high-dose cohort. There were no graft-induced dyskinesias. These data demonstrate safety and support future definitive clinical studies. Parkinson's disease (PD) is a debilitating neurodegenerative disorder characterized by a marked, progressive degeneration of dopaminergic neurons in the substantia nigra and their striatal projections. Motor features include bradykinesia with rigidity and/or resting tremor2,3,4; non-motor manifestations are wide-ranging and include autonomic, psychiatric, sleep and other symptoms that contribute to a decline in quality of life4,5. The global burden of PD is projected to exceed 14 million people by 20401. Medical therapies for PD focus on augmenting striatal dopamine receptor stimulation or acting on other transmitter pathways, such as acetylcholine6,7 or adenosine8. Surgical therapies include deep brain stimulation, which can improve motor symptoms9, and magnetic-resonance-guided focused ultrasound ablation, which can be effective for tremor and other motor features10,11. Experimental approaches under investigation include the use of anti-synuclein antibodies12,13, GLP1 receptor agonists14,15 and gene therapies16,17 aimed at neuroprotection, enhancing dopamine function or potential disease modification. Levodopa remains the cornerstone therapy but is associated with the gradual development of complications such as dyskinesias, a narrowing therapeutic window, off-target effects and exacerbation of non-motor symptoms2,18. While these therapies have an important role in symptom alleviation in the early stages, their efficacy inevitably wanes due to ongoing attrition of dopaminergic neurons, often accompanied by unwanted side effects such as dyskinesias. By delivering new midbrain neurons to the striatum, cell therapy aims to replace the degenerated dopaminergic projections and to achieve a sustained clinical improvement in motor symptoms. Open-label studies of fetal ventral midbrain tissue grafts in the striatum showed some promising results19,20,21,22,23,24 but were not replicated in double-blind, randomized, placebo-controlled trials25,26. The latter trials faced multiple challenges in tissue access, and used variable clinical design, surgical strategies and immunosuppression approaches. They also reported relatively high rates of graft-induced dyskinesia, possibly mediated by serotonergic neuron contaminants27. In other long-term follow-up studies, a subgroup of patients experienced sustained clinical benefit after grafting, including independence of dopaminergic (DA) medical therapy28, with reports demonstrating long-term graft survival and striatal DA innervation after brain autopsy at 14 years29 and 24 years after surgery30. The lessons learned from these studies suggest a potentially substantial clinical benefit for cell therapy in PD; however, these studies also highlighted the need for a scalable cell source of midbrain DA neurons, compatible with stringent quality-control assessments and lacking unwanted cellular contaminants. Over the following two decades, an intense effort ensued across the globe to identify a renewable source of human midbrain DA neurons from pluripotent stem cells (such as hES cells or human induced pluripotent stem (iPS) cells) that is suitable for clinical use. We have previously established a protocol for the derivation of midbrain DA neurons from hES cells31. PS cells are exposed to a carefully determined sequence and combination of patterning factors to undergo directed differentiation into midbrain DA neurons through a floor-plate intermediate stage. The resulting floor-plate-derived DA neurons show transcriptional, biochemical and physiological features of authentic midbrain DA neurons, and exhibit robust cell survival and function when grafted in mouse, rat and rhesus monkey models of Parkinson's disease31. Optogenetic studies in grafted rodents demonstrated that functional improvement is dependent on graft neuronal activity and dopamine release and that the transplanted DA neurons functionally integrate with host striatal neurons32. The protocol was subsequently adapted to GMP-compatible conditions and large-scale cell manufacturing33. Stringent release criteria confirmed midbrain DA neuron identity and the absence of remaining pluripotent stem cells as well as other concerning contaminants such as serotonergic neurons and choroid plexus cells. Cryopreservation was also developed, and four lots comprising around 10 billion cells in total were produced. Investigational New Device (IND)-enabling studies, including tumorigenicity, toxicity and biodistribution confirmed safety, as well as efficacy in reversing rotational behaviour in murine models of PD. Detailed preclinical data were published in 202133,34. The same cryopreserved cell product tested here, named bemdaneprocel (formerly MSK-DA01), was then used in the clinical trial. We conducted a first-in-human, multisite, open-label phase I trial to assess the safety and tolerability of bemdaneprocel in people with PD (NCT04802733). The full eligibility and exclusion criteria are listed in Supplementary Table 1; patients with cognitive impairment (defined by a Montreal Cognitive Assessment score of <26) or with dyskinesia (AIMS rating scale > 2) were excluded. The study included two cohorts: low-dose cohort A (0.9 million cells per putamen) and high-dose cohort B (2.7 million cells per putamen) (Fig. Enrolment occurred sequentially, first into the low-dose cohort, then into the high-dose cohort after a 30-day pause for evaluation by the data safety monitoring committee. There were two surgical sites (New York, n = 9) and Toronto (n = 3) and three screening and follow-up sites (New York, Irvine and Toronto), depending on patient location of residence. The cell doses for injection were selected on the basis of the desired number of surviving DA neurons, and the measured survival rate of DA neurons in our IND-enabling studies, which were performed with the same lot of cells nominated for clinical use. The high dose targets the estimated number of healthy DA neurons in the intact substantia nigra35 (300,000 DA neurons), and the lower dose targets the estimated minimal number of surviving grafted DA neurons required for clinical benefit36 (100,000 DA neurons). The patients underwent a series of clinical assessments and rating scales, as well as magnetic resonance imaging (MRI) and 18Fluoro-DOPA (18F-DOPA) positron emission tomography (PET) imaging at the baseline and at regular intervals after grafting (Fig. Summary of the study design. Ongoing follow-up is anticipated for a minimum of 5 years. The primary objective was to assess the safety and tolerability at 1 year after transplantation. Secondary objectives included the feasibility of stereotactic transplantation; survival of transplanted cells by 18F-DOPA uptake on PET imaging; motor effects measured using MDS-UPDRS Part III OFF scores37 and good ON times (without dyskinesia and with non-troublesome dyskinesia) based on adjusted PD diaries8; and continued safety and tolerability on the basis of the incidence of serious adverse events (SAEs) and adverse events (Supplementary Table 2). Exploratory outcomes included OFF time, ON times with troublesome dyskinesias, the Unified Dyskinesia Rating Scale (UDysRS), use of antiparkinsonian medications, the Non-Motor Symptom Scale (NMSS) and the patient-reported outcome on the basis of the PDQ-39 questionnaire. All safety data from all of the participants were used to conduct the formal analysis of the primary end point at 1 year after transplantation. Secondary and exploratory end points were also assessed at 1 year and 18 months after transplantation. No tests of statistical significance were performed due to the small number of participants. Vials containing cryopreserved cells were thawed under aseptic conditions and suspended in transplantation medium, and the live-cell concentration was adjusted to 100,000 ± 10,000 cells per µl (ref. Cells were loaded into a modified cannula (Smart Flow, Clearpoint Neuro), and administered stereotactically during a single surgical session under general anaesthesia. Cells were delivered into the post-commissural putamen bilaterally through a single burr hole on each side. In total, nine cell deposits were made in each putamen (three passes of the cannula; three deposits per pass). Surgery was performed using a frameless MRI-guided approach with intraoperative imaging at one site (New York) and a frame-based stereotactic approach at the other site (Toronto). To prevent graft rejection, a short-term immunosuppressive regimen similar to that used for solid organ transplantation38,39 was initiated perioperatively and continued for 1 year. The participants received basiliximab 20 mg intravenously intraoperatively and postoperatively on day 4; methylprednisolone 500 mg intravenously immediately before surgery then tapered to oral prednisone 5 mg daily and continued for 1 year; and tacrolimus taken orally beginning on the day after surgery (day 1) and then adjusted to a target trough blood level of 4 to 7 ng ml−1 for a period of 1 year. In total, 12 participants were sequentially enrolled into the low-dose (n = 5) or high-dose (n = 7) cohort (Extended Data Fig. The participants had a median age of 67.0 and 75% were male, with a median time since diagnosis of 9 years (Supplementary Table 3). Predefined safety criteria for the primary outcome were satisfied at 12 months of follow-up after transplantation (Table 1). There were two SAEs reported: one participant in the low-dose cohort was hospitalized overnight with COVID-19 approximately 2 months after the surgery and one participant in the high-dose cohort experienced a single seizure attributed to the surgical procedure within 24 h of surgery that prolonged hospitalization by 1 day. The participant received levetiracetam for 2 months with no recurrence of seizure during or after discontinuation of the anticonvulsant. There were no deaths, no SAEs related to transplanted cells or immunosuppression, and there were no tumours, abnormal tissue overgrowth or intracerebral haemorrhages detected with MRI, which showed the expected postoperative changes (Extended Data Fig. In addition to the two SAEs reported in the 12-month post-transplantation analysis, an SAE of gastrointestinal haemorrhage requiring hospitalization was reported approximately 15 months after surgery, 3 months after discontinuation of prednisone. This was assessed as unrelated to the study procedures and was subsequently resolved. Through 18 months after transplantation, there were 78 cumulative treatment-emergent adverse events (TEAEs), described in Extended Data Tables 1–5. Most were mild or moderate in severity except for one fall that was deemed to be unrelated to treatment. No TEAEs were related to bemdaneprocel. MRI imaging by 18 months did not show evidence of tumours or changes in the putaminal volume. Notably, there were no reports of adverse events or clinical indications of graft-induced dyskinesias (GIDs) to date. Transplant feasibility was achieved, with intraoperative delivery of the total number of all intended cell deposits in all of the participants. The two main secondary clinical readouts were MDS-UPDRS Part III OFF and changes in the number of waking hours in the ON state. A brief description of rating scales is provided in Supplementary Note 1. At 18 months after transplantation, the MDS-UPDRS Part III showed improvement in the OFF state with a decrease in mean (s.d.) Among the exploratory end points (Fig. 2e–h), MDS-UPDRS Part III ON scores also improved with a decrease in the mean (s.d.) MDS-UPDRS Part II scores increased by a mean (s.d.) PDQ-39 summary index scores increased (worsened) by 0.4 points (7.4) from the baseline to 25.4 (6.8) in the low-dose cohort and decreased (improved) by 4.2 points (8.2) from the baseline to 10.9 (8.7) in the high-dose cohort (Extended Data Table 6). Across the total study population, ON times with troublesome dyskinesias (Extended Data Table 7) and use of antiparkinsonian medications were similar to the baseline (Extended Data Table 8). UDysRS scores were similar to the baseline in both cohorts (Fig. The NMSS scores varied across domains, with total scores trending towards stability in the high-dose cohort (Extended Data Table 9). a,b, Secondary end points: individual MDS-UPDRS Part III OFF scores in the low-dose (a) and high-dose (b) cohorts at the baseline and at different timepoints after transplantation. c,d, Individual PD diary good ON time at the baseline and at different timepoints after transplantation in the low-dose (c) and high-dose (d) cohorts. ; exact values are given as mean (s.d.) e–h, Exploratory end points: individual scores on MDS-UPDRS Part III ON (e), Adjusted PD diary OFF time (f), MDS-UPDRS Part II (g) and UDysRS objective subscore (h) at the baseline and at different timepoints after transplantation in the low- and high-dose cohorts. The baseline value (0 months) is defined as the last recorded value before surgery. The double dagger symbol (‡) indicates the exception that, for MDS-UPDRS Part III OFF at 18 months after transplantation, n = 6. The total possible score for the MDS-UPDRS Part III scale is 132, for the MDS-UPDRS Part II scale is 52 and for the UDysRS objective scale is 44. We performed serial 18F-DOPA PET imaging studies to obtain evidence of DA graft survival, a key secondary imaging-based end point. Uptake datapoints are shown for each individual patient in both cohorts (Extended Data Fig. An image of the mean 18F-DOPA uptake signal is shown for the entire group (Fig. 4) across multiple slices through the striatum. These data support DA neuron survival through the 18-month post-transplantation timepoint, 6 months after discontinuation of immunosuppression. Images of the 18F-DOPA PET uptake signal at multiple sections through the striatum, presented as the mean uptake of all patients (n = 12 at all timepoints except at 18 months, for which n = 11). Images of 18F-DOPA uptake were produced by dividing each original PET image by the occipital count and then subtracting 1. Formula: image of 18F-DOPA uptake = (PET image/occipital count − 1). In this first-in-human phase I trial, bilateral putaminal transplantation of bemdaneprocel, a hES cell-derived DA neuron progenitor cell product, was generally well tolerated and achieved predefined safety criteria at 12 months as a primary end point. The treatment process, including surgery and immunosuppression, was generally well tolerated. This was a serious complication and was considered one of the major drawbacks of cell therapy at the time. The absence of GID in our study through 18 months is therefore particularly notable. This may relate to the lack of serotonergic neuron contaminants in bemdaneprocel, as preclinical studies confirmed the absence of serotonergic neurons in vitro and in vivo at 9 months after grafting34. Longer-term clinical and imaging follow-up is ongoing to monitor for the possibility of delayed onset GIDs or other adverse events. Although this phase I study is not designed nor powered for determining efficacy, there is nonetheless a possibility of improvement in the key secondary outcomes, including the MDS-UPDRS OFF motor scores, reflecting improved motor function without the benefit of anti-Parkinsonian medication. This scale was selected as an outcome measure as it is the most widely used PD scale; it is well validated with high inter-rater reliability, and is sensitive to clinical changes40. Our data show a mean change of 8.6 points in the low-dose cohort and 23 points in the high-dose cohort, consistent with a moderate and large ‘clinically important difference' in motor scores41, respectively. Coupled with the stabilization or possible improvement in other secondary and exploratory motor, non-motor and quality-of-life outcomes, the data suggest a promise for potential clinical benefit. However, these data need to be interpreted cautiously in view of the open-label study design, the small patient cohort, the relatively short observation period, the expected fluctuations in PD symptoms and the possibility of placebo effects, which can be substantial in studies of PD. Longer-term and larger studies are required to more rigorously evaluate the safety and efficacy of this cell therapy for PD. The clinical data also show a greater amplitude of changes in UPDRS motor scores and reduction in OFF times, and an earlier onset of improvement, in the high-dose group. Establishing an optimal cell dose involves multiple considerations. While the grafted DA neuron progenitors are expected to become post-mitotic and mature without obvious in vivo expansion, as confirmed in preclinical studies, the rates of progressive graft maturation and functional integration within the host tissue are unknown42. Fetal grafting studies showed evidence of Lewy body formation within the graft; however, such disease transmission occurred only in later stages (more than a decade) after transplantation and affected only a minority of grafted DA neurons43,44. Nevertheless, the precise kinetics of hES cell-derived midbrain DA neuron maturation in grafts and their relative disease vulnerability remains unclear, which complicates the dose-finding process. An encouraging finding in our study was the evidence of likely graft survival on the basis of 18F-DOPA PET, 6 months after cessation of immunosuppression. However, PET data should be interpreted cautiously owing to limitations in the resolution, signal-to-noise ratio and variability across and within patients as well as between the low- and high-dose cohorts. Furthermore, the correlation between 18F-DOPA PET uptake and clinical performance can be variable, and graft survival alone is not sufficient to predict long-term patient benefit. In past fetal allograft studies, the dosing, duration and type of immunosuppression was inconsistent.25,26,45. Data from one of the two randomized clinical trials suggested a potential decline in clinical function after discontinuation of immunosuppression25, yet autopsy reports in other studies demonstrated graft survival many years after surgery and discontinuation of immunosuppression. Our results suggest that the dosing and duration of immunosuppression were sufficient, well-tolerated and protected cell survival and function; longer-term follow-up data will be critical to confirm these findings. An alternative strategy to overcome graft rejection could lie in the use of an autologous iPS-cell-based DA-neuron-grafting approach. An initial single case report has been described to demonstrate feasibility46. However, consistently manufacturing and releasing an autologous product poses new challenges, and the possibility exists that the autologous nature of the cells may entail an increased risk of genetic susceptibility to develop disease. Thus, the feasibility of allografting without long-term immunosuppression, coupled with cryopreservation, offers an off-the-shelf approach33,47 that could facilitate the broader application of this therapy in the future. Finally, our findings should be interpreted carefully within the context of a small and unblinded safety and tolerability study that achieved its primary outcome and was not designed to evaluate efficacy based on clinical outcomes. Despite this, the possible improvement in PD symptoms at 18 months after transplantation may translate in future trials to an important finding. There are currently several ES-cell- or iPS-cell-based grafting studies that are ongoing or close to initiation in the US, Asia and Europe, highlighting considerable enthusiasm for cell therapy for PD. These studies will probably differ in many aspects, including autologous versus allografts, cell dosing, patient selection, immunosuppression, surgical strategies and other criteria; they will therefore provide valuable insights into future study design and the value of cell grafting in PD. Taken together, our data support moving towards larger definitive clinical studies. In brief, master banks of undifferentiated WA09 (H9) cells (passage 28) were obtained from the WiCell Research Institute. The cells were manufactured under cGMP conditions to establish working cell banks that were further certified by Waisman Biomanufacturing. The cells passed a battery of tests including cell authentication by short tandem repeat (STR) analysis, karyotype and marker expression, as well as testing for adventitious viruses and mycoplasma. The full protocol is described in a previous publication34. cGMP manufacturing of MSK-DA01 cells, which were differentiated from hES cells (WA09 at passage 33), was performed at the Cell Therapy and Cell Engineering Facility (CTCEF) at MSKCC. WA09 cells were thawed and expanded on Geltrex coated flasks/dishes in Essential 8 basal medium with supplement for 10–14 days. Cells were split by Dispase (Stem cell Technologies) every 3–5 days at a ratio usually between 1:4 and 1:6. For MSK-DA01 differentiation, single cells WA09 were washed and plated on Geltrex at 400,000 cells per cm2 in Neurobasal medium with N2 and B27 (without vitamin A) containing 2 mM l-glutamine, 10 µM SB431542, 250 nM LDN193189 (LDN), 0.7 µM CHIR99021 and 500 ng ml−1 SHH with 10 µM Y-27632. The medium was replaced daily thereafter without adding Y-27632. On day 4, 7.5 µM CHIR99021 was added. The same medium was replaced on day 6. On day 7 SB, LDN and SHH were withdrawn from the medium. The same medium was changed on day 9. On day 10, Neurobasal medium (with B27) with 2 mM L-glutamine, 20 ng ml−1 BDNF, 20 ng ml−1 GDNF, 200 µM ascorbic acid, 500 µM dibutyryl-cAMP, 1 ng ml−1 TGFβ3 and 3 µM CHIR99021 was added. On day 11, cells were dissociated to single cells with Accutase for 30–40 min and replated on plates coated with poly-ornithine (15 µg ml−1, Sigma Aldrich), fibronectin (1 µg ml−1, Akron Biotech) and laminin (2 µg ml−1, Trevigen) at 800,000 cells per cm2 using the same medium as on day 10. On day 12, the medium was switched to include 10 µM DAPT and CHIR99021 was withdrawn. Complete medium changes were performed daily until collection at day 16. At day 16 of differentiation, MSK-DA01 cells were dissociated with Accutase for 30–40 min and filtered through a 40 µm cell strainer. The cell pellets were resuspended at a cell density of 8 million cells per ml of STEM-CELLBANKER and placed in a controlled-rate freezer (Thermo Fisher Scientific) for cryopreservation. Cryopreserved vials were stored in the secured GMP Facility freezer and monitored 24/7 by a Datatron system. This trial included two cohorts: a low-dose (cohort A; 0.9 million cells per putamen) and a high-dose (cohort B; 2.7 million cells per putamen) cohort (Extended Data Fig. Enrolment occurred sequentially, first into the low-dose cohort, then into the high-dose cohort after a 30-day pause for evaluation by the data safety monitoring committee. The eligible participants (Supplementary Table 1) were aged at least 50 to 78 years (Canada) or at least 60 to 78 years (United States) with a confirmed diagnosis of PD for 3 to 20 years before screening and Hoehn–Yahr ON score of 0 to 3 (Canada) or ON score of 0 to 2 and OFF score of 3 to 4 (United States). The participants attended screening, baseline and preoperative visits before surgery. After transplantation, the participants underwent a postoperative evaluation on day 10. A battery of questionnaires and clinical rating scales were administered during site visits throughout the study, with a total follow-up of 5 years after transplantation; the participants completed PD diaries at home before each visit, and adverse events were assessed throughout the trial. Safety success criteria were defined as: two or fewer participants in either cohort developing two or more SAEs related to surgery, the presence of transplanted cells or immunosuppression; two or fewer participants in either cohort developing a tumour or abnormal tissue overgrowth related to the presence of transplanted cells; two or fewer participants in either cohort developing an intracerebral haemorrhage that was deemed to be life threatening; and one or zero deaths in either cohort. Secondary objectives (Supplementary Table 2) included the feasibility of stereotactic transplantation; survival of transplanted cells by 18F-DOPA uptake on PET imaging; motor effects measured by Movement Disorder Society Unified Parkinson's Disease Rating Scale (MDS-UPDRS) Part III OFF scores and adjusted PD diary good ON times (the sum of ON time without dyskinesia and ON time with non-troublesome dyskinesia); and safety and tolerability by the incidence of SAEs and adverse events. 18F-DOPA PET studies and analysis were performed at a single imaging site by investigators blinded to clinical and dose information. The participants underwent three-dimensional PET imaging at the baseline, and at 12 and 18 months after transplantation. Images were acquired on a high-resolution, high-sensitivity PET tomograph (GE Discovery IQ-5 ring PET/CT Scanner, General Electric Medical Systems; parameters: full-width-at-half-maximum = 5 mm, axial field of view = 26 cm, slice thickness = 3.3 mm, 79 slices). Acquisition began approximately 80 to 100 min after a 5.0 mCi intravenous injection of 18F-DOPA. The participants paused antiparkinsonian medication 12 h before scanning and were given 150 mg of carbidopa orally within 120 min before 18F-DOPA injection in accordance with FDA prescribing information. 18F-FDOPA PET image reconstruction used the VUEPointHD ViP (VPHD) method, a 3D iterative OSEM (ordered subset expectation maximization) algorithm with 4 iterations and 12 subsets after corrections for attenuation and scatter. The image resolution was approximately 5 mm in all directions. We created a priori group mean of striatal to occipital ratio (SOR-1) images computed for each timepoint, as well as a priori VOI-based analyses. 3) were derived to provide an anatomical visualization of potential change in group mean18F-DOPA uptake signal over the striatum across the timepoints. The VOI-based analyses were performed to provide group × time quantification of 18F-DOPA uptake signal within prespecified regions within the striatum (that is, posterior putamen, surgical target region of implanted cells; caudate, control region, no implanted cells). 18F-DOPA PET images for each participant were realigned across individual time frames and then registered together to the high-resolution T1 structural MRI scan (see below) using SPM12 software (https://www.fil.ion.ucl.ac.uk/spm). The registered PET images were converted into volumetric maps of specific 18F-DOPA uptake defined by (image/occipital counts − 1; SOR-1), with the occipital cortex used as a reference. The SOR-1 images of 18F-DOPA uptake before and after treatment were spatially normalized into the standard Montreal Neurological Institute anatomic space (https://mcin.ca/research/neuroimaging-methods/atlases). 18F-DOPA PET images for each participant were realigned across individual time frames and then registered together to the high-resolution T1 structural MRI scan (see below) in native space using SPM12. 18F-DOPA PET images were maintained in the native space without performing spatial normalization. Anatomical structures of the caudate nucleus and putamen were segmented in the MRI scan using an automated processing routine implemented with FSL software48 (https://fsl.fmrib.ox.ac.uk/fsl/docs/#/install/index). 18F-DOPA counts were computed automatically in each striatal VOI, including the caudate nucleus, anterior and posterior putamen and occipital cortex (16 mm diameter sphere in each hemisphere). The 18F-DOPA uptake was then calculated (regional 18F-DOPA counts/occipital counts − 1) using FSL with the occipital cortex used as a reference. Individual datapoints for the anterior putamen, posterior putamen and caudate are shown for both the low- and high-dose cohorts (Extended Data Fig. This method had been previously validated49,50 and used successfully in the PET imaging studies of fetal cell transplantation25,42,51,52. MRI studies of the brain (1.5 Tesla and 3 Tesla) were performed using standardized acquisition protocols at screening, preoperatively for surgical planning, intraoperatively, postoperatively, and 1, 3, 6, 12 and 18 months after transplantation. Between 3 May 2021 and 30 March 2022, 17 participants were assessed for eligibility; five were excluded due to screening failure. The remaining 12 participants were sequentially enrolled into the low-dose (n = 5) or high-dose (n = 7) cohort (Extended Data Fig. The participants had a median age of 67.0 years (quartile 1 (Q1), Q3: 64.5, 70.0); 67% were white and 75% were male (Supplementary Table 3). The median (Q1, Q3) time since diagnosis was 9.0 (5.9, 11.5) years. At the baseline, all of the participants had a Hoehn–Yahr ON score of 2; 11 participants had a Hoehn–Yahr OFF score of 3 and one participant (low-dose cohort; Canada) had a score of 2. TEAEs most frequently experienced by participants (Extended Data Table 6) were in the nervous system disorders (11 TEAEs in eight participants; Extended Data Table 7) and infections and infestations (10 TEAEs in seven participants; Extended Data Table 8) system/organ classes. Of the infections, one TEAE each of oral herpes and urinary tract infection was considered to be related to immunosuppression. No TEAEs were related to bemdaneprocel. Aside from two severe TEAEs (one fall deemed unrelated to treatment and the aforementioned SAE of gastrointestinal haemorrhage), all others were mild or moderate in severity (Extended Data Table 4). All safety data from all of the participants were used to conduct the formal analysis of the primary end point at 1 year after transplantation. Secondary and exploratory end points were also assessed at 1 year and 18 months after transplantation. Inferential statistical testing was not performed due to the small number of participants. Descriptive statistics (mean (standard deviation (s.d.)) or percentage (counts)) were tabulated by cohort. The trial was conducted in accordance with the protocol and all applicable local government laws, regulations and guidance, including policies with foundations in the World Medical Association Declaration of Helsinki, the International Council for Harmonisation of Technical Requirements for Pharmaceuticals for Human Use. An institutional review board at Memorial Sloan Kettering Cancer Center and Weill Cornell Medicine reviewed and approved the clinical protocol (submitted with the Article) and the patient consent form. A data safety monitoring committee and the institutional review boards monitored trial progression and data handling. The Tri-institutional Human Embryonic Stem Cell Research Oversight committee (ESCRO) at MSKCC and WCM approved the study and the consent on 11 October 2019. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data supporting the findings of this study, including de-identified participant-level data, are provided within the Article and its Supplementary Information. The study protocol and statistical analysis plan are also included in Supplementary Note 2. For further information on exploratory end points that are not described in this Article, investigators who provide a methodologically sound proposal may submit a request to medinfo@bluerocktx.com. Requests will be assessed and responded to within 30 days of receipt. Source data are provided with this paper. Dorsey, E. R. & Bloem, B. R. The Parkinson pandemic—a call to action. Morris, H. R., Spillantini, M. G., Sue, C. M. & Williams-Gray, C. H. The pathogenesis of Parkinson's disease. & Okun, M. S. Diagnosis and treatment of Parkinson disease: a review. Poewe, W. et al. Parkinson disease. Heimrich, K. G., Schönenberg, A., Santos-García, D., Mir, P., Coppadis Study Group & Prell, T. The impact of nonmotor symptoms on health-related quality of life in Parkinson's disease: a network analysis approach. Chung, K. A., Lobb, B. M., Nutt, J. G. & Horak, F. B. Effects of a central cholinesterase inhibitor on reducing falls in Parkinson disease. Henderson, E. J. et al. Rivastigmine for gait stability in patients with Parkinson's disease (ReSPonD): a randomised, double-blind, placebo-controlled, phase 2 trial. Hauser, R. A. et al. A home diary to assess functional status in patients with Parkinson's disease with motor fluctuations and dyskinesia. Kremer, N. I. et al. STN-DBS electrode placement accuracy and motor improvement in Parkinson's disease: systematic review and individual patient meta-analysis. Krishna, V. et al. Trial of globus pallidus focused ultrasound ablation in Parkinson's disease. Bond, A. E. et al. Safety and efficacy of focused ultrasound thalamotomy for patients with medication-refractory, tremor-dominant Parkinson disease: a randomized clinical trial. Lang, A. E. et al. Trial of cinpanemab in early Parkinson's disease. Pagano, G. et al. Trial of prasinezumab in early-stage Parkinson's disease. Standaert, D. G. GLP-1, Parkinson's disease, and neuroprotection. Meissner, W. G. et al. Trial of lixisenatide in early Parkinson's disease. Abeliovich, A., Hefti, F. & Sevigny, J. Gene therapy for Parkinson's disease associated with GBA1 mutations. Van Laar, A. D. et al. An update on gene therapy approaches for Parkinson's disease: restoration of dopaminergic function. de Bie, R. M. A., Clarke, C. E., Espay, A. J., Fox, S. H. & Lang, A. E. Initiation of pharmacological therapy in Parkinson's disease: when, why, and how. Wenning, G. K. et al. Short- and long-term survival and function of unilateral intrastriatal dopaminergic grafts in Parkinson's disease. Survival of implanted fetal dopamine cells and neurologic improvement 12 to 46 months after transplantation for Parkinson's disease. Bilateral caudate and putamen grafts of embryonic mesencephalic tissue treated with lazaroids in Parkinson's disease. Hagell, P. et al. Sequential bilateral transplantation in Parkinson's disease: effects of the second graft. Evidence for long-term survival and function of dopaminergic grafts in progressive Parkinson's disease. Bilateral motor improvement and alteration of l-dopa effect in two patients with Parkinson's disease following intrastriatal transplantation of foetal ventral mesencephalon. Olanow, C. W. et al. A double-blind controlled trial of bilateral fetal nigral transplantation in Parkinson's disease. Freed, C. R. et al. Transplantation of embryonic dopamine neurons for severe Parkinson's disease. Politis, M. et al. Serotonin neuron loss and nonmotor symptoms continue in Parkinson's patients treated with dopamine grafts. Long-term clinical outcome of fetal cell transplantation for Parkinson disease: two case reports. Long-term health of dopaminergic neuron transplants in Parkinson's disease patients. Li, W. et al. Extensive graft-derived dopaminergic innervation is maintained 24 years after transplantation in the degenerating parkinsonian brain. Kriks, S. et al. Dopamine neurons derived from human ES cells efficiently engraft in animal models of Parkinson's disease. A. et al. Optogenetics enables functional analysis of human embryonic stem cell-derived grafts in a Parkinson's disease model. Kim, T. W. et al. Biphasic activation of WNT signaling facilitates the derivation of midbrain dopamine neurons from hESCs for translational use. Piao, J. et al. Preclinical efficacy and safety of a human embryonic stem cell-derived midbrain dopamine progenitor product, MSK-DA01. Rudow, G. et al. Morphometry of the human substantia nigra in ageing and Parkinson's disease. Hagell, P. & Brundin, P. Cell survival and clinical outcome following intrastriatal transplantation in Parkinson disease. Goetz, C. G. et al. Movement Disorder Society-sponsored revision of the unified Parkinson's disease rating scale (MDS-UPDRS): process, format, and clinimetric testing plan. Consensus recommendations for use of maintenance immunosuppression in solid organ transplantation: endorsed by the American College of Clinical Pharmacy, American Society of Transplantation, and International Society for Heart and Lung Transplantation: an executive summary. Practical recommendations for long-term management of modifiable risks in kidney and liver transplant recipients: a guidance report and clinical checklist by the Consensus on Managing Modifiable Risk in Transplantation (COMMIT) Group. Martinez–Martin, P., Rodriguez-Blazquez, C. & Forjaz, M. J. in Neuroscience and Biobehavioral Psychology 8–16 (Elsevier, 2017). Ma, Y. et al. Dopamine cell implantation in Parkinson's disease: long-term clinical and 18F-FDOPA PET outcomes. Li, J. Y. et al. Lewy bodies in grafted neurons in subjects with Parkinson's disease suggest host-to-graft disease propagation. Kordower, J. H., Chu, Y., Hauser, R. A., Freeman, T. B. Designing stem-cell-based dopamine cell replacement trials for Parkinson's disease. Schweitzer, J. S. et al. Personalized iPSC-derived dopamine progenitor cells for Parkinson's disease. Kirkeby, A. et al. Preclinical quality, safety, and efficacy of a human embryonic stem cell-derived product for the treatment of Parkinson's disease, STEM-PD. Patenaude, B., Smith, S. M., Kennedy, D. N. & Jenkinson, M. A Bayesian model of shape and appearance for subcortical brain segmentation. Comparative analysis of striatal FDOPA uptake in Parkinson's disease: ratio method versus graphical approach. Simple ratio analysis of 18F-fluorodopa uptake in striatal subregions separates patients with early Parkinson disease from healthy controls. Nakamura, T. et al. Blinded positron emission tomography study of dopamine cell implantation for Parkinson's disease. Ma, Y. et al. Dyskinesia after fetal cell transplantation for parkinsonism: a PET study. We thank the study participants, study investigators, study staff, the staff at the Cell Therapy and Cell Engineering Facility (Memorial Sloan Kettering Cancer Center) and the members of the Feinstein Institutes for Medical Research (Northwell Health) for their contributions to this research. This study was sponsored by BlueRock Therapeutics. The sponsor (BlueRock Therapeutics) collaborated with investigators on trial design and data collection, analysis and interpretation. The initial work to develop bemdaneprocel (MSK-DA01) was supported by NYSTEM grant C028503 and MSK Core Grant P30 CA008748. Under the direction of the authors, B. M. Hiller of Peloton Advantage, an OPEN Health company, provided medical writing support, which was funded by BlueRock Therapeutics. These authors contributed equally: L. Studer, C. Henchcliffe Department of Neurosurgery, Memorial Sloan Kettering Cancer Center, New York, NY, USA Center for Stem Cell Biology, Sloan Kettering Institute, New York, NY, USA Cancer Biology and Genetics Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA Department of Neurology, Weill Cornell Medicine, New York, NY, USA Krembil Brain Institute, Toronto, Ontario, Canada Edmond J. Safra Program in Parkinson's Disease, Morton and Gloria Shulman Movement Disorders Clinic, Toronto Western Hospital, UHN, Toronto, Ontario, Canada Center for Neurosciences, Institute of Molecular Medicine, Feinstein Institutes for Medical Research, Manhasset, NY, USA Department of Molecular Medicine, Donald and Barbara Zucker School of Medicine at Hofstra/Northwell, Hempstead, NY, USA BlueRock Therapeutics, Cambridge, MA, USA Developmental Biology Program, Memorial Sloan Kettering Cancer Center, New York, NY, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar served as a neurosurgeon investigator, performed surgery, contributed to preclinical data, clinical trial design and manuscript writing. served as principal investigator and neurologist on the trial. performed transplantation surgery as investigators on the trial. contributed to the preclinical data, study design and manuscript preparation. was a neurologist investigator on the trial and contributed to the trial design. is a scientific advisor and receives research support from BlueRock Therapeutics. has done consulting work for Novo Nordisk, BlueRock Therapeutics, Neurocrine and Neuroderm; and has also received clinical trial support from BlueRock Therapeutics, Prevail Therapeutics, Neuroderm, Sun Pharma, Meira GTX, Bukwang, Insightec, Biogen, Genentech, Cerevance, UCB and National Institutes of Health (NIH). has done consulting work for Medtronic, Abbott, Boston Scientific and Insightec; and is a scientific officer of Functional Neuromodulation. has stock ownership in Inbrain Pharma and has received payments as consultant and/or speaker from AbbVie, Abbott, Boston Scientific, CereGate, Dompé Farmaceutici, Inbrain Neuroelectronics, Ipsen, Medtronic, Iota, Syneos Health, Merz, Sunovion, Paladin Labs and UCB; and has received research support from AbbVie, BlueRock Therapeutics, Boston Scientific, Medtronic, Praxis and ES, and receives royalties from Springer. has received consultant fees, support and/or speaker honoraria from Novo Nordisk, Abbott, Boston Scientific and Medtronic. is currently an employee of Dompé. is a scientific advisor and receives research support from BlueRock Therapeutics and is a scientific co-founder of DaCapo Brainscience. has received honoraria for consulting from Abbvie, AskBio and Certara; stock options for scientific advisory board participation from Axent Biosciences; honoraria for scientific advisory board participation from Bayer, Canary Health Technologies and ProJenX; honoraria for DSMB meetings from MeiraGTx; research support from BlueRock Therapeutics and Weston Brain Institute; and honoraria for presentations from the American Academy of Neurology, ASGCT and the Parkinson Study Group. The other authors declare no competing interests. Nature thanks William Jagust, Hideyuki Okano, Mark Tuszynski and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Diagram detailing the disposition of all participants. The 12 participants who passed screening were enrolled sequentially into the low-dose cohort (N = 5) or high-dose cohort (N = 7). No evidence of intracerebral haemorrhage, mass, lesion, and/or cellular overgrowth. Transplanted cells were indiscernible; needle tracks exhibit gliosis as expected and are best seen in the lower panels. Volume of interest (VOI)-based analysis of 18F-DOPA PET uptake signal in the anterior putamen (a), posterior putamen (b) and the caudate (c) at baseline, 12 months and 18 months, for individual patients in the low and high dose cohorts. Data is obtained via the VOI method as described in the Methods section. N = 12 for all timepoints with 2 exceptions: one patient missed the 18-month timepoint scan, and another patient had their 18-month timepoint scan performed at week 91 (at ~21 months) post grafting (data included). Bold lines in each panel represent the mean ± SD. Images of 18F-DOPA PET uptake signal at multiple sections through the striatum, presented as mean uptake at baseline, 12 months, and 18 months in the low dose cohort (top) and high dose cohort (bottom). Images were produced by dividing each original PET image by occipital count and then subtracting 1. N = 12 for all timepoints with 2 exceptions: one patient missed the 18-month timepoint scan, and another patient had their 18-month timepoint scan performed at week 91 (at ~21 months) post grafting (data included). Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Tabar, V., Sarva, H., Lozano, A.M. et al. Phase I trial of hES cell-derived dopaminergic neurons for Parkinson's disease. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/alan-turings-lost-work-could-reveal-how-tigers-got-their-stripes/'>Alan Turing's Lost Work Could Reveal How Tigers Got Their Stripes</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-16 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Alan Turing's Lost Work Could Reveal How Tigers Got Their Stripes The world-famous mathematician Alan Turing found a possible mechanism for iconic animal patterns thanks to differential equations Today mathematician Alan Turing is world-famous because he helped the Allies achieve victory against the Axis powers by deciphering an encryption that was considered unbreakable. Turing's cryptographic work remained under wraps until the 1970s, however, so his incredible achievements only became known after his death. During his lifetime, Turing was known among certain experts. He developed the mathematical model of a computer and explained which mathematical quantities it could calculate—and which tasks would exceed even the most sophisticated algorithms. He is also well known for a test that he developed, later named after him, that assesses how “human” artificial intelligence appears to be. For instance, if people cannot tell whether they are chatting to a real person or an AI, then the machine has passed the Turing test. The list of Turing's scientific contributions is long. But one area of his research is rarely mentioned: his work on mathematical biology that dealt with the formation of patterns. He was interested in the question of how animals develop their impressive stripes and spots, and he was convinced that there must be a mechanism by which pigments in skin cells arrange themselves into these patterns. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. After all, what could the pattern of a tiger's skin have to do with abstract mathematics? Until then, I had assumed that some complex biochemical processes led to the tiger's impressive patterns of dots and stripes—not something that could be represented by a tensor (a kind of high-dimensional table). According to his mother, even as a child, he was a dreamer who marveled at the natural world around him. And so Turing found a very simple mechanism that could explain nature's patterns. To understand Turing's ideas, you first need a little biological background. From there, they migrate through the entire body. Although research into these cells was lacking in Turing's time, he recognized that there was a developmental process that formed skin patterns, and he wanted to find out how this occurred. It was impossible to model all the interacting molecules of an animal embryo. Moreover, Turing was not an expert in biochemistry. Therefore, as is usual for mathematicians, he started with a very simple model. Such an interplay is often found in ecology. For example, the black morphogens can be thought of as hares, which reproduce rapidly and thus attract foxes (the orange morphogens). This complex interaction can lead to a variety of situations. Translating that example to morphogens, one can imagine how a dotlike pattern similar to a cheetah's fur could reflect that one morphogen has limited the spread of another. Turing did not use the fox-and-hare visualization when he described how morphogens could move through an embryo's cells, but he did take into account the phenomenon of diffusion. If a cell harbors many black morphogens, for example, and a neighboring cell harbors few of them, then the molecules strive to move such that they are distributed as evenly as possible. All of these processes can be described by so-called differential equations. These equations contain one or more derivatives and can indicate how the number of morphogens per cell changes. In doing so, he was able to adjust several parameters. To put this in terms of our animal analogy: How many foxes and hares are there at the beginning? When Turing investigated this problem, he did not have a powerful computer at his disposal and had to carry out many of the calculations by hand. In some cases, there were stripes; in others, he found dots or sometimes spots similar to those on cows. (If you would also like to experiment with the Turing mechanism but don't feel like doing lengthy calculations, here is a simulator.) Many years later British mathematical biologist James Murray applied Turing's idea to big cats. Unfortunately, this work by Turing attracted little attention during his lifetime. It took about 20 years for biologists to rediscover Turing's work, inspiring a new generation to test whether the Turing mechanism really does occur in nature. To prove Turing's hypothesis, the corresponding morphogens must be identified in animals. Although this is not easy, a few cases are now known. For example, scientists identified two proteins in mice that produce the striped structure of their palate, and the arrangement of the animals' hair follicles appears to be influenced by two other proteins. These examples show that Turing's ideas are not limited to color patterns but also apply to other structures. To date, morphogens have been most clearly detected in mice, which are much easier to study in the lab. But hopefully a future scientific strategy can be found to prove Turing's mechanism beyond a doubt in big cats as well. This article originally appeared in Spektrum der Wissenschaft and was reproduced with permission. Manon Bischoff is a theoretical physicist and an editor at Spektrum der Wissenschaft, the German-language sister publication of Scientific American. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41477-025-01979-x'>Re-analysis of mobile mRNA datasets raises questions about the extent of long-distance mRNA communication</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-16 10:44:16
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Short-read RNA-seq studies of grafted plants have led to the proposal that thousands of messenger RNAs (mRNAs) move over long distances between plant tissues1,2,3,4,5,6,7, potentially acting as signals8,9,10,11,12. Transport of mRNAs between cells and tissues has been shown to play a role in several physiological and developmental processes in plants, such as tuberization13, leaf development14 and meristem maintenance15; yet for most mobile mRNAs, the biological relevance of transport remains to be determined16,17,18,19. Here we perform a meta-analysis of existing mobile mRNA datasets and examine the associated bioinformatic pipelines. Taking technological noise, biological variation, potential contamination and incomplete genome assemblies into account, we find that a high percentage of currently annotated graft-mobile transcripts are left without statistical support from available RNA-seq data. This meta-analysis challenges the findings of previous studies and current views on mRNA communication. A key step in mobile mRNA studies is the assignment of RNA-seq reads to different genotypes. One way of identifying the genotype is based on single nucleotide polymorphisms (SNPs) (Fig. Published criteria are: ≥1 RNA-seq read covering at least two SNPs3, ≥2 reads3, ≥3 reads20 or >3 reads2 covering a single SNP. When these criteria are met, the corresponding transcript is defined as mobile. Transcripts that move root to shoot can be identified by analysing mRNAs in shoot tissue. Natural grafts, such as those established between the parasitic dodder plant and its host plants, can be used in place of artificial grafts. A key challenge in all such approaches is how to assign transcripts to each genotype; methods for doing so are based on (1) SNP identification or on (2) the alignment to different reference genomes. For grafts from the same species, or similar genotypes, SNPs can be used to distinguish between genotypes and thus identify the source genotype of each transcript (1). For grafts between different species, mapping (2) each RNA-seq read to the genome assemblies can be an effective method for determining which transcripts are specific to one species. As previously reported21, criteria based on absolute numbers of reads, such as those above, exhibit a read-depth dependency (Extended Data Fig. This is a consequence of sequencing noise. Illumina sequencing machines produce base-calling errors at a rate of ~0.1–1% per base22,23. Sequencing providers often provide a quality assurance, for instance, that 85% of the reads have a Phred quality score of at least Q30 (that is, a base-calling error of less than 10−3 = 0.1%). However, base-calling inaccuracies are not the only source of error. Before sequencing, reverse transcriptases can introduce base changes with an error rate of ~0.001–0.01%; the reverse transcription reaction error may exhibit a nucleotide bias, for instance, ‘G' to ‘A'24,25, and a range of other artefacts26. The average error rate of next-generation sequencing technologies has been estimated as 0.24 ± 0.06% per base22,27, with RNA-seq errors tending to be higher27. We therefore investigated whether noise in RNA-seq may influence the identification of mobile mRNAs. Figure 2a lists how many reported mobile mRNAs have numbers of reads with SNP occurrences that are consistent with an assumed error rate21,28. As an example, for an accuracy of SNP calling of 99.97% (that is, 0.03% sequencing noise, Phred score Q35, and an error probability for the alternative allele of ~0.01%), the evidence for 1,086 out of 2,006 (54%) and 384 out of 1,130 (34%) previously identified mobile mRNAs2,3 is in line with what would be expected from sequencing noise (Fig. a, Total numbers of reported mobile mRNAs in Arabidopsis thaliana2 and Vitis girdiana3 that can be explained by expected sequencing noise. Two values for the probability of the sequenced nucleotide at a SNP position being assigned to an alternative allele are given: 0.01% and 0.1%. b,c, The distributions of nucleotides at SNP and other positions (‘non-SNP') can be informative for evaluating the evidence for the alternative allele. b, Histograms of the ratio of the number reads that match the alternative allele, n, over the number of reads of local and foreign reads, N, for each SNP position in the mobile population on examples from Arabidopsis2. Several SNPs have reads that match the alternative allele (n/N > 0). c, Histograms of the ratio of the number reads that match the second most frequent nucleotide, m, over the sum of the number of reads over the most frequent and second most frequent nucleotide, M, for neighbouring positions to SNPs. An exact two-sample Kolmogorov–Smirnov test does not find significant differences in the distributions over SNPs and other positions (D = 0.089302, P = 0.3575). A Welch two-sample t-test (P = 0.6421), Wilcox rank sum test (P = 0.6388) or an exact two-sample Kolmogorov–Smirnov test (P = 0.4065) does not support the values for SNPs being higher than other positions. d,e, Of the 2,006 previously identified mobile mRNAs2, 953 unique mobile mRNAs were found in only 1 replicate in different organs of an adult Ped-0:Col-0 (root:shoot) graft. Such high numbers are not consistent with our hypothesis of sequencing noise and biological variation. Investigating the reciprocal relationship between root alleles that were detected in the rosette (d) and vice versa (e), in the root (1,373 mRNAs/867 unique) and rosette (577 mRNAs/151 unique) samples, identifies a strong linear correlation (P = 2 × 10−16) between expression in the source tissue and potential mobility. However, these transcripts have low read numbers only in the ‘source' tissue, whereas they have high read numbers in the sampled tissue and reads over SNPs that are consistent with sequencing errors. These plots thus show two effects: sequencing noise + either non-selective transport across the whole transcriptome or contamination. One way to increase the accuracy of detecting foreign transcripts is to consider multiple SNPs per read. If SNPs are located closely together, then a single RNA-seq read may cover more than one SNP. Accounting for co-occurring SNPs on the same read leads to the multiplication of their probabilities, resulting in higher accuracy (less likely to occur by chance), less pronounced read-depth dependence than single SNP criteria (Extended Data Fig. 1) and greater confidence in these reads being from a foreign genotype. We therefore examined reads over co-occurring SNPs (Extended Data Fig. In the Arabidopsis homograft datasets2, we found a total of 1,753,179 reads covering more than 1 SNP in the root and 1,977,539 in the shoot of Col-0; of these 1,675 (0.10%) and 1,797 (0.091%), respectively, had reads supporting the alternative allele for at least 1 but not all SNPs. These inconsistent calls are in line with the notion that sequencing noise may confound the identification of mobile mRNAs. Investigating these co-occurring SNPs revealed another confounding factor in the identification of mobile mRNA; several loci showed apparent heterozygosity in the Ped-0 ecotype (Extended Data Fig. Such apparent heterozygosity could be caused by a lack of introgression or gene copy-number variation; it has been estimated that 10% of the annotated genes in Arabidopsis have copy-number variation29,30. Differences in gene copy numbers can lead to reads not mapping correctly, which gives rise to pseudo-SNPs and pseudo-heterozygosity29,30. Of the 2,570 genes assigned as pseudo-heterozygous30, we found 188 mobile transcripts2 (Extended Data Figs. We identified 19 transcripts in the Ped-0 samples that are likely caused by mismapping; interestingly, these include transcripts that frequently fulfill the criteria for being classified as mobile (Supplementary Table 3). Thus, in addition to technological noise, there are also biological causes that could be falsely interpreted as SNPs of an alternative allele. As a consequence, it becomes important to not rely solely on Phred scores for estimating errors in SNP assignments. We next sought to estimate this background noise level, that is, the frequency for finding the alternative allele when the alternative allele is not actually present. This value can be estimated from available Arabidopsis homograft data2. For Arabidopsis homograft datasets (ecotypes Col-0 and Ped-0), these background noise levels were 0.084% (Col-0:Col-0 root), 0.082% (Col-0:Col-0 shoot), 0.68% (Ped-0:Ped-0 root) and 0.51% (Ped-0:Ped-0 shoot). The higher background error rate in Ped-0 is consistent with more Col-0 transcripts being identified as mobile in sampled Ped-0 tissue2. For an average background error rate of 0.34%, we find that over 1,455 out of 2,006 (>73%) and over 945 out of 1,130 (>84%) of annotated mobile mRNAs would not be distinguishable from expected errors (Fig. Consistent with this, poor overlap between experiments has been noted18,31, orthologues in closely related species exhibit conflicting mobility, and reported low ratios of mobile to endogenous mRNAs3,5,7 are in line with the level of noise. Another way to distinguish noise from potential evidence for the alternative allele is to investigate the differences in nucleotide distributions at SNP positions compared to other positions in the sequence (non-SNP positions). If a second genotype were present, we would expect the distribution of nucleotides at any SNP position to be enriched in the nucleotide that supports the alternative allele. Furthermore, mRNAs that are transported to cells with low endogenous level (potential signals) would have a value of n/N close to 1, where n is the number of reads that match the alternative allele and N the total number of reads (endogenous + foreign). We investigated the distribution of n/N for each SNP in the mobile population of Arabidopsis2. While we do not find evidence for n/N values close to 1, there are non-zero values of n/N that seem to support the presence of the alternative allele (Fig. However, looking at all neighbouring positions of SNPs and computing the number of reads with the second most frequent nucleotide, m, over the sum of the most frequent and second most frequent nucleotides, M, we find no support for the SNP positions being different (P = 0.3575) (Fig. Thus, the expected shift in the distribution towards higher n/N values, that is n/N > m/M, is not observed. Given the low prevalence, it is important to note that this analysis does not exclude there being instances, potentially even thousands, of reads with SNPs associated with mobile mRNAs in the data, but if so we cannot distinguish them from noise. Interestingly, two samples from Arabidopsis2 do contain numbers of foreign reads that exceed expected noise levels. Investigating further, we find that these samples exhibit a strong linear correlation between the read counts of the grafted tissues (Fig. Similarly, Arabidopsis transcripts found in Cuscuta pentagona correlate with the expression levels in the host genotype1. Another explanation is that the whole transcriptome is transported, with detection being proportional to read depth. Given the available data, we cannot distinguish between these possibilities. Approaches that do not rely on SNPs, such as for cross-species studies, might avoid some of the above issues. A typical pipeline for analysing between-species grafts first maps reads to the reference genome of the sampled tissue (genotype 1 in Fig. Unmapped reads are then compared to the reference genome of the potential source tissue (genotype 2 in Fig. Supplementary Table 2 lists some genome completeness estimates for assemblies that were used in previous mobile mRNA studies. For instance, at the time of the study that investigated the movement of transcripts from a Nicotiana benthamiana scion to a Solanum lycopersicum (tomato) rootstock6, ~15% of the genome was not yet assembled (Extended Data Fig. The authors therefore collected RNA-seq data and applied stringent mapping criteria to mitigate effects of using an incomplete assembly. However, repeating their procedure, we found that many reads that did not map to the tomato genome all aligned to small regions of the N. benthamiana genome, and that coverage was highly uneven over exons (Extended Data Figs. Furthermore, blasting the reads identified as being from N. benthamiana against the whole NCBI nucleotide database resulted in 100% matches to highly conserved sequences contained within many genomes, including N. benthamiana and other Solanaceae species, in particular to 18S ribosomal RNA genes, which accounted for 97.7% of the blast hits to N. benthamiana (Extended Data Fig. To test for false negatives, we mapped the heterograft reads directly to the N. benthamiana genome and found 16 short transcripts that could not be distinguished between genomes (Supplementary Table 4). In addition to genome assembly quality, read depth can also bias the interpretation of RNA-seq data from grafts between different species (Extended Data Fig. For instance, ~30% of the Arabidopsis thaliana transcriptome was reported to move into Cuscuta pentagona, while only 9% of the tomato transcriptome moves to Cuscuta1. However, there is a large discrepancy in the amount of RNA-seq data between tomato (6 Mb) and Arabidopsis experiments (2 Gb). Greater coverage would be expected to lead to more transcripts being detected32,33,34, thus explaining the reported bias in mobility between species. Overall, our study raises questions about published numbers of mobile mRNAs. The experimental evidence for movement of a small number of mRNAs over long distances in plants is compelling5,6,11,15,17,35,36. However, on the basis of RNA-seq studies, several thousand mobile transcripts have been reported1,2,3,4,6,7. Here we question this extrapolation from tens of validated cases to the published vast numbers of potential long-distance signalling agents. We described several challenges in identifying mobile mRNAs from short-read RNA-seq data (Fig. While we do not present solutions, we suggest checks that can be performed to reduce the risk of false positives. We thus end with a list of recommendations. We assume that experimental issues have been taken care of, such as checking the samples for cross-contamination, verifying that graft junctions form functional vascular connections, and every effort has been made to use high-quality genome assemblies. (1) Technological noise can lead to challenges in the assignment of RNA-seq reads. In SNP-based methods of mobile mRNA detection, it is important to be able to differentiate between sequencing-associated errors and genuine SNPs. However, every position has an error rate and the higher the read depth, the more incorrect base calls are to be expected. Base changes could arise for reverse transcriptase or amplification steps, although their error rate is typically orders of magnitude lower than sequencing errors. Conserved regions in gene families can give rise to similar challenges in distinguishing mapping ambiguities from genuine SNPs. Defining an mRNA as being mobile based on thresholds of RNA-seq reads that contain a SNP can result in base-calling errors and mapping ambiguities biasing the interpretation. To reduce the risk of such events occurring, further stringent filters can be applied (for instance, using only SNPs that are bi-allelic2) or applying rigorous statistical comparisons (for instance, estimating the allele calling frequencies and comparing them between homograft and heterograft21). Orthologous sequences (light green) can result in some RNA-seq reads aligning to a different gene and different genotype. Genome assemblies that are not complete (telomere to telomere) from exactly the same genotype as used for grafting can result in potential mismappings. The shaded blue gene in genotype 1 is missing in the reference genome assembly, resulting in RNA-seq reads from this transcript being mapped to genotype 2. A genome mapping visualization tool such as IGV37 can be used to check for pseudo-heterozygosity and contamination in the samples. Observing the distribution of nucleotides at potential SNP positions and comparing to other positions can provide confidence in the SNPs and the alternative allele calls. These distributions should be compared to those from homograft data. RNA-seq reads that cover multiple SNPs can be used to check whether the SNPs that are associated with a certain genotype co-occur in such reads. Long-read and direct RNA sequencing have higher error rates but would allow the full transcript with all SNPs to be assessed. Sequencing protocols that barcode individual molecules by using adapters with unique molecular identifiers (UMIs) can be used to determine the error rates and check whether all reads from the same molecule are consistent in terms of their genotype assignment. Accuracy of experimental and computational procedures for identifying foreign RNA-seq reads. Calculating the ratio of the number of RNA-seq reads assigned to an alternative allele (foreign reads) over the total number of mapped RNA-seq reads for an experiment (foreign + endogenous reads) is a useful metric. This value should be computed for homografts and compared to the value calculated from heterograft data. Reproducibility and consistency of putative mobile transcripts. Independent biological replicates should be used to characterize the inherent variability in the identification of candidate mobile transcripts. Reciprocal grafting is recommended to evaluate whether mobile mRNA and their orthologues are consistently mobile (if mobility motifs are inherent to transcripts, then near-identical sequences would be expected to also be mobile) and, if not, potentially pinpoint determinants of mobility. Definitions for mobile mRNAs using non-validated criteria are best avoided. It is important to test different hypotheses (for example, SNP vs sequencing noise; read from a foreign genotype vs mapping error; transport vs contamination; signalling molecules vs leftovers from differentiating cells) to explain the data. The plausibility of associated mechanisms can lend weight to different hypotheses. All code and scripts are freely available from our GitHub repository at https://github.com/mtomtom/reanalysis-mobile-mrna/tree/main (ref. The expression levels were quantified with Stringtie (v.1.3.5)41 using The raw counts were quantified with bcftools (v.1.10.2)40 using These flags were chosen to compare the raw error rates between the homograft and heterograft to catch all nucleotides. Note that the bcftools mpileup default sequencing depth is 8,000, but the most highly expressed genes have up to 200,000 reads covering a locus within the datasets we considered. The NCBI nucleotide database was downloaded on 21 October 2022 and blast+ (v.2.9.0)42 was utilized for alignments using -outfmt '6 qseqid sseqid pident evalue staxids sscinames scomnames sskingdoms stitle' If we are only interested in the number of reads that contain a SNP that corresponds to the alternative allele, we can use a binomial distribution (q is the probability the SNP matches the alternative allele, 1 − q is the probability that it does not) to evaluate the probability of this event occurring by chance21. The probabilities of errors occurring by chance were calculated from a standard cumulative binomial distribution, P(k ≥ m|N) = 1 − P(k < m∣N), which accounts for the requirement of having k reads, where k is at least m, out of N. Considering replicates can be handled in the same way (the probability of each SNP is computed from the cumulative binomial function and the requirement for a defined number of replicates can likewise be computed from a cumulative binomial function). Multiple SNPs per read results in a multiplication of probabilities. Cumulative binomial function values were computed using standard available functions in Python and R. Rather than ‘defining' a transcript as mobile, we evaluated the probability of the data being consistent with expected noise against the probability of the data being best explained by the presence of two genotypes (and therefore potential candidates for mobile transcripts)21. Essentially, this means that if we find 10 out of 100 reads that match the alternative allele, we compute how likely this would occur by chance for a defined error rate. The implicit but rarely checked assumption in all SNP-based mobile mRNA detection pipelines is that the occurrence of reads that support the alternative allele in the heterograft data is larger than in homograft data. We capture this uncertainty through probability distributions to inform inferences drawn from the data21. This ratio of the statistical evidence of one hypothesis over another is known as the Bayes factor43. 2a are based on the commonly used value of log Bayes factor greater than 1 (refs. The statistical comparison of error rates was performed using baymobil28. To compare the full distributions of n/N and m/M values for different positions of RNA-seq reads, we used an exact two-sample Kolmogorov–Smirnov test, ks.test, available in R44. To evaluate whether the data supported the SNP distributions having higher values of n/N than other positions (m/M), we used an asymptotic two-sample Kolmogorov–Smirnov test. These tests were carried out for histograms with 100 bins. We downloaded the pseudo-heterozygous data from https://zenodo.org/records/6025134 (ref. From the vcf-file we extracted all heterozygous calls for accession 9947 (Ped-0) and obtained 6,303 heterozygous SNPs. We compared these SNPs against the MATRIX_GWAS_raw_position.txt (from https://doi.org/10.5281/zenodo.5702395). We intersected these potential duplicate genes with the list of mobile genes2 and found 19 duplicate genes. These are given in Supplementary Table 3. We analysed the samples of the root (1,373 mRNAs/867 unique) and rosette (577 mRNAs/151 unique), and reciprocally inspected the relationship between root alleles that were detected in the rosette and vice versa. We took the raw sequencing depth for 48,934 previously identified SNPs. The linear fit was performed within gnuplot45. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. We used the following published datasets and the archived reads from NCBI: Cuscuta pentagona1 (PRJNA257158; this dataset was incomplete and partly corrupt); Vitis vinifera3 (SRP058158 and SRP058157); Solanum lycopersicum, Nicotiana benthamiana6 (SRP111187); Arabidopsis thaliana2 (PRJNA271927). We used deposited supplementary datasets of the associated publications to obtain the numbers of identified mRNAs. We used largely available software packages as stated in the Methods. All code and scripts are freely available on GitHub at https://github.com/mtomtom/reanalysis-mobile-mrna/tree/main (ref. Kim, G., LeBlanc, M. L., Wafula, E. K., dePamphilis, C. W. & Westwood, J. H. Genomic-scale exchange of mRNA between a parasitic plant and its hosts. Thieme, C. J. et al. Endogenous Arabidopsis messenger RNAs transported to distant tissues. Yang, Y. et al. Messenger RNA exchange between scions and rootstocks in grafted grapevines. Wang, Y. et al. A universal pipeline for mobile mRNA detection and insights into heterografting advantages under chilling stress. Notaguchi, M., Higashiyama, T. & Suzuki, T. Identification of mRNAs that move over long distances using an RNA-seq analysis of Arabidopsis/Nicotiana benthamiana heterografts. Xia, C. et al. Elucidation of the mechanisms of long-distance mRNA movement in a Nicotiana benthamiana/tomato heterograft system. Zhang, Z. et al. Vascular-mediated signalling involved in early phosphate stress response in plants. Lucas, W. J., Yoo, B.-C. & Kragler, F. RNA as a long-distance information macromolecule in plants. An RNA-based information superhighway in plants. Spiegelman, Z., Golan, G. & Wolf, S. Don't kill the messenger: long-distance trafficking of mRNA molecules. Ham, B.-K. & Lucas, W. J. Phloem-mobile RNAs as systemic signaling agents. & Banerjee, A. K. Multiple mobile mRNA signals regulate tuber development in potato. Kim, M., Canio, W., Kessler, S. & Sinha, N. Developmental changes due to long-distance movement of a homeobox fusion transcript in tomato. Kitagawa, M., Wu, P., Balkunde, R., Cunniff, P. & Jackson, D. An RNA exosome subunit mediates cell-to-cell trafficking of a homeobox mRNA via plasmodesmata. & Kragler, F. Long-distance transported RNAs: from identity to function. Heeney, M. & Frank, M. H. The mRNA mobileome: challenges and opportunities for deciphering signals from the noise. Wang, T. et al. Movement of ACC oxidase 3 mRNA from seeds to flesh promotes fruit ripening in apple. Exact Bayesian inference for the detection of graft-mobile transcripts from sequencing data. Systematic evaluation of error rates and causes in short samples in next-generation sequencing. Performance comparison of benchtop high-throughput sequencing platforms. Reverse transcription errors and RNA–DNA differences at short tandem repeats. Li, W. & Lynch, M. Universally high transcript error rates in bacteria. Verwilt, J., Mestdagh, P. & Vandesompele, J. Artifacts and biases of the reverse transcription reaction in RNA sequencing. & Tomkins, M. baymobil: a Python package for detection of graft-mobile mRNA using exact Bayesian inference on RNA-seq data. Zapata, L. et al. Chromosome-level assembly of Arabidopsis thaliana Ler reveals the extent of translocation and inversion polymorphisms. Jaegle, B. et al. Extensive sequence duplication in Arabidopsis revealed by pseudo-heterozygosity. On the selectivity, specificity and signalling potential of the long-distance movement of messenger RNA. Wang, Z., Gerstein, M. & Snyder, M. RNA-seq: a revolutionary tool for transcriptomics. Conesa, A. et al. A survey of best practices for RNA-seq data analysis. Stark, R., Grzelak, M. & Hadfield, J. RNA sequencing: the teenage years. Zhang, W. et al. tRNA-related sequences trigger systemic mRNA transport in plants. Yang, L., Machin, F., Wang, S., Saplaoura, E. & Kragler, F. Heritable transgene-free genome editing in plants by grafting of wild-type shoots to transgenic donor rootstocks. Robinson, J. T. et al. Integrative genomics viewer. mtomtom/reanalysis-mobile-mrna: Re-analysis of mobile mRNA datasets raises questions about the extent of long-distance mRNA communication. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Pertea, M., Kim, D., Pertea, G. M., Leek, J. T. & Salzberg, S. L. Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown. Camacho, C. et al. BLAST+: architecture and applications. Jaynes, E. T. Probability Theory: the Logic of Science (Cambridge Univ. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2021); https://www.R-project.org/ Williams, T. et al. Gnuplot 4.6: an interactive plotting program. Yates, A. et al. Ensembl Genomes 2022: an expanding genome resource for non-vertebrates. We thank D. Staiger (Bielefeld University), W. Haerty (Earlham Institute), C. Dean (JIC), K. Schneeberger (LMU Munich), M. Mayer (QIB), C. Abreu-Goodger (University of Edinburgh), B. Zagrovic and A. Polyansky (Max Perutz Labs, Vienna) for discussions, insightful comments and constructive feedback on previous versions of the manuscript. The presented reanalysis and the insights derived from it would thus not have been possible without the availability of raw data; we thank all authors who deposited their data, meta-data and methods in public repositories. gratefully acknowledges support from the Biotechnology and Biological Science Research Council Institute Strategic Programme ‘Building Resilience in Crops' (BB/X01102X/1). acknowledge support from National Science Foundation Grants DGE-2139899, DBI-2019674 and IOS-1942437. J.K. acknowledges support from the Deutsche Forschungsgemeinschaft (DFG; Project No. This Article is part of a project that has received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (grant agreement no. These authors contributed equally: Pirita Paajanen, Melissa Tomkins, Franziska Hoerbst. Computational and Systems Biology, John Innes Centre, Norwich, UK Pirita Paajanen, Melissa Tomkins, Franziska Hoerbst, Ruth Veevers & Richard J. Morris School of Integrative Plant Science, Cornell University, Ithaca, NY, USA Cell and Developmental Biology, John Innes Centre, Norwich, UK Department II, Max Planck Institute of Molecular Plant Physiology, Potsdam-Golm, Germany Federico Apelt, Eleftheria Saplaoura, Saurabh Gupta, Dirk Walther & Friedrich Kragler Department of Biology, Institute for Plant Sciences and Microbiology, University of Hamburg, Hamburg, Germany You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar wrote the original draft, and all authors reviewed and edited the manuscript. Correspondence to Pirita Paajanen or Richard J. Morris. The authors declare no competing interests. Nature Plants thanks Marco Catoni, Cankui Zhang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. These plots show the probabilities of transcripts being defined as mobile by chance. Three different mobile mRNA definitions (a, b, c) and their dependence on read-depth (N) and on the rate of a SNP matching to the alternate allele (q) are depicted. The number of read counts over one SNP that correspond to the alternate allele is denoted by n1SNP, over two SNPs by n2SNPs. The probabilities were calculated using a cumulative binomial distribution, that is we account only for the nucleotides that correspond to the two alleles of interest. Note that both axes are on a log-scale. The requirement for co-occurring SNPs on one read (c) is more stringent and less likely to occur by chance at higher read-depths. For low values of q, these criteria are robust up to moderately high (several hundred) read-depths and would be unlikely to occur by chance. a, SNPs can be in close proximity, and therefore it can happen that several SNPs are recorded in the same RNA-Seq read. In this example, genotype 1 has three SNPs very close to each other: A, G and A (yellow bar). In this schematic example, reads from the shoot of Genotype 1 are mapped to Genotype 2. If all covered loci carry the allele of Genotype 2, we are observing evidence for the read being from Genotype 2 and the associated transcript being potentially mobile. On the other hand, if only one loci carries the allele of Genotype 2, the outcome is inconclusive, as it may indicate sequencing errors. b, A. thaliana is a selfing species, so we expect homozygocity at all positions for all reads mapping to the genome at all positions. However, for duplicated genes (magenta) in Genotype 1, which may be single copy genes in Genotype 2 (yellow), short read sequencing and mapping to Genotype 2, can give rise to what appears to be heterozygocity. When there are two alleles present in the homograft data (magenta and yellow), we may be observing pseudo-heterozygocity. These are likely two sequencing errors in the root sample, a C at a non-SNP position and an A at a SNP position (both highlighted in blue circles). In the shoot sample we see potential evidence for mobility at the SNP level but in one case the second SNP is not present and in the other case another sequencing error has occurred (G). Three further sequencing errors (two As on the top left, one A on the right) are also present in the shoot. b, This example shows two positions, A and T in Genotype 1 (Col) and G and G in Genotype 2 (Ped), for which some reads support the alternate allele (green tick), whereas others are likely sequencing errors (red cross). In the latter case, one G is in the correct position but the other G is not present and a further mismatch (T) has occurred. The images are annotated screenshots taken in IGV37. At the highlighted positions there are distinct populations of alleles (depicted as red/blue and red/green bars). This is possibly due to the gene being duplicated in Ped, resulting in pseudoheterozygocity. a, Sequenced transcripts would ideally have RNA-Seq reads covering most of the sequence, that is that all exons of the mRNA are approximately equally covered by sequenc- ing reads (top left). Reads covering all exons in the sample from Genotype 2 provide support for the whole transcript having moved from Genotype 1 to Genotype 2 across the graft junction. Transcripts with coverage only for a subsequence (bottom left) do not support full-length presence of the for- eign transcript. b, Neighbouring positions to SNPs can be used as a negative control to evaluate the strength of the signal at SNP positions. Shown here are neighbouring positions of the identified SNPs at the next nucleotide (SNP position +1). If the neighbouring position shows similar levels of alter- native nucleotides as the SNP position, the these are likely sequencing errors, rather than evidence for the alternate allele. If the SNP positions have a different frequency of Genotype 2 allele than the neighbouring position has errors, then there is evidence for the alternate allele. Analysing the fre- quencies of nucleotides at known SNP positions and their neighbours can aid data interpretation. In the Nico- tiana benthamiana annotation of the depicted gene (Niben101Scf11383g00015.1) we find 5 annotated exons of which all are populated with reads at different levels (grey histograms). In the samples from tomato, non-grafted or grafted we see that not all annotated exons are populated with reads and that the exons with coverage are populated in both grafted and non-grafted samples. Coverage over the full length of the mRNA may help reduce the risk of reads mapping to isolated regions being potentially misinterpreted, Extended Data Figure 5. This is a screenshot taken in IGV37. (a) The presence of Genotype 1 reads in Genotype 2 samples and vice versa, across the whole of genome, especially in genes expressed in both tissues is consistent both with non- selective transport and contamination. (b) The two genes presented in this schematic figure have different relative expression levels. In Experiment 2 the sequencing depth is insufficient to detect lowly expressed genes. A bar plot of the blast results of unmapped reads against the NCBI database that matched Nicotiana benthamiana. Completeness of genome assemblies used in mobile mRNA studies. Potentially pseudo-heterozygous genes in the Arabidopsis Ped-0 ecotype and their functional annotations. Potentially false negatives in the heterograft between Solanum lycopersicum and Nicotiana benthamiana. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Paajanen, P., Tomkins, M., Hoerbst, F. et al. Re-analysis of mobile mRNA datasets raises questions about the extent of long-distance mRNA communication. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            