
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-07</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09896-x'>Mitochondrial transfer from glia to neurons protects against peripheral neuropathy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 16:25:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Primary sensory neurons in dorsal root ganglia (DRG) have long axons and a high demand for mitochondria, and mitochondrial dysfunction has been implicated in peripheral neuropathy after diabetes and chemotherapy1,2. However, the mechanisms by which primary sensory neurons maintain their mitochondrial supply remain unclear. Satellite glial cells (SGCs) in DRG encircle sensory neurons and regulate neuronal activity and pain3. Here we show that SGCs are capable of transferring mitochondria to DRG sensory neurons in vitro, ex vivo and in vivo by the formation of tunnelling nanotubes with SGC-derived myosin 10 (MYO10). Blockade of mitochondrial transfer in naive mice leads to nerve degeneration and neuropathic pain. Single-nucleus RNA sequencing and in situ hybridization revealed that MYO10 is highly expressed in human SGCs. Furthermore, SGCs from DRG of people with diabetes exhibit reduced MYO10 expression and mitochondrial transfer to neurons. Adoptive transfer of human SGCs into the mouse DRG provides MYO10-dependent protection against peripheral neuropathy. This study uncovers a previously unrecognized role of peripheral glia and provides insights into small fibre neuropathy in diabetes, offering new therapeutic strategies for the management of neuropathic pain. Primary sensory neurons within the DRG are pseudounipolar, with long peripheral branches that innervate target organs such as the skin4. These neurons detect sensory inputs—including pain, touch, temperature and proprioception—and transmit them to the central nervous system via action potentials5. Neuronal activity, including rapid action potential firing, relies on tightly regulated mitochondrial function6. The peripheral axons of human DRG sensory neurons can extend up to 100 cm in the sciatic nerve. However, these long axons and their terminal branches in the skin face exceptional challenges in mitochondrial transport and delivery7. This raises the key question of how a single DRG neuron generates and maintains sufficient mitochondria to support its activity and axonal regeneration. SGCs exhibit a unique morphology, tightly enveloping the cell bodies of sensory neurons in the DRG3. However, the structural connections between SGCs and sensory neurons under both physiological and neuropathic conditions remain poorly understood. Mitochondria, often referred to as the powerhouse of the cell owing to their role in energy production8, were traditionally thought to be generated and maintained solely within individual cells. However, accumulating evidence suggests that mitochondria can be transferred between cells. For example, specialized structures called tunnelling nanotubes (TNTs) have been observed connecting different cell types in vitro9,10, and have recently been implicated in mitochondrial transfer between microglia11, from stem cells to endothelial cells12, and from stem cells to T cells13. Although these findings highlight the dynamic interplay between different cell types and the roles of mitochondrial transfer in diverse physiological and pathological processes, the cell-specific mechanisms of mitochondrial transfer remain unclear, and in vivo evidence is still limited. To investigate mitochondrial transfer from SGCs to sensory neurons in vitro, we first prepared primary cultures of SGCs and neurons separately. Mouse primary SGC cultures were immunostained with established SGC markers, including FABP7, Kir4.1, AQP4, connexin 43 and glutamine synthetase. At 5 days in vitro (DIV5), 94.9–97.1% of cultured SGCs were positive for these markers (Extended Data Fig. 1a), validating the identity and purity of our SGC cultures. Next, we labelled SGCs with MitoTracker deep red dye14, co-cultured these cells with primary sensory neurons (without MitoTracker dye labelling) from DRG of Trpv1:Ai9 reporter mice, and captured images of the co-cultures after 24 h (Fig. Remarkably, we observed mitochondrial fluorescence not only in SGCs but also in Trpv1:Ai9+ neurons, indicating that mitochondria were transferred from SGCs to sensory neurons. Notably, we detected TNTs of more than 30 μm in length connecting SGCs and neurons, with mitochondria present inside these structures (Fig. Quantitative analysis revealed that 83.3% of DRG neurons received mitochondria from SGCs, whereas only 31.3% of neurons exhibited visible TNTs (Fig. Live imaging of the co-cultures (Extended Data Fig. 1c) revealed that TNTs were formed transiently and could disassemble within tens of minutes (Extended Data Fig. 1d), which may explain the relatively low percentage of TNT-positive neurons. To determine whether mitochondrial transfer requires intrinsic neuronal activity, we treated co-cultures with tetrodotoxin (TTX) to block sodium channels. TTX reduced mitochondrial transfer (Extended Data Fig. 1e–g) without affecting the overall rate of TNT formation between SGCs and neurons (Extended Data Fig. Together, these results indicate that SGCs can transfer mitochondria to DRG neurons in an activity-dependent manner. a, Schematic of SGC–neuron co-cultures from mouse DRG. b, Left, image from an SGC–neuron co-culture showing a Trpv1+ neuron interacting with an SGC (scale bar, 20 μm). Right, enlarged view of the boxed area showing a TNT (white arrow) and a mitochondrion (Mito; red arrow) within the TNT (scale bar, 5 μm). c, Percentage of MitoTracker-positive (Mito+) and TNT-positive neurons. A total of 109 neurons from 6 independent experiments were included for quantification. d, Schematic for SEM of whole-mount DRG without sectioning. e, High-magnification SEM images of whole-mount mouse DRG showing a TNT-like structure (TNT-LS) with a bulge. f, Schematic for SEM of sectioned mouse DRG. g, Representative SEM images of sectioned mouse DRG showing TNT-LS with bulges from SGCs to neuron. h, Schematic for TEM of mouse DRG. i, Representative low-magnification TEM imaging showing the DRG neuron and SGCs. j–l, Representative high-magnification TEM imaging showing a TNT-LS between SGC and neuron. ‘Mito' in black indicates mitochondria in the cell (j–l); ‘Mito' in red indicates mitochondria in the TNT-LS (k); ‘Vesicle' indicates a vesicle in the TNT-LS (k,l); the black asterisk indicates the nucleus of an SGC. l, Enlarged view from k showing mitochondria (red arrows) and a vesicle inside the smooth TNT-LS. Scale bars: 2 μm (j,k); 800 nm (l). Previous studies have proposed three primary mechanisms for mitochondrial transfer: (1) formation of TNTs11, which can be disrupted by cytochalasin B (CytoB)15; (2) endocytosis and internalization of mitochondria by recipient cells16; and (3) transfer via gap junctions, which can be blocked by carbenoxolone17 (CBX). We investigated these mechanisms using pharmacological approaches. Treatment with CytoB effectively blocked mitochondrial transfer from SGCs to neurons in co-cultures. A similar blockade was observed with Y-27632, which is known to suppress uropod and TNT formation18 (Extended Data Fig. Moreover, mitochondrial transfer was suppressed by the endocytosis inhibitor Pitstop219 and the gap junction blocker CBX (Extended Data Fig. CytoB, Y-27632 and CBX further suppressed TNT formation (Extended Data Fig. Together, our in vitro results support a model in which mitochondrial transfer from SGCs to sensory neurons is mediated by TNTs, endocytosis and gap junctions. To examine mitochondrial transfer ex vivo, whole-mount DRG tissue was co-cultured with SGCs pre-labelled with MitoTracker (Extended Data Fig. We observed extensive TNTs formed between MitoTracker-labelled SGCs and DRG cells, including neurons and SGCs (Extended Data Fig. Moreover, treatment of SGC-whole-mount DRG co-cultures with CytoB (Extended Data Fig. 2d) caused marked inhibition in SGC mitochondrial transfer to whole-mount DRG (Extended Data Fig. Macrophage-to-neuron mitochondrial transfer in mouse DRG contributes to the resolution of acute inflammatory pain20. We conducted a macrophage–whole-mount DRG co-culture experiment and confirmed that macrophages can also transfer mitochondria to DRG neurons (Extended Data Fig. Together, these experiments provide strong evidence for mitochondria transfer from SGCs to sensory neurons both in vitro and ex vivo. To obtain three-dimensional ultrastructures of TNTs in the DRG, we used scanning electron microscopy (SEM; Fig. We found abundant extracellular matrix-like structures (ECM-LS) on the surface of SGCs, which may provide additional structural support for neurons. Notably, we observed TNT-like structures (TNT-LS) connecting adjacent cells, with large vesicle-like bulges present within the TNT-LS in mouse DRG (Fig. To further visualize the detailed contacts between SGCs and neurons, we performed SEM imaging on horizontally half-sectioned DRG (Fig. This cryostat-based sectioning method provided improved visualization of both SGCs and neurons within the DRG architecture (Fig. TNT-LS between SGC and neuron were observed in mouse DRG, and often contained bulges consistent in size with mitochondria (Fig. As expected from a previous study21, the tube structures were largely disrupted by trypsin treatment (Extended Data Fig. Notably, the formation of TNT-LS between SGCs and neurons was impaired in mice with chemotherapy-induced peripheral neuropathy (CIPN), as evidenced by enlarged intercellular gaps and a reduced number of TNT-LS after paclitaxel (PTX) treatment (Extended Data Fig. These results indicate that mitochondria-transferring TNTs are present between SGCs and sensory neurons in the peripheral DRG and are susceptible to neurodegenerative conditions such as CIPN. We further utilized structured illumination microscopy (SIM) to resolve individual mitochondria in mouse SGC. Quantification of mitochondrial length revealed an increase in the number of mitochondria shorter than 2 μm following PTX treatment (Extended Data Fig. This increase in short mitochondria is associated with cell stress22, suggesting that the PTX treatment directly induced mitochondrial dysfunction in SGCs. To validate the presence of mitochondria within TNTs of the DRG, we performed transmission electron microscopy (TEM) on 70-nm sections of mouse DRG (Fig. Smooth TNT-LS were observed between SGCs and neurons (Fig. Both mitochondria and vesicles were present within TNT-LS connecting SGCs and neurons (Fig. Together with the SEM findings, these results provide ultrastructural evidence for the presence of TNT-LS containing mitochondria and vesicular cargo between SGCs and neurons in the DRG. To investigate mitochondrial transfer from SGCs to sensory neurons in vivo, we crossed MitoTag mice, which express eGFP on the mitochondrial outer membrane23, with Aldh1l1-creERT2 mice to drive SGC-specific expression of MitoTag24 (Extended Data Fig. Tamoxifen (Tam) was administered daily for five days to temporally induce Cre-recombinase activity (Fig. DRG were collected five and ten days after the Tam treatment. No MitoTag fluorescence signal was detected in mice without Tam treatment (Extended Data Fig. 5c), whereas MitoTag signal was present in SGCs, marked by the SGC marker FABP7, at both day 5 and day 10 post-Tam injection (Fig. Of note, we observed a substantial increase in MitoTag signal over time, with 23.0% of DRG neurons producing a signal on day 10 compared with only 2.9% on day 5 (Fig. Colocalization of MitoTag with the mitochondrial marker TOM20 further confirmed the mitochondrial identity (Extended Data Fig. 5d), supporting a time-dependent mitochondrial transfer from SGCs to sensory neurons. To explore the subcellular mechanism of this transfer, we administrated CytoB intrathecally and observed a decrease in MitoTag signal in DRG neurons (Fig. 2c), whereas MitoTag expression in SGCs remained detectable (Fig. We also performed intrathecal injection of Pitstop2 to block endocytosis in the DRG (Extended Data Fig. 5e,f), which partially decreased mitochondrial transfer from SGCs to neurons (Fig. Next, we examined whether mitochondrial transfer is altered under pathological conditions. In a diabetic neuropathic pain model, induced by intraperitoneal injection of streptozotocin (STZ; Extended Data Fig. 5e,f), we found that STZ pretreatment but not post-treatment could block SGC-to-neuron mitochondrial transfer (Fig. Intraplantar injection of complete Freund's adjuvant (CFA) is known to elicit inflammatory pain. However, CFA did not alter the percentage of neurons receiving mitochondrial transfer (Fig. b, Top, MitoTag signal in DRG sections. Bottom, enlarged views showing double staining with FABP7 and Nissl. **P = 0.0034. d, Diagram of SNI surgery in Aldh1l1-creERT2:MitoTag mice. e, MitoTag signal in the contralateral and ipsilateral DRG. f, The percentage of MitoTag+ neurons in ipsilateral DRG (n = 4) and contralateral DRG (n = 4). **P = 0.002. g, Diagram of SNI surgery with ipsilateral peri-sciatic bupivacaine film implantation. j, Schematic of injection of AAV-MaCPNS2-Syn-jRGECO1a into Aldh1l1-MitoTag mice, SNI surgery and ex vivo calcium imaging in DRG. k, DRG calcium imaging of jRGECO1a in Aldh1l1-MitoTag mice. n = 45 cells from 9 DRG of 3 different mice. One-way ANOVA followed by Tukey's multiple comparisons test (c); two-sided unpaired t-test (f,i,m). Neuronal activation and action potential firing critically depend on mitochondrial function6, and abnormal neuronal activity after nerve injury drives neuropathic pain25. To investigate the relationship between neuronal activity and mitochondrial transfer, we performed spared nerve injury (SNI) surgery at the time of the first Tam injection and analysed DRG tissue 5 days later (Fig. Additionally, eGFP+ mitochondrial signals were observed in axons, suggesting further mitochondrial transport from neuronal cell bodies to axonal processes (Fig. To further determine the dependence of mitochondrial transfer on neuronal hyperactivity, we implemented a sustained nerve blockade using bupivacaine-containing PEU (poly(ester urea)) films26 wrapped around the sciatic nerve during SNI surgery (Fig. This blockade could increase the mechanical pain threshold above pre-injury baseline levels in SNI mice (Extended Data Fig. Notably, the percentage of MitoTag+ neurons was not different between the ipsilateral and contralateral DRG (Fig. 2g–i), suggesting that neuronal activity is required for mitochondrial transfer. Mitochondria remained present in SGCs after the nerve blockade (Fig. 2h), suggesting that the blockade impaired transfer rather than biogenesis of mitochondria. Together, these results suggest that SGC-to-neuron mitochondrial transfer is regulated by neuronal activity and specific pathological conditions. Given the role of spinal cord astrocytes in regulating physiological and pathological pain, we next investigated whether mitochondrial transfer occurs between astrocytes and neurons in the spinal cord using the same Aldh1l1-creERT2 system. No MitoTag signal was detected in Tam-untreated animals (Extended Data Fig. We did not observe any mitochondrial transfer from astrocytes to spinal neurons at either five or ten days post-Tam induction, but MitoTag labelling clearly delineated individual astrocyte territories (Extended Data Fig. SEM imaging of the spinal cord revealed cilia-like structures, but no tube-like connections between astrocytes and neurons or between astrocytes themselves (Extended Data Fig. To evaluate the directionality of mitochondrial transfer, we crossed MitoTag mice with Advillin-cre mice to drive sensory neuron-specific expression. This resulted in strong MitoTag labelling in DRG neurons but only minimal signal in SGCs (labelled with FABP7), indicating predominantly one-way transfer from SGCs to neurons. Similarly, mitochondrial transfer from neurons to macrophages (labelled with IBA1 (also known as AIF1)) in the DRG was minimal (Extended Data Fig. Finally, enabling MitoTag expression in macrophages using Cx3cr1-creERT2 mice did not result in detectable mitochondrial transfer from macrophages to sensory neurons in the DRG (Extended Data Fig. Together, these in vivo findings strongly support specific and unidirectional mitochondrial transfer from SGCs to sensory neurons under the experimental conditions tested. To investigate SGC-to-neuron mitochondrial transfer in relation to neuronal hyperactivity after SNI, we used ex vivo calcium imaging in DRG using jRGECO1a, a sensitive calcium indicator with red fluorescence27. Adeno-associated viruses (AAVs) encoding jRGECO1a (AAV-MaCPNS2-Syn-jRGECO1a) were injected intraperitoneally into Aldh1l1:MitoTag mice at postnatal day 0 (P0; Fig. Four weeks later, mice underwent SNI surgery followed by five Tam injections, and L4–L5 DRG were collected for ex vivo calcium imaging (Fig. jRGECO1a fluorescence was detected in many small-sized DRG neurons (Fig. Among 334 neurons in the SNI group, only 3 (less than 1%) were positive for both MitoTag and jRGECO1a (Fig. In non-SNI controls, both MitoTag and jRGECO1a signals were greatly reduced, with no colocalization (Extended Data Fig. Cell size analysis showed MitoTag+ signals in medium-to-large neurons, whereas jRGECO1a+ signals appeared mainly in small-to-medium neurons (Extended Data Fig. As a positive control, KCl depolarization evoked jRGECO1a responses in neurons of all sizes, confirming that there was no size-selective labelling (Extended Data Fig. Together, these findings indicate that: (1) SNI induces neuronal hyperactivity primarily in small-sized neurons; and (2) mitochondrial transfer from SGCs may preferentially protect medium-to-large neurons from becoming hyperactive after nerve injury. This protective mechanism may have implications for the pathogenesis of small fibre neuropathy in neuropathic pain28. TNTs contain F-actin but lack microtubules9, contributing to their high elasticity. Ultrastructural analysis by SEM and cryo-electron microscopy has revealed the presence of MYO10 in TNTs29. Analysis of a published mouse DRG single-cell RNA-sequencing (scRNA-seq) dataset30 revealed Myo10 enrichment in SGCs (Fig. Immunostaining confirmed MYO10 expression in mouse SGCs, co-localizing with the SGC marker FABP7. To evaluate the role of MYO10 in TNT formation, we knocked down Myo10 in cultured SGCs with specific small interfering RNA (siRNA) and co-cultured them with DRG neurons (Fig. Immunocytochemistry revealed MYO10 expression in SGCs and TNTs connecting to neurons (Fig. 3f), and impaired mitochondrial transfer to neurons (Fig. We validated these results in vivo by microinjecting siRNA targeting Myo10 (siMyo10) or control siRNA into DRG of Aldh1l1:MitoTag mice (Fig. 3i,j), lowered paw withdrawal threshold at day 3 (Extended Data Fig. 9b,c), and disrupted TNT-LS in the DRG (Extended Data Fig. To further assess MYO10 in TNT formation and pain, we analysed Myo10-knockout (tm1d) mice, which we generated in an earlier study31. Since homozygous deletion is semi-lethal with frequent exencephaly31, we used heterozygous (Myo10+/−) mice for behavioural and anatomical analyses. Compared with wild-type controls, Myo10+/− mice exhibited heightened mechanical and thermal sensitivity (Fig. SEM revealed wider gaps between SGCs and neurons and reduced TNT-LS following Myo10 knockdown (Fig. Together, these findings demonstrate that MYO10 is essential for TNT formation, mitochondrial transfer and SGC–neuron contact, and even partial loss disrupts TNT formation and promotes pain hypersensitivity. a, Analysis of scRNA-seq data30 (reproduced with license number 6122760569440) reveals the expression of Myo10 in mouse DRG. b, Immunohistochemistry of MYO10 in mouse DRG. c, Schematic of mitochondrial transfer from SGCs to neurons and its blockade by knockdown of Myo10. d, Signals of MYO10 and MitoTracker in SGCs and neuron co-cultures treated with control siRNA (siCtrl) or siMyo10. *P = 0.0285. f,g, Percentage of TNT+ neurons (f) and MitoTracker+ neurons (g) in control and Myo10 siRNA groups. h, Diagram of experimental setup for Aldh1l1-creERT2:MitoTag mice treated with Tam and siMyo10. i, MitoTag signal in DRG of siCtrl and siMyo10 groups. j, Quantification of the percentage of MitoTag+ neurons in control (n = 3) and siMyo10 (n = 4) groups. k, Schematic of wild-type (WT) and Myo10+/− mice for behavioural study. m,n, Representative SEM images of DRG from wild-type (m) and Myo10+/− (n) mice. Asterisks indicate neurons with visible gaps between surrounding SGCs. N1 and N2 indicate neurons; S1 to S7 indicate SGCs; the white arrow (m) indicates a TNT with bulge; the blue arrow (n) points to an irregular TNT spanning a gap between SGC and neuron. o, The percentage of DRG neurons having gaps. p, Quantification of TNTs within SGCs. To test the role of TNTs in vivo, we microinjected MitoTracker into the DRG. MitoTracker signal was observed in both neurons and SGCs, but application of CytoB decreased signal intensity and the percentage of labelled neurons (Extended Data Fig. To assess mitochondrial health, we treated SGCs with PTX and co-cultured these cells with healthy neurons (Extended Data Fig. JC-1 aggregate signals were reduced in both SGCs and neurons, suggesting transfer of dysfunctional mitochondria from SGCs to neurons. CytoB blocked this PTX-induced loss (Extended Data Fig. Seahorse analysis further showed that CytoB reduced oxygen consumption rate (OCR) in SGC–neuron co-cultures (Extended Data Fig. Sensory neurons damaged by nerve injury or chemotherapy exhibit abnormal hyperactivity that drives pain sensitivity32. PTX pretreatment enhanced calcium responses to capsaicin in primary cultured DRG neurons (Extended Data Fig. Notably, co-culture with SGCs suppressed this effect, but CytoB abolished this protection, implicating TNTs in SGCs-mediated neuroprotection (Extended Data Fig. Similarly, SGCs prevented PTX-induced increase of reactive oxygen species (ROS), an effect that is lost with CytoB (Extended Data Fig. Capsazepine (CPZ) blocked capsaicin responses under all conditions, confirming TRPV1 specificity (Extended Data Fig. Together, these findings suggest that PTX increases neuronal ROS, driving hyperactivity, whereas SGCs counteract these effects by transferring mitochondria to neurons via TNTs to reduce ROS and suppress hyperexcitability. Given the long axons of DRG neurons, impaired mitochondrial trafficking has been strongly implicated in neuropathic pain conditions including CIPN and diabetic peripheral neuropathy (DPN). To test the role of SGC-derived mitochondria in neuropathic pain, we adoptively transferred MitoTracker-labelled SGCs (treated with vehicle or PTX) into mouse DRG. Mitochondrial signals appeared in both neurons and GFAP+ SGCs and extended into spinal nerve axons, indicating anterograde transport (Extended Data Fig. PTX-treated SGCs exhibited impaired mitochondrial bioenergetics (Extended Data Fig. 12c), and their transfer did not support axonal mitochondrial transport (Extended Data Fig. 12d,e), indicating that dysfunctional SGC mitochondria cannot sustain long-range neuronal delivery. To determine whether dysfunctional SGC mitochondria affect axonal growth, we performed SGC–neuron co-cultures. PTX-treated SGCs impaired axonal growth, an effect that was reversed by CytoB, implicating TNT-mediated transfer (Extended Data Fig. In vivo, intra-DRG CytoB induced dose-dependent mechanical hypersensitivity (days 1–5), similar to the mitochondrial toxin antimycin A (Extended Data Fig. Both treatments also reduced intraepidermal nerve fibre (IENF) density (Extended Data Fig. Similarly, low-dose PTX reduced IENFs two weeks after treatment (Extended Data Fig. Collectively, these findings demonstrate that chemotherapy damages SGCs, and impaired SGC-to-neuron mitochondrial transfer contributes to IENF loss and CIPN. Mouse and human SGCs exhibit distinct transcriptomes, which may hinder translation of pain therapeutics33. To examine whether TNT-LS exist in human DRG, we performed SEM on DRG tissues provided by National Disease Research Interchange (NDRI) from healthy donors and those with diabetes (Fig. Immunostaining confirmed SGC-specific expression of FABP7 (Fig. 4b), with inner SGCs closely enwrapping neurons and others positioned 10–20 μm away as outer SGCs. High-resolution SEM imaging revealed TNT-LS connecting SGCs to neurons and to each other, often containing vesicle-like bulges (Fig. To profile human SGCs, we performed single-nucleus RNA sequencing (snRNA-seq) on DRG from two donors. Clustering of 11,576 human DRG nuclei identified 17 clusters and 6 major cell types (Fig. SGCs represent the largest cell population in the human DRG, comprising 31.1% of total cells (Fig. 4f) and separated into two clusters, cluster 1 and cluster 2, on the basis of gene expression profiles (enriched genes listed in Supplementary Table 3). Consistent with a prior report34, human SGCs were identified by the expression of FABP7 and EDNRB (Extended Data Fig. MYO10 was highly enriched in SGCs (Fig. 4g), with a violin plot showing much higher expression in SGCs than neurons (P < 0.0001; Fig. Notably, MYO10 expression in SGCs was downregulated in DRG from donors with diabetes compared with those without (Fig. a, Schematic of sectioned human DRG processed with SEM and immunostaining using adjacent sections. b, Left, representative three-dimensional SEM images. Neurons are labelled N1 to N8, and N1 is enlarged in the bottom panels. c, High-magnification SEM images showing TNT-LS between an SGC and a neuron (left) and between two SGCs (right). Arrows point to TNT-LS with bulges. e, Uniform manifold approximation and projection (UMAP) of 11,576 DRG nuclei identifies 17 clusters corresponding to 6 major cell types, including SGCs, neurons, connective tissue cells, endothelial cells, immune cells and Schwann cells. f, Proportions of six major DRG cell types. g, UMAP plot showing MYO10 expression across different clusters. h, Violin plot illustrating distinct MYO10 expression levels in single SGCs and neurons. i, In situ hybridization of MYO10 in human DRG tissue from non-diabetic and diabetic donors. Arrows indicate MYO10 mRNA puncta that co-localize with FABP7. j, Quantification of MYO10 puncta surrounding individual DRG neurons per 3,000 μm2 area. n = 4 donors without diabetes; n = 5 donors with diabetes; 4 images (each represented by a small dot) from each donor. k, Schematic of primary SGC–neuron co-culture from non-diabetic and diabetic DRG. l, Representative images of MitoTracker in SGC–neuron co-cultures. Red arrows point to mitochondria within TNTs. m, Quantification of MitoTracker density as the integrated fluorescence intensity in neurons, normalized to the corresponding intensity in interacting SGCs. n = 6 neurons from 2 independent experiments (no diabetes); n = 9 neurons from 3 independent experiments (diabetes). We next examined morphological changes of SGCs and TNTs in human DRG from donors with or without diabetes. DRG from individuals with diabetes showed enlarged neuron–SGC gaps (0.1 μm in control versus 1–4 μm in diabetes), irregular and unsmooth TNTs, and multilayered SGCs (Extended Data Fig. Consistently, SEM of CIPN mice two weeks after PTX revealed enlarged gaps compared with naive and recovered mice, with TNTs appearing more visible owing to gap expansion (Extended Data Fig. Whereas less than 8% of neurons displayed clear gaps under normal conditions, around 50% did so after PTX (Extended Data Fig. These results imply that neuron–SGC separation, enlarged gaps and disrupted TNTs after chemotherapy impair neuro–glial interactions and contribute to peripheral neuropathy. We tested mitochondrial transfer between SGCs and neurons from human DRG donors with and without diabetes. MitoTracker-labelled SGCs transferred mitochondria to neurons via TNTs in non-diabetic cultures, but this transfer was reduced in diabetic DRG (Fig. Directionality assays showed that transfer was more efficient from SGCs to neurons than the reverse (Extended Data Fig. Notably, 83.3% of neurons received mitochondria, with 35.3% being associated with TNT structures (Extended Data Fig. These findings suggest that neurons preferentially acquire mitochondria from SGCs, probably reflecting their high energy demands. To investigate whether adoptive transfer of healthy SGCs alleviates neuropathic pain via mitochondrial transfer, we injected human SGCs from healthy donors into the lumbar DRG of diabetic db/db mice. Pretreatment of SGCs with siRNA targeting MYO10 (siMYO10) abolished the analgesic effect observed with control siRNA-treated SGCs, which reduced mechanical hypersensitivity and enhanced mitochondrial function (Fig. To further evaluate the role of MYO10, we transferred SGCs from wild-type or Myo10+/− mice into PTX-treated mice, followed by behavioural testing (Fig. Wild-type SGCs reduced mechanical hypersensitivity within 1–2 days, whereas Myo10+/− SGCs did not do so (Fig. These results indicate that adoptively transferred SGCs can alleviate neuropathic pain in the CIPN model and that MYO10 is critical for SGC-to-neuron mitochondrial transfer via TNTs. We next explored direct mitochondrial transfer as a strategy for managing neuropathic pain. MitoTracker-labelled mitochondria isolated from cultured SGCs were injected into the DRG (Extended Data Fig. 16a), where they localized to both neurons and SGCs without TNT formation (Extended Data Fig. CytoB had no effect on mitochondrial uptake; however, Pitstop2 reduced MitoTracker signals (Extended Data Fig. 16d–f), indicating that endocytosis mediates internalization of adoptively transferred mitochondria. a, Schematic illustrating human SGC culture treated with siCtrl or siMYO10 and intra-DRG microinjection into db/db diabetic mice. b, Results of von Frey test showing the analgesic effect of transferred SGCs, which is reversed by siMYO10 treatment. 1 day vehicle versus siCtrl SGCs: **P = 0.0099; 2 days vehicle versus siCtrl SGCs: **P = 0.006; 3 days vehicle versus siCtrl SGCs: *P = 0.0241. c, OCR of DRG from mice treated with vehicle, or with SGCs pretreated with siCtrl or siMYO10. e, Analgesic effect following intra-DRG injection of wild-type SGCs. 1 day vehicle versus wild-type SGCs: *P = 0.0225; 2 days vehicle versus wild-type SGCs: *P = 0.0251. f, Schematic of human SGC culture, mitochondria isolation and intra-DRG injection. g, Analgesic effect of mitochondrial transfer from non-diabetic human SGCs. 1 day: ****P < 0.0001 (vehicle versus non-diabetic SGCs). h, Schematic of mitochondrial isolation and intra-DRG injection. i, Analgesic effect of mitochondrial transfer from a non-diabetic donor. Mitochondria from non-diabetic SGCs, baseline versus 1 day: *P = 0.0417; 2 days diabetic versus non-diabetic SGCs: *P = 0.0442. j, Left, images of PGP9.5 staining. Two-way ANOVA with Tukey's multiple comparisons tests (b,e,g,i); one-way ANOVA followed by Tukey's multiple comparisons (c); two-sided unpaired t-test (j). k, Schematic showing potential therapeutic strategies using mitochondrial transfer for neuropathic pain: (1) adoptive transfer of SGCs; and (2) transfer of isolated SGC mitochondria into the DRG. We isolated mitochondria from primary mouse cultures of SGCs, treated them with vehicle or the mitochondrial complex III inhibitor myxothiazol20, and injected them into the DRG of PTX-injected mice (Extended Data Fig. Transfer of healthy mitochondria mitigated mechanical pain, whereas transfer of myxothiazol-treated mitochondria abolished this effect (Extended Data Fig. We also performed conditioned place preference (CPP) to measure ongoing pain35 and found increased CPP scores in the mitochondria-treated group (Extended Data Fig. Furthermore, we conducted a cross-species mitochondrial transfer by administration of human mitochondria, isolated from human SGCs of non-diabetic and diabetic donors, into the lumbar DRG of PTX-treated mice (Fig. Seahorse measurement showed reduced basal OCR in diabetic SGCs compared with non-diabetic controls (Extended Data Fig. Mitochondria from healthy SGCs reduced mechanical pain for two days, whereas those from diabetic SGCs produced only transient relief (6 h; Fig. In db/db mice, transfer of mitochondrial from non-diabetic but not diabetic SGCs reduced neuropathic pain (Fig. 5h,i) and increased IENF density in hindpaw skin (Fig. Taken together, these in vivo results support our in vitro findings and highlight the therapeutic potential of SGC-derived mitochondria in treating nerve degeneration and neuropathic pain. Our study provides comprehensive in vitro, ex vivo and in vivo evidence that SGCs transfer mitochondria to adjacent sensory neurons in the DRG, thus revealing a protective role against peripheral neuropathy. Although TNT-mediated mitochondrial transfer was first identified two decades ago9, in vivo evidence has been elusive. Our scanning and TEM imaging revealed TNT-like tubes containing mitochondria in mouse and human DRG. We also identified MYO10 as a key regulator of TNT formation between SGCs and neurons in the DRG (Supplementary Fig. Our knockdown experiments demonstrate that MYO10-mediated TNT formation is essential for SGC-to-neuron mitochondrial transfer in vitro and in vivo. In addition, endocytosis and connexin 43-mediated gap junction communication are also critically involved. Moreover, connexin 43-containing gap junctions not only regulate pain3 but also stabilize TNTs and facilitate mitochondrial transfer10. Mitochondrial transfer has been implicated in diverse diseases, including obesity36, stroke14, inflammatory pain20 and cancer37,38. Our study demonstrates that SGC-to-neuron mitochondrial transfer occurs under physiological conditions, and its dysregulation drives neuropathic pain in animal models of nerve injury, CIPN and DPN. Mitochondrial dysfunction, a hallmark of CIPN2 and DPN1, is associated with pain, tingling and paraesthesia39, along with IENF loss40. Our findings provide direct evidence that impaired mitochondrial transfer from SGCs to DRG neuronal cell bodies is sufficient to trigger IENF degeneration and neuropathic pain behaviours. Small fibre neuropathy is common in chronic pain conditions, including CIPN, DPN and fibromyalgia41, We found that SGC-derived mitochondrial transfer preferentially targets medium- and large-sized neurons, but not small nociceptors, providing mechanistic insights into small fibre neuropathy. Mitochondria represent important drug targets in neurodegenerative disorders and metabolic diseases42. Our gain-of-function approaches propose multiple strategies for mitochondrial therapy in peripheral neuropathy, including adoptive transfer of SGCs to the DRG and direct delivery of SGC-derived mitochondria (Fig. Cross-species transfer of human mitochondria transfer to mouse DRG also confers protection, consistent with reports of rapid and efficient interspecies mitochondrial fusion43. A limitation of this study is that Aldh1l1 is expressed in other glia, such as astrocytes; future work will use lines with greater SGC specificity. Further studies are needed to validate TNT-like structures using higher-resolution and more specific electron microscopy approaches. In summary, SGCs regulate pain via multiple neuro–glial pathways involving ATP, cytokines, potassium channels and gap junctions3,33. We identified a previously unrecognized mechanism in which SGCs transfer mitochondria to sensory neurons, protecting against neuropathic pain. We demonstrated TNT-like ultrastructures in mouse and human DRG and established that MYO10-mediated mitochondrial transfer is a key protective pathway. These findings offer mechanistic insight into small fibre neuropathy and highlight mitochondrial transfer as a potential therapeutic strategy. Transgenic male and female mice (8 to 12 weeks of age) were used for behavioural and biochemical experiments. Male and female CD1 mice were purchased from Charles River Laboratories. Generation of Myo10 knockout mice (tm1d) and mouse genotyping were described previously31. Mice were maintained at the Duke animal facility. All mouse experiments were approved by the Duke University Institutional Animal Care and Use Committee (IACUC). Mice were housed in an AAALAC-accredited animal facility under a 12 h:12 h light:dark cycle, with food and water provided ad libitum. All animals were housed at 22 ± 1 °C and 30–70 % humidity. A total of 16 human DRG samples were obtained from donors with and without diabetes (Supplementary Table 1) through the NDRI, under exemption approval from the Duke Institutional Review Board (IRB, Pro00051508). Diabetes was identified on the basis of medical history provided in the NDRI reports. PTX (T7402), Tam (T5648), oligomycin (O4876), carbonyl cyanide 4-(trifluoromethoxy) phenylhydrazone (C2920), rotenone (R8875), antimycin A (A8674), myxothiazol (T5580), CBX (C4790), CytoB (C2743) and Y-27632 (688001) were obtained from Sigma. Mouse primary DRG neuron cultures were prepared as described44. In brief, DRG were digested with collagenase (0.2 mg ml−1, Sigma, 10103578001) and Dispase-II (3 mg ml−1, Sigma, D4693) for 60 min at 37 °C. The cells were then mechanically dissociated with a pipette, filtered through a 70-µm nylon mesh, and centrifuged at 300g for 5 min. The cells were plated on glass coverslips coated with 0.1 mg ml−1 poly-d-lysine (Thermo, A3890401) and cultured in Neurobasal medium supplemented with 10% FBS, 2% B27 supplement, and 1% penicillin-streptomycin (Neurobasal-supplemented medium) at 37 °C with 5% CO2. Mouse DRG SGCs were prepared as previously described15. DRG were enzymatically digested with collagenase and Dispase-II for 90 min at 37 °C. The tissue was then mechanically dissociated using a pipette, filtered through a 70-μm nylon mesh, and centrifuged at 300g for 5 min. Cells were seeded onto 35-mm cell culture dishes (VWR, 10861-656) without poly-d-lysine coating and maintained in Neurobasal-supplemented medium at 37 °C with 5% CO2. On day 3 in vitro (DIV3), non-SGCs—including neurons—were removed by vigorously shaking the culture dishes by hand for 15–20 s, leaving behind a purified adherent layer of SGCs. For in vitro mitochondrial transfer experiments, DIV5 SGCs were labelled with 20 nM MitoTracker Deep Red (Thermo Fisher, M22426) in culture medium at 37 °C with 5% CO2 for 30 min. Cells were then washed three times with PBS. MitoTracker-labelled SGCs were collected using 0.25% Trypsin-EDTA (Gibco, 25200-056) and seeded onto coverslips pre-seeded with cultured DRG neurons. Mitochondrial transfer inhibitors, including the TNT formation inhibitor CytoB (3.5 μM) and Y-27632 (10 μM), the endocytosis inhibitor Pitstop 2 (20 μM) or the gap junction blocker CBX (20 μM), were added to the SGC–neuron co-cultures. After 24 h of incubation, cells were imaged using a Zeiss LSM 880 inverted confocal microscope. After PBS washes, cells were incubated with appropriate fluorescent secondary antibodies for 1 h at room temperature. Fluorescent images were acquired using a Zeiss LSM 880 confocal microscope. Primary mouse DRG neurons were co-cultured with SGCs for 72 h and subsequently fixed with 4% PFA. Fixed cells were subjected to immunocytochemistry using a βIII-tubulin antibody (1:1,000, Abcam, ab18207) overnight at 4 °C. Immunofluorescence images were acquired with a Nikon fluorescence microscope, and neurite length was quantified using the SNT plugin in ImageJ. Postmortem L2–L5 DRG were collected from diabetic and healthy donors and delivered in an ice-cold cell culture medium to the laboratory at Duke University within 24–48 h postmortem. Human DRG cultures were prepared as described45. DRG were digested with collagenase and Dispase-II for 120 to 150 min at 37 °C. Cells were then mechanically dissociated using pipettes and centrifuged (300g for 5 min). Cells were seeded onto poly-d-lysine-coated glass coverslips for neuron culture and into uncoated dishes for SGC culture, and grown in Neurobasal-supplemented medium. On DIV2, non-SGCs were removed by vigorously shaking the culture dishes by hand for 15–20 s. Human SGCs were incubated with 20 nM MitoTracker Deep Red in culture medium at 37 °C with 5% CO2 for 30 min, then co-cultured with human DRG neurons for 24 h. Bone marrow-derived macrophages were prepared as described20. Bone marrow was collected from mice and cultured in DMEM supplemented with 10% heat-inactivated fetal bovine serum, 1% penicillin-streptomycin, and 20% L-929 cell-conditioned medium at 37 °C with 5% CO2. Macrophages or SGCs were seeded on coverslips and labelled with 20 nM MitoTracker Deep Red then co-culture with the whole-mount DRG. CytoB, (3.5 μM) or vehicle was added to medium. Images were captured using a Zeiss LSM 880 inverted confocal microscope. Transfection was performed using Lipofectamine 2000 (Thermo Fisher, 11668027). In brief, siRNA targeting Myo10, MYO10 and control siRNA was diluted in 100 μl Opti-MEM, and 0.4 μl of Lipofectamine 2000 was diluted in a separate 100 μl Opti-MEM solution. Two solutions were combined and incubated at room temperature for 5 min before being added to the culture dish to make a 15 nM siRNA concentration. The culture medium was replaced 6 h after transfection. siRNA sequences are included as follows: siMyo10: sense, CCUACAAGCAGAGUACAAUtt; antisense, AUUGUACUCUGCUUGUAGGtg; siMYO10: sense: GGUAUUCACUUACAAGCAGtt; antisense: CUGCUUGUAAGUGAAUACCtg. Mitochondrial membrane potential was assessed using the JC-1 dye (Invitrogen, T3168), a commonly used indicator of mitochondrial health46. Mouse primary SGCs were incubated with JC-1 (2 μg ml−1) for 30 min at 37 °C. JC-1 accumulates in mitochondria in a membrane potential-dependent manner: monomeric forms emit green fluorescence (excitation 485 nm/emission 516 nm), while aggregated forms emit red fluorescence (excitation 579 nm/emission 599 nm), with higher red signal indicating healthier mitochondria. JC-1 fluorescence was monitored in real time using a VivaView FL Incubator Fluorescence Microscope (Olympus) over 24 h. Differential interference contrast images were automatically captured every 30 min at multiple positions. Mitochondria of SGCs were labelled with 20 nM MitoTracker Green (Thermo, M7514). SIM imaging was performed using an inverted Zeiss Elyra 7 microscope equipped with a 63× oil-immersion objective. SIM2 image processing was conducted using Zeiss Zen Black software. Mitochondrial length was quantified using the Mitochondria Analyzer plugin in ImageJ (v.1.53q). OCR was measured using an XFe96 Seahorse Extracellular Flux Analyzer (Agilent) as described11. In brief, primary DRG neurons and SGCs were seeded in XFe96 cell culture microplates (Agilent, 101085-004) and incubated at 37 °C with 5% CO2 before measurement. On the day of the experiment, cells were washed and placed in Seahorse XF base medium (Agilent, 103575-100) containing 10 mM glucose (Agilent, 103577-100), 1 mM pyruvate (Agilent, 103578-100) and 2 mM glutamine (Agilent, 103579-100). OCR values were normalized to the cell number. Fresh tissue mitochondrial bioenergetics were also measured with the XFe96 Seahorse Extracellular Flux Analyzer as described47. In brief, sciatic nerves or whole-mount DRG were freshly isolated and placed into XFe96 spheroid microplates (Agilent, 102978-100) containing Seahorse XF base medium supplemented with 5.5 mM glucose, 0.5 mM sodium pyruvate and 1 mM glutamine. OCR was normalized to protein content. An assay cycle of 3 min mixing, 3 min waiting, and 4 min measurement was repeated 3 times for baseline rates and after each port injection. DRG primary neurons were cultured from Advillin:GCaMP6f mice. Cells were plated on coverslips precoated with poly-D-lysine (Corning, 354087) and grown in a Neurobasal- supplemented medium. Neurons were co-cultured with SGCs following the previously described procedures. The calcium indicator Fura2-AM was used for calcium imaging for the TRPV1 antagonist experiment. Dissociated mouse DRG neurons were loaded with 5 μM Fura2-AM (Invitrogen, Thermo Fisher Scientific, F1221) for 45 min and then replaced with calcium imaging buffer. Cells were pretreated with CPZ (100 μM)48 for 3 min prior to capsaicin perfusion. This perfusion procedure was applied uniformly across all four experimental groups. Calcium signals were captured and expressed as F values, representing fluorescence intensity49. ROS levels were measured using the ROS Assay Kit (Thermo Fisher, 88-5930-74) following the manufacturer's instructions. Primary DRG neuron cultures were assigned to four experimental groups with overnight treatment: (1) vehicle control; (2) PTX (1 μg ml−1) (3) PTX with SGC co-culture; and (4) PTX with SGC co-culture plus CytoB (3.5 μM). For whole-mount mouse DRG and spinal cord imaging, fresh L4–L5 DRG and lumber spinal cord were collected and cut with Vannas spring scissors (FST, 15019-10) to create a flat surface. DRG and spinal cord were fixed in 3% glutaraldehyde in PBS buffer for 1 h at room temperature after tissue collection. After dehydration with 100% ethanol overnight at 4 °C, the samples were ready for the next preparation step. To examine whether trypsin affects the structure of DRG (Extended Data Fig. 3a,b), DRG tissues were treated with 0.25% trypsin for 20 min at 37 °C, followed by hydrolysis with 8 N HCl at room temperature for 20 min, and then fixed in 3% glutaraldehyde, according to a previously described SEM protocol for DRG21. To improve the visualization of TNTs, mice were transcardially perfused with PBS and followed by 4% PFA, and then L4–L5 DRG were collected and immersed in 30% sucrose for over 3 nights at 4 °C. Subsequently, DRG were sectioned by a cryostat (Leica CM 1950). Alternatively, DRG were directly placed in 3% glutaraldehyde for 1 h at room temperature for fixation, followed by dehydration with 100% ethanol overnight at 4 °C. Sample preparation by the critical point drying (Ladd CPD3), then the DRG and spinal cord samples were sputter-coated with gold for 300 s at 12 mA (Denton Desk V). Imaging was performed using an Apreo 2 scanning electron microscope (ThermoFisher Scientific) with a 2.00 kV accelerating voltage and 25 pA emission current at Duke University Shared Materials Instrumentation Facility. Fresh human DRG (L2–L5) were fixed overnight in 4% paraformaldehyde upon delivery, and then immersed in 30% sucrose for at least 3 nights at 4 °C. Free-floating sections (30 μm) were cut using a cryostat. Some sections were processed for SEM, while adjacent sections were used for immunohistochemistry (IHC). For SEM, imaging was performed using an Apreo 2 scanning electron microscope. Multiple detectors (ETD, T1, T2, and T3) were used. For immunohistochemistry, adjacent sections were blocked with 5% bovine serum albumin (BSA) for 1 h at room temperature, followed by overnight incubation at 4 °C with anti-FABP7 antibody (rabbit, 1:100). The next day, sections were incubated with Alexa Fluor 488-conjugated anti-rabbit secondary antibody (1:400) and Nissl/NeuroTrace-640 for 1 h at room temperature. Sections were mounted using DAPI Fluoromount-G and imaged with a Zeiss LSM 780 confocal microscope. Dissected DRG tissue was fixed in 2.5% glutaraldehyde in 0.1 M sodium cacodylate buffer (pH 7.4) for at least 1 h at room temperature, then stored at 4 °C. Samples were washed with 0.1 M sodium cacodylate buffer, followed by post-fixation in 1% osmium tetroxide in 0.1 M sodium cacodylate buffer (pH 7.2) for 2 h at room temperature. After three additional rinses in buffer, tissues were dehydrated through a graded acetone series and embedded in epoxy resin. Semi-thin sections (500 nm) and ultrathin sections (70–80 nm) were cut using a LEICA EM UC7 ultramicrotome (Leica). Ultrathin sections were collected on copper grids, post-stained with uranyl acetate and lead citrate, and imaged using a JEOL 2100 Transmission Electron Microscope (Duke Center for Electron Microscopy and Nanoscale Technology) at an accelerating voltage of 120 kV. CIPN was induced by intraperitoneal injections of PTX at 2 mg kg−1 every other day for 4 doses. Type 2 DPN was modelled using db/db mice (JAX: 000642). Type 1 diabetes was induced by intraperitoneal administration of STZ (150 mg kg−1; Sigma-Aldrich, S0130)50. SNI surgery was performed as previously described51. In brief, the common peroneal and tibial nerves were ligated and transected, while the sural nerve was left intact. Inflammatory pain was induced by intraplantar injection of 20 μl CFA into the hindpaw52. We previously utilized bio-resorbable PEU films for the controlled release of local anaesthetics in pain management26,53. Bupivacaine-loaded films (40% w/w, ~0.3 mg bupivacaine per film) and blank control films were prepared in the laboratory of M.L.B. Bupivacaine-containing films were wrapped around the ipsilateral sciatic nerve during SNI surgery. MitoTag mice were crossed with Aldh1l1-creERT2, Advillin-cre and Cx3cr1-creERT2 lines. In inducible Cre lines, recombination was induced by Tam administration54. Tam was prepared in a solution of 10% ethanol and 90% corn oil (Sigma, C8267). Mice received intraperitoneal injections of Tam at 100 mg kg−1 once daily for 5 consecutive days. DRG and spinal cord tissues were collected for mitochondrial imaging analysis. Mitochondria were isolated from fresh dissected DRG tissue as described23. In brief, fresh DRG were dissected from Aldh1l1-MitoTag mice. The samples were then subjected to nitrogen cavitation, and after depressurization, nuclei and debris were removed by centrifugation. The supernatant was filtered through a pre-separation filter (130-041-407; Miltenyi Biotec) to obtain the crude mitochondrial fraction. For immunocapture of mitochondria against eGFP, the crude mitochondrial fraction was incubated with GFP beads for 1 h. LS columns (Miltenyi Biotec) were then used to separate GFP microbead-coated mitochondria (immunoprecipitate) and mitochondria without GFP (supernatant) from the solution. For western blot analysis, GFP antibodies (goat, 1:1,000, Abcam, ab5450) were used to examine the eGFP, and anti-COX4 (rabbit, 1:1,000, Abcam, ab16056) was used as a mitochondrial loading control. HEK293T cells (CRL-3216) obtained from ATCC, and authentication was performed by ATCC through morphological and STR profiling. HEK293T cells were not tested for mycoplasma contamination. HEK293T cells were cultured in DMEM medium for transfection. After 72 h, cells were collected and lysed in 4 ml lysis buffer (15 mM NaCl, 5 mM Tris-HCl, pH 8.5) through three freeze-thaw cycles. The supernatant was layered onto a stepwise iodixanol gradient (15%, 25%, 40% and 60%) and centrifuged at 67,000 rpm for 1.5 h at 18 °C using a Beckman Ti-70 rotor. The viral fraction was collected from the 40%–60% interface, and concentrated using a 100 kDa molecular weight cutoff filter (Millipore, UFC910008). Purified viral aliquots were stored at –80 °C until use. AAV-MaCPNS2-Syn-jRGECO1a (3 × 1011 viral genomes per mouse) was administered intraperitoneally to MitoTag mice at postnatal day 0. Four weeks post-injection, mice underwent SNI surgery and received Tam (100 mg kg−1, intraperitoneal injection) once daily for 5 consecutive days. Ex vivo calcium imaging was performed 3–7 days after surgery. L4–L6 DRG were dissected and incubated in artificial cerebrospinal fluid. Live imaging was conducted using a Zeiss LSM 780 upright confocal microscope equipped with a 20× water-immersion objective. Images were acquired with 50 cycles over a 15 min time-lapse session for each DRG. Data were analysed using FIJI (ImageJ) software. siMyo10 and siCtrl were mixed with RVG-9R peptide in 5% glucose at a peptide/siRNA molar ratio of 2:1 before use, following a previously described protocol55. Intra-DRG injection was performed as described previously56. In brief, a partial laminectomy was performed to expose the left L4 and L5 DRG. A total volume of 1 μl containing 4 μg of siRNA was delivered to each DRG using a gelatin sponge as the delivery matrix. For intrathecal drug delivery, antimycin A (110 ng), CytoB (70 ng and 350 ng), or Pitstop2 (120 ng) was diluted in 10 μl of PBS and administered via lumbar puncture using a 30-gauge needle between the L5–L6 vertebral levels. Successful intrathecal delivery was confirmed by observing a characteristic tail-flick response57. Mitochondrial labelling in the DRG was performed based on a previously described protocol with minor modifications58. A total volume of 0.6 μl Mitotracker Deep Red FM (2 μM; Thermo, M22426), with or without CytoB, was injected into the L4–L5 DRG using a Hamilton syringe connected to a glass micropipette. Injected DRG were collected 24 h post-injection, embedded in OCT medium, and sectioned at 20 μm thickness using a cryostat. Immunohistochemistry was then performed on these sections using antibodies against GFAP, along with Nissl staining. SGCs were collected from primary cell cultures, and 8,000 cells suspended in 1 μl PBS were injected intra-ganglionically into the L4 and L5 DRG using a Hamilton syringe connected to a glass micropipette. In select experiments, SGCs were pre-labelled with 20 nM MitoTracker Deep Red FM in culture medium for 30 min at 37 °C, followed by three PBS washes prior to injection. Mitochondria were isolated from SGCs following a previously described protocol59 with minor modifications. Mitochondria isolated from 10,000 SGCs in 1 μl of PBS were injected intra-ganglionically into the L4 and L5 DRG using a Hamilton syringe and glass micropipette. In select experiments, to inhibit mitochondrial oxidative phosphorylation, SGCs were pretreated with 1 mM myxothiazol (Sigma, T5580) for 10 min prior to mitochondrial isolation. For von Frey testing, mechanical sensitivity was assessed with a set of von Frey filaments (Stoelting) with logarithmically increasing stiffness ranging from 0.02 g to 2.56 g, which were applied to the hindpaw plantar surface, and a quick withdrawal or licking response to the stimulus was considered a positive response. PWT was calculated using the up-down method. For assessing mechanical allodynia, paw withdrawal frequency (PWF) to a 0.16 g von Frey filament was measured by observing reflexive withdrawal responses within 10 stimulations, and a quick withdrawal or licking response to the stimulus was considered a positive response. Hargreaves test was used to assess heat sensitivity. Paw withdrawal latency was measured with a Hargreaves radiant heat apparatus (IITC Life Science). CPP assay was used to measure ongoing pain in mice44. Mice were habituated for 3 days with 30 min of preconditioning in a two-compartment CPP chamber consisting of white and dark chambers. The baseline behaviour of the mice was recorded on the fourth day using a camera and automatically tracked for 15 min with ANY-Maze software (Stoelting). On the fifth day (conditioning day), mice underwent a 30 min pairing session without intra-DRG injection in the preferred CPP chamber in the morning session, followed by a 30 min pairing session with intra-DRG injection in the non-preferred CPP chamber in the afternoon session. Twenty-four hours later, mice were placed in the CPP test box with access to both chambers, and their behaviour was recorded for 15 min. The time spent in both chambers was analysed using ANY-Maze software. Following terminal anaesthesia with isoflurane, mice were transcardially perfused with PBS, followed by 20 ml of 4% PFA in PBS. The DRG, spinal nerves and hindpaw skins were collected, fixed in 4% PFA and immersed in 30% sucrose at 4 °C for at least 3 nights. Tissues were then embedded in OCT medium (Tissue-Tek), and sections were prepared using a cryostat at the following thicknesses: DRG (20 μm), spinal cord (30 μm), spinal nerve (20 μm), and skin tissue (25 μm). Tissue sections were then blocked in a solution containing 5% BSA and 0.3% Triton X-100 for 1 h at room temperature. Following several washes with PBS, the sections were incubated with species-specific secondary antibodies and Nissl/NeuroTracer-640 (1:100, Thermo, N21483). Finally, the sections were mounted with coverslips using DAPI Fluoromount-G mounting medium (Southern Biotech, 0100-20). Images were acquired using a Zeiss LSM 780 confocal microscope. For quantification of immunofluorescence staining, the same acquisition settings were applied to images requiring comparison under different conditions. Three sections were randomly selected from each mouse, and the integrated density of the fluorescence signal per section was measured using ImageJ software (v.1.53q). For intraepidermal nerve IENF analysis in mouse hindpaw skins, all ascending nerve fibre branches crossing into the epidermis were counted. Three randomly selected skin sections were analysed for each mouse. Fresh human DRG were immediately fixed in 4% paraformaldehyde overnight at 4 °C upon delivery. Free-floating sections (30 µm) were then cut using a cryostat. For MYO10 in situ hybridization, a human MYO10 RNAscope probe (RNAscope Probe Hs-MYO10, 440691), designed by Advanced Cell Diagnostics was used. Following the RNAscope steps, immunohistochemistry was performed as described below. The sections were blocked with 5% BSA for 1 h at room temperature and then incubated with anti-FABP7 antibody (rabbit, 1:100, Thermo, PA5-24949) overnight at 4 °C. Images were captured using a Zeiss LSM 780 confocal microscope with consistent acquisition settings across different samples. The integrated density of the fluorescence signal per section was measured using Image J (v.1.53q). Human DRG tissues were snap-frozen and stored at –80 °C. Nuclei isolation was performed as previously described60. Single-nucleus capture was performed using the 10x Genomics Chromium Single Cell 3′ system (v.3.1). Libraries from individual nuclei samples were pooled and sequenced on an MGISEQ-2000 platform. snRNA-seq reads were processed using Cell Ranger v.4.0.061, and alignment to the Human GRCh38 (GENCODE v.32/Ensembl98) reference genome performed using default parameters. Downstream analysis was conducted with Seurat v.4.3.062 using R v.4.3.0. Cells were filtered out if they expressed fewer than 200 genes or had more than 10% mitochondrial reads. Gene expression was then normalized and scaled using the NormalizeData and ScaleData functions from Seurat, respectively, with default settings, as well as the FindVariableFeatures function to identify the highly variable genes. Principal component analysis was carried out, and downstream analyses were based on the top 20 principal components. To address batch effects, datasets were integrated, re-normalized, scaled, and batch-corrected using Harmony v.0.1.163. Unsupervised clustering was done using the FindClusters function from Seurat with a resolution of 0.8. FindMarkers with default settings to get gene markers, and cell types within each cluster were annotated based on known marker genes and genes that were differentially expressed within each cluster. Statistical analyses were completed with Prism GraphPad 8.4. The sample sizes were based on our previous studies44,45. Each data point corresponds to an individual animal. All data were included in the analyses (no outliers removed). Data were analysed using unpaired t-test (two-sided) or Mann–Whitney test (two-sided) for comparison between two groups, one-way ANOVA followed by Tukey post hoc test, and two-way ANOVA followed by Sidak's multiple comparisons test for two groups and Tukey's multiple comparisons test for more than two groups. A significance level of P < 0.05 was considered statistically significant. Additional statistical details are included in Supplementary Table 4. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data reported in this paper will be shared by the corresponding author upon request. Any additional information required to reanalyse the data reported in this paper is available from the corresponding author upon request. Source data are provided with this paper. Human DRG single-nucleus RNA sequencing (snRNA-seq) data have been deposited in Gene Expression Omnibus (GEO) under accession number GEO: GSE 310000. Eid, S. A. et al. New perspectives in diabetic neuropathy. Canta, A., Pozzi, E. & Carozzi, V. A. Mitochondrial dysfunction in chemotherapy-induced peripheral neuropathy (CIPN). Hanani, M. & Spray, D. C. Emerging importance of satellite glia in nervous system function and dysfunction. Berta, T., Qadri, Y., Tan, P. H. & Ji, R. R. Targeting dorsal root ganglia and primary sensory neurons for the treatment of chronic pain. & Caterina, M. J. Mechanisms of sensory transduction in the skin. & Stojanovski, D. Mitochondria-hubs for regulating cellular biochemistry: emerging concepts and networks. Rustom, A., Saffrich, R., Markovic, I., Walther, P. & Gerdes, H. H. Nanotubular highways for intercellular organelle transport. Borcherding, N. & Brestoff, J. R. The power and potential of mitochondria transfer. Scheiblich, H. et al. Microglia jointly degrade fibrillar alpha-synuclein cargo by distribution through tunneling nanotubes. Lin, R. Z. et al. Mitochondrial transfer mediates endothelial cell engraftment through mitophagy. Baldwin, J. G. et al. Intercellular nanotube-mediated mitochondrial transfer enhances T cell metabolic fitness and antitumor efficacy. Transfer of mitochondria from astrocytes to neurons after stroke. Selective block of tunneling nanotube (TNT) formation inhibits intercellular organelle transfer between PC12 cells. Sun, C. et al. Endocytosis-mediated mitochondrial transplantation: transferring normal human astrocytic mitochondria into glioma cells rescues aerobic respiration and enhances radiosensitivity. Alarcon-Martinez, L. et al. Interpericyte tunnelling nanotubes regulate neurovascular coupling. Reichert, D. et al. Tunneling nanotubes mediate the transfer of stem cell marker CD133 between hematopoietic progenitor cells. Neef, J. et al. Modes and regulation of endocytic membrane retrieval in mouse auditory hair cells. van der Vlist, M. et al. Macrophages transfer mitochondria to sensory neurons to resolve inflammatory pain. Matsuda, S. et al. Phylogenetic investigation of Dogiel's pericellular nests and Cajal's initial glomeruli in the dorsal root ganglion. & van der Bliek, A. M. Mitochondrial fission, fusion, and stress. Fecher, C. et al. Cell-type-specific profiling of brain mitochondria reveals functional and molecular diversity. Avraham, O. Profiling sensory neuron microenvironment after peripheral and central axon injury reveals key pathways for neural repair. Zheng, Q. et al. Synchronized cluster firing, a distinct form of sensory neuron activation, drives spontaneous pain. Controlled release of etoricoxib from poly(ester urea) films for post-operative pain management. Dana, H. Sensitive red protein calcium indicators for imaging neural activity. Levine, T. D. Small fiber neuropathy: disease classification beyond pain and burning. Sartori-Rupp, A. et al. Correlative cryo-electron microscopy reveals the structure of TNTs in neuronal cells. Yang, L. et al. Human and mouse trigeminal ganglia cell atlas implicates multiple cell types in migraine. Heimsath, E. G. Jr, Yim, Y. I., Mustapha, M., Hammer, J. & Cheney, R. E. Myosin-X knockout is semi-lethal and demonstrates that myosin-X functions in neural tube closure, pigmentation, hyaloid vasculature regression, and filopodia formation. Expression and role of voltage-gated sodium channels in human dorsal root ganglion neurons with special focus on Nav1.7, species differences, and regulation by paclitaxel. Bang, S. Satellite glial GPR37L1 and its ligand maresin 1 regulate potassium channel signaling and pain homeostasis. Avraham, O. et al. Profiling the molecular signature of satellite glial cells at the single cell level reveals high similarities between rodents and humans. Mogil, J. S. Animal models of pain: progress and challenges. Brestoff, J. R. et al. Intercellular mitochondria transfer to macrophages regulates white adipose tissue homeostasis and is impaired in obesity. Dong, L. F. et al. Horizontal transfer of whole mitochondria restores tumorigenic potential in mitochondrial DNA-deficient cancer cells. Rebbeck, C. A., Leroi, A. M. & Burt, A. Mitochondrial capture by a transmissible cancer. Brown, T. J., Sedhom, R. & Gupta, A. Chemotherapy-induced peripheral neuropathy. Divisova, S. et al. Intraepidermal nerve-fibre density as a biomarker of the course of neuropathy in patients with type 2 diabetes mellitus. Uceyler, N. et al. Small fibre pathology in patients with fibromyalgia syndrome. Murphy, M. P. & Hartley, R. C. Mitochondria as a therapeutic target for common pathologies. Yoon, Y. G., Haug, C. L. & Koob, M. D. Interspecies mitochondrial fusion between mouse and human mitochondria is rapid and efficient. Donnelly, C. R. et al. STING controls nociception via type I interferon signalling in sensory neurons. Chen, G. et al. PD-L1 inhibits acute and chronic pain by suppressing nociceptive neuron activity via PD-1. Reers, M., Smith, T. W. & Chen, L. B. J-aggregate formation of a carbocyanine as a quantitative fluorescent indicator of membrane potential. Krukowski, K. et al. HDAC6 inhibition effectively reverses chemotherapy-induced peripheral neuropathy. Zhang, F., Challapalli, S. C. & Smith, P. J. Cannabinoid CB1 receptor activation stimulates neurite outgrowth and inhibits capsaicin-induced Ca2+ influx in an in vitro model of diabetic neuropathy. Luo, X. et al. IL-23/IL-17A/TRPV1 axis produces mechanical pain via macrophage-sensory neuron crosstalk in female mice. Single dose streptozotocin-induced diabetes: considerations for study design in islet transplantation models. & Woolf, C. J. Spared nerve injury: an animal model of persistent peripheral neuropathic pain. Chen, G. et al. Hevin/Sparcl1 drives pathological pain through spinal cord astrocyte and NMDA receptor signaling. Dziewior, C., Godwin, K., Judge, N., Dreger, N. & Becker, M. Poly(ester urea)s: synthesis, material properties, and biomedical applications. Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Kumar, P. et al. Transvascular delivery of small interfering RNA to the central nervous system. Xu, J. et al. Oxidative stress induced by NOX2 contributes to neuropathic pain via plasma membrane translocation of PKCepsilon in rat dorsal root ganglion neurons. Xu, J., Yan, Z., Bang, S., Velmeshev, D. & Ji, R. R. GPR37L1 identifies spinal cord astrocytes and protects neuropathic pain after nerve injury. Restoring cellular energetics promotes axonal regeneration and functional recovery after spinal cord injury. Velmeshev, D. et al. Single-cell analysis of prenatal and postnatal human cortical development. Satpathy, A. T. et al. Massively parallel single-cell chromatin landscapes of human immune cell development and intratumoral T cell exhaustion. Stuart, T. et al. Comprehensive integration of single-cell data. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. This study was supported by Duke University Anesthesiology Research Funds, NIH grants R01NS13181201A1 and R61NS138215 and Department of Defense (DoD) grants W81XWH-21-1-0885, W81XWH-21-1-0756, W81XWH-22-1-0267 and W81XWH-22-1-0646 awarded to R.-R.J. M.L.B. was partially supported by DoD grant W81XWH-22-1-0645. was supported by a Paul and Daisy Soros Fellowship and an HHMI Gilliam Fellowship. Fox Foundation (MJFF) and the Aligning Science Across Parkinson's (ASAP) initiative. on behalf of ASAP and the Michael J. is also a Howard Hughes Medical Institute (HHMI) Investigator. received support from Duke University Neurobiology Research Funds and NIH grant R00MH121534. We thank the Duke University Light Microscopy Core Facility for assistance with confocal microscopy and super-resolution microscopy, the Duke University Shared Materials Instrumentation Facility for support with SEM, and the Center for Electron Microscopy and Nanoscale Technology in the Duke Department of Pathology for help with TEM. The Zeiss ELYRA7 super-resolution microscopy in Duke University Light Microscopy Core Facility was funded by NIH Shared Instrumentation Grant 1S10OD28703-01. Cartoon elements were created using BioRender under a license agreement. Center for Translational Pain Medicine, Department of Anesthesiology, Duke University Medical Center, Durham, NC, USA Jing Xu, Yize Li, Charles Novak, Sangsu Bang, Aidan McGinnis, Sharat Chandra, Vivian Zhang, Wei He & Ru-Rong Ji Department of Computational Biology, School of Computer Science, Carnegie Mellon University, Pittsburgh, PA, USA Department of Cell Biology, Duke University Medical Center, Durham, NC, USA Min Lee, Terry Lechler, Maria Pia Rodriguez Salazar, Cagla Eroglu & Ru-Rong Ji Department of Neurobiology, Duke University Medical Center, Durham, NC, USA Zihan Yan, Cagla Eroglu, Dmitry Velmeshev & Ru-Rong Ji Department of Dermatology, Duke University Medical Center, Durham, NC, USA Aligning Science Across Parkinson's (ASAP) Collaborative Research Network, Chevy Chase, MD, USA Department of Chemistry, Duke University, Durham, NC, USA Thomas Lord Department of Mechanical Engineering and Materials Science, Duke University, Durham, NC, USA Department of Cell Biology and Physiology, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA Department of Integrative Immunology, Duke University Medical Center, Durham, NC, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar analysed snRNA-seq data of human DRG. contributed to the experiment of AAV design and packaging. contributed to mitochondrial isolation and immunocapture assay experiments in MitoTag mice. provided Myo10tm1d mice and expertise on MYO10 experiments. contributed to the nerve blockade experiment with bupivacaine-loaded films. All authors reviewed the manuscript and provided input. The authors declare no competing interests Nature thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Immunostaining FABP7, Kir4.1, AQP4, CX43, and GS in mouse SGC cultures, along with quantification of the percentage of marker-positive cells. (b) Triple staining showing mitochondrial (red arrow) transfer from a FABP7-labeled SGC to a Trpv1+ neuron. (c-d) Schematic (c) and representative live-cell imaging (d) of TNT formation between SGCs and neurons. Top panels in d: MitoTracker labeling. Red arrows indicate mitochondria within a TNT, and white arrows indicate TNT breakage. Bottom panels in d: contrast images showing a broken TNT (white arrow). (e-f) Schematic (e) and representative images (f) showing mitochondrial transfer from SGCs to neurons in vehicle- and TTX-treated groups. (g) Quantification of MitoTracker signal density in neurons interacting with SGCs. (i) Top, schematic of SGC and neuron co-cultures treated with different inhibitors. Bottom, representative MitoTracker images showing inhibitory effects of CytoB, Y-27632, Pitstop 2, and CBX on mitochondrial transfer from SGCs to neurons. (j) Quantification of MitoTracker signal density in neurons interacting with SGCs in 5 different groups. n = 41 (vehicle), 33 (CytoB), 32 (Y-27532), 30 (Pitstop2), and 37 (CBX) over 4 independent experiments each group. (k) Percentage of TNT-positive neurons showing inhibitory effects of CytoB, Y-27632, and CBX on TNT formation. Sample sizes are as indicated in j. Unpaired t-test (two-sided, g), one-way ANOVA followed by Tukey's multiple comparisons test (j). (a) Schematic of the ex vivo co-culture setup: SGCs were labeled with MitoTracker and co-cultured with half-cut whole-mount DRG tissue. (b) Low-magnification image of MitoTracker fluorescence showing strong signals in DRG cells, TNTs, and SGCs. Notably, SGC-derived TNTs extend to the edge of the DRG (TNT-4/5, white arrow) and penetrate deep into the tissue (TNT-1 to −3, white arrows). The DRG boundary is outlined by a dotted line. (c) High-magnification image (enlarged from boxed area in b) showing TNTs (TNT-1 to −3, white arrows) between SGCs and neurons (blue arrows) within the DRG. (d) Schematic of co-cultures using primary SGC or macrophage cultures and whole mount DRG from Trpv1:Ai9 reporter mice. (e) Top panels: representative MitoTracker (red) and Ai9 (green) fluorescence images of DRG co-cultures with SGCs without (left) and with CytoB treatment (middle), and with macrophages (right). DRG boundary is marked with a dotted line. Bottom panels: high-magnification images (enlarged from boxed area in top panels) showing mitochondrial transfer to Trpv1+ neurons and the inhibitory effect of CytoB (middle). Notably, TNT-like structures were observed between SGCs and neurons (left and middle, indicated by arrows), but not between macrophages and neurons (right). In the left panels, strong red fluorescence obscures TNT visibility. (f) Quantification of MitoTracker signal density in neurons interacting with SGCs, demonstrating the effect of CytoB. (g) Comparison of mitochondrial transfer to DRG neurons from SGCs versus macrophages. Mice from both sexes were included. (a) Schematic of the trypsin-treated whole mount DRG preparation and SEM imaging. (b) Representative SEM images showing destruction of the tube structures in DRG after trypsin treatment. (c) Schematic for the paclitaxel (PTX) model using intraperitoneal injections (2 mg/kg, every other day, four injections). (d) Representative SEM images showing gap between SGCs and neurons in the DRG at 1-, 2-, and 4-weeks post-paclitaxel treatment. “N” indicates sensory neuron; “S” indicates SGC; and arrow points to TNT-like structures; gap between the SGC and the neuron is also indicated. (f) Quantification of TNT-like structures in the gaps between SGCs and neurons showing a significant increase at 4 weeks compared to 1-week post-PTX. (g-h) Representative SIM images of mitochondria before (g) and 15 min after treatment with 1 μg/mL PTX (h). Right panels showing enlarged views of the boxed areas from the left panels. (i) PTX treatment increases the short-length mitochondria in SGCs. Data are shown as means ± s.e.m and analyzed by one-way ANOVA followed by Tukey's multiple comparisons test (e and f). Mice from both sexes were included. (a) Low magnification TEM image showing DRG neurons and SGCs. (b) Enlarged image (from the boxed area in a), showing a TNT-like structure (blue dotted line). (c) Enlarged image (from the boxed area in b), showing a tube containing mitochondria (red arrows) and vesicles (orange arrows), as well as ER from adjacent SGC and neuron. Black and green asterisks (*) indicate the nuclei of SGCs and neurons, respectively; ER: endoplasmic reticulum. (b) Western blot showing the expression of GFP and mitochondrial marker COX4 in crude mitochondrial fraction (CMF), immunocapture (IP), and supernatant (Sup) groups. (c) DRG fluorescence image of Aldh1l1:MitoTag mice without tamoxifen injection. (d) Colocalization of MitoTag fluorescence signal with mitochondrial marker TOM20 in DRG section. (e) Schematic of experimental design of tamoxifen (Tam) treatment (5 daily injections) with DRG tissue collection on day 10, and additional treatment with Pitstop2 and streptozotocin (STZ). (f) Representative DRG images of MitoTag green fluorescence. n = 4 biological repeats in control, Pitstop2, STZ-5d groups, and n = 3 biological repeats in STZ-15d group. (g) Schematic of experimental design of tamoxifen (Tam) treatment with DRG tissue collection on day 5 and additional treatment with complete Freund's adjuvant (CFA). (h) Representative DRG images of MitoTag green fluorescence. 2b, c. (i) Schematic of pain behavioral testing before (baseline, BL) and after spared nerve injury (SNI) surgery combined with peri-sciatic nerve implantation of bupivacaine-loaded film in Aldh1l1:MitoTag mice. (j) von Frey testing showing paw withdrawal threshold. (a) Left: Timeline of tamoxifen treatment (5 daily injections) followed by spinal cord tissue collection at day 0, day 5, and day 10 from Aldh1l1:MitoTag mice. Right: MitoTag fluorescence imaging showing no signal at day 0 without tamoxifen treatment. (b-c) In the spinal cord dorsal horn, MitoTag signal is only present in astrocytes (GFAP+) but not in neurons (NeuN+) 5 days (b) and 10 days (c) after tamoxifen treatment. Notably, no mitochondrial transfer was observed in neurons (NeuN+). (d-e) SEM imaging on the L4-L5 segment of mouse spinal cord dorsal horn. (d) Schematic of tissue preparation for SEM imaging. (e) Left: low-magnification SEM image showing dorsal root, dorsal horn (grey matter), and white matter. Dotted curve line indicates the dorsal horn boundary. Middle: enlarged boxed area from the left panel. Right: enlarged boxed area from the middle panel. (a) Schematic of crossing Advillin-cre and MitoTag mice. (b) Images of DRG sections showing Advillin:MitoTag (green) and FABP7 (red) fluorescence signal. Red arrows indicate colocalization of MitoTag and FABP7; blue arrows indicate FABP7 without MitoTag. (c) DRG section images showing Advillin:MitoTag fluorescence signal (green) and Iba1 staining (red). Red arrows indicate colocalization of MitoTag and Iba1, and blue arrows indicate macrophages (Iba1+) without MitoTag. (d) Diagram of crossing Cx3cr1-cre/ERT2 and MitoTag mice, along with the timeline of tamoxifen injection and DRG collection at day 5 and day 10. Red arrows indicate colocalization of MitoTag and Iba1 in macrophages, and blue arrows indicate neurons without MitoTag. n = 3 biological repeats (b, c, e, and f). (a) Representative images of calcium signal (jRGECO1a+, red) and mitochondrial signal (MitoTag+, green) in DRG neurons of Aldh1l1-Mito mice with tamoxifen (Tam) treatment for 5 days, followed by tissue collection on day 5. Left, low-magnification image showing both red and green channels. The boxed area was enlarged in the right panels for single red or green channel. (b-c) Heatmap of MitoTag- and jRGECO1a-positive neurons from (a) with MitoTag (green) and jRGECO1a (red) fluorescence signal. (d) Lack of MitoTag and jRGECO1a co-expression in 20 MitoTag+ neurons and 81 jGECO1a+ neurons in naïve mice without nerve injury. (e) Cell size frequency distribution of total DRG neurons (n = 12 DRG from 4 mice), MitoTag+ neurons (n = 9 DRG from 3 mice), and jRGECO1a+ neurons (n = 9 DRG from 3 mice) 5 days after SNI surgery. (f) Representative images of calcium indicator jRGECO1a in DRG of Aldh1l1:MitoTag mice following KCl treatment (60 mM, 1 min). (g) Size frequency distribution of the jRGECO1a+ neurons in DRG of naïve mice with KCl treatment, demonstrating extensive activation in neurons with various diameters. Sample sizes are indicated by individual dots inside columns. (a) Triple staining (left) of MYO10 (red), FABP7 (green), and DAPI (blue) and double staining (right) showing co-localization of MYO10 and FABP7 in SGCs of mouse DRG. (b-c) Altered mechanical pain sensitivity following siRNA knockdown of Myo10 in naïve mice. (b) Schematic of intra-DRG injection of si-RNA (Myo10, 4 μg in 1 μl) and associated pain behavior testing. (c) von Frey test showing decreased paw withdrawal threshold 3 days after Myo10-targeting siRNA treatment. (e-f) SEM images of DRG from control mice (e) and mice with si-RNA knockdown of Myo10 (f), showing a loss of TNT-like structures after Myo10 knockdown. (g-i) Behavioral test in WT and Myo10+/− mice. (h) Paw withdrawal frequency to 0.16 g von Frey filament, showing heightened response in Myo10+/− mice (n = 6) compared to WT mice (n = 6). (i) Hargreaves test showing the decreased paw withdrawal latency in Myo10+/− mice (n = 6) compared to WT mice (n = 6). Unpaired t-test (two-sided, h and i). (a-c) CytoB treatment blocks mitochondrial transfer to mouse neurons in vivo. (a-b) Representative images of vehicle-treated (a) and CytoB-treated (b) mice 1 day following DRG microinjection of MitoTracker dye. (a) DRG images from vehicle-treated mice and the box is enlarged in right panels showing MitoTag, GFAP, Nissl, and merged images. (b) DRG images from CytoB-treated mice and the box is enlarged in right panels showing MitoTag, GFAP, Nissl, and merged images. Notably, MitoTag labeling in neurons is blocked by CytoB. (d-f) CytoB treatment blocks mitochondrial transfer to healthy mouse neurons in vitro. (d) Left panel: schematic of DRG neuron-SGC co-cultures under 3 different conditions: 1) no treatment, 2) SGCs treated with PTX (1 μg/ml, 1 h), 3) SGCs treated with both PTX and CytoB (3.5 μM, 24 h). Right panels: SGC-neuron co-culture images of bright-field (grey color) and JC-1 aggregate signal (red color) under different conditions as indicated. (e) Quantification of JC-1 aggregate signal density in co-cultures. (f) Oxygen consumption rate (OCR) showing an inhibitory effect of CytoB treatment (3.5 μM) in DRG SGC-neuron co-cultures. Mice from both sexes were included. (a) Calcium imaging of DRG neurons co-cultured with SGCs from Advillin:GCamp6f mice before (baseline) and after capsaicin perfusion (100 nM, 2 min), demonstrating the effects of PTX (1 μg/mL) and CytoB (3.5 μM). (b-c) Traces of calcium responses to capsaicin and KCl (b), and Amplitudes of capsaicin-induced calcium responses, showing the effects of paclitaxel and CytoB (c) in dissociated DRG neurons under four indicated conditions. (d) Reactive oxygen species (ROS) assay in DRG SGC-neuron co-cultures showing the effects of paclitaxel and CytoB. (e-f) TRPV1 antagonist capsazepine (CPZ) blocks PTX-induced calcium response in DRG neurons. (e) Traces of calcium responses to capsaicin (100 nM, 2 min) in dissociated DRG neurons with PTX and CPZ (100 μM) treatment in 4 different groups. and analyzed by One-way ANOVA followed by Tukey's multiple comparisons test (c, d, and f). Mice from both sexes were included. (c) PTX treatment decreased OCR in cultured SGCs. (d) Mitochondrial fluorescence signal in the spinal nerve after adoptive transfer of SGCs. (e) Quantification of integrated density of MitoTracker fluorescence in vehicle (n = 6) and PTX-treated (n = 5) group. (f) Left, schematic of neuron-SGC co-cultures. Right, βIII-tubulin immunostaining showing axonal outgrowth of neurons. (g) Quantification of axonal outgrowth in neuron-SGC co-cultures. n = 15 neurons per group from three independent experiments. (i) Paw withdrawal threshold decreased after CytoB and Antimycin A treatment. (k) Representative images of PGP9.5 staining (k) and quantification of IENF density (l) in naïve and PTX-treated group. and were statistically analyzed by unpaired t-test (two-sided, c, e, and l), One-way ANOVA followed by Tukey's multiple comparisons test (g and j), and Two-way ANOVA followed by Tukey's multiple comparisons test (i). (a) Schematic of non-diabetic human donors for whole mount DRG experiment and SEM imaging. Middle, enlarged box area from the left. Right, enlarged boxed area from the middle. (c) Schematic of non-diabetic human donors for sectioned DRG experiment and SEM imaging. (d) low magnification image of N1 and N2 neurons. (e) Left, enlarged boxed area of d, showing multiple SGCs (S1 to S9). Right, enlarged boxed area of the left, showing TNT-like structure with a bulge (arrows) near S1. (f) Schematic of diabetic human donors for sectioned DRG experiment and SEM imaging. (g) Left, SEM image of a neuron (N) and surrounding SGCs (S1 to S7). Right: enlarged boxed area from the left, showing micrometer gap between SGCs and neurons and an unsmooth TNT-LS (indicated by a blue arrow). (h) A neuron is surrounded by multi-layers of non-neuronal cells (presumably SGCs) in diabetic DRG (indicated by black circles). (c) UMAP plots reveal seven neuronal populations including: 1) neurofilament population (FXYD7+), 2) Aβ-LTMR (HS3ST4 + ), 3) proprioceptors (NXPH1+), 4) peptidergic nociceptors (GAL + ), 5) C-fiber low threshold mechanoreceptor (GFRA2+/ and POU4F2+), 6) somatostatin-positive pruriceptors (SST+), and 7) cold-sensing TRPM8+). (d-f) UMAP plots showing Schwann cell markers MPZ and GLDN (d), connective tissue marker COL1A1 (e), and endothelial cell marker PECAM1 (f). (a) Schematic of two different mitochondrial transfer approaches between SGCs and neurons: 1) MitoTracker labeled SGCs and co-culture with neurons and 2) MitoTracker labeled neurons and co-culture with SGCs. (b-c) Representative images of MitoTracker labeled mitochondrial transfer from SGC to neuron (b) and from neuron to SGC (c). White arrows point to TNTs, and red arrows point to mitochondria within TNTs. (d) Quantification of mitochondrial transfer efficiency as the receiver/donor ratio of MitoTracker signal intensity in two groups. **** P < 0.0001. n = 21 cells (left column) and n = 25 cells (right column) from 2 to 4 independent experiments. (e) Percentage of MitoTracker-positive and TNT-positive neurons of the SGC-neuron co-cultures from human DRG. A total of 54 neurons from 6 independent experiments were included for quantification. (a-f) Mitochondrial uptake by DRG cells following intra-DRG injection of MitoTracker labeled mitochondria. (a) Schematic of mitochondrial isolation from MitoTracker labeled SGCs and intra-DRG injection. (b-e) Low magnification DRG images showing triple staining of MitoTracker labeled mitochondria (red), FABP7-labeled SGCs (green), and NeuN-labeled neurons (blue) from untreated mice (b-c), CytoB (3.5 μM)-treated mice (d), and Pitstop2 (25 μM)-treated mice (e). (c) Enlarged image from the box in b showing adoptively transferred mitochondria in both NeuN+ neuron (blue arrow) and FABP7+ SGCs (white arrows). Bottom panels in (d-e) are enlarged images from the boxed areas in top panels. (f) Mito density quantification shows that MitoTracker+ mitochondrial uptake in DRG neurons and SGCs is blocked by Pitstop2, not by CytoB. (g-j) Paclitaxel-induced evoked pain and spontaneous pain and the effects of mitochondrial transfer. (g) Schematic of mitochondrial isolation from mouse SGCs, treatment with the mitochondrial complex III inhibitor myxothiazol (1 mM, 10 min), and intra-DRG injection in the PTX model. (h) Analgesic effects of adoptively transferred mitochondria and its blockade by myxothiazol. (i-j) Assessment of spontaneous pain with conditioned place preference test (CPP) scores. (k) OCR analysis of SGC cultures of human DRG from diabetic and non-diabetic donors, showing mitochondrial dysfunction in diabetes. and were statistically analyzed by One-way ANOVA followed by Tukey's multiple comparisons test (f), Two-way ANOVA followed by Sidak's multiple comparisons test (h), and unpaired t-tests (two-sided, j and k). Mice from both sexes were included for analysis. Middle, disruption of SGC-neuron mitochondrial transfer in peripheral neuropathy-associated diseases, such as chemotherapy-induced peripheral neuropathy (CIPN) and diabetic peripheral neuropathy (DPN), induces nerve degeneration and neuropathic pain. Enriched genes in SGCs cluster 1 and 2 of human DRG snRNA-seq data. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Xu, J., Li, Y., Novak, C. et al. Mitochondrial transfer from glia to neurons protects against peripheral neuropathy. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09852-9'>RNA-triggered Cas12a3 cleaves tRNA tails to execute bacterial immunity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 16:15:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In all domains of life, tRNAs mediate the transfer of genetic information from mRNAs to proteins. As their depletion suppresses translation and, consequently, viral replication, tRNAs represent long-standing and increasingly recognized targets of innate immunity1,2,3,4,5. Here we report Cas12a3 effector nucleases from type V CRISPR–Cas adaptive immune systems in bacteria that preferentially cleave tRNAs after recognition of target RNA. Cas12a3 orthologues belong to one of two previously unreported nuclease clades that exhibit RNA-mediated cleavage of non-target RNA, and are distinct from all other known type V systems. Through cell-based and biochemical assays and direct RNA sequencing, we demonstrate that recognition of a complementary target RNA by the CRISPR RNA triggers Cas12a3 to cleave the conserved 5′-CCA-3′ tail of diverse tRNAs to drive growth arrest and anti-phage defence. Cryogenic electron microscopy structures further revealed a distinct tRNA-loading domain that positions the tRNA tail in the RuvC active site of the nuclease. By designing synthetic reporters that mimic the tRNA acceptor stem and tail, we expanded the capacity of current CRISPR-based diagnostics for multiplexed RNA detection. Overall, these findings reveal widespread tRNA inactivation as a previously unrecognized CRISPR-based immune strategy that broadens the application space of the existing CRISPR toolbox. Immune defences across all domains of life counteract viral infections by clearing the invader or disabling host processes that are essential for viral replication. One growing theme associated with innate immune systems is the inactivation of tRNAs1,2,3,4,5. tRNAs have a critical role in translation, serving as the bridge between mRNAs and nascent proteins. Accordingly, inactivating a portion of the tRNA pool can impair the synthesis of viral proteins or drive systematic cellular shutdown to block viral replication1,2,4,6,7. Representative bacterial defences such as PrrC, VapC, colicin E5 and PARIS cleave the anticodon loop of specific tRNAs5,8,9,10,11. In animals, the type I interferons SLFN11 and SAMD9 cleave the acceptor stem and anticodon loop, respectively, of tRNAs to suppress codon-specific production of viral particles4,7,12,13. Conspicuously absent from the set of immune defences that specifically use tRNA inactivation are CRISPR–Cas systems, the only known source of adaptive immunity in bacteria and archaea14. These widespread systems immunize against future infection by acquiring snippets of viral sequences expressed as CRISPR RNAs (crRNAs) that pair with CRISPR-associated (Cas) effector nucleases. The complex then searches for complementary target RNA or DNA that match the originating virus and, after target recognition, cleaves the bound nucleic acid targets to clear viral genomes or transcripts15,16,17. Some activated Cas nucleases also collaterally cleave non-target RNA or DNA with little sequence preference to halt cellular processes that are essential for viral replication and to drive growth arrest18,19,20,21. One such effector, the RNA-triggered Cas13 nuclease from Leptotrichia shahii (LshCas13a), was recently shown to cleave U-rich anticodon loops associated with a subset of tRNAs when activated in Escherichia coli22. However, LshCas13a also efficiently cleaves its target RNA at U-rich sequences to drive targeted silencing23,24,25,26. Thus, it remains unknown whether any CRISPR–Cas systems have evolved to preferentially cleave tRNAs over other RNA species, including their target RNA, as part of an immune response. Here we report a previously uncharacterized clade of Cas nucleases, which we term Cas12a3. After target RNA recognition, these nucleases preferentially cleave the conserved 3′ CCA tails of tRNAs to drive growth arrest and block phage dissemination. Activated Cas12a3 engages the tRNA tail, acceptor stem and T-arm to load tRNA substrates into its RuvC nuclease domain for cleavage. Harnessing the distinct properties of Cas12a3, we expanded the multiplexing capacity of current CRISPR-based RNA detection approaches to illustrate one of the many applications of programmable RNA-mediated tRNA cleavage. The phylogenetic proximity of the functionally diverse Cas12a nucleases (which target and cleave complementary DNA)17 and Cas12a2 nucleases (which target RNA and then indiscriminately shred RNA and DNA)27,28 indicated that other functionalities might exist in the adjacent phylogenetic space. To explore this possibility, we searched public databases for sequences closely related to Cas12a2. We identified 61 orthologues primarily from environmental metagenomic assemblies that resolved into two clades distinct from Cas12a2, Cas12a and each other (Fig. Most of these were associated with CRISPR arrays and the spacer-acquisition genes cas1, cas2 and cas4 (Supplementary Table 1). However, orthologues from the identified clades showed limited conservation of the aromatic clamp residues in Cas12a2, which are crucial for broad DNA collateral cleavage28, with Cas12a3 lacking these regions entirely (Fig. c, Schematic (left) and quantification (right) of the nucleotide-depletion screen as part of the PFS (Cas12a2, Cas12a3 and Cas12a4) or protospacer-adjacent motif (PAM; Cas12a) determination for representative nucleases. Additional PFS screens are presented in Supplementary Fig. Results are the averages of independent experiments (n = 2). d, Schematic (left) and quantification (right) of the assessment of plasmid clearance versus growth arrest in E. coli based on variations of a plasmid interference assay. e, Quantification of induction of the SOS DNA damage response in E. coli based on a transcriptional fluorescent reporter. Bars and error bars in d and e represent the geometric mean ± geometric s.d. and the mean ± s.d., respectively, of independent experiments starting from separate colonies (n = 4), with grey dots representing each measurement. f, Schematic (top) and measurement (bottom) of the assessment of different targets and cleavage substrates in vitro. Values represent the mean of independent experiments (n = 3 or 4). Complete time courses are shown in Fig. Statistical analyses were performed using two-tailed Welch's t-tests with all biological replicates. Illustrations of thunderbolts and Petri dishes in d reproduced from ref. To investigate the possibility that Cas12a3 and Cas12a4 have distinct activities compared with Cas12a2, we first characterized representative Cas12a3 and Cas12a4 nucleases using our previously established plasmid interference assays in E. coli27. The nucleases were treated with a library of potential PFS sequences encoded in a target plasmid under antibiotic selection (Fig. This assay identified a preference for a purine-rich PFS, a finding in line with the known preferences of Cas12a2 nucleases and the high amino-acid similarity in the PFS-interacting region across the orthologues (Fig. Using a consensus PFS, the Cas12a3 and Cas12a4 nucleases reduced the number of transformants even without antibiotic selection for the target plasmid, similar to a representative Cas12a2 from the microbial community of Microcerotermes parvus (MpCas12a2) (Fig. We obtained comparable results for different target sequences (Supplementary Fig. 2b) and observed impaired growth in liquid culture without antibiotic selection (Supplementary Fig. Cas12a2 and Cas12a3 nucleases further provided defence against T4 phage infection (Supplementary Fig. However, unlike Cas12a2, the Cas12a3 and Cas12a4 nucleases did not induce a measurable SOS DNA damage response, as indicated by a transcriptional reporter driven from the recA promoter27 (Fig. Thus, although Cas12a3 and Cas12a4 nucleases arrest growth after activation in a manner similar to Cas12a2, both nucleases induce a distinct mechanism of immunity. Given that the Cas nucleases could have sequence preferences for specific substrates27,29,30, we used a randomized library of RNA and single-stranded DNA (ssDNA) substrates and a mixed-nucleotide sequence for the double-stranded DNA (dsDNA) substrate. As expected, MpCas12a (as a representative Cas12a nuclease) primarily cleaved the ssDNA substrate library and, to a limited extent, the dsDNA substrate in response to the ssDNA and dsDNA targets, respectively29,31. Moreover, MpCas12a2 cleaved all three substrates in response to the RNA target27,28 (Fig. Similar to MpCas12a2, Cas12a3 from an unknown Bacteroidetes bacterium (Ba1Cas12a3) and Cas12a4 from a microbial community of the Alvinella pompejana hydrothermal vents worm (ApCas12a4) were also activated by the RNA target. However, they exclusively cleaved the RNA substrate library. When presented with additional individual ssRNA, ssDNA or dsDNA substrates in vitro, similar trends were observed for these nucleases and for Cas12a3 from Smithella sp. Notably, Ba1Cas12a3 and Sm3Cas12a3 only partially cleaved their target RNA even after prolonged incubation, which was in contrast to complete cleavage by ApCas12a4 and MpCas12a2. This result suggests that the RNA target is not the preferred cleavage substrate of Cas12a3. RNA-mediated RNA cleavage by Cas12a3 and Cas12a4 is consistent with the lack of a measurable SOS response in E. coli (Fig. These observations reveal that exclusive RNA-activated RNA cleavage also occurs in the highly diverse family of Cas12 nucleases32, similar to the cleavage activities of the phylogenetically distinct Cas13 nucleases33. Building on the limited cleavage of target RNA by these nucleases, we noted that Ba1Cas12a3 also exhibited 3.8-fold less cleavage of the RNA substrate library compared with MpCas12a2 (Fig. By contrast, ApCas12a4 exhibited a continual increase in fluorescence over the course of the reaction without plateauing and efficiently cleaved its target RNA in vitro (Extended Data Fig. We speculated that reduced cleavage of the RNA library by Ba1Cas12a3 resulted from a selective preference for specific RNA substrates. Dots and bars represent the mean ± s.d. from independently mixed in vitro reactions (n = 4). b, Schematic (left) and quantification (right) of the fluorescence time courses of TXTL assays assessing nuclease activity based on silencing of deGFP expression. from independently mixed TXTL reactions (n = 3 or 4). c, Schematic (top) and quantification (bottom) of RNA sequencing of total RNA ≤ 200 nucleotides by Nanopore from TXTL reactions with Ba1Cas12a3 and a target or non-target RNA after 4 h. Values represent independent TXTL reactions (n = 3). The colour map indicates the z score of depletion scores at each position. d, Schematic (top) and quantification (bottom) of Nanopore sequencing of purified E. coli MRE600 tRNAs incubated with Ba1Cas12a3, Sm3Cas12a3 or ca23Cas12a3 under targeting versus non-targeting conditions. Values represent independent reactions (n = 3), with depletion z scores for each nucleotide as a colour map. Shown tRNAs (labelled with the corresponding anticodons) were detected in every nuclease reaction across the three independent experiments. Apparent cleavage sites ending in A could be shorter by one or (in the case of tRNAfMet(CAU)) two nucleotides towards the 3′ end owing to the use of a poly(A) extension to the cleavage product. 3 and 5 for full tRNA sequences with single-nucleotide depletion scores. e, Cleavage patterns of E. coli MRE600 tRNAs incubated with activated nucleases. The tRNA pool was 5′ labelled with a fluorophore, leaving the amino acyl group attached (top), or by replacing the 3′ amino acyl group with a fluorophore (bottom). Gel images are representative of independent cleavage reactions (n = 3). Open and closed circles represent the absence or presence, respectively, of the indicated component. For gel source data, see Supplementary Fig. To identify RNA substrates relevant to an immune response, we tested Ba1Cas12a3 in a cell-free transcription–translation (TXTL) system that closely mimics the bacterial cellular environment34,35. As expected, adding an activated MpCas12a2 or Ba1Cas12a3 ribonucleoprotein complex together with a GFP reporter plasmid reduced fluorescence relative to a non-target control (Fig. To distinguish whether silencing of the reporter resulted from cleavage of the gfp transcript or other essential RNA components (for example, tRNAs or rRNAs), we added activated nucleases 4 h before introducing a GFP-expressing reporter plasmid. Notably, GFP fluorescence was abolished when activated Ba1Cas12a3 was added before the reporter plasmid, whereas MpCas12a2, even when activated, had no effect relative to the non-target control (Fig. The enhanced silencing obtained only by activated Ba1Cas12a3 suggests that this nuclease selectively degrades RNA components essential for gene expression. We aimed to identify the RNA substrates of activated Ba1Cas12a3 that underlie silencing of the reporter in TXTL assays. To that end, we directly sequenced RNA from 4-h reactions using nanopore direct RNA sequencing, which enables complete sequencing of chemically modified RNAs36 (Supplementary Fig. Comparisons of read coverage between the non-target and target conditions showed that no notable cleavage of 5S, 16S or 23S rRNAs or the target RNA occurred. By contrast, many of the reads mapped to tRNAs (27 out of 47) were significantly (z score ≥ 2) truncated roughly 2–4 nucleotides upstream of their 3′ aminoacylated ends up to the discriminator base and the CCA tail conserved across all tRNAs (Fig. Cleavage of selected tRNAs was confirmed by northern blot analysis (Supplementary Fig. Similar cleavage patterns were observed for Sm3Cas12a3 in TXTL assays (25 out of 47) (Supplementary Figs. This specific cleavage of tRNAs accounts for the target-dependent silencing of the reporter observed in TXTL assays, as it would disrupt the translation machinery before the reporter plasmid is added. To further characterize the cleavage patterns in tRNA by Cas12a3 orthologues, we established a more controlled in vitro assay using purified bulk tRNAs from E. coli incubated with activated nucleases (Fig. Activated Ba1Cas12a3 led to significant 3′ cleavage (z score ≥ 2) of all but 3 out of the 49 mapped tRNAs, even though the remaining tRNAs (tRNAAsp(GUC), tRNAVal(GAC)1 and tRNAVal(GAC)2) still underwent measurable cleavage (Supplementary Fig. 5), with cleavage principally occurring three to five nucleotides upstream of each tRNA 3′ end. Similar cleavage patterns were observed with Sm3Cas12a3 and ca23Cas12a3 (identified in a wastewater microbial metagenome). However, the cleavage sites for Sm3Cas12a3 were shifted slightly upstream (Fig. 5), which indicated possible mechanistic differences within the Cas12a3 clade. Consistent with the direct RNA sequencing results, Ba1Cas12a3 trimmed the 3′ end of the entire pool of E. coli tRNAs labelled with a 5′ fluorophore (Fig. 2e, top), including specific tRNAs detected by northern blot analysis (Supplementary Fig. Although these tRNAs were charged with amino acids and contained extensive chemical modifications associated with E. coli37, Ba1Cas12a3 similarly cleaved the same pool of chemically modified tRNAs with the amino acid removed and replaced with a fluorophore (Fig. Ba1Cas12a3 also cleaved in vitro-transcribed tRNAs that lack both chemical modifications and charged amino acids (Supplementary Fig. 7a) and bulk tRNA isolated from yeast (Supplementary Fig. Notably, ApCas12a4 produced a different cleavage pattern of the tRNA pool (Fig. 2e), which suggests that the 3′ tRNA tail is not the preferred substrate of this nuclease. Together, these findings indicate that activated Cas12a3 nucleases preferentially cleave the 3′ tail of tRNAs while sparing the bound RNA target, thereby further differentiating Cas12a3 from Cas12a2 and Cas12a4. Among the diverse set of Cas nucleases, LshCas13a is the only other nuclease reported to cleave tRNAs22. As LshCas13a primarily cleaves its targeted transcript30, and Cas12a3 exhibits minimal target RNA cleavage (Fig. 2a), we sought to directly compare their activities. When targeting the same site in an expressed GFP transcript in TXTL assays, both LshCas13a and Ba1Cas12a3 efficiently reduced GFP fluorescence (Extended Data Fig. However, quantitative PCR with reverse transcription (RT–qPCR) revealed that the gfp transcript underwent cleavage only by LshCas13a (Extended Data Fig. These results show that LshCas13a and Ba1Cas12a3 have different modes of action after target recognition, with LshCas13a but not Cas12a3 substantially cleaving the target RNA. Trimming tRNA tails by Cas12a3 would prevent tRNAs from participating in translation, thereby potentially driving global translational shutdown and arresting cell growth. Alternatively, growth arrest could be mediated by tRNA cleavage products that induce systemic stress responses, such as the stringent response, which is activated through the detection of deacetylated tRNAs bound to the ribosome by the RelA protein38,39,40,41. However, deleting relA did not impair plasmid interference by any of the tested Cas12a3 orthologues in E. coli (Extended Data Fig. Therefore, Cas12a3-mediated immune defence operates independently of the stringent response mediated by RelA, which indicates that Cas12a3-mediated growth arrest results from either a different stress response or the direct disruption of translation. We confirmed that purified Ba1Cas12a3 strongly binds crRNA (dissociation constant (Kd) of about 0.2 nM), target RNA (Kd of about 5 nM) and in vitro-transcribed tRNAAla(UGC) when activated (Ba1Cas12a3, Kd of about  20 nM; catalytically dead Ba1Cas12a3 (dBa1Cas12a3), Kd of about 6 nM) (Supplementary Figs. Therefore, we used single-particle cryogenic electron microscopy (cryo-EM) to determine the 3.1 Å quaternary structure of Ba1Cas12a3 in complex with these three RNAs (Extended Data Table 1 and Supplementary Fig. To limit tRNA cleavage, the complex was reconstituted on ice with a reduced Mg2+ concentration. The resulting quaternary structure revealed that, similar to Cas12a2 from Sulfuricurvum sp. PC08-66 (SuCas12a2)28, Ba1Cas12a3 contains a REC lobe comprising REC1 (unresolved) and REC2 domains and a NUC lobe comprising a wedge (WED), PFS-interacting (PI), RuvC, ZR and an insertion domain (Fig. However, a portion of the insertion domain in Ba1Cas12a3 (residues 855–960) did not have discernible homology with SuCas12a2 and had no structural homologues in the Protein Data Bank (PDB) database (as determined using Foldseek and Dali)42,43. We designate this fold the tRNA-loading domain (tRLD), as it interacts directly with the tRNA in the quaternary structure and probably facilitates tRNA loading into the RuvC nuclease domain (described below). b, Atomic model of the Ba1Cas12a3 quaternary complex. e, Impact of truncating tRNAAla(UGC) on in vitro cleavage by Ba1Cas12a3. A, anticodon; D, D-arm; T, T-arm. Open and closed circles represent the absence or presence, respectively, of the indicated component. f, Mutational analysis of the truncated tRNA substrate h1 as part of in vitro cleavage by Ba1Cas12a3 and Sm3Cas12a3. g, Schematic (left) and quantification (right) of the mutational analysis of tRNA recognition domains in Ba1Cas12a3 in TXTL reactions. The fold reduction in GFP fluorescence was calculated in comparison to a non-target control. The three residues shown in d plus a poorly resolved neighbouring residue (K257) were swapped with alanine residues (AS; R251A, N253A, K256A and K257A) or with residues with an altered charge (CS; R251E, N253D, K256E and K257E). dBa1, Ba1Cas12a3 with one RuvC active site mutation (E1032A); WT, wild type. dBa1, Ba1Cas12a3 with three RuvC active-site mutations (D712A, E1032A and D1137A). i, Schematic (left) and quantification (right) of in vitro cleavage of an arbitrary RNA sequence by selected Ba1Cas12a3 mutants. Images in e and f are representative of independently prepared cleavage reactions (n = 3). of independently mixed TXTL or in vitro reactions. Statistical analysis was performed using two-tailed Welch's t-tests with all biological replicates (n = 4 for g, n = 3 for h and i). For gel source data, see Supplementary Fig. Ba1Cas12a3 interacts with crRNA in a similar way to Cas12a and Cas12a2, with the 5′ pseudoknot of the crRNA anchored between the RuvC and WED domains28,44. Moreover, several electrostatic interactions between basic side chains (for example, K881 and K885) and peptide backbone amines with the tRNA phosphodiester backbone help to position the scissile phosphate in the RuvC endonuclease active site (Fig. Notably, the 3′ hydroxyl group of the terminal adenosine points towards a vacant cavity, which would accommodate the respective charged amino acid (Extended Data Fig. This result suggests that Ba1Cas12a3 cleaves free tRNAs that are not actively engaged in protein synthesis. Stepwise elimination of the tRNA loops did not impair cleavage of the acceptor stem or 3′ CCA tail in vitro (Fig. A minimal substrate comprising the anticodon loop, acceptor stem and 3′ CCA (h1) was also cleaved, although it bound dBa1Cas12a3 with 62-fold lower affinity than the full-length tRNA (Extended Data Fig. Using this truncated tRNA substrate, we next examined the impact of mutating the acceptor stem and the conserved 3′ CCA tail on its activity. Ba1Cas12a3 tolerated stem alterations provided the 3′ CCA was present (Fig. Nonetheless, the catalytic efficiency was higher with a truncated tRNA than with linear RNAs, even when the CCA tail was located at the 3′ end (Extended Data Fig. In the CCA tail, Ba1Cas12a3 was sensitive to cytosine mutations, with transversions (C-to-G) impairing cleavage more strongly than transitions (C-to-U) (Fig. 3f), which suggests that these nucleases have divergent strategies for substrate recognition. However, removing this domain or mutating the residues that stack with the terminal adenosine (R902 or N924) substantially reduced both reporter silencing in TXTL assays and in vitro cleavage of different RNA substrates (Fig. Notably, the Y922A mutant increased cleavage activity in TXTL assays and in vitro with the truncated tRNA substrate (Fig. 3g,h), but not with non-tRNA substrates (Fig. This result suggests that Y922 has a more complex mechanistic role in Ba1Cas12a3. Mutation of the REC2 loop residues that interact with the T-arm impaired targeting in TXTL assays, whereby full-length tRNAs were cleaved, but only partially impeded cleavage of the truncated tRNA substrate lacking a T-arm (Fig. This finding indicates that T-arm interactions help position full-length tRNAs into the active site. Collectively, these data demonstrate that Ba1Cas12a3 preferentially cleaves free tRNAs by positioning the acceptor stem and the 3′ CCA tail of tRNAs into the RuvC active site through shape-specific and charge-specific interactions of tRNAs with the REC2 loop and tRLD. To gain deeper insights into the dynamics of Ba1Cas12a3 nuclease activation and tRNA cleavage, we determined the following cryo-EM structures: Ba1Cas12a3 bound to a crRNA (binary complex, 3.8 Å; Supplementary Fig. 15a,c–e); Ba1Cas12a3 bound to crRNA and target RNA (ternary complex, 3.9 Å; Supplementary Fig. 15b,f–h); and Ba1Cas12a3 bound to the cleaved 3′ ACCA tail from tRNAAla(UGC) (post-cleavage complex, 3.3 Å; Supplementary Fig. Together with the quaternary complex, which contains an uncleaved tRNA (Fig. 3b,c), these structures revealed a series of conformational changes that Ba1Cas12a3 undergoes to recognize a target RNA, selectively bind tRNAs and catalytically cleave the 3′ CCA tail of each tRNA (Fig. a, Cryo-EM structure of the Ba1Cas12a3–crRNA binary complex. b, Cryo-EM structure of Ba1Cas12a3–crRNA–target RNA ternary complex. c. Cryo-EM structure of the Ba1Cas12a3–crRNA–target RNA–tRNAAla(UGC) quaternary complex. g, Ba1Cas12a3 domain motion trajectories followed by release of the cleaved tRNA. Starting from the binary structure, target RNA binding drives multiple conformational changes in Ba1Cas12a3 and the bound crRNA, with the REC2 domain shifting by 28 Å to accommodate the guide–target duplex (Fig. Although similar conformational changes are sufficient to fully activate SuCas12a2 after target RNA recognition28, in Ba1Cas12a3, the tRLD remains positioned near the RuvC active site. However, it has increased flexibility, as evidenced by the decrease in local resolution (Supplementary Fig. 17 and Supplementary Video 1), the tRNA dissociates with the exception of four nucleotides comprising the tRNA tail (5′-ACCA-3′), which remain wedged between the RuvC and tRLD domains through contacts with R902, N924 and Y922 (Fig. 4d), consistent with the mapped cleavage sites across tRNAs (Fig. With the cleavage product bound, Ba1Cas12a3 retains its activated conformation (Fig. 4g), which indicates that another tRNA could be subsequently captured and cleaved without necessitating conformational resetting. Taken together, these structural snapshots reveal an activation pathway in which, after target RNA recognition, Ba1Cas12a3 relies on its unique tRLD to direct cleavage of tRNA 3′ CCA tails. Multiplexed RNA detection has been principally achieved by pairing Cas13 nucleases with orthogonal RNA substrates23,47. However, further expanding multiplexing requires new nucleases with orthogonal substrate preferences. To explore whether Ba1Cas12a3 can offer such an expansion, we adapted the minimal tRNA substrate as a reporter with a conjugated fluorophore and quencher45. A hairpin with a 3′ CCA tail and 3′ fluorophore produced the strongest signal (Fig. Notably, Sm3Cas12a3 and ca23Cas12a3 displayed distinct substrate preferences; however, Ba1Cas12a3 exhibited 226-fold greater activity than either orthologue (Fig. a, Fluorescence production after cleavage of FAM-labelled or quencher-tagged RNA substrates by Ba1Cas12a3, Sm3Cas12a3 and ca23Cas12a3. The area of each circle indicates fluorescence normalized to the highest value for each nuclease, whereas the colour of each circle represents absolute fluorescence production divided by the concentration of nuclease in the reaction. b, Fluorescence production after cleavage of additional FAM-labelled or quencher-tagged RNA substrates by Ba1Cas12a3. c, Fluorescence production after cleavage of FAM-labelled or quencher-tagged RNA substrates specific to Ba1Cas12a3, LwaCas13a or PsmCas13b. d, Schematic of the combined components for multiplexed one-pot detection. In this setup, each fluorophore (FAM, TEX and HEX) can be independently quantified. e, Impact of varying the applied concentration of detected RNAs derived from respiratory syncytial virus (RSV), influenza virus A (IVA) and SARS-CoV-2 (SARS) as part of the one-pot setup. Ratios are based on molarity, assuming an average RNA length of 2 kb in the human total RNA. g, Multiplexed, one-pot RNA detection in the presence of an excess of human total RNA. Dots depict individual measurements of independently prepared reactions, whereas bars and error bars represent the mean ± s.d. Fluorescence measurements depicted as circles or heatmaps represent the average of independent cleavage reactions (n = 3 or 4 in a and b, n = 4 in e and f, n = 6 in g). Using the optimal reporter for Ba1Cas12a3 (h25), we assessed multiplexing with Cas13a from Leptotrichia wadei (LwaCas13a) and Cas13b from Prevotella sp. MA2016 (PsmCas13b), which primarily cleave short RNAs composed only of U or A nucleotides, respectively23,48. As these linear substrates do not contain the CCA sequence and Cas13 nucleases do not efficiently cleave structured RNAs, each nuclease preferentially cleaved its cognate substrate (Fig. Leveraging this specificity, we combined the three nucleases and their cognate probes, each labelled with a distinct fluorophore, into a one-pot reaction for multiplexed RNA detection. This one-pot setup enabled the separate and combinatorial detection of RNA transcripts derived from the respiratory viruses SARS-CoV-2, respiratory syncytial virus (RSV) and influenza A (Fig. Notably, the presence of a large excess of human total RNA did not interfere with, but even enhanced, detection (Fig. These findings show that Cas12a3 nucleases can be readily incorporated into multiplexed detection assays alongside widely used Cas13 platforms45. In this work, we have reported the discovery of crRNA-guided Cas12a3 nucleases that recognize complementary RNA targets and, in response, cleave the conserved 3′ tails of tRNAs to induce growth arrest and block phage dissemination. Integrating genetic, biochemical, sequencing and structural studies, we propose a model (Extended Data Fig. 9) in which the nuclease undergoes a large conformational change that can then bind free tRNAs through multiple sequence-specific and shape-specific contacts. This immune response does not depend on a traditional stringent response but instead probably arises directly from translational inhibition or from a RelA-independent stress response activated by the tRNA cleavage products. Despite the high binding preference for tRNAs in vitro, it also remains possible that the nuclease targets additional RNAs not present in our TXTL system, which may further contribute to immune defence. Cas12a3 adds to a growing set of immune defences and cellular processes that inactivate tRNAs1,2,3,4,5,6,7,8,9,10,11,12,22,49,50,51,52. The anticodon loop is a common target of bacterial defences, with sequence-specific recognition of the loop beyond the anticodon required for translation. In response, some phages encode variant tRNAs that escape cleavage, thereby replenishing the tRNA pool and sustaining phage propagation53. Cleavage of the universally conserved 3′ CCA tail of tRNAs by Cas12a3 represents a distinct strategy that cannot be readily circumvented by viral tRNAs. To overcome this defence mechanism, viruses would require other, yet unknown, means of resistance. tRNA tail cleavage has been implicated with other cellular processes51,54, but so far, has not been directly linked to phage defence. Cas12a3 therefore could represent a previously unrecognized yet widely used phage defence mechanism based on preferential tRNA tail cleavage. The discovery of RNA-mediated tRNA cleavage in CRISPR–Cas systems reflects the rich functional diversity of antiviral defences. Although numerous defence strategies remain uncharacterized55,56, our work reveals that even well-known families can have previously unknown functions. In particular, Cas12a2, Cas12a3 and Cas12a4 are phylogenetically related to DNA-targeting Cas12a nucleases27 but exhibit functions that more closely resemble those of RNA-targeting Cas13 nucleases57. Cas12a3 is particularly notable as a Cas nuclease that can specifically direct its cleavage activity towards a distinct substrate while sparing the bound target, whereas Cas12a4 also exhibits RNA-activated RNA cleavage but functionally deviates from Cas12a3. These findings warrant further investigation. Together, Cas12a2, Cas12a3 and Cas12a4 seem to be hyper-evolvable, with domain alterations dictating whether the RuvC site cleaves broadly across multiple substrates (for example, Cas12a2) or preferentially against specific targets such as tRNA tails (for example, Cas12a3). Deviations are also possible in these clades, as illustrated by the different tRNA cleavage sites and substrate requirements of Ba1Cas12a3 and Sm3Cas12a3. These findings underscore the broader diversity of Cas12 nucleases and indicate the existence of additional, yet to-be-discovered functions. Cas12a3 in particular is an important addition for multiplexed RNA detection. The targeted cleavage of tRNA tails by Cas12a3 led us to the generation of a structured reporter specifically targeted by Cas12a3 but ignored by Cas13 nucleases, which enabled us to independently detect three distinct RNA biomarkers in a one-pot setup. Given the availability of other Cas12a3 and Cas13 orthologues that recognize distinct substrates58 (Fig. 5b), we anticipate that even more RNA biomarkers can be independently detected in a single reaction. Complementary advances in signal post-amplification and rendering CRISPR-based tests compatible with point-of-care settings could further enhance the diagnostic utility of Cas12a3. Beyond molecular diagnostics, the ability to inactivate most tRNAs through cleavage of their universally conserved 3′ CCA tail suggests applications in other areas, such as transcript-dependent cell arrest, viral suppression or selective cell elimination without direct DNA damage. Finally, our structural insights into PFS recognition and tRNA engagement could provide a foundation for engineering Cas12a3 with expanded target access and orthogonal substrates. Together, these advances position Cas12a3 as a versatile addition to CRISPR technologies and a prime candidate for enhancement through protein and guide RNA engineering. We used previously identified Cas12a2 protein sequences27 as queries for tBLASTn and BLASTp searches in the NCBI databases (https://www.ncbi.nlm.nih.gov) and the JGI Integrated Microbial Genomes and Microbiomes database (https://img.jgi.doe.gov) to identify closely related orthologues. The resulting amino acid sequences, along with Cas12a orthologues used as an outgroup, were aligned using Clustal Omega59. The trimmed alignment generated using ClipKIT60 was then used to reconstruct a phylogeny using IQ-TREE (v.2.3.6) (-m MFP -T 8 -B 1000)60,61 with a maximum-likelihood approach. Assignment of putative domains and conserved amino acid residues, as shown in Fig. 1, was performed with reference to SuCas12a2 (refs. Information on each nuclease, including contig accession numbers, source organisms and the presence of spacer acquisition genes (cas1, cas2 and cas4), is provided in Supplementary Table 1. The presence of these genes was determined using DefenseFinder (v.2.0.1) with defense-finder-models (v.2.0.2)62. CRISPR arrays were identified using CRISPRCasFinder (v.4.2.21)63. Strains, plasmids and oligonucleotides used in this study are listed in Supplementary Table 2. Nuclease-encoding sequences were codon-optimized for expression in E. coli and synthesized by Twist Bioscience, unless stated otherwise. A PFS-containing plasmid library (CBS-6873) was constructed by incorporating a target sequence (CAO1: 5′-CAUCAAGCCUUCCUUCAGGUGUUGCUCCA-3′) followed by 1,024 combinations of five randomized nucleotides (NNNNN). Thus, the target, placed under the PJ23119 promoter (https://parts.igem.org/Promoters/Catalog/Anderson) was cloned into a low-copy sc101 plasmid (around 5 copies per cell), which was then amplified using the primers ODpr23 and ODpr24 (Supplementary Table 2), with the forward primer including a 5-nucleotide randomized overhang. After DpnI treatment to remove template DNA, the resulting PCR product was ligated and electroporated into E. coli TOP10, which produced >2 million transformants (about 2,000-fold library coverage). The PFS preferences of Cas12a, Cas12a2, Cas12a3 and Cas12a4 orthologues was assessed by targeting the CBS-6873 library with a CAO1-targeting crRNA plasmid (CBS-6875), using a non-targeting crRNA plasmid (CBS-6876) as a control. The nucleotide-encoding sequences were codon-optimized for E. coli and expressed under a T7 promoter, whereas crRNAs were driven by the PJ23119 promoter. E. coli BL21(AI) cells with the nuclease and crRNA plasmids were electroporated in three separate reactions, each using around 500 ng of library DNA in 50 µl competent cells recovered in LB with 0.1 mM isopropyl β-d-1-thiogalactopyranoside (IPTG) and 0.2% l-arabinose and grown overnight to produce about 2 million transformants (>2,000-fold library coverage). Plasmids were then purified using a ZymoPURE II Plasmid Midiprep kit (D4201). Reactions for each experimental condition were carried out in duplicate. Purified plasmids from both target and non-target conditions were first PCR-amplified using the primers ODpr55 and ODpr56 (Supplementary Table 2) with KAPA HIFI HotStart polymerase (KK2601) for 20 cycles at 64.5 °C following the manufacturer's protocol. After amplification, these PCR products were purified using AMPureXP beads (Beckman Coulter, A63880) and subsequently indexed for Illumina sequencing using standard indexing primers with KAPA HIFI HotStart polymerase (KK2601) for 8 cycles at 61.5 °C with 2 µM forward and reverse primers and 5 ng µl–1 DNA. The resulting indexed PCR products were sequenced on an Illumina NovaSeq 6000 (paired-end, 150 bp reads), which ensured that at least 2 million reads per sample were sequenced. Raw FASTQ files were processed with Trimmomatic (v.0.39)64 using the following parameters: ILLUMINACLIP:TruSeq3-PE.fa:2:30:10, LEADING:3, TRAILING:3, and SLIDINGWINDOW:4:15. Paired-end reads were merged using BBMerge (qtrim=t, trimq=10, minlength=20)64,65. Sequences containing motifs matching “TTCCTTCAGGTGTTGCTCCA (…..) GGTGAGTTCT”, corresponding to the 20-nucleotide target-encoding sequence and the 10-nucleotide downstream sequence, were extracted, excluding any sequences with ambiguous bases (N) or Phred scores below 20. Depletion scores were then calculated using the formula: depletion = (sum(non-target)/sum(target)) × (count(target)/count(non-target)). The log2 fold change values for these scores were computed for the nucleotides at PFS positions (+1 to +5), and scatterplots visualizing the PFS preferences were generated using Matplotlib in Python. To test plasmid clearance, we expressed E. coli codon-optimized orthologues (as in the PFS screen) from plasmids (Supplementary Table 2). Target RNA and crRNA were co-expressed from a single plasmid under separate PJ23119 promoters (plasmids CBS-6177–6182 in Supplementary Table 2 include CAO1, GAPDH and GFP targets with both target and non-target crRNAs). E. coli BL21 (AI) cells with these target–guide constructs were electroporated with 1 µl of high-purity 50 ng µl–1 nuclease plasmid. Cells were then recovered in LB with 0.2% l-arabinose and 1 mM IPTG at 37 °C for 1 h with shaking without antibiotics. Serial tenfold dilutions (up to 10–1 in 1× PBS) were then prepared, and 5 µl of each dilution was spotted onto LB agar plates containing 0.2% l-arabinose, 0.1 mM IPTG and the appropriate antibiotics, including 50 µg ml–1 kanamycin for selection of the nuclease plasmid alone (assessing growth arrest) or 50 µg ml–1 kanamycin plus 25 µg ml–1 chloramphenicol for co-selection (assessing plasmid clearance). Experiments were performed in four biological replicates. Deletion of the relA gene in E. coli BL21(AI) was performed using λ Red recombineering as previously described66. To initiate recombineering, wild-type E. coli BL21(AI) cells were transformed with plasmid pKD46 (Supplementary Table 2), recovered and incubated overnight at 28 °C on LB agar containing ampicillin. The kanamycin resistance (kanR) cassette was PCR-amplified from plasmid pKD13 using the primers ODpr1652 and ODpr1653, which included 50 bp homology arms targeting sequences immediately upstream and downstream of the relA locus (positions 2,715,617–2,717,851 on GenBank accession CP047231), which encodes GTP pyrophosphokinase (NP_311671.1). The resulting PCR product was electroporated into E. coli cells with pKD46, which had been induced with 0.2% l-arabinose to express the λ-RED recombination machinery. Cells were plated on LB agar containing kanamycin and incubated at 37 °C to select for successful recombinants. To remove the inserted kanR cassette, plasmid pCP20 (Supplementary Table 2), which encodes FLP recombinase, was transformed into the strain. The cassette was excised via recombination at the flanking FRT sites, and pCP20 was subsequently cured by incubation at 37 °C. Successful deletion of relA was verified by colony PCR using the primers ODpr1655 and ODpr1654 (Extended Data Fig. 4), and by confirming the absence of growth on kanamycin-containing plates. Four independent mutants were generated and used to perform plasmid clearance (Extended Data Fig. 4), with the GAPDH T17 target under targeting and non-targeting conditions, using the plasmids CBS-6179 and CBS-6180 (Supplementary Table 2), respectively. To quantify the recA-dependent SOS response, E. coli BL21 (AI) cells were co-transformed with the carbenicillin-resistant reporter PrecA-GFP (CBS-3611) or a noGFP control (CBS-3616) along with the nuclease and target–guide plasmids, performed as previously described27. The resulting overnight cultures were grown in LB containing 100 µg ml–1 carbenicillin, 50 µg ml–1 kanamycin, 25 µg ml–1 chloramphenicol and 0.2% glucose to repress nuclease expression. The next day, 1 ml of each culture was pelleted (5,000g, 2 min), resuspended in fresh LB without antibiotics and adjusted to an OD600 of 0.1. Then, 20 µl of each culture was added to 180 µl fresh medium to obtain a final concentration of 0.2% l-arabinose and 0.1 mM IPTG in a 96-well plate (final OD600 = 0.01). Samples were incubated at 37 °C with vigorous shaking in a BioTek Synergy H1 plate reader, with OD600 and GFP fluorescence (excitation of 485/20 nm, emission of 528/20 nm) recorded. Each condition was tested in four biological replicates. Fresh E. coli BL21 (AI) cells at OD600 of 0.3 containing the respective nuclease (CBS-7169 and CBS-7171) and guide (CBS-7181-7183) plasmids (Supplementary Table 2) were grown in LB medium supplemented with 50 µg ml–1 kanamycin, 25 µg ml–1 chloramphenicol, 0.2% l-arabinose and 0.1 mM IPTG to induce nuclease expression. These cultures were challenged with different multiplicities of infection of T4 phage and incubated at 37 °C with vigorous shaking overnight. The next day, the cultures were serially diluted in tenfold increments in SM buffer (50 mM Tris-HCl, 100 mM NaCl and 8 mM MgSO4, pH 7.5). For plaque assays, control E. coli BL21 (AI) cells with an empty carbenicillin-resistant plasmid were mixed with 0.75% top agar and poured onto LB agar plates containing 100 µg ml–1 carbenicillin. Serial dilutions of the overnight cultures were then spotted onto the top agar, and the plates were incubated overnight at 37 °C to allow plaque formation. Each experimental condition was performed in three biological replicates. GFP-targeting crRNA-expression plasmid (CBS-7184) was used as a non-target control (Supplementary Table 2). RNAs were in vitro-transcribed from linear dsDNA templates containing a T7 promoter (Supplementary Table 2). In vitro transcription was performed using a HiScribe T7 High Yield RNA Synthesis kit (NEB, E2040) following the manufacturer's instructions. The generated target RNA along with the complementary crRNA were also used for cryo-EM. For the in vitro experiment, recombinant nucleases were purified primarily as previously described27,28. In brief, the proteins were expressed in E. coli BL21(DE3) star grown in LB medium and induced at an OD600 of about 0.6 with 0.1 mM IPTG, followed by an overnight incubation at 18 °C. Cells were then collected, lysed by sonication in lysis buffer and clarified by centrifugation at 30,000g for 30 min at 4 °C. The soluble fraction was then incubated with Ni-NTA resin (pre-equilibrated in lysis buffer) for 30 min at 6–8 °C, and the column was washed with an IMAC washing buffer supplemented with 2 M NaCl before bound proteins were eluted with imidazole-containing buffer. The eluate was diluted with low-salt buffer and further purified by ion-exchange chromatography on a HiTrap Heparin column using a NaCl gradient. Pooled fractions were concentrated and polished by size-exclusion chromatography on a HiLoad Superdex 200 column. The final purified fractions were pooled, concentrated, flash-frozen in liquid nitrogen and stored at −80 °C. Preference for targets and cleavage substrates was evaluated in vitro using nucleases purified as described above. Reactions contained 500 nM of the respective nuclease, an equimolar amount of T19 GAPDH non-target crRNA and T17 GAPDH target crRNA, and T17 GAPDH target at 50 nM provided either as RNA, ssDNA or dsDNA, as well as respective non-target substrates at 1 µM (Supplementary Table 2). For assays with Ba1Cas12a3 Y922A and ΔtRLD (Supplementary Fig. 13), reactions contained 100 nM nuclease, crRNA and target RNA together with 5 µM RNA substrate library. 1f, initial reaction rates in the linear range were normalized for each nuclease by setting the highest rate across reporters to one. All experiments were performed in at least three biological replicates. Activity of the purified Ba1Cas12a3 and MpCas12a2 nucleases was assessed using the TXTL system (Arbor Biosciences, 540300). Ba1Cas12a3 (25 nM) and MpCas12a2 (50 nM) were individually pre-incubated with 37.5 nM and 75 nM crRNA, respectively, for 15 min at 29 °C in 40 mM Tris-HCl (pH 7.5), 50 mM NaCl and 1 mM DTT, with 2 mM MgCl2. After this pre-incubation step, 50 nM GAPDH T17 RNA target was added and the mixtures were further incubated for 15 min at 29 °C to allow the formation of ribonucleoprotein particle (RNP) complexes. The assembled RNPs were then added to the TXTL reaction and incubated for 4 h at 29 °C. Thereafter, the deGFP plasmid supplied with the TXTL kit was added at 0.5 nM. In parallel, the RNP and the deGFP plasmid were simultaneously introduced into the TXTL mix that had already been pre-incubated for 4 h at 29 °C. GFP signal production was subsequently monitored on a BioTek Synergy H1 plate reader set to an excitation of 485 nm and an emission of 528 nm. All experiments were performed in at least three biological replicates. TXTL reactions were carried out using myTXTL (Arbor Biosciences, 540300) supplemented with the pCBS420 plasmid constitutively expressing deGFP and the pCBS11 p70-T7RNAP plasmid (Supplementary Table 2). Fluorescence was monitored over 16 h on a Synergy H1 plate reader (BioTek) at 485 nm excitation and 528 nm emission. For the monitoring of fluorescence, 3 µl volume of TXTL was used, whereas for RNA extraction, the reactions were scaled to 30 µl and sampled after 4 h, when strong GFP reduction under targeting conditions was evident. All experiments were performed in three biological replicates. Following TXTL, reactions were treated with 10 µl proteinase K for 15 min at room temperature and RNA was purified using a RNA Clean & Concentrator-25 kit (Zymo Research, R1017) with on-column DNase I digestion. RT–qPCR was performed using an iTaq Universal SYBR Green One-Step kit (Bio-Rad, 172-5150) on a CFX Opus 384 Real-Time PCR system (Bio-Rad) with CFX Maestro (v.5.3.022.1030). Primer pair 1 (ODpr1662 and ODpr1663) and primer pair 2 (ODpr1670 and ODpr1671) were validated using a dilution series of RNA from TXTL expressing gfp from plasmid pCBS420 (Supplementary Table 2), which produced efficiencies of 94.7% and 97.4%, respectively. Each primer was used at 300 nM. Cycling conditions included a 63 °C annealing–extension step, with other parameters following the manufacturer's protocol. Melting curve analysis confirmed single amplicons for both primer pairs. No amplification was detected in no-RT controls after 30 cycles. Each condition was tested in three biological replicates, with three technical replicates per sample. Ba1Cas12a3 and Sm3Cas12a3 nucleases (250 nM each) were pre-incubated in 40 mM Tris-HCl (pH 7.5), 50 mM NaCl, 1 mM DTT and 2 mM MgCl2 for 15 min at 29 °C with equimolar amounts of GAPDH T19 non-target crRNA and GAPDH T17 target crRNA. Next, GAPDH T17 target RNA was added to a final concentration of 250 nM and the mixture was incubated for at least 15 min to form the RNP complex. The assembled RNP was then added to 30 µl TXTL reactions. After 4 h of incubation at 29 °C, 6 μl of proteinase K (NEB, P8107) was added and the reactions were incubated for 15 min. RNA was purified from the reactions using a RNA Clean & Concentrator-25 kit (R1017, Zymo Research). To remove full-length rRNAs, RNAs with lengths of ≤200 nucleotides were isolated with a miRNeasy Tissue/Cells Advanced Micro kit (217684, Qiagen) according to the manufacturer's instructions. Subsequently, tRNAs in the purified RNA were deacylated by incubation in 90 mM Tris-HCl (pH 9.0) at 37 °C for 30 min. All reactions were performed in triplicate. For the in vitro total E. coli tRNA cleavage reactions, Ba1Cas12a3, Sm3Cas12a3 and ca23Cas12a3 nucleases (750 nM each) were combined in 40 mM Tris-HCl (pH 7.5), 50 mM NaCl, 1 mM DTT and 2 mM MgCl2 buffer with an equimolar amount of crRNAs and incubated for 15 min at 37 °C. Next, an equimolar amount of target RNA was added and the mixture was incubated for an additional 15 min to allow RNP complex formation. Total tRNAs purified from E. coli MRE600 (10109541001, Roche) were then added to the final concentration of 1.5 µM. The reactions were subsequently incubated at 37 °C for 2 h followed by proteinase K treatment and purification using a RNA Clean & Concentrator-5 kit (R1013, Zymo Research). All reactions were performed in triplicate. For each sample, 10 µg of total RNA was polyadenylated using E. coli poly(A) polymerase (NEB) in a 20 µl reaction containing 50 mM Tris-HCl pH 8, 250 mM NaCl, 10 mM MgCl2, 1 U µl–1 RNasin, 1 mM ATP and 0.25 U µl–1 poly(A) polymerase for 30 min at 37 °C. Polyadenylated RNAs were purified using a RNA Clean & Concentrator-5 kit (Zymo Research) following the manufacturer's instructions. Multiplexed Nanopore direct RNA sequencing was then performed following a previously described strategy67. In brief, custom barcode containing RTA adapters (bc03, bc04 and bc011; Supplementary Table 2) were annealed at a final concentration of 10 µM in 30 mM HEPES-KOH (pH 7.5) and 100 mM potassium acetate by incubation for 1 min at 95 °C followed by a slow cooling to 25 °C at a rate of 0.5 °C min–1. The annealed RTA adapters were then diluted to 0.7 µM and stored at −20 °C until use. To stop the ligation process, 2 µl of 0.5 M EDTA was added and samples were pooled and purified with 0.5 volumes of SPRI beads (Mag-Bind TotalPure NGS, Omega Bio-tek), washed twice with 200 µl fresh 80% ethanol and eluted in 10 µl RNase-free H2O. RT was performed by adding 8 µl of 5× Induro buffer, 2 µl of 10 mM dNTP, 2 µl of 10 µM random hexamer oligonucleotides, 2 µl Induro RT (NEB), 0.5 µl RNasin in a final volume of 40 µl and incubation 2 min at 22 °C (annealing), 15 min at 60 °C (RT) and 10 min at 70 °C (heat-inactivation). The reaction was purified with 1.4 volumes of SPRI beads, washed twice with 200 µl of fresh 80% ethanol and eluted in 10 µl RNase-free water. Sequencing adapter ligation was performed using a SQK-RNA004 library kit (ONT) according to the manufacturer's instructions. In brief, 10 µl of purified library was mixed with 6 µl 5× NEBNext Quick ligation buffer, 5 µl of RLA sequencing adapter (ONT SQK-RNA004) and 3 µl of T4 DNA Ligase high concentration in a final volume of 30 µl and incubated 15 min at 22 °C. The reaction was purified with 0.5 volumes of SPRI beads and washed twice with 100 µl RNA wash buffer (WSB, ONT SQK-RNA004). The library was then eluted in 32 µl RNA elution buffer (REB, ONT SQK-RNA004). For sequencing, the libraries were loaded onto a PromethION (FLO-PRO004RA) flow cell for RNA samples originating from the TXTL reactions or onto a MinION (FLO-MIN004RA) flow cell for RNA samples originating from the in vitro reactions. Data acquisition was performed using MinKNOW (v.24.11). Basecalling was performed with dorado (v.0.8.3) using the sup model. Unaligned bam files were then demultiplexed based on barcode labels using calibrated target performance scores. In silico identification were performed using covariance models from Rfam68 for tRNA (RF00005), tmRNA (RF00023), 5S rRNA (RF00001), 16S rRNA (RF00177) and 23S rRNA (RF02541) with searches executed using cmsearch69. The tRNA hits were subsequently annotated using tRNAscan-SE (v.2.0)70 to generate the final set of tRNA sequences for analysis. The resulting reference fasta files are provided in Supplementary Data 2 and 3. The -M option was enabled to mark shorter split hits as secondary alignments. The resulting SAM files were converted to BAM files, sorted and indexed using SAMtools (v.1.9)72. Reads with a mapping quality (MAPQ) below 20 were filtered out before downstream analyses. Depletion scores were computed for each nucleotide position according to the formula: depletion = (sum(non-target)/sum(target)) × (count(target)/count(non-target)). In this equation, ‘sum' denotes the total number of reads mapped to a reference sequence (that is, the overall read count) under either non-target or target condition, whereas ‘count' refers to the per–nucleotide coverage (the number of reads overlapping that specific position). This normalization procedure produces a relative measure of depletion at each nucleotide position. The z scores for each nucleotide position were calculated using the per-nucleotide depletion score, the overall mean and standard deviation for each sequence reference according to the formula: z = (nucleotide depletion score – reference mean)/reference standard deviation. For visualization purposes, the z scores were then constrained to a range of 0 to 2. Total purified E. coli tRNAs (Roche, 10109541001) were labelled at the 5′ end with Cy5 using a Vector Labs kit (MB-9001) according to the manufacturer's protocol, with Cy5 maleimide monoreactive dye (GEPA15131, Sigma-Aldrich). For 3′ end labelling, total E. coli tRNAs and Saccharomyces cerevisiae tRNA (Life Technologies, 0000010468) were first deacylated in 90 mM Tris-HCl (pH 9.0) at 37 °C for 30 min. Up to 400 pmol of RNA was then combined with 1 mM ATP, 100 µM pCp-Cy5 (NU-1706-CY5, Jena Bioscience), 1 µl T4 RNA ligase (M0202, NEB) and 1 µl of 100% DMSO in the reaction buffer supplied with the ligase. The reaction was incubated overnight at 16 °C. To generate in vitro-transcribed tRNAs, DNA templates were first prepared by PCR using the following primer pairs: ODpr1343 and ODpr1344 for tRNALys(UUU); ODpr1345 and ODpr1346 for tRNAAla(UGC); ODpr1347 and ODpr1348 for tRNASer(GGA); and ODpr1349 and ODpr1350 for tRNATyr(GUA). In vitro transcription was then carried out using a HiScribe T7 High Yield RNA Synthesis kit (NEB, E2040) according to the manufacturer's instructions. In vitro transcription was performed using a HiScribe T7 High Yield RNA Synthesis kit (NEB, E2040) according to the manufacturer's instructions, using up to 1 µg of PCR template. For the in vitro digestion assay, the nucleases MpCas12a, Ba1Cas12a3, Sm3Cas12a3, ca23Cas12a3 and ApCas12a4 were each used at 250 nM, combined with either equimolar GAPDH T19 non-target or T17 target crRNAs. To activate the proteins, 10 nM target RNA was added, followed by the addition of Cy5-labelled tRNA substrates at 50 nM. Reaction products were subsequently analysed by electrophoresis on a 10% polyacrylamide gel containing 7 M urea. All reactions were performed in triplicate. For northern blot analysis, TXTL and the in vitro tRNA cleavage reactions were performed as for Nanopore sequencing. After purification, 10 μg of each RNA sample from the TXTL reactions and 1 µg from the in vitro total purified E. coli tRNA reactions were resolved on an 8% polyacrylamide gel containing 7 M urea at 300 V for 2 h and 25 min using a gel transfer system (Doppel-Gelsystem Twin L, PerfectBlue). Using an electroblotter with an applied voltage of 50 V for 1 h at 4 °C (Tank-Elektroblotter Web M, PerfectBlue), the RNA was transferred onto Hybond-XL membranes (Sigma-Aldrich, 15356-1EA), crosslinked with UV-light for a total of 0.12 Joules (UV-lamp T8C; 254 nm, 8 W) and hybridized overnight in 15 ml of Roti-Hybri-Quick buffer at 42 °C with 5 μl γ-[32P]-ATP end-labelled oligodeoxyribonucleotides (5 pmol μl–1). The labelled RNA was visualized with a Phosphorimager (Typhoon FLA 7000, GE Healthcare). Target RNA and crRNAs for Cas12a3 nucleases were prepared as described above. The respective RNAs were in vitro-transcribed using a HiScribe T7 High Yield RNA Synthesis kit (NEB, E2040). In experiments assessing orthogonality, Ba1Cas12a3 (100 nM), LwaCas13a (10 nM) and PsmCas13b (250 nM) were used with an equimolar amount of target or non-target crRNA, with target RNA added at 50 nM and fluorescent RNA reporters at 1 mM. Reactions were carried out at 29 °C in BioTek Synergy H1 plate reader with fluorescence monitored at an excitation of 492 nm and emission of 520 nm. Fluorescence production rates were determined in the initial linear range, normalized for each nuclease, and analysed as a function of nuclease concentration. All experiments were performed at least in triplicate. For catalytic efficiency calculations with substrates S13, S14 and h25, reactions contained 50 nM Ba1Cas12a3, crRNA and target RNA, together with fluorophore–quencher reporters at 5,000, 2,500, 1,250, 625 or 312.5 nM. Fluorescence was monitored over time, and rates of fluorescence increase were background-corrected using the corresponding non-target controls. Rates were fitted to the Michaelis–Menten equation to extract kinetic parameters, and catalytic efficiency was determined as kcat Km–1. Each condition was tested in at least three independent replicates. To compare truncated tRNA substrates with mutations in the 3′ CCA tail, reactions contained 100 nM Ba1Cas12a3, crRNA and target RNA together with 1 µM reporters h25–h29 in buffer (40 mM Tris-HCl pH 7.5, 50 mM NaCl, 1 mM DTT and 2 mM MgCl2). Fluorescence was monitored over time, with each condition tested in at least three independent replicates. DNA templates for viral RNA targets were generated by PCR using the primers ODpr1684 and ODpr1685 for SARS-CoV-2 (N gene), ODpr1686 and ODpr1687 for influenza A (NP gene), and ODpr1690 and ODpr1691 for RSV (N gene) (Supplementary Table 2). All RNAs were transcribed in vitro using a HiScribe T7 High Yield RNA Synthesis kit (NEB, E2040). Purified Ba1Cas12a3, LwaCas13a and PsmCas13b were each assembled with their cognate crRNAs at 100 nM and tested with target RNA ranging from 50 nM to 0.5 nM. Universal Human Reference RNA (Life Technologies, QS0639) was added at 10 or 50 nM, calculated assuming an average RNA length of 2 kb. For one-pot reactions, nucleases, crRNAs and fluorescent reporters were first combined in a single mixture. Target RNAs were then added to initiate the reactions. Fluorescence was monitored over time. Initial reaction velocities were calculated in the linear range. All reactions were performed in at least four independent replicates. For all measurements, a 16-point 2-fold serial dilution series was prepared, starting at an initial concentration of 2.5 μM of unlabelled components. Each dilution was subsequently mixed in equal volumes with a constant concentration of Cy5-labelled RNA. Samples were transferred into premium-coated capillaries (MO-K025, NanoTemper Technologies), and microscale thermophoresis measurements were conducted using a Monolith Pico instrument (NanoTemper Technologies). Data analysis, including determination of Kd values, was performed using NanoTemper Analysis software and plotted using Origin software (OriginLabs). The cell culture was induced with 0.1 mM IPTG at an OD600 of 0.6 and further grown at 18 °C for 18 h. Cell pellets were collected by centrifugation and resuspended in 100 ml lysis buffer (50 mM Tris pH 7.5, 300 mM NaCl, 2 mM MgCl2, 10 mM imidazole and 10% glycerol) treated with 1 tablet of protease inhibitor cocktail (11873580001, Roche). Cells were disrupted by sonication (Amplitude 58%; 1 s on and 8 s off for 30 min total). The soluble fraction was obtained by centrifugation and incubated with 5 ml (bed volume) nickel resin rotating for 1 h at 4 °C. Subsequently, the purification procedure was conducted in a 20-ml gravity column. Nickel peak elution fractions were collected and diluted 10 times with low-salt buffer (50 mM Tris pH 7.5, 20 mM NaCl, 2 mM MgCl2 and 10% glycerol). Protein samples were then loaded onto a HiTrap SP HP cation-exchange chromatography column using low-salt buffer and eluted with a gradient of high-salt buffer (50 mM Tris pH 7.5, 1.0 M NaCl, 2 mM MgCl2 and 10% glycerol). Peak elution fractions were pooled and loaded onto Superdex 200 increase 10/300 GL (GE Healthcare) using SEC buffer (50 mM Tris pH 7.5, 150 mM NaCl, 2 mM MgCl2 and 5% glycerol). Peak fractions were then pooled and concentrated, and flash-frozen in aliquots in liquid nitrogen. The reaction was performed at 37 °C overnight and followed by DNase I treatment to remove DNA templates. The product was then purified using a DEAE column and heat treatment at 80 °C for 2 min and followed by the slow cooling process to room temperature for the re-annealing process. To obtain a homogenous tRNA population, the samples were subjected to a Superdex 75 Increase gel filtration column, and the tRNA-containing fractions were pooled and stored at −80 °C. For microscale thermophoresis measurements, the internally Cy5-labelled in vitro-transcribed E. coli tRNAs were produced as described above, with an additional 5% of Cy5-CTP introduced in the reaction. To obtain the Ba1Cas12a3 quaternary complex, crRNA was pre-treated by heating at 65 °C for 3 min and slowly cooled down to room temperature. RNA was then added to Ba1Cas12a3 in a 1:1.2 molar ratio. The complex was then incubated at room temperature for 10 min. The pure Ba1Cas12a3 binary complex was purified via Superdex 200 Increase 10/300 GL in buffer without MgCl2 to inhibit nuclease activity (50 mM Tris pH 7.5, 50 mM NaCl, 1 mM DTT and 5% glycerol). Target RNA and tRNAAla were sequentially added to 50 μl of the Mg2+-free peak fraction of the binary complex (about 20 μM) at 1:1.2 and 1:2 molar ratios, respectively. The target RNA was incubated at room temperature for 10 min, followed by the addition of tRNA, and the mixture was kept on ice for 30 min before vitrification. The binary complex sample was vitrified in the presence and absence of 0.02% (w/v) fluorinated octyl maltoside to obtain more particles with different orientations. For the ternary complex dataset, target RNA was added into the Ba1Cas12a3 binary complex (50 mM Tris-HCl pH 7.5, 150 mM NaCl, 1 mM DTT, 2 mM MgCl2 and 5% glycerol) in a 1:1.2 molar ratio. The ternary complex sample was vitrified in the presence of 1 mM ATPγS to obtain more views of particle distribution. Samples were vitrified on cryo-EM grids (Quantifoil R 2/1 300 mesh, Au). In brief, 3.5 μl of the sample was applied onto glow-discharged grids. The grids were blotted for 4 s (blot force: −9; 100% humidity; 4 °C) and plunge-frozen in liquid ethane using a Vitrobot Mark IV (FEI). The vitrified specimens were imaged on a Thermo Fisher Glacios Cryo-TEM operating at 200 kV and equipped with the Falcon 4i Direct Electron Detector camera, an energy filter slit width of 10 eV and a C2 aperture of 50 μm. The videos were recorded in counting mode using Thermo Fisher Scientific EPU software (v.3.7.1) with the total dose of 40 e– A–2 with a nominal magnification of 130,000-fold, corresponding to a pixel size of 0.91 Å. Motion correction, contrast transfer function estimation, particle picking and 2D classification were carried out on-the-fly using CryoSPARC live (v.4.4.0 and v.4.6.0)28,75. The remaining particles were further classified by using heterogeneous refinement, and the best 3D classes were subjected to non-uniform refinement. The statistics for data collection and processing are provided in Extended Data Table 1. All models corresponding to the Ba1Cas12a3 binary, ternary and quaternary complexes were originally generated by AlphaFold3 (ref. 76) and used as initial models for model building. Models were built in ChimeraX (v.1.7)77, and interactive refinements against cryo-EM maps were performed using ISOLDE with restraints for secondary-structure elements of the AlphaFold3-predicted structures77,78. The resulting models were further refined with real-space refinement and validated in Phenix (v.1.20.1)79. Statistics for the final models are described in Extended Data Table 1. All structural figures were generated using Chimerax (v.1.7). For structural comparison, the structures from binary to quaternary were aligned against the Ba1Cas12a3 WED domain. The domain motions from other individual domains were shown with arrows in different domain colours, and the arrows from each two compared models were created by using the PDB-arrows script with slight modification80. The Ba1Cas12a3 mutant constructs were generated either by Q5 PCR site-directed mutagenesis followed by KLD or by Gibson assembly, using the C-terminal 6×His-tagged WT Ba1Cas12a3 plasmid as a template (CBS-7171; Supplementary Table 2). Each plasmid was transformed into E. coli BL21 (DE3) cells, grown on an agar plate and selected by antibiotic resistance selection. A single colony was used to inoculate 60 ml of LB medium, grown at 37 °C at 200 rpm overnight (16–18 h). Next, 20 ml of the overnight growth was used to inoculate 1 litre of LB medium containing 100 µg ml–1 kanamycin. The cells were then cold shocked on ice for 20 min before inducing with 0.1 mM IPTG, followed by 16–18 h of growth at 18 °C. Cell pellets were collected by centrifugation and stored at −80 °C. Other nucleases were expressed using the same procedure. 3f,h,i, Extended Data Fig. 11 and 13, Ba1Cas12a3, mutants thereof, as well as ApCas12a4, MpCas12a2 and Sm3Cas12a3, were purified similarly to the method used for SuCas12a2 (refs. In brief, the cell pellets were thawed on ice before lysis by sonication in a lysis buffer (25 mM Tris pH 8.5, 500 mM NaCl, 10 mM imidazole, 2 mM MgCl2and 10% glycerol) containing protease inhibitors (2 μg ml−1 aprotinin, 10 μM leupeptin and 1.0 μg ml−1 pepstatin) and 1 mg ml−1 lysozyme. The lysate was clarified by centrifugation and added to 5 ml of Ni-NTA resin and batch bound at 4 °C for 30 min, and then washed with 300 ml wash buffer (25 mM Tris pH 8.5, 2 M NaCl, 10 mM imidazole, 2 mM MgCl2 and 10% glycerol). Fractions containing Ba1Cas12a3, as determined by SDS–PAGE, were desalted using a Hiprep 26/10 desalting column into low-salt buffer (25 mM Tris pH 8.5, 50 mM NaCl, 2 mM MgCl2 and 10% glycerol). Ba1Cas12a3 was applied to a Hitrap Heparin HP cation-exchange column and eluted using a gradient of high-salt buffer (25 mM Tris pH 8.5, 1 M NaCl, 2 mM MgCl2 and 10% glycerol). The fractions containing Ba1Cas12a3 were concentrated using a 50 kDa MWKO concentrator to about 1 ml and loaded onto a Hiload 16/600 Superdex 200 pg size-exclusion column using SEC buffer (25 mM HEPES pH 8.5, 150 mM KCl, 2 mM MgCl2 and 5% glycerol). Fractions containing the desired proteins were concentrated and stored at −80 °C. Reactions were made by combining 300 nM crRNA with 250 nM CRISPR-associated enzyme in DTT containing low-salt buffer (40 mM Tris-HCl pH 7.5, 50 mM NaCl, 2 mM MgCl2 and 1 mM DTT) and incubated at room temperature for 15 min. Next, 100 nM of FAM-labelled tRNA substrate was then added followed by 250 nM of target RNA to initiate the reaction. The reaction was performed at 37 °C for 1 h. Reactions were quenched with phenol and nucleic acid was purified by acid phenol–chloroform extraction. FAM-labelled nucleic acid was analysed using 12% urea–PAGE and visualized for fluorescein fluorescence. For determining the cleavage efficiency of the Ba1Cas12a3 TRL mutants, 140 µl reactions were performed with 150 nM Ba1Cas12a3, 100 nM guide RNA, 100 nM FAM 26-nucleotide structured tRNA substrate (Supplementary Table 2) and 25 nM target RNA. Nucleic acid was quenched and visualized as described above. 2), 250 nM crRNA and a given nuclease (250 nM) were incubated in a DTT containing low-salt buffer (40 mM Tris-HCl pH 7.5, 50 mM NaCl, 2 mM MgCl2 and 1 mM DTT) at room temperature for 15 min. Then 100 nM of FAM-labelled target (RNA, ssDNA or dsDNA) was added to initiate the reaction. The samples were analysed using a previously described denaturing FDF–PAGE81 and visualized for fluorescein fluorescence. To determine whether the Ba1Cas12a3 ΔtRLD mutant could still bind target RNA, an electromobility shift assay was performed. To control the amount of binary (protein–guide) complex in the reaction mixture, 5:1 Ba1Cas12a3 to guide RNA concentration ratio was prepared. An initial protein–guide solution was prepared and serial diluted into four reaction concentrations in an EDTA-containing buffer (25 mM HEPES (pH 7.2), 150 mM KCL and 100 mM EDTA). The reaction tubes ranged from 10 nM to 10 µM for Ba1Cas12a3 and 2 nM to 2 µM for the crRNA. The FAM-labelled target RNA concentration was held constant at 100 nM. The samples were run on a 6% TBE polyacrylamide gel and visualized for fluorescein fluorescence. To determine the overall structural components of Ba1Cas12a3 ΔtRLD compared with WT Ba1Cas12a3, far-UV circular dichroism (CD) was performed using a Jasco-J1500 spectropolarimeter. In brief, 0.36 mg ml–1 of the respective protein was prepared in CD buffer (10 mM K2HOP4 and 50 mM Na2SO4 pH 8.74) and CD spectra were obtained from 260 to 190 nm using a scanning speed of 50 nm min−1 (with a 2-s response time and accumulation of three scans). The CD signal was converted to molar ellipticity using Jasco Spectra Manager software. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The Illumina-based PFS screen data and the direct RNA Nanopore sequencing reads have been deposited into the European Nucleotide Archive under accession code PRJEB88250 (https://www.ebi.ac.uk/ena/browser/view/PRJEB88250). Models and associated cryo-EM maps have been deposited into the Electron Microscopy Data Bank (EMD) and PDB databases with the following accession codes: Ba1Cas12a3 binary complex (EMD-52275; PDB: 9HLX); Ba1Cas12a3 ternary complex (EMD-52287; PDB: 9HM6); Ba1Cas12a3 quaternary complex at pre-cleavage state (EMD-52285; PDB: 9HM4); and Ba1Cas12a3 quaternary complex at post-cleavage state (EMD-52286; PDB: 9HM5). Raw gel images are included as Supplementary Fig. Source data are provided with this paper. Elder, J. J. H., Papadopoulos, R., Hayne, C. K. & Stanley, R. E. The making and breaking of tRNAs by ribonucleases. Direct activation of a bacterial innate immune system by a viral capsid protein. Penner, M., Morad, I., Snyder, L. & Kaufmann, G. Phage T4-coded Stp: double-edged effector of coupled DNA and tRNA-restriction systems. Li, M. et al. Codon-usage-based inhibition of HIV protein synthesis by human schlafen 11. Burman, N. et al. A virally encoded tRNA neutralizes the PARIS antiviral defence system. Emerging roles of tRNAs in RNA virus infections. Zhang, F. et al. Human SAMD9 is a poxvirus-activatable anticodon nuclease inhibiting codon-specific protein synthesis. Ogawa, T. et al. A cytotoxic ribonuclease targeting specific transfer RNA anticodons. Winther, K. S. & Gerdes, K. Enteric virulence associated protein VapC inhibits translation by cleavage of initiator tRNA. The optional E. coli prr locus encodes a latent form of phage T4-induced anticodon nuclease. David, M., Borasio, G. D. & Kaufmann, G. Bacteriophage T4-induced anticodon-loop nuclease detected in a host strain restrictive to RNA ligase mutants. Metzner, F. J. et al. Mechanistic understanding of human SLFN11. Structural and biochemical characterization of schlafen11 N-terminal domain. Nussenzweig, P. M. & Marraffini, L. A. Molecular mechanisms of CRISPR–Cas immunity in bacteria. Jinek, M. et al. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Gasiunas, G., Barrangou, R., Horvath, P. & Siksnys, V. Cas9–crRNA ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria. Jiang, W., Samai, P. & Marraffini, L. A. Degradation of phage transcripts by CRISPR-associated RNases enables type III CRISPR–Cas immunity. Athukoralage, J. S. & White, M. F. Cyclic nucleotide signaling in phage defense and counter-defense. Type III CRISPR–Cas systems produce cyclic oligoadenylate second messengers. Kazlauskiene, M., Kostiuk, G., Venclovas, Č, Tamulaitis, G. & Siksnys, V. A cyclic oligonucleotide signaling pathway in type III CRISPR–Cas systems. Jain, I. et al. tRNA anticodon cleavage by target-activated CRISPR–Cas13a effector. A. RNA targeting by functionally orthogonal type VI-A CRISPR–Cas enzymes. Vialetto, E. et al. A target expression threshold dictates invader defense and prevents autoimmunity by CRISPR–Cas13. Zhang, N., Bewick, B., Xia, G., Furling, D. & Ashizawa, T. A CRISPR–Cas13a based strategy that tracks and degrades toxic RNA in myotonic dystrophy type 1. Dmytrenko, O. et al. Cas12a2 elicits abortive infection through RNA-triggered destruction of dsDNA. Bravo, J. P. K. et al. RNA targeting unleashes indiscriminate nuclease activity of CRISPR–Cas12a2. Chen, J. S. et al. CRISPR–Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Abudayyeh, O. O. et al. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. & Sashital, D. G. CRISPR–Cas12a has widespread off-target and dsDNA-nicking effects. Structures, mechanisms and applications of RNA-centric CRISPR–Cas13. Rapid and scalable characterization of CRISPR technologies using an E. coli cell-free transcription-translation system. Silverman, A. D., Karim, A. S. & Jewett, M. C. Cell-free gene expression: an expanded repertoire of applications. Lucas, M. C. et al. Quantitative analysis of tRNA abundance and modifications by nanopore RNA sequencing. Cappannini, A. et al. MODOMICS: a database of RNA modifications and related information. & Block, R. Synthesis of guanosine tetra- and pentaphosphate requires the presence of a codon-specific, uncharged transfer ribonucleic acid in the acceptor site of ribosomes. & Ramakrishnan, V. Ribosome-dependent activation of stringent control. Potrykus, K. & Cashel, M. (p)ppGpp: still magical?. Two compounds implicated in the function of the RC gene of Escherichia coli. Holm, L. Using Dali for protein structure comparison. van Kempen, M. et al. Fast and accurate protein structure search with Foldseek. Stella, S. et al. Conformational activation promotes CRISPR–Cas12a catalysis and resetting of the endonuclease activity. Kaminski, M. M., Abudayyeh, O. O., Gootenberg, J. S., Zhang, F. & Collins, J. J. CRISPR-based diagnostics. Ghouneimy, A., Mahas, A., Marsic, T., Aman, R. & Mahfouz, M. CRISPR-based diagnostics: challenges and potential solutions toward point-of-care applications. Greensmith, R. et al. CRISPR-enabled point-of-care genotyping for APOL1 genetic risk assessment. Meidler, R., Morad, I., Amitsur, M., Inokuchi, H. & Kaufmann, G. Detection of anticodon nuclease residues involved in tRNALys cleavage specificity. Saikia, M. & Hatzoglou, M. The many virtues of tRNA-derived stress-induced RNAs (tiRNAs): discovering novel mechanisms of stress response and effect on human health. Songailiene, I. et al. HEPN-MNT toxin–antitoxin System: the HEPN ribonuclease is neutralized by oligoAMPylation. Lu, J., Huang, B., Esberg, A., Johansson, M. J. O. & Byström, A. S. The Kluyveromyces lactis γ-toxin targets tRNA anticodons. A., Costa, A. R. & Brouns, S. J. J. Phage tRNAs evade tRNA-targeting host defenses through anticodon loop mutations. Wellner, K., Czech, A., Ignatova, Z., Betat, H. & Mörl, M. Examining tRNA 3′-ends in: teamwork between CCA-adding enzyme, RNase T, and RNase R. RNA 24, 361–370 (2018). Protein and genomic language models chart a vast landscape of antiphage defenses. DeWeirdt, P. C., Mahoney, E. M. & Laub, M. T. DefensePredictor: a machine learning model to discover novel prokaryotic immune systems. Structural basis for the activation of a compact CRISPR–Cas13 nuclease. Sievers, F. et al. Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. & Rokas, A. ClipKIT: a multiple sequence alignment trimming software for accurate phylogenomic inference. Nguyen, L.-T., Schmidt, H. A., von Haeseler, A. & Minh, B. Q. IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Tesson, F. et al. A comprehensive resource for exploring antiphage defense: DefenseFinder webservice,wiki and databases. Couvin, D. et al. CRISPRCasFinder, an update of CRISRFinder, includes a portable version, enhanced performance and integrates search for Cas proteins. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. & Singer, E. BBMerge—accurate paired shotgun read merging via overlap. & Wanner, B. L. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. van der Toorn, W. et al. Demultiplexing and barcode-specific adaptive sampling for nanopore direct RNA sequencing. Ontiveros-Palacios, N. et al. Rfam 15: RNA families database in 2025. Nawrocki, E. P. & Eddy, S. R. Infernal 1.1: 100-fold faster RNA homology searches. & Lowe, T. M. tRNAscan-SE 2.0: improved detection and functional classification of transfer RNA genes. Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Abbassi, N.-E. -H. et al. Cryo-EM structures of the human Elongator complex at work. Electrophoretic mobility shift assay (EMSA) and microscale thermophoresis (MST) methods to measure interactions between tRNAs and their modifying enzymes. & Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Pettersen, E. F. et al. UCSF ChimeraX: structure visualization for researchers, educators, and developers. Highly accurate protein structure prediction with AlphaFold. Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM and crystallography. Chaaban, S. & Carter, A. P. Structure of dynein–dynactin on microtubules shows tandem adaptor binding. & Baulcombe, D. C. FDF–PAGE: a powerful technique revealing previously undetected small RNAs sequestered by complementary transcripts. We thank Ł. Koziej for processing of the initial cryo-EM datasets, S. Schmelz for support in cryo-EM, A. Gatzemeier for assistance in the purification of dBa1Cas12a3, R. Rarose for support with the in vitro RNA experiments, M. Kaminski for providing purified PsmCas13b protein, L. Schönemann for protein purification, and C. Krempl and S. Backesfor providing the RSV and influenza A transcript-encoding plasmids. This work was supported through funding by the European Research Council (101001394 to S.G.; 865973 and 101158249 to C.L.B. ), the National Institutes of Health (R35GM138080 to R.N.J. ), the PostDoc Plus Program from the Graduate School of Life Sciences at Julius-Maximilians-Universität Würzburg (to O.D. ), and the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany's Excellence Strategy–The Berlin Mathematics Research Center MATH+ (EXC−2046/1, project ID: 390685689 to M.v.K. Open access funding provided by Helmholtz-Zentrum für Infektionsforschung GmbH (HZI). These authors contributed equally: Oleg Dmytrenko, Biao Yuan Helmholtz Institute for RNA-based Infection Research (HIRI), Helmholtz Centre for Infection Research (HZI), Würzburg, Germany Oleg Dmytrenko, Max Krebel, Xiye Chen, Tatjana Achmedov & Chase L. Beisel Helmholtz Centre for Infection Research (HZI), Braunschweig, Germany Department of Chemistry and Biochemistry, Utah State University, Logan, UT, USA Kadin T. Crosby, Bamidele Filani & Ryan N. Jackson Malopolska Center of Biotechnology, Jagiellonian University, Krakow, Poland Jakub S. Nowak, Andrzej Chramiec-Głąbik & Sebastian Glatt Architecture et Réactivitié de l'ARN, Université de Strasbourg, CNRS, Institute of Molecular and Cellular Biology (IBMC), University of Strasbourg, Strasbourg, France Wiep van der Toorn & Max von Kleist Project Groups, Robert Koch Institute, Berlin, Germany Wiep van der Toorn & Max von Kleist University of Veterinary Medicine, Vienna, Austria Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Structure solution, model building and structural analyses: B.Y., D.W.H., K.T.C., J.P.K.B. Writing, review and editing: all authors. Correspondence to Dirk W. Heinz, Ryan N. Jackson or Chase L. Beisel. have filed patent applications on related technologies. have filed patent applications on the warpdemux technology used as part of direct RNA sequencing. is a co-founder and scientific advisor of Locus Biosciences, a co-founder and officer of Leopard Biosciences, and scientific advisor to Benson Hill. The other authors declare no conflicts of interest. Nature thanks Mitchell O'Connell and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The phylogenetic tree was constructed using IQ-TREE with ultrafast bootstrapping (1,000 replicates). The corresponding simplified phylogeny is shown in Fig. The alignment highlights key motifs and residues, including those involved in protospacer-flanking sequence (PFS) recognition, the aromatic clamp regions interacting with cleaved and non-cleaved strands, CRISPR RNA (crRNA) recognition, RuvC domain motifs, and zinc-ribbon sites. A subset of the amino acid alignment is shown in Fig. (a) In vitro target cleavage using FAM-labeled dsDNA, ssDNA, or RNA resolved on a fully-denaturing formaldehyde (FDF) polyacrylamide gel. (b) In vitro cleavage of FAM-labeled non-target substrates. Gel images are representative of independent cleavage assays (n = 3). For gel source data, see Supplementary Fig. Plasmids encoding the Cas nuclease, GFP-targeting or non-targeting crRNA, and deGFP are combined in a TXTL reaction, and GFP fluorescence is measured over time. Right: fluorescence time course for Ba1Cas12a3 and LshCas13a. The vertical dashed line represents the time-point when a separate TXTL reaction was stopped for RNA extraction and RT-qPCR. Curves and shaded regions represent the mean ± standard deviation of separately prepared reactions (n = 3 or 4). (b) RT-qPCR analysis of the deGFP mRNA to determine the extent of mRNA cleavage. Left: location of the two primer pairs used for qPCR relative to the target sites for Ba1Cas12a3 and LshCas13a. Each dot represents an independent measurement from one TXTL reaction, while bars and error bars represent the mean ± standard deviation from separate reactions (n = 3), with each the average of triplicate technical measurements. Non-significant p-values (p ≥ 0.05) are in gray. Top: two-step approach to generate the complete deletion of relA using lambda-RED recombineering. Primer pairs to verify the deletion are shown at each step. Bottom: Resolved PCR products at each step of generating the ∆relA strain. For gel source data, see Supplementary Fig. (b) Transformation fold-reduction as part of the plasmid interference assay using the ∆relA strain. 1d for an overview of the assay. Bars and error bars represent the mean ± standard deviation of biological replicates (n = 3 or 4). (a) Model of the SuCas12a2-crRNA-target RNA-dsDNA quaternary complex. (b) Model of the Ba1Cas12a3-crRNA-target RNA-tRNAAla(UGC) quaternary complex. (e) Ba1Cas12a3 PI domain (in plum) superimposed with SuCas12a2 PI domain (in dark grey). The PFS for Ba1Cas12a3 is colored in dark orange. The PFS for SuCas12a2 is colored in dim grey. The PFS sequences bind to each PI domain in a distinct manner. The differences between SuCas12a2 and Ba1Cas12a3 in PFS recognition pattern are well correlated with the PFS library screening in Supplementary Fig. (a) Interactions between the REC2 loop of Ba1Cas12a3 and the T-arm of tRNA are shown with underlying EM density map. (c) 2D interaction plot displaying the binding interface between Ba1Cas12a3 and tRNA. Electrostatic contacts are primarily between positively charged side chains or peptide backbone amines located at the ends of helices. (d) The bound tRNA is modeled with a tryptophan residue, demonstrating no steric clashes in the structure to impair binding to aminoacylated tRNAs. Binding of the activated dBa1Cas12a3 ternary complex to the indicated fluorescent RNA substrate was quantified by MST. dBa1Cas12a3 contains the E1065A mutation in the RuvC endonuclease domain to render it catalytically inactive. Symbols and error bars represent the mean ± standard deviation of independent experiments (n = 3). (a) Comparing cleavage of different RNA substrates containing CCCCA. An activated Ba1Cas12a3 RNP was indicated with the indicated substrate, and fluorescence through the separation of the conjugated fluorophore (green circle) and quencher (gray circle) is measured over time for a range of substrate concentrations. Circles with nucleotides are colored to represent a mutation from C to U, a pyrimidine (red), or to G, a purine (yellow). The data from independent measurements (n = 3) were used to calculate catalytic efficiency (kcat/Km), with means ± standard error shown. (b) Comparing different truncated tRNA substrates with mutations to the cytosines in the 3′ CCA tail. Dotted curves and shaded regions represent the mean ± standard deviation of independently prepared and monitored reactions (n = 4). From left to right: Stepwise activation of Ba1Cas12a3 and tRNA cleavage leading to translational inhibition and growth arrest in response to a recognized target RNA expressed from an invading bacteriophage. This file contains the appearance of CRISPR arrays and spacer acquisition genes near cas12a3 and cas12a4 genes. In some cases, cas genes occur on contigs too short to assess the presence of adjacent CRISPR arrays or acquisition genes. This file contains a list of strains, plasmids and RNA and DNA sequences used in this study. Amino acid sequences of Cas12a, Cas12a2, Cas12a3 and Cas12a4 orthologues. tRNA reference sequences of E. coli MRE600 (CP014197). Structural rearrangements for nuclease activation in Ba1Cas12a3. Ba1Cas12a3 complexes are colored as in Figure 4 Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. RNA-triggered Cas12a3 cleaves tRNA tails to execute bacterial immunity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69918587/smiling-medusa/'>Archaeologists Uncovered a Strange Carving of a Terrifying Mythical Figure</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 14:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Depictions of Medusa are typically at least a bit frightening, with her snakes for hair and menacing facial expressions. However, a carved Medusa image recently discovered by archaeologists at an ancient Roman forum offers a fresh take on the iconic character from Greek mythology—the team unearthed ceiling panels above marbled columns showing Medusa smiling like a child. “Our Medusa was made just like an Eros, like the face of a very small child and in a smiling pose,” Fatma Bagdatli Cam, Bartin University professor, said according to Turkey Today. Archaeologists discovered the distinct smiling Medusa during excavations in the ancient city of Amastris (now known as Amasra, and located in modern-day Turkey) while working through a monumental Roman forum with a columned gallery and ceiling tiles. As part of Turkey's Ministry of Culture and Tourism's Heritage for the Future project, in conjunction with the university's Archaeology Application and Research Center, a 30,000-square-foot area has so far yielded marble columns up to 30 feet tall. The research team is attempting to recreate the forum (also known as a Roman stoa), and has uncovered at least seven columns, additional architectural blocks, ceiling coffers, and evidence that a second stoa sits nearby. The discoveries have prompted experts to dub Amasra a city of splendor during Roman rule. Cam said that every artifact helps develop a clearer picture of common life and culture in Amasra during the Roman era. With the character from Greek mythology known for her snakes for hair, sharp teeth, and a terrifying expression designed to turn onlookers into stone, images of Medusa were commonly used to scare would-be visitors. The more welcoming side of Medusa seen at Amasra turns typical mythology upside down, with the smile instead designed to represent peace and prosperity, Cam said. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. A Colossal Squid Might Have Shown Its Secret Face Humans May Be Able to Grow New Teeth in 4 Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69821410/cave-community-incest/'>Scientists Analyzed Ancient DNA—and Discovered a Lost Community with a Hidden Secret</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. The team published its findings in a study in Communications Biology, highlighting the population history, kinship patterns, and cultural practices of this distinct Bronze Age group prior to Greek colonization of the region. DNA analysis shows strong ties to Early Bronze Age Sicily, with little influence from the eastern Mediterranean. “This suggests that, while in contact across the Strait of Messina, Tyrrhenian Calabria followed its own demographic and cultural trajectories during prehistory.” The site—known for copper and iron ore exploitation and funerary use—has an apparent geographic isolation, but it wasn't fully genetically autonomous. Two individuals show ancestral links to populations from northeastern Italy, showing long-distance mobility and interactions across the peninsula. The genomes revealed contributions from European hunter-gatherers, Anatolian Neolithic farmers, and Steppe pastoralists, and while these components are all common in Bronze Age Europe, they form a local signature for the people of the Calabria mountains. This occurred despite their carrying genetic variants associated with adult lactose intolerance. These people had developed dietary strategies that allowed them to thrive in a challenging mountain environment, despite lacking genetic tolerance to lactose.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Experts Find Missing Piece of Ramesses II Statue The Maya Kingdom Collapsed Due to Burning Events A Colossal Squid Might Have Shown Its Secret Face Experts Found Four Mass Graves Filled With Bodies Humans May Be Able to Grow New Teeth in 4 Years Fifth State of Matter Makes a Quantum Breakthrough</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-67360-w'>Associations between preservative food additives and type 2 diabetes incidence in the NutriNet-Santé prospective cohort</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 13:08:08
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Experimental studies suggested potential adverse effects of preservative food additives, but epidemiological data are lacking. We aim to investigate associations between exposure to these compounds and type 2 diabetes incidence in the NutriNet-Santé prospective cohort (n = 108,723; 79.2%women; mean age=42.5 (SD = 14.6); France, 2009-2023). Dietary intakes are assessed using repeated 24h-dietary records. Exposure is evaluated through multiple composition databases and ad-hoc laboratory assays in food matrices. Associations between cumulative exposures to preservatives and diabetes incidence are characterised using multi-adjusted Cox models. The sum of total preservatives encompasses 58 substances. Among those, 17 are consumed by at least 10% of the study population and thus individually investigated. Thirteen (12 after multiple test correction) widely used individual preservatives are associated with higher diabetes incidence (n=1131cases): potassium sorbate, potassium metabisulfite, sodium nitrite, acetic, citric and phosphoric acids, sodium acetates, calcium propionate, sodium ascorbate, alpha-tocopherol, sodium erythorbate, and rosemary extracts. These findings call for their safety re-evaluation and support recommendations to favour fresh and minimally processed foods without superfluous additives. Trial registration: The NutriNet-Santé cohort is registered at clinicaltrials.gov (NCT03335644). Food additives encompass a wide range of substances which can be divided into different categories based on their functional properties, e.g., colours, preservatives, sweeteners, emulsifiers, etc. Our group already investigated artificial sweeteners1, emulsifiers2, nitrites and nitrates3, and most recently, food additive mixtures4 in association with type 2 diabetes incidence in the NutriNet-Santé cohort. The present study focuses on preservative food additives, which are massively used by the food industry globally. Among the three and a half million foods and beverages listed in the Open Food Facts World database in 20245, more than 700,000 contain at least one of these additives. Preservatives (European codes usually within the E200s and E300s ranges) are added to prolong the shelf-life of foods. They protect them against deterioration caused by micro-organisms and/or growth of pathogenic micro-organisms, and by oxidation, such as fat rancidity and colour changes6. From 2004 to 2024, about a sixth (n = 25) of food additive re-evaluations conducted by the European Food Safety Authority (EFSA) concerned preservatives. Sixteen of them led to the derivation of Acceptable Daily Intake (ADI) reference values for these additives or their respective groups7. ADIs were based on a range of toxicological endpoints, including behavioural, carcinogenic, developmental, haematological, reproductive, and thyroid toxicity, as well as growth retardation, increased blood methaemoglobin level, and increased mortality, all relying on experimental data. Moreover, in vivo and in vitro studies highlighted potential metabolic-related generally adverse effects of preservatives on pancreatic tissue8,9, insulin disruption8,9,10,11, the tricarboxylic acid (TCA) cycle11,12, inflammation13,14, and advanced glycation end products (AGEs) activation15, questioning their potential impacts on metabolic disorders such as type 2 diabetes, with direct metabolic aetiology. EFSA suggested that metabolisation of these additives implies conversion into substances, such as acetate into acetyl-CoA, that play a role in the body's metabolism7. For some of these additives, corresponding substances are naturally present in foods and beverages (e.g., certain antioxidant vitamins such as vitamin C – ascorbic acid). In this case, epidemiological studies investigated the link between their dietary intake, supplementation or nutritional status and the risk of type 2 diabetes (suggesting non-to-low protective evidence16), but to our knowledge, none of these have considered food additives sources, while the impact of the substance may depend on the matrix in which it is integrated. Plus, no consumption data is available for other preservative food additives due to the important variability in additive composition between commercial products and the lack of brand-specific data in previous cohorts. Thus, this study aimed to comprehensively quantify the time-dependent exposure to preservative food additives available on the French/European market and examine any potential associations with the incidence of type 2 diabetes in a large prospective cohort. The population study included 108,723 participants (Fig. On average, they completed 21 24HDRs (SD 18); median = 17; 25th–75th percentiles = 6–32; maximum = 84. Participants had a mean age at baseline of 42.5 years (SD 14.6) (range = 15.2–99.0). Compared to lower consumers (tertile 1), higher consumers of food additive preservatives (tertile 3) tended to be younger, slightly less exposed to a family history of diabetes, with a lower prevalence of metabolic diseases (cardiovascular disease, dyslipidaemia, and hypertension), a lower physical activity level, they were more likely to be current smokers, consumed less alcohol and had higher energy intakes (Table 1—descriptive unadjusted analysis). Flowchart of participants included from the NutriNet-Santé cohort, 2009–2023 (n = 108,723). Intakes of preservative food additives are displayed in Tables 2 and 3. Out of the 58 preservative food additives detected and quantified in our databases, 17 were consumed by at least 10% of the participants and thus were individually investigated in relation with type 2 diabetes incidence (Tables 2 and 3), to allow sufficient statistical power in Cox models (but all 58 additives were accounted for in sums of preservative categories). In terms of proportion of consumers, the main preservative food additives were: citric acid (E330) (91.8% consumers), lecithins (E322) (87.1%), total sulfites (83.6%), ascorbic acid (E300) (83.5%), sodium nitrite (E250) (73.7%), potassium sorbate (E202) (65.5%), sodium erythorbate (E316) (52.7%), sodium ascorbate (E301) (50.2%), potassium metabisulfite (E224) (44.4%), and potassium nitrate (E252) (32.6%). No strong correlation between preservative food additives was identified (Supplementary Fig. Manufacturers use preservatives ubiquitously across a wide range of food groups (Fig. In all, 34.6% of food additive preservatives were consumed through ultra-processed foods in this population study. bDetailed % are presented in Supplementary Table 1. No participant exceeded the ADIs set by EFSA7 for sorbates, erythorbates or nitrates. However, 90 participants exceeded the ADI set for sulfites with a mean intake of 0.88 mg of sulfur dioxide/kg body weight per day (SD 0.33, median = 0.80, 25th–75th percentiles [0.75–0.92]) and 55 exceeded the ADI for nitrites with a mean consumption of 0.10 mg nitrite ion/kg body weight per day (0.03, 0.08, [0.08–0.10]). Participants' median follow-up time was 8.05 years (841,296 person-years). Between 2009 and 2023, 1131 incident type 2 diabetes cases were identified. Schoenfeld residuals did not refute the proportional hazard assumption (Supplementary Fig. Restricted cubic spline plots (Supplementary Fig. 3) did not indicate departure from linearity for sodium acetates (E262), total tocopherols, tocopherol-rich extract (E306), alpha-tocopherol (E307), phosphoric acid (E338), and extracts of rosemary (E392) (p values for non-linearity ≥0.05), in this case p values for trend are provided in the following paragraph. For others, restricted cubic splines suggested a plateau effect (p values for non-linearity <0.05); therefore, the likelihood ratio overall p values (requiring no underlying hypothesis of linearity) are displayed thereafter. The retained p values are provided for all tested additives in Supplementary Table 2 for the main model and in Supplementary Table 3 for sensitivity analyses. The results of Cox models are presented in Fig. Overall, these results were stable across all sensitivity analyses testing mutual adjustment for other preservative food additives, additional adjustment for the proportion of ultra-processed foods in the diet, diagnosis and/or treatment for at least one prevalent non-diabetes metabolic disorder, time-dependent intakes of total emulsifier and total artificial sweetener food additives, vitamins C and E supplements, trans fatty acids, dietary patterns rather than individual food groups, polyunsaturated fatty acids and heme iron; models with start of follow-up after the first 2-year period; models not excluding energy under-reporters; marginal structural model; and model using splines for covariates (Supplementary Table 3). An exception was for total ascorbates, which became non-significant with p = 0.4 in the marginal structural model, maybe due to loss of statistical power (however, the specific ascorbate E301 remained stable with this model, p < 0.001). As expected, no association was detected with the hip fracture negative control model (613 incident cases, all p values > 0.05). HR hazard ratio, CI confidence interval. Associations between total preservatives and non-antioxidant preservatives and type 2 diabetes are presented on the left-hand side (A) of the figure, and between antioxidant preservatives and type 2 diabetes on the right-hand side (B). aThe three food additive categories were defined as follows: sex-specific tertiles (tertile 1 = reference) for total preservatives, total preservatives (non-antioxidant), total sorbates, total sulfites, total nitrites, sodium nitrite (E250), total preservatives (antioxidant), total ascorbates, ascorbic acid (E300), lecithins (E322), and citric acid (E330); and otherwise: 1/ non-consumers (=reference), 2/ lower, and 3/ higher consumers, the latter two being separated by the sex-specific median. Cut-offs were re-calculated for each period and are available in Supplementary Table 6. bThe detail of all investigated associations between preservative food additive intakes and type 2 diabetes risk with corresponding HRs, 95% CIs and number of cases/participants per category is provided in Supplementary Table 2. When the log-linearity assumption was not rejected (p for non-linearity ≥0.05 in the Restricted Cubic Splines models), the p for linear trend (marked with *) was retained (obtained by coding the exposure as an ordinal categorical variable 1, 2, 3). When the assumption of log-linearity was not met (p for non-linearity <0.05), it was not adapted to calculate a p for linear trend, thus, the likelihood ratio overall p value (marked with §) was retained (obtained by coding the exposure as a non-ordinal categorical variable and calculating likelihood ratio test between models with and without the studied food additive exposure variable). cMultivariable Cox proportional hazard models (two-sided, statistically significant p value < 0.05) (see Methods for adjustment strategy). FDR-corrected p values for multiple test correction are available in footnote e of Supplementary Table 2. We also tested associations between intakes of acetic acid, citric acid, nitrates, nitrites, sulfites, vitamin C, and vitamin E from naturally occurring sources and type 2 diabetes incidence. Naturally occurring acetic acid and nitrates from drinking water were associated with increased type 2 diabetes incidence, while natural sulfites were associated with lower incidence (Supplementary Table 4). In this population study, ultra-processed food exposure (% weight in the diet) was associated with higher type 2 diabetes incidence (HR = 1.20, 95% CI [1.11–1.29], p value < 0.001). We computed a mediation analysis to assess the proportion of this association mediated by preservative food additives that were related to type 2 diabetes in this study (i.e., potassium sorbate (E202), potassium metabisulfite (E224), sodium nitrite (E250), acetic acid (E260), sodium acetates (E262), calcium propionate (E282), sodium ascorbate (E301), tocopherol-rich extract (E306), alpha-tocopherol (E307), sodium erythorbate (E316), citric acid (E330), phosphoric acid (E338), and extracts of rosemary (E392)). In all, 17% of the association between ultra-processed food and type 2 diabetes was mediated by exposure to these preservatives (p value of the mediated proportion = 0.01). This large prospective cohort study revealed associations of higher type 2 diabetes incidence with higher intakes of several widely used preservative non-antioxidant food additives (potassium sorbate, potassium metabisulfite, sodium nitrite, acetic acid, sodium acetates, and calcium propionate) and preservative antioxidant food additives (sodium ascorbate, tocopherol-rich extract, alpha-tocopherol, sodium erythorbate, citric acid, phosphoric acid, and extracts of rosemary). For some preservative food additives, EFSA data were available to compare intake levels with those observed in our population study (Supplementary Table 5). The order of magnitude was consistent overall. Relatively similar intakes were observed for sorbates (E200-E203), ascorbates (E300-E302), extracts of rosemary (E392), nitrates (E251-E252), propionates (E280-E283), and lecithins (E322). Compared to EFSA, intake tended to be lower in NutriNet-Santé for sulfites (E220-E228), nitrites (E249-E250), and alpha-tocopherol (E307) and higher for erythorbates (E315-E316). These differences may be due to: (1) the differences between exposure assessment methods (with more precise data in the NutriNet-Santé cohort, based on brand-specific repeated 24HDRs versus generic food items and a generally lower number of records or recalls in studies on which EFSA estimates are based); (2) the differences in dates of assessment and study populations, with for instance more women, older, and more-health conscious participants in NutriNet-Santé compared to the French general population. No other cohort study investigated the associations between intakes of preservative food additives and type 2 diabetes incidence, probably due to a lack of data regarding specific industrial foods consumed and thus their additive content, which varies greatly from one brand to another. Thus, comparison with epidemiological literature is not straightforward. Consistent with our findings, a recent meta-analysis observed a higher risk of type 1 or 2 diabetes associated with higher exposure of dietary nitrites and no association with nitrates. This meta-analysis included five studies based on food frequency questionnaires (not enabling differentiation between naturally occurring and food additive sources) as well as previous NutriNet-Santé results (based on brand-specific data, enabling differentiation between sources)17. Indeed, our group previously published a specific study on nitrites and nitrates and type 2 diabetes incidence in NutriNet-Santé3. Despite an updated methodology (now using time-dependent cumulative exposure) and longer follow-up (+2 years), the results remain consistent. As several public health authorities consider a possible ban of nitrites and nitrates as food additives, altogether, these data consolidate the conclusions that higher exposure to additive-originated nitrites was associated with higher type 2 diabetes incidence, while no association with nitrate additives was detected3. To our knowledge, no other study investigated nitrite/nitrate additives and type 2 diabetes incidence. Regarding antioxidants, a recent systematic review and meta-analysis of randomised controlled trials (RCTs) showed no benefit of vitamins E, C or beta-carotene supplementation on type 2 diabetes risk16. None of the included studies provided data specifically on preservative antioxidant food additives. An RCT conducted in Iran focused on patients with non-alcoholic fatty liver disease and detected no effect of rosemary leaf extract on glycaemic status18. Some prospective studies investigated the associations between blood or urinary biomarkers of some compounds that may partly result from exposure to exogenous preservative food additives, but also largely from endogenous metabolism, greatly limiting the comparison with our results. For instance, a nested case-control from the Women's Interagency HIV Study in the United States observed that plasma sorbic acid was associated with greater odds of incident type 2 diabetes19. Another US prospective study showed that elevated serum calcium phosphate, a derived compound of phosphoric acid, was associated with an increased risk of developing type 2 diabetes20. 4, many mechanisms may underpin the associations detected in this study, including metabolic and inflammatory disruption and/or altered insulin signalling pathway. An in vitro study (hepatic cells) on food additives revealed that lecithins showed no cytotoxicity or genotoxicity, while potassium sorbate, sodium nitrite, sodium ascorbate, and sodium erythorbate were cytotoxic21. Potassium sorbate specifically acts as an AGE activator with and without glucose15, while high AGE intake is linked to type 2 diabetes22. Mouse model studies showed that coadministration of sorbate and fructose (a monosaccharide naturally found in many foods and drinks) leads to altered liver function (steatosis, inflammation, fibrosis) associated with altered expression of genes involved in lipid metabolism23. These observations are associated with disturbances in intestinal myco- and microbiota, while other studies have confirmed sorbate's potential proinflammatory effects at hepatic and microbiota levels24. Investigating the impact of potassium sorbate on pancreatic inflammation, related to type 2 diabetes onset25, would be interesting. Regarding the insulin (INS) pathway, experiments on rat models showed the particular role of N-nitroso compounds in the development of INS resistance via disruption of both INS and IGF pathways and dysfunction of pancreatic β-cells8,9 and that co-exposure to N-nitrosodiethylamine (NDEA) and fat (found in nitrited-processed meat) causes INS and IGF-1 resistance associated with type 2 diabetes26. Studies on mice and humans also reported that propionate impaired insulin action27. In a rodent study, chronic supplementations of pharmacological doses of vitamins C and E increased fasting blood glucose, insulin, and homoeostasis model assessment index for insulin resistance (HOMA—a method used to quantify insulin resistance and beta-cell function)10. Vitamin C impaired glucose tolerance by disrupting upstream hepatic insulin action, while vitamin E acted downstream insulin receptors, reducing, for example, glucose transporter-2 expression. Sodium ascorbate, which converts to vitamin C, and vitamin C isomers like erythorbates may share similar roles in metabolism. In contrast, an in vitro study found that alpha-tocopherol decreases superoxide anion release in human monocytes under hyperglycaemic conditions via inhibition of protein kinase C-alpha28. The effects of one substance may vary based on factors such as food matrix (of which composition, structure, pH and other characteristics may affect the way bioactive compounds are assimilated and digested by gut microbiota and host organisms), dosage, and interaction with other elements, impacting the bioavailability of oil-soluble vitamins like vitamin E, for instance29,30. The TCA cycle is connected to several metabolisms (including fat, amino acid, glucose metabolisms)—changes in the levels of its metabolites may impact the overall metabolism. For example, inhibiting citrate cotransporter Slc13a5/mINDY improved hepatic insulin sensitivity11. Interestingly, several food additives like citrate, propionate (after conversion into succinyl-CoA), and acetate (after conversion into acetyl-CoA) connect with the TCA cycle12, and it would be interesting to test the links between such exposure and the activity of this specific metabolism. No direct comparison was possible concerning phosphoric acid; however, a US cross-sectional study found that the phosphate-regulating hormone fibroblast growth factor (FGF23) may be considered a biomarker for declining metabolic function linked to inflammation14. In our study, both linear and non-linear relationships were observed. In terms of public health implications, these results suggest that a reduction in food additive preservative exposure might be beneficial to type 2 diabetes prevention. As per the NOVA definition, food preservatives are not necessarily markers of ultra-processing (unlike other food additives such as artificial sweeteners or colours). The proportion of additive preservatives from ultra-processed foods in this population study was 34.6%. This probably contributed to the fact that results were still statistically significant after adjustment for the proportion of ultra-processed foods in the diet. However, this is not incompatible with the fact that some of these additives could contribute to explaining part of the association between ultra-processed foods and elevated risk of type 2 diabetes31,32. In practice, to reduce exposure to certain food additive preservatives such as sulfites or nitrites, for which the primary sources are specific food groups that, in addition, have no particular nutritional value (i.e., alcoholic beverages and processed meats, respectively), it is advisable to limit consumption of these food/beverage groups. However, several preservative food additives (e.g., potassium sorbate, calcium propionate, erythorbates) are ubiquitously used across many food groups, with a huge variability in ingredient lists depending on the brands for the same generic food item. This has two public health implications. First, in terms of recommendations to the public, this leads to the formulation of a general guideline aimed at limiting unnecessary preservative additives (choosing preservative-free alternatives whenever possible). For example, cooking at home and consuming only unprocessed or minimally processed fruit and vegetables would avoid around 25% of total food preservatives. Second, this reinforces the idea that measures targeting individuals (such as disseminating recommendations) will not be sufficient and that policy actions must be implemented in parallel to deeply transform the food supply and reduce exposure. This includes, for example, re-evaluating these additives and, if necessary, amending regulations on authorised substances and doses in order to better protect the population. This study was based on a large and prospective cohort with highly detailed brand-specific 24HDRs along 14 years of follow-up (allowing time-dependent cumulative exposure assessment), thereby providing access to unique information on exposure to preservative food additives. However, several limitations should be acknowledged. First, the observational design does not allow causal interpretation for the studied associations based on this study alone. Residual confounding cannot be fully ruled out. Yet, multivariable models have been adjusted for a broad spectrum of potential confounding factors, thereby limiting the potential for bias. In particular, the food vectors of preservative additives were very diverse, encompassing contrasted nutritional profiles in terms of sugar, salt, fat and fibre content. For instance, for potassium sorbate, 26.3% of the intake came from fruit- and vegetable-based products while 21.6% came from fats and sauces. To limit as much as possible confounding bias linked to nutritional profiles of vector foods, all models were carefully adjusted for energy, saturated fats, sodium, dietary fibre, and sugar intakes. Second, as in other studies investigating health and diet in which people enroll voluntarily, this study included more women, with a higher educational level and healthier lifestyles than the general French population33,34. Therefore, caution is needed in generalising the findings. However, daily energy intake as well as proportion of energy by ultra-processed foods were similar in our population study compared to estimates from French nationally representative surveys, supporting the generalisability of our findings35,36. Ethnic representation race/ethnicity and religion were not available in the cohort due to a very restrictive ethical/legal regulation policy regarding the collection of these data in French epidemiological studies (specific authorisations needed). Nearly 95% of the French population has access to the Internet38, and we have shown that the study population was not limited to digitally fluent individuals (with about a quarter of participants reporting being inexperienced in terms of computer use)39. In NutriNet-Santé, we aimed to have enough contrast between different categories of dietary exposures to conduct aetiological analyses, while accounting for a wide diversity of lifestyle profiles. Third, besides the characteristics of this population (more women, more highly educated, healthier lifestyles), combined with a young minimum age of enrollment (15+), potentially explains the observed low rate of ascertained type 2 diabetes (1.04% in this study against 5.6% people being medicated for diabetes in France40). Multisource case ascertainment combining participant declaration of a medical diagnosis of type 2 diabetes by a physician with yearly registration of all medication use linked to the Vidal® database and the link with medico-administrative databases limited the risk of not detecting diagnosed cases. Moreover, nationally-representative French data estimated that only 1.7% adults had undiagnosed diabetes in France41, limiting the potential bias due to undiagnosed diabetes cases. Fourth, classification bias can never be totally excluded. However, the estimation of dietary intakes (nutrients and food additives) in NutriNet-Santé is among the most accurate globally in a cohort study. + commercial brands), and their update during follow-up. Nevertheless, specific exposure to preservative food additives was not part of these validation studies. Indeed, validation of additive exposure versus plasmatic or urinary biomarkers would be ideal; however, it can be challenging, and even impossible for most food additives, including most preservatives, due to the lack of measurable biomarkers, since most additives are metabolised into substances that are non-specific and/or ubiquitous in human biofluids. However, the accuracy of the qualitative and quantitative food additive exposure assessment of the NutriNet-Santé cohort represents an important strength of the study, thanks to detailed and repeated 24HDRs, linked to multiple food composition databases (OQALI, Open Food Facts, GNPD, EFSA, and GSFA), ad-hoc laboratory assays in food matrices, and dynamic matching to account for reformulations of industrial food items over time. Besides, intakes of food additive preservatives in this study mostly aligned with EFSA estimates. Using the Australian Food Composition Database to estimate naturally occurring acetic and citric acids was not optimal since country variations may occur, but French/European composition tables were not available or less complete for these natural sources. Similarly, quantifying the natural intake of some substances was impossible due to limited data (e.g., natural lecithins). Fifth, participants were invited to complete three 24HDRs every 6 months. Since the dietary record update was not mandatory, not all participants completed all records (in this case, their intakes of the previous period were kept). Therefore, we used updated intake values for each participant whenever available, making the most of all follow-up data, and systematically adjusting for the number of answered 24HDRs in all models. Next, it was not feasible to examine the association of several less frequently consumed preservative food additives with type 2 diabetes. However, these limited proportions of consumers reflected a low occurrence in the French market, indicating a reduced potential public health impact for these substances. Last, results were presented with and without adjustment for multiple testing by the False Discovery Rate method, with mostly stable results. If adjusting for multiple testing decreases type I error, it also increases type II error (risk of false negative) and may lead miss out on existing associations, which is why this adjustment is debated45. Besides, our results were supported by mechanistic plausibility, and all detected associations in our study showed HRs >1, strongly suggesting they were not going in random directions (which would have been the case if they were due to chance). This large prospective cohort showed associations between exposure to several preservative food additives and higher type 2 diabetes incidence. These findings may have important public health implications given the ubiquitous use of preservatives in a wide range of foods and drinks. Although these results need to be confirmed by other epidemiological studies (since no causal conclusion can be drawn from a single observational study), they are consistent with experimental data suggesting adverse metabolic-related effects of several of these compounds. This calls for a re-evaluation of the safety of these additives and supports recommendations for consumers to favour fresh and minimally processed foods and limit superfluous additives whenever possible. The study is registered at https://clinicaltrials.gov/ct2/show/NCT03335644 and is conducted according to the Declaration of Helsinki guidelines and approved by the Institutional Review Board of the French Institute for Health and Medical Research (IRB-Inserm) and the “Commission Nationale de l'Informatique et des Libertés” (CNIL n°908450/n°909216). Each participant provides an electronic informed consent form before enrollment. Participants aged 15 and above are invited to participate in the study via a dedicated web-based platform (https://etude-nutrinet-sante.fr/) and regularly answer questionnaires on their dietary intakes, health, anthropometric47,48, physical activity49, lifestyle, and socio-demographic data50. No compensation was offered to the participants. There was no specific hypothesis of an interaction between studied preservatives and sex, thus results are presented overall. However, all cut-offs for exposure categories were sex-specific (and presented by sex in Supplementary Table 6). Upon registration and every 6 months, participants were asked to complete in sequences of three validated42,43,44 web-based 24-h dietary records (24HDRs). At each period, 24HDRs were randomly assigned to three non-consecutive days over 2 weeks (2 weekdays and 1 weekend day, to account for variability in the diet across the week and the seasons). At all times throughout their assigned dietary record period, participants had access to a dedicated interface on the study website to declare all foods and beverages consumed during 24 h: three main meals (breakfast, lunch, dinner) and any other eating occasion. Participants were asked to estimate portion sizes either by directly entering the weight consumed in the platform or by using validated photographs or usual containers51. The NutriNet-Santé food composition database (>3500 items) was used to estimate mean daily energy, alcohol, macro- and micro-nutrient intakes (including vitamins C and E)52. This database integrated all available data from the French national composition database (CIQUAL53), and further added information on additional components (e.g., food additive, trans fatty acids, etc.). These estimates included contributions from composite dishes using French recipes validated by food and nutrition professionals. The web-based questionnaires used in the study have been tested and validated against both in-person interviews by trained dietitians and urinary and blood markers42,43. In this analysis, we included participants having at least two 24-h dietary records during the first 2 years of follow-up. Participants who underreported their energy intake were excluded from the analyses and were identified using the method from Black, based on the original method developed by Goldberg et al.54,55. This method relies on the hypothesis that the maintenance of a stable body weight requires a balance between energy intake and expenditure. The equations developed by Black account for the reported dietary energy intake, basal metabolic rate (calculated using Schofield's equations), sex, age, height, weight, number of dietary records, physical activity level (PAL), and intra/inter-individual variability56. As recommended by Black, the intra-individual coefficients of variation for BMR and PAL were fixed at 8.5% and 15%, respectively. In addition, a PAL of 1.55 was used to reflect a “light” physical activity, which is assumed to be attained by healthy, normally active individuals living a sedentary lifestyle. Finally, some individuals identified as under-reporters of energy intakes using Black's method were not excluded if they also reported recent weight variations, adherence to weight-loss restrictive diets, or declared the consumptions entered in their dietary records as unusually low compared to their habitual diets. This ensured that flagged under-reporters have true incoherent reporting, and must be excluded. Although their exclusion may limit the generalisability of the findings, it was necessary to avoid important exposure classification bias. In this study, 23,098 participants (corresponding to 17.2% of the subjects) were considered under-energy reporters and were excluded from the study. This proportion of under-reporters is common, for instance, in the nationally representative INCA 3 study conducted in 2016 by the French Food Safety Agency, 18% of adult participants were identified as under-reporters using the Black method35. Several quality control operations were performed to account for over-reporting. Limitations in the online tool were set when participants reported the quantities of food consumed, aiming to alert them that the number they were about to enter was potentially an outlier, thereby encouraging double-checking and correction. These limitations are based on the 99th percentile of energy intake and are updated every 10,000 new dietary records added to the cohort if more than 10% of reported food items had outliers, then the full record was excluded. In this study, 54 participants (corresponding to 0.04% of the subjects) were considered over-energy reporters and were excluded from the analyses. Participants' intakes of naturally occurring acetic and citric acids, nitrites, nitrates, and sulfites were quantified using multiple sources (see the Methods on Preservative Food Additive Intakes). Assessment of food additive intake in the NutriNet-santé cohort through brand-specific data of the 24HDRs has been previously described57. Each industrial food item consumed and reported in a specific dietary record was matched against three databases to assess the presence of any food additive: OQALI, a national database whose management has been entrusted to the National Research Institute for Agriculture, Food and Environment (INRAE) and the French food safety authority (ANSES) to characterise the quality of the food supply (https://www.oqali.fr/); Open Food Facts, an open collaborative database containing millions of food products marketed worldwide (https://world.openfoodfacts.org/); and the Mintel Global New Products Database (GNPD), an online database of innovative food products in the world (https://www.mintel.com). In a second step, the dose of food additive contained in each food item was estimated based on (1) ad hoc laboratory assays quantifying additives in specific food items (n = 2677 food-additive pairs analysed), (2) doses in generic food categories provided by the European Food Safety Authority (EFSA), or (3) generic doses from the Codex General Standard for Food Additives (GSFA)58. Dynamic matching was applied, meaning that products were matched date-to-date: the date of consumption of each food or beverage declared by each participant was used to match the product to the closest composition data available, thus accounting for potential reformulations. The 80 preservative food additives listed in the Codex General Standard for Food Additives database59 or UK Food Standard Agency60 were eligible for the present study. However, the occurrence of some of them was very low in the French/European markets, thus the proportion of consumers was null; their list is provided in the footnote to Tables 2 and 3. We decided to include as food additive preservatives both preservatives per se as defined by Regulation (EC) No 1333/20086 and antioxidant food additives, as both prevent the spoilage of food (food additive antioxidants preserving food via an antioxidant mode of action). In this paper, the term “preservative food additives” includes both “preservative non-antioxidant food additives” and “preservative antioxidant food additives”. All food additives with preservative properties are included in the present paper. This unique level of detail is permitted by the fact that commercial names/brands of industrial products consumed were collected and matched with Open Food Facts, OQALI, and GNPD databases, providing the ingredient list and thus, the presence of the specific food additive, at the time when the product was consumed. Then, the quantitative assessment of the doses of additives in the products that contain a specific additive is challenging since manufacturers are not compelled to declare this information on the packaging. Hence, the 3-step method was used to assess doses in our cohort. In all, in the framework of the ADDITIVES project, we performed 2677 quantified analyses, corresponding to a total of 61 food additives in 196 different (generic) food items. “Pairs” (i.e., a specific additive in a specific food vector) selected for laboratory assays corresponded to the most frequently consumed and most emblematic commercial food/beverage items for a given additive. Specifically, for preservative food additives, we had access to 1138 laboratory quantified analyses corresponding to 58 preservative food additives (E200, E202, E203, E210, E211, E212, E220, E221, E222, E223, E224, E225, E228, E234, E235, E239, E242, E249, E250, E251, E252, E260, E261, E262, E263, E280, E281, E282, E285, E290, E300, E301, E302, E304, E306, E307, E307b, E307c, E310, E315, E316, E319, E320, E321, E322, E325, E326, E330, E332, E333, E334, E338, E385, E386, E392, E472c, E942, E1105) in 128 (generic) food items (several commercial brands were tested per food item, e.g., in the case of milk chocolate, milk chocolate with nuts, creamy desserts, omega-3 enriched margarines, sausages, jams, chocolate mousse…). In addition to the assays carried out by certified laboratories, which were sent to us by the consumer association UFC Que Choisir, we contacted two companies (Mérieux & Eurofins) and the Direction Générale de la Consommation, de la Concurrence et de la Répression des Fraudes (DGCCRF) to carry out these assays. Only the additives listed in their catalogue could be measured. If data were unavailable from this source, EFSA and GSFA doses were only applied if the specific food item contained the specific food additive in its ingredient list. We used 3122 preservative food additive data from EFSA (data available online in each EFSA Opinion + transmission of specific information by EFSA following an official Public Access to Document request PAD 2020/077), related to 46 preservative food additives present in 977 food categories. EFSA collects much information from manufacturers related to their specific commercial products, but for confidentiality reasons, only transfers information for generic food items or food groups (no brand-specific data). As for EFSA, data from GSFA are not brand-specific but relate to generic food items or food categories (Supplementary Fig. Overall, quantitative dose data from ad hoc assays and from databases were similar in magnitude (e.g., for food additive potassium sorbate (E202) in the brand-specific pre-packed carrot salad that we selected: laboratory assay = 0.987 mg/100 g vs. 1 mg/100 g in the EFSA database for the corresponding generic food item), which was not surprising since EFSA doses for instance correspond to an average of laboratory assay data received by the Agency from EU member states, manufacturers and various contributors. Food additive sulfites are present in many foods and drinks exempted from mandatory nutritional/ingredient declaration (e.g., wines or vinegar), making it sometimes impossible to determine which specific sulfite additive was used. Therefore, in this study, specific food additive codes (E220-E228) were used when the information was available on the packaging, i.e., for food items with mandatory ingredient declaration. The total sulfite variable accounts for all sulfite additives, i.e., both from foods and drinks with a mandatory ingredient list and from other food items with added sulfite (unspecified code). This strategy was established to avoid counting twice the same dose of sulfite (e.g., wine with declared ingredients). In order to adjust for intake from non-additive sources of a given substance, whenever composition data were available: Participants' intake of naturally occurring acetic and citric acids was quantified using the Australian Food Composition Database61, which has been matched to the NutriNet-Santé food composition database for this specific study. The methodology used to quantify intakes from non-food additive sources of nitrites and nitrates in foods and beverages has been previously described3,62,63. Briefly, food-originated nitrites and nitrates were determined by food category using EFSA's concentration levels for natural sources and contamination from agricultural practices7. The publicly available data from the French official regional sanitary control of tap water was used to estimate intakes via water consumption by region of residence64, via a municipality-specific merging according to the NutriNet-Santé participants' postal code, as well as a dynamic temporal merge according to the year of dietary records. Participants' intakes of naturally occurring and food additive sulfites were quantified using the corresponding EFSA Opinion7 and matching to the NutriNet-Santé database for this study. Intakes of non-food additive dietary vitamins C and E were computed from the NutriNet-Santé food composition database52. A multi-source approach was used to detect incident type 2 diabetes cases. Throughout follow-up, participants could report any health-related events, medical treatments, and examinations via the health questionnaires every 6 months or, at any time, directly via the health interface of their profile. Besides, the NutriNet-Santé cohort was linked to the national health insurance system database to collect additional information regarding medical treatments and consultations, and to the French national mortality registry to identify the occurrence and cause of death. We did not perform ad hoc biochemical assessment. Participants were asked to declare major health events through the yearly health questionnaire, through a specific health check-up questionnaire every 6 months, or at any time through a specific interface on the study website. They were also asked to declare all currently taken medications and treatments via the check-up and yearly questionnaires. A search engine with an embedded exhaustive Vidal® drug database is used to facilitate medication data entry for the participants. Besides, our research team was the first in France to obtain authorisation by Decree in the Council of State (n°2013-175) to link data from our general population-based cohorts to medico-administrative databases of the National Health Insurance. Thus, data from the NutriNet-Santé cohort were linked yearly to these medico-administrative databases, providing detailed information about the reimbursement of medication and medical consultations. An incident type 2 diabetes case is detected when a participant has either reported the pathology at least twice or reported it once along with the use of a related medication. All 1131 type 2 diabetes incident cases were primarily detected through the declaration by the participants of a type 2 diabetes diagnosed by a physician and/or diabetes medication use, in follow-up questionnaires. The questions were: “Have you been diagnosed with type 2 diabetes (if yes, indicate the date of diagnosis)” and “Are you treated for type 2 diabetes?”. ATC codes considered for type 2 diabetes medication were A10AB01, A10AB03, A10AB04, A10AB05, A10AB06, A10AC01, A10AC03, A10AC04, A10AD01, A10AD03, A10AD04, A10AD05, A10AE01, A10AE02, A10AE03, A10AE04, A10AE05, A10AE30, A10BA02, A10BB01, A10BB03, A10BB04, A10BB06, A10BB07, A10BB09, A10BB12, A10BD02, A10BD03, A10BD05, A10BD07, A10BD08, A10BD10, A10BD15, A10BD16, A10BF01, A10BF02, A10BG02, A10BG03, A10BH01, A10BH02, A10BH03, A10BX02, A10BX04, A10BX07, A10BX09, A10BX10, A10BX11, A10BX12. In addition to the abovementioned questions about the diagnosis of type 2 diabetes mellitus and/or a medication report, two additional sources of confirmation were considered. First, linkage with the medico-administrative databases confirmed more than 80% of the cases surveyed (ICD-10 codes E11). Second, among participants who provided a blood sample at the clinical/biological examination, 85.3% of those with elevated fasting blood glucose (i.e., ≥1.26 g/L) had consistently reported a diagnosis of type 2 diabetes mellitus and/or medication. However, elevated blood glucose without any declaration of type 2 diabetes diagnosis or treatment was not considered specific enough to classify the participant as a type 2 diabetes case. Participants from the NutriNet-Santé cohort who completed at least two 24HDRs during their first 2 years of follow-up, those who were not under- or over-energy reporters, and who did not have any prevalent type 1 or 2 diabetes at enrollment were included in the analysis (flowchart of participants presented in Fig. Baseline participants' characteristics were described as mean (SD) for quantitative variables and n (%) for qualitative variables for the overall population and per baseline sex-specific tertiles of total preservative food additives. A correlation matrix was generated to visualise the Spearman correlations between intakes of individual food additives (Supplementary Fig. For each studied additive or group of additives, participants were categorised into lower, medium, and higher consumers defined as sex-specific tertiles if the additive was consumed by at least 66% of participants, or non-consumers, and consumers separated by the sex-specific median otherwise (cut-offs provided in Supplementary Table 6). The relationship between preservative food additive intake coded as categorical a cumulative time-dependent exposure and the incidence of type 2 diabetes were investigated using multivariable proportional hazard cause-specific Cox models with age as the time scale to account for the competing mortality risk during the follow-up period. Hazard ratios (HR) and 95% confidence intervals (95%CI) were calculated. Participants contributed person-time to the models from their age at enrollment in the cohort (which corresponds to the completion of the first set of 24HDRs) until their age at the date of type 2 diabetes ascertainment, type 1 diabetes diagnosis, death, last contact, or 31 December 2023, whichever occurred first. A counting process structure was used with cumulative time-dependent dietary variables updated every 2 years. Exposure during a specific period was calculated using a weighted average of the most recent 2-year period and prior periods, thereby using all available dietary record data. The time-to-event data structure was used with time-dependent dietary variables updated every 2 years. Exposure during one period was computed using a weighted average of the most recent 2-year period and prior periods. Each period contained averaged food additive intakes from the available dietary records within each 2-year period. The maximum number of 2-year periods was seven, to cover the maximum follow-up period of 14 years. A cumulative exponential decay weighting scheme was used to attribute lower weights to recent periods and higher weights to more ancient ones, given the fact that physio-pathological mechanisms underlying potential causal associations between additive intake and diabetes onset are expected to take several years (food additive intake consumed the month before diagnosis is less likely to have caused diabetes onset than more ancient usual exposure). The same methodology was applied to all dietary data. Cut-offs for food additives were updated for each follow-up period and are provided in Supplementary Table 1. For incident cases, dietary data collected during periods after type 2 diabetes diagnosis were not accounted for. For incident cases, dietary data collected during periods after type 2 diabetes diagnosis were not accounted for. The principal model was adjusted for age (time-scale), sex, baseline BMI, height, physical activity, smoking status, number of smoked cigarettes in pack-years, educational level, family history of diabetes, number of dietary records completed, time-dependent daily intakes of energy without alcohol, alcohol, saturated fats, sodium, fibre, sugars, fruits and vegetables, dairy products, red and processed meats or heme iron (for nitrites and nitrates models only). Missing values for covariates were handled by a multiple imputation approach using additive regression, followed by bootstrapping, and predictive mean matching (n = 20 imputed dataset) as implemented in the Hmisc R package (version 5.1-0)65. Specifically, the imputation model included a comprehensive set of predictors deemed relevant to the missing covariates. Variables were incorporated to capture the underlying relationships and patterns. The choice of predictors (i.e., age, sex, family history of diabetes, physical activity, incident type 2 diabetes, education level, smoking status, BMI, energy intake, and alcohol intake) was guided by their known or hypothesised associations with the variables containing missing values. Missing values were imputed for the following variables: physical activity level (13.59% of missing values), BMI (2.76%), height (2.76%), smoking status (0.29%), number of smoked cigarettes in pack years (0.30%), education level (0.96%), and family history of diabetes (0.31%). Moreover, whenever relevant, models were adjusted for the intake of the corresponding substance coming from naturally occurring sources. Associations between intakes from these natural sources and type 2 diabetes incidence were also tested. 2) implemented in the survival R package (version 3.5-8)66. Restricted cubic splines with three knots covering each food additive distribution: 27.5th, 72.5th and 95th percentiles (Supplementary Fig. 3)67 were employed to investigate potential non-linear dose-response associations. When the log-linearity assumption was not rejected (p for non-linearity ≥0.05 in the Restricted Cubic Splines models), the p for linear trend was retained (obtained by coding the exposure as an ordinal categorical variable 1, 2, 3). When the assumption of log-linearity was not met (p for non-linearity <0.05), it was not adapted to calculate a p for linear trend; thus, the likelihood ratio overall p value was retained (obtained by coding the exposure as a non-ordinal categorical variable and calculating the likelihood ratio test between models with and without the studied food additive exposure variable). We have tested the associations between food preservative additive exposures and hip fracture (i.e., outcome with no expected causal relationship) as a “negative outcome control model”. We tested the proportion of the association between ultra-processed food68 intake and type 2 diabetes incidence that was mediated by food additive preservatives found associated with diabetes in this study, using the CMAverse R package (version 0.1.0)69 and the same adjustment variables as the main model. Sensitivity analyses were tested based on the main model: additional mutual adjustment for other preservative food additives intakes except the studied one (continuous, mg/d) (model 1); additional adjustment for the baseline weight proportion of ultra-processed food intake (model 2); additional adjustment for the diagnosis and/or treatment for at least one prevalent metabolic disorder (i.e. cardiovascular disease, dyslipidemia or hypertension) (model 3); additional adjustment for time-dependent intakes of total emulsifier and total artificial sweetener food additives (continuous, mg/d) (model 4) (these two categories of food additives have been associated with type 2 diabetes risk in NutriNet-Santé); additional adjustment for baseline intake of vitamin C (continuous, mg) and vitamin E (continuous, mg) from dietary supplements (model 5); additional adjustment for trans fatty acids intake (continuous, mg) (model 6); adjustment for PCA-derived dietary patterns rather than individual food groups (continuous, see Method for deriving dietary patterns by principal component analysis and corresponding factor loadings for determination of dietary patterns) (model 7); additional adjustment for time-dependent intakes of polyunsaturated fatty acids (continuous, g/d) and heme iron (continuous, mg/d) (model 8) (for preservative antioxidant food additives only, since antioxidant may counteract polyunsaturated fatty acid peroxidation by heme iron); follow-up starting at the end of the first 2-year period (model 9); without exclusion of under-reporters (model 10); use of marginal structural models with stabilised and truncated at 99th percentile inverse probability of treatment weighting as recommended by Young et al.70 (model 11); relaxing log-linearity assumption on covariates adding splines with the R package splines (version 4.3.3) (model 12). All statistical analyses were conducted with R version 4.3.3, except for the restricted cubic spline method, which was implemented with SAS version 9.467. Dietary patterns were identified based on 20 food categories, using a principal component analysis conducted with the R package FactoMineR (version 2.8)71. The principal component analysis creates linear combinations (called principal components) of the initial set of variables, with the aim to group those that are correlated while explaining as much variation from the dataset as possible. Finally, we calculated an adherence score for each principal component and for each participant, using the food categories factor loadings to weigh the sum of all observed intakes. Thus, the adherence score measures a participant's diet conformity to the identified dietary pattern intake pattern. In the analyses of dietary patterns, we identified a healthy pattern (explaining 10.88% of the variance), which was characterised by higher intakes of fish and seafood, fruits, unsweetened soft drinks, vegetables, and wholegrains, along with lower intakes of sweetened soft drinks. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The raw data underlying this study are protected and are not available due to data privacy laws, since they are part of a running cohort with multiple ongoing investigations and are subject to national and European regulations for the protection of individual sensitive health data. However, researchers from public institutions without financial conflict of interest can submit a request to have access to the data for strict reproducibility analysis (systematically accepted) or for a new collaboration, including information on the institution and a brief description of the project and sources of funding to collaboration@etude-nutrinet-sante.fr. All requests will be reviewed by the steering committee of the NutriNet-Santé study, and an answer will be provided within 2 months. If the collaboration is accepted, a data access agreement will be necessary, and appropriate authorisations from the competent administrative authorities may be needed. The duration of data access is decided on a case-by-case basis, depending on the complexity of analyses to be run. In accordance with existing regulations, no personal identifying data will be accessible. Source data are provided with this paper. The R and SAS code used in this study is provided in Supplementary Materials R and SAS code for statistical analyses and on Code Ocean. Artificial sweeteners and risk of type 2 diabetes in the prospective NutriNet-Santé cohort. Salame, C. et al. Food additive emulsifiers and the risk of type 2 diabetes: analysis of data from the NutriNet-Santé prospective cohort study. Srour, B. et al. Dietary exposure to nitrites and nitrates in association with type 2 diabetes risk: results from the NutriNet-Santé population-based cohort study. Payen De La Garanderie, M. et al. Food additive mixtures and type 2 diabetes incidence: results from the NutriNet-Santé prospective cohort. Regulation (EC) No 1333/2008 (European Parliament & Council, 2008). Tong, M. et al. Nitrosamine exposure causes insulin resistance diseases: relevance to type 2 diabetes mellitus, non-alcoholic steatohepatitis, and Alzheimer's disease. Tong, M., Longato, L. & de la Monte, S. M. Early limited nitrosamine exposures exacerbate high fat diet-mediated type 2 diabetes and neurodegeneration. Ali, M. A., Eid, R. M. H. M. & Hanafi, M. Y. Vitamin C and E chronic supplementation differentially affect hepatic insulin signaling in rats. Brachs, S. et al. Inhibition of citrate cotransporter Slc13a5/mINDY by RNAi improves hepatic insulin sensitivity and prevents diet-induced non-alcoholic fatty liver disease in mice. Arnold, P. K. & Finley, L. W. S. Regulation and function of the mammalian tricarboxylic acid cycle. Panyod, S. et al. Common dietary emulsifiers promote metabolic disorders and intestinal microbiota dysbiosis in mice. Hanks, L. J., Casazza, K., Judd, S. E., Jenny, N. S. & Gutiérrez, O. M. Associations of fibroblast growth factor-23 with markers of inflammation, insulin resistance and obesity in adults. Taghavi, F. et al. Potassium sorbate as an AGE activator for human serum albumin in the presence and absence of glucose. Nguyen, N. N. et al. Dietary nitrate, nitrite, and nitrosamine in association with diabetes: a systematic review and meta-analysis. Akbari, S. et al. Effect of rosemary leaf powder with weight loss diet on lipid profile, glycemic status, and liver enzymes in patients with nonalcoholic fatty liver disease: a randomized, double-blind clinical trial. Plasma metabolomic analysis indicates flavonoids and sorbic acid are associated with incident diabetes: A nested case-control study among Women's Interagency HIV Study participants. & Haffner, S. M. Calcium and phosphate concentrations and future development of type 2 diabetes: the Insulin Resistance Atherosclerosis Study. Recoules, C., Touvier, M., Pierre, F. & Audebert, M. Evaluation of the toxic effects of food additives, alone or in mixture, in four human cell models. Lu, X. et al. Effect of dietary intake of advanced glycation end products on biomarkers of type 2 diabetes: a systematic review and meta-analysis. Hrncir, T., Trckova, E. & Hrncirova, L. Synergistic effects of fructose and food preservatives on metabolic dysfunction-associated steatotic liver disease (MASLD): from gut microbiome alterations to hepatic gene expression. Effects of potassium sorbate on systemic inflammation and gut microbiota in normal mice: a comparison of continuous intake and washout period. Xourafa, G., Korbmacher, M. & Roden, M. Inter-organ crosstalk during development and progression of type 2 diabetes mellitus. de la Monte, S. M., Tong, M., Lawton, M. & Longato, L. Nitrosamine exposure exacerbates high fat diet-mediated type 2 diabetes mellitus, non-alcoholic steatohepatitis, and neurodegeneration with cognitive impairment. The short-chain fatty acid propionate increases glucagon and FABP4 production, impairing insulin action in mice and humans. Venugopal, S. K., Devaraj, S., Yang, T. & Jialal, I. α-Tocopherol decreases superoxide anion release in human monocytes under hyperglycemic conditions via inhibition of protein kinase C-α. Improving the bioavailability of oil-soluble vitamins by optimizing food matrix effects: a review. Comparison with ancestral diets suggests dense acellular carbohydrates promote an inflammatory microbiota, and may be the primary dietary cause of leptin resistance and obesity. Srour, B. et al. Ultraprocessed food consumption and risk of type 2 diabetes among participants of the NutriNet-Santé prospective cohort. Lane, M. M. et al. Ultra-processed food exposure and adverse health outcomes: umbrella review of epidemiological meta-analyses. Comparison of dietary intakes between a large online cohort study (Etude NutriNet-Santé) and a Nationally Representative Cross-Sectional Study (Etude Nationale Nutrition Santé) in France: addressing the issue of generalizability in E-epidemiology. Etude Individuelle Nationale des Consommations Alimentaires 3 (INCA 3) (ANSES, 2017). Consumption of ultra-processed food and its association with sociodemographic characteristics and diet quality in a representative sample of French adults. Lessons learned from methodological validation research in E-epidemiology. Internet Access and Use in the European Union. Pouchieu, C. et al. How computer literacy and socioeconomic status affect attitudes toward a Web-based cohort: results from the NutriNet-Santé study. Le diabète en France continue de progresser. Prevalence of prediabetes and undiagnosed type 2 diabetes in France: results from the National Survey ESTEBAN, 2014–2016. Lassale, C. et al. Correlations between fruit, vegetables, fish, vitamins, and fatty acids estimated by web-based nonconsecutive dietary records and respective biomarkers of nutritional status. Lassale, C. et al. Validation of a web-based, self-administered, non-consecutive-day dietary record tool against urinary biomarkers. Comparison between an interactive web-based self-administered 24 h dietary record and an interview by a dietitian for large-scale epidemiological studies. No adjustments are needed for multiple comparisons. The Nutrinet-Santé Study: a web-based prospective study on the relationship between nutrition and health and determinants of dietary patterns and nutritional status. Lassale, C. et al. Validity of web-based self-reported weight and height: results of the Nutrinet-Santé study. Comparison between web-based and paper versions of a self-administered anthropometric questionnaire. Committee, I. R. Guidelines for data processing and analysis of the International Physical Activity Questionnaire (IPAQ)-Short and Long Forms. Vergnaud, A.-C. et al. Agreement between web-based and paper versions of a socio-demographic questionnaire in the NutriNet-Santé study. Moullec, N. et al. Validation du manuel-photos utilisé pour l'enquête alimentaire de l'étude SU.VI.MAX. Arnault, N. et al. Table de composition des aliments, étude NutriNet-Santé. [Food composition table, NutriNet-Santé study] (in French) (2013). Ciqual French Food Composition Table (ANSES, 2020). Black, A. E. Critical evaluation of energy intake using the Goldberg cut-off for energy intake:basal metabolic rate. A practical guide to its calculation, use and limitations. Black, A. E. The sensitivity and specificity of the Goldberg cut-off for EI:BMR for identifying diet reports of poor validity. Schofield, W. N. Predicting basal metabolic rate, new standards and review of previous work. Exposure to food additive mixtures in 106,000 French adults from the NutriNet-Santé cohort. Srour, B. et al. Nitrites, nitrates, and cardiovascular outcomes: are we living “La Vie en Rose” with pink processed meats? Desquilbet, L. & Mariotti, F. Dose-response analyses using restricted cubic spline functions in public health research. Martinez-Steele, E. et al. Best practices for applying the Nova food classification system. & VanderWeele, T. CMAverse: A Suite of Functions for Causal Mediation Analysis. A causal framework for classical statistical estimands in failure time settings with competing events. We thank Thi Hong Van Duong, Régis Gatibelza, Amelle Aitelhadj and Aladi Timera (computer scientists) and Selim Aloui (IT manager); Julien Allegre, Nathalie Arnault, Nicolas Dechamp (data managers/statisticians); Paola Yvroud (health event validator & operational coordinator); Maria Gomes and Mirette Foham (participant support); and Marie Ajanohun, Tassadit Haddar (administration and finance) and Nadia Khemache (administrative manager) for their technical contribution to the NutriNet-Santé study. The NutriNet-Santé study was supported by the following public institutions: Ministère de la Santé, Santé Publique France, Institut National de la Santé et de la Recherche Médicale (INSERM), Institut National de la Recherche pour l'agriculture, l'alimentation et l'environnement (INRAE), Conservatoire National des Arts et Métiers (CNAM), and University Sorbonne Paris Nord. This project has received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation program (grant agreement No 864219, ADDITIVES, M.T. ), the French Ministry of Health (arrêté 29.11.19, M.T. ), and a Bettencourt-Schueller Foundation Research Prize 2021 (M.T.). A.H. is funded by a PhD grant from the French Ministry of Health/Sorbonne Paris Nord University. This project was awarded the NACRe (French Network for Nutrition and Cancer Research) Partnership Label (M.T.). ERC-2024-CoG-101170920) from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation program, and the national program “Microbiote” from INSERM. This work only reflects the authors' view, and the funders are not responsible for any use that may be made of the information it contains. Where authors are identified as personnel of the International Agency for Research on Cancer/World Health Organization, the authors alone are responsible for the views expressed in this article and they do not necessarily represent the decisions, policy or views of the International Agency for Research on Cancer/World Health Organization. Université Sorbonne Paris Nord and Université Paris Cité, INSERM, INRAE, CNAM, Centre for Research in Epidemiology and StatisticS (CRESS), Nutritional Epidemiology Research Team (EREN), F-93017, Bobigny, France Anaïs Hasenböhler, Guillaume Javaux, Marie Payen de la Garanderie, Fabien Szabo de Edelenyi, Laurent Bourhis, Cédric Agaësse, Alexandre De Sa, Chantal Julia, Emmanuelle Kesse-Guyot, Benjamin Allès, Léopold K. Fezeu, Serge Hercberg, Mélanie Deschasaux-Tanguy, Emmanuel Cosson, Sopio Tatulashvili, Bernard Srour & Mathilde Touvier Anaïs Hasenböhler, Marie Payen de la Garanderie, Inge Huybrechts, Fabrice Pierre, Xavier Coumoul, Emmanuelle Kesse-Guyot, Serge Hercberg, Mélanie Deschasaux-Tanguy, Benoit Chassaing, Bernard Srour & Mathilde Touvier International Agency for Research on Cancer, World Health Organization, Lyon, France Toxalim (Research Centre in Food Toxicology), Université de Toulouse, INRAE, ENVT, INP-Purpan, UPS, Toulouse, France Public Health Department, Groupe Hospitalier Paris-Seine-Saint-Denis, Assistance Publique-Hôpitaux de Paris (AP-HP), Bobigny, France Diabetology, Endocrinology and Nutrition Department, Avicenne Hospital, AP-HP, Bobigny, France Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors' contributions were as follows to A.H. and M.T. developed the additives composition database and matched consumption/composition data. supervised statistical analysis; A.H.: drafted the manuscript; M.T. : supervised the writing; All authors (A.H., G.J., M.P.G., F.S.E., L.B., C.A., A.D.S., I.H., F.P., X.C., C.J., E.K.G., B.A., L.K.F., S.H., M.D.T., E.C., S.T., B.C., B.S., and M.T. ): contributed to the data interpretation and revised each draft for important intellectual content. All authors read and approved the final manuscript. attest that all listed authors meet authorship criteria and that no others meeting the criteria have been omitted. Correspondence to Anaïs Hasenböhler or Mathilde Touvier. The authors declare no competing interests. Nature Communications thanks Emily Sonestedt, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Hasenböhler, A., Javaux, G., Payen de la Garanderie, M. et al. Associations between preservative food additives and type 2 diabetes incidence in the NutriNet-Santé prospective cohort. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-02021-5'>Monovalent pseudo-natural products supercharge degradation of IDO1 by its native E3 KLHDC3</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 11:15:22
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Targeted protein degradation modulates protein function beyond the inhibition of enzyme activity or protein–protein interactions. Most degrader drugs function by directly mediating the proximity between a neosubstrate and a hijacked E3 ligase. Here we identify pseudo-natural products derived from (−)-myrtanol, termed iDegs, that inhibit and induce degradation of the immunomodulatory enzyme indoleamine-2,3-dioxygenase 1 (IDO1) by a distinct mechanism. iDegs boost IDO1 ubiquitination and degradation by the cullin-RING E3 ligase CRL2KLHDC3, which we identified to natively mediate ubiquitin-mediated degradation of IDO1. Therefore, iDegs increase IDO1 turnover using the native proteolytic pathway. In contrast to clinically explored IDO1 inhibitors, iDegs reduce the formation of kynurenine by both inhibition and induced degradation of the enzyme and thus also modulate the non-enzymatic functions of IDO1. Current strategies for small-molecule-mediated targeted protein degradation (TPD)1 rely on compound-mediated induction of proximity between a neosubstrate and an E3 ubiquitin ligase2. Such ‘degraders' can either be heterobifunctional, harbouring distinct entities for the target and E3 (‘PROTACs'), or monovalent, binding to either the target or the ligase to adapt its surface and induce a cooperative tripartite assembly (molecular glue degraders, MGDs). Both degrader types have entered clinics3, and the discovery of alternative chemotypes that may unlock previously unexplored TPD strategies is highly desirable. Natural products (NPs) and their analogues have yielded diverse inducers of protein degradation4, raising the possibility that new degrader chemotypes could be derived from NPs. They retain the biological relevance of NPs but open new chemical space and therefore may have unexpected and novel targets5,6, so exploration of their bioactivity5,7 may identify novel small-molecule degrader chemotypes and E3 ligases. The haem-binding enzyme indoleamine-2,3-dioxygenase 1 (IDO1) converts tryptophan (Trp) to kynurenine (Kyn), and Trp shortage and Kyn elevation are linked to reduced anti-tumour immunity, though by different mechanisms8,9,10,11,12. Moreover, IDO1 expression and Kyn levels are related to Epstein Barr virus (EBV)-associated lymphoma13 and to neurodegeneration14,15. In general, clinical exploration of different IDO1 inhibitors for cancer treatment has16,17,18,19,20,21,22, despite encouraging preclinical data23,24, met with limited success25,26. Plausible reasons include that IDO1 may have non-enzymatic functions27,28,29,30,31,32,33,34. Nevertheless, IDO1 inhibitors are in active clinical trials35. The limitations of enzymatic inhibitors36 may be overcome by IDO1 degradation, as initially supported by IDO1-directed proteolysis-targeting chimeras (PROTACs)37,38,39, but monovalent IDO1 degraders have yet to be identified. We have identified a class of pseudo-natural products derived from (−)-myrtanol, termed iDegs, that both inhibit IDO1 and induce IDO1 degradation. iDegs induce structural changes in IDO1 that cause enhanced ubiquitination and augmented degradation by CRL2KLHDC3, a ligase we identified to also mediate the ubiquitination and native degradation of IDO1. Our work defines a unique mechanism of action, a previously not identified type of degrader, and reveals an E3-ligase previously not used for small-molecule-mediated protein degradation. A library of 157,332 small molecules were screened in a cell-based assay (Kyn assay; Extended Data Fig. So as to screen under conditions that would be sensitive to IDO1 levels, we considered that IDO1 expression can be induced by interferon gamma (IFN-γ)40. We thus performed the screen in IFN-γ-stimulated BxPC3 cells21. The (−)-myrtanol-derived pseudo-natural product, hereafter termed iDeg-1 (see Supplementary Scheme 1 for the synthesis), inhibited Kyn formation with a half-maximal inhibitory concentration (IC50) of 0.83 ± 0.31 µM in the screening assay (Fig. Further indication that iDeg-1 targets IDO1 came from a cell viability assay in two-dimensional (2D) and 3D cultures41,42: iDeg-1 reduced IDO1-dependent SKOV-3 cell death induced by IFN-γ (Extended Data Fig. iDeg-1 only slightly affected IDO1 enzymatic activity (Extended Data Fig. 1f), did not impair IDO1 transcription (Extended Data Fig. 1c,d) without inhibiting in vitro translation of IDO1 or global protein translation (Extended Data Fig. Data are presented as mean value ± s.d. b, Kyn assay in BxPC3 and SKOV-3 cells after treatment with iDeg-1 and 50 or 5 ng ml−1 IFN-γ, respectively, for 48 h before detection of Kyn levels utilizing p-DMAB. c,d, IDO1 protein levels in BxPC3 cells upon treatment with IFN-γ and iDeg-1 for 24 h. Representative immunoblots (c) and quantified band intensities from c (d). Data are presented as mean values ± s.d. e, Cells were treated for 6 h with 50 µM iDeg-1 before the TUBE pulldown. Representative immunoblots of n = 3 biological replicates are shown. f, Cells were treated with 450 nM carfilzomib (CFZ) 60 min before the addition of iDeg-1 or DMSO for 2 h, followed by TUBE pulldown. Representative immunoblots of n = 3 biological replicates for IDO1 are shown. g,h, HEK239T cells were electroporated with rhIDO1 protein. Cells were treated with CFZ for 30 min before the addition of 3.33 µM iDeg-1 and further incubation for 6 h. g, Representative immunoblot of n = 3 biological replicates. h, Quantified band intensities from g, representing samples treated with compound relative to DMSO (set to 100%). Data are presented as mean values ± s.d. i, Volcano plot of iDeg-1-induced changes in the global proteome. HEK239T cells were electroporated with rhIDO1 protein followed by treatment with 10 µM iDeg-1 or DMSO for 6 h and MS analysis. Statistical significance was assessed using both-sided t-test with error-corrected P values (S0 = 0.5 (non-linear cutoff), FDR = 0.01). j,k, IDO1 immunoprecipitation (IP) and identified peptide of IDO1 (j) or ubiquitin (k) with diGly modification. IFN-γ-stimulated BxPC3 cells were treated with 20 µM iDeg-1 or DMSO for 6 h before the IP. Data are presented as mean values ± s.d. To determine whether iDeg-1 induces degradation via the ubiquitin proteasome system (UPS), IFN-γ-stimulated BxPC3 cells were treated with iDeg-1 for 6 h (Fig. 1e), or for 2 and 4 h after pretreatment with the proteasome inhibitor carfilzomib (CFZ; Fig. Increased polyubiquitination was detected with a tandem ubiquitin binding entity (TUBE) pulldown from cell lysates43. Ubiquitination was induced shortly after compound addition, but protein reduction was only detectable after 24 h in IFN-γ-stimulated BxPC3 cells (Fig. Because the capacity to induce degradation may be masked by the ongoing IFN-γ-stimulated production of IDO1, we directly introduced enzymatically active recombinant human IDO1 protein (rhIDO1) into HEK293T cells by electroporation (HEKrhIDO1 cells). iDeg-1 dose-dependently reduced Kyn amounts with an IC50 value of 0.45 ± 0.1 µM (Extended Data Fig. 2b,c) and lowered IDO1 protein after 6 h by 46% at a concentration of 10 µM (Extended Data Fig. iDeg-1 also inhibited Kyn production in HEK293T cells, which transiently express IDO1, with an IC50 of 0.42 ± 0.2 µM (Extended Data Fig. Global proteome profiling of HEKrhIDO1 cells after 6 h of treatment with iDeg-1 showed that, besides IDO1, reduced levels were detected only for DOCK-8 and RHOBTB3. However, these effects could not be validated (Fig. Analysis of ubiquitinated proteins for a diglycine (diGly) attached to lysine residues (K-ε-diglycine) that were modified with ubiquitin44, after iDeg-1 treatment and IDO1 immunoprecipitation uncovered K389 of IDO1 as a site of ubiquitination (Fig. DiGly analysis further revealed K48 linkages in the ubiquitin chains (Fig. 3e), which are associated with protein degradation via the UPS45. Direct engagement of IDO1 in cells was proven by a cellular thermal shift assay (CETSA)46, which showed that iDeg-1 stabilizes IDO1 with a shift in the melting temperature (ΔTm) of 3.5 ± 0.4 °C (Fig. An isothermal CETSA experiment at 50 °C (Extended Data Fig. 4a,b) proved that stabilization was dose-dependent. a,b, CETSA in intact SKOV-3 cells treated with 50 µM iDeg-1 or DMSO for 1 h followed by heat treatment and immunoblotting. b, Quantification of band intensities from a. Data are presented as mean values ± s.d. Data are presented as mean values ± s.d. d,e, Influence of iDeg-1, iDeg-2 and iDeg-3 on IDO1 protein levels in BxPC3 cells. Cells were treated with IFN-γ and the compounds (3.33 µM) for 24 h before immunoblotting (d). Quantification of the band intensities is shown in e. Data are presented as mean values ± s.d. Dashed lines indicate where lanes of the same blot were spliced together. f, Influence on in vitro rhIDO1 activity. rhIDO1 was pre-incubated with the compounds at 37 °C for 90 min before the detection of Kyn levels using p-DMAB. Data are presented as mean values ± s.d. g, rhIDO1 thermal stability in the presence of 50 µM iDeg-1, iDeg-2 or iDeg-3 or DMSO and the apo-IDO1 inhibitor linrodostat (50 µM) using nanoDSF. rhIDO1 and the compounds were pre-incubated for 3 h at 37 °C before measurement. Representative results are shown (n = 3 independent experiments). Initial structure–activity exploration identified iDeg-2 (IC50 = 138 ± 23 nM) and iDeg-3 (IC50 = 46 ± 21 nM), which embody an iodine or an alkyne substituent, respectively, in the para position of the phenyl carbamate (Fig. In BxPC3 cells exposed continuously to IFN-γ, iDeg-2 and iDeg-3 reduced IDO1 protein levels by 55 ± 3% and 62 ± 11%, respectively, as compared to 42 ± 2% by iDeg-1 at 3.33 µM (Fig. iDeg-2 and iDeg-3 also partially inhibited the enzyme in vitro (Fig. 2f) and stabilized rhIDO1, as determined by nano differential scanning fluorimetry (nanoDSF; Fig. UV–vis spectroscopic analysis revealed that in the presence of iDeg-1, iDeg-2 or iDeg-3, the specific Soret absorbance peak of haem-bound IDO1 (holo-IDO1; Fig. 2h) is reduced, showing that iDegs displace haem and, with different potencies, bind to apo-IDO1. Accordingly, addition of hemin reduced the potency of iDeg-1, iDeg-2 and iDeg-3 in the Kyn assay (Extended Data Fig. 4e–g) and dose-dependently elevated Kyn levels in the presence of iDeg-2 (Extended Data Fig. As we detected both enzymatic inhibition and degradation of IDO1 by iDeg-3, the compound class was termed iDeg. The co-crystal structure of IDO1 in complex with iDeg-1 (PDB 9RIS) and iDeg-2 (PDB 9FOH) at 2.1 Å and 1.6 Å resolution, respectively (Supplementary Table 2 and Extended Data Fig. Interestingly, binding to this latter pocket has previously been observed mainly for holo-IDO1 inhibitors47 (Extended Data Fig. iDegs binding occurs through numerous hydrophobic interactions, a water-bridged hydrogen bond between the carbamate nitrogen and the hydroxyl group of S167 and a hydrogen bond between the sulfonyl oxygen of iDeg-2 and H346 (Fig. As many of the same IDO1 residues are responsible for haem coordination, this binding mode is mutually exclusive with haem and ensures that iDegs can only bind apo-IDO1. Comparisons to all IDO1 structures revealed that iDeg-1 and iDeg-2 induce a striking conformational rearrangement (Fig. In contrast, in the iDeg-2-bound structure, no electron density for the K-helix was detected (Fig. This absence is not attributed to crystal packing forces, as the K-helix is present in the crystal structure of IDO1 with the inhibitor apoxidole with identical space group, unit cell and similar crystallization conditions (PDB 8ABX; Extended Data Fig. An overlay of the structures of IDO1 bound to iDeg-2 or with the apo-IDO1 inhibitor linrodostat revealed that helices B, C, F, H and J are all reoriented and the J-helix is also remodelled in the iDeg complex (Fig. In all previously reported IDO1 structures, the K-helix is embraced through F, J and H helices and the E–F loop (Extended Data Fig. In contrast, in the iDeg-1- and iDeg-2-bound complexes, tethers to the K-helix are weakened due to cumulative movements of several amino acids within the J-helix. Residues R343, F270 and T395 also adopt an alternative conformation compared to all other published IDO1 structures (Fig. As previously reported IDO1 inhibitors do not substantially alter the overall structure of IDO1 compared to holo-IDO147 (Extended Data Fig. 5f), the observed conformational rearrangements represent a novel and unique binding mode of the iDegs. a, Cartoon diagram of the IDO1-iDeg-1 and IDO1-iDeg-2 structures. Dashed lines indicate regions lacking electron density, including the E–F loop, J–K loop and K-helix, with parentheses denoting their proposed flexibility. c, Zoom-in view of the amino acids involved in iDeg-2 binding. Hydrogen bonds are indicated by black dashed lines. d, Comparison of the IDO1 structures with linrodostat (PDB 6DPR-B), iDeg-1 and iDeg-2, illustrating the transition of the K-helix from a rigid to a dynamic conformation upon iDeg binding. Surface regions interacting with the K-helix are highlighted. e, Overlay of IDO1-iDeg-2 (violet) and IDO1–linrodostat (grey; PDB 6DPR-B) structures. iDeg binding triggers local and long-distance structural perturbations (red dashed arrows). f, iDeg-2 binding induces a conformational shift in H346, moving the J-helix towards the iDeg-2 binding pocket, including R343. g, Residues T395, Q281 and R343, which are involved in stabilizing the K-helix through hydrogen bonding, similarly adopt shifted positions in the iDeg-bound state. Mechanistically, because iDegs bind deep in the active site, it is unlikely that they directly contact an E3 ligase. Instead, iDeg-induced degradation possibly involves increased accessibility of the C-terminal K-helix bearing the ubiquitinated K389, as compared to the conformations observed for all published IDO1 inhibitors. To explore the contribution to IDO1-induced degradation of several amino acids that adopt substantially shifted positions in the iDeg-bound structures, we generated an IDO1 stability reporter in KBM7 cells harbouring an inducible Cas9 cassette49 (Fig. Supporting a role of the C-terminal region of IDO1 in degradation, only N-terminally fused BFP enabled iDeg-induced reporter degradation (Fig. Alanine substitutions were introduced at H346, R343 and F270, which interact with haem in the holo-IDO1 structure50 and adopt different conformations in the iDeg-bound state compared to all other published IDO1 structures. In addition, we tested the T395M mutation, as T395 located in the K-helix contributes to its stabilization through hydrogen bonding with R343 (J-helix) and Q281 (E–F loop). All four mutants displayed lower protein stability in cells, indicating their role in maintaining a stable IDO1 conformation (Fig. Degradation induced by iDeg-1, iDeg-2 or iDeg-3 in cells expressing these IDO1 mutants was rescued only by the H346A mutation, highlighting their interaction with H346 as being crucial for degradation (Fig. b, IDO1 levels detected by IDO1 stability reporters. KBM7 IDO1 reporter cells were treated with iDeg-1, iDeg-2 or iDeg-3 (1 µM) for 24 h before detection of IDO1 levels using flow cytometry. BFP) were calculated per genotype, respectively. Data are presented as mean values ± s.d. c, Representative histogram for the iDeg-1-mediated depletion of BFP-IDO1 (24 h, 10 µM). d, Influence of IDO1 mutations on BFP-IDO1 protein levels. Data are presented as mean values ± s.d. Data are presented as mean values ± s.d. f, Identification of genes required for native IDO1 degradation. g, Identification of genes required for iDeg-1-mediated IDO1 degradation. In f and g, genes are highlighted for P < 0.05 (one-sided MAGeCK) and log2(fold-change, FC) > 1.585 (n = 2 biological replicates). h, IDO1 depletion is rescued by 10 h of co-treatment with either CFZ, TAK243 or MLN4924 (1 µM each). Data are presented as mean values ± s.d. i, Proximity labelling and enrichment of biotinylated KLHDC3 in HEK293T cells expressing IDO1–TurboID biotin ligase. The treatment time with iDeg-3 or DMSO was 2.5 h. A representative immunoblot is shown for n = 2 biological replicates, and 2% of the total protein input used for the pulldown was loaded in the input lane. k, Fluorescent ubiquitin transfer from neddylated CRL2KLHDC3-activated UBE2R2 to the indicated C-terminal IDO1 peptides over time. l, IDO1 stability reporter variants in KBM7 cells measured by flow cytometry and depicted normalized to the WT (that is, –EG) reporter. Data are presented as mean values ± s.d. To identify the E3 responsible for iDeg-mediated IDO1 degradation, we conducted a fluorescence-activated cell sorting (FACS)-based CRISPR–Cas9 screen using an sgRNA library targeting 1,301 ubiquitin-associated genes (six sgRNAs per gene)51. Cas9 expression was induced for 72 h before 14 h of compound treatment, then the cells were enriched for increased or decreased BFP levels using FACS and the corresponding sgRNAs quantified by deep sequencing (Fig. 1), revealing the genes functionally required for iDeg-induced degradation. As expected, knockout of proteasome subunits or of genes involved in neddylation counteracted iDeg activity (Fig. 6b,c), thus phenocopying the IDO1 stability reporter behaviour upon chemical perturbation of the UPS (Fig. Importantly, we further identified the cullin-RING ligase (CRL) complex, including cullin2 (CUL2), RBX1, elongin B/C (EloB and EloC) and the Kelch domain containing protein 3 (KLHDC3), as required for IDO1 degradation. Unexpectedly, genetic disruption of the CRL2KLHDC3 complex also affected baseline IDO1 turnover under vehicle (dimethyl sulfoxide, DMSO) treatment. To validate these findings, we employed the TurboID approach and expressed IDO1 fused to the TurboID biotin ligase in HEK293T cells followed by enrichment of biotinylated proteins. Indeed, KLHDC3 was slightly biotinylated in the control condition, which increased upon treatment of cells with iDeg-3 (Fig. 4i), thus confirming that IDO1 and KLHDC3 interact in cells, and that this interaction is enhanced in the presence of the degrader. Contrary to classical degrader modalities such as PROTACs or molecular glue degraders, which typically function by inducing proximity between an E3 and a target that is functionally inconsequential in the absence of the small molecule, iDegs thus appear to promote a native route for IDO1 turnover. The CRL substrate receptor KLHDC3 has not yet been employed for small-molecule-induced protein degradation. KLHDC3 recognizes C-degrons52,53, which explains why the C-terminally fused IDO1-BFP reporter was not degraded, but the BPF-IDO1 reporter was (Fig. The C-terminal EG sequence of human IDO1 is consistent with the distinguishing feature of a KLHDC3 C-degron, which is a C-terminal glycine52,53,54. In agreement, a peptide IDO1C-deg corresponding to the C-terminal region (residues 381–403) bound KLHDC3 with a KD of 103 nM (Fig. Substituting the C-terminal glycine with lysine (peptide IDO1C-deg-G403K) abolished binding to KLHDC3. Replacing the ubiquitination site (peptide IDO1C-deg-K389R) did not impact E3-degron complex formation (Fig. To test whether IDO1 is a direct substrate of CRL2KLHDC3, we biochemically reconstituted ubiquitination. Because KLHDC3-EloB/C can form an autoinhibited self-assembly55, we used a monomeric version of KLHDC3 (C-terminal Gly-to-Lys mutant). Assays were performed in the ‘pulse-chase' format. The pulse reaction generates a thioester-linked E2 conjugate with fluorescently labelled ubiquitin. Next, E3 and substrate are added, and fluorescent ubiquitin transfer from E2 (UBE2R2) to the substrate and subsequently to a substrate-linked ubiquitin is observed over time. The IDO1C-deg peptide was ubiquitinated in vitro in a CRL2KLHDC3-dependent manner, whereas the mutant versions were not (Fig. In cells, mutation of the IDO1 C-terminal glycine increased abundance, whereas mutation of the non-optimal –EG degron to an optimal –RG C terminus reduced IDO1 abundance (Fig. These findings demonstrate that IDO1 is a substrate of KLHDC3 and that the C-terminal glycine-based degron is essential for interaction with the E3 ligase. To benchmark the degradation potency of iDeg-1, iDeg-2 and iDeg-3 and subsequently the identified inhibitor and degrader iDeg-6 (IC50 of 16 ± 5 nM in the Kyn assay; Fig. 7a), IDO1 levels were detected after stimulation of cells for 24 h with IFN-γ followed by a washout (to avoid IDO1 expression) and compound addition. iDeg-6 most potently depleted IDO1 in the cells, with a maximal achievable degradation (Dmax) of 70% at 100 nM and a half-maximal degradation concentration (DC50) for IDO1 degradation of 6.5 ± 3 nM (Fig. In vitro iDeg-6 inhibited IDO1 activity nearly completely and more potently than iDeg-1 to 3 with an IC50 of 1.6 ± 0.3 µM (Extended Data Fig. Direct binding of iDeg-6 to IDO1 was detected using isothermal titration calorimetry (ITC; Supplementary Fig. The compound induced thermal stabilization of the protein and haem displacement to a higher extent as compared to iDeg-1, iDeg-2 and iDeg-3 (Extended Data Fig. We therefore used iDeg-6 for further validation. Data are presented as mean value ± s.d. b,c, Reduction of IDO1 protein levels by iDeg-6. Degradation efficiency was assessed in a modified set-up including IFN-γ washout before compound addition. BxPC3 cells were treated with 50 ng ml−1 IFN-γ for 24 h before washout, addition of iDeg-6 for 24 h, then immunoblotting (b) and quantification of the IDO1 protein levels (c) from b. Data are presented as mean values ± s.d. d–f, IDO1 levels in SKOV-3 (d), SKOV-3 cells stimulated with IFN-γ for 24 h followed by a washout (e) or BT549 (f) cells. Cells were treated with iDeg-6 for 24 h before immunoblotting. g,h, IDO1 protein levels in wild-type (WT) U2OS or KLHDC3 knockout (KO) U2OS cells. Cells were stimulated with 5 ng ml−1 IFN-γ for 24 h before washout, treatment with iDeg-6 or DMSO for 24 h, and immunoblotting (g). Representative data of n = 4 biological replicates (U2OS WT) or n = 2 biological replicates (KO cells) are shown. h, Quantification of the band intensities from g and Extended Data Fig. Data are presented as mean values ± s.d. i,j, IDO1 protein levels in KBM7-BFP-IDO1 (CTRL) or KBM7-BFP-IDO1 KLHDC3 KO1 or KO2 cells in the absence (h, normalized to CTRL) or presence of iDeg-6 (i, normalized to the respective genotype). The treatment time was 24 h. Data are presented as mean values ± s.d. k, Schematic representation of the IDO1 competition ubiquitination assay. l, Quantification of KLHDC3-dependent IDO1 C-terminal peptide ubiquitination (IDO1C-deg) upon titrating increasing concentrations of competing full-length apo-IDO1, iDeg-6-IDO1 or linrodostat-bound IDO1 to measure IC50 values. m, Apo-IDO1 ubiquitination after two sequential 20-min incubations (first and second) with 42 µM of each indicated compound, followed by the addition of NEDD8-CUL2KLHDC3 to initiate ubiquitination. Data are presented as mean values; n = 2 independent experiments. IDO1 degradation accounted for 81% or 59% of the achieved Kyn level decrease at 100 nM or 1 µM, respectively. However, the cellular response to proteasomal inhibition is rather complex and can induce compensatory degradation mechanisms that can impact total IDO1 levels. iDeg-6 also depleted IDO1 protein in SKOV-3 and BT549 cells in the absence of or upon stimulation with IFN-γ in a dose- and time-dependent manner (Fig. The neddylation inhibitor MLN4924 rescued iDeg-6-induced IDO1 depletion (Extended Data Fig. 8f) and increased IDO1 in the absence of iDegs, demonstrating that neddylation is required for both native and iDeg-induced IDO1 degradation. KLHDC3 knockout in U2OS or KBM7-BFP-IDO1 cells increased IDO1 amounts and rescued iDeg-6-dependent degradation (Fig. IDO1 ubiquitination in the presence of iDeg-6 was abolished for KLHDC3 variants carrying mutations in the degron recognition motif (R240A, S241E, R292A)54 (Extended Data Fig. Surprisingly, in vitro, apo-IDO1 was rapidly ubiquitinated by CRL2KLHDC3. IDO1 was also ubiquitinated upon treatment of apo-IDO1 with iDeg-6 (Extended Data Fig. In contrast, both haem and the apo-IDO1 inhibitor linrodostat suppressed IDO1 ubiquitination under these conditions (Extended Data Fig. Hence, biochemically, apo-IDO1 is a better KLHDC3 substrate than holo-IDO1, consistent with structural data showing haem and lindrodostat determining the arrangement of the K-helix, as well as quantitative insights from competitive ubiquitination assays. Briefly, ubiquitination of IDO1 C-terminal peptide (IDO1C-deg) was examined while titrating apo-IDO1 or compound-bound IDO1. If a full-length IDO1 binds KLHDC3, then substrate preference shifts from peptide to protein along the titration, allowing calculation of IC50 values for inhibiting C-terminal peptide ubiquitination. Apo-IDO1 and iDeg-6-bound IDO1 show IC50 values of 119 and 118 nM, respectively, matching the affinity towards the isolated C-terminal peptide measured by ITC (Figs. In contrast, the IDO1 bound to linrodostat showed a >20-fold greater IC50 value (2.6 µM). Thus, apo-IDO1 and iDeg-6-bound IDO1 are high-affinity KLHDC3 substrates, whereas linrodostat binding renders IDO1 less suitable for ubiquitination. Apo-IDO1 was incubated first with an iDeg, or first with hemin or linrodostat (which secure the IDO1 C-terminal helix). Although apo-IDO1 and iDeg-bound IDO1 were readily ubiquitinated in the absence of haem, when the iDeg treatment was first, ubiquitination proceeded after haem addition. Due to its superior occupation of the haem-binding pocket, linrodostat prevented ubiquitination in all assays if added first. Although iDeg-1 showed little effect in these assays, iDeg-2 through iDeg-6 increasingly competed with haem and allowed ubiquitination, trending with their IC50 towards IDO1 activity in vitro and efficacy at eliciting IDO1 degradation in cells. These findings suggest a possible model for the regulation of IDO1. In the presence of haem, apo-IDO1 is charged with haem to form holo-IDO1, and the holo-pool may thus escape degradation. Support for such a mechanism in cells (Fig. In IFN-γ-stimulated BxPC3 cells, succinyl acetone reduced IDO1 protein levels, and the addition of hemin dose-dependently increased IDO1 (Fig. a, Regulation of IDO1 by haem synthesis and succinyl acetone (SA) as an inhibitor of haem synthesis. b,c, Influence of SA and haem on IDO1 protein levels in BxPC3 cells, detected using immunoblotting. Cells were treated with 50 ng ml−1 of IFN-γ with or without 10 µM SA for 24 h, followed by the addition of hemin for a further 24 h. c, Quantification of the band intensities from b. Data are presented as mean values ± s.d. d, Influence of SA and haem on IDO1 levels in SKOV-3 cells. Cells were treated with SA for 48 h before the addition of hemin for another 24 h and immunoblotting. Representative data of n = 3 biological replicates are shown. e,f, Influence of SA and hemin on IDO1 abundance in KBM7-BFP-IDO1 (e) or KBM7-BFP-IDO1 KLHDC3 KO cells (f). Cells were pre-treated with 10 µM SA for 24 h followed by a treatment for 48 h with 100 µM hemin or DMSO as a control. IDO1 levels were quantified using flow cytometry. Data are presented as mean values ± s.d. Cells were incubated with the compounds (5 µM) for 48 h before immunoblotting. Representative data of n = 3 biological replicates are shown. h, Influence of selected IDO1 inhibitors (IDO1i, 1 µM) on IDO1 protein levels in KBM7-BFP-IDO1 or KBM7-BFP-IDO1 KLHDC3 KO cells. Cells were treated with the compounds for 24 h, followed by quantification of IDO1 using flow cytometry. Data are presented as mean values ± s.d. i, Influence of iDeg-6 on wound closure using SKOV-3 cells in the presence of epacadostat (1 µM), iDeg-6 (1 µM) or DMSO as control. Before performing the wound healing assay, cells were pre-conditioned for 24 h with the same compound concentration. Data are displayed as the percentage of wound closure with respect to time 0. Data are presented as mean values ± s.d. The data were analysed by two-way analysis of variance (ANOVA) followed by a post hoc Bonferroni's test. Changes in the IDO1 protein level did not result from altered IDO1 mRNA levels (Extended Data Fig. Knockout of KLHDC3 counteracted succinyl acetone-mediated IDO1 depletion, indicating that apo-IDO1 degradation is dependent on KLHDC3 (Fig. We then asked whether other IDO1 inhibitors might control IDO1 abundance. All investigated inhibitors increased IDO1 protein levels in the cells, which was partially reverted in KLHDC3 knockout cells (Fig. These findings are in agreement with the biochemical data showing lower ubiquitination of IDO1 in the presence of linrodostat (Fig. 5m), and our structural data showing iDegs bind a distinct conformation of IDO1. Hence, the IDO1-KLHDC3 degradation circuit can be modified in both directions by liganding common residues, albeit with distinct overall structural consequences. We analysed the influence of these IDO1 inhibitors on the melting temperature of IDO1 and on haem displacement to assess whether these assays could be used for predicting IDO1 degrader activity. Apo-IDO1 inhibitors shifted the melting curve to higher temperatures, whereas most holo-IDO1 inhibitors shifted the peak corresponding to the haem-bound state (Extended Data Fig. These biophysical assays, however, cannot be used to predict IDO1 degraders as iDegs change the Tm and the Soret band intensity in a similar manner as other apo-IDO1 inhibitors. Finally, we assessed the ability of iDeg-6 to suppress the non-enzymatic activity of IDO1, which has recently been shown to promote tumour proliferation and migration33,34. Specifically, in SKOV-3 cells, IDO1 enzymatic inhibition by epacadostat stabilizes a non-enzymatic IDO1 protein, which results in faster cell migration. In contrast, IDO1 knockdown decreases the migratory capacity of SKOV-3 cells34. We thus compared cell migration, using a wound healing assay, with SKOV-3 cells exposed to iDeg-6 and epacadostat. iDeg-6 significantly slowed down wound closure compared to the control and cells that were treated with epacadostat, demonstrating that iDegs can also affect the non-enzymatic protumorigenic function of IDO1 (Fig. Clinical investigation of IDO1 inhibition has had limited success so far47,56, possibly due to its non-enzymatic signalling function29,32. This limitation could be addressed by the targeted degradation of IDO1. We identified a pseudo-natural product class termed iDegs that potently induce IDO1 ubiquitination and degradation through the native degradation pathway mediated by the E3 ligase CRL2KLHDC3. iDegs bind to apo-IDO1, which is either present in cells or generated by intrinsic haem loss, leading to loss of enzymatic activity. More importantly, iDegs induce a conformational shift of the C-terminal helix and lock IDO1 in a state favouring ubiquitination by KLHDC3. Our data demonstrate that apo-IDO1, but not haem-bound IDO1, is preferentially ubiquitinated and degraded, suggesting also that apo-IDO1 can readily adopt a ubiquitination-sensitive conformation. In contrast, haem binding prevents IDO1 turnover. Therefore, targeting the same binding pocket with the haem cofactor or small-molecule ligands has a completely opposite impact on the IDO1-KLHDC3 degradation circuit and IDO1 fate. So far, degraders employing native degradation mechanisms have largely been overlooked, and only a few examples (targeting BCL6 or EZH257,58) support the notion that monovalent degraders in a more general sense may regulate the native mechanisms of protein homeostasis for the respective targets2. Scholes and colleagues have identified supercharging of endogenous degradation pathways as a frequently observed mechanism of inhibitor-induced kinase degradation59. Functionally differentiated, iDegs represent a group of small molecules that act as a switch to induce an IDO1 conformational state that shifts the equilibrium to the degradation-sensitive state, thereby channelling the IDO1-iDeg complex to the native degradation mechanism. This route to protein degradation complements current degradation strategies that are based on PROTACs or MGDs, as observed for oestrogen and androgen receptors, and may also apply to other proteins60,61. Compared to IDO1-directed PROTACs37,38,39, iDegs are smaller and more drug-like and hijack the native ligase, making them broadly applicable to all cells and tissues expressing IDO1. Due to their dual mode of action, iDegs impair all IDO1 functions, and, unlike other apo-IDO1 inhibitors, they induce a decrease in protein levels. Because the non-enzymatic functions of the protein and an inhibitor-mediated increase in IDO1 concentration32,38 may be linked to failure of IDO1 inhibitors in the clinic, small-molecule-induced inhibition and degradation may open up alternative therapeutic opportunities13,14,15. U2OS KO cells were obtained from the St. Jude Center for Advanced Genome Engineering (CAGE). KBM7 iCas9 cells were a gift from J. Zuber (IMP, Vienna). Lenti-X 293T lentiviral packaging cells were obtained from Clontech. pCMV3-IDO1 was obtained from Sino Biological US. pSpCas9(BB)-2A-GFP (PX458) plasmid was a gift from Feng Zhang (Addgene 48138, ref. pCMVR8.74 was a gift from D. Trono (Addgene 22036). pMD2.G was a gift from D. Trono (Addgene 12259). Fetal bovine serum (FBS), penicillin/streptomycin, anti-IDO1 (14-9750-80) and anti-vinculin (V9131) antibodies were obtained from Thermo Fisher. Anti-rabbit horseradish peroxidase (HRP; 7074) was obtained from Cell Signaling Technology. Anti-IDO1 (ab211017) and anti-β-actin (ab8227) antibodies were obtained from Abcam. Mouse Thy1.1 antibody was purchased from BioLegend. Anti-α-tubulin antibody (ab18251) was obtained from Abcam. HRP-conjugated secondary goat anti-rabbit antibody (31460) was obtained from Pierce Biotechnology. IRDye-conjugated secondary antibodies were obtained from LI-COR Biosciences. Dulbecco's modified Eagle medium (DMEM), McCoy's medium, sodium pyruvate and non-essential amino acids were obtained from PAN Biotech. Where indicated, DMEM was obtained from Gibco. Iscove's modified Dulbecco's medium (IMDM) was obtained from Gibco. Succinyl acetone was obtained from TCI. Doxycycline was obtained from PanReac AppliChem. IDO-IN-1 (19402) and IDO-IN-5 (33178) were obtained from Cayman Chemical. IDO-IN-4 (HY-18769), IDO-IN-7 (HY-13983), PF-06840003 (HY-101111), linrodostat (BMS 986205, HY-101560) and epacadostat (HY- 15689) were obtained from MedChem Express. Navoximod (BD630231), BMS-986242 (BD01401323) and 615-21-4 (BD4825) were obtained from BLD. IDO-IN-13 (TM-T11616) was obtained from CymitQuimica, and 5593-1162 (CAS-no. BxPC3 cells were cultured in RPMI 1640 medium (PAN Biotech) containing 10% heat-inactivated FBS. BT549, HeLa and HEK293T cells were cultured in DMEM (PAN Biotech) supplemented with 10% FBS, sodium pyruvate and non-essential amino acids. SKOV-3 and U2OS WT and KO cells were cultured in McCoy's 5A medium (PAN Biotech) containing 10% heat-inactivated FBS. Lenti-X 293T lentiviral packaging cells were cultured in DMEM (Gibco) supplemented with 100 U ml−1 penicillin/streptomycin and 10% heat-inactivated FBS. IDO1-deficient SKOV-3 cells were produced using the CRISPR–Cas9 system. For this, 100,000 cell well−1 in six-well plates were transfected using Lipofectamine 3000 with PX458 plasmid containing three separated sgRNA located in IDO1. After 48 h, the cells were sorted for single cells based on green fluorescent protein (GFP) positivity. Single clones were expanded and successful KO of IDO1 was verified using immunoblotting. KBM7 iCas9 cells were cultured in IMDM supplemented with 100 U ml−1 penicillin/streptomycin, 10% heat-inactivated FBS and 1 mM L-tryptophan. Cell lines were maintained in a humidified atmosphere at 37 °C and 5% CO2. Regular mycoplasma testing confirmed cell lines as mycoplasma-negative. The Kyn screening assay was performed as previously described21. BxPC3 cells were seeded in phenol red-free RPMI 1640 medium in 1,536-well or 384-well plates (Greiner 782086, Corning 3770) 24 h before the addition of the compounds, IFN-γ (50 ng ml−1, PeproTech) and L-Trp (380 µM, Sigma-Aldrich). After 48 h of incubation, initial cell viability was accessed by the addition of Hoechst 33342. Afterwards, trichloroacetic acid (TCA, Sigma-Aldrich) was added to a final concentration of 7% and incubated for 10 min before a centrifugation step of 10 min at 1,620g. Kyn levels were determined by means of the Kyn sensor21 at a final concentration of 17.5 µM in sensor buffer (50 mM H3PO4 and 120 mM NaCl pH 1). Fluorescence intensity (excitation, 535 nm; emission, 595 nm) was measured using a Spectramax Paradigm reader (Molecular Devices). Data were normalized to the values for cells treated with DMSO. For the manual assay, BxPC3 or SKOV-3 cells were seeded in clear 96-well or 384-well plates, respectively, 24 h before compound treatment and IFN-γ (50 or 5 ng ml−1, respectively) and L-Trp (380 µM) addition, as described in ref. At 48 h after treatment, TCA was added and Kyn levels were detected using 2% wt/vol of para-dimethylaminobenzaldehyde (p-DMAB; Ehrlich reagent) in acetic acid. The absorbance of the Ehrlich reagent was detected at 492 nm and at 650 nm as background control on a Spark multimode microplate plate reader (Tecan). Kyn levels were determined by subtracting the absorbance value at 650 nm and presented relative to the DMSO control. Dose–response curves and IC50 values were generated and fitted with GraphPad Prism 9.2 using a four-parameter variable-slope nonlinear regression curve fit. For label-free Kyn detection, the cell supernatant was analysed by LC–MS/MS following the addition of 30% TCA. Kyn levels were measured by HPLC–MS/MS using an LTQ Velos Pro and Dionex HPLC system (Thermo Fisher Scientific). Data were analysed using Thermo Xcalibur (Thermo Fisher Scientific) and presented with GraphPad Prism 9.2. Values are shown relative to the DMSO control. After seeding, cells were treated with 10 ng ml−1 (2D) or 1 ng ml−1 (3D) of IFN-γ and CellTox Green (Promega, G8731) and added to the culture medium (1:6,000). iDeg-1 was freshly added every three days. The cells were monitored by live phase-contrast microscopy using an IncuCyte S3 system (Sartorius) with a 10x objective, capturing images every day up to four days (2D) and six days (3D). Cell death was quantified by counting the numbers of green fluorescence-positive cells (CellTox Green positive) normalized to the confluence of each respective image (2D) or by measuring the green signal mean intensity of each spheroid (3D). Protein concentrations were determined by a DC protein assay (Bio-Rad), followed by the addition of 1× sodium dodecyl sulfate (SDS) loading buffer and protein separation by 10% SDS–polyacrylamide gel electrophoresis (PAGE). Separated protein samples were transferred to a polyvinylidene difluoride (PVDF) membrane. IDO1 in IFN-γ-stimulated cells as well as vinculin, β-actin or α-tubulin as loading controls were detected by blocking the membranes with Intercept blocking buffer (LI-COR Biosciences, 927-70001) and using the primary antibodies anti-IDO1 (1:2,500, 14-9750-82, Thermo Fisher), anti-vinculin (1:10,000, V9131, Thermo Fisher), anti-β-actin (1:2,500, ab8227, Abcam) or anti-α-tubulin (1:2,500, ab18251, Abcam), and IRDye-conjugated secondary antibodies (IRDye 800CW goat anti-mouse immunoglobulin-G (IgG) secondary antibody (1:5,000, 926-32210), IRDye 680CW goat anti-rabbit IgG secondary antibody (1:5,000, 926-68071) and IRDye 680RD donkey anti-mouse (1:5,000, 926-68072)). For the detection of IDO1 protein levels in BT549 and SKOV-3 cells, the membranes were blocked with 5% milk in PBS + 0.1% Tween-20 (PBS-T), followed by incubation with anti-IDO1 (1:5,000, ab211017, Abcam) and HRP-conjugated secondary antibody (1:5,000, 31460, Pierce Biotechnology). For the detection of DOCK-8 and RHOBTB3 protein levels in HEK293T cells, the membranes were blocked with 5% milk in PBS-T, followed by incubation with anti-DOCK-8 (1:500, 11622-1-AP, Proteintech) or RHOBTB3 (1:800, 13945-1-AP) and HRP-conjugated secondary antibody (1:5,000, 31460, Pierce Biotechnology). KLHDC3 protein levels in U2OS, U2OS KO or HEK293T cells were detected by blocking the membranes with 5% milk in PBS-T and using anti-KLHDC3 antibody (1:1,000, HPA030131, Atlas) and anti-rabbit HRP-conjugated secondary antibody (1:3,000, 7074, Cell Signaling Technology). Primary antibodies were incubated at 4 °C overnight. Washing steps were performed with PBS-T. Chemiluminescence was detected using SuperSignal West Dura Extended Duration Substrate (Thermo Fisher Scientific). Protein bands were visualized using a ChemiDoc MP system (Bio-Rad). Band intensities were quantified using Image Lab software 6.0 (Bio-Rad). BxPC3 cells were incubated for 24 h before treatment with the indicated compound concentrations and 50 ng ml−1 IFN-γ. SKOV-3 cells were directly treated with or without 10 µM succinyl acetone for 24 h before the addition of hemin for a further 24 h. Total RNA was isolated using an RNeasy Mini Kit as described by the manufacturer (Qiagen). The obtained RNA was reverse-transcribed into complementary DNA (cDNA) using a QuantiTect reverse transcription kit according to the manufacturer's instructions (Qiagen) before the quantitative polymerase chain reaction (qPCR) using a QuantiFast SYBR green PCR kit (Bio-Rad). qPCR was performed using a CFX96 real-time PCR detection system (Bio-Rad). Data were analysed using CFX Manager (Bio-Rad). IDO1 expression levels in each sample were normalized to the levels of the reference gene GAPDH. Relative quantification was performed using the 2−ΔΔCt method63. The Renilla (Rluc) luciferase expression plasmid pRL-TK vector (Promega Corporation) was used as a control. HEK293T cells were transiently reverse-transfected with the plasmids using Lipofectamine 2000 (Invitrogen) following a modified protocol of the manufacturer's guidelines. The Fluc plasmid (4 μg) and 300 ng of the Rluc plasmid were mixed in serum-free Opti-MEM in a 1:3 ratio with Lipofectamine 2000. The mixture was added to 9 ml of cell suspension at a cell density of 2.78 × 105 cells ml−1, then 25,000 cells were seeded per well in a clear 96-well plate and incubated for 24 h. The next day, IDO1 promoter-mediated Fluc expression was induced by the addition of 50 ng ml−1 IFN-γ with simultaneous compound treatment. The cells were incubated for 48 h before determining the activity of Fluc and Rluc using the Dual-Glo Luciferase assay system with a Spark multimode microplate reader (Tecan). After 24 h, 5 µM cycloheximide (CHX) was added 30 min before the addition of 5 µM iDeg-1, followed by cell lysis preparation and immunoblotting. Tandem ubiquitin binding entity (TUBE) protein was expressed from pGex-6P-1-GST-4xUBA-His (UBA, ubiquitin-associated domains) in Escherichia coli and purified by affinity purification using a Ni-NTA column. TUBE protein expression was induced by isopropyl β-D-1-thiogalactopyranoside (IPTG) for 4 h at 30 °C. The TUBE protein was eluted using 20 mM sodium phosphate (pH 7.4) buffer containing 500 mM imidazole and 1 mM 2-mercaptoethanol and subsequently applied to a size exclusion column (HiLoad 16/600 Superdex 200 pg) in 20 mM potassium phosphate buffer (pH 7.4, supplemented with 300 mM NaCl, 1 mM dithioerythritol (DTE) and 2% glycerol) to yield pure TUBE protein. For the TUBE pulldown, BxPC3 cells were seeded in either 10- or 15-cm2 cell culture dishes and after cell attachment were treated with 40 ng ml−1 IFN-γ to induce IDO1 expression overnight. For proteasome inhibition, 450 nM CFZ (ab216469, abcam) was added 1 h before the addition of iDeg-1. The samples were kept on ice for 30 min, followed by a centrifugation step for 30 min at 16,000g and 4 °C. Protein concentrations were determined to adjust the protein concentration before the addition of 40 µl of GST magnetic beads (Thermo Fisher). The samples were rotated for 2 h at 4 °C, followed by four washing steps using ice-cold PBS-T for 5 min. To elute ubiquitinated protein species, 20 µl of 1× SDS sample buffer was added to the beads and incubated for 15 min at 55 °C. Afterwards, all samples were boiled for 5 min at 98 °C before separation by SDS–PAGE and immunoblotting for IDO1 and vinculin as a control. HEK239T cells (3,000,000 cells per electroporation) were electroporated with 80 µg of rhIDO1 utilizing a NeonTM transfection kit according to the manufacturer's instructions (Thermo Fisher). For electroporation, 1,000 V (two pulses for 35 ms) were applied using a Neon transfection system pipette station. Afterwards, the cells were washed in pre-warmed PBS followed by resuspension in 3 ml of trypsin/EDTA solution, and were then incubated for 3 min at 37 °C. The electroporated cells were washed again and resuspended in phenol red-free DMEM growth medium (900,000 cells ml−1). For the Kyn assay in HEKrhIDO1 cells, the cells were plated in 96-well plates and supplemented with 150 µM L-Trp and the respective compound concentrations for 24 h before readout as described above for the manual Kyn assay. For immunoblotting, electroporated cells were seeded in a 12-well plate and treated with the indicated compounds for 6, 14 or 24 h before immunoblotting as described above. HEK293T cells (270,000 cells ml−1) were transfected with 0.5 µg of pCMV3-IDO1 (Sino Biological US) using Lipofectamine 2000 (Thermo Fisher) while seeding in a 96-well plate, followed by incubation overnight. L-Trp (500 µM) and compounds at the indicated concentrations were then added, and the cells were incubated for 24 h before the detection of Kyn levels using p-DMAB as described above for the manual Kyn assay. The HEK239T cells were electroporated with rhIDO1 as described above and treated with 10 µM iDeg-1 or DMSO for 6 h before lysate preparation by means of four freeze–thaw cycles in PBS containing 0.4% NP-40 Alternative. The protein concentration was determined, and 200 μg of protein lysates were subjected to sample preparation and MS analysis. Protein lysates were dissolved in an equal volume of 100 mM triethylammonium bicarbonate (TEAB) buffer supplemented with 7.5 µl of 200 mM tris(2-carboxyethyl)phosphine (TCEP) and incubated at 55 °C for 1 h. Afterwards, 7.5 µl of 375 mM iodoacetamide were added and the samples incubated for another 30 min at room temperature in the dark before acetone-based protein precipitation (incubation at −20 °C overnight). The next day, the samples were centrifuged at 8,000g and 4 °C, the supernatant was aspired, and protein pellets were dried at room temperature. The pellets were dissolved in 50 mM TEAB containing trypsin/LysC (25:1 protein:protease ratio (wt/wt), pre-dissolved in 1 mM HCl (trypsin/LysC mix, MS grade, 5× from Promega Corporation)) and incubated at 37 °C overnight while shaking. The next day, the samples were dried in a vacuum concentrator at 30 °C and subjected to MS. All solvents used during MS measurements were LC–MS grade. Digested samples were first dissolved in 120 μl of 20 mM ammonium formate (NH4COOH) at pH 11 in an ultrasonicator for 15 min, followed a 15-min incubation step while rotating, and a subsequent centrifugation step (8,000g for 3 min at room temperature). Each sample (50 μl) was separated into ten fractions using an XBridge C18 column (130 Å, 3.5 μm, 1 mm × 150 mm) and a U3000 capHPLC system (Thermo Fisher Scientific) under high pH conditions to reduce the sample complexity and increase the number of quantified proteins. Separation was performed at a flow rate of 50 μl min−1 starting with 95% solvent A (20 mM NH4COO pH 11 in H2O)/5% solvent B (40% 20 mM NH4COO pH 11, 60% acetonitrile in H2O), isocratic for the first 10 min, followed by a linear gradient increasing solvent B up to 25% over 5 min. Afterwards, a second linear gradient was applied, increasing solvent B up to 65% in 60 min, followed by a third gradient increasing solvent B up to 100% over 10 min. Fractions were detected at a wavelength of 214 nm. The ten fractions were collected at 15 and 100 min (30 s per fraction, circular fractionation using ten vials) and dried in a SpeedVac unit at 30 °C until complete dryness for final analysis by nanoHPLC–MS/MS. The dried samples were then dissolved in 10 μl of 0.1% trifluoroacetic acid (TFA) in water before injection of 9 μl onto an UltiMateTM 3000 RSLCnano system (Thermo Fisher Scientific), coupled online to a Q Exactive HF Hybrid Quadrupole-Orbitrap mass spectrometer equipped with a nanospray source (Nanospray Flex Ion Source, Thermo Fisher Scientific). To detect the peptides within the samples, a mass range of m/z 300–1,650 was achieved with a resolution of 60,000 for a full scan, before 15 high energy collision dissociation (HCD) MS/MS scans of the most intense at least doubly charged ions using a resolution of 15,000 and a nominal collision energy (NCE) of 27%. A full enzymatic trypsin cleavage search was conducted allowing for two miscleavages. Carbamidomethylation was chosen as a fixed modification, and oxidation of methionine and acetylation of the N terminus as variable modifications. A false discovery rate (FDR) of 1% was selected for peptide and protein identification, and only proteins for which at least two peptides were quantified were chosen for further validation. Relative protein quantification was achieved by means of the label-free quantification algorithm implemented in MaxQuant. Statistical data analysis was carried out using Perseus65 v.1.6.14.0. The normalized LFQ intensities were log-transformed (log2) and grouped. Proteins were only retained for further analysis when quantified at least twice in at least one of the groups. Missing values were ascribed as ‘0' and a both-sided t-test with error-corrected P values (FDR = 0.01, S0 = 0.5) was conducted. Results were exported into an Excel file (xlsx) and visualized using Perseus. The detailed analysis procedure is described in refs. The next day, fresh medium was added containing 20 µM of iDeg-1 or DMSO as a control, and incubated for 8 h before cell detachment and cell lysis using immunoprecipitation lysis buffer (20 mM Tris-HCl pH 7.5, 100 mM NaCl, 0.1 mM EDTA and 0.5% NP-40 Alternative) supplemented with fresh N-ethylmaleimide as well as phosphatase (PhosSTOP phosphatase inhibitors, Sigma-Aldrich) and protease inhibitors (cOmplete protease inhibitor cocktail, Sigma-Aldrich). Three freeze–thaw cycles were performed for cell lysis, followed by centrifugation at 14,000g for 20 min at 4 °C and determination of the protein concentration (DC protein assay, Bio-Rad). The lysate (800 µg) was subjected to immunoprecipitation. For the following steps, a magnetic rack was used to separate magnetic beads from the supernatant. First, the cell lysates were pre-cleared using 25 µl of Pierce Protein A/G magnetic beads per sample for 1 h at 4 °C while rotating. The lysates were then added to pre-coated Pierce Protein A/G magnetic beads with IDO1 antibody (ab211017, Abcam), followed by continuous rotation for 2 h at 4 °C. Subsequently, the beads were transferred to a fresh sample tube after being washed three times with 500 µl of PBS, before further washing steps with 500 µl of PBS (thrice) to remove detergents. The beads were resuspended in 50 µl of denaturation buffer (50 mM Tris pH 7.5, 8 M urea and 1 mM DTT) and incubated for 30 min while shaking before the addition of 5.55 μl of 50 mM chloroacetamide solution for another 30 min while shaking. For on-bead digestion, 1 μl of LysC (stock solution of 0.5 μg μl−1 in water) was added for 1 h at 37 °C while shaking. The LysC digestion supernatant was transferred into a fresh sample tube, and to the remaining beads, 165 μl of a 50 mM Tris (pH 7.5) solution containing 0.25 μg trypsin was added and incubated for another 1 h at 37 °C while shaking. The two digestion solutions were combined and supplemented with an additional 0.5 μg of trypsin, and the digestion was continued overnight at 37 °C while shaking. A double layer of C18 chromatography matrix was placed into a 200-µl pipette tip and the matrix was activated by adding 100 μl of 100% methanol. The matrix was washed once with buffer A (0.1% formic acid, 80% acetonitrile in H2O) and twice with buffer B (0.1% formic acid in H2O). For each washing step, centrifugation in a tip centrifuge was performed. Afterwards, the samples were subjected to tip washing once with buffer B, before sample elution, twice, with 20 μl of buffer A. The samples were dried in a vacuum concentrator at 30 °C, and analysed by MS. All solvents used during MS measurements were LC–MS grade. The samples were dissolved in 20 μl of 0.1% TFA in water and 3 μl were subjected to nanoHPLC–MS/MS analysis. The samples were injected into an UltiMateTM 3000 RSLCnano system (Thermo Fisher Scientific), coupled online to a Q Exactive Plus Hybrid Quadrupole-Orbitrap mass spectrometer equipped with a nanospray source (Nanospray Flex Ion Source, Thermo Scientific). Samples were desalted by means of a pre-column cartridge (5 μm, 100 Å, 300-μm ID × 5 mm; Dionex) using 0.1% TFA in water and a flow rate of 30 μl min−1 for 5 min, with eluent flow to waste followed by back-flushing of the sample during the whole analysis from the pre-column to the PepMap100 RSLC C18 nanoHPLC column (2 μm, 100 Å, 75 μm ID × 50 cm, nanoViper, Dionex). A linear gradient starting with 95% solvent A (0.1% formic acid in H2O) and 5% solvent B (0.1% formic acid in acetonitrile) was used, which was increased to 30% solvent B over 90 min (flow rate 300 nl min−1). The obtained data were analysed using MaxQuant software (v.1.6.17.0)64 including the Andromeda search algorithm and using a database containing the IDO1 (UniProt accession no. P14902) or the ubiquitin (UniProt accession no. P0CG48) sequence, together with typical contaminants included in the MaxQuant software. The search was performed for full enzymatic trypsin cleavages allowing two miscleavages. For the protein modifications, carbamidomethylation was chosen as fixed, and oxidation of methionine, acetylation of the N terminus, and GlyGly of lysine as variable modifications. The mass accuracy for the full mass spectra was set to 20 ppm (first search) and 4.5 ppm (second search), respectively, and for the MS/MS spectra to 20 ppm. The FDRs for peptide and protein identification were set to 1%. Results were exported into a txt file and visualized using Prism 6.0 (GraphPad). Details of the analysis procedure are described in ref. SKOV-3 cells were seeded into a T-75 flask (or a six-well plate for the isothermal dose–response experiment) 48 h before compound addition. On day 3, the indicated iDeg-1 concentrations or DMSO as a control were added for 1 h and incubated at 37 °C and 5% CO2 before cell detachment using trypsin. The cells were washed twice with PBS (4 °C, 300g), then the cell pellet was resuspended in PBS and equally divided into ten different PCR sample tubes that were subjected to gradient heat treatment (3 min) using a Mastercycler EP gradient system (37, 40.8, 44.1, 47.4, 50.0, 53.4, 55.7, 56.3, 61.0 and 63.7 °C). For the isothermal dose–response analysis of iDeg-1, the cells were heat-treated at a constant temperature of 50 °C. Afterwards, the SKOV-3 cells were placed on ice and cell lysis was performed by adding 0.4% NP-40 before freeze–thaw lysis. Melting temperatures in the presence or absence of the test compound were determined via nonlinear regression using Prism 6.0 (GraphPad). BxPC3 cells (100,000 cells well−1) were seeded in a 96-well plate and incubated at 37 °C and 5% CO2 overnight before the addition of 25 ng ml−1 IFN-γ and the respective compounds for 24 h. Afterwards, the cells were washed three times with pre-warmed PBS followed by protein translation analysis using a Click-iT HPG Alexa Fluor 488 protein synthesis assay kit (Thermo Fisher). Fresh medium containing the respective compounds as well as 50 µM L-homopropargylglycine (L-HPG) to tag newly synthesized proteins were added to the cells, followed by incubation for 45 min at 37 °C and 5% CO2. The cells were then fixed using 4% para-formaldehyde, and the L-HPG was fluorescently labelled with Alexa 488 reagent provided by the assay kit using a click reaction as described by the manufacturer (Click-iT HPG Alexa Fluor 488 protein synthesis assay, Thermo Fisher). Cell nuclei were stained using the nuclei stain provided in the assay kit. Images were acquired at ×10 magnification using an Axiovert 200M microscope (Zeiss). The images were analysed using MetaXpress software. The L-HPG-mediated fluorescence intensity as well as a cell count analysis were performed, and the obtained data were normalized to the cell count and visualized by Fiji ImageJ. Data were plotted using Prism 6.0 (GraphPad). Full-length IDO1 cDNA was cloned into the pT7CFE vector (pT7CFE1-IDO1) and used in the in vitro translation assay with a 1-Step Human Coupled DNA IVT kit (Thermo Fisher). To obtain the proteins required for protein translation, HeLa cell lysate was prepared according to the manufacturer's instructions. The obtained lysates were combined with pT7CFE1-IDO1 plasmid and supplemented with the reagents provided with the 1-Step Human Coupled DNA IVT kit (Thermo Fisher). Full-length human IDO1 cDNA was cloned into the pGEX6p-2rbs vector using EcoRI and SalI restriction sites. For protein translation assays, full-length IDO1 cDNA was cloned into the pT7CFE vector (pT7CFE1-IDO1). Human recombinant IDO1 protein was expressed in E. coli according to ref. IDO1 protein expression in E. coli was induced by the addition of 300 µM IPTG, and the cells were incubated at 20 °C overnight. Harvested cells were resuspended in lysis buffer (50 mM Tris-HCl pH 7.4, 100 mM NaCl and 1 mM PMSF, DNase), disrupted by sonication and the lysate was centrifuged at 13,000g for 35 min at 10 °C. The supernatant was applied to a GST Trap HP column to enrich GST-tagged IDO1 equilibrated in 50 mM Tris-HCl, 150 mM NaCl and 1 mM DTE, pH 7.0. To obtain untagged full-length IDO1, GST was cleaved from IDO1 overnight using PreScission protease. Eluted IDO1 protein was further purified using a size exclusion column (26/60 G75 HiLoad) in 50 mM Tris-HCl pH 7.4, 100 mM NaCl to yield pure rhIDO1. For hIDO1crystallization, a truncated version (5–400) of IDO1 was cloned into the pGEX6p-2rbs vector using the EcoRI and SalI restriction sites. Haem was removed by overnight incubation with 166 mM 2-mercaptoethanesulfonate (MESNA) to obtain apo-IDO1. Haem occupancy was determined by UV–vis spectrophotometry. Expression constructs for KLHDC3-EloB/C (both monomeric and tetrameric versions) are described in ref. Briefly, KLHDC3, EloB and His-TEV-EloC were first cloned into pLib via Gibson assembly. Cassettes were then generated via PCR as described in ref. 67 and Gibson-assembled into pBig1a to generate a single vector for co-expression of all components. Proper assembly into pBig1a was confirmed by PmeI and SwaI restriction digestion. For the ubiquitination assays, IDO1 (residues 5-C) was cloned into a pRSF Duet based vector with an N-terminal His-TEV tag and purified as described above. KLHDC3-EloB/C was expressed in Hi5 insect cells with an N-terminal 6XHis tag on EloC as described in ref. Tagged proteins were purified from cell lysates by Ni affinity chromatography. Nickel elutions were directly loaded onto a HiTrap Q-HP column and eluted with a gradient of 2–50% NaCl. To introduce a cysteine for fluorescent labelling of UB and K0UB, we mutated the protein kinase A site in the pGEX2TK backbone converting the PKA site from RRASV to RRACV68. UB or UBK0 purified from this expression construct was labelled with AlexaFluor-647-maleimide or fluorescein-5-maleimide, respectively, as described in ref. Briefly, DTT was added to UB or UBK0 at a final concentration of 10 mM and incubated on ice for 20 min to completely reduce the cysteines for labelling. DTT was removed by buffer exchange over an NAP-5 column (GE Healthcare) in labelling buffer (25 mM HEPES, 200 mM NaCl). Labelling reactions consisted of UB or UBK0 at 150 µM final concentration and were initiated by the addition of 600 µM AlexaFluor-647-maleimide or fluorescein-5-maleimide (4× excess over labelling target and <5% final DMSO concentration). Reactions were incubated at room temperature for 2 h and quenched by the addition of DTT to 10 mM. Quenched reactions were desalted over a PD-10 column in labelling buffer containing 1 mM DTT to remove unreacted probe. Desalted protein was concentrated and further purified over a Sephadex SD75 column. The samples were incubated for 60 min, followed by the addition of TCA to a final concentration of 7% and further incubation for 30 min at 37 °C. Kyn levels were detected by the addition of an equal volume of 2% (wt/vol) p-DMAB in acetic acid, and absorbance was measured at 492 nm and 650 nm as a background control. Values are presented relative to the DMSO control. Dose–response curves and IC50 values were generated and fitted with GraphPad Prism 9.2.0 (GraphPad software) using a four-parameter, variable-slope nonlinear regression curve fit. Full-length rhIDO1 (10 µM) in 100 mM potassium phosphate buffer (pH 6.5) was incubated with the respective compound or DMSO as a control at 37 °C for 3 h, followed by a centrifugation step for 10 min at 1,000g. Afterwards, the protein solution was loaded onto a capillary, followed by the measurement of fluorescence intensities at 350 nm and 330 nm from 20 °C up to 90 °C using a Prometheus Panta device (NanoTemper Technologies). Full-length rhIDO1 (10 μM) in 100 mM potassium phosphate buffer (pH 6.5) was incubated with the respective compounds or DMSO as a control for 3 h at 37 °C followed by a centrifugation step for 10 min at 1,000g. Subsequently, UV–vis spectra of the samples were monitored between 250 and 650 nm in 1-nm increments using UV-transparent microplates (UV-STAR, Greiner) and a Spark multimode microplate reader (Tecan). All experiments were performed on a MicroCal AutoITC200 set-up. Lyophilized peptides were dissolved in ITC buffer to a final concentration of 1 mM. Titrations were performed at 25 °C with one injection of 0.4 µl, followed by 12 injections of 3 µl. To evaluate the binding affinity of iDegs to IDO1, IDO1 protein was dissolved in ITC buffer (50 mM KH2PO4, 50 mM K2HPO4 and 1 mM TCEP, pH 6.57) to a final concentration of 125 µM in the syringe, while the sample cell contained iDeg-6 at 50 µM. Titrations were performed at 25 °C with one injection of 0.4 µl, followed by 18 injections of 2 µl. Experiments were performed n = 3 times in a MicroCal PEAQ-ITC system (Malvern), and the data were evaluated with MicroCal PEAQ software. Truncated apo-IDO1 protein (residues 5–400; 10 mg ml−1, 225 µM) was incubated with 660 µM iDeg-1 (1.3% vol/vol DMSO) or 450 µM iDeg-2 (1.8% vol/vol DMSO) in binding buffer (25 mM Tris pH 8.0, 100 µM TCEP) at 37 °C for 2 h. The soluble protein fraction was separated from the precipitate by centrifugation at 20,000g for 15 min at 20 °C. Crystallization drops were prepared by mixing 0.4 µl of the rhIDO1:iDeg solution with 0.2 µl of precipitant solution in a sitting drop set-up (iQ plates, SPT Labtech) at 20 °C. The crystals appeared in one day as thin bars. After five days, the crystals were collected and flash-frozen in liquid nitrogen using the precipitant solution as cryoprotectant. X-ray diffraction data were collected at the Swiss Light Source (SLS, Villigen, Switzerland) on beamline X10SA (iDeg-1) and at the European Synchrotron Radiation Facility (ESRF, Grenoble, France) on beamline ID30B (iDeg-2). Data were processed with XDS and scaled using XSCALE. Iterative refinement cycles were carried out using phenix.refine and COOT71. Figures were generated using the PyMOL Molecular Graphics system (version 2.5.4, Schrödinger). PDB ID codes, data collection and refinement statistics are presented in Supplementary Table 2. First, UBE2R2 or UBE2D2 was pulse-labelled by incubating a mixture of UBA1 (0.3 µM), E2 (10 µM) and fluorescently labelled UB or KOUB (15 µM) in 25 mM HEPES, 100 mM NaCl, 100 mM MgCl2, 2 mM ATP, pH 7.5 at room temperature for 13 min. Pulse-loading reactions were quenched by the addition of EDTA to 50 mM and incubated on ice for 5 min. Ubiquitination chase reactions consisted of mixing the E2~*UB thioester conjugate (0.4 µM final concentration) with the indicated pre-equilibrated NEDD8~CUL2-RBX1-KLHDC3-EloB/C (0.3 µM final concentration) with a threefold excess of IDO1 variants in 25 mM HEPES, 100 mM NaCl, 50 mM EDTA, 0.5 mg ml−1 bovine serum albumin, pH 7.5 at room temperature. Experiments following loading of apo-IDO1 with small molecules consisted of incubating IDO1 (15 µM final concentration) with a twofold excess of the indicated test molecules for 1.5 h at 37 °C. Mixtures were pelleted for 10 min at 14,000 r.p.m. A UV–vis spectrum of the resulting supernatant confirmed haem incorporation, and the samples were directly diluted into ubiquitination reactions. Reactions were quenched at the indicated times with 2× SDS–PAGE sample buffer and separated on 4–12% Bis-Tris gradient gels and scanned for fluorescence on a Typhoon imager. All experiments, except where indicated, utilized a monomeric version of KLHDC3-EloB/C. To assess the relative binding strength of apo-IDO1, or small-molecule-bound variants of IDO1, we employed a competition pulse-chase ubiquitination assay. First, KLHDC3-EloB/C (50 nM) was equilibrated at room temperature for 15 min with 200 nM IDO1C-deg peptide and varied concentrations of full-length apo-IDO1, iDeg-6-bound IDO1 or linrodostat-bound IDO1. Chase reactions were then initiated by the addition of the pre-formed fluorescent UBE2R2~UB thiolester conjugate (100 nM final concentration). Reaction aliquots were removed at 30 s and quenched with 2× SDS–PAGE sample buffer, separated on 4–12% Bis-Tris gradient gels, and scanned for fluorescence on a Typhoon imager. Data were quantified and visualized as the loss of UB~IDO1C-deg ubiquitination upon increasing concentrations of full-length IDO1 variants, yielding apparent IC50 values for KLHDC3 binding by the differing IDO1 full-length variants. Biochemical assays testing the order of addition of small molecules as shown in Fig. 5m, were performed by first incubating apo-IDO1 (14 mM) with the indicated small molecules (42 µM) for 20 min at room temperature. A second test molecule was then added (42 µM) and incubated for an additional 20 min at room temperature. The mixtures were then diluted to a final concentration of 500 nM of IDO1 containing NEDD8~CUL2KLHDC3 (200 nM final concentration). Ubiquitination chase reactions were initiated by the addition of UBE2R2~UBR7 thioester conjugate (300 nM). Reactions were quenched at the indicated times with 2× SDS–PAGE sample buffer and separated on 4–12% Bis-Tris gradient gels and scanned for fluorescence on a Typhoon imager. The N-terminal stability reporter was generated starting from the previously reported C-terminal stability vector49,72. The backbone was digested using SalI and MluI (NEB), and the mCherry and TagBFP inserts were amplified using Phusion High-Fidelity DNA polymerase following the manufacturer's instructions and using the primers indicated in Supplementary Table 3. Fragments were subsequently assembled using an NEBuilder HiFi DNA Assembly cloning kit. As starting material for the genetic versions of IDO1, Addgene plasmid 187026 (pcDNA3.1-IDO1-p2a-eGFP gifted by G. van den Bogaart73) was used. The custom-made sgRNA library targeting elements of the ubiquitin proteasome system51 was generated as previously described in ref. Single sgRNAs were designed using the VBC score74 and ordered as custom oligos (Sigma). After phosphorylation (T4 PNK, NEB) and annealing of the oligos, an NEBridge Golden Gate assembly kit (BsmBI-v2) was used for insertion of the respective sgRNAs (sequences are provided in Supplementary Table 4). Insertion of the correct DNA sequence was verified by Sanger sequencing (Microsynth). For IDO1 F270, R343A, H346A and T395M, pENTR223 IDO1 was mutagenized using Q5 site-directed mutagenesis (NEB), followed by Gibson assembly to yield the respective N-terminal stability reporters. The used oligunucleotides were as follows: Lenti-X cells were transfected at 70% confluence with the reporter or sgRNA plasmids as well as the two packaging plasmids (pCMVR8.74 helper and pMD2.G envelope) using polyethylenimine (PEI MAX Mw 40,000, Polysciences). pCMVR8.74 was a gift from D. Trono (Addgene plasmid 22036). pMD2.G was a gift from D. Trono (Addgene 12259). The virus was collected and filtered using a 0.45-μm poly-ethersulfone filter. One million KBM7 iCas9 cells or KBM7 iCas9 IDO1 N-terminal stability reporter cells per 2 ml were transduced with varying volumes of virus solution and incubated with 8 µg ml−1 polybrene for 24 h before cell expansion. Cells were selected either via FACS using a CytoFLEX SRT Benchtop Cell Sorter (for reporter cell lines, BFP and mCherry positive cells, Extended Data Fig. 7c) or using G418 (1 mg ml−1, Gibco) for sgRNA-harbouring cell lines, which was added 72 h after transduction. KO cell lines were generated by selection of the respective sgRNA (sgAAVS1 = CTRL or sgRNA1/2 KLHDC3 for KO1/KO2) for at least 14 days with G418 followed by subsequent induction with doxycycline (0.4 μg ml−1, PanReac AppliChem) and cell recovery. IDO1 KBM7 reporter cells were treated for the times, compounds and concentrations indicated in the respective figure legends or as specified in the text. All compounds were diluted starting from DMSO stock solutions. Blue fluorescent protein (BFP) and mCherry levels were measured using an LSR Fortessa (BD Biosciences) with BD FACSDiva software (v9.0). To quantify the changes across conditions, the mean BFP and mCherry values were exported after gating for singlets and reporter positive cells (for the gating strategy see Supplementary Fig. Normalization was performed either against the respective wt IDO1 reporter cells treated with DMSO (Fig. 6d) or for each respective genotype (Figs. The screen, library preparation and sequencing analysis were performed as described in ref. In brief, 120 million KBM7 iCas9 IDO1 N-terminal stability reporter cells were transduced at a multiplicity of infection (MOI) of ~0.12 sgRNA-positive cells (>1,000-fold library representation, Extended Data Fig. 7e) with the customized ubiquitin focused library. Transduced cells were selected with G418 (1 mg ml−1, Gibco) for 14 days, expanded, and Cas9 expression was induced with doxycycline (0.4 μg ml−1, PanReac AppliChem). Three days after Cas9 induction, 50 million cells per condition and replicate were treated with DMSO, iDeg-1 or iDeg-2. The experiment was performed in two biological replicates. Cells were washed with PBS, stained with allophycocyanin (APC)-conjugated anti-mouse Thy1.1 antibody (1:400, 202526, BioLegend) and human TruStain FcX Fc receptor blocking solution (1:1,000, 422302, BioLegend) for 5 min at 4 °C. Subsequently, the cells were fixed with BD fixation buffer 4% (Thermo Scientific Pierce) for 45 min at 4 °C. All steps were performed protected from light. Cells were washed twice with PBS and stored in FACS buffer (PBS, 5% FBS and 1 mM EDTA). For sorting, cells were strained using a 35-μm nylon mesh and sorted on a BD FACSAria Fusion instrument (70-µm nozzle, BD Biosciences) with BD FACSDiva software (v8.0.2). Singlets, Cas9 and sgRNA-positive cells were selected following the scheme shown in Extended Data Fig. Next, fractions with different levels of BFP were enriched: the 5% highest or lowest BFP expressing cells were used for the respective HIGH and LOW fractions, and 30% of the population's centre was selected for the MID fractions. For each replicate, cells corresponding to at least a 1,000-fold library representation were sorted. To quantify the sgRNA levels per each fraction and sample, next-generation sequencing (NGS) libraries were prepared. Genomic DNA was isolated by cell lysis (10 mM Tris-HCl, 150 mM NaCl, 10 mM EDTA, 0.1% SDS) and proteinase K treatment (New England Biolabs) overnight at 55 °C, with shaking at 1,200 r.p.m., followed by 2 h of DNAse-free RNAse (Thermo Fisher Scientific) incubation at 37 °C. Next, DNA was extracted by two rounds of phenol extraction and subsequent isopropanol precipitation. Barcoded NGS libraries for each sorted population were generated using a two-step PCR protocol using AmpliTaq Gold polymerase (Invitrogen). The resulting PCR products were purified after each step using Mag-Bind TotalPure NGS beads (Omega Biotek). The final NGS libraries were pooled and sequenced on a NovaSeq 6000 platform (Illumina). The IDO1 coding sequence was PCR-amplified from pCMV3-IDO1 (Sino Biological US) using the following primer pair: Ten million HEK293T cells were transiently transfected with 4.2 µg of Flag-TurboID-IDO1 using Lipofectamine 2000 (Thermo Fisher) according to the manufacturer's instructions. At 48 h after transfection, the cells were treated with CFZ for 1 h, followed by treatment with either DMSO or 5 µM iDeg-3 for 2.5 h. During the final 15 min of the incubation, 500 μM biotin (Sigma-Aldrich) was added to the cells to induce biotinylation of proximal proteins. The cells were subsequently washed four times with cold PBS, pelleted at 300g for 5 min and lysed in urea lysis buffer (4 M urea, 50 mM Tris, 150 mM NaCl, 1% NP-40 Alternative, 2 mM EDTA, 5% glycerol, 20 mM N-ethylmaleimide, protease inhibitor cocktail, pH 7.5) for 1 h on ice, followed by sonication. Lysates were cleared by centrifugation at 17,000g for 15 min at 4 °C), then 500 µg of protein from each condition in 500 µl of lysis buffer was incubated with 60 µl of pre-washed streptavidin magnetic beads (Pierce) (washing buffer 1: 10 mM Tris, 150 mM NaCl, 0.1% Tween-20, 1 mM EDTA, pH 7.5) overnight at 4 °C with gentle rotation. The next day, the beads were washed three times with 1 ml of 4 M urea washing buffer 2 (4 M urea, 10 mM Tris, 150 mM NaCl, 0.1% Tween-20, 1 mM EDTA, pH 7.5) followed by a final wash with PBS. This elution procedure was repeated using 10 µl of 2× LDS buffer at 95 °C, and both eluates were combined. The eluates were then dried in a SpeedVac for 2 h, redissolved in 10 µl of Milli-Q water, and the complete eluate was applied to SDS gel. The scratch wound healing assay was performed as described in ref. SKOV-3 cells were seeded at 0.2 × 106 cells ml−1 in 2 ml in six-well plates and cultured overnight to reach a confluence of over 80%. After 24 h of pre-conditioning with epacadostat, iDeg-6 or vehicle, a 10-µl sterile pipette tip was used to make a scratch line on the monolayer of confluent cells at the bottom of the well. The cells were washed twice to remove cellular debris, re-exposed to stimuli or vehicle, and then incubated at 37 °C in a humidified 5% CO2 incubator for 48 h. Over this incubation time, the wound healing was continuously observed using an EVOS M5000 imaging system (Thermo Fisher Scientific) with ×10 magnification. Pictures were acquired at different time points. The area of the wound healing was determined using the MRI wound healing tool (ImageJ software, NIH) and the data are reported as percentage of wound closure in relation to time 0. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this Article. Data that support the findings of this study have been deposited in MassIVE under accession codes MSV000094270, PXD050474 (global proteome profiling) and MSV000094271, PXD050475 (IDO1 immunoprecipitation). The crystal structures of IDO1 with iDeg-1 and iDeg-2 have been deposited in the PBD under accession nos. Source data are provided with this paper. Luh, L. M. et al. Prey for the proteasome: targeted protein degradation—a medicinal chemist's perspective. Hanan, E. J. et al. Monomeric targeted protein degraders. Bekes, M., Langley, D. R. & Crews, C. M. PROTAC targeted protein degraders: the past is prologue. Grigalunas, M., Brakmann, S. & Waldmann, H. Chemical evolution of natural product structure. & Rathmell, J. C. Metabolic pathways in T cell fate and function. Munn, D. H. & Mellor, A. L. Indoleamine 2,3 dioxygenase and metabolic control of immune responses. Prendergast, G. C. et al. Indoleamine 2,3-dioxygenase pathways of pathogenic inflammation and immune escape in cancer. Fiore, A. et al. Kynurenine importation by SLC7A11 propagates anti-ferroptotic signaling. Opitz, C. A. et al. An endogenous tumour-promoting ligand of the human aryl hydrocarbon receptor. Müller-Durovic, B. et al. A metabolic dependency of EBV can be targeted to hinder B cell transformation. Maddison, D. C. & Giorgini, F. The kynurenine pathway and neurodegenerative disease. Restoring hippocampal glucose metabolism rescues cognition across Alzheimeras disease pathologies. Recent advances in the discovery of indoleamine 2,3-dioxygenase 1 (IDO1) inhibitors. Rohrig, U. F., Majjigapu, S. R., Vogel, P., Zoete, V. & Michielin, O. Challenges in the discovery of indoleamine 2,3-dioxygenase 1 (IDO1) inhibitors. Pradhan, N. et al. Inhibition of immunosuppressive indoleamine 2,3-dioxygenase by targeting the heme and apo-form. Nelp, M. T. et al. Immune-modulating enzyme indoleamine 2,3-dioxygenase is effectively inhibited by targeting its apo-form. Hennes, E. et al. Cell-based identification of new IDO1 modulator chemotypes. Mechanism of tumor rejection with doublets of CTLA-4, PD-1/PD-L1, or IDO blockade involves restored IL-2 production and proliferation of CD8+ T cells directly within the tumor microenvironment. Eynde, B. J. V. D., Baren, N. V. & Baurain, J.-F. Is there a clinical future for IDO1 inhibitors after the failure of epacadostat in melanoma?. & Prendergast, G. C. Inhibiting IDO pathways to treat cancer: lessons from the ECHO-301 trial and beyond. Van den Eynde, B. J., van Baren, N. & Baurain, J. F. Is there a clinical future for IDO1 inhibitors after the failure of epacadostat in melanoma?. Zeitler, L. & Murray, P. J. IL4i1 and IDO1: oxidases that control a tryptophan metabolic nexus in cancer. Zhai, L. et al. Tumor cell IDO enhances immune suppression and decreases survival independent of tryptophan metabolism in glioblastoma. Mellor, A. L. & Munn, D. H. IDO expression by dendritic cells: tolerance and tryptophan catabolism. Chen, W., Liang, X., Peterson, A. J., Munn, D. H. & Blazar, B. R. The indoleamine 2,3-dioxygenase pathway is essential for human plasmacytoid dendritic cell-induced adaptive T regulatory cell generation. Pallotta, M. T. et al. Indoleamine 2,3-dioxygenase is a signaling protein in long-term tolerance by dendritic cells. Rossini, S. et al. Epacadostat stabilizes the apo-form of IDO1 and signals a pro-tumorigenic pathway in human ovarian cancer cells. Brincks, E. L. et al. Indoximod opposes the immunosuppressive effects mediated by IDO and TDO via modulation of AhR function and activation of mTORC1. Identification and characterization of a novel indoleamine 2,3-dioxygenase 1 protein degrader for glioblastoma. Rational design and optimization of a potent IDO1 Proteolysis Targeting Chimera (PROTAC). Théate, I. et al. Extensive profiling of the expression of the indoleamine 2,3-dioxygenase 1 protein in normal and tumoral human tissues. Kenski, J. C. N. et al. An adverse tumor-protective effect of IDO1 inhibition. Liu, Y. et al. Blockade of IDO-kynurenine-AhR metabolic circuitry abrogates IFN-gamma-induced immunologic dormancy of tumor-repopulating cells. Efficient protection and isolation of ubiquitylated proteins using tandem ubiquitin-binding entities. Systematic and quantitative assessment of the ubiquitin-modified proteome. Chau, V. et al. A multiubiquitin chain is confined to specific lysine in a targeted short-lived protein. Kawatkar, A. et al. CETSA beyond soluble targets: a broad application to multipass transmembrane proteins. & Zoete, V. Structure and plasticity of indoleamine 2,3-dioxygenase 1 (IDO1). Identification of a novel pseudo-natural product type IV IDO1 inhibitor chemotype. Targeted protein degradation via intramolecular bivalent glues. Kagiou, C. et al. Alkylamine-tethered molecules recruit FBXO22 for targeted protein degradation. The eukaryotic proteome is shaped by E3 ubiquitin ligases targeting C-terminal degrons. Structural basis for C-degron selectivity across KLHDCX family E3 ubiquitin ligases. Targeting tryptophan catabolism in cancer immunotherapy era: challenges and perspectives. Wang, X. et al. A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination. Slabicki, M. et al. Small-molecule-induced polymerization triggers degradation of BCL6. Scholes, N. S. et al. Inhibitors supercharge kinase turnover through native proteolytic circuits. Novel selective agents for the degradation of androgen receptor variants to treat castration-resistant prostate cancer. & Wang, G. From pure antagonists to pure degraders of the estrogen receptor: evolving strategies for the same target. Ran, F. A. et al. Genome engineering using the CRISPR-Cas9 system. & Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-delta delta C(T)) method. & Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Tyanova, S. & Cox, J. Perseus: a bioinformatics platform for integrative analysis of proteomics data in cancer research. Expression and purification of recombinant human indoleamine 2,3-dioxygenase. Weissmann, F. et al. biGBac enables rapid gene assembly for the expression of large multisubunit protein complexes. Structure of a RING E3 trapped in action reveals ligation mechanism for the ubiquitin-like protein NEDD8. McCoy, A. J. et al. Phaser crystallographic software. Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Xue, G. Discovery of a drug-like, natural product-inspired DCAF11 ligand chemotype. Maassen, S. et al. Mitochondrial interaction of fibrosis-protective 5-methoxy tryptophan enhances collagen uptake by macrophages. Michlits, G. et al. Multilayered VBC score predicts sgRNAs that efficiently generate loss-of-function alleles. Research at the Max Planck Institute of Molecular Physiology and Max Planck Institute of Biochemistry was supported by the Max Planck Society. We thank the European Synchrotron Radiation Facility (ESRF) for provision of synchrotron radiation facilities under proposal no. We thank E. Arnold, S. Pruett-Miller and the St. Jude Center for Advanced Genome Engineering (CAGE) for generating the U2OS-KLHDC3 knockout cells, and St. Jude Children's Research Hospital, ALSAC and NIH P30 CA021765 for providing funding for the CAGE and work by D.C.S. E.H. acknowledges the International Max Planck Research School for a doctoral scholarship. The Compound Management and Screening centre (COMAS) in Dortmund is acknowledged for performing the high-throughput screening. We acknowledge the Core Facility Flow Cytometry of the Medical University of Vienna for access to flow cytometry instruments and assistance, and the CeMM Biomedical Sequencing Facility for NGS support. The research leading to these results also received support from the Innovative Medicines Initiative Joint Undertaking under grant no. 115489, the resources of which are composed of a financial contribution from the European Union's Seventh Framework Programme (FP7/2007–2013) and the EFPIA (The European Federation of Pharmaceutical Industries and Associations) companies' in-kind contribution. CeMM, Aithyra and the Winter laboratory are supported by the Austrian Academy of Sciences, and Aithyra is also supported by the Boehringer Ingelheim Stiftung. The Winter laboratory is supported by the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (grant agreement no. acknowledges research support by grant PID2021-122611NB-100 funded by the Agencia Estatal de Investigación of Ministerio de Ciencia e Innovación (MCIN/AEI/ 10.13039/501100011033). acknowledges the European Union – NextGenerationEU under the Italian Ministry of University and Research (MUR) National Innovation Ecosystem grant ECS00000041–VITALITY–Spoke 8. is further supported by FWF postdoctoral Esprit fellowship ESP 426 and a Marie Skłodowska-Curie postdoctoral fellowship (grant agreement no. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Open access funding provided by Max Planck Society. These authors contributed equally: Elisabeth Hennes, Belén Lucas, Natalie S. Scholes, Xiu-Fen Cheng. Max-Planck-Institut für Molekulare Physiologie, Abteilung Chemische Biologie, Dortmund, Germany Elisabeth Hennes, Belén Lucas, Xiu-Fen Cheng, Katharina Reich, Lisa-Marie Pulvermacher, Lara Dötsch, Alexandra Brause, Siska Führer, Kesava Reddy Naredla, Kamal Kumar, Petra Janning, Slava Ziegler & Herbert Waldmann Elisabeth Hennes, Lara Dötsch, Malte Gersch & Herbert Waldmann CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences, Vienna, Austria Natalie S. Scholes, Hana Imrichova & Georg E. Winter Department of Structural Biology, St. Jude Children's Research Hospital, Memphis, TN, USA Compound Management and Screening Center Otto-Hahn-Str.11, Dortmund, Germany Immunoregulation Research Group, Max Planck Institute of Biochemistry, Martinsried, Germany Chemical Genomics Centre, Max-Planck-Institut für Molekulare Physiologie, Dortmund, Germany AITHYRA Research Institute for Biomedical AI, Vienna, Austria Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar S.S. adapted and performed the screening assays. Correspondence to Georg E. Winter or Herbert Waldmann. is on the Scientific Advisory Board of Proxygen and Nexo Therapeutics. The Winter laboratory has received research funding from Pfizer. The other authors declare no competing interests. Nature Chemistry thanks Elmar Wolf and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic overview of the Kyn screening assay. BxPC3 cells were stimulated with IFN-γ to induce IDO1 expression via the JAK/STAT signaling pathway. After 48 h treatment with IFN-γ, Trp and the compounds, cells were hydrolysed and Kyn levels are quantified in the cell supernatant by means of the Kyn sensor. b, Kyn assay in HeLa cells treated with IFN-γ, Trp and iDeg-1 for 48 h prior Kyn level detection utilizing p-DMAB. Mean values ± SD, n = 3 biological replicates. c, Kyn levels in BxPC3 cells treated with IFN-γ, Trp and iDeg-1 for 48 h. Kyn levels were detected using LC–MS/MS. Mean values ± SD, n = 3 biological replicates. d,e, Influence of iDeg-1 (5 µM) on cell death in SKOV-3 cells WT or IDO1 knockout (KO) cells grown in 2D (d) or 3D (e). Cell death was detected by means of CellTox Green in presence or absence of IFN-γ (1 ng/mL). f, Influence of iDeg-1 on the in vitro IDO1 enzymatic activity. rhIDO1 was pre-incubated with the compounds at 37 °C for 90 min prior to detection of Kyn levels using p-DMAB. The apo-IDO1 inhibitor linrodostat was used as a control. Mean values ± SD, n = 3 biological replicates. BxPC3 cells were treated with IFN-γ and iDeg-1 for 24 h prior to isolation of RNA and RT-qPCR analysis. Mean values ± SD, n = 3 biological replicates. h, Influence on the IDO1 promoter activity. HEK293T cells were transfected with a construct encoding firefly luciferase (Fluc) under the control of the IDO1 promoter and a plasmid for constitutive expression of Renilla luciferase (Rluc). Cells were treated with IFN-γ and the compounds for 24 h. Fluc/Rluc ratios were normalized to samples that were treated with IFN-γ only (set to 100%). Mean values ± SD, n = 3 biological replicates. i,j, In vitro IDO1 protein translation. IDO1 translation was analysed in cell lysates in the presence of iDeg-1, cycloheximide (CHX) or DMSO (CTL). Representative immunoblot (one of n = 3 biological replicates) (i) and quantified band intensities (j). j, Mean values ± SD, n = 3 biological replicates. Representative immunoblot (one of n = 4 biological replicates) (k). (l) Normalized band intensities from k. Mean values ± SD, n = 4 biological replicates. m, n, Analysis of global protein translation in cells 24 h after compound treatment by means of HPG detection. Representative images of n = 3 biological replicates of HPG intensities displaying protein translation (green) and nuclear stain (blue). n, Normalized HPG signal intensities of the images in m. Mean values ± SD, n = 3 biological replicates. a, TUBE pulldown after treatment of IFN-γ-stimulated BxPC3 cells with iDeg-1 or DMSO. Cells were pre-treated for 1 h with the proteasome inhibitor carfilzomib (CFZ, 450 nM) followed by addition of 50 µM iDeg-1 for 4 h prior to lysate preparation and TUBE pulldown. Representative immunoblot (one of n = 3 biological replicates). 24 h later, Kyn levels were detected using p-DMAB. Mean values ± SD, n = 4 biological replicates. c, HEKrhIDO1 cells treated with Trp and iDeg-1 for 24 h prior Kyn level detection using p-DMAB. Mean values ± SD, n = 3 biological replicates. d,e, HEK239T cells were electroporated with rhIDO1 protein followed by treatment with iDeg-1 (10 µM) for 6, 14 or 24 h. IDO1 protein levels were determined via immunoblotting (d). The effect of iDeg-1 in co-treatment conditions is represented relative to the treatment of carfilzomib (CFZ) alone (set to 100 %). Mean values ± SD, n = 3 biological replicates. g, HEK293T cells were transfected with an IDO1 expression plasmid 24 h prior the addition of Trp and iDeg-1. Kyn levels were determined 24 h after compound addition using p-DMAB. Mean values ± SD, n = 3 biological replicates. a,b, Influence of iDeg-1 on the protein levels of DOCK-8 and RHOBTB3 in HEK293T cells treated for 6 h or 24 h. Representative immunoblots (one of n = 3 biological replicates) (a) and quantification of band intensities from a (b). Mean values ± SD, n = 3 biological replicates. c-e, IFN-γ-stimulated BxPC3 cells were treated with iDeg-1 or DMSO as a control for 6 h prior to lysate preparation. IDO1 was immunoprecipitated and the enriched fraction was digested and analysed by MS. c, IDO1 protein sequence. Due to incomplete sequence coverage in the MS experiment, ubiquitination of additional lysines cannot be excluded. d, Measured relative peptide intensity for IDO1 and polyubiquitin-C. Mean values ± SD, n = 3 biological replicates. Identified peptides are displayed in bold. The identified ubiquitination site K48 is indicated. K6, K11, K29 and K33 in ubiquitin could not be identified due to incomplete sequence coverage in the MS experiment. SKOV-3 cells were treated with iDeg-1 or DMSO as a control for 1 h at 37 °C prior to heat treatment at 50 °C, lysis and analysis of the soluble fraction via immunoblotting. Representative immunoblot (one of n = 3 biological replicates) for IDO1 (a) and respective melting curve (b). c, Kyn assay in BxPC3 cells treated with iDegs, Trp and IFN-γ for 48 h prior to detection of Kyn levels utilizing p-DMAB. Mean values ± SD, n = 3 biological replicates. d, HEK293T cells were transiently transfected with pCMV3-IDO1 24 h prior to compound addition. Another 24 h later, Kyn levels were detected utilizing p-DMAB. Mean values ± SD, n = 3 biological replicates. e-g, Influence of hemin on iDeg activity. Kyn assay in IFN-γ-stimulated BxPC3 cells for iDeg-1 (e), iDeg-2 (f) or iDeg-3 (g) in the presence or absence of 5 µM hemin. Kyn levels were detected after 48 h using p-DMAB. Mean values ± SD, n = 3 biological replicates for iDeg-1, 3 and n ≥ 2 biological replicates for iDeg-2. IFN-γ-stimulated BxPC3 cells were incubated with different concentrations of hemin in presence or absence of 0.5 µM iDeg-2 for 48 h prior to detection of Kyn levels with p-DMAB. Mean values ± SD, n = 3 biological replicates. a, Electron density maps (2Fo-Fc) calculated with phases from the final refined models, contoured at 1.5 σ, showing the refined iDeg-1 and iDeg-2 structure in stick representation. b, Overlay of iDeg-2 (violet) with the holo-IDO1 inhibitor epacadostat in complex with haem (PDB ID: 5WN8, dark gray) and with the apo-IDO1 inhibitor linrodostat (PDB ID: 6DPRB, light gray). c, Cartoon representation of the iDeg-1 and iDeg-2 binding sites. Residues involved in hydrophobic interactions and H-bonds are indicated. Hydrophobic interactions between iDegs and IDO1-protein within a range of 4.0 Å are shown with dashed lines in the color corresponding to each iDeg. Hydrogen bonds are indicated as black dashed lines. d. Putty diagrams representation of the relative B-factor changes generated in PyMOL. Blue colors with small diameter tubes indicate lower B-factors and less mobility, while orange to red colors with large diameter tubes indicate regions of higher B-factors and increased mobility within the proteins. e, The highly similar crystal lattice packing of IDO1-iDeg-2 and IDO1-apoxidole indicates that the absence of electron density for the K-helix (dark gray) in the iDeg-2 structure is not an artefact of crystal packing. The J-helix is highlighted in green (8ABX) or purple (9FOH). f, Overlay of the IDO1-iDeg-2 structure with all published IDO1 structures, excluding seven with high coordinate uncertainty. Shown are the front view, a 90° rotated side view, and a zoom-in view of the E- and J-helices, with the positions of H346 and R343 indicated. a, Representative flow-cytometric histogram for iDeg-2 mediated depletion of BFP-IDO1 (24 h, 10 µM) in KBM7 iCas9 cells. b, Identification of genes required for iDeg-2-mediated IDO1 degradation. d, Detection of IDO1 levels using the IDO1 stability reporter. KBM7 IDO1 reporter cells were treated with iDeg-2 or 3 (10 µM) +/- indicated co-treatments for 10 h prior to detection of IDO1 levels using flow cytometry. Mean values ± SD, n = 3 biological replicates. e, IDO1 protein levels in KBM7-BFP-IDO1 CTRL or KBM7-BFP-IDO1 KLHDC3 KO1 or KO2 cells upon treatment with iDeg-1, 2 or 3 (1 µM) for 24 h. Values were normalized to the respective DMSO control, which was set to 1. Mean values ± SD, n = 3 biological replicates. a, Kyn assay in BxPC3 cells after treatment with iDeg-6 and IFN-γ, respectively, for 48 h prior to detection of Kyn levels utilizing p-DMAB. Mean values ± SD, n = 3 biological replicates. b,c, IDO1 protein levels in BxPC3 cells upon treatment with IFN-γ for 24 h followed by washout and addition of iDeg-2, 3 or iDeg-6 for 24 h. Representative immunoblot (one of n = 5 biological replicates) for IDO1 and vinculin as a loading control. c, Quantified band intensities from b. Mean values ± SD, n = 5 biological replicates. d, Reduction of IDO1 levels determined in the BFP-IDO1 reporter cell line. Mean values ± SD, n = 3 biological replicates. rhIDO1 was pre-incubated with the compounds at 37 °C for 90 min prior to detection of Kyn levels using p-DMAB. Mean values ± SD, n = 3 independent experiments. Representative data of n = 3 independent experiments. Incubation at 37 °C for 3 h. Representative data (n = 3 independent experiments). Contribution of IDO1 degradation to the reduction in Kyn levels. BxPC3 cells were stimulated with IFN-γ for 24 h followed by a washout and co-treatment with carfilzomib (CFZ, 500 nM) and iDeg-6 (100 nM or 1 µM) for 7 h. Kyn levels (h-i) and IDO1 protein levels (j) were determined. Kyn levels were normalized to the respective control (DMSO or CFZ alone in h and DMSO in i). Mean values ± SD, n = 3 biological replicates. a, IDO1 protein levels in SKOV-3 and BT549 cells. Quantification of band intensities from Fig. 5d and f. Mean values ± SD, n = 3 biological replicates. b-e, IDO1 protein levels in SKOV-3 (b and c) or BT549 cells (d and e) after treatment with iDeg-6 for different time. IDO1 and vinculin (VCL) were detected by means of immunoblotting. Quantification of band intensities from b and d is shown in c and e, respectively. f, Influence of neddylation inhibition on IDO1 protein levels in BxPC3 cells. BxPC3 cells were stimulated with IFN-γ for 24 h prior to washout, treatment with 500 nM MLN4924 (MLN) and iDeg-6 for 16 h and immunoblotting. Representative blot (one of n = 3 biological replicates). g, Detection of CRISPR–Cas9-mediated KLHDC3 knockout in U2OS cells. Representative blot (one of n = 3 biological replicates). h, IDO1 protein levels in U2OS and U2OS cells carrying a KLHDC3 knockout. Representative blot (one of n = 3 biological replicates). iDeg6-IDO1 was incubated with CRL2KLHDC3 E3 complex and ubiquitination reactions were initiated by the addition of the pre-formed thioester-linked E2~ubiquitin conjugate (UBE2R2 ~ UB) with the ubiquitin fluorescently-labeled. b, Detection of haem load for the conditions in c detected by means of UV/Vis spectroscopy. c, Fluorescent scan of pulse-chase assay monitoring ubiquitination of apo-IDO1 with or without supplementation with small molecules by CRL2KLHDC3 (b). Apo-IDO1 was incubated with the indicated compounds (30 µM) for 90 min at 37 °C prior to incubation with CRL2KLHDC3 E3 complex. Representative data for n = 2 independent experiments. d, KLHDC3-dependent IDO1 C-terminal peptide ubiquitination (IDO1C-deg) upon titrating increasing concentrations of competing full-length apo-IDO1, iDeg-6-IDO1, or linrodostat-bound IDO1 to measure IC50 values (n = 2 independent experiments). e,f, BxPC3 cells were treated with 50 ng/mL IFN-γ with or without 10 µM SA for 24 h followed by the addition of hemin for further 24 h. f, Quantification of band intensities from e. Mean values ± SD, n = 3 biological replicates. g, Quantification of band intensities from Fig. SKOV-3 cells were treated with SA for 48 h prior to the addition of hemin for another 24 h and immunoblotting (see Fig. Mean values ± SD, n = 3 biological replicates. h, Detection of IDO1 mRNA levels in SKOV-3 cells upon treatment with succinyl acetone (SA) for 24 h followed by addition of hemin for another 24 h. Mean values ± SD (n = 3 biological replicates). i, Influence of selected IDO1 inhibitors (10 µM) in comparison to iDeg-6 (1 µM) on IDO1 protein levels in KBM7-BFP-IDO1 cells. Cells were treated with the compounds for 24 h followed by quantification of IDO1 using flow cytometry. Mean values ± SD, n = 3 biological replicates. j, Wound healing assay using SKOV-3 cells in the presence of epacadostat (1 µM), iDeg-6 (1 µM) or DMSO as a control. For each time point, one representative image of three is shown. Representative data of n = 3 independent experiments. Incubation at 37 °C for 3 h. Representative data of n = 3 independent experiments. Measurement of KLHDC3 binding to C-terminal IDO1 peptides using ITC. Measurement of IDO1 binding to iDeg-6 using ITC. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Statistical source data and unprocessed western blots. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Monovalent pseudo-natural products supercharge degradation of IDO1 by its native E3 KLHDC3. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41561-025-01878-y'>Oligocene deep ocean oxygen isotope variations primarily driven by temperature</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 11:00:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Our understanding of the long-term behaviour of global climate and the Antarctic ice sheet relies heavily on the oxygen isotopic composition of marine calcite (δ18Ocalcite), which reflects a combination of ocean temperature and the amount of water stored in ice sheets. On the basis of δ18Ocalcite, the Antarctic ice sheet has been interpreted as extremely dynamic in the Oligocene, 34–23 million years ago. Yet, the proposed continental-scale ice volume changes are challenging to reproduce with models and may be overestimated owing to a larger influence of temperature on the deep-sea δ18Ocalcite than previously assumed. Here we present the first Oligocene record of orbital variability in deep ocean temperature based on benthic foraminiferal clumped isotope thermometry, a method affected only by temperature and independent of seawater chemistry. We find large, eccentricity-paced temperature variations of up to 4 °C, sufficient to explain the δ18Ocalcite cycles without requiring continental-scale ice volume changes. This finding is consistent with the simulated stability of the Antarctic ice sheet, highlighting the importance of robust independent temperature reconstructions. Our results show that the temperature in the deep Southern Ocean, and possibly globally, is highly sensitive to the seasonal distribution of insolation in an Oligocene-like climate state warmer than today. The geological past provides the opportunity to assess the behaviour of the climate system and the long-term stability of the Antarctic ice sheet under atmospheric carbon dioxide levels close to, and exceeding, those projected for the end of this century1. Both the global mean climate state and the behaviour of ice sheets are commonly inferred from deep ocean oxygen isotope records obtained from the microscopic calcite shells of benthic organisms called foraminifera (hereafter δ18Ocalcite)2,3,4. However, owing to a multitude of overlying influences on deep ocean δ18Ocalcite, isolating specific aspects of the climate system, such as continental ice volume or deep ocean temperature, requires additional assumptions or constraints (ref. In the Oligocene (34–23 million years ago, Ma) when atmospheric CO2 ranged between 350 and 800 ppm1, the Arctic was ice-free6, whereas Antarctic-proximal sedimentological records indicate the presence of a continental-sized Antarctic ice sheet7,8. On the basis of the occurrence of large (~0.5–1.0‰) fluctuations in deep ocean δ18Ocalcite records from the Atlantic and Pacific Oceans9,10, Antarctic ice volume in the Oligocene has been interpreted to have fluctuated by 34–90% of the modern ice sheet volume at orbital periodicities of 40 and 110 thousand years (kyr)11,12,13,14. Crucially, the upper range of these estimates would imply that, at elevated atmospheric CO2 relative to pre-industrial levels1, the Antarctic ice sheet could enter a dynamical state similar to the Northern Hemisphere ice sheets during the glacial–interglacial cycles of the Quaternary (the past 2.6 million years)15. Cyclical seaward advances and retreats of at least portions of the Antarctic ice sheet in the Oligocene are supported by ice-proximal sedimentological evidence7,16. However, ice sheet models are unable to reproduce the largest variations in ice volume interpreted from δ18Ocalcite with a range of CO2 even larger than the current best estimates of Oligocene atmospheric CO2 (refs. This is because strong stabilizing feedbacks limit ice sheet retreat from ice surface melting, and a topography without deep subglacial basins reduces the retreat of marine-based ice17. These conflicting lines of evidence currently obscure our understanding of the Antarctic ice sheet behaviour on orbital time scales. A more stable Antarctic ice volume, as indicated by models, could be explained if the documented Oligocene variations in δ18Ocalcite were driven primarily by factors other than continental ice volume. Benthic foraminiferal δ18Ocalcite reflects both the temperature and the isotopic composition of seawater (δ18Osw), with the latter influenced globally by the volume and isotopic composition of continental ice, and to a lesser extent, regionally, by variations in surface ocean salinity at sites of deep-water formation5,20. Owing to the dominant influence of temperature, ice volume estimates from δ18Ocalcite require either assumptions of relatively constant deep ocean temperatures21,22 or independent temperature reconstructions. Here, we derive independent estimates of temperature using clumped isotope thermometry (Δ47)23 to test for orbital-scale deep ocean temperature variability during the interval spanning 28.2–27.2 Ma, which encompasses the largest δ18Ocalcite cycles of the Oligocene9,10 (Fig. Our aim is to test whether deep ocean temperatures vary on eccentricity timescales and, if so, to determine the proportion of the δ18Ocalcite signal driven by temperature variability. Δ47 is independent of δ18Osw and seawater elemental ratios, which affect the traditional trace element-based paleothermometers such as foraminiferal Mg/Ca ratios23,24, and it does not appear to be measurably affected by organism biology23,25,26. Because of their microscopic size, the application of Δ47 to foraminifera has until recently been limited by the requirement of large sample amounts (more than 10 mg)27. Advances in analytical techniques28,29, however, have increased the scope of possible applications of this proxy (for example, refs. Here, we present the first deep ocean temperature record derived by clumped isotopes that resolves the high-amplitude 110-kyr δ18Ocalcite cycles observed in the Oligocene. The 110-kyr cycles (that is, the combined 95-kyr and 125-kyr eccentricity components) reflect the shortest periodicity of changes in the shape of Earth's orbit, which, in association with other orbital parameters, controls the amount and seasonal distribution of incoming solar radiation (Fig. Our independent deep ocean temperature record comes from Ocean Drilling Program (ODP) Site 699 in the subpolar Southern Ocean at a present-day water depth of 3,705 m (ref. 1 and Extended Data Fig. An orbital chronology was developed using sediment colour reflectance and δ18Ocalcite records (Extended Data Figs. A total of 1,082 individual benthic foraminiferal Δ47 measurements from 99 discrete samples (Extended Data Figs. 5–8 and Supplementary Table 2) were binned into 32 temperature data points (Methods) that trace the large δ18Ocalcite cycles characterizing the mid-Oligocene9,10 (Fig. 1), with an average temporal resolution of 28 kyr (Fig. This binning is necessary because, with the small-aliquot approach applied here, the precision of individual Δ47 measurements is insufficient for paleoclimatic interpretation (Methods). The fidelity of our Δ47 binning is confirmed by good agreement with a moving window Gaussian filter, averaging temperature with a resolution of 25 kyr (Methods; Extended Data Fig. a–c, Eccentricity-paced cycles in bottom water temperature (a), together with δ18Ocalcite (‰VPDB) records from Site 699 (b), the Atlantic (Site 1264)10 and Pacific Ocean (Site 1218)9 (c). In b, δ18Ocalcite is shown both for all samples (continuous purple line) and averaged in the same way as the Δ47 record (purple dots) to obtain the Site 699 record of δ18Osw (‰VSMOW) (d). In a and d, the data are presented as mean values with light and dark envelopes in a and stippled and continuous error bars in d representing the 95% and 68% confidence intervals, respectively, derived from 20–46 measurements sourced from two to five individual sediment samples (Methods; Supplementary Table 2). e, The cycles in Earth's orbital eccentricity57 with higher values indicating the times of greater southern hemisphere summer insolation during precession maxima. Black and white bars indicate the magnetic polarity stratigraphy at Site 699 (ref. The yellow bands mark the 110-kyr eccentricity cycles captured at Site 699. The horizontal bars in a, b and d reflect the age ranges of the adjoining two to five samples combined for calculating the mean temperature and δ18Osw values (Main and Methods). The inset map shows the modern location of ODP Sites 699, 1264 and 1218. Map generated with Ocean Data View59. The δ18Ocalcite data obtained for each sample as a co-product of the Δ47 measurements (Methods) provide a higher-resolution record that reproduces the large eccentricity-paced δ18Ocalcite cycles observed at other sites9,10 (Fig. The δ18Ocalcite data were additionally binned in the same way as the Δ47 data and combined with their corresponding temperature data points, to extract the residual variability in δ18Osw (Methods; Fig. Mid-Oligocene bottom water temperatures at Site 699 varied between 4.3 °C (±1.8 °C) and 10.8 °C (±1.5 °C) (uncertainty reported at 68% confidence intervals; Methods) within the study interval and averaged 7.2 °C ± 1.4 °C (Fig. 1), substantially warmer than the present-day bottom water temperature of 0 °C at this site34 (Extended Data Fig. When compared with reconstructions based on δ18Ocalcite for the study interval, our Δ47-derived mean deep ocean temperature is significantly warmer than the 1.4 °C estimate of Rohling et al.13 but within error of the 5.6 °C estimate of Cramer et al.35. Our reconstructed temperature is close to the mean deep ocean temperature of 6.5–7 °C obtained from Mg/Ca thermometry for the interval 28–26 Ma (ref. 35), although the absolute temperatures derived from Mg/Ca are highly dependent on calibration choices (for example, refs. 14,36), and also match with clumped isotope-based estimates for the mean Oligocene temperature in the deep North Atlantic (7.7 °C ± 1.1 °C between 33.5 and 24.4 Ma)37. The overall deep ocean warmth reconstructed for the mid-Oligocene furthermore corresponds well with the reconstructed contemporaneous sea surface temperatures seasonally exceeding 10 °C at sites in the Southern Ocean38 and on the Antarctic margin39,40,41, the likely source region for the bottom water bathing Site 699 (refs. The calculated δ18Osw values average 0.57‰ ± 0.34‰ within the study interval. If reflecting only continental ice volume, this value would indicate a substantially larger ice volume than at present. The direct translation of δ18Osw into continental ice volume (for example, ref. 12), however, is complicated by the additional influences on δ18Osw, including δ18Oice of the ice sheet and regional, salinity-related changes in δ18Osw. Large interbasin salinity gradients have been reconstructed for the deep ocean in the mid-Pliocene44, highlighting that this factor must be considered when interpreting δ18Osw, especially in climate states warmer than today. We note that both the absolute temperature and δ18Osw would be lower by around 1.5 °C and 0.35‰ respectively, with alternative clumped isotope calibrations45,46 (Methods). This difference in absolute temperatures, however, does not exceed uncertainty (Methods), and, crucially, the relative changes in temperature and their relationship with observed variations in δ18Ocalcite remain independent of our calibration choice. At Site 699, sediment properties (Methods), δ18Ocalcite and bottom water temperature show a cyclicity with a dominant 110-kyr (short eccentricity) pacing throughout the studied interval (28.2 to 27.2 Ma; Fig. 1 and Extended Data Figs. The studied interval captures a total of six 110-kyr cycles (Fig. 1 and Extended Data Fig. 9), with the youngest three being the largest 110-kyr cycles of the Oligocene and corresponding to up to 1‰ changes in δ18Ocalcite, globally9,10 (Fig. In the South Atlantic, these δ18Ocalcite cycles have been interpreted to reflect changes in Antarctic ice volume equivalent to 100% of the modern East Antarctic ice sheet11. By contrast, our record shows distinct temperature changes of ±2.8–4.4 °C (±1.4 °C) associated with each of the largest δ18Ocalcite cycles (Δδ18Ocalcite ± 0.6–1‰) (Fig. This amplitude of eccentricity (110-kyr)-paced temperature variations in the mid-Oligocene is slightly larger than 2–3 °C obliquity (40-kyr)-paced temperature variations reconstructed with Mg/Ca thermometry for the late Oligocene deep North Atlantic Ocean12. With an empirical relationship between δ18Ocalcite and temperature of approximately −0.22‰ per degree Celsius20,47, our reconstructed changes in deep ocean temperature alone are in principle sufficient to explain the δ18Ocalcite cycles observed at our site, without requiring continental-scale changes in Antarctic ice volume (Fig. Our data clearly demonstrate that deep ocean temperatures can exhibit a high degree of variability on orbital timescales (Fig. 2), and therefore, δ18Ocalcite cannot be interpreted to reflect δ18Osw (and ice volume) changes in the absence of independent constraints on temperature. This suggests that the previous assumptions that led to the interpretation of continental-scale ice volume fluctuations based on Oligocene δ18Ocalcite alone need revising11. a,b, The largest 110-kyr cycles in the younger part of our record from Site 699, for which the age model is best constrained, showing eccentricity maxima matching maxima in bottom water temperatures (a) and the absence of consistent cyclical changes in δ18Osw (‰VSMOW) (b), suggesting no orbital variability in Antarctic ice volume. c–e, The results in a and b are consistent with Oligocene ice sheet model simulations17 showing minimal ice volume variability between 500 ppm (c) and 840 ppm (d) atmospheric CO2 due to melting in coastal regions being compensated by increased ice thickness inland at higher CO2 (e). The yellow and purple dots in a and b indicate values included in the eccentricity maxima groups for the purpose of testing statistical significance of peak versus background values (Methods; Extended Data Fig. In a and b, the data are presented as mean values with horizontal stippled lines representing the 68% confidence intervals of the proxy data derived from 20–46 measurements sourced from two to five individual sediment samples (Methods; Supplementary Table 2). The vertical lines reflect the age ranges of the adjoining two to five samples combined for calculating the mean temperature and δ18Osw values (Main and Methods). The eccentricity values are from ref. Panels c–e adapted from ref. 17 under a Creative Commons license CC BY 4.0. The large analytical uncertainty of our residual δ18Osw data prevents detailed interpretations of this signal, leaving open the possibility of some ice volume-related changes in δ18Osw. Nonetheless, we do not see any evidence for systematic, large-scale changes in δ18Osw on orbital timescales, as it would be expected if cyclical, continental-scale, waxing and waning of the Antarctic ice sheet was the primary driver behind the δ18Ocalcite record (Methods; Fig. 2 and Extended Data Fig. Given the large deep ocean temperature changes we observe, the δ18Ocalcite cycles could only accommodate for continental-scale fluctuations in ice volume—equivalent to melting the modern East Antarctic ice sheet, resulting in ~59 m of sea-level change14—if large changes in salinity and/or in δ18Oice were counteracting the ice volume component in the δ18Osw signal (Fig. The δ18Oice varies with ice sheet size owing to the effects of changing altitude and transport distance, but these effects are of the opposing sign, and they would probably be insufficient to offset large-scale ice volume changes48. The scale of salinity-related change in δ18Osw in the study area required to offset continental-scale ice volume changes would have to approach the difference observed between deep Atlantic and Pacific water masses in the Pliocene44. Further, any regional salinity-related change in δ18Osw would need to be compensated for by opposing signals elsewhere in the ocean to maintain mass balance, which may be hard to achieve given the volume of deep water that was probably formed in the Southern Ocean. For these reasons, it seems unlikely that bottom water salinity at Site 699 changed to the extent required to fully mask the large ice volume-related changes proposed in earlier studies. Hence, the most parsimonious explanation to account for the mid-Oligocene eccentricity cycles captured in the deep ocean oxygen isotope records is that they were primarily driven by temperature, with possible small variations in ice volume not exceeding the lower end estimates from Oligocene sea-level change reconstructions13,14. The warmest deep ocean temperatures at our site coincide with eccentricity maxima, within age model uncertainty (Methods; Figs. The magnitude of the temperature and δ18Ocalcite changes appears to be proportional to the degree of change in eccentricity, with intervals of less pronounced eccentricity, such as between 28.1 and 27.9 Ma, corresponding to more muted temperature and δ18Ocalcite variability (Fig. Hence, our new temperature reconstructions suggest a strong response of deep ocean temperature in the Oligocene to orbital variations in insolation (Fig. The deep ocean temperature reflects the surface temperature in areas of deep-water formation27, which, for Site 699, and much of the global ocean during the Oligocene, were most probably located in the Southern Ocean42,43. Our record could thus reflect regional eccentricity-paced temperature variations in the surface ocean surrounding Antarctica, with implications for ocean overturning and associated heat and carbon sequestration in the deep ocean. Alternatively, the eccentricity-paced deep ocean temperature variations could reflect switches between two different water masses bathing Site 699, characterized by different temperatures, akin to the Circumpolar Deep Water and Antarctic Bottom Water today (Extended Data Fig. Similar temperature reconstructions from other sites are needed to distinguish between these scenarios. Regardless, given its location in the deep Southern Ocean, Site 699 is probably representative for a substantial portion of the global deep ocean, indicating a strong eccentricity imprint on ocean circulation and heat distribution during the middle Oligocene. If the 110-kyr eccentricity-paced δ18Ocalcite cycles observed in the deep ocean are dominated globally by temperature changes on the order of ~3–4 °C (Fig. 1), this may offer a way to reconcile ice sheet modelling, suggesting a relatively stable Oligocene Antarctic ice volume, with the deep ocean δ18Ocalcite records. In the Oligocene, the Antarctic ice sheet was probably not marine-based as it is today due to elevated bedrock topography17,49 (Fig. The warm surface ocean temperatures proximal to Antarctica39,40,41 would have furthermore prevented an extensive marine-based ice sheet from forming. Models suggest that a land-based ice sheet is much less susceptible to large-scale melting compared with marine-based ice sheets, even under high CO2 concentrations (>1,000 ppm), resulting from self-stabilizing feedbacks caused by surface-elevation mass balance and albedo17,19,49,50,51,52,53,54,55. In model simulations that incorporate data-constrained Oligocene bedrock topographies for Antarctica, the ice volume loss obtained for a CO2 change from 500 to 840 ppm, in line with Oligocene estimates1, is negligible17 (<0.1%) (Fig. For simulations with a low concentration of atmospheric CO2 of 280 ppm, which is lower than proxy reconstructions1, the ice volume change is equivalent to 25% of the modern-day ice volume17,19. In these simulations, melting in Antarctic coastal areas is counteracted by the increased ice thickness in the interior due to more abundant snowfalls in a warmer climate (Fig. Therefore, changes in the spatial extent of the Antarctic ice sheet7,16 may not necessarily correspond to large ice volume changes (Fig. 2) and can explain changes in deep ocean temperatures decoupled from Antarctic ice volume41,56. Our work thus supports a very different response of the middle Oligocene Antarctic ice sheet to orbital variations compared with the ice sheets of the Quaternary at substantially lower levels of atmospheric CO2. Our new data show temperature fluctuations in the deep ocean of up to 4 °C consistent with a mostly terrestrial Oligocene Antarctic ice sheet between 28 and 27 Ma that may have responded to orbital forcing by changes in its spatial extent but with relatively small variations in ice volume. This finding contradicts previous interpretations of δ18Ocalcite fluctuations being the expression of continental-scale glacial–interglacial cycles in Antarctic ice volume. The large, eccentricity-paced (110 kyr) deep water temperature swings we reconstruct for the Southern Ocean (Figs. 1 and 2) suggest a climate state that is highly sensitive to external forcing during the middle Oligocene. At this time, the ocean, rather than ice sheet volume, was the primary component of the climate system mediating and responding to orbital variability in insolation. This may be an inherent feature of a warmer-than-present-day climate state such as the Oligocene. Our findings stress the importance of robust deep ocean temperature reconstructions and the need for more such records to reconcile our understanding of the Antarctic ice sheet from both the geological record and model simulations, and to understand the behaviour of the climate system in an Oligocene-like state characterized by the unipolar glaciation on Antarctica. ODP Site 699 (51°32.537′ S, 30°40.619′ W) was drilled as a single hole (Hole 699A) in the Atlantic sector of the Southern Ocean at a water depth of 3,705 m, underneath the Antarctic Circumpolar Current (ACC) and Circumpolar Deep Water, and is today bathed by Antarctic Bottom Water33,60 (Extended Data Fig. The paleolatitude of Site 699 in the Oligocene was very close to the modern61. The high abundance of siliceous microfossils in studied cores 699A-20H and 21H indicate a high productivity regime typical of the ACC region already in the Oligocene. This is corroborated by micropaleontological evidence62 indicating that, since the early Oligocene, the Atlantic Ocean south of 50 °S was under the influence of the ACC and had a polar oceanographic regime. Sediments recovered at Site 699 possess a clear characteristic remanent magnetization, determined by means of shipboard continuous measurements on the archive halves and integrated by onshore analysis of discrete samples58. The newly generated diatom biostratigraphy supports the magnetic polarity correlation of cores 699A-20H and 699A-21H within subchrons C9n–C10n. This assignment is based on the occurrence of the following marker taxa: Rocella vigilans, Kozloviella minor and Cestodiscus trochus. Especially K. minor is reported from other Southern Ocean deep-sea holes for which diatom biostratigraphy is available, including ODP Holes 748B and 749A63. The best constrained Southern Ocean record of this species is from Hole 748B, where K. minor occurs within a narrow interval spanning Chron C10n. To refine this initial biomagnetostratigraphic framework, red–green–blue (RGB) data was extracted from the core images (Extended Data Fig. We corrected the RGB data for overexposure in the centre of the images and underexposure at the edges of the images. The correction does not considerably change the main patterns in cyclicity. Please note that the main cyclicity in the core images is not an artefact of uneven lighting conditions but a true feature of the sediment cores. The resulting individual red, green, and blue records, were combined into a combined RGB record. High values in the combined RGB signal visually correspond to the lighter coloured strata, which at Site 699 are calcium carbonate dominated. At other South Atlantic and Pacific Ocean sites3,10,64, elevated levels of calcium carbonate (light-coloured sediments, RGB highs) correspond to eccentricity-paced productivity maxima that occur during an eccentricity minimum. The RGB and δ18Ocalcite records both exhibit the dominant 110-kyr cyclicity but are not always perfectly aligned. To better visually align the high-amplitude mid-Oligocene ~110-kyr cycles identified at Sites 699, 1218, and 1264, we made a small correction to the Site 1264 eccentricity-tuned age model by changing the tie point ‘293.52 adjusted revised metres composite depth (armcd) − 27.511 Ma' into ‘293.30 armcd − 27.511 Ma'. Both RGB and δ18Ocalcite were considered when selecting our final tuning tie points (Supplementary Table 1). This tuning approach was independently validated by (1) the convincing alignment (within error) between the magnetostratigraphic reversals from Site 699A with those of the Westerhold et al.3 astronomically calibrated time scales (Extended Data Fig. 3), (2) the identification of between five and six precession forced cycles in the RGB record for some of the best-expressed ~110-kyr cycle (that is, the combined 95- and 125-kyr components of eccentricity) and (3) the coherency in the benthic foraminiferal δ18O stratigraphy from Site 699A and independently astronomically age-calibrated δ18Ocalcite records from Walvis Ridge Site 1264, and equatorial Pacific Ocean Site 1218 (on the revised age model of Westerhold et al.3; Fig. Our fine-tuned astronomically calibrated age model for cores 699A-20H and 21H spans the 26.8- to 28.2-Ma interval and is based on 11 eccentricity based (La2011_ecc3L solution57) tie points and three magnetostratigraphic tie point (Extended Data Fig. We note that the results of this study are independent from the adopted tuning approach as we used δ18Ocalcite from Site 699 and Westerhold et al.3 to guide our sampling strategy and obtain temperature data for the targeted δ18Ocalcite cycles. The samples from cores 699A-20H and 21H are clay-rich and unlithified with generally well-preserved benthic foraminifera (Extended Data Figs. Benthic foraminiferal abundance fluctuates substantially through the studied interval, with several intervals characterized by very low abundances. Levels of low foraminiferal abundance are at 182.6–182.8 and 192.1–192.6 meters below sea floor (mbsf). Benthic foraminiferal abundance can be lower owing to an increase in sedimentation rates, a drop in paleoenvironmental oxygen conditions or carbonate dissolution65,66. Based only on benthic foraminifera abundance variability in the samples, some degree of dissolution at discrete levels in the record cannot be ruled out. However, the generally good foraminiferal preservation through the record does not suggest major dissolution, as supported by scanning electron microscope images of specimens (Extended Data Figs. 5–7) taken from light (Extended Data Figs. 7) sediment intervals (Extended Data Fig. The samples were washed over a 63-μm mesh-size sieve with tap water and oven-dried at 40 °C. A total of 99 samples were picked for clumped isotope analysis from 179.9 to 195.45 mbsf. Benthic foraminifera were picked from the size fraction >150 μm after dry sieving. Benthic foraminifera were grouped according to taxonomy and ecology depending on species and genus abundance as follows: (1) Cibicidoides spp. (composed of C. mundulus, C. eocaenus, C. havanensis, C. grimsdalei, C. micrus, C. brady, C. lamontdoherty and C. dickersoni); (2) Oridorsalis umbonatus; (3) Epifaunals (composed of Laticarinina pauperata, Gyroidinoides gyrardanus, G. planulatus, G. depressus, Nuttallides umbonifera, Alabamina weddellensis, Epistominella exigua, Anomalinoides rubiginosus and A. spissiformis); (4) Nodosarids (Nodosaria spp., Lenticulina spp., Lagenidae and Polymorphinids), Pleurostomellids (Pleurostomella spp.) and Stilostomellids (Stilostomella spp. ); and (5) Infaunals (Pullenia bulloides, P. quinqueloba, Melonis barleanuum, Nonion havanense, Nonionella spp. For several samples, this distinction into different species groups was not possible owing to extremely low foraminifera abundance, requiring all found specimens to be combined for measurements (samples 699A-20H-2, 20–22 cm to 699A-20H-2, 42–44 cm; samples 699A-20H-3, 5–7 cm to 699A-20H-3, 27–29 cm; samples 699A-21H-3, 10.5–13 cm to 699A-21-3. Foraminifera were cracked between glass plates and sonicated in deionized water (3 × 10–20 s) and methanol (1 × 5 s). At the end of the cleaning procedure, the test fragments were rinsed until the solute was no longer cloudy and oven-dried at 50 °C. Individual measurements from multiple adjacent samples were combined to calculate average Δ47 values, covering the minimum and maximum depth ranges of 10 and 30 cm, respectively, except between 182.21–182.67 mbsf (46 cm) and 186.45–187.05 mbsf (60 cm) where several samples were barren of foraminifera. The clumped isotope paleothermometer relies on the thermodynamic bounding of 13C and 18O isotopes in calcite molecules as a function of ambient temperature and is unaffected by the δ18Osw (ref. The influence of non-thermal controls on Δ47, such as pH and the biological partitioning of isotopes in calcite, does not appear resolvable67,68,69. The low natural abundance of 13C–18O bonds within carbonate ions demands large sample sizes to produce data with the precision required for palaeoclimate applications. Here, we used small (~85 μg) carbonate samples and obtained the necessary precision by averaging a mean of 33 Δ47 measurement values (minimum 20 to maximum 46) from neighbouring samples29,70. The Δ47 measurements were performed at the Farlab, University of Bergen, on two Thermo Scientific MAT 253 Plus isotope ratio mass spectrometer connected to Thermo Scientific Kiel IV carbonate preparation devices. The analytical method used here is extensively described in Modestou et al.30 and Leutert et al.71. We used three carbonate standards (ETH 1, 2, 3), which differ in bulk isotopic composition and ordering state to correct for Δ47 scale compression and to transfer results to the Intercarb-Carbon Dioxide Equilibrium Scale (I-CDES)72. An additional standard (International Atomic Energy Agency (IAEA)-C272) was not used for corrections but instead used to check the fidelity of the correction procedure (0.639 ± 0.027‰ (1 s.d. All analytical sessions (~23 h each) included approximately equal numbers of carbonate standards and samples. in Δ47 of the four carbonate standards after correction were between 0.027 and 0.028‰. Oxygen and carbon stable isotopes were obtained from the same groups/specimens as a co-product of Δ47 measurements. Carbonate δ18O and δ13C values are reported relative to the Vienna Pee Dee Belemnite scale (VPDB) and were corrected with the same carbonate standards (ETH 1–3), using the values reported by Bernasconi et al.73, including a scale correction. The δ18O and δ13C values of all the standards have external reproducibilities (1 s.d.) better than or equal to 0.07‰ (δ18O) and 0.03‰ (δ13C). To complement the stable isotope record obtained as a co-product of Δ47 measurements, stable isotopes were additionally measured on several monospecific samples of Cibicidoides (C. munduls/C. eocaenus) for the depth interval 193.6–194.1 mbsf and 194.7–195.0 mbsf at 10-cm resolution on a Thermo Scientific MAT 253 isotope ratio mass spectrometer connected to a Thermo Scientific Kiel IV carbonate preparation device at Farlab, University of Bergen. Carrara Marble (in-house CM12) was used as a working standard, and the values are reported relative to VPDB, calibrated using National Bureau of Standards (NBS) standards 18 and 19. External reproducibility in CM12 was better than or equal to 0.02‰ (δ13C) and 0.05‰ (δ18O) (1 s.d.) over the analysis interval. Owing to the large number of ‘replicate' measurements required, Δ47 values from multiple 10-cm spaced samples were averaged for each data point, using the δ18Ocalcite record as a guide to avoid aliasing and to obtain temperature data for the targeted δ18Ocalcite cycles. On average, each temperature group combines measurements from three samples (from minimum two to maximum five), and a total of 32 temperature groups were obtained with this approach, equivalent to one temperature data point every 28 kyr. Sample standard errors were determined by selecting the higher value between the sample standard deviation and the external reproducibility from IAEA-C2 (0.027‰) and calculating standard errors of the means depending on the number of measurements from each sample. Average Δ47 values for each group were determined as the average of all sample averages in the temperature group weighted by the number of measurements from the respective samples, with errors being estimated through the propagation of the standard errors of the samples. For each temperature group, a mean age was assigned by also averaging the age of each sample in the temperature group by the number of measurements. Temperatures were then estimated with the calibration of Meinicke et al.25 updated to the I-CDES scale74: Errors in temperature estimates (for example, propagated analytical and calibration errors) were determined through Monte Carlo simulations after Meckler et al.37. The final temperature errors are reported as 68% and 95% confidence intervals (Fig. To test the impact of the choice of calibration equation, we also calculated temperatures with the calibrations of Anderson et al.45 and Daëron and Vermeesch46,75 (Supplementary Table 2), which include both biogenic and inorganic (for example, natural and laboratory precipitate) samples. Temperatures calculated with these equations are on average 1.5 °C (ref. 46) colder, respectively, than those obtained with equation (1), and the δ18Osw on average 0.35‰ and 0.28‰ more negative. For this study, we chose to report temperatures with equation (1) because the other two calibrations include very high temperature (>100 °C) samples that can bias the relationship within the ocean temperature range. Although the choice of calibration affects our absolute reconstructed temperature and δ18Osw (albeit within error of our estimates), the relative changes, which are the main focus of this study, are unaffected. We refrain from using the calibration of Daëron and Gray76 as it yields unrealistically cold temperatures when applied to the Plio-Pleistocene section of the Cenozoic record of Meckler et al.37. In addition to the grouping (binning) approach to reconstruct temperature, we used a Gaussian moving window filter to independently confirm the validity of our temperature grouping. This verification process was carried out through a Monte Carlo simulation following the methodology outlined in Rodríguez-Sanz et al.70. In short, for each replicate analysis, we generated 10,000 random values on the basis of the observed external reproducibility of 0.027‰ from IAEA-C2 (Extended Data Fig. 8a), and assuming a normal distribution. To account for both analytical and calibration uncertainties, we computed 10,000 temperature estimates for each replicate using a random slope–intercept pair from the clumped isotope calibration equation (outlined below). Subsequently, we applied a Gaussian filter with a 110,000-year window to each simulation and calculated average temperatures every 25,000 years. The final temperature values are presented as the median (50th percentile), along with the associated uncertainties represented by the 95% and 68% confidence limits (Extended Data Fig. The filter generated a pattern remarkably similar to the temperature groups, indicating that our chosen binning approach did not introduce bias into the temperature reconstruction (Extended Data Fig. Despite the similarity between the temperature patterns obtained with the binning and Gaussian filter approaches, we prefer the binning approach, as it enables variable resolution and minimal smoothing in places where data density is higher (such as between 27.55 and 27.4 Ma). To calculate δ18Osw, we used δ18Ocalcite from the Δ47 analysis of Cibicidoides spp. for most of the samples that compose our temperature groups (86 of 106 samples). However, we also had to rely on other taxa when Cibicidoides spp. were absent or not sufficiently abundant for Δ47 analysis. Whereas Δ47 is not measurably impacted by using different benthic foraminifera taxa77, there are species-specific offsets in δ18Ocalcite (for example, refs. 78,79) that need to be corrected for. Our Site 699 dataset shows distinct offsets between the δ18Ocalcite of Cibicidoides spp. and the other taxonomic groupings used as Δ47 aliquots from the same samples (Extended Data Fig. Averaged across all samples with dual measurements, aliquots of mixed epifaunal benthic foraminifera species were offset in δ18Ocalcite by −0.13‰ (±0.05, 95% confidence intervals), O. umbonatus by −0.22‰ (±0.04, 95% confidence intervals) and mixed samples of all benthic foraminiferal species by −0.15‰ (±0.09, 95% confidence intervals) with respect to Cibicidoides spp. We used these mean offsets to correct δ18Ocalcite with respect to Cibicidoides spp. when Cibicidoides spp. was not available and used the corrected δ18Ocalcite value from the other species/groups with the following order of priority where multiple options were available: (1) mixed epifaunal benthic foraminifera (8/106 samples), (2) O. umbonatus (1/106) and (3) mixed benthic foraminifera, in case of very low abundance of specimens (11/106) (Extended Data Fig. We preferred the mixed epifaunal group over O. umbonatus, as the former consistently display the smallest offset from Cibicidoides spp. It is also worth noting that the offset correction ultimately has limited impact on the Site 699 δ18Ocalcite record and, hence, δ18Osw, owing to the relatively small number of samples lacking Cibicidoides spp. and the small magnitude of the δ18Ocalcite offsets that we corrected for, relative to the changes observed in the δ18Ocalcite record (Extended Data Fig. This notion is corroborated by the similar magnitude of δ18Ocalcite fluctuations at Site 699 and at Sites 1264 (South Atlantic) and 1218 (Equatorial Pacific) where Cibicidoides mundulus and Cibicidoides spp. were used, respectively, with a mean difference of only 0.18‰. To obtain δ18Osw, we calculated the average δ18Ocalcite for each group using the average δ18Ocalcite value for each sample in a temperature group and weighing these in the identical way to the Δ47 data (that is, giving more weight to samples that yielded more measurements for Δ47). This way, any skewing of the group averages towards a given sample owing to an unequal number of Δ47 measurements is also reflected in the δ18Ocalcite and, as a result, δ18Osw. The temperature and the average δ18Ocalcite for each group were combined to obtain δ18Osw following the equation of Marchitto et al.20 where BWT is bottom water temperature. We tested the statistical significance of the temperature and δ18Osw changes associated with the six 110-kyr eccentricity cycles between 28.2 and 27.2 Ma using mean values of temperature and δ18Osw groups (Extended Data Fig. The temperatures and δ18Osw coinciding with the cycle core ~40 kyr of peak eccentricity values were grouped in the ‘eccentricity maxima' group of values and tested against background values (Extended Data Fig. In doing so, we took into account a slight (±5 kyr) misalignment between the data and the eccentricity cycles. The test was performed with a t-test using the R stats package R Core Team80 (Extended Data Fig. For temperature, the t-test returned a P of 0.011, meaning the mean of the groups are statistically different. For δ18Osw, the t-test returned a P value of 0.256, meaning the mean of the groups are not statistically different (Extended Data Fig. All data are available in Supplementary Table 2 and in the EarthChem data repository at https://doi.org/10.60520/IEDA/114211 (accessed 31 October 2025). Toward a Cenozoic history of atmospheric CO2. Zachos, J., Pagani, M., Sloan, L., Thomas, E. & Billups, K. Trends, rhythms, and aberrations in global climate 65 Ma to present. Westerhold, T. et al. An astronomically dated record of Earth's climate and its predictability over the last 66 million years. & Kennett, J. P. Initial Reports of the Deep-Sea Drilling Project Vol. 29 (eds Kennett, J. P., Houtz, R. E. et al.) 743–755 (Washington, US Government Printing Office, 1975). Pearson, P. N. in Paleontological Society Papers Vol. 18 (eds Ivany, L. & Huber, B.) The emergence of modern sea ice cover in the Arctic Ocean. Galeotti, S. et al. Antarctic Ice Sheet variability across the Eocene–Oligocene boundary climate transition. Olivetti, V. et al. Ice volume variations and provenance trends in the Oligocene-early Miocene glaciomarine sediments of the Central Ross Sea, Antarctica (DSDP site 270). Liebrand, D. et al. Cyclostratigraphy and eccentricity tuning of the early Oligocene through early Miocene (30.1–17.1 Ma): Cibicides mundulus stable oxygen and carbon isotope records from Walvis Ridge Site 1264. Liebrand, D. et al. Evolution of the early Antarctic ice ages. Brzelinski, S. et al. Large obliquity-paced Antarctic ice-volume fluctuations suggest melting by atmospheric and ocean warming during late Oligocene. Rohling, E. J. et al. Sea level and deep-sea temperature reconstructions suggest quasi-stable states and critical transitions over the past 40 million years. Global mean and relative sea-level changes over the past 66 Myr: implications for early Eocene ice sheets. Lisiecki, L. E. & Raymo, M. E. A Pliocene–Pleistocene stack of 57 globally distributed benthic δ18O records. Gulick, S. P. S. et al. Initiation and long-term instability of the East Antarctic Ice Sheet. Paxman, G. J. G., Gasson, E. G. W., Jamieson, S. S. R., Bentley, M. J. & Ferraccioli, F. Long-term increase in Antarctic Ice Sheet vulnerability driven by bed topography evolution. The Antarctic ice-sheet: a Paleoclimate modeling perspective. Halberstadt, A. R. W. et al. CO2 and tectonic controls on Antarctic climate and ice-sheet evolution in the mid-Miocene. Improved oxygen isotope temperature calibrations for cosmopolitan benthic foraminifera. Shackleton, N. Oxygen isotope analyses and Pleistocene temperatures re-assessed. Miller, K. G., Fairbanks, R. G. & Mountain, G. S. Tertiary oxygen isotope synthesis, sea level history, and continental margin erosion. Eiler, J. M. ‘Clumped-isotope' geochemistry—the study of naturally-occurring, multiply-substituted isotopologues. Evans, D. & Müller, W. Deep time foraminifera Mg/Ca paleothermometry: nonlinear correction for secular change in seawater Mg/Ca. Meinicke, N. et al. A robust calibration of the clumped isotopes to temperature relationship for foraminifers. Updated calibration of the clumped isotope thermometer in planktonic and benthic foraminifera. Evans, D. et al. Eocene greenhouse climate revealed by coupled clumped isotope-Mg/Ca thermometry. Meckler, A. N., Ziegler, M., Millan, M. I., Breitenbach, S. F. M. & Bernasconi, S. M. Long-term performance of the Kiel carbonate device with a new correction scheme for clumped isotope measurements. A. et al. Carbonate clumped isotope analyses with the long-integration dual-inlet (LIDI) workflow: scratching at the lower sample weight boundaries. Modestou, S. E., Leutert, T. J., Fernandez, A., Lear, C. H. & Meckler, A. N. Warm middle Miocene Indian Ocean bottom water temperatures: comparison of clumped isotope and Mg/Ca-based estimates. Agterhuis, T., Ziegler, M., de Winter, N. J. Warm deep-sea temperatures across Eocene Thermal Maximum 2 from clumped isotope thermometry. Taylor, V. E., Wilson, P. A., Bohaty, S. M. & Meckler, A. N. Transient deep ocean cooling in the eastern equatorial Pacific Ocean at the Eocene–Oligocene Transition. Ciesielski, P. R. et al. Site 699, Shipboard Scientific Party. Ocean Drilling Program, Initial Reports Vol 114 (eds Ciesielski, P. R. et al.) 151–201 (Ocean Drilling Program, 1988). Locarnini, R. A. et al. in World Ocean Atlas 2013 Vol. Cramer, B. S., Miller, K. G., Barrett, P. J. & Wright, J. D. Late Cretaceous–Neogene trends in deep ocean temperature and continental ice volume: reconciling records of benthic foraminiferal geochemistry (d18O and Mg/Ca) with sea level history. Lear, C. H., Elderfield, H. & Wilson, P. A. Cenozoic deep-sea temperatures and global ice volumes from Mg/Ca in benthic foraminiferal calcite. Meckler, A. N. et al. Cenozoic evolution of deep ocean temperature from clumped isotope thermometry. The enigma of Oligocene climate and global surface temperature evolution. Hoem, F. S. et al. Temperate Oligocene surface ocean conditions offshore of Cape Adare, Ross Sea, Antarctica. Duncan, B. et al. Climatic and tectonic drivers of late Oligocene Antarctic ice volume. Hou, S. et al. Reconciling Southern Ocean fronts equatorward migration with minor Antarctic ice volume change during Miocene cooling. Via, R. K. & Thomas, D. J. Evolution of Atlantic thermohaline circulation: early Oligocene onset of deep-water production in the North Atlantic. & Via, R. K. Neogene evolution of Atlantic thermohaline circulation: perspective from Walvis Ridge, southeastern Atlantic Ocean. Braaten, A. H. et al. Limited exchange between the deep Pacific and Atlantic oceans during the warm mid-Pliocene and marine isotope Stage M2 ‘glaciation'. Anderson, N. T. et al. A unified clumped isotope thermometer calibration (0.5–1,100 °C) using carbonate-based standardization. Daëron, M. & Vermeesch, P. Omnivariant generalized least squares regression: theory, geochronological applications, and making the case for reconciled Δ47 calibrations. Epstein, S., Buchsbaum, R., Lowenstam, H. A. & Urey, H. C. Revised carbonate-water isotopic temperature scale. & Schulz, M. Simulating the sea level imprint on marine oxygen isotope records during the middle Miocene using an ice sheet-climate model. Levy, R. et al. Antarctic ice sheet sensitivity to atmospheric CO2 variations in the early to mid-Miocene. Gasson, E., De Conto, R. M., Pollard, D. & Levy, R. H. Dynamic Antarctic ice sheet during the early to mid-Miocene. Pollard, D. & De Conto, R. Hysteresis in Cenozoic Antarctic ice-sheet variations. De Conto, R. & Pollard, D. Contribution of Antarctica to past and future sea-level rise. The multi-millennial Antarctic commitment to future sea-level rise. Oppenheimer, M. in IPCC Special Report on the Ocean and Cryosphere in a Changing Climate (eds Pörtner, H.-O. et al.) 321–445 (IPCC, Cambridge Univ. Garbe, J., Albrecht, T., Levermann, A., Donges, J. F. & Winkelmann, R. The hysteresis of the Antarctic Ice Sheet. Bradshaw, C. D. et al. Hydrological impact of Middle Miocene Antarctic ice-free areas coupled to deep ocean temperatures. Laskar, J., Gastineau, M., Delisle, J. Strong chaos induced by close encounters with Ceres and Vesta. & Clement, B. M. Scientific results, subantarctic South Atlantic. Ocean Drilling Program, Initial Reports Vol. 114 (eds Ciesielski, P. R. et al.) 337–357 (Ocean Drilling Program, 1991). Schlitzer, R. Ocean Data View (2021); https://odv.awi.de Talley, L. D., Pickard, G. L., Emery, W. J. & Swift, J. H. in Descriptive Physical Oceanography: An Introduction 555 (Elsevier, 2011). Van Hinsbergen, D. J. et al. A paleolatitude calculator for paleoclimate studies. Plancq, J., Mattioli, E., Pittet, B., Simon, L. & Gross, V. Productivity and sea-surface temperature changes recorded during the late Eocene–early Oligocene at DSDP Site 511 (South Atlantic). Harwood, D. M. & Maruyama T. Central Kerguelen Plateau. Ocean Drilling Program, Scientific Results Vol. 120 (eds Wise, S. W. et al.) 683–733 (Ocean Drilling Program, 1992). Westerhold, T. et al. Orbitally tuned timescale and astronomical forcing in the middle Eocene to early Oligocene. Jorissen, F. J., Fontanier, C. & Thomas, E. in Developments in Marine Geology: Proxies in Late Cenozoic Paleoceanography Vol. 1 (eds Hillaire-Marcel, C. & de Vernal, A.) The middle Eocene climatic optimum (MECO): a multiproxy record of paleoceanographic changes in the southeast Atlantic (ODP Site 1263, Walvis Ridge). Tang, J., Dietzel, M., Fernandez, A., Tripati, A. K. & Rosenheim, B. E. Evaluation of kinetic effects on clumped isotope fractionation (D47) during inorganic calcite precipitation. Tripati, A. K. et al. Beyond temperature: clumped isotope signatures in dissolved inorganic carbon species and the influence of solution chemistry on carbonate mineral composition. Peral, M. et al. On the combination of the planktonic foraminiferal Mg/Ca, clumped (Δ47) and conventional (δ18O) stable isotope paleothermometers in palaeoceanographic studies. Rodríguez-Sanz, L. et al. Penultimate deglacial warming across the Mediterranean Sea revealed by clumped isotopes in foraminifera. Leutert, T. J., Auderset, A., Martínez-García, A., Modestou, S. & Meckler, A. N. Coupled Southern Ocean cooling and Antarctic ice sheet expansion during the middle Miocene. Bernasconi, S. et al. InterCarb: a community effort to improve interlaboratory standardization of the carbonate clumped isotope thermometer using carbonate standards. Bernasconi, S. M. et al. Reducing uncertainties in carbonate clumped isotope analysis through consistent carbonate-based standardization. Meinicke, N., Reimi, M. A., Ravelo, A. C. & Meckler, A. N. Coupled Mg/Ca and clumped isotope measurements indicate lack of substantial mixed layer cooling in the Western Pacific Warm Pool during the last ∼5 million years. Boscolo-Galazzo, F. et al. Oligocene deep ocean oxygen isotope variations primarily driven by temperature, Version 1.0. Interdisciplinary Earth Data Alliance (IEDA). https://doi.org/10.60520/IEDA/114211 (accessed 31 October 2025). Daëron, M. & Gray, W. R. Revisiting oxygen-18 and clumped isotopes in planktic and benthic foraminifera. Application of clumped isotope thermometry to benthic foraminifera. Duplessy, J. C., Lalou, C. & Vinot, A. C. Differential isotopic fractionation in benthic foraminifera and paleotemperatures reassessed. Shackleton, N. J. Attainment of isotopic equilibrium between ocean water and the benthonic foraminifera genus Uvigerina: isotopic changes in the ocean during the last glacial. Internationaux du CNRS 219, 203–209 (1974). R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, 2022); https://www.r-project.org/ Speijer, R. P., Pälike, H., Hollis, C. J., Hooker, J. J. & Ogg, J. G. in Geologic Time Scale 2020 Ch. Welch, B. L. The generalisation of student's problems when several different population variances are involved. This research used samples and data from the International Ocean Discovery Program (IODP) and the ODP. This research was supported through the European Research Council (starting grant no. ), the Norwegian Research Council (grant no. ), the Natural Environment Research Council (NERC) grant no. NE/T008512/1 to participate in IODP Expedition 378 (F.B.G. ), the Horizon 2020 Framework Programme (grant no. thanks M. Kucera for helpful discussions. Open access funding provided by Staats- und Universitätsbibliothek Bremen. MARUM, Center for Marine Environmental Sciences, University of Bremen, Bremen, Germany Department of Earth Science and Bjerknes Centre for Climate Research, University of Bergen, Bergen, Norway Victoria E. Taylor, Eirik V. Galaasen & A. Nele Meckler Department of Earth and Environmental Sciences, University of Manchester, Manchester, UK Earth Sciences Department, University College London, London, UK Department of Earth and Environmental Sciences, University of Exeter, Cornwall Campus, Exeter, UK Department of Earth Sciences, University of Milan, Milan, Italy Instituto Andaluz de Ciencias de la Tierra (IACT-CSIC), Armilla, Spain Institute of Marine and Environmental Sciences, University of Szczecin, Szczecin, Poland Institute of Earth Sciences, Heidelberg University, Heidelberg, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Project administration and funding acquisition: A.N.M. Correspondence to Flavia Boscolo-Galazzo. The authors declare no competing interests. Nature Geoscience thanks Michael Henehan, Simone Moretti and John Schmelz and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Thomas Richardson and James Super, in collaboration with the Nature Geoscience team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. North-South cross-sections at 30°W showing the modern location of Site 699 with present-day temperature and salinity. Contours for temperature and salinity highlight the two main deep-water masses characterizing the area: CDW (Circumpolar Deep water) and AABW (Antarctic Bottom Water). Figure generated using https://odv.awi.de ref. (a) Hole 699 A Cores 19H to 21H were visually inspected for the presence of sedimentary cycles. Core color images show distinct cyclicity in the depth domain. (b) Magnetostratigraphy from Hole 699A58, where black (white) indicates a normal (reversed) magnetic polarity. Grey corresponds to coring gaps. Red error bars indicate uncertainty intervals of the exact magnetic reversal positions in the depth domain. Specifically, to each chron-reversal is assigned a mean-depth value indicated with a horizontal red bar. Mean-depth values are calculated based on the uppermost and lowermost magnetic chron positions, as reported by ref. 58, also used to calculate the error bars in the figure. The spread from the mean of the upper and lower depth uncertainties (error bars) ranges from ±0.1 to 1.5 meters. Note that the reversal from Chron C10r to C10n falls in the core gap in between Core 22H and Core 21H, and this reversal comes with a greater uncertainty. The study interval spans Cores 699A-20H and 21H, and Subchrons C9n to C10n. (c) Uncorrected red (R), green (G), blue (B), (RGB), data extracted from the core images (a). (d) RGB data corrected for uneven lighting conditions of core photographs. (e) Benthic foraminiferal δ18O data from Site 699. Arrows indicate the stratigraphic depths of foraminiferal samples used for SEM images shown in Extended Data Figs. Our initial age model consisted of assigning the Westerhold et al.3 reversal ages to Site 699 magnetostratigraphic chron boundaries (ref. Black error bars on the y-axis indicate the upper, mean, and lower depth position for each reversal boundary (see Extended Data Fig. 2 caption for an explanation). Black error bars on the x-axis indicate how the uncertainty in the depth-domain (uppermost, mean, and lowermost positions) translates into an age uncertainty. Note that upper and lower depth uncertainties with respect to the mean are equal (see y-axis), but that younger and older age uncertainties with respect to the mean vary from one another (see x-axis). This is resulting from changing depth-age relationships, that is, varying sedimentation rates, throughout our study interval. In the age domain the spread from the mean of the younger and older age uncertainties ranges from -0.009 and +0.005 Myr to -0.072 and +0.141 Myr. Subsequent alignment of the RGB record to the La2011_ecc3L57 eccentricity solution resulted in a more accurately constrained age control for Site 699. Our tuned reversal ages are within error in agreement with those of Westerhold et al.3, and apart from the C8r/C9n reversal, also with the GTS2020 ages81. (b) Site 699 sedimentation rates based on our eccentricity tuning. Sedimentation rates vary between ~1.1 and 2.7 cm kyr−1, values that are typical for a (hemi)pelagic deep marine site. Astronomical age calibration was guided by Site 699 (a) benthic foraminiferal δ18O record, but mainly based upon (b) the RGB record from Hole 699A Cores 20H and 21H. Tuning target was (c) the La2011_ecc3L57 eccentricity solution. Tuning tie points based on an alignment of RGB (and δ18O) to eccentricity are indicated with red vertical lines. Black vertical lines are tie points based on the (d) Site 699 magnetic polarity reversals58, which were assigned the (e) Westerhold et al.3 ages. Within error, (d) and (e) agree with one another. Chron boundary names are indicated. Sample 699A-20H-2, 100-105 cm (182.10–182.13 mbsf). Panels a, b, and c show whole fragments and the images on the right are zoomed in (see panels a, b, c for locations). Sample 699A-20H-3, 45-50 cm (183.05–183.08 mbsf). Panels a, b, and c show whole fragments and the images on the right are zoomed in (see panels a, b, c for locations). Sample 699A-21H-4, 50-53 cm (194.10–194.13 mbsf). Panels a, b, and c show whole fragments and the images on the right are zoomed in (see panels a, b, c for locations). (a) Replicate analyses (grey circles) and filter median values (blue line). (b) Filter median values (blue line) and 68% and 95% confidence intervals shown as dark blue and light blue envelopes. These were calculated with the simulation 16th – 84th and 2.5th – 97.5th percentiles, respectively. Red circles are the temperature estimates for the temperature groups expressed as mean values. Errors on temperature groups are the 68% and 95% confidence intervals derived from 20-46 measurements sourced from 2-5 individual sediment samples. (a) Cross-plot of δ18Ocalcite from Cibicidoides spp. versus other key taxonomic groupings used here (red: O. umbonatus; grey: aliquots of mixed epifaunal benthic foraminifera species; yellow: mix of all benthic foraminifera present) from individual samples where measurements of both Cibicidoides spp. and other were possible. (b) The averaged δ18Ocalcite offset from Cibicidoides spp. of the groupings in (a) for all samples containing dual measurements, with 95% confidence intervals (vertical bars) and number of samples containing measurements of a specific grouping as well as Cibicidoides spp. displayed at the bottom. (c) The composite Site 699 benthic foraminifera δ18Ocalcite record (black line) plotted versus age, with taxonomic groups contributing to the record shown as coloured circles (blue: Cibicidoides spp. ; red: O. umbonatus; grey: all epifaunal benthic foraminifera; yellow: all benthic foraminifera), constructed by correcting for the averaged δ18Ocalcite offsets (see (b)) of the other groupings to that of Cibicidoides spp. (d) Comparison of the composite benthic foraminifera δ18Ocalcite record with the offset-correction applied (blue line, grey circles) and without an offset-correction (black line, yellow circles). (a) Δ47 temperatures plotted against the La2011_ecc3L Eccentricity Index from57 for the six 110-kyr eccentricity cycles captured here. Yellow dots indicate temperatures included in the eccentricity maxima group for the purpose of testing statistical significance of peak vs background values. Temperature data are presented as mean values with horizontal stippled lines representing the 68% confidence intervals of the proxy data derived from 20-46 measurements sourced from 2-5 individual sediment samples. Vertical lines reflect the age ranges of the adjoining 2-5 samples combined for calculating average Δ47 values. (b) Result of the Welch's t-test82, a variation of the standard t-test that is more suitable for distributions with unequal variance, comparing the background temperature values with the ones at the eccentricity maxima. Each box-plot shows the mean (white star), median (green line), 25-75% range (boxes) and full range of values (whiskers) of the two groups. The numerical result of the Welch's t-test is shown in upper-left-hand part of the figure, whereby DF= degree of freedom; LC and HC = 95% lower and higher (respectively) confidence boundaries of the estimated likely true difference between the two groups. (c) Same as (a) but for δ18Osw data. Purple dots indicate values included in the eccentricity maxima group for the purpose of testing statistical significance of peak vs background values. (d) Same as (b) but for δ18Osw data. Raw dataset and all data presented and discussed in the text and Methods. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Boscolo-Galazzo, F., Taylor, V.E., Galaasen, E.V. et al. Oligocene deep ocean oxygen isotope variations primarily driven by temperature. Version of record: 07 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/microbubbles-help-spread-dangerous-microplastics-through-our-water-study/'>‘Microbubbles' Help Spread Dangerous Microplastics Through Our Water, Study Finds</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 11:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>‘Microbubbles' Help Spread Dangerous Microplastics Through Our Water, Study Finds Water plays a crucial role in how tiny pieces of plastic enter our environment—and us A researcher selects microplastics found in sea species at the Hellenic Center for Marine Research in Anávissos, Greece, near Athens, on July 15, 2025. Yet despite these traits, scientists don't fully understand how all the minuscule filaments of plastic get into our environment. A study published last month in Science Advances offers some new clues as to how water may be contributing to their spread. Scientists already knew that plastics degrade through exposure to sunlight and repeated weathering by waves, sand or other debris. But the new study suggests contact with water itself is also a factor: in both marine and river environments, researchers found that microbubbles can form on the surface of a piece of plastic, breaking it down—and releasing tiny, practically invisible plastic bits into the surrounding water. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. From there, nanoplastics and microplastics often enter the food chain—and, in turn, us. An estimated 130 million metric tons of plastic waste enters our bodies and the environment every year, with that number on track to more than double by 2040. “Plastic degradation is an invisible threat to the environment and human health,” said John Boland, a professor in the School of Chemistry at Trinity College Dublin and senior author of the study, in a statement. Jackie Flynn Mogensen is a breaking news reporter at Scientific American. Before joining SciAm, she was a science reporter at Mother Jones, where she received a National Academies Eric and Wendy Schmidt Award for Excellence in Science Communications in 2024. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41589-025-02104-x'>Membrane editing with proximity labeling reveals regulators of lipid homeostasis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-07 10:43:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Cellular lipid metabolism is subject to strong homeostatic regulation, but the players involved in and mechanisms underlying these pathways remain largely uncharacterized. Here we develop a ‘feeding–fishing' approach coupling membrane editing using optogenetic lipid-modifying enzymes (feeding) with organelle membrane proteomics through proximity labeling (fishing) to elucidate molecular players and pathways involved in the homeostasis of phosphatidic acid (PA), a multifunctional lipid central to glycerolipid metabolism. This approach identified several PA-metabolizing enzymes and lipid transfer proteins enriched in and depleted from PA-fed membranes. Mechanistic analysis revealed that PA homeostasis in the cytosolic leaflets of the plasma membrane and lysosomes is mediated by both local PA metabolism and the action of lipid transfer proteins that carry out interorganelle lipid transport before subsequent metabolism. More broadly, the interfacing of membrane editing to controllably modify membrane lipid composition with organelle membrane proteomics using proximity labeling represents a strategy for revealing mechanisms governing lipid homeostasis. Membranes are complex and dynamic assemblies that regulate myriad cellular processes. The lipid composition of membranes, crucial to membrane biophysical properties and the functions of the membrane-associated proteome, is determined by a balance of activities of lipid-metabolizing enzymes and lipid transfer proteins (LTPs). Efforts to decode these lipid–protein networks are complicated by the rapid diffusion, trafficking and, in many cases, short metabolic half-lives of lipids. A central player in lipid metabolism, PA is produced by at least four different biosynthetic routes and metabolized into phospholipids, triglycerides or fatty acids by three pathways2,3. Despite representing only 1% of the lipidome, PA is present in most organelle membranes, and its levels transiently rise downstream of signaling from numerous cell-surface receptors and soluble signaling enzymes4,5. Because such PA bursts induce potent signaling events, PA is subject to strong homeostatic regulation, with its levels tightly controlled in space and time by modulation of both biosynthetic flux and regulated interorganelle transport by LTPs4,5,6,7. Although several PA-metabolizing enzymes and PA-specific LTPs have been characterized, it is not well understood how cells sense changes to levels of PA to regulate the activities of these enzymes and transporters to ultimately restore PA levels and achieve homeostasis. Here, we develop and exploit a ‘feeding–fishing' strategy to map proteins involved in such homeostatic processes and identify mechanisms of how cells respond to changes in the lipid composition of their organelle membranes. In this approach, we combined membrane editing using an optogenetic superactive phospholipase D (superPLD) with proximity-dependent biotinylation using a membrane-tethered TurboID, followed by tandem mass tagging (TMT)-based mass spectrometry (MS) to enable protein identification and quantification. In this manner, feeding PA to a membrane and fishing out proteins associated with that membrane would identify proteins that accumulate on or are depleted from a target membrane in response to changes in PA levels within that membrane. Our studies reveal changes to the membrane recruitment of several PA-metabolizing enzymes and LTPs in PA-fed membranes that collectively facilitate removal of excess PA. Analysis of lipid-metabolizing enzymes points to roles for certain PA metabolic pathways in clearing PA pools from the plasma membrane and lysosomes. Mechanistic investigations using confocal microscopy to visualize PA pools and lipidomics to track changes to the phospholipidome revealed divergent roles for several LTPs recruited to PA-fed membranes in affecting PA metabolism. In particular, we found that Nir2, an LTP known to transport PA between the plasma membrane and ER, impacted PA homeostasis not only at the plasma membrane but also at other organelles, and SCP2, a broad-spectrum LTP implicated in sterol and fatty acid transport, can also mediate PA clearance. We additionally point to unexpected roles for members of the SMP and ORD domain-containing LTP families in PA homeostasis, including PDZD8, TEX2 and ORP1L. Collectively, membrane editing with superPLD coupled to proximity labeling enables identification of regulators of PA homeostasis, revealing cellular mechanisms underlying rapid PA metabolism and interorganelle transport. Membrane editing is an emerging strategy to precisely manipulate the lipid composition of organelle membranes within living cells to study the physiological functions of individual lipids8,9. We previously devoted substantial efforts toward development of membrane editors for PA. First, we designed an optogenetic PLD (optoPLD) to acutely produce PA on target organelle membranes10. PLDs produce PA through the hydrolysis of abundant phospholipids, and optoPLD uses a blue-light-inducible heterodimerization system to recruit PLD onto desired membranes. Moreover, the ability of PLDs to additionally catalyze transphosphatidylation with exogenously supplied primary alcohols to produce a variety of natural and unnatural phospholipids underscores the versatility of optoPLD for editing the phospholipidome11. To overcome the relatively modest activity of the first-generation optoPLD, we greatly improved its performance by using directed evolution in mammalian cells12. The resultant superPLDs, which acquired several mutations that optimized performance of this disulfide-containing, secreted protein from Streptomyces sp. PMF in the mammalian intracellular environment, exhibited up to 100-fold higher activity in cells compared to the original editor (wild-type optoPLD). SuperPLDs were demonstrated as potent membrane editors for spatiotemporally defined editing of phospholipids with organelle-level precision in live cells. Our feeding–fishing strategy to identify proteins associated with PA-fed membranes involves superPLD-enabled membrane editing coupled to proximity proteomics (Fig. Currently, there are two widely used types of proximity labeling enzymes with distinct labeling mechanisms: promiscuous biotin ligase (for example, TurboID13) and engineered peroxidase (for example, APEX2 (ref. Promiscuous biotin ligase releases a biotin-AMP intermediate that reacts with lysine residues on proximal proteins, whereas proximity labeling with engineered ascorbate peroxidase is mediated by biotin-phenoxyl radicals that react with tyrosine residues15. These enzymes are generally expressed in intracellular compartments, where the reactive intermediates are highly confined, and their cross-membrane diffusion is rare. a, Feeding–fishing strategy couples membrane editing and proximity labeling for elucidation of regulators of PA homeostasis on different organelle membranes. b, Comparison of APEX2 and TurboID for proximal organelle membrane proteomics, evaluating their ability to selectively label a model protein (optoPLD) associated with the same versus different membranes. APEX2 labeling was performed using a 30-min incubation with 500 μM biotin-phenol followed by 1 min with H2O2. TurboID labeling was performed using a 10-min incubation with 500 μM biotin. d,e, Immunofluorescence images of cells expressing APEX2 (d) or TurboID (e) targeted to plasma membrane or lysosomes. Shown are representative images from two independent experiments. f, Western blots of cells expressing TurboID targeted to either the plasma membrane (KRAS CAAX domain19), lysosomes (p18/LAMTOR1 lysosomal-targeting sequence20 or TMEM192 (ref. Labeling was performed by 10-min incubation with 500 μM biotin. The rightmost two lanes indicate cells without expressing TurboID treated with or without biotin. Shown are representative blots from two independent experiments. g, One-shot transduction strategy enables efficient and transient coexpression of the feeding–fishing components. h,i, Immunofluorescence images of cells coexpressing TurboID and superPLD targeted to the plasma membrane (h) or lysosomes (i; Lyso). Shown are representative images from three independent experiments. Because there are limited examples of tethering these enzymes to membranes to map membrane-proximal proteomes, we first explored the feasibility of using these two types of proximity labeling systems in this application, that is, to selectively label proteins that reside on the same membrane they are tagged to. We expressed APEX2 or TurboID fused to a plasma membrane targeting sequence, Lyn10, to localize the enzyme to the cytoplasmic leaflet of the plasma membrane. In the same cells, we coexpressed the wild-type optoPLD (from a bicistronic expression vector, CRY2–mCherry–PLD–P2A–CIBN–Tag where Tag is an organelle-targeting sequence) targeted to the cytosolic leaflet of either the plasma membrane or endoplasmic reticulum (ER) membrane. Surprisingly, plasma membrane-tagged APEX2 showed no difference in the labeling of optoPLD on the same (plasma membrane) versus different (ER) membrane, whereas TurboID preferentially labeled optoPLD on the same membrane (Fig. Consistent with this result, TurboID, when targeted to specific membranes, exhibited better colocalization with biotinylated proteins than APEX2 (Fig. From these results, we concluded that TurboID would be an optimal enzyme for mapping proteins associated with membranes of interest. Among various membranes that could be targeted by superPLD and TurboID, we selected the cytoplasmic leaflets of the plasma membrane and lysosomal membrane because of the proposed localization and physiological functions of mammalian PLD1/2, which can initiate PA-based signaling on these membranes16,17,18. We previously showed that CIBN fused to the CAAX domain of KRAS19 (CIBN–CAAX) and the lysosome-targeting sequence of p18/LAMTOR1 (ref. 20) (p18–CIBN) can mediate effective superPLD recruitment and consequent PA enrichment on plasma membrane and lysosomes, respectively12. Using the same sequences for targeting TurboID to these membranes, we confirmed that these membrane-tethered TurboIDs showed equally high and promiscuous labeling activity to the originally reported ER membrane-tethered TurboID13 (Fig. The feeding–fishing strategy requires coexpression of both TurboID and superPLD in cells at high efficiency. To avoid potential long-term effects of chronic expression, we performed transient expression by one-shot, lentivirus-based transduction. Although optoPLD or superPLD is typically expressed using a P2A self-cleavable peptide to ensure equimolar expression of CRY2–mCherry–PLD and CIBN–Tag, we found that for this application, separating these two components enabled more efficient transduction because of the limited size of DNA inserts that each lentivirus can encapsulate21. Therefore, our optimized protocol involved a triple lentivirus transduction system with one lentivirus each harboring TurboID–Tag, CRY2–mCherry–PLD or CIBN–Tag (Fig. To facilitate the incorporation of lentivirus into cells, we used spinfection, which is a method to apply mild centrifugation force during transduction22. The ratio of three lentivirus strains was optimized, on the basis of their titers, to achieve the highest coexpression efficiency while keeping cell viability unaffected. With these modifications, we achieved high efficiency (>75%) for transient cotransduction of HEK 293T cells with TurboID and superPLD targeted to either the plasma membrane or lysosomes (Fig. We next performed proximity proteomics to identify the effects of local PA production on the proteomes of different membranes. We treated HEK 293T cells coexpressing TurboID and superPLD, targeted to either the plasma membrane or lysosomes, for 30 min with intermittent blue light (470 nm, 5 s per 1 min) to activate superPLD, followed by a 3-min labeling with 150 μM biotin (Fig. As a negative control, we used cells transduced with TurboID and a catalytically dead superPLD (H167A;H440A, deadPLD) that does not produce PA. Catalytically dead control samples were subjected to the same 470-nm blue-light irradiation protocol to account for any potential membrane environmental changes because of foreign protein recruitment or light-induced side effects23. Biotinylated proteins from two conditions (superPLD versus deadPLD) and three replicates were combined for TMT 16-plex proteomics analysis to identify proteins that were selectively enriched on or depleted from PA-fed membranes (Fig. a, Schematic of feeding–fishing proximity proteomics workflow. Biotinylated proteins from HEK 293T cells coexpressing TurboID and either superPLD or deadPLD targeted to the same membrane (either plasma membrane or lysosomes) were enriched by streptavidin–agarose pulldown, eluted from resin and subjected to TMT labeling and multiplexed proteomics. b,c, Volcano plots showing differential enrichment of proteins on PA-fed versus unfed plasma membranes (b) or lysosomes (c). Proteins with known functions in lipid transport and lipid metabolism are colored in magenta and green, respectively, and proteins that were further functionally characterized in this study are indicated in italic. d, Table summarizing proteins that showed significance (abundance ratio P < 0.05) in enrichment or depletion on PA-fed versus unfed plasma membranes or lysosomes. Protein hits are ordered by fold enrichment and the hits with particularly high significance (abundance ratio P < 0.001) are shown in bold. Two-sided Student's t-tests were used for P-value calculation of the reported ratios. In these feeding–fishing proteomics experiments, we detected 4,401 and 4,260 biotinylated proteins from the plasma membrane and lysosomes, respectively, with high correlations between replicates (Fig. To identify proteins that were enriched on or depleted from PA-fed membranes, we calculated protein enrichment in the superPLD samples relative to deadPLD controls and plotted abundance ratios and associated P values (Fig. Among the hits that showed a significant enrichment on PA-fed membranes (based on the abundance ratio P value) were two known PA-metabolizing enzymes, Lipins 1 and 2 (LPIN1/2), which degrade PA into diacylglycerol (DAG). Correspondingly, DAG kinase δ (DGKD), which produces PA from DAG, was found to be depleted from PA-fed membranes. Furthermore, the best-characterized mammalian PA transfer protein, Nir2 (PITPNM1), was also enriched on both plasma membranes and lysosomes upon PA production, supporting the validity of the feeding–fishing proteomics. Western blot and immunofluorescence analysis of representative target proteins revealed that the enrichment was because of differences in their labeling and localization patterns rather than their global expression levels (Extended Data Fig. For the plasma membrane feeding–fishing studies, 42 proteins associated with these terms were enriched on this membrane and 18 were depleted; similarly, for lysosomal feeding–fishing experiments, 40 such proteins were enriched and 25 were depleted from lysosomal membranes (Fig. Beyond proteins related to lipid transport and metabolism, interactome analysis using the STRING network revealed additional insights, including notable changes in mitochondrial proteins, which were enriched on PA-fed plasma membranes and depleted from lysosome-depleted membranes (Extended Data Fig. An examination of mitochondrial morphology found alterations in cells expressing superPLD, where mitochondria appeared as round, fragmented structures (Extended Data Fig. Interestingly, a similar phenotype was recently reported upon induction of acute DAG synthesis at the outer mitochondrial membrane, whose lipid composition was found to regulate membrane-shaping proteins that alter mitochondrial morphology24. Consistent with this finding, our data showed that the tendency and kinetics of mitochondrial fragmentation were strongly influenced by the site of PA production, with PA generated on membranes more closely associated with mitochondria (for example, ER) being more effective than PA produced on more distal membranes (for example, plasma membrane) (Extended Data Fig. Nir2 is a well-characterized PA transfer protein that translocates to ER–plasma membrane contact sites upon PA production (for example, following PLC activation) and facilitates PA transfer from the plasma membrane to the ER6. Because our feeding–fishing proteomics detected Nir2 to be enriched not only at the plasma membrane but also on lysosomes upon PA feeding, we overexpressed an miRFP-tagged Nir2 construct to monitor its localization in mammalian cells. miRFP–Nir2 exhibited mostly cytosolic but weakly ER-associated localization under basal conditions, in agreement with a previous study using GFP–Nir2 (ref. Upon light-induced superPLD recruitment to the plasma membrane or to lysosomes, miRFP–Nir2 relocalized to these membranes, strongly colocalizing with superPLD (Fig. 3a,b and Extended Data Fig. a,b, Confocal images of cells coexpressing miRFP–Nir2 and either superPLD or deadPLD targeted to the plasma membrane (a) or lysosomes (b), whose recruitment was induced by intermittent blue-light illumination (470 nm, 5 s per 1 min). c, Schematic of approach to visualize PA localization with GFP–Spo20 upon PA feeding by superPLD in the presence or absence of forced expression of LTPs. d,e, Confocal images of cells coexpressing a PA-binding probe GFP–Spo20 and either PM-superPLD (d) or lyso-superPLD (e), without or with stable expression of V5–Nir2. Each graph shows the colocalization time course in the cells with empty vector (black), V5–Nir2 (red) or a mutant form of V5–Nir2 (T59A) deficient in lipid transfer (blue). Statistical analysis was performed using a two-sided repeated-measures ANOVA (n = 34 cells, **P = 0.0032 for Nir2 and P = 0.1949 for T59A (f); n = 20 cells, **P = 0.0315 for Nir2 and P = 0.2021 for T59A (g)). Three independent experiments were performed with similar results. h,i, Confocal images of cells coexpressing the PA-binding probe GFP–Spo20 and either PM-superPLD (h) or lyso-superPLD (i), without or with expression of proSCP2. j,k, Similar plots to f and g using data from h and i. Each graph shows the colocalization time course in control cells (black) or cells stably expressing V5–Nir2 (red), SCP2 (blue) or proSCP2 (yellow). Note that the V5 tag was omitted from SCP2 and proSCP2 to avoid potential interference with their post-translational cleavage. Three independent experiments were performed with similar results. We then tested whether Nir2 could exert its PA transfer function on these PA-fed membranes. To minimize any potential interference on its function caused by the fluorescent protein fusion, we replaced miRFP with a V5 tag (GKPIPNPLLGLDST) and used lentiviral transduction to generate HEK 293T cells stably expressing V5–Nir2 (Extended Data Fig. To monitor PA localization in these cells, we coexpressed superPLD with the PA reporter GFP–Spo20 (ref. GFP–Spo20 was chosen over GFP–PASS, a version designed with a nuclear export signal for improved sensitivity24, because the predominantly nuclear localization and lack of cytosolic signal of GFP–Spo20 simplified the quantitative colocalization analysis (Fig. In control wild-type cells, superPLD targeted either to the plasma membrane or to lysosomes triggered PA enrichment on these membranes, as illustrated by accumulation of GFP–Spo20 at the membranes where superPLD was localized (Fig. By contrast, in cells stably expressing V5–Nir2, this PA enrichment did not occur. Quantification of the overlap between GFP–Spo20 and superPLD fluorescence upon superPLD activation revealed little colocalization in V5–Nir2-expressing cells but increasing colocalization over time in cells expressing either the T59A lipid transfer mutant of Nir2 or empty vector, as continuous superPLD activation would be expected to elevate PA levels (Fig. We then set out to discover new PA transport proteins from among the feeding–fishing proteomics hits. We selected three additional LTPs—SCP2, PDZD8 and OSBPL1A/ORP1L—that were significantly enriched on PA-fed membranes. As with Nir2, we produced HEK 293T cells stably expressing V5-tagged versions of each LTP (Extended Data Fig. Consistent with previous reports, PDZD8–V5 localized to the ER and lysosomes26, and V5–ORP1L localized to lysosomes27. V5–SCP2 localized exclusively to punctate structures that partially colocalized with lysosomes (Extended Data Fig. Expression of the SCP2 gene leads to two protein isoforms, a full-length 58-kDa SCP2 (also known as SCP-x) and a precursor 15-kDa polypeptide, proSCP2; both of these proteins undergo post-translational cleavage to produce a 13-kDa C-terminal fragment known as mature SCP2 (mSCP2)28. The N-terminal remnant of full-length SCP2/SCP-x, after proteolytic cleavage in the peroxisomes, becomes a functional peroxisomal thiolase29, which explains the punctate localization of our V5–SCP2 construct. In line with previous studies30, a V5 fusion to mSCP2 showed increased cytosolic localization with much weaker association with peroxisomes (Extended Data Fig. We then assessed the effects of these LTPs on local PA enrichment. Whereas stable cell lines expressing the respective LTP did not show any noticeable difference in superPLD expression or activity levels (Extended Data Fig. 4f–h), HEK 293T cells stably expressing SCP2/SCP-x or proSCP2 showed a significant delay in the colocalization between GFP–Spo20 and superPLD to either membrane, indicating reduced PA enrichment on these membranes (Fig. The PA reduction effect with proSCP2 was stronger than with SCP2/SCP-x, which aligns with differences in their maturation efficiencies, as cleavage of SCP-x is partial, whereas proSCP2 undergoes complete cleavage28,31. To directly assess the capacity of SCP2 in mediating PA clearance through its lipid transfer activity, we reconstituted its activity in vitro using a fluorescence resonance energy transfer dequenching assay with tethered donor and acceptor liposomes32. In these studies, purified mSCP2 was anchored to donor liposomes to increase its effective local concentration, and fluorescence dequenching was used to assess lipid transport from donor to acceptor liposomes. We found that mSCP2 exhibited lipid transfer activity in a manner dependent on both PA and mSCP2 concentrations (Extended Data Fig. These studies suggest that SCP2, characterized as a transporter of fatty acids and several other types of phospholipids, may also function as a PA transfer protein in cells. In contrast to SCP2, expression of two other LTPs tested, PDZD8 and ORP1L, did not induce a decrease in PA enrichment from PA-fed membranes (Fig. Because both PDZD8 and ORP1L can bind to PA in vitro33,34, we examined whether superPLD-induced PA enrichment caused any changes in their subcellular localization using GFP fusions to these proteins. PDZD8–GFP was found mostly at the ER, similar to PDZD8–V5 immunofluorescence, and GFP–ORP1L showed some lysosomal association along with a cytosolic pool (Extended Data Fig. It is noteworthy that the lysosomal pool of GFP–ORP1L exhibited limited colocalization with lysosome-targeted superPLD, suggesting that ORP1L and the p18/LAMTOR-targeted superPLD may reside in different microdomains. In addition to PDZD8, another SMP domain-containing LTP35, TEX2, was also enriched on PA-fed lysosomes in feeding–fishing proteomics. Both PDZD8 and TEX2 showed relatively high association with lysosome-targeted superPLD (Fig. We found that acute PA production on lysosomes led to increased association between ER and lysosomes, and knockdown of PDZD8 mitigated this effect (Fig. Because both PDZD8 and TEX2 have been reported to localize to and mediate lipid transfer at ER–lysosome contact sites36, their detection in the feeding–fishing proteomics may reflect increased ER–lysosome association mediated by PA, rather than direct interactions with PA. a,b, Confocal images of cells coexpressing the PA-binding probe GFP–Spo20 and mCherry-fused superPLD targeted to lysosomes, whose recruitment was induced by intermittent blue-light illumination (470 nm, 5 s per 1 min), without or with stable expression of PDZD8–V5 (a) or V5–ORP1L (b). Each graph shows the colocalization time course in control cells (black) or cells stably expressing V5–PDZD8 (blue) or V5–ORP1L (red). Statistical analysis was performed using a two-sided repeated-measures ANOVA (n = 20 cells, P = 0.8562 for PDZD8 and P = 0.5025 for ORP1L (c); n = 20 cells, P = 0.3918 for PDZD8 and P = 0.5803 for ORP1L (d)). Two independent experiments were performed with similar results. e, Confocal images of cells expressing PDZD8–EGFP with coexpression of superPLD targeted to lysosomes. Representative images acquired 60 min after superPLD recruitment by intermittent blue-light illumination (470 nm, 5 s per 1 min) from 3 independent experiments are shown. f, Zoomed-in images of e for the areas marked with the dashed rectangles. g, Confocal images of cells expressing lysosome-anchored LOVPLD, costained with ER and lysosomal markers (calnexin and LAMP2, respectively), treated with or without light. Shown are representative images from three biological replicates. h, Quantification of the Pearson correlation coefficient between ER and lysosomes in LOVPLD-expressing cells treated with (red) or without (gray) light or treated with light in PDZD8-knockdown cells (blue). Black horizontal bars indicate the mean and vertical error bars indicate the s.d. Statistical significance was determined using a one-way ANOVA followed by post hoc honestly significant difference (HSD) Tukey test (n = 100 cells, **P = 0.002 and P = 0.4). In our feeding–fishing proteomics studies, several enzymes that degrade or produce PA were, respectively, either enriched on or depleted from PA-fed membranes. Notably, these enzymes displayed differential abundances between PA-fed plasma membranes and lysosomes (Extended Data Fig. For example, all three isoforms of the lipin PA phosphatases (LPIN1/2/3) were enriched on PA-fed plasma membranes, whereas only LPIN1 was detected on PA-fed lysosomes and at lower abundance levels (Fig. To understand the potential impact of these differences on PA metabolism and overall phospholipid homeostasis, we investigated the global changes to the phospholipidome by liquid chromatography (LC)–MS-based lipidomics after superPLD-mediated membrane editing to feed PA to either the plasma membrane or lysosomes (Fig. Lentiviral spinfection was used to express superPLD or deadPLD targeted to the plasma membrane or lysosomes. Then, 48 h after spinfection, cells were illuminated with or without intermittent blue light (470 nm, 5 s per 1 min) for 60 min to induce PLD recruitment to the plasma membrane or lysosomes. Cellular lipids were then extracted and analyzed by LC–MS. b, The CDP-DAG pathway and Kennedy pathway are two major pathways by which PA is converted to other phospholipids. c, Heat maps depicting fold changes in individual (top) and total (bottom) levels of phospholipid species in cells after (+hv 470 nm) versus before (dark) 60-min illumination with 470-nm blue light to recruit deadPLD or superPLD to either the plasma membrane or lysosomes. d, Heat maps depicting fold changes in individual (top) and total (bottom) levels of phospholipid species, comparing cells expressing superPLD versus deadPLD, after (+hv 470 nm) their targeting to either the plasma membrane or lysosomes (n = 3 replicates per condition). e,f, Heat maps showing fold changes in individual and total levels of phospholipid species in cells stably expressing V5–Nir2, V5–SCP2, PDZD8–V5 or V5–ORP1L, 60 min after recruitment of superPLD to either the plasma membrane (e) or lysosomes (f). Fold changes were calculated on the basis of phospholipid levels in control HEK 293T cells expressing empty V5 vector and otherwise treated identically. We found that HEK 293T cells expressing plasma membrane or lysosome-targeted superPLD exhibited light-dependent increases in PA levels after blue-light-induced superPLD recruitment to the target organelle membrane (Fig. Such increases in PA required superPLD activity, as they were not observed upon recruitment of deadPLD to these membranes (Fig. A more pronounced PA increase was observed at the plasma membrane compared to lysosomes upon PA feeding, indicating a slower net turnover of PA molecules at the plasma membrane. This finding is interesting considering the higher abundance of lipins at PA-fed plasma membranes (Extended Data Fig. 7a), which suggests an increased conversion of plasma membrane PA pools to DAG. The conversion of PA to DAG is bidirectional in cells (where the reverse reaction is catalyzed by DAG kinases), and the rapid turnover of DAG molecules has been observed with an estimated half-life of one to several minutes37,38,39. By contrast, the conversion of PA to cytidine diphosphate (CDP)-DAG (and subsequently to phosphatidylglycerol (PG) and phosphatidylinositol (PtdIns)) is unidirectional. It is possible that the sustained buildup of PA at the plasma membrane results from increased PA flux within the bidirectional PA–DAG pathway, eventually leading to slower net PA turnover, and the increased abundance of lipins and DAG kinase η (DGKH) on PA-fed plasma membrane could be involved in this event (Extended Data Fig. Moreover, despite superPLD consuming phosphatidylcholine (PC; with a preference for abundant species with 34:1 and 36:1 acyl chains10) for PA production, the levels of these lipid species did not change, suggesting that a substantial portion of PA produced on this timescale is routed to the Kennedy pathway as well to replenish these phospholipids (Fig. Concomitantly, the increased levels of PtdIns in PA-fed lysosomes may be accounted for by PA flux into the PA–CDP-DAG pathway. SuperPLD expression results in moderate background PA production activity even in the dark12, which we addressed by using a next-generation, ultralow-background membrane editor for PA production termed LOVPLD40; thus, we hypothesized that superPLD-expressing cells would exhibit chronically elevated PA levels. However, in contrast to the robust PA increase observed during acute PA production (Fig. 5c), the overall difference resulting from chronic background PA production, analyzed by comparing deadPLD-expressing versus superPLD-expressing cells, was modest or even absent (Fig. Instead, a striking increase was observed in PG levels, suggesting that cells maintain PA homeostasis by removing excess PA molecules through the CDP-DAG pathway. Interestingly, small interfering RNA (siRNA)-mediated knockdown of a mitochondrial CDP-DAG synthase (CDS), TAMM41 (refs. 41,42), attenuated this PG increase in superPLD-expressing cells (Extended Data Fig. Because PG synthesis occurs in mitochondria and TAMM41 is responsible for the CDS activity localized there41, our lipidomics analysis and the observed changes in mitochondrial morphology upon acute PA production (Extended Data Fig. 3b,c) suggest that PA trafficking to and metabolism on mitochondria may serve as a mechanism to remove excess PA. Beyond these changes in recruitment of PA-metabolizing enzymes, the feeding–fishing experiments also revealed recruitment of several LTPs that could potentially be involved in restoring PA homeostasis (Extended Data Fig. Indeed, our confocal microscopy analysis in cells coexpressing superPLD and the PA-binding probe demonstrated that forced expression of two such LTPs, Nir2 and SCP2, can diminish local enrichment of PA at the plasma membrane and lysosomal membranes (Fig. To directly investigate the roles of these lipid transporters in the global regulation of PA metabolism, we performed lipidomics analysis on cells subjected to PA feeding of either the plasma membrane or lysosomes that also overexpressed one of these LTPs. We found that forced expression of either Nir2 or SCP2 caused a significant attenuation in the global increase of PA levels that occurs upon PA feeding of either organelle membrane (Fig. These two LTPs had a stronger effect in decreasing excess PA at the plasma membrane compared to at lysosomes, a finding that aligns with our lipidomics analysis indicating slower baseline levels of PA turnover at the plasma membrane (Fig. Combined with the imaging analysis, these lipidomics results suggest that Nir2 and SCP2 can facilitate PA clearance at both local and global levels. Overexpression of PDZD8 also led to a modest but significant decrease in cellular PA levels (Fig. Taken together with the increased ER–lysosome association observed upon PA production (Fig. 4g,h), it is plausible that PDZD8 and TEX2 contribute indirectly to PA clearance either by mediating lipid transfer at or facilitating formation of ER–lysosome contact sites, rather than by directly binding to PA or PA-fed membranes. Lastly, in cells overexpressing ORP1L, we observed an unexpected elevation in PA levels upon PA feeding, particularly at lysosomes (Fig. This PA enrichment persisted when full-length ORP1L was replaced with ORP1L variants bearing amino acid substitutions in either the PtdIns4P-binding site (H651A;H652A)27, in a well-conserved lipid-binding motif, EQVSHHPP, located within the ORD domain43, or its FFAT motif (F476A;Y477A)27, which mediates ER targeting through interactions with VAPA/B. However, the PA enrichment was antagonized by expression of an ORP1L mutant in its ORD domain that is deficient in cholesterol binding (Y583A)44 (Extended Data Fig. Our proteomics and lipidomics analyses suggest that cells maintain PA homeostasis by harnessing two pathways for PA consumption, the Kennedy pathway and CDP-DAG pathway, and multiple LTPs including Nir2 and SCP2 facilitate PA trafficking and ultimate clearance in cells. To further explore how the native forms of these LTPs may regulate PA distribution and signaling as a complement to overexpression studies, we depleted the endogenous pools of these LTPs by siRNA-mediated knockdown (Extended Data Fig. We found that, in SCP2-depleted cells, stable expression of superPLD resulted in substantial cell death, potentially because of exacerbation of lipid metabolic defects upon SCP2 depletion by chronic levels of excess PA production that occur upon superPLD expression, even in the dark45. Therefore, we switched from superPLD to LOVPLD to reduce the effect of chronic background PA production. Whereas SCP2-depleted cells showed drastically reduced the expression levels of all transfected plasmids (~25% and ~50% lower levels of LOVPLD and PA probe expression level, respectively, than in control cells; Extended Data Fig. 9c,d), the patterns of PA enrichment were largely consistent with those in control cells (Fig. 6a,b; quantification shown in Extended Data Fig. Interestingly, in cells depleted of Nir2, PA distribution was markedly different but nonoverlapping with the site of PA production. Instead, in Nir2-depleted cells, PA production by LOVPLD either at the plasma membrane or on lysosomes led to PA accumulation in a perinuclear compartment that corresponded to the Golgi apparatus (Fig. a,b, Confocal microscopy images of HEK 293T cells, with or without depletion of Nir2 (siNir2) or SCP2 (siSCP2), coexpressing a PA-binding probe (GFP–PASS) and LOVPLD (containing mCherry) localized on either plasma membrane (a) or lysosomes (b). Images were acquired 0 and 30 min after incubation with intermittent blue-light illumination (470 nm, 500 ms per 5 s). c, Images of Nir2-depleted HEK 293T cells from b, followed by fixation, permeabilization and immunostaining with Golgi marker GRASP65. d, Relative mTOR signaling activity in control or LTP-depleted HEK 293T cells stably expressing LOVPLD, measured by quantification of phospho-S6K level using western blot. LOVPLD was localized to the plasma membrane, lysosomes or ER, and 30-min incubation with intermittent blue-light illumination (470 nm, 500 ms per 5 s) was used to activate PA production by LOVPLD. DsiRNA targeted to a respective LTP was used to deplete Nir2 (siNir2), SCP2 (siSCP2) or PDZD8 (siPDZD8) (n = 4 biological replicates, except for siPDZD8, where n = 2 biological replicates). Black horizontal bars indicate the mean and vertical error bars indicate the s.d. Lastly, we asked whether changes in PA trafficking and metabolism caused by knockdown of these LTPs led to functional consequences. We focused on mTOR signaling, which is known to be activated by PA produced by PLDs46. In cells stably expressing LOVPLDs at the plasma membrane, lysosome or ER, we observed light-dependent increases in the activity of mTOR signaling, measured by the phosphorylation of p70 S6 kinase 1 (S6K) (Fig. The depletion of Nir2 or SCP2 but not PDZD8 further increased the mTOR activity, correlating with the abilities of these proteins to clear excess PA produced by membrane editing. Notably, a similar degree of increase was observed in LTP-depleted cells without PA overproduction (Extended Data Fig. 9h,i), potentially reflecting changes in their baseline PA metabolism. Collectively, these knockdown studies support that endogenous pools of Nir2 and SCP2 can affect PA localization and cellular signaling pathways. In this study, we introduced feeding–fishing as a general strategy to identify regulators of lipid homeostasis at the organelle level. In this approach, membrane editing is used to feed lipids of interest to designated organelle membranes and proximity labeling is used to fish out proteins recruited to those same membranes. We implemented this concept to reveal proteins and potential mechanisms associated with PA metabolism and transport on the cytosolic leaflets of two distinct organelle membranes: the plasma membrane and the lysosomal membrane. We subsequently analyzed the consequences of the observed enrichment of several protein hits identified in the feeding–fishing proteomics using a combination of confocal microscopy to visualize the subcellular localization of PA using a PA-binding probe and LC–MS-based lipidomics to assess changes to the phospholipidome. A constant challenge in these studies is that local elevations in PA induced by membrane editing with optogenetic superPLDs is opposed by homeostatic mechanisms, that is, local PA metabolism and PA transport to distal membranes followed by subsequent metabolism. Our ability to produce transient elevations in PA on target membranes relies on the high activities of superPLDs to achieve such acute increases to PA levels12. Our studies revealed that, following membrane editing to locally elevate PA on the plasma membrane and lysosomes, certain metabolic enzymes are recruited to deplete the excess PA. Among the several metabolic fates of PA, the enzymes involved in PA–DAG interconversion were dynamically regulated on PA-fed membranes, whereas minor changes in recruitment were observed for enzymes that mediate metabolism to lysophosphatidic acid. The synthases that convert PA to CDP-DAG (CDS1/2) were not reliably detected in our proteomics experiments, likely because these are multipass transmembrane proteins localized in ER membranes. It would be interesting to expand the feeding–fishing approach to additional organelle membranes involved in PA metabolism (for example, ER, Golgi complex and mitochondria). A limitation of our study design is the moderate background activity of superPLD that is present even without light stimulation. Light-independent PG accumulation was observed in cells expressing superPLD, indicating that this background superPLD activity altered lipid metabolism even in this relatively short-term period (that is, 48 h after lentiviral transduction). Thus, future implementations of feeding–fishing proteomics would benefit from using LOVPLD, a next-generation, ultralow-background optogenetic membrane editor, for PA feeding40. In addition to insights related to the contributions of different PA-metabolizing enzymes to PA homeostasis on the plasma membrane and lysosomal membrane, the feeding–fishing studies revealed the translocation of several LTPs to PA-fed plasma membranes and lysosomes that could return excess PA and its metabolites to the ER and/or other organelles for further metabolism. Notably, both a PA-specific LTP, Nir2, and a more broad-spectrum LTP, SCP2, could reduce local PA enrichment and global PA levels under the conditions when PA was supplied to plasma membrane or lysosomes. The finding that SCP2 can reduce local PA enrichment suggests that this soluble LTP can mediate intracellular transport of PA, in addition to cholesterol and several other lipids that it has previously been found to transport28,47,48,49. SCP2 contains an N-terminal amphipathic helix that preferentially binds to membranes enriched in negatively charged lipids such as PtdIns(4,5)P2 (refs. 50,51) and its expression was reported to selectively alter the distribution of phospholipids at the plasma membrane52. Whereas our in vitro lipid transfer assay used SCP2 tethered to donor liposomes to approximate its physiological local concentration (870 nM to micromolar range53), it remains undetermined whether such tethering would occur or be required for SCP2 function in cells. Identifying additional factors, beyond PA itself, that could accelerate SCP2-mediated activity or localization would be an interesting direction for future study. Interestingly, depletion of endogenous Nir2 resulted in striking redistribution of PA produced at either the plasma membrane or lysosomal membrane to the Golgi apparatus. A previous study reported that depletion of Nir2 causes a reduction in the DAG level in the Golgi apparatus and leads to a substantial inhibition of protein transport from the trans-Golgi network to the plasma membrane54. Consistent with those findings, our results indicate that Nir2 depletion can affect PA distribution and negatively affect the PA-to-DAG conversion. Notably, the same study suggested that Nir2 RNA interference increased the levels of PC at the Golgi apparatus by ~20% through upregulation of PC synthesis from DAG and CDP-choline (Fig. 5b), whereas total cellular PC levels remained the same54. In mammalian cells, synthesis of PC from DAG is mainly catalyzed by CEPT1 and CPT1, which localize on the ER and Golgi complex, respectively55,56. Because Nir2 supports PA transport to the ER6, where PA can be metabolized into DAG and, subsequently, PC, it is plausible that Nir2-depleted cells provide compensatory PC synthesis through pathways from PA to DAG and then to PC that at least in part involves Golgi-localized enzymes. Identification of alternative LTPs responsible for PA trafficking to the Golgi complex requires further investigation. In contrast to Nir2, SCP2, PDZD8 and TEX2, whose overexpression antagonized the PA accumulation elicited by membrane editing, ORP1L forced expression led to an unexpected further increase in PA levels. ORP1L binds to PA34, but the functional relevance of such binding and whether it is accompanied by PA extraction and interorganelle transport remain unknown. Our structure–function studies revealed that the PA increase induced by ORP1L required the cholesterol-binding site in its ORD domain but not the EQVSHHPP signature motif found in all ORP-family proteins43,57, which is consistent with the lack of enrichment of other members of the ORP family in our feeding–fishing proteomics results. In other ORP-family proteins, the ORD domain mediates lipid transfer by extracting the lipid from one membrane and depositing it into another58,59,60. It can have differential affinities for two different lipids whose binding sites partially overlap, and such dual affinities are critical for vectorial transport when coupled to other metabolic steps on the origin and destination membranes. The implication of the ORD domain from ORP1L in the paradoxical increase in PA seen upon ORP1L forced expression on PA-fed membranes suggests that mechanisms more complex than simple transport to the ER and subsequent metabolism, as would be predicted by analogy with other ORP proteins, might be involved. Therefore, the roles of ORP1L in PA metabolism warrant further detailed study. Intriguingly, several other proteins related to cholesterol biosynthesis and trafficking were identified in the feeding–fishing proteomics. For example, SOAT1, which catalyzes the formation of fatty acid–cholesterol esters on the ER, and LIPA, which catalyzes the reverse reaction, were both enriched on PA-fed membranes. By contrast, STARD3, an LTP that mediates ER-to-lysosome cholesterol transport61, was depleted from PA-fed membranes. Investigation of these proteins might shed light on new and unexpected regulatory mechanisms between PA and cholesterol metabolism and homeostasis. Lastly, the enrichment of select members of the bridge-like LTP family to PA-fed membranes (VPS13B to the plasma membrane and VPS13C to the lysosome) suggests that increased bulk phospholipid flow between these organelles may occur as a result of changes to PA metabolism induced by superPLD-mediated membrane editing. Beyond elucidating mechanisms that implicate select PA-metabolizing enzymes and transporters in mediating PA homeostasis at the organelle and cellular levels, our study is notable for the introduction of the feeding–fishing strategy. This approach combining membrane editing with proximity labeling is geared to reveal how targeted modifications to the lipid composition of individual organelle membranes impacts the proteomes of such membranes, information that can point to new ways that cells can sense and correct imbalances in lipid metabolism. By exploiting organelle-specific PC hydrolysis by optogenetic PLDs, we focused here on regulators of PA metabolism at the plasma membrane and lysosomes. Yet, our optogenetic PLDs can use exogenously supplied primary alcohols in transphosphatidylation reactions to produce other phospholipids10,12,40,62 (Extended Data Fig. 10), and a growing collection of membrane editors targeting diverse types of lipids (for example, phosphoinositides, DAGs and sterols) is emerging19,62,63,64. The interfacing of such tools with organelle membrane proteomics using proximity labeling in feeding–fishing (or, conversely, fasting–fishing) experiments represents a powerful and generalizable strategy for elucidating mechanisms governing lipid homeostasis. References and/or sequences of plasmids and primers used for this study are provided (Supplementary Tables 2 and 3). Membrane-targeted TurboID and optoPLD were cloned into pCDNA5/FRT/TO for transient transfection and pCDH-CMV-MCS-EF1α-Puro for lentiviral transduction. For plasma membrane targeting, the CAAX domain of KRAS19 (GKKKKKKSKTKCVIM) was fused to the C terminus of the constructs after a linker sequence (GGSGSLYK). For lysosomal membrane targeting, the p18 domain20 (MGCCYSSENEDSDQDREERKLLLDPSSPPTKALNGAEPNY) followed by a linker sequence (GGRGSGSGSGSGSGSGSGSGSG) was fused to the N terminus of the constructs. PM-iLID-LOVPLD40 was cloned into pSBtet-Pur (Addgene, 60507) for stable expression. Plasmids encoding PITPNM1 (Nir2), SCP2, PDZD8, TEX2 or OSBPL1A (ORP1L) were purchased from the DNASU plasmid repository, and their open reading frames were cloned into pCDH-CMV-MCS-EF1α-Puro with an optional V5 tag to generate stable cell lines overexpressing these proteins. V5–Nir2, SCP2, proSCP2 (residues 405–547 of SCP2), V5–ORP1L, PDZD8–V5 and TEX2–V5 were used for analyzing their functions in PA trafficking and metabolism in live cells (note that the V5 tag was omitted from SCP2 and proSCP2 to avoid potential interference with their post-translational cleavage). PITPNM1, PDZD8, TEX2 and OSBPL1A were also cloned into pCDNA3.1 along with EGFP or miRFP fluorescent proteins for localization studies in live cells. For visualization of PA localization in live cells, a Spo20 PA-binding domain25 (MDNCSGSRRRDRLHVKLKSLRNKIHKQLHPNCRFDDATKTS) or PASS domain67 (Addgene, 193970) fused to EGFP was cloned into pCDH-CMV-MCS-EF1α-Puro. Cells were grown in DMEM (Corning) supplemented with 10% FBS (Corning), 1% penicillin–streptomycin (Corning) and 1 mM sodium pyruvate (Thermo Fisher) at 37 °C in a 5% CO2 atmosphere. For poly(l-lysine) pretreatment, cell plates were treated with 0.1 mg ml−1 poly(l-lysine) (Sigma Aldrich, P2636) in PBS for 1 h at 37 °C, followed by triple rinses with autoclaved deionized water. For fibronectin coating, cell plates were treated with 20 μg ml−1 human plasma fibronectin (Millipore Sigma, FC010) in PBS for 5–15 min at 37 °C. For transient transfection, HEK 293T cells were transfected using Lipofectamine 2000 (Invitrogen, 11668019) or PEI MAX (Polysciences, 24765). Cells were incubated in regular DMEM containing plasmids premixed with Lipofectamine 2000 (1–1.5 µg of total plasmids and 3 µl of Lipofectamine 2000 or PEI MAX for cells in 35-mm dish) and the cells were incubated for 20–24 h before experiments. For transient gene knockdown, HEK 293T cells were transfected using Lipofectamine RNAiMAX (Invitrogen, 13778075). Cells were incubated in regular medium containing DsiRNA (Integrated DNA Technologies) premixed with Lipofectamine RNAiMAX (5 pmol of DsiRNA and 1.5 µl of Lipofectamine RNAiMAX for cells in a 24-well plate) and the cells were incubated for 40–48 h before experiments. A list of DsiRNA sequences is provided in Supplementary Table 3. HEK 293T cells were transfected using Lipofectamine 2000 or PEI MAX for lentivirus production. HEK 293T cells seeded on a six-well plate were incubated in Transfectagro (Corning) or Opti-MEM (Gibco) supplemented with 10% FBS containing plasmids premixed with Lipofectamine 2000 or PEI MAX (0.5 µg of envelope plasmid, 1 µg of packaging plasmid, 1.5 µg of transfer plasmid and 6 µl of Lipofectamine 2000 or PEI MAX per well for a six-well plate). Then, 12–16 h after transfection, the transfection medium was replaced with regular DMEM and media were collected 40–48 h and 60–72 h after transfection to obtain virus-containing media. For multivirus transduction (for example, to introduce all feeding–fishing components), the collected virus media were concentrated by centrifugation at 100,000g (24,000 rpm in an SW 41 Ti swinging-bucket rotor) for 90 min at 4 °C. After ultracentrifugation, the supernatant was carefully decanted and the virus concentrate was resuspended in fresh DMEM. The ratio of lentivirus packaging cells and transduced cells used for this study was 1.5:1 for pCDH-CRY2-mCherry-PLD constructs and 0.16:1 for all the other constructs, optimized on the basis of the lentivirus titer of each construct. For spinfection, HEK 293T cells seeded on a six-well plate (pretreated with poly(l-lysine) or fibronectin) were incubated in virus-resuspended medium supplemented with 0.4 µg ml−1 polybrene (Millipore Sigma). The plate was centrifuged at 931 or 1,000g for 2 h at 37 °C, followed by the replacement of virus-containing medium with fresh DMEM (as well as DsiRNA precomplexed with Lipofectamine RNAiMAX for knockdown study). The six-well plate was covered with aluminum foil to keep cells in the dark and the cells were incubated for 40–48 h before experiments. A homemade light box was built by attaching four strips of dimmable, 12-V blue light-emitting diode (LED) tape light (1000Bulbs.com, 2835-60-IP65-B1203) on the inside of a Styrofoam box. For optogenetics experiments, the light box was placed inside the CO2 incubator using an AC outlet power bank (Omars, 24,000 mAh, 80 W) as a power supply. An outlet timer (BN-LINK) was used to switch the light on and off automatically to enable 3-s intervals of 470-nm light per 1 min. For imaging and mTOR signaling experiments using LOVPLD (Figs. 3 and 4a,b), an AMUZA system consisting of a blue LED array (470 nm), an LED array driver and a pulse generator with 10% duty cycle (500 ms on, 5 s off) was used. Light power was typically around 10 mW cm−2 (measured by Thorlabs PM100D). Cells expressing optoPLD and TurboID were illuminated for 30 min with intermittent blue light (470 nm, 5-s pulses every 1 min), followed by 3-min treatment with 500 µM biotin. The cells were then washed five times with PBS and lysed in RIPA lysis buffer (50 mM Tris-HCl pH 7.4, 150 mM NaCl, 1% Triton X-100, 0.5% sodium deoxycholate, 0.1% SDS, 1 mM EDTA and 1× cOmplete protease inhibitor). For proteomics, the cells from all six wells in a six-well plate (~2 × 107 cells total) were combined to prepare each sample. After sonication and centrifugation, the lysate supernatant was incubated with Pierce high-capacity streptavidin–agarose beads (Thermo Fisher Scientific, 20359) overnight at 4 °C on a rotator. A 10-µl volume of bead solution was used for cells per well in 6-well plate. For western blot analysis, the washed beads were boiled for 10 min in 3× Laemmli sample buffer supplemented with 25 mM biotin to elute biotinylated proteins off the beads. The washed beads were boiled for 5 min in sample elution buffer for protein elution. The samples were then quantified by BCA assay and submitted to Proteomics and Metabolomics Facility at Cornell University for TMT labeling and proteomics analysis. Four samples in triplicate were grouped for the labeling and analysis. Detailed workflows of the proteomics analyses are described in Supplementary Information. Antibody and dilutions used for validation experiments (Extended Data Fig. Detection was conducted using chemiluminescence with the SuperSignal West Pico PLUS chemiluminescent substrate (Thermo Fisher Scientific, 34580) or SuperSignal West Atto ultimate-sensitivity substrate (Thermo Fisher Scientific, A38556) and acquisition on a Bio-Rad ChemiDoc MP system. Cells were seeded on 35-mm glass-bottom imaging dishes (Matsunami Glass) coated with poly(l-lysine) or fibronectin. Unless otherwise noted, images were acquired every 1 min for 1 h at 37 °C using Zeiss Zen Blue 2.3 on a Zeiss LSM 800 confocal laser-scanning microscope equipped with Plan Apochromat objectives (×40, numerical aperture: 1.4) and two GaAsP photomultiplier tube detectors. Solid-state lasers (488, 561 and 640 nm) were used to excite GFP, mCherry and miRFP, respectively, and the 488-nm laser irradiation also served as a stimulus for activating optoPLD recruitment to the plasma membrane, ER or lysosomes. Imaging experiments with LOVPLD (Figs. 3 and 4a,b) were acquired using a Zeiss AxioObserver inverted microscope equipped with a Yokogawa spinning-disk confocal head, Cascade II:512 camera and four-color laser launch (405-nm diode, 491-nm diode-pumped solid-state laser (DPSS), 561-nm DPSS and 640-nm diode, all at 50 mW). Images were acquired using Slidebook software (Intelligent Imaging Innovations) through a ×100 oil-immersion objective. Mitochondrial morphology was observed by staining cells with 100 nM MitoTracker Green (Thermo Fisher Scientific, M7514) for 5 min before experiments. 12)) targeted to different organelle membranes and the indicated LTP were imaged as described above. Colocalization analysis between PA-binding probe and superPLD was carried out on ImageJ/FIJI as follows. Firstly, a region of interest (ROI) was drawn around each cell expressing both constructs. Secondly, for each ROI, the mCherry signal of the superPLD was used to generate a binary mask. Cells were fixed in 4% formaldehyde for 10 min at room temperature, rinsed three times with PBS, permeabilized with 0.5% Triton X-100 in PBS for 15 min at room temperature and blocked with 1% BSA and 0.1% Tween-20 in PBS (PBS-T; blocking buffer) for 30 min. The cells were then treated with primary antibody solution in blocking buffer for 1 h at room temperature and rinsed three times with PBS-T. Afterward, the cells were treated with secondary antibody solution in blocking buffer for 1 h at room temperature and rinsed three times with PBS-T. After the final rinse, the cells were incubated in PBS-T for 5 min and stored in PBS supplemented with 1 µg ml−1 DAPI. Image acquisition by laser-scanning confocal microscopy was performed as described above using solid-state lasers (405, 488, 561 and 640 nm) to excite DAPI, AlexaFluor 488, mCherry and AlexaFluor 647, respectively. Pearson colocalization analysis was performed using the Coloc 2 plugin in ImageJ/FIJI. Antibodies, affinity reagents and dilutions used for immunofluorescence staining were as follows: streptavidin–AlexaFluor 488 (Invitrogen, S11223; 1:50), anti-V5 antibody (Bio-Rad, MCA1360GA; 1:100), anti-calnexin antibody as an ER marker (Thermo Fisher Scientific, PA534754; 1:100), anti-LAMP2 antibody as a lysosomal marker (Santa Cruz Biotechnology, sc-18822; 1:100), anti-GRASP65 antibody as a Golgi marker (Santa Cruz Biotechnology, sc-374423; 1:100), anti-LPIN1 antibody (Cell Signaling Technology, 14906; 1:100), anti-mouse–AlexaFluor 488 antibody conjugate (Invitrogen, A21202; 1:500) and anti-mouse–AlexaFluor 647 (Invitrogen, A31571; 1:500). mSCP2 fused with an N-terminal 6×His tag was cloned into a pET28a vector and the construct was transformed into BL21 Rosetta2 Escherichia coli cells for purification. Cells were cultured in Terrific Broth medium at 37 °C to an optical density at 600 nm of 0.5 (approximately 2 h), and then expression was induced by addition of 0.2 mM IPTG followed by overnight incubation at 18 °C. Lysate was clarified by centrifugation at 10,000g for 30 min, followed by incubation with TALON beads for 2 h at 4 °C. The resin was washed four times with lysis buffer, followed by elution with lysis buffer supplemented with 150 mM imidazole. Liposomes were prepared as previously described32. Briefly, lipids were dissolved in chloroform and mixed in glass tubes in the indicated ratio for donor and acceptor liposomes separately. The mixture was dried under a stream of nitrogen to a thin film, followed by further drying in vacuo for 2 h. The lipid film was hydrated with buffer containing 25 mM Tris-HCl, 150 mM NaCl and 0.5 mM TCEP at pH 8.0 to a total lipid concentration of 2.5 mM. Liposomes were formed by ten freeze–thaw cycles in liquid N2 and 37 °C water bath, followed by extrusion 21 times through polycarbonate filters with a pore size of 100 nm. Lipid transfer assays were set up at 25 °C in black flat-bottom 96-well plates, with 100-µl total volume containing 50 µM donor liposomes, 150 µM acceptor liposomes, 62.5 nM 6×His–PH domain tether (used to link donor and acceptor liposomes and, therefore, minimize spurious lipid exchange between donor–donor and acceptor–acceptor liposomes) and 187.5 nM 6×His–mSCP2 (His tag used to anchor mSCP2 to donor liposomes to increase local concentration), for a 1:800 protein-to-lipid ratio and a 1:3 tether-to-LTP ratio. For specific reactions, different compositions of liposomes or concentrations of LTP were used as indicated. The reaction was initiated by addition of proteins and was then monitored for 2 h, using an excitation wavelength of 465 nm and emission wavelength of 540 nm every 10 s, using a BioTek Synergy H1 microplate reader. HEK 293T cells expressing the indicated constructs were illuminated for 30 min with intermittent blue light (470 nm, 5-s pulses every min) and rinsed once in PBS; the cellular lipids were extracted using the Bligh–Dyer method10. Then, 250 μl of methanol was added to cells in a 35-mm dish on ice. Followed by the addition of 125 µl of 20 mM acetic acid and 100 µl of PBS, the cells were scraped off and transferred into a 1.5-ml centrifuge tube. After addition of 500 µl of chloroform, the tube was shaken vigorously for 3 min and centrifuged for 1 min at 10,000g. The bottom organic layer was transferred into a new tube and dried under a stream of N2 gas. The resulting lipid film was dissolved in 150 µl of chloroform and subjected to high-resolution LC–MS analysis. LC–MS measurement was performed on an Agilent 6230 electrospray ionization time-of-flight MS instrument coupled to an Agilent 1260 high-performance LC instrument equipped with a Luna 3-µm silica LC column (Phenomenex; 50 × 2 mm) using a binary gradient elution system where solvent A was chloroform, methanol and ammonium hydroxide (85:15:0.5) and solvent B was chloroform, methanol, water and ammonium hydroxide (60:34:5:0.5). Separation was achieved using a linear gradient from 100% A to 100% B over 10 min. Phospholipid species were detected using an Agilent Jet Stream source operating in positive or negative mode, acquiring in an extended dynamic range of m/z 100–1,700 at one spectrum per second (gas temperature, 325 °C; drying gas, 12 L min−1; nebulizer, 35 psi; fragmentor, 300 V (for positive mode) and 250 V (for negative mode); sheath gas flow, 12 L min−1; Vcap, 3,000 V; nozzle voltage, 500 V. The LC–MS data were analyzed on MassHunter quantitative analysis software using the ‘find compounds by formula' tool. The search parameters were set as follows: source of formulas to confirm, database/library provided in Supplementary Table 4; matches per formula, 1 (automatically increase for isomeric compounds); Values to match, mass and retention time (retention time required); match tolerance, masses ± 20 ppm and retention times ± 0.200 min; expansion of values for chromatogram extraction, m/z ± 20 ppm and retention time 0.500 + 1.00 min; positive ion charge carriers, +H; negative ion charge carriers, −H. HEK 293T cells stably expressing doxycycline-inducible PM-iLID-LOVPLD were seeded in 12-well plates with 2.5 µg ml−1 doxycycline and incubated in the dark for 48 h. Each alcohol was first prepared as a 100× concentrated solution in PBS and then added to cells stably expressing PM-iLID-LOVPLD. The cells were immediately transferred to the light chamber and illuminated for 15 min with intermittent blue light (470 nm, 5-s pulses every 1 min). Bligh–Dyer extraction followed by LC–MS analysis was performed as described above. HEK 293T cells were transduced with LOVPLD using lentivirus and spinfection as described above. The cells were incubated with 10 μM dorsomorphin (AMPK inhibitor) for 1 h at 37 °C, followed by a 30-min stimulation with intermittent blue-light illumination (470 nm, 500-ms pulses every 5 s). The cells were then lysed with RIPA lysis buffer supplemented with protease and phosphatase inhibitor cocktails (Thermo Fisher Scientific, 78439 and 78420). After sonication and centrifugation, the lysate supernatants were mixed with 6× Laemmli sample buffer to prepare the sample for western blot. The membrane was blotted with 1:1,000 dilutions of antibodies for phospho-S6K (T389; Cell Signaling Technology, #9205) or β-actin (Cell Signaling Technology, #5125), with detection by chemiluminescence using the SuperSignal West Pico PLUS chemiluminescent substrate (Thermo Fisher Scientific, 34580) and acquisition on a Bio-Rad ChemiDoc MP system. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data supporting the findings of this study are available within the paper and its Supplementary Information. Proteomics data are available from the PRIDE database under accession codes PXD070270 and PXD070378. Plasmids generated during the current study are listed along with their source information in Supplementary Information. Source data are provided with this paper. Custom Python scripts developed for proteomics and lipidomics analyses are available from GitHub (https://github.com/teipanda/Membrane-Turbo)69, along with the corresponding example datasets. Other scripts used for data processing and plotting were developed using a combination of ImageJ macros and Python and are available from the corresponding author upon request. & Wang, X. in Lipid-Mediated Protein Signaling (ed. Vance, J. E. Phospholipid synthesis and transport in mammalian cells. Selvy, P. E., Lavieri, R. R., Lindsley, C. W. & Brown, H. A. Phospholipase D: enzymology, functionality, and chemical modulation. Shulga, Y. V., Topham, M. K. & Epand, R. M. Regulation and functions of diacylglycerol kinases. Kim, Y. J., Guzman-Hernandez, M.-L., Wisniewski, E. & Balla, T. Phosphatidylinositol–phosphatidic acid exchange by Nir2 at ER–PM contact sites maintains phosphoinositide signaling competence. Hussain, Z., Uyama, T., Tsuboi, K. & Ueda, N. Mammalian enzymes responsible for the biosynthesis of N-acylethanolamines. Chen, P.-H. B., Li, X.-L. & Baskin, J. M. Synthetic lipid biology. Luan, L., Frederick, N. P. & Baskin, J. M. Emerging approaches for studying lipid dynamics, metabolism, and interactions in cells. Tei, R. & Baskin, J. M. Spatiotemporal control of phosphatidic acid signaling with optogenetic, engineered phospholipase Ds. Chiu, D.-C. & Baskin, J. M. Imaging and editing the phospholipidome. Tei, R., Bagde, S. R., Fromme, J. C. & Baskin, J. M. Activity-based directed evolution of a membrane editor in mammalian cells. Efficient proximity labeling in living cells and organisms with TurboID. Directed evolution of APEX2 for electron microscopy and proximity labeling. Qin, W., Cho, K. F., Cavanagh, P. E. & Ting, A. Y. Deciphering molecular interactions by proximity labeling. & Airola, M. V. Structure and regulation of human phospholipase D. Adv. Frias, M. A. et al. Phosphatidic acid drives mTORC1 lysosomal translocation in the absence of amino acids. Hussain, S. S. et al. RalA and PLD1 promote lipid droplet growth in response to nutrient withdrawal. Idevall-Hagren, O., Dickson, E. J., Hille, B., Toomre, D. K. & Camilli, P. D. Optogenetic control of phosphoinositide metabolism. & Okada, M. p18/LAMTOR1: a late endosome/lysosome-specific anchor protein for the mTORC1/MAPK signaling pathway. Kumar, M., Keller, B., Makalou, N. & Sutton, R. E. Systematic determination of the packaging limit of lentiviral vectors. Berggren, W. T., Lutz, M. & Modesto, V. General spinfection protocol. In StemBook (Harvard Stem Cell Institute, 2008). The impact of non-toxic blue light (453 nm) on cellular antioxidative capacity, TGF-β1 signaling, and myofibrogenesis of human skin fibroblasts. Acute diacylglycerol production activates critical membrane-shaping proteins leading to mitochondrial tubulation and fission. Horchani, H., de Saint-Jean, M., Barelli, H. & Antonny, B. Interaction of the Spo20 membrane-sensor motif with phosphatidic acid and other anionic lipids, and influence of the membrane environment. Elbaz-Alon, Y. et al. PDZD8 interacts with Protrudin and Rab7 at ER–late endosome membrane contact sites associated with mitochondria. Zhao, K. & Ridgway, N. D. Oxysterol-binding protein-related protein 1L regulates cholesterol egress from the endo-lysosomal system. Gallegos, A. M. et al. Gene structure, intracellular localization, and functional roles of sterol carrier protein-2. Wirtz, K. W. Phospholipid transfer proteins revisited. Schroeder, F. et al. Pro-sterol carrier protein-2: role of the N-terminal presequence in structure, function, and peroxisomal targeting. Valverde, D. P. et al. ATG2 transports lipids to promote autophagosome biogenesis. & Ji, W.-K. PDZD8-mediated lipid transfer at contacts between the ER and late endosomes/lysosomes is required for neurite outgrowth. Xu, Y., Liu, Y., Ridgway, N. D. & McMaster, C. R. Novel members of the human oxysterol-binding protein family bind phospholipids and regulate vesicle transport. Reinisch, K. M. & De Camilli, P. SMP-domain proteins at membrane contact sites: structure and function. Jeyasimman, D. et al. PDZD-8 and TEX-2 regulate endosomal PI(4,5)P2 homeostasis via lipid transport to promote embryogenesis in C. elegans. Schuhmacher, M. et al. Live-cell lipid biochemistry reveals a role of diacylglycerol side-chain composition for cellular lipid dynamics and protein affinities. Höglinger, D. et al. Trifunctional lipid probes for comprehensive studies of single lipid species in living cells. B. et al. Biophysical physiology of phosphoinositide rapid dynamics and regulation in living cells. Li, X.-L., Tei, R., Uematsu, M. & Baskin, J. M. Ultralow background membrane editors for spatiotemporal control of phosphatidic acid metabolism and signaling. Blunsom, N. J., Gomez-Espinosa, E., Ashlin, T. G. & Cockcroft, S. Mitochondrial CDP-diacylglycerol synthase activity is due to the peripheral protein, TAMM41 and not due to the integral membrane protein, CDP-diacylglycerol synthase 1. Structures of the mitochondrial CDP-DAG synthase Tam41 suggest a potential lipid substrate pathway from membrane to the active site. Functions of oxysterol-binding proteins at membrane contact sites and their control by phosphoinositide metabolism. Dong, J. et al. Allosteric enhancement of ORP1-mediated cholesterol transport by PI(4,5)P2/PI(3,4)P2. Atshaves, B. P. et al. Effect of SCP-x gene ablation on branched-chain fatty acid metabolism. Regulation of mTOR by phosphatidic acid. Galano, M., Venugopal, S. & Papadopoulos, V. Role of STAR and SCP2/SCPx in the transport of cholesterol and other lipids. Puglielli, L., Rigotti, A., Greco, A. V., Santos, M. J. & Nervi, F. Sterol carrier protein-2 is involved in cholesterol transfer from the endoplasmic reticulum to the plasma membrane in human fibroblasts. Schroeder, F. et al. Sterol carrier protein-2 functions in phosphatidylinositol transfer and signaling. Huang, H., Ball, J. M., Billheimer, J. T. & Schroeder, F. The sterol carrier protein-2 amino terminus: a membrane interaction domain. Schroeder, F. et al. Sterol carrier protein-2: new roles in regulating lipid rafts and signaling. Gallegos, A. M. et al. Sterol carrier protein-2 expression alters plasma membrane lipid distribution and cholesterol dynamics. Gianotti, A. R., Ferreyra, R. G. & Ermácora, M. R. Binding properties of sterol carrier protein 2 (SCP2) characterized using Laurdan. Litvak, V., Dahan, N., Ramachandran, S., Sabanay, H. & Lev, S. Maintenance of the diacylglycerol level in the Golgi apparatus by the Nir2 protein is critical for Golgi secretory function. & Sugimoto, H. Differential contributions of choline phosphotransferases CPT1 and CEPT1 to the biosynthesis of choline phospholipids. Dorighello, G., McPhee, M., Halliday, K., Dellaire, G. & Ridgway, N. D. Differential contributions of phosphotransferases CEPT1 and CHPT1 to phosphatidylcholine homeostasis and lipid droplet biogenesis. Depta, L., Whitmarsh-Everiss, T. & Laraia, L. Structure, function and small molecule modulation of intracellular sterol transport proteins. Mesmin, B. et al. A four-step cycle driven by PI4P hydrolysis directs sterol/PI4P exchange by the ER–Golgi tether OSBP. The oxysterol-binding protein cycle: burning off PI4P to transport cholesterol. Kovács, D. et al. Lipid exchange at ER–trans-Golgi contact sites governs polarized cargo sorting. Wilhelm, L. P. et al. STARD3 mediates endoplasmic reticulum-to-endosome cholesterol transport at membrane contact sites. Tei, R. & Baskin, J. M. Induced proximity tools for precise manipulation of lipid signaling. Zewe, J. P. et al. Probing the subcellular distribution of phosphatidylinositol reveals a surprising lack at the plasma membrane. Pham, H. et al. Development of a novel spatiotemporal depletion system for cellular cholesterol. Abu-Remaileh, M. et al. Lysosomal metabolomics reveals V-ATPase- and mTOR-dependent regulation of amino acid efflux from lysosomes. The amino-terminal 29 amino acids of cytochrome P450 2C1 are sufficient for retention in the endoplasmic reticulum. Zhang, F. et al. Temporal production of the signaling lipid phosphatidic acid by phospholipase D2 determines the output of extracellular signal-regulated kinase signaling in cancer cells. & De Camilli, P. Ca2+ releases E-Syt1 autoinhibition to couple ER–plasma membrane tethering with lipid transport. acknowledges support from the National Institutes of Health (R01GM151682). R.T. was supported by Honjo International, Funai Overseas, Cornell and Life Sciences Research Foundation fellowships. was supported by a Natural Sciences and Engineering Research Council of Canada postgraduate fellowship. We acknowledge Cornell University Proteomics and Metabolomics Facility for their support in designing and analyzing proteomics study; specifically, we thank E. Anderson for TMT labeling of proteomics samples, Q. Fu for data analysis and S. Zhang for guidance. We thank A. Y. Ting and her lab at Stanford University for providing resources and instrumentation used in parts of this study. These authors contributed equally: Xiang-Ling Li, Lin Luan. Department of Chemistry and Chemical Biology, Cornell University, Ithaca, NY, USA Reika Tei, Xiang-Ling Li & Jeremy M. Baskin Weill Institute for Cell and Molecular Biology, Cornell University, Ithaca, NY, USA Reika Tei, Xiang-Ling Li, Lin Luan & Jeremy M. Baskin Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar carried out the lipidomics experiments using LOVPLD, L.L. carried out the in vitro liposome assays, and R.T. carried out all other experiments and data analysis. The authors declare no competing interests. Nature Chemical Biology thanks Itay Budin, Carsten Schultz and the other, anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, b, Schematic depiction of optogenetic superPLD (a) for light-dependent recruitment of PLDPMF on the selected membranes, used in the Feeding–Fishing proteomics, and LOVPLD (b), which is constitutively tethered onto the membranes and activated upon 470 nm blue light, used for secondary validation with high temporal resolution. c–e, Quantification of key proteomics hits in supernatant (c) and streptavidin pulldown (d), with quantification shown in (e). SCP2, DGKD, DGKH, and Tubulin were not detected by Western blot in the pulldown samples. n = 2 independent biological replicates, and similar results were obtained in three independent experiments. Statistical analysis was performed using one-way ANOVA (p = 0.0008, 0.05, 0.01, 0.18, 0.11, 0.69, 0.0001, 0.05, respectively). f, g, Confocal microscopy analysis of endogenous Lipin-1, stained using an α-Lipin-1 antibody, with or without LOVPLD activation by 470 nm blue light, revealing changes in Lipin-1 localization pattern upon PA production at the plasma membrane. Shown are representative images from three biological replicates. STRING network analysis of proteins found to be either enriched on (a, plasma membrane; b, lysosomes) or depleted from (c, plasma membrane; d, lysosomes) PA-fed membranes. Proteins that showed p-value < 0.01 were analyzed (interaction score > 0.400), and clustering was performed using k-means. Two-sided Student's t-tests were used for p value calculation. a, Confocal microscopy of HEK 293T cells expressing plasma membrane or lysosome-targeted superPLD or deadPLD. Mitochondria were stained with MitoTracker Green. b, c, Quantification (b) of mitochondrial morphology (Normal, Mix, or Round) in cells expressing LOVPLD anchored on endoplasmic reticulum (ER), lysosome (Lyso), or plasma membrane (PM) over the indicated time periods (0–30 min for ER and 0–180 min for Lyso and PM). 50–100 cells (exact numbers are provided in the Source Data) from each condition were analyzed. Mitochondrial dysmorphology was more rapidly observed in the order of: ER > Lyso > PM. Representative cells showing Normal, Mix, and Round mitochondria are indicated by gray, green, and magenta arrows in (c), respectively. Similar results have been obtained in three independent experiments. a, b, Quantification of miRFP-Nir2 (T59A mutant with lipid transfer activity disabled) co-localization with PA-fed membranes. Black horizontal bars indicate means and vertical error bars indicate standard deviations. Statistical significance was determined using two-sided Student's t-test (n = 10–16 cells with exact numbers provided in the Source Data, p = 8.6E-06 and 8.9E-07). c, Confocal images of HEK 293T cells stably expressing V5-tagged Nir2 (PITPNM1), PDZD8, ORP1L (OSBPL1A), full-length SCP2 (SCP-x), or the mature form of SCP2 (mSCP2). Shown are representative images from two independent experiments. f, Scheme of IMPACT labeling to quantify superPLD activity in live cells. g, h, Flow cytometry results of cells co-expressing superPLD targeted to the plasma membrane (PM; g) or lysosomes (Lyso; h) and the indicated lipid-transfer protein. mCherry signal (readout of superPLD expression level) and BODIPY signal (readout of PLD activity) are plotted. Wild-type cells expressing deadPLD were included as a negative control. Shown are representative plots from two independent experiments. a, Schematic design of lipid transfer assay. Donor liposomes include NBD conjugated to phosphatidic acid (NBD-PA), whose fluorescence signal is quenched by FRET by rhodamine conjugated to phosphatidylethanolamine (Rhod-PE). Lipid transfer from donor liposomes to acceptor liposomes results in dequenching and increased fluorescence signal. b, SDS-PAGE gel of purified mSCP2. c–e, Fluorescence signal readout using donor liposomes with different POPA percentages (c), negative control where mSCP2 or acceptor liposomes were omitted (d), and with different mSCP2 concentrations (e). Raw fluorescence signal is shown except for e, where fluorescence signal was normalized to the No Acceptor control. The concentration of mSCP2 was 187.5 nM unless otherwise noted. Shown are representative plots from two independent experiments. a, b, Confocal images of cells co-expressing PDZD8-EGFP (a) or EGFP-ORP1L (b) and superPLD or deadPLD targeted to lysosomes (Lyso). Images acquired 0, 30, and 60 min following recruitment of superPLD or deadPLD induced by intermittent blue light illumination (470 nm, 5 s per 1 min), representative of three biological replicates, are shown. c, Confocal images of cells expressing TEX2-EGFP, with co-expression of superPLD or deadPLD targeted to lysosomes (Lyso). Images acquired 60 min following superPLD/deadPLD recruitment, representative of three biological replicates, are shown. d, Zoomed-in images of (c) for the areas marked with the dashed rectangles. a, Differential enrichment and depletion of enzymes that mediate PA synthesis and degradation on PA-fed plasma membranes (left) and lysosomes (right). Shown in the table are fold changes in protein abundance between the PA-fed (superPLD-recruited) and the negative control (deadPLD-recruited) membranes and the mean abundance values for each condition (n = 3). The fold changes determined to be statistically significant by two-sided Student's t-tests (abundance ratio p-value < 0.05) are shown in bold. Enzymes that mediate conversion of PA to LPA (for example, PLA2s) were not detected. b, Similar list as in (a), showing lipid transfer proteins that exhibit differential enrichment on or depletion from PA-fed plasma membranes (left) and lysosomes (right). a, Quantification of PG levels by lipidomics upon expression of either deadPLD in control (gray) or TAMM41 knockdown (cyan) cells, or superPLD in control (magenta) or TAMM41 knockdown (green) cells, showing that TAMM41 knockdown attenuates the increase of PG levels in superPLD-expressing cells. Total cellular lipids were extracted and analyzed after a 30 min incubation with intermittent blue light illumination (470 nm, 5 s per 1 min). PG species with average normalized ion counts > 0.05 are shown. Vertical error bars indicate standard deviations. Statistical analysis was performed using one-way ANOVA followed by post-hoc HSD Tukey test (n = 3, p-values are shown in the plots). b, Similar quantification as in (a), showing PA levels. Note that TAMM41 knockdown did not lead to a further increase in PA levels, despite the abolished PG increase, suggesting that cells may compensate for the loss by removing excess PA through an alternative mechanism. c, Domain maps summarizing ORP1L mutants used in the lipidomics study. d, Heat map analysis similar to that shown in Fig. 5f but instead depicting fold changes in individual and total levels of phospholipid species in cells stably expressing V5-ORP1L, V5-ORP1LH651A/H652A (HH/AA: ORD domain mutant lacking PI4P-binding motif), V5-ORP1LY583A (Y583A: ORD domain mutant lacking cholesterol-binding motif), or V5-ORP1LF476A/Y477A (FY/AA: an FFAT mutant deficient in binding to VAP). a, Western blot of LOVPLD-expressing HEK 293T cells treated with DsiRNA to deplete Nir2, SCP-x/SCP2, or PDZD8. DsiRNA #1 (colored red) was used in the subsequent experiments. b, Quantification of (a), with relative levels compared to the control shown in parentheses. Plots are representative of two independent experiments. c, d, Expression levels of LOVPLD (c) and the PA probe GFP-PASS (d) in HEK 293T cells depleted with Nir2 (siNir2), SCP2 (siSCP2), or PDZD8 (siPDZD8), co-expressing PA probe and LOVPLD targeted to either plasma membrane (PM) or lysosomes (Lyso). Plots are representative of two independent experiments. e, f, Quantification of colocalization between LOVPLD and GFP-PASS shown in Fig. Black horizontal bars indicate means and vertical error bars indicate standard deviations. Statistical analysis was performed using one-way ANOVA followed by post-hoc Tukey-HSD test (n = 14–19 cells with exact numbers provided in the Source Data, p = 0.01 and 0.6 for (e) and 0.03 and 0.9 for (f) for Control vs. siRNA samples). g, Western blot of p-S6K to measure mTOR activity in LTP-depleted cells expressing LOVPLD, with quantification presented in Fig. LOVPLD was localized to plasma membrane (PM), lysosomes (Lyso), or endoplasmic reticulum (ER), and 30-min incubation with intermittent blue light illumination (470 nm, 500 ms per 5 s) was used to activate PA production by LOVPLD. β-actin was used as a loading control. h, Similar experiment as (g) but in LTP-depleted HEK 293T cells without LOVPLD activation. Statistical analysis was performed using one-way ANOVA followed by post-hoc Tukey-HSD test (p = 0.004, 0.01, and 0.07 for Control vs. siRNA samples). Black horizontal bars indicate means and vertical error bars indicate standard deviations. a–d, LC–MS quantification of phosphatidyl alcohol lipid species extracted from HEK 293T cells expressing PM-targeted iLID-LOVPLD treated with butanol (a), ethanol (b), butynol (c), or azidopropanol (d), demonstrating that LOVPLD can mediate in situ production of phospholipids with customizable alcohols in light-dependent manner. Total cellular lipids were extracted and analyzed after a 15 min incubation with intermittent blue light illumination (470 nm, 5 s per 1 min). Black horizontal bars indicate means and vertical error bars indicate standard deviations. e–h, Similar quantification of PA species extracted from cells treated with butanol (e), ethanol (f), butynol (g), or azidopropanol (h), showing the lack of noticeable increase in PA levels. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Tei, R., Li, XL., Luan, L. et al. Membrane editing with proximity labeling reveals regulators of lipid homeostasis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            