
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-02-05</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/these-two-habits-are-linked-to-more-than-a-third-of-all-cancer-cases/'>These two habits are linked to more than a third of all cancer cases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-05 16:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>These two habits are linked to more than a third of all cancer cases More than one-third of cancer cases are preventable, a massive study finds Overall, tobacco smoking was the leading contributor to worldwide cancer cases, followed by infections and drinking alcohol. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Cancer continues to be a leading cause of illness and death worldwide, with cases expected to rise over the next decades if current trends continue. Previous studies have estimated that around 44% of global cancer deaths can be attributed to avoidable or controllable causes. Estimates of preventability have mainly focused on the number of deaths rather than cases and have mostly investigated a single risk factor, says Fink. To address this gap, Fink and her colleagues examined global case data from 2022 for 36 different cancers across 185 countries. The study included 30 modifiable risk factors that are well-established causes of cancer — such as tobacco smoking, alcohol consumption and infections. The researchers combined this information with data from 2012 that captured people's exposure to each risk factor. Fink and her colleagues then estimated the proportion of cases that were directly linked to each risk factor. In 2022, there were a total of 18.7 million new cancer cases worldwide. Roughly 38% — or 7.1 million — of these cases could be attributed to avoidable causes. Globally, tobacco smoking was the leading contributor, accounting for around 15% of preventable cases. Lung, stomach and cervical cancers made up nearly half of all preventable cancer cases. Around 30% of the 9.2 million new cases in women were preventable. More than 11% of these were associated with infections, such as those caused by human papillomavirus (HPV) — the leading cause of cervical cancer. Most of these cases occurred in low- and middle-income regions, such as sub-Saharan Africa, where cervical cancer rates are highest. Infections ranked second — mostly occurring in parts of Africa, Asia and South America — followed by drinking alcohol. The study is a “fine piece of work” that signals the need to double down on cancer control efforts, says David Whiteman, a medical epidemiologist at the QIMR Berghofer Medical Research Institute in Brisbane, Australia. This article is reproduced with permission and was first published on February 3, 2026. Gemma Conroy is a freelance science journalist based in Sydney, Australia. First published in 1869, Nature is the world's leading multidisciplinary science journal. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/nasas-next-space-suit-for-artemis-has-out-of-this-world-mobility/'>NASA's next space suit for Artemis has out-of-this-world mobility</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-05 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>An AxEMU (Axiom Extravehicular Mobility Unit) space suit for NASA's Artemis lunar landing missions, as seen during a press conference in Milan, Italy, on October 16, 2024. Launching as early as March for a long-planned there-and-back lunar flyby, NASA's Artemis II mission will bring four astronauts closer to the moon than any humans have been in more than 50 years. Merely traveling so far from Earth is a feat all its own, but the mission is in some ways already overshadowed by its planned follow-up, Artemis III, which is meant to take crew members to the moon's surface to kick off a new, 21st-century era of lunar science and exploration. Of the many obstacles to overcome before any Artemis astronauts begin moonwalking, perhaps the most simple yet significant is figuring out what to wear. One designed for lunar surface operations must protect astronauts from perilous cosmic radiation, extreme temperatures, lung-clogging moon dust and the harsh vacuum of space. It must also carry its own supply of air, as well as water for cooling (and the occasional sip), while also having room for carrying equipment astronauts might need while in orbit or exploring other worlds. Designing a suit that meets all these needs without significantly limiting movement and functionality is an extremely difficult task. But the arduous effort may be essential for achieving longer stays in deep space—as well as building the lunar space station and surface outposts envisioned for future Artemis missions. The bulky white suit looks, in many ways, very similar to previous generations of NASA space suits. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. “There's going to be a lot of new things introduced for the new suit that will be used for moonwalking on Artemis III.” Apollo 11 astronaut Buzz Aldrin has described the landscape of that mission's near-equatorial exploration site as one of “magnificent desolation,” with a nonexistent atmosphere, dust all around and temperatures that swing between hotter and colder than anyone could experience—or endure—on Earth. “This region presents unique challenges including steeper terrain, extreme temperatures and prolonged periods of light and darkness—conditions far harsher than those faced by Apollo astronauts at the lunar equator,” says Victoria Ugalde, a NASA spokesperson. “The space suit will also need to function across different landers, rovers and spacecraft.” To prepare for this desolation, the new suit has robust temperature regulation and is scratch-resistant to reduce damage from jagged rocks and abrasive dust. Whereas Apollo space suits were akin to wearing an inflated balloon that greatly hindered motion, the AxEMU is designed with flexible joints that give astronauts mobility to kneel, jog or even do the splits. Besides allowing space-suit-clad astronauts to fit within multiple different vehicles, the AxEMU's enhanced range of motion should also make working on the lunar surface easier. The new suit's unique joints are shaping up to be a major advancement over previous space suits, but the AxEMU's relatively high weight is still a cause for concern. Axiom Space has not revealed the exact weight of its new suit, but it is more than that of previous space suits, including the Apollo suits, which weighed nearly 200 pounds on Earth. Even under the low gravity of the moon, wearing several hundred pounds still feels heavy and could even cause injuries, NASA astronaut and doctor Mike Barratt told Ars Technica—especially because astronauts are likely to also be toting tools and equipment during lengthy surface operations. “Weight is always a key consideration in space suit design,” NASA's Ugalde says, but she adds that the agency maintains “extremely high confidence” that astronauts will be able to perform mission-necessary tasks when it comes time to don their suits and set foot on the moon. Then, when the Artemis III astronauts return, they will bring with them lessons about how the suit performs in situ, helping to bring forth an even more refined future generation of space suits. “These lessons will shape future Artemis missions, expanding operating ranges and enhancing crew capabilities,” Ugalde says. K. R. Callaway is a freelance journalist specializing in science, health, history and policy. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. There has never been a more important time for us to stand up and show why science matters.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-026-00356-8'>First ‘practical PhDs' awarded in China — for products rather than papers</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-05 12:50:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Civil engineer Zheng Hehui invented Lego-like blocks, used in the Changtai Yangtze River Bridge pylons, as part of his practical PhD.Credit: Cynthia Lee/Alamy Zheng is among the first cohort of Chinese doctoral students to be assessed on the basis of practical achievements that lead to new products, techniques, projects and installations. His invention is being used in a huge cable-stayed rail and road bridge built across the Yangtze River. Since September, at least 11 such ‘practical PhD' students, all engineers, have obtained their doctoral degrees through this route. Universities in other countries offer ‘industrial PhDs', where students work closely with a company, but many of these degrees still require a written thesis. Practical PhDs are part of the Chinese government's broader education reforms, which started in 2010, to cultivate ‘elite engineers' that can help boost innovation in the country. A law passed in 2024 allows universities to let master's and PhD students graduate on the basis of practical achievements. At present, only students in engineering-related subjects are eligible for this no-thesis arrangement. This alternative degree-granting model is important and urgently needed, says Guo Tong, a civil engineer at Southeast University. “[It] can guide students to carry out real research that can solve real-life problems in those industries that carry strategic importance or have technological choke points in China.” Candidates for the practical PhDs have to make prototypes and prove that their inventions can be used in real-life and at scale, according to Sun Yutao, who researches innovation policy at Dalian University of Technology in Dalian, China. The programme is part of China's effort to build a talent pool for key and emerging industries, such as artificial intelligence and semiconductors, to drive innovation, Zhu Xiumei, a deputy director at the Chinese Ministry of Industry and Information Technology, said at a press conference in December. During oral examinations, they are evaluated by a panel comprising both scholars and practising engineers. Tsinghua University, for example, has partnered with 56 companies in 14 key sectors over the past three years, its vice-president Wu Huaqiang said at the press conference. One, Northwestern Polytechnical University in Xi'an, is working with 16 major Chinese groups, including China North Industries Group, which makes weapons and military equipment. These nations are wooing PhD students amid US funding uncertainties China is betting on ‘optical' computer chips — will they power AI? Trump one year on: How six US researchers plan to protect science amid chaos and cuts ‘It means I can sleep at night': how sensors are helping to solve scientists' problems Humanoid robots step up their game: how useful are the latest droids? These nations are wooing PhD students amid US funding uncertainties An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-04191-8'>Anti-CD19 CAR T cells for pediatric patients with treatment-refractory autoimmune diseases</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-05 10:57:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Chimeric antigen receptor (CAR) T cell therapy was recently proposed as a treatment for adults with B-cell-mediated autoimmune diseases (ADs) refractory to conventional immunomodulatory therapy. We present a case series of eight children with severe/refractory AD (four systemic lupus erythematosus, three dermatomyositis, one systemic sclerosis) treated at Ospedale Pediatrico Bambino Gesù, Rome, and University Hospital Erlangen with a single infusion of 1 × 106 kg−1 point-of-care manufactured autologous CD19 CAR T cells (zorpocabtagene autoleucel), in a hospital exemption (HE) program. In Europe, the HE pathway offers the opportunity to treat patients with life-threatening or seriously debilitating disorders who lack valid therapeutic options, using an advanced therapy medicinal product (ATMP) authorized on a nonroutine, single-patient basis. In contrast to the ‘compassionate use' pathway, the ATMP does not necessarily need to have undergone clinical trials or marketing authorization applications. Manufacturing was successful in all patients, yielding several drug product bags. Once infused after lymphodepletion, zorpocabtagene autoleucel cells expanded in vivo, promoting prompt B cell clearance. Grade 1 cytokine release syndrome was reported in six patients, and grade 1 immune effector cell-associated neurotoxicity syndrome was reported in one patient. Late-hematotoxicity was limited to grade 1 in two patients. All these adverse events were manageable and no severe infections occurred. With a median follow-up of 16.5 months (range = 9–24 months), all patients experienced a clinically substantial improvement/resolution of AD, as evidenced by reduction in disease activity scores and signs of reversal of organ damage. This improvement enabled sustained discontinuation of immunomodulators, even after B cell reconstitution. The activation of formal clinical trials enrolling children and adolescents is urgently needed to confirm these preliminary results and to assess the long-term safety of this approach. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Patient identifiable information cannot be made publicly available for reasons of patient privacy and confidentiality. However, nonidentifiable data are available from the corresponding author upon request, based on specific criteria such as the nature of the research request and the necessary ethical approvals. The estimated time frame for responding to such requests is 2–4 weeks. All data requests will be reviewed in accordance with ethical guidelines. Source data are provided with this paper. Müller, F. et al. CD19 CAR T-cell therapy in autoimmune disease—a case series with follow-up. Valenzuela-Almada, M. O. et al. Epidemiology of childhood-onset systemic lupus erythematosus: a population-based study. Arthritis Care Res. Charras, A., Smith, E. & Hedrich, C. M. Systemic lupus erythematosus in children and young people. Ambrose, N. et al. Differences in disease phenotype and severity in SLE across age groups. Massias, J. S. et al. Clinical and laboratory characteristics in juvenile-onset systemic lupus erythematosus across age groups. Md Yusof, M. Y. et al. Management and treatment of children, young people and adults with systemic lupus erythematosus: British Society for Rheumatology guideline scope. US incidence of juvenile dermatomyositis, 1995–1998: results from the National Institute of Arthritis and Musculoskeletal and Skin Diseases Registry. Lu, X., Peng, Q. & Wang, G. Anti-MDA5 antibody-positive dermatomyositis: pathogenesis and clinical progress. Akgün, G. et al. Cardiac evaluation of patients with juvenile dermatomyositis. Volkmann, E. R., Andréasson, K. & Smith, V. Systemic sclerosis. Foeldvari, I. et al. Best clinical practice in the treatment of juvenile systemic sclerosis: expert panel guidance—the result of the International Hamburg Consensus Meeting December 2022. Vasquez-Canizares, N. et al. Developing consensus outcome measures in juvenile systemic sclerosis: a global survey of pediatric rheumatologists and literature review. Kim, H. Juvenile dermatomyositis: updates in pathogenesis and biomarkers, current treatment, and emerging targeted therapies. Northcott, M. et al. Type 1 interferon status in systemic lupus erythematosus: a longitudinal analysis. Arbuckle, M. R. et al. Development of autoantibodies before the clinical onset of systemic lupus erythematosus. Allenbach, Y. et al. Different phenotypes in dermatomyositis associated with anti-MDA5 antibody: study of 121 cases. Kilinc, O. C. & Ugurlu, S. Clinical features of dermatomyositis patients with anti-TIF1 antibodies: a case based comprehensive review. Tansley, S. L. et al. Calcinosis in juvenile dermatomyositis is influenced by both anti-NXP2 autoantibody status and age at disease onset. Hoekstra, E. M. et al. The prognostic power of anti-topoisomerase I and anti-centromere antibodies in systemic sclerosis—a systematic review of the literature. Krustev, E., Clarke, A. E. & Barber, M. R. W. B cell depletion and inhibition in systemic lupus erythematosus. Merrill, J. T. et al. Efficacy and safety of rituximab in moderately-to-severely active systemic lupus erythematosus: the randomized, double-blind, phase II/III systemic lupus erythematosus evaluation of rituximab trial. Brunner, H. I. et al. Safety and efficacy of intravenous belimumab in children with systemic lupus erythematosus: results from a randomised, placebo-controlled trial. Evans, L. S. et al. Povetacicept, an enhanced dual APRIL/BAFF antagonist that modulates B lymphocytes and pathogenic autoantibodies for the treatment of lupus and other B cell-related autoimmune diseases. Oddis, C. V. et al. Rituximab in the treatment of refractory adult and juvenile dermatomyositis and adult polymyositis: a randomized, placebo-phase trial. Ebata, S. et al. Safety and efficacy of rituximab in systemic sclerosis (DESIRES): open-label extension of a double-blind, investigators-initiated, randomised, placebo-controlled trial. Gordon, J. K. et al. Belimumab for the treatment of early diffuse systemic sclerosis: results of a randomized, double-blind, placebo-controlled, pilot trial. Ramwadhdoebe, T. H. et al. Effect of rituximab treatment on T and B cell subsets in lymph node biopsies of patients with rheumatoid arthritis. Leandro, M. J. B-cell subpopulations in humans and their differential susceptibility to depletion with anti-CD20 monoclonal antibodies. Rider, L. G. et al. 2016 ACR-EULAR adult dermatomyositis and polymyositis and juvenile dermatomyositis response criteria-methodological aspects. Rider, L. G. et al. Measures of adult and juvenile dermatomyositis, polymyositis, and inclusion body myositis: Physician and Patient/Parent Global Activity, Manual Muscle Testing (MMT), Health Assessment Questionnaire (HAQ)/Childhood Health Assessment Questionnaire (C-HAQ), Childhood Myositis Assessment Scale (CMAS), Myositis Disease Activity Assessment Tool (MDAAT), Disease Activity Score (DAS), Short Form 36 (SF-36), Child Health Questionnaire (CHQ), physician global damage, Myositis Damage Index (MDI), Quantitative Muscle Testing (QMT), Myositis Functional Index-2 (FI-2), Myositis Activities Profile (MAP), Inclusion Body Myositis Functional Rating Scale (IBMFRS), Cutaneous Dermatomyositis Disease Area and Severity Index (CDASI), Cutaneous Assessment Tool (CAT), Dermatomyositis Skin Severity Index (DSSI), Skindex, and Dermatology Life Quality Index (DLQI). Arthritis Care Res. De Benedetti, F., Diomedi Camassei, F. & Locatelli, F. CAR T-cell therapy in autoimmune disease. Nicolai, R. et al. Autologous CD19-targeting CAR T cells in a patient with refractory juvenile dermatomyositis. Krickau, T. et al. CAR T-cell therapy rescues adolescent with rapidly progressive lupus nephritis from haemodialysis. He, X. et al. Treatment of two pediatric patients with refractory systemic lupus erythematosus using CD19-targeted CAR T-cells. París-Muñoz, A. et al. CD19 CAR-T cell therapy in a pediatric patient with MDA5+ dermatomyositis and rapidly progressive interstitial lung disease. Hanif, M. et al. Contributors to organ damage in childhood lupus: corticosteroid use and disease activity. Del Bufalo, F. et al. Point-of-care fresh CAR T cells for pediatric or young adult BCP-ALL that is relapsed/refractory or in very-high-risk first relapse. Garcia-Prieto, C. A. et al. Epigenetic profiling and response to CD19 chimeric antigen receptor T-cell therapy in B-cell malignancies. J. Natl Cancer Inst. Talleur, A. C. et al. Preferential expansion of CD8+ CD19-CAR T cells postinfusion and the role of disease burden on outcome in pediatric B-ALL. Guzzo, I. et al. Anti-CD19 chimeric antigen receptor T-cell therapy in a highly sensitized patient with focal and segmental glomerulosclerosis. Epperly, R. & Shah, N. N. Long-term follow-up of CD19-CAR T-cell therapy in children and young adults with B-ALL. Elsallab, M. et al. Second primary malignancies after commercial CAR T-cell therapy: analysis of the FDA Adverse Events Reporting System. Lamble, A. J. et al. Risk of T-cell malignancy after CAR T-cell therapy in children, adolescents, and young adults. Youssef, E., Weddle, K., Zimmerman, L. & Palmer, D. Pharmacovigilance in cell and gene therapy: evolving challenges in risk management and long-term follow-up. Hayden, P. J. et al. Management of adults and children receiving CAR T-cell therapy: 2021 best practice recommendations of the European Society for Blood and Marrow Transplantation (EBMT) and the Joint Accreditation Committee of ISCT and EBMT (JACIE) and the European Haematology Association (EHA). Müller, F. et al. CD19-targeted CAR T cells in refractory antisynthetase syndrome. Tur, C. et al. CD19-CAR T-cell therapy induces deep tissue depletion of B cells. Mackensen, A. et al. Anti-CD19 CAR T cell therapy for refractory systemic lupus erythematosus. Merkt, W. et al. Persisting CD19.CAR-T cells in combination with nintedanib: clinical response in a patient with systemic sclerosis-associated pulmonary fibrosis after 2 years. Shimizu, M., Nakaseko, H., Muro, Y. & Iwata, N. Effect of abatacept added to mycophenolate mofetil for refractory calcinosis in juvenile dermatomyositis. Aringer, M. et al. European League Against Rheumatism/American College of Rheumatology classification criteria for systemic lupus erythematosus. Lundberg, I. E. et al. European League Against Rheumatism/American College of Rheumatology classification criteria for adult and juvenile idiopathic inflammatory myopathies and their major subgroups. Van den Hoogen, F. et al. Classification criteria for systemic sclerosis: an American College of Rheumatology/European League Against Rheumatism collaborative initiative. Terrault, N. A. et al. Update on prevention, diagnosis, and treatment of chronic hepatitis B: AASLD 2018 hepatitis B guidance. Bhattacharya, D., Aronsohn, A., Price, J. & Lo Re, V. Hepatitis C guidance 2023 update: American Association for the Study of Liver Diseases– Infectious Diseases Society of America recommendations for testing, managing, and treating hepatitis C virus infection. Regulation (EC) No 1394/2007 of the European Parliament and of the Council of 13 November 2007 on advanced therapy medicinal products and amending Directive 2001/83/EC and Regulation (EC) No 726/2004. Del Bufalo, F., Quintarelli, C. & Locatelli, F. Allogeneic CAR T cells: a new player in the field and the peculiar opportunities of the Hospital Exemption path. & Cometa, M. F. Hospital Exemption in Italy: technical-scientific and regulatory aspects in comparison with Europe. Maschan, M. et al. Multiple site place-of-care manufactured anti-CD19 CAR-T cells induce high remission rates in B-cell malignancy patients. Lee, D. W. et al. ASTCT consensus grading for cytokine release syndrome and neurologic toxicity associated with immune effector cells. Hines, M. R. et al. Immune effector cell-associated hemophagocytic lymphohistiocytosis-like syndrome. Immune effector cell-associated hematotoxicity: EHA/EBMT consensus grading and best practice recommendations. Gladman, D. D., Ibañez, D. & Urowitz, M. B. Systemic lupus erythematosus disease activity index 2000. Standardization of the modified Rodnan skin score for use in clinical trials of systemic sclerosis. La Torre, F. et al. A preliminary disease severity score for juvenile systemic sclerosis. Carsetti, R. et al. Comprehensive phenotyping of human peripheral blood B lymphocytes in healthy conditions. Carsetti, R. et al. Different innate and adaptive immune responses to SARS-CoV-2 infection of asymptomatic, mild, and severe cases. Bagnara, D. et al. A reassessment of IgM memory subsets in humans. The experimental work was supported by grants from Ministero dell'Istruzione, dell'Università e della Ricerca (Ministry of Education, University and Research), including PRIN 2022 (to F.L. ), Ricerca Corrente/5×1000 (to F. Del Bufalo and B.D.A.) ); Ministero delle Imprese e del Made in Italy (MISE)—POS project Life Science Hub Regione Puglia (to F.L. ); European Union—Next Generation EU, Mission 4, Component 2, CUP B93D21010860004, National Center for Gene Therapy and Drugs Based on RNA Technology (to F.L. ); ‘Hub Life Science–Terapia Avanzata (LSH-TA) PNC-E3-2022-23683269–CUP E83C22006230001 from the Italian Ministry of Health—Piano Nazionale Complementare Ecosistema Innovativo della Salute-cod' (PNC-E.3 to F.L. ); ‘PatENts in tHe medicAl aNd for CompaniEs—ENHANCE' (to B.D.A. ); Deutsche Forschungsgemeinschaft (German Research Foundation)—DFG (TRR221 to G.S. We thank Miltenyi Biomedicine for providing the viral supernatant used in the manufacture of zorpocabtagene autoleucel. Our deepest gratitude goes to all the patients and their families, and to the patient associations that support this research—Associazione ‘Raffaele Passarelli' Onlus (B.D.A. ); Associazione ‘Un … due … tre … Alessio' (C.Q. These authors contributed equally: Marco Becilli, Markus Metzler, Claudia Bracaglia. These authors jointly supervised this work: Georg Schett, Fabrizio De Benedetti, Franco Locatelli. Department of Hematology/Oncology, Cell and Gene Therapy, Scientific Institute for Research, Hospitalization and Healthcare (IRCCS), Bambino Gesù Children's Hospital, Rome, Italy Marco Becilli, Francesca Del Bufalo, Chiara Rosignoli, Pietro Merli, Daria Pagliara, Mattia Algeri, Maria Giuseppina Cefalo, Valentina Bertaina, Matilde Sinibaldi, Giuseppina Li Pira, Monica Gunetti, Biagio De Angelis, Concetta Quintarelli & Franco Locatelli Department of Paediatrics and Adolescent Medicine, Universitätsklinikum Erlangen and Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany Markus Metzler, Tobias Krickau & Nora Naumann-Bartsch Deutsches Zentrum Immuntherapie, Universitätsklinikum Erlangen and Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany Markus Metzler, Tobias Krickau, Nora Naumann-Bartsch, Michael Aigner, Simon Völkl, Fabian Müller, Andreas Mackensen & Georg Schett Division of Rheumatology, Scientific Institute for Research, Hospitalization and Healthcare (IRCCS), Bambino Gesù Children's Hospital, Rome, Italy Claudia Bracaglia, Rebecca Nicolai, Emiliano Marasco, Virginia Messia, Antonella Insalaco & Fabrizio De Benedetti Department of Health Sciences, Magna Graecia University, Catanzaro, Italy Advanced Cardiothoracic and Fetal Imaging Unit, Scientific Institute for Research, Hospitalization and Healthcare (IRCCS), Bambino Gesù Children's Hospital, Rome, Italy Complex Operational Unit (UOC) Pathological Anatomy, Scientific Institute for Research, Hospitalization and Healthcare (IRCCS), Bambino Gesù Children's Hospital, Rome, Italy Francesca Diomedi Camassei Department of Internal Medicine 5—Haematology and Oncology, Universitätsklinikum Erlangen and Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany Michael Aigner, Simon Völkl, Fabian Müller & Andreas Mackensen Transfusion Unit, Department of Laboratories, Scientific Institute for Research, Hospitalization and Healthcare (IRCCS), Bambino Gesù Children's Hospital, Rome, Italy Department of Pediatrics, Institute of Clinical Sciences, The Sahlgrenska Academy, University of Gothenburg, Gothenburg, Sweden Miltenyi Biomedicine, Bergisch Gladbach, Germany Department of Clinical Medicine and Surgery, University of Naples Federico II, Naples, Italy Department of Internal Medicine 3—Rheumatology and Immunology, Universitätsklinikum Erlangen and Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany Department of Life Sciences and Public Health, Catholic University of the Sacred Heart, Rome, Italy Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed the treatments and analyses, clinically managed the patients, participated in the data collection, interpreted and analyzed the data, and wrote the paper. G.S., F. De Benedetti and F.L. designed the treatments and analyses, supervised the project conduction and the clinical management of the patients, interpreted and analyzed the data, and wrote the paper. R.N., T.K., F. Del Bufalo, C.R., E.M., N.N.‑B., V.M., P.M., D.P., M. Algeri, A.I., S.B., F.M. and A.M. clinically managed the patients, participated in the data collection, and interpreted and analyzed the data. A.S. performed and interpreted the radiological images. performed pathological examinations of kidney biopsies. performed the flow-cytometry analyses of B cell reconstitution and B cell subsets. performed patients' immune monitoring. manufactured the CAR T cells. performed apheresis collection. performed apheresis manipulation. supervised all the immune-monitoring studies, interpreted and analyzed the data, and wrote the paper. contributed vital reagents. All authors reviewed the paper for final approval before submission. Correspondence to Franco Locatelli. has participated in advisory boards for Novartis. has received consulting fees from Novartis and Pfizer; speaker honoraria from Novartis, Pfizer and Kyowa Kirin; and meeting support from Pfizer, Novartis and AbbVie. P.M. reports personal fees from Sobi, Miltenyi and Amgen. M. Aigner has received research grants from Miltenyi Biotec and Kyverna; consulting fees from Miltenyi Biotec; speaker honoraria from Miltenyi Biotec and Raumedic; expert testimony payments from RUHR‑IP; travel support from Miltenyi Biotec and material support from Miltenyi Biotec. has received research support from the Interdisciplinary Center for Clinical Research at University Hospital Erlangen‑Nuremberg (grant D43). is an employee of Miltenyi Biomedicine. has received research grants from Kite/Gilead; consulting fees from AbbVie, ArgoBio, AstraZeneca, Bristol Myers Squibb, CRISPR Therapeutics, Janssen, Kite and Novartis; speaker honoraria from AbbVie, ArgoBio, AstraZeneca, Bristol Myers Squibb, CRISPR Therapeutics, Janssen, Kite, Kyverna, Miltenyi Biomedicine, Novartis and Sobi; has served on an advisory board for Bristol Myers Squibb and received research funding from Deutsche Krebshilfe (grant 0113695). A.M. has received research grants from Miltenyi Biomedicine and Kyverna; consulting fees from BMS/Celgene, Kite/Gilead, Novartis, BioNTech, Miltenyi Biomedicine and Century Therapeutics; speaker honoraria from BMS/Celgene, Kite/Gilead, Novartis and Miltenyi Biomedicine; and meeting support from AbbVie and Janssen. has received speaker honoraria from Novartis, Bristol Myers Squibb, Kyverna and Cabaletta. F. De Benedetti declares unrestricted research grants from Sobi, Sanofi, Regeneron, Roche, Elixiron and Novartis; participation on the Data Safety Monitoring Board for Regeneron and consulting fees from Sobi, Novartis and Apollo. serves on the scientific advisory board of Amgen, Novartis, Sanofi and Vertex; and speaker's bureau for Miltenyi Biomedicine, Amgen, Novartis, Bristol Myers Squibb, Gilead, MEDAC and Sobi. The other authors declare no competing interests. Nature Medicine thanks Shaun Jackson and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Saheli Sadanand, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Multiparametric flow-cytometry analysis of the starting materials collected from OPBG patients to characterize the memory profile of CD4+ and CD8+ T cells. Proportion of naive (CCR7+/CD45RO− and CD62L+/CD45RA+), central memory (CM, CCR7+/CD45RO+), effector memory (EM, CCR7−/CD45RO+), and terminally differentiated effector memory re-expressing CD45RA (TEMRA, CCR7−/CD45RO−) cells among CD4+ (upper plot) and CD8+ T cells (lower plot). Distribution of ‘naive', central memory (CM), effector memory (EM) T cells, and terminally differentiated effector memory T cells re-expressing CD45RA (TEMRA) in the CD4+CAR+ and CD8+CAR+ T cells in PB (a) and in BM (b), according to different time points after zorpocabtagene-autoleucel infusion. BM samples were collected only in OPBG patients. Each dot represents an individual evaluable patient. Boxes represent the first quartile, median and forth quartile distribution, and boundaries of lower and upper whiskers showing minimum to maximum values. CAR− T cell reconstitution is shown with granular data from individual patients (upper panel). The distribution of CD4+ and CD8+ T cells is depicted as box and whiskers plots in the middle panel (all patients) and lower panel (excluded cSLE-OPBG003), according to different time points, with each dot indicating an individual value from a single patient, boxes representing first quartile, median and forth quartile distribution, and boundaries of lower and upper whiskers showing minimum to maximum values. CD3+CAR− cells recovered in all patients starting from week 4 after lymphodepletion. Both CD4+CAR− and CD8+CAR− increased over time, with a homogeneous distribution in all patients but cSLE-OPBG003 (lower panel). Indeed, in this patient, CD3+CAR− cells achieved significantly higher levels compared to other patients starting from month 6 after zorpocabtagene-autoleucel infusion, with a predominance of CD8+ cells. Importantly, CAR gene copies were persistently undetectable in peripheral blood from month 3 throughout the follow-up. A lymphoproliferative disease was excluded. Percentage of bone marrow PCs expressing CD19+38high138−, 19+38high138+ and CD19−38high138+ are depicted for each OPBG patient, according to different time points. Bone marrow aspirate was not performed before treatment in patient cSLE-OPBG001, and the sample was not available at month 12 for patient JDM-OPBG002. a, Box plot representing the peak levels of IFN-γ, IL-10, IL-6 and TNF. The higher levels of IL-6 refer to patient JDM-OPBG004 and cSLE-FAU001. Individual values derived from individual patients are reported, with boxes representing first quartile, median and forth quartile distribution. Boundaries of lower and upper whiskers show minimum and maximum values. Grade 1 cytokine release syndrome occurred in both patients, while grade 1 immune effector cell-associated neurotoxicity syndrome was experienced by JDM-OPBG004. Tocilizumab was administered to cSLE-FAU001 on day 5. Highest levels of IFN-γ, IL-10 and TNF have been shown in patient JDM-OPBG004. b, Evolution of serum levels of IL-6, IFN-γ, TNF and IL-10 in all patients infused with zorpocabtagene-autoleucel. Serum IL-10 levels were systematically studied only in patients treated at OPBG. a,b, Spaghetti plot representing the evolution of the CRP (a) and ferritin (b) in each patient treated with zorpocabtagene-autoleucel, over the first 4 weeks after infusion. Horizontal straight lines represent the period during which cytokine release syndrome manifested. The evolution of IgG, IgA and IgM levels is shown individually for each patient, according to available data at different time points. The vertical black dotted line indicates the zorpocabtagene-autoleucel infusion, while the horizontal red dotted lines represent the lower limit of normal values for each type of immunoglobulin. Hematotoxicity after zorpocabtagene-autoleucel infusion. Representation of early and late immune effector cell-associated hematotoxicity (ICAHT)62 in each patient treated with autologous zorpocabtagene-autoleucel. Supplementary Tables 1–4 and Figs. Supporting data for Supplementary Figs. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Becilli, M., Metzler, M., Bracaglia, C. et al. Anti-CD19 CAR T cells for pediatric patients with treatment-refractory autoimmune diseases. Version of record: 05 February 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41590-025-02412-3'>Antigen specificity of clonally enriched CD8+ T cells in multiple sclerosis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-05 10:21:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. CD8+ T cells are the dominant clonally expanded lymphocyte population in multiple sclerosis (MS) lesions but their clonal identity, function and antigen specificity are not well understood. A comprehensive single-cell RNA-sequencing and T cell receptor-sequencing analysis of the cerebrospinal fluid and blood from individuals in the MS and control cohorts revealed a subset of 23 highly expanded and activated CD8+ T cell clonotypes that were enriched predominantly in the cerebrospinal fluid in the MS cohort. Using unbiased and targeted antigen discovery approaches, six CD8+ T cell clonotypes recognizing Epstein–Barr virus (EBV) antigens and multiple novel mimotopes were identified. Although the majority of mimotopes did not elicit functional responses, three of the expanded CD8+ T cell receptors from patients with MS were reactive to EBV. EBV DNA and transcripts were detected in cerebrospinal fluid, including in patients with MS who had highly expanded EBV-specific CD8+ T cells. These findings shed vital insight into the role of CD8+ T cells in MS and support an important role of EBV in MS immunopathology. Multiple sclerosis (MS) is an inflammatory demyelinating condition of the central nervous system (CNS) characterized by substantial T cell involvement1. Both CD4+ and CD8+ T cells are found within the perivascular spaces as well as in the parenchyma of MS lesions2,3. CD8+ T cells are enriched and clonally expanded relative to CD4+ T cells in MS lesions4,5, suggesting local antigen-driven expansion. Specific major histocompatibility complex (MHC) I alleles also alter MS susceptibility6, providing additional support for a critical role of cytotoxic CD8+ T cells in MS biology. The goal of this study was to identify T cells, particularly CD8+ T cells, that are uniquely expanded in the CNS and determine their phenotypic characteristics and antigen specificity. Acquisition of CNS tissue to analyze the infiltrating T cell response in MS is invasive and rarely performed early in the disease course. Studies of the CSF repertoire have indeed indicated high clonal overlap with expanded T cell populations in MS lesions8,9. Single-cell-sequencing technologies provide powerful opportunities for deep phenotyping and clonal characterization of T cells in numerous diseases, including MS10,11,12,13,14, yet detailed analyses of CSF-expanded T cells and their antigen specificity in MS are lacking. In this study, CSF and blood from individuals with early untreated MS as well as control participants were subjected to single-cell RNA-sequencing (scRNA-seq) paired with single-cell T cell receptor (TCR)-sequencing (scTCR-seq). After identifying a subset of CSF-expanded CD8+ T cells in the patients with MS, their antigen specificity was investigated using a combination of unbiased and targeted antigen discovery methodologies. A total of 18 individuals were enrolled in the study—11 with relapsing-remitting MS, two with clinically isolated syndrome (CIS), two with other neuro-inflammatory disorders (OND) and three healthy controls (HCs). The demographics of the four cohorts are presented in Table 1 and those of each individual in Supplementary Table 1. All of the patients in the MS and CIS (MS/CIS) cohort were treatment-naive (that is, no previous history of immunomodulatory or immunosuppressive therapies) at the time of sample collection but one of the patients with OND was on immunotherapy with a TNF-α inhibitor. Paired peripheral blood and CSF were collected on the same day for each study participant. Freshly acquired samples comprised of all unseparated cell subsets underwent paired scRNA-seq and scTCR-seq using 10X Genomics 5′ library preparation kits to permit combined single-cell transcriptional phenotyping and TCR clonal analysis. The scRNA-seq data of all participants in this study were previously published15. All major immune cell subsets were readily identified from the scRNA-seq data, with T cell clusters comprising the largest fractions (Fig. To characterize conventional TCRαβ T cells, all subsequent analyses focused on only those T cells with paired scRNA-seq and scTCR-seq data (Fig. A total of 48,468 individual T cells were identified from the blood and CSF across all participants (Fig. As expected, TCR-associated genes (CD3E and CD3D) were highly upregulated with minimal expression of non-T cell-associated genes (for example, CD19; Extended Data Fig. We identified 33,349 CD4+ and 15,119 CD8+ T cells expressing paired TCRαβ genes (described in Methods) for analysis from the combination of blood and CSF of all 18 participants (Supplementary Table 2). a, Major immune cell subsets from combined blood and CSF of all patients were identified by scRNA-seq. b,c,g, T cells were defined after integration of the scRNA-seq and scTCR-seq data, allowing segregation of T cells by CD4/CD8 status (b), compartment (CSF; c) and disease status (g). Differential gene expression comparisons were performed using a two-sided Wilcoxon ranked-sum test with Bonferroni correction (adjusted P). Genes with adjusted P < 0.05 are indicated in red. UMAP, uniform manifold approximation and projection. Pseudotime analysis of T cells in the CSF revealed distinct populations of T cells largely segregated based on T cell subsets (that is, CD4+ or CD8+), highlighting the distinct transcriptional signatures associated with different T cell states and functions (Fig. In contrast, peripheral blood-derived CD8+ T cells expressed significantly higher levels of FOS, JUN, DUSP1 and GADD45B, indicating an alternate activation state. In a comparison of only memory (CD27-expressing) CD8+ T cells, there was significant upregulation of genes associated with T cell activation (HLA-DRA), chemokines (CCL4 and XCL1) and cholesterol metabolism (LDLR and SQLE), and downregulation of genes associated with T cell signaling (FOS, FOSB, JUN and JUNB) in the CSF relative to the blood (Extended Data Fig. In the CSF, CD4+ T cells (Supplementary Table 3) had significantly increased expression of genes similar to their CD8+ T cell counterparts (ITGB1, ITGA4, CXCR3, GZMA, GZMK and CD2) as well as distinct genes (JUN, FOS, DUSP1, CCR7 and HCST). Given the disproportionate number of participants in the different disease categories (Table 1), we grouped the patients with MS or CIS (MS/CIS; n = 13) and performed differential gene expression against the combined group of HCs and patients with OND (HC/OND; non-MS group; n = 5; Fig. In CD8+ T cells combined from the peripheral blood and CSF, various genes were differentially expressed between the MS/CIS and HC/OND groups (Fig. Overall, these data suggest that both CD8+ and CD4+ T cells in the patients with MS/CIS are more activated with increased effector functions and tissue homing capacity than the HC/OND cohort, consistent with other studies11,12. The clonal repertoire of T cell subsets was compared across compartments (that is, peripheral blood versus CSF) and across disease states (that is, MS/CIS versus HC/OND). T cell clonotypes were defined as T cells sharing identical V and J genes and CDR3 amino-acid sequences for paired TCRαβ sequences similar to previous studies10,16. A total of 31,756 unique CD4+ T cell clonotypes and 10,825 unique CD8+ T cell clonotypes were identified from all individuals (Supplementary Table 2). The CD4+ and CD8+ T cell diversities (measured by Shannon entropy) were significantly higher in MS/CIS compared with HC/OND in the peripheral blood and CSF (Extended Data Fig. The diversity of CD4+ T cells was significantly higher in blood compared with CSF but not for CD8+ T cells (Extended Data Fig. These findings suggest a more diverse array of CD4+ and CD8+ T cell clonotypes are present in both the blood and CSF of patients with MS/CIS. T cell clonal expansion is a hallmark of previous antigen encounter; therefore, our analysis focused on T cell clonality in the CSF. Although small fractions of clonally expanded CD4+ T cells were observed in the CSF, much larger populations of highly and moderately expanded CD8+ T cells were observed in similar proportions in the MS/CIS and HC/OND groups (Fig. CD4+ and CD8+ T cell clonal expansion in the CSF was overall similar between the patients with CIS or MS (Supplementary Table 6). b, Clonal frequencies of all T cell clonotypes in the CSF and blood that were highly expanded T cells and enriched at least twofold more frequently than the blood of the same individual are highlighted in red. ; unpaired two-tailed Student's t-test with Welch's correction; NS, not significant. e, Analysis of differential gene expression between highly expanded and unexpanded T cells in the CSF. Two-sided Wilcon ranked-sum test with Bonferroni correction; genes with adjusted P < 0.05 are indicated in red. f,g, Unbiased clustering of all CSF T cells (f; the 11 distinct clusters are numbered) overlaid with highly expanded/enriched T cells (g). To delineate between T cells expanded similarly in the blood and CSF versus those preferentially expanded in the CSF, the abundance of all T cell clonotypes in the blood and CSF was compared in all individuals. The overwhelming majority of T cell clonotypes were detected in the blood or CSF only, whereas only about 1.5% of all clonotypes were found in both compartments (Fig. We postulated that highly expanded T cell clonotypes (that is, CSF frequency of ≥0.75%) that were enriched in the CSF relative to the peripheral blood were more likely to be responsive to local antigens in the CSF and/or CNS (albeit not necessarily CNS-specific antigens). Enriched CSF-expanded T cell clonotypes were defined as those with a CSF frequency at least twofold higher than the peripheral-blood frequency from the same individual. This yielded 33 highly CSF-enriched and expanded T cell clonotype varying from approximately twofold to more than 100-fold higher frequencies in the CSF relative to peripheral blood (Fig. More than 70% of the highly expanded and CSF-enriched T cell clonotypes in the CSF were CD8+ T cells. Although there were no statistically significant differences in the mean frequencies of highly expanded CSF-enriched T cell clonotypes between MS/CIS and HC/OND, only participants in the MS/CIS cohort had CSF-enriched T cells with frequencies greater than 2% (Fig. One patient with MS (patient identifier (ID), MS6) had 11 highly enriched T cell clonotypes, the majority of which were CD8+ T cells, which encompassed nearly 20% of their CSF repertoire (Supplementary Table 7). These findings therefore provide strong support for robust oligoclonal CD8+ T cell expansion and enrichment in the CSF, with the greatest expansion found in MS/CIS. Highly expanded and unexpanded T cells in the CSF were compared by scRNA-seq analysis. Substantial differential gene expression changes were observed in highly expanded T cells in comparison to their unexpanded counterparts (Fig. In particular, genes associated with cytotoxic CD8+ T cell function (CD8A, CD8B, NKG7, KLRD1, GZMA, GZMH, GZMM, GZMK and EOMES) and chemotaxis (CCL5 and CCL4) were significantly increased in highly expanded T cells, whereas genes associated with naive status were significantly reduced (IL7R, LTB and LDHB). A tissue-resident-memory (TRM) phenotype of CSF-expanded T cells coexpressing CD69 and IGTAE was confirmed by the reduced expression of KLF2 and S1PR1 genes (Extended Data Fig. In contrast, CSF-unexpanded T cells expressed higher levels of genes associated with central memory/nonactivation (SELL, CCR7, IL7R, TCF7 and LEF1) as well as the integrin gene ITGB1. The overwhelming majority of the enriched and expanded clonotypes were found in cluster 1, which was defined by a significantly increased expression of a number of genes associated with cytotoxic effector CD8+ T cells, including CD8A, CD8B, PLEK, DUSP2, EOMES, GZMK, GZMA, GZMH, PRF1, NKG7, CCL5 and CCL4 (Supplementary Table 10). Overall, these data indicate that highly clonally expanded T cells in the CSF express gene profiles indicative of substantial antigen experience, cytotoxicity and distinct tissue homing capacities. Only 21 identical TCRs (that is, same V and J genes and CDR3 amino-acid sequences for the paired α and β chains) were found between the peripheral blood of different individuals and another three that were identical between the blood and CSF of different individuals, irrespective of disease status (Fig. To further assess clonal relationships, Grouping of Lymphocyte Interactions with Paratope Hotspots 2 (GLIPH2) was employed, an algorithm to help identify TCRs with potentially shared specificity based on sequence similarity within the CDR3β region18. All CSF T cell clonotypes were analyzed using GLIPH2 and the output was then queried against the 33 CSF high-enriched CDR3 sequences. Using this approach, 19 clonally related networks comprised of a total of 44 clonotypes were identified (Fig. Most of the networks comprised two related clonotypes and two networks were comprised of five clonotypes each. Almost all networks consisted of clonotypes from the same individual and were identified primarily among the individuals with MS or CIS (Fig. Nearly all of the clonally related T cells were CD8+ T cells (Supplementary Table 11), suggesting potential shared antigen specificity. a, Number of unique or shared T cell clonotypes between different compartments from all study participants. b, GLIPH2 analysis of highly expanded and enriched T cell clonotypes in the CSF compared with all other CSF T cell clonotypes. Clonal size is indicated by node size and clonally related populations are connected by lines. c, The indicated GLIPH2-aligned CD8+ TCRs to the EBV-specific TCRs 86333_1456 (left) and 69317_24418 (right) were expressed in reporter Jurkat cells and tested for reactivity to the corresponding EBV peptides (n = 3) or no-stimulation control (n = 3). FLRGRAYGL (EBV EBNA3A193–201) was presented by HLA-B*08:01-expressing APCs (left) and EPLPQGQLTAY (EBV BZLF154–64) was presented by HLA-B*35:01-expressing APCs (right). Several different strategies were undertaken (Fig. An unbiased antigen discovery approach was first employed using a peptide:MHC (pMHC) yeast display library in which approximately 1–10 × 108 random peptides are displayed on a given MHC allele for probing recognition against individual TCRs19. Four TCRs (three MS/CIS and one HC) demonstrated substantial enrichment of specific peptides from three different MHC I libraries (Supplementary Table 12). a, Individual TCRαβ pairs were cloned into plasmids and expressed in primary human CD8+ T cells by nonviral CRISPR knock-in. Candidate antigens for testing specificity were identified in three parallel strategies, screened by pMHC tetramer binding and validated by cytokine production to cognate antigen. b, Candidate antigens for four TCRs identified by pMHC yeast display (unbiased antigen discovery) were tested for tetramer binding and cytokine reactivity experiments. To validate these candidate antigens, each TCR was expressed individually in primary human CD8+ T cells by nonviral CRISPR–Cas9-mediated TCR knock-in (Extended Data Fig. Candidate TCR-expressing CD8+ T cells were then probed for antigen specificity using pMHC I tetramers loaded with peptides identified from the yeast display library screen. Three of the four tested TCRs demonstrated robust tetramer binding to most or all of the library-identified peptides (Fig. The ability of CD8+ T cells expressing these TCRs to respond functionally to the same antigens was tested by intracellular cytokine stimulation using antigen-presenting cells (APCs) expressing the relevant MHC I allele. Strikingly, only TCR clonotype 54189_65570 demonstrated cytokine production to peptide CSERNPWTFY; none of the other TCR-expressing CD8+ T cells were functionally responsive to the respective yeast display-derived peptides (Fig. As nearly all of the yeast display peptides identified by pMHC I tetramers were not naturally occurring (that is, mimotopes), the analysis was extended to an array of foreign and human peptide homologs (Supplementary Table 12). Varying degrees of tetramer binding were observed depending on the TCR tested but none of the peptide homologs elicited cytokine responses above background (Extended Data Fig. Thus, although the unbiased antigen discovery approach yielded novel mimotopes of several CD8+ T cell clonotypes detectable by pMHC I tetramer binding, none exhibited functional reactivity to naturally occurring antigens. One CD8+ T cell clonotype (86333_1456) from participant MS8 demonstrated an exact match to both TCRα and -β sequences with a well-described Epstein–Barr virus (EBV) epitope, EBNA3A193–201 (FLRGRAYGL; Supplementary Table 13), which is restricted by HLA-B*08:01, an allele carried by this individual (Supplementary Table 1). This identical clonotype was also moderately expanded in the CSF from an individual with Alzheimer's disease17. A second CD8+ T cell clonotype (69317_24418) from participant MS6 was a near-exact match to a TCR specific for the EBV epitope BZLF154–64 (EPLPQGQLTAY) restricted by HLA-B*35:01, an allele also carried by this individual. These TCRs were expressed in primary human CD8+ T cells as described earlier and their specificity was again tested by pMHC I tetramer analysis. To confirm functional reactivity, primary human CD8+ T cells expressing each of these TCRs were stimulated with APC lines expressing cognate HLA and loaded with or without cognate EBV peptide. Each TCR demonstrated clear cytokine production to the relevant EBV peptide (Fig. 5b,c), confirming both CSF-expanded and enriched CD8+ T cell clonotypes are specific to EBV antigens. a,b, Representative flow cytometry analysis of tetramer binding (a) and cytokine production (b) of three TCRs (from patients with MS) with predicted reactivity to four different viral epitopes. c, Percentage tetramer binding and cytokine reactivity of each TCR. d, Frequencies and degree of enrichment of the three EBV-specific clonotypes (highlighted in red) relative to all other highly enriched and expanded T cell clonotypes. e, TCR sequencing alignment of expanded CD8+ TCRs to EBV- and CMV-specific TCRs in the peripheral blood (PB) and CSF. f, EBV-specific TCR alignment of all expanded CD8+ T cell clonotypes in the PB and CSF of MS/CIS and HC/OND study participants. g,h, Summary of functional reactivity of Jurkat cells expressing the indicated TCR specific for EBV EBNA3A193–201:B*08:01 (g) or EBV BZLF154–64:B*35:01 (h) to the indicated peptides (n = 3 EBV peptides and 2 peptide homologs). frequency of CD69 and NFAT–mCherry double-positive cells with the no-stimulation background control subtracted. Amino-acid differences between cognate EBV peptides (leftmost of each plot) and self-peptide homologs are indicated in red. Each peptide was tested in a minimum of two independent experiments. In light of these findings, the possibility that additional CSF-enriched and expanded CD8+ T cells may be specific for viral antigens, in particular EBV, was further explored. Note that severe acute respiratory syndrome coronavirus 2 peptides were not tested as all samples were collected previous to the coronavirus disease 2019 pandemic. Nineteen TCRs were tested against panels of pMHC I tetramers loaded with previously identified immunodominant viral epitopes restricted by HLA matching that of the TCR donors. A total of 98 peptides restricted by eight different MHC I alleles were screened (Supplementary Table 14). Each TCR was screened with individual pMHC tetramers, except in the case of HLA-A*02:01 where tetramers were pooled in groups of five due to the large number of candidate peptides. Each TCR was tested against the indicated peptides a minimum of two times using two different T cell donors for TCR expression. No specific tetramer signal was observed for any of the TCRs to any of the peptides beyond the two EBV epitopes already identified for TCRs 86333_1456 and 69317_24418 (Extended Data Fig. Although TCR 86333_17042 from participant MS8 showed an identical match for a TCRβ sequence specific for EBV and cytomegalovirus (CMV) antigens with corresponding MHC I alleles (Supplementary Table 13), it did not show any notable tetramer binding or cytokine reactivity to either viral antigen (Fig. Potential reactivity to EBV was further assessed for the other CSF-expanded and enriched CD8+ T cells given the EBV reactivity of two clonotypes. EBV-transformed lymphoblastoid cell lines (LCLs) were employed to survey a wide array of processed EBV epitopes across a multitude of HLA alleles. NFAT–mCherry-expressing Jurkat cells transfected with the CD8 co-receptor and a single candidate TCR were co-cultured with partially HLA-matched LCLs. Fully HLA-mismatched LCLs and TCR-expressing Jurkat cells from HLA-mismatched patients were used as negative controls. This enabled testing of 16 additional candidate TCRs against at least two different LCLs matching 3–6 MHC I alleles (Supplementary Table 15). Almost all TCRs showed no detectable reactivity; however, TCR 94669_8198 from patient MS27 demonstrated a clear reproducible response to LCLs only when matching the HLA-A*29:02 allele (Fig. No response was elicited from primary B cells from the same donor used to generate the LCLs, indicating this is very likely to be an EBV-specific response rather than a B cell self-antigen or alloreactive response. To identify a potential specific EBV epitope, Jurkat reporter cells expressing TCR 94669_8198 were tested against seven candidate EBV peptides identified from The Immune Epitope Database (FLYALALLL, VFGQQAYFY, AYSSWMYSY, FVYGGSKTSLY, VFSDGRVAC, VSSDGRVAC and ILLARLFLY) presented by HLA-A*29:02-expressing APCs. No functional response was elicited, however, indicating reactivity to a still unspecified EBV epitope (Extended Data Fig. a, Representative flow cytometry analysis of Jurkat reporter cells expressing the indicated TCR and CD8 co-receptor that were co-cultured with partially HLA-matched (matching allele indicated in red) EBV-transformed LCLs. Reactivity was assessed by coexpression of CD69 and NFAT–mCherry. Fully HLA-mismatched LCLs and mismatched TCR-expressing Jurkat cells were used as negative controls. b, Representative flow cytometry analysis of Jurkat reporter cells expressing TCR 94669_8198 were co-cultured with LCLs or primary uninfected B cells from the same donor. c, Summary of all candidate TCRs tested and the corresponding matching MHC I alleles expressed by different LCL lines. The mCherry+CD69+ signal of a given TCR-expressing Jurkat cell line co-cultured with partially MHC-matched LCLs was normalized to the signal observed from completely MHC-mismatched LCLs (left) or mismatched TCR-expressing Jurkat cells (right), which was reported as fold change (FC). We also explored EBV specificity for CD8+ T cell clonotypes that were aligned to the highly CSF-expanded and enriched CD8+ T cell clonotypes that were EBV-reactive (Fig. Only GLIPH2-derived TCR sequences from CD8+ T cells that shared the same MHC I allele as that of the aligned EBV-specific clonotype were tested (Supplementary Table 16). Unlike the EBV-specific clonotype 86333_1456, the GLIPH2-aligned TCRs 86333_17042 and 86333_18519 (all from MS8) showed no detectable reactivity to EBNA3A peptide FLRGRAYGL restricted by HLA-B*08:01 (Fig. Strikingly, TCR 53778_13077 was found by VDJdb search to exactly match a TCR previously demonstrated to be specific for EPLPQGQLTAY23. This specificity was validated by stimulating Jurkat reporter cells expressing TCR 53778_13077 with or without EPLPQGQLTAY presented by HLA-B*35:01-expressing APCs. Notably, TCR 53778_13077 was moderately expanded in the CSF (0.35%) and enriched approximately threefold relative to the blood of MS10 (Supplementary Table 2). These findings indicate that at least three highly expanded CD8+ T cells in the CSF of patients with MS are specific for EBV, but the specificities for most of the enriched T cell clonotypes remain unknown (Fig. 2b and Supplementary Table 2) or the three patients with MS and EBV-specific clonotypes (Extended Data Fig. To assess whether EBV specificity among expanded CD8+ T cells in the CSF was overall enhanced compared with the blood, TCR sequencing alignment (identical V genes, J genes and CDR3 amino-acid sequences for paired TCRα and -β chains) was performed against all expanded CD8+ TCR sequences (>1 TCR per clonotype) in VDJdb20 with CMV used as a comparison. EBV- and CMV-aligned CD8+ TCR sequences in the blood were very similar; however, EBV specificity was markedly increased in the CSF, whereas no CMV specificity was found (Fig. This provides additional support that EBV-specific CD8+ T cell expansion is uniquely increased in the CSF in MS. CD8+ T cells of different viral specificities can exhibit distinct phenotypic characteristics24. The transcriptional profiles of the three CSF-expanded CD8+ T cell clonotypes were therefore compared against all other CSF-expanded and enriched CD8+ T cells. Differential gene expression analysis revealed three genes that were significantly increased in the EBV-specific CD8+ T cells, most notably CXCR5 (Supplementary Table 17), which is associated with migration to B cell follicles and control of chronic infections25. Specific genes associated with memory differentiation, migration and tissue residency were also compared. Consistent with previous reports24, CD27 was particularly abundant in the expanded EBV-specific CD8+ T cells (Extended Data Fig. These findings therefore indicate a distinct phenotype of CSF-expanded CD8+ T cells that are specific for EBV. Rather than expressing TRM markers and genes associated with lymphocyte recruitment, these findings suggest that EBV-expanded CD8+ T cells in the CSF are an effector population associated with follicular homing and B cell interactions. To determine whether the two EBV peptide-reactive CD8+ T cell clonotypes may be cross-reactive against self-antigens, the TCRs were screened against panels of self-peptides with partial sequence homology (Supplementary Table 18). Using NFAT–mCherry-expressing Jurkat cells transfected with the CD8 co-receptor and TCR 86333_1456 or 69317_24418, high reactivity to the respective EBV peptides was confirmed (Fig. Strikingly, no notable reactivity was observed for any of the self-peptide homologs. Although this does not entirely exclude the possibility for self-antigen cross-reactivity, it raises the possibility that the CSF enrichment of these clonally expanded CD8+ T cells may be driven by reactivity to EBV. To assess for the presence of EBV in CSF, DNA was extracted from the CSF of all study participants and PCR amplified with primers specific for the EBV BZLF1 gene (Extended Data Fig. The PCR amplicons were Sanger sequenced for further confirmation (Supplementary Table 19). In this manner, EBV DNA was detected in 6/13 MS/CIS samples and 2/5 HC/OND samples (Supplementary Table 20), including patients MS6 and MS8 who also harbored highly expanded EBV-reactive CD8+ T cells in their CSF. The presence of EBV DNA was further quantified by droplet digital PCR (ddPCR) via amplification of the EBER2 gene normalized to a housekeeping gene. EBV was detected in the CSF of nearly all patients and control study participants, although the relative abundance varied widely with the highest levels found in patients with MS/CIS (Fig. EBV transcripts to several latent and lytic genes were also assessed by complementary DNA quantification. EBER2 cDNA was overall less detectable than DNA and there was no significant difference between MS/CIS and HC/OND (Fig. EBNA3A (latency III gene) and BZLF1 (early lytic gene) were mostly undetectable with no significant difference between the two cohorts (Fig. This therefore suggests that EBV reactivation is enhanced in the CSF in patients with MS/CIS, which may drive the expansion of EBV-specific CD8+ T cells. a, Summary of EBV DNA ddPCR results from CSF supernatant in which EBER2 was normalized to a housekeeping gene (MS/CIS, n = 13; HC/OND, n = 5). b, EBV cDNA for each of the indicated genes were measured by ddPCR and normalized to a housekeeping gene. MS/CIS and HC/OND samples were compared using an unpaired two-tailed Student's t-test with Welch's correction; NS, not significant; n = 13 for MS/CIS for all genes except EBER2 where n = 12 due to lack of sufficient sample for MS27 and n = 5 in HC/OND for all genes except BamHI-W where n = 4 due to a lack of sufficient sample for OND4. CD8+ T cells are the dominant lymphocyte population in MS lesions2,12, where they are highly clonally expanded4,5,8,9, suggesting reactivity to hitherto unknown local antigens. Although previous studies have explored changes in gene expression and T cell clonal expansion in the CSF of patients with MS10,11,12,26, numerous questions remain regarding the identity of clonally expanded CD8+ T cells and their antigen specificity in MS. Our comprehensive transcriptional and clonal analysis identified CSF-infiltrating T cells with increased expression of genes associated with T cell activation, the TRM phenotype and CNS migration in the MS/CIS cohort, consistent with previous reports10,11,12,27. As clonally expanded CD8+ T cells are present in the CSF in normal physiologic conditions and in CNS pathology10,17,26, identification of MS-specific CD8+ T cell clonal populations remains a challenge. Invoking a strategy used to identify disease-relevant T cells in inflammatory arthritis16,28 and cancer29, a subset of highly clonally expanded and CSF-enriched CD8+ T cells that had the highest frequencies in the patients with MS/CIS was identified. It was noteworthy that more than 70% of the highly expanded and CSF-enriched T cell clonotypes were CD8+ given that more than twice as many CD4+ T cells were analyzed. These CSF-enriched T cell clonotypes were widely characterized by a highly differentiated, antigen-experienced and cytotoxic phenotype with high CNS-trafficking potential, consistent with other reports30. These gene signatures were very similar to GZMB and TRM markers enriched in CD8+ T cells in MS lesions3,31, strongly suggesting these T cell clonotypes are CNS-infiltrating. Small networks of highly expanded CSF-enriched T cells with shared TCR sequence features to other less-expanded clonotypes were found, which overwhelmingly occurred within the same individual. These findings suggest that distinct, clonally expanded T cells may be contributory to MS pathology, unlike other autoimmune conditions with preferential TCR usage16. Combined with the inherent technical challenges in T cell antigen discovery, these findings highlight the difficulties in identifying the antigen specificity of clonally expanded T cells in MS. The majority of studies on candidate T cell auto-antigens in MS have focused on CD4+ T cells32,33,34. Through the use of three parallel antigen discovery strategies, our study provides substantial new insight into the antigen specificity of CD8+ T cells in MS. Novel mimotopes to several MS-derived CD8+ TCRs were identified by pMHC yeast display, a powerful unbiased antigen screening tool. Although the majority of mimotopes and naturally occurring peptide homologs were readily detectable by pMHC I tetramers, only one elicited a measurable functional response. The reason for the discrepancy between pMHC tetramer binding and functional reactivity is unclear but could be due to the absence of catch bonds by high-affinity TCR ligands35. Nonetheless, these candidate peptides provide an important framework for identifying TCRs with similar specificities in other individuals. The methodology of testing individual TCRs in primary human T cells by pMHC tetramer screening, followed by validation with functional reactivity is highly rigorous and ensured only genuine positive results. This approach was particularly important in the case of a TCR that demonstrated an exact TCRβ match to another antigen-specific clonotype yet did not share the same specificity, highlighting the need to validate every TCRαβ individually. Antigen specificity should therefore be interpreted cautiously when based solely on partial TCR sequence matching. Three distinct CSF-expanded and enriched CD8+ T cell clonotypes specific for EBV antigens were identified from three different patients in the MS cohort. Although EBV-specific CD8+ T cells have been previously reported in the CSF of MS and other neuro-inflammatory conditions36,37,38,39, the present study used paired TCRαβ analysis to unequivocally demonstrate EBV reactivity of highly enriched and clonally expanded CSF CD8+ T cell populations in MS. These findings are particularly relevant in light of recent evidence that EBV infection is a prerequisite for the subsequent development of MS40. Interestingly, the EBNA3A:B*08:01-specific CD8+ TCR identified in one of the patients with MS participating in this study was highly related to expanded CD8+ T cell clonotypes previously found in several patients with Alzheimer's disease17. Our findings therefore provide further support that EBV may be related to multiple forms of CNS pathology. The mechanism by which EBV is involved in MS pathogenesis remains unresolved. EBV-specific B cells and CD4+ T cells in MS have been suggested to be cross-reactive to CNS autoantigens41,42,43 (that is, molecular mimicry). We were unable to demonstrate cross-reactivity of the two EBV peptide-specific CD8+ T cell clonotypes against partially homologous self-peptides, but this does not completely rule out such a mechanism. Alternatively, the findings of CD8+ T cells reactive against EBV late latent and lytic antigens are consistent with other reports3,36,39 and could indicate EBV reactivation in the CNS44,45. In addition to the detection of EBV DNA in the CSF of most study participants, the increased expression of EBV transcripts in the MS/CIS cohort suggests that EBV reactivation drives expansion of EBV-specific CD8+ T cells. These findings are consistent with other recent results46 and suggest that EBV-specific CD8+ T cell expansion in the CSF could be a protective response to control reactivated EBV in MS. There are multiple mechanisms by which EBV could gain access to the CNS. In addition to primary infection of cells within the CNS47, a number of studies have described the induction of ‘atypical' T-bet+CXCR3+ B cells by EBV48,49, which could enable their migration into the CNS. EBV expression is highly dynamic, permitting the virus to exist in various latency or lytic programs47. Clinical trials using adoptive T cell therapies targeting EBV in MS did not show a clear benefit in progressive MS50, however, it remains unclear how such therapies may alter EBV viral loads and expression as well as relapse and magnetic resonance imaging (MRI) outcomes. It is also important to consider that EBV reactivation in MS may represent an epiphenomenon as memory B cell differentiation into plasma cells is a trigger of EBV reactivation. This study was limited by the smaller population of control participants. Follow-up studies with larger numbers of well-matched MS and control participants are needed to more clearly identify disease-relevant T cell populations in MS. In addition, longitudinal analyses of T cell clonal expansion in earlier versus later stages of MS are needed. Although the transcriptional phenotyping analyses suggest a pro-inflammatory cytotoxic phenotype of CSF-expanded CD8+ T cells, further in vitro and in vivo analyses are needed to determine what role these cells play in MS. It is also important to acknowledge that despite the rigor of the antigen specificity testing, this approach was not exhaustive and was limited in the breadth of antigens that were tested. Given that various foreign and self-antigens are considered viable antigenic targets in MS, future studies will need to incorporate high-throughput approaches to probe multiple target antigens simultaneously. Given that there are only trace B cells in the CSF and CNS of healthy individuals, it is possible that cell-free EBV DNA originated in the blood. Alternatively, there are other cellular reservoirs in the CNS where EBV has been identified even in healthy individuals47. This study provides important progress towards all three aims by demonstrating a small population of predominantly CD8+ T cells that were highly expanded and enriched in the CSF of patients with MS with strongly upregulated genes associated with antigen exposure, CNS migration and cytotoxicity. The studies in this Article have been approved by the University of California San Francisco (UCSF) Institutional Review Board research ethics committee (protocol numbers 10-02389 and 14-15278). Informed consent was obtained from all participants in this study. No compensation was provided to the study participants. The participants, MS/CIS and control, were enrolled through the UCSF ORIGINS or Expression, Proteomics, Imaging, Clinical (EPIC) studies (https://epicstudy.ucsf.edu/). Healthy controls and patients with OND were enrolled in the biobanking study ‘Immunological Studies of Neurologic Subjects'. All enrolled participants with MS or CIS were diagnosed according to the 2017 McDonald criteria51. Basic demographic and clinical information for all research participants is shown in Table 1 and Supplementary Table 1. Blood and CSF samples were collected from the enrolled participants on the same day during clinical and research procedures after informed consent. CSF (20–30 ml) was collected by lumbar puncture from each individual. Blood and CSF were processed immediately after collection in preparation for single-cell library preparation as previously described15. Unfractionated peripheral blood mononuclear cells (PBMCs) were isolated using CPT mononuclear cell preparation tubes (BD Biosciences) and resuspended in 2% fetal bovine serum (FBS). Single-cell sequencing libraries were prepared using 5′ scRNA-seq and 5′ T cell V(D)J scTCR-seq kits (10X Genomics). Raw data for both scRNA-seq and scTCR-seq datasets were processed using CellRanger (v3.0.1 and v3.1.0, respectively) by 10X Genomics. All data were analyzed using a custom bioinformatics pipeline that included Seurat (v3.1.2–v4.3.0), the Spliced Transcripts Alignment to a Reference (STAR) algorithm52 (v2.5.1), SingleR53 (v1.1.7) and DoubletFinder54 (v2.0.2). TCR V(D)J contig assemblies outputted from CellRanger were further annotated and analyzed using Immcantation (v3.1.0). TCR clonal families were identified using Change-O55 (v0.4.6), which generated clone IDs for both TCRα and -β chain assemblies. The scRNA-seq data have been uploaded to the Gene Expression Omnibus (GEO) repository under BioProject PRJNA549712 (GEO accession number GSE133028) and the scTCR-seq data have been uploaded under BioProject PRJNA1232831 (GEO accession number GSE291328). Across both RNA-seq and VDJ data, reads present in more than one sample that shared the same cell barcode and unique molecular identifier were filtered using previously described methods15. The R package DropletUtils was used to filter out these reads in the RNA-seq data and SingleCellVDJdecontamination (https://github.com/UCSF-Wilson-Lab/SingleCellVDJdecontamination) was used to apply the same methods to filter out these reads in the VDJ data. All gene counts from scRNA-seq data were combined using Seurat. Only genes present in two or more cells were included. Only cells containing transcripts of between 700 and 2,500 genes were included. The PercentageFeatureSet function was used to calculate the percentage of mitochondrial transcript expression for each cell. Cells that expressed at least 10% mitochondrial genes were omitted. Gene counts were normalized using the R package SCTransform56. All filtered cells were clustered using 20 principal components (PCs) in Seurat. Clusters were formed using a shared nearest-neighbor graph in combination with dimensional reduction using uniform manifold approximation and projection57. Doublet detection and removal were performed for each sample using DoubletFinder54 with expected doublet rates set based on the 10X Genomics reference manual. Cumulative sums were iteratively calculated for each PC to measure the per cent variance accounted for with the data. To determine a reasonable number of PCs, a threshold of 90% variance was applied, which resulted in 12 PCs being inputted when reclustering cells. Clusters of cells with a high expression of platelet markers (PPBP and PF4) or hemoglobin subunits (HBB, HBA1 and HBA2) were omitted. Among the remaining cells, all V gene transcripts (TRAV, TRBV, IGHV, IGKV and IGLV) were removed and an additional round of reclustering was performed with nine PCs. Assembled TCR contigs outputted from CellRanger were inputted into the Immcantation pipeline for a second round of alignments to the VDJ region using IgBLAST. Contigs containing fewer than three unique molecular identifiers were omitted. Only contigs that aligned in frame (both the FUNCTIONAL and IN_FRAME output fields were TRUE) and across the constant region were retained. Cells in the TCR VDJ data were only kept if these contained one TCRβ chain and one TCRα chain. If cells had multiple chains, TCRα or -β, which passed these thresholds, the contig with the largest number of unique molecular identifiers and or reads was kept. Cell-type annotations were generated using previously described methods15. Cell types were defined by performing differential gene expression analysis for each cluster. The most upregulated genes, with the highest positive average log-transformed fold change, were compared with a custom panel of canonical gene makers (Supplementary Table 21) spanning several key immune cell types—B cells, CD4+ and CD8+ T cells, natural killer (NK) cells, classical monocytes, inflammatory monocytes, macrophages, plasmacytoid dendritic cells and monocyte-derived dendritic cells. In addition to these manual cell-type annotations, another set of cell types was determined using the automated cell-type annotation tool SingleR, which used the combined Blueprint and ENCODE reference dataset for fine-tuning predictions53. A T cell subset was created by filtering for cells that overlap both RNA-seq and TCR VDJ data. All clusters annotated as T cells had their annotations modified by CD8 gene expression. Among the T cells, any cell with CD8A or CD8B expression was annotated as a CD8+ T cell. Differential gene expression analyses were performed using the FindMarkers command in Seurat with the Wilcoxon test and the following parameters: P-adjusted value cutoff = 0.05 and log(fold change) cutoff = 0. The TCR contigs outputted from Immcantation were clustered based on similarities between their TCR variable region genes (TRAV or TRBV), TCR joining region genes (TRAJ or TRBJ) and complementary determining region 3 (CDR3) amino-acid sequences. Cell counts were computed for each clone ID, including separate cell counts for PBMC and CSF samples. Shannon's entropy was calculated between CSF and PBMC samples of different disease groups using the alphaDiversity function in the R package alakazam. Clonal expansion was defined as clones containing more than one cell. Among the expanded clonal families of TCRs, CSF enrichment of highly expanded clones was determined by the ratio of the CSF to peripheral-blood frequencies. Clones that were expanded in CSF (that is, more than singletons), but with frequencies less than highly expanded, were labeled as moderately expanded. Among the expanded clonal families of TCRs, CSF enrichment of highly expanded clones was determined by the ratio of the CSF to peripheral-blood frequencies. CSF highly expanded TCRs were inputted into GLIPH2 to generate glyph groups, which indicated which TCRs were predicted to target the same epitope. These glyph groups were used to create a network using the R package igraph and graphically displayed using Cytoscape. For each sample, 100 ng high-quality DNA was fragmented using a Library preparation enzymatic fragmentation kit 2.0 (Twist Bioscience). After fragmentation, the DNA was repaired, and poly(A) tails were attached and ligated to Illumina-compatible dual-index adapters with unique barcodes. After ligating, the fragments were purified with a 0.8× ratio of AMPure XP magnetic beads (Beckman Coulter). Double size selection was performed (0.42× and 0.15× ratios) and libraries of approximately 800 bp were selected, at which point libraries were amplified and purified using magnetic beads. A Twist target enrichment kit (Twist Bioscience) was then used to perform target capture on pooled samples. Sample volumes were then reduced using magnetic beads and DNA libraries were bound to 1,394 biotinylated probes. Probes were designed specifically to target all exons, introns and regulatory regions of the classical HLA loci, including HLA-A, HLA-B, HLA-C, HLA-DPB1, HLA-DRB1 and HLA-DQB1. Next, streptavidin magnetic beads were used to capture fragments targeted by the probes. The captured fragments were then amplified and purified. A BioAnalyzer instrument (Agilent) was then used to analyze the enriched libraries. After evaluation, the enriched libraries were sequenced using a paired-end 150-bp sequencing protocol on the NovaSeq platform (Illumina). The TCR sequences for each α and β gene pair were codon-optimized and used to generate gene blocks (IDT) in which the TCRβ and TCRα sequences were separated by a P2A sequence. Flanking homology arms were included to permit knock-in into the human TRAC locus, as previously described59. The gene blocks were cloned into pUC19 plasmids by Gibson assembly and the sequence was verified by Sanger sequencing. Primary human CD8+ T cells were isolated from commercially purchased leukopaks (Vitalant or Stemcell; unidentified healthy donors). PBMCs were isolated by Ficoll centrifugation and cryopreserved before each experiment. In all experiments, T cells were cultured in RPMI medium containing 10% FBS, 2-mercaptoethanol, penicillin–streptomycin with L-glutamine, sodium pyruvate, MEM vitamin solution and nonessential amino acids (all Fisher Scientific). TCR knock-in was performed as previously described59, with minor changes. Briefly, CD8+ T cells were isolated from thawed PBMCs by negative selection (Miltenyi) and rested overnight with 5 ng ml−1 human IL-7. The CD8+ T cells were stimulated 1:1 with anti-human CD3/CD28 magnetic Dyna beads (Fisher Scientific), 20 ng ml−1 human IL-2, 5 ng ml−1 human IL-7 and 5 ng ml−1 IL-15 for 48 h previous to T cell electroporation. Guide RNAs specific for the human TRAC locus were generated by incubating CRISPR RNA (crRNA; AGAGTCTCTCAGCTGGTACA) 1:1 with trans-activating crRNA (Dharmacon) for 30 min at 37 °C to yield a final concentration of 80 µM. Polyglutamic acid (0.8× volume) was added to the guide RNA as previously described60. Cas9 (QB3; Macrolab) was added 1:1 with the guide RNA and incubated for 15 min at 37 °C to yield a 20 µM ribonucleoprotein, which was used immediately for electroporation. Dyna beads were removed from the T cell culture using an EasySep separation magnet (StemCell) 48 h after CD8+ T cell stimulation. The T cells were then centrifuged at 200g for 9 min and resuspended in Lonza electroporation P3 buffer with supplement (20 µl per 1 × 106 T cells). The T cells (20 µl) were electroporated with 3.5 µl ribonucleoprotein and 1 µg TCR-encoding plasmid DNA (1–2 µl) using a Lonza 4D Nucleofector 96-well electroporation system and pulse code EH115 (ref. CD8+ T cells were immediately rescued by the addition of 80 µl warmed T cell medium and incubation in a 37 °C incubator for 15 min. The cells were then split into fifths in 96-well round-bottomed plates; T cell medium plus 10 ng ml−1 IL-2 was added to the samples to a final volume of 200 µl. The CD8+ T cells were expanded for a minimum of 96 h before testing for pMHC tetramer binding. The T cells were re-fed with a half volume of fresh medium and IL-2 every 3–4 days. Custom peptide-loaded MHC I monomers were generated by ultraviolet light–ligand exchange as previously described62. HLA-A*31:01 pMHC monomers (Easymers) were purchased from ImmunAware and loaded with custom peptides according to the supplier's instructions. Tetramerization was carried out using streptavidin conjugated to the fluorophores phycoerythrin and allophycocyanin (Life Technologies). CD8+ T cells were treated with 100 nM dasatinib (StemCell) for 30 min at 37 °C, followed by staining with the appropriate tetramers (2–3 µg ml−1) for 30 min at room temperature. All tetramers were used within 3–4 weeks of synthesis. The cells were washed in FACS buffer (1×DPBS without calcium or magnesium, 0.1% sodium azide, 2 mM EDTA and 1% FBS) and stained with anti-CD8 PECy7 (eBioscience; SK1), anti-TCR BV421 (BioLegend; IP26), a PerCP/Cy5.5 dump antibody mixture containing anti-CD4 (BioLegend; RPA-T4), anti-CD14 (BioLegend; HCD14), anti-CD16 (BioLegend; B73.1), anti-CD19 (BioLegend; HIB19; all antibodies at 1:100) and Aqua506 viability dye (1:1,000; Life Technologies) for 30 min at 4 °C. The cells were then washed and resuspended in FACS buffer, and analyzed by flow cytometry (LSRFortessa). Only experiments where the forward versus side-scatter gate contained at least 10% lymphocytes and CD8+ T cells expressed fewer than 20% TCRs were used for analysis to ensure a large number of T cells with high TCR-knockout efficiency was achieved (Extended Data Fig. Yeast allele libraries were thawed in SDCAA (pH 5) medium, passaged, induced in SGCAA (pH 5) and selected using biotinylated soluble TCR coupled to streptavidin-coated magnetic MACS beads (Miltenyi) as previously described63. Briefly, 2 × 109 yeast cells from all four length libraries underwent negative selection with 250 μl beads in 5 ml PBE (PBS containing 0.5% BSA and 1 mM EDTA) for 1 h with rotation at 4 °C. After passage through an LS column (Miltenyi) on a magnetic stand and three washes with 3 ml PBE, the flow-through was incubated with 250 μl beads (pre-incubated with 400 nM biotinylated TCR) for 3 h at 4 °C with rotation. The yeast were magnetically separated through an additional LS column, washed three times with 3 ml PBE and the elution was cultured overnight in SDCAA (pH 5) following an SDCAA wash to remove residual PBE. The yeast were induced in SGCAA (pH 5) for 2–3 days before further selection, with subsequent selections using 50 μl beads or TCR-coated beads in 500 μl PBE. DNA was isolated from 5–10 × 107 yeast cells per selection using a Zymoprep II kit (Zymo Research). Unique barcodes and random eight-mer sequences were added to the sequencing product by PCR and amplified for 25 cycles to allow for downstream demultiplexing and improved clustering. A subsequent PCR added Illumina chip primer sequences, resulting in products containing Illumina P5-Truseq read 1-(N8)-Barcode-pHLA-(N8)-Truseq read 2-IlluminaP7. The library was purified by double-sided SPRI bead isolation (Beckman Coulter), quantified using a KAPA library amplification kit (Illumina) and deep sequenced on an Illumina MiSeq instrument with a 2 × 150 V2 kit for low-diversity libraries. The genes for HLA-A*31:01, HLA-A*29:02, HLA-B*08:01 and HLA-B*35:01 were codon-optimized and synthesized as gene blocks (IDT). The gene blocks were cloned into the pHR-CMV Lacz lentivirus vector by Gibson assembly and sequences were verified by Sanger sequencing. The APCs were pulsed overnight with 10 µg ml−1 peptide or vehicle control in serum-free medium. The cells were washed with FACS buffer and stained with the cell surface antibodies as described in the ‘pMHC tetramer screening' section (anti-CD8, anti-TCR, dump channel antibody mixture and live/dead dye). Next, the cells were washed, fixed and stained with anti-human IFN-γ Alexa 647 (BioLegend; 4S.B3) and anti-human TNF-α Alexa 488 (BioLegend; Mab11) in permeabilization buffer (BD). Finally, the cells were washed and collected on an LSRFortessa system. Jurkat E6-1 T cells (American Type Culture Collection, TIB-152) were maintained in RPMI medium supplemented with L-glutamine and 10% FBS. Endogenous TRAC and TRBC1 expression in Jurkat cells were knocked out with synthetic crRNAs designed using the Alt-R system (IDT) containing the following genomic target sequences: TRBC1, 5′-CGTAGAACTGGACTTGACAG-3′ and TRAC, 5′-CTTCAAGAGCAACAGTGCTG-3′. The crRNA was complexed with 1:1 trans-activating crRNA (IDT; 0.2 nmol each), followed by 0.1 nmol recombinant Cas9 protein (Macrolab). The ribonucleoproteins were then transduced into Jurkat T cells using a Amaxa P3 primary cell nucleofector kit (Lonza; pulse code CK116). TRAC knockout was performed first and loss of surface TCRαβ expression was confirmed by flow cytometry. TRAC-knockout cells underwent subsequent knockout of TRBC1, which had previously been shown to lead to loss of TCRαβ expression in line with overexpressed TCRα. To track TCR activation, a lentiviral vector was constructed that contained the NFAT transcriptional reporter NBV64 upstream of a minimal CMV reporter driving mCherry fluorescent marker expression, and constitutive expression of iRFP670 under a Pgk promoter provided a marker of transduction. Jurkat cells lacking endogenous TCRαβ expression were transduced with the vector and sorted for iRFP fluorescence and lack of mCherry background fluorescence. For TCRs corresponding to CD8+ T cells, an additional lentiviral vector encoding human CD8α was expressed in the Jurkat cells and the cells were sorted for uniform CD8α expression before TCR transduction. For TCR expression, lentiviral expression constructs that encode a human Pgk promoter and the coding sequence of each specific TCRα chain with the IRES-neomycin resistance gene or each TCRβ chain with the IRES-blasticidin resistance gene were generated. Lentiviral particles were packaged in HEK293T cells following standard protocols and concentrated 10× using the Lenti-X Concentrator reagent (Takara). Viral particles were added to T cells at a low multiplicity of infection and expression was ensured by passaging cells for 5 days under antibiotic selection with 10 µg ml−1 blasticidin (Gibco) and 1 mg ml−1 G418 (Teknova). TCR-expressing Jurkat cells (1 × 105) were stimulated for 24 h with HLA-allele-transduced APCs (1 × 105) loaded with 10 µg ml−1 peptide or vehicle control. Antigen-reactive CD8+ cells were identified by coexpression of NFAT–mCherry and anti-human CD69–phycoerythrin (BioLegend; FN50). A Pan B cell isolation kit (Miltenyi) was used to isolate B cells from frozen PBMCs. The B cells (1 × 106 cells ml−1) were incubated 1:1 with pre-warmed EBV supernatant (B95.8 strain) for 1 h at 37 °C, followed by the addition of 1 μg ml−1 R848 and 100 ng ml−1 CD40L. The cells were cultured for two weeks, with medium changes as needed, and expanded into larger plates. The LCLs were cryopreserved for future use in cellular assays. Jurkat cells expressing the TCR of interest (target TCR Jurkat cells) were co-cultured with LCLs carrying at least one matching MHC I allele (HLA-matched LCLs) at a 1:1 ratio (100,000 cells each per well) in a 96-well plate for 24 h at 37 °C. To assess specificity, negative control conditions included co-culture of (1) target TCR Jurkat cells with HLA-mismatched LCLs and (2) HLA-matched LCLs with Jurkat cells expressing an HLA-mismatched TCR. Jurkat reactivity was assessed by measurement of coexpression of NFAT–mCherry and CD69–phycoerythrin as described in the ‘TCR-expressing Jurkat cell assays' section. Cell-free CSF supernatant was obtained after centrifugation as described earlier for single-cell sequencing and stored at −80 °C. DNA and RNA were each extracted from 400 µl CSF supernatant using a ZYMO Quick-DNA/RNA pathogen MagBead kit (Zymo Research) according to the manufacturer's instructions. The extracted DNA was eluted in 50 µl nuclease-free water. DNase I treatment was performed on the RNA-extraction samples before elution into 30 µl nuclease-free water. An EBV-transformed LCL was used as a positive control; Jurkat cells and water only were used as negative controls. The DNA concentration and purity were assessed using a Nanodrop spectrophotometer (Thermo Fisher Scientific). The extracted DNA and RNA were stored at −20 °C until use for PCR amplification. Complementary DNA was synthesized using a ProtoScript first strand cDNA synthesis kit (NEB) using 6 µl RNA per 20 µl reaction volume according to the manufacturer's instructions. PCR reactions were performed using a Qiagen Taq PCR core kit according to the manufacturer's instructions. Each 30-µl reaction contained 1×PCR Buffer with MgCl2, 200 µM dNTPs, 0.2 µM forward and reverse primers, 10–100 ng template DNA, 0.75 U Qiagen Taq DNA polymerase and nuclease-free water to the final volume. Amplification was carried out in a thermal cycler (Bio-Rad thermal cycler with 96-deep-well C1000 block) using the following cycling conditions: an initial denaturation step at 95 °C for 3 min, followed by 34 cycles of denaturation at 95 °C for 30 s, annealing at 58 °C for 30 s and extension at 72 °C for 1 min. The PCR products were analyzed using agarose gel electrophoresis and visualized under ultraviolet light using a gel documentation system (LI-COR Odyssey M Imager). For samples showing positive amplification, the remaining PCR reaction volume was directly submitted to Molecular Cloning Laboratories for Sanger sequencing. CSF samples were considered EBV-positive if sequencing results correctly aligned to the reference sequence of the amplified target. Primers and probes for ddPCR were synthesized as PrimeTime qPCR assays (IDT). Probes targeting EBV genes were labeled with FAM and probes for the housekeeping reference genes (RPP30 or GAPDH) were labeled with HEX (Supplementary Table 22). Oligonucleotides were used as previously published for EBER2 (ref. A volume of 1.25 μl of each 20× target primers–probe mix in Tris–EDTA was used with 2×ddPCR Supermix for probes without deoxyuridine triphosphate and 10 μl DNA. The emulsion of approximately 40 μl was slowly transferred to ddPCR 96-well plates (Bio-Rad, 12001925) and heat-sealed with foil. Amplification was carried out in a thermal cycler (Bio-Rad Thermal Cycler with 96-Deep Well C1000 block) using the following cycling conditions: an initial denaturation step at 95 °C for 10 min; 39 cycles of 30 s at 94 °C and 1 min at 57 °C, followed by 10 min at 98 °C. Data from the droplet reader are given as copies per microlitre and relative expression was calculated as the target gene:reference gene ratio. Any sample that was not detected by the housekeeping gene was repeated and the threshold was set separately according to the negative control with water. All samples were run in duplicate. Differential gene expression comparisons between groups were performed using the two-sided Wilcoxon rank-sum test with Bonferroni correction. Shannon entropy results were compared using Brown–Forsythe's and Welch's analysis of variance with multiple comparisons using Dunnett T3 corrections. Comparisons of CSF-enriched clonotypes T cell (for example, CD8+ versus CD4+ T cells and MS/CIS versus HC/OND) and ddPCR results were performed using unpaired two-tailed Student's t-tests with Welch's correction using GraphPad Prism (v10.6.1). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The scRNA-seq data have been uploaded to the Gene Expression Omnibus (GEO) repository under BioProject PRJNA549712 (GEO accession number GSE133028) and the scTCR-seq data have been uploaded under BioProject PRJNA1232831 at GEO accession number GSE291328. Source data are provided with this paper. The code for all scRNA-seq and scTCR-seq analysis can be found at https://github.com/UCSF-Wilson-Lab/MS_Tcell_CSF_PBMC_single_cell_study_analysis. & Weiner, H. L. Multiple sclerosis: mechanisms and immunotherapy. The compartmentalized inflammatory response in the multiple sclerosis brain is composed of tissue-resident CD8+ T lymphocytes and B cells. van Nierop, G. P. et al. Phenotypic and functional characterization of T cells in white matter lesions of multiple sclerosis patients. Babbe, H. et al. Clonal expansions of CD8+ T cells dominate the T cell infiltrate in active multiple sclerosis lesions as shown by micromanipulation and single cell polymerase chain reaction. Jacobsen, M. et al. Oligoclonal expansion of memory CD8+ T cells in cerebrospinal fluid from multiple sclerosis patients. Sawcer, S. et al. Genetic risk and a primary role for cell-mediated immune mechanisms in multiple sclerosis. Multiple sclerosis: brain-infiltrating CD8+ T cells persist as clonal expansions in the cerebrospinal fluid and blood. Expanded CD8 T-cell sharing between periphery and CNS in multiple sclerosis. Pappalardo, J. L. et al. Transcriptomic and clonal characterization of T cells in the human central nervous system. Integrated single cell analysis of blood and cerebrospinal fluid leukocytes in multiple sclerosis. Ostkamp, P. et al. A single-cell analysis framework allows for characterization of CSF leukocytes and their tissue of origin in multiple sclerosis. Cantoni, C. et al. A single-cell compendium of human cerebrospinal fluid identifies disease-associated immune cell populations. Ramesh, A. et al. A pathogenic and clonally expanded B cell transcriptome in active multiple sclerosis. Yang, X. et al. Autoimmunity-associated T cell receptors recognize HLA-B*27-bound peptides. Gate, D. et al. Clonally expanded CD8 T cells patrol the cerebrospinal fluid in Alzheimer's disease. & Davis, M. M. Analyzing the Mycobacterium tuberculosis immune response by T-cell receptor clustering with GLIPH2 and genome-wide antigen screening. Gee, M. H. et al. Antigen identification for orphan T cell receptors expressed on tumor-infiltrating lymphocytes. Bagaev, D. V. et al. VDJdb in 2019: database extension, new analysis infrastructure and a T-cell receptor motif compendium. Chronister, W. D. et al. TCRMatch: predicting T-cell receptor specificity based on sequence similarity to previously characterized receptors. Rapid single-cell identification of Epstein–Barr virus-specific T-cell receptors for cellular therapy. Schmidt, F. et al. In-depth analysis of human virus-specific CD8+ T cells delineates unique phenotypic signatures for T cell specificity prediction. He, R. et al. Follicular CXCR5-expressing CD8+ T cells curtail chronic viral infection. Expression profiling of cerebrospinal fluid identifies dysregulated antiviral mechanisms in multiple sclerosis. Penkava, F. et al. Single-cell sequencing reveals clonal expansions of pro-inflammatory synovial CD8 T cells expressing tissue-homing receptors in psoriatic arthritis. Peripheral T cell expansion predicts tumour infiltration and clinical response. Ashida, S. et al. Intrathecally expanded GZMK+/GZMH+ CD8 T cells targeting EBV antigens may reduce severity of Multiple Sclerosis. Fransen, N. L. et al. Tissue-resident memory T cells invade the brain parenchyma in multiple sclerosis white matter lesions. Planas, R. et al. GDP-L-fucose synthase is a CD4+ T cell-specific autoantigen in DRB3*02:02 patients with multiple sclerosis. Wang, J. et al. HLA-DR15 molecules jointly shape an autoreactive T cell repertoire in multiple sclerosis. Isolation of a structural mechanism for uncoupling T cell receptor signaling from peptide-MHC binding. van Nierop, G. P., Mautner, J., Mitterreiter, J. G., Hintzen, R. Q. & Verjans, G. M. Intrathecal CD8 T-cells of multiple sclerosis patients recognize lytic Epstein–Barr virus proteins. Erdur, H. et al. EBNA1 antigen-specific CD8+ T cells in cerebrospinal fluid of patients with multiple sclerosis. Gottlieb, A., Pham, H. P. T., Saltarrelli, J. G. & Lindsey, J. W. Expanded T lymphocytes in the cerebrospinal fluid of multiple sclerosis patients are specific for Epstein–Barr-virus-infected B cells. Broader Epstein–Barr virus-specific T cell receptor repertoire in patients with multiple sclerosis. Bjornevik, K. et al. Longitudinal analysis reveals high prevalence of Epstein–Barr virus associated with multiple sclerosis. Lanz, T. V. et al. Clonally expanded B cells in multiple sclerosis bind EBV EBNA1 and GlialCAM. Ineffective control of Epstein–Barr-virus-induced autoimmunity increases the risk for multiple sclerosis. Wucherpfennig, K. W. & Strominger, J. L. Molecular mimicry in T cell-mediated autoimmunity: viral peptides activate human T cell clones specific for myelin basic protein. & Aloisi, F. Epstein–Barr virus-specific CD8 T cells selectively infiltrate the brain in multiple sclerosis and interact locally with virus-infected cells: clue for a virus-driven immunopathological mechanism. Veroni, C., Serafini, B., Rosicarelli, B., Fagnani, C. & Aloisi, F. Transcriptional profile and Epstein–Barr virus infection status of laser-cut immune infiltrates from the brain of patients with progressive multiple sclerosis. Lehikoinen, J. et al. Epstein–Barr virus in the cerebrospinal fluid and blood compartments of patients with multiple sclerosis and controls. Wahbeh, F. & Sabatino, J. J. Epstein–Barr virus in multiple sclerosis. Early multiple sclerosis activity associated with TBX21+CD21loCXCR3+ B cell expansion resembling EBV-induced phenotypes. Pender, M. P. et al. Epstein–Barr virus-specific T cell therapy for progressive multiple sclerosis. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. McGinnis, C. S., Murrow, L. M. & Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. Gupta, N. T. et al. Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data. Hafemeister, C. & Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. Defining KIR and HLA class I genotypes at highest resolution via high-throughput sequencing. Roth, T. L. et al. Reprogramming human T cell function and specificity with non-viral genome targeting. Nguyen, D. N. et al. Polymer-stabilized Cas9 nanoparticles and modified repair templates increase genome editing efficiency. Oh, S. A. et al. High-efficiency nonviral CRISPR/Cas9-mediated gene editing of human T cells using plasmid donor DNA. Sabatino, J. J. et al. Anti-CD20 therapy depletes activated myelin-specific CD8+ T cells in multiple sclerosis. Birnbaum, M. E. et al. Deconstructing the peptide–MHC specificity of T cell recognition. Cetin, M. et al. T-FINDER: A highly sensitive, pan-HLA platform for functional T cell receptor and ligand discovery. Comparison of real-time PCR and digital PCR for detection of plasma Epstein–Barr virus DNA in nasopharyngeal carcinoma. Detection of Epstein–Barr virus DNA by polymerase chain reaction in blood and tissue biopsies from patients with Sjogren's syndrome. Comparison of qPCR with ddPCR for the quantification of JC polyomavirus in CSF from patients with progressive multifocal leukoencephalopathy. We thank the individuals who agreed to participate as research subjects in this study. We thank the UCSF EPIC and Origins Study Teams for valuable aid in subject recruitment. These authors contributed equally: Fumie Hayashi, Kristen Mittl, Ravi Dandekar. UCSF Weill Institute for Neurosciences, Department of Neurology, University of California San Francisco, San Francisco, CA, USA Fumie Hayashi, Kristen Mittl, Ravi Dandekar, Josiah Gerdts, Ebtesam Hassan, Ryan D. Schubert, Lindsay Oshiro, Rita Loudermilk, Ariele Greenfield, Danillo G. Augusto, Gregory Havton, Shriya Anumarlu, Arhan Surapaneni, Akshaya Ramesh, Edwina Tran, Kanishka Koshal, Kerry Kizer, Isabelle J. Fisher, Tiffany Cooper, Meagan Harms, Refujia Gomez, Ahmed Abdelhak, Sergio Baranzini, Riley Bove, Stacy Caillier, Richard Cuneo, Jeffrey Gelfand, Ari Green, Joanne Guo, Sasha Gupta, Harkee Halait, Roland G. Henry, Jill A. Hollenbach, Jorge R. Oksenberg, Nico Papinutto, Samuel Pleasure, Adam Renschen, Simone Sacco, Adam Santaniello, Anna Sindalovsky, Claire D. Clelland, Bruce A. C. Cree, Stephen L. Hauser, Jill A. Hollenbach, Michael R. Wilson, Scott S. Zamvil & Joseph J. Sabatino Jr 3T Biosciences, South San Francisco, CA, USA Joanna Dreux, Alaina K. Cagalingan, Florian Schustek, Lena Flood, Tamson Moore, Lisa L. Kirkemo, Leah Sibener & Marvin Gee Department of Epidemiology and Biostatistics, University of California San Francisco, San Francisco, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Scott S. Zamvil or Joseph J. Sabatino Jr. is a current employee of Genentech. are currently or previously employed by 3T Biosciences. has received research grant funding from Roche/Genentech, Novartis and Kyverna Therapeutics; speaking honoraria from Genentech, Takeda, WebMD and Novartis; and consulting fees from Vertex Pharmaceuticals, Ouro Medicines, Indapta Therapeutics, Pfizer and Delve Bio; is on the Board of Directors of Delve Bio and has received licensing fees from CDI Labs. has received research grant funding from Roche/Genentech and Novartis, advisory board honoraria from IgM Biosciences and TG Therapeutics, and has received stock options as a consultant for Sift BioSciences. The remaining authors declare no competing interests. Nature Immunology thanks Christian Münz and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Expression for the indicated genes is shown for all T cells (blood and CSF combined) after merging scRNA-seq and scTCR-seq data. a, Volcano plot analysis of differential gene expression between memory (CD27+) T cells in the CSF and blood. Only genes expressed in at least 10% of memory CD8+ T cells in the blood and/or CSF were analyzed. Differential gene expression comparisons were performed using two-sided Wilcon ranked-sum tests with Bonferroni correction for adjusted p-values. Genes with adjusted p-values < 0.05 and log2 fold change > 2 are indicated in red and genes with adjusted p-values < 0.05 in blue. c, KLF2 expression levels between Trm negative and positive T cells in the CSF. Shannon entropy analysis (where y axis indicates exponential of Shannon–Wiener index) is shown for CD8+ T cells in the peripheral blood (PB) and CSF by disease status. Abbreviations: MS = multiple sclerosis; CIS = clinically isolated syndrome; HC = healthy control; OND = other neuro-inflammatory disease. Comparisons between groups using Brown–Forsythe and Welch ANOVA with multiple comparisons using Dunnett T3 corrections (****p < 0.0001; ns = not significant). Expression levels and percent expression of the indicated genes is shown for highly expanded CSF T cells versus non-expanded T cell clonotypes. a, Representative flow cytometry analysis of pMHC tetramer-stained CD8+ T cells following TCR knock-in. b, Four patient-derived TCRs were tested for tetramer binding to the indicated peptides. a, Summary tetramer binding analysis of peptide homologs for the three TCRs that yielded yeast display-derived mimotopes (red). b, Summary of tetramer binding and cytokine reactivity of the indicated peptides to two different TCRs. Cytokine reactivity reflects subtracted background from no-stimulation control. The summary of all pMHC tetramer screening for 98 viral peptides for 19 patient-derived TCRs is shown in the heatmap. TCR clonotype ID's are shown on the x axis and peptide:MHC antigens are shown on the y axis. All viral peptides were tested individually except in the case of the peptides for HLA-A*02:01 where tetramers were tested in pools of 5. Each peptide/pool was tested a minimum of two times in using different T cells from different donors. Representative flow cytometry analysis of Jurkat reporter cells expressing TCR 94669_8198 were co-cultured with HLA-A*29:02-expressing K562 cells pulsed with the indicated EBV peptides or no peptide for 24 h (all tested in triplicate). Antigen reactivity was assessed by coexpression of CD69 and NFAT–mCherry. a, The TCR clonotype frequencies for all clonotypes of the three MS patients (MS6, MS8, MS27) with CSF-expanded EBV-specific CD8+ T cells was compared between the blood and CSF. Clonal frequency of T cell clonotypes in the CSF that were highly expanded (at least 0.75%) and enriched at least 2-fold more frequently than the blood of the same individual are highlighted in red. b, The expression of specific genes between the three CSF-expanded EBV-specific CD8+ T cell clonotypes (TCRs 86333_1456, 69317_24418, and 94669_8198 combined) and all other CSF-expanded enriched CD8+ T cells was compared. a, Representative agarose gel of BZLF1 DNA amplification (239-bp fragment) in the CSF supernatant of the indicated patient samples. DNA from EBV-transformed lymphoblastoid cell lines (LCL) were used as positive control and water as a negative control. Samples with a positive band underwent Sanger sequencing for confirmation. b, Representative DNA ddPCR results of EBER2 and RPP30 (housekeeping gene) from CSF supernatant is shown. The two columns with many positive droplets with high signal are from LCLs as positive control. List of all T cell clonotypes. Differential gene expression analysis of CD8+ and CD4+ T cells in CSF versus blood. Differential gene expression analysis of memory (CD27+) T cells in CSF and blood. Differential gene expression analysis of CD8+ and CD4+ T cells in MS/CIS versus HC/OND. CSF T cell clonal expansion in MS and CIS. List of CSF-enriched and expanded T cell clonotypes. Differential gene expression analysis of CSF highly expanded versus unexpanded T cells. Differential gene expression analysis of TRM positive versus negative. List of the tested pMHC yeast display-derived peptides. List of viral peptides tested for antigen discovery. Differential gene expression analysis of EBV-expanded versus all other expanded CD8+ T cells. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Hayashi, F., Mittl, K., Dandekar, R. et al. Antigen specificity of clonally enriched CD8+ T cells in multiple sclerosis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-10085-z'>Imaging the sub-moiré potential using an atomic single electron transistor</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-02-05 03:56:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Electrons in solids owe their properties to the periodic potential landscapes they experience. The advent of moiré lattices has revolutionized our ability to engineer such landscapes on nanometre scales, leading to numerous ground-breaking discoveries. Despite this progress, direct imaging of these electrostatic potential landscapes remains elusive. Here we introduce the atomic single electron transistor (SET), a new scanning probe that uses a single atomic defect in a van der Waals material as an ultrasensitive, high-resolution potential sensor. Built on the quantum twisting microscope (QTM) platform1, this probe leverages the capability of the QTM to form a pristine, scannable two-dimensional interface between vdW heterostructures. Using the atomic SET, we present the first direct images of the electrostatic potential in a canonical moiré interface: graphene aligned to hexagonal boron nitride2,3,4,5,6,7,8,9,10. The measured potential exhibits an approximate C6 symmetry, minimal dependence on carrier density and a substantial amplitude of approximately 60 mV, even in the absence of carriers. Theory indicates that this symmetry arises from a delicate interplay of physical mechanisms with competing symmetries. The measured amplitude significantly exceeds theoretical predictions, suggesting that current understanding may be incomplete. With 1 nm spatial resolution and sensitivity to detect the potential of even a few millionths of an electron charge, the atomic SET enables ultrasensitive imaging of charge order and thermodynamic properties across a wide range of quantum phenomena, including symmetry-broken phases, quantum crystals, vortex charges and fractionalized quasiparticles. The behaviour of electrons in a lattice is governed by the periodic potential of the host material. In naturally occurring materials, this periodicity is determined by the atomic length scale, making it extremely challenging to directly image the local electrostatic potential. Over the past decade, moiré engineering has emerged as a powerful approach to create tunable periodic potentials at van der Waals (vdW) interfaces, achieving length scales substantially larger than those of atomic lattices. A canonical example is the interface between aligned graphene and hexagonal boron nitride (G/hBN), in which their lattice mismatch produces a moiré superlattice that alters the electronic properties through its periodic potential. This moiré interface has enabled numerous discoveries, including the observation of the Hofstadter butterfly3,4,5 and Brown–Zak oscillations11. More recently, when coupled with graphene multilayers, this aligned G/hBN interface has played a crucial role in stabilizing even more exotic phases, such as ferromagnetism in magic-angle twisted bilayer graphene12,13, unconventional ferroelectricity in bilayer graphene14,15 and the fractional quantum anomalous Hall effect (FQAHE) in rhombohedral pentalayer graphene16. Despite its pivotal role, so far the G/hBN moiré potential has only been inferred indirectly from transport17,18 and optical19,20 measurements. Unlike in transition metal dichalcogenides (TMDs), whose band edges can serve as direct markers for potential variations21, mapping the moiré potential in a vdW interface such as G/hBN requires a new imaging technique that combines nanometre resolution with exceptional potential sensitivity. The most sensitive tool currently available for imaging electrostatic potentials is the scanning single electron transistor (SET)22,23,24,25, which uses transport through a small island in the Coulomb blockade regime for detection. However, the spatial resolution of existing scanning SETs is constrained by their lithographic dimensions (>100 nm), preventing them from resolving potentials within a moiré unit cell. Although STM experiments with a graphene sensor layer26,27 have achieved intermediate spatial resolution, and recent advancements in AFM-based techniques have led to high-resolution imaging of molecules28,29,30, direct visualization of moiré potentials within vdW heterostructures remains an unmet challenge. In this work, we develop the atomic SET, a new scanning probe that uses a single atomic defect31,32,33,34,35,36 as a scanning potential sensor, achieving 1 nm spatial resolution, two orders of magnitude better than existing SETs, and a potential sensitivity of 5 μV Hz−1/2. This remarkable sensitivity corresponds to detecting variations of a few parts per million of the potential produced by a single electron charge at the distance given by the spatial resolution (Supplementary Information section 17). Using this tool, we directly image the potential at the G/hBN moiré interface. Our measurements show that even at zero carrier density, the peak-to-peak potential amplitude is large (about 60 mV) and exhibits an approximate C6 symmetry around the moiré centre. Although this symmetry can be explained by a subtle interplay of competing mechanisms, the magnitude of the measured potential is about twice that predicted by existing theory, highlighting that our understanding of even this simple moiré interface remains incomplete. The working principle of the scanning atomic SET is shown in Fig. A single atomic defect embedded in a thin TMD layer acts as a quantum dot, the energy level of which shifts with small changes to the local electrostatic potential ϕ(r). By monitoring this shift, we directly image ϕ(r). This inverted geometry, in which the system of interest is on the tip, enables selecting an optimal defect from a large pool of natural defects within the flat TMD layer. a, Schematic of the atomic SET geometry: the system of interest (purple), positioned on a QTM tip, is scanned across a fixed atomic defect (yellow) embedded in an insulating barrier (blue) above a graphene electrode (grey). At low temperature, the defect functions as a quantum dot (QD) exhibiting single electron transport (white arrows). As the tip scans across this defect, the spatially varying electrostatic potential ϕ(r) of the system on tip gates the defect, and by monitoring its Coulomb blockade peak, we directly map this potential. b, Illustration of an experiment to image individual atomic defects in WSe2: a bias voltage (Vb) is applied to the bottom graphene electrode and the current (I) through the tip is measured while scanning. When the tip does not overlap a defect (off-defect), the current arises from momentum-conserving tunnelling processes. However, when the contact area of the tip (white dashed line) overlaps a defect (on-defect), an additional defect-assisted tunnelling pathway opens. c, Defect imaging experiment using a QTM tip comprising aligned G/hBN layers. As the tip is scanned across trilayer WSe2 on a graphene electrode at Vb = –0.7 V, the measured I map shows multiple replicas of the tip contact area, each produced by a different defect. Within each replica, the G/hBN moiré superlattice appears as periodic modulations of I and the sharp edges (arrows) demonstrate a spatial resolution of approximately 1 nm. d, Schematic energy diagrams (top) and corresponding current maps (bottom) measured with a different G/hBN tip at T = 0.2 K and at three biases: Vb = –1.25 V (left), –0.9 V (centre) and –0.35 V (right), all within the same scan window. The energy diagrams show the interface along the z-direction: a graphene source electrode (grey), defects (yellow and grey) in a bilayer WSe2 barrier (blue) and a G/hBN moiré drain electrode (purple). As |Vb| decreases, the range of transport-accessible defects (highlighted in yellow) is narrowed, resulting in fewer observed tip-shape replicas. At the lowest bias, only a single replica remains in the entire scan window. To locate suitable defects, we map the tunnelling current (I) at a fixed bias voltage (Vb) while scanning the tip across the TMD layer (Fig. When the tip does not overlap a defect, the current reflects only background momentum-conserving elastic1 and inelastic (phonon-assisted)37 tunnelling. However, when it overlaps a defect, an additional tunnelling channel opens, allowing electrons to tunnel preferentially through the defect. This results in an increased \(I\) when the contact area of the tip (white dashed line) coincides with a defect. The scan shows several oblong shapes of increased I, each corresponding to a separate atomic defect imaging the contact area of the tip. Their identical spatial structure and increase in I suggest that these defects have the same chemical origin. In this experiment, the QTM tip consists of aligned G/hBN, forming a moiré superlattice2,3,4,5 that the defect imaging remarkably resolves even at room temperature. This measurement demonstrates extremely high spatial resolution, apparent from the sharpness of the tip image edges (about 1 nm; Supplementary Information section 6). Although we might expect that transferring a moiré heterostructure onto the QTM tip would result in significant strain that would be magnified by the moiré superlattice38, this measurement (Fig. 3a) instead shows minimal moiré heterostrain, typically less than ±0.3% (Supplementary Information section 7). Another important aspect of the defects is their energetics, which we investigate by imaging at different bias voltages at low temperature (T = 0.2 K). Figure 1d presents maps taken with another tip containing aligned G/hBN (λm = 14 nm) scanned over a bilayer WSe2 barrier and graphene electrode. All subsequent data use this tip and are taken at this temperature. Nevertheless, numerous lungs-shaped replicas of the tip contact area appear, reflecting the large set of defects accessible at this energy. As the bias is reduced, progressively fewer defects contribute, and at the lowest bias, only a single low-energy defect remains within the scan window, with minimal residual background tunnelling. We use these relatively rare, low-energy defects for imaging: their sparse distribution ensures single-defect imaging, and their low energies allow us to operate near zero bias, avoiding hot-electron and phonon injection that could interfere with measuring the thermodynamic ground state of the system. In the measurements so far, the defects served only as localized pathways for current. Now we aim to harness an individual defect as a fully functional quantum dot that can probe the local electrostatic potential and thermodynamic quantities at a specific position. This is accomplished by adding top and bottom gates to the QTM junction (Fig. In a prototypical quantum dot, applying a gate voltage Vgate linearly shifts the electrostatic potential of the quantum dot, producing a characteristic Coulomb diamond diagram (Fig. At zero bias, there is a specific Vgate in which the N and N + 1 charge states of the quantum dot are degenerate, permitting current to flow. At finite bias, the conduction window expands linearly with Vb, creating a diamond shape in the Vgate–Vb plane. a, Cross-section of the QTM junction: the tip comprises an aligned G/hBN moiré interface, an hBN layer and a graphite top gate. The sensor device contains a defect-bearing bilayer WSe2 barrier, a graphene electrode, an hBN layer and a metal bottom gate. b, Prototypical quantum dot Coulomb diamond conductance diagram versus gate voltage (Vgate) and source–drain bias (Vb), showing a central conduction region (blue) and Coulomb-blockaded regions with fixed charge states (labelled N and N + 1). c, Electrostatic relations in the junction: for each graphene layer, µ = V – ϕ. The defect potential is ϕD = αϕT + (1 – α)ϕB with α= zB/(zT + zB), where zB and zT are the distances of the defect from the bottom and top layers, respectively. The defect energy ED is referenced to ϕD. This leads to curvature in the Coulomb diamond lines, reflecting the density-dependent µ(n) of the two graphene electrodes. e, Off-defect differential conductance (dI/dV) versus top gate voltage (VTG) and Vb at T = 0.2 K and B = 0 T. Curves of reduced dI/dV correspond to the charge neutrality points (CNPs, n = 0) of the top (purple dashed) and bottom (white dashed) graphene layers. Additional suppression for |Vb| < Vth ≈ 65 mV (horizontal cyan dashed lines) arises from nonlinear contact resistance (see text). f, Off-defect measurement at \({B}_{\perp }=5\,{\rm{T}}\) showing reduced dI/dV along Landau level gaps in the top (\({\nu }_{{\rm{T}}}^{\mathrm{LL}}=\pm 2,\,\pm 6\), purple) and bottom (\({\nu }_{{\rm{B}}}^{\mathrm{LL}}=\pm 2\), white) graphene layers. g, On-defect dI/dV versus VTG and Vb at B = 0 T, exhibiting a Coulomb diamond diagram with curved boundaries that separate blockaded regions with near-zero dI/dV and fixed quantum dot charge (N and N + 1) from a high-conductance region. Arrows mark the deflection points at the CNPs of the top G/hBN (purple) and bottom graphene (white) layers. h, On-defect measurement at \({B}_{\perp }=5\,{\rm{T}}\), where the diamond edges exhibit step-like features due to Landau level gaps in both electrodes. White and purple dashed lines in e–h are fits to the electrostatic model (Supplementary Information sections 2 and 5). 2c, the electrochemical (V), electrostatic (ϕ) and chemical (µ) potentials of the two graphene layers satisfy µ = V – ϕ for each layer. Consequently, when both layers are grounded (VT = VB = 0), their local electrostatic potentials directly track their local chemical potentials. The defect experiences an electrostatic potential ϕD = αϕT + (1 – α)ϕB (Supplementary Information sections 2 and 3), with α = zB/(zT + zB) set by its relative distances to the top and bottom layers (zT, zB). The response of ϕD to a gate voltage V (top or bottom) is thus a weighted sum of the inverse electronic compressibilities dµ/dn of the top and bottom layers \(\frac{{\rm{d}}{\phi }_{{\rm{D}}}}{{\rm{d}}V}=\alpha \frac{{\rm{d}}{\mu }_{{\rm{T}}}}{{\rm{d}}{n}_{{\rm{T}}}}\frac{{\rm{d}}{n}_{{\rm{T}}}}{{\rm{d}}V}+(1-\alpha )\frac{{\rm{d}}{\mu }_{{\rm{B}}}}{{\rm{d}}{n}_{{\rm{B}}}}\frac{{\rm{d}}{n}_{{\rm{B}}}}{{\rm{d}}V}\), where dnT/dV and dnB/dV are capacitive factors determined by the junction electrostatics (Supplementary Information section 2). Consequently, the Coulomb diamond lines will curve in a way that reflects the Dirac-like µ(n) of both layers33,35,40,41,42,43 (Fig. Before performing tunnelling measurements through a defect, we first establish the electrostatics of the QTM junction, which is controlled by the top gate (VTG), the bottom gate (VBG) and the bias (Vb) voltages. Figure 2e shows the tunnelling conductance, dI/dV \(\equiv \) dI/dVb, measured off-defect as a function of VTG and Vb. The two curved lines of reduced dI/dV correspond to the charge neutrality points of the top (nT = 0) and bottom (nB = 0) graphene layers, in agreement with the electrostatic model (dashed lines, Supplementary Information section 2). For |Vb| < Vth ≈ 65 mV, there is a pronounced suppression of dI/dV. Detailed measurements (Supplementary Information section 9) indicate that this suppression is device-specific and arises from a large contact resistance that persists until the threshold bias Vth, then drops sharply. Therefore, for |Vb| < Vth, the bias primarily drops across the contact rather than across the QTM junction itself and the nT, nB = 0 lines remain vertical (essentially unaffected by bias). A similar measurement done in a perpendicular magnetic field of 5 T (Fig. 2f) shows suppressed dI/dV features associated with the Landau level gaps of the top (\({\nu }_{{\rm{T}}}^{\mathrm{LL}}=\pm 2,\,\pm 6\)) and bottom (\({\nu }_{{\rm{B}}}^{\mathrm{LL}}=\pm 2\)) layers, consistent with the electrostatic model. With additional measurements as a function of VBG (Supplementary Information section 5), we fully establish how the three voltages control the carrier densities in the QTM junction. We now measure tunnelling through a low-energy defect as a function of VTG and Vb (Fig. 2g), observing an order of magnitude higher conductance. The conductance forms a curved Coulomb diamond: along its two branches, the defect level is resonant with either the top or bottom electrode (Supplementary Information section 2), whereas outside of this region, dI/dV is strongly suppressed, corresponding to fixed quantum dot charge (labelled as N and N + 1 for generality). For |Vb| < Vth, the Coulomb blockade peak is nearly vertical, consistent with the earlier off-defect observation that the bias drops primarily on the contact in this regime. Specifically, two deflection points (arrows) correspond to the charge neutrality points of the two layers, in which the defect potential responds more strongly to gate voltage (that is, higher slope) due to the reduced compressibility of the top or bottom layer. In a similar measurement at B = 5 T (Fig. 2h), the Coulomb diamond edges show steps reflecting the Landau level gaps. Additional calibration experiments are presented in Supplementary Information section 16. Having shown that a defect can measure µ(n) at a single point, we now turn to imaging potentials in real space. Figure 3a shows a high-resolution map of I(x, y) measured through a single low-energy defect at Vb = –0.35 V, revealing the detailed moiré structure of the tip. Figure 3b shows the zero-bias dI/dV(x, VTG) measured along the white dashed line in Fig. A narrow Coulomb blockade peak is observed at each x, whose gate-voltage position \({V}_{\mathrm{TG}}^{\mathrm{peak}}(x)\) oscillates with the moiré periodicity, following \({V}_{{\rm{T}}{\rm{G}}}^{{\rm{p}}{\rm{e}}{\rm{a}}{\rm{k}}}(x)=c\cdot \phi (x)+{\rm{c}}{\rm{o}}{\rm{n}}{\rm{s}}{\rm{t}}\) (c is determined by the junction electrostatics; Supplementary Information sections 2 and 3). This scan, therefore, directly tracks the moiré electrostatic potential (Fig. a, High-resolution current map I(x, y) at Vb = –0.35 V and T = 0.2 K, showing the shape of the contact area of the G/hBN tip and the detailed moiré structure within. b, Zero-bias dI/dV versus x and VTG, measured at filling v = 3.2 ± 1. The narrow Coulomb-blockade peak oscillates with the moiré periodicity; its gate-voltage position \({V}_{\mathrm{TG}}^{\mathrm{peak}}(x)\) is converted to the electrostatic potential at the moiré interface, ϕ(x) (right y-axis), using the junction electrostatics (Supplementary Information sections 2 and 3). This measurement cuts through the high-symmetry sites of the moiré superlattice with the highest ϕ(r) (white dashed line in a). c, Extension of b to two spatial dimensions (blue dotted region in a), showing dI/dV measured versus x, y and VTG for a few slices of constant VTG. At the lowest VTG, dI/dV is practically zero everywhere. As VTG increases, rings of conductance appear and repeat with the moiré periodicity, corresponding to equipotential lines that expand and merge with increasing VTG. From the full three-dimensional dataset (Supplementary Video 1), we extract \({V}_{\mathrm{TG}}^{\mathrm{peak}}(x,y)\) (Supplementary Information section 4) and from the junction electrostatics directly determine the moiré potential, ϕ(x, y). These maps are obtained by averaging over several moiré sites (uncertainties in v arise from the VTG adjustments used to meet the quantum dot resonance condition; Supplementary Information sections 4 and 13). We set ϕ = 0 at the potential minimum. The centre and right maps use defect D1 at zero bias; the left map uses defect D2 at Vb = –0.21 V. The potential exhibits an approximate C6 symmetry, changes minimally with carrier density (about 10%) and has a substantial amplitude even at zero carrier density. High-symmetry stacking sites are indicated by red, blue and grey dots. We can now apply this technique to obtain a full 2D map of the moiré potential. Figure 3c shows the zero-bias dI/dV measured as a function of x and y (Fig. 3a, dotted box) and at several values of VTG. At the lowest VTG, dI/dV is practically zero at every (x, y) position. With increased VTG, conductance appears along rings, which repeat in the (x, y) plane with the moiré periodicity. These rings correspond to equipotential lines within the moiré unit cell. Further increasing VTG increases the radius of the rings until they merge and disappear. From a full three-dimensional map of dI/dV(x, y, VTG) (Supplementary Video 1), we extract \({V}_{\mathrm{TG}}^{\mathrm{peak}}(x,y)\) (Supplementary Information section 4), and from the relation above, directly determine the 2D moiré potential. Figure 3d shows the 2D moiré potential ϕ(x, y) across a single moiré unit cell at three fillings, ν = 0 ± 0.15, 1.3 ± 0.2 and 4 ± 1 (with ν = 1 corresponding to one electron per moiré cell). First, the moiré potential amplitude is large, ranging from 52 mV to 62 mV (Supplementary Information section 18). Second, this amplitude varies only minimally (about 10%) with moiré filling. The maps also show three distinct high-symmetry points: a central maximum (red) and two minima separated by 60° (grey and blue). The difference between these minima, Δϕminima ≈ 4 mV, is a small fraction of the overall scale. This indicates that although a minor C3 component is present, consistent with the underlying symmetry of the moiré lattice, the overall symmetry is close to C6. To interpret our observations, we consider various mechanisms that have been theoretically proposed to induce potential. 10, the moiré Hamiltonian can be decomposed into three terms: an effective pseudoelectric potential (H0), a pseudomagnetic field (Hxy) and a local mass term (Hz), each associated with a corresponding sublattice Pauli matrix (Supplementary Information section 13). Figure 4 shows the predicted stacking (Fig. 4b) pseudopotential terms comprising H0, along with the pseudomagnetic field Hxy (Fig. 4c), with the three high-symmetry CB (carbon above boron), CN (carbon above nitrogen) and AA (carbon above both boron and nitrogen) stacking sites marked. Generally, a pseudoelectric potential may reflect energy changes that are not electrostatic in nature. However, theory44 and density functional theory calculations45,46 suggest that the pseudopotentials in Fig. 4a,b are directly accounted for by charge polarization perpendicular to the layers and therefore manifest as real electrostatic potentials. a, Stacking pseudopotential due to the relative stacking of G and hBN, which varies within the moiré unit cell. The three high-symmetry points corresponding to local CB, CN and AA stacking are marked. b, Deformation pseudopotential due to atomic relaxation within the graphene layer. d–f, Self-consistent electrostatic potentials obtained after considering the screening by the graphene carriers, calculated by including a self-consistent Hartree potential response using a carrier density corresponding to v = 4. All terms show strong C3 symmetry around the moiré centre, in contrast to the C6 symmetry observed in the experiments. g, Self-consistent stacking and deformation potentials are plotted along a linecut through the moiré centre (dashed white line, bottom inset). The CB, CN and AA high-symmetry points are labelled. Visibly, each of the two terms (blue, pink) shows a strong C3 symmetry. However, owing to cancelling contributions, their sum (purple) exhibits an approximate C6 symmetry, with only a small difference between the potential minima at the CB and AA stacking sites. h, Total self-consistent potential calculated for v = 0. This potential resembles the experiment in terms of the approximate C6 symmetry, but its magnitude is half of that measured experimentally. Graphene carriers redistribute to screen these pseudopotentials, producing the self-consistent electrostatic potential measured by our detector. We model this using self-consistent Hartree calculations (Supplementary Information section 13). Notably, screening preserves the shape of the pseudoelectric potentials but reduces their magnitude by approximately 2.2, consistent with the predicted random-phase-approximation dielectric constant ϵ = 1 + πα/2 ≈ 2.0, where \(\alpha =\frac{{e}^{2}}{4{\rm{\pi }}\kappa {{\epsilon }}_{0}\hbar {v}_{{\rm{F}}}}\) is the fine-structure constant of graphene, vF is its Fermi velocity and κ = 3.5 is the hBN dielectric constant47. Furthermore, we find that electronic screening converts the pseudomagnetic field Hxy into an electrostatic potential (Fig. This potential is small compared with the other two, scales linearly with ν, and becomes identically zero at v = 0 (same for Hz; Supplementary Information section 13). As our experiments show only a minor v dependence, we omit the Hxy and Hz terms in further discussions. Both leading potential terms (Fig. 4d,e) exhibit a clear C3 symmetry around the central CN site, in contrast to the approximate C6 symmetry observed experimentally. However, examining the minima at the CB and AA sites shows that these C3 symmetries are inverted—for the first term ϕAA > ϕCB and for the second term ϕCB > ϕAA. The two terms compensate each other to form an almost C6-symmetric total potential with a pronounced central peak (Fig. The resulting total self-consistent potential (Fig. 4h) strongly resembles the experimental result, with one notable exception—the experimental potential scale is double the theoretical prediction. One explanation might be that theory underestimates strain in the moiré interface21. However, increasing strain alone would yield a more C3-symmetric potential, contrasting our observations. This large discrepancy demonstrates that, despite G/hBN being one of the most relevant and extensively studied moiré interfaces, there are substantial gaps in its theoretical understanding, which has direct consequences for recent experiments that use this interface to design new states of matter (for example, FQAHE in moiré pentalayer graphene). 1 presents potential traces measured by two defects, located at approximately 0.8 nm (D2) and 1.5 nm (D1) from the interface (Supplementary Information section 3). The measured potential decays rapidly, even over these small distances. This significant drop suggests that if the detector were at a moiré distance (h = λm) away from the interface, it would have detected only \({{\rm{e}}}^{-\frac{4{\rm{\pi }}}{\sqrt{3}}\left(\frac{h}{{\lambda }_{{\rm{m}}}}\right)\sqrt{\frac{{{\epsilon }}_{\parallel }}{{{\epsilon }}_{\perp }}}}\) ≈ 10–4 of the potential, underscoring the importance of our atomic SET operating at extremely close standoff distances. At the same time, these measurements also demonstrate that in thin flakes, such as pentalayer graphene, electrons can still experience a significant moiré potential (tens of mV) even in the farthest graphene layer. The atomic SET scanning probe technique demonstrated here has a combination of features that are extremely powerful for studying a wide range of quantum materials. Its QTM geometry allows it to scan within the pristine interfaces of a variety of vdW materials. Similar to existing SETs, this technique will allow quantitative measurements of thermodynamic properties such as the electronic compressibility24,25,48 and entropy49, but now with two orders of magnitude improved spatial resolution—below the Fermi wavelength, magnetic length and moiré scales of many systems. This advance extends this powerful imaging method to a much broader class of physical phenomena occurring on small scales such as Wigner crystals, topological edge states, vortex charges, symmetry-broken phases and fractionally charged quasiparticles. The data shown in this paper are provided at zenodo.org/records/17689195. Additional data that support the plots and other analysis in this work are available from the corresponding author upon request. Emergence of superlattice Dirac points in graphene on hexagonal boron nitride. Massive Dirac fermions and Hofstadter butterfly in a van der Waals heterostructure. Dean, C. R. et al. Hofstadter's butterfly and the fractal quantum Hall effect in moiré superlattices. Ponomarenko, L. A. et al. Cloning of Dirac fermions in graphene superlattices. Generic miniband structure of graphene on a hexagonal substrate. Woods, C. R. et al. Commensurate–incommensurate transition in graphene on hexagonal boron nitride. & MacDonald, A. H. Ab initio theory of moiré superlattice bands in layered two-dimensional materials. San-Jose, P., Gutiérrez-Rubio, A., Sturla, M. & Guinea, F. Electronic structure of spontaneously strained graphene on hexagonal boron nitride. Jung, J. et al. Moiré band model and band gaps of graphene on hexagonal boron nitride. Kumar, R. K. et al. High-temperature quantum oscillations caused by recurring Bloch states in graphene superlattices. Sharpe, A. L. et al. Emergent ferromagnetism near three-quarters filling in twisted bilayer graphene. Serlin, M. et al. Intrinsic quantized anomalous Hall effect in a moiré heterostructure. Unconventional ferroelectricity in moiré heterostructures. Klein, D. R. et al. Electrical switching of a bistable moiré superconductor. Fractional quantum anomalous Hall effect in multilayer graphene. Ballistic miniband conduction in a graphene superlattice. Dynamic band-structure tuning of graphene moiré superlattices with pressure. Shi, Z. et al. Gate-dependent pseudospin mixing in graphene/boron nitride moiré superlattices. Ni, G. X. et al. Plasmons in graphene moiré superlattices. Deep moiré potentials in twisted transition metal dichalcogenide bilayers. Yoo, M. J. et al. Scanning single-electron transistor microscopy: imaging individual charges. Honig, M. et al. Local electrostatic imaging of striped domain order in LaAlO3/SrTiO3. Fractional Chern insulators in magic-angle twisted bilayer graphene. Yu, J. et al. Correlated Hofstadter spectrum and flavour phase diagram in magic-angle twisted bilayer graphene. Imaging two-dimensional generalized Wigner crystals. Chiu, C.-L. et al. High spatial resolution charge sensing of quantum Hall states. Wagner, C. et al. Scanning quantum dot microscopy. Hapala, P. et al. Mapping the electrostatic force field of single molecules from high-resolution scanning probe images. Bolat, R. et al. Electrostatic potentials of atomic nanostructures at metal surfaces quantified by scanning quantum dot microscopy. Chandni, U., Watanabe, K., Taniguchi, T. & Eisenstein, J. P. Evidence for defect-mediated tunneling in hexagonal boron nitride-based junctions. Characterization and manipulation of individual defects in insulating hexagonal boron nitride using scanning tunnelling microscopy. Tunnel spectroscopy of localised electronic states in hexagonal boron nitride. Approaching the intrinsic limit in transition metal diselenides via point defect control. Keren, I. et al. Quantum-dot assisted spectroscopy of degeneracy-lifted Landau levels in graphene. Roelcke, C. et al. Ultrafast atomic-scale scanning tunnelling spectroscopy of a single vacancy in a monolayer crystal. Birkbeck, J. et al. Quantum twisting microscopy of phonons in twisted bilayer graphene. Cosma, D. A., Wallbank, J. R., Cheianov, V. & Fal'ko, V. I. Moiré pattern as a magnifying glass for strain and dislocations in van der Waals heterostructures. & Steinberg, H. Spectroscopy of NbSe2 using energy-tunable defect-embedded quantum dots. Robust quantum oscillation of Dirac fermions in a single-defect resonant transistor. Seo, Y. et al. Defect-assisted tunneling spectroscopy of electronic band structure in twisted bilayer graphene/hexagonal boron nitride moiré superlattices. Vdovin, E. E. et al. A magnetically-induced Coulomb gap in graphene due to electron-electron interactions. & Shockley, W. Deformation potentials and mobilities in non-polar crystals. Bokdam, M., Amlaki, T., Brocks, G. & Kelly, P. J. Band gaps in incommensurable graphene on hexagonal boron nitride. Wang, H. et al. Ferroelectric polarization of graphene/h-BN bilayer of different stacking orders. Hwang, E. H. & Sarma, S. D. Dielectric function, screening, and plasmons in two-dimensional graphene. Zondiner, U. et al. Cascade of phase transitions and Dirac revivals in magic-angle graphene. Rozen, A. et al. Entropic evidence for a Pomeranchuk effect in magic-angle graphene. We thank A. MacDonald, H. Steinberg, O. Hod, W. Cao, and Y. Meir for their discussions. This work was supported by the Israel Science Foundation (grant no. 1621/24), the Leona M. and Harry B. Helmsley Charitable Trust, the Rosa and Emilio Segre Research Award, the ERC-Adg (QTM, grant no. 2020260) and the SNF Sinergia (grant no. acknowledges support from the Zuckerman STEM Leadership Program. M.M.A.E., L.P. and S.A. acknowledge support from the Singapore National Science Foundation Investigator Award (NRFNRFI06-2020-0003) and the Singapore Ministry of Education AcRF Tier 2 grant (MOE-T2EP50220-0016). We thank G. Atiya from the Materials Science and Engineering Department, Electron Microscopy Center, Technion, for her assistance with the focused ion beam patterning. These authors contributed equally: Dahlia R. Klein, Uri Zondiner Department of Condensed Matter Physics, Weizmann Institute of Science, Rehovot, Israel Dahlia R. Klein, Uri Zondiner, Amit Keren, John Birkbeck, Alon Inbar, Jiewen Xiao, Yuval Zamir, Mariia Sidorova & Shahal Ilani Laboratory of Nanooptics and Plasmonics, Moscow Institute of Physics and Technology, Dolgoprudny, Russia Mohammed M. Al Ezzi & Shaffique Adam John A. Paulson School of Engineering and Applied Sciences, Harvard University, Cambridge, MA, USA Department of Physics, Washington University in St Louis, St Louis, MO, USA National Institute for Materials Science, Tsukuba, Japan Department of Physics and Astronomy, University of Pennsylvania, Philadelphia, PA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar built the millikelvin scanning microscope. M.M.A.E., L.P. and S.A. wrote the theoretical model. wrote the paper with input from other authors. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Linecuts along high-symmetry points of the moiré potential, \({\phi }_{m}^{D}\), measured as a function of position, x, using two defects (D1 and D2). These defects are located at different heights from the G/hBN interface (1.5 nm and 0.8 nm, respectively) and measured at moiré fillings v = 4 ± 1 and v = 3.2 ± 1. We also plot the potential at the moiré interface (ϕ(x), dashed lines) deduced using the calibrated junction electrostatics (Supplementary Information sections 2 and 3) from the measurements with both defects. These measurements demonstrate that even at such small heights, the decay of the moiré potential amplitude is substantial (~60% measured decay from 0.8 nm to 1.5 nm). Full measurement of conductance compared with position and top gate voltage. Video showing full three-dimensional measurement of conductance dI/dV as a function of position (x, y) within the moiré superlattice and top gate voltage VTG. The data were obtained using defect D2 at zero d.c. bias Vb and bottom gate voltage VBG = 0 with an a.c. bias excitation of 1 mV root mean square. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Klein, D.R., Zondiner, U., Keren, A. et al. Imaging the sub-moiré potential using an atomic single electron transistor. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            