
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-07-18</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/energy/a65432654/wendelstein-7x-germany-stellarator-fusion-record/'>Scientists Are Now 43 Seconds Closer to Producing Limitless Energy</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 18:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Nuclear energy as we know it today is generated through fission—the process of splitting atomic nuclei, which produces electricity without emitting greenhouse gases. Fission also creates radioactive waste as a byproduct of using uranium fuel. That's where nuclear fusion—or, at least, the promise of it—comes in. Nuclear fusion is the same process that powers stars, like our Sun. Inside a star, hydrogen nuclei slam together under intense pressure and heat, fusing into helium and releasing a huge burst of energy. Scientists have long wanted to recreate fusion on Earth because it could deliver clean, virtually limitless power with no carbon emissions and much less radioactive waste than today's nuclear plants. But the other serious design contender is called a stellarator. In a recent experiment, researchers at the Max Planck Institute for Plasma Physics sustained a stable plasma reaction in the Wendelstein 7-X for 43 seconds, reaching the “triple product” performance level that's required for viable nuclear fusion and achieving the all-time best results for any stellarator. Maybe 43 seconds doesn't sound like much, but it's now the longest plasma duration ever in nuclear fusion, including tokamaks. The triple product measures how close a fusion reactor is to reaching self-sustaining ‘breakeven' energy production levels—the point at which a reactor produces more energy than it consumes. Unlike fission reactors, which use heavy elements such as uranium, fusion reactors rely on lighter elements (like the hydrogen ions deuterium and tritium), which help them avoid creating toxic nuclear waste. The fuel injector for Wendelstein 7-X was created exclusively for this stellarator by scientists at the U.S. Department of Energy's Oak Ridge National Laboratory (ORNL), which has also been instrumental in advancing other types of reactors. As plasma was heated with microwaves over those record-breaking 43 seconds, the fuel injector provided the stellarator with 90 pellets of frozen hydrogen ions. The microwave heating system used electron cyclotron resonance—the most effective method for fusion, which uses electromagnetic waves to heat and ionize plasma to temperatures that (in this case) reached a scorching 30 million degrees Celsius (about 54 million degrees Fahrenheit). Because success depended on balancing heating with pulses of incoming fuel, the pellet injector was programmed with automatic pulse rates, which is an unprecedented move. It impressively demonstrates the potential of Wendelstein 7-X,” Prof. Dr. Thomas Klinger of the Max Planck Institute for Plasma Physics said in a press release. “Elevating the triple product to tokamak levels during long plasma pulses marks another important milestone on the way toward a power-plant-capable stellarator.” Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65436004/ancient-wooden-face-discovery-poland/'>Divers Excavated a Lake—and Stumbled Upon a Haunting Ancient Face</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>While its mouth is closed, this artifact speaks volumes about the medieval past. Many ancient wooden structures have been lost to time, as wood unfortunately decomposes quickly. And recently, archaeologists from Nicolaus Copernicus University (NCU) discovered a striking example of that preservation—a 1,000-year-old wooden face during an underwater research expedition. The team recently presented its findings at a press conference. This isn't the first face of its kind to be discovered. Similar depictions have been found in other Slavic cities including Wolin, Novgorod the Great, and Staraya Ladoga. According to experts, consistencies between these finds suggest that the beam found at Lednica reflects a local artistic and spiritual tradition, rather than Scandinavian or Rus influence. “This discovery not only evokes admiration for the craftsmanship from over a thousand years ago but also opens a fascinating discussion about the spiritual life of early medieval Slavs,” Andrzej Pydyn, director of the Centre for Underwater Archaeology at NCU, said in the press release. The researchers also discovered horse jaws and bones around ramparts, which may have come from ritual offerings or protective sacrifices, Mateusz Popek from NCU said. The team also found similar bones underneath houses. “The [wood] face is a priceless monument and a rare example of an item combining utility and symbolic functions,” a post from the Museum of the First Piasts in Lednica reads. “The discovery from Lake Lednica not only impresses, but also opens new questions about the life, culture and spirituality of the inhabitants of these lands from over a thousand years ago.” Her work appears in several publications including Biography.com and Popular Mechanics. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. Archaeologists Found a Lost Site of Friendly Fire Knight Found Buried Under Former Ice Cream Parlor Archaeologists May Have Found a Maya King's Tomb</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09225-2'>Pathology-oriented multiplexing enables integrative disease mapping</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 13:28:18
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The expression and location of proteins in tissues represent key determinants of health and disease. Although recent advances in multiplexed imaging have expanded the number of spatially accessible proteins1,2,3, the integration of biological layers (that is, cell structure, subcellular domains and signalling activity) remains challenging. This is due to limitations in the compositions of antibody panels and image resolution, which together restrict the scope of image analysis. Here we present pathology-oriented multiplexing (PathoPlex), a scalable, quality-controlled and interpretable framework. It combines highly multiplexed imaging at subcellular resolution with a software package to extract and interpret protein co-expression patterns (clusters) across biological layers. PathoPlex was optimized to map more than 140 commercial antibodies at 80 nm per pixel across 95 iterative imaging cycles and provides pragmatic solutions to enable the simultaneous processing of at least 40 archival biopsy specimens. In a proof-of-concept experiment, we identified epithelial JUN activity as a key switch in immune-mediated kidney disease, thereby demonstrating that clusters can capture relevant pathological features. PathoPlex was then used to analyse human diabetic kidney disease. The framework linked patient-level clusters to organ disfunction and identified disease traits with therapeutic potential (that is, calcium-mediated tubular stress). Finally, PathoPlex was used to reveal renal stress-related clusters in individuals with type 2 diabetes without histological kidney disease. In summary, PathoPlex paves the way towards democratizing multiplexed imaging and establishing integrative image analysis tools in complex tissues to support the development of next-generation pathology atlases. Spatial biology technologies have gained increased attention recently as they provide molecular insights into transcriptomic and proteomic expression while preserving histological context1. The term multiplexed imaging refers to the expansion of antibody-based labelling beyond conventional limits (that is, 3–4 antibodies per section)2,3. Multiple commercial systems are available with varying performance and cost. For example, methods based on mass spectrometry4,5 require specialized equipment and antibody conjugation to metals, enabling spatial projections with high precision and reproducibility at cellular resolution (between 250 and 1,000 nm per pixel). Alternatively, microscopy-based methods6,7 are more economically accessible and rely on the cyclic detection of DNA-conjugated antibody panels or direct immunofluorescence using fixed integrated widefield microscopy. Although such methods achieve an image resolution of 200–300 nm per pixel, there is a trade-off between detection speed and signal amplification. Results from studies that used both mass spectrometry and microscopy-based methods8,9 aligned well with comprehensive reviews of the literature10 that reported panels ranging between 30 and 60 antibodies. This body of work set the foundation for the development of image analysis strategies that focused on the identification of cell identities and states through cell segmentation11,12,13,14. In 2018, iterative indirect immunofluorescence imaging (4i)15 was introduced as an open-source tool for multiplexed imaging and advanced image analysis. These techniques were based on the use of unmodified commercial antibodies in cyclic rounds of immunofluorescence imaging through simple steps of chemical elution and flexible light microscopy. 4i was originally applied in vitro using 41 antibodies at a resolution of 165 nm per pixel, which enabled the detection of functional multilayered subcellular features of cell injury through pixel-level analysis. To our knowledge, there is only one study that recreated the original 4i protocol in multicellular specimens16 with sufficient multiplexed imaging depth (21 imaging cycles for 54 markers) and image resolution (160 nm per pixel) to perform pixel-based image analysis. However, despite being one of the largest and most complex datasets available, the outputs derived from multiplexed imaging have primarily been used to recapitulate known cellular events during organ development. In this context, we postulate that the potential of multiplexed imaging methods to define tissue-based integrative features associated with health and disease remains underexplored. A study10 that discussed the current landscape of antibody-based multiplexed imaging showed that there is a diverse range in performance among the methods. From all the different criteria that can be used to define the advantages and limitations of each method, we propose two criteria to evaluate the potential to support image analysis tools that aim to integrate multiple biological layers (Supplementary Fig. Although it is evident that panel size directly affects the scope of processes that can be analysed, image resolution and the biological insights gained from it are harder to appreciate. To illustrate the importance of image resolution, we compared a mass-spectrometry-based method (Supplementary Fig. 1c) for analysing kidney samples using markers of cell identity and DNA. This comparison highlighted an obvious resolution mismatch that had a clear impact on the ability to delineate subcellular structures (for example, nuclei and even nucleoli) and the borders of neighbouring cells (for example, renal endothelial and epithelial cells). Among the reported multiplexing methods10, the average panel size is approximately 37 markers with an average resolution of 267 nm per pixel. The most used systems, such as imaging mass cytometry (IMC; 40 markers at 1,000 nm per pixel) and co-detection by indexing (CODEX; 56 markers at 250 nm per pixel), provide reliable references of current commercial standards. Thus, it is not surprising that most studies in the field of antibody-based spatial proteomics fundamentally rely on single-cell segmentation as a core step, similar to the approaches used in spatial transcriptomics17,18. Furthermore, most studies of organs that have high cell density (for example, the kidney) typically report cell identity and state19,20 but do not provide integrative data across biological domains. These limitations represent an opportunity for the next generation of multiplexed imaging methods to scale panel sizes beyond current limits. Moreover, computational tools can be built to extract hallmarks of health and disease by weighting and connecting the contributions of each biological layer (Supplementary Fig. Here we introduce PathoPlex, a scalable, quality-controlled and interpretable framework. It combines highly multiplexed imaging at subcellular resolution with an open-source software package to facilitate integrative analyses of formalin-fixed paraffin-embedded (FFPE) specimens (Fig. In brief, multiplexed imaging is performed in iterative cycles, whereby indirect immunofluorescence labelling is conducted first, followed by image acquisition by fluorescence microscopy (for example, widefield or confocal) and subsequent antibody elution (Fig. To prevent tissue lifting, we recommend coating the glass surfaces with poly-d-lysine for small-scale experiments or with (3-aminopropyl)triethoxysilane (APTES) for large-scale experiments, as APTES is more efficient at preventing tissue detachment compared with poly-d-lysine (Methods). In this report, our largest experiment included 95 imaging cycles with antibodies against 150 proteins and 20 quality-control imaging cycles with only secondary antibodies for a total of 170 layers. After detailed examination, we included 142 (122 protein and 20 quality control) layers for analyses, which generated >600 billion available pixels. It is worth noting that the tissues remained stable and did not show signs of damage within 95 imaging cycles, which suggests that this is not the limit of the technology. c, Summary of all experimental datasets in this study. To accommodate the scale of these datasets and to enable modular composition and extendibility of bioinformatic analyses, we developed a high-performance computing library for spatial proteomics (which we term spatiomic) that leverages various algorithms based on graphics processing units (GPUs)21,22, integrates common data formats23 and is freely available as a Python package through the PyPi registry (Fig. The package spatiomic features multiple registration algorithms to align images of individual markers for joint analyses. To identify protein co-expression patterns, spatiomic includes modules to preprocess images, obtain a representative subsample, reduce dimensionality using self-organizing maps (SOMs), construct a similarity-based neighbourhood graph and perform graph clustering24. Co-expression patterns can be consistently identified across all images of an experimental dataset and spatially projected. As these co-expression patterns are generated on the basis of pixel-level clustering, from now on, we refer to them as ‘clusters'. Each cluster has the potential to represent a biological process and warrants further interpretation (Fig. As a first step, the individual contribution of each marker to the cluster was analysed to define the specific co-expression pattern that each cluster represents. As each marker represents proteins with known or predicted locations, distributions and expression patterns, it can be projected back into space for visual validation. Cluster abundance was used as a quantifiable metric to statistically compare conditions and to isolate differentially expressed clusters. Notably, changes in cluster abundance can result not only from differences in protein expression levels but also from changes in protein distribution (for example, cytoplasmic to nuclear shifts). As an overview, we first provided proof-of-principle and quality-control datasets in three different organs (<30 markers at a resolution of 160 nm per pixel). PathoPlex was then validated using the kidney as a model organ with high cellular density and structural complexity through in-depth analyses of three additional datasets (Fig. These datasets were obtained from the following sources: (1) an experimental mouse model of immune-mediated kidney disease (34 markers at 80 nm per pixel); (2) clinical biopsy samples from individuals diagnosed with advanced diabetic kidney disease (DKD) (61 markers at 160 nm per pixel); and (3) research biopsy samples from individuals diagnosed with youth-onset type 2 diabetes (T2D) (142 markers at 80 nm per pixel) without pathological signs of DKD, including a subset of individuals with short-term treatment with SGLT2 inhibitors. Proof-of-principle experiments were performed on the basis of representative samples from autoimmune hepatitis, meningioma and focal segmental glomerulosclerosis (Supplementary Fig. 4) showing broad applicability in pathology and a wide potential for marker selection, including transcription factors, enzymes, structural proteins, subcellular domains, cell surface receptors and phosphorylation targets. Quality-control criteria for PathoPlex were first established in murine tissues and then extended to human specimens. In brief, consecutive imaging cycles of an antibody panel constituted the first level of control. This step was important because incomplete elution might lead to cross-reactivity with subsequent cycles or residual signals from the previous cycle. The second level of control involved direct imaging after elution to confirm the lack of fluorescent signals (Extended Data Fig. The third level of control included imaging cycles using secondary antibodies without previous incubation of primary antibodies (secondary-only cycles). This step ensured the absence of remnant viable primary antibodies and generated additional layers that could be included in image analyses (Extended Data Fig. The fourth level of control involved successful re-staining after multiple imaging cycles (Extended Data Fig. This stage was used to confirm that the epitope is preserved and the effectiveness of antibody elution. Furthermore, we applied practical quality-control steps for human tissue samples throughout 95 imaging cycles. This strategy showed complete elution efficiency using secondary-only cycles (Extended Data Fig. Once all the imaging cycles were completed, image alignment was performed to account for potential shifts during the various cycles. It is well established that nuclei can be easily stained, but commonly used labels are either unstable (for example, 4′,6-diamidino-2-phenylindole (DAPI)) or expensive (for example, DRAQ5). For this reason, we introduce N-hydroxysuccinimide ester (NHS-E), a pan-protein label commonly used in super-resolution microscopy25. NHS-E consistently generated reference images for alignment and showed equally high performance compared with nuclear references (Supplementary Fig. Moreover, NHS-E can be used to segment tissue-containing areas to limit the analysis of regions with potential nonspecific binding. Unlike DAPI or DRAQ5, which need constant re-staining every imaging cycle, NHS-E requires a single application at the beginning of the protocol and remains stable for up to 95 cycles. PathoPlex combines different strategies to optimize performance and to minimize the potential introduction of batch effects, including adaptable microscopy, accessible and customizable imaging set-ups and low-cost automatization of liquid handling (Extended Data Fig. PathoPlex can be implemented using any inverted system for fluorescence microscopy, including widefield, spinning disk and confocal, which provides flexibility in terms of image resolution, scanning time and file size (Extended Data Fig. It is worth mentioning that classical pathology protocols and some multiplexing technologies may inadvertently introduce batch effects, as specimens are processed as individual slides. By contrast, PathoPlex uses imaging chambers that enable the parallel processing of multiple tissues in single runs. Each imaging chamber is organized as an independent and self-contained experiment by including both control and experimental samples (Extended Data Fig. Considering the size of average unmodified histopathological samples, commercial solutions can be used to process between 2 and 24 intact samples at the same time (Extended Data Fig. However, as the number of wells increases, manual pipetting increases the likelihood of user error. Although this source of error can be mitigated through automation, commercially available liquid-handling systems are often expensive and not accessible to the wider scientific community. For this reason, PathoPlex introduces two practical 3D printing-based strategies to simplify liquid handling. The first approach involved the creation of a large unified single-well imaging chamber (11 × 7.4 cm) using a 3D-printed frame (Extended Data Fig. 9a) that can hold 40 intact human kidney biopsy samples (approximately 100 mm2 in size) and even higher numbers of smaller biopsy samples (for example, with size extrapolation, this equates to approximately 77 skin biopsy samples). To achieve this, we repurposed a 3D printer as a low-cost liquid handling system, with the printer head controlling liquid addition and removal (Extended Data Fig. This approach produced successful staining and elution cycles (Extended Data Fig. 2g), saving approximately 70% hands-on time with minimal user input (Supplementary Fig. Although an automated solution for multiplexed imaging using 4i principles has been previously reported26, our universal framework provides users with the flexibility to design their experiment according to their needs, including sample size and image resolution. Next, we performed a proof-of-concept experiment, whereby PathoPlex was used to analyse the pathophysiology of a well-characterized mouse model of immune-mediated kidney disease27. These mice exhibit a clear disease course that ranges from acute injury to crescentic glomerulonephritis (CGN). That is, protein loss in the urine (proteinuria), the subsequent development of pathological lesions (crescents) in the renal-filtering units (glomeruli) and progressive loss of kidney function. A total of 34 markers were used at a resolution of 80 nm per pixel to acquire approximately 5 billion pixels in 40 regions of interest (ROIs) centred on individual glomeruli (Fig. The antibody panel was designed to detect cell identities, subcellular compartments and signalling pathway activity (Supplementary Table 1). Significant changes in cluster abundance during the disease course (Fig. 2b) were used as a metric to define integrative features (Fig. a, Schematic overview for the proof-of-concept experiment in a mouse model of immune-mediated kidney disease before (acute injury) and after (CGN) pathological lesion formation (n = 10 mice; ROIs = 40). NTS, nephrotoxic serum; details of the antibody panel are provided in Supplementary Table 1. b, Spatiotemporal distribution of colour-coded clusters. Each dot represents an ROI, which was used as an independent observation (n = 11 ROIs for controls, n = 11 ROIs for acute injury and n = 18 ROIs for CGN) and red bars represent medians and inter-quartile ranges. d, Identification of C21 (with pJUN as a top contributor) as a key regulated pathomechanism before and after lesion formation. e, Images of the spatiotemporal distribution of C21 (left) and cell-specific frequency (right) among tubular epithelial cells and PECs. f, Treatment with a JNK inhibitor (JNKi) reduces the PDGF-mediated collective migration of murine PECs in vitro. In ‘collective migration', error bars represent upper and lower limits. Data are from four biological replicates. g, Confirmation of pJUN expression in PECs during different lesion stages among human kidney biopsy samples (n = 12 patients and n = 3 healthy individuals), which was also associated with CD44 co-expression. h, Schematic overview of the use of a JNKi as a preventive strategy (before NTS) and a therapeutic strategy (7 days after NTS) during the progression of immune-mediated kidney disease in a rat model of CGN. i,j, Proteinuria (n = 4 rats for all groups) and glomerular damage (n = 4 rats for day 0, n = 6 rats for all other groups; red bars represent medians and interquartile ranges) show a direct preventive (i) and therapeutic (j) effect of the JNKi. k, Using expression of CD44 as a readout of PEC activation, we confirmed the effect of the JNKi on PEC activation (using all rats available from i and j). Differential cluster abundance analysis used a two-sided t-test. Cluster composition analysis relied on a two-sided t-test with Holm–Šidák correction. For other comparisons, two-sided Mann–Whitney, Kruskal–Wallis with Dunn, analysis of variance (ANOVA) with Dunnett T3 or ANOVA with Holm–Šidák tests were used depending on the number of comparisons. Scale bars, 50 µm (c,e,g,k). Diagrams in a, f and h were created using BioRender (https://biorender.com). Additional histopathological sections from the same experimental animals were carefully evaluated by two expert renal pathologists in a blinded fashion, who were specifically asked to diagnose and stage disease and to quantify structural changes (Extended Data Fig. Images of CGN were defined by significant vascular injury and the presence of crescentic lesions, whereas acute injury was determined by the vacuolation of tubular cells (Extended Data Fig. In line with these findings, cluster 26 (which included contributions from early endosome antigen 1 and ezrin) was more abundant in disease than in control samples (Extended Data Fig. Notably, the spatial distribution of cluster 26 corresponded to the location and pattern used by expert pathologists for diagnosis and staging (Extended Data Fig. This result suggests that PathoPlex can detect pathogenesis-related tissue alterations— similar to those used by human experts—in an unsupervised manner. Previous studies have shown that modulation of JNK signalling can lead to substantial protective effects in kidney autoimmunity and fibrosis32,33. The proposed cellular targets of JNK inhibitors have mostly been immune cells (that is, activated macrophages). During crescent formation, these cells are activated through processes that are mediated by growth factors (for example, platelet-derived growth factor (PDGF))34, and their increased potential for proliferation and migration is regulated through the de novo expression of the glycoprotein CD44 (ref. We performed bulk RNA sequencing on nuclei isolated from mice with immune-mediated kidney disease to clarify this issue (Extended Data Fig. Analyses of the differentially expressed genes (Extended Data Fig. 4c), as calculated from the differential expression of JUN-regulated targets (Extended Data Fig. Notably, JUN has a crucial role in AP-1 activation through JNK, which mediates CD44 signalling37. As a readout of JUN activity, its phosphorylated protein product JUN(Ser63) (pJUN) was included in our antibody panel. Cluster 21 featured pJUN as a top contributor and was consistently increased in both acute and CGN disease states compared with controls (Fig. Cluster 21 was essentially restricted to PECs and tubular cells, with a high frequency in tubular cells during acute injury and a gradual increase in PECs during disease progression to CGN (Fig. As tubular cells do not represent an effector population during crescent formation, we turned our full attention to the role of JUN activity in PECs. As an initial validation step, we evaluated the effect of JUN activity modulation in PECs. To this end, PEC activation (that is, increased migration) was induced in vitro using PDGF36. PEC migration was attenuated with the JNK inhibitor (JNKi) CC930 (also known as tanzisertib) in two independent experimental set-ups. Results from both of these experiments confirmed that CC930 has a direct effect on activated murine PECs (Fig. In a second validation step, we analysed human biopsy samples from patients diagnosed with CGN to delineate JUN activity during the progression of human crescentic lesions (n = 12 patients and n = 3 healthy participants). Normal glomeruli from healthy individuals and from individuals with CGN showed that pJUN was expressed in scattered PECs without CD44 expression. As pathological lesions in CGN develop in a focal pattern, some glomeruli appeared normal and only a subset exhibited crescent characteristics, all in the same patient sample. 2g), which indicated an association between JUN activity and PEC activation in human specimens. In a third validation step, CGN was modelled in rats to test the efficacy of CC930 as a preventive strategy (before disease induction) or as a therapeutic strategy initiated 7 days after disease induction (Fig. Proteinuria was substantially decreased in the preventative study (Fig. 2i) and glomerular damage was mitigated with interventional treatment (Fig. 2j), which included substantial modulation of CD44 expression in PECs (Fig. Together, these data confirmed that PathoPlex-derived clusters can identify actionable pathological features with high spatial precision. The performance of PathoPlex was tested in clinical specimens with patient-level heterogeneity in one of the most common clinical features of end-organ damage in diabetes, namely DKD38. A total of 38 human kidney specimens (from 18 individuals without diabetes (controls) and 20 individuals with advanced DKD) were profiled in 422 ROIs using 61 markers (Supplementary Table 3) to obtain >100 billion pixels at 160 nm per pixel (Fig. PathoPlex identified 18 clusters with differential abundance between control and DKD samples (Supplementary Table 4). For example, cluster 19 (with contributors from apoptosis inducing factor mitochondria associated 1 (AIFM1) and transient receptor potential cation channel subfamily C member 6 (TRPC6)) was increased in tissues from individuals with DKD and localized primarily in the proximal tubules. This result was corroborated when projected onto conventional histopathology images (Fig. We validated the expression of TRPC6 in proximal tubules using both immunogold in electron microscopy (Extended Data Fig. 6a) and IMC-based antibody expression (Extended Data Fig. 6c) also confirmed that TRPC6 expression is increased in proximal tubules (Extended Data Fig. a, Schematic overview of the experimental design to compare control and DKD specimens (n = 38 18 controls, 20 DKD; ROIs = 422). Details of the antibody panel are provided in Supplementary Table 3. We show the example of C19, which represents metabolic tubular injury (with TRPC6 and AIFM1 as top contributors). c, Single-cell segmentation reveals disease-specific cell-level metaclusters (MCs). For this example, cluster 28 represents ECM remodelling and is inversely associated with estimated glomerular filtration rate (eGFR). e, Unsupervised bicluster analysis for patient stratification differentiates between DKD and control specimens with perfect accuracy and isolates a subset of biologically meaningful clusters. The top contributors of clusters selected in b were used as a DKD signature that was extended using open-access tools (that is, STRING), and then cross-referenced to the CTD to select a subset of drugs. Multiple medications for the standard care of diabetes interacted with our expanded DKD signature. g, Drug–protein interactions were quantified for our DKD signature. One example is PDE5 inhibitors as potential modulators of TRPC6–AIFM1 through cGMP signalling, which was confirmed through a re-analysis of public single-nucleus RNA-sequencing data47. Differential cluster abundance analysis used a two-sided t-test with Benjamini–Hochberg correction. Cluster composition analysis relied on a two-sided t-test with Holm–Šidák correction. Correlation analysis was performed using two-sided Spearman's rank coefficient. For other comparisons, two-sided t-test, Mann–Whitney or Kruskal–Wallis tests were used depending on the number of comparisons. Diagrams in a,e,f and g were created using BioRender (https://biorender.com). Our analysis revealed multiple additional differentially regulated clusters that reflected known biological processes, including ERK-mediated and integrin-mediated signalling in multiple nephron segments (Extended Data Fig. To connect these findings to cell-level features, we performed deep-learning-based cellular segmentation using CellPose39 (Supplementary Fig. 10) to characterize the co-occurrence of multiple clusters in well-defined cell-level metaclusters. For example, metacluster 16, which was increased in DKD, contained clusters that represented multiple processes associated with proximal tubule injury (Fig. Notably, a subset of clusters showed strong correlations with kidney function (Extended Data Fig. 8), including cluster 28 (extracellular matrix (ECM) remodelling) (Fig. 3d), thereby linking subcellular pathological features to patient-level organ function. To further validate the biological relevance of PathoPlex-derived clusters, we calculated the multivariate cluster join counts for each biopsy sample independently and then averaged them at the condition level (Extended Data Fig. These subcellular and intercellular spatial networks recapitulated aspects of kidney architecture by arranging them into functional compartments, including glomerular and tubular segments, individual cell types (for example, podocytes) and ECM. Moreover, the networks highlighted pathophysiological changes (for example, an increased connection between proximal tubule microtubules and Ca2+ signalling). Next, we applied a nonlinear co-occurrence prediction model across cell-sized windows (MISTy)40, which identified groups of condition-specific mutually predictive clusters that reflected functional (for example, glomerular, tubular and interstitial) and subcellular (for example, nuclear or cytoplasmic) compartments that defined immune activation, ECM remodelling, metabolic stress and cell injury (Extended Data Fig. We also performed image-level pseudotime analysis with a multiscale model41 to propose a path from individuals without diabetes but with varying kidney function to individuals with DKD. This analysis resulted in the identification of two potential trajectories of pathogenesis that showed a strong association with histopathological changes (Extended Data Fig. For trajectory 1, determinant features included tubulointerstitial fibrosis, which reflected the loss of kidney function in a subpopulation of individuals without diabetes (Extended Data Fig. For trajectory 2, specific features included podocyte injury, Ca2+-mediated mitochondrial stress in proximal tubules and glucocorticoid receptor (GR) dysfunction, which reflected diabetic end-organ damage in individuals with impaired kidney function (Extended Data Fig. To reinforce the value of PathoPlex as a foundational tool to perform unsupervised disease phenotyping, we used UnPaSt42 to conduct label-free biclustering based on cluster abundances. UnPaSt was able to accurately discriminate between control and DKD samples (Fig. Bicluster-specific clusters reflected the increased abundance of functional integrity features (that is, podocyte physiology and metabolism) in control specimens, and of pathogenic features in DKD samples (that is, macrophage infiltration, immune activation, AIFM1–TRPC6 signalling, endoplasmic reticulum (ER) stress, ECM remodelling, and GR, β-catenin, histone H2B and ubiquitylation dysfunction). In summary, PathoPlex-derived clusters can be immediately used to extend the scope of computational analyses to add layers of biological context (that is, pseudotime, niche profiling and feature subclassification) and to connect PathoPlex to the broader computational spatial biology ecosystem. Next, we aimed to leverage PathoPlex-derived clusters to infer additional clinically relevant information, such as potential opportunities for drug repurposing. First, we selected the top contributing proteins from each cluster to define a cluster-based DKD signature that was extended using the search tool for the retrieval of interacting genes and proteins (STRING)43. Then we cross-referenced our extended DKD signature with the Comparative Toxicogenomics Database (CTD)44. Notably, different drug classes used in the standard treatment of diabetes45 interacted with the proteins represented in our extended DKD signature (Fig. Next, we analysed a publicly available single-cell RNA-sequencing dataset46 generated from recently diagnosed young individuals with T2D without overt DKD and included a subset of patients receiving an SGLT2 inhibitor. This analysis confirmed our extended DKD signature at the transcriptional level and revealed a partial transcriptional modulation in proximal tubules with SGLT2 inhibitor treatment (Extended Data Fig. This result suggests that individuals with diabetes may benefit from additional interventions to reverse them to the healthy reference state. For this reason, we quantified the number of known drug–protein interactions for members of our extended DKD signature. This analysis led to the identification of potential targets to revert cell communities to the healthy reference state, including phosphodiesterase-5 inhibitors as potential regulators of TRPC6 signalling. As an additional external validation step, we used a public single-nucleus RNA-sequencing dataset from a rat model of DKD47 to assess the link between cGMP signalling and TRPC6-mediated mitochondrial stress in proximal tubules (Fig. Although transcriptomic detection of TRPC6 was insufficient to confirm a direct effect on this target, cGMP modulation was associated with the attenuation of several components of our extended DKD signature. Together, our findings confirm that the applicability of PathoPlex-derived clusters extends beyond the definition of integrative pathological features. Indeed, they can link the spatial context to single-cell transcriptomics and even pharmacological modelling. Up to this point, our experiments included well-defined disease and control groups with recognizable pathological features identifiable through traditional histopathological methods. In our final experiment, PathoPlex was applied to 18 human kidney research biopsy samples without overt histopathological changes to test the limits and added value of PathoPlex. We aimed to identify early stages of kidney stress in T2D and to further profile the impact of SGLT2 inhibitors on these integrative features of cellular stress. A total of 142 markers (122 biological and 20 quality control; Supplementary Table 6) were imaged at 80 nm per pixel, which systematically covered glomerular and non-glomerular regions across 284 ROIs. This strategy generated >600 billion pixels, which contributed to 140 clusters (Fig. A total of 24 clusters showed significant regulation between groups (Fig. 4b), which revealed specific biological processes with distinct subcellular locations (Fig. Significant differences encompassed increases in clusters that represented stromal cell filopodia, the mesangial matrix and vascular smooth muscle cells. Moreover, reductions in clusters associated with structural and functional features of proximal tubules (that is, cell adhesion, brush border integrity, JAK2–H2B-mediated cell cycle, mitochondrial integrity and lactate transport), peritubular capillary integrity, mitochondrial and ER integrity in the distal tubule and nitric oxide production in the collecting duct were observed. a, Schematic overview showing the experimental design (n = 18 cases; ROIs = 284). Details of the antibody panel are provided in Supplementary Table 6. For this experiment, we used human research specimens from healthy individuals and individuals with T2D treated or not with SGLT2 inhibitors (SGLT2i). b, Differential cluster abundance for each comparison. The key for clusters also applies to c and d. c, Examples of integrative subcellular clusters that were differentially regulated. Images were selected from all available ROIs (n = 284). Differential cluster abundance analysis used a two-sided t-test with Benjamini–Hochberg correction. BB, brush border; CD, collecting duct; DT, distal tubule; EC, endothelial cell; FIB, fibroblasts; HSP, heat shock protein; IC, intercalated cell; MAM, mitochondria-associated endoplasmic reticulum membrane; NO, nitric oxide; ROS, reactive oxygen species; TAL, thick ascending limb; VSMC, vascular smooth muscle cell. Diagrams in a were created using BioRender (https://biorender.com). SGLT2 inhibitors were also characterized by the decreased abundance in a cluster representing lysosomal and proteasomal stress in endothelial cells. However, SGLT2i treatment did not fully reverse the T2D-specific changes in cluster abundance. First, the fold changes of significantly differentially abundant clusters in T2D+SGLT2i– samples relative to control samples were defined to represent the baseline effect of T2D. Then, the same comparison was performed between T2D+SGLT2i+ and control samples to represent the effect of SGLT2 inhibitors on T2D. This comparison indicated that SGLT2 inhibitors promoted a reconstitution of peritubular capillary and mitochondrial integrity in proximal tubules, together with a partial reversal of the increase in clusters representing vascular smooth muscle and stromal filopodia (Fig. These data demonstrate the potential of PathoPlex to uncover features of injury before disease onset that are inaccessible to classical histopathology. 3 and 4), we propose a continuum of early glomerular and tubulointerstitial alterations that precede quantifiable reductions in end-organ function and that eventually converge in DKD. These alterations include an impaired glomerular filtration barrier, podocyte loss, ECM remodelling and tubular injury following prolonged hyperglycaemia (Fig. Although some early changes seemed to be attenuated by SGLT2 inhibitors, further studies are required to fully profile the potential long-term preventive effects of this intervention throughout the entire clinical course of DKD. In summary, our results demonstrate the utility of PathoPlex to extract meaningful integrative features from even non-pathological tissues. Notably, the framework also provided further pathophysiological evidence to support the use of SGLT2 inhibtors31 as an early intervention in T2D. Our results highlight the potential need for further treatments to optimally preserve kidney health in individuals with T2D at high risk of DKD. Multiplexed imaging is a rapidly growing field10,48 and its contribution to a deeper understanding of tissue biology is illustrated by the recent generation of organ-level atlases, for example, in placenta8 and intestine9 using MIBI-TOF and CODEX, respectively. Moreover, previous efforts to characterize archival pathological tissues have provided new insights into the tumour microenvironment using IMC in breast cancer49,50 and melanoma51 as well as in post-mortem COVID-19 specimens using IBEX52. Despite these recent successes, widespread application of these technologies has been hampered by several factors. These include access to commercial equipment, antibody panel size (average of 37 markers) and composition (that is, mostly focused on cell identity), limited spatial resolution (average of 267 nm per pixel), high-throughput in single specimens and insufficiently defined quality control steps. Here we provided a detailed protocol for highly multiplexed imaging at subcellular resolution for archival FFPE tissues. The use of PathoPlex for multiplexed imaging includes the following advantages: (1) no dependency on commercial equipment; (2) open-access 3D printing-based solutions for sample preparation and automation; (3) compatibility with any inverted fluorescence microscope, ranging from widefield to high-end confocal microscopy; (4) scalability in antibody panel size (>120), image resolution (up to 80 nm per pixel using confocal microscopy) and sample sizes (that is, intact clinical tissues); (5) use of unmodified antibodies broadly accessible to the scientific community; (6) introduction of stringent quality-control steps to define best practices; and (7) minimization of batch effects through the parallel processing of up to 40 clinical biopsy samples (approximately 4,000 mm2 of available tissue). Together, PathoPlex paves the way for universal access to multiplexed imaging in clinical specimens. It also unlocks one of the largest and most comprehensive biobanks inadvertently created to a wide array of users: FFPE archives in clinical pathology centres and research institutes. It is now up to users to build on this resource to explore well-characterized patient cohorts, generate antibody panels that best address their scientific or clinical questions and to leverage the most efficient and suitable microscopy systems. As emerging multiplexed imaging technologies generate larger and more complex datasets, image analysis tools need to adapt. Although cell segmentation and identification of cell states remain the most accepted methodologies for both technical development53 and biological interpretability54, unsupervised methods are starting to gain attention. Recent examples of pixel-based image analysis tools for multiplexed imaging data include cellular changes during normal retinal development in human organoids16 and the generation of quantitative annotations both independently and in conjunction with cell segmentations in various human tissues55. However, PathoPlex provides integrative features that recapitulate health, stress and overt disease, which can be pharmacologically modulated. As part of PathoPlex, we provide spatiomic, an efficient, scalable and streamlined end-to-end workflow for the community to analyse multiplexed imaging datasets of over half a trillion pixels. Overall, PathoPlex introduces a shift away from characterizing tissues solely at the cellular level (cell typing and their spatial organization) and towards a data-driven approach that captures the most distinctive biological signatures across spatial scales based on spatial co-expression patterns derived from individual pixels. Recent advances in community-based strategies include minimum information guidelines56 and a public repository for antibodies compatible with multiplexed imaging57. These initiatives highlight the importance of continuous development in this new and rapidly growing field. Although PathoPlex shows promise, several areas require further optimization. PathoPlex enables users to perform more imaging cycles, which provides an opportunity to expand antibody panel sizes and in turn can extend their scope. However, time efficiency remains crucial for implementation in research and even more so as a clinical application. For this reason, we consider that robotic automation or sample size enrichment through parallel processing of multiple tissue microarrays may be considered in the future. Furthermore, as antibody panels can rapidly expand, standardizing quality-control metrics (that is, validation, secondary-only cycles and re-staining) will benefit potential PathoPlex users and the growing multiplexed imaging community. Moreover, PathoPlex enables the generation of datasets with sizes beyond current standards (>600 billion pixels), which present an analytical challenge that is currently best addressed by GPU acceleration. In this context, we recognize the need to implement additional features to minimize user reliance on specialized hardware. Finally, our work raises important computational questions regarding the need to establish integrative ontology terms that combine multiple biological layers to facilitate broad interpretability. Moreover, the increasing technical requirements to transfer, share, store and process data at a scale will soon challenge available resources. FFPE tissues were collected and prepared according to institutional protocols. 2, validation kidney biopsy specimens from individuals with ANCA-associated CGN were obtained from the Hamburg Glomerulonephritis Registry (https://www.sfb1192.de/en/register). 3, control kidney specimens were obtained from nephrectomies performed on individuals with renal cell carcinoma in collaboration with the Division of Nephrology and Clinical Immunology, RWTH Aachen University Medical Center. Kidney biopsy samples from individuals with DKD were obtained from the Department of Nephrology and the Department of Pathology Georges Pompidou European Hospital, Assistance Publique–Hôpitaux de Paris. 4, nested protocol research kidney biopsy samples were obtained from volunteers (adolescents and young adults; n = 13) with T2D (12–21 years of age, T2D onset at <18 years of age, T2D duration 1–10 years and HbA1c < 11%) from the Renal HEIR and the IMPROVE-T2D studies. T2D was defined according to criteria of the American Diabetes Association plus the absence of glutamic acid decarboxylase, islet cell, zinc transporter 8 and/or insulin autoantibodies. The Renal HEIR and IMPROVE-T2D cohorts have intentionally harmonized study protocols. Medication use was recorded for all participants, and T2D treatment, including SGLT2 inhibitors, was determined by their medical provider. Normative kidney reference tissue from research biopsy samples were provided by five healthy young adult participants in the CROCODILE study (NCT04074668). 3 and 4, kidney biopsy samples were obtained from the Hamburg Glomerulonephritis Registry (https://www.sfb1192.de/en/register), liver specimens were provided by the Institute of Pathology, University Medical Center Hamburg-Eppendorf and brain specimens were provided by the Institute of Neuropathology, Freiburg University Hospital. All tissue collections were performed in accordance with the ethical principles stated by the Declaration of Helsinki. All experimental animals were housed at an ambient temperature of 20 ± 2 °C, humidity of 55 ± 10% and a light–dark cycle of 12–12 h. In brief, mouse crescentic nephritis was induced according to an established protocol58. Rat tissues were obtained from two experimental set-ups32,59. Administration of a JNK inhibitor (CC930, dose of 60 mg kg–1 in 0.5% carboxymethyl cellulose) or vehicle alone was performed twice daily by oral gavage. The prevention study (therapy started at day 0 and animals were killed on day 1) was performed in outbred male Sprague–Dawley rats, as this strain is known to develop heavy proteinuria59. The therapeutic study (therapy started at day 7 after disease induction and continued until animals were killed on day 28) was performed in inbred male Wistar Kyoto rats, which are prone to developing crescent formation. In both studies, proteinuria measurements and histopathology were performed according to standardized protocols32,59. This model shows morphological and physiological traits of DKD (that is, hyperglycaemia, albuminuria and glomerular hypertrophy). Depending on the number of samples, a suitable-sized glass surface was selected and coated with poly-d-lysine (1 mg ml–1; Merck, A-003-E) for 30 min or with APTES (Merck, 440140) 10% v/v in acetone (Merck, 320110) for 2 min and then dried overnight before mounting the sections. We initially used poly-d-lysine for all our experiments but realized that significant lifting was progressively observed in all organs tested, including kidney, lung, colon, liver and brain. Lifting was initially mild in kidney samples but highly prominent in lung and colon specimens. For example, we observed partial but meaningful tissue lifting in 76 out of 498 ROIs (15%) by the end of 49 imaging cycles (Fig. Furthermore, from 23 lung specimens analysed over 8 imaging cycles, lifting was already observed in 7 of them (30%). These observations across multiple tissue types led us to conclude that poly-d-lysine coating exhibits organ-dependent and time-dependent reliability limitations, for which we recommend potential users to perform pilot studies in their organ of interest. However, after a comprehensive literature review, we identified APTES as an ideal coating agent. Using APTES, tissue lifting occurred in only 1% of kidney specimens (Fig. After coating, FFPE tissues were cut at a thickness of 2–3 μm and carefully mounted on the coated glass surface (for example, µ-Slide 2-well glass-bottom (Ibidi, 80287), µ-Slide 8-well glass-bottom (Ibidi, 80827), Cell Imaging Plate 24-well glass-bottom (Eppendorf, 0030741021) or Nexterion glass (Schott, 1868767)). To prevent dissolution of the plastic components in the chambered coverslips and plates by the solvent used for deparaffinization, the walls of each well were protected by a seal of transparent silicone (Pattex) or a ring of solvent-resistant plastic, respectively. Samples were treated with the following set of solutions: Histo-Clear (National Diagnostic, HS-200) three cycles of 10 min each, followed by an ethanol series consisting of three cycles of 100% ethanol (10 min), two cycles of 70% ethanol (5 min), one cycle of 50% ethanol (5 min) and finally, triple immersion in double-deionized water (ddH2O) for 5 min each. Samples were immersed in target retrieval solution pH 9 (Agilent, S236784-2) and heated for 15 min using a steamer (Braun; FS 3000). Sections were then incubated for 15 min in EnVision FLEX wash buffer (Agilent, K800721-2). To limit nonspecific antibody binding, samples were incubated in a blocking solution consisting of 5% BSA (Merck, A7906) in Dulbecco's PBS (Thermo Fisher Scientific, 14190094) for 1 h at room temperature. Afterwards, samples were washed three times for 5 min with wash buffer. An elution buffer was prepared according to a previously described formulation15, which consisted of 0.5 M glycine (Carl Roth, 3908.2), 3 M urea (Merck, U5378), 3 M guanidine hydrochloride (Merck, G4505) and 70 mM TCEP (Merck, C4706) mixed in ddH2O and adjusted to pH 2.5. Samples were incubated in elution buffer once for 5 min and then three times for 10 min on a shaker, followed by three washes of 5 min with wash buffer. Whenever used as a reference for alignment, NHS-E (Thermo Fisher Scientific, A10168) diluted in PBS (1:400) was added to the samples for 1 h at room temperature. After 1 h, samples were washed three times for 5 min with wash buffer. The following steps were completed for every subsequent cycle of staining and carried out in a light-free environment to prevent the crosslinking of antibodies. Samples were incubated with primary antibodies in EnVision FLEX antibody diluent (Agilent, K800621-2) for either 1 h at room temperature (Fig. 3 and 4), followed by three washes of 5 min with wash buffer. We provide confirmation of each staining pattern for every antibody in Supplementary Data 1 and 2. We also validated the practical feasibility of 1-h incubations at room temperature under non-multiplexed and multiplexed imaging conditions (Supplementary Fig. Appropriately matched secondary antibodies (and directly conjugated primary antibodies) and the nuclear markers DAPI (Merck, D9542, 1:200) or DRAQ5 (Abcam, ab108410, 1:200) were mixed in antibody diluent and incubated for 1 h at room temperature. Afterwards, samples were washed three times for 5 min with wash buffer. An imaging buffer was prepared according to a previously described formulation15, which consisted of 700 mM N-acetyl-l-cysteine (Merck; A9165) mixed in ddH2O and adjusted to pH 7.4. Imaging buffer was added to samples for imaging and then washed three times for 5 min with wash buffer before elution. Samples were incubated with elution buffer once for 5 min and then three times for 10 min on a shaker, followed by three washes of 5 min with wash buffer. Thereafter, these steps were repeated until the desired number of antibodies was reached. Together, each cycle (using 1 h of incubation time for primary antibodies) can be completed in under 4 h of bench work. All cycles per experiment (antibodies and order) are described in Supplementary Table 7. Periodic acid Schiff (PAS) staining was performed after the last immunofluorescence staining only in Fig. 3 following a standard protocol, including incubation with periodic acid (Th. The sections were then counterstained with Mayer's haematoxylin (Agilent Technologies, S330930-2). ABCG2 (Santa Cruz, sc-377176; 1:200); ACE-2 (R&D Systems, AF933; 1:200); adiponectin (Thermo Fisher Scientific, MA1-054; 1:200); AIF (Cell Signaling Technology, 5318; 1:200); AKAP12 (Proteintech, 25199-1-AP; 1:600); AKR1B1 (Thermo Fisher Scientific, PA5-82915; 1:500); AKR1C1 (Thermo Fisher Scientific, MA5-32842; 1:200); alpha B-crystallin (Proteintech, 68001-1-Ig; 1:1,000); ANXA3 (Sigma-Aldrich, HPA013398; 1:200); αSMA–FITC conjugate (Sigma-Aldrich, F3777; 1:800); aquaporin 2 (Alomone Labs, AQP-002; 1:400); β-actin (Sigma-Aldrich, A5441; 1:1,500); β-catenin (Abcam, ab6302; 1:2,000); β-tubulin (Cell Signaling Technology, 2128; 1:150); calbindin-D (Sigma-Aldrich, C9848; 1:3,000); calpain small subunit 1 (Abcam, ab92333; 1:200); calpastatin (Abcam, ab244460; 1:200); calreticulin (Abcam, ab92516; 1:300); carbonic anhydrase IX (R&D Systems, AF2188; 1:50); catalase (Proteintech, 66765-1-Ig; 1:300); CD3 (Abcam, ab11089; 1:200); CD4 (R&D Systems, AF-379-NA; 1:100); CD8 (Agilent, M710301-2; 1:200); CD34 (Agilent, GA63261-2; 1:50); CD41 (Thermo Fisher Scientific, PA5-79526; 1:500); CD42b (Abcam, ab227669; 1:100); CD44 (Cell Signaling Technology, 5640S; 1:200); CD44–Alexa Fluor 647 conjugate (BioLegend, 103018; 1:200); CD68 (BioLegend, 916104; 1:200); CD79α (Agilent, M705001-2; 1:200); CD200 (R&D Systems, AF2724; 1:100); CD206 (Proteintech, 60143-1-Ig; 1:2,000); FOS (Abcam, ab190289; 1:600); claudin 1 (Abcam, ab15098; 1:500); claudin 10 (Thermo Fisher Scientific, 38-8400; 1:100); collagen I (Southern Biotech, 1310-01; 1:200); collagen III (Abcam, ab7778; 1:200); collagen IV (Abcam, ab6586; 1:200); collagen V (Abcam, ab7046; 1:100); cubilin (R&D Systems, AF3700; 1:200), cyclin B1 (Cell Signaling Technology, 12231; 1:100); cytochrome c (Abcam, ab110325; 1:200); cytokeratin 7 (Agilent, GA61961-2; 1:300); cytokeratin 8 (R&D Systems, MAB3165-SP; 1:300); cytokeratin 19 (Abcam, ab52625; 1:300); C1QA (Proteintech, 67063-1-Ig; 1:1,000); DACH1 (Sigma-Aldrich, HPA012672; 1:200); decorin (R&D Systems, AF143, 1:50); E-cadherin (R&D Systems, AF648; 1:200); EEA1 (Santa Cruz, sc-137130; 1:100); EHD3 (LSBio, LS-C133741; 1:150); endomucin (Sigma-Aldrich, HPA005928; 1:100); eNOS (Abcam, ab76198; 1:200); ezrin (Cell Signaling Technology, 3145S; 1:300); FAM189A2 (Thermo Fisher Scientific, PA5-63414; 1:200); fibronectin (Abcam, ab2413; 1:200); FKBP51 (R&D Systems, AF4094-SP; 1:50); FXYD4 (Thermo Fisher Scientific, PA5-63570; 1:200); GFAP (Thermo Fisher Scientific, 14-9892-82; 1:200); glucocorticoid receptor (Cell Signaling Technology, 3660; 1:2,000); glutathione peroxidase 1 (R&D Systems, AF3798; 1:100); glutathione peroxidase 3 (R&D Systems, AF4199; 1:50); glycophorin A (R&D Systems, MAB1228-SP; 1:500); GRP78 (Proteintech, 11587-1-AP; 1:200); HB-EGF (R&D Systems, AF-259; 1:100); histone H3 (Cell Signaling Technology, 4499; 1:400); HMOX1 (Thermo Fisher Scientific, MA1-112; 1:200); HSD11B2 (R&D Systems, MAB8630-SP; 1:100); KIM-1 (R&D Systems, AF1750; 1:200); IBA1 (Thermo Fisher Scientific, MA5-27726; 1:500); IDH1 R132H (Dianova, DIA-H09, 1:200); IL-1RA (Abcam, ab124962; 1:200; specificity issues were raised by the provider after our experiments were completed. We kept it in the panel as none of our findings were affected and we did not perform any biological inferences on the basis of this antibody); iNOS (Thermo Fisher Scientific, MA5-41652; 1:200); integrin-α1 (R&D Systems, AF5676; 1:300); integrin-α3 (Proteintech, 66070-1-Ig; 1:2,000); integrin-β1 (Abcam, ab179471; 1:800); Ki-67 (Agilent, M724029-2; 1:200); laminin (Abcam; ab11575, 1:200); LAMP1 (Cell Signaling Technology, 9091; 1:300); LC3B (Cell Signaling Technology, 3868; 1:300); LEL-DyLight 649 conjugate (Vector Laboratories, DL-1178; 1:300); LTL biotinylated (Vector Laboratories, B-1325-2; 1:500); MCT1 (Thermo Fisher Scientific, MA5-18288; 1:300); MerTK (R&D Systems, AF591; 1:200); MPO (R&D Systems, MAB3174; 1:200); nephrin (Progen, GP-N2; 1:150); neurofilament (Agilent, IR607; 1:200); NOX4 (R&D Systems, MAB8158;, 1:300); NQO1 (Proteintech, 67240-1-Ig; 1:2500); OLIG2 (Bio SB, BSB 2561; 1:200); p62 (Cell Signaling Technology, 39749; 1:400); PCK1 (Proteintech, 66862-1-Ig; 1:400); PCNA (Abcam, ab29; 1:2,000); PDGFRβ (Cell Signaling Technology, 3169; 1:100); PDI (Cell Signaling Technology, 45596S; 1:400); periostin (R&D Systems, AF3548; 1:150); phospho-AMPKα (Cell Signaling Technology, 2535; 1:200); pJUN (Abcam, ab32385; 1:200); phospho-ERK1/2 (Cell Signaling Technology, 4370; 1:250); phospho-ezrin–radixin–moesin (Cell Signaling Technology, 3726; 1:200); phospho-GSK3β (Cell Signaling Technology, 9323; 1:100); phospho-histone H3 (Cell Signaling Technology, 9701; 1:200); phospho-JAK2 (Thermo Fisher Scientific, MA5-42424; 1:100); phospho-ribosomal protein S6 (Cell Signaling Technology, 4858S; 1:300); phospho-SMAD2 (Thermo Fisher Scientific, 44-244G; 1:200); phospho-SMAD3 (Thermo Fisher Scientific, PA5-104940; 1:200); phospho-STAT1 (Cell Signaling Technology, 9167S; 1:400); phospho-STAT3 (Abcam, ab76315; 1:200); PITX2 (R&D Systems, AF7388; 1:100); podocin (Sigma-Aldrich, P0372; 1:3,000); proteasome 20S LMP7 (Abcam, ab3329; 1:400); RAB5A (Cell Signaling Technology, 46449; 1:300); RAB7 (Abcam, ab137029; 1:200); RAP1GAP (Abcam, ab244259; 1:300); RCAS1 (Cell Signaling Technology, 12290; 1:200); sclerostin (Thermo Fisher Scientific, PA5-37943; 1:100); SIRT1 (Cell Signaling Technology, 8469; 1:200); SLC12A3 (Thermo Fisher Scientific, MA5-41643; 1:200); SOD1 (Proteintech, 67480-1-Ig; 1:400); SOD2 (Thermo Fisher Scientific, PA5-30604; 1:300); SRB1 (Abcam, ab217318; 1:300); STAT2 (R&D Systems, MAB16661; 1:200); survivin (Cell Signaling Technology, 2808; 1:300); talin 1 (Abcam, ab71333; 1:200); TRPC6 (Abcam, ab233413; 1:200); ubiquityl-histone H2B (Cell Signaling Technology, 5546T; 1:200); uromodulin (R&D Systems, AF5144; 1:300); villin 1 (Abcam, ab52102; 1:200); vimentin (Progen, GP53; 1:200); von Willebrand factor (Agilent, A008229-2; 1:200); WT1 (Agilent, IS05530-2; 1:200); and ZO-1 (Thermo Fisher Scientific, 61-7300; 1:250). We kept it in the panel as none of our findings were affected and we did not perform any biological inferences on the basis of this antibody); Ki-67 (Abcam, ab15580; 1:200); lamin B1 (Santa Cruz, sc-374015; 1:200); laminin (Abcam, ab11575; 1:200); LTL biotinylated (Vector Laboratories, B-1325-2; 1:500); nephrin (Progen, GP-N2; 1:150); PCNA (Abcam, ab29; 1:2,000); PDI (Cell Signaling Technology, 45596S; 1:400); phospho-ezrin–radixin–moesin (Cell Signaling Technology, 3726; 1:200); podocin (Sigma-Aldrich, P0372; 1:3,000); podoplanin (R&D Systems, AF3244-SP; 1:200); synaptopodin (Synaptic Systems, 163 004; 1:200); tyrosine hydroxylase (Cell Signaling Technology, 45648; 1:200); ubiquityl-histone H2B (Cell Signaling Technology; 5546T; 1:200); β-actin (Sigma-Aldrich; A5441, 1:1500); vimentin (Progen, GP53; 1:200); and von Willebrand factor (Agilent, A008229-2; 1:200). Secondary antibodies were diluted in a ratio ranging from 1:200 to 1:300. The following antibodies were used: goat anti-guinea pig IgG Alexa Fluor 488 (Thermo Fisher Scientific, A-11073); goat anti-guinea pig IgG Alexa Fluor 555 (Thermo Fisher Scientific, A-21435); donkey anti-mouse IgG Alexa Fluor 488 (Thermo Fisher Scientific, A-21202); donkey anti-mouse IgG Alexa Fluor 555 (Thermo Fisher Scientific, A-31570); donkey anti-mouse IgG Alexa Fluor 647 (Thermo Fisher Scientific, A-31571); donkey anti-rabbit IgG Alexa Fluor 488 (Thermo Fisher Scientific, A-21206); donkey anti-rabbit IgG Alexa Fluor 555 (Thermo Fisher Scientific; A-31572); donkey anti-rabbit IgG Alexa Fluor 647 (Thermo Fisher Scientific, A-31573); donkey anti-goat IgG Alexa Fluor 488 (Thermo Fisher Scientific, A-11055); donkey anti-goat IgG Alexa Fluor 555 (Thermo Fisher Scientific, A-21432); donkey anti-rat IgG Alexa Fluor 488 (Thermo Fisher Scientific, A-21208); donkey anti-rat IgG Alexa Fluor 555 (Thermo Fisher Scientific, A78945); donkey anti-sheep IgG Alexa Fluor 488 (Thermo Fisher Scientific, A-11015); donkey anti-sheep IgG Alexa Fluor 555 (Thermo Fisher Scientific, A-21436); streptavidin Alexa Fluor 488 (Thermo Fisher Scientific, S11223); and streptavidin Alexa Fluor 555 (Thermo Fisher Scientific, S21381). FFPE tissues were cut at a thickness of 2–3 μm, carefully affixed onto Superfrost Plus adhesion slides (Epredia, J1800AMNZ) and dried overnight at 37 °C. Subsequently, samples underwent sequential treatment involving triple immersion in xylene (10 min each) followed by an ethanol series (5 min each) consisting of three rounds of 100% ethanol, two rounds of 70% ethanol, one round of 50% ethanol and finally triple immersion in ddH2O (5 min each). The immunostaining procedure mirrored the one used for PathoPlex samples but substituted 5% BSA with SuperBlock blocking buffer (Thermo Fisher Scientific, 37535) during the blocking step. Finally, after immunostaining, samples were mounted using ProLong Gold (Thermo Fisher Scientific, P36930). 2, images were acquired using a LSM 800 confocal microscope plus AiryScan (Zeiss, ZEN2.6) with the optimized ×63 objective (NA: 1.4). 3, a Thunder Imager Live Cell and 3D assay (Leica Microsystems) fitted with a ×40 (NA: 1.10) or ×63 (NA: 1.40) objective was used to acquire images, which were processed using a computational clearing algorithm (Leica Microsystems)60. The positional data of the imaged region for each sample were stored in Leica Application Suite X software (v.3.7.6, Leica Microsystems), which ensured consistent capture of the identical location for each cycle. 4, a CellDiscoverer 7 with LSM 900 (Zeiss, ZEN 3.5 System) and AiryScan Multiplex fitted with a ×50 (NA: 1.20) objective and ×0.5 zoom was used to acquire images. Supplementary Table 8 summarizes the approximate microscopy times per experiment, considering image acquisition as the most important contributing factor. However, there are additional practical contributors, including chamber repositioning, movement delay between the ROI and data storage, which should be accounted for during implementation. Tinkercad (Autodesk; https://www.tinkercad.com) was used to create designs for the 3D-printed parts. The BLTouch Cover Size Fixed was designed by louise_tguk on Thingiverse (https://www.thingiverse.com/thing:5013058). The inner frame was printed using NinjaFlex TPU filament 1.75 mm (NinjaTek). Masking tape was used to create an adhesive surface on the bed. These encompassed manual deparaffinization, antigen retrieval and mounting the Nexterion glass with sample sections onto the chamber frame. The deparaffinization process described above required the use of a dewaxing container, a container holder, a dewaxing carrier and carrier handle printed with PETG. After completion of dewaxing, the Nexterion glass with sections underwent antigen retrieval and washing procedures as outlined above. After washing, any excess wash buffer present at the edges of the glass was carefully removed. To prevent the samples from drying out during this process, regular application of wash buffer to the samples was necessary while ensuring that the silicone did not become excessively wet. The settings of the solution containers corresponding to each g-code are shown in Supplementary Table 9. The solution container stand consists of numbered sections ranging from 1 to 12, which are designated for container installation. Specifically, solution container stand 1 needed to be positioned at the front side of the base plate. The printer then paused for 60 s before resuming operation. The washing, staining and elution processes were then automatically performed by pushing down on the solution containers and table with a rod in the headpiece. The dimensions of the chamber frame match those of ready-made plates used for imaging cultured cells. For an example of how this solution works, see Supplementary Video 1. The maintenance culture was passaged three times a week by gentle trypsinization using trypsin-EDTA 0.05% (Thermo Fisher Scientific, 25300054). Migration assays were performed using Culture-Insert 2 wells in μ-Dish 35 mm (Ibidi, 81176). Each well was seeded with 30,000 PECs in 100 μl ECBM without supplements and with 1% FBS and incubated overnight. The insert was then removed, which created a gap of 500 μm between cells. PECs were stimulated with either PDGF-BB (Thermo Fisher Scientific, 315-18-50UG, per manufacturer's recommendations of 2.0 ng ml–1) or with PDGF-BB and tanzisertib HCl (CC-930; Selleck Chemicals, S8490) or PDGF-BB and vehicle, in this case DMSO (Merck; D2650). All combinations were diluted using ECBM without supplements and with 1% FBS. Images were taken every 10 min for 23 h using the Personal Automated Lab Assistant (Leica Microsystems). Areas of migration were measured using Fiji. Scratch assays were performed using glass-bottom FluoroDishes seeded with 50,000 cells in 2 ml ECBM with 1% FBS at 5% CO2 and 37 °C. After 48 h, cells reached 90% confluence and were ready for the experiment, in which a sterile plastic 1,000 μl micropipette tip was used to scratch the cell monolayer to create a wound of around 1,000 μm. Next, the cell monolayer was gently washed with ECBM with 1% FBS to remove dead cell debris. To use the nucleus for tracking, PECs were stained with 80 nM Hoechst 33342 (Thermo Fisher Scientific) for 20 min at 5% CO2 and 37 °C and washed once with Dulbecco's PBS (Thermo Fisher Scientific, 14190094). Afterwards, 2 ml fresh ECBM with 1% FBS was added for image acquisition. Time-lapse imaging was performed using a Leica DMi8 M/C/A inverted microscope equipped with ×10 Plan Apo objective (Leica Microsystems). Images at both sides of the wound were acquired every 5 min with an ORCA-Flash4.0 digital camera (Hamamatsu Photonics) using MetaMorph (v.7.10.3.279) software (Molecular Devices). To visualize the wound, adjacent positions were stitched using the Stitching plugin from Fiji ImageJ. Tracking of the first 8 h of migration was performed with the TrackMate plugin from Fiji (v7.10.2) and custom-made scripts62. Mean square displacement was calculated using the CelltrackR package63. For electron microscopy with immunogold labelling, kidneys were removed, cut into 2-mm-thick razor blade sections and immersion-fixed in freshly prepared 4% paraformaldehyde for 24 h at 4 °C. The samples were then resliced into 50-µm-thick sections using a vibratome. Vibratome sections were incubated with the primary antibody against TRPC6 (rabbit, final concentration 1:200). After washing and overnight incubation at 4 °C with the secondary antibody, goat anti-rabbit 1:100 (Nanoprobes), sections were silver enhanced with HQ silver (Nanoprobes) for 8 min in the dark at 4 °C, washed in 0.1 M phosphate buffer, treated with OsO4 (0.5% for 45 min at room temperature) and stained with uranyl acetate (1% w/v in 70% v/v ethanol, 30 min at room temperature). After dehydration, sections were embedded in epoxy resin, Durcupan ACM (Sigma-Aldrich). Next, 50-nm ultrathin sections were cut using an UC6 ultramicrotome (Leica Microsystems) and analysed using an 80 kV Zeiss Leo 910 transmission electron microscope. Tissue sections were dewaxed in xylene and rehydrated, followed by staining using a standard protocol for immunohistochemistry according to the protocol by Fluidigm. Nuclei were labelled with iridium, and TRPC6 antibody (Abcam, ab105845) was coupled to 174Yb heavy metal. Data acquisition was performed on a Helios time-of-flight mass cytometer (CyTOF) coupled to a hyperion imaging system (Fluidigm). Areas for ablation were selected on the basis of haematoxylin and eosin staining performed on an adjacent slide. All data were collected using the commercial Fluidigm CyTOF software (v.01). Images were evaluated by two expert pathologists in a blinded fashion to define disease states as either control, acute or crescentic phase. Next, multiple metrics were also calculated in a blinded fashion, including tubular injury score (0–3+), cell numbers per cross section, percentage of cellular crescents and percentage of tubular injury. Glomeruli were isolated at day 4 after NTS treatment and in control groups after kidney perfusion with Dynabeads (Invitrogen), preserved in RNAlater and stored at −80 °C until processing. For preparation of nuclei, nuclei were extracted from the isolated glomeruli according to a modified protocol64. Nuclei were mixed with RLT buffer (Qiagen) and frozen at −80 °C. Total nuclei RNA was extracted using RNeasy Micro kits (Qiagen) according to the manufacturer's recommendations. Bulk RNA-sequencing data were processed using our previously published open-source Snakemake65 workflow for RNA-sequencing analysis with pytximport66. In brief, raw FASTQ files provided by the sequencing facility were assessed for quality with FastQC67, followed by trimming of adapter sequences and removal of low-quality reads with fastp68. Next, processed sequences were selectively aligned to a reference gentrome based on Ensembl GRCh38 (release 112)69 and transcript counts were quantified with Salmon70. Differentially expressed genes were identified using PyDESeq2 (ref. Genes were considered differentially expressed if their log2-transformed fold change value was greater than 0.5 or lower than −0.5 and their false-discovery-rate-adjusted P value was less than 0.01. The results from PyDESeq2 were used to infer transcription factor activity based on the CollecTRI72 gene regulatory network reference with decoupler-py73 univariate linear modelling. Commonly used functions for the analysis of PathoPlex imaging data were combined into the spatiomic Python package. spatiomic comprises different submodules that facilitate data loading, image registration, image preprocessing, dimensionality reduction, spatial analyses, neighbourhood graph construction and clustering. In this section, we describe the architecture of this software package and the general functionality it includes. Parameter choices and detailed workflows are described in subsequent sections. Both the computational analysis library and the code for all analyses are available online through GitHub and Zenodo, as detailed in the Code and Data availability sections. spatiomic comes with support for multiple common microscopy imaging formats and flexibly supports AnnData74 objects, NumPy75 arrays and pandas DataFrames76. It uses cuml, cucim and cugraph from the RAPIDS ecosystem21 as well as cupy77 for GPU-accelerated computations, which enables analyses to scale to billions of data points with affordable hardware with a time requirement of just minutes to hours depending on dataset size. The library was extensively unit-tested, typed and documented. Documentation, including a full example notebook detailing how to apply spatiomic analyses to PathoPlex data, is available at: https://spatiomic.org/. To enable modular composition of analyses, spatiomic encompasses several submodules as described below. The data.read class offers a method for parsing common microscopy imaging formats such as .tiff, .lif and .czi files through readlif, tifffile and aicspylibczi bindings. A random subsample of imaging data can be obtained through the data.subsample class. The process submodule offers common functions for preprocessing imaging data. This includes the clip class for channel-wise histogram clipping to either absolute values or percentiles, the zscore class for channel-wise z scoring and the normalize class for the channel-wise scaling of intensity values to a given range. It also includes the register class, which exposes different methods for image registration and registration evaluation. Dimensionality reduction is an important step in many single-cell and spatial omics analyses. The latter is achieved by incorporating the dimension.som class, which enables GPU-accelerated training of SOMs thanks to an XPySOM22 integration. To train SOMs with the Pearson correlation as distance metric, we extended XPySOM with a CuPy-based function, which is available from GitHub (https://github.com/complextissue/xpysom). The neighbour submodule exposes classes that enable the creation of k-nearest and shared nearest neighbour graphs, which facilitates the construction of similarity-based graphs for graph clustering and distance-based neighbourhood graphs for spatial analysis. Clustering algorithms enable the unsupervised identification of similar (protein co-expression) patterns, which facilitates automatic partitioning of complex signals into biologically meaningful groups. The spatial submodule incorporates functions for the explorative analysis of spatial distribution patterns in both immunofluorescence and clustered images, including global and local univariate and bivariate measures of spatial distribution based on PySAL80. It further includes code for efficient join count quantification and spatial vicinity graph construction and interoperability tools for use together with PySAL80, thereby ensuring compatibility with a wide range of spatial statistics applications81. The tool submodule contains utility functions for additional evaluation or analysis. spatiomic includes plotting functions based on matplotlib82 and seaborn83 that facilitate visualizing common plots, for example, image registration metrics, SOM training-quality metrics, cluster projections, spatial adjacency graphs as well as cluster contributor histograms and volcano plots. spatiomic is designed to be flexible and adaptable to the scale of multiplexed imaging data and available computing resources. The only mandatory system requirement is Python (v.3.10) or higher. Although many individual functions run in seconds, and a complete exemplary workflow (excluding data download) can be completed in less than 3 min on a standard personal laptop, spatiomic substantially benefits from CUDA-enabled GPUs compatible with the RAPIDS ecosystem for larger datasets. At the scale of analysis presented in this paper, three specific steps took several hours to complete on a single GPU. First, image registration time scales linearly with the number of ROIs acquired and imaging cycles. Therefore, users are encouraged to evaluate and test parameter choices to optimize performance for their specific experimental design. For example, smaller, more homogeneous datasets may benefit from smaller training subsamples and SOM sizes, which will result in faster analysis. Moreover, users should select hardware that is appropriate for their analytical needs and desired turnaround time. Given the cyclical nature of image acquisition and imperfect repositioning of standard microscopy systems, aligning signals from all imaging cycles to a common reference is a prerequisite for joint analyses. We propose two different ways of aligning iteratively acquired immunofluorescence images: on the basis of either a nuclear or a pan-protein stain as a registration reference. To combat autofluorescence due to red blood cells (RBCs) in samples with abundant RBCs, we further used a RBC marker to specifically remove this source of noise. Detailed parameter choices and method descriptions are provided for the respective experimental datasets in the subsequent subsections. Differences between datasets, such as the use of NHS-E for foreground segmentation and the use of glycophorin A for erythrocyte segmentation in selected datasets, represent the iterative improvement of PathoPlex, with earlier experiments informing strategies for further scaling in subsequent experiments. In particular, the preprocessing steps described in this section refer only to the transformation of nuclear staining or NHS-E for registration purposes and do not indicate the transformation of marker intensities as used for further analysis, unless indicated otherwise. To align cycles from the CGN mouse dataset (Fig. 2), nuclear reference images (DAPI in cycles 1–17, DRAQ5 in cycles 18–42) were clipped image-wise to the range between the 1st and 99.9th intensity percentiles. Intensity values were z scored and normalized to the range 0–1. Gaussian blur with a sigma of 1 was then applied to the nuclear reference images to reduce noise and to improve comparability across cycles, followed by histogram matching of all subsequent images to the histogram of the reference image from the first cycle. These processed nuclear images were then aligned to the processed refence nuclear image obtained in the first imaging cycle. Finally, the maximum offset detected between any two cycles was subtracted from all sides of all registered images, which resulted in square-sized and equally sized images. For the study of specimens from individuals with advanced DKD and from individuals without diabetes (Fig. 3), all images were first corrected for camera distortion using the remap function of OpenCV85 by linear interpolation based on a reference image of a micrometre-scale microgrid captured with the respective microscope objective. To facilitate registration and further image processing, we applied local mean downscaling with a 2 × 2 pixel kernel to all imaging data. After NHS-E preprocessing, Gaussian blurring with a sigma of 1, thresholding at the 70th percentile and phase correlation-based offset detection with an upsampling factor of 5 were applied to identify the x and y offset between NHS-E images from all cycles, using the first imaging cycle as the reference. After correcting for this offset, SIFT-based homography detection86 was performed and images from subsequent cycles were warped to match the reference from the first cycle. Pre-registration and post-registration structural similarity index metrics (SSIMs) were evaluated for all registration pairs. Manual registration checks were performed for images with the following criteria: the SSIM decreased following registration; the SSIM after registration was lower than 0.3; any offset was greater than 300 pixels; or the SSIM after registration was lower than 0.8 and the SSIM improvement lower than 0.05. To account for RBC-derived autofluorescence, glycophorin A intensities across all images were clipped to the range between the 90th and 99.5th percentiles, and Otsu thresholding was used to binarize signals and to create RBC masks. Masks were smoothed through the removal of small objects <72 pixels, two iterations of binary dilation and hole filling. To remove signals from empty background without tissue, processed NHS-E signals were combined across all cycles by minimum projection (thus excluding areas with lifting at any point) for each image and Otsu thresholding was used to binarize signals. Autofluorescence correction was performed using secondary-antibody cycles acquired repeatedly throughout the experiment. For each primary-antibody imaging cycle, tissue autofluorescence was estimated by interpolating the signals from the nearest preceding and subsequent secondary-only cycles at the same wavelength, thereby accounting for minor fluctuations in autofluorescence. This estimated autofluorescence was then subtracted from the corresponding channels with lower intensity clipping at zero. To achieve reliable and accurate results, we used an iterative registration framework called Elastix87 and a Python wrapper package called PyElastix88. The first cycle was established as a reference, and subsequent cycles were aligned to it using normalized correlation as the optimization metric. The registration process involved iterative steps at six different resolution levels, with 1,000 iterations per level. We used the rigid Euler transform to account for x and y offsets as well as rotation. Further preprocessing was performed for brain samples with reduced contrast by truncating pixel intensity values to high percentiles. This approach focused on the sparsely available nuclei. All registrations were manually verified, and if individual registrations were unsuccessful, minor adjustments were made to the downsampling factor, spatial sample number or iteration number until satisfactory registration was achieved to maximize image inclusion. Owing to differences in size and lens characteristics between the immunofluorescence and PAS images, additional processing steps were required to align the two modalities. First, lens-specific calibration matrices were used to remove optical distortions as described above for the human advanced DKD experiment. As only greyscale images were used for registration, the PAS images were converted from RGB to greyscale using the OpenCV85 Python library. For the greyscale version of the immunofluorescence image stack, three structural marker channels (DRAQ5, LTL and collagen IV) were selected and combined into a single channel using weighted addition. To enhance visual similarity between the greyscale PAS and immunofluorescence variants, the PAS image was inverted, and histogram equalization was applied to both the PAS and immunofluorescence images. In a subsequent investigation, our goal was to evaluate the suitability of DAPI, DRAQ5 and NHS-E as registration reference labels. To achieve this goal, we acquired images of all three marker channels for each imaging spot in each cycle. For each image requiring registration, transformations based on each of the three markers were independently computed. This approach ensured that the different registration references could be compared using DRAQ5 as a quality metric, which consistently provided reliable and high-contrast images. All registrations were manually reviewed, and in instances where registration was not successful, minor adjustments were made to the downsampling factor, the spatial sample number or the iteration number until a satisfactory registration was achieved. This process ensured that no image had to be discarded and that high-quality alignments were obtained. Complete elution of bound antibodies between cycles is key to the cyclic acquisition of images. To quantify elution efficiency, secondary-antibody-only staining cycles were acquired throughout the experiments to establish a baseline autofluorescence profile. For instances when at least one value was equal to or lower than any intensity from the secondary-antibody-only cycles (as may be the case for non-abundant markers, for example, markers for rare immune cells or phosphorylation states), the images were manually re-evaluated to include specific staining. Protein co-expression clusters are the standard output of PathoPlex analyses with spatiomic, which captures specific co-expression patterns of different proteins at the pixel level. These patterns are jointly identified for all imaging data from each respective PathoPlex experiment, which results in a consistent clustering across all samples and facilitates comparisons of spatial expression or co-expression of immunofluorescence markers. Once clusters were identified, their constituting signals were compared to all other clusters and projected in space to infer the biological processes they represent. Last, cluster abundance was quantified and compared across conditions to delineate regulated signals. We applied this overall concept to each experimental dataset with parameters specified as described in the subsequent subsections. To limit bias due to different data sizes between samples and to reduce the computational burden of pixel-based clustering, a weighted subsample of approximately 5 million (10 million for the early human diabetes dataset) random pixel positions per imaging plate were sampled. For each imaging plate (n = 1 for all experiments, except for the human advanced DKD experiment, for which n = 2 imaging plates were used), the number of subsampled pixel positions was equally distributed across all disease states, and for each disease state, all samples were equally weighted. Finally, each FOV of a given sample was given the same weight in the subsample. For the early human diabetes treatment dataset, the subsample also considered equal numbers of pixels from periglomerular, glomerular and tubular images. Immunofluorescence markers differ in intensity range and abundance, which therefore requires preprocessing steps to improve comparability. On the basis of weighted random subsamples, histogram clipping and range normalization classes contained in the spatiomic.process submodule were fitted. After channel-wise fitting of the classes on the random weighted subsample, all channels of all images of each respective dataset were transformed according to the established clipping limits and normalization and scaling settings. For mouse samples and the advanced DKD experiment, histogram clipping was performed based on the 50th (lower) and 99.7th (upper) percentiles for each respective marker in the subsample. Owing to the extensive marker panel for the larger early human diabetes treatment experiment, coupled with the increased sensitivity of the confocal microscopy system used for this dataset, histogram clipping was performed based on absolute intensity thresholds established by human expert annotation in a condition-blind fashion, with each marker evaluated on random patches extracted from random images, followed by normalization of the clipped images. Pixel-based clustering was used to isolate groups of similar immunofluorescence marker signals (clusters), which formed the basis of all downstream analyses. The first step towards this clustering is the fitting of SOMs to the dataset to reduce data-point dimensionality and to ensure computational feasibility of graph clustering as well as to improve representation of signals that are relatively rare in the training data. For each imaging plate (n = 2 for the human advanced diabetes experiment, n = 1 for all other datasets), a SOM was trained on the corresponding weighted random subsample. For human samples, a final learning rate of 10−4 and a final Gaussian neighbourhood sigma of 10−3 (3 × 10−3 for the early human diabetes dataset) were used. On the basis of the representation of the signals contained in each experiment as provided by the respective SOM nodes, we used Leiden graph clustering to identify clusters of protein co-expression patterns and applied the clusters to all pixels from all images for each dataset. First, the similarity-based neighbourhood graph of SOM nodes was built using spatiomic.neighbor.knn_graph using cosine (early diabetes treatment experiment) or Euclidean distance (all other experiments). When image acquisition was performed using multiple plates (Fig. 3), the neighbourhood graph construction step was modified to use an adaptation of the batch-balanced k-nearest neighbours89 algorithm, implemented in spatiomic.neighbor.knn_graph. A neighbour count of k = 40 for each respective plate was used. When only one imaging plate was used (all other experiments), a neighbour count of k = 40 (k = 50 for the early human diabetes experiment) was used without any batch integration. After graph construction, we used the Leiden24 graph clustering algorithm to identify clusters of similar protein co-expression patterns. A Leiden resolution of 2.5 and an iteration count of 10,000 were used for the advanced DKD experiment, a resolution of 2.0 and an iteration count of 1,000 for the early human diabetes treatment experiment and a Leiden resolution of 1.0 and an iteration count of 10,000 for the mouse CGN experiment samples. To assign biological identities to protein co-expression clusters derived from PathoPlex output, we used a two-step approach that combined statistical overrepresentation with spatial distribution analysis and signal interpretation by human experts. Owing to computational constraints, statistical analyses were performed on subsampled data (used with the early diabetes experiment, with 500,000 pixels randomly selected from the weighted random subsample) or the representative SOM nodes (all other datasets). For each cluster, we calculated the mean normalized intensity and the log2-transformed fold change of marker intensity (relative to all pixels or nodes not assigned to that cluster), which reflected the signal strength and cluster specificity, respectively. Significance was assessed using a two-tailed t-test with Benjamini–Hochberg (early diabetes dataset) or Holm–Šidák (all other datasets) correction for multiple testing (implemented in spatiomic.tool.get_stats). Dominant markers in each cluster (termed high contributors) were identified by ranking markers based on the product of their mean intensity and log2-transformed fold change, and retaining only those with mean normalized intensity ≥ 0.2, adjusted P < 0.05 and log2-transformed fold change ≥ 1. Intensity histograms further visualized marker distribution patterns in each cluster for all markers and were also considered for cluster interpretation. Individual clusters were spatially projected onto the corresponding immunofluorescence and—where available—PAS-stained images. Human experts then visually validated and refined cluster identities by assessing their spatial distribution, considering morphological structures and correlations with immunofluorescence signals and integrating contextual biological knowledge. 2 and 3 did not include a pan-protein marker in their panel, no foreground segmentation was performed before pixel-level clustering, which resulted in multiple clusters that corresponded to empty background areas. To account for this limitation (largely circumvented by NHS-E and a larger antibody panel in Fig. 4), we restricted extended analyses (but not differential cluster abundance analyses) to specific foreground clusters only or we treated all background clusters as a single cluster. 4, we similarly assessed the specificity of all clusters and quantified their relative frequency, focusing visualizations and biological assessment to clusters deemed to represent specific signal (excluding minor imaging artefacts, autofluorescence signals and imperfect foreground segmentation) and accounting for >0.1% of foreground pixels. The specificity of clusters, defined as the extent to which clusters correspond to clear biological processes and/or structures, both at the spatial and at the molecular level, was assessed individually by a panel of three experts. In controversial cases, the experts presented their arguments until a unanimous decision for exclusion was reached. Cluster abundances were analysed by quantifying the number of pixels assigned to each cluster for each field of view in a dataset using spatiomic.tool.count_clusters. The abundances were normalized to a range of 0–1 to facilitate comparison and interpretation across FOVs. For the mouse CGN and the advanced diabetes experiment, these normalized abundances were further aggregated to obtain the mean cluster abundance per field of view for each mouse or patient. Differential abundance analysis was performed by quantifying the log2-transformed fold change in cluster abundance between different experimental and clinical conditions by performing a two-sided t-test. To account for the higher number of unique clusters and larger sample sizes, P values for the advanced DKD and the early diabetes treatment experiments were corrected for multiple testing with Benjamini–Hochberg false discovery rate adjustment. Results were visualized with spatiomic.plot.volcano for all datasets. For visualization purposes, the volcano plots were restricted to feature clusters determined to represent specific foreground signal. Moreover, to assess the impact of batch integration, quality control was performed by comparing differences in log2-transformed fold changes in inter-group cluster abundances between the imaging plates (Supplementary Fig. Based on our foundational pixel-level protein co-expression clusters, we used multiple downstream applications to connect the output from PathoPlex with community resources and public knowledge bases, showcasing how pixel-level data can be aggregated at different levels to derive information across multiple biological scales. UnPaSt42 is a biclustering method initially developed for unsupervised patient stratification based on omics data. UnPaSt identifies differentially expressed biclusters in a two-dimensional matrix with samples (for example, images or patients) in columns and features (for example, clusters) in rows. A differentially expressed bicluster is a submatrix consisting of samples and features such that these features are overexpressed or underexpressed in these samples compared with all other samples in the input data matrix. We applied UnPaSt with binarization P value threshold of P = 0.05, direction = ‘both' (to identify biclusters consisting of both upregulated and downregulated clusters) and all other parameters set to default to image-level and patient-level cluster intensities. As UnPaSt is not deterministic, consensus biclusters were built on the basis of results of ten independent runs. To evaluate the druggability of the molecular signature of advanced DKD and to identify potential therapeutics, we combined drug–protein interaction data from the CTD44 with STRING90 protein–protein interaction data. Initially, we manually identified AIFM1, TRPC6, CALR, HSPA5, ITGB1 and CTNNB1 as possible targets involved in the altered signalling cascades revealed by PathoPlex based on the significantly differentially expressed clusters and their molecular composition. Next, we used STRING data for Homo sapiens to link every potential target to a broader network of interacting proteins. We only considered direct interactions with a confidence score exceeding 0.75. In a subsequent step, CTD chemical–gene interaction data were used to extract possible therapeutics that affect proteins in our target networks. This search was limited to compound–protein interactions described in mice (Mus musculus), rats (Rattus norvegicus) or humans. Only interactions that did not involve co-treatment and did not affect the reaction of another externally administered compound were included. Our goal was to determine the impact of existing pharmacological treatments on the extended protein networks and to explore the potential for repurposing drugs authorized for other indications; therefore, we further filtered the results, preserving only entries for which the chemical name had a matching entry in the European Medicines Agency's list of authorized agents (date of consultation: 23 August 2023). In a second step, to further assess the impact of current antidiabetic drugs, results from all protein networks were combined and filtered for compounds containing ‘glutid', ‘gliflozin', ‘glitazon', ‘gliptin', ‘metformin', ‘pril' or ‘sartan'. Although pixel clusters provide valuable subcellular and extracellular information, they can also be used to inform existing cell-level clustering workflows. Statistical testing of nucleus counts per image was performed using statannotations91 with a two-sided nonparametric Mann–Whitney U-test, comparing diabetes and control samples. Centroids of all identified nuclei were then expanded radially by the smaller of either 5 µm or 50% of the shortest Delaunay triangulation edge length to approximate cell areas. Within these estimated areas, the relative abundances of pixel-level clusters were calculated to produce feature vectors for each cell. To identify cell-level metaclusters, a k-nearest neighbour graph was constructed using spatiomic.neighbor.knn_graph with neighbor_count = 50 followed by Leiden24 clustering with a resolution of 1.0. Condition-specificity of meta-clusters was confirmed by averaging their relative abundances across images at the patient level. Within each cell-level metacluster, the fraction of pixels corresponding to each pixel-level cluster was quantified, which provided a quantification of the cell-level co-occurrence patterns of pixel clusters. Differences in normalized metacluster abundance at the patient-level between conditions were visualized through a two-component principal component analysis using spatiomic.dimension.pca with default parameters and significance was established through a Mann–Whitney U-test with Bonferroni correction. PILOT41 is a previously published, multiscale, unsupervised method that uses optimal transport to compute the distance between data points and infer a disease trajectory. In summary, PILOT uses stepwise nonlinear models to determine significantly changing proteins or clusters across the disease trajectory. The immediate spatial neighbourhood analysis was based on join counts between unique clusters that were used to create an adjacency graph using spatiomic.spatial.vicinity_composition. First, for each pixel position, the eight surrounding pixel positions (Chebyshev distance of 1 pixel; that is, first-order queen neighbourhood) were examined to count the instances of nearby clusters. Next, connections between identical clusters were discarded, and the remaining connections were normalized to a range of 0–1. Although the entire adjacency matrix was quantified and evaluated, a neighbourhood cluster abundance of 7.5% was established as minimum value to be included in the graph plot for visualization purposes, focusing the visualization on the most common adjacencies. Condition-specific structural patterns were identified using MISTy40 (mistyR v.1.6.1). To that end, image data (112 images from advanced DKD samples and 310 from control samples) were aggregated at two different resolutions by summing the cluster counts in bins with a side length of 10 μm (62 × 62 pixels). Clusters capturing empty background, unstained tissue parts and nonspecific signals were collapsed into a single background cluster. To account for truncated bins at the edges of slides, we transformed the counts into proportions. We used these cluster proportions per bin as an intrinsic representation of the structure in a bin (MISTy intraview). With these view compositions per aggregation, a MISTy model was independently trained for each sample. MISTy can learn both simple linear relationships (for example, cluster X has a higher proportion if cluster Y has a lower proportion) and complex nonlinear relationships. By combining the predictions from the intraview and paraview for each cluster, MISTy enabled us to disentangle whether the prediction for a given cluster improves, and to what extent, when taking different spatial contexts into account. To compare the MISTy importance scores between conditions, we first computed the mean results per sample due to differing numbers of imaged FOVs. We then aggregated the MISTy results per patient and finally per condition (advanced DKD and controls). For each level of aggregation, group and view, we generated a graph representing the inferred relationships between clusters. The graph layout was calculated using Graphviz94 and NetworkX95. To contrast our findings at the protein-level with transcriptomic data and to establish a bridge to proposed treatments of DKD, we leveraged two public RNA-sequencing datasets with pharmacological intervention or different treatment data, covering rodent and human samples. Data were downloaded from the Gene Expression Omnibus (accession: GSE220939). Files from individual patients were converted to AnnData74 objects, and information on the diabetes and SGLT2i treatment status of each patient was added. To remove ambient RNA contamination, Cellbender96 (v.0.3) training was performed on each sample individually with a training fraction of 0.5 for 100 epochs. Observations with a Cellbender cell probability ≤ 0.5 were discarded. Next, all observations were combined into a single AnnData object, quality-control metrics were quantified and cell clustering was performed using the Python package scanpy97. As part of this step, mitochondrial, ribosomal and haemoglobin genes were identified, and quality control metrics were calculated using the calculate_qc_metrics function. Barcodes containing ≥50% mitochondrial RNA, ≥20% rRNA or ≥5% haemoglobin genes were discarded. Barcodes with reads for less than 500 or more than 5,000 genes were discarded to correct for doublets, as were genes detected in at most 4 observations. To reduce dimensionality, the first 50 principal components were computed using scanpy's pca method with the arpack singular value decomposition solver. As inter-individual batch effects were present, Harmony98 data integration was performed with standard parameters, using the harmonypy99 implementation available through the external module of scanpy. Graph clustering was performed using the Leiden algorithm72 with a resolution of 0.5. On the basis of cell-identity genes (adapted from a previous study46 and PanglaoDb100), clusters were manually annotated to represent different cell types. Cell types and transcripts related to the pathways altered at the protein level were chosen, and gene counts were visualized using the dotplot function of scanpy97. Statistics per cell type of interest were calculated on the depth-normalized gene counts using the get_stats function of spatiomic, configured to revert log1p transformation. Nonparametric independent Wilcoxon rank-sum testing was performed using the ranksum function of scipy101 with Benjamini–Hochberg correction. Preprocessed data were downloaded from the Gene Expression Omnibus (accession: GSE209821). Gene counts were read into R using Seurat and converted to AnnData74 with SeuratData and SeuratDisk. The provided counts had already been filtered for quality-control metrics and cluster labels from a previous study were provided47, thus, only depth normalization to the median read depth was performed. Observations were further filtered to only retain proximal tubule cells from controls, untreated mice or mice treated with soluble guanylate cyclase activators. Differential expression statistics of genes corresponding to defining markers of clusters upregulated in the proximal tubular compartment in the advanced DKD PathoPlex experiment were calculated based on depth-normalized gene counts using the spatiomic.tool.get_stats function, configured to run nonparametric independent Wilcoxon rank-sum testing using the ranksum function of scipy101 with Holm–Šidák correction. To further evaluate the expression of TRPC6 and AIFM1 in the proximal tubule of kidneys with metabolic damage, we assessed their expression in Btbrob/ob mice and control mice at 12 and 24 weeks of age. Immunofluorescence was performed on 5 FOVs per sample, with n = 3 samples per group and time point, which produced a total of n = 12 samples and 60 FOVs. The proximal tubule area was segmented by thresholding the LTL channel to an intensity greater than 1,500, followed by filling holes smaller than 1,000 pixels and removing objects smaller than 20,000 pixels. After proximal tubule segmentation, a random subsample of 50,000 pixel positions was selected from the proximal tubule area of each FOV. These subsamples were combined for each unique condition and time point. Changes in TRPC6 and AIFM1 expression were visualized through histograms of marker intensities, grouped by time point and condition. An independent t-test based on the statannotations91 Python package was used to assess statistical differences in TRPC6 expression between conditions at each time point. Statistical analyses, including cluster composition, differential cluster abundance and differential gene expression, were performed using spatiomic.tool.get_stats and statannotations. These tools internally rely on scipy101 and statsmodels102 and were used to perform either Benjamini-Hochberg or Holm–Šidák-corrected two-tailed t-tests (for cluster composition and differential abundance analysis), Mann–Whitney U-tests (for differences in nucleus counts and cell-level metacluster abundance) or nonparametric Wilcoxon rank-sum tests (for differential gene expression from single-cell and single-nucleus RNA-sequencing data). Bulk RNA-sequencing differential gene expression analysis was performed using PyDESeq2, combining single-factor analysis using Wald tests with log2-transformed fold change shrinkage with approximate posterior estimation generalized linear models103. Detailed descriptions of these tests, including information on input data and additional filters, are provided throughout the Methods. All other statistical analyses, including the quantification of changes in cell migration, clinical parameters (for example, proteinuria or eGFR), sample-specific variables (such as age), PEC activation, injury patterns in CGN and principal component analysis, were performed using GraphPad Prism (v.9). Violin plots report median and interquartile values. Significance was evaluated using the unpaired t-tests with Welch's correction comparing two continuous variables, a paired t-test for before and after settings, and the Brown–Forsythe, Welch ANOVA and Dunnett's tests when comparing three continuous variables. Correlation analyses were performed using Spearman rank coefficients. Statistical significance was defined as P < 0.05 for all analyses, with a threshold of P < 0.01 applied for specific cases as outlined throughout the methods. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The bulk RNA-sequencing data from NTS-treated mice have been deposited into the Hamburg University Research Data Repository (https://doi.org/10.25592/uhhfdm.17394). The public single-cell and single-nucleus RNA-sequencing datasets used in this study are available through Gene Expression Omnibus with accessions GSE220939 and GSE209821. PathoPlex animal multiplexed imaging data are available from Zenodo (https://doi.org/10.5281/zenodo.15212140)104. For human data, as a patient re-identification key is retained internally for scientific continuity of ongoing projects, and historical versions of data containing patient identifiers persist in secure institutional servers and physical laboratory records, the raw microscopy data cannot be fully anonymized and therefore cannot be deposited in a public repository in accordance with General Data Protection Regulation. Raw data can be made available upon reasonable request and subject to a material and data user agreement that ensures appropriate safeguards for data protection and privacy in compliance with General Data Protection Regulation. The senior corresponding author will respond to data requests, aiming to answer within 72 h, and provide data up to 1 month after the material and data user agreement has been signed by both parties. Source data are provided with this paper. The latest development version of spatiomic can be accessed from GitHub (https://github.com/complextissue/spatiomic). The documentation for spatiomic is available at: https://spatiomic.complextissue.com and includes example workflows for common analyses, including code to download example data. Moffitt, J. R., Lundberg, E. & Heyn, H. The emerging landscape of spatial profiling technologies. Angelo, M. et al. Multiplexed ion beam imaging of human breast tumors. Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry. Black, S. et al. CODEX multiplexed tissue imaging with DNA-conjugated antibodies. Highly multiplexed immunofluorescence imaging of human tissues and tumors using t-CyCIF and conventional optical microscopes. Greenbaum, S. et al. A spatially resolved timeline of the human maternal–fetal interface. Hickey, J. W. et al. Organization of the human intestine at single-cell resolution. de Souza, N., Zhao, S. & Bodenmiller, B. Multiplex protein imaging in tumour biology. Stoltzfus, C. R. et al. CytoMAP: a spatial analysis toolbox reveals features of myeloid cell organization in lymphoid tissues. Warchol, S. et al. Visinity: visual spatial neighborhood analysis for multiplexed tissue imaging data. Schapiro, D. et al. histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data. Somarakis, A., Van Unen, V., Koning, F., Lelieveldt, B. & Hollt, T. ImaCytE: visual exploration of cellular micro-environments for imaging mass cytometry data. Gut, G., Herrmann, M. D. & Pelkmans, L. Multiplexed protein maps link subcellular organization to cellular states. Wahle, P. et al. Multimodal spatiotemporal phenotyping of human retinal organoid development. Park, J. et al. Spatial omics technologies at multimodal and single cell/subcellular level. Benjamin, K. et al. Multiscale topology classifies cells in subcellular spatial transcriptomics. Singh, N. et al. Development of a 2-dimensional atlas of the human kidney with imaging mass cytometry. Hansen, J. et al. A reference tissue atlas for the human kidney. & Nolet, C. Machine learning in Python: main developments and technology trends in data science, machine learning, and artificial intelligence. Mancini, R., Ritacco, A., Lanciano, G. & Cucinotta, T. XPySom: High-Performance Self-Organizing Maps. The scverse project provides a computational ecosystem for single-cell omics data analysis. From Louvain to Leiden: guaranteeing well-connected communities. Kylies, D. et al. Expansion-enhanced super-resolution radial fluctuations enable nanoscale molecular profiling of pathology specimens. Pandit, K. et al. An open source toolkit for repurposing Illumina sequencing systems as versatile fluidics and imaging platforms. Wong, M. N., Tharaux, P. L., Grahammer, F. & Puelles, V. G. Parietal epithelial cell dysfunction in crescentic glomerulonephritis. Grigorieva, I. V. et al. A novel role for GATA3 in mesangial cells in glomerular development and injury. & Nikolic-Paterson, D. J. Activation and cellular localization of the p38 and JNK MAPK pathways in rat crescentic glomerulonephritis. Deep learning-based molecular morphometrics for kidney biopsies. Targeting CTGF, EGF and PDGF pathways to prevent progression of kidney disease. Ma, F. Y. et al. Blockade of the c-Jun amino terminal kinase prevents crescent formation and halts established anti-GBM glomerulonephritis in the rat. Ma, F. Y. et al. A pathogenic role for c-Jun amino-terminal kinase signaling in renal fibrosis and tubular cell apoptosis. The role of platelet-derived growth factor in focal segmental glomerulosclerosis. Eymael, J. et al. CD44 is required for the pathogenesis of experimental crescentic glomerulonephritis and collapsing focal segmental glomerulosclerosis. Foster, L. C. et al. Role of activating protein-1 and high mobility group-I(Y) protein in the induction of CD44 gene expression by interleukin-1β in vascular smooth muscle cells. Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: a generalist algorithm for cellular segmentation. Explainable multiview framework for dissecting spatial relationships from highly multiplexed data. Detection of patient-level distances from single cell genomics and pathomics data with optimal transport (PILOT). Hartung, M. et al. UnPaSt: unsupervised patient stratification by differentially expressed biclusters in omics data. STRING 8—a global view on proteins and their functional interactions in 630 organisms. A. et al. SGLT2 inhibitors mitigate kidney tubular metabolic and mTORC1 perturbations in youth-onset type 2 diabetes. Treatment effects of soluble guanylate cyclase modulation on diabetic kidney disease at single-cell resolution. Hickey, J. W. et al. Spatial mapping of protein composition and tissue organization: a primer for multiplexed antibody-based imaging. Kuett, L. et al. Three-dimensional imaging mass cytometry for highly multiplexed molecular and cellular mapping of tissues and the tumor microenvironment. Wang, X. Q. et al. Spatial predictors of immunotherapy response in triple-negative breast cancer. Hoch, T. et al. Multiplexed imaging mass cytometry of the chemokine milieus in melanoma characterizes features of the response to immunotherapy. Radtke, A. J. et al. IBEX: a versatile multiplex optical imaging approach for deep phenotyping and spatial analysis of cells in complex tissues. Kim, J. et al. Unsupervised discovery of tissue architecture in multiplexed imaging. B. et al. An atlas of healthy and injured cell states and niches in the human kidney. Robust phenotyping of highly multiplexed tissue imaging data using pixel-level clustering. Schapiro, D. et al. MITI minimum information guidelines for highly multiplexed tissue images. Organ mapping antibody panels: a community resource for standardized multiplexed tissue imaging. Gnirck, A. C. et al. Endogenous IL-22 is dispensable for experimental glomerulonephritis. Flanc, R. S. et al. A pathogenic role for JNK signaling in experimental anti-GBM glomerulonephritis. Ponzetti, M., Chinna Rao Devarapu, G., Rucci, N., Carlone, A. & Saggiomo, V. HistoEnder: a 3D printer-based histological slide autostainer that retains 3D printer functions. Guidelines for mouse and human DC functional assays. Wortel, I. M. N. et al. CelltrackR: an R package for fast and flexible analysis of immune cell migration data. Gies, S. E. et al. Optimized protocol for the multiomics processing of cryopreserved human kidney tissue. Kuehl, M., Wong, M. N., Wanner, N., Bonn, S. & Puelles, V. G. Gene count estimation with pytximport enables reproducible analysis of bulk RNA sequencing data in Python. Andrews, S. FastQC: a quality control tool for high throughput sequence data (2010); http://www.bioinformatics.babraham.ac.uk/projects/fastqc. & Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Muzellec, B., Teleńczuk, M., Cabeli, V. & Andreux, M. PyDESeq2: a Python package for bulk RNA-seq differential expression analysis. Expanding the coverage of regulons from high-confidence prior knowledge for accurate estimation of transcription factor activities. Badia-I-Mompel, P. et al. decoupleR: ensemble of computational methods to infer biological activities from omics data. Virshup, I., Rybakov, S., Theis, F. J., Angerer, P. & Wolf, F. A. anndata: Annotated data. McKinney, W. pandas: a foundational Python library for data analysis and statistics. 31st Confernce on Neural Information Processing Systems (ed. Pedregosa, F. et al. Scikit-learn: machine learning in Python. McInnes, L., Healy, J., Saul, N. & Großberger, L. UMAP: uniform manifold approximation and projection. & Anselin, L. in Handbook of Applied Spatial Analysis: Software Tools, Methods and Applications (eds Fischer, M. M. & Getis, A.) Emons, M. et al. pasta: pattern analysis for spatial omics data. van der Walt, S. et al. scikit-image: image processing in Python. Lowe, D. G. Distinctive image features from scale-invariant keypoints. & Pluim, J. P. W. elastix: a toolbox for intensity-based medical image registration. Klein, A. PyElastix—Python wrapper for the Elastix nonrigid registration toolkit. Polański, K. et al. BBKNN: fast batch alignment of single cell transcriptomes. The STRING database in 2023: protein–protein association networks and functional enrichment analyses for any sequenced genome of interest. Ellson, J., Gansner, E., Koutsofios, L., North, S. C. & Woodhull, G. in Graph Drawing (eds Mutzel, P. et al.) 483–484 (Springer, 2002). Hagberg, A., Swart, P. & Chult, D. S. Exploring network structure, dynamics, and function using NetworkX. 7th Python in Science Conference (eds Varoquaux, G. et al.) 11–15 (2008). Fleming, S. J. et al. Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender. Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Franzén, O., Gan, L.-M. & Björkegren, J. L. M. PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data. Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Skipper, S. & Perktold, J. statsmodels: econometric and statistical modeling with Python. 9th Python in Science Conference (eds van der Walt, S. & Millman, J.) Zhu, A., Ibrahim, J. G. & Love, M. I. Heavy-tailed prior distributions for sequence count data: removing the noise and preserving large differences. & Puelles, V. Pathology-oriented multiplexing enables integrative disease mapping—imaging files. & Puelles, V. Pathology-oriented multiplexing enablesintegrative disease mapping—processing code and 3D printing files. (project A20); Collaborative Research Center TRR 422 (PodoSigN) to F.B., F.G., N.W., M.L., T.W., N.M.T., C.M.-S., T.B.H. Swiss State Secretariat for Education, Research and Innovation (SERI): to F.M.D.C. Swiss National Science Foundation and InnoSuisse: BRIDGE program to G.G. Swiss National Science Foundation: research grant to M.D. Human Frontier Science Program (HFSP) research grant (RGP0032/2022) to P.J.S. National Heart, Lung, and Blood Institute: research grant (HL165433) to P. Bjornstand. American Heart Association: research grant (20IPA35260142) to P. Bjornstand. American Diabetes Association: research grants (7-23-ICTST2DY-08 and 7-23-ICTST2DY-01) to P. Bjornstand. National Institutes of Health: research grant (DK081943) to M. Kretzler. University of Michigan: research grant (RPC2) to M. Kretzler. European Foundation for the Study of Diabetes: research grant to O.L. Open access funding provided by Universitätsklinikum Hamburg-Eppendorf (UKE). These authors contributed equally: Malte Kuehl, Yusuke Okabayashi, Milagros N. Wong, Lukas Gernhold, Gabriele Gut These authors jointly supervised this work: Tobias B. Huber, Petter Bjornstad, Matthias Kretzler, Olivia Lenoir, David J. Nikolic-Paterson, Lucas Pelkmans, Stefan Bonn and Victor G. Puelles Malte Kuehl, Milagros N. Wong, Jens R. Nyengaard & Victor G. Puelles Malte Kuehl, Milagros N. Wong, Jens R. Nyengaard & Victor G. Puelles Malte Kuehl, Nico Kaiser, Sven Heins, Marina Zimmermann & Stefan Bonn Department of Medicine, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Malte Kuehl, Yusuke Okabayashi, Milagros N. Wong, Lukas Gernhold, Nico Kaiser, Maria Schwerk, Stefanie K. Gräfe, Sam Mezher, Thiago Goldbeck-Strieder, Lukas Klinkert, Ann-Christin Gnirck, Fabian Braun, Oliver Kretz, Hui Wu, Florian Grahammer, Fabian Haas, Dominik Kylies, Nicola Wanner, Jan Czogalla, Bernhard Dumoulin, Nikolay Zolotarev, Maja Lindenmeyer, Nicola M. Tomas, Jan-Eric Turner, Tobias B. Huber & Victor G. Puelles Malte Kuehl, Yusuke Okabayashi, Milagros N. Wong, Lukas Gernhold, Nico Kaiser, Maria Schwerk, Stefanie K. Gräfe, Sam Mezher, Thiago Goldbeck-Strieder, Lukas Klinkert, Fabian Braun, Oliver Kretz, Hui Wu, Florian Grahammer, Marina Zimmermann, Fabian Haas, Dominik Kylies, Nicola Wanner, Jan Czogalla, Bernhard Dumoulin, Nikolay Zolotarev, Maja Lindenmeyer, Nicola M. Tomas, Catherine Meyer-Schwesinger, Jan-Eric Turner, Tobias B. Huber, Stefan Bonn & Victor G. Puelles Gabriele Gut, Jacobo Sarabia del Castillo & Lucas Pelkmans Department of Nephrology, Monash Medical Centre and Department of Medicine, Monash University, Melbourne, Victoria, Australia Jovan Tanevski, Philipp S. L. Schäfer & Julio Saez-Rodriguez Olga Zolotareva, Michael Hartung, Fernando M. Delgado Chaves & Jan Baumbach Data Science in Systems Biology, TUM School of Life Sciences, Technical University of Munich, Freising, Germany Mehdi Joodaki, Victor Parra, Mina Shaigan & Ivan Costa Department of Urology and Kidney Transplantation, Martin-Luther-University, Halle, Germany Cell Communication and Migration Laboratory, Institute of Biochemistry and Molecular Cell Biology, Center for Experimental Medicine, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Department of Medicine, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Department of Pathology, University Medical Centre Hamburg-Eppendorf, Hamburg, Germany Institute of Pathology, Nephropathology Section, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Center for Experimental Medicine, Institute of Cellular and Integrative Physiology, University Medical Center Hamburg-Eppendorf, Hamburg, Germany Patrick Bruneval, Pierre-Louis Tharaux & Olivia Lenoir Laura Pyle, Ye Ji Choi & Petter Bjornstad Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and carried out various biological methodologies (for example, multiplexed imaging, 3D printing, animal experiments, in vitro drug modulation, electron microscopy validation and histopathology). carried out data visualization and figure design. All authors contributed to writing, review and editing of the manuscript. Correspondence to Malte Kuehl, Yusuke Okabayashi or Victor G. Puelles. and L. Pelkmans are listed as inventors on patents related to the 4i and single-pixel clustering methods and L. Pelkmans holds ownership in Apricot Therapeutics, which offers commercial services related to multiplexed histopathology. reports serving or having served as a consultant for AstraZeneca, Bayer, Bristol-Myers Squibb, Boehringer Ingelheim, Eli-Lilly, LG Chemistry, Sanofi, Novo Nordisk and Horizon Pharma. also serves or has served on the advisory boards and/or steering committees of AstraZeneca, Bayer, Boehringer Ingelheim, Novo Nordisk, and XORTX. M. Kretzler reports partial funding for contributions here originated from the Renal Pre-Competitive Consortium (RPC2), as funded by AstraZeneca, Eli Lilly, Janssen Pharmaceuticals, Novo Nordisk and Roche-Genentech. M. Kuehl is an employee of and holds an ownership interest in KH Biotechnology, which provides consulting services to Lamin Labs. All other authors do not report competing interests. Nature thanks Tarek El-Achkar, Beatrice Knudsen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Full elution of antibodies and (b) absence of remnant signals when re-staining with secondary antibodies. (d) Secondary-only cycles do not show signal amplification. (e) Re-staining of SOD1 after 61 cycles, showing very strong agreement between intensity distributions. vWF, von Willebrand factor; SRB1, Scavenger receptor class B member 1; PDI, Protein disulfide isomerase; EZR, Ezrin; p-rp-S6, phospho-ribosomal protein S6; p-H3, phospho-histone-H3; PCNA, Proliferating cell nuclear antigen; COL4, Collagen type IV; LAMP1, lysosomal-associated membrane protein 1;VMT, Vimentin; FN, Fibronectin; WT1, Wilms tumor 1; ACE2, Angiotensin-converting enzyme 2; CK19, Cytokeratin 19; ANXA3, Annexin 3; LTL, Lotus tetragonolobus lectin; α-SMA, alpha-smooth muscle actin (ACTA2); SYNPO, Synaptopodin; EMCN, Endomucin; NPHN, Nephrin; ab, antibody; AF, autofluorescence; PDPN, Podoplanin; LMN, Laminin; AQP2, Aquaporin 2; E-CAD, E-cadherin; SOD1: Superoxide Dismutase 1; HSD11B2: 11-β-hydroxysteroid dehydrogenase type 2; u-H2B: ubiquitylated histone 2B. (a) Schematic overview of all strategies deployed. (b) PathoPlex is compatible with multiple microscopy systems, including widefield, spinning disk and laser confocal, which directly determine imaging time, file size and image resolution. (c) Strategies to minimize bias introduction are based on creating self-contained batches, where tissues are processed in parallel using imaging chambers that contain specimens representing all experimental groups, including controls. (d) Suggested imaging chambers for histopathological studies (2–24 wells). (e) Multi-well pipetting can be performed manually or using liquid handling systems. Alternatively, 3D printing can be used to customize a 1-well imaging chamber, which simplifies liquid handling. The printer head is used to automate liquid addition and removal. (g) Specificity, and elution controls of 3D printer-based imaging cycles. Parts of panels a, c and e were created using BioRender (https://biorender.com). (a) Quantitative pathology analyses to differentiate acute injury and CGN vs controls, showing compartment-specific changes in acute injury and CGN. (b) While mice with CGN have overt lesions, those with acute injury show only minor abnormalities, namely subtle vacuolation in the proximal tubuli (green). (c) Volcano plot showing all regulated clusters comparing controls to acute injury, featured cluster 26 (C26). (d) Principal component (PC) analysis revealed clear separation between images from controls (blue) and acute injury (orange) specimens with C26 being the top contributor to PC1. (e) Spatial projections of cluster 26 (C26) show a regulation pattern in the luminal side of kidney tubuli, a similar location to the one used for pathologists to define acute injury. (a) Schematic description of experimental design with nephrotoxic serum (NTS) glomerulonephritis and glomerular nuclear isolation for bulk RNA sequencing. (b) Volcano plots show transcriptionally regulated genes at day 3 post NTS injection. (c) Transcription factor activity score, and (d) differential expression of JUN-regulated targets, where blue means downregulated by JUN and red means upregulated by JUN. Parts of panel a were created using BioRender (https://biorender.com). (a) Percentage of CD44+phospho-c-Jun+ parietal epithelial cells (PECs) among controls, NTS injected day 1 (alone and vehicle) and NTS injected day 1 with a preventive JNK inhibitor (JNKi) administration. This is a representative image from 3 biological replicates. (b) Expression of TRCP6 using Imaging Mass Cytometry in the human kidney (controls and diabetic kidney disease; DKD). These are representative images from 2 biological replicates per condition. Parts of panel c were created using BioRender (https://biorender.com). Histopathological staining allows cluster assignment to exact structural locations. Cluster abundances correlate with clinical renal function (estimated glomerular filtration rate; eGFR). (a) Schematic representation of our approach to characterize direct spatial interactions between neighbouring clusters. (b) Spatial interactions highlight changes in associations across disease states and scales. GR, Glucocorticoid receptor; PTMs, Post-translational modifications; PTs, proximal tubular cells; DT, distal tubule; CD, collecting duct; ER, Endoplasmic reticulum; MVs, microvesicles; ECM, Extra cellular matrix; activ, activation; podo, podocytes; physio, physiology; metab, metabolism; cytoskel, cytoskeletal; ECs, endothelial cells; GECs, glomerular endothelial cells; mTOR, mammalian target of Rapamycin; macro, macrophages; lyso, lysosomes; microtub, microtubules; mechanotrans, mechanotransduction; Cell adh, cell adhesion; Myofib, myofibroblasts. (a) Schematic representation of our approach to characterize cluster-cluster predictions using MISTy. (b) Cluster predictions highlight changes across disease states and scales. GR, Glucocorticoid receptor; PTMs, Post-translational modifications; PTs, proximal tubular cells; DT, distal tubule; CD, collecting duct; ER, Endoplasmic reticulum; MVs, microvesicles; ECM, Extra cellular matrix; activ, activation; podo, podocytes; physio, physiology; metab, metabolism; cytoskel, cytoskeletal; ECs, endothelial cells; GECs, glomerular endothelial cells; mTOR, mammalian target of Rapamycin; macro, macrophages; lyso, lysosomes; microtub, microtubules; mechanotrans, mechanotransduction; Cell adh, cell adhesion; Myofib, myofibroblasts. (a) Pseudotime analysis was performed based on our interpretable clusters, identifying a path from controls to diabetic kidney disease (DKD) that correlates with histopathological changes. This path can be separated into two trajectories. (b) Trajectory 1 defined a transition based on estimated glomerular filtration rate (eGFR), marking the range in renal function in our non-diabetic controls. Pseudotime was strongly associated with clusters representing loss of tubular integrity, extracellular matrix remodelling (ECM; fibrosis) and myofibroblast expansion (fibrosis). (c) Trajectory 2 was strongly associated with clusters representing podocyte injury, mitochondrial stress in proximal tubuli (PTs) and glucocorticoid receptor (GR) dysfunction. (a) Cell types identified via single-cell RNA-sequencing. Even in early diabetic kidney disease, transcripts related to ER stress, Ca2+-mediated tubular injury, integrin beta 1-, and GR-signaling (b), as well as ECM remodelling and pro-fibrotic signaling (c) showed differential gene expression patterns compatible with differences observed at the protein level found in advanced diabetic kidney disease. SGLT2 inhibitor (SGLT2i)-treated patients exhibited incomplete modulation of transcriptomic changes. TAL: Thick ascending limb; PT: Proximal tubule; PC: Principal cell; ECs: Endothelial cells; IC: Intercalated cell; PC/CNT: Principal cells / Connecting tubule; DCT: Distal convoluted tubule; Mes/VSMC/Fib: Mesangial cells/Vascular smooth muscle cells/Fibroblasts; ATL/PEC: Ascending thin limb/Parietal epithelial cells; GECs: Glomerular endothelial cells; Mac/Mono: Macrophages/Monocytes; Lymph: Lymphocytes. This video shows the use of a 3D printer with custom 3D-printed cartridges and imaging chambers to perform automated liquid handling following the PathoPlex protocol. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Pathology-oriented multiplexing enables integrative disease mapping. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250718031205.htm'>Selfies, sugar, and death: How tourists are endangering elephants</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 12:17:14
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A study led by a scientist at the University of California San Diego offers new warnings on the dangers of human interactions with wildlife. Assistant Professor Shermin de Silva of the School of Biological Sciences studies endangered Asian elephants and has reported on their shrinking habitats, a downturn that has resulted in territorial conflicts between people and elephants. Along with her study coauthors, de Silva now provides fresh evidence in the journal Ecological Solutions and Evidence on the serious consequences of humans supplying food to wild animals. Even for those who live in areas without native elephant populations, the new study provides cautionary information about interactions with any wildlife species living among us. Wild elephants are a prime attraction in Asia, with Sri Lanka and India featuring some of the world's last abundant populations of Asian elephants. In Sri Lanka, de Silva studied 18 years of elephant-tourist interactions at Udawalawe National Park. She found that the elephants congregating near tourists at the park's southern boundary have developed "begging" behavior and have become habituated to sugary foods, sometimes breaking through fences to continue being fed. In India's Sigur region, study coauthors Priya Davidar and Jean-Philippe Puyravaud of the Sigur Nature Trust observed feeding interactions with 11 male Asian elephants, four of which died from suspected human causes. One elephant was successfully rehabilitated and returned to natural foraging behavior. "Many people, especially foreign tourists, think Asian elephants are tame and docile, like domestic pets," said de Silva, a faculty member in the Department of Ecology, Behavior and Evolution and founder of the non-profit conservation organization Trunks & Leaves. "They don't realize these are formidable wild animals and try to get too close in order to take photographs or selfies, which can end badly for both parties." Some elephants, including a popular male named Rambo, became local celebrities as they solicited food from tourists over several years. Since wild elephant feeding cannot be adequately regulated as an ongoing activity, the authors of the study recommend that feeding bans should be strictly enforced. "But this encourages wild animals to seek food from people, attracting them to areas that can put themselves or people at risk," said de Silva. "It can be a conduit for disease transfer between species. Such interactions, de Silva says, can change animals' movement patterns and possibly force them to lose knowledge of natural food sources if they become too dependent on handouts. Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41551-025-01380-1'>Sensitization of tumours to immunotherapy by boosting early type-I interferon responses enables epitope spreading</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 11:14:46
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The success of cancer immunotherapies is predicated on the targeting of highly expressed neoepitopes, which preferentially favours malignancies with high mutational burden. Here we show that early responses by type-I interferons mediate the success of immune checkpoint inhibitors as well as epitope spreading in poorly immunogenic tumours and that these interferon responses can be enhanced via systemic administration of lipid particles loaded with RNA coding for tumour-unspecific antigens. In mice, the immune responses of tumours sensitive to checkpoint inhibitors were transferable to resistant tumours and resulted in heightened immunity with antigenic spreading that protected the animals from tumour rechallenge. Our findings show that the resistance of tumours to immunotherapy is dictated by the absence of a damage response, which can be restored by boosting early type-I interferon responses to enable epitope spreading and self-amplifying responses in treatment-refractory tumours. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 digital issues and online access to articles Prices may be subject to local taxes which are calculated during checkout The main data supporting the results in this study are available within the paper and its Supplementary Information and provided as supplementary source data. Raw data for sequencing results can be found at https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255666. Source data are provided with this paper. Bagchi, S., Yuan, R. & Engleman, E. G. Immune checkpoint inhibitors for the treatment of cancer: clinical impact and mechanisms of response and resistance. Sharma, P. & Allison, J. P. The future of immune checkpoint therapy. Granier, C. et al. Mechanisms of action and rationale for the use of checkpoint inhibitors in cancer. Wei, S. C., Duffy, C. R. & Allison, J. P. Fundamental mechanisms of immune checkpoint blockade therapy. Verma, V. et al. A systematic review of the cost and cost-effectiveness studies of immune checkpoint inhibitors. Chauhan, A., Burkeen, G., Houranieh, J., Arnold, S. & Anthony, L. Immune checkpoint-associated cardiotoxicity: case report with systematic review of literature. Snyder, A. et al. Genetic basis for clinical response to CTLA-4 blockade in melanoma. Rizvi, N. A. et al. Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer. Kandoth, C. et al. Mutational landscape and significance across 12 major cancer types. Fares, C. M., Van Allen, E. M., Drake, C. G., Allison, J. P. & Hu-Lieskovan, S. Mechanisms of resistance to immune checkpoint blockade: why does checkpoint inhibitor immunotherapy not work for all patients? & Flaherty, K. T. Mechanisms of resistance to immune checkpoint inhibitors. A. Hallmarks of response, resistance, and toxicity to immune checkpoint blockade. Fuertes, M. B., Woo, S. R., Burnett, B., Fu, Y. X. & Gajewski, T. F. Type I interferon response and innate immune sensing of cancer. Benci, J. L. et al. Tumour interferon signaling regulates a multigenic resistance program to immune checkpoint blockade. Wang, X. et al. Suppression of type I IFN signaling in tumours mediates resistance to anti-PD-1 treatment that can be overcome by radiotherapy. Sustained type I interferon signaling as a mechanism of resistance to PD-1 blockade. Zhou, L. et al. A dual role of type I interferons in antitumour immunity. Type I interferon is selectively required by dendritic cells for immune rejection of tumours. Host type I IFN signals are required for antitumour CD8+ T cell responses through CD8α+ dendritic cells. Sayour, E. J. et al. Personalized tumour RNA loaded lipid-nanoparticles prime the systemic and intratumoural milieu for response to cancer immunotherapy. Sayour, E. J. et al. Systemic activation of antigen-presenting cells via RNA-loaded nanoparticles. Kariko, K., Buckstein, M., Ni, H. & Weissman, D. Suppression of RNA recognition by Toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA. Baklaushev, V. P. et al. Luciferase expression allows bioluminescence imaging but imposes limitations on the orthotopic mouse (4T1) model of breast cancer. Mendez-Gomez, H. R. et al. RNA aggregates harness the danger response for potent cancer immunotherapy. Ansari, A. M. et al. Cellular GFP toxicity and immunogenicity: potential confounders in in vivo cell tracking experiments. Poloni, C. et al. T-cell activation-induced marker assays in health and disease. Updated analysis of KEYNOTE-024: pembrolizumab versus platinum-based chemotherapy for advanced non-small-cell lung cancer with PD-L1 tumour proportion score of 50% or greater. Samstein, R. M. et al. Tumour mutational load predicts survival after immunotherapy across multiple cancer types. The vigorous immune microenvironment of microsatellite instable colon cancer is balanced by multiple counter-inhibitory checkpoints. Response rates to anti-PD-1 immunotherapy in microsatellite-stable solid tumours with 10 or more mutations per megabase. Hildner, K. et al. Batf3 deficiency reveals a critical role for CD8α+ dendritic cells in cytotoxic T cell immunity. Woo, S. R. et al. STING-dependent cytosolic DNA sensing mediates innate immune recognition of immunogenic tumours. Suschak, J. J., Wang, S., Fitzgerald, K. A. & Lu, S. A cGAS-independent STING/IRF7 pathway mediates the immunogenicity of DNA vaccines. Ayers, M. et al. IFN-γ-related mRNA profile predicts clinical response to PD-1 blockade. Karachi, A. et al. Modulation of temozolomide dose differentially affects T-cell response to immune checkpoint inhibition. Zhai, Y. et al. Cloning and characterization of the genes encoding the murine homologues of the human melanoma antigens MART1 and gp100. Identification of Claudin-6 as a molecular biomarker in pan-cancer through multiple omics integrative analysis. Therapy of established B16-F10 melanoma tumours by a single vaccination of CTL/T helper peptides in VacciMax. Development of cancer immunotherapies based on identification of the genes encoding cancer regression antigens. Overwijk, W. W. & Restifo, N. P. B16 as a mouse model for human melanoma. Balmas, E. et al. Islet-autoreactive CD4+ T cells are linked with response to alefacept in type 1 diabetes. We thank G. Moore for assistance with veterinary animal experiments; D. Li and Biostatistics and Computational Biology Shared Resource for statistical support; L. Fagman and K. Young for help with large animals; and A. Dongtao Fu of the UF Molecular Pathology Core at the Department of Pathology, Immunology and Laboratory Medicine for assistance with genomics experiments. Cartoon images including those for mouse, canine, and schematics in figures were created with BioRender.com. This work was supported by federal awards W81XWH-17-1-0510, K08CA199224, R37CA251978, R01CA266857, R01FD007268 (FDA – OOPD, Office of Orphan Products Development), and P30CA247796; Florida Department of Health 20B11 (Bankhead Coley) and 20L07 (Live Like Bella) awards; and foundation grants from CureSearch (Catapult Award), Alex's Lemonade Stand (R Accelerated Award), Rally Foundation, Hyundai Hope on Wheels (Hope Scholar Award), Stop Children's Cancer and the Bonnie R. Freeman Professorship for Pediatric Oncology Research, Danny's Dream, the Medulloblastoma Initiative (MBI) and Cure Group 4 Medulloblastoma Consortium, Ians Friends Foundation, and the National Pediatric Cancer Foundation. These authors contributed equally: Sadeem Qdaisat, Brandon Wummer, Brian D. Stover. Lillian S. Wells Department of Neurosurgery, Preston A. Wells, Jr. Center for Brain Tumor Therapy, McKnight Brain Institute, University of Florida, Gainesville, FL, USA Sadeem Qdaisat, Brandon Wummer, Dingpeng Zhang, James McGuiness, Frances Weidert, Jonathan Chardon-Robles, Anna DeVries, Chong Zhao, Christiano Marconi, Aida Karachi, Chao Xie, Ruixuan Liu, Stephen Michel, Rachel S. F. Moor, Christina von Roemeling, Nagheme Thomas, Hilary Geffrard, Yodarlynis Campaneria, Elizabeth Ogando-Rivas, Dhruvkumar Soni, Jianping Huang, Maryam Rahman, Catherine T. Flores, Duane A. Mitchell, Hector R. Mendez-Gomez & Elias J. Sayour University of Florida Genetics Institute, University of Florida, Gainesville, FL, USA Department of Pediatrics, Division of Pediatric Hematology-Oncology, UF Health Cancer Center, University of Florida, Gainesville, FL, USA Brian D. Stover, Gabriel Jobin, Xiaojie Ma, Arnav Barpujari, Cathleen Rabideau, John A. Ligon, Paul Castillo & Elias J. Sayour Department of Radiation Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA Department of Mechanical and Aerospace Engineering, University of Florida, Gainesville, FL, USA Duy T. Nguyen & W. Gregory Sawyer Department of Medicine, Division of Hematology and Oncology, UF Health Cancer Center, University of Florida, Gainesville, FL, USA College of Veterinary Medicine, University of Florida, Gainesville, FL, USA Department of Pathology, University of Florida, Gainesville, FL, USA Center of Immunotherapy and Precision Immuno-Oncology/Head and Neck Institute, Cleveland Clinic, Cleveland, OH, USA Department of Biostatistics, University of Florida, Gainesville, FL, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar designed and performed the experiments, interpreted the data and co-wrote the paper. conducted experimental studies and generated RNA vaccine reagents. A.G. assisted with experiments and co-wrote the paper. provided resources and oversight. led H&E organ analyses. provided critical resources and oversight. provided key resources and oversight. conceived and designed studies, performed experiments, provided resources and co-wrote the paper. Correspondence to Elias J. Sayour. hold an ownership interest in iOncologi, Inc. W.G.S. holds interest in Aurita, Inc. E.J.S. is a paid consultant for Siren Biotechnology and serves on the external advisory board of Nature's Toolbox with stock options. The paper discusses pending patent applications from S.Q., J.M., A.G., W.G.S., M.R., D.A.M., P.C., H.R.M.-G. and E.J.S, some of which are licensed to iOncologi, Inc. H.R.M.-G. and E.J.S. receive royalty payments from iOncologi, Inc. Nature Biomedical Engineering thanks Yang-Xin Fu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Source data for supplementary figures. Source data for Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. et al. Sensitization of tumours to immunotherapy by boosting early type-I interferon responses enables epitope spreading. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-02023-w'>Class of 2025: five PhD students reveal realigned priorities in wake of COVID and cuts</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 11:06:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Amanda Heidt is a freelance writer and editor in Moab, Utah. Carys Herbert inspects a telescope used in citizen-science projects at the University of Kent's Beacon Observatory in Canterbury, UK — while she also scans the horizon for a permanent position after her PhD. Others are left wondering whether a research career is still worth pursuing, particularly in places facing economic crises or being torn apart by war, such as Sudan, Ukraine and Gaza. US government funding for scientific research is currently approaching a 25-year low, even though global investment in research and development has tripled over the past three decades. But investment in science and technology hasn't necessarily led to more jobs, meaning that scientists attempting to launch their careers today continue to face shortages in faculty positions in academia alongside a shrinking industry job market. Nature's careers team spoke to five early-career researchers who began their PhDs during the early days of the COVID-19 pandemic and are now planning their next steps. I made that choice before the lockdowns happened, but it was an absolute blessing in disguise to have stayed in a place I knew. For my PhD, I characterized the surface spots on young stars using citizen-science data through a programme called the Hunting Outbursting Young Stars project. Instead, I used data from citizen scientists who submit observations to a database using personal telescopes and cameras. Looking back on my PhD, I think that citizen science has the power to be important and effective in these uncertain and under-funded times. I've been out of the academic world for about eight months now, and I'm a bit disillusioned because it feels like you can do everything right and still end up without a permanent position. I am a queer woman of child-bearing age who wants to have kids one day and, in my opinion, it's not great to be any of those things in the United States right now. I also have some chronic health conditions, and I don't want to have to navigate them in the US health-care system. Unfortunately, things feel only a little bit better in the United Kingdom because it seems like funding for UK research projects is scarce too. I've really enjoyed doing outreach and being in a university setting. I value that kind of devotion to learning and research. Ever since the COVID-19 pandemic, I've been mindful of connecting intentionally with colleagues. Maybe I don't see these people every day, because we're an international community, but I do try to appreciate those bonds and keep in touch. Rachel Gatlin's postdoc position hangs in limbo amid a US federal hiring freeze. We didn't even have pipettes when I started, and my principal investigator went on maternity leave soon after. But the past six years of my life have revolved around using mice to study what happens in our brains when we're isolated physically and socially from each other, which ended up being topical during the pandemic. Then, in February, I had a video meeting with the two PIs who I'd be splitting my time with, and they told me there'd been a federal hiring freeze (now extended until October). Right now, my postdoc position is in limbo, and it's hard not to feel that if I'd defended my PhD even two weeks earlier, I could have avoided this. #OpenToWork: how laid-off US scientists are coping with shattered careers #OpenToWork: how laid-off US scientists are coping with shattered careers If this grant doesn't come through, it's hard to know what I'll do. I've always heard that academia is the more stable career path, and part of me went looking for a postdoc position thinking that I could ride the uncertainty out, but it doesn't feel like that's the case any more. My hope is that if I can get this position settled, the next five years will help me to work out the next steps. Will defend his biochemistry PhD at the University of Wisconsin–Madison later this year. I started my programme in August 2020, so COVID-19 restrictions made it interesting from the start. My cohort wanted to connect, but it was difficult because of social distancing, and it took me a long time to get my bearings in a new city (I'd moved from Illinois). Fortunately, I was still able to do two out of three lab rotations in person during my first year, and by the time I picked my lab in 2021, we were back to doing full-time research in person. I knew Trump was going to start his second term with more of a plan, but I didn't anticipate what would happen. We don't know how long the money is going to last or whether new funding will come in, so we're trying to cut costs. My thesis research is developing methods for sorting out and breaking down plastics in the municipal waste stream. Academia is supposed to be the safe option, but there is little to no upward mobility and I'm not very interested in doing an academic postdoc. Government was a strong contender, but the US administration is cutting funding for government research and seems to be openly against sustainability, from what I've seen. Even if policies get walked back, many government researchers have already been laid off or resigned, and the public's trust has been lost. For years I've heard people say ‘biotechnology is in a slump, but maybe by the time you graduate, things will be better'. So, on all three fronts, US job prospects are not great, and I don't want to compromise on the type of research I want to do. I don't want to sound too alarmist, but it does often feel as if people's livelihoods are on the line. They requested anonymity out of concern for their family's safety and future job prospects. I wrote my master's thesis from a tiny, enclosed balcony in 2020 during lockdown. At the same time, Israel was in another period of intense anti-government protests because of criminal charges against our Prime Minister, Benjamin Netanyahu. #OpenToWork: how laid-off US scientists are coping with shattered careers ‘I don't want this kind of life': graduate students question career options Giant radio telescope was ‘a natural magnet' for African talent Bring science into business schools to train the leaders of tomorrow Giant radio telescope was ‘a natural magnet' for African talent Launch your lab at the Max Perutz Labs and join us in our mission to analyze and reconstitute complex biological systems across scales. IFReC, The University of Osaka in Japan offers Advanced Postdoc Positions for Immunology, Cell Biology, Bioinformatics and Bioimaging. Interested applicants can contact with the relevant department/school and submit their CV directly. #OpenToWork: how laid-off US scientists are coping with shattered careers ‘I don't want this kind of life': graduate students question career options An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-01874-0'>Navigating protein landscapes with a machine-learned transferable coarse-grained model</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 10:46:31
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The most popular and universally predictive protein simulation models employ all-atom molecular dynamics, but they come at extreme computational cost. The development of a universal, computationally efficient coarse-grained (CG) model with similar prediction performance has been a long-standing challenge. By combining recent deep-learning methods with a large and diverse training set of all-atom protein simulations, we here develop a bottom–up CG force field with chemical transferability, which can be used for extrapolative molecular dynamics on new sequences not used during model parameterization. We demonstrate that the model successfully predicts metastable states of folded, unfolded and intermediate structures, the fluctuations of intrinsically disordered proteins and relative folding free energies of protein mutants, while being several orders of magnitude faster than an all-atom model. This showcases the feasibility of a universal and computationally efficient machine-learned CG model for proteins. Over the past 50 years, substantial developments in hardware, software and theory have advanced the simulation of macromolecules from proof of principle to in silico study of protein folding and dynamics1,2. Despite this success, an ongoing challenge of the field is the accurate and efficient representation of large, biologically relevant systems. These systems are inherently multiscale: while fine-grained models must be used to describe local and fast processes, the long-timescale dynamics may be better captured at a coarse-grained (CG) resolution, which is both more computationally efficient and facilitates a more direct understanding of how macroscopic observables arise from interactions between microscopic degrees of freedom. Up to now, the most successful and widely used simulation approach is molecular dynamics (MD) with all-atom resolution1,3. Atomistic MD effectively models macromolecular changes, such as protein folding or protein–ligand binding, and predicts their thermodynamic properties. However, all-atom MD comes at extreme computational costs and requires great efforts to post-process and analyse the data4,5. It is still unclear whether there is a computationally efficient CG scale that lends itself to a general and accurate simulation model. Although deep-learning methods have been wildly successful in predicting protein structure and function by reasoning over large-scale genomic and structure datasets6,7, they often do not tie into a physical level of understanding. In this Article we show that deep learning can be used to develop a universal CG protein force field capable of predicting protein structures, structure transitions, folding mechanisms, folding upon binding of an intrinsically disordered peptide, and changes of free energy upon mutation, similar to all-atom MD methods, but orders of magnitude faster. Most MD simulation studies employ atomistic force fields fitted on a combination of quantum-chemical calculations and experimental data. Modern force fields have been shown to be qualitatively accurate for processes on nanosecond to millisecond timescales and are often quantitatively consistent with experiments2,8. Recently introduced machine-learned force fields9,10,11 may capture the quantum-mechanical interactions between nuclei in the Born–Oppenheimer approximation even more accurately than conventional MD, but they also come at higher computational cost12. Ever since the first protein simulations, the community has striven to develop universal (CG) macromolecular models that are computationally more efficient and more simple to analyse. The feasibility of such models is justified by statistical mechanical descriptions of protein dynamics, such as energy landscape theory13, and results from decades of analysis of atomistic simulations4. These studies suggest that a protein's free energy landscape can be sufficiently described by a reduced number of collective variables with minimal loss of accuracy compared with atomistic MD. Some CG models have shown success in specific systems. These include structure-based models14, which rely on the known native structure of a protein to explore its free energy landscape, the Martini15 CG force field, which can effectively model intermolecular interactions including membrane structure formation and protein interactions, and CG force fields developed to model protein folding and conformational dynamics such as UNRES16 or AWSEM17. These models are limited to system-specific applications. For instance, Martini inaccurately models intramolecular protein dynamics, and UNRES and AWSEM often do not capture alternative metastable states. The main hindrance to the development of an accurate biomolecular CG model is the difficulty in efficiently modelling multi-body interaction terms, which are essential to realistically represent correct protein thermodynamics and implicit solvation effects18,19. In contrast, classical all-atom force fields model most non-bonded interactions as a sum of two-body terms. Bottom–up CG force fields20 are typically fit to match the equilibrium distribution of an all-atom model, so they could in principle reach atomistic-level accuracy and predictiveness. By leveraging recent developments in deep learning, it has become possible to machine-learn such many-body CG force fields using neural networks18,21,22,23,24,25,26,27,28,29,30. In particular, using the variational force-matching approach31,32, such force fields have been shown to accurately reproduce the all-atom distributions of CG observables for single18,21,22,25,27,33 and multiple proteins24. Despite these advancements, a transferable CG force field that could be considered as universal, quantitative, predictive and as reliable as a modern atomistic force field is still missing34. In this Article we propose a neural network-based CG model that is truly transferable in sequence space. We learn the model parameters using a bottom–up approach from atomistic simulations of a set of proteins and then use it to successfully simulate the conformational dynamics of proteins never seen at any learning stage, with low (16–40%) sequence similarities to the training or validation protein set. This CG model is orders of magnitude faster than all-atom MD simulations, predicts metastable folded, unfolded and intermediate states comparable with all-atom MD simulations, and is consistent with experimental data for larger proteins, such as relative folding free energies of protein mutants, where converged all-atom simulations are not available. These results indicate that the CG model ‘learns' to represent effective physical interactions between the CG degrees of freedom and provides strong support for the hypothesis that, using deep-learning methods, a universal CG model for realistic and predictive protein simulations at low computational cost is within reach. We generated a dataset of all-atom explicit solvent simulations of small proteins with diverse folded structures, as well as many combinations of dimers of mono- and dipeptides. Using the training data, we trained a CG force field, CGSchNet22, and conducted extensive simulations of the learned CG model on new, unseen proteins of various sizes and structures (details are presented in Fig. 1, Methods and Supplementary Section 1). Pipeline for building and testing a transferable, bottom–up, machine-learned, CG protein force field from a diverse dataset of all-atom simulations, a chosen CG resolution, and a set of basic physical prior energy terms (bonds, angles, dihedrals and purely repulsive interactions). The CG atom types z and CG coordinates x are transformed into pairwise distances dij are fed into the neural network architecture to predict the CG effective potential energy U and corresponding CG forces F. The trained neural network can subsequently be used to simulate new sequences and predict observables such as root mean square deviations (RMSD), radii of gyration (Rg) or dictionaries of secondary structure in proteins (d.s.s.p. To assess the ability of our approach for learning a transferable CG force field, we first tested how it reproduces the folding/unfolding free energy landscape of all-atom MD simulations for a set of unseen 8-peptides (Fig. 2a,b) and unseen small fast-folding proteins: the 025 mutant of chignolin (PDB 2RVD; Fig. 2e) and the villin headpiece (1YRF; Fig. None of these proteins had sequence similarity >40% to any stretch of sequence from the training or validation datasets (Table 1 and Supplementary Section 5.3). The free energy surfaces of the CG model were obtained through parallel-tempering (PT) simulations to ensure converged sampling of the equilibrium distribution (Supplementary Section 4.2). Long constant-temperature (300 K) Langevin simulations were also performed for comparison, producing consistent results and multiple folding/unfolding events for all proteins (Supplementary Sections 6.4 and 6.9). We also obtained converged folding/unfolding reference landscapes from atomistic simulations for comparison. a–f, 8-residue peptide DYGCSIHP (a), 8-residue peptide SLEAGGRG (b), chignolin (2RVD) (c), TRPcage (2JOF) (d), BBA (1FME) (e) and villin (1YRF) (f). Each subfigure shows the two-dimensional (2D) free energy (FE) surface of the CG model (CGSchNet) and reference atomistic simulations at 300 K as a function of the first two TICA coordinates39 for the two 8-peptides and as a function of the fraction of native contacts, Q, and the Cα r.m.s.d. The structures shown are sampled from the most folded-like metastable basin (or labelled metastable basins for the 8-peptides) for CGSchNet (orange) and atomistic (grey) models. CG free energy landscapes are obtained through Multistate Bennett Acceptance Ratio (MBAR)-reweighted61 parallel-tempering simulations (details are provided in Supplementary Sections 4.2 and 6.4). These peptides are mostly disordered, and their landscapes are mostly determined by the torsional dynamics contained in the prior energy term of the model, whereas the machine-learned multi-body terms have a small effect on the result. In contrast, for the four fast-folding proteins (Fig. 2c–f), the neural network must learn to predict the configurational landscape; control simulations with only the prior energy term only visit the unfolded state for these proteins (Supplementary Fig. For these systems, the CG model predicts metastable folding and unfolding transitions, and the CG folded states are predicted with a fraction of native contacts Q close to 1 and low Cα root-mean-square deviation (r.m.s.d.) values, and they are populated with structures closely resembling the correct native state (Fig. For chignolin, the model is also able to stabilize the same misfolded state with misaligned TYR1 and TYR2 residues, as found in the reference atomistic simulations (Fig. For three of the four fast-folding proteins in Fig. 2, the free energy basin containing the native state is the global minimum, whereas for BBA it is a local minimum, indicating that all proteins are able to fold/unfold correctly (also Supplementary Fig. However, the relative free energy differences between the metastable states do not exactly match the reference. The model performs better on chignolin, TRPcage and villin than on BBA, which contains both helical and anti-parallel β-sheet motifs. Previous CG models have also noted difficulty on this target system24,35. To assess the ability of our CG model to fold and maintain the folded states of larger and more complicated systems, we considered the following proteins: 54-residue engrailed homeodomain (1ENH) and 73-residue de novo designed protein alpha3D (2A3D) (Fig. The sizes of these proteins prevent atomistic simulations from sampling the folding/unfolding transitions in reasonable time, whereas the full free energy landscape can be easily explored by the CG model. Therefore, we simulate the folded state with the atomistic force field and compared these dynamics with those of our CG model, defining the lowest free energy minimum as the CG folded state. From extended configurations, the model simulates the folding of both proteins to their correct native structure (Fig. values of the folded state to the crystal structure are shown (taken from a trajectory window that remained folded for more than one million MD steps; Supplementary Section 5.2) in comparison to reference (AMBER) all-atom simulations (c,g), along with an exploration of the free energy surface as a function of the fraction of native contacts, Q, and the Cα r.m.s.d. to the native state, obtained through MBAR-reweighted61 PT simulations (a,e). A folding trajectory starting from a completely elongated structure is shown (b,f), with orange structures illustrating folding and the crystal structure shown in grey (d,h). We also compared the Cα root-mean-square fluctuations (r.m.s.f.) within the CG folded-state free energy minimum with the reference all-atom simulations. The CG model stabilizes homeodomain in a state very close to the reference native structure, with similar terminal flexibility to the all-atom simulations (Fig. 3a, bottom left) but with slightly higher fluctuations along the length of the sequence. The difference between the folded state predicted by our model and the crystal structure (mean r.m.s.d., ~0.5 nm; fraction of native contacts, ~0.75) is similar to that of the all-atom simulations from ref. 12), suggesting the difficulty in accurately predicting homeodomain's crystalline structure. Our reference simulations of alpha3D show flexibility at the termini as well as between each helical bundle (Fig. 3b), similar to the CG model. The CG model also stabilizes a metastable state of alpha3D very close to the native structure corresponding to an alternative three-helix bundle topology with a different packing of the helices (a detailed analysis is provided in Supplementary Section 6.8 and Supplementary Fig. Alpha3D is a protein designed de novo by iteratively stabilizing the selected native state topology, and precursors of the protein populate both the native state and the alternative topology similar to the one detected by our model36. These results show that the transferable machine-learned CG model can extrapolate to larger unseen proteins, stabilizing the correct native states and reproducing their associated backbone fluctuations. As an additional analysis, we also demonstrate the extrapolative performance of our CG model on stabilizing the folded states of two large proteins for which we only have experimentally determined structures as reference data, and on reproducing the conformational heterogeneity of a partially disordered protein. We compared the characteristics of the learned CG energy landscapes with the reference simulations and with three other CG force fields with similar resolutions: AWSEM17, UNRES16 and Martini15 (Supplementary Sections 4.3, 4.4 and 4.5). We note that AWSEM is parameterized to stabilize native states17, and all presented targets should be stable at this temperature. Similarly, UNRES is parameterized with conformational data for multiple systems at several temperatures at ~300 K (ref. The Martini force field is unable to stabilize the folded state of a protein without elastic restraints37,38; here, we show Martini simulation results without native restraints to compare the force field's ability to explore the conformational landscape of a protein system without prior knowledge of the protein's structure. 27, we show that Martini simulations with an elastic network only allow for small fluctuations around the native structure. Figure 4 shows the free energy landscapes of the four small fast-folding proteins from Fig. 2 as a function of the two slowest time-lagged independent component analysis (TICA) coordinates39, generated from extensive MD simulations using the reference all-atom model, CGSchNet, AWSEM, UNRES and Martini. CGSchNet explores much of the all-atom free energy landscape and it clearly resolves folded and unfolded states as well as other metastable states, whereas this behaviour is rarely observed with the other CG models. Often, AWSEM, UNRES and Martini only stabilize a single metastable state, which is either folded or unfolded. This behaviour is expected, as both the AWSEM and UNRES force fields have been primarily parameterized for stabilizing proteins with a more pronounced fold rather than whole free energy landscapes of less stable proteins. Interestingly, there is also appreciable similarity between the all-atom reference and our machine-learned CGSchNet in structural ensembles besides the folded state prediction. For chignolin, all three all-atom main states (folded, misfolded and unfolded) are visited by both CGSchNet and AWSEM, but these are clearly metastable only with CGSchNet. AWSEM explores but does not stabilize the additional states, and UNRES and Martini do not fold chignolin at all. In the landscapes of TRPcage, BBA and villin, these differences are even more striking, as CGSchnet captures several of the metastable states of the all-atom reference, in particular those with a partial fold, but these states are not explored by the other CG force fields. Nevertheless, substantial differences in the unstructured states between the reference and CGSchNet indicate that there is still room for improvement in our CG model. Results are presented for the reference atomistic simulations, for Langevin simulations of our CG model (CGSchNet), and for three different CG force fields of a comparable resolution to our CG model, AWSEM, UNRES and Martini. Each landscape includes representative structures from different metastable minima. A quantitative comparison between the folded states obtained with an all-atom model for these proteins and the CG models considered here is presented in Supplementary Section 5.4. Not only does our model better populate and stabilize the native state structure than the other CG models, but it is comparable to a reference all-atom model (Supplementary Fig. In particular, in the case of homeodomain (1ENH), the folded-like metastable state visited by the atomistic model is at a Q-value of around 0.6, lower than in our CG model. It is important to note that our model is not designed primarily for structure prediction, but rather for the exploration of free energy landscapes for protein systems through CG MD. Unsurprisingly, structures predicted by AlphaFold36 for these test proteins are very close to the corresponding crystal structures, as AlphaFold models are primarily trained to recover PDB structures. To test our CG model's extrapolative ability beyond protein folding, we consider the PUMA-MCL-1 system as a case study of concerted folding and binding of an intrinsically disordered peptide (IDP). The disordered BH3 motif of the PUMA ligand undergoes coupled folding and binding to the induced myeloid leukaemia cell differentiation protein MCL-140. Starting from an extended structure, we simulated the PUMA peptide with our transferable CG model either alone or in the presence of the folded MCL-1 protein. of the ligand to its helical (folded) state during the simulations in both cases. 5a,b, light blue) exhibit large r.m.s.d. fluctuations, indicating that the peptide remains disordered on its own. By contrast, the peptide simulated in the presence of the MCL-1 protein (orange) rapidly drops to an average r.m.s.d. value of ~2.5 Å, indicating induced folding of the peptide by the binding partner. The final simulation snapshot reported on the top right of Fig. Details about these simulations are provided in Supplementary Section 5.5. a, Time series of the r.m.s.d. of the PUMA ligand to its reference helical (folded) structure from the PDB (2ROC) when simulated alone (light blue traces) or close to the folded MCL-1 protein (orange traces). b, As in a but when the PUMA peptide is simulated close to ubiquitin (PDB 1D3Z), which is not its correct binding partner. The horizontal dashed line marks the 0.25 nm threshold for comparison. As a control, we simulated the unfolded PUMA ligand with a protein that is not known to induce its folding, ubiquitin (PDB 1D3Z). Here, although the peptide remains close to the protein, in none of the simulated trajectories does it fold into a stable helix, as indicated by the much larger deviations of the r.m.s.d. trace (in purple), and by the final simulation snapshot on the right of Fig. Together, these results indicate that, when simulated with our CG model, the PUMA peptide forms a stable helix only when in the presence of its correct binding partner, MCL-1. Although the CG model was trained partially on interacting mono/dipeptide pairs (Supplementary Section 1.2), the training data contain no protein–protein complexes such as MCL-1/PUMA. Despite this, the model learns nontrivial interactions that can correctly model the PUMA peptide both alone and in the presence of its correct binding partner. We illustrate the extrapolative power of our chemically transferable CG model in estimating folding free energy changes upon mutations, comparable to experimentally measured ΔΔG values, as described in Methods and Supplementary Section 5.6. Such mutational analysis is straightforward using our CG model, because the identity of an amino acid is solely defined by the type of the Cβ bead in our model (or the Cα bead for GLY): mutations can be performed simply by changing these bead types as illustrated in Fig. We chose ubiquitin as a test system, given its extensive and available experimental data, focusing specifically on the set of conservative mutations investigated by Went and Jackson41. We note that ubiquitin has only 18% sequence similarly with any proteins in the training/validation datasets. a, The resolution of the CG model allows for straightforward point mutations. b, Comparison of the ΔΔG value estimated from our model and the reference experimental ΔΔG at 298 K from ref. The Pearson correlation coefficient (r = 0.63) and mean absolute error (MAE = 1.25 kcal mol−1) are also reported. The black dotted line shows y = x, and the shaded region marks the interval y = x ± 1 kcal mol−1. Error bars are estimated via bootstrapping, resampling 99 times by taking batches of 10,000 independent elements of the initial folded and unfolded ensembles for each mutation. Figure 6b presents the comparison between the experimental ΔΔG values from ref. 41 and those obtained by our model as described in Supplementary Section 5.6. There is a strong correlation between our results and the experimental values; the Pearson correlation coefficient obtained with our CG model is comparable with what can be obtained with all-atom approaches42,43. This result indicates that the model has learned the general physical interactions among the residues at the CG resolution, thereby allowing useful predictions on new systems such as mutation effects. We have shown that it is possible to machine-learn a transferable, bottom–up, CG effective force field that can be used for MD simulation on proteins with little sequence overlap with the systems used for training the model. Notably, the number of training systems is very small, containing only 50 small protein domains and 1,245 dimers of mono- or dipeptides. We have demonstrated that the model samples similar conformational spaces as an explicit water all-atom model, but is orders of magnitude faster (Supplementary Table 9). With this increased efficiency, the CG model can characterize the folding/unfolding free energy landscapes of larger proteins where comprehensive atomistic MD is unaffordable. Despite this substantial improvement, our current MD code is not optimized, and the simulation throughput can be further improved by implementing speedups and optimizing parallel batch simulation. Our model also excels in more difficult tasks, such as predicting the folding upon binding of an intrinsically disordered peptide in the presence of its protein partner, despite the lack of protein complexes in the training dataset. However, intrinsically disordered proteins appear too structured and compact in our model, which should be a subject for future investigation. Finally, we have used the model to estimate changes in stability upon mutation in the protein ubiquitin, finding good correlation with the experimentally measured values. In contrast to models such as AlphaFold6, our model is not a protein structure prediction tool. Rather, it explores the complete free energy surface of the systems of interest, including but not limited to the folded protein structure. The ultimate goal of our model would be to reproduce the thermodynamics of our systems consistently with the underlying atomistic model, but there are some protein targets, such as BBA, where our model predicts the folded state as being less stable than the unfolded. Yet, even in these systems, our model predicts a metastability in the folded region of BBA's free energy landscape, whereas other CG models do not. Further improving the reliability of free energy predictions is an important future aim that will require both expanding the training dataset and further method development. The key property of our CG model is the deep graph neural network (GNN) representation of its effective energy that can capture multi-body terms without imposing restrictive functional forms. The importance of multi-body terms in CG models has been extensively discussed in the literature18,19,44,45,46. Although it is expected that neural networks can capture the important multi-body effects, it is remarkable that such a CG force field is transferable in sequence space, especially given the rather small sequence coverage of the training set. A trade-off between structural accuracy and transferability has been observed in the past for various CG protein models47,48. However, most CG effective energy functions have been previously parameterized with pre-designed functional forms, limiting the ability to model multi-body interactions. In practice, this precluded the possibility of quantitatively investigating the accuracy/transferability trade-off in protein systems. A deep neural network is the natural answer to such a problem and allows us to address this challenge. Although this is not the first instance of a bottom–up machine-learning-based protein CG model18,21,22,23,24,25,26,27,28,29,30, previously proposed versions were either explicitly parameterized for single specific systems or were not transferable to proteins substantially different from those used in training/validation datasets. The particular neural network chosen here (SchNet49) is quite simple. It consists of a series of continuous-filter convolutions and does not include an attention mechanism, nor explicit long-range interaction terms. This architectural choice was motivated by the goal of developing a ‘proof-of-concept' model, that can be trained and simulated as fast as possible while still yielding the desired results. More sophisticated architectural choices could produce better-performing CG models. In particular, the lack of long-range interactions in our model may affect the model performance on much larger and multi-protein systems, where electrostatic interactions may play an important role50,51. Multiple approaches for including long-range interactions52,53,54,55 and/or attention mechanisms56 have been recently proposed for all-atom resolutions and could be incorporated into our modelling framework in the future. To prevent our model from exploring nonphysical regions of conformational space, we employed a prior energy model (Supplementary Section 3.1); however, the model is quite sensitive to any change in this prior energy (Supplementary Section 3.3). The current functional form and parameterization of the prior model is the result of extensive testing, and this set of terms can be further optimized in future work. It is also important to note that our CG model was trained at a specific thermodynamic condition. Transferability in temperature/pressure or other additional environmental parameters is therefore not expected at this point. In particular, the temperature dependence of the CG effective energy is highly nontrivial, as it really represents a free energy with an entropic component57. An explicit dependence of the model on thermodynamic conditions could, in principle, be included in the framework58,59. However, in practice, its training would require the curation of a substantially larger dataset encompassing multiple simulations at multiple thermodynamic conditions, which would probably require even more large-scale computational resources than those used in this work. The results presented here were obtained with a model that, although aggressively coarse-grained with respect to an explicit water atomistic model, still retains the full backbone heavy atoms and an additional atom per side chain (excluding GLY). We have not yet investigated alternative resolutions for building a transferable model, and we expect transferability to be strongly tied to the chosen resolution. Although different methods have been proposed for the simultaneous optimization of CG effective energy and CG mapping60, it remains unclear if and which additional resolutions allow for the optimal design of a transferable and quantitatively accurate model. We believe that the results presented here open the way to a systematic investigation of this point. We generated a dataset of all-atom explicit solvent simulations of 50 CATH domains62, representing small proteins with diverse folded structures, as well as ~1,200 dimers of mono- and dipeptides. We stored all instantaneous forces on the protein atoms, performed basic force aggregation on a CG backbone representation of the proteins27, and trained a CG force field, CGSchNet22, which combines a deep GNN with physically motivated prior terms. We then conducted a series of extensive Langevin and PT simulations of the learned CG model on new, unseen proteins of various sizes and structures to demonstrate its capabilities and limitations. Wherever feasible, we also performed extensive all-atom MD simulations for the test systems and analysed them with Markov state modelling4,5 for comparison. Our model was built using the deep-learning Python packages PyTorch63 and PyTorch Geometric64. Building on previous efforts22,27,33, we chose to model the optimizable term of our CG effective energy with the GNN architecture CGSchNet, which is based on a previous architecture, SchNet49. See Supplementary Section 2 for a detailed description of network architecture, hyperparameter choices and training routines. The ability to directly learn species-dependent interactions and CG bead-wise features from data represents the primary advantage of using a convolutional GNN such as CGSchNet in this pursuit. More recent GNN architectures65,66 may be used as an alternative. However, we note that newer architectures can require more computational resources and more extensive hyperparameter searches, thereby creating substantial training barriers given the large number of MD simulation frames and the system sizes used in training. We designed our CG model within the framework of variational force-matching31,32. In practice, we optimize the parameters {θ} of a network representing the effective energy \({\tilde{U}}_{\rm{CG}}({\bf{R}};\{\theta \})\), where R are the CG coordinates, by minimizing a loss function in the form Here, N is the number of CG atoms in the system. In equation (1), \({\tilde{{\bf{F}}}}_{{\rm{CG}},\,\Delta }({\bf{R}};\{\theta \})\) are the forces associated with the CG effective energy, \({\tilde{{\bf{F}}}}_{\rm{CG}}({\bf{R}};\{\theta \})=-{\nabla }_{{\bf{R}}}{\tilde{U}}_{\rm{CG}}({\bf{R}};\{\theta \})\), after subtraction of the ‘prior forces': The atomistic force is similarly modified. The definition of a prior energy is discussed in the next section and it has been shown to play an important role in constructing stable and accurate neural network-based CG models by enforcing asymptotic physical behaviour in regions of phase space not covered adequately by training/validation datasets obtained through all-atom MD21,22,33,34. In equation (1), the operator \({{\mathcal{M}}}_{F}\) projects the atomistic forces fAA(r), as a function of the atomistic coordinates r, in CG space. We have shown in previous work that a careful choice of \({{\mathcal{M}}}_{F}\) is crucial to the optimization of the CG model27. In this Article, \({{\mathcal{M}}}_{F}\) is defined for each CG site as the sum of forces on the preserved atom and neighbouring hydrogen atoms connected via constrained bonds27. A good choice of the prior energy model should be connected to the resolution chosen to define the CG model34. Previous non-transferable CG studies22,25,27 have utilized a resolution that retains only the Cα atoms for each amino acid. However, when considering 20 naturally occurring amino acids, the type enumeration for common local energy terms, such as four-body dihedral interactions, becomes very large. Efforts in the past48 have attempted to mitigate such scaling, but this can lead to potentially limiting or overly biasing expressions for the associated prior energies. For this work we chose to retain the following five atoms for each residue: backbone N, Cα, C, O and side-chain Cβ. We label different atoms with an integer atom type, with the Cβ atom having a residue-dependent atom type number. In the case of GLY residues, which do not contain a Cβ, we retain only four atoms—N, Cα, C and O—and assign the residue-dependent atom type number to the Cα atom. 3 provides a graphical description of the CG resolution and atom type labelling. This five-bead-per-residue CG mapping is not unprecedented—the successful AWSEM17 CG force field, which retains the Cα, the Cβ and the O atoms (as well as virtual sites for N and C atoms), utilizes a comparable resolution. This choice of CG resolution allows for a direct interpretation of secondary structures and leads to intuitive prior energy choices (for example, physical bond/angle terms, physical dihedral angles and so on). A description of the terms in the prior force field is provided in Supplementary Section 3.1. It is important to stress that if the prior energy is used alone (without the trainable neural network energy term \({\tilde{U}}_{\rm{CG}}({\bf{R}};\{\theta \})\)), it is completely incapable of stabilizing any secondary or tertiary protein structures (Supplementary Fig. 7 presents the results from control, prior-only simulations). The function of the prior energy is only to prevent the model from visiting configurationally nonphysical regions (for example, configurations involving overlapping atoms), with little to no additional bias on the configurational landscape. To illustrate the relative importance of each prior on model stability/ability to access unphysical configurations, we include a prior ablation study, where the effect of removing each prior subinteraction, one by one, is investigated on a chignolin-specific model at the same five-bead-per-residue resolution (Supplementary Fig. Three strategies were used to create the training dataset. First, to capture sequence and secondary/tertiary structure diversity in proteins, we constructed a dataset of all-atom simulations of 50 protein domains in their native state from the CATH62 database (Supplementary Section 1.1 presents the domain selection procedure). Each simulation represents 100,000 frames of all-atom MD data in which the forces and positions of the solute are saved. In addition to this dataset of folded CATH simulations, we constructed a second dataset wherein ~1,200 mono/dipeptide dimer systems were simulated using umbrella sampling with dimer centre-of-mass distances as a reaction coordinate, and each system consists of 27,000 frames. Although this dataset does not contain direct secondary/tertiary structure information, it contains valuable asymptotic force information for atoms that are brought very close together through the nature of the enhanced sampling strategy. The necessity for both the CATH and dimer datasets was demonstrated by an ablation study in which we systematically remove both entire datasets and selected samples (Supplementary Fig. Finally, additional frames were constructed from the previously described CATH and dimers datasets by taking every 50th frame of each simulation and additively combining bead positions with a Gaussian noise of mean 0 and standard deviation 0.5 Å (Supplementary Section 6.6 provides details of the hyperparameter selection). These distorted ‘decoy' frames were combined with a zero delta-force label and used as additional training data, and are designed to prevent uncontrolled neural network extrapolation on distorted high-energy configurations that may arise transiently during CG simulation. Due to the induced distortion, the prior alone predicts a high baseline energy on the corresponding areas of phase space. Effectively, the decoy training data helps to ensure that the network does not attempt to predict strong forces in configurations that should be dominated by the prior terms. A comprehensive discussion on the training and validation datasets that are used to parameterize the model is provided in Supplementary Section 1. From the CG model, we can estimate ΔΔG values by treating the effect of a single point mutation as a perturbation to the wild-type energy67,68. Under the assumption that the mutation does not significantly perturb the density of states, the effect of the mutation on protein stability can be estimated from perturbation theory67,68 (a detailed analysis is provided in Supplementary Section 5.6). All training data, simulation data, and trained models are available at https://doi.org/10.5281/zenodo.15465782 (ref. Source data are provided with this paper. Code for the generation of the training datasets is available at https://doi.org/10.5281/zenodo.15465782 (ref. The scripts for model training and running simulations are available at https://doi.org/10.5281/zenodo.15482457 (ref. Model training and simulation was done using the mlcg package (https://github.com/ClementiGroup/mlcg). Additional tools for the prior pipeline can be found in the mlcg-tk package (https://github.com/ClementiGroup/mlcg-tk). Finally, analysis tools can be found in the Proteka package (https://github.com/ClementiGroup/proteka). Lindorff-Larsen, K., Piana, S., Dror, R. O. & Shaw, D. E. How fast-folding proteins fold. Plattner, N., Doerr, S., De Fabritiis, G. & Noé, F. Complete protein–protein association kinetics in atomic detail revealed by molecular dynamics simulations and Markov modelling. Voelz, V. A., Bowman, G. R., Beauchamp, K. & Pande, V. S. Molecular simulation of ab initio protein folding for a millisecond folder NTL9(1–39). Noé, F. & Clementi, C. Collective variables for the study of long-time kinetics from molecular trajectories: theory and methods. Husic, B. E. & Pande, V. S. Markov state models: from an art to a science. Accurate structure prediction of biomolecular interactions with AlphaFold 3. Lewis, S. et al. Scalable emulation of protein equilibrium ensembles with generative deep learning. Preprint at bioRxiv https://www.biorxiv.org/content/10.1101/2024.12.05.626885v1 (2024). Casasnovas, R., Limongelli, V., Tiwary, P., Carloni, P. & Parrinello, M. Unbinding kinetics of a p38 map kinase type II inhibitor from metadynamics simulations. Unke, O. T. et al. Biomolecular dynamics with machine-learned quantum-mechanical force fields trained on diverse chemical fragments. Kozinsky, B., Musaelian, A., Johansson, A. & Batzner, S. Scaling the leading accuracy of deep equivariant models to biomolecular simulations of realistic size. International Conference for High Performance Computing, Networking, Storage and Analysis article no. & Roitberg, A. E. ANI-1: an extensible neural network potential with DFT accuracy at force field computational cost. Towards chemical accuracy for alchemical free energy calculations with hybrid physics-based machine learning/molecular mechanics potentials. Preprint at bioRxiv https://doi.org/10.1101/2020.07.29.227959 (2020). & Wolynes, P. G. Theory of protein folding: the energy landscape perspective. Clementi, C., Nymeyer, H. & Onuchic, J. N. Topological and energetic factors: what determines the structural details of the transition state ensemble and ‘en-route' intermediates for protein folding? An investigation for small globular proteins. Souza, P. C. T. et al. Martini 3: a general purpose force field for coarse-grained molecular dynamics. Liwo, A. et al. A general method for the derivation of the functional forms of the effective energy terms in coarse-grained energy functions of polymers. Determination of scale-consistent backbone-local and correlation potentials in the UNRES force field and force-field calibration and validation. Davtyan, A. et al. AWSEM-MD: protein structure prediction using coarse-grained physical potentials and bioinformatically based local structure biasing. Wang, J. et al. Multi-body effects in a coarse-grained protein force field. & Clementi, C. Multibody terms in protein coarse-grained models: a top-down perspective. Jin, J., Pak, A. J., Durumeric, A. E. P., Loose, T. D. & Voth, G. A. Bottom-up coarse-graining: principles and perspectives. Wang, J. et al. Machine learning of coarse-grained molecular dynamics force fields. Husic, B. E. et al. Coarse graining molecular dynamics with graph neural networks. & Zhang, B. Contrastive learning of coarse-grained force fields. Majewski, M. et al. Machine learning coarse-grained potentials of protein thermodynamics. Köhler, J., Chen, Y., Krämer, A., Clementi, C. & Noé, F. Flow-matching: efficient coarse-graining of molecular dynamics without forces. & Rotskoff, G. M. Ensuring thermodynamic consistency with invertible coarse-graining. Krämer, A. et al. Statistically optimal force aggregation for coarse-graining molecular dynamics. Wellawatte, G. P., Hocky, G. M. & White, A. D. Neural potentials of proteins extrapolate beyond training data. Transferable implicit solvation via contrastive learning of graph neural networks. Marloes Arts, V. G. S. et al. Two for one: diffusion models and force fields for coarse-grained molecular dynamics. A multiscale coarse-graining method for biomolecular systems. A rigorous bridge between atomistic and coarse-grained models. Chen, Y. et al. Machine learning implicit solvation for molecular dynamics. Durumeric, A. E. P. et al. Machine learned coarse-grained protein force-fields: are we there yet? Liwo, A., Czaplewski, C., Pillardy, J. & Scheraga, H. A. Cumulant-based expressions for the multibody terms for the correlation between local and electrostatic interactions in the united-residue force field. Bryson, J. W., Desjarlais, J. R., Handel, T. M. & Degrado, W. F. From coiled coils to small globular proteins: design of a native-like three-helix bundle. Combining an elastic network with a coarse-grained molecular force field: structure, dynamics and intermolecular recognition. B., Cieplak, M. & Theodorakis, P. E. Combining the MARTINI and structure-based coarse-grained approaches for the molecular dynamics studies of conformational transitions in proteins. Pérez-Hernández, G., Paul, F., Giorgino, T., De Fabritiis, G. & Noé, F. Identification of slow molecular order parameters for Markov model construction. Interplay between partner and ligand facilitates the folding and binding of an intrinsically disordered protein. Went, H. M. & Jackson, S. E. Ubiquitin folds through a highly polarized transition state. Predicting folding free energy changes upon single point mutations. Gapsys, V., Michielssens, S., Seeliger, D. & Groot, B. L. Accurate and rigorous prediction of the changes in protein free energies in a large scale mutation scan. Vendruscolo, M. & Domany, E. Pairwise contact potentials are unsuitable for protein folding. Ejtehadi, M., Avall, S. & Plotkin, S. Three-body interactions improve the prediction of rate and mechanism in protein folding models. Scherer, C. & Andrienko, D. Understanding three-body contributions to coarse-grained force fields. Kar, P. & Feig, M. Recent advances in transferable coarse-grained modeling of proteins. Hills Jr, R. D., Lu, L. & Voth, G. A. Multiscale coarse-graining of the protein energy landscape. Schütt, K. T., Sauceda, H. E., Kindermans, P.-J., Tkatchenko, A. & Müller, K.-R. SchNet: a deep learning architecture for molecules and materials. On the role of electrostatic interactions in the design of protein–protein interfaces. Zhang, Z., Witham, S. & Alexov, E. On the role of electrostatics in protein–protein interactions. A fourth-generation high-dimensional neural network potential with accurate electrostatics including non-local charge transfer. Unke, O. T. et al. SpookyNet: learning force fields with electronic degrees of freedom and nonlocal effects. Kosmala, A., Gasteiger, J., Gao, N. & Günnemann, S. Ewald-based long-range message passing for molecular graphs. International Conference on Machine Learning (PMLR, 2023) Extending the RANGE of graph neural networks: relaying attention nodes for global encoding. Thölke, P. & Fabritiis, G. D. Equivariant transformers for neural network based molecular potentials. In 10th International Conference on Learning Representations (Curran Associates, 2022). & Noid, W. Energetic and entropic considerations for coarse-graining. Krishna, V., Noid, W. G. & Voth, G. A. Transferring coarse-grained potentials between temperatures. Pretti, E. & Shell, M. S. A microcanonical approach to temperature-transferable coarse-grained models using the relative entropy. Wang, W. & Gómez-Bombarelli, R. Coarse-graining auto-encoders for molecular dynamics. Shirts, M. R. & Chodera, J. D. Statistically optimal analysis of samples from multiple equilibrium states. Sillitoe, I. et al. CATH: comprehensive structural and functional annotations for genome sequences. Paszke, A. et al. PyTorch: an imperative style, high-performance deep learning library. Neural Information Processing Systems Vol. 32 (eds Wallach, H. et al.) article no. Fey, M. & Lenssen, J. E. Fast graph representation learning with PyTorch Geometric. ICLR Workshop on Representation Learning on Graphs and Manifolds (Curran Associates, 2019). Batzner, S. et al. E(3)-equivariant graph neural networks for data-efficient and accurate interatomic potentials. Batatia, I., Kovacs, D. P., Simm, G., Ortner, C. & Csányi, G. MACE: higher order equivariant message passing neural networks for fast and accurate force fields. Advances in Neural Information Processing Systems Vol. 35 (eds Koyejo, S. et al.) 11423–11436 (Curran Associates, 2022). Matysiak, S. & Clementi, C. Optimal combination of theory and experiment for the characterization of the protein folding landscape of S6: how far can a minimalist model go? Matysiak, S. & Clementi, C. Minimalist protein model as a diagnostic tool for misfolding and aggregation. Charron, N., Bonneau, K., Pasos-Trejo, A. & Guljas, A. Navigating protein landscapes with a machine-learned transferable coarse-grained model (data and codes). We thank all members of the Clementi and Noé groups for their help in different phases of this work. We gratefully acknowledge funding from the European Commission (grant no. ERC CoG 772230 ‘ScaleCell'), the International Max Planck Research School for Biology and Computation (IMPRS–BAC), the Bundesministerium für Bildung und Forschung BMBF (Berlin Institute for Learning and Data, BIFOLD, and project FAIME 01IS24076), the Berlin Mathematics Center MATH+ (AA1-6, EF1-2) and the Deutsche Forschungsgemeinschaft DFG (NO 825/2, NO 825/3, NO 825/4; SFB/TRR 186, Project A12; SFB 1114, Projects B03, B08 and A04; SFB 1078, Project C7; and RTG 2433, Projects Q05 and Q04), the National Science Foundation (CHE-1900374 and PHY-2019745) and the Einstein Foundation Berlin (Project 0420815101). We gratefully acknowledge the computing time provided on the supercomputer Lise at NHR@ZIB as part of the NHR infrastructure, and on the supercomputer JUWELS operated by the Jülich Supercomputing Centre. We thank volunteers at GPUGRID.net for contributing computational resources and Acellera for funding. Open access funding provided by Freie Universität Berlin. These authors contributed equally: Nicholas E. Charron, Klara Bonneau, Aldo S. Pasos-Trejo, Andrea Guljas. Nicholas E. Charron, Klara Bonneau, Aldo S. Pasos-Trejo, Andrea Guljas, Félix Musil, Jacopo Venturin, Daria Gusew, Iryna Zaporozhets, Clark Templeton, Frank Noé & Cecilia Clementi Department of Physics and Astronomy, Rice University, Houston, TX, USA Nicholas E. Charron & Cecilia Clementi Center for Theoretical Biological Physics, Rice University, Houston, TX, USA Nicholas E. Charron, Iryna Zaporozhets & Cecilia Clementi Department of Mathematics and Computer Science, Freie Universität Berlin, Berlin, Germany Yaoyi Chen, Andreas Krämer, Atharva Kelkar, Aleksander E. P. Durumeric & Frank Noé Department of Chemistry, Rice University, Houston, TX, USA Iryna Zaporozhets, Frank Noé & Cecilia Clementi Department of Computer Science and Engineering, Chalmers University of Technology and University of Gothenburg, Gothenburg, Sweden Computational Science Laboratory, Universitat Pompeu Fabra, Barcelona, Spain Adrià Pérez, Maciej Majewski & Gianni De Fabritiis Acellera Labs, Barcelona, Spain Adrià Pérez, Maciej Majewski & Gianni De Fabritiis Lewis Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA Department of Neuroscience, Baylor College of Medicine, Houston, TX, USA Department of Electrical and Computer Engineering, Rice University, Houston, TX, USA Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain AI4Science, Microsoft Research, Berlin, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to the development of the models and data analysis. wrote and tested the software. generated part of the reference data. and A. Patel supervised the project. Correspondence to Gianni De Fabritiis, Frank Noé or Cecilia Clementi. The authors declare no competing interests. Nature Chemistry thanks Pratyush Tiwary and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. 1–28, Tables 1–9, Sections 1–6 and Discussion. .csv files with free energy landscape data. .csv files with RMSF and free energy landscape data. .csv files with free energy landscape data. .csv files with RMSD data. .csv files with ddG data. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Charron, N.E., Bonneau, K., Pasos-Trejo, A.S. et al. Navigating protein landscapes with a machine-learned transferable coarse-grained model. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41559-025-02794-5'>Dental microwear texture analysis reveals behavioural, ecological and habitat signals in Late Jurassic sauropod dinosaur faunas</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 10:32:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Ecology & Evolution (2025)Cite this article Most faunas from the Mesozoic era were dominated by sauropod dinosaurs, the largest terrestrial animals to ever exist. These megaherbivores were remarkably diverse and widely distributed. Here we study three Late Jurassic faunas from the USA, Portugal and Tanzania, each approximately 150 million years old, which are known for their extreme sauropod diversity. Whereas general taxonomic composition was similar in these three faunas, the major clades differed in relative abundance. Moreover, their depositional strata record distinct climatic regimes. Using dental microwear texture analysis, we investigated the impact of these climate regimes and the resulting food availability on the different sauropod taxa. Wear patterns in camarasaurid macronarians show minimal variation across different climate regimes, supporting previous studies suggesting that these animals migrated to follow their preferred climate niche and food source. North American camarasaurids show similar wear patterns to those of Portuguese turiasaurs, another broad-crowned taxon, which did not exist in the Jurassic of North America. By contrast, where camarasaurids and turiasaurs co-occurred in Portugal, their microwear patterns are distinct, suggesting niche differentiation to avoid ecological competition between these two clades. Flagellicaudatan diplodocoids display highly variable wear patterns, indicating limited migration (and therefore seasonal variation in diet), which aligns with observed biogeography patterns in the USA. Early-branching titanosauriforms show highly distinct wear patterns between different climate regimes, which can probably be attributed to different abrasive loads in the respective habitats. Our results demonstrate that dental microwear texture analysis not only records dietary preferences but also reveals behaviour such as competition and migration related to dietary niches in past ecosystems. Late Jurassic ecosystems across the planet were dominated by sauropod dinosaurs in terms of size and body mass contribution1. The three best-known and represented Late Jurassic faunas are from the USA, Tanzania and Portugal. These sauropod faunas had very similar taxonomic compositions, with brachiosaurid macronarians and diplodocoids occurring in all three. The North American and Portuguese faunas furthermore shared the presence of camarasaurid macronarians, whereas turiasaurs and early somphospondylans occur both in Portugal and Tanzania. The Tanzanian fauna further included non-neosauropod mamenchisaurids2,3,4,5,6,7, which are otherwise primarily known from the Jurassic of Asia8,9,10,11. Although generally similar, within-clade species diversity and the abundance of individuals from the distinct taxa are different between the three faunas. Whereas camarasaurid macronarians and diplodocid diplodocoids dominated in the USA12,13,14,15, brachiosaurid titanosauriforms and dicraeosaurid diplodocoids predominated in Tanzania5,16 and turiasaurs in Portugal17,18. The reasons for these unequal distributions of taxon abundance remain poorly understood. As the largest terrestrial megaherbivores that ever roamed the Earth, sauropods must have greatly depended on plant productivity and availability. Similarly to sauropods, major plant groups were distributed globally during the Late Jurassic, with conifers forming the canopy together with ginkgoes. Tree ferns reached intermediate heights, whereas other ferns, seed ferns, cycads and horsetails usually composed the understorey. The only notable difference between the floras of the three sauropod-bearing formations was the absence of the conifer clade Pinaceae and of horsetails in Tendaguru, where also cycads were a comparatively minor component of the plant cover19,20,21. Moreover, the relative abundance of the different plant clades differed between the three biomes21, which was correlated to distinct climate regimes. Palaeoclimatic reconstructions indicate different climatic conditions in these three regions during the Late Jurassic20,22,23,24,25. The western USA was reconstructed as semi-arid to arid, with a mean annual temperature ranging between 12 °C and 30 °C, depending on latitude22,25 and probably strong seasonality24. Mean annual precipitation was found to be higher in Portugal and Tanzania compared with the USA, suggesting a more humid climate than in the USA24,25. In Portugal, mean annual temperature ranged between 18 °C and 24 °C, whereas it was considerably warmer, with 24–30 °C, in Tanzania25. Seasonality was probably strong in Tanzania as well, with wet winters22,26 and dry summers20, corresponding to a monsoon-type climate25. We hypothesize that these different climate regimes probably controlled forage availability, which in turn shaped distribution and abundance of different sauropod taxa adapted for foraging on specific plant taxa. Furthermore, we propose that large-bodied herbivores in such habitats affected by seasonality either adopted a generalist feeding strategy, using diverse dietary resources in the same place year-round, or depended upon seasonal migration if they had a narrower dietary niche. Strong niche partitioning between major sauropod groups has been suggested on the basis of their distinct skull shapes and tooth morphologies, and the posture of the neck and forelimbs7,18,27,28,29,30,31,32,33,34,35,36,37,38,39,40. However, morphology can provide only general information about possible feeding adaptations and is not direct evidence whether a certain dietary resource was actually exploited. A promising avenue to assess niche partitioning in sauropods and other archosaurs is the study of dental microwear as a dietary proxy35,41,42,43,44,45,46. We here use three-dimensional dental microwear texture analysis (DMTA), a semi-automated quantitative approach to evaluate microscopic surface wear of enamel wear facets47,48, as a means to test whether distinct sauropod taxa occupied the same niche in three different geographical areas (Fig. 1) irrespective of climate, and how these food preferences and availability may have shaped sauropod distribution during the Late Jurassic period. a, Palaeogeographic map showing the approximate arrangements of continents during the Late Jurassic. Yellow star, North America (Morrison Formation); blue star, Portugal (Lourinhã Formation); and red star, Tanzania (Tendaguru Formation). Silhouettes represent major taxonomic groups of sauropods present in the sampled locations. Taxa that were not available for inclusion into the current study from the respective locations are shown in light grey, those included in the analysis are shown in colour. b, Tooth morphologies for selected clades and phylogenetic tree. All clades except possibly Somphospondyli and Mamenchisauridae could be sampled for the current study. The indeterminate sauropods and macronarians sampled in this study could not be attributed to a less-inclusive clade. Tooth morphologies are shown for well-represented clades (from left to right, Turiasauria, Titanosauriformes, Camarasauridae, Flagellicaudata). Teeth are shown in lingual and buccal view. Approximate sampling areas for occlusal and buccal surfaces are shown in camarasaurid and turiasaur teeth, respectively (grey arrows). Credits: a, Maps are taken from The Paleobiology Database Navigator (https://paleobiodb.org/navigator/) under a CC BY 4.0 license, which uses GPlates as a data source for the maps. GPlates are shared under the GNU software general public license, v.2 (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html). a,b, Silhouettes are from Phylopic (https://phylopic.org). Xinjiangtitan shanshanesis (Mamenchisauridae), created by Jagged Fang Designs under a CC0 1.0 license; Haplocanthosaurus priscus, created by T. M. Keesey under a CC0 1.0 license; Amanzia greppini (Turiasauria), created by T. Dixon under a CC BY 4.0 license; Euhelopus zdanskyi (Somphospondyli), created by DiBgd and modified by T. M. Keesey under CC BY-SA 3.0 license; Giraffatitan brancai (Brachiosauridae), created by S. Hartman under a CC BY 3.0 license; Diplodocus carnegii (Flagellicaudata), created by S. Hartman under a CC BY 3.0 license; Camarasaurus supremus (Camarasauridae), created by M. Wedel under a CC BY 3.0 license. Tooth shapes are modified from ref. We analysed measurements taken from the enamel of buccal and occlusal surfaces from the sauropod dental microwear texture (DMT) dataset acquired by ref. 49, which were categorized as either scan quality 1 or 2 (see Methods and refs. 50 found that measurements from casted tooth surfaces in this dataset yielded inaccurate results, we only exported measurements taken from original teeth and from moulds. Furthermore, as we aimed to compare the impact of diverging climatic regimes on the well-studied faunas of the Morrison (USA), Tendaguru (Tanzania) and Lourinhã (Portugal) Formations, we excluded all measurements of teeth from other geographical areas. The final set of 322 measurements belongs to 39 sauropod individuals, 17 of which were recovered from the Lourinhã Formation (Portugal; all represented by isolated teeth), 13 from the Morrison Formation (USA; between 1 and 8 teeth per specimen) and 9 from the Tendaguru Formation (Tanzania; between 1 and 8 teeth per specimen) (Supplementary Table 1). They include teeth from a range of tooth positions and wear stages, but these differences are expected to have minimal impact on the final results because (1) there are relatively few morphological differences along the tooth rows of sauropods7,18,32,51,52 and (2) based on studies on mammalian dental microwear, recorded wear features in the tooth enamel reflect the ingesta of the past few weeks to months irrespective of how worn they already are53,54,55. Despite our sampling efforts, the three geological formations and four clades could not be equally represented. Owing to the difference in sample size and distribution of clades, our study sample allows for limited direct comparisons of clades in different faunas and climate regimes (Fig. Turiasaur samples were restricted to Portugal. Camarasaurids and flagellicaudatans were sampled from Portugal and the USA. Titanosauriforms could be compared between Portugal and Tanzania. For representative images of typical teeth and analysed microwear patterns for each taxon and geological formation, see Extended Data Fig. First, we tested whether buccal and occlusal surfaces could be compared directly. Most height (matf, metf, Sa, Sdc, Sk, Sp, Sq, Sv, Sz; see Supplementary Table 6 for parameter descriptions) and volume (Vm, Vmc, Vv, Vvc, Vvv) parameter values were larger for occlusal surfaces than for buccal surfaces in Titanosauriformes, Camarasauridae, Turiasauria and an unidentified macronarian (Fig. 2 and Supplementary Table 1). Complexity (Asfc, Sdr) and mean slope (Sdq) parameters showed the same pattern. Mean density of furrows (medf) was comparable for occlusal and buccal surfaces in Camarasauridae and Turiasauria, but it was lower for occlusal surfaces than buccal surfaces in Titanosauriformes. For Flagellicaudata, only four scans of one occlusal surface were of good enough quality for the analyses, all other surfaces were buccal. Height and volume parameters were very similar between buccal and occlusal surfaces, whereas complexity (Asfc, Sdr) and density of furrows (medf) were higher for buccal surfaces than for occlusal surfaces in Flagellicaudata (opposite to macronarians and turiasaurs). Given the ambiguous results, we performed the subsequent analyses both with separate datasets for occlusal and buccal surfaces, as well as with the combined dataset. Owing to the distinct differences between buccal and occlusal surfaces for all parameters and in most clades, we separate surfaces in analyses concerning diet reconstruction. Note that Flagellicaudata from Portugal, unidentified Macronaria from the USA, and unidentified Sauropoda and Macronaria from Tanzania are missing as they are only represented by either buccal or occlusal surfaces. RMS, root-mean square. Boxplots are depicted with a thick horizontal bar that represents the median; the box encloses the first (25%) and third (75%) quartiles; the whiskers extend to the full interquartile range. Amanzia greppini (Turiasauria), created by T. Dixon under a CC BY 4.0 license; Giraffatitan brancai (Titanosauriformes), created by S. Hartman under a CC BY 3.0 license; Diplodocus carnegii (Flagellicaudata), created by S. Hartman under a CC BY 3.0 license; Camarasaurus supremus (Camarasauridae), created by M. Wedel under a CC BY 3.0 license. Of all major clades, flagellicaudatans show the highest variability across all geographical regions and in nearly all parameter values (Supplementary Fig. Sampled titanosauriforms from Tanzania (which are probably all brachiosaurids; E.T., personal observation, 2022) are highly variable as well, whereas camarasaurids and turiasaurs occupy very restricted areas in the principal component analysis (PCA) plots (Fig. In a canonical variate analysis (CVA), taxa could mostly be separated with little overlap and confident assignment to the correct clade (Fig. 4b and Supplementary Table 2). However, Portuguese titanosauriforms and turiasaurs strongly overlap with North American camarasaurids, and to a lesser extent with flagellicaudatans from Portugal (Fig. These four groups are all distinct from Portuguese camarasaurids, North American flagellicaudatans and brachiosaurids from Tanzania (Fig. a, Buccal surfaces only. b, Occlusal surfaces only. c, Buccal and occlusal surfaces combined. d, As in c but separated according to clade. a, Assignment according to clade and fauna. b, Assignment according to clade only. Note that Camarasauridae from the Morrison Formation are greatly overlapping with Turiasauria from Lourinhã and with Titanosauriformes from Lourinhã. Unidentified Macronarians and Sauropoda indet. Comparing taxa measured from two different geographical regions, flagellicaudatans from Portugal had distinctly lower values related to wear surface complexity, slope and density of furrows, than North American flagellicaudatans. Titanosauriforms from Tanzania tend to show larger height and volume parameters, as well as higher complexity values, and significantly larger slope and density than those from Portugal (Fig. 5 and Supplementary Table 4). Camarasaurids from Portugal and the USA are similar in parameter values, except for height and volume parameter values, which are higher in Portuguese camarasaurids compared with those from the USA (Fig. 2 and Supplementary Table 4). Different regions (and thus climate regimes) also seem to have had a distinct impact on wear patterns of their megaherbivore inhabitants. When comparing only buccal surfaces, visible, non-significant differences exist between flagellicaudatans from the USA and Portugal and between titanosauriforms from Tanzania and Portugal. Camarasaurids from the USA and Portugal are very similar in most DMT parameters (Fig. 1 and Supplementary Table 4). When compared with brachiosaurids from Tanzania, significant differences are found with camarasaurids from both Portugal and the USA (Fig. 5) as well as with flagellicaudatans from Portugal (Supplementary Table 4). PC 1 represents surface roughness in terms of height and volume, whereas PC 2 reflects structure of the surface in terms of wear pattern complexity and density of wear marks. Boxplots are depicted with a thick horizontal bar that represents the median; the box encloses the first (25%) and third (75%) quartiles; the whiskers extend to the full interquartile range. Significance according to Dunn's pairwise comparison with Bonferroni adjustment (Supplementary Table 4). Number of surfaces per taxon with number of specimens (in case buccal and occlusal surface of the same specimen were included) given in parenthesis: Turiasauria, n = 5 (7); Titanosauriformes, n = 9 (15); Camarasauridae, n = 12 (20); Flagellicaudata, n = 12 (13); Macronaria indet., n = 3 (4); and Sauropoda indet., n = 1 (1). Credits: Silhouettes are from Phylopic (https://phylopic.org). Xinjiangtitan shanshanesis (Sauropoda indet. ), created by Jagged Fang Designs under a CC0 1.0 license; Amanzia greppini (Turiasauria), created by T. Dixon under a CC BY 4.0 license; Diplodocus carnegii (Flagellicaudata), created by S. Hartman under a CC BY 3.0 license; Giraffatitan brancai (Titanosauriformes), created by S. Hartman under a CC BY 3.0 license; Camarasaurus supremus (Camarasauridae), created by M. Wedel under a CC BY 3.0 license. In Portugal, no differences are seen among taxa regarding complexity parameters (Asfc, Sdr). These parameters, however, are clearly distinct between flagellicaudatans (larger and more variable) and camarasaurids (lower and more consistent) in the USA. Slope (Sdq) and density of furrows (medf) followed a similar pattern to complexity parameters, with little differences between all sauropod clades from Portugal, whereas in the USA, flagellicaudatans showed distinctly larger values than camarasaurids. Only limited comparisons can be made within the Tanzanian sample, because the vast majority of sampled teeth were from brachiosaurid titanosauriforms. Only an indeterminate sauropod tooth and an indeterminate macronarian survived the dataset cleansing for the final analyses. Whereas the absence of enamel wrinkling on the indeterminate sauropod tooth suggests a basal, non-eusauropod affinity (which is surprising because such early forms are currently not represented by other skeletal material in the Late Jurassic biomes discussed above)5,7,14,18,56, the indeterminate macronarian tooth could stem from a brachiosaurid titanosauriform. The single occlusal surface from the indeterminate sauropod and the single buccal surface of the macronarian fell within the range of brachiosaurid buccal surfaces, so all specimens from Tanzania showed similar values within the fauna. Overall, there seems to be a strong location-specific tendency consistent across taxa (Fig. 6 and Supplementary Table 5). Pooled specimens from Tanzania are characterized by significantly larger complexity (Asfc, Sdr) and density of furrows (medf) when compared with pooled specimens from both Portugal and the USA. This location-specific separation also becomes evident in the PCAs, which show a clear separation of the Tanzanian fauna from the Portuguese fauna. The assemblage from the USA partially overlaps with Tanzania when buccal or buccal and occlusal surfaces are considered but is completely separated when only occlusal surfaces are considered. USA and Portugal greatly overlap, regardless of whether occlusal, buccal or all surfaces are compared. Boxplots are depicted with a thick horizontal bar that represents the median; the box encloses the first (25%) and third (75%) quartiles; the whiskers extend to the full interquartile range. Significance according to Dunn's pairwise comparison with Bonferroni adjustment for multiple comparisons (Supplementary Table 5). Number of surfaces per taxon with number of specimens (in case buccal and occlusal surface of the same specimen were included) given in parenthesis: Turiasauria, n = 5 (7); Titanosauriformes, n = 9 (15); Camarasauridae, n = 12 (20); Flagellicaudata, n = 12 (13); Macronaria indet. Credits: Silhouettes are from Phylopic (https://phylopic.org). Xinjiangtitan shanshanesis (Mamenchisauridae), created by Jagged Fang Designs under a CC0 1.0 license; Haplocanthosaurus priscus, created by T. M. Keesey under a CC0 1.0 license; Amanzia greppini (Turiasauria), created by T. Dixon under a CC BY 4.0 license; Euhelopus zdanskyi (Somphospondyli), created by DiBgd and modified by T. M. Keesey under a CC BY-SA 3.0 license; Diplodocus carnegii (Flagellicaudata), created by S. Hartman under a CC BY 3.0 license; Giraffatitan brancai (Brachiosauridae, Titanosauriformes), created by S. Hartman under a CC BY 3.0 license; Camarasaurus supremus (Camarasauridae), created by M. Wedel under a CC BY 3.0 license. DMTA revealed dietary overlap between some sauropod clades, but also distinct niche segregation between others. Moreover, ecosystem (fauna)-specific wear signatures indicate that habitat parameters in the three analysed Late Jurassic ecosystems from Portugal (Lourinã Formation), USA (Morrison Formation) and Tanzania (Tendaguru Formation) were different, probably owing to climatic differences, which affected both plant community and environmental abrasive load. Overall higher roughness and complexity were observed in occlusal versus buccal surfaces in all taxa except for Flagellicaudata, where occlusal surfaces could only be observed in one tooth specimen. Hence, the limited occurrence of flagellicaudatan occlusal surfaces in our dataset hampers direct comparison with taxa that had tooth-to-tooth occlusion (and mammaliaform comparisons). The differences in occlusal versus buccal wear patterns are probably related to the different tooth morphologies. Turiasaurs and early-branching macronarians (represented by camarasaurids, brachiosaurids and possibly early somphospondylans in our dataset; Fig. 1), show true occlusion between antagonistic teeth. Such tooth-to-tooth occlusion resulted in strongly developed, V-shaped wear facets reaching from the apex down the mesial and distal surfaces7,17,18,27,28,35,51. Although the teeth lack adaptations for advanced oral food processing, such as chewing, such attritional contacts increase general wear on the occlusal surfaces compared with buccal surfaces. Through attritional contacts and tooth-to-tooth interaction, food is also trapped between teeth more efficiently, resulting in more tooth-to-food contacts (abrasion). Buccal surfaces hence experience less food contacts than occlusal surfaces, which is also evidenced by the development of the distinct occlusal wear facets with exposed dentin. In flagellicaudatans, the very narrow, pencil-shaped teeth show no distinct occlusal contacts, and only a single wear facet with exposed dentin formed on the tip of the tooth, which can be considered ‘occlusal surface-like' (compare Fig. Camarasaurids and turiasaurs had very similarly shaped, broad-crowned teeth with well-developed tooth-to-tooth occlusion7,17,18,51 and relatively narrow snouts57,58. Although these overlapping morphologies may suggest similarities in feeding niche, there was a distinct separation between camarasaurids and turiasaurs co-occurring in Portugal based on DMT data (Fig. North American camarasaurids, on the other hand, show considerable overlap with Portuguese turiasaurs. They show consistently low (height and roughness) signals, which is interpreted as a low-abrasive, consistent diet. In generalists, we would expect greater diversity of incorporated forage plants, and hence also a less consistent, more variable DMT signal. Portuguese camarasaurids are slightly different from their counterparts from the Morrison Formation, showing overall larger surface roughness. This pattern is best explained by ecological competition between camarasaurids and turiasaurs where they co-occurred in Portugal, which resulted in niche partitioning. Our data suggest that camarasaurids in Lourinã ended up occupying a different dietary niche from turiasaurs, and from contemporary camarasaurids, in North America. The absence of turiasaurs in the North American Morrison Formation allowed camarasaurids on that continent to occupy a similar feeding niche as that occupied by turiasaurs in Portugal (Fig. The unique dietary niche of camarasaurids in Lourinhã indicates that camarasaurids were more flexible in their dietary traits than turiasaurs, or that turiasaurs outcompeted camarasaurids in their preferred niche in Portugal—or a combination of both. At the same time, camarasaurids from North America and Portugal were similar in their very low variability of wear features. Additional evidence for migratory behaviour in sauropods derived from dental wear studies is important, as the currently available evidence is so sparse. On the basis of stable isotope data from a single Camarasaurus tooth from the Morrison Formation, ref. 59 suggested that this animal migrated throughout the year, potentially following its preferred food source and/or climate niche over the seasons. In fact, the strong seasonality in the Morrison Formation habitat24 probably impacted availability of a herbivore's preferred forage. A seasonal signal is known from microwear in mammals53,54,55, and must have been present in sauropods, too, given their high tooth formation and replacement rates52,60,61,62, which led to very short exposure times of a few weeks to maximum 2 or 3 months for each functional tooth. Therefore, if seasonal dietary shifts existed, finding a consistent dental microwear signal within one sauropod taxon is unlikely, as it would imply that all sampled teeth were derived from individuals that died in the same season. The consistently low variation in the DMT signal of camarasaurids from both the Morrison and the Lourinhã Formations is hence remarkable. It hints at a very narrow dietary niche occupied by camarasaurids, which may have forced these taxa to seasonally migrate and follow climate-driven availability patterns of their preferred food source throughout the seasons—not only in northern America but also in Portugal. Their absence from the Tendaguru fauna5 could indicate that camarasaurids were not well-adapted for the hot, tropical, and presumably less seasonal, climate of the region. The strongly variable signal in flagellicaudatans of the Morrison and Lourinhã Formations, in combination with even faster tooth replacement rates than camarasaurids52,60, on the other hand, hints at a broad dietary niche and rather unselective foraging. Unselective foraging has also been inferred for at least adult flagellicaudatans on the basis of their squared snout shape35. Such a broad dietary niche, indicated by variable DMT and the presence of differently sized wear features, would have allowed these animals to shift their diet to adapt to seasonal availability of resources. A seasonal dietary shift would render migration in this clade less necessary. Non-migratory behaviour in flagellicaudatans is also in parts supported by the biogeographic pattern of these clades within the Morrison Formation, where macronarian genera generally seem to have been more widespread compared with flagellicaudatan genera13,15,63,64. The more global distribution of macronarians within the formation may then have been partly due to their migratory behaviour, whereas flagellicaudatans restricted their geographic range by not migrating. The different environments they inhabited seem to have impacted microwear patterns, too. The strongly overlapping ranges of Portuguese titanosauriforms and turiasaurs in the CVA (Fig. 4), controlled by their high similarity in general surface roughness and complexity (Supplementary Fig. 1), show that these clades cannot securely be distinguished on the basis of DMT patterns. Hence, there seems to be a very strong environmental signal that overprints any dietary niche signal in Portugal, at least in these two clades, no strict niche separation between them or a distinct niche separation that could not be captured by DMTA or within the available dental sample. The overall lower values for height, volume and complexity parameters as well as low density of furrows for taxa from Portugal as compared with the Tanzanian fauna indicate that a low-abrasive diet was abundantly available in Portugal. Portuguese titanosauriforms, camarasaurids and turiasaurs are all very similar in their DMT patterns (Fig. 2 and Supplementary Fig. If these taxa were all migrating to some extent, following their preferred low-abrasive food sources, they must have occupied large enough home ranges to avoid competition—which seems to be supported by their large body size65. Camarasaurids from the USA showed very similar dental wear to turiasaurs (and also camarasaurids, to some degree) from Portugal (Fig. The Tanzanian taxa, however, are significantly different (compare Fig. Brachiosaurid titanosauriforms dominate our sample from Tanzania, and the combined sauropod fauna from Tendaguru show significantly larger complexity and density of furrows as compared with the faunas from both Portugal and the USA. Therefore, the sauropod taxa from the Tendaguru ecosystem must have differed in feeding preferences and/or Tendaguru sauropods ingested much higher abrasive loads compared with sauropods from the other two ecosystems, and especially in Portugal, where titanosauriforms showed distinctly lower surface roughness and complexity than in Tanzania (Fig. Although the flora of the Tendaguru Formation is different from that of the Lourinhã and Morrison Formations, in the absence of horsetails and pinacean conifers, and the low abundance of cycads19,20,21, several lines of evidence suggest that this was not the primary cause for the comparatively strong abrasion in Tendaguru sauropod teeth. Intrinsic abrasives in plants are mostly hard parts, such as seeds and silica-rich phytoliths. In the Tendaguru ecosystem, plants with extant relatives that have a high silica content were ginkgoes, cycads and possibly some ferns21,66. These were rather minor components of the flora, however21, with ginkgoes mostly growing in conifer-dominated forests20; conifers are relatively low in phytolith content66. We would therefore not expect overall stronger abrasion in an ecosystem with a lower relative abundance of highly abrasive plants, as is the case in Tendaguru. Moreover, given that titanosauriforms (the only clade we can compare directly between Tendaguru and another biome, in this case Lourinhã) had relatively broad snouts and reached enormous body masses, they were probably rather unselective in their diet, and it is therefore unlikely that they focused on these minor and silica-rich components of the flora. The presence of brachiosaurids and other early-branching titanosauriforms with similar morphologies in all three biomes analysed herein furthermore suggests that food preferences of these sauropods were comparable in the different biomes. The distinct abrasion patterns in sauropod teeth from Tendaguru are thus unlikely to be produced by plant-intrinsic abrasives. In fact, such intrinsic abrasives have already been shown to have a lesser impact on dental wear than external mineral abrasives. Data from mammals67,68,69,70,71 suggest that external mineral abrasives incorporated with the diet have a more pronounced effect on observed DMT signatures than plant type and phytolith content. Feeding experiments have shown that depth and complexity of wear features in particular, as well as density of furrows, were more strongly impacted by diets with high external abrasive load than by diets with high phytolith content. The observed distinct differences in complexity (Asfc) and density of furrows (medf) in Tanzanian brachiosaurids hence are probably stemming from a certain environmental grit load, rather than just phytoliths as internal abrasives of ingesta. Considering that the Tendaguru ecosystem in Tanzania was a riverine environment featuring extended floodplain areas, a potential source of grit would have been forage gathered close to the ground35,41. Low-level forage always comes with substantial amounts of sand and grit attached, if either pulled out of the ground or eaten shortly after rainfall on sandy ground. In extant herbivorous ungulates, soil intake can be up to 33% of dry matter intake in sheep72,73 or 0.5–1.5 kg in cattle74. However, our measurements from Tendaguru are mostly from brachiosaurids, which have obvious adaptations for high-level foraging27,29,34. Moreover, the Tendaguru fauna includes relatively abundant remains from the flagellicaudatans Dicraeosaurus and Tornieria, which have been consistently reconstructed as a low-to-mid-level feeders29,35,52, but no sampled flagellicaudatan tooth from Tendaguru survived our quality checks49,50. The higher abrasive load in Tendaguru brachiosaurids compared with all sauropod teeth except for two teeth referred to Flagellicaudata from the Lourinhã Formation in Portugal (Fig. 6) therefore rather stems from the geographical location of the two ecosystems. Palaeogeographic reconstructions suggest that Tendaguru would have been near an enormous desert belt, which stretched approximately between palaeolatitudes 16° and 32° S (ref. The presence of this desert probably resulted in wind-blown particles frequently covering the vegetation, which in turn caused more pronounced abrasive wear. In fact, sand-sized external quartz abrasives are known to cause large complexity and roughness values in mammalian herbivores69,71 that exceed parameter values attributed to high phytolith load. The recovered DMT patterns in Tendaguru sauropods fit this observation (Figs. Consequently, we think that the hypothesis of the nearby desert as the main contributor of grit to the abrasive load of the Tendaguru environment explains our observed patterns best. DMTs can be impacted by a number of factors related to intrinsic features of the food the individual had ingested, the general external abrasive load in the environment and the behaviour of the feeding individual. Teasing apart these factors in fossil ecosystems is not straightforward. In fact, our recovered within-taxon differences across different ecosystems and climate regimes indicate that inferences based on a limited sampling must be treated with some caution. Nevertheless, our study also shows that with careful investigation and correlation of microwear data with climate reconstructions, palaeobotanical evidence and morphological feeding adaptations, some of these factors can be identified. In fact, our data suggest ecological competition between broad-crowned taxa based on distinct patterns in dental microwear. Additionally, low variation in wear features supports earlier reports of migratory behaviour for camarasaurids in the Morrison and Lourinhã Formations, whereas a non-migratory behaviour can be suggested for flagellicaudatans on the basis of highly variable microwear patterns. Moreover, we could detect distinct microwear texture differences in titanosauriforms from Lourinhã and Tendaguru, which we interpreted in terms of a greater load of external abrasives in Tendaguru, probably stemming from a nearby desert. DMTA can thus be a useful additional proxy to palaeogeography and habitat reconstruction. For the 322 scans included in our analysis (Results), we imported a total of 25 of the 34 parameters computed by ref. 50, which are frequently applied on dental wear surfaces for diet inference47,48,67,69,70,76. These represent surface height, volume, complexity, density, slope, plateau size and anisotropy (Supplementary Table 6). Larger height and volume parameters (and a greater surface roughness in general) have been found to be indicative of feeding on tougher, more fibrous, or mechanically challenging diets (for example, grasses or hard objects such as seeds and molluscs)43,48,70,77, but are also affected by greater abrasive loads (dust or grit ingestion)68,69. Complexity is a good indicator of hard object feeding43,77, while anisotropy and density reflect incorporation of small particles (either external or internal abrasives)70,71,78, but may also be affected by intense, directional oral food processing. The scan data underwent a visual two-step quality screening by D.E.W. and A.S. to exclude scans affected by sediment particles, glue residues, fractures and postprocessing artefacts. Scans from moulds were also checked for imperfections such as air bubbles. The evaluation was guided by ref. 79 for identifying postmortem dental wear. Each scan was assigned a quality rating: good (1), fair (2) or bad (3); only scans attributed to quality 1 and 2 were considered here. For exemplary surface scans of each clade, see Extended Data Fig. For each tooth specimen and surface (occlusal and/or buccal), median values per parameter were calculated from up to four (at least one) non-overlapping scans. If several teeth of one individual had been measured, the mean from all teeth was then calculated for the individual. Where more than four measurements were available from different moulds of a single surface, we excluded the scans from the mould that had less scans qualified as quality 1 by ref. For further details on data acquisition and quality assessment, see ref. Statistical analyses to test for palaeoecological and/or climatic signals in the DMT were conducted in JMP Pro v.17. We used Dunn's test with Bonferroni adjustment for multiple pairwise comparisons. We found pronounced differences between buccal and occlusal surfaces for most clades (Fig. 2 and Supplementary Fig. We therefore analysed buccal and occlusal surfaces both individually and pooled for each clade per fauna (to assess niche partitioning between major taxa). To test for general faunal differences and to assess climate/habitat impact, we solely used the pooled dataset of buccal and occlusal surfaces. PCA including 20 DMT parameters with a factor loading larger than 0.7 (Asfc, matf, medf, metf, Sa, Sdc, Sdq, Sdr, Sk, Smc, Smr, Sp, Sq, Sv, Sz, Vm, Vmc, Vv, Vvc, Vvv) were computed with varimax factor rotation to facilitate interpretation of principal components (Fig. 3 and Supplementary Table 3). One brachiosaurid specimen from Tendaguru (MB.R. 2190) had to be excluded from the PCA, as it was so aberrant that it affected clear separation of groups in the dataset. CVA of all 25 parameters was used to maximize separation between clades for detection of niche partitioning, with either clade or clade + fauna as an identifier (Fig. 4 and Supplementary Table 2). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All measurements used for the analyses herein are included in Supplementary Table 1. They are a subset of measurements exported from a publicly available dataset, which includes the original scan files of tooth surface textures of sauropod teeth more generally (that is, not restricted to the three geological formations we analysed here). The full dataset including all computed measurements based on these scans is available on the UHH Forschungsdatenbank (https://www.fdr.uni-hamburg.de/) at https://doi.org/10.25592/uhhfdm.16992. D'Emic, M. D. The evolution of maximum terrestrial body mass in sauropod dinosaurs. Late Jurassic dinosaurs from the Morrison Formation (USA), the Lourinhã and Alcobaça Formations (Portugal), and the Tendaguru beds (Tanzania): a comparison. Tschopp, E., Mateus, O. A specimen-level phylogenetic analysis and taxonomic revision of Diplodocidae (Dinosauria, Sauropoda). Tschopp, E. et al. in Nature through Time: Virtual Field Trips through the Nature of the Past (eds Martinetto, E. et al.) 209–229 (Springer, 2020). Mannion, P. D., Upchurch, P., Schwarz, D. & Wings, O. Taxonomic affinities of the putative titanosaurs from the Late Jurassic Tendaguru Formation of Tanzania: phylogenetic and biogeographic implications for eusauropod dinosaur evolution. Mocho, P., Royo-Torres, R. & Ortega, F. A new macronarian sauropod from the Upper Jurassic of Portugal. & Tschopp, E. New sauropod teeth from the Upper Jurassic of Portugal and their implications for sauropod dental evolution. Ouyang, H. & Ye, Y. The First Mamenchisaurian Skeleton with Complete Skull, Mamenchisaurus youngi (Sichuan Publishing House of Science and Technology, 2002). Xing, L. et al. A new sauropod dinosaur from the Late Jurassic of China and the diversity, distribution, and relationships of mamenchisaurids. Moore, A. J., Upchurch, P., Barrett, P. M., Clark, J. M. & Xing, X. Osteology of Klamelisaurus gobiensis (Dinosauria, Eusauropoda) and the evolutionary history of Middle–Late Jurassic Chinese sauropods. Moore, A. J. et al. Re-assessment of the Late Jurassic eusauropod Mamenchisaurus sinocanadorum Russell and Zheng, 1993, and the evolution of exceptionally long necks in mamenchisaurids. Foster, J. R. Paleoecological analysis of the vertebrate fauna of the Morrison Formation (Upper Jurassic), Rocky Mountain region, USA. A., Trujillo, K. C. & Hanik, G. M. Assemblage-level structure in Morrison Formation dinosaurs, Western Interior, USA. Woodruff, D. What factors influence our reconstructions of Morrison Formation sauropod diversity? Maidment, S. C. R. Diversity through time and space in the Upper Jurassic Morrison Formation, western U.S.A. J. Vertebr. A. Re-evaluation of Australodocus bohetii, a putative diplodocoid sauropod from the Tendaguru Formation of Tanzania, with comment on Late Jurassic sauropod faunal diversity and palaeoecology. Mocho, P. et al. Turiasauria-like teeth from the Upper Jurassic of the Lusitanian Basin, Portugal. Mocho, P., Royo-Torres, R., Malafaia, E., Escaso, F. & Ortega, F. Sauropod tooth morphotypes from the Upper Jurassic of the Lusitanian Basin (Portugal). Pais, J. Jurassic plant macroremains from Portugal. Schrank, E. Pollen and spores from the Tendaguru Beds, Upper Jurassic and Lower Cretaceous of southeast Tanzania: palynostratigraphical and paleoecological implications. Gee, C. T. in Biology of the Sauropod Dinosaurs: Understanding the Life of Giants (eds Klein, N. et al.) 34–56 (Indiana Univ. Rees, P. M. A., Noto, C. R., Parrish, J. M. & Parrish, J. T. Late Jurassic climates, vegetation, and dinosaur distributions. Sellwood, B. W. & Valdes, P. J. Jurassic climates. Myers, T. S., Tabor, N. J. & Rosenau, N. A. Multiproxy approach reveals evidence of highly variable paleoprecipitation in the Upper Jurassic Morrison Formation (western United States). Cui Q. et al. Monsoon shaped herbivore dinosaur distribution. Preprint at Curr. Aberhan, M. et al. Palaeoecology and depositional environments of the Tendaguru Beds (Late Jurassic to Early Cretaceous, Tanzania). Calvo, J. O. Jaw mechanics in sauropod dinosaurs. Christiansen, P. Feeding mechanisms of the sauropod dinosaurs Brachiosaurus, Camarasaurus, Diplodocus and Dicraeosaurus. Upchurch, P. & Barrett, P. M. in Evolution of Herbivory in Terrestrial Vertebrates: Perspectives from the Fossil Record (ed. Sues, H.-D.) 79–122 (Cambridge Univ. & Parrish, J. M. in The Sauropods: Evolution and Paleobiology (eds Curry-Rogers, K. A. & Parrish, J. M. in Thunder Lizards: The Sauropodomorph Dinosaurs (eds Carpenter, K. & Tidwell, V.) 212–232 (Indiana Univ. First complete sauropod dinosaur skull from the Cretaceous of the Americas and the evolution of sauropod dentition. & Lamanna, M. C. Description of a nearly complete juvenile skull of Diplodocus (Sauropoda: Diplodocoidea) from the Late Jurassic of North America. & Dzemski, G. in Biology of the Sauropod Dinosaurs: Understanding the Life of Giants (eds Klein, N. et al.) Ch. A. Inferences of diplodocoid (Sauropoda: Dinosauria) feeding behavior from snout shape and microwear analyses. Christian, A. et al. Biomechanical reconstructions and selective advantages of neck poses and feeding strategies of sauropods with the example of Mamenchisaurus youngi. Barrett, P. M. Paleobiology of herbivorous dinosaurs. & Barrett, P. M. Cranial biomechanics underpins high sauropod diversity in resource-poor environments. The smallest diplodocid skull reveals cranial ontogeny and growth-related dietary changes in the largest dinosaurs. Bindellini, G. & Sasso, C. D. Sauropod teeth from the Middle Jurassic of Madagascar, and the oldest record of Titanosauriformes. Fiorillo, A. R. Dental micro wear patterns of the sauropod dinosaurs Camarasaurus and Diplodocus: evidence for resource partitioning in the Late Jurassic of North America. Bestwick, J., Unwin, D. M. & Purnell, M. A. Dietary differences in archosaur and lepidosaur reptiles revealed by dental microwear textural analysis. Controlled feeding experiments with juvenile alligators reveal microscopic dental wear texture patterns associated with hard-object feeding. Winkler, D. E., Kubo, T., Kubo, M. O., Kaiser, T. M. & Tütken, T. First application of dental microwear texture analysis to infer theropod feeding ecology. Sakaki, H. et al. Non-occlusal dental microwear texture analysis of a titanosauriform sauropod dinosaur from the Upper Cretaceous (Turonian) Tamagawa Formation, northeastern Japan. Kubo, T. et al. Dental microwear texture analysis reveals a likely dietary shift within Late Cretaceous ornithopod dinosaurs. Dental microwear texture analysis shows within-species diet variability in fossil hominins. Schulz, E., Calandra, I. & Kaiser, T. M. Applying tribology to teeth of hoofed mammals. Saleiro, A. et al. A dataset for dental microwear texture analysis (DMTA) of sauropod dinosaur teeth (Version v2) [Data set]. Saleiro, A. et al. Acquisition of a dataset for dental microwear texture analysis (DMTA) of sauropod teeth pre-analytical and post-processing steps for quality assessment. Preprint at PaleorXiv https://doi.org/10.31233/osf.io/8uhsp (2024). Wiersma, K. & Sander, P. M. The dentition of a well-preserved specimen of Camarasaurus sp. : implications for function, tooth replacement, soft part reconstruction, and food intake. Schwarz, D., Kosch, J. C. D., Fritsch, G. & Hildebrandt, T. Dentition and tooth replacement of Dicraeosaurus hansemanni (Dinosauria, Sauropoda, Diplodocoidea) from the Tendaguru Formation of Tanzania. Investigation of human hunting seasonality through dental microwear analysis of two Caprinae in late Pleistocene localities in Southern France. Gogarten, J. F. & Grine, F. E. Seasonal mortality patterns in primates: implications for the interpretation of dental microwear. Winkler, D. E. et al. The turnover of dental microwear texture: testing the ‘last supper' effect in small mammals in a controlled feeding experiment. Mannion, P. D., Tschopp, E. & Whitlock, J. A. Anatomy and systematics of the diplodocoid Amphicoelias altus supports high sauropod dinosaur diversity in the Upper Jurassic Morrison Formation of the USA. Chatterjee, S. & Zheng, Z. Cranial anatomy of Shunosaurus, a basal sauropod dinosaur from the Middle Jurassic of China. Royo-Torres, R. & Upchurch, P. The cranial anatomy of the sauropod Turiasaurus riodevensis and implications for its phylogenetic relationships. & Hoerner, M. E. Lowland–upland migration of sauropod dinosaurs during the Late Jurassic epoch. D'Emic, M. D., Whitlock, J. A., Smith, K. M., Fisher, D. C. & Wilson, J. A. Evolution of high tooth replacement rates in sauropod dinosaurs. Evidence for niche partitioning among ground-height browsing sauropods from the Upper Jurassic Morrison Formation of North America. Peterson, J. E., Lovelace, D., Connely, M. & McHugh, J. A novel feeding mechanism of diplodocid sauropods revealed in an Apatosaurine skull from the Upper Jurassic Nail Quarry (Morrison Formation) at Como Bluff, Wyoming, USA. Maltese, A., Tschopp, E., Holwerda, F. & Burnham, D. The real Bigfoot: a pes from Wyoming, USA is the largest sauropod pes ever reported and the northern-most occurrence of brachiosaurids in the Upper Jurassic Morrison Formation. Tschopp, E., Maidment, S. C. R., Lamanna, M. C. & Norell, M. A. Reassessment of a historical collection of sauropod dinosaurs from the northern Morrison Formation of Wyoming, with implications for sauropod biogeography. Agusti, J., Alcalá, L. & Santos-Cubedo, A. Did large foraging migrations favor the enormous body size of giant sauropods? The case of Turiasaurus. Hodson, M. J., White, P. J., Mead, A. & Broadley, M. R. Phylogenetic variation in the silicon composition of plants. Schulz, E. et al. Dietary abrasiveness is associated with variability of microwear and dental surface texture in rabbits. Ackermans, N. L. et al. Dust and grit matter: abrasives of different size lead to opposing dental microwear textures in experimentally fed sheep (Ovis aries). Winkler, D. E. et al. Shape, size, and quantity of ingested external abrasives influence dental microwear texture formation in guinea pigs. Winkler, D. E. et al. Forage silica and water content control dental surface texture in guinea pigs and provide implications for dietary reconstruction. Winkler, D. E. et al. Dental microwear texture gradients in guinea pigs reveal that material properties of the diet affect chewing behaviour. & Ludwig, T. G. Wear of sheep's teeth: I. The role of ingested soil. & Ludwig, T. G. Ingestion of soil by sheep in New Zealand in relation to wear of teeth. Mayland, H. F., Florence, A., Rosenau, R. C. & Lazar, V. A. Soil ingestion by cattle on semiarid range as reflected by titanium analysis of feces. Rees, P. M., Ziegler, A. M. & Valdes, P. J. in Warm Climates in Earth History (eds MacLeod, H. B. & Wing, S. K.) 297–318 (Cambridge Univ. Ungar, P. S., Brown, C. A., Bergstrom, T. S. & Walker, A. Quantification of dental microwear by tandem scanning confocal microscopy and scale-sensitive fractal analyses. Winkler, D. E. et al. Microwear textures associated with experimental near-natural diets suggest that seeds and hard insect body parts cause high enamel surface complexity in small mammals. Winkler, D. E., Schulz-Kornas, E., Kaiser, T. M. & Tütken, T. Dental microwear texture reflects dietary tendencies in extant Lepidosauria despite their limited use of oral food processing. Weber, K., Winkler, D. E., Schulz-Kornas, E., Kaiser, T. M. & Tütken, T. The good, the bad and the ugly—a visual guide for common post-mortem wear patterns in vertebrate teeth. We thank A. Daasch, K. Engelkes and L. Schwinger (LIB) for technical support during the measurements. We thank the numerous curators and collection managers (AMNH, CM, CMC, MB.R., ML, NMZ, SMA, USNM and YPM; see ref. 50 for a complete list of people) who have allowed us to sample and/or loan original sauropod teeth. A. Moore (Stony Brook University) provided insights into non-neosauropod tooth morphology. Funding for this work was provided through a Humboldt Research Fellowship for Experienced Researchers (USA 1218977 HFST-E) and a UHH Close the Gap Grant (both to E.T. ); a grant through Synthesys+Transnational Access, 4th call (DE-TAF-TA4-009); a doctoral grant from FCT—Fundação para a Ciência e a Tecnología, IP, (2020.05854.BD); and additional support by FCT—Fundação para a Ciência e a Tecnología, IP, through the Research Unit UIDB/04035/2020 (GeoBioTec; https://doi.org/10.54499/UIDB/04035/2020) and the Project PTDC/CTA-PAL/2217/2021 (BioGeoSauria) (all to A.S.). Open access funding provided by Freie Universität Berlin. Present address: Department of Vertebrate Paleontology, American Museum of Natural History, New York, USA, NY These authors contributed equally: Daniela E. Winkler, Emanuel Tschopp. Centre for Taxonomy and Morphology, Leibniz Institute for the Analysis of Biodiversity Change, Museum of Nature, Hamburg, Germany Daniela E. Winkler, Emanuel Tschopp & Thomas M. Kaiser Emanuel Tschopp, Ria Wiesinger & Thomas M. Kaiser Freie Universität Berlin, Institut für Geologische Wissenschaften, Berlin, Germany GeoBioTec, NOVA School of Science and Technology, University NOVA of Lisbon, Caparica, Portugal Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar : supervision, data analysis, interpretation, writing—original draft, writing —review and editing, visualization. : project design, supervision, interpretation, writing—original draft, writing —review and editing, visualization, funding. : data collection, methods. : resources, methods, data analysis, writing—review and editing. contributed equally to this work. Correspondence to Daniela E. Winkler or Emanuel Tschopp. The authors declare no competing interests. Nature Ecology & Evolution thanks Verónica Díez Díaz, Aaron LeBlanc and Juha Saarinen for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Colour maps represent surface topography: warm colours (red/yellow) indicate higher elevations, while cool colours (green/blue) indicate depressions. All surfaces are to the same scale. Exemplary teeth of the five represented clades are shown in buccal view (Camarasauridae, ML 636; Flagellicaudata, ML 2560; Turiasauria, ML 1178; Titanosauriformes, ML 945, all from Lourinhã, taken from ref. 7; Sauropoda indet., MB.R.2185, from Tendaguru, photo by ET). Credits: Silhouettes are from Phylopic (https://phylopic.org). Xinjiangtitan shanshanesis (Sauropoda indet. ), created by Jagged Fang Designs under a CC0 1.0 license; Amanzia greppini (Turiasauria), created by T. Dixon under a CC BY 4.0 license; Diplodocus carnegii (Flagellicaudata), created by S. Hartman under a CC BY 3.0 license; Giraffatitan brancai (Titanosauriformes), created by S. Hartman under a CC BY 3.0 license; Camarasaurus supremus (Camarasauridae), created by M. Wedel under a CC BY 3.0 license. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Reprints and permissions Winkler, D.E., Tschopp, E., Saleiro, A. et al. Dental microwear texture analysis reveals behavioural, ecological and habitat signals in Late Jurassic sauropod dinosaur faunas. Nat Ecol Evol  (2025). Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Ecology & Evolution (Nat Ecol Evol) © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41551-025-01422-8'>Mannich reaction-based combinatorial libraries identify antioxidant ionizable lipids for mRNA delivery with reduced immunogenicity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 09:10:32
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Biomedical Engineering (2025)Cite this article The immunogenicity of lipid nanoparticles (LNPs) used for the delivery of nucleoside-modified messenger RNA limits the levels and durability of expression of the encoded protein. Here, by leveraging the Mannich reaction for ionizable lipid synthesis, and via the in vitro and in vivo screening of six combinatorial libraries of synthesized lipids, we report the identification of an antioxidant ionizable lipid, C-a16, exhibiting reduced immunogenicity. When incorporated into LNPs for mRNA delivery, C-a16 mitigated the generation of intracellular reactive oxygen species, thereby extending the duration of protein expression. In mice, and compared with commercial LNPs, LNPs incorporating C-a16 and co-delivering Cas9 mRNA and guide RNA for the editing of the transthyretin gene led to 2.8-fold higher editing efficiency; LNPs with C-a16 delivering fibroblast growth factor 21 mRNA increased the expression of the protein 3.6-fold; and when delivering mRNA encoding a tumour neoantigen or the spike protein of SARS-CoV-2, LNPs with C-a16 induced stronger antigen-specific immune responses. Our findings support the further testing of C-a16 as a promising ionizable lipid for mRNA delivery in therapeutic applications. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout The data that support the findings of this study are available within the paper and its Supplementary Information files. Source data are provided with this paper. Pardi, N., Hogan, M. J., Porter, F. W. & Weissman, D. mRNA vaccines—a new era in vaccinology. Drug Discov. Karikó, K. et al. Incorporation of pseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and biological stability. Karikó, K., Buckstein, M., Ni, H. & Weissman, D. Suppression of RNA recognition by Toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA. A., Witzigmann, D., Chen, S., Cullis, P. R. & van der Meel, R. Lipid nanoparticle technology for clinical translation of siRNA therapeutics. Cullis, P. R. & Hope, M. J. Lipid nanoparticle systems for enabling gene therapies. Akinc, A. et al. The Onpattro story and the clinical translation of nanomedicines containing nucleic acid-based drugs. Chen, S. et al. Nanotechnology-based mRNA vaccines. Chaudhary, N., Weissman, D. & Whitehead, K. A. mRNA vaccines for infectious diseases: principles, delivery and clinical translation. Drug Discov. Igyártó, B. Z., Jacobsen, S. & Ndeupen, S. Future considerations for the mRNA-lipid nanoparticle vaccine platform. Ndeupen, S. et al. The mRNA-LNP platform's lipid nanoparticle component used in preclinical vaccine studies is highly inflammatory. Moghimi, S. M. & Simberg, D. Pro-inflammatory concerns with lipid nanoparticles. Minnaert, A.-K. et al. Strategies for controlling the innate immune activity of conventional and self-amplifying mRNA therapeutics: getting the message across. Drug Deliv. Siegwart, D. J. et al. Combinatorial synthesis of chemically diverse core-shell nanoparticles for intracellular delivery. Zhang, Y., Sun, C., Wang, C., Jankovic, K. E. & Dong, Y. Lipids and lipid derivatives for RNA delivery. Eygeris, Y., Gupta, M., Kim, J. & Sahay, G. Chemistry of lipid nanoparticles for RNA delivery. Witten, J., Hu, Y., Langer, R. & Anderson, D. G. Recent advances in nanoparticulate RNA delivery systems. Tramontini, M. Advances in the chemistry of Mannich bases. Tramontini, M. & Angiolini, L. Further advances in the chemistry of Mannich bases. Tramontini, M., Angiolini, L. & Ghedini, N. Mannich bases in polymer chemistry. Zhang, H. & Tsao, R. Dietary polyphenols, oxidative stress and antioxidant and anti-inflammatory effects. Bucciantini, M., Leri, M., Nardiello, P., Casamenti, F. & Stefani, M. Olive polyphenols: antioxidant and anti-inflammatory properties. Yahfoufi, N., Alsadi, N., Jambi, M. & Matar, C. The immunomodulatory and anti-inflammatory role of polyphenols. Grant, C. M. Regulation of translation by hydrogen peroxide. Devoldere, J., Dewitte, H., De Smedt, S. C. & Remaut, K. Evading innate immunity in nonviral mRNA delivery: don't shoot the messenger. Linares-Fernández, S., Lacroix, C., Exposito, J.-Y. & Verrier, B. Tailoring mRNA vaccine to balance innate/adaptive immune response. Forster Iii, J., Nandi, D. & Kulkarni, A. mRNA-carrying lipid nanoparticles that induce lysosomal rupture activate NLRP3 inflammasome and reduce mRNA transfection efficiency. Filomeni, G., De Zio, D. & Cecconi, F. Oxidative stress and autophagy: the clash between damage and metabolic needs. Cell Death Differ. Dasuri, K., Zhang, L. & Keller, J. N. Oxidative stress, neurodegeneration, and the balance of protein degradation and protein synthesis. Cheng, Q. et al. Selective organ targeting (SORT) nanoparticles for tissue-specific mRNA delivery and CRISPR–Cas gene editing. Finn, J. D. et al. A single administration of CRISPR/Cas9 lipid nanoparticles achieves robust and persistent in vivo genome editing. Gillmore, J. D. et al. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. Fisher, F. M. & Maratos-Flier, E. Understanding the physiology of FGF21. Flippo, K. H. & Potthoff, M. J. Metabolic messengers: FGF21. Geng, L., Lam, K. S. & Xu, A. The therapeutic potential of FGF21 in metabolic diseases: from bench to clinic. Jimenez, V. et al. FGF21 gene therapy as treatment for obesity and insulin resistance. Gong, N. et al. Proton-driven transformable nanovaccine for cancer immunotherapy. Buonaguro, L., Petrizzo, A., Tornesello, M. L. & Buonaguro, F. M. Translating tumor antigens into cancer vaccines. Blass, E. & Ott, P. A. Advances in the development of personalized neoantigen-based therapeutic cancer vaccines. Ott, P. A. et al. An immunogenic personal neoantigen vaccine for patients with melanoma. Rojas, L. A. et al. Personalized RNA neoantigen vaccines stimulate T cells in pancreatic cancer. Hu, Z. et al. Personal neoantigen vaccines induce persistent memory T cell responses and epitope spreading in patients with melanoma. Sittplangkoon, C. et al. mRNA vaccine with unmodified uridine induces robust type I interferon-dependent anti-tumor immunity in a melanoma model. Berger, A. Th1 and Th2 responses: what are they? Kaye, P. M., Curry, A. J. & Blackwell, J. M. Differential production of Th1- and Th2-derived cytokines does not determine the genetically controlled or vaccine-induced rate of cure in murine visceral leishmaniasis. Darrah, P. A. et al. Multifunctional TH1 cells define a correlate of vaccine-mediated protection against Leishmania major. Precopio, M. L. et al. Immunization with vaccinia virus induces polyfunctional and phenotypically distinctive CD8+ T cell responses. Giannotta, G. & Giannotta, N. mRNA COVID-19 vaccines and long-lived plasma cells: a complicated relationship. Krienke, C. et al. A noninflammatory mRNA vaccine for treatment of experimental autoimmune encephalomyelitis. Porto, E. M., Komor, A. C., Slaymaker, I. M. & Yeo, G. W. Base editing: advances and therapeutic opportunities. acknowledges support from a US National Institutes of Health (NIH) Director's New Innovator Award (DP2 TR002776), a Burroughs Wellcome Fund Career Award at the Scientific Interface (CASI), a US National Science Foundation CAREER Award (CBET-2145491) and the American Cancer Society (number RSG-22-122-01-ET). acknowledges support from a US National Science Foundation Graduate Research Fellowship (DGE 1845298). C.G.F.-E. acknowledges support from a US National Science Foundation Graduate Research Fellowship (DGE 1845298), a GEM Fellowship and the NIH/National Cancer Institute Pre-doc to Post-doc Transition Award (F99 CA284294). These authors contributed equally: Ningqiang Gong, Dongyoon Kim. Department of Bioengineering, University of Pennsylvania, Philadelphia, PA, USA Ningqiang Gong, Dongyoon Kim, Emily L. Han, Rohan Palanki, Qiangqiang Shi, Xuexiang Han, Lulu Xue, Junchao Xu, Christian G. Figueroa-Espada & Michael J. Mitchell Department of General Surgery, The First Affiliated Hospital of USTC, Division of Life Science and Medicine, University of Science and Technology of China, Hefei, China Ningqiang Gong, Zilin Meng & Tianyu Luo Department of Medicine, University of Pennsylvania, Philadelphia, PA, USA Mohamad-Gabriel Alameh, Rakan El-Mayta, Garima Dwivedi, Drew Weissman & Michael J. Mitchell Penn Institute for RNA Innovation, University of Pennsylvania, Philadelphia, PA, USA Mohamad-Gabriel Alameh, Rakan El-Mayta, Garima Dwivedi, Drew Weissman & Michael J. Mitchell Department of Chemistry, Tsinghua University, Beijing, China Abramson Cancer Center, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Institute for Immunology, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Cardiovascular Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Institute for Regenerative Medicine, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived of and designed the experiments. and C.G.F.-E. performed the experiments. and R.E.-M. analysed the data. involved in results discussion. wrote and edited the paper. supervised the entire project. All authors discussed the results and commented on the paper. Correspondence to Michael J. Mitchell. have filed a patent application related to this study. The other authors declare no competing interests. Nature Biomedical Engineering thanks Bruno De Geest, Shuai Liu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. C-a16, C-a16-Q, DLin-MC3-DMA LNPs or PBS was intravenously injected, and a complete blood counting (CBC) assay was performed 24 hours post-LNP injection. a, The levels of white blood cells (WBC), b, neutrophils (Neu), c, lymphocytes (Lym), and d, monocytes (Mon), in the blood were measured (n = 4). ELISA assay was used to quantify the levels of IL-2 (e) and TNF-α (f) in the mouse serum (n = 4). Statistical differences were calculated using one-way ANOVA with Tukey's post hoc test. a-d, Quantification of LNP distribution in immune cell populations, including DCs (a), macrophages (b), B cells (c), and T cells (d), assessed by flow cytometry. Data are presented as mean ± SD (n = 5), and statistical differences were determined by one-way ANOVA. e, Immunofluorescence images of lymph nodes in mice after intramuscular injection of DiR dye-labeled LNPs. Nuclei are represented in blue, CD11c+ DCs in green, and DiR-labeled LNPs in red. f, Evaluation of OVA-specific T cell generation following two doses of C-a16-mOVA vaccines. Seven days after the boost dose, blood T cells were stained with SIINFEKL-H2Kb tetramer-APC antibody. After that, SIINFEKL-H2Kb tetramer+ cells were detected by flow cytometry (f) and quantified (g). h and i, Investigation of vaccine-induced immune responses' ability to eliminate SIINFEKL epitope+ target cells. A mixture of CFSElow SIINFEKL-loaded and CFSEhigh non-loaded splenocytes (in a 1:1 ratio) was intravenously injected into vaccinated mice. After 24 hours, splenocytes were collected and analyzed by flow cytometry (h), and target cell killing was quantified (i). Data in a-d, g and i are shown as mean ± SD (n = 5). Statistical differences were calculated using one-way ANOVA with Tukey's post hoc test. a, Body weight change after mice were treated with various LNP vaccines. b, Mice survival curves after treatment with various LNP vaccines encapsulating OVA mRNA. c, Survival curves of mice receiving various LNPs encapsulating Pbk-Actn4 mRNA. Data in a are shown as mean ± SD (n = 8). Mice received intramuscular immunization with various LNP vaccines on days 0 and 21 (0.25 mg/kg). On day 35, splenocytes were collected and stimulated with SARS-CoV-2 RBD peptide pools. a and b, T cells were assessed for Th2 (IL-4, IL-5) and c, Th17 (IL-17a) intracellular cytokine expression. d, Evaluation of the expression of the cytotoxic marker CD107α expression in CD8+ T cells. Data are shown as mean ± SD (n = 5). Statistical differences were calculated using one-way ANOVA with Tukey's post hoc test. Mice received intramuscular immunization with various LNP vaccines on days 0 and 21 (0.25 mg/kg). On day 35, splenocytes were collected and stimulated with SARS-CoV-2 RBD peptide pools. CD4+ (a-g) and CD8+ (h-n) polyfunctional T cell percentages (%) were assessed. Data are shown as mean ± SD (n = 5). Statistical differences were calculated using one-way ANOVA with Tukey's post hoc test. a, Mice received intramuscular immunization with various LNP vaccines on days 0 and 21 (0.25 mg/kg). IgG2c to IgG1 ratio was determined. Data are presented as mean ± s.d. b, Enumeration of RBD-specific B cells per spleen. Splenocytes were stimulated with SARS-CoV-2 RBD peptide pools and stained with various antibodies before flow cytometry analysis. c, Percentage of RBD-specific B cells categorized by germinal center (GC) or memory phenotype. GC B cells were defined as CD38−GL7+, and memory B cells were defined as CD38+GL7−. Data are shown as mean ± SD (n = 5). Statistical differences were calculated using one-way ANOVA with Tukey's post hoc test. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Gong, N., Kim, D., Alameh, MG. et al. Mannich reaction-based combinatorial libraries identify antioxidant ionizable lipids for mRNA delivery with reduced immunogenicity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Biomedical Engineering © 2025 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250717013852.htm'>Ancient recipes or rituals? Neanderthal bones reveal a prehistoric culinary mystery</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 04:39:35
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A new study suggests that two groups of Neanderthals living in the caves of Amud and Kebara in northern Israel butchered their food in strikingly different ways, despite living close by and using similar tools and resources. Scientists think they might have been passing down different food preparation practices. “The subtle differences in cut-mark patterns between Amud and Kebara may reflect local traditions of animal carcass processing,” said Anaëlle Jallon, PhD candidate at the Hebrew University of Jerusalem and lead author of the article in Frontiers in Environmental Archaeology. “Even though Neanderthals at these two sites shared similar living conditions and faced comparable challenges, they seem to have developed distinct butchery strategies, possibly passed down through social learning and cultural traditions. “These two sites give us a unique opportunity to explore whether Neanderthal butchery techniques were standardized,” explained Jallon. “If butchery techniques varied between sites or time periods, this would imply that factors such as cultural traditions, cooking preferences, or social organization influenced even subsistence-related activities such as butchering.” Amud and Kebara are close to each other: only 70 kilometers apart. Both groups used the same flint tools and relied on the same prey for their diet — mostly gazelles and fallow deer. But there are some subtle differences between the two. They examined these macroscopically and microscopically, recording the cut-marks' different characteristics. Similar patterns of cut-marks might suggest there were no differences in butchery practices, while different patterns might indicate distinct cultural traditions. The researchers considered several possible explanations for this pattern. Experimental archaeology also suggests this pattern couldn't be accounted for by less skilled butchers or by butchering more intensively to get as much food as possible. However, more research will be needed to investigate these possibilities. While we have made efforts to correct for biases caused by fragmentation, this may limit our ability to fully interpret the data. Future studies, including more experimental work and comparative analyses, will be crucial for addressing these uncertainties — and maybe one day reconstructing Neanderthals' recipes.” Note: Content may be edited for style and length. This Obesity Drug Cuts Calorie Intake by 72% Without Dieting Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/07/250717013903.htm'>Not just hot flashes: The hidden depression crisis in early menopause</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-07-18 03:47:11
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Premature menopause has been described as a life-changing diagnosis with profound physical, psychological, and social consequences. However, some women are more adversely affected by these changes than others. A new study helps explore reasons for these differences. Premature menopause, medically known as premature or primary ovarian insufficiency (POI), is a condition in which the ovaries cease to function normally before the age of 40. It has been linked to an elevated lifetime risk for depression and anxiety. A recent meta-analysis revealed an odds ratio of 3.3 for depression and 4.9 for anxiety in women with POI compared with those without the condition. The increased risk is understandable given the combined experience of infertility and the additional burdens resulting from estrogen deficiency, such as hot flashes, vaginal dryness, reduced bone mineral density, and an increased risk of cardiovascular disease, among others. Their first observation was the high prevalence of depression among participants. Nearly one-third (29.9%) of the women with POI suffered from depressive symptoms. The researchers additionally found that a younger age at POI diagnosis, severe menopause symptoms, fertility-related grief, and lack of emotional support were risk factors. No significant difference was found in depressive symptoms between women using estrogen plus progestogen therapy and those not using hormone therapy, underscoring the role of psychosocial factors. Interestingly, a genetic cause for POI was associated with lower depressive symptoms. Another unexpected result was that, even though a higher burden of menopause symptoms was independently associated with depressive symptoms, hot flashes (specifically night sweats) were not. The researchers believe its results highlight the importance of comprehensive care addressing both physical and psychological aspects of menopause at an early age. Addressing behavioral-health concerns with evidence-based interventions should be part of any comprehensive POI care plan," says Dr. Monica Christmas, associate medical director for The Menopause Society. Unexplained Gases on Venus: Could They Be the Sign of Alien Life? Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            