
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-27</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250527124632.htm'>Nordic studies show the significance of old-growth forests for biodiversity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-27 19:03:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers at the University of Eastern Finland conducted a systematic review of 99 scientific publications that compared the flora or fauna of old-growth forests, managed forests and clearcut sites in boreal Europe. Clearcut sites are also species-rich, but they are inhabited by a distinct set of species in comparison to full-canopy forests. The studies had examined several species groups that cover a large part of the flora and fauna inhabiting forests, including birds, epiphytes (i.e., lichens and bryophytes growing on tree trunks), fungi and insects inhabiting deadwood, soil fungi, ground-dwelling invertebrates, other insects and understory vegetation. Understory vegetation and insects other than those inhabiting deadwood were more diverse in younger forests, especially clearcut sites. The flora and fauna characteristic of old-growth forests are associated with old or dead trees. "In managed forests, trees are cut down before they grow old or die, and thus old and dead trees are scanty in them. Leaving a portion of the trees standing during clearcutting would be an effective solution, but only very few of these so-called retention trees are left in harvests," says Doctoral Researcher Aleksi Nirhamo from the University of Eastern Finland. Clearcut sites are inhabited by species associated with open habitats and young forests. Conditions on clearcut sites are very different from full-canopy forests, and thus they are inhabited largely by distinct sets of species. "Early stages of forest succession would be formed without clearcuts by disturbances such as storms. These kinds of habitats differ from clearcut sites for example by having huge amounts of deadwood that is missing from clearcut sites," Nirhamo points out. "Forest habitats are modified widely in order to enhance wood production, which homogenises and degrades them," Nirhamo says. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250527124640.htm'>Nature-inspired breakthrough enables subatomic ferroelectric memory</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-27 19:03:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A research team led by Prof. Si-Young Choi from the Department of Materials Science and Engineering and the Department of Semiconductor Engineering at POSTECH (Pohang University of Science and Technology) has discovered ferroelectric phenomena occurring at a subatomic scale in the natural mineral Brownmillerite, in collaboration with Prof. Jae-Kwang Lee's team from Pusan National University as well as Prof. Woo-Seok Choi's team from Sungkyunkwan University. The research was published on May 20 in Nature Materials. Electronic devices store data in memory units called 'domains,' whose minimum size limits the density of stored information. However, ferroelectric-based memory has been facing challenges in minimizing domain size due to the collective nature of atomic vibrations. The research team found inspiration to overcome these limitations in nature. They focused on Brownmillerite, a naturally occurring mineral characterized by its unique alternating layers of tetrahedral (FeO4) and octahedral (FeO6) iron-oxygen structures, resembling a sandwich with alternating layers of bread and ham. Strikingly, Brownmillerite exhibits a special phenomenon known as 'phonon decoupling.' The team further demonstrated the practicality of this phenomenon by successfully developing ferroelectric capacitors and thin-film transistor devices based on this structure. If commercialized, this technology is expected to enable the development of memory devices that are tens of times smaller and faster than current models. Prof. Si-Young Choi of POSTECH remarked, "This study exemplifies how wisdom derived from nature can provide critical solutions to technological limitations. Materials provided by Pohang University of Science & Technology (POSTECH). Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59486-8'>Late Paleolithic whale bone tools reveal human and whale ecology in the Bay of Biscay</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-27 15:32:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Reconstructing how prehistoric humans used the products obtained from large cetaceans is challenging, but key to understand the history of early human coastal adaptations. Here we report the multiproxy analysis (ZooMS, radiocarbon, stable isotopes) of worked objects made of whale bone, and unworked whale bone fragments, found at Upper Paleolithic sites (Magdalenian) around the Bay of Biscay. Taxonomic identification using ZooMS reveals at least five species of large whales, expanding the range of known taxa whose products were utilized by humans in this period. Radiocarbon places the use of whale products ca. 20–14 ka cal BP, with a maximum diffusion and diversity at 17.5–16 ka cal BP, making it the oldest evidence of whale-bone working to our knowledge. δ13C and δ15N stable isotope values reflect taxon-specific differences in foraging behavior. The diversity and chronology of these cetacean populations attest to the richness of the marine ecosystem of the Bay of Biscay in the late Paleolithic, broadening our understanding of coastal adaptations at that time. Whales are the largest living animals on Earth and the current populations of many species are a mere fraction of their abundance in the past1. Before intensive whaling depleted the populations of most species, whales were a valuable source of food and other resources (e.g., oil, bone, baleen2). They were thus a key part of subsistence for many coastal human groups worldwide, including hunter-gatherers and Neolithic farmers, with acquisition methods that included scavenging freshly beached animals, opportunistic killing and organized whaling2,3,4,5,6,7,8. However, reconstructing the beginning of whale utilization is challenging because prehistoric coastal sites are an especially fragile part of the archeological record, many of them having been lost to marine erosion or flooded by the last marine transgressions9. In most cases, the only available evidence is indirect, in the form of materials of coastal origin transported by people into inland sites. In Europe, a number of sites attributed to the Middle and Upper Magdalenian culture (ca. 19-14 ka cal BP, or 19-14 millennia before present, at the end of MIS2) in southwestern France and in Atlantic Iberia have yielded an invaluable record of this type of evidence10: whale barnacles attesting to the transport of whale skin, blubber and meat11; unworked whale bones transported and processed at the habitation site12; worked whale teeth13; and, especially, more than 150 tools and projectile heads made of whale bone presumably of Atlantic origin, mostly found scattered from Asturias to the central part of the northern Pyrenean range14. This record, mostly identified within the last ten years, represents (to our knowledge) the oldest evidence of a regular utilization of whale products by humans, for dietary needs and raw materials10. Understanding this Magdalenian utilization of whale products is of crucial importance, not only because it opens a window into early human interactions with whales, but more broadly because of the light it sheds onto the history of human coastal adaptations. Indeed, even though the earliest evidence of the use of coastal resources in Europe dates back to the Middle Paleolithic (older than 40 ka cal BP, and up to ca. 150 ka BP15,16,17), and similar evidence exists for the Early Upper Paleolithic18, the richness and diversity of evidence related to seashore exploitation increases dramatically in association with the Magdalenian culture. Despite a sea level rise of ca. 120 m since that period making the Magdalenian shoreline inaccessible to today's archeologists, depictions of marine animals, remains of seals, dolphins, marine fish and birds, the use of marine mollusks as food and the use of their shells as raw material for ornaments, are all attested with unprecedented frequency and wide diffusion in the Middle and Upper Magdalenian, especially in Iberia and in the Pyrenees10,19,20. This combined evidence points to a stronger link with the seashore compared to the earlier part of the European Upper Paleolithic, suggesting the inception of a settled coastal economy in certain parts of Europe at this time. Understanding the role that whale products played in this process is thus key to the ongoing investigations into the existence, chronology and organization of hunter-gatherer early coastal adaptations, a subject that is today considered central in prehistoric research21,22,23. However, the information provided by the whale remains found in Magdalenian assemblages is limited for several reasons. First, it is unknown to which whale species they correspond. Indeed, these remains were identified only through visual macroscopic criteria and most of them are small, fragmented and/or worked, thus precluding any precise anatomical or taxonomic identification beyond the cetacean category. Furthermore, although these morphological criteria are robust, they have a margin of uncertainty, meaning that some of the remains are classified only as “likely” made of cetacean bone14,24. Since different whale species have different ecologies and feeding behavior, this generic (and sometimes uncertain) identification as cetacean gives only low resolution insight into the range of interactions that foragers could have had with these animals, and deprives us of the environmental information that the presence of certain particular whale species could indicate25. Finally, there is substantial uncertainty regarding the dating of these whale remains. Because of their typology, their stratigraphic context, and the characteristics of the archeological assemblages they were found in, they can be attributed to the Magdalenian period in the broad sense of the term. But since the majority are from ancient excavations with poor chrono-stratigraphic resolution, they can only be collectively ascribed to a broad time span of several millennia, precluding a precise reconstruction of the chronology, rhythm and evolution of the utilization of whale products. Here we analyze a large sample of worked bone objects (n = 83) from 26 Magdalenian cave and rockshelter sites in the Cantabrian region and southwestern France, all visually identified as made of whale bone (Supplementary Data 1 and 2, Supplementary Figs. We also analyze 90 unworked bone fragments from the single assemblage of Santa Catalina cave (Biscay), also ascribed to whale on a visual basis, found among the faunal remains of the site's Upper Magdalenian occupation and showing traces of anthropic processing (notably percussion notches: Supplementary Figs. We sampled these for taxonomic identification through collagen peptide mass fingerprinting (ZooMS, or Zooarcheology by Mass Spectrometry26,27). Thirty-seven of the worked objects and 31 of the unworked bone fragments were also sampled for radiocarbon dating using the compact AMS ECHoMICADAS, and for carbon and nitrogen stable isotope analysis (δ13C and δ15N). In this work, we assess the range of whale taxa whose products were utilized by the Magdalenian foragers and the chronology of this utilization, showing that these groups availed themselves of the carcasses of at least five species starting ca. 20-19 ka cal BP, with a peak time ca. The results of the stable isotope analysis, placed in the context of patterns of isotopic niche partitioning among contemporary whales, contribute to reconstructing whales' relative foraging behavior and past ocean ecology, showing that the distribution of the stable isotope ratios in Paleolithic whales is structured by taxa and overlaps to some extent with those of their modern counterparts, suggesting broadly similar feeding strategies and analogies with today's arctic water communities. Even though the Paleolithic seashore itself is no longer accessible, and the range of taxa identified here might not reflect the full range of species present in the Bay of Biscay at that period, the analysis of these whale bones brought inland by the hunter-gatherers opens a unique window into whale ecology and the marine environments in the northeastern Atlantic at that period, and on the timing and nature of their utilization by human groups. Of the 173 bone specimens (83 worked objects and 90 bone fragments) analyzed with ZooMS, all but four yielded a taxonomic identification, demonstrating the power of this approach to identify taxa on highly transformed and/or fragmented remains of late Pleistocene age. Of the 83 worked objects, 71 were confirmed as cetaceans, while 8 were identified as large terrestrial mammals and 4 did not yield a ZooMS identification, indicating that the macroscopic visual attribution was correct in 90% of the cases (71/79). The visual misidentification of 8 objects made of bone from large terrestrial mammals was due to their thoroughly porous aspect, normally a diagnostic feature of whale bones, but also present in some anatomical elements of certain terrestrial species (in this case mammoth, rhinoceros, reindeer and equids), and that can be misleading when dealing with small, fragmented objects. Of the 90 unworked bone fragments, the attribution as cetacean was confirmed for 60 bones (67%), with the other 30 being identified mostly as large land mammals, but also one seal. The higher error rate for Santa Catalina (33% vs. 10%) is a consequence of the fact that the visual selection of putative whale-bone fragments was more inclusive at this site (see “Methods” below). Overall, ZooMS analyses of 131 cetacean specimens reveal the presence of at least six cetacean taxa in the northeastern Atlantic during the Magdalenian (Fig. 1 and Supplementary Data 3): fin whale, Balaenoptera physalus (n = 65); sperm whale, Physeter macrocephalus (n = 32); gray whale, Eschrichtius robustus (n = 11); blue whale, Balaenoptera musculus (n = 2); one species of porpoise (harbor porpoise or Dall's porpoise, Phocoenidae, n = 1); and at least one species of Balaenid whale (Balaenidae), with 13 samples that can be attributed either to the North Atlantic right whale, Eubalaena glacialis, or to the bowhead whale, Balaena mysticetus (two species that are indistinguishable using ZooMS), both present in the North Atlantic. The remaining 7 samples yielded ZooMS spectra that could not be attributed to a precise cetacean taxon. Among the six cetacean taxa, only the sperm whale had previously been unambiguously documented in the Magdalenian record, through the presence of two carved teeth and several depictions on other portable objects from the Bay of Biscay region28. a worked objects; b unworked bone fragments. 1: blank, Tito Bustillo, sperm whale (#Hum1); 2: projectile point with massive base, Isturitz, blue whale (#15); 3: projectile point, Brassempouy, fin whale (#822); 4: possible foreshaft, Las Caldas, sperm whale (#982); 5: projectile point with massive base, Ermittia, gray whale (#968); 6: unidentified object, Saint-Michel, sperm whale (#799); 7–10: unworked fragments of fin whale bone, Santa Catalina (#SC B6 109 1085, SC B8 144 439, SC B8 124 1180, SC B6 135 24). Whale drawings courtesy of Uko Gorter. Pictures by AL and JMP. Of the 37 worked objects sampled for radiocarbon dating, 5 failed as a result of poor collagen preservation, while the remaining 32 yielded results (Fig. The two earliest dates are from the Cantabrian sites of Rascaño and El Juyo, ca. 20.2-19.6 and 19.6-19 ka cal BP, respectively. The two artifacts were found among assemblages attributed to the Cantabrian Lower Magdalenian (Rascaño IVb and El Juyo 8), and our results confirm this archeological association. All circles correspond to worked objects except site 8, which is the assemblage of unworked fragments of whale bone from Santa Catalina. The size of the circles is proportional to the numbers of remains per site, except in Santa Catalina, where all the unworked remains are considered as a single one so as not to overwrite the representation. Bathymetric data are from Natural Earth (www.naturalearthdata.com). The Sables des Landes region appears in yellow. 1: La Paloma, 2: Las Caldas, 3: La Viña, 4: Tito Bustillo, 5: El Pendo, 6: El Juyo, 7: El Rascaño, 8: Santa Catalina, 9: Iruroin, 10: Ermittia, 11: Urtiaga, 12: Isturitz, 13: Bourrouilla, 14: Duruthy, 15: Brassempouy, 16: Saint-Michel, 17: Tuc d'Audoubert, 18: Mas d'Azil, 19: La Vache, 20: Courbet, 21: La Madeleine, 22: Andernach. Only four worked objects, from three sites in the western Pyrenees and Cantabria (Duruthy, Isturitz and El Pendo), yielded dates in the 19-17.5 ka cal BP time range. At Santa Catalina, from the 31 fragments of unworked whale bone sampled for radiocarbon dating, all but one yielded reliable results (Fig. The two fragments attributed to the North Atlantic right whale or bowhead whale (one of which was dated three times) yielded identical 14C dates and might come from a single individual that died ca. The 28 other dates are all from fragments of fin whale bone and, with one exception, cluster between ca. 16-14.5 ka cal BP (thus after the 17.5-16 ka cal BP time range that yielded most of the dates on worked objects). The number of acquisition events represented by these 27 dates remains undetermined, and the degree of fragmentation of the bones precludes the calculation of an anatomical Minimum Number of Individuals; but the distribution of the dates appears compatible with a minimum of two individuals, one ca. 13.6–13.2 ka cal BP is the only possible evidence of a later episode of bone acquisition on a fin whale individual. Carbon (δ13C) and nitrogen (δ15N) stable isotope ratios were successfully measured in 55 whale bone samples and ranged from −16.8‰ to −11.6‰ in δ13C ratios and from 9.1‰ to 17.5‰ in δ15N ratios (Fig. In total, 8 samples failed collagen extraction, 2 samples failed IRMS analysis and for 7 samples the atomic C:N ratio was too high (ranging between 3.77 and 6.28) or the N peak was below 700 mV (Supplementary Data 1). The distribution of the stable isotope ratios is structured by taxa: fin whale samples are clustered at low δ15N values and (with one exception) intermediate values of δ13C; the single blue whale sample is nested among the fin whales; right/bowhead whale samples show low δ13C ratios and intermediate values of δ15N; gray whales display high δ13C ratios and intermediate values of δ15N; and sperm whales show elevated values both of δ15N values and δ13C (Table 1). The one sample where ZooMS failed to distinguish between right/bowhead and fin whales—due to the overall poor quality of the spectra and specifically the absence of a clear peptide marker at P2_ɑ2 292—is nested within the fin whale samples, strongly suggesting it might have corresponded to a fin whale (Fig. We found no evidence that these patterns of stable isotopic signatures across taxa are driven by differences in the radiocarbon age of samples: there was no significant relationship between conventional 14C age and the δ13C values (see Methods below, and Supplementary Fig. A comparison between the stable isotopic signatures of Paleolithic whale samples with modern counterparts shows in most cases an overlap (at least partial) between the two. The large majority of modern samples used for this comparison originate from the same oceanic basin (Atlantic) as our samples, except for gray whale (Pacific) (Supplementary Data 4). Still, on average we found higher δ13C and δ15N ratios in ancient fin whales, higher δ15N ratios in ancient sperm whales, and higher δ13C ratios in bowhead/right whales and in gray whales when compared with modern whales (Supplementary Discussion, and Supplementary Figs. A previous study of whale-bone objects from the Cantabrian coast suggested that, according to contextual evidence (both archeological and radiocarbon), the utilization of whale bone might have appeared in this region and in this archeological context ca. Our results confirm this archeological association and the earlier evidence of whale-bone working the Cantabrian region, and push back the chronology of its inception by at least one millennium (with the caveat of radiocarbon dating uncertainty; see Methods below). The whale bones used for tool manufacture do not necessarily come from fresh carcasses, but they are unlikely to have been more than a few decades old, given the hazards of bone preservation in the open air on the seashore29, and given that the low density of cetacean bones30 suggests that they will be particularly susceptible to density-mediated attrition31. Therefore, according to our results, the manufacturing of implements made of whale bone began ca. 20-19 ka cal BP on the southern shores of the Bay of Biscay. Since the visual identification of whale-bone objects from more ancient (Gravettian) archeological contexts remains uncertain32,33, these 20-19 ka cal BP dates represent to our knowledge the oldest evidence for the working of whale bone by hunter-gatherers. 17.5 and 16 ka cal BP, the amount of archeological evidence increases sharply and the geographic distribution of the objects expands, encompassing an area between Asturias and the central Pyrenees, perhaps with an extension to the north and east of the Aquitaine Basin (given their archeological context, the objects found at La Madeleine, Dordogne, and Courbet, Tarn, might indeed date to the same period35,36,37). This period, which archeologically corresponds to the Middle Magdalenian in the Cantabrian region and the Late Middle Magdalenian in southwest France38, thus appears as the peak time for the production and diffusion of objects made of whale bone. No chronological trend appears regarding the type of objects manufactured or the taxa used, and this period in general can be considered as representing the maximum diffusion and diversity of this whale-bone industry. This is in accordance with other categories of archeological evidence showing the existence of active exchange/diffusion networks between the Cantabrian coast and the central Pyrenees at that time (lithic raw materials, art, and types of tools14). Most of the objects made of whale bone are weapon elements (projectile points and foreshafts) typologically similar to the antler points that make up a large part of the Magdalenian hunting equipment39. From what can be observed on the finished objects, and on the few pieces of manufacturing waste documented37, the manufacturing techniques used to work whale bone into objects do not differ from those used for working antler. The choice of using whale bone to make one part of the weapon tips might be linked to the dimensions of the raw material, that make it possible to manufacture very long implements, and perhaps the specific mechanical properties of whale bone as compared to antler and to terrestrial bone14,24,40. The chronological distribution of the whale-bone objects ends abruptly after ca. The only worked object that might be attributed to a later date is the whale-bone foreshaft from the Upper Magdalenian site of Andernach, in Rhineland—an object that was found to be made of gray whale bone in this study, and that can be attributed to ca. 16–15.5 ka cal BP according to the AMS dates from the context in which it was discovered40. Only tentative explanations can be put forth to explain this rarefaction. 16–14 ka cal BP corresponds archeologically to the Upper Magdalenian—a cultural phase that, like the preceding one, yielded evidence for the exploitation of a number of coastal resources (such as mollusks10). The acquisition of fin whale carcasses at Santa Catalina after 16 ka cal BP, and the presence of a foreshaft made of gray whale bone at Andernach with contextual dates younger than 16 ka cal BP, both show that hunter-gatherer groups still had access to whale bone at that period. Therefore, the nearly complete lack of evidence for whale-bone working after 16 ka cal BP is not linked to an abandonment of coastal resources, or to a lack of suitable raw material or of technical skill. It could thus be either a purely cultural and technical choice (a disregard for whale bone as a raw material for tool making), or a question of archeological visibility linked to the interruption of the whale-bone diffusion networks. Indeed, since the whale-bone objects were probably manufactured on the seashore14,24, all objects of this type found at inland sites are imports. If the exchange and circulation dynamics that are responsible for the diffusion of these objects ceased to function, the whale-bone objects likely remained at the coastal sites, which are now submerged. There is currently no way to discriminate between those two hypotheses. The taxa identified in this study provide a picture of the rich biodiversity of cetaceans in the northeastern Atlantic during the Magdalenian period that may have been accessible to hunter-gatherers. With its highly productive waters and complex bathymetry, the Bay of Biscay remains today an area of high diversity and abundance of cetacean species41, but it may have been even more so in the past, given that multiple species were subsequently subject to intensive exploitation1. In addition, the climate in the Bay of Biscay was substantially cooler during the Magdalenian period, with sea ice likely present at least seasonally42, which means that the cetacean community may have more closely resembled that of today's arctic waters. Given the role large cetaceans play in marine ecosystem functionality, these differences in cetacean composition and abundance may have translated into non-negligible effects on local primary productivity, with cascading effects across the broader ecosystem43,44,45. Given their wide distribution, including cold waters46, it is not surprising that they were also present during the Magdalenian period, but they may have been more abundant prior to human exploitation1. Our results show that at least one balaenid species was found in the Bay of Biscay at the time, although the imprecision of the ZooMS ID does not allow for a distinction between right and bowhead whales; DNA analysis would be required to verify the species identities for these specimens (analyses which fell outside the scope of this study). Both species were heavily exploited in the western North Atlantic, with right whales now extirpated47 and bowheads nearly so48. Right whales were present in the Bay of Biscay until relatively recent times, where a coastal calving ground formed the basis of a Basque whaling industry from the 11th–18th centuries47. In the cooler Magdalenian waters42, it is more likely that right whales, if present, were using the area as summer feeding grounds. The sea-ice bound bowhead whales could have been a more likely species, with stable isotope data also providing some support in this direction (Supplementary Fig. It is not impossible that both species could have been present, in different seasons. The case of gray whales is more specific. Gray whales today have a strictly North Pacific distribution: they do not seem to have survived in the North Atlantic after the eighteenth century, having likely disappeared due to whaling despite the circumstances of their disappearance remaining poorly understood7,49. Previous evidence of their past presence in the North Atlantic dates either to the Holocene (10 to 0.25 ka cal BP) or to the middle of the Marine Isotope Stage 3 (MIS3), before 40 ka cal BP; with the absence of gray whales in the intervening period attributed to a drastic reduction or even extirpation of Atlantic gray whale50,51,52. Furthermore, molecular analyses revealed little genetic continuity between the late Pleistocene and the Holocene populations: most Atlantic Holocene specimens analyzed by Alter et al.50 are genetically closer to Holocene Pacific gray whales than to Pleistocene Atlantic specimens. This led these authors to suggest that the majority of the Atlantic Holocene population were the result of a second colonization event from the Pacific, after warming temperatures, sea-level rise, and decreases in sea ice permitted passage through the Bering Strait50. The 11 artifacts made of bones from gray whales presented here are evidence of the presence of this species in Atlantic waters at the end of MIS2, thus supporting a continuity between the Pleistocene and the Holocene populations. The stable isotope composition of marine vertebrates is controlled by their diet and habitat preferences53,54 and thus the stable isotope dataset presented in this study sheds light onto past whale ecology. In general, the stable isotopic signatures of Paleolithic whales overlap to some extent with those of their modern counterparts, suggesting broadly similar feeding strategies, while existing differences can be ascribed to variation in stable isotope baseline values, feeding ground locations and possibly trophic level (Supplementary Discussion). As in modern whales, ancient fin whales and blue whale samples have relatively low nitrogen stable isotope ratios in comparison with the other whale samples, consistent with these species' reliance on krill55,56, whereas gray whales display higher δ13C ratios, characteristic for their feeding behavior in benthic environments57, while sperm whales (the only toothed whale species in this dataset) show the most elevated δ15N values, reflecting a diet at a higher trophic level including large squid58 (Fig. The fact that bowhead/right whales (which feed mostly on copepods) have higher δ15N values than fin whales (again, also found in modern whales) is less straightforward to explain, but likely explained by feeding in different water masses, highlighting the complexity of interpreting isotopic signatures (Supplementary Discussion). This said, in most cases the overlap in stable isotope signatures between modern and ancient whale samples is only partial, with ancient whales tending to exhibit higher δ15N and/or δ13C values. These differences may reflect a shift in whale feeding preferences (e.g., if modern whales now target prey at lower trophic levels or warmer waters), large scale temporal changes in the stable isotope baseline59, environmental change, or a combination. Indeed, previous studies found rapid changes in whale feeding behavior (and in corresponding stable isotope signatures) as a result of environmental changes: for example, Jory et al.60 found that a reduction in biomass of artic krill coincided with a dietary niche widening in fin whales in Canadian waters, with 60% of individuals changing their foraging strategy from specialist to generalist feeders in order to reduce intraspecific competition. The stable isotope analysis complements the ZooMS results in showing a clear differentiation between whale species, and furthermore allowed us to identify as likely fin whale one specimen for which identification through ZooMS was imprecise (Fig. It also suggests a higher likelihood that the Balaenidae samples correspond to bowhead whales rather than right whales (Supplementary Fig. 101) although this is less certain. While the arctic conditions in the region would favor the bowhead hypothesis, the region could similarly have functioned as a northerly feeding ground for right whales. These results highlight the added value of combining multiple analytical methods to maximize the information obtained from archeological records of human-whale interactions2,53. With the exception of the object from Andernach, all our specimens come from karstic environments (cave and rockshelter sites) with good preservation of osseous material, including small elements. The relative abundance of the different whale taxa in our sample is thus unlikely to be affected by taphonomic factors such as differential preservation of bone tissues. However, these proportions cannot be used to infer the relative frequency of the whale taxa in the past, because they are biased by the anthropic choices of the Magdalenian hunter-gatherers and by our sampling strategy. For example, the scarcity of the harbor porpoise remains might reflect a lower value of the bones of this smaller species for tool manufacture. Conversely, the large proportion of fin whales in our sample is biased by our decision to systematically analyze the whale bones from Santa Catalina, where this taxon dominates. The worked objects and unworked bone fragments we analyzed are mainly large cetaceans that predominantly forage offshore or in areas with deep water close to shore like the Bay of Biscay—namely fin whale, sperm whale and blue whale. It is extremely unlikely that these species would have been accessible to hunter-gatherers from the European Pleistocene other than through passive acquisition methods, such as the opportunistic acquisition of natural strandings or drift whales. In addition, the list of identified taxa also includes species whose ecology brings them substantially closer to the coast, namely gray whales, right/bowhead whales and harbor porpoises46. Bowhead whales and gray whales are the large cetaceans with the longest recorded history of active whaling, going back at least three millennia in arctic ecosystems53,61. However, there is no evidence that European Pleistocene hunter-gatherers had the necessary technologies for hunting these species, such as seafaring62, or multibarbed points that could have been used as harpoons heads (barbed points appear in the local archeological record only after 16 ka cal BP39). Overall, then, the archeological evidence points towards an opportunistic acquisition of whale resources in the Bay of Biscay during the Magdalenian period, unlike in more recent periods where this same region became an area of active whaling7,47. Whales were likely familiar to coastal communities, and as such may well have played an important role in the Magdalenian cultural world. The Bay of Biscay's highly productive waters make it a cetacean hotspot today41, and it was likely already the case during the Pleistocene. Along the Spanish coast in particular, water depth increases dramatically just a few kilometers from the coastline, bringing species that are typically found offshore, such as fin whales63, unusually close. These species could have been observed from high points along the coastline spouting at a distance, and irregularly closer to shore as dead or moribund individuals. The more coastal right/bowhead whales, gray whales and harbor porpoise were likely even better known. Gray whales in particular are the most coastal of the whale species46, spending summer at low-depth high-latitude feeding grounds, then migrate hugging the coastline to warmer lower-latitude calving areas in sheltered low-depth lagoons. Whereas in today's climate the Bay of Biscay and the Mediterranean would more likely correspond to calving grounds, it is possible that during the cooler Upper Paleolithic climate they would have corresponded to feeding grounds49,64—habitat reconstructions indicate that adequate shallow shelf habitat was available there during the last glaciation50. In any case, they would have been conspicuously present at predictable seasons, and they are likely to have been part of the range of animals whose ecology would have been well known to Paleolithic hunter-gatherers. The whale remains analyzed in this study attest to two different utilization behaviors. The first one is the use of whale bones as raw material for the manufacture of implements; as suggested in previous studies, the choice of this raw material might have been motivated by the large dimensions of these bones, enabling the manufacture of longer implements, and maybe their specific mechanical properties14,24. The two main types of objects manufactured are projectile points and foreshafts, both relating to hunting equipment. Thirty-three projectile points were analyzed with ZooMS, and the results show that the range of species used for their manufacture is varied, with a relative majority of sperm whale (13 of the 31 points whose species could be identified, i.e., 42%; vs. 3-23% for each of the 5 other species identified). For the 11 foreshafts analyzed, the dominance of sperm whale is even more pronounced (8 of 11: 73%). It is uncertain whether this higher frequency compared to the other taxa reflects the relative abundance of sperm whales among stranded individuals or if it testifies to a preference for this species on the part of the Magdalenian carvers. The second hypothesis might be supported by the fact that, among the large cetaceans present in our sample, sperm whales are the only toothed species, displaying a highly characteristic, long, straight, toothed jawbone that may have been considered as a particularly desirable block of raw material. An interest in sperm whale teeth in the Magdalenian is also evidenced by the two carved specimens found at Las Caldas and Mas d'Azil10. Nevertheless, since the precise anatomical parts used for tool-making are so far indeterminate, this supposition remains speculative. The second utilization behavior, evidenced at Santa Catalina, is the transportation of unworked whale bones to habitation sites. The bones were found in the form of fragments, most of which are too small and fragmented to be identified anatomically; still, some of them can be attributed to ribs and to vertebrae (cf. fragments of vertebral disks), evidencing the presence of portions from the trunk, and several display percussion notches indicating anthropic breakage carried out at or outside the site. The transportation of these elements requires explanation, as the species involved (fin whale and right/bowhead whale) are large, and the site was by then 4-5 km from the coast and 70 m up a steep cliff65. Indeed, bones of large whales are bulky elements, usually abandoned on the spot where the whale is processed rather than carried as riders attached to the meat and blubber. Their use as implements at the site (ribs for building huts, vertebrae as stools or anvils, etc.) is implausible given their degree of fragmentation, and evidence for their use as raw material for toolmaking is very scarce. Only one fragment shows traces of scraping, and most of the elements are very spongy, while the whale-bone objects known from other sites show a denser osseous tissue indicating the use of denser parts of the skeleton. An alternative explanation is their use as bone fuel for hearths. The use of the bones of terrestrial mammals as fuel is frequently documented in Paleolithic contexts66, and activities linked to fire, including bone fuel, are intensive in level III of Santa Catalina (presence of charcoal, hearth-like structures, etc.67). However, at Santa Catalina the whale bones we analyzed are not specifically affected by burning, nor concentrated near the hearths; if the collecting of whale bone at this site has to do with fire, it could only be as a reserve for later use. Alternatively, since living animal whale bones are very rich in fat, these large spongy bones might have been brought to collect the oil by letting them drip, or by crushing them2. It is consistent with the strong interest of Paleolithic groups for fat (e.g., the nearly systematic breakage of the bones for marrow68), and with the fact that bone grease rendering is evidenced in the level III of Santa Catalina on the bones of ungulates69. Finally, although our archeological sample includes only bones and bone objects, their presence shows that hunter-gatherers also had access to other whale resources2. Whale skin, blubber and meat found numerous dietary and technical uses among populations of the past, and the transportation of (at least) whale skin to the habitat was already evidenced among European hunter-gatherers through the discovery of right whale barnacles at the sites of Las Caldas, Nerja and Cueva Victoria11. Furthermore, the identification of several species of mysticetes in our sample proves that Magdalenian hunter-gatherers also had access to baleen plates. The baleen of right and bowhead whales in particular have long been appreciated as a strong and flexible material with many practical uses in several populations2. In this case, the osseous material acts as a proxy for other resources with more rapid rates of degradation. This study provided a chronological and taxonomic characterization of Late Paleolithic whale-bone industry. Its inception on the southern shores of the Bay of Biscay takes place within a more general trend towards the intensification of the exploitation of coastal resources at that period and in that region. The acquisition of whale products seems limited to the opportunistic utilization of beached or drift whales, but likely included the use of many resources besides bones—including baleen, the availability of which is now demonstrated, and whale oil, as possibly evidenced by the presence of unworked and fragmented bones at Santa Catalina. For this reason, stranding events certainly affected the groups' mobility patterns by acting as attractive spots. In this perspective, although whales might not have been the primary impulse for the increased interest in the seashore during the Magdalenian, their presence undoubtedly contributed to this evolution and strengthened it, as was suggested for other regions70. Furthermore, the taxonomic diversity and chronological depth of the cetacean populations identified in this study—both previously undocumented for this region at this time period—attest to the richness of the marine and coastal ecosystem of the Bay of Biscay at the end of the Paleolithic, showing how much it represented a favorable milieu for human settlement. The location of each analyzed specimen can be found in Supplementary Data 1. Permission to access, study, borrow or export (when relevant), and sample the specimens was obtained from each institution, curator and/or excavation director following each local procedure (see detailed list in Acknowledgements section). A total of 185 worked objects visually identified as made of whale bone are known from 31 Magdalenian sites (Supplementary Data 2: 165 objects from previous studies14,24,40,71, plus 20 recently discovered through excavations or the reassessment of ancient collections). Minimally-invasive ZooMS analysis was tested on a subset of the samples, but proved to be unsuccessful (see below). In order to maintain the morphological integrity of the worked bone assemblage, a destructive ZooMS approach was applied to 83 of the 185 objects (i.e., 45%), with samples selected to ensure a representative selection of sites (n = 26) and of the objects' typology (these 83 objects include 3 ZooMS results previously published36,37). In addition, at Santa Catalina, 41 unworked bone fragments had previously (i.e., before this study) been attributed to cetaceans on a visual basis12. In this study, the bone assemblage of Santa Catalina was reassessed and sampled to include all fragments that could potentially be whale bone, even if the morphological identification remained somewhat uncertain. A total of 90 unworked bone fragments from Santa Catalina (the 41 previously identified + 49 new specimens) were subjected to ZooMS analysis, mostly from archeological level III. Half of the elements whose identification as whale bone was confirmed by ZooMS were subject to additional sampling for radiocarbon dating and stable isotope analysis: 37 of the 71 worked objects (52%), and 31 of the 60 bones from Santa Catalina (52%). The single object identified as made of porpoise bone through ZooMS analysis was a thin fragment of point, and permission was not granted to sample material sufficient for both ZooMS and 14C dating. Before any sampling, photogrammetry was used to produce a 3D model of each worked object and each unworked bone fragment in its entirety. This was done for conservation reasons, in order to preserve a digital copy of the specimens before they were morphologically altered by the sampling. Each bone selected for sampling was photographed using a Nikon reflex with 40 or 60 mm lens, a portable light tent with different lights to provide diffuse light, and a turntable with markers for metric projection and angular notations. The Adobe® Photoshop® software was used for post-processing (color adjustment and background masking) and the Agisoft® Metashape® software was used for the photogrammetric process. For each artifact, both a dense cloud (for accuracy studies) and a textured mesh (for exports and measurements) were produced. Depending on the shape of each element, sampling was done either: by cutting a piece from one of the extremities of the bone (using either a Stanley® mini hacksaw or a Dremel® rotary tool); by drilling with a Proxxon® Colt 2 pocket drill; or by coring using homemade diamond-coated core drills, 4–6 mm in diameter, mounted on a Moviluty® Minyflex® rotary tool. On average, 75.7 mg of bone was extracted from each specimen for ZooMS, with no significant difference between the worked objects and the unworked bone fragments. For radiocarbon dating only, the average mass of the sample was 95.3 mg on the worked objects, and 340.6 mg on the unworked fragments from Santa Catalina. ZooMS was first developed as a destructive analytical technique, requiring ~10–30 mg of bone or bone powder for analysis26. This project aimed to limit the destructive testing of unique, Magdalenian modified bone objects in order to preserve their morphology. As a result, for the bone objects, a minimally-invasive collagen sampling method was attempted which had been developed for analysis of historic parchments72, and which had previously proved successful on ca. 500 years old modified bone objects from Quebec, Canada73. The 90 unworked cetacean bone samples, and 83 modified bone objects, also underwent a destructive ZooMS protocol to achieve accurate taxonomic identifications (see results above). The bone samples were analyzed using the destructive ZooMS methods listed below, based on a modified protocol as described in Buckley et al.26. The bone objects were gently rubbed with a clean piece of PVC eraser; collagen is transferred from the object to the eraser crumbs through the triboelectric effect72. The collected eraser crumbs from each sample were incubated for four hours at 37 °C in 75 µL of 50 mM ammonium bicarbonate solution (NH4HCO3, pH 8.0, AmBic) and 0.4 µg of trypsin. Trypsin activity was terminated using 1 µL of 5% TFA solution, and peptides were purified using C18 ZipTip® pipette tips and eluted with 50 µL of conditioning solution (50% acetonitrile, 0.1% TFA). 1 µL of eluted peptides was then spotted onto a Bruker ground steel target plate, to which 1 µL of matrix (α-cyano-4-hydroxycinnamic acid) was added. Each sample was spotted in triplicate along with calibration standards, and run on a Bruker Ultraflex III MALDI-ToF-MS. Triplicate spectra were averaged and analyzed using mMass software74 and compared to a database of known collagen peptide masses (Supplementary Data 3)75,76,77,78. This was likely due to a combination of factors, including cetacean bone morphology and sample contamination. Due to the physical requirements of cetaceans, their bones often take on one of two forms depending on the element—either very dense and highly mineralized, or very porous and friable. Both bone conditions are problematic for minimally-invasive sampling methods as they result in reduced availability of surface collagen. Other studies applying the minimally-invasive eraser method also found limited success compared with destructive approaches73,79,80; as a result, a destructive and/or double extraction was performed on the samples. Subsamples of bone or bone powder, sometimes several per specimen and ranging from 20 to 70 mg, were demineralized in 250 µL of 0.6 M HCl (4 °C). The acid was removed and samples were rinsed three times in 200 µL of 50 mM AmBic. Samples were incubated in 100 µL of AmBic for one hour at 65 °C to gelatinize the collagen; then 0.4 µg of trypsin was added to 50 µL of supernatant and incubated overnight at 37 °C. Trypsin activity was terminated using 1 µL of 5% TFA solution, and peptides were purified using C18 ZipTip® pipette tips and eluted with 50 µL of conditioning solution. MALDI-TOF-MS was conducted as described above. Several samples analyzed using the minimally-invasive sampling method returned results that suggested sample contamination, likely as a result of a consolidant that had been added to the specimens post-excavation. While there is no mention in the reports for these excavations of the artifacts having been conserved with any type of consolidant, both minimally-invasive and destructive analyses of a number of these specimens consistently returned collagen identifications of cattle (Bos genus). Upon further inspection of the spectra, peptide markers that appeared to match those of various cetaceans could also be observed, however at a much lower intensity than the cattle fingerprint (Supplementary Fig. This mixed signal in the collagen spectra led us to believe that consolidant contamination was a likely issue, as collagen-based glues made from cattle (and other animals) were known to have been used in conservation practices during the early 1900's. A double extraction method was developed in an attempt to remove or reduce the signal from the contaminating consolidant (Supplementary Fig. In this method, bone or bone powder was immersed in 250 µL of 50 mM ammonium bicarbonate solution (NH4HCO3, pH 8.0, AmBic) and placed on a shaker at room temperature for 3 h. Samples were then centrifuged briefly then incubated at 65 °C for one hour to gelatinize. After incubation, the samples were centrifuged for one minute and the AmBic was transferred to a new eppendorf as the first extraction. Subsequently, 250 µL of 0.6 M HCl (4 °C) was added to each of the original eppendorfs containing the bone chips/powder, then vortexed and left to demineralize at room temperature for four hours. The acid was removed and samples were rinsed three times in 200 µL of 50 mM AmBic. 100 µL of AmBic was then added to the samples, followed by incubation for one hour at 65 °C to gelatinize the collagen for the second extraction. Both the first and second extraction underwent digestion, purification and MALDI-TOF-MS analysis as described above. While this modified protocol resulted in lower peptide recovery, it enabled us to enhance the collagen signal from the actual ‘host' animal (in most cases, cetacean collagen peptide markers) while also confirming our suspicions regarding the use of a cattle-based consolidant (Supplementary Fig. With the exception of 7 samples (Hum-1 to Hum-18) for which collagen extraction was carried out at the EvoAdapta facility and MALDI-TOF-MS analysis at the University of York, all samples were analyzed for ZooMS at the Institute of Environmental Science and Technology (ICTA-UAB) of the Universitat Autònoma de Barcelona. All glassware was cleaned using the following procedure: submerged in Decon 90 at 90 °C and cooled overnight, washed three times with DI water, submerged overnight in 10% HCl, washed three times with Milli-Q. After drying, the glassware was wrapped in aluminum foil to prevent dust contamination during storage and finally baked out in the oven at 450 °C for 5 h. The bone collagen extraction protocol based on Longin82 followed the following steps: Bone samples were first demineralized in 0.2 M HCl for several days, with mechanical and visual checks. The acid was renewed several times during this step. The samples were then rinsed three times with Milli-Q, submerged for 20 min in 0.1 M NaOH (with new NaOH being added for another 20 min if discoloration appeared), before being rinsed again three times with Milli-Q, submerged into 0.1 M HCl for 10 min, then rinsed again three times with Milli-Q. In the next steps, the samples were gelatinized in weak (pH 3) HCl at 90 °C until dissolution, filtered with glass filter units (mesh size 10–20 μm), frozen with liquid nitrogen and lyophilized in clean (baked out) vials. As anomalous peaks were observed in FTIR-ATR spectra of several samples, most likely indicating the presence of glue or consolidant, the XAD resin approach was applied to eliminate this type of contamination. This approach was developed by Stafford et al.83,84,85, is also described in van der Sluis et al.86. Lyophilized collagen samples were dissolved in 1 mL of (sub boiling distilled) 6 M HCl in 10 mL borosilicate tubes with PTFE lines caps before being hydrolyzed at 110 °C for 24 h. The hydrolysate was then passed through pre-conditioned XAD columns fitted with a filter frit at the bottom, filled with ±100 μl (circa 1 cm) of XAD 2 resin slurry and covered with the top filter frit. The latter was pushed down in order to remove air bubbles. The columns were then washed with 20 mL of 1 M HCl and preconditioned with 10 mL of 6 M HCl. After the sample hydrolysate had passed through, the column was washed with 1 bed volume of 6 M HCl in order to collect any amino acids in the void space and added to the collected sample. Samples were dried in small open beakers on the hotplate in the fume hood and were rinsed with Milli-Q to remove any leftover HCl by evaporation. The samples were then transferred in 200 μl (7-8 drops) of Milli-Q to combustion tubes using glass Pasteur pipettes, frozen and lyophilized. After adding 900 mbar pure O2, samples were combusted at 900 °C for a duration of 10 to 20 minutes in the presence of a baked out silver strip (10 mg) to remove contaminants, cleaned on the CO2 extraction line (water trap, NOx oven fitted with copper and silver fiber wool), then volume calculated. The CO2 gas sample was transferred to a fully-automated H2 reduction line using iron as a catalyst, where the vacuum reached 10-7 mbars. Samples were run alongside standards of oxalic acid and phthalic acid. Graphite targets were pressed and analyzed on the same day with the ECHo-MICADAS at the Laboratoire des Sciences du Climat et de l'Environnement (LSCE) in Gif-sur-Yvette, France. Data reduction was performed using the BATS software (version 47)87. The first scans were discarded to eliminate possible contamination of the target with ambient atmosphere between target pressing and AMS measurement. Radiocarbon ages were calculated from F14C88, which is corrected for background and isotopic fractionation. Measurement parameters such as 12C current and 13CH current were monitored during 14C measurement. Time, current and isobar corrections were made before validation. For each individual run, normalization, correction for fractionation and background corrections were applied by measuring the oxalic acid II NIST standard, its 13C/12C ratio and the chemical blanks. The standard deviation of the blanks is generally less than 10% but an overestimated 30% is imposed to the blank value in order to take into account a potential variability of the contaminant which could be added during the sample preparation. Six samples (Hum4, Hum5, Hum8, Hum9, Hum14 and Hum18) were chemically pretreated and prepared at the Higham Lab, University of Vienna, Austria. Sample pretreatment for these samples follows collagen extraction outlined in Brock et al.89. Three samples (Hum9, Hum14 and Hum18) were combusted offline, graphitized using an AGE graphitization unit, and measured at the VERA (Vienna Environmental Research Accelerator) AMS facility, University of Vienna. For details on aspects of target preparation and AMS measurement see Steier et al.90 and Golser & Kutschera91. Three other samples (Hum4, Hum5 and Hum8) were measured at the Keck AMS facility, University of California at Irvine. For details on aspects of target preparation and AMS measurement see Santos et al.92 and Beverly et al.93. These six samples have R-numbers and VERA or UCIAMS numbers instead of MUSE-numbers. As diagenetic alteration can affect stable isotope values as well as radiocarbon ages, only results from good quality collagen with an atomic C:N ratio between 2.9 and 3.6 ought to be used94. However, studies by van Klinken95 on archeological bones and Guiry and Szpak96 on modern bones have suggested that values outside the 3.0-3.3 range for mammals may result in distorted isotopic compositions due to contamination by non-collagenous carbon. We compared the values for whale species that agree with different atomic C:N ratio ranges (2.9–3.3 and 3.3–3.6) (Supplementary Fig. In these figures there is considerable overlap between samples with an atomic C:N ratio up to 3.3 and 3.6, only the gray whale samples show lower carbon stable isotope values (circa 1‰), although this could also be due to the small sample size. We believe most of the variation visible here is due to individual variation between the whales, rather than diagenetic alteration. Whales are exceptionally large (marine) mammals, thus resulting in a larger variation in stable isotope ratios in their bone collagen due to their longevity and slow bone turnover time. Samples with atomic C:N ratios between 2.9–3.6 were included in the discussion of the stable isotopes results. Additionally, stable isotope analysis was done on collagen, while 14C dating was performed on collagen after XAD resin treatment, which would have removed any, if present, remaining contamination. When examining the 14C ages against atomic C:N ratios from a single site (Santa Catalina) (Supplementary Fig. 97), there is no evidence of a correlation that might indicate contamination. Dating of marine animals needs to take into account the marine reservoir effect: given that oceans are depleted in 14C in relation to the atmosphere, conventional radiocarbon dates for marine organisms appear older than for coeval terrestrial individuals97. Some studies use sample pairing between coeval terrestrial and marine taxa to aid interpretations of reservoir ages at archeological sites98. This was not possible in our case, because our samples are all from Pleistocene cave and rockshelter sites with complex stratigraphy (in which each excavated layer is the result of an accumulation of multiple occupation episodes) and most of our samples are from ancient excavations with poor stratigraphic resolution. We thus relied on the correction for the reservoir effect implemented for marine samples in OxCal 4.4, which is based on the Marine20 calibration curve99. In addition, dating of marine organisms can be further refined by taking into account the fact that the marine reservoir effect varies across oceanic masses, being for example stronger (i.e., leading to seemingly older organisms) at higher latitudes and in deeper waters100. This deviation from the standard (globally averaged) marine reservoir effect is expressed as a DeltaR term. In the case of our whale specimens, however, it is not the coastal waters adjacent to the archeological sites that matter, but the water mass (or masses) where whales fed. The whale species identified in this dataset are very diverse in terms of their feeding ground locations and feeding depths. For example, previous studies estimated DeltaR values: between −168 and 504 ± 60 years for sperm whales (that feed in deep waters but can be found in either high latitudes or sub-tropical seas101); at 24 ± 58 years for bowhead whales (that forage in polar/sub-polar waters close to the sea surface102); and about 350 years for a fin whale (a species that forages at depths up to 400 meters and migrates vast distances103). Intra-specific variability in DeltaR values renders it difficult to obtain meaningful values per species. We have not found published estimates of DeltaR for gray whales, but the foraging strategy of this species (which feeds on invertebrates found in shallow sedimentary environments and migrates over very long distances) renders it particularly susceptible to intraspecific variation in DeltaR, especially as it will be dependent on the type of sediment (molluscs feeding on calcareous sediments appear over 2000 years older than suspension-feeding ones104). In addition to these difficulties, DeltaR values published in the literature from modern (Holocene) whales need to be approached with caution when applied to Pleistocene samples. Indeed, the spatial variation in the reservoir effect of water masses likely varied over time, in response to changes in ocean circulation. The foraging grounds of whales are also likely to have been different, given that in the Late Pleistocene, the study region was substantially cooler. For these reasons, we have opted for not attempting to apply a species-specific DeltaR based on information from modern whales, and instead use a single DeltaR as suitable as possible to the period and region of our samples. Choosing the most suitable reservoir correction required careful consideration of various parameters. Foote et al.105 used a DeltaR = 0 to correct for the reservoir effect in bowhead whales during the Pleistocene. However, this no longer seems suitable since the update of the Marine20 calibration curve, as there are large fluctuations in pre-Holocene reservoir ages97. Monge Soares et al.106 published a DeltaR = −117 ± 70 yr for the Late Pleistocene based on three samples pairs (shells and bones/charred wood) from the Cantabrian coast. Mangerud et al.107 showed that there is little difference between the DeltaR from whales and mollusks found at the same location. However, since the update of the Marine20 calibration curve, the DeltaR from Monge Soares et al. needs updating as well. Monge Soares et al. used three paired 14C ages for the calculation of their Late Pleistocene DeltaR (terrestrial 14C ages: 15 656 ± 75, 19 710 ± 120 and 23 040 ± 50 yr BP). Considering the ages of our samples, we decided it would be most suitable to use the two youngest paired 14C ages for an updated DeltaR, because the third sample pair was considerably older. These two paired Late Pleistocene 14C dates from Monge Soares et al. were used in the online application108 to calculate updated DeltaR values. These new DeltaR values are statistically the same (0.037 (χ2:0.05 = 3.84)). The weighted mean of these DeltaR and standard deviation are calculated according to: http://calib.org/marine/AverageDeltaR.html, which gives a new Delta R of -448 ± 39 yr. The calibration was done in OxCal 4.4 (Supplementary Figs. Additionally, using all North Atlantic whales (various species) in the Marine Reservoir Correction Database we get a DeltaR of −168 ± 53, compared with the DeltaR of −117 ± 70 from Monge Soares et al. for the Holocene. This means that the latter slightly overestimate the reservoir age, making samples slightly younger. This may not be the case for the Late Pleistocene samples but it does strengthen our idea that the material used for paired dating in this region by Monge Soares et al. is not that far off from whales themselves. However, the DeltaR used here is not optimal for two species in this dataset, namely sperm whales due to their large variation in ΔR (between –168 and 504 ( ± 60) 14C yr101) and gray whales due to feeding in shallow waters where they might ingest calcareous sediment with different 14C ages104. Heaton et al.100 propose a specific calibration of marine samples from polar regions, essentially calculating a range between a minimum and maximum DeltaR. The minimum is based on the Marine20 curve and the maximum is based on the latitude of the sample and the Large Scale Geostrophic Ocean General Circulation Model (LSG OGCM), which is suggested to use for samples from outside ∼40°S–40°N during glacial periods. Our samples are found between of 43–44 °N, although the whales could of course have been feeding at higher latitudes. However, we do not have the exact location. We state that our results provide evidence of whale bone working from 20ka BP, which is based on two fin whales. If the absolute max DeltaR would be applied, these 2 fin whales would be between 2000-1500-1000 years younger (Fig. 3 in Heaton et al.100, although there is a substantial variation between 71 °N, 88 °N and 53 degrees °N, and again we do not know where these animals were feeding). It would be more realistic to assume sea ice most likely covered a large part of the North Atlantic waters, extending much further south than it does today. If so, most whales would not have been feeding too far north due to sea ice coverage (unlikely reaching the 71 and 88 °N, except for beluga's and bowheads). Additionally, modern fin whales feed in various locations, have summer and winter feeding grounds, and can also adapt their feeding strategy to minimize interspecific competition, suggesting that the maximum value would not be a realistic assumption. Combining all of these assumptions, it is plausible that the maximum DeltaR was ~800 yr, although its value remains extremely uncertain. Given that the results we present here reconstruct the chronology of whale acquisition using very broad time slices of 1.5 millennia (Fig. 2), it is unlikely that our conclusions are significantly affected by an uncertainty on DeltaR in the range of a few centuries. It does however mean that caution is needed when interpreting individual radiocarbon dating results. These can be improved in the future as more information becomes available regarding suitable corrections for the marine reservoir effect. Bone collagen samples (320–380 μg) were weighed into tin capsules (5 × 8 mm) and analyzed with a Thermo Scientific EA Flash 2000 coupled to a Delta V Advantage isotopic mass spectrometer in the Service de Spectrométrie de Masse Isotopique (SSMIM) in the Muséum national d'Histoire naturelle, Paris, France. Isotopic values of all samples were measured relative to the laboratory standard alanine, which has a reproducibility of 0.3 wt% for N and 0.6 wt% for C. The δ13C and δ15N values are reported relative to the VPDB and AIR, respectively. Three primary standards were used to calibrate the Alanine for δ15N air: IAEA USGS-25 (ammonium sulfate), IAEA N1 (ammonium sulfate) and IAEA N2 (ammonium sulfate), while one primary standard (from IAEA) was used to calibrate the Alanine for δ13C V-PDB: IAEA 600 (caffeine). Analytical precision is ± 0.2‰ for both δ13C and δ15N values. Five samples were run in duplicate and the difference fell between 0-0.3‰ for δ13C ratios, and 0–0.34‰ for δ15N ratios. Subsamples (0.2–0.3 mg) of extracted bone collagen from six samples (Hum4, Hum5, Hum8, Hum9, Hum14 and Hum18) were taken for the analysis of isotope ratios of carbon and nitrogen by elemental analyzer-isotope ratio mass spectrometry (EA-IRMS; Thermo Scientific EA-Isolink with a Flash 2000 coupled to a Delta V Advantage isotope ratio mass spectrometer) in the Silver Laboratory (Large-Instrument Facility for Advanced Isotope Research) at the Center of Microbiology and Environmental Systems Science of the University of Vienna. Stable isotope values of all samples are measured relative to the laboratory standard alanine, which has a reproducibility of 0.07 wt% for N and 0.1 wt% for C. δ13C and δ15N values are reported relative to the VPDB and AIR standard, respectively, and are measured with an analytical precision of ± 0.1‰ for both δ13C and δ15N values. Acceptable stable isotope ratios have an atomic C:N ratio that falls between 2.9–3.694. The difference in δ13C ratios between the balaenidae, fin and gray whales suggests these whales were feeding in different bodies of water, which in turn may have affected their radiocarbon ages. To investigate this, we plotted the conventional 14C age against the δ13C ratio from these whale bones (Supplementary Fig. We found no correlation between these two variables (R2 = 8 × 10−5). The ages of the three whale groups studied broadly overlap: balaenidae (right/bowhead whales; n = 8) are dated from 16–16.5ka to 17.1–17.8ka cal BP; fin whales (n = 31) are dated from 13.2–13.5ka to 19.5–20ka cal BP; gray whales (n = 4) are dated from 16.3–16.9ka to 17.4–18ka cal BP. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The location of each analyzed specimen can be found in Supplementary Data 1. 92 through 95, can be found in Supplementary Data 1. 96 through 102 can be found in Supplementary Data 4. Collagen peptide mass spectra have been publicly deposited in the Nakala Repository under following link: https://doi.org/10.34847/nkl.dbbfl6fw. Marine mammal populations: reconstructing historical abundances at the global scale. Charpentier, A. et al. What's in a whale bone? combining new analytical methods, ecology and history to shed light on ancient human-whale interactions. Gusinde, M. et al. Los Indios de Tierra del Fuego: los Yamana, Buenos Aires, CAEA. Turner, L. M. et al. An Aleutian Ethnography, Fairbanks, University of Alaska Press (2008). Lee, S. M. et al. Chasseurs de baleines: la frise de Bangudae, Corée du Sud, Paris, Errance. van den Hurk, Y. et al. The prelude to industrial whaling: identifying the targets of ancient European whaling using zooarchaeology and collagen mass-peptide fingerprinting. Nabais, M., Soares, R. & van den Hurk, Y. The Zooarchaeology of ancient whaling practices in Portugal: A review and a new Roman Republican contribution at Castelo Velho de Safara. Erlandson, J. M. The archaeology of aquatic adaptations: paradigms for a new millennium. Pétillon, J.-M. et al. Life on the shore of the Bay of Biscay in the Late Upper Paleolithic: towards a new paradigm. De la fonction des habitats à l'organisation de l'espace littoral, Paris, Société préhistorique française, 23–36 (2016). Álvarez-Fernández, E. et al. Mise à jour des recherches sur les chasseurs-cueilleurs côtiers dans le sud-est de la péninsule ibérique durant le Magdalénien: les restes archéofauniques d'origine marine. In: Costamagno, S., Boudadi-Maligne, M., Daujeard, C., Fernandez, P., Stoetzel, E. ), Sociétés humaines et environnements dans la zone circumméditerranéenne du Pléistocène au début de l'Holocène, Les Eyzies-de-Tayac-Sireuil, Musée national de Préhistoire (Paléo hors-série), 50–63 (2022). Castaños, P. et al. Estudio de los macromamíferos del yacimiento de Santa Catalina. Restos vegetales, animales y humanos, Bilbao, Kobie serie BAI 4, 331–360 (2014). Lefebvre, A. et al. Interconnected Magdalenian societies as revealed by the circulation of whale bone artefacts in the Pyreneo-Cantabrian region. Earliest known use of marine resources by Neanderthals. Gutiérrez-Zugasti, I. et al. A chrono-cultural reassessment of the levels VI–XIV from El Cuco rock-shelter: a new sequence for the Late Middle Paleolithic in the Cantabrian region (northern Iberia). Zilhão, J. et al. Last Interglacial Iberian Neandertals as fisher-hunter-gatherers. Álvarez-Fernández, E. Marine resource exploitation during the Middle and Early Upper Paleolithic in. Fano, M. A., Gutiérrez-Zugasti, I., Álvarez-Fernández, E., Fernández-García, R. Late Glacial and Postglacial use of marine resources in the Bay of Biscay, North Spain. In: Shell energy: Mollusc shells as coastal resources, Bailey, G. N., Hardy, K., Camara, A. Aura, J. E. et al. Palaeolithic - Epipalaeolithic Seapeople of the Southern Iberian coast (Spain): an overview. De la fonction des habitats à l'organisation de l'espace littoral, Paris, Société préhistorique française, 69–92 (2016). Changing Coastlines and the Antiquity of Coastal Settlement, New York, Springer, (2011). De la fonction des habitats à l'organisation de l'espace littoral, Paris, Société préhistorique française, 425 (2016). Circulation of whale-bone artifacts in the northern Pyrenees during the Late Upper Paleolithic. Rodrigues, A. S. L., Horwitz, L. K., Monsarrat, S. & Charpentier, A. Ancient whale exploitation in the Mediterranean: species matters. & Wilson, J. C. Species identification by analysis of bone collagen using matrix-assisted laser desorption/ionisation time-of-flight mass spectrometry. Collins, M. et al. ZooMS: the collagen barcode and fingerprints. Pétillon, J.-M. et al. Échos de l'océan: phoques et baleines en Europe au Paléolithique récent. In: Cattelain, P., Gillard, M., Smolderen A. Les mammifères au temps de Cro-Magnon en Europe, Treignes, Cedarc, 335–354 (2018). Christensen, M. et al. L'industrie osseuse des chasseurs-cueilleurs: le cas des nomades marins de Patagonie et Terre de Feu, Punta Arenas, Universidad de Magallanes. Gray, N. M., Kainec, K., Madar, S., Tomko, L. & Wolfe, S. Sink or swim? bone density as a mechanism for buoyancy control in early Cetaceans. In: Pokines, J. T., L'Abbe, E. N., Symes, S. A. ), Manual of forensic taphonomy, Boca Raton, CRC Press (2021). Müller, K. & Reiche, I. Differentiation of archaeological ivory and bone materials by micro- PIXE/PIGE with emphasis on two Upper Palaeolithic key sites: Abri Pataud and Isturitz, France. Reiche, I., Müller, K., Staude, A., Goebbels, J. & Riesenmeier, H. Synchrotron radiation and laboratory micro X-ray computed tomography—useful tools for the material identification of prehistoric objects made of ivory, bone or antler. Waelbroeck, C. et al. Sea-level and deepwater temperature changes derived from benthic foraminifera isotopic records. ), Animaux rares, gibiers inattendus: reflets de la biodiversité, Les Eyzies-de-Tayac - Sireuil, Musée national de préhistoire, 92–96 (2019). Pétillon, J.-M. et al. A gray whale in Magdalenian perigord. Species identification of a bone projectile point from La Madeleine (Dordogne, France) using collagen fingerprinting. The Magdalenian osseous industry from Courbet cave (Penne, Tarn, France) in the British Museum collections. Langlais, M. et al. Des segments chronoculturels au modèle archéo-stratigraphique du Magdalénien dans le Sud-Ouest français (21 000-16 000 cal. Magdalenian chrono-stratigraphic correlations and cultural connections between Cantabrian Spain and Southwest France and beyond, Paris, Société préhistorique française, 109–135 (2020). Technological evolution of hunting implements among Pleistocene hunter-gatherers: osseous projectile points in the Middle and Upper Magdalenian (19–14 ky cal BP). Langley, M. C. & Street, M. Long range inland-coastal networks during the Late Magdalenian: evidence for individual acquisition of marine resources at Andernach-Martinsberg. Seasonal distribution and abundance of cetaceans within French waters - Part II: The Bay of Biscay and the English Channel. Roman, J. et al. Whales as marine ecosystem engineers. Global nutrient transport in a world of giants. Gilbert, L., Jeanniard-du-Dot, T., Authier, M., Chouvelon, T. & Spitz, J. Composition of cetacean communities worldwide shapes their contribution to ocean nutrient cycling. Wilson, D. E., Mittermeier, R. A. Handbook of the Mammals of the World: Sea Mammals, Barcelona, Lynx Edicions, 4, 614 (2014). Reeves, R. R., Smith, T. M., Josephson, E. A. Near-annihilation of a species: right whaling in the North Atlantic. ), The Urban Whale: North Atlantic Right Whales at the Crossroads, Harvard, Harvard University Press, 39–74 (2007). Allen, R. C. & Keay, I. Bowhead Whales in the Eastern Arctic, 1611–1911: population reconstruction with historical whaling records. Rodrigues, A. S. L. et al. Forgotten Mediterranean calving grounds of grey and North Atlantic right whales: evidence from Roman archaeological records. Climate impacts on transocean dispersal and habitat in gray whales from the Pleistocene to 2100. Hufthammer, A. K., Arntsen, L., Kitchener, A. C. & Buckley, M. Grey whale (Eschrichtius robustus) in Norwegian waters 2000 years ago. Garrison, E. G., Morgan, G. S., Mcgrath, K., Speller, C. & Cherkinsky, A. Recent dating of extinct Atlantic gray whale fossils (Eschrichtius robustus), Georgia Bight and Florida, western Atlantic Ocean. Buss, D. L. et al. Archaeological evidence of resource utilisation of the great whales over the past two millennia: a systematic review protocol. García-Vernet, R., Borrell, A., Víkingsson, G., Halldórsson, S. D. & Aguilar, A. Ecological niche partitioning between baleen whales inhabiting Icelandic waters. Borrell, A., Abad-Oliva, N., Gómez-Campos, E., Giménez, J. Discrimination of stable isotopes in fin whale tissues and application to diet assessment in cetaceans. Blevins, C. et al. Sex- and age-specific migratory strategies of blue whales in the northeast Pacific. Feeding ecology of gray whales inferred from stable-carbon and nitrogen isotopic analysis of baleen plates. Ruiz-Cooley, R. I., Gendron, D., Aguíñiga, S., Mesnick, S. & Carriquiry, J. D. Trophic relationships between sperm whales and jumbo squid using stable isotopes of C and N. Mar. Reade, H. et al. Nitrogen palaeo-isoscapes: changing spatial gradients of faunal δ 15 N in late Pleistocene and early Holocene Europe. Individual and population dietary specialization decline in fin whales during a period of ecosystem shift. Seersholm, F. V. et al. DNA evidence of bowhead whale exploitation by Greenlandic Paleo-Inuit 4000 years ago. Philippe, M. Un état des connaissances sur la navigation préhistorique en Europe atlantique. MacLeod, K. et al. Distribution and abundance of fin whales and other baleen whales in the European Atlantic. Speller, C. et al. Barcoding the largest animals on Earth: Ongoing challenges and molecular solutions in the taxonomic identification of ancient Cetaceans. Berganza E., Arribas J. L. El entorno físico de las ocupaciones de Santa Catalina. Restos vegetales, animales y humanos, Bilbao, Kobie serie BAI 4, 367–377 (2014). Costamagno, S., Théry-Parisot, I., Castel, J.-C., Brugal, J.-P. Combustible ou non? analyse factorielle et modèles explicatifs sur des ossements brûlés paléolithiques. In: Théry-Parisot, I., Costamagno, S., Henry, A. ), Gestion des combustibles au Paléolithique et au Mésolithique: nouveaux outils, nouvelles interprétations, Oxford, Archaeopress, 69–84 (2009). Berganza E., Arribas J. L. Estructuras de combustión. Restos vegetales, animales y humanos, Bilbao, Kobie serie BAI 4, 33–48 (2014). Costamagno, S., Rigaud, J.-P. L'exploitation de la graisse au Paléolithique. ), Histoire de l'alimentation humaine: entre choix et contraintes, Paris, CTHS, 134–152 (2014). ), La cueva de Santa Catalina (Lekeitio, Bizkaia). Industrias líticas y óseas, colorantes y arte mobiliar, Bilbao, Kobie serie BAI 10, 267–285. Hussain, S. T. et al. Was the Late Glacial human occupation of northernmost Europe facilitated by whales? New data and perspectives on lithic technology and the paleoecology of the Vendsyssel area, Northern Jutland, Denmark. Poplin, F. Le bison en os de baleine d'Isturitz (Pyrénées-Atlantiques). Animal origin of 13th-century uterine vellum revealed using noninvasive peptide fingerprinting. McGrath, K. et al. Identifying archaeological bone via non-destructive ZooMS and the materiality of symbolic expression: Examples from Iroquoian bone points. & Kodíček, M. mMass data miner: an open source alternative for mass spectrometric data analysis. Buckley, M. & Collins, M. Collagen survival and its use for species identification in Holocene-lower Pleistocene bone fragments from British archaeological and paleontological sites. Kirby, D. P., Buckley, M., Promise, E., Trauger, S. A. & Holdcraft, T. R. Identification of collagen-based materials in cultural heritage. Species identification of archaeological marine mammals using collagen fingerprinting. Ancient proteins resolve the evolutionary history of Darwin's South American ungulates. Coutu, A. N. et al. Palaeoproteomics confirm earliest domesticated sheep in southern Africa ca. Evans, Z., Paskulin, L., Rahemtulla, F. & Speller, C. F. A comparison of minimally-invasive sampling techniques for ZooMS analysis of bone artifacts. van der Sluis, L. G. et al. Identification and tentative removal of collagen glue in Palaeolithic worked bone objects: implications for ZooMS and radiocarbon dating. Longin, R. New method of collagen extraction for radiocarbon dating. Stafford, T. W. Jr, Duhamel, R. C., Haynes, C. V. Jr & Brendel, K. Isolation of proline and hydroxyproIine from fossil bone. Stafford, T. W. Jr, Jull, A. J. T., Brendel, K., Duhamel, R. C. & Donahue, D. Study of bone radiocarbon dating accuracy at the University of Arizona NSF accelerator facility for radioisotope analysis. Stafford, T. W. Jr., Brendel, K. & Duhamel, R. C. Radiocarbon, 13C and 15N analysis of fossil bone: removal of humates with XAD-2 resin. van der Sluis, L. G., Zazzo, A., Tombret, O., Thil, F. & Pétillon, J.-M. Testing the use of XAD resin to remove synthetic contamination from archaeological bone prior to radiocarbon dating. Bats: A new tool for AMS data reduction. Brock et al. Current pretreatment methods for AMS radiocarbon dating at the Oxford radiocarbon accelerator unit (ORAU). Steier, P., Liebl, J., Kutschera, W., Wild, E. & Golser, R. Preparation methods of μg carbon samples for 14C Measurements. Golser, R. & Kutschera, W. Twenty years of VERA: toward a universal facility for accelerator mass spectrometry. Santos, G. M. et al. AMS 14 C sample preparation at the KCCAMS/UCI facility: status report and performance of small samples. The keck carbon cycle AMS laboratory, University of California, Irvine: status report. DeNiro, M. J. Postmortem preservation and alteration of in vivo bone collagen isotope ratios in relation to palaeodietary reconstruction. Bone collagen quality indicators for palaeodietary and radiocarbon measurements. & Szpak, P. Quality control for modern bone collagen stable carbon and nitrogen isotope measurements. Heaton, T. J. et al. Marine20 – The marine radiocarbon age calibration curve (0-55,000 cal BP). Dury, J. P. et al. Species-specific reservoir effect estimates: a case study of archaeological marine samples from the Bering Strait. Bronk Ramsey, C. Bayesian analysis of radiocarbon dates. Heaton, T. J. et al. Marine radiocarbon calibration in polar regions: a simple approximate approach using Marine20. Beck, L. et al. Marine reservoir effect of spermaceti, a wax obtained from the head of the sperm whale: a first estimation from museum specimens. Revised marine reservoir offset (Δ R) values for molluscs and marine mammals from Arctic North America. Birkenmajer, K. & Olsson, U. Radiocarbon dating of whale bones from the 17th century whaling sites at Gashamna, Hornsund, South Spitsbergen. England, J., Dyke, A. S., Coulthard, R. D., McNeely, R. & Aitken, A. The exaggerated radiocarbon age of deposit-feeding molluscs in calcareous environments. Ancient DNA reveals that bowhead whale lineages survived Late Pleistocene climate change and habitat shifts. Monge Soares, A. M. et al. Marine radiocarbon reservoir effect in Late Pleistocene and early Holocene coastal waters off Northern Iberia. Mangerud, J., Bondevik, S., Gulliksen, S., Hufthammer, A. K. & Høisæter, T. Marine 14C reservoir ages for 19th century whales and molluscs from the North Atlantic. An online application for ΔR calculation. We thank the people and institutions that granted access and sampling permissions for the material studied: I. Alonso Garcia and M. A. Pedregal Montes (Museo arqueológico de Asturias); F. Bοn, L. Bruxelles, M. Jarry and C. Pallier; J. Cook and C. Lucas (British Museum); J. Darricau (site d'Isturitz) and C. Normand; L. Ducamp (Maison de la dame, Brassempouy); P. Fatás Monforte (Museo nacional y centro de investigación de Altamira); G. Fleury (Muséum d'histoire naturelle de Toulouse); A. Fort, R. Nespoulet and M. Lebon (Musée de l'Homme); N. Fourment (Musée national de Préhistoire); S. Fraile Gracia (Museo nacional de ciencias naturales); E. Galán (Museo arqueológico nacional); I. García Camino (Arkeologi museoa in Bilbao); D. Haro-Gabay (abbaye d'Arthous); the musée de Lespugue; V. Merlin-Anglade (Musée d'art et d'archéologie du Périgord); R. Ontañón (Museo de Prehistoria y Arqueología de Cantabria); J. Primault; L. Rodriguez (Musée de Borda); C. Saint-Martin and P. Alard (musée du Mas d'Azil); S. San Jose Santamarta (Gordailua center); C. Schwab (Musée d'archéologie nationale); A. Simonet; M. Street (RGZM) and C. Langley; and É. Tartar. Thanks to L. Agudo Pérez and A. Cicero Cabañas for their help with the ZooMS analyses, to C. Merlet for his help with the material from Duruthy, to P. Reimer for her help on calculating the new DeltaR for the reservoir correction, and to D. Fiorillo (SSMIM, Service de Spectrométrie de Masse Isotopique du Muséum) for analyzing samples on the IRMS. We thank the York Center for Excellence in Mass Spectrometry (University of York) and the Laboratori de Proteòmica CSIC (Universitat Autònoma de Barcelona), for allowing access to their MALDI-TOF-MS. This work contributes to the ICTA-UAB María de Maeztu Program for Units of Excellence of the Spanish Ministry of Science and Innovation (CEX2019-000940-M), and EarlyFoods (SGR-Cat 2021, 00527). This study was funded by projects HumAntler (PCI2021-122053-2 B) (A.L. This work is dedicated to É. Campmas and G. Marchand. These authors contributed equally: Krista McGrath, Laura G. van der Sluis. These authors jointly supervised this work: Antoine Zazzo, Jean-Marc Pétillon. Department of Prehistory and Institute of Environmental Science and Technology (ICTA-UAB), Universitat Autònoma de Barcelona, Bellaterra, Spain BioArchéologie, Interactions Sociétés Environnements (BioArch), UMR 7209, Muséum national d'Histoire naturelle, CNRS, Paris, France Laura G. van der Sluis, Olivier Tombret & Antoine Zazzo Human Evolution and Archaeological Sciences, Vienna, Austria Alexandre Lefebvre, Ana B. Marín-Arroyo & Leire Torres-Iglesias De la Préhistoire à l'Actuel: Culture, Environnement et Anthropologie (PACEA), UMR 5199, CNRS, Université de Bordeaux, Pessac, France GIR PREHUSAL, Universidad de Salamanca, Facultad de Geografía e Historia, Departamento de Prehistoria, Historia Antigua y Arqueología, Salamanca, Spain Travaux et Recherches Archéologiques sur les Cultures, les Espaces et les Sociétés (TRACES) UMR 5608, CNRS, Université Toulouse Jean Jaurès, Toulouse, France François Baleux, Morgane Dachary & Jean-Marc Pétillon Sociedad de Ciencias Aranzadi, Donostia-San Sebastian, Spain Office du patrimoine et de l'archéologie du canton de Neuchâtel, section Archéologie, Laténium, Hauterive, Switzerland Elsa Duarte Matías & Marco de la Rasilla Vives Université de Franche-Comté, UMR 6249 Chrono-environnement, Besançon, France Laboratoire des Sciences du Climat et de l'Environnement (LSCE/IPSL), UMR 8212, CEA-CNRS-UVSQ, Université Paris Saclay, Gif-sur-Yvette, France Section for Molecular Ecology and Evolution, Globe Institute, University of Copenhagen, Øster Farimagsgade 5, Copenhagen, Denmark You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar All authors revised and approved the manuscript. The authors declare no competing interests. Nature Communications thanks Danielle Buss, Ana Valenzuela-Toro, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. McGrath, K., van der Sluis, L.G., Lefebvre, A. et al. Late Paleolithic whale bone tools reveal human and whale ecology in the Bay of Biscay. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03715-6'>Feasibility of multiomics tumor profiling for guiding treatment of melanoma</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-27 09:44:38
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). There is limited evidence supporting the feasibility of using omics and functional technologies to inform treatment decisions. Here we present results from a cohort of 116 melanoma patients in the prospective, multicentric observational Tumor Profiler (TuPro) precision oncology project. Nine independent technologies, mostly at single-cell level, were used to analyze 126 patient samples, generating up to 500 Gb of data per sample (40,000 potential markers) within 4 weeks. Among established and experimental markers, the molecular tumor board selected 54 to inform its treatment recommendations. The objective response rate in difficult-to-treat palliative, beyond SOC patients (n = 37) was 38%, with a disease control rate of 54%. Progression-free survival of patients with TuPro-informed therapy decisions was 6.04 months, (95% confidence interval, 3.75–12.06) and 5.35 months (95% confidence interval, 2.89–12.06) in ≥third therapy lines. The proof-of-concept TuPro project demonstrated the feasibility and relevance of omics-based tumor profiling to support data-guided clinical decision-making. Clinical decision-making in oncology is based on data derived from clinical trials that provide standardized measures of efficacy and safety. These data are evaluated by expert panels and incorporated into clinical practice guidelines of the main medical societies, which define the evidence-based standard of care (SOC) and guide diagnostic and treatment procedures through a process of shared decision-making between physicians and patients1,2. Adhering to these guidelines has been shown to improve survival rates and other outcome measures in patients with cancer3,4,5. In the case of malignant melanoma, prognosis has improved dramatically over the past decades due to implementation of efficacious SOC treatments6,7. However, many melanoma patients experience recurrence or progression of their disease. Evidence-based treatment options for these patients are limited and clinical management presents a major challenge that often falls outside the scope of clinical guidelines. The advent of reliable and affordable functional and omics technologies, including analyses at single-cell resolution, has rapidly transformed the field of oncology8,9,10,11. Clinical trials and practice guidelines are increasingly incorporating multidimensional data that enable an unprecedented characterization of molecular drivers and therapeutic vulnerabilities specific to the individual patient's cancer, paving the way for personalized precision treatments12,13. Moreover, omics-based patient data can guide enrollment of patients in suitable clinical trials14. To integrate molecular and functional data into clinical decision-making, institutional and national multidisciplinary molecular tumor boards (MTBs) have been established15. Previously, we and others have reported on the development of pipelines that enable the use of molecular data by MTBs10,16,17. Despite the promise of precision medicine, there is limited evidence supporting feasibility and utility of using multiomics and single-cell technologies to guide selection of treatments for patients with cancer. The Tumor Profiler (TuPro) project is a multicentric, prospective, nonrandomized observational project designed to assess the relevance of functional, single-cell and bulk omics readouts to MTB decisions18. A fast diagnostic loop was implemented, enabling an overall turnaround time of 4 weeks from biopsy to reporting in the MTB with single-cell genomics (scDNA-seq)19 and transcriptomics (scRNA-seq)20, targeted proteomics (imaging mass cytometry (IMC)21, cytometry by time of flight (CyTOF)22), proteotyping (using data-independent acquisition (DIA)23,24,25), drug phenotyping (Pharmacoscopy11,26, iterative indirect immunofluorescence imaging (4iDRP)27), targeted next-generation DNA sequencing (NGS) and digital pathology (DigiPath). The selection of these technologies was based on their capability to provide a comprehensive molecular portrait of each tumor with potential clinical relevance. As a result, up to 500 Gb of data were generated and evaluated for each sample, corresponding to over 40,000 potential markers for therapy decisions. The molecular data were integrated with additional clinical and diagnostic information available for each patient during the MTB evaluation. Markers used for treatment recommendations, administered therapies, objective responses and outcomes were captured and analyzed within the project framework. A total of 116 patients with any subtype of melanoma participated in the project, contributing 126 biopsies (including ten longitudinal samples) (Fig. The enrollment period spanned 21 months from January 2019 to November 2020. The demographical, clinical and pathological features of eligible patients were recorded in a good-clinical-practice-compliant database (Extended Data Table 1). Among all biopsy specimens, 62% were obtained from patients who had been treated with at least one therapeutic regime; 43% of these patients had undergone two or more lines of treatment (Extended Data Table 1). Thirteen patients were excluded from further analysis; of those, seven transitioned to best supportive care (BSC) due to deteriorating clinical condition and six early-stage patients opted against the recommended therapy in the adjuvant setting. Cases were discussed at the MTB, including guideline-based treatment options, as well as molecular and functional profiles obtained through tumor profiling, according to three patient groups: a, Project description showing principal aspects for feasibility testing, patient populations included and information utilization. b, CONSORT flow diagram of the TuPro project. A total of 116 patients were included in the TuPro melanoma project. Part I included ten patients to establish the TuPro workflow and analysis/reporting pipeline. Thirteen patients either received BSC (n = 7) or did not receive adjuvant treatment based on shared decision-making (n = 6). In both palliative SOC and beyond SOC groups, more than one sample was evaluated for four patients. Group 1—adjuvant setting: 13 patients with early-stage disease who underwent adjuvant treatment. These patients were administered targeted tyrosine kinase inhibitor (TKI) therapy for BRAF-mutant tumors or immunotherapy involving anti-PD1 agents. One patient received anti-CTLA-4 and anti-PD1 antibodies in the framework of an interventional trial. These treatment regimens included immunotherapy, targeted therapy, talimogene laherparepvec and chemotherapy. These interventions comprised participation in interventional trials during any time of disease course, or the application of individualized treatments (that is, off-label drug use, or the off-label reintroduction of approved therapies following confirmed progression upon initial treatment with these agents). The successful integration of high-throughput analytical technologies into routine clinical practice requires a turnaround time that aligns with clinical decision-making. To meet this need, the TuPro project established a fast diagnostic loop workflow (Fig. The workflow mandates that each technology platform delivers a comprehensive sample-specific report for discussion at the pre-tumor board (pre-TB, see below) and subsequently at the MTB, which takes place within 4 weeks of biopsy. For each sample, data were generated and analyzed through a previously described process and information technology pipeline developed specifically for the TuPro project18. The data were discussed by a pre-TB expert panel and translated into marker-based treatment recommendations that were summarized in a molecular summary report (MSR). Due to limitations of fresh tissue sample abundance or quality, predefined sample access priorities were assigned to individual technologies and sample processing within clinical routine practice (Fig. Technologies performing measurements on formalin-fixed paraffin-embedded tissue prepared for routine diagnostics (for example, IMC, NGS), were generally unaffected by limited sample abundance. scRNA-seq, scDNA-seq and proteotyping each received 68 samples. Most technology nodes successfully analyzed and reported results on the received samples if they passed node-specific quality requirements (Fig. Considering each datapoint, over 40,000 measurements were assessed per sample as potential markers, of which 544 were analyzed using targeted technologies (Supplementary Table M1). Overall, the TuPro framework successfully established a robust multiomics and functional analysis pipeline that yielded rich individual tumor profiles, providing additional and timely insights for clinical decision-making. The analysis phase involved nine technological nodes, although not all samples were analyzed by each node. b, Number of samples (n = 103) analyzed per technology node (n = 9) within the fast diagnostic loop (4-week turnaround time). Given limited sample material in some cases, the assays were prioritized as indicated by color. c, UpSet plot showing the number of samples analyzed by one or more technology nodes (n = 9). d, Concordance (percentage in light blue) between the recommendations agreed upon by the MTB and those inferred from information provided by diagnostic levels level 1 (detailed clinical data, routine molecular testing data in melanoma (BRAF, NRAS and c-KIT mutations) and DigiPath data), level 2 (level 1 plus large panel NGS), or level 3 (all TuPro technology nodes, n = 9), and fraction of actual TuPro-driven therapy decisions (i.e., ≥50% of the applied drugs are supported by markers measured by TuPro). Level 2 additionally included results of a broad targeted NGS panel. Final MTB recommendations were in line with data provided by levels 1, 2 and 3 in 36%, 42% and 75% of samples, respectively. The lower percentage of MTB agreement with level 1 and 2 results mainly from a frequent lack of identifying new options for targeted therapy with standard technologies. The percentage of TuPro-driven actual treatments administered to patients (that is, treatments decided by patients and their physicians after MTB recommendations, with ≥50% of the applied drugs supported by markers measured by TuPro) reached 87% (Fig. In summary, whereas the addition of extended targeted NGS (level 2) to standard clinical workup provided only a modest impact to the MTB, functional and multiomics TuPro data (level 3) provided substantial added value to clinical decision-making compared to current diagnostic standards (NGS and DigiPath). This is reflected by the increased concordance of TuPro-informed recommendations with those of the MTB by 39% (compared to diagnostic level 1) and 33% (compared to diagnostic level 2) of cases, respectively (Fig. Out of all available measurements, the pre-TB and the MTB participants decided on those deemed to be most useful to inform decisions. For actual treatment decisions (that is, treatments prescribed by the treating physician and supported by TuPro measurements), a total of 54 individual markers (recorded as part of the pre-TB discussions) were used (Fig. These 54 markers correspond to 399 individual, validated measurements, including assessment of the same target by different technologies (for example, pERK level was measured by IMC, CyTOF and 4iDRP). The distribution of individual measurements that informed treatment decisions was as follows: adjuvant setting, 50 marker measurements in 13 of 13 samples (Extended Data Fig. 1b,c); palliative SOC group, 153 marker measurements in 38 of 49 samples (Fig. 1d) and beyond SOC group, 196 marker measurements in 39 of 41 samples (Fig. For the remaining 13 samples, no treatments driven by markers measured with TuPro technologies were administered to the patients. Markers used for decision-making in the MTB with a cutoff at >2% amongst all patients, along with the specific technologies used for their assessment, are shown in Fig. Whereas DNA alterations (for example, TMB, mutations in RAS, KIT and BRAF) included United States Food and Drug Administration-recognized biomarkers (according to OncoKB12) measurable via level 2 diagnostics, the TuPro diagnostic level 3 provided markers additional to genetic markers (for example, phosphorylation, expression and drug response). Together, these findings show that multiomics marker data generated via TuPro technologies were readily accepted and implemented by the study physicians. a, OncoPrint showing biomarkers (cutoff of >2%) detected by TuPro and used for treatment decisions (n = 90). Data on the technology node used to measure each marker, the cohort and class of treatment received (chemotherapy, targeted therapy, immunotherapy) are provided. b, Markers (n = 26) used for treatment decision-making in the palliative SOC cohort. c, Markers (n = 44) used for treatment decision-making in the beyond SOC cohort. An important question was whether similar results can be achieved with a smaller subset of technologies and markers, resulting in significantly lower profiling costs. In many instances, one marker can be measured by several technologies. A small subset of these technologies could cover all markers relevant to most patients. We performed an analysis considering all possible subsets of technologies, taking into account assumed equivalences between markers from different technologies (Supplementary Table M2a–c). For each patient, we recorded markers deemed relevant for treatment recommendations per cohort (adjuvant, SOC, beyond SOC), assuming the same recommendations would be made if all relevant markers were available. 2 shows the number of samples (y axis) that had all relevant markers available for a specific combination of technologies. A cost-effectiveness analysis was performed considering all possible subsets of technologies and the fraction of patients likely to have had the same treatment recommendation (Extended Data Fig. Of note, only Pharmacoscopy and 4iDRP provide information for several treatment conditions, whereas the other technologies provide information only for untreated samples. We found that information provided by three technologies (NGS, Pharmacoscopy, IMC) (adjuvant and palliative SOC) and four technologies (NGS, Pharmacoscopy, IMC, scRNA-seq) (beyond SOC) would cover all markers used for pre-TB treatment recommendation (Extended Data Fig. These costs are 1.15-fold higher than the clinical-setting costs for standard NGS analysis (up to 4,000 Swiss francs in Switzerland) for adjuvant or palliative SOC, and 1.8-fold higher for beyond SOC setting. During the project period, approximately 180 Swissmedic-approved anti-cancer agents and 20 investigational drugs within 25 active trials in the participating centers were available. In the adjuvant group, 13 of 13 patients (100%) received treatment supported by markers measured with TuPro technologies. Three different systemic therapy regimens were administered (Fig. After a median observation time of 37.7 months, eight patients remained in remission and five patients had relapsed. In the palliative SOC group, 45 patients received 47 treatments considered SOC (two patients were serially biopsied and received two different treatments) (Fig. In 38 of 49 samples analyzed (77.5%), patients received a treatment supported by markers measured with TuPro technologies, and six different approved systemic treatment regimens were administered (Fig. The top three recommended treatment options were: (1) combination immune checkpoint inhibitors (ICI) (anti-CTLA-4 and anti-PD1 antibodies), (2) TKI combination therapy (BRAF/MEK inhibitors) and (3) chemotherapy (Fig. In the beyond SOC group, 37 patients received 39 treatments (two patients received serial samplings and two sequential but different treatment recommendations), either through participation in an interventional clinical trial or through off-label use of approved oncological drugs. In 39 of 41 samples analyzed, (95%) treatment decisions were informed by TuPro (Fig. Recommendations were highly individualized and 22 different therapy regimens were used for patients in the beyond SOC group. Outer circle, therapies received; inner circle, best responses per patient categorized as CR (dark green), PR (light green), SD (light blue), PD (red) or not evaluable. d, ORR, disease control rate and number of previous treatment lines before biopsy for patients in the palliative SOC and beyond SOC groups. e,f, Alluvial plots showing the association between types of treatment received, line in which the treatment was received and disease control rate for the palliative SOC (e) and beyond SOC (f) groups. Brackets indicate at three or more treatment lines and disease control rate. g,h, Swimmers plots showing PFS and best response to treatment for palliative SOC (g) and beyond SOC (h) groups. Ten patients had serial biopsies (two per patient). Two of these patients did not receive TuPro-informed recommendation and were excluded from the analysis (Supplementary Table M3). Of the remaining eight patients, two received different treatment recommendation in the SOC cohort (patient 2 and 3), one in the beyond SOC cohort (patient 6) and one patient received different recommendations leading to a switch in cohort classification (patient 1) (Supplementary Table M3 and Extended Data Fig. In summary, a total of four out of eight (50%) patients with serial biopsies included in the analysis received different treatment recommendations. For patients in the palliative SOC group, the objective response rate (ORR) was 60% and the disease control rate, which included those with complete response (CR), partial response (PR) or stable disease (SD), was 62% (Fig. The median number of previous therapy lines in the palliative SOC group was one, ranging from zero to five; in the beyond SOC group, it was two, ranging from zero to six (Fig. Analysis of disease control rate for each treatment showed that both palliative SOC and beyond SOC groups benefited from treatments administered in third and further lines (Fig. 4e,f) (brackets indicate at least three lines of treatment and disease control rate). The palliative SOC and beyond SOC TuPro participants (n = 86 lines of treatment) achieved a combined disease control rate (CR, PR or SD) of 61.0% (50 of 82, 4 not evaluable) (Fig. For the subgroups of patients who had already received at least three treatment lines, the disease control rate reached 56.8% (21 of 37) (Fig. Survival analysis showed a median progression-free survival (PFS) for patients in the palliative (both SOC and beyond SOC) of 6.04 months, (95% confidence interval (CI), 3.75–12.06) (Fig. 5c) and of 5.35 months (95% CI, 2.89–12.06) for subjects who were given at least three lines of treatment (n = 37) (Fig. a, Best response (CR, PR, SD, PD) in patients under palliative treatments (SOC and beyond SOC) in the Tumor Profiler cohort (TuPro, n = 82, 4 not evaluable). b, Best response (CR, PR, SD, PD) in patients under palliative at least third line treatment in the TuPro cohort (n = 37). To further evaluate clinical benefit of the TuPro diagnostic pipeline, we retrospectively, in an exploratory way, compared outcomes of patients in the TuPro melanoma cohort with patients with melanoma from the same centers and period who were not part of the TuPro project (non-TuPro patients) (Extended Data Figs. To minimize potential bias, we performed a combined exact (treatment line, intention of treatment, clinical stage, brain metastases status) and propensity score matching (Methods), which resulted in comparable subgroups of 12 adjuvant treatments, 59 palliative treatments and 17 treatments with at least three treatment lines for TuPro and non-TuPro patients each (Supplementary Figure M1 and Extended Data Table 2). The matched analysis of adjuvant patients (n = 12 matched treatments) showed no significant differences in relapse-free survival (RFS) (adjusted hazard ratio (HR) 0.46 (95% CI, 0.06–3.32) (Extended Data Fig. For the matched palliative groups (n = 59 matched treatments), median PFS in TuPro subjects reached 9.59 months (95% CI, 3.75–17.74) and 3.55 months (95% CI, 2.56–9.17) for non-TuPro subjects (adjusted HR of 0.78 (95% CI, 0.43–1.42), adjusted P = 0.4156) (Extended Data Fig. Within these matched groups, the disease control rate was 63.6% (35 of 55, 4 not evaluable) in the TuPro cohort compared to 51.7% (30 of 58, 1 not evaluable) in the non-TuPro cohort (Extended Data Fig. For subjects who had received at least three treatment lines (n = 17 matched treatments) the median PFS in the TuPro cohort was 8.34 months (95% CI, 2.76–NR) compared to 2.0 months (95% CI, 1.08–3.06) in the non-TuPro cohort (adjusted HR of 0.23 (95% CI, 0.07–0.79), adjusted P = 0.0201) and a disease control rate of 64.7% (11 of 17) compared to 23.5% (4 of 17) (Extended Data Fig. Overall, our exploratory analysis suggests an added value for heavily pretreated patients. The TuPro project is a pioneering, prospective, multicohort precision oncology project, integrating cutting-edge technologies for the detailed and comprehensive investigation of tumor biopsies at an unprecedented level18. Here we report several clinical-translational endpoints of the project. First, we successfully generated high-quality molecular datasets from melanoma tumor biopsies using two standard (DigiPath and NGS) and seven experimental, mostly single-cell, high-throughput technologies (Pharmacoscopy, 4iDRP, IMC, CyTOF, proteotyping, scRNA-seq and scDNA-seq), which offer distinct and complementary perspectives on tumor biology. Second, we achieved a clinically meaningful turnaround time from sample acquisition to analysis and recommendation to the MTB of 4 weeks. This fast diagnostic loop supports the notion that incorporation of multiomics technologies into clinical oncology is feasible. Third, we successfully established a data-to-treatment pipeline that presented actionable insights gained from multimodal analysis to the MTB, facilitating actual clinical decision-making. Notably, the integration of MTBs into clinical decision-making correlates with improvements in various outcome measures28. Nevertheless, randomized controlled trials that formally assess these results need to be conducted in the future. The introduction of ICIs targeting PD1, CTLA-4 and LAG-3, as well as therapies addressing dysregulated and oncogenic MAPK pathway alterations (for example, BRAF and MEK mutations), has led to an unprecedented improvement in overall survival. The 60% ORR observed in the TuPro palliative SOC group of first and further line patients compares favorably with published data. Notably, BRAF class I mutations currently are the only predictive biomarker used to determine eligibility for targeted therapy options1,2. However, single biomarker strategies have limitations, which are reflected by (1) a considerable number of patients with poor responses; (2) several negative high-profile clinical trials employing this strategy7,32,33; (3) genotype-matched trials (for example, MatchMel, SHIVA, MOSCATO, IMPACT/COMPACT) resulting in low target-drug matching rates (5–30%) or limited availability of interventional trials, and overall poor outcomes (ORR, 11–19%)34,35,36,37,38,39, and (4) single genetic alterations capturing only a fraction of the vastly complex biology of tumors. Both the successes and limitations outlined above form a strong rational for an expanded biomarker and/or assay panel to enable an in-depth mechanism-based therapy prediction for individual patients. The TuPro project integrated various layers of molecular and cellular information, encompassing phosphorylation, cell–cell interactions, characteristics of the tumor microenvironment, inflammatory/immune markers, and ex vivo drug response. These factors are anticipated to play an important role in advanced and personalized predictive decision-making processes40,41,42. In the beyond SOC group, individual therapy recommendations were generated based on a larger number of individual markers compared to the palliative SOC group (average five versus four, respectively). Although markers used to inform clinical decision-making within the TuPro project are biased toward known mechanisms and available drugs, the swift adoption of the expanded marker panel shows that clinicians rapidly acquire confidence in new readouts to support therapy. The project incorporated drugs that are not traditionally used in clinical care for melanoma patients. Therefore, the response rates in the beyond SOC TuPro cohort compare favorably when viewed from the perspective of other tumor-agnostic or phase I/II clinical trials, where substantial variations in ORR rates of 2–53% and median PFS of 5–7 months have been observed43,44,45,46. Our analysis comparing the TuPro cohort with a synchronous non-TuPro melanoma cohort suggests that omics-guided treatments, such as in the TuPro project, result in comparable or better outcomes in terms of disease control rates and PFS, thus providing a rationale to further explore the analysis of -omics data as a means of therapy prediction in patients with melanoma. These data advocate that the expanded marker assessment via multiomics and functional technologies may contribute to address an unmet clinical need for difficult-to-treat patient populations. The ever-expanding range of therapeutic options necessitates a comprehensive understanding of tumor biology and individual patient characteristics. The TuPro project is uniquely positioned to achieve both immediate patient benefits and biological discoveries at the cohort level, which are currently under investigation. We recognize several limitations of the analyses described and opportunities to improve future study designs. The current approach of selecting markers for treatment decisions by human experts cannot capture all potentially relevant information. The poor response in several patients treated with individualized therapies may reflect this shortcoming. The advent and rapid evolution of artificial intelligence and machine learning methods will probably need to be leveraged to translate the extensive complexity of TuPro datasets into enhanced clinical benefit. Further process optimization and efficacy improvements will facilitate the full implementation of all available technologies. Interpretations regarding clinical utility of serial biopsies are limited by heterogenous results and small sample size. Although we attempted to provide comparable results using matched cohorts, the project was first and foremost a feasibility study. One main goal was to identify patients with melanoma most likely to benefit from extended TuPro analysis. Given key uncertainties (patients who are most likely to benefit, expected effect size, technology performance, feasibility of clinical translation of findings by the MTB) a randomized trial would not have been feasible at the time the project was designed. Clearly blinded, randomized trials are needed to confirm our findings in a next step. Based on our calculations, the per patient cost of analysis in the beyond SOC setting is above the current coverage limits by health insurance for a NGS panel (up to 4,000 Swiss francs in Switzerland) yet only exceeds this limit by a factor of 1.8 when using a limited number of technologies that would possibly deliver the same answers. In addition, we demonstrate that at costs comparable to those covered by insurance (4602 Swiss francs), we were able to successfully profile 38 out of 39 patients (97%) within the SOC setting. The current cost estimates fall within a range that allows for evaluation of potential benefits through a randomized trial. In conclusion, TuPro has successfully pioneered the integration of predictive polymarker assessment with marker-driven, highly individualized treatments, demonstrating the feasibility of complex and comprehensive data-driven approaches to improve patient care. TuPro was conducted as a multicentric precision oncology project according to the Swiss Act and Ordinance on Human Research (HFG and HFV) with approval of, and in compliance with, regulatory authorities (ethics committees of northwestern Switzerland, EKNZ, and of Zurich, KEK). Adult patients diagnosed with any subtype of melanoma, both treatment-naive and those treated either in the adjuvant or palliative setting were eligible for inclusion. The project was conducted at the Department of Dermatology, University Hospital Zurich, at the Department of Oncology, University Hospital Basel and at the Department of Oncology and Hematology, Kantonsspital Baselland, Switzerland, from January 2019 until November 2020 (EC-ID, 2018-02050, 2021-01584) in accordance with the applicable legal and institutional standards. For each specific (single agent or combined) off-label treatment, regardless of whether the Tumor Board decision was based on routine patient data or the MSR, the project allowed inclusion of up to five patients. Treatments were not given as part of the study. Separate consents were required for any selected treatment regimens. The demographical, clinical and pathological features of eligible patients, including age range, sex, melanoma subtype, stage (according to the American Joint Committee on Cancer, eighth edition), together with previous treatments and treatments after biopsy collection, were obtained from our institutional database (Klinikinformationssystem (KISIM)) and entered in the good-clinical-practice-compliant database secuTrial v.6.1.2.5, 2021 (Extended Data Table 1). Sex- and gender-based analyses were not conducted due to the small sample size and the feasibility-focused nature of the study, which did not include sex- or gender-specific stratification. Eligible female and male patients (n = 116), age range 20–89 years, underwent tissue biopsies as part of their diagnostic workup. Leftover tissue samples from routine diagnostics were used in this study, with participants providing project-specific written informed consent. Participants did not receive compensation for their involvement in the study. Experienced pathologists evaluated the biopsy samples for viability and tumor cell content (requiring a minimum of 20% viable tumor cells). Samples meeting these quality criteria were further processed in a central laboratory and underwent paraffin embedding or single-cell dissociation. On average, 2,000,000 viable single tumor cells per sample were recovered. Samples were transferred to the following analytical platforms (referred to as technology nodes) for further processing and analysis: CyTOF22, IMC21, scRNA-seq20, scDNA-seq19, mass spectrometry proteotyping23,24,25, Pharmacoscopy11,26 or 4iDRP27. To establish technical feasibility, robustness, and reproducibility of sample and data handling logistics, ten (n = 10) samples were initially analyzed and results were not discussed at the MTB. Feasibility outcomes: types of molecular information and combinations of molecular information from the biotechnology domain that the pre-TB considers as useful for making a treatment recommendation beyond routine diagnostics (including routine pathology and panel NGS testing). Classification of proposed treatment options based on TuPro measurements: (a) on-label treatment with molecular matched treatment (label of the Swiss Agency for Therapeutic Products, SwissMedic, as a reference) ± radiotherapy or chemotherapy; (b) treatment with classical chemotherapy ± radiotherapy (on label if label available); (c) referral to a suitable clinical trial; (d) off-label treatment (SwissMedic label as a reference) with molecular matched treatment or immunotherapy ± radiotherapy or chemotherapy; (e) off-label treatment (authorization in countries with comparable approval and control systems for medicinal products as defined by SwissMedic) with molecular matched treatment or immunotherapy ± radiotherapy or chemotherapy; (f) immunotherapy; (g) no active anti-tumor treatment (BSC). Clinical outcomes: (a) best response to treatment, assessed radiologically after treatment initiation, classified according to RECIST criteria; (b) PFS as the duration in months between the date of treatment initiation (first medication intake) and the date of the first radiologically confirmed progression, if progression occurred. To comply with applicable laws and regulations (the Swiss Human Research Act), all deidentified clinical data relevant to this publication are provided as supporting information to the paper. Data access can be requested by contacting the TPC (nicola.miglino@usz.ch). The user institution agrees to destroy or discard the data once it is no longer used for the project, and in cases where data must be archived, they must be deleted within 10 years of the project's completion. If data have not been archived, they must be deleted no later than 2 years following the completion of the project. The goal was to analyze and summarize this vast amount of data in a clinically meaningful manner. We aimed to identify datapoints that were valuable for decision-making and determine the necessary steps to ensure the usability and reliability of therapy predictions to enhance patient outcomes. To this end, detailed discussions between members from various technology nodes, data scientists, molecular pathologists and clinicians in the pre-TB condensed this information into the MSR. For instance, response to MEK inhibitors (for example, trametinib) was associated with mutated NRAS, BRAF, GNAQ/GNA11, NF1 and/or an activated MAPK pathway, indicated by pERK. ICIs (for example, nivolumab) response was linked to HLA expression, an inflammatory tumor phenotype and checkpoint expression. However, clinicians adhered to available and applicable guidelines such as from the European Society for Medical Oncology (ESMO) and the National Comprehensive Cancer Network (NCCN), which included recommendations on off-label medication use and the regulatory standards for patient allocation to clinical projects or trials. To assess the additional value of TuPro data in conjunction with standard technologies such as NGS and digital pathology, a stepwise evaluation process at the pre-TB was established. Level 1 included detailed clinical information, patient history, genetic tumor data (BRAF, NRAS, c-KIT) and insights from DigiPath. Level 3 additionally implemented data provided by the seven experimental TuPro technology nodes (CyTOF, IMC, scRNA-seq, scDNA-seq, proteotyping, Pharmacoscopy, 4iDRP). Only medications approved by SwissMedic (on or off label, around 180 at the time of the project) or drugs under investigation in clinical trials at one of the TuPro centers (n = 20) were considered. The assessment of clinical usefulness for each marker and technology was carried out in the TuPro application cohort (n = 93 patients and 99 lines of therapy) (Fig. This population consists of patients who were discussed in both pre-TB and MTB, had all levels of treatment recommendations recorded, received anti-cancer treatment and had available outcome assessments. The TuPro application cohort was categorized into three groups: (1) adjuvant therapy (decision between two standards, TKI therapy for BRAF-mutant tumors versus immunotherapy, and one trial-associated therapy, anti-CTLA-4 + anti-PD1), (2) palliative therapy within the SOC (that is, immunotherapy, targeted therapy, talimogene laherparepvec, chemotherapy) and (3) palliative therapy beyond the SOC (interventional trials, off-label use, or off-label reintroduction of approved therapies after confirmed progression upon treatment with these agents). Two patients were sequentially included first in the SOC and later in the beyond SOC group. Of note, in this diagnostic proof-of-concept project, patients were not preselected to allow for TuPro-directed treatment, that is, there was no inclusion criterion regarding life expectancy. Treatment decisions were considered TuPro-driven if ≥50% of the drugs administered were selected based on markers reported by a TuPro technology node15. This also applied to patients who were already undergoing treatment, when alterations reported by TuPro technologies confirmed ≥50% of the drugs in the ongoing treatment regimen. The costs for multiomics profiling were assessed as marginal costs of production, that is, the cost of one additional analysis on an otherwise fully funded analysis platform running at full capacity. All patients receiving therapy in the TuPro project underwent regular assessments following standard operating procedures at the participating institutions. Response assessment was conducted every 2–3 months using medical imaging scored via RECIST v.1.1 criteria. PFS was calculated from the date of therapy initiation to documented disease progression or until the last follow-up for nonprogressed patients with a cutoff date on 31 May 2023. Clinical endpoints were defined in the project protocol and included the number of cases in which the MTB deemed the MSR useful, number of cases in which treating physicians found the MTB recommendations beneficial, specific information (markers) considered relevant by the MTB for treatment recommendations beyond routine diagnostics (DigiPath, NGS), classification of proposed treatment options (in-label, off-label, trial, BSC or no therapy), ORR (which included CR and PR), disease control rate (CR, PR, SD), duration of response, time to next therapy or BSC and the proportion of patients who discontinued treatment due to toxicity. In an exploratory, retrospective analysis, outcomes in the TuPro cohort were compared to those of melanoma patients who were not recruited into TuPro but received diagnostic assessment and subsequent systemic therapy at the TuPro centers over a comparable period (1 February 2019 to 1 July 2021) (Extended Data Figs. The comparison cohort included 141 patients followed until 31 May 2023, who received 216 therapies (6 neoadjuvant, 75 adjuvant, 96 palliative SOC, 39 beyond SOC). In addition to comparing unmatched RFS, PFS and response rates, a matched analysis was performed to establish comparable subgroups and reduce bias. The TuPro treatments were matched (1:1) to the non-TuPro treatments using a genetic matching algorithm (MatchIt package47) without replacement with propensity scores estimated with a multivariate logistic regression model including the following pretreatment variables: Charlson Comorbidity index, Eastern Cooperative Oncology Group score (0–5), histologic subtype, lactate dehydrogenase levels at baseline, tumor mutational burden and pathogenic mutational status of BRAF (yes/no). To optimize cohort comparability, we combined propensity score matching with exact matching for all palliative comparisons with the following variables: clinical stage (I–IV), intention of treatment (SOC, beyond SOC), treatment line (1–8) and presence of brain metastases (yes/no). In the adjuvant group, exact matching was performed based solely on clinical stage, as treatment intention, treatment line and brain metastases are not applicable in this setting. Unadjusted rates and adjusted odds ratios were calculated for the binary outcome of either disease control (CR, PR, SD) or progression (PD). Mixed-effects adjusted logistic regression models were used to explore the association between disease control and TuPro participation. The models for the unmatched analysis included the following variables as covariates: Eastern Cooperative Oncology Group (binary variable, ≥2), lactate dehydrogenase at baseline, presence of brain metastases (yes/no), age, clinical stage (III or IV), intention of treatment (adjuvant, SOC, beyond SOC) and treatment line (< or ≥ third line). In the subanalysis of treatments for subjects treated with three or more lines of therapy, the treatment line and clinical stage was excluded as a covariate. Median RFS and PFS were estimated using a Cox model with frailty to account for patients receiving several treatments. A mixed-effects Cox proportional-hazards regression model was used to calculate adjusted HRs and P values comparing RFS and PFS between TuPro versus non-TuPro treatments with random effects accounting for patients with several included treatments. In the unmatched adjuvant analysis, presence of brain metastases, intention of treatment and treatment line were excluded as covariates. All analyses tested the null hypothesis using a two-sided 0.05 significance level and included 95% CI calculations. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. To comply with applicable laws and regulations (the Swiss Human Research Act), all deidentified clinical data relevant to this publication are provided as supporting information to the paper. Data access can be requested by contacting the TPC (nicola.miglino@usz.ch). The user institution agrees to destroy or discard the data once it is no longer used for the project, and in cases where data must be archived, it must be deleted within 10 years of the project's completion. If data has not been archived, it must be deleted no later than 2 years following the completion of the project. Michielin, O., Van Akkooi, A. C. J., Ascierto, P. A., Dummer, R. & Keilholz, U. Cutaneous melanoma: ESMO Clinical Practice Guidelines for diagnosis, treatment and follow-up. Coit, D. G. et al. Cutaneous melanoma, version 2.2019, NCCN clinical practice guidelines in oncology. Andreano, A., Rebora, P., Valsecchi, M. G. & Russo, A. G. Adherence to guidelines and breast cancer patients survival: a population-based cohort study analyzed with a causal inference approach. & Long, G. V. Targeted agents and immunotherapies: optimizing outcomes in melanoma. Jenkins, R. W. & Fisher, D. E. Treatment of advanced melanoma in 2020 and beyond. Hoch, T. et al. Multiplexed imaging mass cytometry of the chemokine milieus in melanoma characterizes features of the response to immunotherapy. Kornauth, C. et al. Functional precision medicine provides clinical benefit in advanced aggressive hematologic cancers and identifies exceptional responders. Snijder, B. et al. Image-based ex-vivo drug screening for patients with aggressive haematological malignancies: interim results from a single-arm, open-label, pilot study. Chakravarty, D. et al. OncoKB: a precision oncology knowledge base. Mateo, J. et al. A framework to rank genomic alterations as targets for cancer precision medicine: the ESMO scale for clinical actionability of molecular targets (ESCAT). An overview of precision oncology basket and umbrella trials for clinicians. Singer, F. et al. SwissMTB: establishing comprehensive molecular cancer diagnostics in Swiss clinics. Tamborero, D. et al. Support systems to guide clinical decision-making in precision oncology: The Cancer Core Europe Molecular Tumor Board Portal. Kuipers, J., Tuncel, M. A., Ferreira, P., Jahn, K. & Beerenwinkel, N. Single-cell copy number calling and event history reconstruction. Papalexi, E. & Satija, R. Single-cell RNA sequencing to explore immune cell heterogeneity. Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry. Targeted data extraction of the MS/MS spectra generated by data-independent acquisition: a new concept for consistent and accurate proteome analysis. Standardization and harmonization of distributed multi-center proteotype analysis supporting precision medicine studies. Simultaneous targeted and discovery-driven clinical proteotyping using hybrid-PRM/DIA. Gut, G., Herrmann, M. D. & Pelkmans, L. Multiplexed protein maps link subcellular organization to cellular states. Tsimberidou, A. M. et al. Molecular tumour boards - current and future considerations for precision oncology. Combined nivolumab and ipilimumab or monotherapy in untreated melanoma. Robert, C. et al. Nivolumab in previously untreated melanoma without BRAF mutation. Overall survival in patients with BRAF-mutant melanoma receiving encorafenib plus binimetinib versus vemurafenib or encorafenib (COLUMBUS): a multicentre, open-label, randomised, phase 3 trial. Gogas, H. et al. Cobimetinib plus atezolizumab in BRAFV600 wild-type melanoma: primary results from the randomized phase III IMspire170 study. Long, G. V. et al. Epacadostat plus pembrolizumab versus placebo plus pembrolizumab in patients with unresectable or metastatic melanoma (ECHO-301/KEYNOTE-252): a phase 3, randomised, double-blind study. Stockley, T. L. et al. Molecular profiling of advanced solid tumors and patient outcomes with genotype-matched clinical trials: the Princess Margaret IMPACT/COMPACT trial. Le Tourneau, C. et al. Molecularly targeted therapy based on tumour molecular profiling versus conventional therapy for advanced cancer (SHIVA): a multicentre, open-label, proof-of-concept, randomised, controlled phase 2 trial. Massard, C. et al. High-throughput genomics and clinical outcome in hard-to-treat advanced cancers: results of the MOSCATO 01 trial. Trédan, O. et al. Molecular screening program to select molecular-based recommended therapies for metastatic cancer patients: analysis from the ProfiLER trial. B., Stockley, T. L. & Siu, L. L. Molecular profiling for precision cancer therapies. Bunne, C. et al. Learning single-cell perturbation responses using neural optimal transport. The updated landscape of tumor microenvironment and drug repurposing. Flaherty, K. T. et al. Inhibition of mutated, activated BRAF in metastatic melanoma. Encouraging trends in modern phase 1 oncology trials. & Prasad, V. Phase I trials and therapeutic intent in the age of precision oncology: what is a patient's chance of response? Falchook, G. S. et al. Activity of the oral MEK inhibitor trametinib in patients with advanced melanoma: a phase 1 dose-escalation trial. Ho, D., Imai, K., King, G. & Stuart, E. A. MatchIt: nonparametric preprocessing for parametric causal inference. We thank the patients, their families, as well as involved clinicians and study nurses of University Hospital Basel, Kantonsspital Baselland and University Hospital Zurich for their contribution. We are grateful for the excellent management support by T. Solbach, A. Strauss, L. Reypens and V. M. Johnen. The melanoma cohort of TuPro was also supported by a grant of the Promedica Foundation to A.W. S.G. was supported by funds from the strategic focus area Personalized Health and Related Technologies from the ETH Domain. This work was partially supported by ETH core funding to G.R. Open access funding provided by University of Zurich. These authors contributed equally: Nicola Miglino, Nora C. Toussaint, Alexander Ring, Ximena Bonilla, Marina Tusup. These authors jointly supervised this work: Rudolf Aebersold, Marina Bacac, Gerd Maass, Holger Moch, Michael Weller, Alexandre P. A. Theocharides, Markus G. Manz, Niko Beerenwinkel, Christian Beisel, Lucas Pelkmans, Berend Snijder, Bernd Wollscheid, Viola Heinzelmann, Bernd Bodenmiller, Mitchell P. Levesque, Viktor H. Koelzer, Gunnar Rätsch, Reinhard Dummer, Andreas Wicki. Nicola Miglino, Alexander Ring, Benedict Gosztonyi, Tarun Mehra, Gabriele Gut, Laura Boos, Parisa Rahimzadeh, Manuel Schuerch, Mattheus HE Wildschut, Alexandre P. A. Theocharides, Markus G. Manz & Andreas Wicki Nora C. Toussaint, Natalia Chicherova, Shuqing Yu, Franziska Singer, Anne Bertolini, Michael Prummer, Faisal S. Al-Quaddoomi, Silvana I. Albert, Lars Bosshard, Linda Grob, María L. Rosano-González, Vipin T. Sreedharan & Daniel J. Stekhoven SIB Swiss Institute of Bioinformatics, Lausanne, Switzerland Nora C. Toussaint, Natalia Chicherova, Shuqing Yu, Jack Kuipers, Monica-Andreea Baciu-Drăgan, Pedro F. Ferreira, Franziska Singer, Anne Bertolini, Michael Prummer, Faisal S. Al-Quaddoomi, Sonali Andani, Monica-Andreea Baciu-Drăgan, Lars Bosshard, Joanna Ficek-Pascual, Linda Grob, Alva R. James, André Kahles, Abdullah Kahraman, Laurie Prélot, María L. Rosano-González, Vipin T. Sreedharan, Stefan Stark, Daniel J. Stekhoven, Tinu M. Thomas, Mustafa A. Tuncel, Niko Beerenwinkel & Gunnar Rätsch Swiss Data Science Center SDSC, Zurich, Switzerland Ximena Bonilla, Kjong-Van Lehmann, Sonali Andani, Natalie R. Davidson, Joanna Ficek-Pascual, Alexander Immer, Alva R. James, André Kahles, Philipp Markolin, Laurie Prélot, Stefan Stark, Tanmay Tanna, Tinu M. Thomas, Marc Zimmermann & Gunnar Rätsch Francis Jacob, Ricardo Coelho, André Fedier, Viola Heinzelmann-Schwarz, Flavio C. Lombardo, Monica Nunez Lopéz, Salvatore Piscuoglio, Natalie Rimmer, Mathilde Ritter & Viola Heinzelmann Bettina Sobottka, Anja L. Frei, Marta Nowak, Sonali Andani, Abdullah Kahraman, Norbert Wey, Martin Zoche, Holger Moch & Viktor H. Koelzer Rebekka Wegmann, Julien Mena, Mattheus HE Wildschut, Rudolf Aebersold, Berend Snijder & Bernd Bodenmiller Emanuela S. Milani, Sandra Goetze, Jonas Albinus, Martin Mehnert, Wenguang Shao, Audrey Van Drogen, Fabian Wendt & Bernd Wollscheid Cinzia Esposito, Jacobo Sarabia del Castillo, Esther Danenberg, Rene Holtackers & Lucas Pelkmans Anja Irmisch, Sylvia Herter, Tamara Huesser, Riccardo Murri, Vinko Tosevski & Marina Bacac Department of Biosystems Science and Engineering, ETH Zurich, Basel, Switzerland Jack Kuipers, Monica-Andreea Baciu-Drăgan, Pedro F. Ferreira, Franziska Singer, Anne Bertolini, Michael Prummer, Ulrike Lischetti, Monica-Andreea Baciu-Drăgan, Katja Eschbach, Natascha Santacroce, Tobias Schär, Tanmay Tanna, Mustafa A. Tuncel, Niko Beerenwinkel & Christian Beisel Ilaria Alborelli, Daniel Baumhoer, Philip M. Jermann, Simone Muenst, Charlotte K. Y. Ng, Stefan Nicolet, Salvatore Piscuoglio, Markus Tolnay, Tatjana Vlajnic & Viktor H. Koelzer Sonali Andani, Joanna Ficek-Pascual, Alva R. James, André Kahles, Laurie Prélot, Stefan Stark, Tinu M. Thomas & Gunnar Rätsch Per-Olof Attinger, Maya D'Costa, Tim M. Jaeger, Sebastian Lugert & Severin Schwan Cantonal Hospital Baselland, Medical University Clinic, Liestal, Switzerland Max Planck ETH Center for Learning Systems, Zurich, Switzerland You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar accrued patients and provided patient samples throughout the project. reports being an invited speaker for Sharing Progress in Cancer Care (SPCC) and Indica Labs; advisory board of Takeda; sponsored research agreements with Roche and IAG, all unrelated to the current study. is a participant in patent applications on the assessment of cancer immunotherapy biomarkers by digital pathology and for the prediction of cancer recurrence risk and prediction of treatment efficacy using deep learning unrelated to the current study. received funding from Novartis, Merck Sharp & Dohme (MSD), Bristol-Myers Squibb (BMS), Roche, Amgen, Takeda, Pierre Fabre, Sun Pharma, Sanofi, Catalym, Second Genome, Regeneron, T3 Pharma, MaxiVAX SA, Pfizer and Simcere. has received research grants from Novartis, Quercis and Versameb, and honoraria for lectures or advisory board participation or consulting from Anheart, Bayer, Curevac, Medac, Neurosense, Novartis, Novocure, Orbus, Pfizer, Philogen, Roche and Servier. The other authors declare no competing interests. Nature Medicine thanks Genevieve Boland, Emile Voest and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Jean Nakhle and Saheli Sadanand, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. b, Markers (n = 12) used for treatment decision making in the adjuvant cohort. c, Individual measurements of markers for applied therapeutic decisions in the adjuvant setting. d, Individual measurements of markers used for applied therapeutic decisions in the palliative SOC setting. e, Individual measurements of markers used for applied therapeutic decisions in the beyond SOC setting. The x-axis shows the marginal costs in Swiss francs (CHF) of production, representing the cost of one additional analysis using a combination of technology nodes running at full capacity. The y-axis shows the number of patients that would have all relevant information available with the corresponding subset of technologies when compared to using all technologies. Each dot corresponds to a different combination of technologies associated with a total cost for that combination. Only the best combinations are shown, as other combinations have a higher cost or a smaller number of samples with available relevant information. Second (Sample ID) and third (Treatment) column showing both biopsy samples per patient. Orange: serial biopsies leading to different treatment recommendations, light green: serial biopsies not changing treatment decision. Forth column (cohort) showing serial and biopsy decisions leading to change of classification based on clinical criteria (dark red, either palliative SOC or beyond SOC) or no change in cohort classification (dark green). a, Kaplan-Meier curve and adjusted relapse-free survival (RFS) in months, crude hazard ratio, adjusted hazard ratio and Log-rank P-value for the comparison between unmatched adjuvant treatments. b, Kaplan-Meier curve and adjusted RFS in months, crude hazard ratio, adjusted hazard ratio and Log-rank P-value for the comparison between matched adjuvant treatments. a, Best response (complete response (CR), partial response (PR), stable disease (SD), progressive disease (PD)) in unselected, palliative treatments (SOC and beyond SOC) in the Tumor Profiler cohort (TuPro, n = 86) and the non-TuPro cohort (n = 132, 3 not evaluable). Combined disease control rate in the TuPro cohort of 61.0% (50/82, 4 not evaluable) vs. 57.6% (76/132, 3 not evaluable) in non-TuPro patients. Adjusted odds ratio of 1.54 (95% confidence interval (CI), 0.75 – 3.16). b, Best response (CR, PR, SD, PD) in unselected, palliative ≥ 3rd treatment line in the TuPro cohort (n = 37) and the non-TuPro cohort (n = 26). c, Best response of matched TuPro (n = 59) and non-TuPro (n = 59) palliative treatments (SOC and beyond SOC). Combined disease control rate in the TuPro cohort of 64.7% (11/17) vs. 23.5% (4/17), respectively. e, f, Kaplan-Meier curves and adjusted Progression free survival (PFS) in months, crude hazard ratio, adjusted hazard ratio and Log-rank P-value for the comparison between unmatched treatments: e, palliative (SOC and beyond SOC) TuPro cohort (n = 86 treatment lines) and non-TuPro cohort (n = 135 treatment lines): 6.04 months, (95% CI, 3.75 – 12.06) TuPro cohort vs. 5.62 months (n = 135, 95% CI, 3.09 – 9.46) in the non-TuPro cohort (crude hazard ratio (HR) of 0.93 (95% CI, 0.68 – 1.28), P = 0.5967, adjusted HR 0.89 (95% CI, 0.57 – 1.37). g, h, Kaplan-Meier curves and adjusted PFS in months, crude hazard ratio, adjusted hazard ratio and Log-rank P-value for the comparison between matched treatments: g, matched palliative (SOC and beyond SOC) TuPro cohort (n = 59 treatment lines) and non-TuPro cohort (n = 59 treatment lines): 9.59 months (95% CI, 3.75 – 17.74) vs. 3.55 months (95% CI, 2.56 – 9.17), respectively (adjusted HR of 0.78 (95% CI, 0.43 – 1.42), P = .4156. h, matched patients receiving ≥ 3 treatment line in the TuPro cohort (n = 17) and non-TuPro cohort (n = 17): 8.34 months (95% CI, 2.76 – NR) vs. 2.0 months (95% CI, 1.08 – 3.06), respectively (adjusted HR of 0.23 (95% CI, 0.07 – 0.79), P = .0201. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03735-2'>Early detection of multiple cancer types using multidimensional cell-free DNA fragmentomics</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-27 09:44:33
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. The multicancer early detection (MCED) test has the potential to enhance current cancer-screening methods. We evaluated a new MCED test that analyzes plasma cell-free DNA using genetic- and fragmentomics-based features from whole-genome sequencing. The present study included an internal validation cohort of 3,021 patients with cancer and 3,370 noncancer controls, and an independent cohort of 677 patients with cancer and 687 noncancer individuals. The results demonstrated an overall sensitivity of 87.4%, specificity of 97.8% and tissue-of-origin prediction accuracy of 82.4% in the independent validation cohort. Preliminary results from a prospective study of 3,724 asymptomatic participants showed a sensitivity of 53.5% (predominantly early stage cancers) and specificity of 98.1%. These findings indicate that the MCED test has strong potential to improve early cancer detection and support clinical decision-making. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout The reference human genome used for analysis is GRCh37 (d5; https://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence). The study protocol and statistical analysis plan are available within Supplementary Information. All feature datasets analyzed for the present study are available via the Geneseeq research data portal at https://zh.geneseeq.com/mercury/datause-license-request.html. Approved researchers can access and download these datasets following registration and acceptance of a data-sharing agreement on the portal. All code used for reported analyses has been made available through the Geneseeq research data portal, available at https://zh.geneseeq.com/mercury/datause-license-request.html. Banatvala, N., Akselrod, S., Bovet, P. & Mendis, S. in Noncommunicable Diseases 234–239 (Routledge, 2023). Siegel, R. L., Miller, K. D. & Jemal, A. Global Cancer Statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. Guide to Cancer Early Diagnosis (World Health Organization, 2017). Screening for breast cancer: US Preventive Services Task Force recommendation statement. Screening for cervical cancer: US Preventive Services Task Force recommendation statement. Screening for lung cancer: US Preventive Services Task Force recommendation statement. Screening for colorectal cancer: US Preventive Services Task Force recommendation statement. Patel, M. et al. Hepatocellular carcinoma: diagnostics and screening. The National Lung Screening Trial Research et al. Results of initial low-dose computed tomographic screening for lung cancer. Daskalakis, C. et al. Predictors of overall and test-specific colorectal cancer screening adherence. Parikh, N. D. et al. Biomarkers for the early detection of hepatocellular carcinoma. The origin and mechanism of circulating DNA. Plasma DNA tissue mapping by genome-wide methylation sequencing for noninvasive prenatal, cancer, and transplantation assessments. Fece de la Cruz, F. & Corcoran, R. B. Methylation in cell-free DNA for early cancer detection. Sensitive and specific multi-cancer detection and localization using methylation signatures in cell-free DNA. Detection and characterization of lung cancer using cell-free DNA fragmentomes. Klein, E. A. et al. Clinical validation of a targeted methylation-based multi-cancer early detection test using an independent validation set. Gao, Q. et al. Unintrusive multi-cancer detection by circulating cell-free DNA methylation sequencing (THUNDER): development and independent validation studies. Chen, X. et al. Non-invasive early detection of cancer four years before conventional diagnosis using a blood test. Detection and localization of surgically resectable cancers with a multi-analyte blood test. Cristiano, S. et al. Genome-wide cell-free DNA fragmentation in patients with cancer. Bruhm, D. C. et al. Single-molecule genome-wide mutation profiles of cell-free DNA for non-invasive detection of cancer. Schrag, D. et al. Blood-based tests for multicancer early detection (PATHFINDER): a prospective cohort study. Nicholson, B. D. et al. Multi-cancer early detection test in symptomatic patients referred for cancer investigation in England and Wales (SYMPLIFY): a large-scale, observational cohort study. Swanton C. et al. NHS-Galleri Trial Design: equitable study recruitment tactics for targeted population-level screening with a multi-cancer early detection (MCED) test. & Hartman, A.-R. New genomic technologies for multi-cancer early detection: rethinking the scope of cancer screening. Trosman, J. R. et al. Perspectives of private payers on multicancer early-detection tests: informing research, implementation, and policy. Ma, X. et al. Multi-dimensional fragmentomic assay for ultrasensitive early detection of colorectal advanced adenoma and adenocarcinoma. Bao, H. et al. Letter to the Editor: An ultra-sensitive assay using cell-free DNA fragmentomics for multi-cancer early detection. Yu, P. et al. Multi-dimensional cell-free DNA-based liquid biopsy for sensitive early detection of gastric cancer. Zheng, R. et al. Cancer incidence and mortality in China, 2016. Screening for Lung Cancer With Low-Dose Computed Tomography: An Evidence Review for the U.S. Preventive Services Task Force (Agency for Healthcare Research and Quality (US), 2021). Diagnostic performance of a faecal immunochemical test for patients with low-risk symptoms of colorectal cancer in primary care: an evaluation in the South West of England. Teoh, D. et al. Test performance of cervical cytology among adults with vs without human papillomavirus vaccination. Initial results from a multi-center population-based cluster randomized trial of esophageal and gastric cancer screening in China. Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Fritz, A. et al. International Classification of Diseases for Oncology 3rd edn (World Health Organization, 2000). Jahr, S. et al. DNA fragments in the blood plasma of cancer patients: quantitations and evidence for their origin from apoptotic and necrotic cells. Lai, B. et al. Principles of nucleosome organization revealed by single-cell micrococcal nuclease sequencing. Peneder, P. et al. Multimodal analysis of cell-free DNA whole-genome sequencing for pediatric cancers with low mutational burden. Enhanced detection of circulating tumor DNA by fragment size analysis. Analysis of recurrently protected genomic regions in cell-free DNA found in urine. Doebley, A.-L. et al. A framework for clinical cancer subtyping from nucleosome profiling of cell-free DNA. Sun, K. et al. Orientation-aware plasma cell-free DNA fragmentation analysis in open chromatin regions informs tissue of origin. Zhou, Q. et al. Epigenetic analysis of cell-free DNA by fragmentomic profiling. Prospective changes in global DNA methylation and cancer incidence and mortality. Rodriguez, J. et al. Genome-wide tracking of unmethylated DNA Alu repeats in normal and cancer cells. Ansel, J. et al. PyTorch 2: faster machine learning through dynamic Python bytecode transformation and graph compilation. 29th ACM International Conference on Architectural Support for Programming Languages and Operating Systems 929–947 (Association for Computing Machinery, 2024). LeDell, E. & Poirier, S. H2O AutoML: scalable automatic machine learning. We thank all the patients, study participants and their families for their contributions to this effort. We thank the investigators and research staff at all study sites. We also thank the National Healthcare Big Data (East) Center-Nanjing for assisting with the Jinling cohort follow-up by providing the healthcare database. The present study was sponsored by Nanjing Geneseeq Technology Inc. We thank A. Margaritescu and L. Hummel for their assistance with language editing. These authors contributed equally: Hua Bao, Shanshan Yang, Xiaoxi Chen. Hua Bao, Shanshan Yang, Xiaoxi Chen, Shuyu Wu, Xi Cheng, Xuxiaochen Wu, Wanxiangfu Tang, Min Wu, Shiting Tang, Dongqin Zhu, Ruowei Yang, Shuang Chang, Peng He, Xiuxiu Xu, JinPeng Zhang, Yi Shen, Ya Jiang, Sisi Liu, Xian Zhang, Xue Wu, Xiaonan Wang & Yang Shao Nanjing Jiangbei New Area Center for Public Health Service, Nanjing, China The Fourth Affiliated Hospital of Nanjing Medical University, Nanjing, China Department of Thoracic Surgery and Oncology, The First Affiliated Hospital of Guangzhou Medical University, Guangzhou, China Department of Liver Surgery and Transplantation, Liver Cancer Institute, Zhongshan Hospital, Fudan University, Shanghai, China Colorectal Center, Jiangsu Cancer Hospital, Nanjing, China State Key Laboratory of Molecular Oncology, National Cancer Center, Cancer Hospital of the Chinese Academy of Medical Sciences, Beijing, China Department of Thoracic Surgery, Nanjing Drum Tower Hospital, Nanjing, China Department of Gynecologic Oncology, Sun Yat-sen Memorial Hospital, Sun Yat-sen University, Guangzhou, China Pancreas Center, The First Affiliated Hospital of Nanjing Medical University, Nanjing, China Department of Clinical Laboratory, Guangdong Provincial People's Hospital, Guangzhou, China Department of Thoracic Surgery, Eastern Theater Command Hospital, Nanjing, China Department of Colorectal Surgery, Fudan University Shanghai Cancer Center, Shanghai, China Department of Gynaecology, Second Xiangya Hospital of Central South University, Changsha, China Department of Gynecologic Oncology, Beijing Obstetrics and Gynecology Hospital, Beijing, China Department of Thoracic Surgery, The First Affiliated Hospital of China Medical University, Shenyang, China Department of Gastrointestinal Medical Oncology, Harbin Medical University Cancer Hospital, Harbin, China Department of Respiratory Medicine, First People's Hospital of Changzhou, Third Affiliated Hospital of Soochow University, Changzhou, China You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Conceptualization: H.B., S.Y., X. Zhang, Xue Wu and Y. Shao. Methodology: H.B., S.Y., X. Chen, X. Cheng, S.W., M.W., S.T., S.C., P.H., X. Xu, J.Z., Y. Shen, Y.J., S.L. Validation: H.B., S.Y., X. Chen, Xiuxiu Xu and S.L. Formal analysis: H.B., S.Y., X. Chen and X. Cheng. Visualization: X. Chen, Xiuxiu Xu and W.T. Writing––review and editing: H.B., S.Y., X. Chen, X. Cheng, Xiuxiu Xu, W.T., S.L. Project administration: H.B., S.Y., Xue Wu, G.D., Y.M. Supervision: Xue Wu, X. Wang, X. Zhang and Y. Shao. Correspondence to Yang Shao. H.B., S.Y., X. Chen, S.W., X. Cheng, W.T., M.W., S.T., D.Z., R.Y., S.C., P.H., X. Xu, J.Z., Y. Shen, Y.J., S.L., X.Z., Xue Wu, X. Wang and Y. Shao are employees of Nanjing Geneseeq Technology Inc. The other authors declare no competing interests. Nature Medicine thanks Ziding Feng, Dominic Rothwell and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Ulrike Harjes, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) Confusion matrix displaying the predicted and actual cancer types for the internal validation set. (b) Confusion matrix displaying the predicted and actual cancer types for the independent validation set. (c) TOP2 accuracy rates for different cancer types in the internal validation set, with error bars representing the 95% confidence interval of accuracy. The number of patients in each cancer type is indicated in parentheses. (d) TOP2 accuracy rates for different cancer types in the independent validation set, with error bars representing the 95% confidence interval of accuracy. The number of patients in each cancer type is indicated in parentheses. This analysis is based on independent patient samples, with the unit of study being the individual patient. No technical or biological replicates were performed at the individual patient sample level for this analysis; the ‘n' values represent counts of unique patients. The majority of participants (N=2,110) were followed for 12–15 months, while 724 participants were followed for 15–18 months, and 847 participants were followed for over 18 months. Procedures applied to all participants included assessments of the lungs, liver, thyroid, kidneys, pancreas, and intestines. Gender-specific procedures encompassed evaluations of the breasts, cervix, and prostate. Days to cancer diagnosis for each false negative case. The concordance between cancer cases identified by the MCED test and those detected through existing screening protocols. Screening (USPSTF) or breast ultrasonography: cancers detected by both the MCED test and screening methods recommended by the United States Preventive Services Task Force (USPSTF) or breast ultrasonography. Screening (no USPSTF): cancers detected by both the MCED test and screening methods not currently recommended by USPSTF. Screening at the following year: cancers detected by the MCED test during screening at the following year's appointment. Clinical suspicion only: cancers detected by the MCED test based on clinical suspicion only. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Bao, H., Yang, S., Chen, X. et al. Early detection of multiple cancer types using multidimensional cell-free DNA fragmentomics. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-60099-4'>The APOE isoforms differentially shape the transcriptomic and epigenomic landscapes of human microglia xenografted into a mouse model of Alzheimer's disease</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-27 00:31:34
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Microglia play a key role in the response to amyloid beta in Alzheimer's disease (AD). In this context, the major transcriptional response of microglia is the upregulation of APOE, the strongest late-onset AD risk gene. Of its three isoforms, APOE2 is thought to be protective, while APOE4 increases AD risk. We hypothesised that the isoforms change gene regulatory patterns that link back to biological function by shaping microglial transcriptomic and chromatin landscapes. We use RNA- and ATAC-sequencing to profile gene expression and chromatin accessibility of human microglia xenotransplantated into the brains of male APPNL-G-F mice. We identify widespread transcriptomic and epigenomic differences which are dependent on APOE genotype and are corroborated across the profiling assays. Our results indicate that impaired microglial proliferation, migration and immune responses may contribute to the increased risk for late-onset AD in APOE4 carriers, while increased phagocytic capabilities and DNA-binding of the vitamin D receptor in APOE2 microglia may contribute to the isoform's protective role. Microglia are key players implicated in the genetic susceptibility and progression of Alzheimer's disease (AD). AD genetic risk predominantly falls within regulatory regions of the genome, including those marked by H3K27ac1, an epigenetic modification found at active enhancers and promoters2. H3K27ac is dysregulated in the brains of individuals with AD3,4, and microglial H3K27ac regions are strongly enriched for AD genetic risk5,6. Microglia have also been associated with AD risk in their open chromatin regions7,8,9,10, and at the level of their transcriptome11,12,13. Recent research using single-cell transcriptomics has highlighted that microglia occur in various distinct subtypes and activation states, which are anticipated to exhibit different epigenomic and transcriptomic responses dependent on their environmental niche. This variation is expected to give rise to different downstream effects on AD pathogenesis. Supporting this is the continued characterisation of different microglial phenotypes in AD, including those responsive to amyloid beta (Aβ) aggregates, a pathological hallmark of AD14,15,16. In mouse models, the strongest transcriptional response of microglia to Aβ aggregates is the upregulation of the gene APOE14,17,18, which harbours the strongest genetic risk factor for late-onset AD. APOE is involved in regulating cholesterol and other lipid transport across cells19. In the periphery, it is produced by macrophages in the liver, while in the brain, it is primarily produced by astrocytes19. In humans, APOE has uniquely evolved into three different isoforms encoded by the alleles: APOE2, APOE3, and APOE4. In AD, APOE2 is thought to be protective, while APOE4 increases disease risk up to 12-fold in homozygous individuals of certain human populations20. Several studies have begun to explore the role of microglial APOE in AD, using different samples and methodologies. In mouse models of AD, mice expressing the APOE4 isoform exhibit a higher abundance of microglia stress and inflammatory markers, a phenotype also observed in human tissue15. Furthermore, APOE4 microglia are linked to dysregulated lipid metabolism21,22, followed by tau phosphorylation22. Regarding immune responses, APOE4 microglia induce the signalling of transforming growth factor-β (TGF-β), a multifunctional cytokine23. Clearly, APOE plays a critical role in regulating microglia in response to AD pathology. Thus far, studies have predominantly focused on APOE4, but it is equally important to determine the role of APOE2, which may potentially be antagonistic. We hypothesised that different APOE isoforms would differentially regulate the microglial phenotype in response to Aβ pathology. However, the unique evolution of APOE in humans makes it difficult to faithfully recapitulate its effects on AD risk and pathogenesis in mouse models. As previously suggested, investigating the microglial response in human tissue is challenging due to technical limitations and inconsistencies in biological findings24. To tackle these challenges, researchers have developed a human microglia xenotransplantation model25,26, in which iPSC-derived human microglia are xenografted into the brains of mice. Single-cell profiling of these microglia has identified known and novel amyloid-responsive states27. These microglial states were enriched for different subsets of AD genetic risk genes, highlighting that multiple microglial states are influenced by AD genetic susceptibility. In addition, shift into what is thought to be a protective and human-specific microglial state was impaired in APOE4 microglia16. In addition to demonstrating the usefulness of this model in disentangling the microglial response to Aβ pathology, this highlights the need to investigate the functional role of AD genetic risk factors in a cell type-specific manner. Here, we used ATAC-seq and RNA-seq to profile human microglia expressing the different APOE isoforms, which were xenotransplanted into the AppNL-G-F mouse model of AD. This enabled us to delineate the effects of the different APOE isoforms on the epigenomic and transcriptomic landscapes of microglia in AD. We transplanted iPSC-derived human microglia APOE2/0, APOE3/0, APOE4/0 and an APOE knockout (APOE-KO) into the brains of the AppNL-G-F mouse model of Alzheimer's disease28. At 12 months, by which point Aβ pathology is extensive18,28, microglia were isolated by FACS using human microglia-specific antibodies (CD11b + hCD45+, Fig. This approach results in a scenario where the manipulations of the APOE genotype are restricted to microglia, thereby allowing us to study microglia-autonomous effects of the isoforms. To characterise the epigenomic and transcriptomic landscapes of these microglia, they were profiled using ATAC-seq for open chromatin and RNA-seq for gene expression, respectively. After quality control and pre-processing, we obtained high-quality chromatin accessibility data across 16 mice (APOE2 = 5, APOE3 = 5, APOE4 = 4, APOE-KO = 2; Supplementary Data 1, Supplementary Fig. 1) and high-quality transcriptomic data across 17 mice (APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3; Supplementary Data 1, Supplementary Fig. Overall, we observed widespread differences in gene expression and chromatin accessibility across microglia of the different APOE isoforms, highlighting the complexity of the microglial response to Aβ pathology. In support of the opposing roles of APOE2 and APOE4 in AD risk, the greatest differences were observed between these isoforms. a Experimental design for xenotransplantation of iPSC-derived human microglia into the brains of AppNL-G-F mice, followed by ATAC-seq (APOE2 = 5, APOE3 = 5, E4 = 4, APOE-KO = 2) and RNA-seq (APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3). The two-sided Wilcoxon rank-sum test was used to calculate p-values. The central mark and edges indicate the 50th (median), 25th and 75th percentiles. Whiskers correspond to 1.5 * the interquartile range (IQR). c Genome tracks showing chromatin accessibility signals of all APOE alleles around the APOE locus. d Stacked barplot of the number of differentially expressed genes (DEGs; FDR < 0.05) identified through pairwise comparisons across the experimental groups. APOE expression was significantly lower in three out of five knockout samples (Supplementary Fig. 2), and only these were used in downstream analyses (Fig. Additionally, APOE expression was lower in the APOE4 microglia compared to APOE2 and APOE3. This is in agreement with previous studies investigating the APOE alleles in microglia and astrocytes29,30. Chromatin accessibility around the transcriptional start site (TSS) of APOE was consistent across the APOE groups (Fig. As the most commonly expressed allele20, we used APOE3 as the baseline for pairwise comparisons in the differential expression and chromatin accessibility analyses performed using DESeq231. In addition, we compared the microglia expressing APOE4 with those expressing APOE2. When compared to APOE3 microglia, differential expression analysis revealed 470 (282 up, 188 down, (FDR < 0.05); Figs. 1d, 2a, Supplementary Data 2) and 332 (170 up, 158 down, (FDR < 0.05); Figs. 1d, 2b, Supplementary Data 2) differentially expressed genes (DEGs) in APOE2 and APOE4, respectively. As expected, given their postulated opposing roles in AD risk, the direct comparison of APOE4 with APOE2 revealed the most differences, with 1639 DEGs (751 up, 888 down, (FDR < 0.05; Figs. 126 genes were upregulated and 206 were downregulated in the APOE-KO when compared with the APOE3 isoform (Supplementary Fig. a Differentially expressed genes in APOE2 vs APOE3 microglia. b Differentially expressed genes in APOE4 vs APOE3 microglia. c Differentially expressed genes in APOE4 vs APOE2 microglia. d RRHO heatmap comparing expression signatures between APOE-KO vs APOE3 and APOE2 vs APOE3. e RRHO heatmap comparing expression signatures between APOE-KO vs APOE3 and APOE4 vs APOE3. f Scatterplot of APOE4 vs APOE3 logFC against APOE2 vs APOE3 logFC for genes with expression profiles overlapping with the APOE-KO. g Multi-marker Analysis of GenoMic Annotation (MAGMA) gene set analysis using the differentially expressed genes across the APOE alleles with three independent Alzheimer's disease (AD) genome-wide association study (GWAS)35,43,87. For the RRHO analysis, a one-sided hypergeometric test was used and raw p-values were plotted in the heatmap. Comparison with the APOE-KO enabled us to infer whether the transcriptional mechanisms underlying APOE2 and APOE4 microglia can be explained by loss and/or gain of function. We used Rank-Rank Hypergeometric Overlap (RRHO) to quantify the degree of overlap between expression signatures in the APOE-KO vs APOE3 and APOE2 vs APOE3, and the APOE-KO vs APOE3 and APOE4 vs APOE3. We observed significant overlap between expression patterns in the APOE-KO and APOE2 comparison (Spearman's rank correlation, rho = 0.55, p < 2.2e−16; Fig. 2d), as well as for the APOE-KO and APOE4 variants (Spearman's rank correlation, rho = 0.34, p < 2.2e−16; Fig. The strongest overlap was seen for genes downregulated in both APOE2 and APOE4, suggesting that the mechanisms underlying the different APOE isoforms can be in part explained by a loss of APOE function. However, there were also unique transcriptional changes occurring in APOE2 and APOE4 microglia (Fig. This corroborates findings reported by Machlovi et al.32 in which the authors performed similar analyses investigating the APOE4 and APOE3 isoforms in mouse microglia. All genes that had the same direction of expression change in APOE2 and the APOE-KO, also had the same direction of expression change in APOE4, when compared with APOE3 (Fig. A few genes were only significant in either APOE2 or APOE4, including TSPAN13, which was upregulated in APOE4 microglia and is associated with lipid accumulation in microglia22,33. Due to the pleiotropic nature of APOE, we evaluated whether genes differentially expressed across the APOE groups exhibited differential enrichment for AD genetic risk variants. Using MAGMA gene set analysis34, we found that genes downregulated in the APOE4 and APOE-KO microglia were enriched for risk variants identified in one AD GWAS35 (FDR < 0.05; Fig. This supports previous evidence suggesting that the mechanisms of APOE4 reflect a loss-of-function in the context of AD16,23,36, with APOE4 affecting biological pathways that are consistent with those linked to the polygenic component of AD. To evaluate the upstream regulatory mechanisms associated with the transcriptomic changes across the APOE isoforms, we next investigated changes in chromatin accessibility. When compared to APOE3 microglia, APOE2 microglia had 40 differentially accessible regions (DARs) (24 up, 16 down, FDR < 0.05; Fig. 3a, Supplementary Data 4), APOE4 microglia had 50 DARs (38 up, 12 down, FDR < 0.05; Fig. Again, the direct comparison of APOE4 with APOE2 revealed the most differences, with 72 DARs (52 up, 20 down, FDR < 0.05; Supplementary Fig. 4a, Supplementary Data 4), with the fewest changes observed in the KO (14 up, 8 down, FDR < 0.05; Supplementary Fig. Notably, our analysis revealed consistent epigenomic and transcriptomic responses across microglia expressing the different APOE isoforms (Supplementary Fig. For instance, CHCHD2, a mitochondrial gene involved in promoting cellular migration37 and implicated in Parkinson's disease (PD)38,39, was significantly downregulated in APOE4 microglia when compared to both APOE3 (logFC = –7.8, p = 1.3e−11; Figs. 2b, 3c, Supplementary Data 2) and APOE2 (logFC = –7.4, p = 7.8e−11; Figs. In parallel, chromatin accessibility was significantly reduced close to the TSS of this gene when compared to APOE3 (logFC = −7.8, p = 1.2e−7, distance to TSS = 0; Fig. 3b, d, Supplementary Data 4) and APOE2 (logFC = −6.5, p = 8.3e−9, distance to TSS = 0; Supplementary Fig. Expression of this gene was also significantly reduced in the APOE-KO, suggesting a potential loss of protective function via this gene in the APOE4 microglia (Supplementary Fig. Similarly, the zinc finger protein ZNF248 was upregulated in APOE4 microglia (APOE4 vs APOE3, logFC = 9, p = 5.1e−14; APOE4 vs APOE2, logFC = 7.9, p = 5.2e−19; Supplementary Data 2) and genomic regions in the vicinity of this gene had increased chromatin accessibility (APOE4 vs APOE3, logFC = 6.3, p = 2.3e−5; APOE4 vs APOE2, logFC = 6.6, p = 3.2e−5; Supplementary Data 4). Interestingly, in an in vitro study investigating functional and transcriptional phenotypes of a TREM2 mutant and knockout in iPSC-derived microglia-like cells, ZNF248 was upregulated in the TREM2-KO, while CHCHD2 expression was reduced in the R47H mutant40. For an overall assessment of the concordance between ATAC-seq and RNA-seq, we correlated the logFC values between DEGs and DARs at the corresponding promoter peaks. We observed a strong correlation across all APOE comparisons, indicating general concordance between changes in chromatin and gene expression (Supplementary Fig. Microglia-specific regulatory regions originating from human samples are strongly enriched for AD genetic risk5,6,10,41. To evaluate whether human iPSC-derived microglia xenotransplanted into the mouse brain would recapitulate this enrichment we used stratified linkage disequilibrium score regression (s-LDSC)42 with AD GWAS43. We found that open chromatin regions from the xenotransplanted microglia were enriched for AD heritability (FDR < 0.05, Fig. As the xenotransplanted microglia are predominantly responding to Aβ in our model, this enrichment also suggests that a significant proportion of AD risk is associated with microglial reactions to this pathological hallmark44. By repeating the analysis using GWAS data for autism spectrum disorder45 and amytrophic lateral sclerosis (ALS)46, we confirmed that this enrichment was specific to AD, and not a general brain disease enrichment (Fig. a, b Heatmaps showing differential chromatin accessibility of significant peaks (FDR < 0.05) when comparing (a) APOE2 vs APOE3, and (b) APOE4 vs APOE3. Shown are the genes annotated to the top 20 most significant peaks. Genes marked in bold were also significantly differentially expressed in the RNA-seq analysis. c Boxplot of expression profiles of CHCHD2 shows reduced expression in the APOE4 and the APOE-KO microglia. Expression profiles were derived from biological replicates: APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3. The two-sided Wilcoxon rank-sum test was used to calculate p-values. The central mark and edges indicate the 50th (median), 25th and 75th percentiles. Whiskers correspond to 1.5 * the IQR. d Genome tracks of chromatin accessibility signals around the CHCHD2 locus show a loss of the open chromatin peak at the CHCHD2 promoter in APOE4 and KO. e S-LDSC analysis using all open chromatin regions from the xenotransplanted human microglia with genome-wide association study (GWAS) summary statistics for Alzheimer's disease (AD), amyotrophic lateral sclerosis (ALS), and autism spectrum disorder (ASD) shows a microglia-specific enrichment for AD. Data are presented as enrichment values +/− SD. Open chromatin regions were derived from biological replicates: APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 2. Human microglia from the xenotransplantation model used here were previously profiled using single-cell RNA sequencing27. Mancuso et al. (2024) report eight microglial states responsive to Aβ pathology, including previously characterised disease-associated microglia (DAM), as well as novel states annotated as cytokine response (CRM) and antigen-presenting response (HLA) microglia. Using hypergeometric testing, we found that genes dysregulated across the microglia with different APOE isoforms were strongly enriched within several microglia clusters (Fig. 4a): The strongest association was observed for genes downregulated in APOE4 microglia, which were enriched in the HLA, ribosomal microglia (RM), and DAM clusters (Fig. Furthermore, genes downregulated in APOE4 microglia were enriched in the DAM cluster associated with negative regulation of tumour necrosis (TNF) cytokine production (Fig. Since negative regulation of cytokine production refers to processes that inhibit cytokine production, the downregulation of these genes in APOE4 microglia suggests increased cytokine production in this isoform, which was confirmed in the gene expression data (Fig. CRM microglia mount a pro-inflammatory response driven by the upregulation of chemokines and cytokines, and have only been characterised in human16. Mancuso and colleagues16 showed that in response to Aβ, APOE4 microglia shift to the CRM state rather than HLA. In agreement, Machlovi et al. (2022) report increased cytokine production in APOE4 microglia. Conversely, where the authors found increased TNFα in APOE4 microglia, TNF family members, including TNFRSF25 and TNFRSF21, were upregulated in APOE2 microglia in our study. Taken together with the decreased HLA and DAM response in APOE4 microglia here, these findings suggest that APOE4 microglia fail to transition towards microglial states that are thought to be protective14,16 (Fig. a Heatmap showing enrichment of genes differentially expressed across the APOE groups amongst microglia clusters defined by scRNA-seq. b Dotplot of pathway enrichment analysis using genes downregulated in APOE4 microglia that are enriched in the disease associated microglia (DAM) cluster. c–f Boxplots of gene expression profiles of cytokines (c) CCL3, (d) CCL3L1, (e) CCL4L2, and (f) CXCL16. Expression profiles were derived from biological replicates: APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3). FDR corrected p-values were derived from the differential expression analysis using DESeq2: p < 0.05 (*), p < 0.01 (**), p < 0.001 (***), and for each comparison, can be found in Supplementary Data 2. Whiskers correspond to 1.5 * the IQR. We used weighted gene co-expression network analysis (WGCNA)47 to identify microglial gene modules with similar expression profiles (Fig. These modules were then tested for differential expression across the APOE groups, and the differentially expressed modules were functionally characterised using pathway enrichment analysis. We identified two differentially expressed modules significantly associated with GO biological processes. First, a gene module upregulated in APOE2 microglia when compared to both APOE3 and APOE4 was associated with proliferation and cellular migration pathways (Fig. Such pathways are likely important for microglia being recruited towards the site of Aβ pathology and initiating its clearance, which in some cases, requires APOE48. Second, a module upregulated in APOE2 when compared to APOE4 was significantly associated with a range of immune responses, including both innate immune responses such as complement activation, and adaptive immune responses such as antibody-mediated immunity (Fig. a WGCNA heatmap showing the correlation coefficients and p-values of modules across the APOE groups. b Boxplot of eigengene expression of the WGCNA module associated with proliferation and migration. The central mark and edges indicate the 50th (median), 25th and 75th percentiles. Whiskers correspond to 1.5 * the IQR. Expression profiles were derived from biological replicates: APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3). c Gene ontology (GO) biological processes enriched for genes within the WGCNA module associated with proliferation and migration that is upregulated in APOE2 microglia. d Boxplot of eigengene expression of the WGCNA module associated with immune responses. The central mark and edges indicate the 50th (median), 25th and 75th percentiles. Whiskers correspond to 1.5 * the IQR. Expression profiles were derived from biological replicates: APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3). e GO biological processes enriched for genes within the WGCNA module associated with immune responses, which was upregulated in APOE2 microglia. In the heatmaps, Pearson correlation coefficients and Fisher's asymptotic p-values are shown. For the pathway enrichment analysis, p-values were computed using a one-sided Fisher's exact test. Previous studies have linked the APOE2 isoform to enhanced phagocytic capabilities49,50,51. Using a recently generated scRNA-seq dataset from phagocytic microglia associated with Aβ plaques52, genes upregulated in APOE2-expressing microglia when compared to both APOE3 and APOE4, were exclusively overrepresented in a set of genes upregulated in phagocytic microglia (Fig. To functionally validate this finding, we performed phagocytosis assays using iPSC-derived human microglia using pHrodo E. coli particles and fluorescent myelin. As hypothesised, APOE2-expressing microglia internalised significantly higher amounts of pHrodo E. coli compared to all other APOE groups (Fig. Additionally, a significantly higher proportion of APOE2-expressing microglia successfully took up myelin compared to APOE4-expressing microglia (Fig. We also observed that APOE-KO microglia displayed very high amounts of intracellular myelin. This could indicate an impairment to digest the phagocytosed material, and is in line with previous studies showing lipid accumulation in ApoE−/− mouse microglia in models of demyelination53. Overall, the enrichment of proliferation, migration, phagocytosis, and immune responses suggests enhanced microglial function in APOE2. a Barplot showing the overrepresentation of genes upregulated in APOE2 microglia in differentially expressed genes (DEGs) in phagocytic microglia responsive to Aβ plaques52. The dashed line represents the significance threshold after FDR correction (p < 0.05). b, c Representative images and quantification of pHrodo E. Coli particles (100 µg/ml) uptake by APOE2, APOE3, APOE4 and APOE-KO iPSC-derived microglia. Each data point represents an independent well (n = 7). An average of 1236 + /− 87 cells were quantified per well. d, e Representative images and quantification of PKH67-labelled myelin (200 µg/ml) uptake by APOE2, APOE3, APOE4, and APOE-KO iPSC-derived microglia. Each data point represents an independent well (n = 7). An average of 211 ± 9 cells were quantified per well. Statistical analysis was performed using a one-way ANOVA, with Bonferroni correction for multiple comparisons; * p < 0.05, ** p < 0.02 and *** p < 0.001. Data are presented as the mean +/- standard error of the mean. To better understand the regulatory machinery of human microglia in Alzheimer's disease and the upstream orchestrators of altered transcriptional states, we performed de novo motif enrichment analysis using HOMER54 on the open chromatin regions from the xenotransplanted microglia. Specifically, we used the top 100 peaks with increased and decreased chromatin accessibility across the APOE isoforms as input and defined all ATAC-seq peaks as the background set. Regions with increased accessibility in the APOE2 microglia were strongly enriched for the DNA binding motif of the vitamin D receptor (VDR; Fig. 7a), a ligand-inducible transcription factor (TF) and main mediator of vitamin D signalling55. Importantly, vitamin D deficiency has been linked to increased risk for AD56,57. We next assessed whether VDR target genes were upregulated in APOE2 microglia. A hypergeometric test confirmed an overrepresentation of genes upregulated in APOE2 when compared to both APOE3 and APOE4 (Fig. 7b), in a list of monocytic VDR target genes identified in a previous study58. This confirms the expected downstream transcriptional response predicted by increased VDR binding in APOE2 microglia. Activation of an anti-inflammatory microglia phenotype via IL-10 and vitamin D signalling has been reported previously59 (Fig. Consistent with this, in our data, the alpha subunit of the IL-10 receptor (IL-10RA) was significantly upregulated in APOE2 microglia (Fig. We also checked to see whether VDR target genes were associated with any of the WGCNA modules identified in this study (Supplementary Fig. The ME1 (dark green) module was enriched for upregulated VDR target genes, and this module was also identified as upregulated in APOE2-expressing microglia. The ME1 module was characterised by biological pathways associated with immune responses (Fig. The ME2 (blue) module was also enriched for upregulated VDR target genes, however, this module was not differentially expressed across the APOE groups (FDR < 0.05). Aside from the VDR enrichment, regions with increased chromatin accessibility in APOE4-expressing microglia when compared to APOE2 were enriched for STAT2, a TF involved in interferon response signalling60. De novo motif enrichment analysis results for all APOE isoform comparisons can be found in Supplementary Data 7. a Heatmap showing enrichment of motifs in regions with increased chromatin accessibility in APOE2 microglia. b Barplot showing the overrepresentation of genes upregulated in APOE2 microglia in a list of vitamin D receptor (VDR) target genes in monocytes90. P-values were computed using a one-sided hypergeometric test. c Illustration showing a mechanism of anti-inflammatory microglia activation mediated through vitamin D and IL-10 signalling59. The two-sided Wilcoxon rank-sum test was used to calculate p-values. Expression profiles were derived from biological replicates: APOE2 = 5, APOE3 = 4, APOE4 = 5, APOE-KO = 3). The central mark and edges indicate the 50th (median), 25th and 75th percentiles. Whiskers correspond to 1.5 * the IQR. Increasing evidence points to a highly complex response of microglia to AD pathology. Here, we show that the human APOE2, APOE3, and APOE4 differentially regulate microglia in the context of Aβ aggregates. By profiling human microglia isolated from a xenotransplantation model of AD using RNA-seq and ATAC-seq, we uncovered widespread changes to the transcriptomic and chromatin landscape of this cell type, dependent on the APOE isoform expressed. As anticipated, the largest differences were observed when comparing the AD risk opposing APOE2 and APOE4 microglia. First, we observed consistent epigenomic and transcriptomic responses for several genes, including CHCHD2 and ZNF248. CHCHD2 is involved in promoting cell migration37 and has been linked to familial and sporadic Parkinson's disease (PD)38, where it is transcriptionally downregulated. Its decreased expression and chromatin accessibility in APOE4 microglia, but also in the knockout, suggest a loss of normal function. Conversely, ZNF248 was upregulated in APOE4 and the knockout in both assays, suggesting a potential gain of toxic function. In a study comparing the effects of TREM2 knockout and a TREM2 mutation in a model of human microglia, the TREM2 knockout had deficits in phagocytosis, chemotaxis, and survival that were not observed in the TREM2 mutant40. ZNF248 was one of only four differentially expressed genes with reduced expression in the knockout but increased expression in the mutant. Although the authors argue that it is unlikely that such a limited number of genes, including ZNF248, could explain such vast phenotypic differences40, the overlap between APOE4 microglia and TREM2 knockout microglia is interesting. The convergence of results between the DEGs and DARs highlights the robustness of using multiple independent assays to profile cellular states in a disease context. Overlapping expression signatures with the APOE knockout enabled us to infer whether the APOE2 and APOE4 alleles resulted in a loss or gain-of-function. Although both alleles overlapped with the knockout, only genes downregulated in APOE4 microglia and the knockout were both enriched for AD genetic risk, lending support to previous reports of APOE4 microglia increasing AD risk through loss-of-function mechanisms16,23. Further investigation into the genes shared between the knockout and APOE4 microglia highlighted a strong upregulation of TSPAN13. This gene is also upregulated in microglia that accumulate damaging lipid droplets in the ageing brain33, and in microglia homozygous for APOE4, in response to Aβ22. Our data suggest that lipid dysregulation in APOE4 microglia may be driven by a loss of function. Genes downregulated in APOE4 were enriched within distinct microglial states identified in response to Aβ: HLA, RM, and DAM27. HLA represents a novel, human-specific microglial state that has a pronounced response to Aβ pathology and is thought to play a protective role27. RM are enriched for ribosomal genes. In murine AD models, stage 2 DAM cells, which signal the full activation of the DAM programme that is thought to be protective, are enriched for ribosomal genes14. When considered collectively, these enrichments suggest that APOE4 microglia fail to shift into protective states. Furthermore, our analyses point toward diminished migratory capacity in APOE4 and enhanced migratory capacity in APOE2 microglia. This is supported in previous studies which have shown that migration is decreased in APOE4 microglia-like cells61, APOE4 microglia have reduced motility and responsiveness to ATP, a chemotactic cue of neuronal damage, and amyloid beta62. In contrast, genes associated with microglial migration are upregulated in APOE2 microglia51. Previously, APOE4 microglia were shown to downregulate their expression of cellular migration genes in response to demyelination51, and pericytes derived from APOE4 carriers exhibited downregulation of genes associated with cellular migration63. This also suggests that migratory capacity is not a cell type or pathology-specific mechanism affected by APOE in AD. In concordance with signatures of cellular migration, APOE2-expressing microglia also exhibited enhanced phagocytic activity when compared to APOE3 and APOE4 microglia. This observation is in agreement with previous studies which report a lower accumulation of myelin debris in APOE2-TR mice51, and increased clearance of Aβ from the interstitial fluid of mice expressing the APOE2 isoform50 In addition, APOE2-expressing macrophages derived from transgenic mice with an APP mutation were also more efficient at degrading amyloid beta than both APOE3 and APOE4-expressing macrophages49. Another mechanism through which APOE4 may be exerting its pathogenic role in AD is by mounting a cytokine response. In other studies, mouse microglia expressing the humanised APOE4 allele increased cytokine production32, and xenotransplanted human microglia shifted towards a pro-inflammatory state27. Similarly, we report a general upregulation of cytokines in APOE4 when compared to both APOE2 and APOE3 microglia. However, this chemokine has been reported to drive microglia to an anti-inflammatory phenotype in brain tumours64. In addition to changes to cytokine expression, regions with increased chromatin accessibility in APOE4-expressing microglia were enriched for STAT2, a TF involved in interferon response signalling. Increased interferon signalling as well as upregulation of associated genes has already been shown in mouse microglia expressing APOE432. Machlovi et al. (2022) also reported increased TNFα expression in APOE4 microglia, while we observed the opposite—nine TNF family genes were increased in APOE2 when compared to APOE4 (Supplementary Data 1). Whether a cytokine is pro-inflammatory is context dependent, and it is therefore difficult to conclude whether the upregulation of these TNF family genes is driving a pro-inflammatory response. In addition, it remains unclear to what extent pro-inflammatory responses are protective versus pathogenic in AD, and what triggers the increased production of pro-inflammatory cytokines in the APOE isoforms. For instance, in APOE2 microglia the upregulation of TNF family genes may be triggered by their migration towards and interaction with Aβ plaques. Whereas in APOE4 microglia, the increased production of pro-inflammatory cytokines could be due to a lack of response to Aβ, which in turn would result in continued Aβ deposition and a sustained pro-inflammatory response. Several studies have shown an enrichment of AD genetic risk within microglia-specific genes and regulatory regions from the human brain5,6,11. Here, using the chromatin accessibility profiles from the xenotransplanted microglia we recapitulate this enrichment, highlighting the robustness of this model for investigating human genetic risk in a disease context. At the level of the transcriptome, AD genetic risk was enriched within genes downregulated in APOE4 microglia but also the APOE knockout. Supporting previous studies, this overlap suggests that AD risk increased by the presence of APOE4 is partially mediated through loss of protective function. Our findings underscore the need to consider the interplay between genetic risk factors and microglial states in AD. In addition to arguing for increased proliferation, migration, phagocytosis and immune response in APOE2 microglia as underlying this isoform's protective effect, we report a potential upstream regulatory role for the VDR. In the context of AD, low levels of vitamin D have been associated with a higher incidence of the disease56,57, and vitamin D supplementation has been shown to improve disease outcomes65,66. It's important to take into consideration APOE genotype, as some studies have shown that APOE4 carriers have higher vitamin D levels67,68 and therefore vitamin D supplementation may be more beneficial to non-carriers66. Vitamin D acts via binding to VDR, and enrichment of VDR in regions with increased chromatin accessibility in APOE2 may therefore enable these microglia to be more responsive to vitamin D, regardless of serum levels. Furthermore, the increased expression of the IL-10RA in APOE2 microglia was particularly interesting. IL-10 then activates SOCS3 via the IL-10 receptor, and this mechanism suppresses the expression of pro-inflammatory cytokines. Several other studies have also demonstrated an association between vitamin D and the expression of anti-inflammatory factors in microglia69,70,71,72,73,74. The functional role of VDR activation and binding warrants further studies in terms of mechanisms and therapeutic exploration. Our study has a number of limitations. First, microglia exist in different subtypes and states. For example, microglia associated with Aβ plaques may have distinct transcriptomic and epigenomic profiles compared to less responsive microglia. While it is worth considering the contributions of APOE from mouse microglia and astrocytes, these should remain consistent across the APOE groups and, as such, should not affect differential expression and chromatin accessibility analyses. Although transcriptomic and epigenomic profiling provide valuable insights into gene regulatory mechanisms, other factors, such as histone modifications, also play a significant role in AD3,4,41. It is also worth considering that the APOE isoforms might exert risk or protection prior to the onset of Aβ pathogenesis. Future studies could adopt a time-series approach to investigate this. We linked regions with differential chromatin accessibility to differentially expressed genes based on their proximity. While this approach may capture promoter-gene relationships, many DARs may also function as enhancers. These enhancer-gene links can regulate target genes up to a megabase away, making them more challenging to identify. Using appropriate chromatin interaction data such as Hi-C could help disentangle these connections. Finally, while the absence of an adaptive immune system is necessary to prevent xenograft rejection, it may lead to unaccounted for changes in the microglial response75. Our work sheds light on the regulation of microglia in AD: we show that it is dependent on APOE isoform, at both the level of the transcriptome and epigenome, further highlighting the complexity of this cell type in response to Aβ. Our work suggests that APOE4 microglia have compromised microglial functions including diminished migratory capacity and heightened pro-inflammatory responses compared to APOE2, and these may underlie the increased risk of AD seen in carriers of this isoform. Furthermore, our findings underscore the importance of considering the interplay between genetic risk factors, such as APOE, and microglial states in disease progression. Importantly, we highlight the potential involvement of the VDR in modulating microglial responses, providing new avenues for therapeutic exploration. Overall, the use of the microglia xenotransplantation model coupled with genome-wide profiling has enabled us to dissect the regulatory landscape of microglia expressing the different APOE isoforms. In future, this approach could be extended to other relevant genes. In summary, our study emphasises the complex interplay between genetic, epigenetic, and environmental factors in shaping microglial responses in AD and underscores the need for targeted interventions based on APOE genotype. The lines were obtained via Bioneer from the European Bank for induced pluripotent Stem Cells (EBiSC), where all originating tissue donors give fully informed consent for donation, generation of iPSCs and sharing of lines for research. This set of APOE isogenic cell lines is based on the male iPSC cell line BIONi010-C. Stem cells were cultured on Matrigel in E8 Flex medium, dissociated at ~70–80% confluence, and aggregated in U-bottom 96-well plates with BMP4, VEGF, and SCF. Embryoid bodies were then transferred to six-well plates and cultured in X-VIVO medium with additional cytokines, followed by a media switch on day 11. By day 18, microglial precursors were harvested and transplanted into P4 mouse brains after depleting endogenous microglia with CSF1R inhibitor BLZ945. Ethics approval for transplantation of human iPS cells was obtained by the Ethics Committee Research UZ / KU Leuven (study S65730). AppNL-G-F mice were crossed with homozygous Rag2tm1.1Flv Csf1tm1(CSF1)Flv Il2rgtm1.1Flv Apptm3.1Tcs mice (Jacksons Lab, strain 017708) to generate the Rag2-/- Il2rγ-/- hCSF1KI AppNL-G-F used in this study. In total, we transplanted 500,000 cells bilaterally across 20 mice. Male mice were used for the xenotransplantations to limit biological variability. Mice had access to food and water ad libitum and were housed with a 14/10 h light-dark cycle at 21 °C in groups of two to five animals. Animal experiments were approved by the local Ethical Committee of Laboratory Animals of the KU Leuven (government licence LA1210579 ECD project number P177/2017) following local and EU guidelines. Five biological replicates were prepared per experimental group: APOE2, APOE3, APOE4, APOE KO. Animals were anaesthetised with an overdose of sodium pentobarbital and transcardially perfused with ice cold PBS. Samples were obtained in the morning between 9 and 11 am. From each sample at 12 months, FACS purification of the following cell numbers were attained: 100,000 cells for ATAC-seq, 200,000 cells for RNA-seq. The FACS plots with the gating strategy can be seen in Supplementary Fig. ATAC-seq samples were processed immediately after cell collection for tagmentation and elution of transposed DNA (details in ATAC-seq methods section). RNA-seq was conducted from cell pellets snap frozen in liquid nitrogen. From snap frozen cell pellets of 200,000 cells per sample, RNA was extracted using the Monarch Total RNA Miniprep Kit (T2010) following the manufacturer's instructions. RNA-seq was conducted using the rRNA depletion strategy rather than mRNA enrichment so that noncoding RNAs could be recovered76. rRNA depletion was performed using NEBNext rRNA Depletion Kit v2 Human/Mouse/Rat with RNA Sample Purification Beads (E7405), followed by stranded (directional) library preparation using the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (E7765) following manufacturer's protocols without adjustments. RNA quality was checked using the Agilent RNA 6000 Pico Kit (5067-1513) and final libraries were assessed using Agilent High Sensitivity DNA Kit (5067-2646) where all libraries were appropriate for sequencing apart from one replicate of the APOE3 microglia—therefore this isoform only has four biological replicates for RNA-seq. ATAC-seq was conducted as previously described77. Following FACS collection of 100,000 cells per sample, cells were spun down at 500 g for 5 min at 4 °C, and the supernatant was removed. Cell pellets were gently resuspended in 50 µL of ice cold Lysis Buffer (10 mM Tris-HCl pH 7.4, 10 mM NaCl, 3 mM MgCl2, 0.1% IGEPAL CA-630). 2.5 µL Tagment DNA Enzyme (Illumina; 20034197) was added directly and gently mixed by pipetting. The transposition reaction was incubated at 37 °C for 30 min, then transferred to ice. DNA was purified immediately with the Zymo ChIP DNA Clean and Concentration Kit (D5205) following manufacturer's instructions. The DNA column was spun dry prior to elution of transposed DNA, which was conducted with 11 µL Elution Buffer. Purified DNA was stored at -20 °C until library preparation. 10 µL DNA per sample was transferred into a PCR tube and 34.25 µL PCR master mix was added per sample. 6.25 µL of 10 µM Nextera Primer 2 (with barcode) was added per sample, where a different barcode was used for each sample to enable multiplexing. Amplified library was purified using the Zymo ChIP DNA Clean and Concentration Kit (D5205). The purified library was eluted using 20 µL Elution Buffer. 5 µL of 5x TBE Loading Buffer (Invitrogen; LC6678) was added and loaded in a 12-well 10% TBE gel (Invitrogen; EC62752BOX). A ladder was prepared using 0.25–0.5 µL ORangeRuler 50 bp DNA Ladder (ThermoFisher; SM0613) diluted in 5 µL 5x TBE loading buffer. The gel was run at 70 V until DNA enters the gel, then increased to 140 V for approximately one hour. The gel was stained using 10 mL 1x TBE with SYBR Gold Nucleic Acid Gel stain (Invitrogen; S11494) diluted at 1:10,000 (1 µL). The gel was cut between 175 and 225 bp markers into a 0.5 mL DNA LoBind tube perforated three times with a 22 G needle. The gel was shredded by centrifugation at maximum speed for 2 min at room temperature into a 1.5 mL DNA LoBind tube. 150 µL Diffusion Buffer (0.5 M Ammonium Acetate, 0.1% SDS, 1 mM EDTA, 10 mM Magnesium Acetate, ddH2O) was added to the gel in the 1.5 mL tube and shaken at room temperature for 45 min. The sample was then transferred to filter columns using wide-bore tips and spun at max speed for 2 min. DNA was purified (~140 µL) using the Zymo ChIP DNA Clean and Concentration Kit and eluted with 10 µL Elution Buffer into 1.5 mL DNA LoBind tubes. Final libraries were quantified with the Qubit 1X dsDNA HS Assay Kit (ThermoFisher; Q33230) and stored at −20 °C prior to sequencing (yield: ~0.25 ng/µL). Final library size distributions were assessed by Agilent 2100 Bioanalyser and Agilent 4200 TapeStation for quality control before sequencing. Libraries were pooled to achieve an equal representation of the desired final library size range (equimolar pooling based on Bioanalyser/TapeSation signal in the 150–800 bp range). Paired-end Illumina sequencing using the HiSeq 4000 PE75 strategy was conducted on barcoded libraries at the Imperial Biomedical Research Centre (BRC) Genomics Facility following the manufacturer's protocols. General QC of each sample was assessed using fastQC (https://www.bioinformatics.babraham.ac.uk/projects/fastqc/), followed by adaptor trimming using TrimGalore! Reads were aligned to GRCh38 using bowtie2 with the following arguments: –local –very-sensitive –no-mixed –no-discordant -I 25 -X 1000. Read count generation was performed using featureCounts78. Additionally, peaks were filtered using the filterByExpr() function in DESeq231, retaining only peaks with sufficiently high counts for statistical analysis. This left 167,951 peaks for downstream analyses. The 2 APOE-KO samples with high APOE expression in the RNA-seq data were also excluded from the ATAC-seq dataset. Additionally, another APOE-KO and one APOE4 microglia sample were discarded due to having a low number of reads after QC filtering (<9 million). General QC of each sample was assessed using fastQC (https://www.bioinformaticsf.babraham.ac.uk/projects/fastqc/), followed by adaptor trimming using TrimGalore! Reads were aligned to the GRCh38 genome and transcriptome using STAR79. Duplicate and multi-mapping reads were retained. Transcript quantification was performed using Salmon80, using the GC bias flag. Two of the APOE knockout samples with high APOE expression were excluded from subsequent analyses (Supplementary Fig. DESeq231 was used for the differential expression and differential chromatin accessibility analysis. DESeq231 was designed for the differential analysis of RNA-seq data and has since been widely used for this purpose. In a recent study comparing methods for differential analysis of ATAC-seq read counts, Gontarz et al (2020)81 showed that with five replicates, which we have for most of our samples, DESeq2 had the lowest false positive rate and a true positive recall comparable to other methods available for differential accessibility analysis. For both analyses, the APOE3 microglia samples were used as a baseline for comparison, and we additionally tested for differences between APOE4 and APOE2 microglia. To perform the differential analysis, we used the DESeq() function which provides a wrapper for three functions: estimateSizeFactors() for estimation of size factors, estimateDispersions() for estimation of dispersion, and nbinomWaldTest() for negative binomial GLM fitting and Wald statistics. Genes and peaks were defined as being significant if p < 0.05 after FDR correction. To identify which genes had similar expression profiles across the APOE groups, we used weighted gene co-expression network analysis (WGCNA)47. First, transcripts with zero or low expression counts were filtered out using the filterByExpr() function in edgeR82. As suggested by the authors of WGCNA, the count data was normalised by variance stabilising transformation and explored for outliers using principal component analysis. An appropriate soft thresholding power was chosen to ensure a scale-free network and used as input to the blockwiseModules() function in WGCNA to calculate the adjacency matrix. This function was also used to detect gene co-expression modules and to calculate module eigengenes. As defined by the authors47, the module eigengenes are the first principal component of a given module and can be considered to represent the gene expression profile of that module. The module eigengenes identified in the WGCNA analysis were used to perform differential expression analysis using the lmFit() function in limma83, across the APOE groups. As the only differentially expressed modules were associated with APOE2, the genes belonging to these modules were used to perform pathway enrichment analysis using clusterProfiler84, allowing characterisation of the APOE2-associated modules based on their gene ontology (GO) enrichments. GO terms were considered to be significantly associated with the given modules if p < 0.05 after FDR correction. To estimate the proportion of disease SNP-heritability attributable to open chromatin regions in the xenotransplanted microglia, we performed stratified linkage disequilibrium score regression (s-LDSC). Annotation files were generated and used to compute LD scores. Publicly available GWAS summary statistics for a recent AD GWAS43 were downloaded and converted to the required format for LDSC. Steps for the analysis were followed as instructed here https://github.com/bulik/ldsc/wiki and required files were downloaded from https://alkesgroup.broadinstitute.org/LDSCORE/GRCh38. LDSC was run using the full baseline model, thereby computing the proportion of SNP-heritability associated with the annotation of interest, while taking into account all the annotations in the baseline model. As we observed a significant enrichment, we repeated the analysis using GWAS data for PD85, Amyotropic Lateral Sclerosis46, and Autism spectrum disorder86, to ensure this was not a generic neurological enrichment. MAGMA gene set analysis34 was used to assess the enrichment of AD SNP-based heritability among differentially expressed genes across the APOE isoforms. Summary statistics for three independent AD GWAS were downloaded35,43,87 and formatted for use with MAGMA using MungeSumstats88. P-values were corrected using the FDR method. Microglia progenitors, as generated according to the MIGRATE protocol, were collected on day 18 and plated in differentiation medium (DMEM/F12 supplemented with 200 mM L-glutamine, 1:2000 human insulin, 5 ng/mL N-Acetyl-L-cysteine, 50 mg/mL apo-transferrin, 20 μg/mL sodium selenite, 1 μg/mL heparin sulphate, 50ng/mL M-CSF, 100 ng/mL IL-34, 2 ng/mL transforming growth factor (TGF)-β, 10 ng/mL fractalkine (CX3CL1), 1.5 ng/mL cholesterol) in a concentration of 3.75*105 cells/mL in 8 well chamber slides (Ibidi, 80841). On day 6, microglia were incubated with 100 μg/mL pHrodo Red E.coli BioParticles (Thermofisher, P35361) for 1 h, or 200 μg/mL human myelin debris, as isolated from human brain tissue according to previously published protocol89 and fluorescently labelled according to instructions of the MINI67 cell membrane labelling kit (Sigma-Aldrich, MINI67) for 3 h. Cultures were rinsed with PBS three times and fixed in 4% paraformaldehyde for 15 min. Fixed cells were permeabilized with 1% Triton X-100 in PBS for 5 min and nonspecific binding was blocked with 5% bovine serum albumin (BSA) in PBS for 10 min. Microglia were incubated with anti- ionized calcium-binding adaptor molecule 1 (Iba1; WAKO, 019-1974, 1:500), followed by incubation with secondary Alexa Fluor 488 or 594 donkey anti-rabbit antibodies (Invitrogen A21206 and A21207, 1:400), both in 1% BSA in PBS for 1 h at RT. This was followed by counterstaining with 1.67 μg/mL 4′,6-diamidino-2-phenylindole (DAPI; Merck, D9542) for 15 min at RT. Finally, cells were mounted with Mowiol solution and stored at 4 °C until imaging. For each well (7 per genotype and phagocytosis stimulus), three fields were imaged using a Nikon A1R Eclipse Ti confocal microscope. For each field, a z-stack of three images was acquired with steps of 2.95 μm at 10x magnification for the E.coli conditions and 0.88 μm at 20x magnification for the myelin conditions. Images were further analyzed with a custom made FIJI script (v1.53). Maximum intensity projections were generated from z-stacks, and microglia masks were generated using the ‘Analyze Particles' function on the Iba1 channel, followed by manual verification. The number of microglia positive for E. coli/myelin phagocytosis was counted with the use of the ‘Cell Counter' plugin. Thereafter, the area of E. coli/myelin signal within microglia positive for phagocytosis was measured in μm². Values from three fields per well were averaged for analysis. Differences in phagocytosis between genotypes were assessed with one-way ANOVA and Bonferroni's multiple comparisons post-hoc test in GraphPad Prism (v10.2.0). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. FASTQ and read count files have been deposited in the Gene Expression Omnibus (GEO) database under accession code GSE271384 for the ATAC-seq dataset and GSE271385 for the RNA-seq dataset. All supplementary data files are available on Zenodo: https://doi.org/10.5281/zenodo.15202660. Source data are provided with this paper. https://zenodo.org/records/15202660 Source data are provided with this paper. All the data and code required to reproduce the figures in this manuscript are available in our GitHub repository: https://github.com/Marzi-Lab/APOE_microglia and on Zenodo: https://doi.org/10.5281/zenodo.15176132. Hannon, E., Marzi, S. J., Schalkwyk, L. S. & Mill, J. Genetic risk variants for brain disorders are enriched in cortical H3K27ac domains. Creyghton, M. P. et al. Histone H3K27ac separates active from poised enhancers and predicts developmental state. Marzi, S. J. et al. A histone acetylome-wide association study of Alzheimer's disease identifies disease-associated H3K27ac differences in the entorhinal cortex. Nativio, R. et al. An integrated multi-omics approach identifies epigenetic alterations associated with Alzheimer's disease. Brain cell type–specific enhancer–promoter interactome maps and disease-risk association. Murphy, K. B. et al. CHAS infers cell type-specific signatures in bulk brain histone acetylation studies of neurological and psychiatric disorders. Corces, M. R. et al. Single-cell epigenomic analyses implicate candidate causal variants at inherited risk loci for Alzheimer's and Parkinson's diseases. Morabito, S. et al. Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease. Kosoy, R. et al. Genetics of the human microglia regulome refines Alzheimer's disease risk loci. Xiong, X. et al. Epigenomic dissection of Alzheimer's disease pinpoints causal variants and reveals epigenome erosion. Skene, N. G. & Grant, S. G. N. Identification of vulnerable cell types in major brain disorders using single cell transcriptomes and expression weighted cell type enrichment. Bryois, J. et al. Genetic identification of cell types underlying brain complex traits yields insights into the etiology of Parkinson's disease. Sun, N. et al. Human microglial state dynamics in Alzheimer's disease progression. Keren-Shaul, H. et al. A unique microglia type associated with restricting development of Alzheimer's disease. Millet, A., Ledo, J. H. & Tavazoie, S. F. An exhausted-like microglial population accumulates in aged and APOE4 genotype Alzheimer's brains. Mancuso, R. et al. Xenografted human microglia display diverse transcriptomic states in response to Alzheimer's disease-related amyloid-β pathology. The TREM2-APOE pathway drives the transcriptional phenotype of dysfunctional microglia in neurodegenerative diseases. The major risk factors for Alzheimer's disease: age, sex, and genes modulate the microglia response to Aβ plaques. Liu, C.-C., Liu, C.-C., Kanekiyo, T., Xu, H. & Bu, G. Apolipoprotein E and Alzheimer disease: risk, mechanisms and therapy. & Hyppönen, E. Apolipoprotein E (APOE) genotype-associated disease risks: a phenome-wide, registry-based, case-control study utilising the UK Biobank. Lee, S. et al. APOE modulates microglial immunometabolism in response to age, amyloid pathology, and inflammatory challenge. Haney, M. S. et al. APOE4/4 is linked to damaging lipid droplets in Alzheimer's disease microglia. Yin, Z. et al. APOE4 impairs the microglial response in Alzheimer's disease by inducing TGFβ-mediated checkpoints. Mancuso, R. et al. Stem-cell-derived human microglia transplanted in mouse brain to study human disease. Hasselmann, J. et al. Development of a chimeric model to study and manipulate human microglia in vivo. Fattorelli, N. et al. Stem-cell-derived human microglia transplanted into mouse brain to study human disease. Mancuso, R. et al. Xenografted human microglia display diverse transcriptomic states in response to Alzheimer's disease-related amyloid-ß pathology. Single App knock-in mouse models of Alzheimer's disease. Lin, Y.-T. et al. APOE4 causes widespread molecular and cellular alterations associated with Alzheimer's disease phenotypes in human iPSC-derived brain cell types. Lanfranco, M. F., Sepulveda, J., Kopetsky, G. & Rebeck, G. W. Expression and secretion of apoE isoforms in astrocytes and microglia during inflammation. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Machlovi, S. I. et al. APOE4 confers transcriptomic and functional alterations to primary mouse microglia. Author Correction: Lipid-droplet-accumulating microglia represent a dysfunctional and proinflammatory state in the aging brain. de Leeuw, C. A., Mooij, J. M., Heskes, T. & Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. Kunkle, B. W. et al. Genetic meta-analysis of diagnosed Alzheimer's disease identifies new risk loci and implicates Aβ, tau, immunity and lipid processing. Liu, C.-C. et al. Cell-autonomous effects of APOE4 in restricting microglial response in brain homeostasis and Alzheimer's disease. Seo, M., Lee, W.-H. & Suk, K. Identification of novel cell migration-promoting genes by a functional genetic screen. Shi, C.-H. et al. CHCHD2 gene mutations in familial and sporadic Parkinson's disease. Li, K. et al. Downregulation of CHCHD2 may contribute to Parkinson's disease by reducing expression of NFE2L2 and RQCD1. Hall-Roberts, H. et al. TREM2 Alzheimer's variant R47H causes similar transcriptional dysregulation to knockout, yet only subtle functional phenotypes in human iPSC-derived macrophages. Cell type-specific histone acetylation profiling of Alzheimer's disease subjects and integration with genetics. Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. Jansen, I. E. et al. Genome-wide meta-analysis identifies new loci and functional pathways influencing Alzheimer's disease risk. Novel Alzheimer risk genes determine the microglia response to amyloid-β but not to TAU pathology. Meta-analysis of GWAS of over 16,000 individuals with autism spectrum disorder highlights a novel locus at 10q24. 32 and a significant overlap with schizophrenia. van Rheenen, W. et al. Common and rare variant association analyses in amyotrophic lateral sclerosis identify 15 risk loci with distinct genetic architectures and neuron-specific biology. Langfelder, P. & Horvath, S. WGCNA: an R package for weighted correlation network analysis. The VCAM1-ApoE pathway directs microglial chemotaxis and alleviates Alzheimer's disease pathology. Zhao, L. et al. Macrophage-mediated degradation of beta-amyloid via an apolipoprotein E isoform-dependent mechanism. Castellano, J. M. et al. Human apoE isoforms differentially regulate brain amyloid-β peptide clearance. Opposing effects of apoE2 and apoE4 on microglial activation and lipid metabolism in response to demyelination. Grubman, A. et al. Transcriptional signature in microglia associated with Aβ plaque phagocytosis. A. et al. TREM2 regulates microglial cholesterol metabolism upon chronic phagocytic challenge. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Carlberg, C. Vitamin D and its target genes. Littlejohns, T. J. et al. Vitamin D and the risk of dementia and Alzheimer disease. Chai, B. et al. Vitamin D deficiency as a risk factor for dementia and Alzheimer's disease: an updated meta-analysis. Warwick, T. et al. A hierarchical regulatory network analysis of the vitamin D induced transcriptome reveals novel regulators and complete VDR dependency in monocytes. A., Hayes, C. E. & Olson, J. K. Vitamin D3 alters microglia immune activation by an IL-10 dependent SOCS3 mechanism. & Asgari, N. Interferons in the central nervous system: a few instruments play many tunes. Konttinen, H. et al. PSEN1ΔE9, APPswe, and APOE4 confer disparate phenotypes in human iPSC-derived microglia. Sepulveda, J., Kim, J. Y., Binder, J., Vicini, S. & Rebeck, G. W. APOE4 genotype and aging impair injury-induced microglial behavior in brain slices, including toward Aβ, through P2RY12. Author correction: single-nucleus multiregion transcriptomic analysis of brain vasculature in Alzheimer's disease. Lepore, F. et al. CXCL16/CXCR6 axis drives microglia/macrophages phenotype in physiological conditions and plays a crucial role in glioma. Effects of vitamin D supplementation on cognitive function and blood Aβ-related biomarkers in older adults with Alzheimer's disease: a randomised, double-blind, placebo-controlled trial. Ghahremani, M. et al. Vitamin D supplementation and incident dementia: effects of sex, APOE, and baseline cognitive status. Huebbe, P. et al. APOE ε4 is associated with higher vitamin D levels in targeted replacement mice and humans. Dursun, E. et al. Vitamin D deficiency might pose a greater risk for ApoEɛ4 non-carrier Alzheimer's disease patients. Lefebvre d'Hellencourt, C., Montero-Menei, C. N., Bernard, R. & Couez, D. Vitamin D3 inhibits proinflammatory cytokines and nitric oxide production by the EOC13 microglial cell line. Calvello, R. et al. Vitamin D treatment attenuates neuroinflammation and dopaminergic neurodegeneration in an animal model of Parkinson's disease, shifting M1 to M2 microglia responses. He, J., Guo, X., Liu, Z.-Q., Yang, P.-C. & Yang, S. Vitamin D inhibits the Staphylococcal enterotoxin B-induced expression of tumor necrosis factor in microglial cells. Cui, C. et al. Vitamin D receptor activation regulates microglia polarization and oxidative stress in spontaneously hypertensive rats and angiotensin II-exposed microglial cells: Role of renin-angiotensin system. Clarke, J. et al. Vitamin D regulates MerTK-dependent phagocytosis in human myeloid cells. Lee, P. W. et al. Neuron-specific vitamin D signaling attenuates microglia activation and CNS autoimmunity. The adaptive immune system restrains Alzheimer's disease pathogenesis by modulating microglial function. Chen, X. et al. ATAC-see reveals the accessible genome by transposase-mediated imaging and sequencing. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Patro, R., Duggal, G., Love, M. I., Irizarry, R. A. & Kingsford, C. Salmon: fast and bias-aware quantification of transcript expression using dual-phase inference. Salmon: fast and bias-aware quantification of transcript expression using dual-phase inference. Comparison of differential accessibility analysis strategies for ATAC-seq data. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Identification of novel risk loci, causal insights, and heritable risk for Parkinson's disease: a meta-analysis of genome-wide association studies. Identification of common genetic risk variants for autism spectrum disorder. Bellenguez, C. et al. New insights into the genetic etiology of Alzheimer's disease and related dementias. Murphy, A. E., Schilder, B. M. & Skene, N. G. MungeSumstats: a bioconductor package for the standardization and quality control of many GWAS summary statistics. Rolfe, A. J., Bosco, D. B., Broussard, E. N. & Ren, Y. In vitro phagocytosis of myelin debris by bone marrow-derived macrophages. Nurminen, V., Seuter, S. & Carlberg, C. Primary vitamin D target genes of human monocytes. This work was funded by an Alzheimer's Association grant (grant number ADSF-21-829660-C) to SJM and BDS. This work was also supported by the UK Dementia Research Institute award number UKDRI-6205 through UK DRI Ltd, principally funded by the UK Medical Research Council. SJM received funding from the Edmond and Lily Safra Early Career Fellowship Program (https://www.edmondjsafra.org) and the Medical Research Council (grant number MR/W004984/1). BDS has funding from a Medical Research Council Grant (MR/Y014847/1), Fonds Wetenschappelijk Onderzoek (FWO) #12P5922N, Methusalem grant 3M140280, European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (grant agreement No. RM has funding from the European Research Council (ERC) under the European Union's Horizon 2020 Research and Innovation Programme (project no. 101041867—XenoMicrogliaAD), Fonds voor Wetenschappelijk Onderzoek (grants no. G0C9219N, G056022N and G0K9422N) and is a recipient of a postdoctoral fellowship from the Alzheimer's Association USA (fellowship no. He also receives funding from the Alzheimer's Association (E2A-23-1148152, 23AARF-1026404 and ABA-22-968700), BrightFocus Foundation (A2021034S), SAO-FRA (grant no. 2021/0021) and the University of Antwerp (BOF-TOP 2022-2025). KBM is a recipient of an MRC Doctoral Training Partnership award. We thank Annerieke Sierksma for helpful discussions and feedback. We thank the members of the International Neuroimmune Consortium for their insightful conversations and support in this project. The Imperial BRC Genomics Facility has provided resources and support that have contributed to the research results reported within this paper. The Imperial BRC Genomics Facility is supported by NIHR funding to the Imperial Biomedical Research Centre. UK Dementia Research Institute at Imperial College London, London, UK Kitty B. Murphy & Di Hu Kitty B. Murphy, Di Hu & Sarah J. Marzi Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK Leen Wolfs, Susan K. Rohde, Ivana Geric & Bart De Strooper Leen Wolfs, Susan K. Rohde, Ivana Geric & Bart De Strooper Gonzalo Leguía Fauró & Renzo Mancuso Gonzalo Leguía Fauró & Renzo Mancuso UK Dementia Research Institute at University College London, London, UK UK Dementia Research Institute at King's College London, London, UK You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar contributed to the study design and data interpretation. wrote the manuscript with input from all coauthors. All authors reviewed and approved the final manuscript. is or has been a consultant for Eli Lilly, Biogen, Janssen Pharmaceutica, Eisai, AbbVie and other companies. is also a scientific founder of Augustine Therapeutics and a scientific founder and stockholder of Muna Therapeutics. The remaining authors declare no competing interests. Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Murphy, K.B., Hu, D., Wolfs, L. et al. The APOE isoforms differentially shape the transcriptomic and epigenomic landscapes of human microglia xenografted into a mouse model of Alzheimer's disease. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            