
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-27</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s42255-025-01320-y'>Dietary control of peripheral adipose storage capacity through membrane lipid remodelling</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 17:19:21
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Genetic and dietary cues are known drivers of obesity, yet how they converge at the molecular level is incompletely understood. Here we show that PPARγ supports hypertrophic expansion of adipose tissue via transcriptional control of LPCAT3, an endoplasmic reticulum (ER)-resident O-acyltransferase that selectively enriches diet-derived omega-6 polyunsaturated fatty acids (n-6 PUFAs) in the membrane lipidome. In mice fed a high-fat diet, lowering membrane n-6 PUFA levels through genetic or dietary interventions results in aberrant adipose triglyceride (TG) turnover, ectopic fat deposition and insulin resistance. Additionally, we detail a non-canonical adaptive response in ‘lipodystrophic' Lpcat3–/– adipose tissues that engages a futile lipid cycle to increase metabolic rate and offset lipid overflow to ectopic sites. Live-cell imaging, lipidomics and molecular dynamics simulations reveal that adipocyte LPCAT3 activity enriches n-6 arachidonate in the phosphatidylethanolamine (PE)-dense ER–lipid droplet interface. Functionally, this localized PE remodelling optimizes TG storage by driving the formation of large droplets that exhibit greater resistance to adipose TG lipase activity. These findings highlight the PPARγ–LPCAT3 axis as a mechanistic link between dietary n-6 PUFA intake, adipose expandability and systemic energy balance. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Source data for all figures are provided with the paper. RNA-seq and sNuc-seq datasets generated for this paper are available in the NCBI Gene Expression Omnibus repository under accession code GSE218943. Reads were aligned to the mm10 (GRCm38) mouse reference genome. All source data and input files for MD simulations can be found at Zenodo94. All unique biological materials used are readily available from the authors. Source data are provided with this paper. Sakers, A., De Siqueira, M. K., Seale, P. & Villanueva, C. J. Adipose-tissue plasticity in health and disease. Wang, Q. A., Tao, C., Gupta, R. K. & Scherer, P. E. Tracking adipogenesis during white adipose tissue development, expansion and regeneration. Kim, S. M. et al. Loss of white adipose hyperplastic potential is associated with enhanced susceptibility to insulin resistance. Spalding, K. L. et al. Dynamics of fat cell turnover in humans. Kubota, N. et al. PPAR gamma mediates high-fat diet-induced adipocyte hypertrophy and insulin resistance. Medina-Gomez, G. et al. PPAR gamma 2 prevents lipotoxicity by controlling adipose tissue expandability and peripheral lipid metabolism. Lim, K., Haider, A., Adams, C., Sleigh, A. & Savage, D. B. Lipodistrophy: a paradigm for understanding the consequences of ‘overloading' adipose tissue. Acquired and inherited lipodystrophies. Slawik, M. & Vidal-Puig, A. J. Adipose tissue expandability and the metabolic syndrome. Bluher, M. Metabolically healthy obesity. Lotta, L. A. et al. Integrative genomic analysis implicates limited peripheral adipose storage capacity in the pathogenesis of human insulin resistance. Suzuki, K. et al. Genetic drivers of heterogeneity in type 2 diabetes pathophysiology. Hall, K. D. et al. The energy balance model of obesity: beyond calories in, calories out. Ludwig, D. S. et al. The carbohydrate–insulin model: a physiological perspective on the obesity pandemic. A critique of paradoxes in current advice on dietary lipids. Ludwig, D. S., Willett, W. C., Volek, J. S. & Neuhouser, M. L. Dietary fat: From foe to friend? Mouse models of inherited lipodystrophy. Speakman, J. R. Use of high-fat diets to study rodent obesity as a model of human obesity. Blasbalg, T. L., Hibbeln, J. R., Ramsden, C. E., Majchrzak, S. F. & Rawlings, R. R. Changes in consumption of omega-3 and omega-6 fatty acids in the United States during the 20th century. Ailhaud, G. et al. Temporal changes in dietary fats: role of n-6 polyunsaturated fatty acids in excessive adipose tissue development and relationship to obesity. Abe, I. et al. Lipolysis-derived linoleic acid drives beige fat progenitor cell proliferation. Hilgendorf, K. I. et al. Omega-3 fatty acids activate ciliary FFAR4 to control adipogenesis. Inazumi, T. et al. Prostaglandin E2–EP4 axis promotes lipolysis and fibrosis in adipose tissue leading to ectopic fat deposition and insulin resistance. Oh, D. Y. et al. GPR120 is an omega-3 fatty acid receptor mediating potent anti-inflammatory and insulin-sensitizing effects. Forman, B. M. et al. 15-Deoxy-Δ12,14-prostaglandin J2 is a ligand for the adipocyte determination factor PPARγ. Pietilainen, K. H. et al. Association of lipidome remodeling in the adipocyte membrane with acquired obesity in humans. Grzybek, M. et al. Comprehensive and quantitative analysis of white and brown adipose tissue by shotgun lipidomics. Hishikawa, D. et al. Discovery of a lysophospholipid acyltransferase family essential for membrane asymmetry and diversity. Natl Acad. Hashidate-Yoshida, T. et al. Fatty acid remodeling by LPCAT3 enriches arachidonate in phospholipid membranes and regulates triglyceride transport. Rong, X. et al. Lpcat3-dependent production of arachidonoyl phospholipids is a key determinant of triglyceride secretion. Demeure, O. et al. Regulation of LPCAT3 by LXR. He, M. et al. Inhibiting phosphatidylcholine remodeling in adipose tissue increases insulin sensitivity. Sarvari, A. K. et al. Plasticity of epididymal adipose tissue in response to diet-induced obesity at single-nucleus resolution. Emont, M. P. et al. A single-cell atlas of human and mouse white adipose tissue. Sun, K., Kusminski, C. M. & Scherer, P. E. Adipose tissue remodeling and obesity. Gregor, M. F. & Hotamisligil, G. S. Inflammatory mechanisms in obesity. Reilly, S. M. & Saltiel, A. R. Adapting to obesity with adipose tissue inflammation. Kazak, L. et al. A creatine-driven substrate cycle enhances energy expenditure and thermogenesis in beige fat. Ikeda, K. et al. UCP1-independent signaling involving SERCA2b-mediated calcium cycling regulates beige fat thermogenesis and systemic glucose homeostasis. Rubio-Cabezas, O. et al. Partial lipodystrophy and insulin resistant diabetes in a patient with a homozygous nonsense mutation in CIDEC. EMBO Mol. Reed, A. et al. LPCAT3 inhibitors remodel the polyunsaturated phospholipid content of human cells and protect from ferroptosis. ACS Chem. Ben M'barek, K. et al. ER membrane phospholipids and surface tension control cellular lipid droplet formation. Choudhary, V. et al. Architecture of lipid droplets in endoplasmic reticulum is determined by phospholipid intrinsic curvature. Gao, M., Huang, X., Song, B. L. & Yang, H. The biogenesis of lipid droplets: lipids take center stage. Bacle, A., Gautier, R., Jackson, C. L., Fuchs, P. F. J. & Vanni, S. Interdigitation between triglycerides and lipids modulates surface properties of lipid droplets. Prevost, C. et al. Mechanism and determinants of amphipathic helix-containing protein targeting to lipid droplets. Caillon, L. et al. Triacylglycerols sequester monotopic membrane proteins to lipid droplets. Lands, W. E. Metabolism of glycerolipides; a comparison of lecithin and triglyceride synthesis. Farvid, M. S. et al. Dietary linoleic acid and risk of coronary heart disease: a systematic review and meta-analysis of prospective cohort studies. Wu, J. H. Y. et al. Omega-6 fatty acid biomarkers and incident type 2 diabetes: pooled analysis of individual-level data for 39 740 adults from 20 prospective cohort studies. Lancet Diabetes Endocrinol. DeLong, C. J., Shen, Y. J., Thomas, M. J. & Cui, Z. Molecular distinction of phosphatidylcholine synthesis between the CDP-choline pathway and phosphatidylethanolamine methylation pathway. Horl, G. et al. Sequential synthesis and methylation of phosphatidylethanolamine promote lipid droplet biosynthesis and stability in tissue culture and in vivo. Thiam, A. R., Farese, R. V. Jr. & Walther, T. C. The biophysics and cell biology of lipid droplets. Cell Biol. & Carvalho, P. Dynamics and functions of lipid droplets. Cell Biol. Zadoorian, A., Du, X. & Yang, H. Lipid droplet biogenesis and functions in health and disease. Harayama, T. & Riezman, H. Understanding the diversity of membrane lipid composition. Cell Biol. Vanni, S., Hirose, H., Barelli, H., Antonny, B. & Gautier, R. A sub-nanometre view of how membrane curvature and composition modulate lipid packing and protein recruitment. Shindou, H. et al. Docosahexaenoic acid preserves visual function by maintaining correct disc morphology in retinal photoreceptor cells. Iizuka-Hishikawa, Y. et al. Lysophosphatidic acid acyltransferase 3 tunes the membrane status of germ cells by incorporating docosahexaenoic acid during spermatogenesis. Morigny, P., Houssier, M., Mouisel, E. & Langin, D. Adipocyte lipolysis and insulin resistance. Zimmermann, R. et al. Fat mobilization in adipose tissue is promoted by adipose triglyceride lipase. Kershaw, E. E. et al. Adipose triglyceride lipase: function, regulation by insulin, and comparison with adiponutrin. Tontonoz, P. & Spiegelman, B. M. Fat and beyond: the diverse biology of PPARgamma. Guan, H. P. et al. A futile metabolic cycle activated in adipocytes by antidiabetic agents. Gandotra, S. et al. Perilipin deficiency and autosomal dominant partial lipodystrophy. Chao, L. et al. Adipose tissue is required for the antidiabetic, but not for the hypolipidemic, effect of thiazolidinediones. Kim, J. K. et al. Differential effects of rosiglitazone on skeletal muscle and liver insulin resistance in A-ZIP/F-1 fatless mice. Liu, L. et al. Adipose-specific knockout of Seipin/Bscl2 results in progressive lipodystrophy. Qian, K. et al. CLSTN3β enforces adipocyte multilocularity to facilitate lipid utilization. Senault, C., Hlusko, M. T. & Portet, R. Effects of diet and cold acclimation on lipid composition of rat interscapular brown adipose tissue. Reitman, M. L. Metabolic lessons from genetically lean mice. Deol, P. et al. Soybean oil is more obesogenic and diabetogenic than coconut oil and fructose in mouse: potential role for the liver. PLoS ONE 10, e0132672 (2015). Gong, J. et al. Fsp27 promotes lipid droplet growth by lipid exchange and transfer at lipid droplet contact sites. Cell Biol. Nishino, N. et al. FSP27 contributes to efficient energy storage in murine white adipocytes by promoting the formation of unilocular lipid droplets. Zhou, L. et al. Insulin resistance and white adipose tissue inflammation are uncoupled in energetically challenged Fsp27-deficient mice. Mina, A. I. et al. CalR: a web-based analysis tool for indirect calorimetry experiments. Huynh, F. K., Green, M. F., Koves, T. R. & Hirschey, M. D. Measurement of fatty acid oxidation rates in animal tissues and cell lines. Herman, M. A. et al. A novel ChREBP isoform in adipose tissue regulates systemic glucose metabolism. Cui, L., Mirza, A. H., Zhang, S., Liang, B. & Liu, P. Lipid droplets and mitochondria are anchored during brown adipocyte differentiation. Hsieh, W. Y., Williams, K. J., Su, B. & Bensinger, S. J. Profiling of mouse macrophage lipidome using direct infusion shotgun mass spectrometry. STAR Protoc. Su, B. et al. A DMS shotgun lipidomics workflow application to facilitate high-throughput, comprehensive lipidomics. Patel, R. et al. ATGL is a biosynthetic enzyme for fatty acid esters of hydroxy fatty acids. Gruber, A. et al. The N-terminal region of comparative gene identification-58 (CGI-58) is important for lipid droplet binding and activation of adipose triglyceride lipase. Jo, S., Kim, T., Iyer, V. G. & Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. Jo, S., Lim, J. & Im, W. CHARMM-GUI Membrane Builder for mixed bilayers and its application to yeast membranes. B. et al. Update of the CHARMM all-atom additive force field for lipids: validation on six lipid types. Campomanes, P., Prabhu, J., Zoni, V. & Vanni, S. Recharging your fats: CHARMM36 parameters for neutral lipids triacylglycerol and diacylglycerol. Rep. (N Y) https://doi.org/10.1016/j.bpr.2021.100034 (2021). Gautier, R. et al. PackMem: a versatile tool to compute and visualize interfacial packing defects in lipid bilayers. Humphrey, W., Dalke, A. & Schulten, K. VMD: visual molecular dynamics. Shinoda, W., DeVane, R. & Klein, M. L. Zwitterionic lipid assemblies: molecular dynamics studies of monolayers, bilayers, and vesicles using a new coarse grain force field. Jimenez-Rojo, N. et al. Conserved functions of ether lipids and sphingolipids in the early secretory pathway. Tschöp, M. H. et al. A guide to analysis of mouse energy metabolism. Hu, S. et al. Dietary fat, but not protein or carbohydrate, regulates energy intake and causes adiposity in mice. Tol, M. J. et al. Dietary control of peripheral adipose storage capacity through membrane lipid remodelling. Laakso, M. et al. The Metabolic Syndrome in Men study: a resource for studies of metabolic and cardiovascular diseases. This article is dedicated to the memory of T. C. P. M. Kemper, a beloved friend and enduring source of inspiration. We are grateful to J. Sandhu, C. Priest and B. Clifford for technical assistance. We thank all current and former members of the Tontonoz and Tarling–Vallim labs for valuable discussions and for sharing reagents. PET–CT was performed at the Crump Institute Preclinical Imaging Technology Center, with the assistance of S. Xu and M. Tamboline. Oxylipin analysis was performed at the RIKEN Center for Integrative Medical Sciences with the assistance of M. Honda. TMT labelling was performed at the Pasarow Mass Spectrometry laboratory with the assistance of W. Cohn and J. P. Whitelegge. RNA-seq and sNuc-seq were performed at the Technology Center for Genomics & Bioinformatics. This work was supported by a postdoctoral fellowship from the American Diabetes Association (1-19-PDF-039 to M.J.T. ); the Japan Society for the Promotion of Science abroad and the Osamu Hayaishi Memorial Scholarship for Study Abroad (to Y.S. ); the National Institutes of Health (R01DK129276 to P.T. ); the Leducq Foundation (19CVD04 to P.T. ); the Swiss National Science Foundation (grants 310030_219264 to S.V. ); the European Research Council under European Union's Horizon 2020 Research and Innovation Programme (grant agreement no. and grants of the Swiss National Supercomputing Centre (CSCS) under project ID s1131 and s11876. These authors contributed equally: Marcus J. Tol, Yuta Shimanaka. Department of Pathology and Laboratory Medicine, University of California, Los Angeles, CA, USA Marcus J. Tol, Yuta Shimanaka, Alexander H. Bedard, Liujuan Cui, Alessandra Ferrari, Kevin Qian, John P. Kennelly, Stephen D. Lee, Yajing Gao, Xu Xiao & Peter Tontonoz Department of Biological Chemistry, University of California, Los Angeles, CA, USA Marcus J. Tol, Yuta Shimanaka, Alexander H. Bedard, Liujuan Cui, Alessandra Ferrari, Kevin Qian, John P. Kennelly, Stephen D. Lee, Yajing Gao, Xu Xiao, Kevin J. Williams, Baolong Su & Peter Tontonoz Department of Biology, University of Fribourg, Fribourg, Switzerland Jennifer Sapia & Stefano Vanni Institute of Molecular Biosciences, University of Graz, Graz, Austria Mariana Colaço-Gaspar, Peter Hofer & Rudolf Zechner Department of Neuroscience, The Ohio State University, Wexner Medical Center, Columbus, OH, USA Molecular Biology Institute, University of California, Los Angeles, CA, USA Julia J. Mack Department of Medicine, Division of Cardiology, University of California, Los Angeles, CA, USA Julia J. Mack, Thomas A. Weston, Calvin Pan, Aldons J. Lusis & Stephen G. Young Department of Human Genetics, University of California, Los Angeles, CA, USA Thomas A. Weston, Calvin Pan, Aldons J. Lusis & Stephen G. Young Department of Microbiology, Immunology, and Molecular Genetics, University of California, Los Angeles, CA, USA Calvin Pan & Aldons J. Lusis Edward A. Doisy Department of Biochemistry and Molecular Biology, and Center for Cardiovascular Research, Saint Louis University School of Medicine, St. Louis, MO, USA Daniel P. Pike & David A. Ford Department of Chemistry, The Scripps Research Institute, La Jolla, CA, USA Alex Reed, Natalia Milosevich & Benjamin F. Cravatt Division of Physiological Chemistry and Metabolism, Graduate School of Pharmaceutical Sciences, Keio University, Tokyo, Japan Laboratory of Metabolomics, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan BioTechMed-Graz, Graz, Austria Rudolf Zechner Biohealth Field of Excellence, University of Graz, Graz, Austria Swiss National Center for Competence in Research (NCCR) Bio-inspired Materials, University of Fribourg, Fribourg, Switzerland Stefano Vanni Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar conceived the project. were responsible for the methodology. contributed equally to the computational analyses. performed lipidomic analyses. conducted formal analyses. performed the investigation. provided resources. curated the data. wrote the manuscript. supervised the research. Correspondence to Peter Tontonoz. The authors declare no competing interests. Nature Metabolism thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Revati Dewal, in collaboration with the Nature Metabolism team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. (a) qPCR analysis of Lpcat1-4 mRNA levels in eWAT and iWAT of 12-week-old NCD-fed wild-type mice (n = 5, 5), and (b) in iWAT from wild-type mice fed low-fat diet (LFD) with or without rosiglitazone (Rosi; 50 mg kg–1) for 14 days (n = 7, 7). (c) Immunoblot analysis of LPCAT3 protein levels in iWAT lysates from (b). Tubulin served as a sample processing control. (d) Immunoblot analysis of LPCAT3 protein levels and known adipogenic markers during the course of 10T1/2 adipocyte differentiation. Calnexin served as a loading control for LPCAT3. (e) Sequence alignment of conserved PPAR and LXR response elements in the Lpcat3 promoter across multiple species. The PAM sequence of sgRNAs targeting Lpcat3-PPRE and LXRE are underlined in red. (f) ChIP-qPCR analysis of PPARγ (or normal rabbit IgG) occupancy at Lpcat3, Plin1, Fabp4, Agpat2, or Aqp7 PPRE elements in wild-type and ΔPPRE 10T1/2 adipocytes (n = 3/group; 100–mm plate per replicate from 3 independent experiments). A region of the 36b4 promoter served as a negative control. (g) PCR analysis of a 700-bp genomic segment flanking the PPRE/LXRE sites in the Lpcat3 promoter region of WT, ΔPPRE, and ΔLXRE clonally-derived cell lines, revealing no major deletions introduced by CRISPR/Cas9 gene editing. Primer anneal sites in the Lpcat3 promoter are shown in orange, indels in green (insertions) or red (deletions). (h) Immunoblot analysis of LPCAT3 protein levels in WT, ΔPPRE, and ΔLXRE 10T1/2 adipocytes differentiated for 4 days. Calnexin served as a loading control for LPCAT3. Data are presented as mean ± SEM. ***P < 0.001 by Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (b); or one-way ANOVA with Tukey's multiple comparisons test (f). (a) Graphical illustration of in vivo Lpcat3AKO strategy. (b) qPCR analysis of Lpcat3 mRNA levels in fat depots and insulin target-tissues of 12-week-old NCD-fed control and Lpcat3AKO mice (n = 4, 4). (c) Lipidomic analysis of PC/PE or TG-FA species in the inguinal adipocyte fraction isolated from of 22-week-old NCD-fed control and Lpcat3AKO mice (n = 5, 5). (d) Body weight (BW) gain curves and composition of 22-week-old NCD-fed control and Lpcat3AKO mice (n = 30, 34). (e) Wet weights of the indicated tissues in NCD-fed control and Lpcat3AKO mice (n = 23, 25). (f) H&E stainings of iBAT and iWAT depots (scale bar, 100 µm). (g) qPCR analysis of iWAT of NCD-fed control and Lpcat3AKO mice (n = 7, 7). (h) 14C-oleate uptake in insulin-target tissues of 18-week-old NCD-fed control and Lpcat3AKO mice (n = 5, 7). Radioactivity (counts per min, CPM) was calculated in the extracted lipids from whole-organs 4 h post-gavage with 14C-Triolein and normalized for g/tissue. (i) Ex vivo lipolysis in dissected iWAT explants from 18-week-old NCD-fed control and Lpcat3AKO mice (n = 4, 4). Secreted glycerol was measured under basal and stimulated (2 µM isoproterenol, ISO) conditions. (j) Ex vivo β-oxidation in crude lysates prepared from iBAT, iWAT, and livers of 18-week-old NCD-fed control and Lpcat3AKO mice (n = 5, 5), assessed by conversion of 14C-palmitic acid to 14CO2. (k) Plasma TGs and cholesterol in 22-week-old NCD-fed control and Lpcat3AKO mice (n = 13, 7). Data are presented as mean ± SEM. *P < 0.05, **P < 0.01, ***P < 0.001 by Welch's t-tests with Holm-Sidak's correction on CLR-transformed values of each lipid class (c); or Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (b,g). (a) Immunoblot analysis of phospho-Akt/PKB and rpS6 in iWAT of 12-week-old NCD-fed control and Lpcat3AKO mice after i.p. injection with insulin (1 U kg–1). Total Akt/PKB and rpS6 served as sample processing controls. (b) Coronal, sagittal, and transverse views of 12-week-old NCD-fed control and Lpcat3AKO mice by positron emission tomography–computed tomography (PET–CT) imaging 1 h post-injection with 18F-FDG and insulin (1 U kg–1). (c) 18F-FDG uptake (× 106 pCi g–1 tissue) in the indicated insulin-target tissues (n = 4, 4). (d) glucose tolerance tests (GTT; 1.5 g kg–1) and (e) insulin tolerance tests (ITT; 0.75 U kg–1) were performed on 18-week-old NCD-fed control and Lpcat3AKO mice (n = 10, 9). (f) Respiratory exchange ratios (RER) of 18-week-old NCD-fed control and Lpcat3AKO mice were monitored over a period of 48 h in Oxymax metabolic cages (n = 16, 15). Data are presented as mean ± SEM. (a) GS–MS analysis of dietary fatty acid profiles and presented as n-6/n-3 PUFA ratio, FA species (% of total), or FA class (sfa:mufa:pufa; n = 3, 3, 3). (b) Body weight (BW) gain curves of 18-week-old control and Lpcat3AKO mice fed either HFD (n = 42, 40), HFDsfa (n = 39, 38), or HFDn-3 (n = 40, 37) for 10 weeks. (c) Average daily food intake of singly-housed control and Lpcat3AKO mice fed the indicated HFDs, recorded during the last 5 weeks of HFD-feeding (n = 5–6/group). (d) EchoMRI analysis of lean mass in control and Lpcat3AKO mice fed HFD (n = 42, 40), HFDsfa (n = 39, 38), or HFDn-3 (n = 40, 37) for 10 weeks. (e) PUFA-derived oxylipin species were analysed in the inguinal adipocyte fraction from control and Lpcat3AKO mice fed the indicated HFDs for 10 weeks (n = 4/group). (f-i) Plasma (f) TG, (g) cholesterol, (h) ALT, and (i) AST levels in control and Lpcat3AKO mice fed the indicated HFDs for 10 weeks (n = 10–13/group). (j) VLDL secretion in control and Lpcat3AKO mice fed HFD (n = 5, 5), HFDsfa (n = 6, 6), or HFDn-3 (n = 6, 7) for 6 weeks. Animals were fasted for 12 h before injection with the LPL inhibitor poloxamer-407 (1 g kg–1; i.p.). Plasma TGs were measured in blood collected retro-orbitally at the indicated times post-injection. (k) Ex vivo hepatic β-oxidation rates in control and Lpcat3AKO mice fed the indicated HFDs for 6 weeks, as assessed by the conversion of 14C-palmitic acid (PA) to 14CO2 (n = 3–4/group). Data are presented as mean ± SEM. *P < 0.05, ***P < 0.001 by two-way RM ANOVA with Sidak's multiple comparisons test (b); ***P < 0.001 vs. the HFD-fed control group; #P < 0.05, vs. HFDsfa; same genotype, by two-way ANOVA with Tukey's multiple comparisons test (d); or **P < 0.01, ***P < 0.001 by Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (f,h,i). (a) 14C-oleate uptake in insulin-target tissues from 13-week-old control and Lpcat3AKO mice fed HFD for 5 weeks (n = 7, 8). Radioactivity (counts per min, CPM) was calculated in the extracted lipids from whole-organs 4 h post-gavage with 14C-Triolein and normalized for g/tissue. (b) Ex vivo β-oxidation in crude lysates prepared from iWAT, iBAT, or liver lysates of 13-week-old control and Lpcat3AKO mice fed HFD for 5 weeks, as assessed by the conversion of 14C-palmitic acid (PA) to 14CO2 (n = 4, 4). (c) Ex vivo lipolysis in freshly dissected iWAT explants from 13-week-old control and Lpcat3AKO mice fed HFD for 5 weeks (n = 3, 3). Secreted glycerol was determined under basal and stimulated (2 µM isoproterenol, ISO) lipolytic states. (d) 14C-oleate uptake in insulin-target tissues from 18-week-old control and Lpcat3AKO mice fed HFD for 10 weeks (n = 9, 8). (e,f) 14C-oleate uptake and turnover in (e) iBAT and (f) livers from 18-week-old control and Lpcat3AKO mice fed HFD for 10 weeks. Radioactivity (counts per min, CPM) was calculated in extracted lipids from whole-organs 4 h (n = 9, 8); 4 days (n = 6, 6); and 20 days (n = 7, 7) post-gavage with 14C-Triolein and normalized for g/tissue. (g) UMAP projection of 28,414 sequenced iWAT nuclei or split by genotype (13,466 and 14,948 nuclei for 18-week-old control vs. Lpcat3AKO mice fed HFD for 10 weeks). (h) Violin plots (clusters as columns, genes as rows) of cluster-specific markers. (i) Heatmap of normalized expression values of pan-adipogenic markers in inguinal adipocytes and the SVC fraction. Alternatively activated macrophages (AAMs), lipid-scavenging macrophages (LSMs), adipose stem and progenitor cells (ASPCs). Data are presented as mean ± SEM. ***P < 0.001 by Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (b-d). (a) Gene ontology (GO) terms and circos plots of upregulated genes in iWAT of 18-week-old control and Lpcat3AKO mice fed HFD for 10 weeks. GO terms were determined on the top 250 Nnat or Kcnc2 co-expressing iWAT gene transcripts. (b) Violin plots of representative markers for UCP1-dependent and -independent futile cycles. (a-c) qPCR analysis of pan-adipogenic, DNL, lipid cycling, thermogenic, and ER stress-associated gene expression in (a) iWAT, (b) eWAT, and (c) iBAT of 18-week-old control and Lpcat3AKO mice fed HFD for 10 weeks (n = 6, 6; ND, not detected). Data are presented as mean ± SEM. *P < 0.05, **P < 0.01, ***P < 0.001 by Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (a-c). (a) Heatmap of the top 70 differentially expressed genes (35 up or down) in iWAT of 18-week-old control and Lpcat3AKO mice fed the indicated HFDs for 10 weeks (n = 4/group). (b,c) GSEA analyses of the top 500 upregulated genes in (b) iWAT of HFDn-3 or HFDsfa-fed control mice vs. HFD-fed controls or (c) in eWAT of ob/ob Cidec–/– mice vs. ob/ob controls75. The ranked list metric is the product of log2 (fold-change)* –log10 (adjusted P value) from the differentially expressed genes in iWAT from HFD-fed Lpcat3AKO mice vs. controls (NES). (d) Correlation matrix depicting the association between Lpcat3AKO markers and clinical traits in METSIM95. Node colour and size reflect the correlation directionality and p value. Correlations were analysed by midweight bicorrelation coefficient or corrected p value using the R package WGCNA. (e) Body weight (BW) gain curves and (f) composition of 28-week-old control and Lpcat3AKO female mice fed HFD for 20 weeks (n = 14, 13). (g) Wet weight of the indicated tissues in HFD-fed control and Lpcat3AKO female mice (n = 14, 13). (h) Liver wet weights (% of BW; n = 14, 13). Data are presented as mean ± SEM. ***P < 0.001 by two-way RM ANOVA with Sidak's multiple comparisons test (e); Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (f,g); or Welch's t-test (h). (a) TIDE analysis (left) and Western blot analysis of LPCAT3 protein levels (right) validating Lpcat3-KO in 10T1/2 clonal cell lines transfected with sgRNAs targeting either exon 1 (L3_E1) or exon 3 (L3_E3). Calnexin served as a loading control (b) qPCR analysis of adipogenic markers in Lpcat3KO adipocytes expressing LPCAT3WT or LPCAT3H374A and differentiated for 6 days (n = 4, 4). (c) Shotgun lipidomic analysis of a wild-type 10T1/2 clonal cell line differentiated in the presence of 0.1% DMSO (veh ctrl) or the LPCAT3 inhibitor (R)-HTS-3 (10 µM) for 5 days. (d) Shotgun lipidomic analysis, (e) TG content (n = 3 wells/group), and (f) Oil red-O staining of Lpcat3KO adipocytes reconstituted with GFP Ctrl or the indicated GFP-tagged LPCAT3 fusion proteins (day 7 of differentiation). Data are presented as mean ± SEM. ***P < 0.001 by Welch's t-tests with FDR correction using the Benjamini, Krieger, and Yekutieli procedure (b); or one-way ANOVA with Tukey's multiple comparisons test (e). (a) Shotgun lipidomic analysis of PC composition in ER or LD–ER-enriched fractions from Lpcat3KO adipocytes expressing Flag-LPCAT3WT or Flag-LPCAT3H374A and differentiated for 6 days. Data are presented as the distribution of PC species (% of total PC) within each sample. (b) Schematic depiction of the trilayer model system containing a bulk oil phase flanked by phospholipid monolayers, solvated with water (left), and top/side views of trilayers exhibiting LPDs that expose the oil core to the solvent (right). (c) Deep/shallow LPDs in model bilayers and LD-like trilayer systems (± 3–5% surface tension, ST). Each replicate corresponds to a block-averaged LPD value derived from individual MD simulation trajectories (n = 3/group). WT and Lpcat3KO compositions reflect complex (left) and simplified (right) lipidomic approximations (Supplementary Table 6). (d,e) Immunoblot analysis of Lpcat3KO adipocytes expressing Flag-LPCAT3WT or Flag-LPCAT3H374A and treated with (d) ± isoproterenol (ISO, 10 µM), or (e) ± insulin (100 nM) for 15 min. Actin and Calnexin served as sample processing controls. (f) Tandem mass tag (TMT) labelling on 8,000g LDs isolated from Lpcat3KO adipocytes expressing Flag-LPCAT3WT or Flag-LPCAT3H374A (day 6 of differentiation). TMT data were overlaid with Uniprot mouse proteome for annotated LD localization. (g) Top/side snapshots from CG–MD simulations of an ER–LD neck system. Bilayer/monolayers consist of 50% PUFA (cyan) and 50% MUFA (white) phospholipids, with a triolein (TO; orange) core. (h) Electron microscopy analysis of Lpcat3KO adipocytes expressing Flag-LPCAT3WT or Flag-LPCAT3H374A (day 3 of differentiation). (i) Lipolysis in Lpcat3KO and Lpcat3/AtglDKO adipocytes expressing Flag-LPCAT3WT or Flag-LPCAT3H374A. Secreted glycerol was measured after ISO (10 µM) treatment for 2 h, normalized to intracellular TG stores (n = 3 wells/group). Where indicated, cells were differentiated in the presence of 0.1% DMSO (veh ctrl) or Atglistatin (ATGLi; 40 µM) for 6 days. Data are presented as mean ± SEM. Protein abundance ratios were compared across groups using Student's t-test (P < 0.01) while controlling for false-discovery rate with Benjamini-Hochberg procedure (f); or ***P < 0.001 by two-way ANOVA with Tukey's multiple comparisons test (i). Supplementary Data Fig. 1 and 5–7 uncropped images. Graph data for Figs. Source Data Extended Data Figs. 1, 3, 9 and 10 uncropped images. Graph data for Extended Figs. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Tol, M.J., Shimanaka, Y., Bedard, A.H. et al. Dietary control of peripheral adipose storage capacity through membrane lipid remodelling. Received: 07 February 2025 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250627095029.htm'>World's largest camera just snapped the Universe in 3,200 megapixels</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 15:44:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The camera took nearly two decades to build and involved hundreds of scientists across the globe, including a number of CNRS teams. The world-wide First Look unveiling event is held on June 23 at the National Academy of Sciences in Washington, D.C. The high-definition images, which use six different colour filters, capture the entire southern night-sky in just three nights of shooting. This worldwide premiere is the culmination of 25 years of research and construction by international teams, including several research teams from CNRS2. The exceptional quality of these initial images show that the telescope is ready to start its mission: to scan the entire southern hemisphere sky by taking 1,000 high-definition photographs using six colour filters, every three nights for the next ten years. Studied end-to-end, these scans will provide a high-definition, four-dimensional film of the evolving processes of the Universe. For the first time on a large scale, this vast survey will reveal the slightest changes in the Universe, from nearby celestial phenomena, such as asteroids and comets, to very distant ones, like supernovae. The project paves the way for major advances in cosmology in dark matter and dark energy, as well as our understanding of our solar system. As historic partners, SLAC called on CNRS scientists to help build the focal plane of the camera and help design and build its robotic filter exchange system, which will automatically change the camera's colour filters -- each weighing 24-38 kgs -- 5-15 times per night. Twenty terabytes of collected data will be stored every night. In France, the France Data Facility (IN2P3) (CNRS) in Lyon will store and process 40% of the collected raw image data. This data will be released to scientists around the world at regular intervals to foster groundbreaking discoveries and breakthroughs over the coming decades. Larger and more sensitive, ground-based instruments produce higher-precision exposures as a result. These instruments also record larger volumes of data than space-based ones, as the remote downloading of data from the latter remains a complex process. Last but not least, ground-based telescopes can also be repaired and improved with increasingly efficient tools. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250627095024.htm'>NASA discovers link between Earth's core and life-sustaining oxygen</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 14:57:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>For 540 million years, the ebb and flow in the strength of Earth's magnetic field has correlated with fluctuations in atmospheric oxygen, according to a newly released analysis by NASA scientists. Earth's magnetic field arises from the flow of material in the planet's molten interior, which acts like a giant electromagnet. Before addressing the complexity of the cause-and-effect relationship between magnetic fields and oxygen levels, the study authors decided to see whether Earth's magnetic field and atmosphere have fluctuated in ways that demonstrate a link. When hot minerals that rise with magma at gaps between spreading tectonic plates cool down, they can record the surrounding magnetic field. Scientists can deduce historic oxygen levels from ancient rocks and minerals because their chemical contents depend on the amount of oxygen available when they were formed. Data for both Earth's magnetic field and oxygen extend over comparable ranges in databases that myriad geophysicists and geochemists have compiled. "These two datasets are very similar," said coauthor Weijia Kuang, a geophysicist at NASA's Goddard Space Flight Center in Greenbelt, Maryland. "Earth is the only known planet that supports complex life. When Kuang and colleagues analyzed the two separate datasets, they found that the planetary magnetic field has followed similar rising and falling patterns as oxygen in the atmosphere for nearly a half billion years, dating back to the Cambrian explosion, when complex life on Earth emerged. The researchers hope to examine longer datasets to see if the correlation extends farther back in time. They also plan to investigate the historic abundance of other chemicals essential for life as we know it, such as nitrogen, to determine whether they also support these patterns. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a65221141/dotson-ice-shelf-discovery/'>A Submersible Uncovered Secret Structures. Then, It Vanished Under Antarctic Waters.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 14:48:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>“We have previously used satellite data and ice cores to observe how glaciers change over time,” Anna Wåhlin, the lead author of the study, said in a press statement. Unlike glaciers, which rest on land, ice shelves are actually a part of the ocean. They sort of act like a buttress that keeps ice on land from flowing into the ocean and raising sea levels, and are are vitally important to these polar ecosystem as a result. And because these shelves rest on the ocean, it's possible to get under them. The 20-foot-long Ran submersible used pulsed sound waves (advanced multibeam sonar) on the ice to map its features, but because of its subantarctic location, Wåhlin and her team couldn't communicate with the AUV or track its movements with GPS. After 14 missions—some lasting a couple hours and others stretching longer than a day—Ran mapped around 50 square miles of ice, and the structures imaged were more complex than anyone imagined. “The mapping has given us a lot of new data that we need to look at more closely,” Wåhlin said in a press statement. “It is clear that many previous assumptions about melting of glacier undersides are falling short. Current models cannot explain the complex patterns we see. But with this method, we have a better chance of finding the answers.” This data was supplemented by Ran's measurements of these underwater currents, as well as the high melting rates of fractures that run through the glacier. These surveys were conducted in 2022, and  the team returned earlier this year to see what changes to the ice shelf had occurred. That's when Wåhlin and her team's worst fears were realized—Ran didn't emerge at the pre-planned rendezvous point. These scientific advances were made possible thanks to the unique submersible that Ran was,” Wåhlin said in a press statement. Geologists Found a Ghost Plume Rising From Below Past CO2 Rise Can't Even Compare to Climate Change Are There More Humans on Earth Than We Thought? All Life on Earth Follows This One Simple Rule This ‘Fingerprint' May Explain the Ring of Fire</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250627095022.htm'>Sex swap in seconds: The fish that takes charge and changes gender</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 14:36:09
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Ōtākou Whakaihu Waka scientists have discovered that it takes mere minutes for a species of sex-changing fish to develop dominant behavior after a change in the pecking order. Lead author Haylee Quertermous, a PhD Candidate in the Department of Anatomy, says although the full sex change process takes weeks, it only takes minutes for a second-ranked fish to take advantage of the power vacuum and assert dominant behaviors. "Sometimes the dominant fish will make physical contact with the subordinates, including taking bites at them, usually around their tail and fins. While she expected to be able to see behavior changes within an hour of removing the dominant fish, she was surprised by just how rapid the change could be. "In many of the tanks, second-ranked fish increased their aggression within just a few minutes after removal of the dominant fish." She cautions the dominant behavior that accompanies a female to male sex change in spotties does not indicate a change from typically 'female' to 'male' behavior, as other sex-changing fish species such as clownfish for example, change from male to more dominant female fish. The researchers observed that spotties form linear dominance hierarchies based on size, with larger individuals dominating smaller ones. Results show dominant, larger fish are more likely to change sex, and when social hierarchies are disrupted, less dominant fish can quickly change their behavior to seize new opportunities. "The research has broader implications for understanding social dynamics in other species, even humans." The findings can be applied to other species of sex-changing fish where social dominance appears to be the most common trigger for sex change, and could prove beneficial for aquaculture and open water fisheries, with many commercial valuable fisheries dependent on fishes that change sex, for example, New Zealand's blue cod. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a65187091/historic-black-school-discovery-virginia/'>Archaeologists Excavated One of America's Oldest Schools—and Found a Secret Cellar</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Few schools can say they're older than the countries in which they reside, but the Williamsburg Bray School holds that distinct honor. Luckily, researchers just discovered a few more pieces of the puzzle: the near-complete foundation of the Bray School and an undocumented cellar filled with centuries worth of artifacts. According to a press release from W&M, the remnants of the cellar are sizable, measuring 36 feet by 18 feet. These recent excavations revealed the bottom of the cellar to be nearly 18 inches higher than previous research found. “The roots of our city and university entwine here. Every layer of history that it reveals gives us new insights into our early republic, from the Williamsburg Bray School through the generations that followed, up through the early 20th century.” The newly found broken pottery shards, slate pencil pieces, buttons, and jewelry painted a picture of everyday life at the Bray School. Now that these once-lost stories have come to light, some of the artifacts will be displayed as a permanent installation in W&M's Gates Hall. Other treasures will be lent to The Colonial Williamsburg Foundation and exhibited at the Colin G. and Nancy N. Campbell Archaeology Center once it opens in 2026. “This is exciting,” Maureen Elgersman Lee, director of the W&M Bray School Lab, said in the press release. “What else are we about to learn? We are not done learning the history of this area, and we are certainly not done learning the history of this country.” Her work appears in several publications including Biography.com and Popular Mechanics. When she's not writing, Emma can be found hopping between coffee shops on the hunt for the world's best oat milk cappuccino. This Skull Belongs to a Lost Human Species Archaeologists Found Human Remains in Ancient Urns Odysseus Shrine Discovered in Ithaca After Centuri Lost Revolutionary War Barracks Found in Virginia Experts Found a Slab Bearing the Lord's Prayer</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41562-025-02205-6'>Following news on social media boosts knowledge, belief accuracy and trust</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 11:18:58
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Many worry that news on social media leaves people uninformed or even misinformed. Here we conducted a preregistered two-wave online field experiment in France and Germany (N = 3,395) to estimate the effect of following the news on Instagram and WhatsApp. Participants were asked to follow two accounts for 2 weeks and activate the notifications. In the treatment condition, the accounts were those of news organizations, while in the control condition they covered cooking, cinema or art. The treatment enhanced current affairs knowledge, participants' ability to discern true from false news stories and awareness of true news stories, as well as trust in the news. The treatment had no significant effects on feelings of being informed, political efficacy, affective polarization and interest in news or politics. These results suggest that, while some forms of social media use are harmful, others are beneficial and can be leveraged to foster a well-informed society. Over 5.07 billion people, or 62.6% of the world's population, use social media platforms for communication and information access. Social media have been accused of degrading the quality of public discourse by promoting low-quality content that is toxic, sensationalist, misleading and sometimes harmful1. Falsehoods spread fast on some platforms2, algorithms recommend entertainment over news and public affairs3, and negativity, outrage and content denigrating political opponents receive disproportionate attention online1,4. These trends may exacerbate polarization, radicalization5, misinformation and disinformation-induced confusion6,7, ultimately undermining democratic governance. Scholars have developed numerous interventions to counter these trends. Most have focused on how to minimize or recognize ‘bad' content on platforms (for example, clickbait, misinformative or radical) or how to improve critical thinking5,8, with a focus on making people more sceptical of bad content9. Other scholars have tested various methods for helping people recognize manipulation techniques10. For example, encouraging people to think about accuracy reduces the sharing of unreliable news headlines8 and fact-checking reduces belief in false claims11. These approaches, although effective, may have unintended consequences on the perception of ‘good' content (that is, factual and verified news)12,13,14. A host of other interventions have been tested at the platform level, such as friction15, downranking like-minded sources16, removing reshared content17 or the introduction of chronological feeds18. Yet these interventions may be ineffective in that friction reduces sharing of reliable content15 and chronological feeds increase exposure to untrustworthy, extreme and ideologically congruent sources18,19,20, and can only be implemented by or in collaboration with social media platforms themselves21. Our approach is markedly different. Rather than focusing on minimizing ‘bad' political content, we enhance users' exposure to ‘good' content, namely factual and verified news and public affairs information that is already freely and widely available on platforms22. We incentivized social media users in France and Germany (total N = 3,395) to follow politically balanced news outlets on Instagram and WhatsApp for 2 weeks. Our preregistered field experiment causally estimated the effects of following the news on social media in naturalistic settings on four crucial outcomes: (1) current affairs knowledge about events that occurred between waves, (2) awareness of circulating true and false claims, (3) belief accuracy (that is, discernment between true and false news stories about national and international political issues, such as the wars in Ukraine and Gaza or the European Parliament election) and (4) trust in news media organizations. In addition, we tested whether news consumption on social media impacts participants' feelings of being informed about politics and current events, interest in the news and politics and affective polarization (that is, participants' feelings towards political parties and countries, such as Palestine–Israel or Ukraine–Russia). We tested whether these effects depend on differential levels of compliance, measured using screenshots submitted by the users, as well as self-reported measures of compliance. Last, we used qualitative comments and post-test responses to assess participants' feedback on their experience and their intentions to continue following the recommended news outlets. We expand past literature in three important ways. First, we build on the work on approaches that ‘promote the Internet's potential to bolster rather than undermine democratic societies'23. Although it is feared that social media facilitate the spread of clickbait, misinformative and low-quality and harmful content, past efforts aimed at reducing exposure to such content often overlook the wealth of verified and factual public affairs information already freely available on platforms24,25,26. Despite its abundance, the vast majority of users do not consume much news or political information on platforms27,28,29,30,31,32. This low consumption of public affairs leads to uninformed citizenry, with various negative consequences for democracy33,34,35,36. By contrast, well-informed citizens are better able to discern between true and false news37,38, more resistant to manipulation39 and more likely to hold stable attitudes40. Recognizing this, recent work focuses on positive incentivization and encouraging social media users to consume news25,26 to make citizens more resilient to informational threats39. For instance, Askari et al.26 created bots that contextually replied to users tweeting about non-political topics and encouraged the users to follow verified and balanced news accounts. This intervention slightly enhanced the following and liking of news accounts. In a YouTube field experiment, Yu et al.25 found that nudging the algorithm by playing videos from news channels in the background increases recommendations to and consumption of news over time and promotes more diverse news diets. Such positive incentivizations align with users' desire for accurate information and educational content, instead of divisive, hateful or false content41. Second, we offer a comprehensive causal test of the effects of following news on social media on current affairs knowledge, awareness of both true and false news stories, belief accuracy and trust in news media, outcomes crucial to a well-functioning democracy33,42. Researchers worry that news on social media has negative effects by increasing information overload and political polarization or by increasing the feeling of knowing, without actually learning anything43,44. A recent meta-analysis found no association between social media use and political learning in observational studies and substantively small increases in experimental work45. Most experiments, however, rely on forced exposure to news, which users may avoid in their daily lives, and test outcomes immediately after exposure. It is therefore not clear whether similar effects would emerge in naturalistic settings where users have full control over exposure, myriad content options and where the majority seeks entertainment31,46,47,48. The few field experiments that incentivized internet users to increase exposure to news in general25,49 or to partisan news50,51 found null or very limited effects on various outcomes, such as knowledge or polarization. We build on these field experiments by also maximizing external validity, not forcing exposure, and measuring effects not immediately after (potential) exposure. Furthermore, we assess compliance in several ways (that is, requesting participants to submit screenshots showing that they followed the news accounts and enabled notifications from these accounts and asking them to list and later recognize these accounts), which allows us to systematically test the differential effects of different doses of on-platform exposure to and engagement with news. Third, we examine these effects across two countries and two distinct, popular and understudied platforms that have been accused of facilitating the spread of harmful content52. In general, social media platforms have transformed the relationship between news media and audiences, such that news organizations create and disseminate news in line with platform logics and constrained by platform affordances, such as interactivity or algorithmic curation53,54. For instance, the use of platforms such as Twitter has been shown to increase political knowledge55,56, probably because many used the platform for news and politics4. Yet the use of Instagram, WhatsApp or Facebook has shown to be mostly detrimental37,57. Possibly this is because news outlets publish ‘softer' articles on these platforms58 or because these platforms are not primarily used for news22. Despite the different features and distinct uses of various platforms, cross-platform evidence on the effect of news use is limited53. We focus on Instagram and WhatsApp, platforms with different affordances and distinct usage patterns. Instagram is a well-established image and video-based social media platform with over two billion monthly active users worldwide as of early 2024. WhatsApp is a messaging platform with numerous social media features (such as stories) and almost three billion unique active users worldwide59. Each offers distinct opportunities for incidental news exposure: Instagram provides the most opportunities as news content is displayed on the same page as friends' posts and stories, while WhatsApp offers fewer, as users must actively open the news tab to access news. As of September 2023, WhatsApp users can follow news channels for updates in a one-way broadcast. Crucially for our purposes, Instagram and WhatsApp are not typically used for news22 despite news media organizations' presence on the platforms, making it worthwhile to systematically examine the effects of encouraging users to follow news accounts on these platforms. In particular, 65% of the German population uses WhatsApp and 31% uses Instagram, but less than one-third of the users report getting news on these platforms22. In France, 44% of the population uses WhatsApp and 31% uses Instagram, and between a third and a half of the users report getting news on these platforms22. Focusing on distinct platforms and countries minimizes the chances that the detected effects are attributable to any one context, an important advance over past work, which have mostly studied a single platform, typically Twitter or Facebook, in the USA. Overall, we find that following news accounts on social media increases not only users' current affairs knowledge and their awareness of current events, but also enhances their belief accuracy (that is, discernment between true and false news) and trust in news media—without increasing feeling of being informed, affective polarization or interest in news and politics. These findings show the potential of putting ‘good' information in users' online ecosystems, which disrupts informational silos on social media and provides a robust method to bolster democratic resilience. Our results also offer a positive perspective by challenging the widespread negativity about the impact of social media. Contrary to the idea that news on social media increases the feeling of being informed without informing43,44, our study demonstrates that news on social media can in fact help people become better informed and contribute positively to the democratic process. Additional details on the data and methodology are provided in Methods and Supplementary Information offers a detailed description of all materials and methods used in this study, as well as additional robustness checks. We conducted a preregistered online field experiment in France and Germany (N = 3,395) to causally estimate the effect of following news on Instagram and WhatsApp for 2 weeks in ecological conditions. To take part in the study, participants had to use WhatsApp or Instagram and not already be following the news accounts included in the experiment. WhatsApp users were allocated to the WhatsApp branch while Instagram users were allocated to the Instagram branch. All users were later randomly assigned to the treatment or the control within their branch. Participants in the treatment were asked to follow two news accounts while participants in the control followed non-news accounts. Figure 1 details the flow of the experiment, the number of participants in each branch and the social media accounts participants were asked to follow. The news media accounts were selected because they are generally trusted22, reliable and not partisan (for more information see ‘Treatment (wave 1)' section). The non-news accounts were selected because they do not cover news or politics and could be of interest to most people. WhatsApp and Instagram users were randomly assigned to the treatment or the control condition. Participants answered pre-treatment measures and were then asked to follow accounts on the respective platforms. At this stage, in wave 1 we measured compliance with the screenshots uploaded by the users. Two weeks later participants were surveyed again. In wave 2, we measured compliance with self-reported questions and participants answered post-treatment questions. Below, we focus on the participants who completed both waves and uploaded at least a screenshot in wave 1 (regardless of whether the screenshot is valid). In Supplementary Information we report all exclusions. In total, 1,700 German participants (985 women, mean age of 40.5 years), and 1,695 French participants (1,042 women, mean age of 44.1 years) completed both waves. Age, gender and education were evenly distributed across control and treatment, and we observed no systematic bias in dropout across the conditions (Supplementary Information). Our dependent variables were repeated across waves, and in wave 2 we added new political knowledge items and questions about new true and false news stories that appeared between the waves so that participants could not have possibly known them before the treatment. Supplementary Information details the wording of the questions for all the items used in the analyses and in Methods and Fig. 2 we offer an overview of our dependent variables. The use of dynamic dependent variables allowed us to maximize internal validity and assess over-time changes in current affairs knowledge, awareness and belief accuracy. Our field experiment also maximized external validity as we do not force exposure but instead incentivize it as part of participants' daily platform usage and measure effects up to 14 days after (potential) exposure. Dynamic dependent variables are highlighted in brown boxes, while static dependent variables are shown in blue boxes. We estimated the effects of our treatment using linear mixed-effect models with participants as a random effect, and control for age, gender, education and past vote (as well as country and platform), as preregistered. In particular, we modelled whether within-individual changes across waves are greater in the treatment compared with the control. Figure 3 summarizes our main findings and breaks them down by country and by platform and Fig. 4 breaks them down by compliance levels. The black asterisks are the preregistered combined estimates across platforms and countries. The β represents standardized regression coefficients. Full compliers (N = 1,466) uploaded screenshots showing they followed the accounts and activated the notifications, and reported following the accounts in wave 2. Notifications noncompliers (N = 746) are similar to full compliers except that their screenshots do not show that they activated the notifications. Noncompliers (N = 240) did not upload a screenshot showing that they followed even one account. Partial compliers (N = 942) are the remaining participants; they uploaded screenshots showing they follow only one account and/or only activated the notifications for one account and/or reported having unfollowed the accounts in wave 2. The β represents standardized regression coefficients. First, the treatment led to greater gains in current affairs knowledge across waves, measured as the number of correct answers to the news quiz on national and international news. Current affairs knowledge increased among the treatment group by 0.12 points across waves compared with the control (β = 0.07 (0.01 to 0.12), P = 0.024). The gains are greater among participants who fully complied, with an increase in current affairs knowledge of 0.21 points (β = 0.12 (0.05 to 0.36), P = 0.008), and not significant among noncompliers. Second, the treatments increased gains in awareness of true news stories (but not false news stories), namely the number of news stories participants reported having heard of or read before, while controlling for participants' tendency to report being aware of news stories they have not encountered with a placebo news story. The increase in awareness of true news stories across waves was 0.14 points higher in the treatment group compared with the control (β = 0.08 (0.02 to 0.14), P = 0.006). These gains are greater among participants who fully complied, with an increase in awareness of true news stories of 0.25 points (β = 0.15 (0.06 to 0.24), P = 0.001). The treatments did not significantly increase awareness of false news stories (β = 0.03 (−0.03 to 0.10), P = 0.32), regardless of levels of compliance or country platform-specific effects. Third, we found substantial and statistically significant positive effects of our treatment on gains in belief accuracy across waves, computed as the difference between the perceived accuracy of true and false news stories60. Following two news accounts for 2 weeks increased belief accuracy by 0.70 points compared with the control (β = 0.06 (0.02 to 0.11), P = 0.004). These gains are larger among participants who fully complied, with an increase in belief accuracy of 1.04 points (β = 0.10 (0.03 to 0.16), P = 0.004), and nonsignificant among noncompliers. Overall, gains in belief accuracy are due to a higher acquisition of true beliefs across waves (β = 0.05 (0.02 to 0.09), P = 0.001) rather than to a lower acquisition of false beliefs (β = 0.01 (−0.04 to 0.06), P = 0.80). Even though these results show that following news organizations on social media does increase current affairs knowledge and belief accuracy, participants in the treatments did not significantly feel more informed about current events or politics compared with controls (β = 0.01 (−0.04 to 0.05), P = 0.84)—none of the country platform-specific effects come close to statistical significance. We also clearly show that across countries and platforms, the treatments increased trust in the news as well as trust in the specific news sources participants were asked to follow. Across waves, trust in news increased by 0.13 points on the six-point scale in the treatment compared with the control (β = 0.13 (0.08 to 0.18), P < 0.001). Regarding trust in the specific news outlets, in France, the treatment increased trust in France Info and Hugo Décrypte, while in Germany they increased trust in FAZ and SZ (Supplementary Fig. Finally, the treatments had no statistically significant effect on political efficacy (β = −0.02 (−0.07 to 0.03), P = 0.46), interest in the news (β = −0.02 (−0.09 to 0.04), P = 0.46), interest in politics (β = 0.01 (−0.04 to 0.07), P = 0.61) or on any of the eight feeling thermometers (Supplementary Information). The full results for these outcomes broken down by country and platform are reported in Supplementary Information. We conducted a number of robustness checks and provide additional information in Supplementary Information. In sections 1 and 2 we report detailed results for the static dependent variables. In section 3 we break down the results by waves. In section 4 we report various measures of compliance. In section 5 we report qualitative comments from participants. In section 6 we offer the descriptives statistics of all dependent variables. In section 7 and on Open Science Framework (OSF), we report heterogeneous treatment effects. In section 8 we show no differential attrition and correct randomization. In section 9 we report all exclusions and screens. In section 10 and on OSF we report all the questions in the survey, while in section 11, we show that the results hold while controlling for news use. Social media platforms have been accused of facilitating the spread of toxic, sensationalist, manipulative and otherwise harmful low-quality content, which may foment polarization, increase misinformation endorsement and lower citizen support for democratic processes and institutions. Most interventions have tried to fight exposure to and spread of this ‘bad' political content or teach individuals to recognize or resist such content. In contrast, our approach enhanced the ‘good' content in users' social media ecosystem. As news media organizations have integrated platforms into their communication strategies, we tested whether incentivizing social media users to follow quality news on Instagram and WhatsApp can generate positive democratic outcomes. Our field experiment on 3,395 WhatsApp or Instagram users in France and Germany offers three key findings. First, prompting social media users to follow news organizations increases current affairs knowledge, belief accuracy and awareness of true news stories, as well as trust in news, journalists and the news organizations followed. These effects are (probably) due to increased exposure to news content: participants in the treatment saw more news and learned from it. The effects on trust may be due to participants underestimating the quality of news coverage before the experiment and updating their perceptions after being exposed to this coverage, which in turn spilled over to perceptions of news and journalists more broadly. We note that these findings are more in line with past deactivation studies, which show that deactivating Facebook reduces knowledge61,62,63, than with other, mostly cross-sectional, work, showing that platforms such as WhatsApp, Instagram or Facebook, have null or negative effects on political knowledge25,45. We emphasize that our findings emerged despite the naturalistic setting of daily social media use, which offers myriad distractions that could dilute the impact of the intervention. Exposure to news was not 'forced' and the treatment only consisted of following two news accounts for 2 weeks, which pales in comparison to the hundreds of other, potentially more interesting, accounts users follow. Even in the treatment group, news probably only represented a tiny portion of all the content participants saw on the platform. Still, some news content did manage to cut through the noise and produce measurable beneficial effects. For these reasons, it is important to keep in mind that, despite being statistically significant, the treatment effects are small. We also found numerous nonsignificant effects, such as awareness of and belief in false news stories, the feeling of being informed, interest in news and politics, political efficacy and feelings towards Ukraine, Russia, Palestine, Israel, Europe and specific political parties. These null effects are in line with past work showing that attitudes are very resistant to change16,18,25. Second, the fact that these effects were detected in ecological settings and over time is particularly optimistic. Unlike fact-checking individual news stories or punctual accuracy prompts, our approach may build longer-term capacities and skills among users, helping them situate potentially problematic information in larger associative networks of factual knowledge gained from reliable media. Moreover, the gains in news trust of reliable outlets may translate into further gains in true beliefs, conditional on exposure to such content. These two core findings together suggest that improving exposure to verified information from news media could be more effective than reducing exposure to low-quality information from junk news. The increased belief in true news and trust in news and journalists is important given the evidence that the rejection of true news may be a bigger problem than the acceptance of false news. For instance, people are sceptical of true news64, distrust news media in general, including reliable news outlets22 and trust mainstream news outlets much less than fact checkers do65. This suggests that improving exposure to reliable information may be more beneficial26,66 than reducing exposure to misinformation, which—after all—represents a small portion of the news people see online67. Our third key finding regards the qualitative comments from the participants. These comments are overwhelmingly positive (Supplementary Information) and suggest that some users were largely unaware that it is easy and free to follow news organizations on Instagram and WhatsApp. This lack of awareness emerged especially in the comments from the WhatsApp sample, perhaps because the platform is mainly used for one-to-one communication and WhatsApp channels are a relatively new feature. Crucially, 51% of participants indicated that they intend to keep following these news organizations after the termination of the experiment. These qualitative insights suggest that the lack of awareness about the presence of quality news on platforms may partly contribute to the low levels of news consumption on social media (similar to the unawareness of censored information available via VPNs in China68. These comments add to the recent evidence that social media users report wanting to receive more educational and informative content41 and that increasing recommendations and exposure to news on YouTube does not decrease users' engagement with and activity on the platform25. These comments, and our overall findings, suggest that following news generates positive effects, and calls into question the decisions of some platforms—most notably Meta—to minimize news dissemination. At the time of writing, news is downranked on Facebook, in some countries news outlets have been blocked from posting links to their content on Facebook and Instagram, and by default, the Instagram and Threads algorithms do not recommend news, users have to opt in. Meta claimed that people are not interested in news, that news is ‘highly substitutable'69, and that Meta ‘never thought about news as a way to minimize misinformation/disinformation on our services'70. We suggest that there is a demand for free and reliable news on social media platforms not commonly used for news, and that, in line with past work37,38, news can help people discern facts from fiction. People increasingly consume news on social media, and although they worry about inaccurate news, they find it convenient to use social media for news22,71. Although this project offers new optimistic evidence, we encourage future research to examine the generalizability of our findings. For one, it is unclear whether similar effects would emerge on platforms with different features or uses. In our case, we do not detect statistically significant differences between platforms, but we lack statistical power to detect small differences. Yet there are reasons to expect that platforms with different algorithms and affordances would yield distinct results. For instance, if participants had followed news accounts on TikTok, they may never have been exposed to content from these accounts because users mostly use the ‘For you' feed, which weakly prioritizes followed accounts. Similarly on YouTube, where the algorithms direct users away from news3, subscribing to news channels may not increase news exposure in any pronounced ways (but tweaking the algorithm itself does generate these increases25). However, holding the quantity and quality of news constant, exposure to news on TikTok, YouTube or other platforms should also yield the benefits documented here. Studies should replicate our findings in different countries and different times in electoral politics. Our experiment was run in two multiparty systems with strong public-service broadcasting and during relatively contentious times (with EU elections, the Israel-Gaza war and the Russian-Ukraine war). The effects may be different in other periods and contexts. Further, our approach is premised on the presence of reliable news outlets posting on social media, yet, in some countries or on some platforms, such outlets may be scarce. Moreover, our findings are contextual to the design of the platforms and their algorithms at the moment of the experiment72, and so extending our work to different platforms, countries and time periods is needed. Future work should also investigate the theoretical mechanisms behind the treatment effects. Breaking down the results by compliance levels offers some preliminary insights. The absence of statistically significant effects among participants who followed the news accounts but did not turn on the notifications (compared with participants who turned them on; the full compliers) suggests that notifications may have played an important role. Notifications could expose users to news content directly, cutting through the in-platform noise and informing users about current events and/or serve as a mere reminder for the users to access news. News notifications are not unusual. In wave 1, 47.5% of participants reported having received news notifications on their phones in the past week (in line with ref. The beneficial effects detected may be explained through cognitive processing theories74: the concise, visually-driven nature of short-form news on platforms such as Instagram and WhatsApp may enhance information retention by reducing cognitive overload, while the interactive features of these platforms could foster deeper engagement. Unlike traditional, longer formats, the processing of which requires deeper, more reflective processing (per the Elaboration Likelihood Model75, short-form news may rely more on peripheral cues such as visuals or source credibility. This could mean that the cognitive and emotional pathways through which these formats influence knowledge retention, trust and content discernment differ substantially from those activated by traditional media. We encourage scholars to explore these understudied mechanisms. We also encourage studies to assess the potential cumulative or over-time effects of our intervention25,26. If users engage with news on platforms, the algorithms are more likely to classify those users as politically interested, which increases future recommendations to news and public affairs27,28. Currently, there is a likely feedback loop where platforms deprioritize news, lowering users' exposure to news and leading users to seek out news less. Our approach could break these loops and set in motion a cycle where users receive updates from the news accounts they followed and from other accounts recommended by the algorithms26. On the downside, our approach could have negative substitution effects by increasing social media news consumption while reducing the consumption of higher-quality news. Yet such substitution effects are rare18,76, rely on shaky assumptions and, importantly, are incompatible with our treatment effects. Last, we add to the research on potential interventions to minimize information harms from social media platforms, yet we acknowledge that future research needs to examine whether and how our approach could be scalable. As an intervention, our approach could be implemented at scale either by platforms themselves, which could start prioritizing and recommending, not downranking, news in users' feeds, or by news organizations, which could increase their presence on social media and actively make the users more aware of this presence. Our findings clearly indicate that this could have democratically beneficial effects and—as the aforementioned research shows—would not minimize users' engagement with the platforms25,41. Alternatively, (targeted) social media ads, educational campaigns and informational outreach done by non-profits could be viable encouragement for some users to follow (more) factual and reliable news media organizations. In summary, following news accounts on social media is a relatively rare behaviour among users, and accounts of celebrities or sports teams have disproportionately greater following on platforms48. Yet, as we show, users are willing to engage with news media when encouraged to do so, and this engagement increases current affairs knowledge, awareness of true news stories, belief accuracy and enhances trust in the news and journalists. This means that, with the right incentives, social media can be a powerful tool for promoting informed and engaged citizenship. This research project complied with all ethical regulations for research involving human subjects and received ethical approval from the University of Zürich PhF Ethics Committee (ethics approval no. Participants provided informed consent at the beginning of each survey and were debriefed at the end of wave 2 (all the materials are available on OSF). The preregistrations, materials, R scripts and the data to replicate the findings are publicly available on OSF. In 2024, between 1 and 11 March, we recruited 2,009 German and 2,021 French participants via the market research company Bilendi. Participants were recontacted between 18 and 28 March. In total, 1,700 German participants (985 women, mean age of 40.5 (12.5) years; median education of a 2-year college degree, 38% had a bachelor degree or more) and 1,695 French participants (1,042 women, mean age of 44.1 (11.5) years, median education of finished high school, 31% had a bachelor degree or more) completed the second wave. The median distance between waves was 14 days (mean of 13.4 days, s.d. In Supplementary Information we report all exclusions and show that randomization was successful and that there was no problem of differential attrition. Participants first consented to take part in the study and answered screening questions about (1) whether they have Instagram and WhatsApp accounts, (2) how frequently they use them and (3) whether they follow a list of social media accounts. To be eligible, participants had to have either an Instagram account or WhatsApp, use Instagram or WhatsApp, and not already be following the news accounts included in the experiment. After passing the screens, Instagram users were randomly assigned to the Instagram control or the Instagram treatment, WhatsApp users were randomly assigned to the WhatsApp control or the WhatsApp treatment, and those who used both Instagram and WhatsApp were randomly assigned to one of the four conditions (Fig. After answering the screening questions, participants were told that they would have to follow two (at that moment, not yet specified) accounts on social media for 2 weeks and upload screenshots to show that they complied. At this stage, participants were asked again if they consent to take part in the study. Those who consented answered demographic questions (gender, age, education and vote at the last presidential election) and reported their news consumption frequency in general, on WhatsApp and on Instagram (from ‘Never' to ‘More than ten times a day'). Participants also reported their main source of news (for example, TV, social media and so on) and whether they received news notifications on their phones last week. TV was the main source of news for 37% of participants, followed by social media (22%), online (19%) and radio (17%). In wave 1, participants reported very low news consumption on WhatsApp and Instagram. In the WhatsApp groups, 62% reported never consuming news on WhatsApp and 15% reported consuming it less than once a day (mean of 1.87 on a six-point scale). In the Instagram groups, 29% of participants reported never consuming news on Instagram and 23% reported consuming it less than once a day (mean of 2.61). The questions on news consumption were taken or adapted from the Digital News report22. In both waves, participants reported how informed they feel about the news and politics77, how interested they are in the news and politics (from ‘Not at all' (1) to ‘Extremely' (6)22), the extent to which they trust the news and journalists (‘To what extent do you trust the media and journalists?' from ‘Not at all' (1) to ‘Completely' (7)), as well as the extent to which they trust the specific news sources included in the treatments (from ‘Not at all' (1) to ‘Completely' (7)22). We measured political efficacy with the following questions ‘How well would you say you understand the important political issues facing our country?' (from ‘Not at all' (1) to ‘A lot' (5)) and ‘Do you often think that politics and government are so complicated that you can't understand what's going on?' (reverse coded, from ‘Never' (1) to ‘Yes, most of the time' (5)). These questions are commonly used to measure internal political efficacy78. In wave 1, these outcomes are considered dynamic because they changed between the waves, such that new items were added in wave 2 (Fig. All current affairs knowledge and news stories questions are reported in Supplementary Information. We measured current affairs knowledge with three multiple-choice questions (with seven options, including ‘Don't know') on the date of the next European election, the name of a French/German minister and the name of the southernmost town in the Gaza Strip. Such questions, in contrast with ‘textbook knowledge' about how politics and governments function, are often used in surveys measuring political knowledge acquisition across numerous waves37,55. Our current affairs knowledge questions do not measure how well participants understand current political events, but rather their knowledge related to current affairs. Current affairs knowledge was computed as the sum of correct responses. We used feeling thermometers (from 0 to 100) to measure how people felt towards Palestine, Israel, Ukraine, Russia and the European Union, as well as left-wing and right-wing political parties and the party of the current government. We measured belief in, and awareness of, true and false news stories by asking participants to rate two true news stories selected from mainstream news outlets and two false news stories that have been fact checked as false by independent fact checkers. For each claim, participants were asked ‘Before this survey, had you ever read or heard this statement?' (‘Yes', ‘No' or ‘Don't know/Don't remember') and ‘Do you think this statement is rather true or false?' Awareness in claims was computed as the sum of ‘Yes' responses. All participants were asked to follow two accounts, activate the notifications and upload screenshots showing that they complied. Participants in the WhatsApp conditions followed accounts on WhatsApp while participants in the Instagram conditions followed accounts on Instagram. Participants in the treatment conditions followed news accounts (France Info and Hugo Décrypte in France or FAZ and SZ in Germany), while participants in the control conditions followed non-news accounts about art (Musée Rodin or farbenfroh art), music (les Inrockuptibles or Sie Klassik radio) movies (Allociné or Serienjunkies) or food (Marmiton or Einfach Tasty). While Hugo Décrypte is a news influencer (the most popular one by far in France22), his coverage is very similar to mainstream news outlets such as the ones included in the experiment. All the news accounts are popular on Instagram and enjoy a wide reach (followers count: Hugo Décrypte 3.9m, France Info 809k, SZ 811k and FAZ 631k), while they are markedly less influential on WhatsApp (Hugo Décrypte 406k, France Info 230k, SZ 36k and FAZ 16k). Regarding the formats, on Instagram the accounts use mostly videos and images, while Hugo Décrypte relies almost exclusively on images with text. On WhatsApp, France Info has an audio format with some text, Hugo Décrypte a text format with no links, while FAZ and SZ use a text format with links to (their) news stories. Participants were first asked basic questions about their WhatsApp and Instagram use, and answered questions about their news consumption, their main source of news, as well as the static dependent variables (that is, trust in news, feeling thermometers, political efficacy, news and political interest and feeling of being informed). Participants then answered the dynamic dependent variables. They answered the same three current affairs knowledge questions as in wave 1, plus four new current affairs knowledge questions about events that took place between wave 1 and wave 2. They rated the same two true news stories as in wave 1, plus five new news stories about events that took place between wave 1 and wave 2. They rated the same two false news stories as in wave 1, plus two additional false news stories about events that took place before wave 1 (we had fewer additional false than true questions because we could not find new false news stories circulating in France and Germany between waves). Participants also rated a ‘placebo' news story that we made up (‘Brazil has introduced a new tropical forest monitoring system that detects and prevents illegal deforestation in real time.'). We used responses to this placebo news story to control for participants' tendency to report being aware of news stories they have not encountered (as they could not have possibly encountered the placebo news stories before): 14.2% of participants reported having heard or read the placebo news stories before, while the median number of news stories participants reported being aware of was 4 (out of 11). The dynamic dependent variables allow us to measure the acquisition of beliefs and knowledge, as well as gains in awareness. Such design is typically used in survey research37,55 and reflects an interest in the change in the dependent variables (for example, beliefs and knowledge) rather than changes in the independent variables (for example, news consumption). Yet here, we combine this design with a field experiment in which we experimentally manipulate what is typically considered the independent variable (that is, news consumption), allowing us to draw stronger causal inferences. At the end of the survey, participants were told that they could unfollow the accounts and were asked how likely they were to unfollow them. Fifty-one per cent of participants in the treatments reported being ‘very unlikely' or ‘unlikely' to unfollow the accounts. Qualitative analyses of participants' comments to the open-ended box at the end of the survey also suggest that many participants enjoyed following the news account and will continue to follow them in the future (Supplementary Information). Independent coders rated all the screenshots uploaded by the participants in wave 1. We found that compliance for the account following was high: 87.9% of the participants uploaded screenshots showing they follow both accounts and 92.8% of participants uploaded at least a screenshot showing that they follow one account. Compliance for the notifications was lower: 48.4% of participants uploaded screenshots showing that they activated the notifications for both accounts and 65.1% of participants uploaded at least a screenshot showing that they activated the notifications for one account. At the beginning of wave 2, participants had to write down the names of the accounts they were asked to follow in wave 1 without looking them up: 71% of participants in the treatments correctly mentioned both news accounts. Then, participants were presented with a list of seven social media accounts and had to click on the ones that they follow on Instagram and WhatsApp. While in wave 1 we used these questions as a screener, in wave 2 we used them as a compliance check. Eighty-nine per cent of participants in the treatments reported following the news accounts (compared with 0% in wave 1). Participants were explicitly asked whether they stopped following the accounts (knowing that their response would be anonymous and would not affect their payment), 13% of participants in the treatments reported having done so. In Supplementary Information we report additional measures of compliance showing that overall compliance was high. We ran linear mixed-effect models with participants as a random effect. We preregistered adding wave as a random slope (Wave∣ID) but the models failed to converge. We analysed the responses at the level of the wave participant, that is, two data points per participant, one per wave. We included an interaction term between ‘Condition' and ‘Wave' to estimate treatment effects. This approach allows us to determine whether, for example, within-individual gains in current affairs knowledge between waves are larger in the treatments than in the controls. In all analyses, we controlled for age, gender, education and past vote. All statistical tests are two tailed. Following the preregistration, the main analyses are conducted across countries and platforms, but in Fig. 3 and Supplementary Information we report all treatment effects per country platforms. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The preregistrations, materials, R scripts and the data to replicate the findings are publicly available via OSF at https://osf.io/8tzd2/ (ref. Rathje, S., Van Bavel, J. J. & Van Der Linden, S. Out-group animosity drives engagement on social media. Vosoughi, S., Roy, D. & Aral, S. The spread of true and false news online. Huang, S. & Yang, T. Auditing entertainment traps on YouTube: How do recommendation algorithms pull users away from news. Yu, X., Wojcieszak, M. & Casas, A. Partisanship on social media: In-party love among american politicians, greater engagement with out-party hate among ordinary users. Clayton, K. et al. Elite rhetoric can undermine democratic norms. & Koo, G. H. Comparing the #stopthesteal movement across multiple platforms: differentiating discourse on Facebook, Twitter, and Parler. Lukito, J. et al. “No reason [. ][i]t/should/happen here”: analyzing Flynn's retroactive doublespeak during a QAnon event. Shifting attention to accuracy can reduce misinformation online. Altay, S. How effective are interventions against misinformation? Preprint at PsyArXiv https://doi.org/10.31234/osf.io/sm3vk_v2 (2022). Lewandowsky, S. & Van Der Linden, S. Countering misinformation and fake news through inoculation and prebunking. Factual corrections: concerns and current evidence. Hoes, E., Aitken, B., Zhang, J., Gackowski, T. & Wojcieszak, M. Prominent misinformation interventions reduce misperceptions but increase scepticism. & Valenzuela, S. Studying the downstream effects of fact-checking on social media: experiments on correction formats, belief accuracy, and media trust. & Higham, P. A. Gamified inoculation interventions do not improve discrimination between true and fake news: reanalyzing existing research with receiver operating characteristic analysis. Ershov, D. & Morales, J. S. Sharing news left and right: frictions and misinformation on Twitter. Nyhan, B. et al. Like-minded sources on Facebook are prevalent but not polarizing. Guess, A. M. et al. Reshares on social media amplify political news but do not detectably affect beliefs or opinions. Guess, A. M. et al. How do social media feed algorithms affect attitudes and behavior in an election campaign? Wang, S., Huang, S., Zhou, A. & Metaxa, D. Lower quantity, higher quality: auditing news content and user perceptions on Twitter/X algorithmic versus chronological timelines. & Diakopoulos, N. More accounts, fewer links: how algorithmic curation impacts media exposure in Twitter timelines. Wagner, M. W. Independence by permission. Newman, N., Fletcher, R., Eddy, K., Robertson, C. T. & Nielsen, R. K. Digital News Report 2024 (RISJ: Reuters Institute for the Study of Journalism, 2024). Lorenz-Spreen, P., Lewandowsky, S., Sunstein, C. R. & Hertwig, R. How behavioural sciences can promote truth, autonomy and democratic discourse online. Allen, J., Howland, B., Mobius, M., Rothschild, D. & Watts, D. Evaluating the fake news problem at the scale of the information ecosystem. Yu, X., Haroon, M., Menchen-Trevino, E. & Wojcieszak, M. Nudging the recommendation algorithm increases news consumption and diversity on youtube. Askari, H., Chhabra, A., von Hohenberg, B. C., Heseltine, M. & Wojcieszak, M. Incentivizing news consumption on social media platforms using large language models and realistic bot accounts. Wells, C. & Thorson, K. Combining big data and survey techniques to model effects of political content flows in facebook. Thorson, K., Cotter, K., Medeiros, M. & Pak, C. Algorithmic inference, political interest, and exposure to news and politics on Facebook. Widely viewed content report: what people see on Facebook. & Menchen-Trevino, E. Political content and news are polarized but other content is not in YouTube watch histories. Wojcieszak, M. et al. Non-news websites expose people to more political content than news websites: evidence from browsing data in three countries. McClain, C., Widjaya, R., Rivero, G. & Smith, A. The Behaviors and Attitudes of US Adults on Twitter (Pew Research Center, 2021). & Keeter, S. What Americans Know about Politics and Why it Matters (Yale Univ. In search of the informed citizen: what Americans know about politics and why it matters. Prior, M. News vs. entertainment: how increasing media choice widens gaps in political knowledge and turnout. Bartels, L. M. Uninformed votes: information effects in presidential elections. Altay, S., Nielsen, R. K. & Fletcher, R. News can help! The impact of news media and digital platforms on awareness of and belief in misinformation. The electoral misinformation nexus: how news consumption, platform use, and trust in news influence belief in electoral misinformation. Humprecht, E., Esser, F. & Van Aelst, P. Resilience to online disinformation: a framework for cross-national comparative research. & Keeter, S. in The Civic Web: Online Politics and Democratic Values 129–153 (Routledge, 2003). Rathje, S., Robertson, C., Brady, W. J. People think that social media platforms do (but should not) amplify divisive content. Democracy in America (Perennial Classics, 2000). Lee, S. & Valenzuela, S. A self-righteous, not a virtuous, circle: proposing a new framework for studying media effects on knowledge and political participation in a social media environment. Van Erkel, P. F. & Van Aelst, P. Why don't we learn from social media? Studying effects of and mechanisms behind social media news use on general surveillance political knowledge. Do people learn about politics on social media? A meta-analysis of 76 studies. Arceneaux, K., Johnson, M. & Cryderman, J. Communication, persuasion, and the conditioning value of selective exposure: like minds may unite and divide but they mostly tune out. Stroud, N., Feldman, L., Wojcieszak, M. & Bimber, B. The consequences of forced versus selected political media exposure. Wojcieszak, M., Casas, A., Yu, X., Nagler, J. Most users do not follow political elites on Twitter; those who do show overwhelming preferences for ideological congruity. Wojcieszak, M. et al. No polarization from partisan news: over-time evidence from trace data. Guess, A., Barberá, P., Munzert, S. & Yang, J. The consequences of online partisan media. Levy, R. Social media, news consumption, and polarization: evidence from a field experiment. Political memes and fake news discourses on Instagram. Hase, V., Boczek, K. & Scharkow, M. Adapting to affordances and audiences? A cross-platform, multi-modal analysis of the platformization of news on Facebook, Instagram, TikTok, and Twitter. Dvir-Gvirsman, S. & Tsuriel, K. In an open relationship: platformization of relations between news practitioners and their audiences. Boukes, M. Social network sites and acquiring current affairs knowledge: the impact of Twitter and Facebook usage on learning about the news. Meppelink, C. S., Bos, L., Boukes, M. & Möller, J. A health crisis in the age of misinformation: how social media and mass media influenced misperceptions about COVID-19 and compliance behavior. Does the platform matter? Social media and COVID-19 conspiracy theory beliefs in 17 countries. Lamot, K. What the metrics say. the softening of news on the fFacebook pages of mainstream media outlets. Nobre, G. P., Ferreira, C. H. & Almeida, J. M. A hierarchical network-oriented analysis of user participation in misinformation spread on WhatsApp. Guay, B., Berinsky, A. J., Pennycook, G. & Rand, D. How to think about whether misinformation interventions work. Arceneaux, K., Foucault, M., Giannelos, K., Ladd, J. & Zengin, C. Facebook increases political knowledge, reduces well-being and informational treatments do little to help. Allcott, H., Braghieri, L., Eichmeyer, S. & Gentzkow, M. The welfare effects of social media. Asimovic, N., Nagler, J., Bonneau, R. & Tucker, J. Testing the effects of Facebook usage in an ethnically polarized setting. Pennycook, G. & Rand, D. G. Fighting misinformation on social media using crowdsourced judgments of news source quality. Altay, S., De Angelis, A. & Hoes, E. Media literacy tips promoting reliable news improve discernment and enhance trust in traditional media. Acerbi, A., Altay, S. & Mercier, H. Research Note: Fighting Misinformation or Fighting for Information? (Harvard Kennedy School (HKS) Misinformation Review, 2022). The impact of media censorship: 1984 or brave new world? Taylor, J. Meta says facebook cannot solve media industry's ‘issues' as it defends ending payments for news in australia. The Guardian (14 March 2024); https://www.theguardian.com/media/2024/mar/14/meta-facebook-news-media-bargaining-code Taylor, J. Meta claims news is not an antidote to misinformation on its platforms. The Guardian (9 July 2024); https://www.theguardian.com/media/article/2024/jul/09/meta-facebook-australia-news-ban-misinformation Wang, L. & Forman-Katz, N. Many Americans find Value in Getting News on Social Media, but Concerns about Inaccuracy Have Risen (Pew Research Center, 2024). Munger, K. The limited value of non-replicable field experiments in contexts with low temporal validity. Lu, K. & Matsa, K. E. More Than Half of Smartphone Users Get News Alerts, but Few Get Them Often (Pew Research Center, 2016). Cacioppo, J. T., Petty, R. E., Kao, C. F. & Rodriguez, R. Central and peripheral routes to persuasion: an individual difference perspective. Petty, R. E. & Briñol, P. in Handbook of Theories of Social Psychology, Vol. 1 (eds Van Lange, P. A. M. et al.) 224–245 (Sage, 2011). When corrections fail: the persistence of political misperceptions. Müller, P., Schneiders, P. & Schäfer, S. Appetizer or main dish? Explaining the use of Facebook news posts as a substitute for other news sources. & Hoffmann, C. P. Powerless in the digital age? A systematic review and meta-analysis of political efficacy and digital media use. Altay, S., Hoes, E. & Wojcieszak, M. News on social media boosts knowledge, belief accuracy, and trust. The author(s) gratefully acknowledge the support of the European Research Council (ERC), ‘Europeans exposed to dissimilar views in the media: investigating backfire effects,' Proposal EXPO-756301 (ERC Starting Grant, M.W., PI) and of the Center for Excellence in Social Sciences at the University of Warsaw, which funded the project. The postdoctoral positions of S.A. and E.H. are funded by the ERC under the European Union's Horizon 2020 research and innovation programme (grant agreement no. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. The authors are grateful to D. Della Casa and F. Melliger for their research assistance, especially for coding the screenshots. Any opinions, findings, conclusions, or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the ERC. Open access funding provided by University of Zurich. Sacha Altay & Emma Hoes Department of Communication, University of California, Davis, Davis, CA, USA Center for Excellence in Social Science, University of Warsaw, Warsaw, Poland Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar A coin flip determined that S.A., and not E.H., is corresponding author in addition to M.W. The main purpose of this move is to make the article open access. Correspondence to Sacha Altay or Magdalena Wojcieszak. is part of the US 2020 Facebook and Instagram Election Study, which is a pro bono and fully independent research collaboration of researchers with Meta, and was the PI on two unrelated, unrestricted research grants (US$100k and US$50k) from Meta in 2019 and 2021 (the second with E.H. as the co-PI). Meta did not contribute in any way to this study. The remaining authors declare no competing interests. Nature Human Behaviour thanks Peter Van Aelst and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Treatment effects per country platform. Section B. Null effects on attitudes and interest. Section C. Breaking down the dynamic dependent variables. Compliance in the French sample. Compliance in the German sample. Additional measures of compliance. Section E. Qualitative analysis of participants' comments. Section G. Heterogeneous treatment effects. Section I. Exclusions and screens. Section K. Results while controlling for news use. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Altay, S., Hoes, E. & Wojcieszak, M. Following news on social media boosts knowledge, belief accuracy and trust. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s44161-025-00677-3'>Mechanosensitive PIEZO2 channels shape coronary artery development</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 10:02:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Coronary arteries develop under constant mechanical stress. However, the role of mechanosensitive ion channels in this process remains poorly understood. Here we show that the ion channel PIEZO2, which responds to mechanical stimuli, is expressed in specific coronary endothelial cell populations during a critical phase of coronary vasculature remodeling. These Piezo2+ coronary endothelial cells show distinct transcriptional profiles and have mechanically activated ionic currents. Strikingly, PIEZO2 loss-of-function mouse embryos and mice with human pathogenic variants of PIEZO2 show abnormal coronary vessel development and cardiac left ventricular hyperplasia. We conclude that an optimal balance of PIEZO2 channel function contributes to proper coronary vessel formation, structural integrity and remodeling, and is likely to support normal cardiac function. Our study highlights the importance of mechanical cues in cardiovascular development and suggests that defects in this mechanosensing pathway may contribute to congenital heart conditions. During its formation in the embryo and throughout adult life, the coronary vasculature is uniquely challenged by mechanical loads from the heartbeat more than 100,000 times a day and from coronary perfusion pressure. The coronary arteries are particularly susceptible to disease. In this context, a better understanding of embryonic coronary artery formation processes holds the potential to recreate and enhance the developmental process for coronary artery regeneration under disease conditions. While the role of chemotaxis in guiding angiogenesis is well described, also for coronary vasculature3,4, there is emerging evidence that mechanical guidance cues, termed durotaxis, have a role in the patterning and morphogenesis of blood vessels5. However, there are only few reports on the presence and function of mechanosensitive ion channels in endothelial cells during vasculogenesis or angiogenesis6. In particular, piezo-type mechanosensitive ion channel component 1 (PIEZO1), which is widely expressed in many cell types7, is known to play an important role in the initiation of embryonic vessel formation and function8,9. PIEZO1-deficient mice show disrupted vascularization and a failure in the maturation of endothelial cells into larger blood vessels9,10. The role of its homolog, piezo-type mechanosensitive ion channel component 2 (PIEZO2) in angiogenesis during embryonic development has so far not been explored. However, expression of Piezo2 in endothelial cells of the brain and lungs has been suggested before8,11. One study reported PIEZO2 to be involved in tumor angiogenesis12, and another study described a dramatic loss of pulmonary microvascular endothelial cells in endothelial cell-specific Piezo2 knockout rats13. Ever since the discovery of mechanosensitive PIEZO channels7, PIEZO2 has been primarily characterized as a sensory ion channel, present in a variety of sensory neuron types, the function of which is to sense mechanical forces important for touch, interoception and pain14,15. Importantly, PIEZO2 pathogenic variants are associated with congenital disorders in humans, including joint, craniofacial, brain and cardiovascular defects16,17. Interestingly, the University of Cambridge PhenoScanner V2 database of human genotype–phenotype associations suggests a PIEZO2 genetic linkage with heart failure, diastolic blood pressure, hypertensive heart disease and thoracic aortic aneurysm18. However, the precise function of mechanosensitive PIEZO2 channels in non-sensory cells, including endothelial cells, remains largely unstudied. In particular, a specific function of mechanosensitive ion channels in coronary artery endothelial cells during embryonic development has not been studied. Here we aimed to investigate the role of PIEZO2 in the developing cardiovascular system. Using a combination of genetic fate mapping, single-cell sequencing, physiological analysis and light-sheet imaging, our study uncovers a unique role of PIEZO2 channels in shaping the coronary vasculature during development. Specifically, we show that dysfunction or loss of PIEZO2 leads to aberrant coronary artery branching, impaired vessel morphology and cardiac hyperplasia. To delineate the trajectory and contribution of Piezo2+ cells in the mouse embryonic heart, we used a genetic fate mapping approach in which Piezo2-Cre (ref. 15) drives tdTomato expression in Piezo2-expressing cells and their progeny (Piezo2-tdTomato mice) (Fig. Whole-mount three-dimensional (3D) confocal imaging revealed specific populations of tdTomato-positive (tdTomato+) cells in the heart at embryonic day 11.5 (E11.5) (Fig. We found tdTomato+ cells in the SV and the nascent coronary plexus (CP), and we identified them as endothelial cells positive for endomucin (EMCN) and vascular endothelial cadherin (VE-cadherin) (Fig. At E13.5, the tdTomato+ cell population had expanded to form a nascent coronary endothelial plexus as evidenced by cells staining positive for the pan-endothelial marker platelet/endothelial cell adhesion molecule 1 (PECAM1) (Fig. Whole-mount confocal microscopy also revealed that tdTomato+ cells were positive for Dachshund homolog 1 (DACH1), a transcription factor specifically expressed in embryonic coronary endothelial cells3,19 (Fig. At E13.5, the coronary plexus finally connects to the ascending aorta and triggers plexus remodeling into a mature vasculature. At E18.5, tdTomato+ cells now clearly form the neonatal coronary vasculature as revealed by light-sheet microscopy imaging of the entire heart (Fig. The coronary vessel character was verified on transverse cryosections, and we identified tdTomato+ cells as coronary endothelial cells by fatty acid binding protein 4 (FABP4) staining20 (Fig. Further characterization showed that tdTomato+ cells contribute to both small arteries in the remodeling zone (DACH1 positive)3 (Fig. 2c) and mature arteries, which are positive for SRY-box transcription factor 17 (SOX17)21 (Fig. EMCN-positive cells in the endocardium were negative for tdTomato signals (Fig. 2b–d), supporting the coronary nature of Piezo2 fate-mapped endothelial cells. We observed other non-endothelial cell types with tdTomato labeling, one in the outflow tract with morphological characteristics of vascular smooth muscle cells. These cells were also positive for alpha-smooth muscle actin (αSMA), confirming their identity as vascular smooth muscle cells. Cells making up all four valves of the heart were also tdTomato+ and were surrounded by endothelial cells, indicating their likely identity as valve interstitial cells (Extended Data Fig. As Piezo2-lineage endothelial cells evidently contribute to coronary vasculature formation, we proceeded to analyze both, Piezo2 expression and the transcriptional profile of Piezo2-expressing cells. a, Outline of the genetic strategy (generated in affinity designer 2) used to generate the Piezo2-driven tdTomato fate mapping model. b, A 3D rendering of a representative E11.5 tdTomato+ heart with all cardiac endothelial cells immunolabeled with a cocktail of anti-VE-cadherin (VeCadh) and anti-EMCN antibodies (magenta). c, A 3D rendering of a representative E13.5 tdTomato+ heart confirming the endothelial fate (PECAM1 positive, magenta) of tdTomato+ (yellow) cells in the nascent CP (scale bars, 200 µm and 50 µm in magnification) (n = 2). The co-localization of the tdTomato+ cells (yellow) with DACH1 (magenta) shows that Piezo2 fate-mapped cells were coronary endothelial cells (scale bars, 200 µm and 25 µm in magnification) (n = 2). a, A 3D rendering of a representative E18.5 heart showing that tdTomato+ (yellow) cells contribute to the development of the coronary vasculature (scale bars, 300 µm (3D) and 20 µm (zoom in) (n = 4)). b–d, E18.5 tdTomato+ hearts were immunolabeled for EMCN (transversal sections; EMCN, cyan) and three different coronary markers: FABP4 (magenta) (b), DACH1 (magenta) (c) and SOX17 (magenta) (d). Immunolabeling revealed that tdTomato+ cells comprise coronary endothelial cells of small (DACH1 positive) and mature (SOX17 positive) arteries, as well as endothelial cells in the coronary FABP4-positive population. tdTomato signals did not co-localize with EMCN in the endocardium (cyan), confirming the coronary endothelial cell nature of the tdTomato+ cells (scale bars, 25 µm) (n = 2). While fate mapping provided insights into the trajectories of Piezo2-expressing cells, examining the transcriptional profile of Piezo2-expressing cells could help identify the developmental stages in which this mechanosensitive ion channel contributes to coronary development. By using single-molecule fluorescence in situ hybridization, we could show continued Piezo2 mRNA expression in coronary endothelial cells at E13.5 along with Dach1 expression (Extended Data Fig. In addition, single-cell RNA (scRNA) datasets from fluorescence-activated cell sorting-sorted cardiac endothelial fractions (CD31+ alias PECAM1+/CD45− alias PTPRC−) from four different stages: E12.5, E15.5, postnatal day (P) 2 and adult (8 weeks old) were used to monitor ongoing Piezo2 expression22. Unsupervised clustering was performed using the uniform manifold approximation and projection method, and cell type clusters were classified based on the expression profiles of the top differentially expressed genes (DEGs)22 (Fig. At E12.5, Piezo2 expression was mainly found in the coronary endothelial cells (Fig. Subsequently, at E15.5, Piezo2 expression was observed in two subclusters of coronary endothelial cells designated capillary-to-vein and capillary-to-artery, which are capillary cells22 (Fig. At P2, Piezo2 expression was still present in the same capillary cell types (cap-to-vein and cap-to-artery) and in the proliferating cell cluster (Fig. However, in adult hearts, no Piezo2 expression was found in any of the endothelial cell types analyzed (Fig. To quantify the proportion of Piezo2+ and Fabp4+ cells, we extracted data from our scRNA sequencing (scRNA-seq) analysis across different developmental stages (E12.5, E15.5, P2 and adult). The strong correlation between Piezo2+ and Fabp4+ endothelial cells across developmental stages (Fig. 3b) suggests that PIEZO2 is specifically enriched in a subset of coronary endothelial cells involved in vascular remodeling. While Piezo2+/Fabp4+ cells persist through embryonic and early postnatal stages, Piezo2 expression is lost in adult endothelial cells (Fig. These data show a dynamic Piezo2 expression pattern in distinctive coronary endothelial cell populations during embryonic heart development. Given that Fabp4+ cells contribute to capillary-to-artery transitions, the co-expression of Piezo2 and Fabp4 indicates that PIEZO2-mediated mechanosensing may play a role in endothelial differentiation and artery formation. a, Uniform manifold approximation and projection of cECs from E12.5, E15.5, P2 and murine adult hearts (8 weeks old) showing the following clusters: endocardium (Endoc), capillary-to-vein (Cap-to-vein), capillary-to-artery (Cap-to-artery), proliferating cells (Prolif), valvular endocardium (Valves), arterial and outflow tract cells (Artery and OFT) and venous cells (Vein). Color scale bars indicate log-normalized expression. d, Volcano plots showing DEGs between individual Piezo2-expressing and Piezo2 non-expressing endothelial cells. DEGs for each cluster were identified using the FindAllMarkers function in Seurat, applying the Wilcoxon rank sum test (two sided). Genes were considered for analysis if expressed in at least 25% of cells in either cluster and showed a log2(fold change (FC)) of at least 0.25. Genes with an adjusted P value less than 0.05 were classified as DEGs. We next asked whether Piezo2-expressing coronary endothelial cells show a distinct transcriptional profile when compared with coronary endothelial cells lacking Piezo2 expression (Fig. At E12.5, Piezo2+ coronary endothelial cells showed higher expression of genes such as Neuropilin 2 (Nrp2), a receptor for semaphorin family members, and Apelin (Apln), a sprouting marker, both having been implicated in blood vessel formation23,24,25. We also noticed enrichment of collapsin response mediator protein 1 (Crmp1), a protein involved in neuronal migration and development26. Piezo2 was still enriched at E15.5 along with other well-known markers of coronary endothelial cells such as Fabp4 (ref. 27) and the capillary markers Rgcc (regulator of cell cycle) and Gpihbp1 (glycosylphosphatidylinositol anchored high density lipoprotein binding protein 1)28. At P2, Piezo2+ cells still showed enriched expression of Fabp4 and interestingly were positive for fascin actin-bundling protein 1 (Fscn1), an actin-bundling protein that can induce membrane protrusions and is involved in cell migration and motility29 (Fig. The PIEZO2 homolog PIEZO1 is a stretch-activated ion channel that is widely expressed in the cardiovascular system9,30. Consistently, our analysis revealed that numerous cell types show Piezo1 expression across all stages investigated (Extended Data Fig. By contrast, Piezo2 expression appeared to be maintained primarily in coronary endothelial cells specifically during the angiogenic phase of heart vascularization. The distinct transcriptional signature of Piezo2-expressing coronary endothelial cells, including Nrp2, Apln, Crmp1 and Fscn1, indicates potential roles in cell migration and motility. These findings provide insights into the potential molecular mechanisms underlying the development and maturation of coronary endothelial cells, highlighting the importance of PIEZO2 and associated markers in cardiovascular biology. The electrophysiological function of PIEZO2 in endothelial cells has not been studied. PIEZO2 ion channels are very efficiently activated by substrate deflection but, unlike PIEZO1 channels, are poorly activated by membrane stretch31,32,33. We identified the yolk sac (YS) from E13.5 to E18.5 embryos as being rich in tdTomato+ endothelial cells, which may express functional PIEZO2 channels (Extended Data Fig. We asked whether these cells possess functional mechanosensitive currents characteristic of PIEZO2 channels. We made whole-cell patch–clamp recordings from tdTomato+ and tdTomato− YS endothelial cells and cardiac endothelial cells (cECs) cultured on pillar arrays to study currents gated by substrate deflection31,32 (Fig. We also tried cell indentation, a method often used to activate mechanically activated channels31,32,34,35, but the endothelial cells were too small and flat to make this method feasible. Nanometer-scale pillar displacements move the substrate–cell contact and are limited to the pillar area (10.3 µm2). This mechanical stimulus efficiently evoked inward currents with short latencies (<5 ms) in almost all cells studied owing to the opening of mechanosensitive ion channels sub-adjacent to the stimulated pillus (tdTomato+, 12 of 13 cells; tdTomato−, 11 of 12 cells). Mechanosensitive currents could be classified into three groups with different inactivation time constants similar to what has been shown in sensory neurons32: rapidly adapting (RA) currents (<5 ms), intermediately adapting (IA) currents (5–50 ms) and slowly adapting (SA) currents (>51 ms) (Fig. Notably, the RA currents possessed ultrafast kinetics in that many of the currents inactivated within a millisecond (Fig. Very fast inactivation is a characteristic property of PIEZO2 channels, in contrast to PIEZO1 channels that inactivate much more slowly7,31,32. Consistently, RA currents with very fast inactivation kinetics were found to be significantly more frequent in tdTomato+ cells (chi-squared P < 0.05) (Fig. The frequency with which mechanosensitive currents could be evoked was higher in tdTomato+ than in tdTomato− cells, indicating enhanced mechanosensitivity in Piezo2+ lineage cells. However, no statistically significant differences were observed when comparing deflection–current amplitude relationships (Fig. In summary, almost all YS endothelial cells possess very sensitive mechanically activated currents, but cells from the Piezo2+ lineage possessed more currents with kinetics characteristic of recombinantly expressed PIEZO2 channels7,32. To directly test what proportion of these mechanically activated currents were dependent on PIEZO2 channels, we used small interfering RNA (siRNA) knockdown targeting Piezo2 in these cells followed by electrophysiological analysis. We could still measure mechanically activated currents in cells where Piezo2 was knocked down (Fig. 4d); however, the incidence of such currents was halved compared with controls (Fig. In control cells, we found mechanically gated currents in all cells measured, but after Piezo2 knockdown, around a quarter of the cells showed no mechanically gated currents (Extended Data Fig. The currents measured in the Piezo2 knockdown cells also tended to have slower inactivation kinetics, which is consistent with a loss of ultrafast inactivating currents mediated by PIEZO2 channels (Fig. To get closer to the role of mechanosensitive currents in the development of the coronary circulation, we managed to isolate tdTomato+ cECs from E13.5 hearts and cultured these cells on pillar arrays. Almost all cells identified as tdTomato+ 2 weeks after plating showed very robust and large mechanically activated currents to substrate deflection (14 of 15 cells tdTomato+, 7 of 8 cells tdTomato−) (Fig. However, unlike YS cells, the kinetics and current amplitudes of mechanosensitive currents did not differ between tdTomato+ and tdTomato− cells (Fig. We measured absolute deflection thresholds and currents were evoked by movements of less than 50 nm (Extended Data Fig. 4b); a similar sensitivity to that is seen for PIEZO2-dependent touch receptors32,36,37. However, the frequency with which mechanosensitive currents could be evoked was higher in cECs that were tdTomato+ (Fig. Thus, there appears to be clear PIEZO2-dependent endothelial cell mechanosensitivity, but other mechanosensitive ion channels such as PIEZO1 and ELKIN1 are probably also present9. Consistent with this idea, we could also measure clear stretch-activated currents, typical of PIEZO1 channels, in excised outside-out patches from tdTomato+ YS endothelial cells (Extended Data Fig. In addition, calcium influx responses indicated that the PIEZO1 activator Yoda1 (ref. 38) induces calcium influx in the majority of tdTomato+ and tdTomato− YS cells (Extended Data Fig. Furthermore, 63.6% of the tdTomato− and 66.6% of the tdTomato+ YS cells showed a significant YODA1 response, indicating that both PIEZO1 and PIEZO2 are present in YS endothelial cells. Moreover, Elkin1 transcripts were detected in the same endothelial cell populations in which we had detected Piezo2 (Extended Data Fig. Thus, apart from PIEZO2, other mechanosensitive ion channels contribute to mechanotransduction in yolk sacs and cECs. a, Left: representative picture of a cECtdTomato+ cell cultured on the elastomeric pillar arrays. RE, recording electrode; MS, mechanical stimulator. Right: example traces of RA currents from YStdTomato+ and cECtdTomato+ cells recorded with the pillar array method. b, Stacked histograms showing that the proportion of RA currents in YStdTomato+ cells (n = 12 biological replicates) was higher than in YStdTomato− cells (n = 11 biological replicates, chi-squared test, two sided, P = 0.02). c, YStdTomato+ cells showed deflection-gated currents with faster inactivation kinetics (τinact is the time constant of inactivation) compared with non-red cells (Mann–Whitney test, two sided, P = 0.004). d, YStdTomato+ cells responded statistically similar to YStdTomato− cells, with a trend that shows that they are slightly more sensitive to mechanical stimulation (t-test, P = 0.0699). e,f, YStdTomato+ cells did not show differences in the proportion of mechanosensitive currents (e) nor changes in inactivation kinetics (f) when transfected with Piezo2-siRNA. siRNA-control corresponds to cells transfected with non-targeting control siRNA. g, Plot showing that YStdTomato+ Piezo2-siRNA-transfected cells (n = 12) responded less to deflection stimuli than control cells (n = 10) (Mann–Whitney test, two sided, P = 0.01). h,i, cECtdTomato+ (n = 14) and cECtdTomato− cells (n = 7) did not show differences in the proportion of mechanosensitive currents (h) nor changes in inactivation kinetics (i). The numbers indicate number of currents. j, Histogram showing that cECtdTomato+ cells are more responsive to deflection stimuli than cECtdTomato− cells (Mann–Whitney test, two sided, P = 0.0392). Biological replicates are the number of cells, as indicated by n. Technical replicates for YS and cEC electrophysiology experiments as indicated in Methods. All error bars represent standard errors of the mean (±s.e.m. We next asked whether PIEZO2 channels are necessary for normal heart development. Piezo2−/− mutant hearts were smaller and shorter (base to apex length) at birth than those of wild-type (WT) Piezo2+/+ littermates (Fig. However, measuring heart-to-body weight ratios, we found that the ratios were increased in mutants, indicating that Piezo2−/− embryonic hearts were heavier than WT hearts (Fig. We made a detailed morphological analysis using the endocardial marker EMCN to differentiate between the trabecular myocardium (TM) and compact myocardium (CM) (Fig. The results revealed that Piezo2−/− hearts showed clear ventricular hyperplasia, which was most evident for the left ventricle (LV) (Fig. In addition, the interventricular septum (IVS) was also substantially thickened (Fig. No obvious outflow tract defects, such as common arterial trunk and double outflow tract right ventricle (RV), were observed in the Piezo2−/− mutants. Pulmonary artery and ascending aorta diameters were comparable between WT and Piezo2−/− hearts (Extended Data Fig. Longitudinal and transverse sections of the valves showed clear separation of the leaflets (Extended Data Fig. b, Piezo2−/− mutant hearts show a shorter base-to-apex length (n = 8 per genotype, Mann–Whitney test, P = 0.02). c, Piezo2−/− mutant mice had increased heart-to-body weight ratio (n = 11 per genotype, Mann–Whitney test, P = 0.04). d, Ventricular wall thickness was measured from E18.5 Piezo2+/+ and Piezo2−/− hearts immunolabeled against EMCN (magenta) to distinguish between CM and TM, respectively. Dashed lines indicate border between CM and TM. Hematoxylin and eosin staining was performed for an overall morphological heart assessment. Two representative Piezo2−/− hearts are shown with a milder hyperplasia phenotype (middle row) and a severe hyperplasia phenotype (last row), including the presence of an aberrant outflow tract (OFT). e–g, Piezo2+/+ and Piezo2−/− (n = 4 per genotype) LV (e), RV (f) and IVS (g) wall thickness quantification show that E18.5 Piezo2−/− presents a thicker LV wall and IVS, indicating cardiac hyperplasia (LV, two-way ANOVA, P = 0.04 (CM) and P = 0.17 (TM); RV, two-way ANOVA, P = 0.06 (CM) and P = 0.64 (TM); IVS, Mann–Whitney test, two sided, P = 0.02). i, Piezo2R2756H/R2756H E18.5 heart-to-body weight ratio is similar to that of Piezo2+/+ (n = 13 Piezo2+/+ and n = 11 Piezo2R2756H/R2756H, Mann–Whitney test, P = 0.60). j, Representative macroscopic heart images of 20-week-old mice carrying a Piezo2 knock-in gain-of-function mutation (p. Arg2756His) and Piezo2+/+ mice. k, Piezo2R2756H/R2756H mouse body weight is significantly less than Piezo2+/+ mouse body weight (Mann–Whitney test, two sided, P = 0.003). l, The heart-to-body weight ratio is increased in Piezo2R2756H/R2756H mutants compared with Piezo2+/+ (Mann–Whitney test, two sided, P = 0.003). m, Both groups had similar LV mass (Mann–Whitney test, two sided, P = 0.83). n, Heart size was measured from base to apex showing that Piezo2R2756H/R2756H hearts are smaller than Piezo2+/+ hearts (Mann–Whitney test, two sided, P = 0.004), indicating hyperplasia or hypertrophy. Rare PIEZO2 pathogenic variants in humans are associated with neurodevelopmental disorders such as Gordon syndrome and Marden–Walker syndrome as well as distal arthrogryposis16,17. We generated mice carrying a pathogenic homozygous gain-of-function mutation (Piezo2R2756H/R2756H) that dramatically sensitizes the activation of endogenous PIEZO2 channels by relieving voltage block of the channel39. At E18.5, Piezo2R2756H/R2756H mutant hearts showed no overt phenotype with similar size (Fig. 5h) and heart-to-body weight ratio compared with WT littermate controls (Fig. As homozygous Piezo2R2756H/R2756H mutants are viable, we examined the hearts of these animals at 20 weeks of age postmortem (Fig. Similarly to embryonic Piezo2−/− hearts, we found that the Piezo2R2756H/R2756H mutant hearts were smaller and shorter than hearts from age-matched WT littermates. We noted that Piezo2R2756H/R2756H mutant mice at 10 weeks weighed significantly less than WT littermates (Fig. Indeed, echocardiography (ECG) measurements from living Piezo2R2756H/R2756H and WT littermate mice at 10 weeks of age showed that Piezo2R2756H/R2756H mutant mice displayed a significantly elevated heart-to-body weight ratio compared with controls (Fig. LV mass was comparable between genotypes (Fig. These results suggest that both loss- and gain-of-function PIEZO2 can lead to macroscopic hyperplastic changes in the embryonic and adult heart. The LV hyperplasia observed in Piezo2−/− mice is a phenomenon that can result from reduced perfusion of the cardiac muscle. As PIEZO2 is highly specific to the coronary endothelium, we hypothesized that PIEZO2 mechanosensory function in these cells guides the normal assembly of the coronary vasculature. We performed whole-organ immunostaining against αSMA in immunolabeling-enabled three-dimensional imaging of solvent-cleared organs (iDISCO)-cleared embryonic hearts. Using light-sheet imaging and vessel segmentation, we could visualize the entire coronary vasculature in WT and Piezo2−/− hearts (Fig. a, A 3D rendering of representative Piezo2+/+ and Piezo2−/− E18.5 hearts immunolabeled against αSMA (gray), a smooth muscle marker, used for coronary vasculature segmentation (cyan). The yellow arrowheads on the mutant heart point to an aberrant ectopic coronary branch in a Piezo2−/− heart. c, The circumflex artery of Piezo2−/− hearts branches closer to the left coronary artery ostia (n = 8 per genotype, Mann–Whitney test, P = 0.007). d,e, Number of RCA (d) and LCA (e) branching points (unpaired Student's t-test, P = 0.153 (RCA) and P = 0.08 (LCA)). Piezo2−/− presents with a significantly smaller left LCA diameter (Mann–Whitney test, P = 0.02 (LCA in h) and P = 0.53 (RCA in g)). Representative images are shown in f. Scale bars, 50 µm. All tests were two sided and error bars represent ±s.e.m. In Piezo2−/− hearts, the coronary vasculature topology was clearly altered compared with the WT. 6a (orange asterisks) and Extended Data Fig. Quantitative analyses revealed a significantly shorter segment length between the coronary ostia and the Cx branching point in Piezo2−/− hearts compared with controls (Fig. There was no clear distinction between the main LDA and its branches in the mutant heart compared with the WT (Fig. 6a (white arrows) and Extended Data Fig. The branch points in mutant vessels were frequently trifurcations rather than bifurcations. In addition, in one mutant heart, an ectopic artery branching off the Cx projecting toward the anterior pole of the heart was observed (Fig. Manual and computer-vision-based quantification of branch points revealed that while the number of branch points was not altered in the RCA tree of Piezo2−/− mutants (Fig. 9d), fewer branch points were observed in the left coronary artery tree of mutants compared with controls (Fig. Most importantly, the light-sheet reconstruction indicated that Piezo2−/− hearts have more constricted coronary vessels. As LV hyperplasia was most prominent in Piezo2−/− hearts, we quantified the average diameter of the left and right main coronary artery over 600 µm from the ostia (Fig. We observed significant narrowing of the left coronary artery diameter in Piezo2−/− hearts compared with the WT (Fig. To assess potential perfusion changes caused by the LCA constriction, we applied physical first principles. We considered a cylindrical pipe of constant cross-section where an incompressible, Newtonian fluid is flowing under laminar flow conditions as an idealized model of the LCA. The LCA diameter has narrowed, on average, from 66.377 μm to 58.492 μm, that is, an 11.879% reduction in Piezo2−/− hearts compared with Piezo2+/+ controls. Given a comparable pressure drop between the entrance of the LCA and its post-capillary venous return in both the Piezo2+/+ and Piezo2−/− groups (as can be expected from a hierarchically branched vascular tree with multiple major arteries running in parallel, for example, LCA and RCA), the fact that the largest pressure drop along this path happens in small arteries and arterioles42, and all the remaining variables being equal, it follows from the previous equations and assumptions that the resistance of the LCA will have increased by 65.7% in Piezo2−/− hearts and that this will lead to a very significant reduction of perfusion downstream from it despite the relatively modest reduction in LCA diameter. We next assessed the coronary artery topology in E18.5 Piezo2R2756H/R2756H hearts using iDISCO clearing, αSMA immunostaining and light-sheet imaging followed by vessel segmentation and coronary vasculature surface reconstruction in a total of six Piezo2R2756H/R2756H mutant and seven WT control samples (Fig. Quantification of segment length and branch point numbers was conducted by two different methods (manual and computer vision based) and performed by different researchers in a blinded manner. a, A 3D rendering of representative Piezo2+/+ and Piezo2R2756H/2756H E18.5 hearts immunolabeled against αSMA (gray), a smooth muscle marker, used for coronary vasculature segmentation (cyan). b–d, Analyses of the coronary vascular tree topology, showing that Piezo2R2756H/R2756H presents a significantly increased number of LCA branching points (BP) (n = 7 Piezo2+/+ and n = 6 Piezo2R2756H/R2756H, Mann–Whitney test, P = 0.234 (ostia-to-circumflex length in b), P = 0.125 (RCA BP in c) and P = 0.023 (LCA BP in d)). Representative images are shown in e. Scale bars, 50 µm. All tests were two sided and error bars represent ±s.e.m. Moreover, the number of total left coronary artery branch points was increased in Piezo2R2756H/R2756H gain-of-function mutant hearts (Fig. Taking these results together, we conclude that the proper function of the mechanosensitive PIEZO2 ion channels is necessary for the normal assembly of the cardiac endothelium to form an adequate vasculature supply to the developing myocardium. Our investigation reveals a critical role of mechanosensitive PIEZO2 function in the formation of the coronary vasculature, particularly in developing coronary endothelial cells. We detected a transient expression for Piezo2 within the Fabp4+ endothelial cell population in the heart, which peaked during the embryonic and early postnatal stages but diminished in adulthood, suggesting that PIEZO2 is crucial for coronary vessel formation but not for maintaining mature endothelial identity under healthy conditions. Piezo2-expressing coronary endothelial cells show unique expression profiles characterized by the enrichment of genes involved in cell migration, guidance and angiogenesis, including neuropilin 2, apelin and Crmp1 (ref. Neuropilin 2, originally recognized for its role in neuronal guidance25, also influences angiogenesis and probably guides endothelial cell migration and vessel formation23. Similarly, apelin, a hormone known for supporting angiogenesis, enhances endothelial cell survival and stabilizes blood vessels43. In addition, CRMP isoforms, initially associated with neuronal development26, are present in endothelial cells, where they probably contribute to cytoskeletal regulation, cell movement and angiogenesis. Of note, PIEZO2 expression has been reported in human cultured coronary artery endothelial cells44 and in endothelial cells of cardiac tissue samples from transplanted human hearts45. In an scRNA-seq analysis of P7 murine brain endothelial cells, it was shown that Piezo2 was among the 25 most enriched tip cell transcripts46. PIEZO2 has been implicated in transducing physical cues into mechanobiological responses, such as cytoskeletal rearrangements47. It is therefore plausible to hypothesize that PIEZO2 is needed to guide endothelial navigation at the angiogenic front of the coronary vascular plexus in response to mechanical cues. Additional previously published data further support a potential role for PIEZO2 in endothelial cell motility. Yang et al. described that PIEZO2 siRNA-treated HUVEC cells showed a significant decrease in tube formation compared with the control group. They also showed that shRNA-mediated Piezo2 knockdown in mice caused a significant reduction in the radial extension of vascular plexus from the optic nerve to the periphery at P3 and P5, as well as fewer branch points12. Wei et al. describe impaired cell migration and tube formation in pulmonary microvasculature endothelial cells isolated from endothelial cell-specific Piezo2EC/− knockout rats compared with WT rats13. Chemical signaling (chemotaxis), particularly through pathways such as VEGFA-VEGFR2, BMP2 and CXCL12/CXCR4, is well established in orchestrating endothelial cell movements underlying angiogenesis in various organs, including the heart3,4. However, there is increasing evidence that combinatorial effects of chemotaxis and ion channel-mediated durotaxis fine-tune cellular migration and navigation in diverse cellular systems5,48,49. In particular, the study by Canales Coutiño and Mayor in Xenopus showed that Piezo1 cooperates with chemotactic signaling cues to control neural crest migration48. PIEZO2 was also shown to balance mechanosensitivity and chemosensitivity at the blood–tumor barrier in medulloblastoma50. Indeed, endothelial PIEZO2 could be a major sensor of mechanical cues in the developing myocardium, facilitating cellular migration and motility through durotaxis. We propose that mechanical cues sensed by Piezo2+ coronary endothelial cells are crucial for precise cellular navigation during coronary vasculature plexus formation and remodeling. Our functional analysis showed that Piezo2+ endothelial cells possess fast mechanosensitive currents that have sensitivities and kinetics similar to those of Piezo2-dependent currents in sensory cells32,36,39. However, many cECs that had never expressed Piezo2 also exhibited mechanosensitive currents (Fig. 4), showing that PIEZO2 is only one of several functional mechanosensitive channels in cECs51. Using a knockdown approach in YS cells, we show directly that PIEZO2 contributes to mechanically gated currents in these cells, but it is clearly not the sole mechanosensitive ion channel present (Fig. Indeed, our data show that Elkin1 and Piezo1 are also expressed in cECs, and both channels are efficiently gated by substrate deflection or membrane stretch32. Nevertheless, it is striking that PIEZO2 loss- and gain-of-function experiments revealed a nonredundant role of endothelial PIEZO2-dependent mechanosensitivity in shaping coronary artery formation and branching. In the absence of Piezo2 expression, we observed aberrant coronary artery branching, that is, significantly decreased proximal left coronary artery segment length and less branch points of the left coronary artery tree and most importantly a decreased left coronary artery diameter. We propose that cardiac muscle perfusion deficits result from coronary anomalies in Piezo2−/− mutants during embryonic development. Reduced blood supply could lead to hypoxia-induced compensatory growth. Hemodynamic stress could additionally be a cause of heart malformations and hyperplastic myocardial growth observed in PIEZO2-deficient mouse embryos (Fig. Human pathogenic gain-of-function mutations in PIEZO2, documented in human patients16,17, dramatically increase the channels' sensitivity to mechanical force39. By introducing this gain-of-function mutation into mice, we show here that PIEZO2 hypersensitivity can also lead to coronary artery branch point anomalies (Fig. 9f–h) and disruption of postnatal heart development (Fig. The increased number of branch points in the left coronary artery of Piezo2R2756H/R2756H gain-of-function mutants, even with a normal proximal LCA diameter, could potentially lead to cardiac hyperplasia and hypertrophy. Increased local shear stress due to more branch points and increased local perfusion could stimulate cell cardiomyocyte proliferation in the embryo or cell growth in the adult. However, a direct functional link between PIEZO2 dysfunction in coronary arteries and cardiac muscle anomalies still needs further experimental evidence and a potential role of PIEZO2 in the outflow tract and the valves of cardiovascular phenotypes needs to be assessed in further studies. While the main causes of coronary artery disease are adult-onset conditions such as atherosclerosis, hypertension and diabetes due to lifestyle modifications, genetic predisposition can also have an important role in disease etiology52. Congenital coronary artery defects have been implicated in cardiac hyperplasia and hypertrophy53, and although they might be mild without any symptoms during development and in adolescence, they are considered important factors that can predispose individuals to cardiac disease in adult life54. Indeed, the heart malformations that we have observed are likely to contribute to early perinatal mortality in Piezo2−/− mice, which was attributed to failure to inflate the lungs after birth14,15. The importance of PIEZO2 in a variety of sensory functions, including pain, makes it a potentially attractive target for drug therapies. However, our findings indicate that PIEZO2 is also crucial for the development of coronary arteries in the heart. Considering that developmental programs are often reused in regeneration and remodeling conditions, PIEZO2 might be a promising candidate to be reexpressed in the adult heart to facilitate angiogenesis following cardiac ischemic episodes. Findings from recent studies support this hypothesis, for example, in rodents, Kloth and colleagues observed a clear upregulation of Piezo2 in stressed cardiac tissue55. Further studies describe upregulation of Piezo2 in pharmacologically stressed cultured human coronary artery endothelial cells44 and in cardiac samples from patients with heart failure56. Indeed, cardiovascular problems have been associated with PIEZO2 pathogenic variants in a small number of human patients57,58 and the University of Cambridge PhenoScanner V2 database of human genotype–phenotype associations suggests PIEZO2 genetic linkage with heart failure, diastolic blood pressure, hypertensive heart disease and thoracic aortic aneurysm18. In summary, our study elucidates the indispensable role of PIEZO2-dependent mechanosensitive signaling in coronary vasculature formation and highlights its implications for cardiac development and disease. Our study also raises a new issue for the use of therapeutics targeting PIEZO2 in the somatosensory system as such approaches may have serious consequences for cardiovascular (patho)physiology. All mouse experiments complied with the German Animal Protection Act and were approved by the Max Delbrück Center for Molecular Medicine and the local Berlin authority, Landesamt für Gesundheit und Soziales Berlin, under the following animal protocol numbers: G0233/19, G0017/17 and X9006/21. Mice were housed under standard conditions (22 ± 2 °C, 55 ± 10% humidity, 12:12 h light–dark cycle) with ad libitum access to food and water, and provided with nesting and hiding materials. Timed matings using mice >10 weeks old were set up to collect embryos at specific stages (plug day = E0.5). Pregnant females were killed by cervical dislocation at the desired embryonic stage. B6.Cg-Gt(ROSA)26Sortm14(CAG-tdTomato)Hze/J: Ai14 is a Cre reporter strain (The Jackson Laboratory, 007908) and expresses tdTomato fluorescence following Cre-mediated recombination. Ai14 mice were used for timed matings with B6(SJL)-Piezo2tm1.1(cre)Apat/J mice (The Jackson Laboratory, 027719)15, herein referred to as Piezo2-Cre mice, to obtain Piezo2-cre; Ai14 embryos expressing dtTomato in the Piezo2+ lineage (herein referred to as Piezo2-tdTomato mice). EIIa-cre-mediated recombination leads to germline transmission of the recombined Piezo2 allele. Piezo2−/− embryos were isolated after timed matings of Piezo2+/− parents >10 weeks. The Piezo2R2756H/R2756H strain was generated as described in Sánchez-Carranza et al.39. All these strains were kept on a C57BL/6N background (backcross onto C57BL/6N from C57BL/6J for more than 12 generations). Mouse embryonic hearts were fixed in 4% paraformaldehyde in phosphate-buffered saline (PBS) for 2 h at room temperature (RT), washed in 1× PBS, cryopreserved in 30% sucrose, embedded in optimal cutting temperature embedding matrix (Tissue-Tek, Sakura Finetek) and sectioned at 10 µm using a Leica CM1950 cryostat. Sections were permeabilized with 0.1% Triton X-100 for 30 min and blocked with 10% donkey serum (Biowest) and 1% bovine serum albumin (BSA; Sigma-Aldrich) for 1 h at RT. Primary antibodies diluted in phosphate-buffered saline with Tween 20 (PBST) with 1% serum were applied overnight at 4 °C, followed by secondary antibodies for 1 h at RT. 4′,6-Diamidino-2-phenylindole (DAPI; 62248, Invitrogen) was used for nuclear counterstaining, and DAKO fluorescent medium (S302380-2, Agilent) was used for mounting. Whole-mount immunostaining of E13.5 hearts was performed in 2-ml Eppendorf tubes under rotation. Hearts were permeabilized with 0.5% Triton X-100 in PBS for 1 h, then blocked with 10% donkey serum in 0.5% PBST for 2 h at RT. Primary and secondary antibodies were applied sequentially for 24 h at 4 °C, with PBST washes between steps. DAPI was used for nuclear counterstaining. Samples were mounted in µ-slide eight-well glass-bottom chambers (IBIDI) using Fluoromount G (00-4959-52, eBioscience). E18.5 Piezo2-tdTomato hearts were cleared using the SmartClear protocol (LifeCanvas Technologies), which preserves endogenous fluorescence. Before clearing, paraformaldehyde-fixed samples underwent SHIELD epoxy-based fixation. Clearing was performed for 3 days at 42 °C using the SmartClear device. Samples were incubated overnight at 37 °C in 50% EasyIndex and distilled water with shaking, then in 100% EasyIndex for refractive index matching. Samples were dehydrated, washed and incubated in 66% dichloromethane–33% methanol overnight at RT, then bleached overnight in 5% H2O2–methanol, rehydrated, washed and permeabilized. Samples were blocked and incubated in antibody solution for 4 days at 37 °C. Hearts were embedded in 1% low-melting agarose, dehydrated, incubated for 2.5 h in 66% dichloromethane–33% methanol, followed by 100% dichloromethane, and stored at RT in ethyl cinnamate. The following antibodies were used: goat anti-PECAM1 (1:250, AF3628, R&D Systems), rabbit anti-DACH1 (1:100, 10910-1-AP, Proteintech), rat anti-EMCN (1:100, clone V.C7C7, sc-65495, Santa Cruz), rabbit anti-FABP4 (1:100, ab13979, Abcam), goat anti-SOX17 (1:100, AF1924, R&D Systems), rabbit anti-RFP (1:1000, 600-401-3979S, Rockland), rat anti-VE-Cadherin (555289, clone 11D4.1, BD-Pharmigen), mouse anti-SMA (1:200, clone 11D4.1, A5228, Sigma-Aldrich), mouse anti-SMA-Cy3 (1:250, clone 1A4, C6198, Sigma-Aldrich), Alexa Fluor 488 rat (1:500, ab150153, Abcam), Alexa Fluor 555 rabbit (1:500, ab150074, Abcam), Alexa Fluor 647 rabbit (1:500, ab150075, Abcam), Alexa Fluor 647 goat (1:500, ab150131, Abcam) and Alexa Fluor 647 rat (1:500, ab150155, Abcam). Hematoxylin and eosin staining was performed, following the manufacturer's instructions on 10-µm-thick cryosections. Stained sections were imaged using a Leica DM5000B microscope. Single-molecule fluorescent in situ hybridization was carried out according to the manufacturer's instructions (RNAscope Multiplex Fluorescent V2 assay, 323110, ACD) on E13.5 WT heart sections. Immunofluorescently stained tissue sections and whole-mount E11.5 and E13.5 hearts were imaged using an LSM700 confocal microscope with ZEN software. The pinhole was set to 1 AU for all channels. Cleared and immunolabeled whole E18.5 hearts were imaged with a Zeiss Lightsheet 7 microscope, with the ZEN 3.1 (black edition) LS software. The acquisition was done with dual-side illumination (6.08 µm light-sheet thickness), using LSFM ×5/0.1 foc objectives for illumination and an EC Plan-Neofluar ×5/0.16 foc lens for detection, with the correction collars adjusted to the correct refractive index, depending on the immersion used (1.46 for EasyIndex, 1.56 for Ethylene Cinnamate). Acquisition was done sequentially, using solid-state lasers 405 nm and 561 nm (camera beamsplitter LP 510 nm, emission filters BP (420–470) nm/(575–615) nm), with a PCO.Edge 5.5 sCMOS camera (6.5 µm per pixel, 1,920 × 1,920 pixels), and using a zoom of 0.96 (0.97 × 0.97 × 2.00 µm per voxel). Post-acquisition processing was performed with the ZEN 3.4 (blue edition) software to fuse the dual-side light sheets of each sample while the FIJI BigStitcher plugin was used to stitch the tiles. Identical settings for laser power, detector and pixel size were used for all samples analyzed qualitatively or quantitatively. Ventricular wall thickness was measured from longitudinal sections of E18.5 Piezo2+/+ and Piezo2−/− E18.5 hearts. Six sections per animal were selected, three with the pulmonary artery and three with the aorta. A region of interest below the atria was defined to measure compact and TM thickness. Right and left coronary artery diameters were measured in FIJI from whole-mount E18.5 Piezo2+/+ and Piezo2−/− immunolabeled hearts. These five values per artery were averaged and used in statistical analysis. The diameters of the pulmonary artery and the aorta were measured using FIJI from optical sections obtained from whole-mount E18.5 Piezo2+/+ and Piezo2−/− immunolabeled hearts. Two measurements were taken per vessel and averaged. Coronary vasculature surface reconstruction was based on the iDISCO-cleared, αSMA-immunolabeled Piezo2+/+ and Piezo2−/− hearts. The surface package of IMARIS 10.0 was used to semi-automatically detect, segment and reconstruct the immunofluorescence signal. Filaments plugin from IMARIS was used to measure the ostia-to-circumflex artery length. Branching point analysis was based on the surface reconstruction of the coronary vasculature. We manually labeled the coronary artery root points for the right and left ventricles and refined segmentation masks using napari 0.5.6 (https://napari.org/stable/). Unwanted regions outside the right and left ventricles were removed and broken structures reconnected. Binary closing was used to fill the interior of the blood vessels and skeletonized the resulting 3D masks using Kimimaro (https://github.com/seung-lab/kimimaro), with roots as starting points. From these skeletons, we computed the distances to each branching point and the total number of branching points. We manually identified the branching points of interest and recorded their measurements (ostia–circumflex). The code is available at https://github.com/Kainmueller-Lab/piezo2_branching_point_analysis. Ultrasound examination was performed on anesthetized 10-week-old Piezo2+/+ and Piezo2R2756H/R2756H mice using the Vevo 3100 system (VisualSonics Fujifilm) with an MXD550D ultrasonic probe. Anesthesia was induced with 3–4% isoflurane (anesthesia box) and maintained with 1.5–2% isoflurane (mask ventilation). The examination area was shaved and treated with hair removal cream; eye ointment (Bepanthen) was applied to prevent dryness. The mice were placed on a warming plate with additional heat from a lamp, and body temperature was monitored via a rectal probe. ECG was recorded throughout to monitor heart rate and anesthesia depth. Contact gel ensured proper coupling between the transducer and skin. Recovery occurred under supervision in a warm, quiet environment. ECG data were analyzed using VevoLAB software (version 5.5.0). Primary cECs were isolated from pooled E13.5 Piezo2-tdTomato embryos (10–15 hearts per experiment). The retained tissue was digested in Collagenase/Dispase (10269638001, Sigma-Aldrich) for 45 min at 37 °C, homogenized and filtered into a tube with 5 ml DMEM containing 10% FBS (P40-37500, PAN-Biotech) and Gentamicin (G1272, Sigma-Aldrich). An additional 5 ml of the same medium was added, and the mixture was centrifuged for 5 min at 1,200 r.p.m. The pellet was resuspended in 1 ml PBS–BSA 0.5% and centrifuged for 5 min at 300 × g. The pellet was then resuspended in 100 µl PBS–BSA 0.5%, and 100 µl of bead–antibody solution was added and carefully homogenized. After 30 min of incubation at RT, 800 µl PBS–BSA 0.5% was added, and bead-bound cells were washed several times with MLEC medium containing 400 ml DMEM-F12 (21041-025, Invitrogen), 20% inactivated FBS, 1% penicillin–streptomycin (15140122, Gibco) and 4 ml ECGS-H (C-30120, Promocell). Cells were resuspended in 500 µl of MLEC, and each pool was seeded in one well of a 24-well plate coated with 0.5% gelatin (G1393, Sigma-Aldrich). The next day, cells were washed with 1 ml of PBS–BSA 0.5%, and 500 µl of MLEC was added to each well. After reaching confluency (usually 1 week), the cells were dissociated with trypsin and seeded on pillar arrays for electrophysiology. For the bead–antibody preparation, per heart pool, 6 µl of sheep anti-rat IgG Dynabeads (11035, Dynal Invitrogen) was placed in 1 ml of PBS–BSA 0.5%, washed several times and resuspended in 6 µl of PBS–BSA 0.5% for each pool. Then, 1.5 µl per pool of VE-cadherin antibody was added to the beads and incubated for 1 h at RT with gentle agitation. After incubation, 100 µl of PBS–BSA 0.5% per pool was added and gently homogenized. Technical replicates were generated as follows: embryonic hearts were collected from 5 pregnant females on 2 separate days, yielding over 50 hearts in total. These were divided into groups of 10–15 hearts to create at least 4 independent cEC preparations. Each preparation was plated in a single well for expansion. After expansion, each well was dissociated and the cells were seeded onto multiple pillar arrays, keeping each preparation separate to maintain independent replicates. YS cells were collected on ice in plating medium, consisting of DMEM-F12 (Invitrogen) supplemented with 2 μM l-glutamine (Sigma-Aldrich), 8 mg ml−1 glucose (Sigma-Aldrich), 200 U ml−1 penicillin–streptomycin and 10% fetal horse serum. YS cells were treated with 1 mg ml−1 collagenase IV (Sigma-Aldrich) and incubated at 37 °C for 15 min in a water bath. Cells were washed 3× with PBS and incubated with 0.05% trypsin (Invitrogen) for 15 min at 37 °C. Collected tissue was triturated in plating medium using a pipette tip and then centrifuged at 900 r.p.m. The pellet was resuspended in 1 ml of plating medium and plated on elastomeric pillar arrays or coverslips pre-coated with 4 μg cm−2 laminin (Invitrogen). YS cells were cultured overnight on pillar arrays. Subsequently, 24 h after plating, cells were transfected with Accell siRNA and Accell media as per the manufacturer's protocol (Dharmacon, Horizon) using either non-targeting control siRNA (SMARTpool, D-001960-01-20) or piezo2-siRNA (SMARTpool, E-163012-00-0020). Electrophysiological recordings were carried out 96 h post-transfection. Every culture of YS cells was made from at least five different YSs of embryos at stage E13.5 from three different pregnant females. Three independent transfections were performed. Pillar arrays were prepared following the established protocol32. In summary, silanized negative masters served as templates, which were subsequently coated with polydimethylsiloxane (PDMS) from the syligard 184 silicone elastomer kit (Dow Corning) mixed with a curing agent in a 10:1 ratio (elastomeric base-to-curing agent ratio). The mixture was incubated for 30 min, and glass coverslips were positioned on the top of negative masters containing PDMS, followed by banking for 1 h at 110 °C. Subsequently, pillar arrays were peeled from the negative masters. Each pilus within the array had a radius of 1.79 µm and a length of 5.8 µm. Before cell culture use, the pillar arrays underwent plasma cleaning using a Femto low-pressure plasma system (Deiner Electronic) and were salinized using vapor phase (tridecafluoro-1,1,2,2-tetrahydrooctyl trichlorosilane 97% (AB111444, ABCR GmbH & Co. KG)) for 45 min, followed by a coating with EHS laminin and poly-l-lysine in a 1:1 ratio (v:v) for at least 2 h at 37 °C. Whole-cell patch–clamp recordings were conducted on YS and cEC cells (isolated as described above) using borosilicate glass pipettes (Harvard apparatus, 1.17 mm × 0.87 mm) with a resistance of 3–6 MΩ. The pipettes were filled with intracellular solution containing the following (in mM): 110 KCl, 10 NaCl, 1 MgCl2, 1 ethylene glycol tetraacetic acid (EGTA) and 10 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES); the pH was adjusted to 7.3 with KOH. The extracellular solution contained the following (in mM): 140 NaCl, 4 KCl, 2 CaCl2, 1 MgCl2, 4 glucose and 10 HEPES; the pH was adjusted to 7.4 with NaOH. Pipette and membrane capacitances were compensated using the auto-function of Patchmaster (HEKA, Elektronik) while series resistance was compensated to minimize voltage errors. For pillar arrays, currents were recorded at a holding potential of −60 mV, sampled at 10 kHz and filtered at 3 kHz using an EPC-10 USB amplifier and Patchmaster software version 2×91 (HEKA, Elektronik). Data were analyzed with FitMaster, version 2 × 91 (HEKA, Elektronik). A single pilus was deflected using a heat-polished borosilicate glass pipette driven by an MM3A micromanipulator (Kleindiek Nanotechnik) as previously described in Poole et al.32. Deflection-gated currents were classified according to their inactivation kinetics: RA (τinact < 5 ms), IA (τinact 5–50 ms) and SA (τinact > 50 ms). To calculate pillar deflection, bright-field images (Zeiss 200 inverted microscope) were captured using a ×40 objective and CoolSnapEZ camera (Photometrics) before and after every pillar stimulus. Pillar deflection was determined by comparing the light intensity of the center of each pilus before and after every stimulus using a 2D-Gaussian fit (Igor Software, WaveMetrics). High-speed pressure clamp experiments (ALA Scientific) were conducted on excised outside-out patches from YS cells. Recording pipettes with a final resistance of 6–8 M were used. Positive-pressure pulses were delivered through the recording pipette. The pressure steps protocol involved a series of stimuli ranging from 10 mm Hg to 150 mm Hg, in 20-mm Hg increments, while maintaining the patch potential at −60 mV. Recording solutions were prepared in symmetrical ionic conditions containing the following (in mM): 140 NaCl, 10 HEPES and 5 EGTA adjusted to pH 7.4 with NaOH. For both methods, currents and biophysical parameters were analyzed using FitMaster (HEKA, Elektronik). YS cells were cultured on plating medium (DMEM-F12, Invitrogen) supplemented with 2 μM l-glutamine (Sigma-Aldrich), 8 mg ml−1 glucose (Sigma-Aldrich), 200 U ml−1 penicillin–streptomycin and 10% fetal horse serum, and then loaded with Cal-520 (5 mM, CAL520 AM, AAT Bioquest). Images were acquired with an Olympus BX51WI microscope equipped with a DG4 (Sutter Instruments) and a CoolSNAP ES camera (Visitron). Image acquisition and analysis were performed using MetaFluor (Molecular Devices). Ionomycin (1 µM) was used to normalize the maximum signal of Cal-520. Data were plotted using the formula ΔF/F0 = (F − F0)/F0. Gene count matrices from single-cell sequencing from E12, E15, P2 and 8-week-old C57BL/6J animals were generated by Cano et al.22 and deposited on the Gene Expression Omnibus repository with accession number GSE223266. Sequence data were mapped to the mouse reference genome (mm10, pre-build references v 2.1.0). Custom code was generated using R to analyze the data and generate plots as described in Cano et al.22. Data were analyzed using GraphPad Prism 9.3.1 and tested for normality. Parametric data were compared using unpaired Student's t-test, and nonparametric data were compared using the Mann–Whitney test. Results are reported as mean ± s.e.m. Exact P values and n are provided in the figure legends. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data that support the findings of this study are available in the article and Supplementary Information. Gene count matrices from single-cell sequencing from E12, E15, P2 and 8-week-old C57BL/6J animals (mouse reference genome mm10) were generated by Cano et al.22 and deposited in the Gene Expression Omnibus repository with accession number GSE223266. The code used for computer-vision-based 3D coronary artery reconstruction is available on GitHub at https://github.com/Kainmueller-Lab/piezo2_branching_point_analysis. The code has been deposited in a capsule with Code Ocean as well. Khera, A. V. & Kathiresan, S. Genetics of coronary artery disease: discovery, biology and clinical translation. Cano, E. et al. Extracardiac septum transversum/proepicardial endothelial cells pattern embryonic coronary arterio-venous connections. Chang, A. H. et al. DACH1 stimulates shear stress-guided endothelial cell migration and coronary artery growth through the CXCL12–CXCR4 signaling axis. D'Amato, G. et al. Endocardium-to-coronary artery differentiation during heart development and regeneration involves sequential roles of Bmp2 and Cxcl12/Cxcr4. The guidance receptor plexin D1 is a mechanosensor in endothelial cells. Single stretch-activated ion channels in vascular endothelial cells as mechanotransducers? Coste, B. et al. Piezo1 and Piezo2 are essential components of distinct mechanically activated cation channels. & Kalli, A. C. Force sensing by piezo channels in cardiovascular health and disease. Ranade, S. S. et al. Piezo1, a mechanically activated ion channel, is required for vascular development in mice. Li, J. et al. Piezo1 integration of vascular architecture with physiological force. Wu, Z. et al. Mechanosensory hair cells express two molecularly distinct mechanotransduction channels. Yang, H. et al. Piezo2 protein: a novel regulator of tumor angiogenesis and hyperpermeability. Ranade, S. S. et al. Piezo2 is the major transducer of mechanical forces for touch sensation in mice. Woo, S.-H. et al. Piezo2 is required for Merkel-cell mechanotransduction. The role of PIEZO2 in human mechanosensation. McMillin, M. J. et al. Mutations in PIEZO2 cause Gordon syndrome, Marden–Walker syndrome, and distal arthrogryposis type 5. Kamat, M. A. et al. PhenoScanner V2: an expanded tool for searching human genotype–phenotype associations. Raftrey, B. et al. Dach1 extends artery networks and protects against cardiac injury. Tian, X. et al. De novo formation of a distinct coronary vascular population in neonatal heart. Han, M. & Zhou, B. Sox17 and coronary arteriogenesis in development. Cano, E. et al. Intramyocardial sprouting tip cells specify coronary arterialization. Harman, J. L., Sayers, J., Chapman, C. & Pellet-Many, C. Emerging roles for neuropilin-2 in cardiovascular disease. Red-Horse, K., Ueno, H., Weissman, I. L. & Krasnow, M. A. Coronary arteries form by developmental reprogramming of venous cells. Chen, H., Chédotal, A., He, Z., Goodman, C. S. & Tessier-Lavigne, M. Neuropilin-2, a novel member of the neuropilin family, is a high affinity receptor for the semaphorins Sema E and Sema IV but not Sema III. Yamashita, N. & Goshima, Y. Collapsin response mediator proteins regulate neuronal development and plasticity by switching their phosphorylation status. He, L., Tian, X., Zhang, H., Wythe, J. D. & Zhou, B. Fabp4–CreER lineage tracing reveals two distinctive coronary vascular populations. Young, S. G. et al. A protein of capillary endothelial cells, GPIHBP1, is crucial for plasma triglyceride metabolism. Li, Z. et al. FSCN1 acts as a promising therapeutic target in the blockade of tumor cell motility: a review of its function, mechanism, and clinical significance. The mechanosensitive Piezo1 channel mediates heart mechano-chemo transduction. Poole, K., Herget, R., Lapatsina, L., Ngo, H. D. & Lewin, G. R. Tuning Piezo ion channels to detect molecular-scale movements relevant for fine touch. Structure and mechanogating of the mammalian tactile channel PIEZO2. & Lewin, G. R. Mechanosensitive currents in the neurites of cultured mouse sensory neurones. Direct measurement of TRPV4 and PIEZO1 activity reveals multiple mechanotransduction pathways in chondrocytes. Chakrabarti, S. et al. Touch sensation requires the mechanically gated ion channel ELKIN1. Wetzel, C. et al. Small-molecule inhibition of STOML3 oligomerization reverses pathological mechanical hypersensitivity. Syeda, R. et al. Chemical activation of the mechanotransduction channel Piezo1. Sánchez-Carranza, O. et al. Piezo2 voltage-block regulates mechanical pain sensitivity. Reese, D. E., Mikawa, T. & Bader, D. M. Development of the coronary vessel system. Secomb, T. W. Blood flow in the microcirculation. B. Longitudinal distribution of vascular resistance in the pulmonary arteries, capillaries, and veins. Identification and functional analysis of endothelial tip cell-enriched genes. Different transcriptional profiling between senescent and non-senescent human coronary artery endothelial cells (HCAECs) by omeprazole and lansoprazole treatment. Tucker, N. R. et al. Transcriptional and cellular diversity of the human heart. Sabbagh, M. F. et al. Transcriptional and epigenomic landscapes of CNS and non-CNS vascular endothelial cells. Pardo-Pastor, C. et al. Piezo2 channel regulates RhoA and actin cytoskeleton to promote cell mechanobiological responses. & Mayor, R. The mechanosensitive channel Piezo1 cooperates with semaphorins to control neural crest migration. Pillai, E. K. & Franze, K. Mechanics in the nervous system: from development to disease. Chen, X. et al. Mechanosensitive brain tumor cells construct blood–tumor barrier to mask chemosensitivity. Lim, X. R. & Harraz, O. F. Mechanosensing by vascular endothelium. Schnitzler, G. R. et al. Convergence of coronary artery disease genes onto endothelial cell programs. Bland, E. F., White, P. D. & Garland, J. Congenital anomalies of the coronary arteries: report of an unusual case associated with cardiac hypertrophy. Feng, J., Zhao, J., Li, J., Sun, Z. Classification, diagnosis and clinical strategy of congenital coronary artery disease in children. Kloth, B. et al. Piezo2 is not an indispensable mechanosensor in murine cardiomyocytes. Koenig, A. L. et al. Single-cell transcriptomics reveals cell-type-specific diversification in human heart failure. Klaniewska, M. et al. Case report: Further delineation of neurological symptoms in young children caused by compound heterozygous mutation in the PIEZO2 gene. Mehandru, S. et al. Atypical presentation of Gordon syndrome and its management: a report of three patients. Rios Coronado, P. E. & Red-Horse, K. Enhancing cardiovascular research with whole-organ imaging. We thank A. Scheer (Max Delbrück Center for Molecular Medicine (MDC)), L. Fürst (MDC) and F. Bartelt (MDC) for their help with mouse genotyping. We thank S. Rode, F. Westphal and F. Kratz for animal husbandry at the MDC. We thank V. Begay (MDC) for help with animal protocols. We thank M. Gotthardt (MDC) and members of the Hammes and Lewin laboratories for constructive comments on the paper. We thank A. Patapoutian (Howard Hughes Medical Institute, Department of Neuroscience, Dorris Neuroscience Center, Scripps Research) for providing the B6(SJL)-Piezo2tm1.1(cre)Apat/J and Piezo2+/− mouse strains. This study was funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) CRC 958 (SFB 958) to A. Hammes and G.R.L. ; the DFG (German Research Foundation)—Project-ID 437531118—CRC (SFB) 1470, the Chan Zuckerberg Foundation and an ERC Advanced Grant (AdG788970) to N.H.; an Alexander von Humboldt fellowship (postdoctoral researcher) for W.L. ; the DFG (German Research Foundation) CRC (SFB) 1470-A03 to H.G. These authors contributed equally: Carina Fürst, Oscar Sánchez-Carranza. Molecular Pathways in Cortical Development, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Mireia Pampols-Perez, Carina Fürst & Annette Hammes Molecular Physiology of Somatic Sensation Laboratory, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Carina Fürst, Oscar Sánchez-Carranza, Jonathan Alexis Garcia-Contreras, Wenhan Luo & Gary R. Lewin Integrative Vascular Biology, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Elena Cano, Norbert Hübner & Holger Gerhardt Biomedical Image Analysis, Max-Delbrück-Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Advanced Light Microscopy & Image Analysis, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Cardiovascular and Metabolic Sciences, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Animal Phenotyping, Max Delbrück Center for Molecular Medicine in the Helmholtz Association (MDC), Berlin, Germany Norbert Hübner, Holger Gerhardt & Gary R. Lewin Helmholtz Institute for Translational AngioCardiosciences (HI-TAC), Max Delbrück Center for Molecular Medicine at Heidelberg University, Heidelberg, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Mouse model design: A. Hammes, G.R.L., M.P.-P. and O.S.-C. Developmental characterization, immunohistochemistry, light-sheet imaging and single-molecule fluorescence in situ hybridization: M.P.-P., C.F., E.C., S.R. The authors declare no competing interests. Nature Cardiovascular Research thanks Kristy Red Horse and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a - d, Outflow tract and cardiac valves three-dimensional rendering and immunolabelled sections from an E18.5 tdTomato+ heart. tdTomato signal (yellow) in the outflow tract (a) colocalizes with αSMA (magenta). In the cardiac valves (b - d), tdTomato+ cells (yellow) are surrounded by endothelial cells (PECAM1 positive, magenta), indicating that they are likely valve interstitial cells (n = 2, Scale bar: 100 µm [3D] and 50 µm [sections]). a, Expression of Piezo2 mechanosensitive ion channels in heart tissue. Single-molecule in situ hybridization showing coronary endothelial cells expressing Piezo2 and Dach1 at E13.5. b, scRNAseq identifies distinct cluster representations for Piezo2+ and Piezo1+ cardiac cells. Volcano plots showing Fabp4, Piezo1 and Piezo2 expression levels on cardiac endothelial cells and all cardiac cells from E12, E15, P2 and murine adult hearts (8 weeks old) showing the following clusters: endocardium (Endoc_I to Endoc_V), coronary endothelium (cap-to-vein, cap-to-artery and CoEC_III), proliferating cells (Prolif_I to Prolif_II), valvular endocardium (Valve_I and Valve_II), arterial cells (Arter), venous cells (Vein), outflow tract cells (OFT), pericytes, fibroblasts (FB_I and FB_II), lymphatic cells (lymph), smooth muscle cells (SMC), epicardial cells (Epic), macrophages (macroph) and red blood cells (RBC), Cardiomyocytes (CMs), Immediate Early Genes-expressing Coronary Endothelium (IEG_CoEC). Note: The analysis includes some non-endothelial cells, as the isolation of CD31 + /CD45- endothelial cells by BD FACSAria II sorting following heart dissociation is never completely pure (Cano et al., 2025, Circ. a - c, Immunolabeling of whole yolk sacs showing tdTomato signal (in yellow) colocalized with the endothelial marker PECAM1 (in magenta) at embryonic days E13.5 (a), E16.5 (b), and E18.5 (c) (n = 4 per stage, Scale bar: 50 µm). a - b, Deflection-current amplitude relationships of the yolk sac (YS) cells (a) and cardiac endothelial cells (cEC) (b) showing that deflection-gated currents from tdTomato+ and tdTomato- cells displayed similar mechanical responses (Two-way ANOVA, P = 0,3025 for YS and p = 0,5806 for cEC). c, Stacked histograms showing the percentage of YStdTomato+ either transfected with non-targeting control siRNA or Piezo2-siRNA that responded to pillar deflection (Chi-squared test, two-sided, P = 0.07). d, Deflection-Current amplitude relationship showing similar deflection-responses between YStdTomato+ transfected with control siRNA or Piezo2-siRNA. e, Representative trace of stretch-sensitive currents stimulated with a pressure-step protocol from outside-out patches in YStdTomato+ cells at a resting membrane potential of -60 mV. f, Stretch-response curve of the endogenous stretch-sensitive currents in YStdtomato+ cells. Peak currents were normalized to the maximum current recorded prior to membrane patch rupture. All data is presented as mean ± S.E.M. a - b, above: Examples of individual calcium influx responses from YStdTomato- (grey) and YStdTomato+ (red) cells. The arrows indicate the starting perfusion time of Yoda, Ruthenium red (RR), Extracellular solution (ES) and Ionomycin (iono). below: representative images of calcium imaging experiments from YStdTomato- and YStdTomato+ loaded with Cal520 before (ES) and after perfusion with Yoda1, RR, and Ionomycin. d, Stacked histograms showing that the proportion of responsive cells to Yoda in YStdTomato- (63.6%) and YStdTomato+ (66.6%). Six independent experiments were conducted for each group. a, UMAP of cardiac endothelial cells from E12.5, E15.5, P2 and murine adult hearts (8 weeks old) showing the following clusters: endocardium (Endoc) that includes five different subclusters_(Endoc_I to Endoc_V), coronary endothelium (CoEC) including the capillary-to-vein (cap-to-vein) and capillary-to-artery (cap-to-artery) clusters, proliferating cells (Prolif_I and Prolif_II), valvular endocardium (Valve_I and Valve_II), arterial and outflow tract cells (Artery & OFT) and venous cells (Vein). b, Volcano plots showing Elkin1 expression levels in cardiac endothelial cells. a, Representative images of the aortic and pulmonary valves of Piezo2+/+ and Piezo2-/- E18.5 embryos. Optical sections from whole mount light sheet images are shown. b, Piezo2-/- mutants present a similar pulmonary artery and aorta diameter compared to Piezo2+/+ (n = 5 per genotype, Mann-Whitney test, two-sided, P = 0.420 [Pulmonary artery] and P = 0.547 [Aorta]. a - d, iDisco-cleared and αSMA (gray) immunolabelled hearts from Piezo2+/+ wild types (a and c) and Piezo2-/- mutants (b and d), used for coronary vasculature reconstruction (cyan) (Scale bars: 300 µm). b, three representative images with artery centerline generated by this method, for Piezo2+/+, Piezo2-/- and Piezo2R2756H/R2756H hearts, respectively at E18.5. c - e, Same parameters measures as in Fig. (Unpaired Student t-test, two-sided, P = 0.014 [ostia to circumflex length in panel c], P = 0.59 [RCA branching point number in panel d], P = 0.049 [LCA branching point number in panel e]). f - h, Same parameters measures as in Fig. 7b–d (Unpaired Student t-test, two-sided, P = 0.034 [ostia to circumflex length in panel f], P = 0.31 [RCA branching point number in panel g], P = 0.076 [LCA branching point number in panel h]). A 3D rendering, segmentation and reconstruction of the Piezo2-cre; dtTomato fate-mapped E18.5 heart coronary vasculature, related to Fig. Video recording of the optical view of a representative whole-mount Piezo2-cre; tdTomato+ E18.5 heart, related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of eight Piezo2+/+ hearts (videos 3–10), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering, segmentation and reconstruction of individual E18.5 hearts immunolabeled against alpha-smooth muscle (αSMA) with the coronary vasculature reconstructed. Video recording (360° horizontal view) of seven Piezo2−/− hearts (videos 11–18), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. A 3D rendering of Piezo2+/+ and Piezo2R2756H/R2756H E18.5 hearts (videos 19–31), related to Fig. Supplementary Tables 1–3 present differential gene expression results from scRNA-seq analysis performed on samples collected at three developmental stages: embryonic day 12.5 (E12.5) and day 15.5 (E15.5), and postnatal day 2 (P2). Each table includes genes with associated statistical metrics including raw and adjusted P values, average log2(FC) and the proportion of cells expressing each gene in the two cell populations compared (Piezo2+ and Piezo2− cECs). These datasets support the findings shown in Fig. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Pampols-Perez, M., Fürst, C., Sánchez-Carranza, O. et al. Mechanosensitive PIEZO2 channels shape coronary artery development. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/creating-bird-flu-vaccines-for-humans-at-a-biosecure-laboratory/'>Bird Flu Vaccines Are under Development at Biosecure Laboratories like This</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>This is How We Fight Bird Flu If H5N1 Becomes the Next Human Pandemic By Naeem Amarsy, Jeffery DelViscio, Fonda Mwangi, Alex Sugiura & Rachel Feltman This is the final episode of our three-part series on bird flu. On Wednesday we met scientists who are getting their hands dirty with dairy cows and poultry to better understand how H5N1 bird flu is spreading. Today we'll take a look at efforts to create vaccines for H5N1—and learn why eggs are so critical to the vaccine-making process. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Our host today is Naeem Amarsy, a multimedia journalist based in New York City. While the rest of the city steps out in sandals and shorts, I'm watching a team of scientists at Texas Biomedical Research Institute. They're rummaging through metallic shelves to find the extra layers of protective gear they need to start their day: scrubs, gowns, gloves, shoe covers, hairnets and long white coveralls. Luis Martinez-Sobrido: So, we need to change all our clothes. That's why we have all these cabinets there. He observes with a keen eye as two members of his lab dress up for their next shift. In these highly secure labs scientists handle some of the world's potentially deadliest viruses. As we learned in the first two episodes of this series the virus is moving between species, from wild birds to their domesticated counterparts to cows and other mammals. The currently circulating strains have already infected dozens of people, mostly poultry and cattle farmworkers. Luckily, most of these infections have been mild. Martinez-Sobrido: We do not have any preexisting immunity against this H5N1 virus, and if it's able to transmit, it will potentially cause a pandemic. Amarsy: I wanted to understand what virologists like Luis are discovering about bird flu and how they might try to protect us if it does start jumping between humans. Inside the anteroom, a sealed space between the outside world and the BSL-3 lab, each of Luis's colleagues puts on their final piece of equipment: a battery-powered respirator connected to a large white hood with a transparent front that they call “the bubble.” From left: Ramya Smithaveni Barre and Ahmed Elsayed of the Texas Biomedical Research Institute put on personal protective equipment before entering a BSL-3, or biosafety level three, laboratory. Ahmed Elsayed: So we start with checking our bubble and connecting the battery. [CLIP: Elsayed's respirator battery makes a noise as it's plugged in.] Amarsy: That's Ahmed Elsayed, a staff scientist working in Luis's lab. In his immaculate white coveralls he reminds me of a beekeeper. Elsayed: We are waiting to have everyone ready before opening the second door because we cannot open it twice. Amarsy: As they disappear into the lab I am left wondering what Ahmed and his [colleagues] will do over the next six hours of their shift, working with those potentially deadly viruses. Amarsy: Recombinant viruses are one of the most powerful tools in virology. They let scientists like Luis use genetic sequences to re-create and modify viruses found in the wild—all that without having to rely on samples from the outside world. This helps researchers test a bunch of things about viruses such as H5N1, from how they respond to antiviral treatments to how they mutate and how sick they can make us. And making the annual seasonal flu vaccines is a coordinated global effort. They make these recommendations based on which strains experts think are most likely to spread. In the U.S., the Food and Drug Administration considers that data and then makes its own recommendations, which pharmaceutical manufacturers use to produce millions of doses that are distributed across the country. Along the way, labs like Luis's get involved. Amarsy: What Luis calls the “seed” of the vaccine is a specifically designed virus that pharma companies use to develop flu vaccines. It doesn't make us sick but instead helps our body create antibodies. It's also known as a candidate vaccine virus. Martinez-Sobrido: It could be coming from here, yeah. These so-called pandemic vaccine viruses are an essential line of defense against the threat of an avian flu pandemic. And whether we're talking about flu shots for seasonal or avian strains, making them usually involves a surprising tool. Ahmed Elsayed of the Texas Biomedical Research Institute picks-up fertilized chicken eggs from a crate before inspecting them for infection with an an H5N1 vaccine virus. Amarsy: These aren't your everyday supermarket eggs. They are fertilized eggs, produced in secret biosecure farms across the country. It's believed that we go through millions of these eggs every year. They are used because flu viruses grow very well in the allantoic cavity, which is full of a liquid that contains waste from the embryo as well as various proteins. Luis and his team showed me how they make H5N1 vaccine seeds. They said it all starts in their super-secure BSL-3 lab that only authorized researchers can enter. Amarsy: Once the scientists create the virus they conduct a bunch of tests to make sure it actually isn't dangerous to humans. Only when they know the virus is safe do they move it to another lab with less stringent safety rules—no need for respirators—and this time I could join. Martinez-Sobrido: We have egg incubators like these where we keep the eggs inside, and they actually rotate, like, every 15 minutes, 20 minutes. So now we will start to candle the egg. Amarsy: He holds an egg in one hand. He moves the egg below a small light attached to his workstation, causing it to glow orange—except for a small circular patch at the top, which remains white. He wants to avoid injecting the virus there because it won't grow. Ahmed Elsayed of the Texas Biomedical Research Institute uses a laboratory light to observe the inner contents of a fertilized chicken egg. Elsayed: Then we find [a] certain point in which we can use to inoculate safely the embryonated egg without affecting the embryo. Amarsy: After labeling a dozen eggs Ahmed turns the lights back on. He passes the crate to Ramya Smithaveni Barre, a Ph.D. student who also works in Luis's lab. Amarsy: Under a loud biosafety hood she pokes small holes into the eggs and injects them with the specially designed vaccine virus. These are very thick, so it just helps drill a hole quickly. So you see this point here, which—where the X mark is made? So from here we will be injecting using a syringe. [CLIP: Barre opens a syringe packet and injects virus into an egg.] Each has enough virus to make thousands of flu vaccine doses. After leaving Luis's lab the vials go through a series of quality control evaluations and testing. And then they'll be sent to manufacturing labs, which, in the case of a pandemic, can use them to replicate the process in millions of eggs. And without the science that has—even if most people don't know it—gone into every flu shot you've ever had, we might not be able to prepare for what scientists say is the growing threat of a potential bird flu pandemic. That these vaccines have clearly saved millions of lives. But, clearly [this] is the best mechanism that we have to protect us against any type of infectious disease, including influenza. Amarsy: And if H5N1 does become easily transmissible between humans, the seeds that Luis's lab makes and the H5N1 vaccines that follow, could become central to our pandemic response. Amesh Adalja: So there are stockpiled vaccines against H5N1. This is a process that began during George W. Bush's administration. Luis Martinez-Sobrido of the Texas Biomedical Research Institute holds up a test to detect the presence of an egg-grown H5N1 vaccine virus in diluted samples of allantoic fluid from inside fertilized chicken eggs. In the event of a threat change you would anticipate manufacturers coming online and some of the seasonal flu capacity being shifted to pandemic flu requirements. He's a pandemic-preparedness specialist at Johns Hopkins University and an infectious disease physician. But he sees some challenges in our current system. And how much of it do we have, and how fast can we have it in the arms of those people who are at most risk? However, people recognize that, and there have been specific flocks that have been segregated away and kept under high biosafety to not allow them to be infected. They're not kept in [the] open, where a passing goose can't put its droppings in the chicken cage. The larger concern is that creating vaccines by using chicken eggs takes so long that scientists have to pick the strains about six months before a vaccine gets into our arms. So that's why we sometimes have vaccine mismatches: because of that long lead time required by the egg-based vaccine manufacturers. Amarsy: This means that if the H5N1 viruses currently circulating were to mutate into a new strain that doesn't respond to the vaccine seeds made in labs like Luis's, it would be at least half a year before we even had egg-based shots available. By that time many people could already be infected. Additionally, Amesh says there's another problem with growing bird flu vaccines in eggs. Adalja: When you propagate the virus in chicken eggs, the virus mutates and you might end up at the end with something different than what you started with that might not work as well, so I think it also decreases the efficacy. So, long lead time, which allows mismatches to occur more frequently, and you get egg-based mutations that decrease the efficacy of the vaccine. Amarsy: As mentioned before we still use eggs to make most flu shots, and most of the time it works well. It's the cheapest option, and we have a widespread manufacturing infrastructure built around this process. And though these approaches cost more right now, experts like Amesh have advocated for them to be adopted more widely. Some labs are trying to develop even newer solutions based on messenger RNA, like many of the COVID vaccines. The Trump administration, however, recently canceled $766 million in funding for the pharmaceutical company Moderna to develop an mRNA-based bird flu shot. And that, we think, hasn't happened yet. And how [is] the virus ... able to—these eight genes [able to] code information that take over the whole 30,000 genes and then change completely the cell to make copies of themselves? How can you achieve that with so little information, right? He now runs a lab at the Mount Sinai Icahn School of Medicine in New York City. His lab looks at pretty much everything that has to do with flu—vaccination, treatments, viral evolution, transmission, mutations—but even he was surprised by what H5N1 has done. García-Sastre: I could never imagine in my life that there will be cows infected with flu.... Before that they say, “You think that flu can replicate and establish a cycle in dairy cows by replicating in the mammary glands of the dairy cows?” I say, “What, are you crazy or what?” Amarsy: As we heard my colleague Meghan Bartels explain in Episode Two, scientists were astonished when they learned that H5N1 had jumped from birds to cows. One way that could happen is through reassortment, the genetic mixing of multiple influenza viruses that we learned about in episode one. García-Sastre: So if we get, for example, the 2009 H1N1 pandemic of flu, ... it [had acquired] gene chromosomes coming from four different viruses: one was a virus circulating in humans, the other was a virus circulating in birds, the other virus circulating in pigs and the other virus circulating in pigs but [in] a different geographical area. So somehow these viruses got together into a pig and then created this particular virus. Mutations happen when viruses make copies of themselves and mistakes slip in. García-Sastre: But it can happen that if a new mutation gives you a new characteristic that makes you more likely to replicate faster than your previous brothers and [sisters], then this mutation dominates. Now, let's say H5N1, if it requires 20 different mutations to replicate and transmit in humans and if each mutation by itself doesn't make it better, that is very rare because these 20 mutations, it's very difficult [for them to] happen at the same time. But if it requires only four mutations, that's a different story. Maybe you can get four mutations being [generated] at one moment and it just [happens] to be that this—there is a human that gets infected with this mutant virus; then it starts to propagate in humans. Amarsy: There's an ongoing debate among virologists about how many mutations it would take for the H5N1 viruses we are currently dealing with to better adapt to humans. One study found that for the strain that has been circulating in dairy cows since last year, one single mutation could potentially do the job. García-Sastre: It's very unlikely because I think the number of encounters of this virus with humans has been so many that if it [would] be only one mutation required for [transmissibility] in humans, it would have happened already. But there's one notable exception: experts believe that in 1918 an H1N1 virus thought to have avian origins mutated in a way that made it better adapted to humans. Eventually, the virus caused one of the deadliest pandemics in recorded history, with estimates suggesting it killed anywhere from 50 to 100 million people worldwide. We realized that the virus was a very nasty virus to start with, and many of the deaths happened because that was an extreme case of a very virulent virus for humans of influenza. When we started to try to understand what are the determinants of virulence of this virus, we found that it needed to have a perfect storm—a combination of multiple mutations happening in multiple genes of the virus—and only this combination makes the virus as lethal as it was. Virologists tend to agree that another flu pandemic is inevitable. But there's disagreement about how likely H5N1 is to be the trigger, so I asked Adolfo's longtime colleagues what they thought. Peter Palese: Some people think that it's just a question of time when any of these H5 ... they are bound to, to jump into humans. I turn it around and say, “They have been with us so long already, in all kinds of animals—avian, mammalian—why hasn't it happened?” And so I'm not so sure it will happen, right? And I'm not so sure that this is the next pandemic strain, which will cause us all the grief, which we have seen with other pandemic strains. Amarsy: That's Peter Palese, Adolfo's mentor at Mount Sinai, and a leading figure behind a lot of innovations in influenza research. Florian Krammer: I think it's really hard to predict. I think there is a high chance right now, or higher than before, just because there is so much virus out there and because it seems to adapt to mammals better. From left: Virologists Peter Palese, Florian Krammer, and Adolfo García-Sastre, all of the Mount Sinai Icahn School of Medicine in New York City. García-Sastre: What is really clear is that there's gonna be another flu pandemic—doesn't need to be H5, but there have been, all the time, pandemics. Amarsy: It's not a foregone conclusion that H5N1 will trigger the next pandemic. But if it does, are we ready to respond effectively? I talked to epidemiologists Jennifer Nuzzo and Shira Doron. Jennifer Nuzzo: So historically, H5N1 has been observed to be one of the most deadly viruses we've seen. Meaning that of all the cases we've been able to find, about half of them have died, and that is truly extraordinary when you sort of rank pathogens in terms of their potential to kill the people that we know are infected. She directs the Pandemic Center at Brown University. What we've seen with this particular strain is very mild illness, and so often what we're seeing with the individuals who have developed H5N1 influenza from cows and poultry is just conjunctivitis, or just conjunctivitis with some mild upper respiratory symptoms, like a sore throat. She's the chief infection control officer at Tufts Medicine. And though most cases have been mild we made be undercounting them—by a lot. Doron: We know that our known cases are a relatively small proportion of total actual cases, especially because the disease has been so mild in most of the farmworkers who have become infected. Add to that the fact that many of these workers are undocumented migrants who are trying to stay under the radar. Amarsy: If many cases go undetected, it means that H5N1 could be spreading silently. Nuzzo: I think it's quite concerning that we continue to see new outbreaks on farms being reported and yet no new human cases have been identified in months. So the number of animals who are getting infected continues to climb, and somehow the number of people who are being infected has just remained unchanged. So we have a lot of reason to be concerned that we haven't found new cases because there's been a contraction in the surveillance efforts directed at H5N1. There may be lower H5N1 activity; I'm not ruling that out. But we also know that H5N1 is not going away. Nuzzo: We know just telling people to wear personal protective equipment to protect themselves against the virus every time they're around animals is not working because people have continued to get sick despite making those recommendations. Amarsy: Otherwise, we could be caught off guard. And if H5N1 became able to infect people easily and able to transmit easily between people, thus triggering a pandemic, and anything close to the case fatality that we've seen it have, it would be far and away so much worse than anything we've ever observed with COVID-19. Amarsy: Of course, that's a scenario nobody wants. As we learned from Luis we can make vaccines that are thought to be effective, at least against the current circulating strains of H5N1. Additionally, we have antiviral treatments, some of which researchers such as Adolfo and Peter have tested against the nastiest flu viruses, including viruses with genes from the 1918 strain, and they tend to work well against many different flu viruses. Diluted samples of an inactivated egg-grown H5N1 vaccine virus in petri dishes at the Texas Biomedical Research Institute in San Antonio, Tex. So what seems to concern epidemiologists such as Jennifer and Shira the most is not whether we have the right treatments and prevention mechanisms available to fight bird flu. Instead, the question is whether we have enough resources right now to handle another pandemic. Doron: We have tested a number of strategies and figured out how to do some really hard things, and those are triggers we could pull again if need be. So for example, in my hospital, we now know how to set up, very quickly, a mass testing site, a mass vaccination site. We know how to expand intensive care and expand patient care, so in some ways we're more prepared today because of COVID-19. Amarsy: But in other ways we are less prepared, the experts say. During the start of COVID-19 one of the things that states tried to do was contact tracing, but it was really hard because states didn't have the kind of personnel. Now not only do we not have that workforce anymore, but we've also lost a lot of public health leaders. You know, it's like we had an enormous fire rip through the U.S. and we decided to systematically dismantle all of the fire departments. So I am deeply worried about how the U.S. would fare in another pandemic. Amarsy: Jennifer says that sooner or later there will be another flu pandemic. Whether it will be caused by H5N1 or another bird flu virus, we just don't know. We hope you've enjoyed this week's special series on bird flu. We'll be back with something new on Monday. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper, Naeem Amarsy and Jeff DelViscio. Shayna Posses and Aaron Shattuck fact-check our show. Our theme music was composed by Dominic Smith. Subscribe to Scientific American for more up-to-date and in-depth science news. Naeem Amarsy is a documentary filmmaker and multimedia editor based in New York City. Jeffery DelViscio is currently chief multimedia editor/executive producer at Scientific American. Before that he spent more than eight years at the New York Times, where he worked on five different desks across the paper. He was a Knight Science Journalism Fellow at the Massachusetts Institute of Technology in 2018. Fonda Mwangi is a multimedia editor at Scientific American. She previously worked as an audio producer at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-61008-5'>The microbiota vault initiative: safeguarding Earth's microbial heritage for future generations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 09:58:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Microbial ecosystems are fundamental to planetary and human health, yet human activities are accelerating their loss. Disruptions to microbial communities undermine environmental stability, biodiversity, and health. Urgent action is required to preserve microbial diversity. The Microbiota Vault Initiative provides a global framework to safeguard microbiomes from human, animal, and environmental sources. It proactively archives microbial diversity for future needs, prioritizing depositor sovereignty, equitable collaboration, and ethical governance. By sharing limited information on deposits, the initiative fosters microbial conservation and collaboration between local and global researchers. It complements other efforts to ensure the resilience of microbiomes in an era of rapid environmental change. Microbial ecosystems underpin life on Earth, yet they are increasingly imperiled by human activities, including unsustainable agricultural practices and overuse of antibiotics. These disturbances disrupt microbial networks essential for environmental stability, biodiversity, and animal and human health, contributing to the global rise of chronic diseases and ecosystem degradation. As future restoration efforts will depend on the microbial diversity preserved today, proactive measures are urgently required. In this Perspective, we focus on one such initiative, the Microbiota Vault Initiative (MVI), a not-for-profit non-governmental organization that we have developed to address the urgent need. Inspired by the Svalbard Global Seed Vault (SGSV), the MVI shares a vision of safeguarding diversity, although its goals and implementation differ substantially. Unlike the SGSV, which provides backup for existing seed banks with well-documented restoration outcomes, the MVI proactively archives microbiota across human, animal, and environmental domains in anticipation of future unknown needs in restoration efforts. The MVI embodies an integrative One Health approach by safeguarding microbial biodiversity critical to human and animal health, plant resilience, and environmental functions such as soil fertility, nutrient cycling, climate regulation, and pathogen resistance. By fostering equitable collaboration, honoring diverse knowledge systems, and ensuring depositor sovereignty, the MVI provides a culturally attuned framework for mitigating microbial loss (Table 1). Coordinated action through this initiative offers a timely opportunity to protect health, biodiversity, and ecosystem resilience through the lens of One Health, underscoring the deep interdependence of microbial diversity, environmental integrity, and well-being across species. There is broad recognition of the essential role that naturally evolved microbes play in ecosystems, including in humans1,2,3,4. Human activities have disrupted these evolutionary connections in host-associated microbiomes [humans1,2,3,4, animals5,6,7, and plants8,9,10] and have been associated with an alarming rise in chronic diseases such as allergic, autoimmune, and metabolic disorders4,11,12,13,14,15. The loss of crucial microbes in both human and environmental contexts represent a growing but often overlooked crisis. In human health, several key examples illustrate how modernization disrupts the coevolved microbial partnerships19,20. The decline in prevalence of Bifidobacterium longum subsp. infantis—a bacterium that evolved to digest human milk oligosaccharides and support infant immune development—in industrialized societies has been associated with increased risks of allergies, diabetes, and other immune conditions21,22. This decline, likely due to increased use of Cesarean sections, formula feeding, and antibiotics, has become so severe that some pediatricians now recommend probiotic supplementation23. Similarly, the decrease of Helicobacter pylori and Treponema succinifaciens24 from gut microbiomes in industrialized countries shows broader implications of urbanization. While H. pylori's reduction has decreased stomach ulcers, its absence correlates with increased rates of allergic asthma and metabolic disorders20, highlighting the complex roles that these co-evolved microbes have in human health. In environmental systems, the thawing of Arctic permafrost threatens unique cold-adapted microorganisms like Methanoflorens stordalenmirensis, which helps regulate methane emissions25, and various Acidobacteria species crucial for carbon cycling. The loss of these environmental microbes creates a dangerous feedback loop: as climate change kills off permafrost microbes, their absence accelerates the release of greenhouse gases, further warming the planet26,27. These parallel losses in both human and environmental microbiomes demonstrate how changes caused by human activity can disrupt microbial ecosystems that evolved over millions of years, with cascading effects on both human and planetary health. Just as we strive to preserve endangered macro-organisms, urgent global efforts are needed to understand and preserve microbial diversity. While much microbial biodiversity and mechanisms underlying sustaining natural ecosystems and health remain to be discovered, viewed through the lens of One Health, it is our obligation to future generations to preserve this microbial diversity28. The concept of preserving Earth's microbial heritage draws parallels to seed banks, which safeguard plant genetic diversity. Its primary aim is to support local collections to preserve microbial biodiversity globally, including environmental, plant, animal, and human-associated microbiota. The goal of the MVI is to foster an inclusive, global effort to preserve microbial specimens while fostering extramural research. It is intentionally non-commercial and based on an equitable framework that is designed to address the urgent need for microbial preservation amidst global biodiversity loss. Inspired by the Svalbard Global Seed Vault (www.croptrust.org), the initiative prioritizes microbial biodiversity preservation, fostering the creation of local collections and offering secure cryogenic storage of microbiomes from diverse human and environmental sources (Fig. The Microbiota Vault educates scientists and promotes the creation of Local Collections for preservation and research. These local collections can request that the Microbiota Vault preserve a safe backup of their specimens, maintaining all rights over their deposits; they have the option to request sequencing for open access under the license of their choice, and the MV will publish on its webpage a database of the deposited specimens, including metadata, links to sequences, and contact information about depositors. The MVI acts at the request and on behalf of sovereign depositors, and worldwide local collections, to preserve backup copies of specimens of human, animal, plant, or environmental origin. These samples are not accessible to anyone other than the depositors, or their registered agent. Additionally, upon request by the depositing collections, the Microbiota Vault may sequence microbial DNA (or RNA), depositing in open-access databases under the Creative Commons license of their choice (https://creativecommons.org/licenses/). This helps depositors define the terms of their collaborations29 and foster global research connections, leading to global capacity strengthening in microbiome research. The MVI does not engage in discovery research, therapeutic development, patenting, profit generation, or the assertion of material or intellectual property rights over its collections. Instead, it focuses on building a collaborative, multidisciplinary network. Through initiatives such as the annual Global Microbiome Network Symposia (GloMiNe), the MVI brings together researchers, public health professionals, legal experts, anthropologists, sociologists, and other stakeholders to engage in dialogue around microbiome science, research ethics, standardization, and equitable access and benefit-sharing. The MVI is committed to educating and preserving microbial diversity while ensuring adequate representation of all people. The initiative fully supports multilateral efforts within the Global Biodiversity Framework towards equitable sharing of benefits from the use of biological materials or their data30,31. Its framework aims to support the development of equitable and inclusive approaches to sovereignty and governance in microbiome research. As a non-profit organization, the MVI operates under its bylaws and global policies to maximize inclusion, providing fiduciary oversight and educating prospective depositors on ethical research practices, access and benefit-sharing requirements under the Nagoya Protocol, and standardized methods for metadata design. These events bring together scientists from diverse disciplines—microbiology, anthropology, ethics, public health, and bioinformatics—to share knowledge, discuss microbial biodiversity conservation, and strengthen local research capacity. Over four years, these symposia have fostered interdisciplinary collaboration and contributed to the establishment of a global microbiome network (Table 2). In addition to symposia, the Microbiota Vault supports capacity strengthening through mentoring for scientists from low- and middle-income countries, who thanks to collaborating with the Digital Sequence Information Scientific Network (DSI) are offered travel grants to their meetings, further advancing global equity in microbiome research. As part of its global advocacy efforts, the MVI has engaged with United Nations delegations, coordinating educational screenings to highlight the importance of the microbiome, and presented at the UN COP24 conference in Cali, Colombia. The MVI has just concluded its Launch Phase, refining a model for the interaction between a globally coordinated fundraising initiative and locally operated biobanks. Backup collections are deposited with a contractual legal framework in which Deposit Agreements are signed between depositors and the Microbiota Vault. During this initial phase, Local Working Collections from Benin, Brazil, Ethiopia, Ghana, Laos, Thailand, and Switzerland deposited 1204 human fecal specimens and 190 fermented food samples. These deposits required navigating complex local and global regulations for exporting samples to Switzerland, where they are housed securely in cryo-storage (−80 °C) at the Institute of Medical Microbiology, University of Zurich. The MVI Launch Phase also enabled the development of standardized protocols for sample collection, preservation, transport, metadata development, and data annotation. For metadata, the Microbiota Vault has adopted minimum information about any (x) sequence (MIxS) standards, a modular framework developed by the international scientific community, which offers flexibility to accommodate additional sample types such as environmental or wildlife specimens33. Best practices emerging from this phase include robust traceability of samples, equitable biobanking frameworks, standardized metadata, harmonized storage protocols, advanced biobanking systems, and quality control adhering to the recommendations of the Swiss Biobanking Platform (https://swissbiobanking.ch/). The MVI's Launch Phase focused on human-associated microbiomes and fermented food microbes. To support broader ecological goals, it will expand to include environmental microbiota from threatened ecosystems. This aligns with a One Health framework by preserving microbes vital to environmental and planetary health, and by providing a framework for fostering research and strengthening local capacities. While reintroducing preserved microbes remains speculative, and functional restoration from frozen specimens is unproven, preservation offers a valuable opportunity for future research. Indeed, the application of “microbiome therapy” in humans34, animals35,36, and environmental ecosystems37,38, is promising. Stored samples may one day support fundamental microbiological research, ecosystem restoration, or biotechnological advances. Around the world, other groups of scientists have independently recognized the need to conserve microbial communities through specimen preservation. Among them, the Global Microbiome Conservancy stands out as a research-driven effort actively collecting and studying human microbial samples from diverse geographic regions. While most other initiatives are also research-oriented (Table 3), they align closely with the Microbiota Vault Initiative (MVI) and offer complementary strengths that create opportunities for future collaboration. By partnering with and receiving specimens from local microbial collections, the MVI and similar efforts help safeguard regional ecosystems while advancing education, capacity strengthening, and inclusive governance. In particular, the MVI empowers local stakeholders through depositor sovereignty and strict standards for metadata and preservation protocols. & Curtis, N. Association between the intestinal microbiota and allergic sensitization, eczema, and asthma: a systematic review. & Aarestrup, F. M. Avoparcin used as a growth promoter is associated with the occurrence of vancomycin-resistant Enterococcus faecium on Danish poultry and pig farms. & Varela, M. F. Presence of multidrug-resistant enteric bacteria in dairy farm topsoil. Antibiotics in early life alter the murine colonic microbiome and adiposity. Bulgarelli, D., Schlaeppi, K., Spaepen, S., Ver Loren van Themaat, E. & Schulze-Lefert, P. Structure and functions of the bacterial microbiota of plants. Qu, Q. et al. Rhizosphere microbiome assembly and its impact on plant growth. Associations between infant fungal and bacterial dysbiosis and childhood atopic wheeze in a nonindustrialized setting. Early infancy microbial and metabolic alterations affect risk of childhood asthma. Russell, S. L. et al. Perinatal antibiotic treatment affects murine microbiota, immune responses and allergic asthma. Early life antibiotic-driven changes in microbiota enhance susceptibility to allergic asthma. Effects of antibiotics on human microbiota and subsequent disease. Harnessing the microbiome to prevent global biodiversity loss. Fierer, N. et al. Reconstructing the microbial diversity and function of pre-agricultural tallgrass prairie soils in the United States. Soil microbiomes must be explicitly included in one health policy. Henrick, B. M. et al. Bifidobacteria-mediated immune system imprinting early in life. Huda, M. N. et al. Stool microbiota and vaccine responses of infants. Carter, M. M. et al. Ultra-deep sequencing of Hadza hunter-gatherers recovers vanishing gut microbes. Fu, Q. et al. Genome sequence of a 45,000-year-old modern human from western Siberia. Woodcroft, B. J. et al. Genome-centric view of carbon processing in thawing permafrost. Li, Y. et al. Genomic insights into redox-driven microbial processes for carbon decomposition in thawing Arctic soils and permafrost. Preserving microbial diversity: Microbiota from humans of all cultures are needed to ensure the health of future generations. Halewood, M., Bagley, M. A., Wyss, M. & Scholz, A. H. New benefit-sharing principles for digital sequence information. Scholz, A. H. et al. Multilateral benefit-sharing from digital sequence information will support both science and biodiversity conservation. Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications. Feng, H. et al. Fecal virus transplantation has more moderate effect than fecal microbiota transplantation on changing gut microbial structure in broiler chickens. Ma, Z. et al. Fecal microbiota transplantation improves chicken growth performance by balancing jejunal Th17/Treg cells. Sheth, R. U., Cabral, V., Chen, S. P. & Wang, H. H. Manipulating Bacterial Communities by in situ Microbiome Engineering. Duarte Rosado, J. G. et al. Coral probiotics induce tissue-specific and putative beneficial microbiome restructuring in a coral-dwelling fish. We thank Daniela Vargas-Robles and Joel Colombo for their work on the figure. Department of Biochemistry & Microbiology, Rutgers University, New Brunswick, NJ, USA Department of Anthropology, Rutgers University, New Brunswick, NJ, USA Humans and the Microbiome Program, Canadian Institute for Advanced Research, Toronto, ON, Canada Seerave Foundation, St Helier, Jersey, Channel Islands, UK Adrian Egli, Diana Albertos Torres, Yuen Yi Li & Vanni Benvenga Pascale Vonaesch, Abdifatah Muhummed & Youzheng Teo Department of Food Sciences and Experimental Nutrition, University of São Paulo, São Paulo, Brazil Ecole des Sciences et Techniques de Conservation et de Transformation des Produits Agricoles, Université Nationale d'Agriculture, Sakete, Benin Swiss Tropical and Public Health Institute, Allschwil, Switzerland Armauer Hansen Research Institute, Addis Ababa, Ethiopia Institute of Nutrition, Mahidol University, Nakhon Pathom, Thailand Department of Computer Science and Engineering, University of California, San Diego, La Jolla, CA, USA UCSD Center for Microbiome Innovation, University of California, San Diego, La Jolla, CA, USA Shu Chien-Gene Lay Department of Bioengineering, University of California, San Diego, La Jolla, CA, USA Halıcıoğlu Data Science Institute, University of California, San Diego, La Jolla, CA, USA Scripps Institution of Oceanography, San Diego, La Jolla, CA, USA Center for Advanced Biotechnology and Medicine, Rutgers University, Piscataway, NJ, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar contributed to infrastructure development and coordination of the Launch Phase. All authors approved the final version. R.K. is a co-founder of Biota, Inc., and Micronoma, holds equity; he receives income as a consultant or as a scientific advisory board member for BiomeSense, GenCirq, DayTwo, Cybele, Biota, and Micronoma. The University of California, San Diego has reviewed and approved these arrangements in accordance with its conflict of interest policies. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. The microbiota vault initiative: safeguarding Earth's microbial heritage for future generations. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Microbiology newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-60894-z'>Hepatitis B virus promotes liver cancer by modulating the immune response to environmental carcinogens</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 09:48:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Communications volume 16, Article number: 5360 (2025) Cite this article Hepatitis B virus (HBV) infection is associated with hepatitis and hepatocellular carcinoma (HCC). Considering that most HBV-infected individuals remain asymptomatic, the mechanism linking HBV to hepatitis and HCC remains uncertain. Herein, we demonstrate that HBV alone does not cause liver inflammation or cancer. Instead, HBV alters the chronic inflammation induced by chemical carcinogens to promote liver carcinogenesis. Long-term HBV genome expression in mouse liver increases liver inflammation and cancer propensity caused by a carcinogen, diethylnitrosamine (DEN). HBV plus DEN-activated interleukin-33 (IL-33)/regulatory T cell axis is required for liver carcinogenesis. Pitavastatin, an IL-33 inhibitor, suppresses HBV plus DEN-induced liver cancer. IL-33 is markedly elevated in HBV+ hepatitis patients, and pitavastatin use significantly correlates with reduced risk of hepatitis and its associated HCC in patients. Collectively, our findings reveal that environmental carcinogens are the link between HBV and HCC risk, creating a window of opportunity for cancer prevention in HBV carriers. Hepatitis B virus (HBV) is a hepatotropic DNA virus that can cause persistent infection in primates1,2,3. It is estimated that the prevalence of HBV infection is around 250 million cases worldwide and is associated with an increased risk of hepatitis and HCC4,5. HBV's high prevalence has led to the development of a vaccine using hepatitis B surface antigen (HBsAg)6,7. However, HBV remains a significant global health threat. Key contributing factors include limited access to the HBV vaccine in certain countries, noncompliance with multi-dose regimens, and the presence of vaccine non-responders, estimated at 16% of the population, which are more prevalent among older individuals, those with obesity, and smokers8,9,10,11. Importantly, these same risk factors have well-known carcinogenic effects and may be the primary drivers of hepatitis and HCC in HBV-infected liver12,13. As such, it is essential to determine the underlying mechanism linking HBV infection to hepatitis and HCC in order to effectively address this global challenge. HBV DNA can integrate into the host genome, which has been proposed as a mechanism for genomic mutations with oncogenic effects14. However, HBV is typically a non-cytopathic virus programmed for benign episomal replication3. Although 10-30% of HCC cases are associated with HBV infection, the HBV genome integration ratio in tumor cells is lower than in non-tumor tissues15,16. The other proposed mechanism linking HBV to HCC relates to HBV's ability to induce chronic inflammation in the liver17,18. HBV infection tightly interacts with the host immune system3,19,20,21. CD8+ T cell response to HBV infection, as a potential therapeutic target, can lead to viral clearance but may paradoxically contribute to disease pathogenesis during the early stages22,23. CD8+ T cells' attempt to eliminate infected hepatocytes and HBV-associated HCC cells can instead result in T cell exhaustion, resulting in chronic inflammation and the development of malignancy24,25. The impaired function of CD4+ T cells and natural killer cells is another significant consequence of HBV infection26,27. In addition, the elevated neutrophil/lymphocyte ratio in peripheral blood and increased number of Foxp3+ regulatory T cells (Tregs) are positively correlated with disease progression in patients with HBV-associated HCC4. However, there is limited inflammation in the ‘immune tolerant' and ‘immune inactive' stages of chronic HBV infection28,29,30. While immune imbalance induced by chronic HBV infection is considered the cause of HBV-related HCC development31, the factors that induce this immune imbalance are unclear. In fact, HBV infection often does not cause any overt liver pathology, and up to 60% of HBV-infected individuals remain asymptomatic32. The combination of HBV infection and environmental factors, such as alcohol and smoking, is associated with increased liver injury33,34,35. This suggests that HBV alone may not be sufficient to induce chronic inflammation or liver cancer. Although the incidence of HBV infection is decreasing owing to vaccination, HBV infection remains a serious health problem globally10,36. Thus, it is essential to determine the full biological impact of HBV infection on the liver to uncover suitable therapeutic agents for hepatitis and HCC prevention in HBV-infected individuals. Herein, we investigated the mechanism linking HBV to chronic hepatitis and HCC development. Importantly, mice chronically inoculated with the HBV genome did not develop liver inflammation or cancer. Instead, HBV modulated the liver response to a tobacco smoke carcinogen, diethylnitrosamine (DEN)37,38, resulting in severe liver cancer phenotype. HBV-containing liver exposed to DEN upregulated interleukin-33 (IL-33), which was required for liver cancer development. Loss of IL-33 markedly reduced IL-33 receptor (IL-1 receptor-like 1 (IL1RL1) or ST2) expressing Tregs in HBV+DEN-treated liver. Accordingly, ST2 deletion on Tregs blocked liver cancer development in HBV+DEN-treated mice accompanied by increased cytotoxic CD8+ T cells in the liver. Treatment with pitavastatin, an inhibitor of IL-33 expression39, resulted in reduced risk of chronic hepatitis and liver cancer in mice and humans. Our findings indicate that HBV-infected individuals can be protected from hepatitis and HCC by minimizing their risk of exposure to chemical carcinogens. Furthermore, suppressing IL-33/Treg axis in the liver using statins represents a safe and effective strategy for the prevention and treatment of chronic hepatitis and its cancer sequelae. To establish an HBV infection-like model in mice, we employed a pAAV-HBV1.2 vector to deliver the HBV genome to hepatocytes via hydrodynamic tail vein injection40,41. This method resulted in persistent HBV gene expression in the liver, as evidenced by detectable HBsAg in the circulation for 6 months post-infection (Supplementary Fig. Importantly, HBV genome expression in 28-day-old wild-type (WT) mice did not cause any liver cancer over 12 months post-infection (Fig. In contrast, HBV-expressing WT mice that received a carcinogen, DEN, (HBV+DEN) showed a significantly higher liver tumor burden, reduced survival, and increased PCNA+ proliferating hepatocytes compared with the Sham+DEN group (Fig. Importantly, HBV DNA and gene expression persisted in HBV+DEN and HBV+phosphate-buffered saline (PBS)-treated WT livers up to 8 months post-infection (Supplementary Fig. Thus, the combination of HBV and carcinogen accelerates liver cancer development, whereas HBV alone does not cause hepatocyte proliferation or cancer. a The schematic diagram of liver carcinogenesis protocol. b Representative macroscopic images of the liver from WT mice that underwent liver carcinogenesis protocol at 4, 6, 8, and 12 months (M) post-infection. Arrows point to liver tumors. c Tumor burden measured as % liver surface area of WT mice that underwent liver carcinogenesis protocol at 8 months post-infection. n = 7 mice in HBV+DEN group, n = 8 mice in Sham+DEN group, n = 7 mice in HBV+PBS group, and n = 7 mice in Sham+PBS group. Each dot represents a mouse. Experimental data were verified in two independent experiments. d Survival of WT mice underwent liver carcinogenesis protocol. n = 12 mice in HBV+DEN group; n = 10 mice in Sham+DEN group, n = 16 mice in HBV+PBS group, and n = 10 mice in Sham+PBS group. e Representative images of PCNA stained liver from WT mice that underwent liver carcinogenesis protocol at 8 months post-infection. f Quantification of PCNA+ hepatocytes in WT mice subjected to liver carcinogenesis protocol at 8 months post-infection. PCNA+ cells per 100 hepatocytes were counted in five to eight randomly selected high power field (HPF) images of each liver. Each dot represents an HPF image. Liver samples were from n = 6 mice in HBV+DEN group, n = 7 mice in Sham+DEN group, n = 7 mice in HBV+PBS group, and n = 4 mice in Sham+PBS group. Graphs show mean + SD, (c, f) one-way ANOVA with Tukey's multiple comparison test, scale bars: 1 cm (liver images) and 100 μm (histology). Source data are provided as a Source Data file. To explore the link between chronic hepatitis and liver cancer development in HBV-positive mice, we examined the immune environment of the liver at 4 months post-infection prior to any cancer development (Fig. HBV genome expression alone did not cause any liver damage measured by serum alanine aminotransferase (ALT) levels or liver inflammation assessed by CD45+ leukocyte counts in the liver (Fig. 2a, b, and Supplementary Fig. However, CD45+ leukocyte counts were significantly increased in the HBV+DEN group compared with the Sham+DEN, HBV+PBS, and Sham+PBS groups (Fig. Additionally, HBV+DEN-treated mice exhibited elevated serum ALT levels compared with HBV+PBS and Sham+PBS-treated mice (Supplementary Fig. To investigate the underlying mechanism leading to more severe chronic liver inflammation in HBV+DEN compared with Sham+DEN-treated mice, we performed whole liver RNA sequencing at 4 months post-infection. Gene Set Enrichment Analysis (GSEA) revealed that the HBV+DEN-treated liver upregulated hepatitis-related genes compared with Sham+DEN group (Fig. Further analysis identified six differentially expressed cytokines: Il33, Il1b, Il4, Il16, Il7, and Il18 (Fig. Previous studies have linked IL-33 to cancer-prone chronic inflammation39,42,43, prompting us to explore its role in HBV+DEN-associated liver cancer development. HBV+DEN-treated mice exhibited higher IL-33 expression in their liver compared with Sham+DEN-treated mice, which was expressed by E-cadherin+ hepatocytes and α-SMA+ hepatic stellate cells (Fig. By contrast, IL-1β, IL-4, and IL-16 protein levels were not increased in HBV+DEN- compared with Sham+DEN-treated WT liver (Supplementary Fig. We have demonstrated that the Toll-like receptor (TLR) 3-4/TBK1/IRF3 pathway drives IL-33 expression in chronic inflammation39. Accordingly, activated TBK1 (phospho-TBK1) and IRF3 (phospho-IRF3) were induced in HBV+DEN-treated liver compared with the other treatment groups (Supplementary Fig. DEN induces immunogenic cell death in hepatocytes44,45. Interestingly, HBV+DEN treatment led to significantly greater DNA damage, as indicated by γH2AX+ cells in the liver, compared with Sham+DEN at 4 months post-infection (Supplementary Fig. Likewise, HBV+DEN-treated liver showed elevated high-mobility group box 1 (HMGB1) expression, a well-known damage-associated molecular pattern (DAMP) molecule and TLR4 agonist46,47,48, in hepatocytes compared with Sham+DEN-treated liver at 4 months post-infection (Fig. 2h, i and Supplementary Fig. HBV genome expression alone did not impact HMGB1 expression in the liver (Supplementary Fig. Serum HMGB1 levels were also elevated in HBV+DEN compared with HBV+PBS and Sham+PBS groups (Supplementary Fig. 3f), indicating that increased HMGB1 was secreted from the nucleus to function as a DAMP molecule. These findings indicate that the HMGB1/TLR4/TBK1/IRF3 axis activates IL-33 expression in HBV-positive liver that is exposed to carcinogens, which may lead to hepatitis and cancer development. a Representative images of CD45 stained liver tissues treated with carcinogenesis protocol at 4 months post-infection. b Quantification of CD45+ leukocytes in WT liver subjected to carcinogenesis protocol at 4 months post-infection. Each dot represents the average number of CD45+ cells per 10 randomly selected HPF images from each liver. Liver samples were from n = 8 mice in HBV+DEN group, n = 8 mice in Sham+DEN group, n = 12 mice in HBV+PBS group, and n = 10 mice in Sham+PBS group. c The enrichment plot of hepatitis gene set from differentially expressed gene list of HBV+DEN-treated liver compared with Sham+DEN-treated liver at 4 months post-infection. n = 3 mice in each group, Kolmogorov-Smirnov test. d Heatmap of differentially expressed interleukin genes between HBV+DEN- and Sham+DEN-treated liver at 4 months post-infection. e IL-33 protein levels in HBV+DEN- (n = 10 mice) versus Sham+DEN-treated liver (n = 9 mice) at 4 months post-infection. Each dot represents a mouse. f Representative images of IL-33 stained HBV+DEN- and Sham+DEN-treated liver at 4 months post-infection. g Quantification of IL-33high cells per 100 cells in the HBV+DEN (n = 8 mice) compared with Sham+DEN-treated liver (n = 6 mice) at 4 months post-infection. IL-33high cells per 100 cells were counted in five to eight randomly selected HPF images per liver. Each dot represents an HPF image. h Representative images of HMGB1 stained HBV+DEN- and Sham+DEN-treated liver at 4 months post-infection. i Quantification of HMGB1+ cells per 100 hepatocytes in the HBV+DEN (n = 5 mice) compared with Sham+DEN-treated liver (n = 7 mice) at 4 months post-infection. HMGB1+ cells per 100 hepatocytes were counted in five to eight randomly selected HPF images per liver. Each dot represents an HPF image. Graphs show mean + SD, (b) one-way ANOVA with Tukey's multiple comparison test, (e, g, i) two-sided unpaired t-test, scale bar: 100 μm. Source data are provided as a Source Data file. To investigate the role of IL-33 in HBV+DEN-induced liver cancer, we subjected Il33 knockout (Il33KO), ST2 knockout (ST2KO), and Il33 transcription factor, Irf3, knockout (Irf3KO) mice to HBV+DEN liver carcinogenesis protocol (Fig. HBV+DEN-treated Il33KO, ST2KO, and Irf3KO mice showed significantly lower liver tumor burden and decreased PCNA+ proliferating hepatocytes compared with HBV+DEN-treated WT mice at 8 months post-infection (Fig. The comparable reduction in liver tumor burden observed in HBV+DEN-treated Il33KO and ST2KO mice demonstrated that IL-33's cytokine function, rather than its nuclear function42, was responsible for HBV+DEN-mediated liver cancer development. Thus, we next examined which ST2-expressing immune cell type(s) is most suppressed in Il33KO compared with WT liver treated with HBV+DEN49. ST2+ Tregs exhibited the most significant reduction compared with other immune cells in HBV+DEN-treated Il33KO versus WT liver at 4 months post-infection (Fig. Tregs create a tumor-promoting environment by releasing transforming growth factor beta (TGF-β) and IL-1050,51. IL-2, a critical cytokine for maintaining Tregs52,53,54,55, increased hepatic Tregs proliferation in combination with IL-33 (Fig. Interestingly, IL-33 addition to IL-2 significantly upregulated TGF-β1 and IL-10 expression by ST2+ hepatic Tregs compared with IL-2 alone (Fig. Similarly, IL-33 addition to IL-2 significantly upregulated TGF-β1 and IL-10 expression by ST2+ splenic Tregs (Supplementary Fig. Interestingly, we detected ST2+ Tregs in HBV-associated chronic hepatitis in humans, which were increased compared with healthy control livers (Supplementary Fig. We found that ST2+ Tregs were increased in HBV-associated immune active hepatitis compared with chronic resolved hepatitis and healthy controls (Supplementary Fig. These findings demonstrate that IL-33 supports ST2+ Tregs in HBV+DEN-treated liver, which upregulate TGF-β and IL-10 expression in response to IL-33. a Representative macroscopic images of Il33KO, ST2KO, Irf3KO, and WT liver treated with HBV+DEN at 8 months post-infection. Arrows point to liver tumors. b Tumor burden measured as % liver surface area of Il33KO (n = 7), ST2KO (n = 5), Irf3KO (n = 6), and WT mice (n = 7) that received HBV+DEN at 8 months post-infection. Each dot represents a mouse. Experimental data were verified in two independent experiments. c Representative images of PCNA stained Il33KO, ST2KO, Irf3KO, and WT liver treated with HBV+DEN at 8 months post-infection. d Quantification of PCNA+ hepatocytes in Il33KO (n = 4 mice), ST2KO (n = 7 mice), Irf3KO (n = 6 mice), and WT mice (n = 5 mice) that received HBV+DEN at 8 months post-infection. PCNA+ cells per 100 hepatocytes were counted in five to eight randomly selected HPF images of each liver. Each dot represents an HPF image. e Percent ST2+ immune cell types in Il33KO (n = 6) and WT liver (n = 4) treated with HBV+DEN at 4 months post-infection. Each dot represents a mouse. f Hepatic Treg frequency as % total CD4+ T cells in WT liver after incubation with IL-2 plus IL-33, IL-2, versus no cytokine treatment (PBS) control (n = 6). Each dot represents a mouse. g TGF-β1 mean fluorescence intensity (MFI) of IL-2 plus IL-33, IL-2, versus PBS-treated ST2+ Tregs (n = 6). Each dot represents a mouse. h IL-10 MFI of IL-2 plus IL-33, IL-2, versus PBS-treated ST2+ Tregs (n = 6). Each dot represents a mouse. Graphs show mean + SD, (b, d, f–h) one-way ANOVA with Tukey's multiple comparison test, (e) two-sided unpaired t-test, scale bars: 1 cm (liver images) and 100 μm (histology). Source data are provided as a Source Data file. To determine whether IL-33-activated Tregs contributed to cancer development in HBV+DEN-treated mice, we subjected Foxp3Cre, ST2 flox/flox (TregST2CKO) mice in which ST2 was specifically deleted in Tregs to HBV+DEN liver carcinogenesis protocol (Fig. HBV+DEN-treated TregST2CKO mice showed significantly lower liver tumor burden, improved liver architecture, and decreased PCNA+ proliferating hepatocytes compared with HBV+DEN-treated WT mice at 8 months post-infection (Fig. A significantly fewer Tregs expressed IL-10 in TregST2CKO compared with WT liver (Fig. Accordingly, loss of ST2 expression on Tregs compromised their immunosuppressive function, leading to increased CD3+ T and CD8+ T cell infiltration in the HBV+DEN-treated liver at 4 months post-infection (Fig. In addition, activated CD44+CD8+ T cells were significantly increased in HBV+DEN-treated Treg-ST2CKO compared with WT liver (Supplementary Fig. Thus, the IL-33/Treg axis plays a critical role in mediating cancer development in the liver exposed to HBV plus carcinogens. a Representative macroscopic image of TregST2CKO and WT liver treated with HBV+DEN at 8 months post-infection. The arrow points to a liver tumor. b Tumor burden measured as % liver surface area of TregST2CKO (n = 12) and WT (n = 6) mice that received HBV+DEN at 8 months post-infection. Each dot represents a mouse. Experimental data were verified in two independent experiments. c Representative images of hematoxylin and eosin (H&E) stained TregST2CKO and WT liver treated with HBV+DEN at 8 months post-infection. d Representative images of PCNA stained TregST2CKO and WT liver treated with HBV+DEN at 8 months post-infection. e Quantification of PCNA+ hepatocytes in TregST2CKO (n = 8) and WT (n = 5) mice that received HBV+DEN at 8 months post-infection. PCNA+ cells per 100 hepatocytes were counted in five to eight randomly selected HPF images of each liver. Each dot represents an HPF image. f IL-10+ Treg frequency in TregST2CKO (n = 7) and WT (n = 7) liver treated with HBV+DEN at 4 months post-infection. Each dot represents a mouse. g Representative images of CD3 and CD8 stained TregST2CKO and WT liver treated with HBV+DEN at 4 months post-infection. Insets highlight T cells in the liver. h Quantification of CD3+ T cells in TregST2CKO (n = 4) and WT (n = 3) liver treated with HBV+DEN at 4 months post-infection. CD3+ cells were counted in three to four randomly selected HPF images per liver sample. Each dot represents an HPF image. i Quantification of CD8+ T cells in TregST2CKO (n = 4) and WT (n = 3) liver treated with HBV+DEN at 4 months post-infection. CD8+CD3+ cells were counted in three to four randomly selected HPF images per liver sample. Each dot represents an HPF image. Graphs show mean + SD, two-sided unpaired t-test, scale bars: 1 cm (liver images) and 100 μm (histology). Source data are provided as a Source Data file. Pitavastatin is a safe and widely used lipid-lowering medication that suppresses the TBK1-IRF3-IL-33 signaling pathway39. To determine the role of pitavastatin for liver cancer prevention in HBV-infected individuals, we treated WT mice with 2 mg/kg pitavastatin or PBS alone (carrier control) once a week during the HBV+DEN liver carcinogenesis protocol until harvest, which suppressed IL-33 levels (Supplementary Fig. Importantly, pitavastatin treatment significantly lowered liver tumor burden, improved liver architecture, and decreased PCNA+ proliferating hepatocytes compared with carrier control in HBV+DEN-treated WT mice at 8 months post-infection (Fig. To extend our findings to humans, we measured serum IL-33 levels in hepatitis patients and healthy individuals. HBV+ and HBV- hepatitis patients showed similarly elevated serum ALT levels compared with healthy controls (Supplementary Fig. HBV-associated hepatitis patients showed significantly higher serum IL-33 levels compared with healthy controls, which were also higher than non-HBV hepatitis patients (Fig. Interestingly, serum IL-33 levels were higher in HBV+ versus HBV- patients with other liver diseases, including HCC, cirrhosis, hepatic cyst, fatty liver, and liver hemangioma (Supplementary Fig. Moreover, serum IL-33 levels positively correlated with serum HBsAg levels across all HBV+ patients in the study (Supplementary Fig. Based on these findings, we performed an epidemiological study to determine whether pitavastatin use was associated with a reduced risk of hepatitis and HCC. We compared matched cohorts of patients from the TriNetX diamond network, a global health network comprising electronic medical record-derived data from over 200 million patients across 92 healthcare organizations in North America and Europe (Supplementary Table 1). The risk of hepatitis was significantly reduced in patients treated with pitavastatin compared with those treated with ezetimibe, another cholesterol-lowering agent commonly used in the clinic that doesn't affect the mevalonate pathway (OR 0.727; 95% CI 0.685-0.771; p < 0.0001) (Fig. Furthermore, the risk of HCC associated with hepatitis was markedly decreased in the pitavastatin-treated group compared with the ezetimibe cohort (OR 0.577; 95% CI 0.506-0.657; p < 0.0001) (Fig. Lastly, IL33 and IL1RL1 (ST2) gene expression showed a negative correlation with CD8+ T cell and a positive correlation with Treg infiltration in human HCC samples from the Cancer Genome Atlas (TCGA) datasets (Fig. Collectively, HBV infection link to hepatitis and HCC is mediated by its impact on the immune response to environmental carcinogens, which are the primary drivers of hepatitis and HCC (Fig. a Representative macroscopic images of pitavastatin- and PBS-treated WT liver exposed to HBV+DEN at 8 months post-infection. Arrows point to liver tumors. b Tumor burden measured as % liver surface area of pitavastatin- (n = 10) and PBS-treated (n = 5) WT mice that received HBV+DEN at 8 months post-infection. Each dot represents a mouse. Experimental data were verified in two independent experiments. c Representative images of H&E stained pitavastatin- and PBS-treated WT liver exposed to HBV+DEN at 8 months post-infection. d Representative images of PCNA stained pitavastatin- and PBS-treated WT liver exposed to HBV+DEN at 8 months post-infection. e Quantification of PCNA+ hepatocytes in pitavastatin- (n = 4) and PBS-treated (n = 5) WT mice that received HBV+DEN at 8 months post-infection. PCNA+ cells per 100 hepatocytes were counted in five to eight randomly selected HPF images of each liver. Each dot represents an HPF image. f Serum IL-33 levels in HBV-positive hepatitis patients (n = 40), HBV-negative hepatitis patients (n = 40), and healthy controls (n = 24). Each dot represents a patient serum sample. g Representative images of IL-33 stained liver tissues from HBV-positive hepatitis patients, HBV-negative hepatitis patients, and healthy control. h A retrospective cohort analysis of hepatitis and HCC in hepatitis risk in matched cohorts of patients treated with pitavastatin (test) versus ezetimibe (control) (two-sided two-proportion z-test). i The correlation between IL33 expression levels and CD8+ T cell and Treg cell infiltration in HCC samples (n = 371) from TCGA. j The correlation between IL1RL1 expression levels and CD8+ T cell and Treg cell infiltration in HCC samples (n = 371) from TCGA. k Schematic diagram outlining the mechanism that links HBV infection to increased hepatitis and HCC risk (Created in BioRender. Graphs show mean + SD, (b, e) two-sided unpaired t-test, (f) one-way ANOVA with Tukey's multiple comparison test, (i, j) two-sided t-test for Pearson correlation coefficient, scale bars: 1 cm (liver images) and 100 μm (histology). Source data are provided as a Source Data file. Our findings reveal the essential role of environmental carcinogens in linking HBV to hepatitis and HCC, highlighting the potential for effective strategies to prevent cancer development in HBV-infected individuals. HBV infection alone does not result in liver inflammation or cancer. Instead, HBV infection primarily exacerbates liver inflammation and early cancer development induced by carcinogens. This phenomenon is mediated by HBV, promoting DAMPs (e.g., HMGB1) expression in the HBV-containing hepatocytes that experience carcinogen-induced cellular damage. Importantly, we identify a key immune factor, IL-33, downstream of DAMP induction that significantly contributes to cancer development. IL-33 plays a pivotal role in the pathogenesis of HBV plus carcinogen-induced liver injury by binding to ST2+ Tregs in the liver. IL-33-activated Tregs produce higher levels of cytokines, TGF-β and IL-10, which promote chronic inflammation and early cancer development while inhibiting the recruitment of antitumor cytotoxic CD8+ T cells58,59,60. Our research demonstrates that blocking the IL-33/Treg signaling axis, particularly through statin treatment, can effectively suppress chronic hepatitis and prevent HCC development in HBV-infected individuals. Despite the challenges associated with studying liver cancer development in the context of HBV infection, we have utilized an innovative HBV infection-like model employing AAV vector and hydrodynamic injection techniques61,62 to gain insights into HBV-related immune responses and spontaneous liver cancer development. Chronic HBV infection is associated with a dysregulated immune response characterized by compromised adaptive immunity and increased liver damage63,64. Our research provides innovative insights into the mechanism linking HBV to liver damage, which is governed by exposure to carcinogens. We find that Tregs' role in HBV-associated liver pathogenesis is mediated by IL-33 induction in the liver65. IL-33 is an alarmin cytokine induced in response to cellular injury, which has pro- and anti-cancer effects in various contexts43,66,67,68,69,70. IL-33 induced by TLR3/4-TBK1-IRF3 signaling pathways drives cancer-prone chronic inflammation in the skin and pancreas39,42,71. However, IL-33 can enhance the activity of CD8+ T cells and natural killer cells, thereby inhibiting cancer development72,73. We find that IL-33 induction is associated with HMGB1 upregulation in the liver, a known TLR4 ligand46,74. Interestingly, elevated HMGB1 expression is linked to various liver diseases, including hepatitis B, liver fibrosis, and liver cancer75,76,77,78. Previous research has shown that the HBV X protein could increase HMGB1 expression levels78,79. Furthermore, we find that the TBK1-IRF3 pathway is activated and essential for cancer development in the HBV plus carcinogen-treated liver. Pitavastatin inhibits the TBK1/IRF3 pathway by blocking TBK1 membrane recruitment, which is crucial to activating TBK139. Thus, the TLR3/4-TBK1-IRF3-IL-33 signaling pathway activation by HBV plus carcinogen is a mechanism that explains hepatitis and HCC risk associated with HBV infection, which can be blocked by statin to suppress hepatitis and HCC. Although IL-33 induction in the liver can serve to combat HBV infection80, it is also elevated in other liver pathologies, including nonalcoholic steatohepatitis and intrahepatic cholangiocarcinoma81,82,83. In fact, IL-33 plays an important role in several liver diseases84,85,86,87, including obesity-associated HCC88. IL-33 targets innate lymphoid cell (ILC) 2 and T helper 2 (Th2) cells, which contribute to liver fibrosis by inducing the expression of IL-1385,87. IL-33 supports liver cancer growth by remodeling the tumor microenvironment, leading to the secretion of tumor-promoting factors such as vascular endothelial growth factor86. Therefore, targeting IL-33 signaling pathways using statins has the potential to inhibit liver fibrosis and inhibit HCC development. Our epidemiological data support the use of pitavastatin as an effective therapy for hepatitis and HCC. By inhibiting the mevalonate pathway, statins not only block IL-33 induction but also disrupt viral replication by controlling the expression of adhesion molecules on the surface of host cells90,91,92,93, potentially extending their therapeutic benefits to controlling HBV infection. There are several mouse models that mimic hepatitis B virus infection, including transgenic mouse models and adenovirus-based models94,95,96. Although our hydrodynamic injection model does not involve live HBV infection and may not fully represent HBV infection in humans, it leads to the persistence of the HBV genome and its gene expression in the liver following the hydrodynamic injection. To model liver cancer development in mice, DEN injection into 14-day-old male mice has been widely used97,98. However, to specifically investigate HBV-associated liver cancer development, we established a protocol that incorporates HBV infection in young animals followed by several DEN injections to enable cancer development in adult mice. Although RNA sequencing suggested the elevation of other cytokines (e.g., IL-β and IL-4) in HBV+DEN-treated liver, we did not observe any significant elevation for these cytokines at the protein level. Interestingly, Il18 gene expression was found to be suppressed in the HBV+DEN-treated liver, which warrants future investigations to determine its impact on hepatitis and HCC development. In conclusion, our research sheds light on the intricate interplay between HBV infection and carcinogen exposure to generate immune dysregulation in the liver that leads to cancer development. By targeting the IL-33/Treg axis with pitavastatin, we offer a promising approach for the prevention and treatment of chronic hepatitis and its cancer sequelae in HBV-infected individuals. We propose the potential of combining statin therapy with conventional HBV treatments like entecavir to improve patient outcomes39. Massachusetts General Hospital IACUC approved the animal studies. The First Affiliated Hospital of USTC Ethics Committees approved the human sample studies. All serum and liver biopsy samples were collected from the First Affiliated Hospital of University of Science and Technology of China (Hefei, China). All patients in this study provided written informed consent for sample collection and data analysis. The healthy control liver biopsy came from the distal healthy liver tissue of surgical resection samples from patients with liver hemangioma. HBV+ is defined as HBsAg> 0.08 IU/mL. This study was approved by the Ethics Committees of the First Affiliated Hospital of University of Science and Technology of China (USTC) (2020-KYLS-124). The clinical data used in this study are anonymized and aggregated (Supplementary Data 1). All mice were housed under pathogen-free conditions in an animal facility at Massachusetts General Hospital following animal care regulations. Mice were housed on a 12 light /12 dark cycle, at ~18–23 °C with 40%–60% humidity. Irf3KO mice were purchased from the Riken BioResource Research Center (Ibaraki, Japan). Il33KO mice were a gift from Dr. Marco Colonna, and ST2KO mice were a gift from Dr. Peter Nigrovic. TregST2CKO mice (Foxp3Cre RRID: IMSR_JAX: 016959; ST2flox/flox) were generated by Dr. Richard T. Lee and Dr. Diane Mathis. All mutant mice were maintained on the C57BL/6 background. WT C57BL/6 mice were purchased from the Jackson Laboratory (Bar Harbor, ME). Massachusetts General Hospital IACUC permitted a 20 mm maximal tumor diameter, and the maximal tumor size was not exceeded. Mice were euthanized under anesthesia at predetermined time points, including tumors reaching 20 mm diameter, weigh loss greater than 20% of the initial total body weight, a weigh increase of 20% due to ascitic fluid, a hunchbacked appearance, or a moribund state. The pAAV-HBV1.2 plasmid, which contains a 1.2 fold over-length HBV genotype A genome, was kindly provided by Dr. Per-Jer Chen. 6 μg of pAAV-HBV1.2 plasmid DNA was hydrodynamically injected into the tail veins of 4-week-old male mice. An empty vector (pAAV-sham) is used as a control. Rapidly injecting a large volume of plasmid (10% body weight in 5–7 sec) results in HBV DNA transfection mostly into hepatocytes62 and provides an immunocompetent mouse model for chronic HBV infection61. HBsAg serum levels were determined by ELISA once a month after pAAV-HBV1.2 plasmid injection. To establish an HBV-associated hepatocellular carcinoma model, male C57BL/6 mice received DEN (Diethylnitrosamine, Merck, Darmstadt, Germany, catalog no. 73861) administration within 1 week of pAAV-HBV1.2/empty vector plasmid hydrodynamic injection. DEN was intraperitoneally administrated for eight consecutive weeks at increasing concentrations:10 mg/kg body weight for the first two weeks, 20 mg/kg body weight for the second two weeks, 40 mg/kg body weight for the third two weeks, and 60 mg/kg body weight for the last two weeks. Liver tissues were harvested at different time points following both the HBV challenge and DEN exposure. Pitavastatin in dimethyl sulfoxide (DMSO) (5 mg/mL) was made as a stock solution for the mice experiment. Mice were treated intraperitoneally with 2 mg/kg pitavastatin (Selleck Chemicals LLC, Houston, TX, catalog no. S1759) in 100 μL PBS or the same volume of DMSO with PBS. Pitavastatin was given once a week until harvest. Liver tissues were meshed and lysed by 0.1% TWEEN-20 (Merck, catalog no. Tissue lysates were frozen in liquid nitrogen and thawed by incubation at 37 °C for further lysis. Lysates were sonicated for 10 seconds and centrifugated at high speed (15,000 g). After checking the protein concentration with Pierce BCA protein assay kit (Thermo Fisher Scientific, Waltham, MA, catalog no. 23225) in each sample, the identical amounts of total proteins were loaded onto Mini-PROTEIN TGXTM Gels (BIO-RAD, Hercules, CA, catalog no.456-1083 or 456-1086) with 1X Tris/Glycine/SDS buffer (BIO-RAD, catalog no.1610732). After 40 min (according to protein size, 200 Voltage), samples were transferred to Immobilon–P membrane (Merck, catalog no. IPVH00010) with Transfer buffer (Boston Bioproducts, Ashland, MA, catalog no. For blocking step, the membranes were incubated with 3% bovine serum albumin (Thermo Fisher Scientific, catalog no. BP1600) or 5% Skim milk (BD biosciences, San Jose, CA, catalog no. 232100) in 1X Tris-Buffered Saline (Boston Bioproducts, Milford, MA, catalog no. BM301X) containing 0.1% TWEEN, called TBS-T for 30 min. After washing with TBS-T three times, the membranes were subjected to immunoblot with proper antibodies overnight at 4 °C. The following day, membranes were incubated with appropriate rabbit secondary antibody after washing. Membranes were developed with Pierce ECL Western blotting substrate kit (Thermo Fisher Scientific, catalog no. 32106) and signals were confirmed with BIO-RAD (BIO-RAD, catalog no. Primary and secondary antibodies are listed in Supplementary Data 2. Tissue samples were collected and fixed in 4% paraformaldehyde (Merck, catalog no. Next, tissues were dehydrated in PBS and 30, 50, 70% ethanol, processed, and embedded in paraffin. Five to seven μm sections of paraffin-embedded tissues were placed on slides, deparaffinized, and stained with Hematoxylin & Eosin (H&E). For immunohistochemistry (IHC), antigen retrieval was performed in 500 μL of antigen unmasking solution (VECTOR Laboratories, catalog no. H3300) diluted in 50 mL distilled water using a Cuisinart pressure cooker for 20 min at high pressure. Slides were washed three times for three minutes each in 1X TBS with 0.025% Triton X-100. Sections were blocked with 3% bovine serum albumin (Thermo Fisher Scientific, catalog no. BP1600) and 5% goat serum (Merck, catalog no. G9023) for 1 h. Slides were incubated overnight at 4 °C with a primary antibody diluted in TBS containing 3% BSA (Supplementary Data 2). The next day, slides were washed as above and incubated in 100 μL biotinylated secondary antibody (VECTOR Laboratories, catalog no. Slides were subjected to a 100 μL mixture of reagents A and B from VECTASTAIN Elite ABC universal kit Peroxidase (VECTOR Laboratories, catalog no. After washing, slides were incubated with 100 μL ImmPACT DAB chromogen staining (VECTOR Laboratories, catalog no. SK-4105) for a few minutes (depending on the signal, usually less than 1–2 min). Finally, slides were dehydrated in 70, 90, 100% ethanol and xylene and mounted with a coverslip using three drops of mounting media. For immunofluorescence (IF) staining, rehydrated tissue sections were permeated with 1X PBS supplemented with 0.2% Triton X-100 for 5 min. Antigen retrieval was used similarly to IHC. Slides were washed three times for 3 min each in 1X PBS with 0.1% Tween-20. For the blocking step, slides were blocked with 3% bovine serum albumin and 5% goat serum for 1 h. The slides were incubated overnight at 4 °C with primary antibodies. The next day, slides were washed as above and stained for 2 h at room temperature with secondary antibodies conjugated to fluorochromes. Next, slides were incubated with 1:5000 DAPI (Thermo Fisher Scientific, catalog no. D3571) with PBS for 3 min at room temperature, then washed as above. Slides were covered with Prolong Gold Antifade Reagent (Thermo Fisher Scientific, catalog no. The number of positive cells was counted in randomly selected high-power field (HPF, 200 x magnification) images in a blinded manner by a trained investigator. A pathologist reviewed clinical samples. Total RNA was extracted from the mouse liver samples harvested at 4 months after HBV infection, using the RNeasy Mini Kit (Qiagen, Hilden, Germany, catalog no. 74104) following the manufacturer's instructions. For each sample, 1 μg of RNA was retrotranscribed into cDNA using SuperScript III Reverse Transcriptase (Thermo Fisher Scientific, catalog no.18080093) as indicated by the provider. Novogene performed and analyzed RNA sequencing (RNA-seq). Samples were barcoded and run on a HiSeq 2500 in 50 bp/50 bp paired-end run using HiSeq3000 SBS Kit (Illumina) for 30-40 million paired reads. The raw sequencing FASTQ files were aligned against mm10 assembly by STAR (v2.6.1b). FPKM (Fragments Per Kilobase Million) values were then computed from gene level counts by using fpkm function from the R package “DESeq2”. Original data are available in the NCBI Gene Expression Omnibus (GEO) with accession number GSE269528. The normal liver and chronic hepatitis B single-cell datasets were downloaded from the GEO database. The single-cell RNA-sequencing (scRNA-seq) data of normal human liver tissues (n = 3) and chronic hepatitis B tissues (n = 13) were from GSE23424156. The single-cell RNA sequencing data of immune cells from HBV-associated immune active (IA), chronic resolved (CR), and HBV-free healthy controls (HC) were from GSE18215957. After filtering out cells with a mitochondrial gene ratio of less than 25%, Seurat (v5.1.0) was used for data analysis. Principal component analysis (PCA) was performed on single-cell samples, and the top 20 principal components (PC) were selected for subsequent analysis. The uniform manifold approximation and projection (UMAP) was used to perform an overall dimensionality reduction analysis on the top 20 PC pairs of samples. Data visualization was conducted using the ggplot2 package (v3.5.2). The correlation of IL33 and IL1RL1 expression with CD8+ T cell and Treg infiltration in HCC samples included in the TCGA dataset was obtained from the “GENE” module in the Tumor Immune Estimation Resource (TIMER) (https://cistrome.shinyapps.io/timer/) database99. DNA was extracted from mouse tissue using KAPA Express Extract buffer, and KAPA Express Extract enzyme from Kapa Genotyping kit (Kapa Biosystems Inc, Wilmington, MA, catalog no. After the tissue lysis step, PCR was performed using 2X KAPA2G Fast genotyping mix from the Genotyping kit and the primers that are listed in Supplementary Data 3. Analysis was performed using de-identified data from the TriNetX Diamond Network. A search query was used to identify the cohort of patients within the network who had received pitavastatin. Eligible patients were identified based on the presence of corresponding RxNorm concept unique identifiers (RXCUI) in the patient's electronic medical records. Using International Classification of Diseases Tenth Revision (ICD-10) codes, patients with a history of chronic hepatitis and HCC prior to statin initiation were excluded from the cohorts to reduce confounding. The control cohort for each analysis included all patients within the network who had received ezetimibe but had no recorded statin use and patients with a history of any of the diagnoses mentioned above before ezetimibe initiation were also excluded. The index event for all analyses was the initiation of pitavastatin for test cohorts and the initiation of ezetimibe for the control cohort. Cases and controls were matched using 1:1 propensity score-matching for age at index event, sex, race, and ethnicity using “greedy nearest neighbor matching” and a caliper of 0.1 pooled standard deviations. Baseline characteristics were reported by count and percentage of the total for categorical variables and mean and standard deviations (SD) for continuous variables. Relative risks are presented with 95% confidence intervals. P values are uncorrected and based on Z-tests or Fisher's exact tests. Statistical analyses were performed in real-time using the TriNetX platform. HBsAg in the mouse serum was quantified with QuicktiterTM HBsAg ELISA kit (Cell Biolabs, San Diego, CA, catalog no. VPK-5004) as per the supplier's instructions. HBsAg in human serum was determined by chemiluminescent microparticle immunoassay with anti-HBs Reagent Kit (Abbott, Chicago, IL, catalog no. HBV DNA values were measured in human serum using quantitative real-time PCR HBV DNA assay (DA-AN Gene, China, catalog no. To detect HBV DNA in mouse serum, we used HBV TaqMan PCR kit (NORGEN BIOTEK, Canada, Catalog no. TM29250), which measures relative HBV concentration in comparison to the positive control provided in the kit. Serum alanine aminotransferase (ALT) activity was measured for mice and patients with hepatitis and other liver diseases. ALT activity was determined by Alanine Aminotransferase Activity Assay Kit (Sigma-Aldrich, St. Louis, MO, catalog no. For the colorimetric assay, the absorbance at 570 nm was determined and proportional to the pyruvate generated as the ALT activity. ALT activity is reported as nmole/min/mL = milliunit/mL, where one milliunit (mU) of ALT is defined as the amount of enzyme that generates 1.0 nmole of pyruvate per minute at 37 °C. Livers from mice were harvested, mashed through a 100 µm filter and centrifuged at 55 g for 1 min to remove hepatocytes. The samples were then centrifuged at 3000 g to collect cells. The pellets were resuspended in 40% isotonic Percoll (Merck, Catalog no. GE17-0891-01), a mixture of Percoll with 1X PBS and 10X PBS, and the samples were gently layered onto 70% Percoll. After centrifugation at 2000 g with very low deceleration conditions, cells were collected from the interface between the 40% and 70% Percoll layers. The cells were washed with PBS and incubated in RPMI 1640 (Life Technologies, Carlsbad, CA, catalog no. 21870076) supplemented with 10% FBS (Corning, Manassas, VA, USA) and 1% penicillin and streptomycin (Thermo Fisher Scientific, catalog no. 14140122) (R10 media) along with a cell activator cocktail (Biolegend, San Diego, CA, catalog no. After one hour of incubation, monensin (Biolegend, catalog no. 420701) and brefeldin A (Biolegend, catalog no. 420601) were added, and the cells were incubated for an additional three hours at 37 °C. The cells were stained with the following surface marker antibodies: anti-CD45, anti-CD3, anti-CD4, anti-CD8, anti-ST2 and anti-CD25 antibodies (Supplementary Data 2). They were then fixed and permeabilized using the True-Nuclear Transcription Factor Buffer Set (Biolegend, catalog no. 424401) for staining with anti-FoxP3, anti-GATA3, anti-IL-10, and anti-TGF-β antibodies (Supplementary Data 2). The stained cells were analyzed using a LSR Fortessa X-20 flow cytometer (BD Bioscience) and the data were processed with FlowJo software (Tree Star). Commercial ELISA kits were used according to manufacturers' protocols. ELISA kit for the measurement of HMGB1 was purchased from Thermo Scientific (catalog no. The mouse IL-33, IL-4, and IL-1β quantification kits were purchased from Biolegend (catalog no. The mouse IL-16 quantification kit was purchased from RayBiotech (catalog no. The human IL-33 quantification kit was purchased from MultiSciences (catalog no. Livers and spleens from C57BL/6 mice were harvested and meshed through a 70 µm filter, and the cells were resuspended in RBC lysis buffer for 3 min. After washing with PBS, the CD4+ T cells were isolated using MojoSort mouse CD4+ T cells isolation kits (Biolegend, catalog no. After cell counting, 5 × 106 cells were resuspended in 1 mL of R10 media containing 2 μg/mL of CD28 antibody (BioXcell, Lebanon, NH, catalog no. The cells were then plated in a 12-well plate coated with 10 μg/mL of CD3 antibody (BioXcell, catalog no. After 24 h, PBS, IL-2 (20 ng/mL), or IL-2 (20 ng/mL) plus IL-33 (200 ng/mL) was added to the cells. The next day, monensin and brefeldin A were added and the samples were incubated for an additional four hours at 37 °C. Finally, the samples were stained with the appropriate fluorochrome-conjugated antibodies for flow cytometry analysis. A two-sided t-test for the Pearson correlation coefficient was used for the correlation between IL33 and IL1RL1 expressions and immune cell infiltration, and the correlation between serum IL-33 and HBsAg levels in patients. Statistical differences between the three groups were analyzed using one-way ANOVA. Tukey's multiple comparison test was used to examine the differences in the mean ranks among all three possible pairwise comparisons. A two-sided unpaired t-test was used to test the significance of tumor area, PCNA-positive cells, leukocyte counts, protein expression levels, and other quantitative measurements. Comparisons of survival were performed with the log-rank test. The risk ratios of epidemiological data were compared using a two-tailed, two-proportion z-test. p-value < 0.05 is considered significant. Bar graphs show mean + SD. Sample sizes were not predetermined based on statistical methods but were chosen according to the standards of the field (at least three independent biological replicates for each condition). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All data needed to evaluate the conclusions in the paper are present in the paper and the Supplementary Information. RNA sequencing data can be accessed from NCBI database, GEO accession no. The single-cell RNA sequencing data can be accessed from NCBI database, GEO accession no. Source data supporting the findings of this study are provided with this paper and is available in Figshare at https://doi.org/10.6084/m9.figshare.29270633. Hsu, Y. C., Huang, D. Q. & Nguyen, M. H. Global burden of hepatitis B virus: current status, missed opportunities and a call for action. Hepatitis B virus infection. Iannacone, M. & Guidotti, L. G. Immunobiology and pathogenesis of hepatitis B virus infection. Jiang, Y., Han, Q., Zhao, H. & Zhang, J. The mechanisms of HBV-induced hepatocellular carcinoma. Howell, J. et al. Pathway to global elimination of hepatitis B: HBV cure is just the first step. Wang, W. et al. Dual-targeting nanoparticle vaccine elicits a therapeutic antibody response against chronic hepatitis B. Nat. Mahmood, F. et al. HBV vaccines: advances and development. A. et al. Assessment of hepatitis B vaccination status and hepatitis B surface antibody titres among health care workers in selected public health hospitals in Kenya. & Berger, C. T. A simple clinical score to identify likely hepatitis B vaccination non-responders - data from a retrospective single center study. Roberts, H. et al. Prevalence of HBV infection, vaccine-induced immunity, and susceptibility among at-risk populations: US Households, 2013–2018. LaMori, J. et al. Hepatitis vaccination adherence and completion rates and factors associated with low compliance: a claims-based analysis of U.S. adults. Khandekar, M. J., Cohen, P. & Spiegelman, B. M. Molecular mechanisms of cancer development in obesity. Fane, M. & Weeraratna, A. T. How the ageing microenvironment influences tumour progression. Sung, W. K. et al. Genome-wide survey of recurrent HBV integration in hepatocellular carcinoma. & Nault, J. C. Genomics of viral hepatitis-associated liver tumors. Analysis of viral integration reveals new insights of oncogenic mechanism in HBV-infected intrahepatic cholangiocarcinoma and combined hepatocellular-cholangiocarcinoma. Zong, L. et al. Breakdown of adaptive immunotolerance induces hepatocellular carcinoma in HBsAg-tg mice. Dimitriadis, K., Katelani, S., Pappa, M., Fragkoulis, G. E. & Androutsakos, T. The role of interleukins in HBV infection: a narrative review. Kostyusheva, A., Brezgin, S., Glebe, D., Kostyushev, D. & Chulanov, V. Host-cell interactions in HBV infection and pathogenesis: the emerging role of m6A modification. Carty, M., Guy, C. & Bowie, A. G. Detection of viral infections by innate immunity. Kawai, T. & Akira, S. Innate immune recognition of viral infection. Rossi, M. et al. Phenotypic CD8 T cell profiling in chronic hepatitis B to predict HBV-specific CD8 T cell susceptibility to functional restoration in vitro. Andreata, F. et al. Therapeutic potential of co-signaling receptor modulation in hepatitis B. Khanam, A., Chua, J. V. & Kottilil, S. Immunopathology of chronic hepatitis B infection: role of innate and adaptive immune response in disease progression. Gao, Y. et al. HBV-associated hepatocellular carcinomas inhibit antitumor CD8(+) T cell via the long noncoding RNA HDAC2-AS2. Ye, B. et al. T-cell exhaustion in chronic hepatitis B infection: current knowledge and clinical significance. Cell Death Dis. Peripheral natural killer cells in chronic hepatitis B patients display multiple molecular features of T cell exhaustion. Vyas, A. K., Jindal, A., Hissar, S., Ramakrishna, G. & Trehanpati, N. Immune balance in hepatitis B infection: present and future therapies. Kawanaka, M., Nishino, K., Kawamoto, H. & Haruma, K. Hepatitis B: who should be treated?-managing patients with chronic hepatitis B during the immune-tolerant and immunoactive phases. Li, Z. et al. Histologic changes in immune-tolerant patients with chronic hepatitis B: a systematic review and meta-analysis. & Tian, Z. HBV-induced immune imbalance in the development of HCC. Kumar, M. et al. Virologic and histologic features of chronic hepatitis B virus-infected asymptomatic patients with persistently normal ALT. Levrero, M. & Zucman-Rossi, J. Mechanisms of HBV-induced hepatocellular carcinoma. Chuang, S. C. et al. Interaction between cigarette smoking and hepatitis B and C virus infection on the risk of liver cancer: a meta-analysis. Sayiner, M., Golabi, P. & Younossi, Z. M. Disease burden of hepatocellular carcinoma: a global perspective. Hepatitis B vaccination and risk of hepatitis B infection in HIV-infected individuals. Arboatti, A. S. et al. Diethylnitrosamine increases proliferation in early stages of hepatic carcinogenesis in insulin-treated type 1 diabetic mice. Kumar, A. et al. Piperlongumine inhibits diethylnitrosamine induced hepatocellular carcinoma in rats. Park, J. H. et al. Statin prevents cancer development in chronic inflammation by blocking interleukin 33 expression. & Alino, S. F. Translational advances of hydrofection by hydrodynamic injection. Suda, T. et al. Hydrodynamic delivery: characteristics, applications, and technological advances. Park, J. H. et al. Nuclear IL-33/SMAD signaling axis promotes cancer development in chronic inflammation. Miller, A. M. Role of IL-33 in inflammation and disease. You, Y. et al. Phyllanthin prevents diethylnitrosamine (DEN) induced liver carcinogenesis in rats and induces apoptotic cell death in HepG2 cells. Naylor, G. et al. Immunogenic death of hepatocellular carcinoma cells in mice expressing caspase-resistant ROCK1 is not replicated by ROCK inhibitors. Signaling of high mobility group box 1 (HMGB1) through toll-like receptor 4 in macrophages requires CD14. Jiang, C. et al. Association between the HMGB1/TLR4 signaling pathway and the clinicopathological features of ovarian cancer. Shang, Y. et al. Inhibition of HMGB1/TLR4 signaling pathway by digitoflavone: a potential therapeutic role in alcohol-associated liver disease. Boothby, I. C. et al. Early-life inflammation primes a T helper 2 cell-fibroblast niche in skin. Conrad, M. L., Renz, H. & Blaser, K. Immunological approaches for tolerance induction in allergy. The gut feeling of Treg cells: IL-10 is the silver lining during colitis. Chinen, T. et al. An essential role for the IL-2 receptor in T(reg) cell function. Wong, H. S. et al. A local regulatory T cell feedback circuit maintains immune homeostasis by pruning self-activated T cells. Selective expansion of regulatory T cells using an orthogonal IL-2/IL-2 receptor system facilitates transplantation tolerance. Restoration of HBV-specific CD8(+) T-cell responses by sequential low-dose IL-2 treatment in non-responder patients after IFN-α therapy. Genshaft, A. S. et al. Single-cell RNA sequencing of liver fine-needle aspirates captures immune diversity in the blood and liver in chronic hepatitis B patients. Zhang, C. et al. Single-cell RNA sequencing reveals intrahepatic and peripheral immune characteristics related to disease phases in HBV-infected patients. Smith, L. K. et al. Interleukin-10 Directly inhibits CD8(+) T cell function by enhancing N-glycan branching to decrease antigen sensitivity. Gunderson, A. J. et al. TGFbeta suppresses CD8(+) T cell expression of CXCR3 and tumor trafficking. Rivas, J. R. et al. Interleukin-10 suppression enhances T-cell antitumor immunity and responses to checkpoint blockade in chronic lymphocytic leukemia. Huang, L. R., Wu, H. L., Chen, P. J. & Chen, D. S. An immunocompetent mouse model for the tolerance of human chronic hepatitis B virus infection. Suda, T., Gao, X., Stolz, D. B. & Liu, D. Structural impact of hydrodynamic injection on mouse liver. Tang, L. S. Y., Covert, E., Wilson, E. & Kottilil, S. Chronic Hepatitis B Infection: a Review. Huang, S. C. & Liu, C. J. Chronic hepatitis B with concurrent metabolic dysfunction-associated fatty liver disease: challenges and perspectives. & Wu, H. Regulatory T-cells promote hepatitis B virus infection and hepatocellular carcinoma progression. Mehraj, V., Ponte, R. & Routy, J. P. The dynamic role of the IL-33/ST2 axis in chronic viral-infections: alarming and adjuvanting the immune response. & Nishida, A. Pro- and anti-inflammatory roles of interleukin (IL)-33, IL-36, and IL-38 in inflammatory bowel disease. Cayrol, C. & Girard, J. P. IL-33: an alarmin cytokine with crucial roles in innate immunity, inflammation and allergy. Scott, I. C. et al. Interleukin-33 is activated by allergen- and necrosis-associated proteolytic activities to regulate its alarmin activity during epithelial damage. Hasegawa, T., Oka, T. & Demehri, S. Alarmin cytokines as central regulators of cutaneous immunity. Ameri, A. H. et al. IL-33/regulatory T cell axis triggers the development of a tumor-promoting immune environment in chronic inflammation. Qin, L. et al. Exogenous IL-33 overcomes T cell tolerance in murine acute myeloid leukemia. Xu, L. et al. IL-33 induces thymic involution-associated naive T cell aging and impairs host control of severe infection. Yu, M. et al. HMGB1 signals through toll-like receptor (TLR) 4 and TLR2. Emerging role of high-mobility group box 1 (HMGB1) in liver diseases. Li, J. et al. HMGB1-induced autophagy facilitates hepatic stellate cells activation: a new pathway in liver fibrosis. Rizk, N. I., Sallam, A. M., El-Ansary, A. R. & El-Mesallamy, H. O. HMGB1 and SEPP1 as predictors of hepatocellular carcinoma in patients with viral C hepatitis: Effect of DAAs. Zhang, Y. et al. Elevated HMGB1 expression induced by hepatitis B virus X protein promotes epithelial-mesenchymal transition and angiogenesis through STAT3/miR-34a/NF-kappaB in primary liver cancer. Hepatitis B virus X protein stimulates high mobility group box 1 secretion and enhances hepatocellular carcinoma metastasis. Hepatitis B virus persistence in mice reveals IL-21 and IL-33 as regulators of viral clearance. Brunt, E. M. et al. Nonalcoholic fatty liver disease. Sheka, A. C. et al. Nonalcoholic steatohepatitis: a review. Brindley, P. J. et al. Cholangiocarcinoma. Tan, Z. et al. Interleukin-33 drives hepatic fibrosis through activation of hepatic stellate cells. McHedlidze, T. et al. Interleukin-33-dependent innate lymphoid cells mediate hepatic fibrosis. Wang, W., Wu, J., Ji, M. & Wu, C. Exogenous interleukin-33 promotes hepatocellular carcinoma growth by remodelling the tumour microenvironment. Reissing, J. et al. Th2 Cell activation in chronic liver disease is driven by local IL33 and contributes to IL13-dependent fibrogenesis. Yamagishi, R. et al. Gasdermin D-mediated release of IL-33 from senescent hepatic stellate cells promotes obesity-associated hepatocellular carcinoma. The IL-33-ST2-MyD88 axis promotes regulatory T cell proliferation in the murine liver. Giguere, J. F. & Tremblay, M. J. Statin compounds reduce human immunodeficiency virus type 1 replication by preventing the interaction between virion-associated host intercellular adhesion molecule 1 and its natural cell surface ligand LFA-1. Shrivastava-Ranjan, P. et al. Statins Suppress Ebola Virus Infectivity by Interfering with Glycoprotein Processing. Espano, E. et al. Lipophilic statins inhibit Zika virus production in Vero cells. Gorabi, A. M. et al. Antiviral effects of statins. Iannacone, M. & Guidotti, L. G. Mouse models of hepatitis B virus pathogenesis. Cold Spring Harb Perspect Med. Hwang, J. R. & Park, S. G. Mouse models for hepatitis B virus research. Lai, F., Wee, C. Y. Y. Establishment of humanized mice for the study of HBV. Uehara, T., Pogribny, I. P. & Rusyn, I. The DEN and CCl4 -induced mouse model of fibrosis and inflammation-associated hepatocellular carcinoma. Multiple liver insults synergize to accelerate experimental hepatocellular carcinoma. Li, T. et al. TIMER: A web server for comprehensive analysis of tumor-infiltrating immune cells. We thank Dr. Marco Colonna for the Il33KO mice, which were generated with support from the Mucosal Immunology Studies Team (MIST) (U01; RFA-AI-15-023). We thank Dr. Peter Nigrovic for ST2KO mice, and Dr. Richard T. Lee and Dr. Diane Mathis for TregST2CKO mice. We thank Dr. Per-Jer Chen for the pAAV-HBV1.2 plasmid. SD holds a Career Award for Medical Scientists from the Burroughs Wellcome Fund. were supported by the National Research Foundation of Korea grant funded by the Korea government (MSIT) (RS-2024-00343783). was supported by the Natural Science Foundation of Anhui, China (2008085MH253) and the National Natural Science Foundation of China (82473312). D.W. was supported by the National Natural Science Foundation of China (82370217) and the Natural Science Foundation of Anhui, China (2308085Y46). was supported by the First Affiliated Hospital of Sun Yat-sen University. were supported by grants from the Burroughs Wellcome Fund and NIH (R01CA283214). These authors contributed equally: Mei Huang, Dongyao Wang. Center for Cancer Immunology, Krantz Family Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Mei Huang, Dongyao Wang, Jiao Huang, Yun Xia, Xutu Zhao, Mahsa Mortaja, Marjan Azin, Jong Ho Park & Shadmehr Demehri Cutaneous Biology Research Center, Department of Dermatology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Mei Huang, Dongyao Wang, Jiao Huang, Yun Xia, Xutu Zhao, Mahsa Mortaja, Marjan Azin, Jong Ho Park & Shadmehr Demehri Department of General Surgery, The First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, China Department of Hematology, The First Affiliated Hospital of USTC, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, China Department of Anatomy, School of Medicine, Keimyung University, Daegu, South Korea An-Na Bae & Jong Ho Park Department of Dermatology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA Michael R. Collier & Yevgeniy R. Semenov Laboratory of Systems Pharmacology, Harvard Program in Therapeutic Science, Harvard Medical School, Boston, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar performed the epidemiological study. Correspondence to Jong Ho Park or Shadmehr Demehri. are coinventors on a filed patent for the use of IL-33 inhibition in the treatment of cancer, fibrosis, and inflammation (PCT/US21/40725). Other authors state no conflict of interest. Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Hepatitis B virus promotes liver cancer by modulating the immune response to environmental carcinogens. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250627021845.htm'>AI sees what doctors miss: Fatty liver disease hidden in chest x-rays</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 09:18:36
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>If left untreated, it can lead to serious complications, such as cirrhosis and liver cancer, making it crucial to detect early and initiate treatment. However, the relationship between chest X-rays and fatty liver disease has rarely been a subject of in-depth study. Therefore, a research group led by Associate Professor Sawako Uchida-Kobayashi and Associate Professor Daiju Ueda at Osaka Metropolitan University's Graduate School of Medicine developed an AI model that can detect the presence of fatty liver disease from chest X-ray images. In this retrospective study, a total of 6,599 chest X-ray images containing data from 4,414 patients were used to develop an AI model utilizing controlled attenuation parameter (CAP) scores. We hope it can be put into practical use in the future," stated Professor Uchida-Kobayashi. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250626081540.htm'>This breakthrough turns old tech into pure gold — No mercury, no cyanide, just light and salt</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-27 06:04:51
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Explained in the leading journal Nature Sustainability, the gold-extraction technique promises to reduce levels of toxic waste from mining and shows that high purity gold can be recovered from recycling valuable components in printed circuit boards in discarded computers. The project team, led by Matthew Flinders Professor Justin Chalker, applied this integrated method for high-yield gold extraction from many sources - even recovering trace gold found in scientific waste streams. "The study featured many innovations including a new and recyclable leaching reagent derived from a compound used to disinfect water," says Professor of Chemistry Justin Chalker, who leads the Chalker Lab at Flinders University's College of Science and Engineering. This reagent (trichloroisocyanuric acid) is widely used in water sanitation and disinfection. When activated by salt water, the reagent can dissolve gold. The aim of the Flinders-led project was to provide alternative methods that are safer than mercury or cyanide in gold extraction and recovery. The team also collaborated with experts in the US and Peru to validate the method on ore, in an effort to support small-scale mines that otherwise rely on toxic mercury to amalgamate gold. Gold mining typically uses highly toxic cyanide to extract gold from ore, with risks to the wildlife and the broader environment if it is not contained properly. Professor Chalker says interdisciplinary research collaborations with industry and environmental groups will help to address highly complex problems that support the economy and the environment. "We are especially grateful to our engineering, mining, and philanthropic partners for supporting translation of laboratory discoveries to larger scale demonstrations of the gold recovery techniques." Lead authors of the major new study - Flinders University postdoctoral research associates Dr Max Mann, Dr Thomas Nicholls, Dr Harshal Patel and Dr Lynn Lisboa - extensively tested the new technique on piles of electronic waste, with the aim of finding more sustainable, circular economy solutions to make better use of ever-more-scarce resources in the world. Many components of electronic waste, such as CPU units and RAM cards, contain valuable metals such as gold and copper. Dr Mann says: "This paper shows that interdisciplinary collaborations are needed to address the world's big problems managing the growing stockpiles of e-waste." ARC DECRA Fellow Dr Nicholls, adds: "The newly developed gold sorbent is made using a sustainable approach in which UV light is used to make the sulfur-rich polymer. I hope this research inspires impactful solutions to pressing global challenges." "With the ever-growing technological and societal demand for gold, it is increasingly important to develop safe and versatile methods to purify gold from varying sources," Dr Lisboa concludes. In 2022, an estimated 62 million tonnes of e-waste was produced globally. Only 22.3% was documented as formally collected and recycled. Many of these toxic materials are known or suspected to cause harm to human health, and several are included in the 10 chemicals of public health concern, including dioxins, lead and mercury. Inferior recycling of e-waste is a threat to public health and safety. These are then heated to evaporate the mercury, leaving behind gold but releasing toxic vapours. Studies indicate that up to 33% of artisanal miners suffer from moderate metallic mercury vapor intoxication. These operations, which are often unregulated and unsafe, generate 37% of global mercury pollution (838 tonnes a year) - more than any other sector. Most informal sites lack the funding and training needed to transition towards mercury-free mining. Despite accounting for 20% of the global gold supply and generating approximately US$30 billion annually, artisanal miners typically sell gold at around 70% of its global market value. Additionally, with many gold mines located in rural and remote areas, miners seeking loans are often restricted to predatory interest rates from illegal sources, pushing demand for mercury. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            