
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2026-01-02</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/the-pleiades-star-cluster-has-a-secret-stellar-family/'>The Pleiades Star Cluster Has a Secret Stellar Family Scattered across the Milky Way</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-02 11:45:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>The “Seven Sisters” of the Pleiades are part of a much larger complex that can help reveal our galaxy's deep history Stay connected to The Universe: Get email alerts for this weekly column by Phil Plait The solar system was very likely born in a tremendous gas cloud that also birthed thousands of other stars, forming a large, loosely bound stellar family called an open cluster. These stars may have remained enshrouded in that nebula for 10 million years or more, until their combined stellar winds and light pushed away the surrounding nebulosity. No longer embedded in the gas, the cluster gradually disintegrated as its internal motions flung away some of its members. Throw in collisions with other huge clouds of gas and dust, as well as interactions with the galaxy's gravitational field, and the cluster's fate was sealed: its stars dispersed, eventually mixing with the background “field” stars in the Milky Way. Our solar kin may be long lost, but that's not the case for all stars. Interestingly, we see such loose affiliations of stars (called associations) all over the sky, and they often share similar characteristics, such as age and velocity, through the galaxy. Astronomers now suspect that many of these scattered associations formed in the same gas cloud, where they were born as part of a single, sprawling structure that has since broken up. But we might yet glimpse parts of such long-lost clusters that are still intact because stars from their close-packed cores should be tightly bound to each other via gravity and thus able to survive as a tight-knit group for much longer. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Finding such surviving clusters is important; they can tell us about when and where many of the Milky Way's stars formed and how our galaxy's stellar populations and structures evolved over billions of years. If you're in the Northern Hemisphere and go outside on a clear winter's night, you might notice a fuzzy clutch of stars not too far from Orion. This is the Pleiades (pronounced “PLEE-uh-deez”), a lovely cluster that is about 440 light-years from Earth and located in the constellation Taurus. Six stars can be seen by the unaided eye, although many cultures' ancient myths indicate there are seven. The cluster doesn't have a sharp boundary, but most of the stars are found in a volume that is about 40 light-years across. If it was once part of a bigger structure, this estimated age would align with our expectation that that structure's outer stars would have been shaken loose. Finding any far-flung stars that once were anchored by the Pleiades is not an easy task. To firm up this connection, the researchers next estimated the ages of these stars—a difficult task but one that was made easier by the knowledge that younger stars tend to rotate faster than older ones. (This effect exists because a star's magnetic field can sweep up surrounding material and act like drag on a parachute, slowing the star's spin.) A star's rotation can be determined by looking for tiny variations in brightness as dark star spots—which are like the sun's sunspots but appear on other stars—rotate in and out of view. Using data from NASA's Transiting Exoplanet Survey Satellite, which can precisely measures stellar brightness, the astronomers found stars that rotated with periods shorter than 12 days, which is roughly the expected spin rate for stars as old as the average Pleiades member. Finally, after applying various statistical techniques to look for other stars, the researchers wound up with a list of more than 10,000 possible members of what they called the Greater Pleiades Complex. This includes stars that belong to at least seven previously known associations. Among these associations is the AB Doradus group, which contains a few dozen stars that are only about 70 light-years from the sun. If you go out sometime to look at them—and I really hope you do—gaze upon that tight little cluster and consider that there's still so much we don't know about these stars—and so much more they can tell us about the cosmos. Phil Plait is a professional astronomer and science communicator in Virginia. His column for Scientific American, The Universe, covers all things space. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s43018-025-01068-0'>Tumor ecosystem and microbiome features associated with efficacy and resistance to avelumab plus chemoradiotherapy in head and neck cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-02 10:13:24
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2026)Cite this article Immune checkpoint blockade-based multimodal therapy is widely used across oncology; yet drivers of resistance in most cancer types are not well understood. Here, we comprehensively characterized the tumor genome, microenvironment and microbiome in a phase 3 international randomized trial (NCT02952586) to identify factors that shape outcomes to anti-PD-L1 avelumab plus standard-of-care chemoradiotherapy versus placebo/chemoradiotherapy in individuals with locally advanced head and neck cancer. Patients receiving avelumab whose tumors contained distinct immunologic and genetic features had superior outcomes compared to those receiving placebo. By contrast, patients with increased myeloid/neutrophil activities had worse outcomes with avelumab than those treated with placebo. Strikingly, these tumors possessed telltale intratumoral bacteria, elevated tumor-associated neutrophils, high systemic neutrophil-to-lymphocyte ratios and suppressed adaptive immunity. We define tumor ecosystems associated with benefit to chemoimmunotherapy. Our data demonstrate how intratumoral bacteria affect immune checkpoint blockade response within a randomized trial. These discoveries enhance our understanding of combination immunotherapy, provide a useful multiomic resource and identify unanticipated interactions that may guide future therapeutic strategies. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 digital issues and online access to articles only $9.92 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Any request for additional data outside of that provided in the paper by scientific and medical researchers for legitimate research purposes will be subject to Merck's Data Sharing Policy. All requests should be submitted in writing to Merck's data sharing portal (https://www.merckgroup.com/en/research/our-approach-to-research-and-development/healthcare/clinical-trials/commitment-responsible-data-sharing.html). When Merck has a co-research, co-development, co-marketing or co-promotion agreement or when the product has been out-licensed, the responsibility for disclosure might be dependent on the agreement between parties. Under these circumstances, Merck will endeavor to gain agreement to share data in response to requests. Processed mutation calls, gene expression matrices and TBB data are available in Supplementary Tables 1 and 2 provided in the paper. Raw RNA-seq files have been deposited in the European Genome–Phenome Archive (EGAD00001011290). WES BAM files were also deposited in the European Genome–Phenome Archive (EGAD00001011321). Sequencing data are available per Merck's Data Sharing Policy. Source data are provided with this paper. Ferris, R. L. et al. Nivolumab for recurrent squamous-cell carcinoma of the head and neck. Burtness, B. et al. Pembrolizumab alone or with chemotherapy versus cetuximab with chemotherapy for recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-048): a randomised, open-label, phase 3 study. Ribas, A. & Wolchok, J. D. Cancer immunotherapy using checkpoint blockade. Tang, J. et al. Trial watch: the clinical trial landscape for PD1/PDL1 immune checkpoint inhibitors. Shamseddine, A. A., Burman, B., Lee, N. Y., Zamarin, D. & Riaz, N. Tumor Immunity and immunotherapy for HPV-related cancers. Cancer Discov. Dewan, M. Z. et al. Fractionated but not single-dose radiotherapy induces an immune-mediated abscopal effect when combined with anti-CTLA-4 antibody. Rodriguez-Ruiz, M. E., Vanpouille-Box, C., Melero, I., Formenti, S. C. & Demaria, S. Immunological mechanisms responsible for radiation-induced abscopal effect. Trends Immunol. McLaughlin, M. et al. Inflammatory microenvironment remodelling by tumour cells after radiotherapy. Antonia, S. J. et al. Durvalumab after chemoradiotherapy in stage III non-small-cell lung cancer. Kelly, R. J. et al. Adjuvant nivolumab in resected esophageal or gastroesophageal junction cancer. Stone, H. B., Peters, L. J. & Milas, L. Effect of host immune capability on radiocurability and subsequent transplantability of a murine fibrosarcoma. J. Natl Cancer Inst. Lee, Y. et al. Therapeutic effects of ablative radiation on local tumor require CD8+ T cells: changing strategies for cancer treatment. Deng, L. et al. STING-dependent cytosolic DNA sensing promotes radiation-induced type I interferon-dependent antitumor immunity in immunogenic tumors. Weichselbaum, R. R., Liang, H., Deng, L. & Fu, Y. X. Radiotherapy and immunotherapy: a beneficial liaison?. Zeng, J. et al. Anti-PD-1 blockade and stereotactic radiation produce long-term survival in mice with intracranial gliomas. Deng, L. et al. Irradiation and anti-PD-L1 treatment synergistically promote antitumor immunity in mice. Oweida, A. et al. Ionizing radiation sensitizes tumors to PD-L1 immune checkpoint blockade in orthotopic murine head and neck squamous cell carcinoma. Lee, N. Y. et al. Avelumab plus standard-of-care chemoradiotherapy versus chemoradiotherapy alone in patients with locally advanced squamous cell carcinoma of the head and neck: a randomised, double-blind, placebo-controlled, multicentre, phase 3 trial. Mayadev, J. et al. Durvalumab in combination with chemoradiotherapy (CRT) in locally advanced cervical cancer (LACC): radiotherapy (RT) delivery and subgroup analyses from CALLA. Machiels, J. et al. Primary results of the phase III KEYNOTE-412 study: pembrolizumab (pembro) with chemoradiation therapy (CRT) vs placebo plus CRT for locally advanced (LA) head and neck squamous cell carcinoma (HNSCC). Carbone, D. P. et al. First-line nivolumab in stage IV or recurrent non-small-cell lung cancer. Si, H. et al. A blood-based assay for assessment of tumor mutational burden in first-line metastatic NSCLC treatment: results from the MYSTIC study. Samstein, R. M. et al. Tumor mutational load predicts survival after immunotherapy across multiple cancer types. Haddad, R. I. et al. Influence of tumor mutational burden, inflammatory gene expression profile, and PD-L1 expression on response to pembrolizumab in head and neck squamous cell carcinoma. Cancer Genome Atlas Network. Comprehensive genomic characterization of head and neck squamous cell carcinomas. Leemans, C. R., Snijders, P. J. F. & Brakenhoff, R. H. The molecular landscape of head and neck cancer. Mehanna, H. et al. Prognostic implications of p16 and HPV discordance in oropharyngeal cancer (HNCIG-EPIC-OPC): a multicentre, multinational, individual patient data analysis. Senbabaoglu, Y. et al. Tumor immune microenvironment characterization in clear cell renal cell carcinoma identifies prognostic and immunotherapeutically relevant messenger RNA signatures. Aran, D., Hu, Z. & Butte, A. J. xCell: digitally portraying the tissue cellular heterogeneity landscape. Powles, T. et al. Avelumab maintenance in advanced urothelial carcinoma: biomarker analysis of the phase 3 JAVELIN Bladder 100 trial. Thorsson, V. et al. The immune landscape of cancer. Vanhersecke, L. et al. Standardized pathology screening of mature tertiary lymphoid structures in cancers. Cabrita, R. et al. Tertiary lymphoid structures improve immunotherapy and survival in melanoma. Poropatich, K. et al. Comprehensive T-cell immunophenotyping and next-generation sequencing of human papillomavirus (HPV)-positive and HPV-negative head and neck squamous cell carcinomas. Zilionis, R. et al. Single-cell transcriptomics of human and mouse lung cancers reveals conserved myeloid populations across individuals and species. Galeano Nino, J. L. et al. Effect of the intratumoral microbiota on spatial and cellular heterogeneity in cancer. Nejman, D. et al. The human tumor microbiome is composed of tumor type-specific intracellular bacteria. Battaglia, T. W. et al. A pan-cancer analysis of the microbiome in metastatic cancer. Kostic, A. D. et al. PathSeq: software to identify or discover microbes by deep sequencing of human tissue. Breitwieser, F. P., Baker, D. N. & Salzberg, S. L. KrakenUniq: confident and fast metagenomics classification using unique k-mer counts. Gihawi, A. et al. Major data analysis errors invalidate cancer microbiome findings. Article PubMed Wade, W. G. The oral microbiome in health and disease. Deo, P. N. & Deshmukh, R. Oral microbiome: unveiling the fundamentals. Dohlman, A. B. et al. The cancer microbiome atlas: a pan-cancer comparative analysis to distinguish tissue-resident microbiota from contaminants. Cell Host Microbe 29, 281–298 (2021). Chowell, D. et al. Improved prediction of immune checkpoint blockade efficacy across multiple cancer types. Puram, S. V. et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Gong, Y. et al. The tumor ecosystem in head and neck squamous cell carcinoma and advances in ecotherapy. Chu, Y. et al. Pan-cancer T cell atlas links a cellular stress response state to immunotherapy resistance. Chow, A., Perica, K., Klebanoff, C. A. & Wolchok, J. D. Clinical implications of T cell exhaustion for cancer immunotherapy. Mell, L. K. et al. Radiotherapy with durvalumab vs. cetuximab in patients with locoregionally advanced head and neck cancer and a contraindication to cisplatin: phase II results of NRG-HN004. Clump, D. A. et al. A randomized phase II study evaluating concurrent or sequential fixed-dose immune therapy in combination with cisplatin and intensity-modulated radiotherapy in intermediate- or high-risk, previously untreated, locally advanced head and neck cancer (LA SCCHN). Zhao, X. et al. Somatic 9p24.1 alterations in HPV− head and neck squamous cancer dictate immune microenvironment and anti-PD-1 checkpoint inhibitor activity. Darragh, L. B. et al. A phase I/Ib trial and biological correlate analysis of neoadjuvant SBRT with single-dose durvalumab in HPV-unrelated locally advanced HNSCC. Darragh, L. B. et al. Elective nodal irradiation mitigates local and systemic immunity generated by combination radiation and immunotherapy in head and neck tumors. Wang, L. et al. YTHDF2 inhibition potentiates radiotherapy antitumor efficacy. Cancer Cell 41, 1294–1308 (2023). Nishiga, Y. et al. Radiotherapy in combination with CD47 blockade elicits a macrophage-mediated abscopal effect. Sepich-Poore, G. D. et al. The microbiome and human cancer. Human Microbiome Project Consortium. Structure, function and diversity of the healthy human microbiome. Lamont, R. J., Koo, H. & Hajishengallis, G. The oral microbiota: dynamic communities and host interactions. Nagy, K. N., Sonkodi, I., Szoke, I., Nagy, E. & Newman, H. N. The microflora associated with human oral carcinomas. Oral Oncol. Whitmore, S. E. & Lamont, R. J. Oral bacteria and cancer. Orlandi, E. et al. Potential role of microbiome in oncogenesis, outcome prediction and therapeutic targeting for head and neck cancer. Oral Oncol. Article PubMed Hashim, D. et al. The role of oral hygiene in head and neck cancer: results from International Head and Neck Cancer Epidemiology (INHANCE) consortium. Riquelme, E. et al. Tumor microbiome diversity and composition influence pancreatic cancer outcomes. Kostic, A. D. et al. Fusobacterium nucleatum potentiates intestinal tumorigenesis and modulates the tumor-immune microenvironment. Cell Host Microbe 14, 207–215 (2013). Cogdill, A. P., Gaudreau, P. O., Arora, R., Gopalakrishnan, V. & Wargo, J. The impact of intratumoral and gastrointestinal microbiota on systemic cancer therapy. Trends Immunol. Bender, M. J. et al. Dietary tryptophan metabolite released by intratumoral Lactobacillus reuteri facilitates immune checkpoint inhibitor treatment. Elkrief, A. et al. Intratumoral Escherichia is associated with improved survival to single-agent immune checkpoint inhibition in patients with advanced non-small-cell lung cancer. Goubet, A. G. et al. Escherichia coli-specific CXCL13-producing TFH are associated with clinical efficacy of neoadjuvant PD-1 blockade against muscle-invasive bladder cancer. Cancer Discov. Silver, N. et al. Intratumoral bacteria are immunosuppressive and promote immunotherapy resistance. Cancer https://doi.org/10.1038/s43018-025-01067-1 (in the press). Sturm, G. et al. Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology. Nguyen, H., Nguyen, H., Tran, D., Draghici, S. & Nguyen, T. Fourteen years of cellular deconvolution: methodology, applications, technical evaluation and outstanding challenges. Nucleic Acids Res. Li, H. & Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. DePristo, M. A. et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Koboldt, D. C. et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. Saunders, C. T. et al. Strelka: accurate somatic small-variant calling from sequenced tumor–normal sample pairs. Larson, D. E. et al. SomaticSniper: identification of somatic point mutations in whole genome sequencing data. Rimmer, A. et al. Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications. Ellrott, K. et al. Scalable open science approach for mutation calling of tumor exomes using multiple genomic pipelines. Cell Syst. Amemiya, H. M., Kundaje, A. & Boyle, A. P. The ENCODE blacklist: identification of problematic regions of the genome. Riaz, N. et al. Tumor and microenvironment evolution during immunotherapy with nivolumab. Rosenthal, R., McGranahan, N., Herrero, J., Taylor, B. S. & Swanton, C. DeconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution. Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Kawaguchi, S. & Matsuda, F. High-definition genomic analysis of HLA genes via comprehensive HLA allele genotyping. Methods Mol. Article PubMed Liu, P. et al. Benchmarking the human leukocyte antigen typing performance of three assays and seven next-generation sequencing-based algorithms. Shen, R. & Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. Nucleic Acids Res. Leshchiner, I. et al. Comprehensive analysis of tumour initiation, spatial and temporal progression under multiple lines of treatment. Preprint at bioRxiv https://doi.org/10.1101/508127 (2019). Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Liao, Y., Smyth, G. K. & Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Katz, A. et al. CytoPro: a computational platform for accurate and robust assessment of cell contributions in bulk expression from diverse tissues and conditions. Preprint at bioRxiv https://doi.org/10.1101/2021.08.19.456930 (2021). Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Walker, M. A. et al. GATK PathSeq: a customizable computational tool for the discovery and identification of microbial sequences in libraries from eukaryotic hosts. Oskolkov, N. KrakenUniq database based on complete microbial genomes from NCBI RefSeq, May 2020. FigShare https://doi.org/10.17044/scilifelab.21299541.v1 (2022). Callahan, B. J. et al. DADA2: high-resolution sample inference from Illumina amplicon data. & Holmes, S. phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. PLoS ONE 8, e61217 (2013). We thank the patients and their families, investigators, co-investigators and the study teams at each of the participating centers. We also thank A. Donahue (Pfizer) and P. Robbins (Pfizer) for their roles in establishing the research collaboration with Memorial Sloan Kettering Cancer Center and R.-Y. Tzeng (Memorial Sloan Kettering Cancer Center) for her role in data quality control. We thank colleagues at the Cleveland Clinic and Memorial Sloan Kettering core facilities, including the Cell Imaging Core (CCF), Genomics Core (CCF) and Integrated Genomics Operation (Memorial Sloan Kettering) for processing samples and providing important suggestions. We also thank colleagues at the Department of Pathology (CCF) for pathology advice and the members and alumni of the laboratory of T.A.C. at Memorial Sloan Kettering and CCF for their generous help and support of this study. Data used in this study were generated or collected by the TCGA Research Network. We acknowledge the following funding sources: NIH R01 CA205426 (T.A.C. ), the Sheikha Fatima bint Mubarak Chair (T.A.C.) and NIH/NCI Cancer Center Support Grant P30 CA008748. This trial was sponsored by Pfizer and was previously conducted under an alliance between Merck (CrossRef Funder ID: 10.13039/100009945) and Pfizer. Medical writing support was provided by H. Al-Ashtal of Clinical Thinking and was funded by Merck. The investigators worked with Pfizer on the trial design, collection and analysis of data and interpretation of results. Datasets were reviewed by the authors, and all authors participated fully in developing and reviewing the report for publication. All authors had full access to all data, and the first author had final responsibility for the decision to submit for publication. We vouch for the completeness and accuracy of the data and their analysis and the fidelity of the trial to the protocol and statistical analysis plan. These authors contributed equally: Nadeem Riaz, Tyler J. Alban, Robert I. Haddad. These authors jointly supervised this work: Craig B. Davis, Nancy Y. Lee, Timothy A. Chan. Department of Radiation Oncology, Memorial Sloan Kettering Cancer Center, New York, NY, USA Nadeem Riaz, Yingjie Zhu, Ardijana Novaj & Nancy Y. Lee Department of Cancer Sciences and Center for Immunotherapy and Precision Immuno-Oncology, Cleveland Clinic, Cleveland, OH, USA Tyler J. Alban, Vladimir Makarov, Prerana Bangalore Parthasarathy, Mruniya Gawali, Douglas Hoen, Natalie L. Silver, Ivan Juric, Jennifer Ko, Daniel J. McGrail & Timothy A. Chan Department of Medical Oncology, Dana Farber Cancer Institute, Boston, MA, USA Robert I. Haddad Pfizer, La Jolla, CA, USA Michelle Saul, Phineas Hamilton, Daniel Chawla & Craig B. Davis Moores Cancer Center, University of California San Diego Health, La Jolla, CA, USA UPMC Hillman Cancer Center, Pittsburgh, PA, USA Robert L. Ferris Department of Oncology, Taipei Veterans General Hospital, Taipei, Taiwan Peter Mu-Hsing Chang Department of Radiation Oncology, Changhua Christian Hospital, Changhua, Taiwan National Kapodistrian University of Athens, Athens, Greece Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA National Center for Regenerative Medicine, Cleveland, OH, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Clinical trial organization: R.I.H., N.Y.L., E.E.W.C., R.L.F., P.M.-H.C., J.-C.L. Writing, original draft: N.R., T.J.A., C.B.D. Writing, review and editing: N.R., T.J.A., M.S., R.I.H., N.Y.L. All authors critically discussed and revised the paper for important intellectual content. Correspondence to Craig B. Davis, Nancy Y. Lee or Timothy A. Chan. is a cofounder of Gritstone Oncology and holds equity. also holds equity in An2H. acknowledges grant funding from Bristol-Myers Squibb, AstraZeneca, Illumina, Pfizer, An2H and Eisai. has served as an advisor for Bristol-Myers, MedImmune, Squibb, Illumina, Eisai, AstraZeneca and An2H. is an inventor on intellectual property held by Memorial Sloan Kettering Cancer Center on using TMB to predict immunotherapy response, with pending patent, which has been licensed to PGDx. is on the advisory board of Cell. acknowledges research support from Pfizer, REPARE Therapeutics, Invitae and Bristol-Myers Squibb. The other authors declare no competing interests. Nature Cancer thanks N. Gopalakrishna Iyer, Bertrand Routy, Daniel Spakowicz and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Comparable PFS outcomes are seen in the biomarker analysis cohorts relative to the full dataset. Cox proportional hazards model used to estimate hazard ratio (and 95% CI) between placebo and avelumab in subgroups with genomic data. b. Tumor mutation burden (TMB) is higher in HPV negative (median=5.37 Mut/Mb, IQR=3.95-7.23, N=272) vs. HPV positive (median=4.26 Mut/Mb, IQR=2.53-6.53, N=149) (p < 0.001; Wilcoxon-rank sum test). c. Associations between PFS, treatment, and classical biomarkers were comparable in HPV-positive and HPV-negative patients (Cox proportional hazards model and 95% CI). d. Association between PFS in placebo/CRT arm and expression of PD-L1 by tumor cells (left) vs immune cells (right). PD-L1 high vs low populations (log-rank test). a. Volcano plot showing mutations associated with HPV positive vs HPV negative cancers (Fisher's Exact Test; dotted line represents p-value adjusted for false discovery). Recurrently mutated genes in SCCHN occur at markedly different frequencies between HPV+ (N=149) and HPV- (N=272) tumors. b. Oncoplot of patients identified as HPV-positive (N=149) by p16 status reveals that cases of low coverage by NGS of viral genomes (left side of oncoplot) are more likely to have mutations or copy number changes in TP53 and CDKN2A, suggesting these cases may not be driven by HPV. (left) Concordance between CytoPro and ssGSEA methods for selected signatures related to tertiary lymphoid structures (*** indicates p-value < 0.001) (right) Illustration of relationship between CytoPro signature (Z-normalized) and raw ssGSEA score for Neutrophils (linear regression, shaded region corresponds to 95% CI, N=342). Correlation between IHC CD8 staining (positive cells per total area) with ssGSEA CD8 T cell staining as a validation for ssGSEA measurement of CD8 T cells (r=0.61; p < 0.001, Pearson Correlation; shaded region corresponds to 95% CI, N=342). c. Hierarchical clustering of cell-type contributions estimated by ssGSEA. d. Associations between ssGSEA signatures and PFS (Cox proportional hazards model, 95% CI, and log-rank test, n=344 patients). Note significant association with Tfh (T follicular helper) cells and improved outcomes in Avelumab (N=176, p < 0.001; log-rank test) which is not significant in Placebo and has a significant interaction by treatment arm (N=166, p=0.02). In contrast, Neutrophils are associated with worse outcomes for patients receiving avelumab (p = 0.003; log-rank test). a. Spearman correlation of TBB between 2 analytic pipelines, PathSeq and KrakenUniq, demonstrates good correlation of bacterial burden from tumors (rho=0.93, N=421, p < 2.2*10−16) and normal tissue (rho=0.45; N=421, p < 2.2*10−16). b. Genera counts for each sample in cohort, ordered from highest tumor bacterial abundance (N=421). c. Frequency of species identified in this cohort and HNSC from TCGA are highly correlated (rho = 0.53; p < 0.001; Spearman Correlation). d. LogTBB derived from head and neck cancer cases in TCGA with both WGS and WES (n=155 cancers, with strong linear correlation r=0.88 p < 1*10−32). e Total bacterial read counts aligned to genera in tumor and PBMC samples (split by high vs. low TBB in tumor samples; Wilcoxon test, n=421patients) illustrates markedly higher bacterial reads in tumor compared to blood (p < 0.001; Wilcoxon test). f. Evaluation of log-scale abundance of bacterial reads from WES with T Stage for entire cohort (left) shows a trend towards an association with larger tumors (p=0.064; Wilcoxon Test, n=421 patients) but not N Stage (right, n=421 patients) (p=0.38). g. Evaluation of log-scale abundance of bacterial reads for each Phyla across n=18 samples in 16S compared to WES abundance, shows a strong linear relationship (r=0.97; p=0.00035; Pearson Correlation) h. Correlation of genera abundances between 16S and WES illustrates that samples from same tumor correlate strongly (N=16 tumors). i. Pearson correlation between 16S determined bacterial species and those identified from WES is significantly higher between corresponding samples, than from unrelated samples (Wilcox test, p=0.00016, n=8 cases). Boxplots show IQR (Q1–Q3); line = median; whiskers = most extreme points within 1.5×IQR). 16S IHC staining by RNAscope of selected TBB high (n=4; left) and TBB low (n=4; right) tumors (red: 16S probe; blue DAPI). Note increased bacterial staining in TBB high tumors. Within each cohort, a scrambled 16S probe (red) was utilized as a negative control illustrating specificity of staining. Expression of individual immune related genes and TBB. Note strong inverse relationship (linear regression) with gene markers of adaptive immune cells (CD8A, CD3E, PDCD1, CD19, CD79A) compared with neutrophil markers (NEAT1, NAMPT1, CXCL8) (n=342 cases). b. CD8 T cell abundance determined by IHC and split by TBB status. Increased CD8 T cell staining is noted in the TBB low samples (p < 0.001, N=323, Wilcoxon rank-sum test,). c. Total number of TCR clonotypes per sample is ordered from low to high and colored by TBB status, highlighting expansion of clones in TBB low samples. Boxplot shows IQR (Q1–Q3); line = median; whiskers = most extreme points within 1.5×IQR, N=304 tumors) d. Relative abundance of clones by size shows expansion of small clones in TBB high samples (avelumab high, placebo high), compared to low samples (n=310 patients, avelumab low, placebo low; all comparisons by Wilcoxon test, Yellow NA:NA are those samples for which TCR sequencing was available but TBB was not measured due to lack of WES for that sample). e. Patient TBB is associated with pre-treatment peripheral neutrophil count (R=0.21, p=0.001, N=421) in patients with high TBB but not in patients with low TBB (R=−0.06, p=0.43, Pearson Correlation, shaded region corresponds to 95% CI). Associations between TBB and PFS in patients treated with avelumab in HPV negative patients (left; p = 0.011) and HPV positive patients (right; p=0.077; both log-rank test) b. Multivariate cox proportional hazard model (and 95% CI) in Avelumab treated patients, identifies TBB as associated with PFS after correcting for stratification variables (HR = 2.03 p = 0.002, n=421 patients) c. Comprehensive cut-point analysis illustrates a wide range of TBB cutpoints associate with outcomes, including both the median or 99th% in patients treated with Avelumab (left). Each dot represents a log rank test at a specific cut point, with the y axis showing the p-value for the log-rank test. Every possible cutpoint was tested through the dataset and the summary is plotted. In (right) Placebo + SoC CRT arm TBB is not associated with outcomes regardless of the cutpoint. (N=421 pts) d. Histograms of BLAST E-values for remaining unaligned reads after alignment to GCHR38. Note unaligned reads have significant lower E-values aligning to bacterial genomes rather than alignment to a human genome. e. PFS in Avelumab/CRT arm split by TBB considering only Top 5 genera by abundance and known to be members of the oral microbiome (p=0.012; log-rank test). Schematic showing workflow of elastic net modeling with nested cross-validation. Supplementary Tables 1 and 2. Statistical source data. Statistical source data. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Riaz, N., Alban, T.J., Haddad, R.I. et al. Tumor ecosystem and microbiome features associated with efficacy and resistance to avelumab plus chemoradiotherapy in head and neck cancer. Nat Cancer  (2026). Accepted: 06 October 2025 Published: 02 January 2026 Version of record: 02 January 2026 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative (Nat Cancer) ISSN 2662-1347 (online) © 2026 Springer Nature Limited Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41582-025-01177-5'>Genetic insights from Parkinson disease in African and African admixed populations</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-02 01:28:57
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Emerging genomic studies across Africa are revealing how decades of population bias have constrained understanding of disease mechanisms. New evidence in Parkinson disease genetics illustrates how overlooking Africa's vast genetic diversity not only limits scientific insight but also slows the development of medical and biotechnological innovations that are vital for advancing global health. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Rent or buy this article Prices vary by article type Prices may be subject to local taxes which are calculated during checkout United Nations Department of Economic and Social Affairs, Population Division. World Population Prospects 2024: Online Edition (United Nations, 2024). Fan, S. et al. African evolutionary history inferred from whole genome sequence data of 44 indigenous African populations. Pereira, L., Mutesa, L., Tindana, P. & Ramsay, M. African genetic diversity and adaptation inform a precision medicine agenda. Mills, M. C. & Rahal, C. The GWAS Diversity Monitor tracks diversity by disease in real time. Schumacher-Schuh, A. F. et al. Underrepresented populations in Parkinson's genetics research: current landscape and future directions. Identification of genetic risk loci and causal insights associated with Parkinson's disease in African and African admixed populations: a genome-wide association study. Ojo, O. O. et al. GBA1 rs3115534 is associated with REM sleep behavior disorder in Parkinson's disease in Nigerians. Álvarez Jerez, P. et al. African ancestry neurodegeneration risk variant disrupts an intronic branchpoint in GBA1. Large-scale genetic characterization of Parkinson's disease in the African and African admixed populations. Missing genetic diversity impacts variant prioritisation for rare disorders. Department of Neuromuscular Diseases, UCL Queen Square Institute of Neurology, University College London, London, UK Institute of Global Health and Human Ecology, The American University in Cairo, Cairo, Egypt Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Mie Rizig. are both members of the GP2 programme; however, neither contributed to the data analysis or participated in the writing, reviewing or editing of ref. Rizig, M., Salama, M. Genetic insights from Parkinson disease in African and African admixed populations. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41580-025-00918-0'>Advancing single-cell omics and cell-based therapeutics with quantum computing</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-02 01:15:05
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Molecular Cell Biology (2026)Cite this article The generation of highly accurate models of behaviours of individual cells and cell populations through integration of high-resolution assays with advanced computational tools would transform precision medicine. Recent breakthroughs in single-cell and spatial transcriptomics and multi-omics technologies, coupled with artificial intelligence, are driving rapid progress in model development. Complementing the advances in artificial intelligence, quantum computing is maturing as a novel compute paradigm that may offer potential solutions to overcome the computational bottlenecks inherent to capturing cellular dynamics. In this Roadmap article, we discuss the advancements and challenges in spatiotemporal single-cell analysis, explore the possibility of quantum computing to address the challenges and present a case study on how quantum computing may be integrated into cell-based therapeutics. The specific confluence of quantum and classical computing with high-resolution assays may offer a crucial path towards the generation of transformative models of cellular behaviours and perturbation responses. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Method of the year 2013. Marx, V. Method of the year: spatially resolved transcriptomics. Google Scholar Method of the year 2024: spatial proteomics. The Cancer Genome Atlas Research Network et al. The cancer genome atlas pan-cancer analysis project. Google Scholar de Bruijn, I. et al. Sharing data from the Human Tumor Atlas Network through standards, infrastructure and community engagement. Google Scholar Jain, S. et al. Advances and prospects for the Human BioMolecular Atlas Program (HuBMAP). Cell Biol. Google Scholar Rozenblatt-Rosen, O., Stubbington, M. J., Regev, A. & Teichmann, S. A. The Human Cell Atlas: from vision to reality. Google Scholar Regev, A. et al. The human cell atlas. Google Scholar Zhang, J. et al. Tahoe-100M: a giga-scale single-cell perturbation atlas for context-dependent gene function and cellular modeling. Preprint at bioRxiv https://doi.org/10.1101/2025.02.20.639398 (2025). Klughammer, J. et al. A multi-modal single-cell and spatial expression map of metastatic breast cancer biopsies across clinicopathological features. Google Scholar Mo, C. K. et al. Tumour evolution and microenvironment interactions in 2D and 3D space. Google Scholar Greenbaum, S. et al. A spatially resolved timeline of the human maternal–fetal interface. Google Scholar Ren, J. et al. Spatiotemporally resolved transcriptomics reveals the subcellular RNA kinetic landscape. Google Scholar Wagner, J. et al. A single-cell atlas of the tumor and immune ecosystem of human breast cancer. Google Scholar Paik, D. T., Cho, S., Tian, L., Chang, H. Y. & Wu, J. C. Single-cell RNA sequencing in cardiovascular development, disease and medicine. Google Scholar Segerstolpe, Å et al. Single-cell transcriptome profiling of human pancreatic islets in health and type 2 diabetes. Cell Metab. Google Scholar Li, H. et al. A comprehensive benchmarking with practical guidelines for cellular deconvolution of spatial transcriptomics. Google Scholar Dill, K. A. & MacCallum, J. L. The protein-folding problem, 50 years on. Google Scholar Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Google Scholar Humphreys, I. R. et al. Computed structures of core eukaryotic protein complexes. Google Scholar Bryant, P. et al. Predicting the structure of large protein complexes using AlphaFold and Monte Carlo tree search. Google Scholar Mazurenko, S., Prokop, Z. & Damborsky, J. Machine learning in enzyme engineering. ACS Catal. Google Scholar Dauparas, J. et al. Robust deep learning–based protein sequence design using ProteinMPNN. Google Scholar Zhou, J. & Troyanskaya, O. G. Predicting effects of noncoding variants with deep learning–based sequence model. Google Scholar Greenwald, N. F. et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning. Google Scholar Esteva, A. et al. Dermatologist-level classification of skin cancer with deep neural networks. Google Scholar Ardila, D. et al. End-to-end lung cancer screening with three-dimensional deep learning on low-dose chest computed tomography. Google Scholar & Luo, Y. Statistical and machine learning methods for spatially resolved transcriptomics data analysis. Genome Biol. Google Scholar Kelly, C. J., Karthikesalingam, A., Suleyman, M., Corrado, G. & King, D. Key challenges for delivering clinical impact with artificial intelligence. BMC Med. Google Scholar Greener, J. G., Kandathil, S. M., Moffat, L. & Jones, D. T. A guide to machine learning for biologists. Cell Biol. Google Scholar Biancalani, T. et al. Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram. Google Scholar et al. Batch effects and the effective design of single-cell gene expression studies. Google Scholar Saelens, W., Cannoodt, R., Todorov, H. & Saeys, Y. A comparison of single-cell trajectory inference methods. Google Scholar Dirac, P. A. M. The Principles of Quantum Mechanics (Oxford Univ. Heisenberg, W. in Physicist's Conception of Nature (ed. Heisenberg, W.) (Harcourt, Brace & co., 1958). Levine, H. et al. Parallel implementation of high-fidelity multiqubit gates with neutral atoms. Google Scholar Burkard, G., Ladd, T. D., Pan, A., Nichol, J. M. & Petta, J. R. Semiconductor spin qubits. Google Scholar Conlon, A., Pellegrino, D., Slingerland, J. K., Dooley, S. & Kells, G. Error generation and propagation in Majorana-based topological qubits. Google Scholar Kozhanov, A. et al. Next-generation trapped-ion quantum computing system. In Optica Quantum 2.0 Conference and Exhibition QM3A-2 (Optical Publishing Group, 2023). Bravyi, S., Dial, O., Gambetta, J. M., Gil, D. & Nazario, Z. The future of quantum computing with superconducting qubits. Google Scholar Chamberland, C. et al. Building a fault-tolerant quantum computer using concatenated cat codes. PRX Quantum 3, 010329 (2022). Google Scholar Martinis, J. M. Surface loss calculations and design of a superconducting transmon qubit with tapered wiring. npj Quantum Inf. Chow, J., Dial, O. & Gambetta, J. IBM quantum breaks the 100-qubit processor barrier. IBM Research Blog https://www.ibm.com/quantum/blog/127-qubit-quantum-processor-eagle (2022). Google Quantum, A. I. & Collaborators Quantum error correction below the surface code threshold. Bravyi, S. et al. High-threshold and low-overhead fault-tolerant quantum memory. Google Scholar Cross, A. et al. OpenQASM 3: a broader and deeper quantum assembly language. ACM Trans. Quantum Comput. Google Scholar Jurcevic, P. et al. Demonstration of quantum volume 64 on a superconducting quantum computing system. Quantum Sci. Google Scholar Piveteau, C. & Sutter, D. Circuit knitting with classical communication. IEEE Trans. Google Scholar Patti, T. L., Shehab, O., Najafi, K. & Yelin, S. F. Markov chain Monte Carlo enhanced variational quantum algorithms. Quantum Sci. Google Scholar Van Den Berg, E., Minev, Z. K., Kandala, A. & Temme, K. Probabilistic error cancellation with sparse Pauli–Lindblad models on noisy quantum processors. Google Scholar Kim, Y. et al. Scalable error mitigation for noisy quantum circuits produces competitive expectation values. Google Scholar Caro, M. C. et al. Generalization in quantum machine learning from few training data. Google Scholar Cerezo, M., Verdon, G., Huang, H.-Y., Cincio, L. & Coles, P. J. Challenges and opportunities in quantum machine learning. Google Scholar Abbas, A. et al. Challenges and opportunities in quantum optimization. Google Scholar Krunic, Z., Flöther, F. F., Seegan, G., Earnest-Noble, N. D. & Shehab, O. Quantum kernels for real-world predictions based on electronic health records. IEEE Trans. Quantum Eng. Google Scholar Utro, F., Bose, A., Wang, R., Dubovitskii, V. & Parida, L. A perspective on quantum computing for analyzing cell-cell communication networks. In Conference on Intelligent Systems for Molecular Biology (ISMB, 2024). Doga, H. et al. How can quantum computing be applied in clinical trial design and optimization? Trends Pharmacol. Google Scholar Doga, H. et al. A perspective on protein structure prediction using quantum computers. Theory Comput. Google Scholar Flöther, F. F. et al. How quantum computing can enhance biomarker discovery. Google Scholar Dubovitskii, V., Bose, A., Utro, F. & Pardia, L. On quantum random walks in biomolecular networks. Preprint at https://doi.org/10.48550/arXiv.2506.06514 (2025). Moses, L. & Pachter, L. Museum of spatial transcriptomics. Google Scholar Wang, Z. Cell segmentation for image cytometry: advances, insufficiencies, and challenges. Google Scholar Pang, M., Roy, T. K., Wu, X. & Tan, K. CelloType: a unified model for segmentation and classification of tissue images. Google Scholar Petukhov, V. et al. Cell segmentation in imaging-based spatial transcriptomics. Chen, H., Li, D. & Bar-Joseph, Z. SCS: cell segmentation for high-resolution spatial transcriptomics. Moen, E. et al. Deep learning for cellular image analysis. Walker, B. L., Cang, Z., Ren, H., Bourgain-Chang, E. & Nie, Q. Deciphering tissue structure and function using spatial transcriptomics. Nitzan, M., Karaiskos, N., Friedman, N. & Rajewsky, N. Gene expression cartography. Pesah, A. et al. Absence of barren plateaus in quantum convolutional neural networks. Hur, T., Kim, L. & Park, D. K. Quantum convolutional neural network for classical data classification. Quantum Mach. Born, J. et al. Quantum doubly stochastic transformers. Preprint at https://doi.org/10.48550/arXiv.2504.16275 (2025). Zhao, L., Wan, L. & Luo, M.-X. Quantum algorithm for Markov Random Fields structure learning by information theoretic properties. Khatri, N., Matos, G., Coopmans, L. & Clark, S. Quixer: a quantum transformer model. Preprint at https://doi.org/10.48550/arXiv.2406.04305 (2024). Cherrat, E. A. et al. Quantum vision transformers. Jayakody, M. N., Meena, C. & Pradhan, P. Revisiting one-dimensional discrete-time quantum walks with general coin. Mariella, N. et al. Quantum theory and application of contextual optimal transport. In Proc. 41st International Conference on Machine Learning Article 1416, 34822–34845 (ACM, 2024). Stuart, T. et al. Comprehensive integration of single-cell data. Palla, G. et al. Squidpy: a scalable framework for spatial omics analysis. Dries, R. et al. Giotto: a toolbox for integrative analysis and visualization of spatial expression data. Genome Biol. Li, M. M., Huang, K. & Zitnik, M. Graph representation learning in biomedicine and healthcare. Jaume, G. et al. Quantifying explainers of graph neural networks in computational pathology. In Proc. IEEE/CVF Conference on Computer Vision and Pattern Recognition 8106–8116 (CVPR, 2021). Liu, T. et al. A comprehensive overview of graph neural network-based approaches to clustering for spatial transcriptomics. Article PubMed Li, Y. & Luo, Y. STdGCN: spatial transcriptomic cell-type deconvolution using graph convolutional networks. Genome Biol. Pati, P. et al. Hierarchical graph representations in digital pathology. image Anal. Article PubMed Cook, S. A. The complexity of theorem-proving procedures. Sun, S., Zhu, J. & Zhou, X. Statistical analysis of spatial expression patterns for spatially resolved transcriptomic studies. Govek, K. W., Yamajala, V. S. & Camara, P. G. Clustering-independent analysis of genomic data using spectral simplicial theory. PLoS Comput. Miller, B. F., Bambah-Mukku, D., Dulac, C., Zhuang, X. & Fan, J. Characterizing spatial gene expression heterogeneity in spatially resolved single-cell transcriptomic data with nonuniform cellular densities. Genome Res. Armingol, E., Officer, A., Harismendy, O. & Lewis, N. E. Deciphering cell-cell interactions and communication from gene expression. Arnol, D., Schapiro, D., Bodenmiller, B., Saez-Rodriguez, J. & Stegle, O. Modeling cell-cell interactions from spatial molecular data with spatial variance component analysis. Tanevski, J., Flores, R. O. R., Gabor, A., Schapiro, D. & Saez-Rodriguez, J. Explainable multiview framework for dissecting spatial relationships from highly multiplexed data. Genome Biol. Sahin, M. E. et al. Efficient parameter optimisation for quantum kernel alignment: a sub-sampling approach in variational training. Ray, A. et al. Hybrid quantum–classical graph neural networks for tumor classification in digital pathology. 2024 IEEE Int. Quantum Comput. Mirzasoleiman, B., Bilmes, J. & Leskovec, J. Coresets for data-efficient training of machine learning models. In Proc. 37th International Conference on Machine Learning Vol. Layden, D. et al. Quantum-enhanced Markov chain Monte Carlo. Macaluso, A., Clissa, L., Lodi, S. & Sartori, C. An efficient quantum algorithm for ensemble classification using bagging. IET Quantum Commun. Rhrissorrakrai, K. et al. Quantum ensembling methods for healthcare and life science. Preprint at https://doi.org/10.48550/arXiv.2506.02213 (2025). & Dong, D. Supervised-learning guarantee for quantum AdaBoost. Berry, D. W. et al. Analyzing prospects for quantum advantage in topological data analysis. PRX Quantum 5, 010319 (2024). Hayakawa, R. Quantum algorithm for persistent Betti numbers and topological data analysis. Lloyd, S., Garnerone, S. & Zanardi, P. Quantum algorithms for topological and geometric analysis of data. Ding, J., Sharon, N. & Bar-Joseph, Z. Temporal modelling using single-cell transcriptomics. La Manno, G. et al. RNA velocity of single cells. Trapnell, C. et al. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Bergen, V., Lange, M., Peidli, S., Wolf, F. A. & Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. Teschendorff, A. E. & Enver, T. Single-cell entropy for accurate estimation of differentiation potency from a cell's transcriptome. Wolf, F. A. et al. PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells. Genome Biol. Ji, Z. & Ji, H. TSCAN: pseudo-time reconstruction and evaluation in single-cell RNA-seq analysis. Nucleic Acids Res. Cao, J. et al. The single-cell transcriptional landscape of mammalian organogenesis. Schiebinger, G. et al. Optimal-transport analysis of single-cell gene expression identifies developmental trajectories in reprogramming. Tong, A., Huang, J., Wolf, G., van Dijk, D. & Krishnaswamy, S. TrajectoryNet: a dynamic optimal transport network for modeling cellular dynamics. Chen, R. T., Rubanova, Y., Bettencourt, J. & Duvenaud, D. K. Neural ordinary differential equations. In Proc. 32nd International Conference on Neural Information Processing System 6572–6583 (ACM, 2018). Pham, D. et al. Robust mapping of spatiotemporal trajectories and cell-cell interactions in healthy and diseased tissues. Ji, Y., Lotfollahi, M., Wolf, F. A. & Theis, F. J. Machine learning for perturbational single-cell omics. Cell Syst. Zhang, J., Larschan, E., Bigness, J. & Singh, R. scNODE: generative model for temporal single cell transcriptomic data prediction. Lin, C. & Bar-Joseph, Z. Continuous-state HMMs for modeling time-series single-cell RNA-seq data. Niraula, D., El Naqa, I., Tuszynski, J. & Gatenby, R. A. Modeling non-genetic information dynamics in cells using reservoir computing. Hadaeghi, F. et al. Spatio-temporal feature learning with reservoir computing for T-cell segmentation in live-cell Ca2+ fluorescence microscopy. Lotfollahi, M., Wolf, F. A. & Theis, F. J. scGen predicts single-cell perturbation responses. Rood, J. E., Hupalowska, A. & Regev, A. Toward a foundation model of causal cell and tissue biology with a perturbation cell and tissue atlas. Jaitin, D. A. et al. Dissecting immune circuits by linking CRISPR-pooled screens with single-cell RNA-Seq. McFarland, J. M. et al. Multiplexed single-cell transcriptional response profiling to define cancer vulnerabilities and therapeutic mechanism of action. Frangieh, C. J. et al. Multimodal pooled Perturb-CITE-seq screens in patient models define mechanisms of cancer immune evasion. Xie, S., Duan, J., Li, B., Zhou, P. & Hon, G. C. Multiplexed engineering and analysis of combinatorial enhancer activity in single cells. Gavriilidis, G. I., Vasileiou, V., Orfanou, A., Ishaque, N. & Psomopoulos, F. A mini-review on perturbation modelling across single-cell omic modalities. Nicol, P. B. et al. Robust identification of perturbed cell types in single-cell RNA-seq data. Dann, E., Henderson, N. C., Teichmann, S. A., Morgan, M. D. & Marioni, J. C. Differential abundance testing on single-cell data using k-nearest neighbor graphs. Kamimoto, K. et al. Dissecting cell identity via network inference and in silico gene perturbation. Marouf, M. et al. Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks. Kingma, D. P. Auto-encoding variational bayes. Preprint at https://doi.org/10.48550/arXiv.1312.6114 (2013). Szałata, A. et al. Transformers in single-cell omics: a review and new perspectives. Article PubMed Bunne, C. et al. Learning single-cell perturbation responses using neural optimal transport. Bunne, C., Schiebinger, G., Krause, A., Regev, A. & Cuturi, M. Optimal transport for single-cell and spatial omics. Methods Prim. Cui, H. et al. scGPT: toward building a foundation model for single-cell multi-omics using generative AI. Fu, X. et al. A foundation model of transcription across human cell types. Hao, M. et al. Large-scale foundation model on single-cell transcriptomics. Heimberg, G. et al. A cell atlas foundation model for scalable search of similar human cells. Ghazanfar, S. et al. Investigating higher-order interactions in single-cell data with scHOT. Piatkowski, N. & Zoufal, C. Quantum circuits for discrete graphical models. Quantum Mach. Ghosh, S., Opala, A., Matuszewski, M., Paterek, T. & Liew, T. C. Quantum reservoir processing. NPJ Quantum Inf. Lukoševičius, M. A practical guide to applying echo state networks. Neural Netw. Tanaka, G. et al. Recent advances in physical reservoir computing: a review. Neural Netw. Article PubMed Zhu, C., Ehlers, P. J., Nurdin, H. I. & Soh, D. Practical few-atom quantum reservoir computing. Situ, H., He, Z., Wang, Y., Li, L. & Zheng, S. Quantum generative adversarial network for generating discrete distribution. Zoufal, C., Lucchi, A. & Woerner, S. Quantum generative adversarial networks for learning and loading random distributions. NPJ Quantum Inf. et al. Exploring the advantages of quantum generative adversarial networks in generative chemistry. Wan, K. H., Dahlsten, O., Kristjánsson, H., Gardner, R. & Kim, M. Quantum generalisation of feedforward neural networks. NPJ Quantum Inf. Romero, J., Olson, J. P. & Aspuru-Guzik, A. Quantum autoencoders for efficient compression of quantum data. Quantum Sci. Khoshaman, A. et al. Quantum variational autoencoder. Quantum Sci. Erbe, R., Stein-O'Brien, G. & Fertig, E. J. Transcriptomic forecasting with neural ordinary differential equations. Choi, M., Flam-Shepherd, D., Kyaw, T. H. & Aspuru-Guzik, A. Learning quantum dynamics with latent neural ordinary differential equations. Stassen, S. V., Yip, G. G. K., Wong, K. K. Y., Ho, J. W. K. & Tsia, K. K. Generalized and scalable trajectory inference in single-cell omics data with VIA. Albrecht, B. et al. Quantum feature maps for graph machine learning on a neutral atom quantum processor. Baysoy, A., Bai, Z., Satija, R. & Fan, R. The technological landscape and applications of single-cell multi-omics. Cell Biol. Cao, K., Bai, X., Hong, Y. & Wan, L. Unsupervised topological alignment for single-cell multi-omics integration. & Li, S. C. Probabilistic tensor decomposition extracts better latent embeddings from single-cell multiomic data. Nucleic Acids Res. Cao, Z. J. & Gao, G. Multi-omics single-cell data integration and regulatory inference with graph-linked embedding. Kartha, V. K. et al. Functional inference of gene regulation using single-cell multi-omics. Cell Genom. Szklarczyk, D. et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. Ma, A. et al. Single-cell biological network inference using a heterogeneous graph transformer. Barabasi, A. L., Gulbahce, N. & Loscalzo, J. Network medicine: a network-based approach to human disease. Luck, K. et al. A reference map of the human binary protein interactome. Maniscalco, S. et al. Quantum network medicine: rethinking medicine with network science and quantum algorithms. Preprint at https://doi.org/10.48550/arXiv.2206.12405 (2022). Saarinen, H., Goldsmith, M., Wang, R.-S., Loscalzo, J. & Maniscalco, S. Disease gene prioritization with quantum walks. Bose, A., Platt, D. E., Haiminen, N. & Parida, L. CuNA: cumulant-based network analysis of genotype-phenotype associations in Parkinson's disease. Preprint at medRxiv https://doi.org/10.1101/2021.08.02.21261457 (2021). Parida, L. & Ramakrishnan, N. Redescription mining: structure theory and algorithms. In Proc. 20th National Conference on Artificial Intelligence Vol. Bose, A. & Parida, L. Generating cumulant-based risk scores for diseases. US Patent Application No. Gurnari, D. et al. Probing omics data via harmonic persistent homology. Percus, J. K. Correlation inequalities for Ising spin lattices. Platt, D. E., Basu, S., Zalloua, P. A. & Parida, L. Characterizing redescriptions using persistent homology to isolate genetic pathways contributing to pathogenesis. BMC Syst. Gyurik, C., Cade, C. & Dunjko, V. Towards quantum advantage via topological data analysis. Cui, L., Guo, G., Ng, M. K., Zou, Q. & Qiu, Y. GSTRPCA: irregular tensor singular value decomposition for single-cell multi-omics data clustering. Hastings, M. B. Classical and quantum algorithms for tensor principal component analysis. Burch, M. et al. Towards quantum tensor decomposition in biomedical applications. Preprint at https://doi.org/10.48550/arXiv.2502.13140 (2025). Zhou, L., Basso, J. & Mei, S. Statistical estimation in the spiked tensor model via the quantum approximate optimization algorithm. Neural Inf. Zhu, X., Meng, S., Li, G., Wang, J. & Peng, X. AGImpute: imputation of scRNA-seq data based on a hybrid GAN with dropouts identification. Xu, Y. et al. scIGANs: single-cell RNA-seq imputation using generative adversarial networks. Nucleic Acids Res. Zeng, J., Wu, Y., Liu, J.-G., Wang, L. & Hu, J. Learning and inference on generative adversarial quantum circuits. Li, J., Topaloglu, R. O. & Ghosh, S. Quantum generative models for small molecule drug discovery. IEEE Trans. Quantum Eng. Huang, H.-L. et al. Experimental quantum generative adversarial networks for image generation. Abbas, A. et al. The power of quantum neural networks. Article PubMed Peters, R. et al. Quantum Convolutional HLA Immunogenic Peptide Prediction (Q-CHIPP): next-generation neoantigen prediction with quantum neural networks. Preprint at bioRxiv https://doi.org/10.1101/2025.07.29.667313 (2025). Shende, V. V., Bullock, S. S. & Markov, I. L. Synthesis of quantum-logic circuits. IEEE Trans. Aided Des. Circuits Syst. Plesch, M. & Brukner, Č Quantum-state preparation with universal gate decompositions. Zhang, X.-M., Li, T. & Yuan, X. Quantum state preparation with optimal circuit depth: implementations and applications. Basu, S. et al. Towards quantum-enabled cell-centric therapeutics. Preprint at https://doi.org/10.48550/arXiv.2307.05734 (2023). Moffat, J. G., Vincent, F., Lee, J. A., Eder, J. & Prunotto, M. Opportunities and challenges in phenotypic drug discovery: an industry perspective. Lin, A. et al. Off-target toxicity is a common mechanism of action of cancer drugs undergoing clinical trials. Sams-Dodd, F. Target-based drug discovery: is something wrong? et al. Power of data in quantum machine learning. Utro, F. et al. Enhanced prediction of CAR T-Cell cytotoxicity with quantum-kernel methods. Preprint at https://doi.org/10.48550/arXiv.2507.22710 (2025). Gambetta, J. Expanding the IBM Quantum roadmap to anticipate the future of quantum-centric supercomputing. IBM Research Blog https://www.ibm.com/quantum/blog/ibm-quantum-roadmap-2025 (2022). AbuGhanem, M. IBM quantum computers: evolution, performance, and future directions. Yoder, T. J. et al. Tour de gross: a modular quantum computer based on bivariate bicycle codes. Preprint at https://doi.org/10.48550/arXiv.2506.03094 (2025). Müller, T. et al. Improved belief propagation is sufficient for real-time decoding of quantum memory. Preprint at https://doi.org/10.48550/arXiv.2506.01779 (2025). Mohseni, M. et al. How to build a quantum supercomputer: scaling challenges and opportunities. Preprint at https://doi.org/10.48550/arXiv.2411.10406 (2024). Robledo-Moreno, J. et al. Chemistry beyond the scale of exact diagonalization on a quantum-centric supercomputer. Ding, Q.-M., Huang, Y.-M. & Yuan, X. Molecular docking via quantum approximate optimization algorithm. Chandarana, P., Hegade, N. N., Montalban, I., Solano, E. & Chen, X. Digitized counterdiabatic quantum algorithm for protein folding. Lacroix, N. et al. Improving the performance of deep quantum optimization algorithms with continuous gate sets. PRX Quantum 1, 020304 (2020). Bose, A., Doga, H., Utro, F. & Parida, L. Quantum-enabled multi-omics analysis. In Conference on Intelligent Systems for Molecular Biology (2024). Ghazi Vakili, M. et al. Quantum-computing-enhanced algorithm unveils potential KRAS inhibitors. Article PubMed Larocca, M. et al. Barren plateaus in variational quantum computing. Du, Y., Hsieh, M.-H., Liu, T. & Tao, D. Expressive power of parametrized quantum circuits. Arrazola, J. M., Delgado, A., Bardhan, B. R. & Lloyd, S. Quantum-inspired algorithms in practice. Tang, E. Dequantizing algorithms to understand quantum advantage in machine learning. Chia, N.-H. et al. Sampling-based sublinear low-rank matrix arithmetic framework for dequantizing quantum machine learning. Koch, T. et al. Quantum optimization benchmark library – the intractable decathlon. Preprint at https://doi.org/10.48550/arXiv.2504.03832 (2025). Amitay, Y. et al. CellSighter: a neural network to classify cells in highly multiplexed images. Angermueller, C., Lee, H. J., Reik, W. & Stegle, O. DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning. Genome Biol. Sidhom, J.-W., Larman, H. B., Pardoll, D. M. & Baras, A. S. DeepTCR is a deep learning framework for revealing sequence concepts within T-cell repertoires. Yu, Z., Liu, F. & Li, Y. scTCA: a hybrid Transformer-CNN architecture for imputation and denoising of scDNA-seq data. De Waele, G., Clauwaert, J., Menschaert, G. & Waegeman, W. CpG transformer for imputation of single-cell methylomes. Article PubMed Zhou, J. et al. Robust single-cell Hi-C clustering by convolution-and random-walk-based imputation. Natl Acad. Jin, S. et al. Inference and analysis of cell–cell communication using CellChat. Cole, S., Eckstein, M., Friedland, S. & Życzkowski, K. On quantum optimal transport. Lange, M. et al. Mapping lineage-traced cells across time points with moslin. Genome Biol. Forrow, A. & Schiebinger, G. LineageOT is a unified framework for lineage tracing and trajectory inference. Cang, Z. et al. Screening cell–cell communication in spatial transcriptomics via collective optimal transport. Cao, K., Gong, Q., Hong, Y. & Wan, L. A unified computational framework for single-cell data integration with optimal transport. Verdon, G. et al. Quantum graph neural networks. Preprint at https://doi.org/10.48550/arXiv.1909.12264 (2019). Biswas, B., Kumar, N., Sugimoto, M. & Hoque, M. A. scHD4E: Novel ensemble learning-based differential expression analysis method for single-cell RNA-sequencing data. Mohammed, A. & Kora, R. A comprehensive review on ensemble deep learning: opportunities and challenges. King Saud. Poggiali, A., Berti, A., Bernasconi, A., Del Corso, G. M. & Guidotti, R. Quantum clustering with k-Means: a hybrid approach. Roman-Vicharra, C. & Cai, J. J. Quantum gene regulatory networks. NPJ Quantum Inf. Loers, J. U. & Vermeirssen, V. A single-cell multimodal view on gene regulatory network inference from transcriptomics and chromatin accessibility data. Krovi, H. Improved quantum algorithms for linear and nonlinear differential equations. Aubin-Frankowski, P.-C. & Vert, J.-P. Gene regulation inference from single-cell RNA-seq data with linear differential equations and velocity inference. Ghosh, A., Behl, H., Dupont, E., Torr, P. & Namboodiri, V. Steer: simple temporal regularization for neural ode. Neural Inf. Sha, Y., Qiu, Y., Zhou, P. & Nie, Q. Reconstructing growth and dynamic trajectories from single-cell transcriptomics data. Article PubMed Song, T., Broadbent, C. & Kuang, R. GNTD: reconstructing spatial transcriptomes with graph-guided neural tensor decomposition informed by spatial and functional relations. Zinati, Y., Takiddeen, A. & Emad, A. GRouNdGAN: GRN-guided simulation of single-cell RNA-seq data using causal generative adversarial networks. Huang, Z., Wang, J., Lu, X., Mohd Zain, A. & Yu, G. scGGAN: single-cell RNA-seq imputation by graph-based generative adversarial network. Download references IBM Research, Yorktown Heights, NY, USA Aritra Bose, Kahn Rhrissorrakrai, Filippo Utro, Laxmi Parida, Aldo Guzman-Saenz, Joseph A. Morrone & Daniel Platt Purdue University, West Lafayette, IN, USA Saugata Basu IBM Research, Zurich, Switzerland Jannis Born IBM Research, San Jose, CA, USA Sara Capponi Children's Hospital of Philadelphia, Philadelphia, PA, USA Dimitra Chalkia Center for Immunotherapy and Precision-Immuno-Oncology, Cleveland Clinic, Cleveland, OH, USA Timothy A. Chan National Center for Regenerative Medicine, Cleveland Clinic, Cleveland, OH, USA Timothy A. Chan IBM Quantum, San Jose, CA, USA Hakan Doga, Tanvi Gujarati, Gavin O. Jones, Meltem Tolunay & Jeannette M. Garcia QuantumBasel, uptownBasel Infinity Corp., Arlesheim, Switzerland Frederik F. Flöther Broad Institute of MIT and Harvard, Cambridge, MA, USA Gad Getz Massachusetts General Hospital Cancer, Boston, MA, USA Gad Getz Department of Pathology, Harvard Medical School, Boston, MA, USA Gad Getz Algorithmiq Ltd., Helsinki, Finland Mark Goldsmith, Stefan Knecht & Sabrina Maniscalco Athos Therapeutics Inc, Los Angeles, CA, USA Dimitrios Iliopoulos IBM Research, Gurugram, India Dhiraj Madan IBM Quantum, Dublin, Ireland Nicola Mariella & Sergiy Zhuk Q-Ctrl, Cambridge, MA, USA Khadijeh Najafi Johnson and Johnson, Zurich, Switzerland Pushpak Pati University of Lausanne, Lausanne, Switzerland Maria Anna Rapsomaniki IBM Quantum, Bengaluru, India Anupama Ray IBM Quantum, Yorktown Heights, NY, USA Omar Shehab IBM Quantum, Zurich, Switzerland Ivano Tavernelli & Stefan Woerner Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors contributed equally to all aspects of the article. Correspondence to Laxmi Parida. The authors declare no competing interests. Nature Reviews Molecular Cell Biology thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. A parameterized quantum circuit used to approximate a target state or operation. In variational algorithms, its design and initialization crucially affect the accuracy and efficiency of the solution. The quantum analogue of random walk, in which the evolution of a quantum system over a graph is continuous in time offering advantages in exploring complex networks and solving combinatorial problems. Statistical measures that generalize moments to quantify higher-order correlations and dependencies among random variables. They isolate intrinsic multiway interactions, capturing nonlinear and non-Gaussian structure that ordinary moments or pairwise correlations cannot represent. The exponential increase in data sparsity and computational complexity as the number of features grow. This phenomenon makes it harder for algorithms to detect meaningful patterns and they often require vastly more data for reliable analysis. A quantum circuit that incorporates classical processing within the coherence time of the qubits permitting mid-circuit measurements to enable feedforward operations such as using measured values to determine the next gates to be applied. A machine learning technique that aggregates multiple individual models (known as ‘weak learners') to produce a strong overall model with high prediction accuracy and robustness. A uniquely quantum phenomenon, in which the states of two or more particles become interdependent; consequently measurement of one entangled particle instantaneously affects the state of the other particle. A performance improvement of an algorithm whereby running time decreases exponentially compared to the state-of-the-art algorithmic benchmark. Accuracies in quantum operations that create entanglement between qubits in a 2-qubit gate. If reduced, the effectiveness of quantum computations is decreased, and the complexity of solved problems is more limited. A class of neural networks designed for graph-structured data, which aggregate and update node features based on neighbouring nodes while learning both local and global graph properties. A class of algorithms that use kernel functions, enabling them to transform data features to a high-dimensional, implicit feature space without exactly computing coordinates in the transformed feature space. An unsupervised algorithm that partitions data into k clusters by assigning each point to the nearest centroid. k-means iteratively minimizes within-cluster variance and is widely used in unsupervised learning. A matrix representation of graph connectivity that combines degree and adjacency matrices of the graph, enabling spectral clustering and dimensionality reduction of graphs. A higher-level abstraction of qubits used in a fault-tolerant quantum device where a single qubit is encoded using a collection of physical qubits while protecting quantum information from errors. Qubits encoding quantum information in the internal states of individual, electrically neutral atoms held in optical or magnetic traps, offering long coherence times and scalability for quantum computing applications. A class of computational problems that are at least as hard as problems in the nondeterministic polynomial complexity class, meaning that finding an efficient solution for them would imply efficient solutions for all problem in nondeterministic polynomial. A linear algebraic method to factor a data matrix V to two lower-rank non-negative matrices W and H representing parts of the original data matrix. A mathematical framework for finding the most cost-effective map from one probability distribution to another. A quantum algorithm that produces approximate solutions for combinatorial optimization problems. Structured sequences of quantum gates operating on qubits to perform quantum computation. The quantum analogue of convolutional neural networks, in which quantum circuits perform convolution and pooling operations; it efficiently extracts hierarchical features from quantum data and has applications in quantum state classification and error correction. An emerging interdisciplinary field that leverages quantum computing and network science to analyse complex biological systems by applying quantum-enhanced techniques to biological interactions and reveal insights into disease mechanism and biomarker discovery. A quantum machine learning model that utilizes parameterized quantum circuits inspired by classical neural networks. The basic units of quantum information, analogous to classical bits. Unlike bits, qubits exist in a combination or superposition of 0 and 1 simultaneously, providing the foundation for quantum algorithms that can process information leveraging fundamentals of quantum mechanics. A stochastic process in which a walker on a graph starts at a vertex and repeatedly moves to one of its neighbouring vertices, chosen at random, generating a path through the vertices. Refers to problems that are highly constrained in the number of samples, particularly with respect to the size of the feature space. Qubits storing quantum information in the intrinsic spin states of particles like electrons or nuclei, often implemented in semiconductor quantum dots and controlled through magnetic or electric fields for quantum operations. Qubits that can exist in multiple states simultaneously based on superconducting circuits operating at extremely low temperatures to minimize thermal noise and maintain coherence, enabling fast gate operations for high-speed quantum computations. A fundamental principle in quantum mechanics stating that a quantum system can exist in multiple states simultaneously. A generalization of matrix factorization to multidimensional arrays or tensors, decomposed into latent lower-dimensional factors that capture key patterns in complex data. A mathematical method rooted in algebraic topology that analyses the inherent structure of data to reveal clusters, holes and voids in high-dimensional datasets that conventional data analysis techniques typically overlook. Quantum bits encoded in non-local properties of topological quantum systems, making them inherently resistant to local noise and decoherence. A deep learning architecture that uses self-attention mechanisms to capture long-range dependencies within a sequence and to draw inference. Qubits that encode information in the electronic and nuclear spin states of individual ions, controlled using electromagnetic fields enabling long coherence times, making them suitable for high-accuracy quantum operations. A quantum machine learning model that uses parameterized quantum circuits to classify data, with parameters optimized through a classical optimization method to minimize a cost function. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Bose, A., Rhrissorrakrai, K., Utro, F. et al. Advancing single-cell omics and cell-based therapeutics with quantum computing. Nat Rev Mol Cell Biol  (2026). Download citation Accepted: 13 October 2025 Published: 02 January 2026 Version of record: 02 January 2026 DOI: https://doi.org/10.1038/s41580-025-00918-0 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Molecular Cell Biology (Nat Rev Mol Cell Biol) ISSN 1471-0080 (online) ISSN 1471-0072 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41579-025-01266-8'>Keystone concept revisited: insights into microbial community dynamics and control</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-02 01:01:02
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Microbiology (2026)Cite this article Different definitions of keystone taxa and functions agree that they have an outsized role in maintaining community composition, and thus, the keystone concept continues to attract attention even 50 years after its introduction. In this Perspective, we base our definition of microbial keystones on the original one to explore its implications and limitations. We review different mechanisms behind keystoneness, cover the strengths and weaknesses of current keystone prediction methods and present findings on keystones discovered in recent experiments. In addition, we suggest a new prediction method for keystones based on metabolic modelling. Finally, we discuss the role of keystones in community control strategies. Overall, the development of new prediction methods and the insights from recent experiments illustrate the continued relevance of the keystone concept for microbial communities. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $259.00 per year only $21.58 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Paine, R. T. A note on trophic complexity and community stability. Google Scholar Power, M. E. et al. Challenges in the quest for keystones. Google Scholar Dayton, P. & Hessler, R. R. Role of biological disturbance in maintaining diversity in the deep sea. Deep Sea Res. Google Scholar Hale, S. L. & Koprowski, J. L. Ecosystem-level effects of keystone species reintroduction: a literature review. Google Scholar Banerjee, S., Schlaeppi, K. & van der Heijden, M. G. A. Keystone taxa as drivers of microbiome structure and functioning. Google Scholar Hastings, A. Transients: the key to long-term ecological understanding? Trends Ecol. Article PubMed Google Scholar Becks, L., Hilker, F. M., Malchow, H., Jürgens, K. & Arndt, H. Experimental demonstration of chaos in a microbial food web. Google Scholar Benincà, E. et al. Chaos in a long-term experiment with a plankton community. Article PubMed Google Scholar Rogers, T. L., Johnson, B. J. & Munch, S. B. Chaos is not rare in natural ecosystems. Article PubMed Google Scholar Wang, X.-W. et al. Identifying keystone species in microbial communities using deep learning. Google Scholar Gibson, T. E., Bashan, A., Cao, H.-T., Weiss, S. T. & Liu, Y.-Y. On the origins and control of community types in the human microbiome. PLoS Comput. Google Scholar Lahti, L., Salojärvi, J., Salonen, A., Scheffer, M. & Vos, W. M. D. Tipping elements in the human intestinal ecosystem. Google Scholar Gonze, D., Lahti, L., Raes, J. & Faust, K. Multi-stability and the origin of microbial community types. ISME J. Google Scholar Amit, G. & Bashan, A. Top-down identification of keystone taxa in the microbiome. France, M., Alizadeh, M., Brown, S., Ma, B. & Ravel, J. Towards a deeper understanding of the vaginal microbiota. Gajer, P. et al. Temporal dynamics of the human vaginal microbiota. Agler, M. T. et al. Microbial hub taxa link host and abiotic factors to plant microbiome variation. PLoS Biol. Hajishengallis, G., Darveau, R. P. & Curtis, M. A. The keystone pathogen hypothesis. Rahman, S. A. & Schomburg, D. Observing local and global properties of metabolic pathways: ‘load points' and ‘choke points' in the metabolic networks. Roume, H. et al. Comparative integrated omics: identification of key functionalities in microbial community-wide metabolic networks. npj Biofilms Microbiomes 1, 15007 (2015). Goyal, A., Dubinkina, V. & Maslov, S. Multiple stable states in microbial communities explained by the stable marriage problem. ISME J. Thomas, R. & D'Ari, R. Biological Feedback (CRC Press, 1990). Enke, T. N. et al. Modular assembly of polysaccharide-degrading marine microbial communities. Hu, M. et al. Microbial diversity and keystone species drive soil nutrient cycling and multifunctionality following mangrove restoration. Louis, P., Duncan, S., Sheridan, P., Walker, A. & Flint, H. J. Microbial lactate utilisation and the stability of the gut microbiome. Gut Microbiome 3, e3 (2022). Wang, S. P. et al. Pivotal roles for pH, lactate, and lactate-utilizing bacteria in the stability of a human colonic microbial ecosystem. Matchado, M. S. et al. Network analysis methods for studying microbial communities: a mini review. Röttjers, L. & Faust, K. From hairballs to hypotheses — biological insights from microbial networks. FEMS Microbiol. Zhou, J., Deng, Y., Luo, F., He, Z. & Yang, Y. Phylogenetic molecular ecological network of soil microbial communities in response to elevated CO2. Hubbell, S. P. The Unified Neutral Theory of Biodiversity and Biogeography (Princeton Univ. Rosindell, J., Hubbell, S. P. & Etienne, R. S. The unified neutral theory of biodiversity and biogeography at age ten. Trends Ecol. Article PubMed Hirano, H. & Takemoto, K. Difficulty in inferring microbial community structure based on co-occurrence network approaches. BMC Bioinformatics 20, 329 (2019). Weiss, S. et al. Correlation detection strategies in microbial data sets vary widely in sensitivity and precision. ISME J. Wilmers, C. C. & Schmitz, O. J. Effects of gray wolf-induced trophic cascades on ecosystem carbon cycling. Berry, D. & Widder, S. Deciphering microbial interactions and detecting keystone species with co-occurrence networks. Dubinkina, V., Fridman, Y., Pandey, P. P. & Maslov, S. Multistability and regime shifts in microbial communities explained by competition for essential nutrients. Muller, E. et al. Using metabolic networks to resolve ecological properties of microbiomes. Quinn-Bohmann, N., Carr, A. V., Diener, C. & Gibbons, S. M. Moving from genome-scale to community-scale metabolic models for the human gut microbiome. Croes, D., Couche, F., Wodak, S. J. & Helden, J. V. Metabolic PathFinding: inferring relevant pathways in biochemical networks. Nucleic Acids Res. Faust, K., Croes, D. & van Helden, J. Metabolic pathfinding using RPAIR annotation. Fisher, C. K. & Mehta, P. Identifying keystone species in the human gut microbiome from metagenomic timeseries using sparse linear regression. PLoS ONE 9, e102451 (2014). Friedman, J., Higgins, L. M. & Gore, J. Community structure follows simple assembly rules in microbial microcosms. Estrela, S. et al. Functional attractors in microbial community assembly. Cell Syst. Khazaei, T. et al. Metabolic multistability and hysteresis in a model aerobe–anaerobe microbiome community. Lopes, W., Amor, D. R. & Gore, J. Cooperative growth in microbial communities is a driver of multistability. Orth, J. D., Thiele, I. & Palsson, B. Ø. What is flux balance analysis? Thiele, I. & Palsson, B. Ø. A protocol for generating a high-quality genome-scale metabolic reconstruction. Carr, R. & Borenstein, E. NetSeed: a network-based reverse-ecology tool for calculating the metabolic interface of an organism with its environment. Belcour, A. et al. Metage2Metabo, microbiota-scale metabolic complementarity for the identification of key species. Garza, D., Gonze, D., Zafeiropoulos, H., Liu, B. & Faust, K. Metabolic models of human gut microbiota: advances and challenges. Cell Syst. Papp, B., Pál, C. & Hurst, L. D. Metabolic network analysis of the causes and evolution of enzyme dispensability in yeast. Venturelli, O. S. et al. Deciphering microbial interactions in synthetic human gut microbiome communities. Weiss, A. S. et al. In vitro interaction network of a synthetic gut bacterial community. ISME J. Carlström, C. I. et al. Synthetic microbiota reveal priority effects and keystone strains in the Arabidopsis phyllosphere. Hessler, T. et al. Vitamin interdependencies predicted by metagenomics-informed network analyses and validated in microbial community microcosms. Niu, B., Paulson, J. N., Zheng, X. & Kolter, R. Simplified and representative bacterial community of maize roots. Natl Acad. Brachi, B. et al. Plant genetic effects on microbial hubs impact host fitness in repeated field trials. Natl Acad. Bauchinger, F., Seki, D. & Berry, D. Characteristics of putative keystones in the healthy adult human gut microbiota as determined by correlation network analysis. Arumugam, M. et al. Enterotypes of the human gut microbiome. Costea, P. I. et al. Enterotypes in the landscape of gut microbial community composition. Heinken, A. et al. Genome-scale metabolic reconstruction of 7,302 human microorganisms for personalized medicine. Hertel, J. et al. Integrated analyses of microbiome and longitudinal metabolome data reveal microbial–host interactions on sulfur metabolism in Parkinson's disease. et al. Genetic manipulation of gut microbes enables single-gene interrogation in a complex microbiome. Tanaka, K., Collins, S., Polkoff, K. & Fellner, V. Inhibiting methanogenesis by targeting thermodynamics and enzymatic reactions in mixed cultures of rumen microbes in vitro. Baker, N. R. et al. Nutrient and moisture limitations reveal keystone metabolites linking rhizosphere metabolomes and microbiomes. Natl Acad. Clark, R. L. et al. Design of synthetic human gut microbiome assembly and butyrate production. Oliveira, R. A. et al. Klebsiella michiganensis transmission enhances resistance to Enterobacteriaceae gut invasion by nutrition competition. Angulo, M. T., Moog, C. H. & Liu, Y.-Y. A theoretical framework for controlling complex microbial communities. Amor, D. R., Ratzke, C. & Gore, J. Transient invaders can induce shifts between alternative stable states of microbial communities. Rawstern, A. H., Hernandez, D. J. & Afkhami, M. E. Central taxa are keystone microbes during early succession. Article PubMed Keshteli, A. H., Millan, B. & Madsen, K. L. Pretreatment with antibiotics may enhance the efficacy of fecal microbiota transplantation in ulcerative colitis: a meta-analysis. Mucosal Immunol. Cottee-Jones, H. E. W. & Whittaker, R. J. P erspective: the keystone species concept: a critical appraisal. Bashan, A. et al. Universality of human microbial dynamics. Diaz-Colunga, J., Skwara, A. B., Vila, J. C. C., Bajic, D. J. & Sanchez, A. Global epistasis and the emergence of function in microbial consortia. Gotelli, N. J., Ulrich, W. & Maestre, F. T. Randomization tests for quantifying species importance to ecosystem function. Methods Ecol. Diaz-Colunga, J., Catalan, P., Roman, M. S., Arrabal, A. & Sanchez, A. Full factorial construction of synthetic microbial communities. eLife https://doi.org/10.7554/eLife.101906.1.sa3 (2024). Kehe, J. et al. Massively parallel screening of synthetic microbial communities. Natl Acad. Majzoub, M. E. et al. Refining microbial community metabolic models derived from metagenomics using reference-based taxonomic profiling. Henson, M. A. & Hanly, T. J. Dynamic flux balance analysis for synthetic microbial communities. Garza, D. R. et al. Nutrition or nature: using elementary flux modes to disentangle the complex forces shaping prokaryote pan-genomes. BMC Ecol. Mizrahi, S. P., Lee, H., Goyal, A., Owen, E. & Gore, J. Structured interactions explain the absence of keystone species in synthetic microcosms. ISME J. Faure, L., Mollet, B., Liebermeister, W. & Faulon, J.-L. A neural-mechanistic hybrid approach improving the predictive power of genome-scale metabolic models. Download references The authors are grateful to all current and former members of the Lab of Microbial Systems Biology for stimulating discussions. The authors also thank the reviewers for their thoughtful and stimulating feedback. Department of Microbiology, Immunology and Transplantation, Rega Institute for Medical Research, Laboratory of Molecular Bacteriology, KU Leuven, Leuven, Belgium Daniel Rios Garza & Karoline Faust Université Paris-Saclay, INRAE, PROSE, Antony, France Daniel Rios Garza Unité de Chronobiologie Théorique, Faculté des Sciences, Université Libre de Bruxelles, Bruxelles, Belgium Didier Gonze Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar wrote the text, D.R.G. designed Box 1, D.G. designed the figures and all authors discussed the content, reviewed and edited the manuscript. Correspondence to Karoline Faust. The authors declare no competing interests. Nature Reviews Microbiology thanks Amir Bashan, Yang-Yu Liu and Ramiro Logares for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Garza, D.R., Gonze, D. & Faust, K. Keystone concept revisited: insights into microbial community dynamics and control. Nat Rev Microbiol  (2026). Download citation Accepted: 11 November 2025 Published: 02 January 2026 Version of record: 02 January 2026 DOI: https://doi.org/10.1038/s41579-025-01266-8 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Microbiology (Nat Rev Microbiol) ISSN 1740-1534 (online) ISSN 1740-1526 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41569-025-01227-x'>The cardiac conduction system: development, function and therapeutic targets</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2026-01-02 01:01:01
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Nature Reviews Cardiology (2026)Cite this article The cardiac conduction system (CCS) has a vital role in initiating and coordinating nearly 3 billion heartbeats throughout a person's lifetime. The CCS comprises two primary tissue types: the impulse-generating, slow-conducting nodes and the fast-conducting components of the ventricular conduction system. Dysfunction in this system can give rise to a spectrum of clinical symptoms, including palpitations, syncope, heart failure and even sudden cardiac death. Owing to the limited therapeutic options other than electronic pacemakers, substantial research efforts have been aimed at uncovering the root causes of conduction system disorders. A comprehensive investigative approach integrating genetics, transcriptomics and proteomics has been used to unravel the complex biology of these diseases. Advances in single-cell genomic and transcriptomic technologies, together with spatial transcriptomics, are offering new insights into the cellular microenvironments that govern conduction system function. In this Review, we examine the latest progress in understanding the biology of the CCS, situating new findings within both established and emerging scientific paradigms. Additionally, we discuss how these insights can be leveraged to improve clinical risk assessment, expand drug discovery efforts, accelerate technology aimed at promoting CCS regeneration and foster the development of innovative therapies, including biological pacemakers. The cardiac conduction system (CCS) initiates each heartbeat and regulates the flow of electrical impulses throughout the heart. Diseases of the CCS are a major cause of morbidity and mortality worldwide; research into therapeutic alternatives to electronic pacemakers is driving new discoveries in CCS biology. CCS-restricted gene regulatory networks reinforce specialized conduction phenotypes and simultaneously repress working cardiomyocyte-specific programming. Transcriptomic analyses at single-cell resolution, combined with spatial transcriptomics, are offering unprecedented insights into the local tissue microenvironments of the CCS and are driving advances in next-generation tools for drug discovery and therapies specifically targeting the conduction system. Development of biological pacemakers that use transcription factors to reprogramme working cardiomyocytes into CCS cells holds substantial therapeutic potential. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 12 print issues and online access $189.00 per year only $15.75 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Kusumoto, F. M. et al. 2018 ACC/AHA/HRS guideline on the evaluation and management of patients with bradycardia and cardiac conduction delay: a report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines and the Heart Rhythm Society. Google Scholar Haimovich, J. S. et al. Frequency of electrocardiogram-defined cardiac conduction disorders in a multi-institutional primary care cohort. JACC Adv. Google Scholar Weng, L. C. et al. The impact of common and rare genetic variants on bradyarrhythmia development. Google Scholar Milanesi, R., Baruscotti, M., Gnecchi-Ruscone, T. & DiFrancesco, D. Familial sinus bradycardia associated with a mutation in the cardiac pacemaker channel. Google Scholar Basson, C. T. et al. Mutations in human TBX5 [corrected] cause limb and cardiac malformation in Holt–Oram syndrome. Google Scholar Crasto, S., My, I. & Di Pasquale, E. The broad spectrum of LMNA cardiac diseases: from molecular mechanisms to clinical phenotype. Google Scholar Mond, H. G. & Proclemer, A. The 11th world survey of cardiac pacing and implantable cardioverter-defibrillators: calendar year 2009 — a World society of Arrhythmia's project. Pacing Clin. Article PubMed Google Scholar Simpson, J. et al. Long-term complications related to cardiac implantable electronic devices. Google Scholar Pallante, B. A. et al. Contactin-2 expression in the cardiac Purkinje fiber network. Google Scholar Gillers, B. S. et al. Canonical wnt signaling regulates atrioventricular junction programming and electrophysiological properties. Google Scholar Nerbonne, J. M. & Kass, R. S. Molecular physiology of cardiac repolarization. Google Scholar Lakatta, E. G., Maltsev, V. A. & Vinogradova, T. M. A coupled system of intracellular Ca2+ clocks and surface membrane voltage clocks controls the timekeeping mechanism of the heart's pacemaker. Google Scholar Linscheid, N. et al. Quantitative proteomics and single-nucleus transcriptomics of the sinus node elucidates the foundation of cardiac pacemaking. Google Scholar DiFrancesco, D. The contribution of the ‘pacemaker' current (if) to generation of spontaneous activity in rabbit sino-atrial node myocytes. Google Scholar Kanemaru, K. et al. Spatially resolved multiomics of human cardiac niches. Google Scholar Platzer, J. et al. Congenital deafness and sinoatrial node dysfunction in mice lacking class D L-type Ca2+ channels. Google Scholar Mangoni, M. E. et al. Bradycardia and slowing of the atrioventricular conduction in mice lacking CaV3.1/alpha1G T-type calcium channels. Google Scholar Goodyer, W. R. et al. In vivo visualization and molecular targeting of the cardiac conduction system. Google Scholar Goodyer, W. R. et al. Transcriptomic profiling of the developing cardiac conduction system at single-cell resolution. Google Scholar Ivanov, S. V. et al. Cerebellar ataxia, seizures, premature death, and cardiac abnormalities in mice with targeted disruption of the Cacna2d2 gene. Google Scholar Bogdanov, K. Y., Vinogradova, T. M. & Lakatta, E. G. Sinoatrial nodal cell ryanodine receptor and Na(+)-Ca(2+) exchanger: molecular partners in pacemaker regulation. Google Scholar Kreuzberg, M. M. et al. Connexin30.2 containing gap junction channels decelerate impulse propagation through the atrioventricular node. Natl Acad. Google Scholar Aanhaanen, W. T. et al. Developmental origin, growth, and three-dimensional architecture of the atrioventricular conduction axis of the mouse heart. Google Scholar Saez, J. C., Berthoud, V. M., Branes, M. C., Martinez, A. D. & Beyer, E. C. Plasma membrane channels formed by connexins: their regulation and functions. Google Scholar Al-Khatib, S. M. et al. Risk stratification for arrhythmic events in patients with asymptomatic pre-excitation: a systematic review for the 2015 ACC/AHA/HRS guideline for the management of adult patients with supraventricular tachycardia: a report of the American College of Cardiology/American Heart Association Task Force on clinical practice guidelines and the Heart Rhythm Society. Article PubMed Google Scholar Hiroi, Y. et al. Tbx5 associates with Nkx2-5 and synergistically promotes cardiomyocyte differentiation. Google Scholar Hoogaars, W. M. et al. The transcriptional repressor Tbx3 delineates the developing central conduction system of the heart. Google Scholar Linden, H., Williams, R., King, J., Blair, E. & Kini, U. Ulnar mammary syndrome and TBX3: expanding the phenotype. Article PubMed Google Scholar Meneghini, V., Odent, S., Platonova, N., Egeo, A. & Merlo, G. R. Novel TBX3 mutation data in families with ulnar-mammary syndrome indicate a genotype–phenotype relationship: mutations that do not disrupt the T-domain are associated with less severe limb defects. Article PubMed Google Scholar Stennard, F. A. & Harvey, R. P. T-box transcription factors and their roles in regulatory hierarchies in the developing heart. Google Scholar Liu, N. et al. Functional variants in TBX2 are associated with a syndromic cardiovascular and skeletal developmental disorder. Google Scholar Bruneau, B. G. et al. A murine model of Holt–Oram syndrome defines roles of the T-box transcription factor Tbx5 in cardiogenesis and disease. Google Scholar Garg, V. et al. GATA4 mutations cause human congenital heart defects and reveal an interaction with TBX5. Google Scholar Luna-Zurita, L. et al. Complex interdependence regulates heterotypic transcription factor distribution and coordinates cardiogenesis. Google Scholar Christoffels, V. M. et al. T-box transcription factor Tbx2 represses differentiation and formation of the cardiac chambers. Google Scholar Boogerd, K. J. et al. Msx1 and Msx2 are functional interacting partners of T-box factors in the regulation of Connexin43. Google Scholar Hoogaars, W. M. et al. Tbx3 controls the sinoatrial node gene program and imposes pacemaker function on the atria. Genes Dev. Google Scholar Habets, P. E. et al. Cooperative action of Tbx2 and Nkx2.5 inhibits ANF expression in the atrioventricular canal: implications for cardiac chamber formation. Genes Dev. Google Scholar Burnicka-Turek, O. et al. Coordinated Tbx3/Tbx5 transcriptional control of the adult ventricular conduction system. Google Scholar Schott, J. J. et al. Congenital heart disease caused by mutations in the transcription factor NKX2-5. Google Scholar Oh, Y. et al. Transcriptional regulation of the postnatal cardiac conduction system heterogeneity. Google Scholar Wiesinger, A. et al. A single cell transcriptional roadmap of human pacemaker cell differentiation. Google Scholar Bhattacharyya, S. et al. Using Gjd3-CreEGFP mice to examine atrioventricular node morphology and composition. Google Scholar Liang, D. et al. Cellular and molecular landscape of mammalian sinoatrial node revealed by single-cell RNA sequencing. Google Scholar Farah, E. N. et al. Spatially organized cellular communities form the developing human heart. Google Scholar Arduini, A. et al. Transcriptional profile of the rat cardiovascular system at single-cell resolution. Google Scholar van Eif, V. W. W. et al. Transcriptome analysis of mouse and human sinoatrial node cells reveals a conserved genetic program. Google Scholar Kalyanasundaram, A. et al. Three-dimensional functional anatomy of the human sinoatrial node for epicardial and endocardial mapping and ablation. Heart Rhythm 20, 122–133 (2023). Article PubMed Google Scholar Csepe, T. A. et al. Human sinoatrial node structure: 3D microanatomy of sinoatrial conduction pathways. Article PubMed Google Scholar Mommersteeg, M. T. et al. The sinus venosus progenitors separate and diversify from the first and second heart fields early in development. Google Scholar Mommersteeg, M. T. et al. Molecular pathway for the localized formation of the sinoatrial node. Google Scholar Wiese, C. et al. Formation of the sinus node head and differentiation of sinus node myocardium are independently regulated by Tbx18 and Tbx3. Google Scholar Mori, A. D. et al. Tbx5-dependent rheostatic control of cardiac gene expression and morphogenesis. Google Scholar Blaschke, R. J. et al. Targeted mutation reveals essential functions of the homeodomain transcription factor Shox2 in sinoatrial and pacemaking development. Google Scholar Ye, W. et al. A common Shox2-Nkx2-5 antagonistic mechanism primes the pacemaker cell fate in the pulmonary vein myocardium and sinoatrial node. Google Scholar Espinoza-Lewis, R. A. et al. Ectopic expression of Nkx2.5 suppresses the formation of the sinoatrial node in mice. Google Scholar Li, H. et al. Nkx2-5 defines a subpopulation of pacemaker cells and is essential for the physiological function of the sinoatrial node in mice. Google Scholar Puskaric, S. et al. Shox2 mediates Tbx5 activity by regulating Bmp4 in the pacemaker region of the developing heart. Google Scholar Basson, C. T. et al. The clinical and genetic spectrum of the Holt–Oram syndrome (heart-hand syndrome). Google Scholar Moskowitz, I. P. et al. The T-box transcription factor Tbx5 is required for the patterning and maturation of the murine cardiac conduction system. Google Scholar Mohan, R. A. et al. Embryonic Tbx3(+) cardiomyocytes form the mature cardiac conduction system by progressive fate restriction. Article PubMed Google Scholar Vedantham, V., Galang, G., Evangelista, M., Deo, R. C. & Srivastava, D. RNA sequencing of mouse sinoatrial node reveals an upstream regulatory role for Islet-1 in cardiac pacemaker cells. Google Scholar Frank, D. U. et al. Lethal arrhythmias in Tbx3-deficient mice reveal extreme dosage sensitivity of cardiac conduction system function and homeostasis. Natl Acad. Google Scholar Mohan, R. A. et al. T-box transcription factor 3 governs a transcriptional program for the function of the mouse atrioventricular conduction system. Natl Acad. Google Scholar Wu, M. et al. Baf250a orchestrates an epigenetic pathway to repress the Nkx2.5-directed contractile cardiomyocyte program in the sinoatrial node. Cell Res. Google Scholar Gharibeh, L. et al. GATA6 is a regulator of sinus node development and heart rhythm. Natl Acad. Google Scholar Kapoor, N., Galang, G., Marban, E. & Cho, H. C. Transcriptional suppression of connexin43 by TBX18 undermines cell–cell electrical coupling in postnatal cardiomyocytes. Google Scholar Kapoor, N., Liang, W., Marban, E. & Cho, H. C. Direct conversion of quiescent cardiomyocytes to pacemaker cells by expression of Tbx18. Google Scholar Wolfson, D. W. et al. Transient pacing in pigs with complete heart block via myocardial injection of mRNA coding for the T-box transcription factor 18. Google Scholar Espinoza-Lewis, R. A. et al. Shox2 is essential for the differentiation of cardiac pacemaker cells by repressing Nkx2-5. Google Scholar Hoffmann, S. et al. Functional characterization of rare variants in the SHOX2 gene identified in sinus node dysfunction and atrial fibrillation. Google Scholar Ionta, V. et al. SHOX2 overexpression favors differentiation of embryonic stem cells into cardiac pacemaker cells, improving biological pacing ability. Stem Cell Rep. 4, 129–142 (2015). Google Scholar Hoffmann, S. et al. Islet1 is a direct transcriptional target of the homeodomain transcription factor Shox2 and rescues the Shox2-mediated bradycardia. Basic Res. Google Scholar Cai, C. L. et al. Isl1 identifies a cardiac progenitor population that proliferates prior to differentiation and contributes a majority of cells to the heart. Google Scholar Liang, X. et al. Transcription factor ISL1 is essential for pacemaker development and function. Google Scholar Viragh, S. & Challice, C. E. The development of the conduction system in the mouse embryo heart. Google Scholar Wang, J. et al. Pitx2 prevents susceptibility to atrial arrhythmias by inhibiting left-sided pacemaker specification. Google Scholar Ammirabile, G. et al. Pitx2 confers left morphological, molecular, and functional identity to the sinus venosus myocardium. Google Scholar Gudbjartsson, D. F. et al. Variants conferring risk of atrial fibrillation on chromosome 4q25. Google Scholar Wang, J. et al. Pitx2-microRNA pathway that delimits sinoatrial node development and inhibits predisposition to atrial fibrillation. Google Scholar Baudic, M. et al. TAD boundary deletion causes PITX2-related cardiac electrical and structural defects. Google Scholar Enard, W. et al. A humanized version of Foxp2 affects cortico-basal ganglia circuits in mice. Google Scholar Verweij, N., van de Vegte, Y. J. & van der Harst, P. Genetic study links components of the autonomous nervous system to heart-rate profile during exercise. Google Scholar Bhattacharyya, S. et al. Global chromatin landscapes identify candidate noncoding modifiers of cardiac rhythm. Google Scholar van Eif, V. W. W. et al. Genome-wide analysis identifies an essential human TBX3 pacemaker enhancer. Google Scholar Galang, G. et al. ATAC-Seq reveals an Isl1 enhancer that regulates sinoatrial node development and function. Google Scholar Abassah-Oppong, S. et al. A gene desert required for regulatory control of pleiotropic Shox2 expression and embryonic survival. Google Scholar Ramirez, J. et al. Thirty loci identified for heart rate response to exercise and recovery implicate autonomic nervous system. Google Scholar Anderson, R. H., Mori, S., Spicer, D. E., Sanchez-Quintana, D. & Jensen, B. The anatomy, development, and evolution of the atrioventricular conduction axis. Google Scholar de Carvalho, A. P. & de Almeida, D. F. Spread of activity through the atrioventricular node. Google Scholar Efimov, I. R. & Mazgalev, T. N. High-resolution, three-dimensional fluorescent imaging reveals multilayer conduction pattern in the atrioventricular node. Google Scholar Aanhaanen, W. T. et al. The Tbx2+ primary myocardium of the atrioventricular canal forms the atrioventricular node and the base of the left ventricle. Google Scholar Horsthuis, T. et al. Gene expression profiling of the forming atrioventricular node using a novel tbx3-based node-specific transgenic reporter. Google Scholar Anderson, R. H. et al. Inferior extensions of the atrioventricular node. Google Scholar Ma, L., Lu, M. F., Schwartz, R. J. & Martin, J. F. Bmp2 is essential for cardiac cushion epithelial–mesenchymal transition and myocardial patterning. Google Scholar Chi, N. C. et al. Foxn4 directly regulates tbx2b expression and atrioventricular canal formation. Genes Dev. Google Scholar Liu, F. et al. GATA-binding factor 6 contributes to atrioventricular node development and function. Google Scholar Verhoeven, M. C., Haase, C., Christoffels, V. M., Weidinger, G. & Bakkers, J. Wnt signaling regulates atrioventricular canal formation upstream of BMP and Tbx2. Birth Defects Res. A Clin. Google Scholar Jay, P. Y. et al. Nkx2-5 mutation causes anatomic hypoplasia of the cardiac conduction system. Google Scholar Singh, R. et al. Tbx20 interacts with Smads to confine Tbx2 expression to the atrioventricular canal. Google Scholar Rutenberg, J. B. et al. Developmental patterning of the cardiac atrioventricular canal by Notch and Hairy-related transcription factors. Google Scholar Stefanovic, S. et al. GATA-dependent regulatory switches establish atrioventricular canal specificity during heart development. Article PubMed Google Scholar Singh, R. et al. Tbx2 and Tbx3 induce atrioventricular myocardial development and endocardial cushion formation. Cell Mol. Life Sci. Google Scholar Bressan, M. et al. Reciprocal myocardial–endocardial interactions pattern the delay in atrioventricular junction conduction. Google Scholar Lalani, S. R. et al. 20p12.3 microdeletion predisposes to Wolff–Parkinson–White syndrome with variable neurocognitive deficits. Google Scholar Gaussin, V. et al. Alk3/Bmpr1a receptor is required for development of the atrioventricular canal into valves and annulus fibrosus. Google Scholar Stroud, D. M. et al. Abnormal conduction and morphology in the atrioventricular node of mice with atrioventricular canal targeted deletion of Alk3/Bmpr1a receptor. Google Scholar Aanhaanen, W. T. et al. Defective Tbx2-dependent patterning of the atrioventricular canal myocardium causes accessory pathway formation in mice. Google Scholar Bakker, M. L. et al. Transcription factor Tbx3 is required for the specification of the atrioventricular conduction system. Google Scholar Pfeufer, A. et al. Genome-wide association study of PR interval. Google Scholar Sotoodehnia, N. et al. Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction. Google Scholar Verweij, N. et al. Genetic determinants of P wave duration and PR segment. Google Scholar van Setten, J. et al. PR interval genome-wide association meta-analysis identifies 50 loci associated with atrial and atrioventricular electrical activity. Google Scholar Harrelson, Z. et al. Tbx2 is essential for patterning the atrioventricular canal and for morphogenesis of the outflow tract during heart development. Google Scholar Smith, J. G. et al. Genome-wide association studies of the PR interval in African Americans. PLoS Genet. Google Scholar Hong, K. W. et al. Identification of three novel genetic variations associated with electrocardiographic traits (QRS duration and PR interval) in East Asians. Google Scholar Munshi, N. V. et al. Cx30.2 enhancer analysis identifies Gata4 as a novel regulator of atrioventricular delay. Google Scholar Harris, J. P. et al. MyoR modulates cardiac conduction by repressing Gata4. Google Scholar Pashmforoush, M. et al. Nkx2-5 pathways and congenital heart disease; loss of ventricular myocyte lineage specification leads to progressive cardiomyopathy and complete heart block. Google Scholar Ranganayakulu, G., Elliott, D. A., Harvey, R. P. & Olson, E. N. Divergent roles for NK-2 class homeobox genes in cardiogenesis in flies and mice. Google Scholar Tanaka, M., Chen, Z., Bartunkova, S., Yamasaki, N. & Izumo, S. The cardiac homeobox gene Csx/Nkx2.5 lies genetically upstream of multiple genes essential for heart development. Google Scholar Munoz-Martin, N. et al. Meis transcription factors regulate cardiac conduction system development and adult function. Article PubMed Central Google Scholar Milan, D. J., Giokas, A. C., Serluca, F. C., Peterson, R. T. & MacRae, C. A. Notch1b and neuregulin are required for specification of central cardiac conduction tissue. Google Scholar Rentschler, S. et al. Notch signaling regulates murine atrioventricular conduction and the formation of accessory pathways. Google Scholar Sun, T. et al. Dbh(+) catecholaminergic cardiomyocytes contribute to the structure and function of the cardiac conduction system in murine heart. Google Scholar Blackwell, D. J. et al. The Purkinje–myocardial junction is the anatomic origin of ventricular arrhythmia in CPVT. JCI Insight 7, e151893 (2022). Google Scholar Gourdie, R. G., Mima, T., Thompson, R. P. & Mikawa, T. Terminal diversification of the myocyte lineage generates Purkinje fibers of the cardiac conduction system. Google Scholar Rentschler, S. et al. Neuregulin-1 promotes formation of the murine cardiac conduction system. Google Scholar Choquet, C., Kelly, R. G. & Miquerol, L. Nkx2-5 defines distinct scaffold and recruitment phases during formation of the murine cardiac Purkinje fiber network. Google Scholar Choquet, C. et al. Nkx2-5 loss of function in the His-Purkinje system hampers its maturation and leads to mechanical dysfunction. Google Scholar Moskowitz, I. P. et al. A molecular pathway including Id2, Tbx5, and Nkx2-5 required for cardiac conduction system development. Google Scholar Burnicka-Turek, O. et al. Transcriptional patterning of the ventricular cardiac conduction system. Google Scholar Shekhar, A. et al. ETV1 activates a rapid conduction transcriptional program in rodent and human cardiomyocytes. Google Scholar Shekhar, A. et al. Transcription factor ETV1 is essential for rapid conduction in the heart. Google Scholar Koizumi, A. et al. Genetic defects in a His-Purkinje system transcription factor, IRX3, cause lethal cardiac arrhythmias. Heart J. Google Scholar Meysen, S. et al. Nkx2.5 cell-autonomous gene function is required for the postnatal formation of the peripheral ventricular conduction system. Google Scholar Wakimoto, H., Kasahara, H., Maguire, C. T., Izumo, S. & Berul, C. I. Developmentally modulated cardiac conduction failure in transgenic mice with fetal or postnatal overexpression of DNA nonbinding mutant Nkx2.5. Article PubMed Google Scholar Kasahara, H. et al. Progressive atrioventricular conduction defects and heart failure in mice expressing a mutant Csx/Nkx2.5 homeoprotein. Google Scholar Risebro, C. A. et al. Epistatic rescue of Nkx2.5 adult cardiac conduction disease phenotypes by prospero-related homeobox protein 1 and HDAC3. Google Scholar Yamaguchi, N. et al. An anterior second heart field enhancer regulates the gene regulatory network of the cardiac outflow tract. Google Scholar Lien, C. L., McAnally, J., Richardson, J. & Olson, E. N. Cardiac-specific activity of an Nkx2-5 enhancer requires an evolutionarily conserved Smad binding site. Google Scholar Lien, C. L. et al. Control of early cardiac-specific transcription of Nkx2-5 by a GATA-dependent enhancer. Google Scholar Harris, B. S. et al. Differentiation of cardiac Purkinje fibers requires precise spatiotemporal regulation of Nkx2-5 expression. Google Scholar Arnolds, D. E. et al. TBX5 drives Scn5a expression to regulate cardiac conduction system function. Google Scholar Arnolds, D. E. & Moskowitz, I. P. Inducible recombination in the cardiac conduction system of minK: CreERT2 BAC transgenic mice. Google Scholar Chambers, J. C. et al. Genetic variation in SCN10A influences cardiac conduction. Google Scholar Holm, H. et al. Several common variants modulate heart rate, PR interval and QRS duration. Google Scholar Seyerle, A. A. et al. Genome-wide association study of PR interval in Hispanics/Latinos identifies novel locus at ID2. Google Scholar Christophersen, I. E. et al. Large-scale analyses of common and rare variants identify 12 new loci associated with atrial fibrillation. Google Scholar van den Boogaard, M. et al. Genetic variation in T-box binding element functionally affects SCN5A/SCN10A enhancer. Google Scholar Man, J. C. K. et al. Variant intronic enhancer controls SCN10A-short expression and heart conduction. Google Scholar van den Boogaard, M. et al. A common genetic variant within SCN10A modulates cardiac SCN5A expression. Google Scholar Hua, L. L. et al. Specification of the mouse cardiac conduction system in the absence of Endothelin signaling. Google Scholar Gourdie, R. G., Wei, Y., Kim, D., Klatt, S. C. & Mikawa, T. Endothelin-induced conversion of embryonic heart muscle cells into impulse-conducting Purkinje fibers. Google Scholar Patel, R. & Kos, L. Endothelin-1 and Neuregulin-1 convert embryonic cardiomyocytes into cells of the conduction system in the mouse. Google Scholar Yamaguchi, N. et al. Cardiac pressure overload decreases ETV1 expression in the left atrium, contributing to atrial electrical and structural remodeling. Google Scholar Grego-Bessa, J. et al. Notch signaling is essential for ventricular chamber development. Google Scholar Ribeiro da Silva, A. et al. NOTCH1 is critical for fibroblast-mediated induction of cardiomyocyte specialization into ventricular conduction system-like cells in vitro. Google Scholar Gonzalez, D. M. et al. Transient notch activation converts pluripotent stem cell-derived cardiomyocytes towards a Purkinje fiber fate. Preprint at bioRxiv https://doi.org/10.1101/2024.09.22.614353 (2024). Rentschler, S. et al. Myocardial Notch signaling reprograms cardiomyocytes to a conduction-like phenotype. Google Scholar van Rijen, H. V. et al. Impaired conduction in the bundle branches of mouse hearts lacking the gap junction protein connexin40. Article PubMed Google Scholar Tamaddon, H. S. et al. High-resolution optical mapping of the right bundle branch in connexin40 knockout mice reveals slow conduction in the specialized conduction system. Google Scholar Zhang, S. S. et al. Iroquois homeobox gene 3 establishes fast conduction in the cardiac His–Purkinje network. Google Scholar Kim, K. H. et al. Irx3 is required for postnatal maturation of the mouse ventricular conduction system. Google Scholar Chen, F. et al. Hop is an unusual homeobox gene that modulates cardiac development. Google Scholar Ismat, F. A. et al. Homeobox protein Hop functions in the adult cardiac conduction system. Google Scholar Evans, D. S. et al. Fine-mapping, novel loci identification, and SNP association transferability in a genome-wide association study of QRS duration in African Americans. Google Scholar Vincentz, J. W. et al. Variation in a left ventricle-specific Hand1 enhancer impairs GATA transcription factor binding and disrupts conduction system development and function. Google Scholar Nguyen-Tran, V. T. et al. A novel genetic pathway for sudden cardiac death via defects in the transition between ventricular and conduction system cell lineages. Google Scholar Hewett, K. W. et al. Knockout of the neural and heart expressed gene HF-1b results in apical deficits of ventricular structure and activation. Google Scholar Umbarkar, P. et al. Cardiomyocyte SMAD4-dependent TGF-beta signaling is essential to maintain adult heart homeostasis. JACC Basic Transl. Google Scholar Giri, B. et al. G4 resolvase 1 tightly binds and unwinds unimolecular G4-DNA. Nucleic Acids Res. Google Scholar Gomez-Del Arco, P. et al. The G4 resolvase Dhx36 modulates cardiomyocyte differentiation and ventricular conduction system development. Google Scholar Haubner, B. J. et al. Functional recovery of a human neonatal heart after severe myocardial infarction. Google Scholar Porrello, E. R. et al. Transient regenerative potential of the neonatal mouse heart. Google Scholar Wang, H. et al. Electrophysiologic conservation of epicardial conduction dynamics after myocardial infarction and natural heart regeneration in newborn piglets. Google Scholar Wang, L., Bhakta, M., Fernandez-Perez, A. & Munshi, N. V. Inducible cardiomyocyte injury within the atrioventricular conduction system uncovers latent regenerative capacity in mice. Google Scholar Kahr, P. C. et al. A novel transgenic Cre allele to label mouse cardiac conduction system. Google Scholar Boulgakoff, L. et al. Participation of ventricular trabeculae in neonatal cardiac regeneration leads to ectopic recruitment of Purkinje-like cells. Google Scholar Sayers, J. R. et al. Cardiac conduction system regeneration prevents arrhythmias after myocardial infarction. Google Scholar Meyer, D. & Birchmeier, C. Multiple essential functions of neuregulin in development. Google Scholar Lai, D. et al. Neuregulin 1 sustains the gene regulatory network in both trabecular and nontrabecular myocardium. Google Scholar Kim, E. E. et al. The transcription factor EBF1 non-cell-autonomously regulates cardiac growth and differentiation. Google Scholar Park, D. S. et al. Pocket proteins critically regulate cell cycle exit of the trabecular myocardium and the ventricular conduction system. Google Scholar Csepe, T. A., Kalyanasundaram, A., Hansen, B. J., Zhao, J. & Fedorov, V. V. Fibrosis: a structural modulator of sinoatrial node physiology and dysfunction. Google Scholar Gluck, J. M. et al. Biochemical and biomechanical properties of the pacemaking sinoatrial node extracellular matrix are distinct from contractile left ventricular matrix. PLoS ONE 12, e0185125 (2017). Google Scholar Bressan, M. et al. Dynamic cellular integration drives functional assembly of the heart's pacemaker complex. Google Scholar Sun, Y. H. et al. The sinoatrial node extracellular matrix promotes pacemaker phenotype and protects automaticity in engineered heart tissues from cyclic strain. Google Scholar Kalyanasundaram, A. et al. Fibroblast-specific proteotranscriptomes reveal distinct fibrotic signatures of human sinoatrial node in nonfailing and failing hearts. Google Scholar Restrepo, A. J., Razminia, M., Sanchez-Quintana, D. & Cabrera, J. A. Myoarchitecture of the sinoatrial node and its relevance for catheter ablation: anatomy and histology. JACC Case Rep. 29, 102153 (2024). Article PubMed Google Scholar Efremova, M., Vento-Tormo, M., Teichmann, S. A. & Vento-Tormo, R. CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes. Google Scholar Sawa, H. et al. Expression of the angiotensinogen gene and localization of its protein in the human heart. Google Scholar Ieda, M. et al. Sema3a maintains normal heart rhythm through sympathetic innervation patterning. Google Scholar Hulsmans, M. et al. Macrophages facilitate electrical conduction in the heart. Google Scholar Roselli, C. et al. Multi-ethnic genome-wide association study for atrial fibrillation. Google Scholar Nielsen, J. B. et al. Biobank-driven genomic discovery yields new insight into atrial fibrillation biology. Google Scholar Li, J. et al. Modeling the atrioventricular conduction axis using human pluripotent stem cell-derived cardiac assembloids. Google Scholar Qu, J. et al. Expression and function of a biological pacemaker in canine heart. Article PubMed Google Scholar Miake, J., Marban, E. & Nuss, H. B. Biological pacemaker created by gene transfer. Google Scholar Boink, G. J. et al. HCN2/SkM1 gene transfer into canine left bundle branch induces stable, autonomically responsive biological pacing at physiological heart rates. Google Scholar Cingolani, E. et al. Biological pacemaker created by percutaneous gene delivery via venous catheters in a porcine model of complete heart block. Heart Rhythm 9, 1310–1318 (2012). Article PubMed Google Scholar Hu, Y. F., Dawkins, J. F., Cho, H. C., Marban, E. & Cingolani, E. Biological pacemaker created by minimally invasive somatic reprogramming in pigs with complete heart block. Google Scholar Fan, J. et al. Inhibition of Tgf-beta signaling enables durable ventricular pacing by TBX18 gene transfer. Preprint at bioRxiv https://doi.org/10.1101/2022.06.02.493572 (2022). Keith, A. & Flack, M. The form and nature of the muscular connections between the primary divisions of the vertebrate heart. Google Scholar His, W. Jr. The story of the atrioventricular bundle with remarks concerning embryonic heart activity. Allied Sci. Article PubMed Google Scholar Suma, K. Sunao Tawara: a father of modern cardiology. Pacing Clin. Google Scholar Steiner, I. History of the Purkyne fibres in the heart. On the occasion of bicentenary of Jan Evangelista Purkyne's birth (December 17, 1787). Univerzity Hradci Kralove 30, 7–20 (1987). Google Scholar Roselli, C., Rienstra, M. & Ellinor, P. T. Genetics of atrial fibrillation in 2020: GWAS, genome sequencing, polygenic risk, and beyond. Google Scholar Koizumi, A. et al. Genetic defects in a His–Purkinje system transcription factor, IRX3, cause lethal cardiac arrhythmias. Heart J. Google Scholar Tessadori, F. et al. Identification and functional characterization of cardiac pacemaker cells in zebrafish. PLoS ONE 7, e47644 (2012). Google Scholar Naya, F. J. et al. Mitochondrial deficiency and cardiac sudden death in mice lacking the MEF2A transcription factor. Google Scholar Ritchie, M. D. et al. Genome- and phenome-wide analyses of cardiac conduction identifies markers of arrhythmia risk. Google Scholar Benson, D. W. et al. Mutations in the cardiac transcription factor NKX2.5 affect diverse cardiac developmental pathways. Google Scholar Costa, M. W. et al. Functional characterization of a novel mutation in NKX2-5 associated with congenital heart disease and adult-onset cardiomyopathy. Google Scholar Yuan, F. et al. A novel NKX2-5 loss-of-function mutation predisposes to familial dilated cardiomyopathy and arrhythmias. Google Scholar den Hoed, M. et al. Identification of heart rate-associated loci and their effects on cardiac conduction and rhythm disorders. Google Scholar Takeda, M. et al. Slow progressive conduction and contraction defects in loss of Nkx2-5 mice after cardiomyocyte terminal differentiation. Google Scholar Kääb, S. et al. Large scale replication and meta-analysis of variants on chromosome 4q25 associated with atrial fibrillation. Heart J. Google Scholar Chinchilla, A. et al. PITX2 insufficiency leads to atrial electrical and structural remodeling linked to arrhythmogenesis. Google Scholar Tao, Y. et al. Pitx2, an atrial fibrillation predisposition gene, directly regulates ion transport and intercalated disc genes. Google Scholar Li, N. et al. A SHOX2 loss-of-function mutation underlying familial atrial fibrillation. Google Scholar Hoffmann, S. et al. Coding and non-coding variants in the SHOX2 gene in patients with early-onset atrial fibrillation. Basic Res. Google Scholar Umbarkar, P. et al. Cardiomyocyte SMAD4-dependent TGF-β signaling is essential to maintain adult heart homeostasis. JACC Basic Transl. Google Scholar Tegegne, B. S. et al. Phenotypic but not genetically predicted heart rate variability associated with all-cause mortality. Google Scholar Nolte, I. M. et al. Genetic loci associated with heart rate variability and their effects on cardiac disease risk. Google Scholar Sano, M. et al. Genome-wide association study of absolute QRS voltage identifies common variants of TBX3 as genetic determinants of left ventricular mass in a healthy Japanese population. PLoS ONE 11, e0155550 (2016). Google Scholar van der Harst, P. et al. 52 Genetic loci influencing myocardial mass. Google Scholar Jameson, H. S. et al. Loss of the atrial fibrillation-related gene, Zfhx3, results in atrial dilation and arrhythmias. Google Scholar Hill, M. C. et al. A cellular atlas of Pitx2-dependent cardiac development. Google Scholar Ren, H. et al. Single-cell RNA sequencing of murine hearts for studying the development of the cardiac conduction system. Google Scholar Feulner, L. et al. Transcriptional dynamics of the murine heart during perinatal development at single-cell resolution. Preprint at bioRxiv https://doi.org/10.1101/2024.03.05.583423 (2024). Park, D. S. & Fishman, G. I. in Cardiac Electrophysiology: From Cell to Bedside (eds Zipes, D. P. & Jalife, J.) Download references receives research funding from the National Institutes of Health (R01HL165130 and R01HL171989). receives research funding from the National Institutes of Health (R01 HL171118 and R01HL159869). The authors thank N. Yamaguchi (NYU Grossman School of Medicine, USA) for assistance with the generation of tables and figures for initial submission. Leon H. Charney Division of Cardiology, New York University Grossman School of Medicine, New York, NY, USA David S. Park & Glenn I. Fishman Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar The authors contributed substantially to all aspects of the article. Correspondence to Glenn I. Fishman. The authors declare no competing interests. Nature Reviews Cardiology thanks Nikhil V. Munshi, Kalyanam Shivkumar and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions Park, D.S., Fishman, G.I. The cardiac conduction system: development, function and therapeutic targets. Nat Rev Cardiol  (2026). Download citation Accepted: 07 October 2025 Published: 02 January 2026 Version of record: 02 January 2026 DOI: https://doi.org/10.1038/s41569-025-01227-x Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Nature Reviews Cardiology (Nat Rev Cardiol) ISSN 1759-5010 (online) ISSN 1759-5002 (print) © 2026 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            