
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-10-30</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09801-6'>Nanotyrannus and Tyrannosaurus coexisted at the close of the Cretaceous</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 18:14:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. Tyrannosaurus rex ranks among the most comprehensively studied extinct vertebrates1 and a model system for dinosaur paleobiology1. As one of the last surviving non-avian dinosaurs, Tyrannosaurus is a crucial datum for assessing terrestrial biodiversity, ecosystem structure, and biogeographic exchange immediately preceding the end-Cretaceous mass extinction —one of Earth's greatest biological catastrophes. Paleobiological studies of Tyrannosaurus, including ontogenetic niche partitioning2-4, feeding, locomotor biomechanics,5,6and life history7-9 have drawn upon an expanding skeletal sample comprising multiple hypothesized growth stages—and yet the Tyrannosaurus hypodigm remains controversial10-13. Here, we describe an exceptionally well-preserved, near somatically mature tyrannosaur skeleton (NCSM 40000) from the Hell Creek Formation that shares autapomorphies with the holotype specimen of N. lancensis. We couple comparative anatomy, longitudinal growth models, observations on ontogenetic character invariance, and a novel phylogenetic dataset to test the validity of Nanotyrannus, demonstrating conclusively that this taxon is distinguishable from Tyrannosaurus, sits outside Tyrannosauridae, and unexpectedly contains two species—N. Finally, we document at least two co-occurring, ecomorphologically distinct genera in the Maastrichtian of North America, demonstrating that tyrannosauroid alpha diversity was thriving within one million years of the end-Cretaceous extinction. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout These authors contributed equally: Lindsay E. Zanno, James G. Napoli Department of Biological Sciences, North Carolina State University, Raleigh, NC, USA North Carolina Museum of Natural Sciences, Raleigh, NC, USA Paleontology, Natural History Museum of Utah, Salt Lake City, UT, USA Evolutionary Studies Institute, University of Witwatersrand, Johannesburg, South Africa Supplementary Information 4: Analytical Code for NLFE Growth Models. Supplementary Information 7: Analytical Code for Graphing NLME Growth Models. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09816-z'>Experiments reveal extreme water generation during planet formation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 16:59:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). Before final publication, the manuscript will undergo further editing. Please note there may be errors present which affect the content, and all legal disclaimers apply. The most abundant type of planet discovered in the Galaxy has no analogue in our Solar System and is believed to consist of a rocky interior with an overlying thick H2 dominated envelope. Here we present novel experimental results designed to investigate this interaction. Laser heating diamond anvil cell experiments were conducted between 16 and 60 GPa at temperatures above 4000 K. We find that copious amounts of hydrogen dissolve into the silicate melt with a large dependence on temperature rather than pressure. This is a preview of subscription content, access via your institution Prices may be subject to local taxes which are calculated during checkout Earth and Planets Laboratory, Carnegie Institution for Science, Washington, D.C., USA Composition of the starting material and recovered experiments. Experiments reveal extreme water generation during planet formation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a69192461/nitrogen-fixation-arctic-sea-ice/'>Scientists Found a Strange Lifeform Under Arctic Ice. It Shouldn't Be There.</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 13:30:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. When it comes to important gasses on Earth, oxygen reigns supreme—it's the magic molecule necessary for respiration in nearly all creatures on the planet. An inert gas, nitrogen requires special lifeforms known as “nitrogen-fixers,” or diazotrophs, to transform it into bioavailable ammonia or ammonium. Plants and bacteria perform this magic trick on land, but in the oceans, cyanobacteria are the main purveyors of nitrogen fixation. These organisms form the foundation of the food web, as their byproducts feed algae—the main source of food for marine life. For decades, scientists believed that the process of nitrogen fixation largely took place in warmer waters, but in recent years, scientists have found evidence of the process at work in the Arctic Ocean. Now, a new study from scientists at the University of Copenhagen has gone even further, confirming that nitrogen fixation can occur under sea ice itself, an idea that was once considered impossible. “Until now, it was believed that nitrogen fixation could not take place under the sea ice because it was assumed that the living conditions for the organisms that perform nitrogen fixation were too poor,” Lisa von Friesen, a Ph.D. student at the University of Copenhagen, said in a press statement. In warmer waters (such as tropical and subtropical oceans), water is less soluble, meaning that it holds less dissolved oxygen and thus aids the fixation process. For decades, scientists assumed that Arctic fixation was out of the question, but a 2017 study found evidence of the process. While this new study heads further north into the central Arctic Ocean, it also finds another organism doing this important ammonia-producing work called (uninspiringly) a “non-cyanobacterial diazotroph,” or NCD. With the highest fixation rates occurring near the edge of sea ice as it retreats north due to climate change, the team confirms that the process is also occurring under sea ice, which completely changes our outlook for Arctic marine life in a warming world. “This could mean that the potential for algae production has also been underestimated as climate change continues to reduce the sea ice cover.” Because algae is crucial to the food web and are the yummy munchies of choice for planktonic crustaceans that go on to feed smaller fish, this new source of nitrogen-fixation is important for the viability of marine life moving forward—and the good news doesn't stop there. Algae also absorbs carbon dioxide from the atmosphere, so thi sno-longer-impossible source of nitrogen fixation could also increase the ocean's carbon budget. “But it is clear that we should include an important process such as nitrogen fixation in the equation when we try to predict what will happen to the Arctic Ocean in the coming decades as sea ice declines.” We May Have Been Wrong About the Origin of Life The Place Where Time Moves 9% Slower for Humans For 30 Years, Kodak Ran a Secret Nuclear Device Archaeologists Found the Ruins of a Lost Lake City</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a69180615/hadrians-wall-fort-artifacts/'>An Icon of the Roman Empire Just Revealed a Treasure Trove of Artifacts</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 13:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>At the northern end of Hadrian's Wall, volunteers and archaeology students uncovered pottery, metalwork, soldiers' tools, and more buried at the site of Bremenium Fort “This year's finds are exceptional, in both quantity and quality,” Bob Jackson from Redesdale Archaeological Group, said in a statement from the national park. “The archaeology of High Rochester continues to reveal its secretes, and it's inspiring to see so many people, of all ages, engaged in the uncovering of their shared heritage,” Chris Jones, a historic environment officer with the Northumberland National Park Authority, said in a statement. The pottery discoveries included an amphora made in northern Spain, which may have been used to transport olive oil to the fort. The team located a spearhead and slinger's lead shot (both of which would have been typical for a Roman soldier's kit), a lead seal (used on official documents), a votive oil lamp (often used as decoration for altars), several brooches (including an intact dolphin-style piece with preserved enamel), and two engraved intaglios. The intaglios were likely used as ring inserts, and show that grape harvesting likely took place at the site—a unique aspect of Roman life in Britain and northern Europe. By measuring the wall heights and construction layers found on site, experts came to believe that the building was rebuilt at least three times, with earlier phases demolished and covered in clay before the next project started. “Numerous finds of pottery and other items indicate that the original buildings were infilled, probably in the third century, with later buildings and yard surfaces placed over them,” Richard Carlton, an archaeologist from Newcastle University, said in a statement. The bottom of the trench was fortunately waterlogged, which helped preserve some elements that would have otherwise not survived dry conditions, including a section of wood and what is likely a very, very old plum. Past Bremenium excavations uncovered a gate and a lime kiln likely used to make the fort ramparts—one of three kilns uncovered along Hadrian's Wall or near its outpost forts. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. We May Have Been Wrong About the Origin of Life</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/how-an-error-in-cult-classic-game-doom-sparked-new-appreciation-for-pi/'>How an Error in Cult Classic Game Doom Sparked New Appreciation for Pi</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>How an Error in Cult Classic Game Doom Sparked New Appreciation for Pi Everyone makes mistakes—and sometimes those mistakes can lead to surprising discoveries. (Pi is often truncated without rounding, in which case the ninth decimal place would be filled by 3, but it rounds to 4 because the subsequent digit is 5.) Fortunately, this error has little impact on the game. The game has a great story but terrible graphics. That's not because of the incorrect pi value but rather a reflection of how little computing power was available in the 1990s. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Because Doom is an open-source computer game, you can download the code—and modify it. Gotszling did just that, testing what would happen if he changed the programmed values of pi. When Gotszling set π = 3, for example, the pixelated world of Doom became distorted, with walls and pillars moving in unexpected ways. As a player moved straight in the game world, surrounding objects would move to different sides. Enemies could appear out of nowhere and disappear again. “With enough intoxication, you can re-create this,” Gotszling joked in his presentation. Things got really bad when he set a value of π⁄2 for pi. The game wasn't particularly fun in this state. Originally, pi was defined as the ratio of a circle's circumference to its diameter. In our everyday world, we assume circles are round. But in the narrower mathematical sense, a circle is defined by all points that are equidistant from a common center. For example, imagine you're standing in downtown Manhattan. If you look closely, the value of pi in our world isn't exactly 3.14159…. Instead, the ratio of circumference to diameter takes on a different value—and, worse still, this value varies! If you are standing at the North Pole and looking at all the places exactly 1,000 kilometers away from you, for instance, these points form a circle on the surface of a sphere—but its circumference is smaller than that of a circle on a flat surface, thanks to the curvature of Earth. And this isn't just the case when considering a spherical surface. Any type of curved surface yields variable values for pi. Mathematicians refer to such curved worlds as “non-Euclidean” geometries. (More than 2,000 years ago, the scholar Euclid laid out the rules of geometry most of us learned in school—ideas that only apply on a flat world.) The quirky landscape Gotszling created by tweaking Doom's source code is something else entirely. When developing 3D graphics for a computer game, trigonometric functions such as sine, cosine and tangent play an important role. They can be used to describe how objects move through space over time. To save processing power, Doom's developers calculated important values of the trigonometric functions for various angles in advance and stored them in the lookup tables. And this is where pi comes into play: you have to multiply an angle in degrees by pi to get the corresponding value in radians, which the computer uses. So if Gotszling used a value of pi that was too small, the angles would be converted incorrectly. Because developers only store a finite number of angles, he created lookup tables with values that no longer include the full rotations: Instead of describing an environment in which an object only reappears in the same place after a 360-degree rotation, this occurs after a significantly smaller angle. As a result, in extreme situations, objects suddenly disappear or reappear out of nowhere. The Doom world Gotszling has created is incredibly strange. Nevertheless, it's a nice way to get people thinking about mathematics and the value—in every sense—of pi. This article originally appeared in Spektrum der Wissenschaft and was reproduced with permission. Manon Bischoff is a theoretical physicist and an editor at Spektrum der Wissenschaft, the German-language sister publication of Scientific American. If you enjoyed this article, I'd like to ask for your support. Scientific American has served as an advocate for science and industry for 180 years, and right now may be the most critical moment in that two-century history. I hope it does that for you, too. If you subscribe to Scientific American, you help ensure that our coverage is centered on meaningful research and discovery; that we have the resources to report on the decisions that threaten labs across the U.S.; and that we support both budding and working scientists at a time when the value of science itself too often goes unrecognized. In return, you get essential news, captivating podcasts, brilliant infographics, can't-miss newsletters, must-watch videos, challenging games, and the science world's best writing and reporting. There has never been a more important time for us to stand up and show why science matters. I hope you'll support us in that mission.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/10/251029100154.htm'>Scientists reverse Alzheimer's in mice with groundbreaking nanotech</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 11:09:40
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A team co-led by the Institute for Bioengineering of Catalonia (IBEC) and West China Hospital Sichuan University (WCHSU), in collaboration with partners in the UK, reports a nanotechnology approach that reverses Alzheimer's disease in mouse models. By repairing this interface, the animals showed a reversal of Alzheimer's pathology. That energy arrives through an exceptionally dense vascular network in which each neuron is supplied by its own capillary. With roughly one billion capillaries, the brain relies on healthy vasculature to sustain function and resist disease. These observations reinforce how vascular health connects to conditions such as dementia and Alzheimer's, where damage to the vascular system is closely associated. The BBB is a cellular and physiological shield that separates brain tissue from circulating blood, helping block pathogens and toxins. In Alzheimer's disease, amyloid-β (Aβ) is the primary waste protein, and its buildup disrupts neuronal function. The team worked with mouse models engineered to overproduce Aβ and to develop marked cognitive decline that mirrors Alzheimer's features. Across several behavioral and memory tests conducted over months, animals were assessed at different disease stages. We think it works like a cascade: when toxic species such as amyloid-beta (Aβ) accumulate, disease progresses. But once the vasculature is able to function again, it starts clearing Aβ and other harmful molecules, allowing the whole system to recover its balance. What's remarkable is that our nanoparticles act as a drug and seem to activate a feedback mechanism that brings this clearance pathway back to normal levels," said Giuseppe Battaglia, ICREA Research Professor at IBEC, Principal Investigator of the Molecular Bionics Group and leader of the study. Under normal conditions, the protein LRP1 serves as a molecular gatekeeper. If LRP1 binds too much Aβ too tightly, transport becomes congested and LRP1 itself is degraded within BBB cells, reducing the number of available carriers. Either scenario leads to Aβ accumulation in the brain. The supramolecular drugs act as a reset switch. As this process resumes, the vasculature regains its natural waste-clearing role and returns toward normal function. This precision supports efficient Aβ clearance and helps rebalance the vascular system that safeguards brain health. This therapeutic concept points toward future clinical strategies that address the vascular contribution to Alzheimer's disease and aim to improve patient outcomes. "Our study demonstrated remarkable efficacy in achieving rapid Aβ clearance, restoring healthy function in the blood-brain barrier and leading to a striking reversal of Alzheimer's pathology," concludes Lorena Ruiz Perez, researcher at the Molecular Bionics group from the Institute for Bioengineering of Catalonia (IBEC) and Serra Hunter Assistant Professor at the University of Barcelona (UB). Materials provided by Institute for Bioengineering of Catalonia (IBEC). Note: Content may be edited for style and length. How AI Saved NASA's $10 Billion Webb Telescope From Blurry Vision “Mystery Molecules” Found in Dogs Could Help Humans Live Longer, Healthier Lives Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41557-025-01906-9'>A pipeline for proteome-wide analysis of electrophile selectivity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 10:57:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. In the growing field of chemical proteomics, there is a need for general methods to map the reactivity profiles of covalent probes in complex proteomes. Now, a completely unbiased proteomic workflow has been developed that enables global characterization of the amino acid residues, chemically labelled by reactive electrophilic probes. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Subscribe to this journal Receive 12 print issues and online access Prices may be subject to local taxes which are calculated during checkout Singh, J., Petter, R. C., Baillie, T. A. Kuljanin, M. et al. Nat. Zanon, P. R. A., Lewald, L. & Hacker, S. M. Angew. Zanon, P. R. A. et al. Nat. Kong, A. T., Leprevost, F. V., Avtonomov, D. M., Mellacheruvu, D. & Nesvizhskii, A. I. Nat. Yu, F. et al. Nat. Department of Pathology, Stanford University School of Medicine, Stanford, CA, USA Jeyun Jo & Matthew Bogyo Department of Microbiology and Immunology, Stanford University School of Medicine, Stanford, CA, USA Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Correspondence to Jeyun Jo or Matthew Bogyo. The authors declare no competing interests. Jo, J., Bogyo, M. A pipeline for proteome-wide analysis of electrophile selectivity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Translational Research newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41588-025-02389-7'>Transcription factor switching drives subtype-specific pancreatic cancer</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 10:54:53
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Emerging evidence suggests that lineage-specifying transcription factors control the progression of pancreatic ductal adenocarcinoma (PDAC). We have discovered a transcription factor switching mechanism involving the poorly characterized orphan nuclear receptor HNF4G and the putative pioneer factor FOXA1, which drives PDAC progression. Using our unbiased protein interactome discovery approach, we identified HNF4A and HNF4G as reproducible, FOXA1-associated proteins, in both preclinical models and Whipple surgical samples. A molecular switch occurs in advanced disease, whereby HNF4G expression or activity decreases, unmasking FOXA1's transcriptional potential. Derepressed FOXA1 drives late-stage disease by orchestrating metastasis-specific enhancer–promoter loops to regulate the expression of metastatic genes. Overall survival is influenced by HNF4G and FOXA1 activity in primary tumor growth and in metastasis, respectively. We suggest that the existence of stage-dependent transcription factor activity, triggered by molecular compartmentalization, mediates the progression of PDAC. Despite recent advances in the treatment of pancreatic ductal adenocarcinoma (PDAC), median survival remains less than 12 months1. Recent large-scale genomic approaches have shown that PDAC is composed of at least two distinct molecular subtypes, based on transcriptomic signatures, and these are termed the classical and squamous (or basal/mesenchymal) subtypes. Although it is well described that PDAC harbors gene mutations (that is, KRAS G12D and p53 R172H) that initiate the disease, little is known about the influence of lineage-defining transcription factors on the PDAC molecular subtypes1,2,3,4,5. Pioneer factors (PFs) are a specialized type of transcription factors that can bind directly to condensed chromatin and create enhancer elements within the genome, and changes in the expression levels or fidelity of these specialized transcription factor TF can alter cell lineage. In hormone-dependent breast cancer, FOXA1 functions in a physical complex with GATA3 and the nuclear receptor (NR) estrogen receptor (ER), whereas in prostate cancer, FOXA1 forms a transcriptional complex with GATA2 and the NR androgen receptor (AR)9. Seminal work in metastatic organoid models (mouse) of PDAC demonstrated that elevated FOXA1 and GATA5 led to the creation of new enhancer elements, which were associated with increased metastatic potential10. At a minimum, these two transcription factors could function as biomarkers of this common subtype of PDAC, but as developmental transcription factors11, they potentially could have a functional role in tumor progression. Supporting this, HNF4A was previously shown to cooperate with GATA6 (another transcription factor commonly amplified in PDAC) to maintain a metabolic signature of the classical subtype12. Very little is known about HNF4G, although it has been shown to impact cellular proliferation and invasion13,14,15,16. The SMAD4 deficiency in PDAC has been shown to increase HNF4G expression and oncogenic potential15,17. One of the most statistically enriched germ-line variants is at the HNF4G gene locus and is linked with increased HNF4G expression and increased risk of developing PDAC18. We sought to leverage the link between FOXA1 and PDAC metastasis to purify FOXA1-associated proteins, thereby definitively identifying and characterizing the protein complex involved in disease progression. We identify HNF4 family members as consistent FOXA1-associated proteins and show a stage-specific mutually exclusive role for HNF4G and subsequently for FOXA1 in distinct stages of the disease, from primary tumor to metastasis. We adopted an unbiased approach to identify driving transcription factors in PDAC and performed chromatin immunoprecipitation sequencing (ChIP–seq) of the active enhancer mark H3K27Ac on surplus tissue obtained after the resection of primary PDAC (Whipple surgical samples). We identified all genomic intervals with substantial fold changes in H3K27Ac signal between PDAC tumors (n = 6) and normal adjacent tissue (n = 4; Fig. A median of 65,010 H3K27Ac peaks per sample (range = 43,909–76,827) was observed. 1a, and we termed these sites as PDAC-enriched candidate regulatory elements (PDAC-CREs). Unsupervised hierarchical clustering revealed a substantial H3K27Ac divergence between tumors and the adjacent normal tissue (Fig. Differential transcription factor binding at the tumor-specific regions was assessed using the cistrome DB toolkit, an approach that identifies binding site similarity with known experimental transcription factor datasets (Extended Data Fig. 1c), with FOXA1 being the top correlated transcription factor from published ChIP–seq datasets. Analysis of the H3K27Ac sites showed that the most enriched motifs within all experimentally mapped enhancer elements from patient samples were hepatocyte nuclear factor (HNF) motifs (HNF4G and HNF4A) and Forkhead (that is, FOXA1; Fig. 1a), confirming that these two classes of transcription factors constitute the lineage-defining factors in these patient tumor samples. The analysis of clinical datasets revealed that HNF4A levels did not differ between normal and primary tumor samples but were modestly increased in metastatic samples. We conducted ChIP–seq of HNF4G, HNF4A and FOXA1 from Whipple surgical samples from PDAC patients (n = 6; Fig.1d and Extended Data Fig. 1e,f), revealing 3,344 reproducible, consensus binding sites where all three transcription factors cobound at the same genomic regions (Fig. We also observed a total of 3,461 regions where FOXA1 and HNF4G were corecruited, without any HNF4A binding. 1f), suggesting that these transcription factors bind directly to DNA using consensus motifs. a, ChIP–seq was performed in patient samples taken from Whipple surgery of normal adjacent (n = 4) and PDAC samples (n = 6). Volcano plot represents enriched sites of H3K27Ac, with normalized intensities of called peaks (3,000 sites) in normal and tumor samples. The data have been analyzed using negative binomial GLM as a part of the DiffBind package. b,c, Box plots show the gene expression levels of HNF4G and adapted from ref. d, Heatmaps show coverage intensities mapped onto overlapped sites between FOXA1, H3K27Ac, HNF4A and HNF4G. An illustrative snapshot of a genomic region showing common FOXA1 and HNF4G binding sites. Venn diagram showing the overlapping sites between FOXA1, HNF4G and HNF4A peaks across four independent primary tumors. Distribution of genomic features across FOXA1 and HNF4G binding sites in primary tumors (n = 4). IGV screenshots representing FOXA1 and HNF4G binding from a patient sample. e, ChIP–seq was performed in patient samples of normal adjacent (n = 4) and PDAC samples (n = 6) to pull down HNF4G and H3K27Ac. Correlation of log2-fold changes between H3K27Ac and HNF4G ChIP–seq intensities (tumor versus normal adjacent). f, Kaplan–Meier plot and log-rank test were used to assess the survival curves for patients with HNF4G amplification versus those that were copy number neutral from TCGA data. g, IHC staining performed on frozen sections (n = 10). Representative images of FOXA1, HNF4G and HNF4A restricted to the epithelial cell compartment from one patient are presented. GLM, generalized linear model; FC, fold change. A comparison of HNF4G ChIP–seq from normal adjacent tissue (Fig. 1e) showed that tumor-specific enhancers were occupied by HNF4G. In support of a role for HNF4G, the genomic region encoding the HNF4G gene was shown to be amplified in 9% PDAC patients (Fig. Immunohistochemistry (IHC) analysis of FOXA1, HNF4A and HNF4G from primary PDAC tissue samples confirmed coexpression of these transcription factors, only in the cancer epithelial cells (Fig. These findings suggest that HNF4G mRNA levels and transcriptional activity increase when cells transition from normal to cancer, that HNF4G and FOXA1 are the key lineage-defining transcription factors in PDAC and that amplification of HNF4G is a common event that correlates with poor outcome. Although our initial discoveries from patient samples suggested that HNF4A and HNF4G could co-occupy many genomic regulation elements, we could show that HNF4A binding is dependent on HNF4G (Fig. 2b), with global HNF4A binding sites decreased when HNF4G was silenced. We observed that HNF4G binds directly to CREs at the HNF4A genomic locus, implying direct regulation of HNF4A expression (Fig. a, HNF4A ChIP–seq performed in HPAF-II human tumor cell line +/− HNF4G-KO. Heatmaps illustrate normalized coverage density at lost and gained ChIP–seq sites revealed with DiffBind statistical test at Padj < 0.05. b, UCSC genome browser tracks representing binding of HNF1A and HNF4A in the control and HNF4G-KO HPAF-II cells (chr20: 44,282,766–44,452,506). c, RIME in a panel of pancreatic cancer cell lines. Three biological replicates of FOXA1-RIME and one pooled IgG control RIME were included in each experiment. To identify specific interactors, a label-free quantification method was used. Substantially enriched interactors are highlighted on the heatmap Padj < 0.02. d, MA plot illustrates the changes in ATAC–seq open chromatin regions in HNF4G-KO versus control (Cas9) HPAF-II cells. Statistically significant values are highlighted in red. The heatmap bar above shows correlation in the vicinity between gained ATAC events and active gene loci in patient samples (PanCuRX dataset)37. e,f, HPAF-II cell was orthotopically injected into the pancreas of NSG mice. Data from survival (e) and tumor growth (f) analyses of ultrasound images of tumors are represented for the control (Cas9) and HNF4G-KO tumors. P value of <0.0006 calculation was generated from a two-sided two-sample Mann–Whitney–Wilcoxon test to assess if the median of the difference between the samples of each group is different from 0. P values were calculated with a Welch test with multiplicity corrections for the growth curves were assessed by a likelihood ratio test P value of <0.0001. The lighter lines indicate mouse-specific longitudinal tumor volume measures, color-coded by specific groups. g, DEGs derived from the HNF4G-KO RNA-seq analysis were evaluated by integrating clinical data from the PanCuRX study. Downregulated genes from in Deseq2 test of HNF4G-KO versus control samples were used to visualize expression patterns in the classical subtype tumors from the PanCuRX dataset. Hierarchical clustering was used to order clinical samples column-wise, while the gene expression order was preserved according to descending expression values of RNA-seq. IgG, immunoglobulin G; DEGs, differentially expressed genes. Our data suggest that HNF4G was essential for the growth of classical PDAC models, whereas silencing of HNF4A alone was not sufficient to affect cell proliferation (Extended Data Figs. We subsequently used rapid immunoprecipitation and mass spectrometry of endogenous (RIME) protein complexes, our method for unbiased discovery of endogenous protein complexes19,20,21 and purified FOXA1 as the ‘bait' to discover associated protein complexes (Fig. This was applied to two classical (HPAF-II and AsPC-1) PDAC cell line models (where HNF4A and HNF4G are highly expressed) and one mixed subtype model as a control (Miapaca-2; Fig. After FOXA1-RIME, we identified numerous subunits of the ATP-dependent SWI/SNF complex, known NR cofactors, including CBP, NCOA3 and NCOR1/NCOR2, as well as GATA6 (ref. Notably, this suggests that FOXA1 is present in both major molecular subtypes of PDAC and many of the FOXA1-associated coregulatory proteins were the same, except for HNF4A and HNF4G, which were only seen as FOXA1-associated proteins in models of the classical subtype of the disease. We validated this observation in additional models (Supplementary Table 1)2,22. To confirm that HNF4A and HNF4G were in a complex with both of the metastasis-associated transcription factors, FOXA1 and GATA6 (ref. 23), we also conducted RIME of GATA6 in the AsPC-1 classical model, which confirmed unbiased discovery of interactions among HNF4A, HNF4G, GATA6 and FOXA1 (Supplementary Table 1). The genomic binding profiles and physical interactions among HNF4G, FOXA1, HNF4A and GATA6 were validated in additional models of the classical subtype (Extended Data Figs. We hypothesized that FOXA1 was the PF for HNF4G (and possibly HNF4A), as seen with other NRs in breast and prostate cancer that require FOXA1. However, our in vitro data suggest that, although HNF4G was consistently required for viability of classical PDAC cell line models (Extended Data Figs. These data imply that FOXA1 might not be the PF for HNF4G, because depletion of FOXA1 should phenocopy HNF4G depletion. We also explored GATA6 as a potential PF for HNF4G, but silencing of GATA6 did not result in decreased HNF4G binding sites (Extended Data Fig. 2c) and instead depletion of GATA6 resulted in substantial gains in HNF4G binding sites, suggesting that GATA6 could be a negative regulator of HNF4G activity. We also found little evidence that HNF4A was required for the growth of classical models (Extended Data Fig. We, therefore, focused on HNF4G as a chromatin regulatory protein and specifically silenced HNF4G and subsequently conducted the assay for transposase accessible chromatin with high-throughput sequencing (ATAC–seq; Fig. We could also confirm that the HNF4G-dependent chromatin accessibility regions were adjacent to the HNF4G-dependent target genes (Fig. 2d), suggesting that HNF4G is responsible for creating the enhancers that drive the target gene expression events. Therefore, HNF4G is required for maintaining chromatin accessibility and HNF4G activity is not dependent on FOXA1 or on GATA6. We sought to explore the role of HNF4G in more complex tumor models of human disease. CRISPR knockout (KO) clones were generated in the HPAF-II cells (Extended Data Fig. We orthotopically injected HNF4G-KO pooled clones or control cells into the pancreas of NOD SCID-γ (NSG) mice and observed a significant increase (P < 0.0006) in the median survival of mice bearing HNF4G-KO tumors compared to control tumors (Fig. The HNF4G-KO resulted in slower tumor growth compared to control (Cas9) tumors (P < 0.0001) and substantially smaller tumors (Fig. 2f), confirming that HNF4G is required for optimal tumor growth. We conducted RNA-seq on the orthotopic tumor samples (Extended Data Fig. 3f–j) and identified an HNF4G-specific target gene signature that was assessed in the PanCuRx clinical cohort24. Using unsupervised hierarchical clustering, the HNF4G target genes (discovered in our orthotopic tumor experiments; Fig. 2g) were specifically expressed in primary tumors, as opposed to metastatic samples (Fig. 2g), implying that the HNF4G target genes were restricted to primary tumor contexts. The HNF4G target genes that were differentially expressed in the PanCuRx primary tumor samples were adjacent to the genomic regions that were ‘opened' by HNF4G in our cell line models (Fig. 2d, red panel above MA plot), confirming that HNF4G mediates chromatin accessibility at gene targets that are commonly expressed in patient samples. The findings show that FOXA1 cobinds with HNF4G in the classical subtype of PDAC, but that FOXA1 is not a PF in this PDAC subtype and, instead, the key transcription factor driving primary tumor growth appears to be HNF4G. Thus, HNF4G can induce chromatin opening at the target genes that are consistently expressed in patient primary tumor samples, but these genes are decreased in expression in metastases, mimicking the expression profile of HNF4G mRNA (Fig. HNF4G is a poorly studied NR, and it is currently classified as an ‘orphan' NR, because no endogenous ligands have been implicated for these orphan receptors25,26. Because there are no known or validated chemical inhibitors of HNF4G, we re-examined our quantitative RIME (Fig. 3a) data, with the goal of finding potential druggable enzymes that are associated with the FOXA1, GATA6 and HNF4G protein complex. We found protein arginine methyltransferase (PRMT1) to be a substantial and reproducible interactor of FOXA1 and HNF4G (Fig. Previous data had implicated PRMT1 as an essential gene in a subset of PDAC patient-derived xenograft tumors27. The transcriptionally active HNF4G-specific enhancer elements were cobound by PRMT1, as measured by ChIP–seq, and PRMT1 binding was dependent on HNF4G (Fig. We could also show PRMT1 cobinding with HNF4G (and FOXA1) in the Whipple surgical samples (n = 3; Fig. a, Volcano plot represents FOXA1-RIME candidates discovered in HPAF-II cells. The average of identified peptides in three biological replicates was calculated and proteins absent in IgG control samples were filtered for <3 unique peptides, P values of 0.003. b, Signal intensity plots generated from PRMT1 ChIP–seq in control and HNF4G-KO binding regions. c, Intensities of PRMT1 binding coverage measured at open sites (H3K27Ac) in proximity to HNF4G-regulated genes found in control, HNF4G-KO, HNF4G-KO + rescue conditions. d, ChIP–seq tag density heatmap of FOXA1, HNF4G and PRMT1 co-occupancy at PRMT1 binding sites from a representative patient tumor sample. Venn diagram is an overlap of binding sites of these factors from one representative patient. e, Distance correlation heatmap of ChIP–seq locations illustrates cobinding similarity of PRMT1, FOXA1 and HNF4G from three independent Whipple surgical samples. f,g, Control and HNF4G-KO cells were orthotopically implanted into the pancreas of mice. Mice with palpable tumors were then randomized into four groups and treated with GSK3368715 (oral administration of 75 mg kg−1 five doses per week) for 4 weeks, with n = 9 mice for control vehicle and HNF4G-KO vehicle arms and for control + GSK3368715 and HNF4G-KO + GSK3368715 arms n = 12. The lighter lines indicate mouse-specific longitudinal tumor volume measures, color-coded by specific groups. PRMT1 has been previously shown to be essential for the viability of specific PDAC patient-derived xenograft tumor models27. We could show that the same Type I-PRMT inhibitor (GSK3368715) was more effective at inhibiting growth in the HNF4G-KO models (Extended Data Fig. 4a–c), which was associated with an accumulation of the monomethylation mark, the basal state when type I PRMTs are inactive (Extended Data Fig. FOXA1 overexpression (OE) did not appreciably alter sensitivity to the PRMT inhibitor. We conducted a four-arm in vivo efficacy study in which control or HNF4G-KO cells were injected orthotopically into the pancreas of NSG mice and subsequently treated with vehicle or 75 mg kg−1 GSK3368715 (Fig. We observed a substantial survival advantage of 19 days in HNF4G-KO tumor-bearing mice (P < 0.03) that were treated with GSK3368715 (Fig. These data confirm a functional dependence between the NR HNF4G and the methyltransferase PRMT1 that can potentially be exploited therapeutically. Our data so far suggest that HNF4G can cobind with FOXA1 to the same genomic regions, but only HNF4G is essential for tumor growth, in part, by recruitment of PRMT1. To explore the potential connection further, we depleted FOXA1 and mapped HNF4G binding sites, or vice versa (Extended Data Fig. There was no appreciable loss of FOXA1 binding when the NR (HNF4G) was depleted (Extended Data Fig. 4i) and, notably, we did not see any reproducible decrease in HNF4G binding when FOXA1 was silenced (Extended Data Fig. 4i), confirming the evolving conclusion that FOXA1 is not a PF for the lineage-defining NR (that is, HNF4G) in primary PDAC. Although FOXA1 can contribute to the formation of new enhancers that drive metastasis10, our data suggest that it has no role in primary tumor contexts. To explore and validate the potential role of FOXA1 in PDAC, we conducted an orthotopic tumor experiment (Fig. FOXA1-OE did not change primary tumor growth in vivo compared to control mice (Fig. 4a), whereas HNF4G deletion had a substantial impact on primary tumor growth. FOXA1-OE was unable to reverse the tumor growth retardation resulting from the deletion of HNF4G, suggesting that HNF4G is the dominant transcription factor in the primary tumor context, whereas FOXA1 appears to lack any tumor-promoting activity in this context. Metastasis was not observed in control EV and FOXA1-OE mice due to the rapid primary tumor onset and the ethical requirement to kill the mice before metastases were detectable. a, Cells were orthotopically implanted into the pancreas of NSG mice (n = 5) for the following conditions: control, HNF4G-KO, FOXA1-OE and HNF4G-KO + FOXA1. Multiplicity Padj = 0.001 and 0.04 of random slope parameters were defined by means of an iterative bootstrap. The lighter lines and dots indicate mouse-specific longitudinal tumor volume measures, color-coded by specific groups. b, Number of metastases (y axis) as a function of the number of days since injection (x axis) and group (colors) for n = 10 mice (points). The colored lines show the metastatic burden fit of a Poisson generalized model with group and time as predictors. c, Kaplan–Meier plot showing the survival probability (y axis) as a function of time (x axis) per group (colored lines). Dots represent the time at which observations were right-censored. d, H&E images of the lung metastasis presented in histology. e, Halo analyses were performed to analyze % metastatic burden per lobe of the lungs n = 12 per cohort. Significance calculated by using a Mann–Whitney U test at P = 0.03. Next, to directly explore the role of FOXA1 and/or HNF4G in metastasis, engineered human cancer cell lines (HPAF-II) were injected into the tail vein of mice to directly assess metastatic potential. This confirms that FOXA1 does indeed promote metastasis as previously suggested10, but it does not influence primary tumor growth (Fig. The survival data from the metastatic experiment confirmed that overexpressed FOXA1 substantially shortens overall survival, but this was substantially blunted in the absence of HNF4G (Fig. Interestingly, when FOXA1 is OE in an HNF4G-KO context, these mice had extended survival, because of the HNF4G depletion, although they had widespread and substantial metastasis (Fig.4b,d,e). These data suggest that both transcription factors contribute to overall survival from PDAC, with FOXA1 mediating metastatic potential and HNF4G mediating primary growth, and possibly growth at secondary sites as well. To date, our data suggest that HNF4G is the primary driver of tumor growth, whereas FOXA1 becomes the driver of metastasis, and together, they both contribute to tumor progression and overall survival. To understand how a functional switch occurs from HNF4G-dependent state to a FOXA1-mediated metastatic state in the transition from primary tumor growth to metastatic progression, we explored our engineered human cancer cell line xenograft models. We performed RNA-seq on the collected primary tumors from the pancreatic orthotopic experiment, as well as FOXA1 ChIP–seq and ATAC–seq to map chromatin accessibility (Extended Data Figs. We observed a gain of 4,940 de novo FOXA1 binding sites when FOXA1 was overexpressed compared to EV (Extended Data Fig. These sites correlated with regions that were already open and accessible, as measured by ATAC–seq (Extended Data Fig. 5b), implying that an increase in FOXA1 levels results in opportunistic binding to regions that are already euchromatic and FOXA1 cannot create new enhancers, as would be expected from a genuine PF. Unexpectedly, no differential gene expression changes were observed after FOXA1-OE (Extended Data Fig. When HNF4G was silenced, however, 2,133 of these gained FOXA1 binding sites (following FOXA1-OE) became transcriptionally active (Fig. 5c–e), suggesting that the presence or absence of HNF4G dictates whether FOXA1 is active at these regions or not (Extended Data Fig. a, ChIP–seq heatmap represents coverage intensity in EV, FOXA1-OE, HNF4G-KO and HNF4G-KO + FOXA1-OE at 2,133 functional sites. b, Complex visualization of functional connection between FOXA1 occupancy and gene ontology groups in proximity to the functional binding sites. Color gradient reflects strength of the functional links (chr8: 125,386,596–125,482,166). c, Correlation of FOXA1 ChIP–seq occupancy and ATAC–seq locations in the FOXA1-OE model. The plot shows log2-fold changes in mapping densities between FOXA1 ChIP–seq and FOXA1-OE ATAC–seq. d, PanCuRx dataset was interrogated to compare primary tumor and metastatic samples. The samples were statistically tested using one-sided two-sample t test (P = 0.02). Representative images from one patient are presented. The areas on the right show nuclear staining of FOXA1 and substantial cytosolic staining of HNF4G. These new FOXA1 binding sites (that are normally restricted by HNF4G) result in induced gene expression profiles associated with metastasis (Fig. These HNF4G-restricted FOXA1 metastasis genes were correlated well with the gene expression profiles of metastatic patient samples (mets) when compared to a cohort of classical subtype primary tumor samples (from the PanCuRx dataset; Extended Data Figs. Up to this point, our data suggest that HNF4G is the driver in primary contexts, and it also restricts FOXA1's metastatic activity. We speculated that HNF4G's ability to restrict FOXA1 could be compromised in advanced disease, where metastatic target genes become induced by FOXA1. Analysis of the human PanCuRx dataset showed a global decrease in HNF4G mRNA expression in metastatic samples (mets) relative to primary tumors, with a coincident increase in FOXA1 expression (Fig. We also conducted IHC of HNF4G and FOXA1 from primary tumors and relatively rare metastatic human tissue (two independent cohorts). In primary tumors, we observed clear expression of HNF4G and FOXA1, which was restricted to the epithelial compartment and predominantly exhibited nuclear expression (Fig. However, in two independent cohorts, we observed either decreased HNF4G protein expression or cytoplasmic HNF4G staining in metastases, which was not observed in primary tumor samples (Fig. Our data suggest that multiple mechanisms exist for a tumor to inhibit HNF4G activity, allowing FOXA1 to become active and drive metastatic spread. To support our human clinical and preclinical data, we took an orthogonal approach and exploited the autochthonous KPC mouse model28. We derived and validated cell line models from KPC primary tumors or from matched liver metastases, permitting a comparison of isogenic matched cell line models (Extended Data Fig. There was a substantial gain of global Foxa1 binding in the liver metastasis (mets) cell line compared to the primary tumor cell line (Fig. 6a), with a total of 21,790 metastasis-specific gained Foxa1 binding sites (Fig. The ability of elevated Foxa1 to associate with these enhancers resulted in the induction of adjacent genes (Fig. 6e,f), which were enriched in pathways associated with metastasis. We exploited the published H3K27Ac ChIP–seq data derived from mouse organoid models10 and demonstrated that the Foxa1 binding observed specifically in our metastasis-derived KPC mouse cell line model was located at the same genomic regions where H3K27Ac was mapped in the published mouse metastatic organoids (Fig. a, FOXA1 ChIP–seq performed in KPC mouse-derived primary tumor cell line and matched liver metastasis (mets) cell line. Heatmaps illustrate normalized coverage density at common ChIP–seq sites. b, Foxa1 binding sites that were either common to both tumor and liver mets, tumor-unique or liver metastasis-unique (gained). The data have been analyzed using negative binomial GLM as a part of the DiffBind package at Padj < 0.05 threshold. c, Heatmaps represent occupancy of Foxa1 and Hnf4g at common sites in the metastatic cell line model, ordered by Foxa1 intensity. e, Difference in landscape of Foxa1 binding density across genomic ranges relative to TSS of induced genes, between mets and primary cell lines. f,g, Correlation of log2-fold changes between H3K27Ac (T versus M organoid models extracted from ref. 10) with ChIP–seq of Foxa1 (f) and Hnf4g (g) (mets versus primary tumor cell lines). h, Difference in Foxa1-induced chromatin loops in proximity to upregulated transcripts across genomic ranges relative to TSS, comparing mets and primary cell lines. i,j, An example of Foxa1 (i) and Hnf4g (j) associated Hi-ChIP loops in mets and tumors. The example shows looping between the c-Myc enhancers in mets and Foxa1 and Hnf4g binding is shown in blue. k, Combined genome-wide view of functional relationships between ChIP and Hi-ChIP data in the context of mets and tumor. Corresponding Hi-ChIP loops are shown as scatters in sectors mapped to chromosomes in the ideogram. The top scenario represents primary tumor contexts, where HNF4G is the dominant transcription factor, required for chromatin accessibility, gene expression and cell growth. The bottom scenario represents metastatic contexts, where FOXA1 becomes functional and the role of HNF4G is diminished. The switch from HNF4G dependency to FOXA1 dependency mediates the molecular transition from primary tumor growth to metastasis. We hypothesized that Foxa1 could promote preferential enhancer–promoter interactions in the metastatic context. To this end, we performed Hi-ChIP in the two isogenic mouse cell lines derived from the KPC mice and found that Hnf4g and Foxa1 were associated with chromatin looping (Fig. Hnf4g Hi-ChIP revealed elevated and preferential long-range looping in the primary tumor context compared to metastasis (Fig. In contrast, Foxa1 Hi-ChIP revealed a global gain of enhancer–promoter looping events specifically in the metastatic mouse cell line (Fig. These data implicate FOXA1 (Foxa1) in metastasis-specific chromatin looping interactions that culminate in the metastasis-specific gene expression program (Fig. Therefore, while HNF4G is the dominant transcription factor in primary tumor contexts, a switch occurs in advanced disease, resulting in engagement of FOXA1 to enhancers that create spatial genomic re-organization and ultimately in induction of metastasis-specific gene targets. Recent studies have implicated certain transcription factors as signature genes of specific PDAC subtypes and have linked several of these to disease progression2,5,22. Although transcription factors in the past have been considered ‘undruggable', NR transcription factors constitute realistic drug targets due to the presence of a ligand-binding pocket in the protein. To this end, our new insight reveals the NR HNF4G as a key driver in the primary tumor growth. Currently, HNF4G-targeted drugs do not exist and HNF4G is an orphan NR with no known endogenous ligand. One possibility is that HNF4G might lack an endogenous ligand and could have evolved as an NR-related transcription factor that does not functionally require the conserved ligand-binding domain. Alternatively, there could be an endogenous ligand for HNF4G that is a noncanonical compound. In support of this, X-ray crystallographic structures of HNF4G have revealed a common long-chain fatty acid, palmitic acid, occupying the ligand-binding domain pocket of HNF4G in these structures and palmitic acid has previously been shown to influence PDAC tumor growth and metastatic potential29,30. The family members of HNF transcription factors have been previously implicated as fundamental regulators of gene expression programs in bowel disease, colorectal cancer and PDAC2,11,14. 11 reported BMP/SMAD signaling as a mechanism to stabilize enterocyte identity via HNF4 transcription factors. Specifically, for PDAC, a study in ref. 12 observed that the loss of these endoderm-associated transcription factors (that is, HNF4A and GATA6) could result in altered metabolic profiles of cells associated with the squamous lineage. We did not find a substantial role for HNF4A in the classical subtype; however, a role for HNF4A may exist in different stages of the disease, particularly in late-stage disease. Surprisingly, little is known about the normal physiological role of HNF4G, very little information exists linking it to disease, and gene KO data suggest that it is not an essential gene for developmental processes and mouse viability. In contrast to HNF4G, FOXA1 has been previously implicated as a driver of PDAC metastasis, with elevated FOXA1 creating de novo enhancers associated with metastasis10. In both hormone-dependent breast and prostate cancers, FOXA1 forms a triumvirate with a GATA protein and an NR (GATA3 and ER in breast cancer; GATA2 and AR in prostate cancer) and is the foundational protein in these cancers35,36 due to its ability to create new enhancer elements37. FOXA1 could bind to active enhancers in PDAC (in primary tumor tissue and the various models) at the same places as HNF4G; we could not observe a role for FOXA1 in mediating chromatin accessibility, gene expression or cell viability in primary tumor contexts. This suggests a nonpioneering role for FOXA1 in PDAC and a dominant role for HNF4G in this subtype of the disease. In the primary tumor context, HNF4G was the dominant transcription factor that drives tumor growth, while also negatively regulating FOXA1 transcriptional activity. During the transition to metastasis, however, we observed a substantial decrease in HNF4G:FOXA1 ratios and, mechanistically, we could show that when HNF4G is depleted, FOXA1 becomes active and is associated with chromatin loops that drive expression of a distinct gene expression program associated with metastatic potential10. Matched flash frozen and formalin-fixed paraffin-embedded tissue (FFPE) samples from Whipple surgical samples undergoing surgical resection at the Cambridge University Hospitals were sectioned according to institutional protocols. This study was approved by the East of England—Cambridgeshire and Hertfordshire Research Ethics Committee and was conducted in compliance with Good Clinical Practice, local regulatory requirements and the legal requirements for CAMPAN (08/H0306/32). All procedures performed in studies involving human participants were in accordance with the ethical standards of the institutional and/or national research committee and with the 1964 Helsinki Declaration and its subsequent amendments or comparable ethical standards. Release of data was also pseudo-anonymized as per the UK Human Tissue Act regulations. Clinical data were obtained from archives and electronic health records. AsPC-1, HPAF-II, BXPC-3, Panc-1, MiaPaca (ATCC), KPC-derived cell lines and CRISPR-modified cell lines were grown in DMEM (Gibco, 41966-029), supplemented with 10% FBS (Gibco, 10500-064). Cells were routinely genotyped by short-tandem repeat genetic profiling using the PowerPlex 16HS cell line panel and analyzed using the Applied Biosystems GeneMapper ID (v3.2.1) software by the external provider Genetica DNA Laboratories (LabCorp Specialty Testing Group). Cells were tested using MycoProbe Mycoplasma detection kit (R&D) before experiments every 6 months. Three CRISPR guides were designed against exons 3 and 4 of HNF4G (ENSG00000164749). HPAF-II cells were electroporated (Amaxa 4D nucleofector, Lonza) with 5-μg TrueCut spCas9 protein V2 (Invitrogen) and 100 pmol of single guide RNA (Synthego), and used program DS137 and P3 nucleofector solution (Lonza). A cell pellet was taken 3 days after electroporation, and genomic DNA (gDNA) was extracted from each pool. Exon 3 or 4 of HNF4G was amplified by PCR (Q5 polymerase, NEB). Amplicons were subjected to Sanger sequencing and analyzed using the Synthego ICE web tool to calculate the percent editing. To generate clonal cells, HNF4G edited pools were single cell cloned into 96-well plates. gDNA was extracted from each clone and the editing analysis was performed similarly to the edited pool. Three individual clones were confirmed to be KOs and pooled together to make an HNF4G clonal pool. Three clones with heterozygotic frameshift mutations were used for subsequent experiments. The target frameshift mutations in these three clones were validated by Sanger sequencing. The HNF4G−/− pool of clones were short-tandem repeat genotyped and mycoplasma tested as described under ‘Cell lines'. RIME on cell lines was performed as described previously39. Briefly, cells were cross-linked at room temperature by incubating with 2 mM di(N-succinimidyl) glutarate (DSG) for 20 min followed by 1% formaldehyde for 10 min before quenching in 0.1-M glycine for 10 min. Cells were washed twice in cold PBS and collected in cold PBS containing protease and phosphatase inhibitors(Roche, Complete, 05056489001) and Halt inhibitors (Thermo Fisher Scientific, 78427). Chromatin was sonicated using the Bioruptor Plus (Diagenode) for 15 min (30-s on/30-s off) to generate fragments of around 100–800 bp. Chromatin was immunoprecipitated overnight using protein A Dynabeads (Invitrogen; used for rabbit antibodies) or protein G Dynabeads (Invitrogen; used for goat antibodies) conjugated to specific antibodies against HNF4G (Atlas, HPA005438), HNF4A (Cell Signaling Technology, 3113S), PRMT1 (Atlas, HPA072136), GATA6 (Cell Signaling Technology, 5851; R&D Systems, AF1700) and FOXA1 (Abcam, ab5089). Beads were washed ten times in RIPA buffer (50 mM, pH 7.6, 1 mM EDTA, 0.7% Na-deoxycholate, 1% NP-40, 0.5 M LiCl) followed by two washes in AMBIC (100 mM ammonium hydrogen carbonate). Peptides were analyzed in the Dionex UltiMate 3000 UHPLC system coupled with the Q-Exactive HF (QE-HF) or QE (Thermo Fisher Scientific) mass spectrometers. For peptide separation, the EASY-Spray analytical column (75 μm × 25 cm, C18, 2 μm, 100 Å) was used for multistep gradient elution. For MS2, the ten most intense precursors were selected with resolution of 30k (QE-HF) or 17.5k (QE). The HCD tandem mass spectra were processed with the Sequest HT search engine in Proteome Discoverer 1.4 or 2.2 software. For downstream statistical analysis, the Perseus software was used40. Clinical samples and cell lines ChIP–seq on tumor material were performed essentially as described previously41. Briefly, flash-frozen pancreatic cancer or normal adjacent tissue from patients was cryosectioned into 30-μm sections, cross-linked by 2 mM DSG for 45 min, total together with 1% formaldehyde for the last 20 min and quenched by 0.1 M glycine for 10 min at room temperature. Chromatin was immunoprecipitated, washed and processed for sequencing. ChIP–seq was performed on three or four independent biological replicates. Chromatin was prepared from both cell lines/tumors and immunoprecipitated as described for RIME above. Chromatin was eluted and decrosslinked by incubating overnight at 65 °C in elution buffer (50 mM Tris–HCl, pH 8.0, 10 mM EDTA, 1% SDS). Samples were treated with (20 ng ml−1) RNase A for 30 min–1 h followed by proteinase K (200 ng ml−1) for 1–2 h before DNA was purified by phenol-chloroform extraction. Purified DNA was subjected to library preparation using the SMARTer ThruPLEX DNA-seq Kit (Takara Bio, R400676) and DNA HT Dual Index Kit—96N Set A (Takara Bio, R400660) followed by next-generation sequencing to reach approximately 30 million reads per sample. A total of 50-bp single-end reads were mapped to the hg38 genome using bowtie2 (v2.2.6). Aligned reads with mapping quality <5 were filtered out. The read alignments from all replicates were combined into single library and peaks were called with MACS2 (version 2.1.1.2016) using input sequences as a background control. The peaks that absorbed statistically significant tag density from all replicates were selected for downstream analysis. MEME (version 4.9.1) was used to detect known and discovered new binding motifs among tag-enriched sequences. Differential binding analysis (DiffBind) was performed as described previously42. As a background control, binding sites near an equal number of randomly expressed genes revealed by the RNA-seq analyses were tested in 100 permutations. The fraction of binding sites near these groups of genes was then visualized as curves, including the s.d. for the random constitutively expressed control genes as error bars. Networking analysis of predicted gene associations was performed using the STRING application plugin with Cytoscape (version 3.10.3). Total RNA was purified using RNeasy kit (Qiagen, 74106) and mRNA libraries were prepared using the stranded mRNA library preparation kit (Illumina, 20020595) and IDT for Illumina—TruSeq RNA UD Indexes (20022371). Four biological replicates were conducted for all RNA-seq experiments. A total of 50 bp single-end reads were aligned to the human genome (GRCh37) or mouse genome (GRCm38) using STAR (version 2.6.1a). Differentially expressed genes were identified based on the negative binomial distribution using the DESeq2 package in R (version 1.14.1). Heatmap visualization, clustering and various plotting functions were used from statistical and bioinformatic modules in framework. The cells were incubated on ice for 3 min to lyse the cells and washed with 1 ml of cold ATAC–RSB containing 0.1% Tween-20, but without NP-40 and digitonin. Zymo DNA Clean and Concentrator-5 kit (D4014) was used to clean up the reaction and DNA was eluted in 21 μl elution buffer and stored at −20 °C until amplification. Samples were size selected and purified using Ampure XP beads. Furthermore, NGS was performed using Illumina NovaSeq 6000 with SP flow cell system with 50-bp paired end. Whole-cell extract was sonicated using the Bioruptor Plus (Diagenode) for 2 min (30-s on/30-s off) to degrade the DNA, quantified by Direct Detect spectrometer Millipore) and run on NuPAGE 4–12% Bis–Tris gels (Invitrogen). Proteins were transferred to a membrane by iBlot2 (Invitrogen), and the membrane was then blocked with Odyssey TBS blocking solution (Li-Cor, 927-50000) and incubated with antibody overnight at 4 °C. After washing, the membrane was then incubated with appropriate secondary antibodies (Li-Cor) and developed using the Odyssey CLx Imaging System (Li-Cor). Primary antibodies were as follows: FOXA1 (Abcam, ab23738), HNF4G, HNF4A, GATA6 (Cell Signaling Technology, 5851; monomethyl arginine, 8015; Vinculin, 13901; β-actin, 4970); asymmetric dimethyl (arginine, 13522; Sigma-Aldrich, A5441). Five tumors per treatment arm were fixed in formalin for 24 h and subsequently transferred to 70% ethanol and paraffin embedded. Sections were cut, dewaxed, rehydrated and subsequently stained with antibodies on Ventana (Roche) using the fully automated Ventana Discovery ULTRA (Roche) with Ventana solutions. Paraffin sections were pretreated with heat using standard conditions (40 min) in CC1 solution. The primary antibody was applied and incubated for specific times (listed below). After incubation with a rabbit Impress HRP antibody (Vector Laboratories), chromogenic revelation was performed with ChromoMap DAB kit (Roche). Thus, IHC staining for FOXA1, HNF4G and PRMT1 was performed with antibodies described above. Slides were scanned using an Aperio AT2 (Leica) at ×20 magnification (resolution = 0.5 μm per pixel). Hi-ChIP was performed using a previously published protocol. Once the ligation step was completed by adding 50 µl of 1 U µl−1 T4 DNA ligase (Invitrogen, 15224-025) and incubating for 4 h at room temperature with gentle mixing on shaker (lowest speed), the chromatin was spun down, and the ChIP–seq protocol as described above was followed. Default settings were used to remove duplicate reads, assign reads to Dpn2 restriction fragments, filter for valid interactions and generate binned interaction matrices44,45. All experiments were performed in accordance with the animal project licenses P40BD8F30, P1919999D and PP79993249 in accordance with Home Office guidelines. NSG mice (~8 weeks old) were obtained from Charles River Laboratories, kept in pathogen-free conditions on a 12-h light/12-h dark cycle and allowed to acclimatize for a period of 7 days before any surgery. Mice were grouped, housed with environmental enrichment, fed on a maintenance diet (PicoLab) and housed with a room temperature of +22 °C/−2 °C and humidity of +55%/−10%. For survival studies, humane endpoints were used with the loss of 15% of pre-implantation weight, body condition score <2.0. Grimace scale (MSG > 1.2/2) scoring was adhered to alongside other clinical signs. All mice were randomly assigned to cohorts as necessary, and analyses were performed in a blinded manner. Pancreatic orthotopic injections were performed as described previously46. Mice were allowed to recover from surgery before the onset of routine palpations (weekly from 2 weeks postsurgery) to determine the presence of any pancreatic masses. Once confirmed, mice were serially imaged (weekly) using the Vevo 2100 ultrasound imaging platform (Visual Sonics; 13–24-MHz transducer) to monitor pancreatic tumor progression until clinical endpoint. All study images were randomized and blinded before calculating tumor volumes using 3D volume analysis software (Vevo LAB, version 3.1.1) on images taken at the same orientation for each mouse throughout the study, whenever possible. Mice were randomized into four groups (n = 9 mice for control vehicle and HNF4G-KO vehicle arms; n = 14 for control + GSK3368715 arm and n = 15 for HNF4G-KO + GSK3368715 arm). At 2 weeks postsurgery, mice were treated with either vehicle (control, Vetivex saline) or GSK3368715 (75 mg kg−1) by oral gavage (5 ml kg−1), using a dosing regimen of 5 days on/2 days off until clinical endpoint was reached. Pancreatic cancer cells were injected into the tail vein of NSG mice (n = 12 per group). HPAF-II cells were prepared as a suspension (in PBS) and injected (0.5 × 106 cells in 100 µl) into the tail vein of NSG mice. Mice were monitored daily for signs of ill health or body weight loss for the first 7 days postinjection and then, at routine, periodic intervals until clinical endpoints were reached, whereby necropsies were performed and metastatic burden assessed, complemented by histological analyses. Mice were allowed to recover from surgery for 2 weeks before the onset of routine health checks. Mice were monitored for clinical endpoints, and necropsies were performed to assess metastatic burden by histology. 2b, we used a two-sided two-sample Mann–Whitney–Wilcoxon test (function Wilcox test of R stats package) to test if the median of the difference between a sample of each group is different from 0. 1c, due to the presence of right censoring in the TCGA group, we used a log-rank test (function survdiff of the R survival package) to assess whether there is a difference between the two survival curves. 4a, due to the absence of right censoring, we considered one-sided Welch tests (function t test of the R stats package) to assess whether the average log survival times of each group compared to the reference (control–vehicle) were significant, and the P values by means of a Bonferroni multiplicity correction. Sensitivity analyses, considering Wald t-tests based on a log-normal model and Mann–Whitney–Wilcoxon tests, led to the same conclusions. 2c and 4b,e, we used parametric linear mixed models to fit tumor sizes of mice on the cube root (Figs. These transformations were selected to obtain a linear relationship between tumor size and time on the transformed scales and to tame heteroscedasticity. In each analysis, groups (considered as factor with a reference group), time (in days) and groups interacting with time were used as fixed effects. 2c and 4e) were used to take the longitudinal within-mouse tumor size dependence into account. 2c, the difference in growth between the control and HNF4G-KO groups was assessed by means of a likelihood ratio test comparing the likelihood (of the maximum likelihood estimator) of a restricted model assuming the same average growth for both groups and a full model that allows both groups to have a different average growth. Inference for the parameters related to the growth time lag between the HNF4G and control groups, as well as the difference in growth between each group and HNF4G-KO-GSK, was based on Wald z tests, considering a multiplicity correction for parametric models (function glht of the R multcomp package). 2c and 4e, models were fitted using standard R routines (function lme of the nlme R package). 4e, due to the combination of small sample size per group and the presence of left truncation, we used an estimator based on the iterative bootstrap. This allows us to correct for a potential left-truncation bias and to perform a finite sample correction for the parameters. Its initial estimator was a random slope model with the fixed effects described above and a linear interpolation of left-truncated observations. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All ChIP–seq, Hi-ChIP, ATAC–seq and RNA-seq data have been deposited at Gene Expression Omnibus and can be accessed at GEO submission (GSE245734). Please access data using the link https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE245734. Source data are provided with this paper. Biankin, A. V. et al. Pancreatic cancer genomes reveal aberrations in axon guidance pathway genes. Bailey, P. et al. Genomic analyses identify molecular subtypes of pancreatic cancer. Humphris, J. L. et al. Hypermutation in pancreatic cancer. Whole genomes redefine the mutational landscape of pancreatic cancer. Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma. Opening of compacted chromatin by early developmental transcription factors HNF3 (FoxA) and GATA-4. The androgen receptor cistrome is extensively reprogrammed in human prostate tumorigenesis. Hurtado, A., Holmes, K. A., Ross-Innes, C. S., Schmidt, D. & Carroll, J. S. FOXA1 is a key determinant of estrogen receptor function and endocrine response. Roe, J. S. et al. Enhancer reprogramming promotes pancreatic cancer metastasis. Chen, L. et al. A reinforcing HNF4–SMAD4 feed-forward module stabilizes enterocyte identity. Brunton, H., et al. HNF4A and GATA6 loss reveals therapeutically actionable subtypes in pancreatic cancer. Okegawa, T., Ushio, K., Imai, M., Morimoto, M. & Hara, T. Orphan nuclear receptor HNF4G promotes bladder cancer growth and invasion through the regulation of the hyaluronan synthase 2 gene. Shukla, S. et al. Aberrant activation of a gastrointestinal transcriptional circuit in prostate cancer mediates castration resistance. Wang, C. et al. Metformin inhibits pancreatic cancer metastasis caused by SMAD4 deficiency and consequent HNF4G upregulation. Expression of HNF4G and its potential functions in lung cancer. & Zhao, C. HNF4G stimulates the development of pancreatic cancer by promoting IGF2BP2 transcription. Klein, A. P. et al. Genome-wide meta-analysis identifies five new susceptibility loci for pancreatic cancer. Mohammed, H. & Carroll, J. S. Approaches for assessing and discovering protein interactions in cancer. Papachristou, E. K. et al. A quantitative mass spectrometry-based approach to monitor the dynamics of endogenous chromatin-associated protein complexes. Collisson, E. A. et al. Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy. Martinelli, P. et al. GATA6 regulates EMT and tumour dissemination, and is a marker of response to adjuvant chemotherapy in pancreatic cancer. Transcription phenotypes of pancreatic cancer are driven by genomic events during tumor evolution. Isigkeit, L. & Merk, D. Opportunities and challenges in targeting orphan nuclear receptors. Wisely, G. B. et al. Hepatocyte nuclear factor 4 is a transcription factor that constitutively binds fatty acids. Giuliani, V. et al. PRMT1-dependent regulation of RNA metabolism and DNA damage response sustains pancreatic ductal adenocarcinoma. Hingorani, S. R. et al. Trp53R172H and KrasG12D cooperate to promote chromosomal instability and widely metastatic pancreatic ductal adenocarcinoma in mice. Binker-Cosen, M. J. et al. Palmitic acid increases invasiveness of pancreatic cancer cells AsPC-1 through TLR4/ROS/NF-κB/MMP-9 signaling pathway. Chen, L. et al. HNF4 regulates fatty acid oxidation and is required for renewal of intestinal stem cells in mice. Kalisz, M. et al. HNF1A recruits KDM6A to activate differentiated acinar cell programs that suppress pancreatic cancer. & Zaret, K. S. An early developmental transcription factor complex that is more stable on nucleosome core particles than on free DNA. Lupien, M. et al. FoxA1 translates epigenetic signatures into enhancer-driven lineage-specific transcription. Zhang, G. et al. FOXA1 defines cancer cell specificity. Differential oestrogen receptor binding is associated with clinical outcome in breast cancer. Broome, R. et al. TET2 is a component of the estrogen receptor complex and controls 5mC to 5hmC conversion at estrogen receptor cis-regulatory regions. A., Nagarajan, S. & Carroll, J. S. FOXA1 directs H3K4 monomethylation at enhancers via recruitment of the methyltransferase MLL3. Carroll, J. S. et al. Chromosome-wide mapping of estrogen receptor binding reveals long-range regulation requiring the forkhead protein FoxA1. The Perseus computational platform for comprehensive analysis of (prote)omics data. Siersbæk, R. et al. IL6/STAT3 signaling hijacks estrogen receptor α enhancers to drive breast cancer metastasis. Stark, R. & Brown, G. DiffBind: differential binding analysis of ChIP–seq peak data. Corces, M. R. et al. An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. & Aryee, M. J. Hichipper: a preprocessing pipeline for calling DNA loops from HiChIP data. Mumbach, M. R. et al. HiChIP: efficient and sensitive analysis of protein-directed genome architecture. Erstad, D. J. et al. Orthotopic and heterotopic murine models of pancreatic cancer and their different responses to FOLFIRINOX chemotherapy. Couturier, D. L. et al. CarrollLab: transcription factor switching drives progression of the classical subtype of pancreatic cancer (v1.0). The authors thank T. Halim for his technical support with the in vivo experiments and G. Biffi for her helpful advice and discussion. The authors thank the core facilities at Cancer Research UK (Genomics, Proteomics, Pharmacokinetics and Bioanalytics, Histopathology, Preclinical genome editing, Bioinformatics and Research Instrumentation) for their consistent support and advice. We thank V. Gourgiotis and R. Ranftl for technical support. The authors also thank the Cambridge University Hospitals Tissue Bank for its integral role in the Cambridge NIHR Biomedical Research Center for providing tissue samples. would like to acknowledge NIHR Cambridge Biomedical Research Centre (NIHR203312). The authors would like to acknowledge the funding support from the University of Cambridge, Cancer Research UK Core funds (grant G101107), Pancreatic Cancer Research Fund (PCRF; G110303 to J.S.C. Shalini V. Rao, Lisa Young, Danya Cheeseman, Dominique-Laurent Couturier, Stephanie Mack, Jill Temple, Amy Smith, Evangelia Papachristou, Catarina Pelicano, Chandra Sekhar Reddy Chilamakuri, Krzysztof Herka, Michael Gill, Emily Archer Goode, Steven Kupczak, Yi Cheng, Giacomo Borsari, Duncan Jodrell, Clive D'Santos, Alasdair Russell, Igor Chernukhin & Jason S. Carroll Bridge Institute of Experimental Tumor Therapy (BIT) and Division of Solid Tumour Translational Oncology, German Cancer Consortium (DKTK), University Hospital Essen, Essen, Germany Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar Search author on:PubMed Google Scholar has received past research funding from AstraZeneca, which is unrelated to this work. receives honoraria as a consultant or for continuing medical education presentations from AstraZeneca, Bayer, Boehringer Ingelheim, Bristol Myers Squibb, Immunocore, MSD, Novartis, Roche/Genentech and Servier. His institution receives research funding from Abalos Therapeutics, Boehringer Ingelheim, Bristol Myers Squibb, Celgene, Eisbach Bio and Roche/Genentech; he holds ownership in FAPI Holding (3%); all are unrelated to the submitted work. Nature Genetics thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The MA plot shows differential binding intensity in H3K27Ac ChIP-seq between tumor and normal adjacent tissue. Heatmap represents hierarchical clustering of H3K27Ac-normalized intensities of called peaks (3,000 sites) in normal and tumor samples. c. Giggle cistrome analysis reveals FOXA1 as a dominant TF in the H3K27Ac regions abundant in tumor tissue. d. Box plots show the gene expression levels of FOXA1 in clinical samples. 5. e. Tagged intensity heatmaps showing binding of FOXA1, HNF4G and HNF4A across patient samples. f. Top enriched motifs discovered during analysis of called peaks from clinical samples (n = 4). g. RIME in a panel of pancreatic cancer cell lines. Three biological replicates of the FOXA1 RIME and one pooled immunoglobulin G (IgG) control RIME were included in each experiment. To identify specific interactors, a label-free quantification method was used. Significantly enriched interactors are highlighted on the heatmap padj < 0.02. h. Heatmaps illustrate normalized coverage density at common ChIP-seq sites for FOXA1, HNF4G, HNF4A and GATA6 in the AsPC-1 cell line model. UCSC tracks show common binding sites for the respective pull downs. a. Heatmaps illustrate normalized coverage density at common ChIP-seq sites for HNF4G, HNF4A in the AsPC-1 model. Venn diagram shows shared binding between the factors. b. HNF4A ChIP-seq performed in HPAF-II human tumor cell line +/− HNF4G-KO. Heatmaps illustrate normalized coverage density at common ChIP-seq sites. HNF4A binding sites that were either common, gained or lost in the +/− HNF4G-KO context represented as an MS plot (on the right), subsets measured occupancy for each context using DiffBind at padj < 0.05 threshold. c. Perturbation ChIP-seq performed in HPAF-II human tumor cell line with the following conditions control (pink-GATA6 I.P), GATA-KO (green-HNF4G-KO I.P), HNF4G-KO + GATA6-KO (red-FOXA1 I.P). d. Western blots in PDAC cell lines showing expression levels of FOXA1, GATA6, HNF4A and HNF4G. Normalization controls include vinculin, GAPDH and tubulin. Data presented are from three independent biological replicates. Significance was calculated with multiple comparison ANOVA (p-value ****<0.01, **<0.02). g. Colony formation assay (10 nM siRNA and CRISPR KO in HPAF-II cells) was performed in tandem with growth experiments. Cells were monitored, stained with crystal violet at endpoint (day 15). Representative images in HPAF-II cells are from an independent well of a 24-well plate. h. An independent set of siRNAs 1 (smart pool), siRNAs 2 (stealth pool) were used to treat HPAF-II cell line models, at 10 nM to perform the proliferation assays using the Incucyte (72 h). Non-targeting (NT), death oligo (DO) HNF4G and HNF4A siRNAs were used. Representative western blots showing the knockdown efficiency using the stealth siRNA targeted towards HNF4A and HNF4G in the HPAF-II cells at endpoint (72 h) compared to untreated cells and non-targeting control (NT). An independent set of 2 siRNAs (smart pool and stealth siRNA) were used in the AsPC-1 cell line, at 10 nM. a&b Two independent CRISPR guides were used towards the KO of HNF4G and HNF4A in HPAF-II cell line models, proliferation assays were performed using the Incucyte (72 h). d. Proliferation assay performed in 9 independent CRISPR HNF4G-KO clones compared to control (Cas9) and wildtype (WT) cells using the Incucyte. e. Western blots performed in the HPAF-II cells to determine knockout efficiency of the gRNAs targeting HNF4A and HNF4G. The western blots were normalized using vinculin as a loading control. A representative image of the staining is shown. g. Results of mRNA expression analysis comparing control versus HNF4G-KO tumors. GSEA analysis of mRNA expression revealed meaningful matches with gene sets from the MySigDB hallmark database. h. Differential pathway analysis of the HNF4G-KO tumors, highlighted in red—upregulated, blue—downregulated GSEA gene expression sets. Heatmap represents differentially regulated genes from the EMT pathway extracted from gene set enrichment analysis (GSEA). i&j GSEA analysis comparing HNF4G-KO tumors with control. Snapshots of the top two downregulated pathways. k. Heatmaps anchored on PRMT1 (n = 2759 sites) showing co-binding of FOXA1 and HNF4G at these sites in multiple patient tumor samples. Representative image of PRMT1 IHC staining in patient samples used for ChIP-seq. a. Dose–response curves for GSK3368715 treatment in the SUIT-II (+/− HNF4G-KO) and HPAF-II cells (+/− HNF4G-KO) performed at 72 h using the CellTiter-Glow (CTG). Treatment of SUIT-II (+/− HNF4G-KO) cells with GSK3368715 (10 µM) for 72 h endpoint CTG and western blot representing CRISPR KO efficiency of HNF4G in SUIT-II cells P-values *** <0.02, * <0.05. c. Treatment of HPAF-II (+/− HNF4G-KO) cells with GSK3368715 (10 µM) for 72 h endpoint CTG. d. Colony formation assay (day 15) performed in the HPAF-II cells with control, HNF4G-KO, HNF4G-O.E, FOXA1-O.E and HNF4G-KO + FOXA1 (n = 3) cell were stained with crystal violet. e. Crystal violet-stained colonies were solubilized and absorbance was measured (592 nm) for the control and HNF4G-KO conditions treated with GSK3368715. Two-way ANOVA analysis was performed on n = 3 independent experiments padj <0.02. f. Western blots were performed in the HPAF-II cells treated with +/− GSK3368715 for 72 h. Blots represent monomethyl-arginine (MMA), asymmetric dimethyl-arginine antibody (ADMA) and PRMT1 protein levels. g. Representative image of PRMT1 IHC staining in control, HNF4G-KO tumors +/− GSK3368715 treatment. h. Western blots were performed in the HPAF-II cells treated with non-targeting siRNA and FOXA1-siRNA cells in the HPAF-II (treated for 72 h at 20 nM). Blots represent FOXA1 protein expression levels from three independent biological replicates. Signal intensity heatmaps of FOXA1 binding in the control and HNF4G-KO HPAF-II cells. Signal intensity heatmaps of HNF4G binding in the non-targeting (NT) siRNA and FOXA1-siRNA-transfected HPAF-II cells (transfected for 72 h at 20 nM). Data presented are the cumulative intensity from three independent biological replicates. Blots represent Flag, FOXA1, HNF4G protein expression levels from one representative biological replicate. GSEA snapshot representing EMT gene signature is presented. Representative MA plots with log2-fold change between conditions FOXA1-O.E vs control. Correlation of log2-fold change between FOXA1 ChIP-seq and FOXA1-OE ATAC-seq densities compared to empty vector control (EV). Heatmap represents the open regions from ATAC-seq data between control and FOXA1-OE (n = 4). d. Representative MA plots with log2-fold change between conditions HNF4G-KO + FOXA1-OE vs FOXA1-OE. Binding distribution changes between HNF4G-KO + FOXA1-OE vs control from FOXA1 ChIP-seq data. The sites are classified as highly enriched using FDR (padj < 0.05). e. Binding distribution changes between HNF4G-KO + FOXA1-OE vs control from FOXA1 ChIP-seq data. The sites are classified as highly enriched using FDR (padj < 0.05). Normalized counts of differentially expressed genes represented as a log2-fold change between comparisons. Data extracted from RNA-seq analyses performed on orthotopic tumors and metastatic samples of NSG mice. m. Differentially expressed genes from RNA-seq analysis were evaluated in samples from the PanCuRx study (130 samples). The expression pattern of the selected subset of genes derived from DeSeq2 analysis was assessed in classical subtype tumors and corresponding metastasis (mets) samples. a. Functional relation of FOXA1-dependent sites to gene expression. Curves correspond to the cumulative percentage of FOXA1 binding sites within a distance from the transcription start site (TSS) of either significantly up-regulated or constantly expressed genes in HNF4G-KO + FOXA1-OE. b&c IHC images from advanced PDAC patients (independent cohort). d. Heatmap representing the open, accessible regions extracted from ATAC-seq data comparing KPC-derived primary tumor cell lines and matched liver metastasis cell lines (n = 4). e. Genomic feature distribution from ATAC-seq profiles between primary tumor cell lines and liver metastatic cell lines across four biological replicates. f. RNA-seq analysis was performed in KPC-derived matched primary tumor and liver metastasis cell lines (n = 4). Heatmaps highlight differentially regulated genes, padj < 0.01, using DESeq2 across four independent biological replicates. g. IGV (genome browser) screenshots representative of Foxa1 (red) and Hnf4g (green) binding sites in the KPC-derived isogenic cell lines. Distribution of loop-supporting normalized PETs across samples represented as violin plots. c, Density landscape of Foxa1-associated chromatin loops in proximity to up-regulated transcripts in mets versus tumor cell lines across genomic ranges relative to TSS. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Transcription factor switching drives subtype-specific pancreatic cancer. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='http://www.sciencedaily.com/releases/2025/10/251029100158.htm'>Scientists discover a stunning new golden-tongued lizard in China</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'http://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-10-30 10:24:10
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Researchers in China have identified a previously unknown species of mountain lizard living in the upper Dadu River Valley, deep within the Hengduan Mountains of Sichuan Province. During their expeditions, they came across a population of lizards that displayed unusual traits not seen in other known Diploderma species from the area. Detailed genetic testing and morphological comparisons confirmed their suspicions: this was a species that had never been documented before. Members of this genus are found throughout East Asia and the northern Indochinese Peninsula, where they occupy a wide variety of mountain habitats. Measuring about 6-7 centimeters in body length, D. bifluviale stands out with its distinctive coloring and a wheat-colored tongue -- features that set it apart from closely related species. Their finding underscores how even in well-surveyed regions of China, nature continues to reveal new surprises. Note: Content may be edited for style and length. Why “Walking with Purpose” Could Be the Secret to Better Health Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            