
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-05-14</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250514120231.htm'>New generation of skin substitutes give hope to severe burns patients</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 19:32:26
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Severe burns remain one of the most challenging injuries to treat, causing high disease and death rates worldwide, but Australian researchers have flagged some promising new approaches that could save lives and dramatically improve patient recovery. In a comprehensive review published in Advanced Therapeutics, researchers from the University of South Australia (UniSA), University of Adelaide and Royal Adelaide Hospital (RAH) explore the latest advancements in dermal substitutes -- biochemicals used to replace damaged skin -- with a particular focus on combating infection and enhancing tissue regeneration following catastrophic burns. The researchers say that despite decades of progress, traditional treatments such as skin grafting often fail to provide adequate healing and infection control, leading to prolonged hospital stays and soaring healthcare costs. "We must innovate beyond conventional methods and develop therapies that regenerate tissue while actively preventing infections." Each year, approximately 2423 Australians are admitted to hospital with burn-related injuries, 74% of whom require surgery, including a skin graft. Globally, 180,000 people die from burns each year, and approximately 10 million are hospitalised, costing healthcare systems $112 billion worldwide. The review highlights that while many commercial skin substitutes exist, very few offer integrated antimicrobial protection -- a critical factor given the vulnerability of burn wounds to bacterial invasion and sepsis. The paper discusses emerging technologies such as Kerecis, a novel fish skin graft with inherent antimicrobial properties, and NovoSorb BTM, a synthetic biodegradable matrix that resists bacterial colonisation without relying on antibiotics. Both products represent a new generation of dermal substitutes with enhanced potential to protect and heal complex burns. It stands out for retaining natural omega-3 fatty acids, which have strong antimicrobial effects and promote wound healing. Meanwhile, NovoSorb BTM's unique polyurethane matrix offers structural resilience even in infected wounds, providing a vital scaffold for tissue regeneration. "Such innovations are crucial, particularly as antibiotic-resistant infections continue to rise globally," she says. By combining smart biomaterials with cell-based therapies, scientists aim to regenerate skin that restores its full function -- an outcome that could revolutionise the recovery for millions of burn survivors worldwide. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/05/250514111342.htm'>Sugar-coated nanotherapy dramatically improves neuron survival in Alzheimer's model</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 18:29:15
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Scientists at Northwestern University have developed a new approach that directly combats the progression of neurodegenerative diseases like Alzheimer's disease and amyotrophic lateral sclerosis (ALS). In these devastating illnesses, proteins misfold and clump together around brain cells, which ultimately leads to cell death. "In many of these diseases, proteins lose their functional folded structure and aggregate to make destructive fibers that enter neurons and are highly toxic to them. Early stage, short amyloid fibers, which penetrate neurons, are believed to be the most toxic structures. With further work, we think this could significantly delay progression of the disease." Zijun Gao, a Ph.D. candidate in Stupp's laboratory, is the paper's first author. Co-corresponding author Zaida Alvarez -- a researcher at the Institute for Bioengineering of Catalonia (IBEC) in Spain, former postdoctoral fellow in Stupp's laboratory and current visiting scholar at CRN -- led testing of the therapies in human neurons. While current treatments offer limited relief, a dire need for new therapies remains. To tackle this challenge, the researchers turned to a class of peptide amphiphiles, pioneered by the Stupp laboratory, that contain modified chains of amino acids. Peptide amphiphiles are already used in well-known pharmaceuticals including semaglutide, or Ozempic. "The advantage of peptide-based drugs is that they degrade into nutrients," Stupp said. "The molecules in this novel therapeutic concept break down into harmless lipids, amino acids and sugars. Over the years, Stupp's research group has designed many peptide-based materials for different therapeutic purposes. To develop a peptide amphiphile to treat neurodegenerative diseases, his team added an extra ingredient: a natural sugar called trehalose. "Trehalose is naturally occurring in plants, fungi and insects," Gao said. "It protects them from changing temperatures, especially dehydration and freezing. Others have discovered trehalose can protect many biological macromolecules, including proteins. When added to water, the peptide amphiphiles self-assembled into nanofibers coated with trehalose. Although it seems counterintuitive, this decreased stability exhibited a beneficial effect. That makes it more difficult for other molecules, like misfolded proteins, to integrate into the fibers. Less stable fibers, on the other hand, became more dynamic -- and more likely to find and interact with toxic proteins. "Unstable assemblies of molecules are very reactive," Stupp said. If the nanofibers were stable, they would happily ignore everything around them." Searching for stability, the nanofibers bonded to amyloid-beta proteins, a key culprit implicated in Alzheimer's disease. "Then, it's no longer a peptide amphiphile fiber anymore," Stupp said. That means the nasty amyloid-beta proteins, which would have formed amyloid fibers, are trapped. They can no longer penetrate the neurons and kill them. It's like a clean-up crew for misfolded proteins. "This is a novel mechanism to tackle progression of neurodegenerative diseases, such as Alzheimer's, at an earlier stage. Current therapies rely on the production of antibodies for well-formed amyloid fibers." Stupp says the novel approach of using unstable nanofibers to trap proteins offers a promising avenue for developing new and effective therapies for Alzheimer's, ALS and other neurodegenerative conditions. Much like cancer treatments combine multiple therapies -- like chemotherapy and surgery or hormone therapy and radiation -- Stupp said the nanotherapy might be most effective when combined with other treatments. "Our therapy might work best when targeting diseases at an earlier stage -- before aggregated proteins enter cells," Stupp said. "But it's challenging to diagnose these diseases at early stages. So, it could be combined with therapies that target later-stage symptoms of the disease. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09018-7'>Taurine from tumour niche drives glycolysis to promote leukaemogenesis</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:31:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Signals from the microenvironment are known to be critical for development, stem cell self-renewal and oncogenic progression. Although some niche-driven signals that promote cancer progression have been identified1,2,3,4,5, concerted efforts to map disease-relevant microenvironmental ligands of cancer stem cell receptors have been lacking. Here, we use temporal single-cell RNA-sequencing (scRNA-seq) to identify molecular cues from the bone marrow stromal niche that engage leukaemia stem-enriched cells (LSCs) during oncogenic progression. We integrate these data with our human LSC RNA-seq and in vivo CRISPR screen of LSC dependencies6 to identify LSC–niche interactions that are essential for leukaemogenesis. These analyses identify the taurine–taurine transporter (TAUT) axis as a critical dependency of aggressive myeloid leukaemias. We find that cysteine dioxygenase type 1 (CDO1)-driven taurine biosynthesis is restricted to osteolineage cells, and increases during myeloid disease progression. Blocking CDO1 expression in osteolineage cells impairs LSC growth and improves survival outcomes. Using TAUT genetic loss-of-function mouse models and patient-derived acute myeloid leukaemia (AML) cells, we show that TAUT inhibition significantly impairs in vivo myeloid leukaemia progression. Consistent with elevated TAUT expression in venetoclax-resistant AML, TAUT inhibition synergizes with venetoclax to block the growth of primary human AML cells. Mechanistically, our multiomic approaches indicate that the loss of taurine uptake inhibits RAG-GTP dependent mTOR activation and downstream glycolysis. Collectively, our work establishes the temporal landscape of stromal signals during leukaemia progression and identifies taurine as a key regulator of myeloid malignancies. Signals from the tumour microenvironment (TME) can regulate initiation, progression and immune evasion of tumours1,2,3,4,5,7,8,9,10. While scRNA-seq analysis has identified cellular TME components, especially in solid tumours11,12, concerted efforts to link ligands from the changing TME landscape with cognate receptors on cancer cells have been lacking. As cell surface proteins are particularly amenable to therapeutic targeting, functional characterization of their interactions with the TME are of considerable clinical interest. Aggressive therapy-resistant myeloid leukaemias, such as blast-crisis-phase chronic myeloid leukaemia (bcCML) and AML, initiate and expand in a complex bone marrow microenvironment. Although previous studies have described the cellular composition of normal bone marrow niche13,14,15, their dynamic alterations during leukaemia progression remain undefined. We use scRNA-seq to establish temporal changes in bone marrow microenvironment populations, and in niche-driven signals during disease progression. To define TME ligands that are essential for leukaemogenesis, we focused on cognate cell surface receptors enriched in LSCs as compared to healthy controls, and those essential for in vivo leukaemia progression6. Of these signals, TAUT, encoded by SLC6A6, was strongly associated with poor prognosis in human leukaemias and emerged as a key regulator of AML. As taurine can be neuroprotective, mitigate the side-effects of chemotherapy17 or support anti-cancer immunity18, a cancer-promoting role of taurine has not been considered. We examined if blocking taurine production in the TME impairs LSC function. We used genetic tools to establish whether TAUT expression in cancer cells controls the growth of aggressive myeloid leukaemias. Using metabolomic, proteomic and transcriptomic approaches, we identify downstream mechanisms by which taurine regulates leukaemia growth. To define temporal changes in non-immune bone marrow microenvironment populations during myeloid leukaemia progression, we used bcCML LSCs, which can grow in unirradiated mouse recipients (Extended Data Fig. Bone marrow stromal populations (leukaemia−CD45−TER119−) were isolated from leukaemic cohorts at distinct stages of disease progression and analysed using scRNA-seq (Fig. Gene-expression-based clustering identified 21 lineage clusters with transitional subsets covering endothelial cells, chondrocytes, fibroblasts, pericytes, mesenchymal stromal cells (MSCs) and osteo-associated populations (Fig. 1c,d) and showed significant remodelling with disease progression (Fig. Our flow-cytometry-based analyses of an independent cohort of leukaemic mice validated these changes, showing increases in MSCs, osteolineage and arteriolar endothelial cells, with a concomitant decline in sinusoidal endothelial populations (Fig. a, The experimental strategy used to determine the impact of bcCML progression on bone-marrow microenvironment remodelling. b, A uniform manifold approximation and projection (UMAP) analysis of 15,695 non-haematopoietic cells from bone and bone marrow shows 21 distinct bone marrow stromal cell clusters (n = 3 (naive), n = 6 (initiation), n = 7 (expansion) and n = 9 (end) mice). Chrondro, chondrocyte; fibro, fibroblast; osteo, osteo-associated cell. e, Representative fluorescence-activated cell sorting (FACS) plots and quantification of MSC frequency over time. f, Representative FACS plots and quantification of the osteolineage cell frequency over time. For e and f, data are mean ± s.e.m. g, Unbiased Enrichr analysis showing the top 10 downregulated pathways by population cluster in MSCs, and osteo-associated, arteriolar and sinusoidal endothelial populations. Blue text represents pathways of interest. The mouse image in a is adapted from ref. Clustering analyses of genes expressed in major lineages identified gene sets with similar changes in expression (Extended Data Fig. 1f), suggesting that these may represent altered stromal cell fate. Collectively, our temporal analysis of the TME identifies dynamic changes in gene expression of stromal subpopulations during leukaemia progression. To define the functional relevance of microenvironmental changes on leukaemogenesis, we focused on proteins expressed on LSC cell surface that act as receptors for niche-driven signals. To identify LSC receptors associated with disease progression, we performed RNA-seq analysis of human AML and bcCML CD34+ cells, and healthy donor bone marrow CD34+ haematopoietic stem and progenitor cells (HSPCs; Extended Data Fig. We noted significant (adjusted P (Padj) < 0.05) upregulation of 1,569 genes in bcCML, 2,842 genes in AML and 2,331 genes in both bcCML and AML compared with normal HSPCs. To identify signals that are functionally relevant for disease progression, we found cell surface genes19 that drop out by twofold or more in our genome-wide in vivo leukaemia CRISPR screen6. Of these 38 genes, 16 were misannotated as cell surface in the reference dataset19 and were removed from further analysis (Methods and Extended Data Fig. We used NicheNet and the published literature to identify ligands for these receptors, especially those that were significantly upregulated in our TME scRNA-seq data and in the human AML immune microenvironment20 (Extended Data Fig. This led to the exclusion of receptors with no known ligands (MR1, TMCO3 and TSPAN15) as well as those of which the ligands were not significantly enriched in any TME population (LGALS3BP, CD96, CD274 and CD3D). We therefore generated a unique map of TME ligands for 15 LSC receptors that are essential for disease progression (Fig. a, Circos plot showing leukaemia cell surface receptors and cognate stromal-cell-derived ligands. b, Kaplan–Meier curves of human patients with leukaemia with low (<11.14, n = 80) or high (≥11.14, n = 81) SLC6A6 expression (TCGA-LAML; Xena Browser; log-rank test). CI, confidence interval; HR, hazard ratio. c, Normalized SLC6A6 expression in CD43+ cells from normal human bone marrow (BM) samples and samples from patients with bcCML and AML. Statistical analysis was performed using DESeq2-implemented Wald tests. d–g, Representative IHC images (d,f) and quantification (e,g) of CDO1 expression in matched patient bone marrow biopsies at MDS diagnosis and after AML transformation (d,e), or at AML diagnosis (AML-D) and relapse (AML-R) (f,g). Each colour represents a patient sample. Statistical analysis was performed using two-tailed ratio paired t-tests. h, The strategy used to determine the impact of inhibiting CDO1 in human bone marrow MSCs on co-cultured AML cells (MSCs and AML cells were derived from the same patient). i, The number of live LSCs (left; data are mean ± s.e.m. ; n = 11 independent culture wells per cohort; data were combined from two independent experiments) and their colony-forming ability (right; data are mean ± s.d. ; n = 3 independent culture wells per cohort) after coculture with AML MSCs. j, Taurine quantity per femur in control and leukaemic mice, as determined using colourimetric analysis, 12 days after transplant. Data were combined from two independent experiments. For i and j, statistical analysis was performed using unpaired two-tailed Student's t-tests. k, Experimental strategy and survival curve, showing the impact of blocking taurine production by MSC/osteolineage populations in vivo on bcCML progression in unirradiated recipients. Data were combined from four independent experiments. Statistical analysis was performed using the log-rank test. Scale bars, 50 µm (d and f). The mouse images in h and k are adapted from ref. We next determined whether disease progression could alter the expression of LSC-specific TME ligands, and identified four distinct patterns of ligand expression. While the expression of some genes remained steady, for example, Jam2 in arteriolar endothelial cells and Ihh in chondrocytes, the expression of Vcam1, Pcdh7 and Il7 was lost in the TME, especially at the end point. Expression of Anxa1 first increased but then declined towards the end. The fourth category of genes was always expressed, but the microenvironmental populations expressing them changed. These included Pvr, Cdo1, Apoe and Agt (Extended Data Fig. Our analysis capturing dynamic changes in the TME identifies signals that may be necessary not only at different stages of disease progression, but also those critical during the entire course of disease. Multiple TME–LSC signalling axes identified in our interactome may have a functional role in leukaemia progression. However, we were interested in genes associated with unfavourable outcomes in human patients with AML. Of the 22 genes upregulated in human LSCs, only low-density lipoprotein receptor (LDLR) and SLC6A6 were significantly associated with poor prognosis in AML (Fig. We therefore tested the functional requirement of TME-derived ligands of LDLR and SLC6A6 on LSC growth. One of the primary LDLR ligands, apolipoprotein E (APOE)21, was highly expressed in MSCs (Extended Data Fig. To test the role of APOE from MSCs on LSC function, we co-cultured bcCML LSCs with MSCs transduced with short hairpin RNAs (shRNAs) targeting Apoe or LacZ (control; Extended Data Fig. Co-culture with MSCs lacking Apoe expression led to significant reduction in both the viability and colony-forming ability of LSCs (around 2.1-fold lower than the controls; Extended Data Fig. SLC6A6 encodes TAUT, which has high affinity for the non-essential amino acid taurine, and low affinity for β-alanine (taurine Km = 4.5 μM versus β-alanine Km = 56 μM)22. Our experiments indicate that Slc6a6 expression can be directly induced by oncogenes (Extended Data Fig. While the leukaemia TME did not express enzymes required for β-alanine synthesis (Gadl1 and Cndp1), those needed for taurine biosynthesis were expressed in osteo-associated cells (Cdo1 and Csad; Extended Data Fig. Our analysis of publicly available human bone marrow stromal cell scRNA-seq data using surgical samples from osteoarthritis patients23 showed that CDO1 expression is restricted to MSC and osteolineage populations (Extended Data Fig. To test whether CDO1 is expressed in the human leukaemia TME, we performed an scRNA-seq analysis of CD45− stromal cells from three myelodysplastic syndrome (MDS) and AML bone marrow aspirates. Our analysis of the limited numbers of stromal cells in these samples indicate that CDO1 expression is restricted to the MSC/osteolineage cells (Extended Data Fig. 4k,l), as we see in mouse samples. Notably, we found a marked increase in CDO1 protein expression in nearly all matched biopsies of patients who progressed from MDS to secondary AML, and those who relapsed after AML diagnosis (Fig. While CDO1 protein was expressed in undifferentiated human AML MSCs, its expression increased during in vitro mouse and human AML MSC osteogenic differentiation (Extended Data Fig. Thus, CDO1 may be broadly expressed in immature MSCs as well as differentiating osteolineage cells in human bone marrow. Importantly, taurine levels in extracellular medium increased during MSC differentiation, indicating that elevated CDO1 expression is correlated with taurine production (Extended Data Fig. To determine a functional role of taurine from osteolineage cells, we tested the impact of inhibiting taurine biosynthesis in mouse and human leukaemia bone-marrow-derived MSCs on co-cultured leukaemia cells. Our experiments showed that mouse LSCs co-cultured with osteolineage cells lacking Cdo1 expression were significantly less viable (about 2-fold lower than the controls) and formed fewer colonies (3.4-fold less than controls), which could be rescued by supplementing cultures with taurine (Extended Data Fig. Similarly, inhibiting CDO1 in osteolineage cultures from MSCs derived from patients with AML impaired survival and colony-formation of co-cultured AML cells from the same patient by 2- to 3.2-fold (Fig. 5h), indicating that taurine produced by osteolineage cells promotes LSC growth and survival. Consistent with a functional role of taurine from the TME in leukaemia progression, taurine levels were 1.7-fold higher in leukaemic bone marrow interstitial fluid as compared to the controls (Fig. To determine whether taurine produced locally in the bone marrow niche is essential for leukaemia growth, we generated a new Cdo1fl/fl mouse model and crossed it to MSC/osteolineage-specific Prrx1-cre mice (Extended Data Fig. We used these as recipients for LSCs and monitored survival, and the impact on bone marrow microenvironment populations. Our experiments showed that Cdo1fl/flPrrx1-cre+ mice lived around 13.5% longer than the controls, indicating that taurine produced locally in the leukaemia TME can support disease progression, at least in part (Fig. We did not find any significant changes in the composition of TME in Cdo1fl/flPrrx1-cre+ leukaemic mice as compared to the controls (Extended Data Fig. 5l–n), suggesting that leukaemia-driven remodelling of the bone marrow niche does not depend on taurine produced by osteolineage cells. We finally tested whether exogenous taurine supplements can promote leukaemia growth. Our experiments showed that the colony-forming ability of mouse LSCs and patient-derived AML cells increased by 1.2- to 3.3-fold in the presence of taurine (Extended Data Fig. Taurine supplements could significantly accelerate disease progression in immunocompetent mice (around threefold higher likelihood of death relative to the controls; Extended Data Fig. 5r), indicating that taurine can promote leukaemic progression. Collectively, these data indicate a key role of taurine and APOE from the bone marrow niche in sustaining LSCs. As SLC6A6 expression was significantly enriched in both human bcCML and AML LSCs as compared to the controls, we focused our studies on SLC6A6, as it may be broadly required for growth of aggressive leukaemias. To determine whether TAUT has a functional role in leukaemia progression we used global Slc6a6-knockout mice25 (Fig. While Slc6a6-knockout mice are born at Mendelian ratios, they can develop ageing-related defects in bone mass and retinal degeneration25,26,27. Our experiments showed that TAUT loss in LSCs significantly reduced their ability to grow when co-cultured with osteolineage cells expressing either control shRNA or shRNA against Cdo1 (shCdo1; Extended Data Fig. 6a), indicating that they could not respond to taurine being produced by the niche. 3c,d), which could not be rescued with taurine supplements (Extended Data Fig. TAUT loss also led to functional depletion in bcCML LSCs, as indicated by a 2.3-fold reduction in their colony-forming ability (Fig. 3e), as well as a marked increase in the survival of mice transplanted with Slc6a6−/− LSCs (40%) relative to control LSCs (0%) in secondary transplant assays (21.2-fold higher likelihood of survival; Fig. The 8.5-fold reduction in colony-forming ability of bcCML LSCs from secondary transplants suggests that these leukaemias remain dependent on TAUT expression and taurine uptake for their continued propagation (Fig. We noted no significant differences in bone marrow microenvironmental populations of mice bearing Slc6a6−/− leukaemias as compared to the controls, indicating that leukaemia-driven niche remodelling possibly reflects the extent of the tumour burden, and may be independent of taurine levels within leukaemia cells (Extended Data Fig. a, Relative Slc6a6 mRNA expression in whole bone marrow cells from Slc6a6+/+ (+/+) and Slc6a6−/− (−/−) mice. b, Taurine in normal bone marrow cells. n = 6 pelvic bones from three animals per cohort; data were combined from two independent experiments. Data were combined from three independent experiments. e, CFU analysis of Lin− bcCML cells from primary transplants. f, Survival curve of secondary bcCML transplants. g, CFU of Lin− cells from secondary transplants. n = 3 independent culture wells per cohort. i,j, The experimental strategy (i) and survival curve (j) show the impact of TAUT loss on de novo MLL-AF9-driven AML. m,n, Experimental strategy (m) and survival curve (n), showing the impact of TAUT loss on de novo AML-ETO9a-driven AML. Data were combined from two independent experiments. o, Representative FACS plots and quantification of the Lin−CD150−SCA1+FLT3+ bcCML stem cell frequency in the bone marrow (left) and spleen (right) of recipients. Data were combined from three independent experiments. p, Representative FACS plots and quantification of early apoptosis and necrosis in bcCML at 14 days after transplant. Data were combined from two independent experiments. q, Representative FACS plots and graph showing the frequency of in vivo BrdU incorporation in bcCML. The mouse images in c, i and m are adapted from ref. To determine whether TAUT is broadly required for de novo AML growth, we tested the impact of its loss on initiation of leukaemia driven by MLL-AF9/NRAS(G12V) and by AML-ETO9a/NRAS(G12V). Our experiments showed that TAUT loss markedly delays the initiation of MLL-driven AML relative to the control (6.1-fold higher likelihood of survival; Fig. Slc6a6−/− KIT+ AML LSCs from established disease formed 2.9-fold fewer colonies compared with the controls (Fig. 3l), indicating that TAUT loss depleted functional LSCs. Consistent with a key role of TAUT expression in myeloid leukaemia initiation, TAUT loss in disease driven by AML-ETO9a resulted in a marked increase in survival (70%) relative to the controls (0%) (36.3-fold higher likelihood of survival; Fig. Furthermore, TAUT loss increased necrosis and the Lin+ differentiated cells (Fig. These data demonstrate a critical requirement for TAUT in the initiation, self-renewal and propagation of myeloid leukaemia. We next tested the impact of TAUT loss on normal haematopoiesis. Our analysis indicates that TAUT loss does not severely impact bone marrow cellularity, or the total numbers of HSCs, progenitors and differentiated cells at steady-state (Extended Data Fig. While TAUT loss did not impair initial HSC engraftment (1 month after transplant), donor chimerism dropped over time (Extended Data Fig. Although overall bone marrow chimerism 4 months after transplant was 2.4-fold lower in Slc6a6−/− HSC recipients as compared to Slc6a6+/+ HSC recipients, these Slc6a6−/− HSCs were able to contribute to all haematopoietic lineages (Extended Data Fig. Serial transplantation of Slc6a6+/+ and Slc6a6−/− bone marrow cells showed a similar loss in Slc6a6−/− engraftment over time (Extended Data Fig. These results suggest that, while TAUT loss does not impair steady-state haematopoiesis, it can impact long-term HSC self-renewal and maintenance. These data are consistent with previous studies showing that genetic loss of LSC-enriched genes such as CD981, STAU26, MSI228,29, BRD430 and BCL231 can impair HSC self-renewal. However, therapeutic inhibition of CD98 showed minimal toxicity in AML phase I trials1 and BCL2 inhibitors are approved for AML therapy. It is therefore possible that non-genetic approaches using small-molecule inhibitors or gene silencing may identify a therapeutic window for TAUT targeting in human cells. We therefore tested the impact of TAUT inhibition on growth and proliferation of normal human HSPCs as well as patient-derived AML cells. Our analyses of available gene expression datasets indicate that SLC6A6 expression is enriched in leukaemia stem/progenitors as compared to more mature blasts (Extended Data Fig. While SLC6A6 is broadly expressed in AML irrespective of karyotype or FAB subtype (TCGA; Extended Data Fig. 8b), increased expression correlates with venetoclax resistance (BEAT-AML; Fig. SLC6A6 expression is also enriched in reactive oxygen species (ROS)-low LSCs from human monocytic AMLs (subtype clinically associated with venetoclax resistance; CD45brightSSChighCD117−CD11b+CD68+) compared with primitive AML (Fig. We find increased SLC6A6 expression in leukaemia cells carrying RAS mutations as compared to wild-type cells, consistent with data correlating RAS mutations with venetoclax resistance33 (Fig. SLC6A6 is also highly expressed in relapsed AML originating from stem/progenitor like cells compared with more committed populations34 (Extended Data Fig. These data collectively indicate that inhibiting SLC6A6 may be of value across AML subtypes, including those commonly associated with venetoclax resistance. a, SLC6A6 expression based on venetoclax (Ven) response (BEAT-AML). n = 193; dot plots show individual patients. Statistical analysis was performed using the DESeq2 log-rank test. c, SLC6A6 expression of human NRASG12D+ versus wild-type cells (GEO: GSE253715). d,e, CFU analysis of primary human AML (d) or normal human CD34+ bone marrow HSPCs (e) treated with dimethyl sulfoxide (DMSO)/water (control) or the indicated doses of TAG and GES. For d and e, data are mean ± s.e.m. n = 3 independent culture wells per sample from two independent samples; each colour represents a sample. f,g, CFU analysis of human AML cells treated with DMSO/water (control) or venetoclax (ABT-199) in combination with the indicated doses of TAG (f) or GES (g). n = 2 independent primary human AML samples. i,j, CFU analysis of primary human AML cells (i) or normal CD34+ HSPCs (j) that were transduced with lentiviral shRNAs targeting LacZ (control) or human SLC6A6. Each colour represents an independent sample. k–m, Experimental strategy (k) and representative FACS plots and graph, showing bone marrow engraftment of primary human AML (l) or primary human normal CD34+ HSPC (m) cells. Statistical analysis was performed using two-sample Wilcoxon tests (b), one-way ANOVA (d–g) and unpaired two-tailed Student's t-tests (i, j, l and m). The mouse and culture well images in k are adapted from ref. To determine whether small-molecule inhibitors of TAUT can effectively block leukaemia growth, we used two well-characterized structural analogues of taurine that inhibit uptake: 6-aminomethyl-3-methyl-4H-1,2,4-benzothiadiazine-1,1-dioxide hydrochloride35 (TAG) and guanidinoethyl sulphonate36 (GES; Extended Data Fig. TAG and GES treatment led to a substantial reduction in colonies formed by Slc6a6+/+ mouse LSCs but not Slc6a6−/− cells (Extended Data Fig. Importantly, while TAG and GES impaired growth of primary human AML cells in colony assays by 1.8- to 20-fold, they did not impact normal human CD34+ HSPC colony growth (Fig. We next tested whether venetoclax can exacerbate the impact of TAUT loss and/or inhibition on LSC function. Our experiments showed that venetoclax reduced the viability of mouse Slc6a6−/− LSCs by 3.2-fold and lowered their colony formation by 13.2-fold as compared to the controls (Extended Data Fig. While primary human AML cells that were treated with GES, TAG or venetoclax alone formed 1.3- to 8.3-fold fewer colonies as compared to the controls, combining the treatments substantially impaired colony formation by 2.4- to 150-fold. These data indicate that taurine inhibitors can synergize with venetoclax in blocking the growth of human AML cells (Fig. As TAG and GES did not effectively block taurine uptake in vivo (Extended Data Fig. 9a–j), we used shRNA-based approaches to determine the impact of inhibiting TAUT expression on human AML growth. Knocking down SLC6A6 expression using two independent shRNAs significantly impaired taurine uptake by 2.2 to 3.4-fold, as well as the colony-forming ability of human bcCML, AML and MDS cell lines by 2- to 12-fold (Extended Data Fig. Our experiments showed that SLC6A6 knockdown in samples from patients with AML reduced their colony-forming ability by 2.33- to 9.12-fold as compared to the controls (Fig. Importantly, while SLC6A6 knockdown reduced the engraftment of primary AML cells by 1.2- to 40-fold in patient-derived xenograft models (Fig. 4k,l), it did not significantly impair engraftment of normal human CD34+ HSPCs in xenograft models (Fig. Collectively, our data identify a critical requirement for SLC6A6 expression in primary human AML growth and indicate that blocking taurine transport may be of value in aggressive myeloid leukaemias. To establish the mechanisms by which taurine uptake in leukaemia cells promotes disease progression, we determined metabolic changes in the absence of taurine. Our untargeted metabolomic analysis of mouse LSCs identified significant downregulation of glycolysis/TCA related pathways and metabolites such as pyruvate, glyceraldehyde-3-phosphate and 3-phosphoglycerate in Slc6a6−/− cells, suggesting that taurine may regulate energy metabolism (Fig. Consistent with this, we noted a 1.3- to 1.8-fold reduction in basal glycolysis, glycolytic capacity, maximal oxygen consumption and spare respiratory capacity in Slc6a6−/− LSCs as compared to the controls (Fig. To functionally test the role of glycolysis and TCA associated metabolites, we determined whether bypassing these could rescue Slc6a6−/− defects. Our experiments showed that the colony-forming ability of Slc6a6−/− LSCs could be significantly rescued by pyruvate, sodium acetate and lactate, but not glucagon (Fig. These data suggest that glycolysis, and not gluconeogenesis, is the primary downstream effector of taurine in leukaemia cells. b, Unbiased Enrichr analysis (MetaboAnalyst; Padj ≤ 0.05). Statistical analysis was performed using unpaired Student's t-tests with Welch's correction. AUC, area under the curve; G3P, glyceraldehyde-3-phosphate; 3PG, 3-phosphoglyceric acid. n = 18 independent culture wells per cohort from four bcCML samples. Data are combined from four independent experiments. f,g, CFU analysis of Lin− bcCML cells supplemented with 1 mM pyruvate (f) or 80 μM cell-permeable taurine conjugates (g). n = 3 independent culture wells per cohort. Data were combined from two independent experiments. i, Unbiased Enrichr analysis of transcriptomic data. ChIP–seq, chromatin immunoprecipitation followed by sequencing; g-6-p, glucose-6-phosphate; ncRNA, non-coding RNA. j, Gene sets significantly enriched in Slc6a6+/+ leukaemic mice. k–m, Schematic (k), immunoblot (l) and quantification (m) of the indicated proteins (Extended Data Fig. For d, e and m, statistical analysis was performed using unpaired two-tailed Student's t-tests. n,o, CFU analysis of Lin− bcCML cells treated with rapamycin (n) or 2 μM mTOR activator MHY1485 (o). n = 6 independent culture wells per cohort. Data were combined from two independent experiments. p, CFU analysis of Lin− bcCML cells infected with vector, RAGA(Q66L) or RHEB(Q64L). n = 9 independent culture wells per cohort. Data were combined from three independent experiments. q–s, The strategy to determine mTOR and LAMP1 co-localization in Lin− bcCML cells infected with RAGA(Q66L) or vector (vec) with or without taurine (q), microscopy images (r) and analysis (s). Data were combined from two independent experiments. mTOR and LAMP1 co-colocalization is indicated by white arrows. For f, g, n–p and s, statistical analysis was performed using one-way ANOVA. Blue text in b, i and j represents pathways of interest. The mouse and culture well images in a, h, k and q are adapted from ref. We next tested whether taurine contributed to any cellular metabolite by determining 13C-taurine incorporation in K562 leukaemia cells. Our untargeted metabolomic approach identified 13C label only in known taurine conjugates, N-acetyl taurine and glutaurine (Extended Data Fig. Membrane-permeable taurine fully rescued colonies formed by Slc6a6−/− cells (Fig. 5g), and N-acetyl-taurine partly rescued this defect (Extended Data Fig. By contrast, glutaurine was unable to rescue the Slc6a6−/− colony formation (Extended Data Fig. These experiments indicate that taurine, and not a secondary metabolite, promotes leukaemia growth due to an indirect effect on glycolysis, possibly through downstream signalling. To identify signals downstream of taurine, we performed RNA-seq as well as phosphoproteome and total-proteome analyses of primary mouse leukaemia cells (Fig. Our RNA-seq analysis identified 932 downregulated and 1,158 upregulated genes in the absence of TAUT (Padj < 0.05). The 1,158 upregulated genes included pathways associated with haematopoietic cell lineage and cell cycle regulation (Extended Data Fig. The 932 downregulated genes primarily constituted pathways associated with glycolysis (Fig. 11b–d), consistent with reduced abundance of glycolysis associated metabolites in these cells (Fig. To determine effectors of glycolytic downregulation on TAUT loss, we performed gene set enrichment analysis. This identified profound downregulation of oncogenic MYC and mTOR pathways (Fig. As mTOR signalling is known to regulate expression of glycolysis-related genes38, we tested whether TAUT loss impairs mTOR activation. Western blot analysis showed that TAUT loss results in a 2.2- to 3-fold reduction in activated mTOR, p70S6k and pEIF4B as compared to the controls (Fig. Our global phosphoproteome and total-proteome analyses also showed reduced expression of phosphorylated proteins associated with mTOR signalling, as well as 3.3-fold reduced BCL2 and 1.5-fold increased LC3A expression in the absence of TAUT (Extended Data Fig. Like TAUT loss, TAUT inhibition with TAG and GES downregulated glycolysis-related gene expression (Extended Data Fig. 11h,i) and impaired phosphorylation of mTOR signalling (Extended Data Fig. Consistent with a direct impact of taurine on mTOR signalling, acute treatment in vitro and long-term taurine dosing in vivo could activate mTOR signalling (Extended Data Fig. Inhibiting mTOR with rapamycin impaired the growth of wild-type cells but did not impact Slc6a6−/− cells in vitro or in vivo (Fig. By contrast, mTOR activation with MHY148539 rescued the colony-forming ability of Slc6a6−/− LSCs to 60% of the control, and membrane-permeable taurine rescued mTOR activation (Fig. Collectively, these data indicate that mTOR signalling has a key role downstream of taurine in leukaemia cells. Amino acid availability can be sensed by RAG GTPases to facilitate mTORC1 localization to the outer lysosomal membrane40 and its interaction with RHEB GTPases41, thereby promoting mTORC1 activation. We hypothesized that taurine may activate mTOR by promoting its interaction with RAG GTPases. To test this, we determined the effect of expressing constitutively active RAGA(Q66L) or RHEB(Q64L) on the colony-forming ability of Slc6a6 wild-type and null cells. Our experiments showed that, while RHEB(Q64L) partially rescued Slc6a6−/− colony formation, this could be fully rescued by RAGA(Q66L) (Fig. Furthermore, mTOR phosphorylation in Slc6a6−/− cells could be rescued by RAGA(Q66L) (Extended Data Fig. Finally, we directly tested whether taurine could promote mTOR interactions with lysosomes in wild-type and Slc6a6−/− leukaemia cells expressing RAGA(Q66L) (Fig. Our immunofluorescence-based analysis showed that, at the baseline, Slc6a6−/− cells have 1.2-fold lower mTOR–LAMP1 interaction, which could be fully rescued by RAGA(Q66L). Taurine supplements increased mTOR–LAMP1 co-localization to levels seen in cells expressing RAGA(Q66L). However, this interaction could not be increased further by adding taurine to RAGA(Q66L)-expressing cells (Fig. 5r,s), consistent with our colony-forming assays (Extended Data Fig. Collectively, our studies show that the taurine–TAUT axis regulates glycolysis in myeloid leukaemia cells by promoting RAG-mediated activation of mTOR signalling. We use scRNA-seq to define the changing landscape of the non-immune cancer microenvironment with disease progression, and to identify unique niche driven signals that promote disease progression. scRNA-seq has been effectively used to characterize the immune microenvironment of both solid tumours and leukaemias20,42,43,44,45. Despite technical limitations in detecting non-immune stromal cells, a few studies have identified antigen-presenting fibroblasts and immune-suppressive endothelial cells in lung and pancreatic cancers46,47,48. A reduction in osteoblasts has been noted during leukaemia initiation in bone marrow niche damaged by irradiation13. However, dynamic changes in the TME during leukaemia progression have not been defined. Our temporal scRNA-seq based TME analysis identifies an expansion in MSCs and their immature osteo-associated progeny, along with a loss in mature osteo-associated cells. This skew in osteolineage populations possibly results from downregulation of MSC differentiation signals during leukaemia progression, and may explain conflicting findings using candidate osteoblast markers49,50. Our data indicate that temporal expansion in arteriolar endothelial cells is accompanied by a loss in signals that are essential for endothelial cell integrity and function, consistent with leaky blood vessels seen by in vivo imaging of the AML niche1,3,51. In addition to population-level changes, our studies identify signals such as KIT, thrombospondin, taurine and apolipoproteins from the TME that are essential for cancer progression. While multiple TME ligands such as APOE and PVR are detected through the disease trajectory, the populations expressing these can change over time. Thus, therapeutic approaches aimed at targeting TME-driven signals may be more effective than blocking TME remodelling or inhibiting individual stromal populations. Consistent with clinicopathological similarities between AML and bcCML52, our gene expression analysis of human AML and bcCML CD34+ LSCs identifies distinct overlap. Our unbiased approach to determine LSC-enriched cell surface receptors that are essential for disease progression identified multiple genes that are known to be critical for AML, including CD9653, CD4754 and protein kinase D255. Although these are required for leukaemic progression, they are not associated with poor prognosis (TCGA-LAML), in contrast to LDLR and SLC6A6 that we describe here. It is therefore possible that other cell surface receptors identified by our analysis also have a functional role in disease progression and should be explored further to determine new therapeutically relevant signals. As the APOE–LDLR and taurine–TAUT axes that we identify are known to have a role in ageing27,56, it is possible that cancer-associated signals in our TME–LSC interactome may be of broad relevance in ageing-related disorders such as MDSs, as we see with TAUT. Biosynthesis of taurine from cysteine is known to occur in the liver, kidneys, adipose tissues and pancreas24. Our data identify bone marrow osteolineage cells as a novel source of taurine in the leukaemia niche. Our studies blocking taurine produced by osteolineage cells establish a key role of TME-driven taurine synthesis in LSC survival and self-renewal. However, it is possible that taurine or β-alanine produced outside the bone marrow niche also contribute to disease progression. Consistent with this, our data suggest that taurine supplements can accelerate myeloid leukaemia progression in mouse models. As taurine is a common ingredient in energy drinks, and is often provided as a supplement to mitigate the side-effects of chemotherapy17, our work suggests that it may be of interest to carefully consider the benefits of supplemental taurine in patients with leukaemia. Mechanistically, we identify a critical requirement of taurine from the microenvironment in regulating mTOR-driven glycolysis in leukaemia cells (Extended Data Fig. We can rescue the growth of Slc6a6-null LSCs by circumventing glycolysis with pyruvate or by ectopically activating mTORC1 using GTP-bound RAGA mutants. Our data showing that constitutively active Rag-GTPases can rescue mTOR interactions with lysosomes, and rescue mTOR phosphorylation, in Slc6a6−/− LSCs indicate that taurine levels in leukaemia cells may be detected by hitherto unidentified sensors, like those for arginine and leucine57,58. Our in vivo data showing a strong impact of targeting taurine uptake using genetic approaches in AML suggest that it would be of considerable interest to develop stable and effective in vivo inhibitors of taurine in future studies. In light of early clinical success of glutamine inhibitors in MDS and AML59,60, our work suggests that evaluating taurine-transport inhibitors in normal and leukaemic cells may be of therapeutic interest. For xenograft experiments with human cells, NSG mice (NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ) were used as transplant recipients. All animals used for in vivo studies were randomly selected to receive either control or treatments. Animals were selected based on genotype, age and sex. No statistical method was used to predetermine sample size for experiments. Adequate sample size was determined based on previous publications1,3,6. Mice were bred and maintained in the animal care facilities at the University of Rochester. All animal experiments were performed according to protocols approved by the University of Rochester's Committee on Animal Resources. Mice were housed with the same sex in ventilated cages, under a 12 h–12 h light–dark cycle with temperature (64–79 °F) and humidity (winter levels, <30%; summer levels, >70%) control. Mice were given enrichment material and fed a standard chow diet. Irradiated recipients were maintained on acid water HYDROPAC (pH 2.5–3.0) and irradiated sulfatrim diet (Mod LabDiet 5P00 with 0.025% trimeth and 0.124% sulfameth). Cdo1fl/fl mice were generated in collaboration with the Transgenic and Genome Editing Core Facility at Augusta University using CRISPR–Cas9 gene editing. These mice were bred to the MSC-specific Prrx1-cre (B6.Cg-Tg(Prrx1-cre)1Cjt/J) to generate a Cdo1fl/flPrrx1-cre+ mice, where taurine production is blocked in the MSC/osteolineage cells. As Prrx1-Cre can be expressed in the female germline, only male Prrx1-cre+ mice were used for breeding and experiments. Blinding was not relevant to mouse experiments as researchers needed to know the conditions for each experiment. Flow cytometry, FACS, western blots, Seahorse analysis, sequencing and imaging used analytical machines and blinding was therefore not necessary. Cells were suspended in Hanks' balanced salt solution (Gibco) with 5% FBS and 2 mM EDTA. Cells were prepared for FACS analysis and sorting as previously described1,6. Antibodies used for defining haematopoietic cell populations were as follows: CD3ε, CD4, CD8, GR1, CD11b, TER119, CD45R and CD19 (all for lineage), KIT, SCA1, CD48 and CD150. All antibodies were purchased from, eBioscience, BioLegend or BD Biosciences. A detailed list of antibodies is provided in Supplementary Table 2. Analysis was performed on LSRFortessa (BD Biosciences), and cell sorting was performed on the FACSAria II (BD Biosciences). Lentiviral shRNA constructs were designed and cloned into the pLV-hU6-EF1a-green or pLV-hU6-EF1a-red backbone (Biosettia) according to the manufacturer's protocol. A detailed list of shRNA sequences is provided in Supplementary Table 1. Virus was produced in 293T cells (ATCC) transfected with viral constructs along with VSV-G, Gag-Pol (retroviral production) or pRSV-rev, phCMV and pMDlg/pRRE (lentivirus production) using X-tremeGENE-HP reagent (Roche). Viral supernatants were collected for 3 to 6 days followed by ultracentrifugal concentration at 20,000 rpm for 2 h. Bone marrow KLS cells were sorted from Slc6a6+/+ or Slc6a6−/− mice and cultured overnight in X-VIVO15 (Lonza) medium supplemented with 10% FBS (GeminiBio), 50 μM 2-mercatpoethanol, SCF (100 ng ml−1, R&D Systems), TPO (10 ng ml−1, R&D Systems) and penicillin–streptomycin (Gibco). AML was generated by sorting bone marrow KLS cells from Slc6a6+/+ or Slc6a6−/− mice and culturing in RPMI medium (Gibco) supplemented with 20% FBS, 50 μM 2-mercaptoethanol, 100 ng ml−1 SCF (R&D Systems), 10 ng ml−1 IL-3, and 10 ng ml−1 IL-6 (R&D Systems). Cells were retrovirally infected with MSCV-MLL-AF9-IRES-tNGFR and MSCV-NRAS(G12V)-IRES-YFP cells were collected 48 h after infection, sorted by FACS for BCR-ABL+ and NUP98-HOXA9+ (bcCML only) and retro-orbitally transplanted into cohorts of sublethally irradiated (6 Gy) C57BL/6J mice. For AML-ETOa9 and NRAS, cells were transplanted into lethally irradiated (9.5 Gy) C57BL/6J recipients along with 3 × 105 RBC lysed bone marrow rescue cells. For secondary transplants, Lin− cells from primary bcCML recipient mice were transplanted into secondary sublethally irradiated recipients. The recipients were maintained on acid water HYDROPAC and irradiated sulfatrim diet and evaluated daily. Taurine was dissolved in autoclaved acid water from the HYDROPAC and supplied in sterile water bottles. For in vivo venetoclax treatments, venetoclax (ABT-199; Tocris Bioscience) solution was made fresh daily in a solvent containing 10% ethanol with 60% Phosal 50 PG and 30% PEG-400, and delivered by oral gavage at a final dose of 50 mg per kg. GES (Toronto Research Chemicals or MedChemExpress) and TAG-HCl (synthesized by Enamine) were dissolved in autoclaved acid water from the HYDROPAC and supplied in sterile water bottles (GES) or intraperitoneally (i.p.) For in vivo rapamycin treatments, rapamycin (Selleck Chemicals) stock solution (50 mg ml−1) was made in ethanol (Sigma-Aldrich). Single-use aliquots of the stock were diluted fresh each day in vehicle containing equal parts of 10% PEG-40 with 8% ethanol and 10% Tween-80 solutions. 5 mg per kg rapamycin or vehicle from days 5–10 after transplant. For all experiments, mice were monitored closely for signs of disease or morbidity daily and were euthanized after visible signs of hunched dorsum, failure to thrive or any signs of infection. These limits were not exceeded for any experiment. Relevant tissues were collected and analysed by flow cytometry, RNA-seq, proteomics, metabolomics or fixed for histology. Apoptosis assays were done using annexin V and 7AAD (eBiosciences). Analysis of in vivo bromodeoxyuridine (BrdU) incorporation was performed using the APC BrdU Flow Kit (BD Biosciences) after a single i.p. RNA was extracted using the RNeasy Micro or Mini kits (Qiagen) according to the manufacturer's protocols. RNA concentrations were determined using NanoDrop 1000 Spectrophotometer (Thermo Fisher Scientific). Quantitative PCR with reverse transcription (RT–qPCR) was performed on the Bio-Rad CFX96 C100 Thermocycler using Bio-Rad CFX Manager 1.1 v.4.1 (Bio-Rad) or Thermo Fisher Scientific Quant Studio 12K Flex Real Time PCR using Quant Studio v.1.2 (Thermo Fisher Scientific). RT–qPCR data were analysed using Bio-Rad CFX Manager 1.1 v.4.1 or Quant Studio v.1.2. Microenvironmental populations were isolated as previously described61. In brief, bone and bone marrow were isolated from long bones and pelvis in 1× Media 199 (Gibco) with 2% FBS (GeminiBio). Bone marrow cells from 3–5 mice per timepoint were digested for 30 min in HBSS containing 2 mg ml−1 dispase II (Gibco), 1 mg ml−1 collagenase type IV (Sigma-Aldrich) and 20 ng ml−1 DNase type II (Sigma-Aldrich). Bone spicules were digested for 60 min in PBS supplemented with 2.5 mg ml−1 collagenase type I (Stem Cell Technologies) and 20% FBS. Digested bone marrow was red-blood-cell-lysed using RBC Lysis Buffer (eBioscience). Bone and bone marrow cells were pooled and CD45+TER119+ haematopoietic cells were magnetically depleted on the autoMACS cell separator (Miltenyi Biotec). The CD45−TER119− stromal cells were either stained and analysed for candidate populations by flow cytometry (BD LSRFortessa) or further enriched by sorting (BD FACSAria II) and processed for scRNA-seq. Cell suspensions were processed to generate scRNA-seq libraries using the Chromium Next GEM Single Cell 3′ GEM, Library and Gel Bead Kit v3.1 (10x Genomics) according to the manufacturer's recommendations. The samples were loaded onto the Chromium Single-Cell Instrument (10x Genomics) to generate single-cell gel bead in emulsions (GEMs). GEM reverse transcription (GEM-RT) was performed to produce a barcoded, full-length cDNA from polyadenylated mRNA. After incubation, GEMs were broken, the pooled GEM-RT reaction mixtures were recovered and cDNA was purified with silane magnetic beads (DynaBeads MyOne Silane Beads, Thermo Fisher Scientific). The purified cDNA was further amplified by PCR to generate sufficient material for library construction. Enzymatic fragmentation and size selection was used to optimize the cDNA amplicon size and indexed sequencing libraries were constructed by end repair, A-tailing, adaptor ligation and PCR. Mouse samples were sequenced on the Illumina NovaSeq 6000 S2 flowcell while human samples were sequenced across several lanes of the Illumina NovaSeq X-plus 25B flowcell. The samples were demultiplexed and counted using cellranger v.4.0.0 mkfastq and count, using the default parameters. The samples were aligned to a custom reference containing the 10x provided mm10-2020-A mouse reference and an additional eGFP sequence. Seurat v.4.1.0 within R v.4.1.1 was used for most of processing. Samples were imported and cells were filtered for at least 200 features captured per cell and features were filtered for expression in at least 3 cells. Additional filters were applied, filtering out all cells with higher than 5% mitochondrial content and filtering out all cells positive for CD45, CD71, TER119 and eGFP. All samples were merged using ‘merge' and normalized using SCTransform, regressing out the impact of mitochondrial features. Principal component analysis was performed using the first 40 principal components (RunPCA) and clusters were generated using FindNeighbors and FindClusters (resolution=0.5). Clusters were initially typed using scMCA v.0.2.0. Populations typed as non-stromal (B cell, pre-B cell, pro-erythrocyte, pro-erythroblast, neutrophil and megakaryocyte) were filtered from the dataset. Populations were further typed using published bone marrow stroma markers13. FindAllMarkers was used within the context of specific populations to determine which genes change significantly over time. Expression patterns corresponding to broadly increased or decreased expression over time were passed to gene set enrichment against KEGG_2019, GO_Biological_Process_2021, WikiPathways_2019_Mouse and ChEA_2022 databases using EnrichR-3.0. Bone marrow aspirates were obtained from patients with MDS/AML after written informed consent in accordance with the Declaration of Helsinki and approval of University of Rochester institutional review board (IRB). To isolate the bone spicules, bone marrow aspirates were passed through a 40 μm cell strainer. The filter containing spicules was washed with PBS and placed in a six-well plate. The spicules were digested for 1 h in collagenase (Stem Cell Technologies) at 37 °C. After filtering, the filtered aspirate was used to isolate bone marrow mononuclear cells by density centrifugation (Ficoll-Paque, GE Healthcare). Digested bone cells and bone marrow mononuclear cells were pooled and stained with CD45–APC (BD Biosciences) followed by staining with anti-APC microbeads (Miltenyi Biotec). The stained cells were magnetically depleted for CD45− cell fraction using LD columns (Miltenyi Biotec). The CD45− cells were processed for scRNA-seq as described above. Samples containing cells expressing COL1A1 were integrated into a single dataset using Seurat v.4.1.0 within R v.4.1.1. Cells with mitochondrial features making up greater than 25% of the detected transcripts were removed. Globin-related content was determined using HBB, HBA2, HBA1, HBD and HBM. Further regression was performed using Harmony v.0.1.0 to reduce the impact of sample specific effects. From there, the standard Seurat procedure was used, clustering to a resolution of 0.8. A normal bone marrow reference was created using GEO GSE253355 (ref. Data from this submission were normalized, PCA was run using the first 50 principal components, and UMAP was run using the first 50 principal components within Seurat v.5.0.3.9911. This dataset was then used within Azimuth v.0.5.0 to create the reference using Azimuth standard methods. Total RNA was purified with Qiagen RNeasy PLUS kit according to the manufacturer recommendations and eluted in nuclease-free water. The TruSeq Stranded mRNA Sample Preparation Kit (Illumina) was used for next generation sequencing library construction according to the manufacturer's protocols. In brief, mRNA was purified from 200 ng total RNA with oligo-dT magnetic beads and fragmented. First-strand cDNA synthesis was performed with random hexamer priming followed by second-strand cDNA synthesis using dUTP incorporation for strand marking. End repair and 3′ adenylation was then performed on the double-stranded cDNA. Illumina adaptors were ligated to both ends of the cDNA, purified by gel electrophoresis and amplified with PCR primers specific to the adaptor sequences to generate cDNA amplicons of approximately 200–500 bp in size. Single-end reads of 100 nucleotides were generated for each sample using Illumina's HiSeq2500v4. Raw reads generated from the Illumina basecalls were demultiplexed using bcl2fastq v.2.19.0. Quality filtering and adapter removal was performed using FastP v.0.20.1. Processed reads were then mapped to the human reference genome (hg38 + gencode v36; https://www.gencodegenes.org/human/release_36.html) using STAR_2.7.6a. Reads mapping to genes were counted using subread featurecounts v.2.0.1. Differential expression analysis was performed using DESeq2 v.1.28.1 with a Padj threshold of 0.05 within R v.4.0.2. Gene Ontology analyses were performed using the EnrichR-3.0 package. To determine significantly expressed receptors, the differential expression results from human RNA-seq described above were first filtered for significantly changing (Padj < 0.05) genes with a log-transformed fold change value of greater than 0. A list of potential cell surface receptors was generated using this gene list in conjunction with cell surface proteins detailed within the Cell Surface Protein Atlas19 and essential for LSC growth in vivo6. The resultant gene list was further reviewed for genes that are not truly expressed within the cell surface, leading to the removal of CHST11, ST3GAL4, ACAA1, CLCN6, CHPF2, CTSK, ATP6AP1, CTSD, PNPLA6, DMXL2, TUBB6, MAN2B2, CLN3, MGAT4B, MYH9 and PIGG from further review. Ligand expression from the temporal scRNA-seq dataset corresponding to bcCML and AML expressed receptors was determined by differential expression across the populations. Differentially expressed genes were filtered for corresponding ligands using a combination of nichenetr (v.1.1.0)62 and literature-supported interactions. The Broad Institute hosted dataset20 was used as a stand in for healthy immune microenvironment within the context of AML. Counts (RNA_soupX1.mtx.gz) and metadata (metadata_clustering_w_header_upd.csv) were downloaded from the Broad Single Cell Portal (https://singlecell.broadinstitute.org/single_cell/study/SCP1987/an-inflammatory-state-remodels-the-immune-microenvironment-and-improves-risk-stratification-in-acute-myeloid-leukaemia) and imported using Seurat. FindMarkers was used to determine which genes were upregulated within the microenvironment in relation to the other populations. Significantly expressed genes were then filtered for ligands using the same method as the stromal microenvironment. Significantly expressed ligand and receptor mappings within the stroma microenvironment, the immune microenvironment and within bcCML and AML human bulk RNA-seq data were visualized using circlize v.0.4.15. For illustration purposes, ligands expressed with a log-transformed fold change of 5 or higher were limited to 5. The RNeasy Plus Micro Kit (Qiagen) was used for RNA extraction. A total of 1 ng of total RNA was pre-amplified using the SMARTer Ultra Low Input kit v4 (Clontech) according to the manufacturer's recommendations. Then, 150 pg of cDNA was used to generate Illumina-compatible sequencing libraries using the NexteraXT library preparation kit (Illumina) according to the manufacturer's protocols. Single-end reads of 100 nucleotides were generated for each sample. The mouse bulk RNA-seq samples were processed otherwise identical to the human bulk RNA-seq with two exceptions: m38 + gencode M27 reference (https://www.gencodegenes.org/mouse/release_M27.html) for use within alignment and counting, and ‘-s 0' being used within subread featureCounts. For human RNA-seq experiments and other studies, CD34+ cells were isolated from bone marrow samples of healthy donors and samples from patients with AML and bcCML obtained under University of Rochester institutional review board-approved protocols with written informed consent in accordance with the Declaration of Helsinki. The normal human CD34+ HSPCs used in all functional assays were purchased (Stem Cell Technologies). Cells were cultured in Iscove's modified Dulbecco's medium with 10% FBS, 100 IU ml−1 penicillin–streptomycin (Gibco) and 55 μM 2-mercaptoethanol, 1× LDL (Sigma-Aldrich) and supplemented l-glutamine with 100 ng ml−1 SCF and TPO (R&D Systems). Human leukaemia cell lines K562, THP1 and M-V-411 (ATCC) were maintained in RPMI/IMDM with 10% FBS, 100 IU ml−1 penicillin–streptomycin (Gibco). These cell lines were validated by vendor. MDS-L cells (from K. Tohyama) were authenticated in house by flow cytometry as CD45+CD34+CD38+ and maintained in RPMI with 10% FBS supplemented with 20 ng ml−1 IL-3 (PeproTech). Cell lines were not tested for mycoplasma. For colony-forming assays with shRNAs, leukaemia and normal cells were transduced with the indicated lentiviral shRNAs. Cells were sorted 24 h after infection and plated in CFU assays or transplanted in sublethally irradiated NSG mice. MSCs were isolated from leukaemic mice and cultured in 10 cm dishes in MEMα with no ascorbic acid (Gibco) supplemented with 15% FBS and 100 IU ml−1 penicillin–streptomycin (Gibco). Then, 6 days after culture initiation, the cells were sorted for CD45−CD3−B220−TER119−GR1−CD31−CD51+SCA1+ MSCs. Sorted cells were expanded in the medium descried above. For co-culture experiments, 50,000 MSCs were plated in a 48-well plate, and transduced with the indicated lentiviral shRNAs. Then, 3 days after infection, osteogenic differentiation was induced by switching to MEMα (Gibco) supplemented 15% FBS, 100 IU ml−1 penicillin–streptomycin (Gibco) along with 50 μg ml−1 ascorbic acid (Sigma-Aldrich), 10 mM β-glycerolphosphate (Sigma-Aldrich) and 100 nM dexamethasone (Sigma-Aldrich) for 6 days. On day 6 after differentiation, 100,000 Lin− bcCML cells were added in X-Vivo supplemented with 10% FBS, 50 μM 2-mercatpoethanol and penicillin–streptomycin. Then, 3 days after co-culture, leukaemia cells were analysed for cell viability and plated in methylcellulose for colony-forming assays. For bone marrow transplants, 500 HSCs were isolated from bone marrow of Slc6a6+/+ or Slc6a6−/− mice and transplanted into lethally irradiated (9.5 Gy) CD45.1 mice along with 2 × 105 SCA1-depleted bone marrow rescue cells. For subsequent secondary transplants, 2 × 106 red-blood-cell-lysed bone marrow cells isolated from primary recipient mice were transplanted into lethally irradiated (9.5 Gy) CD45.1 mice. Peripheral blood of recipient mice was collected every 4 weeks for 4 months after transplant and bone marrow analysed at the end of 4 months. For colony assays with human cell lines or patient-derived AML samples, cells were plated in methylcellulose medium (H4434; StemCell Technologies). Colony numbers were counted 10–14 days after plating. Taurine antagonist or TAG (synthesized by Enamine; 92% pure), GES (G827500, Toronto Research Chemicals), taurine (T8691, Sigma-Aldrich), mTOR activator (MHY1485, Sigma-Aldrich) and Venetoclax (ABT-199, Tocris Bioscience) were used as described. Venetoclax and GES Synergy quantification was calculated using Chou-Talalay method72. The Seahorse XF Glycolysis Stress Test Kit (Agilent Technologies, 103020-100) and the Seahorse XF cell mito stress test kit (Agilent Technologies, 103015-100) were used to measure glycolytic flux (ECAR) and oxygen consumption (OCR) respectively. Mouse Lin− bcCML cells were sorted and cultured for 48 h in X-Vivo supplemented with 10% FBS, SCF and TPO. Then, 1 h before the analysis, 50,000–100,000 cells were seeded in Cell-Tak-coated (Corning, 324240) 96-well XF96 well plates in Seahorse XF medium (Agilent Technologies, 102353-100), and the plate was incubated at 37 °C. ECAR data were measured after sequential addition of glucose (10 mM), oligomycin (1μM) and 2-deoxyglucose (50 mM) using the XF96 analyser (Agilent Technologies). Data were analysed using Wave v.2.6.3 (Agilent Technologies). Cell lysates prepared in 1× RIPA (Thermo Fisher Scientific) supplemented with 1× protease, 1× phosphatase inhibitors (Cell Signaling Technology) and 250 IU benzonase nuclease (Millipore Sigma) were separated on gradient polyacrylamide gels and transferred to nitrocellulose blotting membrane (0.45 μM; GE Healthcare). Primary antibodies against phosphorylated mTOR, mTOR, phosphorylated S6K, S6K, pEIF4B, EIF4B and β-actin (Cell Signaling Technology) or tubulin (Abcam) were used. Immunoblots were developed using SuperSignal West Femto Maximum Sensitivity Substrate (Thermo Fisher Scientific). Images were analysed using Empiria Studio v.3.2.0.186 (LI-COR). Raw gel images are provided in Supplementary Fig. Cells were lysed in 50 μl of 5% SDS, 100 mM triethylammonium bicarbonate (TEAB, Thermo Fisher Scientific) and 50 μg of protein from each sample was reduced with dithiothreitol (Sigma-Aldrich). Proteins were alkylated and trapped to S-Trap micros (Protifi), digested with trypsin and extracted by sequential additions of 0.1% trifluoroacetic acid (TFA) in acetonitrile. Then, 1% of each sample was used for global DIA analysis, and the remaining sample was frozen and dried down in the Speed Vac (Labconco) before TMT labelling. The samples were reconstituted in TEAB and labelled with TMT 10-plex reagents (Thermo Fisher Scientific) according to the manufacturer's protocol. All of the samples were combined and dried down in a speed vac before desalting with 0.1% TFA in acetonitrile using the 130 mg C18 sorbent sep-pak attached to a 3 ml syringe (Waters). Desalted samples were frozen and dried down before phosphorylation enrichment using the High-Select FeNTA Enrichment Kit (Thermo Fisher Scientific) according to the manufacturer's protocol. Enriched phosphorylated samples were frozen, dried down and fractionated using C18 spin columns. The fractions were eluted by stepwise addition of 10 mM AmmOH with increasing acetonitrile concentrations: 3.5, 6.5, 9.5, 12.5, 15.5, 18.5, 27, 50%. Fractionated samples were frozen, dried down and reconstituted in 0.1% TFA for MS analysis. Non-TMT-tagged peptides were injected onto a 75 μm × 2 cm trap column before refocusing on a 100 μm × 15 cm C18 column with 1.8 μm beads (Sepax) using the Vanquish Neo UHPLC (Thermo Fisher Scientific) system connected to the Orbitrap Astral mass spectrometer (Thermo Fisher Scientific). Ions were introduced to the mass spectrometer using a Nanospray Flex source operating at 2 kV. Solvent A (0.1% formic acid in water) and solvent B (0.1% formic acid in 80% acetonitrile) formed the gradient starting at 1% B and ramped to 99% B for a total runtime of 14 m. After each run was completed, the column was re-equilibrated with 1% B before the next injection. The Orbitrap Astral was operated in data-independent acquisition (DIA) mode for global proteomic analysis, with MS1 scans acquired at a resolution of 240,000 with a maximum injection time of 5 ms over a range of 390–980 m/z. DIA MS2 scans were acquired using 2 Da windows, a 3 ms maximum injection time, an HCD collision energy of 25% and a normalized automatic gain control (AGC) of 500%. Fragment ions were collected over a scan range of 150–2,000 m/z. Phosphoprotein analysis was carried out on the same Vanquish Neo UHPLC and Orbitrap Astral mass spectrometer system, but with several changes to the LC and instrument settings. To account for ratio compression inherent with TMT, samples were high-pH fractionated to reduce complexity before MS analysis. A longer gradient of 5–30% B over 48 min was used to further separate phosphopeptides. A data-dependent acquisition method using a FAIMS Pro Duo (Thermo Fisher Scientific) was used with three compensation voltages (−40 V, −60 V, −80 V) to further reduce the sample complexity. For each CV, a full scan was acquired over a range of 400–1,500 m/z in the Orbitrap, while MS2 scans were analysed in the Astral analyser for 1 s, after which the instrument switched to the next CV and the process was repeated for a total cycle time of 3 s. Peptides with a charge state of between 2 and 6 were isolated based on intensity with a 0.5 Da isolation window, and were fragmented with an HCD collision energy of 35%. Dynamic exclusion was set to 15 s. The global DIA raw data were processed with DIA-NN v.1.8.1 (https://github.com/vdemichev/DiaNN) using library-free analysis mode. The quantification was set to ‘Robust LC (high precision)' mode with RT-dependent median-based cross-run normalization enabled, MBR enabled, protein inferences set to ‘Genes' and ‘Heuristic protein inference' turned off. Precursors were filtered at library precursor q-value (1%), library protein group q-value (1%) and posterior error probability (50%). Protein quantification was carried out using the MaxLFQ algorithm (https://github.com/vdemichev/diann-rpackage) and the number of peptides quantified in each protein group was determined using the DiannReportGenerator Package (https://github.com/URMC-MSRL/DiannReportGenerator). Further filtering, missing value imputation and statistical tests were performed using Perseus73. Phosphoproteome raw data were searched using the CHIMERYS within the Proteome Discoverer software platform v.3.1 (Thermo Fisher Scientific), allowing for up to two missed cleavages, with a fragment mass tolerance of 10 ppm. Carbamidomethyl on cysteine, and TMT on lysine and peptide N terminus were set as fixed modifications, while oxidation of methionine and phosphorylation of serine, threonine and tyrosine were set as variable modifications. Reporter ions were quantified using the Reporter Ions Quantifier node, with an integration tolerance of 20 ppm, and the integration method was set to ‘most confident centroid'. Paraffin-embedded 4 μm human bone marrow biopsy sections were deparaffinized in xylene and antigen epitopes were retrieved using BOND epitope retrieval solution (pH 9) for 10 min. Endogenous peroxidases were quenched with a 10 min incubation in 3% H2O2-methanol solution. The sections were then blocked in 10% donkey serum for 60 min. The sections were either incubated with anti-CDO1 (Proteintech) overnight or with anti-osterix (Abcam) for 2 h at room-temperature. The sections were washed in PBS and stained with HRP-conjugated donkey anti-rabbit secondary antibody (Jackson Immunoresearch) for 1.5 h at room temperature. After three washes, colour was developed using the ImmPACT DAB Substrate kit (Vector laboratories, SK-4105) according to the manufacturer's protocol. The sections were then counterstained with haematoxylin. Three different areas of each section were imaged on the Olympus BX41 microscope with a ×20 (0.5 NA) objective. Images were analysed using the IHC plugin toolbox in Fiji v.1.54g. Leukaemia cells were cultured in RPMI without amino acids (US Biologicals) in the presence or absence of taurine (Sigma-Aldrich) and seeded on eight-well cover glass chambers (BD biosciences) coated with Cell-Taq (Corning) according to the manufacturer's protocols. MSCs were grown and differentiated in 35 mm glass-bottom dishes with 14 mm microwell (MatTek Life Sciences). Cells were fixed with 4% PFA and blocked in blocking buffer (PBS with 5% donkey serum, 1% BSA and 0.1% Triton X-100) and incubated overnight in primary antibodies diluted in the blocking buffer. Primary antibodies used included mTOR (Cell Signaling Technologies), LAMP1 (DHSB) or CDO1 (Proteintech). Cells were washed in PBS containing 0.1% Tween-20 (Sigma-Aldrich), stained with Alexa-Fluor-conjugated secondary antibodies (Thermo Fisher Scientific), and mounted in Fluormount G (Thermo Fisher Scientific). Immunofluorescence images were acquired with the Teledyne Photometrics Prime BSI express sCMOS camera mounted on the Nikon ECLIPSE Ti2 inverted microscope equipped with the NIS-Elements 6D imaging acquisition module (v.5.42.06). Specific illumination wavelengths were selected by combining a large field of view quad-filter cube (DAPI/FITC/TRITC/CY5; 96378) with specific Lumencor emission filters (FF01-474/27-32, FF01-515/30-32, FF01-595/31-32, FF02-641/75-32). MSC and osteolineage cultures were imaged with the Nikon CFI60 Plan Apochromat Lambda D ×20 (0.8 NA) objective lens, and leukaemia cells were imaged using the Nikon CFI60 Plan Apochromat Lambda D ×100 (1.45 NA) objective lens. Images were deconvoluted using Imaris v.10.2 (Oxford instruments), and Pearson's co-localization analysis was performed using the JACoP BIOP plugin in Fiji v.1.54g. Lineage depleted Slc6a6+/+ and Slc6a6−/− mouse leukaemia cells were fixed using BD Cytofix/Cytoperm Fixation/Permeabilization Kit (BD Bioscience) according to the manufacturer's protocols. Cells were stained with primary antibodies against phosphorylated mTOR (Cell Signaling Technology). Cells were then stained with donkey anti-rabbit secondary antibody conjugated with Alexa Fluro 488 (Invitrogen) to detect p-mTOR. Analysis was performed on the LSRFortessa (BD Biosciences) system. Data were analysed using FlowJo software. Bone marrow cells or bone marrow peripheral fluid were isolated from femurs in Hanks' balanced salt solution (Gibco) with 5% FBS and 2 mM EDTA. For taurine analysis of bone marrow cells after genetic loss of Slc6a6 or treatment of leukaemia cells with taurine inhibitor treatments, cells were lysed in RIPA buffer (Thermo Fisher Scientific) with benzonase nuclease (Sigma-Aldrich). For bone marrow interstitial fluid analysis, one femur was crushed in 1 ml of buffer, filtered and centrifuged at 1,500g for 5 min. The supernatant was concentrated using 10,000 MWCO spin columns (Corning). Then, 25 μl of concentrated samples was quantified using the Taurine Assay kit (Sigma-Aldrich/Abcam) according to the manufacturer's protocols on the BioTek Synergy 2 plate reader using Gen5 v.3.11 (BioTek). The samples were corrected for taurine amounts in unconditioned fresh medium or buffer. Alternatively, cell pellets were processed as described in the liquid chromatography–mass spectrometry (LC–MS) section below, and taurine levels were measured by LC–MS (Orbitrap Exploris 240). For untargeted metabolomics, spleens from mice bearing Slc6a6+/+ or Slc6a6−/− leukaemias were quickly dissected and dissociated in Hanks' balanced salt solution (Gibco) with 5% FBS and 2 mM EDTA at 4 °C. Leukaemia cells were collected and maintained at 4 °C all through the process of isolation, staining and magnetic sorting to minimize metabolic changes. Lin+ (CD3ε−CD4−CD8−GR1−CD11b−TER119−CD45R−CD19−) leukaemia cells were magnetically depleted using LD columns (Milteny Biotec). Lin− leukaemia stem cell fractions were washed with PBS containing 5 mM glucose and centrifuged at 3,000g for 1 min, snap-frozen and processed for metabolomics as described in the ‘LC–MS analysis' section below. K562 cells were cultured in serum-free RPMI-1640 medium in six-well plates for 48 h. The cells were then cultured in serum free RPMI-1640 supplemented with 200 μM taurine (1,2 13C2, 98%; Cambridge Isotope Labs) or no additional taurine for 24 h. The cells were washed with PBS containing 5 mM glucose and centrifuged at 3,000g, snap-frozen and processed for metabolomics as described below in the ‘LC–MS analysis' section. Frozen cell pellets were resuspended at 2 million cells per 1 ml of 80% methanol by vortexing, transferred to −80 °C for 30 min and then to regular ice for 30 min with vortexing every 10 min. Next, the samples were centrifuged at 17,000g for 10 min and 90% of supernatant was dried down in a vacuum evaporator (Thermo Fisher Scientific). The samples were reconstituted in 50% acetonitrile (A955, Thermo Fisher Scientific) at a volume equal to 10% of the dried down volume and transferred to glass vials for LC–MS analysis. The metabolite extracts were analysed by high-resolution MS with the Orbitrap Exploris 240 (Thermo Fisher Scientific) system coupled to the Vanquish Flex LC system (Thermo Fisher Scientific). Then, 2 µl of the samples was injected onto the Waters XBridge XP BEH Amide column (150 mm length × 2.1 mm inner diameter, 2.5 µm particle size) maintained at 25 °C, with a Waters XBridge XP VanGuard BEH Amide (5 mm × 2.1 mm inner diameter, 2.5 µm particle size) guard column. For positive-mode acquisition, mobile phase A was 100% LC–MS-grade H2O with 10 mM ammonium formate and 0.125% formic acid. Mobile phase B was 90% acetonitrile with 10 mM ammonium formate and 0.125% formic acid. For negative-mode acquisition, mobile phase A was 100% LC–MS-grade H2O with 10 mM ammonium acetate, 0.1% ammonium hydroxide and 0.1% medronic acid (Agilent). The H-ESI source was operated in positive mode at spray voltage 3,500 V or negative mode at spray voltage 2,500 V with the following parameters: sheath gas 35 au, aux gas 7 au, sweep gas 0 au, ion transfer tube temperature 320 °C, vaporizer temperature 275 °C, mass range 70 to 1,000 m/z, full scan MS1 mass resolution of 120,000 FWHM, RF lens at 70% and standard AGC. LC–MS data were analysed using Compound Discover (v.3.3, Thermo Fisher Scientific) and El-Maven software74 for peak-area determination and compound identification. Compounds were identified by matching to LC–MS method-specific retention time values of external standards and MS2 spectral matching to external standards and the mzCloud database (Thermo Fisher Scientific). Raw P values were calculated using pairwise Mann–Whitney–Wilcoxon rank-sum tests and Padj values were computed using Benjamini–Hochberg false-discovery rate correction. One-way ANOVA, two-way ANOVA, unpaired two-sided Student's t-tests, multiple unpaired t-tests corrected with the Benjamin–Hochberg method, ratio-paired t-tests, and log-rank tests were used to determine statistical significance. Combination index and isobologram plots were made using CompuSyn72. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The scRNA-seq and bulk RNA-seq data for this publication are available under GEO accessions GSE226372 (human bulk RNA-seq), GSE227082 (mouse bulk RNA-seq), GSE226644 (mouse temporal scRNA-seq) and GSE288862 (scRNA-seq of human MDS and AML bone marrow microenvironment). A shiny app hosting the mouse temporal scRNA-seq is available online (https://wilmot-genomics.shinyapps.io/gse226644/). Receptor and ligand interactions were determined using the NicheNet62, Cell Surface Protein atlas19, AML and healthy immune microenvironment20 datasets. The link between SLC6A6 and LDLR expression and AML prognosis was determined using UCSC Xena Browser (https://xenabrowser.net/). Wild-type and Slc6a6−/− proteomics data are available at ProteomeXchange (PXD062322). The library was annotated using M. musculus UniProt ‘one protein sequence per gene' database (UP000005640_9606, downloaded April 2021). Source data are provided with this paper. Time-course analysis was performed using DEGreport (v.1.30.3; https://lpantano.github.io/DEGreport/index.html). Pathway enrichment was performed using EnrichR (v.3.0; https://cran.r-project.org/web/packages/enrichR/). Sample integration within human scRNA datasets was performed using Harmony (v.0.1.0; https://github.com/immunogenomics/harmony). Pathway enrichment analysis within the bulk RNA datasets was performed using EnrichR (v.3.0, above). A shiny app hosting the mouse temporal scRNA-seq is available online (https://wilmot-genomics.shinyapps.io/gse226644/) and was generated using ShinyCell (v.2.1; https://github.com/SGDDNB/ShinyCell). Bajaj, J. et al. CD98-mediated adhesive signaling enables the establishment and propagation of acute myelogenous leukemia. Kaur, A. et al. sFRP2 in the aged microenvironment drives melanoma metastasis and therapy resistance. Kwon, H. Y. et al. Tetraspanin 3 is required for the development and propagation of acute myelogenous leukemia. Orimo, A. et al. Stromal fibroblasts present in invasive human breast carcinomas promote tumor growth and angiogenesis through elevated SDF-1/CXCL12 secretion. Vermeulen, L. et al. Wnt activity defines colon cancer stem cells and is regulated by the microenvironment. Bajaj, J. et al. An in vivo genome-wide CRISPR screen identifies the RNA-binding protein Staufen2 as a key regulator of myeloid leukemia. Sun, Y. et al. Treatment-induced damage to the tumor microenvironment promotes prostate cancer therapy resistance through WNT16B. Bajaj, J., Diaz, E. & Reya, T. Stem cells in cancer initiation and progression. The next decade of immune checkpoint therapy. The evolving tumor microenvironment: from cancer initiation to metastatic outgrowth. Elhanani, O., Ben-Uri, R. & Keren, L. Spatial profiling technologies illuminate the tumor microenvironment. Baryawno, N. et al. A cellular taxonomy of the bone marrow stroma in homeostasis and leukemia. The bone marrow microenvironment at single-cell resolution. Combined single-cell and spatial transcriptomics reveal the molecular, cellular and spatial bone marrow niche organization. Kamijo, H. et al. Thrombospondin-1 promotes tumor progression in cutaneous T-cell lymphoma via CD47. & Sattari, M. Effect of taurine on attenuating chemotherapy-induced adverse effects in acute lymphoblastic leukemia. Cao, T. et al. Cancer SLC6A6-mediated taurine uptake transactivates immune checkpoint genes and induces exhaustion in CD8+ T cells. Bausch-Fluck, D. et al. A mass spectrometric-derived cell surface protein atlas. Lasry, A. et al. An inflammatory state remodels the immune microenvironment and improves risk stratification in acute myeloid leukemia. & Maeda, N. Spontaneous hypercholesterolemia and arterial lesions in mice lacking apolipoprotein E. Science 258, 468–471 (1992). Liu, Q. R., López-Corcuera, B., Nelson, H., Mandiyan, S. & Nelson, N. Cloning and expression of a cDNA encoding the transporter of taurine and beta-alanine in mouse brain. Bandyopadhyay, S. et al. Mapping the cellular biogeography of human bone marrow niches using single-cell transcriptomics and proteomic imaging. Ueki, I., Roman, H. B., Hirschberger, L. L., Junior, C. & Stipanuk, M. H. Extrahepatic tissues compensate for loss of hepatic taurine synthesis in mice with liver-specific knockout of cysteine dioxygenase. Ito, T. et al. Taurine depletion caused by knocking out the taurine transporter gene leads to cardiomyopathy with cardiac atrophy. Preising, M. N. et al. Biallelic mutation of human SLC6A6 encoding the taurine transporter TAUT is linked to early retinal degeneration. Singh, P. et al. Taurine deficiency as a driver of aging. Ito, T. et al. Regulation of myeloid leukaemia by the cell-fate determinant Musashi. Kharas, M. G. et al. Musashi-2 regulates normal hematopoiesis and promotes aggressive myeloid leukemia. Dey, A. et al. BRD4 directs hematopoietic stem cell development and modulates macrophage inflammatory responses. Veis, D. J., Sorenson, C. M., Shutter, J. R. & Korsmeyer, S. J. Bcl-2-deficient mice demonstrate fulminant lymphoid apoptosis, polycystic kidneys, and hypopigmented hair. Pei, S. et al. Monocytic subclones confer resistance to venetoclax-based therapy in patients with acute myeloid leukemia. Sango, J. et al. RAS-mutant leukaemia stem cells drive clinical resistance to venetoclax. Shlush, L. I. et al. Tracing the origins of relapse in acute myeloid leukaemia to stem cells. Yarbrough, G. G., Singh, D. K. & Taylor, D. A. Neuropharmacological characterization of a taurine antagonist. Wei, W. et al. PTER is a N-acetyltaurine hydrolase that regulates feeding and obesity. Yecies, J. L. & Manning, B. D. Transcriptional control of cellular metabolism by mTOR signaling. The Rag GTPases bind raptor and mediate amino acid signaling to mTORC1. Tirosh, I. et al. Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. Venteicher, A. S. et al. Decoupling genetics, lineages, and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Azizi, E. et al. Single-cell map of diverse immune phenotypes in the breast tumor microenvironment. Steele, N. G. et al. Multimodal mapping of the tumor and peripheral blood immune landscape in human pancreatic cancer. Lambrechts, D. et al. Phenotype molding of stromal cells in the lung tumor microenvironment. Elyada, E. et al. Cross-species single-cell analysis of pancreatic ductal adenocarcinoma reveals antigen-presenting cancer-associated fibroblasts. Lee, J. J. et al. Elucidation of tumor-stromal heterogeneity and the ligand-receptor interactome by single-cell transcriptomics in real-world pancreatic cancer biopsies. Frisch, B. J. et al. Functional inhibition of osteoblastic cells in an in vivo mouse model of myeloid leukemia. Acute myelogenous leukemia-induced sympathetic neuropathy promotes malignancy in an altered hematopoietic stem cell niche. Increased vascular permeability in the bone marrow microenvironment contributes to disease progression and drug response in acute myeloid leukemia. Hehlmann, R. How I treat CML blast crisis. Hosen, N. et al. CD96 is a leukemic stem cell-specific marker in human acute myeloid leukemia. Majeti, R. et al. CD47 is an adverse prognostic factor and therapeutic antibody target on human acute myeloid leukemia stem cells. Protein kinase D2 mediates activation of nuclear factor κB by Bcr-Abl in Bcr-Abl+ human myeloid leukemia cells. Wolfson, R. L. et al. Sestrin2 is a leucine sensor for the mTORC1 pathway. DiNardo, C. D. et al. Glutaminase inhibition in combination with azacytidine in myelodysplastic syndromes: a phase 1b/2 clinical trial and correlative analyses. Liu, Y. et al. A phase 1 study of the amino acid modulator pegcrisantaspase and venetoclax for relapsed or refractory acute myeloid leukemia. Kaszuba, C. M. et al. Identifying bone marrow microenvironmental populations in myelodysplastic syndrome and acute myeloid leukemia. Browaeys, R., Saelens, W. & Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. Jurecka, A., Zikanova, M., Kmoch, S. & Tylki-Szymańska, A. Adenylosuccinate lyase deficiency. & Zylka, M. J. Ecto-5′-nucleotidase (CD73) inhibits nociception by hydrolyzing AMP to adenosine in nociceptive circuits. Kang, Y. P. et al. Cysteine dioxygenase 1 is a metabolic liability for non-small cell lung cancer. Mahootchi, E. et al. GADL1 is a multifunctional decarboxylase with tissue-specific roles in β-alanine and carnosine production. Teufel, M. et al. Sequence identification and characterization of human carnosinase and a closely related non-specific dipeptidase. Amyere, M. et al. KITLG mutations cause familial progressive hyper- and hypopigmentation. Fuchs, A., Cella, M., Giurisato, E., Shaw, A. S. & Colonna, M. Cutting edge: CD96 (tactile) promotes NK cell-target cell adhesion by interacting with the poliovirus receptor (CD155). Läubli, H. et al. Lectin galactoside-binding soluble 3 binding protein (LGALS3BP) is a tumor-associated immunomodulatory ligand for CD33-related Siglecs. Chou, T. C. Drug combination studies and their synergy quantification using the Chou-Talalay method. The Perseus computational platform for comprehensive analysis of (prote)omics data. Melamud, E., Vastag, L. & Rabinowitz, J. D. Metabolomic analysis and visualization engine for LC-MS data. Open access funding was provided by Wilmot Cancer Institute. was supported by NIH Training Grant T32AR076950; and C.T.J. by the Nancy Carroll Allen Chair in Hematology Research and a Leukemia and Lymphoma Society SCOR grant (7033-24). J.B. is a recipient of American Society of Hematology Scholar Award, and Leukemia Research Foundation New Investigator Award. This work was supported by NIH grants R01AG079556 awarded to L.M.C. These authors contributed equally: Sonali Sharma, Benjamin J. Rodems, Cameron D. Baker, Christina M. Kaszuba Sonali Sharma, Benjamin J. Rodems, John M. Ashton & Jeevisha Bajaj Sonali Sharma, Benjamin J. Rodems, Christina M. Kaszuba, Edgardo I. Franco, Francisco A. Chaves, Laura M. Calvi, Michael W. Becker, Jane L. Liesveld, Joshua C. Munger, John M. Ashton & Jeevisha Bajaj Department of Bioscience and Technology, Graduate School of Bioscience and Technology, Fukui Prefectural University, Fukui, Japan Kyle Swovick, Kevin Welle & Sina Ghaemmaghami Yi Zhang, Philip Rock & W. Richard Burack Laura M. Calvi, Michael W. Becker & Jane L. Liesveld You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar carried out the in vivo leukaemia progression experiments. performed all of the mouse and human microenvironment scRNA-seq analyses. provided experimental data and performed image analysis. carried out the metabolomic analysis and provided experimental advice. performed the human leukaemia sequencing experiments and analyses. The authors declare no competing interests. Nature thanks Peter van Galen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Survival curve shows bcCML progression in unirradiated recipients, indicating time course for scRNA-seq experiment (naïve, initiation, expansion, and end). b, Representative FACS plots and graph show changes in leukaemia engraftment over time (2 or 3 mice pooled per timepoint). c, Heatmap of significantly expressed marker genes for non-haematopoietic Bone Marrow (BM) populations. d, Line graphs show temporal changes in the proportion of all major lineages, and in sub-clusters of chondrocyte and fibroblasts. e, Representative FACS plots and graph show changes in BM sinusoidal and arteriolar endothelial cell frequency over time (mean ± s.e.m. ; n = 3 animals per timepoint, two-way ANOVA). f, Gene clusters associated with changes in MSC, chondrocyte, and arteriolar endothelial populations during disease progression. a, Unbiased Enrichr analysis shows top 10 upregulated pathways by population cluster in MSCs, osteo-associated, arteriolar, and sinusoidal endothelial populations. b, PCA-plot shows the distribution of 7 CD34+ healthy donor BM HSPCs, 10 bcCML CD34+ LSCs, and 11 AML CD34+ LSCs from human samples. c, Overlap between genes upregulated in bcCML and AML CD34+ cells compared to normal CD34+ cells, proteins expressed on cell surface19, and those that drop-out by 2-fold or more in the leukaemia in vivo CRISPR screen6. d, Heatmap shows r-log normalized RNA expression of cell surface receptors upregulated in AML, bcCML or both. e, Average expression of ligands for cell surface proteins in primary human adult and paediatric AML cells, human AML immune microenvironment populations, and in normal cells20. a, Kaplan-Meier curves of human leukaemia patients with high (<9.32, n = 80) or low (≥9.32, n = 81) LDLR expression (TCGA-LAML; Xena Browser; log-rank test). b, Normalized LDLR expression in CD34+ cells from human bcCML, AML, or normal BM (n = 7 normal BM, n = 10 bcCML, n = 11 AML; central line, box, and whiskers represent median, interquartile range or IQR, minimum/maximum within 1.5*IQR respectively; DESeq2 implemented Wald test). c, d, Experimental strategy (c) and relative Apoe expression (d) in MSCs transduced with shRNAs targeting LacZ or Apoe (mean ± s.d. e,f, Number of live leukaemia cells (e) and their colony forming ability (CFU) (f) post co-culture with MSCs (mean ± s.e.m. g, Slc6a6 expression in KLS cells infected with BCR-ABL and NUP98-HOXA9 (BA-NH) oncogenes for 48 h (mean ± s.d. i, Dot plot of Cdo1 and Csad expression during disease progression. j, UMAP plot of CDO1 and CSAD expression in normal human BM23. k, UMAP plot of microenvironmental cells in three human MDS or AML BM aspirates. m-p, Representative IHC images and quantification of Osterix expression in matched human BM biopsies at MDS diagnosis and AML transformation (m, n) or AML diagnosis and relapse (o,p) (n = 5 independent patients per cohort; each colour represents a patient sample; two-tailed ratio paired t-test). The mouse image in c is adapted from ref. a, b, Alizarin red staining (a), and Cdo1 and Csad expression (b) in MSCs undergoing osteogenic differentiation (mean ± s.d. c, CDO1 expression in murine leukaemia and patient AML MSCs undergoing osteogenic differentiation (mean ± s.e.m. d, Taurine in MSC culture media during osteogenic differentiation (mean ± s.e.m. e,f, Experimental strategy (e) and Cdo1 expression (f) in MSCs transduced with shRNAs targeting LacZ or Cdo1 (mean ± s.d. g, Live leukaemia cells (left), and CFU (right), post coculture with bcCML MSCs (mean ± s.e.m. ; n = 7 independent culture wells per cohort (live); n = 6 (CFU); data combined from 2 independent experiments; one-way ANOVA). h, CDO1 expression in 293 T transduced with shRNAs targeting LACZ or CDO1 (mean ± s.d. j,k, PCR of Cdo1 (j) and Cdo1 expression (k) in Cdo1fl/fl and Cdo1fl/fl;Prrx1-Cre+ MSCs following osteogenic differentiation (mean ± s.d. l-n, BM stroma in leukaemic control and Cdo1fl/fl;Prrx1-Cre+ mice: MSCs (l), osteolineage cells (m), and, endothelial cells (n) (mean ± s.e.m. o-q, CFU of murine cKit+ AML cells (o), murine Lin− LSCs (p), or primary human AML cells (q) with taurine (mean ± s.e.m. The mouse image in e is adapted from ref. a, Number of live LSCs (left) and their CFU (right) post co-culture with leukaemic MSCs transduced with shCdo1 or shLacZ (mean ± s.e.m. b, Survival curve shows the impact of taurine supplements on murine leukaemia progression in irradiated recipients (n = 10 per cohort; data combined from two independent experiments; log-rank test). c, Representative FACS plots and quantification of BM engraftment in cancer at initiation and end point (mean ± s.e.m. d-g, Graph (d) shows number of microenvironmental cells per mouse leg. h, Representative FACS histogram and quantification of Lin+ frequency in bcCML BM (mean ± s.e.m. ; n = 14 animals per cohort; data combined from three independent experiments; c, e-h unpaired two-tailed Student's t-test). a, Average number of BM cells in +/+ and −/− mice (mean ± s.e.m. b-d Representative FACS plots, and quantification of frequency (left) and cell number (right) of hematopoietic stem cells (b, HSC: Lin−cKit+Sca+CD150+CD48−) and multipotent progenitors (b, MPPs: Lin−cKit+Sca+CD150−CD48−), committed granulocyte–macrophage progenitors (c, GMP: Lin−IL7Ra−Kit+Sca1−CD34+CD16/3+), common myeloid progenitors (c, CMP: Lin−IL7Ra−Kit+Sca1−CD34+CD16/32−), megakaryocyte–erythroid progenitors (c, MEP: Lin−IL7Ra−Kit+Sca1−CD34−CD16/32−), and differentiated haematopoietic cells (d) in the BM of +/+ and −/− mice (mean ± s.e.m. ; n = 5 animals per cohort; data combined from four independent experiments). e, Complete blood count of indicated cells and haemoglobin content in the peripheral blood of age and sex-matched 8-week-old littermates (mean ± s.e.m. f-g, Experimental strategy (f), donor engraftment in peripheral blood over time (g). h-k, Average donor chimerism (h), frequency of KLS (Lin−cKit+Sca+), HSCs, and MPPs (i), committed progenitors GMP, CMP, and MEP (j) and, differentiated haematopoietic cells (k) in primary recipient BM four months post-transplant (f-k: mean ± s.e.m. l, Donor engraftment in peripheral blood of secondary recipients (mean ± s.e.m. ; n = 6 animals per cohort; g, l, two-way ANOVA). m-q, Average donor chimerism (m), frequency of KLS, HSCs, and MPPs (n), committed progenitors GMP, CMP, and MEP (o), and differentiated haematopoietic cells (p, q) in BM of secondary recipients four months post HSC transplant (mean ± s.e.m. ; n = 6 animals per cohort; a-e, h-k, m-q unpaired two-tailed Student's t-test). The mouse images in f are adapted from ref. b, SLC6A6 expression by karyotype or FAB subtype (TCGA-AML, n = 146 karyotype and n = 149 FAB subtype; dot represents an individual patient). c, SLC6A6 expression in relapse origin-committed (ROC) and relapse origin-primitive (ROP) AML34 (n = 29 ROC and n = 10 ROP); (b-c central line, box, and whiskers represent median, interquartile range, minimum/maximum point within 1.5*IQR respectively; unpaired two-samples Wilcoxon test). d, Taurine in Lin− bcCML cells treated with DMSO or TAG (mean±s.e.m. ; n = 8 independent wells combined from 2 experiments). e, Taurine in Lin− bcCML cells treated with GES or water (mean±s.e.m. ; n = 5 independent culture wells; d-e unpaired two tailed Student's t-test). f-h, CFU of Lin− bcCML (f), ckit+ AML LSC (g) with DMSO or indicated doses of TAG, or Lin− bcCML with water or indicated doses of GES (h)(mean±s.e.m. ; n = 3 independent culture wells per cohort; data combined from two independent experiments for f,h). i, j, Viability (i) and CFU (j) of murine Lin− bcCML cells treated with 0.5 mM venetoclax (ABT-199, Ven) for 48 h (n = 3 independent culture wells per cohort; data combined from two (i) or three (j) independent experiments; two-way ANOVA). k, CFU of Lin− bcCML cells with DMSO/water, Ven, or indicated doses of TAG (left) and GES (right), or indicated combinations (mean ± s.e.m. l, CFU of cKit+ AML LSC with DMSO/water, Ven, GES, or indicated combinations (mean±s.e.m. m, Combination index of GES and Ven per fraction affected (left) and isobologram (right) by Chou-Talalay method (n = 3 independent culture wells; data combined from 3 independent experiments; colours represent sample type, shapes represent Ven/GES combinations). a, Survival curve showing the impact of treating mice transplanted with Lin− bcCML cells with GES (2.5%), Ven (50 mg/kg), or their combination (n = 10 control, n = 12 GES, n = 6 Ven, n = 8 GES/Ven); data combined from three independent experiments; lines below graph represent days of treatment; log-rank test). b, Taurine in 107 BM cells from leukaemic mice treated with indicated amounts of GES for 13–16 d (mean ± s.e.m. c, d, Experimental strategy (c) and mass (d) of mice during treatment with GES, TAG, or control (mean ± s.e.m. ; n = 3 animals per cohort; two-way ANOVA). ; n = 6 mice per cohort; data combined from two independent experiments). ; n = 6 animals per cohort; data combined from two independent experiments; two-way ANOVA). i, Haematoxylin & Eosin staining of indicated tissues in treated animals. j, Intracellular taurine levels by LC/MS in BM cells from treated animals (mean ± s.e.m. k, l, SLC6A6 expression (k) and taurine levels (l) in K562 cells transduced with shRNAs targeting LacZ and SLC6A6 (mean ± s.d. m-o, CFU of human leukaemia cell lines transduced with lentiviral shRNAs targeting LacZ or SLC6A6, K562 (m), THP1 (n), and MV-4-11 (o) (mean ± s.d. ; n = 3 independent culture wells per cohort). ; n = 3 technical replicates per cohort) and CFU (right) in MDS-L cells transduced with shRNAs targeting Luc and SLC6A6 (mean ± s.e.m. ; n = 9 independent culture wells per cohort; data combined from 3 independent experiments; b, e-h, j, l-p one-way ANOVA). The mouse image in c is adapted from ref. b, Curve and quantification of normalized of oxygen consumption rate (OCR) in Lin− bcCML cells (mean ± s.e.m. ; n = 10 independent culture wells per cohort from 3 bcCML samples; data combined from three independent experiments; unpaired two-tailed Student's t-test). acetate (c), lactate (d), or indicated amounts of glucagon (e) on the CFU of Lin− bcCML cells (mean ± s.e.m. f, Experimental strategy used to determine 13C taurine tracing in K562 cells by untargeted metabolomics. g, Incorporation of 13C label from taurine in indicated metabolites (mean ± s.e.m. a, Top 20 upregulated pathways in −/− time matched bcCML cells as compared to +/+ controls. b, Heatmap of glycolysis associated genes in bcCML samples. c, GSEA showing downregulation of genes associated with Myc and mTORC1 pathways on TauT loss (a-c, n = 3 animals per cohort). d, Relative expression of glycolysis associated genes in +/+ and −/− Lin− bcCML cells (mean ± s.d. e, Western blot shows phospho-mTOR (pmTOR), mTOR, phosopho-p70S6k (p-p70S6K), p70S6K, and actin protein expression in +/+ and −/− Lin− bcCML cells (also see Fig. f, Volcano plot of global proteome (upper panel) and phosphoproteome (bottom panel) with differential abundance in +/+ and −/− leukaemia cells from time matched recipients (n = 5 animals per cohort). g, Normalized abundance of mTOR pathway proteins (mean ± s.e.m. ; n = 5 animals per cohort; unpaired two-tailed Student's t-test). h, Expression of glycolysis related genes in +/+ Lin− bcCML treated with indicated doses of TAG for 72 h (mean ± s.d. i, Expression of glycolysis related genes in +/+ Lin− bcCML treated with 1 mM GES for 72 h (mean ± s.d. a,b, Immunoblot and quantification of mTOR pathway proteins in +/+ Lin− bcCML treated with indicated doses of TAG (a, mean ± s.d. ; n = 2 replicates per cohort) or 1 mM GES (b, mean ± s.e.m. c-f, Experimental strategy (c), immunoblot (d, f), and quantification (e) of indicated proteins in Lin− bcCML cells +/− 200 mM taurine (mean ± s.e.m. ; n = 3 biological replicates; gels processed in parallel; tubulin run on each gel as sample processing control). g,h, Immunoblots (g), and quantification (h) of mTOR pathway proteins in Lin− bcCML cells from leukaemic mice supplemented with taurine for 10 days (mean ± s.e.m. i, Survival curve shows impact of 5 mg/kg rapamycin treatment for 6 d on leukaemia progression. Line represents treatment days (n = 9 +/+ and n = 5 −/−; data combined from two independent experiments; log-rank test). j, Histogram and geometric Mean Fluorescence Intensity (MFI) of p-mTOR in Lin− bcCML cells treated with DMSO or N-Oleoyl taurine for 36–40 h. (mean ± s.e.m. ; n = 4 samples per cohort; data combined from two independent experiments). k,l, Expression of glycolysis genes in +/+ Lin− bcCML cells treated with rapamycin for 24 h (k; mean ± s.d. ; n = 3 technical replicates per cohort) or 2 mM MHY1485 for 48 h (l; mean ± s.e.m. ; n = 3 technical replicates per cohort; data combined from two independent experiments). m, Histogram and MFI of p-mTOR in Lin− bcCML cells infected with vector or RagA(Q66L) (mean ± s.e.m. ; n = 6 samples per cohort; data combined from two independent experiments). n, CFU of Lin− bcCML cells infected with vector or RagAQ66L and treated with indicated amounts of taurine (mean ± s.e.m. ; n = 3 independent culture wells per cohort; data combined from two independent experiments; j, l-n one-way ANOVA). o, Schematic shows taurine from BM osteolineage cells promotes RagA-dependent mTOR activation and glycolysis in leukaemia cells to drive disease progression. The culture well image in c is adapted from ref. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. et al. Taurine from tumour niche drives glycolysis to promote leukaemogenesis. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09000-3'>STAT5 and STAT3 balance shapes dendritic cell function and tumour immunity</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:31:55
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Immune checkpoint blockade (ICB) has transformed cancer therapy1,2. The efficacy of immunotherapy depends on dendritic cell-mediated tumour antigen presentation, T cell priming and activation3,4. However, the relationship between the key transcription factors in dendritic cells and ICB efficacy remains unknown. Here we found that ICB reprograms the interplay between the STAT3 and STAT5 transcriptional pathways in dendritic cells, thereby activating T cell immunity and enabling ICB efficacy. Mechanistically, STAT3 restrained the JAK2 and STAT5 transcriptional pathway, determining the fate of dendritic cell function. As STAT3 is often activated in the tumour microenvironment5, we developed two distinct PROTAC (proteolysis-targeting chimera) degraders of STAT3, SD-36 and SD-2301. STAT3 degraders effectively degraded STAT3 in dendritic cells and reprogrammed the dendritic cell–transcriptional network towards immunogenicity. Furthermore, STAT3 degrader monotherapy was efficacious in treatment of advanced tumours and ICB-resistant tumours without toxicity in mice. Thus, the crosstalk between STAT3 and STAT5 transcriptional pathways determines the dendritic cell phenotype in the tumour microenvironment and STAT3 degraders hold promise for cancer immunotherapy. Immunotherapy, including ICB and T cell therapy, uses the immune system to eliminate cancer cells1,2,6. However, most patients are either poorly responsive to ICB or develop therapeutic resistance. Emerging evidence suggests that ICB does not trigger potent immune responses in patients with limited and impaired dendritic cells3 (DCs) in the tumour microenvironment (TME). DCs present tumour antigens to prime and activate T cells, thereby driving anti-tumour immunity4,7,8,9,10,11. However, the TME can impede DC1 maturation and function, partly owing to disruptive environmental signals4,15,16,17. Thus, understanding how the DC phenotype is shaped in the TME and exploring strategies to enhance DC1 function are crucial for developing more effective cancer immunotherapy. Alterations in STAT signalling pathways exert profound effects on anti-tumour responses in the TME18. For example, STAT3 is often activated and mediates immune inhibition in the TME5,19. STAT3 activation leads to the production of various pro-tumour factors, including VEGF and IL-6, impeding DC maturation and function20,21. Moreover, STAT3 signalling can inhibit T helper 1 (TH1)-type chemokine expression and subdue DC tumour trafficking, resulting in the exclusion of T cells from the TME21. By contrast, STAT5 is activated in response to cytokine signals, such as GM-CSF and IL-222, and has a positive role in the anti-tumour immune response23,24. On this basis, we hypothesized that the balance between the STAT5 and STAT3 transcriptional pathways in DCs may shape the fate of distinct immune responses in the TME, determining ICB responses in patients with cancer. In this study, we analysed single-cell and bulk RNA-sequencing (RNA-seq) datasets from tumour tissues in patients with cancer who were receiving ICB. We found that the transcriptional pathways of STAT5 and STAT3 in DC1s correlated with T cell immunity and was associated with clinical outcome in patients treated with ICB. Moreover, genetic deletion and pharmacologic inhibition of STAT3 signalling led to DC1 activation and profound anti-tumour T cell immune responses. Furthermore, we provide proof of concept that pharmacological degradation of STAT3 can treat multiple tumour types as monotherapy and sensitize tumours to ICB. The STAT5 and STAT3 transcriptional pathways regulate tumour immune responses21,23,24. We thus reasoned that interplay between these two transcriptional pathways in immune cells, particularly in DCs, influences ICB responses in patients with cancer. To examine this possibility, we analysed RNA-seq data from tumour tissues collected before ICB treatment in a patient cohort (cohort 1) at the University of Michigan25 (Extended Data Table 1). As previously reported25, among these patients, 17% achieved a complete response to ICB, whereas 14% achieved partial response, 17% had stable disease and 51% had progressive disease according to RECIST (response evaluation criteria in solid tumours) criteria (Fig. 1a), and the complete response group exhibited the longest overall survival rate in cohort 1 (Fig. To determine whether the clinical response is associated with the effector T cell-mediated anti-tumour response, we evaluated outcomes stratified by expression of T effector cell signatures26. Patients with a high CD8+ effector T cell score exhibited improved overall survival (Fig. Furthermore, we calculated a DC1 maturation score27. We found that patients with a high DC1 maturation and function score demonstrated improvements in overall survival resembling those in patients with a high CD8+ effector T cell score (Fig. This similarity in survival was mirrored by a strong positive correlation between CD8+ effector T cell and DC1 maturation scores (Fig. Thus, DC-mediated antigen cross-presentation and T cell activation may drive ICB response in patients with cancer. a, Pie chart of number and percentages show RECIST-defined responses to ICB in cohort 1. CR, complete response; PD, progressive disease; PR, partial response; SD, stable disease. b–d, Kaplan–Meier survival curves were stratified by clinical response groups (b), cytotoxic T lymphocyte (CTL) score (c) and median DC1 maturation score (d). e, Pearson correlation analysis shows a correction between CTL score and DC1 maturation score. n = 92. f,g, Kaplan–Meier curves of overall survival in patients from cohort 1 were stratified by DC score and STAT5/STAT3 expression (f) or STAT5 transcriptional pathway/STAT3 expression (g). h, Kaplan–Meier curves of overall survival in cohort 2 were stratified by DC score and the ratio of STAT5 versus STAT3. i–l, STAT5 and STAT3 signalling signatures in DCs in cohort 3. i,j, STAT5 (i) and STAT3 (j) gene signatures in DCs were analysed in post-ICB samples of responders and the respective pre-ICB samples. k,l, STAT5 (k) and STAT3 (l) gene signatures within DCs were compared between non-responders and responders. Outliers are shown as individual points. P values are two-sided without correction for multiple comparisons. Among the four patient groups, those classified as DC1hiSTAT5/STAT3hi had the longest overall survival, whereas the other groups exhibited similar survival trends to each other (Fig. We observed similar results when we stratified patients by the median ratio of STAT5 transcriptional pathway activation and STAT3 expression (Fig. 1g) and by excluding patients with tumours that were inherently unresponsive to ICB (Extended Data Fig. We analysed additional patients with melanoma28,29 (cohort 2), a type of cancer that is sensitive to anti-PD-1 and anti-CTLA-4 therapy (Extended Data Table 2). Thus, the relationship between DC1 maturation and the STAT5 and STAT3 transcriptional pathways may affect responsiveness to ICB. To solidify this finding, we analysed a single-cell RNA-sequencing (scRNA-seq) dataset from patients with triple-negative breast cancer (TNBC) (cohort 3), and examined DCs in tumour tissues collected before and after ICB treatment30. We conducted an analysis of the STAT5 and STAT3 transcriptional pathways in DC clusters. The analysis revealed a bimodal distribution of the STAT3 transcriptional pathway, with the second mode exhibiting a relatively high STAT3 signalling activity (Extended Data Fig. We conducted a detailed analysis of the STAT5 and STAT3 transcriptional pathways in DC clusters with relatively high STAT3 expression. Of note, STAT5 transcriptional signalling increased in post-ICB samples compared with pre-ICB samples among responders to ICB (Fig. 1i), but such an increase was not observed in non-responders (Extended Data Fig. Conversely, STAT3 transcriptional signalling showed a decrease in post-ICB samples of responders compared with their pre-ICB samples (Fig. 1j), with no significant change observed in non-responders (Extended Data Fig. Furthermore, post-ICB samples showed an increase in STAT5 transcriptional signalling and a reduction in STAT3 transcriptional signalling in responders relative to non-responders (Fig. We then examined the STAT5 and STAT3 transcriptional signalling pathways in specific DC subsets in these patients30 (cohort 3). Using the Louvain algorithm for shared nearest neighbours clustering and uniform manifold approximation and projection (UMAP) visualization to distinguish the cells, we conducted an analysis with two major distinct DC clusters, conventional DCs (cDCs) and monocyte-derived DCs (moDCs) (Extended Data Fig. In the cDC subset, the post-ICB samples from responders displayed increased STAT5 transcriptional signalling and decreased STAT3 transcriptional signalling compared with pre-ICB samples (Extended Data Fig. These changes were not discernible in the non-responder group (Extended Data Fig. In moDCs, an increase in STAT5 signalling was observed in the post-ICB samples from responders, whereas STAT3 transcriptional signalling remained unchanged compared with pre-ICB samples in both responders and non-responders (Extended Data Fig. Furthermore, in the post-ICB samples, we observed a noticeable increase in STAT5 transcriptional signalling and a corresponding decrease in STAT3 transcriptional signalling in the responders compared to the non-responders within cDCs (Extended Data Fig. In moDCs, neither STAT5 nor STAT3 transcriptional signalling showed significant differences between post-ICB samples from responders and non-responders (Extended Data Fig. Collectively, ICB treatment dynamically reprograms the STAT5 and STAT3 transcriptional signalling pathways in cDCs, affecting ICB efficacy. Given the negative effect of STAT3 transcriptional activity on DC maturation and ICB response, we examined whether alteration of STAT3 activity would affect DC phenotype. We used short hairpin RNAs (shRNAs) targeting Stat3 (shStat3) in JAWSII cells, a DC line31,32 (Fig. Knocking down STAT3 expression resulted in a robust increase in phosphorylation of STAT5 (pSTAT5) and a minor increase in pSTAT1, but had no effect on pSTAT6 and p-p65 in JAWSII cells cultured with GM-CSF (Fig. We used Stat3fl/flXcr1+/+ mice as the Stat3+/+ controls, enabling us to use mCherry fluorescence to validate knockout specificity in the mouse offspring (Extended Data Fig. No mCherry expression was detected in other immune cells in Stat3−/− mice (Extended Data Fig. We confirmed the absence of STAT3 protein in cDC1s generated from Stat3−/− mice (Fig. At 6 to 10 weeks of age, Stat3+/+ and Stat3−/− mice exhibited similar body size (Extended Data Fig. 2d), similar appearance of lymphoid organs (Extended Data Fig. An unbiased phospho-receptor kinase array showed higher levels of pSTAT5 in Stat3−/− cDC1s than Stat3+/+ cDC1s (Fig. Thus, STAT3 restrains STAT5 activation in DCs. Flow cytometry analysis revealed higher levels of typical DC maturation markers, including major histocompatibility complex class I (MHCI; Fig. 2d), major histocompatibility complex class II (MHCII; Fig. a,b, Immunoblots of transcription factors in shStat3 JAWSII cells (a) and Stat3+/+ and Stat3−/− cDC1s (b). n = 3. c, Scatter plots of relative pixel density versus corresponding −log10(P value) for a phospho-antibody array incubated with lysate from LPS-stimulated Stat3+/+ and Stat3−/− cDC1s. n = 4. d,e, Fluorescence-activated cell sorting (FACS; left) and mean fluorescence intensity (MFI; right) of MHCI (d) and MHCII (e) in Stat3+/+ and Stat3−/− cDC1s. h,i, Immunoblots of transcription factors in JAWSII cells cultured with GM-CSF and transfected with shGmrβ (h) or shJak2 (i). n = 3. j,k, Immunoprecipitation with anti-GMRβ (j) or anti-JAK2 (k) shows the interaction between GMRβ and transcription factors in activated Stat3+/+ and Stat3−/− cDC1s. n = 2. l,m, FACS analysis of MHCII (l) and IL-12 enzyme-linked immunosorbent assay (ELISA; m) in Stat3+/+ and Stat3−/− cDC1s cultured with STAT5i. n–q, Heat map comparing expression of DC maturation genes in Stat3+/+ versus Stat3−/− cDC1s as z-scores (n) or calculated with EdgeR (o), and GSEA shows differences in expression of genes associated with antigen processing and presentation (p) and IL-12 production (q). r,s, RNA-seq analysis of LPS-stimulated Stat3+/+ and Stat3−/− cDC1s shows changes in genes associated with multiple signalling pathways (r) and STAT5 signalling (s). t–w, ChIP–qPCR shows promoter occupancy of H2-D1 (t), H2-Eb1 (u), Cd80 (v) and Cd86 (w) in LPS-treated Stat3+/+ or Stat3−/− cDC1s. To explore how STAT5 activation is regulated in DCs, we assessed the expression profile of a range of cytokine receptors that are capable of activating STAT5, including the receptors for IL-2, IL-3, IL-5, IL-7, IL-15, IL-21 and GM-CSF, in cDC1s. Among these cytokine receptors, we detected higher transcript levels of GM-CSF receptors, Gmrα (also known as Csf2ra) and Gmrβ (also known as Csf2rb), compared with other cytokine receptors in cDC1s (Extended Data Fig. Knocking down GMRβ with specific shRNAs resulted in a decrease in phosphorylation of JAK2, STAT5 and STAT3 (Fig. 2h), but not of JAK1, TYK2, JAK3 and STAT1 in JAWSII cells cultured with GM-CSF (Extended Data Fig. Moreover, knocking down JAK2 caused a profound reduction of pSTAT5 and a moderate reduction of pSTAT3, but not pSTAT1, in JAWSII cells in response to GM-CSF (Fig. GM-CSF stimulation induced the phosphorylation of JAK2, but not JAK1, JAK3 or TYK2 in JAWSII cells (Extended Data Fig. Similar findings have been reported in human neutrophils33. Thus, GM-CSF and signalling pathways downstream of the GM-CSF receptor may regulate STAT5 activation via JAK2 in DCs. We found that Stat3−/− and Stat3+/+ cDC1s expressed similar levels of GM-CSF (Extended Data Fig. Co-immunoprecipitation experiments demonstrated the co-existence of GMRβ, JAK2 and STAT5 in activated cDC1s (Fig. Despite similar levels of GMRβ and JAK2 in the co-immunoprecipitation products of Stat3−/− and Stat3+/+ cDC1s (Fig. 2j,k), genetic deletion of STAT3 led to an increase in the interaction between GMRβ and STAT5 (Fig. 2b) expressed higher levels of pSTAT5 than their counterparts. The data suggest that STAT3 may restrain the GMRβ–JAK2–STAT5 signalling pathway, thereby affecting DC phenotype and function in vitro (Fig. Given that increased IL-6 and activated STAT3 are often observed in the TME, we assessed the expression of IL-6 and its receptors in human and mouse DCs. IL-6-expressing DCs represented a negligible fraction among IL-6-positive cells from patients with TNBC30 (cohort 3) (Extended Data Fig. Mouse Stat3+/+ and Stat3−/− cDC1s expressed low levels of IL-6 (Extended Data Fig. Moreover, cDC1s expressed similar amounts of IL-6 receptor transcripts (Il6ra and Il6st) (Extended Data Fig. These results suggest that a paracrine-activated IL-6–STAT3 pathway regulates the GMRβ–JAK2–STAT5 signalling pathway in DCs, affecting DC phenotype in the TME. To validate this possibility in vivo and elucidate the biological involvement of JAK2, we treated B16F10 or MC38 colon carcinoma-bearing Stat3−/− and Stat3+/+ mice with the JAK2 inhibitor FLLL32. As expected, Stat3−/− cDC1s expressed higher levels of pSTAT5 (Extended Data Fig. 2t,u) and maturation markers compared with Stat3+/+ cDC1s (Extended Data Fig. 2v–x), whereas treatment with FLLL32 abolished this effect (Extended Data Fig. Then, we treated Stat3−/− and Stat3+/+ cDC1s with the STAT5 inhibitor STAT5i34. We observed higher expression levels of MHCII (Fig. Therefore, STAT3 may restrain JAK2-mediated STAT5 activation in DC1s and the functional maturation of these cells. To complement these observations and systemically explore how STAT3 regulates DC phenotype, we conducted RNA-seq analysis of Stat3+/+ and Stat3−/− cDC1s. We observed a noticeable difference in the expression levels of maturation- and function-related genes between Stat3+/+ and Stat3−/− cDC1s (Fig. Gene set enrichment analysis (GSEA) also showed a positive enrichment of maturation and function signatures in differentially expressed genes in Stat3−/− cDC1s relative to Stat3+/+ cDC1s, including antigen processing and presentation (Fig. 2q), CCR chemokine receptor binding (Extended Data Fig. 2y) and IFNγ-production pathways (Extended Data Fig. Consistently, the top upregulated canonical pathways in response to lipopolysaccharide (LPS) in Stat3−/− cDC1s compared with Stat3+/+ cDC1s included antigen presentation, DC maturation and interferon signalling (Fig. Differential gene expression analysis revealed that genes associated with the STAT5 pathway were upregulated with LPS treatment in Stat3−/− cDC1s compared with Stat3+/+ cDC1s (Fig. Furthermore, STAT5 chromatin immunoprecipitation followed by quantitative PCR (ChIP–qPCR) analysis revealed enhanced STAT5 binding at the promoter sites of H2-D1 (Fig. Thus, STAT3 inhibits DC1 maturation and function in a JAK2–STAT5-dependent transcriptional manner. We next investigated the role of STAT3 in DC1-mediated anti-tumour response. 3a–d) or without antigen (Extended Data Fig. 3a–d), and subsequently assessed T cell response. We observed that Stat3−/− cDC1s induced increased proliferation (Fig. These effects were not observed in the absence of SIINFEKL peptide-loaded cDC1s (Extended Data Fig. a–d, OT-I cells were cultured with Stat3+/+ or Stat3−/− cDC1s in the presence of SIINFEKL peptides. a–d, FACS analysis of Ki-67+ (a), IFNγ+ (b), TNF+ (c) and GZMB+ (d) T cells. e–r, MC38 tumours were inoculated into Stat3+/+ and Stat3−/− mice. e–g, Tumour volume (e), images (f) and mass (g). h–r, FACS analysis of pSTAT5 (h–j), MHCII (k,l) and CD86 (m,n) in tumour-infiltrating cDC1s, and the percentage of tumour-infiltrating IFNγ+ (o,p) and GZMB+ (q,r) CD8+ T cells. One representative flow histogram or dot plot is shown (h,k,m,o,q). s–z, B16F10 tumours were inoculated into Stat3+/+ and Stat3−/− mice. u–z, FACS analysis of MHCII (u,v) and CD86 (w,x) in tumour-infiltrating cDC1s, and the percentage of tumour-infiltrating IFNγ+ (y) and GZMB+ (z) CD8+ T cells. One representative flow histogram or dot plot is shown (u,w,y,z). To explore the in vivo effect of STAT3 on DCs in response to tumour challenge, we inoculated MC38 cells subcutaneously into Stat3+/+ and Stat3−/− mice. Loss of STAT3 in cDC1s resulted in reduced tumour progression compared with Stat3+/+ mice, as indicated by tumour size (Fig. To examine whether STAT3 deficiency affected STAT5 activation in intratumoral cDC1s, we gated on tumour-infiltrating cDC1s and detected higher levels of STAT5 activation in Stat3−/− cDC1s than in Stat3+/+ cDC1s (Fig. 3k–n) and tumour-draining lymph nodes (TDLNs) (Extended Data Fig. 3g–j) exhibited higher expression levels of MHCII and CD86 compared with Stat3+/+ cDC1s. This was accompanied by enhanced effector T cell responses in the TME and TDLN, as indicated by higher percentages of IL-2+ (Extended Data Fig. We extended our studies to B16F10 melanoma, a less immunogenic type of tumour. B16F10 tumour-infiltrating Stat3−/− cDC1s exhibited higher levels of maturation and function markers compared with Stat3+/+ cDC1s (Fig. Moreover, we also observed higher percentages of IFNγ+ (Fig. Thus, STAT3 deficiency in DCs enhances T cell function, leading to improved tumour control in vivo. STAT3 has been pursued as a cancer therapeutic target5,35. Small-molecule inhibitors or oligonucleotides have been developed to target STAT35, with limited clinical success35. An indirect strategy to target STAT3 is through inhibition of JAK2 activity, such as with pacritinib5. However, JAK2 inhibition may affect the positive effects of STAT5 on DC1s (as we show here) and T cells24. Thus, we used SD-36, a highly selective and effective PROTAC degrader of the STAT3 protein36 (Extended Data Fig. We treated cDC1s with SD-36 and observed a dose-dependent reduction in the amount of STAT3 protein (Fig. We observed enhanced STAT5 activation in Stat3+/+ cDC1s treated with SD-36 (Fig. Knocking out Stat3 led to increased STAT5 activation, and the addition of SD-36 did not further increase STAT5 activation in Stat3−/− cDC1s (Fig. To pharmacologically validate our genetic studies on the role of STAT3 in DCs (Fig. Pharmacological degradation of STAT3 by SD-36 led to a dose-dependent increase in the interaction between GMRβ and STAT5 (Fig. Moreover, DCs treated with SD-36 exhibited increased expression of MHCI (Fig. 4k,l) compared with the control group. As genetically knocking down STAT3 caused a slight increase in pSTAT1 in JAWSII cells (Fig. 2a), we tested whether STAT1 is involved in SD-36-mediated modification of DC phenotype. Knocking down STAT1 (Extended Data Fig. 4b) had no effect on the expression of MHCI (Extended Data Fig. 4e) compared with the control group, and SD-36 remained equally effective in enhancing the expression of these DC maturation molecules in shStat1 JAWSII cells and wild-type JAWSII cells (Extended Data Fig. Genetic deletion of STAT1 (Extended Data Fig. 4f) had no effect on STAT5 activation in cDC1s (Extended Data Fig. Stat1+/+ and Stat1−/− cDC1s expressed similarly high levels of MHCI, MHCII and CD80 in response to SD-36 (Extended Data Fig. Thus, depletion of STAT3 improves DC function via STAT5 activation. a, Immunoblot showing STAT3 expression in mouse cDC1s treated with different concentrations of SD-36 in vitro. b, Immunoblot showing expression of pSTAT5 and STAT5 in cDC1s from Stat3+/+ and Stat3−/− mice treated with SD-36 (200 nM) for 48 h and LPS (20 ng ml−1) for 1 h. In a,b, one of three experiments is shown. c,d, cDC1s were treated with different doses of SD-36 for 24 h and were immunoprecipitated (IP) with anti-GMRβ (c) or anti-JAK2 (d). One of two experiments with repeats is shown. e–l, FACS analysis of MHCI (e,f), MHCII (g,h), CD80 (i,j) and CD86 (k,l) on cDC1s treated with SD-36 (e–h) and SD-36 plus LPS (i–l). m–p, Mice were inoculated with 4T1 (m), MC38 (n), ID8 (o) or LLC (p) cells and were treated with SD-36 (20 mg kg−1) every 3 days, and tumour volumes were monitored. Mice bearing CT26 tumours (500 mm3 per tumour) were treated with SD-36 or vehicle and tumour volumes (q) and mouse survival (r) were monitored. ; n = 8; *P = 0.0111 (r), log-rank test. s,t, Luciferase radiance images (s) and total fluxes of metastatic 4T1 tumours (t) from mice treated with SD-36. We examined the role of SD-36 in the tumour immune response in vivo. We inoculated B16F10 melanoma into C57BL/6J mice and treated them with different doses of SD-36. SD-36 at 10 mg kg−1 moderately but significantly slowed down B16F10 tumour progression compared with controls (Extended Data Fig. We observed a marked inhibitory effect of SD-36 at 20 mg kg−1 on tumour growth in mice bearing 4T1 mammary carcinoma (Fig. 4o) or Lewis lung carcinoma (LLC) (Fig. Consistently, we observed that mice treated with SD-36 at 100 mg kg−1 also exhibited a slower rate of progression of B16F10, CT26 and EMT6 tumours compared with the control group (Extended Data Fig. 4n–p) and had no obvious effect on the body weight of these mice (Extended Data Fig. Thus, SD-36 monotherapy attenuates the growth of multiple tumours including LLC, an ICB-resistant tumour37. Large, metastatic cancers are often resistant to ICB38,39. We initiated the treatment with SD-36 in CT26 tumour-bearing mice with tumour volumes greater than 500 mm3. Of note, SD-36 remained therapeutically effective at this late stage (Fig. 4t,u) as shown by increased mouse survival compared with the control group (Fig. Finally, we treated 4T1 metastatic tumour-bearing mice with SD-36. This treatment effectively reduced metastatic 4T1 tumour volume (Fig. Thus, SD-36 monotherapy is effective in treating advanced, metastatic and ICB-resistant cancers. We next studied whether the anti-tumour effect of SD-36 was immune- and/or DC-dependent. To this end, we treated MC38 tumour-bearing NOD.SCID γcnull (NSG) mice with a low dose of SD-36 (20 mg kg−1). This dose of SD-36 had no effect on tumours, as shown by tumour volume (Fig. A high dose of SD-36 (100 mg kg−1) had a modest but significant inhibitory effect on tumour progression in NSG mice (Extended Data Fig. Given that both innate and adaptive immunity are deficient in NSG mice, we treated MC38-bearing Rag1−/− mice, which have functional innate immunity, with SD-36. Similarly, a low dose of SD-36 (20 mg kg−1) did not have any effect on tumour volume (Fig. Thus, the therapeutic effect of SD-36 is dependent on an intact adaptive immune system. We treated MC38-bearing wild-type mice with anti-CD8 antibody to deplete CD8+ T cells (Extended Data Fig. CD8+ T cell depletion resulted in the loss of SD-36 therapeutic effectiveness (Fig. Indeed, SD-36 treatment increased the fraction of CD8+ T cells that express TNF (Fig. 5k,l) in 4T1 tumours and TDLNs (Extended Data Fig. Thus, STAT3 degradation enhances CD8+ T cell-mediated anti-tumour efficacy. a–e, Tumours in MC38-bearing NSG (a,b) and Rag1−/− mice (c–e) treated with SD-36 were monitored. n = 5 (a,b) and n = 6 (c,d); two-way ANOVA (a,c) and unpaired two-tailed t-test (b,d). f, Tumour volume was monitored in MC38-bearing mice treated as indicated. Data are mean ± s.e.m., n = 6; **P = 0.0018, two-way ANOVA. g–l, FACS analysis of 4T1-infiltrating TNF+ (g,h), IFNγ+ (i,j) and GZMB+ (k,l) CD8+ T cells from mice treated with SD-36. m,n, Tumour volumes were monitored in Batf3−/− and wild-type mice bearing MC38 (m) or B16F10 (n) tumours and treated with SD-36. o, FACS analysis of STAT3 expression in different cell types from LLC-bearing mice. Data are mean ± s.e.m., n = 4; ****P < 0.0001, one-way ANOVA. p–r, FACS analysis of pSTAT3 (p,q) and pSTAT5 (r) in MC38-infiltrating cDC1s from mice treated with SD-36. s–u, FACS analysis of MHCI (s), MHCII (t) and CD80 (u) in B16F10-infiltrating cDC1s from mice treated with SD-36. v, Tumour volume in B16F10-bearing Stat3+/+ and Stat3−/− mice treated with SD-36. Data are mean ± s.e.m., n = 7; **P = 0.0094, two-way ANOVA. x, Stat5+/+ and Stat5−/− cDC1s were transferred into MC38-bearing Batf3−/− mice, treated with SD-36, and tumour volume was monitored. Data are mean ± s.e.m., n = 7; ****P < 0.0001, two-way ANOVA. y,z, Tumour volume was monitored in mice bearing MC38 (y) and B16F10 (z) tumours and treated as indicated. ; n = 6 (y) and n = 7 (z); **P = 0.0094 (z) and ***P = 0.0009 (y), two-way ANOVA. To investigate whether DC1s were involved in the therapeutic efficacy of SD-36, we treated tumour-bearing Batf3−/− mice lacking cDC1s40. The anti-tumour efficacy of SD-36 therapy was absent in Batf3-deficient mice bearing MC38 (Fig. 5n) or LLC (Extended Data Fig. STAT3 was highly expressed in intratumoral DCs in wild-type mice bearing LLC tumours (Fig. 5o), and SD-36 treatment resulted in a rapid and efficient degradation of STAT3 protein in DCs compared with tumour cells (Extended Data Fig. Of note, the intracellular concentration of SD-36 was four times higher in DCs than in tumour cells (Extended Data Fig. These results suggest that SD-36 effectively induces degradation of STAT3 in DCs to achieve its therapeutic efficacy in vivo. Next, we found that SD-36 treatment reduced pSTAT3 and increased pSTAT5 in MC38 tumour-infiltrating cDC1s (Fig. We also observed enhanced STAT5 activation in DCs isolated from the spleens of MC38-bearing mice after SD-36 treatment (Extended Data Fig. SD-36 treatment inhibited tumour growth in Stat3+/+ mice, whereas this effect was abolished in Stat3−/− mice (Fig. Similar results were observed in MC38 tumour-bearing mice (Extended Data Fig. We next transferred Stat3+/+ or Stat3−/− cDC1s into B16F10 or MC38 tumour-bearing Batf3−/− mice and treated these mice with SD-36. Again, SD-36 improved the anti-tumour effect in Batf3−/− mice that received Stat3+/+ cDC1s, but not in those that received Stat3−/− cDC1s (Fig. We used STAT5i34 to assess the interplay between STAT3 and STAT5 in DCs (Fig. To genetically validate whether STAT5 in DCs contributes to SD-36-mediated anti-tumour immunity, we transferred Stat5b−/− cDC1s or Stat5b+/+ cDC1s into MC38 tumour-bearing Batf3−/− mice and treated these mice with SD-36. SD-36 effectively controlled tumour progression in mice receiving Stat5b+/+ cDC1s, but not Stat5b −/− cDC1s (Fig. Therefore, SD-36 reprograms key transcription factors in cDC1s by degrading STAT3 and enhancing STAT5 signalling, thereby inducing potent anti-tumour immunity in vivo. Finally, we examined whether SD-36 could synergize with ICB. To this end, we treated mice bearing MC38 or B16F10 tumours with PD-L1 monoclonal antibody, SD-36 or both. Anti-PD-L1 inhibited MC38 tumour growth but had no effect on B16F10 tumours, whereas SD-36 monotherapy efficiently slowed down MC38 and B16F10 tumour growth. The combined treatment profoundly inhibited progression of MC38 and B16F10 tumours (Fig. We further tested SD-36 in human DC-mediated T cell activation in the context of ICB. We isolated DCs from fresh human ovarian cancer tissues and treated these DCs with SD-36 and/or anti-human PD-L1 monoclonal antibody, and co-cultured these DCs with T cells. Treatment with SD-36 or anti-PD-L1 increased the numbers of polyfunctional human T cells, as shown by GZMB+IFNγ+ (Extended Data Fig. The combined treatment additionally upregulated the levels of polyfunctional T cells (Extended Data Fig. To further investigate degradation of STAT3 as a potential immunotherapeutic strategy, we have developed SD-2301, another highly potent and selective STAT3 degrader (Extended Data Fig. In contrast to SD-36, which engaged the cereblon–cullin 4A complex for induced STAT3 degradation, SD-2301 used a high-affinity VHL ligand to recruit VHL–cullin 2 for STAT3 degradation. Of note, SD-2301 exhibited four- to five-fold improved potency compared with SD-36 towards STAT3 degradation in DCs (Extended Data Fig. An efficacy experiment showed that 5 mg kg−1 SD-2301 was effective in reducing B16 tumour progression and was four times more potent than SD-36 (Extended Data Fig. Treatment with SD-2301 also increased the number of effector CD8+ T cells, as shown by IFNγ (Extended Data Fig. 6h,i) and GZMB (Extended Data Fig. Treatment with SD-2301 had no effect on CD31+ blood vessel densities in B16F10 (Extended Data Fig. 6p) tumour tissues and did not affect the body weight of mice bearing B16F10 tumours (Extended Data Fig. In addition, whereas anti-PD-L1 monotherapy had no effect on B16F10 tumour growth, combined SD-2301 and anti-PD-L1 synergistically inhibited tumour growth (Extended Data Fig. In the MC38 tumour model, anti-PD-L1 or SD-2301 monotherapy effectively inhibited tumour growth, and the combined therapy profoundly suppressed MC38 tumour progression (Extended Data Fig. Accordingly, SD-2301 treatment increased the proportion of MC38 tumour neoantigen-specific (KSPWFTTL) tetramer+CD8+ T cells compared with the control treatment (Extended Data Fig. These results highlight that SD-2301 may enhance DC-mediated antigen specific T cell priming and presentation in vivo. Next, we explored whether DC1s and STAT3 are direct cellular and molecular targets of SD-2301 therapy in vivo. To address this, we treated B16F10-bearing Stat3+/+ and Stat3−/− mice with SD-2301. As expected, SD-2301 treatment inhibited tumour growth in Stat3+/+ mice, whereas this effect was abolished in Stat3−/− mice (Extended Data Fig. The anti-tumour efficacy of SD-2301 therapy was absent in Batf3-deficient mice bearing MC38 tumours (Extended Data Fig. These results indicate that cDC1s have a key role in the therapeutic efficacy of SD-2301 in vivo. Pharmacokinetic evaluation showed that SD-2301 had an excellent pharmacokinetic profile in mice, as characterized by a slow clearance and high plasma exposure (Extended Data Fig. Moreover, SD-2301 was highly potent and selective in inducing degradation of STAT3 over all other STAT proteins in human peripheral blood monocyte cells in vitro (Extended Data Fig. Together, our results suggest that STAT3 serves as a brake to restrain JAK2-mediated STAT5 activation in DCs and STAT3 degradation in DCs elicits potent anti-tumour immunity, thereby being a promising approach for cancer immunotherapy (Extended Data Fig. This study offers support for several undocumented perspectives in the field of tumour immunology and immunotherapy. First, STAT3 counteracts STAT5 activation, which restrains DC maturation in the TME. Second, STAT3 degradation is a potential immunotherapeutic modality for treating patients with advanced and ICB-resistant cancer. Third, STAT3 degraders work in an DC-dependent manner, supporting STAT5-mediated DC maturation and function. DCs prime and activate T cells against cancer cells3,41,42. We found that ICB can reprogram the STAT5 and STAT3 transcriptional pathways in DCs, thereby normalizing DC function in the TME. It has long been thought that DCs are potential immunotherapeutic targets. However, it is challenging to therapeutically manipulate DCs owing to their numeric scarcity, subset diversity and lack of specific molecular targets. Our results suggest that STAT3 is a druggable molecular target in DC1s using our potent and highly selective PROTAC STAT3 degraders. Despite the identification of STAT3 and STAT5 many years ago20,43,44,45, the interplay between STAT3 and STAT5 signalling pathways in specific immune cells remain poorly understood. STAT3 is frequently activated in the TME owing to abundant stimulating factors, including IL-6, IL-10 and VEGF20,21, whereas STAT5 is often activated by T cell-derived cytokines, such as GM-CSF and IL-2. Given that T cells are limited or dysfunctional in the TME, the balance between STAT5 and STAT3 in immune cells, including DCs, may be biased towards STAT3 overactivation. The interaction between GMR and JAK2 appears to regulate both the STAT3 and STAT5 transcriptional pathways in DC1s. In support of this possibility, we observed that GM-CSF stimulation or genetically knocking down GM-CSF receptor alters the phosphorylation of JAK2, STAT3 and STAT5, but not JAK1, JAK3, TYK2 and STAT1 in cDC1s. Of note, targeting STAT3 has no effect on the expression levels of GM-CSF and GMRβ but enhances the interaction of GMRβ and JAK2 with STAT5, leading to the phosphorylation of STAT5 and DC maturation. Therefore, STAT3 restrains JAK2-mediated STAT5 activation in DCs, thereby impairing DC1-mediated T cell immunity. To rescue the STAT3-impaired DC phenotype, we developed two PROTAC molecules (SD-36 and SD-2301) that target STAT3 protein for degradation. There are several key functional characteristics of our STAT3 degraders. First, they are highly selective and efficacious in degrading STAT3. Second, low doses of SD-36 and SD-2301 effectively target DCs. Furthermore, low doses of SD-36 and SD-2301 mediate an immune-dependent anti-tumour effect, whereas high doses of SD-36 can directly control tumour progression in the absence of a functional immune system36. Intriguingly, SD-36 or SD-2301, as a monotherapy, exhibit remarkable efficacy in treating large, advanced tumours and ICB-resistant tumours. Given that the anti-tumour effect of SD-2301 is four to five times more potent than that of SD-36 in degrading STAT3 and it has excellent pharmacokinetics, we are evaluating SD-2301 as a potential drug candidate for clinical development. Notably, although STAT3 degraders efficaciously target DCs, the involvement of STAT3 in other cell types may also influence immune responses in the TME. Together, our study provides strong support for potential human clinical trials of STAT3 degraders in treating patients with large and metastatic tumours and/or ICB-resistant tumours. Western blot analyses were performed using primary antibodies at a dilution of 1:1,000 unless otherwise specified. Anti-STAT1 (D1K9Y, 14994), anti-STAT2 (D9J7L, 72604), anti-STAT3 (Rabbit, 79D7, 4904; Mouse, 124H6, 9139), anti-STAT4 (C46B10, 2653), anti-STAT5 (D2O6Y, 94205), anti-STAT6 (D3H4, 5397), anti-JAK1 (6G4, 3344), anti-JAK2 (D2E12, 3230), anti-JAK3 (D7B12, 8863), anti-TYK2 (E9H4T, 35615), anti-p65 (D14E12, 8242), anti-CD80 (E6J6N, 54521), anti-CD86 (E5W6H, 19589), anti-IL-6Rα (E7H4J, 39837), anti-GP130 (3732), anti-pSTAT1 (58D6, 9167), anti-pSTAT3 (D3A7, 9145), anti-pSTAT5 (D47E7, 4322), anti-pSTAT6 (D8S9Y, 56554), anti-pJAK1 (D7N4Z, 74129), anti-pJAK2 (C80C3, 3776), anti-pJAK3 (D44E3, 5031), anti-pp65 (93H1, 3033) and anti-β-actin (4967) were from Cell Signaling Technology. Anti-JAK2 (C-10, sc-390539), anti-GMRβ (F-12, sc-393281) and anti-β-actin (C4, sc-47778) were from Santa Cruz Biotechnology. HRP horse anti-mouse IgG antibody (PI-2000, 1:5,000) and HRP goat anti-rabbit IgG (PI-1000, 1:5,000) were from Vector Laboratories. Plasmids expressing shRNAs targeting Stat3 (TRCN0000071453, TRCN0000301946), Stat1 (TRCN0000054924), Jak2 (TRCN0000023649, TRCN0000023651, TRCN0000023652) and Gmrβ (TRCN0000067025, TRCN000067026) were from Sigma-Aldrich. JAWSII, 293T, B16F10, CT26, EMT6, LLC and 4T1 cells were purchased from the American Type Culture Collection (ATCC). MC38 mouse colon cancer cell line was obtained from the University of Texas Southwestern Medical Center (Y.-X. Ovarian cancer cell line luciferase-ID8 cells were previously reported24. The 293T cells were transfected with packaging plasmids and non-targeting lentivirus vector, or a lentivirus vector encoding shRNAs targeting Stat3, Stat1, Jak2 or Gmrβ. The virus-containing supernatant was collected 48 h after transfection. JAWSII cells were transfected with the lentivirus. Then the cell lysates were analysed by immunoblotting. Bone marrow was collected from the femurs and tibias of mice. Red blood cells were lysed using ACK lysis buffer. Bone marrow-derived DCs (BMDCs) were generated with GM-CSF (20 ng ml−1) and FLT3 ligand (100 ng ml−1) in IMDM (Iscove's Modified Dulbecco's Medium; Gibco, 12440-053) supplemented with 10% FBS, 1% penicillin-streptomycin and 55 μΜ β-mercaptoethanol for 7–10 days. BMDCs were either sorted as cDC1s (CD11c+XCR1+) using a FACSAria flow cytometry sorter (BD Biosciences) or purified using biotin anti-mouse XCR1 antibody (BioLegend,148212) and anti-biotin microbeads (Miltenyi Biotec, 130-090-485) following the manufacturer's protocols for subsequent experiments. Primary cell cultures were tested to be mycoplasma-free by MycoAlert Mycoplasma Detection kit. CD8+ T cells were depleted with anti-CD8 (YTS 169.4, BioXCell) antibodies. Anti-CD8 (100 μg per mouse) antibodies were injected intraperitoneally at the beginning of tumour inoculation and continuously administered every three days. DCs and LLC tumour cells were treated with 1 μM SD-36 for 12 h. Then, DCs and LLC (106 each) were centrifuged at 1,500 rpm for 5 min at 4 °C and washed 3 times with ice-cold PBS. The pellets were resuspended in 100 μl of a 50% (v/v) water/methanol solution and subjected to 3 freeze–thaw cycles. After the final thaw cycle, these samples were centrifuged at 14,000 rpm, 4 °C for 20 min, and the supernatant were collected. All samples were submitted to the pharmacokinetics core (University of Michigan) for analysis. Cells were lysed in immunoprecipitation lysis buffer (50 mM Tris-HCl pH 7.4, 120 mM NaCl, 1 mM EDTA and 0.5% NP-40) and supplemented with Halt Protease and Phosphatase Inhibitor Cocktail (Thermo Fisher Scientific). Cells were repeatedly passed through a 21-gauge needle with sonication. Then 1,000 µg total cell lysates were incubated with the appropriate antibody (2 µg), with rotation overnight at 4 °C, followed by a 3-h incubation with Protein A/G Sepharose beads (Santa Cruz Biotechnology). Immune complexes were washed three times with wash buffer (20 mM Tris-HCl pH 7.4, 100 mM NaCl, 1 mM EDTA and 0.2% NP-40); then the immunoprecipitated proteins were denatured by the addition of sample buffer (Bio-Rad) and boiled for 10 min, resolved by SDS–PAGE, and immunoblotted with indicated antibodies. Cells were lysed in RIPA buffer (Thermo Fisher Scientific) supplemented with Halt Protease and Phosphatase Inhibitor Cocktail (Thermo Fisher Scientific). The protein concentrations of cell lysates were determined by BCA protein assay kit (Thermo Fisher Scientific). Equivalent amounts of total cellular protein were separated by SDS–PAGE, transferred to polyvinylidene fluoride membranes, and immunoblotted with the indicated antibodies. Lysates were analysed using the Proteome Profiler Phospho-Kinase Array Kit (R&D Systems, number ARY003C) according to the manufacturer's instructions. Western blotting data were processed and analysed by image lab 6.1 (Bio-Rad) and ImageJ 1.51n (NIH). For histological analysis, tissue samples were collected, fixed in 10% formalin (Sigma) and processed for formalin-fixed paraffin-embedded tissue analysis. Four-micrometre paraffin sections underwent heat-induced epitope retrieval. Staining was performed with anti-mouse CD31 antibody (Cell Signaling Technology, D8V9E, 77699, 1:100) using an automated immunostainer (Biocare Intellipath, Biocare Medical). Sections were counterstained with haematoxylin (Biocare Medical), and slides were scanned with the Aperio AT2 Scanner (Leica Biosystems Imaging). Quantification was performed with QuPath v0.5.1. Total RNA was isolated from cells by column purification (RNeasy Micro Kit, Qiagen) with DNase treatment. cDNA was synthesized using the RevertAid First Strand cDNA Synthesis Kit (Thermo Fisher Scientific) with random hexamer primers. Quantitative PCR was performed on cDNA using Fast SYBR Green Master Mix (Thermo Fisher Scientific) on a QuantStudio 3 Real-Time PCR System (Thermo Fisher Scientific). Gene expression was quantified using specific primers (Extended Data Table 3). Fold changes in messenger RNA expression were calculated with the ΔΔCt method using Actb as an endogenous control. The synthesis of SD-2301 is outlined in scheme 1 (Supplementary Fig. It began with a known compound, 1, which was converted into compound 2 through a two-step process. First, compound 1 was reacted with hept-6-ynoic acid to form an amide, which was then saponified using LiOH in aqueous media to yield acid intermediate 2 in 65% overall yield. Azide 6 was synthesized in 80% yield from commercially available amine 5 using a previously reported method. A click reaction between alkyne acid 2 and azide 6, with sodium ascorbate and CuSO4, resulted in acid intermediate 3 in 78% yield. Amine 7 was prepared from Boc-l-glutamine through a two-step procedure. Initially, Boc-l-glutamine was coupled with substituted benzyl amine in the presence of HATU (hexafluorophosphate azabenzotriazole tetramethyl uronium) and DIPEA (N,N-diisopropylethylamine) in DMF (N,N-dimethylformamide) to form a Boc-protected amide, which was then Boc-deprotected using 4 (N) HCl in DCM to give amine 7 in 70% yield over two steps. The coupling of acid 3 with amine 7 in the presence of HATU and DIPEA in DMF produced a Boc-protected intermediate. This intermediate was then Boc-deprotected using 4 (N) HCl in dioxane to yield amine intermediate 4 in 75% yield over two steps. Finally, compound 4 was converted into SD-2301 in 86% yield through an amide formation reaction with literature known intermediate 8, using HOBt and DIPEA in DMF. The purity of SD-2301 was confirmed by ultra-performance liquid chromatography (UPLC) to be >99%. UPLC−MS (ESI) m/z: calculated, 683.7 for C64H80N13O15PS2 [M + 2H]+/2; found, 684.09 (Supplementary Fig. Proton nuclear magnetic resonance (1H-NMR) and carbon nuclear magnetic resonance (13C-NMR) spectroscopies were performed on Bruker Advance 400 NMR spectrometers, and chemical shifts are reported in parts per million (ppm) relative to an internal standard (Supplementary Figs. Animal studies were approved by the Institutional Animal Care and Use Committee at the University of Michigan. All mice were maintained under specific pathogen-free housing (about 22 °C with approximately 40% humidity) on a 12-h dark:12-h light cycle. The following mice (at 6–8 weeks of age) (The Jackson laboratory) were used for this study: C57BL/6J, BALB/cJ, NOD-scid IL2Rγ null (NSG), Rag1tm1Mom (Rag1–/–), C57BL/6-Tg(TcraTcrb)1100Mjb/J (OT-I), B6.129S(C)-Batf3tm1Kmm/J (Batf3−/−), B6.129S(Cg)-Stat1tm1Dlv/J (Stat1−/−), B6(129S4)-Xcr1tm1.1(cre) Kmm/J (Xcr1cre), B6.129S1-Stat3tm1Xyfu/J (Stat3fl/fl) mice and CD-1 mice were obtained from Charles River Laboratories. Stat5b–/– C57BL/6J mice were from the National Institutes of Health (Warren Leonard). In-house littermates were used in the control arm when specific mouse strains were generated in-house. MC38 (2 × 106), CT26 (105), EMT6 (105), B16F10 (2 × 105) and LLC (2 × 105) cells were injected subcutaneously into syngeneic mice. Luciferase-expressing ID8 cells (2 × 106) were injected into the peritoneal cavity of female mice. Luciferase-expressing 4T1 cells (105) were injected into female mouse mammary fat pad. In 4T1 tumour metastasis model, treatment with SD-36 (20 mg kg−1) started on day 14, followed by every 3 days. In some cases, peritoneal tumour-bearing mice were similarly treated with SD-36 (20 mg kg−1) or SD-2301 (5 mg kg−1) or were injected intraperitoneally with FLLL32 (30 mg kg−1) for 24 h. For the ICB experiments, PD-L1 monoclonal antibody or isotype antibody (BioXcell, 200 μg) were administrated intraperitoneally into tumour-bearing mice, starting on day 3, then repeated this treatment every 3 days. For DC1 transfusion experiments, bone marrow-derived cDC1s (2 × 106) were sorted and injected intravenously into tumour-bearing Batf3−/− mice on day −2 and day 8. SD-36 (20 mg kg−1) was administered intravenously every three days, starting on day 5. Tumour inoculation time was considered day 0. Tumour cells were inoculated into age-, sex- and source-matched mice. Tumour size was measured using calipers with a Vernier scale and calculated as previously46. To evaluate tolerability and toxicity, tumour-bearing C57BL/6J mice received intravenous injections of SD-2301 (5 mg kg−1) every 3 days for 18 days, with body weights were monitored throughout. For pharmacokinetic evaluation, female CD-1 mice were administered a single intravenous dose of SD-2301 at 5 mg kg−1 with 25% of PCP as the dosing vehicle. Then, 250–300 μl blood samples were collected from 5 min to 24 h. Following centrifugation at 15,000 rpm for 10 min, at least 100 μl of plasma was collected. All samples were analysed by the Pharmacokinetic and Mass Spectrometry core at the University of Michigan. Compound concentrations in plasma were determined using a validated LC-MS/MS method with an internal control. Chromatographic separation was achieved with a Waters XBridge-C18 column (5 cm × 2.1 mm, 3.5 μm) on a Shimadzu HPLC system. Detection was performed on an AB Sciex QTrap 5500 mass spectrometer in positive-ion MRM mode. The mobile phases were 0.1% formic acid in water (A) and in acetonitrile (B). The gradient for B was: 10% (0–0.3 min), increased to 95% at 0.7 min, held for 2.3 min, and returned to 10% for a 2-min re-equilibration. Pharmacokinetic parameters were calculated using noncompartmental methods in WinNonlin v.3.2 (Pharsight). All clinical records used in this study were acquired and used with the approval of Institutional Review Boards. Patients (cohort 1) who underwent ICB therapy were recruited from the University of Michigan Hospital, Ann Arbor, MI, USA. Inclusion in the analysis applied to those who were enroled in the Michigan Center for Translational Pathology's (MCTP) continuous comprehensive clinical sequencing programme39,47,48,49, MI-Oncoseq, and who possessed sequenced libraries of pre-treatment tumours. From the initiation of therapy, overall survival times were measured. Treatment responses were determined according to RECIST1.1 criteria50 and imRECIST51. Integrative clinical sequencing was conducted using standard protocols approved for use in MCTP's Clinical Laboratory Improvement Amendments-compliant sequencing laboratory, as previously described47,48,52. After purification with the AllPrep DNA/RNA/miRNA kit (Qiagen), total RNA was sequenced using the exome-capture transcriptome platform on an Illumina HiSeq 2000 or HiSeq 2500 in paired-end mode. Quality control, alignment, and expression quantification was executed through the standard clinical RNA-seq pipeline, CRISP53. Read count tables were then normalized into fragments per kilobase million (FPKM) then transcripts per million (TPM) using the edgeR 4.2.2 package54. Gene scores in RNA-seq data were generated using the rank-based inverse normal transformation (INT) as demonstrated previously55. STAT5/STAT3 expression was calculated as the ratio of combined TPM of STAT5A and STAT5B (STAT5AB) to STAT3 TPM for all patients (Extended Data Table 1). Stratification was performed at the median STAT5AB/STAT3 TPM ratio and at the median DC1 maturation score. This was performed for cohort 2 (Extended Data Table 2) in each dataset, before combining group-wise. STAT5 pathway INT score was calculated using the GM-CSF reactome gene set (R-HSA-512988). STAT5 pathway/STAT3 was calculated by dividing the INT score by the STAT3 TPM in each respective sample, this value was stratified by the median. The single-cell RNA-seq analysis of TNBC (cohort 3) (dataset GSE169246) was conducted in R using Seurat (v.4.3.0), ssGSEA, and standard data wrangling and plotting packages. All analyses were carried out after subsetting the data for immunotherapy treatment. DCs were extracted based on the original authors' annotations. Only cells expressing 200 or more genes in at least 3 cells were included in the analysis, and cells with mitochondrial reads greater than 10% were excluded. The STAT5 signalling score for each cell was calculated by applying the GM-CSF reactome gene set (R-HSA-512988) as input to the enrichIt function from the escape (v.2.0.0) package, which implements ssGSEA. The genes used for the IL-10-driven STAT3 signalling score were obtained from previously reported gene orthologues in murine DCs56. This gene set comprised: RAP1GAP, CAMKK1, CASR, KIF1A, HPCAL4, DRAM1, RAMP2, MT2A, IGFBP6, GATA3, CXCR5, GDA, FAM65C (also known as RIPOR3), PLET1, SOCS3, MUC1 and GBP5. The DC maturation score was computed using the same method with genes canonically associated with DC maturation, including CD40, CD80, CD83, CD86, LAMP3, CCL19, IL12A, IL12B, CCL5, CCL22, CXCL9, CXCL10, NFKB1, NFKB2, NFKBIA, NFKBIB, FSCN1 and CCR7. The binary division of high and low STAT3 signalling scores was determined by visually identifying two distinct clusters. The monocyte-DCs versus conventional DC distinction was determined through a combination of iterative clustering and expression of CD14. For the in vitro human ICB study, lin−CD45+CD11c+MHCII+ DCs were enriched and sorted from fresh cancer tissues of patients with high grade serous ovarian carcinoma. DCs (106 per ml) were pretreated with SD-36 (1 μM), washed, and co-cultured with normal peripheral blood T cells (2 × 106 per ml) in the presence of anti-human PD-L1 (10 μg ml−1), anti-CD3 (2 μg ml−1), and anti-CD28 (1 μg ml−1) for 3 days. T cell cytokine profile was analysed by FACS. For detecting the effect of SD-2301 on STAT members in human immune cells in vitro, normal human peripheral blood mononuclear cells (PBMCs) were treated with SD-2301 for 14 h, followed by immunoblotting to analyse STAT protein levels. Normal human immune cells were collected from commercial buffy coats (Carter BloodCare). cDC1s were purified from BMDCs as described above. The RNA was isolated by column purification (Qiagen, 217048) with DNase treatment. The RNA-seq was conducted by BGI Genomics. Bulk RNA-seq analysis was performed using the edgeR (v.4.2.2) and limma (v.3.60.6) package workflows. Preliminary quality control measures were performed including the filtering of lowly expressed genes, and the calculation and assessment of library sizes across samples. The raw counts were then transformed to log counts per million (CPM) values. A design matrix was created containing cell population information (Stat3−/− versus Stat3+/+) and then contrasts for pairwise comparisons between Stat3−/− and Stat3+/+ were determined in limma using the makeContrasts function. As an additional normalization measure, removal of heteroscedasticity from the normalized counts data was achieved using the voom function. To identify differentially expressed genes, the mean-variance relationship of the normalized counts was assessed and then the normalized data were modelled as a linear combination of factors and covariates. To assess the DC maturation signature, the log fold change and P values of DC maturation genes reported in the literature including (Cd40, Cd80, Cd83, Cd86, Ido1, Irf7, Tnfsf8, H2-Aa, H2-ab1, Tlr4, Il12b, Il12rb1, Il12rb2 and Cxcl9) was observed from the differential expression analysis results. To assess the significantly upregulated DC maturation pathways, gene set enrichment analysis (GSEA) was performed on the Stat3−/− versus Stat3+/+ differentially expressed genes in R using the gseGO function. To assess the significantly upregulated DC activation pathways in the DCs treated with LPS, GSEA was performed on the LPS-treated Stat3−/− versus Stat3+/+ differentially expressed genes in R using the gseGO function. The z-score of the normalized counts were computed and plotted as a heat map using GraphPad Prism 10.2.2. DC maturation genes were selected from the differential expression analysis results and the log fold change and P values were plotted using GraphPad Prism 10.2.2. The GSEA dot plot was generated by plotting key pathways from the GSEA results using the dotplot function from clusterProfiler 4.9.2 package. The volcano plot was generated by first performing differential expression analysis of LPS-treated Stat3−/− versus Stat3+/+ cDC1s and plotting log fold changes and P values of genes from the STAT5 pathway curated in the Hallmark pathway Molecular Signature Database (MSigDB). Chromatin immunoprecipitation was performed using the SimpleChIP Enzymatic Chromatin IP Kit (Cell Signaling Technology, 9003s). Sheared chromatin was then immunoprecipitated with STAT5 (Cell Signaling Technology, 94205s) and IgG (Cell Signaling Technology, 2729) antibodies. SYBR green master mix (Applied Biosystems) was used to measure amplification of DNA using QuantStudio 3 Fast Real-Time PCR system (Applied Biosystems). The promoters of gene were quantified using the specific primers (Extended Data Table 3). After normalization to the Input DNA, the amount of output DNA of each target protein was calculated by subtracting that of the IgG control. Single-cell suspensions were prepared from fresh mouse tumour tissues or spleen, and lymphocytes were enriched by density gradient centrifugation. To assess intracellular cytokine production, cells were cultured for 4 h in the presence of phorbol myristate acetate (5 ng ml−1; Sigma-Aldrich), ionomycin (500 ng ml−1; Sigma-Aldrich) and brefeldin A (1: 1000, BD Biosciences). Cells were fixed and permeabilized with the Transcription factor staining buffer set (Invitrogen, 00-5523-00). Cellular phenotypes were assessed multi-parameter flow cytometry panels. Data were acquired on a BD LSRFortessa. Antibodies against the following mouse antigens were used: CD45 (30-F11, HI30), CD90 (53-2.1, 30-H12), CD3 (17A2), CD45R/B220 (RA3-6B2), CD4 (RM4-5, GK1.5), CD8 (53−6.7), IFNγ (XMG1.2), granzyme B (GB11), IL-2 (JES6-5H4), Ki-67 (B56), I-A/I-E (M5/114.15.2), CD80 (16-10A1), CD86 (GL1), H-2Kb (AF6-88.5.5.3), H-2 (M1/42), XCR1 (ZET), TNF (MP6-XT22), CD11c (HL3), phospho-Tyr694-STAT5 (SRBCZX), phospho-Tyr705-STAT3 (13A3-1). Additionally, antibodies against the following human antigens were used: CD8 (RPA-T8), IFNγ (B27) and TNF (MAb11). The strategies for DC gating and in vitro cultures and tissues are shown in Extended Data Figs. The strategy for T cell gating is presented in Extended Data Fig. Sample sizes were determined based on previously published studies to ensure appropriate statistical power. For in vivo studies, each group consisted of a minimum of five mice, which was considered sufficient to detect meaningful biological differences with good reproducibility, and mice were randomized into treatment groups at the start of the experiment. Experiments were not performed in a blinded manner, as knowledge of the treatment groups were necessary to carry out the study. t-tests were used to assess differences between two independent experimental groups, and one-way analysis of variance (ANOVA) was used to compare three or more groups. Two-way ANOVA was applied to compare tumour growth curves. Data are presented as mean ± s.e.m., and statistical significance was defined as P < 0.05 for all tests. All statistical analyses related to these experiments were performed using GraphPad Prism software v.10.2.2. Kaplan–Meier analysis was conducted to estimate overall survival, with differences between groups assessed using log-rank tests. Cox proportional hazards models were used for multivariate survival analysis. Pearson's correlation coefficient was applied to evaluate linear correlations between variables, and the Wilcoxon rank-sum test was performed to assess group differences. All P values are two-sided and not adjusted for multiple comparisons. The statistical analyses were conducted using R packages. All human studies were conducted under the oversight and approval of the Institutional Review Board at the University of Michigan Medical School. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Clinical sequencing data are publicly available with raw data available upon request from dbGaP phs000673.v5.p1 (refs. RNA-seq data that were newly generated in this study for in vitro analysis have been deposited at the Gene Expression Omnibus (GEO) at NCBI under accession GSE289764. Single-cell RNA-seq data that support the findings of this study were downloaded from GEO accession GSE169246. The two original melanoma RNA-seq datasets were deposited in the European Nucleotide Archive (ENA) under accession PRJEB23709 and in dbGaP under accession phs000452.v2.p1. The processed data for these two melanoma datasets can be found at https://ngdc.cncb.ac.cn/icb/resources. All materials are available from the corresponding authors upon reasonable request. Source data are provided with this paper. Sharma, P. & Allison, J. P. The future of immune checkpoint therapy. Zou, W., Wolchok, J. D. & Chen, L. PD-L1 (B7-H1) and PD-1 pathway blockade for cancer therapy: mechanisms, response biomarkers, and combinations. Wculek, S. K. et al. Dendritic cells in cancer immunology and immunotherapy. & Gajewski, T. F. Tumor-residing Batf3 dendritic cells are required for effector T cell trafficking and adoptive T cell therapy. & Grandis, J. R. Targeting the IL-6/JAK/STAT3 signalling axis in cancer. June, C. H. & Sadelain, M. Chimeric antigen receptor therapy. Inaba, K., Young, J. W. & Steinman, R. M. Direct activation of CD8+ cytotoxic T lymphocytes by dendritic cells. Everts, B. et al. Migratory CD103+ dendritic cells suppress helminth-driven type 2 immunity through constitutive expression of IL-12. Maier, B. et al. A conserved dendritic-cell regulatory program limits antitumour immunity. 3rd, Dutertre, C. A., Ginhoux, F. & Murphy, K. M. Genetic models of human and mouse dendritic cell development and function. Type I interferon activates MHC class I-dressed CD11b+ conventional dendritic cells to promote protective anti-tumor CD8+ T cell immunity. Ganesan, A. P. et al. Tissue-resident memory features are linked to the magnitude of cytotoxic T cell responses in human lung cancer. Bottcher, J. P. et al. NK cells stimulate recruitment of cDC1 into the tumor microenvironment promoting cancer immune control. Gerhard, G. M., Bill, R., Messemaker, M., Klein, A. M. & Pittet, M. J. Tumor-infiltrating dendritic cell states are conserved across solid human cancers. Expansion and activation of CD103+ dendritic cell progenitors at the tumor site enhances tumor responses to therapeutic PD-L1 and BRAF inhibition. Targeting tumors with IL-10 prevents dendritic cell-mediated CD8+ T cell apoptosis. Guo, C. S. et al. SLC38A2 and glutamine signalling in cDC1s dictate anti-tumour immunity. Li, Y. J., Zhang, C., Martincuks, A., Herrmann, A. & Yu, H. STAT proteins in cancer: orchestration of metabolism. Fu, X. Y. STAT3 in immune responses and inflammatory bowel diseases. & Darnell, J. E. Jr. Stat3: a STAT family member activated by tyrosine phosphorylation in response to epidermal growth factor and interleukin-6. Yu, H., Kortylewski, M. & Pardoll, D. Crosstalk between cancer and immune cells: role of STAT3 in the tumour microenvironment. Liao, W., Lin, J. X., Wang, L., Li, P. & Leonard, W. J. Modulation of cytokine receptors by IL-2 broadly regulates differentiation into helper T cell lineages. Eddy, W. E. et al. Stat5 is required for CD103+ dendritic cell and alveolar macrophage development and protection from lung injury. The ubiquitin ligase MDM2 sustains STAT5 stability to control T cell-mediated antitumor immunity. Choi, J. E. et al. PIKfyve, expressed by CD11c-positive cells, controls tumor immunity. Rosenberg, J. E. et al. Atezolizumab in patients with locally advanced and metastatic urothelial carcinoma who have progressed following treatment with platinum-based chemotherapy: a single-arm, multicentre, phase 2 trial. Yuan, X. et al. Vitamin E enhances cancer immunotherapy by reinvigorating dendritic cells via targeting checkpoint SHP1. Van Allen, E. M. et al. Genomic correlates of response to CTLA-4 blockade in metastatic melanoma. Distinct immune cell populations define response to anti-PD-1 monotherapy and anti-PD-1/anti-CTLA-4 combined therapy. Zhang, Y. et al. Single-cell analyses reveal key immune cell subsets associated with response to PD-L1 blockade in triple-negative breast cancer. Theisen, D. J. et al. WDFY4 is required for cross-presentation in response to viral and tumor antigens. Choi, H. W. et al. Perivascular dendritic cells elicit anaphylaxis by relaying allergens to mast cells via microvesicles. Al-Shami, A., Mahanna, W. & Naccache, P. H. Granulocyte-macrophage colony-stimulating factor-activated signaling pathways in human neutrophils—selective activation of Jak2, Stat3, and Stat5B. The transcription factor Fli1 restricts the formation of memory precursor NK cells during viral infection. Huynh, J., Chand, A., Gough, D. & Ernst, M. Therapeutically exploiting STAT3 activity in cancer - using tissue repair as a road map. Bai, L. et al. A potent and selective small-molecule degrader of STAT3 achieves complete tumor regression in vivo. Nishinol, M., Giobbie-Hurder, A., Ramaiyal, N. H. & Hodit, F. S. Response assessment in metastatic melanoma treated with ipilimumab and bevacizumab: CT tumor size and density as markers for response and outcome. Yu, J. et al. Liver metastasis restrains immunotherapy efficacy via macrophage-mediated T cell elimination. Hildner, K. et al. Batf3 deficiency reveals a critical role for CD8α+ dendritic cells in cytotoxic T cell immunity. Durai, V. & Murphy, K. M. Functions of murine dendritic cells. Transcription factor p91 interacts with the epidermal growth factor receptor and mediates activation of the c-fos gene promoter. Sadowski, H. B., Shuai, K., Darnell, J. E. Jr. & Gilman, M. Z. A common nuclear signal transduction pathway activated by growth factor and cytokine receptors. Wakao, H., Gouilleux, F. & Groner, B. Mammary gland factor (MGF) is a novel member of the cytokine regulated transcription factor gene family and confers the prolactin response. Wang, D. et al. Controllable synthesis of dual-MOFs nanostructures for pH-responsive artemisinin delivery, magnetic resonance and optical dual-model imaging-guided chemo/photothermal combinational cancer therapy. Robinson, D. R. et al. Integrative clinical genomics of metastatic cancer. Wu, Y. M. et al. Inactivation of CDK12 delineates a distinct immunogenic class of advanced prostate cancer. Wang, W. et al. CD8+ T cells regulate tumour ferroptosis during cancer immunotherapy. Watanabe, H. et al. New response evaluation criteria in solid tumours-revised RECIST guideline (version 1.1). Hodi, F. S. et al. Immune-modified Response Evaluation Criteria In Solid Tumors (imRECIST): refining guidelines to assess the clinical benefit of cancer immunotherapy. Distinct structural classes of activating FOXA1 alterations in advanced prostate cancer. The use of exome capture RNA-seq for highly degraded RNA with application to clinical cancer sequencing. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Yang, J. et al. FTO genotype is associated with phenotypic variability of body mass index. Chrisikos, T. T. et al. STAT3 inhibits autocrine IFN signaling in type i conventional dendritic cells. The authors thank J. Deng for providing Stat1–/– C57BL/6J mice (originally from The Jackson Laboratory); M. Wang, B. Wen and D. Sun for the pharmacokinetics study; and our laboratory members for their intellectual input. This work was supported in part by research grants from the Breast Cancer Research Foundation, NIH/NCI R01 grants (CA217648, CA123088, CA099985, CA193136, CA152470 and CA244509), and the NIH/NCI through the University of Michigan Rogel Cancer Center Grant (CA46592). are supported by the Division of Intramural Research, the National Heart, Lung and Blood Institute, the National Institutes of Health. Jiajia Zhou, Kole Tison, Michael Pitter, Linda Vatan, Peng Liao, Jiali Yu, Heng Lin, Shuang Wei, Xue Gao, Sara Grove, Ilona Kryczek & Weiping Zou Center of Excellence for Cancer Immunology and Immunotherapy, University of Michigan Rogel Cancer Center, Ann Arbor, MI, USA Jiajia Zhou, Kole Tison, Michael Pitter, Linda Vatan, Peng Liao, Jiali Yu, Heng Lin, Long Jiang, Shuang Wei, Xue Gao, Sara Grove, Ilona Kryczek, Michael D. Green & Weiping Zou Graduate Program in Immunology, University of Michigan, Ann Arbor, MI, USA Department of Computational Medicine and Bioinformatics, University of Michigan Medical School, Ann Arbor, MI, USA Kole Tison, Long Jiang & Marcin Cieslik Haibin Zhou, Longchuan Bai, Ranjan Kumar Acharyya, Donna McEachern, Hoda Metwally, Yu Wang & Shaomeng Wang Jae Eun Choi, Abhijit Parolia, Marcin Cieslik, Arul M. Chinnaiyan & Weiping Zou Michigan Center for Translational Pathology, University of Michigan Medical School, Ann Arbor, MI, USA Jae Eun Choi, Abhijit Parolia, Marcin Cieslik, Arul M. Chinnaiyan & Shaomeng Wang Department of Radiation Oncology, University of Michigan Medical School, Ann Arbor, MI, USA Laboratory of Molecular Immunology and the Immunology Center, National Heart, Lung and Blood Institute, National Institutes of Health, Bethesda, MD, USA Howard Hughes Medical Institute, University of Michigan Medical School, Ann Arbor, MI, USA University of Michigan Rogel Cancer Center, University of Michigan, Ann Arbor, MI, USA Arul M. Chinnaiyan, Shaomeng Wang & Weiping Zou Graduate Program in Cancer Biology, University of Michigan, Ann Arbor, MI, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived the study, designed experiments, conducted experiments and wrote and revised the manuscript. H.M. provided assistance with in vivo experiments. assisted with human bioinformatic analyses. performed flow cytometry sorting of samples. assisted in preparing single-cell suspensions from mouse tumour tissues. assisted with antigen presentation assays. assisted with the identification of Stat3+/+ and Stat3−/− mice. assisted in acquiring human tumour samples and provided bioinformatic support. conducted in vitro human studies, assisted with the analysis of flow cytometry data, provided scientific input, engaged in discussions, revised the manuscript and supervised the study. contributed to the acquisition of human tumour samples and oversaw bioinformatic support. had a role in acquiring of clinical samples and supervised bioinformatic support. S. Wang conceived the study, developed SD-36 and SD-2301, designed experiments, provided scientific input, participated in discussions, revised the manuscript, supervised the study and acquired funding. Correspondence to Shaomeng Wang or Weiping Zou. serves as a scientific advisor or consultant for Cstone, Hanchor Bio and NextCure and was a consultant for Oncopia. S. Wang was a co-founder and a paid consultant for Oncopia Therapeutics and owned equity in Oncopia, which was acquired by Roivant Sciences and Proteovant Therapeutics. S. Wang was a paid consultant for Proteovant Therapeutics and Roivant Sciences and owned stock in Roivant Sciences. S. Wang was the principal investigator for a research contract between Oncopia/Proteovant and the University of Michigan. S. Wang is a co-founder, paid consultant, and member of the board of directors of Ascentage Pharma Group International and owns stock in Ascentage. was a co-founder and a paid consultant for Oncopia and owned equity in Oncopia and was a paid consultant for Proteovant Therapeutics. The University of Michigan owned stock in Oncopia and Ascentage. The University of Michigan has filed multiple patents on STAT3 degraders, which have been licensed by Oncopia/Proteovant (now SK Life Science Labs) for clinical development. and H.M. are inventors on SD-36 and/or SD-2301 and have received royalty payments from the University of Michigan. All other authors declare no competing interests. Nature thanks David Frank, Ignacio Melero and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The Kaplan Meier (KM) survival curve was plotted to visualize the difference in survival probability between patient stratifications. Cohort 1 patients with ICB unresponsive tumors (sarcoma, lymphoma, prostate cancer, and others) were excluded for analysis. n = 69, p = 0.0374, the CoxPH model calculates the effect of the STAT5 pathway/STAT3 DC maturation (high/high group) on survival outcome relative to STAT5 pathway/STAT3 DC maturation (high/low group) when tumor type is controlled. b-c. KM curves of OS in melanoma patients (cohort 2) treated with ICB, by low or high DC score and the ratio of STAT5 versus STAT3 (DC1HiSTAT5/STAT3Hi, DCHiSTAT5/STAT3Lo, DCLoSTAT5/STAT3Hi, DCLoSTAT5/STAT3Lo). n = 72 (b), n = 42 (c), p = 0.038 (b), p = 0.051 (c) by log-rank test. d. High and low STAT3 gene signaling scores in patients with TNBC (cohort 3). Based on the STAT3 ssGSEA score, two distinct clusters were resolved. STAT5 (e) and STAT3 (f) gene signatures were analyzed in DCs in the post-ICB samples of non-responders compared to their pre-ICB samples. NS, not significant, Wilcoxon Rank-sum test. g. UMAP representations were used to identify different DC subtypes from the integrated scRNA-seq datasets of TNBC (cohort 3). The cells were then color-coded according to their respective DC subtypes. Outliers are plotted as individual points. Xcr1 Cre knock-in mice express an mCherry fluorescent protein under the control of the Xcr1 promoter in cDC1s. mCherry expression and XCR1-APC-Cy7 staining in DCs from Stat3+/+ and Stat3−/− mice were analyzed by FACS (a). mCherry expression and XCR1-APC-Cy7 staining in DCs (CD11c), B cells (B220), T cells (CD3 and CD90), macrophages (F4/80) from Stat3−/− mice were analyzed by FACS (b). One of four mice is shown. c. Gating strategy for cDC1s (CD11c+XCR1+) in BMDCs. The representative images show mouse body size (d), lymphoid organs (e), and T cell subset distribution (f-g) in Stat3+/+ and Stat3−/− mice. One of three experiments is shown. Real time qPCR shows the expression of cytokine receptor mRNAs in mouse cDC1s. mean ± SEM, n = 4, ****p < 0.0001, one-way ANOVA. Effect of GMRβ knockdown on the JAK-STAT signaling pathway. JAWSII cells were transfected with shRNAs against Gmrβ or control shRNA and cultured with GM-CSF. Cell lysates were subjected to analysis by immunoblots. One of three experiments is shown. k. Effect of GM-CSF on JAWSII cells. The immunoblots show the expression levels of JAK family members in JAWSII cells. One of three experiments is shown. l-m. Effect of STAT3 deletion on GM-CSF and GMRβ expression in cDC1s. GM-CSF was detected by ELISA in culture supernatants in Stat3+/+ and Stat3−/− cDC1s on day 5 (l). n = 3, NS, not significant, unpaired two-tailed t-test. GMRβ was determined by Western blotting (m). One of three experiments is shown. n. The fractions of IL-6+ DCs (in red) and non-DCs are depicted in the TNBC patient dataset30. 2n) shows the expression of Il6 by TPM (Transcripts Per Million) in mouse Stat3+/+ and Stat3−/− cDC1s. n = 4, NS, not significant, unpaired two-tailed t-test. p. Effect of STAT3 deletion on IL-6 expression in cDC1s. IL-6 was detected by ELISA in culture supernatants in Stat3+/+ and Stat3−/− cDC1s on day 4. n = 3, NS, not significant, unpaired two-tailed t-test. 2n) shows the expression of IL-6 receptors (Il6ra, Il6st) by TPM (Transcripts Per Million) in mouse Stat3+/+ and Stat3−/− cDC1s. n = 4, NS, not significant, unpaired two-tailed t-test. One of three experiments is shown. Effect of FLLL32 on STAT5 activation and DC maturation in vivo. Stat3+/+ and Stat3−/− mice bearing peritoneal B16F10 (t) and MC38 (u-x) tumors were treated with FLLL32 on day 9 for 24 h. FACS shows expression of pSTAT5 (t, u), MHC-I (v), MHC-II (w), and CD80 (x) in peritoneal cDC1s. GSEA analysis reveals significant differences in the CCR chemokine receptor binding (y) and interferon gamma production (z) between Stat3+/+ and Stat3−/− cDC1s. Source data contained unprocessed WB. a-d. Effect of STAT3 deletion on DC-mediated T cell activation. OT- I cells were cultured with Stat3+/+ or Stat3−/− cDC1s without loading SIINFEKL peptides. FACS shows the percentage of Ki67+ (a), IFNγ+ (b), TNFα+ (c), and GZMB+ (d) OT- I cells. e. Gating strategy for cDC1s (CD11c+XCR1+) in MC38 tumor bearing mouse model. Lin− CD11c+XCR1+ cDC1s were enriched and sorted from the TME in Stat3+/+ and Stat3−/− mice bearing peritoneal MC38 tumors. Co-immunoprecipitation (IP) was performed with anti-JAK2 in DC lysates. One of two experiments with repeats is shown. MC38 tumors were inoculated into Stat3+/+ and Stat3−/− mice. MHC-II (g, h), and CD86 (i, j) expression in cDC1 from MC38 tumor-draining lymph nodes (TDLNs) were analyzed by FACS. k. Gating strategy for CD8+ T cells in MC38 tumor bearing mouse model. Effect of STAT3 deletion on MC38 tumor progression. FACS shows the percentages of MC38 tumor infiltrating IL-2+CD8+ T cells (l, m), and the percentages of TDLN IL-2+ (n, o), and IFNγ+ (p, q) CD8+ T cells. For l, n, and p, one representative dot plot is shown in each panel. Effect of STAT3 deletion on the phenotype of DCs and T cells in B16F10 bearing mice. FACS shows the expression of MHC-I (r, s) and CD80 (t, u) in tumor-infiltrating cDC1s, and the percentages of tumor infiltrating IL-2+ (v, w) and TNFα+ (x, y) CD8+ T cells. For r, t, v, and x, one representative dot plot is shown in each panel. Source data contained unprocessed WB. b. Immunoblotting shows STAT1 in shStat1 JAWSII cells. FACS shows the expression of MHC-I (c), MHC-II (d), and CD80 (e) in shStat1 JAWSII cells treated with SD-36. f. Immunoblotting shows STAT1 in bone marrow derived DC1s from Stat1+/+ and Stat1−/− mice. One of three experiments is shown. FACS shows the expression of pSTAT5 in bone marrow derived Stat1+/+ and Stat1−/− cDC1s. n = 3, NS, not significant, unpaired two-tailed t-test. Stat1+/+ and Stat1−/− DCs were treated with SD-36 for 12 h. FACS shows the expression of MHC-I (h), MHC-II (i), and CD80 (j) in cDC1s. k. Effect of SD-36 on B16F10 tumor progression. B16F10 tumor bearing C57/B6 mice were treated with 10 mg/kg SD-36. mean ± SEM, n = 7, *p = 0.0114, two-way ANOVA. l-m. Effect of SD-36 on MC38 tumor progression. MC38 tumor bearing C57/B6 mice were treated with SD-36. Tumor weight (l) and image (m) were shown. mean ± SEM, n = 5, ***p = 0.0004, unpaired two-tailed t-test. n-s. Effect of SD-36 on tumor progression (n-p) and mouse body weights (q-s). Mice bearing B16F10 (n, q), CT26 (o, r), and EMT6 (p, s) were treated with SD-36 (100 mg/kg) three times a week when tumor volume reached 100 mm³. Tumor volumes and mouse body weights were monitored. Effect of SD-36 on large tumor progression. Mice bearing CT26 tumors (500 mm3/tumor) were treated with vehicle (t) or SD-36 (u). Source data contained unprocessed WB. Effect of SD-36 on tumor growth in NSG mice. MC38 tumor-bearing NSG mice were treated with SD-36 (100 mg/kg) or vehicle. mean ± SEM, *p = 0.0326, two-way ANOVA. b. Efficacy of anti-CD8 mAb in CD8+ T cell depletion. MC38 bearing were treated with anti-CD8 mAb. FACS shows CD8+ T cells in the tumor draining lymph node (TDLN). 4T1 tumor bearing mice were treated with SD-36. FACS shows the percentages of TDLN TNFα+ (c, d), and IFNγ+ (e, f) CD8+ T cells. For c, e, one representative dot plot is shown in each panel. FACS shows the percentages of TNFα+ (g, h), IFNγ+ (i, j), and GZMB+ (k, l) CD8+ T cells from tumors (g-l) and tumor-draining lymph nodes (m-r). For g, i, k, m, o, and q, one representative dot plot is shown in each panel. s. Effect of SD-36 on tumor growth in Batf3−/− mice. Batf3−/− mice were inoculated with LLC cells and with SD-36 (n = 8) or vehicle (n = 11). t. Effect of SD-36 on STAT3 protein in DCs and LLC tumor cells. Immune blotting shows the levels of STAT3 in DCs and LLC tumor cells treated with SD-36 for 12 h, and the ratio of STAT3 to actin expression. u. DCs and LLC tumor cells were treated with 1 μM SD-36 for 12 h. Intracellular SD-36 was quantified by LC-MS/MS. mean ± SEM, n = 4, **p = 0.0011, unpaired two-tailed t-test. MC38 tumor-bearing mice were treated with SD-36 or vehicle. Immune blotting shows key transcriptional factors in DCs isolated from spleens. One of three experiments is shown. MC38 tumor bearing Stat3+/+ and Stat3−/− mice were treated with SD-36 or vehicle. mean ± SEM, n = 7, **p = 0.0056, two-way ANOVA. These mice were treated with SD-36. mean ± SEM, n = 5, **p = 0.002, two-way ANOVA. Source data contained unprocessed WB. DCs from human ovarian cancer tissues were pretreated with SD-36, washed, and co-cultured with T cells in the presence of anti-human PD-L1 for 3 days. DCs were treated with SD-2301 (50 nM) for 24 h. Immune blotting shows STAT3 protein. Wild type mice bearing peritoneal B16F10 tumors were treated SD-2301 (5 mg/kg). Immune blotting shows STAT3 expression in peritoneal DCs. One of three experiments is shown. B16F10 bearing mice were treated with SD-2301 (5 mg/kg) or SD-36 (20 mg/kg) every 3 days. Tumor volume was monitored (g). FACS shows the expression of IFNγ (h, i) and GZMB (j, k) in B16F10 infiltrating CD8+ T cells, and the expression of MHC-I (l), MHC-II (m), and CD80 (n) in tumor infiltrating cDC1s. o-p. Effect of SD-2301 on blood vessels in tumors. B16-F10 and MC38 bearing mice were treated with SD-2301. CD31+ vessel density was quantified in B16-F10 (o) and MC38 (p) tumors. Representative immunohistochemical staining images and the quantification of CD31+ vessel densities are shown. n = 4 tumors, NS, not significant, unpaired two-tailed t-test. Effect of SD-2301 on mouse body weights. Mouse body weights were monitored. r-s. Effect of SD-2301 on ICB efficacy in vivo. B16F10 (r) or MC38 (s) bearing mice were treated with SD-2301, anti-PD-L1, and their combination. The group of B16F10 bearing mice (r) treated with SD-2301 is identical to that shown in Extended Data Fig. Effect of SD-2301 on neoantigen specific CD8+ T cell response. MC38 bearing mice were treated with SD-2301 or vehicle. FACS shows the percentage of tumor infiltrating KSPWFTTL tetramer+CD8+ T cells. means ± SEM, n = 6, *p = 0.0216, unpaired two-tailed t-test. B16F10 tumor bearing Stat3+/+ and Stat3−/− mice were treated with SD-2301 or vehicle. mean ± SEM, n = 7, ****p < 0.0001, Two-way ANOVA. MC38 bearing Batf3+/+ and Batf3−/− mice were treated with SD-2301 or vehicle. The group of control mice treated with SD-2301 is identical to that shown in Extended Data Fig. mean ± SEM, n = 7, ****p < 0.0001, Two-way ANOVA. Female CD-1 mice (n = 3) were administered with SD-2301 (5 mg/kg). Blood samples were collected for PK analysis. Cl, clearance; Vss, volume of distribution at steady state; T1/2, elimination half-life; Cmax, maximum drug concentration; AUC, area-under-the-curve. Effect of SD-2301 on STAT3 expression in human immune cells. Human peripheral blood mononuclear cells were treated with different concentrations of SD-2301 for 14 h. The immunoblots show the expression levels of STATs. The schematic diagram shows that STAT3 restrains the JAK2-STAT5 signaling pathway, thereby reducing STAT5-dependent DC maturation and lessening T cell immunity and immunotherapy effectiveness. Source data contained unprocessed WB. This file contains experimental procedure for SD-2301, and Supplementary Figs. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Zhou, J., Tison, K., Zhou, H. et al. STAT5 and STAT3 balance shapes dendritic cell function and tumour immunity. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09008-9'>Dopaminergic action prediction errors serve as a value-free teaching signal</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:31:54
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Choice behaviour of animals is characterized by two main tendencies: taking actions that led to rewards and repeating past actions1,2. Theory suggests that these strategies may be reinforced by different types of dopaminergic teaching signals: reward prediction error to reinforce value-based associations and movement-based action prediction errors to reinforce value-free repetitive associations3,4,5,6. Here we use an auditory discrimination task in mice to show that movement-related dopamine activity in the tail of the striatum encodes the hypothesized action prediction error signal. Causal manipulations reveal that this prediction error serves as a value-free teaching signal that supports learning by reinforcing repeated associations. Computational modelling and experiments demonstrate that action prediction errors alone cannot support reward-guided learning, but when paired with the reward prediction error circuitry they serve to consolidate stable sound–action associations in a value-free manner. Together we show that there are two types of dopaminergic prediction errors that work in tandem to support learning, each reinforcing different types of association in different striatal areas. When animals and humans make choices, they exhibit two key tendencies: pursuing rewarding actions and repeating past actions1,2. Dopamine neurons that encode reward prediction error (RPE)3 provide a critical teaching signal for value-based learning, reinforcing actions that lead to reward. Recently it has been proposed that repetitive ‘habitual' choices may instead be updated by a movement-based teaching signal4,5,6. Although experimental evidence is lacking, theoretical models suggest that this value-free learning system could operate alongside canonical value-based learning in the basal ganglia5,6. These models predict two types of dopaminergic teaching signals: RPE, which reinforces reward-driven actions, and APE, which reinforces repeated actions4,6. Notably, dopaminergic neurons located in the ventral tegmental area (VTA) and medial parts of the substantia nigra pars compacta (SNc) encode RPE7,8, whereas those in the lateral SNc and substantia nigra pars lateralis (SNl), which project to the tail of striatum (TS), prominently respond to movement9,10,11,12. Here we investigated whether movement-related dopaminergic activity encodes an APE, providing a value-free teaching signal to reinforce repeated state–action associations. To address this, we examine the role of dopamine in the TS as mice learn an auditory discrimination, cloud-of-tones (COT)13 task. We show that dopaminergic input to this region is important for learning, and that dopaminergic activity during the task is movement-related. We go on to show that this movement-related dopaminergic activity encodes an APE. Causal manipulations show that this value-free teaching signal can reinforce state–action associations, essentially biasing mice to repeat the actions that they have taken in the past. We propose a model where this movement-based teaching signal works in conjunction with canonical RPE signalling to boost and stabilize learning about consistent state–action associations. In the COT task, mice initiate trials by nose-poking a central port, triggering auditory stimuli composed of a short train of overlapping pure tones (5–40 kHz). They selected a left or right reward port on the basis of whether the stimulus contained primarily low (5–10 kHz) or high (20–40 kHz) frequencies (Fig. In line with previous reports14, bilateral inactivation of the TS with muscimol (a GABAA (γ-aminobutyric acid type A) receptor agonist), but not the dorsomedial striatum (DMS), impaired task performance (Fig. Unilateral optogenetic inactivation of either type of striatal projection neurons (SPNs) in the TS also had an opposing and significant effect on choices of mice (Fig. These results demonstrate that the TS is needed to execute the learned behaviour and that both populations of SPNs exert opposing contributions to the auditory-guided choices. Lines represent logistic fits of the means. Lines represent logistic fits of the means. f, Quantification of the bias for each session shown in e. Scatter dots represent the mean for each session, and error bars illustrate the variation of shuffled data (Methods). g, Example lesion and control mice histology. h, Learning rate of the lesioned TS (n = 11 mice) and control (n = 10 mice) groups (lines represents group means). i,j, Maximum performance (i; P = 0.0023, Kruskal–Wallis test; Cohen's d = −1.70) and maximum learning rate (j; P = 0.0006, Kruskal–Wallis test; Cohen's d = −2.04) between the groups (same mice as in h). k, Example TH staining in the TS for control and lesion mice. l, Learning rate of the TS dopamine-ablated (n = 9 mice) and control (n = 5 mice) groups. m,n, Differences on the maximum performance (m; P = 0.006, Kruskal–Wallis test; Cohen's d = −2.39) and maximum learning rate (n; P = 0.014, Kruskal–Wallis test; Cohen's d = −1.71) between the groups (same mice as l). To test whether the TS was also needed to learn the task, we ablated the TS prior to training using a viral-mediated caspase-based strategy (Fig. Lesions of the TS caused a deficit in learning (Fig. To confirm that the deficits in learning are not due to an inability of the mice to act upon the learned sound–action association we infused an N-methyl-d-aspartate (NMDA) receptor antagonist, d-2-amino-5-phosphonovalerate (d-AP5) in the TS (Extended Data Fig. d-AP5 infusions did not affect performance in expert mice (Extended Data Fig. 3c) but significantly impaired learning when infused during training (Extended Data Fig. In addition, ablation of the TS-projecting dopamine neurons also recapitulated the general TS lesions effects. TS dopamine-ablated mice had deficits in learning (Fig. 3k), or the time taken between trials (Extended Data Fig. Together, these results confirm that both the TS and its dopaminergic innervation are required to facilitate learning and execute the auditory discrimination task. To understand the role of TS dopamine in the task, we measured its dynamics with dLight1.1, a genetically encoded fluorescent dopamine sensor15. TS dopamine responses correlated in time with contralateral movements from the centre port, in sharp contrast to the large reward responses in ventral striatum (VS) (Fig. a, Schematic of the experimental approach for recording dopamine activity in the VS and TS. b, Fluorescent images showing optical fibre locations and dLight expression in the TS and the VS. c, Recording fibre locations from the VS and TS matched to the reference atlas. d, Example of a TS and a VS recording session aligned to time of leaving the centre port to make a contralateral (contra) choice or an ipsilateral (ipsi) choice. White dot shows time of entering contralateral or ipsilateral choice port. e, Average photometry traces in TS (n = 10 mice) and VS (n = 7 mice) aligned to task events. Shaded time windows show significant differences between the two trace types in each subplot, calculated by performing two-sample t-tests on 0.1-s bins and a P value threshold for significance of 0.01. f, Average response kernels to behavioural events for recordings in the TS and VS. Shaded time windows are calculated as in e. of the whole recording session (VS median = 31.3; TS median = 5.50) and for different behavioural kernels for linear regression models fitted on VS (n = 7 mice) and TS (n = 10 mice) recordings. To separate dopamine responses associated with overlapping behavioural events, we applied a linear regression model to photometry data obtained early in training. VS responses were best explained by the outcome kernel, capturing the large responses to rewards and dips for unrewarded trials (Fig. By contrast, TS showed minimal outcome-related dopamine activity (Fig. In line with other studies16,17,18, we observed dopaminergic reward responses when we recorded in the posterior region of the dorsolateral striatum (pDLS), which is located just rostral to the TS and is not prominently innervated by the primary auditory cortex19 (Extended Data Fig. The largest TS dopaminergic response was contralateral movement-locked activity (Fig. 2e–g), which was also seen when mice made contralateral movements to return from the side ports to the centre port (Extended Data Fig. VS movement-related activity was smaller, with no significant difference between contralateral and ipsilateral actions (Extended Data Fig. These results show that VS dopamine activity significantly encodes reward outcome, consistent with RPE, whereas the TS dopamine activity encodes movement information. To confirm that TS dopamine activity was unrelated to sound, we omitted the cue on some trials (Extended Data Fig. 5a–g) and found no significant difference in response. To assess task dependence, we recorded TS dopamine activity as mice explored an open arena. As in-task recordings, TS dopamine increased during contralateral movements (Extended Data Fig. 5h–j), and its signal scaled with movement amplitude (Extended Data Fig. Turn angle significantly correlated with TS dopamine (Extended Data Fig. 5n), a correlation that was absent in VS. These results confirm that TS dopamine encodes information about movement. To determine whether there was any additional sensory response to the sound stimuli, we trained mice on a version of the task in which sounds were played when the mice left the side ports and returned to the centre port to initiate the next trial (Extended Data Fig. There was no significant response to the sound in this version of the task but the response to contralateral orienting movement from the centre port remained (Extended Data Fig. In addition, there was no significant response when the sound cues were played as the mice freely explored an open arena (Extended Data Fig. 5u–y), confirming that the recorded TS dopamine signal correlated with movement and not sound. Our results indicate that TS dopamine is crucial for learning the auditory frequency discrimination task and encodes information about movement rather than reward or sound. Although dopamine RPEs are known to drive cortico-striatal plasticity and learning3,20,21,22, the role of movement-related dopamine remains unclear. Habit-formation models predict that movement-related dopamine could encode a value-free APE, representing the discrepancy between an executed action and its predicted occurrence in a given state4,6. If the movement-related dopaminergic activity in the TS encodes an APE, then dopamine activity in the TS should decrease over time as mice learn to predict the action that they will take in response to the sound. To test this prediction, we recorded TS and VS dopamine signals over the course of learning and compared these signals to values of RPE and APE from a dual value-based/value-free reinforcement learning model (Extended Data Fig. Characteristic of an RPE signal, cue-related dopamine activity in the VS grew over time (Fig. 3a–d) after an initial fall in the response that might be due to changes in stimulus salience or alternative interpretation of VS dopamine signal23,24,25. In line with encoding an APE signal, the movement-related dopamine activity in the TS decreased as mice learned the task (Fig. b, Example VS dopamine response size to contralateral cue binned every 40 trials. c, Average change in contralateral cue-aligned VS dopamine response. d, Size of contralateral cue-aligned dopamine response in VS in the first and last session of training (n = 7 mice), P = 0.016 (paired two-sided t-test), Cohen's d = −1.25. e–g, As a–c but for TS recordings (n = 6 mice). h, As d but for the TS (n = 9 mice), P = 0.006 (paired two-sided t-test). Cohen's d = 1.19. i, Modelled responses for APE at the time of correct contralateral choice if the previous choice for that stimulus was ipsilateral or contralateral. j, As i but for an example average (mean) TS dopamine response. One-sided t-test against zero, corrected using the Bonferroni method for multiple comparisons. VS: n = 7 mice, P = 0.005, 1.0, 1.0, 1.0, 1.0 (left to right), (Cohen's d = 2.23, 0.37, 0.23, 0.17, 0.13 (left to right)). TS: n = 6 mice, P = 0.04, 0.20, 0.20, 0.47, 0.63 (left to right), (Cohen's d = −1.72, −1.13, −1.13, −0.84, −0.75 (left to right)). l,m, As i,j but for the VS response. o, Modelled APE and RPE signals following the state change. q, TS dopamine response to the contralateral action before and after the introduction of the novel state (P = 0.01, paired two-sided t-test) (n = 6 mice), Cohen's d = −1.81. r, As p but for a VS recording aligned to cue. s, As q but for VS recording aligned to cue. (P = 0.02, Wilcoxon signed-rank test) (n = 7 mice), Cohen's d = 1.04. To confirm that this decay in the signal was not due to changes in the way that mice were moving and performing the task, we tracked changes in speed and kinematics over the course of learning. Fitting a linear regression model that considered the log trial number, speed and turn angle in each mouse revealed that the change in TS dopamine signal that is predicted by log trial number persisted even once the proportion of the signal that could be predicted by movement alone was accounted for (Extended Data Fig. The majority of the explained variance in a full model with movement and log trial number was captured by changes in the trial number rather than changes in speed or turn angle (Extended Data Fig. In agreement with this, the TS dopamine response was on average significantly smaller when mice repeated the same action in response to the same stimulus in the past trial (Fig. 3l,m), consistent with an RPE signal as the sound–action pairing would have a larger predicted value on the subsequent trial (Fig. Together these results show that the movement-related dopamine activity in the TS does not reflect a pure movement signal that is invariably linked to action characteristics, rather the size of the signal changes on a trial-by-trial basis depending on how predictable the sound–action association is, as expected from an APE signal. The learning-associated changes in movement-related TS dopamine suggests that responses are modulated by how predictably an action follows a given state (auditory cue). Therefore, if trained mice were to make a familiar movement in response to an unfamiliar sound, the action would not be predicted by the stimulus and consequently the resulting APE should be larger. To examine this hypothesis, we replaced the auditory cue associated with the contralateral action with a novel cue (white noise) (Fig. 6o–t) and significantly increased the movement-related TS dopamine signal (Fig. 3p,q), as expected from an APE signal (Fig. There was no significant TS dopamine response to the unfamiliar white noise stimulus when it was played as mice freely explored an open arena (Extended Data Fig. 3r,s), as there is less value predicted by the novel cue (Fig. c, Average performance (mean) of mice. Pre-stimulation data are pooled across all 13 recording sessions (n = 7 mice, 13 hemispheres). VS: 4 mW P = 0.57, Cohen's d = −0.19; 8 mW P = 0.742, Cohen's d = 0.19; Wilcoxon signed-rank test relative to zero (two-sided). h, Average change in the bias for trials preceded by small or large dopamine choice movement responses (DA) in TS (n = 4, mice), error bars represent 68% confidence interval. i, Regression coefficients from a logistic regression (n = 4 mice) (log uncertainty: P = 0.006, Cohen's d = 3.31; dopamine: P = 0.03, Cohen's d = 2.01; one-sample t-test against zero, two-sided t-test). j, As h but for VS dopamine responses (n = 5 mice). k, As i but for VS dopamine at time of choice. log uncertainty: P = 0.03, Cohen's d = 1.44; dopamine (at time of cue): P = 0.29, Cohen's d = −0.54; one-sample t-test against zero, two-sided t-test. l, Example trajectories with smallest or largest Fréchet distances. One-sample t-test against zero, P = 0.03, Cohen's d = −1.24. Another hallmark of an APE is that it should be value-free and should not be modulated by either the outcome or the predicted value of an action. In agreement with this, the dopamine signal in the TS did not significantly respond to a larger or smaller than predicted reward (Extended Data Fig. By contrast, the VS dopamine signal was altered in a manner consistent with RPE (Extended Data Fig. The size of the TS dopamine signal was also not modulated by changes in the predicted value of an action (Extended Data Fig. 7g–i), in contrast to the RPE-consistent VS signal (Extended Data Fig. Finally, we compared our data to other theoretical models of dopamine activity, movement per se, novelty or salience (Methods). Unlike the models for APE and RPE, none of the models consistently matched the pattern of modulation across conditions that we observed in our recordings (Extended Data Fig. Together, our results reveal that the movement-related TS dopamine signal is consistent with encoding APE. To determine whether the TS dopamine signal can act as a teaching signal, we optogenetically stimulated TS dopamine release at different task epochs. To mimic the endogenous movement-related TS dopamine signal, we stimulated unilaterally at the centre choice port, in trials where there was more sensory evidence for a contralateral choice (Fig. Within sessions, stimulation induced a significant contralateral choice bias (Fig. This bias developed over the course of the session as would be expected if it influenced learning (Extended Data Fig. 9d) and could be replicated in a model where we artificially stimulated APE (Extended Data Fig. Optogenetic stimulation did not bias action directly as there was no choice bias on individually stimulated trials (Extended Data Fig. Stimulating dopamine release in the VS at the time of choice had no significant effect (Fig. In a free choice paradigm TS dopamine stimulation did not induce a choice bias but there was a significant bias towards the stimulated port when we stimulated dopamine release in the VS (Extended Data Fig. Finally, there were no appetitive or aversive effects of the stimulation in a real-time place-preference assay when we stimulated dopamine release in the TS (Extended Data Fig. Together these results show that TS dopamine release can, as predicted by the computational models, reinforce state–action but not state–outcome associations. Other theories suggest that movement-related dopamine facilitates movement initiation or modulates ongoing action9,26,27. However, closed-loop optogenetic stimulation of TS dopamine release in an open-field arena did not influence movement likelihood or alter movement parameters when mice did move (Extended Data Fig. These findings suggest TS dopamine activity reinforces state–action associations rather than influencing ongoing action. Since TS dopamine stimulation reinforced state–action associations, we investigated whether endogenous TS dopamine release functioned similarly. A logistic regression model predicting choice repetition from the previous trial's dopamine response and current log uncertainty showed significant positive correlations for both factors (Fig. 4g–i), indicating that mice were more likely to repeat their previous choice when the TS dopamine response was larger and sensory uncertainty was higher. By contrast, dopamine reward response size in the VS did not correlate with choice bias (Fig. These results suggest that movement-related dopamine at choice timing serves as a value-free teaching signal, reinforcing stimulus–action associations in the TS so that mice learn to repeat the action that they have taken in the past when they hear the auditory stimulus. If the TS dopamine response encodes an APE on the current trial, it should not only bias mice toward repeating stimulus–action associations but also influence trial kinematics, making subsequent actions more similar. To test this, we performed a linear regression between the current trial TS dopamine response at time of choice and the Fréchet distance between current and subsequent trial trajectories (which provides a measure of the similarities between the two trajectories) (Fig. The analysis revealed a significant negative correlation between TS dopamine response size and Fréchet distance, indicating that a larger TS dopamine response in a given trial led to a more similar subsequent trajectory (Fig. These findings show that larger TS dopamine signals bias mice to repeat prior state–action associations with more similar movement trajectories. To examine interactions between striatal regions receiving RPEs or APEs, we built a basal ganglia model that simulated learning with these prediction errors (Fig. Our dual value-based/value-free model had an actor and critic, updated by RPE, that learned to control actions and generate reward predictions. The value-free control system, updated by APE, learned to control action and generate action predictions. A model with only a value-free controller was unable to learn the task, as the model continues to repeat what it has done in the past, which is to randomly choose left or right in equal proportion (Fig. By contrast, a model with only the value-based controller was able to independently learn the task (Fig. Notably, a model with both controllers learned faster than the value-based system alone (Fig. Initially, there was no difference between the learning rate of these two models, because early learning is exclusively driven by RPE updates. However, as the dual model begins to consistently choose a particular action in response to a particular sound, APE updates started to reinforce specific stimulus–action associations, boosting performance and learning rate. This divergence between the value-based and dual-controller model recapitulates the learning deficits observed when we lesioned the TS or removed dopamine innervation to this region (Extended Data Fig. Our model shows that the value-free controller alone cannot support learning but when paired with the value-based controller, it acts to boost learning and store stable associations. a, Schematic of the network model. Bottom, differences in performance between the combined model and the value-based model (12 random agents selected for each). c, Change of the model weights for a reward association (high tone→left action), as means for 100 agents. Vertical lines indicate inactivation time points in d. d, Performance levels before and after (as the mean after ten trials) model inactivations of the TS or the actor networks. e, Schematic of the experimental approach for acute inhibition of D1 SPNs or D2 SPNs in the TS. f, Quantification of the contralateral bias on opto-stimulated trials (Methods) for each session as a function of session's performance. h, Final weights in the TS for each sound–action association. Although both the value-based and value-free controllers can control behaviour, our muscimol inactivation data (Fig. 1c) highlights that in well-trained mice, behavioural performance relies on the TS (value-free controller). In other dual-controller models, this transfer is achieved by an external arbiter4,6. We examined if a similar transfer would occur if the actor's weights decayed over time when RPE was low, consistent with striatal literature28,29,30,31. When the value-based actor's weights decayed, there was indeed a transfer of control in our model. Initial performance was controlled by the value-based controller, and over time, the value-free controller took over (Fig. To test whether the TS also slowly takes over control of performance like the value-free controller in our model, we optogenetically inactivated either type of TS projection neuron throughout learning (Fig. Early in training optogenetic inactivation of neither type of TS-SPNs had a significant effect on behaviour (Fig. However, as training progressed the inactivation began to have a significant effect on choices and these effects grew larger as the mice became experts at the task. These behavioural effects began to consistently increase after the mice were performing above 65% (Fig. This is a comparable training stage to where the control and lesion groups diverged when the TS was lesioned experimentally or in the model (Extended Data Fig. Taken together, this data shows that the TS only begins to contribute to task performance and learning once mice are already preferentially choosing a specific action in response to the auditory cues. In the COT task, frequency-specific cortico-striatal plasticity supporting appropriate sound–action associations develops in each hemisphere of the TS throughout learning32,33. The same appropriate sound–action associations also form in our value-free controller, showing that APEs could be used to drive the frequency-specific changes in cortical striatal plasticity as mice learn the task (Fig. Together, our behavioural results, chronic lesions, acute inactivation, dopamine recordings, optogenetic perturbations and previous synaptic plasticity results32 are consistent with the TS forming the value-free part of a dual value-based/value-free learning system (Extended Data Fig. Here we show that movement-related dopaminergic activity in the TS acts as a teaching signal to reinforce state–action associations. This value-free signal teaches mice to repeat past actions. Alone the value-free system (APE→TS) is not able to support reward-guided learning but in conjunction with the canonical RPE system it learns to mimic and store the value-guided state–action associations. Together we show that there are two types of dopaminergic prediction errors that work in tandem to support learning, each reinforcing different types of association in different areas of the striatum. The identification of dopamine transients prior to movement initiation led to the idea that movement-related dopamine release may act to trigger the initiation of movement9,11,20,26. However, more recent experiments that used optogenetic stimulation parameters that were calibrated to mimic physiological levels of SNc dopamine release did not trigger body movement or lead to an invigoration of ongoing action34,35. The calibrated stimulation was however, as we observe, able to act as a teaching signal to drive conditioned place-preference learning and reinforce the use of particular behavioural syllables. This suggests that, as we have seen in the TS and consistent with APE, movement-related dopamine activity in other striatal areas may also act as a value-free teaching signal rather than as a trigger or modulator of ongoing movement. Notably, learning rules that incorporate terms similar to APE have been used to describe how mice learn to adjust the gain of specific kinematic features28. Yttri & Dudman36 showed that the changes were dependent on the DMS, suggesting that there may be common learning rules and computations across the striatum. In line with this SNc movement-related dopamine activity also decreases with practice in a lever-pressing task9, as it should if the activity reflects an APE5. Furthermore, dopamine transients in the dorsolateral striatum (DLS) occur when sub-second action modules are initiated even in the absence of rewards or task structure35. The size of these transients is proportional to how predictable the upcoming action is with respect to the action that was just taken. This pattern of activity is what would be expected if these dopamine transients reflected an APE that compared the action that is taken (or soon to be taken) to the action that was predicted given the preceding behaviour. Together, the current data on movement-related dopamine activity are consistent with APE, suggesting that all movement-related dopamine activity throughout the striatum may reflect APEs that serve as value-free teaching signals. The caudate tail (CDt)—the primate homologue of the TS—is also innervated by an anatomically distinct population of dopamine neurons38 that are not activated by differences in reward outcome39. Rather, as we observe here, these neurons are preferentially active prior to contralateral orienting eye movement or when stimuli that predict contralateral orienting responses are presented39, as would be expected if they also encode an APE. The contralateral nature of the response in primates was interpreted as a unilaterally processed visual response39. Our results now show that, at least in our task, the TS dopamine responses in mice are related to contralateral movement initiation and not auditory cues. The response to movement rather than sensory cues is also consistent with TS-projecting dopaminergic neurons receiving input from the deeper motor-related layers of the superior colliculus rather than from the superficial sensory-related layers40,41. Whether APE signals are driven by the preparation or execution of action will need to be determined in future work and may help resolve the difference in explanation. While the interpretation of the signal was different, the proposed function of the CDt-projecting dopamine neurons is entirely consistent with what we show here, in that it serves to reinforce stable sensory–motor associations in the CDt in an outcome-independent manner42,43. This suggests that the TS is unique across species in that it exclusively receives a value-free movement-based teaching signal to update stable sensory–motor associations. Other laboratories have also shown that dopaminergic activity in the TS is not activated by reward44,45 but by threatening stimuli, such as an air puff or loud sounds. In addition, large TS dopaminergic transients are active when mice initiate avoidance behaviour, and these responses decrease as mice stop performing avoidance responses44. These observations led to the proposal that TS dopaminergic activity encodes a threat prediction error (TPE)44,46. In agreement with these results, we can also observe threat responses in the same region (and same mice) where we observe APE responses (Extended Data Fig. However, whereas optogenetic stimulation of dopamine release in the TS was able to reinforce state–action associations, we could not detect any aversive effect of this stimulation even when we used higher power than was needed to influence associations in our task. Although our TS dopamine stimulation was not able to induce avoidance behaviour, it does appear clear that TS dopamine release is critical for maintaining innate aversive associations40,44. How TPE and APE work in the TS to support their different functions at different timescales will need to be determined in the future. One option is that threat- and movement-related activity are encoded by separate dopaminergic subpopulations. In support of this, it has recently been reported that threat and acceleration-related dopamine responses are encoded in separate genetic subpopulations that both project to the TS, expressing Slc17a6 (also known as Vglut2) and Anxa1, respectively37,47,48,49. These populations may respectively encode TPE and APE. APE was first conceived as a teaching signal that could update habitual value-free state–action associations4,6. These models offer a parsimonious account for hallmarks of habitual behaviour, such as slower adaptation to contingency degradation or reversal and an insensitivity to outcome devaluation. This is because the stable associations in these models are driven by APE so are stored in an outcome-independent manner and are therefore insensitive to changes in outcome value. Our results show that dopamine neurons that project to the TS encode this type of movement-based prediction error. Interestingly the CDt is where habitual stimulus–action associations are stored29,39,50. Typically for instrumental learning the posterior DMS and the DLS have been shown to be critical for goal-directed and habitual learning respectively31,51,52,53,54,55. Recently the pDMS has been shown to receive reward-related dopaminergic teaching signals30,56. By contrast, movement-related dopamine signals, that could reflect APEs, are prominent in the DLS11,37. Taken together, we propose that RPE and APEs may be used as different teaching signals to drive goal-directed (value-based) and habitual (value-free) learning in different regions of the striatum. This does not seem like an advantageous strategy, but this system must exist for a reason. The first thing to note is that although APE is a value-free teaching signal, this does not mean that the associations that are formed in the TS will be insensitive to value. Rather, the ‘value-free' associations form through repetition of actions that were initially taken in pursuit of value. In this way, the associations that form in the TS can be thought of as storing the long-term ‘value' of an association even though they were never reinforced in value-based manner per se. Sustaining these associations in an outcome-independent manner makes them more stable to short-term fluctuations. Indeed, stimulus–action associations in the CDt system are incredibly stable—once formed they are retained for >100 days in the absence of reinforcement50. Another advantage of the value-free system is that from a normative perspective learning from terms such as APE will bias control systems to repeat stable associations, this can help in learning to act upon regular structure in the environment and forming a compressed default policy—that is, acting the same way in response to the same regular structure irrespective of situational differences57,58,59. Overall, the value-free system could ensure that consistent associations are stored in a more stable and efficient manner that enables them to be acted upon quickly and reliably. Another idea is that stimulus–action associations in the value-free system form a prior for Bayesian inference of action that encodes the policy that has worked in the past6,58,59. During inference this prior would be useful in biasing decisions when there is a high degree of uncertainty regarding the value of the action computed in the value-based controller. Finally, recent theoretical work has also predicted that if dopamine neurons encoded an APE, then it would allow the basal ganglia to implement off-policy learning algorithms5. Future work will be important in establishing the exact nature of the interaction between the value-based and value-free basal ganglia systems and to elucidate the contribution of APEs to habits, priors and off-policy learning. Male and female adult mice, aged between 2 and 7 months old, from the following mouse lines were used: C57BL/6 J wild-type (Charles River), Drd1-Cre (Gensat: EY262), Adora2a-cre (Gensat: KG139), Slc6a3-cre (JAX: 006660), Ai14 (tdTomato, JAX: 007914), Ai35 (Arch-GFP, JAX: 012735) and Ai32 (channelrhodopsin-2/EYFP, JAX: 024109). Mice were housed in HVC cages with free access to chow and water on a 12 h:12 h inverted light:dark cycle and tested during the dark phase. Mice used in the COT task were water deprived. Mice had access to water during each training session, and otherwise 1 ml water per mouse was administered. Water was supplemented as needed if the weight of the mouse was below 85%. All experiments were performed in accordance with the UK Home Office regulations Animal (Scientific Procedures) Act 1986 and the Animal Welfare and Ethical Review Body (AWERB). All viruses were made in house except for pAAV5-CAG-dLight1.1 (Addgene, 111067), which was used for the photometry recordings. dLight1.1 expression was induced by injecting 30–100 nl of pAAV5-CAG-dLight1.1 in the TS and 90 nl in the VS. Chronic lesions of the TS were achieved by injecting a 1:1 mix of AAV2/1-hSyn-Cre at 1014 viral genomes (vg) per ml and AAV2/5-EF1a-DIO-taCasp3-T2A-TEVp at 1014 vg ml−1. The mix was diluted five times in saline buffer prior to injection. For all chronic lesion experiments, 4 injections (30 nl each) were made in each hemisphere at 4 or 5 different depths to distribute the viruses as evenly as possible and to provide enough coverage. Mice were anaesthetized with isoflurane (0.5–2.5% in oxygen, 1 l/min), also used to maintain anaesthesia. Carpofen (5 mg kg−1) was administered subcutaneously before the procedure. Craniotomies were made using a 1-mm dental drill (Meisinger, HP310104001001004). This point usually lies slightly frontal to bregma and is more stereotypic than bregma itself with respect to the brain. Viral injections were delivered using pulled glass pipettes (Drummond, 3.5 inch) in an injection system coupled to a hydraulic micromanipulator (Drummond, Nanoject III) on a stereotaxic frame (Leica, Angle Two), at approximately 10 nl min−1. For optogenetic experiments, we used flat optical fibres of 200 μm diameter (Newdoon: FOC-C-200-1.25-0.37-7) and tapered optical fibres (Optogenix, Lambda Fiber Stubs) of 200 μm diameter with 1.5 mm active length, 4 mm implant length. For photometry experiments, flat fibres (Doric Lenses 0.57NA, 200 or 400 μm diameter, 4 to 7 mm long) were implanted vertically, 0.1 to 0.15 mm dorsal of the injection site of dLight1.1. Eight mice that were targeted for TS recordings were not used in any experiments as initial investigation revealed they had reward responses which have not been observed in the TS44,45. To confirm these fibres were outside the TS in six out of eight of these mice we performed serial two-photon microscopy and confirmed that in all cases the fibres were located outside of the TS. For chronic d-AP5 cannulation experiments, we used 26-gauge guide cannulas (Plastics One, C200GS-5), that were cut to 5 mm below the pedestal. We implanted the guide cannulas in the TS (coordinates: 1.82 posterior, 3.55 lateral, 3.0 depth) and then fit them with a dummy cannula with no projection (Plastics One, C200DCS). All implants were affixed using light-cured dental cement (3 m Espe Relyx U200), which was also used to attach a headbar. In mice that received only injections the wound was either sutured (6-0, Vicryl Rapide) or glued (Vetbond). For photometry experiments a subset of mice had fibres implanted in both the TS and the VS to compare how the signals in these regions developed throughout learning. For muscimol injections, bilateral cranial openings were performed over the DMS and the TS, and a headbar was positioned stereotactically. A landmark that aligned to bregma allowed for future injections in stereotactic positions. Closures of cranial openings were prevented by covering the exposed brain with Duragel (Cambridge Neurotech). Skull was covered with Kwiksil (World Precision Instruments), which was removed before every injection. Before each training session, mice were head-fixed while awake and 30 nl of either muscimol (Sigma-Aldrich) at 0.2 mg ml−1 or saline were injected for experimental and control sessions, respectively, co-injected with cholera toxin B (Cambridge bioscience) of different colours to trace injection sites during histology. After a 15-min period, mice started the training session. Starting from day 3, we administered either saline or 20 mM d-AP5 (Tocris, 0106) bilaterally in the TS using internal cannulas (Plastics One, C200IS) with 0.6 mm projection using a custom-built set up. The mice were awake and head-fixed during the infusion and received an infusion of 500 nl of 20 mM d-AP5 per hemisphere in 3 min, which was followed by a 3 min wait time before internal cannulas were retracted to be replaced by dummy cannulas again. After the mice completed ≥3,000 trials, we administered saline in both groups in the following session. For acute d-AP5 infusion experiments, we let these mice train without infusion till they reached expert-level performance. We then infused either saline and d-AP5 in different sessions using the same method described above and measured their performance in the perceptual auditory two-alternative forced choice task. For dopamine neuron ablations we followed the protocol in ref. In brief, we first injected intraperitoneally (10 mg kg−1), a solution made of 28.5 mg desipramine (Sigma-Aldrich, D3900-1G), 6.2 mg pargyline (Sigma-Aldrich, P8013-500MG), 10 ml water and NaOH to pH 7.4. Subsequently, mice underwent surgery, and a solution of 10 mg ml−1 6-hydroxydopamine (6-OHDA; Sigma-Aldrich, H116-5MG), dissolved in saline buffer was injected. Once prepared, we kept the solution on ice covered from light and injected it within 3 h. If the solution turned brown, it was discarded. For controls we only injected saline buffer. For all manipulation experiments cage mates were assigned to either control or manipulation groups prior to surgery, this was not done through an explicit randomization procedure. Power calculations were not used to define the size of the groups prior to experiments. Experimenters were not blinded to whether mice were in the control or manipulation groups. Mice received a lethal intraperitoneal injection of pentobarbital (0.1 ml per 10 g) inducing unconsciousness and death. Once unresponsive, mice were first perfused with phosphate-buffered saline (PBS) followed by 4% paraformaldehyde (PFA). The mice were decapitated, and the brains were extracted and fixed in 4% PFA for 24 to 48 h at 4 °C. Subsequently, brains were stored in PBS until further processing. Mouse training was carried out in a sound-isolated box containing the behavioural chamber, which consists of a closed arena of 19 cm × 15 cm with 3 ports located in a wall (1.95 cm from the floor, each port 3 cm from the next). To better separate cues and actions and to increase variability in movement times, the three ports were spaced further apart for photometry experiments. The ports are equipped with LEDs so they can be illuminated, and photovoltaic infrared sensors to detect poke events. Water can be delivered through each of the side ports. Ports were purchased from Sanworks or constructed in house to the same standards. Controlling chips were purchased from Sanworks. The chamber was illuminated exclusively with infrared light. Mice were monitored through standard webcams without the infrared filter. Bpod (Sanworks) was used to control the state machine, and the software was run using Matlab. The COT task is a self-initiated two-alternative-choice paradigm. For behavioural experiments the possibility to start a new trial is signalled by turning on the centre port LED. Mice start a trial by poking in the centre port and holding their position for 100 to 300 ms. The centre port LED is turned off after that required time has elapsed. In between 0 and 50 ms after poking in the centre port, sound is triggered, lasting for 500 ms. Following poking out from the centre port, mice were trained to poke in either of the two side ports. Two microlitres of water was delivered in only one of the two ports, contingent on the stimulus. During the first day of training, mice were habituated to the box and the poking sequence by doing a visual version of the task in which both side ports delivered water in every trial. Water amounts were decreased from 5 μl to 2 μl during the first 3 days of training. Therefore, this protocol engages progressively more as the mouse becomes more biased and disengages if the mouse becomes unbiased. This protocol was only active during the initial phases of learning as it uses the proportion of error trials for engagement and calibration. For the psychometric experiments, seven equally spaced ratios of high or low tones were used. Several variations of the COT task were used during photometry recordings. In the outcome value change experiment, unexpected large rewards (6 μl) and omissions were introduced randomly on both sides with a probability of 0.1 each. Mice experience multiple sessions of this protocol to get enough trials of each type. In the predicted value change experiment, the value of one port was changed 100 trials into a session. Mice did multiple sessions of this protocol and experienced both sides becoming the large reward size. The state change experiment was performed only once per mouse. 150 trials into a training session, the sound that had previously corresponded to the contralateral choice to the recording fibre was changed for a white noise stimulus. This new white noise stimulus overlapped with the original COT frequencies. This white noise was louder than the background white noise and was clearly detectable. To test whether the dopamine signals were dependent on an auditory stimulus being present, the sound indicating contralateral turns was omitted in expert mice (silence trials). The response to passive sounds was always tested at the end of a classic COT or a sound on return session. Mice were left inside the training box for an additional 10 min while all three ports were covered with custom made lids. During that time both high and low frequency sounds or white noise lasting 500 ms were presented at random time intervals (mean interval 5 s) and in a random sequence. Sounds consist of a stream of 30 ms pure tones presented at 100 Hz (each tone was introduced 10 ms after the previous one). One of two octaves (5 to 10 kHz, or 20–40 kHz) was selected as the target octave to indicate the side port where water was available on that trial. Each 30 ms tone was randomly drawn from 16 logarithmically spaced frequencies on each octave. For example, in a trial catalogued as 82% of high tones, for each 30 ms tone there are 82% chances of playing a tone from the 20–40 kHz octave, and 18% (100% − 82%) chances of playing a tone from the 5–10 kHz octave. These probabilities are independently computed, so two tones from different octaves can sound simultaneously. Water was delivered into the two side ports for correct choices using a solenoid valve that was carefully calibrated. As previous studies have suggested that the neurons in the TS are responsive to valve click noises45, for the photometry experiments we placed valves outside the training boxes and muffled them using sound insulation foam. In addition, we played quiet white noise constantly inside the training box. We confirmed with a microphone that the valve clicks could not be detected with these precautions. The 465 nm and 405 nm LED amplitudes were modulated using a sinusoid of 211 and 531 Hz respectively. This enabled separation of signal from movement artefacts and bleaching as done by ref. Light was passed from the LED source through optical fibres with NA 0.57, through a commutator (Doric Lenses, FRJ 1 × 1 PT 0.15) to a patch cord (Doric Lenses, FC-ZF1.25 LAF). A mini cube and photodetector (Doric Lenses) were used to collect the signal. The signal was then passed to a NIDAQ (National Instruments) and recorded and analysed using custom Python scripts as described in ‘Statistical analysis'. Behaviour was controlled with a Bpod which sent TTL pulses to the NIDAQ at the start of each trial. Mice were filmed from above during training and recording sessions using a Basler acA640-750um USB 3.0 camera. Videos were acquired at 30 Hz and synchronized with the photometry acquisition using the NIDAQ. For ease of analysis, mice were always trained in the same box and cameras were never moved. For this experiment we used a 40 cm × 40 cm square arena, and experiments were conducted with ambient light. See ‘Optogenetic manipulations' for specifics about the triggered stimulations. Mice were allowed to explore the setup for 20 min, during which video footage and photometry recordings were acquired. For opto-inhibition of either the D1 or the D2 SPNs, in 15–25% of trials, randomly selected, a sustained pulse of green (532 nm) light was delivered after mice initiated the trial, always preceding the onset of sound delivery by at least 50 ms and lasting longer than the sound duration. Light intensity was calibrated to 12 mW at the fibre tip. For dopamine opto-excitation, during the COT task, the first 150 trials of each session were done without stimulation to get a baseline behaviour, and subsequently, stimulated trials were introduced with these parameters: blue (473 nm) light delivered starting at the time of poking and lasting 150 ms in 5 ms pulses at 33 Hz of 4 or 8 mW intensity measured at the fibre tip. For the state–outcome experiment, stimulation was delivered bilaterally on one of the two side ports (for the whole duration of the session) every time the mice chose that port (correct and incorrect trials), coupled with water during correct trials. No anti-bias protocol was employed during these experiments. To test for effects on movement initiation, we performed an experiment similar to the one in ref. We placed mice in an open field where mice received a blue (473 nm) light stimulation (5 ms pulses at 33 Hz during 500 ms of 4 or 8 mW intensity measured at the fibre tip) if they were immobile for at least one second. Trials in which light was not delivered were used as within-animal control. Mice were placed in an arena (50 cm × 20 cm × 28 cm, L × W × H) with a white opaque floor allowing reliable tracking of the dark coated mice. At one end, the arena contained a rectangular shelter (10 cm × 20 cm) made of red Perspex. IR LEDs provided diffuse illumination of the arena. Mice were allowed to explore the entire arena including the shelter for at least 7 min before any looming stimuli were triggered upon their next entry into the threat zone. Visual stimuli were generated using PsychToolBox and MATLAB. A single looming stimulus consisted of five high contrast expanding spots, which expanded linearly from 3°–50° over 0.2 s (235° s−1) and remained at maximum radius for 0.25 s. The inter-spot-interval was 0.4 s. Each mouse was presented with three looming stimuli in one session. All looming stimuli trials were manually triggered for these mice. Data acquisition was controlled using custom scripts in MATLAB or Python and a NIDAQ (National Instruments). Fluorophores were stimulated as described above. Videos were acquired at 30 frames per second using an IR sensitive camera (Basler acA460-750 um USB 3.0) positioned 70 cm away from the arena and 70 cm above its floor. Frame acquisition was triggered using a NIDAQ generated TTL that was also recorded and used for post-hoc synchronization. Real-time stimulus presentation onsets were determined post-hoc using a photodiode (Thorlabs APD430C) and the TTL trigger acquired at 10 kHz. We imaged the fixed brains using serial section63 two-photon64 microscopy. Our microscope was controlled by ScanImage Basic (Vidrio Technologies) using BakingTray, a custom software wrapper for setting up the imaging parameters (https://github.com/SainsburyWellcomeCentre/BakingTray, https://doi.org/10.5281/zenodo.3631609). The 3D coordinates of the injections and fibre placements were determined by aligning the brains to the Allen Reference Atlas–Mouse Brain (available from https://atlas.brain-map.org) using brainreg65 and visualized using custom functions and brainrender66. Brain slices were all stained following the same procedure: Blocking was performed in staining solution (PBS + 1% BSA + 0.5% Triton X-100) for 15 min. Primary antibodies (1:1,000 in staining solution) were incubated for 2–4 h at room temperature or overnight at 4 °C with rocking. Washes were performed for 15 min with a staining solution. Secondary antibodies (1:500 in staining solution) and DAPI were incubated for 2 h at room temperature while rocking. Slices were then washed in PBS and mounted using Mount Medium. Primary antibodies used were NeuN (abcam, ab104225), tyrosine hydroxylase (TH) (Sigma-Aldrich, AB152) and GFP (Aves labs, GFP-1020) (to reveal dLight-expressing cells). Fifteen to 20 slices covering the entire striatum at regular intervals were selected for NeuN staining. Slices were mounted in standard glass slides using standard mounting medium and subsequently imaged in the Slide Scanner (Zeiss) using a 20× objective. Individual slices were registered to the Allen Reference Atlas–Mouse Brain (https://atlas.brain-map.org) using ABBA (https://github.com/BIOP/ijp-imagetoatlas), and the NeuN channel was thresholded automatically, per slice, based on the intensity levels in the cortex. The coverage of NeuN staining in the striatum was determined for each slice, and the inverse was determined as lesioned area. Brains were sectioned and mounted as described for the chronic lesions but stained for TH to specifically label dopamine cell processes. Slices were imaged using the Axio Scan (Zeiss). One mouse was removed from the analysis owing to lack of ablation (except in Extended Data Fig. Sessions with less than 60 trials were omitted, the first 5 trials of each session were discarded, and trials in which the mouse was not engaged (defined as having an inter-trial interval longer than 3 times the median value of that session) were not considered for analysis. Together, this amounts to less than 2% of data discarded. Mice that did not learn the task (end performance less than 55%) were discarded from the analysis. This amounts to a total of three mice in the whole study. This was only used for visualization purposes. To calculate individual learning parameters, per mouse, we modelled the performance of every mouse using a modified Weibull function67,68: \({\rm{performance}}=50+a(1-{2}^{{(\frac{-{\rm{trials}}}{l})}^{s}})\). Parameters were fitted using the scipy package in Python (optimize.minimize function). Statistical differences between the groups for these parameters were calculated using the non-parametric test Kruskal–Wallis from scipy.stats.kruskal. To calculate the differences in performance at different times in learning, we first removed those trials in which mice were extremely biased towards one of the two ports. Bias was determined as described in ‘Behavioural procedures', and extremely biased trials were defined as those having a value larger than twice the standard deviation for the whole dataset. Two mice (one experimental and one control) were removed for the chronic lesions experiment as they did not learn the task at all, and we suspected that they were deaf. Additionally, the two last sessions of one control and one experimental mouse were removed as the performance dropped to chance. One experimental mouse was excluded from the dopamine cell ablation experiment as the lesion quantification showed no ablation (this mouse performance was comparable to controls). At each point in training, performance was defined as the performance of the past 100 trials. To generate a shuffled dataset, experimental labels (for example, lesion or control) were randomly assigned to each mouse, always maintaining the proportion of labels on the original dataset, and differences between groups were calculated the same way. Mixed ANOVA was used from the pingouin package. Each individual session contains a few hundred trials, seven trial types, and between 15 and 25% of stimulated trials. A session of 300 trials can have as few as 6 (300 × 1/7 × 0.15) stimulated trials for a particular type. This can generate a large variability when calculating the proportion of binary choices, as each individual trial will have a large influence (17% in our example). To assess the significance of the biases caused by the optogenetic manipulations, we generated, for every session, a baseline distribution (1,000 shuffles) of port choice proportions for every unstimulated trial type (proportion of high versus low tones), using the same number of trials that were stimulated for that trial type. The total bias for each session was defined as the average difference between opto-stimulated and unstimulated trials for all trial types. Only sessions with more than 150 trials in total were selected for analysis. For mice in which more than data for more than one session and stimulation type was available, we selected the one with the best performance, which is a good indicator of how well a mouse was doing on the task and we reasoned would offer the most stable control to compare the effect of the stimulation to. The statistical significance for each group was calculated using the non-parametric test Kruskal–Wallis from scipy.stats.kruskal, comparing the observed biased values of every session against a randomly-sampled counterpart, per session, using the variance described above. The raw data were demodulated offline using custom Python scripts to produce traces that corresponded to the signal (465 nm) and background (405 nm) channels61. In brief, the demodulated traces were denoised using a median filter and a low-pass Butterworth filter (10 Hz cut off). The resulting signal and background channel traces were high-pass filtered at 0.001 Hz to correct for photo-bleaching. To correct for motion artefacts, the background channel data were fitted to the signal channel data using a linear regression. Finally, dF/F was calculated by dividing this signal by the baseline fluorescence (the signal channel trace filtered using a low-pass filter with a cut off at 0.001 Hz). All traces were z-scored to allow better comparison across mice and sessions. Peak dopamine responses were calculated using the Python package PeakUtils or by taking the maximum value of the trace if no peak was found in a given window. As VS dopamine clearly dips to omitted rewards, instead of calculating peaks, the mean of the dopamine response within this window was used as an estimate of the response. 12, we built a linear regression model to predict the photometry signal at each time point from the behavioural events around this time. In this model, the predicted dLight response is calculated as the convolution of a time series a,b, etc., representing different behavioural events as series of 0 s and 1 s, where 1 s represent the occurrence of the behavioural event. This means that at time t, the predicted dLight response g(t) is given by the weighted sum of the different behavioural events shifted in time within a set window. The model can be expressed as follows: where \([{\tau }_{x}^{-},{\tau }_{x}^{+}]\) gives the shifted time window in which behavioural event x is allowed to influence the predicted photometry signal. When plotted as a time series, these regression weights form the estimated response profile for the dLight signal due to a given behavioural event. These regression coefficients were estimated using the linear regression function of the Python package scikit-learn. Behavioural events were selected from trials in which the mouse did not repeat events (for example, did not repeatedly poke their head in the centre port). The model was fitted for each mouse for each session as the signal in the TS and VS evolved over learning. The calculation of the percentage variance explained by the full model was performed per session per mouse and then averaged across sessions. To calculate the percentage variance explained by each regressor, the predicted dLight signal was recalculated without that particular regressor, inspired by ref. This can be seen for some of the VS recordings without the outcome regressor. This includes extended periods with no behavioural events as the task is self-paced. To account for this the percentage variance explained was recalculated solely on the portions of the photometry trace for which there were behavioural events, a process we refer to as ‘trimming'. This trimming was only done in Extended Data Fig. Additionally, we explored including return to centre movements as behavioural events for the kernel regression model. 4j,k and were considered for inclusion in the regression. Return event kernels had a window from 0.2 s prior to and 1.5 s after leaving the side port. This was chosen to be conservative so as to avoid crossover with the next trial/prior choice movement and to be comparable to choice movements within the task, which had a mean duration of 0.68 ± 0.53 s (ipsi) and 0.68 ± −0.61 s (contra), for which the kernel window also extended 1.5 s after leaving the port. For ease of visualization and alignment, only short return movements (≤1 s) were included for the averages in Extended Data Fig. We performed a linear regression (using statsmodels.api.OLS) predicting the size of the dopamine response (TS dopamine at time of choice, VS dopamine at time of cue) on correct contralateral trials from previous choices for the same stimulus. We included data from throughout training for this analysis. Separate regressions were performed for each ‘lag' (number of trials back value). The position of the mouse was tracked using DeepLabCut71 and variables such as speed, acceleration, angular velocity and angular acceleration were calculated using custom scripts in Python. As the videos for the freely moving experiment were taken at a slight angle, the coordinates were transformed into standard space using custom Python scripts. Videos taken during the COT task were taken from above so there was no need for such a transform. Speed and acceleration were calculated using the nose as a marker. For angular velocity and acceleration, a triangle was formed using the nose and the two ears and a line was drawn from nose to the line between the two ear markers. Turn angles were calculated using the cumulative angular velocity. A regression slope was fitted (using the function stats.linregress from the Python package SciPy) between the average quantile turn angle and average quantile dopamine response for each mouse for each session. Turns were required to last at least 0.06 s and were only included for analysis if no other turns occurred in the preceding or subsequent 0.5 s. The head angle at the times of turn onset was then used to define 0° turn angle and turn angles were determined using the cumulative angular velocity from this time point. To investigate whether changes in turn angle and speed could account for the decrease in TS dopamine response size with trial number seen in Fig. 3, for each mouse we built a linear regression model predicting dopamine response size for all trials from speed and turn angle. To calculate the relative contributions of speed, turn angle and trial number, we also built a regression model with all three predictors to estimate the respective variance explained. To quantify the bias for each session, we calculated the choice differences between the first 150 trials (pre-stimulation), and the subsequent trials, as long as there were more than 100 trials. We averaged the biases if several sessions existed for the same mouse on the same stimulation conditions, so that we had only one observation per condition. The overall threshold to consider that the mouse was moving was determined empirically, but the results presented were invariant across a large range of values tested. Instant speed was calculated as the average speed from movement initiation to 300 ms after. Average movement was calculated from stimulation to 5 s later. We used the t-test on two related samples (as above) to assess the significance of every parameter. Data analysis was inspired by ref. Dopamine responses were categorized as large or small based on whether they were larger than or smaller than the 65th percentile respectively. The average change in percentage of contralateral choices was calculated for large and small dopamine responses on the preceding trial for each level of sensory uncertainty on the current trial (which correspond to the different percentage mixtures of tone clouds described in ‘Behavioural procedures'). A logistic regression was performed (using statsmodels.api.Logit) to investigate the relationship between dopamine response size on the previous trial, perceptual uncertainty and the choice (repeat or switch) on the current trial. Mice were required to meet certain behavioural criteria in order for their data to be included: the slope of the psychometric curve at the Point of Subjective Equality to measure sensitivity and bias at the edges (the ‘easy' trials). Bias was calculated by taking the difference in percentage correct at the two easy trial types. Mice were required to have a slope ≥1 and a bias ≤0.09 to be included, acting as a measure how well the mice had learned the task. This ensured that all mice in the analysis had similar behaviour, with little bias and strong discrimination between stimuli. For this analysis, following Lak et al.72, only trials where mice made a correct choice were included. Note that, as the tail dopamine response is taken between movement onset and reward delivery, for the ambiguous trials (50% high and low) the mouse would have no information as to the outcome of their choice at this time point. Therefore, all trials of this type were included. For the VS mice, the dopamine reward responses were used so only correct trials could be used. For the TS, correct trials were used for all stimuli other than the ambiguous stimulus. For the ambiguous stimulus, both correct and incorrect trials were used. The TS dopamine responses are taken aligned to movement, prior to reward being delivered. We performed a linear regression (using statsmodels.api.OLS) between the current trial TS dopamine response at time of choice and the Fréchet distance73 between trajectories on the current and subsequent trial (which provides a measure of the similarities between the two trajectories that takes into account the location and ordering of the points along a curve, with a low value indicating high similarity between the two trajectories). For the experiment in which opto-inhibition was done at different points in learning, mice only did the simplest version of the task (stimuli were either 98% of high tones or 98% of low tones). The initial 20 trials of every session were discarded, and the baseline distributions of choice proportions and the session bias were calculated as indicated above. Because mice performed close to perfection in this version of the task, we only included in the analysis the choices to the port where biases were expected to happen, in agreement with our previous results (it is not possible to measure any positive contralateral bias in a trial in which the mouse chooses the contralateral port all the time in unstimulated trials). Mice were all implanted bilaterally, and the stimulated implant was alternated every day of training. We only included in the analysis those implant sites which resulted in at least one significantly biased session (P < 0.05, calculated as indicated above). This removed 4 implants from the analysis out of 17. The regression models were done for each individual implant, and the average slope for each genotype was calculated. The significance of the global results were assessed against random datasets, generated by shuffles of the performance values associated with every session. P values were calculated as the proportion of ‘random slopes' being larger (expecting negative and positive contralateral biases for D1-Arch and A2A-Arch, respectively). The RL model used an actor–critic framework with soft-max (inverse temperature = 5.0) policy selection. The model used a semi-Markov state representation, previously introduced by74 to capture the time-course variability of dopamine commonly seen in experimental tasks. In the semi-Markov formalism, the agent keeps track of its current state as well as a representation of ‘dwell time' within that state. Actor, critic and stimulus–action strengths are then updated only when a state transition takes place. This formalism has been previously used in prior work modelling dopamine signals in tasks with variable timing75,76 and it allows us to model the time course of dopamine in a more realistic manner than other previous studies, which have often assumed that each time point within a behavioural stage of the task is an independent state (see ref. The states consisted of Start, High tone, Low tone, Outcome, as well as an action state associated with each of the time of action for each sound and action pairing (in order to align to the time of action to match the data). Actions consisted of Left, Right, Centre and Idle to represent the movement to enter left, right and centre ports (and not taking an action). The behaviour is self-paced for the mice, with sometimes multiple seconds of variability in timing between trials. The critic had dimension 1 × n states. The RPE was scaled according to dwell time in the state to approximate temporal discounting74 as is necessary for the semi-Markov representation. RPE was calculated at each state transition k as \({\rho }_{k}\) was calculated by looking at average reward per time step in the last n (n = 500 in our model) state transitions: and the actor stimulus–action value function was updated in a similar manner: where α and β are both learning rates that were set to 0.005. The non-value dependent model calculated APE, \({\delta }_{{a}_{k}}\), at state transitions (see Extended Data Fig. APEs were rectified, resulting in only one component of the APE vector being non-zero, essentially providing a scalar update signal. This rectification also matched the data better as dips in TS dopamine were never observed at time of action. Stimulus–action pairing strengths were then updated using APE: where ε is the rate at which the stimulus–action association was formed, which was set to 0.01 for all simulations apart from the predicted value change experiment simulation, where it was set to 0.001 to mimic how a value-free controller should update slowly. For other simulations this was less relevant as we aimed to show the direction of change of APE and RPE signals rather than their relative rates of updating (Extended Data Fig. Every time the mouse revisited a state it had visited before, a counter \(I(s)\) was increased to keep track of how many times that state had been visited: Salience L was computed as a weighted combination of value and novelty of the state: In all simulations except for the simulation of the effect of previous choice on subsequent dopamine response (Fig. 3i,l it was set to 0.5, to better capture the more exploratory nature of the behaviour of the mouse and to ensure there were enough trials throughout learning with both ipsi- and contralateral previous choices. The movement model of dopamine comprised a vector with length equal to the number of actions available. When an action was taken, the corresponding element in the vector was set to 1. In all candidate models of dopamine photometry signals, learning rates and constants were not fitted to match the numbers of trials taken to learn the task seen in the experimental data, so trends should be interpreted as approximations of general patterns rather than models of the exact behaviour. We simulated a task with two states (the first element, s = 1, corresponds to low tone, the second element, s = 2, corresponds to high tone) and two actions (first element corresponding to right and the second corresponding to left, a = 1 or a = 2). We modelled two parallel systems, one for the value-based controller that is updated by RPE, this had two components, an actor and a critic. The second system had a value-free controller that was updated by APE. An additional noise term was added, \({A(s)}_{{\rm{noise}}}\) (drawn from a uniform distribution between 0 and 1): The model received a reward, r = 1, if the index of the action taken, a, corresponded to the action rewarded (here r = 1 if: s = 1 and a = 1 or s = 2 and a = 2). We then computed the RPE and APE signals. We then calculated the reward prediction error as To calculate APE and represent it as a scalar value we first binarized the instantaneous predicted action vector: \({A}_{{\rm{tail}}}^{{\rm{binary}}}\), such that the index corresponding to the maximum \({A}_{{\rm{tail}}}\) was 1 and 0 everywhere else. Then we computed the predicted action \({p}_{a}\) (initialized at 0) as a low-pass filter of this binary choice vector: The APE was then calculated as \({\delta }_{{\rm{APE}}}=a-{p}_{a}\) for the action taken. where β = 0.02 is a learning rate; where α = 0.04 is a learning rate. The \({W}_{{\rm{tail}}}\) and \({W}_{{\rm{actor}}}\) weights were bounded to be positive and were initialized to 1. The \({W}_{{\rm{critic}}}\) values were initialized as 0 and were also bounded to be positive. A trial lasted 1,000 timesteps (2,000 for the inactivations). For plotting the performance, we low-pass filtered the reward with a time constant of 10 a.u., starting from 0.5. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. In addition, Allen Brain Atlas connectivity (https://connectivity.brain-map.org/) data were used to demarcate the cortico-striatal projection patterns from AUDp (experiments 100149109, 116903230, 120491896 and 146858006) and S1(SSp) (experiments 112882565, 114290938 and 126908007). Source data are provided with this paper. The analysis code is available at https://doi.org/10.5281/zenodo.15119142. Thorndike, E. L. Animal Intelligence: Experimental Studies (MacMillan, 1911). Schultz, W., Dayan, P. & Montague, P. R. A neural substrate of prediction and reward. & Litwin-Kumar, A. Action-modulated midbrain dopamine activity arises from distributed control policies. Bogacz, R. Dopamine role in learning and action inference. & Uchida, N. Neuron-type-specific signals for reward and punishment in the ventral tegmental area. Specialized coding of sensory, motor and cognitive variables in VTA dopamine neurons. A., Tecuapetla, F., Paixao, V. & Costa, R. M. Dopamine neuron activity before action initiation gates and invigorates future movements. Representation of spontaneous movement by dopaminergic neurons is cell-type selective and disrupted in parkinsonism. Rapid signalling in distinct dopaminergic axons during locomotion and reward. Reward and choice encoding in terminals of midbrain dopamine neurons depends on striatal target. Guo, L., Walker, W. I., Ponvert, N. D., Penix, P. L. & Jaramillo, S. Stable representation of sounds in the posterior striatum during flexible auditory decisions. Patriarchi, T. et al. Ultrafast neuronal imaging of dopamine dynamics with designed genetically encoded sensors. Schmack, K., Bosc, M., Ott, T., Sturgill, J. F. & Kepecs, A. Striatal dopamine mediates hallucination-like perception in mice. Chen, A. P. F. et al. Nigrostriatal dopamine pathway regulates auditory discrimination behavior. Tsutsui-Kimura, I. et al. Dopamine in the tail of the striatum facilitates avoidance in threat-reward conflicts. Hunnicutt, B. J. et al. A comprehensive excitatory input map of the striatum reveals novel functional organization. & Costa, R. M. What, if, and when to move: basal ganglia circuits and self-paced action initiation. B. Striatal circuits for reward learning and decision-making. & Wickens, J. R. A cellular mechanism of reward-related learning. Coddington, L. T., Lindo, S. E. & Dudman, J. T. Mesolimbic dopamine adapts the rate of learning from action. Jeong, H. et al. Mesolimbic dopamine release conveys causal associations. Dynamic nigrostriatal dopamine biases action selection. Coordination of rapid cholinergic and dopaminergic signaling in striatum during spontaneous movement. & Dudman, J. T. Opponent and bidirectional control of movement velocity in the basal ganglia. Distinct basal ganglia circuits controlling behaviors guided by flexible and stable values. Reinhold, K. et al. Striatum supports fast learning but not memory recall. Graybiel, A. M. & Grafton, S. T. The striatum: where skills and habits meet. Xiong, Q., Znamenskiy, P. & Zador, A. M. Selective corticostriatal plasticity during acquisition of an auditory discrimination task. Ghosh, S. & Zador, A. M. corticostriatal plasticity established by initial learning persists after behavioral reversal. Coddington, L. T. & Dudman, J. T. Learning from action: reconsidering movement signaling in midbrain dopamine neuron activity. Markowitz, J. E. et al. Spontaneous behaviour is structured by reinforcement without explicit reward. & Dudman, J. T. A proposed circuit computation in basal ganglia: history-dependent gain. Unique functional responses differentially map onto genetic subtypes of dopamine neurons. Separate groups of dopamine neurons innervate caudate head and tail encoding flexible and stable value memories. & Hikosaka, O. Dopamine neurons encoding long-term memory of object value for habitual behavior. Menegas, W. et al. Dopamine neurons projecting to the posterior striatum form an anatomically distinct subclass. Multiple neuronal circuits for variable object-action choices based on short- and long-term memories. Parallel basal ganglia circuits for voluntary and automatic behaviour to reach rewards. Menegas, W., Akiti, K., Amo, R., Uchida, N. & Watabe-Uchida, M. Dopamine neurons projecting to the posterior striatum reinforce avoidance of threatening stimuli. La Manno, G. et al. Molecular diversity of midbrain development in mouse, human, and stem cells. Poulin, J. F. et al. Mapping projections of molecularly defined dopamine neuron subtypes using intersectional genetic approaches. Defining midbrain dopaminergic neuron diversity by single-cell gene expression profiling. Robust representation of stable object values in the oculomotor basal ganglia. Hart, G., Leung, B. K. & Balleine, B. W. Dorsal and ventral streams: the distinct role of striatal subregions in the acquisition and performance of goal-directed actions. Hart, G., Burton, T. J., Nolan, C. R. & Balleine, B. W. Striatal dopamine release tracks the relationship between actions and their consequences. Origin of perseveration in the trade-off between reward and complexity. Moskovitz, T., Miller, K. J., Sahani, M. & Botvinick, M. M. Understanding dual process cognition via the minimum description length principle. Saanum, T., Éltető, N., Dayan, P., Binz, M. & Schulz, E. Reinforcement learning with simple sequence priors. Robinson, J. E. et al. Optical dopamine monitoring with dLight1 reveals mesolimbic phenotypes in a mouse model of neurofibromatosis type 1. eLife 8, e48983 (2019). Lerner, T. N. et al. Intact-brain analyses reveal distinct information carried by SNc dopamine subcircuits. Threat history controls flexible escape behavior in mice. Mayerich, D., Abbott, L. & McCormick, B. Knife-edge scanning microscopy for imaging and reconstruction of three-dimensional anatomical structures of the mouse brain. Serial two-photon tomography for automated ex vivo mouse brain imaging. Tyson, A. L. et al. A deep learning algorithm for 3D cell detection in whole mouse brain image datasets. Claudi, F. et al. Visualizing anatomically registered data with brainrender. Hunger or thirst state uncertainty is resolved by outcome evaluation in medial prefrontal cortex to guide decision-making. Gallistel, C. R., Fairhurst, S. & Balsam, P. The learning curve: implications of a quantitative analysis. Akam, T. & Walton, M. E. pyPhotometry: open source Python based hardware and software for fiber photometry data acquisition. Mathis, A. et al. DeepLabCut: markerless pose estimation of user-defined body parts with deep learning. Lak, A. et al. Dopaminergic and prefrontal basis of learning from sensory confidence and reward value. Eiter, T. & Mannila, H. Computing Discrete Fréchet Distance. Technical Report CD-TR 94/64 (Christian Doppler Laboratory for Expert Systems, TU Vienna, 1994). Daw, N. D., Courville, A. C. & Touretzky, D. S. Representation and timing in theories of the dopamine system. & Uchida, N. The medial prefrontal cortex shapes dopamine reward prediction errors under state uncertainty. Namboodiri, V. M. K. How do real animals account for the passage of time during associative learning? The authors thank the Advanced Microscopy Facility and Viral Vector Core in the SWC for assistance with histology, imaging and custom production of viral vectors, especially J. Broni-Tabi, R. Campbell, G. Estrada Girona and M. Strom; F. Claudi for advice on analysis; M. Watabe-Uchida, T. Mrsic-Flogel, S. Hofer, T. Branco and A. McAskill for their comments on the manuscript; and all members of the Stephenson-Jones laboratory for their contributions to this project, particularly F. Marbach. This work was supported by an EMBO Long-Term Fellowship (ALTF 827-2018) (H.M.V. ), a Swedish Research Council International Postdoc Grant (2020-06365) (Y.J. ), the Sainsbury Wellcome Centre Core Grant from the Gatsby Charitable Foundation and Wellcome (219627/Z/19/Z) (M.S.-J. ), the Sainsbury Wellcome Centre PhD Programme (F.G.) and a European Research Council grant (starting no. 3k,l, 7a,g, 9g,i, 12b,e) and https://doi.org/10.5281/zenodo.3925901 (Extended Data Figs. These authors contributed equally: Francesca Greenstreet, Hernando Martinez Vergara, Yvonne Johansson Sainsbury Wellcome Centre for Neural Circuits and Behaviour, University College London, London, UK Francesca Greenstreet, Hernando Martinez Vergara, Yvonne Johansson, Sthitapranjya Pati, Laura Schwarz, Stephen C. Lenzi, Jesse P. Geerts, Matthew Wisdom, Alina Gubanova, Lars B. Rollik, Jasvin Kaur, Joseph Cohen, Emmett Thompson, Troy W. Margrie, Claudia Clopath & Marcus Stephenson-Jones Institut d′Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar wrote the software for data analysis. The authors declare no competing interests. Nature thanks Elliot Ludvig, Eric Yttri and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic of the task, left. Example spectrograms showing low (left plot) and high (right plot) “cloud of tone” stimuli. b, Coronal sections along the striatum indicating fiber placement positions (tip of the fiber). Note that fibers were inserted in both hemispheres and are mirrored here for illustration purposes. Primary auditory cortex (AUDp) projections (yellow) and primary somatosensory cortex (SSp) projections (blue) are shown in the other hemisphere. c, Horizontal (top) and side view (bottom) of the same histological data. For the horizontal section the recording location depths are collapsed onto a single horizontal atlas image for illustrative purposes. All error bars represent the standard deviation. a, Representative image of a lesioned brain illustrating the image analysis used to quantify the proportion of lesioned striatum. Slices were registered to the atlas and the area with remaining neurons (as stained by NeuN) was defined. c, Learning rate (performance vs. trial number) of individual mice in the lesion in TS and the control groups. Light blue and light gray traces are from mice that had an initial bias. d, Differences of means in performance between control and lesion groups. e, Learning rate for trials with the low tone stimulus (performance vs. trial number) of lesion TS and the control groups (shaded area indicates standard deviation). f, Same as F but for high tone stimulus trials (shaded area indicates standard deviation). a, Representative image showing the location of an infusion cannula implanted over the TS. b, Coronal sections along the striatum indicating cannula placement positions (center tip of the cannula). Note that cannulas were inserted in both hemispheres and are mirrored here for illustration purposes. Primary auditory cortex projections are also shown (yellow). c, Behavioral effect of acute D-AP5 infusion in expert mice (>5000 trials). d, Learning rate (performance vs. trial number) of D-AP5 and saline infusion groups (shaded area indicates standard deviation). e, Differences in performance between the groups. f, Same as D but showing the data from individual mice. g, Behavioral performance of mice in the last session of chronic D-AP5 infusion (brown), or when saline was infused in the first session after reaching 3000 trials, D-AP5 group (yellow) or the control group (gray). Primary auditory cortex projections are shown. i, Example TH-stained (dopamine axons) coronal slices at the level of the dorsal striatum (DS) for control and lesion mice. j, Correlation between the maximum performance achieved for each mouse and the lesion size (p = 0.049, two-sided Wald test). k, Time elapsed between center port poke and side port pokes, as medians for each animal, for the 6-OHDA and control groups (p = 0.31, Kruskal-Wallis test). l, Time elapsed between trials, as medians for each animal, for the 6-OHDA and control groups (p = 0.73, Kruskal-Wallis test). m, Differences of means in performance between control and lesion groups (6-OHDA n = 9 mice, Control n = 5 mice. n, Learning rate (performance vs. trial number) of individual mice in the 6-OHDA and the control groups (same mice as panel m). o, Learning rate for trials with the low tone stimulus (performance vs. trial number) of 6-OHDA and the control groups (same mice as panel m, shaded area indicates standard deviation). p, Same as O but for high tone stimulus trials. a, Coronal sections along the striatum indicating fiber placement positions (center tip of the fiber). Note that fibers were inserted in both hemispheres and are mirrored here for illustration purposes. Primary auditory cortex projections are shown in the other hemisphere. b, Horizontal (top) and side view (bottom) of the same histological data. For the horizontal section the recording location depths are collapsed onto a single horizontal atlas image for illustrative purposes. c, Average photometry traces from TS (blue, n=10mice) and VS (orange, n = 7 mice) aligned to cue early in training (first three recorded sessions). For TS mice it was comparable to VS mice (TS: min: 51.4%, max: 80.0%, avg: 64.5%; VS: min: 55.8%, max: 70.1%, avg: 63.3%). d, Same as C but aligned to contralateral choice. Movement initiation for choice (leaving the center port) occurs on average (0.19 s +/- 0.05 s) after cue onset. e, Same as C but aligned to reward delivery. f, Example of a TS (blue) and VS (orange) recording session aligned to cues predicting a contralateral choice. g, Comparison of rise time for dopamine response from onset of the cue in the VS (n = 7) and TS (n = 10) early in training (first three recorded sessions). Rise time onset is determined by the time taken for the dopamine trace to reach more than one standard deviation above a baseline period (1.5 s prior to cue onset) (p = 0.002 two-sided independent samples t-test), Cohen's d = 1.90. h, TS dopamine responses for contralateral and ipsilateral choices aligned to movement onset early in training (first three recorded sessions). Shown separately for mice for whom contra-choice corresponded to high frequencies (n = 6) or mice for whom the contra choice corresponded to the low frequencies (n = 4). i, Same as H but for VS recordings aligned to cue onset. Shown separately for mice for whom contra-choice corresponded to high frequencies (n = 2) or mice for whom the contra choice corresponded to the low frequencies (n = 5). j, Average photometry trace across the first 3 recorded sessions early in training, for an example mouse aligned to leaving to the side ports to return to the center port. k, Same as J but an average of all mice (n = 10). l, Average regression kernels across mice (n = 10) for the return to center behavioral events. m, Percentage explained variance for different kernel regression models of TS dopamine. Original kernel regression model with only choice (center port exit): median = 5.1, original model including the return to center (ipsi and contra) behavioral events median = 7.1, original model only allowing photometry data for which there are behavioral events to be included in the explained variance calculation “+ trimming” median = 8.5, model with return events and trimming median = 10.3. For one session in the original regression there was no video recording, so this is why the ‘original' explained variance is slightly different to that reported in the main figure, as this session was not included in this analysis. o, Same as N but aligned to reward. a, Change in percentage of completed trials (where animals made a left or right choice after leaving the center port) in trials with a normal stimulus (tone) or silence trials (p = 0.23, paired two-sided t-test, Cohen's d = 1.15). b, Average photometry traces of one mouse aligned to contralateral choice. d, Change in peak response to the contralateral choice if the associated cloud of tones is replaced with silence (p = 0.31, paired two-sided t-test, Cohen's d = 0.79). e, The number of tone-contralateral action pairings or silence-contralateral action pairings that the mice experienced prior to the recordings in B-D, (p = 0.07, paired two-sided t-test, Cohen's d = 2.10). f, Difference in average speed for tone and silence trials for TS mice (n = 3), (p = 0.28, paired t-test, Cohen's d = 0.84). g, Difference in average turn angle for tone and silence trials for TS mice (n = 3), (p = 0.75, paired two-sided t-test, Cohen's d = 0.21). h, Animals were allowed to move freely in a different arena to the training box whilst dLight signals were recorded from the TS. Inset: Head angle during a detected turn in the freely moving arena. Light blue line shows head orientation 0.5 s later. i, Example photometry response in the TS to contralateral and ipsilateral turn onsets in the freely moving arena. j, As in I but averaged across animals (n = 3 mice). k, Example traces from a TS recording session in the frequency discrimination task separated by size of the response. l, Average turn angle for these quartiles plotted against quartile midpoint (example session). n, Data from early in training (first three recorded sessions) is analyzed as shown in O and a regression slope fitted (TS: n = 18 (6 mice, 3 sessions), VS: n = 21 (7 mice, 3 sessions). The slopes of the regressions (averaged per mouse) were tested against zero (one sample two-sided t-test). The TS slopes were significantly greater than zero (p = 0.03, Cohen's d = 1.20), whereas the VS slopes were not (p = 0.87, Cohen's d = −0.06). o, Schematic of the task structure when sound indicating an upcoming contralateral trial was played as mice return to the center port in 51.66 +/− 0.04 % of trials. p, Average dopamine response aligned to sound played while mice returned to the center during early training (n = 6 mice). q, Same as p but aligned to contralateral choice. r, Average amplitude of dopamine response aligned to the sound and choice across mice (p = 0.0047, Cohen's d = 1.97). Size of the sound response is significantly smaller than zero (p = 0.008793, one-sample one-sided t-test against zero). s, Same as P but ipsi- and contralateral returns are plotted separately and returns without concomitant cue are also shown. t, Difference in response size of returns shown in S (circles, p = 0.2945, one way ANOVA, n = 6 mice) and response sizes of mice when the cue is novel during the first training session (crosses, p = 0.2815, Kruskal Wallis test, n = 3 mice). Passive sounds responses were tested in mice that were at an early stage of training on the CoT task (average performance 60.6 +/− 6.4 % in n = 5 mice). v, Average dopamine response in the TS during contralateral movement in the 2AC task (dark blue) and during passive sound presentation in subsequent exploration (pale blue) (n = 5 mice). w, Average amplitude of choice aligned response in the task and of passive sound response during exploration across mice (p = 0.0092, paired t-test, Cohen's d = 2.11). There is no significant response to the sound (p = 0.45, one-sample two-sided t-test against zero). x, Average dopamine response in the TS during contralateral movement in the 2AC task (dark blue) and during passive presentation of white noise in subsequent exploration (pale blue) (n = 3 mice). Mice had on average experienced 93 presentations +/- 3.30 of white noise as a task cue before these recordings, this is less than half than during the CoT state-change experiment (195 + /−35 trials). y, Average amplitude of choice aligned response in the task and of white noise response during exploration across mice (p = 0.02, paired t-test, Cohen's d = 3.87). There is no significant response to the sound (p = 0.84, one-sample two-sided t-test against zero). a, The model comprises an actor that learns stimulus-action values and guides action choices, a critic that learns a value function that is used to calculate RPE. The RPE signal is broadcast to the actor and critic to update their respective value functions. A value-free system learns to predict actions from those taken in the past and updates its prediction using the difference between its prediction and the action taken (APE). APE and RPE equations are written with respect to time (t), as is common, for illustrative purposes. For the model equations we use dwell time in the state (k) to approximate temporal discounting, see methods. d, Correlation between the average speed of an example mouse and the TS dopamine response for all trials in all sessions. e, Linear regression coefficients for speed and turn angle on single trial TS (n = 6 mice) dopamine responses for the first three sessions of training. Stats: one-sample two-sided t-test against zero, speed: (p = 0.448, Cohen's d = −0.34), turn angle: (p = 0.033, Cohen's d = 1.20). Filled circles represent significant correlations for individual mice. f, Turn angle of an example mouse over the course of training, binned per 40 trials. g, Average speed during a choice of an example mouse over the course of training, binned per 40 trials. h, Linear regression coefficients for the effect of trial number on speed or turn angle at a single trial level (n = 6 mice). Stats: one-sample two-sided t-test against zero, speed: (p = 0.154, Cohen's d = 0.68), turn angle: (p = 0.340, Cohen's d = −0.43). Filled circles represent significant correlations for individual mice. i, TS dopamine response, binned per 40 trials of an example mouse over the course of training (blue). A linear regression model was built using average speed and turn angle to predict the TS dopamine signal. j, The movement model used in panel I was subtracted per trial from the TS dopamine responses to give the remaining signal that was not explained by speed or turn angle (residuals in blue). A new linear regression model was built using log trial number to account for the remaining TS dopamine signal (purple). Both are shown binned per 40 trials. k, The correlation between the individual trial residual dopamine responses and log trial number for an example mouse. l, Regression coefficients for the effect of log trial number on the residual dopamine response (n = 6 mice) (filled circles show significant correlations for individual mice). One-sample two-sided t-test against 0, p = 0.003, Cohen's d = −2.14. m, Total model variance explained by each parameter in a model where speed, turn angle and trial number are used to predict the size of the TS (n = 6 mice) dopamine response throughout learning. n, Difference between TS dopamine response in the last 40 trials of a previous session and next 40 trials of a session (between sessions) and first 40 trials of a session and last 40 trials of the same session (within session) (n = 6, between sessions: p = 0.27 one-sample two-sided t-test against 0, two-sided t-test Cohen's d = 0.51, turn angle p = 0.05 one sample t-test against 0 two-sided t-test, Cohen's d = −1.07). o, Performance in the 50 trials before and after the state change (n = 13 mice, p = 1.98×10-4 paired two-sided t-test, Cohen's d = 1.46). p, Changes in turn angle before and after the state change (n = 13 mice, p = 0.85 two paired two-sided t-test, Cohen's d =−0.08). q, Changes in average speed before and after the state change (n = 13 mice, p = 0.02 paired two-sided t-test, Cohen's d = 1.42). r, Performance before and after state change at trial 150 (black dashed line) binned per 20 trials (n = 13). Green lines show mean, error bars represent sem, grey lines represent data from individual mice. t, Same as R but showing the bias towards ipsilateral choices following the state change. u, behavioral bias towards the large reward port before and after the change in value. The last 50 trials from each block are used for analysis with blocks being a minimum of 70 trials (n = 10 mice, p = 0.002 paired two-sided t-test, Cohen's d = 1.39). v, Percentage of trials where mice did not make a choice before and after the value change is introduced at trial 100 (black dashed line) binned per 20 trials (n = 10 mice). Green lines show mean, error bars represent sem, grey lines represent data from individual mice. w, Same as V but showing the change in performance. x, Same as V but showing the change in response time. a, Schematic of the outcome manipulation task design. b, Modelled responses for how APE and RPE signals would respond to changes in reward outcome. c, Example TS response to omissions, normal sized and large rewards. d, Group data (n = 6 mice) for TS dopamine response size to omission, normal and large reward (p = 0.84, p = 0.54, paired two-sided t-test, adjusted using Bonferroni correction), (Cohen's d: large > normal 0.40, normal > omission 0.57). e, Same as c but for a VS recording. f, Same as d but for VS recording (n = 7 mice, p = 3.16×10-6, p = 8.87×10-6, paired two-sided t-test, adjusted using Bonferroni correction), (Cohen's d: large > normal 7.01, normal > omission 5.89). g, Schematic of the predicted value manipulation task design. h, Model predictions for changes in predicted outcome value. j, Summary data showing the change in movement-aligned response in the TS for relative value changes (p = 0.26, paired two-sided t-test) (n = 5 mice), Cohen's d = 0.58. k, Same as i but for the VS response aligned to cue. l, Same as j but for the VS response aligned to cue (p = 2.41×10-4, paired two-sided t-test) (n = 5 mice), Cohen's d = 5.56. a, Model predictions for how the different models of dopamine respond in the state change experiment. b, Model predictions for how the different models of dopamine respond in the cue value change experiment. c, Salience model dopamine signal aligned to time of high cue, low cue, and reward. Size of response to cues shown for 100 agents over training. d, Novelty model dopamine signal aligned to time of high cue, low cue, and reward. Size of response to cues shown for 100 agents over training. e, Movement model dopamine aligned to time of contralateral choice, ipsilateral choice, and reward. Size of response to contralateral choice shown for 100 agents over training. a, Coronal sections along the striatum indicating fiber placement positions (center tip of the fiber). Note that fibers were inserted in both hemispheres and are mirrored here for illustration purposes. Primary auditory cortex projections are shown in the other hemisphere. b, Horizontal (top) and side view (bottom) of the same histological data. For the horizontal section the recording location depths are collapsed onto a single horizontal atlas image for illustrative purposes. c, Psychometric curves for all 8 mW sessions where dopamine release was optogenetically stimulated in the TS at the time of choice (center port). d, Scatterplot showing that the stimulation choice bias develops over the course of a session (8 mW, p = 0.035; 4 mW, p = 0.006, linear regression). e, Model simulation of the state-action optogenetic experiment. f, Quantification of the contralateral choice bias on stimulated trials when optogenetic stimulation in the TS was delivered on 15% of trials at the center port. session (4 mW, p = 0.67; 8 mW, p = 0.28, Kruskal-Wallis test); 4 mW: n = 9 mice, 18 hemispheres mice, 8 mW n = 7 mice, 13 hemispheres g, Experimental design: animals enter a central port (gray) to initiate trials and then choose between water and water + optogenetic stimulation of dopamine release in the TS by entering one of two side ports. h, Choice bias (n = 5 mice) for 4 Hz optogenetic stimulation (p = 0.99, t-test) or 20 Hz optogenetic stimulation (p = 0.49, t-test). Solid dots indicate mean ± SEM and transparent dots indicate single sessions from individual animals. i, Same as g but with a 50% probability of receiving a water reward at the choice ports. j, Same as h but with the probability of receiving a water reward at the choice ports reduced to 50%. a, Heatmaps showing the occupancy for different sides of an arena during a real-time place-preference experiment where each side of the arena is paired with optogenetic stimulation of dopamine release in the TS. c, Schematic showing the experimental closed-loop setup for detecting immobility and triggering dopamine release. d, Speed heatmap of trials sorted by movement onset for stimulated and control trials (n = 9 biologically independent animals). e, Speed histogram, same data as in n; error bars: SEM. f, g, h, i, Distribution, as means per mouse, of different movement parameters (see methods) (p:4 mW p = 0.21, 8 mW p = 0.42; q: 4 mW p = 0.67, 8 mW p = 0.78; r: 4 mW p = 0.87, 8 mW p = 0.80; s: 4 mW p = 0.93, 8 mW p = 0.50; paired two-sided t-test); same animals as in n. j, Regression coefficients for effect of TS dopamine response on the current trial on difference between turn angle on current and subsequent trial. Stats: one-sample t-test against zero, p = 0.07, Cohen's d = −0.94. b, Change of the model weights during learning, as means for 100 agents, for the Critic, Actor, and TS networks. Dotted arrow indicates the proposed, but as yet unverified, broadcast of APEs to the dorsal striatum. a, Schematic of the experimental approach for recording dopamine activity in TS. Black line illustrates how the looming spot radius changes over time. d, The dlight1.1 photometry traces for all three trials for each mouse (gray lines) and the average (blue line) in response to the five looming stimuli (top). e, Schematic of the frequency discrimination task. f, Average photometry response of the same mice as shown in (D) in the frequency discrimination task aligned to choice (n = 4 mice). g, Same as F aligned to reward. Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Greenstreet, F., Vergara, H.M., Johansson, Y. et al. Dopaminergic action prediction errors serve as a value-free teaching signal. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64770020/dna-last-neanderthal/'>Scientists Sequenced the DNA of the ‘Last Neanderthal'—and It Alters Human History</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:31:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Discover new clues about how our ancient relatives disappeared from time. “There is nothing like looking, if you want to find something,” says Thorin Oakenshield in J.R.R. He and his team had been working the area since 1998 to find remnants of humanity's prehistoric forbearers, and after 17 years, they certainly found something: a piece of a jaw belonging to a Neanderthal. “I began to find {remnants of the Neanderthal's jaw} in 2015,” Slimak told the New Statesman in 2022, “but each year we find one tooth, or one fragment of bone.” Slimak determined that this particular Neanderthal lived 42,000 years ago, towards the end of that species' time on this planet. “Thorin the Neanderthal is also an end of lineage. To confirm his suspicions about Thorin's age and attempt to glean more information about not just when but how this particular specimen lived, Slimak and his colleagues had Thorin's genome analyzed. Nearly a decade before ever finding Thorin, Slimak had already theorized that any Neanderthals who had resided in the Rhône Valley would have been different from those in the surrounding areas. His assessment, at that point, was based on the stone tools found at various sites, noting that those in the Rhône Valley didn't reflect the newer tool-making style found at other locations. “It turns out that what I proposed 20 years ago was predictive,” Slimak told the publication Live Science. “Everything must be rewritten about the greatest extinction in humanity and our understanding of this incredible process that will lead Homo sapiens to remain the only survival of humanity,” Slimak said in assessing what this discovery means. “How can we imagine populations that lived for 50 millennia in isolation while they are only two weeks' walk from each other? Michale Natale is a News Editor for the Hearst Enthusiast Group. As a writer and researcher, he has produced written and audio-visual content for more than fifteen years, spanning historical periods from the dawn of early man to the Golden Age of Hollywood. Archaeologists Find Third Ancient Fort on One Site</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/a64769980/similarities-cats-dogs/'>Cats and Dogs Are Weirdly Starting to Look Like Each Other, Thanks to Human Evolution</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:29:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>We may earn commission if you buy from a link. Whether you're a feline fanatic or with the canine crowd, maybe you've noticed your cat has an eerie resemblance to a popular purebred pug on social media, or that your dog is starting to look like a lot of Instagram cats. This phenomenon isn't a trick of the imagination. Some breeds of dogs and cats really are starting to look uncannily similar, and it's all because of the human aesthetic preferences that go into breeding them. After analyzing the shapes of about 2,800 skulls of cats, dogs and their wild brethren, evolutionary biologists Abby Drake (from Cornell University) and Jonathan Losos (from Washington University) found that there was more variation in the skulls of domesticated animals than wild ones because of artificial selection by breeders. They did a double take, however, after comparing the shapes of dog and cat skulls. “I don't think anyone would have expected that.” Regardless of whether they happen to be dog or cat lovers, humans tend to prefer flat faces and huge eyes, most likely because these features closely mirror infants of our own species and trigger parental instincts. Seeing these features automatically puts many of us into nurturing mode—so much so, in fact, that caring for pets is also sometimes referred to as alloparenting (which refers to parenting anything other than direct offspring). As precocial animals, they are walking, leaping, pawing at the carpet, and knocking over houseplants just weeks after they open their eyes. Human infants don't even begin crawling until about 7 months. The merging of dog and cat features is an example of convergent evolution, which happens when unrelated organisms (which may even live far apart for each other) evolve similar characteristics because of the environmental pressures they face. This has been going on since there was life on Earth—for example, when fossils from what appeared to be a saber-tooth cat emerged in Argentina, scientists examining the bones realized that the creature was actually a monstrous marsupial that had developed fangs to tear through leftover carcasses. Divergent evolution, which results in speciation, is the opposite. Canids (dogs) and felids (cats) diverged from each other 50 million years ago. There are still those of us (including this author) who love creatures that don't even look remotely human as our own. But, in general, people tend to melt at anything that looks human-ish. Problems with this arise when an animal's health is compromised for its looks. And pugs did not evolve these characteristics on their own—they used to be almost unrecognizable by today's standards, with longer faces and legs. Your neighbor's very expensive designer dog went and had puppies with a stray of indeterminate breed? Her work has appeared in Popular Mechanics, Ars Technica, SYFY WIRE, Space.com, Live Science, Den of Geek, Forbidden Futures and Collective Tales. She lurks right outside New York City with her parrot, Lestat. When not writing, she can be found drawing, playing the piano or shapeshifting. A Mutation Lets You Function on 4 Hours of Sleep</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08992-2'>Divergent DNA methylation dynamics in marsupial and eutherian embryos</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:19:07
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Based on seminal work in placental species (eutherians)1,2,3,4,5,6,7,8,9,10, a paradigm of mammalian development has emerged wherein the genome-wide erasure of parental DNA methylation is required for embryogenesis. Whether such DNA methylation reprogramming is, in fact, conserved in other mammals is unknown. Here, to resolve this point, we generated base-resolution DNA methylation maps in gametes, embryos and adult tissues of a marsupial, the opossum Monodelphis domestica, revealing variations from the eutherian-derived model. The difference in DNA methylation level between oocytes and sperm is less pronounced than that in eutherians. Furthermore, unlike the genome of eutherians, that of the opossum remains hypermethylated during the cleavage stages. In the blastocyst, DNA demethylation is transient and modest in the epiblast. However, it is sustained in the trophectoderm, suggesting an evolutionarily conserved function for DNA hypomethylation in the mammalian placenta. Furthermore, unlike that in eutherians, the inactive X chromosome becomes globally DNA hypomethylated during embryogenesis. We identify gamete differentially methylated regions that exhibit distinct fates in the embryo, with some transient, and others retained and that represent candidate imprinted loci. We also reveal a possible mechanism for imprinted X inactivation, through maternal DNA methylation of the Xist-like noncoding RNA RSX11. We conclude that the evolutionarily divergent eutherians and marsupials use DNA demethylation differently during embryogenesis. Marsupials diverged from eutherians 160 million years ago, and provide unique insight into mammalian embryology12. Relative to that of eutherians, formation of the placenta occurs late, and implantation is transient. By contrast, pups are born early and complete development outside the uterus. Like eutherians, marsupials undergo X chromosome inactivation (XCI). However, they lack the eutherian XCI-initiating noncoding RNA Xist. Instead, the alternative noncoding RNA, RSX, is implicated in marsupial XCI11. Furthermore, in eutherians, XCI in the soma is random, but in marsupials, it is imprinted, with the paternal X always chosen for silencing13. Epigenomic profiling has not been performed in preimplantation marsupial embryos. DNA methylation, which in eutherians is globally reprogrammed and linked to key events in embryo development, is of particular interest14. In eutherians, the paternal genome undergoes active demethylation after fertilization. Subsequently, the paternal and maternal genomes undergo passive demethylation, resulting in global hypomethylation in the early blastocyst1,2,3,4,5,6. Global erasure of DNA methylation is not observed in non-mammalian vertebrates15,16, and is thought to serve a mammal-specific function (for example, to permit early embryonic genome activation (EGA), erasure of paternal methylation and epimutations, formation of the trophectoderm, establishment of pluripotency or expression of transposons regulating embryo development)7,8,9,10. These embryonic milestones occur over a more protracted period in the marsupial, making it a useful alternative model to investigate the role of DNA demethylation in mammalian embryogenesis. Marsupials possess the core vertebrate DNA methylation machinery, including de novo methyltransferase genes (DNMT3A and DNMT3B), DNMT3L, TET1–3, UHRF1 and two maintenance methyltransferase genes (DNMT1A and DNMT1B)17. However, the status of DNA methylation in gametes and during preimplantation development has not been explored. To examine global methylation during marsupial embryogenesis, we generated low-input bisulfite sequencing (BS-seq) libraries from opossum sperm, oocytes and male and female preimplantation embryos, collected daily from embryonic day (E) 1.5 to E7.5, as well as triplicate adult somatic tissues representing the three germ layers (Fig. Our embryo series included the time points of EGA (E3.5), XCI (E3.5), blastocyst formation (E5.5) and lineage divergence to form the trophectoderm and epiblast (EPI; E6.5 and E7.5; Supplementary Table 1). Control BS-seq on mice recapitulated published results3, with sperm and brain exhibiting hypermethylation and a bimodal methylation pattern, and blastocysts showed global hypomethylation (mean methylation 76% in sperm, 71% in brain and 16% in blastocysts; Extended Data Fig. a, Light microscopy images of gametes, E1.5–E7.5 opossum embryos and brain. ED, embryonic disc; TE, trophectoderm. Sample numbers are provided in Supplementary Table 1. b, Histograms of DNA methylation distribution at CpG sites captured at ≥5× coverage (note different scales on y axes). c, Mean DNA methylation level across developmental time. d, Micrographs: pronuclear-stage embryos (29.5 h post coitum) immunostained for histone H3 (n = 7), 5mC (n = 8) or 5hmC (n = 3) and co-immunostained for H3K9me3. The two pronuclei (Pat, paternal; Mat, maternal) are of equivalent size (unlike in mice), but the paternal pronucleus is often marked by proximity to the sperm tail (arrowhead in first panel). Plots: quantification of the data. The more granular appearance of H3K9me3 in the middle and bottom panel is due to acid treatment of these samples, which is required for 5mC and 5hmC visualization. In our opossum experiments, we obtained between 47 million and 337 million reads per library, with mapping rates between 15.7% and 71% (Supplementary Table 1). Principal component analysis separated sperm from other samples, with further separation by time point along principal component 2 (Extended Data Fig. After in silico pooling per time point, we captured between 49% and 91% of CpGs at 1× coverage, and 4.2% and 83% of CpG sites at 5× coverage (Extended Data Fig. As genomic coverage did not reach saturation in lower-input time points, we examined the proportion of genomic regions captured in each sample, and found that they were broadly similar between time points (Extended Data Fig. We therefore interpreted the global methylation distributions of each time point as representative of the overall genome. Examination of the opossum methylation data revealed similarities to and differences from eutherians (Fig. 1b,c and Extended Data Fig. In opossums, like eutherians3,4,5,6, the sperm genome was hypermethylated relative to the oocyte. However, in opossums, the magnitude of the difference was smaller (mean methylation 77% in sperm and 65% in oocyte). As a result, the methylation level in oocytes was similar to that in somatic tissues (mean methylation 72%, 69% and 55% in brain, liver and spleen, respectively). Furthermore, following fertilization in the opossum, the genome remained relatively hypermethylated (mean methylation 62.5%). The E1.5 methylome was more similar to that of the oocyte, suggesting that some sites in the sperm are demethylated and reprogrammed to an oocyte-like state (Fig. 1b,c and Extended Data Fig. Nevertheless, the high levels of methylation in the early embryo demonstrated that the global loss of DNA methylation typical of eutherians did not occur in the opossum. To support this conclusion, we immunostained for 5-methylcytosine (5mC) and its derivative 5-hydroxymethylcytosine (5hmC) in opossum zygotes (Fig. In eutherians, demethylation of the paternal genome after fertilization results in lower 5mC and higher 5hmC levels in the paternal than in the maternal pronucleus1,2,18,19. We reproduced these findings in mice (Extended Data Fig. In the opossum, the paternal and maternal pronucleus contained equal levels of histone H3 but could be distinguished by histone 3 Lys9 tri-methylation (H3K9me3) staining, the level of which was lower in the paternal pronucleus (see Fig. In contrast to those of eutherians, 5mC and 5hmC levels in the opossum were similar between the paternal and maternal pronucleus. DNA methylation dynamics in the paternal pronucleus therefore differ between opossums and eutherians. We next tracked DNA methylation levels during cleavage and blastocyst formation. The hypermethylated state persisted up to and including E4.5, when the mean methylation was 64% (Fig. Differential methylation testing of CpG sites between paired samples revealed that more than 90% of captured sites did not undergo significant changes in methylation level (Extended Data Fig. The stability in methylation was observed at all specific genomic features, including genes, intergenic regions and transposons (Extended Data Figs. In the last case, we nevertheless observed expression of multiple transposable element families at EGA (Extended Data Fig. This may be because expression derives from individual subfamily loci for which we did not capture sufficient BS-seq coverage to reveal locus-specific patterns. Alternatively, expression may be triggered independently of DNA demethylation. As observed in eutherians20, promoters and CpG islands (CGIs) were largely hypomethylated across developmental time (Extended Data Fig. We conclude that, in contrast to the case in eutherians, in opposums, EGA and the cleavage divisions take place in the context of a hypermethylated genome. In the opossum, lineage divergence initiates at E6.5. To establish whether the decrease in methylation levels revealed by bulk analysis affected all cells or specific lineages, we performed BS-seq on isolated E7.5 embryonic disc (comprising EPI and primitive endoderm) and trophectoderm. This observation was consistent with the results of previous methylation-sensitive enzyme assays21. One explanation for this finding is that DNA methylation is retained in the EPI but reduced in the trophectoderm. An alternative is that methylation is reduced in both cell types and is then rapidly re-established in the EPI. To distinguish these possibilities, we performed single-cell multi-omics on embryos collected before lineage divergence (E5.5) and after lineage divergence (E6.5 and E7.5; Fig. As embryos transitioned from E5.5 to E6.5, a decrease in methylation was observed in both the EPI and the trophectoderm. Subsequently, at E7.5, the level of methylation increased in the EPI but continued to decrease in the trophectoderm. The level of methylation in the trophectoderm was similar to that described for eutherian post-implantation extraembryonic tissues22. Partially methylated domains, a characteristic of the eutherian trophectoderm, were also present in the opossum (Fig. Thus, relative to that in eutherians, DNA demethylation in opossums occurs late, is transient and modest in the EPI, and is sustained in the trophectoderm. Furthermore, a lower level of methylation and formation of partially methylated domains in the extraembryonic tissue is a conserved feature of therian mammals. a, Mean methylation level in blastocyst single cells before and after lineage segregation. b, Segmentation of the genome into contiguous regions with similar methylation levels (segments 1–3) according to embryonic disc (ED) and trophectoderm (TE) methylation patterns. Presence of long segments with an intermediate level of methylation in the trophectoderm is indicative of partially methylated domains in this tissue. c, Mean expression of DNA methylation machinery derived from previous RNA-seq data23. To investigate the mechanisms behind DNA methylation changes during blastocyst development, we examined the mRNA expression of DNA methylation enzymes in our published opossum embryo single-cell RNA sequencing (RNA-seq)23 data (Fig. 2c) and new multi-omics dataset (Extended Data Fig. Expression of the maintenance methyltransferases DNMT1A, DNMT1B and the DNMT1 binding partner UHRF1 was high during cleavage and then declined before blastocyst formation. Notably, between E6.5 and E7.5, the expression of these factors, and of the establishment methyltransferases DNMT3A and DNMT3B, increased in the EPI concomitant with increased DNA methylation. Conversely, mRNA expression of these enzymes fell in the trophectoderm, concomitant with reduced DNA methylation. The DNMT cofactor DNMT3L was not expressed. Expression of the ten-eleven translocation methylcytosine dioxygenase TET1 was observed; notably, however, levels of expression of TET2 and TET3, the latter of which influences methylation in eutherian embryos18,24,25,26,27,28, were very low. Our findings suggest that differences in DNMT enzyme expression could drive the distinct DNA methylation profiles in the EPI and trophectoderm. We next identified differentially methylated regions (DMRs) in opossum oocytes and sperm. We found 20,800 sperm-specific and 22,921 oocyte-specific DMRs. As in eutherians, oocyte DMRs were relatively enriched in intragenic regions and CGIs, whereas sperm DMRs were enriched in intergenic regions (Fig. To identify transient and long-term autosomal imprints, we searched for DMRs that exhibited intermediate levels of methylation (40–60%) at E3.5, and tracked their fate in E7.5 embryonic disc and trophectoderm, and adult brain, liver and spleen. We found 12,096 intermediately methylated regions at E3.5 (Fig. Around half of these were retained at E7.5, some in a trophectoderm- or embryonic disc-specific manner, and 85 persisted in all 3 adult tissues. As in eutherians, most retained DMRs were of maternal origin (Supplementary Table 2). In adult tissues, we identified 78 DMR-proximal genes, 3 of which are known to be imprinted29 (NKRFL2, ZFP68 and RWDD2A; example locus in Fig. 3c), and the remainder of which are imprinting candidates (Supplementary Table 2). a, Genomic distribution of gamete DMRs. b, Fate of gamete DMRs during embryogenesis. c, DNA methylation levels at individual CpG sites at a representative region of the imprinted NKRFL2 locus. d, DNA methylation profiles in eutherian and opossum oocytes at genes transcribed in oocytes (active), genes not transcribed in oocytes (inactive) and intergenic regions. Eutherian oocytes exhibit divergent DNA methylation patterns, with methylation restricted to the bodies of expressed genes in rodents and extending into non-transcribed regions in other species3,4,5,30. To understand the relationship between oocyte transcription and methylation in opossums, we integrated our methylation datasets with previously published RNA-seq data, and compared our findings with those in other eutherians23,31,32,33,34,35,36 (Fig. Transcribed genes were heavily methylated across the gene body in eutherians and opossum. Non-transcribed regions (inactive genes and intergenic regions) were methylated at low levels in mice and rats, moderately methylated in humans, and more highly methylated in cows, consistent with previous findings. Opossums exhibited an extreme scenario, with non-transcribed regions exhibiting a very high level of methylation. DNMT3L was not expressed in oocytes, indicating that, as in humans37, de novo DNA methylation in opossum oocytes is DNMT3L independent. We also found that non-CpG methylation is relatively enriched in opossum oocytes, as in eutherian oocytes, indicating that this feature is conserved in therians (Extended Data Fig. The active X and inactive X (Xi) exhibit distinct DNA methylation patterns in eutherians. At a chromosome-wide level, the Xi is slightly less methylated than its active counterpart, but the Xi exhibits hypermethylation at CGI promoters38,39,40,41,42. We confirmed these findings in adult mouse somatic tissues using a model exhibiting skewed XCI (Extended Data Fig. By contrast, in marsupials, the Xi is hypomethylated at CGIs and transcription start sites43,44,45,46,47,48. However, patterns of DNA methylation on the X chromosome have not been examined in preimplantation embryos. We found that in adult opossum tissues, the level of X-chromosome DNA methylation in female animals was approximately half of that in male animals, in agreement with findings in another marsupial, the koala48 (Fig. Allele-specific analysis shows that this was due to hypomethylation of the silenced paternal X chromosome (Extended Data Fig. Hypomethylation was observed at all genomic features on the Xi, except at XCI escapees45, where methylation levels were equivalent to those on the active allele (Extended Data Fig. To examine when Xi hypomethylation is established, we assessed X-chromosome DNA methylation levels in embryos (Fig. 4b), which we sexed by quantification of X- and Y-mapping reads (Extended Data Fig. In the sperm and oocyte, X-chromosome DNA methylation levels were equivalent, demonstrating that the Xi is not inherited from the sperm in a hypomethylated state. Loss of methylation on the Xi occurred gradually during the cleavage stages, with hypomethylation clear from the blastocyst stage (Fig. Methylation levels therefore differ between chromosomes, being retained on the active X and the autosomes, but lost on the Xi during embryo development. a, Methylation distribution of the autosomes and X chromosome in adult male and female brain represented as density plots showing the distribution of the data and the probability of a variable being a certain value. c, Methylation at the RSX locus in gametes, embryos and adult tissues. d, Quantitative PCR analysis in DNMT1-knockout (KO) immortalized male fibroblasts. e, RSX RNA fluorescence in situ hybridization in wild-type and DMNT1-KO day-8 immortalized male fibroblasts. f, Quantification of RSX clouds in RNA fluorescence in situ hybridization at 4 and 8 days after DNMT1 deletion. g, X/A gene expression ratios at 4 and 8 days after DNMT1 deletion. Each point represents the median X/A ratio per replicate. h, Schematic of DNA methylation status in eutherian (left) and opossum (right) embryos. The mouse is an important exception, with random XCI preceded by silencing of the paternal X imprinted in the preimplantation embryo51,52. Imprinted XCI in mice is independent of DNA methylation53, relying instead on H3K27me3-mediated silencing of the maternal Xist allele54. The contribution of DNA methylation to XCI imprinting in marsupials is unclear. Methylation of the maternal RSX promoter has been observed in fetal opossum tissue45, but whether this epigenetic mark is inherited from the gamete is unknown. We identified a DMR encompassing the RSX promoter that was highly methylated in the oocyte and hypomethylated in sperm (Fig. Intermediate methylation of this region in female embryos and adult tissues suggests that this pattern is retained, consistent with an instructive role in imprinted XCI. To assess its role in RSX regulation, we ablated DNA methylation in bulk immortalized male opossum fibroblasts using CRISPR-mediated deletion of DNMT1A and DNMT1B (hereafter DNMT1). Loss of DNMT1 reduced methylation genome wide, including at the RSX promoter (Extended Data Fig. It also caused ectopic expression of the normally silent RSX allele in male cells, as assayed by quantitative PCR (Fig. This ectopic expression was accompanied by formation of RSX clouds in male cells (Fig. Ectopic expression of RSX in male fibroblasts was associated with suppression of X-gene activity, with X genes over-represented among all downregulated genes (Extended Data Fig. 6c) and the X-to-autosome (X/A) ratio mildly but significantly decreased (Fig. Loss of DNMT1 also led to a decrease in the level of DNA methylation and an increased level of expression of multiple transposable element families (Extended Data Fig. We also examined expression of the autosomal H19 locus, which is imprinted in marsupials55 (H19 is not present in the opossum assembly and was therefore not recovered in our earlier imprinted screen). The level of H19 expression increased following DNMT1 deletion (Extended Data Fig. 6e), suggesting that DNA methylation may be a general mechanism regulating imprinting in opossums. Here we define the DNA methylation landscape of early development in a marsupial. We demonstrate that opossums do not undergo a eutherian-like global demethylation phase during cleavage (Fig. DNA demethylation is mild and transient in the EPI but sustained in the trophectoderm. On the basis of our findings, we suggest that in therian mammals, DNA demethylation may play an especially important role in trophectoderm formation. Differences in DNA methylation between the EPI and trophectoderm are presumably achieved through cell-autonomous mechanisms, because the marsupial blastocyst is unilaminar. We propose that this is regulated by differential expression of the DNA methylation machinery. DNA demethylation is also proposed to erase germline-acquired epimutations7. This model would predict that epimutations are more stably maintained in marsupials than in eutherians. We also identify new candidate marsupial imprinted genes that will be useful for investigating conservation and evolution of imprints in therians. In eutherians, imprints are maintained in embryos through the Krüppel-associated box zinc-finger proteins ZFP57 and ZFP445. Of the two, only ZFP445 is conserved in marsupials56 and may contribute to imprint maintenance in these mammals. We offer a mechanism for marsupial imprinted XCI, mediated by differential methylation of RSX. Once methods to epigenetically edit the marsupial genome become available, it would be possible to further test this mechanism by targeted ablation of DNA methylation at the RSX promoter. This DNA methylation-mediated mechanism of imprinted XCI differs to that in mice, which achieve imprinted XCI using H3K27me3. We previously identified XSR, an RSX antisense RNA expressed in oocytes and from the maternal X chromosome in embryos23. Although its contribution to imprinted XCI is not established, XSR may help establish RSX promoter methylation through a mechanism reminiscent of Tsix-mediated Xist promoter methylation23. Our work highlights the strength of the marsupial model to understand evolutionary epigenetics. Given that post-fertilization global DNA methylation erasure is absent in the opossum, and zebrafish15,16, we propose that this process evolved after the marsupial–eutherian split. In this context, it is noteworthy that Dppa3 (also known as Stella or Pgc7) evolved after the marsupial–eutherian divergence57. In eutherians, Dppa3 is important for preimplantation development58,59,60, and has been implicated in both passive demethylation and protection against demethylation57,61,62,63,64,65,66,67,68. Our current findings are consistent with the hypothesis that evolutionary acquisition of Dppa3 accompanied the development of oocyte and embryonic DNA demethylation9,57. Opossums and mice (C57BL/6J and Mus spretus) were maintained in the Francis Crick Institute Biological Research Facility in accordance with UK Animal Scientific Procedures Act 1986 regulations (project licence P8ECF28D9) and subject to Francis Crick Institute's internal ethical review. Additional opossums were housed at the University of Texas Rio Grande Valley under Institutional Animal Care and Use Committee protocol AUP-19-31. No randomization or blinding was performed and no statistical methods were used to predetermine sample size. Mice were housed in individually ventilated cages (GM500, Tecniplast) with a 12:12-h light/dark cycle, a temperature of 20–24 °C and humidity of 55% ± 10%. Adults (8 weeks–6 months) were housed in groups of 3–4 animals, with the different sexes housed separately. Mice had free access to water and food and were provided enrichment activities including rodent balls and nesting boxes. Matings for timed collection of embryos were conducted by placing female mice into the cage of male mice at approximately 17:00. Observation of a vaginal plug the following morning was taken to indicate that mating had occurred. Opossums were individually housed in Double Decker cages (GR1800, Tecniplast), with male and female animals housed in separate rooms except during mating periods. The temperature of the housing was maintained between 24 °C and 28 °C, and humidity was maintained between 55% and 75%, with a 14 h/10 h light/dark cycle. Opossums had free access to dried food and water, supplemented every second day with live mealworms, and weekly by fresh fruit. To induce oestrous before mating, adult male and female (≥6 months) mice were placed in single-storey rat cages immediately adjacent to each other for 2 days, and then swapped into each other's cages for an additional 2 days. Subsequently, pairs were placed into the same cage and kept together for 10 days, during which period animals were monitored by infrared CCTV camera for mating behaviour69. Mouse E0.5 (11 h post coitum (hpc)) and E3.5 (82–84 hpc) embryos were recovered by flushing the uteri with PBS (Gibco number 14190-094) from a blunt-ended needle under a Leica MC80 dissecting microscope, aspirated using a Stripper pipette with a 275-μm tip (MXL3-STR and MXL3-275, Cooper Surgical), and washed three times through drops of clean PBS. For immunofluorescence, zygotes were fixed in 4% PFA in PBS with 0.2% Triton X-100 for 60 min at room temperature and washed three times in 0.2% Triton X-100 in PBS with 0.01% polyvinyl alcohol (PVA; number P8136, Sigma). For sequencing of blastocysts, the zona pellucida was removed by incubation in acid Tyrode's solution (T1788, Sigma-Aldrich), and then the embryo was washed three times in PBS, snap-frozen on dry ice and stored at −80 °C until processing. Picking-buffer-only negative controls were collected in parallel, and processed through the library preparation procedure to verify the absence of contamination. In the case of oocyte collection, the female animal was mated to a vasectomized male animal, and oocytes were recovered from the uterus at 36 hpc. Embryos were aspirated using a Stripper pipette with a 600-µm tip (MXL3-600, Cooper Surgical), or a micropipette fitted with a 200-µl tip for large E7.5 embryos, and washed three times in PBS. Samples were imaged using a Leica DMIL LED microscope with a 20× objective and then further processed for sequencing or immunofluorescence. The shell coat was perforated with 1-μm dissecting needles (10130-20, FST) and incubated in 5 mg ml−1 protease in PBS (P8811-100MG, Sigma) at 32 °C for between 2 and 7 min. The sample was transferred into fresh PBS, and any remaining mucoid coat was removed by disaggregation with dissecting needles. For sequencing, the oocytes and E0.5–7.5 embryos were washed through three drops of fresh PBS, dispensed to 0.2 µl tubes, and snap-frozen on dry ice and stored at −80 °C until further processing. We collected multiple single embryos for each time point, and in addition collected pooled litters of oocyte, E2.5 and E3.5 samples (Supplementary Table 1). For E7.5 embryonic disc and trophectoderm samples, after removal of the shell coat, the embryonic disc was isolated from the trophectodermT by dissection with 1-μm needles, washed through three drops of PBS, and snap-frozen as above. For single-cell sequencing, embryos were washed through three drops of fresh PBS with BSA, and incubated in TrypLE (12604013, ThermoFisher) diluted to 0.5× in PBS for 2–8 min at 35 °C. Following TrypLE incubation, embryos were moved to PBS with BSA and disaggregated to single cells by repeated pipetting through a narrow-bore pipette, before nucleosome, methylome and transcriptome (NMT)-sequencing processing. Picking-buffer controls were included as described above. For immunofluorescence, E0.5 zygotes were fixed and washed as above for mouse embryos. For collection of sperm from mouse and opossum adult male animals, epididymides were dissected from the testes and rinsed in PBS. Cauda epididymides were transferred to 2 ml Bigger–Whitten–Whittingham buffer (opossums) or TYH buffer (mice). Several small incisions were made, followed by incubation at 37 °C for 30 min to facilitate sperm swim out. The swim-out was diluted approximately 1:10 in PBS, and visualized on a dissecting microscope using a dark field. Individual sperm were aspirated using a Stripper pipette with a 100-µl tip (MXL3-100, Cooper Surgical) and washed three times in PBS before collection in 5 µl RLT Plus (1053393, Qiagen) containing 2% β-mercaptoethanol. For collection of pools of sperm, multiple sperm cells were picked in one pipette and processed together through washing, collection and freezing as for single sperm cells. Picking-buffer controls were included as described above. For collection of genomic DNA from adult mice, and genomic DNA and RNA samples from adult opossums, brain, liver and spleen tissues were dissected into ≈10-mm pieces, snap-frozen in liquid nitrogen, and stored at −80 °C. To facilitate allele-specific analyses, a cross was set up with parent animals from the LL2 opossum stock, and tissue samples were collected from the parents for whole-genome sequencing (WGS) and genomic variant identification, and from three male and three female littermates for BS-seq and RNA-seq. Mouse tissue samples were collected from a model exhibiting skewed XCI through a targeted Xist deletion70 (that is, C57BL/6J Xisttm1Jae × Mus spretus F1 hybrid cross). We collected three heterozygous female offspring in which XCI is completely skewed, and three male littermates with Xist deleted. Frozen tissue was pulverized using a pestle and mortar pre-cooled on dry ice, and used for genomic DNA extraction with the PureLink Genomic DNA Mini kit (K18290-02, Invitrogen) or RNA extraction using the Ambion RNAqueous-Micro Kit (AM1931, ThermoFisher). Samples were then processed for WGS, BS-seq or RNA-seq library preparation. Library preparation was performed with oligonucleotides compatible with the NEBNext library preparation kit (E7535S) following an established method71 and described briefly here. For gamete and embryo samples, samples were lysed in 2.5 µl RLT Plus and processed following the low-input library method71. For brain, liver, spleen and fibroblast samples, 6 ng of genomic DNA was used to prepare libraries following the bulk method71. Samples were spiked with 6 fg of Lambda DNA (D152A, Promega) and bisulfite-converted using the EZ Methylation Kit (D5020, Zymo). Bisulfite-converted DNA was purified using the PureLink PCR Purification Column Kit (K310050, Invitrogen) with an additional treatment with M-desulfonation buffer (EZ Methylation Kit, Zymo). Samples were eluted into a mixture containing 0.4 µM Preamp primer (5′-CTACACGACGCTCTTCCGATCTNNNNNN-3′) and submitted to one (brain, liver and spleen samples) or five (gamete and embryo samples) rounds of pre-amplification with Klenow exo− (M0212M, NEB). Unused oligonucleotides were degraded by incubation with exonuclease I (M0293L, NEB), and samples were purified with Ampure XP beads (A63881, Beckman Coulter). Samples were resuspended in a mixture containing 0.4 µM Adaptor 2 Oligo for NEB indices (5′-CAGACGTGTGCTCTTCCGATCTNNNNNN-3′) and adaptor-tagged by incubation with Klenow exo−. Samples were purified using Ampure XP beads and resuspended in a mixture containing 0.2 μM NEBNext Universal Adaptor and 0.2 μM NEBNext Index Adaptor (E7535S, NEB), and library amplification was performed using KAPI Hifi HotStart polymerase (KK2502, KAPA Biosystems). Varying numbers of PCR cycles were performed depending on input amount (opossum oocytes and E1.5–E5.5 embryos: 19 cycles; mouse E3.5 embryos and opossum E6.5 and E7.5 embryos: 10–14 cycles; sperm: 10–18 cycles; brain, liver and spleen: 10 cycles). Library sequencing was carried out by the Francis Crick Institute Advanced Sequencing Facility (ASF). Gamete and embryo libraries were sequenced (100-base-pair (bp) paired end) on an Illumina HiSeq 4000, yielding between 47 million and 337 million reads per library. Brain, liver and spleen libraries were sequenced (150-bp paired end) on an Illumina HiSeq 4000, yielding between 198 million and 363 million reads per library. Purified RNA was submitted to the Francis Crick Institute ASF for preparation of cDNA using the SMART-Seq v4 Ultra Low Input RNA Kit (634894, Takara), followed by library preparation using the Nextera XT DNA Library Preparation Kit (FC-131-1096, Illumina). Sequencing (100-bp paired end) was performed on an Illumina HiSeq 4000, generating between 54 million and 156 million reads per library. Disaggregated single cells were deposited into individual wells of a 96-well plate, and NMT-seq libraries were prepared as previously described71. In brief, cells were incubated with M.CviPI for 15 min, followed by separation of mRNA and genomic DNA using Oligo-dT beads, and subsequent preparation of RNA-seq and BS-seq libraries. Sequencing (100-bp paired end) was performed on an Illumina HiSeq 4000. Primary opossum fibroblasts were derived from a newborn male animal and immortalized using SV40-tag virus infection. Single-cell clonal selection was performed to identify an euploid cell line. Opossum fibroblasts were maintained in DMEM (Gibco) supplemented with 20% fetal bovine serum, 1% GlutaMax (Gibco), 1% sodium pyruvate (Gibco) and 1% penicillin–streptomycin (Gibco, 10,000 U ml−1) and were routinely tested and found to be negative for mycoplasma. Single-guide RNAs (sgRNAs) targeting all opossum DNMT1 paralogues (DNMT1A, DNMT1B and DNMT1Ψ) were designed using the online tool CRISPRdirect (https://crispr.dbcls.jp/): DNMT1 gRNA 1: 5′-TCTGAAGGCTTTCATCAAGC-3′; DNMT1 gRNA 2: 5′-CATTGTGGGCCATTGAAATG-3′. sgRNAs were annealed and ligated into the targeting plasmid px333-puro. The plasmid px333-puro was obtained after cloning a puromycin-resistance cassette isolated from px459v2 (gift from F. Zhang, Addgene number 62988) into the px333 vector (gift from A. Ventura, Addgene number 64073)72. Immortalized opossum fibroblasts were seeded onto gelatin-coated wells of 6-well plates. The following day, fibroblasts were transfected using PEI MAX (49553-93-7). A 2 µg quantity of plasmid (with sgRNAs or empty plasmid for negative control) was added to 200 µl of Opti-MEM (Gibco) and 8 µl of PEI MAX (1 mg ml−1). One day after transfection, puromycin (2.5 µg ml−1) was added for 48 h to select successfully transfected cells. Control and DNMT1-knockout cells were fixed for RNA fluorescence in situ hybridization (RNA FISH) or frozen down for quantitative PCR with reverse transcription (qRT–PCR) at specific time points. Cells were washed in cold PBS and treated with ice-cold permeabilizing solution (0.5% Triton X-100, 2 mM vanadyl ribonucleoside complex in PBS) for 10 min. After fixation using ice-cold 4% PFA in PBS for 10 min, cells were rinsed in ice-cold PBS twice, dehydrated through ice-cold 70%, 80%, 95% and 100% ethanol for 3 min each, and air-dried. BAC VM-18-303M7 (CHORI) was used for RSX RNA FISH. BAC DNA was labelled using Nick Translation Kit (Abbott) with fluorescent nucleotides (spectrum orange-dUTP; 02N33-050, Abott), and cells were hybridized with a denatured mix of probes along with 1 µg salmon sperm DNA in hybridization buffer (50% formamide, 10% dextran sulfate, 1 mg ml−1 PVP, 0.05% Triton X-100, 0.5 mg ml−1 BSA, 1 mM vanadyl ribonucleoside complex in 2× SSC) at 37 °C overnight in a humid chamber. Stringency washes were performed on a hot plate, three times for 5 min in 50% formamide in 1× SSC (pH 7.2–7.4) preheated to 45 °C, and three times for 5 min in 2× SSC (pH 7–7.2) preheated to 45 °C. Cells were mounted in antifade containing DAPI (Vector) with a coverslip and stored at −20 °C. RNA from control and DNMT1-knockout male opossum fibroblasts across three different time points and three replicates was purified using RNAqueous-Micro Total RNA Isolation kit (Invitrogen, AM1931). Purified RNA was retrotranscribed using the Maxima First Strand cDNA synthesis kit (Thermo Scientific, K1641). The following primers were used to assess gene expression in RT–PCR assays using PowerUp SYBR Green (Applied Biosystems, A25780): RSX (5′-AGAAGGGACCCCAAGACAC-3′, 5′-TGGGTCACTTCCACTTCCTC-3′); DNMT1 (5′-GACGCAGTAACACTGGAGCA-3′, 5′-ATCCCATTCCAACCTTCCAT-3′); H19 (5′-TCCAGCAGCAGTCAGTGAAC-3′, 5′-TCATCCATCCATGAGCAGAG-3′); ABCD4 (5′-ATCGATAATCCGGACCAGCG-3′, 5′-ATGATCAGCTTGCTGGCCAT-3′), GAPDH (5′-TAAATGGGGAGATGCTGGAG-3′, 5′-ATGCCGAAGTTGTCGTGAA-3′). Libraries from control and DNMT1-knockout RNA samples from day 4 and 8 were prepared with the NEBNext Ultra II Directional PolyA mRNA kit according to the manufacturer's instructions. Libraries were sequenced on the Illumina NovaSeq 6000 system (paired end, 100-bp read length). Raw RNA-seq reads were processed using the RNA-seq nf-core pipeline (v3.2); star_rsem was used to generate raw reads counts. Genes expressed at very low levels were filtered out by applying a rowSums filter of ≥5 to the raw counts table. Raw counts were normalized using the DESeq() function, specifying ~genotype_day in the design formula. log2[fold change] and adjusted P values between DNMT1 knockout and control were calculated using the lfcShrink() function in DESeq2, specifying type = ‘ashr', analysing day 4 and day 8 separately. X/A ratios were calculated using the median expression of X and autosomal genes in each sample after filtering out genes expressed at low levels (transcripts per million (TPM) of <1). X/A ratios between control and DNMT1-knockout samples were compared by Student's t-test. Repetitive element expression was analysed as above for opossum embryos. Samples of 3 μg of genomic DNA prepared from ear snips from one male and one female opossum from the LL2 stock were submitted to the Francis Crick Institute ASF for library construction (KAPA HyperPlus). Libraries were sequenced (150-bp paired end) on a HiSeq 4000, producing 256,834,288 reads (99.23% mapped) for the female animal and 122,371,579 reads (99.27% mapped) for the male animal. Data were used for identification of genomic variants, described below. We modified the MonDom5 reference genome to include a gap-filling long-read sequence of the RSX locus73. We prepared modified gene, repeat and CGI annotation files with corrected coordinates on the gap-filled X chromosome. We also included annotation for both opossum DNMT1 paralogues17 in our modified gene annotation file. Opossum embryo RNA-seq data23 and DNMT1-knockout RNA-seq data were mapped to the modified MonDom5 reference genome and ASM229v1 reference genome, respectively, using HISAT2 (ref. 74) with the command hisat2 -3 0 -5 9 --fr --no-mixed --no-discordant, and read summarization at genes was performed using the Rsubread package75, excluding multi-mapping reads. Fragments per kilobase of transcript per million mapped read (FPKM) values were calculated using the scater package76. Read summarization at repetitive elements and calculation of counts per million were performed using Telescope77. Line plots showing mean and standard error of gene and repeat expression were generated using ggplot2. To generate a bigwig file per time point, the filtered methylKit csv files were converted from csv to bedGraph using awk and then converted to bigwig using the UCSC bedGraphToBigWig utility. Genomic coordinates of the transposable elements, specifically L1, MIR and ERV1, were obtained from the mondom5 RepeatMasker GTF file, selecting for those in the ‘forward' orientation. Using deepTools78, the computeMatrix function in scale-regions mode was used to calculate the methylation score, using the parameters --binSize 50 --averageTypeBins mean -a 1000 -b 1000. The plotProfile function was used to generate the profile plot for each transposable element, using the parameters --perGroup --yMin 0 --yMax 1. The methSeg function from the R package MethylKit79 was used to divide the genome into contiguous stretches of similar methylation level in E7.5 embryonic disc and trophectoderm samples. Parameters used were minSeg = 5, G = 1:3, join.neighbours = TRUE. Individual CpG sites with a minimum coverage of five reads in both samples were used for the analysis. The length and average methylation level of each segment were plotted using ggplot2. Single-cell RNA-seq data were aligned to the opossum ASM229v1 genome, using the nf-core rnaseq pipeline (3.2), using the parameters –aligner star_rsem –bam-csi-index. Raw counts were loaded into Seurat (4.3.0)80 for analysis in R (4.2.2). Each dataset was normalized using NormalizeData, and then datasets were integrated together using the functions SelectIntegrationFeatures, FindIntegrationAnchors and then IntegrateData. The integrated dataset was scaled used ScaleData. Principal component analysis was applied using RunPCA. Uniform manifold approximation and projection was estimated for the integrated dataset using RunUMAP. Module scores for EPI and trophectoderm were calculated using the AddModuleScore function, with the following genes used as markers for each cell lineage. Single-cell BS-seq data were trimmed using the TrimGalore!81 command trim_galore --clip_R1 6 --three_prime_clip_r1 6, mapped using Bismark82 with the command bismark --non_directional --un --ambiguous --multicore 2 and deduplicated using the command deduplicate_bismark. Methylation information was extracted with the command bismark_methylation_extractor --comprehensive --multicore 2 --bedGraph --CX --cytosine_report --nome-seq. Average CpG methylation was calculated per cell and plotted according to the cell lineage determined for the corresponding RNA-seq library (module score, above). Raw RNA-seq and BS-seq data were processed using the nf-core rnaseq (3.12) and methylseq (2.5.0) pipelines, respectively. Gene bodies and intergenic regions for each organism were identified using the GenomicFeatures83 R package. Following a methodology similar to that in ref. 34, genes were classified as active if their TPM was >5 and inactive if their TPM was ≤1, and Bismark CpG methylation calls were imported into R using the methylKit79 R package, then destranded, pooled by sample condition and filtered for CpG sites with a minimum coverage of three reads. The regionCounts function was used to calculate the methylation level across active genes, inactive genes and intergenic regions. These were visualized as violin plots used ggplot2 (ref. WGS data from LL2 parent opossums were used to identify genomic variants. WGS reads were trimmed using TrimGalore!81 with the command trim_galore --cores 4 -- paired --fastqc --gzip --retain_unpaired --clip_R1 10 --clip_R2 10 -- three_prime_clip_R1 5 --three_prime_clip_R2 5. Libraries were mapped to the MonDom5 reference genome using BWA-MEM85 with the command bwa mem -t 32 -M -R. Paired and unpaired mapped reads were then merged, sorted and indexed using SAMTools86. Variants were called using the GATK best practices pipeline87. For the base recalibration step, known variants were not available for opossums. Therefore, variants were initially called independently with three pipelines: BCFtools88, Varscan89 and GATK87. Variants identified by all three pipelines were considered high-confidence variants and were used for GATK base recalibration. Subsequently, variants were called for each opossum, and then combined, and genotypes were annotated to produce a variant call file. BEDtools maskfasta was used to create an N-masked version of the MonDom5 reference genome from the complete set of 25 million variants. Using a custom R script, variants were filtered to include only hemizygous and heterozygous single nucleotide polymorphisms (SNPs). For mice, genomic variant data (41,668,158 SNPs) were derived from the C57BL/6J and M. spretus genomes (Mouse Genomes Project91), and the program SNPsplit90 was used to generate an mm10 reference genome in which parental genomic variants were N-masked. BS-seq reads were trimmed using the TrimGalore!81 command trim_galore --clip_R1 6 --three_prime_clip_r1 6. Reads were mapped to the mm10 (mouse) or MonDom5 (opossum) reference genome using Bismark82 on single-end mode with the command bismark --non_directional --un – ambiguous. For adult libraries, reads were mapped to the N-masked genomes, and bam files were generated for all mapped reads as well as for allele-specific reads using SNPsplit90 with the command SNPsplit --bisulfite –conflicting. Library statistics (Supplementary Table 1) were extracted from TrimGalore! Bam files were deduplicated and methylation calls were extracted using Bismark. CpG methylation calls were imported into R92 using the package methylKit79, and principal component analysis was performed using all CpG methylation calls for each individual sample. Data were destranded and pooled by sample condition, and the number of CpG sites captured at different coverage thresholds was calculated. For subsequent analyses, data were filtered for CpG sites with a minimum coverage of five reads. Methylation distribution histograms, mean methylation plot and genomic coverage plots were generated using ggplot2 (ref. 84). This workflow was independently reproduced to ensure that the results were accurate and robust. DMRs between gametes were identified as follows: 100-bp non-overlapping tiles covered by a minimum of three reads in both oocyte and sperm samples and containing a minimum of one CpG in a tile were identified. The methylation level of each tile was compared between oocyte and sperm using the diffMeth function from the R package methylKit79, with the parameters difference = 80 and qvalue = 0.01. Gamete DMRs with putative transient or life-long retention of differential methylation were defined as tiles with intermediate methylation levels (40−60% methylated), in either E3.5 and E7.5 embryonic disc or trophectoderm (transient embryonic or trophectoderm DMRs), or in E3.5, E7.5 embryonic disc, and all three of brain, liver and spleen (life-long DMRs). The gene nearest to each life-long DMR was identified using the nearest function from the R package GenomicRanges83, and this list was manually checked for previously reported marsupial imprinted genes29,93. To retain maximum read coverage in low-input embryo samples, all libraries per time point were in silico-pooled for this analysis. The dataset therefore included male and female embryos, precluding analysis of sex differences or the X chromosome. CpG methylation calls were destranded, pooled by sample condition (sex and tissue) and filtered for minimum coverage of five reads in all conditions. For embryo samples, sex was inferred from the ratio of reads mapping to chromosome X and pseudo-Y (coding sequence of 19 known opossum Y-chromosome genes)94,95. Autosomes and X-chromosome allelic methylation distributions were represented as ridgeline plots using the R package ggridges96. Allele-specific CpG methylation calls were destranded, pooled by sample condition (sex, tissue and parental genome) and filtered for minimum coverage of five reads in all conditions. To avoid loss of X-linked sites in female samples due to low coverage of the X chromosome in male files, data import and filtering for X chromosomes was performed separately with the coverage parameters as for autosomes, but excluding paternal genome files from male samples. Ridgeline plots were generated using ggridges96 as above. To generate a list of genes expressed in each tissue, RNA-seq reads were trimmed using TrimGalore!81 with the command trim_galore --paired -- clip_R1 10 --clip_R2 10. Trimmed reads were aligned to the N-masked MonDom5 reference genome using HISAT2 (ref. 8) with the command hisat2 --no-softclip --no-mixed --no-discordant. Mapped files were converted to bam files and merged by sample using SAMtools86. Reads overlapping annotated genes were quantified using the command featureCounts from the R package Rsubread75, excluding multi-mapping reads. Files were merged by biological replicate, and FPKM values were calculated using the R package DESeq2 (ref. 97) using the robust median ratio method. Gene models annotated as pseudogenes were excluded, and a threshold of FPKM > 1 was imposed. Expressed genes were categorized as escaping or subject to XCI on the basis of published work45. Methylation level at genes was calculated using the methylKit functions regionCounts and percMethylation, and represented as violin plots for each category using ggplot2 (ref. After 3.5 M HCl treatment for 30 min at room temperature, embryos were washed three times in 0.2% TX–PBS–PVA, blocked for 1–4 h at room temperature in 3% BSA in 0.2% TX–PBS–PVA (0.22-µM-filtered, blocking solution) and incubated overnight at 4 °C in primary antibody in blocking solution. Embryos were washed three times in 0.2% TX–PBS–PVA and incubated in Alexa Fluor-conjugated secondary antibodies at 1:250 in blocking solution for 2 h at room temperature. Three further 0.2% TX–PBS–PVA washes were performed, and DNA was counterstained with 10 µg ml−1 propidium iodide (P4170, Sigma) for 1–2 h at room temperature. Samples were washed three times with PBS with 0.1% PVA and mounted in Vectashield (H-1000-10, Vector Laboratories). Samples were imaged using an LSM710 confocal microscope with 1-µm z-sections. Images were processed and fluorescence intensity was measured using Fiji98. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. BS-seq and RNA-seq data have been deposited at the Gene Expression Omnibus (GEO) under the accession number GSE206499. WGS data have been deposited at the Sequence Read Archive under the accession number PRJNA819000. Publicly available datasets analysed in this study are accessible via the Gene Expression Omnibus under the accession numbers GSE163620, GSE71434, GSE101571, GSE163620 and GSE71985; DDBJ under the accession numbers DRA006642 and DRA000570; and ArrayExpress under the accession number E-MTAB-7515. Source data are provided with this paper. Code used in preparation and analysis of data and the generation of figures is available via GitHub at https://github.com/bleeke/opossum-methylation. Mayer, W., Niveleau, A., Walter, J., Fundele, R. & Haaf, T. Demethylation of the zygotic paternal genome. Smith, Z. D. et al. A unique regulatory phase of DNA methylation in the early mammalian embryo. Smith, Z. D. et al. DNA methylation dynamics of the human preimplantation embryo. The DNA methylation landscape of human early embryos. Ivanova, E. et al. DNA methylation changes during preimplantation development reveal inter-species differences and reprogramming events at imprinted genes. Seisenberger, S. et al. Reprogramming DNA methylation in the mammalian life cycle: building and breaking epigenetic barriers. & Reik, W. Reprogramming the methylome: erasing memory and creating diversity. Greenberg, M. V. C. Get out and stay out: new insights into DNA methylation reprogramming in mammals. & Hore, T. A. DNA methylation in the vertebrate germline: balancing memory and erasure. Grant, J. et al. Rsx is a metatherian RNA with Xist-like properties in X-chromosome inactivation. Frankenberg, S. R., de Barros, F. R., Rossant, J. Late DNA replication in the paternally derived X chromosome of female kangaroos. Adalsteinsson, B. T. & Ferguson-Smith, A. C. Epigenetic control of the genome-lessons from genomic imprinting. & Cairns, B. R. Reprogramming the maternal zebrafish genome after fertilization to match the paternal methylation pattern. Sperm, but not oocyte, DNA methylome is inherited by zebrafish early embryos. Alvarez-Ponce, D., Torres-Sanchez, M., Feyertag, F., Kulkarni, A. & Nappi, T. Molecular evolution of DNMT1 in vertebrates: duplications in marsupials followed by positive selection. Amouroux, R. et al. De novo DNA methylation drives 5hmC accumulation in mouse zygotes. Dean, W., Santos, F. & Reik, W. Epigenetic reprogramming in early mammalian development and following somatic nuclear transfer. Lister, R. et al. Human DNA methylomes at base resolution show widespread epigenomic differences. Stevens, M. E. et al. DNA methylation in the developing marsupial embryo. Smith, Z. D. et al. Epigenetic restriction of extraembryonic lineages mirrors the somatic transition to cancer. Mahadevaiah, S. K., Sangrithi, M. N., Hirota, T. & Turner, J. M. A. A single-cell transcriptome atlas of marsupial embryogenesis and X inactivation. Wossidlo, M. et al. 5-Hydroxymethylcytosine in the mammalian zygote is linked with epigenetic reprogramming. The role of Tet3 DNA dioxygenase in epigenetic reprogramming by oocytes. Iqbal, K., Jin, S. G., Pfeifer, G. P. & Szabo, P. E. Reprogramming of the paternal genome upon fertilization involves genome-wide oxidation of 5-methylcytosine. Active and passive demethylation of male and female pronuclear DNA in the mammalian zygote. Shen, L. et al. Tet3 and DNA replication mediate demethylation of both the maternal and paternal genomes in mouse zygotes. Origin and evolution of marsupial-specific imprinting clusters through lineage-specific gene duplications and acquisition of promoter differential methylation. Deep sequencing and de novo assembly of the mouse oocyte transcriptome define the contribution of transcription to the DNA methylation landscape. Yu, B. et al. Genome-wide, single-cell DNA methylomics reveals increased non-CpG methylation during human oocyte maturation. Brind'Amour, J. et al. LTR retrotransposons transcribed in oocytes drive species-specific and heritable changes in DNA methylation. Mouse oocyte methylomes at base resolution reveal genome-wide accumulation of non-CpG methylation and role of DNA methyltransferases. Evolutionary epigenomic analyses in mammalian early embryos reveal species-specific innovations and conserved principles of imprinting. Zhang, B. et al. Allelic reprogramming of the histone modification H3K4me3 in early mammalian development. Wu, J. et al. Chromatin analysis in human early development reveals epigenetic transition during ZGA. Expression of mRNAs for DNA methyltransferases and methyl-CpG-binding proteins in the human female germ line, preimplantation embryos, and embryonic stem cells. Gendrel, A. V. et al. Smchd1-dependent and -independent pathways determine developmental dynamics of CpG island methylation on the inactive X chromosome. Weber, M. et al. Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells. Keown, C. L. et al. Allele-specific non-CG DNA methylation marks domains of active chromatin in female mouse brain. Dosage compensation and DNA methylation landscape of the X chromosome in mouse liver. & Chess, A. Gene body-specific methylation on the active X chromosome. Kaslow, D. C. & Migeon, B. R. DNA methylation stabilizes X chromosome inactivation in eutherians but not in marsupials: evidence for multistep maintenance of mammalian X dosage compensation. & Johnston, P. G. Methylation analysis of a marsupial X-linked CpG island by bisulfite genomic sequencing. Wang, X., Douglas, K. C., Vandeberg, J. L., Clark, A. G. & Samollow, P. B. Chromosome-wide profiling of X-chromosome inactivation and epigenetic states in fetal brain and placenta of the opossum, Monodelphis domestica. Rens, W., Wallduck, M. S., Lovell, F. L., Ferguson-Smith, M. A. & Ferguson-Smith, A. C. Epigenetic modifications on X chromosomes in marsupial and monotreme mammals and implications for evolution of dosage compensation. Waters, S. A. et al. Landscape of DNA methylation on the marsupial X. Mol. Singh, D. et al. Koala methylomes reveal divergent and conserved DNA methylation signatures of X chromosome regulation. Lyon, M. F. Gene action in the X-chromosome of the mouse (Mus musculus L.). Okamoto, I. et al. Eutherian mammals use diverse strategies to initiate X-chromosome inactivation during development. Okamoto, I., Otte, A. P., Allis, C. D., Reinberg, D. & Heard, E. Epigenetic dynamics of imprinted X inactivation during early mouse development. Mak, W. et al. Reactivation of the paternal X chromosome in early mouse embryos. Sado, T. et al. X inactivation in the mouse embryo deficient for Dnmt1: distinct effect of hypomethylation on imprinted and random X inactivation. Inoue, A., Jiang, L., Lu, F. & Zhang, Y. Genomic imprinting of Xist by maternal H3K27me3. Smits, G. et al. Conservation of the H19 noncoding RNA and H19-IGF2 imprinting mechanism in therians. Takahashi, N. et al. ZNF445 is a primary regulator of genomic imprinting. Recent evolution of a TET-controlled and DPPA3/STELLA-driven pathway of passive DNA demethylation in mammals. Payer, B. et al. Stella is a maternal effect gene required for normal early development in mice. Bortvin, A., Goodheart, M., Liao, M. & Page, D. C. Dppa3 / Pgc7 / stella is a maternal factor and is not required for germ cell specification in mice. Huang, Y. et al. Stella modulates transcriptional and endogenous retrovirus programs during maternal-to-zygotic transition. Funaki, S. et al. Inhibition of maintenance DNA methylation by Stella. Han, L., Ren, C., Zhang, J., Shu, W. & Wang, Q. Differential roles of Stella in the modulation of DNA methylation during oocyte and zygotic development. Du, W. et al. Stella protein facilitates DNA demethylation by disrupting the chromatin association of the RING finger-type E3 ubiquitin ligase UHRF1. Zhang, C. et al. Maternal factor Dppa3 activates 2C-like genes and depresses DNA methylation in mouse embryonic stem cells. Nakamura, T. et al. PGC7/Stella protects against DNA demethylation in early embryogenesis. Nakamura, T. et al. PGC7 binds histone H3K9me2 to protect against conversion of 5mC to 5hmC in early embryos. Effects of dppa3 on DNA methylation dynamics during primordial germ cell development in mice. Shin, S. W., Vogt, E. J., Jimenez-Movilla, M., Baibakov, B. & Dean, J. Cytoplasmic cleavage of DPPA3 is required for intracellular trafficking and cleavage-stage development in mice. Baggott, L. M., Davis-Butler, S. & Moore, H. D. Characterization of oestrus and timed collection of oocytes in the grey short-tailed opossum, Monodelphis domestica. Marahrens, Y., Panning, B., Dausman, J., Strauss, W. & Jaenisch, R. Xist-deficient mice are defective in dosage compensation but not spermatogenesis. Clark, S. J. et al. Genome-wide base-resolution mapping of DNA methylation in single cells using single-cell bisulfite sequencing (scBS-seq). Systematic identification of Y-chromosome gene functions in mouse spermatogenesis. Sprague, D. et al. Nonlinear sequence similarity between the Xist and Rsx long noncoding RNAs suggests shared functions of tandem repeat domains. Kim, D., Paggi, J. M., Park, C., Bennett, C. & Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Liao, Y., Smyth, G. K. & Shi, W. The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads. McCarthy, D. J., Campbell, K. R., Lun, A. T. & Wills, Q. F. Scater: pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R. Bioinformatics 33, 1179–1186 (2017). Bendall, M. L. et al. Telescope: characterization of the retrotranscriptome by accurate estimation of transposable element expression. Ramirez, F., Dundar, F., Diehl, S., Gruning, B. & Manke, T. deepTools: a flexible platform for exploring deep-sequencing data. Akalin, A. et al. methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles. Integrated analysis of multimodal single-cell data. A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (reduced representation bisufite-seq) libraries (2012). Krueger, F. & Andrews, S. R. Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Lawrence, M. et al. Software for computing and annotating genomic ranges. Wickham, H. ggplot2: Elegant graphics for data analysis (Springer-Verlag, 2016). Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. The Sequence Alignment/Map format and SAMtools. Van der Auwera, G. A. et al. From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Narasimhan, V. et al. BCFtools/RoH: a hidden Markov model approach for detecting autozygosity from next-generation sequencing data. Koboldt, D. C. et al. VarScan: variant detection in massively parallel sequencing of individual and pooled samples. Krueger, F. & Andrews, S. R. SNPsplit: allele-specific splitting of alignments between genomes with known SNP genotypes. Mouse genomic variation and its effect on phenotypes and gene regulation. R: A Language and Environment for Statistical Computing. http://www.R-project.org/ (R Foundation for Statistical Computing, 2021). Stringer, J. M., Pask, A. J., Shaw, G. & Renfree, M. B. Post-natal imprinting: evidence from marsupials. Bellott, D. W. et al. Mammalian Y chromosomes retain widely expressed dosage-sensitive regulators. Origins and functional evolution of Y chromosomes across mammals. Wilke, C. ggridges: Ridgeline Plots in ‘ggplot2'. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. We thank the Francis Crick Institute Biological Research Facility, Advanced Sequencing Facility, Bioinformatics and Biostatistics Facility and High-Performance Computing team, M. Sangrithi, N. Fogarty, S. Clark, W. Reik, S. Smallwood, G. Kelsey, P. Skoglund and K. Niakan for advice on the project, and D. Odom and members of the laboratory of J.M.A.T. for comments and discussion on the manuscript. Work in the laboratory of J.M.A.T. is supported by the European Research Council (CoG 647971) and the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC001193), the UK Medical Research Council (FC001193) and the Wellcome Trust (FC001193). Some of the work at the University of Texas Rio Grande Valley was conducted in facilities constructed with support from the NIH grant C06 RR020547. Research in the laboratory of R.J.O. is supported by grants from the Bernice Bibby Research Trust and The Paradifference Foundation, the Wellcome Trust through the research equipment grant (212917/Z/18/Z), the NIHR Biomedical Research Centre at Guy's and St Thomas' Foundation Trust, the MRC (MR/V038664/1) and the King's College London Innovation Fund. A.T. is supported by funding from the European Research Council under the European Union's Horizon 2020 research and innovation programme (grant agreement number 101098236 ERC-2022-AdG). For the purpose of open access, the author has applied a CC BY public copyright licence to any author accepted manuscript version arising from this submission. Open Access funding provided by The Francis Crick Institute. These authors contributed equally: Bryony J. Leeke, Wazeer Varsally Sex Chromosome Biology Laboratory, The Francis Crick Institute, London, UK Bryony J. Leeke, Wazeer Varsally, Sugako Ogushi, Jasmin Zohren, Sergio Menchero, Aurélien Courtois, Daniel M. Snell, Obah Ojarikre, Shantha K. Mahadevaiah & James M. A. Turner MRC Laboratory of Medical Sciences, London, UK Advanced Sequencing Facility, The Francis Crick Institute, London, UK INSERM U934, CNRS UMR3215, Institut Curie, PSL Research University, Paris, France Comparative Medicine, Novartis, Basel, Switzerland Department of Medical and Molecular Genetics, King's College London, London, UK Division of Human Genetics and South Texas Diabetes and Obesity Institute, School of Medicine, The University of Texas Rio Grande Valley, Brownsville, TX, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar are equal first authors, and S.O. performed gamete, embryo, single-cell and adult tissue collections, BS-seq, RNA-seq, WGS and computational analyses. performed adult tissue collections, WGS and computational analyses. performed embryo immunostaining and fibroblast RNA FISH. performed single-cell collections, DNMT1-knockout experiments and analyses. A.C. imaged and quantified RNA FISH experiments. made initial findings of the RSX DMR. Correspondence to Bryony J. Leeke or James M. A. Turner. The authors declare no competing interests. Nature thanks Déborah Bourc'his, Takashi Sado and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a. Histograms of methylation distribution at CpG sites captured at ≥ 5x coverage in mouse sperm (n = 2 libraries of ~100 sperm, in silico pooled), E3.5 embryos (n = 3, in silico pooled), and adult brain (n = 2, in silico pooled) (please note variable scales on y-axes). Principal component analysis of opossum methylation in all samples. c. Percentage of CpG sites captured at different coverage thresholds. d. Percentage of captured CpG sites for different chromosomes and genomic features. a. Histograms of methylation distribution at CpG sites captured at ≥ 5x coverage in opossum adult liver and spleen. b. Pronuclear stage mouse embryos immunostained for 5-methylcytosine (5mC; n = 51), or 5-hydroxymethylcytosine (5hmC; n = 6), each co-immunostained for H3K9me3. mat = maternal, pat = paternal, pb = polar body. c. Pairwise comparisons of percentage significantly differentially methylated sites (Fisher's exact test, q-value <= 0.01, methylation difference 0.25) across entire developmental time course. For each comparison, “higher” and “lower” refers to the status of the second sample listed on the x-axis relative to the first sample d. Histograms of DNA methylation distribution at CpG sites captured at ≥ 5x coverage for each genomic feature (please note variable scale on y-axes). a. Metaplots of methylation across L1, MIR, and ERV1 loci in opossum embryos. Expression of LINE, SINE and LTR repetitive elements in opossum embryos. Repetitive element subfamilies are labelled according to Repeatmasker nomenclature, including ‘?' Expression of methylation enzymes in EPI and TE using the multi-omics dataset. Levels of non-CpG methylation (CHH and CHG sites) in opossum gametes and embryos. a Allele-specific methylation distribution of the autosomes and X chromosome in adult mouse male and female brain and liver, represented as density plots showing the distribution of the data and the probability of a variable being a certain value. b. CGI methylation on the inactive and active X for data in a. c. Methylation distribution of the autosomes and X chromosome in adult opossum male and female liver. d. Allele-specific methylation analysis of the paternal and maternal alleles for data in c. e. Methylation at specific genomic features on the inactive and active X in adult opossum brain, liver and spleen. f. Gene body methylation on the inactive and active X for genes subject to or escaping XCI for female samples in e. g. Sexing of gamete and embryo samples via read mapping to the X and Y chromosome. a. Metaplot of 100,000 randomly selected 1000 nucleotide regions demonstrating global decrease in DNA methylation in DNMT1-deleted fibroblasts at day 4. b. Mosaic loss of DNA methylation at the RSX promoter post-DNMT1 deletion at day 4. c. Proportion of upregulated and downregulated genes by chromosome in DNMT1-deleted fibroblasts at day 8. d. Methylation and expression of repetitive elements following DNMT1-deletion. Above, metaplots showing decreased DNA methylation at L1, MIR and ERV1 repetitive elements 4 days after DNMT1-deletion. Below, line graphs showing transcriptional de-repression of L1, MIR, ERV1 and ERVK elements by day 8 after DNMT1-deletion. e. qPCR analysis of H19 in DNMT1 deletant fibroblasts. BS-seq library information including sample descriptions, numbers of samples per time point, and sequencing, mapping and coverage statistics. Tab 1 contains information for each individual sequencing library. Tab 2 contains information for in silico-pooled data (per sample and time point). Information about gamete DMRs identified as potential imprinted regions. Tab 1 contains information including genomic position, methylation level at different time points, and nearest annotated gene. Tab 2 contains information about the number of gamete DMRs retaining intermediate methylation levels in different time points, broken down by gamete of origin. Sources of publicly available sequencing data used in this study. Tab 1 contains information for BS-seq datasets. Tab 2 contains information for RNA-seq datasets Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Leeke, B.J., Varsally, W., Ogushi, S. et al. Divergent DNA methylation dynamics in marsupial and eutherian embryos. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-08912-4'>Chicago Archaeopteryx informs on the early evolution of the avian bauplan</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 15:10:50
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. (2025)Cite this article Metrics details Here we report on the nearly complete and uncrushed 14th specimen of Archaeopteryx. Exceptional preservation and preparation guided by micro-computed tomographic data make this one of the best exemplars of this iconic taxon, preserving important data regarding skeletal transformation and plumage evolution in relation to the acquisition of flight during early avian evolution. The ventrolaterally exposed skull reveals a palatal morphology intermediate between troodontids1 and crownward Cretaceous birds2,3. Modifications of the skull reflect the shift towards a less rigid cranial architecture in archaeopterygids from non-avian theropods. The complete vertebral column reveals paired proatlases and a tail longer than previously recognized. Skin traces on the right major digit of the hand suggest that the minor digit was free and mobile distally, contrary to previous interpretations4. The morphology of the foot pads indicates that they were adapted for non-raptorial terrestrial locomotion. Specialized inner secondary feathers called tertials5,6 are observed on both wings. Humeral tertials are absent in non-avian dinosaurs closely related to birds, suggesting that these feathers evolved for flight, creating a continuous aerodynamic surface. These new findings clarify the mosaic of traits present in Archaeopteryx, refine ecological predictions and elucidate the unique evolutionary history of the Archaeopterygidae, providing clues regarding the ancestral avian condition. This is a preview of subscription content, access via your institution Access Nature and 54 other Nature Portfolio journals Get Nature+, our best-value online-access subscription cancel any time Subscribe to this journal Receive 51 print issues and online access $199.00 per year only $3.90 per issue Buy this article Prices may be subject to local taxes which are calculated during checkout Raw CT data and segmented models relevant to this manuscript are available via Morphosource (Media ID 000702228; Media ID 000702231). Tsuihiji, T. et al. An exquisitely preserved troodontid theropod with new information on the palatal structure from the Upper Cretaceous of Mongolia. Wang, M., Stidham, T. A., Li, Z.-H., Xu, X. & Zhou, Z.-H. Cretaceous bird with dinosaur skull sheds light on avian cranial evolution. Hu, H. et al. Earliest evidence for fruit consumption and potential seed dispersal by birds. Elzanowski, A. in Mesozoic Birds: above the Heads of Dinosaurs (eds Chiappe, L. M. & Witmer, L. M.) 129–159 (Univ. California Press, 2002). Terrill, R. S. & Shultz, A. J. Feather function and the evolution of birds. Hickman, S. The trouble with tertials. Sereno, P. C. The evolution of dinosaurs. Foth, C., Tischlinger, H. & Rauhut, O. W. M. New specimen of Archaeopteryx provides insights into the evolution of pennaceous feathers. Voeten, D. F. A. E. et al. Wing bone geometry reveals active flight in Archaeopteryx. Wellnhofer, P. Archaeopteryx, English Edition: The Icon Of Evolution (Pfeil, 2009). Rauhut, O. W. M., Foth, C. & Tischlinger, H. The oldest Archaeopteryx (Theropoda: Avialiae): a new specimen from the Kimmeridgian/Tithonian boundary of Schamhaupten, Bavaria. A novel reconstruction of the skull of Archaeopteryx. Mayr, G., Pohl, B., Hartman, S. & Peters, D. S. The tenth skeletal specimen of Archaeopteryx. Yin, Y.-L., Pei, R. & Zhou, C.-F. Cranial morphology of Sinovenator changii (Theropoda: Troodontidae) on the new material from the Yixian Formation of western Liaoning, China. New information on the anatomy and relationships of Dromaeosaurus albertensis (Dinosaura: Theropoda). Hu, H. et al. Evolution of the vomer and its implications for cranial kinesis in Paraves. Wang, M., Stidham, T. A., O'Connor, J. K. & Zhou, Z.-H. Insight into the evolutionary assemblage of cranial kinesis from a Cretaceous bird. & Wellnhofer, P. Cranial morphology of Archaeopteryx: evidence from the seventh skeleton. Mayr, G., Pohl, B. & Peters, D. S. A well-preserved Archaeopteryx specimen with theropod features. Wang, M., Wang, X.-L., Zheng, X.-T. & Zhou, Z.-H. Cranial anatomy of Anchiornis huxleyi (Theropoda: Paraves) sheds new light on bird skull evolution. Wu, Y.-H., Chiappe, L. M., Bottjer, D. J., Nava, W. & Martinelli, A. G. Dental replacement in Mesozoic birds: evidence from newly discovered Brazilian enantiornithines. & Zhang, F. Two new ornithurine birds from the Early Cretaceous of western Liaoning, China. Kundrát, M., Nudds, J., Kear, B. P., Lü, J.-C. & Ahlberg, P. E. The first specimen of Archaeopteryx from the Upper Jurassic Mörnsheim Formation of Germany. Makovicky, P. J., Norell, M. A., Clark, J. M. & Rowe, T. Osteology and relationships of Byronosaurus jaffei (Theropoda: Troodontidae). Pei, R. et al. Osteology of a new Late Cretaceous troodontid specimen from Ukaa Tolgod, Ömnögovi Aimag, Mongolia. Pei, R. et al. Potential for powered flight neared by most close avialan relatives, but few crossed its thresholds. Godefroit, P. et al. A Jurassic avialan dinosaur from China resolves the early phylogenetic history of birds. & Chiappe, L. M. A revision of enantiornithine (Aves: Ornithothoraces) skull morphology. A. Morphology, phylogenetic taxonomy, and systematics of Ichthyornis and Apatornis (Avialae: Ornithurae). Sumida, S. S., Lombard, R. E. & Berman, D. S. The atlas-axis complex of the Late Paleozoic Diadectomorpha and basal amniotes: defining the primitive condition of the atlas-axis complex of amniotes. Norell, M. A. et al. A new dromaeosaurid theropod from Ukhaa Tolgod (Ömnögov, Mongolia). & Norell, M. A. Anatomy and systematics of the Confuciusornithidae (Theropoda: Aves) from the Late Mesozoic of northeastern China. Liu, D. et al. Cranial and dental morphology in a bohaiornithid enantiornithine with information on its tooth replacement pattern. Cretaceous Res. Ostrom, J. H. Osteology of Deinonychus antirrhopus, an unusual theropod from the Lower Cretaceous of Montana. Peabody Mus. Wollin, D. G. The os odontoideum. Separate odontoid process. Bone Joint Surg. Baumel, J. J., King, A. S., Breazile, J. E., Evans, H. E. & Vanden Berge, J. C. in Publ. 779 (Nuttall Ornithological Club, 1993). Turner, A. H., Pol, D. & Norell, M. A. Anatomy of Mahakala omnogovae (Theropoda: Dromaeosauridae), Tögrögiin Shiree, Mongolia. Botelho, J. F. et al. New developmental evidence clarifies the evolution of wrist bones in the dinosaur–bird transition. A. in New Perspectives on the Origin and Early Evolution of Birds (eds Gauthier, J. & Gall, L. F.) 211–235 (Peabody Museum of Natural History, 2001). Pittman, M. et al. Exceptional preservation and foot structure reveal ecological transitions and lifestyles of early theropod flyers. Lennerstedt, I. Pattern of pads and folds in the foot of European Oscines (Aves, Passeriformes). The life style of Archaeopteryx (Aves). VII International Symposium on Mesozoic Terrestrial Ecosystems Vol. 7 Asociación Paleontológica Argentina Publicación Especial (ed. 91–99 (Asociación Paleontológica Argentina, 2001). Yalden, D. W. What size was Archaeopteryx? Wang, M., O'Connor, J., Xu, X. & Zhou, Z.-H. A new Jurassic scansoriopterygid and the loss of membranous wings in theropod dinosaurs. Effects of wing damage and moult gaps on vertebrate flight performance. Jenni, L. & Winkler, R. The Biology of Moult in Birds (Bloomsbury, 2020). Ellis, D. H., Swengel, S. R., Archibald, G. W. & Kepler, C. B. A sociogram for the cranes of the world. Zheng, X.-T. et al. Hind wings in basal birds and the evolution of leg feathers. Zhang, F. & Zhou, Z. Leg feathers in an Early Cretaceous bird. O'Connor, J. in The Evolution of Feathers (eds Foth, C. & Rauhut, O. W. M.) 147–172 (Springer, 2020). Wang, X.-L. et al. Archaeorhynchus preserving significant soft tissue including probable fossilized lungs. Field, D. J. et al. Complete Ichthyornis skull illuminates mosaic assembly of the avian head. We thank T. Lumbsch, B. Lauer and R. Lauer for facilitating the specimen acquisition; A. Stroup for assisting with figures; M. Colbert, J. Maisano and D. Edey for scanning the main slab; C. Wang and X. Zhang for segmentation assistance; S. Selzer for aiding preparation; D. Drummond and J. Stierberger for photographing the specimen; and B. Marks for access to extant bird specimens. is supported by the National Natural Science Foundation of China (42225201). Negaunee Integrative Research Center, Field Museum of Natural History, Chicago, IL, USA Jingmai O'Connor, Alexander Clark, Pei-Chen Kuo, Akiko Shinya & Constance Van Beek Committee on Evolutionary Biology, University of Chicago, Chicago, IL, USA Alexander Clark School of Zoology, Faculty of Life Sciences, Tel Aviv University, Tel Aviv, Israel Steinhardt Museum of Natural History, Tel Aviv University, Tel Aviv, Israel Center for Functional Anatomy and Evolution, Johns Hopkins University School of Medicine, Baltimore, MD, USA Matteo Fabbri Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Chinese Academy of Sciences, Beijing, China Jing Lu, Min Wang & Han Hu You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar led the research and wrote the manuscript; H.H. led the segmentation; M.F., P.-C.K. assisted with segmentation; A.C., P.-C.K., M.F., H.H., Y.K. assisted with research and manuscript preparation; A.S. and C.V.B. prepared the specimen. Correspondence to Jingmai O'Connor or Han Hu. The authors declare no competing interests. Nature thanks Jessie Atterholt, Daniel Field, Johan Lindgren, Patrick O'Connor and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a-c, premaxillae in right lateral, ventral, and caudal view; d-f, vomer in dorsal, left lateral, and ventral view; g-i, right maxilla in lateral, medial and ventromedial view; j-l, right postorbital in medial, rostral, and medial view; m-p, left ectopterygoid in ventral, dorsal, lateral, and medial view; q-v, left pterygoid in ventral, dorsal, cranial, caudal, lateral, and medial view; w-aa, left palatine in rostral, dorsal, ventral, lateral, and medial view. Scale bar equals five millimeters. cf, caudal fork; cp, choanal process; fp, frontal process; jp, jugal process; lp, lateral process; mf, maxillary fenestra; mp, maxillary process; np, nasal process; pf, pterygoid flange; pr, promaxillary fenestra; pp, premaxillary process; pt, pterygoid process; qw, quadrate wing; rp, rostral process; sp, squamosal process. a, modified from Elzanowski (2001); b, modified from Mayr et al.13; c, current reconstruction based on new data from the Chicago Archaeopteryx (FMNH PA 830). The palatal elements are color coded: orange, vomer; green, palatine; dark orange, ectopterygoid; pink, jugal; light blue, pterygoid; dark blue, quadrate; yellow, basicranium. Elements, all figured in right lateral view, are to scale relative to each other but not between taxa. The proatlas is colored olive green; the atlas neurapophysis is pink; the atlas intercentrum is light green. Sinovenator is based on PMOL (Paleontology Museum of Liaoning) -AD00102; Tsaagan is based on IGM, Institute of Geology, Mongolian Academy of Sciences, 100/105; and Deinonychus is based on YPM, Yale Peabody Museum, 5210. Dashed lines indicate poor preservation. Fused atlas (only right neurapophysis could be reconstructed) in Archaeopteryx FMNH PA 830 in caudal (a) and cranial (b) view; right proatlas in lateral (c), medial (d), caudal (e), and cranial (f) view. Scale bar equals 2 mm. Anatomical abbreviations not listed in Fig. 1 caption: sl, semilunate carpal. Grey indicates damaged bone. The carpals were identified based on 2D observations; the smallest carpals (scapholunare and distal carpal 3) could not be reconstructed from the CT data. Extant arboreal taxa Hylocichla mustelina (a) and Cyanocitta cristata (b); terrestrial taxon Coturnix coturnix (c-d); raptorial taxon Buteo jamaicensis (e-f). Fossil taxa represented by (g-h) enantiornithine indet. HPG-15-1 (Xing et al., 2017); (i-j) Archaeopteryx FMNH PA 830; and (k) Microraptor STM5-109 (Pittman et al. 40). All extant specimens represent recently deceased individuals being prepared for the skeletonization at the Field Museum and thus lack catalog numbers. All extant bird photos by A. D. Clark. Scale bar equals five centimeters. a, scapulars and left humeral and ulnar tertials under UVABC; b, c, close up of the left tertials showing the preservation of barbs, rachises, and vane symmetry; d, open pennaceous body feathers preserved dorsal to the proximal caudal vertebrae and projecting off the left femur; e, close up of the open pennaceous feathers preserved in (d). Scale bar in (a) equals 2 cm, 5 mm in (b-c, e), and 1 cm in (d). Anatomical abbreviations not listed in Fig. 1 caption: ba, barbs; cv, wing coverts; op, open pennaceous body feathers; rh, rachis. In many modern birds, the tertial feathers (black arrows; red feathers, inset) are morphologically distinct. These feathers function as protection for the remiges from abrasion when the bird is not flying and are particularly developed in species that feed on the ground, such as (a) wagtails (Motacillidae; pictured, Eastern Yellow Wagtail Motacilla tschutschensis; photo by Y. Kiat), and (b) many shorebirds (Charadriiformes; pictured, Buff-breasted Sandpiper Calidris subruficollis; photo by I. Davies, ML110171881, adapted with permission from Cornell Lab of Ornithology | Macaulay Library). These feathers may also help close the gap between the secondary feathers and the bird's body, thereby improving aerodynamic performance. Additionally, the tertials may be used for display and visual communication, as observed among some cranes (Gruiformes), such as the elongated tertials in the (c) Blue Crane (Anthropoides paradiseus; photo by M. McCloy, ML136331631, adapted with permission from Cornell Lab of Ornithology | Macaulay Library), or the prominent white tertials contrasting with the dark wing and body plumage in the (d) Pale-winged Trumpeter (Psophia leucoptera; photo by T. Palliser, ML62921801, adapted with permission from Cornell Lab of Ornithology | Macaulay Library). The black and white pattern of the feathers is based on previous research that indicates the isolated wing covert was black and white (Carney et al., 2012). This Chicago specimen reveals the presence of the enlarged tertial tract. Supplementary Methods, Discussion, Fig. 1, Tables 1–3 and References. Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Reprints and permissions O'Connor, J., Clark, A., Kuo, PC. et al. Chicago Archaeopteryx informs on the early evolution of the avian bauplan. Published: 14 May 2025 DOI: https://doi.org/10.1038/s41586-025-08912-4 Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative © 2025 Springer Nature Limited Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/science-tells-us-the-u-s-is-heading-toward-a-dictatorship/'>The Science of Autocracy Suggests the U.S. Is Headed for Dictatorship</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 11:36:59
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Science Tells Us the U.S. Is Heading toward a Dictatorship In a split image on television, President Donald Trump delivers address to a joint session of Congress on March 4, 2025. Keep in touch with the Argonaut: Get email alerts for this weekly column by Dan Vergano As president, Donald Trump pretty much checks all the warning boxes for an autocrat. It's now May and sadly, that all checks out. The U.S. is in a bad place and, scholars warn, looks to be headed for worse. His officials have illegally fired federal workers, impounded congressional appropriations and seized people off the street for deportations to foreign prisons, threatening the same for all U.S. citizens. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. We should all be worried that the U.S. is headed toward an autocracy—government by one person—even without political science offering a warning. But scholarship on how nations descend into this unfortunate state, seen in places like Turkey and Hungary, might not surprise you with what it suggests about the U.S. “Since Donald Trump's inauguration, the country has embarked on the slippery slope toward autocracy,” concludes political scientist Daniel Stockemer of the University of Ottawa, in a May report in Politics & Policy. Rather than a coup, Trump's attacks on law firms, universities, immigrants and others constitute “a more incremental form of democratic erosion,” he writes, one that follows a six-step theory of incremental autocratization based on research on the democratic backsliding seen worldwide in recent decades. She looked at the last quarter-century's collapse in Venezuela, examining how states turn from democratic to autocratic in stages, as opposed to a sudden coup. Marked by angry politics, backlash against minorities and immigrants, and distrust in institutions, the U.S. has in the last two decades changed from a “full” to a “flawed” democracy, according to the Economist's global democracy index. “If my theory is correct, the U.S. is still in this transition phase between democracy and autocracy,” says Stockemer, by e-mail. In May, for example, the White House deputy chief of staff suggested Trump could unilaterally suspend habeas corpus, a legal remedy for unlawful detention that dates at least to the Magna Carta and is in the U.S. Constitution, to summarily round up immigrants. He cited an imaginary “invasion”—even though border crossings are at their lowest point in U.S, history, according to Trump's U.S. Customs and Border Protection agency—as a reason. Warnings of the fifth step on the road to autocracy, securing long-term power, come in Trump's musing of seeking an unconstitutional third term as president. He predicts that attacks on minority voting rights in 2026 and 2028 would be an expected outcome of this step. A simpler “competitive authoritarianism” yardstick for measuring democratic collapse comes from political scientists Steven Levitsky, Lucan Way and Daniel Ziblatt earlier this month. By that measure, they add, the U.S. has already crossed that line, ordering Department of Justice investigations into perceived political enemies, donors to the Democratic Party and news outlets ranging from CBS News to the Des Moines Register. “The administration's authoritarian offensive has had a clear impact. It has changed how Americans behave, forcing them to think twice,” they added. Even so, the damage already done is real: “It is very easy to destroy something such as USAID, but it takes a long time to rebuild it both physically and also in a trust sense, both in America and abroad,” says Stockemer, noting the rapid plummet of Canadian attitudes toward the U.S., from positive to sharply negative. This is an opinion and analysis article, and the views expressed by the author or authors are not necessarily those of Scientific American. Dan Vergano is senior opinion editor at Scientific American, where he writes the weekly column Argonaut. Subscribe to Scientific American to learn and share the most exciting discoveries, innovations and ideas shaping our world today.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/podcast/episode/west-texas-measles-cases-threaten-elimination-status-in-the-u-s-heres-why/'>West Texas Measles Cases Threaten Elimination Status in the U.S. Here's Why That Matters</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-05-14 10:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Declining MMR Vaccination Rates Make West Texas Outbreak a Threat to Measles Elimination Here to explain what that means—and why that status could be at risk—is Lauren Young, associate editor for health and medicine at Scientific American. Lauren, thanks so much for coming on to chat with us today. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. Feltman: So, to refresh our listeners' memories, could you give us a brief overview of the current measles outbreaks of concern? The current case count as of May 1 of the [Centers for Disease Control and Prevention's] report says 935 confirmed cases, which is growing at a pretty alarming rate. So it's important to note, too, that nearly 70 percent of the confirmed cases [in the U.S.] have been in younger people, ages 19 and below, and a large proportion of those cases are in unvaccinated people ... It's also known for creating a rash, which is pretty uncomfortable, coughing and runny nose, but it could also cause severe complications: it could open up people to pneumonia, organ failure and death. There've been three people who've died so far from these outbreaks, one adult and two children, and all three have been unvaccinated ... I know a lot of public health experts are keeping an eye on this and trying to understand, too, the public health response that's going on. Feltman: Sure, and just how abnormal is this compared to recent years? Young: Right, so every year we do see cases of measles, and this often happens primarily due to travel—so when someone goes abroad to a place where measles is more common, they'll come back and reintroduce, you know, some cases. What we're seeing now is the highest number of cases since 2019, when we had a pretty large outbreak that started in New York. When these cases started in West Texas, for instance, it was highly concentrated in Gaines County, which is known to have a pretty high population of homeschool children. And so it's hard to understand fully the vaccination rates in kids, since, again, these outbreaks and the, the cases are highly concentrated in children, so yeah, public health experts are definitely keeping an eye on this and are concerned about what's gonna happen in the next few months, yeah. Feltman: Yeah, I think a lot of folks get confused about the statement that we hear a lot lately that measles has been “eliminated” in the United States. Could you explain what that status means and how we got it? It's a little bit of a jargony, like, public health status thing, but the CDC and the World Health Organization define the status for measles as a period of 12 months with zero endemic cases, so that means there needs to be no continuous transmission of the disease over a 12-month period of time—so you can't link one case from another case. The United States achieved its elimination status of measles in 2000, and we've been able to keep that status primarily through prevention measures, particularly through vaccination. Young: Yeah, so there were a few prominent experts in the field of vaccine science who spoke out about this recently. Peter Marks, who was a former [Food and Drug Administration] official and he's a prominent vaccine expert, said he's worried that we're on the way to losing this status. Also Katherine Wells, who is the public health director in Texas, said in March during a news briefing that she's anticipating that this outbreak could go a year long ... Feltman: As you mentioned, this isn't our first big outbreak since 2000, so what factors are coming together to put our elimination status at risk after, you know, 25 years of success? Young: Yeah, so there's a few things that seem to be, you know, folding into play based off of what I'm just hearing from the experts that I've talked to. One, for sure, is we've been seeing kind of this steady decline in vaccination rates, specifically in kids but, you know, just nationally as well, ever since the pandemic. A big part of that was: during the pandemic itself a lot of children missed their well appointments, where they would get their routine vaccinations. We did see, you know, some increase from that, but there's other things at play. We also have, you know, some public health officials in office right now who have a history of endorsing antivaccine rhetoric and are also endorsing studies to reevaluate things like autism and vaccines and that connection there. And when we see things like a decline in vaccination rates it's very important for a disease like measles because it is so highly contagious. And for something like measles we need to see, as some experts have explained to me, very highly uniform vaccine coverage—in other words, high “herd immunity,” which is basically the level of either natural immunity or vaccination immunity you need to have in order to stop the spread of disease. So for measles you need about a 95 percent vaccination rate, and any sort of, you know, even slight decline in that can cause these severe outbreaks. So that's what we're seeing here, where, you know, we have a small pocketed community that had a lower vaccination rate and is, you know, spurring this particular outbreak. And measles is kind of, as some experts have said, a canary in a coal mine for vaccine-preventable diseases because it is so highly contagious, but if we continue to see this overall decrease in vaccinations for things like, you know, other eliminated diseases—like polio, for instance—that's also a little bit of concern for several experts. We did a whole story about this—Tara Haelle, one of our contributors, did a really deep dive on what that exactly would look like. So this is on the forefront of a lot of people's minds, just the general interplay between vaccine recommendations from public health officials and also how that's playing out from, you know, past historical trends. Young: It seems maybe like a little bit beating a dead horse, but getting vaccinated, you know, I think is still an important thing to do. Being active about, you know, going to the hospital or going and getting treatment if you're seeing any signs or symptoms—which, again, include the rash, coughing, and the runny nose and watery eyes. Feltman: Lauren, thank you so much for coming on. Unfortunately, I'm sure this won't be the last time we talk to you about measles, but we really appreciate it. If you haven't already submitted your answers for the Science Quickly listener survey, go check it out at ScienceQuickly.com/survey. Your responses will help us steer the future of the show, and you might just win a fun prize for helping us out. Science Quickly is produced by me, Rachel Feltman, along with Fonda Mwangi, Kelso Harper, Naeem Amarsy and Jeff DelViscio. Subscribe to Scientific American for more up-to-date and in-depth science news. Rachel Feltman is former executive editor of Popular Science and forever host of the podcast The Weirdest Thing I Learned This Week. Young is an associate editor for health and medicine at Scientific American. She has edited and written stories that tackle a wide range of subjects, including the COVID pandemic, emerging diseases, evolutionary biology and health inequities. Young has nearly a decade of newsroom and science journalism experience. Before joining Scientific American in 2023, she was an associate editor at Popular Science and a digital producer at public radio's Science Friday. She has appeared as a guest on radio shows, podcasts and stage events. Young has also spoken on panels for the Asian American Journalists Association, American Library Association, NOVA Science Studio and the New York Botanical Garden. Her work has appeared in Scholastic MATH, School Library Journal, IEEE Spectrum, Atlas Obscura and Smithsonian Magazine. Young studied biology at California Polytechnic State University, San Luis Obispo, before pursuing a master's at New York University's Science, Health & Environmental Reporting Program. Fonda Mwangi is a multimedia editor at Scientific American. She previously worked as an audio producer at Axios, The Recount and WTOP News. He has worked on projects for Bloomberg, Axios, Crooked Media and Spotify, among others.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            