
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-06-04</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64919104/new-evidence-rewrites-the-origins-of-the-dead-sea-scrolls-one-of-judaisms-ancient-texts/'>New Evidence Rewrites the Origins of the Dead Sea Scrolls, One of Judaism's Ancient Texts</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 18:01:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A researcher used an AI program that analyzed handwriting data which, paired with radiocarbon dating, pointed towards the ancient texts being even more ancient than we thought. Experts have a number of techniques they can use to get close, but there are limitations that often can't be overcome without additional information. However, not every scroll was labeled, and as a result, the undated Dead Sea Scrolls have been much harder for scientists to pin down. But when new technologies arise, things can change. According to a new study—in which scientists used AI modeling to study handwriting styles across ancient manuscripts with known dates—some of the undated Dead Sea Scrolls may be older than previously believed. Mladen Popovic (from the University of Groningen in the Netherlands) and his research team claim that their work not only re-dates some Dead Sea Scrolls, but could open a new way to place undated manuscripts on the timeline of ancient history. “It is very exciting to set a significant step in solving the dating problem of the Dead Sea Scrolls and also creating a new tool that could be used to study other partially dated manuscripts from history,” the authors wrote in a statement. “This would not have been possible without the collaboration between so many different scientific disciplines.” The process started with a bounty of ancient texts used to help build datasets. The Enoch model, paired with radiocarbon dating, estimates older ages for “many of the Dead Sea Scrolls” than traditional handwriting analysis methods. Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics. Farmers Dug Up a Skull Unlike Any Human Ancestor A Lost Rock Star Statue Reappeared After 37 Years</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09037-4'>Molecular gradients shape synaptic specificity of a visuomotor transformation</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 15:24:12
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. How does the brain convert visual input into specific motor actions1,2? In Drosophila, visual projection neurons (VPNs)3,4 perform this visuomotor transformation by converting retinal positional information into synapse number in the brain5. The molecular basis of this phenomenon remains unknown. We addressed this issue in LPLC2 (ref. 6), a VPN type that detects looming motion and preferentially drives escape behaviour to stimuli approaching from the dorsal visual field with progressively weaker responses ventrally. This correlates with a dorsoventral gradient of synaptic inputs into and outputs from LPLC2. Here we report that LPLC2 neurons sampling different regions of visual space exhibit graded expression of cell recognition molecules matching these synaptic gradients. Dpr13 shapes LPLC2 outputs by binding DIP-ε in premotor descending neurons mediating escape. Beat-VI shapes LPLC2 inputs by binding Side-II in upstream motion-detecting neurons. Gain-of-function and loss-of-function experiments show that these molecular gradients act instructively to determine synapse number. Similar transcriptomic variation within neuronal types is observed in the vertebrate brain7 and may shape synapse number via gradients of cell recognition molecules acting through both genetically hard-wired programs and experience. Animals rely on visuomotor transformations to convert object locations in eye coordinates into directional movements8. The brain regions and neural circuits regulating this transformation in both vertebrates9,10 and invertebrates11 have been characterized. The precise neuronal connectivity underlying learned12,13 and innate visuomotor14 tasks is shaped by genetically hardwired mechanisms, experience or both. In flies, visuomotor transformation occurs between VPNs and descending neurons15,16 (Fig. VPNs include lobula columnar (LC) and lobula plate/lobula columnar (LPLC) neurons4 comprising approximately 30 cell types with 20–200 cells of each type per hemibrain3,4. For simplicity, we refer to these collectively as VPNs. The dendrites of each VPN type span 20–40° of visual space, forming a retinotopic feature-detecting map17,18 in the optic lobe. Their axons converge and terminate within VPN-type-specific optic glomeruli in the central brain, where they synapse onto the dendrites of descending neurons and other neurons (Fig. Most VPN types lose axonal retinotopy4,5, meaning spatially organized visual inputs onto VPN dendrites do not translate to ordered axonal projections. a, VPN dendrites cover the lobula (Lo) and LoP, and axons converge on optic glomeruli innervated by descending neurons (DNs). b, Confocal projection of LPLC2 and the giant fibre (GF). c, Connectomic reconstructions of LPLC2 neurons (‘hemibrain'21), coloured by LPLC2–GF synapse count (top), and linear regression of synapse number versus dorsoventral (DV) axis (bottom). The bottom panel was adapted from ref. d, Same as panel c (top), using FlyWire22,23. e, GF-mediated short-mode takeoffs in response to lateral (90°) looms at various elevations. The error bars denote ±95% confidence intervals. Numbers indicate total takeoffs (one per animal). f, GF response to looming at different elevations (left), and pooled peak GF responses across five trials (right). Dots denote individual flies (n = 5 biologically independent animals). Repeated measures analysis of variance (rANOVA; P = 0.0048) with Sidak-adjusted post-hoc test, *P = 0.0385 (−25° versus 25°). g, Retinotopic maps transform into synaptic gradients between LPLC2 and GF without axonal retinotopy. h, Single-cell RNA sequencing (scRNA-seq) experimental design. i,j, tSNE plots of the 48 h APF dataset (i) with LPLC2, LPLC1 and LC4 annotated by marker gene expression (j). TP10K, transcripts per 10,000; X, unknown cell type. k, Heatmaps of the top 30 PC1 genes (15% variance explained) in LPLC2 at 48 h APF. l, LPLC2 neuron distributions along PC1–PC2 at 48 h APF, coloured by age, genotype, sex and coverage. lncRNA, long non-coding RNA; nUMI, number of unique molecular identifiers. m, PCA plots of LPLC2 neurons at 48 h APF, 72 h APF and 96 h APF, showing temporal expression changes in select genes from panel k. We recently demonstrated that the transformation from visual input to motor output in VPN–descending neuron circuits relies on synaptic gradients, which form in most cases independent of axonal retinotopy5. We define synaptic gradients as connectivity patterns where the number of synapses from a presynaptic population (for example, a VPN type) to postsynaptic targets (for example, descending neurons) varies topographically along the anteroposterior or dorsoventral axes of visual space. This synaptic organization underlies directional behavioural responses5. Although these gradients originate from retinotopically guided dendritic inputs, the resulting axonal synaptic connections often encode visual space in a non-spatial, abstract manner. The molecular mechanisms underlying this transformation remain unclear. To uncover the molecular basis of synaptic gradients, we examined a VPN type called LPLC2 neurons, a population of approximately 100 cells per hemibrain. Each LPLC2 neuron acts as a local looming detector, responding to dark, radially expanding motion centred on its receptive field6,17. The axons of LPLC2 neurons transmit visual information to the giant fibre (GF), a descending neuron that triggers a rapid looming-evoked escape takeoff19,20 (Fig. Previously, our analysis of two electron microscopy-based connectomic reconstructions21,22,23 revealed that LPLC2 neurons form a dorsoventral synaptic gradient, with dorsal LPLC2 neurons making more synapses with the GF than ventral neurons (Fig. Here we show that flies respond more strongly to dorsal than ventral looming stimuli correlating with the synaptic gradient. To investigate the molecular basis of this gradient, we combined single-cell RNA sequencing (scRNA-seq), spatial transcriptomics and genetics with morphological, behavioural and physiological studies. We identified two cell recognition molecules expressed in a gradient across the LPLC2 population. Dpr13 regulates a gradient of synaptic inputs to the GF, and Beat-VI controls a synaptic gradient onto the dendrites of LPLC2. We demonstrate that varying levels of these proteins in LPLC2 specify different numbers of synaptic inputs and outputs. This represents a new mechanism regulating synaptic specificity. To assess the function of the dorsoventral synaptic gradient between LPLC2 and the GF, we quantified escape responses to looming stimuli at different elevations. Flies take off in two modes: a faster short mode (featuring leg extension only, and taking the fly less than 7 ms to jump24) and a slower long mode (coordinated wing depression and leg extension24). Short-mode takeoffs are driven solely by the GF activation20. If LPLC2 neurons with more dorsal receptive fields form more synapses with the GF than ventral LPLC2 neurons, we predicted more short-mode takeoffs in response to higher-elevation stimuli. To test this hypothesis, we analysed thousands of previously collected25 high-speed videos of takeoffs elicited by looming stimuli at different elevations, and classified takeoffs by mode. The short-mode takeoff percentage increased as stimulus elevation changed from −30° to 77° (Fig. 1a–c), as would be expected from higher engagement of the GF in response to more dorsal looming. The GF receives direct visual input from only one other VPN type (LC4)26 in addition to LPLC2. LC4 neurons do not form a dorsoventral synaptic gradient with the GF5, suggesting the increase in GF-driven short-mode takeoffs at high elevations results directly from the LPLC2–GF synaptic gradient. We tested this by blocking synaptic transmission in LPLC2 and quantifying the looming responses in these flies (Extended Data Fig. Without LPLC2 input, the percentage of short-mode takeoffs did not change across elevations (Extended Data Fig. 1h), with the percentage of short-mode takeoffs greatly reduced at high (77°) elevations compared with control flies. To directly test whether LPLC2–GF connectivity biases GF activation, we performed in vivo whole-cell patch-clamp recordings in the GF during looming stimuli at three elevations. GF membrane depolarization was larger in response to dorsal than to ventral looming (Fig. This provides direct evidence that the LPLC2–GF synaptic gradient generates a corresponding gradient of GF activation, ultimately shaping escape behaviour. Electron microscopy reconstruction studies established that LPLC2 axons projections are not retinotopic (that is, intermingled) and yet form different numbers of synapses onto GF dendrites according to the location of the stimulus in the visual field to which they respond4,5 (Fig. We hypothesized that this pattern of synapses could be achieved through differential molecular recognition: individual LPLC2 neurons sampling different regions of visual space could express different levels of the same cell recognition molecule, which, in turn, would specify the number of synapses they form onto GF dendrites. To investigate whether LPLC2 neurons exhibit molecular variation that correlates with the synaptic gradient, we examined the LPLC2 transcriptome at three developmental stages during and after synaptogenesis (48 h after puparium formation (APF), 72 h APF and 96 h APF; Fig. In addition to LPLC2, we profiled a related cell type, LPLC1, which forms synaptic gradients without axonal retinotopy with other descending neurons5, and LC4, a VPN type with axons arranged in a retinotopic manner5. As neuronal transcriptomes are highly dynamic during development and are affected by genetic background27, we introduced internal controls at each time point to account for transcriptional heterogeneity driven by these factors. We used genetic multiplexing to perform pooled single-cell profiling across biological replicates27,28,29 (including different genetic backgrounds and developmental stages; see Methods). Our dataset included approximately 600 high-quality single-cell transcriptomes per cell type and time point (Fig. We validated the identity of each VPN cell type (LPLC2, LPLC1 and LC4) using known marker genes27 (Fig. This provided approximately 30-times higher per-cell-type coverage than existing single-cell atlases of the Drosophila optic lobes27,30. To explore heterogeneity in gene expression across each VPN cell type, we performed principal component analysis (PCA) separately for each cell type and time point (Fig. At 48 h APF, PC1 captured genes expressed in a graded manner across difference LPLC2 neurons (Fig. For example, the most differentially expressed gene was dpr13, encoding a cell recognition protein of the immunoglobulin superfamily (IgSF)31. There was, however, no clear boundary separating neurons with high versus low dpr13 expression levels. Many of the most differentially expressed genes also encode cell recognition molecules: IgSF (for example, tei, beat-VI and dpr17), leucine-rich-repeat (for example, trn) and cadherin (for example, Cad87A) families31,32 (Fig. To test whether the transcriptional heterogeneity within LPLC2 neurons reflects discrete subtypes or a continuous gradient, we artificially clustered neurons and shuffled gene expression in two ways: across all cells or only within artificial clusters (Extended Data Fig. Shuffling across all cells disrupted the gradient, indicating that the observed variation arises from coordinated gene expression. Shuffling gene expression only within arbitrarily separated clusters of neurons introduced artificial gaps, showing that the original data do not naturally separate into discrete subtypes. Thus, the transcriptomic heterogeneity across LPLC2 neurons forms a continuous gradient. The distribution of neurons along PC1 also did not correlate with developmental age, sex, genetic background or mRNA coverage (Fig. 2e and 3a,b), indicating that molecular heterogeneity had a different origin. The graded expression of the top differentially expressed genes associated with PC1 at 48 h APF (dpr13, beat-VI and tei (encoding IgSF molecules), as well as SiaT (encoding sialyltransferase)) persisted through development (Fig. In LPLC1 neurons, PC1 also captured gradients of IgSF-encoding transcripts (for example, DIP-κ, CG33543, dpr3 and sdk) that persisted through development (Extended Data Fig. Similar to LPLC2, molecular gradients in LPLC1 could not be explained by any of the confounding factors (Extended Data Figs. By contrast, PC1 in LC4 correlated with technical factors (for example, transcript count per neuron; Extended Data Figs. 2e and 3a,b), indicating that its molecular heterogeneity was not biologically relevant. In summary, we identified stable gradients of transcripts encoding recognition molecules in LPLC1 and LPLC2, but not in LC4, suggesting that molecular heterogeneity is a feature of VPNs forming synaptic gradients independent of axonal retinotopy5. To verify gene expression gradients in LPLC2 neurons, we used single-molecule hairpin chain reaction fluorescent in situ hybridization33 (HCR-FISH), an epansion-assisted light-sheet microscopy (ExLSM)34 to visualize and count transcripts within LPLC2 neurons (Fig. The cell bodies of adjacent LPLC2 neurons exhibited striking differences in transcript levels of dpr13 and SiaT, the two most differentially expressed genes in the LPLC2 dataset (Fig. Similar patterns were observed for beat-VI and dpr17 (Extended Data Fig. a, Light-sheet projection of the Drosophila optic lobe showing LPLC2 nuclei and transcripts of dpr13 and SiaT. b, Antiparallel expression of dpr13 and SiaT across LPLC2 neurons (scRNA-seq; Fig. The smoothed lines indicate estimated mean expression trend. The error bands denote ±95% confidence intervals. r, Spearman's rank correlation coefficient. c, Single 0.5-μm-thick slice from panel a (zoomed in region highlighted by the dashed rectangular box). The arrowheads denote individual LPLC2 neurons expressing markedly different levels of dpr13 and SiaT. d, Comparison between scRNA-seq (left) and FISH (right) measuring correlation in expression for two pairs of graded genes: dpr13–SiaT (top) and dpr13–beat-VI (bottom) across LPLC2 neurons. The smoothed lines indicate linear regression fits, and the shaded bands denote ±95% confidence intervals. r, Spearman's rank correlation coefficient (rs). e, Comparison of RNA-seq and HCR-FISH measuring correlation in expression for 12 pairs of genes at three developmental time points across LPLC2 neurons. The error bars denote means ± 95% confidence intervals. g, Positive control for panel f. n = 5 brains (one side per animal tested). h, Correlation between expression levels of dpr13–beat-VI and dpr17–SiaT (from scRNA-seq; Fig 1h–m), along PC1 across the LPLC2 population. i, Subsets of LPLC2 neurons expressing dpr13, dpr17, beat-VI and SiaT. For panels a–d,h–I, 48 h APF developmental time point. Data are from single experiments. We quantified differential gene expression inferred from HCR-FISH analysis using Flyseg35, an automated volumetric instance segmentation algorithm that we previously developed (see Methods). The results of scRNA-seq and HCR-FISH were similar for genes exhibiting antiparallel (for example, dpr13 and SiaT; Fig. Most of these relationships remained consistent throughout development (Fig. 2e and Supplementary Table 2), supporting our findings from scRNA-seq data. One exception was muscleblind(mbl); although this gene exhibited graded expression in scRNA-seq (Fig. 1k), it showed low, uniform expression in HCR-FISH. The reason for this discrepancy is unclear. In summary, most gene expression gradients across LPLC2 neurons from scRNA-seq data were confirmed by HCR-FISH. We next investigated whether there was a correlation between the retinotopic position of LPLC2 dendrites and gene expression levels. As the cell bodies of the LPLC2 neurons were arranged in a salt-and-pepper manner (Fig. 2c), we used a genetic intersection strategy to visualize the expression of transcriptional reporters for different genes in LPLC2 neurons in dendrites in different topographical locations in the visual field (Fig. Genes encoding recognition molecules dpr13, beat-VI and Cad87A, which showed correlated expression in scRNA-seq (Fig. 1k), were predominantly expressed by LPLC2 neurons with dendrites in the dorsal region of the lobula at 48 h APF (Fig. By combining reporter expression and HCR-FISH, we confirmed this heterogeneity, showing significantly higher dpr13 and beat-VI transcript levels in dorsal LPLC2 neurons (Extended Data Fig. Heterogeneous expression of genes encoding recognition molecules in LPLC1 neurons had similar retinotopic correlates (Extended Data Fig. 5e,f), suggesting that such retinotopically biased expression gradients are a general feature of many VPN types. To see whether these transcriptional gradients persist at the protein level, we used MIMIC-based protein traps36 generating GFP-tagged versions of two recognition proteins: Dpr13 and Beat-VI. This facilitated visualization of protein expression under their endogenous regulatory elements. Despite GFP accumulation in cell bodies (probably due to impaired trafficking), significant differences in GFP levels between dorsal and ventral LPLC2 neurons indicated that mRNA expression trends are maintained at the protein level (Extended Data Fig. In summary, individual neurons of the same VPN type that sample different regions of visual space exhibit molecular heterogeneity. These neurons express gradients of recognition molecules that match the orientation of their synaptic gradients. Therefore, despite spatial intermingling, the axons retain distinct molecular identities. Next, we investigated the functional relevance of these molecular gradients. LPLC2 neurons express genes encoding IgSF recognition proteins in a dorsoventral gradient, with dpr13 and beat-VI having higher expression levels in dorsal LPLC2 neurons and dpr17 having higher expression levels in ventral neurons. We hypothesized that one or more of these molecular gradients could specify the gradient in synapse number between LPLC2 axons and GF dendrites (Extended Data Fig. If this were the case, the GF would need to express cell-surface proteins that bind to one or more of these three recognition molecules. This would allow the differential molecular expression in LPLC2 neurons to be converted into differential cell adhesion between individual LPLC2 neurons and the GF. There are multiple paralogues of each, and interactions between them have been characterized37. Similarly, Beat proteins bind to Sides, which are also IgSF members31,38 (Fig. As a step towards testing our hypothesis, we assessed expression levels of genes encoding binding partners of Dpr13, Beat-VI and Dpr17 (DIP-ε, Side-II and DIP-γ, respectively) in the GF using HCR-FISH. DIP-ε, encoding a binding partner of Dpr13, was the only gene with strong expression in the GF during development (Fig. a, Molecular-binding partners of differentially expressed recognition molecules in LPLC2. b,c, Expression levels of candidate genes in the GF. Light-sheet projections of the GF nuclei (b), and quantification of their expression levels across development (c). The red puncta indicate candidate gene mRNA, and the circles denote individual GF neurons (one per animal; n = 3 neurons per gene). d, Confocal projections of colocalized LPLC2 axon terminals and the GF dendrites in wild-type and DIP-εnull animals across development. n, brains (one side per animal); n = 11 and 11 for 48 h APF, n = 10 and 9 at 72 h APF, n = 14 and 13 at 96 h APF for wild type and DIP-εnull, respectively. e, LPLC2–GF axo-dendritic overlap in controls and DIP-εnull across development. Unpaired t-test with Welch's correction (two-sided). f, Same as panel e for DIP-ε rescue in the GF. One-way ANOVA (F = 63.753, P = 3.64 × 10−13) followed by Tukey's (HSD) test for post-hoc pairwise comparisons. g, Same as panel e for controls and GF > DIP-ε RNAi animals. Unpaired t-test with Welch's correction (two-sided). h, Whole-cell patch-clamp recordings in the GF. GF responses to looming at r/v = 40 ms (left). Control (n = 5 flies) and DIP-εnull (n = 7 flies) traces (individual and average) are overlayed. Quantification of expansion peak amplitudes from individual flies is also shown (right). The circles indicate mean values of two recordings per animal, and the error bars denote median and 25–75th percentiles. Mann–Whitney U-test U = 4, *P = 0.0303. i, Same as panel h for controls and GF > DIP-ε RNAi animals (n = 5 flies each). U = 2, *P = 0.03175. j, Violin plots of takeoff sequence durations for lateral stimuli at different elevations in wild-type and DIP-εnull animals (top). Short and long modes are separated by a red dashed line. Short-mode takeoff percentages are also shown (bottom). The error bars denote mean ± 95% confidence intervals. Chi-squared test with post-hoc Bonferroni correction for multiple comparisons: *P = 0.0218 and NS P = 0.1149. k, Same as panel j for controls and GF > DIP-ε RNAi. l, Same as panel j for controls and LPLC2 > UAS-dpr13. Unpaired t-test with Welch's correction (two-sided). *P = 0.022640. n, Same as panel j for wild type and dpr13null. o, Same as panel e for control and LPLC2 > UAS-DIP-ε animals. Unpaired t-test with Welch's correction (two-sided). ***P = 0.000503. p, The model shows that a DV gradient of Dpr13–DIP-ε interactions determines synapse number between individual LPLC2 neurons and the GF. In panels e–g,m–o, n values (images) and circles (plots) represent brains (one side per animal), and the error bars denote median ± 95% confidence intervals. Data are from single experiments. To determine whether DIP-ε regulates synaptic connections between LPLC2 axons and GF dendrites, we explored the interaction between them in DIP-ε-deficient flies. Animals with no DIP-ε (DIP-εnull allele) displayed an approximately tenfold reduction in axo-dendritic overlap (Fig. Knockdown of DIP-ε specifically in the GF using two different RNA interference (RNAi) lines also significantly reduced this overlap (Fig. Wild-type levels of overlap were restored in DIP-εnull flies through targeted expression of DIP-ε in the GF (Fig. Thus, DIP-ε promotes interaction between LPLC2 axons and GF dendrites. To assess whether this effect was associated with a decrease in synapse number, we visualized presynaptic sites (marked with T-bar-associated endogenously tagged Bruchpilot (Brp) protein) in sparsely labelled LPLC2 neurons using a modification of the synaptic tagging with recombination (STaR)39 technique (Extended Data Fig. RNAi knockdown of DIP-ε in the GF led to a significant reduction in the number of LPLC2 T-bars contacting the GF dendrites (Extended Data Fig. Thus, DIP-ε is required in the GF to establish synaptic connectivity with LPLC2 axons. If DIP-ε is required to form functional LPLC2–GF synapses, then we would expect loss of DIP-ε to reduce the GF responses and to disrupt short-mode takeoffs. We also anticipated that, if too few LPLC2–GF synapses formed to make a dorsoventral gradient, then the number of short-mode takeoffs would no longer increase with increasing elevation of the looming stimulus. To test these assumptions, we first performed whole-cell patch-clamp recordings in the GF while presenting looming stimuli to both DIP-εnull and control flies. As predicted, we observed a reduction in the peak magnitude of the GF response in DIP-εnull animals and in those expressing DIP-εRNAi in the GF compared with controls (Fig. DIP-εnull and DIP-ε-RNAi-expressing animals also had dramatically reduced short-mode takeoffs across all stimulus elevations and a smaller difference between 0° and 77° than controls (Fig. These results support a role for DIP-ε in the GF in establishing graded synaptic connectivity with LPLC2 neurons. We did this in two ways. To do this, we increased the level of dpr13 uniformly across the LPLC2 population (heretofore referred to as ‘overexpression', that is, superimposed on the endogenous dpr13 gradient). Compared with controls, animals overexpressing dpr13 in LPLC2 exhibited a higher percentage of short-mode takeoffs (Fig. This gain of function was most pronounced at lower (0°), decreased at medium (45°) and was absent at higher (77°) stimulus elevations, resulting in no significant change in short-mode escape frequency across elevations (that is, animals responded uniformly to dorsal and ventral looming stimuli). Thus, when the dpr13 gradient was flattened, the dorsoventral gradient of short-mode takeoffs was eliminated (Fig. This manipulation also led to a modest increase in LPLC2–GF axo-dendritic overlap (Fig. Our data support a causal relationship between the level of dpr13 and the number and graded distribution of LPLC2 synapses onto the GF. dpr13null flies showed no reduction in the overlap between LPLC2 axons and GF dendrites or the number of LPLC2 T-bars contacting the GF dendrites (Fig. This may be due to redundancy, as LPLC2 neurons also express four other Dprs that bind to DIP-ε (dpr14, dpr18, dpr19 and dpr20; Extended Data Fig. To test this, we ectopically expressed DIP-ε in LPLC2, promoting cis-interactions that block trans-interactions with the GF (an effect observed for other DIP–Dpr pairs40,41). This manipulation reduced LPLC2–GF overlap (Fig. Of the Dpr paralogues that bind to DIP-ε, only Dpr13 is expressed in a graded manner (Extended Data Fig. Although dpr13null flies displayed no changes in the LPLC2–GF axo-dendritic overlap, they no longer maintained the dorsoventral gradient of short-mode takeoffs (Fig. This mainly resulted from a significantly increased short-mode takeoff percentage at lower stimulus elevations. The reason for this is unknown. However, it raises the possibility that it is not the absolute level of Dpr13 expression that determines synapse number, but rather the relative differences in expression levels between different LPLC2 neurons. In summary, DIP-ε and Dpr13 function as a ligand–receptor pair to regulate LPLC2–GF synaptic connectivity. Our findings support a model in which the gradient of Dpr13–DIP-ε interactions shapes the synaptic gradient (Fig. We next investigated the function of other genes displaying dorsoventral expression gradients in LPLC2 neurons (Fig. beat-VI, encoding another IgSF recognition molecule38, exhibited a dorsoventral gradient, with higher expression in dorsal and lower expression in ventral LPLC2 neurons (Fig. Neither the loss of Beat-VI nor its binding partner Side-II affected LPLC2–GF axo-dendritic overlap (Extended Data Fig. Given that LPLC2 dendrites also receive graded synaptic inputs (see below), we tested whether beat-VI controls this gradient. Each LPLC2 neuron has four dendritic branches in the lobula plate (LoP), one in each layer, extending in four cardinal directions and corresponding to motion sensitivity in each layer6 (Fig. Individual LPLC2 neurons thus serve as local looming detectors. They respond most strongly to looming stimuli originating at their receptive field centre extending radially outwards and simultaneously generating motion in all four cardinal directions6. Using the FlyWire22,23 connectome, we found that two of the four classes of T4/T5 neurons, T4c/T5c and T4d/T5d, form antiparallel dorsoventral synaptic gradients onto LPLC2 dendrites in LoP3 and LoP4 layers, respectively (Fig. a, Confocal projections of the LPLC2 population (top; n = 8 brains) and an individual LPLC2 neuron (bottom; n = 11 neurons, one per brain), highlighting the dendritic branches in the LoP. Numbers refer to LoP layers. The inset shows the posterior view of a single LoP dendrite with branches extending into one of the four cardinal directions in each layer. b, In the connectomic reconstructions (FlyWire22,23) of LPLC2 neurons, the skeleton of each neuron is colour coded by the number of inputs each LPLC2 neuron receives from T4c or T4d and T5c or T5d neurons. Scale bars, 50 μm (a,b). c, Confocal projections of LoP dendrites (posterior view) in individual dorsal, central and ventral LPLC2 in control and LPLC2 > beat-VI RNAi flies. The dashed ovals indicate LoP4 dendritic branches. d, Length of LoP4 dendritic branches for dorsal, central and ventral LPLC2 neurons in control and LPLC2 > beat-VI RNAi flies. The circles denote neurons (one neuron per brain), and the error bars indicate median ± 95% confidence intervals. Unpaired t-test with Welch's correction. e, Confocal projections of individual LoP dendrites (posterior view) of dorsal and ventral LPLC2 neurons in T4/T5 > side-II RNAi flies. The dashed ovals denote LoP4 dendritic branches. n = 6 neurons for both positions (one neuron per brain). Data are from single experiments (a,c,e). We and others have shown that Beat-VI regulates synaptic specificity in the Drosophila motion detection system42,43. Beat-VI and its interacting partner Side-II38 are required for matching downwards motion-detecting T4d/T5d neurons with their postsynaptic partners (for example, LLPC3 and VS neurons) in LoP4 (refs. Because LPLC2 is part of this circuit, we hypothesized that the Beat-VI gradient might selectively regulate the inputs of T4/T5d neurons to the dendrites of LPLC2 in LoP4. We next sought to test whether the Beat-VI gradient regulates the gradient of synaptic inputs to LPLC2 in LoP4; that is, the dorsal and ventral LPLC2 neurons may receive different amounts of upwards and downwards motion information, potentially linked to the Beat-VI levels. Supporting this, RNAi knockdown of beat-VI altered LPLC2 dendritic morphology in a location-dependent manner. Removing Side-II from T4/T5 neurons produced a similar graded phenotype affecting dorsal, but not ventral, LPLC2 neurons (Fig. That beat-VI functions in an instructive manner was supported by the finding that overexpression of beat-VI selectively in LPLC2 caused the opposite effect, increasing LoP4 dendritic branch length in ventral, but not dorsal, LPLC2 neurons (Extended Data Fig. This phenotype corresponded to a relatively stronger increase in beat-VI mRNA levels across ventral, rather than dorsal, LPLC2 neurons (280% versus 81%, respectively; Extended Data Fig. These findings indicate that graded Beat-VI–Side-II interactions regulate synaptic connectivity between T4d/T5d axons and LPLC2 dendrites. As the dendrites of ventral LPLC2 neurons are unaffected in beat-VI and side-II removal, these dendrites must be regulated by a Beat-VI-independent mechanism or by such a mechanism acting in a redundant manner with Beat-VI and Side-II. To determine whether loss of Beat-VI in LPLC2 has functional consequences, we assessed the response of LPLC2 neurons in head-fixed animals to small dark bars moving in different directions across a white background using calcium imaging (Fig. Wild-type dorsal and ventral LPLC2 neurons exhibited different responses to these directional stimuli (Fig. Ventral LPLC2 neurons showed little to no response to downwards motion, as would be expected if they received little input from downwards sensing T4d/T5d neurons in LoP4 (Figs. By contrast, dorsal LPLC2 neurons were more sensitive to motion in both upwards and downwards directions (Fig 5d–g and Extended Data Fig. In response to beat-VI knockdown, dorsal LPLC2 neuron responses became biased for upwards motion similar to wild-type ventral neurons. This is consistent with the reduction of downwards-selective T4d/T5d input (that is, loss of LoP4 dendritic branches; Fig. However, beat-VI knockdown had little effect on ventral LPLC2 neurons, which remained biased for upwards motion (Fig. Uniform overexpression of beat-VI in LPLC2 neurons superimposed on the normal beat-VI expression pattern caused the opposite effect. Thus, both beat-VI knockdown and overexpression disrupted the normal directional tuning biases across LPLC2 neurons (Fig. a, Schematic of the fly eye relative to the display for visual stimulation during calcium imaging. b, Display positions probing LPLC2 receptive field with dark edges moving in 24 orientations (top). Mollweide projection of the outermost positions (black boxes) relative to the eye is also shown (bottom). c, Connectomic reconstructions16 of putative LPLC2 neurons at positions 16 (bodyID_28871) and 40 (bodyID_30207). The insets show neurites imaged; single-cell ROIs are overlaid. d, Polar plots of peak responses to moving dark edges in dorsal (red) and ventral (blue) LPLC2 neurons (representative fly). e, Directional sensitivity index (DSI) for dorsal (red) and ventral (blue) LPLC2 neurons in control, beat-VIRNAi and UAS-beat-VI flies. f, Bootstrap of DSI mean in dorsal (16; top) and ventral (40; bottom) regions for control, beat-VI RNAi and UAS-beat-VI flies. g, Filtered heatmap of DSI for all tested positions in control, beat-VI RNAi and UAS-beat-VI flies. h, Average peak responses to looming stimuli above (dorsal) or below (ventral) the eye's equator in UAS-beat-VI (left), control (middle) and beat-VI RNAi (right) flies. Bonferroni-adjusted pairwise t-tests for post-hoc comparisons. NS P = 1.0000 for all comparisons. i, The model shows that a DV gradient of Beat-VI–Side-II interactions determines synapse number between T4d/T5d and LPLC2 neurons. n, biologically independent animals (multiple trials per animal; e,g,h). Data are from single experiments. In summary, Beat-VI and Side-II act as a ligand–receptor pair to control synaptic connectivity between T4d/T5d and LPLC2 neurons. Our data support a model in which the dorsoventral gradient of Beat-VI–Side-II interactions contributes to the formation of the dorsoventral gradient of synaptic weights between T4d/T5d and LPLC2. This results in a decreased response to downwards motion by LPLC2 neurons with ventral receptive fields (Fig. In this study, we identified the molecular origins of two parallel dorsoventral synaptic gradients in the axons and dendrites of looming-detecting LPLC2 neurons. These synaptic gradients translate retinotopic information from the visual field of the fly into specific motor outputs. We showed that a gradient of Beat-VI through interactions with its binding partner Side-II regulates graded synaptic inputs from T4d/T5d axons onto LPLC2 dendrites. This regulates differential integration of the downwards motion component of the looming stimulus across different regions of the eye. Similarly, a gradient of Dpr13 working through its binding partner DIP-ε shapes graded synaptic outputs of LPLC2 onto the GF. This biases the escape towards faster takeoffs in response to looming stimuli from the dorsal visual field. These synaptic gradients, however, are distinct from each other. The dendritic synaptic gradient follows a retinotopic match between populations of presynaptic (T4d/T5d) and postsynaptic LPLC2 neurons (that is, many to many with retinotopy). By contrast, the axonal synaptic gradient follows a many-to-one pattern, in which multiple LPLC2 axons form different numbers of synapses onto a single GF neuron, independent of retinotopy. Despite these differences, both synaptic gradients rely on varying levels of recognition proteins across LPLC2 neurons, which establish synapse numbers that ultimately shape behaviour. The relationship between the gradients of beat-VI and dpr13 expression and the number of synaptic inputs to and outputs from LPLC2 was also explored. In beat-VI loss-of-function and gain-of-function experiments, the severity of functional effects and the extent of branching in LoP4 correlated with beat-VI expression levels. It remains unclear whether this phenotype arises from a failure to extend dendrites or to stabilize them. This protein interaction could influence synapse number either indirectly, by increasing the area of contact between T4d/T5d axons and LPLC2 dendrites, or directly, by controlling synapse formation. Similarly, Dpr13–DIP-ε interactions are required for LPLC2–GF synaptic connectivity. Loss of DIP-ε reduced the overlap between LPLC2 axons and GF dendrites, leading to significant defects in GF responses to looming. Analysing the loss of Dpr13 function is complicated by the expression of four additional Dpr paralogues in LPLC2, which bind to DIP-ε with similar affinities37. This mimicked the DIP-ε loss-of-function phenotype in the GF and provided additional support for the interactions between DIP-ε in the GF dendrites with its cognate Dprs in LPLC2 axons. We propose that multiple Dpr paralogues establish baseline LPLC2–GF connectivity, whereas Dpr13 specifically defines the synaptic gradient. In support of this, both gain and loss of Dpr13 function eliminated the graded behavioural response to dorsal versus ventral looming stimuli. Thus, Dpr13–DIP-ε interactions precisely regulate synapse number, forming a behaviourally relevant synaptic gradient. Future studies will explore the mechanistic basis of this regulation. The Dpr13–DIP-ε gradient differs from Eph–Ephrin gradients in vertebrate retinotectal maps44. Although both relay visual-spatial information, they use different strategies. Ephrins and Ephs form topographical maps by maintaining retinotopy, ensuring the spatial relationship between retinal ganglion cells in the retina and the location of their axon terminals in the midbrain45. By contrast, the Dpr13–DIP-ε gradient translates retinal positional information into synapse numbers in the brain independent of anatomical mapping. As most VPNs form synaptic gradients5, differential expression of recognitions molecules may be a common mechanism for synapse number regulation. In mammals, altering levels of SynCAM1, another IgSF molecule, also affects synapse number in the hippocampus46. Thus, cell recognition molecules not only match synaptic partners32,47 but their expression levels may also influence the number of synapses between matched neurons with functional and behavioural consequences. This variation holds enormous potential for fine-tuning behavioural responses driven by neural circuits. Our work reveals continuous molecular heterogeneity within VPN cell types. This differs from discrete27,30 or stochastic48 variations in gene expression within neuronal types previously described in Drosophila. Beyond recognition molecules, we identified many graded genes in VPNs, suggesting that molecular gradients may regulate broader cellular functions. Cell types in the mammalian cortex also exhibit spatially patterned transcriptomic continuity7,49,50. Thus, within-cell-type molecular variation could be a common mechanism for generating neuronal diversity which, in turn, contributes to synaptic specificity. In conclusion, we have described a molecular strategy regulating neural circuit assembly. This became possible by combining detailed electron microscopy connectomics16,21,22,23, single-cell genomics, genetics, physiology, biochemistry and behaviour. We anticipate that merging these approaches to study other circuits in the fly and mammalian brain51,52 will uncover new molecular principles underlying wiring specificity. Flies were reared under standard conditions at 25 °C and 50% humidity with a 12-h light–12-h dark cycle on a standard cornmeal fly food. Supplementary Table 1 provides detailed descriptions of fly genotypes used in each experiment and origins of transgenic stocks. For developmental experiments, white pre-pupae (0 h APF) were collected and incubated at 25 °C for the indicated number of hours. Virgin female flies carrying LPLC2, LPLC1 or LC4 split-GAL4 driver lines were crossed to male flies expressing a nuclear GFP reporter and carrying unique 3rd chromosomes from the isogenic wild-type strains with known genotypes (Drosophila Reference Genetic Panel (DGRP)29). Each experimental condition (cell type and time point) was crossed to six unique DGRP strains (see Supplementary Table 1 and Source Data for details; the LC4 driver was crossed into four DGRP strains for 72 and 96 h APF time points). Each of these crosses was considered independent biological replicates. F1 generation animals were collected at 0 h APF and incubated for either 48, 72 or 96 h at 25 °C. To introduce a control for developmental age, we split six DGRPs (for each cell type and time point) into equal ‘early' and ‘late' groups (Fig. 1h; see Source Data for a detailed breakdown). Animals from early DGRPs were continuously staged and collected within the 2-h time window, after which we continuously staged and collected animals from late DGRPs within the next 2-h time window. This ensured that by the time brains were dissected (after 48, 72 or 96 h), animals from the early group were on average 2 h older than their late counterparts. Brain dissociation was performed as previously described27. Single-cell suspensions were used to generate scRNA-seq libraries using the 10X Genomics Chromium Next GEM Single Cell 3′ kit (v3.1) following the manufacturer's protocol. Each sample (corresponding to a single time point) was loaded to a single lane of 10X Chromium. Library preparation and sequencing were performed by the Technology Center for Genomics and Bioinformatics at UCLA. Raw scRNA-seq reads were processed using Cell Ranger (10X Genomics, v7.1.0). The reference genome and gene annotations were downloaded from FlyBase54 (release 6.29). Each time point was processed separately. Six biological replicates were tagged with a unique wild-type chromosome, and demultiplexed based on a unique wild-type chromosome using demuxlet28 (v2; https://github.com/statgen/popscle). Demultiplexing was performed using single-nucleotide polymorphisms (SNPs) from seven DGRP strains used in experiments (see Supplementary Table 1 for the full list of genotypes and Source Data for the list of DGRP strains used for each genotype and time point) and three additional DGRP strains as negative controls (line_129, line_427 and line_712). SNPs were filtered using the following criteria: (1) only biallelic SNPs on the 3rd chromosome without missing data (called in all 10 strains); (2) non-reference allele only in 1 of 10 strains. We quantified allelic counts for filtered SNPs using samtools mpileup55 (v1.10). SNPs with a minimum total coverage of 10 in all three samples (time points) and a maximum non-reference allele frequency of 0.25 were kept for downstream analysis (34,655 SNPs). Only a few cells (0.1–0.2%) were erroneously assigned to negative controls; 5–15% of cells were classified as ‘doublets' and ‘ambiguous' (mostly cells with low transcript coverage). The final dataset included 2,595 cells for 48 h APF, 2,369 cells for 72 h APF and 1,039 cells for 96 h APF. The scRNA-seq analysis was performed using Seurat56 package (v5.0.1). The analysis was performed separately for each time point using the standard Seurat workflow: raw transcript counts were normalized, 1,000 highly variable genes were scaled and used for PCA, first 5 PCs were used for clustering (resolution of 0.05) and to calculate tSNE projections. Clusters were annotated based on known marker genes27 (Fig. Most of the cells corresponded to LPLC2, LPLC1 and LC4 neurons. To explore further heterogeneity within VPN type, we subsetted each cell type at each time point, identified 500 highly variable genes and repeated PCA. We plotted known biological and technical covariates along each analysed PC (Fig. 2 and 3), including developmental age, DGRP genotype, sex and coverage (that is, transcripts per cell). PC1s in LPLC2 and LPLC1 did not correlate with any of these covariates and were driven by similar sets of genes (Fig. Further in vivo validations using orthogonal approaches confirmed that these PCs captured true molecular heterogeneity within each of these cell types (Fig. A high-throughput takeoff assay was performed with the FlyPEZ system25, which allows for the near-automated collection of fly behaviours in response to visual stimulation in large sample sizes. FlyPEZ experiments were performed as previously described25. A single stimulus was presented per fly. All behavioural experiments were performed 4 h before incubator lights were switched off, which coincides with the activity peak of the flies in the afternoon light cycle. A 7-inch-diameter back-projection-coated dome was placed centred over the glass platform to present visual stimulation. Specifically, dark looming disks that approach the fly from azimuth of 0° (front looms) or 90° (side looms), at elevations of −30°, 0°, 23°, 45° or 77° in fly coordinates, were used. Looming stimuli were generated using the same equation as described for calcium imaging experiments (see below). All looming stimuli have l/v = 40 ms. 1a–c show trials that were performed in the past (from 2014 to 2024) using control flies shown looming disks with a starting size ranging from 1–30° expanding to either 45°, 90° or 180°. 3 include trials with looming disks expanding from 10° to 180° only. Takeoff sequence durations between 0 ms and 7 ms were considered short-mode takeoffs, and takeoff sequence durations longer than 7 ms were considered long-mode takeoffs, as previously described20. Short-mode takeoff percentage was calculated by the number of short-mode takeoffs divided by the total number of takeoffs. 1a–c, takeoff sequence duration longer than 50 ms was eliminated as outliers. All takeoff sequence durations were less than 50 ms for experiments in Fig. Statistical comparison of the percentages of short-mode takeoffs was performed with the Chi-squared test, with post-hoc Bonferroni correction for multiple comparisons. Statistical comparison of takeoff sequence distributions between two samples was performed with the Mann–Whitney U-test. Statistical comparison of takeoff sequence distributions between more than two samples was performed with the Kruskal–Wallis test, with post-hoc Dunn correction for multiple comparisons. In vivo whole-cell, current-clamp electrophysiology was performed on behaving, tethered flies as previously described57. Visual stimuli were back-projected onto a 4.5-inch diameter mylar cylindrical screen covering 180° in azimuth via two DLP projectors (Texas Instruments Lightcrafter 4500) as previously described57. 1, visual stimuli were back-projected at 360 Hz onto a 4-inch diameterdome at 768 × 768 resolution as previously described5. Looming visual stimuli were generated using Psychtoolbox as previously mentioned. To maximize GF responses, a column of three black looming disks was displayed on a white background on the experimentally accessible visual field of the fly from elevation of −25° to 25°. The looming disks expand from 0° to 30° at a constant velocity of 500° s−1. Looming stimuli from different elevations were shown in randomized order for five times per animal, with a 15-s inter-stimulus interval. To estimate the magnitude of depolarization in response to looming stimuli, membrane potentials were averaged across individual trials, and the peak response (mV) and area (ms × mV) relative to the baseline were calculated in the 150-ms response region using custom MATLAB scripts. Statistical comparison of the looming responses in the GF across elevations was performed with repeated-measures one-way ANOVA test, with post-hoc Sidak correction for multiple comparisons. Calcium imaging was performed with a VIVO Multiphoton Open (Intelligent Imaging Innovation, Inc.) system based on a moveable objective microscope (Sutter Instruments). The excitation of the sample was delivered by a Ti:Sapphire laser (Chameleon Vision I, Coherent) tuned to 920 nm with power ranging from 15 to 30 mW (depending on imaging depth). A dual axis mirror galvanometer was used for x–y laser scanning (RGG scanbox, Sutter Instrument). We imaged with a ×20 water-immersion objective (W Plan-Apochromat ×20/1.0 DIC, Zeiss) and a band-pass filter (Semrock 525/40 nm) was placed in front of the photomultiplier tube (H11706P-40, Hamamatsu) to reduce the blue light from the visual display. Microscope and data acquisition were controlled by Slidebook 2024 (Intelligent Imaging Innovation, Inc.). Single plane images were sampled at 9 Hz with a spatial resolution of approximately 180 × 180 pixels (95.7 × 95.7 μm, pixel size ≅ 0.53 μm and dwell time ≅ 2 μs). Images and external visual stimuli were synchronized a posteriori using frame capture markers (TTL pulses from Slidebook 2024) and stimulus events (analogue outputs from the visual display) sampled with a data acquisition device (USB-6229, National Instruments) at 10 kHz. Flies were prepared and head-fixed to a custom objective stage fly holder as previously described53. GCaMP7f responses of dendritic branches from individual LPLC2 neurons were recorded from a posterior view. For each fly, we identified the most dorsocaudal dendritic arbors in the LoP and then moved the focal plane approximately 10 μm below them to start mapping the receptive field centres of dorsal LPLC2 neurons, or moved approximately 50 μm to probe ventral LPLC2 receptive fields, similarly to previous calcium imaging experiments in LPLC2 (ref. Random steps (±5 μm) between these two bracketed Z-planes were used to probe the receptive field centres of dorsoventrally intermediate LPLC2 neurons. Unstable recordings or recordings from preparations that did not respond during the receptive field scanning trials were not included in the dataset. A visual display composed of 48 8 × 8 dot matrix LED panels arranged in a semi-cylinder58 was used for visual stimulation as previously described53. Four layers of filter (071, LEE Filters) were placed over the display to reduce its light intensity. Visual stimuli were generated and controlled using custom-written MATLAB (MathWorks) scripts that communicated to the display through the microcontroller serial port. The display background was set to 70% maximum intensity, whereas foreground objects (looming or moving bars) were set to 0%. The set of visual stimuli was presented in random block design and repeated two times. Each trial was followed by 3 s of rest in which flies faced the visual background. Neurites in this location were previously shown to have weak responses to a small bar moving in each of four cardinal directions (that is, stimuli exciting LPLC2 dendrite branches in a single LoP layer) and a much larger response to looming (that is, stimuli exciting LPLC2 dendritic branches in all four LoP layers simultaneously)6. We identified an active neurite from a single neuron in the multiphoton field of view (Extended Data Fig. The receptive field centre of that neurite was identified in real-time and subsequently scanned for directional sensitivity. We developed a custom GUI in MATLAB, which allowed for real-time modifications to stimulus positions on the visual display. This interface enabled hand-triggered looming stimuli and the visual inspection of GCaMP responses. To identify the receptive field centre of an LPLC2 dendritic branch, we created a rectangular grid of 48 positions across the right half of the visual display. The positions were spaced every five LEDs in both horizontal and vertical directions, with each LED covering approximately 2.2° on the retina at the eye's equator. Using the GUI, the experimenter presented a looming stimulus centred at each grid position. The looming stimulus simulated a circular object with a 0.5-cm radius, starting from a distance of 50 cm and travelling at 62.5 cm s−1. This caused the object to expand from 0.6° to 14° with a loom velocity (l/v) of 8 ms. If a response was visually detected, the surrounding grid positions were probed next. We tested directional selectivity by moving a dark edge outwards from the centre of the receptive field in 24 different directions (Fig. The edges moved at 20° s−1 with orientations ranging from 0° to 345° in 15° increments. Each edge subtended 15° at the eye's equator and swept 15° orthogonal to its orientation, filling a 15° black square upon completion. In addition, a looming stimulus centred within the receptive field, with the same dynamics as those used for the receptive field scans, was included in this battery of dark moving edges. We sampled neurons along the dorsoventral and anteroposterior axes of the lobula and confirmed their anatomical locations by mapping the receptive field centres onto the fly eye (Fig. We identified specific dorsal and ventral retinal ommatidia and their corresponding columnar LPLC2 in the connectome, verifying the recorded locations of LPLC2 neurons (Fig. Coordinates were calculated using a 3D reconstruction of the fly head, holder and visual display in AutoCAD (Autodesk). We estimated the fly ommatidia with overlapping horizontal coordinates through the following steps: (1) identified the locations of the ommatidia pointing to positions 16 and 40 based on a Mollweide projection of 3D ommatidia directions from a microCT scan59; (2) mapped these ommatidia locations onto identified visual columns of the male optic lobe connectome16; (3) used T4 neurons included in these visual columns to identify downstream LPLC2 neurons. For each recording, the direction sensitivity index (DSI) was computed as follows: where \({R}_{{\rm{up}}}\) is the peak response to an upwards moving edge (0° direction) and \({R}_{{\rm{down}}}\) is the peak response to a downwards moving edge (180° direction). The index ranged from −1 to 1, with negative values indicating downwards sensitivity and positive values indicating upwards sensitivity. Images exported from Slidebook 2024 were processed following established protocols53. We used a custom MATLAB toolbox developed by B. J. Hardcastle (available at https://github.com/bjhardcastle/SlidebookObj) to correct for motion artefacts in the x–y plane and to delineate regions of interest (ROIs) around individual LPLC2 neurites within the dendritic tree. For each recording, a time series was generated by calculating the mean fluorescence intensity of pixels within the ROI (Ft) in each frame. These mean values were then normalized to a baseline value using the formula: where \({F}_{0}\) is the mean of Ft during the 0.5 s preceding stimulus onset. This approach ensures accurate correction for motion artefacts and reliable quantification of fluorescence intensity changes in LPLC2 neurites. The time series for each ROI were then exported from MATLAB and imported in RStudio by using the R package ‘R. Custom R scripts were then written for data plotting and statistical analyses. Given the repeated sampling and unbalanced sample sizes between groups and conditions, we used linear mixed effects models to fit the DSI values. This method maintains statistical power by avoiding averaging procedures and provides more accurate estimates of model parameters, including both fixed and random effects. The fixed effects were defined by the interaction between genotype (control, beat-VI RNAi or UAS-beat-VI) and condition (dorsal or ventral), whereas the random effects were attributed to individual flies61,62. We modelled the data using the R package ‘lme4'63 assuming residuals followed a Gaussian distribution. Pairwise post-hoc comparisons of the fixed effects were conducted using t-tests with Bonferroni adjustments, implemented through the R package ‘emmeans'65. In addition, to estimate the mean DSI differences across groups and conditions without assuming a specific distribution, we performed standard bootstrap simulations with 10,000 replicates using the R package ‘boot'66,67. Dot plots were generated with the R package ‘ggplot2'68. Smoothed heatmaps were generated with the R package ‘spatstat'69. The coding sequences of DIP-ε, dpr13 (isoform RB) and beat-VI were cloned into a modified pJFRC5 vector (Addgene: 5×UAS-IVS-mCD8::GFP, plasmid #26218) by replacing the mCD8::GFP coding sequence. Cloning strategies were designed using SnapGene 4.1.9 (GSL Biotech). Synthesis and cloning were carried out by Genewiz, Inc. Plasmids and sequences are available on request. Flies were generated by injecting the plasmid into embryos for recombination into attP1 sites (BDSC #8621) by BestGene, Inc. The DIP-εnull allele was generated as previously described70. In brief, two single guide RNAs (sgRNAs) were used to generate a frameshift deletion in the DIP-ε coding sequence. High-score spacer sequences were chosen using the SSC tool71. Each sgRNA was cloned into pU6-2-sgRNA-short (Addgene #41700) plasmid and two plasmids were co-injected into the vas-Cas9 line (BDSC #51324) by Bestgene, Inc. Injected larvae were crossed with balancer lines and PCR screened in F1 for single flies carrying the deletion. A mutant stock was established from this single F1. The sgRNA target sequences used for DIP-εnull allele generation were GCTGTTCTGTGGTCATACGATAGC and CTTCAATCGATTGACGGTGGAGC. sgRNA sequences were identified with an efficiency score above 5, as defined by the CRISPR Efficiency Predictor (https://www.flyrnai.org/evaluateCrispr/). The sgRNA sequence oligos were synthesized (Integrated DNA Technologies) and cloned into the pU6b-sgRNA-short vector72 to generate a large approximately 30-kb deletion spanning most of the dpr13 genomic region. All pU6 vectors generated were verified by Sanger sequencing. Two plasmids were co-injected into the vas-Cas9 line (BDSC #51323) in Bestgene, Inc. Injected larvae were crossed with balancer lines and PCR screened in F1 for single flies carrying the deletion. A mutant stock lacking the entire coding sequence of dpr13 was established from this single F1. The sgRNA target sequences used for dpr13null allele generation were CGATATAATCCACTTGATGC and ACGTAGCAGCTCCAGGATGT. Detailed protocols are available on request. All protocols in immunohistochemistry and DPX mounting were performed exactly as described in our previous study5. Immunofluorescence images were acquired using a Zeiss LSM 880 confocal microscope with 488-nm, 561-nm and 633-nm lasers using Zen digital imaging software with a Plan-Apochromat ×63/1.4 oil DIC M27 objective. Serial optical sections were obtained from whole-mount brains with a typical resolution of 1,024 μm × 1,024 μm, and 0.5-μm-thick optical sections. Image stacks were exported to either Fiji 2.0.0-rc-69/1.52k or Imaris 10.1 (Oxford Instruments) for level adjustment, cropping, removal of off-target brain regions and background noise, and 3D volume reconstructions. To measure the axo-dendritic overlap between LPLC2 axons and GF dendrites, confocal image stacks of colocalized LPLC2 glomeruli and GF dendrites were imported into Imaris 10.1 for 3D reconstruction using the Surfaces tool to create masks for membranes of presynaptic and postsynaptic neurons from the corresponding channels. A Surfaces detail value of 1 μm was used for both LPLC2 and GF surfaces to ensure accurate reconstruction. Background subtraction was applied with a diameter of the largest sphere that fits into the object set to 1 μm to minimize noise and nonspecific signals. A similar approach was used to measure the overlap between Brp puncta and the GF dendrites in STaR experiments (Extended Data Fig. 7), but the number of overlapping reconstructed surfaces was considered regardless of the overlap. To measure LoP4 dendritic branch length in sparsely labelled LPLC2 neurons, corresponding image stacks were imported into Fiji 2.0.0-rc-69/1.52k, rotated so that LoP3 and LoP4 branches were oriented antiparallel, and the distance from the point of bifurcation to the most distal tip of the LoP4 branch was measured along Lop4. Dorsoventral differences in Beat-VI and Dpr13 at the protein level were measured in FIJI 2.0.0-rc-69/1.52k. ROIs corresponding to the cell bodies dorsal or ventral LPLC2 clones were drawn manually. Mean grey value (average pixel intensity) was used as a proxy of the GFP fluorescence intensity and was measured for two ROIs per sample after background subtraction. Tissue staining, gelation and expansion for ExLSM protocols were adapted from Sanfilippo et al.73 with minor changes. After dissection, fixation and permeabilization, brains were stored in RNAse-free 0.5% PBST containing anti-GFP nanobody (N0304-At488-L, NanoTag Biotechnologies) overnight at 4 °C. All samples were subsequently processed using a protein and RNA retention ExM protocol with minor modifications73,74 and minor adjustments for the fly brain34. The HCR-FISH protocol was adapted from Wang et al.74 with minor optimizations for the fly brain34. Following digestion with proteinase K, gels with embedded brains were washed three times with PBS, transferred into 24-well plates (4624-24, Laguna Scientific) and digested with DNAse diluted in RDD buffer (RNase-Free DNase Set, 79254, Qiagen) to limit DAPI signal to RNA only and facilitate subsequent analysis for 2 h at 37 °C. After three washes in PBS, gels were equilibrated in probe hybridization buffer (Molecular Instruments) for 30 min at 37 °C, and then transferred to new 24-well plates containing custom-designed probes (Molecular Instruments) diluted in pre-warmed probe hybridization buffer (1 μl of 1 μM stock probe solution per 200 μl of buffer) and left shaking overnight at 37 °C. The following day, gels were washed four times with pre-warmed probe wash buffer (Molecular Instruments) for 20 min at 37 °C, then washed twice for 5 min with SSCT buffer (SSC with 0.05% Triton X-100; AM9763, Thermo Fisher) at room temperature and transferred to new 24-well plates with HCR amplification buffer (Molecular Instruments) for equilibration. Hairpins (HCR Amplifiers, Molecular Instruments) conjugated with AF546 or SeTau647 dyes were diluted in amplification buffer (2 μl of each hairpin per 100 μl of buffer), heat-activated in a thermal cycler (90 s at 95 °C), removed and kept for 30 min at room temperature in the dark. After 30 min, the hairpins were added to the 24-well plates with gels (300 μl per well) and incubated with shaking at room temperature in the dark for 3 h. The hairpin solution was then removed, and the gels were washed four times with SSCT and two times with SSC for 10 min at room temperature in the dark. Gels were subsequently stored at 4 °C in SSC until final expansion. Samples were expanded to approximately three time in 0.5× PBS containing 1:1,000 SYTO-DAPI (S11352, Thermo Fisher) at room temperature for 2 h before mounting onto PLL-coated coverslips (see description for DPX mounting above). The coverslips were then bonded with Bondic UV-curing adhesive (Bondic starter kit, Bondic) onto a custom-fabricated sample holder (Janelia Tech ID 2021-021) to be suspended vertically in the imaging chamber. Mounted samples were imaged in 0.5× PBS with 1:10,000 SYTO-DAPI after a minimum of 1 h of equilibration in the imaging chamber. Unexpanded gels were stored at 4 °C in 1X PBS + 0.02% sodium azide (S8032, Sigma-Aldrich) for up to 14 days before final expansion and imaging. Images were acquired on a Zeiss LS7 microscope equipped with 405-nm, 488-nm, 561-nm and 638-nm lasers. Illumination optics with a ×10/0.2 NA were used for excitation (400900-9020-000, Zeiss). Detection was performed using a W Plan-Apochromat ×20/1.0 DIC M27 water immersion objective (421452-9700-000, Zeiss). The LS7 optical zoom was set to 2.5×, resulting in a total magnification of ×50. DAPI and Alexa Fluor 546 dyes were simultaneously excited by the 405-nm and 561-nm laser lines, and emission light was separated by a dichroic mirror SBS LP 510 with emission filters BP 420-470 (404900-9312-000, Zeiss) and a modified BP 527/23 (ET672/23m, Chroma). Similarly, Alexa Fluor 488 and SeTau647 dyes were simultaneously excited via 488 nm and 638 nm, and the emission was split through a dichroic SBS LP 560 with emission filters BP 505-545 and LP 660 (404900-9318-000, Zeiss). To eliminate laser transmission, a 405/488/561/640 laser blocking filter (404900-9101-000, Zeiss) was added to the emission path. Images were captured using dual PCO.edge 4.2 detection modules (400100-9060-000, Zeiss) with a 50-ms exposure time. Filter and camera alignment were manually calibrated before each imaging session. Image volumes were acquired at optimal Z-step and light-sheet thickness, and the Pivot Scan feature was used to reduce illumination artefacts by sweeping the light sheet in the xy plane. The LS7 microscope was operated using ZEN Black 3.1 (v9.3.6.393). The acquired light-sheet z-stacks, stored in CZI format, were imported and pre-processed to remove noise and artefacts generated by the imaging modality. These artefacts included limited channel contrast, variations in contrast across images within a dataset, background noise fluctuations due to both intra-channel variations and inter-channel crosstalk, and localized brightness changes caused by varying fluorophore concentrations within and among stained nuclei. Pre-processing involved the following steps: full-scale contrast stretching to normalize luminosity across different channels, local background removal using a 3D Gaussian filter, a second full-scale contrast stretching to compensate for any contrast loss due to background removal, and a final median filter to eliminate any remaining localized noise. These pre-processed stacks served as the starting point for instance segmentation of the nuclei. First, nuclear centres were identified using a Laplacian-of-Gaussian filter. Then, the imaged volume was subdivided into 3D Voronoi cells, using the detected centres as seeds and Euclidean distance. Each cell contained one nucleus, which was segmented using a threshold obtained by minimizing an energy functional designed to find the optimal surface separating the nucleus from the surrounding cytoplasm. Once nuclei were segmented, the FISH puncta were identified within the associated 3D volume using a Laplacian-of-Gaussian filter, and only the puncta within the nucleus region and its immediate surrounding volume were counted. Pre-processed products, segmented nuclear features, associated FISH puncta, and their features were stored for further analysis. Puncta counts were normalized by the maximum count for each brain. The significance of gene expression relationships inferred from HCR-FISH (Fig. 2) was assessed using both a linear regression model and a multi-level negative binomial generalized linear mixed model, accounting for inter-animal heterogeneity: random effects were attributed to individual animals (Supplementary Table 2). The package was written in Python, is open source and is available for download on GitHub (https://github.com/avaccari/DrosophilaFISH). To assess gene expression patterns within VPN cell types, we used a combination of GAL4 and LexA binary expression systems. Expression of a LexAop-FRT-stop-FRT-membrane marker controlled by a cell-type-specific LexA driver remained blocked until the FRT-stop-FRT cassette was excised by Flippase, driven by a gene-specific T2A-GAL4 driver line75. In addition, a constitutive membrane marker controlled by the same LexA driver was used to visualize the entire cell population. The list of fly stocks used is available in Supplementary Table 1. To visualize single-cell morphology of LPLC2 dendrites in the LoP (Fig. 4) and to perform HCR-FISH analysis (Extended Data Fig. 4), we used MultiColor FlpOut76, a genetic tool for sparse labelling of individual cells within a population downstream of a given GAL4 driver. Twenty-four-hour-old pupae were heat shocked in a 37 °C water bath for 10 min to achieve the labelling of one LPLC2 neuron per hemibrain on average. To analyse the FlyWire connectome, we developed an open-source Python package, available on GitHub (https://github.com/avaccari/DrosophilaCon). The primary library used by DrosophilaCon to interface with the FlyWire connectome is fafbseg23,77 (v3.0.0). This package enables users to specify the labels of ‘source' and ‘target' neurons and generates a connectivity diagram where target neurons are colour-coded based on the total count of synapses with the source neurons. This information was returned as an adjacency table, providing the counts of synapses between each source–target pair. There are two versions of the synapse datasets: one filtered by synaptic cleft and one unfiltered. We used the unfiltered dataset because the filtered version applies a fixed threshold for distance, resulting in reduced synapse counts. The adjacency table was used to evaluate the total synapse counts for each target neuron. These counts were normalized by the maximum count observed across all target neurons. The mesh representation for each identified target neuron was downloaded from the FlyWire dataset, skeletonized for optimized processing and visualized with colour-coding corresponding to the normalized synapse count, allowing for comparison across different source–target pairs. All statistical analyses were performed using RStudio 1.4.1103, MATLAB 2022b or Prism 9.2.0 (GraphPad). Statistical tests were chosen based on data distribution, which was assessed using the Kolmogorov–Smirnov test in R with a P value threshold of less than 0.05 for normality. Two groups of normally distributed datasets were tested for statistically significant differences using unpaired t-tests with Welch's correction for non-identical variance. For comparisons involving more than two groups, we used either one-way ANOVA followed by Tukey's honestly significant difference test for post-hoc pairwise comparisons or the Kruskal–Wallis test followed by Dunn's multiple comparisons post-hoc test with Bonferroni correction. Binary data were compared using Chi-squared tests. All other statistical tests, number of replicates, significance levels and other elements of the statistical analysis (including measure of centre and error bars) have been reported within the corresponding figure legends. No data were excluded from the analysis unless specified in the corresponding Methods section. All neuroanatomical and behavioural measurements were taken from distinct samples (that is, individual brains or hemibrains and individual flies, one takeoff per fly). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The raw scRNA-seq data and the processed dataset are available at NCBI Gene Expression Omnibus: GSE291561. Raw calcium imaging data are available at: https://doi.org/10.17605/OSF.IO/Z7XFK. All other datasets generated and analysed during the current study (including FlyPEZ videos) are available from the corresponding authors on request. For further information regarding any resources and reagents, please contact S.L.Z. Source data are provided with this paper. The code for scRNA-seq analysis is available on GitHub (https://github.com/kurmangaliyev-lab). The code for the analysis of FlyWire and hemibrain connectome reconstructions is available on GitHub (https://github.com/avaccari/DrosophilaCon and https://github.com/avaccari/DrosophilaVPNWiring, respectively). The code used to analyse HCR-FISH data is available on GitHub (https://github.com/avaccari/DrosophilaFISH). The code related to calcium imaging experiments is available on the Open Science Framework (https://osf.io/z7xfk/). Shin, S., Crapse, T. B., Angeles, L., Mayo, J. P. & Sommer, M. A. in Encyclopedia of Neuroscience (eds Binder, M. D., Hirokawa, N. & Windhorst, U.) Coordinate transformations in the representation of spatial information. Visual projection neurons in the Drosophila lobula link feature detection to distinct behavioral programs. Dombrovski, M. et al. Synaptic gradients transform object location to action. Klapoetke, N. C. et al. Ultra-selective looming detection from radial motion opponency. Cembrowski, M. S. & Menon, V. Continuous variation within cell types of the nervous system. Ghahramani, Z., Wolpert, D. M. & Jordan, M. I. Generalization to local remappings of the visuomotor coordinate transformation. Buneo, C. A., Jarvis, M. R., Batista, A. P. & Andersen, R. A. Helmbrecht, T. O., dal Maschio, M., Donovan, J. C., Koutsouli, S. & Baier, H. Topography of a visuomotor transformation. & Jayaraman, V. Studying sensorimotor integration in insects. Critical periods shaping the social brain: a perspective from Drosophila. Zada, D. et al. Development of neural circuits for social motion perception in schooling fish. Processing of visually evoked innate fear by a non-canonical thalamic pathway. Cande, J. et al. Optogenetic dissection of descending behavioral control in Drosophila. Nern, A. et al. Connectome-driven neural inventory of a complete visual system. Klapoetke, N. C. et al. A functionally ordered visual feature map in the Drosophila brain. Aptekar, J. W., Keleş, M. F., Lu, P. M., Zolotova, N. M. & Frye, M. A. Neurons forming optic glomeruli compute figure–ground discriminations in Drosophila. The dipteran ‘giant fibre' pathway: neurons and signals. Von Reyn, C. R. et al. A spike-timing mechanism for action selection. Scheffer, L. K. et al. A connectome and analysis of the adult Drosophila central brain. Dorkenwald, S. et al. Neuronal wiring diagram of an adult brain. Schlegel, P. et al. Whole-brain annotation and multi-connectome cell typing of Drosophila. Card, G. & Dickinson, M. H. Visually mediated motor planning in the escape response of Drosophila. Williamson, R., Peek, M. Y., Breads, P., Coop, B. & Card, G. M. Tools for rapid high-resolution behavioral phenotyping of automatically isolated Drosophila. Ache, J. M. et al. Neural basis for looming size and velocity encoding in the Drosophila giant fiber escape pathway. Z., Yoo, J., Valdes-Aleman, J., Sanfilippo, P. & Zipursky, S. L. Transcriptional programs of circuit assembly in the Drosophila visual system. Kang, H. M. et al. Multiplexed droplet single-cell RNA-sequencing using natural genetic variation. The Drosophila melanogaster Genetic Reference Panel. Özel, M. N. et al. Neuronal diversity and convergence in a visual system developmental atlas. Özkan, E. et al. An extracellular interactome of immunoglobulin and LRR proteins reveals receptor–ligand networks. Sanes, J. R. & Zipursky, S. L. Synaptic specificity, recognition molecules, and assembly of neural circuits. Third-generation in situ hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust. Rapid reconstruction of neural circuits using tissue expansion and light sheet microscopy. & Dombrovski, M. FlySeg: an automated volumetric instance segmentation algorithm for dense cell populations in Drosophila melanogaster nervous system. In 2023 57th Asilomar Conference on Signals, Systems, and Computers (ed. Nagarkar-Jaiswal, S. et al. A library of MiMICs allows tagging of genes and reversible, spatial and temporal knockdown of proteins in Drosophila. Cosmanescu, F. et al. Neuron-subtype-specific expression, interaction affinities, and specificity determinants of DIP/Dpr cell recognition proteins. Li, H. et al. Deconstruction of the beaten path-sidestep interaction network provides insights into neuromuscular system development. Chen, Y. et al. Cell-type-specific labeling of synapses in vivo through synaptic tagging with recombination. Menon, K. P., Kulkarni, V., Shin-Ya, T., Anaya, M. & Zinn, K. Interactions between dpr11 and dip-y control election of amacrine neurons in Drosophila color ision circuits. Morano, N. C. et al. Members of the DIP and Dpr adhesion protein families use cis inhibition to shape neural development in Drosophila. Brain wiring determinants uncovered by integrating connectomes and transcriptomes. Biased cell adhesion organizes the Drosophila visual motion integration circuit. Brown, A. et al. Topographic mapping from the retina to the midbrain is controlled by relative but not absolute levels of EphA receptor signaling. Kita, E. M., Scott, E. K. & Goodhill, G. J. Topographic wiring of the retinotectal connection in zebrafish. Robbins, E. M. et al. SynCAM 1 adhesion dynamically regulates synapse number and impacts plasticity and learning. & Shen, K. Molecular mechanisms of synaptic specificity. Wernet, M. F. et al. Stochastic spineless expression creates the retinal mosaic for colour vision. Cheng, S. et al. Vision-dependent specification of cell types and function in the developing cortex. Zeng, H. What is a cell type and how to define it? Sievers, M. et al. Connectomic reconstruction of a cortical column. Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. Frighetto, G. & Frye, M. A. Columnar neurons support saccadic bar tracking in Drosophila. Gramates, L. S. et al. FlyBase: a guided tour of highlighted features. Li, H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Dictionary learning for integrative, multimodal and scalable single-cell analysis. & von Reyn, C. R. Azimuthal invariance to looming stimuli in the Drosophila giant fiber escape circuit. & Dickinson, M. H. A modular display system for insect behavioral neuroscience. Eye structure shapes neuron function in Drosophila motion vision. Bengtsson, H. R. MATLAB: read and write MAT files and call MATLAB from within R. CRAN https://doi.org/10.32614/CRAN.package.R.matlab (2018). Understanding mixed-effects models through data simulation. Application of the hierarchical bootstrap to multi-level data in neuroscience. Bates, D., Mächler, M., Bolker, B. M. & Walker, S. C. Fitting linear mixed-effects models using lme4. & Weisberg, S. An R companion to applied regression: appendices. Robust Regression in R (SAGE Publications, 2014). & Ripley B. D. boot: Bootstrap R (S-Plus) Functions. Davison, A. C. & Hinkley, D. V. Bootstrap Methods and their Applications (Cambridge Univ. Wickham H. ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag, 2016). Baddeley, A., Rubak, E. & Turner, R. Spatial Point Patterns: Methodology and Applications with R (Chapman and Hall/CRC Press, 2015). Xu, S. et al. Interactions between the Ig-superfamily proteins DIP-α and Dpr6/10 regulate assembly of neural circuits. Xu, H. et al. Sequence determinants of improved CRISPR sgRNA design. Ren, X. et al. Optimized gene editing technology for Drosophila melanogaster using germ line-specific Cas9. Sanfilippo, P. et al. Mapping of multiple neurotransmitter receptor subtypes and distinct protein complexes to the connectome. Wang, Y. et al. EASI-FISH for thick tissue defines lateral hypothalamus spatio-molecular organization. Lee, P. T. et al. A gene-specific T2A-GAL4 library for Drosophila. Nern, A., Pfeiffer, B. D. & Rubin, G. M. Optimized tools for multicolor stochastic labeling reveal diverse stereotyped cell arrangements in the fly visual system. Zheng, Z. et al. A complete electron microscopy volume of the brain of adult Drosophila melanogaster. Matsliah, A. et al. Neuronal parts list and wiring diagram for a visual system. & Jang, H. Ephys data (whole-cell recordings from the giant fiber) from Dombrovski et al. 2024. We thank the Bloomington Stock Center (US National Institutes of Health P40OD018537), Vienna Drosophila Research Center and Janelia Fly Facility for providing fly stocks; the UCLA Technology Center for Genomics and Bioinformatics for help with library preparation and scRNA-seq; the UCLA BSCRC FACS core for assistance with FACS purification; members of the FlyWire Consortium for connectome annotation in the FAFB dataset; all members of the Zipursky laboratory for advice and comments on the manuscript; and M. Scandola and S. Min Ha for advice on calcium imaging data analysis. This work was funded by the HHMI-HHWF Fellowship (to M.D. is an investigator of the Howard Hughes Medical Institute. 1a was created using BioRender (https://biorender.com). These authors contributed equally: Yixin Zang, Giovanni Frighetto, Andrea Vaccari, HyoJong Jang, Parmis S. Mirshahidi Mark Dombrovski, Parmis S. Mirshahidi, Fangming Xie, Piero Sanfilippo, Aadil Rehan, Roni H. Hussein, Pegah S. Mirshahidi, Catherine Lee, Aileen Morris & S. Lawrence Zipursky Department of Neuroscience, Howard Hughes Medical Institute, The Mortimer B. Zuckerman Mind Brain Behavior Institute, Columbia University, New York, NY, USA Department of Integrative Biology and Physiology, University of California Los Angeles, Los Angeles, CA, USA School of Biomedical Engineering, Science and Health Systems, Drexel University, Philadelphia, PA, USA HyoJong Jang, Bryce W. Hina & Catherine R. von Reyn Department of Biology, Brandeis University, Waltham, MA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar and Pegah S. Mirshahidi conducted the scRNA-seq experiments. and G.M.C., who also contributed to data interpretation. wrote the manuscript, with substantial input and feedback from G.M.C., Y.Z., G.F., Y.Z.K., M.A.F., H.J. All authors reviewed and edited the manuscript. Correspondence to Gwyneth M. Card or S. Lawrence Zipursky. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, GF-mediated short-mode (<7 ms sequence duration) takeoffs in response to frontal (0°) looms at various elevations. Numbers, total takeoffs (one takeoff per animal). b, Histograms showing the distribution of takeoff sequence durations in response to frontal (left) and lateral (right) looming stimuli at different elevations (detailed breakdown of data summarized in Fig. Short-mode and long-mode takeoffs are distinguished by the red dashed lines. c, Total takeoff percentage in response to frontal (0°) and lateral (90°) looms at various elevations. d, Top: whole-cell electrophysiological recordings of the GF in response to looming stimuli at different elevations in control animals. Black traces represent averaged responses of five animals (corresponding to Fig. 1f), green traces represent responses of individual animals. Middle: baseline region (2 s before the onset of stimulus) and response region (150 ms after the onset of stimulus) defined in the traces for analysis of the GF responses. Bottom: change of disk size over time. e, Pooled mean of integrated potentials for the GF in response to looming stimuli at different elevations across five repeated trials. Each dot represents a single fly (n = 5 biologically independent animals). Boxes: quartiles; whiskers: 1.5× interquartile range. Repeated-measures one-way ANOVA (P = 0.01) with post-hoc Sidak correction for multiple comparisons, *P = 0.0496 (−25° vs 25°). f, Violin plots of takeoff sequence durations for lateral stimuli (90°) at different elevations (0°, 45°, 77°) in controls and LPLC2-silenced animals. Short-mode and long-mode durations are separated by a red dashed line. g, Histograms showing the distribution of takeoff sequence durations in response to lateral looming stimuli at different elevations in controls and LPLC2-silenced animals (detailed breakdown of f). Short-mode and long-mode takeoffs are distinguished by the red dashed lines. h, Short-mode takeoff percentages at different elevations for controls and LPLC2-silenced animals. i, Total takeoff percentages at different elevations for controls and LPLC2-silenced animals. a-b, t-SNE plots of 72 h APF (a) and 96 h APF (b) datasets. LPLC2, LPLC1, and LC4 neurons were annotated based on the expression of known transcription factors (right panels). X indicates unknown ectopic cell types. c, Heatmaps of the expression patterns of the top 30 genes with the highest contribution (loading) to differentially expressed genes along Principal Component 1 (PC1, 15.2% variance explained) across LPLC1 neurons at 48 h APF (see Fig. Genes encoding cell recognition molecules (IgSF superfamily) are highlighted in bold. d, PCA plots of LPLC1 neurons at 48, 72, and 96 h APF, colored by the expression levels of two cell recognition molecules from c. e, PCA plots of LPLC2, LPLC1, and LC4 neurons at 48 h APF colored by genotype, age (early vs late collection), genotype (DGRP line), sex (male-specific transcript), and coverage. a-b, PCA plots of LPLC2, LPLC1 and LC4 neurons at 72 h APF (a) and 96 h APF (b) colored by genotype, age (early vs late collection), genotype (DGRP line), sex (male-specific transcript), and coverage. c-e, Scatter plots of LPLC2 cells (48 h APF) embedded in the first two principal components (PC1 and PC2) colored by their cluster labels (n = 2, inferred from K-means clustering). PCs are calculated using the top 1000 highly variable genes based on the actual data (c), after shuffling gene expression levels for each gene independently across all cells (d), and after shuffling each gene only within each cluster (e), respectively. Shuffling gene expression across all cells (d) disrupts this gradient, indicating that the observed differences reflect coordinated gene expression rather than uncorrelated variation. Shuffling within each cluster (e) disrupts the internal gradient of each cluster, creating artificial gaps not present in the original data (c), which suggests that the continuous gradient observed in the original data cannot be represented as a set of discrete clusters. f, Scatter plots of cells in PCA embeddings after shuffling genes within each cluster for an increasing number of fine-grained clusters (n = 2,5,30). Gap size is defined as the minimum distance in PC1 and PC2 space needed to connect 90% of cells into a single graph component. This means that a continuum can be approximated by a large number of discrete clusters. However, a small number of clusters (like 2) is insufficient to capture the continuous nature of the data. a-b, Light sheet projection of the expanded Drosophila optic lobe (48 h APF) with labeled LPLC2 nuclei and transcripts of dpr17 and beat-VI (a) and a single slice (0.5 μm) from the light sheet projection (b, zoomed into the yellow dashed rectangular region); arrows indicate LPLC2 somas expressing markedly different levels of dpr17 and beat-VI. n = 7 brains (one side per animal). c-d, Comparison of scRNA-seq (c) and HCR-FISH (d) measuring the correlation in expression for dpr17 and beat-VI across LPLC2 neurons at 48 h APF. Smoothed lines: linear regression fits; shaded bands: ± 95% confidence intervals. r, Spearman's rank correlation coefficient. e-f, Assessment of dpr13 (e) and beat-VI (f) expression levels in sparsely labeled LPLC2 neurons using HCR-FISH. Left: representative images (scale bar, 20 μm). Right, cell bodies of dorsal, ventral, and central LPLC2 neurons (scale bar, 5 μm). Insets: comparison of dpr13 (e) and beat-VI (f) puncta count in dorsal and ventral LPLC2 neurons. Error bars: mean ± 95% confidence intervals. Unpaired t-test with Welch's correction (two-sided). No statistical test was performed. Error bars, mean ± 95% confidence intervals. i-l, Assessment of Beat-VI and Dpr13 gradients at the protein level. Individual sparsely labeled LPLC2 neurons with dendrites sampling either dorsal, or ventral regions of the visual space, are co-localized with Dpr13 (i) and Beat-VI (k) constitutive protein traps (GFP-fusion proteins). GFP levels in dorsal vs ventral cell bodies are measured via mean pixel intensity (j, l). Error bars: means ± 95% confidence intervals. Unpaired t-test with Welch's correction. D, dorsal; L, lateral; M, medial. a, Subsets of LPLC2 neurons expressing CG44422 and Cad87A at 48 h APF. White arrows indicate ventral regions of the lobula lacking expression of both CG44422 and Cad87A. Dashed ovals, partial overlap of expression in somas, i.e., different LPLC2 neurons express different levels of CG44422 and Cad87A. n = 4. b, Positive correlation between expression levels of Cad87A, CG44422 and dpr13 (inferred from scRNA-seq), along PC1 across the LPLC2 population at 48 h APF, indicating that both CG44422 and Cad87A can be considered “dorsal” genes. d, Positive correlation between expression levels of stacl, Tsp42Ef and CG30419 (inferred from scRNA-seq), along PC1 across the LPLC2 population at 48 h APF. e, Retinotopically biased gene expression across LPLC1 neurons. Red, subsets of LPLC1 neurons expressing DIP-kappa, sdk and dpr17 across development (n = 9, 7, 6 for DIP-kappa; n = 8, 9, 5 for sdk; n = 4, 3, 3 for dpr17). f, Positive correlation between expression levels of DIP-kappa and sdk (top), and negative correlation between expression levels of DIP-kappa and dpr17 (bottom), (from scRNA-seq, Fig. 1h–m), along PC1 across the LPLC1 population at 48 h APF, reflecting the retinotopically biased expression of these genes in e. n, brains (one side per animal). Panels b, d, f: Smoothed lines represent the estimated mean expression trend. D, dorsal; L, lateral; A, anterior. a, Suggested model: to establish a synaptic gradient with LPLC2 based on dorsoventral expression gradient of any of the candidate molecules (Dpr13, Beat-VI, Dpr17), GF needs to express a molecular binding partner to recognize one or more of these molecules. b, Validation of VT049479-GAL4 expression in LPLC2 using GMR75G12-LexA as a reference. A complete overlap confirms that VT049479-GAL4 targets the entire LPLC2 population. c, Confocal projections of LPLC2 axon terminals and the GF dendrites in wild-type animals, as well as animals expressing control RNAi and two different DIP-ε RNAi in the GF (n = 19, 19, 11). d, Same as c for the DIP-ε rescue experiment (overexpression of DIP-ε cDNA in the GF in DIP-null background). n = 14, 15. f, Same as c for control and dpr13null animals. n = 14, 13. g, LPLC2-GF axo-dendritic overlap in control and dpr13null animals. Circles, brains (one side per animal). Error bars: means ± 95% confidence intervals. Unpaired t-test with Welch's correction (two-sided). h, Protein interaction map showing binding strength (affinity values are inversely proportional to edge thickness) between DIP-ε and multiple Dpr paralogs expressed in LPLC2. i, Expression levels of genes encoding DIP-ε binding Dpr paralogs in LPLC2 at 48 h and 96 h APF (inferred from scRNA-seq data generated in this study). j, Correlation between expression levels of genes encoding DIP-ε binding Dprs in LPLC2 at 48 h and 96 h APF (inferred from scRNA-seq), along PC1 across the LPLC2 population at 48 h and 96 h APF. Smoothed lines represent the estimated mean expression trend. n = 15, 15. l, FISH puncta count across LPLC2 neurons in controls and animals overexpressing DIP-ε in LPLC2. Circles, averaged values across all LPLC2 neurons per hemibrain. Error bars: mean ± 95% confidence intervals. Unpaired t-test with Welch's correction (two-sided). Panels b-f, k: n, brains (one side per animal tested); data represent single experiments. Panels b-f, k: scale bars, 5 μm. D, dorsal; L, lateral; M, medial. a, Genetic strategy (LexAop-STaR, Synaptic Tagging with Recombination) for generating sparsely labeled clones of LPLC2 neurons (with a fluorescent membrane marker) and visualizing their presynaptic sites (T-bars) with Brp-smGdP-V5. b, LPLC2 neurons labeled using LexAop-STaR. Left: confocal projection of LPLC2 glomerulus (axon terminals of ~30 LPLC2 neurons, labeled using long heat shock) co-localized with the GF dendrites (n = 6; scale bar, 5 μm). Middle: light sheet projection of a single LPLC2 neuron, imaged with 4x tissue expansion (n = 4; scale bar, 20 μm). Right: high-resolution image of the magnified view of presynaptic sites (T-bars) of a single LPLC2 neuron, co-localized with the GF dendrites (n = 4; scale bar, 1 μm). White arrows indicate individual T-bars, characterized by their distinctive ring-like shape and a typical diameter of 200–250 nm. n, brains (one neuron/optic glomerulus per brain). n, brains (one neuron per brain). d, Fraction of T-bars per single LPLC2 neuron overlapping with the GF dendrites. Error bars: mean ± 95% confidence intervals. Unpaired t-test with Welch's correction (two-sided). e, Same as in d, measuring the number of T-bars overlapping with the GF dendrites. NSP = 0.372; ****P = 0.00003. f, Same as in d, measuring the total number of T-bars per single LPLC2 neuron. In d–f, green lines indicate corresponding values inferred from the hemibrain connectome reconstruction. In d, the discrepancy between connectome-based and anatomy-based values likely reflects additional T-bars in the lobula/lobula plate not included in this analysis. Panels d-f: data from a single experiment. a, GF responses to looming stimuli, in r/v = 10, 20, 40 and 80 ms. Control (average: black, individual fly: grey) and DIP-εnull (average: orange, individual fly: light orange) traces are overlayed. Looming stimulus profile over time is displayed below the GF responses. b, Quantification of expansion peak amplitudes in a from individual flies. n, biologically independent animals; circles, mean values of two recordings per animal. r/v = 10 ms: U = 8, NSP = 0.149. r/v = 20 ms: U = 6, NSP = 0.07323. r/v = 40 ms: U = 4, *P = 0.0303. r/v = 80 ms: U = 14, NSP = 0.6389. c, Same as a for controls (grey) and animals overexpressing DIP-ε RNAi in the GF (light blue). n = 5 for controls and DIP-ε RNAi. d, Quantification of expansion peak amplitudes in c from individual flies. n, biologically independent animals (two trials per animal); circles, mean values of two recordings per animal. a, Left, Histograms showing the distribution of takeoff sequence durations at different stimulus elevations (0°, 45°, and 77°) in wild-type and DIP-εnull flies. Short-mode and long-mode takeoffs are distinguished by red dashed lines. Right, total takeoff percentages at different elevations for wild-type and DIP-εnull flies. b, Same as a for controls and flies expressing DIP-ε RNAi in the GF. c, Same as a for controls and flies overexpressing dpr13 in LPLC2 neurons. d, Same as a for controls and dpr13null flies. Red, subsets of LPLC2 neurons expressing beat-VI at 48 h APF (top) and 96 h APF (bottom). b-c, Comparison of axo-dendritic overlap between LPLC2 and the GF in controls (n = 12) vs side-IInull (n = 12) animals (b), and in animals expressing control RNAi (n = 13, 14) vs two beat-VI RNAi (n = 12, 12) in LPLC2 (c). Circles, brains (one side per animal). Error bars: mean ± 95% confidence intervals. Unpaired t-test with Welch's correction (two-sided). Left panels: posterior views of the LPLC2 population with dashed rectangles indicating the location of cross-sections for dorsal, central, and ventral subsets of LPLC2. e, Same as d for two different LPLC2 GAL4 driver lines and two different beat-VI RNAi lines. f, Confocal projections of LPLC2 dendrites (entire LPLC2 population labeled, dorsal and ventral cross-sections compared) in the lobula plate in control animals (left, n = 4), animals overexpressing beat-VI cDNA (middle, n = 4), and animals overexpressing beat-VI cDNA while also expressing beat-VI RNAi (right, n = 3). Yellow arrowheads indicate changes in LoP4 dendritic density. g, Length of LoP4 dendritic branches for dorsal, central and ventral LPLC2 neurons in control (n = 12, 14, 16 dorsal, central, and ventral) and LPLC2 > UAS-beat-VI (n = 15, 14, 12 dorsal, central, and ventral) flies. n, individual neurons (one cell per brain). Error bars: mean ± 95% confidence intervals. Unpaired t-test with Welch's correction (two-sided). Differences in the length of LoP4 dendritic branches across ventral LPLC2 neurons are highlighted. Panels d–f, h: data represent single experiments. Panels d-f: n, brains (one side per animal). D, dorsal; V, ventral; P, posterior; M, medial; L, lateral. a, 3D reconstruction of LPLC2 neurons from a Z-stack taken using the two-photon microscope in a head-fixed fly (lateral view, scale bar: 10 μm). Dashed lines highlight the two Z-planes where LPLC2 dendrites responded to dark looming at positions 16 (red) and 40 (blue) on the LED display (top). White solid lines approximately define the reference neuropils. Schematic of the procedure used to identify the putative neurons in the fly connectome stimulated by two representative grid positions (bottom, see Methods). b, Polar plots of the peak responses to moving dark edges in dorsal (14) and ventral (30) regions in two representative flies (color-coded by position). c, Polar plots of the average peak responses to moving dark edges in dorsal (16) and ventral (40) regions for control, LPLC2>beat-VI RNAi, and LPLC2 > UAS-beat-VI flies. d, Polar plots of the average peak responses to moving dark edges in aggregated dorsal (above the equator) and ventral (below the equator) regions for control, LPLC2>beat-VI RNAi, and LPLC2 > UAS-beat-VI flies. e, Average calcium transient in response to a dark looming in dorsal (above the equator) and ventral (below the equator) regions for control, LPLC2>beat-VI RNAi, and LPLC2 > UAS-beat-VI flies. f, Average calcium transients in response to dark edges moving in 24 (from 0° to 345°) different orientations for control, LPLC2>beat-VI RNAi, and LPLC2 > UAS-beat-VI flies. Dorsal (left) and ventral (right) regions. Panels c-d: n values represent biologically independent animals (multiple trials per animal). This file provides a summary of the statistical tests conducted for each experiment, including exact P-values Open Access  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by/4.0/. Dombrovski, M., Zang, Y., Frighetto, G. et al. Molecular gradients shape synaptic specificity of a visuomotor transformation. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09063-2'>Maternal iron deficiency causes male-to-female sex reversal in mouse embryos</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 15:04:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Ferrous iron (Fe2+) is essential in all eukaryotic cells for various oxidoreductase reactions, including the demethylation of DNA and proteins. Histone demethylation is required for normal epigenetic regulation of the Y-chromosomal sex-determining gene Sry in developing gonads during male sex determination1,2. Here we investigate the potential connection between iron metabolism, histone demethylation and sex determination in mammals. We found that Fe2+-producing pathways are substantially activated in mouse embryonic gonads during the sex-determining period. Chelation of iron in cultured XY gonads reduced the level of KDM3A-mediated H3K9 demethylation of Sry, mostly abolished Sry expression and caused the gonads to express ovarian markers. In vivo, conditional deletion of the gene Tfrc—which is required for iron incorporation—in fetal XY gonadal somatic cells, or acute pharmaceutical suppression of available iron in pregnant mice, resulted in male-to-female gonadal sex reversal in a proportion of offspring, highlighting the pivotal role of iron metabolism in male sex determination. Finally, long-term feeding of pregnant mice with a low-iron diet, when combined with a heterozygous variant of Kdm3a that by itself has no observable effect, suppressed Sry expression and caused male-to-female sex reversal in some of the progeny, revealing a connection between maternal dietary iron and fetal developmental outcomes. This is a preview of subscription content, access via your institution Get Nature+, our best-value online-access subscription Receive 51 print issues and online access Prices may be subject to local taxes which are calculated during checkout The scRNA-seq data are available at the GEO under GSE184708 (ref. The RNA-seq data have been uploaded to the GEO under GSE287387. All other data are available in the manuscript or its Supplementary Information. Source data are provided with this paper. Kuroki, S. et al. Epigenetic regulation of mouse sex determination by the histone demethylase Jmjd1a. Rescuing the aberrant sex development of H3K9 demethylase Jmjd1a-deficient mice by modulating H3K9 methylation balance. Sinclair, A. H. et al. A gene from the human sex-determining region encodes a protein with homology to a conserved DNA-binding motif. Male development of chromosomally female mice transgenic for Sry. Kashimada, K. & Koopman, P. Sry: the master switch in mammalian sex determination. Stévant, I. et al. Deciphering cell lineage specification during male sex determination with single-cell RNA sequencing. Galy, B., Conrad, M. & Muckenthaler, M. Mechanisms controlling cellular and systemic iron homeostasis. Origin, specification and differentiation of a rare supporting-like lineage in the developing mouse gonad. Guo, J. et al. Single-cell analysis of the developing human testis reveals somatic niche cell specification and fetal germline stem cell establishment. Hirayama, T., Okuda, K. & Nagasawa, H. A highly selective turn-on fluorescent probe for iron(ii) to visualize labile iron in living cells. Qian, Z. M., Li, H., Sun, H. & Ho, K. Targeted drug delivery via the transferrin receptor-mediated endocytosis pathway. Levy, J. E., Jin, O., Fujiwara, Y., Kuo, F. & Andrews, N. C. Transferrin receptor is necessary for development of erythrocytes and the nervous system. Han, Z. et al. Iron homeostasis determines fate of human pluripotent stem cells via glycerophospholipids–epigenetic circuit. Influence on spatiotemporal patterns of a male-specific Sox9 activation by ectopic Sry expression during early phases of testis differentiation in mice. Takase, S. et al. A specific G9a inhibitor unveils BGLT3 lncRNA as a universal mediator of chemically induced fetal globin gene expression. Harima, H. et al. Deferasirox, a novel oral iron chelator, shows antiproliferative activity against pancreatic cancer in vitro and in vivo. Eggers, S. & Sinclair, A. Mammalian sex determination—insights from humans and mice. Sex and the circuitry: progress toward a systems-level understanding of vertebrate sex determination. & McElreavey, K. Human sex-determination and disorders of sex-development (DSD). Capel, B. Vertebrate sex determination: evolutionary plasticity of a fundamental switch. Okashita, N., Maeda, R. & Tachibana, M. CDYL reinforces male gonadal sex determination through epigenetically repressing Wnt4 transcription in mice. Wilkinson, N. & Pantopoulos, K. The IRP/IRE system in vivo: insights from mouse models. Zhou, Z. D. & Tan, E. K. Iron regulatory protein (IRP)-iron responsive element (IRE) signaling pathway in human neurodegenerative diseases. Beyer, S., Kristensen, M. M., Jensen, K. S., Johansen, J. V. & Staller, P. The histone demethylases JMJD1A and JMJD2B are transcriptional targets of hypoxia-inducible factor HIF. Krieg, A. J. et al. Regulation of the histone demethylase JMJD1A by hypoxia-inducible factor 1α enhances hypoxic gene expression and tumor growth. Dynamic change of chromatin conformation in response to hypoxia enhances the expression of GLUT3 (SLC2A3) by cooperative interaction of hypoxia-inducible factor 1 and KDM3A. Disorders of sex development and Diamond–Blackfan anemia: is there an association? Granada, M. L. & Audi, L. The laboratory in the multidisciplinary diagnosis of differences or disorders of sex development (DSD): (III) Biochemical and genetic markers in the 46,XY; (IV) Proposals for the differential diagnosis of DSD. Hashimoto, M. & Takemoto, T. Electroporation enables the efficient mRNA delivery into the mouse zygotes and facilitates CRISPR/Cas9-based genome editing. Integrated analysis of multimodal single-cell data. Okashita, N. et al. PRDM14 promotes active DNA demethylation through the ten-eleven translocation (TET)-mediated base excision repair pathway in embryonic stem cells. Maeda, R. & Tachibana, M. HP1 maintains protein stability of H3K9 methyltransferases and demethylases. Okashita, N., Kuroki, S., Maeda, R. & Tachibana, M. TET2 catalyzes active DNA demethylation of the Sry promoter and enhances its expression. The mouse Sry locus harbors a cryptic exon that is essential for male sex determination. Mukaide, T. et al. Histological detection of catalytic ferrous iron with the selective turn-on fluorescent probe RhoNox-1 in a Fenton reaction-based rat renal carcinogenesis model. Establishment of testis-specific SOX9 activation requires high-glucose metabolism in mouse sex differentiation. The embryonic ontogeny of the gonadal somatic cells in mice and monkeys. The authors acknowledge the NGS core facility of the Genome Information Research Center at the Research Institute for Microbial Diseases of Osaka University for support with RNA-seq and data analysis. This study was supported by the Chubei Itoh Foundation (N.O. ), the Tojuro Iijima Foundation for Food Science and Technology (N.O. ), the Food Science Institute Foundation (N.O. ), the Terumo Life Science Foundation (M.T. ), the Uehara Memorial Foundation (M.T) and JSPS KAKENHI (grant numbers 23K05732 (N.O. Laboratory of Epigenome Dynamics, Graduate School of Frontier Biosciences, Osaka University, Suita, Japan Naoki Okashita, Ryo Maeda, Shunsuke Kuroki, Kyona Sasaki, Yoko Uno & Makoto Tachibana Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar interpreted data and wrote the manuscript. performed most of the experiments. performed all bioinformatic data analysis. performed RNA-expression analysis in Sry-expressing pre-Sertoli cells. supported the analysis and visualization of RNA-seq data. contributed to the establishment of Tfrc-mutant mice and the observation of postnatal mouse gonads. All authors edited and provided comments on the manuscript. The authors declare no competing interests. Nature thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. a, Schematic representation for the isolation of NR5A1+ gonadal somatic cells. Gonadal somatic cells expressing the cell surface marker protein CD271, under the control of a CD271 transgene driven by the Nr5a1 promoter, were immunomagnetically isolated from gonad/mesonephros as described previously1. b, Gene-expression profiles of NR5A1+ and NR5A1− cells examined by quantitative mRNA expression analysis. Schematic illustration of mitochondrial iron metabolism (left). FLVCR1B, locales in the mitochondria, regulates haem export into the cytosol. Ftmt/FTMT catalyses the oxidation of Fe2+ to Fe3+ and stores iron. NR5A1+ and NR5A1− cells were separated from XY gonad/mesonephros at E11.5, and then applied into quantitative mRNA expression analysis. Expression profiles of Slc25a37, Slc25a28, Flvcr1, and Ftmt are shown on the right. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. All differences between means with P < 0.01 are indicated; two-tailed paired t-tests. c, Box-and-whisker plot showing expression (log-normalized counts) of Nr5a1, Kdm3a, Sry, Tfrc, Scara5, Slc11a2, Steap3, Ncoa4, Hmox1, Slc40a1, and Heph of XX or XY gonads in annotated cell populations. Data were extracted from scRNA-seq analysis of XX or XY embryos at E11.5 (GEO: GSE184708). Numbers of cells examined are shown in the legend. Boxes encompass the interquartiles (25th–75th percentiles), The horizontal lines are medians, dots are outliers, and range bars show the maximum and minimum values excluding outliers. d, Comparison of expression levels of KDM3A, SRY, and iron-metabolism genes in human XY NR5A1+ and NR5A1− cells. Data were extracted from scRNA-seq analysis of human embryonic stages at 6, 7, 8, and 12 weeks post-fertilization (GEO: GSE143356). The embryonic stages from 6 to 8 weeks post-fertilization are highlighted as the sex-determining period. a, Experimental scheme to generate Tfrc deletion allele. Protein and genomic structures of TFR1/Tfrc are represented at the top. The position of gRNA target sites is indicated in yellow. Sequence with knock-in two loxP sites (red) and sequence cleaved by Cre recombinase are shown in the middle and bottom, respectively. b, Immunoblot analysis to compare TFR1 expression in E9.5 whole embryos of the indicated genotypes. GAPDH is used as an internal control. c, Immunoblot analysis to compare TFR1 and H-Ferritin expression in E11.5 NR5A1+ cells of the indicated genotypes. The relative values were calculated based on the signal of α-tubulin. d, Effect of Tfrc deficiency on the number of gonadal somatic cells in developing embryos. NR5A1+ cells were isolated from E11.5 embryos of the indicated genotypes and counted. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. e, Expression profiles of Sry, Scara5, Slc11a2, Steap3, Ncoa4 and Hmox1 in E11.5 NR5A1+ cells of the indicated genotypes. NR5A1+ cells were purified from E11.5 embryos and subjected to quantitative RT–qPCR analysis. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. All differences between means with P < 0.01 are indicated; two-tailed paired t-tests. f, External genitalia (Upper) and internal genitalia (Lower) of mice of the indicated genotypes. Frequencies of female-to-male sex reversal are presented in the lower right corner. Sex development frequencies are shown in the lower right corner. Red bar indicates mice carrying two ovaries. The number of mice examined is shown above the bars. a, Development an in vitro culture system to monitor gonadal sex development. Pairs of aorta/gonad/mesonephros/mesenchyme were isolated from E10.8 XY or XX embryos, and then cultured for 14.4 h, 19.2 h and 24 h. NR5A1+ cells were isolated from XY gonads at each time point, and then applied into quantitative mRNA expression analysis. Sry expression reached a peak after 19.2 h of culture. Further 24 h couture (total 43.2 h culture) gave rise to sexually differentiated gonads with either male-type SOX9+ cells or female-type FOXL2+ cells. b, Effect of DFO treatment on the number of gonadal somatic cells in in vitro cultured gonads. NR5A1+ cells were isolated from XY gonads after 19.2-h incubation. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. c, Compared expression levels of H/L-Ferritin between DFO-treated and untreated gonadal somatic cells. NR5A1+ cells were isolated from XY gonads after 19.2-h incubation and introduced into immunoblot. The relative values were calculated based on the signal of α-Tubulin. d, PCA of mRNA expression in NR5A1+ cells from in vitro cultured gonads (19.2 h culture, labelled as IVC) and in utero developed gonads (labelled as E11.5). e, mRNA heat map of genes involved in gonadogenesis and/or Sry regulation in NR5A1+ cells from in vitro cultured gonads (19.2 h, labelled as IVC) and in utero developed gonads (labelled as E11.5). f, Venn diagram illustrates the overlap between genes downregulated by DFO treatment in in vitro cultured NR5A1+ cells and KDM3A target genes in in utero developed NR5A1+ cells. Blue circle indicates 2677 genes downregulated in DFO-treated NR5A1+ cells (TPM > 0; fold change > 2). Merged 40 genes represent KDM3A target genes that were downregulated in DFO-treated NR5A1+ cells. g, RNA-seq based gene-expression values (TPM) of JmjC family in in vitro cultured gonads. NR5A1+ cells were isolated from XY gonads after 19.2-h incubation. Data are presented as mean ± SD. All differences between means with FDR < 0.01 are indicated. h, Compared protein expression levels of KDM3A between DFO-treated and untreated gonadal somatic cells. NR5A1+ cells were isolated from XY gonads after 19.2-h incubation and introduced into immunoblot. The relative values were calculated based on the signal of GAPDH. i, Compared H3K9me2 levels between DFO-treated and untreated gonadal somatic cells. NR5A1+ cells were isolated from XY and XX gonads after 19.2-h incubation and introduced into immunoblot. The relative values were calculated based on the signal of H3.1/2. j, Quantification of H3K9me2 levels at the KDM3A target gene loci Sry, Myl7, and Apoc1 in DFO-treated gonadal somatic cells. NR5A1+ cells were isolated from XY gonads after 19.2-h incubation and introduced into ChIP–qPCR. ChIP experiment was performed independently three times. Data are presented as mean ± s.d. All differences between means with P < 0.01 are indicated; two-tailed paired t-tests. a, Immunoblot analysis of H/L-ferritin in E11.5 XY NR5A1+ cells in embryos carried by DFX-treated mothers and those carried by control mothers. GAPDH is used as an internal control. b, Effect of treating pregnant mothers with DFX an the number of gonadal somatic cells in developing embryos in utero. NR5A1+ cells were isolated from E11.5 XY embryos carried by DFX-treated mothers and those carried by control mothers. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. c, Expression profiles of Sry, Nr5a1, Gata4, Wt1, Kdm3a, Tfrc, Hmox1, Myl7, and Apoc1 in E11.5 XY NR5A1+ cells in the embryos carried by DFX-treated mothers and those carried by control mothers. NR5A1+ cells were purified from E11.5 embryos and subjected to RT–qPCR analysis. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. All differences between means with P < 0.01 are indicated; two-tailed paired t-tests. a, Changes in body weight (left), haemoglobin levels (middle), and red blood cells (right) in IDD-fed mothers and CD-fed mothers were monitored during the pre-feeding (0–4 week) and gestation (4–6 week) periods. Numbers of examined female mice are shown above the markers. Data are presented as mean ± s.d. b, Giemsa staining of a peripheral blood smear from pregnant IDD-fed mothers and CD-fed mothers when embryos were at E11.5. c, Immunoblot analysis of TFR1 and H/L-Ferritin in E11.5 XY gonadal NR5A1+ cells in the embryos carried by IDD-fed mothers and CD-fed mothers. GAPDH is used as an internal control. d, Effect of mother's iron deficiency anaemia induced by IDD feeding on the number of gonadal somatic cells in the corresponding embryos. NR5A1+ cells were isolated from E11.5 XY gonads in the embryos carried by IDD-fed mothers and those carried by CD-fed mothers and counted. Numbers of embryos examined are shown above the bars. Data are presented as mean ± s.d. Uncropped images of immunoblots. Ponceau S staining and uncropped blot images corresponding to the main figures Gene-expression values (TPM) from RNA-seq analysis. Summary of TPM values for genes examined in the RNA-seq dataset Primers used in this study. Nucleotide sequence information of primer sets used in this study Antibodies used in this study. Information on antibodies used in this study Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law. Okashita, N., Maeda, R., Kuroki, S. et al. Maternal iron deficiency causes male-to-female sex reversal in mouse embryos. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41586-025-09071-2'>Concurrent loss of the Y chromosome in cancer and T cells impacts outcome</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 15:04:13
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Loss of the Y chromosome (LOY) in peripheral blood mononuclear cells (PBMCs) is the most common somatic alteration in men and is associated with higher mortality from epithelial cancers1,2,3. In tumours, epithelial LOY is also associated with poor survival4,5,6,7. This raises several fundamental questions, such as why LOY in PBMCs drives cancer mortality and whether there is a relationship between LOY in PBMCs, PBMC-derived immune cells and cancer cells (and, if so, what its consequences are). We sought to answer these questions through a comprehensive pan-cancer analysis of bulk and single-cell RNA sequencing data from 29 human tumour types, along with autochthonous and syngeneic mouse models. In human and mouse tumours, malignant epithelial cells had the highest LOY prevalence, yet LOY was also present in tumour stromal and immune cells, with LOY in malignant epithelial cells predicting LOY in benign cells. LOY also correlated between paired tumour and PBMC samples from patients. Among benign cells, LOY induced the strongest shift in CD4+ and CD8+ T cells, with both showing transcriptomic signatures of immunosuppression. Furthermore, the magnitude of LOY in epithelial cells, CD4+ T cells and CD8+ T cells independently predicts survival, with tumours exhibiting concurrent epithelial and T cell LOY having the worst outcomes. Here we establish a model that links LOY in immune cells to LOY in malignant cells, which may explain in part why LOY in PBMCs is associated with increased cancer mortality. LOY in tumour cells is associated with poor prognosis4,5,6. This was found to be partially due to immune evasion from T cell exhaustion in some tumour types7. LOY in PBMCs is the most common somatic alteration in healthy men and has been associated with increased risk8,9 and mortality from cancer1,2,3. As PBMCs consist of immune cells including lymphocytes (T cells, B cells and natural killer (NK) cells), monocytes and dendritic cells10, these data raise the question of whether LOY in PBMCs, PBMC-derived tumour immune cells and malignant cells are related, and whether this relationship drives increased cancer mortality. Here we address this question through a comprehensive pan-cancer analysis of the genomic, histopathological, biological and clinical consequences of LOY in benign and malignant cells, using bulk and single-cell RNA sequencing (scRNA-seq) from human tumour types. Further, to gain additional insights that are difficult to obtain from human data and to generalize our findings, we also examine numerous data types from autochthonous, syngeneic and human xenograft models of cancer in mice. Our data show that a substantial proportion of non-malignant cells, including both immune and non-immune cell subtypes in the tumour microenvironment (TME), have LOY. The presence of LOY in non-malignant cells is associated strongly with malignant epithelial cell LOY in human data and in mouse models. Furthermore, the proportion of LOY in PBMCs is correlated with that in immune cells and malignant cells in the TME. When present in benign cells, LOY induces the strongest phenotypic shift in CD4+ T and CD8+ T cells, with both cell types showing signs of immunosuppression. Clinically, the prognosis of patients with LOY epithelial tumours deteriorated further when LOY CD4+ T and CD8+ T cells were present in the TME, with tumour cells and T cells independently affecting prognosis. Given the prevalence of LOY in healthy older men, these findings may also have implications for tumour-infiltrating T cell (TIL) and chimeric antigen receptor (CAR) T cell therapies. More broadly, they lay the conceptual foundation for an experimentally tractable model that may explain in part why men with LOY in PBMCs are at higher risk for cancer death. To investigate the impact of LOY at a pan-cancer level, we developed a ‘Y chromosome transcriptional signature' (YchrS) that predicts DNA-based LOY classification (LOY/Y− (LOYDNA) versus wild type/Y+ (WTYDNA))5. Differential analysis of Y chromosome genes between LOYDNA and WTYDNA tumours from the Cancer Genome Atlas (TCGA) with available exome capture data identified 26 genes with lower expression in LOYDNA tumours, which we then filtered further using the Human Protein Atlas to focus on nine genes (DDX3Y, UTY, KDM5D, USP9Y, ZFY, RPS4Y1, TMSB4Y, EIF1AY and NLGN4Y) that are expressed stably at both the RNA and protein levels across several male tissues11 (Fig. a, Expression levels of 284 Y chromosome genes and clinical parameters in 4,127 tumours across 29 types, grouped by LOY status confirmed by DNA analysis5. Genes used to develop YchrS are indicated. b, YchrS score distribution across 29 cancer types (top) and corresponding proportions of LOYDNA and WTYDNA tumours (bottom). Box limits, quartiles; whiskers, data range within 1.5× interquartile range (dots, outliers); centre lines, median. Sample sizes: KIRP (kidney renal papillary cell carcinoma, n = 203), ESCA (oesophageal carcinoma, 115), KICH (37), LUSC (lung squamous cell carcinoma, 284), STAD (212), ACC (adrenocortical carcinoma, 28), PAAD (pancreatic adenocarcinoma, 92), HNSC (272), UVM (uveal melanoma, 44), CHOL (16), KIRC (kidney renal clear cell carcinoma, 316), DLBC (lymphoid neoplasm diffuse large B-cell lymphoma, 20), LUAD (lung adenocarcinoma, 183), READ (rectum adenocarcinoma, 63), COAD (colon adenocarcinoma, 155), BLCA (221), SKCM (skin cutaneous melanoma, 249), MESO (mesothelioma, 61), SARC (sarcoma, 85), LIHC (214), TGCT (testicular germ cell tumours, 120), LGG (brain lower grade glioma, 262), LAML (acute myeloid leukemia, 63), THCA (thyroid carcinoma, 127), PRAD (prostate adenocarcinoma, 457), GBM (glioblastoma multiforme, 89), THYM (thymoma, 58), PCPG (pheochromocytoma and paraganglioma, 73), BRCA (breast invasive carcinoma, 8). c, Area under the receiver operating characteristic curves comparison of YchrS and YwholeS in predicting LOYDNA and WTYDNA (AUC differences tested by two-sided roc.test in the pROC R package). Dots, individual patients; box presentation as in b. Two-sided Wilcoxon rank sum test. Data presentation as in e. Two-sided Wilcoxon rank sum test. g, PCC of YchrS with CTA expression across 26 cancers. Bars and colours represent PCC in each cancer type; sizes represent −log10 adjusted P value, with darker edges denoting significance (P < 0.05). h, Kaplan–Meier survival curves for OS and DSS, stratified by YchrS (LOYBR versus WTYBR). Statistics calculated using a univariate Cox proportional hazard model. HPV, human papilloma virus; LOYBR/WTYBR, LOY/WTY sample by bulk RNA-seq; PCC, Pearson correlation coefficient. Mean YchrS scores are anticorrelated with LOYDNA frequency across cancer types, and, compared with a signature including all Y chromosome genes (YwholeS), YchrS was more predictive of LOYDNA (area under the curve (AUC)) 0.98 versus 0.90; P < 0.01; Fig. We further validated the YchrS signature using data from 778 male cell lines in the Cancer Cell Line Encyclopedia (CCLE), integrating YchrS with copy number alteration (CNA) data from whole-exome sequencing (WES). Cell lines were classified into LOYBR (LOY identified using bulk RNA-seq evaluation) and WTYBR (WTY chromosome identified using bulk RNA-seq evaluation) groups based on the mean YchrS value. Examining CNAs, we found LOYBR cell lines have extremely low average Y chromosome-specific CNAs compared with WTYBR lines, indicating there was prominent LOY in the former group (Extended Data Fig. Next, we used TCGA data to examine whether YchrS was associated with any specific clinical characteristics (Supplementary Table 1). Patient age is correlated inversely with YchrS (Fig. 1d), and we also found significant racial differences in the degree of tumour LOY, with Black/African American patients having the lowest YchrS (Fig. There is also an inverse association between YchrS and smoking: in head and neck squamous cell carcinoma (HNSC) and kidney chromophobe renal cell carcinoma (KICH), patients who have never smoked exhibit significantly higher YchrS scores than current or former smokers (Extended Data Fig. Human papilloma virus (HPV)-positive patients with HNSC have higher YchrS scores than HPV-negative patients (Fig. 1f), but no association is observed between YchrS and hepatitis B virus (HBV) or hepatitis C virus (HCV) in liver hepatocellular carcinoma (LIHC), or between YchrS and Epstein–Barr virus (EBV) in stomach adenocarcinoma (STAD; Extended Data Fig. We found a strong negative correlation between the YchrS and cancer-testis antigen (CTA) signatures (Fig. 1g)—proteins that are normally restricted to germ cells but often overexpressed in cancers and linked to worse survival12. We noted that the YchrS distribution appeared to be bimodal and that the mean served as a better separator of the two peaks while also performing better than the median in a receiver operating characteristic (ROC) analysis, with the mean falling closer to the top-left corner (Extended Data Fig. Given this, we next investigated whether mean YchrS could stratify patient outcomes. Indeed, patients with LOYBR have worse outcomes, including poorer overall survival (OS) and disease-specific survival (DSS; Fig. When analysing TCGA data, we found that the YchrS signature effectively stratifies pan-cancer survival outcomes in white and Asian patients (Extended Data Fig. However, this stratification was not significant in Black patients (Extended Data Fig. Similarly, at the genetic ancestry13,14 level, the YchrS signature stratifies pan-cancer survival in European, African-admixed and East Asian patients, but not in African patients (Extended Data Fig. Moreover, a multivariate Cox proportional hazards analysis found that the WTYBR/LOYBR classification is independent from genetic ancestry, self-reported race and tumour type in predicting OS, suggesting that it has a foundational role in cancer outcomes (Extended Data Table 1). Finally, we also found that higher expression of each of the nine YchrS genes predicted better OS and DSS in TCGA data (Extended Data Fig. To investigate why men with tumour LOY have worse survival1,4,5,7, we examined genomic and transcriptional signatures in TCGA tumours based on their LOYBR classification (Supplementary Table 3). Tumour mutation burden (TMB) and tumour neoantigen burden (TNB) are associated with worse survival without immunotherapy but better immune checkpoint blockade (ICB) response15,16,17,18. We found that LOYBR tumours have elevated TMB and TNB at both the small insertion/deletion mutation (Indel) and single nucleotide variant (SNV) levels, as well as higher mutation rates (both non-silent and silent mutations; Extended Data Fig. YchrS scores are also correlated inversely with aneuploidy and ‘fraction altered' across several cancers (Extended Data Fig. Moreover, homologous recombination defects (HRD, including in HRD score19, loss of heterozygosity (LOH)20, large-scale state transitions21 and telomeric allelic imbalance (TAI)22) are higher in LOYBR tumours (Extended Data Fig. LOYBR tumours also have higher signature scores for homologous recombination, mismatch repair, DNA damage response and DNA repair. In terms of phenotype, LOYBR tumours have higher stemness signature scores, encompassing DNA methylation-based stemness, epigenetically regulated DNA methylation-based stemness, differentially methylated probe-based stemness and enhancer element methylation-based stemness23 (Extended Data Fig. LOYBR tumours also have higher signature scores for proliferation and activation of proliferation-related pathways, including DNA replication, cell cycle regulation, mitotic spindle, KRAS signalling, G2M checkpoint, E2F targets and apoptosis. They also have higher scores for oncogenesis-related pathways such as MYC, MTORC1 and epithelial-mesenchymal transition (EMT; Extended Data Fig. However, on the basis of gene expression, they demonstrate decreased activity of the Hedgehog signalling pathway. Beyond the pathways listed above, LOYBR tumours have elevated angiogenesis, glycolysis and hypoxia pathway scores, suggesting a distinct TME (Extended Data Fig. Furthermore, they display diminished androgen response and elevated oestrogen response scores (Extended Data Fig. Finally, LOYBR tumours have increased expression of immune checkpoints and co-inhibitory genes such as CTLA4, PDCD1, TIGIT, LAG3, CD27, IL2RA, HAVCR2, CD40, CD274 and PDCD1LG2 (Extended Data Fig. 3h), indicating they have an immunosuppressive TME. To study the implications of LOY at the cell-type level, we consolidated scRNA-seq data from 17 epithelial cancer types (Fig. Because we found that YchrS often misclassified low-RNA-count cells as having LOY, we developed a machine-learning model to predict single-cell-level LOY or WT status in scRNA-seq data (LOYSCR and WTYSCR), by training a Random Forest model to predict male versus female cells from adjacent normal scRNA-seq samples using expression levels of the nine YchrS genes (Fig. When the model was used to classify male tumour cells, those predicted to have LOY had minimal expression of Y chromosome genes, but had normal overall sequencing counts, and had normal expression of genes not known to be affected by LOY such as GAPDH, Y gene paralogues on the X chromosome and genes located on other chromosomes (Fig. To further validate model accuracy, we analysed a colorectal cancer (CRC) dataset24 comprised of 23 tumours with paired WES, bulk RNA-seq and scRNA-seq data. Here we found a strong positive correlation between average Y chromosome CNA (WES) and YchrS scores (bulk RNA), and negative correlations between LOYSCR cell proportions and the former two measures (Extended Data Fig. a, scRNA-seq database constituted by 497,055 cells from 105 male tumour samples, coloured by six principal cell types (left) and 12 cancer types (right). b, LOYSCR predictions for cells in a. c, Mean expression (colour) and percentage of expressing cells (size) for the nine Y chromosome genes used in the Random Forest model, their X chromosome paralogues and GAPDH across different cell types and LOYSCR status. d, Database of 157,029 male epithelial cells, coloured by cancer type. e, Normalized enrichment scores (NES) for the top five upregulated and 15 downregulated pathways in LOYSCR versus WTYSCR epithelial cells, categorized into four functional groups. Bar colour denotes −log10-adjusted P value, calculated by gseGO in the clusterProfiler package. f, Glycolysis (left) and hypoxia (right) signatures in LOYSCR (n = 73,576) versus WTYSCR (n = 83,453) epithelial cells. g, MHC class I gene expression in LOYSCR versus WTYSCR epithelial cells. Dot plot (bottom) presentation as in c. Bars (top) show log2FC, with colours indicating −log10-adjusted P value, calculated with sc.tl.rank_genes_group in the scanpy package using two‐sided Wilcoxon rank‐sum test with Benjamini–Hochberg correction. h,i, Scores for proliferation-related (h) and genomic instability-related (i) pathways in LOYSCR versus WTYSCR epithelial cells. Dot colour represents mean scaled pathway score; dot size represents −log10 P value (two-sided Wilcoxon rank sum tests, Benjamini–Hochberg correction). NES > 0 indicates pathway enrichment in Y-KO cells. Statistics calculated as in e. k, Expression of genes related to genomic instability, cell cycle and antigen presentation in CRISPR Y-Scr (n = 3) and CRISPR Y-KO (n = 3) cells. Gene set enrichment analysis (GSEA) revealed that epithelial cells (Fig. 2d) with LOYSCR downregulate pathways associated with immune recognition (major histocompatibility complex (MHC)), aerobic energy metabolism (regulation of aerobic respiration, aerobic electron transport chain, mitochondrial ATP synthesis coupled electron transport, ATP synthesis coupled electron transport) and pathways related to protein synthesis (ribosomal large subunit biogenesis, cytoplasmic translation; Fig. LOYSCR epithelial cells also upregulate pathways associated with glycolysis (glycolytic process through fructose-6-phosphate and glucose catabolic process) and hypoxia, altogether indicating a metabolic shift consistent with tumorigenesis and progression25,26 (Fig. Finally, LOYSCR epithelial cells also have lowered expression of specific MHC class I and MHC class II genes (Fig. 6a), indicating that these cells may have immune evasion capabilities27,28. In terms of oncogenic pathway signature expression, LOYSCR epithelial cells have elevated KRAS, MYC, RAS, TGFβ and oestrogen response activity (Extended Data Fig. They also display upregulation of proliferation-related pathways, such as the cell cycle, DNA replication, mitotic spindle formation and the G2M checkpoint (Fig. As for genomic instability, LOYSCR epithelial cells express high levels of signatures corresponding to repair of double-strand breaks (DSB) through homologous recombination, non-homologous end joining and the p53 pathway (Fig. Inferred copy number variation (inferCNV) analysis also indicated elevated CNAs in LOYSCR epithelial cells, particularly those from bladder cancers (BLCA) and cholangiocarcinoma (CHOL) (Extended Data Fig. To develop a mouse model to study LOY mechanistically, we performed RNA-seq on fluorescence-activated cell sorting (FACS)-isolated CD45− cells from tumours generated by subcutaneous injection of CRISPR induced Y-KO (confirmed by WES to have lost the Y chromosome) (Extended Data Fig. 7a) or control cells that were infected with a scrambled CRISPR guide (Y-Scr). Excluding Y-linked genes to prevent bias, we identified LOY-associated changes through differential expression analysis (Supplementary Table 6). A LOY phenotype score, calculated by comparing scaled signature scores of genes that were upregulated or downregulated between Y-KO and Y-Scr CD45− tumour fractions, was higher in LOYBR BLCA samples than in WTYBR BLCA samples in the TCGA (Extended Data Fig. 7b), confirming the relevance of our mouse model findings. The LOY signature was also higher in LOYBR tumours from other cancer types (Extended Data Fig. To study the genes altered in LOY cancer cells, we used in vitro7 RNA-seq data on MB49 cells with CRISPR–Cas9-mediated selective elimination of the Y chromosome (Y-KO). GSEA pathway enrichment analysis of the Y-KO and Y-Scr in vitro RNA-seq data indicated that the former cells have increased genomic instability (DSB repair) and cell cycle regulation and reduced antigen presentation (Fig. Specifically, key DNA repair genes are overexpressed in Y-KO cells (Fig. Genes that regulate chromosome segregation, such as Bub1b, Aurkb, Ccnb1 and Plk1, are also upregulated in Y-KO cells. Furthermore, cell cycle regulation-related genes (Cdk2, Ccna2, Ccnb1, Ccnd1, Mki67) show upregulation, whereas antigen-presentation genes (H2-D1, B2m) show downregulation (Fig. Finally, WES revealed higher numbers of large-scale chromosomal alterations in Y-KO cells. These include unique single-nucleotide polymorphisms (SNPs) and indels, which appear preferentially enriched in intronic and intergenic regions, suggesting that LOY may predispose cells to widespread genomic alterations rather than targeting specific genomic loci (Extended Data Fig. In studying LOYSCR cell percentages in human scRNA-seq data, we noticed that, whereas LOYSCR frequency is the highest in epithelial cells, other cell types also show significant LOYSCR (Fig. This finding was supported when we examined a signature (YchrSmus) representing Y mouse chromosome genes (Ddx3y, Uty, Kdm5d and Usp9y) in scRNA-seq data from autochthonous mouse bladder tumours generated with the N-butyl-N-(4-hydroxybutyl)-nitrosamine (BBN) carcinogen protocol29,30 (Fig.3a (bottom) and Extended Data Fig. To determine whether there was a relationship between epithelial LOY and the prevalence of LOY in non-epithelial cells in the same tumour, we measured correlations of LOYSCR cell proportions between different cell types. Including all tumour types in our human scRNA-seq dataset, we found that the rates of LOYSCR in any one cell type are highly predictive of LOYSCR rates in other cells (Fig. a, LOYSCR and WTYSCR cell proportions across principal cell populations in human tumours (top) and mouse bladder tumours induced by BBN (bottom). b, Correlation of cell group proportions across human tumours, defined by cell type and LOYSCR status. Cell type proportions across samples are shown in yellow (x axis, left; y axis, top), while LOYSCR cell proportions are in blue (x axis, right; y axis, bottom). Within groups, cell types are ordered alphabetically. c, Proportion of Y chromosome signal-positive nuclei per tissue core (FISH experiment), categorized by CD45 status. Data are presented as mean ± s.e.m., normalized to the normal male Y chromosome signal. Values <0 or >1 arise from normalization and do not reflect absolute signal levels. d, Y chromosome gene (Kdm5d, Uty, Eif2s3y, Ddx3y) DNA levels in each cell line before injection, confirming LOY status. e, DNA qPCR analysis of Y chromosome genes in immune cells (CD45+) across MB49 WTY (Y+), MB49 LOY (Y−) and MB49 clone 5 (C.5) groups. f, DNA qPCR analysis of Y chromosome genes in epithelial (CD45−, TdTomato+, left) and immune cells (CD45+, right) from CRISPR Y-Scr and CRISPR Y-KO groups. g, Genomic DNA qPCR was performed to assess Y chromosome genes (Kdm5d, Uty, Ssty1, Ssty2) in CD3+ T cells from PBMCs of mice bearing CRISPR Y-Scr or CRISPR Y-KO tumours. Fold change (FC) was calculated relative to the average in vivo values and normalized to both endogenous controls separately. Each dot denotes pooled samples from six independent tumours. We also used the inferCNV algorithm on the human scRNA-seq data to assess whether non-malignant cells in the TME, particularly macrophages, display tumour-like copy number variation (CNV) signatures, which could indicate the presence of ambient or phagocytosed nucleic acids from malignant cells. Epithelial cells displayed elevated CNV scores, confirming their distinct genomic profiles, whereas all examined immune cell types, including macrophages, CD8+ T cells and CD4+ T cells, showed minimal CNV signals (Extended Data Fig. These findings suggest that the LOY signals of benign cells are unlikely to be the result of debris from malignant cells. This also indicates that the YchrS scores used in the survival curves above (Fig. 1h) and DNA-based LOY evaluations5 used to assess malignant cell LOY may reflect the overall LOY levels from a variety of cell types within the TME and not just cancer cells. Next, we analysed orthogonal datasets to test for evidence of LOY in benign cells. scRNA-seq data on mouse BLCA (MB49) LOY tumours and normal bladder samples from male and female C57BL/6N mice as controls31 revealed that benign cell types within LOY tumours display higher LOY levels compared with normal bladders (Extended Data Fig. Furthermore, scRNA-seq or single-nucleus RNA sequencing of CD45+ and CD45− sorted mouse hepatocellular carcinoma (HCC) tumour populations indicate that LOY is present in both malignant cells and immune cells whose sequencing libraries were prepared separately from the malignant cell fraction (Extended Data Fig. We also analysed available human scRNA-seq data from CD45-based FACS-sorted HNSC32, CHOL33 and BLCA34 samples (Supplementary Table 7), which revealed that the CD45+ (immune) populations exhibit LOY (Extended Data Fig. Notably, we also observed a positive correlation between LOY proportions in CD45− and CD45+ cells from the same patients in these data (Extended Data Fig. scRNA-seq data from a CRC xenograft model35, in which human LOY CRC cells were subcutaneously injected into immunocompromised mice, also displayed higher LOY levels in benign mouse cells in the TME compared with corresponding cells from normal male mouse tissues (Extended Data Fig. Further, on intact tissue sections, we combined a clinical grade fluorescence in situ hybridization (FISH) assay used for prenatal sex determination with CD45 immunohistochemistry (IHC) to examine LOY in 33 BLCA specimens and found that both CD45− and CD45+ cells in cancer samples exhibit LOY (Fig. As a final validation that stromal and immune cells in the TME display LOY, and to better investigate whether malignant cell LOY promotes LOY in benign populations, we injected lineage-related Y+ and LOY MB49 mouse BLCA cells subcutaneously into C57Bl/6N mice. We used three models for this experiment: (1) naturally occurring Y+ and LOY cells7; (2) a LOY clonal line; and (3) CRISPR Y-KO and CRISPR Y-Scr cells7. For each model, the DNA levels of four Y chromosome genes (Kdm5d, Uty, Eif2s3y and Ddx3y) were assessed, before subcutaneous injection of the cells into immune-competent mice (Fig. After formation, tumours were then disrupted mechanically, and cells separated on the basis of surface CD45 expression using double selection by magnetic beads and FACS (Supplementary Fig. As the genetically engineered Y-KO and Y-Scr lines co-express a TdTomato fluorescent marker and the Cas9 gene, we also sorted these cells by TdTomato expression to minimize epithelial cell contamination in the immune cell fractions. We then used DNA qPCR to detect the Cas9 gene in each sorted population, confirming the purity of the different compartments (Extended Data Fig. As expected, only the TdTomato+ epithelial cells showed a high copy number of Cas9, whereas the immune compartments exhibited negligible Cas9 levels, indicating successful separation (Extended Data Fig. Next, the four Y chromosome genes used to validate LOY above were analysed via DNA qPCR to assess their presence in CD45+ immune and CD45− non-immune cell populations. There were lower DNA levels of Y chromosome genes in the CD45+ cells from tumours generated by LOY MB49 cells compared with those generated with Y+ lines (Fig. Furthermore, when this experiment was repeated with a sorting strategy that allowed T cell LOY to be examined specifically (CD45+CD11b−B220−CD3+TCRb+), we found that infiltrating CD3+TCRb+ T cells in Y-KO tumours had increased LOY compared with those from Y-Scr tumours (Extended Data Fig. However, no evidence of LOY was found in PBMCs from mice bearing Y+ or LOY tumours (Fig. Given the incidence of LOY in PBMCs and its relationship to increased cancer mortality1, other key questions remain, such as how LOY levels in human PBMCs compare with those in tumour-infiltrating immune cells. If the latter exhibit higher LOY levels than PBMCs, that would suggest either preferential accumulation of PBMCs with LOY into tumours or tumour-induced LOY in Y+ PBMCs that have infiltrated the tumour. To begin answering this question, we examined a dataset from patients with renal cell carcinoma (RCC)36 that included tumour and PBMC samples from the same patients. Our analysis indicated that T cells and myeloid cells in tumours exhibit significantly higher LOY proportions than their matched PBMC counterparts (Fig. In both the above and one additional HNSC dataset37,38 on CD45-sorted tumour cells and matched PBMCs, we confirmed that LOY proportions were correlated in the tumour and blood (Fig. We also analysed specific immune cell subtypes and found that tumour immune cells, especially T and B cells, had significantly higher LOY proportions than PBMCs (Fig. To put the mouse data in context, PBMCs from mice treated with the BBN protocol showed no evidence of LOY (Extended Data Fig. a, LOYSCR cell proportions across different immune cell types in paired tumour and blood samples from seven patients with RCC. Paired Wilcoxon rank sum test was used for comparing the LOY proportions of specific cell types from tumour and blood samples, with sample pairs shown at the bottom right of each plot. b, Database of 79,858 cells from 19 patients with HNSC, coloured by five principal cell types. No cluster of epithelial cells was observed. c, LOYSCR and WTYSCR cell proportions of paired TIL and PBMC samples from patients in b. d, LOYSCR cell proportions in matched TIL and PBMC samples from patients in b; each dot represents an individual patient. e, LOYSCR cell proportions in different immune cell types in paired TIL and PBMC samples from 19 patients in b; each dot represents one patient. Paired Wilcoxon rank sum test was used to compare the LOY proportions of specific cell types from TIL and PBMC samples, with sample pairs shown at the bottom right of each plot. To examine the molecular profiles associated with LOY in non-epithelial cell populations in the TME, we conducted a cell-type-specific differential expression and pathway analysis (Fig. GSEA using the Gene Ontology database (https://www.geneontology.org/) found that LOY has the greatest impact on CD8+ and CD4+ T cells (Fig. LOYSCR CD4+ T cells have elevated expression of several immune checkpoint and co-stimulatory molecules (for example, IL2RA, HAVCR2, ICOS, TNFRSF9, TNFRSF4, CTLA4 and TIGIT (Fig. 5b)) as well as increased expression of signatures of exhaustion and the T regulatory (Treg) cell phenotype (Fig. 5c), while displaying lower expression of CD4+ T cell specific signatures of naive and T cell receptor (TCR) signalling. Conversely, LOYSCR CD8+ T cells show reduced expression of HAVCR2, BTLA, TNFRSF9, LAG3, CTLA4, PDCD1, TIGIT, ITGB2 and TOX, as well as signatures of cytotoxicity, TCR signalling and exhaustion (Fig. LOYSCR CD8+ T cells also increase expression of a naive CD8+ T cell signature, indicating these cells are not contributing to an active anti-tumour immune reaction. a, Summary of GSEA analysis between LOYSCR and WTYSCR cells across cell types. Colours represent NES of differentially expressed pathways (adjusted P < 0.05 in any one cell type), calculated by gseGO in the clusterProfiler package. b, Immune checkpoint gene expression in LOYSCR and WTYSCR CD4+ T cells. Dot colour represents mean scaled gene expression; dot size indicates the fraction of expressing cells. Bars above plots show log2FC, coloured by −log10 P value, calculated with sc.tl.rank_genes_group in the scanpy package using two‐sided Wilcoxon rank‐sum test with Benjamini–Hochberg correction. c, Scaled signatures scores of functional pathways in LOYSCR (n = 12,802) and WTYSCR (n = 49,206) CD4+ T cells. ; significance determined via two-sided Wilcoxon rank sum tests. d, Immune checkpoint gene expression in LOYSCR and WTYSCR CD8+ T cells. Data presentation as in b. e, Scaled signatures scores of functional pathways in LOYSCR (n = 19,159) and WTYSCR (n = 69,767) CD8+ T cells. Data presentation as in c. f, Kaplan–Meier survival curves for 4,127 TCGA male samples, stratified by LOYSCR and WTYSCR signature scores from CD4+ T cells, CD8+ T cells and epithelial cells. Statistics (P values, hazard ratio, CI) were calculated using a univariate Cox proportional hazards model. g, Kaplan–Meier survival curves for TCGA male samples based on combinatorial scores of LOYSCR epithelial and CD4+ or CD8+ T cell signatures. P values for pairwise comparisons through log-rank tests are shown at the bottom. h, Nomogram predicting less than 2-year, less than 5-year and less than 8-year OS probabilities based on LOYSCR CD8+ T cell, CD4+ T cell, epithelial signature score levels and age. Bottom, cumulative scores and survival estimates. Example: TCGA-02-2483 (43-year-old, high LOYSCR epithelial/CD8+ score, low CD4+ score) with a total score of 223, predicting 15% (less than 2 years), 28.6% (less than 5 years) and 39.1% (less than 8 years) mortality risk. To further explore the relationship between T cell LOY and T cell exhaustion, we conducted in vitro experiments in which mouse primary T cells were subjected to prolonged activation with anti-CD3 and anti-CD28 coated plates over a 3-week period. Although LOY was not observed in T cells after extended culture and stimulation, markers of exhaustion such as Pdcd1, Tim3 and Ctla4 were induced, whereas the naive T cell marker Tcf7 was reduced as expected (Extended Data Fig. This indicates that chronic activation in vitro alone does not drive LOY in T cells. In addition, we also analysed publicly available transcriptomic datasets from Giles et al.39, in which gp33-specific CD8+ T cells from TCR-transgenic mice were subjected to acute (LCMV Armstrong) and chronic (LCMV Clone 13) LCMV infections. Over the course of the infection, T cells in the chronic model exhibited upregulation of canonical exhaustion markers such as Tox, Pdcd1 and Ctla4 and downregulation of Tcf7 (Extended Data Fig. As in the previous experiment, expression levels of Y-linked genes (Uty, Kdm5d, Ddx3y, Usp9y) remained unchanged in exhausted T cells compared with their non-exhausted counterparts, indicating that chronic activation alone does not drive LOY in T cells (Extended Data Fig. 5a), we further examined their role in patient prognosis and whether combined loss of the Y chromosome in epithelial cells and in these immune cell types may have clinical implications. To this end, we developed cell-type-specific LOYSCR and WTYSCR gene signatures and conducted Kaplan–Meier survival analysis using TCGA data (Supplementary Table 9). This analysis suggests that LOYSCR CD4+ T cells, CD8+ T cells and epithelial cells are associated with patient survival outcomes, including OS and DSS (Fig. Moreover, patients with high predicted levels of LOY epithelial cells in conjunction with elevated predicted levels of LOY CD4+ T or CD8+ T cells have the poorest survival outcomes, whereas those with low predicted levels of LOY epithelial and T cells show the most favourable survival (Fig. To validate the findings above, we analysed a publicly available liver cancer scRNA-seq dataset that includes 94 male patients with matched survival data40. By calculating the proportions of LOYSCR epithelial, CD4+ and CD8+ T cells in each sample and correlating these proportions with long-term outcomes, we find patients with co-existent high proportions of LOYSCR in epithelial cells and in CD8+ or CD4+ T cells have the worst survival outcomes compared with patients with co-existent low proportions of LOYSCR in all cell types (Extended Data Fig. Patients with co-existent high epithelial and CD8+ T cell LOYSCR show worse outcomes than patients with high epithelial cell LOYSCR alone. This pattern was consistent with our bulk pan-cancer RNA-seq findings, reinforcing the link between LOYSCR epithelial and T cells and poor patient prognosis in cancer (Extended Data Fig. We next conducted a multivariate Cox analysis to determine whether the LOYSCR epithelial, LOYSCR CD8+ T cell, and LOYSCR CD4+ T cell signatures independently contribute to patient outcomes. Owing to the unavailability of tumour stage, smoking and race-related data for certain cancer types, and recognizing the previously published correlation between age and LOY, only age was included in our analysis. As the LOYSCR epithelial signature shows non-linearity, we categorized patients into low or high LOYSCR epithelial, CD8+ T cell or CD4+ T cell signature groups, based on corresponding cut-offs determined from the Kaplan–Meier plot analysis above (Fig. 5f), where we observed that all variables are independent risk factors for poor OS (Extended Data Table 2). Finally, we constructed a nomogram that incorporates age, as well as LOYSCR signature scores for epithelial, CD4+ and CD8+ T cells to provide ‘proof of principle' for a potentially clinically beneficial tool that could forecast survival rates among patients with cancer (Fig. Each patient is assigned a score corresponding to each prognostic parameter, wherein a higher total score indicated a poorer prognosis for the patient. Additionally, calibration plots demonstrated that the performance of the nomogram closely resembles that of an ideal model (Extended Data Fig. LOY in a portion of PBMCs is associated with increased risk of death from many cancer types1,2,3. Several possible models could explain how this happens. For example, LOY in PBMCs may be a marker of genome damage due to environment (smoking, carcinogens and so on) or aging41, with such damage driving both LOY and tumour progression in the malignant cells. This latter notion is supported by the finding that LOY in tumours is associated with poor prognosis4,5,6,7. It is also possible that LOY in PBMCs, and thus in key immune cells, may cause these cells to be less competent in restricting tumour growth. Our study extends previous findings that LOY contributes to immune evasion, tumour growth and poorer survival in BLCA1,5,7 to a pan-cancer context. Our WES findings revealed that LOY cells harbour a higher burden of large-scale chromosomal alterations compared with WTY cells. This observation supports a model in which LOY could impair genomic maintenance systems, leading to a ‘vicious cycle' of escalating chromosomal aberrations. Our work found significant LOY in benign cells in the TME and that the prevalence of LOY in malignant epithelial cells is predictive of LOY in other cell types within the same tumour. Speculatively, if LOY tumour cells express specific chemokines, cytokines or adhesion molecules that preferentially attract LOY immune cells, this would explain how a relatively small percentage of PBMCs with LOY can have an impact on cancer death1,2,3. We also show evidence for another possible mechanism to explain this correlation, namely the ability of LOY epithelial cells to induce LOY in benign cells, including T cells. One mechanism underlying such a phenomenon could involve tumour-derived extracellular vehicles42 containing pro-inflammatory or genome-altering cargo transferred from LOY cancer cells that stimulate molecular changes leading to LOY in neighbouring benign cells. Notably, the ‘preferential attraction' and ‘induction' models are not mutually exclusive and could be operating in the same tumour. Treg cells are characterized by their immunosuppressive properties43. Our analysis also revealed that LOY CD8+ T cells exhibit decreased activation, TCR-signalling and cytotoxicity signatures, indicating reduced contribution to an immune response. These findings allow us to posit a speculative model in whichk tumours with high LOY are more aggressive owing to increased levels of dysfunctional LOY T cells and enrichment of LOY Treg cells. Finding LOY in T cells has also potentially important implications for TIL and CAR-T cell therapy in patients with cancer44,45. Given the features of LOY CD8+ T cells compared with those without this change, enrichment or selection of cells for such engineering without knowing their Y chromosome status may lead to variable clinical efficacy and confound clinical trial outcomes. This caveat would be particularly relevant in tumours with high levels of epithelial LOY, which have a proportionately high T cell LOY. This possible dual role of LOY, driving both genomic instability in tumour cells and functional exhaustion in immune cells, provides a plausible explanation for its association with shortened survival. There is an absence of relevant scRNA-seq data for certain tumour types, primarily because of incomplete gender information or quality control issues. Related to this, we were able to identify only one scRNA-seq dataset in which there were enough male patients with long-term follow up to allow analysis of the consequences of combinatorial LOY in epithelial and T cell compartments. Thus, we had to use bulk RNA-seq deconvolution based on cell-type-specific signatures of LOY to perform this analysis in a pan-cancer manner. Also, the nomogram that was developed here needs thorough validation on additional datasets and in specific tumour types. Although our RNA-seq analyses reveal significant changes in gene expression and pathway enrichment, it is important to recognize that these findings are correlative. Functional assays are essential to establish causative relationships between LOY and the observed biological processes. Laboratory experiments are also needed to substantiate the observed associations, particularly concerning the mechanisms by which interactions between epithelial cancer cells with LOY and benign cells in the TME lead to LOY in the latter. In summary, our research lays a new conceptual framework that will serve to accelerate the nascent field of Y chromosome-focused cancer research. These results provide new insights such as showing the profound impact of epithelial LOY on reshaping the TME, the prevalence of LOY in various benign cell types in a tumour, the ability of LOY epithelial cells to promote LOY in benign cells (including T cells) and the consequences of LOY in benign cells. Finally, our study provides evidence that LOY in both epithelial and non-epithelial cells are independent prognostic markers of outcomes, demonstrating the impact of multicellular LOY in cancer. In this study, we used bulk RNA-seq, WES somatic mutation data, and clinical data (Supplementary Table 1) sourced from TCGA project (https://portal.gdc.cancer.gov/). The data matrices obtained from UCSC Xena (https://xena.ucsc.edu/) have been standardized, normalized and corrected for batch effects and platform differences. Additionally, mutation data generated by the PanCancer Atlas consortium (https://gdc.cancer.gov/about-data/publications/pancanatlas) were incorporated. A total of 29 tumour types and 4,127 male participants were included in our analysis. Survival outcome metrics, including OS, OS time, DSS and DSS time were calculated as in Liu et al.46 (Supplementary Table 2). We used the DESeq2 R package (v.1.42.1) to uncover Y chromosome gene expression differences between LOYDNA and WTYDNA47. This gene set resulted in the LOY prediction signature. Subsequently, we conducted a gene set intersection analysis with a gene set in the male-specific region of the Y chromosome seen expressed across 24 human tissues. This analysis required that genes exceeded 0.1 reads per kilobase of transcript per million reads mapped per tissue or that they had presence in IHC data from the Human Protein Atlas RNA-seq11. This approach identified nine signature genes making up our Y chromosome signature (YchrS): DDX3Y, UTY, KDM5D, USP9Y, ZFY, RPS4Y1, TMSB4Y, EIF1AY and NLGN4Y. Based on single-sample GSEA (ssGSEA)48 conducted with the GSVA49 R package (v.1.44.5), we observed that patients with low levels of YchrS exhibited characteristics similar to those of people with LOYDNA, whereas those with high YchrS levels resembled people with an intact Y chromosome (WTYDNA). We partitioned all patients into low YchrS group (LOYBR) and high YchrS group (WTYBR) with a mean value cut-off. Additionally, a similar approach was applied to analyse the Ywhole signature (YwholeS), using a signature comprising all Y chromosome genes. To validate the nine-gene Ychr Signature (YchrS), we downloaded the batch-corrected transcriptomic data and corresponding CNA data for 778 male cancer cell lines from the CCLE project (https://depmap.org/portal/ccle/). YchrS was calculated by the same method as for TCGA data. Cell lines with a YchrS lower than the mean value were categorized as LOYBR, otherwise they were called WTYBR. Average CNA for chromosome i in each cell line was calculated using the following formula to evaluate its integrity: n is the total number of segments in chromosome i. Consensus ancestry for TCGA cases was obtained from ref. 14, determined by combining ancestry inference from five independent classification methods using SNP array and/or WES data. Aneuploidy scores for TCGA cases were obtained from ref. Arm-level statistics were calculated using the GISTIC (v.2.0)51 copy number significance software. These scores were derived by tallying the total number of amplified or deleted arms, collectively termed ‘altered'. Samples were initially categorized by tumour type, alteration type (amplification or deletion) and chromosome arm. Subsequently, samples were clustered on the basis of specific arm attributes, and arms were classified as altered if part of a cluster had a mean fraction altered of at least 80%. Intratumour heterogeneity used to generate DNA damage scores was determined by ABSOLUTE52. ABSOLUTE analysed segmentation data from Affymetrix SNP6.0 arrays and variant calls from the MC3 variant file. We used two measures to assess HRD. The first was derived by Knijnenburg et al.53 and quantifies HRD by aggregating separate metrics of genomic scarring: large (more than 15 Mb) non-arm-level regions with LOH20, large-scale state transitions (breaks between adjacent segments of greater than 10 Mb, LST21 and subtelomeric regions with allelic imbalance. The second measure, introduced by Telli et al.19, incorporates LOH and TAI22. The HRD score for samples analysed via custom hybridization sequencing assay were computed using reads covering SNP positions54. The HRD score was determined as the unweighted sum of LOH, TAI and LST, represented as: HRD score = LOH + TAI + LST. We screened samples using mRNA and DNA methylation profiles to compute four stemness scores: DNA methylation-based stemness score, epigenetically regulated DNA methylation-based stemness score, differentially methylated probe-based stemness score and enhancer elements/DNA methylation-based stemness score as outlined in previous studies23. Two methodologies were used to identify potential neoantigens arising from SNVs and Indels. For SNVs, somatic nonsynonymous coding SNVs were identified and minimal peptides encompassing mutation sites were extracted, followed by prediction of binding to autologous MHC using NetMHCpan v.3.0 (ref. On the other hand, Indel variants meeting specific criteria were extracted, and downstream protein sequences obtained to generate nine-mer peptides. These peptides were then evaluated for their ability to bind MHC molecules using the pVAC-Seq v.4.0.8 pipeline56, also using NetMHCpan v.3.0. The mutation data, specifically encompassing missense mutations and nonsense mutations, were obtained from the PanCancer Atlas consortium and used in this analysis. The dataset (https://gdc.cancer.gov/about-data/publications/pancanatlas) underwent filtering, requiring mutation calls to be generated by two or more mutation callers (NCALLERS > 1). To study the consequences of LOY, a literature review was performed, and a variety of tumour-associated signatures gathered (Supplementary Table 3)57,58,59,60,61,62. Each signature was assessed using ssGSEA implemented through the GSVA R package49 (v.1.44.5). Scoring methods are in the figure legends. To validate the CRISPR–Cas9-mediated Y-KO MB49 (Y-KO)-derived gene expression signature in human BLCA data, we first performed a differential expression analysis between the Y-KO and Y-Scr groups, which identified a robust set of upregulated and downregulated genes. We then calculated a Y phenotype score by dividing the signature scores of upregulated genes by those of downregulated genes, with all values scaled to the [0,1] interval. For pan-cancer scRNA-seq data, transcriptome data of 346 samples from 251 people across 20 scRNA-seq datasets were obtained from public studies63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82, from which tumour samples were selected for later analysis. Accession numbers for each scRNA-seq dataset and detailed clinical information for patients and samples are summarized in Supplementary Table 4. To avoid issues related to platform heterogeneity, only datasets generated from 10x Genomics droplet based scRNA-seq datasets were included. We performed quality control filtering and integration using the Scanpy package (v.1.9.5). Filtering was performed based on (1) confirmation that information was available for all nine Y signature genes, (2) cells had greater than 200 detected genes and (3) the mitochondrial gene counts were below 20%. Additional quality filters were applied to the data to remove barcodes that fell into any of the following categories: possible debris with too few genes expressed (less than 400) and too few unique molecular identifiers (UMIs) (less than 500), possibility of duplicate cells based on genes expressed (more than 5,500) or UMIs (more than 30,000). Count matrices and AnnData objects were then combined using a concatenate function, normalized to log transcripts per million units using the ‘sc.pp.normalize_total' function, and log-transformed using the ‘sc.pp.log1p' function. The normalized HNSC dataset from GSE150430 was then combined. Subsequently, non-tumour samples were removed, and we retained 1,030,968 high-quality cells and 14,689 genes for downstream analysis. We used the scVI Python package (scvi-tools; v.1.0.4)83 to integrate and batch correct scRNA-seq data. The scVI model was trained on the scRNA-seq data, considering samples as covariates. Following batch correction, the corrected data were integrated if multiple batches were present. The effectiveness of batch correction was evaluated by assessing the reduction in batch-specific variation while ensuring preservation of signal. Downstream analyses such as clustering, differential expression analysis or pathway enrichment analysis were performed on the batch-corrected data. Visualization of the results was achieved through two-dimensional UMAP plots, illustrating cell types, batches, datasets, gender, organs and cancer types. To annotate cells, we used the scANVI algorithm from the scVI Python packages (scvi-tools; v.1.0.4) and the Luo et al.71 dataset, where cells were pre-labelled as epithelium, endothelium, fibroblast, lymphocyte, myeloid or plasma cell. Subsequently, we performed unsupervised clustering of the scANVI latent space, and then used Leiden clustering, followed by cluster assignment to specific cell types. The scANVI model was configured with max_epochs = 20 and cluster labels were transferred and predicted, guided by a sample size of n_samples_per_label = 100. The integrated latent embedding provided by scANVI served as the basis for downstream analysis, with the dataset segregated by cell type for further investigation. To delineate cell subtypes within myeloid cell, lymphocyte and plasma cell populations, we merged corresponding AnnData and mitigated batch effects and other sources of variation using scVI. Subsequently, we predicted subtypes using Celltypist84 (v.1.6.2), using ‘majority voting' with default parameters and the pre-trained ‘Immune_All_Low.pkl' model. We collected scRNA-seq pan-cancer data from paired tumour and adjacent normal samples and, following preprocessing, categorized cells from adjacent normal samples as: male cells as having wild-type Y (WTY) chromosomes and female cells as LOY cells. Employing the train_test_split function from sklearn.model_selection, we divided data from the normal samples into training and test sets, with a split ratio of 70% for training and 30% for testing. Next, we trained a Random Forest classifier model (RandomForestClassifier from the sklearn.ensemble Python package (v.1.3.2)) to differentiate LOY and WTY cells based on the expression levels of the nine Y chromosome genes used for the bulk RNA-seq classification of LOY samples. To further validate LOY prediction by a Random Forest model, we obtained 23 samples from Liu et al.24, including sequencing data for single-cell RNA (GSE245552), bulk RNA (GSE255163) and WES (GSE255165). YchrS and average CNA of Y chromosome were calculated using the same method as CCLE data. Owing to absence of RPS4Y1 expression, it was set as 0 for all the cells when applying the Random Forest model. Genomic DNA was isolated from CRISPR Y-KO cells and CRISPR–Cas9-mediated Scr MB49 Y+ control (CRISPR Y-Scr) cell lines using the Invitrogen kit (catalogue no. DNA samples for WES were submitted to Novogene, where library preparation, sequencing and bioinformatics analysis were conducted. The genomic DNA was fragmented randomly into short pieces, end-repaired, A-tailed and ligated with Illumina adaptors. Following PCR amplification, size selection and purification, hybridization capture of libraries was performed. Captured libraries were further enriched by PCR amplification and assessed for quality using Qubit and bioanalyzer systems. Sequencing data were processed using the GATK best practices workflow. Paired-end clean reads were aligned to the mouse reference genome (GRCm39/mm39) using the Burrows–Wheeler aligner. The resulting alignments were sorted with Sambamba and duplicate reads were marked using Picard. A total of 1 × 105 LOY MB49 cells (a LOY clonal line, MB49 clone 5 (C5)) were injected subcutaneously into C57Bl/6N mice (n = 4). Once the tumours reached 500 mm3, they were removed and processed for scRNA-seq. The tumours were cut and transferred immediately to MACS C-tubes along with chilled DMEM and tumour dissociation enzymes for mouse (Miltenyi Biotech, catalogue no. The dissociated tumours were then processed using ACK Lysis buffer (Gibco, catalogue no. 17899) and EasySep Mouse CD45 Positive Selection Kit (Stem Cell Technologies, catalogue no. This mixture of highly enriched CD45+ cells was combined in a 1:1 ratio with live CD45− cells to make a final mixture that was sent for scRNA-seq. The Cedars-Sinai Applied Genomics, Computation and Translational Core used 10x genomics 3′ scRNA-seq to sequence all samples to around 60% saturation. Samples were processed using Cell Ranger (10x genomics) based on a pre-mRNA GRCh38 reference. Since the samples were not hashed, potential doublet cells were identified using Scrublet applied to the filtered feature barcode matrices from Cell Ranger. Scrublet analysed the 10% most variable genes, identified by Scanpy package (v.1.9.5, scanpy.pp.highly_variable_genes function), predicting a 10% doublet rate, and then discarded doublet cells. Finally, nuclei with over 10% of their UMIs linked to mitochondrial genes or those in the top and bottom 5% based on the number of unique genes and UMI count were also removed. One female and one male normal healthy C57BL/6N bladder sample from our previous study31 were also analysed. Filtering was performed based on: (1) cells had more than 200 detected genes and (2) the mitochondrial gene counts were below 20%. Additional quality filters were applied to the data to remove barcodes that fell into any of the following categories: possible debris with too few genes expressed (less than 400) and too few UMIs (less than 500), possibility of duplicate cells based on genes expressed (more than 30,000) or UMIs (more than 5,500). We used sc.pp.combat to remove the batch effect and applied subsequent downstream analyses on the batch-corrected data. To annotate cells, we used the scVI and scANVI algorithm from the scVI Python packages (scvi-tools). To further investigate the ability of LOY malignant cells to induce LOY in benign cells in the TME, we downloaded public scRNA-seq datasets of human xenografts in immunocompromised mice from GSE254890 (ref. 35) (SW480 cells from a male patient with CRC injected into male mice, 14 samples were incorporated as SW480 group), GSE110501 (ref. 85; only eight samples from normal tissues were incorporated as male control) and GSE144236 (ref. Based on scRNA-seq and bulk RNA-seq data provided, SW480 cells used were LOY cells, which matched with the Y chromosome information obtained by our CCLE analysis on RNA-seq data. Mouse cells were selected either by tumour cell depletion using FITC conjugated antibodies, or by expression level of mouse genes compared with human genes. Potential debris (cells with fewer than 200 expressed genes or 400 UMIs) and possible doublets (cells with more than 8,500 expressed genes or 30,000 UMIs) were filtered out. After normalization, batch correction and cell type annotation were performed by scVI and scANVI. YchrS was calculated ‘scanpy.tl.score_genes' using all Ychr gene expression. 87) background 1 were bred and housed under specific pathogen-free conditions in an American Association for Accreditation of Laboratory Animal Care-approved barrier facility at Cedars-Sinai Medical Center. MUP-uPA+ mice were fed a Western diet (Teklad, catalogue no. TD.88137) for 8 months beginning at 8 weeks after birth. HCC development was analysed at 10 months of age. Mice were killed by CO2 inhalation and livers were perfused with PBS containing 2% of heparin (20 USP units ml−1) to remove traces of blood. For single-nucleus preparation, livers were isolated, tumour tissues were dissected and cut into 50 mg tumour tissue pieces for single-nucleus isolation and sequencing. Tissue was frozen in dry ice (solid CO2) and kept in liquid nitrogen for long-term storage. For single-cell preparation, livers were isolated, and tumour tissues were dissected and digested using a cocktail of digestion enzymes containing collagenase I (450 U ml−1) (Sigma-Aldrich, catalogue no. C0130) and DNase I (120 U m−1) (Sigma-Aldrich, catalogue no. D4263) in PBS (with Ca2+/Mg2+) for 30 min at 37°C with gentle shaking at 150 rpm for liver immune cell isolation. Immune cells were enriched by density-gradient centrifugation over Percoll (GE Healthcare, catalogue no. Leukocyte rings on a border of gradient were collected, washed and stained. Immune cell suspensions were stained with Zombie Aqua (BioLegend, catalogue no. 423101) on ice for 15 min to exclude dead cells, incubated with Fc Block TruStain FcX (Clone 93, BioLegend, catalogue no. 101320, RRID: AB_1574975) for 20 min in 2% FBS-PBS and then stained with fluorochrome labelled antibody for 30 min on ice (CD45-PerCP/Cyanine5.5 (QA17A26, BioLegend, catalogue no. All the flow cytometry antibodies were validated by the manufacturer (BioLegend) and were validated in the laboratory in single channel controls. Live, CD45+ cells were sorted by BD sorter Aria III using a 100-µm nozzle. The single-cell droplets were generated with a Chromium X controller using Chromium Next GEM Single Cell 3′ Reagent Kits v3.1 (Dual Index) (10X Genomics, catalogue no. cDNA amplification and library construction were performed according to the manufacturer's instructions. All cDNA and libraries were quantified via Agilent Technologies 2100 Bioanalyzer. Gene expression libraries were sequenced at a targeted depth of 50,000 reads per cell on the Illumina Novaseq X plus (Illumina) at Novogene. Fastq files were obtained and then processed with Cell Ranger v.8.0.1 aligning to the mouse (mm10) 2020-A reference genome on 10x Genomics Cloud Analysis. Single nuclei were isolated from frozen tumour tissues using the Chromium Nuclei Isolation Kit (10x Genomics, catalogue no. cDNA amplification, library construction, sequencing and genome mapping were performed in the same way as for mouse scRNA-seq. Processed scRNA-seq data and corresponding cell type information for 116 liver cancer samples from 94 male patients40 were analysed using our Random Forest model to predict LOY at the single-cell level. Only primary tumour or metastasis samples were included in the survival analysis. For the results presented in Extended Data Fig. 8e,f, CNVs in the scRNA-seq data were predicted using the InferCNV tool (https://github.com/broadinstitute/inferCNV; v.1.13.0), so that differences in the frequencies between the LOYSCR and WTYSCR epithelial cells of gains or deletions of entire chromosomes or large chromosomal segments could be identified. For the results presented in Extended Data Fig. 6c,d, the analysis and figure were generated using Infercnvpy (https://github.com/icbi-lab/infercnvpy; v.0.4.5). We used the ‘scanpy.tl.score_genes' function from the Scanpy Python package (v.1.9.5) to compute gene set scores across cells. This function calculates gene scores for each gene listed62,88,89 in ‘gene_list' across all cells stored within the dataset. We used three distinct models to examine LOY in vivo: (1) naturally occurring Y+ and LOY (Y−) cells7, (2) an established LOY clonal line (C5) and (3) CRISPR-engineered Y-KO and Y-Scr cells7. For the tumour challenge, 1 × 105 cells from each line—MB49 clone 5 (C5), Y− (LOY), Y+ (WTY), CRISPR Y-KO and CRISPR Y-Scr—were injected subcutaneously into the flanks of C57Bl/6N mice (n = 7 per group) obtained from Taconic Biosciences7.The DNA levels of Y chromosome genes in each engineered cell type were checked for abundance before injection. Subcutaneous tumours from each group were disrupted mechanically, filtered through a 70 µm cell strainer (Corning, catalogue no. 352350) with RPMI-1640 cell culture grade media (Gibco, catalogue no. ACK lysis buffer (Gibco, catalogue no. A1049201) was used to disrupt the infiltrating red blood cells. The single cells were then centrifuged and resuspended in 1× HBSS buffer (Gibco, catalogue no. 14025092) and counted in a cell counter machine. Tumours with viability of greater than 60% were used for subsequent procedures. 17899) was used to increase the viability of each tumour and remove the dead cells. Viable cells were next processed with EasySep Mouse CD45 Positive Selection Kit (Stem Cell Technologies, catalogue no. The CD45 cells with the positive magnetic beads were purified and resuspended in EasySep Buffer (Stem Cell Technologies, catalogue no. These cells were next stained with CD45-Alexa Fluor 488 (BioLegend, catalogue no. 103122, 1:40) and Viability ghost dye Red 710 (Cytek, catalogue no. SKU 13-0871-T100) and sorted for only CD45+ cells in BD AriaIII machine. These highly purified CD45+ immune cells were next used for isolating high-quality genomic DNA with Monarch Nucleic Acid Purification Kits (NEB) and quantified. The flow through obtained after the CD45 positive selection, containing stromal, endothelial or other blood cells, was next processed with the EasySep Mouse Epithelial Cell Enrichment Kit II (Stem Cell Technologies, catalogue no. 19868), to isolate only the epithelial cells. The purity of pre- and post-isolation populations was assessed by antibody staining of random samples from each group, using the CD45 AF488 (BioLegend, catalogue no. 103122, 1:40) antibody and the BD Symphony A5, with results being analysed using Flow Jo software v.10.9.0. For isolating T cells from these tumours (n = 6 each group) and their respective PBMCs, the following panel was used: CD45 (BioLegend, catalogue no. 749276, 1:40); TCR β chain (TCRb, BioLegend, catalogue no. Three compartments were sorted with BD FACSymphony S6 machine: epithelial (Ghost Red 710 dye−CD45− or CD45−TdTomato+); non-T immune cells (Ghost Red 710 dye−CD45+CD11b+B220+) and T cells (Ghost Red 710 dye−CD45+CD11b−B220−CD3+TCRb+). The sorting gates and FACS data are shown in Supplementary Fig. Subsequently, the sorted cell compartments from tumours from the both groups were used for high-quality DNA extraction and subsequent qPCR. To get substantial amount of viable T cells for DNA isolation from subcutaneous tumours, the tumours of one group were mashed and pooled together and then proceeded for sorting. However, after sorting, the T cells were aliquoted randomly to four tubes and then processed for DNA isolation, to increase the efficacy of the result and decrease the chances of human error. Therefore, even if each group had six tumours pooled together, the graph in Extended Data Fig. 10c (right panel) shows four dots of one colour. Tumours derived from subcutaneous injection of CRISPR Y_KO and CRISPR Y-Scr cells into the flanks of C57BL/6N mice were removed and processed. Tumour-derived single-cell suspensions were subjected to FACS to isolate CD45− cells as described above. Total RNA was extracted from the isolated cells using the RNeasy Plus Mini Kit with gDNA Eliminator columns (Qiagen), following the manufacturer's protocol. Quality assessment of RNA-seq data, including sequence, alignment and quantification metrics, was conducted using FastQC v.0.12.1 and summarized with MultiQC v.1.13. Illumina Truseq adaptor, polyA and polyT sequences were trimmed using Trimmomatic v.0.39. The trimmed reads were aligned to the mouse genome (GRCm39/mm39) using STAR aligner v.2.5.2b, with parameters aligned to the ENCODE long RNA-seq pipeline recommendations (https://github.com/ENCODE-DCC/long-rna-seq-pipeline). Gene-level expression was quantified using featureCounts v.1.5.3, using Ensembl gene annotations (release v.113) for both alignment and quantification. Genes with low expression were filtered out by applying a threshold of sum of estimated counts (from featureCounts) of at least ten. Differential gene expression analysis was performed on filtered estimated read counts using the R Bioconductor package DESeq2 v.1.42.1, using a generalized linear model with a negative binomial distribution. Differentially expressed genes were identified based on a Benjamini–Hochberg adjusted P value < 0.05 and FC cut-off (≥2 or ≤−2). To validate the Y-KO-derived gene phenotype signature in human cancer data, we first excluded all Y-linked genes to prevent bias in the differential expression results. We then calculated a LOY gene phenotype score by dividing the signature scores of up-regulated genes by those of down-regulated genes, with scores scaled to a [0,1] interval. For the BBN experiment, 8-week-old mice (n = 3–4 mice per timepoint) were administered with 0.5% BBN water for 12 weeks. After 12 weeks, BBN was replaced with standard tap water. Mice were killed at 2, 4, 12, 20 and 25 weeks using isoflurane. Subsequently, blood was drawn directly from the heart using an ethylenediaminetetraacetic acid (EDTA)-prewet insulin syringe and collected in an EDTA Microvette. To enhance yield and DNA quality, mice were pooled in each group. Pooled samples were diluted 1:1 (v:v) with PBS-EDTA 0.1 M and then stratified on Histopaque-1077 Ficoll (Sigma, catalogue no. Samples were centrifuged for 30 min at 400 rcf, with acceleration and break ramps set to 0 to allow gradual phase separation. The resulting PBMC ring was collected and washed once with PBS (1X). A subsequent centrifugation of 5 min at 400 rcf (with acceleration and break ramps set to maximum speed) resulted in a pellet that was processed for DNA extraction. A human BLCA tissue microarray (TMA) with 33 unique cases comprised of triplicate cores from each patient tumour with an individual core size of 1 mm was used. Cores from female patients were used as controls for FISH. The unstained TMA formalin-fixed paraffin-embedded sections (4 μm) were baked at 55–60 °C overnight before subjecting the slide to the following steps on the Abbott VP2000 FISH Instrument: (1) deparaffination of the slide using xylene, (2) pre-treatment of the slide using 0.2 N HCl and 1 M NaSCN, (3) protease treatment with pepsin, (4) fixation in 10% buffered formalin and, finally, (5) dehydration in series of increasing concentration (70%, 85% and 100%) ethanol. The slide was then subjected to a co-denaturation step using ThermoBrite (melting temperature 73 °C, 5 min; hybridization temperature 37 °C overnight). Post hybridization the slide was washed twice with SSC/0.3% NP-4 72 ± 1 °C for 2 min and twice with 2× SSC/0.3% NP-4 15–30 °C for 1 min. Finally, the slide was counterstained with nuclear DAPI before sealing with coverslip for visualization. Formalin-fixed paraffin-embedded samples were sectioned at 4-μm thickness onto Superfrost Plus slides (Fisher Scientific, catalogue no. IHC staining was performed on the Ventana Discovery Ultra Instrument (Roche) as described90. After applying antigen retrieval buffer (CC1 (Tris, pH 8.0) (Roche Ventana, catalogue no. 950-124), CD45 primary antibody (Cell Signaling, catalogue no. DISC anti-Rabbit HQ (Roche Ventana, catalogue no. 760-4815) was then applied for chromagen staining. After DAPI nuclear counterstain, the tissue area was covered coverslipped and mounted with ProLong antifade medium (Invitrogen, catalogue no. XY FISH immunofluorescence slides were scanned using the ZEISS Axio Scan.Z1 whole slide scanner at ×20 magnification (Plan-Apochromat lens (numerical aperture, 0.8; M27)). TMA tissue cores were outlined with permanent marker pen on the coverslip for tissue detection. Region of scan was generated by a polygon tool and the raw focus map was generated using the ‘every-2-tiles' strategy (z-range 150 µm, 21.04-µm step size) under ×5 lens (Fluor ×5/0.25 M27), while a fine focus map was generated using onion skin (z-range 100 µm, 2.06-µm step size, 0.1 density, 24 maximum number of points). Both focus maps were generated in the DAPI channel at 2% LED intensity, 50-ms exposure time. Spectrum Green (X probe) was excited at 495-nm wavelength (5% LED intensity, 150-ms exposure time) and detected at 500–550-nm bandwidth. Spectrum Orange (Y probe) was excited at 548-nm wavelength (25% LED intensity, 150 ms exposure time), detected at 570–640-nm bandwidth. Nuclear DAPI fluorescence was excited at 420-nm wavelength (2.5% LED intensity, 50-ms exposure time), detected at 430–470-nm bandwidth. Spectrum Aqua (18S RNA probe) was excited at 434-nm wavelength (5% LED intensity, 150-ms exposure time) and detected at 460–500-nm bandwidth. All signals were detected by Hamamatsu Orca Flash camera and 16-bit depth format image setting was applied. Standard IHC slide was imaged using the Leica Aperio AT2 whole slide scanner at ×20 magnification. The whole slide image obtained from Zeiss Axio scan system was imported into HALO AI Module (Indica Labs), v.4.0.5107.318, for analysis. Upon import, the TMA image underwent segmentation to identify individual tissue cores. Missing cores were identified and removed from the analysis. For AI training, seven distinct regions of interest were selected, comprising a total of 43 nuclei, to refine the Nuclei Segmentation plugin. Following segmentation, FISH analysis was conducted using the HALO FISH module, v.3.2.3, to detect nuclear signals. The resulting data were exported as.csv files, containing object-level (cell) data for subsequent analysis. To further validate the results and minimize the impact of mis-annotating LOY epithelial cells as LOY immune cells, we analysed 21 CD45-based FACS-sorted samples from three independent public scRNA-seq datasets (HNSC, GSE182227 (ref. This collection included 12 matched CD45+ and CD45− samples from six tumours. Detailed dataset and sample information are provided in Supplementary Table 7. The same quality control, normalization and batch correction procedures described for the pan-cancer human scRNA-seq datasets were applied. LOY cells were predicted using a Random Forest model. To further validate the accumulation of LOY immune cells in tumours, we analysed two additional datasets: RCC dataset36 (accession number, EGAD0001008030), comprising 14 matched tumour (also included in the pan-cancer datasets) and blood samples. 37)), containing 38 matched TIL and PBMC samples. The same quality control, normalization, batch correction and LOY cell prediction methods were applied. Additionally, cell types were identified using the scANVI algorithm and marker gene expression, with annotated pan-cancer scRNA-seq datasets serving as references. Mouse CD8+ T cells were isolated from spleens of C57BL/6N mice using the Mouse CD8+ T Cell Isolation kit (Miltenyi Biotec, catalogue no. CD8+ T cells were then activated by seeding onto six-well plates coated with 1 µg ml−1 anti-CD3 (clone 2C11, BioLegend, catalogue no. T cells were cultured in RPMI supplemented with 10% fetal calf serum, 1% penicillin–streptomycin, 50 µM 2-mercaptoethanol, 1% insulin transferrin sodium selenite as well as 0.1 µg ml−1 IL-7 (Peprotech, catalogue no. T cells were kept at no higher than 1 × 106 ml−1 and transferred to new coated plates every 2–3 days to maintain activation. DNA and RNA was isolated at the timepoints indicated using the Monarch Genomic DNA Purification Kit (New England Biolabs, catalogue no. T3010L) or RNeasy Plus Mini Kit with gDNA Eliminator (Qiagen, catalogue no. CDNA was generated with Maxima H Minus cDNA Synthesis Master Mix (Thermo Fisher, catalogue no. Genomic DNA (10 ng per reaction) was used to detect and quantify Y‐chromosome-specific genes associated with LOY signatures—Kdm5d, Uty, Eif2s3y, Ddx3y, Ssty1, Ssty2 and Zfy1/2—as well as to detect the presence of Cas9 in immune and epithelial compartments. Housekeeping genes (B2m, Gapdh and Actb) served as endogenous controls. All qPCR reactions were carried out using SYBR Green Universal Master Mix (Applied Biosystems, catalogue no. 4309155) on a Quant Studio 6 Flex Real-Time PCR system (Applied Biosystems). To assess Y chromosome copy number in various TIL immune cell populations (T cells and non-T cells), we compared cycle threshold (Ct) values from each sorted population against Ct values from wild-type male tumour cells, using the ΔΔCt method to calculate FC. For Cas9 detection, Ct values in sorted immune populations were compared with DNA from wild-type C57Bl/6N mice, which have no Cas9 integration, using Gapdh and Actb. Absence (or near-absence) of Cas9 amplification in immune compartments confirmed that these cells were not contaminated by, or had phagocytosed, genome-edited epithelial cells (Supplementary Fig. Details of primer sequences used in this manuscript for DNA qPCR are as follows: FC values were calculated using the ΔΔCt method relative to the appropriate wild-type controls. Data were used to quantify the relative copy number of Y chromosome genes in tumour‐derived immune subsets or to confirm the absence of Cas9 in cells not genetically engineered. Chronic stimulation of T cells was validated through the upregulation of canonical exhaustion markers, including Tox, Pdcd1 and Ctla4, and the downregulation of Tcf7. To evaluate LOY, the expression levels of Y-linked genes (Uty, Kdm5d, Ddx3y, Usp9y) were analysed. To generate cell-type-specific gene signatures for LOYSCR/WTYSCR epithelial cells, CD4+ T cells, and CD8+ T cells from scRNA-seq data, we conducted differential analysis using the ‘sc.tl.rank_genes_groups' function of the Scanpy (v.1.9.5) package. This analysis used Wilcoxon rank sum (Mann–Whitney U) tests to identify significant differences across each LOYSCR and WTYSCR cell type. We first identified genes significantly up-regulated (log2FC > 1, adjusted P value < 0.05) in the LOYSCR versus WTYSCR epithelial cells, CD4+ T cells and CD8+ T cells separately. To establish unique signatures for each cell type, we then excluded genes expressed in more than 15% of any other LOYSCR or WTYSCR cell type. We then performed deconvolution on normalized bulk expression data from TCGA cancer types using the ssGSEA algorithm, evaluating the relationship of these signatures with patient outcome. Time-to-event outcomes were presented by using Kaplan–Meier curves and compared by using log-rank test or univariate Cox proportional hazards model (survival R package; v.3.5.8) as noted in each figure. Two multivariable Cox proportional hazards models were fitted, each as a function of (1) YchrS with ancestry and race as known risk factors and confounders; (2) LOY signatures scRNA-seq signatures, including LOYSCR CD4+ T cell, LOYSCR CD8+ T cell, and LOYSCR epithelial cell signatures, with age as known risk factors and confounders. Hazard ratio along with 95% CI based on multivariable Cox proportional hazards models were reported. This method uses maximally selected rank statistics from the maxstat91 R package (v.0.7-25) to classify two groups (low- versus high-risk) based on the optimal cut-point. Moreover, continuous variables included as covariates in the Cox proportional hazards model were evaluated92. Linearity was assessed to ensure model adequacy. According to clinical risk factors and risk scores of multivariate Cox regression coefficients for Extended Data Table 2, a prognostic nomogram was established using the ‘rms' R software package (v.6.8-0), and the prediction accuracy of the nomogram was assessed using the calibration curve to evaluate the match between expected and observed events at 2, 5 and 8 years. All human specimens and associated data were collected following protocols approved by the Institutional Review Board (IRB protocol 43021) at Cedars-Sinai Medical Center, adhering strictly to the Declaration of Helsinki guidelines. Written informed consent was obtained from each participant or their legal guardian. Detailed information regarding patient recruitment, sample collection (including TMA preparation), and data management can be found within IRB protocol 43021. All animal procedures were reviewed and approved by the Institutional Animal Care and Use Committee (IACUC protocol 8253) at Cedars-Sinai Medical Center. Experiments were conducted in strict accordance with the guidelines specified in the National Institutes of Health (NIH) Guide for the Care and Use of Laboratory Animals. Protocol 8253 comprehensively describes animal housing conditions, care standards and experimental methodologies. All animal experiments were performed in accordance with institutional IACUC protocols. All analyses were conducted using R (v.4.3.1) and Python (v.3.10.9). Before commencing tests, data were assessed for normality using the Kolmogorov–Smirnov test, followed by Bartlett or Levene tests to evaluate homogeneity of variances. For normally distributed variables, the unpaired Student's t-test (Stats R package; v.4.3.1) was applied, whereas non-normally distributed variables were analysed using Wilcoxon rank sum tests. Data presentation and multiple comparison corrections are as stated in figure legends. Statistical significance was considered when P values were less than 0.05, including adjusted P values. Discovery analyses involving more than 20 comparisons underwent multiple testing correction using the p.adjust function in R or multipletests function in Python, applying the Benjamini–Hochberg method to control the false discovery rate at 0.05. To compare ROC curves, we used the roc.test function from the pROC R package (v.1.18.5). This allowed us to assess differences between AUC of YchrS and the AUC of YwholeS or LOYDNA. Python packages such as Scanpy (v.1.9.5), Pandas (v.2.0.0), Statsmodels (v.0.14.0), NumPy (v.1.24.2), Scipy (v.1.10.1), Matplotlib (v.3.8.0), Seaborn (v.0.11.2) and Sklearn (v.1.3.2), were used for data analysis. The R package ComplexHeatmap (v.2.11.1) was used to generate heat maps, and visualization was facilitated using ggplot2 (v.3.3.5), ggpubr (v.0.6.0), ggrepel (v.0.9.2), Statannot (v.0.6.0), Circlize (v.0.4.16), GseaVis (v.0.0.5), Enrichplot (v.1.22.0), GridExtra (v.2.3.0), Pheatmap(v.1.0.12) and DEGreport (v.1.38.5) R packages. For data manipulation, Readr (v.2.1.5), Readxl (v.1.4.3), Dplyr (v.1.1.4), Plyr (v.8.9), Apeglm (v.1.24.0), Tidyr (v.1.3.1), Tidyverse (v.2.0.0), Tibble (v. 3.2.1), Iranges (v.2.36.0), Biobase (v.2.62.0), BiocGenerics (v.0.48.1), Lubridate (v.1.9.3), Stringr (v.1.5.1) and AnnotationDbi (v.1.64.1) were used for analysis. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. All bulk RNA-seq, single-cell and nuclear RNA-seq data and WES data generated for this study are available in the Gene Expression Omnibus (accession numbers GSE290112, GSE290113, GSE290114, GSE290115, GSE290587 and GSE290588). Source data are provided with this paper. Software packages, notebooks and scripts used for analysis are available at https://github.com/KnottLab/NATURE_LOY_Pancancer. Forsberg, L. A. et al. Mosaic loss of chromosome Y in peripheral blood is associated with shorter survival and higher risk of cancer. Danielsson, M. et al. Longitudinal changes in the frequency of mosaic chromosome Y loss in peripheral blood cells of aging men varies profoundly between individuals. Duan, Q. et al. Mosaic loss of chromosome Y in peripheral blood cells is associated with age-related macular degeneration in men. Kido, T. & Lau, Y. F. Roles of the Y chromosome genes in human cancers. & Rheinbay, E. Loss of chromosome Y in primary tumors. Downregulation of chromosome Y genes potentially contributes to elevated cancer risk. Abdel-Hafiz, H. A. et al. Y chromosome loss in cancer drives growth by evasion of adaptive immunity. Noveski, P. et al. Loss of Y chromosome in peripheral blood of colorectal and prostate cancer patients. Mattisson, J. et al. Loss of chromosome Y in regulatory T cells. Sonugur, F. G. & Akbulut, H. The role of tumor microenvironment in genomic instability of malignant tumors. Prokop, J. W. & Deschepper, C. F. Chromosome Y genetic variants: impact in animal models and on human disease. The cancer/testis genes: review, standardization, and commentary. Integrated analysis of genetic ancestry and genomic alterations across cancers. Carrot-Zhang, J. et al. Comprehensive analysis of genetic ancestry and its molecular correlates in cancer. McGrail, D. J. et al. High tumor mutation burden fails to predict immune checkpoint blockade response across all cancer types. Rizvi, N. A. et al. Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer. & Jaffee, E. M. Targeting neoantigens to augment antitumour immunity. Lauss, M. et al. Mutational and putative neoantigen load predict clinical benefit of adoptive T cell therapy in melanoma. Telli, M. L. et al. Homologous recombination deficiency (HRD) score predicts response to platinum-containing neoadjuvant chemotherapy in patients with triple-negative breast cancer. Patterns of genomic loss of heterozygosity predict homologous recombination repair defects in epithelial ovarian cancer. Popova, T. et al. Ploidy and large-scale genomic instability consistently identify basal-like breast carcinomas with BRCA1/2 inactivation. Birkbak, N. J. et al. Telomeric allelic imbalance indicates defective DNA repair and sensitivity to DNA-damaging agents. Malta, T. M. et al. Machine learning identifies stemness features associated with oncogenic dedifferentiation. Liu, X. et al. Th17 cells secrete TWEAK to trigger epithelial-mesenchymal transition and promote colorectal cancer liver metastasis. Smoking is associated with mosaic loss of chromosome Y. Bonnet, S. et al. A mitochondria-K channel axis is suppressed in cancer and its normalization promotes apoptosis and inhibits cancer growth. & Ploegh, H. L. A guide to antigen processing and presentation. Dhatchinamoorthy, K., Colbert, J. D. & Rock, K. L. Cancer immune evasion through loss of MHC class I antigen presentation. Fantini, D. et al. A carcinogen-induced mouse model recapitulates the molecular alterations of human muscle invasive bladder cancer. Williams, P. D., Lee, J. K. & Theodorescu, D. Molecular credentialing of rodent bladder carcinogenesis models. Abdel-Hafiz, H. A. et al. Single-cell profiling of murine bladder cancer identifies sex-specific transcriptional signatures with prognostic relevance. Puram, S. V. et al. Cellular states are coupled to genomic and viral heterogeneity in HPV-related oropharyngeal carcinoma. Alvisi, G. et al. Multimodal single-cell profiling of intrahepatic cholangiocarcinoma defines hyperactivated Tregs as a potential therapeutic target. Yu, H. et al. Tumor-infiltrating myeloid cells confer de novo resistance to PD-L1 blockade through EMT-stromal and Tgfβ-dependent mechanisms. Hosohama, L. et al. Colorectal cancer stem cell subtypes orchestrate distinct tumor microenvironments. Li, R. et al. Mapping single-cell transcriptomes in the intra-tumoral and associated territories of kidney cancer. Ruffin, A. T. et al. B cell signatures and tertiary lymphoid structures contribute to outcome in head and neck squamous cell carcinoma. Xue, R. et al. Liver tumour immune microenvironment subtypes and neutrophil heterogeneity. Maluf, S. W., Martinez-Lopez, W. & da Silva, J. DNA damage: health and longevity. Ten, A., Kumeiko, V., Farniev, V., Gao, H. L. & Shevtsov, M. Tumor microenvironment modulation by cancer-derived extracellular vesicles. Sakaguchi, S., Yamaguchi, T., Nomura, T. & Ono, M. Regulatory T cells and immune tolerance. Kverneland, A. H. et al. Adoptive cell therapy with tumor-infiltrating lymphocytes supported by checkpoint inhibition across multiple solid cancer types. Ratto, G. B. et al. A randomized trial of adoptive immunotherapy with tumor-infiltrating lymphocytes and interleukin-2 versus standard therapy in the postoperative treatment of resected nonsmall cell lung carcinoma. Liu, J. et al. An integrated TCGA pan-cancer clinical data resource to drive high-quality survival outcome analytics. & Koltzenburg, M. Microarray analysis after RNA amplification can detect pronounced differences in gene expression using limma. Systematic RNA interference reveals that oncogenic-driven cancers require TBK1. Hänzelmann, S., Castelo, R. & Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. Taylor, A. M. et al. Genomic and functional approaches to understanding cancer aneuploidy. Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Carter, S. L. et al. Absolute quantification of somatic DNA alterations in human cancer. Knijnenburg, T. A. et al. Genomic and molecular landscape of DNA damage repair deficiency across The Cancer Genome Atlas. Timms, K. M. et al. Association of defects with genomic scores predictive of DNA damage repair deficiency among breast cancer subtypes. Nielsen, M. & Andreatta, M. NetMHCpan-3.0; improved prediction of binding to MHC class I molecules integrating information from multiple receptor and peptide length datasets. Hundal, J. et al. pVAC-Seq: a genome-guided in silico approach to identify tumor neoantigens. Cristescu, R. et al. Molecular analysis of gastric cancer identifies subtypes associated with distinct clinical outcomes. Li, Y. et al. Pan-cancer characterization of immune-related lncRNAs identifies potential oncogenic biomarkers. Zeng, D. Q. et al. IOBR: multi-omics immuno-oncology biological research to decode tumor microenvironment and signatures. Turning up the heat on non-immunoreactive tumours: pyroptosis influences the tumor immune microenvironment in bladder cancer. Chen, X. Y. et al. CD8 T effector and immune checkpoint signatures predict prognosis and responsiveness to immunotherapy in bladder cancer. Mariathasan, S. et al. TGFβ attenuates tumour response to PD-L1 blockade by contributing to exclusion of T cells. Li, R. Y. et al. Mapping single-cell transcriptomes in the intra-tumoral and associated territories of kidney cancer. Zhang, Y. P. et al. Single-cell analyses of renal cell cancers reveal insights into tumor microenvironment, cell of origin, and therapy response. Werba, G. et al. Single-cell RNA sequencing reveals the effects of chemotherapy on human pancreatic adenocarcinoma and its tumor microenvironment. Ma, L. C. et al. Tumor cell biodiversity drives microenvironmental reprogramming in liver cancer. Regner, M. J. et al. A multi-omic single-cell landscape of human gynecologic malignancies. Chen, Y. P. et al. Single-cell transcriptomics reveals regulators underlying immune cell diversity and immune subtypes associated with prognosis in nasopharyngeal carcinoma. Xu, J. F. et al. Single-cell RNA sequencing reveals the tissue architecture in human high-grade serous ovarian cancer. Li, C. B. et al. Single-cell transcriptomics reveals cellular heterogeneity and molecular stratification of cervical cancer. Luo, H. et al. Pan-cancer single-cell analysis reveals the heterogeneity and plasticity of cancer-associated fibroblasts in the tumor microenvironment. Sathe, A. et al. Single-cell genomic characterization reveals the cellular reprogramming of the gastric tumor microenvironment. Kim, N. et al. Single-cell RNA sequencing demonstrates the molecular and cellular reprogramming of metastatic lung adenocarcinoma. Lambrechts, D. et al. Phenotype molding of stromal cells in the lung tumor microenvironment. Chen, S. J. et al. Single-cell analysis reveals transcriptomic remodellings in distinct cell types that contribute to human prostate cancer progression. Identification of a distinct luminal subgroup diagnosing and stratifying early stage prostate cancer by tissue-based single-cell RNA sequencing. Luo, H. et al. Characterizing dedifferentiation of thyroid cancer by integrated analysis. Chen, Z. H. et al. Single-cell RNA sequencing highlights the role of inflammatory cancer-associated fibroblasts in bladder urothelial carcinoma. B. et al. A pan-cancer blueprint of the heterogeneous tumor microenvironment revealed by single-cell profiling. Zhang, M. et al. Single-cell transcriptomic architecture and intercellular crosstalk of human intrahepatic cholangiocarcinoma. Peng, J. Y. et al. Single-cell RNA-seq highlights intra-tumoral heterogeneity and malignant progression in pancreatic ductal adenocarcinoma. Lin, W. et al. Single-cell transcriptome analysis of tumor and stromal compartments of pancreatic ductal adenocarcinoma primary tumors and metastatic lesions. & Yosef, N. Deep generative modeling for single-cell transcriptomics. Conde, C. D. et al. Cross-tissue immune cell analysis reveals tissue-specific features in humans. Zhao, Q. et al. Single-cell transcriptome analyses reveal endothelial cell heterogeneity in tumors and changes following antiangiogenic treatment. Ji, A. L. et al. Multimodal analysis of composition and spatial architecture in human squamous cell carcinoma. Weglarz, T. C., Degen, J. L. & Sandgren, E. P. Hepatocyte transplantation into diseased mouse liver. Chu, Y. S. et al. Pan-cancer T cell atlas links a cellular stress response state to immunotherapy resistance. Comparative proteomic analysis of HPV(+) oropharyngeal squamous cell carcinoma recurrence. Grambsch, P. M. & Therneau, T. M. Proportional hazards tests and diagnostics based on weighted residuals. We thank G. Martins and J. Dos Santos for their expert advice and assistance with FACS, H. Ye for interpreting the histological slides used in FISH immunofluorescence experiments and J. Xiao of the Theodorescu laboratory for generating MB49 subcutaneous xenografts and the corresponding scRNA-seq data. This work was supported in part by NIH grants CA278732 (to D.T.) ), as well as by the NIH National Center for Advancing Translational Sciences through UCLA CTSI grant UL1 TR001881 (to V.F.C. Present address: Johns Hopkins University, Baltimore, MD, USA Present address: University of Arizona Comprehensive Cancer Center, Tucson, AZ, USA These authors contributed equally: Yiling Shen, Suhyeon Choi Department of Urology, Cedars-Sinai Medical Center, Los Angeles, CA, USA Xingyu Chen, Yiling Shen, Hany A. Abdel-Hafiz, Mukta Basu, Lena Hoelzen, Martina Tufano, Saravana Kumar Kailasam Mani, Maryam Ranjpour & Dan Theodorescu Department of Biomedical Sciences, Cedars-Sinai Medical Center, Los Angeles, CA, USA Xingyu Chen, Yiling Shen, Suhyeon Choi, Jiani Zhu, Ekaterina K. Koltsova & Simon R. V. Knott School of Medicine, Tsinghua University, Beijing, China Department of Medicine, Cedars-Sinai Medical Center, Los Angeles, CA, USA Department of Pathology and Laboratory Medicine, Cedars-Sinai Medical Center, Los Angeles, CA, USA Department of Computational Biomedicine, Cedars-Sinai Medical Center, Los Angeles, CA, USA Samuel Oschin Comprehensive Cancer Institute, Cedars-Sinai Medical Center, Los Angeles, CA, USA You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conceived the study design and analyses. analysed the mouse HCC sequencing data and generated associated graphs. provided guidance on multivariate Cox analysis and nomogram development. wrote the manuscript with input and specific sections from all authors. All authors reviewed and approved the final manuscript. Correspondence to Simon R. V. Knott or Dan Theodorescu. The authors declare no competing interests. Nature thanks Joshua Rubin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Differential gene expression analysis for Y Chromosome genes comparing LOYDNA and WTYDNA male TCGA samples. Dot color represents -log10 adjusted P-values for each gene. Vertical lines indicate log2FC of -1 and 1 and the horizontal line indicates -log10 adjusted P-values of 200. b. YchrS scores in LOYDNA and WTYDNA TCGA tumor samples, where samples are separated by cancer types. Whiskers extend to either the minimum or maximum values within 1.5 × the interquartile range (IQR). Any data points outside these whiskers are considered outliers. c. Distribution of chromosome segments and corresponding Copy Number Alteration (CNA). Each dot represents one chromosome segment in a particular cell line, with midpoint location shown in X axis and log2 mean CNA shown in Y axis. d. Average CNA at the chromosome level for the 625 CCLE male cell lines with available Y chromosome CNA, with the lines separated into WTYBR (n = 442) and LOYBR (n = 183) groups. e. YchrS and corresponding average CNA of each chromosome for samples in (d). Each dot represents a cell line, with YchrS level shown by color. a. YchrS scores for male TCGA tumors separated by smoking status (Current/Ever, n = 1,019; Never, n = 307) across nine cancer types. Whiskers extend to either the minimum or maximum values within 1.5 × the interquartile range (IQR). Any data points outside these whiskers are considered outliers. b. YchrS scores for Liver Hepatocellular Carcinoma (LIHC) samples, separated by Hepatitis B virus (HBV) or Hepatitis C virus (HCV) infection status (left), and YchrS scores for Stomach Adenocarcinoma (STAD) samples separated by Epstein-Barr Virus (EBV) infection status (right). c. The distribution of YchrS values (Black line) is shown with two proposed thresholds indicated by vertical dashed lines: mean (Red) and median (Light blue). d. AUROC analysis comparing specificity (X-axis, 0–1) against sensitivity (Y-axis, 0–1) for mean and median YchrS cut-offs used to detect LOY. Arrows indicate sensitivity and specificity for each threshold. e. Kaplan-Meier survival curves for Overall Survival (OS) in WTYBR and LOYBR tumors across three ethnicities from the TCGA pan-cancer dataset. Statistics were calculated using the univariate Cox-Proportional-Hazard (coxPH) model. f. Kaplan-Meier survival curves for OS in WTYBR and LOYBR tumors across four genetic ancestries from the TCGA pan-cancer dataset. Statistics were calculated using the univariate coxPH model. g-h. Kaplan-Meier survival curves illustrating OS (g) and Disease-Specific Survival (DSS, h) for TCGA patients, grouped by expression levels of nine Y chromosome genes. Statistics were calculated using the univariate coxPH model. b. Pearson correlation scores and corresponding P-values comparing YchrS scores and aneuploidy scores (left) and altered fractions (right) in 29 different cancer types. c. Comparison of different genomic instability pathway scores between WTYBR and LOYBR tumors. The size of dots represent -log10 P-values adjusted by Benjamini-Hochberg correction as calculated based on two‐sided Wilcoxon rank sum tests. HRD, Homologous Recombination Deficiency; LOH, Loss of Heterozygosity; LST, Large-scale State Transitions; TAI, Telomeric Allelic Imbalance. d. Comparison of different functional pathways scores between WTYBR and LOYBR tumors. The size of squares represents -log10 P-values adjusted by Benjamini-Hochberg correction as assessed by two‐sided Wilcoxon rank sum tests. and significance was assessed based on two‐sided Wilcoxon rank sum tests. f. Comparison of glycolysis (left) and hypoxia (right) signature scores between WTYBR (n = 2,685) and LOYBR (n = 1,442) tumors. and significance was assessed based on two‐sided Wilcoxon rank sum tests. g. Comparison of hormone-related pathway signature scores between WTYBR (n = 2,685) and LOYBR (n = 1,442) tumors. and significance was assessed based on two‐sided Wilcoxon rank sum tests. h. Comparison of expression levels for receptor and co-inhibitory genes between WTYBR and LOYBR tumors. Square color indicates mean scaled gene expression in each group and square size represents -log10 P-values adjusted by Benjamini-Hochberg correction, as assessed by two‐sided Wilcoxon rank sum tests. a. Schematic depicting tumor sources of the human scRNA-seq pan-cancer datasets developed and analyzed in this study. Quality control metrics for the pan-cancer scRNA-seq datasets with vertical lines indicating selection criteria: 400 <Gene counts <5500 (top) and 500 <Total counts <30,000 (center and bottom) for further analysis. c. Cells from the pan-cancer scRNA-seq data colored by original dataset (top left), organ (top right), sex information (bottom left) and cancer type (bottom center) and major cell type (bottom right). d. Expression levels of marker genes across the major cell types identified in the pan-cancer scRNA-seq datasets. Dot size indicates the proportion of expressing cells and color indicates mean expression levels. a. Schematic of the development of the Random Forest model utilized to predict LOYSCR status in individual cells. Expression levels of 9 YchrS signature genes in male and female samples. Dot size indicates the proportion of expressing cells and color indicates mean expression levels. d. Expression levels of the 9 YchrS signature genes in LOYSCR and WTYSCR cells from male tumor samples. Dot size indicates the proportion of expressing cells and color indicates mean expression levels. e. Total counts per cell type, where cells are separated by their cell type and LOYSCR and WTYSCR status. Violin plots show the full distribution of total read counts per cell type. Whiskers extend to either the minimum or maximum values within 1.5 × the interquartile range (IQR). Any data points outside these whiskers (if shown) are considered outliers. f. Scores for chromosome-specific signatures (with each signature comprising all genes from the corresponding chromosome) for male cells, where cells are separated by their LOYSCR and WTYSCR status. Data are presented as mean values ± 95% Confidence Interval (CI). g. Correlations between YchrS from bulk RNA-seq data and average Y chromosome Copy Number Alteration (CNA) from whole-exome sequencing (WES) data (left), the correlation between YchrS and the proportion of LOYSCR cells identified through single-cell RNA sequencing (middle), and the correlation between the proportion of LOYSCR cells and average Ychr CNA (right). Male and female samples are represented by blue and red dots, respectively, with each dot representing a sample. Lines show the linear regression results for male and female samples, with shaded regions showing the 95% confidence interval. Expression levels of Major Histocompatibility Complex (MHC) class II genes in LOYSCR and WTYSCR epithelial cells. Dot sizes (bottom) indicate the proportion of expressing cells, colored by mean standardized expression levels. Bars (top) indicate the log2Fold Change (FC) between in LOYSCR and WTYSCR epithelial cells, where color indicates the corresponding -log10 adjusted P-value, calculated via sc.tl.rank_genes_group in scanpy package using two‐sided Wilcoxon rank‐sum test with Benjamini-Hochberg correction. Scores for pathways related to hormone and oncogenesis in LOYSCR and WTYSCR epithelial cells. Dot color indicates the mean pathway score, while dot size indicates the -log10 adjusted P-value of the difference between cell populations, as calculated by two‐sided Wilcoxon rank sum tests with Benjamini/Hochberg correction. c. Mean Copy Number Variation (CNV) scores of LOYSCR and WTYSCR epithelial cells, as estimated by the inferCNVpy algorithm and where the significance of the difference was calculated based on the two‐sided Wilcoxon rank-sum test. d. Distribution of CNV scores as obtained by the inferCNVpy algorithm for LOYSCR and WTYSCR epithelial cells for 12 cancer types. a. Sequencing depth of Whole Exome Sequencing (WES) data from CRISPR Y-KO and Y-Scr MB49 cells. b. LOY gene phenotype scores in LOYBR (n = 74) versus WTYBR (n = 147) TCGA Bladder Cancer (BLCA) samples. and significance was assessed based on two‐sided Wilcoxon rank sum tests. c. LOY gene phenotype scores across multiple cancer types in the TCGA dataset. Significant LOY gene phenotype trends were observed in Lung Adenocarcinoma (LUAD), Pancreatic Adenocarcinoma (PAAD), and Stomach Adenocarcinoma (STAD), with variable significance across other cancers. NES > 0 indicate the pathway is enriched in Y-KO cells, otherwise in Y-Scr cells. f-g. Stacked bar plots showing counts of Single Nucleotide Polymorphisms (SNPs, f) and Insertion and Deletions (Indels, g) in different genomic regions. h. Ratio of mutation counts (SNPs and Indels) between CRISPR Y-KO cells and Y-Scr MB49 cells across different chromosomes. Single cell RNA sequencing data of 85,759 cells from male and female tumors induced by N-butyl-N-(4-hydroxybutyl)-nitrosamine, where cells are colored by sample (FB prefix indicates female and MB indicates male), cell type and male/female status. Expression levels of the 4 genes used as the mouse chrY signature (YchrSmus) genes in male and female cells from the tumors shown in (a). c. Expression levels of 4 YchrSmus genes in male and female cells from the tumors shown in (a), where cells are also separated by cell type. e-f. Inferred Copy Number Variation (CNV) profiles across chromosomes (columns) in single cells (rows), grouped by major cell types (colored bars on the left). Each column spans chromosome 1 through 22, arranged along the horizontal axis. Differences in chromosomal signal patterns highlight potential aneuploidies and region‐specific alterations linked to LOY status. Mean YchrSmus score across various cell types. Displayed are scRNA-seq data for normal bladder tissues from female (left) and male (middle) C57BL/6 N mice; tumors from C57BL/6 N mice following subcutaneous inoculation with LOY MB49 mouse bladder cancer cell lines (right). Error bars represent the 95% confidence interval (CI) of the mean value, estimated by bootstrap resampling. Analysis of LOY percentage in cell populations from 6 mouse hepatocellular carcinoma (HCC) tumor samples. (b) LOY percentages of sorted CD45+ immune cell populations evaluated by scRNAseq from 3 independent samples. (c) LOY percentage in 3 independent whole tumor cell populations evaluated by snRNAseq. d. Proportion of LOYSCR and WTYSCR in individual CD45+ samples from the HNSC32 (left), CHOL33 (middle), and BLCA34 datasets (right). HNSC, Head and Neck Squamous Cell Carcinoma. f. Mean scaled YchrSmus values across 3 different cell types from a xenograft scRNA-seq dataset—endothelium, fibroblast, and myeloid cells—for 3 groups: Female Control (orange), Group SW480 (male CRC cells, red), Male Control (blue). Error bars represent the 95% confidence interval (CI) of the mean value, estimated by bootstrap resampling. g. Representative FISH images selected from the human patient TMA cores. Smaller regions (80 micron x 80 micron) of images from the normal male and LOY cancer male patient tissue microarray cores (original diameter ~1 mm) are displayed corresponding to CD45-high and CD45-low regions as shown. CD45 signals were probed by immunohistochemistry staining and bright field imaging. Pseudocolor immunofluorescence images of nuclear DAPI, X chromosome FISH probe, Y chromosome FISH probe and the control 18sRNA FISH probe are shown as indicated in the figure. These data are representative of male (n = 18) staining and imaging experiments that yielded similar results. Genomic DNA (10 ng per reaction) was isolated from sorted epithelial cells, non–T cells, and T cells derived from CRISPR‐engineered (Y-Scr or Y-KO) MB49 tumors. Wild‐type (WT) C57Bl/6 N genomic DNA (lacking Cas9) served as the reference in the ΔΔCt analysis. (a) Ct values were normalized to Gapdh and plotted as fold change relative to WT. (b) Ct values were normalized to Actb and similarly plotted as fold change over WT. from 4 biological replicates in each group. c. DNA qPCR analysis of Y chromosome genes in TdTomato+ CD45- epithelial cells (left) and CD3+ T cells (right) from CRISPR Y-Scr and CRISPR Y-KO groups. P-values from two-way ANOVA with Fisher's LSD test and Geisser-Greenhouse correction. ; Each dot denotes pooled samples from six independent tumors. Six independent tumors were mashed and pooled together for substantial number of T cells isolation and then randomly aliquoted to 4 tubes to increase the efficacy of the result and reduce human error. d. DNA qPCR analysis of Y chromosome genes (Kdm5d, Uty, Eif2s3y, and Ddx3y) in PBMCs extracted from BBN-treated mice. Each dot represents the pooled DNA sample from 3-4 mice. On X axis: the time points of samples collection, along with a timeline of BBN administration and bladder tumor onset; on the Y axis the fold change expressed as relative gene expression level compared to untreated ctrl at initiation of BBN. e. Changes in DNA levels of Y chromosome genes (Ddx3y, Kdm5d, Zfy1/2, Ssty1, Ssty2) upon long term activation of primary mouse T cells in vitro. qPCR data normalized to housekeeping genes Actin (left) or B2m (right) relative to day 1 after mouse T cell isolation. Expression changes of T cell exhaustion markers (Ctla4, Pdcd1, Tcf7, Tim3) during long term activation of primary mouse T cells in vitro. Expression normalized to housekeeping genes Actin (f) or Gapdh (g) are shown relative to day 1 post T cell isolation. Mean expression of exhaustion (h) and Y chromosome genes (i) in Naïve P14 CD8+ T cells at baseline (Day 0) and compared to at Days 8, 15, and 30 post-infection under acute (LCMV Armstrong; solid lines) or chronic (LCMV Clone 13; dashed lines) conditions. Data were extracted from publicly available transcriptomic datasets39. a. Kaplan-Meier Disease-Specific Survival (DSS) curves for TCGA samples with high or low signature scores for LOYSCR and WTYSCR CD4+ T cells (two leftmost plots), for LOYSCR and WTYSCR CD8+ T cells (two middle plots), and for LOYSCR and WTYSCR epithelial cells (two rightmost plots). Significance was assessed based on univariate Cox Proportional Hazard (coxPH) model. P-values calculated using univariate coxPH model. c. Kaplan-Meier survival curves of OS for samples in (b) stratified by the combinatorial proportions of LOYSCR tumor cells and LOYSCR CD4+ T cells, and LOYSCR tumor cells and LOYSCR CD8+ T cells. P-values for pairwise comparisons between each combinatorial group are also displayed below Kaplan-Meier curves, calculated using log-rank tests. 5h) at 2,5, and 8 years compared to the observed OS rates (%). Each colored dot represents a group of subjects randomly selected from the whole cohort. Each plot includes the total number of subjects (n = 4,010), the number of events (1,203), and 500 subjects per plotted group. a, Representative FACS plots illustrating the sequential gating strategy for live, single-cell tumour suspensions derived from CRISPR Y-KO and CRISPR Y-Scr tumours. tumours were dissociated mechanically, stained with Ghost Red 710 viability dye (Tonbo Biosciences) and antibodies against CD45, CD3, TCRβ, CD11b and CD45R/B220, followed by sorting of the CD3+TCRb+ T cells using the BD S6 flow cytometry machine. Debris and doublets were excluded through sequential gating (All Events → P1 → P2). Within the CD45− compartment (purple in the Alexa Fluor 700 versus APC‐Cy7 panel), TdTomato+ tumour epithelial cells were identified (bottom left). Immune cells displayed minimal presence of TdTomato (very less percentage population), confirming their purity in both Y-KO and Y-Scr groups (top panels). T cells (CD45+CD11b−B220−CD3+TCRβ+) were gated from the CD45+ population (green or orange in the FITC versus BUV805 panel). The final sorted cell populations included: (1) Ghost Red 710−CD45− (containing TdTomato+ tumour cells), (2) Ghost Red 710−CD45+CD11b+B220+ myeloid and B cells (non-T population) and (3) Ghost Red 710−CD45+CD11b−B220−CD3+TCRβ+ T cells. All sorting was performed using a BD FACSymphony S6. The red events represent both CD45− and CD45+ cells, while black events indicate CD45+ immune cells (top panels), which showed minimal tdTomato expression. This confirms the successful separation of tumour epithelial cells from immune cell fractions in both Y-KO and Y-Scr groups. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Chen, X., Shen, Y., Choi, S. et al. Concurrent loss of the Y chromosome in cancer and T cells impacts outcome. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing: Cancer newsletter — what matters in cancer research, free to your inbox weekly.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.scientificamerican.com/article/math-proves-that-everything-really-is-becoming-more-complicated-over-time/'>It's Not Just You. Everything Really Is Getting More Complicated over Time</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.scientificamerican.com', 'title': 'Scientific American'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Whether we're considering global trade, new technologies or scientific research, everything seems to become more complex over time. But today it is far more difficult for a person to excel in multiple domains, in part because of greater specialization. This is not an “everything used to be better” rant. It's just a sober observation: the world used to be simpler. And that is true from a mathematical point of view. Physicist and philosopher Ludwig Boltzmann recognized it back in 1872. Boltzmann studied the behavior of gases and liquids, among other things. Just decades earlier, it had been proposed that everything in the world was made up of tiny building blocks—specifically, atoms and molecules. If you're enjoying this article, consider supporting our award-winning journalism by subscribing. By purchasing a subscription you are helping to ensure the future of impactful stories about the discoveries and ideas shaping our world today. He therefore developed an equation to describe the movement of particles on average. But this led to a mysterious insight: the individual atoms and molecules seem to follow completely different laws than their collective body. Imagine I show you a short video of balls colliding several times and rolling along a magical, friction-less billiard table. Now I ask you, “Did I play the video forward or backward?” It is actually impossible to answer my question. They give the same result both forward and backward. In the 19th century, experts assumed that the particles that make up gases or liquids would also move like small balls through empty space, bumping into one another from time to time and following Newton's laws of motion, just like the aforementioned billiard balls. Today we know that the truth is more complicated. Atoms and molecules obey quantum mechanics, which is much more complex. But interestingly, quantum mechanics is also often assumed to be invariant under time reversal. That means that, like our hypothetical billiard balls, at the atomic and subatomic scale, such as that of molecules of gas, it makes no difference whether you look at the process forward or backward. At the macro level, however, things look quite different. The different substances mix over time to form a homogeneous liquid. This fundamental difference between the micro and macro worlds was Boltzmann's main concern. How can it be that countless equations that describe the movements of individual particles and are invariant under time reversal cause irreversible behavior? If every collision between individual particles can be theoretically reversed, why can't the milk be separated from the coffee? If you wait long enough, an equilibrium is reached at some point, and the particles all move at the same speed on average. There may be individual outliers, but on average, the particles have roughly the same speed and are evenly distributed in space. Boltzmann was able to express parts of this behavior with an equation that is now named after him. This so-called Boltzmann equation shows how the speed and distribution of particles in space changes depending on time and place. He also introduced a “collision operator,” which, depending on conditions such as density or temperature, takes into account the effects of elastic collisions on the particles. This means that it contains derivatives, among other things. To do this, he first had to define what was meant by the term “complex.” A homogeneous mixture of milk and coffee does not seem particularly complex at first. From a mathematical and physical point of view, however, it is. This means that even if the temperature, density, volume and mixing ratio of the mixture are known precisely, it is not possible to deduce exactly how the molecules are arranged or what their respective speeds are. Many different microscopic states lead to the same end result. If milk and coffee are separate, their system's complexity—and therefore the entropy—is low. Their system's complexity and entropy therefore increase steadily over time. Boltzmann both described this qualitatively and derived it mathematically with his equation. This principle applies to our universe as a whole. This means that entropy—and therefore complexity—is constantly and inexorably increasing over time. This article originally appeared in Spektrum der Wissenschaft and was reproduced with permission. Manon Bischoff is a theoretical physicist and an editor at Spektrum der Wissenschaft, the German-language sister publication of Scientific American.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/archaeology/a64946680/ancient-city-cloud-forest/'>Archaeologists Were Searching a Forest in the Clouds—and Found 100 Structures From an Ancient City</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 12:00:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Rio Abiseo National Park sits over 10,000 feet above sea level in Peru's San Martin region of the Andes Mountains. Plenty of its land is known to hold ancient discoveries, and archaeologists are still uncovering more. According to a release from the World Monuments Fund, a team of archaeologists explored the Gran Pajatén area within the Rio Abiseo National Park, a UNESCO Mixed World Heritage site recognized for both cultural richness and natural beauty. The civilization was known for sophisticated urban centers, ceremonial platforms, cliffside burial structures, and agricultural terraces. The regional chiefdom society also had a distinctive architectural and artistic language that remains on display in circular buildings, geometric friezes, and decorated burials. From 2022 through 2024, researchers used aerial and manual LiDAR scanning, photogrammetry, topographic registration, and technomorphological analysis to create a detailed map of Gran Pajatén—a site with ceremonial buildings decorated with high-relief friezes, views of the cloud forest, and stone mosaics depicting human figures. “What makes this moment so meaningful is not only the scale of what's been uncovered, but how we were able to do it,” Benedicte de Montlaur, president and CEO of WMF, said in a statement. “By using advanced technology, our team was able to gather extraordinary visual and scientific documentation that brings Grant Pajaten to life—all while preserving its delicate environment.” “This discovery radically expands our understating of Gran Pajaten and raises new questions about the site's role in the Chachapoya world,” Juan Pablo de la Puente Brunke, executive director of WMF in Peru, said in a statement. Many of the sites, though, are fragile, overgrown by vegetation. While there, the archaeological team undertook conservation interventions to reinforce stairs and stone reliefs, along with a partial reassembly of a perimeter wall. “Though the site itself remains out of reach for most,” de Montlaur said, “these tools will allow us to share its stories widely through thoughtful, immersive digital storytelling.” Tim Newcomb is a journalist based in the Pacific Northwest. He covers stadiums, sneakers, gear, infrastructure, and more for a variety of publications, including Popular Mechanics.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/d41586-025-01738-0'>Genomics pioneer fired from firm he founded: ‘It was not easy to domesticate me'</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 10:05:25
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. You can also search for this author in PubMed Google Scholar You have full access to this article via your institution. Kári Stefánsson's genomics firm deCODE became an early leader in conducting genome-wide association studies.Credit: Halldor Kolbeins/AFP via Getty It's been a month since Kári Stefánsson, the founder and former chief executive of the pioneering Icelandic genomics company deCODE genetics, says he was summarily fired by deCODE's parent company, the biotechnology giant Amgen, and Stefánsson is his usual candid — some have said abrasive — self. “A significant proportion of my old friends would agree with that.” How a trove of cancer genomes could improve kids' leukaemia treatment How a trove of cancer genomes could improve kids' leukaemia treatment The firm was an early leader in conducting genome-wide association studies (GWAS) that linked common single-letter DNA variations to people's risk of developing complex diseases such as type 2 diabetes1 and schizophrenia2 , as well as other traits (I attended a talk in the 2000s at which Stefánsson said they had even found a gene variant associated with enjoying crossword puzzles). Thanks mainly to deCODE's efforts, Icelanders accounted for 11.5% of GWAS participants between 2007 and 2017, one study found3. “The amazing thing is that in this tiny little country with a population of about 390,000 people, we managed to build up a genetics organization that was second to none,” Stefánsson says. Stefánsson says that deCODE has operated more or less autonomously from Amgen for most of the past 12 years. “We have had a very free-spirited atmosphere where people have worked on the assumption that chaos is a very important beginning point to all research you do,” Stefánsson says. Stefánsson attributes his downfall to a recent shift in deCODE's relationship with Amgen, which, he says, sought closer integration between the two. “When you are working in a very goal-oriented work environment to serve drug discovery and development, this is a very noble activity, but you lose some of that freedom.” Stefánsson says he was blindsided when, on 1 May, while he was visiting Amgen's headquarters in Thousand Oaks, California, the company's vice-president for research and development, Jay Bradner, told him his services were no longer needed at deCODE. “I was asked to come to another building, put into a room where Jay Bradner sat in front of me and said, ‘You are fired', basically. Before I arrived back in Iceland, they had sent out a press release announcing my departure from deCODE,” he says. Amgen declined to comment on Stefánsson's characterization of his dismissal. 23andMe plans to sell its huge genetic database: could science benefit? In vivo haemopoietic stem cell gene therapy enabled by postnatal trafficking Programmable control of spatial transcriptome in live cells and neurons Black Death bacterium has become less lethal after genetic tweak This ancient pathogen became deadlier when humans started wearing wool Biogen is seeking a Director to drive the strategic and operational execution of our clinical development plans. Biogen is seeking a Director to drive the strategic and operational execution of our clinical development plans. 23andMe plans to sell its huge genetic database: could science benefit? An essential round-up of science news, opinion and analysis, delivered to your inbox every weekday. Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41591-025-03719-2'>Individual variations in glycemic responses to carbohydrates and underlying metabolic physiology</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 09:05:39
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Elevated postprandial glycemic responses (PPGRs) are associated with type 2 diabetes and cardiovascular disease. PPGRs to the same foods have been shown to vary between individuals, but systematic characterization of the underlying physiologic and molecular basis is lacking. We measured PPGRs using continuous glucose monitoring in 55 well-phenotyped participants challenged with seven different standard carbohydrate meals administered in replicate. We also examined whether preloading a rice meal with fiber, protein or fat (‘mitigators') altered PPGRs. We performed gold-standard metabolic tests and multi-omics profiling to examine the physiologic and molecular basis for interindividual PPGR differences. Overall, rice was the most glucose-elevating carbohydrate meal, but there was considerable interindividual variability. Individuals with the highest PPGR to potatoes (potato-spikers) were more insulin resistant and had lower beta cell function, whereas grape-spikers were more insulin sensitive. Rice-spikers were more likely to be Asian individuals, and bread-spikers had higher blood pressure. Mitigators were less effective in reducing PPGRs in insulin-resistant as compared to insulin-sensitive participants. Multi-omics signatures of PPGR and metabolic phenotypes were discovered, including insulin-resistance-associated triglycerides, hypertension-associated metabolites and PPGR-associated microbiome pathways. These results demonstrate interindividual variability in PPGRs to carbohydrate meals and mitigators and their association with metabolic and molecular profiles. One in three adults in the United States has prediabetes, and 70% of these will develop type 2 diabetes (T2D), posing a substantial public health burden1 via complications such as kidney disease, vision loss, neuropathy, cardiovascular disease (CVD) and cancer2,3,4,5,6. High postprandial glycemic responses (PPGRs) are a hallmark of prediabetes and T2D and are risk factors for T2D, CVD and all-cause mortality independent of fasting blood glucose (FBG) and HbA1c7,8,9. However, our understanding of glucose dysregulation, especially regarding PPGRs, remains incomplete. Interindividual variability in PPGRs to the same foods has been described7,10. A variety of factors have been identified as contributors to PPGRs, including glycemic index, the total carbohydrate amount, carbohydrate characteristics (starch and simple carbohydrates), food processing, meal macronutrient composition, meal timing, ethnicity and the microbiome11,12,13,14,15,16. However, the contribution of individual physiologic and metabolic factors to PPGRs and whether they interact with food characteristics have not been well studied. A gap in understanding how underlying metabolism and physiology affect PPGRs exists largely because quantifying metabolic functions, such as insulin resistance, insulin secretion and the incretin effect, is laborious and costly and has not been extensively employed. However, these metabolic traits differ between individuals17,18,19 and likely explain, in part, the observed interindividual PPGR differences. Glycemic responses to the oral glucose tolerance test have been linked to underlying physiology such as insulin resistance and beta cell dysfunction19. Prespecified timed blood draws may miss the peak glucose, but continuous glucose monitoring (CGM) devices enable a detailed evaluation of PPGRs and can be linked to individual metabolic traits. Indeed, using CGM, we previously showed that individuals with higher and more variable daily glucose excursions were more insulin resistant (IR) than their lower-glycemic counterparts10. To further study individual PPGRs to foods and their association with metabolic subtypes and omics profiles, we conducted a rigorous investigation of responses to a wide variety of standardized carbohydrate meals and standardized preloads (‘mitigators') to a rice meal in individuals whose metabolic traits were comprehensively profiled with gold-standard tests and whose blood and stool were profiled for metabolites, lipids, proteins and microbiome. We hypothesized that individual PPGRs are associated with the underlying metabolic physiology (for example, insulin resistance and beta cell dysfunction) as well as molecular markers. The overall study design is illustrated in Fig. A cohort of 55 participants with no prior history of T2D underwent deep metabolic phenotyping using gold-standard tests for insulin resistance by steady-state plasma glucose (SSPG), beta cell function by disposition index, hepatic insulin resistance and adipocyte insulin resistance (Methods)19,20,21,22,23,24,25,26,27,28. At baseline, we collected metabolomics, lipidomics, proteomics, microbiome and clinical data. a, Overview of study design and data types. Left, at the baseline, the data on omics and clinical tests were collected. Mid-bottom, each individual ate seven different carbohydrate meals and three different mitigator foods with rice with CGM data and food log collected. Mid-top, an example CGM curve of a day is presented. Right, participants were then stratified into carb-response types based on which meal produced the highest spike and into metabolic traits results. b, Mean CGM curves of PPGR after different meals. c, Glucose AUC above baseline and time from baseline to peak for different meals. CGM curves are extracted into different features (and presented with mean and standard errors (error bar indicates 2 standard errors). Bars of standardized carbohydrate meals were ordered by mean value. The number of CGM curves (each dot) is as follows: rice 115, potatoes 92, bread 99, pasta 65, grapes 98, beans 46 and mixed berries 53. Extracted CGM features are compared between different carbohydrate meals. Each participant was instructed to eat each meal at least twice on two different days. d, Heatmap of extracted CGM PPGR features. Each element is the mean value of extracted CGM features. Each column (feature) is then centered and scaled. e, Association between different extracted features and nutrient contents of the standardized carbohydrate meal. For fiber, proteins and fat in study meals, it's the Spearman correlation with statistics calculated from the R package ‘correlation' under default settings with asymptotic t approximation with a two-sided test. The last column is treated as binary (starch versus non-starch food) with correlation (R) obtained from a simple linear model and P value obtained from a two-sided t-test of the coefficient. P values were FDR-corrected for the whole matrix. Illustration in a created with BioRender.com. To quantify individual PPGRs to different carbohydrates and evaluate potential mitigation, participants underwent seven different standardized carbohydrate meal tests (50 g) and three preloaded mitigator meal tests while wearing a CGM at home and following their regular diet otherwise (Fig. The seven carbohydrate meals, each containing 50 g total carbohydrates, included five starchy meals—rice (jasmine rice), bread (buttermilk bread), potatoes (shredded), pasta (macaroni, precooked per instructions, cooled and frozen) and beans (canned black beans, high fiber)—as well as two different simple-carbohydrate meals—mixed berries (blackberries, strawberries and blueberries, high fiber) and grapes (low fiber) (Fig. 1a and Supplementary Table 6) (details in Methods). The three preloaded mitigators included pea fiber, egg white and cream, representing fiber, protein and fat, respectively (Methods), which were consumed 10 min before a standardized 50 g carbohydrate meal (for example, rice). Each meal test was performed at least twice. After processing, there were 848 CGM curves (55 participants), 267 CGM curves with mitigators (47 participants, 32 for rice + mitigators) and 33,920 glucose time points in total. None of the participants had a prior medical history of diabetes; 27 participants were euglycemic, 26 had prediabetes, and one had T2D based on HbA1c measured during the visit (6.5%) (one was unknown). Eighteen were IR (SSPG ≥ 120 mg dl−1), 25 were insulin sensitive (IS; SSPG < 120 mg dl−1), and the mean SSPG and disposition index were 120.1 ± 72.9 mg dl−1 (mean ± s.d.) We assessed the variability between CGM curve replicates for the same meals consumed by the same individual on different days (Supplementary Fig. CGM curves from most individuals were reproducible between replicates based on the area under the curve above the baseline (AUC(>baseline)) with intraindividual correlation coefficients (ICCs) of AUC(>baseline) from 0.26 for beans to 0.73 for pasta, indicating consistency between meal replicates of the same individual except for meals with low PPGRs (Supplementary Table 9)13. The correlation coefficients between replicates were also similar to previously published studies7. To reduce the effect of within-individual variation, results below were computed based on the average of each participant's replicates. The seven different standardized carbohydrate meals with equivalent carbohydrate loads elicited different PPGRs, emphasizing the impact of food composition (Fig. Typical postprandial CGM curves of standardized carbohydrate meals slightly decreased initially after meal consumption and then increased to a single maximum and returned to baseline within three hours (Fig. Of the five starchy carbohydrate meals, rice, bread and potatoes produced high peaks at approximately 1 hour post consumption. Grapes had high glucose peaks that reached maxima at earlier time points. Beans and pasta reached maxima at similar times as other starchy meals but exhibited significantly lower peaks, with beans eliciting the lowest (Fig. Mixed berries reached maxima at similar times as grapes but had much lower peaks. Beans and mixed berries produced the lowest peak measured by both AUC(>baseline) and delta glucose peak (the difference between the peak and baseline glucose) as compared with other meals. From CGM curves, we extracted 11 different features of principally two types: (1) scale (for example, AUC(>baseline) and delta glucose peak) and (2) rate (for example, time baseline to peak and time return to baseline) (Fig. Different AUC-based features had similar patterns, which were similar to that of delta glucose peak, except for grapes (Fig. 1c,d and Extended Data Fig. Time baseline to peak, time return to baseline and delta glucose peak at 170 min show similar patterns. Using scale and rate dimensions, three exemplar meals (beans, mixed berries and rice) were clustered into partially separated groups (Extended Data Fig. Across different meals, delta glucose peak (Fig. 2a) and AUC(>baseline) (Supplementary Fig. 7a) were also positively correlated with each other. Besides single-meal features, we also defined the potato versus grape ratio (PG-ratio), which is the ratio between the delta glucose peak of potatoes and grapes, highlighting the differences between a starchy carbohydrate with a high amount of resistant starch and a simple-carbohydrate meal. a, The Spearman correlation between PPGRs to all different meals, including both the delta glucose peak of standardized carbohydrate meals and the suppression effect of mitigator foods. The suppression mitigator effects were defined by subtracting the delta glucose peak of rice + mitigator from that of rice and then normalizing it by that of rice. b, Examples of CGM curves of participants with different carbohydrate meals with the highest PPGR (carb-response type). c, Number of participants assigned to each carb-response type. The carb-response types are defined by both delta glucose peak and AUC(>baseline). e, Radar plot of delta glucose to different carbohydrate meals in an IS (SSPG 61 mg dl−1) and an IR (SSPG 239 mg dl−1) participant. Delta glucose values were averaged between replicates and scaled by the carbohydrate with the highest value. Insulin resistance, disposition index (DI) and HbA1c were also presented. PPGRs to different carbohydrate meals were associated with the nutrient composition of the standardized meal. The total dietary fiber content (measured in grams) in the carbohydrate meal negatively correlated with AUC(>baseline) (−0.71, 95% confidence interval (−0.96,0.11), P = 0.088) and delta glucose peak (−0.75, 95% confidence interval (−0.96,0.04), P = 0.066) (Fig. PPGRs varied considerably across individuals, which differed from the conventional idea of a fixed glycemic index value for a single food (Figs. Although overall the PPGRs tended to correlate positively across carbohydrate meals (Fig. 2a), for different individuals, PPGRs to different meals were ranked differently; and for each individual, different meals produced the highest PPGR (Fig. For example, 19 participants exhibited the highest average delta glucose peak from rice (participant #18), whereas others were highest with bread, grapes, potatoes or pasta (Extended Data Fig. We aimed to type individuals based on their PPGRs to different carbohydrate meals. Participants were stratified as carb-response types (also referred to as ‘spikers') according to the meal that produced the highest average PPGR for that individual, as measured by delta glucose peak and AUC(>baseline). Using the delta glucose peak, rice-spikers were the largest group (35%), followed by bread-spikers (24%) and grape-spikers (22%); no participants had the highest spike in response to beans or mixed berries (Fig. 2c), possibly due to higher fiber and/or protein (beans) content (Fig. Stratification based on AUC(>baseline) largely agreed with results based on delta glucose peak, except that even more individuals were classified as rice-spikers and fewer as grape-spikers (Fig. Carb-response-type classification was stable between replicates of meals, and results based on individual meal replicates largely agree with those based on the average (Extended Data Fig. In addition, ICC for delta glucose peak and AUC(>baseline) was moderate to high for most meals (Supplementary Table 9). ICC values were lower for meals with smaller PPGRs (for example, beans), potentially because of a lower signal-to-noise ratio. We investigated individuals' PPGRs as a function of their underlying metabolic physiological traits. PPGRs, measured as both delta glucose peak and AUC(>baseline), were significantly greater in IR (measured by SSPG) compared with IS individuals for potatoes and pasta (179% and 152% for delta glucose, respectively; Figs. We measured starch content and found that potatoes had significantly more resistant starch and pasta had significantly more slowly digestible starch than other starchy foods (bread and rice) (Extended Data Fig. The IR group also exhibited a higher baseline (premeal) glucose and higher absolute glucose peaks compared with IS individuals (Fig. Individuals with impaired beta cell function also demonstrated higher PPGRs to potatoes (Figs. The contributions of both insulin resistance and beta cell function were still significant after adjusting for confounding factors, BMI, age and sex (Supplementary Table 2). When modeled together, only SSPG remained an independent predictor of PPGRs. Normoglycemic IR individuals also exhibited higher PPGRs to potatoes than IS individuals with prediabetes (based on HbA1c), indicating the dominance of the insulin-resistance effect (Supplementary Figs. Other differences noted in potato-spikers included higher HbA1c, FBG and free fatty acids (FFA) during the SSPG test (Fig. Grape-spikers, conversely, exhibited significantly lower SSPG (indicating insulin sensitivity) and the lowest FBG. Among individuals who are IS or with normal beta cell function, grape- and rice-spikers were the most frequent groups. a, Mean CGM curves between IR and IS groups measured by SSPG (i), and groups with normal and dysfunctional beta cells measured by disposition index (ii). b, Delta glucose peaks after eating potatoes and pasta are compared between groups with IR and IS, and delta glucose peaks after eating potatoes are compared beween groups with normal and dysfunctional (dys) beta cell functions. PH from left to right is 3.4 × 10−4, 0.02 and 9.4 × 10−3. Extracted CGM features for each participant are compared between different metabolic subtypes. c, Clinical and metabolic characteristics according to carb-response types. Number of participants for (i) ((ii), (iii)): rice-spiker 18 (13, 19), grape-spiker 12 (10, 12), pasta-spiker 2 (1, 2), bread-spiker 13 (13, 13), potato-spiker 8 (6, 8). Clinical measurements were compared between carb-response-types. Data beyond the whiskers are outliers. d, Average delta glucose peak and AUC(>baseline) between participants with IR and IS. e, Average delta glucose and AUC(>baseline) between participants with different beta cell functions (normal and dysfunctional). We observed that whereas the PPGR to potato consumption differed between IR and IS groups and between beta cell normal and dysfunctional groups, the PPGR to grape consumption changed little (Fig. We thus calculated the PG-ratio based on delta glucose peak as a normalization of the PPGR to potatoes. The PG-ratio was consistently different between IR and IS subtypes (Supplementary Fig. The sex interaction with beta cell function and insulin resistance was also significant in estimating the PG-ratio, with males having less effect from insulin resistance (intersection term coefficient −0.49) and beta cell dysfunction (intersection term coefficient −0.58) (Supplementary Table 2). To test the potential suppressing effect of other foods on PPGRs to standardized carbohydrate meals, other meals containing fiber, protein and fat (mitigators) were tested. Thirty-two individuals consumed fiber (10 g, via pea fiber), protein (10 g, via egg white) or fat (15 g, via cream) 10 min before the consumption of white rice (50 g of carbohydrates), the most glucose-elevating carbohydrate meal on average. The mitigation effect was calculated as a normalized difference based on PPGRs (delta glucose peak and AUC(>baseline)) of rice + mitigator relative to rice alone. A negative mitigation effect indicated that consuming the mitigator reduced the rice PPGR. The effects of different mitigators positively correlated with each other with a correlation of 0.79 between fat and protein and approximately 0.5 between fiber and others (Fig. Across the entire cohort, all three mitigators, when consumed before rice, weakly but significantly decreased the delta peak glucose (Cohen's d: fiber 0.12, protein 0.19, fat 0.05). Fat, but not protein or fiber, increased time (baseline to peak) (Fig. However, similar to the interindividual variability in PPGRs to different carbohydrate meals (carb-response types), highly divergent patterns were observed in mitigation effects between individuals (Fig. Many individuals exhibited mild mitigation effects (for example, participants #18 and #48), one had strong mitigation effects (participant #14), and some even had higher glycemic responses with mitigators (for example, participants #33 and #69) (Fig. For most participants, (fiber 23, protein 22 and fat 20) mitigators reduced glycemic responses (Fig. The mitigation effect of protein was significantly greater in the rice-spikers versus non-rice-spikers (Fig. a, Mean CGM curves of PPGRs to rice and rice preceded by three different mitigators (fiber, protein, fat). Each dot is a CGM curve. c, Mean CGM curves of mitigator test between individuals who are IR and IS. d, Delta glucose for rice and rice + mitigators in IS, IR, normal beta cell and dysfunctional beta cell groups. e, Examples of participants with different mitigation effects. Positive (negative) values indicate increasing (decreasing) delta glucose in the combination meal compared with rice. g, Mitigator foods and their relative mitigation effects. The P value of the t-test (default, two-sided) between rice-spiker and non-rice-spiker is FDR-adjusted. CGM features are presented with mean and two standard errors in b and d. Boxplots in g show the center line as the median and the hinges as the 25th and 75th percentiles. Data beyond the whiskers are outliers. IR individuals showed little beneficial response to mitigators, whereas IS individuals responded more to mitigators, with fiber decreasing delta glucose peak (P = 0.031) and AUC(>baseline) (P = 0.057) (Fig. Individuals with normal beta cell function also had beneficial protein mitigation effects on delta glucose peak (P = 0.028) (Fig. We searched for other potential factors that contributed to differential PPGRs, including demographic and clinical data. In total, 15 continuous variables and three categorical variables (ethnicity, sex and BMI groups) were examined. Asian participants were more likely than non-Asian participants to be rice-spikers (Fisher's exact test, P < 0.05, positive false discovery rate (pFDR) = 0.21) (Fig. No other significant associations were found between PPGRs (carb-response types and mitigator effect) and ethnicity, sex or BMI groups (Supplementary Fig. Among the 15 continuous measurements, associations were uncovered between carb-response types using analysis of variance (ANOVA) with a pFDR threshold of 0.3. Bread-spikers exhibited significantly higher systolic and diastolic blood pressure compared to other carb-response types after correcting for BMI and age (Fig. The analysis in this section is exploratory and has higher risks of random variation than results in other analyses. To identify potential biomarkers and mechanisms of carb-response types, we performed baseline omics analyses, including untargeted metabolomics (731 annotated metabolites, nine classes), targeted lipidomics (652 lipids, 13 classes), targeted proteomics (1,470 proteins) and metagenomics microbiome analysis (Extended Data Fig. In addition to higher blood pressure levels, bread-spikers also presented significantly higher N1-Methyladenosine than other participants (Figs. Potato-spikers had higher levels of certain triglycerides (TAG) and fatty acids (Fig. 21), aligning with their increased insulin resistance23,31. In addition, proteins positively associated with potato PPGRs were significantly enriched in hepatic stellate cell activation and fibroblast activation pathways (Supplementary Data 1). In addition, metabolites enriched in the histidine pathway were both positively (1-methyl-histidine, glutamate and methylimidazole acetic acid) and negatively (histidine and imidazole acetaldehyde) associated with bean-PPGR, indicating variability in histidine catabolism (Supplementary Fig. PPGRs to mixed berries were positively associated with unsaturated fatty acid biosynthesis and unsaturated TAGs in plasma metabolomics and lipidomics (Supplementary Data 1). a, Metabolites and lipids that are distinguished between carb-response types. For the carb-response types, ‘Others' include all other participants. *, based on Holm corrected P value of the Mann–Whitney test (default, two-sided) of the selected pairs: *PH ≤ 0.05; **PH ≤ 0.01. N1-Methyladenosine was selected as it had the lowest P value when comparing bread-spikers with others. The TAGs and fatty acids were selected as lipid-related features with pFDR < 0.2. Data beyond the whiskers are outliers. Number of participants in each carb-response type: rice-spiker (15), grape-spiker (6), pasta-spiker (3), bread-spiker (8), potato-spiker (7). Omics measurements were compared between carb-response types. b, Spearman correlation between T2D-related clinical features and microbiome levels (Genus). Red indicates positive correlation, blue indicates negative correlation, and asterisks indicate P value < 0.05 and pFDR < 0.2. Genera with more than one P value < 0.05 association were selected. Sample weight was controlled by partial correlation. FDR correction was implemented for the whole matrix. c, Mediation effect through the same metabolite. The mediation effect was calculated from microbiome to metabolites (mediator) and then to clinical measurements. Illustration in c created with BioRender.com. Microbiome analyses also revealed taxa and functional pathways associated with PPGRs. PPGRs to most carbohydrate meals were positively associated with glycosyl hydrolase family 85 (GH85), lactate utilization domain and heme exporter protein B and negatively associated with d-alanyl-d-alanine carboxypeptidase (Extended Data Fig. In addition, Klebsiella, Sutterella and starch degradation III were negatively or beneficially correlated with the mitigation effect (Supplementary Table 5). To further investigate the potential mechanisms of different PPGRs, we next explored the associations between multi-omics features and metabolic traits. Correlation matrix analyses between omics and clinical measurements revealed two clusters, with some omics features associated with healthy metabolic function and others with compromised metabolic function (Fig. For instance, tyrosine, hydroxybutyric acid, glutamic acid and TAGs were significantly positively associated with plasma glucose or insulin resistance, whereas phosphatidylcholine, lysophosphatidylcholine, lysophosphatidylethanolamine and threonic acid were significantly negatively correlated (Extended Data Figs. Similarly, among microbiomes, Slackia and Tyzzerella were positively correlated with plasma glucose or insulin resistance, and Butyrivibrio, Asaccharobacter, Adlercreutzia and Fusicatenibacter were negatively correlated (Fig. In an integrated statistical model to predict SSPG, strong predictors include fasting insulin, BMI, FFA and TAGs contributing to the IR state and phosphatidylcholine contributing to the IS state (Extended Data Fig. To further trace potential causal mechanisms, we then conducted a mediation analysis with microbiomes as the input, metabolomics and lipidomics as the mediators and clinical measurements as the outcomes (Supplementary Table 4)36,37. In Bacteroides thetaiotaomicron, pathways involving protein autophosphorylation, lactose catabolism and histidine phosphotransfer kinase activity positively mediated hepatic insulin resistance and fasting insulin via a single metabolite with an inferred formula, C9H15NO3, but no annotated match in public databases (Fig. The results of the current study demonstrate differential PPGRs in response to a broad array of standardized carbohydrate meals and reflect underlying metabolic physiology, including insulin resistance and beta cell dysfunction. Using standardized carbohydrate meals given in replicates with CGM data captured, we showed that on average, rice was the most glucose-elevating carbohydrate, whereas some individuals exhibited higher elevations in response to consumption of potatoes, pasta, bread or grapes (Extended Data Fig. Individuals with the highest PPGR to potato consumption relative to other carbohydrates (potato-spikers) were more IR with lower beta cell function, whereas grape-spikers were more IS. Furthermore, a high glucose response to potatoes relative to grapes (PG-ratio) was statistically significantly associated with insulin resistance (Fig. These results indicate that CGM-detected glycemic responses to carbohydrate meals reflect underlying physiology. The results of this study also highlight that the mitigation effect differs between individuals according to underlying physiology, in that the mitigation effect was only beneficial in reducing delta glucose in the IS or normal beta cell function groups (Supplementary Table 8). The specific mechanisms of mitigator function were not studied in detail in this study, but a potential hypothesis is that mitigators can lower PPGRs by at least two main pathways that work in synergy. The first is decreasing carbohydrate absorption, and the second is through the process of incretin induction, insulin secretion and insulin action38. In healthy individuals, mitigators greatly reduce PPGRs because the two mechanisms work in synergy. However, with insulin resistance or beta cell dysfunction, the single effect of absorption rate reduction may be too small without the synergistic interaction with insulin function. In addition to explaining the reduced mitigator effect in individuals with insulin resistance and beta cell dysfunction, this hypothesis also explains the elevated PPGRs to potatoes and pasta in those individuals. The potatoes provided were cooked and cooled, increasing the level of resistant starch, a type of fiber, compared with other starch meals, and so it was itself a starch food with fiber (Extended Data Fig. Pasta, which was also cooked and cooled, contains some resistant starch and also more slowly digestible starch39. Grapes and rice contain lower levels of fiber and thus do not exhibit PPGR differences between individuals with different metabolic traits. Beans have higher levels of fiber and proteins but lower PPGR in general, limiting the possible effect size to be measured. We also observed that protein has the strongest mitigation effect in individuals with beta cell normal function, potentially indicating a stimulation effect on insulin secretion (Fig. Additional factors affecting the PPGRs and mitigation effect might come from individual microbiomes, accessibility of starch in the meal and the interaction between sex and metabolic traits. Carb-response types are linked to glucose dysregulation subtypes and potential complications. Potato-spikers had significantly higher fasting glucose, insulin resistance and fasting levels of several TAGs and fatty acids. Enriched proteomics pathways such as hepatic stellate cell activation and fibroblast activation (Supplementary Data 1) in potato-spikers might be related to the activation of hepatic stellate cells to myofibroblast-like cells and metabolic dysfunction-associated steatohepatitis (MASH)41. In population-wide T2D studies, the severe IR diabetes group was associated with MASH17. We hypothesize that there is a similar group of individuals with higher potato PPGRs, higher insulin resistance and early signatures of MASH in the stages of prediabetes and even normoglycemia, which needs further validation in larger cohorts. In addition, Asian individuals were more likely to be rice-spikers, which aligns with higher rice glycemic responses among Asian compared to European individuals in the study of glycemic indexes15,16. We note that there may be other confounders in genetics and habits (for example, consuming rice frequently). Bread-spikers exhibited significantly higher systolic and diastolic blood pressure and N1-Methyladenosine, a metabolite associated with hypertension in mice, compared to other carb-response types, suggesting potential CVD risk (Figs. The positive correlation between black bean PPGRs and keto metabolism suggests that individuals with baseline predispositions toward enhanced fatty acid oxidation or amino acid catabolism may exhibit different glucose responses to beans. The variability of histidine metabolism associated with PPGRs to beans might be shaped by differences in gut microbiota composition or habitual diet42. Nonetheless, our results demonstrate that different strategies may be useful for nutrient intervention in groups with different ethnicities and metabolic functions. Microbiome features were also associated with PPGRs and mitigation effects. GH85 in Roseburia intestinalis was positively associated with PPGRs to all carbohydrate meals (Extended Data Fig. The GH85 family is known to degrade glycosylated proteins, and Roseburia is known to degrade complex carbohydrates (for example, fiber)34,35. Elevated GH85 in Roseburia intestinalis might indicate degradation of mucins when fiber from meals was insufficient. D-alanyl-D-alanine carboxypeptidase was negatively associated with PPGRs to most carbohydrate meals and has a role in cell wall synthesis. Klebsiella (the highest level in Klebsiella pneumonia) and Sutterella (the highest level in Sutterella parvirubra) were significantly negatively (beneficially) associated with the mitigation effect (Supplementary Table 5). Klebsiella is known to reside in the small intestine, and the PPGR reduction might come from its competing usage of glucose43. Sutterella is a known factor in glucose control improvement after gastric bypass, and this might be related to its mitigation effect44. The starch degradation III in Eubacterium hallii was significantly negatively associated with the mitigation effect (Supplementary Table 5). This might indicate a competing usage of starch when the human digestive process was slowed by mitigators. Overall, a myriad of microbial biochemical pathways in different microbes could likely impact PPGRs. Our work is built on previous PPGR studies and provides a deeper mechanistic understanding of the PPGR. Interindividual PPGR variation and correlation across foods have been discovered in both early small-scale blood-sample-based and later CGM-based studies7,13,45,46. However, early studies were limited to a small variety of standardized meals, so they did not systematically investigate the real-life heterogeneity of PPGRs7,13. Our work demonstrates that PPGRs were more consistent within individuals than between individuals, and each individual can be classified into one carb-response type with different clinical profiles. We also provided a statistical analysis of interindividual and intraindividual variability of PPGRs based on ICCs to a broad scope of carbohydrate meals. We found associations between PPGRs and blood glucose and omics features, and their relationship to the specific metabolic dysfunctional phenotype, including insulin resistance and beta cell function, enabling a deeper understanding of interindividual variation in PPGRs. Limitations of the current study include its size of 55 well-phenotyped individuals in California. Although there was a balanced distribution of participants by sex and race, there were insufficient numbers in each race to identify more race-specific patterns. Larger studies could address these potential differences. Furthermore, for practical reasons, we cannot compare more than seven carbohydrates (for example, whole wheat bread, bananas and cooked and cooled versus hot potatoes) and include mitigator tests for all carbohydrates. Our meals also lacked variability in cooking time and solid state versus liquid state. The mitigator effects presented were systematically studied but still cannot be extrapolated to a mixed meal. Although participants were instructed to remain sedentary for three hours post-meal, we did not track physical activity and evaluate the effects. Our study explored the association between PPGRs and omics and clinical measurements but presented no direct links to hard clinical endpoints47,48. Further studies are needed to quantify the long-term consequences of different PPGR subtypes and intervention effects objectively and directly. Overall, our results represent an initial step in evaluating the effect of different meals and mitigators on PPGRs and their associations with metabolic physiology, especially because individuals, both healthy and at risk, have increased access to CGMs to optimize health by modifying their diet. Our research also may be relevant for stratification-based approaches that leverage CGM data to cluster individuals based on metabolic health and potentially optimize their diet to prevent progression to T2D and CVD. Our potential approach of stratifying participants into subgroups with relatively homogeneous PPGRs could provide advantages of more precise recommendations than the broad public health recommendation and more accessible data collection than the individualized machine-learning approach. Future studies could enable routine meal recommendations through expanded PPGR profiling with an even broader scope of mitigators in bigger cohorts and novel algorithms. Our study aligns with the NIH definition of Basic Experimental Studies in Humans studies and aims to understand the fundamental process of postprandial responses. Male and female (based on self-report) participants of different ethnicities (based on self-report) were recruited from the San Francisco Bay Area, California. Inclusion criteria were age >18 and <75 years, BMI < 40 kg m−2 and no known diagnosis of diabetes. Exclusion criteria included major organ diseases, uncontrolled hypertension (blood pressure > 160/100 mmHg), pregnancy or lactation, use of diabetogenic medications, malabsorptive disorders (for example, celiac sprue), chronic inflammatory conditions, malignancy not at least five years in remission, alcohol use >2 standard drinks per day, use of weight loss medications or specific diets, weight change of more than 2 kg in the last three weeks and a history of bariatric surgery. Additionally, individuals were excluded if they were deemed to be unable to use the technology correctly, adhere to the study protocol or give written informed consent. The first participant consented on 24 May 2018 and the last one on 15 February 2023. Participants underwent evaluations, screening tests and metabolic tests at the Clinical and Translational Research Unit (CTRU) after an overnight fast. Metabolic tests assessed metabolic health related to T2D and included the oral glucose tolerance test, SSPG test and isoglycemic intravenous glucose infusion (IIGI) test19,20,21,22,49. During the screening and the omics visits, stool, urine, peripheral blood mononuclear cells, plasma and serum samples were collected. Some individuals had multiple omics visits to monitor omics changes throughout the study. Participants were then instructed to complete the at-home diet cycles, during which they collected CGM and food log data. The study protocol was reviewed and approved by the Institutional Review Board at Stanford University School of Medicine Human Research Protection Office (Institutional Review Board no. All participants provided written informed consent. Seven standardized carbohydrate meals (50 g carbohydrates each) and three mitigators were included in the study (Supplementary Table 6). We investigated individual responses to different standardized carbohydrate meals and determined whether other meals could mitigate the PPGRs. The five starchy meals were jasmine rice (precooked frozen Trader Joe's brand), potatoes (shredded, frozen precooked, Trader Joe's brand), pasta (macaroni, cooked in the CTRU kitchen per package instructions and then frozen), white sliced bread (Trader Joe's buttermilk bread) and beans (canned whole black beans). The two simple-carbohydrate meals were grapes (red grapes) and mixed berries (170 g raspberries and a mix of blackberries, strawberries and blueberries, about 85 g each). Participants were instructed to use microwaves to heat the meal at home and consume each on separate days without adding any condiments or other meals. They were to consume the standardized meals as the first meal in the morning, after a 10–12 hour fast, not consume any other food or drink and remain sedentary for 3 hours after consumption of the provided meals. For mitigator tests, participants were instructed to consume the mitigator first, followed by the carbohydrate meal (rice) 10 min later. They were also advised to avoid exercise the afternoon and evening before the test. During the diet cycle, participants were instructed to consume the standardized meals as described in the last section while otherwise following their regular diet. Participants were instructed to keep detailed logs of food intake, including the time and portions of all foods and drinks consumed using the ‘Cronometer' app. A dietitian assisted participants in adhering to the cook and eat instructions. Food logs with specific food and time were collected to evaluate adherence to the assigned instructions. Two versions of CGM were used during the study period: Dexcom G4 and G6 Pro. For the G4, calibration was needed, requiring participants to measure blood glucose levels by fingerstick with provided glucose meters, glucose-reading strips and a lancet device. Participants were instructed to calibrate before the standardized meals. The G6 Pro required no calibration. Glucose values were blinded to participants until after the study. Trained study personnel inserted the sensor and educated participants on its use and care. Multiple CGM sensors were provided as needed. Metadata were downloaded from REDCap and cleaned manually. The data included demographic information, clinical blood test results and responses from health and physical activity surveys19,26. The data were homogenized and manually checked with unreasonable values labeled and removed. Missing clinical values (for example, BMI) were estimated if the corresponding independent variables for calculation existed. If multiple values existed, they were averaged across visits. Participants were instructed to fast for 12 hours before each metabolic test. The tests were conducted on three different days. Details of the metabolic tests are available in previous publications19,26. The OGTT measured postprandial glycemic response to a standard glucose drink (75 g glucose)19. Blood samples were collected through an antecubital intravenous catheter before and after the glucose drink at −10, 0, 15, 30, 60, 90, 120, 150 and 180 min. Blood glucose was checked at the aforementioned time points and at 10, 20, 40, 50, 75, 105, 135 and 150 min. Baseline levels of insulin, C-peptide, glucagon-like peptide-1, glucose-dependent insulinotropic polypeptide and glucagon were measured. The two-stage insulin suppression test (called the SSPG test in the paper) quantified insulin suppression of lipolysis (adipose tissue insulin resistance) and insulin-mediated glucose uptake (muscle insulin resistance), as previously described20,21,22,23,49. During the test, participants received an infusion of octreotide (0.27 μg m−2 min−1) for 240 min to suppress endogenous insulin secretion. During the first stage of the test, insulin was infused at 6 mU m−2 min−1 and glucose at 50 mg m−2 min−1, and blood samples were collected at 100, 110 and 120 min for measurement of glucose, insulin and FFA concentrations23. The glucose, insulin and FFA values collected during the last 20 min of the first stage were averaged to calculate SSPG, steady-state plasma insulin and steady-state plasma FFA concentrations. During the second stage of the test, insulin was infused at 32 mU m−2 min−1 and glucose at 267 mg m−2 min−1. Blood samples were drawn at 210, 220, 230 and 240 min to measure the second-stage steady-state plasma insulin and SSPG concentrations. The IIGI test measured the incretin effect, which is the ability of oral nutrients to stimulate insulin secretion beyond that stimulated by comparable plasma glucose administered intravenously27. It quantified the incretin effect by comparison with OGTT measurements. During the IIGI test, an intravenous catheter was placed in the antecubital vein for administration of continuous dextrose infusion at a rate needed to obtain the desired glucose level to match the curve generated by the OGTT. Blood samples were collected from a second intravenous catheter at the same time points as the OGTT, with immediate results generated by a bedside YSI plasma glucose analyzer and the rate of dextrose infusion adjusted as needed to maintain the desired glucose curve that matched values obtained during the OGTT. Plasma from this test as well as the OGTT was collected for measurement of insulin and C-peptide to calculate the incretin effect. Although all participants were free of diabetes by self-report, they were stratified for analysis purposes according to HbA1c drawn at their baseline visit and American Diabetes Association criteria as follows: normoglycemia (HbA1c < 5.7%), prediabetes (HbA1c 5.7%–6.4%) and T2D (HbA1c ≥ 6.5%). SSPG (mg dl−1) measured the level of insulin resistance, with higher values indicating greater insulin resistance20,21,22,23,49. A cutoff at 120 mg dl−1 was used to classify participants into IR and IS groups19,24. This threshold was based on physiology, prospective studies and a natural separation point between clustering observed in our cohort24. Results followed the default cutoff if not otherwise specified. Beta cell function was expressed as disposition index, reflecting insulin secretion adjusted for insulin resistance. The calculation was by insulin SECretion (ISEC) software, based on C-peptide concentrations measured during OGTT at time points 0, 15 and 30 min and with adjustment for age, sex and BMI19,50. Incretin effect was calculated based on the difference in C-peptide concentrations between OGTT and IIGI at seven time points (0, 15, 30, 60, 90, 120 and 180 min)17,24,28. A higher incretin effect may indicate either increased secretion of incretin (for example, glucagon-like peptide-1, glucose-dependent insulinotropic polypeptide) and/or increased beta cell responsiveness to incretin. An alternative cutoff excluded intermediate results (dysfunctional, incretin effect% < 40; normal, incretin effect% > 60). Results followed the default cutoff if not otherwise specified. Hepatic IR was estimated using a surrogate index based on insulin, HDL, BMI, age and sex. We followed previous papers in the calculation of homeostatic model assessment values52. Details of calculations are available in the previous publications19,26. Group 1 had the most metabolic test measurements on most participants and is the one we focused on for metabolic traits. Group-wise comparisons across metabolic subtypes (different insulin resistances, beta cell function and incretin function) were based on group 1 unless otherwise specified, and other correlation analyses are based on all available data. Group 2 had a distribution shift, was mostly IS and had SSPG only among subtype measurements. CGM data were analyzed over a period from 25 min before to 170 min after the consumption of standardized carbohydrate meals. The CGM data were downloaded from Dexcom devices, processed and integrated with food log data. Standardized meals were identified and selected from the Cronometer food logs. Food names were cleaned and unified for the standardized meals. Duplicate records of identical meals were merged. We only kept those mitigator records correctly following the procedure: (1) rice must be consumed within 30 min after the mitigators; (2) rice must be consumed after the mitigators. Other food intake before and during (2 hours) the standardized meal tests was also recorded. Time shifts from daylight saving were automatically recorded and corrected. We kept CGM records with relatively complete data in the time range of interest ([−25,170] minutes) and removed data with more than 30-minute gaps. We then applied a smoothing spline and resampled the data into a uniform time grid of 40 time points. We manually checked each CGM curve and corrected mistakes in dates and times, including traveling between time zones. CGM curves with mistakes, such as discontinuities after device calibration or an indication of noncompliance, were removed. Traveling between time zones often produced an integer-hour time discrepancy between CGM and food log records, which was corrected by shifting integer hours. The reproducibility and consistency of PPGRs were assessed based on AUC(>baseline). The glucose response was analyzed from the time of carbohydrate meal consumption until either 180 or 120 min post consumption. Replicate 1 and replicate 2 data for each carbohydrate meal type were extracted, and Pearson correlation coefficients were calculated. ICC was also calculated to evaluate consistency within replicates of the same individuals. We projected each CGM time series ([0, 170] minutes) into features19. AUC above a certain value (for example, AUC(>baseline) and AUC(>140)) was calculated by shifting the curves to these thresholds and integrating the positive parts. Delta glucose peak (mg dl−1) was defined as the difference between the peak glucose value and the baseline glucose value. Delta glucose peak at a specific time point (for example, delta glucose peak at 120 min) was defined as the difference between the glucose value at the chosen time point and the baseline glucose value. Time (baseline to peak) (min) was defined as the time difference between the baseline time point and peak value. Time (return to baseline) was defined as the minimal time for the glucose level to decrease to or below the baseline glucose level. If the glucose level had not returned to baseline within 170 min, the time was set as 170 min. Slope (baseline to peak) (mg (dL min)−1) was defined as the division between delta glucose peak value and time from baseline to peak. Mitigation effects were measured by subtracting the delta glucose peak of rice + mitigator from the delta glucose peak of rice alone and then normalizing by the rice delta glucose peak. A similar quantification of the mitigation effect was also calculated based on AUC(>baseline). Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. Data are shared through supplementary tables, public repositories and an integrated web interface. This Article focused on participants with CGM and, particularly, with complete metabolic testing, a subset of the whole cohort. The participant IDs were also reassigned to prevent identification. Metabolomics and lipidomics data were deposited into Metabolomics Workbench with IDs ST003630 and ST003636 (ref. Microbiome data were deposited into NCBI SRA under PRJNA1189566. 54), which will be open after publication. Clinical data, CGM data and omics data were also shared in integration through a web interface for the whole cohort (NCT03919877) when consent for data sharing was obtained from the participants. New types of data will be shared through the web interface when finalized. This manuscript also used datasets/databases including human genome GRCh38 (https://benlangmead.github.io/aws-indexes/bowtie) and metid based database-mapping55. Source data are provided with this paper. The analysis is based on R, Python and MATLAB, and detailed information is provided in the supplementary files. The used packages include Rcpm v.1.0.4, masscleaner v.1.0.6, lipidr v.2.12.0, SummarizedExperiment v.1.28.0, GenomicRanges v.1.50.1, GenomeInfoDb v.1.34.2, MatrixGenerics v.1.10.0, matrixStats v.0.62.0, pathview v.1.38.0, ggfortify v.0.4.15, ComplexUpset v.1.3.3, Rodin v.0.1.43, visNetwork v.2.1.2, gridExtra v.2.3, mediation v.4.5.0, sandwich v.3.0-2, mvtnorm v.1.1-3, Matrix v.1.5-3, doSNOW v.1.0.20, snow v.0.4-4, iterators v.1.0.14, KEGG.db v.2.7.1, MSnbase v.2.24.0, ProtGenerics v.1.30.0, mzR v.2.32.0, Rcpp v.1.0.9, massdataset v.1.0.19, masstools v.1.0.8, metid v.1.2.25, metid v.0.2.0, metpath v.1.0.5, ReactomePA v.1.42.0, org.Hs.eg.db v.3.16.0, AnnotationDbi v.1.60.0, IRanges v.2.32.0, S4Vectors v.0.36.0, Biobase v.2.58.0, BiocGenerics v.0.44.0, clusterProfiler v.4.6.0, ppcor v.1.1, MASS v.7.3-57, igraph v.1.3.5, psych v.2.2.9, reshape2 v.1.4.4, lsr v.0.5.2, circlize v.0.4.15, ggpubr v.0.6.0, ComplexHeatmap v.2.14.0, data.table v.1.14.4, dplyr v.1.0.10, readxl v.1.4.1, npreg v.1.0-9, Hmisc v.5.1-0, tidyr v.1.3.0, ggplot2 v.3.4.0, magrittr v.2.0.3, foreach v.1.5.2, stringr v.1.5.0, caret v.4.4.0, glmnet v.4.4.0, pandas 2.0.3, matplotlib 3.5.1, seaborn 0.11.2, numpy 1.22.3 and scipy 1.7.3. The codes of this study are shared through open-source GitHub (https://github.com/mikeaalv/cgm_meal_manuscript). Bansal, N. Prediabetes diagnosis and treatment: a review. Klil-Drori, A. J., Azoulay, L. & Pollak, M. N. Cancer, obesity, diabetes, and antidiabetic drugs: is the fog clearing? Coles, B. et al. Cardiovascular events and mortality in people with and without type 2 diabetes: an observational study in a contemporary multi-ethnic population. Retnakaran, R. et al. Risk factors for renal dysfunction in type 2 diabetes: U.K. Prevalence and risk of diabetic retinopathy when age at diagnosis is 30 or more years. Callaghan, B. C. et al. Role of neurologists and diagnostic tests on the management of distal symmetric polyneuropathy. Zeevi, D. et al. Personalized nutrition by prediction of glycemic responses. Cavalot, F. et al. Postprandial blood glucose predicts cardiovascular events and all-cause mortality in type 2 diabetes in a 14-year follow-up: lessons from the San Luigi Gonzaga diabetes study. Implications of postprandial glucose and weight control in people with type 2 diabetes: understanding and implementing the International Diabetes Federation guidelines. Hall, H. et al. Glucotypes reveal new patterns of glucose dysregulation. Factors affecting interpretation of postprandial glucose and insulin areas. Berry, S. E. et al. Human postprandial responses to food and potential for precision nutrition. Perera, A., Meda, V. & Tyler, R. T. Resistant starch: a review of analytical protocols for determining resistant starch and of factors affecting the resistant starch content of foods. Wolever, T. M., Giddens, J. L. & Sievenpiper, J. L. Effect of ethnicity on glycaemic index: a systematic review and meta-analysis. Kataoka, M. et al. Glycaemic responses to glucose and rice in people of Chinese and European ethnicity. Novel subgroups of adult-onset diabetes and their association with outcomes: a data-driven cluster analysis of six variables. Zaghlool, S. B. et al. Metabolic and proteomic signatures of type 2 diabetes subtypes in an Arab population. Prediction of metabolic subphenotypes of type 2 diabetes via continuous glucose monitoring and machine learning. Shen, S. W., Reaven, G. M. & Farquhar, J. W. Comparison of impedance to insulin-mediated glucose uptake in normal subjects and in subjects with latent diabetes. Greenfield, M. S., Doberne, L., Kraemer, F., Tobey, T. & Reaven, G. Assessment of insulin resistance with the insulin suppression test and the euglycemic clamp. Pei, D., Jones, C. N., Bhargava, R., Chen, Y. D. & Reaven, G. M. Evaluation of octreotide to assess insulin-mediated glucose disposal by the insulin suppression test. McLaughlin, T., Yee, G., Glassford, A., Lamendola, C. & Reaven, G. Use of a two-stage insulin infusion study to assess the relationship between insulin suppression of lipolysis and insulin-mediated glucose uptake in overweight/obese, nondiabetic women. Yeni-Komshian, H., Carantoni, M., Abbasi, F. & Reaven, G. M. Relationship between several surrogate estimates of insulin resistance and quantification of insulin-mediated glucose disposal in 490 healthy nondiabetic volunteers. Bergman, R. N., Ader, M., Huecking, K. & Van Citters, G. Accurate assessment of beta-cell function: the hyperbolic correction. Park, H. et al. Lifestyle profiling using wearables and prediction of glucose metabolism in individuals with normoglycemia or prediabetes. Nauck, M. A., Vardarli, I., Deacon, C. F., Holst, J. J. & Meier, J. J. Secretion of glucagon-like peptide-1 (GLP-1) in type 2 diabetes: what is up, what is down. Vangipurapu, J. et al. A novel surrogate index for hepatic insulin resistance. Expression of specific inflammasome gene modules stratifies older individuals into two extreme clinical and immunological states. Xiong, W. et al. N1-methyladenosine formation, gene regulation, biological functions, and clinical relevance. Raju, S. C. et al. Microbial-derived imidazole propionate links the heart failure-associated microbiome alterations to disease severity. Miao, G. et al. Longitudinal plasma lipidome and risk of type 2 diabetes in a large sample of American Indians with normal fasting glucose: the strong heart family study. Morze, J. et al. Metabolomics and type 2 diabetes risk: an updated systematic review and meta-analysis of prospective cohort studies. Qin, J. et al. A metagenome-wide association study of gut microbiota in type 2 diabetes. Zhou, X. et al. Longitudinal profiling of the microbiome at four body sites reveals core stability and individualized dynamics during health and disease. Goff, H. D., Repin, N., Fabek, H., El Khoury, D. & Gidley, M. J. Dietary fibre for glycaemia control: towards a mechanistic understanding. Method of food preparation influences blood glucose response to a high-carbohydrate meal: a randomised cross-over trial. Oral L-arginine stimulates GLP-1 secretion to improve glucose tolerance in male mice. Tang, Y., Zheng, S. & Chen, A. Curcumin eliminates leptin's effects on hepatic stellate cell activation via interrupting leptin signaling. Cuparencu, C. et al. Biomarkers of meat and seafood intake: an extensive literature review. Kastl, A. J. Jr., Terry, N. A., Wu, G. D. & Albenberg, L. G. The structure and function of the human small intestinal microbiota: current understanding and future directions. The genus Sutterella is a potential contributor to glucose metabolism improvement after Roux-en-Y gastric bypass surgery in T2D. Vrolix, R. & Mensink, R. P. Variability of the glycemic response to single food products in healthy subjects. Vega-Lopez, S., Ausman, L. M., Griffith, J. L. & Lichtenstein, A. H. Interindividual variability and intra-individual reproducibility of glycemic index values for commercial white bread. & Ruiz-Canela, M. The Mediterranean diet and cardiovascular health. Brand-Miller, J., Hayne, S., Petocz, P. & Colagiuri, S. Low-glycemic index diets in the management of diabetes: a meta-analysis of randomized controlled trials. Role of insulin resistance in human disease. & Young, M. A. ISEC: a program to calculate insulin secretion. Regulation of hepatic glucose metabolism in health and disease. Matthews, D. R. et al. Homeostasis model assessment: insulin resistance and beta-cell function from fasting plasma glucose and insulin concentrations in man. Sud, M. et al. Metabolomics Workbench: an international repository for metabolomics data and metadata, metabolite standards, protocols, tutorials and training, and analysis tools. The PRIDE database at 20 years: 2025 update. Shen, X. et al. metID: an R package for automatable compound annotation for LC–MS-based data. We thank all participants, investigators and CTRU nurses and staffs of the CGM1.0 study. We thank A. Chen and L. Stainton for administrative support. This work was supported by NIH/NIDDK grant no. This research is also supported by the American Diabetes Association Grant 11-23-PDF-76 (Y.W.). It is also supported by National Institute of Diabetes and Digestive and Kidney Diseases of the National Institutes of Health grant no. and National Library of Medicine (grant no. was supported by the NIH institutional research training grant no. NIH 2T32HL09804911 and the Stanford Lifestyle Medicine grant. Research reported in this publication was supported by the National Center for Advancing Translational Sciences of the National Institutes of Health under grant no. This work was partially supported by proteomic profiling conducted through NIH/NIDDK grant no. This study was also supported by Stanford Center for Clinical and Translational Research and Education: MedTech Pilot Grant. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Present address: Google LLC, Mountain View, CA, USA These authors contributed equally: Yue Wu, Ben Ehlert, Ahmed A. Metwally. These authors jointly supervised this work: Tracey McLaughlin, Michael P. Snyder Yue Wu, Ahmed A. Metwally, Dalia Perelman, Heyjun Park, Andrew Wallace Brooks, Basil Michael, Samuel M. Lancaster, Daniel Hornburg, Lucia Ramirez, Frank Wong, Denver Bradley, Tao Wang, Lettie McGuire, Helge Ræder & Michael P. Snyder Systems and Biomedical Engineering Department, Cairo University, Giza, Egypt Dalia Perelman, Fahim Abbasi, Alessandra Celli, Caroline Bejikian, Ekrem Ayhan, Ekanath Srihari Rangan & Tracey McLaughlin Department of International Health, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USA Fahim Abbasi, Tracey McLaughlin & Michael P. Snyder Samuel M. Lancaster, Frank Wong, Denver Bradley & Michael P. Snyder Ultima Genomics, Newark, CA, USA David Bogumil, Sarah Pollock, Zohar Shipony & Doron Lipson Amrita Vishwa Vidyapeetham (University), Kochi, India Department of Pediatrics, Haukeland University Hospital, Bergen, Norway You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar conducted the main data analysis. contributed to clinical data curation. conducted CGM data curation and quality control. contributed to the sample arrangement for omics collection and assays. contributed to microbiome data collection. contributed to microbiome data preprocessing. and D.H. ran the lipidomics analysis. recruited patients and conducted clinical tests. contributed to the website development implementation. contributed to the data sharing and visualization. Correspondence to Tracey McLaughlin or Michael P. Snyder. is a cofounder, scientific advisor and shareholder of Filtricine, Iollo, January AI, Marble Therapeutics, Next Thought AI, Personalis, Protos Biologics, Qbio, RTHM, SensOmics. is a scientific advisor and equity holder of Abbratech, Applied Cognition, Enovone, M3 Helium, Onza. is a scientific advisor and stock option holder of Jupiter Therapeutics, Mitrix, Neuvivo, Sigil Biosciences, WndrHLTH, Yuvan Research. is a cofounder and stock option holder of Crosshair Therapeutics. is an investor in Repair Biotechnologies. is a cofounder, shareholder and director of Exposomics, Fodsel, InVu Health. is a cofounder and equity holder of Mirvie, NiMo Therapeutics, Orange Street Ventures. is currently an employee of Google. The other authors declare no competing interests. Nature Medicine thanks Ellen Blaak, Clemens Wittenbecher and Tao Zuo for their contribution to the peer review of this work. Primary Handling Editor: Ming Yang, in collaboration with the Nature Medicine team. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. The x axis is time (minute), and the y axis is glucose level (mg/dL). a: Delta glucose peak (relative peak value to baseline) (mg/dL) and return time (min) for different food combinations. CGM curves are extracted into different features and presented with mean and standard errors (the error bar indicates 2 standard errors). Bars are ordered by mean value. The number of CGM curves (each dot) is: rice 115, potatoes 92, bread 99, pasta 65, grapes 98, beans 46, mixed berries 53, rice+fiber 64, rice+protein 61, and rice+fat 60. Extracted CGM features are compared between different carbohydrate meals. Each participant was instructed to eat each meal at least twice on two different days. b: the scatter plot for three exemplar meals visualized with 2 relatively independent features. The x axis is time to peak (the slope from baseline to peak, mg/(dL*min)), and the y axis is delta glucose peak. Each point is one CGM curve. The value was averaged among replicates and normalized by the maximum for that participant. Carb-response-type in this paper was classified based on the average of PPGRs (delta glucose peak) of replicates. Here, we also classified each individual into different carb-response-type based on each single meal replicate (1 and 2) and compared it with the classification based on averaged PPGRs. The y axis indicates the percent of average-based classification to be assigned to the same group based on single replicates. For example, in participants classified to be bread-spiker by the averaged PPGR, most meal replicates still hold the same classification (bread as the highest PPGR meal) (62%), and a small chunk of meal replicates (15%) classified bread as the second highest meal. Asterisks indicate Holm corrected p-value of the t-test (default, two-sided) of the selected pairs, *: pFDR < =0.05, **: pFDR < =0.01. The number of replicates is 3 for each food. Data beyond the whiskers are outliers. Omics samples are colored according to sex (row 1) and hepatic insulin resistant (hepatic IR) states (row 2). Color distinguishes different groups, and each point is one sample. Each column is a different omics. Strong microbiome features are selected (pFDR<0.1) from features of GO, KEGG, metacyc pathway, pfam, and reactions. Asterisks indicate correlation original p-value < 0.05. The partial Spearman correlation was run with additional covariates, sample weight, and two-sided t-tests were run. Metabolic features that are significantly associated (pFDR<0.05) with selected T2D-related clinical measurements (A1C, FBG, OGTT @120 mins, SSPG, disposition index (DI), Incretin effect, hepatic insulin resistance (hepatic IR), HOMA IR, adipose IR) are listed in the row. All clinical features are listed in columns. FDR corrections were implemented for each clinical feature. Lipidomics features that are significantly associated (pFDR<0.05) with selected T2D-related clinical measurements (A1C, FBG, OGTT @120 mins, SSPG, DI, Incretin effect, hepatic IR, HOMA IR, adipose IR) are listed in the row. All clinical features are listed in columns. FDR corrections were implemented for each clinical feature. Left: Root mean squared error (RMSE) of SSPG estimation. The x axis indicates different tested models. Right: Coefficient values of the top predictive features of the cooperative model for SSPG estimation. Coefficients with more than 30% of the absolute value of the maximum features are presented. Positive value contributed to higher SSPG. Each of these models was built 100 times with different train/test splits. The model performance is presented with mean and standard errors (the error bar indicates 2 standard errors). Supplementary Tables 1–10 with captions and titles in the file. Pathway enrichment result for metabolomics, lipidomics and proteomics associated with meal-specific postprandial response. Omics features significantly (P < 0.05) associated with PPGRs (delta glucose peak) to each meal were selected and enriched with GO, KEGG, Reactome and HMDB. The P value was adjusted by FDR for each cluster, within each omic and for each database or ontology (for example, GO pathways enriched from proteins positively correlated with rice PPGRs). Details can be found in the Methods. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Wu, Y., Ehlert, B., Metwally, A.A. et al. Individual variations in glycemic responses to carbohydrates and underlying metabolic physiology. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.sciencedaily.com/releases/2025/06/250603213454.htm'>Millions of new solar system objects to be found and 'filmed in technicolor' -- studies predict</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.sciencedaily.com', 'title': 'ScienceDaily'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-06-04 02:01:23
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>A group of astronomers from across the globe, including a team from the University of Washington and led by Queen's University Belfast, have revealed new research showing that millions of new solar system objects will be detected by a brand-new facility, which is expected to come online later this year. The NSF-DOE Vera C. Rubin Observatory is set to revolutionize our knowledge of the solar system's "small bodies" -- asteroids, comets and other minor planets. Together, this "wide-fast-deep" system will generate 20 terabytes of data every night -- creating an unprecedented time-lapse "movie" of the cosmos over the next 10 years, and an incredibly powerful dataset with which to map the solar system. The team of astronomers, led by Queen's University's Meg Schwamb, created Sorcha, an innovative new open-source software used to predict what discoveries are likely to be made. Sorcha is the first end-to-end simulator that ingests Rubin's planned observing schedule. It applies assumptions on how Rubin Observatory sees and detects astronomical sources in its images with the best model of what the solar system and its small body reservoirs look like today. "Accurate simulation software like Sorcha is critical," said Schwamb, a reader in the School of Mathematics and Physics at Queen's University. Our knowledge of what objects fill Earth's solar system is about to expand exponentially and rapidly." Many of these smaller bodies remain essentially unchanged since the solar system's birth, acting as a fossil record of its earliest days. These objects -- numbering in the tens of millions -- -- provide a powerful window into processes such as the delivery of water and organic material to Earth, the reshaping of planetary orbits by giant planets and the ongoing risk posed by those whose paths bring them near our planet. Beyond just finding these new small bodies, Rubin Observatory will observe them multiple times using different optical filters, revealing their surface colors. Past solar system surveys typically observed with a single filter. "With the LSST catalog of solar system objects, our work shows that it will be like going from black-and-white television to brilliant color," said Joe Murtagh, a doctoral student at Queen's University. 127,000 near-Earth objects -- asteroids and comets whose orbits cross or approach Earth. This will cut the risk of undetected asteroid impact of catastrophic proportions by at least two times, making a tremendous contribution to planetary defense. Over 5 million main-belt asteroids, up from about 1.4 million, with precise color and rotation data on roughly one in three asteroids within the survey's first years. Approximately 1,500-2,000 Centaurs, bodies on short-lived giant planet-crossing orbits in the middle solar system. Most Centaurs will eventually be ejected from the solar system, but a few lucky ones will survive to become short-period comets. Rubin Observatory's LSST is a once-in-a-generation opportunity to fill in the missing pieces of our solar system, said Mario Juric, a member of the Sorcha team and a UW professor of Astronomy. "Our simulations predict that Rubin will expand known small-body populations by factors of 4-9x, delivering an unprecedented trove of orbits, colors and light curves," Juric said. "With this data, we'll be able to update the textbooks of solar system formation and vastly improve our ability to spot -- and potentially deflect -- the asteroids that could threaten Earth." It took 225 years of astronomical observations to detect the first 1.5 million asteroids, and researchers found that Rubin will double that number in less than a year, said Jake Kurlander, a doctoral student at the UW. "Rubin's unparalleled combination of breadth and depth make it a uniquely effective discovery machine," Kurlander said. Siegfried Eggl, an assistant professor of Aerospace Engineering at the University of Illinois Urbana-Champaign added: "Only by debiasing LSST's complex observing pattern can we turn raw detections into a true reflection of the solar system's history -- where the planets formed, and how they migrated over billions of years. Rubin Observatory is scheduled to unveil its first spectacular imagery at its "First Look" event on June 23, offering the world an early glimpse of the survey's power. Full science operations are slated to begin later this year. Note: Content may be edited for style and length. Stay informed with ScienceDaily's free email newsletter, updated daily and weekly. Or view our many newsfeeds in your RSS reader: Keep up to date with the latest news from ScienceDaily via social networks: Tell us what you think of ScienceDaily -- we welcome both positive and negative comments.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            