
<html>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/x-icon" href="../images/favicon.ico">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
        <title id="title">News'n'Clues - SCIENCE Article Summaries - 2025-04-26</title>
        <script src="https://cdn.tailwindcss.com"></script>
        <style>
            .menu { color: #444444; }
            .copyright { margin: 0 auto; }
            p { font-family: serif; }
            body {  background-color: #2c2c2c; font-family: Arial, sans-serif; font-size: 20px; color: #f4f4f4;  }
            a { text-decoration: none; color: #f4f4f4; }
            .section { margin-bottom: 20px; }
            .heading {
                font-size: 2rem;
                font-weight: bold;
                background: linear-gradient(90deg, #fc4535, #1a6198);
                -webkit-background-clip: text;
                -webkit-text-fill-color: transparent;
                text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
                line-height: 1.3; /* Prevents letters from being cut off */
                padding-bottom: 5px; /* Ensures space below the text */
                margin: 30px;
            }
            .hidden {
                display: none;
            }            
            
        </style>
    </head>
    <body>
        <div id="banner" class="w-full">
            <img src="../images/banner.jpg" alt="News Banner">
        </div>

        <div id="title" class="w-full flex items-center" style="background-color: #fc4535;">
          <a href="../index.html"><img src="../images/logo.jpg" alt="News Logo" class="h-auto"></a>
          <div class="flex-grow text-center">
              <div id="title_heading" class="text-4xl font-bold mb-4" style="color:#1a6198;">Article Summaries</div>
          </div>
        </div>
        <div id='category_heading' class="section text-center heading">
            SCIENCE
        </div>
        <div id="articles">
            
                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.popularmechanics.com/science/environment/a64590639/scientist-drill-into-earths-mantle/'>Scientists Drilled So Deep Into the Center of the Earth, They Knocked on the Mantle's Door</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.popularmechanics.com', 'title': 'Popular Mechanics'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-26 15:11:00
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>Gear-obsessed editors choose every product we review. We may earn commission if you buy from a link. If you want to understand the geology of our home planet, studying the mantle is a great place to start. But despite its outsized influence on the planet's geologic processes, scientists have never directly sampled rocks from this immensely important geologic layer. On the south side of this massif is an area known as the Lost City—a hydrothermal field whose vent fluids are highly alkaline and rich in hydrogen, methane, and other carbon compounds. This makes the area a particularly compelling candidate for explaining how early life evolved on Earth. Additionally, it contains mantle rock that interacts with seawater in a process known as “serpentinization,” which alters the rock's structure and gives it a green, marble-like appearance. “We had only planned to drill for 200 meters, because that was the deepest people had ever managed to drill in mantle rock,” Johan Lissenberg, a petrologist at Cardiff University and co-author of the study, told Nature. Andrew McCaig—study co-author and University of Leeds scientist—said in an article from The Conversation that, according to a preliminary analysis of the rock, the core's composition contains a variety of peridotite called harzburgite that forms via partial melting of mantle rock. Both of these rocks then chemically reacted with seawater, changing their composition. Future missions could continue exploring this site near the Atlantis Massif, but sadly, those missions won't include JOIDES Resolution—the NSF declined to fund more core drilling past 2024. This Ancient Larva Still Has Its Brain And Guts An Enormous Desert Used to Be a 138-Foot-Deep Lake The Core of North America Is ‘Dripping' Away This “Battery In a Rock” Changes the Energy Game This Dark Ocean Pit Has a Bleak Weather Forecast Does This Evidence Proves Life is a Simulation? A DNA Mutation Helps Some Fish Survive Deep Waters This Bizarre Fossil Is a Whole New Form of Life</p>
                <br/>
                

                <div id='article_title' class='text-3xl text-gray-300 leading-tight'>
                    <a target='_blank' href='https://www.nature.com/articles/s41467-025-59138-x'>Structural basis for the activation of proteinase-activated receptors PAR1 and PAR2</a>
                </div>
                <div id='article_attribution'  style='mb-1'>
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Source:</span> {'href': 'https://www.nature.com', 'title': 'Nature'}&nbsp;&nbsp;&nbsp;
                    <span class='text-1.5xl font-extrabold text-gray-500 leading-tight'>Published:</span> 2025-04-26 11:57:56
                    <i class="fas fa-chevron-down expand-icon text-gray-500" aria-hidden="true"></i>
                </div>
                <p class='hidden' class='text-gray-500'>You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. Members of the proteinase-activated receptor (PAR) subfamily of G protein-coupled receptors (GPCRs) play critical roles in processes like hemostasis, thrombosis, development, wound healing, inflammation, and cancer progression. Comprising PAR1-PAR4, these receptors are specifically activated by protease cleavage at their extracellular amino terminus, revealing a ‘tethered ligand' that self-activates the receptor. This triggers complex intracellular signaling via G proteins and beta-arrestins, linking external protease signals to cellular functions. To date, direct structural visualization of these ligand-receptor complexes has been limited. Here, we present structural snapshots of activated PAR1 and PAR2 bound to their endogenous tethered ligands, revealing a shallow and constricted orthosteric binding pocket. Comparisons with antagonist-bound structures show minimal conformational changes in the TM6 helix and larger movements of TM7 upon activation. These findings reveal a common activation mechanism for PAR1 and PAR2, highlighting critical residues involved in ligand recognition. Additionally, the structure of PAR2 bound to a pathway selective antagonist, GB88, demonstrates how potent orthosteric engagement can be achieved by a small molecule mimicking the endogenous tethered ligand's interactions. Proteinase-activated receptors (PARs) are a distinct subgroup within the broader family of G protein-coupled receptors (GPCRs) and are essential to a myriad of biological processes1,2,3. These receptors are activated through proteolytic cleavage by various serine proteases, which exposes a ‘tethered ligand' within the receptor's N-terminal region. This tethered ligand subsequently binds intramolecularly to initiate signaling, distinguishing PARs from other GPCRs that are typically activated intermolecularly by soluble ligands4. Like many GPCRs, PAR1 and PAR2 play crucial roles in human physiology and pathology, particularly in processes such as inflammation, coagulation, and immune responses4,5,6. PAR1, the first identified member of the PAR family, has been widely studied for its critical roles in hemostasis and thrombosis7,8. Predominantly expressed on platelets, smooth muscle cells and endothelial cells, PAR1 is primarily activated by thrombin, a key enzyme in the coagulation cascade9,10,11,12. This highlights PAR1's pivotal role in vascular biology and its potential as a therapeutic target for novel antithrombotic therapies designed to reduce thrombotic risk while minimizing the bleeding complications often linked to conventional anticoagulants13. In contrast, PAR2 is predominantly activated by trypsin and tryptase, playing diverse roles in inflammation and immune response14. Unlike PAR1, PAR2's functions are implicated in conditions such as chronic inflammation, pulmonary disorders, and cancer progression15,16. Its ability to respond to multiple proteases underscores its broader physiological significance and complexity, making it an attractive target for pharmaceutical development aimed at treating a range of inflammatory and autoimmune disorders17. Mutational scanning methods were first applied to investigate the mechanism of PARs, revealing that residues from the tethered ligands are in contact with extracellular loop 2 (ECL2) of the receptor18. Subsequent crystal structures of thrombin bound to PAR1's N-terminus revealed their interactions and hypothesized how these interactions may cause allosteric effects on GPCR activation, in addition to proteolysis linked activation of PAR119,20,21. The crystal structure of PAR1 in complex with vorapaxar, a highly selective tricyclic himbacine-derived inhibitor of PAR1, marked a significant milestone in the structural study of PARs22. This work revealed unprecedented details of the interaction between PAR1 and the antagonistic compound. Separately, a study reported the crystal structures of PAR2 in complex with the allosteric antagonists AZ3451 and AZ883823. While these structures provided unparalleled detail, the activation mechanism of PARs and details of tethered ligand-GPCR interactions within the orthosteric pocket remain poorly understood. Building on this foundational knowledge, our research delves further into the structural analysis of PAR1 and PAR2, revealing important details of their activation mechanisms. We have resolved the structures of PAR1 and PAR2 in complex with their endogenous tethered ligands and G proteins, providing details of receptor activation rearrangements. Our analysis reveals that both PAR1 and PAR2 feature shallow, constricted orthosteric binding pockets essential for ligand recognition and receptor activation. This structural characterization not only deepens our understanding of their molecular functionality, but also identifies residues that are critical to ligand binding and receptor activation. Surprisingly, our comparative studies with antagonist-bound structures reveal minimal conformational shifts in the TM6 helix, suggesting a subtle yet effective activation mechanism distinct from the dramatic transformations observed in other GPCRs. These insights are vital for comprehending the specificity of receptor activation and signaling and allow us to propose a unified model for the activation of PARs. To further our understanding of PAR2 druggability, we determined the structure of GB88 bound to an activated PAR2 heterotrimeric G-protein complex. GB88 is a biased antagonist of PAR2 that selectively inhibits the PAR2/Gq/11/Ca2+/PKC signaling pathway, resulting in anti-inflammatory effects in vivo24. At the same time, it acts as an agonist in three other PAR2-activated pathways (cAMP, ERK, Rho) in human cells24. Our structure of PAR2 bound to this versatile molecule reveals it to bind deep within the constricted orthosteric pocket by effectively mimicking key interactions that we observe in our tethered ligand structure of PAR2. Here, we report the structural basis of PAR1 and PAR2 activation, revealing key molecular interactions involved in tethered ligand binding and receptor activation. Our findings identify conserved residues critical for orthosteric pocket interactions and reveal minimal conformational shifts in TM6, paired with distinct TM7 movements, driving a unified activation mechanism for PAR1 and PAR2. Additionally, we present the structure of PAR2 bound to GB88, illustrating its mimicry of endogenous ligand interactions for potent orthosteric engagement. Overall, this work enhances our understanding of ligand-receptor and small molecule-receptor interactions, providing a foundation for designing therapies targeting the PAR1 and PAR2 orthosteric binding sites. To elucidate the agonist-binding and activation mechanisms of PAR1 and PAR2, we focused on their natural agonist, the tethered ligand. We employed consistent strategies in designing constructs for both receptors. Briefly, PAR1 (residues 42-425) and PAR2 (residues 37-397) were cloned into the pFastbac vector following the signal peptide (Supplementary Fig. This arrangement simulates protease cleavage, thereby exposing the first amino acid of the tethered ligand, which is crucial for receptor activation. To stabilize the PAR1 and PAR2 complexes with Gαq proteins, we implemented the NanoBiT tethering strategy by attaching the large BiT (LgBiT) to the C-terminus of each receptor, followed by a Flag tag for purification purposes25. Additionally, we utilized an engineered Gαq protein (GqiN), in which the N-terminus (residue 1–32) of Gαq was replaced with the N-terminus (residue 1–28) of Gαi26. This modification significantly enhances the protein's affinity for the scFv16 antibody, a tool that has been instrumental in elucidating numerous receptor/G-protein complexes. To assemble the PAR1-Gαq and PAR2-Gαq complexes, receptor subunits, Gαq, Gβ, Gγ, and scFv16 were co-expressed in Hi5 insect cells. The complexes were then purified using Flag affinity resin (Supplementary Fig. The structures of the PAR1-Gαq and PAR2-Gαq complexes were resolved via cryo-EM to resolutions of 2.7 Å and 3.1 Å (Figs. Both complexes exhibit a highly similar overall structural fold, adopting a canonical seven-transmembrane helix domain (TMD), with root mean square deviation (RMSD) values of 1.07 Å over 205 Cα atoms. A Cryo-EM density map of PAR1-Gαq-scFv16 with tethered ligand. (PAR1, blue; Gαq, teal; Gβ, yellow; Gγ, purple; scFv16, gray). B Cartoon representation of PAR1-Gαq-scFv16 with tethered ligand bound. Inset: tethered ligand (yellow sticks) is shown within the orthosteric pocket of PAR1 (slabbed surface). The tethered ligand cryo-EM map is shown as blue mesh at 5.0 sigma. C Cartoon representation of PAR1 (blue) with the tethered ligand bound (yellow sticks) viewing from the extracellular side of the membrane with TMs and ECLs labeled. D H-bond interactions between the tethered ligand (yellow sticks) and PAR1 orthosteric residues (orange sticks). A Cryo-EM density map of PAR2-Gαq-scFv16 with tethered ligand. (PAR2, salmon; Gαq, teal; Gβ, yellow; Gγ, purple; scFv16, gray) B Cartoon representation of PAR2-Gαq-scFv16 with tethered ligand bound. Inset: tethered ligand (green sticks) is shown within the orthosteric pocket of PAR2 (slabbed surface). The tethered ligand cryo-EM map is shown as blue mesh at 5.0 sigma. C Cartoon representation of PAR2 (salmon) with the tethered ligand bound (green sticks) viewing from the extracellular side of the membrane with TMs and ECLs labeled. D H-bond interactions between the tethered ligand (green sticks) and PAR2 orthosteric residues (yellow sticks). PAR1 is uniquely activated by thrombin through a specialized proteolytic mechanism that sets it apart from most other G protein-coupled receptors. This activation begins when thrombin cleaves PAR1 at residue R41N-term located within its extracellular N-terminal domain27. This proteolytic event exposes a new sequence motif SFLLR, beginning at S42TL, and acts as the tethered ligand. Our PAR1-Gαq cryo-EM map reveals density for these initial five residues (Fig. They adopt an extended conformation that binds into a shallow pocket at the center of the 7 TM bundle of PAR1, forming contacts with residues from TM1, TM5, TM6, TM7, and ECL2 (Fig. The tethered ligand is nestled into a shallow and narrow pocket with the deepest residue, S42TL, in hydrogen bonds distance with residues H255ECL2 of ECL2 and Y3376.59 of TM6 (Fig. The adjacent residues, F43TL, L44TL, and L45TL, contribute van der Waals interactions with Y3507.32 of TM7, and additional H-bonds with D256ECL2 and L258ECL2, illustrating a complex network of interactions that facilitate the activation of the receptor (Fig. In comparison, PAR2 activation involves a direct mechanism by trypsin, which cleaves the receptor at R36N-term. This cleavage reveals a new N-terminal sequence starting with SLIG, beginning at S37TL, that serves as a tethered ligand28. This ligand then folds back onto the receptor itself, binding intramolecularly and inducing significant conformational changes that are essential for receptor activation. Our cryo-EM density map of the PAR2-Gαq complex has effectively resolved the initial three residues, SLI, showing a similar pattern of interactions (Fig. This tethered ligand also binds to a shallow pocket, engaging in key interactions with residues from TM1, TM5, TM6, TM7, and ECL2 (Fig. Notably, residues S37TL and L38TL play important roles in this binding. S37TL reaches into the base of the pocket, forming putative hydrogen bonds with the main chain carbonyl of H227ECL2 and the sidechain of Y3116.59 while L38TL is involved in a polar interaction with L230ECL2 and forms van der Waals interactions with Y3237.32, as well as forming a putative H-bond with D228ECL2 (Fig. 2D), illustrating the detailed mechanism of ligand-receptor interactions that lead to the activation of PAR2. Alignment of the PAR1 and PAR2 structures reveal that they share highly similar binding modes of the tethered ligands and overall conformation (Supplementary Fig. The structure of PAR1 bound to the antagonist vorapaxar (PDB 3VW7) illustrates how vorapaxar partially occupies the orthosteric site of PAR1, effectively inhibiting its activation by sterically preventing rearrangements necessary for activation22. Vorapaxar stabilizes PAR1 in its inactive conformation through multiple hydrogen bonds and hydrophobic interactions with multiple residues. These interactions hinder the conformational changes required for G protein coupling and physically block the tethered ligand binding site, with the tricyclic ring of vorapaxar occupying the same position as S42TL when the tethered ligand is bound. In our cryo-EM structure, the tethered ligand is demonstrated to be positioned higher than vorapaxar's methylpyridine core and distal chlorophenyl ring, that both penetrate deep within the receptor, sterically obstructing the large conformational rearrangements of bulky tyrosine residues Y3507.32 and Y3537.35, (Fig. 3A–D) and virtually irreversibly inhibiting receptor activation by PAR1's tethered ligand22. A Comparison of the tethered ligand-bound to active PAR1 (blue) and the antagonist vorapaxar-bound to inactive PAR1 (pink; PDB 3VW7). Inset: structural superposition illustrating the relative binding positions of vorapaxar (gray sticks) and the tethered peptide (yellow sticks) to PAR1. B Extracellular view of the comparison between tethered ligand-bound active PAR1 (blue tubes) and vorapaxar-bound inactive PAR1 (pink tubes). The tethered peptide is colored in yellow spheres, while vorapaxar is shown in gray spheres. C Key residues conformations in PAR1 bound to the antagonist vorapaxar. D Key residue conformations in PAR1 bound to the tethered ligand. Vorapaxar is a highly lipophilic molecule, speculated to enter the receptor through the lipid bilayer between TMs 6 and 7. In the vorapaxar-bound inactive state, TM6 and TM7 are spaced apart; upon receptor activation, these helices move towards each other by ~3 Å (Fig. 3A, B), eliminating the space necessary to accommodate vorapaxar's ethyl carbamate tail, and possibly vorapaxar's hypothesized entry into the helical bundle via the membrane. The most significant structural change during activation is an inward movement of the extracellular side of TM5 by ~5.6 Å, narrowing the pocket between TM4 and TM5 that accommodates vorapaxar's fluorophenyl pyridine tail (Fig. In contrast, residues of the tethered ligand bind within the relatively superficial orthosteric pocket formed by TM1, TM5, TM6, TM7, and ECL2 (Fig. Unlike other Class A and B GPCRs, which typically exhibit significant activation-associated movements in the intracellular end of TM6 (such as an ~9 Å outward shift in rhodopsin29 and ~16 Å in GLP1R when engaged with G proteins30), PAR1 shows a more modest 4.7 Å outward movement of TM6 in the active state compared to the inactivated vorapaxar-bound state (Fig. Significant conformational shifts induced by the tethered peptide are observed in Y1833.33, F2715.39, Y3376.59, Y3507.32, and Y3537.35 are also observed. Mutagenesis of these residues to alanine significantly diminishes the potency of the tethered ligand in activating PAR131. In the vorapaxar-bound structure, Y3507.32 is located at a position that is poised to form a hydrogen bond with H255ECL2 on ECL2, stabilizing Y3507.32 and helping to maintain TM7 in the inactive conformation (Fig. However, when bound, the tethered ligand forces these residues downward (Fig. The sidechain of Y3507.32 on TM7 breaks the H-bond with H255 ECL2 and forms an H-bond with Y1833.33 of TM3, leading to the downward movement of TM6 and subsequent receptor activation (Fig. Mutation of Y1833.33 exhibited loss of inhibition by vorapaxar and indicates that interactions between Y1833.33 and Y3507.32 may stabilize an inactive conformation22. Previous structures of PAR2 have revealed an inactive conformation and detailed, distinct allosteric binding mechanisms of PAR2 antagonists. The antagonist AZ8838 (PDB 5NDD) demonstrates slow binding kinetics and binds to an occluded pocket23 below the tethered ligand orthosteric pocket we describe here (Fig. Additionally, AZ3451 (PDB 5NDZ), was found to bind to a distant allosteric site on the outside the helical bundle, preventing necessary structural rearrangements for receptor activation and signaling23,32. A Overlay of the tethered ligand-bound active PAR2 (coral) and the antagonist AZ8838-bound inactive PAR2 (yellow; PDB 5NDD). The tethered peptide is colored in green spheres, while AZ8838 is shown in blue spheres. B Rotated extracellular view of the comparison between tethered ligand-bound active PAR2 (coral) and AZ8838-bound inactive PAR2 (yellow). C Conformations of key residues in inactive PAR2 (yellow) bound to the antagonist AZ8838 (blue) compared to active PAR2 (coral). D Detailed view of specific residues involved in H-bond rearrangements in the inactive (yellow) to active conformation (coral) of PAR2. E Detailed view of TM6 and TM7 highlighting residues that propagate agonist binding rearrangements in the active state (coral) compared to relative position in the inactive state (yellow). In our tethered ligand bound PAR2 structure, we also observe a superficial orthosteric site that is comparable to PAR1 (Supplementary Fig. Analogous to our PAR1-Gαq complex, activated PAR2 exhibits minimal movement of TM6, with a 3.4 Å outward tilt and a 2 Å downwards movement towards the intracellular side. Additionally, TM5 exhibits an ~2.6 Å outward movement to accommodate the helix-5 of the G protein (Fig. The tethered ligand occupies a non-overlapping site to that of AZ8838, which is completely solvent inaccessible in a small, deep binding pocket below the orthosteric site of the tethered ligand (Fig. Concomitant with this key difference, the tethered ligand induces a different conformation of TM6 around the binding pocket, causing an ~2.1 Å shift on the extracellular side of TM6 (Fig. Upon binding at the orthosteric pocket, the sidechain of L38TL sterically pushes Y3237.32 of TM7 downwards, forming a stabilizing H-bond with D228ECL2 via the sidechain hydroxyl group (Fig. Commensurate with activation is the breaking of an H-bond between H227ECL2 and Y1563.33 and the formation of a stabilizing H-bond between the sidechains of Y3267.35 and Y1563.33 (Fig. Mutations in Y3237.32, D228 ECL2, Y1563.33, and Y3267.35 each were shown to have critical impacts on PAR2 activation33,34, which correlates with our structural observations. Once stabilized, Y3267.35 of TM7 sterically pushes TM6 outwards through interactions with L3076.55 of TM6, as well as further propagating the shift of TM7 downward via L3307.39 to L3327.41 leading to movement of I2986.46 further down TM6 (Fig. Our structures reveal how tethered ligand induced reorientations of TM6 and TM7 lead to the transmission of the activation signal through a series of conserved tyrosine residues. They further reveal how binding at the orthosteric site results in coupled activation of G-protein binding on the intracellular side of PARs. PAR1 and PAR2 exhibit similar G-protein coupling profiles, both leading to strong Gαq-mediated signaling pathways35,36. Nevertheless, structural alignments between Gαq in PAR1 and PAR2 complexes reveal notable conformational differences. The α5 helix of Gαq in both receptors shows a tilt of ~7°, and a spatial separation of ~6.4 Å is observed in their respective αN helices (Fig. These findings highlight the intricate and receptor-specific nature of GPCR-G protein interactions, emphasizing that insights from one receptor subtype cannot be directly transferred to another. Our study elucidates G-protein coupling specificities in PAR1 and PAR2, revealing the complex interplay of structural elements that govern these interactions. Comparative analysis of the Gαq conformation in the α5 helix (B) and αN domain (C). Our agonist bound structures and comparison with antagonists bound structures of PAR1 and PAR2 reveals a shared activation mechanism involving the downward movement of TM7 and subsequent displacement of TM6 (Fig. The key ligand-induced driver of TM7 movement is the downward force exerted by the second residue of the tethered ligand (L38TL in PAR1 and F43TL in PAR2) directly on Y3507.32 in PAR1 and Y3237.32 in PAR2. This then promotes the sliding downward movement of TM6 through concomitate downward force on Y3537.35 and Y3267.35 in PAR1 and 2 respectively. The downward shift of TM6 and TM7 sets in motion a cascade of rearrangements of other bulky sidechains, most notably F1823.32, Y1833.33, M1863.36 and the PIF motif residue I1903.40 in PAR1 and similarly the corresponding conserved residues F1553.32, Y1563.33, M1593.36 and I1633.40 in PAR2. The rearrangements of these motifs are highly analogous between PAR1 and PAR2, concomitant with their critical role in stabilizing the G-protein binding interface. Correspondingly, in PAR1, mutations of these residues were shown to significantly impede the activation of PAR1 and disrupted G protein signaling31. Schematic representation of PAR1 and PAR2 in antagonist-bound (left; A), ligand-free (middle; B), and agonist-bound (right; C) states. The sidechains of two mechanistically important tyrosines are shown in stick representation. Conformational flexibility in TM7, TM5 and TM6 in the three states are indicated with red arrows, while conformational movements of the tyrosines upon agonist and antagonist binding are highlighted with black arrows in (C). The relative location of the PIF/Y, DRF/NRY and DPxxY motifs are indicated with blue arrows. GB88 is an intriguing molecule, that was initially reported as a PAR2 selective antagonist37. The true functional profile of the compound has recently been shown to be nuanced, demonstrating partial agonist properties against several PAR2 signaling arms, including Gαq/11 directed pathways, and the ability to internalize the receptor post activation24. To gain insight into GB88's ability to act as a small molecule PAR2 agonist, we determined the structure of the co-complex formed between GB88, PAR2, and heterotrimeric G-proteins using a chimeric Gαq (Fig. Model of GB88:PAR2-Gαq-scFv16 complex shown in cartoon representation. C Cartoon representation of PAR2 (green) with GB88 bound (cyan sticks) viewing from the extracellular side of the membrane with TMs and ECLs labeled. D GB88 (cyan sticks) is shown within the orthosteric pocket of PAR2 (slabbed surface). The GB88 cryo-EM map is shown as blue mesh at 5.0 sigma (E). GB88 (cyan) bound to the orthosteric pocket of PAR2 highlighting key residues (yellow) and H-bonds formed between the ligand and receptor. The tethered ligand is shown in green with residues labeled also in green. Our structure reveals that GB88 binds with its isoxazole group deeply situated within the orthosteric pocket of PAR2 (Fig. 7C–E), burying 73.2% of the molecule's accessible surface area (590 Å2) and interfacing with 22 residues from ECL2, TM1, TM6 and TM7 of PAR2. Many of these interactions are contributed by ECL2 residues in the region 217-236. Intriguingly, GB88 overlays very closely with the bound tethered ligand (Fig. The isoxazole group of GB88 has been shown to confer antagonist properties38. In our structure, the isoxazole oxygen accepts an H-bond from H3106.58 with the nitrogen accepting an H-bond from Y3116.59. Adjacent to this, the bulky cyclohexane is positioned where L38TL of the tethered agonist resides, pushing L69N-term and shifting the top of TM1 outwards by ~2-3 Å when compared to the tethered ligand bound structure, to accommodate the bulky C-terminus spiroindene-piperdidine. Of note is the almost identical interaction with the activation residue Y3237.32, which propagates TM7 movement during receptor activation. Similar to interactions formed by the tethered ligand agonist, the backbone amides of GB88 form H-bond interactions by donating to the carbonyl oxygen of D228ECL2 and accepting H-bonds from both L230 ECL2 and S320 ECL3. Previous studies demonstrated key SAR around both of GB88's backbone amides38. Our structure supports the proposed hypothesis that this is due to preference for maintaining a preferred conformation for hydrogen bonding to receptor residues. A robust understanding of the molecular basis of GPCR agonism and antagonism, as well as orthosteric pocket information and ligand-receptor interactions are critical for leveraging rational SBDD during drug discovery campaigns. Until now, such information has largely not been available for the PARs. In this study, we first sought to advance our understanding of the structural mechanisms underlying the activation of PAR1 and PAR2 by their endogenous tethered ligand. Our cryo-EM structures provide a detailed view of the ligand-binding sites in PAR1 and PAR2, outlining residues that define the location and dimensions of their relative orthosteric binding pockets. Unlike previous findings of PAR1 and PAR2 in complex with small molecules17,18 the binding pockets of these proteins for endogenous ligands are shallow and much closer to the extracellular side of the receptor than previous hypothesized32. Unlike most Class A and Class B GPCRs, the movement of TM6 in PAR1 and PAR2 is surprisingly minimal, indicating a G protein coupling mechanism that is so far unique to the PARs. Both our endogenous peptide bound structures reveal a common activation mechanism, where the tethered ligand sterically pushes two key tyrosine residues, which are conserved across all 4 human PARs, downward, leading to TM6 movement and subsequent activation. Given this, it is highly likely that all four PARs share a conserved orthosteric activation mechanism triggered by downward movement of conserved tyrosine residues on TM7 and TM6. The constricted nature of the orthosteric pocket in the region of these bulky agonist switches rationalizes the limited successes in discovery of a clinically successful small molecule antagonist against PAR2 activity. Vorapaxar appears to circumvent triggering orthosteric activation of PAR1, as it is hypothesized to bind the receptor laterally through the membrane22. The crystal structure demonstrates that vorapaxar binds to PAR1 in an atypical manner, using a superficial binding pocket with minimal solvent exposure. Once bound, it sterically prevents the movement of tyrosine switches to initiate activation, explaining how a small molecule can achieve receptor inactivation22. Unlike PAR1, the search for a PAR2 selective orthosteric antagonist has proven extremely difficult17. Here, we present the structure of PAR2 bound to the small molecule GB88, an intriguing orthosteric engager that selectively inhibits the PAR2/Gq/11/Ca2 + /PKC signaling pathway, resulting in anti-inflammatory effects in vivo24. Potent orthosteric engagement is achieved by GB88's ability to mimic many of the same interactions that the tethered agonist forms with orthosteric pocket residues located on ECL2, TM1, TM6 and TM7. A structural understanding of GB88's unique pharmacology holds the potential to rationally develop improved small molecule engagers that can target specific PAR2-linked signaling pathways implicated in diseases without disrupting beneficial PAR2 signaling in normal physiology. During review of this manuscript, two cryo-EM structures of PAR1 bound to the tethered ligand agonist in complex with Gq and Gi heterotrimers were reported31. Overall, these structures are in strong agreement with this study in several key findings, including similar overall structure, proposed activation mechanism, and key residues involved in tethered ligand binding. The human PAR1 and PAR2 gene were subcloned into the pFastBac plasmid with a prolactin-signal peptide sequence on its N-terminus and the LgBiT fused to its c-terminus followed by a Flag tag for purification. HiBiT was fused to the c-terminus of human Gβ1 and cloned into pFastBac plasmid as described in the VIP1R paper25 The N-terminus (residue 1-32) of human Gαq was replaced by the N-terminus of Gi (residue 1–28) and subcloned into pFastBac plasmid. The wild-type human Gγ2 was cloned into pFastBac plasmid. The scFv16 that encodes the single-chain variable fragment of mAb16 was subcloned into pFastBac plasmid. Bacmid preparation and virus production were performed according to the Bac-to-Bac baculovirus system manual (Gibco, Invitrogen). For expression, the Spodoptera frugiperda (Sf9) cells at density of 2 × 106 cells per ml were co-infected with baculovirus encoding the PAR1-LgBiT-Flag, GqiN, Gβ, Gγ, scFv16 and Ric8A protein at a ratio of 1:500 (virus volume vs cells volume). Cells were harvested 48 h after infection. Cell pellets were resuspended in 20 mM Hepes buffer (pH 7.5), 100 mM NaCl, and homogenized by douncing ~30 times. Apyase was added to the lysis at a final concentration of 0.5 mU/ml. To keep the complex stable, the lysate was incubated at room temperature for 1 h with flipping. Then, lauryl maltose neopentyl glycol (LMNG, Anatrace) was added at the final concentration of 0.5% to solubilize the membrane at 4 °C for 2 h. Then the lysis was ultracentrifuged at 56,000 g (45,000 rpm) at 4 °C for 40 min. The supernatant was collected and incubated with anti-FLAG M2 resin for 2 h. The resin was washed with a buffer of 25 mM Hepes (pH 7.5), 100 mM NaCl and 0.001% LMNG, and 0.0005% cholesteryl hemi-succinate (CHS), then eluted with the same buffer plus 100 µb/ml FLAG peptide. The elution was concentrated and separated on a Superdex 200 Increase 10/300 GL (GE health science) gel infiltration column with a buffer of 25 mM Hepes (pH 7.5), 100 mM NaCl, and 0.001% LMNG. The peak corresponding to the PAR1-Gαq and PAR2-Gαq complex was concentrated at about 3 mg/ml for later cryo-EM grid preparation. Three microliters of PAR1-Gαq and PAR2-Gαq complex (including the GB88 preparation) sample at ~3 mg/ml was applied to a glow discharged UltraFoil R1.2/1.3 grids (Quantifoil GmbH). The grids were vitrified in liquid ethane using Vitrobot Mark IV (Thermo Fisher Scientific) instrument with the settings of blot force 5, blot time of 5 s, relative humidity of 100%, chamber temperature of 4 °C. Single-particle cryo-EM data were collected using EPU package on a Thermo Fisher Glacios transmission electron microscope operating at 200 keV using parallel illumination conditions. Micrographs were collected using a Falcon 4i direct electron detector with corrected pixel size of 0.69 Å/pix, a total electron exposure of 45 e– /Å–2, with a defocus range of −0.8 to −2 μm. The cryoSPARC Live application of cryoSPARC v. 4 (Structura Biotechnology) was used to streamline the movie processing, CTF estimation, particle picking, and 2D classification. Preprocessing involved anisotropic motion correction and local CTF estimation. The data were curated by only retaining images with better than 4.5 Å CTF fit resolution. Once a small set of particles was extracted and 2D class averages were obtained, the good 2D classes were used as templates for picking on the entire dataset. Then ab initio 3D reconstruction was carried out in cryoSPARC v. 4 and one out of three classes was giving the reconstruction of the designed complex from ~1.9 million and ~0.4 million particles for PAR1 and PAR2, respectively. Multiple rounds of iterative homogenous refinement, heterogenous refinement, and nonuniform, coupled with global CTF refinements gave the final 3D reconstructions at 2.74 Å (PAR1) and 3.1 Å (PAR2) resolution, respectively (0.143 gold standard FSC with correction of masking effects)39. DeepEMHancer was used to improve the map quality40. A similar strategy was utilized for the PAR2-GB88 complex, where ~4 million particles were initially picked and ~1 million particles were kept from the suitable 2D classes (Supplementary Table 1). During pre-processing with cryoSPARC Live, templated from the PAR2 dataset were used for particle picking. These picks were used in 2 cycles of heterogeneous refinement using models from the PAR1 dataset as seed. The best set of particles was then used for an ab initio reconstruction. Particles from the best model were used in homogeneous and non-uniform refinement, resulting in a 2.9 Å resolution map. DeepEMHancer was further utilized to help with the map quality. Starting models for PAR1, PAR2, Gαq, Gβ1γ2, and scFv16 were based on Protein Data Bank (PDB) entries 3VW722, 5NDD23, 6OIJ41, respectively. All models were docked into the electron microscopy density map using UCSF ChimeraX42. The resulting model was subjected to iterative manual adjustment using Coot43, followed by a Rosetta cryo-EM refinement at relax model and Phenix real_space refinement44. The model statistics were validated using MolProbity. All structure alignments described were done by aligning the GPCR only instead of the whole complex in UCSF Chimera. Structural figures were prepared in UCSF ChimeraX and PyMOL (https://pymol.org/2/). The statistics for data collection and refinement are included in Supplementary Table 1. Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article. The cryo-EM maps have been deposited in the Electron Microscopy Data Bank (EMDB) under accession codes under accession codes EMD-46571 (PAR1-Gαq), EMD-46448 (PAR2-Gαq) and EMD-47687 (PAR2-GB88). The coordinates have been deposited in the Protein Data Bank (PDB) under accession codes 9D4Z (PAR1-Gαq), 9D0A (PAR2-Gαq) and 9E7R (PAR2-GB88). confirmation of the intramolecular tethered liganding hypothesis and discovery of an alternative intermolecular liganding mode. & Coughlin, S. R. Molecular cloning of a functional thrombin receptor reveals a novel proteolytic mechanism of receptor activation. Coughlin, S. R. Thrombin signalling and protease-activated receptors. Macfarlane, S. R., Seatter, M. J., Kanke, T., Hunter, G. D. & Plevin, R. Proteinase-activated receptors. Blackhart, B. D. et al. Ligand cross-reactivity within the protease-activated receptor family. Landis, R. C. Protease activated receptors: clinical relevance to hemostasis and inflammation. Integration of endothelial protease-activated receptor-1 inflammatory signaling by ubiquitin. Leger, A. J., Covic, L. & Kuliopulos, A. Protease-activated receptors in cardiovascular diseases. Coughlin, S. R. Protease-activated receptors in hemostasis, thrombosis and vascular biology. Lin, H., Liu, A. P., Smith, T. H. & Trejo, J. Cofactoring and dimerization of proteinase-activated receptors. & Steinhoff, M. Proteinase-activated receptor-2 in the skin: receptor expression, activation and function during health and disease. Bar-Shavit, R. et al. Protease-activated receptors (PARs) in cancer: novel biased signaling and targets for therapy. Zhao, P., Metcalf, M. & Bunnett, N. W. Corrigendum: biased signaling of protease-activated receptors. Yau, M. K., Liu, L. & Fairlie, D. P. Toward drugs for protease-activated receptor 2 (PAR2). Nanevicz, T. et al. Mechanisms of thrombin receptor agonist specificity. Chimeric receptors and complementary mutations identify an agonist recognition site. & Di Cera, E. Crystal structure of thrombin bound to the uncleaved extracellular fragment of PAR1. Gandhi, P. S., Chen, Z., Mathews, F. S. & Di Cera, E. Structural identification of the pathway of long-range communication in an allosteric enzyme. Mathews, I. I. et al. Crystallographic structures of thrombin complexed with thrombin receptor peptides: existence of expected and novel binding modes. Zhang, C. et al. High-resolution crystal structure of human protease-activated receptor 1. Structural insight into allosteric modulation of protease-activated receptor 2. Suen, J. Y. et al. Pathway-selective antagonism of proteinase activated receptor 2. Duan, J. et al. Cryo-EM structure of an activated VIP1 receptor-G protein complex revealed by a NanoBiT tethering strategy. Xia, R. et al. Cryo-EM structure of the human histamine H(1) receptor/G(q) complex. Ludeman, M. J. et al. PAR1 cleavage and signaling in response to activated protein C and thrombin. The PAR2 signal peptide prevents premature receptor cleavage and activation. Kang, Y. et al. Cryo-EM structure of human rhodopsin bound to an inhibitory G protein. Zhang, Y. et al. Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein. Kennedy, A. J. et al. Protease-activated receptor-2 ligands reveal orthosteric and allosteric mechanisms of receptor inhibition. Structural characterization of agonist binding to protease-activated receptor 2 through mutagenesis and computational modeling. Suen, J. Y. et al. Mapping transmembrane residues of proteinase activated receptor 2 (PAR) that influence ligand-modulated calcium signaling. & Pin, J. P. Interaction of protease-activated receptor 2 with G proteins and beta-arrestin 1 studied by bioluminescence resonance energy transfer. Heuberger, D. M. & Schuepbach, R. A. Protease-activated receptors (PARs): mechanisms of action and potential therapeutic modulators in PAR-driven inflammatory diseases. Suen, J. Y. et al. Modulating human proteinase activated receptor 2 with a novel antagonist (GB88) and agonist (GB110). Yau, M. K. et al. PAR2 Modulators Derived from GB88. 3DFlex: determining structure and motion of flexible proteins from cryo-EM. Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. Maeda, S., Qu, Q., Robertson, M. J., Skiniotis, G. & Kobilka, B. K. Structures of the M1 and M2 muscarinic acetylcholine receptor/G-protein complexes. Pettersen, E. F. et al. UCSF ChimeraX: Structure visualization for researchers, educators, and developers. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Takeda Development Center Americas, Inc, 9625 Towne Centre Drive, San Diego, CA, USA Zongyang Lyu, Xiaoxuan Lyu, Andrey G. Malyutin, Guliang Xia, Daniel Carney, Vinicius M. Alves, Matthew Falk, Nidhi Arora, Hua Zou, Aaron P. McGrath & Yanyong Kang You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar You can also search for this author inPubMed Google Scholar Correspondence to Aaron P. McGrath or Yanyong Kang. All authors are current or former employees of Takeda and own stock/stock options in the company. Nature Communications thanks Fei Xu and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Open Access  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/4.0/. Structural basis for the activation of proteinase-activated receptors PAR1 and PAR2. Anyone you share the following link with will be able to read this content: Sorry, a shareable link is not currently available for this article. Provided by the Springer Nature SharedIt content-sharing initiative Sign up for the Nature Briefing newsletter — what matters in science, free to your inbox daily.</p>
                <br/>
                

        </div>

        <script>
            // Get all article attribution elements
            const articleAttributions = document.querySelectorAll('#article_attribution');

            // Add an event listener to each attribution element
            articleAttributions.forEach(attribution => {
            attribution.addEventListener('click', () => {
                // Get the next paragraph element (the article text)
                const articleText = attribution.nextElementSibling;

                // Toggle the visibility of the article text
                articleText.classList.toggle('hidden');

                // Toggle the expand icon
                const expandIcon = attribution.querySelector('.expand-icon');
                expandIcon.classList.toggle('fa-chevron-down');
                expandIcon.classList.toggle('fa-chevron-up');
            });
            });    
        </script>

        <footer class="text-center text-sm text-gray-500 mt-12">
            <div class="inline-block align-middle">
                <a href="https://www.youtube.com/@news_n_clues" target="_blank" rel="noopener noreferrer"
                    class="flex items-center gap-2 text-red-600 hover:text-red-700 font-semibold transition duration-300 ease-in-out">
                    Watch Daily <span class="italic">News'n'Clues</span> Podcast on
                    <img src="../images/yt.png" width="16" height="16">
                </a>
            </div>
            <div>
                <a href="mailto:newsnclues@gmail.com?subject=News'n'Clues Aggregator Inquiry">SoftMillennium
                    <script>document.write(new Date().getFullYear());</script>
                </a>
                <!--
            <b>Copyright &copy; <script>document.write(new Date().getFullYear());</script> - <a href='mailto:newsnclues@gmail.com?subject=News Aggregator Inquiry'>News And Clues</a></b>
            -->
            </div>
        </footer>
    </body>
</html>
            